<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>25201117</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	
	<br/>
	<div>
		

			
			<h2>文献标题:Genome-wide annotation of the soybean WRKY family and functional characterization of genes involved in response to <italic> Phakopsora pachyrhizi </italic> infection</h2>
			<h3>PMID:25201117</h3>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bencke-Malato </surname>  <given-names> Marta </given-names>  </name>  <address>  <email> mbencke@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cabreira </surname>  <given-names> Caroline </given-names>  </name>  <address>  <email> carol.cabreira@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wiebke-Strohm </surname>  <given-names> Beatriz </given-names>  </name>  <address>  <email> strohm@sinos.net </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000fc;cker-Neto </surname>  <given-names> Lauro </given-names>  </name>  <address>  <email> laurobucker@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mancini </surname>  <given-names> Estefania </given-names>  </name>  <address>  <email> soyestepi@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Osorio </surname>  <given-names> Marina B </given-names>  </name>  <address>  <email> miaosorio@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Homrich </surname>  <given-names> Milena S </given-names>  </name>  <address>  <email> mihomrich@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Turchetto-Zolet </surname>  <given-names> Andreia Carina </given-names>  </name>  <address>  <email> aturchetto@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Carvalho </surname>  <given-names> Mayra CCG </given-names>  </name>  <address>  <email> mayragallo@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stolf </surname>  <given-names> Renata </given-names>  </name>  <address>  <email> renatastolf@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weber </surname>  <given-names> Ricardo LM </given-names>  </name>  <address>  <email> rlmweber@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Westergaard </surname>  <given-names> Gast&#x000f3;n </given-names>  </name>  <address>  <email> gastongw@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Castagnaro </surname>  <given-names> At&#x000ed;lio P </given-names>  </name>  <address>  <email> atilio@eeaoc.org.ar </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo V </given-names>  </name>  <address>  <email> ricardo@cnpso.embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar C </given-names>  </name>  <address>  <email> francmg@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Margis-Pinheiro </surname>  <given-names> M&#x000e1;rcia </given-names>  </name>  <address>  <email> marcia.margis@ufrgs.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Bodanese-Zanettini </surname>  <given-names> Maria Helena </given-names>  </name>  <address>  <email> mhbzanettini@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Programa de P&#x000f3;s-Gradua&#x000e7;&#x000e3;o em Gen&#x000e9;tica e Biologia Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Porto Alegre, Brazil  </aff>  <aff id="Aff2">  <label/> Instituto de Agrobiotecnologia Rosario SA, Rosario, Argentina  </aff>  <aff id="Aff3">  <label/> Empresa Brasileira de Pesquisa Agropecu&#x000e1;ria (Embrapa Soja), Londrina, Brazil  </aff>  <aff id="Aff4">  <label/> Estaci&#x000f3;n Experimental Agroindustrial Obispo Colombres (EEAOC), Tucum&#x000e1;n, Argentina  </aff>
				<br/> 
				年份：<day> 29 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Some of the genes that presented differential expression profiles in response to the fungus were randomly selected from each classification group for more detailed analyses. <italic> <span class="gene">GmWRKY27</span> </italic> (<span class="gene">Glyma15g00570</span>) and <italic> <span class="gene">GmWRKY125</span> </italic> (<span class="gene">Glyma09g41050</span>) were differentially expressed in three of the four experiments, while <italic> <span class="gene">GmWRKY56</span> </italic> (<span class="gene">Glyma08g23380</span>), <italic> <span class="gene">GmWRKY106</span> </italic> (<span class="gene">Glyma07g02630</span>) and <italic> <span class="gene">GmWRKY20</span> </italic> (<span class="gene">Glyma08g02580</span>) in the two microarrays. <italic> <span class="gene">GmWRKY139</span> </italic> (<span class="gene">Glyma13g44730</span>), <italic> <span class="gene">GmWRKY46</span> </italic> (<span class="gene">Glyma05g36970</span>), <italic> <span class="gene">GmWRKY57</span> </italic> (<span class="gene">Glyma18g44560</span>) were also analyzed because they were closely related to at least one of the genes evaluated above. Interestingly, none of these genes was expressed in rust infection lesions at ten days after fungus inoculation (RNA-Seq).To determine which soybean <italic> WRKY </italic> genes are involved in plant defense against <italic> P. pachyrhizi </italic> infection, we performed a series of analyses to examine their expression patterns after infection. We initially compared the microarray data available in the literature [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ] with the results obtained from two additional experiments: SuperSAGE and RNA-Seq. Many genes were differentially expressed in only one library, while a few of them showed differential expression in more than one library. The modulation in the transcript levels of eight genes was validated, showing the reliability of data mining. The similar expression patterns in response to <italic> P. pachyrhizi </italic> infection was observed among closely related genes (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ), such as <italic> <span class="gene">GmWRKY61</span> </italic> (<span class="gene">Glyma06g15220</span>) and <italic> <span class="gene">GmWRKY21</span> </italic> (<span class="gene">Glyma04g39650</span>), <italic> <span class="gene">GmWRKY143</span> </italic> (<span class="gene">Glyma14g11920</span>) and <italic> <span class="gene">GmWRKY63</span> </italic> (<span class="gene">Glyma17g33920</span>), <italic> <span class="gene">GmWRKY106</span> </italic> and <italic> <span class="gene">GmWRKY56</span> </italic> , <italic> <span class="gene">GmWRKY58</span> </italic> (<span class="gene">Glyma04g40130</span>) and <italic> <span class="gene">GmWRKY97</span> </italic> (<span class="gene">Glyma06g14720</span>). This similar expression pattern suggests that these genes might share similar functions in <span class="gene">disease resistance</span>. The redundant function of <italic> GmWRKY </italic> genes might be beneficial in protecting the cell or organism under various stress conditions and eliciting multiple pathways that lead to the wide array of physiological responses that occur following pathogen infections [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ].Global expression data have suggested that the timing and the degree of induction of the defense pathway are determinants for the induction of soybean resistance to <italic> P. pachyrhizi </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. In our study, the induced expression of <italic> <span class="gene">GmWRKY20</span>, <span class="gene">GmWRKY27</span>, <span class="gene">GmWRKY46</span>, <span class="gene">GmWRKY57</span> </italic> , <italic> <span class="gene">GmWRKY56</span>, <span class="gene">GmWRKY106</span>, <span class="gene">GmWRKY125</span> </italic> and <italic> <span class="gene">GmWRKY139</span> </italic> in response to <italic> P. pachyrhizi </italic> was earlier and <bold> / </bold> or stronger in the resistant genotype. The expression of most genes analyzed peaked at 12 hai in the resistant genotype; therefore, we propose that these genes might be involved in non-specific defense responses. Van de Mortel et al. [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and Schneider et al. [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ] reported that <italic> P. pachyrhizi </italic> infections induce biphasic global expression. Gene expression initially peaked at 12 hai, which corresponded with the early infection processes of appressoria formation and epidermal cell penetration. The authors suggested that this peak corresponded to a non-specific defense response similar to pathogen-triggered immunity. A second phase of gene expression, which began at 72 hai and continued until 288 hai, is coincident with haustoria formation and effector protein secretion. The authors suggested that this response is consistent with the activation of <italic> RPP2 </italic> and <italic> RPP3 </italic> -mediated resistance. It has been shown that gene expression is rapid and increased in the incompatible interaction [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ].The closely related genes <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> are putative orthologues of <italic> AtWRKY40, AtWRKY18 </italic> and <italic> At </italic> WRKY60 <italic> Arabidopsis </italic> genes. In both species, these genes were classified into group IIa. The three <italic> Arabidopsis </italic> WRKYs are involved in stress responses, which include resistance against the bacteria <italic> Pseudomonas syringae </italic> and fungus <italic> Botrytis cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> , <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. <italic> At </italic> WRKY18 is a salicylic acid-induced gene that positively regulates SAR [ <xref ref-type="bibr" rid="CR63"> 63 </xref> , <xref ref-type="bibr" rid="CR64"> 64 </xref> ] and modulates PR gene expression; <italic> At </italic> W <italic> RKY18 </italic> overexpression increases resistance to <italic> P. syringae </italic> [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. <italic> At </italic> WRKY40 and <italic> At </italic> WRKY60 proteins antagonize <italic> At </italic> WRKY18 during <italic> P. syringae </italic> infection. The gain or loss of gene function in single, double or triple combination mutants resulted in increased susceptibility to <italic> B. cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. Some rice, barley and <italic> Brassica napus </italic> WRKY members from group IIa are also involved in the response to fungal and bacterial pathogens, as demonstrated using expression studies. O <italic> sWRKY62 </italic> and <italic> OsWRKY76 </italic> are upregulated in <italic> Magnaporthe grisea </italic> infected-leaves <italic> and </italic> downregulated in <italic> Xanthomonas oryzae </italic> -inoculated leaves [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. <italic> HvWRKY1 </italic> and <italic> HvWRKY2 </italic> play an important role in response to <italic> Blumeria graminis </italic> infection [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ], and <italic> BnWRKY18 and BnWRKY40 </italic> play a role in the response to <italic> Sclerotinia sclerotiorum </italic> and <italic> Alternaria brassicae </italic> infections [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ].Most available information concerning soybean gene function is based on data obtained from heterologous expression systems. However, as the activity of many proteins frequently depends on specific interactions that are only found in homologous backgrounds, the present study was based on a homologous expression system. An RNA interference approach was used for the silencing of four soybean homologous genes ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ). The quadruple silencing is an advantage because a single knockout of transcription factors rarely results in altered phenotypes due to functional redundancy among closely related members [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. The transgenic RNAi line used in this study generated a significant reduction in the transcript levels of the four target genes. When infected with <italic> P. pachyrhizi </italic> , the transgenic line showed increased susceptibility to the fungus. Taken together, the results strongly suggest that at least one of the four genes might be involved in the soybean resistance phenotype.Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] silenced 64 soybean WRKYs individually using virus-induced gene silencing (VIGS) to test their involvement in <italic> Rpp2- </italic> mediated resistance against <italic> P. pachyrhizi </italic> infection. Three of these genes ( <italic> GmWRKY45 </italic> , <italic> GmWRKY40 </italic> and <italic> GmWRKY36 </italic> ) compromised the resistance phenotype when silenced. Phenotypic alterations were not evidenced when <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> genes were individually silenced <italic> . </italic> However, in the present study, an increased susceptibility to <italic> P. pachyrhizi </italic> infection was observed in the quadruple-silenced ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span>, <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ) line, suggesting that this phenotype is a consequence of <italic> <span class="gene">GmWRKY27</span> </italic> and/or <italic> <span class="gene">GmWRKY139</span> </italic> silencing. Moreover, the four genes analyzed in this study could also play a synergistic role in the pathogenic defense response.A previous study showed that <italic> <span class="gene">GmWRKY27</span> </italic> is also strongly induced under conditions of drought and salt stress in the soybean [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Altogether, these data suggest that this gene is probably involved in a non-specific response that occurs upstream of biotic and abiotic stress defense routes, in contrast with the specific <italic> Rpp2- </italic> response of the genes identified by Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] in response to the fungal infection.a) SuperSAGE: The libraries were constructed using the leaves of a soybean resistant genotype (PI561356), which carries the <italic> Rpp1 </italic> resistance gene, infected with <italic> P. pachyrhizi </italic> vs. uninfected leaves (mock inoculation/control) collected at 12, 24 and 48 hours after inoculation (hai). A Plant RNeasy kit (Qiagen) was used for RNA extraction and equal amounts of RNA from each sample were used to construct the RNA pools. The libraries (inoculated and mock) were constructed at GenXPro GmbH (Frankfurt, Germany) using previously described methods [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ] and subsequently sequenced using the Illumina Genome Analyzer IIx. The SuperSAGE tags were analyzed using the DiscoverySpace software v.4.01 [ <xref ref-type="bibr" rid="CR79"> 79 </xref> ] to identify unique (unitags) and differentially expressed tags (p&#x02009;&#x02264;&#x02009;0.05). The libraries were constructed as part of the GenoSoja project (Brazilian Soybean Genome Consortium), and the results are available in the LGE (Laborat&#x000f3;rio de Gen&#x000f4;mica e Express&#x000e3;o, UNICAMP) Soybean Genome database [ <xref ref-type="bibr" rid="CR80"> 80 </xref> ] for members of the consortium.b) RNA-Seq of lesion LCM (Laser Capture Microdissection): foliar segments (1 cm <sup> 2 </sup> ) containing <italic> P. pachyrhizi </italic> lesions from two soybean resistant (PI561356) and susceptible BRS231 [ <xref ref-type="bibr" rid="CR81"> 81 </xref> ] genotypes at the V2 growth stage were collected at 10 days after infection. The leaf segments were immediately fixed on ice in Farmer&#x02019;s solution [ <xref ref-type="bibr" rid="CR82"> 82 </xref> ], dehydrated and embedded on paraffin in accord with the methods of Cai and Lashbrook [ <xref ref-type="bibr" rid="CR83"> 83 </xref> ]. Serial sections of 12-&#x003bc;m in thickness were generated using a rotary microtome and transferred to microscope membrane slides. Twenty sections containing a variable number of rust lesions were prepared for each biological replicate/treatment. The PixCell II LCM system (Arcturus) and CapSure Macro LCM (Arcturus) were used to collect the foliar cells within the lesion. Total RNA was extracted using the PicoPure RNA Isolation Kit (Arcturus) from the cells collected at a variable number of infection sites for each biological replicate. The synthesis of cDNA was conducted, and high-performance paired-end (108 bp) sequencing was performed on the Illumina genome analyzer GAAllx. Low-quality RNA-Seq reads were discarded. The reads (a total of 86,301,242) were aligned against the soybean genome, and the corresponding genes were predicted using the TopHat [ <xref ref-type="bibr" rid="CR84"> 84 </xref> ] and SOAP2 [ <xref ref-type="bibr" rid="CR85"> 85 </xref> ] alignment programs. Gene expression was calculated using the FPKM (fragments per kilobase of exon per million fragments mapped) value [ <xref ref-type="bibr" rid="CR86"> 86 </xref> ]. To identify differentially expressed genes, a pair-wise comparison between the FPKM values of both genotypes was performed using a <italic> t </italic> -test at the 99% confidence level. This library was constructed as part of the Biotecsur Consortium and the results are available [ <xref ref-type="bibr" rid="CR87"> 87 </xref> ] for members of the consortium.Soybean plants were grown in a pot-based system maintained in greenhouse conditions at 28&#x02009;&#x000b1;&#x02009;1&#x000b0;C under a 16/8 h light/dark cycle with a light intensity of 22.5 &#x003bc;Em <sup> -2 </sup> s <sup> -1 </sup> in Embrapa Soja, Londrina, PR, Brazil. The Embrapa-48 genotype, which develops a &#x0201c;tan&#x0201d; lesion [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ], was used as the susceptible standard, and the PI561356 genotype, which carries the <italic> Rpp1 </italic> resistance gene [ <xref ref-type="bibr" rid="CR88"> 88 </xref> ], was used as the resistant standard. ASR isolated from Brazilian fields was maintained in a susceptible cultivar. Spores harvested from leaves exhibiting sporulating uredinia and diluted in distilled water containing 0.05% Tween-20 to a final concentration of 3 &#x000d7; 10 <sup> 5 </sup> spores/mL. The spore suspension was sprayed onto plantlets at the V2 developmental stage. The same solution without spores was used for the mock inoculation. Subsequently, the water-misted bags were placed over each pot for one day. One trifoliate leaf from each plant was collected at 1, 12, 24, 48, 96 and 192 hai, frozen in liquid nitrogen, and stored at -80&#x000b0;C. Three biological replicates from each genotype/treatment were analyzed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>