<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>12582561.txt</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>12582561.txt</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:未知</h2>
			<h3>PMID:未知</h3>
				

				
			<p>
				作者：未知
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Clusters of Resistance-like genes (RLGs) have been identified from a variety of plant species. In soybean, RLG-specific primers and BAC-fingerprinting were used to develop a contig of overlapping BACs for a cluster of RLGs on soybean linkage group J. The resistance genes <span class="gene">Rps2</span> ( Phytophthora stem and root rot) and Rmd-c (powdery mildew) and the ineffective nodulation gene <span class="gene">Rj2</span> were previously mapped to this region of linkage group J. PCR hybridization was used to place two TIR/NBD/LRR cDNAs on overlapping BACs from this contig. Both of the cDNAs were present on BAC 34P7. Fingerprinting of this BAC suggested as many as twelve different RLGs were present. Given the high nucleotide identity shared between cDNAs LM6 and MG13 (&gt;90%), direct sequencing of this region would be difficult. More sequence information was needed about the RLGs within this region before sequencing could be undertaken. By comparing the genomic sequences of cDNAs LM6 and MG13 we identified conserved regions from which oligonucleotide primers specific to BAC 34P7 RLGs could be designed. The nine primer pairs spanned the genomic sequence of LM6 and produced overlapping RLG products upon amplification of BAC 34P7. Amplification products from 12 different RLGs were identified. On average, nucleotide identity between RLG sequences was greater than 95%. Examination of RLG sequences also revealed evidence of additions, deletions and duplications within targeted regions of these genes. Using previously mapped cDNAs we were able to quickly and inexpensively access multiple RLGs within a single specific cluster.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>