<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>18831797.nxml</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>18831797.nxml</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:未知</h2>
			<h3>PMID:未知</h3>
				

				
			<p>
				作者：未知
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						内容:<sec><title> Background </title><p> Sudden death syndrome (SDS) of soybean (<italic> Glycine max  </italic>L. Merr.) is an economically important disease, caused by the semi-biotrophic fungus<italic> Fusarium solani  </italic>f. sp.<italic> glycines </italic>, recently renamed<italic> Fusarium virguliforme  </italic>(Fv). Due to the complexity and length of the soybean-Fusarium interaction, the molecular mechanisms underlying plant resistance and susceptibility to the pathogen are not fully understood.<italic> F. virguliforme  </italic>has a very wide host range for the ability to cause root rot and a very narrow host range for the ability to cause a leaf scorch.<italic> Arabidopsis thaliana  </italic>is a host for many types of phytopathogens including bacteria, fungi, viruses and nematodes. Deciphering the variations among transcript abundances (TAs) of functional orthologous genes of soybean and<italic> A. thaliana  </italic>involved in the interaction will provide insights into plant resistance to<italic> F. viguliforme </italic>. </p></sec><sec><title> Results </title><p> In this study, we reported the analyses of microarrays measuring TA in whole plants after<italic> A. thaliana  </italic>cv 'Columbia' was challenged with fungal pathogen<italic> F. virguliforme </italic>. Infection caused significant variations in TAs. The total number of increased transcripts was nearly four times more than that of decreased transcripts in abundance. A putative resistance pathway involved in responding to the pathogen infection in<italic> A. thaliana  </italic>was identified and compared to that reported in soybean. </p></sec><sec><title> Conclusion </title><p> Microarray experiments allow the interrogation of tens of thousands of transcripts simultaneously and thus, the identification of plant pathways is likely to be involved in plant resistance to Fusarial pathogens. Dissection of the set functional orthologous genes between soybean and<italic> A. thaliana  </italic>enabled a broad view of the functional relationships and molecular interactions among plant genes involved in<italic> F. virguliforme  </italic>resistance. </p></sec>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>