<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>28620395.nxml</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>28620395.nxml</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:未知</h2>
			<h3>PMID:未知</h3>
				

				
			<p>
				作者：未知
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						内容:  <p> Glyoxalase is an evolutionary highly conserved pathway present in all organisms. Conventional glyoxalase pathway has two enzymes, glyoxalase I (GLYI) and glyoxalase II (GLYII) that act sequentially to detoxify a highly cytotoxic compound methylglyoxal (MG) to  <sc> D </sc> -lactate with the help of reduced glutathione. Recently, proteins with DJ-1/PfpI domain have been reported to perform the same conversion in a single step without the help of any cofactor and thus termed as &#x0201c;unique glyoxalase III&#x0201d; enzyme. Genome-wide analysis of glyoxalase genes have been previously conducted in  <italic> Arabidopsis </italic> , rice and Soybean plants, but no such study was performed for one of the agricultural important model legume species,  <italic> Medicago truncatula </italic> . A comprehensive genome-wide analysis of  <italic> Medicago </italic>  identified a total of putative 29 GLYI, 14 GLYII genes, and 5 glyoxalase III (DJ-1) genes. All these identified genes and their corresponding proteins were analyzed in detail including their chromosomal distribution, gene duplication, phylogenetic relationship, and the presence of conserved domain(s). Expression of all these genes was analyzed in different tissues as well as under two devastating abiotic stresses- salinity and drought using publicly available transcript data. This study revealed that  <italic> MtGLYI </italic> -4,  <italic> MtGLYII </italic> -6, and  <italic> MtDJ-1 </italic> A are the constitutive members with a high level of expression at all 17 analyzed tissues; while  <italic> MtGLYI </italic> -1,  <italic> MtGLYI </italic> -11,  <italic> MtGLYI </italic> -5,  <italic> MtGLYI- </italic> 7, and  <italic> MtGLYII- </italic> 13 showed tissue-specific expression. Moreover, most of the genes displayed similar pattern of expression in response to both salinity and drought stress, irrespective of stress duration and tissue type.  <italic> MtGLYI </italic> -8,  <italic> MtGLYI </italic> -11,  <italic> MtGLYI </italic> -6,  <italic> MtGLYI </italic> -16,  <italic> MtGLYI </italic> -21, and  <italic> MtGLYII- </italic> 9 showed up-regulation, while  <italic> MtGLYI </italic> -17 and  <italic> MtGLYI </italic> -7/9 showed down-regulation in response to both stresses. Interestingly,  <italic> MtGLYI </italic> -14/15 showed completely opposite pattern of expression in these two stresses. This study provides an initial basis about the physiological significance of glyoxalase genes in plant development and stress response of  <italic> Medicago </italic>  that could be explored further. </p>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>