<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>100-seed weight</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>100-seed weight</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:A PP2C-1 Allele Underlying a Quantitative Trait Locus Enhances Soybean <span class="trait">100-Seed</span> <span class="trait">Weight.</span></h2>
			<h3>PMID:28363587  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu X(1), Xiong Q(1), Cheng T(1), Li QT(1), Liu XL(2), Bi YD(3), Li W(3), Zhang WK(4), Ma B(4), Lai YC(3), Du WG(2), Man WQ(5), Chen SY(6), Zhang JS(7).
				<br/> 
				年份：2017 May 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cultivated soybeans may lose some useful genetic loci during domestication. Introgression of genes from wild soybeans could broaden the genetic background and improve soybean agronomic traits. In this study, through whole-genome sequencing of a recombinant inbred line population derived from a cross between a wild soybean ZYD7 and a cultivated soybean HN44, and mapping of quantitative trait loci for seed <span class="trait">weight,</span> we discovered that a phosphatase 2C-1 (PP2C-1) allele from wild soybean ZYD7 contributes to the increase in seed weight/size. PP2C-1 may achieve this function by enhancing cell size of integument and activating a subset of seed trait-related genes. We found that PP2C-1 is associated with <span class="gene">GmBZR1,</span> a soybean ortholog of Arabidopsis BZR1, one of key transcription factors in brassinosteroid (BR) signaling, and facilitate accumulation of dephosphorylated <span class="gene">GmBZR1.</span> In contrast, the PP2C-2 allele with variations of a few amino acids at the N-terminus did not exhibit this function. Moreover, we showed that <span class="gene">GmBZR1</span> could promote seed weight/size in transgenic plants. Through analysis of cultivated soybean accessions, we found that 40% of the examined accessions do not have the PP2C-1 allele, suggesting that these accessions can be improved by introduction of this allele. Taken together, our study identifies an elite allele PP2C-1, which can enhance seed <span class="trait">weight</span> and/or size in soybean, and pinpoints that manipulation of this allele by molecular-assisted breeding may increase production in soybean and other legumes/crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Genome-Wide Association Study for Agronomic Traits in Soybean Using SNP Markers and SNP-Based Haplotype Analysis.</h2>
			<h3>PMID:28152092  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contreras-Soto RI(1), Mora F(2), de Oliveira MA(3), Higashi W(3), Scapim CA(1), Schuster I(4).
				<br/> 
				年份：2017 Feb 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mapping quantitative trait loci through the use of linkage disequilibrium (LD) in populations of unrelated individuals provides a valuable approach for dissecting the genetic basis of complex traits in soybean (Glycine max). The haplotype-based genome-wide association study (GWAS) has now been proposed as a complementary approach to intensify benefits from LD, which enable to assess the genetic determinants of agronomic traits. In this study a GWAS was undertaken to identify genomic regions that control <span class="trait">100-seed</span> <span class="trait">weight</span> (SW), plant height (PH) and seed <span class="trait">yield</span> (SY) in a soybean association mapping panel using single nucleotide polymorphism (SNP) markers and haplotype information. The soybean cultivars (N = 169) were field-evaluated across four locations of southern Brazil. The genome-wide haplotype association analysis (941 haplotypes) identified eleven, seventeen and fifty-nine SNP-based haplotypes significantly associated with SY, SW and PH, respectively. Although most marker-trait associations were environment and trait specific, stable haplotype associations were identified for SY and SW across environments (i.e., haplotypes <span class="gene">Gm12_Hap12).</span> The haplotype block 42 on Chr19 <span class="gene">(Gm19_Hap42)</span> was confirmed to be associated with PH in two environments. These findings enable us to refine the breeding strategy for tropical soybean, which confirm that haplotype-based GWAS can provide new insights on the genetic determinants that are not captured by the single-marker approach.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T set was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less pod wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant seed <span class="trait">weight</span> and <span class="trait">100-seed</span> <span class="trait">weight</span> than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher seed <span class="trait">weight</span> under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean seed oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased seed <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] seed oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as seed <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between seed oil concentration and important agronomic and seed quality traits, including seed <span class="trait">yield,</span> <span class="trait">100-seed</span> <span class="trait">weight,</span> protein concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with seed oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for protein concentration, four for seed <span class="trait">yield,</span> two for <span class="trait">100-seed</span> <span class="trait">weight,</span> one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with seed protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with seed <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with seed oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for <span class="trait">100-seed</span> <span class="trait">weight,</span> one for protein concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as seed <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher seed <span class="trait">yield</span> and oil concentration or higher seed protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-environment mapping and meta-analysis of <span class="trait">100-seed</span> <span class="trait">weight</span> in soybean.</h2>
			<h3>PMID:22740134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun YN(1), Pan JB, Shi XL, Du XY, Wu Q, Qi ZM, Jiang HW, Xin DW, Liu CY, Hu GH, Chen QS.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">100-Seed</span> <span class="trait">weight</span> (100-SW) of soybean is an important but complicated quantitative trait to <span class="trait">yield.</span> This study was focus on the quantitative trait loci (QTLs) of soybean 100-SW from 2006 to 2010, using recombination inbred lines population that was derived from a cross between Charleston and Dongnong 594. A total of 23 QTLs for 100-SW were detected in the linkage group C2, D1a, F, G and O. Nine QTLs were identified by composite interval mapping including one QTL with the minimum confidence interval (CI) of 1.3 cM, while 14 QTLs by multiple interval mapping. Furthermore, 94 reported QTLs of 100-SW were integrated with our QTL mapping results using BioMercator. As a result, 15 consensus QTLs and their corresponding markers were identified. The minimum <span class="gene">CI</span> was reduced to 1.52 cM by the combination of meta-analysis. These findings may merit fine-mapping of these QTL in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SNPs discovery and CAPS marker conversion in soybean.</h2>
			<h3>PMID:20859693  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shu Y(1), Li Y, Zhu Z, Bai X, Cai H, Ji W, Guo D, Zhu Y.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To discover new SNPs and develop an easy assay method in soybean, we compared the high-throughput pyrosequencing ESTs with whole genome sequences in different soybean varieties and identified 3899 SNPs. Transitions were found to be much more frequent than transversions in these SNPs. We found that SNPs were widely distributed in the soybean genome, targeting numerous genes involved in various physiological and biochemical processes influencing important agronomic traits. A set of 16 SNPs were validated in nine soybean varieties, and seven SNPs were converted into CAPS. From functional gene association analysis, the marker CAPS282 on the 3'-UTR of gene <span class="gene">Glyma07g03490</span> was identified as associated with <span class="trait">100-seed</span> <span class="trait">weight</span> in soybean. The SNP discovery and CAPS markers conversion system developed in this study is fast and cost effective, and holds great promise for molecular-assisted breeding of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL analyses of seed <span class="trait">weight</span> during the development of soybean (Glycine max L. Merr.).</h2>
			<h3>PMID:18971958  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Teng W(1), Han Y, Du Y, Sun D, Zhang Z, Qiu L, Sun G, Li W.
				<br/> 
				年份：2009 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						At harvest traits such as seed <span class="trait">weight</span> are the sum of development and responses to stresses over the growing season and particularly during the reproductive phase of growth. The aim here was to measure quantitative trait loci (QTL) underlying the seed <span class="trait">weight</span> from early development to drying post harvest. One hundred forty-three F(5) derived recombinant inbred lines (RILs) developed from the cross of soybean cultivars 'Charleston' and 'Dongnong 594' were used for the analysis of QTL underlying mean <span class="trait">100-seed</span> <span class="trait">weight</span> at six different developmental stages. QTL x Environment interactions (QE) were analyzed by a mixed genetic mode based on 3 years' data. At an experiment-wise threshold of a=0.05 and by single-point analysis 94 QTL unaffected by QE underlay the mean seed <span class="trait">weight</span> at different developmental stages. Sixty-eight QTL affected by QE that also underlay mean seed <span class="trait">weight</span> were identified. From the 162 QTL 42 could be located on 12 linkage groups by composite interval mapping (LOD&gt;2.0). The numbers, locations and types of the QTL and the genetic effects were different at each developmental stage. On linkage group <span class="gene">C2</span> the distantly linked QTL swC2-1, swC2-2 and swC2-3 each affected mean seed <span class="trait">weight</span> throughout the different developmental stages. The DNA markers linked to the QTL possessed potential for use in marker-assisted selection for soybean seed size. The identification of QTL with genetic main effects and QE interaction effects suggested that such interactions might significantly alter seed <span class="trait">weight</span> during seed development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Genetic effects on seed traits in soybean].</h2>
			<h3>PMID:16318286  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liang HZ(1), Li WD, Wang H, Fang XJ.
				<br/> 
				年份：2005 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genetic effects of seed traits in soybean, including <span class="trait">100-seed</span> <span class="trait">weight,</span> seed length, seed width, seed thickness, length/width, length/thickness and width/thickness, were analyzed using an incomplete diallel cross of eight varieties with its F1 and F2 populations. The results showed that the above seven traits were controlled by direct genetic effects of seed and affected to different extent of maternal and cytoplasmic effects simultaneously. Among the traits, the inheritance of <span class="trait">100-seed</span> <span class="trait">weight,</span> seed length, length/width, length/thickness, and width/thickness were mainly controlled by cytoplasmic effects, while those of seed width and thickness were mainly by maternal effects. Both the seed direct heritabilities and the cytoplsmic heritabilities of <span class="trait">100-seed</span> <span class="trait">weight,</span> seed length, length/width and width/thickness were medium-sized. The individual selection and seed selection of above four traits at late generation may create good results. The maternal heritabilities of seed width and thickness were pretty high. To increase these two traits, an individual maternal selection should be done at early generation. Our results showed that varieties P2 and <span class="gene">P7</span> could be used as ideal parents for improvements of <span class="trait">100-seed</span> <span class="trait">weight,</span> seed length/width, length/thickness and width/thickness, while varieties P1, P4 and P6 are the ideal parents for increasing seed length, width and thickness respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, <span class="trait">100-seed</span> <span class="trait">weight,</span> and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being seed oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>