<!DOCTYPE html>
<html lang="en" id="html">

<head>
	<meta charset="UTF-8">
	<title>21569389</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
		@import url(http://fonts.googleapis.com/css?family=Vollkorn:400,400italic,700,700italic&subset=latin);

img {
	max-width: 100%;
}

table {
	width: 100%;
}

table, th, td {
	padding: 5px;
}


html, body {
        padding:1em;
        margin:auto;
        max-width:80em;
        background:#fefefe;
	}
body {
	font: 1.3em "Vollkorn", Palatino, Times;
	color: #333;
	line-height: 1.4;
	text-align: justify;
	}
header, nav, article, footer {
	width: 700px;
	margin:0 auto;
	}
article {
	margin-top: 4em;
	margin-bottom: 4em;
	min-height: 400px;
	}
footer {
	margin-bottom:50px;
	}
video {
	margin: 2em 0;
	border:1px solid #ddd;
	}

nav {
	font-size: .9em;
	font-style: italic;
	border-bottom: 1px solid #ddd;
	padding: 1em 0;
	}
nav p {
	margin: 0;
	}

/* Typography
-------------------------------------------------------- */

h1 {
	margin-top: 0;
	font-weight: normal;
	}
h2 {
	font-weight: normal;
	}
h3 {
	font-weight: normal;
	font-style: italic;
	margin-top:3em;
	}
p {
	margin-top:0;
	-webkit-hypens:auto;
	-moz-hypens:auto;
	hyphens:auto;
	}
ul {
	list-style: square;
	padding-left:1.2em;
	}
ol {
	padding-left:1.2em;
	}
blockquote {
	margin-left: 1em;
	padding-left: 1em;
	border-left: 1px solid #ddd;
	}
code {
	font-family: "Consolas", "Menlo", "Monaco", monospace, serif;
	font-size: .9em;
	background: white;
	}
a {
	color: #2484c1;
	text-decoration: none;
	}
a:hover {
	text-decoration: underline;
	}
a img {
	border:none;
	}
h1 a, h1 a:hover {
	color: #333;
	text-decoration: none;
	}
hr {
	color : #ddd;
	height : 1px;
	margin: 2em 0;
	border-top : solid 1px #ddd;
	border-bottom : none;
	border-left: 0;
	border-right: 0;
	}
p#heart{
	font-size: 2em;
	line-height: 1;
	text-align: center;
	color: #ccc;
	}
.red {
	color:#B50000;
	}

/* Home Page
--------------------------- */

body#index li {
	margin-bottom: 1em;
	}


/* iPad
-------------------------------------------------------- */
@media only screen and (max-device-width: 1024px) {
body {
	font-size: 120%;
	line-height: 1.4;
	}
} /* @iPad */

/* iPhone
-------------------------------------------------------- */
@media only screen and (max-device-width: 480px) {
body {
	text-align: left;
	}
article, footer {
	width: auto;
	}
article {
	padding: 0 10px;
	}
} /* @iPhone */

.btn2{
	background:#707070;
	position:relative;
	left:-100px;
	color:white;
	display: inline-block;
    margin-bottom: 10px;
    font-family: 'Titillium Web', sans-serif;
    font-size: 14px;
    font-weight: 300;
    line-height: 1.42857143;
	box-sizing: border-box;
	text-align: -webkit-match-parent;
}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
		function change()
		{
		    wi = document.getElementById("body").style.maxWidth
		    if(wi === "42em"){
		        document.getElementById("body").style.maxWidth = "80em";
				document.getElementById("html").style.maxWidth = "80em";
			}
			else {
		    	document.getElementById("body").style.maxWidth = "42em";
				document.getElementById("html").style.maxWidth = "42em";
			}
		}
	</SCRIPT>
</head>

<body id="body">
<button class="btn2" onclick="change()">切换</button>
	
	<br/>
	<div>
		

			
			<h2>文献标题:QTL underlying <span class="trait">resistance</span> to two HG types of <span class="trait">Heterodera</span> <span class="trait">glycines</span> found in soybean cultivar 'L-10'</h2>
			<h3>PMID:21569389</h3>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Chang </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weich0@foxmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Dong </surname>  <given-names> Limin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> donglimin221355@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wang </surname>  <given-names> Zizhen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> milandewang@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Hu </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> nmghuhaibo@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hyp234286@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weiliteng2006@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhang </surname>  <given-names> Hongxia </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hxzhang@sippe.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Guo </surname>  <given-names> Maozu </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> maozuguo@hit.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wenbinli@neau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 5 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Resistance of soybean (Glycine max L. Merr.) cultivars to populations of cyst nematode (SCN; Heterodera glycines I.) was complicated by the diversity of HG Types (biotypes), the multigenic nature of resistance and the temperature dependence of resistance to biotypes. The objective here was to identify QTL for broad-spectrum resistance to SCN and examine the transcript abundances of some genes within the QTL. RESULTS: A Total of 140 F(5) derived F(7) recombinant inbred lines (RILs) were advanced by single-seed-descent from a cross between 'L-10' (a soybean cultivar broadly resistant to SCN) and 'Heinong 37' (a SCN susceptible cultivar). Associated QTL were identified by WinQTL2.1. QTL Qscn3-1 on linkage group (LG) E, Qscn3-2 on LG G, Qscn3-3 on LG J and Qscn14-1 on LG O were associated with <span class="gene">SCN resistance</span> in both year data (2007 and 2008). Qscn14-2 on LG O was identified to be associated with <span class="gene">SCN resistance</span> in 2007. Qscn14-3 on LG D2 was identified to be associated with <span class="gene">SCN resistance</span> in 2008. Qscn14-4 on LG J was identified to be associated with <span class="gene">SCN resistance</span> in 2008. The Qscn3-2 on LG G was linked to Satt309 (less than 4 cM), and explained 19.7% and 23.4% of the phenotypic variation in 2007 and 2008 respectively. Qscn3-3 was less than 5 cM from Satt244 on LG J, and explained 19.3% and 17.95% of the phenotypic variations in 2007 and 2008 respectively. Qscn14-4 could explain 12.6% of the phenotypic variation for the SCN race 14 resistance in 2008 and was located in the same region as Qscn3-3. The total phenotypic variation explained by Qscn3-2 and Qscn3-3 together was 39.0% and 41.3% in 2007 and 2008, respectively. Further, the flanking markers Satt275, Satt309, Sat_350 and Satt244 were used for the selection of resistant lines to SCN race 3, and the accuracy of selection was about 73% in this RIL population. Four genes in the predicted resistance gene cluster of LG J (chromosome 16) were successfully cloned by RT-PCR. The transcript encoded by the gene <span class="gene">Glyma16g30760.1</span> was abundant in the SCN resistant cultivar 'L-10' but absent in susceptible cultivar 'Heinong 37'. Further, the abundance was higher in root than in leaf for 'L-10'. Therefore, the gene was a strong candidate to underlie part of the resistance to SCN. CONCLUSIONS: Satt275, Satt309, Sat_305 and Satt244, which were tightly linked to the major QTL for resistance to SCN on LG G and J, would be candidates for marker-assisted selection of lines resistant to the SCN race 3. Among the six RLK genes, <span class="gene">Glyma16g30760.1</span> was found to accumulate transcripts in the <span class="gene">SCN resistance</span> cultivar 'L-10' but not in 'Heinong 37'. The transcript abundance was higher in root than in leaf for L-10.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>