<!DOCTYPE html>
<html lang="en" id="html">

<head>
	<meta charset="UTF-8">
	<title>26268218</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
		@import url(http://fonts.googleapis.com/css?family=Vollkorn:400,400italic,700,700italic&subset=latin);

img {
	max-width: 100%;
}

table {
	width: 100%;
}

table, th, td {
	padding: 5px;
}


html, body {
        padding:1em;
        margin:auto;
        max-width:80em;
        background:#fefefe;
	}
body {
	font: 1.3em "Vollkorn", Palatino, Times;
	color: #333;
	line-height: 1.4;
	text-align: justify;
	}
header, nav, article, footer {
	width: 700px;
	margin:0 auto;
	}
article {
	margin-top: 4em;
	margin-bottom: 4em;
	min-height: 400px;
	}
footer {
	margin-bottom:50px;
	}
video {
	margin: 2em 0;
	border:1px solid #ddd;
	}

nav {
	font-size: .9em;
	font-style: italic;
	border-bottom: 1px solid #ddd;
	padding: 1em 0;
	}
nav p {
	margin: 0;
	}

/* Typography
-------------------------------------------------------- */

h1 {
	margin-top: 0;
	font-weight: normal;
	}
h2 {
	font-weight: normal;
	}
h3 {
	font-weight: normal;
	font-style: italic;
	margin-top:3em;
	}
p {
	margin-top:0;
	-webkit-hypens:auto;
	-moz-hypens:auto;
	hyphens:auto;
	}
ul {
	list-style: square;
	padding-left:1.2em;
	}
ol {
	padding-left:1.2em;
	}
blockquote {
	margin-left: 1em;
	padding-left: 1em;
	border-left: 1px solid #ddd;
	}
code {
	font-family: "Consolas", "Menlo", "Monaco", monospace, serif;
	font-size: .9em;
	background: white;
	}
a {
	color: #2484c1;
	text-decoration: none;
	}
a:hover {
	text-decoration: underline;
	}
a img {
	border:none;
	}
h1 a, h1 a:hover {
	color: #333;
	text-decoration: none;
	}
hr {
	color : #ddd;
	height : 1px;
	margin: 2em 0;
	border-top : solid 1px #ddd;
	border-bottom : none;
	border-left: 0;
	border-right: 0;
	}
p#heart{
	font-size: 2em;
	line-height: 1;
	text-align: center;
	color: #ccc;
	}
.red {
	color:#B50000;
	}

/* Home Page
--------------------------- */

body#index li {
	margin-bottom: 1em;
	}


/* iPad
-------------------------------------------------------- */
@media only screen and (max-device-width: 1024px) {
body {
	font-size: 120%;
	line-height: 1.4;
	}
} /* @iPad */

/* iPhone
-------------------------------------------------------- */
@media only screen and (max-device-width: 480px) {
body {
	text-align: left;
	}
article, footer {
	width: auto;
	}
article {
	padding: 0 10px;
	}
} /* @iPhone */

.btn2{
	background:#707070;
	position:relative;
	left:-100px;
	color:white;
	display: inline-block;
    margin-bottom: 10px;
    font-family: 'Titillium Web', sans-serif;
    font-size: 14px;
    font-weight: 300;
    line-height: 1.42857143;
	box-sizing: border-box;
	text-align: -webkit-match-parent;
}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
		function change()
		{
		    wi = document.getElementById("body").style.maxWidth
		    if(wi === "42em"){
		        document.getElementById("body").style.maxWidth = "80em";
				document.getElementById("html").style.maxWidth = "80em";
			}
			else {
		    	document.getElementById("body").style.maxWidth = "42em";
				document.getElementById("html").style.maxWidth = "42em";
			}
		}
	</SCRIPT>
</head>

<body id="body">
<button class="btn2" onclick="change()">切换</button>
	
	<br/>
	<div>
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping</h2>
			<h3>PMID:26268218</h3>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <address>  <email> wy11544@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> twlneau@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhiwu </given-names>  </name>  <address>  <email> zhiwu.zhang@wsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Key Laboratory of Soybean Biology in Chinese Ministry of Education (key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry), Northeast Agricultural University, Harbin, 150030 China  </aff>  <aff id="Aff2">  <label/> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI) Chinese Academy of Agricultural Sciences, 100081 Beijing, China  </aff>  <aff id="Aff3">  <label/> Bioinformatics Division, Biomarker Technologies Corporation, 101300 Beijing, China  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode (SCN, Heterodera glycines Ichinohe) is one of the most fatal pests of soybean (Glycine max (L.) Merr.) worldwide and causes huge loss of soybean yield each year. Multiple sources of resistance are urgently needed for effective management of SCN via the development of resistant cultivars. The aim of the present study was to investigate the genetic architecture of resistance to SCN HG Type 0 (race 3) and HG Type 1.2.3.5.7 (race 4) in landraces and released elite soybean cultivars mostly from China. RESULTS: A total of 440 diverse soybean landraces and elite cultivars were screened for resistance to SCN HG Type 0 and HG Type 1.2.3.5.7. Exactly 131 new sources of <span class="gene">SCN resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF) &gt; 4% that were present in 97% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the resistance for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease resistance proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, <span class="gene">MYB</span> and <span class="gene">WRKY</span> transcription activation families were identified. Such proteins <span class="gene">may participate in the resistant reaction to SCN</span> and were frequently found in the tightly linked genomic regions of the peak SNPs. CONCLUSIONS: GWAS extended understanding of the genetic architecture of <span class="gene">SCN resistance</span> in multiple genetic backgrounds. Nineteen association signals were obtained for the resistance to the two Hg Types of SCN. The multiple beneficial alleles from resistant germplasm sources will be useful for the breeding of cultivars with improved resistance to SCN. Analysis of genes near association signals may facilitate the recognition of the causal gene(s) underlying <span class="gene">SCN resistance</span>s.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>