<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Branching</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Branching</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Functional characterization of soybean strigolactone biosynthesis and signaling genes in Arabidopsis MAX mutants and <span class="gene">GmMAX3</span> in soybean nodulation.</h2>
			<h3>PMID:29268717</h3>
				

				
			<p>
				作者：Haq BU(1), Ahmad MZ(2), Ur Rehman N(1), Wang J(1), Li P(2), Li D(1), Zhao J(3)(4).
				<br/> 
				年份：2017 Dec 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Strigolactones (SLs) play important roles in controlling root growth, shoot <span class="trait">branching,</span> and plant-symbionts interaction. Despite the importance, the components of SL biosynthesis and signaling have not been unequivocally explored in soybean. RESULTS: Here we identified the putative components of SL synthetic enzymes and signaling proteins in soybean genome. Soybean genome contains conserved MORE AXILLARY <span class="trait">BRANCHING</span> (MAX) orthologs, <span class="gene">GmMAX1s,</span> <span class="gene">GmMAX2s,</span> <span class="gene">GmMAX3s,</span> and <span class="gene">GmMAX4s.</span> The tissue expression patterns are coincident with SL synthesis in roots and signaling in other tissues under normal conditions. <span class="gene">GmMAX1a,</span> <span class="gene">GmMAX2a,</span> <span class="gene">GmMAX3b,</span> and <span class="gene">GmMAX4a</span> expression in their Arabidopsis orthologs' mutants not only restored most characteristic phenotypes, such as shoot <span class="trait">branching</span> and shoot height, leaf shape, primary root length, and root hair growth, but also restored the significantly changed hormone contents, such as reduced JA and ABA contents in all mutant leaves, but increased auxin levels in atmax1, atmax3 and atmax4 mutants. Overexpression of these <span class="gene">GmMAXs</span> also altered the hormone contents in wild-type Arabidopsis. <span class="gene">GmMAX3b</span> was further characterized in soybean nodulation with overexpression and knockdown transgenic hairy roots. <span class="gene">GmMAX3b</span> overexpression <span class="gene">(GmMAX3b-OE)</span> lines exhibited increased nodule number while <span class="gene">GmMAX3b</span> knockdown <span class="gene">(GmMAX3b-KD)</span> decreased the nodule number in transgenic hairy roots. The expression levels of several key nodulation genes were also altered in <span class="gene">GmMAX3b</span> transgenic hairy roots. <span class="gene">GmMAX3b</span> overexpression hairy roots had reduced ABA, but increased JA levels, with no significantly changed auxin content, while the contrast changes were observed in <span class="gene">GmMAX3b-KD</span> lines. Global gene expression in <span class="gene">GmMAX3b-OE</span> or <span class="gene">GmMAX3b-KD</span> hairy roots also revealed that altered expression of <span class="gene">GmMAX3b</span> in soybean hairy roots changed several subsets of genes involved in hormone biosynthesis and signaling and transcriptional regulation of nodulation processes. CONCLUSIONS: This study not only revealed the conservation of SL biosynthesis and signaling in soybean, but also showed possible interactions between SL and other hormone synthesis and signaling during controlling plant development and soybean nodulation. <span class="gene">GmMAX3b-mediated</span> SL biosynthesis and signaling may be involved in soybean nodulation by affecting both root hair formation and its interaction with rhizobia.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Strigolactones promote rhizobia interaction and increase nodulation in soybean (Glycine max).</h2>
			<h3>PMID:29191709</h3>
				

				
			<p>
				作者：Rehman NU(1), Ali M(1), Ahmad MZ(2), Liang G(3), Zhao J(4).
				<br/> 
				年份：2017 Dec 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Strigolactones (SLs) play an important role in controlling root growth, shoot <span class="trait">branching,</span> and plant-symbionts interaction. Despite the importance, the components of SL biosynthesis and signaling have not been unequivocally explored in soybean. Here we identified the putative components of SL synthesis enzymes <span class="gene">GmMAX1a</span> and <span class="gene">GmMAX4a</span> with tissue expression patterns and were apparently regulated by rhizobia infection and changed during nodule development. <span class="gene">GmMAX1a</span> and <span class="gene">GmMAX4a</span> were further characterized in soybean nodulation with knockdown transgenic hairy roots. <span class="gene">GmMAX1a</span> and <span class="gene">GmMAX4a</span> knockdown lines exhibit decreased nodule number and expression levels of several nodulation genes required for nodule development. Hormone analysis showed that <span class="gene">GmMAX1a</span> and <span class="gene">GmMAX4a</span> knockdown hairy roots had increased physiological level of ABA and JA but significantly decreased auxin content. This study not only revealed the conservation of SL biosynthesis but also showed close interactions between SL and other hormone signaling in controlling plant development and legume-rhizobia interaction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq analysis of nodule development at five different developmental stages of soybean ( <italic> Glycine max </italic> ) inoculated with <italic> Bradyrhizobium japonicum </italic> strain 113-2</h2>
			<h3>PMID:28169364</h3>
				

				
					<h3>PMCID:5294573</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Song L. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Hai F. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chan J. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Li M. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qing N. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shui L. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Zhi H. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Zhong L. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiao J. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> De Z. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xin A. </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Key Laboratory of Oil Crop Biology, Ministry of Agriculture </institution> , Wuhan 430062,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Oil Crops Research Institute of Chinese Academy of Agriculture Sciences </institution> ,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 08 </day>  <month> 01 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nodule development directly affects nitrogen fixation efficiency during soybean growth. Although abundant genome-based information related to nodule development has been released and some studies have reported the molecular mechanisms that regulate nodule development, information on the way nodule genes operate in nodule development at different developmental stages of soybean is limited. In this report, notably different nodulation phenotypes in soybean roots inoculated with Bradyrhizobium japonicum strain 113-2 at five developmental stages <span class="trait">(branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage) were shown, and the expression of nodule genes at these five stages was assessed quantitatively using RNA-Seq. Ten comparisons were made between these developmental periods, and their differentially expressed genes were analysed. Some important genes were identified, primarily encoding symbiotic nitrogen fixation-related proteins, cysteine proteases, cystatins and cysteine-rich proteins, as well as proteins involving plant-pathogen interactions. There were no significant shifts in the distribution of most GO functional annotation terms and KEGG pathway enrichment terms between these five development stages. A cystatin <span class="gene">Glyma18g12240</span> was firstly identified from our RNA-seq, and was likely to promote nodulation and delay nodule senescence. This study provides molecular material for further investigations into the mechanisms of nitrogen fixation at different soybean developmental stages.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Nodule development directly affects nitrogen fixation efficiency during soybean growth. Although abundant genome-based information related to nodule development has been released and some studies have reported the molecular mechanisms that regulate nodule development, information on the way nodule genes operate in nodule development at different developmental stages of soybean is limited. In this report, notably different nodulation phenotypes in soybean roots inoculated with <italic> Bradyrhizobium japonicum </italic> strain 113-2 at five developmental stages <span class="trait">(branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage) were shown, and the expression of nodule genes at these five stages was assessed quantitatively using RNA-Seq. Ten comparisons were made between these developmental periods, and their differentially expressed genes were analysed. Some important genes were identified, primarily encoding symbiotic nitrogen fixation-related proteins, cysteine proteases, cystatins and cysteine-rich proteins, as well as proteins involving plant-pathogen interactions. There were no significant shifts in the distribution of most GO functional annotation terms and KEGG pathway enrichment terms between these five development stages. A cystatin <span class="gene">Glyma18g12240</span> was firstly identified from our RNA-seq, and was likely to promote nodulation and delay nodule senescence. This study provides molecular material for further investigations into the mechanisms of nitrogen fixation at different soybean developmental stages.
					</p>
				
					<p>&nbsp;&nbsp;
						In this report, we investigated ten comparisons between five important developmental stages <span class="trait">(branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage) of soybean and identified a large number of differentially expressed genes (DEGs), including those encoding soybean symbiotic nitrogen fixation-related proteins, cysteine proteases, cystatins and cysteine-rich proteins, as well as proteins involved in plant-pathogen interactions. Seven cystatins are actively transcribed during nodule development and senescence <xref ref-type="bibr" rid="b11"> 11 </xref> , while two other cystatins <span class="gene">(Glyma18g12240</span> and <span class="gene">Glyma18g00690)</span> were identified from our RNA-seq. The symbiotic function of <span class="gene">Glyma18g12240</span> was studied to verify the reliability of our data, and the results showed that it was likely to promote nodulation and delay nodule senescence. The discovered differentially expressed genes (DEGs) and the results described in this study should aid efforts to understand the mechanisms of soybean nodule development and shed new light on the molecular mechanisms of nitrogen fixation at five important developmental stages.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate nodule development during soybean growth, we examined the symbiotic phenotypic of soybean inoculated with <italic> B. japonicum </italic> 113-2 at five developmental stages <span class="trait">(branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage), and the results are shown in <xref ref-type="fig" rid="f1"> Fig. 1 </xref> and <xref ref-type="supplementary-material" rid="S1"> Table S1 </xref> . The classification of these five developmental stages according to the popular classification system <xref ref-type="bibr" rid="b21"> 21 </xref> and the detail defined information for the plants at each stage was shown in materials and methods. The growth of soybean inoculated with <italic> B. japonicum </italic> 113-2 at these five developmental stages was shown in <xref ref-type="fig" rid="f1"> Fig. 1A&#x02013;E </xref> . At the <span class="trait">branching</span> stage, the control (CK, inoculated with media only) had slightly better growth than the soybean inoculated with <italic> B. japonicum </italic> 113-2 ( <xref ref-type="fig" rid="f1"> Fig. 1A </xref> ), maybe because the symbiotic process required more energy from the host plant and nitrogen fixation was weak during this period. The number of nodules per plant and the dry weight per nodule increased from the <span class="trait">branching</span> stage to fruiting stage and decreased from the pod stage ( <xref ref-type="fig" rid="f1"> Fig. 1F&#x02013;J </xref> , <xref ref-type="supplementary-material" rid="S1"> Table S1 </xref> ). At the <span class="trait">branching</span> stage, flowering stage and fruiting stage, the nodules were pink and/or yellow, while at the pod stage, the nodules became brown ( <xref ref-type="fig" rid="f1"> Fig. 1G&#x02013;I </xref> ). The nodules were senescence and filled with water at the harvest stage ( <xref ref-type="fig" rid="f1"> Fig. 1I,J </xref> ). Moreover, differentially expressed levels of <italic> Gm Lb <span class="gene">(Glyma10g34280</span> </italic> ), which is required for nitrogen fixation <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b32"> 32 </xref> , at all five developmental stages were shown ( <xref ref-type="fig" rid="f1"> Fig. 1K </xref> ). These results indicate notable symbiotic phenotypes among different developmental stages of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> (A&#x02013;E) </bold> Soybean growth at five important developmental stages, including the <span class="trait">branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage. ( <bold> F&#x02013;J) </bold> Nodulation phenotypes were examined at five developmental stages after inoculation with 113-2. <bold> (K </bold> ) qPCR analysis of the transcript levels of <italic> Lbc1 </italic> <span class="gene">(Glyma10g34280)</span> at five developmental stages. Bars, 4&#x02009;cm ( <bold> A,B </bold> ); 5&#x02009;cm ( <bold> C&#x02013;E </bold> ); 3&#x02009;cm ( <bold> F&#x02013;J </bold> ). d, days.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:In vivo visualization of F-actin structures during the development of the moss Physcomitrella patens.</h2>
			<h3>PMID:17335498  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Finka A(1), Schaefer DG, Saidi Y, Goloubinoff P, Zrÿd JP.
				<br/> 
				年份：2007
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						* The 'in planta' visualization of F-actin in all cells and in all developmental stages of a plant is a challenging problem. By using the soybean heat inducible <span class="gene">Gmhsp17.3B</span> promoter instead of a constitutive promoter, we have been able to label all cells in various developmental stages of the moss Physcomitrella patens, through a precise temperature tuning of the expression of green fluorescent protein (GFP)-talin. * A short moderate heat treatment was sufficient to induce proper labeling of the actin cytoskeleton and to allow the visualization of time-dependent organization of F-actin structures without impairment of cell viability. * In growing moss cells, dense converging arrays of F-actin structures were present at the growing tips of protonema cell, and at the localization of <span class="trait">branching.</span> Protonema and leaf cells contained a network of thick actin cables; during de-differentiation of leaf cells into new protonema filaments, the thick bundled actin network disappeared, and a new highly polarized F-actin network formed. * The controlled expression of GFP-talin through an inducible promoter improves significantly the 'in planta' imaging of actin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biosynthesis of phytosterols. Kinetic mechanism for the enzymatic C-methylation of sterols.</h2>
			<h3>PMID:12807886  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nes WD(1), Song Z, Dennis AL, Zhou W, Nam J, Miller MB.
				<br/> 
				年份：2003 Sep 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cloned soybean sterol methyltransferase was purified from Escherichia coli to gel electrophoretic homogeneity. From initial velocity experiments, catalytic constants for substrates best suited for the first and second <span class="gene">C1</span> transfer activities, cycloartenol and 24(28)-methylenelophenol, were 0.01 and 0.001 s-1, respectively. Two-substrate kinetic analysis using cycloartenol and S-adenosyl-l-methionine (AdoMet) generated an intersecting line pattern characteristic of a ternary complex kinetic mechanism. The high energy intermediate analog 25-azacycloartanol was a noncompetitive inhibitor versus cycloartenol and an uncompetitive inhibitor versus AdoMet. The dead end inhibitor analog cyclolaudenol was competitive versus cycloartenol and uncompetitive versus AdoMet. 24(28)-Methylenecycloartanol and AdoHcy generated competitive and noncompetitive kinetic patterns, respectively, with respect to AdoMet. Therefore, 24(28)-methylenecycloartanol combines with the same enzyme form as does cycloartenol and must be released from the enzyme before AdoHcy. 25-Azacycloartanol inhibited the first and second <span class="gene">C1</span> transfer activities with about equal efficacy (Ki = 45 nm), suggesting that the successive C-methylation of the Delta 24 bond occurs at the same active center. Comparison of the initial velocity data using AdoMet versus [2H3-methyl]AdoMet as substrates tested against saturating amounts of cycloartenol indicated an isotope effect on VCH3/VCD3 close to unity. [25-2H]24(28)-Methylenecycloartanol, [28E-2H]24 (28)-methylenelanosterol, and [28Z-2H]24(28)-methylene lanosterol were prepared and paired with AdoMet or [methyl-3H3]AdoMet to examine the kinetic isotope effects attending the C-28 deprotonation in the enzymatic synthesis of 24-ethyl(idene) sterols. The stereochemical features as well as the observation of isotopically sensitive <span class="trait">branching</span> during the second C-methylation suggests that the two methylation steps can proceed by a change in chemical mechanism resulting from differences in sterol structure, concerted versus carbocation; the kinetic mechanism remains the same during the consecutive methylation of the Delta 24 bond.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The occurrence of internal (1 --&gt; 5)-linked arabinofuranose and arabinopyranose residues in arabinogalactan side chains from soybean pectic substances.</h2>
			<h3>PMID:11217953  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huisman MM(1), Brül LP, Thomas-Oates JE, Haverkamp J, Schols HA, Voragen AG.
				<br/> 
				年份：2001 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						CDTA-extractable soybean pectic substances were subjected to enzymatic digestion with arabinogalactan degrading enzymes yielding a resistant polymeric pectic backbone and arabino-, galacto-, and arabinogalacto-oligomers. The complex digest was fractionated using size-exclusion chromatography. Monosaccharide composition analysis, HPAEC fractionation and MALDI-TOF <span class="gene">MS</span> analysis of the resulting fractions showed that each contained a mixture of oligosaccharides of essentially the same degree of polymerisation, composed of only arabinose and galactose. MALDI-TOF <span class="gene">MS</span> analysis was used for molecular mass screening of oligosaccharides in underivatised HPAEC fractions. The monosaccharide sequence and the <span class="trait">branching</span> pattern of oligosaccharides (degree of polymerisation from 4 to 8) were determined using linkage analysis and ES-CID tandem <span class="gene">MS</span> analysis of the per-O-methylated oligosaccharides in each of the HPAEC fractions. These analyses indicated the presence of common linear (1 --&gt; 4)-linked galacto-oligosaccharides, and both linear and branched arabino-oligosaccharides. In addition, the results unambiguously showed the presence of oligosaccharides containing (1 --&gt; 4)-linked galactose residues bearing an arabinopyranose residue as the non-reducing terminal residue, and a mixture of linear oligosaccharides constructed of (1 --&gt; 4)-linked galactose residues interspersed with an internal (1 --&gt; 5)-linked arabinofuranose residue. The consequences of these two new structural features of pectic arabinogalactan side chains are discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Reciprocal regulation of mammalian nitric oxide synthase and calcineurin by plant calmodulin isoforms.</h2>
			<h3>PMID:9843363  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cho MJ(1), Vaghy PL, Kondo R, Lee SH, Davis JP, Rehl R, Heo WD, Johnson JD.
				<br/> 
				年份：1998 Nov 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calmodulin (CaM) is the primary mediator of Ca signal transduction processes in cells. Unlike animal cells, plant cells express multiple <span class="gene">CaM</span> isoforms. One cloned soybean <span class="gene">CaM</span> isoform (SCaM-4) half-maximally activated mammalian nitric oxide synthase (NOS) at 180 nM while another (SCaM-1) served as a competitive antagonist (Ki approximately 120 nM) of this activation. The reciprocal was true for the protein phosphatase calcineurin (CaN); <span class="gene">SCaM-1</span> half-maximally activated mammalian CaN at approximately 12 nM, and <span class="gene">SCaM-4</span> competitively antagonized (Ki approximately 70 nM) its activation. The reciprocal enzyme activation and competitive inhibition exhibited by these plant <span class="gene">CaM</span> isoforms suggest that their differential expression in cells could allow selective activation of some target enzymes and the selective inhibition of others. This may allow for a <span class="trait">branching</span> or bifurcation in the Ca2+-CaM signal transduction pathway and to alterations in cell function.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolutionary study of the isoflavonoid pathway based on multiple copies analysis in soybean</h2>
			<h3>PMID:24962214</h3>
				
					<h3>PMCID:4076065</h3>
					<h4>期刊:BMC Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Chu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> 2012216031@njau.edu.cn </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Wang </surname>  <given-names> Jiao </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> wjiao@njau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Cheng </surname>  <given-names> Hao </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jenny21star@njau.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Yang </surname>  <given-names> Qing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qyang19@njau.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> dyyu@njau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The results revealed that 1) only the phenylalanine ammonia-lyase ( <italic> <span class="gene">PAL</span> </italic> ) gene family most upstream from the pathway preserved all of the ancient and recent segmental duplications and maintained a strongly conserved synteny among these duplicated segments; gene families encoding branch-point enzymes with higher pleiotropy tended to retain more types of duplication; and genes encoding chalcone reductase (CHR) and isoflavone synthase (IFS) specific for legumes retained only recent segmental duplications; 2) downstream genes evolved faster than upstream genes and were subject to positive selection or relaxed selection constraints; 3) gene members encoding enzymes with high pleiotropy at the <span class="trait">branching</span> points were more likely to have undergone evolutionary differentiation, which may correspond to their functional divergences.
					</p>
				
					<p>&nbsp;&nbsp;
						Most of the abovementioned evolutionary pattern studies have exclusively considered a single gene copy that encoded each enzyme in the metabolic pathway. However, it remains to be determined whether the above conclusions still apply when multiple copies encoding each enzyme are considered. Homologous copies could incur functional differentiation after gene duplication, including pseudogenization [ <xref ref-type="bibr" rid="B29"> 29 </xref> ], sub-functionalization and [ <xref ref-type="bibr" rid="B30"> 30 </xref> ] neo-functionalization [ <xref ref-type="bibr" rid="B31"> 31 </xref> ], which might lead to evolutionary divergence. If only single-copy encoding enzymes are considered in the study of the relationship between evolutionary patterns and the positions of enzymes in the metabolic pathway, bias toward the evolution of metabolic pathway genes might result. Experimental studies have suggested that several gene copies encoding branch-point enzymes located at the central isoflavonoid biosynthetic pathway in soybeans exhibit functional differentiation, such as 4-coumarate:CoA ligase (4CL) and chalcone isomerase (CHI) [ <xref ref-type="bibr" rid="B32"> 32 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> ]. This phenomenon is expected in soybean due to polyploidy; however, whether there are differential evolution patterns among these functionally differential copies and whether differential evolution patterns are relevant to pleiotropy or to the connectivity of divergent gene copies remain unknown. Above all, considering multiple-copy encoding enzymes in the study of differential evolutionary patterns of genes in metabolic pathways could provide clearer evidence of the way selection power acts on genes in metabolic pathways than previous studies have offered.The isoflavonoid phytoalexin pathway in soybeans provides an excellent system for investigating the influence of metabolic pathway structure on the duplication and evolutionary patterns of enzymes. First, multiple copies of gene-encoding enzymes in the isoflavonoid biosynthetic pathway were retained as the soybean experienced two rounds of WGD events. Second, the pathway contains nine major enzymes acting at different positions and four metabolic nodes of the isoflavonoid biosynthesis pathway. Therefore, the network topology is suitable to investigations of the effect that the positions and functions of enzymes in the pathway on gene duplication and evolutionary patterns (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). Metabolic node substances are defined as the substances that participate in two or <span class="trait">branching</span> pathways. Branch-point enzymes are the enzymes that catalyze these reactions and primarily include4-coumarate:CoA ligase (4CL), chalcone synthase (CHS), chalcone reductase (CHR), <span class="gene">CHI</span> and isoflavone synthase (IFS). In general, <span class="gene">4CL</span> participates in most of the pathways involved in the biosynthesis of lignin, flavone, flavonol, anthocyanin and isoflavonoid. <span class="gene">CHS</span> and <span class="gene">CHI</span> also participate in these pathways, with the exception of the lignin biosynthesis pathway. In contrast, CHR and <span class="gene">IFS</span> only control isoflavonoid biosynthesis. Therefore, we grouped 4CL, <span class="gene">CHI</span> and <span class="gene">CHS</span> into enzymes with greater pleiotropy and CHR and <span class="gene">IFS</span> into enzymes with less pleiotropy in the <span class="trait">branching</span> pathway. To avoid ambiguous boundaries in the <span class="trait">branching</span> pathway, the most upstream enzymes, phenylalanine ammonia-lyase (PAL) and cinnamate 4-hydroxylase (C4H), are classified as the upstream enzymes, whereas the most downstream enzymes, isoflavone O-methyltransferase (IOMT) and isoflavone reductase (IFR), are downstream enzymes.
					</p>
				
					<p>&nbsp;&nbsp;
						Although the recent segmental duplication was preserved in all gene families in the soybean isoflavonoid synthetic pathway, the number of duplications was asymmetrically distributed in the <span class="trait">branching</span> pathway. Gene families encoding enzymes with higher pleiotropy tend to retain more recent segmental duplications (3 to 4) than those with lower pleiotropy (1 to 2). In addition to recent segmental duplications, gene families with higher pleiotropy reserved more gene copies from other types of duplication, such as the old tandem duplication in <italic> <span class="gene">4CL</span> </italic> and <italic> <span class="gene">CHI</span> </italic> and therecent tandem duplication in <italic> <span class="gene">4CL</span> </italic> and <italic> <span class="gene">CHS</span> </italic> . In comparison, branch-point gene families with lower pleiotropy, such as <italic> CHR </italic> and <italic> <span class="gene">IFS</span> </italic> , only retained 4 and 2 copies, respectively, from recent segmental duplication. We hypothesized that enzymes with higher pleiotropy catalyze metabolic node substances that are responsible for a greater range of downstream production and thus required more copies to reinforce their function. However, both <italic> CHR </italic> and <italic> <span class="gene">IFS</span> </italic> , which are legume-specific genes, play a pivotal role in isoflavonoid biosynthesis. An increased number of copies and copy duplication types were detected in one of the downstream enzymes, <italic> IOMT </italic> (14 copies). A previous study revealed that recombinant <italic> M. truncatula IOMT </italic> s had the ability to catalyze differential conformational substrates [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]; thus, we inferred that more copies were needed to maintain distinct catalytic properties.
					</p>
				
					<p>&nbsp;&nbsp;
						According to our study, the gene members encoding the enzymes (such as 4CL, CHS, and CHI) with high pleiotropy at the <span class="trait">branching</span> points tend to retain more gene copies. These duplicated copies are more likely to have evolutionary differences, corresponding to their functional divergences. Divergent members of the <italic> <span class="gene">4CL</span> </italic> gene family have been identified in soybeans and exhibited pronounced differences in the ability of the isoenzymes to catalyze different substrates [ <xref ref-type="bibr" rid="B32"> 32 </xref> ]. For example, <span class="gene">4CL</span> catalyzes the reaction of p-coumarate to produce the final products flavone, flavonol, anthocyanin and isoflavonoid. In addition, the recombinant <span class="gene">4CL1</span> isoform could utilize several other substrates (such as ferulate and sinapate) to channel flux to the lignin biosynthetic pathway. Given that it accepted the broadest range of substrates, <italic> <span class="gene">4CL1</span> </italic> was shown to be under purifying selection. In contrast, positive selection was detected in <italic> 4CL2 </italic> and <italic> 4CL3 </italic> , which acted on fewer substrates with great efficiency.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>