<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Cercospora kikuchii</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Cercospora kikuchii</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Development of a Quantitative Polymerase Chain Reaction Detection Protocol for <span class="trait">Cercospora</span> <span class="trait">kikuchii</span> in Soybean Leaves and Its Use for Documenting Latent Infection as Affected by Fungicide Applications.</h2>
			<h3>PMID:24805074  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chanda AK, Ward NA, Robertson CL, Chen ZY, Schneider RW.
				<br/> 
				年份：2014 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Cercospora</span> leaf blight (CLB) of soybean, caused by <span class="trait">Cercospora</span> <span class="trait">kikuchii,</span> is a serious disease in the southern United States. A sensitive TaqMan probe-based real-time quantitative polymerase chain reaction (qPCR) assay was developed to specifically detect and quantify C. <span class="trait">kikuchii</span> in naturally infected soybean plants. The sensitivity was 1 pg of genomic DNA, which was equivalent to about 34 copies of genome of C. <span class="trait">kikuchii.</span> Using this qPCR assay, we documented a very long latent infection period for C. <span class="trait">kikuchii</span> in soybean leaves beginning at the <span class="gene">V3</span> growth stage (as early as 22 days after planting). The levels of biomass of C. <span class="trait">kikuchii</span> remained low until R1, and a rapid increase was detected from the R2/R3 to R4/R5 growth stages shortly before the appearance of symptoms at R6. The efficacy of various fungicide regimens under field conditions also was evaluated over a 3-year period using this qPCR method. Our results showed that multiple fungicide applications beginning at R1 until late reproductive stages suppressed the development of C. <span class="trait">kikuchii</span> in leaves and delayed symptom expression. Different fungicide chemistries also had differential effects on the amount of latent infection and symptom expression during late reproductive growth stages.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of temperature during soybean seed development on defense-related gene expression and fungal pathogen accumulation.</h2>
			<h3>PMID:21826398  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Upchurch RG(1), Ramirez ME.
				<br/> 
				年份：2011 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr] plants were exposed to three temperature regimens during seed development to investigate the effect of temperature on the expression of eight defense-related genes and the accumulation of two fungal pathogens in inoculated seeds. In seeds prior to inoculation, either a day/night warm (34/26 °C) or a cool temperature (22/18 °C) relative to normal (26/22 °C) resulted in altered patterns of gene expression including substantially lower expression of PR1, PR3 and PR10. After seed inoculation with <span class="trait">Cercospora</span> <span class="trait">kikuchii,</span> pathogen accumulation was lowest in seeds produced at 22/18 °C in which of all defense genes, <span class="gene">MMP2</span> was uniquely most highly induced. For seeds inoculated with Diaporthe phaseolorum, pathogen accumulation was lowest in seeds produced at 34/26 °C in which of all defense genes, PR10 was uniquely most highly induced. Our detached seed assays clearly demonstrated that the temperature regimens we applied during seed development produced significant changes in seed defense-related gene expression both pre- and post inoculation and our findings support the hypothesis that global climate change may alter plant-pathogen interactions and thereby potentially crop productivity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:More <italic> <span class="trait">Cercospora</span> </italic> Species Infect Soybeans across the Americas than Meets the Eye</h2>
			<h3>PMID:26252018</h3>
				
					<h3>PMCID:4529236</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Soares </surname>  <given-names> Ana Paula Gomes </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guillin </surname>  <given-names> Eduardo A. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Borges </surname>  <given-names> Leandro Luiz </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> da Silva </surname>  <given-names> Amanda C. T. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Almeida </surname>  <given-names> &#x000c1;lvaro M. R. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grijalba </surname>  <given-names> Pablo E. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gottlieb </surname>  <given-names> Alexandra M. </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bluhm </surname>  <given-names> Burton H. </given-names>  </name>  <xref ref-type="aff" rid="aff007">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Luiz Orlando </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Fungal mycelia were collected from cultures grown on potato dextrose agar (PDA) [ <xref rid="pone.0133495.ref018" ref-type="bibr"> 18 </xref> ]. Total genomic DNA was extracted from mycelia using either a method described previously [ <xref rid="pone.0133495.ref027" ref-type="bibr"> 27 </xref> ] or the Miniprep DNeasy Plant Mini Kit (QIAgen). Polymerase chain reactions (PCR) were performed to amplify seven nuclear gene regions and the mitochondrial ( <italic> cyb </italic> ) gene region. The seven nuclear gene regions included the five regions previously used to obtain a comprehensive phylogeny of <italic> <span class="trait">Cercospora</span> </italic> [ <xref rid="pone.0133495.ref006" ref-type="bibr"> 6 </xref> ]. The protocols, primers, and PCR conditions to amplify <italic> act </italic> , <italic> <span class="gene">cal</span> </italic> , <italic> his </italic> , ITS, and <italic> <span class="gene">tef</span> </italic> were essentially those described previously [ <xref rid="pone.0133495.ref006" ref-type="bibr"> 6 </xref> ]. For the amplification of the <italic> cyb </italic> gene region, we followed the instructions of a previous study [ <xref rid="pone.0133495.ref028" ref-type="bibr"> 28 </xref> ]. Detailed information on primer design and usage is provided ( <xref rid="pone.0133495.s002" ref-type="supplementary-material"> S1 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple datasets were assembled throughout the study to accommodate the particular requirements of each analysis. From GenBank, we obtained a sequence of <italic> tub </italic> (AY856373) from <italic> C </italic> . <italic> beticola </italic> isolate C-3. Sequences for the remaining five nuclear genes ( <italic> act </italic> , <italic> <span class="gene">cal</span> </italic> , <italic> his </italic> , ITS, and <italic> <span class="gene">tef</span> </italic> ) came from the isolate CPC 11557, which is the strain type of <italic> C </italic> . <italic> beticola </italic> [ <xref rid="pone.0133495.ref006" ref-type="bibr"> 6 </xref> ]. The six sequences of <italic> C </italic> . <italic> beticola </italic> were concatenated and included together with our own set of corresponding sequences to form dataset S1 (N = 27; 3615 bp). In addition to <italic> C </italic> . <italic> beticola </italic> (used as outgroup), therefore, dataset S1 contained 25 fungal isolates associated with CLB and PSS, one isolate of <italic> C </italic> . <italic> sojina </italic> and one isolate of <italic> C </italic> . <italic> beticola </italic> . The sequences were obtained from fungal isolates from Argentina (N = 7), Brazil (11), USA (6), and CPC 5068, the ex-type of <italic> C </italic> . <italic> <span class="trait">kikuchii</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Dataset S2 (N = 60; 1566 bp) contained sequences from a much broader sample of congeneric species. Because sequences of <italic> cfp </italic> and <italic> tub </italic> were not available for all species, we restricted this dataset to sequences of the five nuclear genes (act, <italic> <span class="gene">cal</span> </italic> , <italic> his </italic> , ITS, and <italic> <span class="gene">tef</span> </italic> ) [ <xref rid="pone.0133495.ref006" ref-type="bibr"> 6 </xref> ] together with our own set of corresponding sequences. The sequences belonged to 18 recognized species of <italic> <span class="trait">Cercospora</span> </italic> , with <italic> C </italic> . <italic> apii </italic> as outgroup. Dataset S2 aimed to uncover phylogenetic relationships among soybean-infecting <italic> <span class="trait">Cercospora</span> </italic> and other congeneric species within a broader phylogeny of the genus that has been recently published [ <xref rid="pone.0133495.ref006" ref-type="bibr"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The phylogenetic relationships among the species of <italic> <span class="trait">Cercospora</span> </italic> were inferred by means of Bayesian phylogeny. Subsets of sequences of each of the seven nuclear gene regions were input independently into the software MRMODELTEST v.2.3 [ <xref rid="pone.0133495.ref032" ref-type="bibr"> 32 </xref> ]. For each gene region, the Akaike Information Criterion [ <xref rid="pone.0133495.ref033" ref-type="bibr"> 33 </xref> ] indicated the best-fit models among 24 models of molecular evolution as following: GTR+G ( <italic> act </italic> ), GTR+I ( <italic> <span class="gene">cal</span> </italic> , <italic> his </italic> , ITS, and <italic> <span class="gene">tef</span> </italic> ), HKY+G ( <italic> cfp </italic> ), and GTR+I+G ( <italic> tub </italic> ). The partitioned analysis was performed in MRBAYES v3.1.2 [ <xref rid="pone.0133495.ref034" ref-type="bibr"> 34 </xref> ] with dataset S1. The analysis was carried out using two simultaneous runs of five million generations each, with one cold and seven heated chains in each run; the temperature parameter was set to 0.25 (Temp = 0.25) and the branch length prior mean was set to 0.01 (brlenspr = Unconstrained:Exponential(100)). Trees were sampled once every 5,000 generations.
					</p>
				
					<p>&nbsp;&nbsp;
						A second phylogenetic analysis investigated the relationships of <italic> <span class="trait">Cercospora</span> </italic> and 18 closely-related, congeneric species. The partitioned analysis was performed in MRBAYES v3.1.2 with Dataset S2 and the following models: HKY+G ( <italic> act </italic> ), GTR+G ( <italic> <span class="gene">cal</span> </italic> ), GTR+I ( <italic> his </italic> ), GTR+I (ITS), and GTR+I+G ( <italic> <span class="gene">tef</span> </italic> ). This analysis was carried out using two simultaneous runs of 20 million generations each, with one cold and seven heated chains in each run; the temperature parameter was set to 0.25 and the branch length prior mean was set to 0.01. Trees were sampled once every 20,000 generations.
					</p>
				
					<p>&nbsp;&nbsp;
						The dataset was 3615 bases long and resulted from the concatenation of seven nuclear genes ( <italic> act </italic> , <italic> <span class="gene">cal</span> </italic> , <italic> his </italic> , ITS, <italic> <span class="gene">tef</span> </italic> , <italic> cfp </italic> , and <italic> tub </italic> ). <italic> <span class="trait">Cercospora</span> sojina </italic> is shown for reference purpose; the tree was rooted in <italic> C </italic> . <italic> beticola </italic> . Branch lengths are drawn to scale; nodal support values are given as posterior probabilities (%) above the branches (when &#x02265;85%). Scale <span class="gene">bar</span> corresponds to the expected number of substitutions per site. Color in terminals according to the origin of the isolates: blue, Argentina; green, Brazil; red, United States. Color in branches according to lineage: blue, lineage 1; green, lineage 2; orange, lineage 3; and red, lineage 4. Black arrow indicates the ex-type strain of <italic> C </italic> . <italic> <span class="trait">kikuchii</span> </italic> (CPC_5068). Gene codes: <italic> act </italic> , actin; <italic> <span class="gene">cal</span> </italic> , calmodulin; <italic> his </italic> , histone H3; ITS: internal transcribed spacers and intervening 5.8S nrDNA; <italic> <span class="gene">tef</span> </italic> , translation elongation factor 1-alpha; <italic> cfp </italic> , cercosporin facilitator protein; <italic> tub </italic> , &#x003b2;-tubulin 1.
					</p>
				
					<p>&nbsp;&nbsp;
						Fungal isolates were obtained from either purple-stained soybean seeds or infected leaves. The dataset was1566 bases long and resulted from the concatenation of five nuclear genes (act, cal, his, ITS, and tef). Branch lengths are drawn to scale; nodal support values are given as posterior probabilities (%) above the branches (when &#x0003e; 85%). Scale <span class="gene">bar</span> corresponds to the expected number of substitutions per site. Color in terminals according to the origin of the isolates: blue, Argentina; green, Brazil; red, United States; and black, data from 6]. Color in branches according to lineage: blue, lineage 1; green, lineage 2; orange, lineage 3; and red, lineage 4. Black arrow indicates the ex-type strain of <italic> C </italic> . <italic> <span class="trait">kikuchii</span> </italic> (CPC_5068). Gene codes: <italic> act </italic> , actin; <italic> <span class="gene">cal</span> </italic> , calmodulin; <italic> his </italic> , histone H3; ITS, internal transcribed spacers and intervening 5.8S nrDNA; <italic> <span class="gene">tef</span> </italic> , translation elongation factor 1-alpha.
					</p>
				
					<p>&nbsp;&nbsp;
						The dataset was 714 bases. <italic> <span class="trait">Cercospora</span> sojina </italic> is shown for reference purpose; the tree was rooted in <italic> C </italic> . <italic> beticola </italic> . Branch lengths are drawn to scale; nodal support values are given as posterior probabilities (%) above the branches (when &#x02265;85%). Scale <span class="gene">bar</span> corresponds to the expected number of substitutions per site. Color in ingroup terminals according to the origin of the isolates: blue, Argentina; green, Brazil; red, United States; black, Japan. Color in ingroup branches according to lineage: blue, lineage 1; green, lineage 2; orange, lineage 3; and red, lineage 4. Black arrow indicates the ex-type strain of <italic> C </italic> . <italic> <span class="trait">kikuchii</span> </italic> (CPC_5068).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>