<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Diseases and Pests resistance</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Diseases and Pests resistance</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:The receptor like kinase at <italic> <span class="gene">Rhg1</span> </italic> -a <italic> /Rfs2 </italic> caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome <span class="trait">and</span> soybean cyst nematode as a transgene by altering signaling responses</h2>
			<h3>PMID:22857610</h3>
				

				
					<h3>PMCID:3439264</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Srour </surname>  <given-names> Ali </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> asrour@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Afzal </surname>  <given-names> Ahmed J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> afzal.5@osu.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Blahut-Beatty </surname>  <given-names> Laureen </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> Laureen.Blahut-Beatty@AGR.GC.CA </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Hemmati </surname>  <given-names> Naghmeh </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> naghmeh@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Simmonds </surname>  <given-names> Daina H </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> daina.simmonds@agr.gc.ca </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> wenbinli@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Liu </surname>  <given-names> Miao </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> wenbinli@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Town </surname>  <given-names> Christopher D </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> cdtown@jcvi.org </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Sharma </surname>  <given-names> Hemlata </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> hemlatarca@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Arelli </surname>  <given-names> Prakash </given-names>  </name>  <xref ref-type="aff" rid="I8"> 8 </xref>  <email> prakash.arelli@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Lightfoot </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <xref ref-type="aff" rid="I9"> 9 </xref>  <email> ga4082@siu.edu </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max (L. Merr.)) <span class="trait">resistance</span> to any population of Heterodera glycines (I.), or Fusarium virguliforme (Akoi, O'Donnell, Homma & Lattanzi) required a functional allele at Rhg1/Rfs2. H. glycines, the soybean cyst nematode (SCN) was an ancient, endemic, pest of soybean whereas F. virguliforme causal agent of sudden death syndrome (SDS), was a recent, regional, pest. This study examined the role of a receptor like kinase (RLK) <span class="gene">GmRLK18-1</span> (gene model Glyma_18_02680 at 1,071 kbp on chromosome 18 of the genome sequence) within the Rhg1/Rfs2 locus in causing <span class="trait">resistance</span> to SCN <span class="trait">and</span> SDS. RESULTS: A BAC (B73p06) encompassing the Rhg1/Rfs2 locus was sequenced from a resistant cultivar <span class="trait">and</span> compared to the sequences of two susceptible cultivars from which 800 SNPs were found. Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59 kbp at the center of which the <span class="gene">GmRLK18-1</span> was the most polymorphic gene <span class="trait">and</span> encoded protein. Analyses were made of plants that were either heterozygous at, or transgenic <span class="trait">(and</span> so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <span class="gene">GmRLK18-1.</span> Those plants infested with either H. glycines or F. virguliforme showed that the allele for <span class="trait">resistance</span> was dominant. In the absence of <span class="gene">Rhg4</span> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root <span class="trait">and</span> leaf symptoms of SDS caused by F. virguliforme <span class="trait">and</span> provided partial <span class="trait">resistance</span> to three different populations of nematodes (mature female cysts were reduced by 30-50%). In the presence of <span class="gene">Rhg4</span> the plants with the transgene were nearly classed as fully resistant to SCN (females reduced to 11% of the susceptible control) as well as SDS. A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <span class="gene">GmRLK18-1.</span> Field trials of transgenic plants showed an increase in foliar susceptibility to insect herbivory. CONCLUSIONS: The inference that soybean has adapted part of an existing pathogen recognition <span class="trait">and</span> defense cascade (H.glycines; SCN <span class="trait">and</span> insect herbivory) to a new pathogen (F. virguliforme; SDS) has broad implications for crop improvement. Stable <span class="trait">resistance</span> to many pathogens might be achieved by manipulation the genes encoding a small number of pathogen recognition proteins.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> (L. Merr.)) <span class="trait">resistance</span> to any population of <italic> Heterodera glycines </italic> (I.), or <italic> Fusarium virguliforme </italic> (Akoi, O&#x02019;Donnell, Homma &#x00026; Lattanzi) required a functional allele at <italic> Rhg1/Rfs2 </italic> . <italic> H. glycines </italic> , the soybean cyst nematode (SCN) was an ancient, endemic, pest of soybean whereas <italic> F. virguliforme </italic> causal agent of sudden death syndrome (SDS), was a recent, regional, pest. This study examined the role of a receptor like kinase (RLK) <italic> <span class="gene">GmRLK18-1</span> </italic> (gene model <italic> Glyma_18_02680 </italic> at 1,071&#x02009;kbp on chromosome 18 of the genome sequence) within the <italic> Rhg1/Rfs2 </italic> locus in causing <span class="trait">resistance</span> to SCN <span class="trait">and</span> SDS.
					</p>
				
					<p>&nbsp;&nbsp;
						A BAC (B73p06) encompassing the <italic> Rhg1/Rfs2 </italic> locus was sequenced from a resistant cultivar <span class="trait">and</span> compared to the sequences of two susceptible cultivars from which 800 SNPs were found. Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59&#x02009;kbp at the center of which the <italic> <span class="gene">GmRLK18-1</span> </italic> was the most polymorphic gene <span class="trait">and</span> encoded protein. Analyses were made of plants that were either heterozygous at, or transgenic <span class="trait">(and</span> so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <italic> <span class="gene">GmRLK18-1</span> </italic> . Those plants infested with either <italic> H. glycines </italic> or <italic> F. virguliforme </italic> showed that the allele for <span class="trait">resistance</span> was dominant. In the absence of <italic> <span class="gene">Rhg4</span> </italic> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root <span class="trait">and</span> leaf symptoms of SDS caused by <italic> F. virguliforme </italic> <span class="trait">and</span> provided partial <span class="trait">resistance</span> to three different populations of nematodes (mature female cysts were reduced by 30&#x02013;50%). In the presence of <italic> <span class="gene">Rhg4</span> </italic> the plants with the transgene were nearly classed as fully resistant to SCN (females reduced to 11% of the susceptible control) as well as SDS. A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <italic> <span class="gene">GmRLK18-1</span> </italic> . Field trials of transgenic plants showed an increase in foliar susceptibility to insect herbivory.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Heterodera glycines </italic> I., the soybean cyst nematode (SCN) was probably an ancient pest of soybean since complete resistances to some Hg Types of SCN was found in about 1% of pre-domesticated <span class="trait">and</span> early domesticated Plant Introductions (PIs) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Interestingly most of these PIs were also partially resistant to SDS [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. One locus, the <italic> Rfs2/Rhg1 </italic> locus on chromosome 18, was shown to underlie coinheritance of <span class="trait">resistance</span> to SDS in the roots <span class="trait">and</span> also reduce root infestation by SCN [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> - <xref ref-type="bibr" rid="B17"> 17 </xref> ]. Fine map development did not resolve <italic> Rfs2 </italic> from <italic> <span class="gene">Rhg1</span> </italic> suggesting the underlying gene(s) were either very closely linked or pleiotropic [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Equally the <italic> <span class="gene">Rhg1</span> </italic> locus underlay partial <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> ]. <italic> H.glycines </italic> , like many plant parasitic nematodes, were obligate endoparasites of plant roots. Like <italic> F. virguliforme </italic> <italic> H. glycines </italic> has many alternate hosts. Over the past 50&#x02009;years, the number of Hg Types (ex. races) has expanded from 4 in the 1960&#x02019;s to 16&#x02013;20 [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B20"> 20 </xref> ]. However, the <italic> <span class="gene">Rhg1</span> </italic> locus was constant, being required for partial <span class="trait">resistance</span> to all Hg Types in most PIs <span class="trait">and</span> cultivars. Full <span class="trait">resistance</span> to SCN required 1&#x02013;4 loci in addition to <italic> <span class="gene">Rhg1</span> </italic> , the number depending on the nature of the cyst population parasitizing the roots [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Genetic diversity was found among SCN isolates, even inbred cyst populations like PA3, Hg type 0 [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Further, variation among the host plant roots response to SCN has been associated with temperature [ <xref ref-type="bibr" rid="B27"> 27 </xref> ] such that environmental conditions must be rigorously controlled during assays [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">resistance</span> or susceptible interaction(s) between the nematode <span class="trait">and</span> soybean affected by <italic> <span class="gene">Rhg1</span> </italic> was not induced until females stopped moving through the roots <span class="trait">and</span> established a feeding site comprising several giant cells [ <xref ref-type="bibr" rid="B28"> 28 </xref> - <xref ref-type="bibr" rid="B30"> 30 </xref> ]. Full <span class="trait">resistance</span> to SCN, based on the combined action of the genes at <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">and</span> one or more additional <italic> Rhg </italic> loci, was manifest as; cell wall appositions to surround the feeding site; failure to supply the feeding site a tracheary element; <span class="trait">and</span> a necrosis as the feeding site develops. If the <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg1</span> </italic> was present normal rates of root growth were slightly depressed but the above ground stunting, yellowing <span class="trait">and</span> early senescence did not occur.
					</p>
				
					<p>&nbsp;&nbsp;
						Inheritance of <span class="trait">resistance</span> to SCN was first reported in the PI &#x02018;Peking&#x02019; [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Three recessive loci ( <italic> rhg1 </italic> <italic> rhg3 </italic> ) <span class="trait">and</span> a dominant locus ( <italic> <span class="gene">Rhg4</span> </italic> ) were assigned gene names by parsimony though other dominance models were equally likely. The Peking derived <span class="trait">resistance</span> alleles of <italic> rhg1 </italic> <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> were introgressed into the cv. &#x02018;Forrest&#x02019; [ <xref ref-type="bibr" rid="B31"> 31 </xref> - <xref ref-type="bibr" rid="B33"> 33 </xref> ]. In crosses based on Forrest <span class="trait">and</span> with SCN isolate PA3 the <italic> rhg1-a </italic> was shown to be codominant with the susceptibility allele of &#x02018;Essex&#x02019; ( <italic> rhg1 </italic> -e) <span class="trait">and</span> alone capable of providing partial <span class="trait">resistance.</span> Consequently, the Forrest, &#x02018;Hartwig&#x02019;, Peking <span class="trait">and</span> &#x02018;PI 437654&#x02019; allele was renamed to <italic> Rhg1-a </italic> ([ <xref ref-type="bibr" rid="B17"> 17 </xref> ] <span class="trait">and</span> hereafter) by the Soybean Genetics Committee.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus was located to a sub-telomeric region of the soybean chromosome 18 (molecular linkage group G; Lg G) by many studies [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> ]. All of these segregating populations that were later tested with <italic> F. virguliforme </italic> also had an <italic> Rfs2- </italic> like activity against SDS [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B34"> 34 </xref> ]. However, an <italic> <span class="gene">Rhg1</span> </italic> -like locus was found at other locations in a few SCN resistant PIs, including Lg B1 (chromosome 11) [ <xref ref-type="bibr" rid="B22"> 22 </xref> ], mid LgG [ <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> ] <span class="trait">and</span> Lg B2 (chromosome 14) [ <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> ]. The effects of the <italic> <span class="gene">Rhg1</span> </italic> -like loci found at other locations than chromosome 18 on <span class="trait">resistance</span> to SDS were not reported by 2012.
					</p>
				
					<p>&nbsp;&nbsp;
						It has been shown the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> -a <italic> /Rfs2 </italic> (from Peking) was associated with reduced seed <span class="trait">yield</span> when SCN is not present in the fields [ <xref ref-type="bibr" rid="B39"> 39 </xref> - <xref ref-type="bibr" rid="B41"> 41 </xref> ]. That phenomenon might be related to delayed seedling development <span class="trait">and</span> stand formation [ <xref ref-type="bibr" rid="B42"> 42 </xref> ]. How the root reduction contributes to <span class="trait">resistance</span> may involve a locus on chromosome 7 (LG M) where <italic> Rzd </italic> , an interacting allele needed for <underline> r </underline> esistance to <underline> z </underline> ygote <underline> d </underline> eath, was strictly co-inherited in phase with <italic> Rhg1/Rfs2 </italic> -a [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]. Therefore, effects on development were predicted for the gene(s) underlying <italic> Rhg1-a/Rfs2-a </italic> [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B43"> 43 </xref> - <xref ref-type="bibr" rid="B47"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Fine scale genetic maps <span class="trait">and</span> BAC based genomic analysis identified a 42&#x02009;kbp region encompassed in BAC B73p06 as the <italic> Rhg1/Rfs2 </italic> locus [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. The region encoded an <span class="gene">RLK</span> ( <italic> <span class="gene">Glyma18g02680</span> </italic> named <italic> <span class="gene">GmRLK18-1</span> </italic> hereafter), a variant laccase ( <italic> <span class="gene">Glyma18g02690</span> </italic> ) <span class="trait">and</span> a predicted Na/H ion antiporter ( <italic> <span class="gene">Glyma18g0270</span> </italic> ). The <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> were expressed in roots, shoots <span class="trait">and</span> flowers but not nodules or seeds. The antiporter was not expressed in any organ tested in Essex or Forrest. However, it might be expressed in flowers <span class="trait">and</span> seed at very low abundance [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. Many nucleotide differences were found in the region encompassing the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> genes from fragments of sequences from the resistant allele in Forrest compared to susceptible genotypes &#x02018;Asgrow 3244&#x02019; <span class="trait">and</span> &#x02018;Williams 82&#x02019; [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ]. However, the <italic> <span class="gene">GmRLK18-1</span> </italic> was considered the most likely candidate gene underlying <italic> <span class="gene">Rhg1</span> </italic> -a based on fine maps <span class="trait">and</span> association analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						RLKs are part of the eukaryotic tumor necrosis factor beta receptor super-family [ <xref ref-type="bibr" rid="B44"> 44 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. In plants they represent a major class of <span class="trait">resistance</span> genes <span class="trait">and</span> also a major class of developmental regulators. The <italic> <span class="gene">GmRLK18-1</span> </italic> gene at <italic> Rhg1/Rfs2 </italic> had nine alleles recognized among PIs <span class="trait">and</span> cultivars [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> - <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Of those four alleles were associated with partial <span class="trait">resistance</span> to SCN <span class="trait">and</span> five with susceptibility. The second most important <span class="trait">resistance</span> allele was from PI88788 <span class="trait">and</span> named <italic> rhg1 </italic> -b because it was recessive <span class="trait">and</span> discovered second [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. However, a small scale experiment with RNAi to the <span class="gene">RLK</span> (named <italic> <span class="gene">GmRLK18-1</span> </italic> -b) at the <italic> rhg1 </italic> -b in transgenic hairy roots did not show a large effect on cyst numbers [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. Either dominance or incomplete inhibition of the protein might have occurred. Here, additional experiments were undertaken with the <italic> <span class="gene">GmRLK18-1</span> </italic> at <italic> <span class="gene">Rhg1</span> </italic> -a in stable transformed soybean lines. Here, a molecular basis for <span class="trait">resistance</span> to SDS <span class="trait">and</span> SCN was inferred from functional analyses of the <italic> Rhg1/Rfs2 </italic> -a locus in near isogenic lines (NILs) <span class="trait">and</span> the Forrest allele of the <italic> <span class="gene">GmRLK18-1</span> </italic> gene in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The Forrest <span class="trait">(resistance)</span> allele of the <italic> Rhg1/Rfs2 </italic> locus was analyzed by sequencing the entire BAC B73p06 to 8 fold redundancy (Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ; GenBank HQ008938). The BAC sequence was compared to the sequences of (susceptible) Williams 82 whole genome shotgun sequence [ <xref ref-type="bibr" rid="B54"> 54 </xref> ] <span class="trait">and</span> a BAC contig developed from (susceptible) &#x02018;Asgrow 3244&#x02019; [ <xref ref-type="bibr" rid="B50"> 50 </xref> ] from the same region. The BAC insert encompassed 82,157&#x02009;bp that was predicted to encode nine genes comprised of 57 exons (2&#x02013;8 per gene; Table&#x02009; <xref ref-type="table" rid="T1"> 1 </xref> ). The <italic> <span class="gene">GmRLK18-1</span> </italic> was gene 5, the laccase gene 6 <span class="trait">and</span> the antiporter gene 7. Sequences of the resistant <span class="trait">and</span> susceptible alleles were colinear with no large (more than 100&#x02009;bp) insertions, deletions or inversions. Across the entire BAC there were 800 SNPs between Forrest <span class="trait">and</span> the sequences of alleles for susceptibility. SNPs were found in the promoter <span class="trait">and</span> enhancer regions of all 9 genes. However there were only 31 SNPs within genes. Only 11 of the 31 caused amino acid changes. There were just 6 of the 9 proteins changed by those 11 SNPs.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Marker map of the genomic region around </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> &#x02009; </bold> <bold> <span class="trait">and</span> the most identical paralog (homeolog) of </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> &#x02009; </bold> <bold> with locus, BAC <span class="trait">and</span> gene ideograms. </bold> Panel <bold> A </bold> shows the marker map of the genomic region around Rhg1/Rfs2 (Lg G; chromosome 18) with locus ideograms. Sequence coordinates were from the susceptible cultivar Asgrow 3244 [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> ]. The <italic> <span class="gene">GmRLK18-1</span> </italic> gene encoding the <span class="gene">RLK</span> was shown as a black block arrow. The genes encoding the laccase <span class="trait">and</span> antiporter were shown as opposite white block arrows. All other genes were shown as grey block arrows. Locations of overlapping BAC clones B73p06 <span class="trait">and</span> B21d09 that both encoded the <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus (Lg G; chromosome 18) were shown below the ideogram. Panel <bold> B </bold> shows the BAC clone B73p06 that encoded the <italic> Rhg1/Rfs2 </italic> -a locus. The gene encoding the <span class="gene">RLK</span> was shown as a black block arrow. The genes encoding the laccase <span class="trait">and</span> antiporter were shown as opposite white block arrows. All other genes were shown as grey block arrows. The extent of the pSBHB94 (HQ008939), the 9.772 kbp subclone from BAC B21d09 that was used for soybean transformations was shown as a blue arrow. The plasmid pSBHB94 encompassed from 30,423- 40&#x02013;194&#x02009;bp. Sequence coordinates were from the complete sequence of the BAC derived from resistant cultivar Forrest (HQ008938). Panel <bold> C </bold> showed a syntenic homeolog of <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2/ </italic> found in the sequence of BAC H38F23 from Lg B1 (chromosome 11). The homeolog of the gene encoding the <span class="gene">RLK</span> was show as a black block arrow. The homeologs of the genes encoding the laccase <span class="trait">and</span> antiporter were shown as opposite white block arrows. All other syntenic genes were shown as grey block arrows. The marker TMD1 amplified a fragment from <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2/Rhg1 </italic> -a <italic> /Rfs2 </italic> of 303+ 15&#x02009;bp (resistant allele was the smaller) <span class="trait">and</span> of 362&#x02009;bp from a syntenic homeolog of <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2Rfs2/ </italic> found in the sequence of BAC H38f23 from Lg B1 (chromosome 11). Sequence coordinates were from the complete sequence of the BAC derived from resistant cultivar Forrest (HQ008940). Panel <bold> D </bold> shows an ideogram of markers, the <italic> <span class="gene">GmRLK18-1</span> </italic> gene <span class="trait">and</span> gene features found in the insert of plasmid pSBHB94 that encompassed from 30,423- 40,194&#x02009;bp of B73p06. Panel <bold> E </bold> shows the position of markers in the transcribed region of the <italic> <span class="gene">GmRLK18-1</span> </italic> gene. Panel <bold> F </bold> shows the position of amino acid substitutions in the protein encoded by the cDNA.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Genes predicted from DNA sequence within the </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> -a/ </bold> <bold> &#x02009; <italic> Rfs2 </italic> &#x02009; </bold> <bold> locus encoded by BAC 73p06 (81, 157&#x02009;bp) from Forrest (resistant) <span class="trait">and</span> sequence contig from A3244 (susceptible; AX196295) </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">RLK</span> contained 2 of the 6 SNPs in the BAC predicted to cause amino acid changes (Ruben et al. 2006; Afzal et al. 2012) when comparing Forrest allele to Williams 82 <span class="trait">and</span> Asgrow 3244. However, it contained 7 synonymous SNPs. In comparison the laccase contained 3 SNPs predicted to cause amino acid changes <span class="trait">and</span> only one that was synonymous (Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ). The antiporter alloproteins contained one non-synonymous <span class="trait">and</span> two synonymous SNPs.
					</p>
				
					<p>&nbsp;&nbsp;
						The greater number of differences between alleles of the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> the other candidate genes was taken as evidence this should be the first gene tested in transgenic plants. The <span class="gene">RLK</span> was subcloned on a 9,772&#x02009;bp insert for plant transformation (Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> D) that contained 6&#x02009;Kbp of 5&#x02032; sequence, 2.7&#x02009;kbp of geneic sequence <span class="trait">and</span> 1&#x02009;kbp of 3&#x02032;sequence. The 5&#x02032; sequence extended to the TATA box of the neighboring gene (#4). The region contained 7 potential cis-regulatory elements (CREs) of 8&#x02009;bp that were identical to motifs found in the 5&#x02032; regions hundreds of plant genes. The four closest to the <span class="gene">RLK</span> are shown in Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> D. Only the most distant was polymorphic between cultivars. The Forrest allele of this CRE was the identical to sequences found 5&#x02032; to 1,091 rice genes <span class="trait">and</span> 108 Arabidopsis genes. There was a 53&#x02009;bp deletion from Forrest (at 33,942&#x02009;bp) compared to Essex, Williams 82 <span class="trait">and</span> Asgrow 3244 in the enhancer region of the gene that was named SIUC-indel-(plus)&#x02009;+&#x02009;3.5. There was also a 22&#x02009;bp deletion 3&#x02032; to the gene at (at 40,215&#x02009;bp) named SIUC-indel (minus)-3.5. This deletion was within a complex microsatellite repeat mainly composed of AAAG motifs. Neither deletion appeared to encompass any previously characterized CREs. In the intergeneic region between <span class="gene">RLK</span> <span class="trait">and</span> laccase were four polymorphic small indels or microsatellite-like polymorphisms.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean genome was hypothesized to be the product of a diploidized tetraploid [ <xref ref-type="bibr" rid="B54"> 54 </xref> , <xref ref-type="bibr" rid="B55"> 55 </xref> ]. Therefore, a detailed molecular analysis of the <italic> Rhg1/Rfs2 </italic> locus required that paralogs <span class="trait">and</span> syntenic gene clusters be identified. The most identical paralog (the homeolog) was found on BAC H38f23 (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S1) that was from Lg B1 (chromosome 11) in a region where loci with functions similar to <italic> Rhg1/Rfs2 </italic> were mapped [ <xref ref-type="bibr" rid="B22"> 22 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> ]. Significantly, the sequence of H38f23 contained a complete set of syntenic genes for a second <italic> Rhg1/Rfs2 </italic> locus <span class="trait">and</span> surrounding genes (an RLK, laccase, both antiporters, the kinase <span class="trait">and</span> the helicase; <italic> <span class="gene">Glyma11g35710-</span> <span class="gene">Glyma11g35660</span> </italic> ; Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ; HQ008940). Interrogation of Soybase showed the transcript abundance patterns for the syntenic <span class="gene">RLK</span> pair <span class="trait">and</span> laccase pair were both abundant in roots as reported in the RNA sequence atlas generated from a Williams 82 sister line derived NIL (cultivar &#x02018;P-C609-45-2-2&#x02019; a BC5F5 plant derived from <italic> G. soja </italic> (&#x02018;PI 468916&#x02019;) backcrossed into <italic> G. max </italic> (&#x02018;A81-356022&#x02019;) [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. The laccase transcripts were 15 fold more than either of the <span class="gene">RLK</span> transcripts in roots. Though it must be noted this genotype was either the <italic> rhg1 </italic> -d or <italic> <span class="gene">Rhg1</span> </italic> -e allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparing the syntenic BACs the DNA sequence identity was high (~97% in geneic regions; Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S1E; Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S1). At the RLKs, <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> , with intron boundaries as described by [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] <span class="trait">and</span> [ <xref ref-type="bibr" rid="B25"> 25 </xref> ], the amino acid identity was 93% overall with 94% in the LRR, 93% in the trans-membrane domain <span class="trait">and</span> 97% in the kinase domain (Additional file <xref ref-type="supplementary-material" rid="S2"> 1 </xref> : Table S1). The N terminal signal peptide was most divergent (11 changes in 50 amino acids). The laccase <span class="trait">and</span> the antiporter also showed 85&#x02013;96% amino acid identity with their syntenic paralogs.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmRLK11-1</span> </italic> &#x02019;s amino acid sequence was less identical to <italic> <span class="gene">GmRLK18-1</span> </italic> than the alleles (98&#x02013;99%) of the gene. However, four of the six residues that differ among alloproteins of <italic> <span class="gene">GmRLK18-1</span> </italic> were identical to the Forrest allele in the homeologous protein <span class="gene">GmRLK11-1-a.</span> At the other 2 residues the changes were identical to the susceptible alleles of Essex <span class="trait">and</span> Williams 82. From sequence comparison <span class="trait">and</span> transcript abundances it appears likely the paralogs <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> would share functions <span class="trait">and</span> should hetero-dimerize [ <xref ref-type="bibr" rid="B47"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Since paralogs with homeolog sequence variants (HSVs) appeared to exist for each gene in the cluster, it was important to distinguish alleles precisely <span class="trait">and</span> separately from their most similar HSVs. In the NIL population <span class="gene">RLK</span> alleles <span class="trait">and</span> HSVs were distinguished using both the SIUC-TMD1 marker (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ) <span class="trait">and</span> SNP Ala87Val from the LRR region by Taqman (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> : Figure S2; Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2). In genomic DNA TMD1 distinguished both the Peking <span class="trait">and</span> Peking-like allele of <italic> <span class="gene">GmRLK18-1</span> </italic> from the other 7 alleles <span class="trait">and</span> the homeologous alleles of <italic> <span class="gene">GmRLK11-1</span> </italic> . To distinguish Forrest from both Essex <span class="trait">and</span> X5 the SNP probe to Ala87Val was again used for both genomic DNA <span class="trait">and</span> cDNA.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> &#x02009; </bold> <bold> genomic region altered root development. </bold> Panel <bold> A </bold> ; soybean NILs at 2&#x02009;weeks pre-SCN inoculation show different root morphologies, <italic> Rhg1/Rfs2 </italic> inhibited germination <span class="trait">and</span> early root growth. Panel <bold> B </bold> ; the root morphologies co-segregated with the allele of <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus as shown by the 3 bands generated by intron located marker TMD1 (arrowed). R indicated the resistant allele from Forrest; <bold> S </bold> the susceptible allele from Essex; <span class="trait">and</span> H indicated the heterozygous state for the <italic> Rhg1/Rfs2 </italic> locus. Panel <bold> C </bold> showed the distribution of root mass (rfw was root fresh weight) among 20 heterozygous plants. Panel <bold> D </bold> showed the distribution of root mass among 20 susceptible plants. Panel E showed the distribution of root mass among 20 resistant plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The distribution of recombination events found previously among the six Hg Type 0 susceptible PIs [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] suggested that the action of <italic> Rhg1/Rfs2 </italic> required elements to the 3&#x02032; side of the <span class="gene">RLK</span> intron or in the C terminal portion of the protein. One of the 2 amino acid substitutions found in the intracellular kinase of the complete <span class="gene">RLK</span> (Gly539Ala <span class="trait">and</span> Ser770Pro) may be key to transducing intracellular signaling leading to the <span class="trait">resistance</span> response [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. Some mutations in the kinases of other plant RLKs are known to be lethal [ <xref ref-type="bibr" rid="B58"> 58 </xref> - <xref ref-type="bibr" rid="B60"> 60 </xref> ]. Therefore, it may be the kinase at the <italic> Rhg1/Rfs2 </italic> locus that underlies restricted root growth in resistant genotypes. The effect may be direct or occur after protein-protein interaction(s). The signal transduced is likely to result in a negative reponse since the kinase domain lacks the RD amino acid motif needed for ATP binding [ <xref ref-type="bibr" rid="B44"> 44 </xref> ] <span class="trait">and</span> so protein phosphorylation.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression judged by examination of EST libraries in silico, cDNA libraries by hybridization, RNA sequencing <span class="trait">and</span> mRNA populations assayed by qRT-PCR showed both transcripts encoded by <italic> <span class="gene">GmRLK18-1</span> </italic> -a <span class="trait">and</span> <italic> <span class="gene">GmRLK18-1</span> </italic> -e alleles (Figure&#x02009; <xref ref-type="fig" rid="F3"> 3 </xref> ) were present in both non-infested roots <span class="trait">and</span> SCN-infested roots, as reported in [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. The transcript <span class="trait">and</span> protein abundances were not increased by more than 2 fold in either <italic> H. glycines </italic> or <italic> F. virguliforme </italic> infested plants compared to non-infested plants (Figure&#x02009; <xref ref-type="fig" rid="F3"> 3 </xref> ). Low transcript <span class="trait">and</span> protein abundances appeared to result from largely constitutive expression under the conditions <span class="trait">and</span> stages of development tested to date.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of Forrest (&#x02212;a) <span class="trait">and</span> Essex (&#x02212;e) alleles of the </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> at </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> </bold> <bold> in non-transgenic soybean NIL roots. </bold> Panel ( <bold> A </bold> ); Western hybridization using an anti-RHG1/RFS2 ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) antibody (Afzal et al. 2007) from roots of; Forrest allele in ExF34-23 (a); Essex allele in ExF34-3(b); <italic> E.coli </italic> expressed proteins RHG1/RFS2-LRR-Shrt (c); RHG1/RFS2-LRR-Long (d); <span class="trait">and</span> RHG 4 (e). Panel ( <bold> B </bold> ); agarose gel electrophoresis of cDNA amplified using <italic> Rhg1/Rfs2 </italic> LRR flanking primers from; non-infested NIL 34&#x02013;23 (1); SCN infested NIL 34&#x02013;23 (2); SCN infested NIL 34&#x02013;3 (3); <span class="trait">and</span> Forrest genomic DNA (4). Negative control without template is shown in lane 5.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to test the hypothesis that <italic> <span class="gene">GmRLK18-1</span> </italic> -a underlay part of the activity of the <italic> Rhg1/Rfs2 </italic> -a locus susceptible plants were made transgenic with this allele. Such plants had a <span class="trait">resistance</span> allele at a new location <span class="trait">and</span> a susceptibility allele ( <italic> <span class="gene">GmRLK18-1</span> </italic> -a; <italic> rhg1/rfs2 </italic> -e) on chromosome 18. Several primary transgenic lines were created by biolistics <span class="trait">and</span> fertile lines selected in two different cultivars (&#x02018;X5&#x02019; <span class="trait">and</span> &#x02018;Westag 97&#x02019;) for analyses of SDS <span class="trait">and</span> SCN responses.
					</p>
				
					<p>&nbsp;&nbsp;
						In several transgenic lines from (T1-T3) used for SDS <span class="trait">and</span> SCN assays, lines were identified with the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene in the homozygous state. In these plants the transgenes were both transcribed <span class="trait">and</span> translated. For example, in the progeny of fertile T0 line 6B3 the progeny T1 lines like 7D2 <span class="trait">and</span> sixteen T3 derived plants expressed the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene as both mRNA <span class="trait">and</span> protein (Figure&#x02009; <xref ref-type="fig" rid="F4"> 4 </xref> ). Expression of the transgene alleles was equal to the endogenous gene alleles as judged by protein abundance. <span class="gene">GmRLK1</span> transcripts <span class="trait">and</span> proteins were detected in about 50% of the progenies of the 6B3 primary transformants. In selfed lines expressing <italic> <span class="gene">GmRLK18-1</span> </italic> -a the transgene allele was detected as distinct mRNAs <span class="trait">and</span> alloproteins. Expression was directed by the native promoter <span class="trait">and</span> the enhancer elements contained on the proximal 6&#x02009;kbp of the 9.772&#x02009;kbp fragment of pSBHB94. That plasmid was sub-cloned from BAC B21d09 (Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ; HQ008939) during BAC sequencing.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Soybean transgenic plants expressed the mRNA <span class="trait">and</span> protein from the Forrest allele of </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> , the <span class="gene">RLK</span> at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus. </bold> Panel <bold> A </bold> shows PCR from leaf samples of progeny plants derived from a primary transgenic event 6B3-7D2(1) with TMD1 primers. Lanes contain transgenic plants 1 to 13. The arrow shows the double band for <span class="gene">Gm18RLK-1-a</span> positive sample at 314&#x02009;bp for lines 1,3&#x02013;5, 7,8,10 <span class="trait">and</span> 12. M was the marker; H was the no DNA (water) control; P was the <italic> <span class="gene">Rhg1</span> </italic> plasmid pSBHB94; X5 was the control plant. Panel <bold> B </bold> shows PCR from cDNA leaf samples of sixteen transgenic lines derived from event 6B3-7D2(1) with HRM primers. Green lines are from transgenic plants. Red melt curve was a resistant control blue line was a susceptible control. Panel <bold> C </bold> shows a Western of a 2D gel from roots of a transgenic plant probed with the anti-RLK peptide antibody. An alloprotein at pI 8.42 <span class="trait">and</span> 92.41&#x02009;kDa was found in the non transgenic cv X5 but the presence of the Forrest alloprotein at pI 8.44 <span class="trait">and</span> <span class="trait">and</span> 92.39&#x02009;kDa was found in transgenic plants derived from event 6B3-7D2(1) expressing <italic> <span class="gene">GmRLK18-1</span> </italic> -a. <span class="gene">GmRLK18-1</span> was shown to be a very low abundance protein impossible to visualize without immune-staining.
					</p>
				
					<p>&nbsp;&nbsp;
						In transgenic soybean plants <italic> <span class="gene">GmRLK18-1</span> </italic> -a allele provided <span class="trait">resistance</span> to both root infection <span class="trait">and</span> root rot by <italic> Fusarium virguliforme. </italic> That root <span class="trait">resistance</span> underlay a significant reduction in leaf symptoms <span class="trait">and</span> delay of the senescence caused by SDS (Figure&#x02009; <xref ref-type="fig" rid="F5"> 5 </xref> ; Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> B). <span class="trait">Resistance</span> to SDS was effective throughout the life of the plants which flowered <span class="trait">and</span> set seed. The non-transgenic X5 plants proved to be highly susceptible to SDS <span class="trait">and</span> showed all the expected phenotypes of root rot <span class="trait">and</span> leaf scorch. The phenotypes among the susceptible plants included a gradual worsening of leaf scorch symptoms from 3.0 at 21&#x02009;days after infection (dai) to 8.5 by 56 dai. Only at near maturity did the susceptible plants showed symptoms characteristic of SDS. For example, leaflet abscission occurred from the top of the petiole instead of the base of the petiole. However, the transgenic plants did not show early senescence or reduced pod set. Equally, the characteristic root symptoms of SDS were very reduced in the <span class="gene">RLK</span> transgenic plants. Symptoms reduced included the degree of root rot <span class="trait">and</span> browning of the root cortex. Senescence was delayed by 14&#x02009;days in <italic> <span class="gene">GmRLK18-1</span> </italic> transgenics compared to controls <span class="trait">and</span> more pods were set in each of the three repeats of the experiment. Therefore, the <italic> <span class="gene">GmRLK18-1</span> </italic> -a provided a very high degree of <span class="trait">resistance</span> to the transgenic plants in roots <span class="trait">and</span> leaves <span class="trait">and</span> pods.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Resistance</span> to </bold> <bold> &#x02009; <italic> Fusarium virguliforme </italic> &#x02009; </bold> <bold> <span class="trait">and</span> partial <span class="trait">resistance</span> to </bold> <bold> &#x02009; <italic> Heterodera glycines </italic> &#x02009; </bold> <bold> caused by the Forrest allele of a receptor like kinase ( </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> -a) found at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus as transgene in primary transgenic lines (cv &#x02018;X5&#x02019;). </bold> Panels <bold> A </bold> - <bold> H </bold> show the SDS assays. <italic> F. virguliforme </italic> was used at 10 <sup> 4 </sup> &#x02009;cfu per cm <sup> 3 </sup> of soil. The experiment was carried out on 3 separate occasions. Leaf scorch was scored as DS at 7, 14, 21, 28, 35, 42, 49 <span class="trait">and</span> 56 dai (days after infestation). Derived crossed lines (X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35 <span class="trait">and</span> cultivars X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35xEF2) were included in runs 2 <span class="trait">and</span> 3. Panel <bold> A </bold> &#x00026; <bold> B </bold> shows stable soybean transgenics with <span class="trait">and</span> without the 10 kbp <italic> <span class="gene">GmRLK18-1</span> </italic> ( <italic> Rhg1/Rfs2) </italic> subclone at 21 dai. Panel <bold> C </bold> shows the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene reduced root rot at 28 dai. Panel <bold> D </bold> shows leaf symptoms at 28 dai. Panel <bold> E </bold> shows plants at 56 dai where X5 is senescent with abscission of leaflets from erect petioles <span class="trait">and</span> X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a is still green <span class="trait">and</span> filling pods. Panel <bold> F </bold> shows selected leaflets at 28 dai with a 1&#x02013;9 range in DS scores arranged in order of severity from bottom left to top right. Panels <bold> G </bold> - <bold> J </bold> show the SCN assays. Panel <bold> G </bold> shows SCN development by the <span class="gene">RLK</span> in X5 transgenics. Panel <bold> H </bold> shows SCN development by the <italic> Rhg1-a </italic> allele in resistant NIL 34&#x02013;23. Panel <bold> I </bold> shows normal SCN development in the susceptible X5. Panel <bold> J </bold> shows normal SCN development in susceptible NIL 34&#x02013;3. Panel <bold> K </bold> shows a <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgenic plant of cultivar X5 that was defoliated by insect herbivory in the field. Panel <bold> L </bold> shows an non-transgenic X5 plant with much less leaf area loss.
					</p>
				
					<p>&nbsp;&nbsp;
						The results from X5 transgenics were comaperd to results from Westag97 transgenics in the SDS assay with infested plants (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ). Westag97 plant were bigger <span class="trait">and</span> more stress resistant than X5. However, the Westag plants transgenic with the <span class="gene">RLK</span> again proved more significantly more resistant to SDS caused by <italic> F. virguliforme. </italic> Therefore, the <span class="trait">resistance</span> was not transgenic event or cultivar specific.
					</p>
				
					<p>&nbsp;&nbsp;
						In separate assays with plants from the same lines the <span class="trait">resistance</span> to SCN was partial in the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgenic plants (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> A) judged by female index (FI). Partial <span class="trait">resistance</span> was expected as <italic> rhg4 </italic> -e was present <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> -a absent. SCN FI was reduced by 30&#x02013;50% across four experiments using three Hg Types (P&#x02009;&#x0003c;&#x02009;0.01). Because the X5 selfed transgenic plants reported here had a susceptible allele at <italic> Rhg4, </italic> partial <span class="trait">resistance</span> was the expected outcome. The partial <span class="trait">resistance</span> was confirmed with another isolate of SCN at Harbin University (China). A cross was made to the RIL EF2 via a susceptible NIL WE1. EF2 was <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> . The progeny segregated for the <span class="gene">RLK</span> <span class="trait">and</span> so were <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4:: </italic> RLK18-1 or <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4 </italic> with no transgene. Plants with the <span class="gene">RLK</span> had a 20% lower FI <span class="trait">and</span> were almost in the resistant class (FI&#x02009;&#x0003c;&#x02009;10; Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ). Again the Westag 97 plants were tested with the same assay. Westag 97 appeared to encode a functional <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele. Consequently the <span class="gene">RLK</span> transgenic Westag 97 showed nearly complet <span class="trait">resistance.</span> Therefore, the <italic> <span class="gene">GmRLK18-1</span> </italic> -a was alone sufficient to provide for an <italic> <span class="gene">Rhg1</span> </italic> &#x02013;a like activity at different locations in different cultivars. The linked genes may reduce SCN numbers but did require the action of a second locus ( <italic> <span class="gene">Rhg4</span> </italic> ) to provide full <span class="trait">resistance</span> to Hg Type 0 as expected from [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In both SCN greenhouse assay <span class="trait">and</span> field the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a reduction in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ). The reduction was significant even though the X5 cultivar was innately smaller than the NILs due to a very much earlier maturity date. Reductions occurred in seedlings in the SCN assay (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> A), the SDS assay (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> B) <span class="trait">and</span> whole plants in the field (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> C) at harvest maturity. The reduction in growth in the field lead to a reduction in biomass but not seed <span class="trait">yield</span> <span class="trait">and</span> seed number per plant. Mid- to late-season defoliation by insect <span class="trait">pests</span> rarely reduces soybean <span class="trait">yield.</span> In both SCN <span class="trait">and</span> field assays plant growth was limited by available water. A reduction in root growth might be related to reduced water uptake.
					</p>
				
					<p>&nbsp;&nbsp;
						However, in the water saturated assays of SDS <span class="trait">resistance</span> the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused an increase in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ) in the presence of <italic> F. virguliforme </italic> infestations. The increase in growth lead to a increase in biomass <span class="trait">and</span> seed <span class="trait">yield</span> (Figure&#x02009; <xref ref-type="fig" rid="F5"> 5 </xref> ) <span class="trait">and</span> seed number per plant in plants grown to maturity in the greenhouse (1.7&#x02009;&#x000b1;&#x02009;0.3 compared to 5.7&#x02009;&#x000b1;&#x02009;1.1). Water sufficiency appeared to make the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> beneficial to growth <span class="trait">and</span> <span class="trait">yield</span> when <italic> F. virguliforme </italic> was present. In contrast, plants that were not infested in the SDS assays showed the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a decrease in root <span class="trait">and</span> shoot mass.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus was shown to underlie <span class="trait">resistance</span> to root infection by <italic> F. virguliforme </italic> <span class="trait">and</span> the subsequent leaf scorch, as predicted by [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Some <span class="trait">resistance</span> to SCN was found but this was partial as expected in the absence of a <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ]. The <span class="gene">RLK</span> simultaneously contributed <span class="trait">resistance</span> to both pathogens, establishing pleiotropy (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> ). Reduced seedling root growth was part of the <span class="trait">resistance</span> mechanism <span class="trait">and</span> this may underlie the reduced <span class="trait">yield</span> of resistant cultivars [ <xref ref-type="bibr" rid="B39"> 39 </xref> - <xref ref-type="bibr" rid="B41"> 41 </xref> ]. In Arabidopsis CLAVATA1, the <span class="gene">RLK</span> that regulated meristem development, also had an effect on nematode <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B52"> 52 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Genetic model for the function of the </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> &#x02009; </bold> <bold> locus in association with other known <span class="trait">resistance</span> loci. </bold> Black arrows show positive interactions, blue arrows show inhibitions. In this model four phenotypic events are controlled by the 3 genes at <italic> Rhg1/Rfs2 </italic> <span class="trait">and</span> four unlinked genes <italic> Rhg2-4 </italic> <span class="trait">and</span> <italic> Rzd1 </italic> ( <italic> sup-Rhg1/Rfs2 </italic> ). Root growth inhibition occurs despite suppression of the <span class="gene">RLK</span> by <italic> Rzd1 </italic> . Pleiotropy or close linkage between loci underlying <span class="trait">resistance</span> was found at <italic> Rhg1/Rfs2 </italic> (this work) <span class="trait">and</span> <italic> Rhg5/Rfs3 </italic> [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In field trials of plants transgenic with <italic> <span class="gene">GmRLK18-1</span> </italic> further evidence of pleiotropy was discovered when leaf herbivory was shown to be significantly worsened. Insect <span class="trait">pests</span> are separated into the chewing <span class="trait">and</span> sucking guilds. <span class="trait">Resistance</span> mechanisms to the guilds were known to operate by different pathways but some involved the <italic> <span class="gene">Rhg4</span> </italic> locus or region [ <xref ref-type="bibr" rid="B62"> 62 </xref> ]. Arabidopsis <span class="trait">resistance</span> to <italic> F. graminearum </italic> was recently shown to require operation of both SA <span class="trait">and</span> JA pathways [ <xref ref-type="bibr" rid="B63"> 63 </xref> ]. The same Nils used here when infested with H. glycines decreased proteins involved in salicylic acid responses <span class="trait">and</span> increased those in the jasmonic acid signaling pathways [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. Therefore, plants transgenic with <span class="gene">Gm18RLK18-1</span> might be altered in their responses to a broad range of <span class="trait">pests</span> <span class="trait">and</span> pathogens. Additional pathogens will be tested in future studies.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Rhg1/Rfs2 </italic> locus was shown to center on three genes where recombination was suppressed within a wider region that was high in recombination. Perhaps the failure to recombine within the central 3 gene cluster caused the unusually high frequency of heterozygous plants in the neighboring region. A potential suppressor locus acting on the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> was identified earlier [ <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]. The allele of the gene on chromosome 7 (Lg M) had to be inherited from the resistant parent to prevent zygote or embryo lethality in SCN resistant plants. Consequently, all resistant plants co-inherited <italic> Rhg1/Rfs2 </italic> on chromosome 18 (Lg G) <span class="trait">and</span> the <span class="trait">resistance</span> allele on chromosome 7 (Lg M). That gene may have had a functional homeolog in this study. However, the locus was fixed to the R haplotype in all the NILs, so fine mapping was not possible. Plant transgenic with the <span class="gene">RLK</span> were fertile <span class="trait">and</span> produced seed <span class="trait">and</span> they would lack the <span class="trait">resistance</span> allele of the gene on chromosome 7 (Lg M). Therefore, <italic> <span class="gene">GmRLK18-1</span> </italic> does not cause zygote or embryo lethality. Additional genetic elements must be involved in that phenomenon.
					</p>
				
					<p>&nbsp;&nbsp;
						An absence of recombination events in a region can be caused by; deletions; insertions; inversions; the condensed heterochromatin found near centromeres; <span class="trait">and</span> recombinant allele lethality. The first four phenomena were not occurring at the <italic> Rhg1/Rfs2 </italic> locus since DNA sequences in resistant <span class="trait">and</span> susceptible cultivars were co-linear over at least 87&#x02009;kbp. Further, the <italic> Rhg1/Rfs2 </italic> locus mapped into a region within 1&#x02009;Mbp of the predicted telomere region of chromosome 18. Therefore, recombinant allele lethality may be the cause <span class="trait">and</span> a hypothetical model for its action was developed. The kinase domain of the <span class="gene">RLK</span> at <italic> Rhg1/Rfs2 </italic> was proposed as the element causing lethality to gametes or the zygote carrying a recombination event. The hypothetical anti-porter protein was proposed as the target locus kept in the resistant state when the <span class="gene">RLK</span> is conferring SCN <span class="trait">resistance</span> (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> ). The laccase that was located between these two genes, along with the intergenic regions, are held in phase by the locus. It may be possible that the lethal nature of the three genes linked in association at the <span class="trait">resistance</span> locus are not fully suppressed by the modifier locus (they are leaky) <span class="trait">and</span> results in the inhibition of root growth observed in SCN resistant cultivars (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The linkage disequilibrium studies reported with probes in, <span class="trait">and</span> around, the <span class="gene">RLK</span> inferred a larger more complex structure for <italic> <span class="gene">Rhg1</span> </italic> than simply the <span class="gene">RLK</span> polymorphisms [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B64"> 64 </xref> ]. Possible roles included contributions to additive <span class="trait">resistance;</span> contributions to <span class="trait">resistance</span> in other <span class="trait">resistance</span> types (eg PI88788 <span class="trait">and</span> Toyosuzu; R types 2 <span class="trait">and</span> 3) or contributions to the <span class="trait">resistance</span> to other Hg types. The hypothesis that the linked genes were factors necessary in susceptible genotypes for SCN parasitism was unlikely [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B53"> 53 </xref> ]. The dominance of <italic> Rhg1/Rfs2 </italic> to SCN in NIL segregation also suggested the genes were active in resistant types <span class="trait">and</span> inactive in susceptible genotypes.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Rhg1/Rfs2 </italic> locus was shown to include the <italic> <span class="gene">GmRLK18-1</span> </italic> . Previously [ <xref ref-type="bibr" rid="B53"> 53 </xref> ] also reported inhibition of the <span class="trait">resistance</span> allele increased FI in composite roots, but not to a significant degree in small experiments with few replications. Here <italic> Rhg1/Rfs2 </italic> was inferred to be a complex of three genes assembled <span class="trait">and</span> co-inherited over long periods of selection for <span class="trait">resistance</span> to a endemic pest, root parasitic nematodes. However, the <italic> <span class="gene">GmRLK18-1</span> </italic> alone at a new genomic locations in transgenic plants was sufficient for <span class="trait">resistance</span> to a relatively new pathogen of soybean <italic> F. virguliforme </italic> . Revisions to the sequence of the <span class="gene">RLK</span> made by re-sequencing PCR products [ <xref ref-type="bibr" rid="B25"> 25 </xref> ] were confirmed by BAC fragment sequencing here. Fortunately the amino acid changes made to [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] lay outside of regions under detailed analysis for ligand <span class="trait">and</span> antibody binding [ <xref ref-type="bibr" rid="B65"> 65 </xref> , <xref ref-type="bibr" rid="B66"> 66 </xref> ]. Previously CLE peptide in nematode secretions were shown to be perceived by RLKs in the CLAVATA1, <italic> CLAVATA2 </italic> <span class="trait">and</span> <italic> CORYNE </italic> families [ <xref ref-type="bibr" rid="B52"> 52 </xref> ]. In vitro, the purified <italic> <span class="gene">GmRLK18-1</span> </italic> LRR domain has been shown to bind strongly to the CLE peptide found in nematode secretions as well as two proteins <span class="trait">and</span> two CLE peptide of plant origin involved in tracheary element inhibition [ <xref ref-type="bibr" rid="B43"> 43 </xref> , <xref ref-type="bibr" rid="B66"> 66 </xref> ]. It will be of interest to discover whether the ligand profiles are similar for the two resistances. The biochemical analysis of the <span class="gene">RLK</span> protein will lead to refinements of models for modes of action for the pleiotropic <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The allelic discrimination probes developed provide a high throughput alternative to satellite markers for marker assisted selection. These tools will facilitate molecular breeding for <span class="trait">resistance</span> to two or more important <span class="trait">pests</span> <span class="trait">and</span> <span class="trait">diseases</span> of soybean [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. The discovery of a syntenic set of paralogs to the RLK, laccase <span class="trait">and</span> transporter gene cluster at <italic> Rhg1/Rfs2 </italic> may also assist marker assisted breeding. The discovery of the syntenic homeologs raised significant barriers to reverse genetic approaches to analyses of the <span class="gene">RLK</span> at <italic> Rhg1/Rfs2 </italic> . Another barrier to reverse genetic approaches was the role of <italic> Rhg1/Rfs2 </italic> in normal plant development that can be inferred from the restricted root growth of NILs (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ) <span class="trait">and</span> transgenic plants (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ). A third barrier to reverse genetics was the uncoupling in eukaryotes of transcript <span class="trait">and</span> protein abundances. Both effects may have contributed to the recent report of no significant difference between the SCN counts on composite plant roots with the <span class="gene">GmRLK18-1-b</span> alone targeted for inhibition [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. It must be noted that the <span class="gene">GmRLK18-1-b</span> protein abundance was not measured in that study in spite of the tools available to do so [ <xref ref-type="bibr" rid="B43"> 43 </xref> , <xref ref-type="bibr" rid="B65"> 65 </xref> , <xref ref-type="bibr" rid="B66"> 66 </xref> ]. Further, note that the numbers of cyst did differ between treatments [ <xref ref-type="bibr" rid="B53"> 53 </xref> ] <span class="trait">and</span> that might be significant in larger, replicated studies or under different conditions. Further note that allele was recessive. However, further proofs of the nature of <italic> Rhg1/Rfs2 </italic> locus function may require knock-outs of each of the paralogs, or stable transformation to several new locations, followed by measurements of genetic segregation. In each case, the analysis will be complicated by the co-dominant nature of the <span class="trait">resistance</span> gene in certain experiments. In fact, the possibility that the susceptible Essex allele of <italic> Rhg1/Rfs2 </italic> was functional by promoting root growth <span class="trait">and</span> susceptibility, <span class="trait">and</span> so actively promoted the establishment of parasitism by SCN <span class="trait">and</span> <italic> F. virguliforme </italic> , should be explored with transgenic plants. The model inferring that soybean has adapted part of an existing pathogen recognition <span class="trait">and</span> defense cascade (SCN parasitism <span class="trait">and</span> insect herbivory), to a new pathogen ( <italic> F. virguliforme </italic> ) disease (root rot <span class="trait">and</span> leaf scorch <span class="trait">and</span> syndrome (SDS) has broad implications for crop improvement. Stable <span class="trait">resistance</span> to many pathogens might be achieved by manipulation of the genes encoding a small number of pathogen recognition proteins in the <span class="gene">RLK</span> family.
					</p>
				
					<p>&nbsp;&nbsp;
						Many of the NILs were described previously [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ]. All lines were released <span class="trait">and</span> consequently are available on request as seed. Seeds of NIL 34&#x02013;23 (resistant haplotype between markers Satt 214 to Satt 570) <span class="trait">and</span> NIL 34&#x02013;3 (susceptible haplotype from the marker Satt 214 to the Sat122-Satt 570 interval) were obtained at the F5:13 generation. Genotypes were <italic> rhg1-e/rfs2, rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;3 <span class="trait">and</span> <italic> Rhg1-a/Rfs2, Rhg1-a/Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;23 whereas NIL34-33 contained both those <span class="trait">and</span> <italic> rhg1 </italic> -e <italic> /rfs2, Rhg1- </italic> a <italic> /Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> in different plants (NIL34-33-1 to &#x02212;34).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cv. X5 <span class="trait">and</span> Westag 97 were used for transformation because they could be regenerated to plant efficiently from embryo cultures [ <xref ref-type="bibr" rid="B67"> 67 </xref> ]. They were susceptible to both SCN <span class="trait">and</span> SDS. X5 was judged to be <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, rhg4, rhg4 </italic> <span class="trait">and</span> Westag 97 was judged <italic> rhg1-e/rfs2rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> <span class="trait">and</span> based on the assays reported here. Crosses were made to RIL EF2 for SCN tests which was <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, Rhg4. </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						DNA was isolated following [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Concentrations of DNAs were calculated by measuring absorbances at 260 <span class="trait">and</span> 280&#x02009;nm. Total RNA was isolated with Trizol <sup> TM </sup> (Invitrogen, Carlsbad, CA, USA), according to the manufacturer&#x02019;s instructions. First strand cDNA was synthesis carried out using oligo dT primers using a cDNA synthesis kit, according to manufacturer (Invitrogen). Presence of the <italic> Rhg1/Rfs2 </italic> <span class="trait">resistance</span> alleles was confirmed by PCR analysis using TMD1 an indel marker in the <span class="gene">RLK</span> intron. Several designs of TMD1 primers have been reported [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> ]. Used here were the primers pair; forward 5&#x02032;- CAC CTG CAT CAA GAT GAA CA -3&#x02032; <span class="trait">and</span> reverse 5&#x02032;- GCC TAT TAC TTG GGA CCC AA -3&#x02032; (Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2). Genotyping by markers linked to <italic> Rhg1/Rfs2 </italic> used about 50&#x02009;ng of DNA for microsatellite analysis on PAGE after [ <xref ref-type="bibr" rid="B39"> 39 </xref> ] <span class="trait">and</span> on agarose gels after [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP genotyping assay within the gene encoding the <span class="gene">RLK</span> was performed using a custom Taqman <sup> TM </sup> Kit. Three probes were designed for the SNPs at 1,486&#x02009;bp, 506&#x02009;bp <span class="trait">and</span> 2,040&#x02009;bp (relative to the translation start site) to distinguish the 8 commonest alleles of the <span class="gene">RLK</span> (Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2). Six process were designed to the non-synonymous SNPs in the <span class="gene">GmRLK18-1</span> preotein to distinguish the seven allotypes. The PCR reactions were carried out using a 3 step PCR protocol with one hold at 95&#x000b0;C for 10 minutes followed by 35 cycles that included a denaturation cycle of 95&#x000b0;C for 30&#x02009;sec, annealing at 58&#x000b0;C for 10 seconds <span class="trait">and</span> an extension at 68&#x000b0;C for 20&#x02009;sec.
					</p>
				
					<p>&nbsp;&nbsp;
						Protein from root material was isolated from infested <span class="trait">and</span> non-infested roots was extracted after [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. Total protein concentration was determined using a non-interfering protein assay. For the Western hybridizations, a custom made antibody generated against a peptide [C]TL SRL KTL DIS NNA LNG <span class="gene">NLP</span> ATL SNL S from the LRR domain of <span class="gene">RLK</span> at RHG1/RFS2 was used (Alpha diagnostics, San Antonio, Texas).
					</p>
				
					<p>&nbsp;&nbsp;
						The genotype of X5 was <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, rhg4rhg4. </italic> Purified stable transgenics were of genotype <italic> rhg1 </italic> -e <italic> /rfs2rhg1 </italic> -e <italic> /rfs2, rhg4rhg4:: </italic> <span class="gene">GmRLK18-1,</span> <span class="gene">GmRLK18-1</span> (so <italic> Rhg1- </italic> a <italic> /Rfs2Rhg1- </italic> a <italic> /Rfs2 </italic> ) as shown by markers TMD1 <span class="trait">and</span> A2D8 <italic> . </italic> Expression of the transgene was established by qRT-PCR from cDNA with allele specific Taqman probes <span class="trait">and</span> HRM of amplicons. Protein allotypes were identified by two dimensional PAGE [ <xref ref-type="bibr" rid="B46"> 46 </xref> ] followed by Western hybridization [ <xref ref-type="bibr" rid="B43"> 43 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						TMD1 marker, FJ520231; Corrected <span class="gene">RLK</span> at Rhg1/Rfs2 gene AF506516 <span class="trait">and</span> mRNA AF506517; SIUC-Satt122, bankit1155667; BAC pB73P06 complete sequence JN597009; pSBHB94 complete sequence HQ008939; BAC pH38f23 complete sequence TBD.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Figure S1. Paralogs of </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> in the soybean genome. </bold> Panel (A) shows LRR probe (200bp) hybridized to Forrest MTP. (B) Southern hybridization of LRR probe (200bp) to the MTP positives. Five out of the 7 MTP clones hybridized after BAC clone purification <span class="trait">and</span> restriction digestion with HindIII. The lower panel (C) shows the kinase domain probe (200bp) hybridized to Forrest MTP. Panel (D) shows Southern hybridization of the same kinase probe to the MTP positives. Three out of 5 MTP clones hybridized after BAC clone purification <span class="trait">and</span> restriction digestion with HindIII. Panel F shows ideograms of the genes predicted from the genome sequences centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> Panel F shows an alignment of the genome sequences of 70kbp centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> showing the extent of synteny.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Table S1. Comparisons of sequence identity between Forrest alleles of <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> the most similar <span class="trait">and</span> syntenic <span class="gene">RLK</span> like protein. </bold> The amino acid identity was 78% in the signal peptide (residues 1&#x02013;61); 94% in the ten LRRs (141&#x02013;471), 93% in the transmembrane domain (485&#x02013;507) <span class="trait">and</span> 97% in the kinase domain (569&#x02013;840). Residues that differ in alloproteins of <span class="gene">GmRLK18-1</span> are in bold. Four of the six are identical in the homeoprotein the other 2 are identical to the susceptible allele. The neighboring laccase <span class="trait">and</span> the antiporter also showed 85&#x02013;96% amino acid identity.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Figure S2. Detection of the SNP polymorphism at position 1486 in the LRR region of </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> &#x02013;a <span class="trait">and</span> &#x02013;e using an allelic discriminatory assay. </bold> A Famlabeled probe was used for the detection of resistant haplotypes1 <span class="trait">and</span> 2 (red) <span class="trait">and</span> Hex labeled probe for the detection of susceptible haplotypes2, 3 <span class="trait">and</span> 4 (blue). A total of 16 individuals from the 110 PIs were selected for the analysis. The Panel shows relative fluorescent signal intensity for each of the 16 plant introductions. The two groups form separate clusters.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A <italic> Phytophthora sojae </italic> effector suppresses endoplasmic reticulum stress-mediated immunity by stabilizing plant Binding immunoglobulin Proteins</h2>
			<h3>PMID:27256489</h3>
				
					<h3>PMCID:4895818</h3>
					<h4>期刊:Nature Communications</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jing </surname>  <given-names> Maofeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Baodian </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Haonan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Guanghui </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Huawei </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Wenwu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-7347-8935 </contrib-id>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Suomeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiao </surname>  <given-names> Yongli </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tyler </surname>  <given-names> Brett M. </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Wenbo </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  <xref ref-type="aff" rid="a6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yuanchao </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-5803-5343 </contrib-id>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Plant Pathology, Nanjing Agricultural University </institution> , 210095 Nanjing,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Key Laboratory of Integrated Management of Crop Diseases and Pests (Ministry of Education) </institution> , 210095 Nanjing,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Institute of Crop Science, Chinese Academy of Agricultural Sciences </institution> , 100081 Beijing,  <country> China </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> Center for Genome Research and Biocomputing and Department of Botany and Plant Pathology, Oregon State University </institution> , Corvallis, Oregon 97331,  <country> USA </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Department of Plant Pathology and Microbiology, University of California </institution> , Riverside, California 92521,  <country> USA </country>  </aff>  <aff id="a6">  <label> 6 </label>  <institution> Center for Plant Cell Biology, University of California </institution> , Riverside, California 92521,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 04 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Emerging evidence indicates that ER stress-related cell death is associated with microbe infection. For example, the endophytic fungus <italic> Piriformospora indica </italic> activates ER stress-mediated cell death by inhibiting the UPR-related pro-survival machinery <xref ref-type="bibr" rid="b23"> 23 </xref> . Thus ER-stress pathways could be targeted by pathogens to facilitate infection <xref ref-type="bibr" rid="b24"> 24 </xref> . BiPs can regulate plant responses to abiotic <span class="trait">and</span> biotic stresses. For example, overexpression of <italic> <span class="gene">GmBiP4</span> </italic> in soybean ( <italic> Glycine max </italic> ) inhibited the UPR <span class="trait">and</span> increased tolerance to water loss <xref ref-type="bibr" rid="b25"> 25 </xref> , <span class="trait">and</span> suppressed cell death associated with senescence <span class="trait">and</span> osmotic stress <xref ref-type="bibr" rid="b26"> 26 </xref> , but at the same time intensified cell death <span class="trait">and</span> induction of SA-mediated responses triggered by avirulent <italic> Pseudomonas </italic> species <xref ref-type="bibr" rid="b26"> 26 </xref> . AtBiP2 of <italic> Arabidopsis thaliana </italic> is involved in folding <span class="trait">and</span> secretion of pathogenesis-related (PR) proteins during systemic acquired <span class="trait">resistance</span> (SAR), as loss-of-function mutants of <italic> AtBiP2 </italic> are defective in salicylic acid-elicited PR1 protein secretion <xref ref-type="bibr" rid="b27"> 27 </xref> . Overexpression of a <italic> Nicotiana benthamiana BiP </italic> gene eliminated the triple gene block protein 3 (TGBp3)-induced hypersensitive response (HR), which is consistent with the cyto-protective role of the NbBiP in virus-infected leaves of <italic> N. benthamiana </italic> <xref ref-type="bibr" rid="b28"> 28 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To identify the host targets of PsAvh262, N-terminal GFP-tagged PsAvh262 (without the N-terminal secretion signal peptide 1&#x02013;18 aa) was transiently expressed in <italic> N. benthamiana </italic> leaves followed by immunoprecipitation (IP) using anti-GFP affinity beads. Immuno-purified proteins were then analysed by LC-MS/MS using the genome sequences of the closely related <italic> Nicotiana tabacum </italic> <xref ref-type="bibr" rid="b13"> 13 </xref> . Among the <italic> N. benthamiana </italic> proteins that potentially associated with PsAvh262 ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> ), a protein similar to an ER-luminal BiP5 in <italic> N. tabacum </italic> (GenBank accession &#x00023;729623) was selected for further analysis. Four BiPs, that is, NbBiP1, NbBiP2, NbBiP3 <span class="trait">and</span> NbBiP4, have been reported in <italic> N. benthamiana </italic> <xref ref-type="bibr" rid="b31"> 31 </xref> . A search of the Sol Genomics Network ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://solgenomics.net/"> http://solgenomics.net/ </ext-link> ) using the NtBiP5 sequence as query identified an analogous protein (ID: NbS00040865g0006.1) in the <italic> N. benthamiana </italic> database with only one amino acid difference (a &#x02018;V' versus an &#x02018;I' at the position 13, 99.85&#x00025; identity) ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 3 </xref> ). We therefore designated the PsAvh262-associated <italic> N. benthamiana </italic> protein as NbBiP5, which shares 76&#x02013;96&#x00025; identity in full-length amino-acid sequences with NbBiP1&#x02013;NbBiP4, respectively ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 3 </xref> ). NbBiP5 also has homologues in soybean. The soybean <italic> BiP </italic> gene family consists of at least four members, <italic> <span class="gene">GmBiP1</span> </italic> ( <italic> <span class="gene">Glyma08g02940</span> </italic> ), <italic> <span class="gene">GmBiP2</span> </italic> ( <italic> <span class="gene">Glyma08g02960</span> </italic> ), <italic> <span class="gene">GmBiP3</span> </italic> ( <italic> <span class="gene">Glyma05g36600</span> </italic> ) <span class="trait">and</span> <italic> <span class="gene">GmBiP4</span> </italic> ( <italic> <span class="gene">Glyma05g36620</span> </italic> ), all of which are induced upon ER stress <xref ref-type="bibr" rid="b32"> 32 </xref> . NbBiP5 shares 93.41, 91.77, 92.66 <span class="trait">and</span> 93.87&#x00025; identity with <span class="gene">GmBiP1&#x02013;GmBiP4,</span> respectively ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 4 </xref> ). Transcriptome analysis showed that <italic> <span class="gene">GmBiP1</span> </italic> has the highest transcription level, followed by <italic> <span class="gene">GmBiP4</span> </italic> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 5 </xref> ). These two <span class="gene">GmBiPs</span> were utilized further in our study.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the association of PsAvh262 with NbBiP5 <span class="trait">and</span> <span class="gene">GmBiPs,</span> we carried out Co-IP assays ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 6 </xref> ). GFP-PsAvh262 was co-expressed with NbBiP5-RFP or RFP in <italic> N. benthamiana </italic> . Total proteins were extracted from the infiltrated leaves <span class="trait">and</span> incubated with GFP-Trap_A beads (Chromotek, Germany). NbBiP5-RFP, but not RFP, was significantly enriched in the GFP-PsAvh262 precipitates ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 6a </xref> ). Consistent with this result, GFP-GmBiP1, GFP-GmBiP2, GFP-GmBiP3 <span class="trait">and</span> GFP-GmBiP4, but not GFP, were also enriched in the 3 &#x000d7; Flag-PsAvh262 precipitates ( <xref ref-type="fig" rid="f3"> Fig. 3a </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 6b </xref> ) following transient expression in <italic> N. benthamiana </italic> . We next confirmed this interaction by a semi- <italic> in vivo </italic> GST-pull-down assay because we were unable to express <italic> NbBiP5 </italic> or <italic> <span class="gene">GmBiP</span> </italic> s in <italic> Escherichia coli </italic> ( <italic> E. coli </italic> ). <span class="gene">GmBiP1-RFP</span> was expressed in <italic> N. benthamiana </italic> leaves <span class="trait">and</span> the total protein extracts were incubated with purified glutathione <italic> S </italic> -transferase (GST)-PsAvh262 expressed in <italic> E. coli </italic> . As showed in <xref ref-type="fig" rid="f3"> Fig. 3b </xref> , <span class="gene">GmBiP1-RFP,</span> but not RFP, was specifically enriched in GST-PsAvh262-bound glutathione beads. PsAvh262 also interacted with OsBiP3 from rice (Os02g02410) ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 6c </xref> ). The interaction between PsAvh262-M1 <span class="trait">and</span> <span class="gene">GmBiP1</span> was much weaker than the interaction with WT PsAvh262 ( <xref ref-type="fig" rid="f3"> Fig. 3c </xref> ), indicating that this region plays a key role in mediating the interaction between PsAvh262 <span class="trait">and</span> BiPs.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain insights into the functional significance of the PsAvh262-GmBiP1 interaction, we determined the localization patterns of the proteins in plant cells. Wild-type or mutant PsAvh262 proteins (lacking the signal peptide) were co-expressed with <span class="gene">GmBiP1</span> as a variety of GFP or RFP fusion proteins by agroinfiltration in <italic> N. benthamiana </italic> . Confocal microscopy showed that RFP-PsAvh262, GFP-PsAvh262 <span class="trait">and</span> their mutants (the FPs were attached to the N terminus of PsAvh262) were primarily localized in the nucleus when expressed in the absence of co-expressed BiPs ( <xref ref-type="fig" rid="f4"> Fig. 4a </xref> ). RFP-PsAvh262 <span class="trait">and</span> GFP-PsAvh262 also were present in punctate structures in the cell cortex <xref ref-type="bibr" rid="b33"> 33 </xref> ( <xref ref-type="fig" rid="f4"> Fig. 4a </xref> ). PsAvh262-M2 <span class="trait">and</span> PsAvh262-M3, but not PsAvh262-M1, also were present in punctate structures ( <xref ref-type="fig" rid="f4"> Fig. 4a </xref> ), indicating that the region deleted in M1 is required for the localization of PsAvh262 into the punctate structures. Consistent with the previously observed localization of BiPs to the ER <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b34"> 34 </xref> <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b36"> 36 </xref> , we found that SP-GmBiP1-GFP-HDEL (that is, GFP inserted between BiP <span class="trait">and</span> its HDEL motif) localized in an ER-like network of subcellular structures including the perinuclear ER ( <xref ref-type="fig" rid="f4"> Fig. 4b,c </xref> ). Surprisingly, the localization of BiP was unchanged if the FP tag was placed onto the N terminus of the signal peptide or onto the C terminus of the HDEL motif ( <xref ref-type="fig" rid="f4"> Fig. 4b </xref> ) indicating that the localization of <span class="gene">GmBiP1</span> is primarily controlled by sequences other than these two signals. Co-expression of GFP-SP-GmBiP1-HDEL with an ER marker (SP-RFP-HDEL) confirmed that the localization was to the ER ( <xref ref-type="fig" rid="f4"> Fig. 4c </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Notably, when co-expressed with GFP-SP-GmBiP1-HDEL in <italic> N. benthamiana </italic> , RFP-PsAvh262 was co-localized with GFP-SP-GmBiP1-HDEL to the ER network ( <xref ref-type="fig" rid="f4"> Fig. 4d </xref> ) including the perinuclear ER ( <xref ref-type="fig" rid="f4"> Fig. 4e </xref> ). This re-localization in the presence of <span class="gene">GmBiP1</span> was not observed for RFP-PsAvh262-M1 ( <xref ref-type="fig" rid="f4"> Fig. 4e </xref> ) or RFP ( <xref ref-type="fig" rid="f4"> Fig. 4c </xref> ). To further confirm the location of the <span class="gene">GmBiP1-PsAvh262</span> interaction in plant cells, we conducted a bimolecular fluorescence complementation (BiFC) experiment. PsAvh262 (PsAvh262-YFPN, with or without its signal peptide) <span class="trait">and</span> <span class="gene">GmBiP1</span> (SP-YFPC-GmBiP1-HDEL) were co-expressed in <italic> N. benthamiana </italic> . YFP fluorescence was observed exclusively from the ER ( <xref ref-type="fig" rid="f4"> Fig. 4f </xref> ), suggesting that PsAvh262 could reach <span class="trait">and</span> bind to BiP located inside the ER whether PsAvh262 was targeted to the cytoplasm or whether it was targeted to the ER by a signal peptide. No fluorescence complementation was observed when BiP was removed from the YFPC constructs ( <xref ref-type="fig" rid="f4"> Fig. 4g </xref> ). Taken together, our results suggest that the <span class="gene">GmBiP1-PsAvh262</span> interaction occurs in the ER, though we cannot rule out that the interaction (also) occurs in the cytoplasm <span class="trait">and</span> that the complex subsequent can relocate to the ER.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the localization patterns of <span class="gene">GmBiP1</span> <span class="trait">and</span> PsAvh262 during <italic> Phytophthora </italic> infection, we inoculated <italic> N. benthamiana </italic> leaves expressing GFP-SP-GmBiP1-HDEL with <italic> P. infestans </italic> strain 88069td tagged with cytoplasmic tdTomato. Confocal microscopy showed preferential accumulation of GFP-SP-GmBiP1-HDEL around haustoria in the infected leaves ( <xref ref-type="fig" rid="f5"> Fig. 5a </xref> ), suggesting that relocation of BiPs to the peri-haustorial region is associated with plant responses to infection. To determine PsAvh262 <span class="trait">and</span> BiPs were co-localized during infection, GFP-PsAvh262 (lacking its signal peptide) <span class="trait">and</span> SP-GmBiP1-HDEL-RFP were co-expressed in <italic> N. benthamiana </italic> leaves infected by untagged <italic> P. infestans </italic> strain T30-4. The results showed that both PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> localized to the perihaustorial region, though the distribution of <span class="gene">GmBiP1</span> appeared biased towards the necks of the haustoria, while PsAvh262 appeared biased towards the tips ( <xref ref-type="fig" rid="f5"> Fig. 5b </xref> ). To determine if PsAvh262 was localized to the extra-haustorial membrane (EHM), we co-expressed GFP-PsAvh262 with the <italic> P. infestans </italic> RxLR effector Avrblb2 (RFP-Avrblb2), since Avrblb2 was reported to associate with the extra-haustorial membrane <xref ref-type="bibr" rid="b13"> 13 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> . As shown in <xref ref-type="fig" rid="f5"> Fig. 5c </xref> , GFP-PsAvh262 localized around the haustoria coincident with Avrblb2, demonstrating that PsAvh262 was associated with the EHM. Interestingly, in contrast to <xref ref-type="fig" rid="f4"> Fig. 4a </xref> , PsAvh262 localized to the EHM in the absence of co-expression with <span class="gene">GmBiP1</span> ( <xref ref-type="fig" rid="f5"> Fig. 5c </xref> ). Taken together, these results suggest that PsAvh262 <span class="trait">and</span> its target <span class="gene">GmBiP1</span> co-localize around the haustoria <span class="trait">and</span> so may possibly influence infection.
					</p>
				
					<p>&nbsp;&nbsp;
						We next investigated whether the abundance of <span class="gene">GmBiP1</span> could be affected by PsAvh262. For this purpose, we used western blots with anti-BiP antibodies to determine the endogenous levels of <span class="gene">GmBiPs</span> during <italic> P. sojae </italic> infection of soybean. The hypocotyls of etiolated soybean seedlings were inoculated with wild-type strain P6497 or with two independent <italic> PsAvh262 </italic> -silenced transformants (Avh262-S62 <span class="trait">and</span> Avh262-S76). Total proteins were extracted from the infected tissues at 0, 1.5, 3, 6 <span class="trait">and</span> 9&#x02009;h.p.i. We used 9&#x02009;h.p.i. as our latest time point because <italic> PsAvh262 </italic> -silenced transformants caused cell death at this time point. Abundances of <span class="gene">GmBiPs</span> increased from 3 to 9&#x02009;h.p.i. in tissues inoculated with wild-type <italic> P. sojae </italic> , but much less in samples inoculated with the <italic> PsAvh262 </italic> -silenced transformants ( <xref ref-type="fig" rid="f6"> Fig. 6a </xref> ). In addition, we found a small increase in <span class="gene">GmBiP</span> levels at 9&#x02009;h.p.i. during infection by the <italic> PsAvh262 </italic> -silenced transformants. Transcriptome analysis did not show significant differential accumulation of <italic> <span class="gene">GmBiP</span> </italic> transcripts during <italic> P. sojae </italic> infection ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 5 </xref> ). qRT&#x02013;PCR results confirmed that the transcript levels of <italic> <span class="gene">GmBiP1</span> </italic> remained almost the same in soybean inoculated by either wild type or <italic> PsAvh262 </italic> -silenced <italic> P. sojae </italic> , although there were slight fluctuations at some time points ( <xref ref-type="fig" rid="f6"> Fig. 6b </xref> ). These results suggest that PsAvh262 might have increased the stability of <span class="gene">GmBiPs</span> proteins during <italic> P. sojae </italic> infection, though we cannot rule out that stabilization of the <span class="gene">GmBiPs</span> is caused indirectly by the greater level infection by the wild-type strain, which results in a higher level of ER stress (see below).
					</p>
				
					<p>&nbsp;&nbsp;
						To more directly examine the effect of PsAvh262 on BiP stability, we compared the accumulation of NbBiP5-RFP in <italic> N. benthamiana </italic> when co-expressed with GFP-PsAvh262 or GFP using Agroinfiltration. As shown in <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 8a </xref> , NbBiP5-RFP accumulated to a much higher level in the presence of GFP-PsAvh262 than when co-expressed with GFP. These results provide evidence that the presence of PsAvh262 can enhance the accumulation of NbBiP5. Plant ER-resident proteins can be efficiently transported out of the ER for degradation <xref ref-type="bibr" rid="b38"> 38 </xref> via the ubiquitin/26S proteasome pathway <xref ref-type="bibr" rid="b39"> 39 </xref> . BiPs may also be regulated similarly by this mechanism. It is therefore possible that the interaction with PsAvh262 increases the retention of BiPs in the ER, thereby preventing their degradation. To test this hypothesis, we co-expressed NbBiP5 <span class="trait">and</span> <span class="gene">GmBiP1</span> with wild-type PsAvh262 or the PsAvh262 mutants in <italic> N. benthamiana </italic> , <span class="trait">and</span> treated BiP-expressing leaves with the proteasomal inhibitor MG132. Increased accumulations of <span class="gene">GmBiP1</span> ( <xref ref-type="fig" rid="f7"> Fig. 7a </xref> ) <span class="trait">and</span> NbBiP5 ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 8a </xref> ) were observed in leaves treated with MG132 or co-expressing PsAvh262. Furthermore, MG132 treatment did not further increase the levels of <span class="gene">GmBiP1</span> produced by Avh262 expression, suggesting that Avh262 <span class="trait">and</span> MG132 act via the same process ( <xref ref-type="fig" rid="f7"> Fig. 7a </xref> ). These results suggest that PsAvh262 prevents BiPs from being degraded <italic> in planta </italic> through the 26S proteasomal pathway. Interestingly, we found that PsAvh262-M1, but not PsAvh262-M2, lost the capacity to stabilize <span class="gene">GmBiP1</span> ( <xref ref-type="fig" rid="f7"> Fig. 7a </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 8b </xref> ), indicating that direct PsAvh262-BiP interaction is required for BiP stabilization. PsAvh262-mediated inhibition of BiP degradation is specific because the <italic> P. infestans </italic> effector Avrblb2, which also associates with NbBiP5 (ref. <xref ref-type="bibr" rid="b13"> 13 </xref> ), was unable to stabilize NbBiP5 ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 8b </xref> ). Taken together, these results suggest that PsAvh262 stabilizes BiPs by preventing their degradation.
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous study, overexpression of <span class="gene">GmBiP4</span> in soybean attenuated N-rich protein (NRP)-mediated cell death signalling by inhibiting the induction of several components in the pathway, such as homologues of the vacuolar processing enzyme (VPE) <xref ref-type="bibr" rid="b40"> 40 </xref> . Protein disulfide isomerase (PDI), an ER-resident chaperone involved in mediating ERQC <xref ref-type="bibr" rid="b40"> 40 </xref> , was shown to be induced at both the RNA <span class="trait">and</span> protein levels by ER stress <xref ref-type="bibr" rid="b41"> 41 </xref> . Thus PDI can serve as a marker of ER stress. We therefore examined the transcript levels of <italic> PDI </italic> <span class="trait">and</span> <italic> VPE </italic> during <italic> P. sojae </italic> infection to monitor the induction of ER stress <span class="trait">and</span> the induction of ER stress-triggered cell death, respectively. qRT&#x02013;PCR results showed that <italic> PDI </italic> transcript levels were upregulated nearly two-fold ( <italic> P </italic> &#x0003c;0.01, one-way ANOVA, <italic> n </italic> =3) within 1.5&#x02009;h after infection by wild-type <italic> P. sojae </italic> ( <xref ref-type="fig" rid="f6"> Fig. 6c </xref> ), whereas during infection by the <italic> PsAvh262 </italic> -silenced transformants, minimal changes were observed in the levels of <italic> PDI </italic> transcripts prior to 9&#x02009;h.p.i. PDI induction was observed at 9&#x02009;h.p.i., but the levels were slightly lower than in tissues infected with the wild-type strain ( <xref ref-type="fig" rid="f6"> Fig. 6c </xref> ). These results suggest that the higher virulence of the wild-type strain results in earlier induction of ER stress than in the less virulent-silenced lines. The levels of <italic> VPE </italic> transcripts were not significantly changed during wild-type infection, however <italic> VPE </italic> levels were significantly upregulated at 9&#x02009;h.p.i. during infection by the <italic> PsAvh262 </italic> -silenced transformants ( <xref ref-type="fig" rid="f6"> Fig. 6d </xref> ), at which time cell death symptoms were observed. These results suggest that the <italic> PsAvh262 </italic> -silenced transformants were no longer able to attenuate cell death resulting from the ER stress triggered by infection, even though the level of ER stress imposed by the silenced lines was less than that imposed by the wild type.
					</p>
				
					<p>&nbsp;&nbsp;
						The influence of PsAvh262 on BiP levels during infection suggests that BiPs play a role in plant immunity. To test this, we transiently expressed <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5 (with GFP fused to the N terminus of the signal peptides) in <italic> N. benthamiana </italic> via agroinfiltration, <span class="trait">and</span> inoculated the leaves with <italic> P. capsici </italic> at 2 days post agroinfiltration. At 36&#x02009;h.p.i., larger lesions were observed on leaves expressing each of the four <span class="gene">GmBiPs</span> ( <xref ref-type="fig" rid="f8"> Fig. 8a </xref> ) or NbBiP5 ( <xref ref-type="fig" rid="f8"> Fig. 8b </xref> ), indicating that increased accumulation of BiPs promoted <italic> P. capsici </italic> infection. Consistent with this result, overexpression of GFP-SP-NbBiP5-HDEL in soybean hairy roots also increased their susceptibility to <italic> P. sojae </italic> ( <xref ref-type="fig" rid="f8"> Fig. 8c,d </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 9 </xref> ). In addition, the overexpression of GFP-SP-NbBiP5-HDEL could partly restore the susceptibility of the plants to <italic> PsAvh262 </italic> -silenced <italic> P. sojae </italic> transformants. Taken together, our results suggest that PsAvh262-mediated virulence is affected through host BiPs ( <xref ref-type="fig" rid="f8"> Fig. 8d </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 9 <span class="trait">and</span> 10 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Since PsAvh262 suppresses BAX-triggered cell death <span class="trait">and</span> interacts with all four soybean BiPs ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> <span class="trait">and</span> <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 1c </xref> ; <xref ref-type="supplementary-material" rid="S1"> 6 </xref> ), we also examined the influence of NbBiP5, <span class="gene">GmBiP1</span> <span class="trait">and</span> <span class="gene">GmBiP4</span> on BAX-triggered cell death. ER stress is a component of the mechanism by which BAX triggers cell death <xref ref-type="bibr" rid="b44"> 44 </xref> . Our results showed that these three proteins (expressed with GFP attached to the N terminus of the signal peptide) all significantly alleviated BAX-triggered cell death <span class="trait">and</span> electrolyte leakage compared with a GFP control when expressed in <italic> N. benthamiana </italic> ( <xref ref-type="fig" rid="f8"> Fig. 8e,f </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 13 </xref> ). Thus, these results suggest that BiPs can negatively regulate cell death triggered by ER stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Hemibiotrophic pathogens initially establish a biotrophic relationship with their hosts, but at later stages of the infection they kill the host cells. During the initial biotrophic phase, the pathogen feeds on viable host tissues for nutrition via haustoria <xref ref-type="bibr" rid="b10"> 10 </xref> <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b46"> 46 </xref> ; thus, efficient mechanisms must be employed to suppress or evade host defenses at this stage, especially programmed cell death (PCD) <xref ref-type="bibr" rid="b47"> 47 </xref> . To achieve this, <italic> Phytophthora </italic> pathogens deliver a large repertoire of effectors into host cells to suppress immune responses including PCD. PCD is a widespread mechanism for plants to cope with environmental stresses such as attacks by pathogens. Identifying the target host factors of these effectors <span class="trait">and</span> their functions will lead to detailed understanding of how infection is established. Using a combination of genetic, cellular <span class="trait">and</span> biochemical approaches, we established the <italic> P. sojae </italic> RxLR effector PsAvh262 as a critical virulence factor for infection in soybean. One function of PsAvh262 is to suppress cell death triggered by PAMPs <span class="trait">and</span> effectors. We identified BiPs from soybean as one of the major virulence targets of PsAvh262. PsAvh262 could also interact with <italic> N. benthamiana </italic> <span class="trait">and</span> rice homologues of <span class="gene">GmBiPs.</span> Our experiments showed that PsAvh262 stabilizes host BiPs from degradation by an MG132-sensitive mechanism, that is probably the ERAD pathway. Furthermore, overexpression of BiPs could suppress PCD triggered by BAX; BAX relies partly on ER-stress to trigger PCD. Finally, our results showed that PsAvh262-silenced <italic> P. sojae </italic> lines were ineffective in suppressing PCD associated with infection-induced ER stress. Even when the level of ER stress was reduced by the poor virulence of the <italic> PsAvh262 </italic> -silenced <italic> P. sojae </italic> lines (as measured by PDI transcript levels; <xref ref-type="fig" rid="f6"> Fig. 6c </xref> ), the silenced lines were ineffective in suppressing PCD (as measured by VPE transcript levels; <xref ref-type="fig" rid="f6"> Fig. 6d </xref> ). Together, these results suggest that stabilization of BiPs by PsAvh262 acts to attenuate PCD triggered by infection-associated ER stress ( <xref ref-type="fig" rid="f9"> Fig. 9 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						There is growing evidence that different forms of PCD may harm or benefit pathogens depending on the timing <span class="trait">and</span> nature of the cell death mechanism, <span class="trait">and</span> whether the pathogen has physiologically adapted to benefit from the dying tissue <xref ref-type="bibr" rid="b26"> 26 </xref> . Among the various cellular stress responses that induce plant PCD, the ER stress response governs <span class="trait">resistance</span> to diverse environmental stresses <xref ref-type="bibr" rid="b43"> 43 </xref> <xref ref-type="bibr" rid="b48"> 48 </xref> . BiPs play both broad <span class="trait">and</span> specific roles in regulating ER stress-triggered cell death <xref ref-type="bibr" rid="b41"> 41 </xref> <xref ref-type="bibr" rid="b49"> 49 </xref> . For example, they act as moderators of PCD <span class="trait">and</span> master regulators of the UPR during ER stress <xref ref-type="bibr" rid="b27"> 27 </xref> <xref ref-type="bibr" rid="b42"> 42 </xref> <xref ref-type="bibr" rid="b50"> 50 </xref> <xref ref-type="bibr" rid="b51"> 51 </xref> <xref ref-type="bibr" rid="b52"> 52 </xref> . Thus, attenuation of ER stress-triggered PCD by PsAvh262 through its interaction with BiPs represents a novel <span class="trait">and</span> effective counter-defence strategy by <italic> P. sojae </italic> . BiP-targeting is likely of broad significance, given that BiPs function in systemic acquired <span class="trait">resistance</span> (SAR) <xref ref-type="bibr" rid="b27"> 27 </xref> , pathogen recognition receptor (PRR) biosynthesis <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b53"> 53 </xref> <span class="trait">and</span> regulation of cell surface receptor-mediated innate immunity <xref ref-type="bibr" rid="b54"> 54 </xref> . Silencing of <italic> BiPs </italic> in <italic> N. benthamiana </italic> resulted in delays in PCD induced by <italic> Xanthomonas oryzae </italic> pv. <italic> oryzae </italic> , but not by <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> strain DC3000 (ref. <xref ref-type="bibr" rid="b42"> 42 </xref> ). Silencing of ERD2a and/or ERD2b reduced accumulation of BiPs in the ER, resulting in increased sensitivity of plants to ER stress <span class="trait">and</span> exacerbation of PCD induced by non-host pathogens <xref ref-type="bibr" rid="b42"> 42 </xref> . In contrast, Carvalho <italic> et al </italic> . <xref ref-type="bibr" rid="b41"> 41 </xref> found that soybean transgenic lines overexpressing <span class="gene">GmBiP4</span> displayed enhanced or accelerated PCD induced by <italic> Pst </italic> DC3000 <xref ref-type="bibr" rid="b41"> 41 </xref> . In <italic> Arabidopsis </italic> , transgenic plants overexpressing another ER chaperone, calreticulin 2 (CRT2), were more susceptible to <italic> Pst </italic> DC3000 <xref ref-type="bibr" rid="b55"> 55 </xref> . Thus, the expression level of ER chaperones may increase or decrease plant susceptibility, depending on the pathogen involved. Our results, combined with the observations of these authors, highlight the ER stress response as a key battleground in the struggle between plants <span class="trait">and</span> pathogens to control plant cell death.
					</p>
				
					<p>&nbsp;&nbsp;
						We found that PsAvh262 co-localizes with BiPs to the ER in uninfected plant cells, <span class="trait">and</span> also to the EHM during infection of <italic> N. benthamiana </italic> by <italic> P. infestans </italic> ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ). Co-localization of BiPs <span class="trait">and</span> PsAvh262 to the ER displayed some unexpected features. First, the localization of <span class="gene">GmBiP1</span> to the ER was unaffected by fusion of GFP or RFP to the N terminus of its signal peptide, or to the C terminus of its HDEL ER retention signal ( <xref ref-type="fig" rid="f4"> Fig. 4b,c </xref> ). This suggests that the BiPs may reach the ER by a post-translational pathway <xref ref-type="bibr" rid="b56"> 56 </xref> . Second, our BiFC results revealed that PsAvh262 could co-localize with BiP even when BiP was targeted directly to the ER by an unobstructed signal peptide ( <xref ref-type="fig" rid="f4"> Fig. 4f,g </xref> ). This observation suggests that even with an unobstructed signal peptide, BiP passes through the cytoplasm where it may interact with PsAvh262. In the absence of co-expressed <span class="gene">GmBiP1,</span> PsAvh262 appeared to display little localization to the ER ( <xref ref-type="fig" rid="f4"> Fig. 4a </xref> ). A small amount of localization to punctate structures was observed; this might represent localization to the ER mediated by endogenous BiPs, however the identity of the punctate structures was not confirmed in our study. Furthermore, the PsAvh262 M1 deletion mutant that does not interact with BiP did not colocalize to the ER ( <xref ref-type="fig" rid="f4"> Fig. 4e </xref> ). These results suggest that cytoplasmic PsAvh262 is dependent on BiP binding to localize to the ER. When PsAvh262 was targeted directly to the ER by the signal peptide of <span class="gene">GmBiP1,</span> it could also complex with <span class="gene">GmBiP1</span> as judged by our BiFC results ( <xref ref-type="fig" rid="f4"> Fig. 4f,g </xref> ). Thus PsAvh262 may have the ability to find <span class="trait">and</span> bind to BiP whether it is in the cytoplasm or the ER.
					</p>
				
					<p>&nbsp;&nbsp;
						PsAvh262 was inserted into the vector pGEX4T-2 (GE Healthcare Life Science, Little Chalfont, Buckinghamshire, UK) <span class="trait">and</span> expressed in <italic> E. coli </italic> strain BL21. Soluble total proteins (GST-Avh262) were incubated with 50&#x02009;&#x003bc;l glutathione-agarose beads (GE Healthcare Life Science) at 4&#x02009;&#x000b0;C for 2&#x02009;h, supernatant was removed <span class="trait">and</span> the beads were washed three times with ice-cold phosphate-buffered saline (PBS) buffer. Next, <span class="gene">GmBiP1-RFP</span> protein extracted from <span class="gene">GmBiP1-RFP-expressing</span> <italic> N. benthamiana </italic> plants was added to the above beads for the incubation at 4&#x02009;&#x000b0;C for 2&#x02009;h. After six times of washing with 1 &#x000d7; PBST, the SDS sample loading buffer was added to beads <span class="trait">and</span> heated to 100&#x02009;&#x000b0;C for 5&#x02009;min for the immunoblot analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription data of the four <italic> <span class="gene">GmBiP</span> </italic> genes were obtained from RNA-seq for infected soybean roots (cultivar: Williams 82) by <italic> P. sojae </italic> (P6497) zoospores (NCBI Sequence Read Archive (SRA) accession: SRP073278). Except 0&#x02009;h.p.i., using only soybean roots, the others (0.5, 3, 6 <span class="trait">and</span> 12&#x02009;h.p.i.) were mixed samples of soybean roots <span class="trait">and</span> <italic> P. sojae </italic> . RNA was extracted using the EZNA total RNA kit I (Omega Bio-Tek, USA) <span class="trait">and</span> used to make RNA-seq libraries with the Illumina TruSeq RNA Sample Preparation Kit v2 following the manufacturer's recommendations. These libraries were sequenced using Illumina HiSeq 2000 in paired-end mode with a read length of 90&#x02009;bp. The sequence reads were mapped to the soybean Williams 82 genome (Phytozome database, <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="https://phytozome.jgi.doe.gov/"> https://phytozome.jgi.doe.gov/ </ext-link> ; Glycine max Wm82.a2.v1), using TopHat 2.1.0 ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://ccb.jhu.edu/software/tophat/index.shtml"> http://ccb.jhu.edu/software/tophat/index.shtml </ext-link> ). The gene expression levels were calculated using Cufflinks 2.2.1 ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://cole-trapnell-lab.github.io/cufflinks"> http://cole-trapnell-lab.github.io/cufflinks </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The sequences data (PsAvh262, <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5) that support the finding of this study have been deposited in GenBank ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/genbank/"> http://www.ncbi.nlm.nih.gov/genbank/ </ext-link> ), Phytozome ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="https://phytozome.jgi.doe.gov/pz/portal.html"> https://phytozome.jgi.doe.gov/pz/portal.html </ext-link> ) <span class="trait">and</span> Sol Genomics Network ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://solgenomics.net/"> http://solgenomics.net/ </ext-link> ), respectively. The primary accession codes are JN254253.1 (PsAvh262, GenBank); <span class="gene">Glyma08g02940</span> <span class="gene">(GmBiP1),</span> <span class="gene">Glyma08g02960</span> <span class="gene">(GmBiP2),</span> <span class="gene">Glyma05g36600</span> <span class="gene">(GmBiP3)</span> <span class="trait">and</span> <span class="gene">Glyma05g36620</span> <span class="gene">(GmBiP4)</span> in Phytozome; NbS00040865g0006.1 (NbBiP5, Sol Genomics Network). RNA-seq data described in this study has been deposited in the NCBI sequence read archive under accession code SRP073278. The authors declare that all other data supporting the findings of this study are available from the corresponding author on request.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) PsAvh262 interacts with soybean BiPs <italic> in planta </italic> . Total proteins were extracted from <italic> N. benthamiana </italic> leaves expressing 3 &#x000d7; Flag-PsAvh262 together with GFP-GmBiP1 or GFP-GmBiP4 (GFP was attached to the N terminus of the signal peptides of the <span class="gene">GmBiPs).</span> Protein complexes were pulled down using anti-Flag agarose beads <span class="trait">and</span> the co-precipitation of GFP-GmBiP1 or GFP-GmBiP4 was detected by western blotting using an anti-GFP antibody. ( <bold> b </bold> ) Proteins extracted from <italic> N. benthamiana </italic> leaves expressing <span class="gene">GmBiP1-RFP</span> (RFP was fused to the C-terminal HDEL sequence of <span class="gene">GmBiP1)</span> or RFP were incubated with <italic> E. coli </italic> homogenate containing GST-PsAvh262 or GST. Enrichment of <span class="gene">GmBiP1-RFP</span> in GST-PsAvh262-bound glutathione resins was detected using an anti-RFP antibody. ( <bold> c </bold> ) M1 deletion greatly reduces the interaction of PsAvh262 with <span class="gene">GmBiP1.</span> 3 &#x000d7; Flag-PsAvh262 or its mutants were co-expressed in <italic> N. benthamiana </italic> with <span class="gene">GmBiP1-RFP.</span> Protein complexes were pulled down using anti-Flag agarose gel, <span class="trait">and</span> co-precipitations of PsAvh262 <span class="trait">and</span> its mutants with <span class="gene">GmBiP1</span> were detected using immunoblots. Membranes were stained with Ponceau S to confirm equal loading. &#x0002a;, the objective bands of <span class="gene">GmBiPs.</span> &#x00394;, non-specific bands of <span class="gene">GmBiPs</span> when using anti-GFP or anti-RFP, similar to GFP-PsAvh262 protein. Similar results were obtained in at least three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						In all panels, proteins were expressed in <italic> N. benthamiana </italic> through agroinfiltration. Fluorescence was detected in epidermal cells of the infiltrated leaves by confocal microscopy at 48&#x02009;h after agroinfiltration. Scale bars, 10&#x02009;&#x003bc;m. ( <bold> a </bold> ) Localization of RFP- <span class="trait">and</span> GFP-fusions of PsAvh262 <span class="trait">and</span> mutants in the absence of <span class="gene">GmBiP1.</span> FPs were attached to the N terminus of PsAvh262 without its signal peptide. PsAvh262 <span class="trait">and</span> mutants M2 <span class="trait">and</span> M3 were primarily located in the nucleus <span class="trait">and</span> nucleolus, but also in punctate structures in the cell cortex. RFP-PsAvh262-M1 was no longer present in cell cortex. ( <bold> b </bold> , <bold> c </bold> ) Localization of <span class="gene">GmBiP1</span> FP fusions to the ER. <span class="gene">GmBiP1</span> fusions were targeted to ER-like structures irrespective of the placement of the FP ( <bold> b </bold> ). GFP-SP-GmBiP1-HDEL co-localizes with SP-RFP-HDEL, but not with RFP ( <bold> c </bold> ). ( <bold> d </bold> , <bold> e </bold> ) RFP-PsAvh262 <span class="trait">and</span> GFP-SP-GmBiP1-HDEL co-localize to the ER. Co-localization to the ER network ( <bold> d </bold> ) <span class="trait">and</span> to the peri-nuclear ER ( <bold> e </bold> ). The M1 deletion eliminates co-localization to the peri-nuclear ER. ( <bold> f </bold> , <bold> g </bold> ) Co-localization of PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> detected by bimolecular fluorescence complementation (BiFC) using the constructs shown in <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 7a </xref> . Complementation is only observed with <span class="gene">GmBiP1</span> constructs ( <bold> f </bold> ) but not control constructs ( <bold> g </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Accumulations of <span class="gene">GmBiPs</span> in soybean after hypocotyls were infected with <italic> P. sojae </italic> P6497 (wild-type) or two <italic> PsAvh262 </italic> -silenced transformants (S62 <span class="trait">and</span> S76) were determined by western blotting using an anti-BiP antibody. Antibody against actin was used as an internal standard. Numbers below the blot indicate relative abundances of <span class="gene">GmBiPs.</span> Asterisks (&#x0002a; or &#x0002a;&#x0002a;) denote significant differences ( <italic> P </italic> &#x0003c;0.05 or <italic> P </italic> &#x0003c;0.01, one-way ANOVA, <italic> n </italic> =3) between samples. ( <bold> b </bold> &#x02013; <bold> d </bold> ) Transcript abundances of ER-stress-related genes. The soybean housekeeping gene <italic> CYP2 </italic> (TC224926) was used as an internal standard in each case. <italic> <span class="gene">GmBiP1</span> </italic> ( <bold> b </bold> ), <italic> Protein disulfide isomerase </italic> ( <italic> PDI </italic> ) ( <bold> c </bold> ) <span class="trait">and</span> <italic> vacuolar processing enzyme </italic> ( <italic> VPE </italic> ) ( <bold> d </bold> ) were determined by qRT&#x02013;PCR in soybeans inoculated with wild-type or <italic> PsAvh262 </italic> -silenced mutants of <italic> P. sojae </italic> . Error bars represent the mean&#x000b1;s.d.( <italic> n </italic> =3) <span class="trait">and</span> asterisks (&#x0002a;, &#x0002a;&#x0002a; or &#x0002a;&#x0002a;&#x0002a;) denote significant differences ( <italic> P </italic> &#x0003c;0.05, <italic> P </italic> &#x0003c;0.01 or <italic> P </italic> &#x0003c;0.001, respectively) between samples. The statistical significance of the pairwise differences between <italic> PsAvh262 </italic> -silenced mutants <span class="trait">and</span> wild-type strain at 0&#x02009;d.p.i. was assessed with one-way ANOVA. Similar results were observed in three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Immunoblots showing the stabilization of GFP-GmBiP1 by PsAvh262 or the 26S proteasomal inhibitor MG132, but not by PsAvh262-M1. GFP-GmBiP1 (GFP was attached to the N terminus of the signal peptide of <span class="gene">GmBiP1)</span> was expressed in <italic> N. benthamiana </italic> leaves together with wild-type PsAvh262 or its mutants by agroinfiltration. MG132 (100&#x02009;&#x003bc;M) or DMSO was infiltrated into the leaves at 24&#x02009;h before protein extraction. Abundances of GFP-GmBiP1 were determined using an anti-GFP antibody. Numbers below the blot indicate relative abundances of <span class="gene">GmBiP1.</span> ( <bold> b </bold> ) Endogenous NbBiPs were stabilized by PsAvh262, but not by PsAvh262-M1 in <italic> N. benthamiana </italic> infected with <italic> P. capsici </italic> . <italic> N. benthamiana </italic> leaves expressing GFP, PsAvh262, PsAvh262-M1 or PsAvh262-M2 were inoculated with zoospore suspension of <italic> P. capsici </italic> , <span class="trait">and</span> the NbBiP levels were determined at 0, 3 <span class="trait">and</span> 24&#x02009;h.p.i. Abundances of NbBiPs relative to GFP at 0&#x02009;h were determined using an anti-BiP antibody. Numbers below the blots represent relative abundances of NbBiPs. Asterisks (&#x0002a;, &#x0002a;&#x0002a; or &#x0002a;&#x0002a;&#x0002a;) denote significant differences ( <italic> P </italic> &#x0003c;0.05, <italic> P </italic> &#x0003c;0.01 or <italic> P </italic> &#x0003c;0.001, respectively, one-way ANOVA, <italic> n </italic> =3) between samples. Similar results were observed in at least three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Expression of <span class="gene">GmBiPs</span> in <italic> N. benthamiana </italic> led to increased susceptibility to <italic> P. capsici </italic> . Leaves transiently expressing GFP-GmBiPs (GFP was attached to the N terminus of the signal peptide in each case) or GFP were inoculated with mycelia of <italic> P. capsici </italic> . Disease lesions were visualized by Trypan blue staining of the inoculated leaves at 36&#x02009;h.p.i. (upper panel) <span class="trait">and</span> the sizes of the lesions were statistically analysed (lower panel). ( <bold> b </bold> ) Expression of GFP-NbBiP5 (GFP was attached to the N terminus of the signal peptide of NbBiP5) in <italic> N. benthamiana </italic> led to increased susceptibility to <italic> P. capsici </italic> . Disease lesions were visualized by Trypan blue staining at 36 h.p.i. (left panel) <span class="trait">and</span> the sizes of the lesions were statistically analysed (right panel). ( <bold> c </bold> ) Expression of GFP-SP-NbBiP5-HDEL in soybean hairy roots enhanced the infection by <italic> P. sojae </italic> strain P6497-RFP. Increased numbers of oospores developed in NbBiP5-expressing roots at 48&#x02009;h after inoculation as shown by microscopic images (left panel) <span class="trait">and</span> statistical analysis (right panel). ( <bold> d </bold> ) Overexpression of GFP-SP-NbBiP5-HDEL can partly restore the virulence of <italic> PsAvh262 </italic> -silenced <italic> P. sojae </italic> transformants in soybean roots. Soybean hairy roots expressing NbBiP5 or GFP were inoculated by wild-type <italic> P. sojae </italic> or <italic> PsAvh262 </italic> -silenced transformants (262-S62, 262-S76), respectively. Increased number of oospores was developed in roots expressing NbBiP5 at 48&#x02009;h after infection by <italic> PsAvh262 </italic> -silenced <italic> P. sojae </italic> transformants. ( <bold> e </bold> ) Expression of <span class="gene">GmBiP1</span> or <span class="gene">GmBiP4</span> attenuated cell death triggered by BAX in <italic> N. benthamiana </italic> . GFP-SP-GmBiP1-HDEL, GFP-SP-GmBiP4-HDEL or GFP were expressed in <italic> N. benthamiana </italic> through agroinfiltration. Twelve hours later, the leaves were infiltrated again with <italic> A. tumefaciens </italic> cells carrying <italic> HA </italic> tagged <italic> BAX </italic> . Leaves were bleached with ethanol to visualize the cell death symptoms. Immunoblots confirmed the expression of GFP-SP-GmBiP1-HDEL <span class="trait">and</span> GFP-SP-GmBiP4-HDEL in <italic> N. benthamiana </italic> (lower panel). ( <bold> f </bold> ) Quantification of electrolyte leakage in <italic> N. benthamiana </italic> leaves expressing BAX together with GFP-SP-GmBiP1-HDEL, GFP-SP-GmBiP4-HDEL or GFP, 4 days after agroinfiltration. Electrolyte leakage from the infiltrated leaf discs was measured as a percentage of leakage from boiled discs. Error bars represent the mean&#x000b1;s.d. ( <italic> n </italic> =3), <span class="trait">and</span> asterisks denote significant differences (&#x0002a; <italic> P </italic> &#x0003c;0.05; &#x0002a;&#x0002a; <italic> P </italic> &#x0003c;0.01; &#x0002a;&#x0002a;&#x0002a; <italic> P </italic> &#x0003c;0.001, one-way ANOVA). These experiments were repeated three times with similar results.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Association Study of <span class="trait">Resistance</span> to Soybean Cyst Nematode ( <italic> Heterodera glycines </italic> ) HG Type 2.5.7 in Wild Soybean ( <italic> Glycine soja </italic> )</h2>
			<h3>PMID:27582748</h3>
				
					<h3>PMCID:4987380</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hengyou </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334878/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Chunying </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Davis </surname>  <given-names> Eric L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jinshe </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Griffin </surname>  <given-names> Joshua D. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kofsky </surname>  <given-names> Janice </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/362990/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Bao-Hua </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/356962/overview"/>  </contrib>
				<br/> 
				年份：<day> 02 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A key step in developing SCN-resistant varieties is elucidation of the genetic basis of <span class="trait">resistance.</span> To date, knowledge of the genetic basis of soybean SCN <span class="trait">resistance</span> is largely based on classical quantitative trait locus (QTL) mapping studies. Additionally, most research has focused on dissecting the mechanism of soybean <span class="trait">resistance</span> to races 1 <span class="trait">and</span> 3, which are prevalent in the central US. The identification of two QTLs, <italic> rhg1 </italic> on chromosome 18, conferring HG Type 0 <span class="trait">resistance</span> (Cook et al., <xref rid="B11" ref-type="bibr"> 2012 </xref> ), <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> on chromosome 8, conferring HG Type 7 <span class="trait">resistance</span> (Liu et al., <xref rid="B31" ref-type="bibr"> 2012 </xref> ), has suggested a race-specific <span class="trait">resistance</span> mechanism for the soybean-SCN interaction. However, race specificity is relatively unusual because cloning of the <italic> rhg1 </italic> (Cook et al., <xref rid="B11" ref-type="bibr"> 2012 </xref> ) locus indicated that rather canonical NB-LRR <span class="trait">resistance</span> genes (DeYoung <span class="trait">and</span> Innes, <xref rid="B13" ref-type="bibr"> 2006 </xref> ), the copy number of three tandem genes (an amino acid transporter, an alpha-SNAP, <span class="trait">and</span> the WI12 protein) <span class="trait">and</span> <italic> rhg-4 </italic> (a serine hydroxymethyltransferase; Liu et al., <xref rid="B31" ref-type="bibr"> 2012 </xref> ) confer <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on detailed annotations for the soybean reference genome in SoyBase ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> http://www.soybase.org </ext-link> ), we further predicted candidate genes within the 50 kb region on each side of the 10 associated SNPs (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Only one (ss71561193) of the 10 SNPs is located within a gene, <italic> <span class="gene">Glyma.18g196200</span> </italic> , which encodes an mRNA capping enzyme family protein involved in transferring phosphorous-containing groups (SoyBase, <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> http://www.soybase.org </ext-link> ). The other nine SNPs are located in intergenic regions. A total of 58 gene models were predicted within the search region (Table <xref ref-type="supplementary-material" rid="SM1"> S3 </xref> ), 16 of which are related to disease <span class="trait">resistance</span> based on published literature <span class="trait">and</span> considered to be candidate genes associated with SCN <span class="trait">resistance</span> (Table <xref ref-type="supplementary-material" rid="SM1"> S4 </xref> ). This candidate list includes genes encoding leucine-rich repeat (LRR)-containing <span class="trait">resistance</span> (R) proteins or protein kinases (PKs), a receptor-like protein, a RING/U-box protein, <span class="trait">and</span> MYB family transcription factors. Briefly, two identified LRR-R genes ( <italic> <span class="gene">Glyma.18g077900</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Glyma.18g078000</span> </italic> ) are close to SNP ss715632647. The gene <italic> <span class="gene">Glyma.18g078000</span> </italic> , which showed root-specific expression in soybean (Severin et al., <xref rid="B42" ref-type="bibr"> 2010 </xref> ), is only 5.4 kb away from SNP ss715632647. In addition, two PK-encoding genes ( <italic> <span class="gene">Glyma.18g106800,</span> <span class="gene">Glyma.18g107000</span> </italic> ) <span class="trait">and</span> a phosphate acyltransferase-encoding gene, <italic> <span class="gene">Glyma.18g107100</span> </italic> , within a genomic region containing two tightly linked significant SNPs (ss715628640 <span class="trait">and</span> ss715628650) were identified as candidates (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). <italic> <span class="gene">Glyma.18g106800</span> </italic> , which encodes a mitogen-activated protein kinase (MAPK), is reported to be highly expressed in roots (Severin et al., <xref rid="B42" ref-type="bibr"> 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Wild crop relatives harbor novel <span class="trait">and</span> exotic genetic resources that are crucial for the development of insect pest management (Hajjar <span class="trait">and</span> Hodgkin, <xref rid="B17" ref-type="bibr"> 2007 </xref> ). Some wild species have been applied in breeding practices, such as for tomato (Vidavski et al., <xref rid="B48" ref-type="bibr"> 2008 </xref> ; Kumar et al., <xref rid="B27" ref-type="bibr"> 2014 </xref> ). <italic> G. soja </italic> is widely distributed in China, Korea, Japan, <span class="trait">and</span> Northeast Russia, <span class="trait">and</span> long-term exposure to various environmental stresses in the wild has rendered <italic> G. soja </italic> able to adapt to diverse environments with severe abiotic <span class="trait">and</span> biotic stresses. Some of these beneficial traits, such as salt tolerance <span class="trait">and</span> pest <span class="trait">resistance,</span> may have been lost in cultivated soybean due to the effects of domestication &#x0201c;bottleneck.&#x0201d; Indeed, a recent study showed that the domestication bottleneck of soybean has resulted in more than a 50% reduction in diversity <span class="trait">and</span> the elimination of 81% of the rare alleles present in <italic> G. soja </italic> (Hyten et al., <xref rid="B20" ref-type="bibr"> 2006 </xref> ). For example, Qi et al. ( <xref rid="B38" ref-type="bibr"> 2014 </xref> ) recently identified a salt-tolerance gene, <italic> <span class="gene">GmCHX1</span> </italic> , unique to wild soybean. In addition, the limited availability of SCN-resistant soybean varieties <span class="trait">and</span> the selection for SCN populations that overcome existing <span class="trait">resistance</span> have been two major challenges to SCN management (Mitchum et al., <xref rid="B32" ref-type="bibr"> 2007 </xref> ; Niblack et al., <xref rid="B36" ref-type="bibr"> 2008 </xref> ). As an alternative, our study <span class="trait">and</span> several recent works (Winter et al., <xref rid="B52" ref-type="bibr"> 2007 </xref> ; Kim et al., <xref rid="B25" ref-type="bibr"> 2011 </xref> ; Kim <span class="trait">and</span> Diers, <xref rid="B24" ref-type="bibr"> 2013 </xref> ) have identified resistant accessions among wild soybean populations, suggesting that wild soybean have great potential for improving SCN <span class="trait">resistance.</span> These newly identified resistant resources may contain both conserved loci <span class="trait">and</span> new <span class="trait">resistance</span> genes, as shown in our study (Table <xref ref-type="table" rid="T1"> 1 </xref> ) <span class="trait">and</span> previous studies (Winter et al., <xref rid="B52" ref-type="bibr"> 2007 </xref> ; Kim et al., <xref rid="B25" ref-type="bibr"> 2011 </xref> ; Kim <span class="trait">and</span> Diers, <xref rid="B24" ref-type="bibr"> 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Development of soybean cultivars with broad <span class="trait">resistance</span> to diverse SCN HG types, as well as identification of novel resistant resources/genes, are urgently needed due to SCN HG type shifts caused by evolutionary arms races. The recent identification of two major loci, <italic> rhg1 </italic> (Cook et al., <xref rid="B11" ref-type="bibr"> 2012 </xref> ) <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> (Liu et al., <xref rid="B31" ref-type="bibr"> 2012 </xref> ), conferring SCN <span class="trait">resistance</span> (Concibido et al., <xref rid="B9" ref-type="bibr"> 2004 </xref> ) has provided increasing knowledge <span class="trait">and</span> understanding about the genetic architecture <span class="trait">and</span> <span class="trait">resistance</span> mechanisms of soybean defenses against SCN. The genomic regions linked with <italic> rhg1 </italic> are significantly associated with HG 2.5.7 <span class="trait">resistance.</span> <italic> <span class="gene">Rhg1</span> </italic> , one of the major loci conferring soybean <span class="trait">resistance</span> to multiple SCN HG types (previously called races; Concibido et al., <xref rid="B9" ref-type="bibr"> 2004 </xref> ), has also been previously identified in a resistant <italic> G. soja </italic> accession ( <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="PI468916"> PI468916 </ext-link> ) by linkage mapping (Kim et al., <xref rid="B25" ref-type="bibr"> 2011 </xref> ). Thus, it is not surprising that the <italic> rhg1 </italic> -linked regions were also identified in our study significantly associated with HG 2.5.7 <span class="trait">resistance.</span> In addition, loci located at the opposite end of chromosome 18 as <italic> rhg1 </italic> (Vuong et al., <xref rid="B49" ref-type="bibr"> 2010 </xref> ; Kim et al., <xref rid="B25" ref-type="bibr"> 2011 </xref> ; Kim <span class="trait">and</span> Diers, <xref rid="B24" ref-type="bibr"> 2013 </xref> ; Bao et al., <xref rid="B4" ref-type="bibr"> 2014 </xref> ) <span class="trait">and</span> on chromosome 19 (Vuong et al., <xref rid="B50" ref-type="bibr"> 2015 </xref> ) identified in our study have also recently been suggested as contributing to broad SCN <span class="trait">resistance.</span> The consistent results observed in the present <span class="trait">and</span> previous studies suggest that the significant loci on chromosomes 18 <span class="trait">and</span> 19 might play important roles in conferring SCN <span class="trait">resistance</span> in both <italic> G. max </italic> <span class="trait">and</span> <italic> G. soja </italic> populations. These finding will significantly accelerate molecular marker development <span class="trait">and</span> SCN <span class="trait">resistance</span> breeding programmes (e.g., Kadam et al., <xref rid="B23" ref-type="bibr"> 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Protein phosphorylation/dephosphorylation is one of the major mechanisms involved in plant-pathogen interactions (Xing <span class="trait">and</span> Laroche, <xref rid="B55" ref-type="bibr"> 2011 </xref> ). The process of phosphorylation/dephosphorylation involves various types of protein kinases (PKs), such as calmodulin-dependent receptor-like protein kinases (CDPKs) <span class="trait">and</span> MAPKs. The involvement of various types of PKs in plant defense against nematodes has also been reported in other plant species. For example, it has been suggested that two <italic> Arabidopsis </italic> MAPK genes, <italic> MPK3 </italic> , <span class="trait">and</span> <italic> MPK6 </italic> , are important regulators of the plant-nematode interaction (Sidonskaya et al., <xref rid="B43" ref-type="bibr"> 2016 </xref> ). In tobacco, a <span class="gene">CDPK</span> protein can trigger plant defense via signaling upon virus attack (Romeis et al., <xref rid="B41" ref-type="bibr"> 2000 </xref> ), <span class="trait">and</span> in <italic> Arabidopsis </italic> , mutation of a receptor-like protein kinase ( <italic> <span class="gene">RLK</span> </italic> ) gene, <italic> RPK2 </italic> , results in a decrease in both nematode infection <span class="trait">and</span> syncytium size in the <italic> rpk2 </italic> mutant (Replogle et al., <xref rid="B39" ref-type="bibr"> 2013 </xref> ). Such evidence strongly suggests that PKs are promising candidate genes involved in the soybean response to SCN attack. Regardless, involvement of MAPKs <span class="trait">and</span> CDPKs in the soybean-SCN interaction has not been comprehensively investigated because it remains challenging to pinpoint causal genes from ~200 MAPKs (Neupane et al., <xref rid="B34" ref-type="bibr"> 2013 </xref> ) <span class="trait">and</span> ~500 receptor-like proteins (Zhou et al., <xref rid="B61" ref-type="bibr"> 2016 </xref> ) in the soybean genome. As an alternative, our study identified one MAPK ( <italic> <span class="gene">Glyma.18g106800</span> </italic> ) <span class="trait">and</span> one <span class="gene">CDPK</span> ( <italic> <span class="gene">Glyma.18g064100</span> </italic> ) that might be involved in wild soybean defense against SCN; this result must be further verified.
					</p>
				
					<p>&nbsp;&nbsp;
						A study by Guo et al. ( <xref rid="B16" ref-type="bibr"> 2015 </xref> ) indicated that a soybean ortholog of the <italic> Arabidopsis </italic> gene <italic> RPK2, <span class="gene">GmRPK2B</span> </italic> , is strongly expressed in root tips, vascular tissues, <span class="trait">and</span> nematode feeding sites. Similarly, high or exclusive expression of genes encoding PKs ( <italic> <span class="gene">Glyma.18g106800,</span> <span class="gene">Glyma.18g064100</span> </italic> ) <span class="trait">and</span> LRR-R proteins ( <italic> <span class="gene">Glyma.18g077900,</span> <span class="gene">Glyma.18g078000</span> </italic> ) in roots (Severin et al., <xref rid="B42" ref-type="bibr"> 2010 </xref> ) suggests their potential of playing specific roles in root defense. Future research should validate the functions of these candidate genes using root transformation assays.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 <span class="trait">and</span> HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping.</h2>
			<h3>PMID:26268218  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Han Y(1), Zhao X(2), Cao G(3), Wang Y(4), Li Y(5), Liu D(6), Teng W(7), Zhang Z(8), Li D(9), Qiu L(10), Zheng H(11), Li W(12).
				<br/> 
				年份：2015 Aug 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode (SCN, Heterodera glycines Ichinohe) is one of the most fatal <span class="trait">pests</span> of soybean (Glycine max (L.) Merr.) worldwide <span class="trait">and</span> causes huge loss of soybean <span class="trait">yield</span> each year. Multiple sources of <span class="trait">resistance</span> are urgently needed for effective management of SCN via the development of resistant cultivars. The aim of the present study was to investigate the genetic architecture of <span class="trait">resistance</span> to SCN HG Type 0 (race 3) <span class="trait">and</span> HG Type 1.2.3.5.7 (race 4) in landraces <span class="trait">and</span> released elite soybean cultivars mostly from China. RESULTS: A total of 440 diverse soybean landraces <span class="trait">and</span> elite cultivars were screened for <span class="trait">resistance</span> to SCN HG Type 0 <span class="trait">and</span> HG Type 1.2.3.5.7. Exactly 131 new sources of SCN <span class="trait">resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF) &gt; 4% that were present in 97% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the <span class="trait">resistance</span> for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <span class="gene">Rhg1</span> <span class="trait">and</span> <span class="gene">Rhg4</span> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease <span class="trait">resistance</span> proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, MYB <span class="trait">and</span> WRKY transcription activation families were identified. Such proteins may participate in the resistant reaction to SCN <span class="trait">and</span> were frequently found in the tightly linked genomic regions of the peak SNPs. CONCLUSIONS: GWAS extended understanding of the genetic architecture of SCN <span class="trait">resistance</span> in multiple genetic backgrounds. Nineteen association signals were obtained for the <span class="trait">resistance</span> to the two Hg Types of SCN. The multiple beneficial alleles from resistant germplasm sources will be useful for the breeding of cultivars with improved <span class="trait">resistance</span> to SCN. Analysis of genes near association signals may facilitate the recognition of the causal gene(s) underlying SCN resistances.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Transcriptome Analysis of Two Races of <italic> Heterodera glycines </italic> at Different Developmental Stages</h2>
			<h3>PMID:24662955</h3>
				
					<h3>PMCID:3963861</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Gaofeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Deliang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Bingli </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Wenkun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Lingan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Long </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Huan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jian </surname>  <given-names> Heng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Conceived <span class="trait">and</span> designed the experiments: DP. Performed the experiments: GW. Analyzed the data: GW BG. Contributed reagents/materials/analysis tools: WH HL HP LK. Wrote the paper: GW. Review of the manuscript: DP HJ <span class="gene">BG</span> LK HL HP.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean cyst nematode (SCN), <italic> Heterodera glycines </italic> , causes significant damage to soybean, resulting in annual <span class="trait">yield</span> losses between 1.9 <span class="trait">and</span> 3.5 million tons in the USA <xref rid="pone.0091634-Wrather1" ref-type="bibr"> [1] </xref> . Developing resistant cultivars to SCN has proven to be a highly effective strategy to control this nematode <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> . Several <span class="trait">resistance</span> genes <span class="trait">and</span> QTLs conferring <span class="trait">resistance</span> to <italic> H. glycines </italic> have been identified <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> &#x02013; <xref rid="pone.0091634-Liu1" ref-type="bibr"> [8] </xref> . However, few of these have been introgressed into commercially viable cultivars <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> . Field populations of <italic> H. glycines </italic> have been shown to have high variability <xref rid="pone.0091634-Zheng1" ref-type="bibr"> [9] </xref> , <span class="trait">and</span> the significant diversity of field populations combined with lack of applicable <span class="trait">resistance</span> sources could potentially lead to population shifts over time, resulting in virulent populations. For example, repeated growth of potato cultivars carrying the <span class="gene">H1</span> <span class="trait">resistance</span> gene against <italic> Globodera rostochiensis </italic> has led to selection for <italic> G. pallida </italic> in parts of Europe <xref rid="pone.0091634-Turner1" ref-type="bibr"> [10] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Total RNAs were isolated from preparasitic J2s, parasitic J2s, J3s <span class="trait">and</span> J4 females using TRIzol regent according to the manufacturer's instructions (Invitrogen, USA). mRNAs were amplified using the BD SMART mRNA Amplification Kit (Clontech, USA) as described above. The mRNAs were then reverse-transcribed using oligo (dT) <sub> 20 </sub> primers (Invitrogen, USA) <span class="trait">and</span> SuperScript III Reverse Transcriptase (Invitrogen, USA). qRT-PCR was performed on a Applied Biosystems 7500/7500 Fast Real-Time PCR System (Life technologies, USA) with 20-&#x000b5;L PCR mix containing 2 &#x000b5;L of cDNA, 200 nM of the appropriate oligonucleotide primers, 10 &#x000b5;L of 2&#x000d7; SYBR Premix Ex Taq <span class="trait">and</span> 0.4 &#x000b5;L of 50&#x000d7; ROX Reference Dye II (T&#x003b1;K&#x003b1;R&#x003b1;, Japan). The mix was preheated at 95&#x000b0;C for 15 seconds, followed by 40 cycles of 95&#x000b0;C for 3 seconds, 60&#x000b0;C for 30 seconds. Melting curve analysis was performed from incubation at 65&#x000b0;C followed by 0.5&#x000b0;C incremental ramp up to 95&#x000b0;C. Melting curve lines were used to validate product specificity. The <italic> H. glycines </italic> glyceraldehyde-3-phosphate dehydrogenase encoding gene (GAPDH, GenBank accession: CA939315) was used as an internal reference <xref rid="pone.0091634-Ithal1" ref-type="bibr"> [19] </xref> . The primers used for <span class="gene">GAPDH</span> amplification <span class="trait">and</span> gene-specific primers used for qRT-PCR are listed in <xref ref-type="supplementary-material" rid="pone.0091634.s001"> Table S1 </xref> . Standard curve lines were established with four appropriate serial dilutions of first-strand cDNAs. Relative transcript abundance was calculated using the 2 <sup> &#x02212;&#x00394;&#x00394;Ct </sup> method <xref rid="pone.0091634-Livak1" ref-type="bibr"> [20] </xref> , <span class="trait">and</span> the cut-off was 2.0 <xref rid="pone.0091634-Kong1" ref-type="bibr"> [21] </xref> . qRT-PCR was carried out in triplicate for each of the two different samples from each nematode stage. For relative gene expression level analysis of race 3-enriched genes between race 3 <span class="trait">and</span> race 4, values were expressed as the increase levels relative to the calibration sample from race 4 at the same nematode developmental stage, <span class="trait">and</span> the reverse for that of race 4-enriched genes. For gene expression profile analysis, preparasitic J2 stage was considered as the calibration stage, <span class="trait">and</span> the gene expression level at this stage was defined as 1.0.
					</p>
				
					<p>&nbsp;&nbsp;
						RNA samples were isolated from preparasitic J2s, parasitic J2s (i-J2), J3s (J3) <span class="trait">and</span> J4 females (J4 female) of SCN race 3 <span class="trait">and</span> race 4, respectively. The expression profiles of the 13 genes categorized into similar ( <bold> A </bold> ) <span class="trait">and</span> uncoordinated ( <bold> B </bold> ) expression profiles in both the races. Preparasitic J2 stage was considered as the calibration stage, <span class="trait">and</span> the expression level of these 13 genes at preparasitic J2 stage was arbitrarily set to 1.0. The expression level was analyzed with the 2 <sup> &#x02212;&#x00394;&#x00394;Ct </sup> method using the <span class="gene">GAPDH</span> gene as an internal reference gene for normalization. Mean <span class="trait">and</span> standard errors were determined with data from three independent replicates.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of polymorphism <span class="trait">and</span> transcription of the effector gene <span class="gene">Avr1b</span> in Phytophthora sojae isolates from China virulent to Rps1b.</h2>
			<h3>PMID:21726400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cui L(1), Yin W, Dong S, Wang Y.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effector gene <span class="gene">Avr1b-1</span> of Phytophthora sojae determines the efficacy of the <span class="trait">resistance</span> gene Rps1b in soybean. The sequences of the <span class="gene">Avr1b-1</span> locus in 34 Chinese isolates of P. sojae were obtained <span class="trait">and</span> analysed by polymerase chain reaction (PCR) <span class="trait">and</span> inverse PCR. Four different alleles <span class="trait">and</span> a complete deletion mutation of the <span class="gene">Avr1b-1</span> gene were identified. Molecular analysis of the deletion breakpoints in the <span class="gene">Avr1b-1</span> locus revealed that an 8-kb DNA sequence containing <span class="gene">Avr1b-1</span> was deleted <span class="trait">and</span> a 12.7-kb DNA sequence was inserted at the same locus. A truncated transposase gene was found <span class="trait">and</span> five transposable elements were predicted in the inserted sequence, suggesting that the deletion of <span class="gene">Avr1b-1</span> might be attributed to transposon movement. The transcription of <span class="gene">Avr1b-1</span> was analysed in virulent isolates containing four alleles of <span class="gene">Avr1b-1</span> by real-time reverse transcription-PCR. In all virulent isolates, only those isolates containing the second allele transcripted Avr1b-1.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>