<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Diseases and Pests resistance</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Diseases and Pests resistance</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 <span class="trait">and</span> HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping.</h2>
			<h3>PMID:26268218  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Han Y(1), Zhao X(2), Cao G(3), Wang Y(4), Li Y(5), Liu D(6), Teng W(7), Zhang Z(8), Li D(9), Qiu L(10), Zheng H(11), Li W(12).
				<br/> 
				年份：2015 Aug 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Of the 19 association signals, eight signals overlapped with reported QTL including <span class="gene">Rhg1</span> <span class="trait">and</span> <span class="gene">Rhg4</span> genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The receptor like kinase at <italic> <span class="gene">Rhg1</span> </italic> -a <italic> /Rfs2 </italic> caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome <span class="trait">and</span> soybean cyst nematode as a transgene by altering signaling responses</h2>
			<h3>PMID:22857610</h3>
				

				
					<h3>PMCID:3439264</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Srour </surname>  <given-names> Ali </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> asrour@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Afzal </surname>  <given-names> Ahmed J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> afzal.5@osu.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Blahut-Beatty </surname>  <given-names> Laureen </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> Laureen.Blahut-Beatty@AGR.GC.CA </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Hemmati </surname>  <given-names> Naghmeh </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> naghmeh@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Simmonds </surname>  <given-names> Daina H </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> daina.simmonds@agr.gc.ca </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> wenbinli@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Liu </surname>  <given-names> Miao </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> wenbinli@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Town </surname>  <given-names> Christopher D </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> cdtown@jcvi.org </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Sharma </surname>  <given-names> Hemlata </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> hemlatarca@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Arelli </surname>  <given-names> Prakash </given-names>  </name>  <xref ref-type="aff" rid="I8"> 8 </xref>  <email> prakash.arelli@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Lightfoot </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <xref ref-type="aff" rid="I9"> 9 </xref>  <email> ga4082@siu.edu </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						This study examined the role of a receptor like kinase (RLK) <italic> <span class="gene">GmRLK18-1</span> </italic> (gene model <italic> Glyma_18_02680 </italic> at 1,071&#x02009;kbp on chromosome 18 of the genome sequence) within the <italic> Rhg1/Rfs2 </italic> locus in causing <span class="trait">resistance</span> to SCN <span class="trait">and</span> SDS.
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59&#x02009;kbp at the center of which the <italic> <span class="gene">GmRLK18-1</span> </italic> was the most polymorphic gene <span class="trait">and</span> encoded protein.
					</p>
				
					<p>&nbsp;&nbsp;
						Analyses were made of plants that were either heterozygous at, or transgenic <span class="trait">(and</span> so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <italic> <span class="gene">GmRLK18-1</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the absence of <italic> <span class="gene">Rhg4</span> </italic> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root <span class="trait">and</span> leaf symptoms of SDS caused by <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <italic> <span class="gene">GmRLK18-1</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Equally the <italic> <span class="gene">Rhg1</span> </italic> locus underlay partial <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">Rhg1</span> </italic> locus was constant, being required for partial <span class="trait">resistance</span> to all Hg Types in most PIs <span class="trait">and</span> cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						Full <span class="trait">resistance</span> to SCN required 1&#x02013;4 loci in addition to <italic> <span class="gene">Rhg1</span> </italic> , the number depending on the nature of the cyst population parasitizing the roots [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">resistance</span> or susceptible interaction(s) between the nematode <span class="trait">and</span> soybean affected by <italic> <span class="gene">Rhg1</span> </italic> was not induced until females stopped moving through the roots <span class="trait">and</span> established a feeding site comprising several giant cells [ <xref ref-type="bibr" rid="B28"> 28 </xref> - <xref ref-type="bibr" rid="B30"> 30 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Full <span class="trait">resistance</span> to SCN, based on the combined action of the genes at <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">and</span> one or more additional <italic> Rhg </italic> loci, was manifest as; cell wall appositions to surround the feeding site; failure to supply the feeding site a tracheary element; <span class="trait">and</span> a necrosis as the feeding site develops.
					</p>
				
					<p>&nbsp;&nbsp;
						If the <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg1</span> </italic> was present normal rates of root growth were slightly depressed but the above ground stunting, yellowing <span class="trait">and</span> early senescence did not occur.
					</p>
				
					<p>&nbsp;&nbsp;
						Three recessive loci ( <italic> rhg1 </italic> <italic> rhg3 </italic> ) <span class="trait">and</span> a dominant locus ( <italic> <span class="gene">Rhg4</span> </italic> ) were assigned gene names by parsimony though other dominance models were equally likely.
					</p>
				
					<p>&nbsp;&nbsp;
						The Peking derived <span class="trait">resistance</span> alleles of <italic> rhg1 </italic> <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> were introgressed into the cv.
					</p>
				
					<p>&nbsp;&nbsp;
						However, an <italic> <span class="gene">Rhg1</span> </italic> -like locus was found at other locations in a few SCN resistant PIs, including Lg B1 (chromosome 11) [ <xref ref-type="bibr" rid="B22"> 22 </xref> ], mid LgG [ <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> ] <span class="trait">and</span> Lg B2 (chromosome 14) [ <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The effects of the <italic> <span class="gene">Rhg1</span> </italic> -like loci found at other locations than chromosome 18 on <span class="trait">resistance</span> to SDS were not reported by 2012.
					</p>
				
					<p>&nbsp;&nbsp;
						It has been shown the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> -a <italic> /Rfs2 </italic> (from Peking) was associated with reduced seed <span class="trait">yield</span> when SCN is not present in the fields [ <xref ref-type="bibr" rid="B39"> 39 </xref> - <xref ref-type="bibr" rid="B41"> 41 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The region encoded an <span class="gene">RLK</span> ( <italic> <span class="gene">Glyma18g02680</span> </italic> named <italic> <span class="gene">GmRLK18-1</span> </italic> hereafter), a variant laccase ( <italic> <span class="gene">Glyma18g02690</span> </italic> ) <span class="trait">and</span> a predicted Na/H ion antiporter ( <italic> <span class="gene">Glyma18g0270</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> were expressed in roots, shoots <span class="trait">and</span> flowers but not nodules or seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						Many nucleotide differences were found in the region encompassing the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> genes from fragments of sequences from the resistant allele in Forrest compared to susceptible genotypes &#x02018;Asgrow 3244&#x02019; <span class="trait">and</span> &#x02018;Williams 82&#x02019; [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">GmRLK18-1</span> </italic> was considered the most likely candidate gene underlying <italic> <span class="gene">Rhg1</span> </italic> -a based on fine maps <span class="trait">and</span> association analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> gene at <italic> Rhg1/Rfs2 </italic> had nine alleles recognized among PIs <span class="trait">and</span> cultivars [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> - <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Here, a molecular basis for <span class="trait">resistance</span> to SDS <span class="trait">and</span> SCN was inferred from functional analyses of the <italic> Rhg1/Rfs2 </italic> -a locus in near isogenic lines (NILs) <span class="trait">and</span> the Forrest allele of the <italic> <span class="gene">GmRLK18-1</span> </italic> gene in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> was gene 5, the laccase gene 6 <span class="trait">and</span> the antiporter gene 7.
					</p>
				
					<p>&nbsp;&nbsp;
						Locations of overlapping BAC clones B73p06 <span class="trait">and</span> B21d09 that both encoded the <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus (Lg G; chromosome 18) were shown below the ideogram.
					</p>
				
					<p>&nbsp;&nbsp;
						The marker TMD1 amplified a fragment from <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2/Rhg1 </italic> -a <italic> /Rfs2 </italic> of 303+ 15&#x02009;bp (resistant allele was the smaller) <span class="trait">and</span> of 362&#x02009;bp from a syntenic homeolog of <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2Rfs2/ </italic> found in the sequence of BAC H38f23 from Lg B1 (chromosome 11).
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> D </bold> shows an ideogram of markers, the <italic> <span class="gene">GmRLK18-1</span> </italic> gene <span class="trait">and</span> gene features found in the insert of plasmid pSBHB94 that encompassed from 30,423- 40,194&#x02009;bp of B73p06.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Genes predicted from DNA sequence within the </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> -a/ </bold> <bold> &#x02009; <italic> Rfs2 </italic> &#x02009; </bold> <bold> locus encoded by BAC 73p06 (81, 157&#x02009;bp) from Forrest (resistant) <span class="trait">and</span> sequence contig from A3244 (susceptible; AX196295) </bold>.
					</p>
				
					<p>&nbsp;&nbsp;
						The greater number of differences between alleles of the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> the other candidate genes was taken as evidence this should be the first gene tested in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						In the intergeneic region between <span class="gene">RLK</span> <span class="trait">and</span> laccase were four polymorphic small indels or microsatellite-like polymorphisms.
					</p>
				
					<p>&nbsp;&nbsp;
						Significantly, the sequence of H38f23 contained a complete set of syntenic genes for a second <italic> Rhg1/Rfs2 </italic> locus <span class="trait">and</span> surrounding genes (an RLK, laccase, both antiporters, the kinase <span class="trait">and</span> the helicase; <italic> <span class="gene">Glyma11g35710-</span> <span class="gene">Glyma11g35660</span> </italic> ; Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ; HQ008940).
					</p>
				
					<p>&nbsp;&nbsp;
						Interrogation of Soybase showed the transcript abundance patterns for the syntenic <span class="gene">RLK</span> pair <span class="trait">and</span> laccase pair were both abundant in roots as reported in the RNA sequence atlas generated from a Williams 82 sister line derived NIL (cultivar &#x02018;P-C609-45-2-2&#x02019; a BC5F5 plant derived from <italic> G.
					</p>
				
					<p>&nbsp;&nbsp;
						At the RLKs, <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> , with intron boundaries as described by [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] <span class="trait">and</span> [ <xref ref-type="bibr" rid="B25"> 25 </xref> ], the amino acid identity was 93% overall with 94% in the LRR, 93% in the trans-membrane domain <span class="trait">and</span> 97% in the kinase domain (Additional file <xref ref-type="supplementary-material" rid="S2"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						From sequence comparison <span class="trait">and</span> transcript abundances it appears likely the paralogs <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> would share functions <span class="trait">and</span> should hetero-dimerize [ <xref ref-type="bibr" rid="B47"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In the NIL population <span class="gene">RLK</span> alleles <span class="trait">and</span> HSVs were distinguished using both the SIUC-TMD1 marker (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ) <span class="trait">and</span> SNP Ala87Val from the LRR region by Taqman (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> : Figure S2; Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2).
					</p>
				
					<p>&nbsp;&nbsp;
						In genomic DNA TMD1 distinguished both the Peking <span class="trait">and</span> Peking-like allele of <italic> <span class="gene">GmRLK18-1</span> </italic> from the other 7 alleles <span class="trait">and</span> the homeologous alleles of <italic> <span class="gene">GmRLK11-1</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						One of the 2 amino acid substitutions found in the intracellular kinase of the complete <span class="gene">RLK</span> (Gly539Ala <span class="trait">and</span> Ser770Pro) may be key to transducing intracellular signaling leading to the <span class="trait">resistance</span> response [ <xref ref-type="bibr" rid="B44"> 44 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Expression judged by examination of EST libraries in silico, cDNA libraries by hybridization, RNA sequencing <span class="trait">and</span> mRNA populations assayed by qRT-PCR showed both transcripts encoded by <italic> <span class="gene">GmRLK18-1</span> </italic> -a <span class="trait">and</span> <italic> <span class="gene">GmRLK18-1</span> </italic> -e alleles (Figure&#x02009; <xref ref-type="fig" rid="F3"> 3 </xref> ) were present in both non-infested roots <span class="trait">and</span> SCN-infested roots, as reported in [ <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of Forrest (&#x02212;a) <span class="trait">and</span> Essex (&#x02212;e) alleles of the </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> at </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> </bold> <bold> in non-transgenic soybean NIL roots.
					</p>
				
					<p>&nbsp;&nbsp;
						Such plants had a <span class="trait">resistance</span> allele at a new location <span class="trait">and</span> a susceptibility allele ( <italic> <span class="gene">GmRLK18-1</span> </italic> -a; <italic> rhg1/rfs2 </italic> -e) on chromosome 18.
					</p>
				
					<p>&nbsp;&nbsp;
						In several transgenic lines from (T1-T3) used for SDS <span class="trait">and</span> SCN assays, lines were identified with the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene in the homozygous state.
					</p>
				
					<p>&nbsp;&nbsp;
						For example, in the progeny of fertile T0 line 6B3 the progeny T1 lines like 7D2 <span class="trait">and</span> sixteen T3 derived plants expressed the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene as both mRNA <span class="trait">and</span> protein (Figure&#x02009; <xref ref-type="fig" rid="F4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmRLK1</span> transcripts <span class="trait">and</span> proteins were detected in about 50% of the progenies of the 6B3 primary transformants.
					</p>
				
					<p>&nbsp;&nbsp;
						In selfed lines expressing <italic> <span class="gene">GmRLK18-1</span> </italic> -a the transgene allele was detected as distinct mRNAs <span class="trait">and</span> alloproteins.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Soybean transgenic plants expressed the mRNA <span class="trait">and</span> protein from the Forrest allele of </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> , the <span class="gene">RLK</span> at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						The arrow shows the double band for <span class="gene">Gm18RLK-1-a</span> positive sample at 314&#x02009;bp for lines 1,3&#x02013;5, 7,8,10 <span class="trait">and</span> 12.
					</p>
				
					<p>&nbsp;&nbsp;
						In transgenic soybean plants <italic> <span class="gene">GmRLK18-1</span> </italic> -a allele provided <span class="trait">resistance</span> to both root infection <span class="trait">and</span> root rot by <italic> Fusarium virguliforme.
					</p>
				
					<p>&nbsp;&nbsp;
						Senescence was delayed by 14&#x02009;days in <italic> <span class="gene">GmRLK18-1</span> </italic> transgenics compared to controls <span class="trait">and</span> more pods were set in each of the three repeats of the experiment.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the <italic> <span class="gene">GmRLK18-1</span> </italic> -a provided a very high degree of <span class="trait">resistance</span> to the transgenic plants in roots <span class="trait">and</span> leaves <span class="trait">and</span> pods.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Resistance</span> to </bold> <bold> &#x02009; <italic> Fusarium virguliforme </italic> &#x02009; </bold> <bold> <span class="trait">and</span> partial <span class="trait">resistance</span> to </bold> <bold> &#x02009; <italic> Heterodera glycines </italic> &#x02009; </bold> <bold> caused by the Forrest allele of a receptor like kinase ( </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> -a) found at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus as transgene in primary transgenic lines (cv &#x02018;X5&#x02019;).
					</p>
				
					<p>&nbsp;&nbsp;
						Derived crossed lines (X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35 <span class="trait">and</span> cultivars X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35xEF2) were included in runs 2 <span class="trait">and</span> 3.
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> A </bold> &#x00026; <bold> B </bold> shows stable soybean transgenics with <span class="trait">and</span> without the 10 kbp <italic> <span class="gene">GmRLK18-1</span> </italic> ( <italic> Rhg1/Rfs2) </italic> subclone at 21 dai.
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> E </bold> shows plants at 56 dai where X5 is senescent with abscission of leaflets from erect petioles <span class="trait">and</span> X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a is still green <span class="trait">and</span> filling pods.
					</p>
				
					<p>&nbsp;&nbsp;
						In separate assays with plants from the same lines the <span class="trait">resistance</span> to SCN was partial in the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgenic plants (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> A) judged by female index (FI).
					</p>
				
					<p>&nbsp;&nbsp;
						Partial <span class="trait">resistance</span> was expected as <italic> rhg4 </italic> -e was present <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> -a absent.
					</p>
				
					<p>&nbsp;&nbsp;
						The progeny segregated for the <span class="gene">RLK</span> <span class="trait">and</span> so were <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4:: </italic> RLK18-1 or <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4 </italic> with no transgene.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants with the <span class="gene">RLK</span> had a 20% lower FI <span class="trait">and</span> were almost in the resistant class (FI&#x02009;&#x0003c;&#x02009;10; Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Westag 97 appeared to encode a functional <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Consequently the <span class="gene">RLK</span> transgenic Westag 97 showed nearly complet <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The linked genes may reduce SCN numbers but did require the action of a second locus ( <italic> <span class="gene">Rhg4</span> </italic> ) to provide full <span class="trait">resistance</span> to Hg Type 0 as expected from [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In both SCN greenhouse assay <span class="trait">and</span> field the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a reduction in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						However, in the water saturated assays of SDS <span class="trait">resistance</span> the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused an increase in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ) in the presence of <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						Water sufficiency appeared to make the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> beneficial to growth <span class="trait">and</span> <span class="trait">yield</span> when <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, plants that were not infested in the SDS assays showed the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a decrease in root <span class="trait">and</span> shoot mass.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus was shown to underlie <span class="trait">resistance</span> to root infection by <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						Some <span class="trait">resistance</span> to SCN was found but this was partial as expected in the absence of a <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">RLK</span> simultaneously contributed <span class="trait">resistance</span> to both pathogens, establishing pleiotropy (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis CLAVATA1, the <span class="gene">RLK</span> that regulated meristem development, also had an effect on nematode <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B52"> 52 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> mechanisms to the guilds were known to operate by different pathways but some involved the <italic> <span class="gene">Rhg4</span> </italic> locus or region [ <xref ref-type="bibr" rid="B62"> 62 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, plants transgenic with <span class="gene">Gm18RLK18-1</span> might be altered in their responses to a broad range of <span class="trait">pests</span> <span class="trait">and</span> pathogens.
					</p>
				
					<p>&nbsp;&nbsp;
						A potential suppressor locus acting on the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> was identified earlier [ <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Plant transgenic with the <span class="gene">RLK</span> were fertile <span class="trait">and</span> produced seed <span class="trait">and</span> they would lack the <span class="trait">resistance</span> allele of the gene on chromosome 7 (Lg M).
					</p>
				
					<p>&nbsp;&nbsp;
						The hypothetical anti-porter protein was proposed as the target locus kept in the resistant state when the <span class="gene">RLK</span> is conferring SCN <span class="trait">resistance</span> (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The linkage disequilibrium studies reported with probes in, <span class="trait">and</span> around, the <span class="gene">RLK</span> inferred a larger more complex structure for <italic> <span class="gene">Rhg1</span> </italic> than simply the <span class="gene">RLK</span> polymorphisms [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B64"> 64 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">GmRLK18-1</span> </italic> alone at a new genomic locations in transgenic plants was sufficient for <span class="trait">resistance</span> to a relatively new pathogen of soybean <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						In vitro, the purified <italic> <span class="gene">GmRLK18-1</span> </italic> LRR domain has been shown to bind strongly to the CLE peptide found in nematode secretions as well as two proteins <span class="trait">and</span> two CLE peptide of plant origin involved in tracheary element inhibition [ <xref ref-type="bibr" rid="B43"> 43 </xref> , <xref ref-type="bibr" rid="B66"> 66 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The biochemical analysis of the <span class="gene">RLK</span> protein will lead to refinements of models for modes of action for the pleiotropic <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Stable <span class="trait">resistance</span> to many pathogens might be achieved by manipulation of the genes encoding a small number of pathogen recognition proteins in the <span class="gene">RLK</span> family.
					</p>
				
					<p>&nbsp;&nbsp;
						Genotypes were <italic> rhg1-e/rfs2, rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;3 <span class="trait">and</span> <italic> Rhg1-a/Rfs2, Rhg1-a/Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;23 whereas NIL34-33 contained both those <span class="trait">and</span> <italic> rhg1 </italic> -e <italic> /rfs2, Rhg1- </italic> a <italic> /Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> in different plants (NIL34-33-1 to &#x02212;34).
					</p>
				
					<p>&nbsp;&nbsp;
						X5 was judged to be <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, rhg4, rhg4 </italic> <span class="trait">and</span> Westag 97 was judged <italic> rhg1-e/rfs2rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> <span class="trait">and</span> based on the assays reported here.
					</p>
				
					<p>&nbsp;&nbsp;
						Presence of the <italic> Rhg1/Rfs2 </italic> <span class="trait">resistance</span> alleles was confirmed by PCR analysis using TMD1 an indel marker in the <span class="gene">RLK</span> intron.
					</p>
				
					<p>&nbsp;&nbsp;
						Three probes were designed for the SNPs at 1,486&#x02009;bp, 506&#x02009;bp <span class="trait">and</span> 2,040&#x02009;bp (relative to the translation start site) to distinguish the 8 commonest alleles of the <span class="gene">RLK</span> (Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2).
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> Purified stable transgenics were of genotype <italic> rhg1 </italic> -e <italic> /rfs2rhg1 </italic> -e <italic> /rfs2, rhg4rhg4:: </italic> <span class="gene">GmRLK18-1,</span> <span class="gene">GmRLK18-1</span> (so <italic> Rhg1- </italic> a <italic> /Rfs2Rhg1- </italic> a <italic> /Rfs2 </italic> ) as shown by markers TMD1 <span class="trait">and</span> A2D8 <italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						TMD1 marker, FJ520231; Corrected <span class="gene">RLK</span> at Rhg1/Rfs2 gene AF506516 <span class="trait">and</span> mRNA AF506517; SIUC-Satt122, bankit1155667; BAC pB73P06 complete sequence JN597009; pSBHB94 complete sequence HQ008939; BAC pH38f23 complete sequence TBD.
					</p>
				
					<p>&nbsp;&nbsp;
						Panel F shows ideograms of the genes predicted from the genome sequences centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> Panel F shows an alignment of the genome sequences of 70kbp centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> showing the extent of synteny.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparisons of sequence identity between Forrest alleles of <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> the most similar <span class="trait">and</span> syntenic <span class="gene">RLK</span> like protein.
					</p>
				
					<p>&nbsp;&nbsp;
						Detection of the SNP polymorphism at position 1486 in the LRR region of </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> &#x02013;a <span class="trait">and</span> &#x02013;e using an allelic discriminatory assay.
					</p>
				
					<p>&nbsp;&nbsp;
						This study examined the role of a receptor like kinase (RLK) <italic> <span class="gene">GmRLK18-1</span> </italic> (gene model <italic> Glyma_18_02680 </italic> at 1,071&#x02009;kbp on chromosome 18 of the genome sequence) within the <italic> Rhg1/Rfs2 </italic> locus in causing <span class="trait">resistance</span> to SCN <span class="trait">and</span> SDS
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59&#x02009;kbp at the center of which the <italic> <span class="gene">GmRLK18-1</span> </italic> was the most polymorphic gene <span class="trait">and</span> encoded protein
					</p>
				
					<p>&nbsp;&nbsp;
						Analyses were made of plants that were either heterozygous at, or transgenic <span class="trait">(and</span> so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <italic> <span class="gene">GmRLK18-1</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						In the absence of <italic> <span class="gene">Rhg4</span> </italic> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root <span class="trait">and</span> leaf symptoms of SDS caused by <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <italic> <span class="gene">GmRLK18-1</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Equally the <italic> <span class="gene">Rhg1</span> </italic> locus underlay partial <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">Rhg1</span> </italic> locus was constant, being required for partial <span class="trait">resistance</span> to all Hg Types in most PIs <span class="trait">and</span> cultivars
					</p>
				
					<p>&nbsp;&nbsp;
						Full <span class="trait">resistance</span> to SCN required 1&#x02013;4 loci in addition to <italic> <span class="gene">Rhg1</span> </italic> , the number depending on the nature of the cyst population parasitizing the roots [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">resistance</span> or susceptible interaction(s) between the nematode <span class="trait">and</span> soybean affected by <italic> <span class="gene">Rhg1</span> </italic> was not induced until females stopped moving through the roots <span class="trait">and</span> established a feeding site comprising several giant cells [ <xref ref-type="bibr" rid="B28"> 28 </xref> - <xref ref-type="bibr" rid="B30"> 30 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Full <span class="trait">resistance</span> to SCN, based on the combined action of the genes at <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">and</span> one or more additional <italic> Rhg </italic> loci, was manifest as; cell wall appositions to surround the feeding site; failure to supply the feeding site a tracheary element; <span class="trait">and</span> a necrosis as the feeding site develops
					</p>
				
					<p>&nbsp;&nbsp;
						If the <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg1</span> </italic> was present normal rates of root growth were slightly depressed but the above ground stunting, yellowing <span class="trait">and</span> early senescence did not occur
					</p>
				
					<p>&nbsp;&nbsp;
						Three recessive loci ( <italic> rhg1 </italic> <italic> rhg3 </italic> ) <span class="trait">and</span> a dominant locus ( <italic> <span class="gene">Rhg4</span> </italic> ) were assigned gene names by parsimony though other dominance models were equally likely
					</p>
				
					<p>&nbsp;&nbsp;
						The Peking derived <span class="trait">resistance</span> alleles of <italic> rhg1 </italic> <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> were introgressed into the cv
					</p>
				
					<p>&nbsp;&nbsp;
						However, an <italic> <span class="gene">Rhg1</span> </italic> -like locus was found at other locations in a few SCN resistant PIs, including Lg B1 (chromosome 11) [ <xref ref-type="bibr" rid="B22"> 22 </xref> ], mid LgG [ <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> ] <span class="trait">and</span> Lg B2 (chromosome 14) [ <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The effects of the <italic> <span class="gene">Rhg1</span> </italic> -like loci found at other locations than chromosome 18 on <span class="trait">resistance</span> to SDS were not reported by 2012
					</p>
				
					<p>&nbsp;&nbsp;
						It has been shown the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> -a <italic> /Rfs2 </italic> (from Peking) was associated with reduced seed <span class="trait">yield</span> when SCN is not present in the fields [ <xref ref-type="bibr" rid="B39"> 39 </xref> - <xref ref-type="bibr" rid="B41"> 41 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The region encoded an <span class="gene">RLK</span> ( <italic> <span class="gene">Glyma18g02680</span> </italic> named <italic> <span class="gene">GmRLK18-1</span> </italic> hereafter), a variant laccase ( <italic> <span class="gene">Glyma18g02690</span> </italic> ) <span class="trait">and</span> a predicted Na/H ion antiporter ( <italic> <span class="gene">Glyma18g0270</span> </italic> )
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> were expressed in roots, shoots <span class="trait">and</span> flowers but not nodules or seeds
					</p>
				
					<p>&nbsp;&nbsp;
						Many nucleotide differences were found in the region encompassing the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">Gmlaccase18-1</span> </italic> genes from fragments of sequences from the resistant allele in Forrest compared to susceptible genotypes &#x02018;Asgrow 3244&#x02019; <span class="trait">and</span> &#x02018;Williams 82&#x02019; [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">GmRLK18-1</span> </italic> was considered the most likely candidate gene underlying <italic> <span class="gene">Rhg1</span> </italic> -a based on fine maps <span class="trait">and</span> association analyses
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> gene at <italic> Rhg1/Rfs2 </italic> had nine alleles recognized among PIs <span class="trait">and</span> cultivars [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> - <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Here, a molecular basis for <span class="trait">resistance</span> to SDS <span class="trait">and</span> SCN was inferred from functional analyses of the <italic> Rhg1/Rfs2 </italic> -a locus in near isogenic lines (NILs) <span class="trait">and</span> the Forrest allele of the <italic> <span class="gene">GmRLK18-1</span> </italic> gene in transgenic plants
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> was gene 5, the laccase gene 6 <span class="trait">and</span> the antiporter gene 7
					</p>
				
					<p>&nbsp;&nbsp;
						Locations of overlapping BAC clones B73p06 <span class="trait">and</span> B21d09 that both encoded the <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus (Lg G; chromosome 18) were shown below the ideogram
					</p>
				
					<p>&nbsp;&nbsp;
						The marker TMD1 amplified a fragment from <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2/Rhg1 </italic> -a <italic> /Rfs2 </italic> of 303+ 15&#x02009;bp (resistant allele was the smaller) <span class="trait">and</span> of 362&#x02009;bp from a syntenic homeolog of <italic> <span class="gene">Rhg1</span> </italic> -a / <italic> Rfs2Rfs2/ </italic> found in the sequence of BAC H38f23 from Lg B1 (chromosome 11)
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> D </bold> shows an ideogram of markers, the <italic> <span class="gene">GmRLK18-1</span> </italic> gene <span class="trait">and</span> gene features found in the insert of plasmid pSBHB94 that encompassed from 30,423- 40,194&#x02009;bp of B73p06
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Genes predicted from DNA sequence within the </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> -a/ </bold> <bold> &#x02009; <italic> Rfs2 </italic> &#x02009; </bold> <bold> locus encoded by BAC 73p06 (81, 157&#x02009;bp) from Forrest (resistant) <span class="trait">and</span> sequence contig from A3244 (susceptible; AX196295) </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						The greater number of differences between alleles of the <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> the other candidate genes was taken as evidence this should be the first gene tested in transgenic plants
					</p>
				
					<p>&nbsp;&nbsp;
						In the intergeneic region between <span class="gene">RLK</span> <span class="trait">and</span> laccase were four polymorphic small indels or microsatellite-like polymorphisms
					</p>
				
					<p>&nbsp;&nbsp;
						Significantly, the sequence of H38f23 contained a complete set of syntenic genes for a second <italic> Rhg1/Rfs2 </italic> locus <span class="trait">and</span> surrounding genes (an RLK, laccase, both antiporters, the kinase <span class="trait">and</span> the helicase; <italic> <span class="gene">Glyma11g35710-</span> <span class="gene">Glyma11g35660</span> </italic> ; Figure&#x02009; <xref ref-type="fig" rid="F1"> 1 </xref> ; HQ008940)
					</p>
				
					<p>&nbsp;&nbsp;
						Interrogation of Soybase showed the transcript abundance patterns for the syntenic <span class="gene">RLK</span> pair <span class="trait">and</span> laccase pair were both abundant in roots as reported in the RNA sequence atlas generated from a Williams 82 sister line derived NIL (cultivar &#x02018;P-C609-45-2-2&#x02019; a BC5F5 plant derived from <italic> G
					</p>
				
					<p>&nbsp;&nbsp;
						At the RLKs, <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> , with intron boundaries as described by [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] <span class="trait">and</span> [ <xref ref-type="bibr" rid="B25"> 25 </xref> ], the amino acid identity was 93% overall with 94% in the LRR, 93% in the trans-membrane domain <span class="trait">and</span> 97% in the kinase domain (Additional file <xref ref-type="supplementary-material" rid="S2"> 1 </xref> : Table S1)
					</p>
				
					<p>&nbsp;&nbsp;
						From sequence comparison <span class="trait">and</span> transcript abundances it appears likely the paralogs <italic> <span class="gene">GmRLK18-1</span> </italic> <span class="trait">and</span> <italic> <span class="gene">GmRLK11-1</span> </italic> would share functions <span class="trait">and</span> should hetero-dimerize [ <xref ref-type="bibr" rid="B47"> 47 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In the NIL population <span class="gene">RLK</span> alleles <span class="trait">and</span> HSVs were distinguished using both the SIUC-TMD1 marker (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ) <span class="trait">and</span> SNP Ala87Val from the LRR region by Taqman (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> : Figure S2; Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2)
					</p>
				
					<p>&nbsp;&nbsp;
						In genomic DNA TMD1 distinguished both the Peking <span class="trait">and</span> Peking-like allele of <italic> <span class="gene">GmRLK18-1</span> </italic> from the other 7 alleles <span class="trait">and</span> the homeologous alleles of <italic> <span class="gene">GmRLK11-1</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						One of the 2 amino acid substitutions found in the intracellular kinase of the complete <span class="gene">RLK</span> (Gly539Ala <span class="trait">and</span> Ser770Pro) may be key to transducing intracellular signaling leading to the <span class="trait">resistance</span> response [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Expression judged by examination of EST libraries in silico, cDNA libraries by hybridization, RNA sequencing <span class="trait">and</span> mRNA populations assayed by qRT-PCR showed both transcripts encoded by <italic> <span class="gene">GmRLK18-1</span> </italic> -a <span class="trait">and</span> <italic> <span class="gene">GmRLK18-1</span> </italic> -e alleles (Figure&#x02009; <xref ref-type="fig" rid="F3"> 3 </xref> ) were present in both non-infested roots <span class="trait">and</span> SCN-infested roots, as reported in [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of Forrest (&#x02212;a) <span class="trait">and</span> Essex (&#x02212;e) alleles of the </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> at </bold> <bold> &#x02009; <italic> Rhg1/Rfs2 </italic> </bold> <bold> in non-transgenic soybean NIL roots
					</p>
				
					<p>&nbsp;&nbsp;
						Such plants had a <span class="trait">resistance</span> allele at a new location <span class="trait">and</span> a susceptibility allele ( <italic> <span class="gene">GmRLK18-1</span> </italic> -a; <italic> rhg1/rfs2 </italic> -e) on chromosome 18
					</p>
				
					<p>&nbsp;&nbsp;
						In several transgenic lines from (T1-T3) used for SDS <span class="trait">and</span> SCN assays, lines were identified with the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene in the homozygous state
					</p>
				
					<p>&nbsp;&nbsp;
						For example, in the progeny of fertile T0 line 6B3 the progeny T1 lines like 7D2 <span class="trait">and</span> sixteen T3 derived plants expressed the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgene as both mRNA <span class="trait">and</span> protein (Figure&#x02009; <xref ref-type="fig" rid="F4"> 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmRLK1</span> transcripts <span class="trait">and</span> proteins were detected in about 50% of the progenies of the 6B3 primary transformants
					</p>
				
					<p>&nbsp;&nbsp;
						In selfed lines expressing <italic> <span class="gene">GmRLK18-1</span> </italic> -a the transgene allele was detected as distinct mRNAs <span class="trait">and</span> alloproteins
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Soybean transgenic plants expressed the mRNA <span class="trait">and</span> protein from the Forrest allele of </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> , the <span class="gene">RLK</span> at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus
					</p>
				
					<p>&nbsp;&nbsp;
						The arrow shows the double band for <span class="gene">Gm18RLK-1-a</span> positive sample at 314&#x02009;bp for lines 1,3&#x02013;5, 7,8,10 <span class="trait">and</span> 12
					</p>
				
					<p>&nbsp;&nbsp;
						In transgenic soybean plants <italic> <span class="gene">GmRLK18-1</span> </italic> -a allele provided <span class="trait">resistance</span> to both root infection <span class="trait">and</span> root rot by <italic> Fusarium virguliforme
					</p>
				
					<p>&nbsp;&nbsp;
						Senescence was delayed by 14&#x02009;days in <italic> <span class="gene">GmRLK18-1</span> </italic> transgenics compared to controls <span class="trait">and</span> more pods were set in each of the three repeats of the experiment
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the <italic> <span class="gene">GmRLK18-1</span> </italic> -a provided a very high degree of <span class="trait">resistance</span> to the transgenic plants in roots <span class="trait">and</span> leaves <span class="trait">and</span> pods
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Resistance</span> to </bold> <bold> &#x02009; <italic> Fusarium virguliforme </italic> &#x02009; </bold> <bold> <span class="trait">and</span> partial <span class="trait">resistance</span> to </bold> <bold> &#x02009; <italic> Heterodera glycines </italic> &#x02009; </bold> <bold> caused by the Forrest allele of a receptor like kinase ( </bold> <bold> &#x02009; <italic> <span class="gene">GmRLK18-1</span> </italic> &#x02009; </bold> <bold> -a) found at the </bold> <bold> &#x02009; <italic> Rfs2/Rhg1 </italic> &#x02009; </bold> <bold> locus as transgene in primary transgenic lines (cv &#x02018;X5&#x02019;)
					</p>
				
					<p>&nbsp;&nbsp;
						Derived crossed lines (X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35 <span class="trait">and</span> cultivars X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a x WENIL35xEF2) were included in runs 2 <span class="trait">and</span> 3
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> A </bold> &#x00026; <bold> B </bold> shows stable soybean transgenics with <span class="trait">and</span> without the 10 kbp <italic> <span class="gene">GmRLK18-1</span> </italic> ( <italic> Rhg1/Rfs2) </italic> subclone at 21 dai
					</p>
				
					<p>&nbsp;&nbsp;
						Panel <bold> E </bold> shows plants at 56 dai where X5 is senescent with abscission of leaflets from erect petioles <span class="trait">and</span> X5:: <italic> <span class="gene">GmRLK18-1</span> </italic> -a is still green <span class="trait">and</span> filling pods
					</p>
				
					<p>&nbsp;&nbsp;
						In separate assays with plants from the same lines the <span class="trait">resistance</span> to SCN was partial in the <italic> <span class="gene">GmRLK18-1</span> </italic> -a transgenic plants (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> A) judged by female index (FI)
					</p>
				
					<p>&nbsp;&nbsp;
						Partial <span class="trait">resistance</span> was expected as <italic> rhg4 </italic> -e was present <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> -a absent
					</p>
				
					<p>&nbsp;&nbsp;
						The progeny segregated for the <span class="gene">RLK</span> <span class="trait">and</span> so were <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4:: </italic> RLK18-1 or <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, Rhg4, rhg4 </italic> with no transgene
					</p>
				
					<p>&nbsp;&nbsp;
						Plants with the <span class="gene">RLK</span> had a 20% lower FI <span class="trait">and</span> were almost in the resistant class (FI&#x02009;&#x0003c;&#x02009;10; Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Westag 97 appeared to encode a functional <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele
					</p>
				
					<p>&nbsp;&nbsp;
						Consequently the <span class="gene">RLK</span> transgenic Westag 97 showed nearly complet <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The linked genes may reduce SCN numbers but did require the action of a second locus ( <italic> <span class="gene">Rhg4</span> </italic> ) to provide full <span class="trait">resistance</span> to Hg Type 0 as expected from [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In both SCN greenhouse assay <span class="trait">and</span> field the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a reduction in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						However, in the water saturated assays of SDS <span class="trait">resistance</span> the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused an increase in root <span class="trait">and</span> shoot mass (Table&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> ) in the presence of <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						Water sufficiency appeared to make the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> beneficial to growth <span class="trait">and</span> <span class="trait">yield</span> when <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, plants that were not infested in the SDS assays showed the Forrest allele of <italic> <span class="gene">GmRLK18-1</span> </italic> caused a decrease in root <span class="trait">and</span> shoot mass
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRLK18-1</span> </italic> at the <italic> Rhg1/Rfs2 </italic> locus was shown to underlie <span class="trait">resistance</span> to root infection by <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						Some <span class="trait">resistance</span> to SCN was found but this was partial as expected in the absence of a <span class="trait">resistance</span> allele at <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">RLK</span> simultaneously contributed <span class="trait">resistance</span> to both pathogens, establishing pleiotropy (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis CLAVATA1, the <span class="gene">RLK</span> that regulated meristem development, also had an effect on nematode <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B52"> 52 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> mechanisms to the guilds were known to operate by different pathways but some involved the <italic> <span class="gene">Rhg4</span> </italic> locus or region [ <xref ref-type="bibr" rid="B62"> 62 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, plants transgenic with <span class="gene">Gm18RLK18-1</span> might be altered in their responses to a broad range of <span class="trait">pests</span> <span class="trait">and</span> pathogens
					</p>
				
					<p>&nbsp;&nbsp;
						A potential suppressor locus acting on the <span class="trait">resistance</span> allele of <italic> <span class="gene">Rhg1</span> </italic> was identified earlier [ <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Plant transgenic with the <span class="gene">RLK</span> were fertile <span class="trait">and</span> produced seed <span class="trait">and</span> they would lack the <span class="trait">resistance</span> allele of the gene on chromosome 7 (Lg M)
					</p>
				
					<p>&nbsp;&nbsp;
						The hypothetical anti-porter protein was proposed as the target locus kept in the resistant state when the <span class="gene">RLK</span> is conferring SCN <span class="trait">resistance</span> (Figure&#x02009; <xref ref-type="fig" rid="F6"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The linkage disequilibrium studies reported with probes in, <span class="trait">and</span> around, the <span class="gene">RLK</span> inferred a larger more complex structure for <italic> <span class="gene">Rhg1</span> </italic> than simply the <span class="gene">RLK</span> polymorphisms [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B64"> 64 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> <span class="gene">GmRLK18-1</span> </italic> alone at a new genomic locations in transgenic plants was sufficient for <span class="trait">resistance</span> to a relatively new pathogen of soybean <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						In vitro, the purified <italic> <span class="gene">GmRLK18-1</span> </italic> LRR domain has been shown to bind strongly to the CLE peptide found in nematode secretions as well as two proteins <span class="trait">and</span> two CLE peptide of plant origin involved in tracheary element inhibition [ <xref ref-type="bibr" rid="B43"> 43 </xref> , <xref ref-type="bibr" rid="B66"> 66 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The biochemical analysis of the <span class="gene">RLK</span> protein will lead to refinements of models for modes of action for the pleiotropic <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Stable <span class="trait">resistance</span> to many pathogens might be achieved by manipulation of the genes encoding a small number of pathogen recognition proteins in the <span class="gene">RLK</span> family
					</p>
				
					<p>&nbsp;&nbsp;
						Genotypes were <italic> rhg1-e/rfs2, rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;3 <span class="trait">and</span> <italic> Rhg1-a/Rfs2, Rhg1-a/Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> for NIL 34&#x02013;23 whereas NIL34-33 contained both those <span class="trait">and</span> <italic> rhg1 </italic> -e <italic> /rfs2, Rhg1- </italic> a <italic> /Rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> in different plants (NIL34-33-1 to &#x02212;34)
					</p>
				
					<p>&nbsp;&nbsp;
						X5 was judged to be <italic> rhg1 </italic> -e <italic> /rfs2, rhg1 </italic> -e <italic> /rfs2, rhg4, rhg4 </italic> <span class="trait">and</span> Westag 97 was judged <italic> rhg1-e/rfs2rhg1-e/rfs2, Rhg4, <span class="gene">Rhg4</span> </italic> <span class="trait">and</span> based on the assays reported here
					</p>
				
					<p>&nbsp;&nbsp;
						Presence of the <italic> Rhg1/Rfs2 </italic> <span class="trait">resistance</span> alleles was confirmed by PCR analysis using TMD1 an indel marker in the <span class="gene">RLK</span> intron
					</p>
				
					<p>&nbsp;&nbsp;
						Three probes were designed for the SNPs at 1,486&#x02009;bp, 506&#x02009;bp <span class="trait">and</span> 2,040&#x02009;bp (relative to the translation start site) to distinguish the 8 commonest alleles of the <span class="gene">RLK</span> (Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Table S2)
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> Purified stable transgenics were of genotype <italic> rhg1 </italic> -e <italic> /rfs2rhg1 </italic> -e <italic> /rfs2, rhg4rhg4:: </italic> <span class="gene">GmRLK18-1,</span> <span class="gene">GmRLK18-1</span> (so <italic> Rhg1- </italic> a <italic> /Rfs2Rhg1- </italic> a <italic> /Rfs2 </italic> ) as shown by markers TMD1 <span class="trait">and</span> A2D8 <italic>
					</p>
				
					<p>&nbsp;&nbsp;
						TMD1 marker, FJ520231; Corrected <span class="gene">RLK</span> at Rhg1/Rfs2 gene AF506516 <span class="trait">and</span> mRNA AF506517; SIUC-Satt122, bankit1155667; BAC pB73P06 complete sequence JN597009; pSBHB94 complete sequence HQ008939; BAC pH38f23 complete sequence TBD
					</p>
				
					<p>&nbsp;&nbsp;
						Panel F shows ideograms of the genes predicted from the genome sequences centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> Panel F shows an alignment of the genome sequences of 70kbp centered on <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> showing the extent of synteny
					</p>
				
					<p>&nbsp;&nbsp;
						Comparisons of sequence identity between Forrest alleles of <span class="gene">GmRLK18-1</span> <span class="trait">and</span> <span class="gene">GmRLK11-1</span> the most similar <span class="trait">and</span> syntenic <span class="gene">RLK</span> like protein
					</p>
				
					<p>&nbsp;&nbsp;
						Detection of the SNP polymorphism at position 1486 in the LRR region of </bold> <bold> &#x02009; <italic> <span class="gene">Rhg1</span> </italic> &#x02009; </bold> <bold> &#x02013;a <span class="trait">and</span> &#x02013;e using an allelic discriminatory assay
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of polymorphism <span class="trait">and</span> transcription of the effector gene <span class="gene">Avr1b</span> in Phytophthora sojae isolates from China virulent to Rps1b.</h2>
			<h3>PMID:21726400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cui L(1), Yin W, Dong S, Wang Y.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effector gene <span class="gene">Avr1b-1</span> of Phytophthora sojae determines the efficacy of the <span class="trait">resistance</span> gene Rps1b in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Four different alleles <span class="trait">and</span> a complete deletion mutation of the <span class="gene">Avr1b-1</span> gene were identified.
					</p>
				
					<p>&nbsp;&nbsp;
						Molecular analysis of the deletion breakpoints in the <span class="gene">Avr1b-1</span> locus revealed that an 8-kb DNA sequence containing <span class="gene">Avr1b-1</span> was deleted <span class="trait">and</span> a 12.
					</p>
				
					<p>&nbsp;&nbsp;
						A truncated transposase gene was found <span class="trait">and</span> five transposable elements were predicted in the inserted sequence, suggesting that the deletion of <span class="gene">Avr1b-1</span> might be attributed to transposon movement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Transcriptome Analysis of Two Races of <italic> Heterodera glycines </italic> at Different Developmental Stages</h2>
			<h3>PMID:24662955</h3>
				

				
					<h3>PMCID:3963861</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Gaofeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Deliang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Bingli </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Wenkun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Lingan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Long </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Huan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jian </surname>  <given-names> Heng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, repeated growth of potato cultivars carrying the <span class="gene">H1</span> <span class="trait">resistance</span> gene against <italic> Globodera rostochiensis </italic> has led to selection for <italic> G
					</p>
				
					<p>&nbsp;&nbsp;
						The primers used for <span class="gene">GAPDH</span> amplification <span class="trait">and</span> gene-specific primers used for qRT-PCR are listed in <xref ref-type="supplementary-material" rid="pone
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The primers used for <span class="gene">GAPDH</span> amplification <span class="trait">and</span> gene-specific primers used for qRT-PCR are listed in <xref ref-type="supplementary-material" rid="pone.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A <italic> Phytophthora sojae </italic> effector suppresses endoplasmic reticulum stress-mediated immunity by stabilizing plant Binding immunoglobulin Proteins</h2>
			<h3>PMID:27256489</h3>
				

				
					<h3>PMCID:4895818</h3>
					<h4>期刊:Nature Communications</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jing </surname>  <given-names> Maofeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Baodian </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Haonan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Guanghui </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Huawei </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Wenwu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-7347-8935 </contrib-id>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Suomeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiao </surname>  <given-names> Yongli </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tyler </surname>  <given-names> Brett M. </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Wenbo </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  <xref ref-type="aff" rid="a6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yuanchao </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-5803-5343 </contrib-id>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Plant Pathology, Nanjing Agricultural University </institution> , 210095 Nanjing,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Key Laboratory of Integrated Management of Crop Diseases and Pests (Ministry of Education) </institution> , 210095 Nanjing,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Institute of Crop Science, Chinese Academy of Agricultural Sciences </institution> , 100081 Beijing,  <country> China </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> Center for Genome Research and Biocomputing and Department of Botany and Plant Pathology, Oregon State University </institution> , Corvallis, Oregon 97331,  <country> USA </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Department of Plant Pathology and Microbiology, University of California </institution> , Riverside, California 92521,  <country> USA </country>  </aff>  <aff id="a6">  <label> 6 </label>  <institution> Center for Plant Cell Biology, University of California </institution> , Riverside, California 92521,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 04 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, overexpression of <italic> <span class="gene">GmBiP4</span> </italic> in soybean ( <italic> Glycine max </italic> ) inhibited the UPR <span class="trait">and</span> increased tolerance to water loss <xref ref-type="bibr" rid="b25"> 25 </xref> , <span class="trait">and</span> suppressed cell death associated with senescence <span class="trait">and</span> osmotic stress <xref ref-type="bibr" rid="b26"> 26 </xref> , but at the same time intensified cell death <span class="trait">and</span> induction of SA-mediated responses triggered by avirulent <italic> Pseudomonas </italic> species <xref ref-type="bibr" rid="b26"> 26 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean <italic> BiP </italic> gene family consists of at least four members, <italic> <span class="gene">GmBiP1</span> </italic> ( <italic> <span class="gene">Glyma08g02940</span> </italic> ), <italic> <span class="gene">GmBiP2</span> </italic> ( <italic> <span class="gene">Glyma08g02960</span> </italic> ), <italic> <span class="gene">GmBiP3</span> </italic> ( <italic> <span class="gene">Glyma05g36600</span> </italic> ) <span class="trait">and</span> <italic> <span class="gene">GmBiP4</span> </italic> ( <italic> <span class="gene">Glyma05g36620</span> </italic> ), all of which are induced upon ER stress <xref ref-type="bibr" rid="b32"> 32 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the association of PsAvh262 with NbBiP5 <span class="trait">and</span> <span class="gene">GmBiPs,</span> we carried out Co-IP assays ( <xref ref-type="fig" rid="f3"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						The interaction between PsAvh262-M1 <span class="trait">and</span> <span class="gene">GmBiP1</span> was much weaker than the interaction with WT PsAvh262 ( <xref ref-type="fig" rid="f3"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						PsAvh262 (PsAvh262-YFPN, with or without its signal peptide) <span class="trait">and</span> <span class="gene">GmBiP1</span> (SP-YFPC-GmBiP1-HDEL) were co-expressed in <italic> N
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, our results suggest that the <span class="gene">GmBiP1-PsAvh262</span> interaction occurs in the ER, though we cannot rule out that the interaction (also) occurs in the cytoplasm <span class="trait">and</span> that the complex subsequent can relocate to the ER
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the localization patterns of <span class="gene">GmBiP1</span> <span class="trait">and</span> PsAvh262 during <italic> Phytophthora </italic> infection, we inoculated <italic> N
					</p>
				
					<p>&nbsp;&nbsp;
						The results showed that both PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> localized to the perihaustorial region, though the distribution of <span class="gene">GmBiP1</span> appeared biased towards the necks of the haustoria, while PsAvh262 appeared biased towards the tips ( <xref ref-type="fig" rid="f5"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, these results suggest that PsAvh262 <span class="trait">and</span> its target <span class="gene">GmBiP1</span> co-localize around the haustoria <span class="trait">and</span> so may possibly influence infection
					</p>
				
					<p>&nbsp;&nbsp;
						To test this hypothesis, we co-expressed NbBiP5 <span class="trait">and</span> <span class="gene">GmBiP1</span> with wild-type PsAvh262 or the PsAvh262 mutants in <italic> N
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, MG132 treatment did not further increase the levels of <span class="gene">GmBiP1</span> produced by Avh262 expression, suggesting that Avh262 <span class="trait">and</span> MG132 act via the same process ( <xref ref-type="fig" rid="f7"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						To test this, we transiently expressed <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5 (with GFP fused to the N terminus of the signal peptides) in <italic> N
					</p>
				
					<p>&nbsp;&nbsp;
						3 </xref> <span class="trait">and</span> <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 1c </xref> ; <xref ref-type="supplementary-material" rid="S1"> 6 </xref> ), we also examined the influence of NbBiP5, <span class="gene">GmBiP1</span> <span class="trait">and</span> <span class="gene">GmBiP4</span> on BAX-triggered cell death
					</p>
				
					<p>&nbsp;&nbsp;
						benthamiana </italic> <span class="trait">and</span> rice homologues of <span class="gene">GmBiPs</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The sequences data (PsAvh262, <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5) that support the finding of this study have been deposited in GenBank ( <ext-link xmlns:xlink="http://www
					</p>
				
					<p>&nbsp;&nbsp;
						1 (PsAvh262, GenBank); <span class="gene">Glyma08g02940</span> <span class="gene">(GmBiP1),</span> <span class="gene">Glyma08g02960</span> <span class="gene">(GmBiP2),</span> <span class="gene">Glyma05g36600</span> <span class="gene">(GmBiP3)</span> <span class="trait">and</span> <span class="gene">Glyma05g36620</span> <span class="gene">(GmBiP4)</span> in Phytozome; NbS00040865g0006
					</p>
				
					<p>&nbsp;&nbsp;
						Protein complexes were pulled down using anti-Flag agarose gel, <span class="trait">and</span> co-precipitations of PsAvh262 <span class="trait">and</span> its mutants with <span class="gene">GmBiP1</span> were detected using immunoblots
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Localization of RFP- <span class="trait">and</span> GFP-fusions of PsAvh262 <span class="trait">and</span> mutants in the absence of <span class="gene">GmBiP1</span>
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> f </bold> , <bold> g </bold> ) Co-localization of PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> detected by bimolecular fluorescence complementation (BiFC) using the constructs shown in <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmBiP1</span> </italic> ( <bold> b </bold> ), <italic> Protein disulfide isomerase </italic> ( <italic> PDI </italic> ) ( <bold> c </bold> ) <span class="trait">and</span> <italic> vacuolar processing enzyme </italic> ( <italic> VPE </italic> ) ( <bold> d </bold> ) were determined by qRT&#x02013;PCR in soybeans inoculated with wild-type or <italic> PsAvh262 </italic> -silenced mutants of <italic> P
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The soybean <italic> BiP </italic> gene family consists of at least four members, <italic> <span class="gene">GmBiP1</span> </italic> ( <italic> <span class="gene">Glyma08g02940</span> </italic> ), <italic> <span class="gene">GmBiP2</span> </italic> ( <italic> <span class="gene">Glyma08g02960</span> </italic> ), <italic> <span class="gene">GmBiP3</span> </italic> ( <italic> <span class="gene">Glyma05g36600</span> </italic> ) <span class="trait">and</span> <italic> <span class="gene">GmBiP4</span> </italic> ( <italic> <span class="gene">Glyma05g36620</span> </italic> ), all of which are induced upon ER stress <xref ref-type="bibr" rid="b32"> 32 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the association of PsAvh262 with NbBiP5 <span class="trait">and</span> <span class="gene">GmBiPs,</span> we carried out Co-IP assays ( <xref ref-type="fig" rid="f3"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						The interaction between PsAvh262-M1 <span class="trait">and</span> <span class="gene">GmBiP1</span> was much weaker than the interaction with WT PsAvh262 ( <xref ref-type="fig" rid="f3"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						PsAvh262 (PsAvh262-YFPN, with or without its signal peptide) <span class="trait">and</span> <span class="gene">GmBiP1</span> (SP-YFPC-GmBiP1-HDEL) were co-expressed in <italic> N.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, our results suggest that the <span class="gene">GmBiP1-PsAvh262</span> interaction occurs in the ER, though we cannot rule out that the interaction (also) occurs in the cytoplasm <span class="trait">and</span> that the complex subsequent can relocate to the ER.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the localization patterns of <span class="gene">GmBiP1</span> <span class="trait">and</span> PsAvh262 during <italic> Phytophthora </italic> infection, we inoculated <italic> N.
					</p>
				
					<p>&nbsp;&nbsp;
						The results showed that both PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> localized to the perihaustorial region, though the distribution of <span class="gene">GmBiP1</span> appeared biased towards the necks of the haustoria, while PsAvh262 appeared biased towards the tips ( <xref ref-type="fig" rid="f5"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, these results suggest that PsAvh262 <span class="trait">and</span> its target <span class="gene">GmBiP1</span> co-localize around the haustoria <span class="trait">and</span> so may possibly influence infection.
					</p>
				
					<p>&nbsp;&nbsp;
						To test this hypothesis, we co-expressed NbBiP5 <span class="trait">and</span> <span class="gene">GmBiP1</span> with wild-type PsAvh262 or the PsAvh262 mutants in <italic> N.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, MG132 treatment did not further increase the levels of <span class="gene">GmBiP1</span> produced by Avh262 expression, suggesting that Avh262 <span class="trait">and</span> MG132 act via the same process ( <xref ref-type="fig" rid="f7"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						To test this, we transiently expressed <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5 (with GFP fused to the N terminus of the signal peptides) in <italic> N.
					</p>
				
					<p>&nbsp;&nbsp;
						3 </xref> <span class="trait">and</span> <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 1c </xref> ; <xref ref-type="supplementary-material" rid="S1"> 6 </xref> ), we also examined the influence of NbBiP5, <span class="gene">GmBiP1</span> <span class="trait">and</span> <span class="gene">GmBiP4</span> on BAX-triggered cell death.
					</p>
				
					<p>&nbsp;&nbsp;
						benthamiana </italic> <span class="trait">and</span> rice homologues of <span class="gene">GmBiPs</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The sequences data (PsAvh262, <span class="gene">GmBiPs</span> <span class="trait">and</span> NbBiP5) that support the finding of this study have been deposited in GenBank ( <ext-link xmlns:xlink="http://www.
					</p>
				
					<p>&nbsp;&nbsp;
						1 (PsAvh262, GenBank); <span class="gene">Glyma08g02940</span> <span class="gene">(GmBiP1),</span> <span class="gene">Glyma08g02960</span> <span class="gene">(GmBiP2),</span> <span class="gene">Glyma05g36600</span> <span class="gene">(GmBiP3)</span> <span class="trait">and</span> <span class="gene">Glyma05g36620</span> <span class="gene">(GmBiP4)</span> in Phytozome; NbS00040865g0006.
					</p>
				
					<p>&nbsp;&nbsp;
						Protein complexes were pulled down using anti-Flag agarose gel, <span class="trait">and</span> co-precipitations of PsAvh262 <span class="trait">and</span> its mutants with <span class="gene">GmBiP1</span> were detected using immunoblots.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Localization of RFP- <span class="trait">and</span> GFP-fusions of PsAvh262 <span class="trait">and</span> mutants in the absence of <span class="gene">GmBiP1</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> f </bold> , <bold> g </bold> ) Co-localization of PsAvh262 <span class="trait">and</span> <span class="gene">GmBiP1</span> detected by bimolecular fluorescence complementation (BiFC) using the constructs shown in <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmBiP1</span> </italic> ( <bold> b </bold> ), <italic> Protein disulfide isomerase </italic> ( <italic> PDI </italic> ) ( <bold> c </bold> ) <span class="trait">and</span> <italic> vacuolar processing enzyme </italic> ( <italic> VPE </italic> ) ( <bold> d </bold> ) were determined by qRT&#x02013;PCR in soybeans inoculated with wild-type or <italic> PsAvh262 </italic> -silenced mutants of <italic> P.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Association Study of <span class="trait">Resistance</span> to Soybean Cyst Nematode ( <italic> Heterodera glycines </italic> ) HG Type 2.5.7 in Wild Soybean ( <italic> Glycine soja </italic> )</h2>
			<h3>PMID:27582748</h3>
				

				
					<h3>PMCID:4987380</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hengyou </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334878/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Chunying </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Davis </surname>  <given-names> Eric L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jinshe </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Griffin </surname>  <given-names> Joshua D. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kofsky </surname>  <given-names> Janice </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/362990/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Bao-Hua </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/356962/overview"/>  </contrib>
				<br/> 
				年份：<day> 02 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						, <xref rid="B11" ref-type="bibr"> 2012 </xref> ), <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> on chromosome 8, conferring HG Type 7 <span class="trait">resistance</span> (Liu et al
					</p>
				
					<p>&nbsp;&nbsp;
						, <xref rid="B11" ref-type="bibr"> 2012 </xref> ) <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> (Liu et al
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg1</span> </italic> , one of the major loci conferring soybean <span class="trait">resistance</span> to multiple SCN HG types (previously called races; Concibido et al
					</p>
				
					<p>&nbsp;&nbsp;
						, <xref rid="B41" ref-type="bibr"> 2000 </xref> ), <span class="trait">and</span> in <italic> Arabidopsis </italic> , mutation of a receptor-like protein kinase ( <italic> <span class="gene">RLK</span> </italic> ) gene, <italic> RPK2 </italic> , results in a decrease in both nematode infection <span class="trait">and</span> syncytium size in the <italic> rpk2 </italic> mutant (Replogle et al
					</p>
				
					<p>&nbsp;&nbsp;
						18g106800 </italic> ) <span class="trait">and</span> one <span class="gene">CDPK</span> ( <italic> Glyma
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B16" ref-type="bibr"> 2015 </xref> ) indicated that a soybean ortholog of the <italic> Arabidopsis </italic> gene <italic> RPK2, <span class="gene">GmRPK2B</span> </italic> , is strongly expressed in root tips, vascular tissues, <span class="trait">and</span> nematode feeding sites
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						, <xref rid="B11" ref-type="bibr"> 2012 </xref> ) <span class="trait">and</span> <italic> <span class="gene">Rhg4</span> </italic> (Liu et al.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg1</span> </italic> , one of the major loci conferring soybean <span class="trait">resistance</span> to multiple SCN HG types (previously called races; Concibido et al.
					</p>
				
					<p>&nbsp;&nbsp;
						, <xref rid="B41" ref-type="bibr"> 2000 </xref> ), <span class="trait">and</span> in <italic> Arabidopsis </italic> , mutation of a receptor-like protein kinase ( <italic> <span class="gene">RLK</span> </italic> ) gene, <italic> RPK2 </italic> , results in a decrease in both nematode infection <span class="trait">and</span> syncytium size in the <italic> rpk2 </italic> mutant (Replogle et al.
					</p>
				
					<p>&nbsp;&nbsp;
						18g106800 </italic> ) <span class="trait">and</span> one <span class="gene">CDPK</span> ( <italic> Glyma.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B16" ref-type="bibr"> 2015 </xref> ) indicated that a soybean ortholog of the <italic> Arabidopsis </italic> gene <italic> RPK2, <span class="gene">GmRPK2B</span> </italic> , is strongly expressed in root tips, vascular tissues, <span class="trait">and</span> nematode feeding sites.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>