<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Drought resistance</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Drought resistance</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Mapping of quantitative trait loci for canopy-wilting trait in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:22566068  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Carter TE Jr, Purcell LC, King CA, Ries LL, Chen P, Schapaugh W Jr, Sinclair TR, Boerma HR.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects [Glycine max (L.) Merr] soybean at most developmental stages, which collectively results in <span class="trait">yield</span> reduction. Little information is available on relative contribution and chromosomal locations of quantitative trait loci (QTL) conditioning <span class="trait">drought</span> tolerance in soybean. A Japanese germplasm accession, PI 416937, was found to possess <span class="trait">drought</span> <span class="trait">resistance.</span> Under moisture-deficit conditions, PI 416937 wilted more slowly in the field than elite cultivars and has been used as a parent in breeding programs to improve soybean productivity. A recombinant inbred line (RIL) population was derived from a cross between PI 416937 and Benning, and the population was phenotyped for canopy wilting under rain-fed field conditions in five distinct environments to identify the QTL associated with the canopy-wilting trait. In a combined analysis over environments, seven QTL that explained 75 % of the variation in canopy-wilting trait were identified on different chromosomes, implying the complexity of this trait. Five QTL inherited their positive alleles from PI 416937. Surprisingly, the other two QTL inherited their positive alleles from Benning. These putative QTL were co-localized with other QTL previously identified as related to plant abiotic stresses in soybean, suggesting that canopy-wilting QTL may be associated with additional morpho-physiological traits in soybean. A locus on chromosome 12 <span class="gene">(Gm12)</span> from PI 416937 was detected in the combined analysis as well as in each individual environment, and explained 27 % of the variation in canopy-wilting. QTL identified in PI 416937 could provide an efficient means to augment field-oriented development of drought-tolerant soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmGBP1, a homolog of human ski interacting protein in soybean, regulates flowering and stress tolerance in Arabidopsis.</h2>
			<h3>PMID:23388059  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang Y(1), Zhao L, Li H, Gao Y, Li Y, Wu X, Teng W, Han Y, Zhao X, Li W.
				<br/> 
				年份：2013 Feb 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: SKIP is a transcription cofactor in many eukaryotes. It can regulate plant stress tolerance in rice and Arabidopsis. But the homolog of SKIP protein in soybean has been not reported up to now. RESULTS: In this study, the expression patterns of soybean GAMYB binding protein gene <span class="gene">(GmGBP1)</span> encoding a homolog of SKIP protein were analyzed in soybean under abiotic stresses and different day lengths. The expression of <span class="gene">GmGBP1</span> was induced by polyethyleneglycol 6000, NaCl, gibberellin, abscisic acid and heat stress. <span class="gene">GmGBP1</span> had transcriptional activity in C-terminal. <span class="gene">GmGBP1</span> could interact with R2R3 domain of <span class="gene">GmGAMYB1</span> in SKIP domain to take part in gibberellin flowering pathway. In long-day (16 h-light) condition, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> exhibited earlier flowering and less number of rosette leaves; Suppression of AtSKIP in Arabidopsis resulted in growth arrest, flowering delay and down-regulation of many flowering-related genes (CONSTANS, FLOWERING LOCUS T, LEAFY); Arabidopsis myb33 mutant plants with ectopic overexpression of <span class="gene">GmGBP1</span> showed the same flowering phenotype with wild type. In short-day (8 h-light) condition, transgenic Arabidopsis plants with <span class="gene">GmGBP1</span> flowered later and showed a higher level of FLOWERING LOCUS C compared with wild type. When treated with abiotic stresses, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> enhanced the tolerances to heat and <span class="trait">drought</span> stresses but reduced the tolerance to high salinity, and affected the expressions of several stress-related genes. CONCLUSIONS: In Arabidopsis, <span class="gene">GmGBP1</span> might positively regulate the flowering time by affecting CONSTANS, FLOWERING LOCUS T, LEAFY and GAMYB directly or indirectly in photoperiodic and gibberellin pathways in LDs, but <span class="gene">GmGBP1</span> might represse flowering by affecting FLOWERING LOCUS C and SHORT VEGETATIVE PHASE in autonomous pathway in SDs. <span class="gene">GmGBP1</span> might regulate the activity of ROS-eliminating to improve the <span class="trait">resistance</span> to heat and <span class="trait">drought</span> but reduce the high-salinity tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmERF5,</span> a new member of the soybean EAR motif-containing ERF transcription factor, enhances <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:25779701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dong L(1), Cheng Y(1), Wu J(2), Cheng Q(1), Li W(1), Fan S(1), Jiang L(1), Xu Z(3), Kong F(4), Zhang D(3), Xu P(5), Zhang S(5).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora root and stem rot of soybean [Glycine max (L.) Merr.], caused by Phytophthora sojae Kaufmann and Gerdemann, is a destructive disease throughout the soybean planting regions in the world. Here, we report insights into the function and underlying mechanisms of a novel ethylene response factor (ERF) in soybean, namely <span class="gene">GmERF5,</span> in host responses to P. sojae. <span class="gene">GmERF5-overexpressing</span> transgenic soybean exhibited significantly enhanced <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the PR10, PR1-1, and PR10-1 genes. Sequence analysis suggested that <span class="gene">GmERF5</span> contains an AP2/ERF domain of 58 aa and a conserved ERF-associated amphiphilic repression (EAR) motif in its C-terminal region. Following stress treatments, <span class="gene">GmERF5</span> was significantly induced by P. sojae, ethylene (ET), abscisic acid (ABA), and salicylic acid (SA). The activity of the <span class="gene">GmERF5</span> promoter <span class="gene">(GmERF5P)</span> was upregulated in tobacco leaves with ET, ABA, Phytophthora nicotianae, salt, and <span class="trait">drought</span> treatments, suggesting that <span class="gene">GmERF5</span> could be involved not only in the induced defence response but also in the ABA-mediated pathway of salt and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> could bind to the GCC-box element and act as a repressor of gene transcription. It was targeted to the nucleus when transiently expressed in Arabidopsis protoplasts. <span class="gene">GmERF5</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> and eukaryotic translation initiation factor <span class="gene">(GmEIF)</span> both in yeast cells and in planta. To the best of our knowledge, <span class="gene">GmERF5</span> is the first soybean EAR motif-containing ERF transcription factor demonstrated to be involved in the response to pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Osmoprotective functions conferred to soybean plants via inoculation with Sphingomonas sp. LK11 and exogenous trehalose.</h2>
			<h3>PMID:28942839</h3>
				

				
			<p>
				作者：Asaf S(1), Khan AL(2), Khan MA(1), Imran QM(1), Yun BW(1), Lee IJ(3).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Osmotic stress induced by <span class="trait">drought</span> can hinder the growth and <span class="trait">yield</span> of crop plants. To understand the eco-physiological role of osmoprotectants, the combined utilization of endophytes and osmolytes (trehalose) can be an ideal strategy used to overcome the adverse effects of <span class="trait">drought.</span> Hence, in the present study, we aimed to investigate the role of Sphingomonas sp. LK11, which produces phytohormones and synthesizes trehalose, in improving soybean plant growth under drought-induced osmotic stress (-0.4, -0.9, and -1.2MPa). The results showed that the inoculation of soybean plants with Sphingomonas sp. LK11 significantly increased plant length, dry biomass, photosynthetic pigments, glutathione, amino acids (proline, glycine, and glutamate), and primary sugars as compared to control plants under varying <span class="trait">drought</span> stresses. Trehalose applied to the plant with or without endophyte-inoculation also showed similar plant growth-promoting attributes under stress. Stress exposure significantly enhanced endogenous jasmonic (JA) and abscisic (ABA) acid contents in control plants. In contrast, Sphingomonas sp. LK11-inoculation significantly lowered ABA and JA levels in soybean plants, but these phytohormones increased in response to combined treatments during stress. The drought-induced osmotic stress <span class="trait">resistance</span> associated with Sphingomonas sp. LK11 and trehalose was also evidenced by increased mRNA gene expression of soybean dehydration responsive element binding protein (DREB)-type transcription factors <span class="gene">(GmDREBa</span> and <span class="gene">GmDREB2)</span> and the MYB (myeloblastosis) transcription factor <span class="gene">(GmMYBJ1)</span> as compared to the control. In conclusion, our findings demonstrated that inoculation with this endophyte and trehalose improved the negative effects of drought-induced osmotic stress, and it enhanced soybean plant growth and tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The pathogenic development of Sclerotinia sclerotiorum in soybean requires specific host NADPH oxidases.</h2>
			<h3>PMID:28378935</h3>
				

				
			<p>
				作者：Ranjan A(1), Jayaraman D(2), Grau C(1), Hill JH(3), Whitham SA(3), Ané JM(2)(4), Smith DL(1), Kabbage M(1).
				<br/> 
				年份：2017 Apr 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant membrane-localized NADPH oxidases, also known as respiratory burst oxidase homologues (RBOHs), play crucial roles in various cellular activities, including plant disease responses, and are a major source of reactive oxygen species (ROS). Sclerotinia sclerotiorum is a cosmopolitan fungal pathogen that causes Sclerotinia stem rot (SSR) in soybean. Via a key virulence factor, oxalic acid, it induces programmed cell death (PCD) in the host plant, a process that is reliant on ROS generation. In this study, using protein sequence similarity searches, we identified 17 soybean RBOHs <span class="gene">(GmRBOHs)</span> and studied their contribution to SSR disease development, <span class="trait">drought</span> tolerance and nodulation. We clustered the soybean RBOH genes into six groups of orthologues based on phylogenetic analysis with their Arabidopsis counterparts. Transcript analysis of all 17 <span class="gene">GmRBOHs</span> revealed that, of the six identified groups, group VI <span class="gene">(GmRBOH-VI)</span> was specifically and drastically induced following S. sclerotiorum challenge. Virus-induced gene silencing (VIGS) of <span class="gene">GmRBOH-VI</span> using Bean pod mottle virus (BPMV) resulted in enhanced <span class="trait">resistance</span> to S. sclerotiorum and markedly reduced ROS levels during disease development. Coincidently, <span class="gene">GmRBOH-VI-silenced</span> plants were also found to be <span class="trait">drought</span> tolerant, but showed a reduced capacity to form nodules. Our results indicate that the pathogenic development of S. sclerotiorum in soybean requires the active participation of specific host RBOHs, to induce ROS and cell death, thus leading to the establishment of disease.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Virus-induced down-regulation of <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> genes enhances the stomatal response to abscisic acid and <span class="trait">drought</span> <span class="trait">resistance</span> in soybean.</h2>
			<h3>PMID:28419130  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ogata T(1), Nagatoshi Y(1), Yamagishi N(2), Yoshikawa N(2), Fujita Y(1)(3).
				<br/> 
				年份：2017 Apr 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a major threat to global soybean production. The limited transformation potential and polyploid nature of soybean have hindered functional analysis of soybean genes. Previous research has implicated farnesylation in the plant's response to abscisic acid (ABA) and <span class="trait">drought</span> tolerance. We therefore used virus-induced gene silencing (VIGS) to evaluate farnesyltransferase genes, <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> (Glycine max Enhanced Response to ABA1-A and -B), as potential targets for increasing <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Apple latent spherical virus (ALSV)-mediated <span class="gene">GmERA1-down-regulated</span> soybean leaves displayed an enhanced stomatal response to ABA and reduced water loss and wilting under dehydration conditions, suggesting that <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> negatively regulate ABA signaling in soybean guard cells. The findings provide evidence that the ALSV-VIGS system, which bypasses the need to generate transgenic plants, is a useful tool for analyzing gene function using only a single down-regulated leaf. Thus, the ALSV-VIGS system could constitute part of a next-generation molecular breeding pipeline to accelerate <span class="trait">drought</span> <span class="trait">resistance</span> breeding in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the soybean <span class="gene">GmERF3</span> gene, an AP2/ERF type transcription factor for increased tolerances to salt, <span class="trait">drought,</span> and diseases in transgenic tobacco.</h2>
			<h3>PMID:19602544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Li L, Xu Z, Chen X, Guo J, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new member of the AP2/ERF transcription factor family, <span class="gene">GmERF3,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF3</span> contained an AP2/ERF domain of 58 amino acids and two putative nuclear localization signal (NLS) domains. It belonged to a group IV protein in the ERF (ethylene response factor) subfamily as typified by a conserved N-terminal motif [MCGGAI(I/L)]. Expression of <span class="gene">GmERF3</span> was induced by treatments with high salinity, <span class="trait">drought,</span> abscisic acid (ABA), salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and soybean mosaic virus (SMV), whereas there was no significant <span class="gene">GmERF3</span> mRNA accumulation under cold stress treatment. <span class="gene">GmERF3</span> could bind to the GCC box and DRE/CRT element, and was targeted to the nucleus when transiently expressed in onion epidermal cells. The <span class="gene">GmERF3</span> protein fused to the GAL4 DNA-binding domain to activate transcription of reporter genes in yeast. Ectopic expression of the <span class="gene">GmERF3</span> gene in transgenic tobacco plants induced the expression of some PR genes and enhanced <span class="trait">resistance</span> against infection by Ralstonia solanacearum, Alternaria alternata, and tobacco mosaic virus (TMV), and gave tolerance to high salinity and dehydration stresses. Furthermore, overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions. The overall results suggested that <span class="gene">GmERF3</span> as an AP2/ERF transcription factor may play dual roles in response to biotic and abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel EAR-motif-containing gene <span class="gene">GmERF4</span> from soybean (Glycine max L.).</h2>
			<h3>PMID:19597961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Chen X, Xu Z, Li L, Guo J, Ma Y.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptional repressors are emerging as central regulators of development and stress responses in different organisms. The ERF-associated amphiphilic repression (EAR) motif was identified as essential for transcriptional repression. To gain a better understanding of this type of protein, we reported here a novel <span class="gene">GmERF4</span> protein from soybean. Sequence alignment showed that <span class="gene">GmERF4</span> contains one AP2/ERF domain, two putative nuclear localization signal regions and one EAR motif. The <span class="gene">GmERF4</span> protein was preferentially localized to the nucleus of onion epidermis cells and bound specifically to the GCC box and DRE/CRT element in vitro. Furthermore, the expression of <span class="gene">GmERF4</span> was induced by ethylene, JA, SA, cold, salt, <span class="trait">drought,</span> and soybean mosaic virus, and repressed by ABA. Constitutive expression of <span class="gene">GmERF4</span> in transgenic tobacco plants increased tolerance to salt and <span class="trait">drought</span> stresses compared with wild-type plants, but did not exhibit detectable <span class="trait">resistance</span> against bacterial infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors (MYB TFs) play diverse roles in plant development and stress responses. However, the mechanisms underlying the actions of MYB TFs during stress response remain unclear. In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance.</span> Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> salt stress, H2O2 and ABA. Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary root length, greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content. We also found that ROS could induce SOD/POD/CAT activity in OE lines. In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary root elongation. Some ROS-related genes were up-regulated under abiotic stress in <span class="gene">GmMYB84</span> transgenic plants compared with the WT. Furthermore, electrophoretic mobility shift assay and luciferase reporter analysis demonstrated that <span class="gene">GmMYB84</span> binds directly to the promoter of <span class="gene">GmRBOHB-1</span> and <span class="gene">GmRBOHB-2</span> genes. Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control root growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> in Glycine max L. that enhances abiotic stress tolerance in Saccharomyces cerevisiae.</h2>
			<h3>PMID:20155384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Zhao H, Liu Y, Li Q, Liu X, Tan H, Yuan C, Dong Y.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> was isolated from Glycine max. It is 1,596 bp in length with one ORF of 410 amino acids. Southern blot analysis revealed that it has at least two copies in the G. max genome. <span class="gene">GmGSK,</span> when transiently expressed in Nicotiana tabacum leaves, was localized in both cell membrane and cytoplasm. Northern blot analysis indicated that <span class="gene">GmGSK</span> is expressed in all tissues, with highest expression in the root. <span class="gene">GmGSK</span> can be induced by various abiotic stresses. When transformed with <span class="gene">GmGSK,</span> Saccharomyces cerevisiae exhibited enhanced <span class="trait">resistance</span> to salt and <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue-Specific Regulation of <span class="gene">Gma-miR396</span> Family on Coordinating Development and Low Water Availability Responses.</h2>
			<h3>PMID:28694817</h3>
				

				
			<p>
				作者：Liu W(1), Zhou Y(1), Li X(1), Wang X(1), Dong Y(1), Wang N(1), Liu X(1), Chen H(1), Yao N(1), Cui X(1), Jameel A(1), Wang F(1), Li H(1).
				<br/> 
				年份：2017 Jun 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previously, it was reported that miR396s interact with growth-regulating factors (GRFs) to modulate plant growth, development, and stress <span class="trait">resistance.</span> In soybean, 11 gma-miR396 precursors (Pre-miR396a-k) were found, and 24 <span class="gene">GmGRFs</span> were predicted as targets of seven mature gma-miR396s (gma-miR396a/b/c/e/h/i/k). To explore the roles of the miR396-GRF module in low water availability response of soybean, we analyzed the expression of Pre-miR396a-k, and found that Pre-miR396a/i/bdgk/e/h were up-regulated in leaves and down-regulated in roots; on the contrary, <span class="gene">GmGRF5/6/7/8/15/17/21</span> were down-regulated in leaves and <span class="gene">GmGRF1/2/17/18/19/20/21/22/23/24</span> were up-regulated in roots of low water potential stressed soybean. Any one of gma-miR396a/b/c/e/h/i/k was able to interact with 20 <span class="gene">GmGRFs</span> <span class="gene">(GmGRF1/2/6-11/13-24),</span> confirming that this module represents a multi-to-multi network interaction. We generated Arabidopsis plants over-expressing each of the 11 gma-miR396 precursors (Pre-miR396a-k), and seven of them (miR396a/b/c/e/h/i/k-OE transgenic Arabidopsis) showed altered development. The low water availability of miR396a/b/c/e/h/i/k-OE was enhanced in leaves but reduced in seeds and roots. Contrary to previous reports, miR396a/b/c/i-OE seedlings showed lower survival rate than WT when recovering after rewatering under soil drying. In general, we believe our findings are valuable to understand the role of gma-miR396 family in coordinating development and low water availability responses, and can provide potential strategies and directions for soybean breeding programs to improve seed <span class="trait">yield</span> and plant <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsSNAP33, a novel Glycine soja SNAP25-type protein gene: Improvement of plant salt and <span class="trait">drought</span> tolerances in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:28841544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nisa ZU(1), Mallano AI(2), Yu Y(3), Chen C(4), Duan X(5), Amanullah S(6), Kousar A(7), Baloch AW(8), Sun X(9), Tabys D(10), Zhu Y(11).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The N-ethylmaleimide-sensitive factor attachment protein receptor (SNARE) superfamily, specifically the SNAP25-type proteins and t-SNAREs, have been proposed to regulate cellular processes and plant <span class="trait">resistance</span> mechanisms. However, little is known about the role of SNAP25-type proteins in combating abiotic stresses, specifically in wild soybean. In the current study, the isolation and functional characterization of the putative synaptosomal-associated SNAP25-type protein gene GsSNAP33 from wild soybean (Glycine soja) were performed. GsSNAP33 has a molecular weight of 33,311 Da and comprises 300 amino acid residues along with Qb-Qc SNARE domains. Multiple sequence alignment revealed the highest similarity of the GsSNAP33 protein to <span class="gene">GmSNAP33</span> (91%), VrSNAP33 (89%), PvSNAP33 (86%) and AtSNAP33 (63%). Phylogenetic studies revealed the abundance of SNAP33 proteins mostly in dicotyledons. Quantitative real-time PCR assays confirmed that GsSNAP33 expression can be induced by salt, alkali, ABA and PEG treatments and that GsSNAP33 is highly expressed in the pods, seeds and roots of Glycine soja. Furthermore, the overexpression of the GsSNAP33 gene in WT Arabidopsis thaliana resulted in increased germination rates, greater root lengths, improved photosynthesis, lower electrolyte leakage, higher biomass production and up-regulated expression levels of various stress-responsive marker genes, including KINI, COR15A, P5Cs, RAB18, <span class="gene">RD29A</span> and COR47 in transgenic lines compared with those in WT lines. Subcellular localization studies revealed that the GsSNAP33-eGFP fusion protein was localized to the plasma membrane, while eGFP was distributed throughout whole cytoplasm of onion epidermal cells. Collectively, our findings suggest that GsSNAP33, a novel plasma membrane protein gene of Glycine soja, might be involved in improving plant responses to salt and <span class="trait">drought</span> stresses in Arabidopsis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in high salt and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						YABBY family is a plant specific transcription factor family, with the typical N-terminal C2C2 type zinc finger domain and the C-terminal YABBY conservative structure domain, which plays important biological roles in plant growth, development and morphogenesis. In this study, a total of 17 YABBY genes were identified in the soybean genome. The results of this research showed that 17 soybean YABBY genes were located on 11 chromosomes. Analysis of putative cis-acting elements showed that soybean YABBY genes contained lots of MYB and MYC elements. Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses. And the transient expression in Arabidopsis protoplasts showed that <span class="gene">GmYABBY3</span> protein distributed uniformly the whole cells, while <span class="gene">GmYABBY10</span> protein was mainly localized in the membranes and cytoplasm and <span class="gene">GmYABBY16</span> protein was localized the nucleus and membranes. To further identify the function of <span class="gene">GmYABBY10,</span> we obtained the transgenic Arabidopsis overexpression <span class="gene">GmYABBY10.</span> Based on germination and seedling root arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment. While the root length and root surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings. When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and salt stresses. This study provided valuable information regarding the classification and functions of YABBY genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of the <span class="gene">RAV</span> Family in Soybean and Functional Identification of <span class="gene">GmRAV-03</span> Involvement in Salt and <span class="trait">Drought</span> Stresses and Exogenous ABA Treatment.</h2>
			<h3>PMID:28634481</h3>
				

				
			<p>
				作者：Zhao SP(1)(2), Xu ZS(2), Zheng WJ(1), Zhao W(1)(2), Wang YX(3), Yu TF(1), Chen M(2), Zhou YB(1), Min DH(1), Ma YZ(2), Chai SC(1), Zhang XH(1).
				<br/> 
				年份：2017 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcription factors play vital roles in plant growth and in plant responses to abiotic stresses. The <span class="gene">RAV</span> transcription factors contain a B3 DNA binding domain and/or an APETALA2 (AP2) DNA binding domain. Although genome-wide analyses of <span class="gene">RAV</span> family genes have been performed in several species, little is known about the family in soybean (Glycine max L.). In this study, a total of 13 <span class="gene">RAV</span> genes, named as <span class="gene">GmRAVs,</span> were identified in the soybean genome. We predicted and analyzed the amino acid compositions, phylogenetic relationships, and folding states of conserved domain sequences of soybean <span class="gene">RAV</span> transcription factors. These soybean <span class="gene">RAV</span> transcription factors were phylogenetically clustered into three classes based on their amino acid sequences. Subcellular localization analysis revealed that the soybean <span class="gene">RAV</span> proteins were located in the nucleus. The expression patterns of 13 <span class="gene">RAV</span> genes were analyzed by quantitative real-time PCR. Under <span class="trait">drought</span> stresses, the <span class="gene">RAV</span> genes expressed diversely, up- or down-regulated. Following NaCl treatments, all <span class="gene">RAV</span> genes were down-regulated excepting <span class="gene">GmRAV-03</span> which was up-regulated. Under abscisic acid (ABA) treatment, the expression of all of the soybean <span class="gene">RAV</span> genes increased dramatically. These results suggested that the soybean <span class="gene">RAV</span> genes may be involved in diverse signaling pathways and may be responsive to abiotic stresses and exogenous ABA. Further analysis indicated that <span class="gene">GmRAV-03</span> could increase the transgenic lines <span class="trait">resistance</span> to high salt and <span class="trait">drought</span> and result in the transgenic plants insensitive to exogenous ABA. This present study provides valuable information for understanding the classification and putative functions of the <span class="gene">RAV</span> transcription factors in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peanut gene expression profiling in developing seeds at different reproduction stages during Aspergillus parasiticus infection.</h2>
			<h3>PMID:18248674  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo B(1), Chen X, Dang P, Scully BT, Liang X, Holbrook CC, Yu J, Culbreath AK.
				<br/> 
				年份：2008 Feb 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Peanut (Arachis hypogaea L.) is an important crop economically and nutritionally, and is one of the most susceptible host crops to colonization of Aspergillus parasiticus and subsequent aflatoxin contamination. Knowledge from molecular genetic studies could help to devise strategies in alleviating this problem; however, few peanut DNA sequences are available in the public database. In order to understand the molecular basis of host <span class="trait">resistance</span> to aflatoxin contamination, a large-scale project was conducted to generate expressed sequence tags (ESTs) from developing seeds to identify resistance-related genes involved in defense response against Aspergillus infection and subsequent aflatoxin contamination. RESULTS: We constructed six different cDNA libraries derived from developing peanut seeds at three reproduction stages (R5, R6 and R7) from a resistant and a susceptible cultivated peanut genotypes, 'Tifrunner' (susceptible to Aspergillus infection with higher aflatoxin contamination and resistant to TSWV) and 'GT-C20' (resistant to Aspergillus with reduced aflatoxin contamination and susceptible to TSWV). The developing peanut seed tissues were challenged by A. parasiticus and <span class="trait">drought</span> stress in the field. A total of 24,192 randomly selected cDNA clones from six libraries were sequenced. After removing vector sequences and quality trimming, 21,777 high-quality EST sequences were generated. Sequence clustering and assembling resulted in 8,689 unique EST sequences with 1,741 tentative consensus EST sequences (TCs) and 6,948 singleton ESTs. Functional classification was performed according to <span class="gene">MIPS</span> functional catalogue criteria. The unique EST sequences were divided into twenty-two categories. A similarity search against the non-redundant protein database available from NCBI indicated that 84.78% of total ESTs showed significant similarity to known proteins, of which 165 genes had been previously reported in peanuts. There were differences in overall expression patterns in different libraries and genotypes. A number of sequences were expressed throughout all of the libraries, representing constitutive expressed sequences. In order to identify resistance-related genes with significantly differential expression, a statistical analysis to estimate the relative abundance (R) was used to compare the relative abundance of each gene transcripts in each cDNA library. Thirty six and forty seven unique EST sequences with threshold of R &gt; 4 from libraries of 'GT-C20' and 'Tifrunner', respectively, were selected for examination of temporal gene expression patterns according to EST frequencies. Nine and eight resistance-related genes with significant up-regulation were obtained in 'GT-C20' and 'Tifrunner' libraries, respectively. Among them, three genes were common in both genotypes. Furthermore, a comparison of our EST sequences with other plant sequences in the TIGR Gene Indices libraries showed that the percentage of peanut EST matched to Arabidopsis thaliana, maize (Zea mays), Medicago truncatula, rapeseed (Brassica napus), rice (Oryza sativa), soybean (Glycine max) and wheat (Triticum aestivum) ESTs ranged from 33.84% to 79.46% with the sequence identity &gt;/= 80%. These results revealed that peanut ESTs are more closely related to legume species than to cereal crops, and more homologous to dicot than to monocot plant species. CONCLUSION: The developed ESTs can be used to discover novel sequences or genes, to identify resistance-related genes and to detect the differences among alleles or markers between these resistant and susceptible peanut genotypes. Additionally, this large collection of cultivated peanut EST sequences will make it possible to construct microarrays for gene expression studies and for further characterization of host <span class="trait">resistance</span> mechanisms. It will be a valuable genomic resource for the peanut community. The 21,777 ESTs have been deposited to the NCBI GenBank database with accession numbers ES702769 to ES724546.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant antimicrobial peptides: an overview of SuperSAGE transcriptional profile and a functional review.</h2>
			<h3>PMID:20088771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kido EA(1), Pandolfi V, Houllou-Kido LM, Andrade PP, Marcelino FC, Nepomuceno AL, Abdelnoor RV, Burnquist WL, Benko-Iseppon AM.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Defensin, thionin and lipid transfer protein (LTP) gene families, which antimicrobial activity has an attractive use in protein engineering and transgenic production of agronomical important plants, have been here functionally reviewed. Also, a transcriptional overview of a set of plant SuperSAGE libraries and analysis looking for 26 bp tags possibly annotated for those families is presented. Tags differentially expressed (p = 0.05) or constitutively transcribed were identified from leaves or roots SuperSAGE libraries from important Brazilian plant species [cowpea (Vigna unguiculata (L.) Walp.), soybean (Glycine max (L.) Merr.) and modern sugarcane hybrids (Saccharum spp.)] submitted to abiotic [salt (100 mM NaCl) or <span class="trait">drought]</span> or biotic stresses [fungus inoculation (Phakopsora pachyrhizi; Asiatic Soyben <span class="trait">Rust</span> phytopathogen)]. The diverse transcriptional patterns observed, probably related to the variable range of targets and functions involved, could be the first step to unravel the antimicrobial peptide world and the plant stress response relationship. Moreover, SuperSAGE opens the opportunity to find some SNPs or even rare transcript that could be important on plant stress <span class="trait">resistance</span> mechanisms. Putative defensin or <span class="gene">LTP</span> identified by SuperSAGE following a specific plant treatment or physiological condition could be useful for future use in genetic improvement of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The essence of NAC gene family to the cultivation of drought-resistant soybean (Glycine max L. Merr.) cultivars.</h2>
			<h3>PMID:28241800  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hussain RM(1)(2), Ali M(3), Feng X(4), Li X(5).
				<br/> 
				年份：2017 Feb 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The NAC gene family is notable due to its large size, as well as its relevance in crop cultivation - particularly in terms of enhancing stress tolerance of plants. These plant-specific proteins contain NAC domain(s) that are named after Petunia NAM and Arabidopsis ATAF1/2 and CUC2 transcription factors based on the consensus sequence they have. Despite the knowledge available regarding NAC protein function, an extensive study on the possible use of <span class="gene">GmNACs</span> in developing soybean cultivars with superior <span class="trait">drought</span> tolerance is yet to be done. RESULTS: In response to this, our study was carried out, mainly through means of phylogenetic analysis (rice and Arabidopsis NAC genes served as seeding sequences). Through this, 139 <span class="gene">GmNAC</span> genes were identified and later grouped into 17 clusters. Furthermore, real-time quantitative PCR was carried out on drought-stressed and unstressed leaf tissues of both sensitive (B217 and H228) and tolerant (Jindou 74 and 78) cultivars. This was done to analyze the gene expression of 28 dehydration-responsive <span class="gene">GmNAC</span> genes. Upon completing the analysis, it was found that <span class="gene">GmNAC</span> gene expression is actually dependent on genotype. Eight of the 28 selected genes <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> and <span class="gene">GmNAC087)</span> were discovered to have high expression levels in the drought-resistant soybean varieties tested. This holds true for both extreme and standard <span class="trait">drought</span> conditions. Alternatively, the drought-sensitive cultivars exhibited lower <span class="gene">GmNAC</span> expression levels in comparison to their tolerant counterparts. CONCLUSION: The study allowed for the identification of eight <span class="gene">GmNAC</span> genes that could be focused upon in future attempts to develop superior soybean varieties, particularly in terms of <span class="trait">drought</span> <span class="trait">resistance.</span> This study revealed that there were more dehydration-responsive <span class="gene">GmNAC</span> genes as <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC020</span> and <span class="gene">GmNAC021)</span> in addition to what were reported in earlier inquiries. It is important to note though, that discovering such notable genes is not the only goal of the study. It managed to put emphasis on the significance of further understanding the potential of soybean <span class="gene">GmNAC</span> genes, for the purpose of enhancing tolerance towards abiotic stress in general. This scientific inquiry has also revealed that cultivar genotypes tend to differ in their drought-induced gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmCYP82A3, a Soybean Cytochrome <span class="gene">P450</span> Family Gene Involved in the Jasmonic Acid and Ethylene Signaling Pathway, Enhances Plant <span class="trait">Resistance</span> to Biotic and Abiotic Stresses.</h2>
			<h3>PMID:27588421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Q(1), Cui X(2), Lin S(1), Gan S(2), Xing H(2), Dou D(1).
				<br/> 
				年份：2016 Sep 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cytochrome <span class="gene">P450</span> monooxygenases (P450s) represent a large and important enzyme superfamily in plants. They catalyze numerous monooxygenation/hydroxylation reactions in biochemical pathways, P450s are involved in a variety of metabolic pathways and participate in the homeostasis of phytohormones. The CYP82 family genes specifically reside in dicots and are usually induced by distinct environmental stresses. However, their functions are largely unknown, especially in soybean (Glycine max L.). Here, we report the function of <span class="gene">GmCYP82A3,</span> a gene from soybean CYP82 family. Its expression was induced by Phytophthora sojae infection, salinity and <span class="trait">drought</span> stresses, and treatment with methyl jasmonate (MeJA) or ethephon (ETH). Its expression levels were consistently high in resistant cultivars. Transgenic Nicotiana benthamiana plants overexpressing <span class="gene">GmCYP82A3</span> exhibited strong <span class="trait">resistance</span> to Botrytis cinerea and Phytophthora parasitica, and enhanced tolerance to salinity and <span class="trait">drought</span> stresses. Furthermore, transgenic plants were less sensitive to jasmonic acid (JA), and the enhanced <span class="trait">resistance</span> was accompanied with increased expression of the JA/ET signaling pathway-related genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL for increased fibrous roots in soybean.</h2>
			<h3>PMID:21165732  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Lee GJ, Boerma RH.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects soybean at various developmental stages, which collectively results in <span class="trait">yield</span> reduction. Unpredictable rainfall has been reported to contribute about 36% to variation of <span class="trait">yield</span> difference between the rain-fed and irrigated fields. Among the <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms, <span class="trait">drought</span> avoidance in genotypes with fibrous roots was recognized to be associated with <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Plant introduction PI416937 was shown to possess fibrous roots and has been used as a parent in breeding programs to improve soybean productivity. Little information is available on relative contribution and chromosomal location of quantitative trait loci (QTL) conditioning fibrous roots in soybean. To identify the genomic locations and genetic bases of this trait, a recombinant inbred line population was derived from a cross between PI416937 and 'Benning'. To detect associated QTLs, phenotypic data were collected and analyzed for 2 years under rain-fed field conditions. The selective genotyping approach was used to reduce the costs and work associated with conducting the QTL analysis. A total of five QTLs were identified on chromosomes <span class="gene">Gm01</span> (Satt383), <span class="gene">Gm03</span> (Satt339), <span class="gene">Gm04</span> (Sct_191), <span class="gene">Gm08</span> (Satt429), and <span class="gene">Gm20</span> (Sat_299), and together explained 51% of the variation in root score. Detected QTLs were co-localized with QTLs related to root morphology, suggesting that fibrous roots QTL may be associated with other morpho-physiological traits and seed <span class="trait">yield</span> in soybean. Genetic dissection of the fibrous roots trait at the individual marker loci will allow for marker-assisted selection to develop soybean genotypes with enhanced levels of fibrous roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsLRPK, a novel cold-activated leucine-rich repeat receptor-like protein kinase from Glycine soja, is a positive regulator to cold stress tolerance.</h2>
			<h3>PMID:24388511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang L(1), Wu K(1), Gao P(2), Liu X(1), Li G(3), Wu Z(4).
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant LRR-RLKs serve as protein interaction platforms, and as regulatory modules of protein activation. Here, we report the isolation of a novel plant-specific <span class="gene">LRR-RLK</span> from Glycine soja (termed GsLRPK) by differential screening. GsLRPK expression was cold-inducible and shows Ser/Thr protein kinase activity. Subcellular localization studies using GFP fusion protein indicated that GsLRPK is localized in the plasma membrane. Real-time PCR analysis indicated that temperature, salt, <span class="trait">drought,</span> and ABA treatment can alter GsLRPK gene transcription in G. soja. However, just protein induced by cold stress not by salinity and ABA treatment in tobacco was found to possess kinase activity. Furthermore, we found that overexpression of GsLRPK in yeast and Arabidopsis can enhance <span class="trait">resistance</span> to cold stress and increase the expression of a number of cold responsive gene markers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of gma-MIR394a confers tolerance to <span class="trait">drought</span> in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:23000164  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni Z(1), Hu Z, Jiang Q, Zhang H.
				<br/> 
				年份：2012 Oct 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs, key posttranscriptional regulators of eukaryotic gene expression, play important roles in plant development and response to stress. In this study, a soybean gma-MIR394a gene was functionally characterized, especially with regard to its role in <span class="trait">drought</span> stress <span class="trait">resistance.</span> Expression analysis revealed that gma-MIR394a was expressed differentially in various soybean tissues and was induced by <span class="trait">drought,</span> high salinity, low temperature stress, and abscisic acid treatment in leaves. One target gene of gma-miR394a, <span class="gene">Glyma08g11030,</span> was predicted and verified using a modified 5' RLM-RACE (RNA ligase-mediated rapid amplification of 5' cDNA ends) assay. Overexpression of gma-MIR394a resulted in plants with lowered leaf water loss and enhanced <span class="trait">drought</span> tolerance. Furthermore, overexpression of gma-MIR394a in Arabidopsis reduced the transcript of an F-box gene <span class="gene">(At1g27340)</span> containing a miR394 complementary target site. These results suggest that the gma-MIR394a gene functions in positive modulation of <span class="trait">drought</span> stress tolerance and has potential applications in molecular breeding to enhance <span class="trait">drought</span> tolerance in crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>