<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Drought</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Drought</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Diurnal Oscillations of Soybean Circadian Clock and <span class="trait">Drought</span> Responsive Genes</h2>
			<h3>PMID:24475115</h3>
				

				
					<h3>PMCID:3903518</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Marcolino-Gomes </surname>  <given-names> Juliana </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana Aparecida </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fuganti-Pagliarini </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bendix </surname>  <given-names> Claire </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakayama </surname>  <given-names> Thiago Jonas </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Celaya </surname>  <given-names> Brandon </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Molinari </surname>  <given-names> Hugo Bruno Correa </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Maria Cristina Neves </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harmon </surname>  <given-names> Frank G. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhythms produced by the endogenous circadian clock play a critical role in allowing plants to respond and adapt to the environment. While there is a well-established regulatory link between the circadian clock and responses to abiotic stress in model plants, little is known of the circadian system in crop species like soybean. This study examines how <span class="trait">drought</span> impacts diurnal oscillation of both <span class="trait">drought</span> responsive and circadian clock genes in soybean. <span class="trait">Drought</span> stress induced marked changes in gene expression of several circadian clock-like components, such as LCL1-, <span class="gene">GmELF4-</span> and PRR-like genes, which had reduced expression in stressed plants. The same conditions produced a phase advance of expression for the <span class="gene">GmTOC1-like,</span> <span class="gene">GmLUX-like</span> and <span class="gene">GmPRR7-like</span> genes. Similarly, the rhythmic expression pattern of the soybean drought-responsive genes DREB-, bZIP-, GOLS-, RAB18- and Remorin-like changed significantly after plant exposure to <span class="trait">drought.</span> In silico analysis of promoter regions of these genes revealed the presence of cis-elements associated both with stress and circadian clock regulation. Furthermore, some soybean genes with upstream ABRE elements were responsive to abscisic acid treatment. Our results indicate that some connection between the <span class="trait">drought</span> response and the circadian clock may exist in soybean since (i) <span class="trait">drought</span> stress affects gene expression of circadian clock components and (ii) several stress responsive genes display diurnal oscillation in soybeans.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The circadian clock is an endogenous timer that plays a key role in the coordination of plant biological activities with diurnal variations, conferring adaptive advantages to organisms. In this system, environmental cues like light and temperature play important role in entrain the circadian clock responses. Recent studies have demonstrated a correlation between the circadian clock and plant responses to <span class="trait">drought,</span> suggesting a close connection between both pathways <xref rid="pone.0086402-Legnaioli1" ref-type="bibr"> [10] </xref> , <xref rid="pone.0086402-Wilkins1" ref-type="bibr"> [11] </xref> . The more recent existing model of the circadian clock in plants comprises interlocked feedback loops, which includes a ring of three sequential negative steps: (a)the inhibition of evening complex (EC) genes (ELF3, LUX, and ELF4) by the rise of LHY/CCA1 in the late night, (b) the inhibition of PRR genes by EC in the early night, and (c) the inhibition of LHY/CCA1 by PRRs in the day <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> . The evening complex is suggested to represent the structure of the evening loop <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> . The two partially redundant morning Myb-like transcription factors CCA1 and <span class="gene">LHY</span> regulate the expression of <span class="gene">TOC1</span> and GI. TOC1/PRR1 is a member of the clock-specific transcription factor family of pseudo-response regulator (PRR) proteins, and GI is a vascular plant-specific protein with a poorly understood molecular function. CCA1 and <span class="gene">LHY</span> reach peak levels in the morning to repress the daytime expression of <span class="gene">TOC1</span> and GI. CCA1 and <span class="gene">LHY</span> directly bind to the <span class="gene">TOC1</span> promoter at a 9 bp target sequence referred to as the evening element (EE; AAATATCT), and this interaction suppresses transcription at this promoter <xref rid="pone.0086402-Harmer1" ref-type="bibr"> [13] </xref> , <xref rid="pone.0086402-Alabad1" ref-type="bibr"> [14] </xref> . Recently, Pokhilko and colleagues extended the inhibitory action of LHY/CCA1 to all evening genes (TOC1, LUX, ELF4, ELF3 and GI) <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> . <span class="gene">TOC1</span> appears to repress CCA1 and <span class="gene">LHY</span> expression inside the morning loop <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0086402-Gendron1" ref-type="bibr"> [15] </xref> , while the EC indirectly activate CCA1/LHY expression, by suppressing the expression of CCA1/LHY&#x02019;s repressors <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0086402-Dixon1" ref-type="bibr"> [16] </xref> &#x02013; <xref rid="pone.0086402-Kikis1" ref-type="bibr"> [18] </xref> . Two additional loops maintain the adequate expression of CCA1/LHY and TOC1. In the morning, PRR5, PRR7 and PRR9 are activated through the activity of CCA1/LHY, and in turn PRR5, PRR7 and PRR9 feedback repress the expression of CCA1/LHY <xref rid="pone.0086402-Farr1" ref-type="bibr"> [19] </xref> , <xref rid="pone.0086402-Nakamichi1" ref-type="bibr"> [20] </xref> . In a posttranslational loop, the F-box protein ZTL targets <span class="gene">TOC1</span> for degradation <xref rid="pone.0086402-Kim1" ref-type="bibr"> [21] </xref> . GI controls the activity of ZTL, and the physical interaction between GI and ZTL (light dependent) stabilizes both ZTL and <span class="gene">TOC1</span> expression during the day <xref rid="pone.0086402-Kim1" ref-type="bibr"> [21] </xref> . According to the most recent circadian clock model in Arabidopsis, GI increases <span class="gene">TOC1</span> expression by the inhibition of the EC, which is a negative regulator of <span class="gene">TOC1</span> expression <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> . Together, the three interlocking feedback loops ensure that the clock produces robust and accurate rhythms.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized cycle threshold (Ct) values were calculated based on the geometric average of two endogenous genes (Elongation factor 1-&#x003b2;, <span class="gene">Glyma13g04050,</span> and &#x003b2;-actin, <span class="gene">Glyma15g05570)</span> previously identified as stably expressed under many developmental stages and stress situations <xref rid="pone.0086402-Jian1" ref-type="bibr"> [32] </xref> , <xref rid="pone.0086402-StolfMoreira1" ref-type="bibr"> [33] </xref> . The normalized expression (NE) was calculated using the following formula: NE&#x0200a;=&#x0200a;2&#x02227; <sup> -(&#x00394;Ct) </sup> , where &#x00394;Ct&#x0200a;=&#x0200a; Ct experimental &#x02013; Ct normalizer (n). The primers for the <italic> Glycine max </italic> -like clock genes were designed using soybean Glyma ortholog sequences from Phytozome <italic> G. max </italic> v.1.0 (see <xref ref-type="table" rid="pone-0086402-t001"> Table 1 </xref> ). The soybean drought-responsive genes were chosen according to previous work confirming the upregulation of these genes under conditions of dehydration <xref rid="pone.0086402-Rodrigues1" ref-type="bibr"> [34] </xref> , <xref rid="pone.0086402-Mizuno1" ref-type="bibr"> [35] </xref> . The primer sequences ( <xref ref-type="table" rid="pone-0086402-t001"> Table 1 </xref> ) were designed using the PrimerQuest tool (Integrated DNA Technologies, Coralville, IA, USA), employing the sequence from the 3&#x02032; untranslated region with the default settings. Gene expression was evaluated in 3 independent biological replicates, each one evaluated in 2 technical replicates. To compare gene expression between control and stressed/treated plants along a time course we performed statistical analysis using &#x00394;Ct values, as described by Yuan and colleges <xref rid="pone.0086402-Yuan1" ref-type="bibr"> [36] </xref> . After performing descriptive and exploratory data analysis, the effects of water regimes (control, moderate and severe <span class="trait">drought)</span> or ABA treatment, day time (ZT0 to ZT20), and their interactions on determined variables were analyzed using ANOVA. When results from the overall significance test led to rejection of the null hypothesis, Duncan&#x02019;s multiple range test for multiple comparisons among groups tests (5%) was performed. The above analyses were conducted using SAS 9.2 software (SAS Institute, Cary, NC).
					</p>
				
					<p>&nbsp;&nbsp;
						To characterize the effects of <span class="trait">drought</span> stress on the transcriptional regulatory networks of the soybean circadian clock effectors, genes encoding probable circadian clock components were identified through in-house alignments of the amino acid sequences of core <italic> Arabidopsis </italic> circadian clock proteins to the <italic> G. max </italic> genome. We identified potential soybean homologs of TOC1, CCA1/LHY, LUX, ELF4, CHE, GI, ZTL, and several PRR-like proteins ( <xref ref-type="table" rid="pone-0086402-t001"> Table 1 </xref> ). The soybean genome encodes at least two <italic> <span class="gene">LHY</span> </italic> and <italic> CCA1- </italic> like homologs named <italic> <span class="gene">LCL1</span> </italic> and <italic> <span class="gene">LCL2</span> </italic> ( <italic> LHY/CCA1- </italic> like 1 and 2), however it is not possible distinguish which gene (LCL1 or LCL2) corresponds to the Arabidopsis <span class="gene">LHY</span> or CCA1 <xref rid="pone.0086402-Liu1" ref-type="bibr"> [23] </xref> . In this study we evaluated specifically <italic> <span class="gene">GmLCL1</span> </italic> gene expression. The PRR proteins are highly similar in amino acid sequence, and the soybean genome is highly redundant <xref rid="pone.0086402-Severin1" ref-type="bibr"> [46] </xref> ; therefore, a phylogenetic analysis was used to identify the most probable homologs for each PRR class ( <xref ref-type="supplementary-material" rid="pone.0086402.s001"> Fig. S1 </xref> ). In total, 11 genes encoding potential circadian clock genes were identified for our analysis ( <xref ref-type="table" rid="pone-0086402-t001"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The great plasticity exhibited by plant responses to <span class="trait">drought</span> calls for studies that simulate field conditions in order to obtain information on the plants responses under more natural conditions. A recent study showed that molecular analyses under constant environmental conditions (eg. constant light, temperature) may give partial or misleading indications of the plant responses to the natural fluctuating conditions <xref rid="pone.0086402-Jnknp1" ref-type="bibr"> [47] </xref> . Therefore, studies showed that light and temperature cycles entrain the clock and thereby ensure appropriate phasing of circadian rhythms <xref rid="pone.0086402-Kikis1" ref-type="bibr"> [18] </xref> , <xref rid="pone.0086402-Thines1" ref-type="bibr"> [48] </xref> . In this context, to assess the impact of <span class="trait">drought</span> stress on the soybean circadian clock components, the expression profiles for each candidate soybean circadian clock gene were determined in V2-stage plants under simulated field light (14h light/10h dark) and temperature (28&#x000b0;C/20&#x000b0;C) cycles, over a 24 h time course under well-watered conditions (control) or one of two water-limited (stress) conditions. Prior to the implementation of the stress treatments, all of the plants were maintained under well-watered conditions, which corresponded to soil at 70% GH. After reaching the <span class="gene">V2</span> stage of development, in accordance with the definition of previous study <xref rid="pone.0086402-Fehr1" ref-type="bibr"> [31] </xref> , irrigation was removed for the water-limited plants. The moderate stress condition was obtained when the soil achieved 30% GH, corresponding to 3 days of water withholding, whereas the severe stress treatment corresponded to a GH of 15%, achieved with 5 days of water withholding. To overcome possible differences in gene expression due disparity in plants developmental stages we compared age-matched stress and control plants, since authors have demonstrated changes in gene expression between plants in different developmental stages <xref rid="pone.0086402-Baerenfaller1" ref-type="bibr"> [49] </xref> , <xref rid="pone.0086402-Jiang1" ref-type="bibr"> [50] </xref> . As we can see in <xref ref-type="fig" rid="pone-0086402-g001"> Figures 1 </xref> and <xref ref-type="fig" rid="pone-0086402-g002"> 2 </xref> , for some genes we observed differences in the expression of control plants from the moderate and severe stress, which points the importance of using age-matched plants for comparisons between control and stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Under normal well-watered conditions, the potential soybean circadian clock genes showed expression patterns compatible with the circadian oscillation described in literature for soybean <xref rid="pone.0086402-Liu1" ref-type="bibr"> [23] </xref> , <xref rid="pone.0086402-Sullivan1" ref-type="bibr"> [24] </xref> , <xref rid="pone.0086402-Thakare1" ref-type="bibr"> [26] </xref> and Arabidopsis <xref rid="pone.0086402-Pokhilko1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0086402-Lu1" ref-type="bibr"> [51] </xref> &#x02013; <xref rid="pone.0086402-Mizoguchi1" ref-type="bibr"> [53] </xref> . On the other hand, the plants exposed to the moderate and severe <span class="trait">drought</span> stress conditions showed significant changes in the diurnal expression of several potential core circadian clock genes. The expression of <italic> <span class="gene">GmTOC1</span> </italic> -like under normal hydration conditions was compatible to previous circadian oscillation reports <xref rid="pone.0086402-Liu1" ref-type="bibr"> [23] </xref> , <xref rid="pone.0086402-Thakare1" ref-type="bibr"> [26] </xref> with transcript levels for this gene peaked during the transition from day to night at the time corresponding to ZT12 ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> ). The moderate stress condition did not affect the timing of <italic> <span class="gene">GmTOC1-</span> </italic> like expression ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1A </xref> ), whereas lower amplitude of <italic> <span class="gene">GmTOC1-</span> </italic> like peak expression was observed 4 hours earlier under severe stress conditions ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). The evening-expressed <italic> <span class="gene">GmLUX</span> </italic> -like also displayed advanced-phase expression under severe hydration stress ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). Similar to <italic> <span class="gene">TOC1</span> </italic> , the <italic> LUX </italic> gene expressed in the evening were reported to be direct regulatory targets of CCA1 and <span class="gene">LHY</span> in <italic> Arabidopsis </italic> <xref rid="pone.0086402-Farr1" ref-type="bibr"> [19] </xref> , <xref rid="pone.0086402-Hazen1" ref-type="bibr"> [54] </xref> . The advance in the peak expression of these evening genes was similar to that observed in <italic> Arabidopsis </italic> mutants lacking both the CCA1 and <span class="gene">LHY</span> functions <xref rid="pone.0086402-Mizoguchi1" ref-type="bibr"> [53] </xref> . The early phase expression observed in the <italic> cca1 lhy </italic> double mutants reflected the absence of the normal repression activities of these two transcription factors, suggesting that <span class="trait">drought</span> stress conditions modify the regulation of soybean <italic> LHY/CCA1 </italic> -like gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine whether <span class="trait">drought</span> stress influenced the morning loop components of the soybean circadian clock, the expression of the <italic> <span class="gene">GmLCL1</span> </italic> -like gene was examined under conditions of <span class="trait">drought</span> stress. The expression of <italic> <span class="gene">GmLCL1</span> </italic> -like under conditions of normal hydration was comparable to that of its <italic> Arabidopsis </italic> ortholog ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> ), with the peak expression of this gene being observed at dawn (ZT0). Strikingly, <italic> <span class="gene">GmLCL1</span> </italic> -like expression was substantially reduced in those plants experiencing severe dehydration stress ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). Therefore, the advancement of evening-phased gene expression under severe <span class="trait">drought</span> stress might reflect, at least in part, the loss of normal <italic> <span class="gene">GmLCL1</span> </italic> -like expression.
					</p>
				
					<p>&nbsp;&nbsp;
						The lack of normal <italic> <span class="gene">GmLCL1</span> </italic> -like expression under <span class="trait">drought</span> conditions might also negatively affect the expression of <italic> <span class="gene">GmPRR9-</span> </italic> like and <italic> <span class="gene">GmPRR3-</span> </italic> like, as the transcript levels of these genes were markedly lower in the plants exposed to severe <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). The two <italic> Arabidopsis </italic> genes that participate in the morning loop, <italic> PRR9 </italic> and <italic> PRR7 </italic> , are directly upregulated through CCA1 and <span class="gene">LHY</span> <xref rid="pone.0086402-Farr1" ref-type="bibr"> [19] </xref> . Indeed, the genes we identified as <italic> <span class="gene">GmPRR9</span> </italic> -like and <italic> <span class="gene">GmPRR3</span> </italic> -like could potentially represent these components in the soybean clock system.
					</p>
				
					<p>&nbsp;&nbsp;
						Although <span class="trait">drought</span> stress potentially influences Gm <italic> LCL1- </italic> like regulation, a potential contributor to the strong reduction in <italic> <span class="gene">LCL1</span> </italic> expression is the severe drought-induced loss of a <italic> <span class="gene">GmLCL1-</span> </italic> like activator. <italic> <span class="gene">GmELF4-</span> </italic> like is a putative activator of <italic> <span class="gene">GmLCL1-</span> </italic> like because <italic> Arabidopsis ELF4 </italic> is required for the phytochrome-mediated light induction of <italic> AtCCA1 </italic> and <italic> AtLHY </italic> <xref rid="pone.0086402-Kikis1" ref-type="bibr"> [18] </xref> . Consistent with this idea, the <italic> <span class="gene">GmELF4-</span> </italic> like expression was low in the plants exposed to severe <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> ). In <italic> Arabidopsis </italic> , ELF4 is repressed through the action of a multiprotein complex containing CCA1 and <span class="gene">LHY</span> and the transcription factors FAR1 and FHY3 <xref rid="pone.0086402-Kikis1" ref-type="bibr"> [18] </xref> , <xref rid="pone.0086402-McWatters1" ref-type="bibr"> [55] </xref> , <xref rid="pone.0086402-Li1" ref-type="bibr"> [56] </xref> . FAR1 and FHY3 also activate ELF4 when CCA1 and <span class="gene">LHY</span> are absent <xref rid="pone.0086402-Li1" ref-type="bibr"> [56] </xref> . Thus, severe <span class="trait">drought</span> stress could negatively affect the function of the soybean orthologs FAR1 and/or FHY3, resulting in limited <italic> <span class="gene">GmELF4-</span> </italic> like expression. <xref ref-type="supplementary-material" rid="pone.0086402.s002"> Fig. S2 </xref> shows a proposed scheme for the effect of severe <span class="trait">drought</span> on the expression of <italic> <span class="gene">GmLCL</span> </italic> -like, <italic> <span class="gene">GmTOC1</span> </italic> -like, <italic> <span class="gene">GmLUX</span> </italic> -like, and <italic> <span class="gene">GmELF4</span> </italic> -like genes.
					</p>
				
					<p>&nbsp;&nbsp;
						A general reduction in the amplitude of expression was observed after severe <span class="trait">drought</span> stress for most of the clock genes evaluated, including the <italic> <span class="gene">GmPRR3-</span> </italic> like, <italic> <span class="gene">GmPRR7-</span> </italic> like, <italic> <span class="gene">GmPRR9-</span> </italic> like, <italic> <span class="gene">GmGI-</span> </italic> like, <italic> <span class="gene">GmZTL-</span> </italic> like, and <italic> <span class="gene">GmCHE-</span> </italic> like genes ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). This observation is consistent with the effect of other abiotic stresses on the circadian rhythms in plants. Cold stress reduces the amplitude of cycles for clock components and reduces/disrupts the cycles of output genes in Arabidopsis, poplar, and chestnut <xref rid="pone.0086402-Bieniawska1" ref-type="bibr"> [27] </xref> , <xref rid="pone.0086402-Ibez1" ref-type="bibr"> [57] </xref> , <xref rid="pone.0086402-Ramos1" ref-type="bibr"> [58] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Unlike the strong repression observed under the severe stress condition, the <italic> <span class="gene">GmLCL1</span> </italic> -like expression under moderate stress was slightly higher relative to that in the control plants at ZT0 and ZT4, with the most noticeable effect occurring at ZT0 and ZT4 ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1A </xref> ). This increase in <italic> <span class="gene">LCL1</span> </italic> during early stress stages might be involved in the signaling and activation of <span class="trait">drought</span> response mechanisms. CCA1 directs the circadian regulation of the <italic> CBF/DREB </italic> genes in <italic> Arabidopsis </italic> <xref rid="pone.0086402-Fowler1" ref-type="bibr"> [28] </xref> , and CCA1 participates in the transcription factor complex that promotes the acute induction of CBF/DREB genes under cold stress <xref rid="pone.0086402-Dong1" ref-type="bibr"> [59] </xref> . A similar system might be induced in response to <span class="trait">drought,</span> as cross-talk between cold and dehydration response pathways is observed.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Jumonji </italic> genes are upregulated by <span class="trait">drought</span> in soybean <xref rid="pone.0086402-Rodrigues1" ref-type="bibr"> [34] </xref> and <italic> Arachis hypogaea </italic> <xref rid="pone.0086402-Govind1" ref-type="bibr"> [60] </xref> . The Jumonji transcription factors act as demethylases, controlling chromatin structure and, thus, gene expression <xref rid="pone.0086402-Chen1" ref-type="bibr"> [61] </xref> . The silencing of an <italic> A. hypogaea Jumonji </italic> ortholog in tobacco improved <span class="trait">drought</span> tolerance <xref rid="pone.0086402-Govind1" ref-type="bibr"> [60] </xref> . The <italic> Arabidopsis JMJD5 </italic> (or <italic> JMJ30 </italic> ) gene shows circadian oscillation and acts as a regulator of period length <xref rid="pone.0086402-Lu1" ref-type="bibr"> [51] </xref> . It has been proposed that <italic> JMJD5 </italic> and <italic> AtTOC1 </italic> act in combination to control the circadian clock and that both genes are repressed through the activity of <italic> AtCCA1 </italic> and <italic> AtLHY </italic> <xref rid="pone.0086402-Jones1" ref-type="bibr"> [62] </xref> . The soybean <italic> <span class="gene">GmJMJ-</span> </italic> like gene exhibited strong diurnal expression ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> ). Severe <span class="trait">drought</span> treatment induced the upregulation of <italic> <span class="gene">GmJMJ-</span> </italic> like expression, particularly at ZT4, and the downregulation of <italic> <span class="gene">GmJMJ-</span> </italic> like expression at ZT12, relative to the control plants ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1B </xref> ). Whether these changes in expression during <span class="trait">drought</span> stress might be important for the regulation of <span class="trait">drought</span> responses remains unknown. However, epigenetic mechanisms, such as DNA methylation and histone modification, play a crucial role in regulating gene expression during plant responses to environmental stress <xref rid="pone.0086402-Boyko1" ref-type="bibr"> [63] </xref> &#x02013; <xref rid="pone.0086402-Wang1" ref-type="bibr"> [66] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						For crop species, such as soybean, corn or cotton, productivity is directly connected with flowering intensity/stability to maximize number of seeds produced. Clock genes expression is directly controlled by day length and temperature, interacting with other environmental factors (water status, soil <span class="trait">fertility,</span> presence of pathogens, etc) to establish flowering initiation, intensity and duration. Previous studies were able to increase soybean productivity by overexpressing the AtBBX32 transcription factor <xref rid="pone.0086402-Preuss1" ref-type="bibr"> [67] </xref> . They demonstrated that the constitutive expression of AtBBX32 in soybean altered the abundance of transcript levels of the soybean clock genes <italic> <span class="gene">GmTOC1</span> and LHY-CCA1-like2 </italic> ( <italic> <span class="gene">GmLCL2</span> </italic> ), and by altering the abundance of circadian clock genes during the transition from dark to light, the timing of critical phases of reproductive development was altered <xref rid="pone.0086402-Preuss1" ref-type="bibr"> [67] </xref> . Flower, pod, and seed number were increased, where the authors proposed that it was caused by changes in the timing of reproductive development in the transgenic soybean that lead to the increased duration of the pod and seed development period. <span class="trait">Drought</span> is well known to alter time of flowering in soybean where maintenance/abortion of flowers/seeds will depend on the intensity and duration of the stress. Our results are the first to show in soybean that water deficit alters the expression of circadian clock genes. Thus, in this context, an understanding of the means by which the circadian clock gene expression is altered by this environmental stress may give clues to how we can genetically manipulate the circadian clock to reduce <span class="trait">yield</span> losses during <span class="trait">drought</span> events.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of <italic> <span class="gene">GmDREB2-</span> </italic> like suggested that this gene is only involved during the early responses to <span class="trait">drought</span> stress. Under moderate hydration stress, the expression of <italic> <span class="gene">GmDREB2-</span> </italic> like was induced ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2A </xref> ), however, a significant reduction in <italic> <span class="gene">GmDREB2-</span> </italic> like expression was observed under severe conditions ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2B </xref> ). In contrast, <italic> <span class="gene">GmDREB1-</span> </italic> like and <italic> <span class="gene">GmDREB3-</span> </italic> like gene expression induction was observed under this treatment ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of <italic> DREB </italic> genes under cold stress is suppressed upon the expression of PRR transcription factors that also act in the core circadian oscillator <xref rid="pone.0086402-Dong1" ref-type="bibr"> [59] </xref> . Because significant cross-talk occurs between cold and <span class="trait">drought</span> responses, these expression patterns indicate similar regulatory relationships in soybean. The peak expression of the <italic> <span class="gene">GmPRR-</span> </italic> like genes corresponds with the reduced expression of the <italic> <span class="gene">GmDREB1</span> </italic> -like gene (ZT8) ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> , <xref ref-type="fig" rid="pone-0086402-g002"> 2 </xref> and <xref ref-type="fig" rid="pone-0086402-g003"> 3 </xref> ). Additionally, the reduced expression of some <italic> <span class="gene">GmPRR-</span> </italic> like genes was observed under <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0086402-g001"> Fig. 1 </xref> ), which is consistent with a mechanism for promoting DREB expression by reducing expression of these repressors.
					</p>
				
					<p>&nbsp;&nbsp;
						We also evaluated the expression of a <italic> <span class="gene">GmbZIP1</span> </italic> transcription factor that belongs to the AREB subfamily and is reported to be an abiotic stress- and ABA-responsive gene <xref rid="pone.0086402-Gao1" ref-type="bibr"> [72] </xref> . The authors suggest that the overexpression of <italic> ZIP1 </italic> in <italic> Arabidopsis </italic> and wheat increases <span class="trait">drought</span> tolerance and improves the ABA-mediated control of stomatal aperture in plants. The results obtained in the present study show diurnal oscillation for this gene expression in response to moderate stress at dawn (ZT0) compared with the control treatment ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2A </xref> ). In addition, the <italic> <span class="gene">GmbZIP1</span> </italic> expression was similar to that of the <italic> <span class="gene">GmDREB-</span> </italic> like genes, suggesting a putative mechanism involving transcription factors in plant stress defenses that are activated in the early morning before sunlight.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Arabidopsis </italic> orthologs for the <italic> <span class="gene">GmRemorin-</span> </italic> like, <italic> <span class="gene">GmGols-</span> </italic> like, and <italic> <span class="gene">GmRAB18-</span> </italic> like genes used in our study showed diurnal oscillation and ABA upregulation <xref rid="pone.0086402-Mizuno1" ref-type="bibr"> [35] </xref> . This is consistent with the results of previous studies of the <italic> Arabidopsis </italic> Remorin ortholog showing the upregulation of gene expression in response to <span class="trait">drought</span> <xref rid="pone.0086402-Raffaele1" ref-type="bibr"> [8] </xref> . Our results show the upregulation of the <italic> <span class="gene">GmRemorin-</span> </italic> like gene under both moderate and severe <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2 </xref> ). Interestingly, this gene expression was much higher in response to severe stress and showed a significant diurnal oscillation, with lower expression values between ZT8 and ZT16 ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2B </xref> ), whereas the expression differences were not significant under the control and stressed treatments at ZT0. Studies that characterize the function of Remorins are important to understand <span class="trait">drought</span> tolerance mechanisms, as Remorins have only been identified in plants <xref rid="pone.0086402-Raffaele1" ref-type="bibr"> [8] </xref> . The Remorins have a hydrophilic profile and attach to the plasma membrane. The evolution of the cell wall composition is likely associated with the emergence of different classes of proteins that maintain cell membrane/cell wall integrity and the acquisition of vascular tissue. Thus, it is reasonable to propose that these proteins played an important role during the plant colonization of land in that the chemical adaptation of the cell wall was vital to cope with the particularly rough selection pressure in a dry environment.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AtRAB18 </italic> is a well-known <span class="trait">drought-</span> and ABA-responsive gene used in several studies as a marker for <span class="trait">drought</span> and ABA treatments <xref rid="pone.0086402-Wang1" ref-type="bibr"> [66] </xref> , <xref rid="pone.0086402-Rushton1" ref-type="bibr"> [77] </xref> , <xref rid="pone.0086402-Ying1" ref-type="bibr"> [78] </xref> . <italic> RAB18 </italic> encodes a protein involved in membrane vesicle transport, and its function has been associated with the recycling of molecules, the removal of existing molecules from cellular compartments, and their replacement with newly synthesized molecules during stress adaptation <xref rid="pone.0086402-Agarwal1" ref-type="bibr"> [7] </xref> . The <italic> <span class="gene">GmRAB18</span> </italic> gene showed diurnal oscillations under the stress conditions, with higher expression during the day between ZT0 and ZT4, in moderate stress ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2A </xref> and <xref ref-type="fig" rid="pone-0086402-g003"> 3 </xref> ), and ZT4 and ZT12 in severe stress ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2B </xref> ). The moderate <span class="trait">drought</span> stress applied appeared to specifically induce the peak expression of <italic> <span class="gene">GmRAB18-</span> </italic> like ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2A </xref> ), whereas severe stress caused a significantly higher expression level that was approximately 2000 times greater than that in the control plants ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2B </xref> ). The expression profile of the soybean gene suggests that the product of <italic> <span class="gene">GmRAB18-</span> </italic> like is important during the day because the moderate stress treatment specifically amplified the daytime expression.
					</p>
				
					<p>&nbsp;&nbsp;
						The phytohormone ABA is integral for the response to stressful environments and for the regulation of growth and development <xref rid="pone.0086402-Zhu1" ref-type="bibr"> [80] </xref> . To understand the effects of ABA as a <span class="trait">drought</span> stress response component in soybean and to link the activity of this phytohormone with the diurnal expression patterns observed under <span class="trait">drought</span> stress, selected soybean genes were characterized for their responsiveness to exogenous ABA treatment. Plants grown under the control environmental conditions were sprayed with a solution of 100 mM ABA just after the start of the light period (ZT3), and gene expression was determined at 4 h intervals thereafter, beginning at ZT4. We selected representative ABA- and drought-responsive genes ( <italic> <span class="gene">GmRAB18-</span> </italic> like, <italic> <span class="gene">GmGOLS-</span> </italic> like, <italic> <span class="gene">GmABAR-</span> </italic> like, and <italic> <span class="gene">GmbZIP</span> </italic> ) as well as several circadian clock genes ( <italic> <span class="gene">GmTOC1-</span> </italic> like, <italic> <span class="gene">GmLUX-</span> </italic> like, <italic> <span class="gene">GmLCL1-</span> </italic> like, <italic> <span class="gene">GmELF4-</span> </italic> like, <italic> <span class="gene">GmPRR3-</span> </italic> like, <italic> <span class="gene">GmPRR9-</span> </italic> like, and <italic> <span class="gene">GmJMJ-</span> </italic> like) for the analyses. All of these genes contain putative ABA-responsive elements in their promoter regions ( <xref ref-type="supplementary-material" rid="pone.0086402.s004"> Tables S1 </xref> and <xref ref-type="supplementary-material" rid="pone.0086402.s005"> S2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The candidate ABA- and drought-related genes <italic> <span class="gene">GmRAB18</span> </italic> -like, <italic> <span class="gene">GmDREB2</span> </italic> -like and <italic> <span class="gene">GmbZIP</span> </italic> -like responded to ABA application ( <xref ref-type="fig" rid="pone-0086402-g004"> Fig. 4 </xref> ; <xref ref-type="supplementary-material" rid="pone.0086402.s008"> Table S5 </xref> ). On the other hand, <italic> <span class="gene">GmABAR-</span> </italic> like, pointed as an ABA receptor for some authors, showed no significant change in its expression levels or patterns following the ABA treatment ( <xref ref-type="supplementary-material" rid="pone.0086402.s003"> Fig. S3 </xref> ). Thus, considering that <italic> <span class="gene">GmABAR-</span> </italic> like was nonresponsive to both moderate and severe <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0086402-g002"> Fig. 2 </xref> and <xref ref-type="fig" rid="pone-0086402-g003"> 3 </xref> ), it is likely that this soybean homolog is not involved in the ABA-mediated responses to <span class="trait">drought.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Although we observed that ABA treatment induced ABA- and drought-related genes under our experimental conditions, we could not detect a strong effect on the circadian clock genes evaluated, with the exception of the evening genes <italic> <span class="gene">GmJumonji-like</span> </italic> and <italic> <span class="gene">GmTOC1-like</span> </italic> ( <xref ref-type="fig" rid="pone-0086402-g004"> Fig. 4 </xref> ; Fig.S3; <xref ref-type="supplementary-material" rid="pone.0086402.s008"> Table S5 </xref> ). The <italic> <span class="gene">GmTOC1-</span> </italic> like responses to ABA were of particular interest, as exogenous ABA treatment induces the expression of <italic> Arabidopsis <span class="gene">TOC1</span> </italic> <xref rid="pone.0086402-Legnaioli1" ref-type="bibr"> [10] </xref> . Furthermore, <italic> Arabidopsis </italic> <span class="gene">TOC1</span> participates in a feedback loop with ABAR to modulate ABA responsiveness, a process which is involved in <span class="trait">drought</span> tolerance <xref rid="pone.0086402-Legnaioli1" ref-type="bibr"> [10] </xref> . However <italic> <span class="gene">GmTOC1</span> </italic> -like expression showed a significant decrease in response to ABA treatment at ZT16 ( <xref ref-type="fig" rid="pone-0086402-g004"> Fig. 4A </xref> ), requiring further investigation of this gene responses to ABA.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Model of the impact of severe <span class="trait">drought</span> on circadian clock genes. </bold> Model of the impact of severe <span class="trait">drought</span> stress on <italic> <span class="gene">GmLCL1</span> </italic> , <italic> <span class="gene">GmTOC1</span> </italic> , <italic> <span class="gene">GmLUX</span> </italic> and <italic> <span class="gene">GmELF4-like</span> </italic> gene expression.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptional response of soybean to thiamethoxam seed treatment in the presence and absence of <span class="trait">drought</span> stress</h2>
			<h3>PMID:25467808</h3>
				

				
					<h3>PMCID:4265413</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Stamm </surname>  <given-names> Mitchell D </given-names>  </name>  <address>  <email> mitchell.stamm@basf.com </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Enders </surname>  <given-names> Laramy S </given-names>  </name>  <address>  <email> laramy.enders@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Donze-Reiner </surname>  <given-names> Teresa J </given-names>  </name>  <address>  <email> tdonze@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baxendale </surname>  <given-names> Frederick P </given-names>  </name>  <address>  <email> fbaxendale1@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Siegfried </surname>  <given-names> Blair D </given-names>  </name>  <address>  <email> bsiegfried1@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heng-Moss </surname>  <given-names> Tiffany M </given-names>  </name>  <address>  <email> thengmoss2@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <aff id="Aff6"> Department of Entomology, University of Nebraska, 103 Entomology Hall, Lincoln, NE 68583 USA  </aff>
				<br/> 
				年份：<day> 24 </day>  <month> 11 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Neonicotinoid insecticides are widely known for their broad-spectrum control of arthropod pests. Recently, their effects on plant physiological mechanisms have been characterized as producing a stress shield, which is predicted to enhance tolerance to adverse conditions. Here we investigate the molecular underpinnings of the stress shield concept using the neonicotinoid thiamethoxam in two separate experiments that compare gene expression. We hypothesized that the application of a thiamethoxam seed treatment to soybean would alter the expression of genes involved in plant defensive pathways and general stress response in later vegetative growth. First, we used next-generation sequencing to examine the broad scale transcriptional effects of the thiamethoxam seed treatment at three vegetative stages in soybean. Second, we selected ten target genes associated with plant defense pathways in soybean and examined the interactive effects of thiamethoxam seed treatment and <span class="trait">drought</span> stress on expression using qRT-PCR. RESULTS: Direct comparison of thiamethoxam-treated and untreated soybeans revealed minor transcriptional differences. However, when examined across vegetative stages, the thiamethoxam seed treatment induced substantial transcriptional changes that were not observed in untreated plants. Genes associated with photosynthesis, carbohydrate and lipid metabolism, development of the cell wall and membrane organization were uniquely upregulated between vegetative stages in thiamethoxam-treated plants. In addition, several genes associated with phytohormone and oxidative stress responses were downregulated between vegetative stages. When we examined the expression of a subset of ten genes associated with plant defense and stress response, the application of thiamethoxam was found to interact with <span class="trait">drought</span> stress by enhancing or repressing expression. In <span class="trait">drought</span> stressed plants, thiamethoxam induced (upregulated) expression of a thiamine biosynthetic enzyme (THIZ2) and gibberellin regulated protein (GRP), but repressed (downregulated) the expression of an apetala 2 <span class="gene">(GmDREB2A;2),</span> lipoxygenase (LIP), and SAM dependent carboxyl methyltransferase (SAM). CONCLUSIONS: We found evidence that a thiamethoxam seed treatment alters the expression soybean genes related to plant defense and stress response both in the presence and absence of <span class="trait">drought</span> stress. Consistent with the thiamethoxam stress shield concept, several genes associated with phytohormones showed enhanced expression in <span class="trait">drought</span> stressed plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Direct comparison of thiamethoxam-treated and untreated soybeans revealed minor transcriptional differences. However, when examined across vegetative stages, the thiamethoxam seed treatment induced substantial transcriptional changes that were not observed in untreated plants. Genes associated with photosynthesis, carbohydrate and lipid metabolism, development of the cell wall and membrane organization were uniquely upregulated between vegetative stages in thiamethoxam-treated plants. In addition, several genes associated with phytohormone and oxidative stress responses were downregulated between vegetative stages. When we examined the expression of a subset of ten genes associated with plant defense and stress response, the application of thiamethoxam was found to interact with <span class="trait">drought</span> stress by enhancing or repressing expression. In <span class="trait">drought</span> stressed plants, thiamethoxam induced (upregulated) expression of a thiamine biosynthetic enzyme (THIZ2) and gibberellin regulated protein (GRP), but repressed (downregulated) the expression of an apetala 2 <span class="gene">(GmDREB2A;2),</span> lipoxygenase (LIP), and SAM dependent carboxyl methyltransferase (SAM).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress had several negative effects on the physiological parameters measured in soybean. First, the effects of <span class="trait">drought</span> stress were observed in delayed vegetative growth of both thiamethoxam-treated and untreated plants relative to unstressed plants. <span class="trait">Drought</span> stressed plants did not develop past the <span class="gene">V2</span> stage in contrast to unstressed plants that reached the V4 stage (13d), indicating a reduction in developmental rate. Second, <span class="trait">drought</span> stress caused a significant reduction in plant biomass. When averaged across all thiamethoxam-treated and untreated control plants, <span class="trait">drought</span> stressed plants showed a 1.4, 3.5, and 6.1-fold reduction in biomass compared to unstressed plants at 7, 10, and 13d, respectively (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> ). Third, <span class="trait">drought</span> stress had a negative effect on plant height. <span class="trait">Drought</span> stressed thiamethoxam-treated and untreated control plants were on average 0.6, 5.0, and 10.4&#x000a0;cm shorter than unstressed plants at 7, 10, and 13d, respectively (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> ). Taken together these results demonstrate that <span class="trait">drought</span> stress had a significant impact on soybean physiology and that the intensity of stress progressed over the three sampling dates both in thiamethoxam-treated and untreated control plants. However, we did not find that the thiamethoxam treatment enhanced overall plant vigor or response to <span class="trait">drought</span> stress. There were no significant differences in shoot height and plant biomass between thiamethoxam-treated and untreated control plants in the presence and absence of <span class="trait">drought</span> stress (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> ). <table-wrap id="Tab6"> <label> Table 6 </label> <caption> <p> <bold> The effects of <span class="trait">drought</span> stress and seed treatment on plant biomass (wet &#x02013; dry weight) and shoot height (&#x000b1; SEM) at the three time points (7, 10, and 13d) </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress significantly altered gene expression of the ten target genes in at least one time point. A significant seed treatment and <span class="trait">drought</span> stress interaction was identified in the expression of several target genes. (Tables&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> and <xref rid="Tab8" ref-type="table"> 8 </xref> ). Specifically, the effect of <span class="trait">drought</span> stress on gene expression varied between thiamethoxam-treated and untreated control plants at 7d <span class="gene">(GmDREB2A;2,</span> LIP, and SAM) and 13d (THIZ2, GRP, and <span class="gene">GmDREB2A;2),</span> as indicated by a significant seed treatment &#x000d7; <span class="trait">drought</span> stress interaction (Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). We found patterns consistent with increased expression in treated plants under stress and also decreased expression in treated plants under stress, which demonstrate interactive effects between thiamethoxam and <span class="trait">drought</span> stress. <table-wrap id="Tab7"> <label> Table 7 </label> <caption> <p> <bold> Summaries of ANOVA with </bold> <bold> <italic> P- </italic> </bold> <bold> values for the overall effects of the stress <span class="trait">(drought</span> stress and no stress), seed treatment (thiamethoxam-treated and untreated control) and the two-way interaction of stress and seed treatment on target genes at the three time points (7, 10 and, 13d) </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of <span class="gene">GmDREB2A;2,</span> a drought-responsive transcription factor [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ], was affected by thiamethoxam. This gene was also shown to be stress-responsive and had significant interactions with the seed treatment at 7 and 13d ( <italic> F </italic> &#x02009;=&#x02009;10.76; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0112 and <italic> F </italic> &#x02009;=&#x02009;13.83; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0059, respectively; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). In thiamethoxam-treated plants at 7d, <span class="gene">GmDREB2A;2</span> expression was repressed by -1.3-fold under <span class="trait">drought</span> stress compared to untreated control plants, which showed a significant 9.7 fold upregulation under stress ( <italic> t </italic> &#x02009;=&#x02009;4.38; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0101; Table&#x000a0; <xref rid="Tab8" ref-type="table"> 8 </xref> ). This similar expression pattern was also observed at 13d ( <italic> t </italic> &#x02009;=&#x02009;3.75; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0563) with <span class="gene">GmDREB2A;2</span> being repressed in thiamethoxam-treated plants under <span class="trait">drought</span> stress by -1.7-fold compared to the untreated controls, which showed a significant 4.1-fold upregulation (Table&#x000a0; <xref rid="Tab8" ref-type="table"> 8 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Thiamethoxam also affected the expression of THIZ2, GRP, LIP, and SAM, which are associated with phytohormones. THIZ2, a thiamine biosynthetic enzyme, had a significant seed treatment &#x000d7; <span class="trait">drought</span> stress interaction at 13d ( <italic> F </italic> &#x02009;=&#x02009;14.25; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0054; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ), which was further investigated by comparing gene expression between drought-stressed and unstressed plants using Fisher&#x02019;s LSD test (Table&#x000a0; <xref rid="Tab8" ref-type="table"> 8 </xref> ). Specifically, THIZ2 expression was upregulated under <span class="trait">drought</span> stress in thiamethoxam-treated plants, but significantly downregulated in the untreated controls at 13d ( <italic> t </italic> &#x02009;=&#x02009;4.95; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0049; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). GRP, a GA regulated protein, also had a significant seed treatment x <span class="trait">drought</span> stress interaction at 13d ( <italic> F </italic> &#x02009;=&#x02009;10.25; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0126; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). This gene was downregulated in response to <span class="trait">drought</span> stress in both the thiamethoxam-treated and untreated control plants. However, the magnitude of the effect of <span class="trait">drought</span> stress on <span class="gene">GRP</span> expression in thiamethoxam-treated plants was less than observed in untreated controls (-1.6-fold compared to -2.6 fold) ( <italic> t </italic> &#x02009;=&#x02009;4.05; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0155 and <italic> t </italic> &#x02009;=&#x02009;8.58; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0001, respectively; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ), which suggests thiamethoxam caused a slight upregulation of expression under <span class="trait">drought</span> stress. Lastly, LIP and SAM, a lipoxygenase and SAM dependent carboxyl methyltransferase (respectively), displayed a significant seed treatment &#x000d7; <span class="trait">drought</span> stress interaction at 7d ( <italic> F </italic> &#x02009;=&#x02009;37.65; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0003 and <italic> F </italic> &#x02009;=&#x02009;114.87; df&#x02009;=&#x02009;1, 8; <italic> P </italic> &#x02009;=&#x02009;0.0048, respectively; Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). LIP, a lipoxygenase associated with JA, showed decreased expression (-6.0-fold) under <span class="trait">drought</span> stress in thiamethoxam-treated plants compared ( <italic> t </italic> &#x02009;=&#x02009;7.14; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0004) to a 1.5-fold upregulation in the untreated control under <span class="trait">drought</span> stress (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). SAM expression followed a similar expression pattern and was significantly repressed (-4.2-fold) in thiamethoxam-treated plants under <span class="trait">drought</span> stress ( <italic> t </italic> &#x02009;=&#x02009;4.52; df&#x02009;=&#x02009;8; <italic> P </italic> &#x02009;=&#x02009;0.0084) compared to a 1.3-fold upregulation in the untreated controls (Table&#x000a0; <xref rid="Tab8" ref-type="table"> 8 </xref> ). <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> <bold> Expression pattern of <span class="gene">GmDREB2A;2</span> (apetala 2). </bold> There was significant seed treatment x <span class="trait">drought</span> stress interaction under no stress (NS) and <span class="trait">drought</span> stress (S) at 7 and 13d. Under <span class="trait">drought</span> stress, the expression of <span class="gene">GmDREB2A;2</span> was downregulated in thiamethoxam-treated (Thx) plants and upregulated in the untreated control (Unt).
					</p>
				
					<p>&nbsp;&nbsp;
						An aspartic protease ( <italic> <span class="gene">Glyma11g03500</span> </italic> ) was the only gene shared across all three thiamethoxam-treated vegetative stage comparisons (VC-V2, VC-V4, and V2-V4; Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Aspartic proteases have been shown to be an active part of inducible <span class="trait">resistance</span> mechanisms in response to plant pathogens, like <italic> Pseudomonas syringae, </italic> in <italic> A. thaliana </italic> [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Its response to thiamethoxam and importance in soybean suggests this gene is an excellent candidate for future studies. We were also interested in genes associated with phytohormones and oxidative enzymes due to their importance in plant defense response [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. Previously, imidacloprid has been shown to induce genes involved with the salicylic acid pathway [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ], an important defense mechanism for environmental stress protection [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. We found some evidence to support our hypothesis that thiamethoxam will affect plant defenses and oxidation-reduction mechanisms; however, the majority of these genes were downregulated in thiamethoxam-treated plants. It was also revealed that most of these genes were similarly responsive in untreated control plants, indicating that thiamethoxam did not significantly alter the expression of the major defensive pathways of interest.
					</p>
				
					<p>&nbsp;&nbsp;
						The effects of the seed treatment and <span class="trait">drought</span> stress were only quantified during vegetative stages of soybean development, and therefore long-term impacts were not evaluated in this study. In addition to the negative impacts of stress on physiology, interactions between seed treatment and <span class="trait">drought</span> stress were also observed at the molecular level (Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). We found five cases to support our hypothesis that thiamethoxam affects gene expression under <span class="trait">drought</span> stress. Of particular interest was <span class="gene">GmDREB2A;2,</span> which has been documented as inducing the expression of dehydration-responsive elements in <italic> A. thaliana </italic> under <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ]. In our study, the expression of this gene was consistently repressed across time points by thiamethoxam and upregulated in the untreated controls under <span class="trait">drought</span> stress (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). Further studies with other drought-responsive genes are needed to investigate the stress shield concept. If reduced expression similar to that observed for <span class="gene">GmDREB2A;2</span> in the current study is found in additional drought-responsive genes, this may suggest thiamethoxam could limit a plant&#x02019;s ability to perceive and respond to stress.
					</p>
				
					<p>&nbsp;&nbsp;
						This study provides insight into the effect of a thiamethoxam seed treatment on gene expression in soybean. Transcriptional changes observed across vegetative stages in thiamethoxam-treated plants were primarily stage-specific. When <span class="trait">drought</span> stress was imposed on the soybean, gene expression was affected by both thiamethoxam and <span class="trait">drought</span> stress. Interactive effects between these two treatments were also observed, with genes in thiamethoxam-treated plants showing both induced (THIZ2 and GRP) and repressed <span class="gene">(GmDREB2A;2,</span> LIP, and SAM) expression under <span class="trait">drought</span> stress relative to untreated plants. In this study, thiamethoxam had a minimal effect on the upregulation of genes associated with phytohormones previously associated with the plant stress shield concept [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. However, thiamethoxam was not found to enhance plant health in the presence or absence of <span class="trait">drought</span> stress, which may in part explain this minimal induction of defense mechanisms.
					</p>
				
					<p>&nbsp;&nbsp;
						For gene expression analysis the first and second trifoliates were removed from each plant and handled in the same manner as described previously for the RNA-Seq study. Ten target genes were selected based on possible response to thiamethoxam or <span class="trait">drought</span> stress, including a lipoxygenase (LIP), SAM dependent carboxyl methyltransferase (SAM), aspartic protease (AP), apetala 2 <span class="gene">(GmDREB2A;2),</span> cytochrome <span class="gene">P450</span> (P450), two thiamine biosynthetic enzymes (THIZ1 and THIZ2), gibberellin regulated protein (GRP), WRKY51 DNA-binding protein 51 (WKRY51) and cartenoid oxygenase (CAR). LIP and SAM are jasmonic and salicylic acid-associated genes, respectively [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ] and <span class="gene">GmDREB2A;2</span> was previously documented to be <span class="trait">drought</span> stress responsive in soybean [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide transcriptional analysis of two soybean genotypes under dehydration and rehydration conditions</h2>
			<h3>PMID:24093224</h3>
				

				
					<h3>PMCID:3827939</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Chen </surname>  <given-names> Li M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ccllmm_008@aliyun.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A2">  <name>  <surname> Zhou </surname>  <given-names> Xin A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhouocri@sina.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A3">  <name>  <surname> Li </surname>  <given-names> Wen B </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> wenbinli@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Chang </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> weich0@foxmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Zhou </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhourong@oilrops.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Wang </surname>  <given-names> Cheng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> wcgigi@163.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Sha </surname>  <given-names> Ai H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> aihuasha@163.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Shan </surname>  <given-names> Zhi H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhihuishan@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Zhang </surname>  <given-names> Chan J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhangchanjuan@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Qiu </surname>  <given-names> De Z </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> qiudz6677@yahoo.cn </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Yang </surname>  <given-names> Zhong L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yangzhonglu@126.com </email>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Chen </surname>  <given-names> Shui L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> chenshuilianjy@163.com </email>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 9 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is an important crop that provides valuable proteins and oils for human use. Because soybean growth and development is extremely sensitive to water deficit, quality and crop yields are severely impacted by <span class="trait">drought</span> stress. In the face of limited water resources, drought-responsive genes are therefore of interest. Identification and analysis of dehydration- and rehydration-inducible differentially expressed genes (DEGs) would not only aid elucidation of molecular mechanisms of stress response, but also enable improvement of crop stress tolerance via gene transfer. Using Digital Gene Expression Tag profiling (DGE), a new technique based on Illumina sequencing, we analyzed expression profiles between two soybean genotypes to identify drought-responsive genes. RESULTS: Two soybean genotypes - drought-tolerant Jindou21 and drought-sensitive Zhongdou33 - were subjected to dehydration and rehydration conditions. For analysis of DEGs under dehydration conditions, 20 cDNA libraries were generated from roots and leaves at two different time points under well-watered and dehydration conditions. We also generated eight libraries for analysis under rehydration conditions. Sequencing of the 28 libraries produced 25,000-33,000 unambiguous tags, which were mapped to reference sequences for annotation of expressed genes. Many genes exhibited significant expression differences among the libraries. DEGs in the drought-tolerant genotype were identified by comparison of DEGs among treatments and genotypes. In Jindou21, 518 and 614 genes were differentially expressed under dehydration in leaves and roots, respectively, with 24 identified both in leaves and roots. The main functional categories enriched in these DEGs were metabolic process, response to stresses, plant hormone signal transduction, protein processing, and plant-pathogen interaction pathway; the associated genes primarily encoded transcription factors, protein kinases, and other regulatory proteins. The seven most significantly expressed (|log2 ratio| ≥ 8) genes - <span class="gene">Glyma15g03920,</span> <span class="gene">Glyma05g02470,</span> <span class="gene">Glyma15g15010,</span> <span class="gene">Glyma05g09070,</span> <span class="gene">Glyma06g35630,</span> <span class="gene">Glyma08g12590,</span> and <span class="gene">Glyma11g16000</span> - are more likely to determine <span class="trait">drought</span> stress tolerance. The expression patterns of eight randomly-selected genes were confirmed by quantitative RT-PCR; the results of QRT-PCR analysis agreed with transcriptional profile data for 96 out of 128 (75%) data points. CONCLUSIONS: Many soybean genes were differentially expressed between drought-tolerant and drought-sensitive genotypes. Based on GO functional annotation and pathway enrichment analysis, some of these genes encoded transcription factors, protein kinases, and other regulatory proteins. The seven most significant DEGs are candidates for improving soybean <span class="trait">drought</span> tolerance. These findings will be helpful for analysis and elucidation of molecular mechanisms of <span class="trait">drought</span> tolerance; they also provide a basis for cultivating new varieties of drought-tolerant soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Two soybean genotypes&#x02014;drought-tolerant Jindou21 and drought-sensitive Zhongdou33&#x02014;were subjected to dehydration and rehydration conditions. For analysis of DEGs under dehydration conditions, 20 cDNA libraries were generated from roots and leaves at two different time points under well-watered and dehydration conditions. We also generated eight libraries for analysis under rehydration conditions. Sequencing of the 28 libraries produced 25,000&#x02013;33,000 unambiguous tags, which were mapped to reference sequences for annotation of expressed genes. Many genes exhibited significant expression differences among the libraries. DEGs in the drought-tolerant genotype were identified by comparison of DEGs among treatments and genotypes. In Jindou21, 518 and 614 genes were differentially expressed under dehydration in leaves and roots, respectively, with 24 identified both in leaves and roots. The main functional categories enriched in these DEGs were metabolic process, response to stresses, plant hormone signal transduction, protein processing, and plant-pathogen interaction pathway; the associated genes primarily encoded transcription factors, protein kinases, and other regulatory proteins. The seven most significantly expressed (|log <sub> 2 </sub> ratio|&#x02009;&#x02265;&#x02009;8) genes&#x02014; <span class="gene">Glyma15g03920,</span> <span class="gene">Glyma05g02470,</span> <span class="gene">Glyma15g15010,</span> <span class="gene">Glyma05g09070,</span> <span class="gene">Glyma06g35630,</span> <span class="gene">Glyma08g12590,</span> and <span class="gene">Glyma11g16000&#x02014;are</span> more likely to determine <span class="trait">drought</span> stress tolerance. The expression patterns of eight randomly-selected genes were confirmed by quantitative RT-PCR; the results of QRT-PCR analysis agreed with transcriptional profile data for 96 out of 128 (75%) data points.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes with similar expression patterns are usually functionally correlated. We performed cluster analysis of gene expression patterns using Cluster and Java TreeView software [ <xref ref-type="bibr" rid="B39"> 39 </xref> , <xref ref-type="bibr" rid="B40"> 40 </xref> ]. The results of hierarchical cluster analysis of DEGs under dehydration and rehydration treatments are shown in Figure&#x000a0; <xref ref-type="fig" rid="F8"> 8 </xref> . In the drought-tolerant genotype, 48 genes were identified between (JW2L vs. JD2L)&#x02229;(JR0.5 L vs. JD2L)&#x02229;(JR2L vs. JD2L)&#x02229;(JW2R vs. JD2R)&#x02229;(JR0.5R vs. JD2R)&#x02229;(JR2R vs. JD2R); 35 genes were found between (ZW2L vs. ZD2L)&#x02229;(ZR0.5 L vs. ZD2L)&#x02229;(ZR2L vs. ZD2L)&#x02229;(ZW2R vs. ZD2R)&#x02229;(ZR0.5R vs. ZD2R)&#x02229;(ZR2R vs. ZD2R) in the drought-sensitive genotype. Under rehydration conditions, some drought-inducible genes <span class="gene">(Glyma03g29790,</span> <span class="gene">Glyma11g33760,</span> <span class="gene">Glyma08g12590,</span> <span class="gene">Glyma11g16000,</span> and <span class="gene">Glyma01g43180)</span> were down-regulated, while drought-repressed genes <span class="gene">(Glyma11g05550,</span> <span class="gene">Glyma15g15200,</span> <span class="gene">Glyma13g21350,</span> and <span class="gene">Glyma13g35820)</span> were up-regulated upon re-watering. These genes may facilitate adaptation to <span class="trait">drought</span> conditions and promote plant growth recovery upon rewatering.
					</p>
				
					<p>&nbsp;&nbsp;
						The seven most differentially expressed genes (|log <sub> 2 </sub> ratio|&#x02009;&#x02265;&#x02009;8) <span class="gene">(Glyma15g03920,</span> <span class="gene">Glyma05g02470,</span> <span class="gene">Glyma15g15010,</span> <span class="gene">Glyma05g09070,</span> <span class="gene">Glyma06g35630,</span> <span class="gene">Glyma08g12590,</span> and <span class="gene">Glyma11g16000)</span> identified by analysis of DEGs in the drought-tolerant genotype and by cluster analysis warrant further study. Functional annotations of these genes are given in Tables&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> , <xref ref-type="table" rid="T3"> 3 </xref> , and <xref ref-type="table" rid="T4"> 4 </xref> . Functional analysis of these genes should assist efforts to improve soybean <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to their identification, DEGs were annotated according to biological process using GO functional enrichment analysis. The primary enriched biological processes associated with DEGs in the drought-tolerant genotype were metabolic processes, response to stress, response to stimulus, transcription, hormone-mediated signaling pathways, regulation of gene expression, protein modification processes, and plant-type cell wall biogenesis (Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> ). A large proportion of metabolic processes were related to carbohydrate metabolism, which could provide most of the energy required for these pathways under dehydration. Other biological process categories enriched in DEGs included aspartate family amino acid and Acetyl-CoA metabolic processes <span class="gene">(Glyma14g37440.1,</span> <span class="gene">Glyma11g27720.2</span> and <span class="gene">Glyma15g15010.1)</span> that also contribute to energy release [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Proline is an important osmolyte, with its synthesis derived from glutamine metabolic processes; consequently, genes associated with glutamine family amino acid and proline metabolisms <span class="gene">(Glyma14g13480.2,</span> <span class="gene">Glyma11g27720.2,</span> <span class="gene">Glyma14g37440.1,</span> and <span class="gene">Glyma13g07110.1)</span> are necessary for response to osmotic stress [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Dicarboxylic acid metabolism <span class="gene">(Glyma14g13480.2)</span> is also reportedly involved in <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="B52"> 52 </xref> ]. In addition, two regulatory proteins <span class="gene">(Glyma01g01190.1</span> and <span class="gene">Glyma15g03920.1)</span> were identified in the tolerant genotype. These proteins may protect plants by promoting the formation of a tolerant morphological structure to reduce the impact of adversity [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. In addition to genes associated with metabolic processes and morphogenesis, DEGs involved in other processes were mainly regulatory proteins, such as transcription factors and protein kinases.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to TFs and PKs, some regulatory proteins related to plant growth, development, and metabolism were detected in this study (Table&#x000a0; <xref ref-type="table" rid="T4"> 4 </xref> ). It has been reported that ATP synthesis during <span class="trait">drought</span> stress is essential for preserving plastid function [ <xref ref-type="bibr" rid="B74"> 74 </xref> ]. Among our DEGs, some ATP-binding proteins identified in the tolerant genotype were down-regulated, which might make more ATP available for maintenance of cellular activities under water deficit. Cytochrome P450s are involved in ABA catabolism in <span class="trait">drought</span> stress response [ <xref ref-type="bibr" rid="B75"> 75 </xref> ]. They can reduce and eliminate reactive oxygen damage, serve as precursors to ABA synthesis, and also participate in photosynthesis. In our study, the expression of several identified cytochrome P450s was suppressed. Mechanisms involving two-component gene systems play important roles in cellular stress signal transduction [ <xref ref-type="bibr" rid="B22"> 22 </xref> ]. A two-component histidine kinase 1 (ATHK1) is reported to function as an osmosensor, and phospholipase C (PLC) is thought to function upstream of the drought-responsive <span class="gene">DREB2</span> system [ <xref ref-type="bibr" rid="B76"> 76 </xref> ]. In our study, four two-component system genes were identified. Two genes separately encoding histone and aspartate aminotransferase were down-regulated, and another gene encoding asparagine synthetase was up-regulated. Regulatory factors of two-component systems are induced by <span class="trait">drought,</span> salt, and osmotic stress [ <xref ref-type="bibr" rid="B77"> 77 </xref> , <xref ref-type="bibr" rid="B78"> 78 </xref> ]. These identified DEGs thus likely play an important role in improving <span class="trait">drought</span> tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of gma-MIR394a confers tolerance to <span class="trait">drought</span> in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:23000164  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni Z(1), Hu Z, Jiang Q, Zhang H.
				<br/> 
				年份：2012 Oct 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs, key posttranscriptional regulators of eukaryotic gene expression, play important roles in plant development and response to stress. In this study, a soybean gma-MIR394a gene was functionally characterized, especially with regard to its role in <span class="trait">drought</span> stress <span class="trait">resistance.</span> Expression analysis revealed that gma-MIR394a was expressed differentially in various soybean tissues and was induced by <span class="trait">drought,</span> high salinity, low temperature stress, and abscisic acid treatment in leaves. One target gene of gma-miR394a, <span class="gene">Glyma08g11030,</span> was predicted and verified using a modified 5' RLM-RACE (RNA ligase-mediated rapid amplification of 5' cDNA ends) assay. Overexpression of gma-MIR394a resulted in plants with lowered leaf water loss and enhanced <span class="trait">drought</span> tolerance. Furthermore, overexpression of gma-MIR394a in Arabidopsis reduced the transcript of an F-box gene <span class="gene">(At1g27340)</span> containing a miR394 complementary target site. These results suggest that the gma-MIR394a gene functions in positive modulation of <span class="trait">drought</span> stress tolerance and has potential applications in molecular breeding to enhance <span class="trait">drought</span> tolerance in crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis and expression profiling under heat and <span class="trait">drought</span> treatments of <italic> <span class="gene">HSP70</span> </italic> gene family in soybean ( <italic> Glycine max </italic> L.)</h2>
			<h3>PMID:26442082</h3>
				

				
					<h3>PMCID:4585176</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276424/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Hong-Kun </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276428/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Qian-Li </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276425/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yuan-Yu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276439/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yu-Min </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276429/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hai-Yun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276431/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xing </surname>  <given-names> Guo-Jie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276432/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Qi-Yun </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/266484/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Ying-Shan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/251453/overview"/>  </contrib>
				<br/> 
				年份：<day> 09 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heat shock proteins (HSPs) perform a fundamental role in protecting plants against abiotic stresses. Previous studies have made great efforts in the functional analysis of individual family members, but there has not yet been an overall analysis or expression profiling of the <span class="gene">HSP70</span> gene family in soybeans (Glycine max L.). In this study, an investigation of the soybean genome revealed 61 putative <span class="gene">HSP70</span> genes, which were evaluated. These genes were classified into eight sub-families, denoted I-VIII, based on a phylogenetic analysis. In each sub-family, the constituent parts of the gene structure and motif were relatively conserved. These <span class="gene">GmHSP70</span> genes were distributed unequally on 17 of the 20 chromosomes. The analysis of the expression profiles showed that 53 of the 61 <span class="gene">GmHSP70</span> genes were differentially expressed across the 14 tissues. However, most of the <span class="gene">GmHSP70s</span> were differentially expressed in a tissue-specific expression pattern. Furthermore, the expression of some of the duplicate genes was partially redundant, while others showed functional diversity. The quantitative real-time PCR (qRT-PCR) analysis of the 61 soybean <span class="gene">HSP70</span> genes confirmed their stress-inducible expression patterns under both <span class="trait">drought</span> and heat stress. These findings provide a thorough overview of the evolution and modification of the <span class="gene">GmHSP70</span> gene family, which will help to determine the functional characteristics of the <span class="gene">HSP70</span> genes in soybean growth and development.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Heat shock proteins (HSPs) are a group of proteins induced by heat shock, and they are found in virtually all living organisms, from bacteria to humans (De Maio, <xref rid="B5" ref-type="bibr"> 1999 </xref> ). In plants, the <italic> HSP </italic> gene family members play important roles in developmental processes, as well as different kinds of environmental stress conditions, such as heat and <span class="trait">drought</span> (Swindell et al., <xref rid="B53" ref-type="bibr"> 2007 </xref> ; Cho and Choi, <xref rid="B3" ref-type="bibr"> 2009 </xref> ), low and high temperatures (Krishna et al., <xref rid="B28" ref-type="bibr"> 1995 </xref> ; Sabehat et al., <xref rid="B45" ref-type="bibr"> 1998 </xref> ; Lopez-Matas et al., <xref rid="B37" ref-type="bibr"> 2004 </xref> ), salinity (Zou et al., <xref rid="B65" ref-type="bibr"> 2012 </xref> ), and heavy metals (Kim et al., <xref rid="B26" ref-type="bibr"> 2014 </xref> ). HSPs have recently been discovered to be associated with plant responses to infection by pathogens such as nematodes (Maimbo et al., <xref rid="B38" ref-type="bibr"> 2007 </xref> ). Based on their molecular weight, HSPs are classified into five major families: sHSP, small heat shock protein; HSP60, chaperonin family; HSP70, 70-kDa-heat shock protein; <span class="gene">HSP90</span> and HSP100 family (Wang et al., <xref rid="B59" ref-type="bibr"> 2004 </xref> ). Of these, the <span class="gene">HSP70</span> family is of greatest interest, as it is fundamental to plant developmental processes and functions during heat stress (Sung et al., <xref rid="B52" ref-type="bibr"> 2001 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> L.) is one of the most economically and nutritionally crucial crops in the world. It provides not only vegetable protein and edible oil, but also essential amino acids for humans and animals. However, soybean production is threatened by <span class="trait">drought,</span> as well as other environmental stresses. With the sequencing of the soybean genome (Schmutz et al., <xref rid="B48" ref-type="bibr"> 2010 </xref> ), a few members of the HSP family were characterized in this species (Jinn et al., <xref rid="B23" ref-type="bibr"> 1995 </xref> ; Fuganti et al., <xref rid="B14" ref-type="bibr"> 2010 </xref> ; Mohammadi et al., <xref rid="B42" ref-type="bibr"> 2012 </xref> ; Liu and Whitham, <xref rid="B34" ref-type="bibr"> 2013 </xref> ; Lopes-Caitar et al., <xref rid="B36" ref-type="bibr"> 2013 </xref> ; Wang et al., <xref rid="B56" ref-type="bibr"> 2014 </xref> ). However, most of their functions remain to be determined. There remains no genome-wide characterization of the <span class="gene">HSP70</span> family in soybean to date.
					</p>
				
					<p>&nbsp;&nbsp;
						In our study, all of the non-redundant sets of the soybean <italic> <span class="gene">HSP70</span> </italic> genes were scanned and integrated, and we also determined their chromosomal locations, predicted their protein structures with available software and network stations, and analyzed the expression levels of the soybean <italic> <span class="gene">HSP70</span> </italic> genes using qRT-PCR in order to determine their tolerance to <span class="trait">drought</span> and heat stresses. Our findings will be useful resources for future studies to unravel the functions of the <italic> <span class="gene">GmHSP70</span> </italic> genes and will contribute to our understanding of the evolutionary history of the <italic> <span class="gene">HSP70</span> </italic> genes in different species.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to gain more insight into the roles of soybean <italic> <span class="gene">HSP70</span> </italic> genes in heat and <span class="trait">drought</span> tolerance, the expression profiles of 61 soybean <italic> <span class="gene">HSP70</span> </italic> genes in response to <span class="trait">drought</span> (0, 3, 6, 12, 24 h) and heat (0, 3, 6, 12, 24 h) stresses were reanalyzed by qRT-PCR. These genes expressed diversely under both stresses (Figures <xref ref-type="fig" rid="F6"> 6A,B </xref> ). For the <span class="trait">drought</span> treatment (15% PEG), these genes could be divided into four clusters. Cluster 1 contains 31 (50.8%) members of detectable <italic> <span class="gene">GmHSP70</span> </italic> genes, which were widely upregulated after 6 h by <span class="trait">drought</span> treatment, and up to the highest expression level at 12 h, then downregulated at 24 h after <span class="trait">drought</span> treatment. Cluster 2 mainly consists of 13 (21.3%) genes, which were widely downregulated by <span class="trait">drought</span> treatment. Cluster 3 has 6 (9.8%) members, which showed a weakly upregulated at 6 or 12 h, but downregulated at 24 h after <span class="trait">drought</span> treatment. Cluster 4 contains 11 (18%) genes, which were mainly upregulated with the increased level of <span class="trait">drought</span> treatment after 24 h, and the <italic> <span class="gene">Glyma02g36700</span> </italic> in particular was highly induced at 3 h after <span class="trait">drought</span> treatment. Overall, 30 <italic> <span class="gene">GmHSP70</span> </italic> genes were up-regulated from two-fold to 222-fold after <span class="trait">drought</span> treatment relative to the control, and the other <italic> <span class="gene">GmHSP70</span> </italic> genes were down-regulated (&#x0003c;0.5-fold) at some stage or showed no changes during the <span class="trait">drought</span> stress treatment (Additional File <xref ref-type="supplementary-material" rid="SM14"> 14 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Quantitative RT-PCR analysis of the <italic> <bold> <span class="gene">GmHSP70</span> </bold> </italic> gene expression in soybean leaves in response to <span class="trait">drought</span> stress and 42&#x000b0;C heat stress </bold> . <bold> (A) </bold> Expression levels of 61 <italic> <span class="gene">GmHSP70</span> </italic> family genes under <span class="trait">drought</span> stress. <bold> (B) </bold> Expression levels of 61 <italic> <span class="gene">GmHSP70</span> </italic> family genes under heat stress. The normalized relative expression level of the 0 h treatment time point was set up as 1. Leaves collected at 0, 3, 6, 12, and 24 h post-drought or at 0, 3, 6, 12, and 24 h post heat stress. Means are calculated from technical triplicate qRT-PCR measurements within three biological replicates. The qRT-PCR data was shown in the Additional File <xref ref-type="supplementary-material" rid="SM14"> 14 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Notably, 29 <italic> <span class="gene">GmHSP70</span> </italic> genes showed upregulation under both <span class="trait">drought</span> and heat stress conditions. Five soybean <italic> <span class="gene">HSP70</span> </italic> genes were downregulated both under the <span class="trait">drought</span> and heat stress. 26 upregulated genes typically induced by heat were downregulated during <span class="trait">drought</span> stress. In contrast, only one gene, <italic> <span class="gene">Glyma18g11520</span> </italic> , was downregulated by the heat treatment and upregulated by the <span class="trait">drought</span> stress (Figure <xref ref-type="fig" rid="F6"> 6 </xref> and Additional File <xref ref-type="supplementary-material" rid="SM14"> 14 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Analyses of <italic> cis </italic> -elements in the promoters suggest that <italic> <span class="gene">GmHSP70s</span> </italic> might respond to regulate plant growth and development, as well as different environmental stresses and stimuli (Additional File <xref ref-type="supplementary-material" rid="SM6"> 6 </xref> ). In a previous study, the promoters for 11 Arabidopsis <italic> <span class="gene">HSP70</span> </italic> genes were examined for the presence of two major temperature responsive <italic> cis </italic> -elements, heat shock element (HSE), and C-repeat or dehydration responsive element (CRT/DRE) (Sung et al., <xref rid="B52" ref-type="bibr"> 2001 </xref> ). HSE has been linked with the heat inducible expression of many heat shock genes (Czarnecka et al., <xref rid="B4" ref-type="bibr"> 1989 </xref> ). CRT/DRE is known to be associated with <span class="trait">drought-</span> and cold-inducible expression of many genes (Yamaguchi-Shinozaki and Shinozaki, <xref rid="B61" ref-type="bibr"> 1994 </xref> ). In our <italic> cis </italic> -acting elements analysis of <italic> <span class="gene">GmHSP70s</span> </italic> , we found that 45 of 61 <italic> <span class="gene">GmHSP70</span> </italic> genes contain the HSE element, and one gene ( <italic> <span class="gene">Glyma03g17870</span> </italic> ) contains the DRE element. These results suggest that most of the <italic> <span class="gene">GmHSP70s</span> </italic> might be significantly related to heat-stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies have shown that <italic> <span class="gene">HSP70</span> </italic> genes play an important role in plants in response to various stresses, such as <span class="trait">drought,</span> salt, freezing temperatures, and heat (Kurepa et al., <xref rid="B29" ref-type="bibr"> 2009 </xref> ). In this study, the expression profiles of the <italic> <span class="gene">GmHSP70</span> </italic> genes under <span class="trait">drought</span> and heat stresses (Figure <xref ref-type="fig" rid="F6"> 6 </xref> and Additional File <xref ref-type="supplementary-material" rid="SM14"> 14 </xref> ) revealed that the soybean <italic> <span class="gene">HSP70</span> </italic> genes were widely involved in responding to <span class="trait">drought</span> and heat stresses. It is interesting to note that most of <italic> <span class="gene">GmHSP70</span> </italic> genes (55) were strongly upregulated during heat stress, which indicates that those genes might be mainly involved in the heat stress biological pathway. Some <italic> <span class="gene">GmHSP70</span> </italic> genes showed the same expression profile under the heat or <span class="trait">drought</span> treatment, being either upregulated or downregulated. This result suggests that these <italic> <span class="gene">GmHSP70</span> </italic> genes were co-expressed in responses to <span class="trait">drought</span> and heat stresses, and those genes might play shared roles in <span class="trait">drought</span> and heat stresses. However, the gene expressions profiles which responded to the different stress conditions usually tended to be different; some heat-regulated genes were downregulated in <span class="trait">drought</span> stress, which indicated that two sets of <italic> <span class="gene">GmHSP70</span> </italic> genes were involved in the response to the <span class="trait">drought</span> and heat stress, respectively. These results also imply that the signaling pathways in the plants' abiotic stress responses are very complicated systems. From an applied perspective, the identification of the <italic> HSP70s </italic> with potential value in the improvement of stress <span class="trait">resistance</span> of the soybean would likely benefit from targeting such genes which are part of the abiotic and biotic stress response networks.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean ( <italic> Glycine max </italic> L.) genome contains 61 members of the <italic> <span class="gene">HSP70</span> </italic> gene family, and these genes are located within 17 chromosomes. The genome distribution, gene organizations, and gene structures suggest a complex evolutionary history of this family in soybeans. The soybean <italic> <span class="gene">HSP70</span> </italic> family is greatly contributed to by segment/chromosomal duplications, which may be associated with the soybean genome fusion events. Their expression profiles in various tissues and responses to <span class="trait">drought</span> and heat stress conditions demonstrate that this gene family is widely involved in soybean organ development, as well as <span class="trait">drought</span> and heat stress responses. The overall picture of this gene family and their potential involvement in growth, development, and stress responses, will facilitate further research on the <italic> <span class="gene">HSP70</span> </italic> gene family, particularly in regards to their evolutionary history and biological functions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean C2H2-Type Zinc Finger Protein <span class="gene">GmZFP3</span> with Conserved QALGGH Motif Negatively Regulates <span class="trait">Drought</span> Responses in Transgenic <italic> Arabidopsis </italic></h2>
			<h3>PMID:27047508</h3>
				

				
					<h3>PMCID:4796006</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dayong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn005">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/254519/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tong </surname>  <given-names> Jinfeng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn005">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhaolong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Peipei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wan </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yihong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiaolan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Jiayin </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/314160/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shao </surname>  <given-names> Hongbo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/59382/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hongxiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 03 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant response to environmental stresses is regulated by a complicated network of regulatory and functional genes. In this study, we isolated the putative stress-associated gene <span class="gene">GmZFP3</span> (a C2H2-type Zinc finger protein gene) based on the previous finding that it was one of two genes located in the QTL region between the Satt590 and Satt567 markers related to soybean tolerance to <span class="trait">drought.</span> Temporal and spatial expression analysis using quantitative real-time PCR indicated that <span class="gene">GmZFP3</span> was primarily expressed in roots, stems and leaf organs and was expressed at low levels in flowers and soybean pods. Moreover, <span class="gene">GmZFP3</span> expression increased in response to polyethylene glycol (PEG) and Abscisic acid (ABA) treatments. In addition, subcellular localization analysis indicated that <span class="gene">GmZFP3</span> was ubiquitously distributed in plant cells. Transgenic experiments indicated that <span class="gene">GmZFP3</span> played a negative role in plant tolerance to <span class="trait">drought.</span> Analysis of ABA-related marker gene expression in Arabidopsis suggested that <span class="gene">GmZFP3</span> might be involved in the ABA-dependent pathway during the <span class="trait">drought</span> stress response. Taken together, these results suggest that soybean <span class="gene">GmZFP3</span> negatively regulates the <span class="trait">drought</span> response.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Luo et al. ( <xref rid="B21" ref-type="bibr"> 2012 </xref> ) cloned soybean <italic> GsZFP1 </italic> from <italic> Glycine soja </italic> and found that it played a crucial role in the plant response to cold and <span class="trait">drought</span> stress. They further found that <italic> GsZFP1 </italic> over-expression in <italic> Arabidopsis </italic> reduced ABA sensitivity and decreased stomata size under ABA treatment. Kim et al. ( <xref rid="B15" ref-type="bibr"> 2001 </xref> ) found that the C2H2-type protein <italic> <span class="gene">SCOF-1</span> </italic> was specifically induced by low temperature and ABA, but not by dehydration or salinity. Furthermore, <span class="gene">SCOF-1</span> interacted with SGBF-1 in a yeast two-hybrid system, suggesting that <span class="gene">SCOF-1</span> functions as a positive regulator of ABRE-mediated <italic> COR </italic> gene expression through protein-protein interactions, which in turn, enhances plant cold tolerance. Yu et al. ( <xref rid="B37" ref-type="bibr"> 2014 </xref> ) reported that soybean <span class="gene">GmZF1</span> enhanced <italic> Arabidopsis </italic> tolerance to cold stress by regulating cold-regulation gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Specht et al. ( <xref rid="B28" ref-type="bibr"> 2001 </xref> ) identified a WUE (Water Use Efficiency)-related QTL region between the Satt590 and Satt567 markers on chromosome 7 in soybean, and the <span class="gene">Glyma07g01900</span> and <span class="gene">Glyma07g05820</span> gene loci were borders of this QTL region. All 393 genes between these two loci were downloaded using the Perl program and isolated. One such gene was <italic> <span class="gene">GmZFP3</span> </italic> , a putative stress-associated gene, was selected for further study. <italic> <span class="gene">GmZFP3</span> </italic> was primarily expressed in the root and stem, while <span class="gene">GmZFP3</span> protein was ubiquitously distributed among plant cells. Transgenic experiments indicated that <span class="gene">GmZFP3</span> played a negative role in plant tolerance to <span class="trait">drought</span> and that it might be involved in the ABA-dependent pathway during response to <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The QTL region sequences between markers Satt590 and Satt567 related to soybean <span class="trait">drought</span> tolerance were downloaded from <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/cgi-bin/gbrowse/soybean/"> http://www.phytozome.net/cgi-bin/gbrowse/soybean/ </ext-link> using the perl program. The functional annotation of genes was confirmed with BLAST2GO software. The <italic> <span class="gene">GmZFP3</span> </italic> gene primers (see Table <xref ref-type="table" rid="T1"> 1 </xref> ) were designed according to the full-length coding sequence and used to clone the genes from soybean root tissue cDNA using RT-PCR (Reverse transcriptase-polymerase chain reaction).
					</p>
				
					<p>&nbsp;&nbsp;
						Before treatment, the total root length, water loss rate from 0 to 24 h, stomatal width/length and density were measured in <italic> <span class="gene">GmZFP3</span> </italic> -overexpressing transgenic <italic> Arabidopsis </italic> plants and WT control plants. After air <span class="trait">drought</span> treatment application, including no watering and rewatering, the phenotype change of transgenic, and wild type were recorded using camera (Canon,Japan). The experiments were repeated independently at least three times.
					</p>
				
					<p>&nbsp;&nbsp;
						To test whether <span class="trait">drought</span> stress induced <italic> <span class="gene">GmZFP3</span> </italic> expression, we treated the roots of soybean seedlings with PEG and ABA, then analyzed <italic> <span class="gene">GmZFP3</span> </italic> expression by quantitative real-time RT-PCR. ABA treatment (100 &#x003bc;M) initially significantly suppressed <italic> <span class="gene">GmZFP3</span> </italic> expression after 30 min treatment ( <italic> p </italic> &#x0003c; 0.05), but then it increased back to its initial expression level from 45 to 60 min, then continuously decreased from 90 to 120 min (Figure <xref ref-type="fig" rid="F3"> 3A </xref> ). However, <italic> <span class="gene">GmZFP3</span> </italic> expression decreased within 2 h of PEG-6000 (20%) treatment, then its mRNA expression continuously increased from 4 to 12 h and reached a maximum at 12 h. Finally, its expression decreased from 12 to 48 h (Figure <xref ref-type="fig" rid="F3"> 3B </xref> ). Taken together, these data suggest that <italic> <span class="gene">GmZFP3</span> </italic> expression changed in response to PEG and ABA.
					</p>
				
					<p>&nbsp;&nbsp;
						To further validate <span class="gene">GmZFP3</span> function in <span class="trait">drought</span> stress, we did not water 4-week-old transgenic <italic> Arabidopsis </italic> seedlings for 3 weeks and used wild type (WT) <italic> Arabidopsis </italic> plants as controls. Seedlings from transgenic <italic> <span class="gene">GmZFP3</span> Arabidopsis </italic> lines #5 and #6 grew worse compared to controls (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ). The survival rate of transgenic lines was significantly lower than that of wild type(WT), suggesting that <italic> <span class="gene">GmZFP3</span> </italic> overexpression did not improve <span class="trait">drought</span> tolerance, likely due to the decreased root length and increased water loss rate in transgenic plants (Figures <xref ref-type="fig" rid="F5"> 5B,C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Performance of transgenic <italic> <span class="gene">GmZFP3</span> </italic> - overexpressing <italic> Arabidopsis </italic> plants under <span class="trait">drought</span> stress </bold> . <bold> (A) </bold> The response of transgenic plants and wild type (WT) to <span class="trait">drought</span> stress. The transgenic plants (#5 and #6) showed the more sensitive to <span class="trait">drought</span> treatment compared with the WT plants. <bold> (B) </bold> The root length of transgenic was significantly shorter than that of WT. The roots of transgenic plants (#5 and #6) had the shorter primary root compared with the WT plants. <bold> (C) </bold> The water loss rate under air condition in transgenic and WT. When detached leaves from transgenic and WT plant were placed the air condition, the transgenic plants (#5 and #6) showed a faster water loss compared with the WT plants.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the function of <span class="gene">GmZFP3</span> in <span class="trait">drought</span> response, we performed RT-PCR and qRT-PCR for nine genes related to stress/or ABA signaling pathway in transgenic lines #5 and #6, including <italic> DREB2A, LHY1, AT5G02840, MYB2, RCI3, PAD3, CCA1 </italic> , and <italic> UGT71B6 </italic> . Among the nine genes tested, six genes related to the ABA-dependent signaling pathway were up-regulated, with the exception of <italic> <span class="gene">DREB2A</span> </italic> related to the ABA-independent pathway and <italic> AT5G02840 </italic> compared to wild-type (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). Because of the higher expression in transgenic line #5, we measured the relative germination rate, stoma width/length, and density for line #5 under different ABA concentrations. We found that leaf stoma in transgenic plants were primarily completely open (CO), while stoma in wild-type plants were primarily completely closed (CC) (Figures <xref ref-type="fig" rid="F7"> 7A&#x02013;C </xref> ). The relative germination rate was significantly lower in <span class="gene">GmZFP3-overexpressing</span> plants compared to wild-type after ABA treatment (Figure <xref ref-type="fig" rid="F7"> 7D </xref> ), suggesting that <span class="gene">GmZFP3-overexpressing</span> plants were sensitive to ABA. Moreover, the width/length was significantly higher compared to wild-type, but the stoma density was significantly decreased under the same conditions (Figures <xref ref-type="fig" rid="F7"> 7E,F </xref> ). Taken together, these data suggest that <span class="gene">GmZFP3</span> functioned in the ABA-dependent pathway during the <span class="trait">drought</span> response.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the many reports about plant <span class="trait">drought</span> tolerance, we selected two aquaporin gene <span class="gene">TIP</span> (TIP2;3, <span class="gene">Glyma07g02060)</span> and NIP <span class="gene">(Glyma07g02800),</span> six transcription factors HD (Homeodomain)-zip <span class="gene">(Glyma07g01940),</span> ERF <span class="gene">(Glyma07g02380),</span> <span class="gene">bZIP38</span> <span class="gene">(Glyma07g02050),</span> Zinc Finger Protein (ZFP3, <span class="gene">Glyma07g02880),</span> bHLH <span class="gene">(Glyma07g02120),</span> and WRKY27 <span class="gene">(Glyma07g02630)</span> as candidate genes for soybean <span class="trait">drought</span> tolerance. Among them, the <italic> <span class="gene">GmTIP2;3</span> </italic> and <italic> <span class="gene">GmZFP3</span> </italic> genes showed similar spatial and temporal expression patterns after 10%PEG6000 or 100 &#x003bc;M ABA treatments for different time periods. More importantly, when we co-transformed pEarlygate103-HD-zip(OE), <span class="gene">GmERF(OE),</span> <span class="gene">GmbZIP38(OE),</span> <span class="gene">GmZFP3(OE),</span> bHLH(OE), and pGUS- <italic> <span class="gene">GmTIP2;3</span> </italic> -promoter(Zhang et al., <xref rid="B39" ref-type="bibr"> 2016 </xref> ), only <span class="gene">GmZFP3</span> could altered and inhibited GUS expression, leading to the lack of blue color(Data not shown), indicating that <span class="gene">GmZFP3</span> inhibited the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, <span class="gene">GmZF1</span> contained a conserved QALGGH motif, which has been previously been shown to enhance <italic> Arabidopsis </italic> tolerance to cold stress (Yu et al., <xref rid="B37" ref-type="bibr"> 2014 </xref> ). However, studies on <italic> Arabidopsis </italic> transformed with GsZFP1 lacking the typical QALGGH motif revealed that it played an important role in withstanding cold and <span class="trait">drought</span> stresses (Luo et al., <xref rid="B21" ref-type="bibr"> 2012 </xref> ), suggesting that the QALGGH motif in the GsZFP1 protein was not necessary for adaptation to abiotic stress in wild soybean. In our study, <span class="gene">GmZFP3</span> also had a plant-specific typical QALGGH motif, suggesting that this motif was crucial for plant response to abiotic stresses in cultivated soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Xie et al. ( <xref rid="B34" ref-type="bibr"> 2009 </xref> ) found that soybean trihelix transcription factors <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> improved plant tolerance to abiotic stresses in transgenic <italic> Arabidopsis </italic> , as the expression of <italic> CCA1, LHY1, MYB2 </italic> , and <italic> AT5G02840 </italic> was down-regulated in transgenic plants compared to WT, while the expression of <italic> PAD3, LTP3, RCI3, UGT71B6 </italic> , and <italic> <span class="gene">DREB2A</span> </italic> was up-regulated. In this study, we selected these nine stress-responsive genes for further quantitative real-time PCR analysis in transgenic and WT plants. Six genes, including <italic> LHY1, MYB2, CCA1, PAD3, LTP3 </italic> , and <italic> UGT71B6 </italic> , were up-regulated in transgenic lines expressing <italic> <span class="gene">GmZFP3</span> </italic> . <italic> RCI3 </italic> , while <italic> AT5G02840 </italic> expression did not significantly change between transgenic and WT plants, in contrast to the report by Xie et al, especially for <italic> LHY1, MYB2 </italic> , and <italic> CCA1 </italic> , implying a difference in gene function. <italic> LTP3, PAD3 </italic> , and <italic> UGT71B6 </italic> are involved in ABA responses (Arondel et al., <xref rid="B3" ref-type="bibr"> 2000 </xref> ; Priest et al., <xref rid="B23" ref-type="bibr"> 2006 </xref> ; Kaliff et al., <xref rid="B14" ref-type="bibr"> 2007 </xref> ). RCI3 encodes a peroxidase, and its overexpression conferred dehydration and salt tolerance (Llorente et al., <xref rid="B20" ref-type="bibr"> 2002 </xref> ). <span class="gene">DREB2A</span> acts as a trans-acting factor in the signal transduction pathway under dehydration conditions, and dehydration induced its expression (Liu et al., <xref rid="B18" ref-type="bibr"> 1998 </xref> ). <italic> <span class="gene">DREB2A</span> </italic> expression does not activate downstream genes under normal growth conditions. However, overexpression of its constitutively active form leads to <span class="trait">drought</span> stress tolerance and slight freezing tolerance (Sakuma et al., <xref rid="B26" ref-type="bibr"> 2006 </xref> ). <italic> LHY1, CCA1, At5g02840 </italic> , and <italic> MYB2 </italic> have been found to be responsive to stress and/or ABA (Boxall et al., <xref rid="B4" ref-type="bibr"> 2005 </xref> ). In our study, <italic> <span class="gene">GmZFP3</span> </italic> -overexpressing <italic> Arabidopsis </italic> plants were more sensitive to <span class="trait">drought</span> stress than wild-type plants, consistent with <italic> AtMYB2 </italic> function, whose expression was induced by dehydration, high salt stress or exogenous ABA (Urao et al., <xref rid="B33" ref-type="bibr"> 1993 </xref> ). However, transgenic plants overexpressing <italic> MYB2 </italic> were more sensitive to ABA and showed stress tolerance, while co-expression of <italic> AtMYB2 </italic> and <italic> AtMYC2 </italic> conferred moderate stress tolerance (Abe et al., <xref rid="B1" ref-type="bibr"> 2003 </xref> ). The seed germination rate of transgenic <italic> <span class="gene">GmZFP3</span> </italic> plants was influenced by ABA and decreased as ABA concentrations increased. In addition, we observed the stomata by SEM to gain further insight into the mechanism of <span class="gene">GmZFP3</span> during stress. Stomatal aperture (width/length) of transgenic plant leaves under <span class="trait">drought</span> conditions was significantly larger than wild-type, and the percentage of completely open (CO) stoma in transgenic plants was higher than wild-type. However, the stomatal density was lower in transgenic plants compared to wild-type. Huang et al. ( <xref rid="B11" ref-type="bibr"> 2009 </xref> ) examined the percentage of three types of stomata, stomatal density, and stomatal conduction between ZH11 and <italic> dst </italic> mutants and found that the completely open level, stomatal density, and aperture in mutants was lower than in wild-type, which was consistent with the positive role under <span class="trait">drought</span> stress in <italic> dst </italic> mutants. Therefore, the <italic> <span class="gene">GmZFP</span> </italic> -overexpressing plants might lose large volumes of water by increasing the width/length and number of completely open stoma, but not the density, leading to <span class="trait">drought</span> stress sensitivity. Moreover, stomatal development and change are closely related to ABA (Trejo et al., <xref rid="B32" ref-type="bibr"> 1993 </xref> ). By examining the expression pattern of marker genes related to stress or ABA and the effect of ABA on germination rate, we conclude that the function of <span class="gene">GmZFP3</span> under <span class="trait">drought</span> stress involves the ABA pathway but not the ROS pathway.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Expression pattern of <span class="trait">drought</span> stress marker genes in soybean roots under two water deficit systems</h2>
			<h3>PMID:22802707</h3>
				

				
					<h3>PMCID:3392874</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Neves-Borges </surname>  <given-names> Anna Cristina </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-212">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-212">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guimar&#x000e3;es-Dias </surname>  <given-names> F&#x000e1;bia </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-212">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-212">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cruz </surname>  <given-names> Fernanda </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-212">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mesquita </surname>  <given-names> Rosilene Oliveira </given-names>  </name>  <xref ref-type="aff" rid="af5-gmb-35-1-sup-212">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-212">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Romano </surname>  <given-names> Eduardo </given-names>  </name>  <xref ref-type="aff" rid="af4-gmb-35-1-sup-212">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Loureiro </surname>  <given-names> Marcelo Ehlers </given-names>  </name>  <xref ref-type="aff" rid="af5-gmb-35-1-sup-212">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de F&#x000e1;tima Grossi-de-S&#x000e1; </surname>  <given-names> Maria </given-names>  </name>  <xref ref-type="aff" rid="af4-gmb-35-1-sup-212">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alves-Ferreira </surname>  <given-names> M&#x000e1;rcio </given-names>  </name>  <xref ref-type="corresp" rid="c1-gmb-35-1-sup-212"/>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-212">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The study of tolerance mechanisms for <span class="trait">drought</span> stress in soybean is fundamental to the understanding and development of tolerant varieties. Using in silico analysis, four marker genes involved in the classical ABA-dependent and ABA-independent pathways of <span class="trait">drought</span> response were identified in the Glycine max genome in the present work. The expression profiles of the marker genes ERD1-like, <span class="gene">GmaxRD20A-like,</span> <span class="gene">GmaxRD22-like</span> and <span class="gene">GmaxRD29B-like</span> were investigated by qPCR in root samples of <span class="trait">drought</span> sensitive and tolerant soybean cultivars (BR 16 and Embrapa 48, respectively), submitted to water deficit conditions in hydroponic and pot-based systems. Among the four putative soybean homologs to Arabidopsis genes investigated herein, only <span class="gene">GmaxRD29B-like</span> was not regulated by water deficit stress. Distinct expression profiles and different induction levels were observed among the genes, as well as between the two drought-inducing systems. Our results showed contrasting gene expression responses for the <span class="gene">GmaxRD20A-like</span> and <span class="gene">GmaxRD22-like</span> genes. <span class="gene">GmaxRD20A-like</span> was highly induced by continuous <span class="trait">drought</span> acclimating conditions, whereas <span class="gene">GmaxRD22-like</span> responses decreased after abrupt water deprivation. <span class="gene">GmaxERD1-like</span> showed a different expression profile for the cultivars in each system. Conversely, <span class="gene">GmaxRD20A-like</span> and <span class="gene">GmaxRD22-like</span> genes exhibited similar expression levels in tolerant plants in both systems.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The dendrogram analysis was performed on the selected protein sequences to uncover putative soybean homologs for the <italic> Arabidopsis </italic> DMGs ( <xref ref-type="fig" rid="f1-gmb-35-1-sup-212"> Figure 1 </xref> ). When the dendrogram analysis could not pinpoint a single putative homolog, the top e-value obtained in the BLASTP analysis was consider as selection criteria. This strategy allowed us to identify the four putative soybean homolog genes, <italic> <span class="gene">GmaxERD1-</span> </italic> like <span class="gene">(Glyma04g38050),</span> <italic> <span class="gene">GmaxRD20A-</span> </italic> like <span class="gene">(Glyma03g41030),</span> <italic> <span class="gene">GmaxRD22-</span> </italic> like <span class="gene">(Glyma14g20450)</span> and <italic> <span class="gene">GmaxRD29B-</span> </italic> like <span class="gene">(Glyma16g31330)</span> <italic> , </italic> for the <italic> Arabidopsis </italic> DMGs <italic> ERD1 </italic> (AT5G51070), <italic> RD20A </italic> (AT2G33380), <italic> RD22 </italic> (AT5G25610) and <italic> RD29B </italic> (AT5G52300), respectively ( <xref ref-type="fig" rid="f1-gmb-35-1-sup-212"> Figure 1 </xref> ). As previously mentioned, the <italic> RD20A </italic> , <italic> RD22 </italic> and <italic> RD29B </italic> genes belong to the ABA-dependent pathways, while the <italic> ERD1 </italic> gene belongs to the ABA-independent pathways of <span class="trait">drought</span> stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the four DMGs identified by our analysis, <italic> <span class="gene">GmaxRD29B-</span> </italic> like was not induced under any of the test conditions (data not shown), whereas <italic> <span class="gene">Gmaxerd1-</span> </italic> like, <italic> <span class="gene">GmaxRD20A-</span> </italic> like and <italic> <span class="gene">GmaxRD22-</span> </italic> like were induced in both cultivars in the two cultivation conditions, PSys and HSys. Nevertheless, these genes showed distinct expression dynamics and induction levels depending on <span class="trait">drought</span> conditions assessed.
					</p>
				
					<p>&nbsp;&nbsp;
						We employed the POBO program ( <xref ref-type="bibr" rid="b25-gmb-35-1-sup-212"> Kankainen and Holm, 2004 </xref> ) to evaluate the enrichment of 17 <italic> cis </italic> -elements related to <span class="trait">drought,</span> ABA, or osmotic stress, such as ABRE, DRE, MYB, MYC or NAC <italic> cis </italic> -elements ( <xref ref-type="supplementary-material" rid="SD1"> Table S1 </xref> ), that we had found previously in the PLACE data bank ( <xref ref-type="bibr" rid="b20-gmb-35-1-sup-212"> Higo <italic> et al. </italic> , 1999 </xref> ). The statistical significance of their enrichment in the promoters of <italic> <span class="gene">GmaxRD20A-</span> </italic> like, <italic> <span class="gene">GmaxRD22-</span> </italic> like and <italic> <span class="gene">GmaxERD1-</span> </italic> like was also evaluated. <italic> <span class="gene">GmaxRD29B-</span> </italic> like was excluded from this analysis because it was not responsive to the <span class="trait">drought</span> conditions evaluated in this work. To do so, we compared the frequency of the 17 motifs in a group comprising these genes and in a background set (comprising all promoters regions of the <italic> G. max </italic> genome &#x02013; <span class="gene">BG</span> model). This analysis revealed that three <italic> cis </italic> -elements (ACGT, WAACCA and CANNTG) are significantly enriched in the promoter region of the three genes analyzed when compared to the background set, while the ACCGAC motif is significantly enriched in the promoter region of the <italic> <span class="gene">GmaxRD20A</span> </italic> -like and <italic> <span class="gene">GmaxRD22</span> </italic> -like genes, also compared to the background set ( <xref ref-type="supplementary-material" rid="SD2"> Table S2 </xref> ). The most striking result was found for the ACGT motif, an element functionally that is important in a variety of promoters responding to ABA, among other stimuli ( <xref ref-type="bibr" rid="b18-gmb-35-1-sup-212"> Guiltinan <italic> et al. </italic> , 1990 </xref> ). The analysis in POBO indicated that the ACGT motif is present up to 16 times in the promoter regions of the DMG genes, with an average of 5.33 compared to an average of 3.02 for the genome background ( <xref ref-type="supplementary-material" rid="SD2"> Table S2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The expression profiles of the putative soybean homologs <italic> <span class="gene">GmaxRD20A-</span> </italic> like, <italic> <span class="gene">GmaxRD22-</span> </italic> like and <italic> <span class="gene">GmaxERD1-</span> </italic> like were notably different in HSys when compared to PSys ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-212"> Figure 2 </xref> ). <italic> <span class="gene">GmaxRD20A-</span> </italic> like and <italic> <span class="gene">GmaxRD22-</span> </italic> like putative homologs of <italic> Arabidopsis </italic> genes from the ABA-dependent pathway, were predominantly expressed in PSys, whereas <italic> <span class="gene">GmaxERD1-</span> </italic> like, a putative homolog of an <italic> Arabidopsis </italic> gene from the ABA-independent pathway, was mainly expressed in HSys ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-212"> Figure 2C </xref> ). The expression levels and profiles also varied among genes. For instance, <italic> <span class="gene">GmaxRD22-</span> </italic> like was highly expressed in both cultivars in PSys, and its expression levels in severe stress (&#x003a8;w &#x02212;3.0 MPa) were nine to eleven times higher than in moderate stress (&#x003a8;w &#x02212;1.5 MPa). In contrast, <italic> <span class="gene">GmaxRD20A-</span> </italic> like showed lower expression levels and only a minor difference in gene expression between severe and moderate stress. Considerable differences in DMG expression between the sensitive and tolerant soybean cultivars were exclusively found for <italic> <span class="gene">GmaxERD1-</span> </italic> like in plants grown under HSys. The difference was limited to the expression levels, but this may have biological significance. These results suggest that <span class="trait">drought</span> stress levels and pathway activation may vary considerably between the two systems investigated in this work. <xref ref-type="bibr" rid="b33-gmb-35-1-sup-212"> Martins <italic> et al. </italic> (2008) </xref> also observed differences in the expression profiles of <italic> Axi 1 </italic> , <italic> PITP </italic> and <italic> bHLH </italic> soybean genes between the two systems and reported significant differences among time points in stress recognition and the subsequent adaptive response between mild or more severe stress levels. <italic> <span class="gene">GmaxERD1-</span> </italic> like showed expression levels of about four times higher in the sensitive cultivar in the T150 condition of HSys ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-212"> Figure 2C </xref> ), suggesting that the sensitive cultivar may have a stronger response to water deficit stress at the molecular level due to the lack of traits that are important for plant adaptation to <span class="trait">drought.</span> Another explanation could be the occurrence of damage-related responses that may not have been induced in the tolerant cultivar under the same stress level. <italic> <span class="gene">GmaxRD20A-</span> </italic> like and <italic> <span class="gene">GmaxRD22-</span> </italic> like did not show such responses, which may indicate that <italic> <span class="gene">GmaxERD1-</span> </italic> like belongs to an alternative response pathway. In fact, <italic> <span class="gene">GmaxERD1-</span> </italic> like should be part of the ABA-dependent and <italic> <span class="gene">GmaxRD20A-</span> </italic> like and <italic> <span class="gene">GmaxRD22-</span> </italic> like of the ABA-independent pathway. However, more genes from each pathway are needed to confirm this hypothesis. <xref ref-type="bibr" rid="b47-gmb-35-1-sup-212"> Stolf-Moreira <italic> et al. </italic> (2011) </xref> identified other putative <italic> Arabidopsis ERD1 </italic> homolog genes in the <italic> G.max </italic> genome, named <italic> <span class="gene">GmERD1</span> </italic> , which showed low expression in the tolerant cultivar in all <span class="trait">drought</span> conditions tested in the HSys condition. <italic> <span class="gene">GmERD1</span> </italic> was not included in our analysis because it has a very low similarity to <italic> Arabidopsis ERD1 </italic> (data not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmaxRD20A-</span> </italic> like showed a similar expression profile in the sensitive and tolerant cultivars. However, <italic> <span class="gene">GmaxRD20A-</span> </italic> like was more expressed in the tolerant cultivar under severe stress (&#x003a8;w &#x02212;3.0 MPa) in PSys ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-212"> Figure 2A </xref> ). In <italic> Arabidopsis </italic> , the RD20A gene is regulated by the AREB1/ABF2 transcription factor, which is also involved in the regulation of the RD29B gene ( <xref ref-type="bibr" rid="b17-gmb-35-1-sup-212"> Fujita <italic> et al. </italic> , 2005 </xref> ; <xref ref-type="bibr" rid="b42-gmb-35-1-sup-212"> Shinozaki and Yamaguchi-Shinozaki, 2007 </xref> ). Even after exhaustive searches in the soybean genome, the most similar gene we found to <italic> Arabidopsis RD29B, <span class="gene">Glyma16g31330</span> </italic> , showed low similarity (e-value 10 <sup> &#x02212;18 </sup> ). Furthermore, this putative <italic> RD29B </italic> soybean homolog is not regulated by the water deficit stress according to our experiments (data not shown). These results indicate that <italic> RD29B </italic> homologs are absent in the soybean genome. Recently, it was shown that <italic> Arabidopsis RD29B </italic> is unlikely to perform any protective effects during <span class="trait">drought</span> stress ( <xref ref-type="bibr" rid="b35-gmb-35-1-sup-212"> Msanne <italic> et al. </italic> , 2011 </xref> ). Therefore, the lack of <italic> RD29B </italic> homologs in the soybean genome should not interfere in the tolerance of this species to <span class="trait">drought.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, we identified three genes as homologs of traditional <italic> Arabidopsis </italic> DMGs, which were induced by water deficit in soybean. The <italic> <span class="gene">GmaxRD20A-</span> </italic> like and <italic> <span class="gene">GmaxRD22-</span> </italic> like genes, homologs of <italic> Arabidopsis </italic> genes of the ABA-independent pathway, are highly induced by water deficit in the PSys condition, whereas <italic> <span class="gene">GmaxERD1-</span> </italic> like, a homolog of an <italic> Arabidopsis </italic> gene of the ABA-dependent pathway, was highly induced by water deficit in HSys. Hence, these three genes can be very useful as stress markers for studying water deficit in soybean. Moreover, the differences in gene expression in the two systems revealed by our work emphasize that both systems need to be employed to get a <italic> bona fide </italic> picture of the pathways activated by water deficit in soybean. Finally, the differences in gene expression of the <italic> <span class="gene">GmaxERD1-</span> </italic> like in the HSys condition between sensitive and tolerant cultivars are robust and may provide a means to better understand the <span class="trait">drought</span> tolerance phenotype in soybean cultivars.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Virus-induced down-regulation of <italic> <span class="gene">GmERA1A</span> </italic> and <italic> <span class="gene">GmERA1B</span> </italic> genes enhances the stomatal response to abscisic acid and <span class="trait">drought</span> <span class="trait">resistance</span> in soybean</h2>
			<h3>PMID:28419130</h3>
				

				
					<h3>PMCID:5395220</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ogata </surname>  <given-names> Takuya </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nagatoshi </surname>  <given-names> Yukari </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamagishi </surname>  <given-names> Noriko </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yoshikawa </surname>  <given-names> Nobuyuki </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujita </surname>  <given-names> Yasunari </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a major threat to global soybean production. The limited transformation potential and polyploid nature of soybean have hindered functional analysis of soybean genes. Previous research has implicated farnesylation in the plant's response to abscisic acid (ABA) and <span class="trait">drought</span> tolerance. We therefore used virus-induced gene silencing (VIGS) to evaluate farnesyltransferase genes, <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> (Glycine max Enhanced Response to ABA1-A and -B), as potential targets for increasing <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Apple latent spherical virus (ALSV)-mediated <span class="gene">GmERA1-down-regulated</span> soybean leaves displayed an enhanced stomatal response to ABA and reduced water loss and wilting under dehydration conditions, suggesting that <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> negatively regulate ABA signaling in soybean guard cells. The findings provide evidence that the ALSV-VIGS system, which bypasses the need to generate transgenic plants, is a useful tool for analyzing gene function using only a single down-regulated leaf. Thus, the ALSV-VIGS system could constitute part of a next-generation molecular breeding pipeline to accelerate <span class="trait">drought</span> <span class="trait">resistance</span> breeding in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The plant hormone ABA coordinates the plant&#x02019;s responses to reduced water availability and influences diverse developmental processes [ <xref rid="pone.0175650.ref025" ref-type="bibr"> 25 </xref> &#x02013; <xref rid="pone.0175650.ref027" ref-type="bibr"> 27 </xref> ]. Seed maturation and osmotic stresses such as <span class="trait">drought</span> and high salinity result in cellular dehydration, which increases endogenous ABA levels to trigger multiple developmental and physiological responses, including stomatal closure and changes in gene expression [ <xref rid="pone.0175650.ref006" ref-type="bibr"> 6 </xref> , <xref rid="pone.0175650.ref028" ref-type="bibr"> 28 </xref> ]. Many key regulators of ABA signaling have been identified using forward and reverse genetic approaches, primarily in <italic> Arabidopsis </italic> . The <italic> Arabidopsis </italic> ABA-hypersensitive mutant <italic> era1 </italic> ( <italic> enhanced response to ABA1 </italic> ), harboring a disruption or deletion of the gene for the &#x003b2;-subunit of protein farnesyltransferase [ <xref rid="pone.0175650.ref029" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="pone.0175650.ref032" ref-type="bibr"> 32 </xref> ], exhibits enhanced ABA-induced stomatal closure due to the activation of a guard cell S-type anion channel and increased cytosolic Ca <sup> 2+ </sup> levels in guard cells [ <xref rid="pone.0175650.ref029" ref-type="bibr"> 29 </xref> ]. Although the role of ERA1-mediated protein farnesylation remains unclear, farnesylated CYP85A2 (a cytochrome <span class="gene">P450</span> enzyme implicated in brassinosteroid biosynthesis) and ASG2 (a WD40 protein involved in seed germination) proteins have recently been shown to act as negative regulators of ABA responses in <italic> Arabidopsis </italic> [ <xref rid="pone.0175650.ref033" ref-type="bibr"> 33 </xref> , <xref rid="pone.0175650.ref034" ref-type="bibr"> 34 </xref> ]. Furthermore, repression of <italic> ERA1 </italic> has been shown to enhance <span class="trait">drought</span> <span class="trait">resistance</span> in <italic> Arabidopsis </italic> [ <xref rid="pone.0175650.ref031" ref-type="bibr"> 31 </xref> , <xref rid="pone.0175650.ref035" ref-type="bibr"> 35 </xref> ], <italic> Brassica napus </italic> (canola) [ <xref rid="pone.0175650.ref035" ref-type="bibr"> 35 </xref> , <xref rid="pone.0175650.ref036" ref-type="bibr"> 36 </xref> ], <italic> Triticum aestivum </italic> (wheat) [ <xref rid="pone.0175650.ref037" ref-type="bibr"> 37 </xref> ], and <italic> Oryza sativa </italic> (rice) [ <xref rid="pone.0175650.ref038" ref-type="bibr"> 38 </xref> ]. Therefore, <italic> ERA1 </italic> represents a promising candidate gene for increasing <span class="trait">drought</span> <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we employed VIGS to evaluate whether <italic> <span class="gene">GmERA1A</span> </italic> and <italic> <span class="gene">GmERA1B</span> </italic> , which are homologs of <italic> Arabidopsis ERA1 </italic> , enhance <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Soybean leaves subjected to ALSV-mediated <italic> <span class="gene">GmERA1s</span> </italic> -down-regulation showed an increased stomatal closure response to ABA and reduced water loss, gas exchange, and wilting under water-limiting conditions compared to control plants. These results suggest that <italic> <span class="gene">GmERA1</span> </italic> s act as negative regulators of ABA signaling during stomatal responses in soybean under dehydration conditions. These findings support the proposal that <italic> ERA1 </italic> can be downregulated to increase <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. In addition, they demonstrate that the ALSV-VIGS system is a useful tool for evaluating candidate drought-resistance genes in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis ERA1 </italic> plays an important role in ABA signaling [ <xref rid="pone.0175650.ref029" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="pone.0175650.ref031" ref-type="bibr"> 31 </xref> ] and is a promising candidate gene for genetic manipulation of <span class="trait">drought</span> stress tolerance in <italic> Arabidopsis </italic> and canola plants [ <xref rid="pone.0175650.ref035" ref-type="bibr"> 35 </xref> , <xref rid="pone.0175650.ref036" ref-type="bibr"> 36 </xref> ]. Although <span class="trait">drought</span> is considered to be a major problem in soybean production worldwide [ <xref rid="pone.0175650.ref043" ref-type="bibr"> 43 </xref> &#x02013; <xref rid="pone.0175650.ref045" ref-type="bibr"> 45 </xref> ], <italic> ERA1 </italic> homologs in soybean have not been reported to date. Therefore, we searched the Soybean Knowledge Base (SoyKB, <ext-link ext-link-type="uri" xlink:href="http://soykb.org/"> http://soykb.org/ </ext-link> ) using BlastP program and the amino acid sequence of <italic> Arabidopsis </italic> ERA1 as query. Two soybean genes, <italic> <span class="gene">Glyma06g19740</span> </italic> and <italic> <span class="gene">Glyma13g23780</span> </italic> , were identified as homologs of <italic> Arabidopsis ERA1 </italic> and named <italic> <span class="gene">GmERA1A</span> </italic> and <italic> <span class="gene">GmERA1B</span> </italic> , respectively ( <xref ref-type="supplementary-material" rid="pone.0175650.s001"> S1 </xref> and <xref ref-type="supplementary-material" rid="pone.0175650.s002"> S2 </xref> Figs, <xref ref-type="supplementary-material" rid="pone.0175650.s011"> S2 Table </xref> ). <xref ref-type="supplementary-material" rid="pone.0175650.s002"> S2 Fig </xref> shows the sequence alignment of ERA1 proteins from soybean and <italic> Arabidopsis </italic> . <span class="gene">GmERA1A</span> and <span class="gene">GmERA1B</span> share 56.0% and 56.3% amino acid sequence identity with <italic> Arabidopsis </italic> ERA1, respectively. <span class="gene">GmERA1A</span> shares 92.9% amino acid sequence identity with <span class="gene">GmERA1B.</span> These data are consistent with the previous finding that soybean is a partially diploidized tetraploid [ <xref rid="pone.0175650.ref012" ref-type="bibr"> 12 </xref> ]. Although these proteins have high levels of amino acid sequence identity, the N-terminus of <italic> Arabidopsis </italic> ERA1 is longer than that of <span class="gene">GmERA1A</span> or <span class="gene">GmERA1B</span> ( <xref ref-type="supplementary-material" rid="pone.0175650.s002"> S2 Fig </xref> ). Five prenyltransferase domains are well-conserved in the ERA proteins ( <xref ref-type="supplementary-material" rid="pone.0175650.s002"> S2 Fig </xref> ). An analysis of public databases in the Soybean and Arabidopsis eFP Browsers ( <ext-link ext-link-type="uri" xlink:href="http://bar.utoronto.ca/"> http://bar.utoronto.ca/ </ext-link> )) showed that both <italic> <span class="gene">GmERA1A</span> </italic> and <italic> <span class="gene">GmERA1B</span> </italic> , like <italic> Arabidopsis ERA1 </italic> , are expressed in various tissues ( <xref ref-type="supplementary-material" rid="pone.0175650.s003"> S3 Fig </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> ERA1 </italic> , which is expressed in guard cells, regulates the transpiration rate in <italic> Arabidopsis </italic> by modulating ABA-mediated stomatal responses [ <xref rid="pone.0175650.ref029" ref-type="bibr"> 29 </xref> , <xref rid="pone.0175650.ref031" ref-type="bibr"> 31 </xref> ]. We therefore measured water loss in the leaves of <italic> <span class="gene">GmERA1</span> </italic> -down-regulated plants to evaluate their response to <span class="trait">drought</span> stress. The water loss rate was lower in the detached fourth or fifth trifoliate leaves of soybean plants inoculated with ALSV-GmERA1s than in those of plants inoculated with ALSV-VC ( <xref ref-type="fig" rid="pone.0175650.g002"> Fig 2A </xref> ), suggesting that transpiration from the leaves of <italic> <span class="gene">GmERA1</span> </italic> -down-regulated plants was reduced under limited water conditions. This result is consistent with our infrared thermography observation that the leaf temperature of detached fourth trifoliate leaflets of plants inoculated with ALSV-GmERA1s was higher than that of detached fourth trifoliate leaflets of plants inoculated with ALSV-VC ( <xref ref-type="fig" rid="pone.0175650.g002"> Fig 2B </xref> ). To confirm these results, we analyzed the leaf surface temperature over time using a LI-6400XT portable photosynthesis system. At 3 or more min after detachment, the surface temperature of detached fourth or fifth trifoliate leaflets of <italic> <span class="gene">GmERA1</span> </italic> -down-regulated plants was ~1&#x000b0;C higher than that of control plants ( <xref ref-type="fig" rid="pone.0175650.g002"> Fig 2C </xref> ). Furthermore, using the LI-6400XT system, we measured the rate of gas exchange in the detached <italic> <span class="gene">GmERA1</span> </italic> -down-regulated leaflets over time. At 4 to 16 min after detachment, the water-stressed fourth or fifth trifoliate leaves of <italic> <span class="gene">GmERA1</span> </italic> -down-regulated plants exhibited statistically significant reductions in stomatal conductance compared to that of control plants inoculated with ALSV-VC empty viral vector, suggesting that down-regulation of <italic> <span class="gene">GmERA1s</span> </italic> reduces gas exchange under water-limiting conditions ( <xref ref-type="fig" rid="pone.0175650.g002"> Fig 2D </xref> ). These results support the notion that down-regulation of <italic> <span class="gene">GmERA1s</span> </italic> reduces water loss from water-stressed soybean leaves through its influence on stomatal regulation.
					</p>
				
					<p>&nbsp;&nbsp;
						We demonstrated that the <italic> <span class="gene">GmERA1</span> </italic> -down-regulated leaves displayed reduced water loss and gas exchange through ABA-mediated stomatal regulation compared to the vector control plants. Next, we examined whether whole <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants exhibit enhanced <span class="trait">drought</span> tolerance. When watering was withheld from soybean plants at the V6 growth stage [ <xref rid="pone.0175650.ref039" ref-type="bibr"> 39 </xref> ] for 24 h, the leaf temperature was higher in plants inoculated with ALSV-GmERA1N and ALSV-GmERA1C than in mock- or ALSV-VC-inoculated plants ( <xref ref-type="supplementary-material" rid="pone.0175650.s008"> S8 Fig </xref> ). However, during the first 24 h of dehydration, no wilting was observed among the plants ( <xref ref-type="supplementary-material" rid="pone.0175650.s008"> S8 Fig </xref> ). We then scored leaf rolling in the fourth to sixth trifoliate leaves after approximately 60 h of dehydration stress treatment in six independent experiments ( <xref ref-type="fig" rid="pone.0175650.g004"> Fig 4A </xref> ). At 60 h post-treatment, the leaf rolling scores of the plants were similar ( <xref ref-type="supplementary-material" rid="pone.0175650.s009"> S9 Fig </xref> ). However, after 72&#x02013;75 h of dehydration stress treatment, although ambient humidity could not be controlled under our experimental conditions, in five out of six independent experiments, the ratio of severely wilted leaves to total leaves (score 3) was lower in <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants than in the control, indicating that in many cases, the <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants exhibited less wilting than the control ( <xref ref-type="fig" rid="pone.0175650.g004"> Fig 4B </xref> ). At one day after rehydration, the survival rates of the plants in each experiment were quite variable ( <xref ref-type="fig" rid="pone.0175650.g004"> Fig 4C </xref> , <xref ref-type="supplementary-material" rid="pone.0175650.s012"> S3 Table </xref> ). Nevertheless, taking all six independent experiments into account, approximately 60% of the <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants survived, whereas only approximately 30% of the control plants survived ( <xref ref-type="fig" rid="pone.0175650.g004"> Fig 4C </xref> , <xref ref-type="supplementary-material" rid="pone.0175650.s012"> S3 Table </xref> ). Overall, the <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants survived dehydration stress better than did the vector control plants in terms of wilting, supporting the view that <italic> <span class="gene">GmERA1</span> </italic> -down-regulated soybean plants exhibit enhanced <span class="trait">resistance</span> to water-deficit conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Our data show that <italic> <span class="gene">GmERA1</span> </italic> -down-regulated leaves displayed reduced water loss rates under water-limiting conditions through ABA-mediated stomatal regulation and, consequently, that these plants exhibited enhanced <span class="trait">resistance</span> to water deficit conditions, consistent with previous studies of plants with similar levels of down-regulation of <italic> ERA1 </italic> homologs [ <xref rid="pone.0175650.ref035" ref-type="bibr"> 35 </xref> &#x02013; <xref rid="pone.0175650.ref037" ref-type="bibr"> 37 </xref> ]. These findings demonstrate that <italic> <span class="gene">GmERA1s</span> </italic> function as negative regulators of ABA signaling in guard cells in soybean. Therefore, <italic> <span class="gene">GmERA1s</span> </italic> appear to play an important role in the dehydration stress response in soybean, as does <italic> ERA1 </italic> in <italic> Arabidopsis </italic> [ <xref rid="pone.0175650.ref029" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="pone.0175650.ref031" ref-type="bibr"> 31 </xref> , <xref rid="pone.0175650.ref035" ref-type="bibr"> 35 </xref> ]. Our data support the hypothesis that <italic> ERA1 </italic> genes can be downregulated to increase sensitivity to ABA, thereby leading to more rapid ABA-induced stomatal closure under water-limiting conditions. Thus, our study suggests a useful strategy for enhancing <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Plants harboring mutations in <italic> <span class="gene">GmERA1s</span> </italic> could be readily identified through high-throughput methods, such as targeting-induced local lesions in genomes (TILLING), high-resolution melting (HRM), and indexed amplicon sequencing by next generation sequencing (NGS), taking advantage of high-density mutant resources that have already been developed in soybean [ <xref rid="pone.0175650.ref049" ref-type="bibr"> 49 </xref> &#x02013; <xref rid="pone.0175650.ref051" ref-type="bibr"> 51 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Nonetheless, the weaknesses of the VIGS system described above can also be perceived as strengths. Although the growth of <italic> ERA1 </italic> -repressed <italic> Arabidopsis </italic> plants is retarded [ <xref rid="pone.0175650.ref055" ref-type="bibr"> 55 </xref> &#x02013; <xref rid="pone.0175650.ref058" ref-type="bibr"> 58 </xref> ], under our experimental conditions, no significant negative effect on plant growth was observed in the <italic> <span class="gene">GmERA1</span> </italic> -down-regulated plants ( <xref ref-type="supplementary-material" rid="pone.0175650.s007"> S7 Fig </xref> ). As specific leaf positions and growth stages (e.g., the fourth trifoliate leaves at the V4 growth stage) are more readily recognized in soybean than in other staple crops such as wheat and <italic> Solanum lycopersicum </italic> (tomato) [ <xref rid="pone.0175650.ref039" ref-type="bibr"> 39 </xref> ], the ALSV-VIGS system is ideal for validating gene function in soybean. Thus, our findings demonstrate that the ALSV-VIGS system is a useful tool for validating candidate genes without the need for generating transgenic plants; however, the selected candidate genes will need to be evaluated using appropriate mutant or transgenic plants to assess the impact on physiological parameters, such as soil water contents, before applying this technology in the field. Therefore, the ALSV-VIGS system, combined with recent methods for screening for desired mutations from high-density mutant resources and the CRISPR/CAS9-mediated introduction of desired mutations, could contribute to next-generation molecular breeding strategies to improve <span class="trait">drought</span> <span class="trait">resistance</span> in soybean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Salt Tolerance 1 ( <italic> <span class="gene">GmST1</span> </italic> ) Reduces ROS Production, Enhances ABA Sensitivity, and Abiotic Stress Tolerance in <italic> Arabidopsis thaliana </italic></h2>
			<h3>PMID:27148284</h3>
				

				
					<h3>PMCID:4826999</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ren </surname>  <given-names> Shuxin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/305911/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lyle </surname>  <given-names> Chimera </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Guo-liang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/229420/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Penumala </surname>  <given-names> Abhishek </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Abiotic stresses, including high soil salinity, significantly reduce crop production worldwide. Salt tolerance in plants is a complex trait and is regulated by multiple mechanisms. Understanding the mechanisms and dissecting the components on their regulatory pathways will provide new insights, leading to novel strategies for the improvement of salt tolerance in agricultural and economic crops of importance. Here we report that soybean salt tolerance 1, named <span class="gene">GmST1,</span> exhibited strong tolerance to salt stress in the Arabidopsis transgenic lines. The <span class="gene">GmST1-overexpressed</span> Arabidopsis also increased sensitivity to ABA and decreased production of reactive oxygen species under salt stress. In addition, <span class="gene">GmST1</span> significantly improved <span class="trait">drought</span> tolerance in Arabidopsis transgenic lines. <span class="gene">GmST1</span> belongs to a 3-prime part of <span class="gene">Glyma.03g171600</span> gene in the current version of soybean genome sequence annotation. However, comparative reverse transcription-polymerase chain reaction analysis around <span class="gene">Glyma.03g171600</span> genomic region confirmed that <span class="gene">GmST1</span> might serve as an intact gene in soybean leaf tissues. Unlike <span class="gene">Glyma.03g171600</span> which was not expressed in leaves, <span class="gene">GmST1</span> was strongly induced by salt treatment in the leaf tissues. By promoter analysis, a TATA box was detected to be positioned close to <span class="gene">GmST1</span> start codon and a putative ABRE and a DRE cis-acting elements were identified at about 1 kb upstream of <span class="gene">GmST1</span> gene. The data also indicated that <span class="gene">GmST1-transgenic</span> lines survived under <span class="trait">drought</span> stress and showed a significantly lower water loss than non-transgenic lines. In summary, our results suggest that overexpression of <span class="gene">GmST1</span> significantly improves Arabidopsis tolerance to both salt and <span class="trait">drought</span> stresses and the gene may be a potential candidate for genetic engineering of salt- and drought-tolerant crops.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Previously, we characterized soybean salt tolerance in WF-7 and mapped a major QTL in the same region on linkage group N that was closely linked with a microsatellite marker, JD33&#x02013;432 ( <xref rid="B45" ref-type="bibr"> Ren et al., 2009 </xref> , <xref rid="B46" ref-type="bibr"> 2012 </xref> ). Using the first draft of the soybean genome sequence, and through bioinformatics analysis, we identified two putative cation/proton antiporters on scaffold 63 nearby JD33&#x02013;432. These two genes were, at the time, named <span class="gene">Gm0063&#x000d7;00340</span> and <span class="gene">Gm0063&#x000d7;00341.</span> According to the update soybean sequence database available currently, these two genes are overlapped with <span class="gene">Glyma03g32900</span> ( <xref rid="B21" ref-type="bibr"> Guan et al., 2014b </xref> ; <xref rid="B44" ref-type="bibr"> Qi et al., 2014 </xref> ) and are predicted as one gene, <span class="gene">Glyma.03g171600</span> ( <xref rid="B44" ref-type="bibr"> Qi et al., 2014 </xref> ), in the current Wm82.a2 assembly. Here we present the evidence that overexpression of <span class="gene">Gm0063&#x000d7;00340,</span> renamed <italic> <span class="gene">GmST1</span> </italic> , produced strong tolerance to salt stress in <italic> Arabidopsis </italic> seedling and adult stages. In addition, the <italic> <span class="gene">GmST1</span> </italic> -transgenic <italic> Arabidopsis </italic> lines also increased sensitivity to ABA and decreased production of ROS under salt stress. Furthermore, the <italic> Arabidopsis <span class="gene">GmST1</span> </italic> -transgenic lines also improved tolerance to <span class="trait">drought</span> stress. Based on the current version of soybean genome sequence annotation, <italic> <span class="gene">GmST1</span> </italic> was located within <span class="gene">Glyma03g171600,</span> and <span class="gene">Glyma03g171600</span> overlaps with <span class="gene">Glyma03g32900.</span> Thus we propose that at least two different models of gene prediction exist for this region. By comparative RT-PCR analysis, <italic> <span class="gene">GmST1</span> </italic> has been demonstrated as an intact gene functioning in soybean leaf tissues, not a part of <span class="gene">Glyma03g171600.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						To produce <italic> Arabidopsis </italic> transgenic lines, <italic> Arabidopsis </italic> ecotype Col-0 was used. The DNA fragment covering start and stop codons of <italic> <span class="gene">GmST1</span> </italic> was amplified from WF-7 genomic DNA using primers <span class="gene">GmST1-F</span> (P4) and <span class="gene">GmST1-R</span> (P6), cloned into TOPO vector PCR2.1 and sequence-confirmed. The confirmed DNA fragment was then inserted into binary vector pCBK05 between XbaI (5&#x02032; end) and SacI (3&#x02032; end) sites, driven by the CaMV 35S promoter. The plasmid containing 35S::GmST1 construct was transformed first into <italic> Agrobacterium tumefaciens </italic> GV3101 and then into <italic> Arabidopsis </italic> ecotype Col-0 by the floral dipping method ( <xref rid="B15" ref-type="bibr"> Clough and Bent, 1998 </xref> ). Two independent single-copy homozygous transgenic lines were generated through herbicide <span class="trait">resistance</span> selection. Briefly, primary transformants were identified and determined based on the presence of herbicide <span class="trait">resistance</span> that is completely linked with the target gene. Individual transgenic lines were further genetically analyzed using herbicide <span class="trait">resistance</span> as the indicator in the following T <sub> 1 </sub> and T <sub> 2 </sub> generations to identify single copy T-DNA insertion lines. Two independent homozygous transgenic lines were then bred and the seeds from T <sub> 3 </sub> and/or T <sub> 4 </sub> generations were subjected to various stress tests.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine the possible role of <italic> <span class="gene">GmST1</span> </italic> in <span class="trait">drought</span> stress tolerance, we first conducted water loss experiments using detached leaves. Three leaves were excised from each 4-week-old plant grown in the growth chamber, and the fresh weight was immediately determined. All leaves were placed on a laboratory bench at room temperature for 210 min. Every 30 min, leaf weight was recorded in the same order that leaves were detached from their corresponding plants. Water loss was calculated as the percentage of the initial fresh weight at each time point. For each genotype (Col-0, <italic> <span class="gene">GmST1&#x02013;1</span> </italic> , and <italic> <span class="gene">GmST1&#x02013;2</span> </italic> ), five plants were tested for water loss.
					</p>
				
					<p>&nbsp;&nbsp;
						To test <span class="trait">drought</span> tolerance at adult stage, wild type and a transgenic line were grown under normal watering conditions for 4 weeks and then subjected to <span class="trait">drought</span> stress by withdrawing irrigation. After 11 days of <span class="trait">drought</span> stress, <span class="trait">drought</span> tolerance phenotypes were examined, and pictures were captured to show the difference between the wild type and the transgenic line. A total of 15 plants for each of Col-0 and <italic> <span class="gene">GmST1&#x02013;1</span> </italic> were evaluated for <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine the role of <italic> <span class="gene">GmST1</span> </italic> in salt tolerance in plants, we cloned a genomic DNA fragment covering the full length of cDNA of <italic> <span class="gene">GmST1</span> </italic> from soybean variety WF-7. Driven by CAMV 35S promoter, we transferred the construct into <italic> Arabidopsis </italic> ecotype Col-0. Transformants were selected based on herbicide <span class="trait">resistance</span> as the maker, and two independent homozygous transgenic lines carrying a single copy of T-DNA insertion were developed for further characterization of the roles of <italic> <span class="gene">GmST1</span> </italic> in abiotic stress tolerance. RT-PCR analysis showed high expression of <italic> <span class="gene">GmST1</span> </italic> in both <italic> Arabidopsis </italic> transgenic lines ( <bold> Figure <xref ref-type="fig" rid="F1"> 1 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Since both ABA and ROS are regarded as classical signal pathways that plants evolved for defending against abiotic stresses, the fact that <italic> <span class="gene">GmST1</span> </italic> both increased ABA sensitivity and decreased ROS production under salt stress prompted us to further examine if it could also strengthen <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> . We first conducted water loss experiments using detached leaves. As shown in <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> , during a 210-min period, wild type Col-0 lost more than 40% of water due to evaporation, while two <italic> <span class="gene">GmST1</span> </italic> transgenic lines evaporated only less than 30% of water. The significant difference in water loss between Col-0 and the transgenic lines indicated a possible role of <italic> <span class="gene">GmST1</span> </italic> in plants&#x02019; <span class="trait">drought</span> tolerance. We further compared the <span class="trait">drought</span> tolerance between Col-0 and a transgenic line for an extended period of <span class="trait">drought</span> stress. After 11 days of water withholding, Col-0 plants died of stress ( <bold> Figure <xref ref-type="fig" rid="F6"> 6B </xref> </bold> ), while almost all the plants from the transgenic line survived. It suggests that <italic> <span class="gene">GmST1</span> </italic> may play an important role in <span class="trait">drought</span> stress tolerance in plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Effect of <span class="gene">GmST1</span> overexpression on <italic> Arabidopsis </italic> <span class="trait">drought</span> tolerance. (A) </bold> Overexpression of <span class="gene">GmST1</span> reduces water loss through transpiration. Leaves were excised from 4-week-old <italic> Arabidopsis </italic> lines (Col-0, <span class="gene">GmST1&#x02013;1,</span> and <span class="gene">GmST1&#x02013;1)</span> and monitored their water losses under ambient temperature for 210 min. Water loss was calculated as the percentage of initial fresh weight. <bold> (B) </bold> Overexpression of <span class="gene">GmST1</span> confers <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> . 4-week-old <italic> Arabidopsis </italic> plants (Col-0 and <span class="gene">GmST1&#x02013;1)</span> were subjected to a water stress by withholding water. Photos were taken after 11 days of <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In the <span class="gene">Glyma.03g171600</span> model, <span class="gene">Gm0063&#x000d7;00341</span> belongs to the 5&#x02032; part of the <span class="gene">Glyma.03g171600</span> gene, and the primer P3 ( <bold> Figure <xref ref-type="fig" rid="F7"> 7A </xref> </bold> ), located at the 3&#x02032; end of <span class="gene">Gm0063&#x000d7;00341,</span> is predicted to be in the intron of the <span class="gene">Glyma.03g171600</span> gene. We conducted RT-PCR using <span class="gene">P1</span> and P3. If <span class="gene">Gm0063&#x000d7;00341</span> was an intact gene, then the <span class="gene">P1</span> and P3 combination should <span class="trait">yield</span> 720 bp RT-PCR product; and if the <span class="gene">Glyma.03g171600</span> prediction model was correct, then it should not amplify any RT-PCR product. Shown in <bold> Figure <xref ref-type="fig" rid="F7"> 7B </xref> </bold> , we successfully amplified an RT-PCR product at about 720 bp in both salt-treated and untreated plants of the variety Union. These results clearly demonstrate that both <italic> <span class="gene">GmST1</span> </italic> and <span class="gene">Gm0063&#x000d7;00341</span> are not part of the <span class="gene">Glyma.03g171600</span> gene in soybean leaf tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						ABA is considered an abiotic stress hormone, and in most cases, an ABA-dependent signaling pathway regulates plant responses to multiple stresses ( <xref rid="B28" ref-type="bibr"> Larkindale and Knight, 2002 </xref> ; <xref rid="B67" ref-type="bibr"> Zhu, 2002 </xref> ; <xref rid="B19" ref-type="bibr"> Fujita et al., 2005 </xref> ; <xref rid="B12" ref-type="bibr"> Chen L.T. et al., 2010 </xref> ). Thus, we further tested <italic> <span class="gene">GmST1</span> </italic> &#x02019;s ability to defend against <span class="trait">drought</span> stress, the major cause of crop losses worldwide. The results indicated that <italic> <span class="gene">GmST1</span> </italic> expression prevented water loss from leaves and improved <span class="trait">drought</span> tolerance during the adult stage ( <bold> Figure <xref ref-type="fig" rid="F6"> 6 </xref> </bold> ). This is not surprising, though, as many other cation/proton antiporters from different species also enhanced both salt and <span class="trait">drought</span> tolerance when being overexpressed in <italic> Arabidopsis </italic> ( <xref rid="B9" ref-type="bibr"> Brini et al., 2007 </xref> ; <xref rid="B58" ref-type="bibr"> Wei et al., 2011 </xref> ). Such dual functions in salt and <span class="trait">drought</span> tolerance would allow scientists to efficiently engineer the abiotic stress tolerance of crops, thus leading to improved agricultural production on lands that are marginal due to salt and/or <span class="trait">drought</span> problems. Both <span class="trait">drought</span> and salt tolerance in soybean were shown to be related to the accumulation of soluble saccharides and proline, and less damage of chlorophyll when stresses occurred ( <xref rid="B1" ref-type="bibr"> Abd El-Samad and Shaddad, 1997 </xref> ). Such physiological traits and Na <sup> + </sup> , K <sup> + </sup> , and Cl <sup> - </sup> accumulations in shoots, together with possible alternations of the expressions for genes on ABA signaling pathways, would be excellent targets for the future investigation on the precise physiological and molecular mechanism(s) that drive <italic> <span class="gene">GmST1</span> </italic> &#x02019;s abiotic stress tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently <span class="gene">Glyma.03g171600</span> was identified as <italic> <span class="gene">GmCHX1</span> </italic> and contributed to soybean salt tolerance ( <xref rid="B44" ref-type="bibr"> Qi et al., 2014 </xref> ). The intact <italic> <span class="gene">GmCHX1</span> </italic> gene was highly expressed in soybean roots, but was barely detected in soybean shoots ( <xref rid="B21" ref-type="bibr"> Guan et al., 2014b </xref> ; <xref rid="B44" ref-type="bibr"> Qi et al., 2014 </xref> ). Through RT-PCR analysis, we were able to examine if <italic> <span class="gene">GmST1</span> </italic> and <span class="gene">Gm0063x0041</span> belong to parts of <italic> <span class="gene">GmCHX1</span> </italic> or if they act as individual genes. Using primer combinations of <span class="gene">Gm0063x0041</span> forward (P1 and P2, respectively) and <italic> <span class="gene">GmST1</span> </italic> reverse (P6), we failed to amplify any RT-PCR products from either salt-treated or untreated soybean leaves. Though, we could not rule out the <italic> <span class="gene">GmCHX1</span> </italic> model given that <italic> <span class="gene">GmCHX1</span> </italic> was not expressed in leaf tissues ( <xref rid="B21" ref-type="bibr"> Guan et al., 2014b </xref> ; <xref rid="B44" ref-type="bibr"> Qi et al., 2014 </xref> ). By examining the expressions of <italic> <span class="gene">GmST1</span> </italic> and <span class="gene">Gm0063x0041</span> as individual genes, however, we found that both genes were expressed in soybean leaves. <italic> <span class="gene">GmST1</span> </italic> was highly induced by salt treatment in both salt-sensitive and salt-tolerant soybean varieties, but less expressed in the conditions with no salt treatment. On the other hand, <span class="gene">Gm0063x0041</span> was highly expressed in the salt-sensitive soybean Union under both control and salt-treated conditions but only a very low level of expression was detected in the salt-tolerant soybean WF-7. Interestingly, according to the <italic> <span class="gene">GmCHX1</span> </italic> model, <span class="gene">Gm0063x0041</span> reverse primer (P3) was located in the intron of <italic> <span class="gene">GmCHX1</span> </italic> and should not be able to amplify the RT-PCR product if the <italic> <span class="gene">GmCHX1</span> </italic> model was correct, but it did amplify the RT-PCR product. Furthermore, we designed another reverse primer, P5, to further verify the model. In the <italic> <span class="gene">GmST1</span> </italic> model, <span class="gene">P5</span> was located in the intron region, but in the <italic> <span class="gene">GmCHX1</span> </italic> model, it was located at 3&#x02032; UTR. Compared with the P4+P6 combination, the P4+P5 combination did not <span class="trait">yield</span> any RT-PCR product at the predicted size. It is clearly indicated that both <italic> <span class="gene">GmST1</span> </italic> and <span class="gene">Gm0063&#x000d7;00341</span> are not part of the <span class="gene">Glyma.03g171600</span> gene but two individual genes in soybean leaf tissues, though they are overlapped by the latter. It also implies that the current <span class="gene">Glyma.03g171600</span> gene annotation might need further revision. Based on our data and those reported by <xref rid="B21" ref-type="bibr"> Guan et al. (2014b) </xref> and <xref rid="B44" ref-type="bibr"> Qi et al. (2014) </xref> , we hypothesize that different intron-exon splicing sites exist in this genomic region. In soybean root tissues, a splicing pattern forms one gene, <italic> <span class="gene">GmCHX1</span> </italic> , while in soybean leaf tissues, different splicing patterns create two distinct genes in the same genomic region. This phenomena is an exciting observation and warrants further investigation into the details of intron-exon site reorganization in different organs and genome structures in this genomic region.
					</p>
				
					<p>&nbsp;&nbsp;
						We have cloned <italic> <span class="gene">GmST1</span> </italic> from the salt-tolerant soybean variety WF-7, and we have demonstrated that <italic> <span class="gene">GmST1</span> </italic> enhances both salt and <span class="trait">drought</span> tolerance through <italic> Arabidopsis in planta </italic> study. Furthermore, <italic> <span class="gene">GmST1</span> </italic> also increases ABA sensitivity and reduces ROS production upon salt treatment. Additionally, <italic> <span class="gene">GmST1</span> </italic> is strongly induced by salt stress in soybean leaf tissues and functions as an intact gene, rather than as a part of the <span class="gene">Glyma.03g171600</span> gene. Further comparative studies between <italic> <span class="gene">GmST1</span> </italic> and <span class="gene">Glyma.03g171600,</span> as well as on the possible different intron-exon splicing patterns in different organs, will be required to fully describe the real gene structure and gene models within this genomic region. Regardless, the fact that <italic> <span class="gene">GmST1</span> </italic> overexpression improved both salt and <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> suggests that it may be worthwhile to elucidate the contribution of this gene to multi-abiotic tolerance in crops. The overexpression of <italic> <span class="gene">GmST1</span> </italic> in crops, including soybean, may provide a new approach to genetically engineering agricultural and economical plants of importance with tolerance to multiple abiotic stresses simultaneously.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Functional and regulatory conservation of the soybean ER stress-induced DCD/NRP-mediated cell death signaling in plants</h2>
			<h3>PMID:27405371</h3>
				

				
					<h3>PMCID:4943007</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Reis </surname>  <given-names> Pedro A. B. </given-names>  </name>  <address>  <email> pedrobqi@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Carpinetti </surname>  <given-names> Paola A. </given-names>  </name>  <address>  <email> paolacarpinetti@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Freitas </surname>  <given-names> Paula P.J. </given-names>  </name>  <address>  <email> ppjfreitas@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Santos </surname>  <given-names> Eul&#x000e1;lio G.D. </given-names>  </name>  <address>  <email> eulaliobqi@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Camargos </surname>  <given-names> Luiz F. </given-names>  </name>  <address>  <email> luizfcamargos@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oliveira </surname>  <given-names> Igor H.T. </given-names>  </name>  <address>  <email> igorhiroshi@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silva </surname>  <given-names> Jos&#x000e9; Cleydson F. </given-names>  </name>  <address>  <email> cleysinhonv@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Carvalho </surname>  <given-names> Humberto H. </given-names>  </name>  <address>  <email> humberto_henriquec@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dal-Bianco </surname>  <given-names> Maximiller </given-names>  </name>  <address>  <email> maximiller@ufv.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soares-Ramos </surname>  <given-names> Juliana R.L. </given-names>  </name>  <address>  <email> juliramos@ufv.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Fontes </surname>  <given-names> Elizabeth P. B. </given-names>  </name>  <address>  <email> bbfontes@ufv.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> Departamento de Bioqu&#x000ed;mica e Biologia Molecular, Universidade Federal de Vi&#x000e7;osa, Vi&#x000e7;osa, MG Brazil  </aff>  <aff id="Aff2">  <label/> National Institute of Science and Technology in Plant-Pest Interactions, Bioagro, Universidade Federal de Vi&#x000e7;osa, Vi&#x000e7;osa, MG Brazil  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The developmental and cell death domain (DCD)-containing asparagine-rich proteins (NRPs) were first identified in soybean (Glycine max) as transducers of a cell death signal derived from prolonged endoplasmic reticulum (ER) stress, osmotic stress, <span class="trait">drought</span> or developmentally-programmed leaf senescence via the <span class="gene">GmNAC81/GmNAC30/GmVPE</span> signaling module. In spite of the relevance of the DCD/NRP-mediated signaling as a versatile adaptive response to multiple stresses, mechanistic knowledge of the pathway is lacking and the extent to which this pathway may operate in the plant kingdom has not been investigated. RESULTS: Here, we demonstrated that the DCD/NRP-mediated signaling also propagates a stress-induced cell death signal in other plant species with features of a programmed cell death (PCD) response. In silico analysis revealed that several plant genomes harbor conserved sequences of the pathway components, which share functional analogy with their soybean counterparts. We showed that <span class="gene">GmNRPs,</span> <span class="gene">GmNAC81and</span> VPE orthologs from Arabidopsis, designated as AtNRP-1, AtNRP-2, ANAC036 and gVPE, respectively, induced cell death when transiently expressed in N. benthamiana leaves. In addition, loss of AtNRP1 and AtNRP2 function attenuated ER stress-induced cell death in Arabidopsis, which was in marked contrast with the enhanced cell death phenotype displayed by overexpressing lines as compared to Col-0. Furthermore, atnrp-1 knockout mutants displayed enhanced sensitivity to PEG-induced osmotic stress, a phenotype that could be complemented with ectopic expression of either <span class="gene">GmNRP-A</span> or <span class="gene">GmNRP-B.</span> In addition, AtNRPs, ANAC036 and gVPE were induced by osmotic and ER stress to an extent that was modulated by the ER-resident molecular chaperone binding protein (BiP) similarly as in soybean. Finally, as putative downstream components of the NRP-mediated cell death signaling, the stress induction of AtNRP2, ANAC036 and gVPE was dependent on the AtNRP1 function. BiP overexpression also conferred tolerance to water stress in Arabidopsis, most likely due to modulation of the drought-induced NRP-mediated cell death response. CONCLUSION: Our results indicated that the NRP-mediated cell death signaling operates in the plant kingdom with conserved regulatory mechanisms and hence may be target for engineering stress tolerance and adaptation in crops.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The developmental and cell death domain (DCD)-containing asparagine-rich proteins (NRPs) were first identified in soybean ( <italic> Glycine max </italic> ) as transducers of a cell death signal derived from prolonged endoplasmic reticulum (ER) stress, osmotic stress, <span class="trait">drought</span> or developmentally-programmed leaf senescence via the <span class="gene">GmNAC81/GmNAC30/GmVPE</span> signaling module. In spite of the relevance of the DCD/NRP-mediated signaling as a versatile adaptive response to multiple stresses, mechanistic knowledge of the pathway is lacking and the extent to which this pathway may operate in the plant kingdom has not been investigated.
					</p>
				
					<p>&nbsp;&nbsp;
						As a branch of the ER stress response that connects with other environmentally induced responses, the NRP&#x02013;mediated cell death signaling pathway may allow for the versatile adaptation of cells to different stresses [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. Accordingly, we have previously showed that this pathway is activated by <span class="trait">drought</span> and the modulation of this signaling event by the constitutive expression of the ER&#x02013;resident molecular chaperone binding protein (BiP) promotes a better adaptation of transgenic lines to <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. <italic> BiP </italic> overexpression also increased tolerance of soybean transgenic seedlings to tunicamycin, an inducer of ER stress, and to PEG, which induces osmotic stress [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. In soybean, BiP attenuates the propagation of the stress&#x02013;induced cell death signal by modulating the expression and activity of the components of the cell death pathway <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> A </italic> , G <italic> mNRP </italic> &#x02013; <italic> B </italic> , <italic> <span class="gene">GmNAC81</span> </italic> and <italic> VPE </italic> [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR26"> 26 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The stress&#x02013;induced NRP&#x02013;mediated cell death response has been shown to be modulated by BiP [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. Overexpression of <italic> soyBiPD </italic> <span class="gene">(Glyma.05G219400.1.p)</span> in soybean attenuates and delays the cell death response induced by osmotic stress, ER stress and <span class="trait">drought,</span> a phenotype that has been linked to the BiP&#x02013;mediated increases in the water stress tolerance [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. Among the Arabidopsis BiP paralogs, AtBiP1 is most related to AtBiP2 (98&#x000a0;% amino acid sequence identity) and they share the highest sequence conservation with soyBiPD (91&#x000a0;% sequence identity), whereas AtBiP3 is 77&#x000a0;% identical to soyBiPD. Thereby, <italic> AtBiP1 </italic> and <italic> AtBiP2 </italic> genes were selected to examine whether the NRP&#x02013;mediated cell death response in Arabidopsis would share similar regulatory mechanisms as in soybean. Then, we transformed Arabidopsis Col&#x02013;0 with <italic> soyBiPD </italic> and also with the Arabidopsis BiP genes, <italic> AtBiP1 </italic> and <italic> AtBiP2 </italic> , and monitored the BiP attenuation of the stress&#x02013;induced expression of pathway components. The ectopically expressed soybean <span class="gene">BiPD</span> protein accumulated to high levels in the independently transformed Arabidopsis T07, T10, T13, T23 lines and was correctly localized in microsomal fraction (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> a and b). UGPase was used as a cytosolic fraction&#x02013;associated marker to demonstrate that soyBiPD was confined to the microsomal fraction (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> c). Likewise, Arabidopsis transformed with AtBiP1&#x02013;GFP&#x02013;HDEL and AtBiP2&#x02013;GFP&#x02013;HDEL fusions accumulated higher levels of BiP mRNA (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> a) and protein (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> b) than Col&#x02013;0. Accumulation of BiP&#x02013;GFP&#x02013;HDEL was detected by immunoblotting total protein with anti&#x02013;GFP serum (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> b, lower blot) and the endogenous BiP levels&#x02009;+&#x02009;fusion proteins were monitored with an anti&#x02013;soyBiPD serum (upper blot). Like the endogenous BiPs (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> c, Col0), BiP&#x02013;GFP&#x02013;HDEL was correctly localized in the microsomal fraction (AtBiP1).
					</p>
				
					<p>&nbsp;&nbsp;
						The BiP&#x02013;mediated attenuation of the stress&#x02013;induced NRP&#x02013;mediated cell death response has been linked to its capacity to confer tolerance to <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. We next examined whether <italic> BiP </italic> overexpression in Arabidopsis also conferred tolerance to <span class="trait">drought.</span> For the <span class="trait">drought</span> treatment, water was withheld from 5&#x02013;week&#x02013;old plants for 20&#x000a0;days and the pictures and samples were taken at the time points, as indicated in Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> . A water stress tolerant phenotype was clearly developed by the transgenic lines overexpressing <italic> soyBiPD </italic> (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8a </xref> ), <italic> AtBiP1 </italic> and <italic> AtBiP2 </italic> (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ). This phenotype was typical of tobacco and soybean <italic> BiP </italic> &#x02013;overexpressing lines, such as maintenance of leaf turgidity (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8a </xref> and Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ), higher relative water content (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8b </xref> ) under a water deficit regime and attenuation of drought&#x02013;mediated induction of the <italic> AtNRP1 </italic> gene (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8c </xref> ). These results indicate that conserved regulatory mechanisms account for the BiP modulation of <span class="trait">drought</span> tolerance and NRP&#x02013;mediated cell death signaling in different plant species. <fig id="Fig8"> <label> Fig. 8 </label> <caption> <p> <italic> <span class="gene">BiPD</span> </italic> &#x02013;overexpressing lines are more tolerant to water stress. <bold> a </bold> Transgenic lines under a water deficit regime. Arabidopsis plants, genotypes Col&#x02013;0, BiPDox line T10 and BiPDox line T23 were grown in soil and water stress was induced by withholding irrigation for 20&#x000a0;days. Photography was taken 10&#x000a0;days after withholding irrigation. <bold> b </bold> Relative water content of wild&#x02013;type and transgenic leaves exposed to the <span class="trait">drought</span> regime. <bold> c </bold> Expression of <italic> AtNRP1 </italic> and <italic> AtNRP2 </italic> during the water stress treatment. Total RNA was isolated from leaves of Col&#x02013;0 and BiPDox lines at the indicated time points during the water stress treatment. The transcript levels of <italic> AtNRP1 </italic> were quantified by qRT&#x02013;PCR. Gene expression was calculated using the 2 <sup> -&#x00394;&#x00394;Ct </sup> method and <italic> Actin </italic> as endogenous control. cDNAs were obtained from five biological replicates and validated individually. Error bars indicate the 95&#x000a0;% confidence interval based on a t-test ( <italic> p </italic> &#x02009;&#x0003c;&#x02009;0,05)
					</p>
				
					<p>&nbsp;&nbsp;
						A functional conservation between <italic> AtNRP1 </italic> , <italic> AtNRP2 </italic> , <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> A </italic> and <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> B </italic> orthologs was further demonstrated in complementation assays of the hypersensitive phenotype to osmotic stress displayed by the atnrp1 null mutants. Under osmotic conditions, a further enhancement of root growth inhibition is clearly observed in the <italic> atnrp1 </italic> line (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a </xref> ). Stable expression of <italic> AtNRP1 </italic> , <italic> AtNRP2 </italic> , <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> A </italic> or <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> B </italic> in the atnrp1 null mutant rescued the wild&#x02013;type phenotype and the root growth of transgenic lines under PEG treatment was compared to the untransformed wild&#x02013;type line. These results indicated that NRPs may be involved in root growth under osmotic stress. Consistent with this hypothesis, overexpression of <italic> BiP </italic> , a negative regulator of <italic> AtNRP1 </italic> expression, recapitulates the enhanced phenotype of PEG&#x02013;mediated root growth inhibition of atnrp1 null alleles (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5b </xref> ). Likewise, <italic> BiP </italic> overexpression in soybean also promotes root inhibition under osmotic stress and <span class="trait">drought</span> and represses <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> A </italic> and <italic> <span class="gene">GmNRP</span> </italic> &#x02013; <italic> B </italic> expression [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. The <span class="gene">NRP</span> involvement in root growth is not likely associated with a cell death function but rather it may be linked to the pro&#x02013;survival activity of UPR. Two lines of evidence argue favorably for this hypothesis. First, both <span class="gene">GmNARP&#x02013;A</span> and <span class="gene">GmNRP&#x02013;B</span> promoters harbor putative unfolded protein response elements (UPRE2AT on <span class="gene">GmNRP&#x02013;A</span> promoter and UPRE1AT on <span class="gene">GmNRP&#x02013;B</span> promoter; Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ), and the AtNRP1 promoter contains two putative UPREs (UPRE1A and UPRE2A; Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ), which have the potential to be activated by UPR branches under ER stress ( <ext-link ext-link-type="uri" xlink:href="http://plantpan2.itps.ncku.edu.tw/"> http://plantpan2.itps.ncku.edu.tw </ext-link> ) [ <xref ref-type="bibr" rid="CR37"> 37 </xref> ]. Second, loss of <italic> IRE1a </italic> and <italic> IRE1b </italic> function causes root inhibition indicating the relevance of an activated UPR in root development [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Under stress conditions, root growth requires a functional <span class="gene">bZIP60</span> in addition of IRE1a/IRE1b, implicating the IRE1a/IRE1b&#x02013;bZIP60 module of UPR as a critical player not only in stress tolerance but also in root growth. However, it remains to be determined whether induction and function of NRPs under osmotic stress are linked to the IRE1a/IRE1b&#x02013;bZIP60 arm of the UPR.
					</p>
				
					<p>&nbsp;&nbsp;
						For transient expression in protoplasts and <italic> N. benthamiana </italic> leaves and for Arabidopsis transformation <italic> AtNRP1 </italic> , <italic> AtNRP2 </italic> , <italic> ANAC036 </italic> , <italic> AtBiP1 </italic> , <italic> AtBiP2 </italic> , <italic> gVPE </italic> and <italic> soyBiPD </italic> were amplified from Arabidopsis and soybean cDNA using specific primers (AtNRP1 Fwd and AtNRP1 Rvs; AtNRP2 Fwd and AtNRP2 Rvs; ANAC036 Fwd and ANAC036 Rvs, BiP1 Fwd and BiP1 Rvs, <span class="gene">BiPD</span> Fwd and <span class="gene">BiPD</span> Rvs, VPE&#x02013;gFwd and VPE&#x02013;g Rvs) (Additional file <xref rid="MOESM10" ref-type="media"> 10 </xref> ) and inserted by recombination into the entry vectors pDONR201 and pDONR221 (Invitrogen) to <span class="trait">yield</span> pDONR201Ns&#x02013;AtNRP1, pDONR201Ns&#x02013;AtNRP2, pDONR201&#x02013;ANAC036, pDONR221BiPD, pDONR201BiP1, pDONR201BiP2 and pDONR201VPE&#x02013;g. These genes were then transferred from the entry vectors to different expression vector (pK7WG2, pEarleygate100 and pEarleygate 104) by recombination using the enzyme LR clonase (Invitrogen) to generate the clones described here: p35S: AtNRP1, p35S: AtNRP2, p35S: ANAC036, p35S: BiPD, p35S: BiP1, p35S: BiP2, p35S: VPE&#x02013;g. For the transient expression in the protoplasts and tobacco, AtNRP1, AtNRP2, ANAC036, BiPD, BiP1 and BiP2 were fused to YFP (pYFP&#x02013;AtNRP1,pYFP&#x02013;AtNRP2, pYFP&#x02013;ANAC036, pYFP&#x02013;BiPD, pYFP&#x02013;BiP1 and pYFP&#x02013;BiP2) or to GFP (gVPE&#x02013;GFP) and placed under the control of the 35S promoter in the respective binary vector for plant transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						Arabidopsis BiP1 and BiP2 modulate the NRP&#x02013;mediated cell death signaling and cause tolerance to <span class="trait">drought.</span> a Enhanced accumulation of Arabidopsis BiP1 or BiP2 transcripts in transgenic lines. Total RNA was isolated from wild&#x02013;type leaves, <italic> BiP1 </italic> &#x02013;overexpressing (BiP1ox) and <italic> BiP2 </italic> &#x02013;overexpressing (BiP2ox) transgenic leaves and <span class="gene">BIP</span> transcript levels were determined by quantitative RT&#x02013;PCR. The relative expression was quantified using the 2 <sup> &#x02013;&#x00394;&#x00394;Ct </sup> method and <italic> UBQ5 </italic> as an endogenous control. The values are relative to the control treatment (wild&#x02013;type), and error bars indicate the 95&#x000a0;% confidence interval based on a t&#x02013;test ( <italic> p </italic> &#x02009;&#x0003c;&#x02009;0,05, <italic> n </italic> &#x02009;=&#x02009;3). b AtBiP1&#x02013;GFP&#x02013;HDEL and AtBiP2&#x02013;GFP&#x02013;HDEL accumulation in Arabidopsis transgenic lines. Total protein from Col&#x02013;0 and transgenic lines were separated in SDS&#x02013;PAGE and immunoblotted with an anti&#x02013;soyBiPD serum, which recognizes both the endogenous BiP1 and BiP2 (lower bands) and the fusion proteins (upper bands). In the lower blot, the recombinant proteins were probed with an anti&#x02013;GFP serum to recognize specifically BiP&#x02013;GFP fusion proteins. c The BiP&#x02013;GFL&#x02013;HDEL fusion protein localized in the microsomal fraction in transgenic lines. Equal amounts of whole cell protein extract (TF), cytosolic fraction (CF) and microsomal fraction (MF) from seedlings of Col&#x02013;0 (upper blot) were separated by SDS&#x02013;PAGE, immunoblotted with an anti&#x02013;BiP serum and reprobed with an anti&#x02013;UGPase, as a cytosolic marker. Asterisks indicate anti&#x02013;UGPase cross&#x02013;reacting proteins. In the lower blot, the BiP&#x02013;GFP&#x02013;HDEL fusion was probed with an anti&#x02013;GFP serum. d and e Arabidodpis <italic> BiP </italic> overexpression attenuates the tunicamycin and PEG induction of DCD/NRP&#x02013;mediated cell death signaling genes. Total RNA was isolated from 15&#x000a0;days&#x02013;old Arabidopsis plants treated with PEG (10&#x000a0;%&#x000a0;w/v) and Tunicamycin (2,5&#x000a0;&#x003bc;g/mL) for 2&#x000a0;h, 6&#x000a0;h, 12&#x000a0;h and 24&#x000a0;h. H <sub> 2 </sub> O was used as control for PEG and DMSO for Tunicamycin. The transcript levels of selected genes were quantified by qRT&#x02013;PCR. Gene expression was calculated using the 2 <sup> -&#x00394;&#x00394;Ct </sup> method and <italic> UBQ5 </italic> as endogenous control. cDNAs were obtained from four biological replicates and validated individually. The error bars indicate the 95&#x000a0;% confidence interval based on a t&#x02013;test ( <italic> p </italic> &#x02009;&#x0003c;&#x02009;0,05, <italic> n </italic> &#x02009;=&#x02009;4). (TIF 969&#x000a0;kb)
					</p>
				
					<p>&nbsp;&nbsp;
						PABR carried out the molecular genetics and functional studies, participated in the <italic> in silico </italic> analysis and co&#x02013;wrote the manuscript. JCFS and MDB carried out the in silico analysis. PPJF, EGDS, LFC, IHTO participated in the molecular genetics and functional studies. <span class="gene">PAC</span> and HHC obtained transgenic lines, monitored transgene expression and <span class="trait">drought</span> tolerance. JRLSR participated in the design of the study. EPBF conceived the study, participated in its design and coordination and co&#x02013;wrote the manuscript. All authors read and approved the final manuscript.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Genes Encoding Methionine-Rich Proteins in <italic> Arabidopsis </italic> and Soybean Suggesting Their Roles in the Adaptation of Plants to Abiotic Stress</h2>
			<h3>PMID:27635394</h3>
				

				
					<h3>PMCID:5007304</h3>
					<h4>期刊:International Journal of Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chu </surname>  <given-names> Ha Duc </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 19 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Oxidation and reduction of methionine (Met) play important roles in scavenging reactive oxygen species (ROS) and signaling in living organisms. To understand the impacts of Met oxidation and reduction in plants during stress, we surveyed the genomes of Arabidopsis and soybean (Glycine max L.) for genes encoding Met-rich proteins (MRPs). We found 121 and 213 genes encoding MRPs in Arabidopsis and soybean, respectively. Gene annotation indicated that those with known function are involved in vital cellular processes such as transcriptional control, calcium signaling, protein modification, and metal transport. Next, we analyzed the transcript levels of MRP-coding genes under normal and stress conditions. We found that 57 AtMRPs were responsive either to <span class="trait">drought</span> or to high salinity stress in Arabidopsis; 35 <span class="gene">GmMRPs</span> were responsive to <span class="trait">drought</span> in the leaf of late vegetative or early reproductive stages of soybean. Among the MRP genes with a known function, the majority of the abiotic stress-responsive genes are involved in transcription control and calcium signaling. Finally, Arabidopsis plant which overexpressed an MRP-coding gene, whose transcripts were downregulated by abiotic stress, was more sensitive to paraquat than the control. Taken together, our report indicates that MRPs participate in various vital processes of plants under normal and stress conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To obtain transcription levels, microarray data from previous studies were mined, including datasets for <italic> Arabidopsis </italic> under <span class="trait">drought</span> and salinity stress [ <xref rid="B22" ref-type="bibr"> 34 </xref> , <xref rid="B23" ref-type="bibr"> 35 </xref> ] and soybean under experimental <span class="trait">drought</span> conditions [ <xref rid="B16" ref-type="bibr"> 36 </xref> ]. In that study, for <span class="trait">drought</span> treatment of <italic> Arabidopsis </italic> , 2-week-old plants were transferred to soil and allowed to grow for one more week; the 3-week-old plants were then withheld from watering for 10 days. After the tenth day, rosette leaves were collected from both well-watered and drought-stressed plants, frozen in liquid nitrogen, and stored at &#x02212;80&#x000b0;C until RNA extraction. For high salinity treatment, 10-day-old plants grown on GM media were transferred onto 0.5x <span class="gene">MS</span> plates without sucrose, containing either 0&#x02009;mM (untreated) or 200&#x02009;mM NaCl and maintained for a period of 24&#x02009;h. Samples were collected in three biological replicates, frozen in liquid nitrogen, and stored at &#x02212;80&#x000b0;C until used for RNA extraction. The <span class="trait">drought</span> treatment of soybean plants and data acquisition was described previously [ <xref rid="B16" ref-type="bibr"> 36 </xref> ]. Data analyses were performed with functions integrated in <span class="gene">MS</span> EXCEL.
					</p>
				
					<p>&nbsp;&nbsp;
						To identify stress-responsive genes among MRP-coding genes, we performed data mining with the microarray experiments conducted earlier. In these experiments, <span class="trait">drought</span> treatments were carried out by withholding water. Leaves of V6 (vegetative) and R2 (reproductive) stages were collected and microarray analyses performed as reported [ <xref rid="B16" ref-type="bibr"> 36 </xref> ]. Transcript levels of all 213 MRP-coding genes were obtained, of which 11 were repressed and 12 genes were induced under <span class="trait">drought</span> in V6 <italic> trifolia </italic> . In reproductive leaves (R2 <italic> trifolia </italic> ), <span class="trait">drought</span> treatment induced 24 genes, whereas only 6 genes were repressed (Table S2). A total of 13 MRP-coding genes were found to be transcriptionally responsive to <span class="trait">drought</span> in both vegetative and reproductive-stage leaves ( <xref ref-type="table" rid="tab2"> Table 2 </xref> ). The gene <span class="gene">Glyma04g37040,</span> which encodes a calmodulin-binding protein CML38, was the most induced gene by <span class="trait">drought</span> and its transcript levels were induced 40- and 15-fold in R2 and V6 <italic> trifolia </italic> , respectively. The most repressed gene by <span class="trait">drought</span> was <span class="gene">Glyma02g10620,</span> encoding a 98-residue protein of unknown function whose transcript levels were repressed 44- and 4-fold in V6 and R2 <italic> trifolia </italic> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						In <italic> Arabidopsis </italic> , 7 AtMRPs were upregulated and 3 other AtMRPs were downregulated under both <span class="trait">drought</span> and high salinity. In soybean <span class="trait">drought,</span> 8 and 5 <span class="gene">GmMRPs</span> were up- or downregulated, respectively, in both V6 and R2 <italic> trifolia </italic> ( <xref ref-type="fig" rid="fig2"> Figure 2(a) </xref> ). Among 6 <italic> AtMRPs </italic> encoding calcium signaling proteins, three genes were transcriptionally responsive to either <span class="trait">drought</span> or salt (Table S1). At the same time, 14 <italic> <span class="gene">GmMRPs</span> </italic> encoding calmodulin-like proteins were transcriptionally responsive to <span class="trait">drought</span> in either V6 or R2 <italic> trifolia </italic> or both. This data suggested that calcium signaling plays an important role in the plant's signaling during abiotic stress exposure. In light of previous studies, it is very likely that Met oxidation and reduction of calmodulin may also contribute significantly to the plant's signaling response to abiotic stresses. A number of <italic> AtMRPs </italic> and <italic> <span class="gene">GmMRPs</span> </italic> encoding transcription factors were also responsive to abiotic stresses in <italic> Arabidopsis </italic> and soybean (Tables <xref ref-type="table" rid="tab1"> 1 </xref> and <xref ref-type="table" rid="tab2"> 2 </xref> ), indicating the involvement of the MRPs in the important cellular activities.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes encoding <span class="gene">GmMRPs</span> whose expression is responsive to <span class="trait">drought</span> stress in V6 and R2 leaves.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Dual functioning of plant arginases provides a third route for putrescine synthesis.</h2>
			<h3>PMID:28716421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Patel J(1), Ariyaratne M(1), Ahmed S(1), Ge L(1), Phuntumart V(1), Kalinoski A(2), Morris PF(3).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two biosynthetic routes are known for putrescine, an essential plant metabolite. Ornithine decarboxylase (ODC) converts ornithine directly to putrescine, while a second route for putrescine biosynthesis utilizes arginine decarboxylase (ADC) to convert arginine to agmatine, and two additional enzymes, agmatine iminohydrolase (AIH) and N-carbamoyl putrescine aminohydrolase (NLP1) to complete this pathway. Here we show that plants can use ADC and arginase/agmatinase (ARGAH) as a third route for putrescine synthesis. Transformation of Arabidopsis thaliana ADC2, and any of the arginases from A. thaliana (ARGAH1, or ARGHA2) or the soybean gene <span class="gene">Glyma.03g028000</span> <span class="gene">(GmARGAH)</span> into a yeast strain deficient in ODC, fully complemented the mutant phenotype. In vitro assays using purified recombinant enzymes of AtADC1 and AtARGAH2 were used to show that these enzymes can function in concert to convert arginine to agmatine and putrescine. Transient expression analysis of the soybean genes <span class="gene">(Glyma.06g007500,</span> ADC; <span class="gene">Glyma.03g028000</span> <span class="gene">GmARGAH)</span> and the A. thaliana ADC2 and ARGAH genes in leaves of Nicotiana benthamiana, showed that these proteins are localized to the chloroplast. Experimental support for this pathway also comes from the fact that expression of AtARGAH, but not AtAIH or AtNLP1, is co-regulated with AtADC2 in response to <span class="trait">drought,</span> oxidative stress, wounding, and methyl jasmonate treatments. Based on the high affinity of ARGAH2 for agmatine, its co-localization with ADC2, and typically low arginine levels in many plant tissues, we propose that these two enzymes can be major contributors to putrescine synthesis in many A. thaliana stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						Published by Elsevier B.V.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptomic changes due to water deficit define a general soybean response and accession-specific pathways for <span class="trait">drought</span> avoidance</h2>
			<h3>PMID:25644024</h3>
				

				
					<h3>PMCID:4322458</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Shin </surname>  <given-names> Jin Hee </given-names>  </name>  <address>  <email> kaloseidos@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Vaughn </surname>  <given-names> Justin N </given-names>  </name>  <address>  <email> jnvaughn@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdel-Haleem </surname>  <given-names> Hussein </given-names>  </name>  <address>  <email> hussein@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chavarro </surname>  <given-names> Carolina </given-names>  </name>  <address>  <email> mcch@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abernathy </surname>  <given-names> Brian </given-names>  </name>  <address>  <email> bla@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Kyung Do </given-names>  </name>  <address>  <email> kkim@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jackson </surname>  <given-names> Scott A </given-names>  </name>  <address>  <email> sjackson@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <address>  <email> zli@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Center for Applied Genetic Technologies &#x00026; Department of Crop and Soil Science, University of Georgia, Athens, GA 30602 USA  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chromatin remodeling genes have an unusual Trough expression pattern in both genotypes (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> and Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). Chromatin regulation responses to <span class="trait">drought,</span> cold, and salinity stress have been described in Arabidopsis [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. It was reported that the histone <span class="gene">H3</span> modification correlates with gene activation of the <span class="trait">drought</span> stress-inducible genes, such as responsive to dehydration (RD) 29A, RD29B, and related to AP2.4 (RAP2.4) [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. Moreover some chromatin remodeling and modifying enzymes such as histone modification enzymes, linker histone H1, and components of chromatin remodeling complex have been shown to function in plant abiotic stress responses [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. The initial down-regulation of these genes may reflect the expansion of euchromatin associated with the major transcriptional reprogramming that is occurring even at early stages of water-deficit, while the late up-regulation counters this trend, returning much of the genome to heterochromatin, under extreme physiological stress [ <xref ref-type="bibr" rid="CR29"> 29 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The large majority of <italic> GxE </italic> genes exhibited quantitative differences in expression levels at particular points rather than qualitatively different profiles (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> A). The exception to this trend was a small group of regulatory proteins that peaked in PI 416937 and remained relatively low and constant in Benning until 24&#x000a0;hr (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> B). Though none of these genes fell directly within the range specified by the QTL mapping discussed above, chromosomes 5 and 17 contain QTLs nearby two of the most striking <italic> GxE </italic> profiles, <span class="gene">Glyma02g45280</span> and <span class="gene">Glyma17g05520.</span> These genes are another set of promising leads in identifying solutions to problems posed by <span class="trait">drought.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The large majority of <italic> GxE </italic> genes exhibited quantitative differences in expression levels at particular points rather than qualitatively different profiles (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> A). The exception to this trend was a small group of regulatory proteins that peaked in PI 416937 and remained relatively low and constant in Benning until 24&#x000a0;hr (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> B). Though none of these genes fell directly within the range specified by the QTL mapping discussed above, chromosomes 5 and 17 contain QTLs nearby two of the most striking <italic> GxE </italic> profiles, <span class="gene">Glyma02g45280</span> and <span class="gene">Glyma17g05520.</span> These genes are another set of promising leads in identifying solutions to problems posed by <span class="trait">drought.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Expression analysis in response to <span class="trait">drought</span> stress in soybean: Shedding light on the regulation of metabolic pathway genes</h2>
			<h3>PMID:22802708</h3>
				
					<h3>PMCID:3392875</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guimar&#x000e3;es-Dias </surname>  <given-names> F&#x000e1;bia </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-222">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-222">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neves-Borges </surname>  <given-names> Anna Cristina </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-222">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-222">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Viana </surname>  <given-names> Antonio Americo Barbosa </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-222">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mesquita </surname>  <given-names> Rosilene Oliveira </given-names>  </name>  <xref ref-type="aff" rid="af5-gmb-35-1-sup-222">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Romano </surname>  <given-names> Eduardo </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-222">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de F&#x000e1;tima Grossi-de-S&#x000e1; </surname>  <given-names> Maria </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-222">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="af4-gmb-35-1-sup-222">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Loureiro </surname>  <given-names> Marcelo Ehlers </given-names>  </name>  <xref ref-type="aff" rid="af5-gmb-35-1-sup-222">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alves-Ferreira </surname>  <given-names> M&#x000e1;rcio </given-names>  </name>  <xref ref-type="corresp" rid="c1-gmb-35-1-sup-222"/>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-222">  <sup> * </sup>  </xref>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-222">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In order to identify the best candidates in the soybean genome for the AGMPs, we performed dendrogram analyses. These included the genes <italic> <span class="gene">GmaxLKR/SDH</span> </italic> -like1, <italic> <span class="gene">GmaxLKR/SDH</span> </italic> -like2 and <italic> <span class="gene">GmaxADC2</span> </italic> -like1 ( <xref ref-type="fig" rid="f3-gmb-35-1-sup-222"> Figure 3 </xref> ) and also <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1, <italic> <span class="gene">GmaxGOLS2</span> </italic> -like2, and <italic> <span class="gene">GmaxGOLS2</span> </italic> -like3 ( <xref ref-type="fig" rid="f4-gmb-35-1-sup-222"> Figure 4 </xref> ). These genes are part of the metabolic pathways of lysine degradation II, putrescine biosynthesis I and stachyose biosynthesis, respectively ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-222"> Figure 2 </xref> ). For those soybean genes where the neighbor-joining analysis was not able to determine the closest <italic> Arabidopsis </italic> ortholog, the selection of the soybean gene(s) for posterior analysis was based on their expression frequency in the <span class="trait">drought</span> induced subtractive library of the GENOSOJA LGE database ( <xref ref-type="supplementary-material" rid="SD2"> Table S2 </xref> ). The putative soybean orthologs of AGMPs were identified through Blastp searches in the soybean genome on the Phytozome website, followed by dendrogram analysis. For each AGMP, we identified a putative ortholog in the <italic> G. max </italic> and <italic> O. sativa </italic> genomes. The dendrogram analysis indicated that the <italic> Arabidopsis </italic> genes <italic> AtLKR/SDH </italic> (At4g33150) and <italic> AtGOLS2 </italic> <span class="gene">(At1g56600)</span> have two putative orthologs in the soybean genome. For the gene <italic> <span class="gene">GmaxLKR/SDH</span> </italic> the putative orthologs are <span class="gene">Glyma13g17580</span> and <span class="gene">Glyma17g0492</span> <italic> , </italic> while for the gene <italic> <span class="gene">GmaxGOLS2</span> </italic> the putative orthologs are <span class="gene">Glyma20g22700,</span> <span class="gene">Glyma03g38080</span> and <span class="gene">Glyma19g40680</span> ( <xref ref-type="fig" rid="f3-gmb-35-1-sup-222"> Figures 3A </xref> and <xref ref-type="fig" rid="f4-gmb-35-1-sup-222"> 4 </xref> ). The dendrogram analysis of <italic> ADC2 </italic> pointed to <span class="gene">Glyma04g00960</span> as being the closest gene to <italic> AGMP </italic> . However, <italic> <span class="gene">Glyma04g00960</span> </italic> was present only in a single subtractive library whereas <span class="gene">Glyma06g00990</span> was represented in four. Therefore, <italic> <span class="gene">Glyma06g00990</span> </italic> was also selected to be validated by qPCR ( <xref ref-type="fig" rid="f3-gmb-35-1-sup-222"> Figure 3B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The genes <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like1 and <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like2 showed higher expression in PSys compared to HSys ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5A, B </xref> ). The expression profile in the sensitive cultivars showed a gradual increase in all conditions tested. Interestingly, the expression of <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like1 and <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like2 in the tolerant cultivar was down-regulated in the PSys when exposed to <span class="trait">drought.</span> In the HSys condition, these genes showed a higher increase in expression at a later time (T100 min and T150 min) in both cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 gene presented a quite different expression profile during <span class="trait">drought</span> stress in the two tested systems when compared with the other two <italic> <span class="gene">GmaxGOLS2</span> </italic> soybean orthologs, <italic> <span class="gene">GmaxGOLS2</span> </italic> -like2 and <italic> <span class="gene">GmaxGOLS2</span> </italic> -like3. It is worthy of note that the level of expression of <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 is eight times higher in the tolerant cultivar at an early time point (T50 min) in HSys compared to the non-stress sample, while the sensitive cultivar showed a level of expression four times higher for the same time point (T50 min) compared to the control sample. In PSys, the tolerant cultivar showed a subtle increase in the <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 expression level under moderate stress (&#x02212;1.5 MPa) compared to the control, while the sensitive cultivar exhibited mild repression under the same stress level ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5D </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Arginine decarboxylase (ADC) is a key plant enzyme that converts arginine into putrescine, an important mediator of abiotic stress tolerance ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-222"> Figure 2B </xref> ) ( <xref ref-type="bibr" rid="b39-gmb-35-1-sup-222"> Peremarti <italic> et al. </italic> , 2010 </xref> ). The over-expression of <italic> ADC2 </italic> in transgenic <italic> Arabidopsis </italic> showed that higher levels of putrescine increased <span class="trait">drought</span> tolerance ( <xref ref-type="bibr" rid="b1-gmb-35-1-sup-222"> Alcazar <italic> et al. </italic> , 2006 </xref> , <xref ref-type="bibr" rid="b2-gmb-35-1-sup-222"> 2010 </xref> ). Dendrogram analysis allowed us to identify two paralogs, <italic> <span class="gene">GmaxADC2</span> </italic> -like1 <span class="gene">(Glyma06g00990)</span> and <italic> <span class="gene">GmaxADC2</span> </italic> -like2 <span class="gene">(Glyma04g0960)</span> ( <xref ref-type="fig" rid="f3-gmb-35-1-sup-222"> Figure 3B </xref> ). An analysis by qPCR was not done for <italic> <span class="gene">GmaxADC2</span> </italic> -like2 because previous information from subtractive library data did not indicate its expression during water deficit. The <italic> <span class="gene">GmaxADC2-</span> </italic> like1 reached peak expression at a water deficit of &#x02212;1.5 MPa in the PSys and at the T100 time point in the HSys condition in both cultivars. Interestingly, unlike the <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like1 and <italic> <span class="gene">GmaxLKR/SDH</span> </italic> -like2 genes, the expression levels of <italic> <span class="gene">GmaxADC2-</span> </italic> like1 were lower in the PSys when compared to the HSys condition ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5C </xref> ). This indicates that the regulation of <italic> <span class="gene">GmaxADC2-</span> </italic> like1 expression may be early and transient after the onset of a water deficit sensitivity.
					</p>
				
					<p>&nbsp;&nbsp;
						The conversion of myo-inositol to galactinol or to other raffinose series oligosaccharides ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-222"> Figure 2C </xref> ) under abiotic stress was studied in <italic> Arabidopsis </italic> ( <xref ref-type="bibr" rid="b44-gmb-35-1-sup-222"> Seki <italic> et al. </italic> , 2002 </xref> ; <xref ref-type="bibr" rid="b51-gmb-35-1-sup-222"> Taji <italic> et al. </italic> , 2002 </xref> ; <xref ref-type="bibr" rid="b47-gmb-35-1-sup-222"> Shinozaki and Yamaguchi-Shinozaki, 2007 </xref> ; <xref ref-type="bibr" rid="b58-gmb-35-1-sup-222"> Urano <italic> et al. </italic> , 2009 </xref> , <xref ref-type="bibr" rid="b57-gmb-35-1-sup-222"> 2010 </xref> ). Among the key genes of this pathway, <italic> AtGOLS1 </italic> and <italic> AtGOLS2 </italic> are the best studied. Their expression patterns are tightly regulated by <span class="trait">drought</span> stress and the over-expression of <italic> AtGOLS2 </italic> in <italic> Arabidopsis </italic> increases dehydration tolerance ( <xref ref-type="bibr" rid="b51-gmb-35-1-sup-222"> Taji <italic> et al. </italic> , 2002 </xref> ). The neighbor joining analysis suggests that there are six genes in the soybean genome related to <italic> AtGOLS1, AtGOLS2 and AtGOLS3: <span class="gene">GmGOLS</span> </italic> <span class="gene">(Glyma10g28610),</span> <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 <span class="gene">(Glyma20g22700)</span> <italic> <span class="gene">GmaxGOLS2</span> </italic> -like2 <span class="gene">(Glyma19g40680),</span> <italic> <span class="gene">GmaxGOLS2-</span> </italic> like3, <italic> <span class="gene">GmaxGOLS2-</span> </italic> like4 <span class="gene">(Glyma03g33910)</span> and <italic> <span class="gene">GmaxGOLS2-</span> </italic> like5 <span class="gene">(Glyma19g41550)</span> ( <xref ref-type="fig" rid="f4-gmb-35-1-sup-222"> Figure 4 </xref> ). The genes <italic> <span class="gene">GmaxGOLS2-</span> </italic> like4 and <italic> <span class="gene">GmaxGOLS2-</span> </italic> like5 were not selected for validation by qPCR because they were absent in the subtractive libraries ( <xref ref-type="supplementary-material" rid="SD2"> Table S2 </xref> ). Our analysis in HSys revealed that <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 shows higher levels of gene expression at earlier stages (T50 min) in the tolerant cultivar (Embrapa 48), while the sensitive cultivar (BR16) shows a slower response to water deficit ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5D </xref> ). A similar expression profile was also observed in PSys, but expression levels were significantly lower when compared with HSys ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5D </xref> ). In contrast, <italic> <span class="gene">GmaxGOLS2</span> </italic> -like2 and <italic> <span class="gene">GmaxGOLS2</span> </italic> -like3 were induced exclusively in the PSys condition ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5E, F </xref> ). Moreover, the expression levels in the tolerant cultivars were dramatically higher under severe stress ( <xref ref-type="fig" rid="f5-gmb-35-1-sup-222"> Figure 5E, F </xref> ). This result indicates that the expression of <italic> <span class="gene">GmaxGOLS2-</span> </italic> like2 and <italic> <span class="gene">GmaxGOLS2-</span> </italic> like3 is not regulated during the sudden water deficit promoted by the HSys treatment, but may be fundamental during the slow adaptation to <span class="trait">drought</span> in a PSys condition. The disparity observed in the regulation of gene expression between <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1 and the two paralogs <italic> <span class="gene">GmaxGOLS2-</span> </italic> like2 and <italic> <span class="gene">GmaxGOLS2-</span> </italic> like3 fits with the well-accepted model according to which changes in the transcriptional regulation of duplicated genes play an important role for their fixation in the genome ( <xref ref-type="bibr" rid="b12-gmb-35-1-sup-222"> Carroll, 2000 </xref> ). The distinct regulation of expression of the <italic> <span class="gene">GmaxGOLS2</span> </italic> genes may be important to soybean plants to promote tight control of GOLS2 expression under a multitude of environmental conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Dendrogram of the <italic> <span class="gene">GOLS</span> </italic> gene using a gene model based on amino acid sequences for <span class="trait">drought</span> of responsive genes in <italic> Arabidopsis thaliana </italic> , <italic> Oryza sativa </italic> and <italic> Glycine max, </italic> . The green solid arrow indicates soybean candidates, while the blue dotted arrow point to <italic> Arabidopsis </italic> reference gene, and the black dot indicates another <italic> <span class="gene">GmGOLS</span> </italic> candidate gene. Bootstrap values (1,000 replications) are indicated at the base of each branch.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression profile analyses of <span class="trait">drought</span> stress-related genes in pot-based (PSys) and hydroponic (HSys) cultivation conditions. A) <italic> <span class="gene">GmaxLKR/SDH</span> </italic> -like1, B) <italic> <span class="gene">GmaxLKR/SDH-</span> </italic> like2, C) <italic> <span class="gene">GmaxADC2-</span> </italic> like1, D) <italic> <span class="gene">GmaxGOLS2</span> </italic> -like1, E) <italic> <span class="gene">GmaxGOLS2</span> </italic> -like2 and F) <italic> <span class="gene">GmaxGOLS2</span> </italic> -like3. The PSys condition is represented by green solid bars and HSys by blue solid gray bars. The sensitive (BR16) and tolerant (Embrapa 48) cultivars are indicated at the bottom of the bars. Relative expression levels of these genes are represented on the Y-axis, relation to the reference genes <span class="gene">ACT</span> and FBOX in both cultivars and systems. The C, &#x02212;1.5 and &#x02212;3.0 represent control and the water potentials of soybean plants (&#x003a8;w) measured after the second day of the interruption of watering. Seven days after the interruption of watering the &#x003a8;w was &#x02212;1.5 &#x000b1; 0.2 MPa (moderate stress level) and after ten days &#x02212;3.0 &#x000b1; 0.2 MPa (severe stress level). The T0 (control), T50 (50 min), T100 (100 min) and T150 (150 min) indicate the different times under <span class="trait">drought</span> stress which the soybean seedlings were submitted after removed of the hydroponic solution.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and <italic> in silico </italic> characterization of soybean trihelix-GT and bHLH transcription factors involved in stress responses</h2>
			<h3>PMID:22802709</h3>
				
					<h3>PMCID:3392876</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Osorio </surname>  <given-names> Marina Borges </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-233">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000fc;cker-Neto </surname>  <given-names> Lauro </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-gmb-35-1-sup-233">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Castilhos </surname>  <given-names> Graciela </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Turchetto-Zolet </surname>  <given-names> Andreia Carina </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wiebke-Strohm </surname>  <given-names> Beatriz </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bodanese-Zanettini </surname>  <given-names> Maria Helena </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Margis-Pinheiro </surname>  <given-names> M&#x000e1;rcia </given-names>  </name>  <xref ref-type="corresp" rid="c1-gmb-35-1-sup-233"/>  </contrib>  <aff id="af1-gmb-35-1-sup-233"> Programa de P&#x000f3;s-Gradua&#x000e7;&#x000e3;o em Gen&#x000e9;tica e Biologia Molecular, Departamento de Gen&#x000e9;tica, Universidade Federal do Rio Grande do Sul, Porto Alegre, RS, Brazil </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Basic helix-loop-helix (bHLH) proteins constitute one of the largest families of transcription factors. They are found in all three eukaryotic kingdoms and are involved in a myriad of regulatory processes. Members of this family share the bHLH signature domain, which consists of &#x0223c;60 amino acids comprising two distinct regions, a basic stretch at the N-terminus consisting of &#x0223c;15 amino-acids involved in DNA binding, and a C-terminal region of &#x0223c;40 amino-acids composed of two amphipathic &#x003b1;-helices, mainly consisting of hydrophobic residues linked by a variable loop (the &#x0201c;helix-loop-helix&#x0201d; region). This region is responsible for promoting protein-protein interactions through the formation of homo- and hetero-dimeric complexes ( <xref ref-type="bibr" rid="b60-gmb-35-1-sup-233"> Toledo-Ortiz <italic> et al. </italic> , 2003 </xref> ; <xref ref-type="bibr" rid="b6-gmb-35-1-sup-233"> Carretero-Paulet <italic> et al. </italic> , 2010 </xref> ; <xref ref-type="bibr" rid="b46-gmb-35-1-sup-233"> Pires and Dolan, 2010 </xref> ). The Lc protein from <italic> Zea mays, </italic> reported as a transcriptional activator in the anthocyanin biosynthetic pathway ( <xref ref-type="bibr" rid="b33-gmb-35-1-sup-233"> Ludwig <italic> et al. </italic> , 1989 </xref> ), was the first plant bHLH member identified. The involvement of bHLH members in plant developmental processes ( <xref ref-type="bibr" rid="b55-gmb-35-1-sup-233"> Szecsi <italic> et al. </italic> , 2006 </xref> ; <xref ref-type="bibr" rid="b36-gmb-35-1-sup-233"> Menand <italic> et al. </italic> , 2007 </xref> ), light perception ( <xref ref-type="bibr" rid="b29-gmb-35-1-sup-233"> Liu <italic> et al. </italic> , 2008 </xref> ), iron and phosphate homeostasis ( <xref ref-type="bibr" rid="b63-gmb-35-1-sup-233"> Yi <italic> et al. </italic> , 2005 </xref> ; <xref ref-type="bibr" rid="b31-gmb-35-1-sup-233"> Long <italic> et al. </italic> , 2010 </xref> ; <xref ref-type="bibr" rid="b66-gmb-35-1-sup-233"> Zheng <italic> et al. </italic> , 2010 </xref> ), and phytohormone signalling pathways ( <xref ref-type="bibr" rid="b1-gmb-35-1-sup-233"> Abe <italic> et al. </italic> , 1997 </xref> ; <xref ref-type="bibr" rid="b16-gmb-35-1-sup-233"> Friedrichsen <italic> et al. </italic> , 2002 </xref> ; <xref ref-type="bibr" rid="b32-gmb-35-1-sup-233"> Lorenzo <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="b3-gmb-35-1-sup-233"> Anderson <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="b14-gmb-35-1-sup-233"> Fernandez-Calvo <italic> et al. </italic> , 2011 </xref> ; <xref ref-type="bibr" rid="b25-gmb-35-1-sup-233"> Hiruma <italic> et al. </italic> , 2011 </xref> ; <xref ref-type="bibr" rid="b53-gmb-35-1-sup-233"> Seo <italic> et al. </italic> , 2011 </xref> ) has also been reported. In fact, <italic> Arabidopsis </italic> MYC2 is to date the most extensively characterized plant bHLH transcription factor, and it seems to be a global regulator of hormone signalling. MYC2 has been described as an activator of ABA-mediated <span class="trait">drought</span> stress-response ( <xref ref-type="bibr" rid="b1-gmb-35-1-sup-233"> Abe <italic> et al. </italic> , 1997 </xref> , <xref ref-type="bibr" rid="b2-gmb-35-1-sup-233"> 2003 </xref> ). It also regulates JA/ET-induced genes, either as an activator in response to wounding, or as a suppressor in pathogen responses ( <xref ref-type="bibr" rid="b3-gmb-35-1-sup-233"> Anderson <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="b32-gmb-35-1-sup-233"> Lorenzo <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="b25-gmb-35-1-sup-233"> Hiruma <italic> et al. </italic> , 2011 </xref> ). In these cases, the activity of MYC2 is itself subject to regulation by JAZ proteins, in a SCF <sup> <span class="gene">COI1</span> </sup> proteosome degradation &#x02013; dependent pathway ( <xref ref-type="bibr" rid="b9-gmb-35-1-sup-233"> Chini <italic> et al. </italic> , 2007 </xref> ). Additionally, MYC2 seems to form homo- and heterodimers with two other closely-related bHLH proteins (MYC3 and MYC4), and their interaction is essential for full regulation of JA responses in <italic> Arabidopsis </italic> ( <xref ref-type="bibr" rid="b14-gmb-35-1-sup-233"> Fernandez-Calvo <italic> et al. </italic> , 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Lately, a growing number of studies accessing the functional role of specific plant bHLH transcription factors have been reported ( <xref ref-type="bibr" rid="b16-gmb-35-1-sup-233"> Friedrichsen <italic> et al. </italic> , 2002 </xref> ; <xref ref-type="bibr" rid="b55-gmb-35-1-sup-233"> Sz&#x000e9;csi <italic> et al. </italic> , 2006 </xref> ; <xref ref-type="bibr" rid="b29-gmb-35-1-sup-233"> Liu <italic> et al. </italic> , 2008 </xref> ; <xref ref-type="bibr" rid="b7-gmb-35-1-sup-233"> Chandler <italic> et al. </italic> , 2009 </xref> ; <xref ref-type="bibr" rid="b59-gmb-35-1-sup-233"> Todd <italic> et al. </italic> , 2010 </xref> ; <xref ref-type="bibr" rid="b66-gmb-35-1-sup-233"> Zheng <italic> et al. </italic> , 2010 </xref> ). Nevertheless, a deeper (and broader) functional characterization of this family, focusing on the connection of members/subgroups to the biological processes they control, remains to be done. A first step in this direction has been recently taken by <xref ref-type="bibr" rid="b6-gmb-35-1-sup-233"> Carretero-Paulet <italic> et al. </italic> (2010) </xref> and <xref ref-type="bibr" rid="b46-gmb-35-1-sup-233"> Pires and Dolan (2010) </xref> , where comprehensive information relating both classification and function of previously characterized plant bHLH transcription factors was assembled. More specifically, information regarding the function of subgroup 25 members is still scarce and concerns <italic> Arabidopsis </italic> members only. An alternative transcript of <span class="gene">At1g59640</span> ( <italic> ZCW </italic> 32/ <italic> BPE </italic> ) seems to be involved in the control of petal size, whereas its counterpart is expressed ubiquitously ( <xref ref-type="bibr" rid="b55-gmb-35-1-sup-233"> Sz&#x000e9;csi <italic> et al. </italic> , 2006 </xref> ). Furthermore, At4g34530 ( <italic> CIB </italic> 1) and <span class="gene">At1g26260</span> ( <italic> CIB </italic> 5) were shown to interact with blue-light receptor CRY2 and promote floral initiation ( <xref ref-type="bibr" rid="b29-gmb-35-1-sup-233"> Liu <italic> et al. </italic> , 2008 </xref> ). Of most interest for this study, is the redundant role of <span class="gene">At1g18400</span> ( <italic> BEE </italic> 1, <italic> Brassinosteroid Enhanced Expression </italic> 1), At4g36540 ( <italic> BEE </italic> 2) and <span class="gene">At1g73830</span> ( <italic> BEE </italic> 3) in brassinosteroids (BRs)/ABA antagonistic cross-talk during cell elongation ( <xref ref-type="bibr" rid="b16-gmb-35-1-sup-233"> Friedrichsen <italic> et al. </italic> , 2002 </xref> ). According to these authors, <italic> BEE </italic> 1, 2 and 3 are early-response genes induced by BRs through the BRI1 receptor complex, and their expression is repressed by ABA through a yet unknown ABA receptor. Whether this pathway is also related to the ABA-dependent stress-responsive network, still requires further study. Moreover, <xref ref-type="bibr" rid="b47-gmb-35-1-sup-233"> Poppenberger <italic> et al. </italic> (2011) </xref> have demonstrated that <span class="gene">At1g25330</span> ( <italic> CESTA </italic> ), a close homolog of <italic> BEE </italic> 1 and <italic> BEE </italic> 3 ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-233"> Figure 2 </xref> ), is also involved in BR signaling, possibly by heterodimerization with its closest homologs. Remarkably, it has also been shown that lack of CESTA activity results in the misregulation of genes that are not only BR-responsive but also stress-responsive, such as <italic> Arabidopsis ERD </italic> 5 ( <italic> Early Responsive to Dehydration </italic> 5), <italic> TTL4 </italic> ( <italic> Tetratricopetide-Repeat Thioredoxin-Like </italic> 4), <italic> <span class="gene">WRKY18</span> </italic> and a putative LRR-disease <span class="trait">resistance</span> protein ( <xref ref-type="bibr" rid="b47-gmb-35-1-sup-233"> Poppenberger <italic> et al. </italic> , 2011 </xref> ), further suggesting that these pathways might indeed share common features.
					</p>
				
					<p>&nbsp;&nbsp;
						With all that in mind, an integrated analysis of both the expression profile ( <xref ref-type="fig" rid="f1-gmb-35-1-sup-233"> Figure 1 </xref> ) and the phylogeny ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-233"> Figure 2 </xref> ) presented herein provides a hint at the roles of subgroup 25 bHLH soybean genes. By focusing on soybean-near homologs shown in the tree ( <xref ref-type="fig" rid="f2-gmb-35-1-sup-233"> Figure 2 </xref> on left) we could see that for most of the paralogs whose expression has been detected, a divergent profile seems to prevail. An exception would be the cases of <span class="gene">Glyma03g31510</span> and <span class="gene">Glyma19g34360,</span> which were both repressed during <span class="trait">drought</span> stress, with a broadly negative response in the latter, as its mRNA levels were down-regulated in both the susceptible and the tolerant cultivars analyzed. Moreover, the transcripts from <span class="gene">Glyma19g32570</span> were up-regulated during ASR infection in the resistant genotype, whereas its counterpart <span class="gene">Glyma03g29710</span> exhibited opposite differential expression. The near paralogs <span class="gene">Glyma05g01590</span> and <span class="gene">Glyma17g10290</span> also seem to be moving in different directions. Whereas the first seems to be up-regulated in response to fungal stress, the latter seems to be broadly down-regulated, in both susceptible and tolerant cultivars submitted to <span class="trait">drought,</span> as well as in <italic> P. pachyrhiz </italic> i&#x02019;s infection. Furthermore, while <span class="gene">Glyma15g33020</span> seems to be positively involved in soybean defense against ASR and during <span class="trait">drought</span> stress in tolerant Embrapa-48 cultivar, its nearest paralog <span class="gene">(Glyma09g14380)</span> was not differentially expressed in any of the situations assessed, and their near homolog <span class="gene">Glyma17g08300</span> seems to be negatively involved in <span class="trait">drought</span> stress responses, since it was down-regulated in the same cultivar. Whether the examples mentioned above reflect functional divergence or subfunctionalization among duplicate homologs still requires further analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Even though comparison of soybean genes with their orthologs in other species (such as <italic> Arabidopsis </italic> ) is a tentative approach, and as such needs to be performed carefully. In this context it would be interesting to address the function of <italic> BEE </italic> orthologs in soybean, so as to determine whether they are similar to their <italic> Arabidopsis </italic> counterparts, and whether they somehow connected to stress responses. In this respect, special attention should be given to <span class="gene">Glyma05g35060,</span> which clustered together with the <italic> Arabidopsis </italic> BR-responsive genes, and whose transcripts turned out to be up-regulated in Embrapa-48 tolerant cultivar in response to <span class="trait">drought.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Mining the LGE gene expression superSAGE experiments revealed that 11 soybean trihelix-GT genes were differentially expressed in the abiotic/biotic conditions tested ( <xref ref-type="fig" rid="f3-gmb-35-1-sup-233"> Figure 3 </xref> ). In accordance with our analyses, five trihelix-GT genes were up-regulated under <span class="trait">drought</span> in the tolerant cultivar (Embrapa-48), whereas only two genes were down-regulated in this genotype. In the susceptible cultivar (BR16), <span class="gene">Glyma10g34520</span> had its transcript levels increased in response to water deficit and the opposite situation occurred with <span class="gene">Glyma10g36950.</span> When plants were infected with <italic> P. pachyrhiz </italic> i, only two genes displayed up-regulation of mRNA levels in response to biotic stress whereas two others seemed to be down-regulated. Interestingly, none of the soybean trihelix-GT previously reported as responsive to stress conditions and particularly to abiotic stress <span class="gene">[GmGT-2A</span> <span class="gene">(Glyma04g39400)</span> and <span class="gene">GmGT-2B</span> <span class="gene">(Glyma10g30300)]</span> were detected in the superSAGE experiments herein assessed. Divergence in experimental parameters and genotypes used might explain this unexpected result.
					</p>
				
					<p>&nbsp;&nbsp;
						The superSAGE experiments suggested that, at least in some cases, the same gene has variable transcript levels in different cultivars and/or in response to different stresses or agents. For example, when water deficit was imposed on soybean plants, <span class="gene">Glyma10g36950</span> was down-regulated in the susceptible (BR16) and the tolerant (Embrapa-48) cultivars, whereas its transcript levels did not change in response to ASR. In another case, <span class="gene">Glyma09g38050</span> was up-regulated in response to <span class="trait">drought</span> stress in Embrapa-48, but no differences were detected in BR16. Furthermore, <span class="gene">Glyma13g26550</span> was down-regulated in response to <span class="trait">drought</span> stress in the tolerant cultivar, whereas its expression in cultivar BR16 did not exhibit any alterations. In these cases, in addition to differential gene regulation, there may be other factors contributing to distinct regulatory function, such as post-translational modifications or variation in dimerization partners ( <xref ref-type="bibr" rid="b67-gmb-35-1-sup-233"> Zhou, 1999 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Molecular Chaperone Binding Protein BiP Prevents Leaf Dehydration-Induced Cellular Homeostasis Disruption</h2>
			<h3>PMID:24489761</h3>
				
					<h3>PMCID:3906070</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Carvalho </surname>  <given-names> Humberto H. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brustolini </surname>  <given-names> Ot&#x000e1;vio J. B. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pimenta </surname>  <given-names> Maiana R. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mendes </surname>  <given-names> Giselle C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gouveia </surname>  <given-names> Bianca C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silva </surname>  <given-names> Priscila A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silva </surname>  <given-names> Jos&#x000e9; Cleydson F. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mota </surname>  <given-names> Clenilso S. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soares-Ramos </surname>  <given-names> Juliana R. L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fontes </surname>  <given-names> Elizabeth P. B. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Because it is the sole molecular chaperone involved in the activation of UPR, BiP expression changes could indicate alterations in the folding environment and ER processing capacity. Accordingly, BiP overexpression in mammals and plants attenuates ER stress and suppresses UPR activation <xref rid="pone.0086661-LeborgneCastel1" ref-type="bibr"> [29] </xref> , <xref rid="pone.0086661-Morris1" ref-type="bibr"> [38] </xref> . In addition to alleviating ER stress <xref rid="pone.0086661-LeborgneCastel1" ref-type="bibr"> [29] </xref> , BiP overexpression in plants has also been shown to increase water deficit tolerance <xref rid="pone.0086661-Valente1" ref-type="bibr"> [33] </xref> , <xref rid="pone.0086661-Alvim1" ref-type="bibr"> [39] </xref> , <xref rid="pone.0086661-Cascardo1" ref-type="bibr"> [40] </xref> . BiP-mediated increases in water stress tolerance have been associated with the capacity to delay drought-induced leaf senescence, which is mediated by NRP-mediated cell death signaling. In transgenic lines, BiP overexpression attenuates ER and osmotic stress-induced cell death phenotypes, such as foliar necrotic lesions and wilting, the percentage of dead cells, senescence-associated gene marker induction, DNA fragmentation, caspase activity, lipid peroxidation and the induction of cell death marker genes NRP-A, NRP-B and <span class="gene">GmNAC6</span> <xref rid="pone.0086661-Reis1" ref-type="bibr"> [31] </xref> . These results implicate BiP as a negative regulator of the stress-induced NRP-mediated cell death response. Nevertheless, whether BiP controls cellular homeostasis under <span class="trait">drought</span> conditions or simply delays dehydration-induced leaf senescence as the underlying mechanism for BiP-mediated increase in <span class="trait">drought</span> tolerance remains to be determined. To address this issue, we compared the drought-induced transcriptomes of wild type (WT) soybean leaves and BiP-overexpressing leaves at a similar leaf water potential (&#x003c8; <sub> w </sub> ), which was induced by a slow soil-drying regime. Our results revealed that a gradual reduction of the leaf &#x003c8; <sub> w </sub> to &#x02212;1.0 MPa imposed a sufficiently high level of dehydration stress to induce tremendous reprogramming of the gene expression in WT soybean leaves. In contrast, the BiP-overexpressing leaves maintained cellular homeostasis at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 MPa because they displayed only a slight change in the expression profile compared to the irrigated, well-watered leaves. However, as the dehydration stress increased to a &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.6 or &#x02212;1.7 MPa, the global gene expression variation in BiP-overexpressing lines and WT lines approached similar levels, although the expression of senescence-associated genes and UPR markers remained lower in BiP-overexpressing lines.
					</p>
				
					<p>&nbsp;&nbsp;
						The microarray and heatmap results for the differential expression of selected stress-responsive genes were confirmed by quantitative RT-PCR. For this assay, we also used a second independently transformed BiP-overexpressing line, called 35S::BiP-2, which has also been shown to be tolerant of water deficits <xref rid="pone.0086661-Valente1" ref-type="bibr"> [33] </xref> . A gene expression analysis by qRT-PCR first confirmed the high levels of <italic> soy </italic> <span class="gene">BiPD</span> transgene accumulation in transgenic lines 35S::BiP-4 and 35S::BiP-2 ( <xref ref-type="fig" rid="pone-0086661-g005"> Figure 5 </xref> , <xref ref-type="fig" rid="pone-0086661-g009"> Figure 9A </xref> ). Progressive <span class="trait">drought</span> induced the UPR marker genes, CNX <span class="gene">(Glyma04g38000</span> and <span class="gene">Glyma05g33330),</span> endogenous BiP and <span class="gene">bZIP60</span> ( <xref ref-type="fig" rid="pone-0086661-g009"> Figures 9B, 9C, 9D and 9E </xref> ). We also examined the UPR transducers <span class="gene">bZIP28</span> and <span class="gene">bZIP17</span> as they were induced by <span class="trait">drought</span> in the UPR heatmap ( <xref ref-type="fig" rid="pone-0086661-g005"> Figure 5 </xref> ). The quantititative RT-PCR results confirmed the <span class="trait">drought</span> induction of the UPR marker genes ( <xref ref-type="fig" rid="pone-0086661-g009"> Figures 9B&#x02013;9G </xref> ) Nevertheless, the ectopic expression of BiP inhibited the <span class="trait">drought</span> induction of UPR marker genes, because the UPR transcript level was lower in the transgenic lines than in the WT under the same leaf &#x003c8; <sub> w </sub> status ( <xref ref-type="fig" rid="pone-0086661-g005"> Figure 5 </xref> , <xref ref-type="fig" rid="pone-0086661-g009"> Figures 9B&#x02013;9G </xref> ),). These results confirmed that gradual dehydration stress induces UPR in the WT line, and BiP overexpression negatively impacts the drought-induced activation of UPR gene expression. The expression of representative drought-responsive gene samples ( <xref ref-type="fig" rid="pone-0086661-g007"> Figure 7 </xref> ; <xref ref-type="supplementary-material" rid="pone.0086661.s003"> Tables S2 </xref> and <xref ref-type="supplementary-material" rid="pone.0086661.s004"> S3 </xref> ), including the functional genes glutathionae-S-transferase (GST; <span class="gene">Glyma15g40190)</span> <xref rid="pone.0086661-Irsigler1" ref-type="bibr"> [34] </xref> , seed maturation protein (SMP; <span class="gene">Glyma13g44020)</span> <xref rid="pone.0086661-Irsigler1" ref-type="bibr"> [34] </xref> , and the regulatory genes NAC2, <span class="gene">(Glyma06g11970)</span> <xref rid="pone.0086661-Aguirrezabal1" ref-type="bibr"> [53] </xref> , also designated <span class="gene">GmNAC35,</span> and <span class="gene">NAC3</span> <span class="gene">(Glyma06g38410)</span> <xref rid="pone.0086661-Costa1" ref-type="bibr"> [28] </xref> , also designated <span class="gene">GmNAC043,</span> were also examined by qRT-PCR. Consistent with the microarray results ( <xref ref-type="supplementary-material" rid="pone.0086661.s003"> Tables S2 </xref> and <xref ref-type="supplementary-material" rid="pone.0086661.s004"> S3 </xref> , <xref ref-type="fig" rid="pone-0086661-g007"> Figure 7 </xref> ), progressive <span class="trait">drought</span> induced the expression of these drought-responsive genes ( <xref ref-type="fig" rid="pone-0086661-g009"> Figures 9H&#x02013;9K </xref> ). However, the induction of drought-responsive genes in 35S::BiP-4 and 35S::BiP-2 lines was lower than in WT leaves under a similar leaf water potential status. These results are congruent with our data on the global variation of gene expression and confirmed that BiP overexpression inhibits the induction of functional and regulatory drought-responsive genes, suggesting that BiP may alleviate endogenous dehydration stress.
					</p>
				
					<p>&nbsp;&nbsp;
						BiP overexpression has been demonstrated to confer <span class="trait">drought</span> tolerance in tobacco <xref rid="pone.0086661-Alvim1" ref-type="bibr"> [39] </xref> and, more recently, in soybeans <xref rid="pone.0086661-Valente1" ref-type="bibr"> [33] </xref> . In addition to a more efficient photosynthetic rate, the maintenance of leaf turgidity is prolonged in BiP-overexpressing plants when compared with WT lines under the same water deficit regime. To determine whether BiP-mediated protection against leaf dehydration is associated with a delay in the decline of leaf water potential under progressive <span class="trait">drought</span> or the amelioration of endogenous stress, the BiP-overexpressing and WT leaf transcriptomes induced by a slow soil drying regime were evaluated under similar levels of leaf &#x003c8; <sub> w </sub> (similar RWC). As previously observed, exposing soybeans to a gradual dehydration regime caused a slower decline of the leaf &#x003c8; <sub> w </sub> of the 35S::BiP-4 and 35S::BiP-2 lines than in WT leaves, which maintained the leaf turgor for a longer period ( <xref ref-type="fig" rid="pone-0086661-g001"> Figures 1A and 1B </xref> ). An analysis of global gene expression variations by gradual leaf dehydration clearly demonstrated that the leaf water status of &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 MPa is low enough to promote extensive gene expression reprogramming in soybean leaves. These results indicate that leaf &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 induces endogenous cellular stress that is efficiently sensed by leaf cells to activate typical responses to dehydration stress, such as the up-regulation of typical drought-responsive regulatory and functional genes ( <italic> DREB </italic> , <italic> NAC </italic> , <italic> AREB </italic> , <italic> LEA </italic> , <italic> <span class="gene">GST</span> </italic> , peroxidases, etc) as well as the down-regulation of photosynthesis-related genes. This scenario of high variations in the number of differentially expressed genes is consistent with the drought-induced transcriptome identified by Le et al. <xref rid="pone.0086661-Le1" ref-type="bibr"> [49] </xref> in soybean leaves exposed to <span class="trait">drought.</span> In contrast, the 35S::BiP-4 leaves at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 MPa displayed a considerably lower number of differentially expressed genes. These results indicate that BiP overexpression prevents variations in gene expression as induced by a leaf water status of &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 MPa, most likely because of its ability to maintain a degree of cellular homeostasis that ensures a lower level of cellular stress compared to WT leaves at the same leaf &#x003c8; <sub> w </sub> .
					</p>
				
					<p>&nbsp;&nbsp;
						The Venn diagram ( <xref ref-type="fig" rid="pone-0086661-g004"> Figure 4 </xref> ) revealed the remarkable predominance of differentially expressed genes in WT (&#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0) leaves in comparison to the barely detected gene variations in the 35S::BiP (&#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0) leaves. In the WT, <span class="trait">drought</span> conditions rapidly induced a variety of PCD-mediated genes, including caspases ( <italic> CASP-like proteins 3, 8 </italic> and <italic> 10 </italic> ), which have been characterized in heat-induced PCD events in tobacco cells <xref rid="pone.0086661-Xu1" ref-type="bibr"> [56] </xref> and <italic> cysteine proteinase </italic> , which is involved in plant PCDs <xref rid="pone.0086661-Xu2" ref-type="bibr"> [57] </xref> . These results are consistent with recent studies demonstrating that PEG-induced osmotic stress promotes PCD in soybean seedlings <xref rid="pone.0086661-Reis1" ref-type="bibr"> [31] </xref> . In contrast, BiP overexpression attenuated alterations in cell death-associated gene expression under progressive <span class="trait">drought</span> ( <xref ref-type="fig" rid="pone-0086661-g008"> Figures 8 </xref> and <xref ref-type="fig" rid="pone-0086661-g009"> 9 </xref> ), which is consistent with the BiP-mediated delay in drought-induced leaf senescence as previously reported <xref rid="pone.0086661-Valente1" ref-type="bibr"> [33] </xref> . A precedent in the literature has linked <span class="trait">drought</span> tolerance to the impairment of stressed leaves to undergo senescence <xref rid="pone.0086661-Rivero1" ref-type="bibr"> [58] </xref> . The underlying mechanism for the BiP-mediated increase in water stress tolerance may be associated at least in part with its capacity to attenuate stress-induced cell death events. Our global gene expression variation results revealed that multiple stress-induced NRP-mediated cell death signaling is also induced by <span class="trait">drought</span> and can be modulated by BiP overexpression. In addition to detecting a down-regulation in the cell death signaling components VPE (at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0, <xref ref-type="fig" rid="pone-0086661-g009"> Figure 9N </xref> and <xref ref-type="supplementary-material" rid="pone.0086661.s019"> Table S18 </xref> ) and <italic> Gm </italic> NAC81 (at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.6, <xref ref-type="fig" rid="pone-0086661-g009"> Figure 9L </xref> , <xref ref-type="supplementary-material" rid="pone.0086661.s023"> Table S22 </xref> ), NRPs were induced in the WT leaves at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0, but not in 35S::BiP-4 at &#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;1.0 ( <xref ref-type="fig" rid="pone-0086661-g009"> Figures 9J and 9K </xref> ). The BiP modulation of drought-induced expression for NRP-mediated cell death signaling components was further confirmed by qRT-PCR. VPE displays caspase 1-like activity and is an executioner of plant-specific PCD mediated by vacuole collapse <xref rid="pone.0086661-Hatsugai1" ref-type="bibr"> [59] </xref> , <xref rid="pone.0086661-Shimada1" ref-type="bibr"> [60] </xref> . Recently, VPE has been shown to be a target of the transcriptional factors <span class="gene">GmNAC30</span> and <span class="gene">GmNAC81,</span> which function in the osmotic and ER stress-induced NRP-mediated cell death response <xref rid="pone.0086661-Reis1" ref-type="bibr"> [31] </xref> , <xref rid="pone.0086661-Mendes1" ref-type="bibr"> [37] </xref> . <span class="gene">NRP</span> expression up-regulation mediated by the ER stress-, <span class="trait">drought-</span> and osmotic stress-induced <span class="gene">GmERD15</span> induces <span class="gene">GmNAC81</span> and <span class="gene">GmNAC30,</span> which in turn act in concert to activate the VPE promoter and expression, resulting in PCD activation. We propose that the BiP-mediated modulation of drought-induced NRP-mediated cell death signaling attenuates the dehydration-induced cell death and thereby promotes better adaptation of BiP-overexpressing lines to <span class="trait">drought.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The heatmap of UPR-specific genes revealed that progressive <span class="trait">drought</span> induces the UPR key transducer <span class="gene">bZIP28</span> and UPR marker genes, such as SP1, <span class="gene">bZIP60</span> and a subset of CNX and CRT (compare WTirr&#x000d7;WT&#x003c8; <sub> w </sub> &#x0200a;=&#x0200a;&#x02212;2.0; <xref ref-type="fig" rid="pone-0086661-g005"> Figure 5 </xref> ). The UPR induction caused by a gradual decline in the leaf &#x003c8; <sub> w </sub> was further confirmed by a qRT-PCR of selected UPR marker genes ( <xref ref-type="fig" rid="pone-0086661-g009"> Figure 9 </xref> ). The activation of bZIP28- and bZIP60-supported branches in the UPR up-regulates ERAD genes as part of the quality control of the organelle <xref rid="pone.0086661-Walter1" ref-type="bibr"> [16] </xref> , <xref rid="pone.0086661-Deng2" ref-type="bibr"> [61] </xref> , <xref rid="pone.0086661-Srivastava2" ref-type="bibr"> [62] </xref> . The heatmap of ERAD marker genes at different levels of leaf &#x003c8; <sub> w </sub> showed that these ERAD genes may be induced as a result of the decline of leaf &#x003c8; <sub> w </sub> , which supports the notion that UPR may be activated in soybean leaves in response to a gradual loss of water.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Metabolic and Transcriptional Reprogramming in Developing Soybean ( <italic> Glycine max </italic> ) Embryos</h2>
			<h3>PMID:24957996</h3>
				
					<h3>PMCID:3901275</h3>
					<h4>期刊:Metabolites</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Collakova </surname>  <given-names> Eva </given-names>  </name>  <xref ref-type="aff" rid="af1-metabolites-03-00347"> 1 </xref>  <xref rid="c1-metabolites-03-00347" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Aghamirzaie </surname>  <given-names> Delasa </given-names>  </name>  <xref ref-type="aff" rid="af2-metabolites-03-00347"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fang </surname>  <given-names> Yihui </given-names>  </name>  <xref ref-type="aff" rid="af1-metabolites-03-00347"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Klumas </surname>  <given-names> Curtis </given-names>  </name>  <xref ref-type="aff" rid="af2-metabolites-03-00347"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tabataba </surname>  <given-names> Farzaneh </given-names>  </name>  <xref ref-type="aff" rid="af3-metabolites-03-00347"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kakumanu </surname>  <given-names> Akshay </given-names>  </name>  <xref ref-type="aff" rid="af4-metabolites-03-00347"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Myers </surname>  <given-names> Elijah </given-names>  </name>  <xref ref-type="aff" rid="af2-metabolites-03-00347"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heath </surname>  <given-names> Lenwood S. </given-names>  </name>  <xref ref-type="aff" rid="af3-metabolites-03-00347"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grene </surname>  <given-names> Ruth </given-names>  </name>  <xref ref-type="aff" rid="af1-metabolites-03-00347"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 07 </day>  <month> 5 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The relevant responses underlying desiccation acquisition or <span class="trait">drought</span> tolerance were in the past regarded as being purely stress related. In fact, in the case of developing seeds, this genetically programmed developmental process proves to be more complex than a simple acquisition of desiccation tolerance. There is good evidence from previous high-throughput studies that desiccation acquisition and dormancy induction in orthodox seeds involve a specific and active transcriptional and metabolomic program [ <xref ref-type="bibr" rid="B20-metabolites-03-00347"> 20 </xref> , <xref ref-type="bibr" rid="B22-metabolites-03-00347"> 22 </xref> ], with 30% of genes expressed during Arabidopsis seed development being expressed during this late and final stage of seed development. That same pattern is reflected in the data reported here, where 22% of all genes expressed over the course of seed development were present in clusters 78&#x02013;105, which showed a common pattern of up regulation at the last time points ( <xref ref-type="supplementary-material" rid="metabolites-03-00347-s001"> Supplementary Figure 1, Table 2 </xref> ). Some of the genes that are part of this late phase of seed development are well known as components of stress adaptation in other plant tissues, such as the LEA and heat shock proteins [ <xref ref-type="bibr" rid="B53-metabolites-03-00347"> 53 </xref> , <xref ref-type="bibr" rid="B54-metabolites-03-00347"> 54 </xref> , <xref ref-type="bibr" rid="B55-metabolites-03-00347"> 55 </xref> ]. Representatives from these gene families (91&#x02013;94: <span class="gene">Glyma13g16510,</span> <span class="gene">Glyma01g39260,</span> <span class="gene">Glyma07g04860,</span> <span class="gene">Glyma16g01440,</span> <span class="gene">Glyma03g31380,</span> <span class="gene">Glyma08g26100,</span> <span class="gene">Glyma19g32920,</span> 100-104: <span class="gene">Glyma07g02480,</span> <span class="gene">Glyma08g23540,</span> <span class="gene">Glyma07g32070,</span> <span class="gene">Glyma11g37450,</span> and <span class="gene">Glyma16g23750)</span> are present in the late clusters of this data set.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Daytime soybean transcriptome fluctuations during water deficit stress</h2>
			<h3>PMID:26149272</h3>
				
					<h3>PMCID:4491896</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana Aparecida </given-names>  </name>  <address>  <email> deganuti@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fuganti-Pagliarini </surname>  <given-names> Renata </given-names>  </name>  <address>  <email> renatafuganti@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcolino-Gomes </surname>  <given-names> Juliana </given-names>  </name>  <address>  <email> marcolino.ju@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakayama </surname>  <given-names> Thiago Jonas </given-names>  </name>  <address>  <email> thiago84naka@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Molinari </surname>  <given-names> Hugo Bruno Correa </given-names>  </name>  <address>  <email> hugo.molinari@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lobo </surname>  <given-names> Francisco Pereira </given-names>  </name>  <address>  <email> francisco.lobo@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harmon </surname>  <given-names> Frank G </given-names>  </name>  <address>  <email> fharmon@berkeley.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <address>  <phone> +55 43 3371 6218 </phone>  <email> alexandre.nepomuceno@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <aff id="Aff1">  <label/> Brazilian Agricultural Research Corporation- Embrapa Soybean, Embrapa Soybean- Rod. Carlos Jo&#x000e3;o Strass, s/n, Londrina, 86001-970 PR Brazil  </aff>  <aff id="Aff2">  <label/> Department of Biology, State University of Londrina, Londrina, PR Brazil  </aff>  <aff id="Aff3">  <label/> Department of Crop Science, Federal University of Vi&#x000e7;osa, Vi&#x000e7;osa, MG Brazil  </aff>  <aff id="Aff4">  <label/> Genetics and Biotechnology Laboratory, Embrapa Agroenergy (CNPAE), Bras&#x000ed;lia, DF Brazil  </aff>  <aff id="Aff5">  <label/> Brazilian Agricultural Research Corporation-Embrapa Agricultural Informatics, Campinas, SP Brazil  </aff>  <aff id="Aff6">  <label/> Plant Gene Expression Center, USDA-ARS, Albany, CA USA  </aff>  <aff id="Aff7">  <label/> Department of Plant and Microbial Biology, UC Berkeley, Berkeley, CA USA  </aff>  <aff id="Aff8">  <label/> Embrapa LABEX US Plant Biotechnology at ARS/USDA Plant Gene Expression Center, Albany, CA USA  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Recently, our research team has demonstrated how <span class="trait">drought</span> impacts diurnal oscillation of both drought-responsive and circadian clock genes in soybean [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. <span class="trait">Drought</span> stress induced marked reduction in gene expression levels of several circadian clock-like components, such as <italic> <span class="gene">GmLCL1</span> </italic> -, <italic> <span class="gene">GmELF4</span> </italic> -, and <italic> <span class="gene">GmPRR</span> </italic> -like genes. The same conditions produced a phase advance of expression for the <italic> <span class="gene">GmTOC1-</span> </italic> , <italic> <span class="gene">GmLUX</span> </italic> - and <italic> <span class="gene">GmPRR7</span> </italic> -like genes. Similarly, the daily oscillation pattern of the soybean drought-responsive genes <italic> DREB- </italic> , <italic> bZIP- </italic> , <italic> GOLS- </italic> , <italic> RAB18- </italic> and <italic> Remorin- </italic> like changed significantly following plant exposure to water deficit.
					</p>
				
					<p>&nbsp;&nbsp;
						All DEGs that exhibited oscillating expression (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ) were mapped to the main pathways involved in plant response to stress (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Genes expressed in ZT0 were included in many pathways, including signaling and cell wall metabolism, that were repressed in response to water deficit during this time period. Conversely, some genes from the abiotic stress class were highly induced in a similar manner to hormone (ABA) signaling pathways and heat-shock proteins (HSPs) (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). The jasmonate pathway was also induced, as indicated by the expression of three genes related to lipoxygenase ( <italic> <span class="gene">Glyma07g00920</span> </italic> ), allene oxide synthase ( <italic> <span class="gene">Glyma07g21100</span> </italic> ), and 12-oxo-PDA-reductase ( <italic> <span class="gene">Glyma14g39790</span> </italic> ). The jasmonate pathway remained induced in ZT4 but was represented by two other genes encoding allene oxide synthase ( <italic> <span class="gene">Glyma11g13070</span> </italic> ) and 12-oxo-PDA-reductase ( <italic> <span class="gene">Glyma11g00980</span> </italic> ). <italic> <span class="gene">Glyma07g00920</span> </italic> , which was moderately expressed in the early morning (ZT0), also was highly up-regulated during pre-dawn (ZT20) (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). According to these results, some genes detected in a specific pathway were not expressed during the entire <italic> diel </italic> period, but genes encoding different proteins seem to be involved in the same stress-related pathways. The expression of genes related to PR-proteins is another example of how different genes are involved in maintaining increased or decreased metabolic activity, since most genes were distinct in each time period, and only one down-regulated gene at dawn (ZT0) ( <italic> <span class="gene">Glyma20g33740</span> </italic> ) was positively regulated in ZT12 (transition light&#x02013;dark). All other genes related to PR-proteins in ZT12 were exclusively detected at this time period (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). In the abiotic stress class (heat, cold, <span class="trait">drought,</span> salt, and wounding), genes were predominantly up-regulated (expression ranged from 6.05 to 14.05&#x000a0;fc). Similar profiles were observed for redox-state metabolism; HSPs; ABA and other genes involved in hormone signaling; the transcription factors MYB, WRKY, DOF, and ERF; and for the class of secondary metabolites. Because the sets of DEGs were smaller in the ZT8 and ZT16 time periods, the profiles observed for the metabolic pathways were under-represented. In ZT8, few genes related to signaling and hormone signaling (ABA and jasmonate) were up-regulated. Conversely, such signaling and hormone signaling (ethylene) processes became repressed in ZT16. Although many genes were down-regulated during pre-dawn (ZT20), up-regulated ones were identified as involved in the processes of secondary metabolism, transcription factors, ABA and ethylene hormone signaling, and redox-state metabolism (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). <fig id="Fig8"> <label> Fig. 8 </label> <caption> <p> Functional roles triggered in soybean plants under water deficit stress. Genes that were differentially expressed in response to water deficit and that exhibited oscillations during the time periods analyzed were mapped to specific stress-related pathways. The color scale shows the log <sub> 2 </sub> fold change: red&#x02009;=&#x02009;up-regulated and blue&#x02009;=&#x02009;down-regulated
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, the gene coding for the oxidative stress 3 (OXS3) protein (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ) exhibited up-regulation in response to light (ZT0) but not dark (ZT20) (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). This OXS3 protein is related to cadmium ion tolerance, as demonstrated in mutants of <italic> AtOXS3 </italic> that were unable to enhance stress tolerance [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. The authors have also associated the <italic> AtOXS3 </italic> gene with regulation by light, suggesting a putative role for this protein in protecting the cell against photooxidation [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. We found eight paralogs in the soybean genome that were related to the <italic> Arabidopsis OXS3 </italic> (AT5G56550.1); three of them were identified in this study as differentially expressed in response to water deficit. In addition to <italic> <span class="gene">Glyma11g33040</span> </italic> reported above, <italic> <span class="gene">Glyma01g00930</span> </italic> and its paralog <italic> <span class="gene">Glyma07g15070</span> </italic> also were up-regulated in ZT0 (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). Although we do not have information about the functional role of those proteins in the water deficit stress scenario, it is tempting to speculate on their importance in the soybean transcriptome since <span class="trait">drought</span> events can produce oxidative stress in the plants [ <xref ref-type="bibr" rid="CR28"> 28 </xref> , <xref ref-type="bibr" rid="CR29"> 29 </xref> ]. Additionally, <italic> <span class="gene">Glyma11g33040</span> </italic> showed a cycling pattern when expressed in normal condition (control plants) (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ), suggesting that its expression can be influenced by time of day.
					</p>
				
					<p>&nbsp;&nbsp;
						Conversely, genes that were positively regulated in response to water deficit, such as the BURP domain-containing protein (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7d </xref> &#x02009;&#x02212;&#x02009; <xref rid="Fig7" ref-type="fig"> g </xref> ), were detected in various time periods (ZT0, ZT4, ZT12, and ZT20). This class of protein contains a conserved domain found in diverse plants and is putatively involved in the localization of proteins within the cell wall matrix through association with a structural domain that might target sites for intermolecular interaction [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Although the function of many BURP proteins is unknown, specific elements have been characterized, and the functional role of these proteins is associated with normal plant metabolic processes, such as seed development [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. Moreover, the genes encoding the BURP domain-containing proteins also are involved in rice responses to abiotic stresses [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Some members of the rice BURP family are responsive to cold, ABA, and <span class="trait">drought</span> and salt stresses and can be induced by a single stress condition or combination of treatments. This family also exhibits temporal and spatial expression pattern differences [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. In soybean plants, the BURP family contains 23 genes that have been classified into five subfamilies (BNM2, USP, RD22, PG1&#x003b2;, and BURP V). Although these genes possess no tissue specificity, they are expressed in response to stress. In particular, genes from the RD22 subfamily are the most responsive to ABA, PEG treatments, and salt stress [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. For the <span class="gene">GmRD22</span> protein ( <italic> <span class="gene">Glyma06g085400</span> </italic> ), the BURP domain seems to have an important role for determining its apoplast localization [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. Furthermore, this protein-coding gene ( <italic> <span class="gene">GmRD22</span> </italic> [ <italic> <span class="gene">Glyma06g085400</span> </italic> ]) exhibits potential responses to salt and osmotic stresses [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. In this study, we identified four up-regulated BURP genes from the classes previously determined by Xu et al. [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]: <italic> RD22 </italic> ( <italic> <span class="gene">Glyma06g08540</span> </italic> and <italic> <span class="gene">Glyma04g08410</span> </italic> ), <italic> USP </italic> ( <italic> <span class="gene">Glyma12g34570</span> </italic> ), and <italic> <span class="gene">Glyma12g34550</span> </italic> (a gene not included in the classification). Expression levels exhibited by these genes indicate the responsiveness of BURP-domain-containing proteins to water deficit, and the oscillation patterns detected in plants under normal development show the regulation of gene expression also might be influenced by time of the day.
					</p>
				
					<p>&nbsp;&nbsp;
						The DEGs that showed oscillating expression in control or drought-stressed conditions (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ) played functional roles as regulatory genes in hormone signaling, cell communication, and abiotic stress-related pathways (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Few genes involved in biotic stress responses (signaling and PR-proteins) also were down- or up-regulated during this time. Genes responsive to multiple stresses are often detected in plants under adverse conditions, since a common set of biological processes triggered by genes induced in both events converge on similar downstream responses [ <xref ref-type="bibr" rid="CR46"> 46 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. Our results showed that several hormone signaling genes, including genes related to jasmonate hormone metabolism, were observed in the early and late morning (ZT0 and ZT4), and redox reactions were up-regulated in most of the time periods (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Increased hormone signaling and fluctuations in the cellular redox status have been associated with plant responses to stress, and, in addition, circadian regulation also has been implicated in these processes [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. With respect to the classification of functional roles assigned to DEGs, a similar fraction of genes down- and up-regulated in the same class can be observed. For instance, <italic> <span class="gene">Glyma07g04310</span> </italic> (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7a </xref> ) and <italic> <span class="gene">Glyma16g00980</span> </italic> (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7c </xref> ), both coding for the same type of protein, showed distinct expression patterns at different time periods. Evaluation of gene expression performed at a single time point during the day can only provide information about the plant&#x02019;s responsiveness to <span class="trait">drought</span> during that specific period in which plants were sampled. In this context, soybean gene networks induced in response to stress and modulated over a <italic> diel </italic> period appear to be a significant feature in the acclimation process, and exploring these interactions might provide novel insight into how plants respond to water deficit.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean transcription factor ORFeome associated with <span class="trait">drought</span> <span class="trait">resistance:</span> a valuable resource to accelerate research on abiotic stress <span class="trait">resistance</span></h2>
			<h3>PMID:26268547</h3>
				
					<h3>PMCID:4534118</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chai </surname>  <given-names> Chenglin </given-names>  </name>  <address>  <email> chaic@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Yongqin </given-names>  </name>  <address>  <email> wangyq@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas </given-names>  </name>  <address>  <email> princesj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Michel </surname>  <given-names> Lydia </given-names>  </name>  <address>  <email> lkm8vb@mail.missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> Xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Department of Computer Science, Informatics Institute, Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 30 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Mainly based on microarray data of soybean root and leaf under dehydration and <span class="trait">drought</span> conditions generated by our group (Valliyodan et al., unpublished data) and other researchers [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ], soybean TFs with a fold change&#x02009;&#x02265;&#x02009;1.5 upon treatments and a <italic> p </italic> -value &#x0003c;0.05 were selected as candidates for building this soybean TF-ORFeome (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). We also included in this TF-ORFeome 19 TFs, which showed a fold change &#x02265;1.5 in at least one of the two tissues (shoots and roots) upon mild <span class="trait">drought</span> stress in our quantitative reverse transcription-PCR (qRT-PCR) analysis (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> A) but without support from our microarray data due to lack of probes [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. A total of 207 soybean full-length TF ORFs were cloned into pENTR&#x02122;/D TOPO or pDONR&#x02122;/Zeo vectors, which meets the &#x0201c;gold standard&#x0201d; criteria as previously defined [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. Detailed information of these clones is provided (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ), including gene locus number, transcript, GenBank accession number, gene family, gene size, vector, ORF sequence, primer sequences, with/without stop codon, and others. These TFs were not equally distributed among 21 gene families, of which the top seven families are MYB, bHLH, APETALA2 (AP2)-ethylene-responsive element binding protein (EREBP), NAC, WRKY, bZIP and Cys2(C2)His2(H2)-type zinc fingers (ZFs), constituting 88&#x000a0;% of the total clones (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Genes from these TF families were found to play important roles in responding to various abiotic stresses, which was very well summarized in a recent review paper [ <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. Several genes from our ORFeome collection were reported as major regulators in the soybean abiotic stress responses, such as <italic> <span class="gene">GmbZIP1</span> </italic> <span class="gene">(Glyma02g14880)</span> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ], <italic> <span class="gene">GmERF3</span> </italic> <span class="gene">(Glyma03g42450)</span> [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ], <italic> <span class="gene">GmDREB2</span> </italic> <span class="gene">(Glyma06g04490)</span> [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ], and ( <italic> <span class="gene">GmNAC004</span> </italic> <span class="gene">(Glyma12g35000)</span> [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. However, functions of the vast majority soybean TFs are yet to be explored. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> Distribution of cloned soybean TF-ORFs among different gene families
					</p>
				
					<p>&nbsp;&nbsp;
						Compared to <span class="trait">drought,</span> dehydration leads to a much lower water potential, and it is also considered as a common stress induced by <span class="trait">drought,</span> extreme temperature, or salinity conditions. Under dehydration treatments (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ), approximately 50&#x000a0;% of the selected genes were regulated the same way as in the <span class="trait">drought</span> conditions, while the expression patterns of other genes were quite different, indicating both pathways share some signaling components while remaining relatively independent. Notably, one <span class="gene">GmNAC</span> gene, <span class="gene">Glyma12g35000.1,</span> showed a dramatic up-regulation upon the dehydration treatment (75 fold change). A very recent study showed that over-expression of <span class="gene">Glyma12g35000.1</span> in <italic> Arabidopsis </italic> enhanced lateral root-growth under both normal and mild <span class="trait">drought</span> stress conditions [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> Expression of randomly selected 50 TFs under dehydration conditions for one hour (DH 1.0&#x000a0;h), five hours (DH 5.0&#x000a0;h) and ten hours (DH 10&#x000a0;h), respectively
					</p>
				
					<p>&nbsp;&nbsp;
						Salinity is another abiotic stress that significantly reduces soybean <span class="trait">yield,</span> and plant responses to salt stress and <span class="trait">drought</span> are very closely related due to their overlapping mechanisms [ <xref ref-type="bibr" rid="CR28"> 28 </xref> , <xref ref-type="bibr" rid="CR29"> 29 </xref> ]. Surprisingly, about 80&#x000a0;% of the same 50 TFs were differentially expressed upon salt stress (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ), suggesting their possible role in salt stress adaption and salinity tolerance. While there was a significant overlap of TFs co-regulated by drought/dehydration/salinity (Figs.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , <xref rid="Fig4" ref-type="fig"> 4 </xref> and <xref rid="Fig5" ref-type="fig"> 5 </xref> ), several of them showed opposite expression patterns, such as <span class="gene">Glyma19g20090.1,</span> <span class="gene">Glyma14g11030.1</span> and <span class="gene">Glyma10g03820.1.</span> Discrepant expressions under these two conditions suggested their distinct roles in response to different stresses. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Expression of randomly selected 50 TFs under salt stress conditions for one hour (Salt 1.0&#x000a0;h), five hours (Salt 5.0&#x000a0;h) and ten hours (Salt 10&#x000a0;h), and twenty-four hours (Salt 24&#x000a0;h), respectively
					</p>
				
					<p>&nbsp;&nbsp;
						The plant hormone abscisic acid (ABA) plays a pivotal role in plant responses to biotic and abiotic stresses [ <xref ref-type="bibr" rid="CR30"> 30 </xref> - <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. When plants are exposed to abiotic stress such as <span class="trait">drought</span> and salinity, ABA regulates stomata aperture to limit water loss through transpiration [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]; on the other hand, the localized ABA signaling, by working together with other phytohormones, regulates root growth, especially lateral root growth plasticity [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. qRT-PCR analysis was performed in order to investigate whether the same set of selected genes are involved in the ABA-dependent signaling pathway (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). Expression of the selected genes showed dramatic changes in both roots and shoots under ABA treatments, and 98&#x000a0;% of them showed&#x02009;&#x02265;&#x02009;1.5 fold change at one or more of the time points. This result indicated that most of the TFs might function dependently on the ABA signal transduction pathway. More than half of the genes showed a similar expression pattern in roots and shoots, while some other TFs exhibited an opposite pattern in different tissues (such as <span class="gene">Glyma05g32850.1</span> and <span class="gene">Glyma20g31500.1),</span> suggesting different roles in shoots and roots (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). <fig id="Fig6"> <label> Fig. 6 </label> <caption> <p> Expression of randomly selected 50 TFs in roots ( <bold> a </bold> ) and shoots ( <bold> b </bold> ) under ABA treatment for half hour (0.5&#x000a0;h), one hour (1.0&#x000a0;h), three hours (3.0&#x000a0;h) and five hours (5&#x000a0;h), respectively
					</p>
				
					<p>&nbsp;&nbsp;
						Since the results presented here are from various comprehensive analyses, plant biologists, especially researchers in the field of abiotic stresses, may find our genomic resources very informative in their search for candidate genes as a starting point. Two examples are given below to demonstrate what function a certain soybean TF may have by putting all data together. <span class="gene">Glyma06g17420,</span> one TF from our ORFeome collection, is annotated as a member of the bHLH superfamily, of which 393 members have been <italic> in-silico </italic> characterized in the soybean genome but until now, none have been functionally characterized in terms of <span class="trait">drought</span> <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. Its subcellular localization in the nucleus suggested it might function as a TF (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Its expression was highly up-regulated in shoots upon <span class="trait">drought</span> and ABA treatments (Figs.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> and <xref rid="Fig6" ref-type="fig"> 6 </xref> ), indicating a role in responding to <span class="trait">drought</span> and probably through an ABA dependent pathway. Since it has little similarity with well characterized MYC2 or ICE1, which are positive regulators of <span class="trait">drought</span> tolerance [ <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR48"> 48 </xref> ], exploring the possible novel function of <span class="gene">Glyma06g17420</span> might be interesting.
					</p>
				
					<p>&nbsp;&nbsp;
						NAC is one of the largest plant-specific gene families with 152 genes in soybean, and 58 of them are putative stress-responsive genes [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. Ectopic expression of several of these stress-responsive genes in Arabidopsis enhanced <span class="trait">resistance</span> to salinity and freezing [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. According to our qRT-PCR analysis, <span class="gene">Glyma13g35550</span> <span class="gene">(GmNAC101)</span> was highly up-regulated by <span class="trait">drought,</span> dehydration, salt and ABA. Recent studies reported that higher expressions of this gene were detected in both shoots and roots of the drought-tolerant cultivar DT51 in comparison with the drought-sensitive cultivar MTD720 under <span class="trait">drought</span> conditions [ <xref ref-type="bibr" rid="CR51"> 51 </xref> , <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. More interestingly, a total of 10 CRE motifs were identified within its 1&#x000a0;kb promoter sequence, indicating that this gene is under complex regulations. All of this evidence suggests that <span class="gene">Glyma13g35550</span> is a potential candidate for in-depth investigation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of root transcriptomes from two contrasting drought-responsive Williams 82 and DT2008 soybean cultivars under normal and dehydration conditions</h2>
			<h3>PMID:26300889</h3>
				
					<h3>PMCID:4528160</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ha </surname>  <given-names> Chien Van </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/243836/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Yasuko </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/243942/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Uyen Thi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/243979/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Le </surname>  <given-names> Dung Tien </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/139256/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Maho </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Kien Huu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/239797/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seki </surname>  <given-names> Motoaki </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/166545/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Dong Van </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/243858/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/64534/overview"/>  </contrib>
				<br/> 
				年份：<day> 06 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Apart from the regulatory TFs, a number of DEGs encoding other types of regulatory proteins, such as kinases and hormone signaling-related proteins, were found in the signaling and protein modification categories. Some of them were predicted to be SnRK (sucrose non-fermenting-related), <span class="gene">RLK</span> (receptor-like), and MAP (mitogen-activated protein) kinases and PP2C (protein phosphatase 2C) proteins based on sequence homology with their <italic> Arabidopsis </italic> counterparts (Supplementary Figure <xref ref-type="supplementary-material" rid="SM13"> S2 </xref> ; Supplementary Tables <xref ref-type="supplementary-material" rid="SM6"> S6E,F </xref> , <xref ref-type="supplementary-material" rid="SM7"> S7E,F </xref> ). These proteins have been shown to be involved in regulation of plant responses to various stresses, including <span class="trait">drought</span> (Umezawa, <xref rid="B52" ref-type="bibr"> 2011 </xref> ; Osakabe et al., <xref rid="B36" ref-type="bibr"> 2013 </xref> ). Among many dehydration-inducible genes coding for non-regulatory proteins are those encoding proteins of transporters, osmoprotectant biosynthesis-related proteins, and detoxification enzymes. Some are deserved to be mentioned, such as ABC (ATP-binding cassette) transporters, the ABA-importing transporter 1 (AIT1)-like proteins that might have ABA importer activity (Kanno et al., <xref rid="B16" ref-type="bibr"> 2012 </xref> ), aquaporins, galactinol synthases, and polyamine oxidases (Supplementary Tables <xref ref-type="supplementary-material" rid="SM6"> S6E </xref> , <xref ref-type="supplementary-material" rid="SM7"> S7E </xref> ). An appropriate change of their levels during stress may lead to a better adaptation of the plants (Osakabe et al., <xref rid="B36" ref-type="bibr"> 2013 </xref> ; Himuro et al., <xref rid="B11" ref-type="bibr"> 2014 </xref> ; Minocha et al., <xref rid="B31" ref-type="bibr"> 2014 </xref> ; Rangan et al., <xref rid="B38" ref-type="bibr"> 2014 </xref> ; Srivastava et al., <xref rid="B43" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Apart from the TF genes, many other dehydration-inducible genes also displayed higher expression levels in DT2008 roots than W82 roots under normal or dehydration conditions (Supplementary Table <xref ref-type="supplementary-material" rid="SM11"> S11 </xref> ), which might contribute to differential <span class="trait">drought</span> tolerance of DT2008 and W82. Results summarized in Figure <xref ref-type="fig" rid="F5"> 5 </xref> indicated that the short-term dehydration-induced expression changes might be more highly required for enhanced <span class="trait">drought</span> tolerance of DT2008 vs. W82 than the prolonged dehydration-induced ones, as significantly higher number of DEGs were identified in DT-D2/WT-D2 comparison than DT-D10/WT-D10 comparison. With regard to dehydration-upregulated genes with higher expression levels in DT2008 roots vs. W82 roots under well-watered conditions (Supplementary Tables <xref ref-type="supplementary-material" rid="SM11"> S11A&#x02013;C </xref> ), <italic> <span class="gene">Glyma.14G216500</span> </italic> encodes an ortholog STH2 (salt tolerance homolog2) (Table <xref ref-type="table" rid="T1"> 1 </xref> ), a B-box TF that can act as a positive regulator of photomorphogenesis and anthocyanin biosynthesis (Datta et al., <xref rid="B4" ref-type="bibr"> 2007 </xref> ). This gene might play a role in enhanced tolerance of DT2008 as anthocyanins are known to protect plants against various environmental stresses, including <span class="trait">drought</span> (Pourcel et al., <xref rid="B37" ref-type="bibr"> 2007 </xref> ). Modulation of the signaling molecule phospholipids, which involved in regulation of plant response to environmental stimuli, through <italic> <span class="gene">Glyma.04G083000</span> </italic> (induced by dehydration in both DT2008 and W82 background, Table <xref ref-type="table" rid="T1"> 1 </xref> ) and <italic> <span class="gene">Glyma.20G189100</span> </italic> that encode putative proteins with function in phosphoinositide signaling, might also be responsible for increased tolerance of DT2008 to <span class="trait">drought</span> (Liu et al., <xref rid="B28" ref-type="bibr"> 2013 </xref> ). As for the dehydration-inducible genes showing higher transcription levels in DT2008 roots vs. W82 roots under dehydration (Supplementary Tables <xref ref-type="supplementary-material" rid="SM11"> S11D&#x02013;F </xref> ), <italic> <span class="gene">Glyma.19G227800,</span> <span class="gene">Glyma.16G003500</span> </italic> , and <italic> <span class="gene">Glyma.13G095200</span> </italic> encoding osmoprotectant biosynthesis-, detoxification- or cell wall-related proteins (Table <xref ref-type="table" rid="T1"> 1 </xref> ), such as the orthologs of <italic> Arabidopsis </italic> AtGOLS2 ( <italic> Arabidopsis thaliana </italic> galactinol synthase 2), glyoxalase I family protein and xyloglucan endotransglycosylase, may play important roles in better adaptation of DT2008 to <span class="trait">drought</span> relative to W82 as supported by previous studies (Xu et al., <xref rid="B57" ref-type="bibr"> 1995 </xref> ; Taji et al., <xref rid="B45" ref-type="bibr"> 2002 </xref> ; Kaur et al., <xref rid="B18" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Additionally, dehydration-repressible genes with lower expression level in drought-tolerant DT2008 roots than drought-sensitive W82 might also contribute to the better performance of DT2008 vs. W82 under <span class="trait">drought.</span> Among the dehydration-repressible genes that had lower expression levels in DT2008 than W82 under well-watered conditions (Supplementary Tables <xref ref-type="supplementary-material" rid="SM12"> S12A&#x02013;C </xref> ), <italic> <span class="gene">Glyma.02G069400</span> </italic> codes for an ortholog of <italic> Arabidopsis </italic> BIN2 (brassinosteroid-insensitive 2) (Table <xref ref-type="table" rid="T1"> 1 </xref> ). It was reported that the rice ortholog of BIN2, the OsGSK1 (glycogen synthase 3-like protein kinase), acts as a negative regulator of plant responses to multiple stresses, including <span class="trait">drought</span> (Koh et al., <xref rid="B21" ref-type="bibr"> 2007 </xref> ). Repression of <italic> <span class="gene">Glyma.02G069400</span> </italic> might therefore contribute to increased <span class="trait">drought</span> tolerance of DT2008. Another example is <italic> <span class="gene">Glyma.12G202400</span> </italic> encoding a protein with high homology to an <italic> Arabidopsis </italic> phosphatase 2C protein (AT4G03415) (Table <xref ref-type="table" rid="T1"> 1 </xref> ) that might be involved in <span class="trait">drought</span> response perhaps through its interaction with CPK16 of Ca <sup> 2+ </sup> -dependent protein kinase/sucrose non-fermenting related kinase (CPK/SnRK) superfamily (Curran et al., <xref rid="B3" ref-type="bibr"> 2011 </xref> ). CPK16 has been known to be implicated in regulation of root gravitropism that is a trait important for plant response to water stress (Kirkham, <xref rid="B20" ref-type="bibr"> 2008 </xref> ; Huang et al., <xref rid="B14" ref-type="bibr"> 2013 </xref> ). With regard to the dehydration-repressible genes that displayed lower expression levels in DT2008 roots than W82 roots under dehydration (Figure <xref ref-type="fig" rid="F5"> 5C </xref> ; Supplementary Tables <xref ref-type="supplementary-material" rid="SM12"> S12D&#x02013;F </xref> ), <italic> <span class="gene">Glyma.06G050900</span> </italic> and <italic> <span class="gene">Glyma.08G254400</span> </italic> encoding phosphatase 2C and bZIP orthologs of <italic> Arabidopsis </italic> , respectively, caught our attention (Table <xref ref-type="table" rid="T1"> 1 </xref> ). <italic> <span class="gene">Glyma.06G050900</span> </italic> exhibited lower expression in DT2008 roots than W82 roots under both normal and dehydration conditions (Table <xref ref-type="table" rid="T1"> 1 </xref> ). It is well established that many members of the phosphatase 2C family are involved in stress signaling (Schweighofer et al., <xref rid="B42" ref-type="bibr"> 2004 </xref> ). Several phosphatase 2C proteins have been found to act as negative regulators of ABA signaling (Umezawa et al., <xref rid="B53" ref-type="bibr"> 2010 </xref> ). As for <italic> <span class="gene">Glyma.08G254400</span> </italic> , although there is no specific information either for this gene or its highest <italic> Arabidopsis </italic> homolog (AT3G30530/AtbZIP42), there have been published reports that several bZIP TFs act as negative regulators of <span class="trait">drought</span> tolerance. For instance, overexpression of <italic> OsbZIP52 </italic> in rice significantly enhanced sensitivity of transgenic plants to cold and <span class="trait">drought</span> stresses (Liu et al., <xref rid="B27" ref-type="bibr"> 2012 </xref> ). Thus, downregulation of such a gene would allow plants to adapt better to adverse environmental conditions.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Association Study of Ureide Concentration in Diverse Maturity Group IV Soybean [ <italic> Glycine max </italic> (L.) Merr.] Accessions</h2>
			<h3>PMID:26374596</h3>
				
					<h3>PMCID:4632059</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ray </surname>  <given-names> Jeffery D. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-8879-0884 </contrib-id>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dhanapal </surname>  <given-names> Arun Prabhu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-9686-7260 </contrib-id>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Shardendu K. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-7437-330X </contrib-id>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoyos-Villegas </surname>  <given-names> Valerio </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> &#x000a7; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> James R. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Purcell </surname>  <given-names> Larry C. </given-names>  </name>  <xref ref-type="aff" rid="aff5"> ** </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> King </surname>  <given-names> C. Andy </given-names>  </name>  <xref ref-type="aff" rid="aff5"> ** </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boykin </surname>  <given-names> Debbie </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> &#x02020;&#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> &#x02021;&#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> &#x02021;&#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fritschi </surname>  <given-names> Felix B. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Crop Genetics Research Unit, USDA-ARS, Stoneville, Mississippi 38776 </aff>  <aff id="aff2">  <label> &#x02020; </label> Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211 </aff>  <aff id="aff3">  <label> &#x02021; </label> Crop Systems and Global Change Lab, USDA-ARS, Beltsville, Maryland 20705 </aff>  <aff id="aff4">  <label> &#x000a7; </label> Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan 48824 </aff>  <aff id="aff5">  <label> ** </label> Department of Crop, Soil, and Environmental Sciences, University of Arkansas, Fayetteville, Arkansas 72704 </aff>  <aff id="aff6">  <label> &#x02020;&#x02020; </label> Southeast Area Statistics, USDA-ARS, Stoneville, Mississippi 38776 </aff>  <aff id="aff7">  <label> &#x02021;&#x02021; </label> Soybean Genomics and Improvement Lab, USDA-ARS, Beltsville, Maryland 20705 </aff>
				<br/> 
				年份：<day> 08 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<xref rid="bib38" ref-type="bibr"> Werner <italic> et al. </italic> (2013) </xref> examined two gene copies of allantoate amidohydrolase <span class="gene">(GmAAH1</span> and <span class="gene">GmAAH2),</span> ureidoglycine aminohydrolase <span class="gene">(GmUGlyAH1</span> and <span class="gene">GmUGlyAH2),</span> and ureidoglycolate amidohydrolase <span class="gene">(GmUAH1</span> and <span class="gene">GmUAH2),</span> which are all involved in ureide hydrolysis ( <xref rid="bib38" ref-type="bibr"> Werner <italic> et al. </italic> 2013 </xref> ). Three of these genes, <span class="gene">GmUAH1</span> <span class="gene">(Glyma10g32850,</span> chromosome 10), <span class="gene">GmAAH2</span> <span class="gene">(Glyma15g16870,</span> chromosome 15), and <span class="gene">GmUAH2</span> <span class="gene">(Glyma20g34790,</span> chromosome 20), were each near a different locus of the 53 putative loci (see <xref ref-type="table" rid="t5"> Table 5 </xref> and <xref ref-type="fig" rid="fig3"> Figure 3 </xref> ). For <span class="gene">GmUAH2</span> (chromosome 20), one nearby significant SNP association was detected ( <xref ref-type="table" rid="t4"> Table 4 </xref> , locus 53), while three ( <xref ref-type="table" rid="t4"> Table 4 </xref> , locus 26) and two ( <xref ref-type="table" rid="t4"> Table 4 </xref> , locus 40) nearby significant SNP associations marked the loci near <span class="gene">GmUAH1</span> (chromosome 10) and <span class="gene">GmAAH2</span> (chromosome 15), respectively. For all three loci, the major allele was associated with an increase in ureide concentration ( <xref ref-type="table" rid="t4"> Table 4 </xref> ) and, thus, the minor allele was associated with a decrease in ureide concentration. Potentially, the reduced ureide concentration in the accessions with the minor alleles near these genes might be associated with more active/efficient ureide metabolism. Reduced petiole ureide concentrations have been associated with increased <span class="trait">drought</span> tolerant N-fixation, possibly through the elimination/reduction of feedback inhibition caused by a buildup of ureides ( <xref rid="bib29" ref-type="bibr"> Sinclair and Serraj 1995 </xref> ; <xref rid="bib26" ref-type="bibr"> Serraj and Sinclair 1996 </xref> ; <xref rid="bib4" ref-type="bibr"> de Silva <italic> et al. </italic> 1996 </xref> ; <xref rid="bib11" ref-type="bibr"> Gordon <italic> et al. </italic> 1997 </xref> ; <xref rid="bib22" ref-type="bibr"> Purcell <italic> et al. </italic> 1998 </xref> ; <xref rid="bib27" ref-type="bibr"> Serraj <italic> et al. </italic> 1999 </xref> , <xref rid="bib28" ref-type="bibr"> 2001 </xref> ; <xref rid="bib35" ref-type="bibr"> Vadez <italic> et al. </italic> 2000 </xref> ; <xref rid="bib16" ref-type="bibr"> King and Purcell 2005 </xref> ; <xref rid="bib19" ref-type="bibr"> Ladrera <italic> et al. </italic> 2007 </xref> ). Thus, if these three loci are associated with greater ureide catabolism as a result of more active/efficient alleles of these hydrolases, then the accessions with the minor allele may also exhibit greater drought-tolerant N-fixation. While no SNPs evaluated in this study were located between the start and stop positions of <span class="gene">GmUAH1,</span> <span class="gene">GmAAH2,</span> and <span class="gene">GmUAH2,</span> comparisons of the sequences of these genes between genotypes with the major and minor alleles for the nearby significant SNP could be of interest. Additionally, confirmation of tolerance and more detailed investigation of these accessions may aid in the identification of the molecular mechanisms associated with drought-tolerant N-fixation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome-Wide Identification of Reference Genes for Expression Analysis of Soybean Responses to <span class="trait">Drought</span> Stress along the Day</h2>
			<h3>PMID:26407065</h3>
				
					<h3>PMCID:4583485</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Marcolino-Gomes </surname>  <given-names> Juliana </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana Aparecida </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fuganti-Pagliarini </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakayama </surname>  <given-names> Thiago Jonas </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ribeiro Reis </surname>  <given-names> Rafaela </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bou&#x000e7;as Farias </surname>  <given-names> Jose Renato </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harmon </surname>  <given-names> Frank G. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Correa Molinari </surname>  <given-names> Hugo Bruno </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Correa Molinari </surname>  <given-names> Mayla Daiane </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The soybean transcriptome displays strong variation along the day in optimal growth conditions and also in response to adverse circumstances, like <span class="trait">drought</span> stress. However, no study conducted to date has presented suitable reference genes, with stable expression along the day, for relative gene expression quantification in combined studies on <span class="trait">drought</span> stress and diurnal oscillations. Recently, water deficit responses have been associated with circadian clock oscillations at the transcription level, revealing the existence of hitherto unknown processes and increasing the demand for studies on plant responses to <span class="trait">drought</span> stress and its oscillation during the day. We performed data mining from a transcriptome-wide background using microarrays and RNA-seq databases to select an unpublished set of candidate reference genes, specifically chosen for the normalization of gene expression in studies on soybean under both <span class="trait">drought</span> stress and diurnal oscillations. Experimental validation and stability analysis in soybean plants submitted to <span class="trait">drought</span> stress and sampled during a 24 h timecourse showed that four of these newer reference genes ( <italic> FYVE </italic> , <italic> NUDIX </italic> , <italic> Golgin-84 </italic> and <italic> CYST </italic> ) indeed exhibited greater expression stability than the conventionally used housekeeping genes ( <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2;-actin </italic> ) under these conditions. We also demonstrated the effect of using reference candidate genes with different stability values to normalize the relative expression data from a drought-inducible soybean gene ( <italic> <span class="gene">DREB5</span> </italic> ) evaluated in different periods of the day.
					</p>
				
					<p>&nbsp;&nbsp;
						In this context, a reference gene must be experimentally validated for specific tissues, genotypes and experimental designs. The soybean genes <italic> TUA </italic> <span class="gene">(Glyma08g12140),</span> <italic> <span class="gene">TUB</span> </italic> <span class="gene">(Glyma03g27970),</span> <italic> ELF1-&#x003b2; </italic> <span class="gene">(Glyma13g04050),</span> <italic> &#x003b2;-actin </italic> <span class="gene">(Glyma15g05570)</span> and <italic> <span class="gene">GAPDH</span> </italic> <span class="gene">(Glyma06g01850)</span> have been widely used as references in gene expression studies on <span class="trait">drought</span> responses [ <xref rid="pone.0139051.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0139051.ref019" ref-type="bibr"> 19 </xref> ]. On the other hand, isopentenyl diphosphate ( <italic> IPP2 </italic> ), actin and ubiquitin are the most commonly used reference genes in studies investigating circadian/diurnal oscillations [ <xref rid="pone.0139051.ref020" ref-type="bibr"> 20 </xref> &#x02013; <xref rid="pone.0139051.ref025" ref-type="bibr"> 25 </xref> ]. Thus, no study conducted to date has evaluated the expression stability of reference genes for the study of both water deficit stress and circadian oscillations in soybean. Hence, in this study, after evaluating gene expression in response to <span class="trait">drought</span> during the day, we present a novel set of reference genes suitable for the normalization of relative expression data from combined studies on water deficit and diurnal oscillations.
					</p>
				
					<p>&nbsp;&nbsp;
						Under a second approach, we evaluated the expression stability of genes from 36 RNA-seq libraries. These libraries were synthesized from leaves of the drought-sensitive soybean genotype BR16, subjected to moderate <span class="trait">drought</span> stress on <span class="gene">V2</span> developmental stage, sampled over a 24 h timecourse, with 4h intervals [ <xref rid="pone.0139051.ref010" ref-type="bibr"> 10 </xref> ]. These data are deposited in the NCBI&#x02019;s Gene Expression Omnibus [GEO; <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/"> http://www.ncbi.nlm.nih.gov/geo/ </ext-link> ] repository and are accessible through GEO Series accession number GSE69469 (geospiza.com/Products/AnalysisEdition.shtml). To compare gene expression between different times and conditions, we log <sub> 2 </sub> -transformed the normalized reads per mapped million (RPM) value.
					</p>
				
					<p>&nbsp;&nbsp;
						After carrying out the efficiency analysis, the expression levels of the candidate reference genes were analyzed separately at 6 time points (ZT0, ZT4, ZT8, ZT12, ZT16 and ZT20) in plants under control <italic> vs </italic> . <span class="trait">drought</span> stress condition in order to assess their expression stability along the day. The expression of the target gene ( <italic> <span class="gene">GmDREB5</span> </italic> -like; <span class="gene">Glyma12g33020)</span> [ <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> ] was also measured under the experimental conditions described above. The reactions were performed in triplicate with cycling parameters similar to those described above for the amplification efficiency analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						The relative expression level of the drought-responsive gene <italic> <span class="gene">GmDREB5</span> </italic> -like [ <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> ] was measured in leaf samples from BR16 plants subjected to moderate <span class="trait">drought,</span> sampled over a 24 h timecourse, under the experimental conditions described above. For each time point (ZT0, ZT4, ZT8, ZT12, ZT16 and ZT20), three biological replicates, with three technical replicates each, were analyzed. Target expression was normalized using a combination of 2 reference genes with high ( <italic> FYVE </italic> and <italic> GOL84 </italic> ), intermediate ( <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2; -actin </italic> ) and low ( <italic> <span class="gene">DNAJ</span> </italic> and <italic> NCL1 </italic> ) expression stabilities. Plants grown under normal water conditions (control plants) were used to calibrate relative expression.
					</p>
				
					<p>&nbsp;&nbsp;
						The differential expression of some of the selected candidate genes has been reported during biotic and abiotic stress responses. The gene <span class="gene">Glyma13g24060,</span> for example, encodes a protein similar to a NUDIX hydrolase protein from <italic> Arabidopsis </italic> . The NUDIX hydrolase family is widespread, from eukaryotes to <italic> Archaea </italic> , and consists of pyrophosphohydrolases that act upon substrates with a general nucleoside diphosphate structure, including (deoxy)ribonucleoside diphosphates and triphosphates, nucleotide sugars, coenzymes and RNA caps [ <xref rid="pone.0139051.ref042" ref-type="bibr"> 42 </xref> , <xref rid="pone.0139051.ref043" ref-type="bibr"> 43 </xref> ]. Members of the NUDIX family have been reported to be induced by salt, <span class="trait">drought,</span> heat, and cold in <italic> Chrysanthemum lavandulifolium </italic> [ <xref rid="pone.0139051.ref044" ref-type="bibr"> 44 </xref> ]. Similarly, <span class="gene">Glyma10g44020</span> encodes a protein from the DnaJ/Hsp40 cysteine-rich domain superfamily, which is described as being involved in diverse cellular processes (protein folding, translocation, and degradation) [ <xref rid="pone.0139051.ref045" ref-type="bibr"> 45 </xref> ], including biotic and abiotic stress responses [ <xref rid="pone.0139051.ref046" ref-type="bibr"> 46 </xref> &#x02013; <xref rid="pone.0139051.ref049" ref-type="bibr"> 49 </xref> ]. However, analysis of the soybean <italic> NUDIX </italic> ( <xref rid="pone.0139051.g001" ref-type="fig"> Fig 1 </xref> ) and <italic> <span class="gene">DNAJ</span> </italic> ( <xref rid="pone.0139051.g002" ref-type="fig"> Fig 2 </xref> ) genes in microarray and RNA-Seq databases showed high expression stability in response to <span class="trait">drought</span> and diurnal oscillations, suggesting that these soybean genes could be reliable candidate reference genes for <span class="trait">drought</span> studies.
					</p>
				
					<p>&nbsp;&nbsp;
						Although most of the candidate reference genes performed well in response to the applied experimental conditions, the <italic> NCL1 </italic> and <italic> <span class="gene">DNAJ</span> </italic> genes showed lower stability ( <xref rid="pone.0139051.g003" ref-type="fig"> Fig 3A and 3B </xref> ). The <italic> NCL1 </italic> gene produces an RNA (cytosine-5)-methyltransferase involved in epigenetic modifications of tRNA [ <xref rid="pone.0139051.ref039" ref-type="bibr"> 39 </xref> , <xref rid="pone.0139051.ref040" ref-type="bibr"> 40 </xref> ]. The analysis of intragroup variation performed on NormFinder for <italic> NCL1 </italic> revealed that under control conditions this was one of the least stable genes ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4A </xref> ), whereas in stressed plants it was among the most stable ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4B </xref> ), indicating that <italic> NCL1 </italic> is not a suitable reference gene for studies on gene expression oscillation during the day in plants under control conditions. Previous studies show that the oscillation of genes that cycle daily in normal conditions may be altered due the imposition of abiotic stresses, like cold and <span class="trait">drought</span> stress. The mRNA levels of some chestnut genes, like <italic> <span class="gene">TOC1</span> </italic> and <italic> <span class="gene">LHY</span> </italic> cycle daily in seedlings and adult plants, as expected. However, during chilling stress (4&#x000b0;C), mRNA levels of these genes were higher and did not oscillate [ <xref rid="pone.0139051.ref051" ref-type="bibr"> 51 </xref> ]. Similar events were observed in soybean plants subjected to severe <span class="trait">drought</span> stress, where a general reduction in the amplitude of the daily oscillation was observed for most clock genes, including the <italic> <span class="gene">GmPRR3-like</span> </italic> , <italic> <span class="gene">GmPRR7-like</span> </italic> , <italic> <span class="gene">GmPRR9-</span> like </italic> , <italic> <span class="gene">GmGI-like</span> </italic> , <italic> <span class="gene">GmZTL-like</span> </italic> , and <italic> <span class="gene">GmCHE-like</span> </italic> genes [ <xref rid="pone.0139051.ref010" ref-type="bibr"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, the expression of the <italic> <span class="gene">DNAJ</span> </italic> gene was unstable in both control and stressed conditions ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4 </xref> ). Genes from the <span class="gene">DNAJ</span> family have been reported to be <span class="trait">drought</span> responsive in many species, and its lack of stability may therefore be explained in part by its possible involvement in <span class="trait">drought</span> responses in soybean, as studies have reported that drought-responsive genes oscillate in response to diurnal oscillations [ <xref rid="pone.0139051.ref010" ref-type="bibr"> 10 </xref> , <xref rid="pone.0139051.ref012" ref-type="bibr"> 12 </xref> , <xref rid="pone.0139051.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> , <xref rid="pone.0139051.ref052" ref-type="bibr"> 52 </xref> ]. Additionally, the intragroup analysis shows that <italic> Golgin-84 </italic> and <italic> FYVE </italic> ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4A </xref> ) are the most reliable reference genes for gene expression normalization when studying diurnal oscillations in plants under normal water conditions, whereas <italic> NUDIX </italic> and <italic> NCL1 </italic> ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4B </xref> ) are the most strongly indicated for use in studies under <span class="trait">drought</span> conditions. Furthermore, the intergroup variation analysis allowed us to identify the genes <italic> NUDIX </italic> and <italic> <span class="gene">DNAJ</span> </italic> as the most and least stable genes, respectively, for data normalization in studies comparing gene expression under control and <span class="trait">drought</span> stress, without considering the time of day ( <xref rid="pone.0139051.g004" ref-type="fig"> Fig 4C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A previous study showed that the normalization of soybean genes under circadian regulation using unstable reference genes may lead to erroneous data interpretation [ <xref rid="pone.0139051.ref019" ref-type="bibr"> 19 </xref> ]. To demonstrate the effect of data normalization using reference genes with different stability values in response to <span class="trait">drought</span> stress during the day, we evaluated the relative expression of the drought-responsive gene <italic> <span class="gene">GmDREB5</span> </italic> -like [ <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> ] using a combination of 2 reference genes with high ( <italic> FYVE and GOL84 </italic> ), intermediate ( <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2; -actin </italic> ) and low ( <italic> <span class="gene">DNAJ</span> and NCL1 </italic> ) expression stabilities ( <xref rid="pone.0139051.g005" ref-type="fig"> Fig 5 </xref> ). The <italic> <span class="gene">GmDREB5</span> </italic> gene expression was previously investigated in response to <span class="trait">drought</span> in short-term stress conditions [ <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> ], and its expression was normalized by <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2; &#x02013;actin </italic> . As expected, our results show that the expression of <italic> <span class="gene">GmDREB5</span> </italic> -like in response to <span class="trait">drought</span> oscillates during the day, as previously described for this gene [ <xref rid="pone.0139051.ref030" ref-type="bibr"> 30 </xref> ] and for other genes of the DREB subfamily [ <xref rid="pone.0139051.ref010" ref-type="bibr"> 10 </xref> , <xref rid="pone.0139051.ref053" ref-type="bibr"> 53 </xref> &#x02013; <xref rid="pone.0139051.ref058" ref-type="bibr"> 58 </xref> ] ( <xref rid="pone.0139051.g005" ref-type="fig"> Fig 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Gene expression was measured in leaf tissues of soybean subjected to <span class="trait">drought</span> stress at different times of day (ZT0, ZT4, ZT8, ZT12, ZT16 and ZT20). The raw data were normalized using a combination of 2 candidate reference genes with high ( <italic> FYVE </italic> and <italic> GOL84 </italic> ), intermediate ( <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2; -actin </italic> ) and low ( <italic> <span class="gene">DNAJ</span> </italic> and <italic> NCL1 </italic> ) expression stabilities. The relative expression of <italic> <span class="gene">GmDREB5</span> </italic> -like under <span class="trait">drought</span> stress was determined after comparison with the control sample (non-stressed plants). Asterisks indicate statistically significant changes in gene expression between <span class="trait">drought</span> stressed and non-stressed plants. The analyses were performed using REST 2009 software.
					</p>
				
					<p>&nbsp;&nbsp;
						However, our results demonstrated that choosing reference genes with diverse stability of expression can lead to differences on gene expression data interpretation when evaluating combined studies on water deficit and diurnal oscillations. As shown in <xref rid="pone.0139051.g005" ref-type="fig"> Fig 5 </xref> , at ZT16 the normalization of <italic> <span class="gene">GmDREB5</span> </italic> -like gene using the least stable genes ( <italic> <span class="gene">DNAJ</span> and NCL1 </italic> ) resulted in higher expression levels than observed for normalization using genes with high (FYVE and GOL84) and intermediate expression stability (ELF1-&#x003b2; and &#x003b2; &#x02013;actin). Additionally, slight changes in gene expression, such as the down-regulation of the target gene at ZT0 and ZT20, were detected only by normalization using genes with high ( <italic> FYVE </italic> and <italic> GOL84 </italic> ) and intermediate expression stability ( <italic> ELF1-&#x003b2; </italic> and <italic> &#x003b2; &#x02013;actin </italic> ) ( <xref rid="pone.0139051.g005" ref-type="fig"> Fig 5A </xref> ). These results emphasize the importance of selecting reference genes with stable expression for accurate gene expression analysis on <span class="trait">drought</span> responses and diurnal oscillations.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Medicago truncatula </italic> and <italic> Glycine max </italic> : Different <span class="trait">Drought</span> Tolerance and Similar Local Response of the Root Nodule Proteome</h2>
			<h3>PMID:26503705</h3>
				
					<h3>PMCID:4673605</h3>
					<h4>期刊:Journal of Proteome Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="ath1">  <name>  <surname> Gil-Quintana </surname>  <given-names> Erena </given-names>  </name>  <xref rid="aff1" ref-type="aff"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author" id="ath2">  <name>  <surname> Lyon </surname>  <given-names> David </given-names>  </name>  <xref rid="aff2" ref-type="aff"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author" id="ath3">  <name>  <surname> Staudinger </surname>  <given-names> Christiana </given-names>  </name>  <xref rid="aff2" ref-type="aff"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author" corresp="yes" id="ath4">  <name>  <surname> Wienkoop </surname>  <given-names> Stefanie </given-names>  </name>  <xref rid="cor2" ref-type="other"> * </xref>  <xref rid="aff2" ref-type="aff"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author" corresp="yes" id="ath5">  <name>  <surname> Gonz&#x000e1;lez </surname>  <given-names> Esther M. </given-names>  </name>  <xref rid="cor1" ref-type="other"> * </xref>  <xref rid="aff1" ref-type="aff"> &#x02020; </xref>  </contrib>  <aff id="aff1">  <label> &#x02020; </label> Department
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The nodule proteins were analyzed to compare the response to <span class="trait">drought</span> stress of both legumes. The mass spectra obtained were searched against MT-fasta and <span class="gene">GM-fasta.</span> This resulted in a stringent identification of 304 <italic> M. truncatula </italic> plant nodule proteins and 341 soybean plant nodule proteins. The complete lists of identified proteins and their spectral count values for relative quantification are included in the <xref rid="notes-1" ref-type="notes"> Supporting Information </xref> in tables in the electronic appendix ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.5b00617/suppl_file/pr5b00617_si_002.xlsx"> Table S2 </ext-link> for <italic> M. truncatula </italic> proteins and <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.5b00617/suppl_file/pr5b00617_si_003.xlsx"> Table S3 </ext-link> for soybean proteins).
					</p>
				
					<p>&nbsp;&nbsp;
						Venn diagrams were employed to examine the <span class="trait">drought</span> response of the nodule proteomes ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). The Gene Ontology Annotation Database from UniProt was utilized to classify proteins into ten functional groups for both <italic> M. truncatula </italic> and <italic> G. max </italic> . These diagrams provided an important overview of whether the changes represented specific or common responses. Of the relatively quantified proteins in <italic> Medicago </italic> , 28 changed significantly and exclusively in the total <span class="trait">drought</span> comparison (D/C), 17 proteins were altered exclusively in the partial <span class="trait">drought</span> comparison (PDD/PDC), and 16 proteins were shared between both comparisons ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). In soybean, however, 20 proteins varied significantly and exclusively in the total <span class="trait">drought</span> comparison (D/C), 15 were altered in the partial <span class="trait">drought</span> comparison (PDD/PDC), and 4 were shared between both comparisons ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). The main functional groups that changed in the total (D/C) and partial <span class="trait">drought</span> (PDD/PDC) comparisons in both legumes were the glycolysis and TCA cycle and amino acid metabolism, followed by redox- and stress-related proteins ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). It should be noted that in soybean, the proteins with an unknown function represent almost 49% of the studied proteins while in <italic> M. truncatula </italic> , these are around 6%. The identified proteins that change exclusively in total or partial <span class="trait">drought</span> comparisons are listed in <xref rid="tbl2" ref-type="other"> Table <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> for <italic> M. truncatula </italic> and <xref rid="tbl3" ref-type="other"> Table <xref rid="tbl3" ref-type="other"> 3 </xref> </xref> for <italic> G. max </italic> ; unknown proteins are not displayed. A common pattern was observed in all the identified proteins: a decrease in the relative content of proteins from the treatments directly subjected to <span class="trait">drought</span> (D and PDD) in comparison with their controls (C and PDC) ( <xref rid="tbl2" ref-type="other"> Tables <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <xref rid="tbl3" ref-type="other"> 3 </xref> and <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). Glycolysis and TCA was the most numerous group that changed in total and partial <span class="trait">drought</span> comparisons, and similar proteins were altered significantly in both legumes (mainly sucrose synthase (SuSy), fructose-bisphosphate aldolase (FBPA), phosphoenolpyruvate carboxylase (PEPC), and malate dehydrogenase (MDH) ( <xref rid="tbl2" ref-type="other"> Tables <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <xref rid="tbl3" ref-type="other"> 3 </xref> ). Amino-acid metabolism was the second largest protein functional group and included nitrogen assimilation enzymes such as aspartate aminotransferase (AAT), glutamate synthase (GOGAT), glutamine synthetase (GS), and asparagine synthetase (AS); as well as enzymes involved in the metabolism of sulfur-containing amino acids ( <xref rid="tbl2" ref-type="other"> Tables <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <xref rid="tbl3" ref-type="other"> 3 </xref> ). In <italic> M. truncatula </italic> , protein synthesis and degradation and nitrogen fixation-related proteins were also numerous, while this latter functional group was almost absent in soybean plants ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> and <xref rid="tbl2" ref-type="other"> Tables <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <xref rid="tbl3" ref-type="other"> 3 </xref> ). Additionally in soybean, in contrast to <italic> M. truncatula </italic> , some exceptions were found to the general pattern of decline in D and PDD nodules. A total of three redox- and stress-related proteins presented a higher relative content in PDD nodules in comparison with PDC (an In2-1 protein and two ATP synthase proteins) ( <xref rid="tbl3" ref-type="other"> Table <xref rid="tbl3" ref-type="other"> 3 </xref> </xref> and <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). The proteins that showed the highest significant fold-change were two SuSys (P13708; <span class="gene">Glyma13g17421.1</span> and I1L1U4; and <span class="gene">Glyma09g08550.5)</span> in soybean plants and one PEPC (G7IH71), two AS (TC180197; O24483; and G7JZK0) and a lipoxygenase (G7LIY0) in <italic> M. truncatula </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						The main feature of the SRS experimental setup is that it enables the identification of systemic changes. Proteins responding systemically have been categorized by comparing the proteins from C nodules with those from PDC nodules. Therefore, significant changes in the PDC nodules in comparison with those in C conditions were understood as systemic changes caused by <span class="trait">drought</span> stress. The proteins that changed systemically are listed in <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ; unknown proteins are excluded. From the relatively quantified proteins, 25% changed systemically in <italic> Medicago </italic> and 10% in soybean ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ). An overall and marked increase in the relative quantification of PDC proteins compared with C conditions was seen as being a general systemic response in both legumes ( <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ). The only exception to this general pattern was a <italic> Medicago </italic> FBPA that showed a higher relative content in C nodules than in the other treatments ( <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ). It is worth noting that the proteins changing systemically also changed in the partial and total <span class="trait">drought</span> comparisons; in other words, none of the proteins exhibited an exclusively systemic response, with the exception of an uncharacterized soybean protein (I1KK66 and <span class="gene">Glyma07g16910.1)</span> ( <xref rid="notes-1" ref-type="notes"> Table S3 </xref> ). The rest of the proteins that varied significantly in the systemic comparison (PDC/C) also altered significantly in the total <span class="trait">drought</span> comparison (D/C: 2 proteins in <italic> M. truncatula </italic> and 7 proteins in soybean), in the partial <span class="trait">drought</span> comparison (PDD/PDC: 19 proteins in <italic> M. truncatula </italic> and 4 proteins in soybean) or were divided among the three comparisons (one protein in soybean and 6 in <italic> M. truncatula </italic> ) ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The most numerous functional groups in the <italic> M. truncatula </italic> systemic comparison were glycolysis and TCA cycle and protein synthesis and degradation, which was in agreement with the local response of the proteome ( <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ). Within the set of proteins showing a systemic response in <italic> M. truncatula </italic> , it is remarkable that three cell-wall- and organization-related proteins changed significantly (two actins: G7IL85 and G7JAX5, and one tubulin &#x003b2; chain: G7J588), while this functional group was not significantly altered in the total <span class="trait">drought</span> comparison (D/C) ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> and <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ). From the 13 proteins that changed in the systemic comparison ( <xref rid="fig3" ref-type="fig"> Figure <xref rid="fig3" ref-type="fig"> 3 </xref> </xref> ) in soybean plants, only two were characterized: a transketolase <span class="gene">(Glyma03g03200.1)</span> that also altered in the partial <span class="trait">drought</span> comparison and a peptidyl-prolyl cis&#x02013;trans isomerase (K7LSN8 and <span class="gene">Glyma12g02790.2)</span> that was shared between all of the comparisons ( <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Together with the above-cited proteins, the relative content of <span class="gene">LOX</span> proteins declines significantly in the D and PDD nodules of both legumes ( <xref rid="tbl2" ref-type="other"> Tables <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <xref rid="tbl3" ref-type="other"> 3 </xref> ). Furthermore, in <italic> M. truncatula </italic> , one of the nodule proteins that decreased the most after the water deprivation treatment was a <span class="gene">LOX</span> (G7LIY0) ( <xref rid="tbl2" ref-type="other"> Table <xref rid="tbl2" ref-type="other"> 2 </xref> </xref> and <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.5b00617/suppl_file/pr5b00617_si_002.xlsx"> Table S2 </ext-link> ). <span class="gene">LOX</span> proteins have been identified in numerous legume nodules, <sup> <xref ref-type="bibr" rid="ref32"> 32 </xref> , <xref ref-type="bibr" rid="ref51"> 51 </xref> &#x02212; <xref ref-type="bibr" rid="ref53"> 53 </xref> </sup> and it has been suggested that they play a role in nodule development. <sup> <xref ref-type="bibr" rid="ref54"> 54 </xref> </sup> However, this is the first time <span class="gene">LOX</span> proteins have been implicated in the response of the nodule proteome to <span class="trait">drought</span> stress. LOXs catalyze the oxygenation of polyunsaturated fatty acids, <sup> <xref ref-type="bibr" rid="ref31"> 31 </xref> </sup> which can be further metabolized into volatile aldehydes and jasmonates in plants. <sup> <xref ref-type="bibr" rid="ref55"> 55 </xref> </sup> These molecules play important signaling roles in defense processes, responses to biotic and abiotic stresses, and in plant growth and development. <sup> <xref ref-type="bibr" rid="ref56"> 56 </xref> , <xref ref-type="bibr" rid="ref57"> 57 </xref> </sup> It is known that nodules express a variety of LOXs in diverse tissues indicating their possible different functions. <sup> <xref ref-type="bibr" rid="ref58"> 58 </xref> </sup> However, more research needs to be conducted to further understand the implication of <span class="gene">LOX</span> in the metabolism of legume nodules when subjected to <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The up-regulation of glycolysis- and TCA-related proteins in PDC nodules could be indicative of a higher energy demand and consumption on that side of the root system. Furthermore, it could be a response to guaranteeing the SNF and, therefore, the nitrogen supply required in the aerial part. Carbon consumption is limited in the drought-treated part of the root system, and this may favor a switch of photosynthate supply to the watered root of PD plants to enhance the PDC root system activity and, therefore, the nitrogen supply to the shoot. The active metabolism in PDC nodules could promote the growth of the root and nodules in response to the nitrogen demand of the aerial part, which can only be supplied by the PDC root system. Although not significant, the nodule dry weight for the PDC treatment was higher than in C conditions in both <italic> M. truncatula </italic> and soybeans (data not shown). The new metabolic arrangement proposed would require active plant cell growth. This suggestion is in line with the increase in PDC nodules when compared with C conditions of the relative content of diverse ribosomal proteins (G7ZUR7; G7J9M6; Contig_55945; G7JYX5; and G7J4S6), enzymes known to play important roles in protein synthesis, and a peptidyl-prolyl cis&#x02013;trans isomerase <span class="gene">(Glyma12g02790.2),</span> which is related to protein folding and is implicated in stress-response signaling and tolerance <sup> <xref ref-type="bibr" rid="ref59"> 59 </xref> &#x02212; <xref ref-type="bibr" rid="ref61"> 61 </xref> </sup> ( <xref rid="tbl4" ref-type="other"> Table <xref rid="tbl4" ref-type="other"> 4 </xref> </xref> ). Similarly, the quantity of diverse actin (G7IL85 and G7JAX5) and tubulin &#x003b2; chain proteins (G7J588) increased in PDC nodules when compared with C treatment. These components of the plant cytoskeleton are involved in several subcellular processes, including cell division, cell elongation, cell trafficking, and cell-wall formation. <sup> <xref ref-type="bibr" rid="ref62"> 62 </xref> &#x02212; <xref ref-type="bibr" rid="ref64"> 64 </xref> </sup> The increase of the relative content of the above-mentioned proteins in PDC nodules reflects the general adaptive response of the root system to the partial <span class="trait">drought</span> treatment applied to the other side of the root, pointing to an enhanced metabolism in the watered side of the PD root system aiming for cell growth.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, the <italic> M. truncatula </italic> and soybean proteomes had a high degree of similarity, and comparable responses in the diverse functional groups were identified in the SNF process that showed the same pattern of decline in both legumes, although <italic> M. truncatula </italic> seemed more tolerant to <span class="trait">drought</span> stress. In nodules directly subjected to <span class="trait">drought</span> stress, a local down-regulation of the metabolic processes was observed. Carbon, nitrogen, and sulfur metabolism were the most down-regulated processes, and SuSy, in soybean plants, and AS, in <italic> M. truncatula </italic> , were identified as being the enzymes showing the greatest relative abundance changes. Furthermore, new evidence in the regulation of SNF has come to light, such as the implication of <span class="gene">LOX</span> and protein turnover that seem to be important in the response of legume nodules to <span class="trait">drought</span> stress. Water deprivation in partial-drought conditions leads to a new metabolic arrangement in PDC nodules, increasing diverse metabolic processes involved in energy supply and cell growth. The high degree of similarity between <italic> M. truncatula </italic> and soybean proteomes suggests, in our view, that the vast amount of research conducted on <italic> M. truncatula </italic> could be extended to economically important crop legumes such as soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Understanding Water-Stress Responses in Soybean Using Hydroponics System&#x02014;A Systems Biology Perspective</h2>
			<h3>PMID:26734044</h3>
				
					<h3>PMCID:4685135</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Tripathi </surname>  <given-names> Prateek </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/55951/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rabara </surname>  <given-names> Roel C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn005">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/283767/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shulaev </surname>  <given-names> Vladimir </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Qingxi J. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/301671/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rushton </surname>  <given-names> Paul J. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn006">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/202471/overview"/>  </contrib>
				<br/> 
				年份：<day> 02 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ammonia detoxification observed in the metabolomics study suggests that nitrogen accumulates during water stress from various plausible sources (Figure <xref ref-type="fig" rid="F1"> 1C </xref> ). An increase in the level of asparagine and glutamine signifies the accumulation of nitrogen in root tissues (Oliver et al., <xref rid="B23" ref-type="bibr"> 2011 </xref> ). Interestingly, studies in <italic> Arabidopsis </italic> and <italic> Brassica </italic> during <span class="trait">drought</span> and osmotic stress have shown that an increase in proline is directly related to the effect of increasing asparagine levels (Chiang and Dandekar, <xref rid="B4" ref-type="bibr"> 1995 </xref> ). The accumulation of asparagine and glutamine facilitates the transport of nitrogen and carbon for the resumption of growth and metabolism upon rehydration in soybean (Martinelli et al., <xref rid="B19" ref-type="bibr"> 2007 </xref> ; Oliver et al., <xref rid="B23" ref-type="bibr"> 2011 </xref> ). This transport of nutrients and amino acids could also be facilitating the process of seed filling as soybean accumulates large amounts of seed storage proteins. Also, it was observed that the expression of nitrate transporter1.5 <span class="gene">(Glyma09g37220.1)</span> in the roots drastically increased at mRNA level between 2- (10-fold) and 3 h (41-fold) of dehydration. It seems plausible plant's mechanism for cost management during stress conditions over energy metabolism, whereby plants break down complex sugars and transport pyruvate and ADP to the mitochondrial matrix. It is well-understood that a decline in starch level is observed due to a decrease in the rate of photosynthesis. Thus, as a result, the ATP/ADP ratio decreases due to less demand for ATP, which leads to initiation of alternate pathways (Ribas-Carbo et al., <xref rid="B28" ref-type="bibr"> 2005 </xref> ). This study also follows the same assumption. Taken together, we hypothesize that there is a flow of nitrogen into amino acid metabolism and <italic> de novo </italic> biosynthesis of amino acids during water stress, and this constitutes a potential water stress tolerance strategy in soybean. This finding is consistent with previous studies that compared <span class="trait">drought</span> tolerant and <span class="trait">drought</span> sensitive varieties of grasses (Oliver et al., <xref rid="B23" ref-type="bibr"> 2011 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide transcriptome analysis of soybean primary root under varying water-deficit conditions</h2>
			<h3>PMID:26769043</h3>
				
					<h3>PMCID:4714440</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <address>  <email> songli@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas </given-names>  </name>  <address>  <email> princesj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maldonado dos Santos </surname>  <given-names> Joao V. </given-names>  </name>  <address>  <email> maldonadodossantosj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jiaojiao </given-names>  </name>  <address>  <email> jwqrd@mail.missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Li </given-names>  </name>  <address>  <email> Linl@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wan </surname>  <given-names> Jinrong </given-names>  </name>  <address>  <email> wanj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yongqin </given-names>  </name>  <address>  <email> yongqin.w@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Science and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Department of Computer Science, and Christopher S Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff3">  <label/> MU Informatics Institute, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label/> Department of Molecular Microbiology and Immunology, School of Medicine, University of Missouri, Columbia, MO 65212 USA  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 1 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The phenotypic response of the soybean shoot and leaf was monitored to determine the water deficit levels. The reduction of plant height was first observed under a very mild water deficit stress treatment (VMS). The height of the plants under a severe stress (SS) treatment was nearly half compared to the plant height in the well-watered treatment (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ). The level of water deficit imposed was evident from the depletion of soil moisture. Compared with the well-watered treatment (13.9&#x000a0;%), after 5&#x000a0;days of no water, the soil moisture percentage decreased under very mild stress to 9.7, mild stress to 9.1 and severe stress to 8.3. The soil moisture increased to 17.9 after water recovery (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1b </xref> ). The plant water status was also revealed by the leaf water potential (LWP) values in <span class="trait">drought</span> stressed pots (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1c </xref> ). The leaf water potential reached in this experiment reflected the severe <span class="trait">drought</span> stress treatment in the field with a LWP of &#x02212;2.5&#x000a0;MPa (megapascals). During the stress recovery treatment, the severely stressed pots were re-watered for two days to recover, reaching a water potential of &#x02212;0.65&#x000a0;MPa. In addition to the general decrease in growth caused by the water deficit, several physiological indicators were monitored under different water deficit stages. The plants under the <span class="trait">drought</span> stress treatment showed declined stomatal conductance values up to 80&#x02013;90&#x000a0;% in the <span class="gene">MS</span> and <span class="gene">SS</span> stress treatments, compared to the VMS treatment (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1d </xref> ). This in turn affected the canopy temperature (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1e </xref> ). The leaf relative water content (RWC) was also found to decrease with the varying water deficit levels from 85.2&#x000a0;% (VMS) to 68.6&#x000a0;% (SS) (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1f </xref> ). The effect of different stress levels is evident in the soybean plants with varying physiological responses. In the water recovery treatment, the leaf water potential and stomatal conductance values of the plants were the same as observed in the VMS treatment. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> Soybean (Williams 82 genotype) phenotypic responses to different water-deficit treatments. Soybean plant height ( <bold> a </bold> ), soil moisture ( <bold> b </bold> ), leaf water potential ( <bold> c </bold> ), leaf stomatal conductance ( <bold> d </bold> ), leaf canopy temperature ( <bold> e </bold> ), and leaf relative water content ( <bold> f </bold> ) were monitored to indicate the water-deficit levels. CK (well-watered control), VMS (Very Mild Stress), <span class="gene">MS</span> (Mild Stress), <span class="gene">SS</span> (Severe Stress), SR (water Recovery after severe stress). Bars indicate standard error of the mean of nine plants
					</p>
				
					<p>&nbsp;&nbsp;
						Cluster analysis of the 6,609 genes with dChiP software indicated that the regulated genes under <span class="gene">SS</span> showed higher similarity with <span class="gene">MS</span> and VMS exhibited a higher correlation with SR (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3a </xref> ). The clustered genes were further divided into seven subgroups to identify their connection with different pathways (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3b </xref> ). The genes in Group 2 showed up-regulation in VMS, down-regulation in <span class="gene">MS</span> and SS, and the expression pattern in SR was very similar to that of VMS. However, the genes in Group 6 showed the opposite regulation pattern (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3b </xref> ). <fig id="Fig3"> <label> Fig. 3 </label> <caption> <p> Expression profile of differentially expressed genes (DEGs) clustering. <bold> a </bold> Hierarchical clustering display of the up- and down-regulated genes in the root indicated the gene expression patterns of severe stress were more similar to mild <span class="trait">drought</span> stress. Very mild stress and recovery showed a correlation. <bold> b </bold> Root gene expression profile clustering showed seven groups with normalization values between 2 and -2. Standardized expression levels are shown on y-axis and stress level on x-axis. Each group is represented by the gene expression profiles under different <span class="trait">drought</span> levels. Group1: genes up-regulated under SR treatment, but down-regulated under <span class="gene">MS</span> treatment, constantly under VMS and <span class="gene">SS</span> treatments; Group2: genes up-regulated under VMS and SR treatments, but down-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments; Group 3: genes up-regulated under VMS treatment, but downregulated under <span class="gene">SS</span> treatment, constantly under <span class="gene">MS</span> and SR treatments; Group4: genes up-regulated under <span class="gene">MS</span> treatment, but down-regulated under VMS and <span class="gene">SS</span> treatments, constantly under SR treatment. Group 5: genes up-regulated under <span class="gene">SS</span> and SR treatments, but down-regulated under VMS and <span class="gene">MS</span> treatments; Group6: genes up-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments, but down-regulated under VMS and SR treatments. Group 7: genes up-regulated under VMS treatment, but down-regulated under SR treatments, constantly under <span class="gene">MS</span> and <span class="gene">SS</span> treatments
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple metabolic related pathways that respond to water-deficit treatments have been identified. Transcripts associated with cell wall modification, such as expansins, xyloglucan endotransglucosylase/hydrolases (XTHs), and pectin methylesterases (PMEs), were identified in the root tissues. The transcripts within these three gene families were down-regulated during <span class="gene">MS</span> and <span class="gene">SS</span> treatments. Several gene families (mannan-xylose-arabinose-fucose, pectate lyases, and polygalacturonases) involved in cell wall degradation were mainly down-regulated (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4a </xref> and <xref rid="Fig4" ref-type="fig"> b </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). Most of the differentially expressed genes involved in lipid metabolism (including fatty acid (FA) synthesis, elongation, desaturation, and degradation) were inhibited under <span class="gene">MS</span> and SS, which could limit lipid biosynthesis. None of the amino acid metabolic pathway genes was found to be differentially expressed in roots, but some related genes were found to be down-regulated in shoot tissues under <span class="trait">drought</span> stress (data not shown). Several gene families involved in the secondary metabolism pathways were found to be down-regulated under the <span class="gene">MS</span> and <span class="gene">SS</span> treatments, but most of them returned to normal level after water re-application. These include flavonoids, isoprenoids, lignin, fermentation, and glycolysis related genes (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4a </xref> and <xref rid="Fig4" ref-type="fig"> b </xref> and Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> ). <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> MAPMAN visualization of metabolism-related gene expression in water-deficit stressed soybean roots compared with control. The log2 fold changes of significantly DEGs were imported and visualized in MapMan for the root samples under <span class="gene">MS</span> condition ( <bold> a </bold> ) and <span class="gene">SS</span> condition ( <bold> b </bold> ). Genes were assigned to their associated metabolic pathways. Each square corresponds to a gene. Red squares indicate genes suppressed in comparison to isolated or untreated plants. Green squares indicate genes induced. MapMan version 3.6.0 was used to generate these images from gene expression data. The scale bar is shown in log2 from -4.5 to 4.5
					</p>
				
					<p>&nbsp;&nbsp;
						Many genes encoding sucrose invertases that are present in the cell wall and vacuoles were decreased at a transcript level and the genes associated with trehalose synthesis were also down-regulated. Moreover, most of the transcripts showed a significant reduction in the <span class="gene">SS</span> treatment, which sends the oxidative stress signal to the plant. Conversely, galactinol and raffinose synthase related genes were up-regulated, which might act as an energy source to maintain root growth in <span class="trait">drought</span> stress (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a </xref> , Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). No sucrose biosynthesis genes were found to change in the primary root during <span class="trait">drought</span> stress. Several starch <span class="trait">branching</span> and synthase enzyme related genes were slightly down-regulated in VMS, but were increased under <span class="gene">MS</span> and <span class="gene">SS</span> treatments. At the same time, the enzymes involved in starch degradation (starch cleavage) were also down-regulated (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5b </xref> , Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Overall, there was an increase in transcripts associated with starch synthesis and a decrease in transcripts associated with starch degradation in <span class="gene">MS</span> and <span class="gene">SS</span> treatments. These results indicated that water-deficit induced transcripts change with carbohydrate metabolism, which in turn regulates sucrose and starch content. The increased content in galactinol and raffinose will enhance osmotic tolerance in the root under <span class="trait">drought</span> stress. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> The regulations of carbohydrate metabolism in soybean roots under water-deficit stress. The transcripts associated with starch synthesis increased ( <bold> a </bold> ) and the transcripts associated with starch decreased ( <bold> b </bold> ) in <span class="gene">MS</span> and <span class="gene">SS</span> treatments. Red arrow: up-regulated; Green arrow: down-regulated. Blue arrow: no changes
					</p>
				
					<p>&nbsp;&nbsp;
						It has been reported that the effect of water-deficiency on ethylene synthesis was linked with the plant&#x02019;s rate of stressed [ <xref ref-type="bibr" rid="CR21"> 21 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. Rapid induction of water-deficit stress induces ethylene production and a slower imposition of water-deficit stress inhibits ethylene biosynthesis [ <xref ref-type="bibr" rid="CR21"> 21 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. In our results, 102 transcripts associated with the ethylene pathway were regulated, and most of the ethylene signal transduction and synthesis related genes were down-regulated in <span class="gene">MS</span> and <span class="gene">SS</span> treatments, but resumed at the recovery phase. They were mainly grouped into subgroup1, subgroup2 and subgroup3 (genes up-regulated or constantly under VMS and SR, down-regulated or constantly under <span class="gene">MS</span> and SS, Additional file <xref rid="MOESM5" ref-type="media"> 5 </xref> ). The ACC oxidases, ACO1 and ACO4, which catalyze the last step of the ethylene biosynthesis pathway, were found to be inhibited by water-deficit stress. Several ethylene response factors (ERFs) and ACC synthase (ACS) genes in ethylene pathway were also found to be down-regulated (Additional file <xref rid="MOESM5" ref-type="media"> 5 </xref> ). These results suggested the stress treatment in this study was a slow progression of <span class="trait">drought</span> induction. Ethylene synthesis and signal transduction played an important role in the soybean primary root growth under the water stress treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes potentially involved in other hormonal pathways were also identified in the present study: abscisic acid (ABA, 22 genes), gibberellin (GA, 25 genes), jasmonic acid (JA, nine genes), salicylic acid (SA, nine genes) and cytokinin (16 genes) (Additional files <xref rid="MOESM6" ref-type="media"> 6 </xref> and <xref rid="MOESM7" ref-type="media"> 7 </xref> ). No brassinosteroid related genes were found. Although ABA is the major hormone known to be involved in <span class="trait">drought</span> stress response, there were only a few ABA related genes that changed in expression level under water deficient treatments. Among them, the transcripts of ABA synthesis genes, <italic> NCED </italic> (9-cis-epoxycarotenoid dioxygenase) and <italic> <span class="gene">CCD8</span> </italic> (carotenoid cleavage dioxygenases), increased with respect to the level of <span class="trait">drought</span> imposed. This trend was found to be reversed after water recovery. Several genes involved in the crosstalk between the two hormones were also found here. Two GA synthesis related genes, <italic> <span class="gene">GA2OX1</span> </italic> and <italic> <span class="gene">GA2OX2</span> </italic> (gibberellin 2-oxidase), were found to be down-regulated, which were also involved in the ethylene pathway. <span class="gene">CCD7</span> and <span class="gene">CCD8</span> (carotenoid cleavage dioxygenases), which function in root <span class="trait">branching</span> by involving ABA and the strigolactones synthesis pathway, were shown to change expression patterns under water-deficit treatments. Another gene, <italic> <span class="gene">GmJAR1</span> </italic> (JASMONATE RESISTANT 1), which is involved in both the auxin and JA pathways, was also inhibited during the <span class="trait">drought</span> stress treatment and recovered after re-watering. All of the above results indicated that the ethylene and auxin synthesis and signal transduction play a more important role than other hormones in the soybean taproot response to water deficient stress. On the other hand, hormone crosstalk probably plays a significant role in soybean root responses to <span class="trait">drought.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The TFs specifically regulated in both <span class="gene">MS</span> and <span class="gene">SS</span> treatment were selected for further gene family analysis (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Genes belonging to the bHLH (basic helix-loop-helix), <span class="gene">AP2</span> (activating Protein 2), MYB (myeloblastosis) and WRKY (is a class of DNA-binding proteins that recognize the TTGAC(C/T) W-box elements) families represent most of the differentially expressed TFs. MYB represents the highest number of transcripts under <span class="trait">drought</span> followed by WRKY and bHLH. Six TFs were found to change under every stress level and they probably play the conserved or fundamental roles in taproot responses to stress. One of them, a heat shock related TF <span class="gene">(Glyma04g05500),</span> was up-regulated in all stress levels. Its homologue was found to regulate the heat stress response in Arabidopsis [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Another ERF (Ethylene response factor, containing the <span class="gene">AP2</span> domain) TF was down-regulated. Two <span class="gene">MYB15</span> TFs and one U-box TF were found to be up-regulated under VMS and SR treatments and were down-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments. In Arabidopsis, the AtMYB15 gene was found to be involved in the ABA, ethylene, and jasmonic acid mediated signaling pathways. Overexpression of AtMYB15 showed an improved survival rate and reduced water loss under water deficit treatments [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						First, to specifically uncover the regulatory mechanisms of <span class="trait">drought</span> responses in root tissue, all of the TFs in the DEGs were identified from the leaf and shoot tissues. As shown in Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7a </xref> , the maximum number (603 TFs) of TFs were found in roots, with more than half of them (372 TFs) being root specific. A fewer number of TFs (229 TFs) encoding transcripts were found in the shoot in comparison to root and leaf transcriptomes. In addition, more than half of the TFs in the shoot were shared in the root and leaf as well. Sixty-five TFs were found in all three tissues (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> and Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7a </xref> ). These results indicated that the gene regulation network between root and leaf plays more important roles under water-deficit treatments. Second, based on stress level specificity from the whole plant expression pattern, the TFs identified in the present study were classified. Minimum numbers (69 TFs) of stress level specific TFs were found in <span class="gene">MS</span> treatment, which also indicated that the mild stress level share more common TFs among other stress levels. Only a few specific TFs involved in the VMS stress were found in the root, but more were found at the whole plant level. This indicates more TFs were regulated in the shoot and leaf as an early <span class="trait">drought</span> stress response. Nearly half of the TFs regulated by <span class="gene">SS</span> or SR treatment were not adjusted by other stress level. Irrespective of the tissues used, 27 TFs found were involved in all of the stress treatments (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> and Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7b </xref> ). Finally, 13 TFs were found to be regulated in all three tissues under all stress levels. Among them, five <span class="gene">AP2</span> TFs were found to be highly induced under <span class="gene">MS</span> and <span class="gene">SS</span> treatments in the root and SR in the leaf tissue, and they were down-regulated in <span class="gene">MS</span> and <span class="gene">SS</span> treatments in the shoot tissues. UBIQUITINATION FACTOR E4 <span class="gene">(Glyma.03G202600)</span> and bHLH <span class="gene">(Glyma.15G170500)</span> TFs were decreased in number under <span class="gene">MS</span> and <span class="gene">SS</span> in all tissues studied. These results indicated that these common TFs are likely to play a key role in regulating the whole plant response under varying levels of <span class="trait">drought</span> stress, but might also be involved spatio-temporal regulation among different tissues (Additional file <xref rid="MOESM10" ref-type="media"> 10 </xref> ). <fig id="Fig7"> <label> Fig. 7 </label> <caption> <p> Comparative analysis with transcription factors (TFs) revealed the number of overlapping TFs from those three tissues or water-deficit treatment levels. <bold> a </bold> Venn diagram showing the TF number of DEGs under all water-deficit treatments in the different tissues (root, leaf, and shoot). <bold> b </bold> Venn diagram showing the transcription factor number of DEGs under different water-deficit treatments in three tissues (root, leaf, and shoot). (Blue: VMS, Pink: MS, Green: SR, Purple: SS)
					</p>
				
					<p>&nbsp;&nbsp;
						In this analysis, two motifs (AAAAAAAA and TATATATA) over-represented in each group were identified. The second motif was normally found approximately 25&#x02013;35 base pairs upstream of the start site. Several known motifs involved in the known <span class="trait">drought</span> response pathway or BR pathway were also identified, such as CRTGCAY (ABI3), GCCRCS (ERF1), CACRC and ACGHGK (BES1), CTCYCYC (ABI4), and <span class="gene">GMCACGB</span> (ABF1, BZR1, and BES1). This result indicated that those enriched motifs were probably involved in the <span class="trait">drought</span> regulations network induced at different stress levels.
					</p>
				
					<p>&nbsp;&nbsp;
						Because groups 2 and 6 (group2: genes up-regulated under VMS and SR treatments, but down-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments; group6: genes up-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments, but down-regulated under VMS and SR treatments;) stand for oppositely regulated mechanisms in response to <span class="trait">drought</span> in the taproot tissues, motif analysis on genes from these two groups in the root tissues were further assessed. Fifty-seven over-represented motifs were identified in the <span class="gene">MS</span> and <span class="gene">SS</span> up-regulated <span class="trait">drought</span> response genes group (Group 2) and 49 over-represented motifs were isolated in the <span class="gene">MS</span> and <span class="gene">SS</span> down-regulated <span class="trait">drought</span> response genes group (Group 6). Only one motif (CACGHG) overlapping in these two groups. The CACGHG motif was a &#x0201c;G-box&#x0201d;, which is the binding site of the GBF4 TF in Arabidopsis. It was also conserved in the PIF, BES ((BRI1-EMS-SUPPRESSOR 1), and MYC TF target element. Enriched sequences regulated by ARR10, ANT, and Dof3 TFs were found only in Group 2 and not in Group 6. The ARR10 TF is involve in the cytokinin signaling pathway and often shows a down-regulated response to abiotic stress [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. <italic> ANT </italic> encodes an AP2-like ethylene-responsive TF and its function is most closely related to floral development [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. It has been reported that the expression pattern of the DOF TF responds to abiotic stress in several species [ <xref ref-type="bibr" rid="CR29"> 29 </xref> , <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. Two enriched cis-elements (CWCCACS and CAYSCAC) were found in Group 6, which were regulated by TF GAMYB. It has been reported that <span class="gene">GmGBP1,</span> which is one soybean GAMYB binding protein, regulates flowering and improves <span class="trait">drought</span> tolerance in Arabidopsis [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. These results indicated that different enriched motifs were involved in very specific regulation mechanism responses to different <span class="trait">drought</span> levels.
					</p>
				
					<p>&nbsp;&nbsp;
						The 18 selected genes for this comparison included nine genes associated with metabolism and nine genes encoding TFs. As shown in Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ( <xref rid="Fig8" ref-type="fig"> a&#x02013;i </xref> ), two cell wall-related genes (xyloglucan endo-transglycosylase) and two carbohydrate metabolism related genes (glycosyl transferase and raffinose synthase) showed up-regulation under <span class="gene">MS</span> and <span class="gene">SS</span> treatments. In addition, five genes (auxin responsive protein, hydrolase activity, arylacetamide deacetylase, rare lipoprotein A, and polygalacturonase) were significantly down-regulated under <span class="gene">MS</span> and <span class="gene">SS</span> treatments, and the gene expression never returned back to normal treatments at VMS and water recovery phases. Among the TFs selected, bHLH TFs showed a reduced expression pattern in all stress levels with increased transcripts during the recovery phase (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8j </xref> ). However, the expression pattern of NY-YA was highly induced at all stress levels (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8l </xref> ). All other transcription factors were significantly induced only under the <span class="gene">MS</span> and <span class="gene">SS</span> treatments in the RNA-Seq results (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8k&#x02013;r </xref> ). <fig id="Fig8"> <label> Fig. 8 </label> <caption> <p> qRT-PCR analysis of genes identified to be water-deficit responsive by RNA-seq. Nine selected metabolism ( <bold> a-i </bold> ) and nine transcription factor ( <bold> j-r </bold> ) related DEGs with different <span class="trait">drought</span> stress treatments compared with the RNA-Seq data. Gray and black bars for RNA-Seq and qRT-PCR results, respectively. Error bars represent the Standard error for three independent experiments and two technical replicates
					</p>
				
					<p>&nbsp;&nbsp;
						It has been known that a variety of osmolytes which includes sugars (such as raffinose, trehalose, sucrose and sorbitol), sugar alcohols (such as mannitol), amino acids (such as proline), and secondary metabolites (such as isoflavonoids and saponins) will accumulate in the plants when it responses to the abiotic stress, [ <xref ref-type="bibr" rid="CR33"> 33 </xref> &#x02013; <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. These metabolism products are able to stabilize proteins and cellular structures to maintain cell turgor by osmotic adjustment. Redox metabolism removes excess levels of ROS (Reactive oxygen species) and re-establishes the cellular redox balance [ <xref ref-type="bibr" rid="CR36"> 36 </xref> , <xref ref-type="bibr" rid="CR37"> 37 </xref> ]. The type of metabolism and related genes involved in the adjustment of soybean roots under water deficit conditions are still unclear. Galactinol and raffinose act as osmoprotectants and provide an adaptation to the water stress conditions [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Arabidopsis plants over-expressing GolS1 and GolS2 (galactinol synthase) accumulated high levels of galactinol and raffinose and were tolerant to <span class="trait">drought</span> and salinity stress [ <xref ref-type="bibr" rid="CR38"> 38 </xref> , <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Several GOLS1 and GOLS2 genes that were up-regulated under the VMS, MS, and <span class="gene">SS</span> conditions were inhibited at the water-recovery phase in the present study. These candidate genes will be very promising and can be used for soybean transgenic development to improve soybean tolerance to water deficit conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Starch serves as a primary carbohydrate source and can be mobilized to soluble sugars if the plants experience an environmental challenge. It is reported that the starch content will be depleted by <span class="trait">drought</span> and the soluble sugars will accumulate in the leaves by enhanced beta-amylase enzyme activity [ <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Here, we found the reverse adjustment for the starch metabolism in the soybean root, which indicates that metabolites involved with energy production and growth were increased or shifted from shoots to roots. The total metabolism is accelerated to adapt to the water-deficit environment when water is a limiting factor. It is reported that the shoots and roots respond to <span class="trait">drought</span> in an opposite metabolic way, while shoots decrease their growth metabolism (lower concentrations of sugars, amino acids, nucleosides, nitrogen, phosphorus and potassium) and roots increase metabolism in a mirrored response [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. BMY8 (&#x003b2;-amylase) antisense Arabidopsis plants accumulated high starch levels during cold shock (4&#x000a0;&#x000b0;C for 6&#x000a0;h) [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Three BMY8 genes were also found down-regulated under the <span class="gene">MS</span> and <span class="gene">SS</span> conditions in the soybean root, which indicates the <span class="gene">GmBMY8</span> gene probably plays a role under <span class="trait">drought</span> stress by modulating starch metabolism.
					</p>
				
					<p>&nbsp;&nbsp;
						As important membrane components, the changes in the composition of lipids may help to maintain membrane integrity and preserve cell compartmentation under water stress conditions. In response to <span class="trait">drought,</span> the total leaf lipid content decreased progressively. The time-course of the decrease in Arabidopsis leaf lipid content correlated well with the expression of genes involved in lipid degradation [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. Most of the lipid biosynthesis related genes were down-regulated in the root in this study. In transgenic Arabidopsis with suppression of the inositol polyphosphate 1-phosphatase (IPP) signal by heterologous expression of the human type I inositol polyphosphate 5-phosphatase exhibited an increased <span class="trait">drought</span> tolerance phenotype, which was mediated in part via a DREB2A-dependent pathway [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. The content and composition may change when plants are exposed to different abiotic and biotic stresses [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. Little is known about the effects of water-deficit on lignin synthesis in roots until now. Research has shown that there was an increased expression of genes involved in lignin biosynthesis during the intermediate and final stages of water stress (from 48&#x000a0;h to 72&#x000a0;h) in rice roots ( <italic> Oryza sativa </italic> L.), such as those coding for PAL, C3H, 4-coumarate: coenzyme A ligase (4CL), caffeoyl coenzyme A O-methyltransferase (CCoAOMT), cin-namyl alcohol dehydrogenase (CAD), and peroxidase [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. Similarly, lignin synthesis-related proteins were induced mostly in the later stage of <span class="trait">drought</span> stress in the roots of wild watermelon ( <italic> Citrullus lanatus </italic> sp.), which may function in the enhancement of physical desiccation tolerance and <span class="trait">drought</span> adaptation [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. These results indicated that growth reduction and tolerance to water-deficit were associated with more lignin production in the roots. However, most of the lignin biosynthesis related genes in the soybean root showed a down-regulated pattern under the <span class="gene">MS</span> and <span class="gene">SS</span> conditions. The reason for this phenomenon may be caused by different regions of root and stress levels or degrees.
					</p>
				
					<p>&nbsp;&nbsp;
						Three hormones,auxin, ethylene and ABA, were identified to be involved in <span class="trait">drought</span> responsive pathways in soybean [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. In this study, ethylene and auxin synthesis and signal transduction had more important roles than other hormones in the soybean taproot response to water deficient stress. At present, hormone interactions involved in abiotic stress were mainly derived from <italic> Arabidopsis thaliana </italic> . Until now, it was unknown how the hormone crosstalk regulated primary root elongation under water-deficit stress in soybean roots. Several genes (i.e., <italic> <span class="gene">GA2OX1</span> </italic> , <italic> <span class="gene">GA2OX2</span> </italic> , <italic> <span class="gene">ACS</span> </italic> , <italic> <span class="gene">CCD7</span> </italic> , <italic> <span class="gene">CCD8</span> </italic> , and <italic> JAR1 </italic> ) involved in different hormone crosstalk and related TFs were found in this study (Additional files <xref rid="MOESM4" ref-type="media"> 4 </xref> , <xref rid="MOESM5" ref-type="media"> 5 </xref> , <xref rid="MOESM6" ref-type="media"> 6 </xref> and <xref rid="MOESM7" ref-type="media"> 7 </xref> ), which indicated that the interactions between different hormones play a crucial role in soybean root development.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of genes associated with the auxin transporter has been shown to be regulated by ethylene [ <xref ref-type="bibr" rid="CR53"> 53 </xref> ]. Conversely, auxin was found to affect the synthesis of ethylene [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ]. Here, the expression of <italic> PIN2 </italic> and <italic> <span class="gene">PIN5</span> </italic> were found to be regulated by the water-deficit stress, and several <italic> <span class="gene">ACS</span> </italic> genes ( <italic> <span class="gene">ACS1</span> </italic> , <italic> ACS6 </italic> , <italic> ACS7 </italic> , <italic> ACS8 </italic> , and <italic> ACS10 </italic> ) were regulated under <span class="gene">MS</span> and <span class="gene">SS</span> conditions. <italic> <span class="gene">CCD7</span> </italic> and <italic> <span class="gene">CCD8</span> </italic> genes were involved in the ABA and strigolactones (SL) synthesis pathway. SL, as plant hormone, functions as a positive regulator of primary root elongation and as a negative regulator of adventitious root formation due to modulation of auxin flux in the root [ <xref ref-type="bibr" rid="CR55"> 55 </xref> &#x02013; <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. Wang et al. reported that ABA treatment increased soybean <italic> <span class="gene">CCD7</span> </italic> and <italic> <span class="gene">CCD8</span> </italic> transcription [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. Based on this study, hormone balance between SL and ABA might play an important role in primary root development under <span class="gene">MS</span> and <span class="gene">SS</span> treatments. Under <span class="trait">drought</span> stress, higher levels of ABA may up-regulate the activities of downstream TFs (MYBs), which would modify the auxin biosynthesis and trigger auxin responsive cascades, controlling root development. The increase in ABA content and signal transduction promotes the synthesis of ethylene, then inhibits the elongation of root. This complex network of hormone crosstalk allows cell differentiation to balance cell division and the formation of the root system architecture.
					</p>
				
					<p>&nbsp;&nbsp;
						TFs DREB1A and DREB1D were found to be highly up-regulated by <span class="gene">MS</span> and <span class="gene">SS</span> conditions in the root. In transgenic Arabidopsis, overexpression of the DREB1/CBF family genes showed increasing tolerance to freezing, <span class="trait">drought,</span> and high salinity [ <xref ref-type="bibr" rid="CR62"> 62 </xref> &#x02013; <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. Overexpression of DREB/CBF TFs has been reported to enhance <span class="trait">drought</span> tolerance in soybeans [ <xref ref-type="bibr" rid="CR65"> 65 </xref> , <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. <span class="trait">Drought</span> is often accompanied by elevated air and leaf temperatures in the field. Heat shock TF (HSF) genes have been implicated not only in thermotolerance but also in plant growth and development and response to various stresses [ <xref ref-type="bibr" rid="CR67"> 67 </xref> ]. Six <span class="gene">HSF</span> transcription factors were found highly induced under the <span class="gene">MS</span> and <span class="gene">SS</span> conditions in soybean roots. After water recovery, the expression level decreased compared with <span class="gene">SS</span> treatment (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Several auxin, salt, and cold pathway related TFs were found in the DEG group in the root tissue. <span class="gene">ICE1</span> encodes an upstream bHLH TF that regulates the transcription of <span class="gene">CBF</span> genes in the cold [ <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. One soybean ICE TF <span class="gene">(Glyma06g17330.1)</span> was induced by the <span class="gene">MS</span> and <span class="gene">SS</span> treatments. Two auxin response TFs were affected; IAA26 <span class="gene">(Glyma07g01800.1)</span> was up-regulated and IAA19 <span class="gene">(Glyma13g43780.1)</span> was down-regulated by the stress. The induced IAA 26 in rice leaves could dynamically regulate the phytohormone IAA level to respond to <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="CR69"> 69 </xref> ]. Two PIF3 (phytochrome-interacting factor) genes <span class="gene">(Glyma19g40980.1</span> and <span class="gene">Glyma10g28290.1)</span> were induced by the <span class="gene">MS</span> and <span class="gene">SS</span> treatments in the root. It was reported that the rice PIF-like protein Os-PIL1/PIL13 was a key regulator of reduced internode elongation in rice under <span class="trait">drought</span> conditions [ <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. Overexpressing the STO (salt tolerance) gene in Arabidopsis conferred a higher salt tolerance phenotype than the wildtype through binding to a MYB TF [ <xref ref-type="bibr" rid="CR71"> 71 </xref> ]. One soybean STO <span class="gene">(Glyma11g13570.1)</span> gene was found induced under the <span class="gene">MS</span> and <span class="gene">SS</span> conditions, but was down-regulated after water recovery. Two GA pathway TFs; SCARECROW (SCR) and RGA-LIKE 2 della protein (RGL2), were also found to be down-regulated in all levels of <span class="trait">drought</span> stress. SCR is known to be a key regulator of stem cell renewal and radial patterning in the root of Arabidopsis [ <xref ref-type="bibr" rid="CR72"> 72 </xref> , <xref ref-type="bibr" rid="CR73"> 73 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Williams 82 soybean seeds were planted in 5-gallon pots filled with soil media (turface and sand mixed in 2:1 ratio) in the growth chamber at the Sears Plant Growth Facility at the University of Missouri. The controlled conditions of the growth chamber was 27/21&#x000a0;&#x000b0;C&#x000a0;day/night temperature, photoperiod of 16/8&#x000a0;day/night, 60&#x000a0;% relative humidity. Three seeds were sown per pot and thinned to one plant per pot when the plants had two sets of unfolded trifoliate leaves (Vegetative (V) 2 stage). All pots were kept well-watered until <span class="gene">V3</span> (three unfolded trifoliate leaves) stage [ <xref ref-type="bibr" rid="CR74"> 74 </xref> ]. For very mild stress (VMS), <span class="trait">drought</span> stress was imposed by withholding water for 5&#x000a0;days. For mild stress (MS), <span class="trait">drought</span> stress was imposed by withholding water for 12&#x000a0;days. For severe stress (SS), <span class="trait">drought</span> stress was imposed by withholding water for 19&#x000a0;days. For water recovery after severe stress (SR), plants were re-watered for 2&#x000a0;days after withholding water for 19&#x000a0;days. Each treatment had corresponding control plants and all control plants were kept well-watered until sampling. The soil moisture of each pot was measured on a daily basis to monitor the treatment condition by using the PR2 soil moisture profile probe (Delta-T Devices Ltd., UK). The value recorded at the sampling day was used for Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1b </xref> . Plant height, water potential, stomatal conductance, canopy temperature, and leaf relative water content (RWC) were recorded or determined at the sampling day in all control and stressed plants. These physiological traits were collected from an attached leaflet of the fourth trifoliate leaf from the main-stem apex at midday. Full leaves were used in the determination of RWC. Water potential was measured using a pressure chamber (PMS Instrument Co., Albany, CA, USA). A LI-COR 6400XT portable photosynthesis system (LI-COR, Lincoln, NE) was used to measure net stomatal conductance. Canopy temperatures were collected with an infrared meter (FLIP Ex severe infra-red meter, Everett, WA). RWC was determined by the ratio of tissue fresh weight to tissue turgid weight. This is termed relative tissue weight.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution of an intron-poor cluster of the CIPK gene family and expression in response to <span class="trait">drought</span> stress in soybean</h2>
			<h3>PMID:27311690</h3>
				
					<h3>PMCID:4911590</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Kaikai </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Fei </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jinyi </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xinlu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hewezi </surname>  <given-names> Tarek </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Zong-Ming (Max) </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Plant Sciences, University of Tennessee </institution> , Knoxville, Tennessee 37996,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 01 </day>  <month> 06 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To investigate gene expression change of the soybean CIPK family members under <span class="trait">drought,</span> we analyzed gene expression profiles of individual <italic> CIPK </italic> genes using publicly available Affymetrix microarray datasets. GSE29663 and GSE40604 datasets provide gene expression profiling of soybean leaves at early and late developmental stages under <span class="trait">drought</span> stress, respectively <xref ref-type="bibr" rid="b9"> 9 </xref> . Gene expression data were available only for 48 <italic> CIPK </italic> genes, which have probes in this microarray platform ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ). Twenty genes were found to be up-regulated and 28 were down-regulated in the leaves during the vegetative growth stage in response to <span class="trait">drought.</span> During the reproductive stage, 33 were found to be up-regulated and 15 down-regulated, in the leaves in response to <span class="trait">drought</span> stress. Eighteen genes were up-regulated in both developmental stages under <span class="trait">drought,</span> including 3 in subgroup II ( <italic> <span class="gene">GmCIPK8,</span> 20 </italic> and <italic> &#x02212;41 </italic> ), 2 in subgroup III ( <italic> <span class="gene">GmCIPK29</span> </italic> and <italic> &#x02212;42 </italic> ), and 13 in subgroup IV ( <italic> <span class="gene">GmCIPK4,</span> &#x02212;7, &#x02212;9, &#x02212;12, &#x02212;24, &#x02212;27, &#x02212;28, &#x02212;31, &#x02212;32, &#x02212;38, &#x02212;47, &#x02212;49 </italic> and &#x02212; <italic> 51 </italic> ). In contrast, 13 genes showed constant down-regulation during both developmental stages. These genes included 3 in subgroup I ( <italic> <span class="gene">GmCIPK10,</span> 30 </italic> and <italic> &#x02212;45 </italic> ), 1 in subgroup II ( <italic> <span class="gene">GmCIPK15</span> </italic> ), 1 in subgroup III ( <italic> <span class="gene">GmCIPK43</span> </italic> ), and 8 in subgroup IV ( <italic> <span class="gene">GmCIPK2,</span> &#x02212;11, &#x02212;14, &#x02212;17, &#x02212;22, &#x02212;33, &#x02212;37 </italic> and &#x02212; <italic> 46 </italic> ). The remaining 17 genes showed opposite expression patterns at both developmental stages ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the microarray data ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ), we selected 18 candidate genes ( <italic> <span class="gene">GmCIPK2,</span> &#x02212;4, &#x02212;8, &#x02212;9, &#x02212;11, &#x02212;12, &#x02212;14, &#x02212;20, &#x02212;22, &#x02212;24, &#x02212;28, &#x02212;30, &#x02212;31, &#x02212;33, &#x02212;38, &#x02212;41, &#x02212;47, </italic> and <italic> &#x02212;49 </italic> ) for further confirmation using qRT-PCR in leaves, stems and roots at 4-, 8-, and 12-days after imposing <span class="trait">drought</span> stress ( <xref ref-type="supplementary-material" rid="S1"> Figure S2 </xref> ). The qRT-PCR results showed that all these 18 selected <italic> CIPK </italic> genes are drought-responsive and expressed differently in three different tissues under <span class="trait">drought</span> treatment ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ). In leaf samples, <italic> <span class="gene">GmCIPK2,</span> &#x02212;14, &#x02212;31 </italic> and <italic> &#x02212;33 </italic> genes were down-regulated in the entire period tested under <span class="trait">drought</span> stress. However, <italic> <span class="gene">GmCIPK9,</span> &#x02212;12, &#x02212;20, &#x02212;24, &#x02212;38, </italic> and <italic> &#x02212;49 </italic> were up-regulated during all the three time points. Notably, <italic> <span class="gene">GmCIPK49</span> </italic> was induced gradually and showed the highest gene expression (&#x0003e;60-fold) after 12 days of <span class="trait">drought</span> application. Seven genes, including <italic> <span class="gene">GmCIPK9,</span> &#x02212;12, &#x02212;20, &#x02212;24, &#x02212;38, &#x02212;41, </italic> and <italic> &#x02212;49 </italic> , were highly up-regulated at 8-day and 12-day time points. In the stem samples, <italic> <span class="gene">GmCIPK4,</span> &#x02212;8, &#x02212;9, &#x02212;24, &#x02212;38, </italic> and &#x02212; <italic> 49 </italic> all showed higher gene expression levels (&#x0003e;20-fold) at 12-day than the untreated control. With the exception of <italic> <span class="gene">GmCIPK14,</span> </italic> which showed down-regulation, the remaining 17 genes exhibited up-regulation at the 3 time points tested. In root tissues, <italic> <span class="gene">GmCIPK9</span> </italic> showed the highest gene expression level at 12-day point, or 120-fold of that in the control. Most of these 18 genes were up-regulated at 8-day and 12-day <span class="trait">drought</span> treatment except <italic> <span class="gene">GmCIPK11,</span> &#x02212;14, &#x02212;20 &#x02212;30 </italic> and &#x02212; <italic> 33 </italic> . While our qRT-PCR data are overall consistent with the microarray results ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ), a few genes displayed distinct expression patterns. This includes, for example, <italic> <span class="gene">GmCIPK31</span> </italic> , which was down-regulated in qRT-PCR assays of leave samples, but was up-regulated in the two microarray experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the up-regulated <italic> <span class="gene">GmCIPK</span> </italic> genes, we identified <italic> CIPK9, &#x02212;12, &#x02212;24, </italic> and &#x02212; <italic> 49 </italic> as the most highly expressed genes under <span class="trait">drought</span> stress in the three tissues tested. These genes represent bona find targets for improving soybean tolerance to <span class="trait">drought</span> and deserve further analysis to reveal their functional roles in <span class="trait">drought</span> response and the underlying molecular mechanisms, which is currently underway.
					</p>
				
					<p>&nbsp;&nbsp;
						Fifteen-day-old plants were subjected to <span class="trait">drought</span> stress by withholding water for 0 (control), 4, 8 and 12 days. Leaf, stem and root samples were collected at these four time points for RNA extraction and qRT-PCR quantification of the expression levels of 14 intron-poor genes (top panels) and 4 intron-rich genes (bottom panels). <italic> Ribosomal protein s20e </italic> gene ( <italic> <span class="gene">Glyma.03G142300</span> </italic> ) was used as an internal control to normalize gene expression levels. The fold-change values represent changes of mRNA levels in <span class="trait">drought</span> stressed samples relative to non-stressed control samples. Data are average of three independent biological samples&#x02009;&#x000b1;&#x02009;SE. Mean values significantly different from the control plants were determined by t tests (P&#x02009;&#x0003c;&#x02009;0.05) and are indicated by an asterisk.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Stress Responses in Soybean Roots and Nodules</h2>
			<h3>PMID:27462339</h3>
				
					<h3>PMCID:4941547</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kunert </surname>  <given-names> Karl J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/197577/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vorster </surname>  <given-names> Barend J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/255151/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fenta </surname>  <given-names> Berhanu A. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/358850/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kibido </surname>  <given-names> Tsholofelo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/353430/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dionisio </surname>  <given-names> Giuseppe </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/122360/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Foyer </surname>  <given-names> Christine H. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/64213/overview"/>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptome analysis and Next-Generation Sequencing (NGS) are current strategies to particularly study plant responses to abiotic stress ( <xref rid="B16" ref-type="bibr"> <span class="gene">Fan</span> et al., 2013 </xref> ). Identification of genes underpinning root traits and related <span class="trait">drought</span> responses have recently received intensive interest ( <xref rid="B49" ref-type="bibr"> Manavalan et al., 2009 </xref> ; <xref rid="B47" ref-type="bibr"> Libault et al., 2010 </xref> ; <xref rid="B12" ref-type="bibr"> Comas et al., 2013 </xref> ; <xref rid="B74" ref-type="bibr"> Thao et al., 2013 </xref> ; <xref rid="B65" ref-type="bibr"> Satbhai et al., 2015 </xref> ). Among 3,000 genes strongly up-regulated in roots by <span class="trait">drought</span> were several transcription factors, receptor-like kinases, calcium signaling components as well as jasmonate and abscisic acid biosynthetic genes ( <xref rid="B77" ref-type="bibr"> Tripathi et al., 2016 </xref> ). Transcriptome responses to <span class="trait">drought</span> are also highly dependent on stress intensity and duration as well as species and organs investigated. In the case of soybean roots, 145 root genes were for example differentially expressed due to <span class="trait">drought.</span> Identified gene functions demonstrated a complex <span class="trait">drought</span> response with genes involved in different multiple biochemical pathways related to <span class="trait">drought</span> adaptation ( <xref rid="B71" ref-type="bibr"> Stolf-Moreira et al., 2011 </xref> ). Applying the deep SuperSAGE method, increased expression of 1,127 unitags in a stress-tolerant soybean accession were associated with responses to hormone stimuli, water stress, as well as oxidative stresses ( <xref rid="B56" ref-type="bibr"> Neto et al., 2013 </xref> ). Other transcriptome studies were carried out with soybean cultivars W82 and DT2008. The genome of W82, often used as a model cultivar, was sequenced several years ago ( <xref rid="B66" ref-type="bibr"> Schmutz et al., 2010 </xref> ). DT2008, an economically important soybean cultivar and widely grown in Vietnam ( <xref rid="B82" ref-type="bibr"> Vinh et al., 2010 </xref> ; <xref rid="B72" ref-type="bibr"> Sulieman et al., 2015 </xref> ), has high <span class="trait">drought</span> tolerance ( <xref rid="B28" ref-type="bibr"> Ha et al., 2013 </xref> ; <xref rid="B72" ref-type="bibr"> Sulieman et al., 2015 </xref> ) and better nodule development under <span class="trait">drought</span> when compared to W82 ( <xref rid="B72" ref-type="bibr"> Sulieman et al., 2015 </xref> ). By comparing the root transcriptomes of DT2008 and W82, seedlings under normal and dehydration conditions (2 and 10 h treatment), 38172 soybean genes, which changed in expression, could be annotated with high confidence ( <xref rid="B29" ref-type="bibr"> Ha et al., 2015 </xref> ). Data suggested that higher <span class="trait">drought</span> tolerability of DT2008 roots, when compared to W82, might be attributed to a higher number of root genes induced by early dehydration than by prolonged dehydration. The higher <span class="trait">drought</span> tolerability of DT2008 vs. W82 might be further attributed to differential expression of genes associated in osmo-protectant biosynthesis, detoxification, cell wall-related proteins, kinases, transcription factors as well as phosphatase 2C proteins ( <xref rid="B29" ref-type="bibr"> Ha et al., 2015 </xref> ). In particular, the levels of transcripts encoding the auxin responsive factors (ARFs) <italic> <span class="gene">GmARF33</span> </italic> and <italic> <span class="gene">GmARF50</span> </italic> were greatly increased in shoots and roots. For example, <italic> <span class="gene">GmARF50</span> </italic> transcripts were rapidly increased by 15- and 30-fold after 2 and 10 h of dehydration, respectively ( <xref rid="B28" ref-type="bibr"> Ha et al., 2013 </xref> ). Further, subjecting Williams 82 to increasing <span class="trait">drought</span> conditions caused the total differential expression of 6609 transcripts including many genes involved in hormone (auxin/ethylene), carbohydrate, cell wall-related secondary metabolism as well as transcription factors controlling root growth ( <xref rid="B69" ref-type="bibr"> Song et al., 2016 </xref> ). However, a more in-depth functional characterization is still required to determine how these transcripts will lead to better <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Improving the soybean-rhizobia symbiosis might also contribute to better <span class="trait">drought</span> tolerance. More robust rhizobia with better osmo-tolerance of rhizobia to persist for longer in droughted soils might thereby be a contributor ( <xref rid="B53" ref-type="bibr"> Mhadhbi et al., 2013 </xref> ). Recent research has also provided evidence that plant growth-promoting rhizobacterium (PGPR) improve plant adaptation to <span class="trait">drought</span> by stimulating lateral root formation and increasing shoot growth ( <xref rid="B64" ref-type="bibr"> Rolli et al., 2015 </xref> ) with stimulation partly caused by bacterium-produced volatile organic compounds ( <xref rid="B83" ref-type="bibr"> Wintermans et al., 2016 </xref> ). Also, salicylic acid to assemble a better root microbiome might play a role, since salicylic acid can modulate colonization of the root by specific bacterial families ( <xref rid="B45" ref-type="bibr"> Lebeis et al., 2015 </xref> ). Pyrrolizidine alkaloids (PAs), involved in plant cell re-programming for micro-symbiont entry, might be further a contributor and a target for investigation. A plant-homo-spermidine synthase (HSS), the first pathway-specific enzyme of PA biosynthesis, is exclusively localized in nodules ( <xref rid="B38" ref-type="bibr"> Irmer et al., 2015 </xref> ) suggesting that the plant is the main PA producer. Investigation how <span class="trait">drought</span> affects expression of soybean nodule HSS <span class="gene">(Glyma.06g126700)</span> might be therefore interesting.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Stress Causes a Reduction in the Biosynthesis of Ascorbic Acid in Soybean Plants</h2>
			<h3>PMID:28663755</h3>
				
					<h3>PMCID:5471321</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Seminario </surname>  <given-names> Amaia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/156596/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103322/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zulet </surname>  <given-names> Amaia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gonz&#x000e1;lez </surname>  <given-names> Esther M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/143837/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Larrainzar </surname>  <given-names> Est&#x000ed;baliz </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/136395/overview"/>  </contrib>
				<br/> 
				年份：<day> 30 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Glycine max </italic> (L.) Merr. cv Oxumi seeds were sterilized as previously described ( <xref rid="B26" ref-type="bibr"> Labhilili et al., 1995 </xref> ), germinated for 3 days in the darkness at 26&#x000b0;C and grown in 1-L pots containing 2:1 (v/v) vermiculite:perlite under controlled conditions (24&#x000b0;C/18&#x000b0;C day/night temperature, 60/70% day/night relative humidity and 16-h photoperiod). Plants were watered three times a week with a nutrient solution ( <xref rid="B39" ref-type="bibr"> Rigaud and Puppo, 1975 </xref> ) supplemented with 5 mM KNO <sub> 3 </sub> . Four-week-old plants were separated into two sets: control and <span class="trait">drought.</span> Control plants were supplied daily with the nutrient solution to field capacity, whereas <span class="trait">drought</span> was achieved by withholding water/nutrients. Two hours after the start of the photoperiod, leaf and root &#x003a8; <sub> w </sub> were measured. Leaf &#x003a8; <sub> w </sub> was measured in the first fully expanded leaf using a pressure chamber (Soil Moisture Equipment, Santa Barbara, CA, United States) ( <xref rid="B42" ref-type="bibr"> Scholander et al., 1966 </xref> ). Root &#x003a8; <sub> w </sub> was measured using C52 sample chambers coupled to a HR-33T Dew Point microvoltmeter (Wescor). Soybean leaf &#x003a8; <sub> w </sub> was daily monitored to classify plants into control (C; leaf &#x003a8; <sub> w </sub> = -0.23 &#x000b1; 0.04 MPa), VMS (leaf &#x003a8; <sub> w </sub> = -0.54 &#x000b1; 0.02 MPa) and <span class="gene">MS</span> (leaf &#x003a8; <sub> w </sub> = -1.13 &#x000b1; 0.1 MPa). Plant tissue FW was measured and aliquots were collected, immediately frozen in liquid nitrogen and stored at - 80&#x000b0;C for analytical determinations. The remaining tissue was employed for DW determinations after drying for 48 h at 70&#x000b0;C. WC was calculated using the following formula: WC = [(FW-DW)/FW] &#x000d7; 100.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean plants were subjected to progressive <span class="trait">drought</span> stress by withholding water/nutrients and leaf &#x003a8; <sub> w </sub> was monitored to determine the level of stress. Plants were assigned to the following groups: well-watered plants were used as <underline> c </underline> ontrols (C; leaf &#x003a8; <sub> w </sub> = -0.23 &#x000b1; 0.04 MPa); 3 days after the onset of <span class="trait">drought</span> plants were classified as VMS (leaf &#x003a8; <sub> w </sub> = -0.54 &#x000b1; 0.02 MPa) and 7 days after the onset of <span class="trait">drought</span> plants were under <span class="gene">MS</span> conditions (MS; leaf &#x003a8; <sub> w </sub> = -1.13 &#x000b1; 0.1 MPa) ( <bold> Figure <xref ref-type="fig" rid="F1"> 1A </xref> </bold> ). To better characterize the plant water status, we also measured root &#x003a8; <sub> w </sub> ( <bold> Figure <xref ref-type="fig" rid="F1"> 1A </xref> </bold> ) and the relative WC ( <bold> Figure <xref ref-type="fig" rid="F1"> 1B </xref> </bold> ). A significant decline in &#x003a8; <sub> w </sub> was observed both in leaves and roots of plants subjected to progressive <span class="trait">drought.</span> In leaves, <span class="gene">MS</span> caused an almost fivefold decline in &#x003a8; <sub> w </sub> compared to C plants. In roots, &#x003a8; <sub> w </sub> values declined fourfold in VMS and sixfold in <span class="gene">MS</span> conditions, reaching more negative values compared to the aerial part. Leaf and root WC in C plants showed values around 80&#x02013;90% throughout the study period ( <bold> Figure <xref ref-type="fig" rid="F1"> 1B </xref> </bold> ). Water deprivation caused a gradual decrease in WC in both organs, which was more pronounced in roots (-63% vs. -26%; <bold> Figure <xref ref-type="fig" rid="F1"> 1B </xref> </bold> ). This response is consistent with the observed decline in &#x003a8; <sub> w </sub> values in the different <span class="trait">drought</span> treatments and provides a physiological context to the progressive <span class="trait">drought</span> stress imposed.
					</p>
				
					<p>&nbsp;&nbsp;
						Effects of progressive <span class="trait">drought</span> stress on leaf and root water potentials <bold> (A) </bold> and WC <bold> (B) </bold> in soybean plants under well-watered control (C), VMS and <span class="gene">MS</span> conditions. Values represent the average &#x000b1; SE ( <italic> n </italic> = 4 biological replicates). An asterisk ( <sup> &#x02217; </sup> ) denotes significant differences (Student&#x02019;s <italic> t </italic> -test, <italic> p </italic> &#x02264; 0.05) with respect to C plants.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain further insights into the regulation of the biosynthesis of AsA at the transcriptional level, we analyzed the expression patterns of the genes involved in the pathway in soybean plants subjected to a progressive <span class="trait">drought</span> stress followed by a rewatering treatment ( <xref rid="B43" ref-type="bibr"> Song et al., 2016 </xref> ). Based on the Smirnoff-Wheeler pathway described for <italic> A. thaliana </italic> , we first searched for the putative orthologous genes in the soybean genome using a BLASTP (Basic Local Alignment Search Tool Protein <sup> <xref ref-type="fn" rid="fn05"> 5 </xref> </sup> ) approach. Since the first steps in the pathway generate compounds also involved in other metabolic pathways, we focused our analysis on the following enzymes: VTC1 (GDP-D-mannose pyrophosphorylase), <span class="gene">GME</span> (GDP-D-mannose 3&#x02032;,5&#x02032;-epimerase), VTC2/VTC5 (GDP-L-galactose phosphorylase), VTC4 (L-galactose-1-P phosphatase), GalDH (L-galactose dehydrogenase) and GalLDH. Unlike the situation in <italic> A. thaliana </italic> , the soybean genome contains multiple gene copies for each enzymatic step in the pathway, many of which are yet annotated as &#x0201c;unnamed protein product.&#x0201d; Four genes were found to code for VTC1 <span class="gene">(Glyma02g41820,</span> <span class="gene">Glyma11g34550,</span> <span class="gene">Glyma14g07150</span> and <span class="gene">Glyma18g03840),</span> five genes for <span class="gene">GME</span> <span class="gene">(Glyma01g09540,</span> <span class="gene">Glyma03g40720,</span> <span class="gene">Glyma10g30400,</span> <span class="gene">Glyma19g43410</span> and <span class="gene">Glyma20g36740),</span> the same set of four were retrieved when VTC2 and VTC5 were queried <span class="gene">(Glyma02g46230,</span> <span class="gene">Glyma08g43200,</span> <span class="gene">Glyma14g02500</span> and <span class="gene">Glyma18g10430),</span> three for VTC4 <span class="gene">(Glyma07g39620,</span> <span class="gene">Glyma09g01380</span> and <span class="gene">Glyma15g12230),</span> two for GalDH <span class="gene">(Glyma07g30395</span> and <span class="gene">Glyma08g06840)</span> and two for GalLDH <span class="gene">(Glyma02g27260</span> and <span class="gene">Glyma10g17370).</span> A known regulator of the pathway in <italic> A. thaliana </italic> , VTC3, for which only one candidate ortholog was found in the soybean genome, <span class="gene">Glyma02g21970,</span> was also included. A similar BLASTP analysis was carried out in the pasture legume <italic> M. truncatula </italic> , leading to the identification of two genes coding for VTC1 (Medtr3g069070 and Medtr5g080770), <span class="gene">GME</span> (Medtr1g080950 and Medtr7g115080), and VTC2/VTC5 (Medtr3g053020 and Medtr5g093390), and single-copy genes for VTC4 (Medtr2g026060), GalDH (Medtr4g092750) and GalLDH (Medtr1g050360). In the case of <italic> AMR1 </italic> (At2g40860.1; <xref rid="B58" ref-type="bibr"> Zhang et al., 2009 </xref> ), BLASTP searches did not retrieve any good match in either the soybean or the <italic> M. truncatula </italic> genome.
					</p>
				
					<p>&nbsp;&nbsp;
						In terms of <span class="trait">drought</span> responses in soybean, the mildest stress treatment showed one of the strongest responses in leaf tissue, with a reduction in the transcript levels of most of the genes in the pathway compared to control plants. The expression of three out of the five <italic> <span class="gene">GME</span> </italic> genes and two <italic> VTC4 </italic> genes, however, increased at later <span class="trait">drought</span> stages, a situation that was reverted upon rewatering of the plants. In contrast, the <italic> GalLDH </italic> genes, coding for the last enzyme in the pathway and supposedly one of the key regulators of the pathway, did not show a differential expression during <span class="trait">drought</span> ( <bold> Figure <xref ref-type="fig" rid="F5"> 5A </xref> </bold> ). In root tissue, however, the first stage of <span class="trait">drought</span> stress caused an increase in the transcript levels of most genes in the pathway, with the exception of the <italic> <span class="gene">GME</span> </italic> gene <span class="gene">Glyma03g40720</span> and the <italic> GalDH </italic> gene <span class="gene">Glyma08g06840,</span> which showed a reduction in gene expression. Interestingly, the <italic> VTC4 </italic> gene <span class="gene">Glyma09g01380</span> showed an induction under severe <span class="trait">drought</span> conditions, a similar expression pattern to this observed in leaf tissue ( <bold> Figure <xref ref-type="fig" rid="F5"> 5B </xref> </bold> ). Regarding <italic> VTC3 </italic> , only in root tissue there was a drought-related response of this putative regulator, whose transcript levels were lower under water deficit, a situation that was reverted by rewatering the plants.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate whether this regulation at the transcriptional level was also found in other legume species, we queried the expression of genes in the Smirnoff-Wheeler pathway in the Medicago Gene Atlas database <sup> <xref ref-type="fn" rid="fn06"> 6 </xref> </sup> ( <xref rid="B9" ref-type="bibr"> Benedito et al., 2008 </xref> ), which has been now extended with expression data corresponding to a progressive <span class="trait">drought</span> stress followed by a recovery treatment ( <xref rid="B57" ref-type="bibr"> Zhang et al., 2014 </xref> ). In terms of <span class="trait">drought</span> responses, <italic> M. truncatula </italic> showed a differential response compared to soybean (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> 1 </xref> ). For instance, in leaves progressive <span class="trait">drought</span> caused a gradual increase in the expression of one of the <italic> <span class="gene">GME</span> </italic> genes, Medtr7g115080, while the levels <italic> VTC1 </italic> gene Medtr5g080770 and the regulatory gene <italic> VTC3 </italic> (Medtr1g050520) were reduced compared to well-watered plants. Interestingly, in roots <span class="trait">drought</span> induced the expression of the same <italic> <span class="gene">GME</span> </italic> gene observed in leaves, along with the <italic> VTC2 </italic> gene Medtr5g093390, while there was a reduction of the transcript levels of the two <italic> VTC1 </italic> genes (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> 1 </xref> ). It is worth noting that the two genes putatively coding for <span class="gene">GME</span> show an opposite pattern of expression both in roots and in leaves, which suggests a functional specialization of the genes.
					</p>
				
					<p>&nbsp;&nbsp;
						To check whether there was a regulation of the pathway at the transcriptional level, we analyzed the expression patterns of enzymes in the Smirnoff-Wheeler pathway, including putative orthologs described as regulators in other plant species, in soybean and <italic> M. truncatula </italic> plants exposed to <span class="trait">drought</span> stress. Since the pathway has not been specifically described in legumes, we used the protein sequences of enzymes described in <italic> A. thaliana </italic> to identify the closest orthologous genes. Interestingly, we identified several closely related genes encoding for each of the enzymes in the pathway in soybean, as confirmed by the phylogenetic analysis ( <bold> Figure <xref ref-type="fig" rid="F4"> 4 </xref> </bold> ). This genetic redundancy can be explained by the fact that the soybean genome has undergone at least two polyploidy events, estimated to occur &#x0223c;13 and &#x0223c;59 million years ago, which has led to 75% of the genes in the soybean genome being present in multiple copies ( <xref rid="B41" ref-type="bibr"> Roulin et al., 2013 </xref> ). It is interesting to note, however, how these multiple gene copies show differential patterns of expression, as observed for the soybean <italic> <span class="gene">GME</span> </italic> and <italic> VTC4 </italic> orthologs ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ), suggesting certain level of specialization at the functional level. These contrasting expression patterns may be also explained by a differential localization of the gene expression at the tissue level, a dimension that is lost when harvesting the complete tissue. Interestingly, the expression of genes coding for the full AsA biosynthetic pathway was detected not only in leaves but also in roots of both legume species. This suggests that although the vast majority of works in the literature have focused on AsA biosynthesis in leaves, roots have the potential of synthesizing this antioxidant compounds as well ( <xref rid="B22" ref-type="bibr"> Hancock et al., 2003 </xref> ; <xref rid="B28" ref-type="bibr"> Liso et al., 2004 </xref> ), as shown in pea roots ( <xref rid="B56" ref-type="bibr"> Zabalza et al., 2007 </xref> ), <italic> Lotus japonicus </italic> root nodules ( <xref rid="B31" ref-type="bibr"> Matamoros et al., 2006 </xref> ), and a number of non-legume species.
					</p>
				
					<p>&nbsp;&nbsp;
						The biosynthesis of AsA in plants is postulated to be regulated by several enzymes in the pathway, namely, GalLDH ( <xref rid="B54" ref-type="bibr"> Yabuta et al., 2007 </xref> ), GDP-D-mannose pyrophosphorylase (VTC1; <xref rid="B14" ref-type="bibr"> Conklin et al., 1997 </xref> , <xref rid="B13" ref-type="bibr"> 1999 </xref> ; <xref rid="B48" ref-type="bibr"> Veljovic-Jovanovic et al., 2001 </xref> ; <xref rid="B2" ref-type="bibr"> Badejo et al., 2007 </xref> , <xref rid="B3" ref-type="bibr"> 2008 </xref> ; <xref rid="B49" ref-type="bibr"> Wang et al., 2013 </xref> ), GDP-D-mannose 3&#x02032;, 5&#x02032;-epimerase <span class="gene">(GME)</span> in association with GDP-L-galactose phosphorylase (VTC2/VTC5; <xref rid="B16" ref-type="bibr"> Dowdle et al., 2007 </xref> ; <xref rid="B52" ref-type="bibr"> Wolucka and Van Montagu, 2007 </xref> ; <xref rid="B27" ref-type="bibr"> Linster and Clarke, 2008 </xref> ; <xref rid="B10" ref-type="bibr"> Bulley et al., 2009 </xref> ), the predicted F-box protein AMR1 ( <xref rid="B58" ref-type="bibr"> Zhang et al., 2009 </xref> ) and the protein kinase::protein phosphatase VTC3 ( <xref rid="B12" ref-type="bibr"> Conklin et al., 2013 </xref> ). These works are mainly based on the analysis of <italic> A. thaliana </italic> mutants defective in the production of AsA, or on correlations between the levels of AsA and gene expression of these enzymes in other plant species. Nevertheless, the regulation of the pathway under abiotic stress, particularly <span class="trait">drought,</span> has been barely studied. In the case of legume plants, this is the first work to our knowledge in which the genes coding for enzymes in the Smirnoff-Wheeler have been identified.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, <italic> GalLDH </italic> was not found differentially expressed in either soybean or <italic> M. truncatula </italic> plants under <span class="trait">drought</span> stress ( <bold> Figure <xref ref-type="fig" rid="F4"> 4B </xref> </bold> and Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> 1 </xref> , respectively). If <italic> GalLDH </italic> is not regulated at the transcriptional level under water deficit conditions, it could be hypothesized that the observed decline in AsA levels and biosynthetic activity may be due to post-translational regulatory mechanisms of the GalLDH enzyme. This is further supported by the fact that <italic> in silico </italic> prediction of phosphorylation sites <sup> <xref ref-type="fn" rid="fn07"> 7 </xref> </sup> ( <xref rid="B20" ref-type="bibr"> Gao et al., 2010 </xref> ) identify two conserved Ser residues (Ser472 and 476, and Ser469 and 473 in <span class="gene">Glyma.02G166300</span> and <span class="gene">Glyma.10G104100,</span> respectively; &#x0003e; %99 specificity) as potential targets for phosphorylation. Furthermore, several potential ubiquitylation sites are also found in the predicted soybean proteins [support vector machine probability &#x0003e; 0.83 <sup> <xref ref-type="fn" rid="fn08"> 8 </xref> </sup> ( <xref rid="B11" ref-type="bibr"> Chen et al., 2013 </xref> )]. Thus, one hypothesis to explain the reduction in the content of AsA and biosynthesis activity detected in leaves of drought-stressed soybeans could be differential phosphorylation of the enzyme or activation of its degradation via the ubiquitin-proteasome pathway.
					</p>
				
					<p>&nbsp;&nbsp;
						Regarding the rest of genes tested, we could not find a direct link between their expression pattern and the reduction of AsA levels and biosynthetic rates measured under water-deficit conditions. In some cases transcript levels declined at VMS conditions but increased afterward. This is for instance the case of three out of the five genes encoding <span class="gene">GME,</span> the <italic> VTC2/VTC5 </italic> <span class="gene">Glyma02g46230</span> and two <italic> VTC4 </italic> paralogs in soybean leaves. Furthermore, we often observed that putative paralogs presented differential gene expression patterns, suggesting the occurrence of functional specialization. For example, the expression of the <italic> M. truncatula <span class="gene">GME</span> </italic> gene Medtr7g115080 was induced both in leaves and roots, while the transcript levels of the other putative paralog, Medtr1g080950, were reduced in roots of <span class="trait">drought</span> stressed plants (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> 1B </xref> ). Similarly, the transcript levels of <italic> VTC3 </italic> , another known regulator of the pathway ( <xref rid="B12" ref-type="bibr"> Conklin et al., 2013 </xref> ), were found to decline only at early <span class="trait">drought</span> stages in soybean roots, but not subsequently ( <bold> Figure <xref ref-type="fig" rid="F5"> 5B </xref> </bold> ), while the content of AsA showed a continuous progressive decline as <span class="trait">drought</span> stress levels increased.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean DREB1/CBF-type transcription factors function in heat and <span class="trait">drought</span> as well as cold stress-responsive gene expression.</h2>
			<h3>PMID:25495120  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kidokoro S(1), Watanabe K, Ohori T, Moriwaki T, Maruyama K, Mizoi J, Myint Phyu Sin Htwe N, Fujita Y, Sekita S, Shinozaki K, Yamaguchi-Shinozaki K.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is a globally important crop, and its growth and <span class="trait">yield</span> are severely reduced by abiotic stresses, such as <span class="trait">drought,</span> heat, and cold. The cis-acting element DRE (dehydration-responsive element)/CRT plays an important role in activating gene expression in response to these stresses. The Arabidopsis DREB1/CBF genes that encode DRE-binding proteins function as transcriptional activators in the cold stress responsive gene expression. In this study, we identified 14 DREB1-type transcription factors <span class="gene">(GmDREB1s)</span> from a soybean genome database. The expression of most <span class="gene">GmDREB1</span> genes in soybean was strongly induced by a variety of abiotic stresses, such as cold, <span class="trait">drought,</span> high salt, and heat. The <span class="gene">GmDREB1</span> proteins activated transcription via DREs (dehydration-responsive element) in Arabidopsis and soybean protoplasts. Transcriptome analyses using transgenic Arabidopsis plants overexpressing <span class="gene">GmDREB1s</span> indicated that many of the downstream genes are cold-inducible and overlap with those of Arabidopsis DREB1A. We then comprehensively analyzed the downstream genes of <span class="gene">GmDREB1B;1,</span> which is closely related to DREB1A, using a transient expression system in soybean protoplasts. The expression of numerous genes induced by various abiotic stresses were increased by overexpressing <span class="gene">GmDREB1B;1</span> in soybean, and DREs were the most conserved element in the promoters of these genes. The downstream genes of <span class="gene">GmDREB1B;1</span> included numerous soybean-specific stress-inducible genes that encode an ABA receptor family protein, <span class="gene">GmPYL21,</span> and translation-related genes, such as ribosomal proteins. We confirmed that <span class="gene">GmDREB1B;1</span> directly activates <span class="gene">GmPYL21</span> expression and enhances ABRE-mediated gene expression in an ABA-independent manner. These results suggest that <span class="gene">GmDREB1</span> proteins activate the expression of numerous soybean-specific stress-responsive genes under diverse abiotic stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the activated form of the AtAREB1 gene (AtAREB1ΔQT) improves soybean responses to water deficit.</h2>
			<h3>PMID:25158254  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Leite JP(1), Barbosa EG(2), Marin SR(2), Marinho JP(3), Carvalho JF(4), Pagliarini RF(2), Cruz AS(2), Oliveira MC(5), Farias JR(4), Neumaier N(4), Guimarães FC(2), Yoshida T(6), Kanamori N(6), Fujita Y(6), Nakashima K(6), Shinozaki KY(6), Desidério JA(1), Nepomuceno AL(7).
				<br/> 
				年份：2014 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Abscisic acid-responsive element binding protein (AREB1) is a basic domain/leucine zipper transcription factor that binds to the abscisic acid (ABA)-responsive element motif in the promoter region of ABA-inducible genes. Because <span class="gene">AREB1</span> is not sufficient to direct the expression of downstream genes under non-stress conditions, an activated form of <span class="gene">AREB1</span> (AREB1ΔQT) was created. Several reports claim that plants overexpressing <span class="gene">AREB1</span> or AREB1ΔQT show improved <span class="trait">drought</span> tolerance. In our studies, soybean plants overexpressing AREB1ΔQT were characterized molecularly, and the phenotype and <span class="trait">drought</span> response of three lines were accessed under greenhouse conditions. Under conditions of water deficit, the transformed plants presented a higher survival rate (100%) than those of their isoline, cultivar BR 16 (40%). Moreover, the transformed plants displayed better water use efficiency and had a higher number of leaves than their isoline. Because the transgenic plants had higher stomatal conductance than its isoline under well-watered conditions, it was suggested that the enhanced <span class="trait">drought</span> response of AREB1ΔQT soybean plants might not be associated with altered transpiration rates mediated by ABA-dependent stomatal closure. However, it is possible that the smaller leaf area of the transgenic plants reduced their transpiration and water use, causing delayed stress onset. The difference in the degree of wilting and percentage of survival between the 35S-AREB1ΔQT and wildtype plants may also be related to the regulation of genes that protect against dehydration because metabolic impairment of photosynthesis, deduced by an increasing internal CO2 concentration, was not observed in the transgenic plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential Gene Expression in Soybean Leaf Tissues at Late Developmental Stages under <span class="trait">Drought</span> Stress Revealed by Genome-Wide Transcriptome Analysis</h2>
			<h3>PMID:23189148</h3>
				

				
					<h3>PMCID:3505142</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Le </surname>  <given-names> Dung Tien </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishiyama </surname>  <given-names> Rie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Yasuko </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Maho </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seki </surname>  <given-names> Motoaki </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ham </surname>  <given-names> Le Huy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi-Shinozaki </surname>  <given-names> Kazuko </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shinozaki </surname>  <given-names> Kazuo </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 10 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The availability of complete genome sequence of soybean has allowed research community to design the 66 K Affymetrix Soybean Array GeneChip for genome-wide expression profiling of soybean. In this study, we carried out microarray analysis of leaf tissues of soybean plants, which were subjected to <span class="trait">drought</span> stress from late vegetative V6 and from full bloom reproductive R2 stages. Our data analyses showed that out of 46,093 soybean genes, which were predicted with high confidence among approximately 66,000 putative genes, 41,059 genes could be assigned with a known function. Using the criteria of a ratio change &gt; = 2 and a q-value&lt;0.05, we identified 1458 and 1818 upregulated and 1582 and 1688 downregulated genes in drought-stressed V6 and R2 leaves, respectively. These datasets were classified into 19 most abundant biological categories with similar proportions. There were only 612 and 463 genes that were overlapped among the upregulated and downregulated genes, respectively, in both stages, suggesting that both conserved and unconserved pathways might be involved in regulation of <span class="trait">drought</span> response in different stages of plant development. A comparative expression analysis using our datasets and that of <span class="trait">drought</span> stressed Arabidopsis leaves revealed the existence of both conserved and species-specific mechanisms that regulate <span class="trait">drought</span> responses. Many upregulated genes encode either regulatory proteins, such as transcription factors, including those with high homology to Arabidopsis DREB, NAC, AREB and ZAT/STZ transcription factors, kinases and two-component system members, or functional proteins, e.g. late embryogenesis-abundant proteins, glycosyltransferases, glycoside hydrolases, defensins and glyoxalase I family proteins. A detailed analysis of the <span class="gene">GmNAC</span> family and the hormone-related gene category showed that expression of many <span class="gene">GmNAC</span> and hormone-related genes was altered by <span class="trait">drought</span> in V6 and/or R2 leaves. Additionally, the downregulation of many photosynthesis-related genes, which contribute to growth retardation under <span class="trait">drought</span> stress, may serve as an adaptive mechanism for plant survival. This study has identified excellent drought-responsive candidate genes for in-depth characterization and future development of improved drought-tolerant transgenic soybeans.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Keeping all these in mind, in this study we have performed a microarray analysis using the 66 K Affymetrix soybean GeneChip to gain an overall picture of transcriptome-wide changes in soybean leaves under <span class="trait">drought</span> stress. In this study, we imposed <span class="trait">drought</span> stress on soybean plants from late vegetative stage (V6) till early bloom reproductive stage (R1) and during full bloom R2 stage, and examined differential gene expression in V6 and R2 leaves of soybean plants grown under well-watered and <span class="trait">drought</span> conditions. The period from late V6 stage toward the end of R2 stage is known as one of critical periods that hurts <span class="trait">yield</span> ( <ext-link ext-link-type="uri" xlink:href="http://www.okstate.edu/OSU_Ag/oces/timely/soybean.htm"> http://www.okstate.edu/OSU_Ag/oces/timely/soybean.htm </ext-link> ), giving rise to the need of studying mechanisms of soybean responses to <span class="trait">drought</span> stress during this period aimed at developing drought-tolerant transgenic soybeans. Since the expression profiles of many genes obtained by qRT-PCR and microarray analysis were in good accordance, this array platform was found to be suitable for a high-throughput genome-wide analysis. Furthermore, the microarray data showed transcriptional changes of various well-known functional and regulatory genes; including transcription factors (TFs), kinases, heat shock proteins, late embryogenesis-abundant (LEA) proteins, osmoprotectant biosynthesis-related proteins, hormone-related proteins, transporters and detoxification enzymes. In addition, we have performed a comparative expression analysis of V6 and R2 microarray datasets to search for the conserved and unconserved sets of genes which are involved in regulation of <span class="trait">drought</span> response in different stages of plant development. We have then expanded our comparative analysis to species level to identify conserved and species-specific drought-responsive genes in soybean and <italic> Arabidopsis </italic> by comparing our soybean transcriptome datasets and that of <span class="trait">drought</span> stressed <italic> Arabidopsis </italic> leaves. Finally, our interest in research on the functions of the <italic> NAC </italic> TF family members and genes involved in hormone metabolism and hormone signaling pathways under <span class="trait">drought</span> stress has prompted us to carry out a detailed analysis of the <italic> <span class="gene">GmNAC</span> </italic> TF family and the hormone-related gene category. This study ultimately provides excellent candidates for in-depth characterization and future development of improved drought-tolerant transgenic soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						With the availability of the whole genomic sequence of soybean and the Glyma1 annotation, which predicted 46430 protein-coding genes with high confidence out of approximately 66000 putative genes <xref ref-type="bibr" rid="pone.0049522-Schmutz1"> [15] </xref> , Affymetrix has designed the newest version of the Soybean Array GeneChip covering all of the soybean putative genes. To gain a comprehensive overview of the transcriptome-wide regulation in soybean leaves under <span class="trait">drought</span> stress during the critical period spanning from late V6 stage toward the end of R2 stage, we performed microarray analyses to compare the transcriptome changes of drought-stressed V6 leaves vs. well-watered V6 leaves (comparison V6-D/V6-C) and drought-stressed R2 leaves vs. well-watered R2 leaves (comparison R2-D/R2-C) using this 66 K Affymetrix Soybean Array GeneChip. Our analysis indicated that this array version contains 66555 probes. Within this array, 66195 probes matched with the respective gene IDs used in the Glyma1 model, corresponding to 99.46% of the total probes ( <xref ref-type="supplementary-material" rid="pone.0049522.s001"> Tables S1 </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s002"> S2 </xref> , Gene Expression Omnibus accession number: GSE40627). With reference to the annotation of the Glyma1 version <xref ref-type="bibr" rid="pone.0049522-Schmutz1"> [15] </xref> , of all the genes identified on the 66 K Affymetrix Soybean Array GeneChip, 46093 genes could be annotated ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3A </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s004"> S4A </xref> ). Among the genes that could be annotated, 41059 genes could be assigned with a known function, and these genes were used in further analyses ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3B </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s004"> S4B </xref> ). Using the criteria of a ratio change &#x0003e;&#x0200a;=&#x0200a;2 and a q-value&#x0003c;0.05, 1458 and 1582 genes were found to be upregulated and downregulated, respectively, in drought-stressed V6 leaves ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3C, S3D </xref> ). When the same criteria were applied for comparison R2-D/R2-C, we were able to detect 1818 upregulated and 1688 downregulated genes. For verification of our microarray data, first we selected three well-known senescence-inducible genes, two soybean stay-green <italic> <span class="gene">GmSGR1</span> </italic> and <italic> <span class="gene">GmSGR2</span> </italic> genes <xref ref-type="bibr" rid="pone.0049522-Park1"> [20] </xref> and a senescence-associated receptor-like kinase <italic> <span class="gene">GmSARK</span> </italic> gene <xref ref-type="bibr" rid="pone.0049522-Xu1"> [21] </xref> , and assessed their expression by qRT-PCR. Next, as another mean of verification we compared expression profiles of 7 <italic> <span class="gene">GmCKX</span> </italic> genes that were obtained by systematic expression analysis of cytokinin (CK)-metabolic genes using qRT-PCR <xref ref-type="bibr" rid="pone.0049522-Le1"> [16] </xref> with their respective expression data obtained by our microarray analysis. Results shown in <xref ref-type="table" rid="pone-0049522-t001"> Table 1 </xref> indicated good accordance between the microarray and qRT-PCR data, suggesting that the newly designed 66 K Affymetrix Soybean Array GeneChip is useful for genome-wide expression profiling of soybean genes in leaf tissue, and perhaps in other tissues as well, under <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						A closer look at the drought-induced gene sets classified the upregulated genes identified in comparisons V6-D/V6-C and R2-D/R2-C into regulatory and functional categories. For the regulatory category, many upregulated genes were grouped into TF, signaling and protein modification groups ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3C, S3D </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s004"> S4C, S4D </xref> ). In the TF group, many soybean genes encoding TFs with high homology to the well-known drought/abscisic acid (ABA)-inducible TFs, such as RD26/ANAC072 and ATAF1/ANAC002 of the NAC family (see detailed analysis of the NAC TF family below) and <span class="gene">AREB1</span> of the bZIP family <xref ref-type="bibr" rid="pone.0049522-Tran4"> [23] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0049522-Mochida3"> [27] </xref> , were upregulated in both two comparisons ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3C </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s004"> S4C </xref> ). Heat shock TF and heat shock protein encoding genes were also found among the genes with increased transcript abundance. All of these types of TFs are known to function in plant adaptation to various stresses, including <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0049522-Tran1"> [2] </xref> , <xref ref-type="bibr" rid="pone.0049522-Thao1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0049522-Tran2"> [5] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0049522-Hadiarto1"> [9] </xref> , <xref ref-type="bibr" rid="pone.0049522-Tran5"> [28] </xref> , <xref ref-type="bibr" rid="pone.0049522-Jogaiah1"> [29] </xref> . Interestingly, the soybean genes encoding soybean othologs of the <italic> Arabidopsis </italic> DREB1A and DREB1D of the AP2_EREBP family and ZAT10/STZ of the C2H2_Zn family were induced in drought-stressed V6 leaves but not in drought-stressed R2 leaves, suggesting that these TFs may be involved in regulation of <span class="trait">drought</span> response during vegetative growth (V6) rather than reproductive growth (R2); a phenomenon indicating developmental stage-specific function of TFs. For the signaling and protein modification groups, we identified many drought-inducible genes encoding kinases, such as CIPKs and MAP kinases, PP2C proteins and hormone-signaling related proteins (see detailed analysis below), which were reported to be involved in the regulation of the <span class="trait">drought</span> response <xref ref-type="bibr" rid="pone.0049522-Hadiarto1"> [9] </xref> , <xref ref-type="bibr" rid="pone.0049522-Umezawa1"> [30] </xref> , <xref ref-type="bibr" rid="pone.0049522-Takahashi1"> [31] </xref> . The functional category contained many upregulated genes encoding LEA proteins, ABA metabolism-related proteins, osmoprotectant biosynthesis-related proteins, transporters and detoxification enzymes ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3C, S3D </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Given the biotechnological potentials of the NAC family, in this section we aimed to analyze in detail the drought-responsive expression profiles of NAC TF family in drought-stressed V6 and R2 leaves. Among 41059 genes that could be assigned with a known function, 175 putatively annotated <italic> <span class="gene">GmNAC</span> </italic> genes were identified ( <xref ref-type="supplementary-material" rid="pone.0049522.s003"> Tables S3B </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s004"> S4B </xref> ). Out of these <italic> <span class="gene">GmNAC</span> </italic> genes, a total of 18 and 4 genes were found to be upregulated and downregulated by more than two-fold (q-value&#x0003c;0.05) in the soybean V6 leaves by <span class="trait">drought</span> stress ( <xref ref-type="table" rid="pone-0049522-t002"> Table 2 </xref> ), while using the same criteria 23 and 4 genes were upregulated and downregulated in drought-treated R2 leaves ( <xref ref-type="table" rid="pone-0049522-t003"> Table 3 </xref> ). Previously, expression analysis of 38 <italic> <span class="gene">GmNAC</span> </italic> genes in dehydrated shoots and roots of 12-d-old soybean seedlings using qRT-PCR has found 29 and 6 <italic> <span class="gene">GmNAC</span> </italic> genes upregulated and downregulated, respectively, in dehydrated shoot and/or root tissues <xref ref-type="bibr" rid="pone.0049522-Le3"> [41] </xref> . Comparison of the differential expression of the <italic> <span class="gene">GmNAC</span> </italic> genes in drought-stressed V6 and R2 leaves and dehydrated shoot and root tissues of 12-d-old soybean seedlings revealed that all the <italic> <span class="gene">GmNAC</span> </italic> genes upregulated in the drought-stressed V6 and R2 leaves were also upregulated in the dehydrated shoot and/or root tissues of 12-d-old soybean seedlings if their expression was examined in these tissues ( <xref ref-type="table" rid="pone-0049522-t002"> Tables 2 </xref> , <xref ref-type="table" rid="pone-0049522-t003"> 3 </xref> ). In addition, more than half of the <italic> <span class="gene">GmNAC</span> </italic> genes induced in drought-stressed V6 leaves were also upregulated in drought-stressed R2 leaves and vice versa ( <xref ref-type="table" rid="pone-0049522-t002"> Tables 2 </xref> , <xref ref-type="table" rid="pone-0049522-t003"> 3 </xref> ). Furthermore, a comparative analysis against the expression of the <italic> Arabidopsis NAC </italic> orthologous genes in leaves of 35-d-old <italic> Arabidopsis </italic> plants, which were subjected to a soil <span class="trait">drought</span> treatment <xref ref-type="bibr" rid="pone.0049522-Harb1"> [22] </xref> , indicated that the majority of the soybean and <italic> Arabidopsis NAC </italic> orthologous genes were drought-responsive in a similar manner, suggesting the existence of a relatively well-conserved <span class="trait">drought</span> response in the leaves of the two dicotic species at similar developmental stage. This comparative analysis may help us select drought-responsive soybean <italic> <span class="gene">GmNAC</span> </italic> genes with more confidence for further studies and genetic engineering. On the other hand, among 5 and 4 <italic> <span class="gene">GmNAC</span> </italic> genes significantly downregulated in the drought-stressed V6 and R2 leaves, respectively, two genes, <italic> <span class="gene">GmNAC057</span> </italic> and <italic> <span class="gene">GmNAC102</span> </italic> , were examined transcriptionally in the dehydrated shoot and root tissues of 12-d-old soybean seedlings. Interestingly, unlike in the drought-stressed V6 or R2 leaves the expression of these two genes was upregulated in the dehydrated root tissue and unchanged in the dehydrated shoot tissue of young soybean seedlings in comparison with the untreated controls ( <xref ref-type="table" rid="pone-0049522-t002"> Tables 2 </xref> , <xref ref-type="table" rid="pone-0049522-t003"> 3 </xref> ). These results together suggest that the dynamics of <span class="trait">drought</span> -responsive expression of the <italic> NAC </italic> genes in soybean is complex. Stresses may trigger different stress-responsive gene expression in different tissues at the same developmental stage or in the same tissue at different developmental stages. This characteristic of the <italic> <span class="gene">GmNAC</span> </italic> gene family, and perhaps other gene families, will enable us to perform genetic engineering in an organ-specific and/or developmental stage-specific manner.
					</p>
				
					<p>&nbsp;&nbsp;
						It is well established that various plant hormones, such as ABA, cytokinin (CK) and brassinosteroid (BR), and their respective hormone pathways are involved in regulation of <span class="trait">drought</span> stress responses <xref ref-type="bibr" rid="pone.0049522-Divi1"> [42] </xref> , <xref ref-type="bibr" rid="pone.0049522-Ha1"> [43] </xref> . Conversely, stresses are known to influence the expression of hormone-related genes, including those involved in hormone metabolisms and hormone signaling pathways, leading to changes in hormone homeostasis, redistribution and signaling <xref ref-type="bibr" rid="pone.0049522-Matsui1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0049522-Divi1"> [42] </xref> , <xref ref-type="bibr" rid="pone.0049522-Wang1"> [44] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0049522-Nishiyama2"> [46] </xref> . To have an overview on the expression profiles of hormone-related genes in the V6 and R2 leaves under <span class="trait">drought</span> stress, MapMan was used to visualize the gene expression data of the annotated hormone-related genes which were generated by our microarray analysis. In our study, we examined the expression levels of both biosynthetic and signaling genes related to auxin (IAA, indole-3-acetic acid), ethylene (ET), ABA, jasmonic acid (JA), giberelline (GA), brassinosteroid (BR), CK and salicylic acid (SA).Within the Glyma 1 annotation, we detected hormone-related genes which have significant change in expression levels by at least two-fold in drought-stressed V6 leaves ( <xref ref-type="supplementary-material" rid="pone.0049522.s005"> Table S5 </xref> ). The expression of these genes in drought-stressed R2 leaves was also extracted from comparison R2-D/R2-C for comparative analysis ( <xref ref-type="fig" rid="pone-0049522-g003"> Figure 3 </xref> , <xref ref-type="supplementary-material" rid="pone.0049522.s005"> Table S5 </xref> ). Auxin-related gene family showed the highest number with 40 members having differential expression. All the auxin-related genes identified were downregulated in the drought-stressed V6 leaves, and the majority of these genes showed reduced expression in the drought-stressed R2 leaves as well. This finding is in agreement with the results reported previously, in which the authors reported that almost all the auxin-related genes were downregulated in <italic> Arabidopsis </italic> whole plants <xref ref-type="bibr" rid="pone.0049522-Matsui1"> [12] </xref> and <italic> Sorghum bicolor </italic> leaves <xref ref-type="bibr" rid="pone.0049522-Wang1"> [44] </xref> under <span class="trait">drought</span> stress. Among the genes related to the hormones analyzed, ET-related genes formed the second major group with 34 genes displaying altered expression profiles in drought-stressed R2 and/or V6 leaves. A significant proportion of ET-related genes showed the same expression patterns in both V6 and R2 leaves, while only a few genes exhibited opposite expression profiles in the two leaves of different developmental stages under <span class="trait">drought</span> stress. ABA-related genes made the third biggest group with 14 members out of which 11 and 12 genes were remarkably induced in V6 and R2 leaves, respectively, by <span class="trait">drought</span> stress. Recently, BRs and CKs were shown to be involved in regulation of plant responses to <span class="trait">drought</span> stress, and genetic engineering of the homeostasis of these two hormones at biosynthesis or signaling levels enhanced tolerance to various stresses, including <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0049522-Divi1"> [42] </xref> , <xref ref-type="bibr" rid="pone.0049522-Ha1"> [43] </xref> , <xref ref-type="bibr" rid="pone.0049522-Nishiyama1"> [45] </xref> , <xref ref-type="bibr" rid="pone.0049522-Peleg1"> [47] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0049522-Choudhary3"> [50] </xref> . Our microarray analysis identified 8 BR-related and 8 CK-related genes showing altered expression in drought-stressed R2 and/or V6 leaves. The majority of the <italic> <span class="gene">GmCKX</span> </italic> genes were downregulated in both two types of soybean leaves under <span class="trait">drought</span> stress which is in consistence with the result reported previously in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0049522-Nishiyama1"> [45] </xref> . Similar to BR and CK, GA plays an important role in plant responses to both biotic and abiotic stresses <xref ref-type="bibr" rid="pone.0049522-Divi1"> [42] </xref> , <xref ref-type="bibr" rid="pone.0049522-Wang2"> [51] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0049522-Qin1"> [53] </xref> . Ten GA-related genes were recorded with altered expression in our study, and the majority of these genes exhibited downregulated expression in both the drought-stressed leaves. SA and JA are known as hormones regulating mainly biotic stress responses. Eleven JA-related and 2 SA-related genes were found to have differential expression in R2 and/or V6 leaves under imposed <span class="trait">drought</span> conditions, providing evidence that JA and SA may be involved in regulation of <span class="trait">drought</span> responses as well.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Expression Patterns of <span class="gene">GmAP2/EREB-Like</span> Transcription Factors Involved in Soybean Responses to Water Deficit</h2>
			<h3>PMID:23667465</h3>
				

				
					<h3>PMCID:3646874</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Marcolino-Gomes </surname>  <given-names> Juliana </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana Aparecida </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oliveira </surname>  <given-names> Maria Cristina Neves </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farias </surname>  <given-names> Jose Renato Bou&#x000e7;as </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neumaier </surname>  <given-names> Norman </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo Vilela </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar Corr&#x000ea;a </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 3 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean farming has faced several losses in productivity due to <span class="trait">drought</span> events in the last few decades. However, plants have molecular mechanisms to prevent and protect against water deficit injuries, and transcription factors play an important role in triggering different defense mechanisms. Understanding the expression patterns of transcription factors in response to water deficit and to environmental diurnal changes is very important for unveiling water deficit stress tolerance mechanisms. Here, we analyzed the expression patterns of ten APETALA2/Ethylene Responsive Element Binding-like (AP2/EREB-like) transcription factors in two soybean genotypes (BR16: drought-sensitive; and Embrapa 48: drought-tolerant). According to phylogenetic and domain analyses, these genes can be included in the DREB and ERF subfamilies. We also analyzed a <span class="gene">GmDRIP-like</span> gene that encodes a DREB negative regulator. We detected the up-regulation of 9 <span class="gene">GmAP2/EREB-like</span> genes and identified transcriptional differences that were dependent on the levels of the stress applied and the tissue type analyzed (the expression of the <span class="gene">GmDREB1F-like</span> gene, for example, was four times higher in roots than in leaves). The <span class="gene">GmDRIP-like</span> gene was not induced by water deficit in BR16 during the longest periods of stress, but was significantly induced in Embrapa 48; this suggests a possible genetic/molecular difference between the responses of these cultivars to water deficit stress. Additionally, RNAseq gene expression analysis over a 24-h time course indicates that the expression patterns of several <span class="gene">GmDREB-like</span> genes are subject to oscillation over the course of the day, indicating a possible circadian regulation.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmDREB2A;2, a canonical DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2-type transcription factor in soybean, is posttranslationally regulated and mediates dehydration-responsive element-dependent gene expression.</h2>
			<h3>PMID:23151346  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mizoi J(1), Ohori T, Moriwaki T, Kidokoro S, Todaka D, Maruyama K, Kusakabe K, Osakabe Y, Shinozaki K, Yamaguchi-Shinozaki K.
				<br/> 
				年份：2013 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is an important crop around the world. Abiotic stress conditions, such as <span class="trait">drought</span> and heat, adversely affect its survival, growth, and production. The DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2 (DREB2) group includes transcription factors that contribute to <span class="trait">drought</span> and heat stress tolerance by activating transcription through the cis-element dehydration-responsive element (DRE) in response to these stress stimuli. Two modes of regulation, transcriptional and posttranslational, are important for the activation of gene expression by <span class="gene">DREB2A</span> in Arabidopsis (Arabidopsis thaliana). However, the regulatory system of <span class="gene">DREB2</span> in soybean is not clear. We identified a new soybean <span class="gene">DREB2</span> gene, <span class="gene">GmDREB2A;2,</span> that was highly induced not only by dehydration and heat but also by low temperature. <span class="gene">GmDREB2A;2</span> exhibited a high transactivation activity via DRE and has a serine/threonine-rich region, which corresponds to a negative regulatory domain of <span class="gene">DREB2A</span> that is involved in its posttranslational regulation, including destabilization. Despite the partial similarity between these sequences, the activity and stability of the <span class="gene">GmDREB2A;2</span> protein were enhanced by removal of the serine/threonine-rich region in both Arabidopsis and soybean protoplasts, suggestive of a conserved regulatory mechanism that involves the recognition of serine/threonine-rich sequences with a specific pattern. The heterologous expression of <span class="gene">GmDREB2A;2</span> in Arabidopsis induced DRE-regulated stress-inducible genes and improved stress tolerance. However, there were variations in the growth phenotypes of the transgenic Arabidopsis, the induced genes, and their induction ratios between <span class="gene">GmDREB2A;2</span> and DREB2A. Therefore, the basic function and regulatory machinery of <span class="gene">DREB2</span> have been maintained between Arabidopsis and soybean, although differentiation has also occurred.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of soybean ubiquitin-conjugating enzyme gene <italic> <span class="gene">GmUBC2</span> </italic> confers enhanced <span class="trait">drought</span> and salt tolerance through modulating abiotic stress-responsive gene expression in <italic> Arabidopsis </italic></h2>
			<h3>PMID:19941154</h3>
				

				
					<h3>PMCID:2816239</h3>
					<h4>期刊:Plant Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Guo-An </given-names>  </name>  <address>  <email> guoanzhou2001@yahoo.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Ru-Zhen </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <address>  <phone> +86-01082105843 </phone>  <fax> +86-01082105840 </fax>  <email> qiu_lijuan@263.net </email>  <email> qiu_lijuan@mail.caas.net.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China  </aff>
				<br/> 
				年份：<day> 9 </day>  <month> 11 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies have shown that ubiquitination plays important roles in plant abiotic stress responses. In the present study, the ubiquitin-conjugating enzyme gene <span class="gene">GmUBC2,</span> a homologue of yeast RAD6, was cloned from soybean and functionally characterized. <span class="gene">GmUBC2</span> was expressed in all tissues in soybean and was up-regulated by <span class="trait">drought</span> and salt stress. Arabidopsis plants overexpressing <span class="gene">GmUBC2</span> were more tolerant to salinity and <span class="trait">drought</span> stresses compared with the control plants. Through expression analyses of putative downstream genes in the transgenic plants, we found that the expression levels of two ion antiporter genes AtNHX1 and AtCLCa, a key gene involved in the biosynthesis of proline, AtP5CS, and the copper chaperone for superoxide dismutase gene AtCCS, were all increased significantly in the transgenic plants. These results suggest that <span class="gene">GmUBC2</span> is involved in the regulation of ion homeostasis, osmolyte synthesis, and oxidative stress responses. Our results also suggest that modulation of the ubiquitination pathway could be an effective means of improving salt and <span class="trait">drought</span> tolerance in plants through genetic engineering.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						E2s were originally defined as proteins capable of accepting ubiquitin from an <span class="gene">E1</span> through a thioester linkage via a cysteinyl-sulfhydryl group (Glickman and Ciechanover <xref ref-type="bibr" rid="CR15"> 2002 </xref> ). The UBC domain also interacts with the E3 enzyme and, in some cases it also interacts with the substrate (Kalchman et al. <xref ref-type="bibr" rid="CR21"> 1996 </xref> ). Many genes encoding the ubiquitin-conjugating enzyme have been identified in eukaryotic organism. The yeast ( <italic> S. cerevisiae </italic> ) genome encodes 11 ubiquitin E2s and there are approximately 50 and 37 E2s in the human and <italic> Arabidopsis </italic> genome, respectively (Bachmair et al. <xref ref-type="bibr" rid="CR4"> 2001 </xref> ; Jiang and Beaudet <xref ref-type="bibr" rid="CR20"> 2004 </xref> ; Kraft et al. <xref ref-type="bibr" rid="CR25"> 2005 </xref> ). The <italic> RAD6 </italic> gene in <italic> S. cerevisiae </italic> encodes a 172-amino acid, 20-kDa E2 enzyme (Ubc2) and is required for post-replication repair of UV-damaged DNA, induced mutagenesis, and sporulation (Reynolds et al. <xref ref-type="bibr" rid="CR31"> 1985 </xref> ). Rad6 appears to have diverse functions since <italic> rad6 </italic> mutants display a slow-growth phenotype, show defectiveness in DNA damage-induced mutagenesis and are sensitive to killing by various DNA-damaging agents such as UV, ionizing radiations, and alkylating agents. rad6 diploids are also deficient in sporulation (Prakash et al. <xref ref-type="bibr" rid="CR30"> 1993 </xref> ). Mouse <italic> HR6A </italic> encodes an ubiquitin-conjugating enzyme highly homologous to RAD6 and <italic> mHR6A </italic> -deficient cells appear to have normal DNA damage <span class="trait">resistance</span> properties, but <italic> mHR6A </italic> knockout male and female mice display a small decrease in body weight (Roest et al. <xref ref-type="bibr" rid="CR32"> 2004 </xref> ). AtUBC2 of <italic> A. thaliana </italic> encodes a structural homologue of the <italic> RAD6 </italic> gene of <italic> S. cerevisiae </italic> with approximately 65% amino acid identity and it can partially rescue the UV sensitivity and reduced growth rate of rad6 mutants at elevated temperatures (Zwirn et al. <xref ref-type="bibr" rid="CR48"> 1997 </xref> ). The <italic> Atubc1-1 </italic> and <italic> Atubc2-1 </italic> double mutant shows a dramatically reduced number of rosette leaves, an early-flowering phenotype, and reduced transcript levels of a set of floral repressor genes, including Flowering Locus C (FLC), MADS Associated Flowering 4 (MAF4) and MAF5 (Xu et al. <xref ref-type="bibr" rid="CR42"> 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Although E2s have been studied in regulating various aspects of plant growth, development, and cell process in eukaryotes, little is known about its role in abiotic stress responses. In this study, we identified a soybean gene <italic> <span class="gene">GmUBC2</span> </italic> encoding an ubiquitin-conjugating enzyme. Expression of <italic> <span class="gene">GmUBC2</span> </italic> is induced by salt and <span class="trait">drought</span> stress. Our molecular and physiological results suggest that <italic> <span class="gene">GmUBC2</span> </italic> is involved in the regulation of ion homeostasis, osmolyte synthesis, oxidative stress and abiotic stress responses. We demonstrate that overexpression of <italic> <span class="gene">GmUBC2</span> </italic> in <italic> Arabidopsis </italic> conferred improved salt and <span class="trait">drought</span> tolerance by regulating the expression of a set of important stress-responsive genes.
					</p>
				
					<p>&nbsp;&nbsp;
						We performed quantitative RT-PCR analysis to examine the expression levels of <italic> <span class="gene">GmUBC2</span> </italic> in different tissues of soybean seedlings under abiotic stress conditions. <italic> <span class="gene">GmUBC2</span> </italic> transcripts increased in soybean seedlings treated with a solution containing 200&#x000a0;mM NaCl, 20% PEG, or 100&#x000a0;&#x003bc;M ABA for 6&#x000a0;h (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> a). The <italic> <span class="gene">GmUBC2</span> </italic> transcripts were detected in leaves, stems and roots of soybean plants without stress treatments, but the level of transcript was more abundant in roots than in stems and leaves (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> b). After the 6&#x000a0;h of treatment with 200&#x000a0;mM NaCl, the mRNA level of <italic> <span class="gene">GmUBC2</span> </italic> is about twofold higher in stems and leaves than that in the control, but the level in roots only slightly increased (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> b). These results suggest that the expression of <italic> <span class="gene">GmUBC2</span> </italic> is responsive to salt, osmotic stress, and exogenous ABA treatments. <fig id="Fig2"> <label> Fig.&#x000a0;2 </label> <caption> <p> Expression of <italic> <span class="gene">GmUBC2</span> </italic> is regulated by salt, <span class="trait">drought</span> and ABA treatments. The seeds were first germinated in vermiculite irrigated with water. After opening of the first trifoliate, the seedlings were transferred into Hoagland&#x02019;s solution supplemented with 200&#x000a0;mM NaCl, 20% polyethylene glycol (PEG), or 100&#x000a0;&#x003bc;M ABA for 6&#x000a0;h. Seedlings, roots, stems and leaves were collected for total RNA extraction and real time RT-PCR analysis. <bold> a </bold> <span class="gene">GmUBC2</span> is response to NaCl, <span class="trait">drought,</span> and exogenous ABA treatments. <bold> b </bold> <italic> <span class="gene">GmUBC2</span> </italic> is up-regulated in different organs under NaCl stress. The soybean actin gene was used as an internal control. Ck, untreated control. <italic> Error bars </italic> represent SD ( <italic> n </italic> &#x000a0;=&#x000a0;3)
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the molecular mechanisms underlying the enhanced tolerance to salt and <span class="trait">drought</span> by <italic> 35S-GmUBC2 </italic> plants, we examined transcript levels of a set of selected abiotic stress-responsive genes in WT and <italic> 35S-GmUBC2 </italic> plants grew under normal conditions by real time PCR. Because plants reestablish ion homeostasis under salt stress, several genes modulating ion homeostasis were selected. As shown in Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> , expression of several genes were elevated in the 35S-GmUBC2 transgenic plants compared to wild type plants, including the vacuolar antiporter gene <italic> AtNHX1 </italic> , <italic> AtCLCa </italic> gene encoding a NO <sub> 3 </sub> <sup> &#x02212; </sup> /H <sup> + </sup> exchanger (Angeli et al. <xref ref-type="bibr" rid="CR2"> 2006 </xref> ), At <italic> P5CS1 </italic> encoding a rate-limiting enzyme in the biosynthesis of Proline and <italic> AtCCS </italic> encoding a copper chaperone for superoxide dismutase(Strizhov et al. <xref ref-type="bibr" rid="CR37"> 1997 </xref> ; Chu et al. <xref ref-type="bibr" rid="CR10"> 2005 </xref> ). However, the levels of <italic> <span class="gene">SOS1</span> </italic> encoding a plasma membrane antiporter (Shi et al. <xref ref-type="bibr" rid="CR35"> 2002 </xref> ) and the stress-responsive transcription factor <italic> <span class="gene">DREB2A</span> </italic> and its target genes <italic> RD29b </italic> appear to be insensitive to <italic> <span class="gene">GmUBC2</span> </italic> abundance under normal growth conditions (Sakuma et al. <xref ref-type="bibr" rid="CR33"> 2006 </xref> ). <fig id="Fig7"> <label> Fig.&#x000a0;7 </label> <caption> <p> Expression of stress-responsive genes in <italic> 35S-GmUBC2 </italic> lines. Relative RNA levels of stress-responsive genes were determined by Real time PCR using total RNAs isolated from the shoots of 3-week-old plants grown under normal conditions in soil. Two independent assays were performed and similar results were obtained (each <italic> n </italic> &#x000a0;=&#x000a0;3). The <italic> Arabidopsis </italic> <italic> ACT3 </italic> (ATU39480) gene was used as a loading control
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we identified a soybean ubiquitin-conjugating enzyme gene, <italic> <span class="gene">GmUBC2</span> </italic> , which contains a conserved consensus E2 catalytic site with an invariant active-site cysteine of E2 enzymes (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> b). Previous phylogenetic analyses of UBC proteins in <italic> Arabidopsis </italic> , <italic> S. cerevisiae, </italic> and <italic> H. sapiens </italic> have classified AtUBC1 and AtUBC2 as well as AtUBC3 together with RAD6 (also named ScUBC2) and HsUBC2 as members of group III (Kraft et al. <xref ref-type="bibr" rid="CR25"> 2005 </xref> ). <span class="gene">GmUBC2</span> has high sequence similarity to AtUBC2 and RAD6 and phylogenetically clusters together with AtUBC2 and RAD6. These sequence analyses suggest that <span class="gene">GmUBC2</span> could be an active ubiquitin-conjugating enzyme as RAD6 and AtUBC2, further biochemical analysis is needed to further confirm whether <span class="gene">GmUBC2</span> binds ubiquitin via a thiol ester linkage. Previous studies have demonstrated that AtUBC1 and AtUBC2 have E2 activity in vitro (Sullivan and Vierstra <xref ref-type="bibr" rid="CR38"> 1993 </xref> ), that AtUBC2 could rescue the UV-sensitive phenotype of the yeast rad6 mutant (Zwirn et al. <xref ref-type="bibr" rid="CR48"> 1997 </xref> ), and that AtUBC1 and AtUBC2 have redundant functions in suppression of flowering (Xu et al. <xref ref-type="bibr" rid="CR42"> 2009 </xref> ). Here, we showed that <italic> <span class="gene">GmUBC2</span> </italic> played an important role in salt and <span class="trait">drought</span> stress responses by regulating stress-responsive genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Accumulating evidences suggest that ubiquitination may play an important role in plant tolerance against abiotic stresses (Yan et al. <xref ref-type="bibr" rid="CR44"> 2003 </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR46"> 2007 </xref> ; Lee et al. <xref ref-type="bibr" rid="CR26"> 2001 </xref> ; Lopez-Molina et al. <xref ref-type="bibr" rid="CR27"> 2003 </xref> ; Luo et al. <xref ref-type="bibr" rid="CR28"> 2006 </xref> ; Stone et al. <xref ref-type="bibr" rid="CR36"> 2006 </xref> ). The mRNA level of <italic> <span class="gene">GmUBC2</span> </italic> increased after exposure to salt and <span class="trait">drought</span> stress, suggesting that <italic> <span class="gene">GmUBC2</span> </italic> might be involved in plant tolerance against salt and <span class="trait">drought</span> stress. As a prove-of-concept study, we overexpressed the <italic> <span class="gene">GmUBC2</span> </italic> under the control of 35S CaMV promoter to examine its role in salt and <span class="trait">drought</span> stress in <italic> Arabidopsis </italic> given the technical difficulty of transforming soybean. The transgenic <italic> Arabidopsis </italic> plants were more tolerant to NaCl during the seedling stage. As shown in Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> , seedlings expressing <italic> <span class="gene">GmUBC2</span> </italic> grew in <span class="gene">MS</span> medium containing 200&#x000a0;mM NaCl for 7&#x000a0;days exhibited higher survival rates compared with the control plants. The root length of transgenic seedlings is longer than that of WT seedlings (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). In addition, our data indicated that plants overexpressing <italic> <span class="gene">GmUBC2</span> </italic> were more tolerant to <span class="trait">drought</span> compared to control plants (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). These results strongly suggest that <span class="gene">GmUBC2</span> is a positive regulator of salt and <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Salt and <span class="trait">drought</span> stress leads to accumulation of high levels of reactive oxygen species (ROS) (Borsani et al. <xref ref-type="bibr" rid="CR6"> 2001 </xref> ). The excessive production of ROS such as superoxide (O <sub> 2 </sub> <sup> &#x02212; </sup> ), hydrogen peroxide (H <sub> 2 </sub> O <sub> 2 </sub> ), and hydroxyl radicals (OH <sup> &#x02212; </sup> ) may disturb cellular redox homeostasis, leading to oxidative injuries. Superoxide dismutase (SOD) belongs to the enzymatic defense system against ROS and catalyzes the dismutation of superoxide radicals to molecular oxygen and hydrogen peroxide. The increased expression level of <italic> AtCCS </italic> suggests that SOD activity may increase in <italic> 35S-GmUBC2 </italic> plants. Indeed, we observed a higher SOD activity in transgenic plants compared with control plants with or without NaCl treatment (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ), suggesting that <span class="gene">GmUBC2</span> regulates SOD activity to scavenge ROS and is involved in oxidative stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						Many abiotic-stress-inducible genes are controlled by ABA, but some are not, which indicates that both ABA-dependent and ABA-independent regulatory systems are involved in stress-responsive gene expression. The ABA-dependent signaling pathway is mediated mainly by two types of transcription factors: one is ABRE-binding factors (ABFs); the other is the transcription factor <italic> AtMYC, </italic> which activates the expression of <italic> RD22 </italic> genes (Yamaguchi-Shinozaki and Shinozaki <xref ref-type="bibr" rid="CR43"> 2005 </xref> ). Although the <italic> P5CS1 </italic> gene responsive to salinity, <span class="trait">drought,</span> and ABA was affected in <italic> 35S-GmUBC2 </italic> transgenic plants, the expressions of <italic> ABF3 </italic> , <italic> ABF4 </italic> and <italic> AtMYC2 </italic> , and some of their target genes (such as <italic> RD22 </italic> ), were not affected in <italic> 35S-GmUBC2 </italic> plants (data not shown). A substantial number of studies have reported that overexpression of genes in the ABA signal pathway such as ABF3, ABF4, AtMYC2 and SDIR1, confer several ABA- or stress-associated phenotypes in transgenic plants, such as ABA hypersensitivity, sugar hypersensitivity, salt sensitivity and enhanced <span class="trait">drought</span> tolerance (Kang et al. <xref ref-type="bibr" rid="CR22"> 2002 </xref> ; Abe et al. <xref ref-type="bibr" rid="CR1"> 2003 </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR46"> 2007 </xref> ). However, <italic> 35S-GmUBC2 </italic> transgenic plants were insensitive to various ABA concentrations (data not shown) and had enhanced tolerance to salt stress at seedling stages (Figs.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> , <xref rid="Fig5" ref-type="fig"> 5 </xref> ). In addition, two genes of ABA-independent signaling pathway regulated by DREB2A, <italic> Rd29b, </italic> and <italic> <span class="gene">DREB2A</span> </italic> (Sakuma et al. <xref ref-type="bibr" rid="CR33"> 2006 </xref> ), showed no significant changes in <italic> 35S-GmUBC2 </italic> plants. Therefore, our results suggest that <span class="gene">GmUBC2</span> could be involved in ABA-independent signaling pathway in the response to dehydration stress. However, it is also possible that the expressed levels of <italic> <span class="gene">GmUBC2</span> </italic> in <italic> 35S-GmUBC2 </italic> lines might not be enough to trigger the expression of various ABA-responsive genes even though <span class="gene">GmUBC2</span> could be involved in ABA-dependent pathway. Further studies are required to definitely resolve this issue.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Allantoate amidohydrolase transcript expression is independent of <span class="trait">drought</span> tolerance in soybean.</h2>
			<h3>PMID:19129162  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Charlson DV(1), Korth KL, Purcell LC.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a limiting factor for N(2) fixation in soybean [Glycine max (L.) Merr.] thereby resulting in reduced biomass accumulation and <span class="trait">yield.</span> Drought-sensitive genotypes accumulate ureides, a product of N(2) fixation, during <span class="trait">drought</span> stress; however, drought-tolerant genotypes have lower shoot ureide concentrations, which appear to alleviate <span class="trait">drought</span> stress on N(2) fixation. A key enzyme involved in ureide breakdown in shoots is allantoate amidohydrolase (AAH). It is hypothesized that <span class="gene">AAH</span> gene expression in soybean determines shoot ureide concentrations during water-deficit stress and is responsible for the differential sensitivities of the N(2)-fixation response to <span class="trait">drought</span> among soybean genotypes. The objectives were to examine the relationship between <span class="gene">AAH</span> transcript levels and shoot ureide concentration and <span class="trait">drought</span> tolerance. Drought-tolerant (Jackson) and drought-sensitive (Williams) genotypes were subjected to three water-availability treatments: well-watered control, moderate water-deficit stress, and severe water-deficit stress. Shoot ureide concentrations were examined, in addition to gene expression of <span class="gene">AAH</span> and DREB2, a gene expressed during water-deficit stress. As expected, <span class="gene">DREB2</span> expression was detected only during severe water-deficit stress, and shoot ureide concentrations were greatest in the drought-sensitive genotype relative to the drought-tolerant genotype during water-deficit stress. However, expression of <span class="gene">AAH</span> transcripts was similar among water treatments and genotypes, indicating that <span class="gene">AAH</span> mRNA was not closely associated with <span class="trait">drought</span> tolerance. Ureide concentrations in shoots were weakly associated with <span class="gene">AAH</span> mRNA levels. These results indicate that <span class="gene">AAH</span> expression is probably not associated with the increased ureide catabolism observed in drought-tolerant genotypes, such as Jackson. Further study of <span class="gene">AAH</span> at the post-translational and enzymatic levels is warranted in order to dissect the potential role of this gene in <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induced over-expression of AtDREB2A CA improves <span class="trait">drought</span> tolerance in sugarcane.</h2>
			<h3>PMID:24656336  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reis RR(1), da Cunha BA(2), Martins PK(2), Martins MT(2), Alekcevetch JC(3), Chalfun A Jr(3), Andrade AC(4), Ribeiro AP(1), Qin F(5), Mizoi J(6), Yamaguchi-Shinozaki K(6), Nakashima K(5), Carvalho Jde F(7), de Sousa CA(2), Nepomuceno AL(7), Kobayashi AK(2), Molinari HB(8).
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is one of the most challenging agricultural issues limiting sustainable sugarcane production and, in some cases, <span class="trait">yield</span> losses caused by <span class="trait">drought</span> are nearly 50%. DREB proteins play vital regulatory roles in abiotic stress responses in plants. The transcription factor <span class="gene">DREB2A</span> interacts with a cis-acting DRE sequence to activate the expression of downstream genes that are involved in <span class="trait">drought-,</span> salt- and heat-stress response in Arabidopsis thaliana. In the present study, we evaluated the effects of stress-inducible over-expression of AtDREB2A CA on gene expression, leaf water potential (ΨL), relative water content (RWC), sucrose content and gas exchanges of sugarcane plants submitted to a four-days water deficit treatment in a rhizotron-grown root system. The plants were also phenotyped by scanning the roots and measuring morphological parameters of the shoot. The stress-inducible expression of AtDREB2A CA in transgenic sugarcane led to the up-regulation of genes involved in plant response to <span class="trait">drought</span> stress. The transgenic plants maintained higher RWC and ΨL over 4 days after withholding water and had higher photosynthetic rates until the 3rd day of water-deficit. Induced expression of AtDREB2A CA in sugarcane increased sucrose levels and improved bud sprouting of the transgenic plants. Our results indicate that induced expression of AtDREB2A CA in sugarcane enhanced its <span class="trait">drought</span> tolerance without biomass penalty.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of soybean R2R3-MYB transcription factor, <span class="gene">GmMYB12B2,</span> and tolerance to UV radiation and salt stress in transgenic Arabidopsis.</h2>
			<h3>PMID:27323089  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li XW(1)(2), Wang Y(1), Yan F(1), Li JW(1), Zhao Y(3), Zhao X(4), Zhai Y(3), Wang QY(1).
				<br/> 
				年份：2016 May 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB, v-myb avian myeloblastosis viral oncogene homolog, proteins play central roles in plant stress response. Previously, we identified a novel R2R3-MYB transcription factor, <span class="gene">GmMYB12B2,</span> which affected the expression levels of some key enzyme genes involved in flavonoid biosynthesis in transgenic Arabidopsis. In the present study, we analyzed the expression levels of <span class="gene">GmMYB12B2</span> under salt, low temperature, <span class="trait">drought,</span> abscisic acid (ABA), and ultraviolet (UV) radiation treatments in soybean using semi-quantitative reverse transcription polymerase chain reaction. The expression of <span class="gene">GmMYB12B2</span> was drastically induced by UV irradiation and salt treatment, but no response was detected under low temperature, <span class="trait">drought,</span> and ABA stresses. A detailed characterization of the <span class="gene">GmMYB12B2</span> overexpression lines revealed that <span class="gene">GmMYB12B2</span> might be involved in response of plants to UV radiation and salt stresses. Transgenic Arabidopsis lines constitutively expressing <span class="gene">GmMYB12B2</span> showed an increased tolerance to salt and UV radiation treatment compared with wild-type plants. The expression levels of certain salt stress-responsive genes, such as <span class="gene">DREB2A</span> and RD17, were found to be elevated in the transgenic plants. These results indicate that <span class="gene">GmMYB12B2</span> acts as a regulator in the plant stress response.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">GmDi19-5</span> interacts with <span class="gene">GmLEA3.1</span> and increases sensitivity of transgenic plants to abiotic stresses</h2>
			<h3>PMID:25852726</h3>
				

				
					<h3>PMCID:4371698</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Feng </surname>  <given-names> Zhi-Juan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Xiao-Yu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Xi-Yan </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Guang-Xiao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> You-Zhi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Guang-Yuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/213018"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao-Shi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/203437"/>  </contrib>
				<br/> 
				年份：<day> 05 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Drought-induced (Di19) proteins played important roles in plant growth, development, and abiotic stress responses. In the present study, a total of seven Di19 genes were identified in soybean. Each soybean Di19 gene showed specific responses to salt, <span class="trait">drought,</span> oxidative, and ABA stresses based on expression profiles. With a relatively higher transcript level among Di19 members under four stress treatments, <span class="gene">GmDi19-5</span> was selected for detailed analysis. Inhibitor assays revealed that ABA inhibitor (Fluridone) or H2O2 inhibitor (DMTU) was involved in the <span class="trait">drought-</span> or salt-induced transcription of <span class="gene">GmDi19-5.</span> The GUS activity driven by the <span class="gene">GmDi19-5</span> promoter was induced by salt, PEG, ABA, and MV treatments and tended to be accumulated in the vascular bundles and young leaves. A subcellular localization assay showed that <span class="gene">GmDi19-5</span> protein localized in the nucleus. Further investigation showed that <span class="gene">GmDi19-5</span> protein was involved in the interaction with <span class="gene">GmLEA3.1.</span> Overexpression of <span class="gene">GmDi19-5</span> increased sensitivity of transgenic Arabidopsis plants to salt, <span class="trait">drought,</span> oxidative, and ABA stresses and regulated expression of several ABA/stress-associated genes. This present investigation showed that <span class="gene">GmDi19-5</span> functioned as a negative factor under abiotic stresses and was involved in ABA and SOS signaling pathway by altering transcription of stress-associated genes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the effect of abiotic stresses on <italic> Di19s </italic> , expression pattern was measured by qRT-PCR in soybean seedlings subjected to salt, <span class="trait">drought,</span> H <sub> 2 </sub> O <sub> 2 </sub> ,and ABA treatments. As shown in Figure <xref ref-type="fig" rid="F4"> 4 </xref> , there were wide variations in accumulation of different mRNAs following different types of stress. For salt treatment most of the transcript levels of <italic> Di19 </italic> genes increased and remained constant during the first 12 h of treatment. <italic> <span class="gene">GmDi19-5</span> </italic> was upregulated more than 10-fold after 12 h of salt treatment (Figure <xref ref-type="fig" rid="F4"> 4A </xref> ). <span class="trait">Drought</span> and H <sub> 2 </sub> O <sub> 2 </sub> stresses also significantly modulated the expressions of most <italic> Di19 </italic> genes. As shown in Figure <xref ref-type="fig" rid="F4"> 4B </xref> , transcripts of <italic> <span class="gene">GmDi19-5</span> </italic> and <italic> <span class="gene">GmDi19-6</span> </italic> were increased by more than 5-fold after 5 h of <span class="trait">drought</span> stress. Figure <xref ref-type="fig" rid="F4"> 4C </xref> showed that transcript levels of <italic> Di19 </italic> genes, except <italic> <span class="gene">GmDi19-2</span> </italic> , instantaneously increased by more than 2-fold with H <sub> 2 </sub> O <sub> 2 </sub> treatment; after 5 h <italic> <span class="gene">GmDi19-3</span> </italic> and <italic> <span class="gene">GmDi19-6</span> </italic> were upregulated by more than 5-fold. With ABA treatment, <italic> Di19 </italic> transcript levels were increased more than 3-fold, except for <italic> <span class="gene">GmDi19-6</span> </italic> (Figure <xref ref-type="fig" rid="F4"> 4D </xref> ). The transcript of <italic> <span class="gene">GmDi19-5</span> </italic> was increased more than 6-fold at 0.5 h of ABA stress.
					</p>
				
					<p>&nbsp;&nbsp;
						To explore whether ABA and H <sub> 2 </sub> O <sub> 2 </sub> were involved in up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under <span class="trait">drought</span> and salt stresses, fluridone, and DMTU were chosen as ABA and H <sub> 2 </sub> O <sub> 2 </sub> inhibitors, respectively (Hu et al., <xref rid="B19" ref-type="bibr"> 2012 </xref> ; Ma et al., <xref rid="B35" ref-type="bibr"> 2014 </xref> ; Yoshida et al., <xref rid="B59" ref-type="bibr"> 2014 </xref> ; You et al., <xref rid="B60" ref-type="bibr"> 2014 </xref> ; Zhang et al., <xref rid="B61" ref-type="bibr"> 2014 </xref> ). Treatment with fluridone or DMTU had no effect on transcript of <italic> <span class="gene">GmDi19-5</span> </italic> under normal treatments (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Pretreatment with the fluridone inhibitor partially prevented up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under NaCl, while fluridone was not so effective in block of up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under <span class="trait">drought</span> stress in soybean. Pretreatment with the DMTU inhibitor partially prevented up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under NaCl or <span class="trait">drought</span> stresses in soybean. Therefore, ABA was involved in up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under salt stresses, and H <sub> 2 </sub> O <sub> 2 </sub> was involved in up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> under both salt and <span class="trait">drought</span> stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Effect of salt, <span class="trait">drought,</span> oxidative and ABA stresses on seed germination of <italic> <span class="gene">GmDi19-5</span> </italic> transgenic and WT seeds. (A) </bold> Germination rate of WT and transgenic seeds on <span class="gene">MS</span> medium with or without NaCl, PEG, MV, and ABA. <bold> (B) </bold> Statistics analysis of germination rate. Mean values and standard errors (bar) were shown from three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Effect of salt, <span class="trait">drought,</span> oxidative, and ABA stresses on root length of <italic> <span class="gene">GmDi19-5</span> </italic> transgenic and WT seedlings. (A) </bold> Root length of seedlings were transferred to a medium with or without NaCl, PEG, MV, and ABA before the images were taken. Statistics analysis of root length <bold> (B) </bold> and fresh weight <bold> (C) </bold> . Mean values and standard errors (bar) were shown from three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						Di19 was involved in multiple stress responses in plants (Milla et al., <xref rid="B36" ref-type="bibr"> 2006 </xref> ; Parkinson et al., <xref rid="B40" ref-type="bibr"> 2009 </xref> ; Li et al., <xref rid="B29" ref-type="bibr"> 2010a </xref> , <xref rid="B30" ref-type="bibr"> b </xref> ; Liu et al., <xref rid="B33" ref-type="bibr"> 2013b </xref> ). <italic> Arabidopsis Di19 </italic> family genes displayed various transcriptional regulations in response to specific abiotic stress, including cold, <span class="trait">drought,</span> osmotic, oxidative, and salt stresses, in roots or shoots (Supplementary Figure <xref ref-type="supplementary-material" rid="SM13"> S3 </xref> ; Supplementary Table <xref ref-type="supplementary-material" rid="SM10"> S7 </xref> ). Promoter analysis showed there were many abiotic stress-related elements in the upstream region of soybean <italic> Di19 </italic> family genes (Supplementary Table <xref ref-type="supplementary-material" rid="SM9"> S6 </xref> ). Expression profiles analysis revealed that transcripts of soybean <italic> Di19 </italic> genes were increased by salt, <span class="trait">drought,</span> oxidative, and ABA stresses (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). Meanwhile, GUS activity driven by the <italic> <span class="gene">GmDi19-5</span> </italic> promoter tended to increase under NaCl, PEG, ABA, and MV treatments (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). It was reported that several genes responsive to salt and <span class="trait">drought</span> might be involved in ABA and H <sub> 2 </sub> O <sub> 2 </sub> signaling pathways that were conserved components of these stress signal pathways (Hu et al., <xref rid="B19" ref-type="bibr"> 2012 </xref> ; Ma et al., <xref rid="B35" ref-type="bibr"> 2014 </xref> ; Yoshida et al., <xref rid="B59" ref-type="bibr"> 2014 </xref> ; You et al., <xref rid="B60" ref-type="bibr"> 2014 </xref> ; Zhang et al., <xref rid="B61" ref-type="bibr"> 2014 </xref> ). Pretreatment with the inhibitors of these pathways prevented up-regulation of <italic> <span class="gene">GmDi19-5</span> </italic> in NaCl- and drought-treated soybean seedlings (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Thus, it is plausible that <italic> <span class="gene">GmDi19-5</span> </italic> could be part of the ABA- and ROS-mediation process. However, more research is needed to confirm this mechanism through other biological technologies.
					</p>
				
					<p>&nbsp;&nbsp;
						It was demonstrated that salt and <span class="trait">drought</span> stresses were able to induce ABA biosynthesis and trigger ABA-dependent signaling pathways, and ABA could maintain seed dormancy, prevented germination and inhibited seedling growth (Finkelstein et al., <xref rid="B14" ref-type="bibr"> 1998 </xref> , <xref rid="B13" ref-type="bibr"> 2002 </xref> ; Leung and Giraudat, <xref rid="B28" ref-type="bibr"> 1998 </xref> ; Achard et al., <xref rid="B1" ref-type="bibr"> 2006 </xref> ; Piskurewicz et al., <xref rid="B41" ref-type="bibr"> 2008 </xref> ; Bari and Jones, <xref rid="B2" ref-type="bibr"> 2009 </xref> ). In cotton, Di19-like genes <italic> GhDi19-1 </italic> and <italic> GhDi19-2 </italic> were identified to be positive nuclear regulators in the ABA signaling pathway (Li et al., <xref rid="B29" ref-type="bibr"> 2010a </xref> ). Wheat <italic> TaDi19A </italic> was positively regulated by ABA signaling and was responsive to NaCl and high osmotic stress (Li et al., <xref rid="B30" ref-type="bibr"> 2010b </xref> ). <italic> <span class="gene">GmDi19-5</span> </italic> affected the expression of genes related to ABA/stress signaling in transgenic plants, including <italic> ABF3, ABF4, ABI1, ABI5, RAB18 </italic> , and <italic> SOS2 </italic> genes (Figure <xref ref-type="fig" rid="F11"> 11 </xref> ). <italic> ABF3 </italic> and <italic> ABF4 </italic> were two bZIP transcription factors which positively regulated ABA signaling. The constitutive overexpression of <italic> ABF3 </italic> or <italic> ABF4 </italic> in <italic> Arabidopsis </italic> exhibited enhanced <span class="trait">drought</span> tolerance and ABA hypersensitivity (Kang et al., <xref rid="B22" ref-type="bibr"> 2002 </xref> ). The relative expression levels of these two genes were higher after ABA treatment. <italic> ABI1 </italic> encoded one ser/thr protein phosphatase of the PP2C family that acted as negative regulator of ABA response (Koornneef et al., <xref rid="B26" ref-type="bibr"> 1989 </xref> ). <italic> ABI5 </italic> was one bZIP transcription factor, positively regulating ABA-mediated control of seed germination and early seedling development (Leung and Giraudat, <xref rid="B28" ref-type="bibr"> 1998 </xref> ). The level of <italic> ABI1 </italic> and <italic> ABI5 </italic> were significantly increased under ABA treatment. The transcription level of known ABA up-regulated gene <italic> RAB18 </italic> was also elevated in <italic> <span class="gene">GmDi19-5</span> </italic> transgenic plants. These findings suggested that ABA was more effectively synthesized in transgenic plants than in WT plants. <italic> <span class="gene">GmDi19-5</span> </italic> might act a positive regulator in ABA response. In addition, <italic> ABF3, ABF4 </italic> , and <italic> SOS2 </italic> were significantly low in <italic> <span class="gene">GmDi19-5</span> </italic> transgenic plants after NaCl and PEG treatments. Overexpression of <italic> <span class="gene">GmDi19-5</span> </italic> displayed increased sensitivities to salt, <span class="trait">drought,</span> and oxidative stress during the germination and seedling stages in <italic> Arabidopsis </italic> (Figures <xref ref-type="fig" rid="F7"> 7 </xref> , <xref ref-type="fig" rid="F8"> 8 </xref> ). Recently, it was reported that <italic> AtDi19-3 </italic> displayed <span class="trait">drought-</span> and salt-sensitive phenotype in overexpression lines (Qin et al., <xref rid="B42" ref-type="bibr"> 2014 </xref> ). Similar to <italic> AtDi19-3, <span class="gene">GmDi19-5</span> </italic> might act as a negative regulator in response to salt and <span class="trait">drought</span> stresses (Qin et al., <xref rid="B42" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Zinc-finger domain is involved in protein-DNA interactions, protein-protein interactions between cytoskeleton dystrophin and calmodulin, and between transcriptional adapters (such as CREB-binding proteins and p300) and transcriptional activators (Dure, <xref rid="B11" ref-type="bibr"> 1993 </xref> ; Davies et al., <xref rid="B10" ref-type="bibr"> 1996 </xref> ; Takatsuji, <xref rid="B52" ref-type="bibr"> 1999 </xref> ; Searles et al., <xref rid="B47" ref-type="bibr"> 2000 </xref> ; Wolfe et al., <xref rid="B55" ref-type="bibr"> 2000 </xref> ). The Cys2/His2 zinc finger-like domain may have a similar function in protein-protein interaction. LEA proteins possessed high hydrophilicity and chaperone-like activity and involved in stress <span class="trait">resistance</span> in plants, such as chilling, <span class="trait">drought,</span> and high salinity (Battaglia et al., <xref rid="B4" ref-type="bibr"> 2008 </xref> ; Bies-Etheve et al., <xref rid="B5" ref-type="bibr"> 2008 </xref> ; Battaglia and Covarrubias, <xref rid="B3" ref-type="bibr"> 2013 </xref> ). Physical interaction with other proteins was one of the important features of LEAs, such as between hydrophilic proteins and LEA proteins (Hanin et al., <xref rid="B16" ref-type="bibr"> 2011 </xref> ; Cuevas-Velazquez et al., <xref rid="B9" ref-type="bibr"> 2014 </xref> ). Maize LEA protein Rab17 interacted with and was phosphorylated protein kinase CK2 (Riera et al., <xref rid="B44" ref-type="bibr"> 2004 </xref> ). <italic> Opuntia streptacantha </italic> LEA proteins were shown to form dimer (Hern&#x000e1;ndez-S&#x000e1;nchez et al., <xref rid="B17" ref-type="bibr"> 2014 </xref> ). LEAs belonged to a multigene family, which was classified into seven groups based on expression patterns and sequences. Recently, there were extensive correlative data linking the expression of Group 3 LEA (LEA3) proteins with tolerance suggested various biochemical mechanisms for LEA3s, such as the repair of improperly folded proteins as a chaperone, the binding of metal ions, the stabilization of membrane structure and the increase of cellular mechanical strength through the generation of filaments (Browne et al., <xref rid="B8" ref-type="bibr"> 2002 </xref> ; Tolleter et al., <xref rid="B53" ref-type="bibr"> 2007 </xref> ; Tunnacliffe and Wise, <xref rid="B54" ref-type="bibr"> 2007 </xref> ). We found that <span class="gene">GmDi19-5</span> interacted with <span class="gene">GmLEA3.1</span> (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ), thus we might suppose that <span class="gene">GmLEA3.1</span> protein might improve the stability of <span class="gene">GmDi19-5</span> protein.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Trihelix Transcription Factors <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> Improve Plant Tolerance to Abiotic Stresses in Transgenic Arabidopsis</h2>
			<h3>PMID:19730734</h3>
				

				
					<h3>PMCID:2731930</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xie </surname>  <given-names> Zong-Ming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zou </surname>  <given-names> Hong-Feng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lei </surname>  <given-names> Gang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Qi-Yun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Niu </surname>  <given-names> Can-Fang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Ai-Guo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Wan-Ke </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jin-Song </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shou-Yi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 8 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trihelix transcription factors play important roles in light-regulated responses and other developmental processes. However, their functions in abiotic stress response are largely unclear. In this study, we identified two trihelix transcription factor genes <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> from soybean and further characterized their roles in abiotic stress tolerance. FINDINGS: Both genes can be induced by various abiotic stresses, and the encoded proteins were localized in nuclear region. In yeast assay, <span class="gene">GmGT-2B</span> but not <span class="gene">GmGT-2A</span> exhibits ability of transcriptional activation and dimerization. The N-terminal peptide of 153 residues in <span class="gene">GmGT-2B</span> was the minimal activation domain and the middle region between the two trihelices mediated the dimerization of the <span class="gene">GmGT-2B.</span> Transactivation activity of the <span class="gene">GmGT-2B</span> was also confirmed in plant cells. DNA binding analysis using yeast one-hybrid assay revealed that <span class="gene">GmGT-2A</span> could bind to GT-1bx, GT-2bx, mGT-2bx-2 and D1 whereas <span class="gene">GmGT-2B</span> could bind to the latter three elements. Overexpression of the <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> improved plant tolerance to salt, freezing and <span class="trait">drought</span> stress in transgenic Arabidopsis plants. Moreover, <span class="gene">GmGT-2B-transgenic</span> plants had more green seedlings compared to Col-0 under ABA treatment. Many stress-responsive genes were altered in <span class="gene">GmGT-2A-</span> and <span class="gene">GmGT-2B-transgenic</span> plants. CONCLUSION: These results indicate that <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> confer stress tolerance through regulation of a common set of genes and specific sets of genes. <span class="gene">GmGT-2B</span> also affects ABA sensitivity.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Both genes can be induced by various abiotic stresses, and the encoded proteins were localized in nuclear region. In yeast assay, <span class="gene">GmGT-2B</span> but not <span class="gene">GmGT-2A</span> exhibits ability of transcriptional activation and dimerization. The N-terminal peptide of 153 residues in <span class="gene">GmGT-2B</span> was the minimal activation domain and the middle region between the two trihelices mediated the dimerization of the <span class="gene">GmGT-2B.</span> Transactivation activity of the <span class="gene">GmGT-2B</span> was also confirmed in plant cells. DNA binding analysis using yeast one-hybrid assay revealed that <span class="gene">GmGT-2A</span> could bind to GT-1bx, GT-2bx, mGT-2bx-2 and D1 whereas <span class="gene">GmGT-2B</span> could bind to the latter three elements. Overexpression of the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> improved plant tolerance to salt, freezing and <span class="trait">drought</span> stress in transgenic Arabidopsis plants. Moreover, <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic plants had more green seedlings compared to Col-0 under ABA treatment. Many stress-responsive genes were altered in <italic> <span class="gene">GmGT-2A</span> </italic> - and <italic> <span class="gene">GmGT-2B-</span> </italic> transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean seedlings were treated with various stresses and the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> gene expressions were investigated. Both gene expressions can be enhanced by ABA, cold, <span class="trait">drought</span> and salt treatments ( <xref ref-type="fig" rid="pone-0006898-g002"> Fig. 2B </xref> ). However, the expression patterns were different. The <italic> <span class="gene">GmGT-2A</span> </italic> transcripts accumulated to a higher level at 12 h after initiation of the treatments with ABA, cold or <span class="trait">drought</span> stress, whereas the <italic> <span class="gene">GmGT-2B</span> </italic> expression had peak levels at 1 h and/or 3 h after the three treatments. The upregulation of <italic> <span class="gene">GmGT-2A</span> </italic> expression after 12-h ABA treatment may be an indirect effect. At 12 h after these treatments, the <italic> <span class="gene">GmGT-2B</span> </italic> expression was apparently declined ( <xref ref-type="fig" rid="pone-0006898-g002"> Fig. 2B </xref> ). Under salt stress, the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> inductions were similar in patterns. These results suggest that the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> may be involved in regulation of plant responses to abiotic stresses. Because the <italic> GT-2 </italic> gene showed similarity to the <italic> <span class="gene">GT-2A</span> </italic> and <italic> <span class="gene">GT-2B</span> </italic> , we further tested if this gene is responsive to stresses. However, after the four treatments, the <italic> GT-2 </italic> expression was not significantly altered, indicating that the gene may not be involved in abiotic stress responses ( <xref ref-type="fig" rid="pone-0006898-g002"> Fig. 2B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Because expression of the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> genes can be induced under <span class="trait">drought</span> and cold treatments, we investigated if the performance of their transgenic plants was altered under these stresses. Plants (12-day-old) in pots were subjected to <span class="trait">drought</span> stress by withholding water for 16 d. After this stress, only &#x0223c;40&#x00025; of the wild type plants survived whereas more than 90&#x00025; of the <italic> <span class="gene">GmGT-2A</span> </italic> - and <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic plants can survive and grow well ( <xref ref-type="fig" rid="pone-0006898-g006"> Fig. 6A, B </xref> ). All the plants under normal condition grew normally ( <xref ref-type="fig" rid="pone-0006898-g006"> Fig. 6A </xref> ). Plants that start to have siliques were further treated under <span class="trait">drought</span> stress by withholding water from plants for 16 d. The aerial parts of these plants were harvested and the dry weights of the transgenic plants were significantly higher than that of the wild type plants ( <xref ref-type="fig" rid="pone-0006898-g006"> Fig. 6C </xref> ). These results indicate that the <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> confer <span class="trait">drought</span> tolerance to the transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Water loss represents another parameter for estimation of the plant tolerance to <span class="trait">drought</span> stress. Transgenic plants overexpressing the <italic> <span class="gene">GmGT-2A</span> </italic> or <italic> <span class="gene">GmGT-2B</span> </italic> all had reduced water loss in comparison with the wild type plants when detached leaves were used for the desiccation analysis ( <xref ref-type="fig" rid="pone-0006898-g006"> Fig. 6D </xref> ). The whole plants (six-week-old) were also withheld from water for various times and the aerial parts were measured for water content. <xref ref-type="fig" rid="pone-0006898-g006"> Figure 6E </xref> showed that, at day 8 and day 12 after the treatment, the water content was significantly higher in one <italic> <span class="gene">GmGT-2A</span> </italic> -transgenic line and two <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic lines compared to the control plants, suggesting that these transgenic plants are tolerant to <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> may function in plant stress tolerance through regulation of downstream genes. We selected 33 stress-responsive genes for further quantitative real-time PCR analysis. <xref ref-type="fig" rid="pone-0006898-g007"> Figure 7 </xref> showed that 17 genes including <italic> AZF1, MYB74, MYB75, PAD3, LTP3, STZ, MYB73, LTP4, At4g30650, UGT71B6, RHL41, COR13, MYB77, CYP707A3, LTI30, RCI3 </italic> , and <italic> <span class="gene">DREB2A</span> </italic> were enhanced in all the transgenic lines harboring the <italic> <span class="gene">GmGT-2A</span> </italic> or <italic> <span class="gene">GmGT-2B</span> </italic> . The <italic> AZF1 </italic> , <italic> STZ </italic> and <italic> RHL41/Zat12 </italic> encoded plant-specific transcription factors with Cys-2/His-2 zinc finger motifs, and can be induced by various stresses. Overexpression of the STZ, Zat10 and Zat12 increased stress tolerance <xref ref-type="bibr" rid="pone.0006898-Davletova1"> &#x0005b;37&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0006898-Mittler1"> &#x0005b;39&#x0005d; </xref> . The four MYB genes <italic> MYB73, MYB74, <span class="gene">MYB75</span> </italic> and <italic> <span class="gene">MYB77</span> </italic> can be induced by salt stress <xref ref-type="bibr" rid="pone.0006898-Kamei1"> &#x0005b;40&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0006898-Chen1"> &#x0005b;41&#x0005d; </xref> . The <italic> LTP3, LTP4, PAD3 </italic> and UGT71B6 were involved in ABA responses <xref ref-type="bibr" rid="pone.0006898-Aronde1"> &#x0005b;42&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0006898-Priest1"> &#x0005b;44&#x0005d; </xref> . The At4g30650, <italic> COR13 </italic> and <italic> LTI30 </italic> were also involved in abiotic stress response <xref ref-type="bibr" rid="pone.0006898-Gong1"> &#x0005b;45&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0006898-Nylander1"> &#x0005b;46&#x0005d; </xref> . <italic> CYP707A3 </italic> encodes an ABA 8&#x02032;-hydroxylase. The <italic> cyp707a3 </italic> mutant plants are hypersensitive to ABA and exhibited enhanced <span class="trait">drought</span> tolerance <xref ref-type="bibr" rid="pone.0006898-Umezawa1"> &#x0005b;47&#x0005d; </xref> . <italic> RCI3 </italic> encodes a peroxidase and overexpression of the gene conferred dehydration and salt tolerance <xref ref-type="bibr" rid="pone.0006898-Llorente1"> &#x0005b;48&#x0005d; </xref> . <span class="gene">DREB2A</span> expression does not activate downstream genes under normal growth condition. However, overexpression of its constitutive active form leads to <span class="trait">drought</span> stress tolerance and slight freezing tolerance <xref ref-type="bibr" rid="pone.0006898-Sakuma1"> &#x0005b;49&#x0005d; </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> showed differential expression in response to ABA and other stresses. However, expression of their homologue <italic> <span class="gene">GmGT-2</span> </italic> was not induced by these treatments, suggesting that the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> have specific roles in abiotic stress responses. Overexpression of both the <italic> <span class="gene">GmGT-2A</span> </italic> and <italic> <span class="gene">GmGT-2B</span> </italic> greatly improved plant tolerance to salt, freezing and <span class="trait">drought</span> stresses as can be seen from the survival rates of the transgenic plants, the dry weight and the water loss ( <xref ref-type="fig" rid="pone-0006898-g005"> Fig. 5 </xref> , <xref ref-type="fig" rid="pone-0006898-g006"> 6 </xref> ). The regulation of stress tolerance may be achieved through control at levels of transcriptional activation, DNA binding, and dimerization and/or by other unknown mechanisms. These controls at different levels will finally affect gene expressions, through which the stress tolerance can be achieved in plants. Actually, many genes have been found to be upregulated in the transgenic plants. Among these, three C2/H2 zinc finger-type transcription factors were increased and may play significant roles since two of the genes <italic> STZ </italic> and <italic> RHL41/Zat12 </italic> have been found to promote stress tolerance <xref ref-type="bibr" rid="pone.0006898-Davletova1"> &#x0005b;37&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0006898-Sakamoto1"> &#x0005b;38&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0006898-Vogel1"> &#x0005b;51&#x0005d; </xref> . <italic> <span class="gene">DREB2A</span> </italic> gene and a peroxidase gene <italic> RCI3 </italic> were also highly expressed in the <italic> <span class="gene">GmGT-2A</span> </italic> - and <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic plants. These two genes have been found to improve stress tolerance in transgenic plants <xref ref-type="bibr" rid="pone.0006898-Llorente1"> &#x0005b;48&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0006898-Sakuma1"> &#x0005b;49&#x0005d; </xref> . Therefore, the <span class="gene">GmGT-2A</span> and <span class="gene">GmGT-2B</span> may at least activate these gene expressions either through direct binding to promoter of each gene or in a manner of indirect regulation. Expression of a few genes were also suppressed by the two <italic> <span class="gene">GmGT</span> </italic> genes. The <italic> VAMP711 </italic> , a gene encoding a protein related to vesicle trafficking, was downregulated. Suppression of the <italic> VAMP711 </italic> inhibited the fusion of the H <sub> 2 </sub> O <sub> 2 </sub> -containing vesicle to the tonoplast, leading to improved vacuolar functions for plant salt tolerance <xref ref-type="bibr" rid="pone.0006898-Leshem1"> &#x0005b;56&#x0005d; </xref> . It is therefore possible that the present two <span class="gene">GmGT</span> genes conferred stress tolerance at least through activation of STZ/ZAT and <span class="gene">DREB2A</span> transcription factors as well as the antioxidative system. Moreover, the <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic plants appeared to be slightly more tolerant to stresses than the <italic> <span class="gene">GmGT-2A</span> </italic> -transgenic plants did ( <xref ref-type="fig" rid="pone-0006898-g005"> Fig. 5B </xref> , <xref ref-type="fig" rid="pone-0006898-g006"> 6G </xref> ). This difference is most likely due to the higher expressions of the specific genes in <italic> <span class="gene">GmGT-2B</span> </italic> -transgenic plants ( <xref ref-type="fig" rid="pone-0006898-g007"> Fig. 7 </xref> ). It should be mentioned that overexpression of the transcription factors in Arabidopsis plants might induce tolerance observations not related to activation of specific pathways but rather indirect or pleiotropic effects. Further transgenic analysis in soybean plants may disclose such a possibility.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of the <italic> <span class="gene">GmGT-2B</span> </italic> gene resulted in reduced ABA sensitivity ( <xref ref-type="fig" rid="pone-0006898-g004"> Fig. 4 </xref> ), but still improved plant tolerance to salt, <span class="trait">drought</span> and freezing stress ( <xref ref-type="fig" rid="pone-0006898-g005"> Fig. 5 </xref> , <xref ref-type="fig" rid="pone-0006898-g006"> 6 </xref> ). This phenomenon appears to be inconsistent with the notion that ABA insensitivity would generally lead to reduced stress tolerance. However, our results were in line with several reports showing that genes conferring reduced ABA sensitivity can increase stress tolerance. Yang et al. <xref ref-type="bibr" rid="pone.0006898-Yang1"> &#x0005b;60&#x0005d; </xref> found that Lily hydrophilin gene <italic> LLA23 </italic> -transgenic seeds showed reduced sensitivity to ABA, however, its transgenic plants exhibited tolerance to salt, osmotic and <span class="trait">drought</span> stresses. An ABF2-interacting protein gene <italic> ARIA </italic> -overexpressing plants are hypersensitive to ABA but also sensitive to high osmolarity during germination <xref ref-type="bibr" rid="pone.0006898-Kim1"> &#x0005b;61&#x0005d; </xref> . Transgenic plants overexpressing <italic> <span class="gene">GmbZIP44</span> </italic> , <italic> <span class="gene">GmbZIP62</span> </italic> and <italic> <span class="gene">GmbZIP78</span> </italic> from soybean show reduced sensitivity to ABA but enhanced tolerance to salt and freezing stress <xref ref-type="bibr" rid="pone.0006898-Liao1"> &#x0005b;5&#x0005d; </xref> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmDREB2, a soybean DRE-binding transcription factor, conferred <span class="trait">drought</span> and high-salt tolerance in transgenic plants.</h2>
			<h3>PMID:17178106  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen M(1), Wang QY, Cheng XG, Xu ZS, Li LC, Ye XG, Xia LQ, Ma YZ.
				<br/> 
				年份：2007 Feb 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel DREB (dehydration responsive element binding protein) homologous gene, <span class="gene">GmDREB2,</span> was isolated from soybean. Based on its similarity with <span class="gene">AP2</span> domains, <span class="gene">GmDREB2</span> was classified into A-5 subgroup in DREB subfamily in AP2/EREBP family. Expression of <span class="gene">GmDREB2</span> gene was induced by <span class="trait">drought,</span> high salt, and low temperature stresses and abscisic acid treatment. The <span class="gene">GmDREB2</span> bound specifically to DRE element in vitro. Furthermore, the overexpression of <span class="gene">GmDREB2</span> activated expression of downstream genes in transgenic Arabidopsis, resulting in enhanced tolerance to <span class="trait">drought</span> and high-salt stresses and did not cause growth retardation. Analysis of free proline contents in transgenic tobacco indicated that the overexpression of <span class="gene">GmDREB2</span> accumulated higher level of free proline compared to the wild type plants under <span class="trait">drought</span> condition. The results from this study indicate that this novel soybean <span class="gene">GmDREB2</span> gene functions as an important transcriptional activator and may be useful in improving of plant tolerance to abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seaweed extract improve <span class="trait">drought</span> tolerance of soybean by regulating stress-response genes</h2>
			<h3>PMID:29308122</h3>
				

				
					<h3>PMCID:5751077</h3>
					<h4>期刊:AoB Plants</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shukla </surname>  <given-names> Pushp S </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shotton </surname>  <given-names> Katy </given-names>  </name>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Norman </surname>  <given-names> Erin </given-names>  </name>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neily </surname>  <given-names> Will </given-names>  </name>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Critchley </surname>  <given-names> Alan T </given-names>  </name>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Prithiviraj </surname>  <given-names> Balakrishnan </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="corresp" rid="c1"/>  <!--<email> bprithiviraj@dal.ca </email> --> </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 10 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						There is an increasing global concern about the availability of water for agricultural use. <span class="trait">Drought</span> stress negatively impacts plant physiology and crop productivity. Soybean (Glycine max) is one of the important oilseed crops, and its productivity is often reduced by <span class="trait">drought.</span> In this study, a commercial extract of Ascophyllum nodosum (ANE) was evaluated for its potential to alleviate <span class="trait">drought</span> stress in soybean. The aim of this study was to determine the effects of ANE on the response of soybean plants to <span class="trait">drought</span> stress by monitoring stomatal conductance, relative leaf water content, antioxidant activity and expression of stress-responsive genes. Plants treated with ANE had higher relative water content and higher stomatal conductance under <span class="trait">drought</span> stress. During early recovery in the post-drought phase, ANE treated plants had significantly higher stomatal conductance. The antioxidant activity was also found higher in the plants treated with ANE. In addition, ANE-treatment led to changes in the expression of stress-responsive genes: <span class="gene">GmCYP707A1a,</span> <span class="gene">GmCYP707A3b,</span> <span class="gene">GmRD22,</span> <span class="gene">GmRD20,</span> <span class="gene">GmDREB1B,</span> <span class="gene">GmERD1,</span> <span class="gene">GmNFYA3,</span> FIB1a, <span class="gene">GmPIP1b,</span> <span class="gene">GmGST,</span> <span class="gene">GmBIP</span> and <span class="gene">GmTp55.</span> Taken together, these results suggest that applications of ANE improve the <span class="trait">drought</span> tolerance of soybean by changing physiology and gene expression.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Applications of ANE improved stomatal conductance by regulating the ABA biosynthesis pathways. ABA plays an important role in plant responses to <span class="trait">drought</span> stress ( <xref rid="CIT0037" ref-type="bibr"> Zheng <italic> et al. </italic> 2012 </xref> ). <italic> <span class="gene">GmCYP707A1a</span> </italic> and <italic> <span class="gene">GmCYP707A3b</span> </italic> , identified as ABA 8&#x02032;-hydroxylases, which are known to play a central role in regulating ABA levels during dehydration and rehydration, respectively ( <xref rid="CIT0031" ref-type="bibr"> Umezawa <italic> et al. </italic> 2006 </xref> ; <xref rid="CIT0037" ref-type="bibr"> Zheng <italic> et al. </italic> 2012 </xref> ). Treatment of soybean plants with the 7 mL L <sup> &#x02013;1 </sup> of ANE induced the expression of <italic> CYP707A1a </italic> by 1.6-fold during dehydration ( <xref ref-type="fig" rid="F4"> Fig. 4a </xref> ), while the expression of <italic> CYP707A3b </italic> was significantly increased by 1.9-fold after <span class="trait">drought</span> relief in the treated soybean plants ( <xref ref-type="fig" rid="F4"> Fig. 4b </xref> ). Thus, treatment with ANE modulate the biosynthesis of ABA by regulating the expression of <italic> CYP707A </italic> genes during both dehydration and rehydration. ANE-treated soybean showed a significant increase in the expression of <italic> <span class="gene">GmRD22</span> </italic> (2-fold increase) during the applied <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="F4"> Fig. 4c </xref> ). <italic> <span class="gene">GmDREB1B</span> </italic> , an ABA-dependent DRE-binding transcription factor involved in plant <span class="trait">drought</span> tolerance, showed a 3.5-fold greater expression in the drought-stressed ANE-treated plants ( <xref ref-type="fig" rid="F4"> Fig. 4d </xref> ). In contrast, <italic> <span class="gene">GmRD20</span> </italic> , an ABA-independent stress responsive gene was not induced by the extract applications ( <xref ref-type="fig" rid="F4"> Fig. 4e </xref> ). This suggested that the ANE treatment-induced, ABA-dependent regulation of stomata (via regulating expression of <italic> <span class="gene">GmRD22</span> </italic> ) helped plants to withstand those <span class="trait">drought</span> conditions. In addition to this, the greater expression of <italic> <span class="gene">GmRD22</span> </italic> is able to induce the expression of cell wall peroxidases and improve cell wall integrity under the stress conditions ( <xref rid="CIT0034" ref-type="bibr"> Wang <italic> et al. </italic> 2012 </xref> ). The expression of <italic> <span class="gene">GmNFYA3</span> </italic> , which encodes for nuclear factor Y subunit of the NF-Y complex ( <xref rid="CIT0022" ref-type="bibr"> Ni <italic> et al. </italic> 2013 </xref> ), was found to remain unaltered in ANE-treated plants ( <xref ref-type="fig" rid="F4"> Fig. 4f </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Effect of ANE on the relative fold expression of genes of selected soybean genes during different stages of <span class="trait">drought</span> stress by qPCR: (a) <italic> <span class="gene">GmCYP707A1a</span> </italic> , (b) <italic> <span class="gene">GmCYP707A3b</span> </italic> , (c) <span class="gene">GmRD22,</span> (d) <italic> <span class="gene">GmRD20</span> </italic> , (e) <italic> <span class="gene">GmDREB1B</span> </italic> , (f) <span class="gene">GmERD1,</span> (g) <italic> <span class="gene">GmNFYA3</span> </italic> , (h) <italic> FIB1a </italic> , (i) <italic> <span class="gene">GmPIP1b</span> </italic> , (j) <italic> <span class="gene">GmGST</span> </italic> , (k) <italic> <span class="gene">GmBIP</span> </italic> and (l) <italic> <span class="gene">GmTp55</span> </italic> . The values represented in the graphs were calculated from three independent experiments ( <italic> n </italic> = 3). Significantly different values between the treatment and control are represented by different letters.
					</p>
				
					<p>&nbsp;&nbsp;
						It is postulated that in plants treated with 7 mL L <sup> &#x02013;1 </sup> of ANE, ABA induced fibrillin accumulation and the ABA-response regulators (ABI1, ABI2) regulated the fibrillin expression. Fibrillins are lipid binding proteins of the plastid involved in abiotic stress tolerance ( <xref rid="CIT0035" ref-type="bibr"> Yang <italic> et al. </italic> 2006 </xref> ). ANE extract at the rate applied, significantly induced the expression of <italic> FIB1a </italic> during <span class="trait">drought,</span> as compared to the control plants ( <xref ref-type="fig" rid="F4"> Fig. 4g </xref> ). The higher expression of <italic> FIB1a </italic> in Arabidopsis was shown to enhance photosystem II (PSII) photo-tolerance, while reducing <italic> FIB1a </italic> expression which decreases PSII photo-tolerance ( <xref rid="CIT0020" ref-type="bibr"> Mutava <italic> et al. </italic> 2015 </xref> ). ANE-treated plants showed relatively higher transcripts of <italic> FIB1a </italic> , resulting in increased photo-tolerance and reduced stress-induced photo-inhibition of PSII. The expression of <italic> FIB1a </italic> is regulated by ABA, and is involved in ABA-mediated photo-protection during stress ( <xref rid="CIT0035" ref-type="bibr"> Yang <italic> et al. </italic> 2006 </xref> ). Thus, taking all of these together we conclude that treatment with ANE improved the <span class="trait">drought</span> tolerance of soybean plants by ABA-induced photo-protection, by regulating <italic> FIB1a </italic> . In contrast to other genes evaluated in this work, <italic> <span class="gene">GmERD1</span> </italic> , which belongs to the ABA-independent pathway, was also induced 2-fold in the ANE-treated soybeans under the <span class="trait">drought</span> conditions tested (as compared to the control) ( <xref ref-type="fig" rid="F4"> Fig. 4h </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Aquaporins are family of small integral membrane proteins which acts as an osmo-sensor in plant membranes involved in the control of inter-cellular water movement between cells during <span class="trait">drought</span> stress ( <xref rid="CIT0028" ref-type="bibr"> Stolf-Moreira <italic> et al. </italic> 2010 </xref> ). Application of ANE extract (at the rates tested) helped soybean plants in recovering from <span class="trait">drought</span> stress by increasing the expression of <italic> <span class="gene">GmPIP1b</span> </italic> by 1.8-fold during the recovery phase (as compared to control; <xref ref-type="fig" rid="F4"> Fig. 4i </xref> ). Those plants treated with ANE showed an increased transcript abundance of <italic> <span class="gene">GmPIP1b</span> </italic> during <span class="trait">drought</span> stress and yet maintained internal water movement in the plants during the water stress condition. <span class="trait">Drought</span> stress in plants leads to the generation of ROS at the chloroplast PSI and PSII and the mitochondrial complex I and III of the electron transport chain. Glutathione S transferase (GST) functions to remove ROS during stress. Under the conditions of this study, the expression of <span class="gene">GST</span> increased in the ANE treatments, as compared to controls, but that change was not significant ( <xref ref-type="fig" rid="F4"> Fig. 4j </xref> ). <italic> <span class="gene">GmGST</span> </italic> protect cell from oxidative damage by quenching reactive molecules with the addition of glutathione ( <xref rid="CIT0019" ref-type="bibr"> McGonigle <italic> et al. </italic> 2000 </xref> ). <italic> <span class="gene">GmBIP</span> </italic> , a molecular chaperone that increase <span class="trait">drought</span> tolerance in soybean by delaying leaf senescence ( <xref rid="CIT0032" ref-type="bibr"> Valente <italic> et al. </italic> 2009 </xref> ), was induced in ANE-treated plants (as compared to control), during <span class="trait">drought</span> stress, as well as the recovery stages ( <xref ref-type="fig" rid="F4"> Fig. 4k </xref> ). Thus, as demonstrated here, application of ANE mediates the <span class="trait">drought</span> response in soybean by regulating expression of <italic> <span class="gene">GmBIP</span> </italic> and <italic> <span class="gene">GmGST,</span> </italic> and protected treated plants by delaying drought-induced senescence and reduced any oxidative damage to tissues. <italic> <span class="gene">GmTP55</span> </italic> , an antiquitin-like soybean protein was expressed significantly higher in ANE-treated plants during <span class="trait">drought</span> stress, however, the increase was not significantly different during the recovery stage ( <xref ref-type="fig" rid="F4"> Fig. 4i </xref> ). Therefore, it is possible that ANE-induced <italic> <span class="gene">GmTP55</span> </italic> could be involved in the adaptive response by reducing lipid peroxidation-derived, reactive aldehydes, under oxidative stress ( <xref rid="CIT0024" ref-type="bibr"> Rodrigues <italic> et al. </italic> 2006 </xref> ).Thus, under the rate of application and <span class="trait">drought</span> conditions presented in this study, ANE was able to mitigate the <span class="trait">drought</span> stress by regulating the expression of genes involved in ABA biosynthesis and ROS detoxification.
					</p>
				
					<p>&nbsp;&nbsp;
						The fact that there was no significant difference in gene expression between treated and control plants before exposure to <span class="trait">drought</span> stress suggests that ANE only activates the <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms of the plant when the plants are under stress. With the exception of genes such as <italic> <span class="gene">GmPIP1b</span> </italic> , <italic> <span class="gene">GmGST</span> </italic> and <italic> <span class="gene">GmCYP707A3b</span> </italic> which showed higher expression in both control and treated plants, the level of expression in the ANE-treated plants in the recovery time point was already decreasing compared to the period during stress, again indicating that the response is only present while the plants are under stress. This is important, as <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms such as stomatal closure can lead to reduced productivity if extended beyond the period of stress ( <xref rid="CIT0021" ref-type="bibr"> Neumann 2008 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Of the genes investigated, <italic> <span class="gene">GmRD20</span> </italic> and <italic> <span class="gene">GmNFYA3</span> </italic> showed little response to the <span class="trait">drought</span> stress in either control or treated plants. This suggests that the mode of action of ANE works by amplifying the plants&#x02019; natural response to <span class="trait">drought</span> stress in terms of increased gene expression. Thus, our physiological and gene expression study revealed that treatment with 7 mL L <sup> &#x02013;1 </sup> ANE resulted in the successful adaptation and survival of treated soybean plants under the conditions tested (as compared to control).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Cold-induced modulation and functional analyses of the DRE-binding transcription factor gene, <span class="gene">GmDREB3,</span> in soybean (Glycine max L.).</h2>
			<h3>PMID:18988621  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen M(1), Xu Z, Xia L, Li L, Cheng X, Dong J, Wang Q, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DREB (dehydration-responsive element-binding protein) transcription factors have important roles in the stress-related regulation network in plants. A DREB orthologue, <span class="gene">GmDREB3,</span> belonging to the A-5 subgroup of the DREB subfamily, was isolated from soybean using the RACE (rapid amplification of cDNA ends) method. Northern blot analysis showed that expression of <span class="gene">GmDREB3</span> in soybean seedlings was induced following cold stress treatment for 0.5 h and was not detected after 3 h. However, it was not induced by <span class="trait">drought</span> and high salt stresses or by abscisic acid (ABA) treatment. This response was similar to those of members in the A-1 subgroup and different from those of other members in the A-5 subgroup, suggesting that the <span class="gene">GmDREB3</span> gene was involved in an ABA-independent cold stress-responsive signal pathway. Furthermore, analysis of the <span class="gene">GmDREB3</span> promoter elucidated its cold-induced modulation. A promoter fragment containing bases -1058 to -664 was involved in response to cold stress, and its effect was detected for 1 h after treatment, but a transcriptional repressor appeared to impair this response by binding to a cis-element in the region -1403 to -1058 at 24 h after the beginning of cold stress. Moreover, the <span class="gene">GmDREB3</span> protein could specifically bind to the DRE element in vitro, and activated expression of downstream reporter genes in yeast cells. In addition, overexpression of <span class="gene">GmDREB3</span> enhanced tolerance to cold, <span class="trait">drought,</span> and high salt stresses in transgenic Arabidopsis. Physiological analyses indicated that the fresh weight and osmolality of <span class="gene">GmDREB3</span> transgenic Arabidopsis under cold stress were higher than those of wild-type controls. <span class="gene">GmDREB3</span> transgenic tobacco accumulated higher levels of free proline under <span class="trait">drought</span> stress and retained higher leaf chlorophyll levels under high salt stress than wild-type tobacco. In addition, constitutive expression of <span class="gene">GmDREB3</span> in transgenic Arabidopsis caused growth retardation, whereas its expression under control of the stress-inducible Rd29A promoter minimized negative effects on plant growth under normal growth conditions, indicating that a combination of the Rd29A promoter and <span class="gene">GmDREB3</span> might be useful for improving tolerance to environmental stresses in crop plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean WRKY-type transcription factor genes, <span class="gene">GmWRKY13,</span> <span class="gene">GmWRKY21,</span> and <span class="gene">GmWRKY54,</span> confer differential tolerance to abiotic stresses in transgenic Arabidopsis plants.</h2>
			<h3>PMID:18384508  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou QY(1), Tian AG, Zou HF, Xie ZM, Lei G, Huang J, Wang CM, Wang HW, Zhang JS, Chen SY.
				<br/> 
				年份：2008 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						WRKY-type transcription factors have multiple roles in the plant defence response and developmental processes. Their roles in the abiotic stress response remain obscure. In this study, 64 <span class="gene">GmWRKY</span> genes from soybean were identified, and were found to be differentially expressed under abiotic stresses. Nine <span class="gene">GmWRKY</span> proteins were tested for their transcription activation in the yeast assay system, and five showed such ability. In a DNA-binding assay, three proteins <span class="gene">(GmWRKY13,</span> <span class="gene">GmWRKY27</span> and <span class="gene">GmWRKY54)</span> with a conserved WRKYGQK sequence in their DNA-binding domain could bind to the W-box (TTGAC). However, <span class="gene">GmWRKY6</span> and <span class="gene">GmWRKY21,</span> with an altered sequence WRKYGKK, lost the ability to bind to the W-box. The function of three stress-induced genes, <span class="gene">GmWRKY13,</span> <span class="gene">GmWRKY21</span> and <span class="gene">GmWRKY54,</span> was further investigated using a transgenic approach. <span class="gene">GmWRKY21-transgenic</span> Arabidopsis plants were tolerant to cold stress, whereas <span class="gene">GmWRKY54</span> conferred salt and <span class="trait">drought</span> tolerance, possibly through the regulation of <span class="gene">DREB2A</span> and STZ/Zat10. Transgenic plants over-expressing <span class="gene">GmWRKY13</span> showed increased sensitivity to salt and mannitol stress, but decreased sensitivity to abscisic acid, when compared with wild-type plants. In addition, <span class="gene">GmWRKY13-transgenic</span> plants showed an increase in lateral roots. These results indicate that the three <span class="gene">GmWRKY</span> genes play differential roles in abiotic stress tolerance, and that <span class="gene">GmWRKY13</span> may function in both lateral root development and the abiotic stress response.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean DRE-binding transcription factors that are responsive to abiotic stresses.</h2>
			<h3>PMID:15841365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li XP(1), Tian AG, Luo GZ, Gong ZZ, Zhang JS, Chen SY.
				<br/> 
				年份：2005 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Three DREB homologue genes, <span class="gene">GmDREBa,</span> <span class="gene">GmDREBb,</span> and <span class="gene">GmDREBc,</span> were isolated from soybean, Glycine max (L.) Merr. Each of the deduced proteins contains an <span class="gene">AP2</span> domain of 64 amino acids. Yeast one-hybrid assay revealed that all of the three dehydration-responsive, element-binding proteins specifically bound to the dehydration-responsive element. Analysis of transcriptional activation abilities of these proteins in yeast indicated that <span class="gene">GmDREBa</span> and <span class="gene">GmDREBb</span> could activate the expression of a reporter gene, whereas <span class="gene">GmDREBc</span> could not. The transcriptions of <span class="gene">GmDREBa</span> and <span class="gene">GmDREBb</span> were induced by salt, <span class="trait">drought,</span> and cold stresses in leaves of soybean seedlings. The expression of <span class="gene">GmDREBc</span> was not significantly affected in leaves but apparently induced in roots by salt, <span class="trait">drought,</span> and abscisic acid treatments. These results suggest that these three genes function specifically in response to abiotic stresses in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Osmoprotective functions conferred to soybean plants via inoculation with Sphingomonas sp. LK11 and exogenous trehalose.</h2>
			<h3>PMID:28942839</h3>
				

				
			<p>
				作者：Asaf S(1), Khan AL(2), Khan MA(1), Imran QM(1), Yun BW(1), Lee IJ(3).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Osmotic stress induced by <span class="trait">drought</span> can hinder the growth and <span class="trait">yield</span> of crop plants. To understand the eco-physiological role of osmoprotectants, the combined utilization of endophytes and osmolytes (trehalose) can be an ideal strategy used to overcome the adverse effects of <span class="trait">drought.</span> Hence, in the present study, we aimed to investigate the role of Sphingomonas sp. LK11, which produces phytohormones and synthesizes trehalose, in improving soybean plant growth under drought-induced osmotic stress (-0.4, -0.9, and -1.2MPa). The results showed that the inoculation of soybean plants with Sphingomonas sp. LK11 significantly increased plant length, dry biomass, photosynthetic pigments, glutathione, amino acids (proline, glycine, and glutamate), and primary sugars as compared to control plants under varying <span class="trait">drought</span> stresses. Trehalose applied to the plant with or without endophyte-inoculation also showed similar plant growth-promoting attributes under stress. Stress exposure significantly enhanced endogenous jasmonic (JA) and abscisic (ABA) acid contents in control plants. In contrast, Sphingomonas sp. LK11-inoculation significantly lowered ABA and JA levels in soybean plants, but these phytohormones increased in response to combined treatments during stress. The drought-induced osmotic stress <span class="trait">resistance</span> associated with Sphingomonas sp. LK11 and trehalose was also evidenced by increased mRNA gene expression of soybean dehydration responsive element binding protein (DREB)-type transcription factors <span class="gene">(GmDREBa</span> and <span class="gene">GmDREB2)</span> and the MYB (myeloblastosis) transcription factor <span class="gene">(GmMYBJ1)</span> as compared to the control. In conclusion, our findings demonstrated that inoculation with this endophyte and trehalose improved the negative effects of drought-induced osmotic stress, and it enhanced soybean plant growth and tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning, expression profiling and trans-activation property studies of a DREB2-like gene from chrysanthemum (Dendranthema vestitum).</h2>
			<h3>PMID:18224275  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu L(1), Zhu K, Yang Y, Wu J, Chen F, Yu D.
				<br/> 
				年份：2008 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Dehydration responsive element binding (DREB) proteins are important transcription factors in plant stress response and signal transduction. In this study, a DREB homolog gene, DvDREB2A, was isolated from chrysanthemum (Dendranthema vestitum) by reverse transcriptase-polymerase chain reaction (RT-PCR) and rapid amplification of cDNA ends (RACE) methods. It contained an open reading frame (ORF) of 1,471 bp encoding 366 amino acid residues and was classified as a <span class="gene">DREB2</span> subfamily member based on multiple sequence alignment. The predicted protein sequence contained a typical AP2/EREBP DNA-binding domain near the N-terminal region. In yeast one-hybrid analysis DvDREB2A protein was specifically bound to DRE elements (core sequence, A/GCCGAC) and activated the expression of the reporter <span class="gene">HIS3</span> and LacZ. Transient expression experiment suggested that DvDREB2A protein was localized to the nucleus of onion epidermis cells. Quantitative real-time PCR (QRT-PCR) experiments showed that expression level of DvDREB2A was significantly affected by heat, low temperature, <span class="trait">drought,</span> abscisic acid (ABA) and high salinity treatments. These results indicated that the DvDREB2A gene is a new member of the DREB transcription factors, which may play an important role in providing tolerance to environmental stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <italic> GhWRKY27a </italic> reduces tolerance to <span class="trait">drought</span> stress and <span class="trait">resistance</span> to <italic> Rhizoctonia solani </italic> infection in transgenic <italic> Nicotiana benthamiana </italic></h2>
			<h3>PMID:26483697</h3>
				
					<h3>PMCID:4586331</h3>
					<h4>期刊:Frontiers in Physiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Yan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jia </surname>  <given-names> Haihong </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Fang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Chen </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shuchang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Xingqi </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/228979/overview"/>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To date, many group III WRKY TFs have been identified in <italic> Arabidopsis </italic> , rice, Pak-choi, <italic> Thlaspi caerulescens </italic> , and <italic> Vitis pseudoreticulata </italic> (Kalde et al., <xref rid="B19" ref-type="bibr"> 2003 </xref> ; Xie et al., <xref rid="B50" ref-type="bibr"> 2005 </xref> ; Wei et al., <xref rid="B49" ref-type="bibr"> 2008 </xref> ; Li et al., <xref rid="B25" ref-type="bibr"> 2010 </xref> ; Wang et al., <xref rid="B46" ref-type="bibr"> 2012 </xref> ). Additionally, more group III WRKY TFs have been reported to be involved in plant responses to abiotic stress. For instance, <italic> BcWRKY46 </italic> overexpression in tobacco enhanced the tolerance of transgenic tobacco to <span class="trait">drought,</span> cold, and salt stress (Wang et al., <xref rid="B46" ref-type="bibr"> 2012 </xref> ). FcWRKY70, a WRKY protein of <italic> Fortunella crassifolia </italic> , functions in <span class="trait">drought</span> tolerance and modulates putrescine synthesis by regulating the arginine decarboxylase gene (Gong et al., <xref rid="B13" ref-type="bibr"> 2015 </xref> ). WRKY70 and WRKY54 co-operate as negative regulators of stomatal closure and, consequently, osmotic stress tolerance in <italic> Arabidopsis </italic> (Li et al., <xref rid="B26" ref-type="bibr"> 2013 </xref> ). Another group III WRKY TF, <italic> AtWRKY46 </italic> , has been shown to play dual roles in regulating plant responses to <span class="trait">drought</span> and salt stress and light-dependent stomatal opening in guard cells (Ding et al., <xref rid="B9" ref-type="bibr"> 2014 </xref> ). Furthermore, the phytohormone abscisic acid (ABA) plays central role in the stress responses of plants exposed to environmental challenges. A recent study has demonstrated that WRKY TFs constitute key nodes in ABA-responsive signaling networks (Rushton et al., <xref rid="B37" ref-type="bibr"> 2012 </xref> ). Several rice WRKY proteins have been found to act as repressors or activators of an ABA-inducible promoter in aleuronic cells (Xie et al., <xref rid="B50" ref-type="bibr"> 2005 </xref> ). Ren et al. ( <xref rid="B36" ref-type="bibr"> 2010 </xref> ) also showed that <italic> AtWRKY63 </italic> can bind the W-box in the promoter of <italic> <span class="gene">AREB1</span> </italic> / <italic> ABF2 in vitro </italic> , and the <italic> wrky63 </italic> mutant is more sensitive to <span class="trait">drought</span> stress than wild-type plants.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, emerging evidence has indicated that group III WRKY TFs are central components of many aspects of the plant innate immune system, including basal defense and systemic-acquired <span class="trait">resistance</span> (Rushton et al., <xref rid="B38" ref-type="bibr"> 2010 </xref> ; Jiang et al., <xref rid="B18" ref-type="bibr"> 2014 </xref> ). The WRKY TFs bind to and regulate the expression of several well-characterized plant defense-related genes, all of which contain W-box elements in their promoter regions (Yu et al., <xref rid="B55" ref-type="bibr"> 2001 </xref> ). For example, Mao et al. ( <xref rid="B32" ref-type="bibr"> 2007 </xref> ) reported that <span class="gene">WRKY62</span> acts downstream of cytosolic NPR1 and negatively regulates JA-responsive gene expression in <italic> Arabidopsis </italic> . <italic> Arabidopsis </italic> WRKY70 has been shown to modulate the crosstalk between SA- and JA-mediated signaling by promoting SA-dependent and suppressing JA-dependent responses (Li et al., <xref rid="B28" ref-type="bibr"> 2006 </xref> ). In rice plants, overexpression of the elicitor-induced <italic> OsWRKY53 </italic> gene leads to enhanced <span class="trait">resistance</span> to the blast fungus <italic> Magnaporthe grisea </italic> (Chujo et al., <xref rid="B7" ref-type="bibr"> 2007 </xref> ). OsWRKY31 acts as a transcriptional activator, and overexpression of the <italic> OsWRKY31 </italic> gene enhances <span class="trait">resistance</span> against infection with <italic> M. grisea </italic> (Zhang et al., <xref rid="B58" ref-type="bibr"> 2008 </xref> ). A pepper ( <italic> Capsicum annuum </italic> L.) WRKY gene, <italic> CaWRKY30 </italic> , is involved in pathogen stress responses (Zheng et al., <xref rid="B59" ref-type="bibr"> 2011 </xref> ). Another group III member, AtWRKY52/RRS1, forms a receptor complex by combining with several NB-LRR proteins, and this receptor complex integrates a &#x0201c;decoy&#x0201d; domain that enables the detection of effectors that target WRKY proteins (Sarris et al., <xref rid="B39" ref-type="bibr"> 2015 </xref> ). These findings further emphasize the significance of group III WRKY proteins for plant immunity.
					</p>
				
					<p>&nbsp;&nbsp;
						To ascertain the possible mechanisms underlying the reduced antioxidant defense abilities observed in the transgenic plants, the levels of defense-related genes were assessed in WT and OE plants via qRT-PCR under <span class="trait">drought</span> stress. As shown in Figure <xref ref-type="fig" rid="F8"> 8E </xref> , the levels of superoxide dismutase gene ( <italic> SOD </italic> ), glutathione S-transferase gene ( <italic> <span class="gene">GST</span> </italic> ), ascorbate peroxidase gene ( <italic> APX </italic> ), and catalase gene ( <italic> CAT </italic> ), which encode ROS-scavenging enzymes, were increased to a greater extent in WT plants than in OE plants under <span class="trait">drought</span> conditions. However, the levels of the ROS producers (the respiratory burst oxidase homolog genes <italic> RbohA </italic> and <italic> RbohB </italic> ) were increased to a greater extent in OE plants than in WT plants (Figure <xref ref-type="fig" rid="F8"> 8E </xref> ). Together, these data demonstrate that <italic> GhWRKY27a </italic> may play a critical role in the regulation of the ROS network pathway.
					</p>
				
					<p>&nbsp;&nbsp;
						To expand our previous analysis of the biological roles of <italic> GhWRKY27a </italic> , we explored the possible contribution of <italic> GhWRKY27a </italic> to abiotic stress responses using <span class="trait">drought</span> stress as a model. Our results showed that silencing <italic> GhWRKY27a </italic> in cotton enhanced the tolerance to <span class="trait">drought</span> stress (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ) and <italic> GhWRKY27a </italic> overexpression in <italic> N. benthamiana </italic> reduced its tolerance to <span class="trait">drought</span> stress (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ), indicating that GhWRKY27a is a negative regulator of tolerance to <span class="trait">drought</span> stress. The function of GhWRKY27a in <span class="trait">drought</span> stress tolerance is similar to <italic> Arabidoposis </italic> WRKY53, which is phylogenetically related to GhWRKY27a (Dou et al., <xref rid="B10" ref-type="bibr"> 2014 </xref> ). It was shown that activated expression of AtWRKY53 negatively regulated <span class="trait">drought</span> tolerance (Sun and Yu, <xref rid="B43" ref-type="bibr"> 2015 </xref> ). The phytohormone ABA is essential in plant responses to <span class="trait">drought</span> stress, and an increased ABA content is beneficial for plants under <span class="trait">drought</span> stress as a result of ABA-induced changes at the cellular and whole-plant levels (Xiong and Zhu, <xref rid="B52" ref-type="bibr"> 2003 </xref> ). The OE plants accumulated less ABA under <span class="trait">drought</span> stress compared with the WT plants (Figure <xref ref-type="fig" rid="F7"> 7C </xref> ), which indicated that GhWRKY27a functioned in the ABA response in <italic> N. benthamiana </italic> . In addition, the transpirational water loss through the stomata is a key determinant of <span class="trait">drought</span> tolerance (Xiong et al., <xref rid="B51" ref-type="bibr"> 2002 </xref> ). We found that the ectopic expression of <italic> GhWRKY27a </italic> in OE plants driven by the CaMV35S promoter did not result in the impairment of stomatal closure under normal conditions (Figure <xref ref-type="fig" rid="F6"> 6D </xref> ). However, the more rapid water loss and impaired stomatal closure were observed in OE plants under <span class="trait">drought</span> stress (Figure <xref ref-type="fig" rid="F6"> 6D </xref> ). We speculated that <italic> GhWRKY27a </italic> may not be expressed in guard cells of stomata, and impair stomatal closure indirectly. The regulation of stomatal movement may result from the interference of GhWRKY27a with the endogenous WRKY regulatory networks. Besides that, compared with normal condition, <italic> GhWRKY27a </italic> may interfere with more endogenous regulators under <span class="trait">drought</span> condition. Of course, further research is required to examine whether GhWRKY27a localizes to the guard cells of stomata and affects ABA level. Moreover, the results of this study showed that OE plants antagonistically regulated the expression of ABA- and drought-responsive genes, including <italic> SnRK2.3, AREB, LEA </italic> , and <italic> <span class="gene">P5CS</span> </italic> (Figure <xref ref-type="fig" rid="F7"> 7D </xref> ). Under <span class="trait">drought</span> stress conditions, these stress-related genes are induced and are considered to play a role in defense response (Shinozaki and Yamaguchi-Shinozaki, <xref rid="B41" ref-type="bibr"> 2007 </xref> ). Based on these data, we infer that <italic> GhWRKY27a </italic> might confer reduced <span class="trait">drought</span> tolerance, which was coupled with impaired stomatal closure and downregulated expression of drought-responsive genes. On the other hand, we found that OE plants showed greater accumulation of ROS than WT plants after <span class="trait">drought</span> treatment, and <italic> GhWRKY27a </italic> overexpression resulted in reduced tolerance to oxidative stress (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ), indicating that GhWRKY27a is also a negative regulator of tolerance to oxidative stress. It has been hypothesized that ROS production may be the primary symptom of phytotoxicity under abiotic stress (Choudhury et al., <xref rid="B6" ref-type="bibr"> 2013 </xref> ). Thus, we infer that GhWRKY27a may function in <span class="trait">drought</span> stress by regulating production of ROS. This is similar to the observation that overexpression of <italic> GhWRKY17 </italic> in <italic> N. benthamiana </italic> reduced <span class="trait">drought</span> stress tolerance by enhancing ROS accumulation (Yan et al., <xref rid="B54" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The increased transcript levels of <italic> GhWRKY27a </italic> under <span class="trait">drought</span> stress and pathogen attack is paradoxical, because GhWRKY27a functions in negative responses to <span class="trait">drought</span> tolerance and in <span class="trait">resistance</span> to <italic> R. solani </italic> infection. Similar results have been reported in other plant species. In soybean, a stress-induced gene, <italic> <span class="gene">GmWRKY13</span> </italic> , negatively regulates <span class="trait">drought</span> stress responses (Zhou et al., <xref rid="B61" ref-type="bibr"> 2008 </xref> ). <italic> C. annuum CaWRKY1 </italic> , which is strongly induced by pathogen infections and the signal molecular SA, acts as a negative regulator to prevent spurious activation of defense responses at suboptimal concentrations of SA (Oh et al., <xref rid="B33" ref-type="bibr"> 2008 </xref> ). In <italic> Arabidopsis </italic> , Wang et al. ( <xref rid="B45" ref-type="bibr"> 2006 </xref> ) showed that <italic> <span class="gene">WRKY58</span> </italic> acts as a negative regulator in defense, and the <italic> wrky58 </italic> mutant was more resistant to a pathogen than WT plants after treatment with a suboptimal level of benzothiadizole. We infer that the role of <italic> GhWRKY27a </italic> may be similar to that of these negative regulators, which is to prevent unnecessary activation of defense responses at suboptimal levels of signal molecules. In addition, another possibility is that <italic> GhWRKY27a </italic> , which is induced by multiple stresses, adjusts the intensity of defense responses in cooperation with stress-responsive positive regulators, and prevents overactivation of defense responses whenever stress diminishes. Further studies are needed to elucidate the role of GhWRKY27a in various biological processes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> OsDREB2A </italic> , a Rice Transcription Factor, Significantly Affects Salt Tolerance in Transgenic Soybean</h2>
			<h3>PMID:24376625</h3>
				
					<h3>PMCID:3869746</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Xiu-xiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Tang </surname>  <given-names> Yu-juan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Qi-bin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Cun-yi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mu </surname>  <given-names> Ying-hui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Suo </surname>  <given-names> Hai-cui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Luo </surname>  <given-names> Lai-hui </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						DRE <italic> cis- </italic> acting element is mainly involved in regulation of genes by <span class="trait">drought,</span> salt and cold stress under ABA-independent pathways <xref ref-type="bibr" rid="pone.0083011-Li1"> [26] </xref> . Previous studies have shown that DREB genes are resistant to different abiotic stresses, including salt, <span class="trait">drought,</span> oxidation, and freezing <xref ref-type="bibr" rid="pone.0083011-Dubouzet1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0083011-Liu2"> [27] </xref> , <xref ref-type="bibr" rid="pone.0083011-Gutha1"> [28] </xref> . <span class="gene">DREB2</span> are involved in <span class="trait">drought</span> and salt stresses but not in cold stress while DREB1/CBF-type transcription factors function in response to cold stress in another ABA-independent pathway <xref ref-type="bibr" rid="pone.0083011-Liu1"> [7] </xref> , <xref ref-type="bibr" rid="pone.0083011-Lata1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0083011-Chen2"> [17] </xref> , <xref ref-type="bibr" rid="pone.0083011-Nakashima2"> [29] </xref> . In the present study, our results showed that <italic> OsDREB2A </italic> overexpression improved high-salt tolerance in transgenic soybeans. And interestingly, <italic> OsDREB2A </italic> does not affect the developmental growth of soybean under normal growth conditions (Data not shown). Effect of salt stress on both seed germination and seedling growth showed that the transgenic lines increased tolerance to salt stress than the WT ( <xref ref-type="fig" rid="pone-0083011-g001"> Figure 1B, 1D and 1E </xref> ). The results demonstrated that <italic> OsDREB2A </italic> acted as a positive regulator in response to salt stress at the germination and seedling stages in transgenic soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">P5CS</span> </italic> also increased stress tolerance of transgenic potato, rice and wheat as a result of the increased proline content <xref ref-type="bibr" rid="pone.0083011-HmidaSayari1"> [39] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0083011-Vendruscolo1"> [42] </xref> . We analyzed the expression of <italic> <span class="gene">GmP5CS</span> </italic> , the key enzyme in proline synthesis <xref ref-type="bibr" rid="pone.0083011-Delauney1"> [43] </xref> , <xref ref-type="bibr" rid="pone.0083011-Hu1"> [44] </xref> , and some other genes known to be related to salt <span class="trait">resistance.</span> Similarly, qRT-PCR analysis revealed that the expression of <italic> OsDREB2A </italic> increased <italic> <span class="gene">GmP5CS</span> </italic> transcript level and thus led to higher level of free proline accumulation ( <xref ref-type="fig" rid="pone-0083011-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0083011-g003"> 3 </xref> ). Promoter analysis of the <italic> <span class="gene">GmP5CS</span> </italic> revealed that no DRE region exists. The interaction of <italic> OsDREB2A </italic> with <italic> <span class="gene">GmP5CS</span> </italic> promoter must not contribute to the increased expression of <italic> <span class="gene">GmP5CS</span> </italic> . With the discovery of miRNAs, a new mechanism to regulate protein expression has been revealed. Considering the inconsistency between <italic> <span class="gene">P5CS</span> </italic> mRNA and protein expression and the importance of miRNAs in cancer, the regulation of miRNAs on <italic> <span class="gene">P5CS</span> </italic> represent a very promising hypothesis <xref ref-type="bibr" rid="pone.0083011-Liu3"> [45] </xref> . In this study, we propose that <italic> OsDREB2A </italic> might be involved in many pathways, in addition to combination with DRE region, including regulation of miRNAs and other pathways. A lot of efforts are still required to uncover in detail of each products of gene induced by <italic> <span class="gene">P5CS</span> </italic> and their interacting partners to understand the complexity of the high salinity stress signal transduction pathways.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hydrogen peroxide spraying alleviates <span class="trait">drought</span> stress in soybean plants.</h2>
			<h3>PMID:21377755  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ishibashi Y(1), Yamaguchi H, Yuasa T, Iwaya-Inoue M, Arima S, Zheng SH.
				<br/> 
				年份：2011 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To ascertain the effect of exogenously applied hydrogen peroxide (H2O2) on <span class="trait">drought</span> stress, we examined whether the spraying of soybean leaves with H2O2 would alleviate the symptoms of <span class="trait">drought</span> stress. Pre-treatment by spraying leaves with H2O2 delayed foliar wilting caused by <span class="trait">drought</span> stress compared to leaves sprayed with distilled water (DW). Additionally, the relative water content of drought-stressed leaves pre-treated with H2O2 was higher than that of leaves pre-treated with DW. Therefore, we analyzed the effect of H2O2 spraying on photosynthetic parameters and on the biosynthesis of oligosaccharides related to water retention in leaves during <span class="trait">drought</span> stress. Under conditions of <span class="trait">drought</span> stress, the net photosynthetic rate and stomatal conductance of leaves pre-treated with H2O2 were higher than those of leaves pre-treated with DW. In contrast to DW spraying, H2O2 spraying immediately caused an increase in the mRNA levels of d-myo-inositol 3-phosphate synthase 2 <span class="gene">(GmMIPS2)</span> and galactinol synthase (GolS), which encode key enzymes for the biosynthesis of oligosaccharides known to help plants tolerate <span class="trait">drought</span> stress. In addition, the levels of myo-inositol and galactinol were higher in H2O2-treated leaves than in DW-treated leaves. These results indicated that H2O2 spraying enabled the soybean plant to avoid <span class="trait">drought</span> stress through the maintenance of leaf water content, and that this water retention was caused by the promotion of oligosaccharide biosynthesis rather than by rapid stomatal closure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel MYB transcription factor, <span class="gene">GmMYBJ1,</span> from soybean confers <span class="trait">drought</span> and cold tolerance in Arabidopsis thaliana.</h2>
			<h3>PMID:24440241  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Su LT(1), Li JW(1), Liu DQ(1), Zhai Y(2), Zhang HJ(1), Li XW(3), Zhang QL(1), Wang Y(4), Wang QY(5).
				<br/> 
				年份：2014 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB transcription factors play important roles in the regulation of plant growth, developmental metabolism and stress responses. In this study, a new MYB transcription factor gene, <span class="gene">GmMYBJ1,</span> was isolated from soybean [Glycine max (L.)]. The <span class="gene">GmMYBJ1</span> cDNA is 1296bp in length with an open reading frame (ORF) of 816 bp encoding for 271 amino acids. The amino acid sequence displays similarities to the typical R2R3 MYB proteins reported in other plants. Transient expression analysis using the <span class="gene">GmMYBJ1-GFP</span> fusion gene in onion epidermal cells revealed that the <span class="gene">GmMYBJ1</span> protein is targeted to the nucleus. Quantitative RT-PCR analysis demonstrated that <span class="gene">GmMYBJ1</span> expression was induced by abiotic stresses, such as <span class="trait">drought,</span> cold, salt and exogenous abscisic acid (ABA). Compared to wild-type (WT) plants, transgenic Arabidopsis overexpressing <span class="gene">GmMYBJ1</span> exhibited an enhanced tolerance to <span class="trait">drought</span> and cold stresses. These results indicate that <span class="gene">GmMYBJ1</span> has the potential to be utilized in transgenic breeding lines to improve abiotic stress tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping of quantitative trait loci for canopy-wilting trait in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:22566068  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Carter TE Jr, Purcell LC, King CA, Ries LL, Chen P, Schapaugh W Jr, Sinclair TR, Boerma HR.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects [Glycine max (L.) Merr] soybean at most developmental stages, which collectively results in <span class="trait">yield</span> reduction. Little information is available on relative contribution and chromosomal locations of quantitative trait loci (QTL) conditioning <span class="trait">drought</span> tolerance in soybean. A Japanese germplasm accession, PI 416937, was found to possess <span class="trait">drought</span> <span class="trait">resistance.</span> Under moisture-deficit conditions, PI 416937 wilted more slowly in the field than elite cultivars and has been used as a parent in breeding programs to improve soybean productivity. A recombinant inbred line (RIL) population was derived from a cross between PI 416937 and Benning, and the population was phenotyped for canopy wilting under rain-fed field conditions in five distinct environments to identify the QTL associated with the canopy-wilting trait. In a combined analysis over environments, seven QTL that explained 75 % of the variation in canopy-wilting trait were identified on different chromosomes, implying the complexity of this trait. Five QTL inherited their positive alleles from PI 416937. Surprisingly, the other two QTL inherited their positive alleles from Benning. These putative QTL were co-localized with other QTL previously identified as related to plant abiotic stresses in soybean, suggesting that canopy-wilting QTL may be associated with additional morpho-physiological traits in soybean. A locus on chromosome 12 <span class="gene">(Gm12)</span> from PI 416937 was detected in the combined analysis as well as in each individual environment, and explained 27 % of the variation in canopy-wilting. QTL identified in PI 416937 could provide an efficient means to augment field-oriented development of drought-tolerant soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmGBP1, a homolog of human ski interacting protein in soybean, regulates flowering and stress tolerance in Arabidopsis.</h2>
			<h3>PMID:23388059  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang Y(1), Zhao L, Li H, Gao Y, Li Y, Wu X, Teng W, Han Y, Zhao X, Li W.
				<br/> 
				年份：2013 Feb 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: SKIP is a transcription cofactor in many eukaryotes. It can regulate plant stress tolerance in rice and Arabidopsis. But the homolog of SKIP protein in soybean has been not reported up to now. RESULTS: In this study, the expression patterns of soybean GAMYB binding protein gene <span class="gene">(GmGBP1)</span> encoding a homolog of SKIP protein were analyzed in soybean under abiotic stresses and different day lengths. The expression of <span class="gene">GmGBP1</span> was induced by polyethyleneglycol 6000, NaCl, gibberellin, abscisic acid and heat stress. <span class="gene">GmGBP1</span> had transcriptional activity in C-terminal. <span class="gene">GmGBP1</span> could interact with R2R3 domain of <span class="gene">GmGAMYB1</span> in SKIP domain to take part in gibberellin flowering pathway. In long-day (16 h-light) condition, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> exhibited earlier flowering and less number of rosette leaves; Suppression of AtSKIP in Arabidopsis resulted in growth arrest, flowering delay and down-regulation of many flowering-related genes (CONSTANS, FLOWERING LOCUS T, LEAFY); Arabidopsis myb33 mutant plants with ectopic overexpression of <span class="gene">GmGBP1</span> showed the same flowering phenotype with wild type. In short-day (8 h-light) condition, transgenic Arabidopsis plants with <span class="gene">GmGBP1</span> flowered later and showed a higher level of FLOWERING LOCUS C compared with wild type. When treated with abiotic stresses, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> enhanced the tolerances to heat and <span class="trait">drought</span> stresses but reduced the tolerance to high salinity, and affected the expressions of several stress-related genes. CONCLUSIONS: In Arabidopsis, <span class="gene">GmGBP1</span> might positively regulate the flowering time by affecting CONSTANS, FLOWERING LOCUS T, LEAFY and GAMYB directly or indirectly in photoperiodic and gibberellin pathways in LDs, but <span class="gene">GmGBP1</span> might represse flowering by affecting FLOWERING LOCUS C and SHORT VEGETATIVE PHASE in autonomous pathway in SDs. <span class="gene">GmGBP1</span> might regulate the activity of ROS-eliminating to improve the <span class="trait">resistance</span> to heat and <span class="trait">drought</span> but reduce the high-salinity tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and quantitative expression analysis of drought-induced genes in soybean.</h2>
			<h3>PMID:20467978  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Stolf-Moreira R(1), Medri ME, Neumaier N, Lemos NG, Brogin RL, Marcelino FC, de Oliveira MC, Farias JR, Abdelnoor RV, Nepomuceno AL.
				<br/> 
				年份：2010 May 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We determined the expression levels of DREB transcription factor <span class="gene">(Gmdreb1)</span> and of the genes <span class="gene">Gmgols,</span> <span class="gene">Gmpip1b,</span> <span class="gene">Gmereb,</span> and <span class="gene">Gmdefensin</span> in drought-tolerant (MG/BR46-Conquista) and drought-sensitive (BR16) genotypes of soybean, during <span class="trait">drought.</span> The trial was carried out in a controlled-environment chamber, set up to provide <span class="trait">drought</span> conditions. Sequences of Arabidopsis thaliana DREB-family proteins were used to build a phylogenetic tree through the alignment of the conserved regions near the <span class="gene">AP2</span> domain. We found that <span class="gene">Gmdreb1</span> is similar to Atrap2.1, which is located near the AtDREB1 and AtDREB2 families. The amplified fragment was cloned and sequenced; alignment with the sequence available at Genbank showed total similarity. Expression analysis showed that under <span class="trait">drought:</span> a) <span class="gene">Gmdreb1</span> expression increased in leaves and roots of both genotypes and expression level changes occurred that were correlated with the length of the water-deficit period; b) there were increased expression levels of <span class="gene">Gmdefensin</span> in roots of MG/BR46; c) expression of <span class="gene">Gmgols</span> increased in leaves and roots of the two genotypes; d) <span class="gene">Gmpip1b</span> expression generally increased, except in roots of BR16, and e) the same was found for <span class="gene">Gmereb,</span> except in roots of MG/BR46.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmWRKY27 interacts with <span class="gene">GmMYB174</span> to reduce expression of <span class="gene">GmNAC29</span> for stress tolerance in soybean plants.</h2>
			<h3>PMID:25990284  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang F(1), Chen HW(1), Li QT(1), Wei W(1), Li W(2), Zhang WK(1), Ma B(1), Bi YD(2), Lai YC(2), Liu XL(3), Man WQ(3), Zhang JS(1), Chen SY(1).
				<br/> 
				年份：2015 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is an important crop for oil and protein resources worldwide. The molecular mechanism of the abiotic stress response in soybean is largely unclear. We previously identified multiple stress-responsive WRKY genes from soybean. Here, we further characterized the roles of one of these genes, <span class="gene">GmWRKY27,</span> in abiotic stress tolerance using a transgenic hairy root assay. <span class="gene">GmWRKY27</span> expression was increased by various abiotic stresses. Over-expression and RNAi analysis demonstrated that <span class="gene">GmWRKY27</span> improves salt and <span class="trait">drought</span> tolerance in transgenic soybean hairy roots. Measurement of physiological parameters, including reactive oxygen species and proline contents, supported this conclusion. <span class="gene">GmWRKY27</span> inhibits expression of a downstream gene <span class="gene">GmNAC29</span> by binding to the W-boxes in its promoter region. The <span class="gene">GmNAC29</span> is a negative factor of stress tolerance as indicated by the performance of transgenic hairy roots under stress. <span class="gene">GmWRKY27</span> interacts with <span class="gene">GmMYB174,</span> which also suppresses <span class="gene">GmNAC29</span> expression and enhances <span class="trait">drought</span> stress tolerance. The <span class="gene">GmWRKY27</span> and <span class="gene">GmMYB174</span> may have evolved to bind to neighbouring cis elements in the <span class="gene">GmNAC29</span> promoter to co-reduce promoter activity and gene expression. Our study discloses a valuable mechanism in soybean for regulation of the stress response by two associated transcription factors. Manipulation of these genes should facilitate improvements in stress tolerance in soybean and other crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmERF5,</span> a new member of the soybean EAR motif-containing ERF transcription factor, enhances <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:25779701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dong L(1), Cheng Y(1), Wu J(2), Cheng Q(1), Li W(1), Fan S(1), Jiang L(1), Xu Z(3), Kong F(4), Zhang D(3), Xu P(5), Zhang S(5).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora root and stem rot of soybean [Glycine max (L.) Merr.], caused by Phytophthora sojae Kaufmann and Gerdemann, is a destructive disease throughout the soybean planting regions in the world. Here, we report insights into the function and underlying mechanisms of a novel ethylene response factor (ERF) in soybean, namely <span class="gene">GmERF5,</span> in host responses to P. sojae. <span class="gene">GmERF5-overexpressing</span> transgenic soybean exhibited significantly enhanced <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the PR10, PR1-1, and PR10-1 genes. Sequence analysis suggested that <span class="gene">GmERF5</span> contains an AP2/ERF domain of 58 aa and a conserved ERF-associated amphiphilic repression (EAR) motif in its C-terminal region. Following stress treatments, <span class="gene">GmERF5</span> was significantly induced by P. sojae, ethylene (ET), abscisic acid (ABA), and salicylic acid (SA). The activity of the <span class="gene">GmERF5</span> promoter <span class="gene">(GmERF5P)</span> was upregulated in tobacco leaves with ET, ABA, Phytophthora nicotianae, salt, and <span class="trait">drought</span> treatments, suggesting that <span class="gene">GmERF5</span> could be involved not only in the induced defence response but also in the ABA-mediated pathway of salt and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> could bind to the GCC-box element and act as a repressor of gene transcription. It was targeted to the nucleus when transiently expressed in Arabidopsis protoplasts. <span class="gene">GmERF5</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> and eukaryotic translation initiation factor <span class="gene">(GmEIF)</span> both in yeast cells and in planta. To the best of our knowledge, <span class="gene">GmERF5</span> is the first soybean EAR motif-containing ERF transcription factor demonstrated to be involved in the response to pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich protein) gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genes coding for PGYRPs (proline-, glycine- and tyrosine-rich proteins) are widely distributed across eukaryotes and have been proposed to have critical role in plant development, especially in response to environmental stresses. In this study, total of 12 soybean PGYRPs <span class="gene">(GmPGYRP1-12)</span> were identified from the soybean genome database for the first time and full-length cDNA and DNA sequences of <span class="gene">GmPGYRP7</span> was cloned. <span class="gene">GmPGYRP1-12</span> genes encoded a set of small predicted proteins (&lt;120 aa) with molecular mass of 7.20-13.29 kDa and isoelectric point of 4.06-6.57. All <span class="gene">GmPGYRPs</span> contained three exons and two introns with fixed occurring sites within genomic DNA sequences. In the putative <span class="gene">GmPGYRP</span> sequences, 4 amino acids (proline, glycine, tyrosine, and glutamine) account for more than 39% of the total protein composition. <span class="gene">GmPGYRPs</span> had a relatively flexible GYPPX motif followed by a highly conserved cysteine-rich domain (GCLAAXCCCCXLXC) and showed high similarity to other known PGYRPs, especially in C-terminal region. Most of PGYRPs can be divided into five subgroups according to phylogenetic analysis. The transcripts of <span class="gene">GmPGYRP1,</span> 3, 5, and 7, representing different PGYRP subgroups, appeared in different organs including seedling leaves, stems, roots, flowers, and developing seeds, but mainly accumulated in seedling roots. Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage. Our data suggest that <span class="gene">GmPGYRP</span> genes encoding a class of conservative XYPPX-repeat proteins probably play an important role in plant development as well as in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcription analyses of <span class="gene">GmICHG,</span> a gene coding for a β-glucosidase that catalyzes the specific hydrolysis of isoflavone conjugates in Glycine max (L.) Merr.</h2>
			<h3>PMID:23683924  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yoo D(1), Hara T, Fujita N, Waki T, Noguchi A, Takahashi S, Nakayama T.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavone conjugate-hydrolyzing β-glucosidase <span class="gene">(GmICHG)</span> of soybeans [Glycine max (L.) Merr.] catalyzes the specific hydrolysis of isoflavone conjugates (β-7-O-(malonyl)glucosides of isoflavones) to produce free isoflavones. In this study, changes in the transcription levels of <span class="gene">GmICHG</span> in the individual organs of soybean seedlings (cv. Enrei) in response to microbial infection and abiotic stresses were analyzed and compared with those of genes coding for 2-hydroxyisoflavanone synthase <span class="gene">(GmIFS)</span> and isoflavone 7-O-glucosyltransferase <span class="gene">(GmIF7GT).</span> <span class="gene">GmICHG</span> was originally expressed in abundance only in the roots and at low levels only in the other organs. The transcription of <span class="gene">GmICHG</span> in the roots and other organs was suppressed upon infection of the roots by Phytophthora sojae. Upon wounding of the cotyledon, a transient long-distance up-regulation of <span class="gene">GmICHG</span> transcription in the roots was observed; upon fungal infection in the cotyledon, however, a delayed elevation of <span class="gene">GmICHG</span> transcription took place in the roots with the maximum at 10 h after the infection. Such long-distance up-regulation patterns were not observed with either <span class="gene">GmIFS</span> or <span class="gene">GmIF7GT.</span> The transcription levels of <span class="gene">GmICHG</span> remained essentially unchanged upon treatment of the roots with Bradyrhizobium japonicum. The transcription of <span class="gene">GmICHG</span> in the roots was also sensitive to a variety of stresses on the roots, such as flooding, elicitation with yeast extract, <span class="trait">drought,</span> and treatment with plant hormones such as abscisic, salicylic, and jasmonic acids and ethylene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Expressing and functional analysis of <span class="gene">GmMYBJ6</span> from soybean].</h2>
			<h3>PMID:19586866  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang WJ(1), Wu YM, Tang YX.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB transcription factor is one of the largest families in plants, which plays an important role in regulating plant development and physiological metabolism. In this study, the expression and function of the new MYB transcription factor gene <span class="gene">GmMYBJ6</span> (GenBank No. DQ902863), isolated from soybean (Glycine max L.), were characterized. The expression pattern of <span class="gene">GmMYBJ6</span> in different organs was examined using Northern blotting analysis. The expression of <span class="gene">GmMYBJ6</span> was detected only in the leaves. The transcriptional activation ability of <span class="gene">GmMYBJ6</span> protein was confirmed by the yeast assay system and the activity of beta-galactosidase was 28.48 U/mL. The green fluorescent protein expression vector p163-GFP-GmMYBJ6 was constructed and transformed into the epidermal cells of onion via particle bombardmental method. The results of instantaneous expression showed that <span class="gene">GmMYBJ6</span> proteins were localized in cell nucleus. Semi-quantitative RT-PCR analysis indicated that <span class="gene">GmMYBJ6</span> improved the expression of certain flavonoid biosynthetic genes, such as <span class="gene">PAL</span> (Phenylalanine ammonia lyase), <span class="gene">C4H</span> (cinnamate-4-hydroxylase), <span class="gene">4CL</span> (4-coumaroyl-CoA ligase), <span class="gene">CHS</span> (Chalcone synthase), <span class="gene">CHI</span> (Chalcone isomerase), <span class="gene">F3H</span> (Flavanone 3-hydroxylase), and FLS (Flavonol synthase), resulting an increase of the total flavonoid levels in positive tobacco transformants. Additionally, the increasing expression of <span class="gene">GmMYBJ6</span> in soybean cultivar Zhongdou 27, induced by UV-B radiation, <span class="trait">drought,</span> and high-salt treatment, indicated that <span class="gene">GmMYBJ6</span> was associated with response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">GmMYB76,</span> <span class="gene">GmMYB92,</span> and <span class="gene">GmMYB177</span> genes confer stress tolerance in transgenic Arabidopsis plants.</h2>
			<h3>PMID:18725908  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liao Y(1), Zou HF, Wang HW, Zhang WK, Ma B, Zhang JS, Chen SY.
				<br/> 
				年份：2008 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors contain the conserved MYB DNA-binding domain of approximately 50 amino acids and are involved in the regulation of many aspects of plant growth, development, metabolism and stress responses. From soybean plants, we identified 156 <span class="gene">GmMYB</span> genes using our previously obtained 206 MYB unigenes, and 48 were found to have full-length open-reading frames. Expressions of all these identified genes were examined, and we found that expressions of 43 genes were changed upon treatment with ABA, salt, <span class="trait">drought</span> and/or cold stress. Three <span class="gene">GmMYB</span> genes, <span class="gene">GmMYB76,</span> <span class="gene">GmMYB92</span> and <span class="gene">GmMYB177,</span> were chosen for further analysis. Using the yeast assay system, <span class="gene">GmMYB76</span> and <span class="gene">GmMYB92</span> were found to have transactivation activity and can form homodimers. <span class="gene">GmMYB177</span> did not appear to have transactivation activity but can form heterodimers with <span class="gene">GmMYB76.</span> Yeast one-hybrid assay revealed that all the three <span class="gene">GmMYBs</span> could bind to cis-elements TAT AAC GGT TTT TT and CCG GAA <span class="gene">AAA</span> AGG AT, but with different affinity, and <span class="gene">GmMYB92</span> could also bind to TCT CAC CTA CC. The transgenic Arabidopsis plants overexpressing <span class="gene">GmMYB76</span> or <span class="gene">GmMYB177</span> showed better performance than the <span class="gene">GmMYB92-transgenic</span> plants in salt and freezing tolerance. However, these transgenic plants exhibited reduced sensitivity to ABA treatment at germination stage in comparison with the wild-type plants. The three <span class="gene">GmMYB</span> genes differentially affected a subset of stress-responsive genes in addition to their regulation of a common subset of stress-responsive genes. These results indicate that the three <span class="gene">GmMYB</span> genes may play differential roles in stress tolerance, possibly through regulation of stress-responsive genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors (MYB TFs) play diverse roles in plant development and stress responses. However, the mechanisms underlying the actions of MYB TFs during stress response remain unclear. In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance.</span> Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> salt stress, H2O2 and ABA. Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary root length, greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content. We also found that ROS could induce SOD/POD/CAT activity in OE lines. In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary root elongation. Some ROS-related genes were up-regulated under abiotic stress in <span class="gene">GmMYB84</span> transgenic plants compared with the WT. Furthermore, electrophoretic mobility shift assay and luciferase reporter analysis demonstrated that <span class="gene">GmMYB84</span> binds directly to the promoter of <span class="gene">GmRBOHB-1</span> and <span class="gene">GmRBOHB-2</span> genes. Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control root growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">GmbZIP44,</span> <span class="gene">GmbZIP62</span> and <span class="gene">GmbZIP78</span> genes function as negative regulator of ABA signaling and confer salt and freezing tolerance in transgenic Arabidopsis.</h2>
			<h3>PMID:18365246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liao Y(1), Zou HF, Wei W, Hao YJ, Tian AG, Huang J, Liu YF, Zhang JS, Chen SY.
				<br/> 
				年份：2008 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						From soybean plant, 131 bZIP genes were identified and named as <span class="gene">GmbZIPs.</span> The <span class="gene">GmbZIPs</span> can be classified into ten groups and more than one third of these <span class="gene">GmbZIPs</span> are responsive to at least one of the four treatments including ABA, salt, <span class="trait">drought</span> and cold stresses. Previous studies have shown that group A bZIP proteins are involved in ABA and stress signaling. We now chose four non-group A genes to study their features. The four proteins <span class="gene">GmbZIP44,</span> <span class="gene">GmbZIP46,</span> <span class="gene">GmbZIP62</span> and <span class="gene">GmbZIP78</span> belong to the group S, I, C and G, respectively, and can bind to GLM (GTGAGTCAT), ABRE (CCACGTGG) and PB-like (TGAAAA) elements with differential affinity in both the yeast one-hybrid assay and in vitro gel-shift analysis. <span class="gene">GmbZIP46</span> can form homodimer or heterodimer with <span class="gene">GmbZIP62</span> or <span class="gene">GmMYB76.</span> Transgenic Arabidopsis plants overexpressing the <span class="gene">GmbZIP44,</span> <span class="gene">GmbZIP62</span> or <span class="gene">GmbZIP78</span> showed reduced ABA sensitivity. However, all the transgenic plants were more tolerant to salt and freezing stresses when compared with the Col plants. The <span class="gene">GmbZIP44,</span> <span class="gene">GmbZIP62</span> and <span class="gene">GmbZIP78</span> may function in ABA signaling through upregulation of ABI1 and ABI2 and play roles in stress tolerance through regulation of various stress-responsive genes. These results indicate that <span class="gene">GmbZIP44,</span> <span class="gene">GmbZIP62</span> and <span class="gene">GmbZIP78</span> are negative regulators of ABA signaling and function in salt and freezing tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rice heterotrimeric G-protein gamma subunits (RGG1 and RGG2) are differentially regulated under abiotic stress.</h2>
			<h3>PMID:22751322  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yadav DK(1), Islam SM, Tuteja N.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heterotrimeric G-proteins (α, β and γ subunits) are primarily involved in diverse signaling processes by transducing signals from an activated transmembrane G-protein coupled receptor (GPCR) to appropriate downstream effectors within cells. The role of α and β G-protein subunits in salinity and heat stress has been reported but the regulation of γ subunit of plant G-proteins in response to abiotic stress has not heretofore been described. In the present study we report the isolation of full-length cDNAs of two isoforms of Gγ [RGG1(I), 282 bp and RGG2(I), 453 bp] from rice (Oryza sativa cv Indica group Swarna) and described their transcript regulation in response to abiotic stresses. Protein sequence alignment and pairwise comparison of γ subunits of Indica rice [RGG(I)] with other known plant G-protein γ subunits demonstrated high homology to barley (HvGs) while soybean <span class="gene">(GmG2)</span> and Arabidopsis (AGG1) were least related. The numbers of the exons and introns were found to be similar between RGG1(I) and RGG2(I), but their sizes were different. Analyses of promoter sequences of RGG(I) confirmed the presence of stress-related cis-regulatory signature motifs suggesting their active and possible independent roles in abiotic stress signaling. The transcript levels of RGG1(I) and RGG2(I) were upregulated following NaCl, cold, heat and ABA treatments. However, in <span class="trait">drought</span> stress only RGG1(I) was upregulated. Strong support by transcript profiling suggests that γ subunits play a critical role via cross talk in signaling pathways. These findings provide first direct evidence for roles of Gγ subunits of rice G-proteins in regulation of abiotic stresses. These findings suggest the possible exploitation of γ subunits of G-protein machinery for promoting stress tolerance in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The pathogenic development of Sclerotinia sclerotiorum in soybean requires specific host NADPH oxidases.</h2>
			<h3>PMID:28378935</h3>
				

				
			<p>
				作者：Ranjan A(1), Jayaraman D(2), Grau C(1), Hill JH(3), Whitham SA(3), Ané JM(2)(4), Smith DL(1), Kabbage M(1).
				<br/> 
				年份：2017 Apr 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant membrane-localized NADPH oxidases, also known as respiratory burst oxidase homologues (RBOHs), play crucial roles in various cellular activities, including plant disease responses, and are a major source of reactive oxygen species (ROS). Sclerotinia sclerotiorum is a cosmopolitan fungal pathogen that causes Sclerotinia stem rot (SSR) in soybean. Via a key virulence factor, oxalic acid, it induces programmed cell death (PCD) in the host plant, a process that is reliant on ROS generation. In this study, using protein sequence similarity searches, we identified 17 soybean RBOHs <span class="gene">(GmRBOHs)</span> and studied their contribution to SSR disease development, <span class="trait">drought</span> tolerance and nodulation. We clustered the soybean RBOH genes into six groups of orthologues based on phylogenetic analysis with their Arabidopsis counterparts. Transcript analysis of all 17 <span class="gene">GmRBOHs</span> revealed that, of the six identified groups, group VI <span class="gene">(GmRBOH-VI)</span> was specifically and drastically induced following S. sclerotiorum challenge. Virus-induced gene silencing (VIGS) of <span class="gene">GmRBOH-VI</span> using Bean pod mottle virus (BPMV) resulted in enhanced <span class="trait">resistance</span> to S. sclerotiorum and markedly reduced ROS levels during disease development. Coincidently, <span class="gene">GmRBOH-VI-silenced</span> plants were also found to be <span class="trait">drought</span> tolerant, but showed a reduced capacity to form nodules. Our results indicate that the pathogenic development of S. sclerotiorum in soybean requires the active participation of specific host RBOHs, to induce ROS and cell death, thus leading to the establishment of disease.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of CCA1-Like Proteins in Soybean and Functional Characterization of <span class="gene">GmMYB138a</span></h2>
			<h3>PMID:28937654</h3>
				

				
					<h3>PMCID:5666722</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bian </surname>  <given-names> Shaomin </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-ijms-18-02040"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> Donghao </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-ijms-18-02040"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ruihua </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-ijms-18-02040"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Xin </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Guoli </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Weikang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yuejia </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhai </surname>  <given-names> Lulu </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xuyan </given-names>  </name>  <xref rid="c1-ijms-18-02040" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant CIRCADIAN CLOCK ASSOCIATED1 (CCA1)-like proteins are a class of single-repeat MYELOBLASTOSIS ONCOGENE (MYB) transcription factors generally featured by a highly conserved motif SHAQK(Y/F)F, which play important roles in multiple biological processes. Soybean is an important grain legume for seed protein and edible vegetable oil. However, essential understandings regarding CCA1-like proteins are very limited in soybean. In this study, 54 CCA1-like proteins were identified by data mining of soybean genome. Phylogenetic analysis indicated that soybean CCA1-like subfamily showed evolutionary conservation and diversification. These CCA1-like genes displayed tissue-specific expression patterns, and analysis of genomic organization and evolution revealed 23 duplicated gene pairs. Among them, <span class="gene">GmMYB138a</span> was chosen for further investigation. Our protein-protein interaction studies revealed that <span class="gene">GmMYB138a,</span> but not its alternatively spliced isoform, interacts with a 14-3-3 protein <span class="gene">(GmSGF14l).</span> Although <span class="gene">GmMYB138a</span> was predominately localized in nucleus, the resulting complex of <span class="gene">GmMYB138a</span> and <span class="gene">GmSGF14l</span> was almost evenly distributed in nucleus and cytoplasm, supporting that 14-3-3s interact with their clients to alter their subcellular localization. Additionally, qPCR analysis suggested that <span class="gene">GmMYB138a</span> and <span class="gene">GmSGF14l</span> synergistically or antagonistically respond to <span class="trait">drought,</span> cold and salt stresses. Our findings will contribute to future research in regard to functions of soybean CCA1-like subfamily, especially regulatory mechanisms of <span class="gene">GmMYB138a</span> in response to abiotic stresses.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several pieces of evidence indicated that CCA1-like genes might be involved in plant stress responses [ <xref rid="B11-ijms-18-02040" ref-type="bibr"> 11 </xref> , <xref rid="B16-ijms-18-02040" ref-type="bibr"> 16 </xref> , <xref rid="B21-ijms-18-02040" ref-type="bibr"> 21 </xref> ]. To explore if <italic> <span class="gene">GmMYB138a</span> </italic> performs function in response to stresses, we examined its expression in whole seedlings under the stresses of cold, <span class="trait">drought</span> and salinity.
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> , the expression of <italic> <span class="gene">GmMYB138a</span> </italic> was significantly increased by cold, <span class="trait">drought</span> and salt stresses. Briefly, the transcripts of <italic> <span class="gene">GmMYB138a</span> </italic> were accumulated to its maximum level at 24 h after cold stress with 5.2 folds increase as compared to control ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> A), while a dramatic increase in <italic> <span class="gene">GmMYB138a</span> </italic> transcripts (2.3 and 2.1 folds, respectively as compared to control) was observed on Days 2 and 4 of <span class="trait">drought</span> treatment ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> B). When young seedlings were subjected to salt stress, the expression of <italic> <span class="gene">GmMYB138a</span> </italic> was remarkably pronounced by 2.4 folds at 12 h, and then remained a relatively stable level as stress prolongs ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> C).
					</p>
				
					<p>&nbsp;&nbsp;
						It is well known that 14-3-3s play important roles in response to stresses in plants. Since <span class="gene">GmSGF14l</span> exhibited interaction with <span class="gene">GmMYB138a,</span> we also investigated the expression of <italic> <span class="gene">GmSGF14l</span> </italic> under the above stresses. The expression patterns of <italic> <span class="gene">GmSGF14l</span> </italic> were similar to the ones of <italic> <span class="gene">GmMYB138a</span> </italic> when subjected to <span class="trait">drought</span> and salinity stresses. As shown in <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> B,C, the expression of <italic> <span class="gene">GmSGF14l</span> </italic> was increased to its maximum level at some stress point. In contrast, <italic> <span class="gene">GmSGF14l</span> </italic> was clearly decreased from 6 h to 48 h after cold stress ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> A), basically opposite to the expression pattern of <italic> <span class="gene">GmMYB138a</span> </italic> under cold stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Although plant CCA1-like proteins might perform important functions in multiple biological processes, the functional roles of <italic> <span class="gene">GmMYB138a</span> </italic> remain totally unknown in soybean. Recently, several studies indicated that CCA1-like proteins might be involved in response to various stresses [ <xref rid="B7-ijms-18-02040" ref-type="bibr"> 7 </xref> , <xref rid="B11-ijms-18-02040" ref-type="bibr"> 11 </xref> , <xref rid="B16-ijms-18-02040" ref-type="bibr"> 16 </xref> , <xref rid="B21-ijms-18-02040" ref-type="bibr"> 21 </xref> , <xref rid="B46-ijms-18-02040" ref-type="bibr"> 46 </xref> ]. Our study revealed that <span class="gene">GmMYB138a</span> interacts with a 14-3-3 protein, <span class="gene">GmSGF14l.</span> It has been well known that 14-3-3s are important players in the regulation of cellular responses to environmental stresses in plant [ <xref rid="B23-ijms-18-02040" ref-type="bibr"> 23 </xref> , <xref rid="B26-ijms-18-02040" ref-type="bibr"> 26 </xref> , <xref rid="B27-ijms-18-02040" ref-type="bibr"> 27 </xref> , <xref rid="B30-ijms-18-02040" ref-type="bibr"> 30 </xref> ]. In the present study, the expression patterns indicated that <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> might respond to cold, <span class="trait">drought</span> and salt stresses synergistically or antagonistically. Consistent with previous reports that <italic> CCA1-like </italic> gene expression is affected by cold, heat, osmotic, salt and oxidative stresses [ <xref rid="B49-ijms-18-02040" ref-type="bibr"> 49 </xref> , <xref rid="B50-ijms-18-02040" ref-type="bibr"> 50 </xref> ], the expression of <italic> <span class="gene">GmMYB138a</span> </italic> was significantly increased by cold, <span class="trait">drought</span> and salt stresses (5.20, 2.30 and 3.86 folds changes, respectively) ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> ), implying that <italic> <span class="gene">GmMYB138a</span> </italic> possibly functions as a modulator of stress-induced signaling pathways. Likewise, the expression patterns of <italic> <span class="gene">GmSGF14l</span> </italic> were also clearly increased when subjected to <span class="trait">drought</span> and salinity stresses ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> B,C), which are in agreement with previous reports that <italic> GsGF14o </italic> expression was greatly induced by <span class="trait">drought</span> stress [ <xref rid="B51-ijms-18-02040" ref-type="bibr"> 51 </xref> ], while <italic> BdGF14d </italic> confers salt tolerance in transgenic tobacco plants [ <xref rid="B52-ijms-18-02040" ref-type="bibr"> 52 </xref> ]. The similarity in response to <span class="trait">drought</span> and salt stress supported that signaling pathway activated by <span class="trait">drought</span> and salt are largely overlapping in plants [ <xref rid="B53-ijms-18-02040" ref-type="bibr"> 53 </xref> ]. In addition, similar expression patterns and protein interaction suggested that <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> might be in the same pathway in response to <span class="trait">drought</span> and salt stresses. Unlike its response to <span class="trait">drought</span> and salt stresses, however, <italic> <span class="gene">GmSGF14l</span> </italic> was clearly decreased from 6 h to 48 h after cold stress ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> A), consistent with previous reports that 14-3-3s can function as negative regulators of freezing tolerance and/or cold acclimation in Arabidopsis [ <xref rid="B54-ijms-18-02040" ref-type="bibr"> 54 </xref> , <xref rid="B55-ijms-18-02040" ref-type="bibr"> 55 </xref> ]. Although <span class="gene">GmSGF14l</span> and <span class="gene">GmMYB138a</span> can interact with each other, they were conversely expressed in response to cold stress ( <xref ref-type="fig" rid="ijms-18-02040-f008"> Figure 8 </xref> A), which is consistent with or previous study expressions of two 14-3-3 genes ( <italic> PvGF14e </italic> and <italic> PvGF14h </italic> ) were increased by cold stress and decreased by salinity and <span class="trait">drought</span> stress [ <xref rid="B56-ijms-18-02040" ref-type="bibr"> 56 </xref> ]. We assume that they are possibly involved in different pathways responsive for cold tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression analysis of <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> in response to abiotic stresses. Ten-day-old soybean seedlings were exposed to stress treatment as indicated below. Gene expression analysis was conducted by qRT-PCR using gene specific primers. ( <bold> A </bold> ) Gene expression pattern of <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> in seedlings exposed to cold stress for 0, 6, 12, 24, 48 and 72 h; ( <bold> B </bold> ) Gene expression pattern of <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> in seedlings exposed to <span class="trait">drought</span> stress for 0, 2, 4, 6, 8 and 10 days; ( <bold> C </bold> ) Gene expression pattern of <italic> <span class="gene">GmMYB138a</span> </italic> and <italic> <span class="gene">GmSGF14l</span> </italic> in seedlings exposed to salinity stress for 0, 6, 12, 24, 48 and 72 h. Error bars indicate SE (Standard error) of two biological and three technical replicates. Values were normalized against <italic> SUBI-3 </italic> . Significant differences are denoted by asterisks ( <italic> p </italic> &#x0003c; 0.01).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Heterologous Expression of Panax ginseng PgTIP1 Confers Enhanced Salt Tolerance of Soybean Cotyledon Hairy Roots, Composite, and Whole Plants.</h2>
			<h3>PMID:28769947</h3>
				

				
			<p>
				作者：An J(1), Hu Z(1), Che B(1), Chen H(2), Yu B(1), Cai W(2).
				<br/> 
				年份：2017 Jul 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Panax ginseng <span class="gene">TIP</span> gene PgTIP1 was previously demonstrated to have high water channel activity by its heterologous expression in Xenopus laevis oocytes and in yeast; it also plays a significant role in growth of PgTIP1-transgenic Arabidopsis plants under favorable conditions and has enhanced tolerance toward salt and <span class="trait">drought</span> treatment. In this work, we first investigated the physiological effects of heterologous PgTIP1 expression in soybean cotyledon hairy roots or composite plants mediated by Agrobacterium rhizogenes toward enhanced salt tolerance. The PgTIP1-transgenic soybean plants mediated by the pollen tube pathway, represented by the lines N and J11, were analyzed at the physiological and molecular levels for enhanced salt tolerance. The results showed that in terms of root-specific heterologous expression, the PgTIP1-transformed soybean cotyledon hairy roots or composite plants displayed superior salt tolerance compared to the empty vector-transformed ones according to the mitigatory effects of hairy root growth reduction, drop in leaf RWC, and rise in REL under salt stress. Additionally, declines in K+ content, increases in Na+ content and Na+/K+ ratios in the hairy roots, stems, or leaves were effectively alleviated by PgTIP1-transformation, particularly the stems and leaves of composite soybean plants. At the whole plant level, PgTIP1-trasgenic soybean lines were found to possess stronger root vigor, reduced root and leaf cell membrane damage, increased SOD, POD, CAT, and APX activities, steadily increased leaf Tr, RWC, and Pn values, and smaller declines in chlorophyll and carotenoid content when exposed to salt stress compared to wild type. Moreover, the distribution patterns of Na+, K+, and Cl- in the roots, stems, and leaves of salt-stressed transgenic plants were readjusted, in that the absorbed Na+ and Cl- were mainly restricted to the roots to reduce their transport to the shoots, and the transport of root-absorbed K+ to the shoots was simultaneously promoted. PgTIP1 transformation into soybean plants enhanced the expression of some stress-related genes <span class="gene">(GmPOD,</span> <span class="gene">GmAPX1,</span> <span class="gene">GmSOS1,</span> and <span class="gene">GmCLC1)</span> in the roots and leaves under salt treatment. This indicates that the causes of enhanced salt tolerance of heterologous PgTIP1-transformed soybean are associated with the positive regulation on water relations, ion homeostasis, and ROS scavenging under salt stress both at root-specific and whole plant levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peanut gene expression profiling in developing seeds at different reproduction stages during Aspergillus parasiticus infection.</h2>
			<h3>PMID:18248674  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo B(1), Chen X, Dang P, Scully BT, Liang X, Holbrook CC, Yu J, Culbreath AK.
				<br/> 
				年份：2008 Feb 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Peanut (Arachis hypogaea L.) is an important crop economically and nutritionally, and is one of the most susceptible host crops to colonization of Aspergillus parasiticus and subsequent aflatoxin contamination. Knowledge from molecular genetic studies could help to devise strategies in alleviating this problem; however, few peanut DNA sequences are available in the public database. In order to understand the molecular basis of host <span class="trait">resistance</span> to aflatoxin contamination, a large-scale project was conducted to generate expressed sequence tags (ESTs) from developing seeds to identify resistance-related genes involved in defense response against Aspergillus infection and subsequent aflatoxin contamination. RESULTS: We constructed six different cDNA libraries derived from developing peanut seeds at three reproduction stages (R5, R6 and R7) from a resistant and a susceptible cultivated peanut genotypes, 'Tifrunner' (susceptible to Aspergillus infection with higher aflatoxin contamination and resistant to TSWV) and 'GT-C20' (resistant to Aspergillus with reduced aflatoxin contamination and susceptible to TSWV). The developing peanut seed tissues were challenged by A. parasiticus and <span class="trait">drought</span> stress in the field. A total of 24,192 randomly selected cDNA clones from six libraries were sequenced. After removing vector sequences and quality trimming, 21,777 high-quality EST sequences were generated. Sequence clustering and assembling resulted in 8,689 unique EST sequences with 1,741 tentative consensus EST sequences (TCs) and 6,948 singleton ESTs. Functional classification was performed according to <span class="gene">MIPS</span> functional catalogue criteria. The unique EST sequences were divided into twenty-two categories. A similarity search against the non-redundant protein database available from NCBI indicated that 84.78% of total ESTs showed significant similarity to known proteins, of which 165 genes had been previously reported in peanuts. There were differences in overall expression patterns in different libraries and genotypes. A number of sequences were expressed throughout all of the libraries, representing constitutive expressed sequences. In order to identify resistance-related genes with significantly differential expression, a statistical analysis to estimate the relative abundance (R) was used to compare the relative abundance of each gene transcripts in each cDNA library. Thirty six and forty seven unique EST sequences with threshold of R &gt; 4 from libraries of 'GT-C20' and 'Tifrunner', respectively, were selected for examination of temporal gene expression patterns according to EST frequencies. Nine and eight resistance-related genes with significant up-regulation were obtained in 'GT-C20' and 'Tifrunner' libraries, respectively. Among them, three genes were common in both genotypes. Furthermore, a comparison of our EST sequences with other plant sequences in the TIGR Gene Indices libraries showed that the percentage of peanut EST matched to Arabidopsis thaliana, maize (Zea mays), Medicago truncatula, rapeseed (Brassica napus), rice (Oryza sativa), soybean (Glycine max) and wheat (Triticum aestivum) ESTs ranged from 33.84% to 79.46% with the sequence identity &gt;/= 80%. These results revealed that peanut ESTs are more closely related to legume species than to cereal crops, and more homologous to dicot than to monocot plant species. CONCLUSION: The developed ESTs can be used to discover novel sequences or genes, to identify resistance-related genes and to detect the differences among alleles or markers between these resistant and susceptible peanut genotypes. Additionally, this large collection of cultivated peanut EST sequences will make it possible to construct microarrays for gene expression studies and for further characterization of host <span class="trait">resistance</span> mechanisms. It will be a valuable genomic resource for the peanut community. The 21,777 ESTs have been deposited to the NCBI GenBank database with accession numbers ES702769 to ES724546.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Investigation of Genes Encoding Calcineurin B-Like Protein Family in Legumes and Their Expression Analyses in Chickpea ( <italic> Cicer arietinum </italic> L.)</h2>
			<h3>PMID:25853855</h3>
				
					<h3>PMCID:4390317</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Meena </surname>  <given-names> Mukesh Kumar </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ghawana </surname>  <given-names> Sanjay </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sardar </surname>  <given-names> Atish </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dwivedi </surname>  <given-names> Vikas </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khandal </surname>  <given-names> Hitaishi </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Roy </surname>  <given-names> Riti </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chattopadhyay </surname>  <given-names> Debasis </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Like other gene families, CBL genes have mostly been studied in <italic> Arabidopsis </italic> and information on CBL genes from other plants is limited. Overexpression of soybean ( <italic> Glycine max </italic> ) CBL1 <span class="gene">(GmCBL1),</span> an ortholog of AtCBL1, enhances tolerance to salinity and <span class="trait">drought</span> stress in <italic> Arabidopsis </italic> [ <xref rid="pone.0123640.ref022" ref-type="bibr"> 22 </xref> ]. CBL3 from chickpea ( <italic> Cicer arietinum </italic> ) was shown to interact with CIPK6 and mediate response to dehydration and salinity stresses [ <xref rid="pone.0123640.ref023" ref-type="bibr"> 23 </xref> ]. PeCBL1 interacts with CIPK24, CIPK25 and CIPK26 to regulate Na <sup> + </sup> /K <sup> + </sup> homeostasis in <italic> Populus euphratica </italic> [ <xref rid="pone.0123640.ref024" ref-type="bibr"> 24 </xref> ]. In <italic> Brassica napus </italic> , BnaCBL1-CIPK6 complex mediates plant response to high salinity, phosphorous deficiency and ABA signaling [ <xref rid="pone.0123640.ref025" ref-type="bibr"> 25 </xref> ]. Grapevine ( <italic> Vitis vinifera </italic> ) VvCBL1-VvCIPK4 and VvCBL2-VvCIPK3 complexes control activity of K <sup> + </sup> channel [ <xref rid="pone.0123640.ref026" ref-type="bibr"> 26 </xref> ]. CBL proteins also play role in plant immunity. Tomato CBL10 and CIPK6 interact with respiratory burst homolog B protein (RbohB) at the plasma membrane and contribute to ROS generation during effector-triggered immunity [ <xref rid="pone.0123640.ref027" ref-type="bibr"> 27 </xref> ]. From all the studies mentioned above and from the numbers of CBLs and CIPKs in a species, it is evident that a particular CBL can interact with and activate multiple CIPKs depending upon circumstances and CBL-CIPK pathways are essential for important cellular processes mostly for stress signaling.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A 14-3-3 Family Protein from Wild Soybean ( <italic> Glycine Soja </italic> ) Regulates ABA Sensitivity in <italic> Arabidopsis </italic></h2>
			<h3>PMID:26717241</h3>
				
					<h3>PMCID:4696740</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Xiaoli </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Mingzhe </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jia </surname>  <given-names> Bowei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Chao </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qin </surname>  <given-names> Zhiwei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Kejun </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meiping </surname>  <given-names> Zhang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mingyang </surname>  <given-names> Cong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Yanming </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The 14-3-3 family proteins are phosphor-serine/threonine-binding proteins that regulate a wide array of targets via direct protein-protein interaction [ <xref rid="pone.0146163.ref033" ref-type="bibr"> 33 </xref> , <xref rid="pone.0146163.ref034" ref-type="bibr"> 34 </xref> ]. Up to now, researches have demonstrated the crucial regulatory roles of 14-3-3s in ABA stress responses [ <xref rid="pone.0146163.ref035" ref-type="bibr"> 35 </xref> , <xref rid="pone.0146163.ref036" ref-type="bibr"> 36 </xref> ]. The 14-3-3s and ABFs interaction has been identified in <italic> Arabidopsis </italic> [ <xref rid="pone.0146163.ref036" ref-type="bibr"> 36 </xref> ], <italic> Thellungiella </italic> [ <xref rid="pone.0146163.ref032" ref-type="bibr"> 32 </xref> ] and barley [ <xref rid="pone.0146163.ref029" ref-type="bibr"> 29 </xref> ]. By conducting genome-wide analysis, previous studies have identified the 14-3-3 family proteins in soybean [ <xref rid="pone.0146163.ref037" ref-type="bibr"> 37 </xref> , <xref rid="pone.0146163.ref038" ref-type="bibr"> 38 </xref> ]. Among them, SGF14c and SGF14l are found to play critical roles during the early developmental stages of soybean nodules [ <xref rid="pone.0146163.ref039" ref-type="bibr"> 39 </xref> ]. In addition, SGF14l also could affect isoflavonoid synthesis by regulating the intracellular localization of the <span class="gene">GmMYB176</span> transcription factor [ <xref rid="pone.0146163.ref040" ref-type="bibr"> 40 </xref> , <xref rid="pone.0146163.ref041" ref-type="bibr"> 41 </xref> ]. In a previous study, we functionally characterized a 14-3-3 family gene from wild soybean ( <italic> Glycine soja </italic> ), <italic> GsGF14o </italic> , which participated in stomata and root hair development, and negatively regulated plant <span class="trait">drought</span> tolerance [ <xref rid="pone.0146163.ref037" ref-type="bibr"> 37 </xref> ]. However, until now, there is no direct genetic evidence showing the involvement of soybean 14-3-3 proteins in ABA responses.
					</p>
				
					<p>&nbsp;&nbsp;
						ABA is generally considered as an important hormone, and helps plants to deal with <span class="trait">drought</span> stress [ <xref rid="pone.0146163.ref083" ref-type="bibr"> 83 </xref> , <xref rid="pone.0146163.ref084" ref-type="bibr"> 84 </xref> ]. However, a serial of researches have suggested that an increased ABA sensitivity was not necessarily accompanied with an increased stress tolerance in plants. Plants showing ABA hypersensitivity may also display hypersensitivity to <span class="trait">drought</span> stress, in other words, <span class="trait">drought</span> stress responses could be controlled by ABA-dependent or ABA-independent pathway. One of the best studied examples is that the ABA overly sensitive mutant <italic> abo3 </italic> displayed decreased <span class="trait">drought</span> tolerance [ <xref rid="pone.0146163.ref085" ref-type="bibr"> 85 </xref> ]. Similar results were also found for <italic> Arabidopsis </italic> histone deacetylase gene <italic> AtHD2C </italic> and Lily ABA-, stress-, and ripening-induced gene <italic> LLA23 </italic> , overexpression of which decreased plant ABA sensitivity but increased <span class="trait">drought</span> tolerance [ <xref rid="pone.0146163.ref086" ref-type="bibr"> 86 </xref> , <xref rid="pone.0146163.ref087" ref-type="bibr"> 87 </xref> ]. Very recently, transgenic <italic> Arabidopsis </italic> overexpressing a rice group A PP2C gene <italic> OsPP108 </italic> was found to be highly insensitive to ABA but tolerant to <span class="trait">drought</span> stress [ <xref rid="pone.0146163.ref052" ref-type="bibr"> 52 </xref> ]. Similarly, our research also revealed that <italic> GsGF14o </italic> overexpression increased ABA sensitivity, but decreased <span class="trait">drought</span> tolerance. Remarkably, expression of the ABA inducible gene <italic> <span class="gene">RD29A</span> </italic> in <italic> OsPP108 </italic> transgenic lines was down-regulated under ABA treatment, but up-regulated under <span class="trait">drought</span> stress. Similarly, our results also showed that expression of the ABA inducible stress responsive genes was up-regulated under ABA treatment ( <xref rid="pone.0146163.g010" ref-type="fig"> Fig 10 </xref> ), but down-regulated under <span class="trait">drought</span> stress [ <xref rid="pone.0146163.ref037" ref-type="bibr"> 37 </xref> ]. Generally, ABA induced stomata closure is an important adaptive response to <span class="trait">drought</span> stress, resulting in reduced water loss [ <xref rid="pone.0146163.ref088" ref-type="bibr"> 88 </xref> , <xref rid="pone.0146163.ref089" ref-type="bibr"> 89 </xref> ]. The <italic> OsPP108 </italic> transgenic lines showed ABA insensitivity, with much more open stomata after ABA treatment than WT; in contrast, the water loss of <italic> OsPP108 </italic> transgenic lines obviously decreased under <span class="trait">drought</span> stress [ <xref rid="pone.0146163.ref052" ref-type="bibr"> 52 </xref> ]. Opposite phenomenon was also observed for <italic> GsGF14o </italic> . <italic> GsGF14o </italic> transgenic lines displayed ABA hypersensitivity concerning stomata closure ( <xref rid="pone.0146163.g009" ref-type="fig"> Fig 9 </xref> ), but exhibited decreased <span class="trait">drought</span> tolerance, even though with lower water loss rates [ <xref rid="pone.0146163.ref037" ref-type="bibr"> 37 </xref> ]. The contradiction in ABA sensitivity and <span class="trait">drought</span> tolerance could be explained by the hypothesis that this kind of genes might regulate stress tolerance through ABA-independent mechanism.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular, anatomical and physiological properties of a genetically modified soybean line transformed with rd29A:AtDREB1A for the improvement of <span class="trait">drought</span> tolerance.</h2>
			<h3>PMID:22033903  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Polizel AM(1), Medri ME, Nakashima K, Yamanaka N, Farias JR, de Oliveira MC, Marin SR, Abdelnoor RV, Marcelino-Guimarães FC, Fuganti R, Rodrigues FA, Stolf-Moreira R, Beneventi MA, Rolla AA, Neumaier N, Yamaguchi-Shinozaki K, Carvalho JF, Nepomuceno AL.
				<br/> 
				年份：2011 Oct 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We evaluated the molecular, anatomical and physiological properties of a soybean line transformed to improve <span class="trait">drought</span> tolerance with an rd29A:AtDREB1A construct. This construct expressed dehydration- responsive element binding protein DREB1A from the stress-inducible rd29A promoter. The greenhouse growth test included four randomized blocks of soybean plants, with each treatment performed in triplicate. Seeds from the non-transformed soybean cultivar BR16 and from the genetically modified soybean P58 line (T(2) generation) were grown at 15% gravimetric humidity for 31 days. To induce water deficit, the humidity was reduced to 5% gravimetric humidity (moderate stress) for 29 days and then to 2.5% gravimetric humidity (severe stress). AtDREB1A gene expression was higher in the genetically modified P58 plants during water deficit, demonstrating transgene stability in T(2) generations and induction of the rd29A promoter. Drought-response genes, including <span class="gene">GmPI-PLC,</span> <span class="gene">GmSTP,</span> <span class="gene">GmGRP,</span> and <span class="gene">GmLEA14,</span> were highly expressed in plants submitted to severe stress. Genetically modified plants had higher stomatal conductance and consequently higher photosynthetic and transpiration rates. In addition, they had more chlorophyll. Overexpression of AtDREB1A may contribute to a decrease in leaf thickness; however, a thicker abaxial epidermis was observed. Overexpression of AtDREB1A in soybean appears to enhance <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis and expression profiling of glyoxalase gene families in soybean ( <italic> Glycine max </italic> ) indicate their development and abiotic stress specific response</h2>
			<h3>PMID:27083416</h3>
				

				
					<h3>PMCID:4833937</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Ghosh </surname>  <given-names> Ajit </given-names>  </name>  <address>  <phone> +880-821-713491 </phone>  <phone> +88-01679436700 </phone>  <email> ajitghoshbd@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Islam </surname>  <given-names> Tahmina </given-names>  </name>  <address>  <email> subarna.islam@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> Department of Biochemistry and Molecular Biology, Shahjalal University of Science and Technology, Sylhet, 3114 Bangladesh  </aff>  <aff id="Aff2">  <label/> Plant Breeding and Biotechnology Laboratory, Department of Botany, Dhaka University, Dhaka, 1000 Bangladesh  </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Glyoxalase pathway consists of two enzymes, glyoxalase I (GLYI) and glyoxalase II (GLYII) which detoxifies a highly cytotoxic metabolite methylglyoxal (MG) to its non-toxic form. MG may form advanced glycation end products with various cellular macro-molecules such as proteins, DNA and RNA; that ultimately lead to their inactivation. Role of glyoxalase enzymes has been extensively investigated in various plant species which showed their crucial role in salinity, <span class="trait">drought</span> and heavy metal stress tolerance. Previously genome-wide analysis of glyoxalase genes has been conducted in model plants Arabidopsis and rice, but no such study was performed in any legume species. RESULTS: In the present study, a comprehensive genome database analysis of soybean was performed and identified a total of putative 41 GLYI and 23 GLYII proteins encoded by 24 and 12 genes, respectively. Detailed analysis of these identified members was conducted including their nomenclature and classification, chromosomal distribution and duplication, exon-intron organization, and protein domain(s) and motifs identification. Expression profiling of these genes has been performed in different tissues and developmental stages as well as under salinity and <span class="trait">drought</span> stresses using publicly available RNAseq and microarray data. The study revealed that <span class="gene">GmGLYI-7</span> and <span class="gene">GmGLYII-8</span> have been expressed intensively in all the developmental stages and tissues; while <span class="gene">GmGLYI-6,</span> <span class="gene">GmGLYI-9,</span> <span class="gene">GmGLYI-20,</span> <span class="gene">GmGLYII-5</span> and <span class="gene">GmGLYII-10</span> were highly abiotic stress responsive members. CONCLUSIONS: The present study identifies the largest family of glyoxalase proteins to date with 41 <span class="gene">GmGLYI</span> and 23 <span class="gene">GmGLYII</span> members in soybean. Detailed analysis of <span class="gene">GmGLYI</span> and <span class="gene">GmGLYII</span> genes strongly indicates the genome-wide segmental and tandem duplication of the glyoxalase members. Moreover, this study provides a strong basis about the biological role and function of <span class="gene">GmGLYI</span> and <span class="gene">GmGLYII</span> members in soybean growth, development and stress physiology.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, a comprehensive genome database analysis of soybean was performed and identified a total of putative 41 GLYI and 23 GLYII proteins encoded by 24 and 12 genes, respectively. Detailed analysis of these identified members was conducted including their nomenclature and classification, chromosomal distribution and duplication, exon-intron organization, and protein domain(s) and motifs identification. Expression profiling of these genes has been performed in different tissues and developmental stages as well as under salinity and <span class="trait">drought</span> stresses using publicly available RNAseq and microarray data. The study revealed that <italic> <span class="gene">GmGLYI</span> </italic> -7 and <italic> <span class="gene">GmGLYII</span> </italic> -8 have been expressed intensively in all the developmental stages and tissues; while <italic> <span class="gene">GmGLYI</span> </italic> -6, <italic> <span class="gene">GmGLYI</span> </italic> -9, <italic> <span class="gene">GmGLYI</span> </italic> -20, <italic> <span class="gene">GmGLYII-</span> </italic> 5 and <italic> <span class="gene">GmGLYII-</span> </italic> 10 were highly abiotic stress responsive members.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain deep insights into the function of glyoxalase genes in the abiotic stress adaptation of soybean, the expression profiles of all <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes were analyzed in response to salinity and <span class="trait">drought</span> stresses using publicly available microarray data. Expression data sets were retrieved from gene expression omnibus database of GSE41125 and GSE40627 for salinity and <span class="trait">drought</span> stress, respectively. Data were not available for all these genes due to the limitation of respective probes. Among the total of 24 <italic> <span class="gene">GmGLYI</span> </italic> and 12 <italic> <span class="gene">GmGLYII</span> </italic> genes; data for 19 <italic> <span class="gene">GmGLYI</span> </italic> and eight <italic> <span class="gene">GmGLYII</span> </italic> genes were analyzed for salinity, while expression data of 21 <italic> <span class="gene">GmGLYI</span> </italic> and 12 <italic> <span class="gene">GmGLYII</span> </italic> genes were found for <span class="trait">drought</span> stress. Different glyoxalase members responded differentially in terms of their expression towards both these stresses (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> and <xref rid="Fig10" ref-type="fig"> b </xref> ). In response to salinity stress, four <italic> <span class="gene">GmGLYI</span> </italic> genes and two <italic> <span class="gene">GmGLYII</span> </italic> genes showed up-regulation, while seven <italic> <span class="gene">GmGLYI</span> </italic> genes and four <italic> <span class="gene">GmGLYII</span> </italic> genes showed down-regulation and rest of them remained unaltered (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> and <xref rid="Fig10" ref-type="fig"> b </xref> ). Similarly, <span class="trait">drought</span> stress causes upregulation of seven <italic> <span class="gene">GmGLYI</span> </italic> genes and five <italic> <span class="gene">GmGLYII</span> </italic> genes, and down-regulation of eight <italic> <span class="gene">GmGLYI</span> </italic> genes and six <italic> <span class="gene">GmGLYII</span> </italic> genes (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> and <xref rid="Fig10" ref-type="fig"> b </xref> ). Among 24 <italic> <span class="gene">GmGLYI</span> </italic> genes; <italic> <span class="gene">GmGLYI</span> </italic> -6 and <italic> <span class="gene">GmGLYI</span> </italic> -9 showed significant up-regulation, while other two of them ( <italic> <span class="gene">GmGLYI</span> </italic> -3 and <italic> <span class="gene">GmGLYI</span> </italic> -10) showed down-regulation in both the stresses (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> ). In case of <italic> <span class="gene">GmGLYII</span> </italic> ; <italic> <span class="gene">GmGLYII</span> </italic> -4 and <italic> <span class="gene">GmGLYII</span> </italic> -5 showed up-regulation, while <italic> <span class="gene">GmGLYII</span> </italic> -10 showed significant down-regulation in both salinity and <span class="trait">drought</span> stresses (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10b </xref> ). Rest of the members of both <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> family showed variable expression pattern. This indicates diverse role of different glyoxalase members in the stress modulation pathways of soybean plant. <fig id="Fig10"> <label> Fig. 10 </label> <caption> <p> Expression analyses of soybean glyoxalase genes in response to salinity, <span class="trait">drought</span> and hormonal treatment. Relative expression data of all available <italic> <span class="gene">GmGLYI</span> </italic> ( <bold> a </bold> ) and <italic> <span class="gene">GmGLYII</span> </italic> ( <bold> b </bold> ) genes under salinity and <span class="trait">drought</span> stresses were obtained from the National Center for Biotechnology Information GEO database ( <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/"> http://www.ncbi.nlm.nih.gov/geo/ </ext-link> ). Expression data is presented as fold-change by comparing with the corresponding mock samples. Blue colour bar represents data for salinity stress, while red colour indicates <span class="trait">drought</span> stress. <bold> c </bold> Semiquantitative RT-PCR of four <italic> <span class="gene">GmGLYI</span> </italic> genes ( <italic> <span class="gene">GmGLYI</span> </italic> -3, <italic> <span class="gene">GmGLYI</span> </italic> -6, <italic> <span class="gene">GmGLYI</span> </italic> -16 and <italic> <span class="gene">GmGLYI</span> </italic> -20), four <italic> <span class="gene">GmGLYII</span> </italic> genes ( <italic> <span class="gene">GmGLYI</span> </italic> -4, <italic> <span class="gene">GmGLYI</span> </italic> -5, <italic> <span class="gene">GmGLYI-9</span> </italic> and <italic> <span class="gene">GmGLYI-10</span> </italic> ) and one house-keeping control gene, <italic> Tubulin </italic> under different conditions such as control, salinity, <span class="trait">drought</span> and ABA treatment. <bold> d </bold> Relative expression analysis of the representative eight glyoxalase genes were analyzed by measuring the PCR band intensity using Image J software and represent as relative fold change in expression
					</p>
				
					<p>&nbsp;&nbsp;
						To screen the role of soybean glyoxalase genes in response to salinity, <span class="trait">drought</span> and hormonal treatment (Abscisic Acid, ABA), a semi quantitative RT-PCR was performed to validate four candidate <italic> <span class="gene">GmGLYI</span> </italic> genes ( <italic> <span class="gene">GmGLYI-3</span> </italic> , &#x02212; <italic> 6 </italic> ,- <italic> 16 </italic> and &#x02212; <italic> 20 </italic> ) and four candidate <italic> <span class="gene">GmGLYII</span> </italic> genes ( <italic> <span class="gene">GmGLYII-4</span> </italic> , &#x02212; <italic> 5 </italic> ,- <italic> 9 </italic> and &#x02212; <italic> 10 </italic> ) which were highly responsive in microarray data analysis (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> and <xref rid="Fig10" ref-type="fig"> b </xref> ). For this purpose, 15&#x000a0;days old soybean seedlings were subjected to normal water (as control), 200&#x000a0;mM NaCl (for salinity) or withdrawn of water (for <span class="trait">drought)</span> or 10&#x000a0;mM ABA (for hormonal treatment) for 8&#x000a0;h. Expression of all the candidate genes were compared with that of <italic> Tubulin </italic> (act as a house-keeping control gene) (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10c </xref> ). Relative transcript abundance of all these transcripts was measured by scanning the gel image using Image J software relative fold change in expression was calculated considering <italic> Tubulin </italic> as internal control (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10d </xref> ). It could be clearly inferred from Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10d </xref> that <italic> <span class="gene">GmGLYI-</span> </italic> 6, <italic> <span class="gene">GmGLYII-</span> </italic> 4 and <italic> <span class="gene">GmGLYII-</span> </italic> 5 showed strong up-regulation in response to both salinity and <span class="trait">drought,</span> while <italic> <span class="gene">GmGLYI-</span> </italic> 20, <italic> <span class="gene">GmGLYII-</span> </italic> 9 and <italic> <span class="gene">GmGLYII-</span> </italic> 10 showed clear down-regulation (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10d </xref> ). The remaining two members, <italic> <span class="gene">GmGLYI-</span> </italic> 3 and <italic> <span class="gene">GmGLYI-</span> </italic> 16 showed slight up/down regulation as compared to control sample. Overall, the pattern of expression of these eight candidate genes (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10c </xref> ) was found to be almost similar to that of microarray data (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> and <xref rid="Fig10" ref-type="fig"> b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In order to comment on stress responsive expression of <italic> <span class="gene">GmGLY</span> </italic> genes in response to salinity, <span class="trait">drought</span> and ABA treatment, 1&#x000a0;kb upstream promoter region of each <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes were retrieved from soybase ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org/dlpages/flank/index.php"> http://www.soybase.org/dlpages/flank/index.php </ext-link> ) and analyzed for the presence of cis-acting elements using PlantCARE [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. This analysis leads to the identification of several stress-responsive cis elements such as abscisic acid responsive element (ABRE), auxin responsive element (AuxRR-core), fungal elicitor responsive element (BOX-W1), ethylene responsive element (ERE), gibberellin-responsive element (GARE), heat shock element (HSE), jasmonate and elicitor responsive element (JERE), low temperature responsive element (LTR), MYB-binding site (MBS), defence and stress responsive element (TC-rich), wounding and pathogen responsive elements (W-box and WUN-motif), salicylic acid responsive element (TCA), Methyl jasmonate-responsive element (CGTCA box and TGACG motif), element conferring high transcription level (5&#x02019; UTR Py-rich stretch). All these motifs are very crucial for plant stress modulation pathways and thus play important role to regulate the expression of various stress responsive genes [ <xref ref-type="bibr" rid="CR45"> 45 </xref> , <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. All these motifs were found to be distributed randomly in both the positive and negative strands of promoter sequences (Fig.&#x000a0; <xref rid="Fig11" ref-type="fig"> 11 </xref> ). Among <italic> <span class="gene">GmGLYI</span> </italic> members, <italic> <span class="gene">GmGLYI-1</span> </italic> and <italic> <span class="gene">GmGLYI-24</span> </italic> have maximum cis-elements (12 elements), while <italic> <span class="gene">GmGLYI-14</span> </italic> promoter has minimum two elements on it. In case of <italic> <span class="gene">GmGLYII</span> </italic> members, <italic> <span class="gene">GmGLYII-7</span> </italic> has maximum ten elements, while <italic> <span class="gene">GmGLYII-8</span> </italic> has only one element. ABRE, HSE, and TGACG motif are found to be present in almost every promoter of <italic> <span class="gene">GmGLY</span> </italic> genes with few exceptions. Although correlation between the presence of cis-acting regulatory elements and the observed transcript abundance needs to be confirmed experimentally, these results indicated the stress-responsive nature of <italic> <span class="gene">GmGLY</span> </italic> genes. <fig id="Fig11"> <label> Fig. 11 </label> <caption> <p> <italic> In silico </italic> promoter analysis of <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes. One kb 5&#x02019; upstream sequence of all <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes was downloaded from soybase database and scanned through PlantCARE for the identification of number and position of various cis-acting regulatory elements. Different regulatory elements were indicated by different colour symbols and placed in their relative position on the promoter. Symbols presented above the line indicate forward strand of DNA, while below one indicates the reverse strand
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of glyoxalase genes has been found to be highly specific towards certain tissue or developmental stages in <italic> Arabidopsis </italic> and rice [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. Thus, the expression pattern of <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYI</span> </italic> genes was analyzed at different developmental stages and tissues (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ). These data revealed the tissue specific expression pattern of glyoxalase genes in soybean too. Out of all, <italic> <span class="gene">GmGLYI</span> </italic> -7, <italic> <span class="gene">GmGLYI</span> </italic> -21 and <italic> <span class="gene">GmGLYII</span> </italic> -8 are found to be the constitutively expressed members of soybean glyoxalase system. A cluster of <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes maintained high level of expression in all the underground and aerial tissues, followed by low level of expression in the different stages of seed development (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ). This indicates the presence of functional distribution among the multiple members in different tissue/developmental stimuli. On contrary, some of the <italic> <span class="gene">GmGLYI</span> </italic> genes such as <italic> <span class="gene">GmGLYI</span> </italic> -1, <italic> <span class="gene">GmGLYI</span> </italic> -11, and <italic> <span class="gene">GmGLYI</span> </italic> -22 showed medium level of expression in the seed tissues only, with low or no expression in other parts (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ). This indicates development specific transition/regulation of <italic> <span class="gene">GmGLYI</span> </italic> genes. In case of <italic> <span class="gene">GmGLYII</span> </italic> genes, two distinguishable clades were observed in their expression where one set showed high level expression in all the tissues and another have no expression at all. These low expressive genes might have other cellular/metabolic regulation expect developmental/tissue regulation. Another interesting expression pattern was observed for soybean glyoxalase genes under abiotic stresses (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10 </xref> ). Different members of <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> families responded differentially against salinity, <span class="trait">drought</span> and hormone (ABA) treatment (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10c </xref> ). <italic> <span class="gene">GmGLYII</span> </italic> -9 and <italic> <span class="gene">GmGLYII</span> </italic> -10 showed strong down-regulation in response to all three conditions, while <italic> <span class="gene">GmGLYII</span> </italic> -16 showed sharp up-regulation (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10d </xref> ). Presence of various cis-acting regulatory elements on the putative promoter sequence of <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes might be a probable reason behind this altered expression (Fig.&#x000a0; <xref rid="Fig11" ref-type="fig"> 11 </xref> ). Similar pattern of expression was observed previously in rice and <italic> Arabidopsis </italic> glyoxalase genes [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ], where each member shows specific pattern of expression towards the particular type of stress treatment. Overall, the observed information in the present study will facilitate to find out the appropriate candidate gene(s) for further functional characterization and raising stress-tolerant transgenic crop plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression data of glyoxalase genes in response to salinity and <span class="trait">drought</span> stresses were retrieved from the National Center for Biotechnology Information Gene Expression Omnibus (GEO) database [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ] with accession numbers GSE41125 and GSE40627, respectively. Corresponding probe sets for <italic> <span class="gene">GmGLYI</span> </italic> and <italic> <span class="gene">GmGLYII</span> </italic> genes were identified using NetAffx Analysis Center ( <ext-link ext-link-type="uri" xlink:href="http://www.affymetrix.com/analysis/index.affx?navMode=cat530006&#x00026;aId=netaffxNav"> http://www.affymetrix.com/analysis/index.affx?navMode=cat530006&#x00026;aId=netaffxNav </ext-link> ) online Probe Match tool. More than one gene with the same probe set, were considered as same transcriptional profile, while in case of gene having more than one probe set, the highest value was considered. Expression data were normalized using that of mock, and represented as bar diagram.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> in Glycine max L. that enhances abiotic stress tolerance in Saccharomyces cerevisiae.</h2>
			<h3>PMID:20155384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Zhao H, Liu Y, Li Q, Liu X, Tan H, Yuan C, Dong Y.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> was isolated from Glycine max. It is 1,596 bp in length with one ORF of 410 amino acids. Southern blot analysis revealed that it has at least two copies in the G. max genome. <span class="gene">GmGSK,</span> when transiently expressed in Nicotiana tabacum leaves, was localized in both cell membrane and cytoplasm. Northern blot analysis indicated that <span class="gene">GmGSK</span> is expressed in all tissues, with highest expression in the root. <span class="gene">GmGSK</span> can be induced by various abiotic stresses. When transformed with <span class="gene">GmGSK,</span> Saccharomyces cerevisiae exhibited enhanced <span class="trait">resistance</span> to salt and <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue-Specific Regulation of <span class="gene">Gma-miR396</span> Family on Coordinating Development and Low Water Availability Responses.</h2>
			<h3>PMID:28694817</h3>
				

				
			<p>
				作者：Liu W(1), Zhou Y(1), Li X(1), Wang X(1), Dong Y(1), Wang N(1), Liu X(1), Chen H(1), Yao N(1), Cui X(1), Jameel A(1), Wang F(1), Li H(1).
				<br/> 
				年份：2017 Jun 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previously, it was reported that miR396s interact with growth-regulating factors (GRFs) to modulate plant growth, development, and stress <span class="trait">resistance.</span> In soybean, 11 gma-miR396 precursors (Pre-miR396a-k) were found, and 24 <span class="gene">GmGRFs</span> were predicted as targets of seven mature gma-miR396s (gma-miR396a/b/c/e/h/i/k). To explore the roles of the miR396-GRF module in low water availability response of soybean, we analyzed the expression of Pre-miR396a-k, and found that Pre-miR396a/i/bdgk/e/h were up-regulated in leaves and down-regulated in roots; on the contrary, <span class="gene">GmGRF5/6/7/8/15/17/21</span> were down-regulated in leaves and <span class="gene">GmGRF1/2/17/18/19/20/21/22/23/24</span> were up-regulated in roots of low water potential stressed soybean. Any one of gma-miR396a/b/c/e/h/i/k was able to interact with 20 <span class="gene">GmGRFs</span> <span class="gene">(GmGRF1/2/6-11/13-24),</span> confirming that this module represents a multi-to-multi network interaction. We generated Arabidopsis plants over-expressing each of the 11 gma-miR396 precursors (Pre-miR396a-k), and seven of them (miR396a/b/c/e/h/i/k-OE transgenic Arabidopsis) showed altered development. The low water availability of miR396a/b/c/e/h/i/k-OE was enhanced in leaves but reduced in seeds and roots. Contrary to previous reports, miR396a/b/c/i-OE seedlings showed lower survival rate than WT when recovering after rewatering under soil drying. In general, we believe our findings are valuable to understand the role of gma-miR396 family in coordinating development and low water availability responses, and can provide potential strategies and directions for soybean breeding programs to improve seed <span class="trait">yield</span> and plant <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Trihelix transcription factor <italic> GT </italic> - <italic> 4 </italic> mediates salt tolerance via interaction with <italic> TEM2 </italic> in <italic> Arabidopsis </italic></h2>
			<h3>PMID:25465615</h3>
				
					<h3>PMCID:4267404</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Xiao-Hong </given-names>  </name>  <address>  <email> xhwang@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Qing-Tian </given-names>  </name>  <address>  <email> qtli@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Hao-Wei </given-names>  </name>  <address>  <email> hwchen@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Wan-Ke </given-names>  </name>  <address>  <email> wkzhang@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Biao </given-names>  </name>  <address>  <email> mabiao@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Chen </surname>  <given-names> Shou-Yi </given-names>  </name>  <address>  <email> sychen@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhang </surname>  <given-names> Jin-Song </given-names>  </name>  <address>  <email> jszhang@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> State Key Lab of Plant Genomics, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, 100101 China  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 11 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Arabidopsis <italic> PETAL LOSS </italic> ( <italic> PTL </italic> ), which belongs to the GT-2 group, was the first trihelix gene identified associating with a morphogenetic function. PTL regulates petal and sepal growth, and sepal fusion [ <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR28"> 28 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. Rice Shattering 1( <italic> SHA1 </italic> ) gene encoding a SH4-type factor plays an important role in activation of cell separation, and a mutation in the trihelix domain results in the elimination of seed shattering in cultivated rice [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. More recently, ASIL1, belonging to a new subfamily of trihelix family, has been found to function as a negative regulator of a large subset of Arabidopsis embryonic and seed maturation genes in Arabidopsis seedlings [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. The role for GT-3b in responding to salt and pathogen stress is also identified in Arabidopsis. <italic> GT-3b </italic> expression is rapidly induced by NaCl and Pseudo-monas syringae infection and GT-3b binds to a GT-like element (GAAAAA) in the promoter of a calmodulin gene ( <italic> <span class="gene">SCaM-4</span> </italic> ) [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. We demonstrates that overexpression of <italic> <span class="gene">GmGT-2A</span> </italic> or <italic> <span class="gene">GmGT-2B</span> </italic> from soybean enhanced tolerance to salt, <span class="trait">drought</span> and freezing stresses in transgenic Arabidopsis plants [ <xref ref-type="bibr" rid="CR29"> 29 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification of Glyoxalase Genes in <italic> Medicago truncatula </italic> and Their Expression Profiling in Response to Various Developmental and Environmental Stimuli</h2>
			<h3>PMID:28620395</h3>
				
					<h3>PMCID:5452422</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ghosh </surname>  <given-names> Ajit </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/256089/overview"/>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Different glyoxalase members of <italic> Arabidopsis </italic> and rice such as <italic> AtGLYI </italic> 4, <italic> AtGLYI </italic> 7, <italic> OsGLYI </italic> 6, and <italic> OsGLYI </italic> 11 were previously reported for their highly stress inducible expression pattern ( <xref rid="B37" ref-type="bibr"> Mustafiz et al., 2011 </xref> ). Similarly, several soybean glyoxalase members such as <italic> <span class="gene">GmGLYI</span> </italic> 6, <italic> <span class="gene">GmGLYI</span> </italic> 9, <italic> <span class="gene">GmGLYI</span> </italic> 20, <italic> <span class="gene">GmGLYII</span> </italic> 5, and <italic> <span class="gene">GmGLYII</span> </italic> 10 were reported for their highly abiotic stress responsive expression ( <xref rid="B10" ref-type="bibr"> Ghosh and Islam, 2016 </xref> ). Moreover, few other soybean glyoxalase members such as <italic> <span class="gene">GmGLYI</span> </italic> -6/9, <italic> <span class="gene">GmGLYI</span> </italic> -20, <italic> <span class="gene">GmGLYII</span> </italic> -6, and <italic> <span class="gene">GmGLYII</span> </italic> -10 showed biotic stress specific modulation in response to various pathogenic infection ( <xref rid="B9" ref-type="bibr"> Ghosh, 2016 </xref> ). Stress specific transcript alteration is a highly complex process and vary depending on the duration and type of stress; and genotype of plant. Expression pattern of all <italic> Medicago </italic> glyoxalase genes was further analyzed in response to two devastating abiotic stresses-salinity and <span class="trait">drought</span> ( <bold> Figure <xref ref-type="fig" rid="F10"> 10 </xref> </bold> ). Different members of <italic> Medicago </italic> glyoxalase gene families responded differently in response to the duration of stress (salinity and <span class="trait">drought)</span> and types of tissue analyzed <span class="trait">(drought)</span> ( <bold> Figure <xref ref-type="fig" rid="F10"> 10 </xref> </bold> ). <italic> MtGLYI </italic> -8, <italic> MtGLYI </italic> -21, <italic> MtGLYII </italic> -9, and <italic> MtDJ-1 </italic> C found to be the highly up-regulated member in response to salinity; while <italic> MtGLYI </italic> -8, <italic> MtGLYI </italic> -11, <italic> MtGLYI </italic> -21, <italic> MtGLYII </italic> -9, <italic> MtGLYII </italic> -13, and <italic> MtDJ-1 </italic> A are the <span class="trait">drought</span> specific up-regulated members ( <bold> Figure <xref ref-type="fig" rid="F10"> 10 </xref> </bold> ). Thus <italic> MtGLYI </italic> -8, <italic> MtGLYI </italic> -21, and <italic> MtGLYII </italic> -9 found to be the highly up-regulated abiotic stress responsive member that could be characterized further. Overall the present study provides basic information about <italic> Medicago </italic> glyoxalase members that will facilitate the identification of appropriate candidate gene(s) for generating abiotic stress resistant crop plants.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression analysis of a subset of <span class="gene">GmNAC</span> genes in shoots of two contrasting drought-responsive soybean cultivars DT51 and MTD720 under normal and <span class="trait">drought</span> conditions.</h2>
			<h3>PMID:24985975  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Thu NB(1), Hoang XL, Doan H, Nguyen TH, Bui D, Thao NP, Tran LS.
				<br/> 
				年份：2014 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						NAC transcription factors are known to be involved in regulation of plant responses to <span class="trait">drought</span> stress. In this study, the expression of 23 drought-responsive <span class="gene">GmNAC</span> genes was assessed in the shoot tissues of DT51 and MTD720, the two soybean varieties with contrasting drought-responsive phenotypes, by real-time quantitative PCR (RT-qPCR) under normal and <span class="trait">drought</span> conditions. Results indicated that expression profile of <span class="gene">GmNAC</span> genes was genotype-dependent, and six <span class="gene">GmNACs</span> <span class="gene">(GmNAC019,</span> 043, 062, 085, 095 and 101) had higher transcript levels in the shoots of the drought-tolerant DT51 in comparison with the drought-sensitive MTD720 under <span class="trait">drought.</span> Our study suggests a positive correlation between the higher <span class="trait">drought</span> tolerance degree of DT51 versus MTD720 and the up-regulation of at least these six drought-responsive <span class="gene">GmNACs</span> in the shoot tissues. Furthermore, on the basis of our analysis, three genes, <span class="gene">GmNAC043,</span> 085 and 101, were identified as promising candidates for development of drought-tolerant soybean cultivars by genetic engineering.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential Expression Analysis of a Subset of Drought-Responsive <italic> <span class="gene">GmNAC</span> </italic> Genes in Two Soybean Cultivars Differing in <span class="trait">Drought</span> Tolerance</h2>
			<h3>PMID:24322442</h3>
				

				
					<h3>PMCID:3876080</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Thao </surname>  <given-names> Nguyen Phuong </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-23828"> 1 </xref>  <xref rid="c1-ijms-14-23828" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Thu </surname>  <given-names> Nguyen Binh Anh </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-23828"> 1 </xref>  <xref rid="fn1-ijms-14-23828" ref-type="author-notes"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoang </surname>  <given-names> Xuan Lan Thi </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-23828"> 1 </xref>  <xref rid="fn1-ijms-14-23828" ref-type="author-notes"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Van Ha </surname>  <given-names> Chien </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-14-23828"> 2 </xref>  <xref ref-type="aff" rid="af3-ijms-14-23828"> 3 </xref>  <xref ref-type="aff" rid="af4-ijms-14-23828"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-14-23828"> 2 </xref>  <xref rid="c1-ijms-14-23828" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant-specific NAC transcription factors play important roles in plant response to <span class="trait">drought</span> stress. Here, we have compared the expression levels of a subset of <span class="gene">GmNAC</span> genes in drought-tolerant DT51 and drought-sensitive MTD720 under both normal and <span class="trait">drought</span> stress conditions aimed at identifying correlation between <span class="gene">GmNAC</span> expression levels and <span class="trait">drought</span> tolerance degree, as well as potential <span class="gene">GmNAC</span> candidates for genetic engineering. The expression of 23 selected dehydration-responsive <span class="gene">GmNACs</span> was assessed in both stressed and unstressed root tissues of DT51 and MTD720 using real-time quantitative PCR. The results indicated that expression of <span class="gene">GmNACs</span> was genotype-dependent. Seven and 13 of 23 tested <span class="gene">GmNACs</span> showed higher expression levels in roots of DT51 in comparison with MTD720 under normal and <span class="trait">drought</span> stress conditions, respectively, whereas none of them displayed lower transcript levels under any conditions. This finding suggests that the higher <span class="trait">drought</span> tolerance of DT51 might be positively correlated with the higher induction of the <span class="gene">GmNAC</span> genes during water deficit. The drought-inducible <span class="gene">GmNAC011</span> needs to be mentioned as its transcript accumulation was more than 76-fold higher in drought-stressed DT51 roots relative to MTD720 roots. Additionally, among the <span class="gene">GmNAC</span> genes examined, <span class="gene">GmNAC085,</span> 092, 095, 101 and 109 were not only drought-inducible but also more highly up-regulated in DT51 roots than in that of MTD720 under both treatment conditions. These data together suggest that <span class="gene">GmNAC011,</span> 085, 092, 095, 101 and 109 might be promising candidates for improvement of <span class="trait">drought</span> tolerance in soybean by biotechnological approaches.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a nutritionally important crop due to its great supplies of protein- and oil-rich food for human consumption and animal feed [ <xref rid="b31-ijms-14-23828" ref-type="bibr"> 31 </xref> ]. According to the Food and Agricultural Organization of United Nations&#x02019;s statistic (2012), worldwide soybean production is more than 250 million metric tons, mainly from United States, Brazil, Argentina, China and India. Vietnam, with distribution of 175,295 metric tons, also belongs to the top 25 soybean-producing countries [ <xref rid="b32-ijms-14-23828" ref-type="bibr"> 32 </xref> ]. However, <span class="trait">drought</span> stress has led to significant reductions in soybean <span class="trait">yield</span> (24%&#x02013;50%) at various locations over the world [ <xref rid="b33-ijms-14-23828" ref-type="bibr"> 33 </xref> , <xref rid="b34-ijms-14-23828" ref-type="bibr"> 34 </xref> ]. To cope with <span class="trait">drought</span> stress, in recent years intensive research has been conducted to gain a better insight into molecular mechanisms underlying <span class="trait">drought</span> responses in soybean, especially at transcriptional and translational levels to discover and functionally analyze the genes involved [ <xref rid="b1-ijms-14-23828" ref-type="bibr"> 1 </xref> , <xref rid="b35-ijms-14-23828" ref-type="bibr"> 35 </xref> &#x02013; <xref rid="b38-ijms-14-23828" ref-type="bibr"> 38 </xref> ]. Thanks to the completion of the soybean genomic sequence in 2010, at least 61 TF families were identified in soybean by computational prediction, among which the NAC TF family was predicted to consist of more than 180 members by several research groups [ <xref rid="b15-ijms-14-23828" ref-type="bibr"> 15 </xref> , <xref rid="b39-ijms-14-23828" ref-type="bibr"> 39 </xref> , <xref rid="b40-ijms-14-23828" ref-type="bibr"> 40 </xref> ]. The first six <italic> <span class="gene">GmNAC</span> </italic> genes called <italic> <span class="gene">GmNAC1&#x02013;6</span> </italic> were identified by Meng <italic> et al </italic> . in 2007 [ <xref rid="b41-ijms-14-23828" ref-type="bibr"> 41 </xref> ], and the expression of these genes under osmotic stress was thoroughly examined [ <xref rid="b42-ijms-14-23828" ref-type="bibr"> 42 </xref> ]. Later on, in the first large-scale study of the <italic> <span class="gene">GmNAC</span> </italic> family, expression profiling of 31 <italic> <span class="gene">GmNAC</span> </italic> genes in soybean seedlings demonstrated that 9 <italic> <span class="gene">GmNAC</span> </italic> genes were induced by dehydration, high salinity, cold and/or ABA treatments [ <xref rid="b31-ijms-14-23828" ref-type="bibr"> 31 </xref> ]. More recently, a comprehensive analysis of <italic> <span class="gene">GmNAC</span> </italic> family by Le <italic> et al </italic> . (2011) identified 152 full-length <span class="gene">GmNAC</span> TFs, including 11 membrane-bound members within soybean genome [ <xref rid="b43-ijms-14-23828" ref-type="bibr"> 43 </xref> ]. Furthermore, out of 38 <italic> <span class="gene">GmNAC</span> </italic> genes the authors found 25 and 6 <italic> <span class="gene">GmNAC</span> </italic> s induced and repressed 2-fold or more, respectively, in roots and/or shoots of soybean seedlings by dehydration treatment using real-time quantitative PCR (RT-qPCR). In addition, the same group demonstrated the complexity in the dynamics of drought-responsive expression of the <italic> <span class="gene">GmNAC</span> </italic> genes as they reported that several <italic> <span class="gene">GmNAC</span> </italic> genes displayed different drought-responsive expression profiles in different tissues at the same development stage or in the same tissue at different development stages [ <xref rid="b44-ijms-14-23828" ref-type="bibr"> 44 </xref> ]. Strong lines of evidence obtained from various model and crop plants, including soybean, collectively suggest that NAC TFs play an important role in plant adaptation to various stresses, thereby providing novel tools and resources for improvement of stress tolerance in economically important crops.
					</p>
				
					<p>&nbsp;&nbsp;
						In spite of their important roles in plant responses to stresses, the potential of <span class="gene">GmNACs</span> has not been fully explored in soybean. In this study, we have carried out differential expression analysis of 23 selected dehydration-responsive <italic> <span class="gene">GmNAC</span> </italic> genes in DT51 (a drought-tolerant soybean variety) and MTD720 (a drought-sensitive soybean variety) that have contrasting drought-responsive phenotypes. We hypothesized that differential expression of <italic> <span class="gene">GmNAC</span> </italic> genes might contribute to <span class="trait">drought</span> tolerance of DT51 <italic> versus </italic> MTD720. Therefore, we examined the correlation between expression of <italic> <span class="gene">GmNAC</span> </italic> genes and <span class="trait">drought</span> tolerance degrees to identify possible role of <italic> <span class="gene">GmNACs</span> </italic> in DT51 that has enhanced drought-tolerant trait. Additionally, the differentially expressed <italic> <span class="gene">GmNAC</span> </italic> s could serve as potential genetic resources for development of soybean elite cultivars with improved <span class="trait">drought</span> tolerance by genetic engineering.
					</p>
				
					<p>&nbsp;&nbsp;
						Previously, among 152 <italic> <span class="gene">GmNAC</span> </italic> s, which have putative full-length open reading frame, 50 stress-related genes were predicted based on phylogenetic analyses of <italic> <span class="gene">GmNAC</span> </italic> , <italic> ANAC </italic> and <italic> ONAC </italic> families [ <xref rid="b43-ijms-14-23828" ref-type="bibr"> 43 </xref> ]. Out of these 50 genes, 38 genes were checked by RT-qPCR, and 25 up-regulated and 6 down-regulated genes were identified in roots and/or shoots of 12-day-old soybean seedlings in response to dehydration [ <xref rid="b43-ijms-14-23828" ref-type="bibr"> 43 </xref> ]. To determine whether the differential expression of <italic> <span class="gene">GmNAC</span> </italic> genes would contribute to enhanced <span class="trait">drought</span> tolerance we have carried out differential expression analysis of selected genes in a drought-tolerant (DT51) and in a drought-sensitive soybean variety (MTD720). These varieties have contrasting drought-tolerant phenotype under the same experimental conditions by comparing their relative water content (RWC) and drought-tolerant index (DTI) with 11 other soybean cultivars (data submitted for publication to BioMed Research International, Thu <italic> et al. </italic> ). In addition, under non-stressed conditions, DT51 has longer root and shoot lengths than MTD720, especially at V3-stage (not shown). From the identified 31 dehydration-responsive <italic> <span class="gene">GmNAC</span> </italic> genes [ <xref rid="b43-ijms-14-23828" ref-type="bibr"> 43 </xref> ], we selected 23 genes that displayed the highest expression change by dehydration treatment, including 17 up-regulated and all 6 down-regulated genes, grouped to Group A and B in this work, respectively, for comparative expression analyses (Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						Root plasticity has been considered as an important physiological trait in genotypic adaptation to <span class="trait">drought</span> stress. Plants with desirable root traits, such as longer primary root and/or larger lateral root system, can adapt better to <span class="trait">drought</span> stress as they can reach water at lower soil layers and forage subsoil surface moisture [ <xref rid="b45-ijms-14-23828" ref-type="bibr"> 45 </xref> &#x02013; <xref rid="b47-ijms-14-23828" ref-type="bibr"> 47 </xref> ]. Since regulation of root plasticity has been implicated as one of the most important activities reflecting the plant responses against <span class="trait">drought</span> stress, in this study we had the highest interest in analyzing differential expression of the selected <italic> <span class="gene">GmNAC</span> </italic> genes in roots of DT51 and MTD720 to identify the possible correlation between enhanced <span class="trait">drought</span> tolerance of DT51, as well as potential <italic> <span class="gene">GmNAC</span> </italic> genes for genetic engineering.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the criteria of the ratio change &#x02265;2 and <italic> p </italic> -value &#x0003c; 0.05, out of 23 selected dehydration-responsive <italic> <span class="gene">GmNAC</span> </italic> s a total of 19 genes displayed altered expression in roots of DT51 and/or MTD720 after the <span class="trait">drought</span> treatment ( <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1 </xref> , <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ). We found that among 14 genes of Group A, 9 genes ( <italic> <span class="gene">GmNAC011</span> </italic> , <italic> 043 </italic> , <italic> 085 </italic> , <italic> 092 </italic> , <italic> 095 </italic> , <italic> 099 </italic> , <italic> 101 </italic> , <italic> 102 </italic> and <italic> 109 </italic> ) were up-regulated in DT51 roots, whereas <italic> <span class="gene">GmNAC148</span> </italic> was down-regulated as a consequence of the <span class="trait">drought</span> treatment ( <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1A </xref> , <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ). The remaining 4 genes of Group A ( <italic> <span class="gene">GmNAC006</span> </italic> , <italic> 019 </italic> , <italic> 038 </italic> and <italic> 062 </italic> ) did not show significant transcriptional change under our experimental conditions. As for the <italic> <span class="gene">GmNAC</span> </italic> genes in Group B, our data showed that <italic> <span class="gene">GmNAC022</span> </italic> and <italic> 027 </italic> were induced, while the expression of the remaining genes ( <italic> <span class="gene">GmNAC017</span> </italic> , <italic> 071 </italic> , <italic> 083 </italic> and <italic> 113 </italic> ) was considered as unaltered in drought-treated roots of DT51 based on the predefined criteria ( <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1B </xref> , <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In drought-sensitive MTD720, as shown in <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1A </xref> and <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> , 12 genes in Group A displayed significant expression change in the roots under <span class="trait">drought</span> stress. Among these genes, apart from <italic> <span class="gene">GmNAC006</span> </italic> , <italic> 011 </italic> , <italic> 019 </italic> and <italic> 148 </italic> , whose expression was assigned as down-regulated, 8 remaining genes, including <italic> <span class="gene">GmNAC038</span> </italic> , <italic> 043 </italic> , <italic> 062 </italic> , <italic> 085 </italic> , <italic> 095 </italic> , <italic> 099 </italic> , <italic> 101 </italic> and <italic> 109 </italic> , showed up-regulation in response to <span class="trait">drought</span> stress. The remaining two genes of Group A ( <italic> <span class="gene">GmNAC092</span> </italic> and <italic> 102 </italic> ) did not have their gene expression changed during <span class="trait">drought</span> stress. Regarding to the <italic> <span class="gene">GmNAC</span> </italic> s in Group B, 4 genes ( <italic> <span class="gene">GmNAC027</span> </italic> , <italic> 071 </italic> , <italic> 083 </italic> and <italic> 113 </italic> ) were down-regulated, while <italic> <span class="gene">GmNAC022</span> </italic> was induced in drought-treated MTD720 roots ( <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1B </xref> , <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The diverse dehydration/drought-responsive expression of <italic> <span class="gene">GmNAC</span> </italic> genes observed in the roots of DT51 and MTD720 might suggest that the expression of <italic> <span class="gene">GmNAC</span> </italic> genes is genotype-dependent. This diversification would suggest that there might be a positive correlation between drought-responsive expression levels of <italic> <span class="gene">GmNAC</span> </italic> genes and <span class="trait">drought</span> tolerance degrees of DT51 and MTD720, as more up-regulated and less down-regulated genes were identified in drought-treated roots of DT51, whereas the opposite tendency was observed in that of MTD720 ( <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ). Specifically, among 19 <italic> <span class="gene">GmNAC</span> </italic> genes with altered drought-responsive expression in DT51 and/or MTD720, there were 11 up-regulated, 7 unaltered and only 1 down-regulated genes in drought-stressed roots of DT51, whereas 9 up-regulated, 2 unaltered and 8 down-regulated genes were found in the respective tissues of MTD720.
					</p>
				
					<p>&nbsp;&nbsp;
						As a means to find further evidence for the existence of the positive correlation between the expression levels of the examined <italic> <span class="gene">GmNAC</span> </italic> genes and the different tolerance degrees of DT51 and MTD720, we compared the expression levels of the tested <italic> <span class="gene">GmNAC</span> </italic> s in the root tissues of the two cultivars under both normal and <span class="trait">drought</span> conditions. Our data showed that under unstressed conditions, 7 genes, namely <italic> <span class="gene">GmNAC085</span> </italic> , <italic> 092 </italic> , <italic> 095 </italic> , <italic> 101 </italic> , <italic> 109 </italic> and <italic> 148 </italic> of Group A, and <italic> <span class="gene">GmNAC017</span> </italic> of Group B, had significantly higher expression levels in DT51 roots relative to MTD720 roots ( <xref rid="t2-ijms-14-23828" ref-type="table"> Table 2 </xref> ). The expression levels of the remaining genes did not significantly differ in DT51&#x02019;s and MTD720&#x02019;s untreated root tissues. On the other hand, there were more genes, 13 out of 23 tested <italic> <span class="gene">GmNAC</span> </italic> s, showing significantly higher expression levels in roots of DT51 than in that of MTD720 under <span class="trait">drought</span> stress. These included <italic> <span class="gene">GmNAC006</span> </italic> , <italic> 011 </italic> , <italic> 019 </italic> , <italic> 085 </italic> , <italic> 092 </italic> , <italic> 095 </italic> , <italic> 101 </italic> and <italic> 109 </italic> of Group A, and <italic> <span class="gene">GmNAC017</span> </italic> , <italic> 022 </italic> , <italic> 027 </italic> , <italic> 071 </italic> and <italic> 083 </italic> of Group B. Not a single <italic> <span class="gene">GmNAC</span> </italic> gene was observed to have higher expression in MTD720 roots relative to DT51 roots under either condition. In addition, we found that <italic> <span class="gene">GmNAC085</span> </italic> , <italic> 095 </italic> , <italic> 101 </italic> and <italic> 109 </italic> were not only drought-inducible in roots of DT51 and MTD720 but also had higher transcript levels in roots of DT51 than in that of MTD720 under both normal and <span class="trait">drought</span> conditions ( <xref rid="t1-ijms-14-23828" ref-type="table"> Tables 1 </xref> and <xref rid="t2-ijms-14-23828" ref-type="table"> 2 </xref> ). It is also worthy to note that <italic> <span class="gene">GmNAC011</span> </italic> and <italic> 027 </italic> exhibited opposite tendency in regulation between the two varieties. Since these two genes were induced in drought-stressed roots of DT51 but repressed in the respective tissues of MTD720, <italic> <span class="gene">GmNAC011</span> </italic> and <italic> 027 </italic> showed more than 76- and 20-fold higher expression, respectively, in roots of DT51 than in that of MTD720 under <span class="trait">drought</span> stress ( <xref rid="f1-ijms-14-23828" ref-type="fig"> Figure 1 </xref> ; <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ). Taken together, these results suggest that the better <span class="trait">drought</span> tolerance of DT51 is, at least in part, associated with the enhanced expression of a subset of the <italic> <span class="gene">GmNAC</span> </italic> genes under normal and/or <span class="trait">drought</span> stress conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						One of the objectives of this study was to identify drought-responsive genes that could be used for development of soybean cultivars with improved <span class="trait">drought</span> tolerance via genetic engineering technologies. One strategy, which is often used to identify promising candidate genes, is differential expression analysis of cultivars with contrasting drought-tolerant phenotypes [ <xref rid="b35-ijms-14-23828" ref-type="bibr"> 35 </xref> ]. Thus, taking the advantage of this study we searched for <italic> <span class="gene">GmNAC</span> </italic> genes that have more highly expression levels in DT51 than in MTD720, especially under stress conditions [ <xref rid="b35-ijms-14-23828" ref-type="bibr"> 35 </xref> ]. Furthermore, on the basis of the hypothesis that a potential candidate gene should be applicable to any genotype to overcome the <span class="trait">drought</span> stress and the genetic engineering approach could be either over-expression of knockdown, the candidate gene should be in general either induced or repressed in response to <span class="trait">drought</span> in both contrasting cultivars [ <xref rid="b35-ijms-14-23828" ref-type="bibr"> 35 </xref> ]. The candidate genes may also be those that are unaltered in drought-sensitive cultivar but induced in drought-tolerant cultivar, or repressed in drought-sensitive genotype and unaltered or induced in drought-tolerant genotype [ <xref rid="b48-ijms-14-23828" ref-type="bibr"> 48 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						According to these principles, we have identified a number of genes that can be considered for further functional characterizations prior to using them in genetic engineering. When comparing the gene expression levels in the root tissues of the two cultivars exposed to <span class="trait">drought</span> stress, <italic> <span class="gene">GmNAC085</span> </italic> , <italic> 095 </italic> , <italic> 101 </italic> and <italic> 109 </italic> were drought-inducible in root tissues of both cultivars and showed significantly higher expression levels in DT51 not only under <span class="trait">drought</span> but also under normal conditions ( <xref rid="t1-ijms-14-23828" ref-type="table"> Tables 1 </xref> and <xref rid="t2-ijms-14-23828" ref-type="table"> 2 </xref> ). Among these 4 genes, of particular interest is <italic> <span class="gene">GmNAC085</span> </italic> since it shares 39% identity at protein level with <italic> SNAC1 </italic> , a rice <italic> NAC </italic> gene, whose overexpression enhanced <span class="trait">drought</span> tolerance of transgenic rice plants under field conditions [ <xref rid="b26-ijms-14-23828" ref-type="bibr"> 26 </xref> ]. <italic> <span class="gene">GmNAC011</span> </italic> ( <italic> <span class="gene">GmNAC20</span> </italic> in [ <xref rid="b22-ijms-14-23828" ref-type="bibr"> 22 </xref> ]) is also a very attractive candidate gene as its overexpression in <italic> Arabidopsis </italic> was shown to enhance salt tolerance and improved lateral root development [ <xref rid="b22-ijms-14-23828" ref-type="bibr"> 22 </xref> ], that could thus benefit plant adaptation to <span class="trait">drought</span> stress as well [ <xref rid="b45-ijms-14-23828" ref-type="bibr"> 45 </xref> &#x02013; <xref rid="b47-ijms-14-23828" ref-type="bibr"> 47 </xref> ]. The expression level of <italic> <span class="gene">GmNAC011</span> </italic> , which was found to be up- and down-regulated in drought-stressed roots of DT51 and MTD720, respectively, was approximately 76.6-fold higher in DT51 roots than in that of MTD720 under <span class="trait">drought</span> stress. <italic> <span class="gene">GmNAC092</span> </italic> (previous named <italic> <span class="gene">GmNAC4</span> </italic> in [ <xref rid="b31-ijms-14-23828" ref-type="bibr"> 31 </xref> , <xref rid="b41-ijms-14-23828" ref-type="bibr"> 41 </xref> , <xref rid="b42-ijms-14-23828" ref-type="bibr"> 42 </xref> ]), which was up-regulated in drought-stressed roots of DT51 but unaltered in that of MTD720 and expressed at higher level in DT51 roots under both conditions, can be considered a potential candidate gene, too. <italic> <span class="gene">GmNAC092</span> </italic> was grouped together with <italic> ANAC019 </italic> , <italic> ANAC055 </italic> and <italic> ANAC072 </italic> in phylogenetic analyses [ <xref rid="b41-ijms-14-23828" ref-type="bibr"> 41 </xref> , <xref rid="b43-ijms-14-23828" ref-type="bibr"> 43 </xref> ], which were demonstrated to enhance <span class="trait">drought</span> tolerance when overexpressed in transgenic <italic> Arabidopsis </italic> plants [ <xref rid="b10-ijms-14-23828" ref-type="bibr"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, we identified six potential candidate genes including <italic> <span class="gene">GmNAC011</span> </italic> , <italic> 085 </italic> , <italic> 092 </italic> , <italic> 095 </italic> , <italic> 101 </italic> and <italic> 109 </italic> . Not only their coding regions, but also their drought-inducible promoters may be promising for genetic engineering. Increasing evidence has indicated that application of stress-inducible promoters in biotechnology can overcome the negative effect of excessive overproduction of the protein resulted from the usage of constitutive promoters [ <xref rid="b49-ijms-14-23828" ref-type="bibr"> 49 </xref> , <xref rid="b50-ijms-14-23828" ref-type="bibr"> 50 </xref> ]. To gain an insight into the potential applications of the promoters of these candidate <italic> <span class="gene">GmNAC</span> </italic> genes in transgenic technologies under various stress conditions, we have searched for the well-known stress-responsive <italic> cis </italic> -motifs, including ABRE1 (ABA responsive element 1), ABRE2 (ABA responsive element 2), CE1 (coupling element 1), CRT (C-repeat), ICEr1 (induction of <span class="gene">CBF</span> expression region 1), ICEr2 (induction of <span class="gene">CBF</span> expression region 2), LTRE (low temperature-responsive elements), MYBR (MYB recognition site), MYCR (MYC recognition site), NACR (NAC recognition site) [ <xref rid="b8-ijms-14-23828" ref-type="bibr"> 8 </xref> , <xref rid="b51-ijms-14-23828" ref-type="bibr"> 51 </xref> ], G-box [ <xref rid="b52-ijms-14-23828" ref-type="bibr"> 52 </xref> ], CE3 (coupling element 3), DRE (dehydration-responsive element), T/G box, EE (evening element) [ <xref rid="b53-ijms-14-23828" ref-type="bibr"> 53 </xref> , <xref rid="b54-ijms-14-23828" ref-type="bibr"> 54 </xref> ] and ZFHDRS (ZFHD recognition sequence) [ <xref rid="b55-ijms-14-23828" ref-type="bibr"> 55 </xref> ], in their promoter regions (3000-bp upstream sequences from transcription start site [ <xref rid="b15-ijms-14-23828" ref-type="bibr"> 15 </xref> ]). Among these 15 motifs, 9 (ABRE2, CE3, DRE, ICEr2, MYBR, MYCR, G-box, T/G box and EE) were found in the promoter regions of the <italic> <span class="gene">GmNAC</span> </italic> candidate genes (Table S2). ICEr2 has been known as a cold-responsive <italic> cis </italic> -element [ <xref rid="b51-ijms-14-23828" ref-type="bibr"> 51 </xref> ], while ABRE2, CE3, DRE, MYBR, MYCR, G-box, T/G box and EE were identified as dehydration-inducible <italic> cis </italic> -motifs [ <xref rid="b51-ijms-14-23828" ref-type="bibr"> 51 </xref> &#x02013; <xref rid="b54-ijms-14-23828" ref-type="bibr"> 54 </xref> ]. Consistent with their drought-inducible expression profile ( <xref rid="t1-ijms-14-23828" ref-type="table"> Table 1 </xref> ), all six <italic> <span class="gene">GmNAC</span> </italic> s contain one or more dehydration-inducible <italic> cis </italic> -motifs in their promoter regions. Furthermore, our results indicated that in addition to dehydration-inducible <italic> cis </italic> -motifs, ICEr2 was also found in the promoter region of <italic> <span class="gene">GmNAC109</span> </italic> , suggesting that <italic> <span class="gene">GmNAC109</span> </italic> might also be induced by cold stress, and its promoter could be used in genetic engineering of soybean plants against not only <span class="trait">drought</span> but also cold stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Drought-responsive genes were defined if the change in expression was at least 2-fold induction or repression under the water deficit treatment. When comparing expression of <italic> <span class="gene">GmNAC</span> </italic> genes between cultivars, differential expression ratio with at least 2-fold was considered as significant. The mean values of relative expression to <italic> Fbox </italic> were used to plot figures, and error bars on the top of bars represent standard errors of 3 biological replicates. The data were analyzed by Student&#x02019;s <italic> t </italic> -test (one tail, unpaired, equal variance) to identify the statistical significance of differential gene expression within or between two cultivars under either normal or <span class="trait">drought</span> treatment with <italic> p- </italic> value &#x0003c; 0.05.
					</p>
				
					<p>&nbsp;&nbsp;
						The results of this study suggested that in response to <span class="trait">drought</span> stress, transcriptional regulation of <italic> <span class="gene">GmNAC</span> </italic> genes may vary dependently on the genotypes of the cultivars. In addition, our study demonstrated that there is a positive correlation between the root-related expression of a subset <italic> <span class="gene">GmNAC</span> </italic> genes and the improved <span class="trait">drought</span> tolerance of DT51, in comparison with the drought-sensitive MTD720. This study also enabled us to identify a number of <italic> <span class="gene">GmNAC</span> </italic> candidates that may be subjected to in-depth functional characterizations aimed at improving <span class="trait">drought</span> tolerance in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of 17 selected dehydration-inducible ( <bold> A </bold> ) and six selected dehydration-repressible ( <bold> B </bold> ) <italic> <span class="gene">GmNAC</span> </italic> genes in roots of MTD720 and DT51 under normal (white bars) and <span class="trait">drought</span> (black bars) conditions. Asterisks on the top of bars indicate statistically significant differences ( <bold> * </bold> <italic> p </italic> -value &#x0003c; 0.05; <bold> ** </bold> <italic> p </italic> &#x0003c; 0.01; <bold> *** </bold> <italic> p </italic> &#x0003c; 0.001) between treated and untreated root samples within a cultivar.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide expression analysis of soybean NF-Y genes reveals potential function in development and <span class="trait">drought</span> response</h2>
			<h3>PMID:25542200</h3>
				

				
					<h3>PMCID:4435856</h3>
					<h4>期刊:Molecular Genetics and Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Quach </surname>  <given-names> Truyen N. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Hanh T. M. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <phone> 573-882-5494 </phone>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Field Crop Research Institute, Vietnam Academy of Agricultural Sciences, Hanoi, Vietnam  </aff>  <aff id="Aff3">  <label/> The Center for Plant Science Innovation, University of Nebraska, Lincoln, NE USA  </aff>  <aff id="Aff4">  <label/> Department of Computer Science, Christopher S. Bond Life Sciences Center, National Center for Soybean Biotechnology and Informatics Institute, University of Missouri, Columbia, MO USA  </aff>
				<br/> 
				年份：<day> 10 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nuclear factor-Y (NF-Y), a heterotrimeric transcription factor, is composed of NF-YA, NF-YB and NF-YC proteins. In plants, there are usually more than 10 genes for each family and their members have been identified to be key regulators in many developmental and physiological processes controlling gametogenesis, embryogenesis, nodule development, seed development, abscisic acid (ABA) signaling, flowering time, primary root elongation, blue light responses, endoplasmic reticulum (ER) stress response and <span class="trait">drought</span> tolerance. Taking the advantages of the recent soybean genome draft and information on functional characterizations of nuclear factor Y (NF-Y) transcription factor family in plants, we identified 21 <span class="gene">GmNF-YA,</span> 32 <span class="gene">GmNF-YB,</span> and 15 <span class="gene">GmNF-YC</span> genes in the soybean (Glycine max) genome. Phylogenetic analyses show that soybean's proteins share strong homology to Arabidopsis and many of them are closely related to functionally characterized NF-Y in plants. Expression analysis in various tissues of flower, leaf, root, seeds of different developmental stages, root hairs under rhizobium inoculation, and drought-treated roots and leaves revealed that certain groups of soybean NF-Y are likely involved in specific developmental and stress responses. This study provides extensive evaluation of the soybean NF-Y family and is particularly useful for further functional characterization of <span class="gene">GmNF-Y</span> proteins in seed development, nodulation and <span class="trait">drought</span> adaptation of soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In addition to the roles in plant growth and development, NF-Y proteins are important regulators of stress tolerance, especially in response to <span class="trait">drought</span> stress (Petroni et al. <xref ref-type="bibr" rid="CR57"> 2012 </xref> ; Laloum et al. <xref ref-type="bibr" rid="CR28"> 2013 </xref> ). Overexpression of AtNF-YA5, a strongly drought-induced gene with strong expression in the guard cells, improves <span class="trait">drought</span> tolerance and reduces water loss in the leaves (Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ). In contrast to the overexpression of AtNF-YA5, the nfya5 knockout plants and plants overexpressing miR169a showed enhanced leaf water loss and were more sensitive to <span class="trait">drought</span> stress than wild-type plants. In soybean, <span class="gene">GmNF-YA3</span> is induced by various stress treatments. The <italic> Arabidopsis </italic> transgenic plants overexpressing <span class="gene">GmNF-YA3</span> show reduced leaf water loss and enhanced <span class="trait">drought</span> tolerance through regulating the common drought-responsive genes (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ). In rice, the transgenic plants overexpressing HAP2E (NF-YA), a gene responsive to wounding, are tolerant to the pathogen <italic> Xanthomonas oryzae </italic> pv. <italic> oryzae </italic> , <span class="trait">drought</span> and salinity stresses, and improve photosynthesis and tiller numbers (Alam et al. <xref ref-type="bibr" rid="CR1"> 2014 </xref> ). <span class="trait">Drought</span> tolerance regulation is also reported in NF-YB subgroup. Overexpression of the drought-inducible NF-YB results in improved performance under <span class="trait">drought</span> conditions in <italic> Arabidopsis </italic> , maize and poplar (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ; Han et al. <xref ref-type="bibr" rid="CR15"> 2013 </xref> ). The transgenic plants have significant improvement in number of stress-related parameters, including chlorophyll content, stomatal conductance, leaf temperature, reduced wilting, maintenance of photosynthesis and the seed <span class="trait">yield</span> under water-limited conditions (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ). Further characterizations of drought-inducible NF-Y genes will facilitate genetic engineering for improvement of <span class="trait">drought</span> tolerance in crop plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is one of the most important crops for its nutrient sources for human and animal consumption, and potential biofuel and economic values. Extensive research has focused on oil seed development (Severin et al. <xref ref-type="bibr" rid="CR64"> 2010 </xref> ) and plant and symbiotic microbe interaction (Libault et al. <xref ref-type="bibr" rid="CR38"> 2010 </xref> ), making soybean an ideal plant species for both basic and applied research. Taking the advantage of broad understanding of the recent functional genomics of plant NF-Y genes and the current draft of soybean genome (Schmutz et al. <xref ref-type="bibr" rid="CR63"> 2010 </xref> ), we attempted to characterize the soybean NF-Y gene family. Together with analysis of NF-Y family in growth and development, we included an analysis of the family members in response to <span class="trait">drought</span> and <italic> Bradyrhizobium japonicum </italic> treatments which are particularly important to identify candidate genes involved in <span class="trait">drought</span> tolerance and nodulation and nitrogen fixation in soybean. Our study of transcript expression of soybean NF-Y gene families, therefore, would provide an initial step in understanding gene function that may facilitate functional characterization and genetic engineering to increase the efficiency of soybean in response to <span class="trait">drought</span> and <italic> B. japonicum </italic> . In this study, we identified and investigated 68 NF-Y (21 <span class="gene">GmNF-YA,</span> 32 <span class="gene">GmNF-YB,</span> 15 <span class="gene">GmNF-YC)</span> and 13 NC2 genes for their expression patterns in soybean tissues and in response to <span class="trait">drought</span> stress. Expression analysis showed that there are strong associations between gene expression patterns and homology-based predicted functions, allowing us to assign putative functions to the individual gene groups in regulation of plant growth, development and adaptation to <span class="trait">drought</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Each frozen tissue collected from the three-biological repeat <span class="trait">drought</span> experiment was separated into two technical repeats and grinded in liquid nitrogen. RNA was extracted using Trizol (Invitrogen Inc.). RNA integrity was verified by electrophoresis using 1&#x000a0;% agarose in buffer TAE. The RNA was purified from DNA contamination using the Turbo DNA-free DNaseI kit (Ambion, Austin, TX, USA). One microgram of total RNA and the primer mix of oligodT and random hexamer were used to synthesize cDNA using iScript cDNA Synthesis Kit (Bio-Rad, Hercules, CA, USA) in a reaction volume of 20&#x000a0;&#x000b5;L. Quantitative real-time PCR (qRT-PCR) was performed to quantify mRNA expression under <span class="trait">drought</span> conditions. Initially, PRIMEGENS (Srivastava and Xu <xref ref-type="bibr" rid="CR71"> 2007 </xref> ) was used to design primers for soybean NF-Y gene family and all the obtained sequences were manually verified for specific alignment by Blast against the soybean genome database housed in Phytozome to have the final primer list (Supplementary Table S2). Primer pairs with acceptable PCR efficiency and the unique melting disassociation curve were used for qRT-PCR. All reference genes (Supplementary Table S3) were selected based on recommendations from current literatures (Jian et al. <xref ref-type="bibr" rid="CR19"> 2008 </xref> ; Libault et al. <xref ref-type="bibr" rid="CR37"> 2008 </xref> ; Hu et al. <xref ref-type="bibr" rid="CR18"> 2009 </xref> ; Le et al. <xref ref-type="bibr" rid="CR31"> 2012 </xref> ). qRT-PCR results were analyzed using a common PCR efficiency for each primer obtained from linregPCR analysis software (Ramakers et al. <xref ref-type="bibr" rid="CR58"> 2003 </xref> ). The obtained efficiencies were consistently higher than 1.8 with average of about 1.95 for whole primer sets. GeNorm (Vandesompele et al. <xref ref-type="bibr" rid="CR78"> 2002 </xref> ) was used to test the stability (M index) to determine a set of reference genes to be used to normalize individual expression data. In our experiment, we calculated relative expression by normalizing data to the four best reference genes: IDE, UNK1, UNK2 and <span class="gene">ACT</span> (Supplementary Figure S1). Fold change of gene expression was calculated only when there was a significant difference in means of relative gene expression between the <span class="trait">drought</span> and control samples from a Student <italic> T </italic> test.
					</p>
				
					<p>&nbsp;&nbsp;
						Publicly available gene expression in 14 soybean tissue types including leaf, flower, pod, root, nodules and various stages of seed development (Severin et al. <xref ref-type="bibr" rid="CR64"> 2010 </xref> ) for 54 soybean NF-Y genes was retrieved and used to construct a hierarchical clustering heat map (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). The data show that there are some preferences in expression of each gene subfamily in certain plant tissues. A small group of genes highly specific to 1- to 3-week-old seeds (R3 growth-staged seeds) belong strictly to NF-YC <span class="gene">(GmNF-YC05,</span> <span class="gene">GmNF-YC09,</span> <span class="gene">GmNF-YC10</span> and <span class="gene">GmNF-YC11)</span> while majority of seed deferential genes with stronger expression belong to NF-YA and NF-YB genes. A large group of 14 genes strictly to NF-YA and NF-YB are highly specific to root and/or nodules. We also included the data retrieved from a mRNA expression analysis of root hairs in response to treatments of <italic> B. japonicum </italic> (Libault et al. <xref ref-type="bibr" rid="CR38"> 2010 </xref> ) which clearly show that all the genes that are strongly induced (10- to 63-fold change) by <italic> B. japonicum </italic> in root hair are nodule-specific <span class="gene">GmNF-YA</span> members: NF-YA01, NF-YA03, NF-YA10 and NF-YA21 (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ; Supplementary Figure S2). There are several smaller gene groups which are more universal in expression and can be sub-grouped due to slight preference in green organs, flowers and leaves. Members of this group belonging to all families of NF-Y and NC2 including NF-YC14, NF-YB13, NF-YA07 and NF-YA20 are differentially expressed in green organs (leaves and pods) while NF-YB02, NF-Y06, Dpb3-1, Dpb3-2 and NC2&#x003b2;3 differentially expressed in leaves. <fig id="Fig1"> <label> Fig.&#x000a0;1 </label> <caption> <p> Expression of soybean NF-Y in various tissues, <span class="trait">drought</span> stress and rhizobium inoculation. Data of 54 <span class="gene">GmNF-Y</span> genes were retrieved from the normalized RNA sequencing databases (Libault et al. <xref ref-type="bibr" rid="CR38"> 2010 </xref> ; Severin et al. <xref ref-type="bibr" rid="CR64"> 2010 </xref> ) for clustering analysis of tissue-specific and stress-response expression. For <span class="trait">drought</span> and rhizobium treatments (+) and (&#x02013;) indicate up and down gene expression in both root and leaf tissues tested, compared to non-stress condition. The <italic> color bar </italic> represents number of RPKM (color figure online)
					</p>
				
					<p>&nbsp;&nbsp;
						A number of functional groups can be putatively assigned (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). Group A consists of <span class="gene">GmNF-YA01,</span> <span class="gene">GmNF-YA03</span> and <span class="gene">GmNF-YA10</span> which are expressed highly in nodules and have sequences highly similar to the <italic> M. truncatula </italic> MtNF-YA1 (Combier et al. <xref ref-type="bibr" rid="CR10"> 2006 </xref> ), <italic> Lotus japonicus </italic> LjNF-YA1 (Soyano et al. <xref ref-type="bibr" rid="CR70"> 2013 </xref> ) and <italic> Arabidopsis </italic> AtNF-YA2 (Zhao et al. <xref ref-type="bibr" rid="CR90"> 2011 </xref> ), which are important in nodule development and nitrogen nutrition. These three nodule-specific genes are also induced strongly by inoculation with <italic> B. japonicum </italic> (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Group B contains the largest number of soybean <span class="gene">GmNF-YA</span> which have differential expression in flower and seeds and share sequence similarity with at least five functionally characterized proteins relating to ABA/dehydration signaling in seed development. Of these, <span class="gene">GmNF-YA12</span> [known as <span class="gene">GmNF-YA3</span> (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> )] is induced by ABA, NaCl, PEG and cold treatments and controlled by miRNA169 to regulate <span class="trait">drought</span> tolerance in the <italic> Arabidopsis </italic> model (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ). Genes in group B have diverse functions: AtNF-YA3 and AtNF-YA8 which regulate embryogenesis (Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ) and AtNF-YA5 and AtNF-YA6 which regulate ABA and blue light signaling and <span class="trait">drought</span> tolerance (Warpeha et al. <xref ref-type="bibr" rid="CR80"> 2007 </xref> ; Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ), and AtNF-YA4 which controls ER stress tolerance and flowering time (Wenkel et al. <xref ref-type="bibr" rid="CR82"> 2006 </xref> ; Liu and Howell <xref ref-type="bibr" rid="CR39"> 2010 </xref> ). In contrast, genes in group C might be particularly functioning in ABA-related seed development and maturation which are relevant to the functions of <italic> Arabidopsis </italic> AtNF-YA1 and AtNF-YA9 (Levesque-Lemay et al. <xref ref-type="bibr" rid="CR33"> 2003 </xref> ; Wenkel et al. <xref ref-type="bibr" rid="CR82"> 2006 </xref> ; Li et al. <xref ref-type="bibr" rid="CR35"> 2013 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ). Sequence analysis predicted that 12 soybean <span class="gene">GmNF-YA</span> genes are targeted by miR169d and miR169e (TAG_5193213 and TAG_1196776, respectively). Expression miR169d was not associated with any of the NF-YA; however, expression of miRNA169e was negatively associated with <span class="gene">GmNF-YA02,</span> <span class="gene">GmNF-YA12,</span> <span class="gene">GmNF-YA14</span> and <span class="gene">GmNF-YA20</span> (data not shown). <span class="gene">GmNF-YA12</span> has been shown to be cleaved by miR169e in a coexpression study in <italic> Nicotiana benthamiana </italic> and 5&#x02032; RACE assays (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ). <fig id="Fig4"> <label> Fig.&#x000a0;4 </label> <caption> <p> Phylogenetic tree of soybean NF-YA. The tree was constructed using MEGA5 software (see " <xref rid="Sec2" ref-type="sec"> Materials and methods </xref> ") for the NF-YA domain sequences. <italic> Bootstrap values </italic> (from 1,000 replicates) were shown to represent the reliability of tree branches. The referred sequences are from <italic> Arabidopsis </italic> (At), <italic> Medicago </italic> (Mt), <italic> L. japonicus </italic> (Lj) human (Hs), mouse (Rn) and yeast (Sc). Function prediction for soybean genes were based on gene expression and homology to the referred characterized proteins which are written in <italic> bold </italic> for AtNF-YA1 (Wenkel et al. <xref ref-type="bibr" rid="CR82"> 2006 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ), AtNF-YA2 (Zhao et al. <xref ref-type="bibr" rid="CR90"> 2011 </xref> ), AtNF-YA3 (Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ), AtNF-YA4 (Wenkel et al. <xref ref-type="bibr" rid="CR82"> 2006 </xref> ; Liu and Howell <xref ref-type="bibr" rid="CR39"> 2010 </xref> ), AtNF-YA5 (Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ), AtNF-YA6 (Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ), AtNF-YA8 (Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ), AtNF-YA9 (Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ), MtNF-YA1 (Combier et al. <xref ref-type="bibr" rid="CR10"> 2006 </xref> ), LjNF-YA4 (Soyano et al. <xref ref-type="bibr" rid="CR70"> 2013 </xref> ) and <span class="gene">GmNF-YA12</span> (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						NF-YB proteins can be putatively assigned to several functional groups (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). Group D contains proteins that are closely related to the wheat TaNF-YB3 which regulates photosynthesis (Stephenson et al. <xref ref-type="bibr" rid="CR73"> 2011 </xref> ), <italic> Arabidopsis </italic> AtNF-YB2 and AtNF-YB3, and the rice OsHAP3S which controls flowering time control (Kumimoto et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Wei et al. <xref ref-type="bibr" rid="CR81"> 2010 </xref> ). Surprisingly, many of the soybean genes are highly expressed in nodule suggesting that they may function in nodulations and nitrogen nutrition. Members of group E including <span class="gene">GmNF-YB03,</span> <span class="gene">GmNF-YB10,</span> <span class="gene">GmNF-YB26</span> and <span class="gene">GmNF-YB30</span> share homology to LEC1 and LEC1-like (AtNF-YB9 and AtNF-YB6). All these sequence have an aspartate at position 28 (D <sub> 28 </sub> ) which is critical for LEC1 function and is not present in all other members of NF-YB proteins (Lee et al. <xref ref-type="bibr" rid="CR32"> 2003 </xref> ) indicating the unique function of this LEC1 group in embryogenesis and ABA signaling (Lotan et al. <xref ref-type="bibr" rid="CR40"> 1998 </xref> ; Kwong et al. <xref ref-type="bibr" rid="CR27"> 2003 </xref> ; Warpeha et al. <xref ref-type="bibr" rid="CR80"> 2007 </xref> ). LEC1 and LEC1-like orthologs have a conserved function in fatty acid biogenesis and seed oil production in <italic> Brassica napus </italic> (Tan et al. <xref ref-type="bibr" rid="CR75"> 2011 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ) and maize (Shen et al. <xref ref-type="bibr" rid="CR65"> 2010 </xref> ). Group F composed a cluster of root/nodule differentially expressed NF-YB genes homologous to the <italic> L. japonicus </italic> protein LjNF-YB1, which together with LjNF-YA1 regulates division of cortical cells, an initial step in root nodule organogenesis (Soyano et al. <xref ref-type="bibr" rid="CR70"> 2013 </xref> ). Surprisingly, majority of nodule differential and rhizobium inducible genes in this group are <span class="trait">drought</span> repressed. Their close <italic> Arabidopsis </italic> homolog, <italic> AtNF </italic> - <italic> YB1 </italic> , is an important gene regulating <span class="trait">drought</span> tolerance (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ). <fig id="Fig6"> <label> Fig.&#x000a0;6 </label> <caption> <p> Phylogenetic tree of soybean NF-YB. Function prediction for soybean genes were based on gene expression and homology to the referred characterized proteins which are written in <italic> bold </italic> for AtNF-YB2 and AtNF-YB3 (Kumimoto et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ), AtNF-YB6 (Lotan et al. <xref ref-type="bibr" rid="CR40"> 1998 </xref> ; Kwong et al. <xref ref-type="bibr" rid="CR27"> 2003 </xref> ; Lee et al. <xref ref-type="bibr" rid="CR32"> 2003 </xref> ; Tan et al. <xref ref-type="bibr" rid="CR75"> 2011 </xref> ), AtNF-YB9 (Kwong et al. <xref ref-type="bibr" rid="CR27"> 2003 </xref> ; Lee et al. <xref ref-type="bibr" rid="CR32"> 2003 </xref> ; Warpeha et al. <xref ref-type="bibr" rid="CR80"> 2007 </xref> ), OSHAP3 (Miyoshi et al. <xref ref-type="bibr" rid="CR49"> 2003 </xref> ), AtNF-YB1 and ZmNF-YB2 (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ) and LjNF-YB1 (Soyano et al. <xref ref-type="bibr" rid="CR70"> 2013 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean genome has 15 genes encoding NF-YC subunits which have between 112 and 291 amino acids in length. <span class="gene">GmNF-YC</span> possesses of a highly conserved domain for NF-YA and NF-YB interaction (phylogenetic tree bootstrap values are substantially low, Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ) and have structural and amino acid homology to H2A histone (Mantovani <xref ref-type="bibr" rid="CR44"> 1999 </xref> ). The majority of the required amino acids for the subunit interaction and DNA binding of NF-YC (Kim et al. <xref ref-type="bibr" rid="CR23"> 1996 </xref> ; Sinha et al. <xref ref-type="bibr" rid="CR69"> 1996 </xref> ) are highly conserved with minimal replacement of alternative residues of the same functional properties. The two amino acids arginine (R <sub> 52 </sub> ) and aspartate (D <sub> 59 </sub> ) necessary in stabilizing the structural conformations of NF-YB and NF-YC through the HFMs for mutual interaction between the NF-YB and NF-YC subunits (Hackenberg et al. <xref ref-type="bibr" rid="CR14"> 2012 </xref> ) are present in most soybean NF-YC, except for <span class="gene">GmNF-YC11</span> (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ). <span class="gene">GmNF-YC</span> genes are expressed in various tissues except a small group of genes which are expressed specific in early seed development stages. Compared to NF-YA and NF-YB subfamily, fewer NF-YC genes showed response to <span class="trait">drought</span> treatment with only <span class="gene">GmNF-YC15</span> is induced and <span class="gene">GmNF-YC01,</span> <span class="gene">GmNF-YC07</span> and <span class="gene">GmNF-YC12</span> are suppressed (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). <fig id="Fig7"> <label> Fig.&#x000a0;7 </label> <caption> <p> Alignment of soybean NF-YC domains. The referred sequences are from <italic> Arabidopsis </italic> (At), <italic> Medicago </italic> (Mt), human (Hs), mouse (Rn) and yeast (Sc). <italic> Numbers in parentheses </italic> are the actual amino acid numbers of the start and end of domain NF-YC. Required amino acids (Required AA) are important for DNA binding and subunit interaction from yeast (Xing et al. <xref ref-type="bibr" rid="CR84"> 1993 </xref> ) and rat (Maity and de Crombrugghe <xref ref-type="bibr" rid="CR42"> 1992 </xref> ). The putative H-bonds between arginine (R) and aspartate (D) were indicated in the bottom line. NC2&#x003b1; and Dpb (or DRAP1) groups which are homologous to NF-YC are included
					</p>
				
					<p>&nbsp;&nbsp;
						The involvement of NF-Y genes in developmental processes, particularly seed development and embryogenesis, has been identified in all three subfamilies (Petroni et al. <xref ref-type="bibr" rid="CR57"> 2012 </xref> ; Laloum et al. <xref ref-type="bibr" rid="CR28"> 2013 </xref> ). As expected, our transcript analysis identified several groups of soybean NF-YA, NF-YB and NF-YC which have highly specific or differential expression in soybean seeds and close relationship to the known genes. <span class="gene">GmNF-FA</span> subfamily appears to have more genes (8 out of total 16 <span class="gene">GmNF-YA</span> genes with available expression data) highly and differentially expressed in the seeds (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Their <italic> Arabidopsis </italic> orthologs, AtNF-YA1, AtNF-YA3, AtNF-YA5, AtNF-YA6, AtNF-YA8 and AtNF-YA9, have conserved and redundant function in male gametogenesis, embryogenesis and seed development (Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ). Because the majority of the NF-YA genes specifically or differentially expressed in seeds are also induced by <span class="trait">drought</span> treatment (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ), they might be involved in the dehydration-response maturation process during seed development. Similar to the soybean seed development-related <span class="gene">GmNF-YA,</span> the soybean NF-YB <span class="gene">(GmNF-YB03,</span> <span class="gene">GmNF-YB10,</span> <span class="gene">GmNF-YB26</span> and <span class="gene">GmNF-YB30,</span> Figs.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , <xref rid="Fig6" ref-type="fig"> 6 </xref> ) are highly similar to the <italic> Arabidopsis </italic> AtLEC1 and L1L (Meinke <xref ref-type="bibr" rid="CR47"> 1992 </xref> ; West et al. <xref ref-type="bibr" rid="CR83"> 1994 </xref> ; Parcy et al. <xref ref-type="bibr" rid="CR55"> 1997 </xref> ; Lotan et al. <xref ref-type="bibr" rid="CR40"> 1998 </xref> ; Kwong et al. <xref ref-type="bibr" rid="CR27"> 2003 </xref> ; Junker et al. <xref ref-type="bibr" rid="CR20"> 2012 </xref> ). LEC1 and L1L are particularly different from other NF-YB proteins and appear to possess unique residues that are important to regulate hormonal and light signaling during embryogenesis (Lee et al. <xref ref-type="bibr" rid="CR32"> 2003 </xref> ; Junker et al. <xref ref-type="bibr" rid="CR20"> 2012 </xref> ). Indeed, their orthologs in other plant species showed the highly conserved function in regulation of the transition from embryo to adult status (Zhang et al. <xref ref-type="bibr" rid="CR89"> 2002 </xref> ; Yazawa et al. <xref ref-type="bibr" rid="CR85"> 2004 </xref> ; Alemanno et al. <xref ref-type="bibr" rid="CR2"> 2008 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR51"> 2008 </xref> ; Schellenbaum et al. <xref ref-type="bibr" rid="CR62"> 2008 </xref> ; Shen et al. <xref ref-type="bibr" rid="CR65"> 2010 </xref> ; Cao et al. <xref ref-type="bibr" rid="CR8"> 2011 </xref> ; Tan et al. <xref ref-type="bibr" rid="CR75"> 2011 </xref> ). In addition, overexpression of LEC1, L1L and their orthologs resulted in seed oils via activation of various genes regulating fatty acid biosynthesis (Mu et al. <xref ref-type="bibr" rid="CR51"> 2008 </xref> ; Shen et al. <xref ref-type="bibr" rid="CR65"> 2010 </xref> ; Tan et al. <xref ref-type="bibr" rid="CR75"> 2011 </xref> ). Seed-specific expression is also seen in soybean <span class="gene">GmNF-YC</span> group for NF-YC05, NF-YC09, NF-YC10, NF-YC11. These genes share sequence similarity with the <italic> Arabidopsis </italic> AtNF-YC12, a flower differentially expressed gene (Siefers et al. <xref ref-type="bibr" rid="CR66"> 2009 </xref> ). During seed developmental stages, AtNF-YC12 is highly expressed in chalazal endosperm and chalazal seed coat of green mature seed ( <ext-link ext-link-type="uri" xlink:href="http://www.seedgenenetwork.net/"> http://www.seedgenenetwork.net/ </ext-link> ). AtNF-YC12 interacts bilaterally with LEC1 and L1L in yeast&#x02014;two hybrid assays (Hackenberg et al. <xref ref-type="bibr" rid="CR14"> 2012 </xref> ) indicating that they co-regulate seed development and embryogenesis. Therefore, it would be interesting to investigate the functions of soybean gene combinations in the control of embryogenesis and seed oil production in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Increasing evidences have shown that individual NF-Y subunits, particularly NF-YA and NF-YB, are important to regulate <span class="trait">drought</span> and ER stress tolerance in both dicots and monocots with various mechanisms including tolerance to reduced water loss and low tissue water potential (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ; Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ; Liu and Howell <xref ref-type="bibr" rid="CR39"> 2010 </xref> ; Han et al. <xref ref-type="bibr" rid="CR15"> 2013 </xref> ; Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ). The present investigation identified a large number of soybean NF-YA and NF-YB genes responsive to <span class="trait">drought</span> treatments and possibly involved in <span class="trait">drought</span> adaptation (Figs.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , <xref rid="Fig2" ref-type="fig"> 2 </xref> ). Soybean NF-YA family appears to have more genes induced by <span class="trait">drought</span> (7 of 21 genes). Most these genes belong to the groups that have differential expression in the seeds, suggesting that they are involved in maturation and dehydration signaling. Three soybean genes: <span class="gene">GmNF-YA09,</span> <span class="gene">GmNF-YA17</span> and <span class="gene">GmNF-YA19</span> have sequence homology to the <span class="trait">drought</span> tolerance <span class="gene">GmNF-YA12</span> [or <span class="gene">GmNF-YA3</span> (Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> )], AtNF-YA5 and AtNF-YA6 proteins (Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ; Fornari et al. <xref ref-type="bibr" rid="CR11"> 2013 </xref> ; Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ). <italic> <span class="gene">GmNF</span> </italic> - <italic> YA12 </italic> was induced by abscisic acid (ABA) and abiotic stresses, such as NaCl and cold, and has a role in <span class="trait">drought</span> tolerance by reduced leaf water loss via regulation of the ABA signaling pathway. <italic> AtNF </italic> - <italic> YA5 </italic> is induced by ABA, NaCl, cold and PEG treatment, and control water loss in <italic> Arabidopsis </italic> (Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ). AtNF-YA5 also has a redundant role with AtNF-YA1, AtNF-YA6 and AtNF-YA9 which are downstream of LEC1 and regulate of embryogenesis and seed development in <italic> Arabidopsis </italic> (Mu et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ). Most of drought-inducible soybean genes (5/7 genes) in this group are predicted to be targets of microRNA169 (data not shown), which is known to cleave <span class="gene">GmNF-YA12</span> and AtNF-YA5 in response to <span class="trait">drought</span> stress (Li et al. <xref ref-type="bibr" rid="CR34"> 2008 </xref> ; Ni et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ), indicating a conserved function in seed development and drought-response regulation.
					</p>
				
					<p>&nbsp;&nbsp;
						Our phylogenetic analysis shows that two clades of soybean homology to the drought-tolerant <italic> Arabidopsis </italic> AtNF-YB1 and maize ZmNF-YB2 (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ). AtNF-YB1 and ZmNF-YB2 function in <span class="trait">drought</span> tolerance by maintaining leaf water potential, stomatal conductance and photosynthesis (Nelson et al. <xref ref-type="bibr" rid="CR53"> 2007 </xref> ). Surprisingly, most of the genes sharing homology to AtNF-YB1 (group F) are repressed by <span class="trait">drought</span> treatment. The majority of these genes are in fact specific/differential to nodule. Therefore, it is unlikely that they are involved in <span class="trait">drought</span> tolerance in soybean. <span class="gene">GmNF-YB01,</span> the only homolog of drought-tolerant genes AtNF-YB1 and ZmNF-YB2, is expressed ubiquitously in various tissues and induced by <span class="trait">drought</span> (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ) suggesting that it is potentially involved in <span class="trait">drought</span> tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmNFYA3, a target gene of miR169, is a positive regulator of plant tolerance to <span class="trait">drought</span> stress.</h2>
			<h3>PMID:23483290  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni Z(1), Hu Z, Jiang Q, Zhang H.
				<br/> 
				年份：2013 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nuclear factor Y (NF-Y) is a heterotrimeric transcription factor composed of NF-YA, NF-YB and NF-YC proteins. In this study, we identified and characterized a gene, <span class="gene">GmNFYA3,</span> which encodes the NF-YA subunit of the NF-Y complex in soybeans (Glycine max L.). Real time RT-PCR analysis indicated that <span class="gene">GmNFYA3</span> was induced by abscisic acid (ABA) and abiotic stresses, such as polyethylene glycol, NaCl and cold. Subcellular localization analysis suggested that <span class="gene">GmNFYA3</span> may activate its specific targets in the nucleus. Histochemical β-glucuronidase (GUS) staining revealed that the expression of the GUS gene driven by the <span class="gene">GmNFYA3</span> promoter occurred in various transgenic Arabidopsis tissues. Coexpression in Nicotiana benthamiana and 5' RACE assays indicated that miR169 directs <span class="gene">GmNFYA3</span> mRNA cleavage in vivo. Overexpression of <span class="gene">GmNFYA3</span> resulted in Arabidopsis with reduced leaf water loss and enhanced <span class="trait">drought</span> tolerance. In addition, the transgenic Arabidopsis exhibited increased sensitivity to high salinity and exogenous ABA. Moreover, the transcript levels of ABA biosynthesis (ABA1, ABA2), ABA signaling (ABI1, ABI2) and stress-responsive genes, including <span class="gene">RD29A</span> and CBF3, were generally higher in <span class="gene">GmNFYA3</span> plants than in wild-type controls under normal conditions. These results suggest that the <span class="gene">GmNFYA3</span> gene functions in positive modulation of <span class="trait">drought</span> stress tolerance and has potential applications in molecular breeding to enhance <span class="trait">drought</span> tolerance in crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcription factors expressed in soybean roots under <span class="trait">drought</span> stress.</h2>
			<h3>PMID:22033904  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pereira SS(1), Guimarães FC, Carvalho JF, Stolf-Moreira R, Oliveira MC, Rolla AA, Farias JR, Neumaier N, Nepomuceno AL.
				<br/> 
				年份：2011 Oct 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To gain insight into stress-responsive gene regulation in soybean plants, we identified consensus sequences that could categorize the transcription factors MYBJ7, BZIP50, C2H2, and <span class="gene">NAC2</span> as members of the gene families myb, bzip, c2h2, and nac, respectively. We also investigated the evolutionary relationship of these transcription factors and analyzed their expression levels under <span class="trait">drought</span> stress. The NCBI software was used to find the predicted amino acid sequences of the transcription factors, and the Clustal X software was used to align soybean and other plant species sequences. Phylogenetic trees were built using the Mega 4.1 software by neighbor joining and the degree of confidence test by Bootstrap. Expression level studies were carried out using hydroponic culture; the experiments were designed in completely randomized blocks with three repetitions. The blocks consisted of two genotypes, MG/BR46 Conquista (drought-tolerant) and BR16 (drought-sensitive) and the treatments consisted of increasingly long dehydration periods (0, 25, 50, 75, and 100 min). The transcription factors presented domains and/or conserved regions that characterized them as belonging to the bzip, c2h2, myb, and nac families. Based on the phylogenetic trees, it was found that the myb, bzip and nac genes are closely related to myb78, bzip48 and nac2 of soybean and that c2h2 is closely related to c2h2 of Brassica napus. Expression of all genes was in general increased under <span class="trait">drought</span> stress in both genotypes. Major differences between genotypes were due to the lowering of the expression of the mybj7 and c2h2 genes in the drought-tolerant variety at some times. Over-expression or silencing of some of these genes has the potential to increase stress tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of stress-inducible <span class="gene">GmNAC</span> genes in soybean.</h2>
			<h3>PMID:19277718  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tran LS(1), Quach TN, Guttikonda SK, Aldrich DL, Kumar R, Neelakandan A, Valliyodan B, Nguyen HT.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is detrimental to plant growth and development, and often results in significant losses to the yields of economically important crops such as soybeans (Glycine max L.). NAC transcription factors (TFs), which consist of a large family of plant-specific TFs, have been reported to enhance <span class="trait">drought</span> tolerance in a number of plants. In this study, 31 unigenes that contain the complete open reading frames encoding <span class="gene">GmNAC</span> proteins were identified and cloned from soybean. Analysis of C-terminal regulatory domain using yeast one-hybrid system indicated that among 31 <span class="gene">GmNAC</span> proteins, 28 have transcriptional activation activity. Expression analysis of these <span class="gene">GmNAC</span> genes showed that they are differentially expressed in different organs, suggesting that they have diverse functions during plant growth and development. To search for the drought-inducible <span class="gene">GmNAC</span> genes, we prescreened and re-confirmed by quantitative real-time PCR analysis that nine <span class="gene">GmNAC</span> genes are induced by dehydration stress with differential induction levels in both shoot and root. The expression profiles of these nine <span class="gene">GmNAC</span> genes were also examined under other stresses such as high salinity, cold and with abscisic acid hormone treatments. Phylogenetic analysis of the <span class="gene">GmNAC</span> proteins with previously reported drought-inducible NAC proteins of Arabidopsis and rice revealed that the nine drought-inducible <span class="gene">GmNAC</span> proteins belong to the "stress-inducible" NAC group. The results of this systematic analysis of the <span class="gene">GmNAC</span> family will provide novel tools and resources for the development of improved <span class="trait">drought</span> tolerant transgenic soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The essence of <italic> NAC </italic> gene family to the cultivation of drought-resistant soybean <italic> (Glycine max L. Merr.) </italic> cultivars</h2>
			<h3>PMID:28241800</h3>
				

				
					<h3>PMCID:5330122</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Hussain </surname>  <given-names> Reem M. </given-names>  </name>  <address>  <email> rmrommh@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ali </surname>  <given-names> Mohammed </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Feng </surname>  <given-names> Xing </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Xia </given-names>  </name>  <address>  <email> xli@mail.hzau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1790 4137 </institution-id>  <institution-id institution-id-type="GRID"> grid.35155.37 </institution-id>  <institution> State Key Laboratory of Agricultural Microbiology, College of Plant Science and Technology,  </institution>  <institution> Huazhong Agricultural University,  </institution>  </institution-wrap> Wuhan, 430070 People&#x02019;s Republic of China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0696 1046 </institution-id>  <institution-id institution-id-type="GRID"> grid.412741.5 </institution-id>  <institution> Tishreen University, Faculty of Agriculture, Crop Field Department,  </institution>  <institution> Tishreen University,  </institution>  </institution-wrap> Lattakia, Syria  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1790 4137 </institution-id>  <institution-id institution-id-type="GRID"> grid.35155.37 </institution-id>  <institution> National Key Lab of Crop Genetic Improvement, College of Life science and Technology, Bioinformatics Lab,  </institution>  <institution> Huazhong Agriculture University,  </institution>  </institution-wrap> Wuhan, 430070 Hubei People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 13 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The NAC gene family is notable due to its large size, as well as its relevance in crop cultivation - particularly in terms of enhancing stress tolerance of plants. These plant-specific proteins contain NAC domain(s) that are named after Petunia NAM and Arabidopsis ATAF1/2 and CUC2 transcription factors based on the consensus sequence they have. Despite the knowledge available regarding NAC protein function, an extensive study on the possible use of <span class="gene">GmNACs</span> in developing soybean cultivars with superior <span class="trait">drought</span> tolerance is yet to be done. RESULTS: In response to this, our study was carried out, mainly through means of phylogenetic analysis (rice and Arabidopsis NAC genes served as seeding sequences). Through this, 139 <span class="gene">GmNAC</span> genes were identified and later grouped into 17 clusters. Furthermore, real-time quantitative PCR was carried out on drought-stressed and unstressed leaf tissues of both sensitive (B217 and H228) and tolerant (Jindou 74 and 78) cultivars. This was done to analyze the gene expression of 28 dehydration-responsive <span class="gene">GmNAC</span> genes. Upon completing the analysis, it was found that <span class="gene">GmNAC</span> gene expression is actually dependent on genotype. Eight of the 28 selected genes <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> and <span class="gene">GmNAC087)</span> were discovered to have high expression levels in the drought-resistant soybean varieties tested. This holds true for both extreme and standard <span class="trait">drought</span> conditions. Alternatively, the drought-sensitive cultivars exhibited lower <span class="gene">GmNAC</span> expression levels in comparison to their tolerant counterparts. CONCLUSION: The study allowed for the identification of eight <span class="gene">GmNAC</span> genes that could be focused upon in future attempts to develop superior soybean varieties, particularly in terms of <span class="trait">drought</span> <span class="trait">resistance.</span> This study revealed that there were more dehydration-responsive <span class="gene">GmNAC</span> genes as <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC020</span> and <span class="gene">GmNAC021)</span> in addition to what were reported in earlier inquiries. It is important to note though, that discovering such notable genes is not the only goal of the study. It managed to put emphasis on the significance of further understanding the potential of soybean <span class="gene">GmNAC</span> genes, for the purpose of enhancing tolerance towards abiotic stress in general. This scientific inquiry has also revealed that cultivar genotypes tend to differ in their drought-induced gene expression.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> NAC </italic> gene family is notable due to its large size, as well as its relevance in crop cultivation &#x02013; particularly in terms of enhancing stress tolerance of plants. These plant-specific proteins contain NAC domain(s) that are named after Petunia NAM and Arabidopsis ATAF1/2 and CUC2 transcription factors based on the consensus sequence they have. Despite the knowledge available regarding NAC protein function, an extensive study on the possible use of <italic> <span class="gene">GmNACs</span> </italic> in developing soybean cultivars with superior <span class="trait">drought</span> tolerance is yet to be done.
					</p>
				
					<p>&nbsp;&nbsp;
						In response to this, our study was carried out, mainly through means of phylogenetic analysis (rice and <italic> Arabidopsis NAC </italic> genes served as seeding sequences). Through this, 139&#x02009; <italic> <span class="gene">GmNAC</span> </italic> genes were identified and later grouped into 17 clusters. Furthermore, real-time quantitative PCR was carried out on drought-stressed and unstressed leaf tissues of both sensitive (B217 and H228) and tolerant (Jindou 74 and 78) cultivars. This was done to analyze the gene expression of 28 dehydration-responsive <italic> <span class="gene">GmNAC</span> </italic> genes. Upon completing the analysis, it was found that <italic> <span class="gene">GmNAC</span> </italic> gene expression is actually dependent on genotype. Eight of the 28 selected genes ( <italic> <span class="gene">GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> </italic> and <italic> <span class="gene">GmNAC087</span> </italic> ) were discovered to have high expression levels in the drought-resistant soybean varieties tested. This holds true for both extreme and standard <span class="trait">drought</span> conditions. Alternatively, the drought-sensitive cultivars exhibited lower <italic> <span class="gene">GmNAC</span> </italic> expression levels in comparison to their tolerant counterparts.
					</p>
				
					<p>&nbsp;&nbsp;
						The study allowed for the identification of eight <italic> <span class="gene">GmNAC</span> </italic> genes that could be focused upon in future attempts to develop superior soybean varieties, particularly in terms of <span class="trait">drought</span> <span class="trait">resistance.</span> This study revealed that there were more dehydration-responsive <italic> <span class="gene">GmNAC</span> </italic> genes as ( <italic> <span class="gene">GmNAC004,</span> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC020</span> </italic> and <italic> <span class="gene">GmNAC021)</span> </italic> in addition to what were reported in earlier inquiries. It is important to note though, that discovering such notable genes is not the only goal of the study. It managed to put emphasis on the significance of further understanding the potential of soybean <italic> <span class="gene">GmNAC</span> </italic> genes, for the purpose of enhancing tolerance towards abiotic stress in general. This scientific inquiry has also revealed that cultivar genotypes tend to differ in their drought-induced gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						NAC transcription factors are relevant in both abscisic acid (ABA) dependent and independent pathways in <span class="trait">drought</span> stress signaling [ <xref ref-type="bibr" rid="CR28"> 28 </xref> , <xref ref-type="bibr" rid="CR29"> 29 </xref> ]. This was first revealed in the discovery of multiple stress-responsive genes in <italic> Arabidopsis </italic> , namely <italic> ANAC019, <span class="gene">GmNAC055</span> </italic> , and <italic> <span class="gene">GmNAC072</span> </italic> . The overexpression of the aforementioned genes led to enhanced <span class="trait">drought</span> tolerance [ <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. In rice, one notable <italic> NAC </italic> gene is <italic> SNAC1 </italic> , which is vital to <span class="trait">drought</span> stress signaling in guard cells. The overexpression of <italic> SNAC1 </italic> improves the <span class="trait">drought</span> tolerance of transgenic varieties [ <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. <italic> OsNAC10 </italic> , a root-specific transcription factor, is another noteworthy <italic> NAC </italic> . Overexpression of this gene enhances <span class="trait">drought</span> tolerance and provides another valuable benefit &#x02013; increased grain production (under normal circumstances) [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. <italic> SNAC2/OsNAC6, Os045, </italic> and <italic> Os063 </italic> are other genes in rice whose overexpression boosts tolerance to abiotic stress factors (such as salinity, <span class="trait">drought</span> and cold) [ <xref ref-type="bibr" rid="CR34"> 34 </xref> &#x02013; <xref ref-type="bibr" rid="CR37"> 37 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the recognized role of NAC transcription factors in various physiological and biological processes, a more comprehensive study of the <italic> NAC </italic> gene family in soybean was done. This was to establish the possible application of the said genes in creating highly drought-tolerant transgenic cultivars. The research involved a total of 28 <italic> <span class="gene">GmNAC</span> </italic> genes from drought-sensitive (B217 and H228) and drought-tolerant (Jindou74 and 78) soybean varieties. It was hypothesized that the varying expression of <italic> <span class="gene">GmNAC</span> </italic> genes might have an impact on <span class="trait">drought</span> tolerance. Furthermore, the possible correlation between <italic> <span class="gene">GmNACs</span> </italic> and tolerance degrees was looked into.
					</p>
				
					<p>&nbsp;&nbsp;
						Usually similar functions are performed by the genes which have a close evolutionary relationship. Predicting the functions of gene is important in many functional studies and polygenetic analysis can be used for this purpose. Furthermore study of phylogenetic relationships is effective for prediction of stress-related functional genes [ <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. From the phylogenetic tree, it can be established that <italic> <span class="gene">GmNAC100</span> </italic> , <italic> <span class="gene">GmNAC101,</span> <span class="gene">GmNAC102,</span> <span class="gene">GmNAC103</span> <span class="gene">GmNAC118</span> </italic> and <italic> <span class="gene">GmNAC119</span> </italic> might play important part in the formation and development of shoot apical meristem as these are combined together in single with <italic> CUC1 </italic> and <italic> CUC3 </italic> [ <xref ref-type="bibr" rid="CR44"> 44 </xref> , <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. <italic> OsNAC52 </italic> , <italic> ANAC017, ONAC063, <span class="gene">GmNAC004,</span> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC116</span> </italic> and <italic> <span class="gene">GmNAC117</span> </italic> which are combined together in a single group are responsible for <span class="trait">drought</span> tolerance and oxidative stress in transgenic plants [ <xref ref-type="bibr" rid="CR36"> 36 </xref> , <xref ref-type="bibr" rid="CR37"> 37 </xref> , <xref ref-type="bibr" rid="CR46"> 46 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A close evolutionary relationship is shown by <italic> <span class="gene">GmNAC020</span> </italic> and <italic> <span class="gene">GmNAC021</span> </italic> with <italic> ANAC096 </italic> . ABF2 and ABF4 functions synergistically with <italic> ANAC096 </italic> and through their combined function plants can survive under osmotic stress and dehydration [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. <italic> <span class="gene">GmNAC064</span> </italic> to <italic> <span class="gene">GmNAC082</span> </italic> are orthologous with <italic> ORE1/ANAC092/AtNAC2, NAC047, OsNAC1, ANAC072/RD26, ANAC019, OsNAC9, OsNAC6, ANAC032, ANAC055 </italic> and <italic> ANAC102 </italic> and their functions in plants is to respond to stress conditions thereby resulting in improved <span class="trait">yield</span> in different stress conditions such as limited water [ <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> , <xref ref-type="bibr" rid="CR48"> 48 </xref> &#x02013; <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. From <italic> <span class="gene">GmNAC128</span> </italic> to <italic> <span class="gene">GmNAC139</span> </italic> , <italic> VNDs </italic> (including <italic> VND1-VND7 </italic> ) and <italic> NSTs </italic> (including <italic> NST1, NST2, NST3/SND1 </italic> ) combined together in a single group are considered as regulators in vascular vessel development [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. In Group <italic> XVI, OsNAC10 </italic> and <italic> ANAC042 </italic> are present and they are responsible for oxidative stress response functions [ <xref ref-type="bibr" rid="CR33"> 33 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ]. From <italic> <span class="gene">GmNAC008</span> </italic> to <italic> <span class="gene">GmNAC009</span> </italic> were orthologues to <italic> VN12 </italic> which functions in environmental stress responses and senescence and also interacts with gemeniviral replication initiator protein [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ]. Drought-resistance response is inducted by <italic> <span class="gene">GmNAC040,</span> <span class="gene">GmNAC041,</span> <span class="gene">GmNAC042,</span> <span class="gene">GmNAC043,</span> NTL6 </italic> and <italic> NTL9 </italic> which are combined together in a single group and lead senescence is also affected by this drought-resistance response [ <xref ref-type="bibr" rid="CR55"> 55 </xref> , <xref ref-type="bibr" rid="CR56"> 56 </xref> ]. For cold response and co-ordination of flowering time, an important role is played by <italic> <span class="gene">GmNAC046,</span> <span class="gene">GmNAC047,</span> <span class="gene">GmNAC052,</span> <span class="gene">GmNAC053</span> </italic> and <italic> LOV1 </italic> which are clustered in one group [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. These results indicated different possible functions which are performed by <italic> NAC </italic> genes in the soybean plant.
					</p>
				
					<p>&nbsp;&nbsp;
						The productivity of soybean is severely affected by <span class="trait">drought</span> stress and abiotic stress and therefore study of genes of this crop important to increase crop <span class="trait">yield.</span> The research conducted by Dung et al. (2011) [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ] indicated that among 152 <italic> <span class="gene">GmNACs</span> </italic> having putative full-length open reading frame, about fifty stress-related genes have been predicted using phylogenetic analyses of <italic> ANAC, <span class="gene">GmNAC</span> </italic> and <italic> ONAC </italic> families. According to the results of their study seven of our 139&#x02009; <italic> <span class="gene">GmNAC</span> </italic> genes (namely <italic> <span class="gene">GmNAC033,</span> 064, 065, 068, 066, 067and 069 </italic> ) were found ubiquitously in all eight tissues, while 22 genes were found expressed strongly in stressed leaves as shown (Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ). Silico analysis of <italic> <span class="gene">GmNACs</span> </italic> was conducted in the study along with rice and <italic> Arabidopsis </italic> counterparts and similar <italic> NAC </italic> architecture was revealed [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. These results indicate that the stress-related <italic> NAC </italic> genes are differentially expressed spatially and temporally in response to abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						Later on, the study of Dung Tien Le et al. [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ] indicated that our 139&#x02009; <italic> <span class="gene">GmNAC</span> </italic> genes displayed different expression profiles in soybean leaves at V6 and R2 stages under <span class="trait">drought</span> stress as shown in (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). Among these genes, <italic> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC040,</span> <span class="gene">GmNAC041</span> </italic> , <italic> <span class="gene">GmNAC064,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC070,</span> <span class="gene">GmNAC071,</span> <span class="gene">GmNAC072,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC086,</span> <span class="gene">GmNAC087</span> </italic> and <italic> <span class="gene">GmNAC093</span> </italic> genes were upregulated at both V6 and R2 stages with at least two-fold change <span class="trait">(drought</span> vs. well-water). Interestingly, Nguyen Phuong Thao, et al. [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ] suggested that among our 139&#x02009; <italic> <span class="gene">GmNAC</span> </italic> genes, <italic> <span class="gene">GmNAC064</span> </italic> , <italic> <span class="gene">GmNAC065</span> </italic> , <italic> <span class="gene">GmNAC070</span> </italic> , <italic> <span class="gene">GmNAC071</span> </italic> , <italic> <span class="gene">GmNAC072</span> </italic> and <italic> <span class="gene">GmNAC111</span> </italic> genes were not only drought-inducible but also highly upregulated in soybean roots of drought-tolerant cultivar with at least two-fold change (unstressed vs. stressed conditions). <fig id="Fig3"> <label> Fig. 3 </label> <caption> <p> Differential expression data of 139 <italic> <span class="gene">GmNAC</span> </italic> genes in V6 and R2 leaves under <span class="trait">drought</span> stress. Genes shown are either up-regulated or down-regulated at least by two-fold; the colors indicate expression intensity ( <italic> red </italic> , high expression; <italic> green </italic> , low expression; <italic> grey </italic> , no expression). 66&#x000a0;K Affymetrix Soybean Array GeneChip source: Dung Tien Le et al. [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ] see Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						As previously discussed, dehydration stress has altered expression of many <italic> <span class="gene">GmNAC</span> </italic> genes in the soybean leaves [ <xref ref-type="bibr" rid="CR23"> 23 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Pinpointing the specific function of genes is of great importance in agriculture and other related industries. It is, after all, through such means that superior cultivars could be developed with great accuracy. In this study, the goal was to gather information on gene expression patterns in relation to dehydration stress, potentially revealing genes that could allow for new drought-tolerant varieties to be engineered. The study involved the analysis of 28 predicted stress-related <italic> <span class="gene">GmNAC</span> </italic> genes, mainly done through qRT-PCR. As to be expected, gene-specific primers were utilized (Additional file <xref rid="MOESM5" ref-type="media"> 5 </xref> ), and the samples were subjected to simulated <span class="trait">drought</span> conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Despite being subjected to the same conditions, though as to be expected, the drought-tolerant and drought-sensitive cultivars had different phenotypic expressions in response to <span class="trait">drought.</span> Both H288 and B217 (drought-sensitive varieties) exhibited wilting at a much greater extent compared to the Jindou 74 and 78 (drought-tolerant varieties). The impact on growth was also much more prominent on the former pair than on the latter. These effects became more noticeable as the experiment progressed, and the differences were most evident upon reaching the eighth day (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> ). As for gene expression, the <italic> <span class="gene">GmNAC</span> </italic> genes analyzed were expressed differently among drought-tolerant and drought-sensitive cultivars (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). The drought-tolerant varieties (which include Jindou 74 and 78) exhibited high expression levels for <italic> <span class="gene">GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> </italic> and <italic> <span class="gene">GmNAC087</span> </italic> . The drought-sensitive cultivars (B217, H228), on the other hand, were detected to have lower expressions for the aforesaid genes. Do note that this trend was observed in both extreme <span class="trait">drought</span> and standard stress setups. Interestingly, the Jindou varieties still maintained higher expressions for all genes mentioned so far, even under non-stress conditions. Although associated with higher expression in more genes while under <span class="trait">drought</span> stress (24 out of 28 <italic> <span class="gene">GmNAC</span> </italic> genes to be exact), the Jindou varieties failed to match H228 and B217&#x02019;s expression for <italic> <span class="gene">GmNAC067,</span> <span class="gene">GmNAC072</span> </italic> , and <italic> <span class="gene">GmNAC080</span> </italic> under normal conditions (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> Expression of 28 selected dehydration-inducible <italic> <span class="gene">GmNAC</span> </italic> genes in leaf tissues of soybean plants. qPCR analysis of the selected genes of <span class="trait">drought</span> sensitive (B217, H228) and <span class="trait">drought</span> tolerant (Jindou74 and 78) cultivars under normal (ck.) and <span class="trait">drought</span> (dr.) conditions. Error bars above means denote LSD value of three replicates. Different alphabetical letters (a, b, c&#x02026;) above means show the significant differences (LSD&#x02009;&#x0003c;&#x02009;0.05) among treatments of a cultivar at control and 8&#x000a0;days after <span class="trait">drought.</span> Error bars above means denote LSD value of three biological replicates
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, the drought-linked <italic> <span class="gene">GmNAC</span> </italic> genes ( <italic> <span class="gene">GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> </italic> and <italic> <span class="gene">GmNAC087</span> </italic> ) were found to be inducible regardless of <span class="trait">drought</span> sensitivity of cultivars, although in both <span class="trait">drought</span> and non-drought conditions, such genes showed greater transcript levels in the Jindou or drought-tolerant soybean varieties than that in drought-sensitive varieties. The same set of genes were also upregulated throughout different stages of leaf growth. The results suggest that these genes may play conserved roles in leaf growth and in plant responses to <span class="trait">drought</span> stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the data gathered throughout the course of the study, it may be said that <span class="trait">drought</span> tolerance is influenced by the expression of certain <italic> <span class="gene">GmNAC</span> </italic> genes, particularly <italic> <span class="gene">GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> </italic> and <italic> <span class="gene">GmNAC087</span> </italic> . It is crucial to mention, however, that only three among the six have been previously known to be connected to <span class="trait">drought.</span> It may also be concluded that there is a correlation between gene expression, transcriptional regulation, and superior tolerance to <span class="trait">drought,</span> as seen from the different stages of leaf development across tolerant and sensitive cultivars. All in all, the study has revealed a number of potential gene candidates that could be given focus by those attempting to develop soybean cultivars with even greater <span class="trait">drought</span> tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Investigating the Control of Chlorophyll Degradation by Genomic Correlation Mining</h2>
			<h3>PMID:27618630</h3>
				
					<h3>PMCID:5019398</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ghandchi </surname>  <given-names> Frederick P. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Caetano-Anolles </surname>  <given-names> Gustavo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Clough </surname>  <given-names> Steven J. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0002-5435-4387 </contrib-id>  <name>  <surname> Ort </surname>  <given-names> Donald R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Three experiments on the analysis of global gene expression under conditions that should influence PAO gene expression in <italic> Arabidopsis thaliana </italic> were chosen as the expression datasets to mine for genes that co-expressed with PAO. These transcriptomic experiments were conducted using the Affymetrix ATH1 microarray chip technology containing gene expression data for 22,810 predicted genes and downloaded from the NIH GEO database [ <xref rid="pone.0162327.ref018" ref-type="bibr"> 18 </xref> ]. The GSE55907 dataset originated from freezing tolerance induction experiments (4&#x000b0;C for 24 hrs) or warm (22&#x000b0;C) conditions on wild-type Columbia (Col-0) and several transgenics that were over-expressing genes associated with cold responses: a C-repeat binding factor (CBF), heat shock transcription factor <span class="gene">C1</span> (HSFC), and a DREB and EAR Motif protein (DEAR) [ <xref rid="pone.0162327.ref019" ref-type="bibr"> 19 </xref> ]. The GSE5727 dataset originated from leaves sampled during and prior to senescence, and were performed on wild-type Col-0, a jasmonic acid (JA) coi1 mutant, an ethylene signaling pathway ein2 mutant, and a NahG transgenic that would dampen salicylic acid (SA) signaling pathways [ <xref rid="pone.0162327.ref020" ref-type="bibr"> 20 </xref> ]. The GSE72050 dataset originated from a study on the effect of <span class="trait">drought</span> on global gene expression in leaf tissue from wild-type Col-0 and NAC26-transcription factor overexpressing lines [ <xref rid="pone.0162327.ref021" ref-type="bibr"> 21 </xref> ]. Microarray data were formatted for statistical analysis using the Python scripting language such that all data were normalized and log2 transformed in a similar manner.
					</p>
				
					<p>&nbsp;&nbsp;
						Further correlation analysis and regression modeling pointed to the same conclusion that ATAF1 significantly predicts PAO and that PAO is positively correlated with ATAF1 ( <xref ref-type="table" rid="pone.0162327.t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone.0162327.g002"> Fig 2 </xref> ). Frequency analysis showed that PAO and ATAF1 expression levels were either both high or both low in the various plant lines, again reinforcing the positive correlation between ATAF1 and PAO ( <xref ref-type="table" rid="pone.0162327.t003"> Table 3 </xref> ). In particular, PAO and ATAF1 were both overexpressed under warmer conditions, during senescence, and under <span class="trait">drought</span> conditions as well as in certain plant lines like HSFC overexpressing lines. PAO and ATAF1 were also overexpressed in jasmonic acid/ethylene pathway mutant lines and salicylic acid pathway transgenic line despite pathway disruption in these lines, suggesting that jasmonic acid/ethylene pathway and salicylic acid pathway, which are involved in senescence-induction, work independently of ATAF1 to induce chlorophyll degradation. PAO levels were also higher despite slightly lower ATAF1 levels in the NAC26-overexpressing line, suggesting that <span class="gene">NAC26</span> up-regulates PAO in a manner similar to ATAF1. Meanwhile both PAO and ATAF1 were underexpressed in the DEAR overexpressing line. These findings are compiled into a working model showing hypothesized relationships among the different elements that interact to regulate PAO expression ( <xref ref-type="fig" rid="pone.0162327.g003"> Fig 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						PAO is represented by probe name _246335_at, while Plant class variable represents the 4 plant lines of the study representing wild-type or <span class="gene">NAC26</span> line in well-water or <span class="trait">drought</span> conditions. Note significant factor of Plant (p = 0.0092). Box plots displayed lower to upper quartiles with central horizontal line representing median, and diamond representing mean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dehydrin expression in soybean.</h2>
			<h3>PMID:23792826  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yamasaki Y(1), Koehler G, Blacklock BJ, Randall SK.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is a relatively cold intolerant plant. In most stress tolerant plants the responsive expression of dehydrin proteins in vegetative tissues can be a significant contributor to protection against environmental stresses. The purpose of this study was to examine the expression of dehydrins in various organs and the cold-responses of dehydrin genes in vegetative tissues of soybean. Examination of the soybean genome indicated the presence of genes encoding ten distinct dehydrins. Levels of dehydrin proteins were probed with several antibodies specific to dehydrins or to the signature K-sequence. A single vegetatively expressed dehydrin protein was detected and the levels were insignificantly altered in response to cold, <span class="trait">drought,</span> or salt stress, nor was the transcript responsive to ABA. This SK2-type, acidic dehydrin family member <span class="gene">(GmERD14)</span> was purified, identified by mass spectroscopy, and shown to be in vivo phosphorylated; indicating characteristics similar to other known acidic dehydrins. The lack of cold stress-regulated acidic dehydrin expression may contribute to the inability of soybean to cold acclimate. While transcripts for all ten dehydrins could be detected in various tissues, only three accumulated to significant levels in vegetative tissues (two of the KS type and one of SK2 type). One of these transcripts, a KS dehydrin, was accumulated following cold treatments. The accumulation of the KS dehydrin was also responsive to exogenous ABA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification of Soybean U-Box E3 Ubiquitin Ligases and Roles of <span class="gene">GmPUB8</span> in Negative Regulation of <span class="trait">Drought</span> Stress Response in Arabidopsis.</h2>
			<h3>PMID:27057003  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang N(1), Liu Y(1), Cong Y(1), Wang T(1), Zhong X(1), Yang S(1), Li Y(2), Gai J(2).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant U-box (PUB) E3 ubiquitin ligases play important roles in hormone signaling pathways and response to abiotic stresses, but little is known about them in soybean, Glycine max. Here, we identified and characterized 125 PUB genes from the soybean genome, which were classified into eight groups according to their protein domains. Soybean PUB genes <span class="gene">(GmPUB</span> genes) are broadly expressed in many tissues and are a little more abundant in the roots than in the other tissues. Nine <span class="gene">GmPUB</span> genes, <span class="gene">GmPUB1-GmPUB9,</span> showed induced expression patterns by <span class="trait">drought,</span> and the expression of <span class="gene">GmPUB8</span> was also induced by exogenous ABA and NaCl. <span class="gene">GmPUB8</span> was localized to post-Golgi compartments, interacting with <span class="gene">GmE2</span> protein as demonstrated by yeast two-hybrid (Y2H) and bimolecular fluorescence complementation (BiFC) experiments, and showed E3 ubiquitin ligase activity by in vitro ubiquitination assay. Heterogeneous overexpression of <span class="gene">GmPUB8</span> in Arabidopsis showed decreased <span class="trait">drought</span> tolerance, enhanced sensitivity with respect to osmotic and salt stress inhibition of seed germination and seedling growth, and inhibited ABA- and mannitol-mediated stomatal closure. Eight <span class="trait">drought</span> stress-related genes were less induced in <span class="gene">GmPUB8-overexpressing</span> Arabidopsis after <span class="trait">drought</span> treatment compared with the wild type and the pub23 mutant. Taken together, our results suggested that <span class="gene">GmPUB8</span> might negatively regulate plant response to <span class="trait">drought</span> stress. In addition, Y2H and BiFC showed that <span class="gene">GmPUB8</span> interacted with soybean COL (CONSTANS LIKE) protein. <span class="gene">GmPUB8-overexpressing</span> Arabidopsis flowered earlier under middle- and short-day conditions but later under long-day conditions, indicating that <span class="gene">GmPUB8</span> might regulate flowering time in the photoperiod pathway. This study helps us to understand the functions of PUB E3 ubiquitin ligases in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and molecular characterization of <span class="gene">GmERF7,</span> a soybean ethylene-response factor that increases salt stress tolerance in tobacco.</h2>
			<h3>PMID:23111158  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhai Y(1), Wang Y, Li Y, Lei T, Yan F, Su L, Li X, Zhao Y, Sun X, Li J, Wang Q.
				<br/> 
				年份：2013 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ethylene-response factors (ERFs) play an important role in regulating gene expression in plant responses to biotic and abiotic stresses. In this study, a new ERF transcription factor, <span class="gene">GmERF7,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF7</span> contained an AP2/ERF domain with 58 amino acids, two putative nuclear localization signal (NLS) domains, an acidic amino acid-rich transcriptional activation domain and a conserved N-terminal motif [MCGGAI(I/L)]. The expression of <span class="gene">GmERF7</span> was induced by <span class="trait">drought,</span> salt, methyl jasmonate (MeJA), ethylene (ETH) and abscisic acid (ABA) treatments. However, the expression of <span class="gene">GmERF7</span> decreased under cold treatment. <span class="gene">GmERF7</span> localized to the nucleus when transiently expressed in onion epidermal cells. Furthermore, <span class="gene">GmERF7</span> protein bound to the GCC-box element in vitro and activated the expression of the β-glucuronidase (GUS) reporter gene in tobacco leaves. Activities of <span class="gene">GmERF7</span> promoter <span class="gene">(GmERF7P)</span> upregulated in tobacco leaves with 10h <span class="trait">drought,</span> salt and ETH treatments. However, activities of <span class="gene">GmERF7P</span> decreased with 10h cold and ABA treatments. Overexpression of <span class="gene">GmERF7</span> in tobacco plants led to higher levels of chlorophyll and soluble carbohydrates and a lower level of malondialdehyde compared with wild-type tobacco plants under salt stress conditions, which indicated that <span class="gene">GmERF7</span> enhanced salt tolerance in transgenic plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the soybean <span class="gene">GmERF3</span> gene, an AP2/ERF type transcription factor for increased tolerances to salt, <span class="trait">drought,</span> and diseases in transgenic tobacco.</h2>
			<h3>PMID:19602544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Li L, Xu Z, Chen X, Guo J, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new member of the AP2/ERF transcription factor family, <span class="gene">GmERF3,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF3</span> contained an AP2/ERF domain of 58 amino acids and two putative nuclear localization signal (NLS) domains. It belonged to a group IV protein in the ERF (ethylene response factor) subfamily as typified by a conserved N-terminal motif [MCGGAI(I/L)]. Expression of <span class="gene">GmERF3</span> was induced by treatments with high salinity, <span class="trait">drought,</span> abscisic acid (ABA), salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and soybean mosaic virus (SMV), whereas there was no significant <span class="gene">GmERF3</span> mRNA accumulation under cold stress treatment. <span class="gene">GmERF3</span> could bind to the GCC box and DRE/CRT element, and was targeted to the nucleus when transiently expressed in onion epidermal cells. The <span class="gene">GmERF3</span> protein fused to the GAL4 DNA-binding domain to activate transcription of reporter genes in yeast. Ectopic expression of the <span class="gene">GmERF3</span> gene in transgenic tobacco plants induced the expression of some PR genes and enhanced <span class="trait">resistance</span> against infection by Ralstonia solanacearum, Alternaria alternata, and tobacco mosaic virus (TMV), and gave tolerance to high salinity and dehydration stresses. Furthermore, overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions. The overall results suggested that <span class="gene">GmERF3</span> as an AP2/ERF transcription factor may play dual roles in response to biotic and abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel EAR-motif-containing gene <span class="gene">GmERF4</span> from soybean (Glycine max L.).</h2>
			<h3>PMID:19597961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Chen X, Xu Z, Li L, Guo J, Ma Y.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptional repressors are emerging as central regulators of development and stress responses in different organisms. The ERF-associated amphiphilic repression (EAR) motif was identified as essential for transcriptional repression. To gain a better understanding of this type of protein, we reported here a novel <span class="gene">GmERF4</span> protein from soybean. Sequence alignment showed that <span class="gene">GmERF4</span> contains one AP2/ERF domain, two putative nuclear localization signal regions and one EAR motif. The <span class="gene">GmERF4</span> protein was preferentially localized to the nucleus of onion epidermis cells and bound specifically to the GCC box and DRE/CRT element in vitro. Furthermore, the expression of <span class="gene">GmERF4</span> was induced by ethylene, JA, SA, cold, salt, <span class="trait">drought,</span> and soybean mosaic virus, and repressed by ABA. Constitutive expression of <span class="gene">GmERF4</span> in transgenic tobacco plants increased tolerance to salt and <span class="trait">drought</span> stresses compared with wild-type plants, but did not exhibit detectable <span class="trait">resistance</span> against bacterial infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">GmbZIP1</span> transcription factor enhances multiple abiotic stress tolerances in transgenic plants.</h2>
			<h3>PMID:21331631  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao SQ(1), Chen M, Xu ZS, Zhao CP, Li L, Xu HJ, Tang YM, Zhao X, Ma YZ.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Abscisic acid (ABA)-responsive element binding proteins (AREBs) are basic domain/leucine zipper transcription factors that bind to the ABA-responsive element (ABRE) in the promoter regions of ABA-inducible genes in plants. A novel bZIP transcription factor gene, <span class="gene">GmbZIP1,</span> encoding 438 amino acids with a conserved bZIP domain composed of 60 amino acids was isolated from salt-tolerant soybean cv. Tiefeng 8. Southern blotting showed that only one copy was present in the soybean genome. Phylogenetic analyses showed that <span class="gene">GmbZIP1</span> belonged to the AREB subfamily of the bZIP family and was most closely related to AtABF2 and OsTRAB1. The expression of <span class="gene">GmbZIP1</span> was highly induced by ABA, <span class="trait">drought,</span> high salt and low temperature; and <span class="gene">GmbZIP1</span> was expressed in soybean roots, stems and leaves under different stress conditions. <span class="gene">GmbZIP1</span> was localized inside the nuclei of transformed onion epidermal cells. Overexpression of <span class="gene">GmbZIP1</span> enhanced the responses of transgenic plants to ABA and triggered stomatal closure under stresses, potentially leading to improved tolerances to several abiotic stresses such as high salt, low temperature and <span class="trait">drought</span> in transgenic plants. Furthermore, overexpression of <span class="gene">GmbZIP1</span> affected the expression of some ABA or stress-related genes involved in regulating stomatal closure in Arabidopsis under ABA, <span class="trait">drought</span> and high salt stress conditions. A few AREB elements were detected in the promoter region of those ABA or stress-related genes, suggesting that <span class="gene">GmbZIP1</span> regulates the ABA response or stomatal closure mediated by those downstream genes in transgenic Arabidopsis. Moreover, <span class="gene">GmbZIP1</span> was used to improve the <span class="trait">drought</span> tolerance trait of Chinese wheat varieties BS93. Functional analysis showed that overexpression of <span class="gene">GmbZIP1</span> enhanced the <span class="trait">drought</span> tolerance of transgenic wheat, and transcripts of <span class="gene">GmbZIP1</span> were detected in transgenic wheat using RT-PCR. In addition, <span class="gene">GmbZIP1</span> overexpression did not result in growth retardation in all transgenic plants, suggesting that <span class="gene">GmbZIP1</span> may be a valuable genetic resource for engineering stress tolerance of crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterologous expression of <span class="gene">GmSIP1;3</span> from soybean in tobacco showed and growth retardation and tolerance to hydrogen peroxide.</h2>
			<h3>PMID:28818377  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang D(1), Huang Y(2), Kumar M(3), Wan Q(2), Xu Z(2), Shao H(4), Pandey GK(5).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aquaporins (AQPs) are transmembrane protein channels that are members of Major Intrinsic Proteins (MIP) superfamily. AQPs play important roles in plant reproduction, cell elongation, osmoregulation, influence leaf physiology and are responsive to <span class="trait">drought</span> and salt tolerance. Small intrinsic proteins (SIPs)belongs to one of the groups of AQPs, which are mainly localized to endoplasmic reticulum(ER). While this group of aquaporin is being well studied in Arabidopsis, grape and other plant species, not much is known about the molecular regulatory mechanisms driven by ER-type AQPs in Glycine Max. In this study, the function of <span class="gene">GmSIP1;3</span> is studied in detail by using both yeast and plant systems. <span class="gene">GmSIP1;3</span> showed a ubiquitous expression pattern in all different tissues in Glycine Max. Heterologous expression of <span class="gene">GmSIP1;3</span> in Nicotiana tabacum conferred a short root phenotype,growth retardation at seedling stage and significant tolerance to oxidative stress (H2O2) both in yeast and plant systems. Auxin (IAA) content significantly increased in transgenic plants compared with that of wild type, however, the abscisic acid (ABA) content was significantly reduced. Subcellular localization and colocalization analyses showed <span class="gene">GmSIP1;3</span> localized to ER plasma membrane. On the basis of these observations, we postulate that <span class="gene">GmSIP1;3</span> is involved in oxidative stress pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of soybean <span class="gene">GmbZIP132</span> under abscisic acid and salt stresses.</h2>
			<h3>PMID:18713445  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liao Y(1), Zhang JS, Chen SY, Zhang WK.
				<br/> 
				年份：2008 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant basic-leucine zipper (bZIP) transcription factors play important roles in many biological processes. In the present study, a bZIP gene, <span class="gene">GmbZIP132,</span> was cloned from soybean and its biological function under abiotic stresses was studied. The transcription of <span class="gene">GmbZIP132</span> was induced by <span class="trait">drought</span> and high salt treatments. Among all of the organs analyzed, its expression was the highest in cotytledon and stems. <span class="gene">GmbZIP132</span> could weakly bind to the GCN4-like motif (GLM) (5'-GTGAGTCAT-3') in yeast one-hybrid assay. Compared with wild-type (WT) Arabidopsis plants, transgenic plants overexpressing <span class="gene">GmbZIP132</span> showed reduced abscisic acid sensitivity and increased water loss rate. At the stage of germination, transgenic plants were more tolerant to salt treatment than wild-type plants. The expression of some abiotic stress-related genes, such as rd29B, DREB2A, and P5CS, were upregulated in the transgenic plants. These results indicated that <span class="gene">GmbZIP132</span> was an abiotic stress-related gene, and its overexpression could increase the salt tolerance of transgenic Arabidopsis plants during germination, yet no significant difference of tolerance to abiotic stresses was found between transgenic and wild type plants at the seedling stage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Expression Analysis of Cytokinin Metabolic Genes in Soybean under Normal and <span class="trait">Drought</span> Conditions in Relation to Cytokinin Levels</h2>
			<h3>PMID:22900018</h3>
				

				
					<h3>PMCID:3416864</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Le </surname>  <given-names> Dung Tien </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishiyama </surname>  <given-names> Rie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Yasuko </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vankova </surname>  <given-names> Radomira </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Maho </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seki </surname>  <given-names> Motoaki </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ham </surname>  <given-names> Le Huy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi-Shinozaki </surname>  <given-names> Kazuko </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shinozaki </surname>  <given-names> Kazuo </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cytokinins (CKs) mediate cellular responses to <span class="trait">drought</span> stress and targeted control of CK metabolism can be used to develop drought-tolerant plants. Aiming to manipulate CK levels to improve <span class="trait">drought</span> tolerance of soybean cultivars through genetic engineering of CK metabolic genes, we surveyed the soybean genome and identified 14 CK biosynthetic (isopentenyltransferase, <span class="gene">GmIPT)</span> and 17 CK degradative (CK dehydrogenase, <span class="gene">GmCKX)</span> genes. Comparative analyses of <span class="gene">GmIPTs</span> and <span class="gene">GmCKXs</span> with Arabidopsis counterparts revealed their similar architecture. The average numbers of abiotic stress-inducible cis-elements per promoter were 0.4 and 1.2 for <span class="gene">GmIPT</span> and <span class="gene">GmCKX</span> genes, respectively, suggesting that upregulation of <span class="gene">GmCKXs,</span> thereby reduction of CK levels, maybe the major events under abiotic stresses. Indeed, the expression of 12 <span class="gene">GmCKX</span> genes was upregulated by dehydration in R2 roots. Overall, the expressions of soybean CK metabolic genes in various tissues at various stages were highly responsive to <span class="trait">drought.</span> CK contents in various organs at the reproductive (R2) stage were also determined under well-watered and <span class="trait">drought</span> stress conditions. Although tRNA-type <span class="gene">GmIPT</span> genes were highly expressed in soybean, cis-zeatin and its derivatives were found at low concentrations. Moreover, reduction of total CK content in R2 leaves under <span class="trait">drought</span> was attributable to the decrease in dihydrozeatin levels, suggesting a role of this molecule in regulating soybean's responses to <span class="trait">drought</span> stress. Our systematic analysis of the <span class="gene">GmIPT</span> and <span class="gene">GmCKX</span> families has provided an insight into CK metabolism in soybean under <span class="trait">drought</span> stress and a solid foundation for in-depth characterization and future development of improved drought-tolerant soybean cultivars by manipulation of CK levels via biotechnological approach.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Taking into account the importance of CKs and CK signaling in the regulation of stress tolerance, which provides multiple biotechnological strategies for agronomy, we have previously identified and characterized expression profiles of each TCS member in soybean seedlings under dehydration stress <xref ref-type="bibr" rid="pone.0042411-Le1"> [36] </xref> , <xref ref-type="bibr" rid="pone.0042411-Mochida1"> [37] </xref> . In this report, we have identified and systematically characterized all of the <span class="gene">IPT</span> and CKX encoding genes in soybean. We have found that a large number of putative <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes may be the result of genome duplication. Since there is a wealth of structural and functional information for <italic> Arabidopsis </italic> IPTs and CKXs, we performed sequence analyses and phylogenetic relationship studies of IPTs and CKXs of soybean and <italic> Arabidopsis </italic> to classify the functions of <span class="gene">GmIPT</span> and <span class="gene">GmCKX</span> proteins in CK metabolism based on their sequence architecture. To clarify the regulation of CK metabolism in soybean during normal growth and <span class="trait">drought</span> stress, we have analyzed the expression patterns of <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes under normal and <span class="trait">drought</span> stress conditions in a tissue-specific fashion. Expression profiles of <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes were examined in various tissues of both dehydrated young seedlings and soil-dried plants at vegetative and flowering stages. Additionally, we have investigated the correlation between the <span class="trait">drought</span> stress-dependent alterations of CK metabolic gene expression and CK biosynthesis by determining the endogenous CK levels in soybean drought-treated leaf tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Tissue-specific and development stage-related expression data are useful in the identification of genes that are involved in defining the precise nature of individual tissues in a given developmental stage. Moreover, the mechanisms controlling the response to <span class="trait">drought</span> stress may be associated with root- and/or shoot-related traits. For instance, suppression of shoot growth and/or promotion of primary root growth are considered morphological adjustments enabling plants to adapt better to <span class="trait">drought</span> stress <xref ref-type="bibr" rid="pone.0042411-Manavalan1"> [1] </xref> , <xref ref-type="bibr" rid="pone.0042411-Thao1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0042411-Werner1"> [12] </xref> . CKs are well-known to positively regulate shoot growth but negatively regulate root growth. As a result, the appropriate control of shoot- and root-related morphological traits, via the modulation of endogenous CK levels prior to the occurrence of a stress, as a preventive measure, is a promising approach for developing economically important drought-tolerant crops <xref ref-type="bibr" rid="pone.0042411-Werner1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0042411-Ha1"> [32] </xref> . Apart from their biochemical characteristics, tissue-specific and development stage-related expression of the CK metabolic genes indicate their functional specification and potential utility for the genetic engineering of specific traits. A well-known example is that among the seven <italic> Arabidopsis </italic> ATP/ADP <italic> <span class="gene">IPT</span> </italic> genes, <italic> IPT1 </italic> , <italic> 3 </italic> , <italic> 5 </italic> and <italic> 7 </italic> are expressed in the vegetative phase and <italic> IPT4 </italic> , <italic> 6 </italic> and <italic> 8 </italic> are not. Thus, the <italic> ipt1,3,5,7 </italic> quadruple mutant has reduced active CK levels which results in morphological adjustment (shorter shoot and longer primary root), hypersensitivity to ABA and enhanced cell membrane integrity contributing to enhanced drought-tolerant phenotypes <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0042411-Miyawaki2"> [19] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Among the <italic> <span class="gene">IPT</span> </italic> transcripts expressed in remarkably high abundance, the <italic> <span class="gene">GmIPT08</span> </italic> transcript was consistently increased in the leaves and young seedling shoots under <span class="trait">drought</span> or dehydration conditions ( <xref ref-type="fig" rid="pone-0042411-g005"> Figures 5A </xref> , <xref ref-type="fig" rid="pone-0042411-g006"> 6A </xref> and <xref ref-type="fig" rid="pone-0042411-g007"> 7A </xref> ). The induction level of this transcript was the highest in young seedling shoots (&#x0223c;300-fold, <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5A </xref> ). In the V6 leaves, the induction levels were correlated with the age of the trifoliate leaves; the older the leaf is the higher the induction ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6A </xref> ). In R2 leaves, <italic> <span class="gene">GmIPT08</span> </italic> was the only gene whose expression was significantly induced (&#x0223c;7-fold) by <span class="trait">drought</span> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ). Other <italic> <span class="gene">GmIPT</span> </italic> genes, which were significantly induced by <span class="trait">drought</span> in the V6-stage leaves, are <italic> <span class="gene">GmIPT09</span> </italic> and <italic> 13 </italic> , and the degree of induction of these genes was higher in the younger trifoliate leaves ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6A </xref> ). As for the drought-repressible <italic> <span class="gene">IPT</span> </italic> genes identified in the leaf tissues, the <italic> <span class="gene">GmIPT05</span> </italic> transcripts were repressed by <span class="trait">drought</span> in the leaves of various stages ( <xref ref-type="fig" rid="pone-0042411-g005"> Figures 5A </xref> , <xref ref-type="fig" rid="pone-0042411-g006"> 6A </xref> and <xref ref-type="fig" rid="pone-0042411-g007"> 7A </xref> ). In the R2-stage leaves, expression of almost all highly expressed <italic> <span class="gene">GmIPT</span> </italic> genes, except that of <italic> <span class="gene">GmIPT08</span> </italic> , was repressed by <span class="trait">drought.</span> The most substantial repression was observed in case of the <italic> <span class="gene">GmIPT07</span> </italic> gene ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ). In the roots of young seedlings, the most abundant <italic> <span class="gene">GmIPT08</span> </italic> transcript ( <xref ref-type="fig" rid="pone-0042411-g004"> Figure 4A </xref> ) was slightly induced by dehydration after 2 h of treatment, being subsequently repressed after 10 h of dehydration ( <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5A </xref> ). In the R2 auxiliary roots, the transcripts with high abundance did not change significantly upon dehydration. These data are in contrast with the observation in root hairs where all major transcripts, such as <italic> <span class="gene">GmIPT01</span> </italic> , <italic> 07 </italic> , <italic> 08 </italic> , <italic> 09 </italic> and <italic> 13 </italic> , were induced upon 5-h dehydration ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						(A) <italic> <span class="gene">GmIPT</span> </italic> genes. (B) <italic> <span class="gene">GmCKX</span> </italic> genes. Black bars; expression under normal condition; white bars, expression under <span class="trait">drought</span> condition.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) <italic> <span class="gene">GmIPT</span> </italic> genes. (B) <italic> <span class="gene">GmCKX</span> </italic> genes. Black bars; expression under normal condition; white bars, expression under <span class="trait">drought</span> condition.
					</p>
				
					<p>&nbsp;&nbsp;
						As for the <italic> <span class="gene">GmCKX</span> </italic> genes, five <italic> <span class="gene">GmCKX</span> </italic> s ( <italic> <span class="gene">GmCKX01</span> </italic> , <italic> 02 </italic> , <italic> 06 </italic> , <italic> 13 </italic> and <italic> 15 </italic> ) were found to be severely reduced at transcriptional level by <span class="trait">drought</span> in the V6 leaves ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6B </xref> ), and four of which, except <italic> <span class="gene">GmCKX06</span> </italic> , were also significantly downregulated in R2 leaves under <span class="trait">drought</span> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ). Nevertheless, in young seedling shoots, the expression of <italic> <span class="gene">GmCKX01</span> </italic> and <italic> 02 </italic> was induced by dehydration ( <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5B </xref> ). In addition, the <italic> <span class="gene">GmCKX14</span> </italic> transcript was found to be significantly induced by <span class="trait">drought</span> in various tissues at different stages. Furthermore, most of the <italic> <span class="gene">GmCKX</span> </italic> s were induced by <span class="trait">drought</span> in the R2 roots ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ). Among 17 <italic> <span class="gene">GmCKX</span> </italic> genes, only <italic> <span class="gene">GmCKX16</span> </italic> , whose encoded protein might be involved in degradation of <italic> c </italic> Z-type CKs as its AtCKX7 ortholog <xref ref-type="bibr" rid="pone.0042411-Gajdosova1"> [46] </xref> , was significantly upregulated by <span class="trait">drought</span> in the roots of the R2 stage soybean plants ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Strong lines of evidence have also suggested that CKs play an important role in the development of reproductive organs and seed <span class="trait">yield.</span> Disruption of <italic> <span class="gene">CKX3</span> </italic> and <italic> CKX5 </italic> genes in <italic> Arabidopsis </italic> resulted in higher CK levels, which subsequently led to larger inflorescences and floral meristems, increased size of the WUSCHEL expression domain, supernumerary ovules and increased seed <span class="trait">yield</span> of the <italic> ckx3,5 </italic> double mutant plants <xref ref-type="bibr" rid="pone.0042411-Bartrina2"> [55] </xref> . An increase in CK accumulation caused by a null mutation in the <italic> OsCKX2 </italic> gene was also shown to enhance the size of inflorescence meristems and increase the number of reproductive organs, resulting in enhanced grain <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0042411-Ashikari1"> [56] </xref> . Therefore, to gain an insight into the CK metabolism in reproductive organs, we analyzed the expression of CK metabolic genes in flowers, full pods and R5 seeds. Results shown in <xref ref-type="fig" rid="pone-0042411-g008"> Figure 8A </xref> indicated that <italic> <span class="gene">GmIPT02</span> </italic> is ubiquitously expressed in all three reproductive tissues examined while five other <italic> <span class="gene">GmIPT</span> </italic> s ( <italic> <span class="gene">GmIPT03</span> </italic> , <italic> 05 </italic> , <italic> 08 </italic> , <italic> 10 </italic> and <italic> 12 </italic> ) were not expressed. <italic> <span class="gene">GmIPT02</span> </italic> was the major transcript in flowers, <italic> <span class="gene">GmIPT01</span> </italic> and <italic> 02 </italic> expressed in the greatest abundance in full pods while <italic> <span class="gene">GmIPT01</span> </italic> , <italic> 02 </italic> and <italic> 11 </italic> were the most abundant transcripts in R5 seeds. The variation in the <italic> <span class="gene">GmIPT</span> </italic> transcript levels in flowers, pods and R5 seeds suggested that each of these organs required different <italic> <span class="gene">GmIPT</span> </italic> genes for CK biosynthesis. <italic> <span class="gene">GmIPT02</span> </italic> and <italic> <span class="gene">GmIPT11</span> </italic> might play the most important role in flowers and R5 seeds, respectively, while <italic> <span class="gene">GmIPT01</span> </italic> and <italic> 02 </italic> appear to be equally important in full pods as judged by their abundant expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Since an increasing amount of evidence suggests that CKs play an important role in the regulation of the <span class="trait">drought</span> response, we examined the effects of <span class="trait">drought</span> stress on overall CK metabolism in soybean plants by comparing the CK content in R2 leaves under control and <span class="trait">drought</span> conditions. A CK analysis was performed in leaves collected at the R2 stage, because this stage is considered critical, where <span class="trait">drought</span> stress may cause detrimental effects on soybean productivity ( <ext-link ext-link-type="uri" xlink:href="http://www.uwex.edu/ces/ag/issues/drought2003/soybeansrespondstress.html"> http://www.uwex.edu/ces/ag/issues/drought2003/soybeansrespondstress.html </ext-link> ). The results shown in <xref ref-type="fig" rid="pone-0042411-g009"> Figure 9 </xref> (right panel) indicated that CK content was lower in <span class="trait">drought</span> stressed leaves, which is in agreement with the results observed in other plant species <xref ref-type="bibr" rid="pone.0042411-Havlova1"> [25] </xref> , <xref ref-type="bibr" rid="pone.0042411-Kudoyarova1"> [62] </xref> . The most pronounced changes were observed with active CKs, <italic> t </italic> Z and DZ, which were significantly reduced in the R2 leaves upon <span class="trait">drought</span> treatment. The total content of <italic> t </italic> Z-type CKs was relatively unchanged. Our data suggest that a decrease in DZ content, which is associated with the downregulation of the majority of the <italic> <span class="gene">GmIPT</span> </italic> genes and the upregulation of <italic> <span class="gene">GmCKX03</span> </italic> and <italic> <span class="gene">GmCKX14</span> </italic> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7 </xref> ), contributes significantly to the overall reduction of CK content in <span class="trait">drought</span> stressed soybean leaves. Additionally, DZ may be involved in regulation of <span class="trait">drought</span> stress responses through <span class="gene">GmHK12,</span> which has high homology to AHK3. Both <italic> <span class="gene">GmHK12</span> </italic> and <italic> AHK3 </italic> were upregulated by <span class="trait">drought</span> stress, and AHK3 was shown to act as a negative regulator of <span class="trait">drought</span> stress signaling in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0042411-Tran2"> [5] </xref> , <xref ref-type="bibr" rid="pone.0042411-Le1"> [36] </xref> , <xref ref-type="bibr" rid="pone.0042411-Mochida1"> [37] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Research in the last several years has indicated that CKs play an essential role in the regulation of plant adaptation to various environmental stresses, including <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0042411-Peleg2"> [63] </xref> . Repression of CK metabolism under adverse stress conditions, which leads to a downregulation of CK signaling, is known as one of the mechanisms used by plants to adapt to adverse environmental conditions <xref ref-type="bibr" rid="pone.0042411-Ha1"> [32] </xref> . The results of this study provided the first insight into the previously uncharacterized CK metabolic genes encoding <span class="gene">GmIPTs,</span> which are involved in the rate-limiting step of CK biosynthesis. In addition, we also investigated <italic> <span class="gene">GmCKX</span> </italic> s, encoding the main CK degrading enzymes, which contribute to maintenance or reestablishment of CK homeostasis. Throughout our investigation, we placed a particular emphasis on their tissue-specific and/or drought-responsive expression. Collectively, these data enable us to understand the molecular mechanisms regulating CK homeostasis in various tissues/organs at different developmental stages under both normal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification of calcium-dependent protein kinases in soybean and analyses of their transcriptional responses to insect herbivory and <span class="trait">drought</span> stress</h2>
			<h3>PMID:26733237</h3>
				

				
					<h3>PMCID:4702179</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hettenhausen </surname>  <given-names> Christian </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Guiling </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Yanbiao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhuang </surname>  <given-names> Huifu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Ting </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qi </surname>  <given-names> Jinfeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Jianqiang </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Economic Plants and Biotechnology, Kunming Institute of Botany, Chinese Academy of Sciences </institution> , Kunming 650201,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 02 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calcium-dependent protein kinases (CDPKs) are plant-specific calcium sensors that play important roles in various aspects of plant physiology. Here, we investigated phylogenic relationships, chromosomal locations, gene structures, and tissue-specific, herbivory- and drought-induced expression profiles of soybean (Glycine max) <span class="gene">GmCDPKs.</span> Fifty <span class="gene">GmCDPK</span> genes were identified, which phylogenetically grouped into 4 distinct clusters and distributed across 13 sub-clusters. Individual classes of <span class="gene">GmCDPKs</span> harbor highly conserved mRNA splicing sites, and their exon numbers and lengths were consistent with the phylogenetic relationships, suggesting that at least 13 ancestral <span class="gene">CDPK</span> genes had emerged before the split of monocots and eudicots. Gene expression analysis indicated that several <span class="gene">GmCDPKs</span> were tissue-specific expressed. <span class="gene">GmCDPKs'</span> transcript levels changed after wounding, exhibited specific expression patterns after simulated Spodoptera exigua feeding or soybean aphid (Aphis glycines) herbivory, and were largely independent of the phytohormones jasmonic acid and salicylic acid. The most pronounced transcriptional responses were detected after <span class="trait">drought</span> and abscisic acid treatments with more than half of all <span class="gene">GmCDPKs</span> being upregulated, suggesting their important roles during abiotic stress responses in soybean. Our data provide an important foundation for further functional dissection of <span class="gene">GmCDPKs,</span> especially in the context of soybean-insect interactions and <span class="trait">drought</span> stress adaptation.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Recent findings have pointed to the roles of CDPKs in wounding- and herbivory-induced plant responses. The expression and enzymatic activity of maize ( <italic> Zea mays </italic> ) ZmCDPK11 are up-regulated by wounding, linolenic acid, and methyl jasmonate (MeJA), and the wound-induced activation of ZmCPK11 is dependent on phosphatidic acid produced in the phospholipase D (PLD) pathway <xref ref-type="bibr" rid="b7"> 7 </xref> <xref ref-type="bibr" rid="b8"> 8 </xref> . Arabidopsis AtCPK3 and AtCPK13 phosphorylate the transcription factor (TF) HsfB2a, and HsfB2a in turn regulates the transcript accumulation of several stress-related genes after <italic> Spodoptera littoralis </italic> feeding <xref ref-type="bibr" rid="b9"> 9 </xref> . Tomato ( <italic> Solanum lycopersicum </italic> ) LeCPK1 inhibits the plasma membrane H <sup> + </sup> -ATPase to induce extracellular alkalization as a part of the wound and pathogen defense signaling cascades <xref ref-type="bibr" rid="b10"> 10 </xref> , and LeCDPK2 phosphorylates LeACS2 in response to wounding, stabilizing this rate-limiting enzyme for ethylene (ET) biosynthesis by direct phosphorylation <xref ref-type="bibr" rid="b11"> 11 </xref> . In addition, Zhang <italic> et al. </italic> showed that in grape wine, several <italic> <span class="gene">CDPK</span> </italic> genes&#x02019; expression was increased by ET-treatment <xref ref-type="bibr" rid="b12"> 12 </xref> . <italic> N. attenuata </italic> CDPKs, NaCDPK4 and NaCDPK5, play redundant and negative roles in mediating wounding- and herbivory-elicited JA accumulations, and silencing <italic> NaCDPK4 </italic> and <italic> NaCDPK5 </italic> greatly elevates plant <span class="trait">resistance</span> to the specialist insect <italic> Manduca sexta </italic> <xref ref-type="bibr" rid="b13"> 13 </xref> <xref ref-type="bibr" rid="b14"> 14 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we carried out a genome-wide analysis and identified 50 <italic> <span class="gene">GmCDPK</span> </italic> s. Phylogenetic analysis was performed and <italic> <span class="gene">GmCDPK</span> </italic> s gene structures were compared to reveal their evolutionary relationships. Furthermore, we examined the expression of all <italic> <span class="gene">GmCDPK</span> </italic> s after wounding, simulated <italic> Spodoptera exigua </italic> herbivory, <italic> Aphis glycines </italic> feeding, and treatments of JA, ET, and SA, phytohormones important for plant defense against insects, as well as in response to <span class="trait">drought</span> and ABA. We found that the transcript levels of many <italic> <span class="gene">GmCDPK</span> </italic> s were not dependent on JA or SA signaling but were specifically altered in response to herbivory, suggesting that <span class="gene">GmCDPKs</span> may play important roles in soybean defense against insects. Moreover, almost 80% of all <italic> <span class="gene">GmCDPKs</span> </italic> were induced after <span class="trait">drought</span> or ABA treatments, highlighting their important role in regulating <span class="trait">drought</span> stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain insight into the functions of <italic> <span class="gene">GmCDPK</span> </italic> s in abiotic stress responses, we investigated the patterns of <italic> <span class="gene">GmCDPK</span> </italic> s&#x02019; expression after ABA and <span class="trait">drought</span> treatments ( <xref ref-type="fig" rid="f8"> Fig. 8 </xref> ). With 23 and 27 up-regulated <italic> <span class="gene">GmCDPKs</span> </italic> respectively, about half of the <italic> <span class="gene">GmCDPKs</span> </italic> responded to the individual treatments and only 10 were not regulated by at least one of them. As expected, the transcriptional responses partially overlapped ( <xref ref-type="fig" rid="f8"> Fig. 8C </xref> ) and many paralogous <italic> <span class="gene">GmCDPKs</span> </italic> , such as <italic> <span class="gene">GmCDPK19</span> </italic> and <italic> <span class="gene">GmCDPK47</span> </italic> and <italic> <span class="gene">GmCDPK6</span> </italic> and <italic> <span class="gene">GmCDPK36</span> </italic> showed consistent expression patterns. These findings confirm the importance of CDPKs in <span class="trait">drought</span> stress and suggest that the transcriptional regulation of <italic> <span class="gene">GmCDPKs</span> </italic> in response to <span class="trait">drought</span> is of ancient nature and existed long before the recent soybean genome duplication event.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, CDPKs have been shown to play important roles in plants responses to abiotic stresses, especially ABA and <span class="trait">drought</span> <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b46"> 46 </xref> <xref ref-type="bibr" rid="b47"> 47 </xref> <xref ref-type="bibr" rid="b48"> 48 </xref> . Our analyses revealed that almost 80% of all <italic> <span class="gene">GmCDPK</span> </italic> s responded to either ABA, <span class="trait">drought</span> treatment, or both, supporting the notion that CDPKs are central regulators of plant <span class="trait">drought</span> stress responses. ABA- and drought-induced transcriptional patterns of <italic> <span class="gene">GmCDPK</span> </italic> greatly overlap, nevertheless, about half of the drought-induced transcripts were not regulated by ABA, indicating that other regulatory factors are also upstream of the transcriptional regulation of <italic> <span class="gene">GmCDPK</span> </italic> s. As expected, <span class="trait">drought-</span> and herbivore-induced expression patterns did not show much overlap.
					</p>
				
					<p>&nbsp;&nbsp;
						Duplicated genes might undergo different evolutionary processes including nonfunctionalization, neofunctionalization, and subfunctionalization, and these are usually associated with divergence of gene expression profiles of the paralogous genes <xref ref-type="bibr" rid="b49"> 49 </xref> . Among the 24 paralogous pairs, 3 pairs <italic> <span class="gene">GmCDPK10/46</span> </italic> , <italic> <span class="gene">GmCDPK8/34</span> </italic> and <italic> <span class="gene">GmCDPK23/43</span> </italic> were induced by wounding ( <xref ref-type="fig" rid="f3"> Fig. 3A </xref> ); <italic> <span class="gene">GmCDPK23/43</span> </italic> was also induced by W+OS. Three paralogous pairs, <italic> <span class="gene">GmCDPK19/47</span> </italic> , <italic> <span class="gene">GmCDPK18/44</span> </italic> , and <italic> <span class="gene">GmCDPK6/36</span> </italic> were suppressed after W+OS ( <xref ref-type="fig" rid="f3"> Fig. 3A </xref> ); 2 paralogous pairs <italic> <span class="gene">GmCDPK5/24</span> </italic> and <italic> <span class="gene">GmCDPK6/36</span> </italic> were induced by aphid feeding ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ); ET increased the expression of the paralogous <italic> <span class="gene">GmCDPK9/45</span> </italic> but suppressed <italic> <span class="gene">GmCDPK7/35</span> </italic> ( <xref ref-type="fig" rid="f7"> Fig. 7C </xref> ). ABA induced 5 paralogous gene pairs, 3 of which were also induced by <span class="trait">drought,</span> and <span class="trait">drought</span> stress increased the expression of 10 pairs in total ( <xref ref-type="fig" rid="f8"> Fig. 8 </xref> ). In contrast, the expression patterns of two paralogous pairs, <italic> <span class="gene">GmCDPK4/26</span> </italic> and <italic> <span class="gene">GmCDPK37/41,</span> </italic> differed dramatically. <italic> <span class="gene">GmCDPK4</span> </italic> was induced after W+W, W+OS and ABA treatments but the paralog <italic> <span class="gene">GmCDPK26</span> </italic> was suppressed following W+OS treatment ( <xref ref-type="fig" rid="f3"> Fig. 3A </xref> ) and induced by <span class="trait">drought</span> ( <xref ref-type="fig" rid="f8"> Fig. 8 </xref> ). Similarly, W+W and W+OS elicited the expression of <italic> <span class="gene">GmCDPK37</span> </italic> , but <italic> <span class="gene">GmCDPK41</span> </italic> was suppressed by W+OS ( <xref ref-type="fig" rid="f3"> Fig. 3A </xref> ); <italic> <span class="gene">GmCDPK37</span> </italic> tended to be suppressed by SA, while <italic> <span class="gene">GmCDPK41</span> </italic> was induced by SA ( <xref ref-type="fig" rid="f7"> Fig. 7B </xref> ). These differences suggest possible neofunctionalization of <italic> <span class="gene">GmCDPK</span> </italic> s following gene duplication. Several other duplicated gene pairs (such as <italic> <span class="gene">GmCDPK14/21</span> </italic> , <italic> <span class="gene">GmCDPK27/48</span> </italic> and <italic> <span class="gene">GmCDPK2/30</span> </italic> ) demonstrated divergent expression patterns, with one gene responding to the treatment and the other remaining at control levels, indicating that they might have had undergone subfunctionalization. Consistent with the deviating expression patterns of these <italic> <span class="gene">GmCDPK</span> </italic> paralogs, in silico promoter analysis indicated that the distributions of <italic> cis </italic> -elements are different between many paralogs&#x02019; promoters, probably due to the divergence of the promoters after genome duplication events ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Dataset 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Recent advances have revealed CDPKs as central regulators of Ca <sup> 2+ </sup> -mediated immune and stress responses that are crucial for plant survival <xref ref-type="bibr" rid="b50"> 50 </xref> <xref ref-type="bibr" rid="b51"> 51 </xref> . Various CDPKs mediate transient and sustained transcriptional reprogramming and play key roles in the regulation of JA, SA, ET and ABA signaling, facilitating plant responses to wounding, herbivory and <span class="trait">drought.</span> Our data shed light on the potential roles of <span class="gene">GmCDPKs</span> in regulating soybean responses to biotic and abiotic stresses and provide evidence that many <span class="gene">GmCDPKs</span> may have functionally diverged since the two soybean genome duplications. Future genetic and biochemical studies are needed to unravel <span class="gene">GmCDPKs&#x02019;</span> functions in plant adaptation to environmental factors and the underlying mechanisms.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> A </bold> ) Leaves were sprayed with 1&#x02009;mM ABA or water as control and the transcript levels of <italic> <span class="gene">GmCDPK</span> </italic> s were measured in leaves from five replicate plants at 8&#x02009;h post-treatment. ( <bold> B </bold> ) For <span class="trait">drought</span> treatments, watering was stopped for 5 days and samples were harvested when leaves begun to lose turgor and wilted. Values represent mean (&#x02009;&#x000b1;&#x02009;SE; N&#x02009;=&#x02009;5) expression ratios of significantly ( <italic> t </italic> -test, p&#x02009;&#x0003c;&#x02009;0.05) and more than 1.5-fold up- or down-regulated <italic> <span class="gene">GmCDPK</span> </italic> transcript levels normalized to the <italic> <span class="gene">GmCDPK</span> </italic> levels in the respective controls; <italic> <span class="gene">GmCDPK</span> </italic> s whose expression levels did not change are not shown. ( <bold> C </bold> ) Venn diagram illustration of overlapping and specific transcriptional responses after ABA and <span class="trait">drought</span> treatments. Numbers show significantly ( <italic> t </italic> -test, p&#x02009;&#x0003c;&#x02009;0.05) and more than 1.5-fold up- or down-regulated genes in the respective group.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of soybean <span class="gene">GmCBL1</span> enhances abiotic stress tolerance and promotes hypocotyl elongation in Arabidopsis.</h2>
			<h3>PMID:23044418  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li ZY(1), Xu ZS, He GY, Yang GX, Chen M, Li LC, Ma YZ.
				<br/> 
				年份：2012 Nov 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Although extensive studies and remarkable progress have been made with Arabidopsis calcineurin B-like proteins (CBLs), knowledge of their functions in other plant species is still limited. Here we isolated gene <span class="gene">GmCBL1</span> from soybean, a homolog of AtCBL1 in Arabidopsis. <span class="gene">GmCBL1</span> was differentially induced by multiple abiotic stress and plant hormones, and its transcripts were abundant in seedlings and mature roots. We over-expressed <span class="gene">GmCBL1</span> in Arabidopsis and found that it enhanced tolerances to both high salt and <span class="trait">drought</span> stresses in the transgenic plants. Overexpression of <span class="gene">GmCBL1</span> also promoted hypocotyl elongation under light conditions. <span class="gene">GmCBL1</span> may regulate stress tolerance through activation of stress-related genes, and may control hypocotyl development by altering the expression of gibberellin biosynthesis-related genes. This study identifies a putative soybean CBL gene that functions in both stress tolerance and light-dependent hypocotyl development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and expression analysis of the Glycine max CYP707A gene family in response to <span class="trait">drought</span> and salt stresses.</h2>
			<h3>PMID:22751653  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng Y(1), Huang Y, Xian W, Wang J, Liao H.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIMS: Abscisic acid (ABA) plays crucial roles in plants' responses to abiotic stresses. ABA 8'-hydroxylation controlled by CYP707A genes has been well studied in Arabidopsis and rice, but not in legumes. The aims of the present study were to identify and functionally analyse the soybean CYP707A gene family, and to explore their expression patterns under dehydration and salt stresses. METHODS: A complementation experiment was employed to verify the function of soybean CYP707A1a in ABA catabolism. Genomic and cDNA sequences of other soybean CYP707A genes were isolated from the Phytozome database based on soybean CYP707A1a. The structure and phylogenetic relationship of this gene family was further analysed. The expression patterns of soybean CYP707A genes under dehydration and salt stress were analysed via quantitative real-time PCR. KEY RESULTS: Over-expression of <span class="gene">GmCYP707A1a</span> in the atcyp707a2 T-DNA insertion mutant decreased its sensitivity to ABA, indicating that <span class="gene">GmCYP707A1a</span> indeed functions as an ABA 8'-hydroxylase in higher plants. The soybean genome contains ten CYP707A genes. Gene structure and phylogenetic analysis showed high conservation of ten <span class="gene">GmCYP707A</span> genes to the other CYP707A genes from monocots and dicots. Seed imbibition induced expression of A1a, A1b, A2a, A2b, A2c, A3a and A5 in embryo, and expression of A1a, A1b, A2a and A2b in cotyledon. Dehydration induced expression of A1a, A1b, A2b, A2c, A3a, A3b, A4a, A4b and A5 both in roots and in leaves, whereas rehydration stimulated transcription of A2a, A2b, A3b, A4a and A5 in roots, and only A3b and A5 in leaves. Expression of all soybean CYP707A genes was induced either by short- or by long-term salt stress. CONCLUSIONS: The first biological evidence is provided that <span class="gene">GmCYP7071a</span> encodes an ABA 8'-hydroxylase through transgenic studies. Ten soybean <span class="gene">GmCYP707A</span> genes were identified, most of them expressed in multiple soybean tissues, and were induced by imbibition, dehydration and salinity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> <span class="gene">GmCYP82A3</span> </italic> , a Soybean Cytochrome <span class="gene">P450</span> Family Gene Involved in the Jasmonic Acid and Ethylene Signaling Pathway, Enhances Plant <span class="trait">Resistance</span> to Biotic and Abiotic Stresses</h2>
			<h3>PMID:27588421</h3>
				

				
					<h3>PMCID:5010195</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Qiang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Xiaoxia </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Shuai </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gan </surname>  <given-names> Shuping </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xing </surname>  <given-names> Han </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dou </surname>  <given-names> Daolong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cytochrome <span class="gene">P450</span> monooxygenases (P450s) represent a large and important enzyme superfamily in plants. They catalyze numerous monooxygenation/hydroxylation reactions in biochemical pathways, P450s are involved in a variety of metabolic pathways and participate in the homeostasis of phytohormones. The CYP82 family genes specifically reside in dicots and are usually induced by distinct environmental stresses. However, their functions are largely unknown, especially in soybean (Glycine max L.). Here, we report the function of <span class="gene">GmCYP82A3,</span> a gene from soybean CYP82 family. Its expression was induced by Phytophthora sojae infection, salinity and <span class="trait">drought</span> stresses, and treatment with methyl jasmonate (MeJA) or ethephon (ETH). Its expression levels were consistently high in resistant cultivars. Transgenic Nicotiana benthamiana plants overexpressing <span class="gene">GmCYP82A3</span> exhibited strong <span class="trait">resistance</span> to Botrytis cinerea and Phytophthora parasitica, and enhanced tolerance to salinity and <span class="trait">drought</span> stresses. Furthermore, transgenic plants were less sensitive to jasmonic acid (JA), and the enhanced <span class="trait">resistance</span> was accompanied with increased expression of the JA/ET signaling pathway-related genes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The cytochrome <span class="gene">P450</span> (CYP) monooxygenases represent a large and important superfamily in plants. The enzymes catalyze a wide variety of monooxygenation/hydroxylation reactions in biochemical pathways involved in primary and secondary metabolism pathways [ <xref rid="pone.0162253.ref024" ref-type="bibr"> 24 </xref> , <xref rid="pone.0162253.ref025" ref-type="bibr"> 25 </xref> ]. <italic> <span class="gene">P450</span> </italic> genes also participate in the homeostasis of phytohormones [ <xref rid="pone.0162253.ref025" ref-type="bibr"> 25 </xref> ]. In soybean, the cytochrome <italic> <span class="gene">P450</span> </italic> gene family is comprised of 322 genes and 378 pseudogenes, but the biological functions most of them have not been elucidated [ <xref rid="pone.0162253.ref026" ref-type="bibr"> 26 </xref> ]. The CYP82 family, which belongs to the CYP71 clan, is only present in dicots [ <xref rid="pone.0162253.ref027" ref-type="bibr"> 27 </xref> ]. Some CYP82 members are reported to be highly induced by environmental stress in tobacco ( <italic> Nicotiana tabacum </italic> L.), pea ( <italic> Pisum sativum </italic> L.), soybean, and <italic> Arabidopsis </italic> [ <xref rid="pone.0162253.ref028" ref-type="bibr"> 28 </xref> &#x02013; <xref rid="pone.0162253.ref031" ref-type="bibr"> 31 </xref> ]. A few CYP82 family members are involved in a variety of metabolic pathways. CYP82E4v1, from <italic> N </italic> . <italic> tabacum </italic> , was identified as a nicotine N-demethylase and the synthesis of nornicotine was suppressed in plants when the gene was silenced [ <xref rid="pone.0162253.ref032" ref-type="bibr"> 32 </xref> ]. CYP82N2v2, from <italic> Eschscholzia californica </italic> , is involved in sanguinarine biosynthesis by catalyzing the conversion of protopine to dihydrosanguinarine by the <span class="gene">P450</span> reaction [ <xref rid="pone.0162253.ref033" ref-type="bibr"> 33 </xref> ]. CYP82D is reported to be involved in lipophilic flavone biosynthesis in sweet basil ( <italic> Ocimum bacilicum </italic> L.) [ <xref rid="pone.0162253.ref034" ref-type="bibr"> 34 </xref> ]. In <italic> Arabidopsis </italic> , CYP82G1 catalyzes the final step in the synthesis of the common plant homoterpene volatiles TMTT/DMNT, CYP82C2 and CYP82C4 hydroxylate the therapeutic compound 8-methoxypsoralen [ <xref rid="pone.0162253.ref035" ref-type="bibr"> 35 </xref> , <xref rid="pone.0162253.ref036" ref-type="bibr"> 36 </xref> ]. The <italic> CYP82 </italic> family genes also participate in the interaction between plants and pathogens. For example, cotton ( <italic> Gossypium hirsutum </italic> L.) <italic> CYP82D </italic> can regulate systemic cell death by modulating the octadecanoid pathway and negatively regulate disease <span class="trait">resistance</span> to <italic> Verticillium dahliae </italic> by controlling JA biosynthesis [ <xref rid="pone.0162253.ref037" ref-type="bibr"> 37 </xref> ]. <italic> AtCYP82C2 </italic> can increase expressions of the JA-induced defense-related genes and contents of JA-induced IGs, and enhance <span class="trait">resistance</span> to <italic> Botrytis cinerea </italic> [ <xref rid="pone.0162253.ref038" ref-type="bibr"> 38 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmCYP82A3</span> </italic> was highly stress responsive and located in the <italic> P </italic> . <italic> sojae </italic> resistant QTL region [ <xref rid="pone.0162253.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0162253.ref028" ref-type="bibr"> 28 </xref> ]. But the accurate function remains unknown. In this study, we characterized the functions of <italic> <span class="gene">GmCYP82A3</span> </italic> in response to biotic and abiotic stresses. Ectopic expression of <italic> <span class="gene">GmCYP82A3</span> </italic> in <italic> N </italic> . <italic> benthamiana </italic> enhanced <span class="trait">resistance</span> to the pathogen <italic> Botrytis cinerea </italic> and <italic> Phytophthora parasitica </italic> , tolerance to the abiotic salinity and <span class="trait">drought</span> stresses. Furthermore, we demonstrated that JA/ET signaling pathway was altered in the <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants. These findings provide valuable information on soybean partial <span class="trait">resistance</span> mechanisms.
					</p>
				
					<p>&nbsp;&nbsp;
						We used several approaches to evaluate the effect of <italic> <span class="gene">GmCYP82A3</span> </italic> on plant <span class="trait">resistance</span> to pathogens. The uniform leaves cut from 7-week-old WT (wild type plant), EV (a transgenic line expressing empty vector as a negative control), and two <italic> <span class="gene">GmCYP82A3</span> </italic> overexpressing lines (2&#x02013;3 and 4&#x02013;1) were placed in Petri dishes, as described above for the soybean leaves. The detached leaves were infected with both necrotrophic <italic> B </italic> . <italic> cinerea </italic> and semi-biotrophic <italic> P </italic> . <italic> parasitica </italic> . For <italic> B </italic> . <italic> cinerea </italic> inoculation, we placed a 5 mm circular potato dextrose agar (PDA) agar containing mycelia cut from the edge of fresh cultured <italic> B </italic> . <italic> cinerea </italic> on the leaves. The diameters of the disease lesions were measured and the infected leaves were photographed at 4 days post-inoculation (dpi). For <italic> P </italic> . <italic> parasitica </italic> inoculation, the detached leaves were inoculated with 20 &#x003bc;l (approximately 5&#x000d7;10 <sup> 4 </sup> zoospores ml <sup> -1 </sup> ) of zoospore suspension. The WT and EV leaves were used as controls. The diameters of disease lesions were measured at 24 and 48 hpi, and the lesion area was calculated. Statistical significance was determined according to the Dunnett t-test method.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Gma.3136.2.A1_s_at</span> was highly induced by <italic> P </italic> . <italic> sojae </italic> in cultivars with high level of partial <span class="trait">resistance</span> based on an Affymetrix GeneChip microarray data analysis [ <xref rid="pone.0162253.ref023" ref-type="bibr"> 23 </xref> ]. This gene was up-regulated about 5-fold at both 3 and 5 dpi in Conrad, a cultivar with high level of partial <span class="trait">resistance,</span> and about 10-fold at 5 dpi in General, another highly partial resistant cultivar. However, the gene was weakly expressed in the moderately resistant cultivar Williams and the susceptible cultivar Sloan ( <xref ref-type="fig" rid="pone.0162253.g001"> Fig 1A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> (A) </bold> The microarray data of relative expression levels of <italic> <span class="gene">GmCYP82A3</span> </italic> . The expression levels in four soybean cultivars with different <span class="trait">resistance</span> level (Conrad and General with highly partial <span class="trait">resistance,</span> Williams with moderately partial <span class="trait">resistance</span> and Sloan is highly susceptible). The relative expression levels were normalized to the mock samples. <bold> (B) </bold> Expression pattern of <italic> <span class="gene">GmCYP82A3</span> </italic> induced by <italic> P </italic> . <italic> sojae </italic> determined by RT-PCR. Soybean leaves of Sloan, Williams and Conrad were inoculated with <italic> P </italic> . <italic> sojae </italic> and samples were taken at 0, 3, 6, 12 and 24 hpi. The <italic> <span class="gene">GmTUA</span> </italic> gene serves as a control.
					</p>
				
					<p>&nbsp;&nbsp;
						To explore the role of this <italic> <span class="gene">P450</span> </italic> family gene, <italic> <span class="gene">GmCYP82A3</span> </italic> was overexpressed in <italic> N </italic> . <italic> benthamiana </italic> plants driven by the 35S promoter and two independent T2 generations were selected for functional characterizations. Transgene integration and expression were confirmed by genomic PCR and RT-PCR, respectively ( <xref ref-type="supplementary-material" rid="pone.0162253.s003"> S2 Fig </xref> ). No obvious phenotypic differences were observed between the WT, EV, and <italic> <span class="gene">GmCYP82A3</span> </italic> overexpressing (2&#x02013;3 and 4&#x02013;1) seedlings. We first challenged the detached leaves with <italic> B </italic> . <italic> cinerea </italic> , a necrotrophic pathogen and causes necrotic symptoms on <italic> N </italic> . <italic> benthamiana </italic> ( <xref ref-type="fig" rid="pone.0162253.g003"> Fig 3A </xref> , <xref ref-type="supplementary-material" rid="pone.0162253.s004"> S3 Fig </xref> ). The average sizes of the lesion areas were ~ 357.15 and ~ 344.54 mm <sup> 2 </sup> at 4 dpi in WT and EV leaves, respectively. However, the necrotic areas were significantly reduced in the two independent overexpression lines. The average sizes of the lesion areas were ~ 201.97 and ~ 250.12 mm <sup> 2 </sup> ( <xref ref-type="fig" rid="pone.0162253.g003"> Fig 3B </xref> ). The results showed that <italic> <span class="gene">GmCYP82A3</span> </italic> could enhance <span class="trait">resistance</span> to <italic> B </italic> . <italic> cinerea </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						To further study the disease <span class="trait">resistance</span> function of <italic> <span class="gene">GmCYP82A3</span> </italic> , we also characterized the <span class="trait">resistance</span> level of transgenic plants to <italic> P </italic> . <italic> parasitica </italic> , a hemibiotrophic pathogen that is similar to <italic> P </italic> . <italic> sojae </italic> . When the hydroponically cultured 2-week-old plants were inoculated with <italic> P </italic> . <italic> parasitica </italic> zoospores, the WT and EV plants withered at 2 dpi, then exhibited stem rot and collapse of the whole plant at 5 dpi. In contrast, the two overexpression lines were more resistant and the symptoms were much weaker ( <xref ref-type="fig" rid="pone.0162253.g004"> Fig 4A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The detached transgenic leaves were also used to test for <span class="trait">resistance.</span> The average lesion areas caused by <italic> P </italic> . <italic> parasitica </italic> zoospores infection were significantly smaller on <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression leaves than those on WT and EV leaves, at both 24 and 48 hpi ( <xref ref-type="fig" rid="pone.0162253.g004"> Fig 4C and 4D </xref> ). Trypan blue staining showed fewer hyphae on <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression leaves ( <xref ref-type="fig" rid="pone.0162253.g004"> Fig 4E </xref> ). These results imply that <italic> <span class="gene">GmCYP82A3</span> </italic> contributes to <span class="trait">resistance</span> to the two tested pathogens.
					</p>
				
					<p>&nbsp;&nbsp;
						Because of changes of <span class="trait">resistance</span> levels to <italic> P </italic> . <italic> parasitica </italic> infection, we investigated <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants for the expression pattern of defense related marker genes in the defense cascade, and to gain further insight into possible regulation pathways. Four well known <italic> PR </italic> ( <italic> pathogenesis related </italic> ) genes were firstly selected for comparison. The SA marker <italic> PR1 </italic> ( <italic> Pathogenesis-related protein 1 </italic> ) and <italic> PR2 </italic> ( <italic> Pathogenesis-related protein 2 </italic> ) [ <xref rid="pone.0162253.ref048" ref-type="bibr"> 48 </xref> ] were significantly lower in <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants than those in WT and EV plant before and after infection, except <italic> PR1 </italic> showed no different at 12 hpi. In contrast, the basal expression levels of another two genes response to JA/ET, <italic> PR3 </italic> ( <italic> B-chitinase </italic> ) and <italic> PR4 </italic> ( <italic> Hevein-like protein </italic> ) [ <xref rid="pone.0162253.ref049" ref-type="bibr"> 49 </xref> ], were significantly higher in the two overexpression lines than in EV and WT plants. Both <italic> PR3 </italic> and <italic> PR4 </italic> greatly increased after infection with <italic> P </italic> . <italic> parasitica </italic> , and also had a higher expression level in overexpression plants. JA and ET regulated defense related marker gene <italic> PDF1 </italic> . <italic> 2 </italic> ( <italic> Plant defensing 1 </italic> . <italic> 2 </italic> ) [ <xref rid="pone.0162253.ref049" ref-type="bibr"> 49 </xref> ] significantly increased in overexpression plants and the basal expression levels were about 11 and 8-fold higher in 2&#x02013;3 and 4&#x02013;1 lines than that in WT. Meanwhile, <italic> PDF1 </italic> . <italic> 2 </italic> expression induced by <italic> P </italic> . <italic> parasitica </italic> infection was also significantly higher in overexpression plants at 12 and 24 hpi ( <xref ref-type="fig" rid="pone.0162253.g006"> Fig 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Given that the expression of <italic> <span class="gene">GmCYP82A3</span> </italic> was significantly induced by salt and <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone.0162253.g002"> Fig 2 </xref> ), we conducted tests to determine whether <italic> <span class="gene">GmCYP82A3</span> </italic> could regulate the response of the transgenic plants to abiotic stresses. Germination rates of the transgenic and WT seeds were calculated under abiotic stress. The germination of WT, EV, and overexpression seeds showed no obvious differences on the <span class="gene">MS</span> medium without NaCl or PEG6000 ( <xref ref-type="fig" rid="pone.0162253.g008"> Fig 8A </xref> ). Under treatment with 100 mM NaCl and 8% PEG 6000, germination rates of the seeds showed significant difference. The germination of WT and EV seeds were severely suppressed under the stresses. However, the germination rates of <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression seeds were higher compared with the WT and EV seeds at 4&#x02013;7 and 4&#x02013;5 days after sowing on <span class="gene">MS</span> medium containing 100 mM NaCl and 8% PEG 6000 respectively ( <xref ref-type="fig" rid="pone.0162253.g008"> Fig 8B and 8C </xref> ). Imply that enhanced tolerance to salt and <span class="trait">drought</span> stress of <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						Since partial <span class="trait">resistance</span> is important for soybean breeding <span class="trait">resistance</span> to Phytophthora root and stem rot, attempts have been made to identify the functional candidate genes conferring this <span class="trait">resistance.</span> A wide list of candidate genes underlying the soybean QTL conferring <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> were identified by whole-genome transcription profiling analysis of cultivars with different levels of <span class="trait">resistance</span> [ <xref rid="pone.0162253.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0162253.ref014" ref-type="bibr"> 14 </xref> , <xref rid="pone.0162253.ref020" ref-type="bibr"> 20 </xref> ]. In this study, we performed a functional analysis of the soybean <italic> <span class="gene">P450</span> </italic> gene, called <italic> <span class="gene">GmCYP82A3</span> </italic> , which is highly expressed in partial resistant soybean cultivars during <italic> P </italic> . <italic> sojae </italic> infection and underlying the QTL conferring <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> [ <xref rid="pone.0162253.ref014" ref-type="bibr"> 14 </xref> ]. Our results indicated that the expression level of <italic> <span class="gene">GmCYP82A3</span> </italic> can be induced by various abiotic stresses and phytohormones molecules. The overexpressed transgenic <italic> N </italic> . <italic> benthamiana </italic> plants showed enhanced <span class="trait">resistance</span> to the necrotrophic pathogen <italic> B </italic> . <italic> cinerea </italic> and the hemibiotrophic pathogen <italic> P </italic> . <italic> parasitica </italic> , implying that this gene underlying the QTL region may contribute to plant <span class="trait">resistance.</span> Further examination found the transgenic plants were less sensitive to jasmonic acid (JA), and the enhanced <span class="trait">resistance</span> accompanied with increased expression of the JA/ET signaling pathway genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies revealed that members of CYP82 family are highly stress responsive in tobacco, pea, soybean, and <italic> Arabidopsis </italic> [ <xref rid="pone.0162253.ref028" ref-type="bibr"> 28 </xref> &#x02013; <xref rid="pone.0162253.ref031" ref-type="bibr"> 31 </xref> ]. In this study, <italic> <span class="gene">GmCYP82A3</span> </italic> was induced by <italic> P </italic> . <italic> sojae </italic> infection and showed a different transcription profiles in soybean cultivars with diverse partial resistant levels, which consists with that from the candidate genes identification of soybean partial resistant QTL [ <xref rid="pone.0162253.ref013" ref-type="bibr"> 13 </xref> ]. Besides the pathogen induction, <italic> <span class="gene">GmCYP82A3</span> </italic> was remarkably up-regulated by salt and <span class="trait">drought</span> treatment. Phytohormones are widely believed to play key roles in signaling transduction involved in plant responses biotic and abiotic stresses [ <xref rid="pone.0162253.ref057" ref-type="bibr"> 57 </xref> &#x02013; <xref rid="pone.0162253.ref059" ref-type="bibr"> 59 </xref> ]. <italic> <span class="gene">GmCYP82A3</span> </italic> was highly induced by MeJA and ETH, but reduced in some degree when treated with SA, and did not respond to ABA treatment. This was consistent with previous reports, in which cotton and <italic> Arabidopsis CYP82 </italic> genes were both highly induced by JA/MeJA treatment [ <xref rid="pone.0162253.ref037" ref-type="bibr"> 37 </xref> , <xref rid="pone.0162253.ref038" ref-type="bibr"> 38 </xref> ]. Thus, we speculated that this soybean <italic> <span class="gene">P450</span> </italic> gene may play important role in abiotic stress and defense processes.
					</p>
				
					<p>&nbsp;&nbsp;
						JA, ET, SA signaling and the cross-talk between them play a role in plant defense response activation upon pathogen infection. For example, SA mediates defense responses to biotrophic pathogens, JA/ET mediates defense against necrotrophic or hemibiotrophic pathogens and usually antagonism with SA [ <xref rid="pone.0162253.ref058" ref-type="bibr"> 58 </xref> , <xref rid="pone.0162253.ref059" ref-type="bibr"> 59 </xref> ]. Transcriptional analysis of phytohormone signaling marker genes revealed that <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression affected the phytohormone signaling transduction. Two SA signaling marker genes, <italic> PR1 </italic> and <italic> PR2 </italic> were suppressed in transgenic plants both before and after pathogen infection compare with control, especially <italic> PR2 </italic> . The results proved that <italic> <span class="gene">GmCYP82A3</span> </italic> partly suppressed SA signaling pathway. <italic> PR3 </italic> , <italic> PR4 </italic> and <italic> PDF1 </italic> . <italic> 2 </italic> , which involved in JA and ET dependent <span class="trait">resistance</span> were significantly higher expression levels in overexpression plants [ <xref rid="pone.0162253.ref049" ref-type="bibr"> 49 </xref> ]. We speculated that <italic> <span class="gene">GmCYP82A3</span> </italic> might be involved in the JA/ET signaling pathway. In this study, the root growth in response to JA of <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants showed a similar phenotype of <italic> AtCYP82C2 </italic> , which the <italic> Arabidopsis </italic> overexpression plants also showed less sensitive to JA-mediated root growth inhibition [ <xref rid="pone.0162253.ref038" ref-type="bibr"> 38 </xref> ], implying that <italic> <span class="gene">GmCYP82A3</span> </italic> is involved in JA signaling. <italic> LOX1 </italic> has been widely demonstrated to be involved in JA biosynthesis and signaling pathway [ <xref rid="pone.0162253.ref050" ref-type="bibr"> 50 </xref> ], JAR1 catalyzes the formation of a biologically active JA-Ile conjugate [ <xref rid="pone.0162253.ref051" ref-type="bibr"> 51 </xref> ]. <italic> <span class="gene">COI1</span> </italic> encodes an F-box protein to assemble SCF <sup> <span class="gene">COI1</span> </sup> protein complex, act as JA-Ile receptor in JA signaling. The binding of JA-Ile to <span class="gene">COI1</span> leads degradation of JASMONATE ZIM-domain (JAZ) transcriptional repressor proteins, and results in the activation of JA responsive genes [ <xref rid="pone.0162253.ref052" ref-type="bibr"> 52 </xref> ]. The higher expression levels of <italic> LOX1 </italic> , <italic> JAR1 </italic> , and <italic> <span class="gene">COI1</span> </italic> in <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants after <italic> P </italic> . <italic> parasitica </italic> infection suggest that the JA signaling transduction may be enhanced. At least two distinct branches lie in JA signaling pathway to regulates downstream genes expression [ <xref rid="pone.0162253.ref049" ref-type="bibr"> 49 </xref> , <xref rid="pone.0162253.ref053" ref-type="bibr"> 53 </xref> ]. ERF branch confer <span class="trait">resistance</span> to necrotrophic pathogens and MYC2 branch is associated with wound response and insect herbivores <span class="trait">resistance,</span> but MYC2 branch also has been demonstrated to play a role for enhance pathogen defense [ <xref rid="pone.0162253.ref049" ref-type="bibr"> 49 </xref> , <xref rid="pone.0162253.ref060" ref-type="bibr"> 60 </xref> , <xref rid="pone.0162253.ref061" ref-type="bibr"> 61 </xref> ]. Alternatively, the MYC branch has an antagonistic effect with ERF branch, <italic> AtMYC2 </italic> represses the expression of ERF branch genes while activating the wound responsive genes such as <italic> VSP2 </italic> in MYC branch [ <xref rid="pone.0162253.ref053" ref-type="bibr"> 53 </xref> ]. The ERF branch and downstream defense relate genes <italic> ERF1 </italic> , <italic> PDF1 </italic> . <italic> 2 </italic> , <italic> PR3 </italic> and <italic> PR4 </italic> showed significant up-regulated in <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants. In contrast, the expression of <italic> MYC2 </italic> and <italic> VSP2 </italic> which involved in the MYC branch were suppressed. The results demonstrated that ectopic expression <italic> <span class="gene">GmCYP82A3</span> </italic> enhanced the signaling transduction of JA ERF branch during the transgenic <italic> N </italic> . <italic> benthamiana </italic> plants and <italic> P </italic> . <italic> parasitica </italic> interaction.
					</p>
				
					<p>&nbsp;&nbsp;
						Ethylene regulates wide physiological responses in plants. Ethylene is the first plant hormone, the signaling pathway has provided a framework in <italic> Arabidopsis </italic> . EIN3 is a plant specific nuclear transcription factor and positive regulate downstream transcription of ethylene response, such as the target gene of <italic> ERF1 </italic> [ <xref rid="pone.0162253.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0162253.ref056" ref-type="bibr"> 56 </xref> ]. The critical regulatory mechanism of ethylene signaling in the nucleus is controlled by EIN3 protein levels. In the absence of ethylene, EIN3 is rapidly degraded by 26S proteasomal under the regulation of two F-box proteins EBF1 and EBF2. In the presence of ethylene, EBF1 and EBF2 are degraded, thus allowing EIN3 protein accumulation and activated ethylene responsive gene expression [ <xref rid="pone.0162253.ref056" ref-type="bibr"> 56 </xref> ]. In this study, transcription profiles of <italic> EIN3 </italic> during the pathogen infection was elevated in <italic> <span class="gene">GmCYP82A3</span> </italic> overexpression plants, whereas, <italic> EBF2 </italic> showed suppression effect. In <italic> Arabidopsis </italic> , the ET and ERF branch of JA signaling act synergistically on the defense related genes expression, such as <italic> PDF1 </italic> . <italic> 2 </italic> , <italic> ERF1 </italic> , <italic> PR3 </italic> and <italic> PR4 </italic> [ <xref rid="pone.0162253.ref054" ref-type="bibr"> 54 </xref> , <xref rid="pone.0162253.ref062" ref-type="bibr"> 62 </xref> &#x02013; <xref rid="pone.0162253.ref064" ref-type="bibr"> 64 </xref> ]. <italic> ERF1 </italic> transcript was induced by <italic> B </italic> . <italic> cinerea </italic> infection, and the overexpressed <italic> Arabidopsis </italic> plants showed enhanced <span class="trait">resistance</span> to necrotrophic fungi such as <italic> B </italic> . <italic> cinerea </italic> and <italic> Plectospaerella cucumerina </italic> , but reduced tolerance to biotrophic <italic> Pst </italic> DC3000 [ <xref rid="pone.0162253.ref065" ref-type="bibr"> 65 </xref> ]. We believed that the ET signaling transduction was activated and contributed to the enhanced <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, ectopic expression soybean <italic> <span class="gene">GmCYP82A3</span> </italic> in <italic> N </italic> . <italic> benthamiana </italic> enhanced <span class="trait">resistance</span> to two different kinds of pathogens and salt, <span class="trait">drought</span> stress tolerance. The transcription analysis of phytohormones pathway genes revealed that <italic> PR1 </italic> and <italic> PR2 </italic> involved in SA signaling was suppressed, on the other hand, the ERF branch of JA and ET signaling genes were upregulated in the transgenic plants, indicating that the ERF branch of JA and ET signaling pathways were activated by <italic> <span class="gene">GmCYP82A3</span> </italic> . But the potential mechanism is still needed to be explored. Since several CYP82 family members participate in a variety of metabolic pathways [ <xref rid="pone.0162253.ref032" ref-type="bibr"> 32 </xref> &#x02013; <xref rid="pone.0162253.ref038" ref-type="bibr"> 38 </xref> ], so further characterization of the biochemical function is needed.
					</p>
				
					<p>&nbsp;&nbsp;
						As a form of incomplete <span class="trait">resistance</span> in the <italic> P </italic> . <italic> sojae </italic> -soybean system and non-race-specific <span class="trait">resistance,</span> partial <span class="trait">resistance</span> has been proposed as a way to improve the breeding efforts for soybean <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> . Previous studies showed that some cultivars with high levels of partial <span class="trait">resistance</span> were difficult to distinguish from the <italic> Rps </italic> genotype when inoculated with the simple isolate [ <xref rid="pone.0162253.ref071" ref-type="bibr"> 71 </xref> ]. In this study, we found that <italic> <span class="gene">GmCYP82A3</span> </italic> affected defense related genes expression associated with the JA/ET signaling pathway, conferring transgenic plants resistant to two different types of pathogens. From our understanding of the mechanism of soybean partial <span class="trait">resistance,</span> we believe that the strategy of generate highly partial resistant soybean cultivars offer a promising approach for Phytophthora root rot control.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <italic> AmRosea1 </italic> Gene Confers <span class="trait">Drought</span> and Salt Tolerance in Rice</h2>
			<h3>PMID:28025485</h3>
				
					<h3>PMCID:5297637</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dou </surname>  <given-names> Mingzhu </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-00002"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-18-00002"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sanhong </given-names>  </name>  <xref ref-type="aff" rid="af3-ijms-18-00002"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Suxin </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-00002"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Rongfeng </given-names>  </name>  <xref ref-type="aff" rid="af4-ijms-18-00002"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Huiyun </given-names>  </name>  <xref ref-type="aff" rid="af3-ijms-18-00002"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Feng </surname>  <given-names> Xianzhong </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-00002"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-18-00002"> 2 </xref>  <xref rid="c1-ijms-18-00002" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						KEGG pathway analysis results of both ethylene signal and JA signal pathways are shown in <xref ref-type="fig" rid="ijms-18-00002-f004"> Figure 4 </xref> . One gene (Os06t0605900-01), encoding EBF1, was up-regulated under both <span class="trait">drought</span> and salt stresses; another gene (Os02t0527600-01), encoding CTR1, was up-regulated under salt stress in transgenic plants compared to WT plants. These differences of gene expression repressed the ethylene signal transduction pathway in transgenic rice plants ( <xref ref-type="fig" rid="ijms-18-00002-f004"> Figure 4 </xref> ). Under <span class="trait">drought</span> stress, one gene (Os05t0449500-01) encoding <span class="gene">COI1</span> was up-regulated and under salt stress, and two genes (Os03t0402800-01 and Os07t0615200-01) encoding JAZ were down-regulated in transgenic plants compared to WT plants. In general, the JAZ signal transduction pathway was promoted in transgenic plants compared to WT plants under both <span class="trait">drought</span> and salt stresses ( <xref ref-type="fig" rid="ijms-18-00002-f004"> Figure 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Reactive oxygen species (ROS) induced by stress can destroy cell membranes, proteins, and nucleic acids [ <xref rid="B28-ijms-18-00002" ref-type="bibr"> 28 </xref> , <xref rid="B29-ijms-18-00002" ref-type="bibr"> 29 </xref> ]. The accumulation of ROS such as superoxide and hydroxyl radicals is largely counteracted by an intricate antioxidant defense system [ <xref rid="B30-ijms-18-00002" ref-type="bibr"> 30 </xref> , <xref rid="B31-ijms-18-00002" ref-type="bibr"> 31 </xref> ]. Three catalases (CATs) and fifteen peroxidases (PODs) are mainly up-regulated in transgenic plants compared to WT under <span class="trait">drought</span> and salt treatments ( <xref ref-type="table" rid="ijms-18-00002-t003"> Table 3 </xref> and <xref ref-type="table" rid="ijms-18-00002-t004"> Table 4 </xref> ). However, one ascorbate peroxidase (APX; Os04t0602100-01) and one <span class="gene">GST</span> (Os04t0435500-01) were always down-regulated in transgenic plants with both stresses. These results suggested that protection from oxidative damage might be the main reason for causing stress tolerance in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the expression profiles of genes in RNA-seq results, we performed qRT-PCR on a few DEGs, including stress responsible genes, <italic> RD22 </italic> (Os10t0409400-01), heat shock protein ( <italic> <span class="gene">HSP90</span> </italic> , Os06t0716700-01), LEA protein (Os05t0542500-02), <italic> RAB16A </italic> (Os11t0454300-01), catalase ( <italic> CAT </italic> , Os06t0727200-02) and submergence-induced protein 2 ( <italic> Sip2 </italic> , Os10t0419400-01); transcriptional factors, dehydration responsive element binding protein 1F ( <italic> DREB1F </italic> , Os04t0549700-01), <italic> <span class="gene">MYB48</span> </italic> (Os01t0975300-01) and <italic> NAC134 </italic> (Os12t0156100-01); and serine-threonine kinase, CBL-interacting protein kinase 14 ( <italic> CIPK14 </italic> , Os12t0113500-01). The qPCR outcomes correlated closely with the transcript abundances estimated from the RNA-seq output, and all the confirmed qPCR results in <xref ref-type="fig" rid="ijms-18-00002-f005"> Figure 5 </xref> supported the defense mechanisms against <span class="trait">drought</span> and salinity in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						On the contrast, the JA pathway positively regulates <span class="trait">drought</span> tolerance and salt tolerance in plants [ <xref rid="B54-ijms-18-00002" ref-type="bibr"> 54 </xref> , <xref rid="B55-ijms-18-00002" ref-type="bibr"> 55 </xref> , <xref rid="B56-ijms-18-00002" ref-type="bibr"> 56 </xref> , <xref rid="B57-ijms-18-00002" ref-type="bibr"> 57 </xref> , <xref rid="B58-ijms-18-00002" ref-type="bibr"> 58 </xref> ]. In the JA linear signal pathway, <span class="gene">COI1</span> is a positive regulator and JAZ is a negative regulator. In our study, the transcriptional level of <italic> <span class="gene">COI1</span> </italic> gene was up-regulated and <italic> JAZ </italic> genes were down-regulated in OXR plants as compared to WT rice plants under stress conditions ( <xref ref-type="fig" rid="ijms-18-00002-f004"> Figure 4 </xref> ). That means the JA signal pathway was up-regulated by ectopic expression of <italic> AmRosea1 </italic> , which was beneficial to confer <span class="trait">drought</span> and salt tolerance in transgenic rice.
					</p>
				
					<p>&nbsp;&nbsp;
						T2 transgenic homozygous lines were used for the stress tolerance treatment. On the seventh day after germination on half-strength <span class="gene">MS</span> medium, 48 seedlings of each line were transplanted into small pots containing nutritious soil (vermiculite:soil = 2:1). Four seedlings were grown per pot. One half of the seedlings were exposed to salinity stress, and the other half were exposed to <span class="trait">drought</span> stress. The small pots with holes on the bottom were placed in large plastic trays that held a depth of 7&#x02013;8 cm of water.
					</p>
				
					<p>&nbsp;&nbsp;
						Seeds were washed with distilled water and germinated in the dark at 28 &#x000b0;C for 3 days. Germinated seeds were transferred to 96-well plates with no bottom, floating on 1/2 <span class="gene">MS</span> solution in a growth chamber (16/8 h light/dark photoperiod at 25 &#x000b0;C with 70% RH). Seedlings at the three-leaf stage were treated with 200 mM NaCl or 15% PEG4000 for salinity and <span class="trait">drought</span> stress, respectively.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Transcriptomic Comparison of Two Bambara Groundnut Landraces under Dehydration Stress</h2>
			<h3>PMID:28420201</h3>
				
					<h3>PMCID:5406868</h3>
					<h4>期刊:Genes</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Faraz </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-08-00121"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Hui Hui </given-names>  </name>  <xref ref-type="aff" rid="af2-genes-08-00121"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ajmera </surname>  <given-names> Ishan </given-names>  </name>  <xref ref-type="aff" rid="af3-genes-08-00121"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hodgman </surname>  <given-names> Charlie </given-names>  </name>  <xref ref-type="aff" rid="af3-genes-08-00121"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mayes </surname>  <given-names> Sean </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-08-00121"> 1 </xref>  <xref ref-type="aff" rid="af2-genes-08-00121"> 2 </xref>  <xref rid="c1-genes-08-00121" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Chungui </given-names>  </name>  <xref ref-type="aff" rid="af4-genes-08-00121"> 4 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ability to grow crops under low-water conditions is a significant advantage in relation to global food security. Bambara groundnut is an underutilised crop grown by subsistence farmers in Africa and is known to survive in regions of water deficit. This study focuses on the analysis of the transcriptomic changes in two bambara groundnut landraces in response to dehydration stress. A cross-species hybridisation approach based on the Soybean Affymetrix GeneChip array has been employed. The differential gene expression analysis of a water-limited treatment, however, showed that the two landraces responded with almost completely different sets of genes. Hence, both landraces with very similar genotypes (as assessed by the hybridisation of genomic DNA onto the Soybean Affymetrix GeneChip) showed contrasting transcriptional behaviour in response to dehydration stress. In addition, both genotypes showed a high expression of dehydration-associated genes, even under water-sufficient conditions. Several gene regulators were identified as potentially important. Some are already known, such as <italic> WRKY40 </italic> , but others may also be considered, namely <italic> PRR7 </italic> , <italic> ATAUX2 </italic> -11, CONSTANS-like 1, <italic> <span class="gene">MYB60</span> </italic> , <italic> AGL-83 </italic> , and a Zinc-finger protein. These data provide a basis for <span class="trait">drought</span> trait research in the bambara groundnut, which will facilitate functional genomics studies. An analysis of this dataset has identified that both genotypes appear to be in a dehydration-ready state, even in the absence of dehydration stress, and may have adapted in different ways to achieve <span class="trait">drought</span> <span class="trait">resistance.</span> This will help in understanding the mechanisms underlying the ability of crops to produce viable yields under <span class="trait">drought</span> conditions. In addition, cross-species hybridisation to the soybean microarray has been shown to be informative for investigating the bambara groundnut transcriptome.
					</p>
				
					<p>&nbsp;&nbsp;
						Understanding the mechanisms underlying the ability of crops to produce viable yields under <span class="trait">drought</span> conditions is a priority for global food security. This study has examined the transcriptomic reponse to dehydration and recovery in two genotypes derived from landraces of bambara groundnut, in an attempt to investigate the molecular mechanisms occurring in the two landraces. In addition, this study also tested whether the cross-species hybridisation to the soybean microarray is suitable for investigating the bambara groundnut transcriptome. It was shown that many potential dehydration-responsive genes are expressed, even under water-sufficient conditions, in both landraces, suggesting that bambara groundnut could at least be in a partially ready state for dehydration, even in the absence of dehydration stress. In terms of differential expression, there were only a very small number of genes differentially expressed under water-limited treatment in both landraces, suggesting that the slow and progressive dehydration stress might not cause a significant shock to the plants. Although the transcription factors and dehydration-response genes were largely different between the two landraces, they may achieve the same effect in terms of survival under <span class="trait">drought</span> conditions. The DipC genotype displayed the differential expression of some well-known dehydration-associated transcriptions factors (especially <italic> WRKY40 </italic> ), while TN showed the differential expression of CONSTANS-LIKE 1 and <italic> <span class="gene">MYB60</span> </italic> . Cross-species hybridisation to the soybean microarray has been shown to be informative for investigating the bambara groundnut transcriptome, as good gene retention was observed at high gDNA hybridisation thresholds.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparison of qPCR and microarray intensity values: Rows ( <bold> A </bold> ) and ( <bold> B </bold> ) respectively refer to results for DipC and TN. The left- and right-hand pairs of columns correspond to the qPCR and microarray values for DipC and TN, respectively. The gene under study is named at the top of each panel. In order, the investigated genes are Beta-fructofuranosidase, <italic> COMT </italic> , <italic> UBC-2 </italic> , and <italic> <span class="gene">PAL1</span> </italic> . qPCR and Microarray values are shown as fold changes with respect to the water-sufficient treatment (Irrigated). Error bars denote the standard error. Single and double asterisks indicate that <italic> p </italic> -value is less than 0.05 and 0.01, respectively, which was assessed by the paired <italic> t </italic> -test between groups. Irrigated and <span class="trait">Drought</span> refer to water-sufficient and water-limited treatments, respectively.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the stress associated microRNAs in <italic> Glycine max </italic> by deep sequencing</h2>
			<h3>PMID:22112171</h3>
				
					<h3>PMCID:3267681</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Li </surname>  <given-names> Haiyan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hyli99@163.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Dong </surname>  <given-names> Yuanyuan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dongyuanyuan_dyy@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Yin </surname>  <given-names> Hailong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hlyin05@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Wang </surname>  <given-names> Nan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wangnanlunwen@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Yang </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yangjing5122010@163.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Liu </surname>  <given-names> Xiuming </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xiumingliu@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Wang </surname>  <given-names> Yanfang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> nifengcao_2000@163.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Wu </surname>  <given-names> Jinyu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> iamwujy@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> Li </surname>  <given-names> Xiaokun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> xiaokunli@163.net </email>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To confirm and validate the results obtained from the Solexa library, we examined the expression patterns of four known miRNAs and two novel miRNAs. These six miRNAs (miR166b, miR169d, miR482b, miR1507a, <span class="gene">Gma-m001,</span> and <span class="gene">Gma-m002)</span> were individually selected and experimentally verified by northern blotting hybridization. The sequences of antisense RNA probes are listed in Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> . By comparing the miRNA results by Solexa sequencing to northern hybridization, three stress-responsive miRNAs (miR166b, miR169d, and miR1507a) were identified with identical expression patterns. MiR166b and MiR1507a were up-regulated under <span class="trait">drought,</span> salinity, and alkalinity conditions. MiR169d was up-regulated under <span class="trait">drought</span> and alkalinity (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). While the expression patterns of miR482b and <span class="gene">Gma-m002</span> remained unchanged by the three stress conditions when tested by northern blotting. However, these were up-regulated under <span class="trait">drought</span> stress according to the Solexa results. Based on the northern blot analysis, the expression level of <span class="gene">Gma-m001</span> decreased under salinity stress, but identical patterns were observed under <span class="trait">drought</span> and alkalinity when compared with Solexa sequencing (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). Therefore, the expression pattern obtained by RNA blot analysis may reflect the result from deep sequencing.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Northern blotting </bold> . Northern blotting confirming differential expression of miRNAs. Total RNA (30 &#x003bc;g) from each of the four conditions (mock, <span class="trait">drought,</span> salinity, and alkalinity) was loaded and probed with miRNAs probe. The blot was hybridized with six miRNAs (miR166b, miR169d, miR482b, miR1507a, <span class="gene">Gma-m001,</span> and <span class="gene">Gma-m002).</span> The rRNA bands were shown as a loading control. miR166b, miR169d, and miR1507a expressed identical patterns when compared with Solexa sequencing.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">GmPHD-Type</span> Transcription Regulators Improve Stress Tolerance in Transgenic Arabidopsis Plants</h2>
			<h3>PMID:19789627</h3>
				

				
					<h3>PMCID:2747011</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wei </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Huang </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Yu-Jun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zou </surname>  <given-names> Hong-Feng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Hui-Wen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Jing-Yun </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xue-Yi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Wan-Ke </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jin-Song </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shou-Yi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 9 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean [Glycine max (L.) Merr.] is one of the most important crops for oil and protein resource. Improvement of stress tolerance will be beneficial for soybean seed production. PRINCIPAL FINDINGS: Six <span class="gene">GmPHD</span> genes encoding Alfin1-type PHD finger protein were identified and their expressions differentially responded to <span class="trait">drought,</span> salt, cold and ABA treatments. The six <span class="gene">GmPHDs</span> were nuclear proteins and showed ability to bind the cis-element "GTGGAG". The N-terminal domain of <span class="gene">GmPHD</span> played a major role in DNA binding. Using a protoplast assay system, we find that <span class="gene">GmPHD1</span> to <span class="gene">GmPHD5</span> had transcriptional suppression activity whereas <span class="gene">GmPHD6</span> did not have. In yeast assay, the <span class="gene">GmPHD6</span> can form homodimer and heterodimer with the other <span class="gene">GmPHDs</span> except <span class="gene">GmPHD2.</span> The N-terminal plus the variable regions but not the PHD-finger is required for the dimerization. Transgenic Arabidopsis plants overexpressing the <span class="gene">GmPHD2</span> showed salt tolerance when compared with the wild type plants. This tolerance was likely achieved by diminishing the oxidative stress through regulation of downstream genes. SIGNIFICANCE: These results provide important clues for soybean stress tolerance through manipulation of PHD-type transcription regulator.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Six <italic> <span class="gene">GmPHD</span> </italic> genes encoding Alfin1-type PHD finger protein were identified and their expressions differentially responded to <span class="trait">drought,</span> salt, cold and ABA treatments. The six <span class="gene">GmPHDs</span> were nuclear proteins and showed ability to bind the cis-element &#x0201c;GTGGAG&#x0201d;. The N-terminal domain of <span class="gene">GmPHD</span> played a major role in DNA binding. Using a protoplast assay system, we find that <span class="gene">GmPHD1</span> to <span class="gene">GmPHD5</span> had transcriptional suppression activity whereas <span class="gene">GmPHD6</span> did not have. In yeast assay, the <span class="gene">GmPHD6</span> can form homodimer and heterodimer with the other <span class="gene">GmPHDs</span> except <span class="gene">GmPHD2.</span> The N-terminal plus the variable regions but not the PHD-finger is required for the dimerization. Transgenic Arabidopsis plants overexpressing the <italic> <span class="gene">GmPHD2</span> </italic> showed salt tolerance when compared with the wild type plants. This tolerance was likely achieved by diminishing the oxidative stress through regulation of downstream genes.
					</p>
				
					<p>&nbsp;&nbsp;
						A gene fragment (256 bp) encoding a PHD finger was identified during cDNA-AFLP analysis using stress-tolerant lines and stress-sensitive lines from the population of the recombined inbred lines derived from the soybean Jindou No. 23 (JD23, <span class="trait">drought-</span> and salt-tolerant) and Huibuzhi (HBZ, <span class="trait">drought-</span> and salt-sensitive). Expression of the corresponding gene was higher in the stress-tolerant pool than that in the stress-sensitive pool (data not shown). After EST assembly, a full-length gene <italic> <span class="gene">GmPHD1</span> </italic> (DQ973812), was obtained, which encoded a PHD finger protein of 253 amino acids. Further searching and assembly of soybean EST sequences revealed five other members of this gene family, namely <italic> <span class="gene">GmPHD2</span> </italic> to <italic> <span class="gene">GmPHD6</span> </italic> (DQ973807, DG973808, DQ973809, DQ973810, and DQ973811). The <italic> <span class="gene">GmPHD3</span> </italic> and <italic> <span class="gene">GmPHD6</span> </italic> were partial in 5&#x02032;- and 3&#x02032;-end respectively, and their full-length open reading frames were obtained using RACE method.
					</p>
				
					<p>&nbsp;&nbsp;
						Expressions of the six <italic> <span class="gene">GmPHDs</span> </italic> were investigated in JD23 and HBZ in response to different treatments, including high NaCl, <span class="trait">drought,</span> ABA and cold ( <xref ref-type="fig" rid="pone-0007209-g003"> Fig. 3 </xref> ). All of the genes were induced in response to <span class="trait">drought,</span> but showed differences in responses to the other stresses. One of the genes, <italic> <span class="gene">GmPHD4</span> </italic> , was induced in response to all four conditions while the other five genes were induced in response to two or three conditions. Interestingly, the <italic> <span class="gene">GmPHD4</span> </italic> and <italic> <span class="gene">GmPHD5</span> </italic> were the only genes induced in response to low temperature and in both cases, this was only observed in the more stress-tolerant line JD23. These results indicate that the six <italic> <span class="gene">GmPHD</span> </italic> genes were differentially regulated in response to various treatments, and in most cases, the inductions of the <italic> <span class="gene">GmPHD</span> </italic> genes were stronger in stress-tolerant JD23 than those in stress-sensitive HBZ.
					</p>
				
					<p>&nbsp;&nbsp;
						Eighty-three canonical PHD finger proteins have been identified in Arabidopsis <xref ref-type="bibr" rid="pone.0007209-Lee1"> &#x0005b;43&#x0005d; </xref> . Only several proteins containing the PHD finger domain have been studied in plants. However, except the conserved C4HC3 residues, other sequences in the PHD domains are divergent. The functions of these proteins are also different, ranging from regulation of anther development and male meiosis <xref ref-type="bibr" rid="pone.0007209-Reddy1"> &#x0005b;46&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0007209-Yang1"> &#x0005b;48&#x0005d; </xref> to regulation of vernalization and flowering <xref ref-type="bibr" rid="pone.0007209-Pieiro1"> &#x0005b;49&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0007209-Greb1"> &#x0005b;52&#x0005d; </xref> , disease <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0007209-Eulgem1"> &#x0005b;53&#x0005d; </xref> , apical meristem maintenance <xref ref-type="bibr" rid="pone.0007209-Saiga1"> &#x0005b;54&#x0005d; </xref> , specification of vasculature and primary root meristem <xref ref-type="bibr" rid="pone.0007209-Thomas1"> &#x0005b;55&#x0005d; </xref> , and embryogenesis and sister-chromatid cohesion <xref ref-type="bibr" rid="pone.0007209-Sebastian1"> &#x0005b;56&#x0005d; </xref> . Unlike the PHD-containing proteins above, the present <span class="gene">GmPHDs</span> shared high homology only with <italic> Alfin1 </italic> from <italic> Medicago sativa. </italic> The <italic> Alfin1 </italic> can be induced by salt stress and enhance salt tolerance in the transgenic plants <xref ref-type="bibr" rid="pone.0007209-Winicov1"> &#x0005b;20&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0007209-Winicov2"> &#x0005b;21&#x0005d; </xref> . The present six <italic> <span class="gene">GmPHD</span> </italic> genes were differentially expressed in <span class="trait">drought-</span> and salt-tolerant JD23 and <span class="trait">drought-</span> and salt-sensitive HBZ in response to salt, <span class="trait">drought,</span> cold and ABA treatment, indicating that this subset of genes may have specific roles in multiple stress responses. In most cases, these genes were induced to a higher intensity in the tolerant JD23 cultivar than that in the sensitive HBZ, suggesting that the genes may contribute to the stress tolerance of the JD23 cultivar. Different from the specific <italic> Alfin1 </italic> expression in roots, the six <italic> <span class="gene">GmPHD</span> </italic> genes were expressed in multiple organs. We selected the <span class="gene">GmPHD2</span> for transgenic analysis because this protein had the least homology (67&#x00025;) with the Alfin1. Overexpression of the <italic> <span class="gene">GmPHD2</span> </italic> improved salt tolerance of the transgenic plants, indicating that proteins with transcriptional repression can also confer stress tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A comprehensive resource of <span class="trait">drought-</span> and salinity- responsive ESTs for gene discovery and marker development in chickpea ( <italic> Cicer arietinum </italic> L.)</h2>
			<h3>PMID:19912666</h3>
				
					<h3>PMCID:2784481</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> r.k.varshney@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Hiremath </surname>  <given-names> Pavana J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> p.hiremath@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lekha </surname>  <given-names> Pazhamala </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lekha_5@rediffmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kashiwagi </surname>  <given-names> Junichi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> jkashi@res.agr.hokudai.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Balaji </surname>  <given-names> Jayashree </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> balaji.jayashree@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Deokar </surname>  <given-names> Amit A </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> aadeokar@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Vadez </surname>  <given-names> Vincent </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> v.vadez@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Xiao </surname>  <given-names> Yongli </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> yxiao@jcvi.org </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Srinivasan </surname>  <given-names> Ramamurthy </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> srinivasan53@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Gaur </surname>  <given-names> Pooran M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> p.gaur@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Siddique </surname>  <given-names> Kadambot HM </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> ksiddique@fnas.uwa.edu.au </email>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Town </surname>  <given-names> Christopher D </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> cdtown@jcvi.org </email>  </contrib>  <contrib contrib-type="author" id="A13">  <name>  <surname> Hoisington </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> d.hoisington@cgiar.org </email>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 11 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The sub-cluster IIa contains genes specifically expressed in the ICC 4958_dehydration root library. Two contigs homologous to the biotic stress-responsive 'probable pleiotropic drug <span class="trait">resistance'</span> ( <italic> PDR </italic> ) protein (UniProt ID: Q7PC86) and 'phenylalanine ammonia-lyase' ( <italic> <span class="gene">PAL</span> </italic> ) (UniProt ID: P45732) were identified. Both <italic> PDR </italic> and <italic> <span class="gene">PAL</span> </italic> are involved in the phenylpropanoid and flavanoid/isoflavanoid pathways leading to phenylpropanoid biosynthesis in plants in response to various biotic and abiotic stresses [ <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> ]. The relative transcript counts of <italic> PDR </italic> and <italic> <span class="gene">PAL</span> </italic> signify their associated role in stress signaling processes.
					</p>
				
					<p>&nbsp;&nbsp;
						The sub-cluster IIId includes 23 (21.9%) contigs whose transcripts are highly expressed in both the ICC 4958_Drought_Field library and ICC 1882_Drought_Field library and also contains a majority of stress-responsive functionally important genes. Up-regulation of membrane spanning genes such as 'probable aquaporin <italic> <span class="gene">PIP</span> </italic> -type' (UniProt ID: P25794; Q9ATM4), which mediate regulation of root hydraulic conductivity in response to environmental stimuli [ <xref ref-type="bibr" rid="B45"> 45 </xref> ] were observed also in ICCV2_Salinity library as well as their significant expression in the other two libraries mentioned above and is not unexpected since its involvement is well reported in plant <span class="trait">drought</span> stress response. It is noteworthy that the transcripts annotating to 'metallothionein- like proteins' (MTs) (UniProt ID: Q39458, Q39459, Q9SSK5) involved in heavy metal detoxification and accumulate in response to high metal concentration, nutrient deprivation and heat shock [ <xref ref-type="bibr" rid="B46"> 46 </xref> ], identified in this cluster were highly expressed in ICC 4958_Drought_Field library and even more so in ICCV 2_Salinity library. Their high expression level in the ICCV 2_Salinity library may be correlated with their putative role in detoxification of salts accumulated in salinity stressed roots. Contigs similar to classical 'arabinogalactan proteins' (AGPs) (UniProt ID: Q9ZT16), abundant in the plant cell wall and plasma membrane [ <xref ref-type="bibr" rid="B47"> 47 </xref> ] were identified to be highly expressed in the ICC 4958_Drought_Field library. 'Fasciclin like Arabinogalactans' (FLAs) is a class of AGPs known to be regulated both during developmental processes and stress responses. Different classes of FLAs have been associated with ABA and down-regulated by <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. Likewise, a high copy number of transcripts corresponding to the ABR gene (UniProt ID: Q06931) were highly expressed in ICC 1882_Drought_Field library (423 transcripts), ICC 4958_Drought_Field library (291 transcripts) and moderately expressed in ICCV 2_Salinity library (55 transcripts). The predicted expression pattern of AGPs and ABRs identified in this study is also consistent with experimental observations in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B47"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						RKV planned and coordinated the study and wrote the <span class="gene">MS</span> together with PJH and finalized the MS, PJH executed majority of experiments and helped RKV in writing and finalizing the MS, PL, JK, AAD and VV were involved in setting up <span class="trait">drought</span> and salinity experiments and isolation of RNA, JB was involved in bioinformatics analysis, YX and CDT sequenced several cDNA libraries, RS, PMG, KHMS and DAH collaborated with RKV in planning the study. All authors read and approved the final manuscript.
					</p>
				
					<p>&nbsp;&nbsp;
						RKV planned and coordinated execution of study, wrote the <span class="gene">MS</span> together with PJH and finalized the MS. PJH conducted some experiments, analyzed data and helped RKV in writing and finalizing the MS. PL, JK, AAD and VV were involved in setting up <span class="trait">drought</span> and salinity experiments and isolation of RNA, JB together with RKV and PJH was involved in bioinformatics analysis, YX and CDT sequenced several cDNA libraries, RS, PMG, KHMS and DAH collaborated with RKV in planning the study. All authors read and approved the final manuscript.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A G-Protein &#x003b2; Subunit, AGB1, Negatively Regulates the ABA Response and <span class="trait">Drought</span> Tolerance by Down-Regulating AtMPK6-Related Pathway in <italic> Arabidopsis </italic></h2>
			<h3>PMID:25635681</h3>
				
					<h3>PMCID:4312036</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong-bei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="econtrib001">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ming </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="econtrib001">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Ya-nan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao-shi </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Lian-cheng </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Yao-feng </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> You-zhi </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Heterotrimeric G-proteins are versatile regulators involved in diverse cellular processes in eukaryotes. In plants, the function of G-proteins is primarily associated with ABA signaling. However, the downstream effectors and the molecular mechanisms in the ABA pathway remain largely unknown. In this study, an <italic> AGB1 </italic> mutant ( <italic> agb1-2 </italic> ) was found to show enhanced <span class="trait">drought</span> tolerance, indicating that <italic> AGB1 </italic> might negatively regulate <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> . Data showed that AGB1 interacted with protein kinase AtMPK6 that was previously shown to phosphorylate <italic> AtVIP1 </italic> , a transcription factor responding to ABA signaling. Our study found that transcript levels of three ABA responsive genes, <italic> AtMPK6 </italic> , <italic> AtVIP1 </italic> and <italic> AtMYB44 </italic> (downstream gene of <italic> AtVIP1 </italic> ), were significantly up-regulated in <italic> agb1-2 </italic> lines after ABA or <span class="trait">drought</span> treatments. Other ABA-responsive and drought-inducible genes, such as <italic> <span class="gene">RD29A</span> </italic> (downstream gene of <italic> AtMYB44 </italic> ), were also up-regulated in <italic> agb1-2 </italic> lines. Furthermore, overexpression of <italic> AtVIP1 </italic> resulted in hypersensitivity to ABA at seed germination and seedling stages, and significantly enhanced <span class="trait">drought</span> tolerance in transgenic plants. These results suggest that <italic> AGB1 </italic> was involved in the ABA signaling pathway and <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> through down-regulating the <italic> AtMPK6 </italic> , <italic> AtVIP1 </italic> and <italic> AtMYB44 </italic> cascade.
					</p>
				
					<p>&nbsp;&nbsp;
						For ABA sensitivity and PEG tolerance analyses, seeds of WT and OE-VIP1&#x02013;4, 5, 6 lines were germinated on <span class="gene">MS</span> medium for 3 d, and then transferred to normal medium or medium contained ABA or PEG. Root lengths of seedlings grown in 16 h light/8 h darkness were recorded after 14 d. For <span class="trait">drought</span> tolerance analyses, three-week-old plants in soil were withheld from watering for 21 d and then were re-watered at the 21 <sup> st </sup> day of <span class="trait">drought.</span> Observations were recorded after 3 d or 5 d of re-watering.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate <span class="trait">drought</span> stress tolerance of G-protein &#x003b2; subunit mutant lines, we selected a knockout mutant <italic> agb1-2 </italic> , and <span class="trait">drought</span> treated by withholding water from 3-week-old plants in pots for 3 weeks; 54.76% of WT plants survived for 5 d after re-watering, significantly lower than the 76.19% survival rate of <italic> agb1-2 </italic> plants (**P&#x0003c;0.01) ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1B and C </xref> ). Additionally, analysis showed that there were no differences in root length between wild-type (ecotype, Col-0) and <italic> agb1 </italic> mutants on <span class="gene">MS</span> medium, however, in wild-type seedlings, roots length was significantly shorter than that of <italic> agb1-1 </italic> and <italic> agb1-2 </italic> mutant lines when the <span class="gene">MS</span> medium contained 4% and 8% PEG (*P&#x0003c;0.05) ( <xref ref-type="supplementary-material" rid="pone.0116385.s003"> S3 Fig. A, B, C and E </xref> ). Physiological measurements showed that, under normal conditions, there was no significant difference in proline contents between WT and <italic> agb1-2 </italic> ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1D </xref> ). However, the proline contents in <italic> agb1-2 </italic> plants were significantly higher at 1 h (31% higher) and 3 h (47% higher) than those in WT under dehydration at the same time point (**P&#x0003c;0.01) ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1D </xref> ). Moreover, the expression levels of two key regulators of proline biosynthesis, <italic> AtProDH </italic> and <italic> AtP5CS2 </italic> , were increased in <italic> agb1-2 </italic> in comparison to WT plants after <span class="trait">drought</span> treatment ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1E and F </xref> ), leading to proline accumulation and enhancement of <span class="trait">drought</span> <span class="trait">resistance</span> in <italic> agb1-2 </italic> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						Real-time PCR was performed to analyze the expression of <italic> <span class="gene">RD29A</span> </italic> , a downstream gene of <italic> AtMYB44 </italic> [ <xref rid="pone.0116385.ref040" ref-type="bibr"> 40 </xref> ], and stress-responsive genes <italic> RAB18 </italic> and <italic> ERD10 </italic> [ <xref rid="pone.0116385.ref041" ref-type="bibr"> 41 </xref> , <xref rid="pone.0116385.ref042" ref-type="bibr"> 42 </xref> ] in <italic> agb1-2 </italic> and WT plants. When subjected to dehydration stress treatments, a set of stress-responsive genes, such as <italic> RD29A, RAB18 </italic> and <italic> ERD10 </italic> in WT and <italic> agb1-2 </italic> were induced. For example, the expression of <italic> <span class="gene">RD29A</span> </italic> in <italic> agb1-2 </italic> was higher than those in WT, reaching nearly 17-fold at 16 h after <span class="trait">drought</span> treatment compared with WT ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A </xref> ). Except at 12 h, the transcript levels of <italic> RAB18 </italic> and <italic> ERD10 </italic> in <italic> agb1-2 </italic> were higher than those in WT, and the transcript levels of <italic> RAB18 </italic> reached nearly 3-fold and that of <italic> ERD10 </italic> nearly 6.5-fold higher at 6 h in <italic> agb1-2 </italic> compared with WT ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A </xref> ). ABA-biosynthesis genes, such as <italic> ABA1 </italic> and <italic> ABA2 </italic> , were also up-regulated in <italic> agb1-2 </italic> after <span class="trait">drought</span> treatment, and the expression of <italic> ABA1 </italic> in <italic> agb1-2 </italic> reached nearly 5.2-fold compared with that of WT at 1 h ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5B </xref> ). These results indicated that <italic> AGB1 </italic> negatively regulated the expression of a subset of drought-responsive genes involved in ABA signaling pathway, including <italic> RD29A, RAB18, ERD10, ABA1 </italic> and <italic> ABA2 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						ABA functions to inhibit seed germination and seedling growth. Under normal conditions without ABA treatment, the <italic> 35S:AtVIP1 </italic> and WT lines showed similar germination rates (80&#x02013;88%) at 72 h. However, the germination rates of <italic> 35S:AtVIP1 </italic> lines were 40&#x02013;63% compared to 76% in WT at the same time point following treatment with 2.5 &#x003bc;M ABA ( <xref ref-type="fig" rid="pone.0116385.g006"> Fig. 6A, B and C </xref> ). The root lengths of WT and <italic> 35S:AtVIP1 </italic> seedlings were likewise similar under normal conditions, but those of WT were significantly longer than <italic> 35S:AtVIP1 </italic> seedlings when grown on medium supplemented with 5 &#x003bc;M ABA (*P&#x0003c;0.05) ( <xref ref-type="fig" rid="pone.0116385.g006"> Fig. 6D and E </xref> ). Germination rate of transgenic lines OE-VIP1&#x02013;4 (40%) and OE-VIP1&#x02013;7 (45%) were lower than that of line OE-VIP1&#x02013;5 (63%) ( <xref ref-type="fig" rid="pone.0116385.g006"> Fig. 6C </xref> ) after treatment with 2.5 &#x003bc;M ABA and the phenotypes of the transgenic lines were related to the expression levels of <italic> AtVIP1 </italic> ( <xref ref-type="fig" rid="pone.0116385.g006"> Fig. 6F </xref> ), indicating that the expression level of the <italic> AtVIP1 </italic> transgene was positively related with sensitivity to ABA treatment in transgenic plants during germination and early seedling growth. With regard to osmotic stress, there were no differences between WT and <italic> 35S:AtVIP1 </italic> lines on <span class="gene">MS</span> medium, but the root lengths of WT seedlings were significantly shorter than those of <italic> 35S:AtVIP1 </italic> lines OE-VIP1&#x02013;5 (P&#x0003c;0.05) and OE-VIP1&#x02013;7 (P&#x0003c;0.01) when grown on <span class="gene">MS</span> medium contained 4% PEG ( <xref ref-type="fig" rid="pone.0116385.g007"> Fig. 7A and B </xref> ). For plants grown in soil, <italic> 35S:AtVIP1 </italic> lines OE-VIP1&#x02013;5 (51.7% survival) and OE-VIP1&#x02013;7 (68.3% survival) retained better appearance and higher survival rates than WT (18.3% survival) at 3 d after re-watering following 21 d of water deprivation ( <xref ref-type="fig" rid="pone.0116385.g007"> Fig. 7C </xref> ), suggesting that overexpression of <italic> AtVIP1 </italic> conferred <span class="trait">drought</span> tolerance in transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The downstream effectors of G proteins and associated signaling pathways were not well-defined. AGB1 was previously considered to be a primary regulator of the G-protein complex in ABA signaling [ <xref rid="pone.0116385.ref007" ref-type="bibr"> 7 </xref> , <xref rid="pone.0116385.ref011" ref-type="bibr"> 11 </xref> ], whereas the regulatory mechanism of AGB1 in ABA responses in <italic> Arabidopsis </italic> remained unclear [ <xref rid="pone.0116385.ref011" ref-type="bibr"> 11 </xref> ]. Recently, although several AGB1 interaction proteins in <italic> Arabidopsis </italic> have been identified in the G-signaling Interactome Database (AGIdb, <ext-link ext-link-type="uri" xlink:href="http://bioinfolab.unl.edu/AGIdb"> http://bioinfolab.unl.edu/AGIdb </ext-link> ), few protein kinases have been shown to interact with AGB1 [ <xref rid="pone.0116385.ref043" ref-type="bibr"> 43 </xref> , <xref rid="pone.0116385.ref044" ref-type="bibr"> 44 </xref> ]. In the present study, we used Y2H, Pull-down ( <italic> in vitro </italic> ), BiFC and Co-IP assay ( <italic> in vivo </italic> ) to demonstrate that AGB1 interacts with AtMPK6, a key protein kinase involved in stress responses ( <xref ref-type="fig" rid="pone.0116385.g002"> Fig. 2 </xref> ). These results established the link between G-proteins and the MAP-like kinase cascade. Protein kinases are major components in cellular signal transduction pathways, and mediate responses to various biotic and abiotic stresses, including hormone signaling pathways [ <xref rid="pone.0116385.ref045" ref-type="bibr"> 45 </xref> ]. For example, <italic> AtMPK6 </italic> was transiently activated after ABA application and overexpression of <italic> AtMPK6 </italic> enhanced ABA response during germination. However, the <italic> atmpk6 </italic> mutant was insensitive to ABA during germination and under <span class="trait">drought</span> stress condition [ <xref rid="pone.0116385.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0116385.ref046" ref-type="bibr"> 46 </xref> , <xref rid="pone.0116385.ref047" ref-type="bibr"> 47 </xref> ], suggesting that the interaction between MPK6 and AGB1 may play a role in ABA signaling. In addition, both AtVIP1 and AtMYB44 can be phosphorylated by AtMPK6 [ <xref rid="pone.0116385.ref037" ref-type="bibr"> 37 </xref> , <xref rid="pone.0116385.ref038" ref-type="bibr"> 38 </xref> ]. <italic> AtMYB44 </italic> is regulated by <italic> AtVIP1 </italic> [ <xref rid="pone.0116385.ref039" ref-type="bibr"> 39 </xref> ], and the dehydration-responsive gene, <italic> <span class="gene">RD29A</span> </italic> , is a downstream target gene of <italic> AtMYB44 </italic> [ <xref rid="pone.0116385.ref040" ref-type="bibr"> 40 </xref> ]. Furthermore, overexpression of <italic> AtMYB44 </italic> also enhanced ABA sensitivity and <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> [ <xref rid="pone.0116385.ref040" ref-type="bibr"> 40 </xref> ]. The evidence suggested that a signaling cascade that involves <italic> AtMPK6, AtVIP1, AtMYB44 </italic> , and <italic> <span class="gene">RD29A</span> </italic> might be involved in the ABA signaling pathway in <italic> Arabidopsis </italic> . In this report, the <italic> agb1 </italic> mutants showed hypersensitivity to ABA ( <xref ref-type="supplementary-material" rid="pone.0116385.s002"> S2 Fig. </xref> ), and enhanced <span class="trait">drought</span> tolerance ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1B and C </xref> ). On the other hand, the target genes involved in this pathway were up-regulated in <italic> agb1-2 </italic> under ABA or <span class="trait">drought</span> treatments ( <xref ref-type="fig" rid="pone.0116385.g004"> Fig. 4 </xref> , <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A </xref> and <xref ref-type="supplementary-material" rid="pone.0116385.s004"> S4 Fig. </xref> ). Overexpression of <italic> AtVIP1 </italic> enhanced the sensitivity of transgenic plants to ABA treatment ( <xref ref-type="fig" rid="pone.0116385.g006"> Fig. 6A-E </xref> ). These results indicating that a novel role of AGB1 may be involved in transcriptional regulation through the <italic> AtMPK6, AtVIP1, AtMYB44 </italic> , and <italic> <span class="gene">RD29A</span> </italic> cascade in <italic> Arabidopsis </italic> in response to ABA or dehydration.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we found that, except for <italic> <span class="gene">RD29A</span> </italic> , some stress-responsive genes such as <italic> RAB18 </italic> and <italic> ERD10 </italic> ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A </xref> ), ABA-biosynthesis genes containing <italic> ABA1 </italic> and <italic> ABA2 </italic> ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5B </xref> ), and proline-biosynthesis genes including <italic> AtProDH </italic> and <italic> AtP5CS2 </italic> ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1E and F </xref> ) in <italic> agb1-2 </italic> were significantly up-regulated after <span class="trait">drought</span> treatment, suggesting that in addition to the <italic> AtMPK6, AtVIP1, AtMYB44 </italic> , and <italic> <span class="gene">RD29A</span> </italic> cascade, other factors might also be involved in ABA response and <span class="trait">drought</span> tolerance of plants regulated by AGB1. It was reported that G&#x003b2; interacts with MAPK-like protein, PsMPK3, and responds to ABA treatment in <italic> Pisum sativum </italic> [ <xref rid="pone.0116385.ref043" ref-type="bibr"> 43 </xref> ]. As PsMPK3 is highly homologous with AtMPK3 and AtMPK6, two MAPKs that are key nodal and functionally redundant factors in the ABA signaling pathway [ <xref rid="pone.0116385.ref055" ref-type="bibr"> 55 </xref> ], we hypothesized that AGB1 acts as a molecular switch in regulating the ABA signaling pathway in plants by interacting simultaneously with AtMPK3 and AtMPK6. However, further studies are necessary to confirm this hypothesis.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AGB1 </italic> not only affects the adaptability of plants to the environment, but also alters seed weight under water deficit [ <xref rid="pone.0116385.ref056" ref-type="bibr"> 56 </xref> ]. However, the physiological and molecular mechanisms involved in the regulation of <span class="trait">drought</span> response are not completely understood. In this report, <italic> AGB1 </italic> was demonstrated to negatively affect <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1B, C </xref> and <xref ref-type="supplementary-material" rid="pone.0116385.s003"> S3E Fig. </xref> ). Stress-responsive genes <italic> RAB18 </italic> and <italic> ERD10 </italic> ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A </xref> ), <italic> ABA1 </italic> and <italic> ABA2 </italic> ( <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5B </xref> ), and <italic> AtProDH </italic> and <italic> AtP5CS2 </italic> ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1E and 1F </xref> ) in <italic> agb1-2 </italic> were significantly up-regulated after <span class="trait">drought,</span> along with proline accumulation in the <italic> agb1-2 </italic> mutant ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1D </xref> ). It has been reported that overexpression of <italic> AtMPK6 </italic> increased expression of the ABA biosynthesis genes <italic> NCED3 </italic> and <italic> ABA2 </italic> [ <xref rid="pone.0116385.ref047" ref-type="bibr"> 47 </xref> ], and overexpression of stress-responsive genes, including <italic> <span class="gene">RD29A</span> </italic> and <italic> RAB18 </italic> , which resulted in enhanced dehydration tolerance in transgenic plants [ <xref rid="pone.0116385.ref057" ref-type="bibr"> 57 </xref> , <xref rid="pone.0116385.ref058" ref-type="bibr"> 58 </xref> ]. Accumulation of proline is an important indicator in determining salt/drought tolerance, and overproduction of proline led to increased tolerance against abiotic stress in transgenic tobacco plants [ <xref rid="pone.0116385.ref059" ref-type="bibr"> 59 </xref> ]. In this study, overexpression of <italic> AtVIP1 </italic> enhanced <span class="trait">drought</span> tolerance of transgenic plants ( <xref ref-type="fig" rid="pone.0116385.g007"> Fig. 7A, B and C </xref> ), indicating that inducible expression of stress-responsive genes contributed to enhanced <span class="trait">drought</span> tolerance in <italic> agb1-2 </italic> . Moreover, we detected some cis-acting elements associated with abiotic stress responses, such as DRE <span class="trait">(drought</span> response element), LTRE (low temperature response element) and ABRE (ABA response element) in the promoter region of <italic> AGB1 </italic> using the database PLACE Signal Scan Search ( <xref ref-type="supplementary-material" rid="pone.0116385.s005"> S1 Table </xref> ) [ <xref rid="pone.0116385.ref060" ref-type="bibr"> 60 </xref> ], suggesting that <italic> AGB1 </italic> is indeed involved in <span class="trait">drought</span> responses in plants. Interestingly, <italic> AGB1 </italic> expression was down-regulated by ABA and <span class="trait">drought</span> treatment ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1A </xref> ). Therefore, it was hypothesized that <italic> AGB1 </italic> negatively regulated expression of some ABA response and <span class="trait">drought</span> tolerance genes in wild type plants under normal conditions [ <xref rid="pone.0116385.ref040" ref-type="bibr"> 40 </xref> , <xref rid="pone.0116385.ref061" ref-type="bibr"> 61 </xref> ]. Whereas under stress conditions, expression of <italic> AGB1 </italic> was inhibited ( <xref ref-type="fig" rid="pone.0116385.g001"> Fig. 1A </xref> ), thereby expression of ABA response genes and <span class="trait">drought</span> tolerance genes were activated ( <xref ref-type="fig" rid="pone.0116385.g004"> Fig. 4A, B </xref> and <xref ref-type="fig" rid="pone.0116385.g005"> Fig. 5A, B </xref> ), thus contributing to enhanced stress tolerance and better survival under <span class="trait">drought</span> conditions. In brief, AGB1 interacts with AtMPK6 ( <xref ref-type="fig" rid="pone.0116385.g002"> Fig. 2A-D </xref> ), and regulates ABA response and <span class="trait">drought</span> tolerance by down-regulating the ABA responsive transcriptional factor <italic> AtMPK6 </italic> in <italic> Arabidopsis </italic> ( <xref ref-type="fig" rid="pone.0116385.g008"> Fig. 8 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and functional characterization of the sulfate transporter gene <span class="gene">GmSULTR1;2b</span> in soybean.</h2>
			<h3>PMID:27206527  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ding Y(1), Zhou X(1), Zuo L(1), Wang H(2), Yu D(3).
				<br/> 
				年份：2016 May 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is a major source of oil and protein in the human diet and in animal feed. However, as soybean is deficient in sulfur-containing amino acids, its nutritional value is limited. Increasing sulfate assimilation and utilization efficiency is a valuable approach to augment the concentration of sulfur-containing amino acids in soybean seeds, and sulfate transporters play important roles in both sulfate uptake and translocation within plants. RESULTS: In this study, we isolated and characterized a soybean sulfate transporter gene: <span class="gene">GmSULTR1;2b.</span> The gene was found to be specifically expressed in root tissues and induced by low-sulfur stress. In addition, <span class="gene">GmSULTR1;2b</span> expression in yeast could complement deficiency in the sulfate transporter genes SUL1 and SUL2. Under +S conditions, <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco plants accumulated higher levels of organic matter and exhibited enhanced biomass and seed weight compared to control plants. Under -S conditions, acclimation of <span class="gene">GmSULTR1;2b-overexpressing</span> plants was much better than control plants. <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, but no significant difference was observed under salt stress. Transcriptome analysis revealed 515 genes with at least a 2-fold change in expression in the leaves of tobacco plants overexpressing <span class="gene">GmSULTR1;2b.</span> Of these, 227 gene annotations were classified into 12 functional categories associated with 123 relevant pathways, including biosynthesis and metabolism-related proteins, stress-related proteins, and transporters. CONCLUSIONS: The findings reported here indicate that the increased biomass and seed <span class="trait">yield</span> observed in transgenic tobacco plants could have resulted from greater nutrient uptake and transport capability as well as enhanced development and accumulation of organic matter. Taken together, our results indicate that <span class="gene">GmSULTR1;2b</span> plays an important role in sulfur uptake and could alter the sulfur status of plants. Our study suggests that overexpressing <span class="gene">GmSULTR1;2b</span> may enhance plant <span class="trait">yield</span> under +S conditions, reduce plant production loss under -S conditions, and improve plant tolerance to sulfur deficiency stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genomic characterization of the S-adenosylmethionine decarboxylase genes from soybean.</h2>
			<h3>PMID:14618239  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tian AG(1), Zhao JY, Zhang JS, Gai JY, Chen SY.
				<br/> 
				年份：2004 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length gene <span class="gene">GmSAMDC1,</span> encoding the S-adenosylmethionine decarboxylase (SAMDC), a key enzyme involved in polyamine biosynthesis, was identified from soybean expressed sequence tags and was characterized. <span class="gene">GmSAMDC1</span> encoded a peptide of 355 amino acids. When compared with other plant SAMDCs, the <span class="gene">GmSAMDC1</span> protein had several highly conserved regions including a putative pro-enzyme cleavage site and a PEST sequence. The 5' leader sequence of the the <span class="gene">GmSAMDC1</span> mRNA contained two additional open reading frames (ORFs), which may regulate the translational process. The genomic sequence of the <span class="gene">GmSAMDC1</span> gene contained three introns in the 5' leader sequence, but no intron in the 3'-UTR or the main pro-enzyme ORF. A simple sequence repeat (SSR) was found in intron 2, and the <span class="gene">GmSAMDC1</span> gene was mapped to linkage group D1 using this SSR. The genomic organization of the <span class="gene">GmSAMDC1</span> gene in the subgenus Glycine and the subgenus Soja was found to be different by Southern-blot and PCR analysis. A pseudogene, <span class="gene">GmSAMDC2,</span> was also identified. This gene contained no intron and lost its two uORFs. Northern-blot analysis showed that the <span class="gene">GmSAMDC1</span> gene expression was induced by salt, <span class="trait">drought</span> and cold, but not induced by wounding; suggesting that the gene was implicated in response to multiple-stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsSNAP33, a novel Glycine soja SNAP25-type protein gene: Improvement of plant salt and <span class="trait">drought</span> tolerances in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:28841544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nisa ZU(1), Mallano AI(2), Yu Y(3), Chen C(4), Duan X(5), Amanullah S(6), Kousar A(7), Baloch AW(8), Sun X(9), Tabys D(10), Zhu Y(11).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The N-ethylmaleimide-sensitive factor attachment protein receptor (SNARE) superfamily, specifically the SNAP25-type proteins and t-SNAREs, have been proposed to regulate cellular processes and plant <span class="trait">resistance</span> mechanisms. However, little is known about the role of SNAP25-type proteins in combating abiotic stresses, specifically in wild soybean. In the current study, the isolation and functional characterization of the putative synaptosomal-associated SNAP25-type protein gene GsSNAP33 from wild soybean (Glycine soja) were performed. GsSNAP33 has a molecular weight of 33,311 Da and comprises 300 amino acid residues along with Qb-Qc SNARE domains. Multiple sequence alignment revealed the highest similarity of the GsSNAP33 protein to <span class="gene">GmSNAP33</span> (91%), VrSNAP33 (89%), PvSNAP33 (86%) and AtSNAP33 (63%). Phylogenetic studies revealed the abundance of SNAP33 proteins mostly in dicotyledons. Quantitative real-time PCR assays confirmed that GsSNAP33 expression can be induced by salt, alkali, ABA and PEG treatments and that GsSNAP33 is highly expressed in the pods, seeds and roots of Glycine soja. Furthermore, the overexpression of the GsSNAP33 gene in WT Arabidopsis thaliana resulted in increased germination rates, greater root lengths, improved photosynthesis, lower electrolyte leakage, higher biomass production and up-regulated expression levels of various stress-responsive marker genes, including KINI, COR15A, P5Cs, RAB18, <span class="gene">RD29A</span> and COR47 in transgenic lines compared with those in WT lines. Subcellular localization studies revealed that the GsSNAP33-eGFP fusion protein was localized to the plasma membrane, while eGFP was distributed throughout whole cytoplasm of onion epidermal cells. Collectively, our findings suggest that GsSNAP33, a novel plasma membrane protein gene of Glycine soja, might be involved in improving plant responses to salt and <span class="trait">drought</span> stresses in Arabidopsis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The auxin response factor transcription factor family in soybean: genome-wide identification and expression analyses during development and water stress.</h2>
			<h3>PMID:23810914  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ha CV(1), Le DT, Nishiyama R, Watanabe Y, Sulieman S, Tran UT, Mochida K, Dong NV, Yamaguchi-Shinozaki K, Shinozaki K, Tran LS.
				<br/> 
				年份：2013 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In plants, the auxin response factor (ARF) transcription factors play important roles in regulating diverse biological processes, including development, growth, cell division and responses to environmental stimuli. An exhaustive search of soybean genome revealed 51 <span class="gene">GmARFs,</span> many of which were formed by genome duplications. The typical <span class="gene">GmARFs</span> (43 members) contain a DNA-binding domain, an ARF domain and an auxin/indole acetic acid (AUX/IAA) dimerization domain, whereas the remaining eight members lack the dimerization domain. Phylogenetic analysis of the ARFs from soybean and Arabidopsis revealed both similarity and divergence between the two ARF families, as well as enabled us to predict the functions of the <span class="gene">GmARFs.</span> Using quantitative real-time polymerase chain reaction (qRT-PCR) and available soybean Affymetrix array and Illumina transcriptome sequence data, a comprehensive expression atlas of <span class="gene">GmARF</span> genes was obtained in various organs and tissues, providing useful information about their involvement in defining the precise nature of individual tissues. Furthermore, expression profiling using qRT-PCR and microarray data revealed many water stress-responsive <span class="gene">GmARFs</span> in soybean, albeit with different patterns depending on types of tissues and/or developmental stages. Our systematic analysis has identified excellent tissue-specific and/or stress-responsive candidate <span class="gene">GmARF</span> genes for in-depth in planta functional analyses, which would lead to potential applications in the development of genetically modified soybean cultivars with enhanced <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide characterization of the aldehyde dehydrogenase gene superfamily in soybean and its potential role in <span class="trait">drought</span> stress response</h2>
			<h3>PMID:28687067</h3>
				

				
					<h3>PMCID:5501352</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Wei </given-names>  </name>  <address>  <email> 543434717@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Wei </given-names>  </name>  <address>  <email> jiangwei19901006@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Juge </given-names>  </name>  <address>  <email> luoyeyanhua@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yang </given-names>  </name>  <address>  <email> 327086855@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <address>  <email> sri@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Yan </given-names>  </name>  <address>  <phone> +86-25-84399519 </phone>  <email> yanli1@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9750 7019 </institution-id>  <institution-id institution-id-type="GRID"> grid.27871.3b </institution-id>  <institution> National Key Laboratory of Crop Genetics and Germplasm Enhancement / National Center for Soybean Improvement / Key Laboratory for Biology and Genetic Improvement of Soybean (General, Ministry of Agriculture) / Jiangsu Collaborative Innovation Center for Modern Crop Production,  </institution>  <institution> Nanjing Agricultural University,  </institution>  </institution-wrap> Nanjing, Jiangsu China  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Aldehyde dehydrogenases (ALDHs) represent a group of enzymes that detoxify aldehydes by facilitating their oxidation to carboxylic acids, and have been shown to play roles in plant response to abiotic stresses. However, the comprehensive analysis of ALDH superfamily in soybean (Glycine max) has been limited. RESULTS: In present study, a total of 53 <span class="gene">GmALDHs</span> were identified in soybean, and grouped into 10 ALDH families according to the ALDH Gene Nomenclature Committee and phylogenetic analysis. These groupings were supported by their gene structures and conserved motifs. Soybean ALDH superfamily expanded mainly by whole genome duplication/segmental duplications. Gene network analysis identified 1146 putative co-functional genes of 51 <span class="gene">GmALDHs.</span> Gene Ontology (GO) enrichment analysis suggested the co-functional genes of these 51 <span class="gene">GmALDHs</span> were enriched (FDR &lt; 1e-3) in the process of lipid metabolism, photosynthesis, proline catabolism, and small molecule catabolism. In addition, 22 co-functional genes of <span class="gene">GmALDHs</span> are related to plant response to water deprivation/water transport. <span class="gene">GmALDHs</span> exhibited various expression patterns in different soybean tissues. The expression levels of 13 <span class="gene">GmALDHs</span> were significantly up-regulated and 14 down-regulated in response to water deficit. The occurrence frequencies of three drought-responsive cis-elements (ABRE, CRT/DRE, and GTGCnTGC/G) were compared in <span class="gene">GmALDH</span> genes that were up-, down-, or non-regulated by water deficit. Higher frequency of these three cis-elements was observed for the group of up-regulated <span class="gene">GmALDH</span> genes as compared to the group of down- or non- regulated <span class="gene">GmALDHs</span> by <span class="trait">drought</span> stress, implying their potential roles in the regulation of soybean response to <span class="trait">drought</span> stress. CONCLUSIONS: A total of 53 ALDH genes were identified in soybean genome and their phylogenetic relationships and duplication patterns were analyzed. The potential functions of <span class="gene">GmALDHs</span> were predicted by analyses of their co-functional gene networks, gene expression profiles, and cis-regulatory elements. Three <span class="gene">GmALDH</span> genes, including <span class="gene">GmALDH3H2,</span> <span class="gene">GmALDH12A2</span> and <span class="gene">GmALDH18B3,</span> were highly induced by <span class="trait">drought</span> stress in soybean leaves. Our study provides a foundation for future investigations of <span class="gene">GmALDH</span> gene function in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Heap map of the expression profiles of <italic> <span class="gene">GmALDHs</span> </italic> in soybean plants subjected to <span class="trait">drought</span> stress for 0, 6, 12 and 24&#x000a0;h. The color scale is based on the log10 (FPKM&#x000a0;+&#x000a0;1) values. FPKM values of 53 <italic> <span class="gene">GmALDHs</span> </italic> were obtained from the RNA-seq data using leaf samples of the drought-tolerant soybean cultivar, PI416937. The heat map with hierarchical clustering of <italic> <span class="gene">GmALDH</span> </italic> genes was constructed using MeV 4.9 software by average linage with Euclidean distance
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of the <italic> cis </italic> -elements in the promoter regions of soybean <italic> ALDH </italic> genes. <bold> a </bold> . Predicted <italic> cis </italic> -elements in the 1500&#x000a0;bp upstream regions of six <italic> <span class="gene">GmALDH</span> </italic> genes. The names of the <italic> cis </italic> -elements are indicated at the bottom of the figure. The scale indicates the relative position of each <italic> cis </italic> -element relative to the translation start codon, ATG. <bold> b </bold> . Frequency of three drought-responsive <italic> cis </italic> -elements present in the promoter regions of different groups of soybean <italic> ALDHs </italic> . The frequencies of ABRE, CRT/DRE, and GTGCnTGC/G <italic> cis </italic> -elements in up-regulated, down-regulated and non-responsive <italic> <span class="gene">GmALDH</span> </italic> genes under <span class="trait">drought</span> stress are shown in different colored bars. Changes in expression were determined by the criteria of <italic> P </italic> value &#x0003c;&#x000a0;0.05 and &#x02502;log <sub> 2 </sub> fold change&#x02502;&#x000a0;&#x02265;&#x000a0;2 comparing drought-stressed soybean plants with control
					</p>
				
					<p>&nbsp;&nbsp;
						FPKM values of <italic> <span class="gene">GmALDH</span> </italic> genes in the drought-tolerant soybean variety PI416937 under control and <span class="trait">drought</span> stress conditions. (XLSX 14 kb)
					</p>
				
					<p>&nbsp;&nbsp;
						Fold change and <italic> p </italic> -value of <italic> <span class="gene">GmALDH</span> </italic> genes in the drought-tolerant soybean variety PI416937 under <span class="trait">drought</span> stress compared with control. (XLSX 13 kb)
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of the soybean L-asparaginase gene induced by low temperature stress.</h2>
			<h3>PMID:17646702  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cho CW(1), Lee HJ, Chung E, Kim KM, Heo JE, Kim JI, Chung J, Ma Y, Fukui K, Lee DW, Kim DH, Chung YS, Lee JH.
				<br/> 
				年份：2007 Jun 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						L-asparaginase (EC 3.5.1.1) catalyzes the hydrolysis of the amide group of L-asparagine, releasing aspartate and NH4+. We isolated a low temperature-inducible cDNA sequence encoding L-asparaginase from soybean leaves. The full-length L-asparaginase cDNA, designated <span class="gene">GmASP1,</span> contains an open reading frame of 1,258 bp coding for a protein of 326 amino acids. Genomic DNA blotting and fluorescence in situ hybridization showed that the soybean genome has two copies of <span class="gene">GmASP1.</span> <span class="gene">GmASP1</span> mRNA was induced by low temperature, ABA and NaCl, but not by heat shock or <span class="trait">drought</span> stress. E. coli cells expressing recombinant <span class="gene">GmASP1</span> had 3-fold increased L-asparaginase activity. A possible function of L-asparaginase in the early response to low temperature stress is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsZFP1, a new Cys2/His2-type zinc-finger protein, is a positive regulator of plant tolerance to cold and <span class="trait">drought</span> stress.</h2>
			<h3>PMID:22160567  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Luo X(1), Bai X, Zhu D, Li Y, Ji W, Cai H, Wu J, Liu B, Zhu Y.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant acclimation to environmental stress is controlled by a complex network of regulatory genes that compose distinct stress-response regulons. The C2H2-type zinc-finger proteins (ZFPs) have been implicated in different cellular processes involved in plant development and stress responses. Through microarray analysis, an alkaline (NaHCO(3))-responsive ZFP gene GsZFP1 was identified and subsequently cloned from Glyycine soja. GsZFP1 encodes a 35.14 kDa protein with one C2H2-type zinc-finger motif. The QALGGH domain, conserved in most plant C2H2-type ZFPs, is absent in the GsZFP1 protein sequence. A subcellular localization study using a GFP fusion protein indicated that GsZFP1 is localized to the nucleus. Real-time RT-PCR analysis showed that GsZFP1 was induced in the leaf by ABA (100 μM), salt (200 mM NaCl), and cold (4°C), and in the root by ABA (100 μM), cold (4°C), and <span class="trait">drought</span> (30% PEG 6000). Over-expression of GsZFP1 in transgenic Arabidopsis resulted in a greater tolerance to cold and <span class="trait">drought</span> stress, a decreased water loss rate, and an increase in proline irrespective of environmental conditions. The over-expression of GsZFP1 also increased the expression of a number of stress-response marker genes, including CBF1, CBF2, CBF3, NCED3, COR47, and <span class="gene">RD29A</span> in response to cold stress and RAB18, NCED3, P5CS, RD22, and <span class="gene">RD29A</span> in response to <span class="trait">drought</span> stress, especially early during stress treatments. Our studies suggest that GsZFP1 plays a crucial role in the plant response to cold and <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A toolbox of genes, proteins, metabolites and promoters for improving <span class="trait">drought</span> tolerance in soybean includes the metabolite coumestrol and stomatal development genes.</h2>
			<h3>PMID:26861168  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tripathi P(1)(2), Rabara RC(3)(4), Reese RN(5), Miller MA(6), Rohila JS(7), Subramanian S(8), Shen QJ(9), Morandi D(10), Bücking H(11), Shulaev V(12), Rushton PJ(13)(14).
				<br/> 
				年份：2016 Feb 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The purpose of this project was to identify metabolites, proteins, genes, and promoters associated with water stress responses in soybean. A number of these may serve as new targets for the biotechnological improvement of <span class="trait">drought</span> responses in soybean (Glycine max). RESULTS: We identified metabolites, proteins, and genes that are strongly up or down regulated during rapid water stress following removal from a hydroponics system. 163 metabolites showed significant changes during water stress in roots and 93 in leaves. The largest change was a root-specific 160-fold increase in the coumestan coumestrol making it a potential biomarker for <span class="trait">drought</span> and a promising target for improving <span class="trait">drought</span> responses. Previous reports suggest that coumestrol stimulates mycorrhizal colonization and under certain conditions mycorrhizal plants have improved <span class="trait">drought</span> tolerance. This suggests that coumestrol may be part of a call for help to the rhizobiome during stress. About 3,000 genes were strongly up-regulated by <span class="trait">drought</span> and we identified regulators such as ERF, MYB, NAC, bHLH, and WRKY transcription factors, receptor-like kinases, and calcium signaling components as potential targets for soybean improvement as well as the jasmonate and abscisic acid biosynthetic genes JMT, LOX1, and ABA1. <span class="trait">Drought</span> stressed soybean leaves show reduced mRNA levels of stomatal development genes including FAMA-like, MUTE-like and SPEECHLESS-like bHLH transcription factors and leaves formed after <span class="trait">drought</span> stress had a reduction in stomatal density of 22.34 % and stomatal index of 17.56 %. This suggests that reducing stomatal density may improve <span class="trait">drought</span> tolerance. MEME analyses suggest that ABRE (CACGT/CG), CRT/DRE (CCGAC) and a novel GTGCnTGC/G element play roles in transcriptional activation and these could form components of synthetic promoters to drive expression of transgenes. Using transformed hairy roots, we validated the increase in promoter activity of <span class="gene">GmWRKY17</span> and <span class="gene">GmWRKY67</span> during dehydration and after 20 μM ABA treatment. CONCLUSIONS: Our toolbox provides new targets and strategies for improving soybean <span class="trait">drought</span> tolerance and includes the coumestan coumestrol, transcription factors that regulate stomatal density, water stress-responsive WRKY gene promoters and a novel DNA element that appears to be enriched in water stress responsive promoters.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of wild soybean <span class="gene">WRKY20</span> in Arabidopsis enhances <span class="trait">drought</span> tolerance and regulates ABA signalling.</h2>
			<h3>PMID:23606412  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Luo X(1), Bai X, Sun X, Zhu D, Liu B, Ji W, Cai H, Cao L, Wu J, Hu M, Liu X, Tang L, Zhu Y.
				<br/> 
				年份：2013 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The WRKY-type transcription factors are involved in plant development and stress responses, but how the regulation of stress tolerance is related to plant development is largely unknown. GsWRKY20 was initially identified as a stress response gene using large-scale Glycine soja microarrays. Quantitative reverse transcription-PCR (qRT-PCR) showed that the expression of this gene was induced by abscisic acid (ABA), salt, cold, and <span class="trait">drought.</span> Overexpression of GsWRKY20 in Arabidopsis resulted in a decreased sensitivity to ABA during seed germination and early seedling growth. However, compared with the wild type, GsWRKY20 overexpression lines were more sensitive to ABA in stomatal closure, and exhibited a greater tolerance to <span class="trait">drought</span> stress, a decreased water loss rate, and a decreased stomatal density. Moreover, microarray and qRT-PCR assays showed that GsWRKY20 mediated ABA signalling by promoting the expression of negative regulators of ABA signalling, such as AtWRKY40, ABI1, and ABI2, while repressing the expression of the positive regulators of ABA, for example ABI5, ABI4, and ABF4. Interestingly, GsWRKY20 also positively regulates the expression of a group of wax biosynthetic genes. Further, evidence is provided to support that GsWRKY20 overexpression lines have more epicuticular wax crystals and a much thicker cuticle, which contribute to less chlorophyll leaching compared with the wild type. Taken together, the findings reveal an important role for GsWRKY20 in enhancing <span class="trait">drought</span> tolerance and regulating ABA signalling.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmWRKY53, a water- and salt-inducible soybean gene for rapid dissection of regulatory elements in BY-2 cell culture.</h2>
			<h3>PMID:23511199  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tripathi P(1), Rabara RC, Lin J, Rushton PJ.
				<br/> 
				年份：2013 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is the major cause of crop losses worldwide. Water stress-inducible promoters are important for understanding the mechanisms of water stress responses in crop plants. Here we utilized tobacco (Nicotiana tabacum L.) Bright Yellow 2 (BY-2) cell system in presence of polyethylene glycol, salt and phytohormones. Extension of the system to 85 mM NaCl led to inducibility of up to 10-fold with the water stress and salt responsive soybean <span class="gene">GmWRKY53</span> promoter. Upon ABA and JA treatment fold inducibility was up to 5-fold and 14-fold, respectively. Thus, we hypothesize that <span class="gene">GmWRKY53</span> could be used as potential model candidate for dissecting <span class="trait">drought</span> regulatory elements as well as understanding crosstalk utilizing a rapid heterologous system of BY-2 culture.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The differential expression of soybean [Glycine max (L.) Merrill] WRKY genes in response to water deficit.</h2>
			<h3>PMID:27343875  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dias LP(1), de Oliveira-Busatto LA(2), Bodanese-Zanettini MH(3).
				<br/> 
				年份：2016 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is today, and perhaps even more in the future, the main challenge for grain crops, resulting in a drastic <span class="trait">yield</span> reduction. Thus, it is of great interest to obtain soybean genotypes tolerant to water deficit. The <span class="trait">drought</span> tolerance trait is difficult to obtain through classical breeding due to its polygenic basis. In this context, genetic engineering is presented as a way to achieve this attribute. The ability to modulate the expression of many genes placed the transcription factors as promising biotechnological targets to develop stress tolerant cultivars. The WRKY proteins form a large family of transcription factors that are involved in important physiological and biochemical processes in plants, including the response to water deficit. In this study, the expression pattern determined by qPCR showed that, <span class="gene">GmWRKY6,</span> <span class="gene">GmWRKY46,</span> <span class="gene">GmWRKY56,</span> <span class="gene">GmWRKY106</span> and <span class="gene">GmWRKY149</span> genes are differentially expressed between a <span class="trait">drought</span> tolerant and a susceptible soybean genotype in water stress conditions. The in silico promoter and coexpression analysis indicate that these genes act in a stress physiological background.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Core clock, SUB1, and ABAR genes mediate flooding and <span class="trait">drought</span> responses via alternative splicing in soybean.</h2>
			<h3>PMID:26314767  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Syed NH(1), Prince SJ(2), Mutava RN(2), Patil G(2), Li S, Chen W(2), Babu V(2), Joshi T(3), Khan S(4), Nguyen HT(5).
				<br/> 
				年份：2015 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Circadian clocks are a great evolutionary innovation and provide competitive advantage during the day/night cycle and under changing environmental conditions. The circadian clock mediates expression of a large proportion of genes in plants, achieving a harmonious relationship between energy metabolism, photosynthesis, and biotic and abiotic stress responses. Here it is shown that multiple paralogues of clock genes are present in soybean (Glycine max) and mediate flooding and <span class="trait">drought</span> responses. Differential expression of many clock and <span class="gene">SUB1</span> genes was found under flooding and <span class="trait">drought</span> conditions. Furthermore, natural variation in the amplitude and phase shifts in PRR7 and <span class="gene">TOC1</span> genes was also discovered under <span class="trait">drought</span> and flooding conditions, respectively. PRR3 exhibited flooding- and drought-specific splicing patterns and may work in concert with PRR7 and <span class="gene">TOC1</span> to achieve energy homeostasis under flooding and <span class="trait">drought</span> conditions. Higher expression of <span class="gene">TOC1</span> also coincides with elevated levels of abscisic acid (ABA) and variation in glucose levels in the morning and afternoon, indicating that this response to abiotic stress is mediated by ABA, endogenous sugar levels, and the circadian clock to fine-tune photosynthesis and energy utilization under stress conditions. It is proposed that the presence of multiple clock gene paralogues with variation in DNA sequence, phase, and period could be used to screen exotic germplasm to find sources for <span class="trait">drought</span> and flooding tolerance. Furthermore, fine tuning of multiple clock gene paralogues (via a genetic engineering approach) should also facilitate the development of flooding- and drought-tolerant soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Arabidopsis and tobacco plants ectopically expressing the soybean antiquitin-like ALDH7 gene display enhanced tolerance to <span class="trait">drought,</span> salinity, and oxidative stress.</h2>
			<h3>PMID:16595581  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rodrigues SM(1), Andrade MO, Gomes AP, Damatta FM, Baracat-Pereira MC, Fontes EP.
				<br/> 
				年份：2006
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Despite extensive studies in eukaryotic aldehyde dehydrogenases, functional information about the ALDH7 antiquitin-like proteins is lacking. A soybean antiquitin homologue gene, designated <span class="gene">GmTP55,</span> has been isolated which encodes a dehydrogenase motif-containing 55 kDa protein induced by dehydration and salt stress. <span class="gene">GmTP55</span> is closely related to the stress-induced plant antiquitin-like proteins that belong to the ALDH7 family. Transgenic tobacco (Nicotiana tabacum) and Arabidopsis (Arabidopsis thaliana) plants constitutively expressing <span class="gene">GmTP55</span> have been obtained in order to examine the physiological role of this enzyme under a variety of stress conditions. Ectopic expression of <span class="gene">GmTP55</span> in both Arabidopsis and tobacco conferred tolerance to salinity during germination and to water deficit during plant growth. Under salt stress, the germination efficiency of both transgenic tobacco and Arabidopsis seeds was significantly higher than that of their control counterparts. Likewise, under progressive <span class="trait">drought,</span> the transgenic tobacco lines apparently kept the shoot turgidity to a normal level, which contrasted with the leaf wilt phenotype of control plants. The transgenic plants also exhibited an enhanced tolerance to H(2)O(2)- and paraquat-induced oxidative stress. Both <span class="gene">GmTP55-expressing</span> Arabidopsis and tobacco seeds germinated efficiently in medium supplemented with H(2)O(2), whereas the germination of control seeds was drastically impaired. Similarly, transgenic tobacco leaf discs treated with paraquat displayed a significant reduction in the necrotic lesions as compared with control leaves. These transgenic lines also exhibited a lower concentration of lipid peroxidation-derived reactive aldehydes under oxidative stress. These results suggest that antiquitin may be involved in adaptive responses mediated by a physiologically relevant detoxification pathway in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification of Major Intrinsic Proteins in Glycine soja and characterization of <span class="gene">GmTIP2;1</span> function under salt and water stress.</h2>
			<h3>PMID:28646139</h3>
				

				
			<p>
				作者：Zhang DY(1), Kumar M(2)(3), Xu L(1), Wan Q(1), Huang YH(1), Xu ZL(1), He XL(1), Ma JB(4), Pandey GK(5), Shao HB(6)(7).
				<br/> 
				年份：2017 Jun 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In different plant species, aquaporins (AQPs) facilitate water movement by regulating root hydraulic conductivity under diverse stress conditions such as salt and water stresses. To improve survival and <span class="trait">yield</span> of crop plants, a detailed understanding of stress responses is imperative and required. We used Glycine soja genome as a tool to study AQPs, considering it shows abundant genetic diversity and higher salt environment tolerance features and identified 62 Gs AQP genes. Additionally, this study identifies major aquaporins responsive to salt and <span class="trait">drought</span> stresses in soybean and elucidates their mode of action through yeast two-hybrid assay and BiFC. Under stress condition, the expression analysis of AQPs in roots and leaves of two contrasting ecotypes of soybean revealed diverse expression patterns suggesting complex regulation at transcriptional level. Based on expression analysis, we identify <span class="gene">GmTIP2;1</span> as a potential candidate involved in salinity and <span class="trait">drought</span> responses. The overexpression of <span class="gene">GmTIP2;1</span> in Saccharomyces cerevisiae as well as in-planta enhanced salt and <span class="trait">drought</span> tolerance. We identified that <span class="gene">GmTIP2;1</span> forms homodimers as well as interacts with <span class="gene">GmTIP1;7</span> and <span class="gene">GmTIP1;8.</span> This study augments our knowledge of stress responsive pathways and also establishes <span class="gene">GmTIP2;1</span> as a new stress responsive gene in imparting salt stress tolerance in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmRFP1 encodes a previously unknown RING-type E3 ubiquitin ligase in Soybean (Glycine max).</h2>
			<h3>PMID:19373563  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du QL(1), Cui WZ, Zhang CH, Yu DY.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						RING-finger proteins with E3 ubiquitin ligase activity play important roles in the regulation of plant growth and development. In this study, a cDNA clone encoding a novel RING-finger protein, designated as <span class="gene">GmRFP1,</span> was isolated and characterized from soybean. <span class="gene">GmRFP1</span> was an intronless gene encoding a predicted protein product of 392 amino acid residues with a molecular mass of ~43 kDa. The protein contained a RING-H2 motif and an N-terminal transmembrane domain. The transcript was observed in all detected organs and was up-regulated by abscisic acid (ABA) and salt stress, but down-regulated by cold and <span class="trait">drought</span> treatments. We further expressed and purified both wild type and mutant version of <span class="gene">GmRFP1</span> in E. coli. In vitro assays showed that the purified <span class="gene">GmRFP1</span> induced the formation of polyubiquitin chains while mutation within the RING-finger region abolished the ubiquitination activity. These findings suggest that <span class="gene">GmRFP1</span> is a previously unknown E3 ubiquitin ligase in soybean and that the RING domain is required for its activity. It may play unappreciated roles in ABA signaling and stress responses via mediating the ubiquitination and degradation of target proteins through the ubiquitin-proteasome pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and characterization of the soybean DEAD-box RNA helicase gene induced by low temperature and high salinity stress.</h2>
			<h3>PMID:19463922  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chung E(1), Cho CW, Yun BH, Choi HK, So HA, Lee SW, Lee JH.
				<br/> 
				年份：2009 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel gene encoding a DEAD-box RNA helicase designated as <span class="gene">GmRH</span> was isolated from soybean. Amino acid sequence alignment and phylogenetic tree analysis revealed a close relationship between <span class="gene">GmRH</span> and other orthologous DEAD-box RNA helicases from other plant species. Structural motif analysis revealed that the bipartite lysine rich nuclear localization signal (NLS) is present in the N-terminal variable region of <span class="gene">GmRH</span> and that there are ten conserved motifs found in DEAD-box RNA helicase proteins. Southern blot analysis revealed the presence of 2 copies of <span class="gene">GmRH</span> in the soybean genome. Northern blot analysis demonstrated that the RNA expression of the <span class="gene">GmRH</span> was induced during low temperature or high salinity stress, but not by the exogenous application of abscisic acid or <span class="trait">drought</span> stress. Subcellular localization studies showed that <span class="gene">GmRH((1-355))-GFP</span> is localized in the nucleus, whereas <span class="gene">GmRH((130-355))-GFP</span> is localized both in the cytoplasm and in the nucleus. This provides the evidence that the N-terminal region predicted as NLS is essential for nuclear targeting of the <span class="gene">GmRH</span> protein in the plant cell. Purified GST-GmRH recombinant protein was shown to unwind dsRNA independent of ATP in vitro. Here, we propose that <span class="gene">GmRH</span> plays an important role in RNA processing during low temperature and high salinity stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of a glutathione S-transferase gene, GsGST, from wild soybean (Glycine soja) enhances <span class="trait">drought</span> and salt tolerance in transgenic tobacco.</h2>
			<h3>PMID:20383560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ji W(1), Zhu Y, Li Y, Yang L, Zhao X, Cai H, Bai X.
				<br/> 
				年份：2010 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine soja is a species of soybean that survives in adverse environments including high salt and <span class="trait">drought</span> conditions. We constructed a cDNA library from G. soja seedlings treated with NaCl and isolated a glutathione S-transferase gene (GsGST: GQ265911) from the library. The cDNA encoding GsGST contains an open reading frame of 660 bp and the predicted protein belongs to the tau class of <span class="gene">GST</span> family proteins. Tobacco plants over-expressing the GsGST gene showed sixfold higher <span class="gene">GST</span> activity than wild-type plants. Transgenic tobacco plants exhibited enhanced dehydration tolerance. T(2) transgenic tobacco plants showed higher tolerance at the seedling stage than wild-type plants to salt and mannitol as demonstrated by longer root length and less growth retardation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biological Networks Underlying Abiotic Stress Tolerance in Temperate Crops&#x02014;A Proteomic Perspective</h2>
			<h3>PMID:26340626</h3>
				
					<h3>PMCID:4613235</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kosov&#x000e1; </surname>  <given-names> Kl&#x000e1;ra </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-20913"> 1 </xref>  <xref rid="c1-ijms-16-20913" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> V&#x000ed;t&#x000e1;mv&#x000e1;s </surname>  <given-names> Pavel </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-20913"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Urban </surname>  <given-names> Milan Old&#x00159;ich </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-20913"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kl&#x000ed;ma </surname>  <given-names> Miroslav </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-20913"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Roy </surname>  <given-names> Amitava </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-16-20913"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pr&#x000e1;&#x00161;il </surname>  <given-names> Ilja Tom </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-20913"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Changes in gene expression are coupled with changes in protein metabolism including both protein biosynthesis and degradation. Plant adjustment to an altered environment requires myriads of novel proteins to be synthesized as well as myriads of proteins to be degraded. Therefore, several changes in the abundance of ribosomal proteins include proteins belonging to both eukaryotic and prokaryotic-type (mitochondrial and plastidic) ribosomal subunits. For example, an increase in ribosomal protein L39 involved in accuracy of translation in drought-treated grapevine was found by Vincent <italic> et al. </italic> [ <xref rid="B66-ijms-16-20913" ref-type="bibr"> 66 </xref> ]. A decrease in chloroplast 30S ribosomal protein S10 indicates a down-regulation of chloroplast protein biosynthesis in salt-sensitive canola cultivar Sarigol since protein S10 seems to be crucial for tRNA binding to ribosomal surface and the stability of 30S ribosomal subunit [ <xref rid="B67-ijms-16-20913" ref-type="bibr"> 67 </xref> ]. Moreover, differential abundance of several eukaryotic translation initiation and elongation factors was found. However, the observed change may be related also to processes other than protein biosynthesis. For example, eukaryotic translation initiation factor 5A (eIF5A) reveals multiple regulatory roles in cell cycle regulation&#x02014;different forms of eIF5A are proposed to affect a switch between cell proliferation and cell death [ <xref rid="B68-ijms-16-20913" ref-type="bibr"> 68 </xref> ]. A lower decrease of eIF5A3 isoform in salt-treated <italic> Triticum aestivum </italic> &#x000d7; <italic> Thinopyrum ponticum </italic> hybrid with respect to its <italic> T. aestivum </italic> parent indicates a higher anti-senescence ability of salt-tolerant hybrid compared to salt-sensitive parent under salinity [ <xref rid="B54-ijms-16-20913" ref-type="bibr"> 54 </xref> ]. Factor eIF5A was also detected in salt-tolerant oilseed rape Hyola 308 while it was absent in salt-sensitive cultivar Sarigol under salt stress [ <xref rid="B67-ijms-16-20913" ref-type="bibr"> 67 </xref> ]. Moreover, alterations in several regulatory proteins involved in mRNA stabilization, processing, and editing, e.g., nuclear-encoded chloroplastic ribonucleoprotein cp29, were found in stressed plants [ <xref rid="B69-ijms-16-20913" ref-type="bibr"> 69 </xref> , <xref rid="B70-ijms-16-20913" ref-type="bibr"> 70 </xref> ]. Regulation of cell cycle and programmed cell death (PCD) is also associated with translationally controlled tumor protein homolog (TCTP) which has been characterized as an agent decreasing cytosolic Ca <sup> 2+ </sup> levels and thus inhibiting PCD [ <xref rid="B71-ijms-16-20913" ref-type="bibr"> 71 </xref> ]. An increased abundance of <span class="gene">TCTP</span> was observed under several stresses including salinity [ <xref rid="B72-ijms-16-20913" ref-type="bibr"> 72 </xref> ], <span class="trait">drought</span> [ <xref rid="B56-ijms-16-20913" ref-type="bibr"> 56 </xref> ], and others.
					</p>
				
					<p>&nbsp;&nbsp;
						Photosynthesis is highly sensitive to imbalances between primary electron-transport processes and secondary chemical reactions associated with CO <sub> 2 </sub> assimilation. A discrepancy between the rate of primary and secondary photosynthetic reactions enhances a risk of ROS formation. Changes in photosystem-associated proteins, especially OEE proteins as components of PSII OEC center involved in photolysis of water, were found in several studies [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> , <xref rid="B56-ijms-16-20913" ref-type="bibr"> 56 </xref> , <xref rid="B60-ijms-16-20913" ref-type="bibr"> 60 </xref> , <xref rid="B77-ijms-16-20913" ref-type="bibr"> 77 </xref> , <xref rid="B83-ijms-16-20913" ref-type="bibr"> 83 </xref> , <xref rid="B86-ijms-16-20913" ref-type="bibr"> 86 </xref> , <xref rid="B89-ijms-16-20913" ref-type="bibr"> 89 </xref> ]. A decrease in RubisCO large and small subunits (RubisCO LSU and SSU) as well as Calvin cycle enzymes phosphoglycerokinase (PGK), phosphoribulokinase (PRK), and transketolase was found in <span class="trait">drought-</span> and salt-treated durum wheat [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> , <xref rid="B30-ijms-16-20913" ref-type="bibr"> 30 </xref> ] as well as in cold-treated spring Iranian wheat Kohdasht [ <xref rid="B60-ijms-16-20913" ref-type="bibr"> 60 </xref> ]. In contrast, an increase in proteins with protective functions such as RubisCO activase A [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> , <xref rid="B30-ijms-16-20913" ref-type="bibr"> 30 </xref> , <xref rid="B76-ijms-16-20913" ref-type="bibr"> 76 </xref> , <xref rid="B90-ijms-16-20913" ref-type="bibr"> 90 </xref> ], a Triticeae-specific thermostable RubisCO activase B [ <xref rid="B91-ijms-16-20913" ref-type="bibr"> 91 </xref> ], and carbonic anhydrase [ <xref rid="B30-ijms-16-20913" ref-type="bibr"> 30 </xref> , <xref rid="B56-ijms-16-20913" ref-type="bibr"> 56 </xref> ], were found in stressed plants. Changes in OEE1 and OEE2 proteins were frequently found in salt-treated barley [ <xref rid="B76-ijms-16-20913" ref-type="bibr"> 76 </xref> , <xref rid="B77-ijms-16-20913" ref-type="bibr"> 77 </xref> ], durum wheat [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> ], and tobacco [ <xref rid="B92-ijms-16-20913" ref-type="bibr"> 92 </xref> ], and an increase in OEE1 protein was observed in drought-treated barley infected by <italic> Piriformospora indica </italic> [ <xref rid="B56-ijms-16-20913" ref-type="bibr"> 56 </xref> ]. Proteomic studies have usually shown an increase in OEE proteins under milder stress and in tolerant plant materials while a decrease under severe stress or in sensitive plant materials [ <xref rid="B93-ijms-16-20913" ref-type="bibr"> 93 </xref> , <xref rid="B94-ijms-16-20913" ref-type="bibr"> 94 </xref> ]. Moreover, an increase in RubisCO chaperones CPN60-&#x003b1;, CPN60-&#x003b2;, and 20-kDa co-chaperonin was found under several stresses [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> , <xref rid="B55-ijms-16-20913" ref-type="bibr"> 55 </xref> , <xref rid="B62-ijms-16-20913" ref-type="bibr"> 62 </xref> , <xref rid="B69-ijms-16-20913" ref-type="bibr"> 69 </xref> , <xref rid="B95-ijms-16-20913" ref-type="bibr"> 95 </xref> , <xref rid="B96-ijms-16-20913" ref-type="bibr"> 96 </xref> , <xref rid="B97-ijms-16-20913" ref-type="bibr"> 97 </xref> ]. Stress also leads to profound changes in aerobic metabolism. Due to an enhanced risk of ROS formation, a decrease in some components of mitochondrial electron-transport chain such as NADH-dependent ubiquinone oxidoreductase was found in some studies [ <xref rid="B83-ijms-16-20913" ref-type="bibr"> 83 </xref> , <xref rid="B95-ijms-16-20913" ref-type="bibr"> 95 </xref> ]. Imbalances in the rates of several aerobic processes lead to an enhanced risk of oxidative stress. Therefore, an increase in alternative electron-transport pathways using alternative oxidase [ <xref rid="B98-ijms-16-20913" ref-type="bibr"> 98 </xref> , <xref rid="B99-ijms-16-20913" ref-type="bibr"> 99 </xref> ] as well as in enzymes involved in anaerobic ATP-producing processes such as glycolysis (glyceraldehyde-3-phosphate dehydrogenase GAPDH, triosephosphate isomerase TPI, enolase ENO, pyruvate kinase) and alcoholic fermentation (alcohol dehydrogenase ADH, aldehyde dehydrogenase, formate dehydrogenase) has been found in several studies dealing with waterlogged root cells, but also with seedlings and young plants exposed to severe dehydration stresses such as <span class="trait">drought</span> and salinity [ <xref rid="B29-ijms-16-20913" ref-type="bibr"> 29 </xref> , <xref rid="B30-ijms-16-20913" ref-type="bibr"> 30 </xref> , <xref rid="B62-ijms-16-20913" ref-type="bibr"> 62 </xref> , <xref rid="B73-ijms-16-20913" ref-type="bibr"> 73 </xref> , <xref rid="B83-ijms-16-20913" ref-type="bibr"> 83 </xref> , <xref rid="B95-ijms-16-20913" ref-type="bibr"> 95 </xref> , <xref rid="B100-ijms-16-20913" ref-type="bibr"> 100 </xref> ]. In contrast, a decline in the levels of glycolytic enzymes was found during the flax seed development in radioactivity-contaminated environment [ <xref rid="B35-ijms-16-20913" ref-type="bibr"> 35 </xref> ]. However, an increased need for energy can lead to an increase in several proteins related to aerobic metabolism (Krebs cycle, mitochondrial electron-transport chain) as indicated by an enhanced level of thiamine thiazole synthase, dihydrolipoamine acetyltransferase, and other cofactors of dehydrogenase complex catalyzing pyruvate conversion to acetyl-CoA [ <xref rid="B101-ijms-16-20913" ref-type="bibr"> 101 </xref> ]. Differential patterns of changes in several isoforms of glycolysis enzymes were frequently reported in one proteomic experiment as shown for enolase isoforms in drought-treated sunflower roots [ <xref rid="B33-ijms-16-20913" ref-type="bibr"> 33 </xref> ]. Moreover, nuclear isoforms of cytoplasmic glycolytic enzymes can act as important regulators of stress-responsive pathways as reported for a nuclear isoform of ENO encoded by <italic> LOS2 </italic> locus and involved in regulation of cold-inducible <span class="gene">CBF</span> pathway [ <xref rid="B102-ijms-16-20913" ref-type="bibr"> 102 </xref> ] or for a nuclear <span class="gene">GAPDH</span> involved in tRNA transport [ <xref rid="B103-ijms-16-20913" ref-type="bibr"> 103 </xref> , <xref rid="B104-ijms-16-20913" ref-type="bibr"> 104 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Several stress factors including <span class="trait">drought,</span> salinity, but often also cold, frost, and heat, induce cellular dehydration. A decrease in cell water content in plant cells results in a lack of hydration envelopes and an increased risk of an improper protein folding. Protein disulfide isomerase (PDI) catalyzes a reversible cleavage of disulfide bonds and thus affects protein conformation. Alterations in PDI abundance were found in several studies [ <xref rid="B37-ijms-16-20913" ref-type="bibr"> 37 </xref> , <xref rid="B38-ijms-16-20913" ref-type="bibr"> 38 </xref> , <xref rid="B75-ijms-16-20913" ref-type="bibr"> 75 </xref> , <xref rid="B88-ijms-16-20913" ref-type="bibr"> 88 </xref> , <xref rid="B105-ijms-16-20913" ref-type="bibr"> 105 </xref> ]. An increased accumulation of several hydrophilic proteins from COR/LEA family including LEA-II dehydrins [ <xref rid="B106-ijms-16-20913" ref-type="bibr"> 106 </xref> , <xref rid="B107-ijms-16-20913" ref-type="bibr"> 107 </xref> , <xref rid="B108-ijms-16-20913" ref-type="bibr"> 108 </xref> , <xref rid="B109-ijms-16-20913" ref-type="bibr"> 109 </xref> ] and LEA-III proteins such as chloroplast-located COR14b protein [ <xref rid="B110-ijms-16-20913" ref-type="bibr"> 110 </xref> , <xref rid="B111-ijms-16-20913" ref-type="bibr"> 111 </xref> , <xref rid="B112-ijms-16-20913" ref-type="bibr"> 112 </xref> ] and others [ <xref rid="B101-ijms-16-20913" ref-type="bibr"> 101 </xref> ] was found under cold and <span class="trait">drought.</span> Other proteins with chaperone functions include heat shock cognate proteins (HSC) and proteins from HSP superfamily encompassing five families of HSP proteins, HSP110, HSP90, HSP70, HSP60, and small HSP (sHSP) proteins. An increase in several small HSP proteins, but also HSP82 from <span class="gene">HSP90</span> family was found in wheat grain endosperm in developing wheat grains subjected to a heat period [ <xref rid="B113-ijms-16-20913" ref-type="bibr"> 113 </xref> , <xref rid="B114-ijms-16-20913" ref-type="bibr"> 114 </xref> ]. An increase in <span class="gene">HSP70</span> and HSC70 was found in watermelon exposed to <span class="trait">drought</span> and in tomato exposed to waterlogging, respectively [ <xref rid="B115-ijms-16-20913" ref-type="bibr"> 115 </xref> , <xref rid="B116-ijms-16-20913" ref-type="bibr"> 116 </xref> ]. Not only an increase, but also a decrease in some HSP proteins was found in several proteomic studies, e.g., a decrease in <span class="gene">HSP70</span> in drought-treated soybean roots [ <xref rid="B74-ijms-16-20913" ref-type="bibr"> 74 </xref> ] and salt-treated tobacco chloroplast stroma [ <xref rid="B92-ijms-16-20913" ref-type="bibr"> 92 </xref> ], and a decrease in <span class="gene">HSP90</span> in cold-treated winter wheats [ <xref rid="B75-ijms-16-20913" ref-type="bibr"> 75 </xref> ]. An opposite pattern of changes was reported for several sHSP26 protein isoforms in drought-stressed maize leaves [ <xref rid="B117-ijms-16-20913" ref-type="bibr"> 117 </xref> ]. Other proteins with chaperone functions found in proteomic studies include copper chaperone [ <xref rid="B75-ijms-16-20913" ref-type="bibr"> 75 </xref> , <xref rid="B89-ijms-16-20913" ref-type="bibr"> 89 </xref> , <xref rid="B118-ijms-16-20913" ref-type="bibr"> 118 </xref> ], cystatin [ <xref rid="B62-ijms-16-20913" ref-type="bibr"> 62 </xref> , <xref rid="B119-ijms-16-20913" ref-type="bibr"> 119 </xref> ] (cysteine protease inhibitor), serpins [ <xref rid="B73-ijms-16-20913" ref-type="bibr"> 73 </xref> , <xref rid="B84-ijms-16-20913" ref-type="bibr"> 84 </xref> , <xref rid="B120-ijms-16-20913" ref-type="bibr"> 120 </xref> , <xref rid="B121-ijms-16-20913" ref-type="bibr"> 121 </xref> ] (serine protease inhibitor), Zn-dependent metalloproteases [ <xref rid="B86-ijms-16-20913" ref-type="bibr"> 86 </xref> ], DnaK [ <xref rid="B60-ijms-16-20913" ref-type="bibr"> 60 </xref> ], <italic> etc. </italic> Besides their roles as protein chaperones, an interaction of <span class="gene">HSP70</span> with glutathione-related enzymes such as GPX and <span class="gene">GR</span> was reported in animal cells thus indicating a role of <span class="gene">HSP70</span> in regulation of cell redox homeostasis [ <xref rid="B122-ijms-16-20913" ref-type="bibr"> 122 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Hypoxia stimulates anaerobic metabolism such as glycolysis and fermentation processes, but also biosynthesis of efficient O <sub> 2 </sub> captures such as hemoglobin. In waterlogged maize root cells, an enhanced abundance of coproporphyrinogen III oxidase which catalyzes oxidative decarboxylation of coproporphyrinogen III to protoporphyrinogen IX in heme, a crucial step in hemoglobin biosynthesis, was found by Yu <italic> et al. </italic> [ <xref rid="B100-ijms-16-20913" ref-type="bibr"> 100 </xref> ]. Hypoxia also results in decreased soil pH. Maintenance of stable cellular pH necessary for a proper enzyme function is achieved by an enhanced accumulation of cytoplasmic enzymes with buffering capacity such as glutamate decarboxylase (GDC), malate dehydrogenase (MDH), and NADP-malic enzyme (NADP-ME) [ <xref rid="B100-ijms-16-20913" ref-type="bibr"> 100 </xref> ]. An enhanced abundance of GDC, NADP-ME3, and NADP-ME4 was found in waterlogging-tolerant maize line with respect to the sensitive one [ <xref rid="B100-ijms-16-20913" ref-type="bibr"> 100 </xref> ]. Glutathione S-transferases (GST) represent a large family of enzymes catalyzing glutathione (GSH) conjugation to various substrates. They encompass several structural classes marked by Greek letters &#x003c6;, &#x003c4;, &#x003b6;, &#x003b8;. They are known as enzymes involved in detoxification of heavy metals and xenobiotics including several herbicides; however, their glutathionylating activity could also play an important role in regulation of protein activity ( <italic> S </italic> -glutathionylation as a posttranslational modification) and in secondary metabolism [ <xref rid="B125-ijms-16-20913" ref-type="bibr"> 125 </xref> ] ( <italic> S </italic> -glutathionylated intermediates in metabolism of terpenes, glucosinolates, thiophenes, alliins, <italic> etc. </italic> ). Increased <span class="gene">GST</span> levels were found under several stresses including wheat exposed to enhanced copper levels [ <xref rid="B37-ijms-16-20913" ref-type="bibr"> 37 </xref> ], cold [ <xref rid="B69-ijms-16-20913" ref-type="bibr"> 69 </xref> , <xref rid="B75-ijms-16-20913" ref-type="bibr"> 75 </xref> ], <span class="trait">drought</span> [ <xref rid="B88-ijms-16-20913" ref-type="bibr"> 88 </xref> , <xref rid="B94-ijms-16-20913" ref-type="bibr"> 94 </xref> , <xref rid="B96-ijms-16-20913" ref-type="bibr"> 96 </xref> , <xref rid="B105-ijms-16-20913" ref-type="bibr"> 105 </xref> ], salinity [ <xref rid="B77-ijms-16-20913" ref-type="bibr"> 77 </xref> , <xref rid="B81-ijms-16-20913" ref-type="bibr"> 81 </xref> ], and cadmium [ <xref rid="B31-ijms-16-20913" ref-type="bibr"> 31 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Li <italic> et al. </italic> [ <xref rid="B143-ijms-16-20913" ref-type="bibr"> 143 </xref> ] studied the effect of a spring freezing in combination with either <span class="trait">drought</span> or waterlogging (water stresses) on winter wheat cv. Yannong leaves sampled from plants in anther connective tissue formation stage. Differences between the individual treatments and combined treatments were observed. For example, <span class="gene">HSP70</span> decreased in response to a single freeze stress treatment while the same protein increased in response to combined stress treatments. In contrast, decreased levels of chloroplast ATP synthase &#x003b2; subunit and mitochondrial ATP synthase &#x003b1; subunit in both single freezing and combined freezing and waterlogging treatments are consistent with a decrease in Ca <sup> 2+ </sup> and Mg <sup> 2+ </sup> -ATPase activities and an observed damage of PSII under waterlogging stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Yang <italic> et al. </italic> [ <xref rid="B144-ijms-16-20913" ref-type="bibr"> 144 </xref> ] investigated an effect of <span class="trait">drought</span> and heat (32 &#x000b0;C) treatment on proteome composition of wheat grain in the stage of terminal spikelet and anthesis. Several proteins revealed a specific response to each stress treatment while only a few common proteins involved in redox metabolism, defense, carbohydrate metabolism, and storage revealed an analogous response to multiple stress treatments. Proteins responding exclusively to heat stress include increased cinnamoyl-CoA reductase, <span class="gene">TCTP</span> (translationally-controlled tumor protein), cell division control protein, and heat shock cognate 70 (HSC70) and a decreased 14-3-3 protein.
					</p>
				
					<p>&nbsp;&nbsp;
						A comparison of contrasting water stresses&#x02014;drought and flooding&#x02014;was carried out by Oh and Komatsu [ <xref rid="B78-ijms-16-20913" ref-type="bibr"> 78 </xref> ] in soybean seedlings. The results have shown differentially regulated stress responses&#x02014;an increase in enzymes involved in regulation of redox homeostasis was found in drought-stressed plants while an increase in anaerobic metabolism-related (fermentation) enzymes was found in flooded plants. Moreover, an opposite pattern of changes in SAM synthetase (SAMS) was found in drought-treated plants vs flooded ones indicating an increase in <span class="gene">SAMS</span> under <span class="trait">drought</span> while a decrease in <span class="gene">SAMS</span> under flooding. Since SAM is a universal cell methylation agent involved in lignin biosynthesis, the differential pattern of <span class="gene">SAMS</span> may be related to changes observed in root cell wall lignification in soybean seedlings under the two treatments corresponding to an increased cell wall lignification under <span class="trait">drought</span> while a decreased cell wall lignification under flooding, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Effects of <span class="trait">drought,</span> cold, and herbicide paraquat treatments on pea mitochondrial proteome were compared by Taylor <italic> et al. </italic> [ <xref rid="B42-ijms-16-20913" ref-type="bibr"> 42 </xref> ]. The strongest adverse effects on mitochondrial proteins resulting in an oxidative damage were observed under paraquat treatment, followed by chilling while <span class="trait">drought</span> revealed the mildest effects. Mitochondria isolated from stressed pea plants maintained their electron transport chain activity; complexes of mitochondrial electron transport chain were least damaged by oxidative stress, F <sub> 1 </sub> F <sub> o </sub> ATP synthase complex was more damaged while enzymes of carbon metabolism in mitochondrial matrix were significantly modified by oxidation. Moreover, increased lipid peroxidation and a decrease in inner membrane import proteins were observed. Differential changes in abundance of several HSP proteins with an induction of <span class="gene">HSP22</span> and opposite patterns in several isoforms of <span class="gene">HSP70</span> were also found under both chilling and <span class="trait">drought</span> treatments.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Drought-Responsive Universal Stress Proteins in Viridiplantae</h2>
			<h3>PMID:21423406</h3>
				
					<h3>PMCID:3045048</h3>
					<h4>期刊:Bioinformatics and Biology Insights</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Isokpehi </surname>  <given-names> Raphael D. </given-names>  </name>  <xref ref-type="corresp" rid="c1-bbi-2011-041"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Simmons </surname>  <given-names> Shaneka S. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cohly </surname>  <given-names> Hari H.P. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ekunwe </surname>  <given-names> Stephen I.N. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Begonia </surname>  <given-names> Gregorio B. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ayensu </surname>  <given-names> Wellington K. </given-names>  </name>  </contrib>  <aff id="af1-bbi-2011-041"> Department of Biology, Jackson State University, Jackson MS 39217, USA </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Twelve USP sequences were annotated with both the ArrayExpress and Ortholog Matrix Project (OMA) Browser ( <xref ref-type="fig" rid="f4-bbi-2011-041"> Fig. 4 </xref> ). Three <italic> Arabidopsis </italic> USP genes (Q93W91 [At3g62550], Q9LPF5 <span class="gene">[At1g44760]</span> and Q9M328 [AT3g53990]) were up regulated in a <span class="trait">drought</span> microarray experiment stored in ArrayExpress and were annotated in the OMA Browser. Box plots of the three genes obtained from ArrayExpess as well as multiple sequence alignment of orthologs are presented in <xref ref-type="fig" rid="f5-bbi-2011-041"> Figure 5 </xref> . The OMA Browser provides multiple sequence alignment for groups of orthologs for each protein sequence ( <xref ref-type="fig" rid="f5-bbi-2011-041"> Fig. 5 </xref> ). Orthologous sequences were from <italic> Oryza sativa </italic> , <italic> Sorghum bicolor </italic> , <italic> Populus trichocarpa </italic> and <italic> Vitis vinifera </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comprehensive Analysis of the Soybean ( <italic> Glycine max </italic> ) <italic> <span class="gene">GmLAX</span> </italic> Auxin Transporter Gene Family</h2>
			<h3>PMID:27014306</h3>
				

				
					<h3>PMCID:4783406</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Chenglin </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/65376/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yongqin </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/65688/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The phytohormone auxin plays a critical role in regulation of plant growth and development as well as plant responses to abiotic stresses. This is mainly achieved through its uneven distribution in plant via a polar auxin transport process. Auxin transporters are major players in polar auxin transport. The AUXIN RESISTENT 1/LIKE AUX1 (AUX/LAX) auxin influx carriers belong to the amino acid permease family of proton-driven transporters and function in the uptake of indole-3-acetic acid (IAA). In this study, genome-wide comprehensive analysis of the soybean AUX/LAX <span class="gene">(GmLAX)</span> gene family, including phylogenic relationships, chromosome localization, and gene structure, was carried out. A total of 15 <span class="gene">GmLAX</span> genes, including seven duplicated gene pairs, were identified in the soybean genome. They were distributed on 10 chromosomes. Despite their higher percentage identities at the protein level, <span class="gene">GmLAXs</span> exhibited versatile tissue-specific expression patterns, indicating coordinated functioning during plant growth and development. Most <span class="gene">GmLAXs</span> were responsive to <span class="trait">drought</span> and dehydration stresses and auxin and abscisic acid (ABA) stimuli, in a tissue- and/or time point- sensitive mode. Several <span class="gene">GmLAX</span> members were involved in responding to salt stress. Sequence analysis revealed that promoters of <span class="gene">GmLAXs</span> contained different combinations of stress-related cis-regulatory elements. These studies suggest that the soybean <span class="gene">GmLAXs</span> were under control of a very complex regulatory network, responding to various internal and external signals. This study helps to identity candidate <span class="gene">GmLAXs</span> for further analysis of their roles in soybean development and adaption to adverse environments.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is one of the most <span class="trait">drought</span> and salinity sensitive crops. Its <span class="trait">yield</span> is significantly influenced by these abiotic stresses. In order to explore whether any <italic> <span class="gene">GmLAX</span> </italic> genes are involved in abiotic stress response, expressions of <italic> <span class="gene">GmLAXs</span> </italic> were investigated under <span class="trait">drought,</span> salinity, and dehydration using qRT-PCR (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ). Twelve <italic> <span class="gene">GmLAXs</span> </italic> were responsive to <span class="trait">drought</span> stresses. Under mild <span class="trait">drought</span> stress, six <italic> <span class="gene">GmLAXs</span> </italic> were transcriptionally regulated, with most of them (four out of six) being up-regulated in either shoot or root. While upon moderate <span class="trait">drought</span> stress, all 10 responsive <italic> <span class="gene">GmLAXs</span> </italic> were down-regulated, with two specifically in shoot, six solely in root, and two in both shoot and root. The response of <italic> <span class="gene">GmLAXs</span> </italic> to <span class="trait">drought</span> stresses was in a tissue-specific and stress magnitude-specific mode (Figure <xref ref-type="fig" rid="F5"> 5B </xref> ). For example, seven <italic> <span class="gene">GmLAXs</span> </italic> were responsive to only one <span class="trait">drought</span> stress treatment (mild or moderate) and in only one tissue (shoot or root). In some cases, the same gene was differentially regulated in different tissues. For instance, <italic> <span class="gene">GmLAX9</span> </italic> and <italic> <span class="gene">GmLAX15</span> </italic> were both up-regulated in shoots by mild <span class="trait">drought</span> stress but were down-regulated in roots by moderate <span class="trait">drought</span> stress (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Differential expression of <italic> <span class="gene">GmLAXs</span> </italic> in response to <span class="trait">drought,</span> salt, and dehydration. (A) </bold> Fold changes of <italic> <span class="gene">GmLAXs</span> </italic> gene expressions in shoots and roots under mild <span class="trait">drought</span> stress (&#x02212;7 bar) and moderate <span class="trait">drought</span> stress (&#x02212;13 bar) treatments, and in whole seedlings under salt (250 mM NaCl) and dehydration stress treatments. Data showed the means of three biological repeats in qRT-PCR analysis. Asterisks indicated fold changes (absolute value) &#x02265;2 and <italic> P </italic> &#x0003c; 0.05 (Student's <italic> t </italic> -test). <bold> (B) </bold> Venn diagram analysis of data in <bold> (A) </bold> . MS-L, mild <span class="trait">drought</span> stress-shoots; MS-R, mild <span class="trait">drought</span> stress-roots; SS-L, moderate <span class="trait">drought</span> stress-shoots; SS-R, moderate <span class="trait">drought</span> stress-roots; S-1 h, salt 1 h; S-5 h, salt 5 h; S-10 h, salt 10 h; S-24 h, salt 24 h; D-1 h, dehydration 1 h; D-5 h, dehydration 5 h; D-10 h, dehydration 10 h.
					</p>
				
					<p>&nbsp;&nbsp;
						Quantitative-PCR analysis revealed that six <italic> <span class="gene">GmLAXs</span> </italic> were differentially expressed under salinity conditions and all of them were down-regulated (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). However, thirteen <italic> <span class="gene">GmLAXs</span> </italic> were down-regulated by dehydration. Of the 15 <italic> <span class="gene">GmLAXs</span> </italic> , three genes, i.e., <italic> <span class="gene">GmLAX11,</span> 2 </italic> , and <italic> 13 </italic> , were involved in responses to all three abiotic stresses, eight were exclusively responsive to <span class="trait">drought</span> and dehydration stresses, and two were specifically regulated by salt and dehydration stresses (Figure <xref ref-type="fig" rid="F5"> 5B </xref> ). Only one gene was specifically responsive to salt stress, one to <span class="trait">drought</span> stress, and none of the 15 <italic> <span class="gene">GmLAXs</span> </italic> were dehydration-specific.
					</p>
				
					<p>&nbsp;&nbsp;
						Under abiotic stresses such as <span class="trait">drought</span> and salinity, plants usually first adaptively decrease growth rate before growth stops or death occurs. The uneven distribution of auxin, which is mainly mediated via auxin transporters, plays a key role in plants' adaptation to adverse conditions by adjusting growth rate. Crosstalk between auxin and biotic and abiotic stress signaling has been reported in some plant species (Ghanashyam and Jain, <xref rid="B9" ref-type="bibr"> 2009 </xref> ; Jain and Khurana, <xref rid="B15" ref-type="bibr"> 2009 </xref> ). In soybean, genome-wide transcriptome analyses showed that many hormone-related genes were differentially expressed in leaf and root under water deficit conditions (Le et al., <xref rid="B21" ref-type="bibr"> 2012 </xref> ; Song et al., <xref rid="B45" ref-type="bibr"> 2016 </xref> ). Most members of the soybean <italic> PIN </italic> gene family were responsive to various abiotic stresses and phytohormone stimuli (Wang et al., <xref rid="B58" ref-type="bibr"> 2015 </xref> ). However, the precise molecular mechanism regarding regulation of auxin transport and distribution, which were achieved by coordination of different auxin transporters, is largely unknown in soybean. In this study, responses of <italic> <span class="gene">GmLAXs</span> </italic> , putative auxin influx carriers in soybean, to abiotic stresses and hormone signals, including auxin and ABA, were investigated (Figures <xref ref-type="fig" rid="F5"> 5 </xref> , <xref ref-type="fig" rid="F6"> 6 </xref> ). Most <italic> <span class="gene">GmLAXs</span> </italic> were down-regulated by <span class="trait">drought</span> and dehydration, while only six <italic> <span class="gene">GmLAXs</span> </italic> were responsive to salt stress, and all of them were down-regulated. Decreased expression levels of <italic> <span class="gene">GmLAX</span> </italic> s might reflect down-regulation of auxin up-taking and/or transport, which might result in decreased or ceased growth of soybean sink tissues. This could at least partially explain the lower biomass and <span class="trait">yield</span> of soybean under abiotic stresses (Liu et al., <xref rid="B26" ref-type="bibr"> 2003 </xref> ). The expression patterns of <italic> <span class="gene">GmLAX</span> </italic> s under abiotic stresses were different from those of the maize <italic> AUX/LAX </italic> s, which were up-regulated by salt and <span class="trait">drought</span> stresses in shoots, but were repressed in the roots (Yue et al., <xref rid="B63" ref-type="bibr"> 2015 </xref> ). Interestingly, the sorghum <italic> SbLAXs </italic> exhibited irregular expression patterns in response to <span class="trait">drought</span> and salt (Shen et al., <xref rid="B43" ref-type="bibr"> 2010 </xref> ). These studies suggested that the three plant species might have different mechanisms in responding to these unfavorable environments.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Soybean LATERAL ORGAN BOUNDARIES Domain-Containing Genes: A Functional Investigation of <span class="gene">GmLBD12.</span></h2>
			<h3>PMID:28464070</h3>
				

				
			<p>
				作者：Yang H, Shi G, Du H, Wang H, Zhang Z, Hu D, Wang J, Huang F, Yu D.
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-specific () genes play critical roles in various plant growth and development processes. However, the number and characteristics of genes in soybean [ (L.) Merr.] remain unknown. Here, we identified 90 homologous genes in the soybean genome that phylogenetically clustered into two classes (I and II). The majority of the genes were evenly distributed across all 20 soybean chromosomes, and 77 (81.11%) of them were detected in segmental duplicated regions. Furthermore, the exon-intron organization and motif composition for each were analyzed. A close phylogenetic relationship was identified between the soybean genes and 41 previously reported genes of different plants in the same group, providing insights into their putative functions. Expression analysis indicated that more than half of the genes were expressed, with the two gene classes showing differential tissue expression characteristics; in addition, they were differentially induced by biotic and abiotic stresses. To further explore the functions of genes in soybean, was selected for functional characterization. <span class="gene">GmLBD12</span> was mainly localized to the nucleus and showed high expression in root and seed tissues. Overexpressing in (L.) Heynh resulted in increases in lateral root (LR) number and plant height. Quantitative real-time polymerase chain reaction (qRT-PCR) analysis demonstrated that was induced by <span class="trait">drought,</span> salt, cold, indole acetic acid (IAA), abscisic acid (ABA), and salicylic acid SA treatments. This study provides the first comprehensive analysis of the soybean gene family and a valuable foundation for future functional studies of genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">TIP</span> Gene Family Analysis and Characterization of <span class="gene">GmTIP1;5</span> and <span class="gene">GmTIP2;5</span> Water Transport Activity.</h2>
			<h3>PMID:27818669</h3>
				

				
			<p>
				作者：Song L(1), Nguyen N(1), Deshmukh RK(2), Patil GB(1), Prince SJ(1), Valliyodan B(1), Mutava R(1), Pike SM(3), Gassmann W(3), Nguyen HT(1).
				<br/> 
				年份：2016 Oct 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean, one of the most important crops worldwide, is severely affected by abiotic stress. <span class="trait">Drought</span> and flooding are the major abiotic stresses impacting soybean <span class="trait">yield.</span> In this regard, understanding water uptake by plants, its utilization and transport has great importance. In plants, water transport is mainly governed by channel forming aquaporin proteins (AQPs). Tonoplast intrinsic proteins (TIPs) belong to the plant-specific AQP subfamily and are known to have a role in abiotic stress tolerance. In this study, 23 soybean <span class="gene">TIP</span> genes were identified based on the latest soybean genome annotation. TIPs were characterized based on conserved structural features and phylogenetic distribution. Expression analysis of soybean <span class="gene">TIP</span> genes in various tissues and under abiotic stress conditions demonstrated tissue/stress-response specific differential expression. The natural variations for <span class="gene">TIP</span> genes were analyzed using whole genome re-sequencing data available for a set of 106 diverse soybean genotypes including wild types, landraces and elite lines. Results revealed 81 single-nucleotide polymorphisms (SNPs) and several large insertions/deletions in the coding region of TIPs. Among these, non-synonymous SNPs are most likely to have a greater impact on protein function and are candidates for molecular studies as well as for the development of functional markers to assist breeding. The solute transport function of two TIPs was further validated by expression in Xenopus laevis oocytes. <span class="gene">GmTIP1;5</span> was shown to facilitate the rapid movement of water across the oocyte membrane, while <span class="gene">GmTIP2;5</span> facilitated the movement of water and boric acid. The present study provides an initial insight into the possible roles of soybean <span class="gene">TIP</span> genes under abiotic stress conditions. Our results will facilitate elucidation of their precise functions during abiotic stress responses and plant development, and will provide potential breeding targets for modifying water movement in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel Glycine soja tonoplast intrinsic protein gene responds to abiotic stress and depresses salt and dehydration tolerance in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:21397356  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Ji W, Bai X, Cai H, Zhu D, Sun XL, Chen LJ, Zhu YM.
				<br/> 
				年份：2011 Jul 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Tonoplast intrinsic protein (TIP) is a subfamily of the aquaporin (AQP), also known as major intrinsic protein (MIP) family, and regulates water movement across vacuolar membranes. Some reports have implied that <span class="gene">TIP</span> genes are associated with plant tolerance to some abiotic stresses that cause water loss, such as <span class="trait">drought</span> and high salinity. In our previous work, we found that an expressed sequence tag (EST) representing a <span class="gene">TIP</span> gene in our Glycine soja EST library was inducible by abiotic stresses. This <span class="gene">TIP</span> was subsequently isolated from G. soja with cDNA library screening, EST assembly and PCR, and named as GsTIP2;1. The expression patterns of GsTIP2;1 in G. soja under low temperature, salt and dehydration stress were different in leaves and roots. Though GsTIP2;1 is a stress-induced gene, overexpression of GsTIP2;1 in Arabidopsis thaliana depressed tolerance to salt and dehydration stress, but did not affect seedling growth under cold or favorable conditions. Higher dehydration speed was detected in Arabidopsis plants overexpressing GsTIP2;1, implying GsTIP2;1 might mediate stress sensitivity by enhancing water loss in the plant. Such a result is not identical to previous reports, providing some new information about the relationship between <span class="gene">TIP</span> and plant abiotic stress tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Leaf Proteome Analysis Reveals Prospective <span class="trait">Drought</span> and Heat Stress Response Mechanisms in Soybean</h2>
			<h3>PMID:27034942</h3>
				

				
					<h3>PMCID:4808539</h3>
					<h4>期刊:BioMed Research International</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Das </surname>  <given-names> Aayudh </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 1 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> and heat are among the major abiotic stresses that affect soybean crops worldwide. During the current investigation, the effect of <span class="trait">drought,</span> heat, and <span class="trait">drought</span> plus heat stresses was compared in the leaves of two soybean varieties, Surge and Davison, combining 2D-DIGE proteomic data with physiology and biochemical analyses. We demonstrated how 25 differentially expressed photosynthesis-related proteins affect RuBisCO regulation, electron transport, Calvin cycle, and carbon fixation during <span class="trait">drought</span> and heat stress. We also observed higher abundance of heat stress-induced <span class="gene">EF-Tu</span> protein in Surge. It is possible that <span class="gene">EF-Tu</span> might have activated heat tolerance mechanisms in the soybean. Higher level expressions of heat shock-related protein seem to be regulating the heat tolerance mechanisms. This study identifies the differential expression of various abiotic stress-responsive proteins that regulate various molecular processes and signaling cascades. One inevitable outcome from the biochemical and proteomics assays of this study is that increase of ROS levels during <span class="trait">drought</span> stress does not show significant changes at the phenotypic level in Davison and this seems to be due to a higher amount of carbonic anhydrase accumulation in the cell which aids the cell to become more resistant to cytotoxic concentrations of H2O2.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Proteomic evaluation for the purpose of differentially regulated proteins identification in response to the <span class="trait">drought,</span> heat, and the combined stress has been monitored in various plants in context of the morphological and physiological changes [ <xref rid="B10" ref-type="bibr"> 10 </xref> &#x02013; <xref rid="B12" ref-type="bibr"> 12 </xref> ]. Particularly, changes in proteomic expression during <span class="trait">drought</span> stress have been observed in major crops like rice and wheat which shows differential regulation of various proteins [ <xref rid="B13" ref-type="bibr"> 13 </xref> &#x02013; <xref rid="B15" ref-type="bibr"> 15 </xref> ]. Earlier studies using proteomic and transcriptomic approaches also implicated the <span class="trait">drought</span> stress response mechanisms including alteration in the signal transduction pathway and plant's metabolism [ <xref rid="B16" ref-type="bibr"> 16 </xref> &#x02013; <xref rid="B19" ref-type="bibr"> 19 </xref> ]. Moreover, in response to heat stress, many stress proteins were originally identified as HSP which indicates plant's tolerance threshold [ <xref rid="B20" ref-type="bibr"> 20 </xref> ]. To improve the understanding of the mechanisms underlying soybean responses to <span class="trait">drought</span> and heat stress, in the present investigation we have used a global proteomic approach combined with physiological and computational analysis [ <xref rid="B10" ref-type="bibr"> 10 </xref> , <xref rid="B21" ref-type="bibr"> 21 </xref> , <xref rid="B22" ref-type="bibr"> 22 </xref> ]. The two major soybean varieties, Surge <span class="trait">(yield</span> 3,272&#x02009;Kg/ha) and Davison <span class="trait">(yield</span> 4,074&#x02009;Kg/ha), showing distinct physiological characteristics were used in this study. Proteomic analysis revealed differentially expressed proteins related to key biological processes (such as photosynthesis and respiration), various metabolic pathways (nitrogen metabolism and carbohydrate metabolism), and several other molecular processes (protein biosynthesis and ATP synthesis). Computational analysis predicted a protein network displaying the likely interactions between protein abundances and plant stress responses. These analyses will enable the development of plant molecular improvement programs to produce more effective strategies contributing to greater food security in the coming years. Since the proteins are the translated version of mRNA (proteomics approach may have advantage over transcriptomics approach [ <xref rid="B4" ref-type="bibr"> 4 </xref> ]) several research groups have utilized the power of proteomic evaluations and combined it with physiological studies to create a link between plant physiology and the molecular signatory responses [ <xref rid="B10" ref-type="bibr"> 10 </xref> , <xref rid="B23" ref-type="bibr"> 23 </xref> ]. The primary aim of this study was to investigate differential protein expression between two contrasting genotypes for <span class="trait">drought</span> and heat stress responses, and to reveal the <span class="trait">drought</span> and heat stress-responsive mechanisms in soybean at early stage of growth. Based on physiological analyses, we found that <span class="trait">drought</span> and heat stress decrease photosynthesis and reduce stomatal conductance and transpiration rates, which ultimately alter the net CO <sub> 2 </sub> concentrations in leaves. The proteomic analyses facilitated the characterization of a set of proteins whose expressions were altered under heat and <span class="trait">drought</span> stresses. We found that, in Surge, more proteins were downregulated during <span class="trait">drought</span> stress than during heat stress, but the combined stress conditions exerted a drastic effect at both the molecular and the phenotypic levels. In this study, we have identified genes involved in photosynthesis that were differentially expressed during <span class="trait">drought</span> and heat stress conditions. This differential expression is most likely the result of blocked interactions between RuBisCO, RuBisCO activase, electron transport chain, and downregulation of various Calvin cycle enzymes, which ultimately results in a net reduced level of photosynthesis [ <xref rid="B24" ref-type="bibr"> 24 </xref> ]. This study identified greater accumulations of <span class="gene">EF-Tu</span> protein during heat stress in Surge. The results also indicated the accumulation of a 70&#x02009;kDa stromal HSP, which we assume is crucial for plant survival during heat stress by activating heat tolerance responses [ <xref rid="B25" ref-type="bibr"> 25 </xref> ]. Previously, it has been reported that <span class="trait">drought</span> and heat stresses may trigger the formation of reactive oxygen species (ROS) [ <xref rid="B26" ref-type="bibr"> 26 </xref> , <xref rid="B27" ref-type="bibr"> 27 </xref> ]. Likewise, we also examined the <span class="trait">drought</span> and heat stress-induced changes in the production of ROS and found that, during <span class="trait">drought</span> stress, soybeans exhibited a significant increase in the level of hydrogen peroxide and an enhanced ROS detoxification capacity via carbonic anhydrase, which protects the plant against oxidative damage [ <xref rid="B28" ref-type="bibr"> 28 </xref> ]. Plants use ABA as a signaling molecule while maintaining a positive water balance throughout the system, as physiological studies have shown [ <xref rid="B29" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="B31" ref-type="bibr"> 31 </xref> ]. Elevated ABA levels during heat stress are shown to protect the maize plant against heat-induced oxidative damage [ <xref rid="B32" ref-type="bibr"> 32 </xref> ]. We quantified the ABA levels in soybean leaves during all the stress conditions and have discussed it in <span class="trait">drought</span> and heat stress contexts for soybean crop. At the level of computational analysis, we have constructed a protein-protein network to predict the interactions between the differentially expressed proteins [ <xref rid="B17" ref-type="bibr"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						2D-DIGE analysis revealed that the chloroplast protein synthesis elongation factor, <span class="gene">EF-Tu</span> protein (Spot 31), was upregulated by 4.6-fold in Surge compared to Davison under heat stress. Additionally, we found that, in Surge, under heat stress, fewer (13) proteins were downregulated than under <span class="trait">drought</span> stress, whereas 31 proteins were downregulated in Davison under heat stress. Chloroplast <span class="gene">EF-Tu</span> is a protein that is involved in the elongation of polypeptides during the translational process, and it belongs to a nuclear-encoded multigene family [ <xref rid="B62" ref-type="bibr"> 60 </xref> , <xref rid="B63" ref-type="bibr"> 61 </xref> ]. The heat stress-induced accumulation of <span class="gene">EF-Tu</span> in mature plants is thought to protect the photosynthesis machinery [ <xref rid="B64" ref-type="bibr"> 62 </xref> ]. It has been reported that maize <span class="gene">EF-Tu</span> plays a crucial role in heat tolerance by acting as a molecular chaperone, and it has been found to protect heat-labile citrate synthase, RuBisCO activase, and malate dehydrogenase from thermal accumulations [ <xref rid="B65" ref-type="bibr"> 63 </xref> , <xref rid="B66" ref-type="bibr"> 64 </xref> ]. The results indicate that, upon heat stress, <span class="gene">EF-Tu</span> assembles in the cytosol in Surge (as seen by an increase in the protein levels of Spots 29&#x02013;31). Therefore, based on these results, <span class="gene">EF-Tu</span> could serve as a biomarker of heat stress in soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						Reduced stomatal conductance is one of the most sensitive responses to <span class="trait">drought</span> stress, observed in kidney beans and various <span class="gene">C3</span> plants [ <xref rid="B50" ref-type="bibr"> 48 </xref> , <xref rid="B87" ref-type="bibr"> 83 </xref> ]. At the level of soybean physiology, we concluded that <span class="trait">drought</span> stress, heat stress, and <span class="trait">drought</span> plus heat stress reduce stomatal conductance, which is caused by decreasing the leaf water potential via transpiration rate alterations in both varieties. Stomatal conductance is also reliant on leaf temperature via transpiration rate, and we observed that the increase of leaf temperature is correlated to the stomatal conductance and transpiration rates. Photosynthesis is among the primary processes affected by the <span class="trait">drought</span> and heat stresses [ <xref rid="B16" ref-type="bibr"> 16 </xref> , <xref rid="B88" ref-type="bibr"> 84 </xref> , <xref rid="B89" ref-type="bibr"> 85 </xref> ]. As soil moisture decreases during <span class="trait">drought</span> stress and cooccurring <span class="trait">drought</span> and heat stress, the water content of mesophyll tissue also reduces, ultimately affecting the photosynthetic physiology, principally carbon assimilation and energy use of the plant. Surprisingly, there was no such significant effect observed during heat stress alone, which actually establishes the fact that consequences of water deficiency have an enormous impact on altering the photosynthetic machinery [ <xref rid="B90" ref-type="bibr"> 86 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						This study shows how <span class="trait">drought,</span> heat, and cooccurring <span class="trait">drought</span> and heat stresses alter the soybean leaf proteome. Differential expression of 44 abiotic stress-responsive proteins suggests that various signaling cascades and molecular processes are affected due to <span class="trait">drought</span> and heat stresses. Furthermore, the result suggests that many differentially expressed photosynthesis-related proteins perturb RuBisCO regulation, electron transport, and Calvin cycle during abiotic stress. These findings, as well as upregulation of <span class="gene">EF-Tu</span> and higher level expressions of HSP, will help in better understanding of the heat tolerance mechanisms in the soybean varieties. The biochemical and proteomic assays also explain how higher amount of carbonic anhydrase accumulation in the cell alleviates cytotoxic concentrations of H <sub> 2 </sub> O <sub> 2 </sub> during <span class="trait">drought</span> stress. Moreover, we found that cooccurring <span class="trait">drought</span> and heat stresses have the highest level of ABA accumulations in Davison, which also correlates with our stomatal conductance measurements. Taken together, the results presented provide a proteomic level explanation for the abiotic stress effects on physiological processes of soybean plants during abiotic stress conditions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Productivity, absorbed photosynthetically active radiation, and light use efficiency in crops: implications for remote sensing of crop primary production.</h2>
			<h3>PMID:25723474  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gitelson AA(1), Peng Y(2), Arkebauer TJ(3), Suyker AE(4).
				<br/> 
				年份：2015 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Vegetation productivity metrics such as gross primary production (GPP) at the canopy scale are greatly affected by the efficiency of using absorbed radiation for photosynthesis, or light use efficiency (LUE). Thus, close investigation of the relationships between canopy GPP and photosynthetically active radiation absorbed by vegetation is the basis for quantification of LUE. We used multiyear observations over irrigated and rainfed contrasting <span class="gene">C3</span> (soybean) and <span class="gene">C4</span> (maize) crops having different physiology, leaf structure, and canopy architecture to establish the relationships between canopy GPP and radiation absorbed by vegetation and quantify LUE. Although multiple LUE definitions are reported in the literature, we used a definition of efficiency of light use by photosynthetically active "green" vegetation (LUE(green)) based on radiation absorbed by "green" photosynthetically active vegetation on a daily basis. We quantified, irreversible slowly changing seasonal (constitutive) and rapidly day-to-day changing (facultative) LUE(green), as well as sensitivity of LUE(green) to the magnitude of incident radiation and <span class="trait">drought</span> events. Large (2-3-fold) variation of daily LUE(green) over the course of a growing season that is governed by crop physiological and phenological status was observed. The day-to-day variations of LUE(green) oscillated with magnitude 10-15% around the seasonal LUE(green) trend and appeared to be closely related to day-to-day variations of magnitude and composition of incident radiation. Our results show the high variability of LUE(green) between <span class="gene">C3</span> and <span class="gene">C4</span> crop species (1.43 g C/MJ vs. 2.24 g C/MJ, respectively), as well as within single crop species (i.e., maize or soybean). This implies that assuming LUE(green) as a constant value in GPP models is not warranted for the crops studied, and brings unpredictable uncertainties of remote GPP estimation, which should be accounted for in LUE models. The uncertainty of GPP estimation due to facultative and constitutive changes in LUE(green) can be considered as a critical component of the total error budget in the context of remotely sensed based estimations of GPP. The quantitative framework of LUE(green) estimation presented here offers a way of characterizing LUE(green) in plants that can be used to assess their phenological and physiological status and vulnerability to <span class="trait">drought</span> under current and future climatic conditions and is essential for calibration and validation of globally applied LUE algorithms.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing <span class="trait">drought</span> and diminishing benefits of elevated carbon dioxide for soybean yields across the US Midwest.</h2>
			<h3>PMID:29110424</h3>
				

				
			<p>
				作者：Jin Z(1), Ainsworth EA(2)(3), Leakey ADB(2), Lobell DB(1).
				<br/> 
				年份：2017 Nov 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elevated atmospheric CO2 concentrations ([CO2 ]) are expected to increase <span class="gene">C3</span> crop <span class="trait">yield</span> through the CO2 fertilization effect (CFE) by stimulating photosynthesis and by reducing stomatal conductance and transpiration. The latter effect is widely believed to lead to greater benefits in dry rather than wet conditions, although some recent experimental evidence challenges this view. Here we used a process-based crop model, the Agricultural Production Systems sIMulator (APSIM), to quantify the contemporary and future CFE on soybean in one of its primary production area of the US Midwest. APSIM accurately reproduced experimental data from the Soybean Free-Air CO2 Enrichment site showing that the CFE declined with increasing <span class="trait">drought</span> stress. This resulted from greater radiation use efficiency (RUE) and above-ground biomass production at elevated [CO2 ] that outpaced gains in transpiration efficiency (TE). Using an ensemble of eight climate model projections, we found that <span class="trait">drought</span> frequency in the US Midwest is projected to increase from once every 5 years currently to once every other year by 2050. In addition to directly driving <span class="trait">yield</span> loss, greater <span class="trait">drought</span> also significantly limited the benefit from rising [CO2 ]. This study provides a link between localized experiments and regional-scale modeling to highlight that increased <span class="trait">drought</span> frequency and severity pose a formidable challenge to maintaining soybean <span class="trait">yield</span> progress that is not offset by rising [CO2 ] as previously anticipated. Evaluating the relative sensitivity of RUE and TE to elevated [CO2 ] will be an important target for future modeling and experimental studies of climate change impacts and adaptation in <span class="gene">C3</span> crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Enhancing soybean photosynthetic CO2 assimilation using a cyanobacterial membrane protein, ictB.</h2>
			<h3>PMID:28273517  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hay WT(1), Bihmidine S(2), Mutlu N(3), Hoang KL(4), Awada T(2), Weeks DP(3), Clemente TE(5), Long SP(6).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="gene">C3</span> photosynthesis can suffer a severe loss in efficiency due to photorespiration and the lack of a carbon concentrating mechanism (CCM) such as those present in other plant species or cyanobacteria. Transgenic soybean (Glycine max cv. Thorne) plants constitutively expressing cyanobacterial ictB (inorganic carbon transporter B) gene were generated using Agrobacterium-mediated transformation. Although more recent data suggest that ictB does not actively transport HCO3-/CO2, there is nevertheless mounting evidence that transformation with this gene can increase higher plant photosynthesis. The hypothesis that expression of the ictB gene would improve photosynthesis, biomass production and seed <span class="trait">yield</span> in soybean was tested, in two independent replicated greenhouse and field trials. Results showed significant increases in photosynthetic CO2 uptake (Anet) and dry mass in transgenic relative to wild type (WT) control plants in both the greenhouse and field trials. Transgenic plants also showed increased photosynthetic rates and biomass production during a <span class="trait">drought</span> mimic study. The findings presented herein demonstrate that ictB, as a single-gene, contributes to enhancement in various <span class="trait">yield</span> parameters in a major commodity crop and point to the significant role that biotechnological approaches to increasing photosynthetic efficiency can play in helping to meet increased global demands for food.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Novel characteristics of cassava, Manihot esculenta Crantz, a reputed C3-C 4 intermediate photosynthesis species.</h2>
			<h3>PMID:24317831</h3>
				

				
			<p>
				作者：Angelov MN(1), Sun J, Byrd GT, Brown RH, Black CC.
				<br/> 
				年份：1993 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cassava plant, Manihot esculenta, grows exceptionally well in low <span class="trait">fertility</span> and <span class="trait">drought</span> prone environments, but the mechanisms that allow this growth are unknown. Earlier, and sometimes contradictory, work speculated about the presence of a C4-type photosynthesis in cassava leaves. In the present work we found no evidence for a <span class="gene">C4</span> metabolism in mature attached cassava leaves as indicated i) by the low, 2 to 8%, incorporation of (14)CO2 into <span class="gene">C4</span> organic acids in short time periods, 10 s, and the lack of (14)C transfer from <span class="gene">C4</span> acids to other compounds in (12)CO2, ii) by the lack of <span class="gene">C4</span> enzyme activity changes during leaf development and the inability to detect <span class="gene">C4</span> acid decarboxylases, and iii) by leaf CO2 compensation values between 49 and 65 μl of CO2 1(-1) and by other infrared gas exchange photosynthetic measurements. It is concluded that the leaf biochemistry of cassava follows the <span class="gene">C3</span> pathway of photosynthesis with no indication of a C3-C4 mechanism.However, cassava leaves exhibit several novel characteristics. Attached leaves have the ability to effectively partition carbon into sucrose with nearly 45% of the label in sucrose in about one min of (14)CO2 photosynthesis, contrasting with 34% in soybean (C3) and 25% in pigweed (C4). Cassava leaves displayed a strong preference for the synthesis of sucrose versus starch. Field grown cassava leaves exhibited high rates of photosynthesis and curvilinear responses to increasing sunlight irradiances with a tendency to saturate only at high irradiances, above 1500 μmol m(-2) s(-1). Morphologically, the cassava leaf has papillose epidermal cells on its lower mesophyll surface that form 'fence-like' arrangements encircling guard cells. It is proposed that the active synthesis of sugars has osmotic functions in the cassava plant and that the papillose epidermal cells function to maintain a healthy leaf water status in various environments.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression analysis of a stress-induced <span class="gene">GmIMT1</span> gene in soybean (Glycine max).</h2>
			<h3>PMID:24615045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang HT(1), Guo N(1), Zhao JM(1), Karthikeyan A(1), Xue D(1), Xue CC(1), Xu JY(1), Xu ZH(1), Gai JY(1), Xing H(1).
				<br/> 
				年份：2014 Feb 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Here, we aimed to clone and identify the <span class="gene">GmIMT1</span> gene related to the salt stress response in soybean. The full-length cDNA sequence of the <span class="gene">GmIMT1</span> gene was amplified in soybean using degenerate primers of Mesembrythmum crystallium. To understand the stress response, the <span class="gene">GmIMT1</span> gene was cloned and sequenced. Then, the expression vectors of the gene were constructed, and introduced into the model plant Arabidopsis thaliana through Agrobacterium mediated transformation, and the salt tolerance was analyzed in the transgenic plants. In addition, the expression patterns of <span class="gene">GmIMT1</span> gene in soybean were analyzed. The expression was examined in different organs (roots, leaves, flower seeds, and stem) and under different stress conditions <span class="trait">(drought,</span> high salt, low temperature, salicylic acid, ethane, abscisic acid, and methyl jasmonate) by real-time fluorescent quantitative polymerase chain reaction analysis. The results showed that the root, leaves, and stems exhibited high level of <span class="gene">GmIMT1</span> gene expression, whereas there was no expression in the seeds. In addition, the <span class="gene">GmIMT1</span> gene expression was upregulated under all stress conditions. Overall, the results clearly indicate that <span class="gene">GmIMT1</span> might be involved in multiple plant response pathways to the different environmental conditions. Furthermore transgenic plants exhibited higher salt-tolerance compared to wild type plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increased salt and <span class="trait">drought</span> tolerance by D-ononitol production in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:22079289  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ahn C(1), Park U, Park PB.
				<br/> 
				年份：2011 Dec 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The methylation of myo-inositol forms O-methyl inositol (D-ononitol) when plants are under abiotic stress in a reaction catalyzed by myo-inositol methyltransferase (IMT). D-Ononitol can serve as an osmoprotectant that prevents water loss in plants. We isolated the IMT cDNA from Glycine max and found by RT-PCR analysis that <span class="gene">GmIMT</span> transcripts are induced by <span class="trait">drought</span> and salinity stress treatments in the leaves of soybean seedlings. We confirmed the protein product of <span class="gene">GmIMT</span> and its substrate using a recombinant system in E. coli. Transgenic Arabidopsis plants over-expressing <span class="gene">GmIMT</span> displayed improved tolerance to dehydration stress treatment and to a lesser extent high salinity stress treatment. These results indicate that <span class="gene">GmIMT</span> is functional in heterologous Arabidopsis plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Exploring the role of Inositol 1,3,4-trisphosphate 5/6 kinase-2 <span class="gene">(GmITPK2)</span> as a dehydration and salinity stress regulator in Glycine max (L.) Merr. through heterologous expression in E. coli.</h2>
			<h3>PMID:29289899</h3>
				

				
			<p>
				作者：Marathe A(1), Krishnan V(1), Vinutha T(1), Dahuja A(1), Jolly M(1), Sachdev A(2).
				<br/> 
				年份：2017 Dec 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (PA) is implicative in a spectrum of biochemical and physiological processes involved in plant stress response. Inositol 1,3,4, Tris phosphate 5/6 kinase (ITPK), a polyphosphate kinase that converts Inositol 1,3,4 trisphosphate to Inositol 1,3,4,5/6 tetra phosphate, averting the inositol phosphate pool towards PA biosynthesis, is a key regulator that exists in four different isoforms in soybean. In the present study, in-silico analysis of the promoter region of ITPKs was done and among the four isoforms, promoter region of <span class="gene">GmITPK2</span> showed the presence of two MYB binding elements for <span class="trait">drought</span> inducibility and one for ABA response. Expression profiling through qRT-PCR under <span class="trait">drought</span> and salinity stress showed higher expression of <span class="gene">GmITPK2</span> isoform compared to the other members of the family. The study revealed <span class="gene">GmITPK2</span> as an early dehydration responsive gene which is also induced by dehydration and exogenous treatment with ABA. To evaluate the osmo-protective role of <span class="gene">GmITPK2,</span> attempts were made to assess the bacterial growth on Luria Broth media containing 200 mM NaCl, 16% PEG and 100 μM ABA, individually. The transformed E. coli BL21 (DE3) cells harbouring the <span class="gene">GmITPK2</span> gene depicted better growth on the media compared to the bacterial cells containing the vector alone. Similarly, the growth of the transformed cells in the liquid media containing 200 mM NaCl, 16% PEG and 100 μM ABA showed higher absorbance at 600 nm compared to control, at different time intervals. The <span class="gene">GmITPK2</span> recombinant E. coli cells showing tolerance to <span class="trait">drought</span> and salinity thus demonstrated the functional redundancy of the gene across taxa. The purity and specificity of the recombinant protein was assessed and confirmed through PAGE showing a band of ∼35 kDa on western blotting using Anti- Penta His- HRP conjugate antibody. To the best of our knowledge, the present study is the first report exemplifying the role of <span class="gene">GmITPK2</span> isoform in <span class="trait">drought</span> and salinity tolerance in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Water Stress Modulates Soybean Aphid Performance, Feeding Behavior, and Virus Transmission in Soybean</h2>
			<h3>PMID:27200027</h3>
				
					<h3>PMCID:4847208</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Nachappa </surname>  <given-names> Punya </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/301869/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Culkin </surname>  <given-names> Christopher T. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saya </surname>  <given-names> Peter M. </given-names>  <suffix> II </suffix>  </name>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Jinlong </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nalam </surname>  <given-names> Vamsi J. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/76969/overview"/>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To determine the level of water stress to be applied, a modified water stress procedure was used ( <xref rid="B64" ref-type="bibr"> Porcel and Ruiz-Lozano, 2004 </xref> ). Briefly, 500 g of Mastermix 830 soilless media (Mastermix, Quakertown, PA, USA) was weighed, and fully saturated with water in 6&#x02032;&#x02032; pots (Hummert International, Earth City, MO, USA). Saturated media was weighed and a Waterscout <sup> &#x000ae; </sup> SM100 soil moisture probe (Spectrum Technologies, Aurora, IL, USA) was used to determine percent volumetric water content (VWC). The saturated media was allowed to air dry until all moisture was lost. VWC and weight of water lost were monitored daily. A calibration curve based on average VWC and corresponding mass of water was computed based on which the volume of water to be used to maintain each water stress treatment was determined (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). Upon reaching <span class="gene">V1</span> or first trifoliate leaf stage, plants were exposed to three water-stress treatments: drought-stress conditions (25% field capacity or FC corresponding to 7.6% VWC), well-watered conditions (75% FC corresponding to 17.9% VWC), and water-saturated or flooding condition (100% FC corresponding to 24.8% VWC). The plants were maintained at such conditions for 3 days prior to the start of the experiment and for 7 days during the duration of the experiment. In order to maintain the water stress conditions, soil water content of the soilless media was measured daily (early evening) and re-watered to restore the soil water content to required levels. During the 24 h period between measurement and re-watering, the water content only decreased by about 2&#x02013;6% VWC in each of the treatments. Leaf water potential measurements were not performed because the protocol used has been previously shown to reduce water potential in <span class="trait">drought</span> stressed plants ( <xref rid="B64" ref-type="bibr"> Porcel and Ruiz-Lozano, 2004 </xref> ). With respect to the saturation treatment, previous studies have found that flooding or soil water saturation does not impact leaf water potential ( <xref rid="B59" ref-type="bibr"> Oosterhuis et al., 1990 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						There was significant interaction between water stress and aphid infestation on ABA marker genes, <italic> RD20A </italic> but not <italic> SCOF1 </italic> (Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S2 </xref> ). <italic> RD20A </italic> has been previously shown to be highly induced by <span class="trait">drought</span> stress, and indeed we found a significant increase in <italic> RD20A </italic> expression in response to <span class="trait">drought</span> stress ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). In contrast, saturation resulted in the suppression of <italic> RD20A </italic> expression ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). Feeding by non-viruliferous aphids increased <italic> RD20A </italic> expression under all treatments ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). There was a moderate increase in <italic> SCOF1 </italic> expression under <span class="trait">drought</span> stress compared to well-watered and saturated conditions albeit not statistically significant. Similar to <italic> RD20A </italic> , non-viruliferous aphid feeding up-regulated <italic> SCOF1 </italic> expression under <span class="trait">drought</span> stress ( <bold> Figure <xref ref-type="fig" rid="F6"> 6B </xref> </bold> ). The SA pathway marker, <italic> PR1 </italic> was affected by the interaction between water stress and aphid infestation but not <italic> <span class="gene">PAL2</span> </italic> , a gene involved in SA biosynthesis (Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S2 </xref> ). <italic> PR1 </italic> expression was down-regulated in uninfested drought-stressed plants and significantly up-regulated in saturated plants. In addition, <italic> PR1 </italic> expression was induced in response to both non-viruliferous and viruliferous aphid feeding in all treatments ( <bold> Figure <xref ref-type="fig" rid="F6"> 6C </xref> </bold> ). <italic> <span class="gene">PAL2</span> </italic> expression was reduced under <span class="trait">drought</span> stress, moreover, expression was down-regulated in response to feeding by viruliferous aphids ( <bold> Figure <xref ref-type="fig" rid="F6"> 6D </xref> </bold> ). Expression of the JA marker, <italic> JAR1 </italic> was also affected by the interaction between water stress and aphid infestation, but only water stress had a significant main effect (Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S2 </xref> ). <italic> JAR1 </italic> expression was suppressed in drought-stressed plants and in well-watered plants whereas, expression was significantly up-regulated in saturated plants ( <bold> Figure <xref ref-type="fig" rid="F6"> 6E </xref> </bold> ). The expression of <italic> AOS </italic> , involved in JA biosynthesis, was not affected by water stress (Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S2 </xref> ). However, the pattern of expression was similar to <italic> JAR1 </italic> . In general, feeding by either non-viruliferous or viruliferous aphids significantly up-regulated <italic> AOS </italic> expression as compared to the uninfested control ( <bold> Figure <xref ref-type="fig" rid="F6"> 6F </xref> </bold> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Agricultural practices altered soybean seed protein, oil, fatty acids, sugars, and minerals in the Midsouth USA</h2>
			<h3>PMID:25741347</h3>
				
					<h3>PMCID:4332302</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bellaloui </surname>  <given-names> Nacer </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/63229"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bruns </surname>  <given-names> H. Arnold </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/204010"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abbas </surname>  <given-names> Hamed K. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/132311"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mengistu </surname>  <given-names> Alemu </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/86055"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fisher </surname>  <given-names> Daniel K. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> Krishna N. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						An experiment was performed under field conditions in 2009 and 2010 on Sharkey clay (very-fine, smectitic, thermic Chromic Epiaquerts) and sandy loam (Beulah fine sandy loam, coarse-loamy, mixed, active, thermic Typic Dystrudepts) soils in Stoneville, MS, USA. The current research focused on seed nutrition only, and the agronomic component, including <span class="trait">yield,</span> was previously published ( <xref rid="B17" ref-type="bibr"> Bruns, 2011 </xref> ). Field management and growth conditions were described in detail previously ( <xref rid="B17" ref-type="bibr"> Bruns, 2011 </xref> ). Briefly, single-row plots were planted using an Almaco cone plot planter (Allen Machine Company, Nevada, IA, USA), and twin-row plots were planted using a four unit Monosem NG-3 (Monosem, Edwardsville, KS, USA) twin-row planter set on 102-cm centers and 25 cm between rows. In the Sharkey clay soil, soybean was planted in 2009 on 22 April, 20 May, and 17 June. In 2010, plantings occurred on 12 April, 11 May, and 2 June. In the sandy loam soil, PDs were 8 April, 11 May, and 8 June in 2009, and 14 April, 11 May, and 17 June in 2010. Cultivar Pioneer brand, 94B73, representative of most of the cultivars grown in the ESPS in the Midsouth was used. For weed control, a pre-plant application of trifluralin [2,6-dinitro- <italic> N </italic> , <italic> N </italic> -dipropyl-4-(trifluoromethyl)aniline] at 0.7 kg ai ha <sup> -1 </sup> was applied, followed by two post-emergence applications of metolachlor [2-chloro- <italic> N </italic> -(2-ethyl-6-methylphenyl)- <italic> N </italic> -(2-methoxy- 1-methylethyl) acetamide] and glyphosate [2-[(phosphonomethyl)amino]acetic acid] at growth stage <span class="gene">V2</span> to <span class="gene">V3</span> (two to three trifoliolates) and at V5 to V6 (five to six trifoliolates). To control fungus, pyraclostrobin (carbamic acid, [[[[1-(4-cholrophenyl)-H-pyrazol-3-yl]oxy]methyl]phenyl]methoxy-,methyl ester) was applied at factory label directions at V5 to V6. The experiments were furrow irrigated starting at R1 (beginning flowering) through R6 (full seed-fill), and an equivalent of &#x0223c;25 mm ha <sup> -1 </sup> water was applied at 10-days intervals. Soil analysis during the vegetative stage at both sites indicated that there were no nutrient deficiencies in soil in either year. Mature seeds at the R8 growth stage were collected, processed, and analyzed for seed composition constituent concentrations as described below.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and genetic diversity analysis of a new <span class="gene">P5CS</span> gene from common bean (Phaseolus vulgaris L.).</h2>
			<h3>PMID:20143043  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen J(1), Zhang X, Jing R, Blair MW, Mao X, Wang S.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Delta(1)-pyrroline-5-carboxylate synthetase (P5CS) is the rate-limiting enzyme involved in the biosynthesis of proline in plants. By the 3' rapid amplification of cDNA ends (3'-RACE) approach, a 2,246-bp cDNA sequence was obtained from common bean (Phaseolus vulgaris L.), denominated PvP5CS2 differing from another <span class="gene">P5CS</span> gene that we cloned previously from common bean (PvP5CS). The predicted amino acid sequence of PvP5CS2 has an overall 93.2% identity <span class="gene">GmP5CS</span> (Glycine max L. P5CS). However, PvP5CS2 shows only 83.7% identity in amino acid sequence to PvP5CS, suggesting PvP5CS2 represents a homolog of the soybean <span class="gene">P5CS</span> gene. Abundant indel (insertion and deletion events) and SNP (single nucleotide polymorphisms) were found in the cloned PvP5CS2 genome sequence when comparing 24 cultivated and 3 wild common bean accessions and these in turn reflected aspects of common bean evolution. Sequence alignment showed that genotypes from the same gene pool had similar nucleotide variation, while genotypes from different gene pools had distinctly different nucleotide variation for PvP5CS2. Furthermore, diversity along the gene sequence was not evenly distributed, being low in the glutamic-g-semialdehyde dehydrogenase catalyzing region, moderate in the Glu-5-kinase catalyzing region and high in the intervening region. Neutrality tests showed that PvP5CS2 was a conserved gene undergoing negative selection. A new marker (Pv97) was developed for genetic mapping of PvP5CS2 based on an indel between DOR364 and G19833 sequences and the gene was located between markers Bng126 and BMd045 on chromosome b01. The relationship of PvP5CS2 and a previously cloned pyrroline-5-carboxylate synthetase gene as well as the implications of this work on selecting for <span class="trait">drought</span> tolerance in common bean are discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Stress-Induced Declines in Soybean N2 Fixation Are Related to Nodule Sucrose Synthase Activity.</h2>
			<h3>PMID:12223754</h3>
				

				
			<p>
				作者：Gordon AJ(1), Minchin FR, Skot L, James CL.
				<br/> 
				年份：1997 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max L.) plants were subjected to a number of treatments <span class="trait">(drought,</span> 10 mM nitrate, 150 mM NaCl, shoot meristem removal, and removal of approximately 50% of the nodules) to test the hypothesis that metabolic responses contribute to the regulation of N2 fixation. Nitrogenase activity was correlated with the activity of nodule sucrose synthase (SS), but not with that of glutamine oxoglutarate amino transferase. Leghemoglobin levels and other enzyme activities were not significantly or consistently affected by the treatments. <span class="gene">SS</span> mRNA was greatly reduced in nodules of <span class="trait">drought-,</span> salt-, and nitrate-treated plants; however, this was not correlated with changes in soluble carbohydrate, starch, amino acids, or ureides. Leghemoglobin mRNA was only slightly affected by the treatments. The time course of <span class="trait">drought</span> stress showed a decline in the <span class="gene">SS</span> transcript level by 1 d, but levels of leghemoglobin, glutamine synthetase, and ascorbate peroxidase mRNA were not markedly affected by 4 d. <span class="gene">SS</span> activity at 4 d was reduced by 46%. We propose that N2 fixation in soybean nodules is mediated by both the oxygen-diffusion barrier and the potential to metabolize sucrose via SS. The response to environmental perturbation may involve down-regulation of the nodule <span class="gene">SS</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Abscisic acid induces a decline in nitrogen fixation that involves leghaemoglobin, but is independent of sucrose synthase activity.</h2>
			<h3>PMID:11283173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：González EM(1), Gálvez L, Arrese-Igor C.
				<br/> 
				年份：2001 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS) activity has been suggested to be a key point of regulation in nodule metabolism since this enzyme is down-regulated in response to different stresses which lead to decreased nitrogen fixation. In soybean, a dramatic decline of <span class="gene">SS</span> transcripts has been observed within 1 d from the onset of <span class="trait">drought.</span> Such a quick response suggests mediation by a signal transduction molecule. Abscisic acid (ABA) is a likely candidate to act as such a molecule as it mediates in a significant number of plant responses to environmental constraints. The hypothesis of ABA controlling nodule metabolism was approached in this work by assessing nodule responses to exogenous ABA supply in pea. Under the experimental conditions, ABA did not affect plant biomass, nodule numbers or dry weight. However, nitrogen fixation rate was reduced by 70% within 5 d and by 80% after 9 d leading to a reduced plant organic nitrogen content. Leghaemoglobin (Lb) content declined in parallel with that of nitrogen fixation. <span class="gene">SS</span> activity, however, was not affected by ABA treatment, and neither were the activities of the enzymes aspartate amino transferase, alkaline invertase, malate dehydrogenase, glutamate synthase, uridine diphosphoglucose pyrophosphorylase, isocitrate dehydrogenase, and glutamine synthetase. Nodule bacteroid-soluble protein content was reduced in nodules only after 9 d of ABA treatment. These results do not support the hypothesis that ABA directly regulates <span class="gene">SS</span> activity. However, they do suggest the occurrence of at least two different control pathways in nodules under environmental constraints, which include ABA being involved in a Lb/oxygen-related control of nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Modeling the impacts of temperature and precipitation changes on soil CO2 fluxes from a Switchgrass stand recently converted from cropland.</h2>
			<h3>PMID:27155405  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lai L(1), Kumar S(2), Chintala R(3), Owens VN(4), Clay D(3), Schumacher J(3), Nizami AS(5), Lee SS(6), Rafique R(7).
				<br/> 
				年份：2016 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Switchgrass (Panicum virgatum L.) is a perennial <span class="gene">C4</span> grass native to North America and successfully adapted to diverse environmental conditions. It offers the potential to reduce soil surface carbon dioxide (CO2) fluxes and mitigate climate change. However, information on how these CO2 fluxes respond to changing climate is still lacking. In this study, CO2 fluxes were monitored continuously from 2011 through 2014 using high frequency measurements from Switchgrass land seeded in 2008 on an experimental site that has been previously used for soybean (Glycine max L.) in South Dakota, USA. DAYCENT, a process-based model, was used to simulate CO2 fluxes. An improved methodology CPTE [Combining Parameter estimation (PEST) with "Trial and Error" method] was used to calibrate DAYCENT. The calibrated DAYCENT model was used for simulating future CO2 emissions based on different climate change scenarios. This study showed that: (i) the measured soil CO2 fluxes from Switchgrass land were higher for 2012 which was a <span class="trait">drought</span> year, and these fluxes when simulated using DAYCENT for long-term (2015-2070) provided a pattern of polynomial curve; (ii) the simulated CO2 fluxes provided different patterns with temperature and precipitation changes in a long-term, (iii) the future CO2 fluxes from Switchgrass land under different changing climate scenarios were not significantly different, therefore, it can be concluded that Switchgrass grown for longer durations could reduce changes in CO2 fluxes from soil as a result of temperature and precipitation changes to some extent.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GMCHI, cloned from soybean [Glycine max (L.) Meer.], enhances survival in transgenic Arabidopsis under abiotic stress.</h2>
			<h3>PMID:18825384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cheng L(1), Huan S, Sheng Y, Hua X, Shu Q, Song S, Jing X.
				<br/> 
				年份：2009 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants respond to cold stress by modifying the expression of a battery of cold-responsive genes. Using cDNA-AFLP techniques, <span class="gene">GMCHI</span> (G lycine m ax chilling-inducible) (accession no. EU699765) was isolated from the embryonic axis of a chilling-resistant cultivar of soybean seed imbibed at 4 degrees C for 24 h. The full-length <span class="gene">GMCHI</span> cDNA which consisted of a single open reading frame (ORF) encoded a putative polypeptide of 129 amino acids. Sequence analysis revealed neither significant similarity of <span class="gene">GMCHI</span> to known proteins, nor any conserved domains found. Soybean seed imbibed at 4 degrees C dramatically enhanced transcript level of <span class="gene">GMCHI</span> after 1 h, and reached a maximum at 18 h, while the expression was only detected in the embryonic axis. <span class="gene">GMCHI</span> expression was strongly induced by treatment with ABA and PEG, but weakly by 250 mM NaCl which suggests that <span class="gene">GMCHI</span> is probably regulated by ABA-dependent signal transduction pathway during cold acclimation. Overexpression of <span class="gene">GMCHI</span> in Arabidopsis under the control of CaMV35S promoter enhanced the tolerance to cold, <span class="trait">drought</span> and NaCl stresses. Therefore, <span class="gene">GMCHI</span> may play an important role in the adaptation of chilling-resistant soybean seed to chilling imbibition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsLRPK, a novel cold-activated leucine-rich repeat receptor-like protein kinase from Glycine soja, is a positive regulator to cold stress tolerance.</h2>
			<h3>PMID:24388511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang L(1), Wu K(1), Gao P(2), Liu X(1), Li G(3), Wu Z(4).
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant LRR-RLKs serve as protein interaction platforms, and as regulatory modules of protein activation. Here, we report the isolation of a novel plant-specific <span class="gene">LRR-RLK</span> from Glycine soja (termed GsLRPK) by differential screening. GsLRPK expression was cold-inducible and shows Ser/Thr protein kinase activity. Subcellular localization studies using GFP fusion protein indicated that GsLRPK is localized in the plasma membrane. Real-time PCR analysis indicated that temperature, salt, <span class="trait">drought,</span> and ABA treatment can alter GsLRPK gene transcription in G. soja. However, just protein induced by cold stress not by salinity and ABA treatment in tobacco was found to possess kinase activity. Furthermore, we found that overexpression of GsLRPK in yeast and Arabidopsis can enhance <span class="trait">resistance</span> to cold stress and increase the expression of a number of cold responsive gene markers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolutionary history and stress regulation of the lectin superfamily in higher plants</h2>
			<h3>PMID:20236552</h3>
				
					<h3>PMCID:2846932</h3>
					<h4>期刊:BMC Evolutionary Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Jiang </surname>  <given-names> Shu-Ye </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> shuye@tll.org.sg </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Ma </surname>  <given-names> Zhigang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhigang@tll.org.sg </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A3">  <name>  <surname> Ramachandran </surname>  <given-names> Srinivasan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sri@tll.org.sg </email>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 3 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We try to further elucidate the biological benefits to drive the expansion and retention of the lectin superfamily by surveying the effect of tandem and/or segmental duplications on the expression divergence under biotic and abiotic stresses. Previous reports have showed that tandemly duplicated genes tend to be involved in biotic and abiotic stresses [ <xref ref-type="bibr" rid="B72"> 72 </xref> , <xref ref-type="bibr" rid="B83"> 83 </xref> ]. Many of the <span class="gene">RLK</span> family members are B-lectin or Lectin_legB domain-containing proteins in rice and <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B68"> 68 </xref> ]. Interestingly, the expression data also support the importance of this family in stress response [ <xref ref-type="bibr" rid="B92"> 92 </xref> ]. These results further confirmed the co-evolution of B-lectin/Lectin_legB and kinase domains. Our expression data suggest a link between biotic/abiotic stresses and tandem/segmental duplications in lectin families and also suggest that not only tandem but also segmental duplications of lectin genes may be regarded as drivers for plants to adapt various environmental stresses through duplication followed by expression divergence, thus, providing an explanation for why some of lectin families exhibit large expansion. Highly divergent expression profiles also demonstrate that each member of this gene superfamily may play specialized roles in a specific stress condition and function as a regulator of various environmental factors such as cold <span class="trait">drought</span> and high salinity stresses as well as biotic stresses. The detection of tissue-specifically expressed genes further demonstrated the comprehensive biological functions of this superfamily.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the Hsf family in soybean and functional identification of <span class="gene">GmHsf-34</span> involvement in <span class="trait">drought</span> and heat stresses.</h2>
			<h3>PMID:25416131  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li PS, Yu TF, He GH, Chen M, Zhou YB, Chai SC(1), Xu ZS, Ma YZ.
				<br/> 
				年份：2014 Nov 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: High temperature affects organism growth and metabolic activity. Heat shock transcription factors (Hsfs) are key regulators in heat shock response in eukaryotes and prokaryotes. Under high temperature conditions, Hsfs activate heat shock proteins (Hsps) by combining with heat stress elements (HSEs) in their promoters, leading to defense of heat stress. Since the first plant Hsf gene was identified in tomato, several plant Hsf family genes have been thoroughly characterized. Although soybean (Glycine max), an important oilseed crops, genome sequences have been available, the Hsf family genes in soybean have not been characterized accurately. RESULT: We analyzed the Hsf genetic structures and protein function domains using the GSDS, Pfam, SMART, PredictNLS, and NetNES online tools. The genome scanning of dicots (soybean and Arabidopsis) and monocots (rice and maize) revealed that the whole-genome replication occurred twice in soybean evolution. The plant Hsfs were classified into 3 classes and 16 subclasses according to protein structure domains. The A8 and B3 subclasses existed only in dicots and the A9 and <span class="gene">C2</span> occurred only in monocots. Thirty eight soybean Hsfs were systematically identified and grouped into 3 classes and 12 subclasses, and located on 15 soybean chromosomes. The promoter regions of the soybean Hsfs contained cis-elements that likely participate in <span class="trait">drought,</span> low temperature, and ABA stress responses. There were large differences among Hsfs based on transcriptional levels under the stress conditions. The transcriptional levels of the A1 and A2 subclass genes were extraordinarily high. In addition, differences in the expression levels occurred for each gene in the different organs and at the different developmental stages. Several genes were chosen to determine their subcellular localizations and functions. The subcellular localization results revealed that <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> were located in the nucleus. Overexpression of the <span class="gene">GmHsf-34</span> gene improved the tolerances to <span class="trait">drought</span> and heat stresses in Arabidopsis plants. CONCLUSIONS: This present investigation of the quantity, structural features, expression characteristics, subcellular localizations, and functional roles provides a scientific basis for further research on soybean Hsf functions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of quantitative trait loci for <span class="trait">yield</span> and <span class="trait">drought</span> tolerance traits in soybean using a recombinant inbred line population.</h2>
			<h3>PMID:19723246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Yu D, Fu S.
				<br/> 
				年份：2009 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the genetic basis of <span class="trait">drought</span> tolerance in soybean (Glycine max L. Merr.) a recombinant inbred population with 184 F(2:7:11) lines developed from a cross between Kefeng1 <span class="trait">(drought</span> tolerant) and Nannong1138-2 <span class="trait">(drought</span> sensitive) were tested under water-stressed and well-watered conditions in field and greenhouse trials. Traits measured included leaf wilting coefficient, excised leaf water loss and relative water content as indicators of plant water status and seed <span class="trait">yield.</span> A total of 40 quantitative trait loci (QTLs) were identified: 17 for leaf water status traits under <span class="trait">drought</span> stress and 23 for seed <span class="trait">yield</span> under well-watered and drought-stressed conditions in both field and greenhouse trials. Two seed <span class="trait">yield</span> QTLs were detected under both well-watered and drought-stressed conditions in the field on molecular linkage group H and D1b, while two seed <span class="trait">yield</span> QTLs on molecular linkage group <span class="gene">C2</span> were found under greenhouse conditions. Several QTLs for traits associated with plant water status were identified in both field and greenhouse trials, including two leaf wilting coefficient QTLs on molecular linkage group A2 and one excised leaf water loss QTL on molecular linkage group H. Phenotypic correlations of traits suggested several QTLs had pleiotropic or location-linked associations. These results will help to elucidate the genetic basis of <span class="trait">drought</span> tolerance in soybean, and could be incorporated into a marker-assisted selection breeding program to develop high-yielding soybean cultivars with improved tolerance to <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping QTLs for seed <span class="trait">yield</span> and <span class="trait">drought</span> susceptibility index in soybean (Glycine max L.) across different environments.</h2>
			<h3>PMID:20129399  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Wang M, Fu S, Yu D.
				<br/> 
				年份：2009 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress has long been a major constraint in maintaining <span class="trait">yield</span> stability of soybean (Glycine max (L.) Merr.) in rainfed ecosystems. The identification of consistent quantitative trait loci (QTL) involving seed <span class="trait">yield</span> per plant (YP) and <span class="trait">drought</span> susceptibility index (DSI) in a population across different environments would therefore be important in molecular marker-assisted breeding of soybean cultivars suitable for rainfed regions. The YP of a recombinant line population of 184 F(2:7:11) lines from a cross of Kefeng1 and Nannong1138-2 was studied under water-stressed (WS) and well-watered (WW) conditions in field (F) and greenhouse (G) trials, and DSI for <span class="trait">yield</span> was calculated in two trials. Nineteen QTLs associated with YP-WS and YP-WW, and 10 QTLs associated with DSI, were identified. Comparison of these QTL locations with previous findings showed that the majority of these regions control one or more traits related to <span class="trait">yield</span> and other agronomic traits. One QTL on molecular linkage group (MLG) K for YP-F, and two QTLs on MLG <span class="gene">C2</span> for YP-G, remained constant across different water regimes. The regions on MLG <span class="gene">C2</span> for YP-WW-F and MLG H for YP-WS-F had a pleiotropic effect on DSI-F, and MLG A1 for YP-WS-G had a pleiotropic effect on DSI-G. The identification of consistent QTLs for YP and DSI across different environments will significantly improve the efficiency of selecting for <span class="trait">drought</span> tolerance in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						2009 Institute of Genetics and Developmental Biology and the Genetics Society of China. Published by Elsevier Ltd. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Implications of ethylene biosynthesis and signaling in soybean <span class="trait">drought</span> stress tolerance</h2>
			<h3>PMID:26335593</h3>
				
					<h3>PMCID:4557918</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Arraes </surname>  <given-names> Fabricio Barbosa Monteiro </given-names>  </name>  <address>  <email> fabricio.arraes@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Beneventi </surname>  <given-names> Magda Aparecida </given-names>  </name>  <address>  <email> magbeneventi@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lisei de Sa </surname>  <given-names> Maria Eugenia </given-names>  </name>  <address>  <email> maria-eugenia.sa@colaborador.embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Paixao </surname>  <given-names> Joaquin Felipe Roca </given-names>  </name>  <address>  <email> joaquinfrp@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Albuquerque </surname>  <given-names> Erika Valeria Saliba </given-names>  </name>  <address>  <email> erika.albuquerque@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marin </surname>  <given-names> Silvana Regina Rockenbach </given-names>  </name>  <address>  <email> silvana.marin@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Purgatto </surname>  <given-names> Eduardo </given-names>  </name>  <address>  <email> epurgatt@usp.br </email>  </address>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <address>  <email> alexandre.nepomuceno@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Grossi-de-Sa </surname>  <given-names> Maria Fatima </given-names>  </name>  <address>  <phone> +55 61 3448 4705 </phone>  <email> fatima.grossi@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <aff id="Aff1">  <label/> Federal University of Rio Grande do Sul, Campus do Vale, Av. Bento Gon&#x000e7;alves 9500, Postal Code 15005 CEP 91501&#x02013;970, Porto Alegre, RS Brazil  </aff>  <aff id="Aff2">  <label/> Embrapa Genetic Resources and Biotechnology, PqEB, Av. W5-Norte, Postal Code 02372 CEP 70770&#x02013;910, Brasilia, DF Brazil  </aff>  <aff id="Aff3">  <label/> Brasilia University &#x02013; Biology Institute, Brasilia, DF Brazil  </aff>  <aff id="Aff4">  <label/> Agricultural Research Company of Minas Gerais State, Rua Afonso Rato 1301, Postal Code 311 CEP 38001&#x02013;970, Uberaba, MG Brazil  </aff>  <aff id="Aff5">  <label/> Embrapa Soybean, Rodovia Carlos Jo&#x000e3;o Strass, SN, Acesso Orlando Amaral, Distrito de Warta, Postal Code 231 CEP 86001&#x02013;970, Londrina, PR Brazil  </aff>  <aff id="Aff6">  <label/> Food Chemistry and Biochemistry Laboratory, Sao Paulo University, Av. Lineu Prestes 580, Bloco 14, Cidade Universitaria, CEP 05508&#x02013;000, Sao Paulo, SP Brazil  </aff>  <aff id="Aff7">  <label/> Catholic University of Brasilia, SGAN 916, Modulo B, Av. W5, Asa Norte, CEP 70790&#x02013;160, Brasilia, DF Brazil  </aff>
				<br/> 
				年份：<day> 20 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The expression of the genes <italic> MAT </italic> , <italic> <span class="gene">ACS</span> </italic> and <italic> <span class="gene">ACO</span> </italic> were found to have the same differential trends as the data obtained <italic> in silico, </italic> although with variations in the expression profiles (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a, b and c </xref> . Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S8a, S8b and S8c). This result could be due to limitations in the construction of the GENOSOJA subtractive libraries and general experimental variations. As an example, the expression of <italic> <span class="gene">ACS</span> </italic> is different in both cultivars and tissues with RT-qPCR, but it was detected in only the transcriptome of the roots of EMBRAPA48. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Expression of Ethylene-Related Genes in Soybean Under <span class="trait">Drought</span> Stress Conditions. The graphs show the expression levels, obtained by RT-qPCR, of five soybean genes related to ethylene biosynthesis [ <italic> MAT </italic> ( <bold> a </bold> ), <italic> <span class="gene">ACS</span> </italic> ( <bold> b </bold> ) and <italic> <span class="gene">ACO</span> </italic> ( <bold> c </bold> )] and ethylene signal transduction [ <italic> ETR </italic> ( <bold> d </bold> ) and <italic> CTR </italic> ( <bold> e </bold> )]. The expression of these genes in the experiment was compared in roots and leaves of soybean cultivars BR16 and EMBRAPA48 after different durations of <span class="trait">drought</span> stress. In the column graphs <italic> 1 </italic> and <italic> 2 </italic> , the statistical analysis was performed by comparing similar tissues in both cultivars under the same conditions of <span class="trait">drought</span> stress (same durations). The asterisks represent the level of statistical significance: ( <bold> * </bold> ) <italic> p </italic> &#x02009;&#x02264;&#x02009;0.05; ( <bold> ** </bold> ) 0.01&#x02009;&#x02264;&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05; ( <bold> *** </bold> ) 0.001&#x02009;&#x02264;&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01. Each dot represents the average amount (&#x000b1; standard error) of three experimental replicates (same sample) in three biological samples (different plants), totaling nine replicates. The standard error is not presented with some of the dots because their absolute values are lower than the scale. After normalization based on housekeeping genes, the values given in the graph are relative to the lowest expression, whose value was set at 1 (one). Information about the target genes is presented (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> : Method S1)
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Responses of nitrogen metabolism and seed nutrition to <span class="trait">drought</span> stress in soybean genotypes differing in slow-wilting phenotype <xref ref-type="fn" rid="fn0001"> <sup> 1 </sup> </xref></h2>
			<h3>PMID:24339829</h3>
				
					<h3>PMCID:3857554</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bellaloui </surname>  <given-names> Nacer </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gillen </surname>  <given-names> Anne M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mengistu </surname>  <given-names> Alemu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kebede </surname>  <given-names> Hirut </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fisher </surname>  <given-names> Daniel K. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> James R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> Krishna N. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A repeated greenhouse experiment was conducted at USDA-ARS, Crop Genetics Research Unit. Soybean genotypes in maturity group (MG) VI and differing in wilting were used. Genotypes were: NC-Roy (fast wilting), Boggs (intermediate in wilting), and slow-wilting (SLW) genotypes: NTCPR94-5157 and N04-9646. Soybean seeds were germinated in flat trays in vermiculite. Uniform size seedlings at about the <span class="gene">V1</span> stage (fully developed unifoliate leaves and one unrolled trifoliate leaf) were transplanted into 9.45 L pots filled with field soil. Soil characteristics were silty loam with pH 6.3, 1.1% organic matter, a cation exchange capacity of 15 cmol/kg, and soil textural fractions of 26% sand, 56% silt, and 18% clay. The soil contained an abundant native population of <italic> B. japonicum </italic> . Soil in pots, weighed, saturated with deionized water, and drained. The pots were then weighed to obtain the water field capacity using Watermark 200SS soil sensors inserted in the pots and read daily with a Soil Moisture Meter (Irrometer Company, Inc., Riverside, California, USA). Well watered plants were kept between &#x02212;15 and &#x02212;20 kPa (this was considered field capacity for the control plants), and <span class="trait">drought</span> stressed plants were kept between &#x02212;90 and &#x02212;100 kPa (Bellaloui et al., <xref ref-type="bibr" rid="B11"> 2010a </xref> ). Plants were considered fully matured when they reached R8 (full maturity) according to Fehr and Caviness ( <xref ref-type="bibr" rid="B26"> 1977 </xref> ). At full maturity, 95% of pods reached full maturity). Three replicates were used in each treatment, and experiment was repeated. Therefore, the total replicates used in this experiment is 6. Each pot had three plants. Greenhouse temperature conditions were maintained at 34 &#x000b1; 9&#x000b0;C during the day and 28 &#x000b1; 7&#x000b0;C at night. Photosynthetic photon flux density (PPFD) during the day of about 800-2300 &#x003bc;mol&#x000b7;m <sup> &#x02212;2 </sup> &#x000b7;s <sup> &#x02212;1 </sup> was measured by a Quantum Meter (Spectrum Technologies, Inc., Aurora. Illinois, USA). The wide range of light intensity reflects a bright, sunny, or cloudy day. The two experiments were conducted simultaneously in two different bays during the normal growing season (from April to September) to simulate the growing season photoperiod of soybean production in the midsouth USA. The fully expanded leaves at seed-fill stages (R5&#x02013;R6) were analyzed for NF, NA, and mineral nutrition. Seed fill stage was chosen as it is considered as the most critical stage for soybean for seed evaluation, and any impact on nutrients uptake or movement from leaves to seed will impact seed quality. Mature seeds at R8 were harvested and analyzed for seed protein, oil, fatty acids, sugars, and minerals. In a separate preliminary experiment to further investigate the role of K and B (among the responsive nutrients to SLW trait under <span class="trait">drought),</span> a foliar K application at a rate of 1.75% K <sub> 2 </sub> SO <sub> 4 </sub> and foliar B application at a rate of 1.1 kg ha <sup> &#x02212;1 </sup> H <sub> 3 </sub> BO <sub> 3 </sub> were made at seed-fill stages (R5&#x02013;R6). Leaves were collected for K and B analysis 2 weeks after K and B application. Foliar K and B were applied only to SLW genotypes under moderate <span class="trait">drought</span> stress (&#x02212;99 to &#x02212;100 kPa) and severe <span class="trait">drought</span> stress (soil water potential = &#x02212;199 kPa).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> SlDEAD31 </italic> , a Putative DEAD-Box RNA Helicase Gene, Regulates Salt and <span class="trait">Drought</span> Tolerance and Stress-Related Genes in Tomato</h2>
			<h3>PMID:26241658</h3>
				
					<h3>PMCID:4524616</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Mingku </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Guoping </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Tingting </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lingling </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jianling </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Zhiping </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Zongli </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 2 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The expression of multiple abiotic stress-related genes was significantly upregulated in the <italic> SlDEAD31 </italic> -overexpressing tomato plants ( <xref rid="pone.0133849.g007" ref-type="fig"> Fig 7 </xref> ), something considered to be advantageous for the <span class="trait">resistance</span> to abiotic stress, also indicating the protective role of <italic> SlDEAD31 </italic> against environmental stresses. The expression of stress-related genes results in the accumulation of metabolites and the alteration of biochemical and physiological pathways that are vital for the plant adaptation to adverse stress conditions. Our results are in a good agreement with previous report, in which enhanced tolerance to various stresses is mainly attributed to increased expression of biotic and abiotic stress-related genes in the <italic> OsBIRH1- </italic> overexpressing plants [ <xref rid="pone.0133849.ref017" ref-type="bibr"> 17 </xref> ]. In plants, biotic and abiotic stresses can trigger the generation of ROS, which include hydrogen peroxide (H <sub> 2 </sub> O <sub> 2 </sub> ) that can oxidize and damage cellular components [ <xref rid="pone.0133849.ref042" ref-type="bibr"> 42 </xref> ]. The ROS defense mechanisms involve the synthesis of antioxidant compounds (such as AsA) and antioxidant enzymes (such as CAT1/2 and APX2). An increase in AsA content can confer tolerance to various stresses including salt, ozone, and chilling [ <xref rid="pone.0133849.ref030" ref-type="bibr"> 30 </xref> ]. The upregulated expression of <italic> <span class="gene">GME2</span> </italic> was detected in the transgenic plants suggesting an elevated AsA content, and thus conferring tolerance to salt stress in <italic> SlDEAD31 </italic> -overexpressing plants. In this context, CAT and APX can detoxify ROS by converting H <sub> 2 </sub> O <sub> 2 </sub> to H <sub> 2 </sub> O in plant cells under oxidative stress [ <xref rid="pone.0133849.ref031" ref-type="bibr"> 31 </xref> , <xref rid="pone.0133849.ref032" ref-type="bibr"> 32 </xref> ]. Therefore, the enhanced tolerance observed in transgenic tomato plants might be due to the upregulation of <italic> APX2 </italic> and <italic> Cat1 </italic> / <italic> 2 </italic> , and thus an increased protection against ROS-induced damages. In addition, a significant upregulation in <italic> ERF1 </italic> expression was observed in transgenic plants. It has been shown that overexpression of <italic> ERF1 </italic> provides improved <span class="trait">drought</span> tolerance in transgenic tomato [ <xref rid="pone.0133849.ref034" ref-type="bibr"> 34 </xref> ]. We have reported previously that overexpression of <italic> SlERF5 </italic> in tomato promotes adaptation to <span class="trait">drought</span> and salt tolerance [ <xref rid="pone.0133849.ref043" ref-type="bibr"> 43 </xref> ]. Furthermore, <italic> ER5 </italic> , an ethylene-responsive <italic> LEA </italic> gene confers desiccation tolerance [ <xref rid="pone.0133849.ref033" ref-type="bibr"> 33 </xref> ], was also increased in transgenic plants, indicating a less damage of the transgenic lines under abiotic stress. Interestingly, transcripts encoding pathogenesis-related proteins, <italic> PR1 </italic> and <italic> PR5 </italic> , were also upregulated in transgenic lines. Similarly, they were also elevated in the O <italic> sBIRH1 </italic> -overexpressing <italic> Arabidopsis </italic> plants [ <xref rid="pone.0133849.ref017" ref-type="bibr"> 17 </xref> ]. Thus our results imply that <italic> SlDEAD31 </italic> , besides having a role in plant adaptation to abiotic stress, may also integrates signals derived from both biotic and abiotic stresses in tomato, although this remains to be elucidated in detail. Collectively, our data indicate that biotic or abiotic stresses can induce the expression of <italic> SlDEAD31 </italic> , leading to the upregulation of transcripts encoding functional and regulatory proteins involved in antistress metabolism. Therefore, their consistent upregulated expression likely increases the expression of downstream genes, probably enhancing integrative tolerance to multiple stresses due to cross-talk between various environmental stresses, especially salt and <span class="trait">drought</span> [ <xref rid="pone.0133849.ref044" ref-type="bibr"> 44 </xref> , <xref rid="pone.0133849.ref045" ref-type="bibr"> 45 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of a chickpea (Cicer arietinum L.) NAC family gene, CarNAC5, which is both developmentally- and stress-regulated.</h2>
			<h3>PMID:19800808  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Peng H(1), Cheng HY, Yu XW, Shi QH, Zhang H, Li JG, Ma H.
				<br/> 
				年份：2009 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been documented that the plant-specific NAC (for NAM, ATAF1,2 and CUC2) transcription factors play an important role in plant development and stress responses. In this study, a chickpea NAC gene CarNAC5 (for Cicer arietinum L. NAC gene 5) was isolated from a cDNA library from chickpea leaves treated by polyethylene glycol (PEG). CarNAC5, as a single/low copy gene, contained three exons and two introns within genomic DNA sequence and encoded a polypeptide with 291 amino acids. CarNAC5 protein had a conserved NAC domain in the N-terminus and showed high similarity to other NACs, especially ATAF subgroup members. The CarNAC5:GFP fusion protein was localized in the nucleus of onion epidermal cells. Furthermore, CarNAC5 protein activated the reporter genes LacZ and <span class="gene">HIS3</span> in yeast. The transactivation activity was mapped to the C-terminal region. The transcripts of CarNAC5 appeared in many chickpea tissues including seedling leaves, stems, roots, flowers, seeds and pods, but mostly accumulated in flowers. Meanwhile, CarNAC5 was strongly expressed during seed maturation and in embryos of the early germinating seeds. It was also significantly induced by <span class="trait">drought,</span> heat, wounding, salicylic acid (SA), and indole-3-acetic acid (IAA) treatments. Our results suggest that CarNAC5 encodes a novel NAC-domain protein and acts as a transcriptional activator involved in plant developmental regulation and various stress responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">PIP</span> aquaporin gene expression in arbuscular mycorrhizal Glycine max and Lactuca sativa plants in relation to <span class="trait">drought</span> stress tolerance.</h2>
			<h3>PMID:16514562  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Porcel R(1), Aroca R, Azcón R, Ruiz-Lozano JM.
				<br/> 
				年份：2006 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Although the discovery of aquaporins in plants has resulted in a paradigm shift in the understanding of plant water relations, the relationship between aquaporins and plant responses to <span class="trait">drought</span> still remains elusive. Moreover, the contribution of aquaporin genes to the enhanced tolerance to <span class="trait">drought</span> in arbuscular mycorrhisal (AM) plants has never been investigated. Therefore, we studied, at a molecular level, whether the expression of aquaporin-encoding genes in roots is altered by the AM symbiosis as a mechanism to enhance host plant tolerance to water deficit. In this study, genes encoding plasma membrane aquaporins (PIPs) from soybean and lettuce were cloned and their expression pattern studied in AM and nonAM plants cultivated under well-watered or <span class="trait">drought</span> stressed conditions. Results showed that AM plants responded to <span class="trait">drought</span> stress by down-regulating the expression of the <span class="gene">PIP</span> genes studied and anticipating its down-regulation as compared to nonAM plants. The possible physiological implications of this down-regulation of <span class="gene">PIP</span> genes as a mechanism to decrease membrane water permeability and to allow cellular water conservation is further discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmFDL19</span> enhances tolerance to <span class="trait">drought</span> and salt stresses in soybean.</h2>
			<h3>PMID:28640834  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li Y(1)(2), Chen Q(2), Nan H(2), Li X(2)(3), Lu S(2)(4), Zhao X(2)(4), Liu B(2)(4), Guo C(1), Kong F(2)(4), Cao D(2).
				<br/> 
				年份：2017 Jun 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The basic leucine zipper (bZIP) family of transcription factors plays an important role in the growth and developmental process as well as responds to various abiotic stresses, such as <span class="trait">drought</span> and high salinity. Our previous work identified <span class="gene">GmFDL19,</span> a bZIP transcription factor, as a flowering promoter in soybean, and the overexpression of <span class="gene">GmFDL19</span> caused early flowering in transgenic soybean plants. Here, we report that <span class="gene">GmFDL19</span> also enhances tolerance to <span class="trait">drought</span> and salt stress in soybean. <span class="gene">GmFDL19</span> was determined to be a group A member, and its transcription expression was highly induced by abscisic acid (ABA), polyethylene glycol (PEG 6000) and high salt stresses. Overexpression of <span class="gene">GmFDL19</span> in soybean enhanced <span class="trait">drought</span> and salt tolerance at the seedling stage. The relative plant height (RPH) and relative shoot dry weight (RSDW) of transgenic plants were significantly higher than those of the WT after PEG and salt treatments. In addition, the germination rate and plant height of the transgenic soybean were also significantly higher than that of WT plants after various salt treatments. Furthermore, we also found that <span class="gene">GmFDL19</span> could reduce the accumulation of Na+ ion content and up-regulate the expression of several ABA/stress-responsive genes in transgenic soybean. We also found that <span class="gene">GmFDL19</span> overexpression increased the activities of several antioxidative enzyme and chlorophyll content but reduced malondialdehyde content. These results suggested that <span class="gene">GmFDL19</span> is involved in soybean abiotic stress responses and has potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression of three fatty acid desaturase genes from cold-sensitive lima bean (Phaseolus lunatus L.).</h2>
			<h3>PMID:20953666  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang YM(1), Wang CC, Hu HH, Yang L.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The critical chilling temperature of cold-sensitive lima bean (Phaseolus lunatus L.) seedlings was determined to be approx. 8°C. The full-length cDNAs of PlSAD, PlFAD2, and PlFAD3 encoding three fatty acid desaturases were isolated from lima bean leaves. The PlSAD open reading frame (ORF) had 1,317 bp and a single intron of 601 bp, encoding a polypeptide of 438 amino acids that showing 86% homology with the plastidial stearoyl-acyl carrier protein desaturase in soybean. The PlFAD2 ORF contained 1,059 bp and was uninterrupted, encoding a polypeptide of 352 amino acids having 85% identity with the microsomal omega-6 desaturase <span class="gene">GmFAD2-3</span> in soybean. PlFAD3 ORF had 1,116 bp and seven introns, encoding a polypeptide of 371 amino acids showing 91% homology with the microsomal omega-3 desaturase <span class="gene">FAD3</span> in cowpea. PlSAD and PlFAD2 were expressed highly in leaves but poorly in roots and stems, while PlFAD3 was expressed in three tissues. All three genes were significantly induced in leaves by <span class="trait">drought.</span> PlSAD and PlFAD3 in leaves were down-regulated by high temperature and salinity, whereas PlFAD2 was up-regulated by the two stresses. The expression patterns of the three genes in lima bean leaves under suboptimal temperature, 18°C, were different from those under 8°C. The down-regulation of PlSAD transcript at chilling temperature might be an important factor contributing to chilling susceptibility for lima bean plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Early transcriptional response of soybean contrasting accessions to root dehydration.</h2>
			<h3>PMID:24349513  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ferreira Neto JR(1), Pandolfi V(2), Guimaraes FC(3), Benko-Iseppon AM(2), Romero C(3), Silva RL(1), Rodrigues FA(3), Abdelnoor RV(3), Nepomuceno AL(4), Kido EA(1).
				<br/> 
				年份：2013 Dec 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a significant constraint to <span class="trait">yield</span> increase in soybean. The early perception of water deprivation is critical for recruitment of genes that promote plant tolerance. DeepSuperSAGE libraries, including one control and a bulk of six stress times imposed (from 25 to 150 min of root dehydration) for drought-tolerant and sensitive soybean accessions, allowed to identify new molecular targets for <span class="trait">drought</span> tolerance. The survey uncovered 120,770 unique transcripts expressed by the contrasting accessions. Of these, 57,610 aligned with known cDNA sequences, allowing the annotation of 32,373 unitags. A total of 1,127 unitags were up-regulated only in the tolerant accession, whereas 1,557 were up-regulated in both as compared to their controls. An expression profile concerning the most representative Gene Ontology (GO) categories for the tolerant accession revealed the expression "protein binding" as the most represented for "Molecular Function", whereas <span class="gene">CDPK</span> and CBL were the most up-regulated protein families in this category. Furthermore, particular genes expressed different isoforms according to the accession, showing the potential to operate in the distinction of physiological behaviors. Besides, heat maps comprising GO categories related to abiotic stress response and the unitags regulation observed in the expression contrasts covering tolerant and sensitive accessions, revealed the unitags potential for plant breeding. Candidate genes related to "hormone response" (LOX, ERF1b, XET), "water response" (PUB, BMY), "salt stress response" (WRKY, MYB) and "oxidative stress response" (PER) figured among the most promising molecular targets. Additionally, nine transcripts (HMGR, XET, WRKY20, RAP2-4, EREBP, NAC3, PER, GPX5 and BMY) validated by RT-qPCR (four different time points) confirmed their differential expression and pointed that already after 25 minutes a transcriptional reorganization started in response to the new condition, with important differences between both accessions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two genes that encode Ca(2+)-dependent protein kinases are induced by <span class="trait">drought</span> and high-salt stresses in Arabidopsis thaliana.</h2>
			<h3>PMID:8078458  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Urao T(1), Katagiri T, Mizoguchi T, Yamaguchi-Shinozaki K, Hayashida N, Shinozaki K.
				<br/> 
				年份：1994 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two cDNA clones, cATCDPK1 and cATCDPK2, encoding Ca(2+)-dependent, calmodulin-independent protein kinases (CDPK) were cloned from Arabidopsis thaliana and their nucleotide sequences were determined. Northern blot analysis indicated that the mRNAs corresponding to the ATCDPK1 and ATCDPK2 genes are rapidly induced by <span class="trait">drought</span> and high-salt stress but not by low-temperature stress or heat stress. Treatment of Arabidopsis plants with exogenous abscisic acid (ABA) had no effect on the induction of ATCDPK1 or ATCDPK2. These findings suggest that a change in the osmotic potential of the environment can serve as a trigger for the induction of ATCDPK1 and ATCDPK2. Putative proteins encoded by ATCDPK1 and ATCDPK2 which contain open reading frames of 1479 and 1488 bp, respectively, are designated ATCDPK1 and ATCDPK2 and show 52% identity at the amino acid sequence level. ATCDPK1 and ATCDPK2 exhibit significant similarity to a soybean <span class="gene">CDPK</span> (51% and 73%, respectively). Both proteins contain a catalytic domain that is typical of serine/threonine protein kinases and a regulatory domain that is homologous to the Ca(2+)-binding sites of calmodulin. Genomic Southern blot analysis suggests the existence of a few additional genes that are related to ATCDPK1 and ATCDPK2 in the Arabidopsis genome. The ATCDPK2 protein expressed in Escherichia coli was found to phosphorylate casein and myelin basic protein preferentially, relative to a histone substrate, and required Ca2+ for activation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL for increased fibrous roots in soybean.</h2>
			<h3>PMID:21165732  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Lee GJ, Boerma RH.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects soybean at various developmental stages, which collectively results in <span class="trait">yield</span> reduction. Unpredictable rainfall has been reported to contribute about 36% to variation of <span class="trait">yield</span> difference between the rain-fed and irrigated fields. Among the <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms, <span class="trait">drought</span> avoidance in genotypes with fibrous roots was recognized to be associated with <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Plant introduction PI416937 was shown to possess fibrous roots and has been used as a parent in breeding programs to improve soybean productivity. Little information is available on relative contribution and chromosomal location of quantitative trait loci (QTL) conditioning fibrous roots in soybean. To identify the genomic locations and genetic bases of this trait, a recombinant inbred line population was derived from a cross between PI416937 and 'Benning'. To detect associated QTLs, phenotypic data were collected and analyzed for 2 years under rain-fed field conditions. The selective genotyping approach was used to reduce the costs and work associated with conducting the QTL analysis. A total of five QTLs were identified on chromosomes <span class="gene">Gm01</span> (Satt383), <span class="gene">Gm03</span> (Satt339), <span class="gene">Gm04</span> (Sct_191), <span class="gene">Gm08</span> (Satt429), and <span class="gene">Gm20</span> (Sat_299), and together explained 51% of the variation in root score. Detected QTLs were co-localized with QTLs related to root morphology, suggesting that fibrous roots QTL may be associated with other morpho-physiological traits and seed <span class="trait">yield</span> in soybean. Genetic dissection of the fibrous roots trait at the individual marker loci will allow for marker-assisted selection to develop soybean genotypes with enhanced levels of fibrous roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean physiology and gene expression during <span class="trait">drought.</span></h2>
			<h3>PMID:20927713  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Stolf-Moreira R(1), Medri ME, Neumaier N, Lemos NG, Pimenta JA, Tobita S, Brogin RL, Marcelino-Guimarães FC, Oliveira MC, Farias JR, Abdelnoor RV, Nepomuceno AL.
				<br/> 
				年份：2010 Oct 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes MG/BR46 (Conquista) and BR16, drought-tolerant and -sensitive, respectively, were compared in terms of morphophysiological and gene-expression responses to water stress during two stages of development. Gene-expression analysis showed differential responses in <span class="gene">Gmdreb1a</span> and <span class="gene">Gmpip1b</span> mRNA expression within 30 days of water-deficit initiation in MG/BR46 (Conquista) plants. Within 45 days of initiating stress, <span class="gene">Gmp5cs</span> and <span class="gene">Gmpip1b</span> had relatively higher expression. Initially, BR16 showed increased expression only for <span class="gene">Gmdreb1a,</span> and later (45 days) for <span class="gene">Gmp5cs,</span> <span class="gene">Gmdefensin</span> and <span class="gene">Gmpip1b.</span> Only BR16 presented down-regulated expression of genes, such as <span class="gene">Gmp5cs</span> and <span class="gene">Gmpip1b,</span> 30 days after the onset of moisture stress, and <span class="gene">Gmgols</span> after 45 days of stress. The faster perception of water stress in MG/BR46 (Conquista) and the better maintenance of up-regulated gene expression than in the sensitive BR16 genotype imply mechanisms by which the former is better adapted to tolerate moisture deficiency.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Soybean HD-Zip Gene Family and Expression Profiling under Salinity and <span class="trait">Drought</span> Treatments</h2>
			<h3>PMID:24498296</h3>
				
					<h3>PMCID:3911943</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Zhu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Hualin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Beijiu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xiang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To gain more insights into the roles of soybean HD-Zip genes in salinity and <span class="trait">drought</span> tolerance, we reanalyzed the expression profiles of all soybean HD-Zip genes in response to <span class="trait">drought</span> and salinity stresses using publicly available microarray data. Fifty-nine HD-Zip genes were included on both GSE40627 and GSE41125,including class I (23 genes), class II (17 genes), class III(8 genes) and class IV(11 genes) ( <xref ref-type="supplementary-material" rid="pone.0087156.s006"> Table S4 </xref> ). The expression of most HD-Zip genes was suppressed or induced under both stresses ( <xref ref-type="fig" rid="pone-0087156-g004"> Fig. 4 </xref> ). Salt stress caused upregulation of 16 genes, including 12 genes from HD-Zip I ( <italic> <span class="gene">Gmhdz2</span> </italic> , - <italic> 9 </italic> , - <italic> 19 </italic> , - <italic> 20 </italic> , - <italic> 24 </italic> , - <italic> 27 </italic> , - <italic> 32 </italic> , - <italic> 38 </italic> , - <italic> 51 </italic> , - <italic> 72 </italic> , - <italic> 75 </italic> and - <italic> 83 </italic> ), four gene from HD-Zip II ( <italic> <span class="gene">Gmhdz14</span> </italic> , - <italic> 70 </italic> , - <italic> 71 </italic> and <italic> -86 </italic> ) and two genes in the HD-Zip III subfamilies ( <italic> <span class="gene">Gmhdz59</span> </italic> and - <italic> 61 </italic> ). In response to <span class="trait">drought</span> stress, 40 genes showed downregulation, whereas the transcripts of 18 were significantly upregulated ( <xref ref-type="fig" rid="pone-0087156-g004"> Fig. 4 </xref> ). Nine genes were upregulated under both salt and <span class="trait">drought</span> stress treatments, including eight genes from HD-Zip I ( <italic> <span class="gene">Gmhdz2</span> </italic> , - <italic> 9 </italic> , - <italic> 19 </italic> , - <italic> 20 </italic> , - <italic> 27 </italic> , - <italic> 32 </italic> , - <italic> 72 </italic> and - <italic> 75 </italic> ) and one gene from HD-Zip III ( <italic> <span class="gene">Gmhdz59</span> </italic> ), while 33 genes were downregulated under both stresses. The responses of HD-Zip genes to stress also differed among some genes. For instance, eight genes ( <italic> <span class="gene">Gmhdz14</span> </italic> , - <italic> 24 </italic> , - <italic> 38 </italic> , <italic> -51 </italic> , - <italic> 70 </italic> , - <italic> 71 </italic> , - <italic> 83 </italic> and - <italic> 86 </italic> ) were significantly upregulated under salt stress, whereas downregulation were observed in these genes under <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0087156-g004"> Fig. 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Duplicate genes may have different evolutionary fates, i.e., nonfunctionalization, neofunctionalization or subfunctionalization, which may be indicated by the divergence in their expression patterns. From the above results, we determined that there are 41 paralogous gene pairs in the soybean HD-Zip gene family. Based on in silico expression data from 23 paralogous pairs in response to salt and <span class="trait">drought</span> stress, expression divergence of <span class="trait">drought</span> stress was evident in three pairs( <italic> <span class="gene">Gmhdz69/-45,</span> <span class="gene">Gmhdz7/-56,</span> <span class="gene">Gmhdz25/-36</span> </italic> ), which supports the notion that the expression of paralogs can diverge significantly after duplication. For example, <italic> <span class="gene">Gmhdz45</span> </italic> was upregulated in response to <span class="trait">drought</span> stress, whereas its duplicated counterpart <italic> <span class="gene">Gmhdz69</span> </italic> was downregulated. As shown in <xref ref-type="fig" rid="pone-0087156-g004"> Figure 4 </xref> , most pairs of paralogs in soybean share similar expression patterns and thus show functional redundancy. From the online database, 8 <span class="gene">Gmhdz</span> genes (2 for HD-Zip I, 5 for HD-Zip II and 1 for HD-Zip IV), were undetectable at the transcription level in all 14 tissues ( <xref ref-type="supplementary-material" rid="pone.0087156.s007"> Table S5 </xref> and <xref ref-type="fig" rid="pone-0087156-g005"> Figure 5 </xref> ). Most soybean HD-Zip genes have a broad expression spectrum. The genes in different subfamilies had their primary abundant transcripts in different tissues, such as HD-Zip I and HD-Zip II in flower and root, both HD-Zip III and IV in young leaf, one com pod and pod shell of 10 days after flowering, some of HD-Zip IV genes also highly expressed in flower. There are a few genes strongly expressed in seed and node. From the expression data of the 34 detected paralogous pairs in14 soybean tissues, expression divergence was also obviously evidenced. For example, <italic> <span class="gene">Gmhdz35</span> </italic> from HD-Zip II highly expressed in seed of 14 days after flowering while <italic> <span class="gene">Gmhdz88</span> </italic> was undetectable. However, the paralogous pairs also have the same expression pattern, such as both <italic> <span class="gene">Gmhdz19</span> </italic> and <italic> <span class="gene">Gmhdz27</span> </italic> from HD-Zip I strongly expressed in flower and barely in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						The HD-Zip I genes have been shown to play major roles in abiotic stress <xref rid="pone.0087156-Harris1" ref-type="bibr"> [61] </xref> . Soil salinity, one of the most severe abiotic stresses, hinders the improvement of agricultural productivity on nearly 20% of irrigated land worldwide <xref rid="pone.0087156-Rhoades1" ref-type="bibr"> [46] </xref> . As water becomes limited, plants redistribute this valuable resource by restricting transpiration and growth, and they frequently flower early <xref rid="pone.0087156-Skirycz1" ref-type="bibr"> [62] </xref> . Therefore, there is a need to identify the master regulators and their regulatory pathways involved in stress adaptation. To screen soybean HD-Zip genes regulated by salt and <span class="trait">drought</span> stress, qRT-PCR was employed to validate 12 candidate genes ( <italic> <span class="gene">Gmhdz2</span> </italic> , - <italic> 9 </italic> , - <italic> 19 </italic> , - <italic> 20 </italic> , - <italic> 24 </italic> , - <italic> 27 </italic> , - <italic> 32 </italic> , - <italic> 38 </italic> , - <italic> 51 </italic> , - <italic> 72 </italic> , - <italic> 75 </italic> and - <italic> 83 </italic> ) from HD-Zip I subfamily that are highly induced by salt stress based on microarray data. The 12 genes are also highly induced by <span class="trait">drought</span> stress (except for <italic> <span class="gene">Gmhdz24</span> </italic> , <italic> -38 </italic> , <italic> -51 </italic> and <italic> -83 </italic> , which are downregulated). In addition, the 12 genes comprise six paralogous pairs. The qRT-PCR results showed that all 12 HD-Zip genes were drought-and salinity-responsive, but some differences were observed among these genes ( <xref ref-type="fig" rid="pone-0087156-g006"> Fig. 6 </xref> ). Under NaCl treatment, only <italic> <span class="gene">Gmhdz20</span> </italic> was highly expressed at a comparatively early stage (6 h after treatment), whereas the levels of <italic> <span class="gene">Gmhdz2</span> </italic> , - <italic> 9 </italic> , - <italic> 19 </italic> , - <italic> 27 </italic> , - <italic> 32 </italic> , - <italic> 38 </italic> , - <italic> 61 </italic> , - <italic> 75 </italic> and - <italic> 83 </italic> expression peaked at 12 h after treatment. <italic> <span class="gene">Gmhdz24</span> </italic> was downregulated by NaCl treatment across all time points. Notably, <italic> <span class="gene">Gmhdz51</span> </italic> and - <italic> 83 </italic> were strongly upregulated (&#x0003e;400-fold) at 12 h after NaCl treatment but were dramatically downregulated thereafter ( <xref ref-type="fig" rid="pone-0087156-g006"> Fig. 6 </xref> ). Under <span class="trait">drought</span> stress, the highest expression levels of <italic> <span class="gene">Gmhdz9</span> </italic> , <italic> -27 </italic> , - <italic> 51 </italic> , - <italic> 75 </italic> and - <italic> 83 </italic> were found at 6 h after treatment, while those of <italic> <span class="gene">Gmhdz24</span> </italic> , - <italic> 32 </italic> and <italic> -38 </italic> were observed later, at 12 h after treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						Notably, <italic> <span class="gene">Gmhdz72</span> </italic> was highly expressed at 24 h after treatment. Similar to their response to NaCl treatment, <italic> <span class="gene">Gmhdz51</span> </italic> and - <italic> 83 </italic> were also obviously upregulated by <span class="trait">drought</span> stress. Furthermore, the induced expression levels of <italic> <span class="gene">Gmhdz51</span> </italic> and <italic> -83 </italic> were higher under <span class="trait">drought</span> treatment than under salinity treatment. By comparing the expression patterns of the 12 segmental duplicated genes, we found that three paralogous pairs, <italic> <span class="gene">Gmhdz2</span> </italic> and <italic> -9 </italic> , <italic> <span class="gene">Gmhdz19</span> </italic> and <italic> -27 </italic> , <italic> <span class="gene">Gmhdz51</span> </italic> and <italic> -83 </italic> , exhibited similar expression profiles under both stress treatments. In summary, the expression patterns of the soybean HD-Zip genes detected by qRT-PCR were roughly consistent with those observed with microarray analyses, even in response to two different abiotic stress <span class="trait">(drought</span> and salt) treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						Legumes, the third major crop worldwide, can also generate on their roots other <italic> de novo </italic> meristems, leading to the formation of lateral roots and symbiotic nitrogen-fixing nodules <xref rid="pone.0087156-Crespi1" ref-type="bibr"> [67] </xref> . In the model legume <italic> Medicago truncatula </italic> , the HD-Zip I transcription factor <italic> HB1 </italic> is expressed in primary and lateral root meristems and induced by salt stress. The mutant is also affected in nodule density (nodule number per root centimeter). In <italic> silico </italic> analysis, three HD-Zip I genes( <italic> <span class="gene">Gmhdz2,</span> -78, -84 </italic> ), three HD-Zip II genes( <italic> <span class="gene">Gmhdz16,</span> -57, -77 </italic> ) and one HD-ZIP III gene( <italic> <span class="gene">Gmhdz30</span> </italic> ) were detected higher both in the root and noodle ( <xref ref-type="fig" rid="pone-0087156-g005"> Figure 5 </xref> ). The result indicates that these genes may affect roots and symbiotic nodules. Roots sense the edaphic water deficit, send chemical signals to shoots, and maintenance of root growth despite reduced water availability can contribute to <span class="trait">drought</span> tolerance through water foraging <xref rid="pone.0087156-Grant1" ref-type="bibr"> [68] </xref> , <xref rid="pone.0087156-Tardieu1" ref-type="bibr"> [69] </xref> . The genes from HD-Zip I and II expressed broadly in the root are potentially related to regulation of developmental adaptation to environmental stress conditions such as <span class="trait">drought.</span> In a previous study, it was reported that <italic> HAHB10 </italic> , a sunflower HD-Zip transcription factor belonging to subfamily II, was able to accelerate flowering and thereby shortens the plant's life cycle when ectopically expressed in <italic> Arabidopsis </italic> plants <xref rid="pone.0087156-Rueda2" ref-type="bibr"> [70] </xref> . <italic> Arabidopsis ATHB2/HAT4 </italic> induces early flowering when ectopically expressed and it has been described as a master switch in the shade avoidance response <xref rid="pone.0087156-Ciarbelli1" ref-type="bibr"> [33] </xref> . The tissues expression data analyses shows that 11 genes( <italic> <span class="gene">Gmhdz16,</span> -21, -22, -35, -64, -70, -71, -74, -76, 77, -88 </italic> ) are expressed in flowers, suggesting their putative roles in the induction of flowering. In <italic> Arabidopsis </italic> and rice, the function of HD-Zip III genes is related to apical embryo patterning, embryonic shoot meristem formation, organ polarity, vascular development, and meristem function <xref rid="pone.0087156-Prigge1" ref-type="bibr"> [36] </xref> , <xref rid="pone.0087156-Byrne1" ref-type="bibr"> [71] </xref> . Leaves, the principal lateral organ of the shoot, develop as polar structures typically with distinct dorsoventrality. In comparison, the 12 soybean genes from HD-ZIP III subfamily identified in the present study showed preferentially high expression levels in young leaf, suggesting their putative roles in the regulation adaxial leaf fate. HD-ZIP IV genes have been identified in several plant species including Arabidopsis, maize, and rice, and a common feature of the vast majority of these genes resides in their epidermis-specific expression pattern <xref rid="pone.0087156-Nakamura1" ref-type="bibr"> [42] </xref> , <xref rid="pone.0087156-Ito1" ref-type="bibr"> [72] </xref> . In maize, <italic> OCL1 </italic> (OUTER CELL LAYER1) is expressed in the epidermis of embryo, endosperm, meristematic tissues, and organ primordia <xref rid="pone.0087156-Ingram1" ref-type="bibr"> [73] </xref> . In soybean, seven genes ( <italic> <span class="gene">Gmhdz48</span> </italic> , <italic> -49 </italic> , <italic> - 55 </italic> , <italic> -56 </italic> , <italic> -62 </italic> , <italic> -68 </italic> , <italic> -73 </italic> ) from HD-ZIP IV show comparatively higher transcript abundances in seed. How these soybean HD-ZIPs perform their functional roles in the seed remains to be elucidated and further functional analyses will be required to understand their biological roles in soybean. In addition, <italic> fwa </italic> (flowering late) semi-dominant mutants are late flowering <xref rid="pone.0087156-Koornneef1" ref-type="bibr"> [74] </xref> , <xref rid="pone.0087156-Soppe1" ref-type="bibr"> [75] </xref> , a phenotype shared by plants overexpressing <italic> PDF </italic> 2 <xref rid="pone.0087156-Abe1" ref-type="bibr"> [20] </xref> . Two orthologous genes ( <italic> <span class="gene">Gmhdz7,-56</span> </italic> ) of <italic> PDF2 </italic> were strongly expressed in the flower( <xref ref-type="fig" rid="pone-0087156-g005"> Figure 5 </xref> ). Overexpression of <italic> <span class="gene">Gmhdz7</span> </italic> and <italic> <span class="gene">Gmhdz56</span> </italic> in soybean probably leads to a strong delay in flowering time. Soybean HD-Zip genes may also involve in other biological processes, such as fruit and seed development, for their abundant expression in the one cm pod and pod shell.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, 12 soybean HD-Zip genes from subfamily I were predicted to be stress-related genes based on microarray data analysis, and their expression patterns under <span class="trait">drought</span> and salinity treatments were investigated. The results show that the 12 soybean HD-Zip genes are responsive to the two stressors. Phylogenetic analysis places <italic> Os04g45810 (Oshox22) </italic> , <italic> Os02g43330 (Oshox24) </italic> , <italic> ATHB-7 </italic> and <italic> ATHB-12 </italic> in the same subgroup (&#x003b3; clade) of the HD-Zip family I, and they all regulate the salt and <span class="trait">drought</span> stress. Based on in <italic> silico </italic> analysis and our RT-qPCR, <italic> <span class="gene">Gmhdz19/-27</span> </italic> and <italic> <span class="gene">Gmhdz32/-72</span> </italic> belonging to &#x003b3; clade also response to the two stressors. <italic> <span class="gene">Gmhdz51/-83</span> </italic> and <italic> <span class="gene">Gmhdz24/-38</span> </italic> have a close relationship with <italic> ATHB2/-20 </italic> and <italic> ATHB5/-6/-16 </italic> respectively. While <italic> ATHB2/-20 </italic> and <italic> ATHB5/-6/-16 </italic> play a role as a negative regulator under the two stress, <italic> <span class="gene">Gmhdz51/-83</span> </italic> and <italic> <span class="gene">Gmhdz24/-38</span> </italic> increased their expression level. In particular, <italic> <span class="gene">Gmhdz51</span> </italic> and <italic> -83 </italic> were strongly induced by both stimuli. We conclude that <italic> <span class="gene">Gmhdz51</span> </italic> and <italic> -83 </italic> may play essential roles in responses to abiotic stress. <italic> <span class="gene">Gmhdz2/-9</span> </italic> and <italic> <span class="gene">Gmhdz19/-27</span> </italic> contained to the clade &#x003b6; which is unique to the soybean are also up-regulated under the two stress. As noted above, though the closely related genes share many similar characteristics, they may have different expression patterns. Throughout evolution, they diversified and acquired new genes that may have important roles in plant development.
					</p>
				
					<p>&nbsp;&nbsp;
						By comparing the six pairs of these duplicated genes, we observed that <italic> <span class="gene">Gmhdz2/-9</span> </italic> , <italic> <span class="gene">Gmhdz19/-27</span> </italic> and <italic> <span class="gene">Gmhdz51/-83</span> </italic> exhibit similar expression patterns, indicating that the responses of paralogs to stress conditions did not undergo much divergence along with the evolution of each gene after duplication and that the duplicated genes may have redundant functions in response to salinity and <span class="trait">drought</span> stress. By contrast, the functions of the gene pair <italic> <span class="gene">Gmhdz24/-38</span> </italic> apparently diverged under salinity treatment while those of <italic> <span class="gene">Gmhdz20/-72</span> </italic> diverged under <span class="trait">drought</span> treatment. The new information obtained in this study may aid in the selection of appropriate candidate genes for further functional characterization.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotypic variation in sorghum [ <italic> Sorghum bicolor </italic> (L.) Moench] exotic germplasm collections for <span class="trait">drought</span> and disease tolerance</h2>
			<h3>PMID:24349954</h3>
				
					<h3>PMCID:3863401</h3>
					<h4>期刊:SpringerPlus</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kapanigowda </surname>  <given-names> Mohankumar H </given-names>  </name>  <address>  <email> mohanhk@k-state.edu </email>  </address>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Perumal </surname>  <given-names> Ramasamy </given-names>  </name>  <address>  <email> perumal@ksu.edu </email>  </address>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Djanaguiraman </surname>  <given-names> Maduraimuthu </given-names>  </name>  <address>  <email> jani@k-state.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Aiken </surname>  <given-names> Robert M </given-names>  </name>  <address>  <email> raiken@ksu.edu </email>  </address>  <xref ref-type="aff" rid="Aff9"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tesso </surname>  <given-names> Tesfaye </given-names>  </name>  <address>  <email> ttesso@k-state.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prasad </surname>  <given-names> PV Vara </given-names>  </name>  <address>  <email> vara@k-state.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Little </surname>  <given-names> Christopher R </given-names>  </name>  <address>  <email> clittle@ksu.edu </email>  </address>  <xref ref-type="aff" rid="Aff10"/>  </contrib>  <aff id="Aff7">  <label/> Ramasamy Perumal, Sorghum Breeder, Agricultural Research Center, Kansas State University, 1232 240th Avenue, Hays, Kansas 67601 USA  </aff>  <aff id="Aff8">  <label/> Department of Agronomy, Kansas State University, Manhattan, KS 66506 USA  </aff>  <aff id="Aff9">  <label/> Kansas State University, Northwest Research-Extension Center, Colby, KS 67701 USA  </aff>  <aff id="Aff10">  <label/> Department of Plant Pathology, Kansas State University, Manhattan, KS 66506 USA  </aff>
				<br/> 
				年份：<day> 28 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						PCA on combined <span class="trait">drought</span> and disease related traits showed positive correlation between PS II quantum <span class="trait">yield,</span> chlorophyll content and HI indicating these traits are associated with <span class="trait">drought</span> and disease tolerance (Nazir <span class="gene">Mir</span> et al. <xref ref-type="bibr" rid="CR32"> 1998 </xref> ). However, these have to be verified using proven disease tolerant and susceptible genotypes under controlled and field experiments. Plant height was positively correlated with lesion length indicating that disease development is directly proportional to plant height which was also confirmed by component loading value (Bazzalo et al. <xref ref-type="bibr" rid="CR5"> 1991 </xref> ). Genotypes PI267109, IS12706, PI533916, BTx3042 and PI291382 were separated from others and showed increased PS II quantum <span class="trait">yield</span> (0.74 to 0.77), SPAD (50 to 60) and HI (0.30 to 0.47) with early maturity (61 to 66&#x000a0;days to flowering) under dryland condition (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Table S3). Likewise, genotype IS24463 showed increased tolerance to both pathogens. Combined PCA analysis of <span class="trait">drought</span> and disease related traits clearly indicated that chlorophyll content, PS II quantum <span class="trait">yield</span> and HI can be used as a screening index for post-flowering <span class="trait">drought</span> and disease tolerance (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). The aforementioned sources can be used to develop potential lines for post-flowering <span class="trait">drought</span> tolerance, Fusarium stalk and charcoal rot <span class="trait">resistance.</span> <fig id="Fig3"> <label> Figure 3 </label> <caption> <p> <bold> Two dimensional plot of PC1 (principle component) versus PC2 for 140 sorghum genotypes segregating for combined <span class="trait">drought</span> and disease related traits. </bold> Note: List of 140 genotypes described in Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression profiling on soybean leaves reveals integration of ER- and osmotic-stress pathways</h2>
			<h3>PMID:18036212</h3>
				
					<h3>PMCID:2242807</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Irsigler </surname>  <given-names> Andr&#x000e9; ST </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> irsigler@bio.fsu.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Costa </surname>  <given-names> Maximiller DL </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> maximiller@vicosa.ufv.br </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Ping </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> pzhang3@unity.ncsu.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Reis </surname>  <given-names> Pedro AB </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> pedrobqi@yahoo.com.br </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Dewey </surname>  <given-names> Ralph E </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> ralph_dewey@ncsu.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Boston </surname>  <given-names> Rebecca S </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> boston@unity.ncsu.edu </email>  </contrib>  <contrib id="A7" corresp="yes" contrib-type="author">  <name>  <surname> Fontes </surname>  <given-names> Elizabeth PB </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> bbfontes@ufv.br </email>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The identification of wheat <span class="gene">MLO</span> (transmembrane domain mildew <span class="trait">resistance)</span> allelic variants as endogenous substrates of an ERAD-related quality-control machinery provided direct evidence that an ERAD-like mechanism operates constitutively in plants [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. As part of the ER-stress response, the activation of this turnover mechanism has been observed in genome-wide analyses of <italic> Arabidopsis </italic> treated with inducers of ER stress and in a transcript-profiling assay of maize endosperm mutants that display a long-term ER stress response [ <xref ref-type="bibr" rid="B34"> 34 </xref> , <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Here we also observed a tunicamycin and AZC up-regulated repertoire of putative ERAD-related genes in soybean, such as those encoding polyubiquitin, ubiquitin conjugating enzyme, alpha subunit of the proteasome, <span class="gene">CDC48</span> and Derlin (see Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ). These results further confirmed that, like in mammalian cells and in yeast, an ER stress-induced quality control mechanism in plants integrates the cellular response to conditions that alter protein folding in the ER.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant antimicrobial peptides: an overview of SuperSAGE transcriptional profile and a functional review.</h2>
			<h3>PMID:20088771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kido EA(1), Pandolfi V, Houllou-Kido LM, Andrade PP, Marcelino FC, Nepomuceno AL, Abdelnoor RV, Burnquist WL, Benko-Iseppon AM.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Defensin, thionin and lipid transfer protein (LTP) gene families, which antimicrobial activity has an attractive use in protein engineering and transgenic production of agronomical important plants, have been here functionally reviewed. Also, a transcriptional overview of a set of plant SuperSAGE libraries and analysis looking for 26 bp tags possibly annotated for those families is presented. Tags differentially expressed (p = 0.05) or constitutively transcribed were identified from leaves or roots SuperSAGE libraries from important Brazilian plant species [cowpea (Vigna unguiculata (L.) Walp.), soybean (Glycine max (L.) Merr.) and modern sugarcane hybrids (Saccharum spp.)] submitted to abiotic [salt (100 mM NaCl) or <span class="trait">drought]</span> or biotic stresses [fungus inoculation (Phakopsora pachyrhizi; Asiatic Soyben <span class="trait">Rust</span> phytopathogen)]. The diverse transcriptional patterns observed, probably related to the variable range of targets and functions involved, could be the first step to unravel the antimicrobial peptide world and the plant stress response relationship. Moreover, SuperSAGE opens the opportunity to find some SNPs or even rare transcript that could be important on plant stress <span class="trait">resistance</span> mechanisms. Putative defensin or <span class="gene">LTP</span> identified by SuperSAGE following a specific plant treatment or physiological condition could be useful for future use in genetic improvement of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Substantial equivalence of antinutrients and inherent plant toxins in genetically modified novel foods.</h2>
			<h3>PMID:10828499  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Novak WK(1), Haslberger AG.
				<br/> 
				年份：2000 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For a safety evaluation of foodstuff derived from genetically modified crops, the concept of the substantial equivalence of modified organisms with their parental lines is used following an environmental safety evaluation. To assess the potential pleiotropic effect of genetic modifications on constituents of modified crops data from US and EC documents were investigated with regard to inherent plant toxins and antinutrients. Analysed were documents of rape (glucosinolates, phytate), maize (phytate), tomato (tomatine, solanine, chaconine, lectins, oxalate), potato (solanine, chaconine, protease-inhibitors, phenols) and soybean (protease-inhibitors, lectins, isoflavones, phytate). In several documents used for notifications no declarations even on essential inherent plant toxins and antinutrients could be found, for instance data on phytate in modified maize were provided only in one of four documents. Significant variations in the contents of these compounds in parental and modified plants especially due to environmental influences were observed: <span class="trait">drought</span> stress, for example, was made responsible for significantly increased glucosinolate levels of up to 72.6micromol/g meal in modified and parental rape plants in field trials compared to recommended standard concentrations of less than 30micromol/g. Taking into account these wide natural variations generally the concentrations of inherent plant toxins and antinutrients in modified products were in the range of the concentrations in parental organisms. The results presented indicate that the concept of the substantial equivalence is useful for the risk assessment of genetically modified organisms <span class="gene">(GMOs)</span> used for novel foods but possible environmental influences on constituents of modified crops need more attention. Consistent guidelines, specifying data of relevant compounds which have to be provided for notification documents of specific organisms have to be established. Because of the importance of inherent plant toxins and antinutrients on nutritional safety, also coherent databases of standard parental lines and clear criteria for mandatory declarations are necessary.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and annotation of abiotic stress responsive candidate genes in peanut ESTs</h2>
			<h3>PMID:23275722</h3>
				

				
					<h3>PMCID:3530874</h3>
					<h4>期刊:Bioinformation</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kumari </surname>  <given-names> Archana </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Ashutosh </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wany </surname>  <given-names> Aakanksha </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prajapati </surname>  <given-names> Gopal Kumar </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pandey </surname>  <given-names> Dev Mani </given-names>  </name>  <xref ref-type="corresp" rid="COR1"> * </xref>  </contrib>  <aff> Department of Biotechnology, Birla Institute of Technology, Mesra, Ranchi, Jharkhand-835215, India </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 11 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peanut (Arachis hypogaea L.) ranks fifth among the world oil crops and is widely grown in India and neighbouring countries. Due to its large and unknown genome size, studies on genomics and genetic modification of peanut are still scanty as compared to other model crops like Arabidopsis, rice, cotton and soybean. Because of its favourable cultivation in semi-arid regions, study on abiotic stress responsive genes and its regulation in peanut is very much important. Therefore, we aim to identify and annotate the abiotic stress responsive candidate genes in peanut ESTs. Expression data of <span class="trait">drought</span> stress responsive corresponding genes and EST sequences were screened from dot blot experiments shown as heat maps and supplementary tables, respectively as reported by Govind et al. (2009). Some of the screened genes having no information about their ESTs in above mentioned supplementary tables were retrieved from NCBI. A phylogenetic analysis was performed to find a group of utmost similar ESTs for each selected gene. Individual EST of the said group were further searched in peanut ESTs (1,78,490 whole EST sequences) using stand alone BLAST. For the prediction as well as annotation of abiotic stress responsive selected genes, various tools (like Vec-Screen, Repeat Masker, EST-Trimmer, DNA Baser, WISE2 and I-TASSER) were used. Here we report the predicted result of Contigs, domain as well as 3D structure for HSP 17.3KDa protein, <span class="gene">DnaJ</span> protein and Type 2 Metallothionein protein.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Peanut ( <italic> Arachis hypogaea </italic> L.) ranks fifth among the world oil crops and is widely grown in India and neighbouring countries. Due to its large and unknown genome size, studies on genomics and genetic modification of peanut are still scanty as compared to other model crops like Arabidopsis, rice, cotton and soybean. Because of its favourable cultivation in semi-arid regions, study on abiotic stress responsive genes and its regulation in peanut is very much important. Therefore, we aim to identify and annotate the abiotic stress responsive candidate genes in peanut ESTs. Expression data of <span class="trait">drought</span> stress responsive corresponding genes and EST sequences were screened from dot blot experiments shown as heat maps and supplementary tables, respectively as reported by Govind <italic> et al </italic> . (2009). Some of the screened genes having no information about their ESTs in above mentioned supplementary tables were retrieved from NCBI. A phylogenetic analysis was performed to find a group of utmost similar ESTs for each selected gene. Individual EST of the said group were further searched in peanut ESTs (1,78,490 whole EST sequences) using stand alone BLAST. For the prediction as well as annotation of abiotic stress responsive selected genes, various tools (like Vec-Screen, Repeat Masker, EST-Trimmer, DNA Baser, WISE2 and I-TASSER) were used. Here we report the predicted result of Contigs, domain as well as 3D structure for HSP 17.3KDa protein, <span class="gene">DnaJ</span> protein and Type 2 Metallothionein protein.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, we aim to identify the <span class="trait">drought</span> stress responsive candidate genes in peanut ESTs available in databases. Experimental data of expression analysis of <span class="trait">drought</span> stress responsive genes and its corresponding EST sequences as heat maps and supplementary tables reported previously [ <xref ref-type="bibr" rid="R06"> 6 </xref> ] were screened. Furthermore, a phylogenetic analysis was performed to find a cluster of utmost similar ESTs for each selected gene. Individual EST of the mentioned cluster were further searched in peanut ESTs (1, 78,490) using stand alone BLAST. To predict as well as annotate abiotic stress responsive selected genes various tools like Vecscreen (a system that is developed by NCBI for quickly identifying and removing segments of a nucleic acid sequence that may be of vector origin and act as contaminant, before sequence analysis), RepeatMasker (a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences), EST trimmer (a tool used for trimming EST sequences concerning ambiguous sequences, removing of distal oligoN series from either the 5' or 3' end and size cut off), DNA Baser (that allows to assemble a set of contiguous sequences, contigs) and WISE2 (that compares a protein sequence to a genomic DNA sequence, allowing for introns and frameshifting errors) were used. Here we report six best putative candidate genes belonging to three different gene families for <span class="trait">drought</span> stress response in peanut. Further, Predicted result of contigs, domain as well as 3D structure for HSP 17.3KDa protein, <span class="gene">DnaJ</span> protein and Type 2 Metallothionein protein were also discussed.
					</p>
				
					<p>&nbsp;&nbsp;
						For identifying <span class="trait">drought</span> stress responsive putative genes in peanut ( <italic> A. hypogaea </italic> ) ESTs, five <span class="trait">drought</span> stress responsive gene families and its corresponding EST sequences were shortlisted from the experimental data of the heat map [ <xref ref-type="bibr" rid="R06"> 6 </xref> ]. For each shortlisted gene family, their corresponding ESTs were selected. Total 22 ESTs belonging to five <span class="trait">drought</span> stress responsive gene families were selected from the supplementary data described by Govind <italic> et al </italic> [ <xref ref-type="bibr" rid="R06"> 6 </xref> ] and shown in <xref ref-type="supplementary-material" rid="SD1"> Table 1 </xref> (see supplementary material). The whole ESTs of <italic> A. hypogaea </italic> (1, 78,490 ESTs) including 22 ESTs screened during this study were downloaded from the public NCBI database. For the identification of similar ESTs of each screened 22 ESTs in whole peanut ESTs, Standalone BLAST (2.2.26) analysis was performed. The screened ESTs were taken as a query sequence and the whole peanut ESTs as a database. Similar EST sequences were shortlisted on the basis of lowest e-value and highest bit score. We found that total of 3073 ESTs in which 826, 286, 17, 66 and 1878 ESTs belongs to Hsp17.3 KDa, Desiccation protective protein LEA5, <span class="gene">DnaJ</span> heat shock protein, Hsp70-60 KDa chaperonin and Type 2 metallothionein, respectively ( <xref ref-type="supplementary-material" rid="SD1"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The functional characterization of protein depends upon their 3D structure, therefore from the resulting assembled contig sequences, conserved protein domains were assessed from InterProScan ( <ext-link ext-link-type="uri" xlink:href="http://www.ebi.ac.uk/Tools/pfa/iprscan"> http://www.ebi.ac.uk/Tools/pfa/iprscan </ext-link> ) [ <xref ref-type="bibr" rid="R19"> 19 </xref> ] which allows to scan protein sequence for matches against the InterPro collection of protein signature databases. The different combinations of domains give rise to the diverse range of proteins found in nature. The identification of domains that occur within proteins can thus provide insights into their function. We also predicted the possible domains in three protein sequences of HSP17.3KDa protein ( <xref ref-type="fig" rid="F3"> Figure 3a </xref> ), <span class="gene">DnaJ</span> protein ( <xref ref-type="fig" rid="F3"> Figure 3b </xref> ) and Type 2 Metallothionein protein ( <xref ref-type="fig" rid="F3"> Figure 3c </xref> ) by using InterProScan. InterProScan of HSP17.3KDa protein has Alpha crystallin/HSP20 domain (IPR002068) as well as HSP20-like Chaperone and other unassigned HSP20 domains ( <xref ref-type="fig" rid="F3"> Figure 3a </xref> ). Likewise, <span class="gene">DnaJ</span> protein showed N-terminal HSP domain (IPRO01623), C-terminal HSP domain (IPRO02939) and others ( <xref ref-type="fig" rid="F3"> Figure 3b </xref> ). Again, metallothionein type 2 has entirely a single domain of metallothionein family-15 type 2 domains (IPRO00347) ( <xref ref-type="fig" rid="F3"> Figure 3c </xref> ).Earlier it was reported that Small heat shock proteins (sHSPs) varies from 15&#x02013;42 kDa [ <xref ref-type="bibr" rid="R20"> 20 </xref> ] that play a significant role in providing <span class="trait">resistance</span> to variety of abiotic stresses such as <span class="trait">drought,</span> salt, cold and oxidants besides high temperature [ <xref ref-type="bibr" rid="R21"> 21 </xref> , <xref ref-type="bibr" rid="R22"> 22 </xref> ]. Tzi <italic> et al </italic> [ <xref ref-type="bibr" rid="R23"> 23 </xref> ] reported that <span class="gene">DnaJ</span> protein plays an important role in various stress response mechanism. Clement <italic> et al </italic> [ <xref ref-type="bibr" rid="R24"> 24 </xref> ] reported that <span class="gene">DnaJ</span> protein interacts with Hsp70 that might also be involved with ABA signalling. Metallothioneins are associated in metal detoxification, homeostasis, and protection during oxidative damage [ <xref ref-type="bibr" rid="R25"> 25 </xref> ]. It was reported that metallothionein transcripts were present in both cultivated peanut and wild species [ <xref ref-type="bibr" rid="R26"> 26 </xref> , <xref ref-type="bibr" rid="R01"> 1 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The present <italic> in silico </italic> study allowed identification of six best putative candidate genes belonging to three different gene families for <span class="trait">drought</span> stress response in peanut. Contigs, domains as well as 3D structure of HSP 17.3KDa protein, <span class="gene">DnaJ</span> protein and Type 2 Metallothionein protein were also predicted. Thus, EST approach play key role in providing an excellent resource for novel gene discovery and its annotation for <span class="trait">drought</span> stresses.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GsAPK, an ABA-Activated and Calcium-Independent SnRK2-Type Kinase from <italic> G. soja </italic> , Mediates the Regulation of Plant Tolerance to Salinity and ABA Stress</h2>
			<h3>PMID:22439004</h3>
				

				
					<h3>PMCID:3306294</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Yang </surname>  <given-names> Liang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ji </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Peng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cai </surname>  <given-names> Hua </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bai </surname>  <given-names> Xi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Qin </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Yanming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 2 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant <span class="gene">Snf1</span> (sucrose non-fermenting-1) related protein kinase (SnRK), a subfamily of serine/threonine kinases, has been implicated as a crucial upstream regulator of ABA and osmotic signaling as in many other signaling cascades. In this paper, we have isolated a novel plant specific ABA activated calcium independent protein kinase (GsAPK) from a highly salt tolerant plant, Glycine soja (50109), which is a member of the SnRK2 family. Subcellular localization studies using GFP fusion protein indicated that GsAPK is localized in the plasma membrane. We found that autophosphorylation and Myelin Basis Protein phosphorylation activity of GsAPK is only activated by ABA and the kinase activity also was observed when calcium was replaced by EGTA, suggesting its independence of calcium in enzyme activity. We also found that cold, salinity, <span class="trait">drought,</span> and ABA stress alter GsAPK gene transcripts and heterogonous overexpression of GsAPK in Arabidopsis alters plant tolerance to high salinity and ABA stress. In summary, we demonstrated that GsAPK is a Glycine soja ABA activated calcium independent SnRK-type kinase presumably involved in ABA mediated stress signal transduction.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plant <span class="gene">Snf1</span> (sucrose non-fermenting-1) related protein kinasea (SnRK) represent a subfamily of serine/threonine kinases that are highly conserved throughout evolution. According to sequence similarity, domain structure and cellular function, the SnRK could be classified as 3 subgroups: SnRK1, SnRK2, and SnRK3. The <italic> SnRK2 </italic> and <italic> SnRK3 </italic> genes are unique to plants and have 42&#x0223c;45&#x00025; amino acid sequence identity with SnRK1 in the kinase catalytic domain <xref ref-type="bibr" rid="pone.0033838-Halford1"> [23] </xref> . To date, reports indicate that SnRK2 and SnRK3 are implicated to function in ABA and/or abiotic stress signaling. There are 10 <italic> SnRK2 </italic> genes and 25 <italic> SnRK3 </italic> genes encoded by the <italic> Arabidopsis </italic> genome <xref ref-type="bibr" rid="pone.0033838-Halford2"> [24] </xref> , <xref ref-type="bibr" rid="pone.0033838-Hrabak1"> [25] </xref> . <italic> SRK2C </italic> , an <italic> Arabidopsis </italic> SnRK2, has been shown to improve <span class="trait">drought</span> tolerance by controlling stress-responsive gene expression <xref ref-type="bibr" rid="pone.0033838-Umezawa1"> [26] </xref> . A guard cell specific Ca <sup> 2&#x0002b; </sup> -independent and ABA-activated protein kinase, AAPK from <italic> Vicia faba </italic> and its <italic> Arabidopsis </italic> ortholog OST1/SRK2E regulate ABA-induced stomatal closure during <span class="trait">drought</span> stress <xref ref-type="bibr" rid="pone.0033838-Li1"> [27] </xref> , <xref ref-type="bibr" rid="pone.0033838-Mustilli1"> [28] </xref> , <xref ref-type="bibr" rid="pone.0033838-Yoshida1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0033838-Assmann1"> [30] </xref> . In rice, 10 members of <italic> SnRK2 </italic> gene family were identified and all of them are activated by hyperosmotic stress. Three of these are also activated by ABA. Surprisingly, there were no members that were only activated by ABA <xref ref-type="bibr" rid="pone.0033838-Kobayashi1"> [31] </xref> . PKABA1 (ABA-responsive protein kinase 1) from wheat also belongs to the SnRK2 family, which is involved in mediating ABA-induced changes in gene expression <xref ref-type="bibr" rid="pone.0033838-GomezCadenas1"> [32] </xref> . Unlike SnRK1 and SnRK2, SnRK3 is calcium-dependent for its interactions with a calcium-binding protein <xref ref-type="bibr" rid="pone.0033838-Guo1"> [33] </xref> . The <italic> Arabidopsis </italic> SnRK3 family includes SOS2 (salt overly sensitive 2), which functions in ion homeostasis and is involved in conferring salt tolerance <xref ref-type="bibr" rid="pone.0033838-Liu2"> [34] </xref> , <xref ref-type="bibr" rid="pone.0033838-Halfter1"> [35] </xref> . There is biochemical evidence that PKS3, PKS18 or CIPK3, members of the SnRK3 family, modulate ABA sensitivity in seed germination, stomatal closure and seedling growth <xref ref-type="bibr" rid="pone.0033838-Kim2"> [9] </xref> , <xref ref-type="bibr" rid="pone.0033838-Guo1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0033838-Gong1"> [36] </xref> . Moreover, PKS3 and SOS2 were found to interact with ABA insensitive 2 (ABI2) phosphatase with specificity <xref ref-type="bibr" rid="pone.0033838-Guo1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0033838-Ohta1"> [37] </xref> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The contribution of bacteroidal nitrate and nitrite reduction to the formation of nitrosylleghaemoglobin complexes in soybean root nodules.</h2>
			<h3>PMID:17259612  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Meakin GE(1), Bueno E, Jepson B, Bedmar EJ, Richardson DJ, Delgado MJ.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is becoming recognized that leghaemoglobin constitutes an important buffer for the cytotoxic nitric oxide radical (NO(*)) in root nodules, although the sources of this NO(*) within nodules are unclear. In Bradyrhizobium japonicum bacteroids, NO(*) can be produced through the denitrification process, during which nitrate is reduced to nitrite by the periplasmic nitrate reductase Nap, and nitrite is reduced to NO(*) by the respiratory nitrite reductase NirK. To assess the contribution of bacteroidal denitrification to the NO(*) within nitrate-treated soybean nodules, electron paramagnetic resonance and UV-visible spectroscopy were employed to study the presence of nitrosylleghaemoglobin (LbNO) within nodules from plants inoculated with wild-type, <span class="gene">napA</span> or <span class="gene">nirK</span> B. japonicum strains. Since it has been found that hypoxia induces NO(*) production in plant root tissue, and that plant roots can be subjected to hypoxic stress during <span class="trait">drought</span> and flooding, the effect of hypoxic stress on the formation of LbNO complexes within nodules was also investigated. Maximal levels of LbNO were observed in nodules from plants treated with nitrate and subjected to hypoxic conditions. It is shown that, in the presence of nitrate, all of the LbNO within normoxic nodules arises from nitrate reduction by the bacteroidal periplasmic nitrate reductase, whereas Nap activity is only responsible for half of the LbNO within hypoxic nodules. In contrast to Nap, NirK is not essential for LbNO formation under any condition tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis and expression profiling suggest diverse roles of <italic> <span class="gene">GH3</span> </italic> genes during development and abiotic stress responses in legumes</h2>
			<h3>PMID:25642236</h3>
				

				
					<h3>PMCID:4294127</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Vikash K. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/190534"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jain </surname>  <given-names> Mukesh </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/94275"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Garg </surname>  <given-names> Rohini </given-names>  </name>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/94284"/>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Growth hormone auxin regulates various cellular processes by altering the expression of diverse genes in plants. Among various auxin-responsive genes, <span class="gene">GH3</span> genes maintain endogenous auxin homeostasis by conjugating excess of auxin with amino acids. <span class="gene">GH3</span> genes have been characterized in many plant species, but not in legumes. In the present work, we identified members of <span class="gene">GH3</span> gene family and analyzed their chromosomal distribution, gene structure, gene duplication and phylogenetic analysis in different legumes, including chickpea, soybean, Medicago, and Lotus. A comprehensive expression analysis in different vegetative and reproductive tissues/stages revealed that many of <span class="gene">GH3</span> genes were expressed in a tissue-specific manner. Notably, chickpea CaGH3-3, soybean <span class="gene">GmGH3-8</span> and -25, and Lotus LjGH3-4, -5, -9 and -18 genes were up-regulated in root, indicating their putative role in root development. In addition, chickpea CaGH3-1 and -7, and Medicago MtGH3-7, -8, and -9 were found to be highly induced under <span class="trait">drought</span> and/or salt stresses, suggesting their role in abiotic stress responses. We also observed the examples of differential expression pattern of duplicated <span class="gene">GH3</span> genes in soybean, indicating their functional diversification. Furthermore, analyses of three-dimensional structures, active site residues and ligand preferences provided molecular insights into function of <span class="gene">GH3</span> genes in legumes. The analysis presented here would help in investigation of precise function of <span class="gene">GH3</span> genes in legumes during development and stress conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Gretchen Hagen3 family of proteins maintain auxin level by catalyzing conjugation of amino acids with indole-3-acetic acid, salicylic acid (SA), and jasmonic acid (JA; <xref rid="B50" ref-type="bibr"> Staswick et al., 2002 </xref> , <xref rid="B49" ref-type="bibr"> 2005 </xref> ). The first <italic> <span class="gene">GH3</span> </italic> gene was identified by <xref rid="B19" ref-type="bibr"> Hagen et al. (1984) </xref> as an early auxin-responsive gene in soybean. Since then, a large number of <italic> <span class="gene">GH3</span> </italic> homologs have been identified in numerous plant species ranging from mosses to angiosperms ( <xref rid="B23" ref-type="bibr"> Jain et al., 2006 </xref> ; <xref rid="B53" ref-type="bibr"> Terol et al., 2006 </xref> ; <xref rid="B35" ref-type="bibr"> Ludwig-M&#x000fc;ller et al., 2008 </xref> ; <xref rid="B29" ref-type="bibr"> Kumar et al., 2012 </xref> ; <xref rid="B66" ref-type="bibr"> Yuan et al., 2013 </xref> ). The studies on <span class="gene">GH3</span> proteins have revealed their regulatory function in plant growth, organ development, light signaling, abiotic stress tolerance, and plant defense responses ( <xref rid="B62" ref-type="bibr"> Woodward and Bartel, 2005 </xref> ; <xref rid="B40" ref-type="bibr"> Park et al., 2007 </xref> ; <xref rid="B24" ref-type="bibr"> Jain and Khurana, 2009 </xref> ; <xref rid="B34" ref-type="bibr"> Ludwig-Muller, 2011 </xref> ; <xref rid="B11" ref-type="bibr"> Du et al., 2012 </xref> ; <xref rid="B29" ref-type="bibr"> Kumar et al., 2012 </xref> ; <xref rid="B66" ref-type="bibr"> Yuan et al., 2013 </xref> ). In <italic> Arabidopsis </italic> , <span class="gene">GH3</span> gene family has been classified into three groups (I&#x02013;III) based on sequence similarity and substrate specificities ( <xref rid="B50" ref-type="bibr"> Staswick et al., 2002 </xref> ). Group I <span class="gene">GH3</span> proteins of <italic> Arabidopsis </italic> are JA-amido synthetases ( <xref rid="B50" ref-type="bibr"> Staswick et al., 2002 </xref> , <xref rid="B49" ref-type="bibr"> 2005 </xref> ). AtGH3-11, a group I <span class="gene">GH3</span> protein, was characterized based on analysis of <italic> jar1 </italic> mutant, which was insensitive to JA and was required for the formation of bioactive jasmonate JA-isoleucine ( <xref rid="B50" ref-type="bibr"> Staswick et al., 2002 </xref> ). A different allele of this gene ( <italic> FIN219 </italic> ) was identified as a phytochrome A signaling component, having crucial role in photomorphogenesis ( <xref rid="B21" ref-type="bibr"> Hsieh et al., 2000 </xref> ). Group II <span class="gene">GH3</span> proteins of <italic> Arabidopsis </italic> are involved in conjugation of IAA to various amino acids ( <xref rid="B50" ref-type="bibr"> Staswick et al., 2002 </xref> , <xref rid="B49" ref-type="bibr"> 2005 </xref> ). <italic> AtGH3-2 </italic> gain-of-function mutant, <italic> Ydk1-D </italic> , was shown to be responsible for short primary root, reduced lateral root number, and apical dominance ( <xref rid="B51" ref-type="bibr"> Takase et al., 2004 </xref> ). In another report <italic> AtGH3-6 </italic> mutant, <italic> dfl1 </italic> , was shown to regulate shoot elongation and lateral root formation negatively, but positively regulate the light responses to hypocotyl length ( <xref rid="B37" ref-type="bibr"> Nakazawa et al., 2001 </xref> ). Some Group II <span class="gene">GH3</span> proteins of rice (TLD1/OsGH3-13, OsGH3-2, and OsGH3-8) have also been characterized that conjugate IAA with aspartate or alanine ( <xref rid="B6" ref-type="bibr"> Chen et al., 2009 </xref> , <xref rid="B5" ref-type="bibr"> 2010 </xref> ; <xref rid="B67" ref-type="bibr"> Zhang et al., 2009 </xref> ). A gain-of-function mutant of rice <italic> OsGH3-13 </italic> gene, <italic> tld1-D </italic> , resulted in increased tillers, enlarged leaf angles, dwarfism and improved <span class="trait">drought</span> tolerance ( <xref rid="B67" ref-type="bibr"> Zhang et al., 2009 </xref> ). AtGH3-12/PBS3 is the only characterized member of group III, which catalyzes the conjugation of glutamic acid (Glu) to 4-aminobenzoate and 4-hydroxybenzoate and is involved in SA signaling ( <xref rid="B22" ref-type="bibr"> Jagadeeswaran et al., 2007 </xref> ; <xref rid="B38" ref-type="bibr"> Nobuta et al., 2007 </xref> ; <xref rid="B39" ref-type="bibr"> Okrent et al., 2009 </xref> ). Recently, the crystal structure and mechanism of catalytic action of AtGH3-12 and JAR1/AtGH3-11 ( <xref rid="B61" ref-type="bibr"> Westfall et al., 2012 </xref> ) in <italic> Arabidopsis </italic> , and VvGH3-1 in grapevine ( <xref rid="B41" ref-type="bibr"> Peat et al., 2012 </xref> ) have also been reported.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants are constantly exposed to various abiotic stresses in their life cycle. Several recent studies have implicated auxin in abiotic stress responses ( <xref rid="B24" ref-type="bibr"> Jain and Khurana, 2009 </xref> ; <xref rid="B58" ref-type="bibr"> Wang et al., 2010 </xref> ; <xref rid="B11" ref-type="bibr"> Du et al., 2012 </xref> ; <xref rid="B29" ref-type="bibr"> Kumar et al., 2012 </xref> ; <xref rid="B66" ref-type="bibr"> Yuan et al., 2013 </xref> ). Some studies have revealed that <italic> <span class="gene">GH3</span> </italic> genes are regulated by abiotic stresses, like <span class="trait">drought,</span> salt, and cold stresses ( <xref rid="B40" ref-type="bibr"> Park et al., 2007 </xref> ; <xref rid="B24" ref-type="bibr"> Jain and Khurana, 2009 </xref> ). The transcript level of <italic> AtGH3-5 </italic> ( <italic> WES1 </italic> ) has been shown to be induced by various abiotic stress conditions, like <span class="trait">drought,</span> high salt, and cold ( <xref rid="B40" ref-type="bibr"> Park et al., 2007 </xref> ). In rice, the transcript abundance of <italic> OsGH3-1, OsGH3-2 </italic> , <italic> OsGH3-8, </italic> and <italic> OsGH3-13 </italic> were markedly higher in seedlings subjected to salt, <span class="trait">drought</span> and cold stresses ( <xref rid="B24" ref-type="bibr"> Jain and Khurana, 2009 </xref> ; <xref rid="B67" ref-type="bibr"> Zhang et al., 2009 </xref> ; <xref rid="B11" ref-type="bibr"> Du et al., 2012 </xref> ). In <italic> Sorghum </italic> , at least six <italic> <span class="gene">GH3</span> </italic> genes were found to be induced upon salt and <span class="trait">drought</span> treatments in leaf ( <xref rid="B58" ref-type="bibr"> Wang et al., 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the role of legume <italic> <span class="gene">GH3</span> </italic> genes in abiotic stress responses, we performed scanning of <italic> cis </italic> -acting regulatory DNA elements within promoter regions (2 kb upstream from the start codon) using PLACE database. This analysis predicted several elements responsive to auxin (IAA), abscisic acid (ABA), SA, JA, <span class="trait">drought,</span> salinity, and disease (Supplemental Table S4), suggesting that the function of these genes may be associated with various phytohormone signals and/or environmental stresses. Considering regulatory role of <italic> cis </italic> -elements, we analyzed expression of <italic> <span class="gene">GH3</span> </italic> genes under abiotic stress conditions to know their function during abiotic stresses. For chickpea, we analyzed RNA-seq data from root and shoot tissues subjected to desiccation, salinity and cold conditions ( <xref rid="B13" ref-type="bibr"> Garg et al., 2014 </xref> ), and performed real-time PCR analysis for validation. In our analysis, paralogous gene pair, <italic> CaGH3-1 </italic> and <italic> -9, </italic> showed induction under both desiccation and salinity stresses in root ( <bold> Figures <xref ref-type="fig" rid="F5"> 5A,B </xref> </bold> ), and also their promoter sequences harbor desiccation (S000414) and salinity (S000453) responsive <italic> cis </italic> -regulatory elements (Supplemental Table S4), indicating their role in desiccation and salinity stress. Recently, rice group-I gene, <italic> OsGH3-12 </italic> , has also been found to be markedly induced by <span class="trait">drought</span> stress ( <xref rid="B10" ref-type="bibr"> Du et al., 2013b </xref> ). Similarly, promoter of <italic> CaGH3-4 </italic> harbor salinity responsive <italic> cis </italic> -element (S000453) and showed higher expression in root under salt stress ( <bold> Figures <xref ref-type="fig" rid="F5"> 5A,B </xref> </bold> ). Its ortholog, <italic> AtGH3-1 </italic> , has also been found to be up-regulated under salt stress ( <xref rid="B44" ref-type="bibr"> Sani et al., 2013 </xref> ), corroborating our result. Group-I paralogous genes, <italic> CaGH3-7 </italic> and <italic> -8 </italic> , were found to be induced in root under salinity stress ( <bold> Figures <xref ref-type="fig" rid="F5"> 5A,B </xref> </bold> ), implying their involvement in homeostasis of auxin under salinity stress in root. <italic> CaGH3-5 </italic> and <italic> -6 </italic> showed enhanced expression under desiccation, salt and cold stresses in shoot and root ( <bold> Figures <xref ref-type="fig" rid="F5"> 5A,B </xref> </bold> ), respectively, suggesting their role during multiple abiotic stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						In <italic> Medicago </italic> , <italic> MtGH3-8 </italic> and <italic> -9 </italic> genes were induced under salt stress in root, and <italic> MtGH3-7 </italic> was induced under <span class="trait">drought</span> stress in root (Supplemental Figure <xref ref-type="supplementary-material" rid="SM1"> S2 </xref> ). Previous reports suggest that IAA, SA, JA, ethylene, and ABA regulate the protective responses of plants against both biotic and abiotic stress responses via signaling crosstalk ( <xref rid="B2" ref-type="bibr"> Bostock, 2005 </xref> ; <xref rid="B33" ref-type="bibr"> Lorenzo and Solano, 2005 </xref> ; <xref rid="B36" ref-type="bibr"> Mauch-Mani and Mauch, 2005 </xref> ; <xref rid="B7" ref-type="bibr"> Ding et al., 2008 </xref> ; <xref rid="B8" ref-type="bibr"> Domingo et al., 2009 </xref> ; <xref rid="B12" ref-type="bibr"> Fu et al., 2011 </xref> ). In addition, orthologous genes, <italic> CaGH3-10 </italic> and <italic> MtGH3-8 </italic> , showed induced expression under salt stress in root ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ; Supplemental Figure <xref ref-type="supplementary-material" rid="SM1"> S2 </xref> ); suggesting their conserved function in both legumes. Taken together, these findings indicated that members of <span class="gene">GH3</span> gene family might be involved in stress adaptation in legumes.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in high salt and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						YABBY family is a plant specific transcription factor family, with the typical N-terminal C2C2 type zinc finger domain and the C-terminal YABBY conservative structure domain, which plays important biological roles in plant growth, development and morphogenesis. In this study, a total of 17 YABBY genes were identified in the soybean genome. The results of this research showed that 17 soybean YABBY genes were located on 11 chromosomes. Analysis of putative cis-acting elements showed that soybean YABBY genes contained lots of MYB and MYC elements. Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses. And the transient expression in Arabidopsis protoplasts showed that <span class="gene">GmYABBY3</span> protein distributed uniformly the whole cells, while <span class="gene">GmYABBY10</span> protein was mainly localized in the membranes and cytoplasm and <span class="gene">GmYABBY16</span> protein was localized the nucleus and membranes. To further identify the function of <span class="gene">GmYABBY10,</span> we obtained the transgenic Arabidopsis overexpression <span class="gene">GmYABBY10.</span> Based on germination and seedling root arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment. While the root length and root surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings. When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and salt stresses. This study provided valuable information regarding the classification and functions of YABBY genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variability in Bradyrhizobium japonicum and B. elkanii seven years after introduction of both the exotic microsymbiont and the soybean host in a cerrados soil.</h2>
			<h3>PMID:17265000  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Batista JS(1), Hungria M, Barcellos FG, Ferreira MC, Mendes IC.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plasticity of rhizobial genomes is far greater than previously thought, with complex genomic recombination events that may be accelerated by the often stressful environmental conditions of the tropics. This study aimed at evaluating changes in soybean rhizobia due to adaptation to inhospitable environmental conditions (high temperatures, <span class="trait">drought,</span> and acid soils) in the Brazilian Cerrados. Both the host plant and combinations of four strains of soybean Bradyrhizobium were introduced in an uncropped soil devoid of rhizobia capable of nodulating soybean. After the third year, seeds were not reinoculated. Two hundred and sixty-three isolates were obtained from nodules of field-grown soybean after the seventh year, and their morphological, physiological, serological, and symbiotic properties determined, followed by genetic analysis of conserved and symbiotic genes. B. japonicum strain CPAC 15 (same serogroup as USDA 123) was characterized as having high saprophytic capacity and competitiveness and by the seventh year represented up to 70% of the cultivable population, in contrast to the poor survival and competitiveness of B. japonicum strain CPAC 7 (same serogroup as CB 1809). In general, adapted strains had increased mucoidy, and up to 43% of the isolates showed no serological reaction. High variability, presumably resulting from the adaptation to the harsh environmental conditions, was verified in rep-PCR (polymerase chain reaction) profiles, being lower in strain CPAC 15, intermediate in B. elkanii, and higher in CPAC 7. Restriction fragment length polymorphism (RFLP)-PCR types of the 16S rDNA corresponded to the following: one type for B. elkanii species, two for B. japonicum, associated to CPAC 15 and CPAC 7, and unknown combinations of profiles. However, when <span class="gene">nodC</span> sequences and RFLP-PCR of the <span class="gene">nifH</span> region data were considered, only two clusters were observed having full congruence with B. japonicum and B. elkanii species. Combining the results, variability was such that even within a genetically more stable group (such as that of CPAC 15), only 6.4% of the isolates showed high similarity to the inoculant strain, whereas none was similar to CPAC 7. The genetic variability in our study seems to result from a variety and combination of events including strain dispersion, genomic recombination, and horizontal gene transfer. Furthermore, the genetic variability appears to be mainly associated with adaptation, saprophytic capacity, and competitiveness, and not with symbiotic effectiveness, as the similarity of symbiotic genes was higher than that of conserved regions of the DNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of the <span class="gene">RAV</span> Family in Soybean and Functional Identification of <span class="gene">GmRAV-03</span> Involvement in Salt and <span class="trait">Drought</span> Stresses and Exogenous ABA Treatment.</h2>
			<h3>PMID:28634481</h3>
				

				
			<p>
				作者：Zhao SP(1)(2), Xu ZS(2), Zheng WJ(1), Zhao W(1)(2), Wang YX(3), Yu TF(1), Chen M(2), Zhou YB(1), Min DH(1), Ma YZ(2), Chai SC(1), Zhang XH(1).
				<br/> 
				年份：2017 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcription factors play vital roles in plant growth and in plant responses to abiotic stresses. The <span class="gene">RAV</span> transcription factors contain a B3 DNA binding domain and/or an APETALA2 (AP2) DNA binding domain. Although genome-wide analyses of <span class="gene">RAV</span> family genes have been performed in several species, little is known about the family in soybean (Glycine max L.). In this study, a total of 13 <span class="gene">RAV</span> genes, named as <span class="gene">GmRAVs,</span> were identified in the soybean genome. We predicted and analyzed the amino acid compositions, phylogenetic relationships, and folding states of conserved domain sequences of soybean <span class="gene">RAV</span> transcription factors. These soybean <span class="gene">RAV</span> transcription factors were phylogenetically clustered into three classes based on their amino acid sequences. Subcellular localization analysis revealed that the soybean <span class="gene">RAV</span> proteins were located in the nucleus. The expression patterns of 13 <span class="gene">RAV</span> genes were analyzed by quantitative real-time PCR. Under <span class="trait">drought</span> stresses, the <span class="gene">RAV</span> genes expressed diversely, up- or down-regulated. Following NaCl treatments, all <span class="gene">RAV</span> genes were down-regulated excepting <span class="gene">GmRAV-03</span> which was up-regulated. Under abscisic acid (ABA) treatment, the expression of all of the soybean <span class="gene">RAV</span> genes increased dramatically. These results suggested that the soybean <span class="gene">RAV</span> genes may be involved in diverse signaling pathways and may be responsive to abiotic stresses and exogenous ABA. Further analysis indicated that <span class="gene">GmRAV-03</span> could increase the transgenic lines <span class="trait">resistance</span> to high salt and <span class="trait">drought</span> and result in the transgenic plants insensitive to exogenous ABA. This present study provides valuable information for understanding the classification and putative functions of the <span class="gene">RAV</span> transcription factors in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Impacts of elevated ozone concentration on N2O emission from arid farmland].</h2>
			<h3>PMID:26031093  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu YZ, Hu ZH, Li CZ, Chen ST, Xie Y, Xiao QT.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the impact of elevated surface ozone (O3) concentration on nitrous oxide (N2O) emission from arid farmland, field experiments were carried out during winter-wheat and soybean growing seasons under the condition of simulating O3 concentrations, including free air (CK), 100 nL x L(-1) O3 concentration (T1), and 150 nL x L(-1) O3 concentration (T2). N2O emission fluxes were measured by static dark chamber-gas chromatograph method. The results showed that the accumulative amount of N2O (AAN) were decreased by 37.8% (P = 0.000 ) and 8.8% (P = 0.903 ) under T1 and T2 treatments, respectively, in the turning-green stage of winter wheat. In the elongation-booting stage, <span class="gene">ANN</span> were decreased by 15.0% (P = 0.217) and 39.1% (P = 0.000) under T1 and T2 treatments, respectively. <span class="gene">ANN</span> were decreased by 18.9% (P = 0.138) and 25.6% (P = 0.000) under T1 and T2 treatments, respectively, during the whole winter-wheat growing season. No significant impact of elevated O3 concentration on N2O emission from soil-soybean system was found due to the less rainfall during the soybean growing season, <span class="trait">drought</span> had a stronger stress on soybean than O3 concentration. The results of this study suggested that elevated O3 concentration could reduce N2O emission from arid farmland.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Response of archaeal communities in the rhizosphere of maize and soybean to elevated atmospheric CO2 concentrations.</h2>
			<h3>PMID:21209969  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nelson DM(1), Cann IK, Mackie RI.
				<br/> 
				年份：2010 Dec 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Archaea are important to the carbon and nitrogen cycles, but it remains uncertain how rising atmospheric carbon dioxide concentrations ([CO(2)]) will influence the structure and function of soil archaeal communities. METHODOLOGY/PRINCIPAL FINDINGS: We measured abundances of archaeal and bacterial 16S rRNA and <span class="gene">amoA</span> genes, phylogenies of archaeal 16S rRNA and <span class="gene">amoA</span> genes, concentrations of KCl-extractable soil ammonium and nitrite, and potential ammonia oxidation rates in rhizosphere soil samples from maize and soybean exposed to ambient (∼385 ppm) and elevated (550 ppm) [CO(2)] in a replicated and field-based study. There was no influence of elevated [CO(2)] on copy numbers of archaeal or bacterial 16S rRNA or <span class="gene">amoA</span> genes, archaeal community composition, KCl-extractable soil ammonium or nitrite, or potential ammonia oxidation rates for samples from maize, a model C(4) plant. Phylogenetic evidence indicated decreased relative abundance of crenarchaeal sequences in the rhizosphere of soybean, a model leguminous-C(3) plant, at elevated [CO(2)], whereas quantitative PCR data indicated no changes in the absolute abundance of archaea. There were no changes in potential ammonia oxidation rates at elevated [CO(2)] for soybean. Ammonia oxidation rates were lower in the rhizosphere of maize than soybean, likely because of lower soil pH and/or abundance of archaea. KCl-extractable ammonium and nitrite concentrations were lower at elevated than ambient [CO(2)] for soybean. CONCLUSION: Plant-driven shifts in soil biogeochemical processes in response to elevated [CO(2)] affected archaeal community composition, but not copy numbers of archaeal genes, in the rhizosphere of soybean. The lack of a treatment effect for maize is consistent with the fact that the photosynthesis and productivity of maize are not stimulated by elevated [CO(2)] in the absence of <span class="trait">drought.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of isoflavone biosynthetic genes in soybean during water deficits.</h2>
			<h3>PMID:20430761  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gutierrez-Gonzalez JJ(1), Guttikonda SK, Tran LS, Aldrich DL, Zhong R, Yu O, Nguyen HT, Sleper DA.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Numerous environmental factors influence isoflavone accumulation and have long hampered their genetic dissection. Temperature and water regimes are two of the most significant abiotic factors. However, while the effects of temperature have been widely studied, little is known about how water scarcity might affect isoflavone concentration in seeds. Studies have shown that accumulation of isoflavones is promoted by well-watered conditions, but the molecular basis remains elusive. The length and severity of the water stress required to induce changes are also still unknown. In the present work, several intensities of water stress were evaluated at various critical stages for soybean [Glycine max (L.) Merr.] seed development, in both field and controlled environments. The results suggested that only long-term progressive <span class="trait">drought,</span> spanning most of the seed developmental stages, significantly decreased isoflavone content in seeds. The reduction is proportional to the intensity of the stress and appears to occur in a genotype-dependent manner. However, regardless of water regime, isoflavone compounds were mainly accumulated in the later seed developmental stages. Transcripts of the most important genes for isoflavone biosynthesis were also quantified from samples collected at key seed developmental stages under well-watered and long-term water deficit conditions. Expression of CHS7, CHS8 and <span class="gene">IFS2</span> correlated with isoflavone accumulation under well-watered conditions. Interestingly, we found that the two isoflavone synthase genes in soybean (IFS1 and IFS2) showed different patterns of expression. The abundance of <span class="gene">IFS1</span> transcripts was maintained at a constant rate, whereas <span class="gene">IFS2</span> was down-regulated and highly correlated with isoflavone accumulation under both water deficit and well-watered conditions, suggesting <span class="gene">IFS2</span> as a main contributor to isoflavone diminution under <span class="trait">drought.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution and expression analysis of the soybean glutamate decarboxylase gene family.</h2>
			<h3>PMID:25431418  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hyun TK(1), Eom SH, Han X, Kim JS.
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glutamate decarboxylase (GAD) is an enzyme that catalyses the conversion of L-glutamate into gamma-aminobutyric acid (GABA), which is a four-carbon non-protein amino acid present in all organisms. Although plant <span class="gene">GAD</span> plays important roles in GABA biosynthesis, our knowledge concerning <span class="gene">GAD</span> gene family members and their evolutionary relationship remains limited. Therefore, in this study, we have analysed the evolutionary mechanisms of soybean <span class="gene">GAD</span> genes and suggested that these genes expanded in the soybean genome partly due to segmental duplication events. The approximate dates of duplication events were calculated using the synonymous substitution rate, and we suggested that the segmental duplication of <span class="gene">GAD</span> genes in soybean originated 9.47 to 11.84 million years ago (Mya). In addition, all segmental duplication pairs <span class="gene">(GmGAD1/3</span> and <span class="gene">GmGAD2/4)</span> are subject to purifying selection. Furthermore, <span class="gene">GmGAD</span> genes displayed differential expression either in their transcript abundance or in their expression patterns under abiotic stress conditions like salt, <span class="trait">drought,</span> and cold. The expression pattern of paralogous pairs suggested that they might have undergone neofunctionalization during the subsequent evolution process. Taken together, our results provide valuable information for the evolution of the <span class="gene">GAD</span> gene family and represent the basis for future research on the functional characterization of <span class="gene">GAD</span> genes in higher plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>