<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Fusarium infection</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Fusarium infection</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:FvSNF1, the sucrose non-fermenting protein kinase gene of <span class="trait">Fusarium</span> virguliforme, is required for cell-wall-degrading enzymes expression and sudden death syndrome development in soybean.</h2>
			<h3>PMID:28132080</h3>
				

				
			<p>
				作者：Islam KT(1)(2), Bond JP(1), Fakhoury AM(3).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Fusarium</span> virguliforme is a soil-borne pathogenic fungus that causes sudden death syndrome (SDS) in soybean. Its pathogenicity is believed to require the activity of cell-wall-degrading enzymes (CWDEs). The sucrose non-fermenting protein kinase 1 gene (SNF1) is a key component of the glucose de-repression pathway in yeast, and a regulator of gene expression for CWDEs in some plant pathogenic fungi. To elucidate the functional role of the <span class="gene">SNF1</span> homolog in F. virguliforme, FvSNF1 was disrupted using a split-marker strategy. Disruption of FvSNF1 in F. virguliforme abolishes galactose utilization and causes poor growth on xylose, arabinose and sucrose. However, the resulting Fvsnf1 mutant grew similar to wild-type and ectopic transformants on glucose, fructose, maltose, or pectin as the main source of carbon. The Fvsnf1 mutant displayed no expression of the gene-encoding galactose oxidase (GAO), a secretory enzyme that catalyzes oxidation of D-galactose. It also exhibited a significant reduction in the expression of several CWDE-coding genes in contrast to the wild-type strain. Greenhouse pathogenicity assays revealed that the Fvsnf1 mutant was severely impaired in its ability to cause SDS on challenged soybean plants. Microscopy and microtome studies on infected roots showed that the Fvsnf1 mutant was defective in colonizing vascular tissue of infected plants. Cross and longitudinal sections of infected roots stained with fluorescein-labeled wheat germ agglutinin and Congo red showed that the Fvsnf1 mutant failed to colonize the xylem vessels and phloem tissue at later stages of <span class="trait">infection.</span> Quantification of the fungal biomass in inoculated roots further confirmed a reduced colonization of roots by the Fvsnf1 mutant when compared to the wild type. These findings suggest that FvSNF1 regulates the expression of CWDEs in F. virguliforme, thus affecting the virulence of the fungus on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tanscriptomic Study of the Soybean- <italic> <span class="trait">Fusarium</span> virguliforme </italic> Interaction Revealed a Novel Ankyrin-Repeat Containing Defense Gene, Expression of Whose during <span class="trait">Infection</span> Led to Enhanced <span class="trait">Resistance</span> to the Fungal Pathogen in Transgenic Soybean Plants</h2>
			<h3>PMID:27760122</h3>
				

				
					<h3>PMCID:5070833</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ngaki </surname>  <given-names> Micheline N. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Bing </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sahu </surname>  <given-names> Binod B. </given-names>  </name>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Srivastava </surname>  <given-names> Subodh K. </given-names>  </name>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farooqi </surname>  <given-names> Mohammad S. </given-names>  </name>  <xref ref-type="author-notes" rid="currentaff003">  <sup> &#x000a4;c </sup>  </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kambakam </surname>  <given-names> Sekhar </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Swaminathan </surname>  <given-names> Sivakumar </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-5639-9706 </contrib-id>  <name>  <surname> Bhattacharyya </surname>  <given-names> Madan K. </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 9 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Fusarium</span> virguliforme causes the serious disease sudden death syndrome (SDS) in soybean. Host <span class="trait">resistance</span> to this pathogen is partial and is encoded by a large number of quantitative trait loci, each conditioning small effects. Breeding SDS <span class="trait">resistance</span> is therefore challenging and identification of single-gene encoded novel <span class="trait">resistance</span> mechanisms is becoming a priority to fight this devastating this fungal pathogen. In this transcriptomic study we identified a few putative soybean defense genes, expression of which is suppressed during F. virguliforme <span class="trait">infection.</span> The F. virguliforme infection-suppressed genes were broadly classified into four major classes. The steady state transcript levels of many of these genes were suppressed to undetectable levels immediately following F. virguliforme <span class="trait">infection.</span> One of these classes contains two novel genes encoding ankyrin repeat-containing proteins. Expression of one of these genes, <span class="gene">GmARP1,</span> during F. virguliforme <span class="trait">infection</span> enhances SDS <span class="trait">resistance</span> among the transgenic soybean plants. Our data suggest that <span class="gene">GmARP1</span> is a novel defense gene and the pathogen presumably suppress its expression to establish compatible interaction.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="trait">Fusarium</span> virguliforme </italic> causes the serious disease sudden death syndrome (SDS) in soybean. Host <span class="trait">resistance</span> to this pathogen is partial and is encoded by a large number of quantitative trait loci, each conditioning small effects. Breeding SDS <span class="trait">resistance</span> is therefore challenging and identification of single-gene encoded novel <span class="trait">resistance</span> mechanisms is becoming a priority to fight this devastating this fungal pathogen. In this transcriptomic study we identified a few putative soybean defense genes, expression of which is suppressed during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> The <italic> F </italic> . <italic> virguliforme </italic> infection-suppressed genes were broadly classified into four major classes. The steady state transcript levels of many of these genes were suppressed to undetectable levels immediately following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> One of these classes contains two novel genes encoding ankyrin repeat-containing proteins. Expression of one of these genes, <italic> <span class="gene">GmARP1</span> </italic> , during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> enhances SDS <span class="trait">resistance</span> among the transgenic soybean plants. Our data suggest that <italic> <span class="gene">GmARP1</span> </italic> is a novel defense gene and the pathogen presumably suppress its expression to establish compatible interaction.
					</p>
				
					<p>&nbsp;&nbsp;
						We hypothesize that pathogens suppress defense-related genes to overcome potent host defense mechanisms to establish in host cells, multiply and spread. To our knowledge, no attempt has been made to alter the expression of down-regulated putative host defense genes to enhance disease <span class="trait">resistance</span> in transgenic plants. This study was undertaken primarily to uncover candidate defense-related genes repressed during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> and to determine if altered expression of such a gene can enhance <span class="trait">resistance</span> against <italic> F </italic> . <italic> virguliforme </italic> . We examined the expression profile of soybean genes in roots of young etiolated seedlings infected with <italic> F </italic> . <italic> virguliforme </italic> conidial spore suspensions or treated with sterile water. We observed that following inoculation with <italic> F </italic> . <italic> virguliforme </italic> , transcripts of more genes were up-regulated than down-regulated. We altered the expression of one member, of a family of two down-regulated genes, <italic> <span class="gene">GmARP1</span> </italic> and <italic> <span class="gene">GmARP2</span> </italic> , encoding ankyrin repeat-containing proteins during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> in transgenic soybean plants. Several independent transgenic soybean plants showing induced expression of <italic> <span class="gene">GmARP1</span> </italic> exhibited enhanced SDS <span class="trait">resistance.</span> Our study suggests that (i) <italic> F </italic> . <italic> virguliforme </italic> somehow suppresses defense-related genes to cause susceptibility and (ii) <italic> <span class="gene">GmARP1</span> </italic> encoding ankyrin repeats containing protein is a defense gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Three root specific and <span class="trait">infection</span> inducible promoters (Prom) and the <italic> <span class="gene">GmARP1</span> </italic> gene ( <xref ref-type="supplementary-material" rid="pone.0163106.s001"> S1 </xref> and <xref ref-type="supplementary-material" rid="pone.0163106.s002"> S2 </xref> Figs) were amplified from soybean cv. Williams 82 DNA. Prom 1 ( <italic> <span class="gene">Glyma18g47390</span> </italic> ) was discovered in our lab (B.B. Sahu and M.K. Bhattacharyya, unpublished); Prom 2 ( <italic> <span class="gene">Glyma10g31210</span> </italic> ) and Prom 3 ( <italic> <span class="gene">Glyma20g36300</span> </italic> ) are two root specific promoters, reported earlier ( <ext-link ext-link-type="uri" xlink:href="http://www.oardc.ohio-state.edu/SURE/GmROOT/GmRoot.htm"> http://www.oardc.ohio-state.edu/SURE/GmROOT/GmRoot.htm </ext-link> ). Genomic DNA was isolated using a modified CTAB extraction method [ <xref rid="pone.0163106.ref036" ref-type="bibr"> 36 </xref> ] adapted from Doyle and Doyle [ <xref rid="pone.0163106.ref037" ref-type="bibr"> 37 </xref> ]. Promoter sequences were amplified using the following pairs of primers: <italic> Prom1F </italic> - <italic> Prom1R </italic> ; <italic> Prom2F </italic> - <italic> Prom2R </italic> ; <italic> Prom3F </italic> - <italic> Prom3R </italic> ( <xref ref-type="supplementary-material" rid="pone.0163106.s009"> S1 Table </xref> ). The sequence of <italic> <span class="gene">GmARP1</span> </italic> was amplified using the primers <span class="gene">GmARP1G-F</span> and <span class="gene">GmARP1G-R.</span> In these primers, sequences in bold font indicate cloning sites. The binary vector pTF102 [ <xref rid="pone.0163106.ref038" ref-type="bibr"> 38 </xref> ] ( <xref ref-type="supplementary-material" rid="pone.0163106.s003"> S3 Fig </xref> ) was used to create three <italic> <span class="gene">GmARP1</span> </italic> transgenes: <italic> Prom1-GmARP1 </italic> , <italic> Prom2-GmARP1 </italic> , and <italic> Prom3-GmARP1 </italic> as follows. First, the CaMV 35S promoter was removed from pTF102 by digesting with <italic> Xba </italic> I and replaced it with any of the three new promoters. The restriction site for cloning <italic> <span class="gene">GmARP1</span> </italic> ( <italic> Bst </italic> XI) was inserted at the 3&#x02019;-end of the promoter primers. Next, we excised the <italic> GUS </italic> gene and CaMV 35S terminator (containing the Poly(A+) signal) by digesting with <italic> Bst </italic> XI and the <italic> Hind </italic> III. The CaMV 35S terminator was reinserted with addition of the <italic> Bst </italic> XI restriction site at the 5&#x02019;-end-specific PCR primer, and cloned in the <italic> Bst </italic> XI and <italic> Hind </italic> III sites. Finally, the created vectors were digested with <italic> Bst </italic> XI and the <italic> <span class="gene">GmARP1</span> </italic> sequence including 84 nucleotides upstream of the ATG start codon and 45 nucleotides beyond the TAA stop codon was inserted. The constructs were cloned in to <italic> Escherichia coli </italic> strain DH10B and sequenced to confirm their identity. The constructs were then transferred by electroporation into <italic> Agrobacterium tumefaciens </italic> strain EH101 for transformation of Williams 82 at the Plant Transformation Facility, Iowa State University. R <sub> 0 </sub> transgenic soybean plants carrying <italic> <span class="gene">GmARP1</span> </italic> transgenes were maintained in a greenhouse. R <sub> 1 </sub> seeds were harvested for further characterization in growth chambers and then under field conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						A field test of transgenic soybean plants was carried out in the Hinds Research Farm, Iowa State University located in north of Ames, Iowa between June 11 and October 30, 2015. Each transgenic line carrying <italic> <span class="gene">GmARP1</span> </italic> transgenes were grown in two replications along with the SDS resistant cultivar, MN1606, and the SDS susceptible transgene recipient line, Williams 82. Seeds of individual genotypes were mixed with <italic> F </italic> . <italic> virguliforme </italic> NE305S inoculum grown on sorghum grains during planting with a push planter. At 1 to 2-trifoliate stage, all transgenic lines were sprayed with basta herbicide (glufosinate at a 250 mg/L concentration) mixed with 0.1% Tween 20 twice with an interval of two days ( <xref ref-type="supplementary-material" rid="pone.0163106.s010"> S2 Table </xref> ). DNA samples were harvested from twelve plants that showed <span class="trait">resistance</span> to the herbicide. The plants were heavily irrigated in the last week of August that followed by heavy rains. SDS symptoms appeared following heavy rainfall and flood. Individual plants were scored on September 11 <sup> th </sup> , 22 <sup> nd </sup> , 30 <sup> th </sup> , and October 7 <sup> th </sup> , based on a scale of 1 to 9, with 1 being symptomless to 9 for severe symptoms with death of soybean plants ( <xref ref-type="supplementary-material" rid="pone.0163106.s011"> S3 Table </xref> ) ( <ext-link ext-link-type="uri" xlink:href="http://www.siu.edu/~soybean"> www.siu.edu/~soybean </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Ten day-old seedlings of cultivar Williams 82 were either treated with water (water treatment) or infected with <italic> F </italic> . <italic> virguliforme </italic> isolates. In order to monitor the expression of genes during <span class="trait">infection,</span> roots tissues were harvested at different time periods: (i) S1, early time period (ETP) of pooled root samples, 3 and 5 days following water treatment; (ii) S2, late time period (LTP) of pooled root samples, 10 and 25 days following water treatment; (iii) S3, ETP of pooled root samples, 3 and 5 days following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection;</span> (iv) S4, <span class="gene">LTP</span> of pooled root samples, 10 and 24 days following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> Total RNA samples were extracted from the root tissues and sequenced using Illumina HiSeq 2500 (Illumina, San Diego, CA) and deposited in GEO (accession GSE86201).
					</p>
				
					<p>&nbsp;&nbsp;
						We observed that 48 of the 54 genes induced in the infected roots of <span class="gene">LTP</span> were induced also during ETP ( <xref ref-type="fig" rid="pone.0163106.g001"> Fig 1B </xref> ; <xref ref-type="supplementary-material" rid="pone.0163106.s013"> S1 </xref> and <xref ref-type="supplementary-material" rid="pone.0163106.s015"> S3 </xref> Datasets). Functionally, 10 of the 50 genes up-regulated at both ETP and <span class="gene">LTP</span> belong to the cytochrome <span class="gene">P450</span> CYP2 subfamily protein involved in the synthesis of defense metabolites [ <xref rid="pone.0163106.ref051" ref-type="bibr"> 51 </xref> ]. Seven of these genes encode RmlC-like cupins with a nutrient reservoir activity; five genes encode peroxidases probably associated with signaling [ <xref rid="pone.0163106.ref052" ref-type="bibr"> 52 </xref> ]; four genes encode receptor-like kinases (RLKs) presumably to modulate the induction of immunity [ <xref rid="pone.0163106.ref053" ref-type="bibr"> 53 </xref> ]; three genes are members of the thaumatin family of pathogenesis-related protein that were shown to be induced during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> [ <xref rid="pone.0163106.ref021" ref-type="bibr"> 21 </xref> ], and three encode chitinase-related proteins involved in plant defense mechanisms [ <xref rid="pone.0163106.ref054" ref-type="bibr"> 54 </xref> , <xref rid="pone.0163106.ref055" ref-type="bibr"> 55 </xref> ]. Among the genes induced only in infected roots during <span class="gene">LTP</span> include: (i) a cupin-like gene, (ii) two defense/pathogenesis-related genes, and (iii) a gene with unknown function ( <xref ref-type="supplementary-material" rid="pone.0163106.s015"> S3 Dataset </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 51 genes repressed in soybean roots during ETP, only 15 with FC &#x02265;15 continued to be suppressed during <span class="gene">LTP</span> following <span class="trait">infection</span> ( <xref ref-type="fig" rid="pone.0163106.g001"> Fig 1C </xref> ; Tables <xref ref-type="table" rid="pone.0163106.t001"> 1 </xref> and <xref ref-type="table" rid="pone.0163106.t002"> 2 </xref> , <xref ref-type="supplementary-material" rid="pone.0163106.s014"> S2 Dataset </xref> ). These genes include <italic> Glyma </italic> . <italic> 02g054200 </italic> encoding a SAM (salicylic acid methyltransferase) dependent carboxyl methyltransferase, <italic> Glyma </italic> . <italic> 15g082200 </italic> encoding a cysteine proteinase Cathepsin F, <italic> Glyma </italic> . <italic> 13g282200 </italic> and <italic> Glyma </italic> . <italic> 13g282400 </italic> encoding wound-induced proteins, <italic> Glyma </italic> . <italic> 16g178000 </italic> encoding a lipid transfer protein, and <italic> Glyma </italic> . <italic> 06g294400 </italic> and <italic> Glyma </italic> . <italic> 12g111500 </italic> encoding ankyrin-repeat containing proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate down-regulation of a few selected genes during <span class="trait">infection,</span> we performed a semi-quantitative reverse transcriptase polymerase chain reaction (RT-PCR) for one member from each of the four selected classes of down-regulated genes ( <xref ref-type="table" rid="pone.0163106.t001"> Table 1 </xref> ). They are: (i) <italic> Glyma </italic> . <italic> 01g171600 </italic> encoding an uncharacterized salicylate o-methyltransferase (SAM)-like protein, we termed <italic> <span class="gene">GmSAM1</span> </italic> , most likely involved in the conversion of salicylic acid to the volatile methyl salicylate, a plant defense signal [ <xref rid="pone.0163106.ref056" ref-type="bibr"> 56 </xref> ] for defending attack of necrotrophic pathogens including <italic> F </italic> . <italic> virguliforme </italic> ; (ii) <italic> <span class="gene">Glyma12g12470</span> </italic> encoding an uncharacterized ankyrin repeat-containing protein, we termed it as <italic> <span class="gene">GmARP1;</span> </italic> (iii) <italic> Glyma </italic> . <italic> 10g185900 </italic> encodes a sieve element-occlusion protein (SEO)-like protein ( <ext-link ext-link-type="uri" xlink:href="http://www.uniprot.org/"> http://www.uniprot.org </ext-link> ; <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> http://www.phytozome.net </ext-link> ); and (iv) <italic> Glyma </italic> . <italic> 10g094800 </italic> encoding an uncharacterized, putative transmembrane protein [ <xref rid="pone.0163106.ref046" ref-type="bibr"> 46 </xref> ]( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> http://www.phytozome.net </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A. Expression levels of four selected soybean genes following water treatment at early (S1: 3 and 5 d) and late time (S2: 10 and 24 d) periods and <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> at early (S3: 3 and 5 d) and late time (S4: 10 and 24 d) periods. B. RT-PCR analyses of the selected soybean genes. RT-PCR products of each of the four selected soybean amplified from RNAs of root tissues harvested 8 and 12 h, and 1, 2, 3, 5 days following (i) water treatment or (ii) <span class="trait">infection</span> with the <italic> F </italic> . <italic> virguliforme </italic> Mont-1. The results presented here are from one of three independent experiments showing similar results. <italic> <span class="gene">Glyma12g12470</span> </italic> is from the Glyma.Wm82.a1.v1.1 version of the soybean genome sequence. Other three genes are from the recent version of the soybean genome sequence (Glyma.Wm82.a2v1). <italic> Elf1b </italic> , elongation factor 1-&#x003b2; encoded by <italic> <span class="gene">Glyma02g44460</span> </italic> . C&#x02013;F. Quantified expression levels of four selected genes. Gel pictures of the three biological replications are presented in <xref ref-type="supplementary-material" rid="pone.0163106.s008"> S8 Fig </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						We hypothesized that <italic> F </italic> . <italic> virguliforme </italic> down-regulates transcription of some of the defense-related genes to cause susceptibility in soybean. To test this hypothesis, we induced the expression of <italic> <span class="gene">GmARP1</span> </italic> ( <italic> <span class="gene">Glyma12g12470</span> </italic> ) in transgenic soybean plants following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> <italic> Glyma </italic> . <italic> 12g111500 </italic> and <italic> Glyma </italic> . <italic> 06g294400 </italic> are two highly similar genes that encode ankyrin repeats containing proteins with 76% identity ( <xref ref-type="supplementary-material" rid="pone.0163106.s004"> S4 </xref> and <xref ref-type="supplementary-material" rid="pone.0163106.s005"> S5 </xref> Figs), but with no identity to any functionally characterized ankyrin repeat-containing genes including <italic> <span class="gene">GmNPR1</span> </italic> [ <xref rid="pone.0163106.ref057" ref-type="bibr"> 57 </xref> ]. <span class="gene">GmARP1</span> protein contains 218 residues and slightly larger than all known ankyrin repeat-containing proteins including ANK superfamily with 71&#x02013;199 residues; ANK_2 with 81&#x02013;171; ANK_4 with 145&#x02013;199 residues.
					</p>
				
					<p>&nbsp;&nbsp;
						We used three <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">-infection</span> inducible promoters to induce the expression of <italic> <span class="gene">GmARP1</span> </italic> during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> Three fusion genes were generated by ligating <italic> <span class="gene">GmARP1</span> </italic> individually to these three promoters ( <xref ref-type="supplementary-material" rid="pone.0163106.s001"> S1A Fig </xref> ) and used to transform soybean cultivar &#x02018;Williams 82&#x02019;. R <sub> 0 </sub> plants were grown in a greenhouse and analyzed for integration of <italic> <span class="gene">GmARP1</span> </italic> transgenes by genomic PCR ( <xref ref-type="supplementary-material" rid="pone.0163106.s001"> S1B Fig </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						R <sub> 1 </sub> progenies of individual transformants (R <sub> 0 </sub> ) carrying any of the three fusion <italic> <span class="gene">GmARP1</span> </italic> genes were inoculated with <italic> F </italic> . <italic> virguliforme </italic> Mont-1 isolate in a growth chamber ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9A&#x02013;9E </xref> ). Approximately 30 to 50% of the R <sub> 1 </sub> plants showed no symptoms to slight yellowing with disease scores 1 and 2. The rest showed severe disease symptoms with interveinal to severe <span class="trait">chlorosis</span> and necrosis with disease scores 3 to 7 ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9B </xref> ). On the contrary, susceptible R <sub> 1 </sub> plants presumably lacking a functional transgene exhibited typical SDS foliar symptoms from the second week of <span class="trait">infection</span> and were severely diseased by the end of the fourth week following inoculation, with severe <span class="trait">chlorosis</span> and necrosis (disease scores 3 to 7), and reduced chlorophyll content as compared to the resistant R <sub> 1 </sub> progeny plants ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9C </xref> ). Moreover, resistant transgenic plants showed increased root weight as compared to the susceptible R <sub> 1 </sub> progenies implying root <span class="trait">resistance</span> of those resistant R <sub> 1 </sub> progenies to the pathogen ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9D </xref> ) and is supported by the levels of root <span class="trait">resistance</span> observed among the R <sub> 1 </sub> progenies ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9E </xref> ). Molecular characterization of the infected R <sub> 1 </sub> plants indicated that expression of foliar and root SDS <span class="trait">resistance</span> among the R <sub> 1 </sub> progenies of multiple transformants was associated with the expression of <italic> <span class="gene">GmARP1</span> </italic> transgenes ( <xref ref-type="fig" rid="pone.0163106.g009"> Fig 9F </xref> ). No transcript of the endogenous <italic> <span class="gene">GmARP1</span> </italic> gene was detected during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						R <sub> 1 </sub> plants were tested for <span class="trait">resistance</span> to <italic> F </italic> . <italic> virguliforme </italic> under growth chamber conditions. A. Root phenotype of a resistant (R) and a susceptible (S) R <sub> 1 </sub> progeny of a transformant, Prom2-ARP1-7, carrying the <italic> Prom2-GmARP1 </italic> fusion gene. B. Enhanced foliar SDS <span class="trait">resistance</span> among R <sub> 1 </sub> progenies. W82, the SDS susceptible line Williams 82; MN1606, the SDS resistant line. C. Chlorophyll content per individual R <sub> 1 </sub> progeny carrying <italic> <span class="gene">GmARP1</span> </italic> of three independent transformants. &#x02018;Resistant&#x02019; and &#x02018;Susceptible&#x02019; classes are defined as in (A). D. Average root weight of R <sub> 1 </sub> progeny of three independent transformants. &#x02018;Resistant&#x02019; and &#x02018;Susceptible&#x02019; classes are defined as in A. E. Enhanced root <span class="trait">resistance</span> among R <sub> 1 </sub> progenies. Extent of root <span class="trait">resistance</span> to the pathogen was expressed in percent; e.g., 100%, healthy roots with no obvious blackening caused by necrosis and rotting due to <span class="trait">infection</span> of <italic> F </italic> . <italic> virguliforme </italic> . F. Expression of <italic> <span class="gene">GmARP1</span> </italic> transgenes. Two random SDS resistant and susceptible R <sub> 1 </sub> progenies from each R <sub> 0 </sub> line were analyzed. Top panel, resistant plants (two representatives from each line). Bottom panel, susceptible plants (two representatives from each line). Red arrow, <italic> <span class="gene">GmARP1</span> </italic> ; black arrow, <italic> ELF1b </italic> internal control. *, significantly different at <italic> p&#x0003c;0 </italic> . <italic> 01 </italic> . Results are means &#x000b1;SE of three independent experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						We evaluated whether <italic> <span class="gene">GmARP1</span> </italic> transgenes could provide enhanced SDS <span class="trait">resistance</span> in transgenic plants under field conditions. We observed that 70 to 100% of the transgenic R <sub> 1 </sub> plants descended from six independent transgenic R <sub> 0 </sub> plants showed enhanced SDS <span class="trait">resistance</span> under field conditions and did not develop SDS symptoms. Non-trangenic Williams 82 control plants developed severe SDS symptoms including <span class="trait">chlorosis</span> and necrosis of leaves that caused total defoliation of plants before maturity stage ( <xref ref-type="fig" rid="pone.0163106.g010"> Fig 10A and 10C </xref> ; <xref ref-type="supplementary-material" rid="pone.0163106.s006"> S6 Fig </xref> ). The transgene copy number assay conducted using qPCR revealed that all six transgenic lines contain at least one copy of the transgene, sufficient to enhance <span class="trait">resistance</span> against <italic> F </italic> . <italic> virguliforme </italic> ( <xref ref-type="table" rid="pone.0163106.t003"> Table 3 </xref> ; <xref ref-type="supplementary-material" rid="pone.0163106.s016"> S4 Dataset </xref> ). The line Prom2-ARP1-9 carrying single copy <italic> Prom2-ARP1 </italic> transgene showed 100% resistant R <sub> 1 </sub> progenies; whereas, Prom2-ARP1-9 line carrying on the average 4 copies of the same <italic> Prom2-ARP1 </italic> transgene showed the lowest number SDS resistant plants (70%).
					</p>
				
					<p>&nbsp;&nbsp;
						R <sub> 1 </sub> plants were tested for <span class="trait">resistance</span> to <italic> F </italic> . <italic> virguliforme </italic> under field conditions. A. Average disease scores gathered at four scoring dates for R <sub> 1 </sub> progenies of each transgenic line carrying <italic> <span class="gene">GmARP1</span> </italic> under the control of Prom1, Prom2, or Prom3 promoters. n, number of R <sub> 1 </sub> plants. B. Average disease scores of Williams 82 (susceptible control) and MN1606 (resistant control). n, number of plants and results in A and B are means &#x000b1;SE of n plants. C. Percentage of R <sub> 1 </sub> plants resistant to <italic> F </italic> . <italic> virguliforme </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the Arabidopsis- <italic> F </italic> . <italic> oxysporum </italic> interaction a large number of genes were repressed as opposed to induction as observed in most plant-pathogen interactions [ <xref rid="pone.0163106.ref066" ref-type="bibr"> 66 </xref> ]. Despite a large number of soybean genes were up-regulated upon <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection,</span> in our analysis of the soybean- <italic> F </italic> . <italic> virguliforme </italic> interaction revealed that 70 soybean genes were suppressed following <span class="trait">infection.</span> Of these, only four genes were down-regulated in both ETP and <span class="gene">LTP</span> following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> ( <xref ref-type="fig" rid="pone.0163106.g001"> Fig 1C </xref> ; <xref ref-type="supplementary-material" rid="pone.0163106.s014"> S2 Dataset </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Among the genes down-regulated following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection,</span> seven soybean genes encode glycosyl hydrolases ( <xref ref-type="table" rid="pone.0163106.t001"> Table 1 </xref> ). Of these three genes <italic> Glyma </italic> . <italic> 03g024400 </italic> , <italic> Glyma </italic> . <italic> 03g024300 </italic> , and <italic> Glyma </italic> . <italic> 03g024200 </italic> encoding the uncharacterized glycosyl hydrolase family 18 were highly expressed during ETP of the water treated root tissues (RPKM &#x0003e;1,102). The expression of these genes was strongly suppressed upon <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> (FC &#x0003e;65) ( <xref ref-type="table" rid="pone.0163106.t001"> Table 1 </xref> ). Members of the glycosyl hydrolase 18 with high identity to some pathogen related ( <italic> PR </italic> ) genes are not only implicated in metabolic processes of cell wall, but also considered to have defense and signaling functions [ <xref rid="pone.0163106.ref067" ref-type="bibr"> 67 </xref> &#x02013; <xref rid="pone.0163106.ref070" ref-type="bibr"> 70 </xref> ]. The glycosyl hydrolases encoded by <italic> Glyma </italic> . <italic> 03g024400 </italic> , <italic> Glyma </italic> . <italic> 03g024300 </italic> , and <italic> Glyma </italic> . <italic> 03g024200 </italic> are 100% identical and contain a GH18_chitinase-like domain; therefore most likely they function in plant defense against pathogens with chitin molecules [ <xref rid="pone.0163106.ref071" ref-type="bibr"> 71 </xref> , <xref rid="pone.0163106.ref072" ref-type="bibr"> 72 </xref> ]. Four additional soybean genes, <italic> <span class="gene">Glyma03g02810</span> </italic> , <italic> Glyma </italic> . <italic> 03g025000 Glyma </italic> . <italic> 03g024800 </italic> , and <italic> Glyma </italic> . <italic> 08g285400 </italic> encoding glycosyl hydrolases with moderate expression levels (FC from 6 to 56), were also repressed during ETP following <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						More than half of plant genes encode proteins of unknown functions [ <xref rid="pone.0163106.ref076" ref-type="bibr"> 76 </xref> ]. We observed down-regulation of many genes encoding proteins of unknown functions during both ETP and <span class="gene">LTP</span> (FC &#x0003e; 10; Tables <xref ref-type="table" rid="pone.0163106.t001"> 1 </xref> and <xref ref-type="table" rid="pone.0163106.t002"> 2 </xref> ). During LTP, four genes have RPKM values in water treated roots over 1,000 (1,375 to 32,802) and are strongly repressed (FC &#x02265; 20) in <italic> F </italic> . <italic> virguliforme </italic> infected roots. The most highly expressed, <italic> <span class="gene">Glyma17g03850</span> </italic> (RPKM in water treated roots 32,802 and repressed after <span class="trait">infection</span> to FC = 24.0), is an orphan gene of <italic> Glycine max </italic> [ <xref rid="pone.0163106.ref077" ref-type="bibr"> 77 </xref> ] ( <ext-link ext-link-type="uri" xlink:href="http://www.greenphyl.org/"> http://www.greenphyl.org/ </ext-link> ) with no annotated functional domain. This gene is only co-expressed with <italic> Glyma </italic> . <italic> 10G216000 </italic> (correlation 0.87, <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.com/"> www.phytozome.com </ext-link> ) that encodes a gibberellin-regulated protein 2 involved in gibberellic acid mediated signaling pathway [ <xref rid="pone.0163106.ref078" ref-type="bibr"> 78 </xref> ]. Similarly, infection-suppressed gene <italic> Glyma </italic> . <italic> 10g232100 </italic> (RPKM 2,130 in water treated roots vs. FC 27 following <span class="trait">infection)</span> encoding an unknown protein also co-expressed with the gibberellin-regulated protein 2. Finally, <italic> Glyma </italic> . <italic> 17g039400 </italic> (RPKM 2,206 in water treated roots vs. FC 24 following <span class="trait">infection).</span> It is highly co-expressed with a fasciclin-like arabinogalactan protein gene (correlation 0.88, <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.com/"> www.phytozome.com </ext-link> ) implicated in the cell-wall composition [ <xref rid="pone.0163106.ref079" ref-type="bibr"> 79 </xref> ]. Thus, these genes with unknown functions could be involved in plant immunity. Future studies on these four genes might <span class="trait">yield</span> new insights about the soybean- <italic> F </italic> . <italic> virguliforme </italic> interaction.
					</p>
				
					<p>&nbsp;&nbsp;
						Aknyrin repeat-containing proteins widely exist in plants [ <xref rid="pone.0163106.ref080" ref-type="bibr"> 80 </xref> ]. We identified five <italic> <span class="gene">GmARP1</span> </italic> homologs in <italic> G </italic> . <italic> soja </italic> and <italic> Phaseolus vulgaris </italic> exhibiting &#x02265; 62% identities among them, specifically in the ankyrin domain ( <xref ref-type="supplementary-material" rid="pone.0163106.s007"> S7 Fig </xref> ). Two to more than 20 ankyrin-repeats involved in protein-protein interactions could be present in ankyrin-repeat containing proteins [ <xref rid="pone.0163106.ref081" ref-type="bibr"> 81 </xref> &#x02013; <xref rid="pone.0163106.ref083" ref-type="bibr"> 83 </xref> ]. Ankyrin-repeats are involved in many metabolic processes including defense against pathogens and signaling [ <xref rid="pone.0163106.ref081" ref-type="bibr"> 81 </xref> , <xref rid="pone.0163106.ref084" ref-type="bibr"> 84 </xref> &#x02013; <xref rid="pone.0163106.ref086" ref-type="bibr"> 86 </xref> ]. The ankyrin repeat-containing ACD6 is implicated in salicylic acid signaling in Arabidopsis [ <xref rid="pone.0163106.ref087" ref-type="bibr"> 87 </xref> ]. The ankyrin repeat-containing protein CaKR1 in <italic> Capsicum annuum </italic> is responsive to both abiotic and biotic stresses [ <xref rid="pone.0163106.ref088" ref-type="bibr"> 88 </xref> ]. Ankyrin repeat-containing protein NPR1 modulates plant immunity cooperatively with transcription factors [ <xref rid="pone.0163106.ref089" ref-type="bibr"> 89 </xref> ]. In Arabidopsis, BDA1 containing ankyrin repeats has been shown to regulate immunity [ <xref rid="pone.0163106.ref090" ref-type="bibr"> 90 </xref> ]. Similarly, Mou and colleagues [ <xref rid="pone.0163106.ref060" ref-type="bibr"> 60 </xref> ] reported that overexpression of the aknyrin repeat-containing protein gene <italic> OsPIANK1 </italic> enhanced immunity against <italic> Magnaporthe oryzae </italic> in transgenic rice. <italic> <span class="gene">GmARP1</span> </italic> ( <italic> <span class="gene">Glyma12g12470)</span> </italic> has been shown to have a protein binding function and putatively localized to plasma membrane [ <xref rid="pone.0163106.ref080" ref-type="bibr"> 80 </xref> ]. Here we have shown that the steady state transcript level of this gene is not detectable immediately after <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> ( <xref ref-type="fig" rid="pone.0163106.g008"> Fig 8 </xref> ). Expression of this gene using infection-inducible promoters in transgenic plants enhanced <span class="trait">resistance</span> to the SDS pathogen, indicating that like other known aknyrin repeat-containing protein genes, <italic> <span class="gene">GmARP1</span> </italic> is involved in soybean defense against this fungal pathogen.
					</p>
				
					<p>&nbsp;&nbsp;
						Our transcriptomic study revealed many genes that are induced, whereas expression of only a few genes including a previously uncharacterized <italic> <span class="gene">GmARP1</span> </italic> gene is strongly suppressed by <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> Many of the genes that are suppressed during <span class="trait">infection</span> might be defense-related and, are somehow suppressed by pathogens to establish the compatible interaction or to cause susceptibility. This hypothesis was tested by expressing <italic> <span class="gene">GmARP1</span> </italic> during <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection</span> in transgenic soybean lines. We observed that expression of <italic> <span class="gene">GmARP1</span> </italic> led to induction of foliar SDS <span class="trait">resistance.</span> How <italic> <span class="gene">GmARP1</span> </italic> regulates immunity against SDS pathogen is yet to be determined. It is also unknown how the gene is down-regulated by <italic> F </italic> . <italic> virguliforme </italic> <span class="trait">infection.</span> Our transgenic study suggests that altered expression of pathogen-repressed host genes could be a suitable strategy in engineering disease <span class="trait">resistance</span> in crop plants.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The plant immunity inducer pipecolic acid accumulates in the xylem sap and leaves of soybean seedlings following <span class="trait">Fusarium</span> virguliforme <span class="trait">infection.</span></h2>
			<h3>PMID:26795155  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abeysekara NS(1), Swaminathan S(2), Desai N(3), Guo L(3), Bhattacharyya MK(4).
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The causal agent of the soybean sudden death syndrome (SDS), <span class="trait">Fusarium</span> virguliforme, remains in infected roots and secretes toxins to cause foliar SDS. In this study we investigated the xylem sap, roots, and leaves of F. virguliforme-infected and -uninfected soybean seedlings for any changes in a set of over 3,000 metabolites following pathogen <span class="trait">infection</span> by conducting GC/MS and LC/MS/MS, and detected 273 biochemicals. Levels of many intermediates of the TCA cycle were reduced suggesting suppression of this metabolic pathway by the pathogen. There was an increased accumulation of peroxidated lipids in leaves of F. virguliforme-infected plants suggesting possible involvement of free radicals and lipoxygenases in foliar SDS development. Levels of both isoflavone conjugates and isoflavonoid phytoalexins were decreased in infected roots suggesting degradation of these metabolites by the pathogen to promote root necrosis. The levels of the plant immunity inducer pipecolic acid (Pip) and the plant hormone salicylic acid (SA) were significantly increased in xylem sap (in case of Pip) and leaves (in case of both Pip and SA) of F. virguliforme-infected soybean plants compared to the control plants. This suggests a major signaling role of Pip in inducing host defense responses in above ground parts of the F. virguliforme-infected soybean. Increased accumulation of pipecolic acid in foliar tissues was associated with the induction of <span class="gene">GmALD1,</span> the soybean homolog of Arabidopsis ALD1. This metabolomics study generated several novel hypotheses for studying the mechanisms of SDS development in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aggressiveness of <span class="trait">Fusarium</span> species and impact of root <span class="trait">infection</span> on growth and <span class="trait">yield</span> of soybeans.</h2>
			<h3>PMID:23514263  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Arias MM(1), Leandro LF, Munkvold GP.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Fusarium</span> spp. are commonly isolated from soybean roots but the pathogenic activity of most species is poorly documented. Aggressiveness and <span class="trait">yield</span> impact of nine species of <span class="trait">Fusarium</span> were determined on soybean in greenhouse (50 isolates) and field microplot (19 isolates) experiments. Root rot severity and shoot and root dry weights were compared at growth stages <span class="gene">V3</span> or R1. Root systems were scanned and digital image analysis was conducted; <span class="trait">yield</span> was measured in microplots. Disease severity and root morphology impacts varied among and within species. <span class="trait">Fusarium</span> graminearum was highly aggressive (root rot severity &gt;90%), followed by F. proliferatum and F. virguliforme. Significant variation in damping-off (20 to 75%) and root rot severity (&lt;20 to &gt;60%) was observed among F. oxysporum isolates. In artificially-infested microplots, root rot severity was low (&lt;25%) and mean <span class="trait">yield</span> was not significantly reduced. However, there were significant linear relationships between <span class="trait">yield</span> and root symptoms for some isolates. Root morphological characteristics were more consistent indicators of <span class="trait">yield</span> loss than root rot severity. This study provides the first characterization of aggressiveness and <span class="trait">yield</span> impact of <span class="trait">Fusarium</span> root rot species on soybean at different plant stages and introduces root image analysis to assess the impact of root pathogens on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homo-dimerization and ligand binding by the leucine-rich repeat domain at RHG1/RFS2 underlying <span class="trait">resistance</span> to two soybean pathogens.</h2>
			<h3>PMID:23497186  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Afzal AJ(1), Srour A, Goil A, Vasudaven S, Liu T, Samudrala R, Dogra N, Kohli P, Malakar A, Lightfoot DA.
				<br/> 
				年份：2013 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The protein encoded by <span class="gene">GmRLK18-1</span> (Glyma_18_02680 on chromosome 18) was a receptor like kinase (RLK) encoded within the soybean (Glycine max L. Merr.) Rhg1/Rfs2 locus. The locus underlies <span class="trait">resistance</span> to the soybean cyst nematode (SCN) Heterodera glycines (I.) and causal agent of sudden death syndrome (SDS) <span class="trait">Fusarium</span> virguliforme (Aoki). Previously the leucine rich repeat (LRR) domain was expressed in Escherichia coli. RESULTS: The aims here were to evaluate the LRRs ability to; homo-dimerize; bind larger proteins; and bind to small peptides. Western analysis suggested homo-dimers could form after protein extraction from roots. The purified LRR domain, from residue 131-485, was seen to form a mixture of monomers and homo-dimers in vitro. Cross-linking experiments in vitro showed the H274N region was close (&lt;11.1 A) to the highly conserved cysteine residue C196 on the second homo-dimer subunit. Binding constants of 20-142 nM for peptides found in plant and nematode secretions were found. Effects on plant phenotypes including wilting, stem bending and <span class="trait">resistance</span> to <span class="trait">infection</span> by SCN were observed when roots were treated with 50 pM of the peptides. Far-Western analyses followed by <span class="gene">MS</span> showed methionine synthase and cyclophilin bound strongly to the LRR domain. A second LRR from <span class="gene">GmRLK08-1</span> (Glyma_08_g11350) did not show these strong interactions. CONCLUSIONS: The LRR domain of the <span class="gene">GmRLK18-1</span> protein formed both a monomer and a homo-dimer. The LRR domain bound avidly to 4 different CLE peptides, a cyclophilin and a methionine synthase. The CLE peptides <span class="gene">GmTGIF,</span> <span class="gene">GmCLE34,</span> <span class="gene">GmCLE3</span> and HgCLE were previously reported to be involved in root growth inhibition but here <span class="gene">GmTGIF</span> and HgCLE were shown to alter stem morphology and <span class="trait">resistance</span> to SCN. One of several models from homology and ab-initio modeling was partially validated by cross-linking. The effect of the 3 amino acid replacements present among <span class="gene">RLK</span> allotypes, A87V, Q115K and H274N were predicted to alter domain stability and function. Therefore, the LRR domain of <span class="gene">GmRLK18-1</span> might underlie both root development and disease <span class="trait">resistance</span> in soybean and provide an avenue to develop new variants and ligands that might promote reduced losses to SCN.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Separate loci underlie <span class="trait">resistance</span> to root <span class="trait">infection</span> and leaf scorch during soybean sudden death syndrome.</h2>
			<h3>PMID:18324383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kazi S(1), Shultz J, Afzal J, Johnson J, Njiti VN, Lightfoot DA.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] cultivars show differences in their <span class="trait">resistance</span> to both the leaf scorch and root rot of sudden death syndrome (SDS). The syndrome is caused by root colonization by <span class="trait">Fusarium</span> virguliforme (ex. F. solani f. sp. glycines). Root susceptibility combined with reduced leaf scorch <span class="trait">resistance</span> has been associated with <span class="trait">resistance</span> to Heterodera glycines HG Type 1.3.6.7 (race 14) of the soybean cyst nematode (SCN). In contrast, the rhg1 locus underlying <span class="trait">resistance</span> to Hg Type 0 was found clustered with three loci for <span class="trait">resistance</span> to SDS leaf scorch and one for root <span class="trait">infection.</span> The aims of this study were to compare the inheritance of <span class="trait">resistance</span> to leaf scorch and root <span class="trait">infection</span> in a population that segregated for <span class="trait">resistance</span> to SCN and to identify the underlying quantitative trait loci (QTL). "Hartwig", a cultivar partially resistant to SDS leaf scorch, F. virguliforme root <span class="trait">infection</span> and SCN HG Type 1.3.6.7 was crossed with the partially susceptible cultivar "Flyer". Ninety-two F5-derived recombinant inbred lines and 144 markers were used for map development. Four QTL found in earlier studies were confirmed. One contributed <span class="trait">resistance</span> to leaf scorch on linkage group (LG) <span class="gene">C2</span> (Satt277; P = 0.004, R2 = 15%). Two on LG G underlay root <span class="trait">infection</span> at R8 (Satt038; P = 0.0001 R2 = 28.1%; Satt115; P = 0.003, R2 = 12.9%). The marker Satt038 was linked to rhg1 underlying <span class="trait">resistance</span> to SCN Hg Type 0. The fourth QTL was on LG D2 underlying <span class="trait">resistance</span> to root <span class="trait">infection</span> at R6 (Satt574; P = 0.001, R2 = 10%). That QTL was in an interval previously associated with <span class="trait">resistance</span> to both SDS leaf scorch and SCN Hg Type 1.3.6.7. The QTL showed repulsion linkage with <span class="trait">resistance</span> to SCN that may explain the relative susceptibility to SDS of some SCN resistant cultivars. One additional QTL was discovered on LG G underlying <span class="trait">resistance</span> to SDS leaf scorch measured by disease index (Satt130; P = 0.003, R2 = 13%). The loci and markers will provide tagged alleles with which to improve the breeding of cultivars combining resistances to SDS leaf scorch, root <span class="trait">infection</span> and SCN HG Type 1.3.6.7.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A pyramid of loci for partial <span class="trait">resistance</span> to <span class="trait">Fusarium</span> solani f. sp. glycines maintains Myo-inositol-1-phosphate synthase expression in soybean roots.</h2>
			<h3>PMID:12582889</h3>
				

				
			<p>
				作者：Iqbal J(1), Afzal J, Yaegashi S, Ruben E, Triwitayakorn K, Njiti N, Ahsan R, Wood J, Lightfoot A.
				<br/> 
				年份：2002 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Myo-inositol 1-phosphate synthase (MIPS; EC 5.5.1.4) converts glucose 6-phosphate to myo-inositol 1-phosphate in the presence of NAD(+). It catalyzes the first step in the synthesis of myo-inositol and pinitol, and is a rate limiting step in the de novo biosynthesis of inositol in eukaryotes. Therefore, <span class="gene">MIPS</span> is involved in biotic and abiotic stress via Ca(2+) signalling. Seedlings of four soybean genotypes were inoculated with <span class="trait">Fusarium</span> solani f. sp. glycines, the causative agent of sudden death syndrome (SDS), and differentially abundant mRNAs were identified by differential display. The genotypes carried either zero, two, four or six alleles of the quantitative trait loci (QTLs) that control <span class="trait">resistance</span> to SDS in an additive manner. The mRNA abundance of <span class="gene">MIPS</span> did not decrease following inoculation in a recombinant inbred line (RIL 23) containing all six <span class="trait">resistance</span> alleles of the QTLs conferring <span class="trait">resistance</span> to SDS of soybean. However, the abundance of <span class="gene">MIPS</span> mRNA was decreased in genotypes containing four, two or no <span class="trait">resistance</span> alleles. The specific activity of the <span class="gene">MIPS</span> enzyme in vitro followed the same pattern across genotypes. The IP(3) content in the inoculated roots of genotypes with two, four or six <span class="trait">resistance</span> alleles were higher compared to the non-inoculated root. The results suggests that a non-additive effect on transcription and translation of <span class="gene">MIPS</span> is established in RIL 23 roots by pyramiding six QTLs for <span class="trait">resistance</span> to SDS. A role of <span class="gene">MIPS</span> in the partial <span class="trait">resistance</span> or response of soybean roots to F. solani <span class="trait">infection</span> is suggested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Involvement of specific calmodulin isoforms in salicylic acid-independent activation of plant disease <span class="trait">resistance</span> responses.</h2>
			<h3>PMID:9892708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heo WD(1), Lee SH, Kim MC, Kim JC, Chung WS, Chun HJ, Lee KJ, Park CY, Park HC, Choi JY, Cho MJ.
				<br/> 
				年份：1999 Jan 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Ca2+ signal is essential for the activation of plant defense responses, but downstream components of the signaling pathway are still poorly defined. Here we demonstrate that specific calmodulin (CaM) isoforms are activated by <span class="trait">infection</span> or pathogen-derived elicitors and participate in Ca2+-mediated induction of plant disease <span class="trait">resistance</span> responses. Soybean <span class="gene">CaM</span> (SCaM)-4 and <span class="gene">SCaM-5</span> genes, which encode for divergent <span class="gene">CaM</span> isoforms, were induced within 30 min by a fungal elicitor or pathogen, whereas other SCaM genes encoding highly conserved <span class="gene">CaM</span> isoforms did not show such response. This pathogen-triggered induction of these genes specifically depended on the increase of intracellular Ca2+ level. Constitutive expression of <span class="gene">SCaM-4</span> and <span class="gene">SCaM-5</span> in transgenic tobacco plants triggered spontaneous induction of lesions and induces an array of systemic acquired <span class="trait">resistance</span> (SAR)-associated genes. Surprisingly, these transgenic plants have normal levels of endogenous salicylic acid (SA). Furthermore, coexpression of <span class="gene">nahG</span> gene did not block the induction of SAR-associated genes in these transgenic plants, indicating that SA is not involved in the SAR gene induction mediated by <span class="gene">SCaM-4</span> or SCaM-5. The transgenic plants exhibit enhanced <span class="trait">resistance</span> to a wide spectrum of virulent and avirulent pathogens, including bacteria, fungi, and virus. These results suggest that specific <span class="gene">CaM</span> isoforms are components of a SA-independent signal transduction chain leading to disease <span class="trait">resistance.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>