<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Heterodera glycines</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Heterodera glycines</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:A harpin elicitor induces the expression of a coiled-coil nucleotide binding leucine rich repeat (CC-NB-LRR) defense signaling gene and others functioning during defense to parasitic nematodes.</h2>
			<h3>PMID:29107936</h3>
				

				
			<p>
				作者：Aljaafri WAR(1), McNeece BT(2), Lawaju BR(3), Sharma K(4), Niruala PM(5), Pant SR(6), Long DH(7), Lawrence KS(8), Lawrence GW(9), Klink VP(10).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The bacterial effector harpin induces the transcription of the Arabidopsis thaliana NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1/HARPIN INDUCED1 (NDR1/HIN1) coiled-coil nucleotide binding leucine rich repeat (CC-NB-LRR) defense signaling gene. In Glycine max, <span class="gene">Gm-NDR1-1</span> transcripts have been detected within root cells undergoing a natural resistant reaction to parasitism by the syncytium-forming nematode <span class="trait">Heterodera</span> <span class="trait">glycines,</span> functioning in the defense response. Expressing <span class="gene">Gm-NDR1-1</span> in Gossypium hirsutum leads to <span class="trait">resistance</span> to Meloidogyne incognita parasitism. In experiments presented here, the heterologous expression of <span class="gene">Gm-NDR1-1</span> in G. hirsutum impairs Rotylenchulus reniformis parasitism. These results are consistent with the hypothesis that <span class="gene">Gm-NDR1-1</span> expression functions broadly in generating a defense response. To examine a possible relationship with harpin, G. max plants topically treated with harpin result in induction of the transcription of <span class="gene">Gm-NDR1-1.</span> The result indicates the topical treatment of plants with harpin, itself, may lead to impaired nematode parasitism. Topical harpin treatments are shown to impair G. max parasitism by H. <span class="trait">glycines,</span> M. incognita and R. reniformis and G. hirsutum parasitism by M. incognita and R. reniformis. How harpin could function in defense has been examined in experiments showing it also induces transcription of G. max homologs of the proven defense genes ENHANCED DISEASE SUSCEPTIBILITY1 (EDS1), TGA2, galactinol synthase, reticuline oxidase, xyloglucan endotransglycosylase/hydrolase, alpha soluble N-ethylmaleimide-sensitive fusion protein (α-SNAP) and serine hydroxymethyltransferase (SHMT). In contrast, other defense genes are not directly transcriptionally activated by harpin. The results indicate harpin induces pathogen associated molecular pattern (PAMP) triggered immunity (PTI) and effector-triggered immunity (ETI) defense processes in the root, activating defense to parasitic nematodes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of wild soybean (Glycine soja) response to soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines).</span></h2>
			<h3>PMID:28710561  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang H(1), Song Q(2), Griffin JD(3), Song BH(4).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean cyst nematode (SCN) is one of the most destructive pathogens of soybean plants worldwide. Host-plant <span class="trait">resistance</span> is an environmentally friendly method to mitigate SCN damage. To date, the resistant soybean cultivars harbor limited genetic variation, and some are losing <span class="trait">resistance.</span> Thus, a better understanding of the genetic mechanisms of the SCN <span class="trait">resistance,</span> as well as developing diverse resistant soybean cultivars, is urgently needed. In this study, a genome-wide association study (GWAS) was conducted using 1032 wild soybean (Glycine soja) accessions with over 42,000 single-nucleotide polymorphisms (SNPs) to understand the genetic architecture of G. soja <span class="trait">resistance</span> to SCN race 1. Ten SNPs were significantly associated with the response to race 1. Three SNPs on chromosome 18 were localized within the previously identified quantitative trait loci (QTLs), and two of which were localized within a strong linkage disequilibrium block encompassing a nucleotide-binding (NB)-ARC disease <span class="trait">resistance</span> gene <span class="gene">(Glyma.18G102600).</span> Genes encoding methyltransferases, the calcium-dependent signaling protein, the leucine-rich repeat kinase family protein, and the NB-ARC disease <span class="trait">resistance</span> protein, were identified as promising candidate genes. The identified SNPs and candidate genes can not only shed light on the molecular mechanisms underlying SCN <span class="trait">resistance,</span> but also can facilitate soybean improvement employing wild genetic resources.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant elicitor peptides promote plant defences against nematodes in soybean.</h2>
			<h3>PMID:28600875</h3>
				

				
			<p>
				作者：Lee MW(1), Huffaker A(2), Crippen D(3), Robbins RT(3), Goggin FL(1).
				<br/> 
				年份：2017 Jun 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant elicitor peptides (Peps) are widely distributed among angiosperms, and have been shown to amplify immune responses in multiple plant families. Here, we characterize three Peps from soybean (Glycine max) and describe their effects on plant defences against two damaging agricultural pests, the root-knot nematode (Meloidogyne incognita) and the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines).</span> Seed treatments with exogenous <span class="gene">GmPep1,</span> <span class="gene">GmPep2</span> or <span class="gene">GmPep3</span> significantly reduced the reproduction of both nematodes. Pep treatment also protected plants from the inhibitory effects of root-knot nematodes on above-ground growth, and up-regulated basal expression levels of nematode-responsive defence genes. <span class="gene">GmPep1</span> induced the expression of its propeptide precursor <span class="gene">(GmPROPEP1),</span> a nucleotide-binding site leucine-rich repeat protein (NBS-LRR), a pectin methylesterase inhibitor (PMEI), Respiratory Burst Oxidase Protein D (RBOHD) and the accumulation of reactive oxygen species (ROS) in leaves. In addition, <span class="gene">GmPep2</span> and <span class="gene">GmPep3</span> seed treatments up-regulated RBOHD expression and ROS accumulation in roots and leaves. These results suggest that <span class="gene">GmPeps</span> activate plant defences through systemic transcriptional reprogramming and ROS signalling, and that Pep seed treatments represent a potential strategy for nematode management.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <italic> <span class="gene">GmSNAP18</span> </italic> gene underlies two types of <span class="trait">resistance</span> to soybean cyst nematode</h2>
			<h3>PMID:28345654</h3>
				

				
					<h3>PMCID:5378975</h3>
					<h4>期刊:Nature Communications</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kandoth </surname>  <given-names> Pramod K. </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lakhssassi </surname>  <given-names> Naoufal </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Jingwen </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Colantonio </surname>  <given-names> Vincent </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heinz </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yeckel </surname>  <given-names> Greg </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Zhou </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bekal </surname>  <given-names> Sadia </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dapprich </surname>  <given-names> Johannes </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rotter </surname>  <given-names> Bjorn </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cianzio </surname>  <given-names> Silvia </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Plant, Soil and Agricultural Systems, Southern Illinois University </institution> , 1205 Lincoln Drive RM176, Carbondale, Illinois 62901,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Division of Plant Sciences and Bond Life Sciences Center, University of Missouri </institution> , Columbia, Missouri 65211,  <country> USA </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Generation Biotech </institution> , Lawrenceville, New Jersey 08648,  <country> USA </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> GenXPro-GmbH </institution> , Altenh&#x000f6;ferallee 3, 60438 Frankfurt am Main,  <country> Germany </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Department of Agronomy, Iowa State University </institution> , Ames, Iowa 50011,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 06 </day>  <month> 02 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two types of resistant soybean (Glycine max (L.) Merr.) sources are widely used against soybean cyst nematode (SCN, <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe). These include Peking-type soybean, whose <span class="trait">resistance</span> requires both the rhg1-a and <span class="gene">Rhg4</span> alleles, and PI 88788-type soybean, whose <span class="trait">resistance</span> requires only the rhg1-b allele. Multiple copy number of PI 88788-type <span class="gene">GmSNAP18,</span> <span class="gene">GmAAT,</span> and <span class="gene">GmWI12</span> in one genomic segment simultaneously contribute to rhg1-b <span class="trait">resistance.</span> Using an integrated set of genetic and genomic approaches, we demonstrate that the rhg1-a Peking-type <span class="gene">GmSNAP18</span> is sufficient for <span class="trait">resistance</span> to SCN in combination with Rhg4. The two SNAPs (soluble NSF attachment proteins) differ by only five amino acids. Our findings suggest that Peking-type <span class="gene">GmSNAP18</span> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <span class="gene">GmSNAP18.</span> As such, this is an example of a pathogen <span class="trait">resistance</span> gene that has evolved to underlie two types of <span class="trait">resistance,</span> yet ensure the same function within a single plant species.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Two types of resistant soybean ( <italic> Glycine max </italic> (L.) Merr.) sources are widely used against soybean cyst nematode (SCN, <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe). These include Peking-type soybean, whose <span class="trait">resistance</span> requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles, and PI 88788-type soybean, whose <span class="trait">resistance</span> requires only the <italic> rhg1-b </italic> allele. Multiple copy number of PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic> , <italic> <span class="gene">GmAAT</span> </italic> , and <italic> <span class="gene">GmWI12</span> </italic> in one genomic segment simultaneously contribute to <italic> rhg1-b </italic> <span class="trait">resistance.</span> Using an integrated set of genetic and genomic approaches, we demonstrate that the <italic> rhg1-a </italic> Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is sufficient for <span class="trait">resistance</span> to SCN in combination with <italic> <span class="gene">Rhg4</span> </italic> . The two SNAPs (soluble NSF attachment proteins) differ by only five amino acids. Our findings suggest that Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic> . As such, this is an example of a pathogen <span class="trait">resistance</span> gene that has evolved to underlie two types of <span class="trait">resistance,</span> yet ensure the same function within a single plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						Peking-type and PI 88788-type soybeans are both used as sources of <span class="trait">resistance</span> against soybean cyst nematode. Here the authors show that in contrast to PI 88788-type <span class="trait">resistance,</span> where <italic> <span class="gene">GmSNAP18</span> </italic> acts in combination with two neighbouring genes, in Peking-type <span class="trait">resistance</span> <italic> <span class="gene">GmSNAP18</span> </italic> acts with <italic> <span class="gene">GmSHMT08</span> </italic> to confer <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The first <italic> Rhg </italic> genes (for <span class="trait">resistance</span> to <italic> H. <span class="trait">glycines</span> </italic> ) were identified in the early 1960s. Numerous reports are available on the identification and mapping of quantitative trait loci (QTL) in soybeans with underlying <span class="trait">resistance</span> to SCN from a variety of different germplasm sources. QTL on chromosomes 18 ( <italic> rhg1 </italic> ) and 8 ( <italic> <span class="gene">Rhg4</span> </italic> ) are the two major <span class="trait">resistance</span> QTL that have been consistently mapped and reported in a variety of soybean germplasm. In some accessions, such as plant introduction (PI) 88788, <italic> rhg1 </italic> is sufficient to provide <span class="trait">resistance</span> against the nematode with the gene itself displaying incomplete dominance <xref ref-type="bibr" rid="b2"> 2 </xref> . In other cases, such as the soybean cultivar ( <italic> cv </italic> .) Forrest, <span class="trait">resistance</span> to SCN requires both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> (ref. <xref ref-type="bibr" rid="b3"> 3 </xref> ). Brucker <italic> et al </italic> . classified the resistant <italic> rhg1 </italic> into two types: <italic> rhg1-a </italic> in Peking-type soybeans and <italic> rhg1-b </italic> in PI 88788-type soybeans <xref ref-type="bibr" rid="b4"> 4 </xref> . Here we define the susceptible <italic> rhg1 </italic> type in Essex and Williams 82 as <italic> rhg1-s </italic> ; otherwise, we use <italic> rhg1 </italic> to refer to the locus in general.
					</p>
				
					<p>&nbsp;&nbsp;
						Resistant soybean lines carrying the <italic> Rhg </italic> <span class="trait">resistance</span> genes display an incompatible interaction with the nematode. Although infective juvenile nematodes penetrate the plant root, the feeding cells eventually degenerate causing the nematodes to die before they reach the adult stage. <italic> H. <span class="trait">glycines</span> </italic> exhibits genetic variability and nematodes carrying the as yet undefined <italic> ror </italic> (reproduction on a resistant host) alleles are able to survive on <span class="trait">resistance</span> cultivars thus causing population shifts in the field associated with monoculture or resistant soybean varieties <xref ref-type="bibr" rid="b5"> 5 </xref> . Recently, progress has been made in the identification of the major soybean genes underlying SCN <span class="trait">resistance</span> at the <italic> rhg1-b </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, and in the development of specific DNA markers for SCN <span class="trait">resistance</span> selection. Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b6"> 6 </xref> identified and functionally validated a serine hydroxymethyltransferase on chromosome 8 ( <italic> <span class="gene">GmSHMT08</span> </italic> ) as the <italic> <span class="gene">Rhg4</span> </italic> gene by high-density genetic mapping, targeting-induced local lesions in genomes (TILLING), gene silencing (virus induced gene silencing (VIGS) and RNA interference) and genetic complementation. Serine hydroxymethyltransferases (SHMTs) are important in one-carbon folate metabolism, but how this particular soybean <span class="gene">SHMT</span> functions in <span class="trait">resistance</span> to the nematode remains unknown. Cook <italic> et al </italic> . reported that the <span class="trait">resistance</span> of soybean <italic> rhg1-b </italic> to SCN is simultaneously mediated by the high copy number of three genes ( <italic> <span class="gene">Glyma18g02580</span> </italic> , <italic> <span class="gene">Glyma18g02590</span> </italic> and <italic> <span class="gene">Glyma18g02610</span> </italic> ) within a 31&#x02009;kb PI 88788 <italic> rhg1-b </italic> segment <xref ref-type="bibr" rid="b7"> 7 </xref> . This segment also includes partial gene sequences for <italic> <span class="gene">Glyma18g02570</span> </italic> and <italic> <span class="gene">Glyma18g02610</span> </italic> . Direct repeats (2&#x02013;10) are present in SCN-resistant soybean lines, whereas only one copy is present in SCN-susceptible soybean lines <xref ref-type="bibr" rid="b7"> 7 </xref> <xref ref-type="bibr" rid="b8"> 8 </xref> <xref ref-type="bibr" rid="b9"> 9 </xref> . Using the <italic> <span class="gene">Glyma18g02590</span> </italic> gene sequences, two specific KASP (kompetitive allele-specific PCR) single-nucleotide polymorphism (SNP) markers were developed to select the <italic> rhg1 </italic> <span class="trait">resistance</span> alleles and the assay can be used to differentiate Peking-type and PI 88788-type <span class="trait">resistance</span> <xref ref-type="bibr" rid="b10"> 10 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Using a positional cloning approach, we identified <italic> rhg1-a </italic> candidate genes within a contrasting 370&#x02009;kb chromosomal interval between the resistant line Forrest and the susceptible line Essex. We then applied &#x02018;region-specific extraction sequencing' (RSE-Seq), a capture technology developed to enrich a targeted chromosomal segment for genome sequencing <xref ref-type="bibr" rid="b11"> 11 </xref> <xref ref-type="bibr" rid="b12"> 12 </xref> <xref ref-type="bibr" rid="b13"> 13 </xref> , to identify SCN <span class="trait">resistance</span> genes within the identified 300&#x02009;kb chromosomal segment carrying the <italic> rhg1 </italic> locus. RSE-Seq has been applied successfully to study yeast, human and zebrafish <xref ref-type="bibr" rid="b11"> 11 </xref> <xref ref-type="bibr" rid="b12"> 12 </xref> <xref ref-type="bibr" rid="b13"> 13 </xref> <xref ref-type="bibr" rid="b14"> 14 </xref> <xref ref-type="bibr" rid="b15"> 15 </xref> . The SNPs and insertions and deletions (InDels) of four soybean lines (Essex, Forrest, Peking and PI 88788) were analysed using the Williams 82 genomic sequence as a reference <xref ref-type="bibr" rid="b16"> 16 </xref> . The analysis of the relevant SNPs and InDels identified the <italic> <span class="gene">GmSNAP18</span> </italic> as the strongest candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus. Genetic complementation analyses of the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> confirmed its major role in <span class="trait">resistance</span> to SCN. The complementation analyses also showed the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is specifically required for Peking-type <span class="trait">resistance,</span> as the resistant <italic> <span class="gene">GmSNAP18</span> </italic> allele of PI 88788 or the susceptible <italic> rhg1-s </italic> could not restore <span class="trait">resistance</span> in the Forrest (Peking-type) <italic> <span class="gene">Rhg4</span> </italic> background.
					</p>
				
					<p>&nbsp;&nbsp;
						To perform positional cloning by high-density genetic mapping of the <italic> rhg1-a </italic> gene conferring <span class="trait">resistance</span> to SCN, three F <sub> 5:6 </sub> recombinant inbred line (RIL) populations segregating for <span class="trait">resistance</span> to SCN were developed from crosses of the SCN-resistant <italic> cv. </italic> Forrest (F) with the SCN-susceptible soybean <italic> cvs </italic> . Williams 82 (W) and Essex (E). As Forrest-type <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> , identification of recombinants was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1-a </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> ). We identified a total of 222 recombinant lines with chromosomal breakpoints around the <italic> rhg1-a </italic> locus, of which three recombinants (ExF4361, ExF3126 and WxF6034) were crucial in defining the interval carrying the <italic> rhg1-a </italic> or <italic> rhg1-s </italic> gene. All three lines carried the Forrest allele at the <italic> <span class="gene">Rhg4</span> </italic> locus; ExF4361 and ExF3126 were resistant, whereas WxF6034 was susceptible to SCN ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ). The ExF4361 recombinant suggested that the <italic> rhg1-a </italic> gene(s) was (were) located on the left side of marker 37591. The WxF6034 recombinant indicated that the <italic> rhg1-a </italic> gene(s) was (were) located to the left of marker 600, excluding the DNA marker 600. The ExF3126 recombination event showed that the <italic> rhg1-a </italic> gene(s) was (were) located in the interval between DNA maker 560 and the DNA marker SIUC-SAT185, excluding DNA marker 560. Taken together, the recombinant analyses suggested that the interval (&#x0223c;14.3&#x02009;kb) carrying DNA markers 570, 580 and 590 was underlying <span class="trait">resistance</span> to SCN at the <italic> rhg1-a </italic> locus in Forrest. Three genes were identified within the 14.3&#x02009;kb interval; one codes for an armadillo/&#x003b2;-catenin-like repeat ( <italic> <span class="gene">Glyma18g02570</span> </italic> ) at the interval carrying marker 570; the second is an amino acid transporter ( <italic> <span class="gene">AAT</span> </italic> , <italic> <span class="gene">Glyma18g02580</span> </italic> ) at the interval carrying marker 580; the third is a soluble N-ethylmelaimide sensitive factor (NSF) attachment protein ( <italic> <span class="gene">GmSNAP18</span> </italic> , <italic> <span class="gene">Glyma18g02590</span> </italic> ) at the interval carrying marker 590. The complementary DNAs for each of the three identified genes were cloned and sequenced. Amino acid alignments of the predicted armadillo/&#x003b2;-catenin-like repeat and amino acid transporter protein sequences of Forrest and Essex revealed identical amino acid sequences between both the resistant and susceptible lines ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 1 </xref> ). A <italic> <span class="gene">GmSNAP18</span> </italic> genomic DNA sequence comparison of Forrest and PI 88788 with Essex identified six SNPs (C2447A, C2464G, G4203C, G4206C, G4206T and C4215A) and four InDels (&#x02212;4211G, &#x02212;4212G, &#x02212;4213T and &#x02212;4213G) within the exons ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ), resulting in nine amino acid changes ( <xref ref-type="fig" rid="f1"> Fig. 1c </xref> ). Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was deemed as the strongest candidate gene for conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1-a </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Compared with susceptible soybeans, we identified 408 SNPs (FP88-S127, FP-S232 and 88-S49), 71 insertions (FP88-I21, FP-I47 and 88-I3) and 59 deletions (FP88-D27, FP-D28 and 88-D4) in the resistant Forrest, Peking and/or PI 88788 ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Data 1 </xref> ). Among them, 175 SNPs and InDels, including 127 SNPs (FP88-S127), 21 insertions (PF88-I21) and 27 deletions (PF88-D27), were considered to be associated with the alleles underlying <span class="trait">resistance</span> to SCN. Furthermore, 26 SNPs and InDels resulted in 25 amino acid changes ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 2 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Data 1 </xref> ). However, when the genomic DNA sequences of the <italic> <span class="gene">GmSNAP18</span> </italic> from Forrest and PI 88788 were compared with those of the susceptible line Essex, only six SNPs (C2447A, C2464G, G4203C, G4206C, G4206T and C4215A) and four InDels (&#x02212;4211G, &#x02212;4212G, &#x02212;4213T and &#x02212;4213G) were mapped to the exons of gene <italic> <span class="gene">Glyma18g02590</span> </italic> ( <italic> <span class="gene">GmSNAP18</span> </italic> ). The identified polymorphisms resulted in nine amino acid changes ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 2 </xref> ). Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was identified to be the most probable candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus, consistent with the results of map-based cloning ( <xref ref-type="fig" rid="f1"> Fig. 1 </xref> ). The predicted <italic> <span class="gene">GmSNAP18</span> </italic> protein of Forrest and Peking is different from that of PI 88788. This is consistent with previous genetic results (termed as <italic> rhg1-a </italic> in Peking-type soybeans and <italic> rhg1-b </italic> in PI 88788-type soybeans) <xref ref-type="bibr" rid="b4"> 4 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To establish a link between <italic> <span class="gene">GmSNAP18</span> </italic> alleles and soybean <span class="trait">resistance</span> to SCN, we scored 81 soybean lines (including PIs, landraces and elite cultivars, as described previously <xref ref-type="bibr" rid="b6"> 6 </xref> ) representing 95&#x00025; of the sequence diversity of soybeans for their SCN female index (FI) <xref ref-type="bibr" rid="b17"> 17 </xref> and then determined their SNP-based <italic> <span class="gene">GmSNAP18</span> </italic> haplotype after genotyping them at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci ( <xref ref-type="table" rid="t3"> Table 3 </xref> ). The genotyping data were clustered (Fig. 2 and Supplementary Table 1 of Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b6"> 6 </xref> ) and 11 soybean lines were selected to fully sequence their <italic> <span class="gene">GmSNAP18</span> </italic> genes. As a result, 14 polymorphisms (10 SNPs and 4 InDels) at 13 positions were identified within the exons of <italic> <span class="gene">GmSNAP18</span> </italic> ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ). In total, three different <italic> <span class="gene">GmSNAP18</span> </italic> haplotypes were identified. Soybean lines with Haplotype I carry resistant alleles at both <italic> <span class="gene">GmSNAP18</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ), and are resistant to SCN ( <xref ref-type="table" rid="t3"> Table 3 </xref> and Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b6"> 6 </xref> ). This includes soybean lines PI 548402 (Peking), Forrest, PI 90763, PI 437654 and PI 89772, all of which exhibit &#x02018;Peking-type' <span class="trait">resistance</span> and require both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ). Soybean lines with Haplotype III carry the susceptible <italic> <span class="gene">GmSNAP18</span> </italic> , but vary for either the resistant or susceptible allele at the <italic> <span class="gene">Rhg4</span> <span class="gene">GmSHMT08</span> </italic> ( <xref ref-type="table" rid="t3"> Table 3 </xref> and Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b6"> 6 </xref> ). These lines are susceptible to SCN, regardless of the <italic> <span class="gene">Rhg4</span> <span class="gene">GmSHMT08</span> </italic> genotype, and include soybean cultivars Essex, Williams 82 and PI 603428C. Soybean lines with Haplotype II carry the resistant allele at <italic> <span class="gene">GmSNAP18</span> </italic> , but vary for either the resistant or susceptible allele at the <italic> <span class="gene">Rhg4</span> <span class="gene">GmSHMT08</span> </italic> ( <xref ref-type="table" rid="t3"> Table 3 </xref> and Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b6"> 6 </xref> ). These lines are resistant to SCN and include PI 88788, PI 209332 and PI 548316 ( <italic> cv </italic> . Cloud), all of which exhibit &#x02018;PI 88788-type' <span class="trait">resistance.</span> In summary, the <italic> <span class="gene">GmSNAP18</span> </italic> haplotyping analysis is in agreement with previous SCN-resistance QTL reports and confirms the requirement of <italic> rhg1 </italic> for the SCN <span class="trait">resistance</span> in both PI 88788 and Peking.
					</p>
				
					<p>&nbsp;&nbsp;
						We successively analysed the <italic> <span class="gene">GmSNAP18</span> </italic> expression in the ExF RILs together with <italic> <span class="gene">GmSNAP18</span> </italic> and <italic> <span class="gene">GmSHMT08</span> </italic> genotyping and phenotyping ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Data 2 </xref> ). In this work, <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> and <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> , and <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x0002b; </sup> and <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x02212; </sup> represent Forrest and Essex <italic> <span class="gene">GmSNAP18</span> </italic> , and Forrest and Essex <italic> <span class="gene">GmSHMT08</span> </italic> , respectively; <italic> <span class="gene">GmSNAP18-P</span> </italic> <sup> &#x0002b; </sup> represents PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> . The results demonstrate that <italic> <span class="gene">GmSNAP18</span> </italic> transcripts were induced in the ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> (ExF7) and <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x02212; </sup> (ExF12), whereas the ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> (ExF68) did not show significant changes in the expression of <italic> <span class="gene">GmSNAP18</span> </italic> , after SCN infection (3 days). ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> display <span class="trait">resistance</span> to SCN, whereas the ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> or <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x02212; </sup> do not show <span class="trait">resistance</span> to SCN ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Data 2 </xref> ). These results further support a role for <italic> <span class="gene">GmSNAP18</span> </italic> in <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In Forrest, both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles are required for <span class="trait">resistance</span> to SCN. VIGS silencing of Forrest <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-a </italic> ) in resistant soybean RIL ExF67 carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> / <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x0002b; </sup> resulted in a strong cell death phenotype in the shoots, ultimately killing the plant ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 2 </xref> ). Therefore, to test the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> for contribution to <italic> rhg1-a </italic> <span class="trait">resistance,</span> we used the SCN-susceptible RIL ExF50 carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> / <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x0002b; </sup> , which has the resistant Forrest allele at the <italic> <span class="gene">Rhg4</span> </italic> locus and the susceptible Essex allele at the <italic> rhg1 </italic> ( <italic> rhg1-s </italic> ) locus. As <italic> <span class="gene">GmSNAP18</span> </italic> occurs in multiple copies at this locus in the resistant soybean <xref ref-type="bibr" rid="b7"> 7 </xref> <xref ref-type="bibr" rid="b8"> 8 </xref> <xref ref-type="bibr" rid="b9"> 9 </xref> , the construct was made to express the gene under the control of a CaMV 35S promoter <italic> in planta </italic> . Roots of RIL ExF67 transformed with the plasmid vector were used as a positive control for <span class="trait">resistance.</span> The transgenic hairy roots overexpressing Forrest <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-a </italic> ) grew normally, but showed a significant reduction in SCN development compared with the control transgenic hairy roots in infection assays ( <xref ref-type="fig" rid="f5"> Fig. 5a </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 3a and 4 </xref> ). We also tested the amino acid transporter ( <italic> <span class="gene">AAT</span> </italic> , <italic> <span class="gene">Glyma18g02580</span> </italic> , the predicted protein sequences between Forrest and Essex are identical ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 1 </xref> ) and did not observe a significant reduction in SCN development compared with the control transgenic hairy roots in infection assays ( <xref ref-type="fig" rid="f5"> Fig. 5a </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs 3b and 4 </xref> ). The predicted protein sequences of <italic> <span class="gene">Glyma18g02570</span> </italic> of both Forrest and Essex are identical to that of Williams 82 ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://soybase.org"> http://soybase.org </ext-link> ) ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 1 </xref> ). <italic> <span class="gene">Glyma18g02570</span> </italic> did not contribute to SCN <span class="trait">resistance</span> <xref ref-type="bibr" rid="b7"> 7 </xref> and therefore was not tested in this study. These data suggest that the <italic> <span class="gene">GmSNAP18</span> </italic> gene at the <italic> rhg1-a </italic> locus is sufficient to confer SCN <span class="trait">resistance</span> in Forrest. We also expressed the <italic> <span class="gene">GmSNAP18</span> </italic> genes from two other soybean lines: Essex ( <italic> rhg1-s </italic> ) and PI 88788 ( <italic> rhg1-b </italic> ). Expression of the <italic> <span class="gene">GmSNAP18</span> </italic> gene from either Essex or PI 88788 in RIL ExF50 did not result in a significant reduction in SCN development compared with the control ( <xref ref-type="fig" rid="f5"> Fig. 5b </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 4 </xref> ), demonstrating the specificity of Forrest <italic> <span class="gene">GmSNAP18</span> </italic> in conferring Peking-type soybean <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the locations of the different identified haplotypes of <italic> <span class="gene">GmSNAP18</span> </italic> and how they may confer SCN <span class="trait">resistance,</span> the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> was modelled and then each haplotype of the other soybean lines ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ) was mapped. The modelling showed that most of the identified differences were at the carboxy-terminus of the <span class="gene">GmSNAP18.</span> These haplotypes not only varied between PI 88788-type and Peking-type resistant lines, but also between the resistant and susceptible lines ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ). Through structural analyses, variations in the <italic> <span class="gene">GmSNAP18</span> </italic> C-terminal residues 285&#x02013;289 (E285, Y286, E287, V288 and I289) may play a role in establishing protein&#x02013;protein interactions or modulating vesicular exocytosis <xref ref-type="bibr" rid="b18"> 18 </xref> . It has been reported that the C-terminus of soluble N-ethylmaleimide sensitive fusion attachment proteins (SNAPs; last 25 residues) determines its functionality in vesicle trafficking and localization <xref ref-type="bibr" rid="b19"> 19 </xref> . The Q203K haplotype in PI 88788-type soybeans ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ) has an altered charge of the immediate environment from an uncharged glutamine to a positively charged lysine. The T261A variation only occurring in PI 90763 ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ) shows a polarity shift from a polar threonine in all other soybean lines to a non-polar alanine, which probably enables PI 90763 to acquire new <span class="trait">resistance</span> to certain SCN Hg Types. Similarly, the T251S variation in PI 209332 and A39D in PI 548316 may play a role in nematode effector recognition, as this ability differs from its closest soybean haplotypes ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1-a </italic> gene conferring <span class="trait">resistance</span> to SCN using an integrated set of genomic and genetic approaches in combination with genetic complementation analyses. A high-density genetic map of a 370&#x02009;kb chromosomal segment carrying the Forrest <italic> rhg1-a </italic> locus was developed using three RIL populations ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ). Three recombination events (ExF3126, ExF4361 and WxF6034) narrowed the <italic> rhg1-a </italic> gene candidate to a region containing three genes, <italic> <span class="gene">Glyma18g02570</span> </italic> , <italic> <span class="gene">Glyma18g02580</span> </italic> and <italic> <span class="gene">Glyma18g02590</span> </italic> ( <italic> <span class="gene">GmSNAP18</span> </italic> ) ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ). Similar to Peking <xref ref-type="bibr" rid="b7"> 7 </xref> , three tandem copies of the <italic> <span class="gene">Glyma18g02570</span> </italic> , <italic> <span class="gene">Glyma18g02580</span> </italic> and <italic> <span class="gene">Glyma18g02590</span> </italic> are present within the <italic> rhg1 </italic> -a locus in the cultivar Forrest. The recombinant WxF6034 carried the Forrest allele for <italic> <span class="gene">Glyma18g02610</span> </italic> , but was susceptible to SCN, indicating that the <italic> <span class="gene">Glyma18g02610</span> </italic> gene is not contributing to the SCN <span class="trait">resistance</span> in the Forrest background. Previously, the leucine-rich repeat receptor-like kinase ( <italic> <span class="gene">LRR-RLK</span> </italic> ) genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci were claimed as the <span class="trait">resistance</span> genes <xref ref-type="bibr" rid="b20"> 20 </xref> <xref ref-type="bibr" rid="b21"> 21 </xref> <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> <xref ref-type="bibr" rid="b24"> 24 </xref> . However, Melito <italic> et al </italic> . <xref ref-type="bibr" rid="b25"> 25 </xref> and Liu <italic> et al </italic> . <xref ref-type="bibr" rid="b26"> 26 </xref> demonstrated that the <italic> <span class="gene">LRR-RLK</span> </italic> genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci did not confer SCN <span class="trait">resistance.</span> In the present work, the <italic> rhg1-a </italic> genetic map of susceptible RIL WxF6034 clearly showed that WxF6034 carried the Forrest genotype at the TMD1 marker and the flanking region carrying the <italic> <span class="gene">LRR-RLK</span> </italic> gene ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ), excluding the <italic> rhg1-a <span class="gene">LRR-RLK</span> </italic> candidate <span class="trait">resistance</span> gene in support of earlier work.
					</p>
				
					<p>&nbsp;&nbsp;
						To further confirm the reliability of the constructed genetic map, RSE-Seq was used to study the SNPs and InDels among resistant and susceptible genotypes to pinpoint candidate genes for <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus. RSE has been successfully applied to yeast, human and zebrafish <xref ref-type="bibr" rid="b11"> 11 </xref> <xref ref-type="bibr" rid="b12"> 12 </xref> <xref ref-type="bibr" rid="b13"> 13 </xref> <xref ref-type="bibr" rid="b14"> 14 </xref> <xref ref-type="bibr" rid="b15"> 15 </xref> , but has not yet been applied to plants. Here we show that RSE-Seq of a targeted 300&#x02009;kb genomic DNA segment <span class="gene">(Gm18:</span> 1480001..1780000) of contrasting chromosomal regions underlying <span class="trait">resistance</span> was effective in the direct identification of a candidate <italic> rhg1 </italic> SCN <span class="trait">resistance</span> gene ( <xref ref-type="table" rid="t1"> Table 1 </xref> and <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). We postulated that the SNPs and InDels at <italic> rhg1 </italic> most likely to be associated with the alleles underlying <span class="trait">resistance</span> to SCN would be shared by Forrest, Peking, and/or PI 88788, but not present in the genomic DNA of susceptible line Essex at the <italic> rhg1 </italic> locus. Furthermore, those with changes within exons that resulted in amino acid changes would be considered high priority candidates. Compared with the genomic sequence of Williams 82 at that targeted 300&#x02009;kb <italic> rhg1 </italic> genomic DNA segment, the sequence analysis of Forrest, Peking and PI 88788 resulted in the identification of 835, 872, and 1021 SNPs and InDels, respectively. In contrast, Essex showed 886 SNPs and InDels ( <xref ref-type="table" rid="t1"> Table 1 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Data 1 </xref> ), of which only 26 SNPs and InDels were potentially associated with the alleles underlying <span class="trait">resistance</span> to SCN ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 2 </xref> ). Furthermore, of those 26 SNPs and InDels, 10 were specific to Forrest, Peking, and PI 88788, and all were located within the exons of one gene <italic> <span class="gene">Glyma18g02590</span> </italic> ( <italic> <span class="gene">GmSNAP18</span> </italic> ). This resulted in nine amino acid changes to the predicted <italic> <span class="gene">GmSNAP18</span> </italic> protein ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 2 </xref> ). The other SNPs and InDels identified did not result in any amino acid changes. Thus, consistent with the genetic mapping results, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1 </italic> candidate gene probably conferring <span class="trait">resistance</span> to SCN. The results obtained in this study suggest that the RSE method is a powerful tool for the preparation of specific genomic regions for next-generation sequencing (NGS). However, the design of specific primers is critical when working with organisms possessing repetitive genome sequences. For instance, the soybean genome is about 1.115&#x02009;Gb in size <xref ref-type="bibr" rid="b27"> 27 </xref> , but 40&#x02013;60&#x00025; of the soybean genome is repetitive sequence and heterochromatic <xref ref-type="bibr" rid="b28"> 28 </xref> <xref ref-type="bibr" rid="b29"> 29 </xref> <xref ref-type="bibr" rid="b30"> 30 </xref> . To enrich the specific region <span class="gene">(Gm18:</span> 1480001..1780000), one primer was designed approximately every 5&#x02009;kb for a total of 60 primers spanning the targeted genomic DNA region. Specific primers ensured that other homeologous genomic DNA segments (that is, chromosome 11) were not enriched to avoid biased sequence reads which could have complicated the sequence data analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic complementation analyses of <italic> <span class="gene">GmSNAP18</span> </italic> clearly showed that the transgenic ExF50 ( <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> <italic> &#x0002b; </italic> </sup> ) gained <span class="trait">resistance</span> to SCN after being complemented with the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> , but not when complemented with Forrest <italic> <span class="gene">AAT</span> </italic> ( <xref ref-type="fig" rid="f5"> Fig. 5a </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 4 </xref> ), indicating that the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is the gene at the <italic> rhg1-a </italic> locus that confers <span class="trait">resistance</span> to SCN. In addition, the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-b </italic> ) or Essex <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-s </italic> ) could not restore <span class="trait">resistance</span> to ExF50 ( <xref ref-type="fig" rid="f5"> Fig. 5b </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 4 </xref> ), showing that the only form of <italic> <span class="gene">GmSNAP18</span> </italic> that could complement the ExF50 RIL is the Forrest-type <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-a </italic> ). These data suggest that the resistant form of the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> is unable to communicate with the Forrest <italic> <span class="gene">GmSHMT08</span> </italic> ( <italic> <span class="gene">Rhg4</span> </italic> ), to confer <span class="trait">resistance</span> to SCN. Our genetic mapping results did not include <italic> <span class="gene">Glyma18g02610</span> </italic> among the <italic> rhg1-a </italic> candidate genes ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ). Thus, the complementation results indicate that the SCN <span class="trait">resistance</span> at the <italic> rhg1-a </italic> locus in Forrest is triggered by <italic> <span class="gene">GmSNAP18</span> </italic> alone, contrasting with the <italic> rhg1-b </italic> in PI 88788, whose <span class="trait">resistance</span> is due to the high copy number and concerted activity of three genes <xref ref-type="bibr" rid="b7"> 7 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Forrest, Peking and PI 88788 require the presence of either the resistant <italic> rhg1-a </italic> or <italic> rhg1-b </italic> allele to exhibit <span class="trait">resistance</span> to SCN <xref ref-type="bibr" rid="b2"> 2 </xref> <xref ref-type="bibr" rid="b3"> 3 </xref> <xref ref-type="bibr" rid="b6"> 6 </xref> <xref ref-type="bibr" rid="b7"> 7 </xref> . In addition, the <span class="trait">resistance</span> of Forrest and Peking to SCN requires the <italic> <span class="gene">Rhg4</span> </italic> gene <xref ref-type="bibr" rid="b6"> 6 </xref> . The <span class="trait">resistance</span> reaction of Peking-type soybeans to nematodes is quick and forceful leading to a rapid degeneration of the syncytium, whereas PI 88788-type soybeans react to nematodes comparatively slower <xref ref-type="bibr" rid="b31"> 31 </xref> . SCN populations that have adapted to reproduce on soybeans with either Peking-type or PI 88788-type <span class="trait">resistance</span> have been reported <xref ref-type="bibr" rid="b32"> 32 </xref> . According to the different haplotypes of <italic> <span class="gene">GmSNAP18</span> </italic> identified from a collection of 81 soybean lines, Peking, Forrest, PI 90763, PI 437654 and PI 89772 were classified as Peking-type soybeans; PI 88788, PI 209332 and PI 548316 were classified as PI 88788-type soybeans; and Essex, Williams 82 and PI 603428C were classified as SCN-susceptible-type soybeans ( <xref ref-type="table" rid="t3"> Table 3 </xref> and <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ). All are in agreement with the previous genetic results, showing that the SCN <span class="trait">resistance</span> of Peking-type soybeans such as Forrest requires both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ) <xref ref-type="bibr" rid="b2"> 2 </xref> <xref ref-type="bibr" rid="b3"> 3 </xref> <xref ref-type="bibr" rid="b6"> 6 </xref> . The genomic and cDNA sequences of the <italic> <span class="gene">GmSNAP18</span> </italic> conferring SCN <span class="trait">resistance</span> were different between Peking-type soybeans and PI 88788-type soybeans ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ), consistent with the different <italic> rhg1 </italic> loci identified in both lines ( <italic> rhg1-a </italic> and <italic> rhg1-b </italic> ) <xref ref-type="bibr" rid="b4"> 4 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						SNAPs are highly conserved proteins and belong to the tetratricopeptide repeat containing protein family. Tetratricopeptide repeat proteins are known to be involved in vesicular trafficking, cytokinesis and plasma membrane repair and stability <xref ref-type="bibr" rid="b33"> 33 </xref> <xref ref-type="bibr" rid="b34"> 34 </xref> <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b36"> 36 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> . SNAP proteins have been described as the member of the SNARE complex involved in many pathogen <span class="trait">resistance</span> pathways. It has been reported that SNAP, together with syntaxins SYP121 and SYP132, contributes to the <span class="trait">resistance</span> to fungi and bacteria <xref ref-type="bibr" rid="b38"> 38 </xref> <xref ref-type="bibr" rid="b39"> 39 </xref> . The major amino acid changes of <italic> <span class="gene">GmSNAP18</span> </italic> between susceptible, Peking-type and PI 88788-type soybean lines were mostly mapped in the C-terminus of the protein ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ), which we classified into different haplotypes ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ). Considering that the C-terminus of SNAP proteins have been found to control vesicular trafficking <xref ref-type="bibr" rid="b19"> 19 </xref> , the haplotype differences among susceptible, Peking-type and PI 88788-type soybeans may alter the destination of a <span class="gene">GmSNAP18-guided</span> vesicle. In addition to the two major resistant haplotypes (Peking and PI 88788), there are also the PI 90763 (T261A), PI 548316 (A39D), and PI 209332 (T251S) haplotypes. SNAP proteins have also been found to have their activity altered by phosphorylation. As most kinases act on serine or threonine, the T261A and T251S may have altered phosphorylation capabilities. Various &#x003b1;-SNAP haplotypes may also play a role in nematode effector recognition <xref ref-type="bibr" rid="b40"> 40 </xref> or in establishing protein&#x02013;protein interactions <xref ref-type="bibr" rid="b41"> 41 </xref> <xref ref-type="bibr" rid="b42"> 42 </xref> . Thus, the differences in <italic> <span class="gene">GmSNAP18</span> </italic> among soybean genotypes could determine the type of interactions between the nematode and the soybean host.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, our results support a model wherein <italic> <span class="gene">GmSNAP18</span> </italic> is a major factor mediating the different types of soybean <span class="trait">resistance</span> to SCN. Our findings suggest that the Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic> . This is an example of a pathogen <span class="trait">resistance</span> gene that has evolved to underly two types of <span class="trait">resistance,</span> yet ensure the same function within a single plant species. Now that the major genes for <span class="trait">resistance</span> to SCN have been discovered, the molecular and biochemical mechanisms of soybean <span class="trait">resistance</span> to SCN can be explored in detail.
					</p>
				
					<p>&nbsp;&nbsp;
						As Forrest <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes <xref ref-type="bibr" rid="b3"> 3 </xref> , genotyping was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1 </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> ). The simple sequence repeat (SSR) markers Sat_210, Satt309 and SIUC-SAT143 were used to identify chromosomal breakpoints at the <italic> rhg1-a </italic> locus. PCR amplifications were performed using DNA from individuals from each of the three genetic populations. Cycling parameters were as follows: 35 cycles of 94&#x02009;&#x000b0;C for 30&#x02009;s, 50&#x02009;&#x000b0;C for 30&#x02009;s and 72&#x02009;&#x000b0;C for 30&#x02009;s with 7&#x02009;min of extension at 72&#x02009;&#x000b0;C. The PCR products were separated on 3&#x02013;4&#x00025; metaphor agarose gels. The identified recombinants were subjected to a second screening by using the EcoTILLING marker <span class="gene">SHMT</span> <xref ref-type="bibr" rid="b6"> 6 </xref> and the simple sequence repeat (SSR) marker Sat_162 to identify the <italic> <span class="gene">Rhg4</span> </italic> genotype of each recombinant.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> How to cite this article: </bold> Liu, S. <italic> et al </italic> . The soybean <italic> <span class="gene">GmSNAP18</span> </italic> gene underlies two types of <span class="trait">resistance</span> to soybean cyst nematode. <italic> Nat. Commun. </italic> <bold> 8 </bold> , 14822 doi: 10.1038/ncomms14822 (2017).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A Glycine max homolog of NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1 (NDR1) alters defense gene expression while functioning during a <span class="trait">resistance</span> response to different root pathogens in different genetic backgrounds.</h2>
			<h3>PMID:28273511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：McNeece BT(1), Pant SR(2), Sharma K(1), Niruala P(1), Lawrence GW(3), Klink VP(4).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A Glycine max homolog of the Arabidopsis thaliana NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1 (NDR1) coiled-coil nucleotide binding leucine rich repeat (CC-NB-LRR) defense signaling gene <span class="gene">(Gm-NDR1-1)</span> is expressed in root cells undergoing a defense response to the root pathogenic nematode, <span class="trait">Heterodera</span> <span class="trait">glycines.</span> <span class="gene">Gm-NDR1-1</span> overexpression in the H. glycines-susceptible genotype G. max[Williams 82/PI 518671] impairs parasitism. In contrast, <span class="gene">Gm-NDR1-1</span> RNA interference (RNAi) in the H. glycines-resistant genotype G. max[Peking/PI 548402] facilitates parasitism. The broad effectiveness of <span class="gene">Gm-NDR1-1</span> in impairing parasitism has then been examined by engineering its heterologous expression in Gossypium hirsutum which is susceptible to the root pathogenic nematode Meloidogyne incognita. The heterologous expression of <span class="gene">Gm-NDR1-1</span> in G. hirsutum effectively impairs M. incognita parasitism, reducing gall, egg mass, egg and juvenile numbers. In contrast to our prior experiments examining the effectiveness of the heterologous expression of a G. max homolog of the A. thaliana salicyclic acid signaling (SA) gene NONEXPRESSOR OF PR1 <span class="gene">(Gm-NPR1-2),</span> no cumulative negative effect on M. incognita parasitism has been observed in G. hirsutum expressing <span class="gene">Gm-NDR1-1.</span> The results indicate a common genetic basis exists for plant <span class="trait">resistance</span> to parasitic nematodes that involves <span class="gene">Gm-NDR1.</span> However, the <span class="gene">Gm-NDR1-1</span> functions in ways that are measurably dissimilar to <span class="gene">Gm-NPR1-2.</span> Notably, <span class="gene">Gm-NDR1-1</span> overexpression leads to increased relative transcript levels of its homologs of A. thaliana genes functioning in SA signaling, including NPR1-2, TGA2-1 and LESION SIMULATING DISEASE1 (LSD1-2) that is lost in <span class="gene">Gm-NDR1-1</span> RNAi lines. Similar observations have been made regarding the expression of other defense genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Disease <span class="trait">resistance</span> through impairment of α-SNAP-NSF interaction and vesicular trafficking by soybean Rhg1.</h2>
			<h3>PMID:27821740</h3>
				

				
			<p>
				作者：Bayless AM(1), Smith JM(1), Song J(1), McMinn PH(1), Teillet A(1), August BK(2), Bent AF(3).
				<br/> 
				年份：2016 Nov 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						α-SNAP [soluble NSF (N-ethylmaleimide-sensitive factor) attachment protein] and NSF proteins are conserved across eukaryotes and sustain cellular vesicle trafficking by mediating disassembly and reuse of SNARE protein complexes, which facilitate fusion of vesicles to target membranes. However, certain haplotypes of the <span class="gene">Rhg1</span> <span class="trait">(resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> 1) locus of soybean possess multiple repeat copies of an α-SNAP gene <span class="gene">(Glyma.18G022500)</span> that encodes atypical amino acids at a highly conserved functional site. These <span class="gene">Rhg1</span> loci mediate <span class="trait">resistance</span> to soybean cyst nematode (SCN; H. <span class="trait">glycines),</span> the most economically damaging pathogen of soybeans worldwide. <span class="gene">Rhg1</span> is widely used in agriculture, but the mechanisms of <span class="gene">Rhg1</span> disease <span class="trait">resistance</span> have remained unclear. In the present study, we found that the resistance-type <span class="gene">Rhg1</span> α-SNAP is defective in interaction with NSF. Elevated in planta expression of resistance-type <span class="gene">Rhg1</span> α-SNAPs depleted the abundance of SNARE-recycling 20S complexes, disrupted vesicle trafficking, induced elevated abundance of NSF, and caused cytotoxicity. Soybean, due to ancient genome duplication events, carries other loci that encode canonical (wild-type) α-SNAPs. Expression of these α-SNAPs counteracted the cytotoxicity of resistance-type <span class="gene">Rhg1</span> α-SNAPs. For successful growth and reproduction, SCN dramatically reprograms a set of plant root cells and must sustain this sedentary feeding site for 2-4 weeks. Immunoblots and electron microscopy immunolocalization revealed that resistance-type α-SNAPs specifically hyperaccumulate relative to wild-type α-SNAPs at the nematode feeding site, promoting the demise of this biotrophic interface. The paradigm of disease <span class="trait">resistance</span> through a dysfunctional variant of an essential gene may be applicable to other plant-pathogen interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of soybean genotype <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> and Meloidogyne species via <span class="trait">resistance</span> gene analogs.</h2>
			<h3>PMID:27706689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vieira PM(1), Arêdes FA(2), Ferreira A(3), Ferreira MF(4).
				<br/> 
				年份：2016 Sep 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nematodes are important pests of soybean throughout the world and cause high <span class="trait">yield</span> losses. As a control strategy, the identification of <span class="trait">resistance</span> genes is an important aim of breeding studies. Plants possess <span class="trait">resistance</span> genes (R), which are responsible for the recognition of pathogens and activation of the defense system. R genes and <span class="trait">resistance</span> gene analogs (RGAs) possess conserved domains, from which nucleotide-binding site is the most common. Using degenerate primers originating from these domains, it is possible to identify and isolate sequences of R and RGA genes. In this study, soybean genotypes resistant to the nematodes <span class="trait">Heterodera</span> <span class="trait">glycines,</span> Meloidogyne incognita, M. javanica, and M. enterolobii were compared by the use of RGAs and simple sequence repeat (SSR) markers. Forty-six soybean genotypes were studied, including plant introductions (PIs), commercial crops, and source of <span class="trait">resistance</span> genotypes. Thirteen combinations of RGA primers and different SSRs linked to QTLs were used to confirm <span class="trait">resistance</span> to soybean cyst nematodes (SCN). Fragments associated with <span class="trait">resistance</span> to the studied nematodes were amplified in the source of <span class="trait">resistance</span> and PI genotypes. RGA markers were efficient at distinguishing groups of genotypes that were resistant and susceptible to Meloidogyne <span class="gene">spp</span> and SCN. Combinations of specific primers were identified through their ability to amplify nucleotide sequences from possible <span class="trait">resistance</span> candidate genes. SSR markers contributed to the analysis of SCN race specificity, showing that the QTLs identified by these markers are distinct from those identified by RGA markers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impact of <span class="gene">Rhg1</span> copy number, type, and interaction with <span class="gene">Rhg4</span> on <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> in soybean.</h2>
			<h3>PMID:27581541  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yu N(1), Lee TG(1)(2), Rosa DP(3), Hudson M(1), Diers BW(4).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Evaluations of soybean populations showed that both <span class="gene">Rhg1</span> copy number and type were important in determining soybean cyst nematode <span class="trait">resistance</span> with higher copy number within <span class="gene">Rhg1</span> type conferring greater <span class="trait">resistance.</span> <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> are important loci conferring <span class="trait">resistance</span> to soybean cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines).</span> Alleles at <span class="gene">Rhg1</span> have been shown to vary for copy number and type and the importance of this variation in conferring <span class="trait">resistance</span> is not well defined. The repeat number ranges from one to 10 and there are three variant repeat sequence types [plant introduction (PI) 88788-'Fayette' type (F), 'Peking' type (P) and Williams 82 type (W)] across diverse soybean germplasm. We developed populations segregating for <span class="gene">Rhg1</span> copy number and type and <span class="gene">Rhg4</span> allele type to investigate the effect of these factors and their interaction on SCN <span class="trait">resistance.</span> F2 plants from each cross were evaluated for the segregation of <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> alleles and for SCN reproduction after infesting plants with HG type 2.5.7 and HG type 7 populations. Within repeat types, an increase in repeat number was associated with greater <span class="trait">resistance.</span> The P type <span class="gene">Rhg1</span> showed an advantage over F + W type for SCN population HG type 2.5.7 but this was not observed for SCN HG type 7. While plants with P type <span class="gene">Rhg1</span> required <span class="gene">Rhg4</span> to achieve full <span class="trait">resistance,</span> <span class="gene">Rhg4</span> did not increase <span class="trait">resistance</span> in the background of F + W type <span class="gene">Rhg1</span> repeat. This study demonstrates the importance of both <span class="gene">Rhg1</span> copy number and type in determining <span class="trait">resistance</span> and can assist soybean breeders in determining what alleles would best fit their breeding goals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soybean overexpressing <span class="gene">GmSAMT1</span> exhibits <span class="trait">resistance</span> to multiple-HG types of soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:27064027  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M(1), Zhao N(2), Hatcher CN(1), Wuddineh WA(1), Rudis M(1), Tschaplinski TJ(2), Pantalone VR(1), Arelli PR(3), Hewezi T(1), Chen F(1), Stewart CN Jr(4).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) salicylic acid methyl transferase <span class="gene">(GmSAMT1)</span> catalyses the conversion of salicylic acid to methyl salicylate. Prior results showed that when <span class="gene">GmSAMT1</span> was overexpressed in transgenic soybean hairy roots, <span class="trait">resistance</span> is conferred against soybean cyst nematode (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe. In this study, we produced transgenic soybean overexpressing <span class="gene">GmSAMT1</span> and characterized their response to various SCN races. Transgenic plants conferred a significant reduction in the development of SCN HG type 1.2.5.7 (race 2), HG type 0 (race 3) and HG type 2.5.7 (race 5). Among transgenic lines, <span class="gene">GmSAMT1</span> expression in roots was positively associated with SCN <span class="trait">resistance.</span> In some transgenic lines, there was a significant decrease in salicylic acid titer relative to control plants. No significant seed <span class="trait">yield</span> differences were observed between transgenics and control soybean plants grown in one greenhouse with 22 °C day/night temperature, whereas transgenic soybean had higher <span class="trait">yield</span> than controls grown a warmer greenhouse (27 °C day/23 °C night) temperature. In a 1-year field experiment in Knoxville, TN, there was no significant difference in seed <span class="trait">yield</span> between the transgenic and nontransgenic soybean under conditions with negligible SCN infection. We hypothesize that <span class="gene">GmSAMT1</span> expression affects salicylic acid biosynthesis, which, in turn, attenuates SCN development, without negative consequences to soybean <span class="trait">yield</span> or other morphological traits. Thus, we conclude that <span class="gene">GmSAMT1</span> overexpression confers broad <span class="trait">resistance</span> to multiple SCN races, which would be potentially applicable to commercial production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A SNARE-Like Protein and Biotin Are Implicated in Soybean Cyst Nematode Virulence</h2>
			<h3>PMID:26714307</h3>
				

				
					<h3>PMCID:4699853</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bekal </surname>  <given-names> Sadia </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Domier </surname>  <given-names> Leslie L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gonfa </surname>  <given-names> Biruk </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lakhssassi </surname>  <given-names> Naoufal </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lambert </surname>  <given-names> Kris N. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytoparasitic nematodes that are able to infect and reproduce on plants that are considered resistant are referred to as virulent. The mechanism(s) that virulent nematodes employ to evade or suppress host plant defenses are not well understood. Here we report the use of a genetic strategy (allelic imbalance analysis) to associate single nucleotide polymorphisms (SNPs) with nematode virulence genes in <span class="trait">Heterodera</span> <span class="trait">glycines,</span> the soybean cyst nematode (SCN). To accomplish this analysis, a custom SCN SNP array was developed and used to genotype SCN F3-derived populations grown on resistant and susceptible soybean plants. Three SNPs reproducibly showed allele imbalances between nematodes grown on resistant and susceptible plants. Two candidate SCN virulence genes that were tightly linked to the SNPs were identified. One SCN gene encoded biotin synthase (HgBioB), and the other encoded a bacterial-like protein containing a putative SNARE domain (HgSLP-1). The two genes mapped to two different linkage groups. HgBioB contained sequence polymorphisms between avirulent and virulent nematodes. However, the gene encoding HgSLP-1 had reduced copy number in virulent nematode populations and appears to produce multiple forms of the protein via intron retention and alternative splicing. We show that HgSLP-1 is an esophageal-gland protein that is secreted by the nematode during plant parasitism. Furthermore, in bacterial co-expression experiments, HgSLP-1 co-purified with the SCN <span class="trait">resistance</span> protein <span class="gene">Rhg1</span> α-SNAP, suggesting that these two proteins physically interact. Collectively our data suggest that multiple SCN genes are involved in SCN virulence, and that HgSLP-1 may function as an avirulence protein and when absent it helps SCN evade host defenses.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genetic studies have indicated that <span class="trait">resistance</span> to SCN is polygenic, and numerous quantitative trait loci (QTLs) for SCN <span class="trait">resistance</span> have been identified. However a single accession Plant Introduction (PI) 88788, and to a lesser extent soybean cultivar (cv) Peking, predominate the commercial seed market [ <xref rid="pone.0145601.ref009" ref-type="bibr"> 9 </xref> ]. Recently, <span class="trait">resistance</span> genes to SCN were map-based cloned at two loci and both were shown to be atypical plant <span class="trait">resistance</span> genes [ <xref rid="pone.0145601.ref010" ref-type="bibr"> 10 </xref> , <xref rid="pone.0145601.ref011" ref-type="bibr"> 11 </xref> ]. The <italic> <span class="gene">Rhg1</span> </italic> locus from PI88788, was analyzed using an RNA interference (RNAi)-based approach which identified three genes that were part of a tandem repeat encoding a soybean &#x003b1;-SNAP protein, a wound inducible protein and a potential amino acid transporter [ <xref rid="pone.0145601.ref010" ref-type="bibr"> 10 </xref> ]. The <italic> <span class="gene">Rhg4</span> </italic> SCN <span class="trait">resistance</span> gene from cv Forrest (Peking-type <span class="trait">resistance)</span> was recently map-based cloned using targeting induced local lesions in genomes (TILLING), in combination with gene complementation and gene silencing [ <xref rid="pone.0145601.ref011" ref-type="bibr"> 11 </xref> ]. <italic> <span class="gene">Rhg4</span> </italic> also encoded a novel type of plant <span class="trait">resistance</span> gene, a serine hydroxymethyltransferase (SHMT), which is an enzyme involved in one carbon folate metabolism. The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg4</span> </italic> also requires <italic> <span class="gene">Rhg1</span> </italic> to function fully, indicating the two seemingly different SCN <span class="trait">resistance</span> mechanisms work together in some unknown, but important way [ <xref rid="pone.0145601.ref011" ref-type="bibr"> 11 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						While proteins expressed from esophageal glands are undoubtedly important for nematode parasitism, it has been also suggested that glutathione peroxidases secreted from the hypodermis could protect cyst nematodes against reactive oxygen species [ <xref rid="pone.0145601.ref023" ref-type="bibr"> 23 </xref> ]. Likewise, a lipid binding protein secreted from the cuticle Gp-FAR-1 has been hypothesized to be an inhibitor of jasmonic acid signaling [ <xref rid="pone.0145601.ref024" ref-type="bibr"> 24 </xref> ]. Although not secreted, biosynthetic enzymes involved in the production of vitamin B metabolites, such as pantothenate (VB5), biotin (VB7), thiamin (VB1) and pyridoxal 5-phosphate (VB6), could play a role in circumventing a starvation-based mechanism of nematode <span class="trait">resistance</span> [ <xref rid="pone.0145601.ref025" ref-type="bibr"> 25 </xref> , <xref rid="pone.0145601.ref026" ref-type="bibr"> 26 </xref> ]. Two avirulence genes tied to specific host plant <span class="trait">resistance</span> genes have been identified in root knot nematodes ( <italic> Meloidogyne <span class="gene">spp</span> </italic> .); map-1protein is secreted from nematode amphids [ <xref rid="pone.0145601.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0145601.ref028" ref-type="bibr"> 28 </xref> ] and Mj-Cg-1 [ <xref rid="pone.0145601.ref029" ref-type="bibr"> 29 </xref> ] which when silenced via RNAi increased the level of root-knot nematode virulence. However, the mechanism by which these genes alter nematode virulence is unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						Much of how plant parasitic nematodes evade or suppress host plant <span class="trait">resistance</span> mechanisms depends on the corresponding plant-resistance genes involved in preventing the nematode from completing its life cycle. Thus, one might expect the unusual nematode <span class="trait">resistance</span> genes found at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci would require SCN to deploy equally unique mechanisms to overcome these atypical types of <span class="trait">resistance.</span> In this paper, we describe the use of whole genome allelic imbalance or bulk segregant analysis to identify two candidate SCN virulence genes.
					</p>
				
					<p>&nbsp;&nbsp;
						A pool of F3 single cyst derived populations was used to inoculate soybean plants containing the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, or susceptible plants harboring only the susceptible <italic> <span class="gene">Rhg1</span> </italic> alleles. The resulting cysts were harvested from all plants and the DNA was extracted. The SCN populations used in the selection experiments contained both virulent and avirulent SCN, thus one would expect the frequency of SCN virulence genes to increase in SCN populations grown on soybean harboring the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, but not on the susceptible plants. Furthermore, genetic recombination in the SCN genome should create a condition in the SCN population where allelic ratios of single-nucleotide polymorphisms (SNPs) in or near SCN virulence genes should be altered, but SNPs physically farther away from or unlinked to virulence genes will not have distorted ratios of SNP alleles in comparison to the susceptible control. This type of allelic imbalance or bulk segregant analysis can be a useful genetic approach if a way of conducting high-throughput SNP analysis is available.
					</p>
				
					<p>&nbsp;&nbsp;
						The identification of the molecular mechanisms that plant parasitic nematodes use to evade or suppress host plant <span class="trait">resistance</span> is of great practical significance, since understanding this process could lead to broader and more durable resistant plants or to rapid diagnostic tests to predict the virulence profile of field nematode populations. A genetic approach to the identification of nematode virulence genes makes few assumptions about the underlying nature of the genes controlling the virulence phenotype. Past genetic studies on SCN virulence indicated one or two genes control the nematodes ability to reproduce on <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> -resistant plants [ <xref rid="pone.0145601.ref033" ref-type="bibr"> 33 </xref> ]. This study showed that SCN was a viable genetic system; however, it was also clear that SCN lacked a genetic infrastructure (a genome sequence and sequence polymorphisms) needed for map-based cloning of the genes controlling the virulence phenotype.
					</p>
				
					<p>&nbsp;&nbsp;
						The region of the SCN genome that contains SNP 212 encodes <italic> HgSLP-1 </italic> , a gene that appears to have entered the SCN genome via HGT. Interestingly, <italic> HgSLP-1 </italic> appears to vary in copy number in SCN populations. Some of the SCN strains tested for HgSLP-1 copy number variation were the genetically characterized inbred SCN strains OP20, OP25 and OP50 [ <xref rid="pone.0145601.ref033" ref-type="bibr"> 33 </xref> ]. The genetic study of the OP SCN inbred strains established that SCN virulence was controlled by one or two genes, depending on the type of resistant plants and nematode strains used. This landmark study established that virulence on <italic> <span class="gene">Rhg1</span> </italic> -type <span class="trait">resistance</span> was genetically controlled by the dominant <italic> Ror1 </italic> gene (reproduction on resistant) in the virulent strain OP50, but OP50 growth on <italic> <span class="gene">Rhg4</span> </italic> / <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance</span> was controlled by the recessive <italic> ror2 </italic> gene. The virulent inbred SCN strain OP20 was shown to require two virulence genes to reproduce on the <italic> <span class="gene">Rhg1</span> </italic> source of <span class="trait">resistance.</span> This study suggested that SCN utilizes several mechanisms to overcome <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance,</span> but only one gene, <italic> ror2 </italic> , to overcome <italic> Rhg1/Rhg4 </italic> -type <span class="trait">resistance.</span> When these nematodes were tested for the presence or absence of <italic> HgSLP-1 </italic> , the gene was present in OP25 (avirulent SCN), but nearly absent or reduced in copy number in OP20 and OP50 (both virulent on <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg1</span> </italic> / <italic> <span class="gene">Rhg4</span> </italic> resistant plants). It is interesting to note, that even though OP20, OP50 and TN20 are not related to each other (two are from North Carolina and the other from Missouri) the nematode strains have a similar virulence profile in that they grow on most SCN resistant plants. More work is needed to confirm the correlation between lower copy number of <italic> HgSLP-1 </italic> and broad SCN virulence. However, it is interesting that the <italic> ror2 </italic> virulence gene that controls growth on the <italic> <span class="gene">Rhg1</span> </italic> / <italic> <span class="gene">Rhg4</span> </italic> -type <span class="trait">resistance</span> is recessive. The deleted <italic> HgSLP-1 </italic> gene in most of the nematodes in this strain would be expected to act as a recessive gene, thus it may be that <italic> ror2 </italic> is in fact this deletion or lack of <italic> HgSLP- </italic> 1.
					</p>
				
					<p>&nbsp;&nbsp;
						Likewise, the &#x003b1;-SNAP proteins can bind to t-SNAREs via coiled-coil domains both in trans-SNARE complexes and independently [ <xref rid="pone.0145601.ref048" ref-type="bibr"> 48 </xref> ]. When &#x003b1;-SNAP binds to free t-SNAREs in a cell, this interaction prevents membrane fusion events [ <xref rid="pone.0145601.ref049" ref-type="bibr"> 49 </xref> ]. Thus, it seemed possible that if the HgSLP-1 could act as a t-SNARE mimic and bind to an &#x003b1;-SNAP, or a soybean v-SNARE, it might prevent defense-related membrane fusion events in the nematode feeding cell by sequestering these required plant membrane fusion proteins. However in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> plants have up to 10 copies of <italic> <span class="gene">Rhg1</span> </italic> &#x003b1;-SNAP, which may allow them to overcome this putative sequestration. While the data indicates HgSLP-1 is secreted from the nematode and interacts with a soybean &#x003b1;-SNAP, further work will be required to experimentally demonstrate that HgSLP-1 is injected into the feeding cell and interacts <italic> in planta </italic> with host proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of plant membrane fusion proteins in SCN-resistance is supported by the finding that one of the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> genes encodes a &#x003b1;-SNAP protein. <span class="gene">Rhg1</span> &#x003b1;-SNAP proteins may confer SCN <span class="trait">resistance</span> by themselves [ <xref rid="pone.0145601.ref036" ref-type="bibr"> 36 </xref> , <xref rid="pone.0145601.ref050" ref-type="bibr"> 50 </xref> ] or in combination with the other <span class="trait">resistance</span> genes at the <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref rid="pone.0145601.ref010" ref-type="bibr"> 10 </xref> ]. Exactly how HgSLP-1 might mitigate host defenses in soybean lacking the <italic> Rhg-1 </italic> loci is unclear, but over expression of &#x003b1;-SNAP can induce syntaxin 31 in soybean and this t-SNARE protein enhances SCN <span class="trait">resistance</span> [ <xref rid="pone.0145601.ref050" ref-type="bibr"> 50 </xref> ], thus blocking this interaction may promote susceptibility. It has also been demonstrated in tobacco ( <italic> Nicotiana tabacum </italic> ) that by blocking syntaxin SYP132, a t-SNARE protein involved with targeting vesicles containing PR proteins, increased bacterial susceptibility [ <xref rid="pone.0145601.ref051" ref-type="bibr"> 51 </xref> ]. Similarly, <span class="trait">resistance</span> to <italic> Peronospora parasitica </italic> is mediated via VAMP721/722 vesicles in <italic> Arabidopsis </italic> [ <xref rid="pone.0145601.ref052" ref-type="bibr"> 52 </xref> ], suggesting vesicle transport is a critical component for host-plant <span class="trait">resistance</span> to several plant pathogens. Thus, it seems plausible that HgSLP-1 could block the vesicle-mediated transport of anti-nematode proteins or metabolites in a similar way and inactivate this type if basic host defense mechanism.
					</p>
				
					<p>&nbsp;&nbsp;
						However, our data suggests the roles of HgSLP-1 and <span class="gene">Rhg1</span> are not simple. Different isolates of SCN populations had variable copies of the <italic> HgSLP-1 </italic> gene and the RNA-Seq data suggests that the mRNA encoded by the gene is alternatively spliced and forms containing intron sequences may be produced. Similarly, the Rhg-1 &#x003b1;-SNAP proteins show significant polymorphisms between and within SCN resistant and susceptible soybean lines [ <xref rid="pone.0145601.ref010" ref-type="bibr"> 10 </xref> , <xref rid="pone.0145601.ref053" ref-type="bibr"> 53 </xref> ]. The diversity of SCN SNARE-like proteins could reflect the nematodes attempt to adapt to different soybean &#x003b1;-SNAP proteins or other binding partners of HgSLP-1. Indeed, the apparent reduction in copy number of <italic> HgSLP-1 </italic> from TN20 and other nematode populations may reflect a <span class="trait">resistance</span> evasion mechanism used by the nematode as several highly virulent SCN populations appear to have reduced copy numbers of the gene. However, if HgSLP-1 is important for establishing a feeding cell, one might expect that the lack of this protein may impair the growth and development of virulent SCN lacking the gene. While the lack of HgSLP-1 may play a role in evading host plant defense mechanisms, other potential virulence genes, such as the <italic> HgBioB </italic> or the gene near SNP 1533 are probably also required for virulence.
					</p>
				
					<p>&nbsp;&nbsp;
						Overall, HgSLP-1 could play a role in SCN virulence, via its absence, but it could be involved with feeding cell formation if alterations of membrane fusion would be beneficial in this process. It seems plausible that HgSLP-1 may have other binding partners in the plant cell in addition to the <italic> <span class="gene">Rhg1</span> </italic> &#x003b1;-SNAP, but it will be important to localize HgSLP-1 at higher resolution in nematode feeding cells to validate its <italic> in planta </italic> role in nematode parasitism. It will be interesting in the future to test other &#x003b1;-SNAP variants to determine their binding kinetics to <italic> HgSLP-1 </italic> in its various forms. It will also be important to follow up on the involvement in biotin in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> by further analysis of the biotin biosynthetic enzymes in the plant and nematode. Increasing our knowledge of how nematodes evade/suppress host plant <span class="trait">resistance</span> may lead to more durable SCN resistant plants or to improved methods of monitoring virulent nematode populations, which in turn will aid in the management of these damaging pathogens.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genomic-assisted phylogenetic analysis and marker development for next generation soybean cyst nematode <span class="trait">resistance</span> breeding.</h2>
			<h3>PMID:26566850  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kadam S(1), Vuong TD(2), Qiu D(3), Meinhardt CG(4), Song L(5), Deshmukh R(6), Patil G(7), Wan J(8), Valliyodan B(9), Scaboo AM(10), Shannon JG(11), Nguyen HT(12).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN, <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe) is a serious soybean pest. The use of resistant cultivars is an effective approach for preventing <span class="trait">yield</span> loss. In this study, 19,652 publicly available soybean accessions that were previously genotyped with the SoySNP50K iSelect BeadChip were used to evaluate the phylogenetic diversity of SCN <span class="trait">resistance</span> genes <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> in an attempt to identify novel sources of <span class="trait">resistance.</span> The sequence information of soybean lines was utilized to develop KASPar (KBioscience Competitive Allele-Specific PCR) assays from single nucleotide polymorphisms (SNPs) of Rhg1, Rhg4, and other novel quantitative trait loci (QTL). These markers were used to genotype a diverse set of 95 soybean germplasm lines and three recombinant inbred line (RIL) populations. SNP markers from the <span class="gene">Rhg1</span> gene were able to differentiate copy number variation (CNV), such as resistant-high copy (PI 88788-type), low copy (Peking-type), and susceptible-single copy (Williams 82) numbers. Similarly, markers for the <span class="gene">Rhg4</span> gene were able to detect Peking-type <span class="trait">(resistance)</span> genotypes. The phylogenetic information of SCN <span class="trait">resistance</span> loci from a large set of soybean accessions and the gene/QTL specific markers that were developed in this study will accelerate SCN <span class="trait">resistance</span> breeding programs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping</h2>
			<h3>PMID:26268218</h3>
				

				
					<h3>PMCID:4542112</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <address>  <email> wy11544@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> twlneau@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhiwu </given-names>  </name>  <address>  <email> zhiwu.zhang@wsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Key Laboratory of Soybean Biology in Chinese Ministry of Education (key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry), Northeast Agricultural University, Harbin, 150030 China  </aff>  <aff id="Aff2">  <label/> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI) Chinese Academy of Agricultural Sciences, 100081 Beijing, China  </aff>  <aff id="Aff3">  <label/> Bioinformatics Division, Biomarker Technologies Corporation, 101300 Beijing, China  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode (SCN, <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe) is one of the most fatal pests of soybean (Glycine max (L.) Merr.) worldwide and causes huge loss of soybean <span class="trait">yield</span> each year. Multiple sources of <span class="trait">resistance</span> are urgently needed for effective management of SCN via the development of resistant cultivars. The aim of the present study was to investigate the genetic architecture of <span class="trait">resistance</span> to SCN HG Type 0 (race 3) and HG Type 1.2.3.5.7 (race 4) in landraces and released elite soybean cultivars mostly from China. RESULTS: A total of 440 diverse soybean landraces and elite cultivars were screened for <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7. Exactly 131 new sources of SCN <span class="trait">resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF) &gt; 4% that were present in 97% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the <span class="trait">resistance</span> for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease <span class="trait">resistance</span> proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, MYB and WRKY transcription activation families were identified. Such proteins may participate in the resistant reaction to SCN and were frequently found in the tightly linked genomic regions of the peak SNPs. CONCLUSIONS: GWAS extended understanding of the genetic architecture of SCN <span class="trait">resistance</span> in multiple genetic backgrounds. Nineteen association signals were obtained for the <span class="trait">resistance</span> to the two Hg Types of SCN. The multiple beneficial alleles from resistant germplasm sources will be useful for the breeding of cultivars with improved <span class="trait">resistance</span> to SCN. Analysis of genes near association signals may facilitate the recognition of the causal gene(s) underlying SCN resistances.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 440 diverse soybean landraces and elite cultivars were screened for <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7. Exactly 131 new sources of SCN <span class="trait">resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF)&#x02009;&#x0003e;&#x02009;4&#x000a0;% that were present in 97&#x000a0;% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the <span class="trait">resistance</span> for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease <span class="trait">resistance</span> proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, MYB and WRKY transcription activation families were identified. Such proteins may participate in the resistant reaction to SCN and were frequently found in the tightly linked genomic regions of the peak SNPs.
					</p>
				
					<p>&nbsp;&nbsp;
						The knowledge of genetic architecture of SCN <span class="trait">resistance</span> is very important for breeding <span class="trait">resistance</span> varieties [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. So far, genetic architecture analyses mainly depended on traditional quantitative trait locus (QTL) linkage mapping based on bi-parental populations. Once the desired QTL were mapped, molecular markers that were tightly linked to the QTL could be applied in marker-assisted breeding to improve and shorten the process for developing cultivars with <span class="trait">resistance</span> to SCN. Many QTL conferring <span class="trait">resistance</span> to SCN in soybean have been mapped on almost all chromosomes (except of <span class="gene">Gm02;</span> LG D1b) based on SSR markers [ <xref ref-type="bibr" rid="CR12"> 12 </xref> &#x02013; <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. Of them, two major QTL, <italic> <span class="gene">Rhg1</span> </italic> on <span class="gene">Gm18</span> and <italic> <span class="gene">Rhg4</span> </italic> on <span class="gene">Gm08</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], were identified in many different <span class="trait">resistance</span> sources. Genes and their alleles underlying both loci were identified from PI88788 and cultivar Forrest, respectively [ <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Fine mapping, sequencing and gene analysis of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were reported [ <xref ref-type="bibr" rid="CR19"> 19 </xref> &#x02013; <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. The receptor like kinase at <italic> Rhg1-a </italic> caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome and soybean cyst nematode as a transgene by altering signaling responses [ <xref ref-type="bibr" rid="CR23"> 23 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. The study on <italic> Rhg1-b </italic> allele&#x02019;s locus by Cook <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] considered the SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was conferred by three genes, and the copy number variation of these genes caused the phenotypic differences between susceptible and resistant varieties. The SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg4</span> </italic> in Forrest was controlled by a serine hydroxymethyl transferase gene, that acted in cellular one-carbon metabolism [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. The mutation of this gene could disturb the concentration of folate and lead to a nutritional deficiency detrimental to the life cycle of SCN [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Although few SCN <span class="trait">resistance</span> genes have been cloned based on the identified QTL, the large confidence genomic intervals of most QTL impaired the precise identification of causative genes.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic basis of natural variation for soybean <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7 were explored by using an association panel of 440 diverse germplasm genotyped with 36,976 SNP markers. A total of 19 association signals were identified with <italic> P </italic> &#x02009;&#x0003c;&#x02009;1.35&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;6 </sup> for the two traits in a genome wide scan under compressed mixed linear model that controlled the population structure and familial relatedness (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). Among the 19 association signals, eighteen were declared significant under the threshold of <italic> P </italic> &#x02009;&#x0003c;&#x02009;2.70&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;7 </sup> . The 19 peak SNPs explained 6.2&#x02013;15.4&#x000a0;% of the total phenotypic variation. Twelve signals were significantly associated with the <span class="trait">resistance</span> to Hg Type 0 and another seven signals were found to be related to the <span class="trait">resistance</span> to Hg Type 1,2,3,5,7. The known SCN resistant loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were identified by the present study for soybean <span class="trait">resistance</span> to Hg Type 0. The <italic> <span class="gene">Rhg4</span> </italic> locus was also found to contribute <span class="trait">resistance</span> to Hg Type 1.2.3.5.7. The rs33704130 located on <span class="gene">Gm16</span> was significantly associated with the <span class="trait">resistance</span> for both Hg Types. It could be an important locus that offered <span class="trait">resistance</span> to multiple Hg Types. Eight signals overlapped with reported QTL, and another eight were located in the linked region of known QTL (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Three QTL were newly found in the present study. <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> Genome wide association signals for soybean <span class="trait">resistance</span> to HG Type 0 and HG Type 1.2.3.5.7. <bold> a </bold> , <bold> c </bold> Genome wide Manhattan plots of associations from compressed mixed linear model. <italic> x </italic> axis showed the SNPs along each chromosome; <italic> y </italic> axis was the &#x02212;log 10 ( <italic> P </italic> -value) for the association for soybean <span class="trait">resistance</span> to Hg Type 0 and Hg Type 1.2.3.5.7, respectively. <bold> b </bold> , <bold> d </bold> Quantile-Quantile plots for mixed linear models of the <span class="trait">resistance</span> to Hg Type 0 and Hg Type 1.2.3.5.7
					</p>
				
					<p>&nbsp;&nbsp;
						The candidate genes related to the identified loci against SCN Hg Type 0 and Hg Type 1.2.3.5.7 was further evaluated. Gene models that located in 200 Kbp genomic region upstream and downstream of each peak SNP in the reference soybean genome (Wm82.a2.v1, see &#x02018;www. phetozome.net&#x02019;) were considered to be <span class="trait">resistance</span> gene candidates since the mean LD decay distance of soybean genome was around 200 Kbp according to the present study and the reports of Hwang <italic> et al. </italic> (~360 Kbp) and Lam <italic> et al. </italic> (~150Kbp) [ <xref ref-type="bibr" rid="CR37"> 37 </xref> , <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. A total of 395 soybean gene models were found in the flanking regions of each peak SNP. Of these gene models, 64 genes had no functional annotation. Fourteen genes belonged to the domains of unknown function families. Among the other 317 gene models, several genes encoded serine/threonine protein kinases known to be involved in plant disease response. Genes encoding leucine-rich repeat-containing proteins, which might function in plant disease <span class="trait">resistance</span> pathways in response to a variety of external stimuli from pathogens, were also identified. Five other domain types were common including cytochrome P450s, protein kinases, zinc fingers, RING domains, MYB and WRKY families (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Of these gene families, there were five clusters of genes with disease <span class="trait">resistance</span> protein and/or LRR protein domains that were found in both side of rs33704130 on <span class="gene">Gm16</span> that was associated with the <span class="trait">resistance</span> to both Hg Type 0 and Hg Type 1.2.3.5.7. The Hg Type 0 <span class="trait">resistance</span> loci (rs46050545 on <span class="gene">Gm04,</span> rs33843704 on <span class="gene">Gm05,</span> rs33488383 on <span class="gene">Gm11</span> and rs43617172 on <span class="gene">Gm14),</span> as well as Hg Type 1.2.3.5.7 <span class="trait">resistance</span> locus (rs172388849 on <span class="gene">Gm17)</span> were tightly linked with the genes of RING/U-box superfamily protein or RING-H2 finger C1A gene. The distance of each RING gene to their corresponding peak SNP ranged from 8.9 Kbp to 161.7 Kbp with an average distance of 59.1 Kbp. It could be inferred that the RING/U-box superfamily protein might be involved in the disease <span class="trait">resistance</span> responsive to Hg Type 0 and Hg Type 1.2.3.5.7. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Main gene families in the flanking genomic regions of peak SNP
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 12 QTN (nucleotides) associated with QTL underlying <span class="trait">resistance</span> to SCN Hg Type 0, one QTN (rs8050006) was identified on chromosome 8 (Gm 08, LG A2), where a known and major QTL region associated with <italic> <span class="gene">Rhg4</span> </italic> locus had been commonly reported before [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> &#x02013; <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. Concibido <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ] reported that a QTL near the <italic> <span class="gene">Rhg4</span> </italic> locus could explain 15&#x000a0;% of the total phenotypic variation in PI 209332 to SCN Hg Type 0. The <italic> <span class="gene">Rhg4</span> </italic> locus was confirmed as one of the major QTL controlling soybean <span class="trait">resistance</span> against SCN Hg Type 0 [ <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ]. Moreover, Kazi <italic> et al. </italic> and Ferreira <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ] confirmed the effect of <italic> <span class="gene">Rhg4</span> </italic> on <span class="trait">resistance</span> to race 3 and 14, while <italic> <span class="gene">Rhg4</span> </italic> was less associated with <span class="trait">resistance</span> to races 1, 2, 4 and 5. However, this locus was also detected to be related to the <span class="trait">resistance</span> to Hg Type 1.2.3.5.7 (race 4) in our study, indicating that it could underlie SCN <span class="trait">resistance</span> more than one races simultaneously and might facilitate to breed new soybean varieties resistant to SCN race 3, 4 and 14.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, Gm 18 (LG G) was very important for SCN <span class="trait">resistance.</span> The <italic> rhg1 </italic> loci, located on Gm 18, had been conformed to have an impact on the depression of SCN development in several <span class="trait">resistance</span> lines and provided SCN <span class="trait">resistance</span> in many commercial varieties from several <span class="trait">resistance</span> sources [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR55"> 55 </xref> &#x02013; <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. For example, Concibido <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ] reported the recessive gene <italic> rhg 1 </italic> that located on Gm 18 was identified to resistant to SCN race 3 and 14 as a major SCN <span class="trait">resistance</span> source. Similarly, Wu <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] also detected a major QTL near <italic> rhg1 </italic> with a relative large R <sup> 2 </sup> , resistant to SCN race 1, 2, 3, 5 and 14, respectively. Here, the SNP rs1643660, associated with the <span class="trait">resistance</span> to SCN Hg Type 0 (race 3), was detected to fall within the <italic> rhg1 </italic> locus and was located in the 31 Kbp copy number variation (CNV) region of <italic> <span class="gene">Rhg1</span> </italic> locus. The SNP marker rs1643660 was 5.1 Kpb, 4.6 Kbp and 5.1&#x000a0;K&#x000a0;bp distances from the three key genes ( <italic> Glyma.18&#x000a0;g022400, Glyma.18&#x000a0;g022500 </italic> ) in CNV region [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. However, the distance between SSR marker BARCSOYSSR_18_0090 and <italic> rhg 1 </italic> -b was 67 Kbp in a fine mapping study of <italic> rhg 1 </italic> locus that was reported by Kim <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. The distance from marker to causal genes in our study was much shorter than that reported by Kim <italic> et al., </italic> indicating that better mapping result could be available from GWAS using SNP markers. The locus rs1643660 could be more effective for marker-assisted selection of lines resistant to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, rs2085816 on Gm 20 (LG I), rs21804864 on <span class="gene">Gm02</span> (LG D1b) and rs30581306 on <span class="gene">Gm14</span> (LG B2) were the novel loci reported for SCN <span class="trait">resistance.</span> They might have the potential to be used in MAS for enhanced SCN Hg Type 0 and Hg Type 1.2.3.5.7 <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we mapped 19 QTL representing 17 genomic regions associated with soybean <span class="trait">resistance</span> to Hg Type 0 and/or Hg Type 1.2.3.5.7. More than 300 candidate genes were found in the linked region of these QTL, and as many as 25&#x000a0;% might have SCN response. Disease resistance-like proteins were found to be induced by different Hg Types of SCN in other studies too such as by Mazarei <italic> et al. </italic> and Matsye <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR65"> 65 </xref> , <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. That suggested that disease <span class="trait">resistance</span> like proteins was a class of genes with broad-spectrum <span class="trait">resistance</span> to SCN. In addition, expression patterns of genes that encoded cytochrome P450, zinc finger protein, MYB domain protein, and WRKY DNA-binding protein that were mapped here were changed in SCN induced soybean roots [ <xref ref-type="bibr" rid="CR65"> 65 </xref> &#x02013; <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. SAUR-like genes has been suggested to encode short-lived nuclear proteins involved in auxin signaling by interacting with calmodulin [ <xref ref-type="bibr" rid="CR69"> 69 </xref> , <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. A report about auxin-responsive factor (ARF) in <italic> Arabidopsis thaliana </italic> described that double mutants of ARF gene showed a strong auxin phenotype that resulted in the absence of lateral root formation than single mutant expression. In addition, <italic> <span class="gene">Glyma.02g161600</span> </italic> that encoded the RING-H2 finger domain was the nearest gene to rs21804864 which is a novel locus associated with <span class="trait">resistance</span> to SCN. RING-H2 finger as an domain in E3 ubiquitin ligase was considered to participate in the degradation of proteins from the endoplasmic reticulum in Human [ <xref ref-type="bibr" rid="CR71"> 71 </xref> ]. E3 ubiquitin ligases was known to play an important role in mediating plant defence signaling [ <xref ref-type="bibr" rid="CR72"> 72 </xref> ]. Therefore, <italic> <span class="gene">Glyma.02g161600</span> </italic> with RING-H2 finger domain might be a new gene source of SCN <span class="trait">resistance.</span> However, many such genes alter the degree of susceptibility (60&#x02009;&#x0003c;&#x02009;FI&#x02009;&#x0003c;&#x02009;300) more than the degree of <span class="trait">resistance</span> (FI&#x02009;&#x0003c;&#x02009;10).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe). Although this approach successfully mapped a large number of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse set of 553 soybean plant introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as seed coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The syntaxin 31-induced gene, LESION SIMULATING DISEASE1 (LSD1), functions in Glycine max defense to the root parasite <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:25530246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pant SR(1), Krishnavajhala A, McNeece BT, Lawrence GW, Klink VP.
				<br/> 
				年份：2015
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments show the membrane fusion genes α soluble NSF attachment protein (α-SNAP) and syntaxin 31 <span class="gene">(Gm-SYP38)</span> contribute to the ability of Glycine max to defend itself from infection by the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span> Accompanying their expression is the transcriptional activation of the defense genes ENHANCED DISEASE SUSCEPTIBILITY1 (EDS1) and NONEXPRESSOR OF PR1 (NPR1) that function in salicylic acid (SA) signaling. These results implicate the added involvement of the antiapoptotic, environmental response gene LESION SIMULATING DISEASE1 (LSD1) in defense. Roots engineered to overexpress the G. max defense genes <span class="gene">Gm-α-SNAP,</span> SYP38, EDS1, NPR1, BOTRYTIS INDUCED KINASE1 (BIK1) and xyloglucan endotransglycosylase/hydrolase (XTH) in the susceptible genotype G. max[Williams 82/PI 518671] have induced <span class="gene">Gm-LSD1</span> <span class="gene">(Gm-LSD1-2)</span> transcriptional activity. In reciprocal experiments, roots engineered to overexpress <span class="gene">Gm-LSD1-2</span> in the susceptible genotype G. max[Williams 82/PI 518671] have induced levels of SYP38, EDS1, NPR1, BIK1 and XTH, but not α-SNAP prior to infection. In tests examining the role of <span class="gene">Gm-LSD1-2</span> in defense, its overexpression results in ∼52 to 68% reduction in nematode parasitism. In contrast, RNA interference (RNAi) of <span class="gene">Gm-LSD1-2</span> in the resistant genotype G. max[Peking/PI 548402] results in an 3.24-10.42 fold increased ability of H. <span class="trait">glycines</span> to parasitize. The results identify that <span class="gene">Gm-LSD1-2</span> functions in the defense response of G. max to H. <span class="trait">glycines</span> parasitism. It is proposed that LSD1, as an antiapoptotic protein, may establish an environment whereby the protected, living plant cell could secrete materials in the vicinity of the parasitizing nematode to disarm it. After the targeted incapacitation of the nematode the parasitized cell succumbs to its targeted demise as the infected root region is becoming fortified.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and evaluation of quantitative trait loci underlying <span class="trait">resistance</span> to multiple HG types of soybean cyst nematode in soybean PI 437655</h2>
			<h3>PMID:25316311</h3>
				

				
					<h3>PMCID:4282714</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Jiao </surname>  <given-names> Yongqing </given-names>  </name>  <address>  <phone> 573-882-5494 </phone>  <email> yongqing.jiao@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meinhardt </surname>  <given-names> Clinton </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <phone> 573-882-5494 </phone>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology (NCSB), University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Department of Computer Science, Informatics Institute and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff3">  <label/> Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, MD 20705 USA  </aff>  <aff id="Aff4">  <label/> Division of Plant Sciences and NCSB, University of Missouri, Delta Center, P.O. Box 160, Portageville, MO 63873 USA  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: We performed QTL analysis for SCN <span class="trait">resistance</span> in PI 437655 in two mapping populations, characterized CNV of <span class="gene">Rhg1</span> through whole-genome resequencing and evaluated the effects of QTL pyramiding to enhance <span class="trait">resistance.</span> Soybean cyst nematode (SCN, <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe) is one of the most serious pests of soybean worldwide. PI 437655 has broader <span class="trait">resistance</span> to SCN HG types than PI 88788. The objectives of this study were to identify quantitative trait loci (QTL) underlying SCN <span class="trait">resistance</span> in PI 437655, and to evaluate the QTL for their contribution to SCN <span class="trait">resistance.</span> Two F6:7 recombinant inbred line populations, derived from cv. Williams 82 × PI 437655 and cv. Hutcheson × PI 437655 crosses, were evaluated for <span class="trait">resistance</span> to SCN HG types 1.2.5.7 (PA2), 0 (PA3), 1.3.5.6.7 (PA14), and 1.2.3.4.5.6.7 (LY2). The 1,536 SNP array was used to genotype the mapping populations and construct genetic linkage maps. Two significant QTL were consistently mapped on chromosomes (Chr.) 18 and 20 in these two populations. One QTL on Chr. 18, which corresponds to the known <span class="gene">Rhg1</span> locus, contributed <span class="trait">resistance</span> to SCN HG types 1.2.5.7, 0, 1.3.5.6.7, and 1.2.3.4.5.6.7 (PA2, PA3, PA14, and LY2, respectively). Copy number variation (CNV) analysis by whole-genome resequencing showed that PI 437655 and PI 88788 had similar CNV at the <span class="gene">Rhg1</span> locus. The QTL on Chr. 20 contributed <span class="trait">resistance</span> to SCN HG types 1.3.5.6.7 (PA14) and 1.2.3.4.5.6.7 (LY2). Evaluation of both QTL showed that pyramiding of <span class="gene">Rhg1</span> and the QTL on Chr. 20 significantly improved the <span class="trait">resistance</span> to SCN HG types 1.3.5.6.7 (PA14) and 1.2.3.4.5.6.7 (LY2) in both populations. Our studies provided useful information for deploying PI 437655 as a donor for SCN <span class="trait">resistance</span> in soybean breeding through marker-assisted selection.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> We performed QTL analysis for SCN <span class="trait">resistance</span> in PI 437655 in two mapping populations, characterized CNV of </bold> <bold> <italic> <span class="gene">Rhg1</span> </italic> </bold> <bold> through whole-genome resequencing and evaluated the effects of QTL pyramiding to enhance <span class="trait">resistance.</span> </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN, <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe) is one of the most serious pests of soybean worldwide. PI 437655 has broader <span class="trait">resistance</span> to SCN HG types than PI 88788. The objectives of this study were to identify quantitative trait loci (QTL) underlying SCN <span class="trait">resistance</span> in PI 437655, and to evaluate the QTL for their contribution to SCN <span class="trait">resistance.</span> Two F <sub> 6:7 </sub> recombinant inbred line populations, derived from cv. Williams 82&#x000a0;&#x000d7;&#x000a0;PI 437655 and cv. Hutcheson&#x000a0;&#x000d7;&#x000a0;PI 437655 crosses, were evaluated for <span class="trait">resistance</span> to SCN HG types 1.2.5.7 (PA2), 0 (PA3), 1.3.5.6.7 (PA14), and 1.2.3.4.5.6.7 (LY2). The 1,536 SNP array was used to genotype the mapping populations and construct genetic linkage maps. Two significant QTL were consistently mapped on chromosomes (Chr.) 18 and 20 in these two populations. One QTL on Chr. 18, which corresponds to the known <italic> <span class="gene">Rhg1</span> </italic> locus, contributed <span class="trait">resistance</span> to SCN HG types 1.2.5.7, 0, 1.3.5.6.7, and 1.2.3.4.5.6.7 (PA2, PA3, PA14, and LY2, respectively). Copy number variation (CNV) analysis by whole-genome resequencing showed that PI 437655 and PI 88788 had similar CNV at the <italic> <span class="gene">Rhg1</span> </italic> locus. The QTL on Chr. 20 contributed <span class="trait">resistance</span> to SCN HG types 1.3.5.6.7 (PA14) and 1.2.3.4.5.6.7 (LY2). Evaluation of both QTL showed that pyramiding of <italic> <span class="gene">Rhg1</span> </italic> and the QTL on Chr. 20 significantly improved the <span class="trait">resistance</span> to SCN HG types 1.3.5.6.7 (PA14) and 1.2.3.4.5.6.7 (LY2) in both populations. Our studies provided useful information for deploying PI 437655 as a donor for SCN <span class="trait">resistance</span> in soybean breeding through marker-assisted selection.
					</p>
				
					<p>&nbsp;&nbsp;
						Quantitative trait loci (QTL) mapping is a powerful tool to identify genomic regions responsible for expression of important agronomic traits. Once the desired QTL are mapped, molecular markers that are tightly linked to the QTL can be applied in marker-assisted breeding to improve and shorten the process of developing resistant cultivars. To date, many QTL conferring <span class="trait">resistance</span> to SCN in soybean have been mapped on almost all chromosomes except for Chr. 2 (D1b) (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ; Vuong et al. <xref ref-type="bibr" rid="CR32"> 2010 </xref> ; Winter et al. <xref ref-type="bibr" rid="CR36"> 2007 </xref> ; Wang et al. <xref ref-type="bibr" rid="CR35"> 2004 </xref> ; Kabelka et al. <xref ref-type="bibr" rid="CR22"> 2005 </xref> ; Guo et al. <xref ref-type="bibr" rid="CR18"> 2005 </xref> , <xref ref-type="bibr" rid="CR19"> 2006 </xref> ; Wu et al. <xref ref-type="bibr" rid="CR38"> 2009 </xref> ). Among these, two QTL, <italic> <span class="gene">Rhg1</span> </italic> on Chr. 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr. 8 (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ), were commonly mapped in various sources of <span class="trait">resistance,</span> primarily plant introductions (PIs). Recently, these two QTL have been successfully cloned from PI 88788 and cultivar Forrest, respectively (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ; Liu et al. <xref ref-type="bibr" rid="CR24"> 2012 </xref> ). The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was found to be controlled by three genes (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ). The alteration of expression level caused by copy number variation (CNV) rather than sequence mutation for these three genes explained the phenotypic differences between susceptible and resistant varieties (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ). <italic> <span class="gene">Rhg4</span> </italic> encodes a serine hydroxymethyltransferase that is essential for cellular one-carbon metabolism (Liu et al. <xref ref-type="bibr" rid="CR24"> 2012 </xref> ). Two point mutations in Forrest altered a key regulatory property of this enzyme, which may disturb the folate homeostasis and lead to a nutritional deficiency preventing life of the nematode (Liu et al. <xref ref-type="bibr" rid="CR24"> 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The QTL significantly associated with <span class="trait">resistance</span> to multi-SCN HG types 1.2.5.7 (PA2), 0 (PA3), 1.3.5.6.7 (PA14), and 1.2.3.4.5.6.7 (LY2) were detected and consistently mapped to Chr. 18 in both genetic populations (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ; Supplements S1 and S2). In Pop1, the total phenotypic variation explained by this QTL was 10.8&#x000a0;% for HG type 1.2.5.7 (PA2), 56.2&#x000a0;% for HG Type 0 (PA3), 51.7&#x000a0;% for HG Type 1.3.5.6.7 (PA14), and 30.7&#x000a0;% for 1.2.3.4.5.6.7 (LY2) (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). In Pop2, this QTL explained 9.2&#x000a0;% of total phenotypic variation for HG type 1.2.5.7 (PA2), 58.1&#x000a0;% for HG type 0 (PA3), 27.1&#x000a0;% for HG type 1.3.5.6.7 (PA14), and 20.9&#x000a0;% for 1.2.3.4.5.6.7 (LY2) (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). Although the confidence intervals mapped in the two populations were slightly different, they overlapped in the same genomic region (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ), indicating the same QTL was detected in both populations. The QTL on Chr. 18 has been commonly mapped in various sources, which corresponds to the known QTL <italic> <span class="gene">Rhg1</span> </italic> (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ). Moreover, previous studies showed that there are two different types of <italic> <span class="gene">Rhg1</span> </italic> : Peking-type <italic> <span class="gene">Rhg1</span> </italic> and PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ). PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> can confer soybean SCN <span class="trait">resistance</span> by itself (Kim et al. <xref ref-type="bibr" rid="CR23"> 2010 </xref> ; Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ; Brucker et al. <xref ref-type="bibr" rid="CR7"> 2005 </xref> ). In contrast, Peking-type <italic> <span class="gene">Rhg1</span> </italic> functions only when <italic> <span class="gene">Rhg4</span> </italic> is present (Meksem et al. <xref ref-type="bibr" rid="CR25"> 2001 </xref> ; Brucker et al. <xref ref-type="bibr" rid="CR7"> 2005 </xref> ; Liu et al. <xref ref-type="bibr" rid="CR24"> 2012 </xref> ). In our study, a significant association was not observed between markers in the <italic> <span class="gene">Rhg4</span> </italic> interval and <span class="trait">resistance</span> to any SCN isolate tested (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). Therefore, it was evident that PI 437655 might contain the same <italic> <span class="gene">Rhg1</span> </italic> allele as PI 88788 rather than the <italic> <span class="gene">Rhg1</span> </italic> allele from Peking. To investigate if <italic> <span class="gene">Rhg1</span> </italic> between PI 437655 and PI 88788 were the same, we conducted CNV analysis of <italic> <span class="gene">Rhg1</span> </italic> using whole-genome resequencing. Compared to the susceptible soybean cv. Hutcheson, both PI 437655 and PI 88788 had a significant increase in copy numbers of <italic> <span class="gene">Rhg1</span> </italic> (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). No significant difference was detected for copy numbers for <italic> <span class="gene">Rhg1</span> </italic> between PI 437655 and PI 88788 (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ), which indicated that PI 437655 and PI 88788 might have the same <italic> <span class="gene">Rhg1</span> </italic> allele. <table-wrap id="Tab3"> <label> Table&#x000a0;3 </label> <caption> <p> Significant QTL for <span class="trait">resistance</span> to SCN HG types 1.2.5.7 (PA2), 0 (PA3), 1.3.5.6.7 (PA14), and 1.2.3.4.5.6.7 (LY2) identified and mapped in two recombinant inbred line (RIL) mapping populations derived from crosses of Hutcheson&#x000a0;&#x000d7;&#x000a0;PI 437655 (Pop1) and Williams 82&#x000a0;&#x000d7;&#x000a0;PI 437655 (Pop2)
					</p>
				
					<p>&nbsp;&nbsp;
						We evaluated the <italic> <span class="gene">Rhg1</span> </italic> locus and the QTL on Chr. 20 for their contribution to the <span class="trait">resistance</span> to different SCN HG types in these two mapping populations. Our results showed that <italic> <span class="gene">Rhg1</span> </italic> significantly reduced the FI % of SCN HG types 1.2.5.7, 0, 1.3.5.6.7, and 1.2.3.4.5.6.7 (PA2, PA3, PA14, and LY2, respectively) in both populations (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ). Addition of the QTL on Chr. 20 in genotypes with <italic> <span class="gene">Rhg1</span> </italic> reduced FI (%) of SCN HG type 1.3.5.6.7 (PA14) and 1.2.3.4.5.6.7 (LY2) by approximately 40&#x000a0;% (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ). This indicated that pyramiding different QTL for <span class="trait">resistance</span> could significantly confer more stable and higher <span class="trait">resistance</span> in soybean to multiple HG types of SCN. <table-wrap id="Tab4"> <label> Table&#x000a0;4 </label> <caption> <p> Evaluation of confirmed QTL for their contribution for <span class="trait">resistance</span> to SCN HG types 1.2.5.7 (PA2), 0 (PA3), 1.3.5.6.7 (PA14), and 1.2.3.4.5.6.7 (LY2)
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been widely mapped in various sources of SCN <span class="trait">resistance</span> (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ; Guo et al. <xref ref-type="bibr" rid="CR18"> 2005 </xref> , <xref ref-type="bibr" rid="CR19"> 2006 </xref> ; Vuong et al. <xref ref-type="bibr" rid="CR33"> 2011 </xref> ; Wu et al. <xref ref-type="bibr" rid="CR38"> 2009 </xref> ). It was shown that there were two different types of <italic> <span class="gene">Rhg1</span> </italic> , PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> and Peking-type <italic> <span class="gene">Rhg1</span> </italic> . These two <italic> <span class="gene">Rhg1</span> </italic> were believed to be different functional alleles (Meksem et al. <xref ref-type="bibr" rid="CR25"> 2001 </xref> ; Brucker et al. <xref ref-type="bibr" rid="CR7"> 2005 </xref> ). PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> was recently cloned (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ). However, the <italic> <span class="gene">Rhg1</span> </italic> allele from Peking has not been cloned. To date, many sources of SCN <span class="trait">resistance</span> contain <italic> <span class="gene">Rhg1</span> </italic> (Concibido et al. <xref ref-type="bibr" rid="CR10"> 2004 </xref> ). Because the level of SCN <span class="trait">resistance</span> conferred by the <italic> <span class="gene">Rhg1</span> </italic> gene was associated with CNVs (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ), it is important to determine CNVs of <italic> <span class="gene">Rhg1</span> </italic> among those sources of <span class="trait">resistance.</span> In our study, we predicted that PI 437655 might carry PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> because <italic> <span class="gene">Rhg4</span> </italic> was not detected in PI 437655. CNV analysis showed that PI 437655 had the same high number of copies of <italic> <span class="gene">Rhg1</span> </italic> as PI 88788, which indicated PI 437655 and PI 88788 might have the same level of SCN <span class="trait">resistance</span> or the same <italic> <span class="gene">Rhg1</span> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Currently, PI 88788 has been widely employed as a predominant donor source in breeding for soybean cultivars with SCN <span class="trait">resistance.</span> The most important gene underlying SCN <span class="trait">resistance</span> in PI 88788 is <italic> <span class="gene">Rhg1</span> </italic> (Cook et al. <xref ref-type="bibr" rid="CR11"> 2012 </xref> ). However, several studies have reported that more and more SCN populations overcame the SCN <span class="trait">resistance</span> conferred by PI 88788 (Diers and Arelli <xref ref-type="bibr" rid="CR12"> 1999 </xref> ; Faghihi et al. <xref ref-type="bibr" rid="CR14"> 2008 </xref> ), which highlighted the need to develop soybean cultivars with broader and more stable <span class="trait">resistance</span> to multiple HG types. In addition to <italic> <span class="gene">Rhg1</span> </italic> , PI 437655 also contained a QTL on Chr. 20, not found in PI 88788 (Glover et al. <xref ref-type="bibr" rid="CR16"> 2004 </xref> ). This additional QTL might be the reason why PI 437655 showed better SCN <span class="trait">resistance</span> than PI 88788 to the four HG types evaluated in this study. RILs with the combination of the <italic> <span class="gene">Rhg1</span> </italic> locus and the QTL on Chr. 20 in each of two populations had enhanced SCN <span class="trait">resistance</span> than genotypes with the <italic> <span class="gene">Rhg1</span> </italic> allele alone. Thus, pyramiding different <span class="trait">resistance</span> QTL will likely be an effective approach to reduce the problem of increased virulence on <span class="trait">resistance</span> sources like PI 88788 due to genetic shifts in SCN populations. Moreover, it has been reported that there was no linkage drag for <span class="trait">yield</span> when either cv. Fayette or its derived cultivars were utilized in soybean breeding for Northern cultivars. The newly identified QTL in PI 437655 can be introgressed into either to cv. Fayette or its derived cultivars for broadening <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Supplementary material 2 A genetic linkage map constructed using an F <sub> 6:7 </sub> recombinant inbred line (RIL) population from a Williams 82&#x000a0;&#x000d7;&#x000a0;PI 437655 cross. The confidence intervals of SCN <span class="trait">resistance</span> QTL were shown by the bars on the right of chromosome. The bars filled with black color represent the QTL resistant to HG type 1.2.5.7 (PA2). The <span class="gene">bar</span> filled no color represents the QTL resistant to HG type 0 (PA3). The bars filled with slashes represent the QTL resistant to HG type 1.3.5.6.7 (PA14). The bars filled with black dots represent the QTL resistant to HG type 1.2.3.4.5.6.7 (LY2) (TIFF 2817&#x000a0;kb)
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Manipulation of two α-endo-β-1,4-glucanase genes, AtCel6 and <span class="gene">GmCel7,</span> reduces susceptibility to <span class="trait">Heterodera</span> <span class="trait">glycines</span> in soybean roots.</h2>
			<h3>PMID:24844661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Woo MO(1), Beard H, MacDonald MH, Brewer EP, Youssef RM, Kim H, Matthews BF.
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant endo-β-1,4-glucanases (EGases) include cell wall-modifying enzymes that are involved in nematode-induced growth of syncytia (feeding structures) in nematode-infected roots. EGases in the α- and β-subfamilies contain signal peptides and are secreted, whereas those in the γ-subfamily have a membrane-anchoring domain and are not secreted. The Arabidopsis α-EGase <span class="gene">At1g48930,</span> designated as AtCel6, is known to be down-regulated by beet cyst nematode <span class="trait">(Heterodera</span> schachtii) in Arabidopsis roots, whereas another α-EGase, AtCel2, is up-regulated. Here, we report that the ectopic expression of AtCel6 in soybean roots reduces susceptibility to both soybean cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines)</span> and root knot nematode (Meloidogyne incognita). Suppression of <span class="gene">GmCel7,</span> the soybean homologue of AtCel2, in soybean roots also reduces the susceptibility to SCN. In contrast, in studies on two γ-EGases, both ectopic expression of AtKOR2 in soybean roots and suppression of the soybean homologue of AtKOR3 had no significant effect on SCN parasitism. Our results suggest that secreted α-EGases are likely to be more useful than membrane-bound γ-EGases in the development of an SCN-resistant soybean through gene manipulation. Furthermore, this study provides evidence that Arabidopsis shares molecular events of cyst nematode parasitism with soybean, and confirms the suitability of the Arabidopsis-H. schachtii interaction as a model for the soybean-H. <span class="trait">glycines</span> pathosystem.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distinct Copy Number, Coding Sequence, and Locus Methylation Patterns Underlie Rhg1-Mediated Soybean <span class="trait">Resistance</span> to Soybean Cyst Nematode.</h2>
			<h3>PMID:24733883</h3>
				

				
			<p>
				作者：Cook DE(1), Bayless AM(1), Wang K(1), Guo X(1), Song Q(1), Jiang J(1), Bent AF(2).
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Copy number variation of kilobase-scale genomic DNA segments, beyond presence/absence polymorphisms, can be an important driver of adaptive traits. <span class="trait">Resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> (Rhg1) is a widely utilized quantitative trait locus that makes the strongest known contribution to <span class="trait">resistance</span> against soybean cyst nematode (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines,</span> the most damaging pathogen of soybean (Glycine max). <span class="gene">Rhg1</span> was recently discovered to be a complex locus at which resistance-conferring haplotypes carry up to 10 tandem repeat copies of a 31-kb DNA segment, and three disparate genes present on each repeat contribute to SCN <span class="trait">resistance.</span> Here, we use whole-genome sequencing, fiber-FISH (fluorescence in situ hybridization), and other methods to discover the genetic variation at <span class="gene">Rhg1</span> across 41 diverse soybean accessions. Based on copy number variation, transcript abundance, nucleic acid polymorphisms, and differentially methylated DNA regions, we find that SCN <span class="trait">resistance</span> is associated with multicopy <span class="gene">Rhg1</span> haplotypes that form two distinct groups. The tested high-copy-number <span class="gene">Rhg1</span> accessions, including plant introduction (PI) 88788, contain a flexible number of copies (seven to 10) of the 31-kb <span class="gene">Rhg1</span> repeat. The identified low-copy-number <span class="gene">Rhg1</span> group, including PI 548402 (Peking) and PI 437654, contains three copies of the <span class="gene">Rhg1</span> repeat and a newly identified allele of <span class="gene">Glyma18g02590</span> (a predicted α-SNAP [α-soluble N-ethylmaleimide-sensitive factor attachment protein]). There is strong evidence for a shared origin of the two resistance-conferring multicopy <span class="gene">Rhg1</span> groups and subsequent independent evolution. Differentially methylated DNA regions also were identified within <span class="gene">Rhg1</span> that correlate with SCN <span class="trait">resistance.</span> These data provide insights into copy number variation of multigene segments, using as the example a disease <span class="trait">resistance</span> trait of high economic importance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Syntaxin 31 functions in Glycine max <span class="trait">resistance</span> to the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:24452833  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pant SR(1), Matsye PD, McNeece BT, Sharma K, Krishnavajhala A, Lawrence GW, Klink VP.
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A Glycine max syntaxin 31 homolog <span class="gene">(Gm-SYP38)</span> was identified as being expressed in nematode-induced feeding structures known as syncytia undergoing an incompatible interaction with the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span> The observed <span class="gene">Gm-SYP38</span> expression was consistent with prior gene expression analyses that identified the alpha soluble NSF attachment protein <span class="gene">(Gm-α-SNAP)</span> <span class="trait">resistance</span> gene because homologs of these genes physically interact and function together in other genetic systems. Syntaxin 31 is a protein that resides on the cis face of the Golgi apparatus and binds α-SNAP-like proteins, but has no known role in <span class="trait">resistance.</span> Experiments presented here show <span class="gene">Gm-α-SNAP</span> overexpression induces <span class="gene">Gm-SYP38</span> transcription. Overexpression of <span class="gene">Gm-SYP38</span> rescues G. max [Williams 82/PI 518671], genetically rhg1 (-/-), by suppressing H. <span class="trait">glycines</span> parasitism. In contrast, <span class="gene">Gm-SYP38</span> RNAi in the rhg1 (+/+) genotype G. max [Peking/PI 548402] increases susceptibility. <span class="gene">Gm-α-SNAP</span> and <span class="gene">Gm-SYP38</span> overexpression induce the transcriptional activity of the cytoplasmic receptor-like kinase BOTRYTIS INDUCED KINASE 1 <span class="gene">(Gm-BIK1-6)</span> which is a family of defense proteins known to anchor to membranes through a 5' MGXXXS/T(R) N-myristoylation sequence. <span class="gene">Gm-BIK1-6</span> had been identified previously by RNA-seq experiments as expressed in syncytia undergoing an incompatible reaction. <span class="gene">Gm-BIK1-6</span> overexpression rescues the resistant phenotype. In contrast, <span class="gene">Gm-BIK1-6</span> RNAi increases parasitism. The analysis demonstrates a role for syntaxin 31-like genes in <span class="trait">resistance</span> that until now was not known.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean salicylic acid methyltransferase gene confers <span class="trait">resistance</span> to soybean cyst nematode.</h2>
			<h3>PMID:24034273  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M, Zhao N, Zhu JJ, Zhuang X, Liu W, Pantalone VR, Arelli PR, Stewart CN Jr, Chen F.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Salicylic acid plays a critical role in activating plant defence responses after pathogen attack. Salicylic acid methyltransferase (SAMT) modulates the level of salicylic acid by converting salicylic acid to methyl salicylate. Here, we report that a SAMT gene from soybean <span class="gene">(GmSAMT1)</span> plays a role in soybean defence against soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines</span> Ichinohe, SCN). <span class="gene">GmSAMT1</span> was identified as a candidate SCN defence-related gene in our previous analysis of soybean defence against SCN using GeneChip microarray experiments. The current study started with the isolation of the full-length cDNAs of <span class="gene">GmSAMT1</span> from a SCN-resistant soybean line and from a SCN-susceptible soybean line. The two cDNAs encode proteins of identical sequences. The <span class="gene">GmSAMT1</span> cDNA was expressed in Escherichia coli. Using in vitro enzyme assays, E. coli-expressed <span class="gene">GmSAMT1</span> was confirmed to function as salicylic acid methyltransferase. The apparent Km value of <span class="gene">GmSAMT1</span> for salicylic acid was approximately 46 μM. To determine the role of <span class="gene">GmSAMT1</span> in soybean defence against SCN, transgenic hairy roots overexpressing <span class="gene">GmSAMT1</span> were produced and tested for SCN <span class="trait">resistance.</span> Overexpression of <span class="gene">GmSAMT1</span> in SCN-susceptible backgrounds significantly reduced the development of SCN, indicating that overexpression of <span class="gene">GmSAMT1</span> in the transgenic hairy root system could confer <span class="trait">resistance</span> to SCN. Overexpression of <span class="gene">GmSAMT1</span> in transgenic hairy roots was also found to affect the expression of selected genes involved in salicylic acid biosynthesis and salicylic acid signal transduction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ectopic expression of AtPAD4 broadens <span class="trait">resistance</span> of soybean to soybean cyst and root-knot nematodes.</h2>
			<h3>PMID:23617694  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Youssef RM(1), MacDonald MH, Brewer EP, Bauchan GR, Kim KH, Matthews BF.
				<br/> 
				年份：2013 Apr 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The gene encoding <span class="gene">PAD4</span> (PHYTOALEXIN-DEFICIENT4) is required in Arabidopsis for expression of several genes involved in the defense response to Pseudomonas syringae pv. maculicola. AtPAD4 (Arabidopsis thaliana PAD4) encodes a lipase-like protein that plays a regulatory role mediating salicylic acid signaling. RESULTS: We expressed the gene encoding AtPAD4 in soybean roots of composite plants to test the ability of AtPAD4 to deter plant parasitic nematode development. The transformed roots were challenged with two different plant parasitic nematode genera represented by soybean cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines)</span> and root-knot nematode (RKN; Meloidogyne incognita). Expression of AtPAD4 in soybean roots decreased the number of mature SCN females 35 days after inoculation by 68 percent. Similarly, soybean roots expressing AtPAD4 exhibited 77 percent fewer galls when challenged with RKN. CONCLUSIONS: Our experiments show that AtPAD4 can be used in an economically important crop, soybean, to provide a measure of <span class="trait">resistance</span> to two different genera of nematodes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homo-dimerization and ligand binding by the leucine-rich repeat domain at <italic> RHG1/RFS2 </italic> underlying <span class="trait">resistance</span> to two soybean pathogens</h2>
			<h3>PMID:23497186</h3>
				

				
					<h3>PMCID:3626623</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Afzal </surname>  <given-names> Ahmed J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> afzal.5@osu.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Srour </surname>  <given-names> Ali </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> asrour@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Goil </surname>  <given-names> Abhishek </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> abhishekgoil81@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Vasudaven </surname>  <given-names> Sheeja </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> isheeja@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Liu </surname>  <given-names> Tianyun </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> isheeja@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Samudrala </surname>  <given-names> Ram </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ram@compbio.washington.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Dogra </surname>  <given-names> Navneet </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> navneet@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Kohli </surname>  <given-names> Punit </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> pkohli@chem.siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Malakar </surname>  <given-names> Ayan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mailme.ayan@siu.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Lightfoot </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ga4082@siu.edu </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The protein encoded by <span class="gene">GmRLK18-1</span> (Glyma_18_02680 on chromosome 18) was a receptor like kinase (RLK) encoded within the soybean (Glycine max L. Merr.) Rhg1/Rfs2 locus. The locus underlies <span class="trait">resistance</span> to the soybean cyst nematode (SCN) <span class="trait">Heterodera</span> <span class="trait">glycines</span> (I.) and causal agent of sudden death syndrome (SDS) Fusarium virguliforme (Aoki). Previously the leucine rich repeat (LRR) domain was expressed in Escherichia coli. RESULTS: The aims here were to evaluate the LRRs ability to; homo-dimerize; bind larger proteins; and bind to small peptides. Western analysis suggested homo-dimers could form after protein extraction from roots. The purified LRR domain, from residue 131-485, was seen to form a mixture of monomers and homo-dimers in vitro. Cross-linking experiments in vitro showed the H274N region was close (&lt;11.1 A) to the highly conserved cysteine residue C196 on the second homo-dimer subunit. Binding constants of 20-142 nM for peptides found in plant and nematode secretions were found. Effects on plant phenotypes including wilting, stem bending and <span class="trait">resistance</span> to infection by SCN were observed when roots were treated with 50 pM of the peptides. Far-Western analyses followed by <span class="gene">MS</span> showed methionine synthase and cyclophilin bound strongly to the LRR domain. A second LRR from <span class="gene">GmRLK08-1</span> (Glyma_08_g11350) did not show these strong interactions. CONCLUSIONS: The LRR domain of the <span class="gene">GmRLK18-1</span> protein formed both a monomer and a homo-dimer. The LRR domain bound avidly to 4 different CLE peptides, a cyclophilin and a methionine synthase. The CLE peptides <span class="gene">GmTGIF,</span> <span class="gene">GmCLE34,</span> <span class="gene">GmCLE3</span> and HgCLE were previously reported to be involved in root growth inhibition but here <span class="gene">GmTGIF</span> and HgCLE were shown to alter stem morphology and <span class="trait">resistance</span> to SCN. One of several models from homology and ab-initio modeling was partially validated by cross-linking. The effect of the 3 amino acid replacements present among <span class="gene">RLK</span> allotypes, A87V, Q115K and H274N were predicted to alter domain stability and function. Therefore, the LRR domain of <span class="gene">GmRLK18-1</span> might underlie both root development and disease <span class="trait">resistance</span> in soybean and provide an avenue to develop new variants and ligands that might promote reduced losses to SCN.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The protein encoded by <span class="gene">GmRLK18-1</span> ( <italic> Glyma_18_02680 </italic> on chromosome 18) was a receptor like kinase (RLK) encoded within the soybean (Glycine max L. Merr.) <italic> Rhg1/Rfs2 </italic> locus. The locus underlies <span class="trait">resistance</span> to the soybean cyst nematode (SCN) <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> (I.) and causal agent of sudden death syndrome (SDS) <italic> Fusarium virguliforme </italic> (Aoki). Previously the leucine rich repeat (LRR) domain was expressed in <italic> Escherichia coli </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						The aims here were to evaluate the LRRs ability to; homo-dimerize; bind larger proteins; and bind to small peptides. Western analysis suggested homo-dimers could form after protein extraction from roots. The purified LRR domain, from residue 131&#x02013;485, was seen to form a mixture of monomers and homo-dimers <italic> in vitro </italic> . Cross-linking experiments <italic> in vitro </italic> showed the H274N region was close (&#x0003c;11.1 A) to the highly conserved cysteine residue C196 on the second homo-dimer subunit. Binding constants of 20&#x02013;142 nM for peptides found in plant and nematode secretions were found. Effects on plant phenotypes including wilting, stem bending and <span class="trait">resistance</span> to infection by SCN were observed when roots were treated with 50 pM of the peptides. Far-Western analyses followed by <span class="gene">MS</span> showed methionine synthase and cyclophilin bound strongly to the LRR domain. A second LRR from <span class="gene">GmRLK08-1</span> (Glyma_08_g11350) did not show these strong interactions.
					</p>
				
					<p>&nbsp;&nbsp;
						The LRR domain of the <span class="gene">GmRLK18-1</span> protein formed both a monomer and a homo-dimer. The LRR domain bound avidly to 4 different CLE peptides, a cyclophilin and a methionine synthase. The CLE peptides <span class="gene">GmTGIF,</span> <span class="gene">GmCLE34,</span> <span class="gene">GmCLE3</span> and HgCLE were previously reported to be involved in root growth inhibition but here <span class="gene">GmTGIF</span> and HgCLE were shown to alter stem morphology and <span class="trait">resistance</span> to SCN. One of several models from homology and <italic> ab-initio </italic> modeling was partially validated by cross-linking. The effect of the 3 amino acid replacements present among <span class="gene">RLK</span> allotypes, A87V, Q115K and H274N were predicted to alter domain stability and function. Therefore, the LRR domain of <span class="gene">GmRLK18-1</span> might underlie both root development and disease <span class="trait">resistance</span> in soybean and provide an avenue to develop new variants and ligands that might promote reduced losses to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants employ both cell surface and cytoplasmic receptors to respond to a wide array of signals from pathogens [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. The receptor protein kinases (RPKs) represent one of the two large gene families implicated to underlie the recognition events that lead to pathogen <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. Two of the most destructive pathogens in soybean ( <italic> Glycine max </italic> L. Merr <italic> . </italic> ) are the soybean cyst nematode (SCN; <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> I.) and sudden death syndrome (SDS) agent <italic> Fusarium virguliforme </italic> (Aoki) [ <xref ref-type="bibr" rid="B3"> 3 </xref> ]. The complex genetics of the cyst nematode populations, the partial nature of plant <span class="trait">resistance</span> and temperature sensitivity makes controlling the nematode a difficult task [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Elicitation of plant defenses in response to the pathogens were shown to involve the activity of <span class="gene">RLK</span> proteins [ <xref ref-type="bibr" rid="B5"> 5 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ] introgressed from Peking. Two loci, <italic> <span class="gene">Rhg4</span> </italic> on chromosome 8 (linkage group (Lg) A2) and <italic> Rhg1/Rfs2 </italic> on chromosome 18 (Lg G), contain genes that encode receptor like kinase (RLK) proteins within the RPK gene family implicated in <span class="trait">resistance.</span> <span class="gene">GmRLK08-1</span> (Glyma_08_11350) is near <italic> <span class="gene">Rhg4</span> </italic> and <span class="gene">GmRLK18-1</span> (Glyma_18_02680) is within <italic> Rhg1/Rfs2 </italic> [ <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> - <xref ref-type="bibr" rid="B10"> 10 </xref> ]. The translated proteins are both RLKs with extra-cellular leucine rich repeats (LRR). Many other genes can alter SCN responses. However, only <span class="gene">GmRLK18-1</span> has been shown to underlie <span class="trait">resistance</span> to both pathogens in transgenic plants [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						At <italic> Rhg1/Rfs2, </italic> the <span class="trait">resistance</span> phenotypes were perfectly associated with the <span class="gene">GmRLK18-1</span> allotype 1 [ <xref ref-type="bibr" rid="B5"> 5 </xref> ] and that allele in transgenic plants provided partial <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Combined, the amino acid changes (A87V, Q115K and H274N) were sufficient to differentiate between plant introductions possessing type I <span class="trait">resistance</span> (Peking based <span class="trait">resistance)</span> and four other allotypes. No studies to date have attempted to analyze the role of these amino acid changes on overall protein structure, hence the molecular basis of <span class="trait">resistance</span> to SCN and SDS pathogenesis remains unexplored.
					</p>
				
					<p>&nbsp;&nbsp;
						Previously a three dimensional model for an RPK protein [ <xref ref-type="bibr" rid="B27"> 27 </xref> ] was predicted but the modeled RPK protein was not an RLK. Equally, a model for and <span class="gene">RLK</span> was predicted, but the protein acted in symbiosis not defense [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Here a model of the LRR domain from a <span class="gene">RLK</span> protein involved in <span class="trait">resistance</span> is reported based on homology modeling of the extracellular LRR domain (residue 141&#x02013;471) of the <span class="gene">GmRLK18-1</span> protein. Modeling for <span class="gene">GmRLK18-1-LRR</span> was based on PRI (2BNH) [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. The effect of the amino acid substitutions on protein stability was inferred; structure for fold and class analysis was made; and secondary structure was analyzed <italic> in vitro </italic> and <italic> in silico </italic> to predict whether <span class="gene">GmRLK18-1</span> may homo-dimerize <italic> in vivo </italic> . The models were tested with proteins cross linked <italic> in vitro </italic> . Ligand binding was measured with short CLE-like peptides.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">GmRLK08-1</span> LRR domain (from the <span class="gene">RLK</span> protein at <italic> <span class="gene">Rhg4</span> </italic> ) showed a lower affinity for most of the CLE peptides tested (50&#x02013;338 nM). However, the long and short versions of <span class="gene">GmCLE34</span> and short version of <span class="gene">GmCLV3</span> bound with the highest affinity (50&#x02013;52 nM) suggesting these were among the ligand signals integrated by <span class="gene">GmRLK08-1.</span> The nematode peptide HgCLV3 was bound weakly (78 nM). This result would agree with the conclusion that <span class="gene">GmRLK08-1</span> protein was not the sole element underlying the <span class="trait">resistance</span> reaction encoded at the <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The <span class="gene">GmRLK08-1</span> LRR domain protein bound very weakly to the symbiosis associated <span class="gene">GmRIC</span> and <span class="gene">GmNIC,</span> as did <span class="gene">GmRLK18-1.</span> Unlike <span class="gene">GmRLK18-1</span> the <span class="gene">GmRLK08-1</span> protein bound weakly to <span class="gene">GmTDIF.</span> Therefore, the LRRs showed distinct peptide ligand specificities reflecting their different sequence and structures.
					</p>
				
					<p>&nbsp;&nbsp;
						CLE-like protein derived consensus peptides are defined set of peptides found in plant genomes and involved in both short and long distance signaling [ <xref ref-type="bibr" rid="B26"> 26 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> - <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Five of the 16 consensus motifs found among CLE peptides are expressed in roots (Table1), as was the <span class="gene">GmRLK18-1.</span> The <span class="gene">GmRLK18-1</span> LRR domain had a strong binding constant for <span class="gene">GmCLV3</span> and N that are thought to be involved in meristem differentiation [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. During SCN pathogenesis a new meristem is initiated to bring a tracheary element close to the feeding site, N might mediate that and be detected by the <span class="trait">resistance</span> protein <span class="gene">GmRLK18-1.</span> T is the tracheary element differentiation inhibitory factor <span class="gene">(GmTDIF)</span> that might provide inhibition of feeding site induced developmental processes during defence. <span class="gene">GmCLE34</span> peptides were produced in pro-vascular tissues [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. CLE domains thought to be involved in symbiosis [ <xref ref-type="bibr" rid="B26"> 26 </xref> ] were not strongly bound suggesting they were not ligands of physiological relevance, although nematode parasitism does decrease nodulation [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Previously a perfect association between allele 1 (Forrest based <span class="trait">resistance)</span> and <span class="trait">resistance</span> to the SCN (Hg Type 0 or race 3) was shown in three segregating populations and an association study of unrelated PIs [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Recently, previously susceptible plants transgenic with the Forrest allele of <span class="gene">GmRLK18-1</span> were found to be resistant to both SCN and SDS [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Based on multiple sequence alignment of predicted <span class="gene">GmRLK18-1</span> haplotypes, a quantitative trait nucleotide in the LRR of <italic> rhg1 </italic> was inferred, that alters A87 to V87 in the context of Q115 and H274 rather than K115 and N274 (Additional file <xref ref-type="supplementary-material" rid="S6"> 6 </xref> : Figure S3). There is a closely linked copy number variation that may also contribute to <span class="trait">resistance</span> [8].
					</p>
				
					<p>&nbsp;&nbsp;
						This report describes a functional analysis and structural prediction of the LRR domain from a <span class="gene">RLK</span> protein involved in plant pest and disease <span class="trait">resistance.</span> Structural predictions were made and validated. The results showed that the isolated LRR of <span class="gene">GmRLK18-1</span> binds peptides found in both nematode and plant secretions with high affinity. A question might arise over whether isolated LRRs behave in the same way as in whole RLKs and RPKs. However, the LRR was previously shown to form an active structure by CD [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. Homo-dimerization was detected in 3 different assays (2 <italic> in vitro </italic> ) and the avidity estimated by a peptide binding assay. Larger proteins were also shown to bind the LRR domain including cyclophilin, methionine synthase and a protein of the same mass and charge as the <span class="gene">RLK</span> monomer. Therefore, the isolate LRR behaved in ways similar to the RLK. The LRR domain was large and appeared to be binding different ligands. The <span class="gene">RLK</span> might integrate those signals into a single appropriate response.
					</p>
				
					<p>&nbsp;&nbsp;
						Some important features of the candidate <span class="gene">GmRLK18-1</span> protein were predicted and validated. The three polymorphisms to the C-terminal side, A87V, Q115K, and H274N may affect protein stability. The quantitative trait nucleotide underlying H274N that differentiates the Peking allotype (for <span class="trait">resistance)</span> from all others may have structural or functional significance. That amino acid polymorphism, H274N, may play an important role in the stability of the monomer as well as the homo-dimer. Future experiments will involve mutation of this amino acid and correlating it with <span class="gene">GmRLK18-1</span> stability and/or function in binding the CLE like peptide motifs found in all plants and their many of their pathogens [ <xref ref-type="bibr" rid="B30"> 30 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence diversity among the seven <span class="gene">GmRLK18-1</span> allotypes. Alanine at position 87 is only present in the &#x02018;Peking&#x02019; sequence. Two additional changes Q to K at position 115 and H to N at position 274 are not exclusive to <span class="trait">resistance</span> type I.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Engineered <span class="trait">resistance</span> and hypersusceptibility through functional metabolic studies of 100 genes in soybean to its major pathogen, the soybean cyst nematode</h2>
			<h3>PMID:23389673</h3>
				

				
					<h3>PMCID:3634990</h3>
					<h4>期刊:Planta</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F. </given-names>  </name>  <address>  <phone> 301-504-5730 </phone>  <fax> 301-504-5728 </fax>  <email> ben.matthews@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Beard </surname>  <given-names> Hunter </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kabir </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Youssef </surname>  <given-names> Reham M. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hosseini </surname>  <given-names> Parsa </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brewer </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Soybean Genomics and Improvement Laboratory, United States Department of Agriculture, Agricultural Research Service, 10300 Baltimore Ave, Bldg 006, Beltsville, MD 20705 USA  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						During pathogen attack, the host plant induces genes to ward off the pathogen while the pathogen often produces effector proteins to increase susceptibility of the host. Gene expression studies of syncytia formed in soybean root by soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines)</span> identified many genes altered in expression in resistant and susceptible roots. However, it is difficult to assess the role and impact of these genes on <span class="trait">resistance</span> using gene expression patterns alone. We selected 100 soybean genes from published microarray studies and individually overexpressed them in soybean roots to determine their impact on cyst nematode development. Nine genes reduced the number of mature females by more than 50 % when overexpressed, including genes encoding ascorbate peroxidase, β-1,4-endoglucanase, short chain dehydrogenase, lipase, DREPP membrane protein, calmodulin, and three proteins of unknown function. One gene encoding a serine hydroxymethyltransferase decreased the number of mature cyst nematode females by 45 % and is located at the <span class="gene">Rhg4</span> locus. Four genes increased the number of mature cyst nematode females by more than 200 %, while thirteen others increased the number of mature cyst nematode females by more than 150 %. Our data support a role for auxin and ethylene in susceptibility of soybean to cyst nematodes. These studies highlight the contrasting gene sets induced by host and nematode during infection and provide new insights into the interactions between host and pathogen at the molecular level. Overexpression of some of these genes result in a greater decrease in the number of cysts formed than recognized soybean cyst nematode <span class="trait">resistance</span> loci.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> in the host can be multigenic, and numerous researchers have studied the genetics of major loci of SCN <span class="trait">resistance</span> in soybean (Matson and Williams <xref ref-type="bibr" rid="CR94"> 1965 </xref> ; Concibido et al. <xref ref-type="bibr" rid="CR21"> 1994 </xref> , <xref ref-type="bibr" rid="CR23"> 2004 </xref> ; Diers and Arelli <xref ref-type="bibr" rid="CR28"> 1999 </xref> ; Diers et al. <xref ref-type="bibr" rid="CR29"> 1997 </xref> ; Schuster et al. <xref ref-type="bibr" rid="CR122"> 2001 </xref> ). For example, several genes ( <italic> rhg1, rhg2, rhg3 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ) in soybean cv Peking confer <span class="trait">resistance</span> to SCN race 1 (Caldwell et al. <xref ref-type="bibr" rid="CR15"> 1960 </xref> ; Matson and Williams <xref ref-type="bibr" rid="CR94"> 1965 </xref> ). The <italic> rhg1 </italic> locus confers the most <span class="trait">resistance</span> of the known <italic> Rhg </italic> loci, and it reduces by half the number of SCN cysts that form on soybean roots (Concibido et al. <xref ref-type="bibr" rid="CR21"> 1994 </xref> , <xref ref-type="bibr" rid="CR22"> 1997 </xref> , <xref ref-type="bibr" rid="CR23"> 2004 </xref> ; Ruben et al. <xref ref-type="bibr" rid="CR117"> 2006 </xref> ). In PI 437654, <span class="trait">resistance</span> to SCN race 3 was mapped to three loci, including the <italic> <span class="gene">Rhg4</span> </italic> locus on linkage group A within 1&#x000a0;cM of the <italic> I </italic> locus (Webb et al. <xref ref-type="bibr" rid="CR132"> 1995 </xref> ). Molecular markers near the <italic> <span class="gene">Rhg4</span> </italic> locus have been identified including PCR (Matthews et al. <xref ref-type="bibr" rid="CR97"> 1998 </xref> ), SSR (Cregan et al. <xref ref-type="bibr" rid="CR25"> 1999 </xref> ) and SNP (Qui et al. <xref ref-type="bibr" rid="CR111"> 2003 </xref> ) markers. Recently, candidate genes for both <italic> <span class="gene">Rhg1</span> </italic> (Cook et al. <xref ref-type="bibr" rid="CR24"> 2012 </xref> ; Matsye et al. <xref ref-type="bibr" rid="CR96"> 2012 </xref> ) and <italic> <span class="gene">Rhg4</span> </italic> (Liu et al. <xref ref-type="bibr" rid="CR93"> 2012 </xref> ) have been identified and tested. However, these genes alone do not provide <span class="trait">resistance</span> to all SCN populations. Scientists have been pursuing other approaches to provide <span class="trait">resistance</span> to SCN, including transforming plant roots with DNA constructs to silence nematode genes (Bakhetia et al. <xref ref-type="bibr" rid="CR6"> 2007 </xref> , <xref ref-type="bibr" rid="CR7"> 2008 </xref> ; Klink et al. <xref ref-type="bibr" rid="CR79"> 2009a </xref> , <xref ref-type="bibr" rid="CR80"> b </xref> , <xref ref-type="bibr" rid="CR83"> 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The gene expression vector pRAP15 is designed to overexpress genes using the figwort mosaic virus (FMV) promoter. The vector contains the gene encoding enhanced green fluorescent protein (eGFP) driven by the <italic> Agrobacterium rhizogenes </italic> <italic> rol </italic> D promoter so transformed roots can be identified easily. It also contains the attR1 and attR2 sites for Gateway <sup> &#x000ae; </sup> cloning, a tetracycline <span class="trait">resistance</span> gene (TetR) for bacterial selection, and the <italic> <span class="gene">bar</span> </italic> gene for chemical selection of transformed plant cells using the herbicide BASTA <sup> &#x000ae; </sup>
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of gene A35 decreased the female index 55&#x000a0;% of the control, and it is of interest, because it encodes a serine hydroxymethyltransferase located on chromosome eight <span class="gene">(Glyma08g11490)</span> of the soybean genome at the <italic> <span class="gene">Rhg4</span> </italic> locus (Matson and Williams <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) encoding a <span class="trait">resistance</span> gene for SCN (Cregan et al. <xref ref-type="bibr" rid="CR25"> 1999 </xref> ; Lewers et al. <xref ref-type="bibr" rid="CR90"> 2002 </xref> ; Liu et al. <xref ref-type="bibr" rid="CR93"> 2012 </xref> ). Microarray analysis of the expression of this gene indicates that the gene is increased in expression&#x000a0;3.1-fold at 2 dai and 2.1-fold at 5 dai in syncytia formed by SCN in the compatible interaction with Williams 82 (Ithal et al. <xref ref-type="bibr" rid="CR62"> 2007 </xref> ), but does not change in expression in the incompatible interaction with Peking (Klink et al. <xref ref-type="bibr" rid="CR77"> 2007b </xref> ) and with PI 88788 (Klink et al. <xref ref-type="bibr" rid="CR81"> 2010a </xref> , <xref ref-type="bibr" rid="CR82"> b </xref> ). We overexpressed transcript <span class="gene">Glyma08g11490.2,</span> which was recently mapped and tested by complementation by Liu et al. ( <xref ref-type="bibr" rid="CR93"> 2012 </xref> ) to show that it confers <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene encoding serine hydroxymethyltransferase (Gene A35) is located on chromosome 8 <span class="gene">(Glyma08g11490)</span> of the soybean genome (Schmutz et al. <xref ref-type="bibr" rid="CR121"> 2010 </xref> ; <ext-link ext-link-type="uri" xlink:href="http://www.Phytozome.com"> www.Phytozome.com </ext-link> ). Very recently, it has been shown to coincide with the <italic> Rhg </italic> 4 locus conferring <span class="trait">resistance</span> to SCN (Liu et al. <xref ref-type="bibr" rid="CR93"> 2012 </xref> ) through mutation analysis, gene silencing and by transforming roots of a susceptible soybean recombinant inbred line, ExF63, with the gene to restore <span class="trait">resistance.</span> When we overexpressed this gene in susceptible roots, the female index of SCN decreased by 45&#x000a0;%. Microarray analysis of the expression of this gene indicates that the gene is expressed more in syncytia formed by SCN in Williams 82 in the compatible interaction as compared to cells from uninfected control roots. It is expressed at 3.1-fold at 2 dai and 2.1-fold at 5 dai (Ithal et al. <xref ref-type="bibr" rid="CR62"> 2007 </xref> ), but does not change in expression in the incompatible interaction with Peking (Klink et al. <xref ref-type="bibr" rid="CR77"> 2007b </xref> ) nor with PI 88788 (Klink et al. <xref ref-type="bibr" rid="CR81"> 2010a </xref> ). Matson and Williams ( <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) first identified the <italic> <span class="gene">Rhg4</span> </italic> gene as a dominant gene required for <span class="trait">resistance</span> to SCN race 3. Rao-Arelli et al. ( <xref ref-type="bibr" rid="CR113"> 1992 </xref> ) demonstrated that Peking and PI 90763, resistant to SCN race 3, had three genes in common, one of which was <italic> <span class="gene">Rhg4</span> </italic> . The <italic> <span class="gene">Rhg4</span> </italic> gene was mapped to linkage group A2 using molecular markers (Weisemann et al. <xref ref-type="bibr" rid="CR134"> 1992 </xref> ; Matthews et al. <xref ref-type="bibr" rid="CR97"> 1998 </xref> ; Cregan et al. <xref ref-type="bibr" rid="CR25"> 1999 </xref> ; Qui et al. <xref ref-type="bibr" rid="CR111"> 2003 </xref> ); then it was physically mapped using BAC libraries by Lewers et al. <xref ref-type="bibr" rid="CR89"> 2001 </xref> ; <xref ref-type="bibr" rid="CR90"> 2002 </xref> ). TILLING experiments demonstrated that <span class="trait">resistance</span> is not conferred by the gene encoding an <span class="gene">LRR-RLK</span> that resides in the region (Liu et al. <xref ref-type="bibr" rid="CR92"> 2011 </xref> ). The historically defined SCN <span class="trait">resistance</span> loci had eluded identification for decades, and the gene expression experiments of syncytia undergoing the resistant reaction (Matsye et al. <xref ref-type="bibr" rid="CR95"> 2011 </xref> ) have provided a complimentary approach to fine mapping (Kim et al. <xref ref-type="bibr" rid="CR73"> 2010 </xref> ). For example, the expression of genes at the <italic> rhg1 </italic> locus was examined by Matsye et al. ( <xref ref-type="bibr" rid="CR95"> 2011 </xref> ), and the gene encoding alpha soluble <italic> N </italic> -ethylmaleimide-sensitive factor attachment protein (&#x003b1;-SNAP; <span class="gene">Glyma18g02590)</span> that maps to the <italic> rhg1 </italic> locus was recently examined for its effect on the female index of SCN (Matsye et al. <xref ref-type="bibr" rid="CR96"> 2012 </xref> ). Overexpression of that gene decreased the female index of SCN by approximately 50&#x000a0;% when overexpressed in soybean roots. The &#x003b1;-SNAP gene has been studied in numerous genetically tractable animal and fungal model systems (Weidman et al. <xref ref-type="bibr" rid="CR133"> 1989 </xref> ; Clary et al. <xref ref-type="bibr" rid="CR16"> 1990 </xref> ) and is part of the vesicular fusion and transport system that is involved in various types of plant defense responses (Collins et al. <xref ref-type="bibr" rid="CR20"> 2003 </xref> ; Assaad et al. <xref ref-type="bibr" rid="CR5"> 2004 </xref> ; Kalde et al. <xref ref-type="bibr" rid="CR67"> 2007 </xref> ; Patel and Dinesh-Kumar <xref ref-type="bibr" rid="CR108"> 2008 </xref> ; Hofius et al. <xref ref-type="bibr" rid="CR58"> 2009 </xref> ; Lenz et al. <xref ref-type="bibr" rid="CR88"> 2011 </xref> ; Lai et al. <xref ref-type="bibr" rid="CR86"> 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Annexins have been implicated in the regulation of cell growth and in signaling during stress. A nematode effector protein was recently described from <italic> H. schachtii </italic> , and it is similar to annexins (Patel et al. <xref ref-type="bibr" rid="CR109"> 2010 </xref> ). This effector is a homolog of the <italic> H. <span class="trait">glycines</span> </italic> effector <italic> Hg4F01 </italic> gene. The protein encoded by <italic> Hs4F01 </italic> had 33&#x000a0;% identity with annexin-1 ( <italic> annAt1 </italic> ) from <italic> Arabidopsis </italic> . Annexins are membrane and Ca2+&#x000a0;-binding proteins with many different functions (Talukdar et al. <xref ref-type="bibr" rid="CR128"> 2009 </xref> ). We overexpressed two genes encoding soybean annexin, <span class="gene">Glyma13g27020</span> (C12) and <span class="gene">Glyma13g27020</span> (A60) in soybean roots. Neither had a very significant effect on the female index of SCN when overexpressed, although C12 decreased the female index of SCN to 66&#x000a0;% of the control ( <italic> P </italic> &#x000a0;=&#x000a0;0.25). When the <italic> H. schachtii </italic> effector <italic> Hg4F01 </italic> was overexpressed in <italic> Arabidopsis </italic> , the plants were more susceptible to <italic> H. schachtii </italic> and had approximately 25&#x000a0;% more cysts than wild-type plants (Patel et al. <xref ref-type="bibr" rid="CR109"> 2010 </xref> ). However, when <italic> Arabidopsis </italic> plants transformed with <italic> Hg4F01 </italic> were infected with RKN, <italic> M. incognita, </italic> there was no significant change in the number of galls formed (Patel et al. <xref ref-type="bibr" rid="CR109"> 2010 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A soybean cyst nematode <span class="trait">resistance</span> gene points to a new mechanism of plant <span class="trait">resistance</span> to pathogens.</h2>
			<h3>PMID:23235880  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu S(1), Kandoth PK, Warren SD, Yeckel G, Heinz R, Alden J, Yang C, Jamai A, El-Mellouki T, Juvale PS, Hill J, Baum TJ, Cianzio S, Whitham SA, Korkin D, Mitchum MG, Meksem K.
				<br/> 
				年份：2012 Dec 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) is an important crop that provides a sustainable source of protein and oil worldwide. Soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines</span> Ichinohe) is a microscopic roundworm that feeds on the roots of soybean and is a major constraint to soybean production. This nematode causes more than US$1 billion in <span class="trait">yield</span> losses annually in the United States alone, making it the most economically important pathogen on soybean. Although planting of resistant cultivars forms the core management strategy for this pathogen, nothing is known about the nature of <span class="trait">resistance.</span> Moreover, the increase in virulent populations of this parasite on most known <span class="trait">resistance</span> sources necessitates the development of novel approaches for control. Here we report the map-based cloning of a gene at the <span class="gene">Rhg4</span> (for <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> 4) locus, a major quantitative trait locus contributing to <span class="trait">resistance</span> to this pathogen. Mutation analysis, gene silencing and transgenic complementation confirm that the gene confers <span class="trait">resistance.</span> The gene encodes a serine hydroxymethyltransferase, an enzyme that is ubiquitous in nature and structurally conserved across kingdoms. The enzyme is responsible for interconversion of serine and glycine and is essential for cellular one-carbon metabolism. Alleles of <span class="gene">Rhg4</span> conferring <span class="trait">resistance</span> or susceptibility differ by two genetic polymorphisms that alter a key regulatory property of the enzyme. Our discovery reveals an unprecedented plant <span class="trait">resistance</span> mechanism against a pathogen. The mechanistic knowledge of the <span class="trait">resistance</span> gene can be readily exploited to improve nematode <span class="trait">resistance</span> of soybean, an increasingly important global crop.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic relationships of soybean cyst nematode <span class="trait">resistance</span> originated in Gedenshirazu and PI84751 on <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci</h2>
			<h3>PMID:23136498</h3>
				

				
					<h3>PMCID:3406782</h3>
					<h4>期刊:Breeding Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Suzuki </surname>  <given-names> Chika </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-602"> 1 </xref>  <xref rid="c1-bs-61-602" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Yoshinori </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-602"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takeuchi </surname>  <given-names> Toru </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-602"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yumoto </surname>  <given-names> Setsuzo </given-names>  </name>  <xref ref-type="aff" rid="af4-bs-61-602"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shirai </surname>  <given-names> Shigehisa </given-names>  </name>  <xref ref-type="aff" rid="af3-bs-61-602"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) <span class="trait">(Heterodera</span> <span class="trait">glycines</span> Ichinohe) is one of the most damaging pests of soybean (Glycine max (L.) Merr.). Host plant <span class="trait">resistance</span> has been the most effective control method. Because of the spread of multiple SCN races in Hokkaido, the Tokachi Agricultural Experiment Station has bred soybeans for SCN <span class="trait">resistance</span> since 1953 by using 2 main <span class="trait">resistance</span> resources PI84751 (resistant to races 1 and 3) and Gedenshirazu (resistant to race 3). In this study, we investigated the genetic relationships of SCN <span class="trait">resistance</span> originating from major SCN <span class="trait">resistance</span> genes in Gedenshirazu and PI84751 by using SSR markers. We confirmed that race 1 <span class="trait">resistance</span> in PI84751 was independently controlled by 4 genes, 2 of which were rhg1 and Rhg4. We classified the PI84751- type allele of <span class="gene">Rhg1</span> as rhg1-s and the Gedenshirazu-type allele of <span class="gene">Rhg1</span> as rhg1-g. In the cross of the Gedenshirazu-derived race 3-resistant lines and the PI84751-derived races 1- and 3-resistant lines, the presence of rhg1-s and <span class="gene">Rhg4</span> was responsible for race 1-resistance. These results indicated that it was possible to select race 1 resistant plants by using marker-assisted selection for the rhg1-s and <span class="gene">Rhg4</span> alleles through a PI84751 origin × Gedenshirazu origin cross.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe) is one of the most damaging pests of soybean ( <italic> Glycine max </italic> (L.) Merr.). Host plant <span class="trait">resistance</span> has been the most effective control method. Because of the spread of multiple SCN races in Hokkaido, the Tokachi Agricultural Experiment Station has bred soybeans for SCN <span class="trait">resistance</span> since 1953 by using 2 main <span class="trait">resistance</span> resources PI84751 (resistant to races 1 and 3) and Gedenshirazu (resistant to race 3). In this study, we investigated the genetic relationships of SCN <span class="trait">resistance</span> originating from major SCN <span class="trait">resistance</span> genes in Gedenshirazu and PI84751 by using SSR markers. We confirmed that race 1 <span class="trait">resistance</span> in PI84751 was independently controlled by 4 genes, 2 of which were <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> . We classified the PI84751- type allele of <italic> <span class="gene">Rhg1</span> </italic> as <italic> rhg1-s </italic> and the Gedenshirazu-type allele of <italic> <span class="gene">Rhg1</span> </italic> as <italic> rhg1-g </italic> . In the cross of the Gedenshirazu-derived race 3-resistant lines and the PI84751-derived races 1- and 3-resistant lines, the presence of <italic> rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> was responsible for race 1-resistance. These results indicated that it was possible to select race 1 resistant plants by using marker-assisted selection for the <italic> rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles through a PI84751 origin &#x000d7; Gedenshirazu origin cross.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) is one of the most damaging pests of soybean. Populations of SCN exhibit diversity in their ability to infect resistant soybean cultivars. In Japan, races 1, 3 and 5 have been reported ( <xref rid="b8-bs-61-602" ref-type="bibr"> Inagaki 1979 </xref> , <xref rid="b15-bs-61-602" ref-type="bibr"> Shimizu <italic> et al. </italic> 1992 </xref> ). Host plant <span class="trait">resistance</span> based on race diversity has been the most effective control method. Gedenshirazu is a local soybean ( <italic> Glycine max </italic> (L.) Merr.) variety from Akita Prefecture, Japan. It has been used as a genetic source of <span class="trait">resistance</span> to SCN race 3 since the 1950s because of its large, high-quality seed and yellow hilum, making it a good source of soyfood. In Japan, most of the race 3-resistant varieties inherited <span class="trait">resistance</span> from Gedenshirazu. In Tokachi Agricultural Experiment Station (AES), several leading varieties of soybean were bred as progenies of Gedenshirazu, namely, Toyomusume ( <xref rid="b13-bs-61-602" ref-type="bibr"> Sasaki <italic> et al. </italic> 1988 </xref> ), Toyokomachi ( <xref rid="b14-bs-61-602" ref-type="bibr"> Sasaki <italic> et al. </italic> 1990 </xref> ), Yukihomare ( <xref rid="b19-bs-61-602" ref-type="bibr"> Tanaka <italic> et al. </italic> 2003 </xref> ) and Toyoharuka ( <xref rid="b20-bs-61-602" ref-type="bibr"> Tanaka <italic> et al. </italic> 2009 </xref> ). Another source of SCN <span class="trait">resistance,</span> PI84751, was introduced into the United States as a forage crop from China, but it is not suitable for food because it has a small, low-quality seed with a black seed coat. However, because of its excellent <span class="trait">resistance</span> to SCN, PI84751 has been used as a genetic source of <span class="trait">resistance</span> to SCN races 1 and 3 since the 1960s. Four genes were involved in SCN <span class="trait">resistance</span> in PI84751 ( <xref rid="b12-bs-61-602" ref-type="bibr"> Sakai and Sunada 1987 </xref> ), and substantial efforts were required to breed varieties resistant to races 1 and 3. <xref rid="b2-bs-61-602" ref-type="bibr"> Caldwell <italic> et al. </italic> (1960) </xref> reported that 3 recessive genes, <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> and <italic> rhg3 </italic> , control <span class="trait">resistance</span> to SCN in Peking, and that PI90763 and PI84751 possessed the same 3 <span class="trait">resistance</span> genes as Peking. <italic> <span class="gene">Rhg4</span> </italic> was reported as a dominant <span class="trait">resistance</span> gene that is tightly linked to the <italic> I </italic> locus, which controls pigment distribution on the seed coat ( <xref rid="b11-bs-61-602" ref-type="bibr"> Matson and Williams 1965 </xref> ). Two decades after the initiation of breeding, the Tokachi AES developed a variety of soybeans called Suzuhime with <span class="trait">resistance</span> to SCN races 1 and 3 ( <xref rid="b18-bs-61-602" ref-type="bibr"> Sunada <italic> et al. </italic> 1981 </xref> ), for special use in natto. PI84751 has a black seed coat, but Suzuhime and our race 1-resistant strains have yellow seed coats; we have already generated a number of strains through recombination between the <italic> <span class="gene">Rhg4</span> </italic> and <italic> I </italic> loci. The seed appearance of the progenies derived from PI84751 was gradually improved, but other agronomic traits such as seed size and adaptability for tofu processing are not sufficient.
					</p>
				
					<p>&nbsp;&nbsp;
						After a decade of soybean genetic mapping studies ( <xref rid="b4-bs-61-602" ref-type="bibr"> Cregan <italic> et al. </italic> 1999 </xref> ), the findings revealed <italic> rhg1 </italic> to be the most important gene in resistant soybean varieties. This gene was mapped to linkage group (LG) G from many sources ( <xref rid="b3-bs-61-602" ref-type="bibr"> Concibido <italic> et al. </italic> 2004 </xref> ). In addition, the <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> has been mapped to LG A2 from several sources ( <xref rid="b3-bs-61-602" ref-type="bibr"> Concibido <italic> et al. </italic> 2004 </xref> ). Recently candidate genes for <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> have been reported using positional cloning techniques ( <xref rid="b7-bs-61-602" ref-type="bibr"> Hauge <italic> et al. </italic> 2001 </xref> , <xref rid="b10-bs-61-602" ref-type="bibr"> Lightfoot and Meksem 2002 </xref> ). With regard to the genetic background of Gedenshirazu, the most effective quantitative trait locus (QTL) for <span class="trait">resistance</span> to race 3 was mapped to the same region as <italic> rhg1 </italic> ( <xref rid="b5-bs-61-602" ref-type="bibr"> Ferdous <italic> et al. </italic> 2006 </xref> , <xref rid="b9-bs-61-602" ref-type="bibr"> Kamiya <italic> et al. </italic> 2000 </xref> ). However, the genetic relationship of <span class="trait">resistance</span> genes between Gedenshirazu and PI84751 based on molecular markers has not been reported.
					</p>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to explain the genetic relationships of SCN <span class="trait">resistance</span> derived from Gedenshirazu and PI84751 by focusing on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci. This study was composed of 3 parts. In the first part, we investigated the relationships between the proportion of race 1-resistant plants and genotypes by using recombinant inbred lines (RILs). In the second part, we evaluated the adaptability of the candidate markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which when used with our varieties conferred <span class="trait">resistance</span> to SCN. In the last part, we explained the relationships between PI84751 and Gedenshirazu with regard to the markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which were used in the analysis of the SCN <span class="trait">resistance</span> of BC <sub> 1 </sub> F <sub> 2 </sub> individuals.
					</p>
				
					<p>&nbsp;&nbsp;
						Eighty-eight breeding strains were used to confirm the relationship between SCN <span class="trait">resistance</span> and the genotypes of simple sequence repeat (SSR) markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> . The analysis of pedigree indicated that their <span class="trait">resistance</span> to SCN was originated in PI84751 or Gedenshirazu. Their <span class="trait">resistance</span> to SCN was evaluated in the test of specific characteristics in the breeding program. <span class="trait">Resistance</span> to SCN was classified into 3 groups: resistant to race 1, resistant to race 3, and susceptible. DNA was isolated from the leaves of each strain in 2003.
					</p>
				
					<p>&nbsp;&nbsp;
						The sizes of the SSR fragments for Satt309 and Satt632 of the resistant parent To-8E were consistent with those of PI84751. There were polymorphisms of these markers in the susceptible parent Tokei-758, resistant parent PI84751, and progeny lines segregated by these 2 loci. The 10 lines exhibiting race 1 <span class="trait">resistance</span> harbored both PI84751 alleles on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci. Race 1-susceptible and race 3-resistant lines harbored the PI84751 allele of the <italic> <span class="gene">Rhg1</span> </italic> locus and the susceptible type allele of the <italic> <span class="gene">Rhg4</span> </italic> locus ( <xref rid="t2-bs-61-602" ref-type="table"> Table 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Our results for TC9703RILs were in accordance with Sakai&#x02019;s findings ( <xref rid="b12-bs-61-602" ref-type="bibr"> Sakai and Sunada 1987 </xref> ) that the inheritance of <span class="trait">resistance</span> from PI84751 fitted a 4-gene model ( <xref rid="t1-bs-61-602" ref-type="table"> Table 1 </xref> ). In all race 1-resistant lines of TC9703RILs, the genotypes of Satt309 and Satt632 were the same as those of PI84751 ( <xref rid="t2-bs-61-602" ref-type="table"> Table 2 </xref> ). Therefore, we assumed that the genotype of Satt309 was closely linked to the <italic> <span class="gene">Rhg1</span> </italic> locus, the genotype of Satt632 was closely linked to the <italic> <span class="gene">Rhg4</span> </italic> locus and <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> were 2 of the 4 <span class="trait">resistance</span> genes of PI84751. Conversely, race 3-resistant and race 1-susceptible lines did not express the PI84751 type of <italic> <span class="gene">Rhg4</span> </italic> ( <xref rid="t2-bs-61-602" ref-type="table"> Table 2 </xref> ), indicating that <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In our breeding strains, all race 1-resistant strains had the same genotypes of <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> as PI84751 and the same genotypes as race 1-resistant lines of TC9703RILs. These results suggested that Satt309 and Satt632 conferred adaptability as linked markers of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci in our breeding strains. At the <italic> <span class="gene">Rhg1</span> </italic> locus, the sizes of the SSR fragments of Satt309 of PI84751 and Gedenshirazu origin were polymorphic. We detected a PI84751 type, a susceptible type, and a Gedenshirazu type ( <xref rid="f1-bs-61-602" ref-type="fig"> Fig. 1 </xref> ). In the cross between race 3-resistant strains derived from Gedenshirazu and race 1-resistant strains derived from PI84751, the lines that expressed Gedenshirazu alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus did not exhibit <span class="trait">resistance</span> to race 1; only the lines that expressed PI84751 alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus exhibited race 1 <span class="trait">resistance</span> ( <xref rid="t5-bs-61-602" ref-type="table"> Table 5 </xref> ). The result of this cross indicated that PI84751 and Gedenshirazu each have a different <span class="trait">resistance</span> allele at the <italic> <span class="gene">Rhg1</span> </italic> locus and supported the existence of <italic> rhg1 </italic> alleles as reported by <xref rid="b1-bs-61-602" ref-type="bibr"> Brucker (2005) </xref> in the cross between PI437654 and PI88788. We assigned the PI84751 type of <italic> rhg1 </italic> as <italic> rhg1-s </italic> and the Gedenshirazu type of <italic> rhg1 </italic> as <italic> rhg1-g </italic> . On the relationship between <span class="trait">resistance</span> to SCN and genotypes of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, the combination of alleles on the two loci could explain the type of <span class="trait">resistance</span> to SCN ( <xref rid="t3-bs-61-602" ref-type="table"> Table 3 </xref> ). Both alleles of <italic> rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> were indispensable for race 1 <span class="trait">resistance;</span> however, either <italic> rhg1-s </italic> or <italic> rhg1-g </italic> was necessary for race 3 <span class="trait">resistance.</span> In addition, <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance</span> as shown by the analysis of TC9703RILs. The QTLs for <span class="trait">resistance</span> to SCN race 3 originating from Gedenshirazu were not detected on LG A2, where <italic> <span class="gene">Rhg4</span> </italic> lies ( <xref rid="b5-bs-61-602" ref-type="bibr"> Ferdous <italic> et al. </italic> 2006 </xref> ). There were 2 exceptions to the relationships between SCN <span class="trait">resistance</span> and genotypes. The two strains that expressed <italic> rhg1-s </italic> or <italic> rhg1-g </italic> did not exhibit SCN <span class="trait">resistance</span> ( <xref rid="t3-bs-61-602" ref-type="table"> Table 3 </xref> ). Because <span class="trait">resistance</span> to SCN is controlled by multiple <span class="trait">resistance</span> genes, we speculate that the strains lacked <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> and/or <italic> rhg3 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the cross between race 3-resistant strains derived from Gedenshirazu and race 1-resistant strains derived from PI84751, the proportion of segregation for the race 1 response at BC <sub> 1 </sub> F <sub> 2 </sub> fitted a 1-dominant gene and 1-recessive gene model. When the BC <sub> 1 </sub> F <sub> 2 </sub> plants were <italic> rhg1-s </italic> homozy-gous and <italic> <span class="gene">Rhg4</span> </italic> homozygous or heterozygous, they were resistant to race 1 ( <xref rid="t5-bs-61-602" ref-type="table"> Table 5 </xref> ). From our preliminary experiments, it was estimated that all lines exhibit <span class="trait">resistance</span> to race 3 in this cross. These results demonstrated that 2 SCN resistant genes differed in Gedenshirazu and PI84751, and differences in 2 loci such as <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> between Gedenshirazu and PI84751 controlled <span class="trait">resistance</span> to race 1. In addition, they indicated that the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg4</span> </italic> locus was dominant and the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg1</span> </italic> locus was recessive. This indicates that the other 2 <span class="trait">resistance</span> genes had the same or similar functions in PI84751 and Gedenshirazu, and race3 <span class="trait">resistance</span> in Gedenshirazu was controlled by 3 genes. <xref rid="b9-bs-61-602" ref-type="bibr"> Kamiya (2000) </xref> reported a race 3 progeny test of RILs derived from a cross of Toyomusume (race3 resistant) and Tsurukogane (susceptible) and the proportion of race3-resistant lines matched the probability of <span class="trait">resistance</span> in the RILs segregating for 3 genes. However, in a previous study, <xref rid="b16-bs-61-602" ref-type="bibr"> Shirai (1991) </xref> reported that race 3 <span class="trait">resistance</span> in Gedenshirazu was controlled by 2 genes. The reason our result differed from Shirai&#x02019;s is that their data was not based on the evaluation of a progeny test but on the evaluation of F <sub> 2 </sub> individuals, and it is possible that the susceptible parents had <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> or <italic> rhg3 </italic> , which could not detected by the phenotypes. We illustrated the genetic relationships of SCN <span class="trait">resistance</span> originating from Gedenshirazu and PI84751 in <xref rid="f3-bs-61-602" ref-type="fig"> Fig. 3 </xref> . Using a cross of PI84751-derived and Gedenshirazu-derived varieties, it is possible to select race 1-resistant plants by using only 2 loci such as <italic> Rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> and by omitting the remaining 2 loci. This method requires crossing PI84751-derived and Gedenshirazu-derived varieties. In practical soybean breeding programs, when we pyramid useful agronomical traits such as cool weather tolerance, yielding, and adaptability for tofu processing in addition to SCN <span class="trait">resistance,</span> this system will promote soybean breeding by using marker-assisted selection effectively.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The receptor like kinase at Rhg1-a/Rfs2 caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome and soybean cyst nematode as a transgene by altering signaling responses.</h2>
			<h3>PMID:22857610  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Srour A(1), Afzal AJ, Blahut-Beatty L, Hemmati N, Simmonds DH, Li W, Liu M, Town CD, Sharma H, Arelli P, Lightfoot DA.
				<br/> 
				年份：2012 Aug 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max (L. Merr.)) <span class="trait">resistance</span> to any population of <span class="trait">Heterodera</span> <span class="trait">glycines</span> (I.), or Fusarium virguliforme (Akoi, O'Donnell, Homma & Lattanzi) required a functional allele at Rhg1/Rfs2. H. <span class="trait">glycines,</span> the soybean cyst nematode (SCN) was an ancient, endemic, pest of soybean whereas F. virguliforme causal agent of sudden death syndrome (SDS), was a recent, regional, pest. This study examined the role of a receptor like kinase (RLK) <span class="gene">GmRLK18-1</span> (gene model Glyma_18_02680 at 1,071 kbp on chromosome 18 of the genome sequence) within the Rhg1/Rfs2 locus in causing <span class="trait">resistance</span> to SCN and SDS. RESULTS: A BAC (B73p06) encompassing the Rhg1/Rfs2 locus was sequenced from a resistant cultivar and compared to the sequences of two susceptible cultivars from which 800 SNPs were found. Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59 kbp at the center of which the <span class="gene">GmRLK18-1</span> was the most polymorphic gene and encoded protein. Analyses were made of plants that were either heterozygous at, or transgenic (and so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <span class="gene">GmRLK18-1.</span> Those plants infested with either H. <span class="trait">glycines</span> or F. virguliforme showed that the allele for <span class="trait">resistance</span> was dominant. In the absence of <span class="gene">Rhg4</span> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root and leaf symptoms of SDS caused by F. virguliforme and provided partial <span class="trait">resistance</span> to three different populations of nematodes (mature female cysts were reduced by 30-50%). In the presence of <span class="gene">Rhg4</span> the plants with the transgene were nearly classed as fully resistant to SCN (females reduced to 11% of the susceptible control) as well as SDS. A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <span class="gene">GmRLK18-1.</span> Field trials of transgenic plants showed an increase in foliar susceptibility to insect herbivory. CONCLUSIONS: The inference that soybean has adapted part of an existing pathogen recognition and defense cascade (H.glycines; SCN and insect herbivory) to a new pathogen (F. virguliforme; SDS) has broad implications for crop improvement. Stable <span class="trait">resistance</span> to many pathogens might be achieved by manipulation the genes encoding a small number of pathogen recognition proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Recombination suppression at the dominant Rhg1/Rfs2 locus underlying soybean <span class="trait">resistance</span> to the cyst nematode.</h2>
			<h3>PMID:22200919  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Afzal AJ(1), Srour A, Saini N, Hemmati N, El Shemy HA, Lightfoot DA.
				<br/> 
				年份：2012 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Host <span class="trait">resistance</span> to "yellow dwarf" or "moonlight" disease cause by any population (Hg type) of <span class="trait">Heterodera</span> <span class="trait">glycines</span> I., the soybean cyst nematode (SCN), requires a functional allele at rhg1. The host <span class="trait">resistance</span> encoded appears to mimic an apoptotic response in the giant cells formed at the nematode feeding site about 24-48 h after nematode feeding commences. Little is known about how the host response to infection is mediated but a linked set of 3 genes has been identified within the rhg1 locus. This study aimed to identify the role of the genes within the locus that includes a receptor-like kinase (RLK), a laccase and an ion antiporter. Used were near isogeneic lines (NILs) that contrasted at their rhg1 alleles, gene-based markers, and a new Hg type 0 and new recombination events. A syntenic gene cluster on Lg B1 was found. The effectiveness of SNP probes from the <span class="gene">RLK</span> for distinguishing homolog sequence variants on LgB1 from alleles at the rhg1 locus on LgG was shown. The resistant allele of the rhg1 locus was shown to be dominant in NILs. None of the recombination events were within the cluster of the three candidate genes. Finally, rhg1 was shown to reduce the plant root development. A model for rhg1 as a dominant multi-gene <span class="trait">resistance</span> locus based on the developmental control was inferred.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL underlying <span class="trait">resistance</span> to two HG types of <span class="trait">Heterodera</span> <span class="trait">glycines</span> found in soybean cultivar 'L-10'</h2>
			<h3>PMID:21569389</h3>
				

				
					<h3>PMCID:3224386</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Chang </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weich0@foxmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Dong </surname>  <given-names> Limin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> donglimin221355@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wang </surname>  <given-names> Zizhen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> milandewang@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Hu </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> nmghuhaibo@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hyp234286@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weiliteng2006@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhang </surname>  <given-names> Hongxia </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hxzhang@sippe.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Guo </surname>  <given-names> Maozu </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> maozuguo@hit.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wenbinli@neau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 5 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Resistance</span> of soybean (Glycine max L. Merr.) cultivars to populations of cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines</span> I.) was complicated by the diversity of HG Types (biotypes), the multigenic nature of <span class="trait">resistance</span> and the temperature dependence of <span class="trait">resistance</span> to biotypes. The objective here was to identify QTL for broad-spectrum <span class="trait">resistance</span> to SCN and examine the transcript abundances of some genes within the QTL. RESULTS: A Total of 140 F(5) derived F(7) recombinant inbred lines (RILs) were advanced by single-seed-descent from a cross between 'L-10' (a soybean cultivar broadly resistant to SCN) and 'Heinong 37' (a SCN susceptible cultivar). Associated QTL were identified by WinQTL2.1. QTL Qscn3-1 on linkage group (LG) E, Qscn3-2 on LG G, Qscn3-3 on LG J and Qscn14-1 on LG O were associated with SCN <span class="trait">resistance</span> in both year data (2007 and 2008). Qscn14-2 on LG O was identified to be associated with SCN <span class="trait">resistance</span> in 2007. Qscn14-3 on LG D2 was identified to be associated with SCN <span class="trait">resistance</span> in 2008. Qscn14-4 on LG J was identified to be associated with SCN <span class="trait">resistance</span> in 2008. The Qscn3-2 on LG G was linked to Satt309 (less than 4 cM), and explained 19.7% and 23.4% of the phenotypic variation in 2007 and 2008 respectively. Qscn3-3 was less than 5 cM from Satt244 on LG J, and explained 19.3% and 17.95% of the phenotypic variations in 2007 and 2008 respectively. Qscn14-4 could explain 12.6% of the phenotypic variation for the SCN race 14 <span class="trait">resistance</span> in 2008 and was located in the same region as Qscn3-3. The total phenotypic variation explained by Qscn3-2 and Qscn3-3 together was 39.0% and 41.3% in 2007 and 2008, respectively. Further, the flanking markers Satt275, Satt309, Sat_350 and Satt244 were used for the selection of resistant lines to SCN race 3, and the accuracy of selection was about 73% in this RIL population. Four genes in the predicted <span class="trait">resistance</span> gene cluster of LG J (chromosome 16) were successfully cloned by RT-PCR. The transcript encoded by the gene <span class="gene">Glyma16g30760.1</span> was abundant in the SCN resistant cultivar 'L-10' but absent in susceptible cultivar 'Heinong 37'. Further, the abundance was higher in root than in leaf for 'L-10'. Therefore, the gene was a strong candidate to underlie part of the <span class="trait">resistance</span> to SCN. CONCLUSIONS: Satt275, Satt309, Sat_305 and Satt244, which were tightly linked to the major QTL for <span class="trait">resistance</span> to SCN on LG G and J, would be candidates for marker-assisted selection of lines resistant to the SCN race 3. Among the six <span class="gene">RLK</span> genes, <span class="gene">Glyma16g30760.1</span> was found to accumulate transcripts in the SCN <span class="trait">resistance</span> cultivar 'L-10' but not in 'Heinong 37'. The transcript abundance was higher in root than in leaf for L-10.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A Total of 140 F <sub> 5 </sub> derived F <sub> 7 </sub> recombinant inbred lines (RILs) were advanced by single-seed-descent from a cross between 'L-10' (a soybean cultivar broadly resistant to SCN) and 'Heinong 37' (a SCN susceptible cultivar). Associated QTL were identified by WinQTL2.1. QTL Qscn3-1 on linkage group (LG) E, Qscn3-2 on LG G, Qscn3-3 on LG J and Qscn14-1 on LG O were associated with SCN <span class="trait">resistance</span> in both year data (2007 and 2008). Qscn14-2 on LG O was identified to be associated with SCN <span class="trait">resistance</span> in 2007. Qscn14-3 on LG D2 was identified to be associated with SCN <span class="trait">resistance</span> in 2008. Qscn14-4 on LG J was identified to be associated with SCN <span class="trait">resistance</span> in 2008. The Qscn3-2 on LG G was linked to Satt309 (less than 4 cM), and explained 19.7% and 23.4% of the phenotypic variation in 2007 and 2008 respectively. Qscn3-3 was less than 5 cM from Satt244 on LG J, and explained 19.3% and 17.95% of the phenotypic variations in 2007 and 2008 respectively. Qscn14-4 could explain 12.6% of the phenotypic variation for the SCN race 14 <span class="trait">resistance</span> in 2008 and was located in the same region as Qscn3-3. The total phenotypic variation explained by Qscn3-2 and Qscn3-3 together was 39.0% and 41.3% in 2007 and 2008, respectively. Further, the flanking markers Satt275, Satt309, Sat_350 and Satt244 were used for the selection of resistant lines to SCN race 3, and the accuracy of selection was about 73% in this RIL population. Four genes in the predicted <span class="trait">resistance</span> gene cluster of LG J (chromosome 16) were successfully cloned by RT-PCR. The transcript encoded by the gene <span class="gene">Glyma16g30760.1</span> was abundant in the SCN resistant cultivar 'L-10' but absent in susceptible cultivar 'Heinong 37'. Further, the abundance was higher in root than in leaf for 'L-10'. Therefore, the gene was a strong candidate to underlie part of the <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Satt275, Satt309, Sat_305 and Satt244, which were tightly linked to the major QTL for <span class="trait">resistance</span> to SCN on LG G and J, would be candidates for marker-assisted selection of lines resistant to the SCN race 3. Among the six <span class="gene">RLK</span> genes, <span class="gene">Glyma16g30760.1</span> was found to accumulate transcripts in the SCN <span class="trait">resistance</span> cultivar 'L-10' but not in 'Heinong 37'. The transcript abundance was higher in root than in leaf for L-10.
					</p>
				
					<p>&nbsp;&nbsp;
						Inheritance of <span class="trait">resistance</span> to SCN was first reported in Peking when three <span class="trait">resistance</span> genes ( <italic> rhg1,rhg2 and rhg3 </italic> ) were identified [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. The <italic> rhg1 </italic> locus has been shown to have the greatest impact on the development of SCN from all HG Types in several <span class="trait">resistance</span> sources including Peking, PI437654 and PI88788. Although originally reported as a recessive locus, the <italic> rhg1 </italic> locus has more recently been characterized as incomplete dominant or dominant depending on the HG Type 0 used [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. The fourth <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> had been confirmed as being necessary for full <span class="trait">resistance</span> to some races of SCN in Peking but not PI88788. It behaved as a dominant gene in Peking-derived sources. Both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were necessary for the full <span class="trait">resistance</span> to SCN race 3 and 14. An additional dominant gene was identified in PI88788 and was designated <italic> Rhg5 </italic> that also had been characterized as a dominant gene [ <xref ref-type="bibr" rid="B9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> was inferred to be linked with pBLT24, pBLT56 and <italic> I </italic> gene for seed coat color on linkage group A2 (chromosome 8) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. QTL identified and localized for SCN <span class="trait">resistance</span> based on both genotypic and phenotypic data in a segregating population was first reported by Concibido et al. (1994)[ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. During the following decade, a total of 60 SCN resistant QTL were reported on the following linkage groups: A1, A2, B1, B2, C1, C2, D1a, D2, E, F, G, H, I, J, L, M, and N [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. QTL on the LG G near <italic> rhg1 </italic> was detected in most studies and was considered to be one of the major genes underlying <span class="trait">resistance</span> to SCN. Nearly 80% of these studies used the SCN HG Type 0 (race 3) <span class="trait">resistance</span> because it was the main cause of <span class="trait">yield</span> reduction among soybeans grown in many areas, including Northeast of China. A minor QTL S16-5 on LG K provided 5% additive effects for <span class="trait">resistance</span> to SCN race 1 had been reported in 2009, this was the first report of a QTL on linkage group K associated with SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. Another QTL was detected on LG D2 (Satt574; P = 0.001, R2 = 11%) and associated with the <span class="trait">resistance</span> to HG Types 1.3- and 1.2.5- [ <xref ref-type="bibr" rid="B6"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Candidate genes underlying QTL were identified and one, a lecine-rich repeat (LRR) trans-membrane (TM) receptor like kinase (RLK) was tested in transgenic plants [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. <span class="trait">Resistance</span> was provided to SCN and a fungal pathogen, <italic> Fusarium virguliforme </italic> . In contrast, another <span class="gene">RLK</span> in the <italic> <span class="gene">Rhg4</span> </italic> interval was shown not to underlie the <span class="trait">resistance</span> locus [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. Candidate genes underlying other loci were not reported to date.
					</p>
				
					<p>&nbsp;&nbsp;
						All the QTL mentioned above were projected on the genetic map <span class="gene">GmComposite2003</span> based on the homothetic function. For the interval between Sat_350 and Satt244 that contained the genes underlying <span class="trait">resistance</span> to both SCN race 3 and SCN race 14, the QTL was first projected on the genome map according to the physical distance ratio of genetic distance of 412 kbp/cM between the two DNA markers. A total of 343 putatively gene encoded sequences encompassed in this region were used for <span class="trait">resistance</span> gene analog (RGA) detection by the hmmsearch program. Six encoded sequences were considered to be candidate SCN R-genes (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). According to the conversed domain shared by all the R-genes, all of these six encoded sequences were identified to encode LRR-TM-PK protein [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. It was notable that these encoded sequences were organized in a cluster on the chromosome 16 (inferred to be LG J). Four encoded sequences within this cluster of chromosome 16 were successfully amplified by RT-PCR and showed a high homology with other LRR-TM-PK type of <span class="trait">resistance</span> genes (data not shown). The results of semi-quantitative RT-PCR showed that <span class="gene">Glyma16g30760.1</span> was only abundant in the SCN resistant cultivar 'L-10' but not in susceptible 'Heinong 37'. Further, the transcript abundance was higher in root than in leaf for 'L-10'. Therefore, the gene might play a role in the SCN <span class="trait">resistance</span> (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). Future analyses would be focused to determine the role in the <span class="trait">resistance</span> mechanism underlain by <span class="gene">Glyma16g30760.1.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The interval on LG J contained both Qscn3-3 and Qscn14-4 (including <italic> rhg5) </italic> might underlie multi-races <span class="trait">resistance</span> to SCN in soybean [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. In this study, no QTL was identified on the LG A2. The <span class="gene">Rhg4</span> locus was considered to be another main QTL for SCN <span class="trait">resistance,</span> tightly linked to <italic> I </italic> locus [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Four alleles were known at <italic> I </italic> , the dominant allele inhibited seed coat pigmentation, leading to possession of a yellow seed coat, whereas the recessive i allele conferred a pigmented seed coat. Two other alleles, i <sup> i </sup> and i <sup> k </sup> , gave rise to restricted pigmentation of the hilum and the saddle-shaped region, respectively [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. In our work, there was no significant correlation between the color of seed coat (or hilum) and SCN <span class="trait">resistance.</span> This result indicated that the line 'L-10' had another locus for <span class="trait">resistance</span> to SCN race 3 independent of the <span class="gene">Rhg4</span> locus. This locus may be similar to the LG J locus found in PI 88788 [ <xref ref-type="bibr" rid="B7"> 7 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Qscn14-4 might underlie multi-races <span class="trait">resistance</span> in soybean. Kanazin et al. (1996) identified a gene cluster that contained eight RGAs on LG J in the similar region [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. Further, six members of the <span class="gene">RLK</span> family were involved in one cluster on Chromosome J. Among these the <span class="gene">RLK</span> gene <span class="gene">Glyma16g30760.1</span> was found to accumulate transcripts in the SCN <span class="trait">resistance</span> cultivar 'L-10' but not in 'Heinong 37' and the transcript abundance was higher in root than in leaf for 'L-10', suggesting that the gene might play a role in the SCN <span class="trait">resistance</span> to race 3 and race 14. Melito et al. (2010) suggested that the <span class="gene">RLK</span> at <italic> rhg1 </italic> was involved in root development but not nematode development as shown by others [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Unfortunately, the study of Melito et al. (2010) was flawed by the lack of an analysis of the target <span class="gene">RLK</span> protein abundance despite the available antibodies. Further studies of the RLKs at <italic> rhg1 </italic> and <italic> rhg5 </italic> will be needed for the clarification of the <span class="trait">resistance</span> mechanism and involvement of the candidate genes.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Glyma16g30760.1</span> was tightly linked to the QTL Qscn3-3 and Qscn14-4, indicating that this gene might be associated with both SCN race 3 and race 14 <span class="trait">resistance.</span> Therefore, a real-time PCR combined with molecular marker-based selection could be used for the future breeding of SCN resistant lines.
					</p>
				
					<p>&nbsp;&nbsp;
						The four main SSR markers, Satt275, Satt309, Sat_305 and Satt244, which were tightly linked to the major QTL for <span class="trait">resistance</span> to SCN on LG G and J, would be candidates for marker-assisted selection of lines resistant to the SCN race 3. Among the six <span class="gene">RLK</span> genes, <span class="gene">Glyma16g30760.1</span> was found to accumulate transcripts in the SCN <span class="trait">resistance</span> cultivar 'L-10' but not in 'Heinong 37'. The transcript abundance was higher in root than in leaf for 'L-10', suggesting that the gene might play a role in the SCN <span class="trait">resistance</span> to race 3 and race 14.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Gene expression profiling and shared promoter motif for cell wall-modifying proteins expressed in soybean cyst nematode-infected roots.</h2>
			<h3>PMID:21430185  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Murphy CA, Yang R.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We hypothesized that soybean cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines)</span> co-opts part or all of one or more innate developmental process in soybean (Glycine max) to establish its feeding structure, syncytium, in soybean roots. The syncytium is formed within the vascular bundle by partial degradation of cell walls and membranes between adjacent parenchyma cells. A mature syncytium incorporates as many as 200 cells into one large multinucleated cell. Gene expression patterns for several cell wall-modifying proteins were compared in multiple tissues undergoing major shifts in cell wall integrity. These included SCN-colonized roots, root tips where vascular differentiation occurs, flooded roots (aerenchyma), adventitious rooting in hypocotyls, and leaf abscission zones. A search in the 5' upstream promoters of these genes identified a motif (SCNbox1: WGCATGTG) common to several genes that were up-regulated in several different tissues. The polygalacturonase 11 promoters <span class="gene">(GmPG11a/b)</span> include the SCNbox1 motif. The expression pattern for <span class="gene">GmPG11a</span> was examined further in transgenic soybean containing a PG11a promoter fused to a β-glucuronidase (GUS) reporter gene. GUS expression was highest in cells undergoing radial expansion in the stele and/or cell wall dissolution. GUS staining was not observed in cortical cells where a lateral root tip or a growing nematode emerged through the root cortex.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Soybean <span class="gene">Rhg1</span> locus for <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> regulates the expression of a large number of stress- and defense-related genes in degenerating feeding cells.</h2>
			<h3>PMID:21335526  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kandoth PK(1), Ithal N, Recknor J, Maier T, Nettleton D, Baum TJ, Mitchum MG.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To gain new insights into the mechanism of soybean (Glycine max) <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines),</span> we compared gene expression profiles of developing syncytia in soybean near-isogenic lines differing at <span class="gene">Rhg1</span> (for <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines),</span> a major quantitative trait locus for <span class="trait">resistance,</span> by coupling laser capture microdissection with microarray analysis. Gene expression profiling revealed that 1,447 genes were differentially expressed between the two lines. Of these, 241 (16.8%) were stress- and defense-related genes. Several stress-related genes were up-regulated in the resistant line, including those encoding homologs of enzymes that lead to increased levels of reactive oxygen species and proteins associated with the unfolded protein response. These results indicate that syncytia induced in the resistant line are undergoing severe oxidative stress and imbalanced endoplasmic reticulum homeostasis, both of which likely contribute to the <span class="trait">resistance</span> reaction. Defense-related genes up-regulated within syncytia of the resistant line included those predominantly involved in apoptotic cell death, the plant hypersensitive response, and salicylic acid-mediated defense signaling; many of these genes were either partially suppressed or not induced to the same level by a virulent soybean cyst nematode population for successful nematode reproduction and development on the resistant line. Our study demonstrates that a network of molecular events take place during Rhg1-mediated <span class="trait">resistance,</span> leading to a highly complex defense response against a root pathogen.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differences in gene expression amplitude overlie a conserved transcriptomic program occurring between the rapid and potent localized resistant reaction at the syncytium of the Glycine max genotype Peking (PI 548402) as compared to the prolonged and potent resistant reaction of PI 88788.</h2>
			<h3>PMID:21153862  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Klink VP(1), Hosseini P, Matsye PD, Alkharouf NW, Matthews BF.
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine max L. Merr. (soybean) <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe occurs at the site of infection, a nurse cell known as the syncytium. <span class="trait">Resistance</span> is classified into two cytologically-defined responses, the G. max ([Peking])- and G. max ([PI 88788])-types. Each type represents a cohort of G. max genotypes. <span class="trait">Resistance</span> in G. max ([Peking]) occurs by a potent and rapid localized response, affecting parasitic second stage juveniles (p-J2). In contrast, <span class="trait">resistance</span> occurs by a potent but more prolonged reaction in the genotype G. max ([PI 88788]) that affects nematode development at the J3 and J4 stages. Microarray analyses comparing these cytologically and developmentally distinct resistant reactions reveal differences in gene expression in pericycle and surrounding cells even before infection. The differences include higher relative levels of the differentially expressed in response to arachidonic acid 1 gene (DEA1 <span class="gene">[Gm-DEA1])</span> (+224.19-fold) and a protease inhibitor (+68.28-fold) in G. max ([Peking/PI 548402]) as compared to G. max ([PI 88788]). Gene pathway analyses compare the two genotypes (1) before, (2) at various times during, (3) constitutively throughout the resistant reaction and (4) at all time points prior to and during the resistant reaction. The amplified levels of transcriptional activity of defense genes may explain the rapid and potent reaction in G. max ([Peking/PI 548402]) as compared to G. max ([PI 88788]). In contrast, the shared differential expression levels of genes in G. max ([Peking/PI 548402]) and G. max ([PI 88788]) may indicate a conserved genomic program underlying the G. max <span class="trait">resistance</span> on which the genotype-specific gene expression programs are built off.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A nematode demographics assay in transgenic roots reveals no significant impacts of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-Kinase on soybean cyst nematode <span class="trait">resistance</span></h2>
			<h3>PMID:20529370</h3>
				

				
					<h3>PMCID:3095272</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Melito </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> melito@wisc.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Heuberger </surname>  <given-names> Adam L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> adam.heuberger@colostate.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Cook </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dcook3@wisc.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Diers </surname>  <given-names> Brian W </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> bdiers@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> MacGuidwin </surname>  <given-names> Ann E </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> aemacgui@wisc.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Bent </surname>  <given-names> Andrew F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> afbent@wisc.edu </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 6 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines,</span> SCN) is the most economically damaging pathogen of soybean (Glycine max) in the U.S. The <span class="gene">Rhg1</span> locus is repeatedly observed as the quantitative trait locus with the greatest impact on SCN <span class="trait">resistance.</span> The <span class="gene">Glyma18g02680.1</span> gene at the <span class="gene">Rhg1</span> locus that encodes an apparent leucine-rich repeat transmembrane receptor-kinase (LRR-kinase) has been proposed to be the SCN <span class="trait">resistance</span> gene, but its function has not been confirmed. Generation of fertile transgenic soybean lines is difficult but methods have been published that test SCN <span class="trait">resistance</span> in transgenic roots generated with Agrobacterium rhizogenes. RESULTS: We report use of artificial microRNA (amiRNA) for gene silencing in soybean, refinements to transgenic root SCN <span class="trait">resistance</span> assays, and functional tests of the <span class="gene">Rhg1</span> locus LRR-kinase gene. A nematode demographics assay monitored infecting nematode populations for their progress through developmental stages two weeks after inoculation, as a metric for SCN <span class="trait">resistance.</span> Significant differences were observed between resistant and susceptible control genotypes. Introduction of the <span class="gene">Rhg1</span> locus LRR-kinase gene (genomic promoter/coding region/terminator; Peking/PI 437654-derived SCN-resistant source), into rhg1- SCN-susceptible plant lines carrying the resistant-source Rhg4+ locus, provided no significant increases in SCN <span class="trait">resistance.</span> Use of amiRNA to reduce expression of the LRR-kinase gene from the <span class="gene">Rhg1</span> locus of Fayette (PI 88788 source of Rhg1) also did not detectably alter <span class="trait">resistance</span> to SCN. However, silencing of the LRR-kinase gene did have impacts on root development. CONCLUSION: The nematode demographics assay can expedite testing of transgenic roots for SCN <span class="trait">resistance.</span> amiRNAs and the pSM103 vector that drives interchangeable amiRNA constructs through a soybean polyubiqutin promoter <span class="gene">(Gmubi),</span> with an intron-GFP marker for detection of transgenic roots, may have widespread use in legume biology. Studies in which expression of the <span class="gene">Rhg1</span> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> , SCN) is the most economically damaging pathogen of soybean ( <italic> Glycine max </italic> ) in the U.S. The <italic> <span class="gene">Rhg1</span> </italic> locus is repeatedly observed as the quantitative trait locus with the greatest impact on SCN <span class="trait">resistance.</span> The <span class="gene">Glyma18g02680.1</span> gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor-kinase (LRR-kinase) has been proposed to be the SCN <span class="trait">resistance</span> gene, but its function has not been confirmed. Generation of fertile transgenic soybean lines is difficult but methods have been published that test SCN <span class="trait">resistance</span> in transgenic roots generated with <italic> Agrobacterium rhizogenes </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						We report use of artificial microRNA (amiRNA) for gene silencing in soybean, refinements to transgenic root SCN <span class="trait">resistance</span> assays, and functional tests of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene. A nematode demographics assay monitored infecting nematode populations for their progress through developmental stages two weeks after inoculation, as a metric for SCN <span class="trait">resistance.</span> Significant differences were observed between resistant and susceptible control genotypes. Introduction of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene (genomic promoter/coding region/terminator; Peking/PI 437654-derived SCN-resistant source), into <italic> rhg1 </italic> <sup> - </sup> SCN-susceptible plant lines carrying the resistant-source <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> locus, provided no significant increases in SCN <span class="trait">resistance.</span> Use of amiRNA to reduce expression of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus of Fayette (PI 88788 source of <italic> <span class="gene">Rhg1</span> </italic> ) also did not detectably alter <span class="trait">resistance</span> to SCN. However, silencing of the LRR-kinase gene did have impacts on root development.
					</p>
				
					<p>&nbsp;&nbsp;
						The nematode demographics assay can expedite testing of transgenic roots for SCN <span class="trait">resistance.</span> amiRNAs and the pSM103 vector that drives interchangeable amiRNA constructs through a soybean polyubiqutin promoter <span class="gene">(Gmubi),</span> with an intron-GFP marker for detection of transgenic roots, may have widespread use in legume biology. Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been shown to have the greatest impact on SCN development in several <span class="trait">resistance</span> sources including Peking, PI 88788, PI 437654, PI 209332 and PI 90763. This locus provides <span class="trait">resistance</span> to many common SCN populations such as Hg type 0 (race 3) [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Multiple research groups have mapped the <italic> <span class="gene">Rhg1</span> </italic> locus to a sub-telomeric region on chromosome 18 [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> ], to a location approximately 0.4 centimorgans (cM) from the simple sequence repeat (SSR) marker Satt309 [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] (chromosome 18 was formerly known as linkage group G; <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> http://www.phytozome.net/ </ext-link> [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]). Although originally reported as a recessive locus, " <italic> rhg1" </italic> has more recently been characterized as exhibiting incomplete dominance. Soybean lines heterozygous for resistant and susceptible alleles at the <italic> <span class="gene">Rhg1</span> </italic> locus often allow SCN cyst formation at a rate intermediate between that of plants genotyped with Satt309 as homozygous resistant or homozygous susceptible at the <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref ref-type="bibr" rid="B18"> 18 </xref> - <xref ref-type="bibr" rid="B20"> 20 </xref> ](Kim et al. submitted; incomplete dominance also has been observed in unpublished work with the Ina &#x000d7; E98076 material used in this study). A second QTL ( <italic> <span class="gene">Rhg4</span> </italic> ) has been identified as being necessary for full <span class="trait">resistance</span> to some SCN populations in Peking and in PI 437654, but not PI 88788 or PI 209332 [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. <italic> <span class="gene">Rhg4</span> </italic> exhibits dominant gene action, and in Peking-derived material the relevant alleles of both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> are necessary to exhibit the full <span class="trait">resistance</span> phenotype. Other loci that make smaller and/or more race-specific contributions to SCN <span class="trait">resistance</span> have also been identified throughout the soybean genome, but often a given locus was identified in only one study [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Cytological studies suggest that Peking-type <span class="trait">resistance</span> displays host cell necrosis and cell wall appositions not seen in PI 88788 type resistances in response to SCN Hg type 0 [ <xref ref-type="bibr" rid="B26"> 26 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> ]. The Peking and PI 88788 <italic> <span class="gene">Rhg1</span> </italic> sources also exhibit distinct differential behaviors in their strength of <span class="trait">resistance</span> against particular SCN test populations, suggesting at least partially different mechanisms in the SCN <span class="trait">resistance</span> controlled by different <italic> <span class="gene">Rhg1</span> </italic> alleles [ <xref ref-type="bibr" rid="B28"> 28 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Two groups first filed applications with the U.S. Patent Office in 2000 identifying apparent leucine-rich repeat transmembrane receptor-kinase (LRR-kinase) genes currently annotated as <span class="gene">Glyma18g02680.1</span> and <span class="gene">Glyma08g11350.1</span> <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> http://www.phytozome.net/ </ext-link> [ <xref ref-type="bibr" rid="B17"> 17 </xref> ] as the likely SCN <span class="trait">resistance</span> genes at both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ], see also [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ]. Sequences supporting the claims were released to Genbank between 2000 and 2005. The basis for these claims was the presence of these genes at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, their similarity to the known rice bacterial blight <span class="trait">resistance</span> gene <italic> Xa21 </italic> , and the presence of derived amino acid sequence differences between the alleles from resistant and susceptible plant genotypes. However, a decade later, no functional evidence for a role of these LRR-kinase genes in SCN <span class="trait">resistance</span> has been reported in a peer-reviewed forum. There are numerous reasons why identification of the <italic> <span class="gene">Rhg1</span> </italic> gene that confers SCN <span class="trait">resistance</span> is a high priority, including the extreme economic significance of SCN for <span class="trait">yield</span> loss, the major reliance on the <italic> <span class="gene">Rhg1</span> </italic> locus in commercial soybean breeding, the detection of SCN populations that overcome currently available <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> the potential to respond to these challenges with engineered improvements in <italic> <span class="gene">Rhg1</span> </italic> , and basic scientific interest in the nature of plant <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						For the present study we refined assays that test SCN <span class="trait">resistance</span> in transgenic roots generated with <italic> A. rhizogenes </italic> . A nematode demographic assay was developed that discriminates resistant and susceptible responses by monitoring the infecting population for progression through nematode life stages. We also developed an amiRNA vector system for induction of gene silencing in legume roots using <italic> A. rhizogenes </italic> assay systems. We used these tools to investigate the impact on SCN <span class="trait">resistance</span> of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> genomic region (the candidate SCN <span class="trait">resistance</span> gene). Our experiments expressing the LRR-kinase from a resistant (Peking/PI 437654-source) <italic> <span class="gene">Rhg1</span> </italic> locus in susceptible test lines, with or without the resistant allele at <italic> <span class="gene">Rhg4</span> </italic> , and silencing the LRR-kinase (PI 88788 source) in resistant lines, provided no evidence for a contribution of this gene to SCN <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographics assay for SCN <span class="trait">resistance</span> in roots transformed with a full-length <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-Kinase gene </bold> . <bold> A, B, C </bold> , Representative examples of SCN developmental stages on transgenic soybean hairy roots 15 days after infection; cleared root sections were stained with acid fuchsin, which stains nematodes as well as root vascular bundles. <bold> A </bold> , Early parasitic second-stage juveniles (J2). <bold> B </bold> , Late parasitic J2s and early J3s. <bold> C </bold> , J4 or adult female. <bold> D </bold> , Ratio of total infecting nematode population that developed to J4 or adult female stage two weeks after inoculation onto <italic> A. rhizogenens </italic> -transformed roots of Ina, E98076 (both transformed with empty vector), or E98076 + LRR-K Ina (transformed with the <span class="gene">Glyma18g02680.1</span> <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-Kinase promoter/gene/terminator). Mean &#x000b1; std error of mean; data combined from two independent experiments; bars with the same letter above are not significantly different (ANOVA, Tukey pairwise comparison, p &#x0003c; 0.001). <bold> E </bold> , Similar to D, except that " <italic> rhg1 </italic> <sup> - </sup> <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> are putative <italic> rhg1 </italic> <sup> - </sup> <bold> / </bold> <italic> rhg1 </italic> <sup> - </sup> ; <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> <bold> / </bold> <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> progeny lines from Ina &#x000d7; E98076 that were genotyped as carrying the E98076 (susceptible) allele of a marker linked to <italic> <span class="gene">Rhg1</span> </italic> and the Ina (resistant) allele of a marker linked to <italic> <span class="gene">Rhg4</span> </italic> . All the lines were transformed with empty vector except " <italic> rhg1 </italic> <sup> - </sup> <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> + LRR-K Ina" was transformed with <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-kinase gene (including native promoter and terminator). Mean &#x000b1; std error of mean are from two independent experiments; bars with the same letter above are not significantly different (ANOVA, Tukey pairwise comparison, p &#x0003c; 0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus of soybean has been shown in numerous studies to be the QTL with the greatest phenotypic impact on SCN <span class="trait">resistance,</span> and previous patents and other reports have indicated but not demonstrated that the <span class="gene">Glyma18g02680.1</span> gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor kinase (LRR-K) is the source of this SCN <span class="trait">resistance.</span> We isolated this LRR-K gene from the <italic> <span class="gene">Rhg1</span> </italic> locus using genomic DNA from the soybean cultivar Ina that we confirmed to exhibit SCN <span class="trait">resistance</span> in a standard whole plant/cyst formation assay immediately prior to its use in DNA isolation. Ina was used because of available progeny lines that are described below; Ina has Peking, PI 437654 and PI 88788 in its pedigree but carries an LRR-K at the <italic> <span class="gene">Rhg1</span> </italic> locus that is identical in sequence to the Peking allele and also similar to the PI437654 allele. Multiple independent PCR products were isolated and sequenced, and the sequences were compared to each other and to the Peking sequence published in [ <xref ref-type="bibr" rid="B30"> 30 </xref> ] to identify the best clone. An <italic> Agrobacterium </italic> -compatible binary vector pGWB1::erGFP7INT carrying a screenable intron-GFP marker was constructed to transform soybean with candidate SCN <span class="trait">resistance</span> genes in full-gene complementation experiments. An insert of 6.9 kb was added to pGWB1::erGFP7INT and included approximately 2 kb of genomic upstream DNA, the predicted 2757 bp LRR-K coding region + intron, and approximately 800 bp of downstream DNA. SCN-susceptible E98076 was then transformed with this construct to test the impact of this LRR-K allele on SCN development. Using the nematode demographics assay described above, roots from soybean line E98076 transformed with the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K did not display increased <span class="trait">resistance</span> to SCN; nematode development progressed similarly to what was observed on E98076 susceptible control roots transformed with the empty pGWB1::erGFP7INT vector (Figure <xref ref-type="fig" rid="F1"> 1D </xref> ). E98076 transformed with either an empty plasmid vector or with the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K were, however, both significantly more susceptible to SCN than the resistant line Ina (p &#x0003c; 0.001).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus in Peking and PI 437654 also contributes to the full SCN <span class="trait">resistance</span> phenotype [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. SCN-susceptible soybean genotypes with endogenous SCN-susceptible alleles at both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which are then transformed with a resistant allele only of <italic> <span class="gene">Rhg1</span> </italic> , might not display a resistant phenotype. To further test the impact of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene on SCN development, <italic> rhg1 </italic> <sup> - </sup> lines carrying the SCN-resistant Peking <italic> <span class="gene">Rhg4</span> </italic> locus were tested by complementation. Microsatellite markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were used to identify Ina &#x000d7; E98076 progeny lines with a (-/-, +/+) genotype ( <italic> rhg1 </italic> <sub> E98076 </sub> <italic> /rhg1 </italic> <sub> E98076 </sub> , <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> / <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> ). These lines were transformed with the cloned Ina <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene and assayed for SCN development. In the combined results from two independent experiments, no significant change in the <span class="trait">resistance</span> to SCN was detected after transformation of these genotypes with the Ina (Peking)-derived <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene (Figure <xref ref-type="fig" rid="F1"> 1E </xref> ). The two <italic> rhg1 </italic> <sup> - </sup> lines transformed with an empty vector were significantly more susceptible to SCN than resistant Ina (p &#x0003c; 0.01), while the <italic> rhg1 </italic> <sub> E98076 </sub> <italic> /rhg1 </italic> <sub> E98076 </sub> , <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> / <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> genotype transformed with the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene was not significantly more resistant than the same genotype transformed with empty vector. The mean <span class="trait">resistance</span> of roots transformed with the Ina LRR-K gene may have been slightly greater but not to a significant extent, in contrast to the statistically significant difference between the Ina and E98076 control lines. When the nematode demographics data for these same experiments were expressed as [(J3+J4)/total], the results for <italic> rhg1 </italic> <sub> E98076 </sub> <italic> /rhg1 </italic> <sub> E98076 </sub> , <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> / <italic> <span class="gene">Rhg4</span> </italic> <sub> Ina </sub> genotypes transformed with empty vector or vector + <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene were even more similar (mean &#x000b1; std error 0.59 &#x000b1; 0.09 for empty vector and 0.49 &#x000b1; 0.09 for LRR-K Ina; P = 0.85 for ANOVA Tukey simultaneous tests).
					</p>
				
					<p>&nbsp;&nbsp;
						We sought to use gene silencing as an independent means to test for contributions of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene toward SCN <span class="trait">resistance.</span> To facilitate this work, we constructed a new vector for gene silencing in transgenic soybean roots. The binary vector pSM103 (pCAMBIA1300::35SP-erGFP7INT-NOS, <span class="gene">Gmubi</span> promoter, mi319RNAa) was designed to meet the following requirements: 1) efficient cloning of artificial microRNAs, 2) constitutive strong amiRNA expression in legume roots as well as other tissues [ <xref ref-type="bibr" rid="B54"> 54 </xref> ], 3) reliable identification of transgenic roots by GFP screening using an intron-containing GFP that is not expressed in bacteria, 4) competence for transferring inserted DNA into plant genomes via <italic> Agrobacterium </italic> , and 5) high transformation efficiency. The steps taken to develop the vector are shown in Figure <xref ref-type="fig" rid="F2"> 2 </xref> , as the intermediate constructs may also be of use and are available upon request. To initially confirm GFP7 expression <italic> in planta </italic> , pSM101 and pSM103 were infiltrated into <italic> N. bentamiana </italic> leaves using <italic> A. tumefaciens </italic> <italic> GV3101 </italic> . Positive and negative controls (pGWB1::35SP-erGFP7INT-NOS, and <italic> A. tumefaciens GV3101 </italic> without any vector) did or did not produce green fluorescence respectively, as expected (Figure <xref ref-type="fig" rid="F3"> 3A </xref> and <xref ref-type="fig" rid="F3"> 3B </xref> ). erGFP7INT expression was phenotypically detected with use of vectors pSM101 and pSM103 (Figure <xref ref-type="fig" rid="F3"> 3C </xref> and <xref ref-type="fig" rid="F3"> 3D </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Healthy transgenic GFP-positive roots expressing amiRNA constructs that target the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene (or empty vector controls) were sub-cultured by propagating the young growing tips (2-3 cm), and were infected with ~250 sterile J2 SCN for each root. The nematode demographics assay was then used to detect relative SCN <span class="trait">resistance.</span> The experiment was repeated with three independent biological replicates each using 4-10 separate transgenic hairy roots of each treatment-construct. Two weeks after nematode inoculation, all nematode stages (except eggs) were observed in the transgenic roots tested, and the number of nematodes at each stage was recorded. Roots carrying five or fewer nematodes were excluded from further analysis. As expected, Fayette control roots allowed significantly fewer nematodes to molt and progress to an apparent J3 or J4/adult female stage within 14 days relative to Williams 82 control roots (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ; p = 0.007). In the key comparison to test if silencing of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene altered <span class="trait">resistance</span> to SCN, no statistical difference in nematode development was observed between Fayette and Fayette carrying amiRNA <span class="gene">GmLRR-K</span> I or amiRNA <span class="gene">Gm-K</span> II (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographic assay for SCN <span class="trait">resistance</span> in roots transformed with amiRNA silencing the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-kinase gene </bold> . Ratio of total infecting nematode population that gained appearance of J3 and J4/adult female stages two weeks after inoculation onto <italic> A. rhizogenes </italic> -transformed roots. Williams 82 (susceptible) and Fayette (resistant) soybean tissue (left two bars) were transformed using <italic> A. rhizogenes </italic> carrying the pSM103 negative control plasmid; Fayette + amiRNA received the designated amiRNAs that target the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-kinase gene (amiRNA LRR-K I and amiRNA LRR-K II). Graph shows mean &#x000b1; std error of mean from three independent nematode infection assays that used roots from three independent transformation experiments (overall n &#x0003e; 18 for each treatment). P values above each <span class="gene">bar</span> are color-coded for contrasts with the <span class="gene">bar</span> of the designated color (null hypothesis: the observed ratio between the two treatments was similar; tested by ANOVA using SAS Proc Mixed).
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to reporting amiRNA use for gene silencing in soybean, a vector to facilitate this approach, and the nematode demographics assay for tests of genes that may contribute to soybean SCN <span class="trait">resistance,</span> we also report negative data regarding impacts of the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene on SCN <span class="trait">resistance.</span> Using complementation with a full-length gene under control of its native promoter, and gene knockdown using amiRNA technology, we were unable to detect a contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene from either Peking/PI 437654 or PI 88788, the most commonly used sources of SCN <span class="trait">resistance</span> in U.S. soybean cultivars. An impact on root development was observed when amiRNA for the LRR-K gene were expressed in transgenic roots, and effects on root development have also been preliminarily reported by the Lightfoot research group <ext-link ext-link-type="uri" xlink:href="http://hdl.handle.net/10101/npre.2008.2726.1"> http://hdl.handle.net/10101/npre.2008.2726.1 </ext-link> . It remains possible that this <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene may in some way impact SCN infection of or development on soybean roots, but no such effect was detected in our studies.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus is of extremely high economic significance to soybean cultivation in the United States, and has received attention from multiple research groups. It is notable that more than ten years after the filing of patent applications claiming SCN <span class="trait">resistance</span> function for the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene, no groups have published evidence (beyond its presence as one of many candidate genes at the <italic> <span class="gene">Rhg1</span> </italic> locus) for a functional contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene. We note that, simultaneous with the present study, a group led by Drs. Khalid Meksem and Melissa Mitchum has completed tests for functional contributions of the <italic> <span class="gene">Rhg4</span> </italic> -locus LRR-K gene to SCN <span class="trait">resistance</span> and, similar to our work, has not detected a contribution for this <italic> <span class="gene">Rhg4</span> </italic> candidate gene (K. Meksem, personal communication). We are also aware of work led by Dr. Brian Diers (Kim et al., submitted)[ <xref ref-type="bibr" rid="B63"> 63 </xref> ], in which fine-structure genetic mapping of the <italic> <span class="gene">Rhg1</span> </italic> locus has revealed chromosomal recombination events that genetically separate <italic> rhg1-b </italic> (PI 88788-source) SCN <span class="trait">resistance</span> function from the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene. Hence evidence is accumulating that the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K genes seem unlikely to be the primary source of SCN <span class="trait">resistance</span> polymorphism. A tone of caution is appropriate, however, as our present study only provides negative data on this subject. For example, gene silencing is often partial (whether mediated by VIGS, hairpin RNA-generating constucts or amiRNA), and the expression knockdown we achieved with amiRNA may have been insufficient to detectably block hypothesized SCN <span class="trait">resistance</span> contributions (although the other phenotypic impacts we observed with the amiRNA LRR-K I suggest that the gene expression knockdown achieved was functionally significant). As another example, the quantitative nature of the functional contribution of any given gene to SCN <span class="trait">resistance</span> may be difficult to detect in <italic> A. rhizogenes </italic> -generated soybean roots. Hence we are cautious with our conclusion, but we can state that when SCN-susceptible sources were complemented with an <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K allele from an SCN-resistant source, or when expression of the gene in resistant plants was reduced by amiRNA-mediated silencing, no significant impacts on SCN <span class="trait">resistance</span> were detected.
					</p>
				
					<p>&nbsp;&nbsp;
						Our findings suggest that the nematode demographics assay can expedite testing of transgenic roots for SCN <span class="trait">resistance.</span> amiRNAs may have widespread use in legume biology, and we have developed the vector pSM103 that drives interchangeable amiRNA constructs through a <span class="gene">Gmubi</span> promoter, with an intron-GFP marker for detection of transgenic roots. Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance,</span> suggesting that one or more other genes at the <italic> <span class="gene">Rhg1</span> </italic> locus may be the primary determinant of differences in SCN <span class="trait">resistance</span> among different soybean types.
					</p>
				
					<p>&nbsp;&nbsp;
						EST TC204550 UP|Q8L3Y5 (Q8L3Y5) (Glycine max EST <span class="gene">GmGI-12.0;</span> wmd2 database <ext-link ext-link-type="uri" xlink:href="http://wmd2.weigelworld.org/cgi-bin/mirnatools.pl"> http://wmd2.weigelworld.org/cgi-bin/mirnatools.pl </ext-link> ) corresponding to the <span class="gene">Glyma18g02680.1</span> putative LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus was used to design PCR primers for isolation of the <italic> <span class="gene">Rhg1</span> </italic> locus receptor-like kinase-encoding allele from the resistant source Fayette, which carries the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele <italic> rhg1-b </italic> from PI 88788. The primers RTPCR_TC204550F CAA CTT CAA GGC CTC AGG AA; RTPCR_TC204550R GCT ACC CAA AGA AGC AGG AA, amplified a PCR product of 450 bp. The DNA sequences of PCR products were determined by the dideoxy chain-termination method using ABI big dye cycle sequencing kit (3.1) and the ABI 377 automated sequencing service at the University of Wisconsin - Madison Biotechnology Center. Blast analysis of the Fayette version of TC204550 (the EST for this gene in earlier WMD versions) was conducted in March 2008 using the Phytozome database <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/soybean"> http://www.phytozome.net/soybean </ext-link> . For amiRNA design, the sequence of this Fayette LRR-kinase gene was submitted to <ext-link ext-link-type="uri" xlink:href="http://wmd2.weigelworld.org"> http://wmd2.weigelworld.org </ext-link> . The wmd2 tool identified a total of 85 candidate amiRNA sequences to potentially silence the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene, with the first 31 candidates classified by wmd2 as having excellent (most promising) properties (data not shown). The two amiRNA target sequences selected for primer design to modify pRS300 for silencing of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene were identical to Fayette chromosome 18 sequence as well as the paralogous chromosome 11 sequence from the Williams 82 genome: amiRNA <span class="gene">GmLRR-KI:</span> TAAGACTATAAGGGATTGCTG; and amiRNA LRR-KII: TAAGACTATAAGGGATTGCTC. The Arabidopsis miR319a sequence from pRS300 was used as a template to create three PCR products (a, b, c) with soybean target transcript specificity. These products were used as template for a fourth PCR using overlapping primers to create gene specific amiRNA [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. The final PCR product (containing target transcript sequence) was cloned into pSM103 using BamHI-PstI restriction sites, replacing the miR319a sequence. Presence of the desired amiRNA was confirmed by DNA sequencing.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:1-Aminocyclopropane-1-carboxylic acid (ACC) concentration and ACC synthase expression in soybean roots, root tips, and soybean cyst nematode <span class="trait">(Heterodera</span> glycines)-infected roots.</h2>
			<h3>PMID:19861652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Xue P, Yang R.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Colonization of plant roots by root knot and cyst nematodes requires a functional ethylene response pathway. However, ethylene plays many roles in root development and whether its role in nematode colonization is direct or indirect, for example lateral root initiation or root hair growth, is not known. The temporal requirement for ethylene and localized synthesis of ethylene during the life span of soybean cyst nematode (SCN) on soybean roots was further investigated. Although a significant increase in ethylene evolution was not detected from SCN-colonized roots, the concentration of the immediate precursor to ethylene, 1-aminocyclopropane-1-carboxylic acid (ACC), was higher in SCN-colonized root pieces and root tips than in other parts of the root. Moreover, expression analysis of 17 ACC synthase (ACS) genes indicated that a select set of <span class="gene">ACS</span> genes is expressed in SCN-colonized root pieces that is clearly different from the set of genes expressed in non-colonized roots or root tips. Semi-quantitative real-time PCR indicated that <span class="gene">ACS</span> transcript accumulation correlates with the high concentration of ACC in root tips. In addition, an ACS-like sequence was found in the public SCN nucleotide database. Acquisition of a full-length sequence for this mRNA (accession GQ389647) and alignment with transcripts for other well-characterized <span class="gene">ACS</span> proteins indicated that the nematode sequence is missing a key element required for <span class="gene">ACS</span> activity and therefore probably is not a functional ACS. Moreover, no significant amount of ACC was found in any growth stage of SCN that was tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cellulose binding protein from the parasitic nematode <span class="trait">Heterodera</span> schachtii interacts with Arabidopsis pectin methylesterase: cooperative cell wall modification during parasitism.</h2>
			<h3>PMID:19001564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hewezi T(1), Howe P, Maier TR, Hussey RS, Mitchum MG, Davis EL, Baum TJ.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-parasitic cyst nematodes secrete a complex of cell wall-digesting enzymes, which aid in root penetration and migration. The soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> also produces a cellulose binding protein (Hg CBP) secretory protein. To determine the function of CBP, an orthologous cDNA clone (Hs CBP) was isolated from the sugar beet cyst nematode <span class="trait">Heterodera</span> schachtii, which is able to infect Arabidopsis thaliana. <span class="gene">CBP</span> is expressed only in the early phases of feeding cell formation and not during the migratory phase. Transgenic Arabidopsis expressing Hs <span class="gene">CBP</span> developed longer roots and exhibited enhanced susceptibility to H. schachtii. A yeast two-hybrid screen identified Arabidopsis pectin methylesterase protein 3 (PME3) as strongly and specifically interacting with Hs CBP. Transgenic plants overexpressing PME3 also produced longer roots and exhibited increased susceptibility to H. schachtii, while a pme3 knockout mutant showed opposite phenotypes. Moreover, <span class="gene">CBP</span> overexpression increases PME3 activity in planta. Localization studies support the mode of action of PME3 as a cell wall-modifying enzyme. Expression of <span class="gene">CBP</span> in the pme3 knockout mutant revealed that PME3 is required but not the sole mechanism for <span class="gene">CBP</span> overexpression phenotype. These data indicate that <span class="gene">CBP</span> directly interacts with PME3 thereby activating and potentially targeting this enzyme to aid cyst nematode parasitism.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of a horizontally transferred pathway involved in vitamin B6 biosynthesis from the soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:18586696  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Craig JP(1), Bekal S, Hudson M, Domier L, Niblack T, Lambert KN.
				<br/> 
				年份：2008 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Heterodera</span> <span class="trait">glycines</span> is an obligate plant parasite capable of biochemically and developmentally altering its host's cells in order to create a specialized feeding cell. Although the exact mechanism of feeding cell morphogenesis remains a mystery, the nematode's ability to manipulate the plant is thought to be due in part to horizontal gene transfers (HGTs). A bioinformatic screen of the nematode genome has revealed homologues of the genes <span class="gene">SNZ</span> and SNO, which comprise a metabolic pathway for the de novo biosynthesis of pyridoxal 5'-phosphate, the active form of vitamin B(6) (VB(6)). Analysis of the 2 genes, HgSNZ and HgSNO, show that they contain nematode-like introns, generate polyadenylated mRNAs, and map to the soybean cyst nematode genetic linkage map, indicating that they are part of the nematode genome. However, gene synteny, protein homology, and phylogenetic evidence suggest prokaryotic origin. This would represent the first case of the HGT of a complete pathway into a nematode or terrestrial animal. VB(6) acts as a cofactor in over 140 different enzymes, and recent studies point toward an important role as a potent quencher of reactive oxygen species. With H. <span class="trait">glycines'</span> penchant for acquiring parasitism genes through HGT along with the absence of this pathway in other land-based animals suggests a specific need for VB(6) which may involve the parasite-host interaction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterisation by RNAi of pioneer genes expressed in the dorsal pharyngeal gland cell of <span class="trait">Heterodera</span> <span class="trait">glycines</span> and the effects of combinatorial RNAi.</h2>
			<h3>PMID:18579145  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bakhetia M(1), Urwin PE, Atkinson HJ.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Changes in transcript abundance of 24 genes expressed in the dorsal pharyngeal gland cell of <span class="trait">Heterodera</span> <span class="trait">glycines</span> encoding for putative secretions of unknown function were monitored by quantitative PCR (qPCR) at 0, 2, 7, 14 and 21 days post-invasion (pi) of soybean plantlets. Five groups of temporal patterns (A, B1, B2, C and D) were defined for the 24 genes plus data for two previously studied genes expressed in the same cell. Group D (two genes) showed no significant increase between 0 and 2 days <span class="gene">pi</span> and were the least abundantly expressed at 7-21 days pi. Transcripts of group C (nine genes including one studied previously) increased in abundance from 0 to 2 days <span class="gene">pi</span> but were the second least expressed for 7-21 days pi. Groups A (three genes), B1 (seven genes) and B2 (five genes including one studied previously) were all abundant at 7-21 days pi. B1 and B2 were discriminated by their relative abundance at 0 and 2 days pi. RNA interference (RNAi) targeting two genes of group A and one each of B1 and B2 in nematodes prior to invasion resulted in phenotypic effects on total parasites per plant and sexual fate at 10 days pi. Phenotype penetrance was reduced for three genes showing such effects and one with a strong effect in earlier work when two genes rather than one were concurrently targeted for RNAi. One gene (dg13) was more abundantly expressed after combinatorial RNAi than for either control nematodes or when targeting singly by RNAi. This work reports the unexpected elevation in mRNA expression after combinatorial RNAi that requires understanding before combinatorial RNAi can be advanced for highly effective cyst nematode control via plant biotechnology.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Separate loci underlie <span class="trait">resistance</span> to root infection and leaf scorch during soybean sudden death syndrome.</h2>
			<h3>PMID:18324383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kazi S(1), Shultz J, Afzal J, Johnson J, Njiti VN, Lightfoot DA.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] cultivars show differences in their <span class="trait">resistance</span> to both the leaf scorch and root rot of sudden death syndrome (SDS). The syndrome is caused by root colonization by Fusarium virguliforme (ex. F. solani f. sp. <span class="trait">glycines).</span> Root susceptibility combined with reduced leaf scorch <span class="trait">resistance</span> has been associated with <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> HG Type 1.3.6.7 (race 14) of the soybean cyst nematode (SCN). In contrast, the rhg1 locus underlying <span class="trait">resistance</span> to Hg Type 0 was found clustered with three loci for <span class="trait">resistance</span> to SDS leaf scorch and one for root infection. The aims of this study were to compare the inheritance of <span class="trait">resistance</span> to leaf scorch and root infection in a population that segregated for <span class="trait">resistance</span> to SCN and to identify the underlying quantitative trait loci (QTL). "Hartwig", a cultivar partially resistant to SDS leaf scorch, F. virguliforme root infection and SCN HG Type 1.3.6.7 was crossed with the partially susceptible cultivar "Flyer". Ninety-two F5-derived recombinant inbred lines and 144 markers were used for map development. Four QTL found in earlier studies were confirmed. One contributed <span class="trait">resistance</span> to leaf scorch on linkage group (LG) <span class="gene">C2</span> (Satt277; P = 0.004, R2 = 15%). Two on LG G underlay root infection at R8 (Satt038; P = 0.0001 R2 = 28.1%; Satt115; P = 0.003, R2 = 12.9%). The marker Satt038 was linked to rhg1 underlying <span class="trait">resistance</span> to SCN Hg Type 0. The fourth QTL was on LG D2 underlying <span class="trait">resistance</span> to root infection at R6 (Satt574; P = 0.001, R2 = 10%). That QTL was in an interval previously associated with <span class="trait">resistance</span> to both SDS leaf scorch and SCN Hg Type 1.3.6.7. The QTL showed repulsion linkage with <span class="trait">resistance</span> to SCN that may explain the relative susceptibility to SDS of some SCN resistant cultivars. One additional QTL was discovered on LG G underlying <span class="trait">resistance</span> to SDS leaf scorch measured by disease index (Satt130; P = 0.003, R2 = 13%). The loci and markers will provide tagged alleles with which to improve the breeding of cultivars combining resistances to SDS leaf scorch, root infection and SCN HG Type 1.3.6.7.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene expression profiles for cell wall-modifying proteins associated with soybean cyst nematode infection, petiole abscission, root tips, flowers, apical buds, and leaves.</h2>
			<h3>PMID:17916637  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Burke A, Murphy CA, Thai VK, Ehrenfried ML.
				<br/> 
				年份：2007
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Changes in transcript accumulation for cell wall-modifying proteins were examined in excised soybean root pieces colonized by soybean cyst nematodes (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines,</span> using RT-PCR and soybean Affymetrix GeneChips. Sequence-specific PCR primer pairs were prepared from sequence data for core sequences in the GenBank soybean database and consensus sequences derived from the assembly of soybean ESTs. In addition, to identify previously uncharacterized soybean transcripts, degenerate primers were prepared for conserved motifs in cellulases (endo-1,4-beta-glucanases, EGases) and polygalacturonases (PGs) and these were used to amplify segments of transcripts that were then extended with 3' and 5' RACE. Several novel EGase and PG transcripts were identified. Gene expression patterns were determined by real-time RT-PCR for 11 EGases, three expansins (EXPs), 14 PGs, two pectate lyases (PLs), and two xyloglucan endotransglucosylase/hydrolases (XTHs) in soybean roots inoculated with SCN, non-inoculated roots, serial dissections of root tips, leaf abscission zones, flowers, apical buds, and expanding leaves. A large number of genes associated with cell wall modifications are strongly up-regulated in root pieces colonized by SCN. However, in contrast to most of the transcripts for cell wall proteins, two <span class="gene">XTH</span> transcripts were specifically down-regulated in the colonized root pieces. Gene expression in serial dissections of root tips (0-2 mm, and 2-7 mm) and whole roots indicate that the SCN up-regulated genes are associated with a wide range of developmental processes in roots. Also of interest, many of the cDNAs examined were up-regulated in petiole abscission zones induced to abscise with ethylene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:'Candidatus Paenicardinium endonii', an endosymbiont of the plant-parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> (Nemata: Tylenchida), affiliated to the phylum Bacteroidetes.</h2>
			<h3>PMID:16825653  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Noel GR(1), Atibalentja N.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacteria-like endosymbionts of females of the plant-parasitic nematodes Globodera rostochiensis and <span class="trait">Heterodera</span> goettingiana and juveniles of <span class="trait">Heterodera</span> <span class="trait">glycines</span> were first observed during transmission electron microscopy (TEM) studies conducted in the 1970s. These organisms were characterized as being rod-shaped, ranging in size from 0.3 to 0.5 microm in diameter and 1.8 to 3 microm in length and containing structures labelled as striated inclusion bodies or tubular structures. A population of H. <span class="trait">glycines</span> was obtained from the soybean field where infected nematodes were first discovered in order to conduct TEM studies of females and males and to determine the phylogenetic position of the H. <span class="trait">glycines</span> endosymbiont among bacteria by studying the 16S rRNA and <span class="gene">gyrB</span> gene sequences. The bacterium was observed in the pseudocoelom and intestine of juveniles, females and males, in hypodermal chords of juveniles and males, in ovary walls and in oocytes and spermatozoa. The bacterium was polymorphic, measuring 0.4-0.8 x 2.5-4.5 microm, and many specimens contained an array of microfilament-like structures similar to those observed in "Candidatus Cardinium hertigii", the endosymbiont of Encarsia spp. wasps. Phylogenetic analysis of the 16S rRNA and <span class="gene">gyrB</span> genes of the H. glycines-infecting bacterium revealed 93 % and 81 % sequence identity, respectively, to the homologous genes in "Candidatus C. hertigii". Thus, the name "Candidatus Paenicardinium endonii" is proposed for the bacterial endosymbiont of the plant-parasitic nematode H. <span class="trait">glycines.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Laser capture microdissection (LCM) and expression analyses of Glycine max (soybean) syncytium containing root regions formed by the plant pathogen <span class="trait">Heterodera</span> <span class="trait">glycines</span> (soybean cyst nematode).</h2>
			<h3>PMID:16307369  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Klink VP(1), Alkharouf N, MacDonald M, Matthews B.
				<br/> 
				年份：2005 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Roots of soybean, Glycine max cv. Kent L. Merr., plants susceptible to the soybean cyst nematode (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe, were inoculated and allowed to develop feeding sites (syncytia) for 8 days. Root samples enriched in syncytial cells were collected using laser capture microdissection (LCM). RNA was extracted and used to make a cDNA library and expressed sequence tags (ESTs) were produced and used for a Gene Ontology (GO) analysis. RT-PCR results indicated enhanced expression of an aquaporin <span class="gene">(GmPIP2,2),</span> alpha-tubulin <span class="gene">(GmTubA1),</span> beta-tubulin <span class="gene">(GmTubB4)</span> and several other genes in syncytium-enriched samples as compared to samples extracted from whole roots. While RT-PCR data showed increased transcript levels of <span class="gene">GmPIP2,2</span> from LCM tissue enriched in syncytial cells, in situ hybridization showed prominent <span class="gene">GmPIP2,2</span> hybridization to RNA in the parenchymal cells tightly juxtaposed to the syncytium. Immunolocalization indicated stronger alpha-tubulin signal within the syncytium as compared to surrounding tissue. However, alpha-tubulin labeling appeared diffuse or clumped. Thus, LCM allowed for the isolation of tissue enriched for syncytial cells, providing material suitable for a variety of molecular analyses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">Rhg1</span> alleles from soybean PI 437654 and PI 88788 respond differentially to isolates of <span class="trait">Heterodera</span> <span class="trait">glycines</span> in the greenhouse.</h2>
			<h3>PMID:15883792  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brucker E(1), Carlson S, Wright E, Niblack T, Diers B.
				<br/> 
				年份：2005 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The production of resistant soybean [Glycine max (L.) Merr.] cultivars is the most effective means for controlling losses from soybean cyst nematode (SCN) <span class="trait">(Heterodera</span> <span class="trait">glycines</span> Ichinohe). The major <span class="trait">resistance</span> gene in most SCN <span class="trait">resistance</span> sources is rhg1, which has been mapped as a quantitative trait locus onto linkage group G. Our objective was to determine whether the SCN <span class="trait">resistance</span> sources PI 437654 and PI 88788 have different functional alleles at rhg1 based on <span class="trait">resistance</span> phenotypes. Populations segregating for <span class="trait">resistance</span> alleles at rhg1 from both PI 88788 and PI 437654 and at Rhg4, a second SCN <span class="trait">resistance</span> gene from PI 437654, were developed. These populations were screened for <span class="trait">resistance</span> to the H. <span class="trait">glycines</span> inbred isolates PA3 (HG type 7) and TN14 (HG type 1.2.5.7) in the greenhouse and evaluated with molecular markers linked to both rhg1 and Rhg4. Each isolate test was repeated, and the evaluations were done on a single-plant and a line-mean basis in Test 1, and solely on a single-plant basis in Test 2. Across two tests with the TN14 isolate, plants with the PI 437654 allele for a marker linked to rhg1 had significantly (P&lt;0.0001) less SCN reproduction than plants carrying the PI 88788 allele. A marker linked to Rhg4, however, was not significantly associated with <span class="trait">resistance</span> to TN14. Across two tests with the PA3 isolate, alleles of rhg1 from both sources gave a resistant reaction, although plants homozygous for the PI 88788 allele had significantly (P&lt;0.05) greater <span class="trait">resistance</span> than plants with the PI 437654 allele. The marker allele from PI 437654 linked to <span class="gene">Rhg4</span> was significantly (P&lt;0.0005) associated with greater <span class="trait">resistance</span> than the PI 88788 allele in both PA3 tests, and <span class="trait">resistance</span> was dominant. There was a significant interaction between alleles at rhg1 and <span class="gene">Rhg4</span> in both PA3 tests. These results suggest that PI 437654 and PI 88788 each have a different functional SCN <span class="trait">resistance</span> allele at or close to rhg1. These allelic differences have implications that breeders should consider before incorporation into cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homologous soybean and Arabidopsis genes share responsiveness to cyst nematode infection.</h2>
			<h3>PMID:20565617</h3>
				

				
			<p>
				作者：Mazarei M(1), Lennon KA, Puthoff DP, Rodermel SR, Baum TJ.
				<br/> 
				年份：2004 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SUMMARY We previously isolated a partial soybean cDNA clone (D17.1) whose corresponding transcript increases in susceptible roots 1 day post inoculation (dpi) with the soybean cyst nematode, <span class="trait">Heterodera</span> <span class="trait">glycines.</span> Here we isolated the corresponding full-length cDNA from a soybean cDNA library and designated this gene of unknown function <span class="gene">Gm17.1.</span> Time course RNA gel blot analyses revealed that <span class="gene">Gm17.1</span> mRNA steady-state levels were elevated in soybean roots following H. <span class="trait">glycines</span> infection up to at least 6 dpi. For further in-depth study we identified a homologous Arabidopsis thaliana gene and designated this gene At17.1. Arabidopsis is successfully infected by the sugar beet cyst nematode (H. schachtii), a close relative of H. <span class="trait">glycines.</span> We isolated the At17.1 promoter, fused it to the beta-glucuronidase (GUS) reporter gene, and transformed this construct into Arabidopsis plants as well as soybean hairy roots. Histochemical analysis of plant materials containing the At17.1::GUS construct revealed that the At17.1 promoter is functional in Arabidopsis as well as in soybean and that during normal plant development the At17.1 promoter directs GUS expression predominantly to the vascular tissues and root tips of both plant species. When At17.1::GUS Arabidopsis plants and soybean hairy roots were inoculated with cyst nematodes, strong GUS activity was detected within the cyst nematode-induced feeding structures. Further tests of At17.1 promoter activity in Arabidopsis revealed that this promoter was induced by auxin, jasmonic acid, mannitol and dehydration. Quantitative real-time reverse transcription-polymerase chain reaction assays of At17.1 expression confirmed the observed promoter characteristics. Based on our expression data and the observation that both the soybean and the Arabidopsis homologues behaved in a similar fashion following cyst nematode infection, it is likely that these genes are closely associated with cyst nematode parasitism of plants, potentially with hormone and osmotic changes occurring in the developing nematode feeding cells. Furthermore, these data provide additional insights into the strengths of the Arabidopsis-H. schachtii pathosystem to study cyst nematode-plant interactions in lieu of less tractable pathosystems. This finding is supported by the fact that the Arabidopsis promoter tested here produced similar results in Arabidopsis and soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of an Arabidopsis phosphoglycerate mutase homologue is localized to apical meristems, regulated by hormones, and induced by sedentary plant-parasitic nematodes.</h2>
			<h3>PMID:15010616  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mazarei M(1), Lennon KA, Puthoff DP, Rodermel SR, Baum TJ.
				<br/> 
				年份：2003 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We previously isolated a partial soybean cDNA clone whose transcript abundance is increased upon infection by the sedentary, endoparasitic soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span> We now isolated the corresponding full-length cDNA and determined that the predicted gene product was similar to the group of cofactor-dependent phosphoglycerate mutase/bisphosphoglycerate mutase enzymes (PGM/bPGM; EC 5.4.2.1/5.4.2.4). We designated the corresponding soybean gene <span class="gene">GmPGM.</span> <span class="gene">PGM</span> and bPGM are key catalysts of glycolysis that have been well characterized in animals but not plants. Using the <span class="gene">GmPGM</span> cDNA sequence, we identified a homologous Arabidopsis thaliana gene, which we designated AtPGM. Histochemical GUS analyses of transgenic Arabidopsis plants containing the AtPGM promoter ::GUS construct revealed that the AtPGM promoter directs GUS expression in uninfected plants only to the shoot and root apical meristems. In infected plants, GUS staining also is evident in the nematode feeding structures induced by the cyst nematode <span class="trait">Heterodera</span> schachtii and by the root-knot nematode Meloidogyne incognita. Furthermore, we discovered that the AtPGM promoter was down-regulated by abscisic acid and hydroxyurea, whereas it was induced by sucrose, oryzalin, and auxin, thereby revealing expression characteristics typical of genes with roles in meristematic cells. Assessment of the auxin-inducible AUX1 gene promoter (a gene coding for a polar auxin transport protein) similarly revealed feeding cell and meristem expression, suggesting that auxin may be responsible for the observed tissue specificity of the AtPGM promoter. These results provide first insight into the possible roles of PGM/bPGM in plant physiology and in plant-pathogen interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pasteuria spp.: Systematics and Phylogeny of These Bacterial Parasites of Phytopathogenic Nematodes.</h2>
			<h3>PMID:19265995</h3>
				

				
			<p>
				作者：Preston JF, Dickson DW, Maruniak JE, Nong G, Brito JA, Schmidt LM, Giblin-Davis RM.
				<br/> 
				年份：2003 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pasteuria spp. include endospore-forming bacterial pathogens of cladoceran crustaceans and plant-parasitic nematodes. Propagation of these nematode pathogens requires attachment of soilborne endospores to nematode hosts, infection, growth, sporulation, and release of endospores to repeat the cycle of infection and propagation. The ability of these bacteria to suppress the levels of plant-parasitic nematodes in the field has made them particularly promising candidates for biocontrol of nematode diseases of plants. Genes encoding 16S ribosomal RNA have been sequenced for the cladoceran (water flea) parasite and type species, Pasteuria ramosa, and for Pasteuria spp. isolated from root-knot (Meloidogyne arenaria race 1 and Meloidogyne sp.), soybean cyst <span class="trait">(Heterodera</span> <span class="trait">glycines),</span> and sting (Belonolaimus longicaudatus) nematodes. These have provided a phylogenetic basis for their designation to a distinct clade within the family Alicyclobacillaceae of the gram-positive endospore-forming bacteria. Two apparent biotypes of P. penetrans demonstrating a host preference for different Meloidogyne spp. showed identical 16S rDNA sequences, suggesting host-recognition evolves within a given species. The sequences of genes encoding sporulation transcription factors, <span class="gene">sigE</span> and sigF, from P. penetrans biotype P-20 show different phylogenetic relationships to other endospore-forming bacteria, supporting their application to further discriminate Pasteuria spp. and biotypes. Distribution of an adhesin-associated epitope on polypeptides from different Pasteuria isolates provides an immunochemical approach to differentiate species and biotypes with specific host preferences. Application of bioinformatics to genomic data, as well as further characterization of the biochemical basis for host recognition, will facilitate development of Pasteuria spp. as benign alternatives to chemical nematicides.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A physical map of a gene-dense region in soybean linkage group A2 near the black seed coat and Rhg (4) loci.</h2>
			<h3>PMID:12582695</h3>
				

				
			<p>
				作者：Lewers K(1), Heinz R, Beard H, Marek L, Matthews B.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( Glycine max L. Merrill) linkage group A2 contains a major <span class="trait">resistance</span> gene to the soybean cyst nematode ( <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe) at the Rhg (4) locus near a gene encoding aspartokinase homoserine dehydrogenase (AK-HSDH) and also near the I locus affecting seed coat color. To identify clones related to this region of the genome, we used a PCR assay using primers designed from a gene encoding <span class="gene">AK-HSDH</span> to screen approximately 40,000 clones from a bacterial artificial chromosome (BAC) library constructed from genomic DNA of the susceptible cv. Williams 82. The identified BACs were screened with a second PCR assay using primers designed from DNA sequence associated with the I locus to confirm the location of the BACs. Only BAC <span class="gene">Gm_ISb001_056_G02</span> (56G2) was positive for both assays. BAC 56G2 contains several genes previously associated with stress or defense response including genes with high sequence similarity to those encoding chalcone synthase, glucosyl-transferase, a heat-shock transcription factor, a membrane-associated salt-inducible protein, adenosyl homocysteinase, a protein kinase, and a G10-like protein. The map contributes to our understanding of the organization of the soybean genome and to the completion of a physical map of the soybean genome. In addition, the genes identified provide landmarks to identify BAC clones near the Rhg (4) locus in resistant soybean genomic libraries and provide a foundation for comparison of soybean cyst nematode resistant and -susceptible DNA sequences in this region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a soybean ethylene-responsive element-binding protein gene whose mRNA expression changes during soybean cyst nematode infection.</h2>
			<h3>PMID:12059106  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mazarei M(1), Puthoff DP, Hart JK, Rodermel SR, Baum TJ.
				<br/> 
				年份：2002 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ethylene-responsive element-binding proteins (EREBPs) are members of a family of plant transcription factors. Conserved EREBP domains of these proteins bind to the GCC box, an ethylene-responsive promoter element found in many pathogenesis-related (PR) genes. Using degenerate primers to the EREBP domain from diverse plant species, an EREBP homolog was isolated from a soybean cDNA library. Gel mobility-shift assays revealed that the translation product of this cDNA bound specifically to GCC box sequences. We, therefore, named this gene Glycine max ethylene-responsive element-binding protein 1 <span class="gene">(GmEREBP1),</span> i.e., a gene coding for the first confirmed GCC box-binding protein of soybean. <span class="gene">GmEREBP1</span> mRNA abundance was analyzed by RNA blot hybridizations in soybean roots and shoots of cultivars Corsoy 79 and Hartwig, which are susceptible and resistant, respectively, to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines).</span> These analyses revealed that <span class="gene">GmEREBP1</span> is expressed in a root-preferential manner and that <span class="gene">GmEREBP1</span> mRNA abundance is changed after H. <span class="trait">glycines</span> infection. <span class="gene">GmEREBP1</span> mRNA abundance decreased in infected (susceptible) 'Corsoy 79' roots, whereas it increased in abundance in infected (resistant) 'Hartwig' roots. Furthermore, ethephon treatment repressed <span class="gene">GmEREBP1</span> mRNA accumulation in both cultivars, whereas wounding increased expression in both cultivars. These changes in mRNA steady-state levels suggest that <span class="gene">GmEREBP1</span> plays a role in soybean-H. <span class="trait">glycines</span> interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physical mapping of resistant and susceptible soybean genomes near the soybean cyst nematode <span class="trait">resistance</span> gene Rhg4.</h2>
			<h3>PMID:11768209  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lewers KS(1), Nilmalgoda SD, Warner AL, Knap HT, Matthews BF.
				<br/> 
				年份：2001 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean cyst nematode (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe, is the foremost pest of soybean (Glycine max L. Merr.). The rhg1 allele on linkage group (LG) G and the <span class="gene">Rhg4</span> allele on LG A2 are important in conditioning <span class="trait">resistance.</span> Markers closely linked to the <span class="gene">Rhg4</span> locus were used previously to screen a library of bacterial artificial chromosome (BAC) clones from susceptible 'Williams 82' and identified a single 150-kb BAC, <span class="gene">Gm_ISb001_056_G02</span> (56G2). End-sequenced subclones positioned onto a restriction map provided landmarks for identifying the corresponding region from a BAC library from accession PI 437654 with broad <span class="trait">resistance</span> to SCN. Seventy-three PI 437654 BACs were assigned to contigs based upon HindIII restriction fragment profiles. Four contigs represented the PI 437654 counterpart of the 'Williams 82' BAC, with PCR assays connecting these contigs. Some of the markers on the PI 437654 contigs are separated by a greater physical distance than in the 'Williams 82' BAC and some primers amplify bands from BACs in the mid-portion of the connected PI 437654 BAC contigs that are not amplified from the 'Williams 82' BAC. These observations suggest that there is an insertion in the PI 437654 genome relative to the 'Williams 82' genome in the <span class="gene">Rhg4</span> region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conversion of AFLP bands into high-throughput DNA markers.</h2>
			<h3>PMID:11361330  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Meksem K(1), Ruben E, Hyten D, Triwitayakorn K, Lightfoot DA.
				<br/> 
				年份：2001 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The conversion of AFLP bands into polymorphic sequence-tagged-site (STS) markers is necessary for high-throughput genotype scoring. Technical hurdles that must be overcome arise from genome complexity (particularly sequence duplication), from the low-molecular-weight nature of the AFLP bands and from the location of the polymorphism within the AFLP band. We generated six <span class="gene">STS</span> markers from ten AFLP bands (four AFLPs were from co-dominant pairs of bands) in soybean (Glycine max). The markers were all linked to one of two loci, rhg1 on linkage group G and <span class="gene">Rhg4</span> on linkage group A2, that confer <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines</span> I.). When the polymorphic AFLP band sequence contained a duplicated sequence or could not be converted to a locus-specific <span class="gene">STS</span> marker, direct sequencing of BAC clones anchored to a physical map generated locus-specific flanking sequences at the polymorphic locus. When the polymorphism was adjacent to the restriction site used in the AFLP analysis, single primer extension was performed to reconstruct the polymorphism. The six converted AFLP markers represented 996 bp of sequence from alleles of each of two cultivars and identified eight insertions or deletions, two microsatellites and eight single-nucleotide polymorphisms (SNPs). The polymorphic sequences were used to design a non-electrophoretic, fluorometric assay (based on the TaqMan technology) and/or develop electrophoretic <span class="gene">STS</span> markers for high-throughput genotype determination during marker-assisted breeding for <span class="trait">resistance</span> to cyst nematode. We conclude that the converted AFLP markers contained polymorphism at a 10- to 20-fold higher frequency than expected for adapted soybean cultivars and that the efficiency of AFLP band conversion to <span class="gene">STS</span> can be improved using BAC libraries and physical maps. The method provides an efficient tool for SNP and <span class="gene">STS</span> discovery suitable for marker-assisted breeding and genomics.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional screening yields a new beta-1,4-endoglucanase gene from <span class="trait">Heterodera</span> <span class="trait">glycines</span> that may be the product of recent gene duplication.</h2>
			<h3>PMID:11194873  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Y(1), Smant G, Davis E.
				<br/> 
				年份：2001 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Clones with secreted cellulolytic activity were identified when a cDNA library constructed from poly A(+) RNA of preparasitic second-stage juveniles of <span class="trait">Heterodera</span> <span class="trait">glycines,</span> the soybean cyst nematode, was expressed in the Escherichia coli SOLR strain and overlaid with a carboxymethylcellulose (CMC) substrate. Twenty CMC-degrading clones were analyzed, and all were either identical or strongly similar to a beta-1,4-endoglucanase gene (HG-eng-2), previously isolated from H. <span class="trait">glycines.</span> A subgroup of identical "HG-eng-2-like" clones had considerable differences in the 5' untranslated region compared with <span class="gene">HG-eng-2</span> and were designated HG-eng-3. One H. <span class="trait">glycines</span> genomic clone contained <span class="gene">HG-eng-2</span> and HG-eng-3 full-length genes, separated by a distance of approximately 8 kb, and a second genomic clone contained two copies of HG-eng-2, separated by approximately 6.5 kb, suggesting the presence of endoglucanase gene clusters in H. <span class="trait">glycines.</span> The <span class="gene">HG-eng-2</span> and HG-eng-3 genes were in opposite transcriptional orientation, with considerable nucleotide differences in their 5' flanking regions. The highly conserved nucleotide sequence in the introns and exons and their close proximity within the genome suggest that <span class="gene">HG-eng-2</span> and HG-eng-3 are the products of recent gene duplication and inversion.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hgm1, a novel male-specific gene from <span class="trait">Heterodera</span> <span class="trait">glycines</span> identified by differential mRNA display.</h2>
			<h3>PMID:9763288  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Llado RR(1), Urwin P, Atkinson H, Gilmartin PM.
				<br/> 
				年份：1998 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> is amphimictic: populations consist of both male and female individuals that reproduce sexually. However, under conditions of environmental stress, unbalanced male and female sex ratios are found. This observation requires an explanation of how sexual fate is determined in these nematodes. As a step towards identifying genes that are differentially regulated between male and female nematodes we have used cDNA differential display to screen for male-specific cDNA sequences. These studies have led to the isolation of a full-length male specific cDNA, Hgm1, that is up regulated in H. <span class="trait">glycines</span> males. Sequence analysis of this cDNA reveals that it represents a novel gene which encodes a 49.5 kDa acidic protein with a pI of 4.72 and a predicted 21 amino acid leader sequence. Neither the nucleotide nor the predicted amino acid sequence of this gene show any homology to sequences in the databases suggesting that <span class="gene">Hgm1</span> is a previously uncharacterised gene. Here we report the isolation, molecular characterisation and expression profile of Hgm1.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Enzymes of the Phenylpropanoid Pathway in Soybean Infected with Meloidogyne incognita or <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:19277292</h3>
				

				
			<p>
				作者：Edens RM, Anand SC, Bolla RI.
				<br/> 
				年份：1995 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcription of genes encoding several enzymes and the activity of some of these enzymes of the phenylpropanoid pathway leading to synthesis of chemical and physical barriers for defense of plants against root pathogens was estimated in susceptible and resistant soybean infected with <span class="trait">Heterodera</span> <span class="trait">glycines</span> race 3 or with Meloidogyne incognita race 3. Transcription of genes encoding phenylalanine ammonia lyase (PAL) and the activity of this enzyme increased in resistant, but not susceptible, soybean cultivars after nematode infection. Likewise, transcription of the gene encoding 4-coumaryl CoA ligase and activity of this enzyme were enhanced in resistant, but not susceptible, soybean cultivars after nematode infection. Activity of <span class="gene">PAL</span> decreased in susceptible soybean after H. <span class="trait">glycines</span> or M. incognita infection. Transcription of enzymes later in the phenylpropanoid pathway leading to glyceollin synthesis increased in both resistant and susceptible soybean in response to nematode infection; the increase was greater in resistant cultivars. These results suggest possible reasons for the rapid induction of glyceollin synthesis immediately after infection of resistant soybean cultivars with H. <span class="trait">glycines</span> or M. incognita and the failure of this response in infected, susceptible soybean cultivars. Nematode infection had no effect on the activity of enzymes in the branch of the pathway leading to lignin synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of <span class="trait">Heterodera</span> <span class="trait">glycines</span> pathotypes as affected by soybean cultivars.</h2>
			<h3>PMID:19305614</h3>
				

				
			<p>
				作者：Riggs RD, Hamblen ML, Rakes L.
				<br/> 
				年份：1977 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The reproductive abilities of four races of <span class="trait">Heterodera</span> <span class="trait">glycines</span> were compared on soybean cultivars by using single cyst or mass inoculations. Progeny transfers were used to determine changes in reproduction of H. <span class="trait">glycines.</span> Reproduction of all races (1, 2, 3, 4) was best on 'Lee' and poorest on <span class="gene">P1</span> 88788. The size of females produced and numbers of eggs/female of the different races varied with the cultivar. Races 1 and 3 appeared to contain low populations of Races 2 or 4. Races 2 and 4 were best selected by a series of transfers on 'Pickett' soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of stress-related hormones in plant defence during early infection of the cyst nematode <italic> <span class="trait">Heterodera</span> schachtii </italic> in Arabidopsis</h2>
			<h3>PMID:25825039</h3>
				
					<h3>PMCID:4657489</h3>
					<h4>期刊:The New Phytologist</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kammerhofer </surname>  <given-names> Nina </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Radakovic </surname>  <given-names> Zoran </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Regis </surname>  <given-names> Jully M A </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dobrev </surname>  <given-names> Petre </given-names>  </name>  <xref ref-type="aff" rid="au3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vankova </surname>  <given-names> Radomira </given-names>  </name>  <xref ref-type="aff" rid="au3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grundler </surname>  <given-names> Florian M W </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Siddique </surname>  <given-names> Shahid </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hofmann </surname>  <given-names> Julia </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wieczorek </surname>  <given-names> Krzysztof </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  <xref ref-type="author-notes" rid="fn1"> * </xref>  <xref ref-type="corresp" rid="cor1"/>  </contrib>  <aff id="au1">  <label> 1 </label>  <institution> Division of Plant Protection, Department of Crop Sciences, University of Natural Resources and Life Sciences, UFT Tulln </institution>  <addr-line> Konrad Lorenz Str. 24, 3430, Tulln, Austria </addr-line>  </aff>  <aff id="au2">  <label> 2 </label>  <institution> Institute of Crop Science and Resource Conservation, Department Molecular Phytomedicine, University Bonn </institution>  <addr-line> Karlrobert-Kreiten-Str. 13, D-53115, Bonn, Germany </addr-line>  </aff>  <aff id="au3">  <label> 3 </label>  <institution> Institute of Experimental Botany, Academy of Sciences of the Czech Republic </institution>  <addr-line> Rozvojov&#x000e1; 263, 165 02, Prague 6, Lysolaje, Czech Republic </addr-line>  </aff>
				<br/> 
				年份：<day> 03 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Interestingly, although ET plays a positive role in CN attraction, at later stages its suppression in syncytia indicates a rather negative function for nematode development, which is supported by studies of Mazarei <italic> et&#x000a0;al </italic> . ( <xref rid="b45" ref-type="bibr"> 2003 </xref> ) and Ali <italic> et&#x000a0;al </italic> . ( <xref rid="b2" ref-type="bibr"> 2013 </xref> ). The ethylene-responsive element binding protein gene <span class="gene">GmEREBP1</span> was found to be down-regulated in susceptible soybean infected with CN, whereas it is up-regulated in resistant roots (Mazarei <italic> et&#x000a0;al </italic> ., <xref rid="b45" ref-type="bibr"> 2003 </xref> ). Similarly, it was shown that the ET-responsive RAP2.6 gene is down-regulated in syncytia and its overexpression leads to enhanced <span class="trait">resistance,</span> which the authors suggest is a result of the activated JA pathway and callose deposition (Ali <italic> et&#x000a0;al </italic> ., <xref rid="b2" ref-type="bibr"> 2013 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SNP identification and marker assay development for high-throughput selection of soybean cyst nematode <span class="trait">resistance</span></h2>
			<h3>PMID:25903750</h3>
				
					<h3>PMCID:4407462</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Zi </given-names>  </name>  <address>  <email> shizi@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <address>  <email> smliu@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Noe </surname>  <given-names> James </given-names>  </name>  <address>  <email> jpnoe@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arelli </surname>  <given-names> Prakash </given-names>  </name>  <address>  <email> Prakash.Arelli@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <address>  <email> meksemk@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <address>  <email> zli@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Center for Applied Genetic Technologies &#x00026; Dep. of Crop and Soil Sciences, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff2">  <label/> Department of Plant, Soil and Agriculture Systems, Southern Illinois University, Carbondale, IL 62901 USA  </aff>  <aff id="Aff3">  <label/> Department of Plant Pathology, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff4">  <label/> USDA-ARS-SEA, Jackson, TN 38301 USA  </aff>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) is the most economically devastating pathogen of soybean. Two <span class="trait">resistance</span> loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> primarily contribute <span class="trait">resistance</span> to SCN race 3 in soybean. Peking and PI 88788 are the two major sources of SCN <span class="trait">resistance</span> with Peking requiring both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles and PI 88788 only the <italic> <span class="gene">Rhg1</span> </italic> allele. Although simple sequence repeat (SSR) markers have been reported for both loci, they are linked markers and limited to be applied in breeding programs due to accuracy, throughput and cost of detection methods. The objectives of this study were to develop robust functional marker assays for high-throughput selection of SCN <span class="trait">resistance</span> and to differentiate the sources of <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the genomic DNA sequences of 27 soybean lines with known SCN phenotypes, we have developed Kompetitive Allele Specific PCR (KASP) assays for two Single nucleotide polymorphisms (SNPs) from <span class="gene">Glyma08g11490</span> for the selection of the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele. Moreover, the genomic DNA of <span class="gene">Glyma18g02590</span> at the <italic> <span class="gene">Rhg1</span> </italic> locus from 11 soybean lines and cDNA of Forrest, Essex, Williams 82 and PI 88788 were fully sequenced. Pairwise sequence alignment revealed seven SNPs/insertion/deletions (InDels), five in the 6th exon and two in the last exon. Using the same 27 soybean lines, we identified one SNP that can be used to select the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele and another SNP that can be employed to differentiate Peking and PI 88788-type <span class="trait">resistance.</span> These SNP markers have been validated and a strong correlation was observed between the SNP genotypes and reactions to SCN race 3 using a panel of 153 soybean lines, as well as a bi-parental population, F <sub> 5 </sub> &#x02013;derived recombinant inbred lines (RILs) from G00-3213 x LG04-6000.
					</p>
				
					<p>&nbsp;&nbsp;
						Three functional SNP markers (two for <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> locus) were identified that could provide genotype information for the selection of SCN <span class="trait">resistance</span> and differentiate Peking from PI 88788 source for most germplasm lines. The robust KASP SNP marker assays were developed. In most contexts, use of one or two of these markers is sufficient for high-throughput marker-assisted selection of plants that will exhibit SCN <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN, <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> L.) is the most economically devastating pathogen of soybean and causes losses in soybean producing areas worldwide with no practical means of eradication [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. The annual losses exceed one billion dollars in the United States alone [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. Two <span class="trait">resistance</span> genes for SCN, <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">(Resistance</span> to <italic> H. <span class="trait">glycines</span> </italic> ) on Chromosome (Chr) 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr08, have been identified in the soybean germplasm, and have been shown to be the key sources of <span class="trait">resistance</span> in soybean cultivars [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. The <italic> <span class="gene">Rhg1</span> </italic> locus, which is crucial for broad spectrum <span class="trait">resistance</span> to SCN, has been identified in various <span class="trait">resistance</span> sources, such as PI 88788 and Peking [ <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						SCN field populations are genetically variable [ <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Sixteen SCN races have been described and the race designation was determined by comparing its reproduction on a set of four soybean differentials with a standard SCN-susceptible cultivar. To avoid the implication of genetic predictability, a revised classification scheme for field SCN population introduced the <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> (HG) types determined by reactions to seven indicator lines [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. The distribution of SCN populations in the field varied among different geographic regions. However, race 3 is the most frequently found SCN populations in major soybean producing states in the USA [ <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. Soybean lines PI 88788 and Peking are the two main sources of <span class="trait">resistance</span> to SCN race 3 [ <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. Over 90% of the commercially available SCN-resistant soybean cultivars in the USA carry the <italic> rhg1-b </italic> allele, which is derived from PI 88788 due to its desirable agronomic characteristics [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. While PI 88788-type <span class="trait">resistance</span> only requires the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, Peking-type <span class="trait">resistance</span> needs both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles for a resistant phenotype [ <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Although the molecular basis of SCN-resistance is still unknown, recent studies have shed light on the genes involved in the <span class="trait">resistance.</span> Map-based cloning showed that a single gene at <italic> <span class="gene">Rhg4</span> </italic> locus encoding a serine hydroxymethyltransferase (SHMT, <span class="gene">Glyma08g11490)</span> is responsible for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Two polymorphisms have been identified in the coding sequence of SHMT, 389&#x000a0;G/C in the first exon and 1,165 A/T in the second exon, which resulted in the amino acid change of Arginine (R) to Proline (P) and Tyrosine(Y) to Asparagine (N), respectively. These polymorphisms change the property of the enzyme and may correlate with <italic> <span class="gene">Rhg4</span> </italic> -dependent <span class="trait">resistance</span> and susceptibility [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Genetic mapping and gene functional analysis of the <italic> rhg1-b </italic> locus exhibited a 31.2&#x000a0;kb genomic segment encoding a predicted amino acid transporter <span class="gene">(Glyma18g02580),</span> an &#x003b1;&#x02013;SNAP protein <span class="gene">(Glyma18g02590)</span> and a protein with wound-inducible protein 12 (WI12) <span class="gene">(Glyma18g02610)</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. Gene silencing and overexpression analysis showed that all three genes contribute to SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Interestingly, the phenotypic reaction to SCN is associated with the copy number variation of this 31.2&#x000a0;kb fragment. Only one copy is present in susceptible varieties, such as Williams 82, and multiple copies in SCN resistant lines. For example, Peking possesses three copies and seven to 10 copies were detected in PI 88788-derived resistant lines [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Sequencing of cDNA from several soybean lines revealed that multiple alleles of <span class="gene">Glyma18g02590</span> were present in the different multi-copy <italic> <span class="gene">Rhg1</span> </italic> categories [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Planting of resistant soybean cultivars combined with crop rotation formulates the best management strategy for reducing <span class="trait">yield</span> losses due to SCN. However, traditional breeding for resistant lines is challenging, not just because the phenotyping of SCN is time-consuming, but it is also affected by the different <span class="trait">resistance</span> genes [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and varied SCN populations in the field [ <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Therefore, the use of molecular markers to assist the selection of SCN <span class="trait">resistance</span> is a crucial improvement in breeding programs. Major efforts have been made to develop genetic markers linked to SCN <span class="trait">resistance</span> genes. Restriction fragment length polymorphisms (RFLPs) markers have been identified and reported to facilitate soybean breeding for both <italic> <span class="gene">Rhg1</span> </italic> [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> - <xref ref-type="bibr" rid="CR20"> 20 </xref> ] and <italic> <span class="gene">Rhg4</span> </italic> loci [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> - <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Simple sequence repeats (SSRs) have further been discovered and applied in the marker-assisted selection of SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> - <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. SNP/InDel markers have also been developed at the flanking region of both loci [ <xref ref-type="bibr" rid="CR11"> 11 </xref> , <xref ref-type="bibr" rid="CR28"> 28 </xref> - <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. However, they are not diagnostic markers, and both SSRs and RFLPs are labor/time-consuming and cost ineffective. Thus, they are very limited to be applied in breeding programs in a high throughput setting.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we report the development of three robust, functional SNP marker assays, two for the <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> . These functional SNP markers provide genotype information which can be used for the high-throughput selection of SCN <span class="trait">resistance,</span> as well as differentiating Peking and PI 88788 <span class="trait">resistance</span> sources for breeding of SCN <span class="trait">resistance</span> cultivars and germplasm introgression.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate which SNPs correlate with known SCN reactions of soybean lines, Kompetitive Allele Specific PCR (KASP) assays were developed and tested for all 13 SNPs at the <italic> <span class="gene">Rhg1</span> </italic> locus and two previously identified SNPs (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> A) at <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ] using 27 soybean lines with known SCN reactions. Two SNPs from <span class="gene">Glyma08g11490</span> are associated with the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele, and they are completely linked (data not shown). Three SNPs in <span class="gene">Glyma18g02580</span> and two SNP/InDels in the last exon of <span class="gene">Glyma18g02590</span> are associated with the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, and all five markers are completely linked. Five SNPs in the 6th exon of <span class="gene">Glyma18g02590</span> could distinguish the sources of Peking-type and PI88788-type <span class="trait">resistance.</span> However, three SNPs identified in the 5&#x02019;-UTR of <span class="gene">Glyma18g02610</span> showed no correlation with SCN <span class="trait">resistance.</span> Three SNPs were selected to provide genotype information for the selection of SCN <span class="trait">resistance</span> (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ), which were designated as GSM383, GSM381 and GSM191. GSM381 and GSM383 are located in the 6th and last exons of <span class="gene">Glyma18g02590</span> at the <italic> <span class="gene">Rhg1</span> </italic> locus, respectively, while GSM191 is in the first exon of <span class="gene">Glyma08g11490.</span> The KASP assays of three SNPs performed with 27 soybean lines exhibited a good clustering of the homozygous alleles, suggesting that it is feasible to employ those markers for selection. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> Positions of three SNPs in <span class="gene">Glyma18g02590</span> and <span class="gene">Glyma08g11490</span> at <italic> Rgh1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, respectively. GSM numbers were used to designate SNP markers. Blue squares represent exons and black lines represent introns and UTRs. The genome coordinates (Database: <span class="gene">Gmax_275_v2.0.softmasked)</span> of each SNP are shown.
					</p>
				
					<p>&nbsp;&nbsp;
						Haplotype analysis was performed using the Soy50k SNP Infinium chip data of 27 lines and nine cultivars including Woodruff, Hartwig, Boggs, Gordon, Bryan, G00-3880, Prichard, Bedford and Carver. The <italic> <span class="gene">Rhg1</span> </italic> region was compared between Peking and the Peking-type lines: Forrest, Woodruff, Hartwig, Boggs, Gordon, Bryan, PI 89772, PI 90763 and PI 437654, revealing that these lines have same haplotypes to Peking at the <italic> <span class="gene">Rhg1</span> </italic> locus as large as 5.9 Mbp. A 788&#x000a0;kb haplotype window block was shown in Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A. Hartwig, PI 89772, PI 90763 and PI 437654 showed a relatively small haplotype window (710&#x000a0;kb) when compared to Peking. Woodruff had a larger haplotype block that is about 3.8Mbp and over 5.9Mbp was observed in Forrest, Gordon and Bryan. However, PI 88788 and other susceptible-type lines showed no introgression from Peking at the <italic> <span class="gene">Rhg1</span> </italic> locus. All eight PI 88788-type lines, G00-3880, Prichard, Bedford, Carver, PI 548316, PI 209332, PI 603372 and PI 603587A exhibited haplotype window blocks ranging from 1.6Mbp to 4.3Mbp, when compared to PI 88788 at the <italic> <span class="gene">Rhg1</span> </italic> region. Haplotype analysis of <italic> <span class="gene">Rhg4</span> </italic> locus on Chr08 indicated haplotype window blocks from Peking and nine Peking-type lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B), with a smaller region of 348&#x000a0;kb in Woodruff, Boggs and Bryan. However, the introgressed haplotype blocks are as large as 2.6Mbp in Forrest, Hartwig and Gordon. Examination of the pedigrees of these lines indicated that Peking is present somewhere in the extended pedigree of these six cultivars ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/npgs/acc/acc_queries.html"> http://www.ars-grin.gov/npgs/acc/acc_queries.html </ext-link> ) [ <xref ref-type="bibr" rid="CR32"> 32 </xref> - <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. These data are consistent with genotype indications of our three SNPs alleles and further substantiated the effectiveness of these SNP markers. <fig id="Fig3"> <label> Figure 3 </label> <caption> <p> Haplotype analysis at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci using Soy50k SNP Infinium chip data. Twenty-seven soybean plant introductions and nine cultivars, Woodruff, Hartwig, Boggs, Gordon, Bryan, G00-3880, Prichard, Bedford and Carver were used. <bold> A </bold> . Haplotype analysis at <italic> <span class="gene">Rhg1</span> </italic> locus. SNPs alleles at the <italic> <span class="gene">Rhg1</span> </italic> region on Chr18 were compared to Peking and PI 88788, respectively. A 788 kbp region (Coordinate of 1,266,210-2,054,952 on Chr18 based on <span class="gene">Gmax_275_v2.0.softmasked)</span> consisting of 82 SNPs is shown in the graph. <bold> B </bold> . Haplotype analysis at <italic> <span class="gene">Rhg4</span> </italic> locus. SNPs alleles at the <italic> <span class="gene">Rhg4</span> </italic> region on Chr08 were compared to Peking. A 1.5Mbp region (Coordinate of 6,903,359-8,399,514 on Chr08 based on <span class="gene">Gmax_275_v2.0.</span> softmasked) consisting of 82 SNPs is shown here. The reference line is shown in dark red. Red squares represent same alleles as the reference and green squares represent alleles that are different from the reference. The position of the locus and the <span class="trait">resistance</span> types are as indicated.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the SNP markers with more diverse lines, we explored the data from the previous three years&#x02019; USDA Uniform Soybean Tests for Southern States where the phenotypes of SCN race 3 are available for all the tested lines. We requested a total of 153 lines from eight institutions and performed KASP assays on these lines with SNP markers, GSM381, GSM383 and GSM191 (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> A). The SNP genotypes, SCN race 3 reactions and pedigrees were listed (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). The rating of SCN phenotype is on a scale of 1 to 5 based on the number of cysts, thus, lines with ratings smaller than 3 were considered as resistant and lines with ratings equal to or greater than 3 as susceptible. A strong accordance was observed between SNP genotypes and reactions to SCN race 3 for 145 germplasm lines with an exception of 10 lines (successful rate&#x02009;&#x0003e;&#x02009;93%). Due to the large discrepancy of ratings for same line between two years, eight lines were excluded for analysis. Single factor analysis using General Linear Model (GLM procedure, SAS 9.3) showed a significant correlation between SCN <span class="trait">resistance</span> phenotypes and genotypes consisted of three SNP markers (R <sup> 2 </sup> &#x02009;=&#x02009;0.69, P&#x02009;&#x0003c;&#x02009;0.001). Moreover, we have examined the extended pedigrees (data not shown) of these lines with <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> allele (GSM381&#x02009;=&#x02009;G) to identify the sources of <span class="trait">resistance</span> (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). The SNP genotypes are in complete agreement with the <span class="trait">resistance</span> sources based on extended pedigrees: Peking-source with a G allele at GSM383 and PI 88788-source with a C allele, indicating that GSM383 is a functional SNP to differentiate Peking and PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele. <fig id="Fig4"> <label> Figure 4 </label> <caption> <p> KASP graphs of three SNPs for validation and precision genotyping test population. <bold> A </bold> . KASP SNP graphs of 153 soybean lines from 2011&#x02013;2013 USDA Southern Uniform Soybean Tests with known SCN race 3 reactions. <bold> B </bold> . KASP SNP graphs of F <sub> 5 </sub> -derived RILs from the population derived from G00-3213 x LG04-6000.
					</p>
				
					<p>&nbsp;&nbsp;
						We also performed a precision genotyping test on a bi-parental population of F <sub> 5 </sub> -derived Recombinant inbred lines (RILs) from G00-3213 x LG04-6000. The line G00-3213 carries a Peking-source <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles (GSM381&#x02009;=&#x02009;G, GSM383&#x02009;=&#x02009;G, and GSM191&#x02009;=&#x02009;G) and is resistant to SCN race 3, while LG04-6000 is a susceptible line with T, C and C alleles for GSM381, GSM383, and GSM191, respectively. Three replicates of 150 RILs with two parents were arranged in a randomized complete block design in greenhouse and each entry was genotyped with three SNP markers (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> B). Ten weeks after inoculation, phenotyping was carried out by counting the numbers of cysts on the roots of each individual plant. Resulting phenotypes demonstrated that combination of the <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> (GSM381&#x02009;=&#x02009;G and GSM383&#x02009;=&#x02009;G) and <span class="trait">resistance</span> <italic> <span class="gene">Rhg4</span> </italic> alleles (GSM191&#x02009;=&#x02009;G) brought a resistant phenotype with only 0.4 cyst/plant and all the other combinations exhibited a more susceptible phenotype (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ), suggesting that these three markers can be employed for the selection of SCN race 3 <span class="trait">resistance</span> from a population derived from Peking source. Single factor analysis using a GLM procedure in SAS 9.3 was performed for three subsets individually and combined. All of them showed a significant correlation between the average number of cysts and genotypes of three SNPs, GSM381, GSM383, and GSM191 (P&#x02009;&#x0003c;&#x02009;0.0001), with an R-square of 0.64, 0.40, 0.24, and 0.3 for RIL sets 1, 2, 3 and combined, respectively. The different R-square values among sets might be due to the conditions at different inoculation times. <fig id="Fig5"> <label> Figure 5 </label> <caption> <p> Reactions of race3 to different combinations of SNP marker alleles for 150&#x000a0;F <sub> 5 </sub> families of G00-3213 x LG04-6000. <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> genotypes are represented as combinations of alleles of SNP markers, GSM381, GSM383 and GSM191. Boxes show median, 25% to 75% of the data and whiskers extend to minimum and maximum of the data.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic basis of <span class="trait">resistance</span> to SCN is complex in soybean, not only because <span class="trait">resistance</span> to SCN is multi-genic, but also because different sets of genes/ Quantitative trait loci (QTLs) confer <span class="trait">resistance</span> to the diverse populations of SCN due to their genetic heterogeneity. Field populations of soybean cyst nematode have been classified into 16 races based on their ability to develop on resistant cultivars [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ]. <span class="trait">Resistance</span> to SCN is race-cultivar specific [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ], for example, soybean cultivars derived from Peking source are commonly claimed to be resistant to race 1 and 3, however, the cultivars with SCN <span class="trait">resistance</span> from PI 88788 are usually shown to be resistant to race 3 and 14 [ <xref ref-type="bibr" rid="CR37"> 37 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. A growing body of evidence has shown different defense loci that underlie <span class="trait">resistance</span> to different races. The <italic> <span class="gene">Rhg1</span> </italic> locus on Chr18 is necessary for <span class="trait">resistance</span> to all SCN races [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Many other cultivar-race specific QTLs have been demonstrated in <span class="trait">resistance</span> sources, such as <italic> <span class="gene">Rhg4</span> </italic> for race 1 and 3, <italic> Rhg3 </italic> and <italic> Rhg5 </italic> for race 2 and <italic> Rhg2 </italic> for race 5 and 14 [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Although two functional SNPs have been identified for <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], no high-throughput marker assays have been reported to date. Moreover, several SNPs have been found at <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], but no specific functional SNPs as well as detection assays have been reported to associate with the <span class="trait">resistance</span> allele, nor the SNPs to differentiate sources of <span class="trait">resistance.</span> Built on the previous discoveries of copy number variation, gene dosage, and DNA methylation at the <italic> <span class="gene">Rhg1</span> </italic> locus by Cook et al. [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] and SNPs at the <italic> <span class="gene">Rhg4</span> </italic> locus by Liu et al. [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], in this study, we have developed SNP marker assays, GSM381, GSM383, and GSM191 at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> locus for the selection of <span class="trait">resistance</span> alleles, (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). The SNPs reside in the coding sequences of <span class="trait">resistance</span> genes <span class="gene">(Glyma18g02590</span> and <span class="gene">Glyma08g11490)</span> and also introduce changes in protein sequences, thus they can be considered as functional markers for SCN race 3 <span class="trait">resistance</span> and applied in high throughput marker-assisted selection with high accuracy. Similarly, the knowledge of additional loci along with the genome sequence of soybean may be used in the marker development for <span class="trait">resistance</span> to different SCN races.
					</p>
				
					<p>&nbsp;&nbsp;
						PI 88788 has been the primary choice in soybean cultivar development as a source for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ], however, SCN populations have demonstrated the capability to mutate and recombine to new races, which can overcome the soybean <span class="trait">resistance</span> derived from PI 88788 [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Reports have shown that PI 88788 has not been providing sufficient protection against SCN in some fields [ <xref ref-type="bibr" rid="CR42"> 42 </xref> - <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. Thus, additional <span class="trait">resistance</span> sources are desired to be used for developing new resistant cultivars and Peking is believed to be an attractive option. Although SNP GSM381 discriminates resistant and susceptible <italic> <span class="gene">Rhg1</span> </italic> alleles, it does not distinguish PI 88788- and Peking-type <italic> <span class="gene">Rhg1</span> </italic> loci. Thus, to identify the sources of <span class="trait">resistance,</span> we demonstrated that one SNP marker at <italic> <span class="gene">Rhg1</span> </italic> locus, GSM383 was able to distinguish Peking from PI 88788 loci for most soybean germplasm (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). This SNP can be employed in differentiating Peking from PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> allele for breeding programs.
					</p>
				
					<p>&nbsp;&nbsp;
						The successful prediction rate for three SNP markers at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci is over 93% in this study (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). A few lines (10 lines from USDA Uniform Test (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1) and two lines from 27 validation lines (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> )) did not show a correlation between the expected phenotypes and the <span class="trait">resistance</span> SNP alleles. A number of factors may result in the discrepancy between genotyping and phenotyping results, including 1) phenotyping/ or seed source errors: some of the lines have been tested for only one year, and one-time phenotyping of SCN <span class="trait">resistance</span> may not be definitive; 2) sources other than PI 88788 and Peking may be providing <span class="trait">resistance;</span> and 3) the complexity of the <italic> <span class="gene">Rhg1</span> </italic> region is not fully understood. A future study will be designed to understand why the lines with the <span class="trait">resistance</span> genotypes did not have a <span class="trait">resistance</span> reaction to SCN race 3. The 31.2&#x000a0;kb region at <italic> Rgh1 </italic> locus is repeated in resistant genotypes [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], and the sequences of the individual copies are also variant within a single genotype, which makes marker development challenging for the <italic> <span class="gene">Rhg1</span> </italic> locus. We also tested marker performance with artificial heterozygous DNA, where PI 88788/ Peking and Williams 82 DNA were mixed at 1:1 ratio. Although the artificial heterozygous allele cluster was located a little closer towards the resistant homozygous allele cluster in the KASP SNP graphs (data not shown) because of the gene repeats in resistant lines, the alleles can still be called clearly. Therefore, the functional SNP markers that we identified can be used not only to select the <span class="trait">resistance</span> allele, but also to differentiate <span class="trait">resistance</span> sources, which were also confirmed by analysis of the <italic> <span class="gene">Rhg1</span> </italic> region with Soy50k SNP Infinium chip data using 38 soybean lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						A recent study of the <italic> <span class="gene">Rhg1</span> </italic> locus revealed that copy number variation of the 31&#x000a0;kb DNA fragment, DNA sequence variation, and differentially methylated patterns were correlated with the <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. These discoveries may also offer additional selection strategies for the <italic> <span class="gene">Rhg1</span> </italic> allele, in addition to the functional SNP detection assays we developed. Cook et al. [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] explored the whole genome data for <italic> <span class="gene">Rhg1</span> </italic> locus in 42 soybean lines, and multiple lines are in common in both studies. In this study, we got the same C-terminal protein sequence of <span class="gene">Glyma18g02590</span> for both low-copy and high-copy <italic> <span class="gene">Rhg1</span> </italic> as described in the previous study ([ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B), indicating that GSM 381 maybe also exist in their DNA sequences. Interestingly, they identified a novel splice isoform of <span class="gene">Glyma18g02590</span> cDNA in Peking-type <italic> <span class="gene">Rhg1</span> </italic> lines, however, we did not detect this isoform in our cDNA sequencing, probably due to the low copy number of this isoform and use of a different sequencing method. Thus, more studies are needed to investigate the <italic> <span class="gene">Rhg1</span> </italic> locus in various soybean lines for new insights into the development of improved selection approaches.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic sequence of <span class="gene">Glyma08g11490</span> at <italic> <span class="gene">Rhg4</span> </italic> locus in 28 soybean lines and <span class="gene">Glyma18g02590</span> at <italic> <span class="gene">Rhg1</span> </italic> locus in 11 lines, along with the known reactions to soybean cyst nematode race 3 of these lines, allowed us to identify functional SNPs and develop robust KASP SNP assays for marker-assisted breeding. Using a panel of 153 soybean germplasm and a segregating bi-parent population, we have validated the usefulness of these three SNP markers, two at <italic> <span class="gene">Rhg1</span> </italic> locus and one at <italic> Rhg4. </italic> The results of this work demonstrated that these markers could be applied for high throughput selection of SCN <span class="trait">resistance</span> from PI 88788 and Peking sources and for accelerating breeding of SCN <span class="trait">resistance</span> in an accurate and efficient manner.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A decline in transcript abundance for <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> homologs of <italic> Caenorhabditis elegans uncoordinated </italic> genes accompanies its sedentary parasitic phase</h2>
			<h3>PMID:17445261</h3>
				
					<h3>PMCID:1867819</h3>
					<h4>期刊:BMC Developmental Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> heartwood27@hotmail.com </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Martins </surname>  <given-names> Veronica E </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> veronicamartins@yahoo.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Alkharouf </surname>  <given-names> Nadim W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> nalkharouf@gmail.com </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Overall </surname>  <given-names> Christopher C </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> coverall@gmu.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> macdonam@ba.ars.usda.gov </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> matthewb@ba.ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 4 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The genetically-defined <italic> <underline> unc </underline> oordinated </italic> ( <italic> unc </italic> ) genes perform many functions in <italic> C. elegans </italic> . The protein products of the <italic> unc </italic> genes are involved in muscle focal adhesion, architecture and stimulation (via neuromuscular connections). However, null alleles of <italic> unc </italic> genes can exhibit <underline> P </underline> aralyzed <underline> A </underline> rrested at <underline> T </underline> wo-fold stage ( <italic> <span class="gene">pat</span> </italic> ) phenotypes. The <italic> unc </italic> mutants all display uncoordinated motion, slow movement, or paralysis [ <xref ref-type="bibr" rid="B3"> 3 </xref> ]. The <italic> unc </italic> family of mutants contains 114 different members [ <xref ref-type="bibr" rid="B3"> 3 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ]. We believe that much of the muscle degeneration observed in <italic> H. <span class="trait">glycines</span> </italic> would likely involve transcriptional regulation of <italic> H. <span class="trait">glycines</span> </italic> homologs of <italic> unc </italic> genes whose protein products are involved in (1) the acto-myosin complex, (2) muscle focal adhesion or (3) other aspects of muscle composition and regulation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Population-specific gene expression in the plant pathogenic nematode <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> exists prior to infection and during the onset of a resistant or susceptible reaction in the roots of the <italic> Glycine max </italic> genotype Peking</h2>
			<h3>PMID:19291306</h3>
				
					<h3>PMCID:2662880</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" corresp="yes" contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> vklink@biology.msstate.edu </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Hosseini </surname>  <given-names> Parsa </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> phosse1@towson.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Margaret.MacDonald@ARS.USDA.GOV </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Alkharouf </surname>  <given-names> Nadim W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> nalkharouf@gmail.com </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Ben.Matthews@ARS.USDA.GOV </email>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 3 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Experiments designed at comparing R to S reactions at multiple time points have the advantage of permitting the filtering out differential gene expression that is common between the infections caused by diverse <italic> H. <span class="trait">glycines</span> </italic> populations. This, theoretically, may allow the identification of genes expressed uniquely in the R or S reaction. Klink et al. [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B20"> 20 </xref> ] performed those experiments and demonstrated that while much plant gene expression was common, signature gene expression profiles existed for the R and S reactions [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B20"> 20 </xref> ]. This dichotomy in expression occurs early during infection and prior to the selection of feeding sites [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Thus, gene expression that is important to the R or S reaction is not limited to the syncytium because they have not even formed by the 12 h time point studied in that analysis [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Of note, a <italic> G. max </italic> <underline> h </underline> eat <underline> s </underline> hock <underline> p </underline> rotein <underline> 90 </underline> (HSP90) gene was found to be induced specifically during the R reaction [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Recent experiments using <underline> v </underline> irus <underline> i </underline> nduced <underline> g </underline> ene <underline> s </underline> ilencing (VIGS) of the <italic> S. lycopersicum </italic> <span class="gene">HSP90</span> resulted in attenuation of <span class="trait">resistance</span> of the root knot nematode ( <italic> Meloidogyne <span class="gene">spp</span> </italic> .) [ <xref ref-type="bibr" rid="B32"> 32 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						MA demonstrated that the only differential gene expression occurring in NL1-RHg at 12 h is induced gene expression. Comparatively few genes were induced as well. Actin, a gene composing muscles is induced in nematodes experiencing an R reaction at 12 h. Prior analyses have shown that actin1 ( <italic> Hg-act-1 </italic> ) is induced during the pi-L2 stage of development [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. The analysis identified a <italic> H. <span class="trait">glycines</span> </italic> homolog of glutathione-S-transferase as being induces in <italic> Hg-GST-1 </italic> nematodes experiencing an R reaction at 12 h. <span class="gene">GST</span> is a component of the detoxification pathway. In <italic> Meloidogyne incognita </italic> , a secreted <span class="gene">GST</span> was shown to be induced 27 times more abundantly in the L3 stage than in the L2 [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. However, in our case, an induced expression was noted in the i-L2 stage in <italic> H. <span class="trait">glycines</span> </italic> experiencing a resistant reaction as compared to those experiencing a susceptible reaction at 12 h. Another gene, carnitine palmitoyl transferase (CPT) family member (cpt-1) was identified as being induced in <italic> H. <span class="trait">glycines</span> </italic> experiencing a resistant reaction as compared to those experiencing a susceptible reaction at 12 h. The CPTase system is composed of two mitochondrial membrane-bound enzymes. The first is CPTase I and the second is CPTase II. The location of CPTase I is the inner side of the outer mitochondrial membrane. In contrast, the location of CPTase II is the inner mitochondrial membrane. These enzymes, along with acyl-CoA synthetase and the carnitine/acylcarnitine translocase, provide the mechanism for how long-chain fatty acids are transferred cytosolically to the mitochondrial matrix for &#x003b2;-oxidation. In humans, mutations in CPT are a clinically heterogeneous autosomal recessive disorder of energy metabolism. CPT I genetic defects impair liver activity while CPT II genetic defects are characterized, clinically, by muscle weakness and rhabdomyolysis (rapid breakdown of skeletal muscle) [ <xref ref-type="bibr" rid="B52"> 52 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Very little difference in gene expression was observed between NL1-RHg and TN8 at 3d. Histological analyses of root tissue development during R and S reactions at 3d have been performed. Those analyses have demonstrated that <italic> H. <span class="trait">glycines</span> </italic> initiates the formation of syncytia that appear similar at the anatomical level. While syncytia appear similar at the anatomical level at 3d, molecular differences are present [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Syncytia collected from Peking roots undergoing R or S reactions revealed differences in gene expression between the two reactions at 3d [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Some of those differences involved induced expression of genes involved in important aspects of the defense response. Some of the genes induced in syncytia undergoing an R reaction as compared to syncytia undergoing an S reaction were lipoxygenase, <italic> G. max </italic> <span class="gene">HSP70</span> <span class="gene">(GmHSP70),</span> superoxide dismutase, WRKY transcription factor and <span class="gene">GmRLK3</span> [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Therefore, localized changes in the feeding environment of the nematode (the syncytium) are underway even though they are not yet evident at 3d when examining NL1-RHg and TN8 gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						A time course analysis of contrasting <italic> H. <span class="trait">glycines</span> </italic> gene expression was presented. The comparison demonstrated gene expression as <italic> H. <span class="trait">glycines</span> </italic> experienced an R or S reaction. Six of these profile types had greater than 100 genes each. Comparing their expression with <italic> G. max </italic> gene expression during an R reaction may identify deficiencies in those <italic> H. <span class="trait">glycines</span> </italic> genes involved in defense. It may also explain how the nematodes are being overcome as the <italic> G. max </italic> defense response is becoming established. For example, in whole roots undergoing R, a substantial number of <italic> G. max </italic> probe sets for HSPs (i.e., ClpB/HSP101, <span class="gene">HSP90</span> and HSP70), <span class="gene">LOX</span> and genes involved in energy metabolism measured induced gene expression [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Laser capture microdissection and MA analyses of syncytia undergoing a resistant reaction resulted in the identification of genes known to be involved in various defense responses (i.e., HSP70, WRKY-like transcription factor, <span class="gene">GmRLK3,</span> Superoxide dismutase and LOX) [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Therefore, <italic> G. max </italic> gene expression at the site of infection and distally in the cortex tissue surrounding <italic> H. <span class="trait">glycines</span> </italic> is likely playing roles during the R reaction.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Developing a Systems Biology Approach to Study Disease Progression Caused by <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> in <italic> Glycine max </italic></h2>
			<h3>PMID:19936075</h3>
				
					<h3>PMCID:2759149</h3>
					<h4>期刊:Gene Regulation and Systems Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P. </given-names>  </name>  <xref ref-type="aff" rid="af1-grsb-2007-017"> 1 </xref>  <xref ref-type="corresp" rid="c1-grsb-2007-017"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Overall </surname>  <given-names> Christopher C. </given-names>  </name>  <xref ref-type="aff" rid="af2-grsb-2007-017"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F. </given-names>  </name>  <xref ref-type="aff" rid="af1-grsb-2007-017"> 1 </xref>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The identification of the induction of WRKY gene expression during the susceptible reaction was particularly intriguing. This is because WRKY transcription factors are involved in the defense response ( <xref ref-type="bibr" rid="b104-grsb-2007-017"> Shen et al. 2007 </xref> ). Recent microarray analyses have demonstrated WRKY gene expression during infection of <italic> A. thaliana </italic> by the bacterium <italic> Pseudomonas syringae </italic> . Those analyses demonstrated that the activation and downstream targets of the WRKY gene family is complex ( <xref ref-type="bibr" rid="b114-grsb-2007-017"> Wang et al. 2006 </xref> ). Those microarray analyses have aided their understanding of the activation of gene networks during infection ( <xref ref-type="bibr" rid="b114-grsb-2007-017"> Wang et al. 2006 </xref> ). WRKY genes have been shown to be rapidly induced during the <span class="trait">resistance</span> response in pepper ( <italic> Capsicum annum </italic> ) ( <xref ref-type="bibr" rid="b84-grsb-2007-017"> Oh et al. 2006 </xref> ). <italic> A. thaliana </italic> mutants of WRKY33 become susceptible to the fungal pathogens <italic> Botrytis cinerea </italic> and <italic> Alternaria brassicicola </italic> ( <xref ref-type="bibr" rid="b124-grsb-2007-017"> Zheng et al. 2006 </xref> ). However, over-expression of WRKY33 was shown to enhance susceptibility to <italic> P. syringae </italic> and is coincident with reduced PR1 gene expression ( <xref ref-type="bibr" rid="b124-grsb-2007-017"> Zheng et al. 2006 </xref> ). <xref ref-type="bibr" rid="b96-grsb-2007-017"> Ryu et al. (2006) </xref> explored the expression of 45 WRKY genes in <italic> Oryza sativa </italic> . They demonstrated that 15 WRKY genes were significantly induced during the <span class="trait">resistance</span> response after infection with the fungus <italic> Magnaportha grisaei </italic> ( <xref ref-type="bibr" rid="b96-grsb-2007-017"> Ryu et al. 2006 </xref> ). Twelve of those WRKY genes were also induced during infection with the bacterial pathogen <italic> Xanthomonas oryzae </italic> pv. oryzae (Xoo) ( <xref ref-type="bibr" rid="b96-grsb-2007-017"> Ryu et al. 2006 </xref> ). <xref ref-type="bibr" rid="b122-grsb-2007-017"> Xu et al. (2006) </xref> demonstrated that, in <italic> A. thaliana </italic> , WRKY18, WRKY40 and <span class="gene">WRKY60</span> form a multimeric complex. The influence of WRKY gene expression during <span class="trait">resistance</span> to <italic> P. syringae </italic> and <italic> B. cinerea </italic> was tested by mutational analysis. Single mutants of wrky18, wrky40 or wrky60 had little effect ( <xref ref-type="bibr" rid="b122-grsb-2007-017"> Xu et al. 2006 </xref> ). However, wrky18-wrky40 and wrky18-wrky60 double mutants exhibited <span class="trait">resistance</span> to <italic> P. syringae </italic> , but were susceptible to <italic> B. cinerea </italic> ( <xref ref-type="bibr" rid="b122-grsb-2007-017"> Xu et al. 2006 </xref> ). The wrky18-wrky40-wrky60 triple mutant was also resistant to <italic> P. syringae </italic> and susceptible to <italic> B. cinerea </italic> ( <xref ref-type="bibr" rid="b122-grsb-2007-017"> Xu et al. 2006 </xref> ). Enhanced disease <span class="trait">resistance</span> was also observed in <italic> A. thaliana </italic> wrky11 and wrky17 single mutants and enhanced <span class="trait">resistance</span> was observed in wrky11-wrky17 double mutants infected with <italic> P. syringae </italic> ( <xref ref-type="bibr" rid="b63-grsb-2007-017"> Journot-Catalino et al. 2006 </xref> ). The WRKY gene family is also large in <italic> G. max </italic> . It is likely that complex gene expression patterns will emerge as we better understand <italic> G. max </italic> infection by <italic> H. <span class="trait">glycines</span> </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Microarray Detection Call Methodology as a Means to Identify and Compare Transcripts Expressed within Syncytial Cells from Soybean ( <italic> Glycine max </italic> ) Roots Undergoing Resistant and Susceptible Reactions to the Soybean Cyst Nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> )</h2>
			<h3>PMID:20508855</h3>
				
					<h3>PMCID:2875038</h3>
					<h4>期刊:Journal of Biomedicine and Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P. </given-names>  </name>  <xref ref-type="aff" rid="I1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Overall </surname>  <given-names> Christopher C. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2, 3, 4 </sup>  </xref>  <xref ref-type="aff" rid="I3"/>  <xref ref-type="aff" rid="I4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alkharouf </surname>  <given-names> Nadim W. </given-names>  </name>  <xref ref-type="aff" rid="I5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 2 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The genetic basis of <italic> G. max </italic> to overcome <italic> H. <span class="trait">glycines</span> </italic> infection (an incompatible reaction resulting in <span class="trait">resistance)</span> is complex (reviewed in [ <xref ref-type="bibr" rid="B18"> 15 </xref> ]). Several recessive <span class="trait">resistance</span> loci ( <italic> rhg1 </italic> , <italic> rhg2, </italic> and <italic> rhg3 </italic> ) [ <xref ref-type="bibr" rid="B14"> 16 </xref> ] and dominant <span class="trait">resistance</span> loci ( <italic> <span class="gene">Rhg4</span> </italic> ) [ <xref ref-type="bibr" rid="B67"> 17 </xref> ] and ( <italic> Rhg5 </italic> ) [ <xref ref-type="bibr" rid="B81"> 18 </xref> ] have been identified (reviewed in [ <xref ref-type="bibr" rid="B18"> 15 </xref> ]). The understanding of <span class="trait">resistance</span> to <italic> H. <span class="trait">glycines</span> </italic> has also been aided by other genetic marker technology (e.g., quantitative trait loci (QTL) mapping). Those studies have identified QTLs that map to 17 linkage groups. <italic> G. </italic> <italic> m </italic> <italic> a </italic> <italic> x </italic> <sub> [PI&#x02009;&#x02009;548402/Peking] </sub> has nine QTLs that map to different linkage groups (reviewed in [ <xref ref-type="bibr" rid="B18"> 15 </xref> ]). One of those QTLs present in <italic> G. </italic> <italic> m </italic> <italic> a </italic> <italic> x </italic> <sub> [PI&#x02009;&#x02009;548402/Peking] </sub> that maps to linkage group G explains more than 50% of <span class="trait">resistance</span> to <italic> H. <span class="trait">glycines</span> </italic> [ <xref ref-type="bibr" rid="B92"> 19 </xref> ]. It also is responsible for <span class="trait">resistance</span> to several different populations of <italic> H. <span class="trait">glycines</span> </italic> . The major QTL that is located on linkage group G was identified using the RFLP marker C006V and is designated as <italic> rhg1 </italic> [ <xref ref-type="bibr" rid="B17"> 20 </xref> ]. Further studies have shown that molecular marker Satt309 is only 0.4 centiMorgans from <italic> rhg1 </italic> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]. Importantly, much of the <span class="trait">resistance</span> that has been bred into elite <italic> G. max </italic> varieties originates from the <italic> G. </italic> <italic> m </italic> <italic> a </italic> <italic> x </italic> <sub> [PI&#x02009;&#x02009;548402/Peking] </sub> genotype.
					</p>
				
					<p>&nbsp;&nbsp;
						The DCM analyses identified genes that were present only in the incompatible syncytial cell sample as compared to the pericycle sample. The genes included various defense response genes. Some of these were DRRG49-C (CF809336), Pto-interacting-kinase (Pti) (BI970555), BOTRYTIS-INDUCED KINASE 1 (BIK1) (AW311265), and two leucine-rich repeat genes (LRRs) (AW348555, AW317282). LRRs near <italic> rhg1 </italic> exist [ <xref ref-type="bibr" rid="B116"> 65 </xref> ]. The DRRG49-C gene is induced during pathogen attack [ <xref ref-type="bibr" rid="B16"> 66 </xref> ]. The Pti-kinase [ <xref ref-type="bibr" rid="B115"> 67 </xref> ] and BIK1 [ <xref ref-type="bibr" rid="B106"> 68 </xref> ] are examples of genes that are induced during a response to pathogenic attack and are involved in important defense responses. The identification of a Pti-like kinase was particularly interesting. In <italic> L. esculentum </italic> , <italic> Pti4 </italic> and <italic> Pti5 </italic> are induced by the virulent <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> , the nonhost pathogenic bacterium <italic> Xanthomonas oryzae </italic> pv. <italic> oryzae </italic> (strain PXO <sup> A </sup> &#x02009;&#x02009; <italic> avrXa10 </italic> ), and the nonpathogenic bacterium <italic> Pseudomonas fluorescens </italic> (strain 2&#x02013;79) [ <xref ref-type="bibr" rid="B101"> 69 </xref> ]. Interestingly, Pti kinases are observed at 3&#x02009;dpi in both compatible and incompatible reactions. The Pti-kinase identified in the 3&#x02009;dpi incompatible reaction was most closely related to <italic> <span class="gene">Pti1</span> </italic> isolated from tomato [ <xref ref-type="bibr" rid="B119"> 70 </xref> ]. In that analysis, <italic> <span class="gene">Pti1</span> </italic> was shown to be phosphorylated by <italic> Pto </italic> and to be involved in the hypersensitive response [ <xref ref-type="bibr" rid="B119"> 70 </xref> ]. The LRR genes have a long history as being important for plant defense [ <xref ref-type="bibr" rid="B9"> 71 </xref> , <xref ref-type="bibr" rid="B49"> 72 </xref> ]. The genes also have been shown to confer <span class="trait">resistance</span> to parasitic nematodes [ <xref ref-type="bibr" rid="B13"> 73 </xref> &#x02013; <xref ref-type="bibr" rid="B43"> 78 </xref> ]. Thus, due to the transient nature of expression of some of these genes in other systems, it is not surprising that they were not identified as being differentially expressed in syncytium samples [ <xref ref-type="bibr" rid="B56"> 26 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative trait loci underlying <span class="trait">resistance</span> to sudden death syndrome (SDS) in MD96-5722 by &#x02018;Spencer&#x02019; recombinant inbred line population of soybean</h2>
			<h3>PMID:未知</h3>
				
					<h3>PMCID:4362743</h3>
					<h4>期刊:3 Biotech</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> J. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Akond </surname>  <given-names> M. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kassem </surname>  <given-names> M. A. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Kantartzi </surname>  <given-names> S. K. </given-names>  </name>  <address>  <email> kantart@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Plant, Soil and Agricultural Systems, Southern Illinois University, Carbondale, IL 62901-4415 USA  </aff>  <aff id="Aff2">  <label/> Plant Genomics and Biotechnology Lab, Department of Biological Sciences, Fayetteville State University, Fayetteville, NC 28301-4298 USA  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although simple sequence repeat (SSR) markers were used successfully for localizing QTL in soybeans (Hnetkovsky et al. 1995; Kassem et al. <xref ref-type="bibr" rid="CR21"> 2006 </xref> ; Kazi et al. <xref ref-type="bibr" rid="CR24"> 2008 </xref> ), single nucleotide polymorphism (SNP) markers are the most ample genetic markers available for taking research to the next level (Nicod and Largiader <xref ref-type="bibr" rid="CR25"> 2003 </xref> ; <span class="gene">Fan</span> et al. <xref ref-type="bibr" rid="CR10"> 2003 </xref> ; Barbazuk et al. <xref ref-type="bibr" rid="CR4"> 2007 </xref> ; Hsu et al. <xref ref-type="bibr" rid="CR15"> 2008 </xref> ). Currently, very few SNP markers/QTL are available for SDS <span class="trait">resistance</span> in soybean. Recently, Kassem et al. ( <xref ref-type="bibr" rid="CR23"> 2012 </xref> ) mapped 14 significant QTL linked to SDS <span class="trait">resistance</span> through a high-density SNP-based genetic linkage map using PI438489B by &#x02018;Hamilton&#x02019; RIL population of soybean. Another SNP-based high-density genetic linkage map (Akond et al. <xref ref-type="bibr" rid="CR1"> 2013 </xref> ) was constructed using MD 96-5722&#x000a0;&#x000d7;&#x000a0;&#x02018;Spencer&#x02019; RIL population to map QTL for SDS <span class="trait">resistance.</span> Therefore, the objective of this research was to identify QTL conferring SDS <span class="trait">resistance</span> in a population advanced from a cross between soybean line MD 96-5722 and variety &#x02018;Spencer&#x02019;.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Soybean MicroRNAs Involved in Soybean Cyst Nematode Infection by Deep Sequencing</h2>
			<h3>PMID:22802924</h3>
				
					<h3>PMCID:3384596</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiaoyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xue </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shaopeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dawei </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Duan </surname>  <given-names> Yuxi </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs (miRNAs) are a class of 21&#x02013;24 nucleotide long non-coding RNAs that negatively regulate gene expression in animals and plants <xref ref-type="bibr" rid="pone.0039650-Carthew1"> [14] </xref> . MiRNAsrole in plant was first demonstrated in plant growth, development, hormone signaling, and RNA metabolism <xref ref-type="bibr" rid="pone.0039650-Nonogaki1"> [15] </xref> , <xref ref-type="bibr" rid="pone.0039650-Liu1"> [16] </xref> . Recently, it was reported that miR393 was induced by a bacterial flagellin-derived PAMP, Flg22 in <italic> Arabidopsis </italic> . MiR393negatively regulates the expression level of the F-box auxin receptor genes and restricts <italic> Pseudomonas syringae </italic> growth. Combined with race-specific <span class="trait">resistance</span> test, the results suggest that miR393 has a role in imparting basal <span class="trait">resistance</span> but not race-specific <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0039650-Navarro1"> [17] </xref> . Researchers find 10 miRNAs are down-regualted in loblolly pine response to <span class="trait">rust</span> fungus <xref ref-type="bibr" rid="pone.0039650-Lu1"> [18] </xref> . Two miRNAs, bra-miR158 and bra-miR1885, were greatly upregulated when <italic> Brassica rapa </italic> was infected by Turnip mosaic virus <xref ref-type="bibr" rid="pone.0039650-He1"> [19] </xref> . Genome-wide small RNA profiling analysis from nodules or from roots inoculated with symbiotic rhizobia identified a number of nodulation-associated miRNAs from different legumes <xref ref-type="bibr" rid="pone.0039650-Subramanian1"> [20] </xref> , <xref ref-type="bibr" rid="pone.0039650-Wang2"> [21] </xref> , <xref ref-type="bibr" rid="pone.0039650-LelandaisBriere1"> [22] </xref> , <xref ref-type="bibr" rid="pone.0039650-ArenasHuertero1"> [23] </xref> , <xref ref-type="bibr" rid="pone.0039650-Simon1"> [24] </xref> . Mtr-miR166, Mtr-miR169 of <italic> M. truncatula </italic> and three soybean miRNAs, <span class="gene">Gma-miR482,</span> <span class="gene">Gma-miR1511,</span> <span class="gene">Gma-miR1512</span> have been functionally analysed and associated to rhizobial symbiosis <xref ref-type="bibr" rid="pone.0039650-Boualem1"> [25] </xref> , <xref ref-type="bibr" rid="pone.0039650-Combier1"> [26] </xref> , <xref ref-type="bibr" rid="pone.0039650-Li1"> [27] </xref> . These increasing evidences show that miRNAs are involved in plants response to biotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						MiR319 was shown to be drought-responsive in <italic> Arabidopsis </italic> , rice, and sugarcane, while we firstly report miR319 was involved in biotic stress <xref ref-type="bibr" rid="pone.0039650-Sunkar2"> [61] </xref> , <xref ref-type="bibr" rid="pone.0039650-Lv1"> [62] </xref> , <xref ref-type="bibr" rid="pone.0039650-Thiebaut1"> [63] </xref> . <span class="gene">Gma-miR319</span> was the single up-regulated conserved miRNA family in both two cultivars. In contrast to deeply conserved miRNA families, most young miRNAs were weakly expressed, more divergent, and tend to lack targets. Thus, function information about non-conserved miRNA is rare. The 20 legume- or soybean specific miRNAs related to SCN infection were identified in current study, suggesting a number of non-conserved miRNAs function as regulators of genes in plant genomes, and play an important role in plant pathogen defense. The complex plant response to SCN infection spans the early migratory stages during penetration and migration through roots into the later sedentary stages of syncytium induction and feeding which have inspired numberous studies to identify genes involved in SCN. Here we report the miRNAs related to SCN infection. In previous study, 43 SCN-responsive miRNAs were identified. More than 80% of miRNAs were suppressed by SCN infection in both cultivars. Similarity, 10 of 11 miRNAs were down-regulated in the stem galls induced by the <span class="trait">rust</span> fungus <italic> Cronartium quercuum </italic> f. sp. <italic> Fusiforme </italic> <xref ref-type="bibr" rid="pone.0039650-Lu1"> [18] </xref> . While in <italic> Arabidopsis </italic> - <italic> H. schachtii </italic> interaction study, all the 14 differentially expressed miRNAs were statistically significantly down-regulated at the 4 dpi (day post-inoculation) while at 7 dpi, 5 miRNAs were up-regulated and 7 were down-regulated <xref ref-type="bibr" rid="pone.0039650-Hewezi1"> [59] </xref> . Indicating the tested miRNAs have complex expression profiles at different stage of cyst nematode infection which is consistent with gene expression pattern during SCN-infection <xref ref-type="bibr" rid="pone.0039650-Ithal1"> [9] </xref> , <xref ref-type="bibr" rid="pone.0039650-Ithal2"> [10] </xref> . Targets of SCN-responsive miRNAs include transcription factors, general stress-responsive genes, disease <span class="trait">resistance</span> responsive proteins, PR proteins. All these genes expression pattern were checked in previous studies, about 50% miRNAs showed negative correlations between miRNA accumulation and target gene mRNA abundance <xref ref-type="bibr" rid="pone.0039650-Li2"> [60] </xref> . These discrepancies suggest that other regulatory mechanisms could be involved in regulating these genes.
					</p>
				
					<p>&nbsp;&nbsp;
						MiR319 was the single up-regulated conserved miRNA family in both two cultivars, which was shown to be drought-responsive in <italic> Arabidopsis </italic> , rice, and sugarcane <xref ref-type="bibr" rid="pone.0039650-Sunkar2"> [61] </xref> , <xref ref-type="bibr" rid="pone.0039650-Lv1"> [62] </xref> , <xref ref-type="bibr" rid="pone.0039650-Thiebaut1"> [63] </xref> . In tomato, miR319 was involved in <italic> Solanum lycopersicum </italic> , and showed opposite expression patterns in leaves <xref ref-type="bibr" rid="pone.0039650-Expression1"> [64] </xref> . The diverse functions of miR319 were due to the target gene TCP that presumably control cell divisions in stress response. Interestingly, during the same stress, several miRNAs had differential express pattern in the two cultivars. MiR169 was up-regulated in the sensitive cultivar, but down-regulated in the tolerant. MiR169 targeted nuclear factor Y subunit, which can regulate expression level of some stress-responsive genes <xref ref-type="bibr" rid="pone.0039650-Li3"> [65] </xref> . MiR169 guided regulation of these transcription factors appeared to be important in variety of abiotic stress, including <span class="trait">drought,</span> cold, salinity, UV-B radiation, and heat stress <xref ref-type="bibr" rid="pone.0039650-Sunkar2"> [61] </xref> , <xref ref-type="bibr" rid="pone.0039650-Lv1"> [62] </xref> , <xref ref-type="bibr" rid="pone.0039650-Thiebaut1"> [63] </xref> , <xref ref-type="bibr" rid="pone.0039650-Li3"> [65] </xref> , <xref ref-type="bibr" rid="pone.0039650-Zhou1"> [66] </xref> . Furthermore, <italic> MtHAP2-1 </italic> , a new transcription factor of the CCAAT-binding family identified in <italic> M. truncatula </italic> , is regulated by miR169. <italic> MtHAP2-1 </italic> is essential for nodule meristematic persistence in <italic> M. truncatula </italic> , and that miR169 confers spatial and temporal accuracy of nodules development <xref ref-type="bibr" rid="pone.0039650-Zhao1"> [67] </xref> . Contrarily, gma-miR390b was induced by SCN in HB and suppressed in L10. MiR390 was involved in auxin signaling and regulated root growth <xref ref-type="bibr" rid="pone.0039650-Marin1"> [68] </xref> . <span class="gene">Gma-miR390c</span> were down-regulated in HB but without significant changes in L10. Expression patterns of MIR390 members are different even in single cultivar which indicating function diversity of this miRNA family. With other 15 miRNAs up- or down-regulated only in one cultivar, the genotype-specific regulation of miRNAs might be part of the reason why the two soybean cultivars had differential tolerance to soybean cyst nematode.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Arabidopsis </italic> genes, <italic> AtNPR1, AtTGA2 </italic> and <italic> AtPR-5 </italic> , confer partial <span class="trait">resistance</span> to soybean cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> ) when overexpressed in transgenic soybean roots</h2>
			<h3>PMID:24739302</h3>
				
					<h3>PMCID:4021311</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ben.matthews@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Beard </surname>  <given-names> Hunter </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hunter.beard@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Brewer </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> eric.brewer@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kabir </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sara_kabir1@hotmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> margaret.macdonald@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Youssef </surname>  <given-names> Reham M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> reham.youssef@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plant parasitic nematodes cause billions of dollars in losses each year worldwide [ <xref ref-type="bibr" rid="B1"> 1 </xref> - <xref ref-type="bibr" rid="B3"> 3 </xref> ]. The root-knot nematode, genus <italic> Meloidogyne </italic> , attacks over 3000 species of plants [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ], while the soybean cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> ) has a much narrower host range and is responsible for almost one billion dollars per year in losses in the US [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B3"> 3 </xref> ]. Although some soybean genotypes are resistant to specific populations of SCN, no soybean genotype is known to be resistant to all SCN populations. Several genes conferring partial <bold> R </bold> esistance to <bold> <italic> H </italic> </bold> <italic> eterodera </italic> <bold> <italic> g </italic> </bold> <italic> lycines </italic> (Rhg) have been mapped, and, recently, genes at the rhg1 and <span class="gene">Rhg4</span> loci have been elucidated [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ].The defense response of soybean to SCN has been examined at the physiological, genetic, and molecular level, and several reports indicate that salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and indole acetic acid (IAA) play a role in <span class="trait">resistance</span> and susceptibility of plants to nematodes [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B10"> 10 </xref> - <xref ref-type="bibr" rid="B15"> 15 </xref> ]. However, the roles of defense-related hormones and specific components of their synthesis and signaling pathways in providing <span class="trait">resistance</span> in plants to nematodes are unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						Little is known of the role of specific components of SA regulation and signaling in the interaction of soybean with the soybean cyst nematode (SCN; <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> ), the major pest of soybean in the US. Although soybean genes conferring <span class="trait">resistance</span> to SCN have been identified, these do not provide <span class="trait">resistance</span> to all SCN populations. <span class="trait">Resistance</span> in soybean to SCN is multigenic, and several major loci for <span class="trait">resistance</span> have been identified [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> - <xref ref-type="bibr" rid="B31"> 31 </xref> ]. For example, in soybean cv Peking, several genes ( <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> , <italic> rhg3 </italic> , and <italic> <span class="gene">Rhg4</span> </italic> ) have been reported that confer <span class="trait">resistance</span> to SCN race 1 [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> ], yet none of these genes confers complete <span class="trait">resistance</span> to all SCN populations. Therefore, we are applying to soybean a portion of the vast knowledge that has been gained from studies on the model plant <italic> Arabidopsis </italic> and its large array of mutants on the role of SA and JA in the plant defense response to identify important components that may be useful in decreasing susceptibility of plants to nematodes, and especially of soybean to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> has been used widely as a model system to study plant defense pathways, usually with bacterial and fungal pathogens [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> - <xref ref-type="bibr" rid="B39"> 39 </xref> ]. Much attention has been focused on SA production and signaling pathways using <italic> Arabidopsis </italic> mutants infected with bacterial and fungal pathogens (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ) [ <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B35"> 35 </xref> - <xref ref-type="bibr" rid="B44"> 44 </xref> ]. For example, when a virulent strain of the biotrophic pathogen, <italic> Pseudomonas syringae </italic> , attacks <italic> Arabidopsis </italic> , the AvrRPS4 effector protein of <italic> P. syringae </italic> secreted by the type III secretion system is detected by the plant receptor RPS4, a Toll-interleukin-nucleotide binding-leucine-rich repeat (TIR-NB-LRR) that mediates the induction of antimicrobial defenses to provide disease <span class="trait">resistance.</span> The nucleo-cytoplasmic protein ENHANCED DISEASE SUSCEPTIBILITY 1 (EDS1), which is a lipase-like protein, connects RPS4 with downstream defense pathways and regulates the accumulation of SA [ <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> ]. EDS1 is essential for production of the hypersensitive response and mobilization of defense pathways [ <xref ref-type="bibr" rid="B47"> 47 </xref> - <xref ref-type="bibr" rid="B49"> 49 </xref> ]. EDS1 can dimerize and interact with another lipase-like protein, phytoalexin deficient 4 (PAD4) [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> ]. Both <italic> EDS1 </italic> and <italic> <span class="gene">PAD4</span> </italic> are required for the accumulation of SA and they are involved in ROS signaling [ <xref ref-type="bibr" rid="B46"> 46 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]. <span class="gene">PAD4</span> protein is required for amplification of weak signals resulting from pathogen infection. Another important component of the defense response is EDS5, a multi-drug transporter member of the <span class="gene">MATE</span> family of transporters [ <xref ref-type="bibr" rid="B50"> 50 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						SA can also be regulated independently of EDS1 and PAD4. NDR1 (NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1) is a positive regulator of SA that works independently of EDS1 and <span class="gene">PAD4</span> [ <xref ref-type="bibr" rid="B55"> 55 </xref> ] NDR1 is an integrin-like protein that can associate with RIN4, while RIN4 can associate with RPM1 and RPS2 [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]. NDR1 may play a role in electrolyte release upon infection of <italic> Arabidopsis </italic> by <italic> P. syringae </italic> , while RIN4 regulates stomatal apertures in conjunction with H&#x02009;+&#x02009;&#x02212;ATPases of the plasma membrane of <italic> Arabidopsis </italic> during pathogen attack [ <xref ref-type="bibr" rid="B57"> 57 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> AtDND1 </italic> gene AT5G15410.1 encodes the cyclic nucleotide-gated cation channel protein DEFENSE NO DEATH 1 (DND1), and is involved in production of the hypersensitive response [ <xref ref-type="bibr" rid="B86"> 86 </xref> ]. The <italic> Arabidopsis dnd1 </italic> mutant produces elevated amounts of SA. Overproduction of <italic> AtDND1 </italic> in soybean roots did not provide <span class="trait">resistance</span> to SCN; rather, it enhanced susceptibility. The FI of transgenic soybean roots containing <italic> AtDND1 </italic> was 175% of the control, the largest increase in susceptibility of the genes tested here (Table&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> ). The protein sequence of DND1 is highly conserved between <italic> Arabidopsis </italic> and soybean <span class="gene">(Glyma18g49890.1)</span> as indicated in Figure&#x000a0; <xref ref-type="fig" rid="F9"> 9 </xref> . It contains a cyclic nucleotide-binding domain as indicated by a significant (e-value&#x02009;=&#x02009;5.8) Pfam-A match.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> to SCN is a multigenic trait and several genetic loci have been mapped [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> - <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B90"> 90 </xref> - <xref ref-type="bibr" rid="B93"> 93 </xref> ]. Recently, the identity of genes residing at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci have been reported [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ] which confer some <span class="trait">resistance</span> to SCN. However, none of these loci alone provides full <span class="trait">resistance</span> to any one SCN population. For example, in a cross between soybean cv Essex and Forrest, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> accounted for about 65% of the variation in <span class="trait">resistance</span> found in the resultant inbred population to SCN [ <xref ref-type="bibr" rid="B94"> 94 </xref> ]. Other soybean genes have been identified that confer partial <span class="trait">resistance</span> to SCN when overexpressed in roots [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B60"> 60 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Examination of <italic> Arabidopsis </italic> mutants has played a key role in our understanding of the defense response and is the subject of many reviews [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> , <xref ref-type="bibr" rid="B39"> 39 </xref> ]. It is postulated that SA can be synthesized through two different biochemical pathways [ <xref ref-type="bibr" rid="B52"> 52 </xref> , <xref ref-type="bibr" rid="B97"> 97 </xref> ]. In the first pathway, chorismate is converted to isochorismate via the action of ICS; then, SA is produced from isochorismate by isochorismate pyruvate lyase. Examination of <italic> ICS1 </italic> mutants, <italic> sid2-1 </italic> and <italic> sid2-2, </italic> of <italic> Arabidopsis </italic> indicate that loss of ICS1 activity dramatically decreases SA levels [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. Most SA synthesized and relevant to plant defense in Arabidopsis appears to be made through this pathway. The <italic> sid2 </italic> mutant does not accumulate SA upon inoculation with <italic> P. syringae </italic> , and <italic> PR-1 </italic> expression is reduced greatly. However, <italic> PR-2 </italic> and <italic> PR-5 </italic> are expressed [ <xref ref-type="bibr" rid="B90"> 90 </xref> ]. The second possible pathway diverts chorismate via CM to phenylalanine, which is converted to cinnamic acid by phenylalanine ammonia lyase (PAL) and progresses through a series of reactions to form SA. Previously, we demonstrated that overexpression in transgenic soybean roots of two different soybean genes encoding <span class="gene">PAL</span> did not greatly affect SCN maturation, with FI values of 94 and 111% [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. However, here we show that overexpression of CM and ICS, representatives of the two different pathways, decrease the FI to 57% and 67% of the control, respectively. However, overexpression of PAL, CM, or ICS alone does not confer <span class="trait">resistance</span> to the level provided by overexpression of several other SA-related genes individually.
					</p>
				
					<p>&nbsp;&nbsp;
						If the SA-related defense response is a major factor in soybean <span class="trait">resistance</span> to SCN, then components regulated by SA may confer <span class="trait">resistance</span> to SCN when overexpressed. Our results show that several <italic> Arabidopsis </italic> genes involved in SA regulation, synthesis, and signaling conferred <span class="trait">resistance</span> to SCN when overexpressed in soybean roots. The <italic> Arabidopsis </italic> genes <italic> AtNPR1, AtTGA2, AtPR-5, </italic> and several others related to SA strongly decreased the FI of SCN in transgenic soybean roots (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). NPR1 is a master regulator of the SA-related defense response, and it is a receptor for SA [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. NPR1 binds SA and interacts with TGA transcription factors, such as the transcription factor AHBP-1b/TGA2, perhaps through its ankyrin domain [ <xref ref-type="bibr" rid="B54"> 54 </xref> ]. NPR1 and TGA transcription factors work downstream of SA and are important to the expression of the genes encoding PR-1, PR-5, and others [ <xref ref-type="bibr" rid="B98"> 98 </xref> - <xref ref-type="bibr" rid="B102"> 102 </xref> ]. Expression of these genes is completely abolished in <italic> Arabidopsis </italic> plants carrying the <italic> npr1 </italic> mutation [ <xref ref-type="bibr" rid="B83"> 83 </xref> ]. Recently, Pant et al. [ <xref ref-type="bibr" rid="B103"> 103 </xref> ] demonstrated that a Gm ortholog, <span class="gene">Glyma09g02430,</span> of <italic> Arabidopsis </italic> NPR1 reduced SCN cysts to approximately 30% of the control, in agreement with our data for overexpression of <italic> AtNPR </italic> (FI&#x02009;=&#x02009;33%) <italic> . </italic> There are reports in numerous plants indicating that overexpression of NPR1 results in defense against fungal and bacterial pathogens. Overexpression of <italic> AtNPR1 </italic> in <italic> Arabidopsis </italic> conferred <span class="trait">resistance</span> to <italic> P. syringae </italic> and <italic> Peronospora parasitica </italic> [ <xref ref-type="bibr" rid="B104"> 104 </xref> ]. Overexpression of <italic> AtNPR1 </italic> in rice conferred <span class="trait">resistance</span> to the rice bacterial blight pathogen <italic> Xanthomonas oryzae </italic> [ <xref ref-type="bibr" rid="B105"> 105 </xref> ]. Overexpression of <italic> AtNPR1 </italic> in wheat conferred <span class="trait">resistance</span> to fusarium head blight, caused by <italic> Fusarium graminearum. </italic> The apple <italic> MpNPR1 </italic> gene confers <span class="trait">resistance</span> to two fungal pathogens of apple, <italic> Venturia inaequalis </italic> and <italic> Gymnosporangium juniper-virginianae </italic> [ <xref ref-type="bibr" rid="B106"> 106 </xref> , <xref ref-type="bibr" rid="B107"> 107 </xref> ] complemented <italic> Arabidopsis npr1-1 </italic> mutants with soybean homologs <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> NPR1-2, </italic> and PR-1 was induced in the transformed plants after infection with <italic> P. syringae </italic> and after treatment with the SAR inducer, 2,6-dichloroisonicotinic acid.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">PAD4</span> is found upstream of SA production ([ <xref ref-type="bibr" rid="B113"> 113 </xref> ]; Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). <span class="gene">PAD4</span> is a lipase like protein [ <xref ref-type="bibr" rid="B48"> 48 </xref> ] that can form molecular complexes with EDS1 to modulate SA defense signaling [ <xref ref-type="bibr" rid="B114"> 114 </xref> ]. Ectopic expression of <italic> <span class="gene">PAD4</span> </italic> reduces feeding time of green peach aphids on transgenic <italic> Arabidopsis </italic> plants. The aphid spends more time actively feeding on <italic> pad4 </italic> mutants [ <xref ref-type="bibr" rid="B115"> 115 </xref> ]. Previously, we showed that <italic> AtPAD4 </italic> conferred <span class="trait">resistance</span> to SCN and RKN [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. EDS1 is also upstream of SA, and it can interact with <span class="gene">PAD4</span> [ <xref ref-type="bibr" rid="B114"> 114 </xref> , <xref ref-type="bibr" rid="B116"> 116 </xref> , <xref ref-type="bibr" rid="B117"> 117 </xref> ]. Recently, Pant et al. [ <xref ref-type="bibr" rid="B103"> 103 </xref> ] demonstrated that <italic> <span class="gene">GmEDS1</span> </italic> <span class="gene">(Glyma06g19890)</span> had a great effect on SCN and reduced SCN cysts by approximately 80%. This is in contrast to our data indicating that overexpression of <italic> AtEDS1 </italic> did not decrease the FI of SCN. <span class="gene">GmEDS1</span> is composed of 620 aa and AtEDS1 is composed of 623 aa (Figure&#x000a0; <xref ref-type="fig" rid="F10"> 10 </xref> ). The amino acid sequences of <span class="gene">GmEDS1</span> and AtEDS1 have 239 aa in common. Furthermore, they have another 140 aa that are closely related substitutions. Apparently, this conservation is not enough for AtEDS1 to provide <span class="trait">resistance</span> to SCN as did <span class="gene">GmEDS1.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of several <italic> Arabidopsis </italic> genes provided protection to soybean against SCN. In fact, several genes provided increases in <span class="trait">resistance</span> comparable to or better than that provided by <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , two naturally occurring <span class="trait">resistance</span> gene loci in soybean. However, not all <italic> Arabidopsis </italic> genes provided <span class="trait">resistance.</span> In fact, overexpression of <italic> AtEDS1 </italic> did not increase <span class="trait">resistance,</span> although a <italic> <span class="gene">GmEDS1</span> </italic> ortholog was recently reported as providing <span class="trait">resistance.</span> These data indicate that some <italic> Arabidopsis </italic> genes can be used directly in soybean to confer <span class="trait">resistance,</span> especially genes associated with SA regulation, signaling, and synthesis, but not all <italic> Arabidopsis </italic> orthologs will provide the same results as the orthologous soybean gene. These and similar studies may provide useful insights into protein conservation and function, and several of these <italic> Arabidopsis </italic> genes may prove useful in engineering plants with broad <span class="trait">resistance</span> to nematodes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole-genome gene expression profiling revealed genes and pathways potentially involved in regulating interactions of soybean with cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe)</h2>
			<h3>PMID:25880563</h3>
				
					<h3>PMCID:4351908</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wan </surname>  <given-names> Jinrong </given-names>  </name>  <address>  <email> wanj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Vuong </surname>  <given-names> Tri </given-names>  </name>  <address>  <email> vuongt@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiao </surname>  <given-names> Yongqing </given-names>  </name>  <address>  <email> yongqing.jiao@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hongxin </given-names>  </name>  <address>  <email> jackson.zhang.828@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Current address: Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan, Hubei 430062 China  </aff>  <aff id="Aff3">  <label/> Department of Computer Sciences, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label/> Informatics Institute and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 3 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Annual soybean <span class="trait">yield</span> losses caused by soybean cyst nematode (SCN, <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe) in the United States alone were estimated at 1.5 billion dollars [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. One of the effective ways to control this disease is through deployment of genetic <span class="trait">resistance</span> in soybean cultivars. Two of the major sources of <span class="trait">resistance</span> employed in breeding programs are derived from soybean lines PI 88788 and Peking (PI 548402), especially the former [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. Recently, the genes underlying two major QTL (quantitative trait loci), <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , responsible for <span class="trait">resistance</span> to SCN in the soybean lines, have been cloned [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. However, due to the overuse of the <span class="trait">resistance</span> from these sources, SCN populations showing increasing reproduction on resistant soybean cultivars have emerged [ <xref ref-type="bibr" rid="CR5"> 5 </xref> - <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Thus, new sources of <span class="trait">resistance</span> are needed for sustainable soybean production. For this purpose, a number of studies have screened plant introductions (PIs) in the USDA Soybean Germplasm Collection for new sources of <span class="trait">resistance</span> to SCN. As a result, new resistant lines were identified (e.g., [ <xref ref-type="bibr" rid="CR8"> 8 </xref> - <xref ref-type="bibr" rid="CR12"> 12 </xref> ]). Among these identified resistant lines, PI 437654 and PI 567516C were highly resistant to multiple SCN races [Races 1 (HG Type 2.5.7), 2 (HG Type 1.2.5.7), 3 (HG Type 0), 5 (HG Type 2.5.7), and 14 (HG Type 1.3.5.6.7)] [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR11"> 11 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> , <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Different from PI 437654, PI 567516C is also highly resistant to the synthetic nematode population LY1, which was derived from a mass mating of SCN races 2 (HG Type 1.2.5.7) and 3 (HG Type 0) [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. This uniqueness is in agreement with the findings that PI 567516C is genetically different from most other SCN resistant lines (including PI 88788 and Peking) [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and contains two novel QTL, which were genetically mapped on chromosomes (Chr.) 10 and 18 for <span class="trait">resistance</span> to multiple SCN races [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Although PI 437654 was shown to contain two known QTL ( <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> ), a new QTL for <span class="trait">resistance</span> to SCN HG types 0, 2.5.7, and 1.3.5.6.7 (PA 3, 5, and 14, respectively), was mapped to Chr. 20 [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. Therefore, PI 437654 and PI 567516C are two important sources with <span class="trait">resistance</span> to multiple SCN HG Types, different from most other SCN resistant lines, including PI 88788 and Peking.
					</p>
				
					<p>&nbsp;&nbsp;
						To reveal potentially important pathways or groups of genes involved in <span class="trait">resistance</span> to SCN, the constitutively-regulated 297 genes were grouped into the following functional categories using MapMan [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]: cell wall metabolism (seven genes), lipid metabolism (seven genes), secondary metabolism (five genes), abiotic stress (six genes), biotic stress (24 genes), transcription regulation (transcription factors, TFs, 24 genes), signaling (16 genes), hormonal pathways (eight genes), protein modification and degradation (11 genes), transport (12 genes), development (8 genes), enzyme families (25 genes: 4 UDP glucosyl and glucoronyl transferase genes, 3 cytochrome <span class="gene">P450</span> genes, etc.), other groups of genes (46 genes: 6 genes in RNA processing, 3 genes in DNA synthesis, and 3 genes in cell organization ,etc.), and unclassified genes (82 genes) (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ; see Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> ). It is worth noting that 23 nucleotide-binding site-leucine-rich repeat (NBS-LRR) <span class="trait">resistance</span> genes and one leucine-rich repeat receptor-like kinase (in the biotic stress category), seven protease and ubiquitin ligase genes (in the protein modification and degradation category), 24 TF genes (in the transcription regulation category), and four jasmonate pathway genes (in the hormone category) were constitutively regulated in the resistant PI lines, suggesting a possible role in regulating <span class="trait">resistance</span> to multiple SCN races in these soybean lines. <fig id="Fig3"> <label> Figure 3 </label> <caption> <p> <bold> Functional categories of the constitutively-regulated genes. </bold> The 297 genes constitutively regulated (at least 2 fold, <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05) in the two resistant lines were categorized using MapMan.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes constitutively regulated (commonly) in both resistant PI lines: Among the 297 genes constitutively regulated in the two resistant PI lines, 34 genes were up-regulated and 51 genes down-regulated commonly in both resistant PI lines (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ; see Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). For example, <span class="gene">Glyma.01G046900</span> <span class="gene">(Glyma01g05710)</span> and <span class="gene">Glyma.03G034400</span> <span class="gene">(Glyma03g04030),</span> both encoding an NBS-LRR type <span class="trait">resistance</span> protein, were highly up-regulated in both resistant lines, and genes <span class="gene">Glyma.08G318000</span> <span class="gene">(Glyma08g43020)</span> and <span class="gene">Glyma.08G317400</span> <span class="gene">(Glyma08g42930),</span> both also encoding an NBS-LRR type <span class="trait">resistance</span> protein, were significantly down-regulated in both resistant lines (see Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). These commonly regulated defense genes may play a common role, possibly in defense against SCN, in both resistant PI lines. <fig id="Fig4"> <label> Figure 4 </label> <caption> <p> <bold> Venn diagrams to show commonality and uniqueness of the constitutively-regulated genes between the two resistant lines. </bold> In total, 297 genes were constitutively regulated in the two resistant soybean lines (at least 2 fold, <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						Genes uniquely regulated by the SCN inoculation in individual genotypes (at 3 dpi): In addition to the commonly regulated genes by SCN in multiple soybean lines, 77 genes were up-regulated and 58 genes down-regulated only in cv. Magellan (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ; see Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ); 363 genes were up-regulated and 81 genes down-regulated only in PI 437654 (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ; see Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ); and 94 genes were up-regulated and 65 genes down-regulated only in PI 567516C (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ; see Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ). For example, <span class="gene">Glyma.06G275100</span> <span class="gene">(Glyma06g42750,</span> encoding a papain family cysteine protease) was up-regulated only in cv. Magellan, and <span class="gene">Glyma.04G247500</span> (Glyma 04&#x000a0;g42640, encoding a F-box domain protein) was down-regulated only in Magellan; <span class="gene">Glyma.05G148300</span> <span class="gene">(Glyma05g28090,</span> encoding a calmodulin-binding protein) was up-regulated only in PI 437654), and <span class="gene">Glyma.08G182100</span> <span class="gene">(Glyma08g19410,</span> encoding a Cytochrome P450), was down-regulated only in PI 437654; <span class="gene">Glyma.20G148600</span> <span class="gene">(Glyma20g28680,</span> encoding a inositol 5-phosphatase) was up-regulated, and <span class="gene">Glyma.07G010800</span> <span class="gene">(Glyma07g01300,</span> encoding a EF hand protein) was down-regulated only in PI 567516C. These uniquely regulated genes possibly contribute to the observed difference in <span class="trait">resistance</span> to SCN in these soybean lines.
					</p>
				
					<p>&nbsp;&nbsp;
						Thirty genes that were regulated constitutively in the two resistant PIs were also responsive to the SCN inoculation (see Additional file <xref rid="MOESM5" ref-type="media"> 5 </xref> ), e.g., <span class="gene">Glyma.08G256600</span> <span class="gene">(Glyma08g28750,</span> encoding an allene oxide cyclase) and <span class="gene">Glyma.09G053700</span> <span class="gene">(Glyma09g05910,</span> encoding an ankyrin repeat-containing protein). These genes possibly play an important role in mediating the soybean-SCN interactions, and therefore may be good targets to manipulate for enhancing soybean <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Defense genes: Similar to published microarray studies [ <xref ref-type="bibr" rid="CR23"> 23 </xref> - <xref ref-type="bibr" rid="CR36"> 36 </xref> ], our present work revealed that 47 defense genes were regulated either constitutively or by SCN (see Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> and Additional file <xref rid="MOESM10" ref-type="media"> 10 </xref> ). Among them are five genes [e.g., <span class="gene">Glyma.02G042500</span> <span class="gene">(Glyma02g04820)</span> and <span class="gene">Glyma.15G206800</span> <span class="gene">(Glyma15g25060)]</span> encoding proteins similar to Arabidopsis pathogenesis-related (PR) proteins, one gene <span class="gene">[Glyma.06G187300</span> <span class="gene">(Glyma06g19900)]</span> encoding an protein similar to Arabidopsis EDS1 (enhanced disease susceptibility 1), one gene <span class="gene">[Glyma.06G162300</span> <span class="gene">(Glyma06g17030)]</span> encoding a protein similar to Arabidopsis RBOHD (respiratory burst oxidase protein D), one gene <span class="gene">[Glyma.15G209300</span> <span class="gene">(Glyma15g26790)]</span> encoding a protein similar to Arabidopsis PGIP1 (polygalacturonase inhibiting protein 1, and five genes [e.g., <span class="gene">Glyma.08G3413000</span> <span class="gene">(Glyma08g45510)</span> and <span class="gene">Glyma.08G341400</span> <span class="gene">(Glyma08g45520)]</span> encoding trypsin and protease inhibitor family proteins. Most importantly, 28 genes [e.g., <span class="gene">Glyma.06G268600</span> <span class="gene">(Glyma06g41880)</span> and <span class="gene">Glyma.06G311100</span> <span class="gene">(Glyma06g46810)]</span> encode NBS-LRR type disease <span class="trait">resistance</span> proteins, accounting for a significant number of the total 319 NBS-LRR <span class="trait">resistance</span> proteins encoded by the soybean genome [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. All this information together supports that defense genes, especially NBS-LRR <span class="trait">resistance</span> genes, are important in mediating soybean <span class="trait">resistance</span> to SCN. The defense genes revealed in the present study are possibly good targets to regulate for soybean defense against SCN. Indeed, Matthews and colleagues recently showed that the over-expression of <span class="gene">Glyma.05G204600</span> <span class="gene">(Glyma05g38130,</span> encoding a thaumatin-like PR protein) significantly enhanced soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription factors (TFs): TFs are important regulators of gene expression and are involved in plant defense against pathogens [ <xref ref-type="bibr" rid="CR44"> 44 </xref> , <xref ref-type="bibr" rid="CR45"> 45 </xref> ] as well as in many other events. Overall, 82 TF genes were found regulated by the SCN inoculation in the current work (see Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). Additionally, 24 TF genes were regulated constitutively in the resistant lines (three of them were also regulated by SCN) (see Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). These TF genes were primarily from the following TF families: MYB (myeloblastosis) domain-containing proteins (19 genes), basic helix-loop-helix (bHLH, 16 genes), APETALA2-ethylene-responsive element binding proteins (AP2-EREBP, 12 genes), WRKY domain transcription factors (12 genes), C2C2 zinc finger proteins (10 genes), and C2H2 zinc finger proteins (9 genes). For example, <span class="gene">Glyma.03G002300</span> <span class="gene">(Glyma03g00460),</span> <span class="gene">Glyma.14G103100</span> <span class="gene">(Glyma14g11960),</span> and <span class="gene">Glyma.16G219800</span> <span class="gene">(Glyma16g34590)</span> (all encoding a WRKY TF) were significantly induced by SCN in all three genotypes; <span class="gene">Glyma.19G262700</span> <span class="gene">(Glyma19g45200,</span> encoding an AP2-EREBP TF) was constitutively up-regulated, and <span class="gene">Glyma.13G321900</span> <span class="gene">(Glyma13g39620,</span> encoding a putative TF) was constitutively down-regulated in both resistant lines. Considering that many similar TFs have been shown to be involved in plant defense against different pathogens in various plants [ <xref ref-type="bibr" rid="CR46"> 46 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> ], these soybean TF genes deserve further investigation to explore their possible role in mediating soybean <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Protein degradation: Although many microarray studies have been done on soybean-SCN interactions, only limited information was revealed in these studies about protein ubiquitination and degradation during such interactions (e.g., [ <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR32"> 32 </xref> ]), likely due to the limited coverage of the soybean genome by the arrays used in those studies. Surprisingly, in the present study, we found that 64 genes potentially involved in protein degradation were regulated by SCN and another seven genes were constitutively regulated in two resistant lines (see Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> and Additional file <xref rid="MOESM11" ref-type="media"> 11 </xref> ). For example, <span class="gene">Glyma.07G196500</span> <span class="gene">(Glyma07g31630,</span> encoding a ubiquitin-protein ligase) was constitutively up-regulated in both resistant PI lines, and <span class="gene">Glyma.09G023700</span> <span class="gene">(Glyma09g02760,</span> encoding a ubiquitin) was constitutively down-regulated in both resistant PI lines; <span class="gene">Glyma.18G242900</span> <span class="gene">(Glyma18g47820,</span> encoding a serine carboxypeptidase) and <span class="gene">Glyma.09G243700</span> <span class="gene">(Glyma09g37910,</span> encoding a subtilase) were induced in all three soybean genotypes; and <span class="gene">Glyma.02G264200</span> <span class="gene">(Glyma02g43190,</span> encoding a U-box domain-containing family protein) and <span class="gene">Glyma.06G174800</span> <span class="gene">(Glyma06g18390,</span> encoding a cysteine proteinase) were down-regulated, especially at 3 dpi, in all three genotypes. Protein degradation is implicated in mediating plant-pathogen interactions as well as in many other processes [ <xref ref-type="bibr" rid="CR48"> 48 </xref> - <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. However, very little is known about their involvement in mediating plant-nematode interactions. Further investigation of the genes revealed in the present work is needed to confirm their potential roles in mediating soybean-SCN interactions, and such studies could lead to effective ways to enhance soybean <span class="trait">resistance</span> to SCN through modulating protein degradation during soybean-SCN interactions.
					</p>
				
					<p>&nbsp;&nbsp;
						The phenylpropanoid pathway: The phenylpropanoid pathway plays an important role in plant growth and development as well as in biotic and abiotic stress responses, likely through regulating the formation of lignin, flavonoids, phytoalexins, etc. [ <xref ref-type="bibr" rid="CR60"> 60 </xref> - <xref ref-type="bibr" rid="CR63"> 63 </xref> ]. Our current microarray data showed that 20 genes potentially involved in this pathway were regulated by the SCN inoculation (see Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> and Additional file <xref rid="MOESM12" ref-type="media"> 12 </xref> ). For example, <span class="gene">Glyma.01G187700</span> <span class="gene">(Glyma01g39460)</span> and <span class="gene">Glyma.06G286600</span> <span class="gene">(Glyma06g43970)</span> (both encoding an O-methyltransferase) were significantly induced in all three lines by SCN. Consistent with our findings, previous microarray work also revealed multiple genes in the phenylpropanoid pathway regulated by the SCN inoculation [ <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR30"> 30 </xref> ]; however, our present work revealed more genes, once again, likely due to the broader coverage of the soybean genome by the array used in our study. Additionally, an early report by Edens et al. [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ] showed that transcription of the genes encoding phenylalanine ammonia lyase and 4-coumaryl CoA ligase and the activities of these enzymes increased in resistant, but not in susceptible, soybean cultivars after nematode infection. And such increase in transcription and enzymatic activities led to increased synthesis of glyceollin, a phytoallaxin that inhibits multiple soybean pathogens [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. Therefore, the phenylpropanoid pathway possibly plays a critical role in mediating soybean <span class="trait">resistance</span> to SCN, and further investigation is needed to understand how this pathway is involved in the process.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg1</span> </italic> and <italic> Rgh4 </italic> are the two major QTL involved in soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. Our current data showed that two out of three genes at the <span class="gene">Rhg1</span> locus [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ], <span class="gene">Glyma.18G022400</span> <span class="gene">(Glyma18g02580,</span> encoding a predicted amino acid transporter) and <span class="gene">Glyma18g02610</span> [encoding a protein with a WI12 (wound-inducible protein 12) region; not present in the newest soybean genome annotation], had a higher expression level in both resistant PI lines than in the susceptible cv. Magellan (2.4&#x02009;~&#x02009;4.5-fold higher) (see Additional file <xref rid="MOESM14" ref-type="media"> 14 </xref> ). However, the third gene, <span class="gene">Glyma.18G022500</span> <span class="gene">(Glyma18g02590,</span> encoding an &#x003b1;-SNAP (&#x003b1;-soluble N-ethylmaleimide-sensitive factor attachment protein) vesicle-trafficking protein), was not significantly more expressed in the resistant lines, suggesting that the <italic> <span class="gene">Rhg1</span> </italic> loci in these two resistant PIs may be different from the <italic> <span class="gene">Rhg1</span> </italic> locus present in PI 88788, which shows significantly elevated expression of all three genes due to 10 copies of the 31-kb <italic> <span class="gene">Rhg1</span> </italic> repeat [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. Consistent with this, the recent study by Cook et al. [ <xref ref-type="bibr" rid="CR69"> 69 </xref> ] showed that there were only 3 copies of the <italic> <span class="gene">Rhg1</span> </italic> repeat at the <italic> <span class="gene">Rhg1</span> </italic> locus in PI 437654 and the genes encoded by the <italic> <span class="gene">Rhg1</span> </italic> locus were moderately expressed relative to PI 88788. Therefore, our current data, together with findings by other researchers may help explain why <italic> <span class="gene">Rhg1</span> </italic> loci from soybean PI 437654 and PI 88788 respond differentially to SCN isolates [ <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. <italic> <span class="gene">Rhg4</span> </italic> gene, <span class="gene">Glyma.08G108900</span> <span class="gene">(Glyma08g11490),</span> encodes a serine hydroxymethyltransferase [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. We found that this gene was constitutively expressed in the resistant line PI 437654 relative to the susceptible cv. Magellan (3.2&#x02009;~&#x02009;4.3-fold higher), but not in the resistant line PI 567516C. This result is in agreement with the mapping data that showed the presence of the <italic> <span class="gene">Rhg4</span> </italic> QTL in PI 437654 [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ] and the absence of the typical <italic> <span class="gene">Rhg4</span> </italic> QTL in PI 567516C [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Further studies, such as re-sequencing the corresponding <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> regions in these PI lines, are needed to elucidate the observed differences. Some efforts are already under way in our laboratory and other laboratories (e.g., [ <xref ref-type="bibr" rid="CR69"> 69 </xref> ]).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of a transformation system for <italic> Hirsutella </italic> spp. and visualization of the mode of nematode infection by GFP-labeled <italic> H. minnesotensis </italic></h2>
			<h3>PMID:26190283</h3>
				
					<h3>PMCID:4507137</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Jingzu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Park </surname>  <given-names> Sook-Young </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Seogchan </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xingzhong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Junzhi </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xiang </surname>  <given-names> Meichun </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> State Key Laboratory of Mycology, Institute of Microbiology, Chinese Academy of Sciences </institution> , No. 3 Park 1, West Beichen Road, Chaoyang District, Beijing 100101,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Department of Plant Pathology and Environmental Microbiology </institution> , Pennsylvania State University, University Park, PA 16802,  <country> USA </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Department of Life Science, Fujian Agriculture and Forestry University </institution> , No. 15 Shangxiadian Road, Cangshan District, Fuzhou 350002,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 14 </day>  <month> 04 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The percentages of nematodes parasitized by AS3.9869 and AS3G1 were analyzed 24 hours after inoculation, which showed no significant differences in their ability to parasitize <italic> H. <span class="trait">glycines</span> </italic> J2 (Hg-J2) and four different larval stages of <italic> C </italic> . <italic> elegans </italic> (Ce-L1, Ce-L2, Ce-L3, and Ce-L4). Error <span class="gene">bar</span> indicates the standard deviations within three replications.
					</p>
				
					<p>&nbsp;&nbsp;
						A, a spore of AS3G1 (a <italic> H </italic> . <italic> minnesotensis </italic> transformant expressing ZsGreen) attached to <italic> H. <span class="trait">glycines</span> </italic> second stage juvenile (J2); B, formation of an infection peg; C, the apex of the infection peg becoming globose; D, initial colonization of coelom by hyphae; E-G, hyphal growth through coelom; H-J, hyphal growth out of coelom; K-L, formation of new conidia. The scale <span class="gene">bar</span> in A-F, H-I, and L corresponds to 20&#x02009;&#x003bc;m, whereas the scale <span class="gene">bar</span> in G, J and K equals 40&#x02009;&#x003bc;m.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Horizontal gene transfer of <italic> acetyltransferases, invertases </italic> and <italic> chorismate mutases </italic> from different bacteria to diverse recipients</h2>
			<h3>PMID:27068610</h3>
				
					<h3>PMCID:4828791</h3>
					<h4>期刊:BMC Evolutionary Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Noon </surname>  <given-names> Jason B. </given-names>  </name>  <address>  <email> jnoon15@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Baum </surname>  <given-names> Thomas J. </given-names>  </name>  <address>  <email> tbaum@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Department of Plant Pathology and Microbiology, Iowa State University, Ames, IA 50011 USA  </aff>
				<br/> 
				年份：<day> 5 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						From our above searches for the three HGT genes in nematode genomes and transcriptomes, we found potentially complex patterns of gene duplications in Hoplolaimina following HGT, in particular for the <italic> INVs </italic> and <italic> CMs </italic> . Also, as mentioned above, INVs are understood to be non-secreted in root-knot nematodes [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ], to be secreted in the nematode digestive system in the potato cyst nematode <italic> G. pallida </italic> [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ], while in the soybean cyst nematode <italic> H. <span class="trait">glycines</span> </italic> , they are believed to be secreted effectors (or at least Hg-GLAND13 [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]). Furthermore, previously reported CMs are relatively small proteins that have been documented in all Hoplolaimna PPN with considerable sequence datasets. However, <span class="gene">GLAND16</span> CMs are over four times larger than other CMs [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ] and in our above searches were only found in cyst nematodes. Therefore, we were interested in elucidating the complex post-HGT evolution of these gene families in Hoplolaimina. In these analyses, we evaluated the subtrees of the Hoplolaimina recipients and bacterial donors specifically within the ML phylogenetic trees that resulted from our comprehensive phylogenetic analyses, re-evaluated the multiple sequence alignments, and evaluated the protein sequences for predicted protein domains, signal peptides and TM regions. Results from these analyses are detailed in the supplementary text, Additional file <xref rid="MOESM11" ref-type="media"> 11 </xref> , and in Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> : Figures S6-S8. In summary, results from these analyses indicated that a <italic> FAM7 GNAT </italic> was acquired from actinomycetes in an ancestor of cyst and reniform nematodes and remains as a single effector gene in each species. Also, <italic> INVs </italic> and <italic> CMs </italic> were acquired from rhizobacteria and <italic> Burkholderia </italic> -related bacteria, respectively, in ancestral Hoplolaimina PPN, and since HGT have experienced multiple duplications with neofunctionalization; some are likely effectors while others are either secreted or TM proteins that function within the nematodes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Host-Derived Artificial MicroRNA as an Alternative Method to Improve Soybean <span class="trait">Resistance</span> to Soybean Cyst Nematode</h2>
			<h3>PMID:27941644</h3>
				
					<h3>PMCID:5192498</h3>
					<h4>期刊:Genes</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Bin </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-07-00122"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Jiarui </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-07-00122"> 1 </xref>  <xref ref-type="aff" rid="af2-genes-07-00122"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oakley </surname>  <given-names> Thomas R. </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-07-00122"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Todd </surname>  <given-names> Timothy C. </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-07-00122"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Trick </surname>  <given-names> Harold N. </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-07-00122"> 1 </xref>  <xref rid="c1-genes-07-00122" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 01 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To construct the pUFamiR vector for amiRNA-mediated gene silencing in legume roots ( <xref ref-type="fig" rid="genes-07-00122-f001"> Figure 1 </xref> ), a 918 bp fragment of the soybean polyubiquitin (GenBank: EU310508.1) promoter <span class="gene">(GmUbi)</span> [ <xref rid="B26-genes-07-00122" ref-type="bibr"> 26 </xref> ] flanked by <italic> Not </italic> I and <italic> Bam </italic> HI restriction sites was amplified via PCR from soybean genomic DNA (cultivar &#x0201c;Fayette&#x0201d;) and cloned into a modified pBI121 construct. The superfolder green fluorescent protein (sGFP) gene from HBT&#x02013;sGFP&#x02013;NOS construct (kind gift from Jen Sheen, Massachusetts General Hospital) was mutagenized via PCR to remove its stop codon and to flank the gene with <italic> Bam </italic> HI and <italic> Xba </italic> I sites. The gene was then cloned into the pBI121 vector to <span class="trait">yield</span> the pUF construct. The pre-amiRNA containing miR319a of <italic> Arabidopsis thaliana </italic> was cloned from pRS300, which was a gift from Detlef Weigel (Addgene plasmid # 22846) [ <xref rid="B27-genes-07-00122" ref-type="bibr"> 27 </xref> , <xref rid="B28-genes-07-00122" ref-type="bibr"> 28 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Total RNA was treated with RQ1 DNase (Promega, Madison, WI, USA) to remove genomic DNA (gDNA) contaminants. One microgram of treated total RNA was used as template in the Reverse Transcript System (Promega). The RT-qPCRs were performed in triplicate using the PowerUp <sup> TM </sup> SYBR Green PCR Master Mix (Applied Biosystems, Foster City, CA, USA) on 7900HT Fast Real-Time PCR system (Applied Biosystems) with the following parameters: 95 &#x000b0;C for 3 min; 40 cycles (95 &#x000b0;C for 10 s; 60 &#x000b0;C for 1 min); 95 &#x000b0;C for 10 s, and a melt curve analysis from 60 &#x000b0;C to 95 &#x000b0;C at a ramp rate of 0.1 &#x000b0;C/s. The melt curve analysis was used to confirm the specificity, and RT-qPCR amplicons were sequenced to confirm that the target gene was amplified. The soybean <span class="gene">GmACTII</span> and <span class="gene">GmEF1b</span> [ <xref rid="B30-genes-07-00122" ref-type="bibr"> 30 </xref> ] were both used as internal controls to normalize expression of each target gene in plant, and the <italic> H. <span class="trait">glycines</span> </italic> &#x003b2;-actin (AF318603) was used as the internal control in nematodes. The primers of stem-loop PCR were designed according to Chen et al. [ <xref rid="B31-genes-07-00122" ref-type="bibr"> 31 </xref> ] with some modifications [ <xref rid="B32-genes-07-00122" ref-type="bibr"> 32 </xref> ]. Finally, the comparison of relative gene expression levels was calculated by 2 <sup> &#x02212;&#x00394;&#x00394; <italic> C </italic> t </sup> method [ <xref rid="B33-genes-07-00122" ref-type="bibr"> 33 </xref> ]. Three biological replications, each consisting of a pooled sample from 4 of the 12 positive transgenic roots, as well as three technical replicates, were used for each construct in each independent experiment. A list of primers used in this study can be found in <xref ref-type="app" rid="app1-genes-07-00122"> Supplementary Materials Table S1 </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetics and Adaptation of Soybean Cyst Nematode to Broad Spectrum Soybean <span class="trait">Resistance</span></h2>
			<h3>PMID:28064187</h3>
				
					<h3>PMCID:5345713</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gardner </surname>  <given-names> Michael </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heinz </surname>  <given-names> Robert </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jianying </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1"> Division of Plant Sciences and Bond Life Sciences Center, University of Missouri, Columbia, Missouri 65211 </aff>
				<br/> 
				年份：<day> 01 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						It was also noted that SCN populations adapted to reproduce on one resistant soybean line were able to reproduce better on some, but not all, other resistant lines. For example, it was shown that culturing nematodes on soybean lines PI 88788, PI 209332, or PI 548316 (Cloud) resulted in SCN populations that grow well on these lines, but not on the soybean lines PI 548402 (Peking), PI 90763, and PI 89772 ( <xref rid="bib1" ref-type="bibr"> Anand and Brar 1983 </xref> ; <xref rid="bib15" ref-type="bibr"> McCann <italic> et al. </italic> 1982 </xref> ; <xref rid="bib27" ref-type="bibr"> Young 1994 </xref> ). Based on these observations, two genetic groups of soybeans were proposed, with group one containing PI 88788, PI 209332, and Cloud, while group two contains Peking, PI 90763, and PI 89772 ( <xref rid="bib14" ref-type="bibr"> Luedders and Dropkin 1983 </xref> ; <xref rid="bib6" ref-type="bibr"> Colgrove and Niblack 2008 </xref> ). In recent years, much has been discovered about the genes responsible for <span class="trait">resistance</span> in both groups, specifically underlying two quantitative trait loci named <italic> <span class="gene">Rhg1</span> </italic> (for <span class="trait">resistance</span> to <italic> H. <span class="trait">glycines</span> </italic> ), and <italic> <span class="gene">Rhg4</span> </italic> (reviewed in <xref rid="bib16" ref-type="bibr"> Mitchum 2016 </xref> ). Resistant soybean lines in group one only require <italic> <span class="gene">Rhg1</span> </italic> for <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations. These soybean lines have a variable number of tandem repeats containing a set of four dissimilar genes ( <italic> i.e. </italic> , 7&#x02013;10 copies) at <italic> <span class="gene">Rhg1</span> </italic> ( <xref rid="bib7" ref-type="bibr"> Cook <italic> et al. </italic> 2012 </xref> ; <xref rid="bib11" ref-type="bibr"> Lee <italic> et al. </italic> 2015 </xref> ). Three of the genes within the repeat were shown to be required for <span class="trait">resistance.</span> By contrast, soybean lines in group two contain only a few repeats ( <italic> i.e. </italic> , 1&#x02013;3 copies) at <italic> <span class="gene">Rhg1</span> </italic> ( <xref rid="bib7" ref-type="bibr"> Cook <italic> et al. </italic> 2012 </xref> ; <xref rid="bib11" ref-type="bibr"> Lee <italic> et al. </italic> 2015 </xref> ), but also require a second QTL, <italic> <span class="gene">Rhg4</span> </italic> , where a single serine hydroxymethyltransferase (SHMT) gene controls <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations ( <xref rid="bib12" ref-type="bibr"> Liu <italic> et al. </italic> 2012 </xref> ). While great strides have been made to identify some of the soybean genes controlling <span class="trait">resistance,</span> the mechanisms by which virulent SCN overcome these two major types of <span class="trait">resistance,</span> often referred to as Peking-type and PI 88788-type, remain unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous reports have implicated independent genes inherited in both a dominant and a recessive manner in SCN virulence on a variety of <span class="trait">resistance</span> sources ( <xref rid="bib8" ref-type="bibr"> Dong and Opperman 1997 </xref> ). The offspring of the PA3&#x000a0;&#x000d7;&#x000a0;TN20 F <sub> 1 </sub> generation support these previous findings, with high FI on PI 88788, where the <italic> ror-1 </italic> gene is inherited in a dominant manner, and recessive genes, <italic> ror-2 </italic> and <italic> ror-3 </italic> , for reproduction on PI 90763 and Peking, respectively ( <xref rid="bib9" ref-type="bibr"> Dong <italic> et al. </italic> 2005 </xref> ). However, nothing was known about the inheritance of virulence genes on PI 437654-derived <span class="trait">resistance,</span> or the impact of sex linkage upon virulence gene inheritance. Our tests sought to describe the SCN genes responsible for virulence on PI 437654, and <span class="gene">try</span> to distinguish if these genes are independent of those required for virulence on PI 88788 and Peking <span class="trait">resistance.</span> To determine if there was any relationship between sex of the parent in the initial cross, and the virulence profile of the offspring, we performed the cross with the virulent parent as both a male and a female. The resulting F <sub> 1 </sub> and F <sub> 2 </sub> offspring had a nearly identical HG type, indicating that the virulence genes were likely inherited independently of sex.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Embryogenesis in the parasitic nematode <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> is independent of host-derived hatching stimulation</h2>
			<h3>PMID:28077087</h3>
				
					<h3>PMCID:5225516</h3>
					<h4>期刊:BMC Developmental Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Thapa </surname>  <given-names> Sita </given-names>  </name>  <address>  <email> sthapa2@illinois.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patel </surname>  <given-names> Jayna A. </given-names>  </name>  <address>  <email> japatel3@illinois.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reuter-Carlson </surname>  <given-names> Ursula </given-names>  </name>  <address>  <email> reuterca@illinois.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-3327-2136 </contrib-id>  <name>  <surname> Schroeder </surname>  <given-names> Nathan E. </given-names>  </name>  <address>  <email> nes@illinois.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Department of Crop Sciences, University of Illinois at Urbana-Champaign, 1102 S. Goodwin Ave, Urbana, 61801 IL USA  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To better understand the hatching behavior of <italic> H. <span class="trait">glycines</span> </italic> , we first resolved the timeline of pre-hatch development using several observational techniques. Single cell eggs were rarely found. We, therefore, started most of our observations at the two-celled stage. We observed a similar sequence of pre-hatch developmental stages (2, 3, and 4-celled stages, gastrula, tadpole, J1, and J2) as recorded for other plant-parasitic nematodes [ <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Furthermore, we traced the early cell lineage of some embryos. The first cell division was transverse and slightly unequal, producing the P <sub> 1 </sub> and AB blastomeres (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 7&#x000a0;h). This asymmetry determined the anterior-posterior axis. P <sub> 1 </sub> divided to produce EMS and P <sub> 2 </sub> (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 10.5&#x000a0;h). Following this division, AB divided to form the ABa (a-anterior) and ABp (p-posterior) blastomeres (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 13.5&#x000a0;h). Similar to other previously described Clade 12 nematodes, we observed a linear arrangement in the four-celled embryo (ABa, ABp, EMS, and P <sub> 2 </sub> ) indicative of an I <sub> 3 </sub> -Type blastomere arrangement [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. Following the four-celled linearly arranged embryo, the P <sub> 2 </sub> cell divided to produce C and P <sub> 3 </sub> (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 21.5&#x000a0;h). ABa and ABp then divided to produce anterior and posterior pairs (ABaa, ABap, ABpa, ABpp) (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 24&#x000a0;h). Following the AB cell divisions, EMS divided to form <span class="gene">MS</span> and E, resulting in an eight-celled embryo (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , 27&#x000a0;h). The cell lineage of individual animals was not traced following the eight-cell stage due to the refractive optical properties of the embryo, as described for other plant-parasitic nematodes [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> Time-lapse DIC micrographs of <italic> H. <span class="trait">glycines</span> </italic> early embryo development. Individual eggs were observed every half hour until the eight-cell stage. The time in hours ( <italic> lower left corner </italic> ) was recorded for each of the cell divisions. Cell labels are based on <italic> Caenorhabditis elegans </italic> and inferred based on positional homology in other nematode species [ <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. Scale, 15&#x000a0;&#x003bc;m
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ethylene response pathway modulates attractiveness of plant roots to soybean cyst nematode <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic></h2>
			<h3>PMID:28112257</h3>
				
					<h3>PMCID:5256374</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Yanfeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> You </surname>  <given-names> Jia </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Chunjie </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Williamson </surname>  <given-names> Valerie M. </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Congli </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Key Laboratory of Mollisols Agroecology, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences </institution> , Harbin 150081  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Department of Plant Pathology, University of California </institution> , Davis, CA 95616  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 19 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We proposed to treat soybean roots with the ET-synthesis inhibitor AVG prior to the attraction assay as a strategy to evaluate whether ET signaling played a role in attractiveness of soybean roots to <italic> H. <span class="trait">glycines</span> </italic> . To confirm that AVG treatment of soybean roots under our conditions had the anticipated effect, the expression of ethylene-response marker genes <italic> <span class="gene">GmERF5</span> </italic> and <italic> <span class="gene">GmERS2</span> </italic> was measured by qRT-PCR after 6, 12 and 24&#x02009;h of treatment with AVG. <italic> <span class="gene">GmERF5</span> </italic> transcript levels were significantly reduced compared to untreated roots at 12 and 24&#x02009;h after AVG treatment, whereas <italic> <span class="gene">GmERS2</span> </italic> levels were reduced at all three time points ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). These results indicated the AVG-treatment down-regulated expression of the ethylene-response pathway in soybean roots.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean cyst nematode culture collections and field populations from North Carolina and Missouri reveal high incidences of infection by viruses</h2>
			<h3>PMID:28141854</h3>
				
					<h3>PMCID:5283738</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ruark </surname>  <given-names> Casey L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Koenning </surname>  <given-names> Stephen R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Davis </surname>  <given-names> Eric L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Opperman </surname>  <given-names> Charles H. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lommel </surname>  <given-names> Steven A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sit </surname>  <given-names> Tim L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						RNAs were extracted from <italic> Glycine max </italic> (internal control ELF1B), <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> (internal controls HgFAR1, <span class="gene">GAPDH</span> and PPN18 <italic> S </italic> ) or other nematode species (internal control PPN18 <italic> S </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						New viral primers were designed to account for a potentially high degree of sequence dissimilarity for viruses in other PPN species. Therefore, viral &#x02018;Q&#x02019; primers were designed within conserved regions of 500 nucleotide sequences of viral polymerases ( <xref ref-type="table" rid="pone.0171514.t001"> Table 1 </xref> ). The nematode species <italic> <span class="trait">Heterodera</span> schachtii </italic> (beet cyst), <italic> H </italic> . <italic> trifolii </italic> (clover cyst), <italic> Vittatidera zeaphila </italic> (corn cyst) [ <xref rid="pone.0171514.ref024" ref-type="bibr"> 24 </xref> ], <italic> Globodera rostochiensis </italic> (golden potato cyst), <italic> G </italic> . <italic> tabacum </italic> (tobacco cyst) were tested for viruses. Additionally, <italic> Meloidogyne arenaria </italic> (peanut root-knot), <italic> M </italic> . <italic> hapla </italic> (northern root-knot), <italic> M </italic> . <italic> incognita </italic> (southern root-knot), <italic> M </italic> . <italic> javanica </italic> (Javanese root-knot), and <italic> Rotylenchus reniformis </italic> (reniform) were examined. Due to either unavailable or dissimilar <span class="gene">GAPDH</span> sequences across nematode species, internal control primers were designed from conserved regions of the 18 <italic> S </italic> ribosomal RNA gene among sequences available from NCBI Genbank including <italic> H </italic> . <italic> <span class="trait">glycines</span> </italic> (AY043247), <italic> H </italic> . <italic> schachtii </italic> (KJ636284), <italic> Meloidogyne javanica </italic> (AF442193) and <italic> Aphelenchoides besseyi </italic> (AY508035). The resulting primers are also listed in <xref ref-type="table" rid="pone.0171514.t001"> Table 1 </xref> . Products were analyzed on an Applied Biosystems QuantStudio&#x02122; 6 Flex Real-Time PCR System under the previously described settings. Samples positive for viruses were electrophoresed on a 2% TAE agarose gel.
					</p>
				
					<p>&nbsp;&nbsp;
						Future work will determine if nematode viruses are contributing factors to host development or pathogenicity. Relatedly, treating human infections of the nematode species <italic> Onchocerca volvulus </italic> with doxycycline in conjunction with ivermectin enables greater treatment success. The antibiotic sterilizes female nematodes by attacking an endosymbiotic bacterium, <italic> Wolbachia <span class="gene">spp</span> </italic> ., which controls host development [ <xref rid="pone.0171514.ref028" ref-type="bibr"> 28 </xref> ]. If viruses can function similarly in other parasitic nematodes, targeting viruses could help to disrupt the parasitic cycles of important nematode species. More in depth studies of microbial symbionts of nematodes may <span class="trait">yield</span> interesting results that can further development of novel control strategies, and it is likely that future next-generation sequencing projects will continue to unveil additional novel viral sequences within nematode species. Although SCN viruses were not detectable in soybean plant tissue, more intensive testing of SCN weed hosts may <span class="trait">yield</span> interesting results as to additional hosts for these viruses. From the presented research, we hope to demonstrate that nematode viruses are more widespread than previously thought and may have important impacts on nematode biology. Thus in the future, viruses could prove useful for development of novel biological controls to target nematodes. Upcoming studies include examining how viral transmission occurs and localizing cellular and tissue tropisms.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loci and candidate genes conferring <span class="trait">resistance</span> to soybean cyst nematode HG type 2.5.7</h2>
			<h3>PMID:28615053</h3>
				
					<h3>PMCID:5471737</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> tengweili@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 1136 </institution-id>  <institution-id institution-id-type="GRID"> grid.412243.2 </institution-id>  <institution> Key Laboratory of Soybean Biology in Chinese Ministry of Education (Key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry),  </institution>  <institution> Northeast Agricultural University,  </institution>  </institution-wrap> Harbin, 150030 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0526 1937 </institution-id>  <institution-id institution-id-type="GRID"> grid.410727.7 </institution-id>  <institution> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI),  </institution>  <institution> Chinese Academy of Agricultural Sciences,  </institution>  </institution-wrap> Beijing, 100081 China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.410751.6 </institution-id>  <institution> Bioinformatics Division,  </institution>  <institution> Biomarker Technologies Corporation,  </institution>  </institution-wrap> Beijing, 101300 China  </aff>
				<br/> 
				年份：<day> 5 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The inheritance of SCN <span class="trait">resistance</span> is complicated [ <xref ref-type="bibr" rid="CR6"> 6 </xref> &#x02013; <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. The advance of DNA markers has enabled the detection of many quantitative trait loci (QTL) underlying <span class="trait">resistance</span> to SCN. To date, several putative QTL have been reported to be associated with <span class="trait">resistance</span> to SCN, derived from both cultivated and wild soybeans ( <italic> Glycine soja </italic> ) [ <xref ref-type="bibr" rid="CR7"> 7 </xref> , <xref ref-type="bibr" rid="CR11"> 11 </xref> &#x02013; <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. These loci have provided <span class="trait">resistance</span> to various HG Types (previously races) and have been mapped onto 17 chromosomes (Chr) or linkage groups (LG). Among the identified QTL, <italic> rhg1 </italic> alleles a and b on Chr 18 and <italic> <span class="gene">Rhg4</span> </italic> allele a on Chr 8 [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ] were isolated from &#x02018;PI88788&#x02019; and &#x02018;Forrest&#x02019;, respectively [ <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. The copy number variation (CNV) of 31 Kbp DNA segment conferred the SCN <span class="trait">resistance</span> of <italic> rhg1 </italic> allele b in &#x02018;PI 88788&#x02019; and three disparate genes presented on each repeat contribute to SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. Differentially methylated DNA regions were also identified within <italic> rhg1 </italic> , that correlate with soybean cyst nematode <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Two point mutations in <italic> <span class="gene">Rhg4</span> </italic> of &#x02018;Forrest&#x02019; altered a key regulatory property of serine hydroxymethyltransferase. It had been hypothesized that this mutation might result in a nutritional deficiency among female nematodes [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 248 soybean genes were identified in the flanking region of each peak SNP (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Among these genes, 53 genes had no functional annotation, and seven genes belonged to the domains of unknown function families. To predict potential functions of genes in the flanking region of SNPs associated with the <span class="trait">resistance</span> to SCN, the 196 genes were grouped into the following functional categories using MapMap [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]: cell wall metabolism (eight genes), lipid metabolism (six genes), secondary metabolism (nine genes), biotic stress (five genes), signaling (nine genes), transcription regulation (transcription factors, TFs, 32 genes), hormonal metabolism (13 genes), redox group (two genes), protein modification and degradation (31 genes), transport (eight genes), development (two genes), DNA synthesis or chromatin structure (six genes), miscellaneous group (15 genes: 3 cytochrome <span class="gene">P450</span> genes, 2 GDSL-motif lipase, 2 nitrile lyases etc.), light reaction (seven genes), other groups of genes (12 genes), and unclassified genes (12 genes) (Fig. <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Of them, many genes have been implicated in plant disease defense or plant disease <span class="trait">resistance</span> pathways, including protein kinase family (belonging to the signal group), leucine-rich repeat-containing proteins and receptor-like protein. Some domain types, such as cytochrome P450s (belonging to the miscellaneous group), zinc fingers and RING (belonging to the transcription factors), have been implicated in soybean responses to SCN [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ] <italic> . </italic> To more accurately predict the candidate genes, the genes in a 50 kbp genomic region of each side of the peak SNP were further selectively analyzed. A total of 21 candidate genes were eventually verified. <span class="gene">Glyma.07G195500,</span> encoding a transcription factor-related gene, was 12.78 kbp away from SNP rs36423980 on Chr7. <span class="gene">Glyma.07G195400</span> and <span class="gene">Glyma.07G196000,</span> with RING domains, were associated with SCN <span class="trait">resistance.</span> Gruenwald et al. reported that the auxin-inducible transcription factor AtWRKY23 was expressed during the infection of Arabidopsis roots with <italic> H. schachtii </italic> and demonstrated that the regulation of AtWRKY23 was controlled through auxin response factor 7 (ARF7) and the ARF19 pathway [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Herein, an auxin response factor gene <span class="gene">(Glyma.08G100100)</span> 2.4 kbp from rs7671170 on Chr 8 was implicated in the soybean reaction to SCN infection. In the present study, Laccase genes <span class="gene">(Glyma.18G193200,</span> <span class="gene">Glyma.18G193300,</span> and <span class="gene">Glyma.18G193400),</span> which participate in lignin synthesis and phenolic compound metabolism in plants [ <xref ref-type="bibr" rid="CR37"> 37 </xref> ], might also contribute to HG Type 2.5.7 <span class="trait">resistance</span> in soybean. Ithal et al. tested three time points after SCN feeding (2, 5, and 10 dpi) and also noted an increase in the expression of genes involved in lignin biosynthesis and phenolic compound metabolism [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Except for the above genes, nine genes out of the 196 were regulated by SCN in different soybean lines according to the report by Wan et al. [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Of them, Leucine-rich repeat receptor-like protein kinase family protein <span class="gene">(Glyma.14G047900)</span> might be involved in SCN <span class="trait">resistance</span> since the gene showed up-regulated after SCN inoculation (Table <xref rid="Tab2" ref-type="table"> 2 </xref> ) [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Functional categories of the predicted resistant genes to HG Type 2.5.7
					</p>
				
					<p>&nbsp;&nbsp;
						To date, numbers of SCN-resistant QTL have been reported [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Most of which were verified using different cross populations from limited <span class="trait">resistance</span> sources. Two major QTL across multiple resistant sources were <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Additionally, QTL qSCN11, located on Chr11, has also been consistently identified from PI 437654, PI 90763 and PI 404198B [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. In the present study, a total of thirteen SNPs distributed on five chromosomes (Chr 7, 8, 14, 15 and 18) were associated with SCN HG Type 2.5.7 <span class="trait">resistance.</span> Among the thirteen association signals, nine SNPs overlapped with or were located near known QTL (Table <xref rid="Tab2" ref-type="table"> 2 </xref> ). A resistant genomic region in Chr8, containing the gene loci of rs7631207, rs7640250, rs7671170, rs7662003, rs7664479, rs7622492 and rs7661660, were significantly associated with SCN <span class="trait">resistance</span> to HG Type 2.5.7, and the relation between these genomic regions and SCN <span class="trait">resistance</span> has been reported in previous studies [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> &#x02013; <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. Similarly, two SNPs, rs38522986 and rs46625879, located on Chr15 and Chr18, respectively, were identified inside two marker intervals previously reported by Kabelka et al. [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ] and Winter et al. [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Moreover, two genomic regions (rs7671170 on Chr8 and rs46625879 on Chr18) could stably be identified using three models, including CMLM, ECMLM and FARMCPU, which further verified the importance of these two genomic regions for <span class="trait">resistance</span> to HG Type 2.5.7. These consistent genomic regions in the present and previous studies showed that Chr8, Chr15 and Chr18 might play important roles in conferring SCN <span class="trait">resistance</span> in the soybean germplasms of China and North America.
					</p>
				
					<p>&nbsp;&nbsp;
						Major QTL, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were valuable resources for SCN <span class="trait">resistance,</span> but were frequently not durable, reflecting shifts in the SCN population that resulted in a loss of SCN <span class="trait">resistance</span> in major QTL [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. Breeding soybean with durable <span class="trait">resistance</span> to SCN through the identification and utilization of novel QTL is an effective strategy to cope with the loss of SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. One novel QTL on Chr10 (qSCN10) was identified in PI 567516C [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ], which could confer high SCN <span class="trait">resistance</span> to soybean lacking the two known major genes, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> . Additionally, four novel QTL (rs35898587 and rs36423980 on Chr7, rs16268025 on Chr8, and rs3853672 on Chr14) were also identified, which were significantly different from the major QTL reported in previous studies. Although the molecular mechanisms of the novel <span class="trait">resistance</span> loci were not clear, these loci possessed high potential to breed cultivars with durable <span class="trait">resistance</span> to SCN through the pyramid of the novel and previously reported QTL [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Presently, for the molecular mechanism of SCN <span class="trait">resistance</span> genes, only two genes, <italic> rhg1 </italic> and <italic> Rhg4, </italic> were clearly associated with the molecular mechanism of SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ], and other candidate genes or QTL underlying SCN <span class="trait">resistance</span> were less investigated. Thus, it was difficult to predict and confirm the SCN candidates from large QTL intervals with a number of genes. However, GWAS could still offer some valuable clues to identify and confirm SCN <span class="trait">resistance</span> genes, particularly within a linkage disequilibrium (LD) block (150-200 kbp in length on average) [ <xref ref-type="bibr" rid="CR23"> 23 </xref> , <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. In the present study, a total of 196 potential candidate genes, located in 200 kbp flanking regions up- and downstream of thirteen peak SNPs, possessed the canonical SCN <span class="trait">resistance</span> domains (including cytochrome P450s, zinc fingers and RING) [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ], which are involved in plant disease responses or plant disease <span class="trait">resistance</span> pathways. Among these candidate genes, six genes <span class="gene">(Glyma.07G195400,</span> <span class="gene">Glyma.07G196000,</span> <span class="gene">Glyma.08G100100,</span> <span class="gene">Glyma.18G193200,</span> <span class="gene">Glyma.18G193300</span> and <span class="gene">Glyma.18G193400)</span> have been reported to be responsible for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Furthermore, 15 novel genes <span class="gene">(Glyma.07G190900,</span> <span class="gene">Glyma.07G191000,</span> <span class="gene">Glyma.07G191100,</span> <span class="gene">Glyma.07G191200,</span> <span class="gene">Glyma.07G191500,</span> <span class="gene">Glyma.07G195500,</span> <span class="gene">Glyma.08G099400,</span> <span class="gene">Glyma.08G099700,</span> <span class="gene">Glyma.08G100700,</span> <span class="gene">Glyma.08G100800,</span> <span class="gene">Glyma.08G200800,</span> <span class="gene">Glyma.08G201000,</span> <span class="gene">Glyma.08G201100,</span> <span class="gene">Glyma.14G048600,</span> and <span class="gene">Glyma.14G049500),</span> located in 50 kbp flanking regions up- and downstream of peak SNPs, were associated with SCN HG Type 2.5.7 <span class="trait">resistance</span> in the present study. The clear function of these candidates should be discussed in future studies.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Complex Cell Wall Composition of Syncytia Induced by Plant Parasitic Cyst Nematodes Reflects Both Function and Host Plant</h2>
			<h3>PMID:28680436</h3>
				
					<h3>PMCID:5478703</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/345048/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lilley </surname>  <given-names> Catherine J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/359806/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Imren </surname>  <given-names> Mustafa </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/431497/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Knox </surname>  <given-names> J. Paul </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/48523/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Urwin </surname>  <given-names> Peter E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/346015/overview"/>  </contrib>
				<br/> 
				年份：<day> 06 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Immuno-fluorescence imaging of syncytia induced by soybean cyst nematode <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> within soybean roots (cv. Toliman, 14 dpi). <bold> (A) </bold> The extent of the syncytium is indicated in the Toluidine Blue O stained bright field image with a red line. Indirect immunofluorescence (green) resulting from the binding of specific mAbs is shown for corresponding serial sections: <bold> (B) </bold> LM11 to heteroxylan; <bold> (C) </bold> LM15 to xyloglucan (XyG); <bold> (D) </bold> LM24 to xyloglucan (XyG); <bold> (E) </bold> LM25 to xyloglucan (XyG); <bold> (F,G) </bold> LM19 to non/low methyl-esterified homogalacturonan (HG); <bold> (H) </bold> LM20 to methyl-esterified HG; <bold> (I) </bold> LM5 to pectic galactan; <bold> (J) </bold> LM6 to arabinan; <bold> (K) </bold> JIM20 to extensin. LM11 binds only to the xylem vessels in the vascular cylinder <bold> (B) </bold> so serves to identify these cells in all sections. Control section <bold> (L) </bold> was processed without primary antibody. PL, pre-treated with pectate lyase; Na <sub> 2 </sub> CO <sub> 3 </sub> , pre-treated with Na <sub> 2 </sub> CO <sub> 3 </sub> ; Scale <span class="gene">bar</span> = 50 &#x003bc;m.
					</p>
				
					<p>&nbsp;&nbsp;
						Immuno-fluorescence imaging of syncytia induced by the cereal cyst nematodes (CCNs) <italic> <span class="trait">Heterodera</span> avenae </italic> and <italic> <span class="trait">Heterodera</span> filipjevi </italic> within wheat roots (cv. Cadenza, 21 dpi and 28 dpi respectively). <bold> (A,K) </bold> Bright field images of Toluidine Blue O stained sections. The extent of the syncytium is indicated with a red line. Indirect immunofluorescence (green) resulting from the binding of specific mAbs is shown for corresponding serial sections: <bold> (B,L) </bold> LM10 to heteroxylan; <bold> (C,M) </bold> LM11 to heteroxylan; <bold> (D,N) </bold> LM12 to feruloylated heteroxylan; <bold> (E,O) </bold> MLG to mixed linkage glucan (MLG); <bold> (F,P) </bold> LM15 to xyloglucan (XyG); <bold> (G,Q) </bold> LM25 to xyloglucan (XyG); <bold> (H,R) </bold> LM20 to methyl-esterified homogalacturonan (HG); <bold> (I,S) </bold> LM5 to pectic galactan; <bold> (J,T) </bold> LM6 to arabinan. Scale <span class="gene">bar</span> = 50 &#x003bc;m.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>