<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Heterodera glycines</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Heterodera glycines</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:A harpin elicitor induces the expression of a coiled-coil nucleotide binding leucine rich repeat (CC-NB-LRR) defense signaling gene and others functioning during defense to parasitic nematodes.</h2>
			<h3>PMID:29107936</h3>
				

				
			<p>
				作者：Aljaafri WAR(1), McNeece BT(2), Lawaju BR(3), Sharma K(4), Niruala PM(5), Pant SR(6), Long DH(7), Lawrence KS(8), Lawrence GW(9), Klink VP(10).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In Glycine max, <span class="gene">Gm-NDR1-1</span> transcripts have been detected within root cells undergoing a natural resistant reaction to parasitism by the syncytium-forming nematode <span class="trait">Heterodera</span> <span class="trait">glycines,</span> functioning in the defense response.
					</p>
				
					<p>&nbsp;&nbsp;
						Expressing <span class="gene">Gm-NDR1-1</span> in Gossypium hirsutum leads to <span class="trait">resistance</span> to Meloidogyne incognita parasitism.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <italic> <span class="gene">GmSNAP18</span> </italic> gene underlies two types of <span class="trait">resistance</span> to soybean cyst nematode</h2>
			<h3>PMID:28345654</h3>
				

				
					<h3>PMCID:5378975</h3>
					<h4>期刊:Nature Communications</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kandoth </surname>  <given-names> Pramod K. </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lakhssassi </surname>  <given-names> Naoufal </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Jingwen </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Colantonio </surname>  <given-names> Vincent </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heinz </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yeckel </surname>  <given-names> Greg </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Zhou </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bekal </surname>  <given-names> Sadia </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dapprich </surname>  <given-names> Johannes </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rotter </surname>  <given-names> Bjorn </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cianzio </surname>  <given-names> Silvia </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Plant, Soil and Agricultural Systems, Southern Illinois University </institution> , 1205 Lincoln Drive RM176, Carbondale, Illinois 62901,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Division of Plant Sciences and Bond Life Sciences Center, University of Missouri </institution> , Columbia, Missouri 65211,  <country> USA </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Generation Biotech </institution> , Lawrenceville, New Jersey 08648,  <country> USA </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> GenXPro-GmbH </institution> , Altenh&#x000f6;ferallee 3, 60438 Frankfurt am Main,  <country> Germany </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Department of Agronomy, Iowa State University </institution> , Ames, Iowa 50011,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 06 </day>  <month> 02 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						These include Peking-type soybean, whose <span class="trait">resistance</span> requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles, and PI 88788-type soybean, whose <span class="trait">resistance</span> requires only the <italic> rhg1-b </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple copy number of PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic> , <italic> <span class="gene">GmAAT</span> </italic> , and <italic> <span class="gene">GmWI12</span> </italic> in one genomic segment simultaneously contribute to <italic> rhg1-b </italic> <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Using an integrated set of genetic and genomic approaches, we demonstrate that the <italic> rhg1-a </italic> Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is sufficient for <span class="trait">resistance</span> to SCN in combination with <italic> <span class="gene">Rhg4</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Our findings suggest that Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Here the authors show that in contrast to PI 88788-type <span class="trait">resistance,</span> where <italic> <span class="gene">GmSNAP18</span> </italic> acts in combination with two neighbouring genes, in Peking-type <span class="trait">resistance</span> <italic> <span class="gene">GmSNAP18</span> </italic> acts with <italic> <span class="gene">GmSHMT08</span> </italic> to confer <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						QTL on chromosomes 18 ( <italic> rhg1 </italic> ) and 8 ( <italic> <span class="gene">Rhg4</span> </italic> ) are the two major <span class="trait">resistance</span> QTL that have been consistently mapped and reported in a variety of soybean germplasm.
					</p>
				
					<p>&nbsp;&nbsp;
						) Forrest, <span class="trait">resistance</span> to SCN requires both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> (ref.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, progress has been made in the identification of the major soybean genes underlying SCN <span class="trait">resistance</span> at the <italic> rhg1-b </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, and in the development of specific DNA markers for SCN <span class="trait">resistance</span> selection.
					</p>
				
					<p>&nbsp;&nbsp;
						Serine hydroxymethyltransferases (SHMTs) are important in one-carbon folate metabolism, but how this particular soybean <span class="gene">SHMT</span> functions in <span class="trait">resistance</span> to the nematode remains unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						reported that the <span class="trait">resistance</span> of soybean <italic> rhg1-b </italic> to SCN is simultaneously mediated by the high copy number of three genes ( <italic> <span class="gene">Glyma18g02580</span> </italic> , <italic> <span class="gene">Glyma18g02590</span> </italic> and <italic> <span class="gene">Glyma18g02610</span> </italic> ) within a 31&#x02009;kb PI 88788 <italic> rhg1-b </italic> segment <xref ref-type="bibr" rid="b7"> 7 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the <italic> <span class="gene">Glyma18g02590</span> </italic> gene sequences, two specific KASP (kompetitive allele-specific PCR) single-nucleotide polymorphism (SNP) markers were developed to select the <italic> rhg1 </italic> <span class="trait">resistance</span> alleles and the assay can be used to differentiate Peking-type and PI 88788-type <span class="trait">resistance</span> <xref ref-type="bibr" rid="b10"> 10 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						The analysis of the relevant SNPs and InDels identified the <italic> <span class="gene">GmSNAP18</span> </italic> as the strongest candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Genetic complementation analyses of the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> confirmed its major role in <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						The complementation analyses also showed the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is specifically required for Peking-type <span class="trait">resistance,</span> as the resistant <italic> <span class="gene">GmSNAP18</span> </italic> allele of PI 88788 or the susceptible <italic> rhg1-s </italic> could not restore <span class="trait">resistance</span> in the Forrest (Peking-type) <italic> <span class="gene">Rhg4</span> </italic> background.
					</p>
				
					<p>&nbsp;&nbsp;
						As Forrest-type <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> , identification of recombinants was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1-a </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was deemed as the strongest candidate gene for conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1-a </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was identified to be the most probable candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus, consistent with the results of map-based cloning ( <xref ref-type="fig" rid="f1"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						To establish a link between <italic> <span class="gene">GmSNAP18</span> </italic> alleles and soybean <span class="trait">resistance</span> to SCN, we scored 81 soybean lines (including PIs, landraces and elite cultivars, as described previously <xref ref-type="bibr" rid="b6"> 6 </xref> ) representing 95&#x00025; of the sequence diversity of soybeans for their SCN female index (FI) <xref ref-type="bibr" rid="b17"> 17 </xref> and then determined their SNP-based <italic> <span class="gene">GmSNAP18</span> </italic> haplotype after genotyping them at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci ( <xref ref-type="table" rid="t3"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						This includes soybean lines PI 548402 (Peking), Forrest, PI 90763, PI 437654 and PI 89772, all of which exhibit &#x02018;Peking-type' <span class="trait">resistance</span> and require both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, the <italic> <span class="gene">GmSNAP18</span> </italic> haplotyping analysis is in agreement with previous SCN-resistance QTL reports and confirms the requirement of <italic> rhg1 </italic> for the SCN <span class="trait">resistance</span> in both PI 88788 and Peking.
					</p>
				
					<p>&nbsp;&nbsp;
						ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> display <span class="trait">resistance</span> to SCN, whereas the ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> or <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x02212; </sup> do not show <span class="trait">resistance</span> to SCN ( <xref ref-type="fig" rid="f4"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						These results further support a role for <italic> <span class="gene">GmSNAP18</span> </italic> in <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In Forrest, both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles are required for <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, to test the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> for contribution to <italic> rhg1-a </italic> <span class="trait">resistance,</span> we used the SCN-susceptible RIL ExF50 carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> / <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x0002b; </sup> , which has the resistant Forrest allele at the <italic> <span class="gene">Rhg4</span> </italic> locus and the susceptible Essex allele at the <italic> rhg1 </italic> ( <italic> rhg1-s </italic> ) locus.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Glyma18g02570</span> </italic> did not contribute to SCN <span class="trait">resistance</span> <xref ref-type="bibr" rid="b7"> 7 </xref> and therefore was not tested in this study.
					</p>
				
					<p>&nbsp;&nbsp;
						These data suggest that the <italic> <span class="gene">GmSNAP18</span> </italic> gene at the <italic> rhg1-a </italic> locus is sufficient to confer SCN <span class="trait">resistance</span> in Forrest.
					</p>
				
					<p>&nbsp;&nbsp;
						4 </xref> ), demonstrating the specificity of Forrest <italic> <span class="gene">GmSNAP18</span> </italic> in conferring Peking-type soybean <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the locations of the different identified haplotypes of <italic> <span class="gene">GmSNAP18</span> </italic> and how they may confer SCN <span class="trait">resistance,</span> the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> was modelled and then each haplotype of the other soybean lines ( <xref ref-type="fig" rid="f3"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1-a </italic> gene conferring <span class="trait">resistance</span> to SCN using an integrated set of genomic and genetic approaches in combination with genetic complementation analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						The recombinant WxF6034 carried the Forrest allele for <italic> <span class="gene">Glyma18g02610</span> </italic> , but was susceptible to SCN, indicating that the <italic> <span class="gene">Glyma18g02610</span> </italic> gene is not contributing to the SCN <span class="trait">resistance</span> in the Forrest background.
					</p>
				
					<p>&nbsp;&nbsp;
						Previously, the leucine-rich repeat receptor-like kinase ( <italic> <span class="gene">LRR-RLK</span> </italic> ) genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci were claimed as the <span class="trait">resistance</span> genes <xref ref-type="bibr" rid="b20"> 20 </xref> <xref ref-type="bibr" rid="b21"> 21 </xref> <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> <xref ref-type="bibr" rid="b24"> 24 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="bibr" rid="b26"> 26 </xref> demonstrated that the <italic> <span class="gene">LRR-RLK</span> </italic> genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci did not confer SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						1a </xref> ), excluding the <italic> rhg1-a <span class="gene">LRR-RLK</span> </italic> candidate <span class="trait">resistance</span> gene in support of earlier work.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, consistent with the genetic mapping results, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1 </italic> candidate gene probably conferring <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic complementation analyses of <italic> <span class="gene">GmSNAP18</span> </italic> clearly showed that the transgenic ExF50 ( <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> <italic> &#x0002b; </italic> </sup> ) gained <span class="trait">resistance</span> to SCN after being complemented with the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> , but not when complemented with Forrest <italic> <span class="gene">AAT</span> </italic> ( <xref ref-type="fig" rid="f5"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						4 </xref> ), indicating that the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is the gene at the <italic> rhg1-a </italic> locus that confers <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-b </italic> ) or Essex <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-s </italic> ) could not restore <span class="trait">resistance</span> to ExF50 ( <xref ref-type="fig" rid="f5"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						These data suggest that the resistant form of the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> is unable to communicate with the Forrest <italic> <span class="gene">GmSHMT08</span> </italic> ( <italic> <span class="gene">Rhg4</span> </italic> ), to confer <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, the complementation results indicate that the SCN <span class="trait">resistance</span> at the <italic> rhg1-a </italic> locus in Forrest is triggered by <italic> <span class="gene">GmSNAP18</span> </italic> alone, contrasting with the <italic> rhg1-b </italic> in PI 88788, whose <span class="trait">resistance</span> is due to the high copy number and concerted activity of three genes <xref ref-type="bibr" rid="b7"> 7 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the <span class="trait">resistance</span> of Forrest and Peking to SCN requires the <italic> <span class="gene">Rhg4</span> </italic> gene <xref ref-type="bibr" rid="b6"> 6 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						All are in agreement with the previous genetic results, showing that the SCN <span class="trait">resistance</span> of Peking-type soybeans such as Forrest requires both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ) <xref ref-type="bibr" rid="b2"> 2 </xref> <xref ref-type="bibr" rid="b3"> 3 </xref> <xref ref-type="bibr" rid="b6"> 6 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic and cDNA sequences of the <italic> <span class="gene">GmSNAP18</span> </italic> conferring SCN <span class="trait">resistance</span> were different between Peking-type soybeans and PI 88788-type soybeans ( <xref ref-type="fig" rid="f1"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, our results support a model wherein <italic> <span class="gene">GmSNAP18</span> </italic> is a major factor mediating the different types of soybean <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Our findings suggest that the Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						As Forrest <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes <xref ref-type="bibr" rid="b3"> 3 </xref> , genotyping was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1 </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean <italic> <span class="gene">GmSNAP18</span> </italic> gene underlies two types of <span class="trait">resistance</span> to soybean cyst nematode.
					</p>
				
					<p>&nbsp;&nbsp;
						These include Peking-type soybean, whose <span class="trait">resistance</span> requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles, and PI 88788-type soybean, whose <span class="trait">resistance</span> requires only the <italic> rhg1-b </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple copy number of PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic> , <italic> <span class="gene">GmAAT</span> </italic> , and <italic> <span class="gene">GmWI12</span> </italic> in one genomic segment simultaneously contribute to <italic> rhg1-b </italic> <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Using an integrated set of genetic and genomic approaches, we demonstrate that the <italic> rhg1-a </italic> Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is sufficient for <span class="trait">resistance</span> to SCN in combination with <italic> <span class="gene">Rhg4</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Our findings suggest that Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Here the authors show that in contrast to PI 88788-type <span class="trait">resistance,</span> where <italic> <span class="gene">GmSNAP18</span> </italic> acts in combination with two neighbouring genes, in Peking-type <span class="trait">resistance</span> <italic> <span class="gene">GmSNAP18</span> </italic> acts with <italic> <span class="gene">GmSHMT08</span> </italic> to confer <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						QTL on chromosomes 18 ( <italic> rhg1 </italic> ) and 8 ( <italic> <span class="gene">Rhg4</span> </italic> ) are the two major <span class="trait">resistance</span> QTL that have been consistently mapped and reported in a variety of soybean germplasm
					</p>
				
					<p>&nbsp;&nbsp;
						) Forrest, <span class="trait">resistance</span> to SCN requires both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> (ref
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, progress has been made in the identification of the major soybean genes underlying SCN <span class="trait">resistance</span> at the <italic> rhg1-b </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, and in the development of specific DNA markers for SCN <span class="trait">resistance</span> selection
					</p>
				
					<p>&nbsp;&nbsp;
						Serine hydroxymethyltransferases (SHMTs) are important in one-carbon folate metabolism, but how this particular soybean <span class="gene">SHMT</span> functions in <span class="trait">resistance</span> to the nematode remains unknown
					</p>
				
					<p>&nbsp;&nbsp;
						reported that the <span class="trait">resistance</span> of soybean <italic> rhg1-b </italic> to SCN is simultaneously mediated by the high copy number of three genes ( <italic> <span class="gene">Glyma18g02580</span> </italic> , <italic> <span class="gene">Glyma18g02590</span> </italic> and <italic> <span class="gene">Glyma18g02610</span> </italic> ) within a 31&#x02009;kb PI 88788 <italic> rhg1-b </italic> segment <xref ref-type="bibr" rid="b7"> 7 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						Using the <italic> <span class="gene">Glyma18g02590</span> </italic> gene sequences, two specific KASP (kompetitive allele-specific PCR) single-nucleotide polymorphism (SNP) markers were developed to select the <italic> rhg1 </italic> <span class="trait">resistance</span> alleles and the assay can be used to differentiate Peking-type and PI 88788-type <span class="trait">resistance</span> <xref ref-type="bibr" rid="b10"> 10 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						The analysis of the relevant SNPs and InDels identified the <italic> <span class="gene">GmSNAP18</span> </italic> as the strongest candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus
					</p>
				
					<p>&nbsp;&nbsp;
						Genetic complementation analyses of the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> confirmed its major role in <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						The complementation analyses also showed the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is specifically required for Peking-type <span class="trait">resistance,</span> as the resistant <italic> <span class="gene">GmSNAP18</span> </italic> allele of PI 88788 or the susceptible <italic> rhg1-s </italic> could not restore <span class="trait">resistance</span> in the Forrest (Peking-type) <italic> <span class="gene">Rhg4</span> </italic> background
					</p>
				
					<p>&nbsp;&nbsp;
						As Forrest-type <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> , identification of recombinants was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1-a </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was deemed as the strongest candidate gene for conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1-a </italic> locus
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, <italic> <span class="gene">GmSNAP18</span> </italic> was identified to be the most probable candidate gene conferring <span class="trait">resistance</span> to SCN at the <italic> rhg1 </italic> locus, consistent with the results of map-based cloning ( <xref ref-type="fig" rid="f1"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						To establish a link between <italic> <span class="gene">GmSNAP18</span> </italic> alleles and soybean <span class="trait">resistance</span> to SCN, we scored 81 soybean lines (including PIs, landraces and elite cultivars, as described previously <xref ref-type="bibr" rid="b6"> 6 </xref> ) representing 95&#x00025; of the sequence diversity of soybeans for their SCN female index (FI) <xref ref-type="bibr" rid="b17"> 17 </xref> and then determined their SNP-based <italic> <span class="gene">GmSNAP18</span> </italic> haplotype after genotyping them at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci ( <xref ref-type="table" rid="t3"> Table 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						This includes soybean lines PI 548402 (Peking), Forrest, PI 90763, PI 437654 and PI 89772, all of which exhibit &#x02018;Peking-type' <span class="trait">resistance</span> and require both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> )
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, the <italic> <span class="gene">GmSNAP18</span> </italic> haplotyping analysis is in agreement with previous SCN-resistance QTL reports and confirms the requirement of <italic> rhg1 </italic> for the SCN <span class="trait">resistance</span> in both PI 88788 and Peking
					</p>
				
					<p>&nbsp;&nbsp;
						ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> display <span class="trait">resistance</span> to SCN, whereas the ExF RILs carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x0002b; </sup> or <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x0002b; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> &#x02212; </sup> do not show <span class="trait">resistance</span> to SCN ( <xref ref-type="fig" rid="f4"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						These results further support a role for <italic> <span class="gene">GmSNAP18</span> </italic> in <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						In Forrest, both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles are required for <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, to test the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> for contribution to <italic> rhg1-a </italic> <span class="trait">resistance,</span> we used the SCN-susceptible RIL ExF50 carrying <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> / <italic> <span class="gene">GmSHMT08</span> </italic> <sup> &#x0002b; </sup> , which has the resistant Forrest allele at the <italic> <span class="gene">Rhg4</span> </italic> locus and the susceptible Essex allele at the <italic> rhg1 </italic> ( <italic> rhg1-s </italic> ) locus
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Glyma18g02570</span> </italic> did not contribute to SCN <span class="trait">resistance</span> <xref ref-type="bibr" rid="b7"> 7 </xref> and therefore was not tested in this study
					</p>
				
					<p>&nbsp;&nbsp;
						These data suggest that the <italic> <span class="gene">GmSNAP18</span> </italic> gene at the <italic> rhg1-a </italic> locus is sufficient to confer SCN <span class="trait">resistance</span> in Forrest
					</p>
				
					<p>&nbsp;&nbsp;
						4 </xref> ), demonstrating the specificity of Forrest <italic> <span class="gene">GmSNAP18</span> </italic> in conferring Peking-type soybean <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the locations of the different identified haplotypes of <italic> <span class="gene">GmSNAP18</span> </italic> and how they may confer SCN <span class="trait">resistance,</span> the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> was modelled and then each haplotype of the other soybean lines ( <xref ref-type="fig" rid="f3"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1-a </italic> gene conferring <span class="trait">resistance</span> to SCN using an integrated set of genomic and genetic approaches in combination with genetic complementation analyses
					</p>
				
					<p>&nbsp;&nbsp;
						The recombinant WxF6034 carried the Forrest allele for <italic> <span class="gene">Glyma18g02610</span> </italic> , but was susceptible to SCN, indicating that the <italic> <span class="gene">Glyma18g02610</span> </italic> gene is not contributing to the SCN <span class="trait">resistance</span> in the Forrest background
					</p>
				
					<p>&nbsp;&nbsp;
						Previously, the leucine-rich repeat receptor-like kinase ( <italic> <span class="gene">LRR-RLK</span> </italic> ) genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci were claimed as the <span class="trait">resistance</span> genes <xref ref-type="bibr" rid="b20"> 20 </xref> <xref ref-type="bibr" rid="b21"> 21 </xref> <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> <xref ref-type="bibr" rid="b24"> 24 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="bibr" rid="b26"> 26 </xref> demonstrated that the <italic> <span class="gene">LRR-RLK</span> </italic> genes at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci did not confer SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						1a </xref> ), excluding the <italic> rhg1-a <span class="gene">LRR-RLK</span> </italic> candidate <span class="trait">resistance</span> gene in support of earlier work
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, consistent with the genetic mapping results, <italic> <span class="gene">GmSNAP18</span> </italic> was identified as the <italic> rhg1 </italic> candidate gene probably conferring <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic complementation analyses of <italic> <span class="gene">GmSNAP18</span> </italic> clearly showed that the transgenic ExF50 ( <italic> <span class="gene">GmSNAP18</span> </italic> <sup> &#x02212; </sup> <italic> <span class="gene">/GmSHMT08</span> </italic> <sup> <italic> &#x0002b; </italic> </sup> ) gained <span class="trait">resistance</span> to SCN after being complemented with the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> , but not when complemented with Forrest <italic> <span class="gene">AAT</span> </italic> ( <xref ref-type="fig" rid="f5"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						4 </xref> ), indicating that the Forrest <italic> <span class="gene">GmSNAP18</span> </italic> is the gene at the <italic> rhg1-a </italic> locus that confers <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-b </italic> ) or Essex <italic> <span class="gene">GmSNAP18</span> </italic> ( <italic> rhg1-s </italic> ) could not restore <span class="trait">resistance</span> to ExF50 ( <xref ref-type="fig" rid="f5"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						These data suggest that the resistant form of the PI 88788 <italic> <span class="gene">GmSNAP18</span> </italic> is unable to communicate with the Forrest <italic> <span class="gene">GmSHMT08</span> </italic> ( <italic> <span class="gene">Rhg4</span> </italic> ), to confer <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, the complementation results indicate that the SCN <span class="trait">resistance</span> at the <italic> rhg1-a </italic> locus in Forrest is triggered by <italic> <span class="gene">GmSNAP18</span> </italic> alone, contrasting with the <italic> rhg1-b </italic> in PI 88788, whose <span class="trait">resistance</span> is due to the high copy number and concerted activity of three genes <xref ref-type="bibr" rid="b7"> 7 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the <span class="trait">resistance</span> of Forrest and Peking to SCN requires the <italic> <span class="gene">Rhg4</span> </italic> gene <xref ref-type="bibr" rid="b6"> 6 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						All are in agreement with the previous genetic results, showing that the SCN <span class="trait">resistance</span> of Peking-type soybeans such as Forrest requires both <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> ( <italic> <span class="gene">GmSHMT08</span> </italic> ) <xref ref-type="bibr" rid="b2"> 2 </xref> <xref ref-type="bibr" rid="b3"> 3 </xref> <xref ref-type="bibr" rid="b6"> 6 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic and cDNA sequences of the <italic> <span class="gene">GmSNAP18</span> </italic> conferring SCN <span class="trait">resistance</span> were different between Peking-type soybeans and PI 88788-type soybeans ( <xref ref-type="fig" rid="f1"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, our results support a model wherein <italic> <span class="gene">GmSNAP18</span> </italic> is a major factor mediating the different types of soybean <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Our findings suggest that the Peking-type <italic> <span class="gene">GmSNAP18</span> </italic> is performing a different role in SCN <span class="trait">resistance</span> than PI 88788-type <italic> <span class="gene">GmSNAP18</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						As Forrest <span class="trait">resistance</span> to SCN requires both the <italic> rhg1-a </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes <xref ref-type="bibr" rid="b3"> 3 </xref> , genotyping was conducted using DNA markers flanking both loci to detect informative recombinants at the <italic> rhg1 </italic> locus ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean <italic> <span class="gene">GmSNAP18</span> </italic> gene underlies two types of <span class="trait">resistance</span> to soybean cyst nematode
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A Glycine max homolog of NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1 (NDR1) alters defense gene expression while functioning during a <span class="trait">resistance</span> response to different root pathogens in different genetic backgrounds.</h2>
			<h3>PMID:28273511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：McNeece BT(1), Pant SR(2), Sharma K(1), Niruala P(1), Lawrence GW(3), Klink VP(4).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A Glycine max homolog of the Arabidopsis thaliana NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1 (NDR1) coiled-coil nucleotide binding leucine rich repeat (CC-NB-LRR) defense signaling gene <span class="gene">(Gm-NDR1-1)</span> is expressed in root cells undergoing a defense response to the root pathogenic nematode, <span class="trait">Heterodera</span> <span class="trait">glycines</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The results indicate a common genetic basis exists for plant <span class="trait">resistance</span> to parasitic nematodes that involves <span class="gene">Gm-NDR1</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Disease <span class="trait">resistance</span> through impairment of α-SNAP-NSF interaction and vesicular trafficking by soybean Rhg1.</h2>
			<h3>PMID:27821740</h3>
				

				
			<p>
				作者：Bayless AM(1), Smith JM(1), Song J(1), McMinn PH(1), Teillet A(1), August BK(2), Bent AF(3).
				<br/> 
				年份：2016 Nov 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						However, certain haplotypes of the <span class="gene">Rhg1</span> <span class="trait">(resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> 1) locus of soybean possess multiple repeat copies of an α-SNAP gene (Glyma.
					</p>
				
					<p>&nbsp;&nbsp;
						These <span class="gene">Rhg1</span> loci mediate <span class="trait">resistance</span> to soybean cyst nematode (SCN; H.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rhg1</span> is widely used in agriculture, but the mechanisms of <span class="gene">Rhg1</span> disease <span class="trait">resistance</span> have remained unclear.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impact of <span class="gene">Rhg1</span> copy number, type, and interaction with <span class="gene">Rhg4</span> on <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> in soybean.</h2>
			<h3>PMID:27581541  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yu N(1), Lee TG(1)(2), Rosa DP(3), Hudson M(1), Diers BW(4).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Evaluations of soybean populations showed that both <span class="gene">Rhg1</span> copy number and type were important in determining soybean cyst nematode <span class="trait">resistance</span> with higher copy number within <span class="gene">Rhg1</span> type conferring greater <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> are important loci conferring <span class="trait">resistance</span> to soybean cyst nematode (SCN; <span class="trait">Heterodera</span> <span class="trait">glycines)</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Alleles at <span class="gene">Rhg1</span> have been shown to vary for copy number and type and the importance of this variation in conferring <span class="trait">resistance</span> is not well defined.
					</p>
				
					<p>&nbsp;&nbsp;
						We developed populations segregating for <span class="gene">Rhg1</span> copy number and type and <span class="gene">Rhg4</span> allele type to investigate the effect of these factors and their interaction on SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						While plants with P type <span class="gene">Rhg1</span> required <span class="gene">Rhg4</span> to achieve full <span class="trait">resistance,</span> <span class="gene">Rhg4</span> did not increase <span class="trait">resistance</span> in the background of F + W type <span class="gene">Rhg1</span> repeat.
					</p>
				
					<p>&nbsp;&nbsp;
						This study demonstrates the importance of both <span class="gene">Rhg1</span> copy number and type in determining <span class="trait">resistance</span> and can assist soybean breeders in determining what alleles would best fit their breeding goals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soybean overexpressing <span class="gene">GmSAMT1</span> exhibits <span class="trait">resistance</span> to multiple-HG types of soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:27064027  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M(1), Zhao N(2), Hatcher CN(1), Wuddineh WA(1), Rudis M(1), Tschaplinski TJ(2), Pantalone VR(1), Arelli PR(3), Hewezi T(1), Chen F(1), Stewart CN Jr(4).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Prior results showed that when <span class="gene">GmSAMT1</span> was overexpressed in transgenic soybean hairy roots, <span class="trait">resistance</span> is conferred against soybean cyst nematode (SCN), <span class="trait">Heterodera</span> <span class="trait">glycines</span> Ichinohe.
					</p>
				
					<p>&nbsp;&nbsp;
						Among transgenic lines, <span class="gene">GmSAMT1</span> expression in roots was positively associated with SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						We hypothesize that <span class="gene">GmSAMT1</span> expression affects salicylic acid biosynthesis, which, in turn, attenuates SCN development, without negative consequences to soybean <span class="trait">yield</span> or other morphological traits.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, we conclude that <span class="gene">GmSAMT1</span> overexpression confers broad <span class="trait">resistance</span> to multiple SCN races, which would be potentially applicable to commercial production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A SNARE-Like Protein and Biotin Are Implicated in Soybean Cyst Nematode Virulence</h2>
			<h3>PMID:26714307</h3>
				

				
					<h3>PMCID:4699853</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bekal </surname>  <given-names> Sadia </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Domier </surname>  <given-names> Leslie L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gonfa </surname>  <given-names> Biruk </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lakhssassi </surname>  <given-names> Naoufal </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lambert </surname>  <given-names> Kris N. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Furthermore, in bacterial co-expression experiments, HgSLP-1 co-purified with the SCN <span class="trait">resistance</span> protein <span class="gene">Rhg1</span> &#x003b1;-SNAP, suggesting that these two proteins physically interact.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> SCN <span class="trait">resistance</span> gene from cv Forrest (Peking-type <span class="trait">resistance)</span> was recently map-based cloned using targeting induced local lesions in genomes (TILLING), in combination with gene complementation and gene silencing [ <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> also encoded a novel type of plant <span class="trait">resistance</span> gene, a serine hydroxymethyltransferase (SHMT), which is an enzyme involved in one carbon folate metabolism.
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg4</span> </italic> also requires <italic> <span class="gene">Rhg1</span> </italic> to function fully, indicating the two seemingly different SCN <span class="trait">resistance</span> mechanisms work together in some unknown, but important way [ <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, one might expect the unusual nematode <span class="trait">resistance</span> genes found at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci would require SCN to deploy equally unique mechanisms to overcome these atypical types of <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						A pool of F3 single cyst derived populations was used to inoculate soybean plants containing the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, or susceptible plants harboring only the susceptible <italic> <span class="gene">Rhg1</span> </italic> alleles.
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN populations used in the selection experiments contained both virulent and avirulent SCN, thus one would expect the frequency of SCN virulence genes to increase in SCN populations grown on soybean harboring the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, but not on the susceptible plants.
					</p>
				
					<p>&nbsp;&nbsp;
						This landmark study established that virulence on <italic> <span class="gene">Rhg1</span> </italic> -type <span class="trait">resistance</span> was genetically controlled by the dominant <italic> Ror1 </italic> gene (reproduction on resistant) in the virulent strain OP50, but OP50 growth on <italic> <span class="gene">Rhg4</span> </italic> / <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance</span> was controlled by the recessive <italic> ror2 </italic> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						The virulent inbred SCN strain OP20 was shown to require two virulence genes to reproduce on the <italic> <span class="gene">Rhg1</span> </italic> source of <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						This study suggested that SCN utilizes several mechanisms to overcome <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance,</span> but only one gene, <italic> ror2 </italic> , to overcome <italic> Rhg1/Rhg4 </italic> -type <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						However, it is interesting that the <italic> ror2 </italic> virulence gene that controls growth on the <italic> <span class="gene">Rhg1</span> </italic> / <italic> <span class="gene">Rhg4</span> </italic> -type <span class="trait">resistance</span> is recessive.
					</p>
				
					<p>&nbsp;&nbsp;
						However in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> plants have up to 10 copies of <italic> <span class="gene">Rhg1</span> </italic> &#x003b1;-SNAP, which may allow them to overcome this putative sequestration.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of plant membrane fusion proteins in SCN-resistance is supported by the finding that one of the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> genes encodes a &#x003b1;-SNAP protein.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rhg1</span> &#x003b1;-SNAP proteins may confer SCN <span class="trait">resistance</span> by themselves [ <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						ref050" ref-type="bibr"> 50 </xref> ] or in combination with the other <span class="trait">resistance</span> genes at the <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						It will also be important to follow up on the involvement in biotin in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> by further analysis of the biotin biosynthetic enzymes in the plant and nematode.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, in bacterial co-expression experiments, HgSLP-1 co-purified with the SCN <span class="trait">resistance</span> protein <span class="gene">Rhg1</span> &#x003b1;-SNAP, suggesting that these two proteins physically interact
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> SCN <span class="trait">resistance</span> gene from cv Forrest (Peking-type <span class="trait">resistance)</span> was recently map-based cloned using targeting induced local lesions in genomes (TILLING), in combination with gene complementation and gene silencing [ <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> also encoded a novel type of plant <span class="trait">resistance</span> gene, a serine hydroxymethyltransferase (SHMT), which is an enzyme involved in one carbon folate metabolism
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg4</span> </italic> also requires <italic> <span class="gene">Rhg1</span> </italic> to function fully, indicating the two seemingly different SCN <span class="trait">resistance</span> mechanisms work together in some unknown, but important way [ <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, one might expect the unusual nematode <span class="trait">resistance</span> genes found at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci would require SCN to deploy equally unique mechanisms to overcome these atypical types of <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						A pool of F3 single cyst derived populations was used to inoculate soybean plants containing the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, or susceptible plants harboring only the susceptible <italic> <span class="gene">Rhg1</span> </italic> alleles
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN populations used in the selection experiments contained both virulent and avirulent SCN, thus one would expect the frequency of SCN virulence genes to increase in SCN populations grown on soybean harboring the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> locus, but not on the susceptible plants
					</p>
				
					<p>&nbsp;&nbsp;
						This landmark study established that virulence on <italic> <span class="gene">Rhg1</span> </italic> -type <span class="trait">resistance</span> was genetically controlled by the dominant <italic> Ror1 </italic> gene (reproduction on resistant) in the virulent strain OP50, but OP50 growth on <italic> <span class="gene">Rhg4</span> </italic> / <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance</span> was controlled by the recessive <italic> ror2 </italic> gene
					</p>
				
					<p>&nbsp;&nbsp;
						The virulent inbred SCN strain OP20 was shown to require two virulence genes to reproduce on the <italic> <span class="gene">Rhg1</span> </italic> source of <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						This study suggested that SCN utilizes several mechanisms to overcome <italic> <span class="gene">Rhg1</span> </italic> -based <span class="trait">resistance,</span> but only one gene, <italic> ror2 </italic> , to overcome <italic> Rhg1/Rhg4 </italic> -type <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						However, it is interesting that the <italic> ror2 </italic> virulence gene that controls growth on the <italic> <span class="gene">Rhg1</span> </italic> / <italic> <span class="gene">Rhg4</span> </italic> -type <span class="trait">resistance</span> is recessive
					</p>
				
					<p>&nbsp;&nbsp;
						However in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> plants have up to 10 copies of <italic> <span class="gene">Rhg1</span> </italic> &#x003b1;-SNAP, which may allow them to overcome this putative sequestration
					</p>
				
					<p>&nbsp;&nbsp;
						The role of plant membrane fusion proteins in SCN-resistance is supported by the finding that one of the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> genes encodes a &#x003b1;-SNAP protein
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rhg1</span> &#x003b1;-SNAP proteins may confer SCN <span class="trait">resistance</span> by themselves [ <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						ref050" ref-type="bibr"> 50 </xref> ] or in combination with the other <span class="trait">resistance</span> genes at the <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						It will also be important to follow up on the involvement in biotin in <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> by further analysis of the biotin biosynthetic enzymes in the plant and nematode
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genomic-assisted phylogenetic analysis and marker development for next generation soybean cyst nematode <span class="trait">resistance</span> breeding.</h2>
			<h3>PMID:26566850  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kadam S(1), Vuong TD(2), Qiu D(3), Meinhardt CG(4), Song L(5), Deshmukh R(6), Patil G(7), Wan J(8), Valliyodan B(9), Scaboo AM(10), Shannon JG(11), Nguyen HT(12).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, 19,652 publicly available soybean accessions that were previously genotyped with the SoySNP50K iSelect BeadChip were used to evaluate the phylogenetic diversity of SCN <span class="trait">resistance</span> genes <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> in an attempt to identify novel sources of <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, markers for the <span class="gene">Rhg4</span> gene were able to detect Peking-type <span class="trait">(resistance)</span> genotypes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The syntaxin 31-induced gene, LESION SIMULATING DISEASE1 (LSD1), functions in Glycine max defense to the root parasite <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:25530246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pant SR(1), Krishnavajhala A, McNeece BT, Lawrence GW, Klink VP.
				<br/> 
				年份：2015
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments show the membrane fusion genes α soluble NSF attachment protein (α-SNAP) and syntaxin 31 <span class="gene">(Gm-SYP38)</span> contribute to the ability of Glycine max to defend itself from infection by the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and evaluation of quantitative trait loci underlying <span class="trait">resistance</span> to multiple HG types of soybean cyst nematode in soybean PI 437655</h2>
			<h3>PMID:25316311</h3>
				

				
					<h3>PMCID:4282714</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Jiao </surname>  <given-names> Yongqing </given-names>  </name>  <address>  <phone> 573-882-5494 </phone>  <email> yongqing.jiao@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meinhardt </surname>  <given-names> Clinton </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <phone> 573-882-5494 </phone>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology (NCSB), University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Department of Computer Science, Informatics Institute and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff3">  <label/> Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, MD 20705 USA  </aff>  <aff id="Aff4">  <label/> Division of Plant Sciences and NCSB, University of Missouri, Delta Center, P.O. Box 160, Portageville, MO 63873 USA  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> We performed QTL analysis for SCN <span class="trait">resistance</span> in PI 437655 in two mapping populations, characterized CNV of </bold> <bold> <italic> <span class="gene">Rhg1</span> </italic> </bold> <bold> through whole-genome resequencing and evaluated the effects of QTL pyramiding to enhance <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						18, which corresponds to the known <italic> <span class="gene">Rhg1</span> </italic> locus, contributed <span class="trait">resistance</span> to SCN HG types 1.
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was found to be controlled by three genes (Cook et al.
					</p>
				
					<p>&nbsp;&nbsp;
						PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> can confer soybean SCN <span class="trait">resistance</span> by itself (Kim et al.
					</p>
				
					<p>&nbsp;&nbsp;
						In our study, a significant association was not observed between markers in the <italic> <span class="gene">Rhg4</span> </italic> interval and <span class="trait">resistance</span> to any SCN isolate tested (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been widely mapped in various sources of SCN <span class="trait">resistance</span> (Concibido et al.
					</p>
				
					<p>&nbsp;&nbsp;
						To date, many sources of SCN <span class="trait">resistance</span> contain <italic> <span class="gene">Rhg1</span> </italic> (Concibido et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Because the level of SCN <span class="trait">resistance</span> conferred by the <italic> <span class="gene">Rhg1</span> </italic> gene was associated with CNVs (Cook et al.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="bibr" rid="CR11"> 2012 </xref> ), it is important to determine CNVs of <italic> <span class="gene">Rhg1</span> </italic> among those sources of <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						CNV analysis showed that PI 437655 had the same high number of copies of <italic> <span class="gene">Rhg1</span> </italic> as PI 88788, which indicated PI 437655 and PI 88788 might have the same level of SCN <span class="trait">resistance</span> or the same <italic> <span class="gene">Rhg1</span> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						The most important gene underlying SCN <span class="trait">resistance</span> in PI 88788 is <italic> <span class="gene">Rhg1</span> </italic> (Cook et al.
					</p>
				
					<p>&nbsp;&nbsp;
						20 in each of two populations had enhanced SCN <span class="trait">resistance</span> than genotypes with the <italic> <span class="gene">Rhg1</span> </italic> allele alone.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> We performed QTL analysis for SCN <span class="trait">resistance</span> in PI 437655 in two mapping populations, characterized CNV of </bold> <bold> <italic> <span class="gene">Rhg1</span> </italic> </bold> <bold> through whole-genome resequencing and evaluated the effects of QTL pyramiding to enhance <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						18, which corresponds to the known <italic> <span class="gene">Rhg1</span> </italic> locus, contributed <span class="trait">resistance</span> to SCN HG types 1
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> conferred by <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was found to be controlled by three genes (Cook et al
					</p>
				
					<p>&nbsp;&nbsp;
						PI 88788-type <italic> <span class="gene">Rhg1</span> </italic> can confer soybean SCN <span class="trait">resistance</span> by itself (Kim et al
					</p>
				
					<p>&nbsp;&nbsp;
						In our study, a significant association was not observed between markers in the <italic> <span class="gene">Rhg4</span> </italic> interval and <span class="trait">resistance</span> to any SCN isolate tested (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been widely mapped in various sources of SCN <span class="trait">resistance</span> (Concibido et al
					</p>
				
					<p>&nbsp;&nbsp;
						To date, many sources of SCN <span class="trait">resistance</span> contain <italic> <span class="gene">Rhg1</span> </italic> (Concibido et al
					</p>
				
					<p>&nbsp;&nbsp;
						Because the level of SCN <span class="trait">resistance</span> conferred by the <italic> <span class="gene">Rhg1</span> </italic> gene was associated with CNVs (Cook et al
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="bibr" rid="CR11"> 2012 </xref> ), it is important to determine CNVs of <italic> <span class="gene">Rhg1</span> </italic> among those sources of <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						CNV analysis showed that PI 437655 had the same high number of copies of <italic> <span class="gene">Rhg1</span> </italic> as PI 88788, which indicated PI 437655 and PI 88788 might have the same level of SCN <span class="trait">resistance</span> or the same <italic> <span class="gene">Rhg1</span> </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						The most important gene underlying SCN <span class="trait">resistance</span> in PI 88788 is <italic> <span class="gene">Rhg1</span> </italic> (Cook et al
					</p>
				
					<p>&nbsp;&nbsp;
						20 in each of two populations had enhanced SCN <span class="trait">resistance</span> than genotypes with the <italic> <span class="gene">Rhg1</span> </italic> allele alone
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Manipulation of two α-endo-β-1,4-glucanase genes, AtCel6 and <span class="gene">GmCel7,</span> reduces susceptibility to <span class="trait">Heterodera</span> <span class="trait">glycines</span> in soybean roots.</h2>
			<h3>PMID:24844661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Woo MO(1), Beard H, MacDonald MH, Brewer EP, Youssef RM, Kim H, Matthews BF.
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Arabidopsis α-EGase <span class="gene">At1g48930,</span> designated as AtCel6, is known to be down-regulated by beet cyst nematode <span class="trait">(Heterodera</span> schachtii) in Arabidopsis roots, whereas another α-EGase, AtCel2, is up-regulated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distinct Copy Number, Coding Sequence, and Locus Methylation Patterns Underlie Rhg1-Mediated Soybean <span class="trait">Resistance</span> to Soybean Cyst Nematode.</h2>
			<h3>PMID:24733883</h3>
				

				
			<p>
				作者：Cook DE(1), Bayless AM(1), Wang K(1), Guo X(1), Song Q(1), Jiang J(1), Bent AF(2).
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rhg1</span> was recently discovered to be a complex locus at which resistance-conferring haplotypes carry up to 10 tandem repeat copies of a 31-kb DNA segment, and three disparate genes present on each repeat contribute to SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on copy number variation, transcript abundance, nucleic acid polymorphisms, and differentially methylated DNA regions, we find that SCN <span class="trait">resistance</span> is associated with multicopy <span class="gene">Rhg1</span> haplotypes that form two distinct groups.
					</p>
				
					<p>&nbsp;&nbsp;
						Differentially methylated DNA regions also were identified within <span class="gene">Rhg1</span> that correlate with SCN <span class="trait">resistance</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Syntaxin 31 functions in Glycine max <span class="trait">resistance</span> to the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines.</span></h2>
			<h3>PMID:24452833  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pant SR(1), Matsye PD, McNeece BT, Sharma K, Krishnavajhala A, Lawrence GW, Klink VP.
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A Glycine max syntaxin 31 homolog <span class="gene">(Gm-SYP38)</span> was identified as being expressed in nematode-induced feeding structures known as syncytia undergoing an incompatible interaction with the plant parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The observed <span class="gene">Gm-SYP38</span> expression was consistent with prior gene expression analyses that identified the alpha soluble NSF attachment protein <span class="gene">(Gm-α-SNAP)</span> <span class="trait">resistance</span> gene because homologs of these genes physically interact and function together in other genetic systems.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean salicylic acid methyltransferase gene confers <span class="trait">resistance</span> to soybean cyst nematode.</h2>
			<h3>PMID:24034273  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M, Zhao N, Zhu JJ, Zhuang X, Liu W, Pantalone VR, Arelli PR, Stewart CN Jr, Chen F.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Here, we report that a SAMT gene from soybean <span class="gene">(GmSAMT1)</span> plays a role in soybean defence against soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines</span> Ichinohe, SCN).
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the role of <span class="gene">GmSAMT1</span> in soybean defence against SCN, transgenic hairy roots overexpressing <span class="gene">GmSAMT1</span> were produced and tested for SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <span class="gene">GmSAMT1</span> in SCN-susceptible backgrounds significantly reduced the development of SCN, indicating that overexpression of <span class="gene">GmSAMT1</span> in the transgenic hairy root system could confer <span class="trait">resistance</span> to SCN.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homo-dimerization and ligand binding by the leucine-rich repeat domain at <italic> RHG1/RFS2 </italic> underlying <span class="trait">resistance</span> to two soybean pathogens</h2>
			<h3>PMID:23497186</h3>
				

				
					<h3>PMCID:3626623</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Afzal </surname>  <given-names> Ahmed J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> afzal.5@osu.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Srour </surname>  <given-names> Ali </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> asrour@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Goil </surname>  <given-names> Abhishek </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> abhishekgoil81@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Vasudaven </surname>  <given-names> Sheeja </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> isheeja@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Liu </surname>  <given-names> Tianyun </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> isheeja@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Samudrala </surname>  <given-names> Ram </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ram@compbio.washington.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Dogra </surname>  <given-names> Navneet </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> navneet@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Kohli </surname>  <given-names> Punit </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> pkohli@chem.siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Malakar </surname>  <given-names> Ayan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mailme.ayan@siu.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Lightfoot </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ga4082@siu.edu </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The CLE peptides <span class="gene">GmTGIF,</span> <span class="gene">GmCLE34,</span> <span class="gene">GmCLE3</span> and HgCLE were previously reported to be involved in root growth inhibition but here <span class="gene">GmTGIF</span> and HgCLE were shown to alter stem morphology and <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the LRR domain of <span class="gene">GmRLK18-1</span> might underlie both root development and disease <span class="trait">resistance</span> in soybean and provide an avenue to develop new variants and ligands that might promote reduced losses to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Two loci, <italic> <span class="gene">Rhg4</span> </italic> on chromosome 8 (linkage group (Lg) A2) and <italic> Rhg1/Rfs2 </italic> on chromosome 18 (Lg G), contain genes that encode receptor like kinase (RLK) proteins within the RPK gene family implicated in <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						However, only <span class="gene">GmRLK18-1</span> has been shown to underlie <span class="trait">resistance</span> to both pathogens in transgenic plants [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						At <italic> Rhg1/Rfs2, </italic> the <span class="trait">resistance</span> phenotypes were perfectly associated with the <span class="gene">GmRLK18-1</span> allotype 1 [ <xref ref-type="bibr" rid="B5"> 5 </xref> ] and that allele in transgenic plants provided partial <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Here a model of the LRR domain from a <span class="gene">RLK</span> protein involved in <span class="trait">resistance</span> is reported based on homology modeling of the extracellular LRR domain (residue 141&#x02013;471) of the <span class="gene">GmRLK18-1</span> protein.
					</p>
				
					<p>&nbsp;&nbsp;
						This result would agree with the conclusion that <span class="gene">GmRLK08-1</span> protein was not the sole element underlying the <span class="trait">resistance</span> reaction encoded at the <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="B9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						During SCN pathogenesis a new meristem is initiated to bring a tracheary element close to the feeding site, N might mediate that and be detected by the <span class="trait">resistance</span> protein <span class="gene">GmRLK18-1</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						This report describes a functional analysis and structural prediction of the LRR domain from a <span class="gene">RLK</span> protein involved in plant pest and disease <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The CLE peptides <span class="gene">GmTGIF,</span> <span class="gene">GmCLE34,</span> <span class="gene">GmCLE3</span> and HgCLE were previously reported to be involved in root growth inhibition but here <span class="gene">GmTGIF</span> and HgCLE were shown to alter stem morphology and <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the LRR domain of <span class="gene">GmRLK18-1</span> might underlie both root development and disease <span class="trait">resistance</span> in soybean and provide an avenue to develop new variants and ligands that might promote reduced losses to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Two loci, <italic> <span class="gene">Rhg4</span> </italic> on chromosome 8 (linkage group (Lg) A2) and <italic> Rhg1/Rfs2 </italic> on chromosome 18 (Lg G), contain genes that encode receptor like kinase (RLK) proteins within the RPK gene family implicated in <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						However, only <span class="gene">GmRLK18-1</span> has been shown to underlie <span class="trait">resistance</span> to both pathogens in transgenic plants [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						At <italic> Rhg1/Rfs2, </italic> the <span class="trait">resistance</span> phenotypes were perfectly associated with the <span class="gene">GmRLK18-1</span> allotype 1 [ <xref ref-type="bibr" rid="B5"> 5 </xref> ] and that allele in transgenic plants provided partial <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Here a model of the LRR domain from a <span class="gene">RLK</span> protein involved in <span class="trait">resistance</span> is reported based on homology modeling of the extracellular LRR domain (residue 141&#x02013;471) of the <span class="gene">GmRLK18-1</span> protein
					</p>
				
					<p>&nbsp;&nbsp;
						This result would agree with the conclusion that <span class="gene">GmRLK08-1</span> protein was not the sole element underlying the <span class="trait">resistance</span> reaction encoded at the <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						During SCN pathogenesis a new meristem is initiated to bring a tracheary element close to the feeding site, N might mediate that and be detected by the <span class="trait">resistance</span> protein <span class="gene">GmRLK18-1</span>
					</p>
				
					<p>&nbsp;&nbsp;
						This report describes a functional analysis and structural prediction of the LRR domain from a <span class="gene">RLK</span> protein involved in plant pest and disease <span class="trait">resistance</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A soybean cyst nematode <span class="trait">resistance</span> gene points to a new mechanism of plant <span class="trait">resistance</span> to pathogens.</h2>
			<h3>PMID:23235880  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu S(1), Kandoth PK, Warren SD, Yeckel G, Heinz R, Alden J, Yang C, Jamai A, El-Mellouki T, Juvale PS, Hill J, Baum TJ, Cianzio S, Whitham SA, Korkin D, Mitchum MG, Meksem K.
				<br/> 
				年份：2012 Dec 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Here we report the map-based cloning of a gene at the <span class="gene">Rhg4</span> (for <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines</span> 4) locus, a major quantitative trait locus contributing to <span class="trait">resistance</span> to this pathogen.
					</p>
				
					<p>&nbsp;&nbsp;
						Alleles of <span class="gene">Rhg4</span> conferring <span class="trait">resistance</span> or susceptibility differ by two genetic polymorphisms that alter a key regulatory property of the enzyme.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic relationships of soybean cyst nematode <span class="trait">resistance</span> originated in Gedenshirazu and PI84751 on <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci</h2>
			<h3>PMID:23136498</h3>
				

				
					<h3>PMCID:3406782</h3>
					<h4>期刊:Breeding Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Suzuki </surname>  <given-names> Chika </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-602"> 1 </xref>  <xref rid="c1-bs-61-602" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Yoshinori </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-602"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takeuchi </surname>  <given-names> Toru </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-602"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yumoto </surname>  <given-names> Setsuzo </given-names>  </name>  <xref ref-type="aff" rid="af4-bs-61-602"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shirai </surname>  <given-names> Shigehisa </given-names>  </name>  <xref ref-type="aff" rid="af3-bs-61-602"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We confirmed that race 1 <span class="trait">resistance</span> in PI84751 was independently controlled by 4 genes, 2 of which were <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> (1960) </xref> reported that 3 recessive genes, <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> and <italic> rhg3 </italic> , control <span class="trait">resistance</span> to SCN in Peking, and that PI90763 and PI84751 possessed the same 3 <span class="trait">resistance</span> genes as Peking.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> was reported as a dominant <span class="trait">resistance</span> gene that is tightly linked to the <italic> I </italic> locus, which controls pigment distribution on the seed coat ( <xref rid="b11-bs-61-602" ref-type="bibr"> Matson and Williams 1965 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> has been mapped to LG A2 from several sources ( <xref rid="b3-bs-61-602" ref-type="bibr"> Concibido <italic> et al.
					</p>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to explain the genetic relationships of SCN <span class="trait">resistance</span> derived from Gedenshirazu and PI84751 by focusing on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci.
					</p>
				
					<p>&nbsp;&nbsp;
						In the second part, we evaluated the adaptability of the candidate markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which when used with our varieties conferred <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In the last part, we explained the relationships between PI84751 and Gedenshirazu with regard to the markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which were used in the analysis of the SCN <span class="trait">resistance</span> of BC <sub> 1 </sub> F <sub> 2 </sub> individuals.
					</p>
				
					<p>&nbsp;&nbsp;
						Eighty-eight breeding strains were used to confirm the relationship between SCN <span class="trait">resistance</span> and the genotypes of simple sequence repeat (SSR) markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						The 10 lines exhibiting race 1 <span class="trait">resistance</span> harbored both PI84751 alleles on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, we assumed that the genotype of Satt309 was closely linked to the <italic> <span class="gene">Rhg1</span> </italic> locus, the genotype of Satt632 was closely linked to the <italic> <span class="gene">Rhg4</span> </italic> locus and <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> were 2 of the 4 <span class="trait">resistance</span> genes of PI84751.
					</p>
				
					<p>&nbsp;&nbsp;
						Conversely, race 3-resistant and race 1-susceptible lines did not express the PI84751 type of <italic> <span class="gene">Rhg4</span> </italic> ( <xref rid="t2-bs-61-602" ref-type="table"> Table 2 </xref> ), indicating that <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the cross between race 3-resistant strains derived from Gedenshirazu and race 1-resistant strains derived from PI84751, the lines that expressed Gedenshirazu alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus did not exhibit <span class="trait">resistance</span> to race 1; only the lines that expressed PI84751 alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus exhibited race 1 <span class="trait">resistance</span> ( <xref rid="t5-bs-61-602" ref-type="table"> Table 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The result of this cross indicated that PI84751 and Gedenshirazu each have a different <span class="trait">resistance</span> allele at the <italic> <span class="gene">Rhg1</span> </italic> locus and supported the existence of <italic> rhg1 </italic> alleles as reported by <xref rid="b1-bs-61-602" ref-type="bibr"> Brucker (2005) </xref> in the cross between PI437654 and PI88788.
					</p>
				
					<p>&nbsp;&nbsp;
						On the relationship between <span class="trait">resistance</span> to SCN and genotypes of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, the combination of alleles on the two loci could explain the type of <span class="trait">resistance</span> to SCN ( <xref rid="t3-bs-61-602" ref-type="table"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Both alleles of <italic> rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> were indispensable for race 1 <span class="trait">resistance;</span> however, either <italic> rhg1-s </italic> or <italic> rhg1-g </italic> was necessary for race 3 <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance</span> as shown by the analysis of TC9703RILs.
					</p>
				
					<p>&nbsp;&nbsp;
						The QTLs for <span class="trait">resistance</span> to SCN race 3 originating from Gedenshirazu were not detected on LG A2, where <italic> <span class="gene">Rhg4</span> </italic> lies ( <xref rid="b5-bs-61-602" ref-type="bibr"> Ferdous <italic> et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Because <span class="trait">resistance</span> to SCN is controlled by multiple <span class="trait">resistance</span> genes, we speculate that the strains lacked <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> and/or <italic> rhg3 </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						These results demonstrated that 2 SCN resistant genes differed in Gedenshirazu and PI84751, and differences in 2 loci such as <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> between Gedenshirazu and PI84751 controlled <span class="trait">resistance</span> to race 1.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, they indicated that the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg4</span> </italic> locus was dominant and the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg1</span> </italic> locus was recessive.
					</p>
				
					<p>&nbsp;&nbsp;
						The reason our result differed from Shirai&#x02019;s is that their data was not based on the evaluation of a progeny test but on the evaluation of F <sub> 2 </sub> individuals, and it is possible that the susceptible parents had <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> or <italic> rhg3 </italic> , which could not detected by the phenotypes.
					</p>
				
					<p>&nbsp;&nbsp;
						We confirmed that race 1 <span class="trait">resistance</span> in PI84751 was independently controlled by 4 genes, 2 of which were <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> (1960) </xref> reported that 3 recessive genes, <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> and <italic> rhg3 </italic> , control <span class="trait">resistance</span> to SCN in Peking, and that PI90763 and PI84751 possessed the same 3 <span class="trait">resistance</span> genes as Peking
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> was reported as a dominant <span class="trait">resistance</span> gene that is tightly linked to the <italic> I </italic> locus, which controls pigment distribution on the seed coat ( <xref rid="b11-bs-61-602" ref-type="bibr"> Matson and Williams 1965 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> has been mapped to LG A2 from several sources ( <xref rid="b3-bs-61-602" ref-type="bibr"> Concibido <italic> et al
					</p>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to explain the genetic relationships of SCN <span class="trait">resistance</span> derived from Gedenshirazu and PI84751 by focusing on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci
					</p>
				
					<p>&nbsp;&nbsp;
						In the second part, we evaluated the adaptability of the candidate markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which when used with our varieties conferred <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						In the last part, we explained the relationships between PI84751 and Gedenshirazu with regard to the markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which were used in the analysis of the SCN <span class="trait">resistance</span> of BC <sub> 1 </sub> F <sub> 2 </sub> individuals
					</p>
				
					<p>&nbsp;&nbsp;
						Eighty-eight breeding strains were used to confirm the relationship between SCN <span class="trait">resistance</span> and the genotypes of simple sequence repeat (SSR) markers linked to <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						The 10 lines exhibiting race 1 <span class="trait">resistance</span> harbored both PI84751 alleles on the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, we assumed that the genotype of Satt309 was closely linked to the <italic> <span class="gene">Rhg1</span> </italic> locus, the genotype of Satt632 was closely linked to the <italic> <span class="gene">Rhg4</span> </italic> locus and <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> were 2 of the 4 <span class="trait">resistance</span> genes of PI84751
					</p>
				
					<p>&nbsp;&nbsp;
						Conversely, race 3-resistant and race 1-susceptible lines did not express the PI84751 type of <italic> <span class="gene">Rhg4</span> </italic> ( <xref rid="t2-bs-61-602" ref-type="table"> Table 2 </xref> ), indicating that <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In the cross between race 3-resistant strains derived from Gedenshirazu and race 1-resistant strains derived from PI84751, the lines that expressed Gedenshirazu alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus did not exhibit <span class="trait">resistance</span> to race 1; only the lines that expressed PI84751 alleles of the <italic> <span class="gene">Rhg1</span> </italic> locus exhibited race 1 <span class="trait">resistance</span> ( <xref rid="t5-bs-61-602" ref-type="table"> Table 5 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The result of this cross indicated that PI84751 and Gedenshirazu each have a different <span class="trait">resistance</span> allele at the <italic> <span class="gene">Rhg1</span> </italic> locus and supported the existence of <italic> rhg1 </italic> alleles as reported by <xref rid="b1-bs-61-602" ref-type="bibr"> Brucker (2005) </xref> in the cross between PI437654 and PI88788
					</p>
				
					<p>&nbsp;&nbsp;
						On the relationship between <span class="trait">resistance</span> to SCN and genotypes of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, the combination of alleles on the two loci could explain the type of <span class="trait">resistance</span> to SCN ( <xref rid="t3-bs-61-602" ref-type="table"> Table 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Both alleles of <italic> rhg1-s </italic> and <italic> <span class="gene">Rhg4</span> </italic> were indispensable for race 1 <span class="trait">resistance;</span> however, either <italic> rhg1-s </italic> or <italic> rhg1-g </italic> was necessary for race 3 <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, <italic> <span class="gene">Rhg4</span> </italic> was not related to race 3 <span class="trait">resistance</span> as shown by the analysis of TC9703RILs
					</p>
				
					<p>&nbsp;&nbsp;
						The QTLs for <span class="trait">resistance</span> to SCN race 3 originating from Gedenshirazu were not detected on LG A2, where <italic> <span class="gene">Rhg4</span> </italic> lies ( <xref rid="b5-bs-61-602" ref-type="bibr"> Ferdous <italic> et al
					</p>
				
					<p>&nbsp;&nbsp;
						Because <span class="trait">resistance</span> to SCN is controlled by multiple <span class="trait">resistance</span> genes, we speculate that the strains lacked <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> and/or <italic> rhg3 </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						These results demonstrated that 2 SCN resistant genes differed in Gedenshirazu and PI84751, and differences in 2 loci such as <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> between Gedenshirazu and PI84751 controlled <span class="trait">resistance</span> to race 1
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, they indicated that the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg4</span> </italic> locus was dominant and the <span class="trait">resistance</span> gene at the <italic> <span class="gene">Rhg1</span> </italic> locus was recessive
					</p>
				
					<p>&nbsp;&nbsp;
						The reason our result differed from Shirai&#x02019;s is that their data was not based on the evaluation of a progeny test but on the evaluation of F <sub> 2 </sub> individuals, and it is possible that the susceptible parents had <span class="trait">resistance</span> genes such as <italic> <span class="gene">rhg2</span> </italic> or <italic> rhg3 </italic> , which could not detected by the phenotypes
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The receptor like kinase at Rhg1-a/Rfs2 caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome and soybean cyst nematode as a transgene by altering signaling responses.</h2>
			<h3>PMID:22857610  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Srour A(1), Afzal AJ, Blahut-Beatty L, Hemmati N, Simmonds DH, Li W, Liu M, Town CD, Sharma H, Arelli P, Lightfoot DA.
				<br/> 
				年份：2012 Aug 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This study examined the role of a receptor like kinase (RLK) <span class="gene">GmRLK18-1</span> (gene model Glyma_18_02680 at 1,071 kbp on chromosome 18 of the genome sequence) within the Rhg1/Rfs2 locus in causing <span class="trait">resistance</span> to SCN and SDS.
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence alignments inferred that the <span class="trait">resistance</span> allele was an introgressed region of about 59 kbp at the center of which the <span class="gene">GmRLK18-1</span> was the most polymorphic gene and encoded protein.
					</p>
				
					<p>&nbsp;&nbsp;
						Analyses were made of plants that were either heterozygous at, or transgenic (and so hemizygous at a new location) with, the <span class="trait">resistance</span> allele of <span class="gene">GmRLK18-1</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the absence of <span class="gene">Rhg4</span> the <span class="gene">GmRLK18-1</span> was sufficient to confer nearly complete <span class="trait">resistance</span> to both root and leaf symptoms of SDS caused by F.
					</p>
				
					<p>&nbsp;&nbsp;
						A reduction in the rate of early seedling root development was also shown to be caused by the <span class="trait">resistance</span> allele of the <span class="gene">GmRLK18-1</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Soybean <span class="gene">Rhg1</span> locus for <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> regulates the expression of a large number of stress- and defense-related genes in degenerating feeding cells.</h2>
			<h3>PMID:21335526  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kandoth PK(1), Ithal N, Recknor J, Maier T, Nettleton D, Baum TJ, Mitchum MG.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To gain new insights into the mechanism of soybean (Glycine max) <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines),</span> we compared gene expression profiles of developing syncytia in soybean near-isogenic lines differing at <span class="gene">Rhg1</span> (for <span class="trait">resistance</span> to <span class="trait">Heterodera</span> <span class="trait">glycines),</span> a major quantitative trait locus for <span class="trait">resistance,</span> by coupling laser capture microdissection with microarray analysis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A nematode demographics assay in transgenic roots reveals no significant impacts of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-Kinase on soybean cyst nematode <span class="trait">resistance</span></h2>
			<h3>PMID:20529370</h3>
				

				
					<h3>PMCID:3095272</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Melito </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> melito@wisc.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Heuberger </surname>  <given-names> Adam L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> adam.heuberger@colostate.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Cook </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dcook3@wisc.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Diers </surname>  <given-names> Brian W </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> bdiers@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> MacGuidwin </surname>  <given-names> Ann E </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> aemacgui@wisc.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Bent </surname>  <given-names> Andrew F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> afbent@wisc.edu </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 6 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus is repeatedly observed as the quantitative trait locus with the greatest impact on SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						1 gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor-kinase (LRR-kinase) has been proposed to be the SCN <span class="trait">resistance</span> gene, but its function has not been confirmed.
					</p>
				
					<p>&nbsp;&nbsp;
						We report use of artificial microRNA (amiRNA) for gene silencing in soybean, refinements to transgenic root SCN <span class="trait">resistance</span> assays, and functional tests of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Introduction of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene (genomic promoter/coding region/terminator; Peking/PI 437654-derived SCN-resistant source), into <italic> rhg1 </italic> <sup> - </sup> SCN-susceptible plant lines carrying the resistant-source <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> locus, provided no significant increases in SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Use of amiRNA to reduce expression of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus of Fayette (PI 88788 source of <italic> <span class="gene">Rhg1</span> </italic> ) also did not detectably alter <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been shown to have the greatest impact on SCN development in several <span class="trait">resistance</span> sources including Peking, PI 88788, PI 437654, PI 209332 and PI 90763.
					</p>
				
					<p>&nbsp;&nbsp;
						A second QTL ( <italic> <span class="gene">Rhg4</span> </italic> ) has been identified as being necessary for full <span class="trait">resistance</span> to some SCN populations in Peking and in PI 437654, but not PI 88788 or PI 209332 [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> exhibits dominant gene action, and in Peking-derived material the relevant alleles of both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> are necessary to exhibit the full <span class="trait">resistance</span> phenotype.
					</p>
				
					<p>&nbsp;&nbsp;
						The Peking and PI 88788 <italic> <span class="gene">Rhg1</span> </italic> sources also exhibit distinct differential behaviors in their strength of <span class="trait">resistance</span> against particular SCN test populations, suggesting at least partially different mechanisms in the SCN <span class="trait">resistance</span> controlled by different <italic> <span class="gene">Rhg1</span> </italic> alleles [ <xref ref-type="bibr" rid="B28"> 28 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						net/ </ext-link> [ <xref ref-type="bibr" rid="B17"> 17 </xref> ] as the likely SCN <span class="trait">resistance</span> genes at both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ], see also [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The basis for these claims was the presence of these genes at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, their similarity to the known rice bacterial blight <span class="trait">resistance</span> gene <italic> Xa21 </italic> , and the presence of derived amino acid sequence differences between the alleles from resistant and susceptible plant genotypes.
					</p>
				
					<p>&nbsp;&nbsp;
						There are numerous reasons why identification of the <italic> <span class="gene">Rhg1</span> </italic> gene that confers SCN <span class="trait">resistance</span> is a high priority, including the extreme economic significance of SCN for <span class="trait">yield</span> loss, the major reliance on the <italic> <span class="gene">Rhg1</span> </italic> locus in commercial soybean breeding, the detection of SCN populations that overcome currently available <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> the potential to respond to these challenges with engineered improvements in <italic> <span class="gene">Rhg1</span> </italic> , and basic scientific interest in the nature of plant <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						We used these tools to investigate the impact on SCN <span class="trait">resistance</span> of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> genomic region (the candidate SCN <span class="trait">resistance</span> gene).
					</p>
				
					<p>&nbsp;&nbsp;
						Our experiments expressing the LRR-kinase from a resistant (Peking/PI 437654-source) <italic> <span class="gene">Rhg1</span> </italic> locus in susceptible test lines, with or without the resistant allele at <italic> <span class="gene">Rhg4</span> </italic> , and silencing the LRR-kinase (PI 88788 source) in resistant lines, provided no evidence for a contribution of this gene to SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographics assay for SCN <span class="trait">resistance</span> in roots transformed with a full-length <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-Kinase gene </bold>.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus of soybean has been shown in numerous studies to be the QTL with the greatest phenotypic impact on SCN <span class="trait">resistance,</span> and previous patents and other reports have indicated but not demonstrated that the <span class="gene">Glyma18g02680</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						1 gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor kinase (LRR-K) is the source of this SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						We isolated this LRR-K gene from the <italic> <span class="gene">Rhg1</span> </italic> locus using genomic DNA from the soybean cultivar Ina that we confirmed to exhibit SCN <span class="trait">resistance</span> in a standard whole plant/cyst formation assay immediately prior to its use in DNA isolation.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the nematode demographics assay described above, roots from soybean line E98076 transformed with the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K did not display increased <span class="trait">resistance</span> to SCN; nematode development progressed similarly to what was observed on E98076 susceptible control roots transformed with the empty pGWB1::erGFP7INT vector (Figure <xref ref-type="fig" rid="F1"> 1D </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus in Peking and PI 437654 also contributes to the full SCN <span class="trait">resistance</span> phenotype [ <xref ref-type="bibr" rid="B10"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In the combined results from two independent experiments, no significant change in the <span class="trait">resistance</span> to SCN was detected after transformation of these genotypes with the Ina (Peking)-derived <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene (Figure <xref ref-type="fig" rid="F1"> 1E </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We sought to use gene silencing as an independent means to test for contributions of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene toward SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the key comparison to test if silencing of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene altered <span class="trait">resistance</span> to SCN, no statistical difference in nematode development was observed between Fayette and Fayette carrying amiRNA <span class="gene">GmLRR-K</span> I or amiRNA <span class="gene">Gm-K</span> II (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographic assay for SCN <span class="trait">resistance</span> in roots transformed with amiRNA silencing the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-kinase gene </bold>.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to reporting amiRNA use for gene silencing in soybean, a vector to facilitate this approach, and the nematode demographics assay for tests of genes that may contribute to soybean SCN <span class="trait">resistance,</span> we also report negative data regarding impacts of the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene on SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Using complementation with a full-length gene under control of its native promoter, and gene knockdown using amiRNA technology, we were unable to detect a contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene from either Peking/PI 437654 or PI 88788, the most commonly used sources of SCN <span class="trait">resistance</span> in U.
					</p>
				
					<p>&nbsp;&nbsp;
						It is notable that more than ten years after the filing of patent applications claiming SCN <span class="trait">resistance</span> function for the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene, no groups have published evidence (beyond its presence as one of many candidate genes at the <italic> <span class="gene">Rhg1</span> </italic> locus) for a functional contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Khalid Meksem and Melissa Mitchum has completed tests for functional contributions of the <italic> <span class="gene">Rhg4</span> </italic> -locus LRR-K gene to SCN <span class="trait">resistance</span> and, similar to our work, has not detected a contribution for this <italic> <span class="gene">Rhg4</span> </italic> candidate gene (K.
					</p>
				
					<p>&nbsp;&nbsp;
						, submitted)[ <xref ref-type="bibr" rid="B63"> 63 </xref> ], in which fine-structure genetic mapping of the <italic> <span class="gene">Rhg1</span> </italic> locus has revealed chromosomal recombination events that genetically separate <italic> rhg1-b </italic> (PI 88788-source) SCN <span class="trait">resistance</span> function from the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Hence evidence is accumulating that the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K genes seem unlikely to be the primary source of SCN <span class="trait">resistance</span> polymorphism.
					</p>
				
					<p>&nbsp;&nbsp;
						Hence we are cautious with our conclusion, but we can state that when SCN-susceptible sources were complemented with an <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K allele from an SCN-resistant source, or when expression of the gene in resistant plants was reduced by amiRNA-mediated silencing, no significant impacts on SCN <span class="trait">resistance</span> were detected.
					</p>
				
					<p>&nbsp;&nbsp;
						Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance,</span> suggesting that one or more other genes at the <italic> <span class="gene">Rhg1</span> </italic> locus may be the primary determinant of differences in SCN <span class="trait">resistance</span> among different soybean types.
					</p>
				
					<p>&nbsp;&nbsp;
						1 putative LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus was used to design PCR primers for isolation of the <italic> <span class="gene">Rhg1</span> </italic> locus receptor-like kinase-encoding allele from the resistant source Fayette, which carries the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele <italic> rhg1-b </italic> from PI 88788.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus is repeatedly observed as the quantitative trait locus with the greatest impact on SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						1 gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor-kinase (LRR-kinase) has been proposed to be the SCN <span class="trait">resistance</span> gene, but its function has not been confirmed
					</p>
				
					<p>&nbsp;&nbsp;
						We report use of artificial microRNA (amiRNA) for gene silencing in soybean, refinements to transgenic root SCN <span class="trait">resistance</span> assays, and functional tests of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene
					</p>
				
					<p>&nbsp;&nbsp;
						Introduction of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene (genomic promoter/coding region/terminator; Peking/PI 437654-derived SCN-resistant source), into <italic> rhg1 </italic> <sup> - </sup> SCN-susceptible plant lines carrying the resistant-source <italic> <span class="gene">Rhg4</span> </italic> <sup> <italic> + </italic> </sup> locus, provided no significant increases in SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Use of amiRNA to reduce expression of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus of Fayette (PI 88788 source of <italic> <span class="gene">Rhg1</span> </italic> ) also did not detectably alter <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus has been shown to have the greatest impact on SCN development in several <span class="trait">resistance</span> sources including Peking, PI 88788, PI 437654, PI 209332 and PI 90763
					</p>
				
					<p>&nbsp;&nbsp;
						A second QTL ( <italic> <span class="gene">Rhg4</span> </italic> ) has been identified as being necessary for full <span class="trait">resistance</span> to some SCN populations in Peking and in PI 437654, but not PI 88788 or PI 209332 [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg4</span> </italic> exhibits dominant gene action, and in Peking-derived material the relevant alleles of both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> are necessary to exhibit the full <span class="trait">resistance</span> phenotype
					</p>
				
					<p>&nbsp;&nbsp;
						The Peking and PI 88788 <italic> <span class="gene">Rhg1</span> </italic> sources also exhibit distinct differential behaviors in their strength of <span class="trait">resistance</span> against particular SCN test populations, suggesting at least partially different mechanisms in the SCN <span class="trait">resistance</span> controlled by different <italic> <span class="gene">Rhg1</span> </italic> alleles [ <xref ref-type="bibr" rid="B28"> 28 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						net/ </ext-link> [ <xref ref-type="bibr" rid="B17"> 17 </xref> ] as the likely SCN <span class="trait">resistance</span> genes at both <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ], see also [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The basis for these claims was the presence of these genes at the <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, their similarity to the known rice bacterial blight <span class="trait">resistance</span> gene <italic> Xa21 </italic> , and the presence of derived amino acid sequence differences between the alleles from resistant and susceptible plant genotypes
					</p>
				
					<p>&nbsp;&nbsp;
						There are numerous reasons why identification of the <italic> <span class="gene">Rhg1</span> </italic> gene that confers SCN <span class="trait">resistance</span> is a high priority, including the extreme economic significance of SCN for <span class="trait">yield</span> loss, the major reliance on the <italic> <span class="gene">Rhg1</span> </italic> locus in commercial soybean breeding, the detection of SCN populations that overcome currently available <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance,</span> the potential to respond to these challenges with engineered improvements in <italic> <span class="gene">Rhg1</span> </italic> , and basic scientific interest in the nature of plant <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						We used these tools to investigate the impact on SCN <span class="trait">resistance</span> of the LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> genomic region (the candidate SCN <span class="trait">resistance</span> gene)
					</p>
				
					<p>&nbsp;&nbsp;
						Our experiments expressing the LRR-kinase from a resistant (Peking/PI 437654-source) <italic> <span class="gene">Rhg1</span> </italic> locus in susceptible test lines, with or without the resistant allele at <italic> <span class="gene">Rhg4</span> </italic> , and silencing the LRR-kinase (PI 88788 source) in resistant lines, provided no evidence for a contribution of this gene to SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographics assay for SCN <span class="trait">resistance</span> in roots transformed with a full-length <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-Kinase gene </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus of soybean has been shown in numerous studies to be the QTL with the greatest phenotypic impact on SCN <span class="trait">resistance,</span> and previous patents and other reports have indicated but not demonstrated that the <span class="gene">Glyma18g02680</span>
					</p>
				
					<p>&nbsp;&nbsp;
						1 gene at the <italic> <span class="gene">Rhg1</span> </italic> locus that encodes an apparent leucine-rich repeat transmembrane receptor kinase (LRR-K) is the source of this SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						We isolated this LRR-K gene from the <italic> <span class="gene">Rhg1</span> </italic> locus using genomic DNA from the soybean cultivar Ina that we confirmed to exhibit SCN <span class="trait">resistance</span> in a standard whole plant/cyst formation assay immediately prior to its use in DNA isolation
					</p>
				
					<p>&nbsp;&nbsp;
						Using the nematode demographics assay described above, roots from soybean line E98076 transformed with the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K did not display increased <span class="trait">resistance</span> to SCN; nematode development progressed similarly to what was observed on E98076 susceptible control roots transformed with the empty pGWB1::erGFP7INT vector (Figure <xref ref-type="fig" rid="F1"> 1D </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus in Peking and PI 437654 also contributes to the full SCN <span class="trait">resistance</span> phenotype [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In the combined results from two independent experiments, no significant change in the <span class="trait">resistance</span> to SCN was detected after transformation of these genotypes with the Ina (Peking)-derived <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K gene (Figure <xref ref-type="fig" rid="F1"> 1E </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						We sought to use gene silencing as an independent means to test for contributions of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene toward SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In the key comparison to test if silencing of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene altered <span class="trait">resistance</span> to SCN, no statistical difference in nematode development was observed between Fayette and Fayette carrying amiRNA <span class="gene">GmLRR-K</span> I or amiRNA <span class="gene">Gm-K</span> II (Figure <xref ref-type="fig" rid="F6"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Nematode demographic assay for SCN <span class="trait">resistance</span> in roots transformed with amiRNA silencing the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-kinase gene </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to reporting amiRNA use for gene silencing in soybean, a vector to facilitate this approach, and the nematode demographics assay for tests of genes that may contribute to soybean SCN <span class="trait">resistance,</span> we also report negative data regarding impacts of the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene on SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Using complementation with a full-length gene under control of its native promoter, and gene knockdown using amiRNA technology, we were unable to detect a contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene from either Peking/PI 437654 or PI 88788, the most commonly used sources of SCN <span class="trait">resistance</span> in U
					</p>
				
					<p>&nbsp;&nbsp;
						It is notable that more than ten years after the filing of patent applications claiming SCN <span class="trait">resistance</span> function for the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene, no groups have published evidence (beyond its presence as one of many candidate genes at the <italic> <span class="gene">Rhg1</span> </italic> locus) for a functional contribution to SCN <span class="trait">resistance</span> by the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene
					</p>
				
					<p>&nbsp;&nbsp;
						Khalid Meksem and Melissa Mitchum has completed tests for functional contributions of the <italic> <span class="gene">Rhg4</span> </italic> -locus LRR-K gene to SCN <span class="trait">resistance</span> and, similar to our work, has not detected a contribution for this <italic> <span class="gene">Rhg4</span> </italic> candidate gene (K
					</p>
				
					<p>&nbsp;&nbsp;
						, submitted)[ <xref ref-type="bibr" rid="B63"> 63 </xref> ], in which fine-structure genetic mapping of the <italic> <span class="gene">Rhg1</span> </italic> locus has revealed chromosomal recombination events that genetically separate <italic> rhg1-b </italic> (PI 88788-source) SCN <span class="trait">resistance</span> function from the <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K gene
					</p>
				
					<p>&nbsp;&nbsp;
						Hence evidence is accumulating that the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-K genes seem unlikely to be the primary source of SCN <span class="trait">resistance</span> polymorphism
					</p>
				
					<p>&nbsp;&nbsp;
						Hence we are cautious with our conclusion, but we can state that when SCN-susceptible sources were complemented with an <italic> <span class="gene">Rhg1</span> </italic> -locus LRR-K allele from an SCN-resistant source, or when expression of the gene in resistant plants was reduced by amiRNA-mediated silencing, no significant impacts on SCN <span class="trait">resistance</span> were detected
					</p>
				
					<p>&nbsp;&nbsp;
						Studies in which expression of the <italic> <span class="gene">Rhg1</span> </italic> locus LRR-kinase gene from different <span class="trait">resistance</span> sources was either reduced or complemented did not reveal significant impacts on SCN <span class="trait">resistance,</span> suggesting that one or more other genes at the <italic> <span class="gene">Rhg1</span> </italic> locus may be the primary determinant of differences in SCN <span class="trait">resistance</span> among different soybean types
					</p>
				
					<p>&nbsp;&nbsp;
						1 putative LRR-kinase gene from the <italic> <span class="gene">Rhg1</span> </italic> locus was used to design PCR primers for isolation of the <italic> <span class="gene">Rhg1</span> </italic> locus receptor-like kinase-encoding allele from the resistant source Fayette, which carries the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele <italic> rhg1-b </italic> from PI 88788
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Separate loci underlie <span class="trait">resistance</span> to root infection and leaf scorch during soybean sudden death syndrome.</h2>
			<h3>PMID:18324383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kazi S(1), Shultz J, Afzal J, Johnson J, Njiti VN, Lightfoot DA.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						One contributed <span class="trait">resistance</span> to leaf scorch on linkage group (LG) <span class="gene">C2</span> (Satt277; P = 0.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:'Candidatus Paenicardinium endonii', an endosymbiont of the plant-parasitic nematode <span class="trait">Heterodera</span> <span class="trait">glycines</span> (Nemata: Tylenchida), affiliated to the phylum Bacteroidetes.</h2>
			<h3>PMID:16825653  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Noel GR(1), Atibalentja N.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> endosymbiont among bacteria by studying the 16S rRNA and <span class="gene">gyrB</span> gene sequences.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">Rhg1</span> alleles from soybean PI 437654 and PI 88788 respond differentially to isolates of <span class="trait">Heterodera</span> <span class="trait">glycines</span> in the greenhouse.</h2>
			<h3>PMID:15883792  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brucker E(1), Carlson S, Wright E, Niblack T, Diers B.
				<br/> 
				年份：2005 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a soybean ethylene-responsive element-binding protein gene whose mRNA expression changes during soybean cyst nematode infection.</h2>
			<h3>PMID:12059106  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mazarei M(1), Puthoff DP, Hart JK, Rodermel SR, Baum TJ.
				<br/> 
				年份：2002 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmEREBP1</span> mRNA abundance was analyzed by RNA blot hybridizations in soybean roots and shoots of cultivars Corsoy 79 and Hartwig, which are susceptible and resistant, respectively, to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines)</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physical mapping of resistant and susceptible soybean genomes near the soybean cyst nematode <span class="trait">resistance</span> gene Rhg4.</h2>
			<h3>PMID:11768209  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lewers KS(1), Nilmalgoda SD, Warner AL, Knap HT, Matthews BF.
				<br/> 
				年份：2001 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The rhg1 allele on linkage group (LG) G and the <span class="gene">Rhg4</span> allele on LG A2 are important in conditioning <span class="trait">resistance</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conversion of AFLP bands into high-throughput DNA markers.</h2>
			<h3>PMID:11361330  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Meksem K(1), Ruben E, Hyten D, Triwitayakorn K, Lightfoot DA.
				<br/> 
				年份：2001 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The markers were all linked to one of two loci, rhg1 on linkage group G and <span class="gene">Rhg4</span> on linkage group A2, that confer <span class="trait">resistance</span> to the soybean cyst nematode <span class="trait">(Heterodera</span> <span class="trait">glycines</span> I.
					</p>
				
					<p>&nbsp;&nbsp;
						The polymorphic sequences were used to design a non-electrophoretic, fluorometric assay (based on the TaqMan technology) and/or develop electrophoretic <span class="gene">STS</span> markers for high-throughput genotype determination during marker-assisted breeding for <span class="trait">resistance</span> to cyst nematode.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional screening yields a new beta-1,4-endoglucanase gene from <span class="trait">Heterodera</span> <span class="trait">glycines</span> that may be the product of recent gene duplication.</h2>
			<h3>PMID:11194873  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Y(1), Smant G, Davis E.
				<br/> 
				年份：2001 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> genomic clone contained <span class="gene">HG-eng-2</span> and HG-eng-3 full-length genes, separated by a distance of approximately 8 kb, and a second genomic clone contained two copies of HG-eng-2, separated by approximately 6.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SNP identification and marker assay development for high-throughput selection of soybean cyst nematode <span class="trait">resistance</span></h2>
			<h3>PMID:25903750</h3>
				

				
					<h3>PMCID:4407462</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Zi </given-names>  </name>  <address>  <email> shizi@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <address>  <email> smliu@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Noe </surname>  <given-names> James </given-names>  </name>  <address>  <email> jpnoe@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arelli </surname>  <given-names> Prakash </given-names>  </name>  <address>  <email> Prakash.Arelli@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <address>  <email> meksemk@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <address>  <email> zli@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Center for Applied Genetic Technologies &#x00026; Dep. of Crop and Soil Sciences, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff2">  <label/> Department of Plant, Soil and Agriculture Systems, Southern Illinois University, Carbondale, IL 62901 USA  </aff>  <aff id="Aff3">  <label/> Department of Plant Pathology, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff4">  <label/> USDA-ARS-SEA, Jackson, TN 38301 USA  </aff>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two <span class="trait">resistance</span> loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> primarily contribute <span class="trait">resistance</span> to SCN race 3 in soybean
					</p>
				
					<p>&nbsp;&nbsp;
						Peking and PI 88788 are the two major sources of SCN <span class="trait">resistance</span> with Peking requiring both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles and PI 88788 only the <italic> <span class="gene">Rhg1</span> </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the genomic DNA sequences of 27 soybean lines with known SCN phenotypes, we have developed Kompetitive Allele Specific PCR (KASP) assays for two Single nucleotide polymorphisms (SNPs) from <span class="gene">Glyma08g11490</span> for the selection of the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele
					</p>
				
					<p>&nbsp;&nbsp;
						Using the same 27 soybean lines, we identified one SNP that can be used to select the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele and another SNP that can be employed to differentiate Peking and PI 88788-type <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Three functional SNP markers (two for <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> locus) were identified that could provide genotype information for the selection of SCN <span class="trait">resistance</span> and differentiate Peking from PI 88788 source for most germplasm lines
					</p>
				
					<p>&nbsp;&nbsp;
						Two <span class="trait">resistance</span> genes for SCN, <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">(Resistance</span> to <italic> H
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> </italic> ) on Chromosome (Chr) 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr08, have been identified in the soybean germplasm, and have been shown to be the key sources of <span class="trait">resistance</span> in soybean cultivars [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus, which is crucial for broad spectrum <span class="trait">resistance</span> to SCN, has been identified in various <span class="trait">resistance</span> sources, such as PI 88788 and Peking [ <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						While PI 88788-type <span class="trait">resistance</span> only requires the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, Peking-type <span class="trait">resistance</span> needs both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles for a resistant phenotype [ <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Map-based cloning showed that a single gene at <italic> <span class="gene">Rhg4</span> </italic> locus encoding a serine hydroxymethyltransferase (SHMT, <span class="gene">Glyma08g11490)</span> is responsible for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						These polymorphisms change the property of the enzyme and may correlate with <italic> <span class="gene">Rhg4</span> </italic> -dependent <span class="trait">resistance</span> and susceptibility [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Two SNPs from <span class="gene">Glyma08g11490</span> are associated with the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele, and they are completely linked (data not shown)
					</p>
				
					<p>&nbsp;&nbsp;
						Three SNPs in <span class="gene">Glyma18g02580</span> and two SNP/InDels in the last exon of <span class="gene">Glyma18g02590</span> are associated with the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, and all five markers are completely linked
					</p>
				
					<p>&nbsp;&nbsp;
						Five SNPs in the 6th exon of <span class="gene">Glyma18g02590</span> could distinguish the sources of Peking-type and PI88788-type <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						However, three SNPs identified in the 5&#x02019;-UTR of <span class="gene">Glyma18g02610</span> showed no correlation with SCN <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, we have examined the extended pedigrees (data not shown) of these lines with <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> allele (GSM381&#x02009;=&#x02009;G) to identify the sources of <span class="trait">resistance</span> (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1)
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP genotypes are in complete agreement with the <span class="trait">resistance</span> sources based on extended pedigrees: Peking-source with a G allele at GSM383 and PI 88788-source with a C allele, indicating that GSM383 is a functional SNP to differentiate Peking and PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele
					</p>
				
					<p>&nbsp;&nbsp;
						The line G00-3213 carries a Peking-source <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles (GSM381&#x02009;=&#x02009;G, GSM383&#x02009;=&#x02009;G, and GSM191&#x02009;=&#x02009;G) and is resistant to SCN race 3, while LG04-6000 is a susceptible line with T, C and C alleles for GSM381, GSM383, and GSM191, respectively
					</p>
				
					<p>&nbsp;&nbsp;
						Resulting phenotypes demonstrated that combination of the <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> (GSM381&#x02009;=&#x02009;G and GSM383&#x02009;=&#x02009;G) and <span class="trait">resistance</span> <italic> <span class="gene">Rhg4</span> </italic> alleles (GSM191&#x02009;=&#x02009;G) brought a resistant phenotype with only 0
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus on Chr18 is necessary for <span class="trait">resistance</span> to all SCN races [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Many other cultivar-race specific QTLs have been demonstrated in <span class="trait">resistance</span> sources, such as <italic> <span class="gene">Rhg4</span> </italic> for race 1 and 3, <italic> Rhg3 </italic> and <italic> Rhg5 </italic> for race 2 and <italic> Rhg2 </italic> for race 5 and 14 [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, several SNPs have been found at <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], but no specific functional SNPs as well as detection assays have been reported to associate with the <span class="trait">resistance</span> allele, nor the SNPs to differentiate sources of <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						[ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], in this study, we have developed SNP marker assays, GSM381, GSM383, and GSM191 at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> locus for the selection of <span class="trait">resistance</span> alleles, (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The SNPs reside in the coding sequences of <span class="trait">resistance</span> genes <span class="gene">(Glyma18g02590</span> and <span class="gene">Glyma08g11490)</span> and also introduce changes in protein sequences, thus they can be considered as functional markers for SCN race 3 <span class="trait">resistance</span> and applied in high throughput marker-assisted selection with high accuracy
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, to identify the sources of <span class="trait">resistance,</span> we demonstrated that one SNP marker at <italic> <span class="gene">Rhg1</span> </italic> locus, GSM383 was able to distinguish Peking from PI 88788 loci for most soybean germplasm (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1)
					</p>
				
					<p>&nbsp;&nbsp;
						A number of factors may result in the discrepancy between genotyping and phenotyping results, including 1) phenotyping/ or seed source errors: some of the lines have been tested for only one year, and one-time phenotyping of SCN <span class="trait">resistance</span> may not be definitive; 2) sources other than PI 88788 and Peking may be providing <span class="trait">resistance;</span> and 3) the complexity of the <italic> <span class="gene">Rhg1</span> </italic> region is not fully understood
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the functional SNP markers that we identified can be used not only to select the <span class="trait">resistance</span> allele, but also to differentiate <span class="trait">resistance</span> sources, which were also confirmed by analysis of the <italic> <span class="gene">Rhg1</span> </italic> region with Soy50k SNP Infinium chip data using 38 soybean lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A)
					</p>
				
					<p>&nbsp;&nbsp;
						A recent study of the <italic> <span class="gene">Rhg1</span> </italic> locus revealed that copy number variation of the 31&#x000a0;kb DNA fragment, DNA sequence variation, and differentially methylated patterns were correlated with the <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Peking and PI 88788 are the two major sources of SCN <span class="trait">resistance</span> with Peking requiring both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles and PI 88788 only the <italic> <span class="gene">Rhg1</span> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the genomic DNA sequences of 27 soybean lines with known SCN phenotypes, we have developed Kompetitive Allele Specific PCR (KASP) assays for two Single nucleotide polymorphisms (SNPs) from <span class="gene">Glyma08g11490</span> for the selection of the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the same 27 soybean lines, we identified one SNP that can be used to select the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele and another SNP that can be employed to differentiate Peking and PI 88788-type <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Three functional SNP markers (two for <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> locus) were identified that could provide genotype information for the selection of SCN <span class="trait">resistance</span> and differentiate Peking from PI 88788 source for most germplasm lines.
					</p>
				
					<p>&nbsp;&nbsp;
						Two <span class="trait">resistance</span> genes for SCN, <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">(Resistance</span> to <italic> H.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> </italic> ) on Chromosome (Chr) 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr08, have been identified in the soybean germplasm, and have been shown to be the key sources of <span class="trait">resistance</span> in soybean cultivars [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus, which is crucial for broad spectrum <span class="trait">resistance</span> to SCN, has been identified in various <span class="trait">resistance</span> sources, such as PI 88788 and Peking [ <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						While PI 88788-type <span class="trait">resistance</span> only requires the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, Peking-type <span class="trait">resistance</span> needs both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles for a resistant phenotype [ <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Map-based cloning showed that a single gene at <italic> <span class="gene">Rhg4</span> </italic> locus encoding a serine hydroxymethyltransferase (SHMT, <span class="gene">Glyma08g11490)</span> is responsible for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						These polymorphisms change the property of the enzyme and may correlate with <italic> <span class="gene">Rhg4</span> </italic> -dependent <span class="trait">resistance</span> and susceptibility [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Two SNPs from <span class="gene">Glyma08g11490</span> are associated with the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele, and they are completely linked (data not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						Three SNPs in <span class="gene">Glyma18g02580</span> and two SNP/InDels in the last exon of <span class="gene">Glyma18g02590</span> are associated with the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, and all five markers are completely linked.
					</p>
				
					<p>&nbsp;&nbsp;
						Five SNPs in the 6th exon of <span class="gene">Glyma18g02590</span> could distinguish the sources of Peking-type and PI88788-type <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						However, three SNPs identified in the 5&#x02019;-UTR of <span class="gene">Glyma18g02610</span> showed no correlation with SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, we have examined the extended pedigrees (data not shown) of these lines with <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> allele (GSM381&#x02009;=&#x02009;G) to identify the sources of <span class="trait">resistance</span> (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP genotypes are in complete agreement with the <span class="trait">resistance</span> sources based on extended pedigrees: Peking-source with a G allele at GSM383 and PI 88788-source with a C allele, indicating that GSM383 is a functional SNP to differentiate Peking and PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						The line G00-3213 carries a Peking-source <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles (GSM381&#x02009;=&#x02009;G, GSM383&#x02009;=&#x02009;G, and GSM191&#x02009;=&#x02009;G) and is resistant to SCN race 3, while LG04-6000 is a susceptible line with T, C and C alleles for GSM381, GSM383, and GSM191, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Resulting phenotypes demonstrated that combination of the <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> (GSM381&#x02009;=&#x02009;G and GSM383&#x02009;=&#x02009;G) and <span class="trait">resistance</span> <italic> <span class="gene">Rhg4</span> </italic> alleles (GSM191&#x02009;=&#x02009;G) brought a resistant phenotype with only 0.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg1</span> </italic> locus on Chr18 is necessary for <span class="trait">resistance</span> to all SCN races [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Many other cultivar-race specific QTLs have been demonstrated in <span class="trait">resistance</span> sources, such as <italic> <span class="gene">Rhg4</span> </italic> for race 1 and 3, <italic> Rhg3 </italic> and <italic> Rhg5 </italic> for race 2 and <italic> Rhg2 </italic> for race 5 and 14 [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, several SNPs have been found at <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], but no specific functional SNPs as well as detection assays have been reported to associate with the <span class="trait">resistance</span> allele, nor the SNPs to differentiate sources of <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						[ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], in this study, we have developed SNP marker assays, GSM381, GSM383, and GSM191 at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> locus for the selection of <span class="trait">resistance</span> alleles, (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The SNPs reside in the coding sequences of <span class="trait">resistance</span> genes <span class="gene">(Glyma18g02590</span> and <span class="gene">Glyma08g11490)</span> and also introduce changes in protein sequences, thus they can be considered as functional markers for SCN race 3 <span class="trait">resistance</span> and applied in high throughput marker-assisted selection with high accuracy.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, to identify the sources of <span class="trait">resistance,</span> we demonstrated that one SNP marker at <italic> <span class="gene">Rhg1</span> </italic> locus, GSM383 was able to distinguish Peking from PI 88788 loci for most soybean germplasm (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						A number of factors may result in the discrepancy between genotyping and phenotyping results, including 1) phenotyping/ or seed source errors: some of the lines have been tested for only one year, and one-time phenotyping of SCN <span class="trait">resistance</span> may not be definitive; 2) sources other than PI 88788 and Peking may be providing <span class="trait">resistance;</span> and 3) the complexity of the <italic> <span class="gene">Rhg1</span> </italic> region is not fully understood.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the functional SNP markers that we identified can be used not only to select the <span class="trait">resistance</span> allele, but also to differentiate <span class="trait">resistance</span> sources, which were also confirmed by analysis of the <italic> <span class="gene">Rhg1</span> </italic> region with Soy50k SNP Infinium chip data using 38 soybean lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						A recent study of the <italic> <span class="gene">Rhg1</span> </italic> locus revealed that copy number variation of the 31&#x000a0;kb DNA fragment, DNA sequence variation, and differentially methylated patterns were correlated with the <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping</h2>
			<h3>PMID:26268218</h3>
				

				
					<h3>PMCID:4542112</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <address>  <email> wy11544@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> twlneau@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhiwu </given-names>  </name>  <address>  <email> zhiwu.zhang@wsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Key Laboratory of Soybean Biology in Chinese Ministry of Education (key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry), Northeast Agricultural University, Harbin, 150030 China  </aff>  <aff id="Aff2">  <label/> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI) Chinese Academy of Agricultural Sciences, 100081 Beijing, China  </aff>  <aff id="Aff3">  <label/> Bioinformatics Division, Biomarker Technologies Corporation, 101300 Beijing, China  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many QTL conferring <span class="trait">resistance</span> to SCN in soybean have been mapped on almost all chromosomes (except of <span class="gene">Gm02;</span> LG D1b) based on SSR markers [ <xref ref-type="bibr" rid="CR12"> 12 </xref> &#x02013; <xref ref-type="bibr" rid="CR19"> 19 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Of them, two major QTL, <italic> <span class="gene">Rhg1</span> </italic> on <span class="gene">Gm18</span> and <italic> <span class="gene">Rhg4</span> </italic> on <span class="gene">Gm08</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], were identified in many different <span class="trait">resistance</span> sources
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] considered the SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was conferred by three genes, and the copy number variation of these genes caused the phenotypic differences between susceptible and resistant varieties
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg4</span> </italic> in Forrest was controlled by a serine hydroxymethyl transferase gene, that acted in cellular one-carbon metabolism [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The known SCN resistant loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were identified by the present study for soybean <span class="trait">resistance</span> to Hg Type 0
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus was also found to contribute <span class="trait">resistance</span> to Hg Type 1
					</p>
				
					<p>&nbsp;&nbsp;
						The rs33704130 located on <span class="gene">Gm16</span> was significantly associated with the <span class="trait">resistance</span> for both Hg Types
					</p>
				
					<p>&nbsp;&nbsp;
						Of these gene families, there were five clusters of genes with disease <span class="trait">resistance</span> protein and/or LRR protein domains that were found in both side of rs33704130 on <span class="gene">Gm16</span> that was associated with the <span class="trait">resistance</span> to both Hg Type 0 and Hg Type 1
					</p>
				
					<p>&nbsp;&nbsp;
						The Hg Type 0 <span class="trait">resistance</span> loci (rs46050545 on <span class="gene">Gm04,</span> rs33843704 on <span class="gene">Gm05,</span> rs33488383 on <span class="gene">Gm11</span> and rs43617172 on <span class="gene">Gm14),</span> as well as Hg Type 1
					</p>
				
					<p>&nbsp;&nbsp;
						7 <span class="trait">resistance</span> locus (rs172388849 on <span class="gene">Gm17)</span> were tightly linked with the genes of RING/U-box superfamily protein or RING-H2 finger C1A gene
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 12 QTN (nucleotides) associated with QTL underlying <span class="trait">resistance</span> to SCN Hg Type 0, one QTN (rs8050006) was identified on chromosome 8 (Gm 08, LG A2), where a known and major QTL region associated with <italic> <span class="gene">Rhg4</span> </italic> locus had been commonly reported before [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> &#x02013; <xref ref-type="bibr" rid="CR51"> 51 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus was confirmed as one of the major QTL controlling soybean <span class="trait">resistance</span> against SCN Hg Type 0 [ <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ] confirmed the effect of <italic> <span class="gene">Rhg4</span> </italic> on <span class="trait">resistance</span> to race 3 and 14, while <italic> <span class="gene">Rhg4</span> </italic> was less associated with <span class="trait">resistance</span> to races 1, 2, 4 and 5
					</p>
				
					<p>&nbsp;&nbsp;
						Here, the SNP rs1643660, associated with the <span class="trait">resistance</span> to SCN Hg Type 0 (race 3), was detected to fall within the <italic> rhg1 </italic> locus and was located in the 31 Kbp copy number variation (CNV) region of <italic> <span class="gene">Rhg1</span> </italic> locus
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, rs2085816 on Gm 20 (LG I), rs21804864 on <span class="gene">Gm02</span> (LG D1b) and rs30581306 on <span class="gene">Gm14</span> (LG B2) were the novel loci reported for SCN <span class="trait">resistance</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Of them, two major QTL, <italic> <span class="gene">Rhg1</span> </italic> on <span class="gene">Gm18</span> and <italic> <span class="gene">Rhg4</span> </italic> on <span class="gene">Gm08</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], were identified in many different <span class="trait">resistance</span> sources.
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] considered the SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was conferred by three genes, and the copy number variation of these genes caused the phenotypic differences between susceptible and resistant varieties.
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg4</span> </italic> in Forrest was controlled by a serine hydroxymethyl transferase gene, that acted in cellular one-carbon metabolism [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The known SCN resistant loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were identified by the present study for soybean <span class="trait">resistance</span> to Hg Type 0.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus was also found to contribute <span class="trait">resistance</span> to Hg Type 1.
					</p>
				
					<p>&nbsp;&nbsp;
						The rs33704130 located on <span class="gene">Gm16</span> was significantly associated with the <span class="trait">resistance</span> for both Hg Types.
					</p>
				
					<p>&nbsp;&nbsp;
						Of these gene families, there were five clusters of genes with disease <span class="trait">resistance</span> protein and/or LRR protein domains that were found in both side of rs33704130 on <span class="gene">Gm16</span> that was associated with the <span class="trait">resistance</span> to both Hg Type 0 and Hg Type 1.
					</p>
				
					<p>&nbsp;&nbsp;
						The Hg Type 0 <span class="trait">resistance</span> loci (rs46050545 on <span class="gene">Gm04,</span> rs33843704 on <span class="gene">Gm05,</span> rs33488383 on <span class="gene">Gm11</span> and rs43617172 on <span class="gene">Gm14),</span> as well as Hg Type 1.
					</p>
				
					<p>&nbsp;&nbsp;
						7 <span class="trait">resistance</span> locus (rs172388849 on <span class="gene">Gm17)</span> were tightly linked with the genes of RING/U-box superfamily protein or RING-H2 finger C1A gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 12 QTN (nucleotides) associated with QTL underlying <span class="trait">resistance</span> to SCN Hg Type 0, one QTN (rs8050006) was identified on chromosome 8 (Gm 08, LG A2), where a known and major QTL region associated with <italic> <span class="gene">Rhg4</span> </italic> locus had been commonly reported before [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> &#x02013; <xref ref-type="bibr" rid="CR51"> 51 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Rhg4</span> </italic> locus was confirmed as one of the major QTL controlling soybean <span class="trait">resistance</span> against SCN Hg Type 0 [ <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ] confirmed the effect of <italic> <span class="gene">Rhg4</span> </italic> on <span class="trait">resistance</span> to race 3 and 14, while <italic> <span class="gene">Rhg4</span> </italic> was less associated with <span class="trait">resistance</span> to races 1, 2, 4 and 5.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, the SNP rs1643660, associated with the <span class="trait">resistance</span> to SCN Hg Type 0 (race 3), was detected to fall within the <italic> rhg1 </italic> locus and was located in the 31 Kbp copy number variation (CNV) region of <italic> <span class="gene">Rhg1</span> </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, rs2085816 on Gm 20 (LG I), rs21804864 on <span class="gene">Gm02</span> (LG D1b) and rs30581306 on <span class="gene">Gm14</span> (LG B2) were the novel loci reported for SCN <span class="trait">resistance</span>.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Population-specific gene expression in the plant pathogenic nematode <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> exists prior to infection and during the onset of a resistant or susceptible reaction in the roots of the <italic> Glycine max </italic> genotype Peking</h2>
			<h3>PMID:19291306</h3>
				

				
					<h3>PMCID:2662880</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" corresp="yes" contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> vklink@biology.msstate.edu </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Hosseini </surname>  <given-names> Parsa </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> phosse1@towson.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Margaret.MacDonald@ARS.USDA.GOV </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Alkharouf </surname>  <given-names> Nadim W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> nalkharouf@gmail.com </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Ben.Matthews@ARS.USDA.GOV </email>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 3 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						lycopersicum </italic> <span class="gene">HSP90</span> resulted in attenuation of <span class="trait">resistance</span> of the root knot nematode ( <italic> Meloidogyne spp </italic>
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Microarray Detection Call Methodology as a Means to Identify and Compare Transcripts Expressed within Syncytial Cells from Soybean ( <italic> Glycine max </italic> ) Roots Undergoing Resistant and Susceptible Reactions to the Soybean Cyst Nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> )</h2>
			<h3>PMID:20508855</h3>
				

				
					<h3>PMCID:2875038</h3>
					<h4>期刊:Journal of Biomedicine and Biotechnology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Klink </surname>  <given-names> Vincent P. </given-names>  </name>  <xref ref-type="aff" rid="I1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Overall </surname>  <given-names> Christopher C. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2, 3, 4 </sup>  </xref>  <xref ref-type="aff" rid="I3"/>  <xref ref-type="aff" rid="I4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alkharouf </surname>  <given-names> Nadim W. </given-names>  </name>  <xref ref-type="aff" rid="I5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F. </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 2 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Several recessive <span class="trait">resistance</span> loci ( <italic> rhg1 </italic> , <italic> rhg2, </italic> and <italic> rhg3 </italic> ) [ <xref ref-type="bibr" rid="B14"> 16 </xref> ] and dominant <span class="trait">resistance</span> loci ( <italic> <span class="gene">Rhg4</span> </italic> ) [ <xref ref-type="bibr" rid="B67"> 17 </xref> ] and ( <italic> Rhg5 </italic> ) [ <xref ref-type="bibr" rid="B81"> 18 </xref> ] have been identified (reviewed in [ <xref ref-type="bibr" rid="B18"> 15 </xref> ])
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:QTL underlying <span class="trait">resistance</span> to two HG types of <span class="trait">Heterodera</span> <span class="trait">glycines</span> found in soybean cultivar 'L-10'</h2>
			<h3>PMID:21569389</h3>
				

				
					<h3>PMCID:3224386</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Chang </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weich0@foxmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Dong </surname>  <given-names> Limin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> donglimin221355@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wang </surname>  <given-names> Zizhen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> milandewang@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Hu </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> nmghuhaibo@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hyp234286@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weiliteng2006@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhang </surname>  <given-names> Hongxia </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hxzhang@sippe.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Guo </surname>  <given-names> Maozu </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> maozuguo@hit.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wenbinli@neau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 5 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The fourth <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> had been confirmed as being necessary for full <span class="trait">resistance</span> to some races of SCN in Peking but not PI88788
					</p>
				
					<p>&nbsp;&nbsp;
						Both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were necessary for the full <span class="trait">resistance</span> to SCN race 3 and 14
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> was inferred to be linked with pBLT24, pBLT56 and <italic> I </italic> gene for seed coat color on linkage group A2 (chromosome 8) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, another <span class="gene">RLK</span> in the <italic> <span class="gene">Rhg4</span> </italic> interval was shown not to underlie the <span class="trait">resistance</span> locus [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Future analyses would be focused to determine the role in the <span class="trait">resistance</span> mechanism underlain by <span class="gene">Glyma16g30760</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">Rhg4</span> locus was considered to be another main QTL for SCN <span class="trait">resistance,</span> tightly linked to <italic> I </italic> locus [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						This result indicated that the line 'L-10' had another locus for <span class="trait">resistance</span> to SCN race 3 independent of the <span class="gene">Rhg4</span> locus
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were necessary for the full <span class="trait">resistance</span> to SCN race 3 and 14.
					</p>
				
					<p>&nbsp;&nbsp;
						The SCN <span class="trait">resistance</span> gene <italic> <span class="gene">Rhg4</span> </italic> was inferred to be linked with pBLT24, pBLT56 and <italic> I </italic> gene for seed coat color on linkage group A2 (chromosome 8) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, another <span class="gene">RLK</span> in the <italic> <span class="gene">Rhg4</span> </italic> interval was shown not to underlie the <span class="trait">resistance</span> locus [ <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Future analyses would be focused to determine the role in the <span class="trait">resistance</span> mechanism underlain by <span class="gene">Glyma16g30760</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">Rhg4</span> locus was considered to be another main QTL for SCN <span class="trait">resistance,</span> tightly linked to <italic> I </italic> locus [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						This result indicated that the line 'L-10' had another locus for <span class="trait">resistance</span> to SCN race 3 independent of the <span class="gene">Rhg4</span> locus.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Engineered <span class="trait">resistance</span> and hypersusceptibility through functional metabolic studies of 100 genes in soybean to its major pathogen, the soybean cyst nematode</h2>
			<h3>PMID:23389673</h3>
				

				
					<h3>PMCID:3634990</h3>
					<h4>期刊:Planta</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F. </given-names>  </name>  <address>  <phone> 301-504-5730 </phone>  <fax> 301-504-5728 </fax>  <email> ben.matthews@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Beard </surname>  <given-names> Hunter </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kabir </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Youssef </surname>  <given-names> Reham M. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hosseini </surname>  <given-names> Parsa </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brewer </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Soybean Genomics and Improvement Laboratory, United States Department of Agriculture, Agricultural Research Service, 10300 Baltimore Ave, Bldg 006, Beltsville, MD 20705 USA  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, several genes ( <italic> rhg1, rhg2, rhg3 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ) in soybean cv Peking confer <span class="trait">resistance</span> to SCN race 1 (Caldwell et al
					</p>
				
					<p>&nbsp;&nbsp;
						In PI 437654, <span class="trait">resistance</span> to SCN race 3 was mapped to three loci, including the <italic> <span class="gene">Rhg4</span> </italic> locus on linkage group A within 1&#x000a0;cM of the <italic> I </italic> locus (Webb et al
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of gene A35 decreased the female index 55&#x000a0;% of the control, and it is of interest, because it encodes a serine hydroxymethyltransferase located on chromosome eight <span class="gene">(Glyma08g11490)</span> of the soybean genome at the <italic> <span class="gene">Rhg4</span> </italic> locus (Matson and Williams <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) encoding a <span class="trait">resistance</span> gene for SCN (Cregan et al
					</p>
				
					<p>&nbsp;&nbsp;
						Matson and Williams ( <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) first identified the <italic> <span class="gene">Rhg4</span> </italic> gene as a dominant gene required for <span class="trait">resistance</span> to SCN race 3
					</p>
				
					<p>&nbsp;&nbsp;
						TILLING experiments demonstrated that <span class="trait">resistance</span> is not conferred by the gene encoding an <span class="gene">LRR-RLK</span> that resides in the region (Liu et al
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In PI 437654, <span class="trait">resistance</span> to SCN race 3 was mapped to three loci, including the <italic> <span class="gene">Rhg4</span> </italic> locus on linkage group A within 1&#x000a0;cM of the <italic> I </italic> locus (Webb et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of gene A35 decreased the female index 55&#x000a0;% of the control, and it is of interest, because it encodes a serine hydroxymethyltransferase located on chromosome eight <span class="gene">(Glyma08g11490)</span> of the soybean genome at the <italic> <span class="gene">Rhg4</span> </italic> locus (Matson and Williams <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) encoding a <span class="trait">resistance</span> gene for SCN (Cregan et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Matson and Williams ( <xref ref-type="bibr" rid="CR94"> 1965 </xref> ) first identified the <italic> <span class="gene">Rhg4</span> </italic> gene as a dominant gene required for <span class="trait">resistance</span> to SCN race 3.
					</p>
				
					<p>&nbsp;&nbsp;
						TILLING experiments demonstrated that <span class="trait">resistance</span> is not conferred by the gene encoding an <span class="gene">LRR-RLK</span> that resides in the region (Liu et al.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Arabidopsis </italic> genes, <italic> AtNPR1, AtTGA2 </italic> and <italic> AtPR-5 </italic> , confer partial <span class="trait">resistance</span> to soybean cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> ) when overexpressed in transgenic soybean roots</h2>
			<h3>PMID:24739302</h3>
				

				
					<h3>PMCID:4021311</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ben.matthews@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Beard </surname>  <given-names> Hunter </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hunter.beard@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Brewer </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> eric.brewer@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kabir </surname>  <given-names> Sara </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sara_kabir1@hotmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> margaret.macdonald@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Youssef </surname>  <given-names> Reham M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> reham.youssef@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, in soybean cv Peking, several genes ( <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> , <italic> rhg3 </italic> , and <italic> <span class="gene">Rhg4</span> </italic> ) have been reported that confer <span class="trait">resistance</span> to SCN race 1 [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> ], yet none of these genes confers complete <span class="trait">resistance</span> to all SCN populations
					</p>
				
					<p>&nbsp;&nbsp;
						NDR1 (NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1) is a positive regulator of SA that works independently of EDS1 and <span class="gene">PAD4</span> [ <xref ref-type="bibr" rid="B55"> 55 </xref> ] NDR1 is an integrin-like protein that can associate with RIN4, while RIN4 can associate with RPM1 and RPS2 [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, the identity of genes residing at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci have been reported [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ] which confer some <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						For example, in a cross between soybean cv Essex and Forrest, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> accounted for about 65% of the variation in <span class="trait">resistance</span> found in the resultant inbred population to SCN [ <xref ref-type="bibr" rid="B94"> 94 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> The apple <italic> MpNPR1 </italic> gene confers <span class="trait">resistance</span> to two fungal pathogens of apple, <italic> Venturia inaequalis </italic> and <italic> Gymnosporangium juniper-virginianae </italic> [ <xref ref-type="bibr" rid="B106"> 106 </xref> , <xref ref-type="bibr" rid="B107"> 107 </xref> ] complemented <italic> Arabidopsis npr1-1 </italic> mutants with soybean homologs <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> NPR1-2, </italic> and PR-1 was induced in the transformed plants after infection with <italic> P
					</p>
				
					<p>&nbsp;&nbsp;
						Apparently, this conservation is not enough for AtEDS1 to provide <span class="trait">resistance</span> to SCN as did <span class="gene">GmEDS1</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, several genes provided increases in <span class="trait">resistance</span> comparable to or better than that provided by <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , two naturally occurring <span class="trait">resistance</span> gene loci in soybean
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, overexpression of <italic> AtEDS1 </italic> did not increase <span class="trait">resistance,</span> although a <italic> <span class="gene">GmEDS1</span> </italic> ortholog was recently reported as providing <span class="trait">resistance</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						NDR1 (NON-RACE SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1) is a positive regulator of SA that works independently of EDS1 and <span class="gene">PAD4</span> [ <xref ref-type="bibr" rid="B55"> 55 </xref> ] NDR1 is an integrin-like protein that can associate with RIN4, while RIN4 can associate with RPM1 and RPS2 [ <xref ref-type="bibr" rid="B56"> 56 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, the identity of genes residing at the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci have been reported [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ] which confer some <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						For example, in a cross between soybean cv Essex and Forrest, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> accounted for about 65% of the variation in <span class="trait">resistance</span> found in the resultant inbred population to SCN [ <xref ref-type="bibr" rid="B94"> 94 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> The apple <italic> MpNPR1 </italic> gene confers <span class="trait">resistance</span> to two fungal pathogens of apple, <italic> Venturia inaequalis </italic> and <italic> Gymnosporangium juniper-virginianae </italic> [ <xref ref-type="bibr" rid="B106"> 106 </xref> , <xref ref-type="bibr" rid="B107"> 107 </xref> ] complemented <italic> Arabidopsis npr1-1 </italic> mutants with soybean homologs <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> NPR1-2, </italic> and PR-1 was induced in the transformed plants after infection with <italic> P.
					</p>
				
					<p>&nbsp;&nbsp;
						Apparently, this conservation is not enough for AtEDS1 to provide <span class="trait">resistance</span> to SCN as did <span class="gene">GmEDS1</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, several genes provided increases in <span class="trait">resistance</span> comparable to or better than that provided by <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , two naturally occurring <span class="trait">resistance</span> gene loci in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, overexpression of <italic> AtEDS1 </italic> did not increase <span class="trait">resistance,</span> although a <italic> <span class="gene">GmEDS1</span> </italic> ortholog was recently reported as providing <span class="trait">resistance</span>.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Whole-genome gene expression profiling revealed genes and pathways potentially involved in regulating interactions of soybean with cyst nematode ( <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> Ichinohe)</h2>
			<h3>PMID:25880563</h3>
				

				
					<h3>PMCID:4351908</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wan </surname>  <given-names> Jinrong </given-names>  </name>  <address>  <email> wanj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Vuong </surname>  <given-names> Tri </given-names>  </name>  <address>  <email> vuongt@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiao </surname>  <given-names> Yongqing </given-names>  </name>  <address>  <email> yongqing.jiao@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hongxin </given-names>  </name>  <address>  <email> jackson.zhang.828@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Current address: Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan, Hubei 430062 China  </aff>  <aff id="Aff3">  <label/> Department of Computer Sciences, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label/> Informatics Institute and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 3 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recently, the genes underlying two major QTL (quantitative trait loci), <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , responsible for <span class="trait">resistance</span> to SCN in the soybean lines, have been cloned [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Although PI 437654 was shown to contain two known QTL ( <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> ), a new QTL for <span class="trait">resistance</span> to SCN HG types 0, 2
					</p>
				
					<p>&nbsp;&nbsp;
						To reveal potentially important pathways or groups of genes involved in <span class="trait">resistance</span> to SCN, the constitutively-regulated 297 genes were grouped into the following functional categories using MapMan [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]: cell wall metabolism (seven genes), lipid metabolism (seven genes), secondary metabolism (five genes), abiotic stress (six genes), biotic stress (24 genes), transcription regulation (transcription factors, TFs, 24 genes), signaling (16 genes), hormonal pathways (eight genes), protein modification and degradation (11 genes), transport (12 genes), development (8 genes), enzyme families (25 genes: 4 UDP glucosyl and glucoronyl transferase genes, 3 cytochrome <span class="gene">P450</span> genes, etc
					</p>
				
					<p>&nbsp;&nbsp;
						03G034400 <span class="gene">(Glyma03g04030),</span> both encoding an NBS-LRR type <span class="trait">resistance</span> protein, were highly up-regulated in both resistant lines, and genes Glyma
					</p>
				
					<p>&nbsp;&nbsp;
						08G317400 <span class="gene">(Glyma08g42930),</span> both also encoding an NBS-LRR type <span class="trait">resistance</span> protein, were significantly down-regulated in both resistant lines (see Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						06G311100 <span class="gene">(Glyma06g46810)]</span> encode NBS-LRR type disease <span class="trait">resistance</span> proteins, accounting for a significant number of the total 319 NBS-LRR <span class="trait">resistance</span> proteins encoded by the soybean genome [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						05G204600 <span class="gene">(Glyma05g38130,</span> encoding a thaumatin-like PR protein) significantly enhanced soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg1</span> </italic> and <italic> Rgh4 </italic> are the two major QTL involved in soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although PI 437654 was shown to contain two known QTL ( <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> ), a new QTL for <span class="trait">resistance</span> to SCN HG types 0, 2.
					</p>
				
					<p>&nbsp;&nbsp;
						To reveal potentially important pathways or groups of genes involved in <span class="trait">resistance</span> to SCN, the constitutively-regulated 297 genes were grouped into the following functional categories using MapMan [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]: cell wall metabolism (seven genes), lipid metabolism (seven genes), secondary metabolism (five genes), abiotic stress (six genes), biotic stress (24 genes), transcription regulation (transcription factors, TFs, 24 genes), signaling (16 genes), hormonal pathways (eight genes), protein modification and degradation (11 genes), transport (12 genes), development (8 genes), enzyme families (25 genes: 4 UDP glucosyl and glucoronyl transferase genes, 3 cytochrome <span class="gene">P450</span> genes, etc.
					</p>
				
					<p>&nbsp;&nbsp;
						03G034400 <span class="gene">(Glyma03g04030),</span> both encoding an NBS-LRR type <span class="trait">resistance</span> protein, were highly up-regulated in both resistant lines, and genes Glyma.
					</p>
				
					<p>&nbsp;&nbsp;
						08G317400 <span class="gene">(Glyma08g42930),</span> both also encoding an NBS-LRR type <span class="trait">resistance</span> protein, were significantly down-regulated in both resistant lines (see Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						06G311100 <span class="gene">(Glyma06g46810)]</span> encode NBS-LRR type disease <span class="trait">resistance</span> proteins, accounting for a significant number of the total 319 NBS-LRR <span class="trait">resistance</span> proteins encoded by the soybean genome [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						05G204600 <span class="gene">(Glyma05g38130,</span> encoding a thaumatin-like PR protein) significantly enhanced soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Rhg1</span> </italic> and <italic> Rgh4 </italic> are the two major QTL involved in soybean <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetics and Adaptation of Soybean Cyst Nematode to Broad Spectrum Soybean <span class="trait">Resistance</span></h2>
			<h3>PMID:28064187</h3>
				

				
					<h3>PMCID:5345713</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gardner </surname>  <given-names> Michael </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heinz </surname>  <given-names> Robert </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jianying </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1"> Division of Plant Sciences and Bond Life Sciences Center, University of Missouri, Columbia, Missouri 65211 </aff>
				<br/> 
				年份：<day> 01 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In recent years, much has been discovered about the genes responsible for <span class="trait">resistance</span> in both groups, specifically underlying two quantitative trait loci named <italic> <span class="gene">Rhg1</span> </italic> (for <span class="trait">resistance</span> to <italic> H
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> </italic> ), and <italic> <span class="gene">Rhg4</span> </italic> (reviewed in <xref rid="bib16" ref-type="bibr"> Mitchum 2016 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Resistant soybean lines in group one only require <italic> <span class="gene">Rhg1</span> </italic> for <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> 2015 </xref> ), but also require a second QTL, <italic> <span class="gene">Rhg4</span> </italic> , where a single serine hydroxymethyltransferase (SHMT) gene controls <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations ( <xref rid="bib12" ref-type="bibr"> Liu <italic> et al
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">glycines</span> </italic> ), and <italic> <span class="gene">Rhg4</span> </italic> (reviewed in <xref rid="bib16" ref-type="bibr"> Mitchum 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Resistant soybean lines in group one only require <italic> <span class="gene">Rhg1</span> </italic> for <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations.
					</p>
				
					<p>&nbsp;&nbsp;
						</italic> 2015 </xref> ), but also require a second QTL, <italic> <span class="gene">Rhg4</span> </italic> , where a single serine hydroxymethyltransferase (SHMT) gene controls <span class="trait">resistance</span> to SCN HG type 0 (Race 3) populations ( <xref rid="bib12" ref-type="bibr"> Liu <italic> et al.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean cyst nematode culture collections and field populations from North Carolina and Missouri reveal high incidences of infection by viruses</h2>
			<h3>PMID:28141854</h3>
				

				
					<h3>PMCID:5283738</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ruark </surname>  <given-names> Casey L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Koenning </surname>  <given-names> Stephen R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Davis </surname>  <given-names> Eric L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Opperman </surname>  <given-names> Charles H. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lommel </surname>  <given-names> Steven A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mitchum </surname>  <given-names> Melissa G. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sit </surname>  <given-names> Tim L. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						RNAs were extracted from <italic> Glycine max </italic> (internal control ELF1B), <italic> <span class="trait">Heterodera</span> <span class="trait">glycines</span> </italic> (internal controls HgFAR1, <span class="gene">GAPDH</span> and PPN18 <italic> S </italic> ) or other nematode species (internal control PPN18 <italic> S </italic> )
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Loci and candidate genes conferring <span class="trait">resistance</span> to soybean cyst nematode HG type 2.5.7</h2>
			<h3>PMID:28615053</h3>
				

				
					<h3>PMCID:5471737</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> tengweili@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 1136 </institution-id>  <institution-id institution-id-type="GRID"> grid.412243.2 </institution-id>  <institution> Key Laboratory of Soybean Biology in Chinese Ministry of Education (Key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry),  </institution>  <institution> Northeast Agricultural University,  </institution>  </institution-wrap> Harbin, 150030 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0526 1937 </institution-id>  <institution-id institution-id-type="GRID"> grid.410727.7 </institution-id>  <institution> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI),  </institution>  <institution> Chinese Academy of Agricultural Sciences,  </institution>  </institution-wrap> Beijing, 100081 China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.410751.6 </institution-id>  <institution> Bioinformatics Division,  </institution>  <institution> Biomarker Technologies Corporation,  </institution>  </institution-wrap> Beijing, 101300 China  </aff>
				<br/> 
				年份：<day> 5 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To predict potential functions of genes in the flanking region of SNPs associated with the <span class="trait">resistance</span> to SCN, the 196 genes were grouped into the following functional categories using MapMap [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]: cell wall metabolism (eight genes), lipid metabolism (six genes), secondary metabolism (nine genes), biotic stress (five genes), signaling (nine genes), transcription regulation (transcription factors, TFs, 32 genes), hormonal metabolism (13 genes), redox group (two genes), protein modification and degradation (31 genes), transport (eight genes), development (two genes), DNA synthesis or chromatin structure (six genes), miscellaneous group (15 genes: 3 cytochrome <span class="gene">P450</span> genes, 2 GDSL-motif lipase, 2 nitrile lyases etc
					</p>
				
					<p>&nbsp;&nbsp;
						Major QTL, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were valuable resources for SCN <span class="trait">resistance,</span> but were frequently not durable, reflecting shifts in the SCN population that resulted in a loss of SCN <span class="trait">resistance</span> in major QTL [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						One novel QTL on Chr10 (qSCN10) was identified in PI 567516C [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ], which could confer high SCN <span class="trait">resistance</span> to soybean lacking the two known major genes, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Major QTL, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were valuable resources for SCN <span class="trait">resistance,</span> but were frequently not durable, reflecting shifts in the SCN population that resulted in a loss of SCN <span class="trait">resistance</span> in major QTL [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						One novel QTL on Chr10 (qSCN10) was identified in PI 567516C [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ], which could confer high SCN <span class="trait">resistance</span> to soybean lacking the two known major genes, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic>.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>