<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>High protein</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>High protein</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Extracellular Enzyme Composition and Functional Characteristics of Aspergillus niger An-76 Induced by Food Processing Byproducts and Based on Integrated Functional-omics.</h2>
			<h3>PMID:29334221</h3>
				

				
			<p>
				作者：Liu L, Gong W, Sun X, Chen G, Wang L.
				<br/> 
				年份：2018 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Byproducts of food processing can be utilized for the production of high-value-added enzyme cocktails. In this study, we utilized integrated functional-omics technology to analyze composition and functional characteristics of extracellular enzymes produced by Aspergillus niger grown on food processing byproducts. The results showed that oligosaccharides constituted by arabinose, xylose, and glucose in wheat bran were able to efficiently induce the production of extracellular enzymes of A. niger. Compared with other substrates, wheat bran was more effective at inducing the secretion of β-glucosidases from <span class="gene">GH1</span> and <span class="gene">GH3</span> families, as well as &gt;50% of proteases from A1-family aspartic proteases. Compared with proteins induced by single wheat bran or soybean dregs, the <span class="trait">protein</span> <span class="trait">yield</span> induced by their mixture was doubled, and the time required to reach peak enzyme activity was shortened by 25%. This study provided a technical platform for the complex formulation of various substrates and functional analysis of extracellular enzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide scan for seed composition provides insights into soybean quality improvement and the impacts of domestication and breeding.</h2>
			<h3>PMID:29305230</h3>
				

				
			<p>
				作者：Zhang J(1), Wang X(1), Lu Y(1), Bhusal SJ(1), Song Q(2), Cregan PB(2), Yen Y(3), Brown M(4), Jiang GL(5).
				<br/> 
				年份：2018 Jan 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complex genetic architecture of quality traits has hindered efforts to modify seed nutrients in soybean (Glycine max). Genome-wide association studies were conducted for the seed composition including <span class="trait">protein,</span> oil, fatty acids and amino acids using 313 diverse soybean germplasm accessions genotyped with a high-density SNP array. A total of 87 chromosomal regions were identified to be associated with seed composition, explaining 8-89% of genetic variances. The candidate genes <span class="gene">GmSAT1,</span> AK-HSDH, <span class="gene">SACPD-C</span> and <span class="gene">FAD3A</span> of known function, and putative MtN21 nodulin, FATB and steroid-5-α-reductase involved in N2 fixation, amino acid biosynthesis and fatty acid metabolism were found at the major-effect loci. Further analysis of additional germplasm accessions indicated that these major-effect loci had been subjected to domestication or modern breeding selection, and the allelic variants and distributions were relevant to geographic regions. This study also revealed that amino acid concentrations related to seed weight and to total <span class="trait">protein</span> had different genetic basis. It helps uncover the in-depth genetic mechanism of the intricate relationships among the seed compounds. This study not only provides valuable genes and markers for soybean nutrient improvement both quantitatively and qualitatively, but also offers insights into the alteration of soybean quality during domestication and breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Novel mechanisms for organic acid-mediated aluminium tolerance in roots and leaves of two contrasting soybean genotypes.</h2>
			<h3>PMID:29302304</h3>
				

				
			<p>
				作者：Huang SC(1)(2)(3), Chu SJ(1)(2), Guo YM(1)(2), Ji YJ(1)(2), Hu DQ(1)(2), Cheng J(1)(2), Lu GH(1)(2), Yang RW(1)(2), Tang CY(1)(2), Qi JL(1)(2), Yang YH(1)(2).
				<br/> 
				年份：2017 Nov 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aluminium (Al) toxicity is one of the most important limiting factors for crop <span class="trait">yield</span> in acidic soils. However, the mechanisms that confer Al tolerance still remain largely unknown. To understand the molecular mechanism that confers different tolerance to Al, we performed global transcriptome analysis to the roots and leaves of two contrasting soybean genotypes, BX10 (Al-tolerant) and BD2 (Al-sensitive) under 0 and 50 μM Al3+ treatments, respectively. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes analyses revealed that the expression levels of the genes involved in lipid/carbohydrate metabolism and jasmonic acid (JA)-mediated signalling pathway were highly induced in the roots and leaves of both soybean genotypes. The gene encoding enzymes, including pyruvate kinase, phosphoenolpyruvate carboxylase, ATP-citrate lyase and glutamate-oxaloacetate transaminase 2, associated with organic acid metabolism were differentially expressed in the BX10 roots. In addition, the genes involved in citrate transport were differentially expressed. Among these genes, <span class="gene">FRD3b</span> was down-regulated only in BD2, whereas the other two multidrug and toxic compound extrusion genes were up-regulated in both soybean genotypes. These findings confirmed that BX10 roots secreted more citrate than BD2 to withstand Al stress. The gene encoding enzymes or regulators, such as lipoxygenase, 12-oxophytodienoate reductase, acyl-CoA oxidase and jasmonate ZIM-domain proteins, involved in JA biosynthesis and signalling were preferentially induced in BD2 leaves. This finding suggests that the JA defence response was activated, possibly weakening the growth of aerial parts because of excessive resource consumption and ATP biosynthesis deficiency. Our results suggest that the Al sensitivity in some soybean varieties could be attributed to the low level of citrate metabolism and exudation in the roots and the <span class="trait">high</span> level of JA-mediated defence response in the leaves.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean (Glycine max) WRINKLED1 transcription factor, <span class="gene">GmWRI1a,</span> positively regulates seed oil accumulation.</h2>
			<h3>PMID:29138932</h3>
				

				
			<p>
				作者：Chen L(1), Zheng Y(2), Dong Z(3), Meng F(1), Sun X(1), Fan X(1), Zhang Y(1), Wang M(1), Wang S(4).
				<br/> 
				年份：2017 Nov 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is the world's most important leguminous crop producing high-quality <span class="trait">protein</span> and oil. Elevating oil accumulation in soybean seed is always many researchers' goal. WRINKLED1 (WRI1) encodes a transcription factor of the APETALA2/ethylene responsive element-binding <span class="trait">protein</span> (AP2/EREBP) family that plays important roles during plant seed oil accumulation. In this study, we isolated and characterized three distinct orthologues of <span class="gene">WRI1</span> in soybean (Glycine max) that display different organ-specific expression patterns, among which <span class="gene">GmWRI1a</span> was highly expressed in maturing soybean seed. Electrophoretic mobility shift assays and yeast one-hybrid experiments demonstrated that the <span class="gene">GmWRI1a</span> <span class="trait">protein</span> was capable of binding to AW-box, a conserved sequence in the proximal upstream regions of many genes involved in various steps of oil biosynthesis. Transgenic soybean seeds overexpressing <span class="gene">GmWRI1a</span> under the control of the seed-specific napin promoter showed the increased total oil and fatty acid content and the changed fatty acid composition. Furthermore, basing on the activated expressions in transgenic soybean seeds and existence of AW-box element in the promoter regions, direct downstream genes of <span class="gene">GmWRI1a</span> were identified, and their products were responsible for fatty acid production, elongation, desaturation and export from plastid. We conclude that <span class="gene">GmWRI1a</span> transcription factor can positively regulate oil accumulation in soybean seed by a complex gene expression network related to fatty acid biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in <span class="trait">high</span> salt and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						YABBY family is a plant specific transcription factor family, with the typical N-terminal C2C2 type zinc finger domain and the C-terminal YABBY conservative structure domain, which plays important biological roles in plant growth, development and morphogenesis. In this study, a total of 17 YABBY genes were identified in the soybean genome. The results of this research showed that 17 soybean YABBY genes were located on 11 chromosomes. Analysis of putative cis-acting elements showed that soybean YABBY genes contained lots of MYB and MYC elements. Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses. And the transient expression in Arabidopsis protoplasts showed that <span class="gene">GmYABBY3</span> <span class="trait">protein</span> distributed uniformly the whole cells, while <span class="gene">GmYABBY10</span> <span class="trait">protein</span> was mainly localized in the membranes and cytoplasm and <span class="gene">GmYABBY16</span> <span class="trait">protein</span> was localized the nucleus and membranes. To further identify the function of <span class="gene">GmYABBY10,</span> we obtained the transgenic Arabidopsis overexpression <span class="gene">GmYABBY10.</span> Based on germination and seedling root arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment. While the root length and root surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings. When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and salt stresses. This study provided valuable information regarding the classification and functions of YABBY genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Neodiversification of homeologous CLAVATA1-like receptor kinase genes in soybean leads to distinct developmental outcomes.</h2>
			<h3>PMID:28827708</h3>
				

				
			<p>
				作者：Mirzaei S(1)(2), Batley J(3), El-Mellouki T(4), Liu S(4), Meksem K(4), Ferguson BJ(1), Gresshoff PM(5).
				<br/> 
				年份：2017 Aug 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The CLAVATA pathway that regulates stem cell numbers of the shoot apical meristem has exclusively been studied in Arabidopsis; as such insight into other species is warranted. In this study, a <span class="gene">GmCLV1A</span> mutant (F-S562L) with altered lateral organ development, and two mutants of <span class="gene">GmNARK,</span> isolated from a Forrest M2 population (EMS-mutated soybean) were studied. <span class="gene">GmCLV1A</span> and <span class="gene">GmNARK</span> encode for LRR receptor kinases, and share 92% of <span class="trait">protein</span> sequence. While <span class="gene">GmNARK</span> is critical for systemic regulation of nodulation (new organ made on the root through symbiosis), we show that <span class="gene">GmCLV1A</span> functions locally and has no apparent function in nodulation or root development. However, a recessive, loss-of-function mutation (S562L) in a putative S-glycosylation site of <span class="gene">GmCLV1A</span> causes stem nodal identity alterations as well as flower and pod abnormalities (deformed flower and pod). The mutant also exhibits a homeotic phenotype, displaying abnormal leaf development/number, vein-derived leaf emergence, and a thick, faciated stem. The mutant phenotype is also temperature-sensitive. Interestingly, a novel truncated version of <span class="gene">GmCLV1A</span> was identified upstream of <span class="gene">GmCLV1A</span> that is absent from <span class="gene">GmNARK,</span> but is present upstream of the <span class="gene">GmNARK</span> orthologues, MtSUNN and PvNARK. Taken together, our findings indicate that <span class="gene">GmCLV1A</span> acts on shoot architecture, whereas <span class="gene">GmNARK,</span> functions in controlling nodule numbers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation, characterization and selection of indigenous <italic> Bradyrhizobium </italic> strains with outstanding symbiotic performance to increase soybean yields in Mozambique</h2>
			<h3>PMID:28775390</h3>
				

				
					<h3>PMCID:5521954</h3>
					<h4>期刊:Agriculture, Ecosystems &#x00026; Environment</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chibeba </surname>  <given-names> Amaral Machaculeha </given-names>  </name>  <email> amaral_chibeba@yahoo.com </email>  <xref rid="aff0005" ref-type="aff"> a </xref>  <xref rid="aff0010" ref-type="aff"> b </xref>  <xref rid="aff0015" ref-type="aff"> c </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kyei-Boahen </surname>  <given-names> Stephen </given-names>  </name>  <email> s.boahen@cgiar.org </email>  <xref rid="aff0015" ref-type="aff"> c </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guimar&#x000e3;es </surname>  <given-names> Maria de F&#x000e1;tima </given-names>  </name>  <email> mfatima@uel.br </email>  <xref rid="aff0005" ref-type="aff"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nogueira </surname>  <given-names> Marco Antonio </given-names>  </name>  <email> marco.nogueira@embrapa.br </email>  <xref rid="aff0010" ref-type="aff"> b </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hungria </surname>  <given-names> Mariangela </given-names>  </name>  <email> mariangela.hungria@embrapa.br </email>  <email> biotecnologia.solo@hotmail.com </email>  <email> hungria@cnpq.br </email>  <email> hungria@pq.cnpq.br </email>  <xref rid="aff0010" ref-type="aff"> b </xref>  <xref rid="cor0005" ref-type="corresp"> &#x0204e; </xref>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean inoculation with effective rhizobial strains makes unnecessary the use of N-fertilizers in the tropics. A frequently reported problem is the failure of the inoculant strains to overcome the competition imposed by indigenous rhizobial populations. The screening of indigenous rhizobia, already adapted to local conditions, searching for highly effective strains for use as inoculants represents a promising strategy in overcoming inoculation failure. The objective of this study was to isolate and characterize indigenous rhizobia and to identify strains that hold potential to be included in inoculant formulations for soybean production, with both promiscuous and non-promiscuous soybean cultivars, in Mozambican agro-climatic conditions. A total of 105 isolates obtained from nodules of promiscuous soybean grown at 15 sites were screened for N2-fixation effectiveness in the greenhouse along with five commercial strains. Eighty-seven isolates confirmed the ability to form effective nodules on soybean and were used for genetic characterization by rep-PCR (BOX) and sequencing of the 16S rRNA gene, and also for symbiotic effectiveness. BOX-PCR fingerprinting revealed remarkable genetic diversity, with 41 clusters formed, considering a similarity level of 65%. The 16S rRNA analysis assigned the isolates to the genera Bradyrhizobium (75%) and Agrobacterium/Rhizobium (25%). Great variability in symbiotic effectiveness was detected among the indigenous rhizobia from Mozambique, with ten isolates performing better than the commercial strain B. diazoefficiens USDA 110, the best reference strain, and 51 isolates with lower performance than all reference strains. Thirteen of the best isolates from the first greenhouse trial were evaluated, along with the five commercial strains, in two promiscuous (TGx 1963-3F and TGx 1835-10E) and one non-promiscuous (BRS 284) soybean cultivars in a second greenhouse trial. In general the promiscous soybeans responded better to inoculation. The 13 isolates were also characterized for tolerance to acidity and alkalinity (pH 3.5 and 9.0, respectively), salinity (0.1, 0.3 and 0.5 mol L-1 of NaCl) and <span class="trait">high</span> temperatures (35, 40 and 45 °C) in vitro. Five isolates, three (Moz 4, Moz 19 and Moz 22) belonging to the superclade B. elkanii and two (Moz 27 and Moz 61) assigned to the superclade B. japonicum, consistently showed <span class="trait">high</span> symbiotic effectiveness, suggesting that the inoculation with indigenous rhizobia adapted to local conditions represents a possible strategy for increasing soybean yields in Mozambique. Phylogenetic position of the five elite isolates was confirmed by the MLSA with four protein-coding housekeeping genes, dnaK, glnII, <span class="gene">gyrB</span> and recA.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean inoculation with effective rhizobial strains makes unnecessary the use of N-fertilizers in the tropics. A frequently reported problem is the failure of the inoculant strains to overcome the competition imposed by indigenous rhizobial populations. The screening of indigenous rhizobia, already adapted to local conditions, searching for highly effective strains for use as inoculants represents a promising strategy in overcoming inoculation failure. The objective of this study was to isolate and characterize indigenous rhizobia and to identify strains that hold potential to be included in inoculant formulations for soybean production, with both promiscuous and non-promiscuous soybean cultivars, in Mozambican agro-climatic conditions. A total of 105 isolates obtained from nodules of promiscuous soybean grown at 15 sites were screened for N <sub> 2 </sub> -fixation effectiveness in the greenhouse along with five commercial strains. Eighty-seven isolates confirmed the ability to form effective nodules on soybean and were used for genetic characterization by rep-PCR (BOX) and sequencing of the 16S rRNA gene, and also for symbiotic effectiveness. BOX-PCR fingerprinting revealed remarkable genetic diversity, with 41 clusters formed, considering a similarity level of 65%. The 16S rRNA analysis assigned the isolates to the genera <italic> Bradyrhizobium </italic> (75%) and <italic> Agrobacterium/Rhizobium </italic> (25%). Great variability in symbiotic effectiveness was detected among the indigenous rhizobia from Mozambique, with ten isolates performing better than the commercial strain <italic> B. diazoefficiens </italic> USDA 110, the best reference strain, and 51 isolates with lower performance than all reference strains. Thirteen of the best isolates from the first greenhouse trial were evaluated, along with the five commercial strains, in two promiscuous (TGx 1963-3F and TGx 1835-10E) and one non-promiscuous (BRS 284) soybean cultivars in a second greenhouse trial. In general the promiscous soybeans responded better to inoculation. The 13 isolates were also characterized for tolerance to acidity and alkalinity (pH 3.5 and 9.0, respectively), salinity (0.1, 0.3 and 0.5&#x000a0;mol&#x000a0;L <sup> &#x02212;1 </sup> of NaCl) and <span class="trait">high</span> temperatures (35, 40 and 45&#x000a0;&#x000b0;C) <italic> in vitro </italic> . Five isolates, three (Moz 4, Moz 19 and Moz 22) belonging to the superclade <italic> B. elkanii </italic> and two (Moz 27 and Moz 61) assigned to the superclade <italic> B. japonicum, </italic> consistently showed <span class="trait">high</span> symbiotic effectiveness, suggesting that the inoculation with indigenous rhizobia adapted to local conditions represents a possible strategy for increasing soybean yields in Mozambique. Phylogenetic position of the five elite isolates was confirmed by the MLSA with four protein-coding housekeeping genes, <italic> <span class="gene">dnaK</span> </italic> , <italic> <span class="gene">glnII</span> </italic> , <italic> <span class="gene">gyrB</span> </italic> and <italic> recA. </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						For the five isolates with outstanding symbiotic performance (from Section <xref rid="sec0115" ref-type="sec"> 3.3 </xref> ), a better phylogenetic definition was achieved by the MLSA analysis with four housekeeping genes, <italic> dnaK, glnII, <span class="gene">gyrB</span> </italic> and <italic> <span class="gene">recA</span> </italic> . Each single-gene based phylogeny showed the five strains from Mozambique separated in two groups (Supplementary Figs. S2 to S5). In the first group, strains Moz 27 and Moz 61 shared 100%, 100%, 99.2% and 99.2% of nucleotide identity in the <italic> dnaK, glnII, <span class="gene">gyrB</span> </italic> and <italic> <span class="gene">recA</span> </italic> phylogenies, respectively (Supplementary Table S4), and were clustered tightly (96&#x02013;99% bootstrap support) with <italic> B. japonicum </italic> USDA 6 <sup> T </sup> . In the second group, Moz 4, Moz 19 and Moz 22 shared 99.5&#x02013;100%, 99.3&#x02013;99.7%, 95.7&#x02013;100% and 99.7&#x02013;100% similarities in the <italic> dnaK, glnII, <span class="gene">gyrB</span> </italic> and <italic> <span class="gene">recA</span> </italic> phylogenies, respectively, with <italic> B. elkanii </italic> USDA 76 <sup> T </sup> (Supplementary Table S4), also sharing <span class="trait">high</span> bootstrap support (91&#x02013;99%) (Supplementary Figs. S2 to S5), except in the <italic> <span class="gene">gyrB</span> </italic> phylogeny, where strain Moz 4 was separated from this group (Supplementary Fig. S4).
					</p>
				
					<p>&nbsp;&nbsp;
						In the analysis of the concatenated genes, strains Moz 27 and Moz 61 shared 99.5% of nucleotide identity (Supplementary Table S4) and in the phylogenetic tree formed a well supported cluster (100% of bootstrap) with <italic> B. japonicum </italic> USDA 6 <sup> T </sup> ( <xref rid="fig0020" ref-type="fig"> Fig. 4 </xref> ). Likewise, strains Moz 4, Moz 19 and Moz 22 shared 98.5&#x02013;99.7% of nucleotide identity of the concatenated genes (Supplementary Table S4) and clustered with <span class="trait">high</span> bootstrap support (98%) with <italic> B. elkanii </italic> USDA 76 <sup> T </sup> ( <xref rid="fig0020" ref-type="fig"> Fig. 4 </xref> ). <fig id="fig0020"> <label> Fig. 4 </label> <caption> <p> Maximum &#x02013; likelihood phylogeny based on concatenated gene sequences [ <italic> <span class="gene">dnaK</span> </italic> (223&#x000a0;bp), <italic> <span class="gene">glnII</span> </italic> (480&#x000a0;bp), <italic> <span class="gene">gyrB</span> </italic> (419&#x000a0;bp) and <italic> <span class="gene">recA</span> </italic> (375&#x000a0;bp)] showing the relationships among five rhizobial isolates from Mozambique (in bold) with type ( <sup> T </sup> ) and reference strains used in commercial inoculants, <italic> B. elkanii </italic> SEMIA 587 and SEMIA 5019, <italic> B. japonicum </italic> SEMIA 5079, and <italic> B. diazoefficiens </italic> SEMIA 5080 and USDA 110 (with an asterix). <italic> Ensifer fredii </italic> USDA 205 <sup> T </sup> was inluded as an outgroup. Only bootstrap confidence levels &#x0003e;70% are shown at the internodes. The scale <span class="gene">bar</span> indicates 5 substitutions per 100 nucleotides; a and b represent the clustering of the five best nitrogen fixer strains from Mozambique.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of complete plastid genomes from wild soybean ( <italic> Glycine soja </italic> ) and nine other <italic> Glycine </italic> species</h2>
			<h3>PMID:28763486</h3>
				

				
					<h3>PMCID:5538705</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0001-8216-7549 </contrib-id>  <name>  <surname> Asaf </surname>  <given-names> Sajjad </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Abdul Latif </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Aaqil Khan </surname>  <given-names> Muhammad </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Muhammad Imran </surname>  <given-names> Qari </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Sang-Mo </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Al-Hosni </surname>  <given-names> Khdija </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data Curation </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jeong </surname>  <given-names> Eun Ju </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Software </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Ko Eun </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> In-Jung </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 7 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plastid genomes of different plant species exhibit significant variation, thereby providing valuable markers for exploring evolutionary relationships and population genetics. Glycine soja (wild soybean) is recognized as the wild ancestor of cultivated soybean (G. max), representing a valuable genetic resource for soybean breeding programmes. In the present study, the complete plastid genome of G. soja was sequenced using Illumina paired-end sequencing and then compared it for the first time with previously reported plastid genome sequences from nine other Glycine species. The G. soja plastid genome was 152,224 bp in length and possessed a typical quadripartite structure, consisting of a pair of inverted repeats (IRa/IRb; 25,574 bp) separated by small (178,963 bp) and large (83,181 bp) single-copy regions, with a 51-kb inversion in the large single-copy region. The genome encoded 134 genes, including 87 protein-coding genes, eight ribosomal RNA genes, and 39 transfer RNA genes, and possessed 204 randomly distributed microsatellites, including 15 forward, 25 tandem, and 34 palindromic repeats. Whole-plastid genome comparisons revealed an overall <span class="trait">high</span> degree of sequence similarity between G. max and G. gracilis and some divergence in the intergenic spacers of other species. Greater numbers of indels and SNP substitutions were observed compared with G. cyrtoloba. The sequence of the <span class="gene">accD</span> gene from G. soja was highly divergent from those of the other species except for G. max and G. gracilis. Phylogenomic analyses of the complete plastid genomes and 76 shared genes yielded an identical topology and indicated that G. soja is closely related to G. max and G. gracilis. The complete G. soja genome sequenced in the present study is a valuable resource for investigating the population and evolutionary genetics of Glycine species and can be used to identify related species.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The plastid genomes of different plant species exhibit significant variation, thereby providing valuable markers for exploring evolutionary relationships and population genetics. <italic> Glycine soja </italic> (wild soybean) is recognized as the wild ancestor of cultivated soybean (G. <italic> max </italic> ), representing a valuable genetic resource for soybean breeding programmes. In the present study, the complete plastid genome of <italic> G </italic> . <italic> soja </italic> was sequenced using Illumina paired-end sequencing and then compared it for the first time with previously reported plastid genome sequences from nine other <italic> Glycine </italic> species. The <italic> G </italic> . <italic> soja </italic> plastid genome was 152,224 bp in length and possessed a typical quadripartite structure, consisting of a pair of inverted repeats (IRa/IRb; 25,574 bp) separated by small (178,963 bp) and large (83,181 bp) single-copy regions, with a 51-kb inversion in the large single-copy region. The genome encoded 134 genes, including 87 protein-coding genes, eight ribosomal RNA genes, and 39 transfer RNA genes, and possessed 204 randomly distributed microsatellites, including 15 forward, 25 tandem, and 34 palindromic repeats. Whole-plastid genome comparisons revealed an overall <span class="trait">high</span> degree of sequence similarity between <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> gracilis </italic> and some divergence in the intergenic spacers of other species. Greater numbers of indels and SNP substitutions were observed compared with <italic> G </italic> . <italic> cyrtoloba </italic> . The sequence of the <italic> <span class="gene">accD</span> </italic> gene from <italic> G </italic> . <italic> soja </italic> was highly divergent from those of the other species except for <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> gracilis </italic> . Phylogenomic analyses of the complete plastid genomes and 76 shared genes yielded an identical topology and indicated that <italic> G </italic> . <italic> soja </italic> is closely related to <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> gracilis </italic> . The complete <italic> G </italic> . <italic> soja </italic> genome sequenced in the present study is a valuable resource for investigating the population and evolutionary genetics of <italic> Glycine </italic> species and can be used to identify related species.
					</p>
				
					<p>&nbsp;&nbsp;
						The chloroplast (cp) is a key organelle in photosynthesis and in the biosynthesis of fatty acids, starches, amino acids, and pigments [ <xref rid="pone.0182281.ref001" ref-type="bibr"> 1 </xref> , <xref rid="pone.0182281.ref002" ref-type="bibr"> 2 </xref> ]. In angiosperms, plastomes are typically circular and highly conserved, ranging from 115 to 165 kb in length and comprising a small single-copy region (SSC; 16&#x02013;27 kb) and a large-single-copy region (LSC; 80&#x02013;90 kb), separated by a pair of inverted repeats (IRs) [ <xref rid="pone.0182281.ref003" ref-type="bibr"> 3 </xref> , <xref rid="pone.0182281.ref004" ref-type="bibr"> 4 </xref> ]. Most plastomes also contain 110&#x02013;130 genes encoding up to 80 unique proteins and approximately 4 rRNAs and 30 tRNAs. Most of the protein-coding genes are associated with photosynthesis or other biochemical processes in plant cells, such as synthesis of amino acids, sugars, vitamins, lipids, pigments, and starches, storage, nitrogen metabolism, sulphate reduction, and immune responses [ <xref rid="pone.0182281.ref005" ref-type="bibr"> 5 </xref> , <xref rid="pone.0182281.ref006" ref-type="bibr"> 6 </xref> ]. In contrast to mitochondrial and nuclear genomes, the plastomes of plants are highly conserved in regard to gene structure, organization, and content [ <xref rid="pone.0182281.ref004" ref-type="bibr"> 4 </xref> ]. However, gene duplications, mutations, rearrangements, and losses have been observed in some angiosperm lineages [ <xref rid="pone.0182281.ref007" ref-type="bibr"> 7 </xref> ]. Rearrangements of plastid gene order are generally observed in taxa with plastomes that exhibit at least one of the following qualities: variable IR region size, loss of one IR region, a <span class="trait">high</span> frequency of small dispersed repeats, complete or near-complete lack of photosynthesis, or biparental cp inheritance [ <xref rid="pone.0182281.ref008" ref-type="bibr"> 8 </xref> ]. In addition, plastome inversions have been reported in a number of angiosperm families, including Asteraceae [ <xref rid="pone.0182281.ref009" ref-type="bibr"> 9 </xref> ], Campanulaceae [ <xref rid="pone.0182281.ref010" ref-type="bibr"> 10 </xref> ], Onagraceae [ <xref rid="pone.0182281.ref011" ref-type="bibr"> 11 </xref> ], Leguminosae [ <xref rid="pone.0182281.ref012" ref-type="bibr"> 12 </xref> ], and Geraniaceae [ <xref rid="pone.0182281.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0182281.ref014" ref-type="bibr"> 14 </xref> ]. The plastomes of several members of the Papilionoideae also exhibit significant variation and rearrangement, including the loss of an IR region [ <xref rid="pone.0182281.ref015" ref-type="bibr"> 15 </xref> ] and inversion of a 50-kb portion of the LSC [ <xref rid="pone.0182281.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0182281.ref017" ref-type="bibr"> 17 </xref> ]. These features, as well as the loss of introns from the <italic> <span class="gene">rps12</span> </italic> and <italic> <span class="gene">clpP</span> </italic> genes [ <xref rid="pone.0182281.ref018" ref-type="bibr"> 18 </xref> , <xref rid="pone.0182281.ref019" ref-type="bibr"> 19 </xref> ] and transfer of <italic> <span class="gene">rpl22</span> </italic> to the nucleus [ <xref rid="pone.0182281.ref020" ref-type="bibr"> 20 </xref> , <xref rid="pone.0182281.ref021" ref-type="bibr"> 21 </xref> ], have been well documented, and their occurrence has been mapped onto the phylogeny of Leguminosae [ <xref rid="pone.0182281.ref019" ref-type="bibr"> 19 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Among these codons, the most and least frequently used were <span class="gene">AAA</span> (n = 1,181), which encodes lysine, and ATC and ATT (n = 1, n = 1), which both encode methionine. The AT contents of the 1 <sup> st </sup> , 2 <sup> nd </sup> , and 3 <sup> rd </sup> codon positions of CDSs were 55.7%, 62.9%, and 72.4%, respectively ( <xref ref-type="table" rid="pone.0182281.t003"> Table 3 </xref> ). The <span class="trait">high</span> AT content observed at the 3 <sup> rd </sup> codon position is similar to that reported for the plastomes of other terrestrial plants [ <xref rid="pone.0182281.ref054" ref-type="bibr"> 54 </xref> , <xref rid="pone.0182281.ref057" ref-type="bibr"> 57 </xref> , <xref rid="pone.0182281.ref058" ref-type="bibr"> 58 </xref> ]. In addition, 46.36% and 57.65% of the preferred synonymous codons (RSCU &#x0003e; 1) ended with A or U and C or G, respectively, and 44.30% and 55.20% of the non-preferred synonymous codons (RSCU &#x0003c; 1) ended with C or G and A or U, respectively. However, there was no bias in start codon usage (AUG or UGG; RSCU = 1; <xref ref-type="table" rid="pone.0182281.t004"> Table 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Pairwise alignment of the new <italic> G </italic> . <italic> soja </italic> plastome with the old <italic> G </italic> . <italic> soja </italic> plastome and those of nine other genomes showed a <span class="trait">high</span> degree of synteny. The annotation of the new <italic> G </italic> . <italic> soja </italic> plastome was used as a reference for plotting the overall sequence identity of the plastomes of the other ten <italic> Glycine </italic> species in mVISTA ( <xref ref-type="fig" rid="pone.0182281.g004"> Fig 4 </xref> ). In the results, relatively lower sequence identity was observed between the plastomes of the seven other perennial species, especially in the <italic> <span class="gene">rpoC1</span> </italic> , <italic> <span class="gene">atpF</span> </italic> , <italic> <span class="gene">accD</span> </italic> , <italic> <span class="gene">clpP</span> </italic> , <italic> <span class="gene">rpl2</span> </italic> , <italic> <span class="gene">ndhA</span> </italic> , <italic> <span class="gene">ndhF</span> </italic> , <italic> <span class="gene">rps8</span> </italic> , <italic> <span class="gene">rps19</span> </italic> , and <italic> <span class="gene">ycf1</span> </italic> genes ( <xref ref-type="fig" rid="pone.0182281.g004"> Fig 4 </xref> ). In addition, the LSC and SSC regions were less similar than the two IR regions in all <italic> Glycine </italic> species, and the non-coding regions were more divergent than the coding regions. Highly divergent regions included the <italic> <span class="gene">matK</span> </italic> - <italic> <span class="gene">rbcL</span> </italic> , <italic> ycf3-psaA </italic> , <italic> trnC-rpoB </italic> , <italic> rpl20-clpP </italic> , <italic> rps16-trnQ </italic> , <italic> trnfM-trnM </italic> , <italic> psbM-petN </italic> , <italic> atpI-atpH </italic> , <italic> petA-psbJ </italic> , and <italic> ycf1-rps15 </italic> spacers, as reported previously [ <xref rid="pone.0182281.ref054" ref-type="bibr"> 54 </xref> , <xref rid="pone.0182281.ref055" ref-type="bibr"> 55 </xref> ]. Our results also confirmed similar differences among various coding regions in the analysed species, as suggested by Kumar et al. [ <xref rid="pone.0182281.ref094" ref-type="bibr"> 94 </xref> ]. On the other hand, <italic> G </italic> . <italic> soja </italic> exhibited <span class="trait">high</span> sequence identity with annual <italic> Glycine </italic> species ( <xref ref-type="supplementary-material" rid="pone.0182281.s006"> S1 Fig </xref> ), which suggest that they are highly conserved. However, the variation in similarity levels revealed various coding and non-coding regions where the <italic> G </italic> . <italic> soja </italic> exhibits divergence from these annual <italic> Glycine </italic> species ( <xref ref-type="supplementary-material" rid="pone.0182281.s006"> S1 Fig </xref> ). Similarly, we detected 10 relatively highly variable regions, including 4 gene regions and 6 intergenic regions of the cp genomes, that might be undergoing more rapid nucleotide substitution at species and cultivar levels ( <xref ref-type="supplementary-material" rid="pone.0182281.s007"> S2 Fig </xref> ) ( <italic> atpB-rbcL </italic> , <italic> trnT-trnL </italic> , <italic> trnS(GGA) </italic> - <italic> trnG(UCC) </italic> , <italic> psbD-trnT </italic> , <italic> <span class="gene">rps16</span> </italic> , <italic> rpl33-rpl18 </italic> , <italic> rpl16-rps3 </italic> , <italic> <span class="gene">ndhB</span> </italic> , <italic> <span class="gene">ycf1</span> </italic> and <italic> ycf15 </italic> ). These regions can be used as potential molecular markers for application in phylogenetic analyses of <italic> Glycine </italic> . Furthermore, various researchers have determined coding and non-coding regions of particularly <span class="trait">high</span> variability as potential molecular markers for <italic> Glycine </italic> species, such as <italic> trnS(GGA </italic> )- <italic> trnG(UCC) </italic> , <italic> rpl16-rps3 </italic> , <italic> trnT-trnL </italic> and <italic> atpB-rbcL </italic> [ <xref rid="pone.0182281.ref095" ref-type="bibr"> 95 </xref> &#x02013; <xref rid="pone.0182281.ref097" ref-type="bibr"> 97 </xref> ]. Similarly, it has been reported that non-coding regions in cp DNA show greater variability in nucleotide regions than coding regions, and these regions have become a major source of variability for phylogenetic studies in various species, including studies within <italic> Glycine </italic> species [ <xref rid="pone.0182281.ref098" ref-type="bibr"> 98 </xref> &#x02013; <xref rid="pone.0182281.ref100" ref-type="bibr"> 100 </xref> ]. Furthermore, comparison of the plastomes of <italic> G </italic> . <italic> soja </italic> and related species revealed 72 SNPs and 26 indels in relation to <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> gracilis </italic> , respectively ( <xref ref-type="supplementary-material" rid="pone.0182281.s002"> S2 Table </xref> ). These results confirmed that the highly conserved plastome can include interspecific mutations that may be useful for analysing both genetic diversity and phylogenetic relationships.
					</p>
				
					<p>&nbsp;&nbsp;
						Plastid genomes have been useful in phylogenetic, evolutionary, and molecular studies. During the last decade, many analyses based on the comparison of plastid protein-coding genes [ <xref rid="pone.0182281.ref110" ref-type="bibr"> 110 </xref> , <xref rid="pone.0182281.ref111" ref-type="bibr"> 111 </xref> ] and complete genome sequences [ <xref rid="pone.0182281.ref112" ref-type="bibr"> 112 </xref> ] have addressed phylogenetic questions at deep nodes and enhanced our understanding of enigmatic evolutionary relationships among angiosperms. The genus <italic> Glycine </italic> includes 28 species, separated into two subgenera ( <italic> Soja </italic> and <italic> Glycine </italic> ), the former of which includes both cultivated soybean ( <italic> G </italic> . <italic> max </italic> ) and its wild annual progenitor ( <italic> G </italic> . <italic> soja </italic> ), which are distributed in East Asia, including Japan, Korea, China, Russia, and Taiwan. <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> soja </italic> are both diploid (2n = 40) and interfertile and are thought to share highly similar genetic variation, although <italic> G </italic> . <italic> soja </italic> is much more variable than <italic> G </italic> . <italic> max </italic> [ <xref rid="pone.0182281.ref025" ref-type="bibr"> 25 </xref> , <xref rid="pone.0182281.ref113" ref-type="bibr"> 113 </xref> ]. Polymorphisms in the cpDNA of <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> soja </italic> have been used in numerous studies to assess maternal lineages and cytoplasmic diversity [ <xref rid="pone.0182281.ref114" ref-type="bibr"> 114 </xref> &#x02013; <xref rid="pone.0182281.ref119" ref-type="bibr"> 119 </xref> ]. Continued efforts have expanded our ability to differentiate and understand the genomic structure and phylogenetic relationships of <italic> Glycine </italic> species [ <xref rid="pone.0182281.ref028" ref-type="bibr"> 28 </xref> , <xref rid="pone.0182281.ref120" ref-type="bibr"> 120 </xref> , <xref rid="pone.0182281.ref121" ref-type="bibr"> 121 </xref> ]. The phylogeny and taxonomy of <italic> Glycine </italic> species in the <italic> Soja </italic> subgenus have been extensively investigated based on DNA variation, including nucleotide variation in nuclear ribosomal DNA (rDNA), intergenic spacer (ITS) regions [ <xref rid="pone.0182281.ref122" ref-type="bibr"> 122 </xref> ], cpDNA restriction sites [ <xref rid="pone.0182281.ref024" ref-type="bibr"> 24 </xref> , <xref rid="pone.0182281.ref029" ref-type="bibr"> 29 </xref> ], the histone gene <span class="gene">H3-D</span> [ <xref rid="pone.0182281.ref031" ref-type="bibr"> 31 </xref> ], A-199a [ <xref rid="pone.0182281.ref123" ref-type="bibr"> 123 </xref> ], and cpDNA intergenic spacer regions [ <xref rid="pone.0182281.ref025" ref-type="bibr"> 25 </xref> ]. However, the complete genome sequence provides more detailed insight [ <xref rid="pone.0182281.ref052" ref-type="bibr"> 52 </xref> , <xref rid="pone.0182281.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0182281.ref124" ref-type="bibr"> 124 </xref> ]. In the present study, the phylogenetic position of <italic> G </italic> . <italic> soja </italic> within its genus was established using the complete plastomes ( <xref ref-type="supplementary-material" rid="pone.0182281.s005"> S5 Table </xref> ) and shared genes of 10 <italic> Glycine </italic> species and various methods of phylogenetic analysis. Phylogenetic analysis indicated that the complete plastome and the 76 shared genes contained the same phylogenetic signal. In both datasets, <italic> G </italic> . <italic> soja </italic> formed a clade with <italic> G </italic> . <italic> max </italic> and <italic> G </italic> . <italic> gracilis </italic> , with <span class="trait">high</span> BI and bootstrap support values ( <xref ref-type="fig" rid="pone.0182281.g007"> Fig 7 </xref> , <xref ref-type="supplementary-material" rid="pone.0182281.s009"> S4 Fig </xref> ). Moreover, the tree topology confirmed previously reported relationships based on SSR and plastome data [ <xref rid="pone.0182281.ref114" ref-type="bibr"> 114 </xref> , <xref rid="pone.0182281.ref125" ref-type="bibr"> 125 </xref> ]. These results of the present study are in general agreement with the results of Gao and Gao (2016) [ <xref rid="pone.0182281.ref037" ref-type="bibr"> 37 </xref> ], who reported that <italic> G </italic> . <italic> gracilis </italic> is intermediate between the two species and is more closely related to <italic> G </italic> . <italic> max </italic> than <italic> G </italic> . <italic> soja </italic> . Furthermore, the results of the present study suggest that there is no conflict between the complete genome and the 76 shared gene datasets.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Dual functioning of plant arginases provides a third route for putrescine synthesis.</h2>
			<h3>PMID:28716421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Patel J(1), Ariyaratne M(1), Ahmed S(1), Ge L(1), Phuntumart V(1), Kalinoski A(2), Morris PF(3).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two biosynthetic routes are known for putrescine, an essential plant metabolite. Ornithine decarboxylase (ODC) converts ornithine directly to putrescine, while a second route for putrescine biosynthesis utilizes arginine decarboxylase (ADC) to convert arginine to agmatine, and two additional enzymes, agmatine iminohydrolase (AIH) and N-carbamoyl putrescine aminohydrolase (NLP1) to complete this pathway. Here we show that plants can use ADC and arginase/agmatinase (ARGAH) as a third route for putrescine synthesis. Transformation of Arabidopsis thaliana ADC2, and any of the arginases from A. thaliana (ARGAH1, or ARGHA2) or the soybean gene <span class="gene">Glyma.03g028000</span> <span class="gene">(GmARGAH)</span> into a yeast strain deficient in ODC, fully complemented the mutant phenotype. In vitro assays using purified recombinant enzymes of AtADC1 and AtARGAH2 were used to show that these enzymes can function in concert to convert arginine to agmatine and putrescine. Transient expression analysis of the soybean genes <span class="gene">(Glyma.06g007500,</span> ADC; <span class="gene">Glyma.03g028000</span> <span class="gene">GmARGAH)</span> and the A. thaliana ADC2 and ARGAH genes in leaves of Nicotiana benthamiana, showed that these proteins are localized to the chloroplast. Experimental support for this pathway also comes from the fact that expression of AtARGAH, but not AtAIH or AtNLP1, is co-regulated with AtADC2 in response to <span class="trait">drought,</span> oxidative stress, wounding, and methyl jasmonate treatments. Based on the <span class="trait">high</span> affinity of ARGAH2 for agmatine, its co-localization with ADC2, and typically low arginine levels in many plant tissues, we propose that these two enzymes can be major contributors to putrescine synthesis in many A. thaliana stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						Published by Elsevier B.V.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary Supplementation of Selenium-Enriched Probiotics Enhances Meat Quality of Broiler Chickens (Gallus gallus domesticus) Raised Under <span class="trait">High</span> Ambient Temperature.</h2>
			<h3>PMID:28702872</h3>
				

				
			<p>
				作者：Khan AZ(1)(2)(3), Kumbhar S(1)(2), Liu Y(1)(2), Hamid M(1)(2), Pan C(1)(2), Nido SA(1)(2), Parveen F(1)(2), Huang K(4).
				<br/> 
				年份：2017 Jul 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We investigated the effects of selenium-enriched probiotics (SP) on broiler meat quality under <span class="trait">high</span> ambient temperature and explore their underlying mechanisms. A total of 200 1-day-old male broiler chicks (Ross 308) were randomly allotted to four treatment groups, each with five replicates, in groups of ten birds. These birds were fed a corn-soybean basal diet (C), a basal diet plus probiotics supplementation (P), a basal diet plus Se supplementation in the form of sodium selenite (SS, 0.30 mg Se/kg), and a basal diet with the addition of selenium-enriched probiotics (SP, 0.30 mg Se/kg). The experiment lasted for 42 days. The birds were sacrificed by cervical dislocation, and the breast muscles were removed for further process. Our results showed that SP diet significantly increased (p &lt; 0.05) the physical (pH, colors, water holding capacity, drip loss, shear force) and sensory characteristics of breast meat. All P, SS, and SP supplementation enhanced the antioxidant system by increasing (p &lt; 0.05) the Se concentrations, glutathione (GSH) levels, activities of glutathione peroxidase (GSH-Px), and superoxide dismutase (SOD) whereas decreasing (p &lt; 0.05) malondialdehyde (MDA) levels, with SP being higher than P and SS. Moreover, SP diet significantly upregulated (p &lt; 0.05) the mRNA levels of glutathione peroxidase genes (GPx1, GPx4) while it downregulated heat stress biomarkers such as heat shock <span class="trait">protein</span> (HSP) 70 as compared to C, P, and <span class="gene">SS</span> diets. In conclusion, our findings suggest that SP may function as beneficial nutritive supplement that is capable of improving meat quality during the summer season.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The genetic architecture of water-soluble <span class="trait">protein</span> content and its genetic relationship to total <span class="trait">protein</span> content in soybean.</h2>
			<h3>PMID:28698580</h3>
				

				
			<p>
				作者：Zhang D(1), Lü H(2), Chu S(2), Zhang H(3), Zhang H(4), Yang Y(5), Li H(2), Yu D(6).
				<br/> 
				年份：2017 Jul 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water-soluble <span class="trait">protein</span> content (WSPC) is a critical factor in both soybean <span class="trait">protein</span> quality and functionality. However, the underlying genetic determinants are unclear. Here, we used 219 soybean accessions and 152 recombinant inbred lines genotyped with high-density markers and phenotyped in multi-environments to dissect the genetic architectures of WSPC and <span class="trait">protein</span> content (PC) using single- and multi-locus genome-wide association studies. In the result, a total of 32 significant loci, including 10 novel loci, significantly associated with WSPC and PC across multi-environments were identified, which were subsequently validated by linkage mapping. Among these loci, only four exhibited pleiotropic effects for PC and WSPC, explaining the low correlation coefficient between the two traits. The largest-effect WSPC-specific loci, GqWSPC8, was stably identified across all six environments and tagged to a linkage disequilibrium block comprising two promising candidate genes AAP8 and 2 S albumin, which might contribute to the <span class="trait">high</span> level of WSPC in some soybean varieties. In addition, two genes, <span class="gene">Glyma.13G123500</span> and <span class="gene">Glyma.13G194400</span> with relatively <span class="trait">high</span> expression levels at seed development stage compared with other tissues were regarded as promising candidates associated with the PC and WSPC, respectively. Our results provide new insights into the genetic basis of WSPC affecting soybean <span class="trait">protein</span> quality and <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmFDL19</span> enhances tolerance to <span class="trait">drought</span> and salt stresses in soybean.</h2>
			<h3>PMID:28640834  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li Y(1)(2), Chen Q(2), Nan H(2), Li X(2)(3), Lu S(2)(4), Zhao X(2)(4), Liu B(2)(4), Guo C(1), Kong F(2)(4), Cao D(2).
				<br/> 
				年份：2017 Jun 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The basic leucine zipper (bZIP) family of transcription factors plays an important role in the growth and developmental process as well as responds to various abiotic stresses, such as <span class="trait">drought</span> and <span class="trait">high</span> salinity. Our previous work identified <span class="gene">GmFDL19,</span> a bZIP transcription factor, as a flowering promoter in soybean, and the overexpression of <span class="gene">GmFDL19</span> caused early flowering in transgenic soybean plants. Here, we report that <span class="gene">GmFDL19</span> also enhances tolerance to <span class="trait">drought</span> and salt stress in soybean. <span class="gene">GmFDL19</span> was determined to be a group A member, and its transcription expression was highly induced by abscisic acid (ABA), polyethylene glycol (PEG 6000) and <span class="trait">high</span> salt stresses. Overexpression of <span class="gene">GmFDL19</span> in soybean enhanced <span class="trait">drought</span> and salt tolerance at the seedling stage. The relative plant height (RPH) and relative shoot dry weight (RSDW) of transgenic plants were significantly higher than those of the WT after PEG and salt treatments. In addition, the germination rate and plant height of the transgenic soybean were also significantly higher than that of WT plants after various salt treatments. Furthermore, we also found that <span class="gene">GmFDL19</span> could reduce the accumulation of Na+ ion content and up-regulate the expression of several ABA/stress-responsive genes in transgenic soybean. We also found that <span class="gene">GmFDL19</span> overexpression increased the activities of several antioxidative enzyme and chlorophyll content but reduced malondialdehyde content. These results suggested that <span class="gene">GmFDL19</span> is involved in soybean abiotic stress responses and has potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of the <span class="gene">RAV</span> Family in Soybean and Functional Identification of <span class="gene">GmRAV-03</span> Involvement in Salt and <span class="trait">Drought</span> Stresses and Exogenous ABA Treatment.</h2>
			<h3>PMID:28634481</h3>
				

				
			<p>
				作者：Zhao SP(1)(2), Xu ZS(2), Zheng WJ(1), Zhao W(1)(2), Wang YX(3), Yu TF(1), Chen M(2), Zhou YB(1), Min DH(1), Ma YZ(2), Chai SC(1), Zhang XH(1).
				<br/> 
				年份：2017 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcription factors play vital roles in plant growth and in plant responses to abiotic stresses. The <span class="gene">RAV</span> transcription factors contain a B3 DNA binding domain and/or an APETALA2 (AP2) DNA binding domain. Although genome-wide analyses of <span class="gene">RAV</span> family genes have been performed in several species, little is known about the family in soybean (Glycine max L.). In this study, a total of 13 <span class="gene">RAV</span> genes, named as <span class="gene">GmRAVs,</span> were identified in the soybean genome. We predicted and analyzed the amino acid compositions, phylogenetic relationships, and folding states of conserved domain sequences of soybean <span class="gene">RAV</span> transcription factors. These soybean <span class="gene">RAV</span> transcription factors were phylogenetically clustered into three classes based on their amino acid sequences. Subcellular localization analysis revealed that the soybean <span class="gene">RAV</span> proteins were located in the nucleus. The expression patterns of 13 <span class="gene">RAV</span> genes were analyzed by quantitative real-time PCR. Under <span class="trait">drought</span> stresses, the <span class="gene">RAV</span> genes expressed diversely, up- or down-regulated. Following NaCl treatments, all <span class="gene">RAV</span> genes were down-regulated excepting <span class="gene">GmRAV-03</span> which was up-regulated. Under abscisic acid (ABA) treatment, the expression of all of the soybean <span class="gene">RAV</span> genes increased dramatically. These results suggested that the soybean <span class="gene">RAV</span> genes may be involved in diverse signaling pathways and may be responsive to abiotic stresses and exogenous ABA. Further analysis indicated that <span class="gene">GmRAV-03</span> could increase the transgenic lines <span class="trait">resistance</span> to <span class="trait">high</span> salt and <span class="trait">drought</span> and result in the transgenic plants insensitive to exogenous ABA. This present study provides valuable information for understanding the classification and putative functions of the <span class="gene">RAV</span> transcription factors in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of hlyB and <span class="gene">mdh</span> genes confers halotolerance in Fremyella diplosiphon, a freshwater cyanobacterium.</h2>
			<h3>PMID:28554380</h3>
				

				
			<p>
				作者：Tabatabai B(1), Arumanayagam AS(2), Enitan O(1), Mani A(3), Natarajan SS(4), Sitther V(5).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fremyella diplosiphon is a freshwater cyanobacterium that has great potential as a biofuel agent due to its ability to grow in low light intensity and acclimation to different wavelengths. To enhance its halotolerance for growth in 35gL-1 sodium chloride (NaCl), plasmids harboring hemolysin B (hlyB) and malate dehydrogenase (mdh) genes were transformed into wild type F. diplosiphon (WT-Fd33). Electroporation-mediated overexpression of the genes resulted in two transformants, HSF33-1 and HSF33-2, with 9- and 20-fold increases in hlyB and <span class="gene">mdh</span> transcript levels. In addition, up-regulation of proteins at the expected size ranges of 50-60kDa for HlyB and 40-50kDa for MDH was observed. Two-dimensional polyacrylamide gel electrophoresis and matrix-assisted laser desorption/ionization time-of-flight/time-of-flight mass spectrometry revealed a <span class="trait">protein</span> spot corresponding to HlyB in HSF33-1 with a significant MOWSE score of 164 and 3% sequence coverage, and a spot corresponding to MDH in HSF33-2 gave a significant MOWSE score of 124 with 10% sequence coverage. Physiological evaluation in BG11/HEPES medium and seawater adjusted to 35gL-1 NaCl confirmed that the transformants could thrive in <span class="trait">high</span> salinity with no loss of photosynthetic pigments. Results of the study indicate that overexpression of hlyB and <span class="gene">mdh</span> genes confer halotolerance in F. diplosiphon, thus maximizing its potential as a large-scale biofuel agent.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gm1-MMP is involved in growth and development of leaf and seed, and enhances tolerance to <span class="trait">high</span> temperature and humidity stress in transgenic Arabidopsis.</h2>
			<h3>PMID:28483053  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu S(1), Liu Y(1), Jia Y(1), Wei J(1), Wang S(1), Liu X(1), Zhou Y(1), Zhu Y(1), Gu W(2), Ma H(3).
				<br/> 
				年份：2017 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Matrix metalloproteinases (MMPs) are a family of zinc- and calcium-dependent endopeptidases. <span class="gene">Gm1-MMP</span> was found to play an important role in soybean tissue remodeling during leaf expansion. In this study, <span class="gene">Gm1-MMP</span> was isolated and characterized. Its encoding <span class="trait">protein</span> had a relatively low phylogenetic relationship with the MMPs in other plant species. Subcellular localization indicated that <span class="gene">Gm1-MMP</span> was a plasma membrane <span class="trait">protein.</span> <span class="gene">Gm1-MMP</span> showed higher expression levels in mature leaves, old leaves, pods, and mature seeds, as well as was involved in the development of soybean seed. Additionally, it was involved in response to <span class="trait">high</span> temperature and humidity (HTH) stress in R7 leaves and seeds in soybean. The analysis of promoter of <span class="gene">Gm1-MMP</span> suggested that the fragment from -399 to -299 was essential for its promoter activity in response to HTH stress. The overexpression of <span class="gene">Gm1-MMP</span> in Arabidopsis affected the growth and development of leaves, enhanced leaf and developing seed tolerance to HTH stress and improved seed vitality. The levels of hydrogen peroxide (H2O2) and ROS in transgenic Arabidopsis seeds were lower than those in wild type seeds under HTH stress. <span class="gene">Gm1-MMP</span> could interact with soybean metallothionein-II <span class="gene">(GmMT-II),</span> which was confirmed by analysis of yeast two-hybrid assay and BiFC assays. All the results indicated that <span class="gene">Gm1-MMP</span> plays an important role in the growth and development of leaves and seeds as well as in tolerance to HTH stress. It will be helpful for us understanding the functions of <span class="gene">Gm1-MMP</span> in plant growth and development, and in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of aroma volatiles and understanding 2-acetyl-1-pyrroline biosynthetic mechanism in aromatic mung bean (Vigna radiata (L.) Wilczek).</h2>
			<h3>PMID:28461731</h3>
				

				
			<p>
				作者：Attar U(1), Hinge V(1), Zanan R(1), Adhav R(1), Nadaf A(1).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mung bean having <span class="trait">high</span> food value and easily digestible proteins, is one of the socioeconomically important crop of India. Among the varied cultivars, Sona mung is having aroma and hence popularly cultivated in the pockets of Ganga river basin at Bhutnir char village of Malda District in the West Bengal state. In the present study, aroma volatiles with special reference to 2-acetyl-1-pyrroline (2AP) were analyzed using HS-SPME-GCMS from Sona mung bean and compared with non-scented mung bean (PHULE M-9339). 26 volatiles in seeds of Sona mung and 20 in non-scented mung bean were identified, in which 3,7-dimethyl-6-octenal, (2E)-2-decen-1-ol, 2-ethyl-1-dodecanol and 3,5,5-trimethyl-2-cyclohexene-1-one are first time reported. 0.19 ± 0.001 ppm 2AP was recorded in Sona mung seeds whereas it was not detected in non-scented mung bean. PCA analysis indicated that 2AP, octanal, 1 pentanol, decanal, phenylmethanol and 2-nonen-1-ol were the major contributors in the aroma of Sona mung bean. The significantly higher level proline, methylglyoxal and lower level of <span class="gene">BADH2</span> transcript were detected in Sona mung than non-scented mung, suggesting similar 2AP biosynthesis mechanism in Sona mung bean as reported in scented rice, sorghum and soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Stearoyl-Acyl Carrier <span class="trait">Protein</span> Desaturase Mutations Uncover an Impact of Stearic Acid in Leaf and Nodule Structure.</h2>
			<h3>PMID:28461402  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lakhssassi N(1), Colantonio V(1)(2), Flowers ND(3), Zhou Z(1), Henry J(3), Liu S(1), Meksem K(4).
				<br/> 
				年份：2017 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Stearoyl-acyl carrier <span class="trait">protein</span> desaturase (SACPD-C) has been reported to control the accumulation of seed stearic acid; however, no study has previously reported its involvement in leaf stearic acid content and impact on leaf structure and morphology. A subset of an ethyl methanesulfonate mutagenized population of soybean (Glycine max) 'Forrest' was screened to identify mutants within the <span class="gene">GmSACPD-C</span> gene. Using a forward genetics approach, one nonsense and four missense <span class="gene">Gmsacpd-c</span> mutants were identified to have <span class="trait">high</span> levels of seed, nodule, and leaf stearic acid content. Homology modeling and in silico analysis of the <span class="gene">GmSACPD-C</span> enzyme revealed that most of these mutations were localized near or at conserved residues essential for diiron ion coordination. Soybeans carrying <span class="gene">Gmsacpd-c</span> mutations at conserved residues showed the highest stearic acid content, and these mutations were found to have deleterious effects on nodule development and function. Interestingly, mutations at nonconserved residues show an increase in stearic acid content yet retain healthy nodules. Thus, random mutagenesis and mutational analysis allows for the achievement of <span class="trait">high</span> seed stearic acid content with no associated negative agronomic characteristics. Additionally, expression analysis demonstrates that nodule leghemoglobin transcripts were significantly more abundant in soybeans with deleterious mutations at conserved residues of <span class="gene">GmSACPD-C.</span> Finally, we report that <span class="gene">Gmsacpd-c</span> mutations cause an increase in leaf stearic acid content and an alteration of leaf structure and morphology in addition to differences in nitrogen-fixing nodule structure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The essence of NAC gene family to the cultivation of drought-resistant soybean (Glycine max L. Merr.) cultivars.</h2>
			<h3>PMID:28241800  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hussain RM(1)(2), Ali M(3), Feng X(4), Li X(5).
				<br/> 
				年份：2017 Feb 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The NAC gene family is notable due to its large size, as well as its relevance in crop cultivation - particularly in terms of enhancing stress tolerance of plants. These plant-specific proteins contain NAC domain(s) that are named after Petunia NAM and Arabidopsis ATAF1/2 and CUC2 transcription factors based on the consensus sequence they have. Despite the knowledge available regarding NAC <span class="trait">protein</span> function, an extensive study on the possible use of <span class="gene">GmNACs</span> in developing soybean cultivars with superior <span class="trait">drought</span> tolerance is yet to be done. RESULTS: In response to this, our study was carried out, mainly through means of phylogenetic analysis (rice and Arabidopsis NAC genes served as seeding sequences). Through this, 139 <span class="gene">GmNAC</span> genes were identified and later grouped into 17 clusters. Furthermore, real-time quantitative PCR was carried out on drought-stressed and unstressed leaf tissues of both sensitive (B217 and H228) and tolerant (Jindou 74 and 78) cultivars. This was done to analyze the gene expression of 28 dehydration-responsive <span class="gene">GmNAC</span> genes. Upon completing the analysis, it was found that <span class="gene">GmNAC</span> gene expression is actually dependent on genotype. Eight of the 28 selected genes <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC021,</span> <span class="gene">GmNAC065,</span> <span class="gene">GmNAC066,</span> <span class="gene">GmNAC073,</span> <span class="gene">GmNAC082,</span> <span class="gene">GmNAC083</span> and <span class="gene">GmNAC087)</span> were discovered to have <span class="trait">high</span> expression levels in the drought-resistant soybean varieties tested. This holds true for both extreme and standard <span class="trait">drought</span> conditions. Alternatively, the drought-sensitive cultivars exhibited lower <span class="gene">GmNAC</span> expression levels in comparison to their tolerant counterparts. CONCLUSION: The study allowed for the identification of eight <span class="gene">GmNAC</span> genes that could be focused upon in future attempts to develop superior soybean varieties, particularly in terms of <span class="trait">drought</span> <span class="trait">resistance.</span> This study revealed that there were more dehydration-responsive <span class="gene">GmNAC</span> genes as <span class="gene">(GmNAC004,</span> <span class="gene">GmNAC005,</span> <span class="gene">GmNAC020</span> and <span class="gene">GmNAC021)</span> in addition to what were reported in earlier inquiries. It is important to note though, that discovering such notable genes is not the only goal of the study. It managed to put emphasis on the significance of further understanding the potential of soybean <span class="gene">GmNAC</span> genes, for the purpose of enhancing tolerance towards abiotic stress in general. This scientific inquiry has also revealed that cultivar genotypes tend to differ in their drought-induced gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Selection for a Zinc-Finger <span class="trait">Protein</span> Contributes to Seed Oil Increase during Soybean Domestication.</h2>
			<h3>PMID:28184009  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li QT(1)(2)(3), Lu X(1)(2)(3), Song QX(1)(2)(3), Chen HW(1)(2)(3), Wei W(1)(2)(3), Tao JJ(1)(2)(3), Bian XH(1)(2)(3), Shen M(1)(2)(3), Ma B(1)(2)(3), Zhang WK(1)(2)(3), Bi YD(1)(2)(3), Li W(1)(2)(3), Lai YC(1)(2)(3), Lam SM(1)(2)(3), Shui GH(1)(2)(3), Chen SY(4)(5)(6), Zhang JS(4)(5)(6).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed oil is a momentous agronomical trait of soybean (Glycine max) targeted by domestication in breeding. Although multiple oil-related genes have been uncovered, knowledge of the regulatory mechanism of seed oil biosynthesis is currently limited. We demonstrate that the seed-preferred gene <span class="gene">GmZF351,</span> encoding a tandem CCCH zinc finger <span class="trait">protein,</span> is selected during domestication. Further analysis shows that <span class="gene">GmZF351</span> facilitates oil accumulation by directly activating WRINKLED1, BIOTIN CARBOXYL CARRIER PROTEIN2, 3-KETOACYL-ACYL CARRIER <span class="trait">PROTEIN</span> SYNTHASE III, DIACYLGLYCEROL O-ACYLTRANSFERASE1, and OLEOSIN2 in transgenic Arabidopsis (Arabidopsis thaliana) seeds. Overexpression of <span class="gene">GmZF351</span> in transgenic soybean also activates lipid biosynthesis genes, thereby accelerating seed oil accumulation. The ZF351 haplotype from the cultivated soybean group and the wild soybean (Glycine soja) subgroup III correlates well with <span class="trait">high</span> gene expression level, seed oil contents and promoter activity, suggesting that selection of <span class="gene">GmZF351</span> expression leads to increased seed oil content in cultivated soybean. Our study provides novel insights into the regulatory mechanism for seed oil accumulation, and the manipulation of <span class="gene">GmZF351</span> may have great potential in the improvement of oil production in soybean and other related crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A 90-day toxicity study of <span class="gene">GmTMT</span> transgenic maize in Sprague-Dawley rats.</h2>
			<h3>PMID:28132846  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fang J(1), Feng Y(1), Zhi Y(1), Zhang L(2), Yu Z(3), Jia X(4).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmTMT</span> transgenic maize is a genetically modified maize plant that overexpresses the γ-tocopherol methyltransferase (γ-TMT) from Glycine max (Gm). The γ-TMT gene was introduced into maize line Zhen58 to encode the <span class="gene">GmTMT2a</span> <span class="trait">protein</span> which can convert γ-tocopherol into α-tocopherol. Overexpression of <span class="gene">GmTMT2a</span> significantly increased the α-tocopherol content in transgenic maize. The present study was designed to investigate any potential effects of <span class="gene">GmTMT</span> maize grain in a 90-day subchronic rodent feeding study. Maize grains from <span class="gene">GmTMT</span> or Zhen58 were incorporated into rodent diets at low (12.5%), medium (25%) or <span class="trait">high</span> (50%) concentrations and administered to Sprague-Dawley rats (n = 10/sex/group) for 90 days. The negative control group of rats (n = 10/sex/group) were fed with common maize diets. Results from body weights, feed consumption, clinical chemistry, hematology, absolute and relative organ weights indicated no treatment-related side effects of <span class="gene">GmTMT</span> maize grain on rats in comparison with rats consuming diets containing Zhen58 maize grain. In addition, no treatment-related changes were found in necropsy and histopathology examinations. Altogether, our data indicates that <span class="gene">GmTMT</span> transgenic maize is as safe and nutritious as its conventional non-transgenic maize.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Oxidative cleavage and hydrolytic boosting of cellulose in soybean spent flakes by Trichoderma reesei Cel61A lytic polysaccharide monooxygenase.</h2>
			<h3>PMID:28110665  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pierce BC(1), Agger JW(2), Wichmann J(3), Meyer AS(4).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The auxiliary activity family 9 (AA9) copper-dependent lytic polysaccharide monooxygenase (LPMO) from Trichoderma reesei (EG4; TrCel61A) was investigated for its ability to oxidize the complex polysaccharides from soybean. The substrate specificity of the enzyme was assessed against a variety of substrates, including both soy spent flake, a by-product of the soy food industry, and soy spent flake pretreated with sodium hydroxide. Products from enzymatic treatments were analyzed using mass spectrometry and <span class="trait">high</span> performance anion exchange chromatography. We demonstrate that TrCel61A is capable of oxidizing cellulose from both pretreated soy spent flake and phosphoric acid swollen cellulose, oxidizing at both the <span class="gene">C1</span> and <span class="gene">C4</span> positions. In addition, we show that the oxidative activity of TrCel61A displays a synergistic effect capable of boosting endoglucanase activity, and thereby substrate depolymerization of soy cellulose, by 27%.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome-wide transcript profiling to identify differentially expressed genes associated with seed field emergence in two soybean low phytate mutants.</h2>
			<h3>PMID:28100173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan F(1), Yu X(1), Dong D(1), Yang Q(1), Fu X(1), Zhu S(1), Zhu D(2).
				<br/> 
				年份：2017 Jan 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Seed germination is important to soybean (Glycine max) growth and development, ultimately affecting soybean <span class="trait">yield.</span> A lower seed field emergence has been the main hindrance for breeding soybeans low in phytate. Although this reduction could be overcome by additional breeding and selection, the mechanisms of seed germination in different low phytate mutants remain unknown. In this study, we performed a comparative transcript analysis of two low phytate soybean mutants (TW-1 and TW-1-M), which have the same mutation, a 2 bp deletion in <span class="gene">GmMIPS1,</span> but show a significant difference in seed field emergence, TW-1-M was higher than that of TW-1 . RESULTS: Numerous genes analyzed by RNA-Seq showed markedly different expression levels between TW-1-M and TW-1 mutants. Approximately 30,000-35,000 read-mapped genes and ~21000-25000 expressed genes were identified for each library. There were ~3900-9200 differentially expressed genes (DEGs) in each contrast library, the number of up-regulated genes was similar with down-regulated genes in the mutant TW-1and TW-1-M. Gene ontology functional categories of DEGs indicated that the ethylene-mediated signaling pathway, the abscisic acid-mediated signaling pathway, response to hormone, ethylene biosynthetic process, ethylene metabolic process, regulation of hormone levels, and oxidation-reduction process, regulation of flavonoid biosynthetic process and regulation of abscisic acid-activated signaling pathway had <span class="trait">high</span> correlations with seed germination. In total, 2457 DEGs involved in the above functional categories were identified. Twenty-two genes with 20 biological functions were the most highly up/down- regulated (absolute value Log2FC &gt;5) in the <span class="trait">high</span> field emergence mutant TW-1-M and were related to metabolic or signaling pathways. Fifty-seven genes with 36 biological functions had the greatest expression abundance (FRPM &gt;100) in germination-related pathways. CONCLUSIONS: Seed germination in the soybean low phytate mutants is a very complex process, which involves a series of physiological, morphological and transcriptional changes. Compared with TW-1, TW-1-M had a very different gene expression profile, which included genes related to plant hormones, antioxidation, anti-stress and energy metabolism processes. Our research provides a molecular basis for understanding germination mechanisms, and is also an important resource for the genetic analysis of germination in low phytate crops. Plant hormone- and antioxidation-related genes might strongly contribute to the <span class="trait">high</span> germination rate in the TW-1-M mutant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean cyclophilin <span class="gene">GmCYP1</span> interacts with an isoflavonoid regulator <span class="gene">GmMYB176</span></h2>
			<h3>PMID:28074922</h3>
				

				
					<h3>PMCID:5225424</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mainali </surname>  <given-names> Hemanta Raj </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vadivel </surname>  <given-names> Arun Kumaran Anguraj </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xuyan </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gijzen </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dhaubhadel </surname>  <given-names> Sangeeta </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Biology, University of Western Ontario </institution> , 1151 Richmond St, London, ON, N6A 5B7,  <country> Canada </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> London Research and Development Centre, Agriculture and Agri-Food Canada </institution> , 1391 Sandford Street, London, ON, N5V 4T3,  <country> Canada </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> College of Plant Science, Jilin University </institution> , Changchun, 130062,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cyclophilins (CYPs) belong to the immunophilin superfamily with peptidyl-prolyl cis-trans isomerase (PPIase) activity. They catalyze the interconversion of the cis- and trans-rotamers of the peptidyl-prolyl amide bond of peptides. A yeast-two-hybrid screening using the isoflavonoid regulator <span class="gene">GmMYB176</span> as bait identified <span class="gene">GmCYP1</span> as one of the interacting proteins in soybean embryos. <span class="gene">GmCYP1</span> localizes both in the nucleus and cytoplasm, and interacts in planta with <span class="gene">GmMYB176,</span> in the nucleus, and with SGF14l (a soybean 14-3-3 <span class="trait">protein)</span> in the nucleus and the cytoplasm. <span class="gene">GmCYP1</span> contains a single cyclophilin-like domain and displays a <span class="trait">high</span> sequence identity with other plant CYPs that are known to have stress-specific function. Tissue-specific expression of <span class="gene">GmCYP1</span> revealed higher expression in developing seeds compared to other vegetative tissues, suggesting their seed-specific role. Furthermore, <span class="gene">GmCYP1</span> transcript level was reduced in response to stress. Since isoflavonoids are involved in plant stress <span class="trait">resistance</span> against biotic and abiotic factors, the interaction of <span class="gene">GmCYP1</span> with the isoflavonoid regulators <span class="gene">GmMYB176</span> and 14-3-3 <span class="trait">protein</span> suggests its role in defense in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Cyclophilins (CYPs) belong to the immunophilin superfamily with peptidyl-prolyl <italic> cis-trans </italic> isomerase (PPIase) activity. They catalyze the interconversion of the <italic> cis- </italic> and <italic> trans- </italic> rotamers of the peptidyl-prolyl amide bond of peptides. A yeast-two-hybrid screening using the isoflavonoid regulator <span class="gene">GmMYB176</span> as bait identified <span class="gene">GmCYP1</span> as one of the interacting proteins in soybean embryos. <span class="gene">GmCYP1</span> localizes both in the nucleus and cytoplasm, and interacts <italic> in planta </italic> with <span class="gene">GmMYB176,</span> in the nucleus, and with SGF14l (a soybean 14-3-3 <span class="trait">protein)</span> in the nucleus and the cytoplasm. <span class="gene">GmCYP1</span> contains a single cyclophilin-like domain and displays a <span class="trait">high</span> sequence identity with other plant CYPs that are known to have stress-specific function. Tissue-specific expression of <italic> <span class="gene">GmCYP1</span> </italic> revealed higher expression in developing seeds compared to other vegetative tissues, suggesting their seed-specific role. Furthermore, <italic> <span class="gene">GmCYP1</span> </italic> transcript level was reduced in response to stress. Since isoflavonoids are involved in plant stress <span class="trait">resistance</span> against biotic and abiotic factors, the interaction of <span class="gene">GmCYP1</span> with the isoflavonoid regulators <span class="gene">GmMYB176</span> and 14-3-3 <span class="trait">protein</span> suggests its role in defense in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> ) is a grain legume belonging to the family <italic> Fabaceae </italic> . Soybean seeds provide a major supply of oil, <span class="trait">protein</span> and beneficial plant natural compounds such as isoflavonoids and saponins. The soybean genome contains 88,647 predicted transcripts and 56,044 <span class="trait">protein</span> coding loci located on 20 different chromosomes <xref ref-type="bibr" rid="b30"> 30 </xref> . Previously, we performed a genome-wide analysis of soybean <italic> CYPs </italic> and identified 62 <italic> CYP </italic> genes <xref ref-type="bibr" rid="b15"> 15 </xref> . Among these, <span class="gene">GmCYP1</span> has been shown to act as a &#x0201c;helper&#x0201d; to <italic> Phytophthora sojae </italic> RXLR effector Avr3b by activating its hydrolase activity in plant cells <xref ref-type="bibr" rid="b31"> 31 </xref> . The protein-protein interaction between <span class="gene">GmCYP1</span> and Avr3b was shown to be isoform-specific since <span class="gene">GmCYP1</span> paralogs failed to interact with Avr3b. Here we present a molecular characterization of <italic> <span class="gene">GmCYP1</span> </italic> covering its sequence analysis, phylogeny, temporal and spatial expression, subcellular localization, and provide the evidence for its possible role in isoflavonoid biosynthesis and stress response in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmCYP1</span> was identified in our Y2H screening as a <span class="trait">protein</span> that demonstrated protein-protein interaction with the isoflavonoid regulator <span class="gene">GmMYB176.</span> The Y2H assay was performed to identify <span class="gene">GmMYB176-interacting</span> proteins using <span class="gene">GmMYB176</span> as the bait <span class="trait">protein</span> and proteins from soybean embryos (50&#x02013;60 days after pollination) as prey. Of the several hundred yeast colonies screened, 6.5% contained a sequence corresponding to <italic> <span class="gene">GmCYP1</span> </italic> (accession #AF456323, locus <span class="gene">Glyma.11G098700).</span> <italic> <span class="gene">GmCYP1</span> </italic> is predicted to contain only one exon (519&#x02009;bp), and is located on the long arm (q arm) of chromosome 11, approximately 16&#x02009;Mb from the centromere. It encodes a single domain <span class="trait">protein</span> of 172 amino acid residues with a calculated molecular mass of 18.22&#x02009;kDa and a pI of 8.69. The cyclophilin-like domain in <span class="gene">GmCYP1</span> is predicted in between the amino acid residues 7 and 169.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to find sequences closely related to <span class="gene">GmCYP1,</span> a protein-protein BLAST (BLASTP) was performed using <span class="gene">GmCYP1</span> as a query against the NCBI non-redundant <span class="trait">protein</span> database. A list of 12 high-scoring and previously characterized CYPs is shown in <xref ref-type="supplementary-material" rid="S1"> Table S1 </xref> . Alignment of the deduced sequence of <span class="gene">GmCYP1</span> with previously characterised CYPs from several different plant species, human, yeast, and two <italic> Arabidopsis </italic> multi-domain CYPs (AtCYP40 and AtCYP63) revealed two general features ( <xref ref-type="fig" rid="f1"> Fig. 1 </xref> ). First, three amino acid residues that critically affect PPIase activity (R <sub> 55 </sub> , F <sub> 60 </sub> and H <sub> 126 </sub> ) <xref ref-type="bibr" rid="b32"> 32 </xref> are conserved in all CYPs aligned. Second, the tryptophan residue (W <sub> 121 </sub> ) implicated in substrate cyclosporinA binding <xref ref-type="bibr" rid="b32"> 32 </xref> <xref ref-type="bibr" rid="b33"> 33 </xref> is present in all of the CYPs studied except in the multi domain CYPs.
					</p>
				
					<p>&nbsp;&nbsp;
						A phylogenetic analysis of <span class="gene">GmCYP1</span> and other functionally characterized plant CYPs clustered <span class="gene">GmCYP1</span> close to <italic> GhCYP1, CcCYP1 </italic> and <italic> PvCYP </italic> from cotton, pigeon pea and common bean, respectively. Both <italic> GhCYP1 </italic> and <italic> CcCYP1 </italic> are known to have stress-specific function ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). Overexpression of <italic> GhCYP1 </italic> in tobacco conferred increased tolerance to biotic and abiotic stress <xref ref-type="bibr" rid="b29"> 29 </xref> . Similarly, <italic> Arabidopsis </italic> plants overexpressing <italic> CcCYP1 </italic> showed higher PPIase activity during stress and increased tolerance against multiple abiotic stresses as compared to control <xref ref-type="bibr" rid="b28"> 28 </xref> . Differential accumulation of <italic> PvCYP </italic> transcripts in response to various external stimuli suggested that it may possess a stress-related function <xref ref-type="bibr" rid="b24"> 24 </xref> . The <span class="trait">high</span> amino acid sequence identity of <span class="gene">GmCYP1</span> with GhCYP1 (96%), CcCYP1 (81%) and PvCYP (96%), suggests similar possible functions of <span class="gene">GmCYP1</span> in stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Molecules of size smaller than 20&#x02013;40&#x02009;kDa, such as ions, water, and small proteins, can pass through the nuclear pore complex by diffusion <xref ref-type="bibr" rid="b34"> 34 </xref> , whereas movement of larger molecules (70&#x02009;kDa or higher) entails an active transport system <xref ref-type="bibr" rid="b35"> 35 </xref> , mediated by transport receptors and signal peptides <xref ref-type="bibr" rid="b36"> 36 </xref> . The size of <span class="gene">GmCYP1-YFP</span> (45.22&#x02009;kDa) is not considerably larger than the size of molecules that can pass through the nuclear pore complex by diffusion. It is also possible that heterologous <span class="trait">protein</span> <span class="gene">GmCYP1-YFP,</span> expressed in <italic> N. benthamiana </italic> is cleaved by the endogenous host proteases and only the cleaved YFP fragments are localized to the nucleus. The confirmation of the YFP signal in the nucleus arising from the intact <span class="gene">GmCYP1-YFP</span> and not from the cleaved product of a fusion <span class="trait">protein</span> was performed by Western blot analysis ( <xref ref-type="fig" rid="f3"> Fig. 3b </xref> ). Therefore, it is not clear whether <span class="gene">GmCYP1-YFP</span> localization in the nucleus was due to passive diffusion or to active transport. Regardless, its nuclear localization indicates a possible role in the regulation of gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmCYP1</span> was identified as one of the interacting proteins of <span class="gene">GmMYB176</span> in the Y2H assay. <span class="trait">Protein</span> chaperones often bind with misfolded bait proteins when a <span class="trait">protein</span> is overexpressed or heterologously expressed. To confirm that <span class="gene">GmCYP1</span> is a true <span class="gene">GmMYB176-interacting</span> <span class="trait">protein,</span> we performed a targeted Y2H assay using <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176.</span> However, our targeted Y2H assay failed to verify the interaction between <span class="gene">GmCYP1</span> with <span class="gene">GmMYB176.</span> This result led us to hypothesize that there may be an indirect interaction between <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176</span> <italic> via </italic> involvement of <span class="trait">protein</span> (s) that may not be conserved between the species. Therefore, a BiFC assay <xref ref-type="bibr" rid="b37"> 37 </xref> was carried to further investigate <italic> in planta </italic> interaction between <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176.</span> Translational fusions of <span class="gene">GmMYB176</span> and <span class="gene">GmCYP1</span> were created with N-terminal (YN) or C-terminal (YC) halves of YFP <xref ref-type="bibr" rid="b38"> 38 </xref> and co-expressed in <italic> N. benthamiana </italic> leaf epidermal cells in the following combinations: (A) <span class="gene">GmCYP1-YN</span> and <span class="gene">GmMYB176-YC,</span> (B) <span class="gene">GmMYB176-YN</span> and <span class="gene">GmCYP1-YC.</span> The negative controls used for the experiment were co-expression of <span class="gene">GmCYP1-YN</span> and -YC only or <span class="gene">GmCYP1-YC</span> and -YN only. As shown in <xref ref-type="fig" rid="f4"> Fig. 4a </xref> , <span class="gene">GmCYP1</span> interacts with <span class="gene">GmMYB176</span> <italic> in planta </italic> , and the interaction between <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176</span> was strong in the nucleus. Similar results were obtained for the reciprocal combinations. No YFP signal was detected during co-expression of control constructs -YC or -YN with <span class="gene">GmCYP1-YN</span> or <span class="gene">GmCYP1-YC,</span> respectively, confirming that YFP signal was due to the protein-protein interaction <italic> in planta. </italic> Furthermore, we measured the strength of interaction by using FRET approach. It is a powerful tool for non-invasive monitoring of protein-protein interactions that involves transfer of energy between two closely positioned fluorophores <xref ref-type="bibr" rid="b39"> 39 </xref> , and has been widely used to determine the interactions between proteins in living plant cells <xref ref-type="bibr" rid="b40"> 40 </xref> <xref ref-type="bibr" rid="b41"> 41 </xref> <xref ref-type="bibr" rid="b42"> 42 </xref> . Translational fusions of <span class="gene">GmMYB176</span> with YFP and <span class="gene">GmCYP1</span> with CFP were co-expressed in <italic> N. benthamiana </italic> leaf epidermal cells and analysed for FRET efficiency between the reporters <italic> in vivo </italic> . The results revealed a FRET efficiency of 23.8%, indicating a close proximity of <span class="gene">GmMYB176</span> and <span class="gene">GmCYP1</span> and their co-existence in a complex ( <xref ref-type="fig" rid="f4"> Fig. 4b </xref> ). Empty vectors containing YFP only and CFP only were used as negative control which showed a FRET efficiency of 0.5%. A FRET value of 8&#x02013;10% was considered as the background level in previous studies <xref ref-type="bibr" rid="b43"> 43 </xref> <xref ref-type="bibr" rid="b44"> 44 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						When <italic> A. tumefaciens </italic> GV3101 strains carrying BiFC plasmids containing <span class="gene">GmCYP1-YN</span> or <span class="gene">GmCYP1-YC</span> were co-infiltrated into <italic> N. benthamiana </italic> leaves and visualized by confocal microscope, a strong yellow fluorescence was observed in the nucleus and relatively weaker fluorescence in the cytoplasm, suggesting that <span class="gene">GmCYP1</span> forms a homodimer in both cellular compartments <italic> in planta </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4a </xref> ). Even though recombinant hCYP-A has been reported to form monomers, dimers, and trimers when expressed in <italic> E. coli </italic> <xref ref-type="bibr" rid="b45"> 45 </xref> , there is no published literature on homo-dimerization of plant CYPs. Search for predicted motifs in <span class="gene">GmCYP1</span> identified a putative phosphorylation (pST binding) site within the <span class="gene">GmCYP1</span> sequence (97-ENFVKKH <bold> T </bold> GPGILSM-112), where T <sub> 105 </sub> is potentially phosphorylated. The pST binding motifs are binding sites for 14-3-3 family of proteins. 14-3-3 <span class="trait">protein</span> functions as a dimer to bind with its client proteins where each monomer in the dimer is capable of interacting with a separate client <span class="trait">protein.</span> The dimeric nature of 14-3-3 proteins allows them to serve as scaffolds by bringing two regions of the same proteins into proximity or two different proteins together <xref ref-type="bibr" rid="b46"> 46 </xref> . We have previously demonstrated that <span class="gene">GmMYB176</span> interacts with 14-3-3 proteins, thereby affecting its subcellular localization <xref ref-type="bibr" rid="b38"> 38 </xref> . The interactions between <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176</span> in the Y2H screen and in BiFC assay where soybean and <italic> N. benthamiana </italic> 14-3-3&#x02009;s possibly bring the two proteins together could be explained if <span class="gene">GmCYP1</span> is a true client of 14-3-3 <span class="trait">protein.</span> Therefore, we performed a BiFC assay between SGF14l (a soybean 14-3-3) and <span class="gene">GmCYP1.</span> Indeed, <span class="gene">GmCYP1</span> interacted with SGF14l <italic> in planta </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ), suggesting that 14-3-3 proteins may act as a scaffold to facilitate binding of <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176.</span> Despite that the mechanism and consequence of <span class="gene">GmCYP1</span> dimerization are not yet known, it is possible that binding of 14-3-3 with <span class="gene">GmCYP1</span> could bring two <span class="gene">GmCYP1</span> monomers together to produce fluorescence in the BiFC assay. Further, it is not clear whether the binding of <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176</span> mediated by SGF14l is involved in the <italic> CHS8 </italic> gene regulation, and subsequent isoflavonoid biosynthesis in soybean, or to some other, as-yet unknown function.
					</p>
				
					<p>&nbsp;&nbsp;
						Several studies have shown that pathogen effectors interact with plant helper proteins for their activation and proper function <xref ref-type="bibr" rid="b57"> 57 </xref> <xref ref-type="bibr" rid="b58"> 58 </xref> . Activated effector proteins bind with plant targets to suppress plant defenses and otherwise enable pathogen growth. For effectors encoded by <italic> Avr </italic> genes, activation can also result in detection by an immune receptor encoded by a <span class="trait">resistance</span> ( <italic> R </italic> ) gene. Thus, activation of Avr3b by <span class="gene">GmCYP1</span> triggers immunity in soybean cultivars containing Rps3b, whereas activation of Avr3b in soybean cultivars lacking Rps3b enables pathogen growth <xref ref-type="bibr" rid="b31"> 31 </xref> . Unlike many other plant CYPs that function in protecting plants during biotic and abiotic stress <xref ref-type="bibr" rid="b29"> 29 </xref> <xref ref-type="bibr" rid="b59"> 59 </xref> , <span class="gene">GmCYP1</span> acts as a susceptibility factor in soybean, at least in instances when Avr3b does not trigger immunity <xref ref-type="bibr" rid="b31"> 31 </xref> . The expression of susceptibility proteins is generally induced during infection in susceptible plants <xref ref-type="bibr" rid="b60"> 60 </xref> . However, our results show that <italic> <span class="gene">GmCYP1</span> </italic> transcripts are reduced in stressed plants compared to controls. This finding, together with the results that show <span class="gene">GmCYP1</span> interacts with <span class="gene">GmMYB176,</span> suggests a role for <span class="gene">GmCYP1</span> as a negative regulator of plant defense and isoflavonoid biosynthesis in soybean. The <span class="trait">high</span> expression of <span class="gene">GmCYP1</span> in the late stage developing embryos cannot be explained by this hypothesis because isoflavonoid biosynthesis is active in seed tissues, albeit this is seed isoflavonoid and not phytoalexin isoflavonoid biosynthesis.
					</p>
				
					<p>&nbsp;&nbsp;
						Overall, this study presents a detailed analysis of <italic> <span class="gene">GmCYP1</span> </italic> . The presence of predicted cyclophilin domain, subcellular localization and its sequence homology with other identified CYPs from other organisms provides insights into its putative function. The interaction of <span class="gene">GmCYP1</span> and <span class="gene">GmMYB176</span> is particularly intriguing because of the conditional functionality of <span class="gene">GmCYP1</span> in effector-triggered immunity or susceptibility to the pathogen <italic> P. sojae </italic> . It seems more than coincidence that the biosynthesis of isoflavonoid phytoalexins, being necessary for <span class="trait">resistance</span> to <italic> P. sojae </italic> , is also connected to <span class="gene">GmCYP1.</span> Since isoflavonoids are involved in plant stress <span class="trait">resistance</span> against biotic and abiotic factors <xref ref-type="bibr" rid="b61"> 61 </xref> <xref ref-type="bibr" rid="b62"> 62 </xref> <xref ref-type="bibr" rid="b63"> 63 </xref> , the interaction of <span class="gene">GmCYP1</span> with isoflavonoid regulators and its potential role as a suppressor of plant defense merits further investigation.
					</p>
				
					<p>&nbsp;&nbsp;
						The subcellular localization of <span class="gene">GmCYP1</span> was studied by infiltrating <italic> A. tumefaciens </italic> GV3101 carrying pEG101-GmCYP1 into <italic> N. benthamiana </italic> leaves, as described by Sparkes <italic> et al </italic> . <xref ref-type="bibr" rid="b67"> 67 </xref> . For co-expression, equal volumes of two construct-bearing strains, suspended in Gamborg&#x02019;s solution, were mixed together and then infiltrated into <italic> N. benthamiana </italic> leaf epidermal cells. The <span class="trait">protein</span> expression was visualized by confocal microscopy using a Leica TCS SP2 inverted confocal microscope. An excitation wavelength of 514&#x02009;nm was used for YFP imaging, and 525&#x02013;545&#x02009;nm emissions were collected. For visualization of CFP, an excitation wavelength of 458&#x02009;nm was used, and emissions were collected between 465&#x02013;495&#x02009;nm.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> A. tumefaciens </italic> GV3101 carrying the plasmids with <span class="gene">GmCYP1-YFP</span> and nuclear localizing CFP (NLS-CFP) constructs were co-infiltrated into <italic> N. benthamiana </italic> leaves and visualized by confocal microscopy. Expression of <bold> (a) </bold> <span class="gene">GmCYP1-YFP,</span> <bold> (b) </bold> NLS-CFP, <bold> (c) </bold> images of A and B merged to confirm nuclear localization of <span class="gene">GmCYP1.</span> Scale bars indicate 50&#x02009;&#x003bc;m. <bold> (d) </bold> Western blot analysis of translational fusion of <span class="gene">GmCYP1-YFP</span> proteins. <span class="gene">GmCYP1-YFP</span> was transiently expressed in <italic> N. benthamiana </italic> leaves for 1, 2 or 3 days, and <span class="trait">protein</span> accumulation was measured. Proteins (30&#x02009;&#x003bc;g) were separated on a SDS-PAGE and transferred to PVDF membrane by electroblotting. <span class="gene">GmCYP1-YFP</span> <span class="trait">protein</span> was detected by sequential incubation of the blot with anti-GFP antibody and anti-mouse IgG conjugated with HRP, followed by chemiluminescent reaction. eGFP is shown as a positive control.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soy Isoflavone Intake and Sleep Parameters over 5 Years among Chinese Adults: Longitudinal Analysis from the Jiangsu Nutrition Study.</h2>
			<h3>PMID:27956174  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cao Y, Taylor AW, Zhen S, Adams R, Appleton S, Shi Z.
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soy isoflavone is beneficial for menopausal/postmenopausal symptoms, including sleep complaints. However, little is known about its longitudinal association with sleep in the general population. OBJECTIVE: Our aim was to investigate the association between soy isoflavone intake and sleep duration and daytime falling asleep among Chinese adults. DESIGN: A longitudinal analysis was performed. Soy isoflavone intake was assessed by food frequency questionnaire. Sleep duration was self-reported at two time points. Occurrence of daytime falling asleep was determined at follow-up. Short and long sleep were defined as sleep &lt;7 h/day or ≥9 h/day, respectively. PARTICIPANTS/SETTING: Adults aged 20 years and older from the Jiangsu Nutrition Study (2002-2007) with complete isoflavone intake and sleep duration data at both time points (n=1,474) were analyzed (follow-up, n=1,492). MAIN OUTCOME MEASURES: We measured sleep duration in 2002 and 2007 and daytime falling asleep occurrence in 2007. STATISTICAL ANALYSES PERFORMED: Mixed-effects logistic regression was performed for repeated measures between isoflavone intake and sleep duration. Logistic regression was performed for daytime falling asleep at follow-up. Demographic, anthropometric, and social factors were adjusted in the analyses. RESULTS: The prevalence of long sleep duration was 18.9% in 2002 and 12.6% in 2007, and the prevalence of daytime falling asleep was 5.3%. Compared with the lowest quartile of isoflavone intake, the highest quartile was associated with a lower risk of long sleep duration (odds ratio=0.66; 95% <span class="gene">CI</span> 0.48 to 0.90; P for trend=0.018) over 5 years. Compared with persistent low intake of isoflavone (less than median intake of isoflavone at two time points), persistent <span class="trait">high</span> intake was associated with a reduced risk of daytime falling asleep in women (odds ratio=0.20; 95% <span class="gene">CI</span> 0.06 to 0.68), but not men. No consistent association between soy isoflavone intake and short sleep duration was found. CONCLUSIONS: Soy isoflavone intake was associated with a low risk of long sleep duration in both sexes and a low risk of daytime falling asleep in women but not men.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Candidate Gene Sequence Analyses toward Identifying -Type <span class="trait">Resistance</span> to.</h2>
			<h3>PMID:27898808</h3>
				

				
			<p>
				作者：Redekar NR, Clevinger EM, Laskar MA, Biyashev RM, Ashfield T, Jensen RV, Jeong SC, Tolin SA, Saghai Maroof MA.
				<br/> 
				年份：2016 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						is one of three genetic loci conferring strain-specific <span class="trait">resistance</span> to (SMV). The locus has been mapped to a 154-kb region on chromosome 14, containing a cluster of five nucleotide-binding leucine-rich repeat (NB-LRR) <span class="trait">resistance</span> genes. <span class="trait">High</span> sequence similarity between the candidate genes challenges fine mapping of the locus. Among the five, <span class="gene">Glyma14g38533</span> showed the highest transcript abundance in 1 to 3 h of SMV-G7 inoculation. Comparative sequence analyses were conducted with the five candidate NB-LRR genes from susceptible (-type) soybean [ (L.) Merr.] cultivar Williams 82, resistant (-type) cultivar Hwangkeum, and resistant lines L29 and RRR. Sequence comparisons revealed that <span class="gene">Glyma14g38533</span> had far more polymorphisms than the other candidate genes. Interestingly, <span class="gene">Glyma14g38533</span> gene from -type lines exhibited 150 single-nucleotide polymorphism (SNP and six insertion-deletion (InDel) markers relative to -type line, Furthermore, the polymorphisms identified in three -type lines were highly conserved. Several polymorphisms were validated in 18 -type resistant and six -type susceptible lines and were found associated with their disease response. The majority of the polymorphisms were located in LRR domain encoding region, which is involved in pathogen recognition via protein-protein interactions. These findings associating <span class="gene">Glyma14g38533</span> with -type <span class="trait">resistance</span> to SMV suggest it is the most likely candidate gene for .
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Integrating QTL mapping and transcriptomics identifies candidate genes underlying QTLs associated with soybean tolerance to low-phosphorus stress.</h2>
			<h3>PMID:27815671  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang D(1), Zhang H(2), Chu S(3), Li H(3), Chi Y(4), Triebwasser-Freese D(2), Lv H(3), Yu D(4).
				<br/> 
				年份：2017 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a <span class="trait">high</span> phosphorus (P) demand species that is sensitive to low-P stress. Although many quantitative trait loci (QTL) for P efficiency have been identified in soybean, but few of these have been cloned and agriculturally applied mainly due to various limitations on identifying suitable P efficiency candidate genes. Here, we combined QTL mapping, transcriptome profiling, and plant transformation to identify candidate genes underlying QTLs associated with low-P tolerance and response mechanisms to low-P stress in soybean. By performing QTL linkage mapping using 152 recombinant inbred lines (RILs) that were derived from a cross between a P-efficient variety, Nannong 94-156, and P-sensitive Bogao, we identified four major QTLs underlying P efficiency. Within these four QTL regions, 34/81 candidate genes in roots/leaves were identified using comparative transcriptome analysis between two transgressive RILs, low-P tolerant genotype B20 and sensitive B18. A total of 22 phosphatase family genes were up-regulated significantly under low-P condition in B20. Overexpression of an acid phosphatase candidate gene, <span class="gene">GmACP2,</span> in soybean hairy roots increased P efficiency by 15.43-24.54 % compared with that in controls. Our results suggest that integrating QTL mapping and transcriptome profiling could be useful for rapidly identifying candidate genes underlying complex traits, and phosphatase-encoding genes, such as <span class="gene">GmACP2,</span> play important roles involving in low-P stress tolerance in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Leader Intron of a Soybean Elongation Factor 1A (eEF1A) Gene Interacts with Proximal Promoter Elements to Regulate Gene Expression in Synthetic Promoters.</h2>
			<h3>PMID:27806110  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang N(1), McHale LK(2), Finer JJ(1).
				<br/> 
				年份：2016 Nov 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Introns, especially the first intron in the 5' untranslated region (5'UTR), can significantly impact gene expression via intron-mediated enhancement (IME). In this study, we demonstrate the leader intron of a soybean elongation factor 1A (eEF1A) gene <span class="gene">(GmScreamM8)</span> was essential for the <span class="trait">high</span> activity of the native promoter. Furthermore, the interaction of the <span class="gene">GmScreamM8</span> leader intron with regulatory element sequences from several soybean eEF1A promoters was studied using synthetic promoters, which consisted of element tetramers upstream of a core promoter used to regulate a green fluorescent <span class="trait">protein</span> (gfp) reporter gene. Element tetramers, placed upstream of a <span class="gene">GmScreamM8</span> core promoter, showed very <span class="trait">high</span> activity using both transient expression in lima bean cotyledons and stable expression in soybean hairy roots, only if the native leader intron was included, suggesting an interaction between intronic sequences and promoter elements. Partial deletions of the leader intron showed that a 222 bp intronic sequence significantly contributed to very <span class="trait">high</span> levels of GFP expression. Generation of synthetic intron variants with a monomeric or trimeric repeat of the 222 bp intronic sequence, yielded almost two-fold higher expression compared to the original intron, while partial deletion of the 222 bp intronic repeated sequence significantly decreased gene expression, indicating that this intronic sequence was essential for the intron-element interaction enhancement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An efficient method for measuring copy number variation applied to improvement of nematode <span class="trait">resistance</span> in soybean.</h2>
			<h3>PMID:27310152  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lee TG(1), Diers BW(1), Hudson ME(2).
				<br/> 
				年份：2016 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Copy number variation (CNV) is implicated in important traits in multiple crop plants, but can be challenging to genotype using conventional methods. The <span class="gene">Rhg1</span> locus of soybean, which confers <span class="trait">resistance</span> to soybean cyst nematode (SCN), is a CNV of multiple 31.2-kb genomic units each containing four genes. Reliable, high-throughput methods to quantify <span class="gene">Rhg1</span> and other CNVs for selective breeding were developed. The CNV genotyping assay described here uses a homeologous gene copy within the paleopolyploid soybean genome to provide the internal control for a single-tube TaqMan copy number assay. Using this assay, CNV in breeding populations can be tracked with <span class="trait">high</span> precision. We also show that extensive CNV exists within Fayette, a released, inbred SCN-resistant soybean cultivar with a <span class="trait">high</span> copy number at <span class="gene">Rhg1</span> derived from a single donor parent. Copy number at <span class="gene">Rhg1</span> is therefore unstable within a released variety over a relatively small number of generations. Using this assay to select for individuals with altered copy number, plants were obtained with both increased copy number and increased SCN <span class="trait">resistance</span> relative to control plants. Thus, CNV genotyping technologies can be used as a new type of marker-assisted selection to select for desirable traits in breeding populations, and to control for undesirable variation within cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Direct stacking of sequence-specific nuclease-induced mutations to produce <span class="trait">high</span> oleic and low linolenic soybean oil</h2>
			<h3>PMID:27733139</h3>
				

				
					<h3>PMCID:5062912</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Demorest </surname>  <given-names> Zachary L. </given-names>  </name>  <address>  <email> zdemorest@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Coffman </surname>  <given-names> Andrew </given-names>  </name>  <address>  <email> acoffman@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baltes </surname>  <given-names> Nicholas J. </given-names>  </name>  <address>  <email> nbaltes@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stoddard </surname>  <given-names> Thomas J. </given-names>  </name>  <address>  <email> tstoddard@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Clasen </surname>  <given-names> Benjamin M. </given-names>  </name>  <address>  <email> bclasen@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Luo </surname>  <given-names> Song </given-names>  </name>  <address>  <email> sluo@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Retterath </surname>  <given-names> Adam </given-names>  </name>  <address>  <email> aretterath@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yabandith </surname>  <given-names> Ann </given-names>  </name>  <address>  <email> ayabandith@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gamo </surname>  <given-names> Maria Elena </given-names>  </name>  <address>  <email> mgamo@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bissen </surname>  <given-names> Jeff </given-names>  </name>  <address>  <email> jbissen@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mathis </surname>  <given-names> Luc </given-names>  </name>  <address>  <email> lmathis@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Voytas </surname>  <given-names> Daniel F. </given-names>  </name>  <address>  <email> dvoytas@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhang </surname>  <given-names> Feng </given-names>  </name>  <address>  <email> fzhang@calyxt.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Calyxt, Inc., 600 County Road D West Suite 8, New Brighton, MN 55112 USA  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 9 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The ability to modulate levels of individual fatty acids within soybean oil has potential to increase shelf-life and frying stability and to improve nutritional characteristics. Commodity soybean oil contains <span class="trait">high</span> levels of polyunsaturated linoleic and linolenic acid, which contribute to oxidative instability - a problem that has been addressed through partial hydrogenation. However, partial hydrogenation increases levels of trans-fatty acids, which have been associated with cardiovascular disease. Previously, we generated soybean lines with knockout mutations within fatty acid desaturase 2-1A (FAD2-1A) and <span class="gene">FAD2-1B</span> genes, resulting in oil with increased levels of monounsaturated oleic acid (18:1) and decreased levels of linoleic (18:2) and linolenic acid (18:3). Here, we stack mutations within <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> with mutations in fatty acid desaturase 3A (FAD3A) to further decrease levels of linolenic acid. Mutations were introduced into <span class="gene">FAD3A</span> by directly delivering TALENs into fad2-1a fad2-1b soybean plants. RESULTS: Oil from fad2-1a fad2-1b fad3a plants had significantly lower levels of linolenic acid (2.5 %), as compared to fad2-1a fad2-1b plants (4.7 %). Furthermore, oil had significantly lower levels of linoleic acid (2.7 % compared to 5.1 %) and significantly higher levels of oleic acid (82.2 % compared to 77.5 %). Transgene-free fad2-1a fad2-1b fad3a soybean lines were identified. CONCLUSIONS: The methods presented here provide an efficient means for using sequence-specific nucleases to stack quality traits in soybean. The resulting product comprised oleic acid levels above 80 % and linoleic and linolenic acid levels below 3 %.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ability to modulate levels of individual fatty acids within soybean oil has potential to increase shelf-life and frying stability and to improve nutritional characteristics. Commodity soybean oil contains <span class="trait">high</span> levels of polyunsaturated linoleic and linolenic acid, which contribute to oxidative instability &#x02013; a problem that has been addressed through partial hydrogenation. However, partial hydrogenation increases levels of <italic> trans </italic> -fatty acids, which have been associated with cardiovascular disease. Previously, we generated soybean lines with knockout mutations within fatty acid desaturase 2-1A ( <italic> <span class="gene">FAD2-1A</span> </italic> ) and <italic> <span class="gene">FAD2-1B</span> </italic> genes, resulting in oil with increased levels of monounsaturated oleic acid (18:1) and decreased levels of linoleic (18:2) and linolenic acid (18:3). Here, we stack mutations within <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> with mutations in fatty acid desaturase 3A ( <italic> <span class="gene">FAD3A</span> </italic> ) to further decrease levels of linolenic acid. Mutations were introduced into <italic> <span class="gene">FAD3A</span> </italic> by directly delivering TALENs into <italic> fad2-1a fad2-1b </italic> soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Decreasing levels of linolenic acid is predicted to improve soybean oil characteristics by decreasing total levels of polyunsaturated fatty acids, and subsequently increasing frying and oxidative stability. Conversion of linoleic to linolenic acid is catalyzed by the fatty acid desaturase 3 (FAD3) enzyme, which is produced by a family of genes consisting of <italic> <span class="gene">FAD3A</span> </italic> <span class="gene">(Glyma14g37350),</span> <italic> <span class="gene">FAD3B</span> </italic> <span class="gene">(Glyma02g39230)</span> and <italic> <span class="gene">FAD3C</span> </italic> <span class="gene">(Glyma18g06950).</span> Consistent with its <span class="trait">high</span> expression in developing seeds, <italic> <span class="gene">FAD3A</span> </italic> has the greatest effect on linolenic acid concentrations in soybean oil [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Combining mutations within <italic> <span class="gene">FAD3A</span> </italic> with <italic> <span class="gene">FAD3B</span> </italic> and/or <italic> <span class="gene">FAD3C</span> </italic> resulted in oil having &#x0003c;3&#x000a0;% linolenic acid [ <xref ref-type="bibr" rid="CR16"> 16 </xref> &#x02013; <xref ref-type="bibr" rid="CR20"> 20 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						With the advent of sequence-specific nucleases, including TALENs and CRISPR/Cas, it has become possible to introduce targeted knockout mutations within genes of interest [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. When delivered to plant cells, sequence-specific nucleases generate targeted DNA double-strand breaks. These double-strand breaks are then repaired predominantly by non-homologous end joining (NHEJ), which may result in the introduction of small insertions or deletions at the repair site. If double-strand breaks are generated within gene coding sequences, imprecise repair by NHEJ has potential to introduce frameshift mutations or in-frame deletions that destroy <span class="trait">protein</span> function. The objective of this study was to create <span class="trait">high</span> oleic and low linolenic soybean lines by stacking targeted mutations within <italic> <span class="gene">FAD2-1A</span> </italic> , <italic> <span class="gene">FAD2-1B</span> </italic> and <italic> <span class="gene">FAD3</span> </italic> genes. With the current industrial standard for low linolenic acid soybean oil being 3&#x000a0;% [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ], we sought to inactivate a sufficient number of <italic> <span class="gene">FAD3</span> </italic> genes to achieve linolenic levels &#x0003c;3&#x000a0;%.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean oil is primarily composed of palmitic acid, stearic acid, oleic acid, linoleic acid and linolenic acid. In oil from wild type plants, these five fatty acids are present at approximately 13, 4, 20, 55 and 8&#x000a0;%, respectively (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ). Previously, we used TALENs to generate knockout mutations within both <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Oil from the resulting plants contained higher levels of oleic acid (~79&#x000a0;%) and lower levels of linoleic and linolenic acids (~5&#x000a0;% each), compared to oil from WT plants. Here, we sought to further improve oil characteristics by knocking out genes involved in the conversion of linoleic to linolenic acid. We predicted that by knocking out the <italic> <span class="gene">FAD3</span> </italic> linoleate desaturase genes, levels of linolenic acid would further decrease. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> Design of TALENs targeting <italic> <span class="gene">FAD3</span> </italic> genes within <italic> Glycine max </italic> . <bold> a </bold> Illustration of the fatty acid pathway. Relative percent composition of individual fatty acids in the oil from WT and <italic> fad2-1 </italic> knockout plants is shown on the right. <bold> b </bold> Schematic of the <italic> <span class="gene">FAD3A</span> </italic> genomic sequence. Triangles, approximate TALEN binding sites; <italic> black boxes </italic> , exons; <italic> gray boxes </italic> , 5&#x02019; and 3&#x02019; untranslated regions. <bold> c </bold> Nucleotide sequences of the predicted TALEN target sites within the <italic> <span class="gene">FAD3A</span> </italic> , <italic> <span class="gene">FAD3B</span> </italic> , and <italic> <span class="gene">FAD3A</span> </italic> genes. Bold and underlined nucleotides indicate TALEN binding sequence. Lower case nucleotides indicate positions of SNPs. <bold> d </bold> Illustration of a TALEN monomer expression vector. P <sub> NOS </sub> , nopaline synthase promoter; SV40 NLS, simian virus 40 large T-antigen nuclear localization signal; T <sub> NOS </sub> , nopaline synthase terminator; AmpR, ampicillin <span class="trait">resistance</span> gene
					</p>
				
					<p>&nbsp;&nbsp;
						In 2015, the FDA ruled that <italic> trans </italic> fat is no longer &#x02018;generally recognized as safe&#x02019; for use in food, and has set a 3&#x000a0;year deadline to remove partially hydrogenated oils from food products. In an effort to improve shelf life and cooking characteristics, soybean oil is partially hydrogenated. However, partial hydrogenation results in increased levels of <italic> trans </italic> fats. Generating soybean oil with lower levels of polyunsaturated fatty acids promises to enhance shelf life and heat stability, thereby reducing the need for hydrogenation. Previously, we generated soybean that produce <span class="trait">high</span> oleic acid oil by knocking out both <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes <italic> . </italic> Here, we further improved oil characteristics by decreasing polyunsaturated fatty acids (linoleic and linolenic) to levels below 3&#x000a0;%. The methods and products presented here provide solutions for the demand of soybean oil with increased oxidative stability.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean plants with <span class="trait">high</span> oleic acid have been developed by research groups using different approaches. These approaches include RNAi [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ], combining naturally occurring or induced mutations [ <xref ref-type="bibr" rid="CR13"> 13 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> ], and site-directed mutagenesis using sequence-specific nucleases [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Of the characterized <span class="trait">high</span> oleic soybean lines, most have distinct differences in the levels of individual fatty acids. For example, Monsanto&#x02019;s Vistive&#x000ae; Gold <span class="trait">high</span> oleic acid soybean oil (MON 87705), created using RNAi targeting <italic> FATB </italic> and <italic> <span class="gene">FAD2-1</span> </italic> and harboring naturally occurring mutations in <italic> <span class="gene">FAD3</span> </italic> genes, has oleic, linoleic, linolenic and saturated fatty acids levels of 71.7, 16.9, 2.9 and 6.8&#x000a0;%, respectively [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. Further, Pioneer/DuPont&#x02019;s Plenish&#x000ae; <span class="trait">high</span> oleic soybean oil (HOSO 305423), created using RNAi against <italic> <span class="gene">FAD2-1</span> </italic> , has oleic, linoleic, linolenic and saturated fatty acids levels of 70.6, 5.5, 7.2 and 14.3&#x000a0;% respectively [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. The <italic> fad2-1a fad2-1b fad3a </italic> lines created here have oleic, linoleic, linolenic and saturated fatty acids levels of 82.2, 2.7, 2.5, and 11.3&#x000a0;%, respectively. Variations in fatty acid levels may be due to differences in gene targets (e.g., <italic> FATB </italic> vs <italic> <span class="gene">FAD3</span> </italic> ), genetic background, growth conditions, and possibly incomplete silencing of gene expression when using RNAi technology.
					</p>
				
					<p>&nbsp;&nbsp;
						We observed lower levels of linoleic acid and higher levels of oleic acid within oil from <italic> fad2-1a fad2-1b fad3a </italic> plants, when compared to oil from <italic> fad2-1a fad2-1b </italic> plants. It would be expected that knockout mutations within desaturase genes would result in accumulation of the corresponding substrate. Indeed, this is the case for plants containing mutations in either <italic> <span class="gene">FAD3A</span> </italic> or <italic> <span class="gene">FAD2-1A</span> <span class="gene">FAD2-1B</span> </italic> ; mutations in <italic> <span class="gene">FAD3A</span> </italic> resulted in increased levels of linoleic acid, and mutations in <italic> <span class="gene">FAD2-1A</span> <span class="gene">FAD2-1B</span> </italic> resulted in increased levels of oleic acid. When we introduced <italic> <span class="gene">FAD3A</span> </italic> mutations within <italic> fad2-1a fad2-1b </italic> soybeans, the level of linoleic acid decreased and the levels of oleic acid increased. This trend was also observed in <span class="trait">high</span> oleic and low linolenic soybean plants generated after combining different sources of mutant <italic> FAD2-1A, <span class="gene">FAD2-1B</span> </italic> and <italic> <span class="gene">FAD3A</span> </italic> genes [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]; however, fatty acids levels were significantly affected by environmental conditions. Further, and unexpectedly, we observed that oil within <italic> fad3a </italic> and <italic> fad2-1a fad2-1b </italic> plants had deceased levels of the two fatty acids immediately preceding the substrate of the inactivated desaturase (i.e., palmitic and stearic acid in <italic> fad2-1a fad2-1b </italic> plants, or stearic and oleic acid in <italic> fad3a </italic> plants). Understanding properties of additional soybean desaturase proteins and the effects of genetic background and environmental conditions may provide a better understanding of the lipid biosynthetic pathway in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Coding sequences for the TALEN pairs used in this study <span class="gene">(GmFAD3_T01.1,</span> <span class="gene">GmFAD3_T02.1,</span> and <span class="gene">GmFAD3_T03.1)</span> were synthesized as previously described [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. Individual TALEN monomers were cloned into protoplast expression vectors harboring a nopaline synthase (NOS) promoter and terminator. TALEN backbone architecture comprised N-terminal truncations (N152: TAAAKFERQHMDSIDIADLRTLGYSQQQQEKIKPKVRSTVAQHHEALVGHGFTHAHIVALSQHPAALGTVAVKYQDMIAALPEATHEAIVGVGKQWSGARALEALLTVAGELRGPPLQLDTGQLLKIAKRGGVTAVEAVHAWRNALTGAPLN) and C-terminal truncations (C40: SIVAQLSRPDPALAALTNDHLVALACLGGRPALDAVKKGL). Each TALEN monomer comprised 15 repeat domains for targeting 15 nucleotides of <italic> <span class="gene">FAD3</span> </italic> sequence, as shown in Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1c </xref> . Repeat variable diresidues within the TALE repeats included NI (for targeting adenine), HD (for targeting cytosine), NN (for targeting guanine), and NG (for targeting thymine). To facilitate trafficking to plant cell nuclei, an SV40 NLS (PKKKRKV) was added to the N-terminus of the TALEN <span class="trait">protein.</span> The size of plasmids encoding TALE monomers was 6151&#x000a0;bp. Plasmids were isolated from bacteria using the QIAGEN&#x000ae; maxiprep kit.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:NtPHYB1K326, a homologous gene of Arabidopsis PHYB, positively regulates the content of phenolic compounds in tobacco.</h2>
			<h3>PMID:27636822  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao J(1), Han J(2), Zhang J(3), Li Z(4), Yu J(5), Yu S(6), Guo Y(7), Fu Y(8), Zhang X(9).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polyphenols are important secondary metabolites and bioactive compounds in plants. Light is a vital abiotic factor that greatly impacts the content of polyphenols in plants. In spite of their importance the mechanism of polyphenol regulation still remains unknown in tobacco. A phytochrome B homolog, NtPHYB1K326, was isolated from Nicotiana tabacum cv. K326 to investigate the role of light receptors in the regulation of polyphenol metabolism in tobacco leaves. Furthermore, role of NtPHYB1K326 in polyphenol metabolism was analyzed by over-expression and RNAi-silencing approaches. Consistent and complemented results indicated involvement of NtPHYB1K326 in the regulation of polyphenol metabolism in tobacco leaves. Moreover, <span class="trait">high</span> levels of NtPHYB1K326 transcripts favor the accumulation of chlorogenic acid and its isomers, the key polyphenol component in tobacco leaves. Transcriptome analysis was also carried out for exploring the regulation mechanism of NtPHYB1K326 in the polyphenol metabolism. Compared with WT, 1665 and 1421 differentially-expressed genes were found in NtPHYB1K326-GFP and NtPHYB1K326-RNAi transgenic lines, respectively. Among these, about 30 genes were related to phenylpropanoid pathway, which is predominantly involved in synthesis of polyphenols. Further evidences from quantitative RT-PCR confirmed that NtPHYB1K326 may control phenylpropanoid pathway through regulating the transcription of PAL4 (phenylalanine ammonialyase 4), <span class="gene">4CL1</span> (4-coumarate:coenzyme A ligase 1) and COMT (caffeic acid 3-O-methyltransferase) genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Genes Encoding Methionine-Rich Proteins in Arabidopsis and Soybean Suggesting Their Roles in the Adaptation of Plants to Abiotic Stress.</h2>
			<h3>PMID:27635394</h3>
				

				
			<p>
				作者：Chu HD(1), Le QN(2), Nguyen HQ(3), Le DT(1).
				<br/> 
				年份：2016
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Oxidation and reduction of methionine (Met) play important roles in scavenging reactive oxygen species (ROS) and signaling in living organisms. To understand the impacts of Met oxidation and reduction in plants during stress, we surveyed the genomes of Arabidopsis and soybean (Glycine max L.) for genes encoding Met-rich proteins (MRPs). We found 121 and 213 genes encoding MRPs in Arabidopsis and soybean, respectively. Gene annotation indicated that those with known function are involved in vital cellular processes such as transcriptional control, calcium signaling, <span class="trait">protein</span> modification, and metal transport. Next, we analyzed the transcript levels of MRP-coding genes under normal and stress conditions. We found that 57 AtMRPs were responsive either to <span class="trait">drought</span> or to <span class="trait">high</span> salinity stress in Arabidopsis; 35 <span class="gene">GmMRPs</span> were responsive to <span class="trait">drought</span> in the leaf of late vegetative or early reproductive stages of soybean. Among the MRP genes with a known function, the majority of the abiotic stress-responsive genes are involved in transcription control and calcium signaling. Finally, Arabidopsis plant which overexpressed an MRP-coding gene, whose transcripts were downregulated by abiotic stress, was more sensitive to paraquat than the control. Taken together, our report indicates that MRPs participate in various vital processes of plants under normal and stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmCYP82A3, a Soybean Cytochrome <span class="gene">P450</span> Family Gene Involved in the Jasmonic Acid and Ethylene Signaling Pathway, Enhances Plant <span class="trait">Resistance</span> to Biotic and Abiotic Stresses.</h2>
			<h3>PMID:27588421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Q(1), Cui X(2), Lin S(1), Gan S(2), Xing H(2), Dou D(1).
				<br/> 
				年份：2016 Sep 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cytochrome <span class="gene">P450</span> monooxygenases (P450s) represent a large and important enzyme superfamily in plants. They catalyze numerous monooxygenation/hydroxylation reactions in biochemical pathways, P450s are involved in a variety of metabolic pathways and participate in the homeostasis of phytohormones. The CYP82 family genes specifically reside in dicots and are usually induced by distinct environmental stresses. However, their functions are largely unknown, especially in soybean (Glycine max L.). Here, we report the function of <span class="gene">GmCYP82A3,</span> a gene from soybean CYP82 family. Its expression was induced by Phytophthora sojae infection, salinity and <span class="trait">drought</span> stresses, and treatment with methyl jasmonate (MeJA) or ethephon (ETH). Its expression levels were consistently <span class="trait">high</span> in resistant cultivars. Transgenic Nicotiana benthamiana plants overexpressing <span class="gene">GmCYP82A3</span> exhibited strong <span class="trait">resistance</span> to Botrytis cinerea and Phytophthora parasitica, and enhanced tolerance to salinity and <span class="trait">drought</span> stresses. Furthermore, transgenic plants were less sensitive to jasmonic acid (JA), and the enhanced <span class="trait">resistance</span> was accompanied with increased expression of the JA/ET signaling pathway-related genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association of Effector <span class="gene">Six6</span> with Vascular Wilt Symptoms Caused by Fusarium oxysporum on Soybean.</h2>
			<h3>PMID:27349740  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lanubile A(1), Ellis ML(1), Marocco A(1), Munkvold GP(1).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Fusarium oxysporum species complex (FOSC) is a widely distributed group of fungi that includes both pathogenic and nonpathogenic isolates. In a previous study, isolates within the FOSC collected primarily from soybean were assessed for the presence of 12 fungal effector genes. Although none of the assayed genes was significantly associated with wilt symptoms on soybean, the secreted in xylem 6 (Six6) gene was present only in three isolates, which all produced <span class="trait">high</span> levels of vascular wilt on soybean. In the current study, a collection of F. oxysporum isolates from soybean roots and F. oxysporum f. sp. phaseoli isolates from common bean was screened for the presence of the <span class="gene">Six6</span> gene. Interestingly, all isolates for which the <span class="gene">Six6</span> amplicon was generated caused wilt symptoms on soybean, and two-thirds of the isolates showed <span class="trait">high</span> levels of aggressiveness, indicating a positive association between the presence of the effector gene <span class="gene">Six6</span> and induction of wilt symptoms. The expression profile of the <span class="gene">Six6</span> gene analyzed by quantitative reverse-transcription polymerase chain reaction revealed an enhanced expression for the isolates that caused more severe wilt symptoms on soybean, as established by the greenhouse assay. These findings suggest the suitability of the <span class="gene">Six6</span> gene as a possible locus for pathogenicity-based molecular diagnostics across the various formae speciales.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional Characterization of Soybean <italic> <span class="gene">Glyma04g39610</span> </italic> as a Brassinosteroid Receptor Gene and Evolutionary Analysis of Soybean Brassinosteroid Receptors</h2>
			<h3>PMID:27338344</h3>
				

				
					<h3>PMCID:4926431</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Suna </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tao </surname>  <given-names> Ping </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Aiping </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huo </surname>  <given-names> Weige </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jinxiang </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00897"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00897"> 2 </xref>  <xref rid="c1-ijms-17-00897" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BR) play important roles in plant growth and development. Although BR receptors have been intensively studied in Arabidopsis, the BR receptors in soybean remain largely unknown. Here, in addition to the known receptor gene <span class="gene">Glyma06g15270</span> <span class="gene">(GmBRI1a),</span> we identified five putative BR receptor genes in the soybean genome: <span class="gene">GmBRI1b,</span> <span class="gene">GmBRL1a,</span> <span class="gene">GmBRL1b,</span> <span class="gene">GmBRL2a,</span> and <span class="gene">GmBRL2b.</span> Analysis of their expression patterns by quantitative real-time PCR showed that they are ubiquitously expressed in primary roots, lateral roots, stems, leaves, and hypocotyls. We used rapid amplification of cDNA ends (RACE) to clone <span class="gene">GmBRI1b</span> <span class="gene">(Glyma04g39160),</span> and found that the predicted amino acid sequence of <span class="gene">GmBRI1b</span> showed <span class="trait">high</span> similarity to those of AtBRI1 and pea PsBRI1. Structural modeling of the ectodomain also demonstrated similarities between the BR receptors of soybean and Arabidopsis. GFP-fusion experiments verified that <span class="gene">GmBRI1b</span> localizes to the cell membrane. We also explored <span class="gene">GmBRI1b</span> function in Arabidopsis through complementation experiments. Ectopic over-expression of <span class="gene">GmBRI1b</span> in Arabidopsis BR receptor loss-of-function mutant (bri1-5 bak1-1D) restored hypocotyl growth in etiolated seedlings; increased the growth of stems, leaves, and siliques in light; and rescued the developmental defects in leaves of the bri1-6 mutant, and complemented the responses of BR biosynthesis-related genes in the bri1-5 bak1-D mutant grown in light. Bioinformatics analysis demonstrated that the six BR receptor genes in soybean resulted from three gene duplication events during evolution. Phylogenetic analysis classified the BR receptors in dicots and monocots into three subclades. Estimation of the synonymous (Ks) and the nonsynonymous substitution rate (Ka) and selection pressure (Ka/Ks) revealed that the Ka/Ks of BR receptor genes from dicots and monocots were less than 1.0, indicating that BR receptor genes in plants experienced purifying selection during evolution.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BR) play important roles in plant growth and development. Although BR receptors have been intensively studied in <italic> Arabidopsis </italic> , the BR receptors in soybean remain largely unknown. Here, in addition to the known receptor gene <italic> <span class="gene">Glyma06g15270</span> </italic> ( <italic> <span class="gene">GmBRI1a</span> </italic> ), we identified five putative BR receptor genes in the soybean genome: <italic> <span class="gene">GmBRI1b</span> </italic> , <italic> <span class="gene">GmBRL1a</span> </italic> , <italic> <span class="gene">GmBRL1b</span> </italic> , <italic> <span class="gene">GmBRL2a</span> </italic> , and <italic> <span class="gene">GmBRL2b</span> </italic> . Analysis of their expression patterns by quantitative real-time PCR showed that they are ubiquitously expressed in primary roots, lateral roots, stems, leaves, and hypocotyls. We used rapid amplification of cDNA ends (RACE) to clone <italic> <span class="gene">GmBRI1b</span> </italic> ( <italic> <span class="gene">Glyma04g39160</span> </italic> ), and found that the predicted amino acid sequence of <span class="gene">GmBRI1b</span> showed <span class="trait">high</span> similarity to those of AtBRI1 and pea PsBRI1. Structural modeling of the ectodomain also demonstrated similarities between the BR receptors of soybean and <italic> Arabidopsis </italic> . GFP-fusion experiments verified that <span class="gene">GmBRI1b</span> localizes to the cell membrane. We also explored <italic> <span class="gene">GmBRI1b</span> </italic> function in <italic> Arabidopsis </italic> through complementation experiments. Ectopic over-expression of <italic> <span class="gene">GmBRI1b</span> </italic> in <italic> Arabidopsis </italic> BR receptor loss-of-function mutant ( <italic> bri1-5 bak1-1D </italic> ) restored hypocotyl growth in etiolated seedlings; increased the growth of stems, leaves, and siliques in light; and rescued the developmental defects in leaves of the <italic> bri1-6 </italic> mutant, and complemented the responses of BR biosynthesis-related genes in the <italic> bri1-5 bak1-D </italic> mutant grown in light. Bioinformatics analysis demonstrated that the six BR receptor genes in soybean resulted from three gene duplication events during evolution. Phylogenetic analysis classified the BR receptors in dicots and monocots into three subclades. Estimation of the synonymous ( <italic> K </italic> <sub> s </sub> ) and the nonsynonymous substitution rate ( <italic> K </italic> <sub> a </sub> ) and selection pressure ( <italic> K </italic> <sub> a </sub> / <italic> K </italic> <sub> s </sub> ) revealed that the <italic> K </italic> <sub> a </sub> / <italic> K </italic> <sub> s </sub> of BR receptor genes from dicots and monocots were less than 1.0, indicating that BR receptor genes in plants experienced purifying selection during evolution.
					</p>
				
					<p>&nbsp;&nbsp;
						Work in <italic> Arabidopsis </italic> and rice has revealed the BR signaling pathway. BR signals are detected by BR receptors such as BRASSINOSTEROID INSENSITIVE 1 (BRI1) in the cell membrane [ <xref rid="B15-ijms-17-00897" ref-type="bibr"> 15 </xref> , <xref rid="B16-ijms-17-00897" ref-type="bibr"> 16 </xref> ]. In the absence of BRs, BRI1 is bound by the membrane-localized BRI1 KINASE INHIBITOR 1 (BKI1) [ <xref rid="B17-ijms-17-00897" ref-type="bibr"> 17 </xref> ]. Upon perception of BR, BRI1 disassociates from BKI1 [ <xref rid="B17-ijms-17-00897" ref-type="bibr"> 17 </xref> ] and interacts with BRI1-ASSOCIATED RECEPTOR KINASE 1 (BAK1), a membrane kinase and co-receptor of BRI1 [ <xref rid="B18-ijms-17-00897" ref-type="bibr"> 18 </xref> ]. BRI1 and BAK1 <italic> trans </italic> -phosphorylate each other [ <xref rid="B17-ijms-17-00897" ref-type="bibr"> 17 </xref> ]. BRASSINOSTEROID INSENSITIVE 2 (BIN2), the downstream regulator of BRI1, is a highly conserved <span class="gene">GSK</span> kinase and a negative regulator of BR signaling; BIN2 phosphorylates the transcription factors BRASSINAZOLE-RESISTANT 1 (BZR1) and BZR2 thus inactivating them [ <xref rid="B19-ijms-17-00897" ref-type="bibr"> 19 </xref> , <xref rid="B20-ijms-17-00897" ref-type="bibr"> 20 </xref> , <xref rid="B21-ijms-17-00897" ref-type="bibr"> 21 </xref> , <xref rid="B22-ijms-17-00897" ref-type="bibr"> 22 </xref> ]. In contrast, <span class="trait">PROTEIN</span> PHOSPHATASE 2A (PP2A) mediates the dephosphorylation, and thus activation of BZR1 [ <xref rid="B23-ijms-17-00897" ref-type="bibr"> 23 </xref> ]. <span class="trait">High</span> levels of BR in plants leads to the inactivation of BIN2. The dephosphorylated BZR1 and BZR2 shuttle from the cytoplasm to the nucleus and bind to the promoters of numerous downstream genes, thus strengthening BR signaling [ <xref rid="B24-ijms-17-00897" ref-type="bibr"> 24 </xref> ]. BIN2 and BZR1 are regulated by proteasome-dependent pathways in <italic> Arabidopsis </italic> [ <xref rid="B21-ijms-17-00897" ref-type="bibr"> 21 </xref> , <xref rid="B25-ijms-17-00897" ref-type="bibr"> 25 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Identifying BR receptors in other plants and deciphering their functions provides an important initial step toward deciphering BR signaling networks and understanding their evolution. <italic> Arabidopsis </italic> has three functional BR receptors, BRI1, BRL1, and BRL3. AtBRL2 appears to be non-functional in BR signaling [ <xref rid="B29-ijms-17-00897" ref-type="bibr"> 29 </xref> , <xref rid="B30-ijms-17-00897" ref-type="bibr"> 30 </xref> ]. The rice genome contains four BR receptor genes, <italic> OsBRI1 </italic> , <italic> OsBRL1 </italic> , <italic> OsBRL2 </italic> , and <italic> OsBRL3 </italic> [ <xref rid="B31-ijms-17-00897" ref-type="bibr"> 31 </xref> , <xref rid="B32-ijms-17-00897" ref-type="bibr"> 32 </xref> ]. BR receptors have also been identified in tomato [ <xref rid="B33-ijms-17-00897" ref-type="bibr"> 33 </xref> ], pea [ <xref rid="B34-ijms-17-00897" ref-type="bibr"> 34 </xref> ], barley [ <xref rid="B35-ijms-17-00897" ref-type="bibr"> 35 </xref> ], cotton [ <xref rid="B36-ijms-17-00897" ref-type="bibr"> 36 </xref> ], maize [ <xref rid="B37-ijms-17-00897" ref-type="bibr"> 37 </xref> ], and wheat [ <xref rid="B38-ijms-17-00897" ref-type="bibr"> 38 </xref> ]. Although the BR signaling pathway has been well studied in <italic> Arabidopsis </italic> and rice, it is not well understood in soybean. Recent work reported that soybean <italic> <span class="gene">Glyma06g12570</span> </italic> encodes a functional BR receptor [ <xref rid="B39-ijms-17-00897" ref-type="bibr"> 39 </xref> ]. Considering the <span class="trait">high</span> levels of duplication in the soybean genome [ <xref rid="B40-ijms-17-00897" ref-type="bibr"> 40 </xref> ], we postulated that soybean may have other functional BR receptors.
					</p>
				
					<p>&nbsp;&nbsp;
						To clone soybean brassinosteroid receptors, we used the AtBRI1 <span class="trait">protein</span> sequence to search the soybean EST database [ <xref rid="B41-ijms-17-00897" ref-type="bibr"> 41 </xref> ], using the BLASTP algorithm. We found that the amino acid sequence encoded by a tentative contig (TA51665) showed a <span class="trait">high</span> similarity to a region of AtBRI1. Therefore, we used the contig to design 5&#x02032;-RACE and 3&#x02032;-RACE primers to amplify the flanking regions of TA51665. After cloning and sequencing the flanking region, a cDNA fragment approximately 4 kb long containing a poly (A) tail was obtained (data not shown). Using ORF finder [ <xref rid="B42-ijms-17-00897" ref-type="bibr"> 42 </xref> ], the full cDNA was predicted to contain a long open reading frame (Genbank Accession No. KU360113) that encodes a <span class="trait">protein</span> of 1187 amino acids. Alignment with the soybean genome sequence indicated that this <span class="trait">protein</span> is encoded by <italic> <span class="gene">Glyma04g39610</span> </italic> [ <xref rid="B43-ijms-17-00897" ref-type="bibr"> 43 </xref> ]. As <italic> <span class="gene">Glyma06g15270</span> </italic> ( <italic> <span class="gene">GmBRI1</span> </italic> ) was reported to encode a BR receptor [ <xref rid="B39-ijms-17-00897" ref-type="bibr"> 39 </xref> ], we named <italic> <span class="gene">Glyma04g39610</span> </italic> as <italic> <span class="gene">GmBRI1b</span> </italic> and renamed <italic> <span class="gene">Glyma06g15270</span> </italic> as <italic> <span class="gene">GmBRI1a</span> </italic> . Further bioinformatics analysis showed that <span class="gene">GmBRI1b</span> contains a membrane-localized signal peptide in the N-terminus followed by 25 LRRs, a transmembrane domain, and a Ser/Thr kinase domain in the C-terminus ( <xref ref-type="table" rid="ijms-17-00897-t001"> Table 1 </xref> and <xref ref-type="app" rid="app1-ijms-17-00897"> Table S1 </xref> ). Alignment analysis indicated that <span class="gene">GmBRI1b</span> has 69% and 81% identity to AtBRI1 and pea BRI1 (PsBRI1), respectively ( <xref ref-type="app" rid="app1-ijms-17-00897"> Figure S1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Additionally, we investigated the expression levels of the BR receptors in soybean based on previous RNA-Seq studies [ <xref rid="B47-ijms-17-00897" ref-type="bibr"> 47 </xref> ]. As shown in <xref ref-type="fig" rid="ijms-17-00897-f001"> Figure 1 </xref> G, the transcript levels of <italic> <span class="gene">GmBRI1a</span> </italic> and <italic> <span class="gene">GmBRI1b</span> </italic> were relatively <span class="trait">high</span> in young leaf, flower, pod, seed, root, and nodule tissues, indicating their important roles in soybean growth and development. <italic> <span class="gene">GmBRL1a</span> </italic> and <italic> <span class="gene">GmBRL1b</span> </italic> were also expressed in all tested organs although their transcript levels were lower than those of <italic> <span class="gene">GmBRI1a</span> </italic> and <italic> <span class="gene">GmBRI1b</span> </italic> . By contrast, <italic> <span class="gene">GmBRI2a</span> </italic> and <italic> <span class="gene">GmBRI2b</span> </italic> had relatively low transcript levels in seeds and nodules ( <xref ref-type="fig" rid="ijms-17-00897-f001"> Figure 1 </xref> G). The relatively <span class="trait">high</span> expression levels of <italic> <span class="gene">GmBRI1a</span> </italic> and <italic> <span class="gene">GmBRI1b</span> </italic> in nodules suggest their important roles in nodulation. Based on similarities of their expression patterns, the six soybean BR receptor genes can be classified into three groups, <italic> <span class="gene">GmBRI1a</span> </italic> and <italic> <span class="gene">GmBRI1b</span> </italic> ; <italic> <span class="gene">GmBRL1a</span> </italic> and <italic> <span class="gene">GmBRL1b</span> </italic> ; and <italic> <span class="gene">GmBRI2a</span> </italic> and <italic> <span class="gene">GmBRI2b</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						As mentioned above, a signal peptide in the N-terminus and a transmembrane (TM) domain were predicted in <span class="gene">GmBRI1b</span> [ <xref rid="B44-ijms-17-00897" ref-type="bibr"> 44 </xref> ] ( <xref ref-type="table" rid="ijms-17-00897-t001"> Table 1 </xref> ). This indicated that <span class="gene">GmBRI1b</span> might be a cell membrane <span class="trait">protein.</span> To determine the subcellular localization of <span class="gene">GmBRI1b,</span> we constructed a fusion <span class="trait">protein</span> of <span class="gene">GmBRI1b::GFP</span> using the gateway vector pMDC43. We then co-transformed tobacco leaf epidermal cells with constructs encoding <span class="gene">GmBRI1b::GFP</span> and the plasma membrane marker <span class="trait">protein</span> AtPIP2A::mCherry [ <xref rid="B48-ijms-17-00897" ref-type="bibr"> 48 </xref> ]. Using laser confocal microscopy, we detected fluorescence signals only in the plasma membrane and the GFP fluorescence co-localized with mCherry fluorescence ( <xref ref-type="fig" rid="ijms-17-00897-f002"> Figure 2 </xref> ). This suggested that <span class="gene">GmBRI1b</span> is a cell membrane <span class="trait">protein.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AtBRI1 </italic> and <italic> OsBRI1 </italic> are ubiquitously expressed in all tissues [ <xref rid="B31-ijms-17-00897" ref-type="bibr"> 31 </xref> , <xref rid="B65-ijms-17-00897" ref-type="bibr"> 65 </xref> ]. We found that <italic> <span class="gene">GmBRI1b</span> </italic> is expressed in apical buds, cotyledons, epicotyls, hypocotyls, leaves, lateral roots, and primary roots ( <xref ref-type="fig" rid="ijms-17-00897-f001"> Figure 1 </xref> A), implying a crucial and universal role in soybean growth and development. The soybean RNA-Seq data [ <xref rid="B47-ijms-17-00897" ref-type="bibr"> 47 </xref> ] supported our results ( <xref ref-type="fig" rid="ijms-17-00897-f001"> Figure 1 </xref> G). Moreover, the transcript abundance of two soybean BR receptor genes, <italic> <span class="gene">GmBRI1a</span> </italic> and <italic> <span class="gene">GmBRI1b</span> </italic> , in nodules was relatively <span class="trait">high,</span> suggesting that these two genes play important roles in nodulation ( <xref ref-type="fig" rid="ijms-17-00897-f001"> Figure 1 </xref> G). Application of Brz on mature leaves or into the culture media increased the nodule number and inhibited internode growth in the soybean cultivar Enrei and foliar applications of BR inhibited nodulation and root growth in the super-nodulating mutant <italic> En6500 </italic> , indicating the existence of BR signaling modules in soybean [ <xref rid="B66-ijms-17-00897" ref-type="bibr"> 66 </xref> ]. Moreover, these results indicate that endogenous BR homeostasis or BR signaling might control nodulation. Recent studies demonstrated that pea BR biosynthesis mutant ( <italic> lk </italic> ), and BR receptor mutant ( <italic> lkb </italic> ) had fewer lateral roots and nodules and the decreased nodule number did not seem to be attributed to changes in endogenous GA or auxin levels [ <xref rid="B8-ijms-17-00897" ref-type="bibr"> 8 </xref> ]. Therefore, the roles of BR receptors in legume nodulation need further study.
					</p>
				
					<p>&nbsp;&nbsp;
						The loss of function mutation in BR signaling or biosynthesis genes in Arabidopsis results in a dwarf phenotype and late flowering [ <xref rid="B6-ijms-17-00897" ref-type="bibr"> 6 </xref> , <xref rid="B68-ijms-17-00897" ref-type="bibr"> 68 </xref> , <xref rid="B69-ijms-17-00897" ref-type="bibr"> 69 </xref> , <xref rid="B70-ijms-17-00897" ref-type="bibr"> 70 </xref> ]. We found that ectopic over-expression of <italic> <span class="gene">GmBRI1b</span> </italic> in transgenic <italic> bri1-6 </italic> , and <italic> bri1-5 bak1-1D </italic> mutant restored the normal wild-type phenotype, including the height of the seedlings and the length of petioles and siliques, leaf growth. Thus, we concluded that <italic> <span class="gene">GmBRI1b</span> </italic> functions as a BR receptor in <italic> Arabidopsis </italic> at the physiological and genetic level. We also demonstrated that <span class="gene">GmBRI1b</span> acts as a functional BR receptor in <italic> Arabidopsis </italic> at the molecular level. Ectopic over-expression of <italic> <span class="gene">GmBRI1b</span> </italic> repressed the relatively <span class="trait">high</span> expression of <italic> DWF4 </italic> , <italic> CPD </italic> , <italic> BR6OX-1 </italic> , and <italic> BR6OX-2 </italic> in the transgenic <italic> bri1-5 bak1-1D </italic> mutant. <span class="gene">GmBRI1a</span> was classified into the same subclade with <span class="gene">GmBRI1b</span> in this study. In a previous study, when <italic> <span class="gene">GmBRI1a</span> </italic> was expressed in the <italic> Arabidopsis bri1-5 </italic> mutant, reversed the developmental defects of the <italic> bri1-5 </italic> mutant [ <xref rid="B39-ijms-17-00897" ref-type="bibr"> 39 </xref> ], although the subcellular localization of <span class="gene">GmBRI1a</span> was not determined [ <xref rid="B39-ijms-17-00897" ref-type="bibr"> 39 </xref> ]. The BR-binding activity of <span class="gene">GmBRI1b</span> remains unknown. Results from this study combined with previous studies suggest that <span class="gene">GmBRIa</span> and <span class="gene">GmBRI1b</span> are BR receptors.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the subcellular localization of <span class="gene">GmBRI1b,</span> we constructed the fusion <span class="trait">protein</span> of <span class="gene">GmBRI1b</span> with GFP. The ORF of <italic> <span class="gene">GmBRI1b</span> </italic> , in which no stop codon exists, was amplified by PCR using the primers 5&#x02032;-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCATGAAAGCTCTGTACAGAAGCT-3&#x02032; and 5&#x02032;-GGGGACCACTTTGTACAAGAAAGCTGGGTCCTAATGCTTGCTCAATTCAGGG-3&#x02032;. The amplified cDNA fragment was then recombined into the pMDC43 vector, thus producing the GFP::BRI1b construct under the control of the 35S promoter. AtPIP2A, which was shown to be a plasma membrane aquaporin [ <xref rid="B48-ijms-17-00897" ref-type="bibr"> 48 </xref> ], fused with mCherry was used as the plasma membrane marker <span class="trait">protein.</span> <italic> Agrobacterium tumefaciens </italic> mediated transient expression in <italic> Nicotiana </italic> <italic> benthamiana </italic> tobacco leaves was conducted as described [ <xref rid="B79-ijms-17-00897" ref-type="bibr"> 79 </xref> ] with minor modifications. The <italic> Agrobacterium </italic> GV3101 strain harboring the constructs of GFP::GmBRI1b and AtPIP2A-mCherry [ <xref rid="B56-ijms-17-00897" ref-type="bibr"> 56 </xref> ] was inoculated in YEP medium with the appropriate antibiotics and incubated for 16 h with shaking at 28 &#x000b0;C. After centrifugation at 5000 rpm for 10 min, the cell pellet was re-suspended to OD <sub> 600 </sub> = 1.0 in the infiltration medium (10 mM MgCl <sub> 2 </sub> , 10 mM MES, and 150 &#x000b5;M acetosyringone). The cell suspension was then allowed to standing at 22 to &#x02212;24 &#x000b0;C for 2 to 3 h before infection into the tobacco leaves. A mix of cells containing the same quantities of <italic> GFP::GmBRI1b </italic> and <italic> AtPIP2A-mCherry </italic> , were then infiltrated into the leaves of three- to four-week-old tobacco plants. After three days, we observed the fluorescence distribution in the tobacco epidermal cells at 488 nm (GFP) and 587 nm (mCherry) wave lengths by confocal laser scanning microscopy (LSM780, Zeiss, Jena, Germany).
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the full sequence of the <italic> <span class="gene">GmBRI1b</span> </italic> cDNA, we designed primers at the regions of the initiation codon and the stop codon to amply the genomic DNA fragment. After sequencing, we compared the genomic and cDNA sequences of <italic> <span class="gene">GmBRI1b</span> </italic> using SIM4 software [ <xref rid="B80-ijms-17-00897" ref-type="bibr"> 80 </xref> ] to determine the <span class="trait">protein</span> structure of <span class="gene">GmBRI1b.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Glyma04g39610</span> </italic> encodes <italic> <span class="gene">GmBRI1b</span> </italic> , which functions as a BR receptor. <span class="gene">GmBRI1b</span> is cell membrane <span class="trait">protein.</span> Ectopic over-expression of <italic> <span class="gene">GmBRI1b</span> </italic> in <italic> bri1-6 </italic> and <italic> bri1-5 bak1-1D </italic> rescues the BR signaling-related growth and development defects of the two mutants. The <italic> Glycine max </italic> genome contains six BR receptor-encoding genes, which are grouped into three clades and are generated from three gene duplication events during evolution. BR receptors in plants were subjected to purifying selection during evolution.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of structurally related soybean <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> in plant growth and development</h2>
			<h3>PMID:27335454</h3>
				

				
					<h3>PMCID:4973743</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						WRKY transcription factors constitute a large <span class="trait">protein</span> superfamily with a predominant role in plant stress responses. In this study we report that two structurally related soybean WRKY proteins, <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76,</span> play a critical role in plant growth and flowering. <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> are both Group III WRKY proteins with a C2HC zinc finger domain and are close homologs of AtWRKY70 and AtWRKY54, two well-characterized Arabidopsis WRKY proteins with an important role in plant responses to biotic and abiotic stresses. <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> are both localized to the nucleus, recognize the TTGACC W-box sequence with a <span class="trait">high</span> specificity, and function as transcriptional activators in both yeast and plant cells. Expression of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> was detected at low levels in roots, stem, leaves, flowers, and pods. Expression of the two genes in leaves increased substantially during the first 4 weeks after germination but steadily declined thereafter with increased age. To determine their biological functions, transgenic Arabidopsis plants were generated overexpressing <span class="gene">GmWRKY58</span> or <span class="gene">GmWRKY76</span> Unlike AtWRKY70 and AtWRKY54, overexpression of <span class="gene">GmWRKY58</span> or <span class="gene">GmWRKY76</span> had no effect on disease <span class="trait">resistance</span> and only small effects on abiotic stress tolerance of the transgenic plants. Significantly, transgenic Arabidopsis plants overexpressing <span class="gene">GmWRKY58</span> or <span class="gene">GmWRKY76</span> flowered substantially earlier than control plants and this early flowering phenotype was associated with increased expression of several flowering-promoting genes, some of which are enriched in W-box sequences in their promoters recognized by <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76.</span> In addition, virus-induced silencing of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> in soybean resulted in stunted plants with reduced leaf expansion and terminated stem growth. These results provide strong evidence for functional divergence among close structural homologs of WRKY proteins from different plant species.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						WRKY transcription factors constitute a large <span class="trait">protein</span> superfamily with a predominant role in plant stress responses. In this study we report that two structurally related soybean WRKY proteins, <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76,</span> play a critical role in plant growth and flowering. <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> are both Group III WRKY proteins with a C2HC zinc finger domain and are close homologs of AtWRKY70 and AtWRKY54, two well-characterized Arabidopsis WRKY proteins with an important role in plant responses to biotic and abiotic stresses. <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> are both localized to the nucleus, recognize the TTGACC W-box sequence with a <span class="trait">high</span> specificity, and function as transcriptional activators in both yeast and plant cells. Expression of <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> was detected at low levels in roots, stem, leaves, flowers, and pods. Expression of the two genes in leaves increased substantially during the first 4 weeks after germination but steadily declined thereafter with increased age. To determine their biological functions, transgenic Arabidopsis plants were generated overexpressing <italic> <span class="gene">GmWRKY58</span> </italic> or <italic> <span class="gene">GmWRKY76</span> </italic> . Unlike <italic> AtWRKY70 </italic> and <italic> AtWRKY54 </italic> , overexpression of <italic> <span class="gene">GmWRKY58</span> </italic> or <italic> <span class="gene">GmWRKY76</span> </italic> had no effect on disease <span class="trait">resistance</span> and only small effects on abiotic stress tolerance of the transgenic plants. Significantly, transgenic Arabidopsis plants overexpressing <italic> <span class="gene">GmWRKY58</span> </italic> or <italic> <span class="gene">GmWRKY76</span> </italic> flowered substantially earlier than control plants and this early flowering phenotype was associated with increased expression of several flowering-promoting genes, some of which are enriched in W-box sequences in their promoters recognized by <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76.</span> In addition, virus-induced silencing of <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> in soybean resulted in stunted plants with reduced leaf expansion and terminated stem growth. These results provide strong evidence for functional divergence among close structural homologs of WRKY proteins from different plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						For generation of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> recombinant <span class="trait">protein,</span> their full-length cDNAs were PCR-amplified using gene-specific primers (5&#x02032;- AGCGGATCCATGAGTATTCTCTTCCC AAGAAGT-3&#x02032; and 5&#x02032;- AGCAAGCTTCTAAAGCAATTGGCTTT CATCAAAG -3&#x02032;), cloned into pET32a (Novagen, San Diego, CA, USA). The cloned <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> genes in the recombinant pET32a vector were confirmed by sequencing, and transformed into <italic> E. coli </italic> strain BL21 (DE3). Induction of <span class="trait">protein</span> expression and purification of recombinant His-tagged proteins were performed according to the protocol provided by Novagen. The recombinant proteins were purified according to the Novagen manual. Oligo nucleotides were labeled using the Biotin 3&#x02032; End DNA Labeling Kit (Thermo Scientific, Waltham, MA, USA) according to the manufacturer&#x02019;s instructions. A DNA binding assay was performed based on the Light Shift Chemiluminescent EMSA Kit (Thermo Scientific, Waltham, MA, USA).
					</p>
				
					<p>&nbsp;&nbsp;
						For generating transgenic <italic> <span class="gene">GmWRKY58</span> </italic> - and <italic> <span class="gene">GmWRKY76</span> </italic> -expressing Arabidopsis plants, the full-length coding sequences were amplified using gene specific primers (5&#x02032;-AGCGTCGACATG AGTATTCTCTTCCCAAGAAGTTC-3&#x02032; and 5&#x02032;-AGCGGATCCCT AAAGCAATTGGCTTTCATCAA-3&#x02032;). The amplified fragments were digested using appropriate restriction enzymes and inserted into the plant transformation vector pFGC5941 containing the CaMV <italic> 35S </italic> promoter. The fused plasmids were transformed into Col-0 wild-type plants using the <italic> Agrobacterium </italic> -mediated floral-dip procedure ( <xref rid="CIT0008" ref-type="bibr"> Clough and Bent, 1998 </xref> ). Transformants were identified for <span class="trait">resistance</span> to Basta herbicide. Transgenic plants overexpressing the transformed <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> transgene were identified using RT-qPCR.
					</p>
				
					<p>&nbsp;&nbsp;
						Our study of soybean <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> originated from our interest in identifying those members of the soybean WRKY superfamily that are capable of enhancing pathogen <span class="trait">resistance</span> and stress tolerance when overexpressed in transgenic Arabidopsis plants. From the sequenced soybean genome, we initially identified more than 170 WRKY genes and analyzed their expression in response to pathogens, abiotic stresses, and defense- and stress-related phytohormones such as salicylic acid (SA) and ABA. From the expression analysis, we identified more than 100 defense/stress-responsive soybean WRKY genes. To analyze more directly their ability to affect plant defense and stress responses, we generated overexpression constructs under the strong CaMV <italic> 35S </italic> promoter for the stress/defense-responsive soybean WRKY genes and transformed some of them into the model plant <italic> Arabidopsis thaliana </italic> . Transgenic Arabidopsis plants overexpressing the vast majority of the stress-responsive soybean WRKY genes displayed no significant alteration in plant morphology, growth, and development. As will be described later in detail, however, transgenic plants overexpressing soybean <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> were not altered in plant disease <span class="trait">resistance</span> or stress tolerance but were substantially altered in flowering time when compared to control plants, suggesting their specific role in plant growth and development.
					</p>
				
					<p>&nbsp;&nbsp;
						WRKY transcription factors recognize the TTGACC/T W-box sequences, which occur at <span class="trait">high</span> frequencies in the promoters of defense/stress-induced genes ( <xref rid="CIT0014" ref-type="bibr"> Dong <italic> et al. </italic> , 2003 </xref> ). To examine the DNA-binding activity of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76,</span> we expressed the genes in <italic> E. coli </italic> , purified the recombinant proteins, and analyzed their binding to an oligonucleotide that contains two direct TTGACC repeats (Pchn5; <xref ref-type="fig" rid="F3"> Fig. 3A </xref> ) using an electrophoretic mobility shifting assay (EMSA). A protein/DNA complex with a reduced mobility was detected when purified recombinant <span class="gene">GmWRKY58</span> or <span class="gene">GmWRKY76</span> <span class="trait">protein</span> was incubated with the Pchn5 probe ( <xref ref-type="fig" rid="F3"> Fig. 3B </xref> ). Binding of <span class="gene">GmWRKY58</span> or <span class="gene">GmWRKY76</span> was not detected with a mutant probe (mPchn5) in which both TTGACC sequences were changed to TTGAAC ( <xref ref-type="fig" rid="F3"> Fig. 3 </xref> ). Thus, binding of the two soybean Group III WRKY proteins to the TTGACC W-box sequence is highly specific.
					</p>
				
					<p>&nbsp;&nbsp;
						W-box-binding activities of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76.</span> (A) Nucleotide sequences of probes used for DNA binding assays. Pchn5 contains two TTGACC W-box sequences, which are mutated into TTGAAC in mPchn5. (B) Recombinant <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> were purified from <italic> E. coli </italic> cells and used for DNA binding assays with Pchn5 and mPchn5 as probes. The binding reactions (20 &#x000b5;l) contained 2ng labeled oligo DNA, 5 &#x000b5;g polydeoxyinosinic-deoxycytidylic acid, and 1 &#x000b5;g recombinant <span class="trait">protein.</span> The binding assays were repeated twice with independently prepared recombinant proteins with similar results.
					</p>
				
					<p>&nbsp;&nbsp;
						As described earlier, we were interested in functionally analyzing a large number of stress-responsive soybean WRKY genes through overexpression in Arabidopsis. Functional analysis of genes from crop plants, including soybean, through overexpression in Arabidopsis is a common and widely reported approach as the technique for transformation of the model plant is efficient and easy. In addition, many Arabidopsis WRKY genes have been well characterized and their close sequence similarity to many WRKY genes from soybean allows for comparative functional analysis. To analyze soybean WRKY genes in Arabidopsis, we first generated overexpression constructs of stress-responsive soybean WRKY genes under control of the constitutive and strong CaMV <italic> 35S </italic> promoter. Overexpression constructs for more than 60 stress-responsive soybean WRKY genes were then transformed into Arabidopsis using the floral-dip procedure and transformants were identified for their herbicide <span class="trait">resistance.</span> From independent transgenic Arabidopsis plants for each of &#x0003e;60 soybean WRKY genes examined at the same time, only those transgenic plants transformed with the <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> overexpression constructs flowered substantially earlier than the control plants. Because of the importance of this phenotype, these transgenic <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> plants were further characterized for possible alterations in growth, development, and stress responses. From the &#x0003e;10 independent transgenic plants for either of the two soybean WRKY genes, a majority of them had the early flowering phenotype and expressed <span class="trait">high</span> levels of the soybean WRKY transgenes based on qRT-PCR analysis (see <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw252/-/DC1"> Supplementary Fig. S1 </ext-link> ). Homozygous progeny containing a single transgene locus from these <italic> <span class="gene">GmWRKY58</span> </italic> - or <italic> <span class="gene">GmWRKY76</span> </italic> -expressing lines were subsequently obtained and used for phenotypic characterization. As shown in <xref ref-type="fig" rid="F8"> Fig. 8A </xref> , these transgenic <italic> <span class="gene">GmWRKY58</span> </italic> - or <italic> <span class="gene">GmWRKY76</span> </italic> -expressing plants grew mostly normally without major morphological abnormalities in the vegetative stages. However, these plants flowered substantially earlier than control plants ( <xref ref-type="fig" rid="F8"> Fig. 8B </xref> ). The numbers of leaves at the time of bolting in these transgenic plants were only about 60% of those in wild-type plants ( <xref ref-type="fig" rid="F8"> Fig. 8C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Arabidopsis AtWRKY70 has been extensively analyzed in plant defense responses. Analysis using both loss-of-function mutants and overexpression lines revealed that AtWRKY70 promotes SA-dependent defense pathways but suppresses JA-dependent signaling ( <xref rid="CIT0035" ref-type="bibr"> Li <italic> et al. </italic> , 2004 </xref> , <xref rid="CIT0034" ref-type="bibr"> 2006 </xref> ; <xref rid="CIT0021" ref-type="bibr"> Hu <italic> et al. </italic> , 2012 </xref> ; <xref rid="CIT0051" ref-type="bibr"> Shim and Choi, 2013 </xref> ; <xref rid="CIT0051" ref-type="bibr"> Shim <italic> et al. </italic> , 2013 </xref> ). To determine whether overexpression of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> altered plant defense in transgenic plants, we analyzed the responses of Arabidopsis plants overexpressing the soybean WRKY genes to the virulent bacterial pathogen PstDC3000. A large number of studies have shown that Arabidopsis <span class="trait">resistance</span> to the virulent bacterial pathogen is dependent on SA signaling ( <xref rid="CIT0015" ref-type="bibr"> Dong, 1998 </xref> ). When inoculated with the virulent bacteria, however, these transgenic <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> plants developed disease symptoms and supported bacterial growth at levels similar to those in control wild-type plants (see <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw252/-/DC1"> Supplementary Fig. S2 </ext-link> ). By contrast, the SA-signaling mutant <italic> npr1 </italic> developed more severe disease symptoms and supported higher levels of bacterial growth when inoculated with the same virulent bacteria ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw252/-/DC1"> Supplementary Fig. S2 </ext-link> ). Thus, overexpression of <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> did not affect SA-dependent defense against the bacterial pathogen in Arabidopsis plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Unlike Arabidopsis, many plants including important crop species such as soybean are still relatively difficult to routinely and reliably transform. Virus-induced gene silencing (VIGS) provides an alternative approach for targeted down-regulation and functional analysis of genes. VIGS mediated by Bean pod mottle virus (BPMV) has been successfully used in the functional investigation of soybean genes involved in defense and other processes, including soybean MAPK4 <span class="gene">(GmMPK4)</span> ( <xref rid="CIT0037" ref-type="bibr"> Liu <italic> et al. </italic> , 2011 </xref> ). To determine the role of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> in soybean, we attempted to silence their gene expression using BPMV-mediated gene silencing. Gene fragments from <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> were separately cloned into the BPMV silencing vector and the viral vectors were delivered into soybean plants through particle bombardment. To determine the extent of silencing, we performed qRT-PCR of the transcript levels of the two soybean WRKY genes in inoculated plants. As shown in <xref ref-type="fig" rid="F11"> Fig. 11A </xref> , when compared to the plants inoculated with empty vectors, those inoculated with either <italic> <span class="gene">GmWRKY58</span> </italic> - or <italic> <span class="gene">GmWRKY76</span> </italic> -silencing vectors displayed a reduction of approximately 60% in the transcript levels of the targeted gene but also of the close homolog as well. As described earlier, <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> share a <span class="trait">high</span> nucleotide sequence identify (&#x0003e;90%) and as a result either the construct for silencing <italic> <span class="gene">GmWRKY58</span> </italic> or for silencing <italic> <span class="gene">GmWRKY76</span> </italic> can silence both genes. Plants in which <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> were silenced had consistent phenotypes characterized by stunted growth ( <xref ref-type="fig" rid="F11"> Fig. 11B </xref> ). The soybean cultivar used in the experiments (&#x02018;Williams 82&#x02019;) is a Group III indeterminate variety. Under the growth conditions with 12/12h photoperiod, the control plants maintained vegetative growth even after flowering and pod setting. Those plants inoculated with <italic> <span class="gene">GmWRKY58</span> </italic> - and <italic> <span class="gene">GmWRKY76</span> </italic> -silencing vectors were normal during the first 3&#x02013;4 weeks after inoculation. However, around 4 weeks after inoculation, those plants with severe silencing of <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> started to display stunted phenotypes characterized by slower emergence and smaller size of new leaves and shorter length of upper parts of the elongating stems ( <xref ref-type="fig" rid="F11"> Fig. 11B </xref> , <xref ref-type="fig" rid="F11"> 11C </xref> ). As a result, the stature of the upper part of plants with <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> silenced was substantially reduced, almost mimicking a determinate growth pattern, even though the stature of their lower parts was largely normal, with normal-sized leaves and pods.
					</p>
				
					<p>&nbsp;&nbsp;
						As a large superfamily of plant transcription factors, WRKY proteins have been subjected to extensive functional analysis over the last two decades. The overwhelming majority of these studies have focused on the roles of WRKY transcription factors in broad plant responses to biotic and abiotic stresses. By contrast, a broad role of WRKY proteins in plant growth and development has not emerged, although a few WRKY proteins have been shown to play roles in certain specific plant developmental processes ( <xref rid="CIT0025" ref-type="bibr"> Johnson <italic> et al. </italic> , 2002 </xref> ; <xref rid="CIT0038" ref-type="bibr"> Luo <italic> et al. </italic> , 2005 </xref> , <xref rid="CIT0039" ref-type="bibr"> 2013 </xref> ; <xref rid="CIT0011" ref-type="bibr"> Li <italic> et al. </italic> , 2015 </xref> ). In the present study, we have provided strong evidence that two closely related soybean WRKY proteins, <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76,</span> have a critical role in plant growth and development. Even though they are close homologs of Arabidopsis AtWRKY70 and AtWRKY54, overexpression of <span class="gene">GmWRKY58</span> and <span class="gene">GmWRKY76</span> did not result in altered <span class="trait">resistance</span> to the bacterial pathogen <italic> P. syringae </italic> (see <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw252/-/DC1"> Supplementary Fig. S2 </ext-link> ) or in tolerance to osmotic stress &#x02013; two phenotypes previously established for AtWRKY70 and AtWRKY54 ( <xref rid="CIT0035" ref-type="bibr"> Li <italic> et al. </italic> , 2004 </xref> , <xref rid="CIT0034" ref-type="bibr"> 2006 </xref> ; <xref rid="CIT0021" ref-type="bibr"> Hu <italic> et al. </italic> , 2012 </xref> ; <xref rid="CIT0051" ref-type="bibr"> Shim and Choi, 2013 </xref> ; <xref rid="CIT0051" ref-type="bibr"> Shim <italic> et al. </italic> , 2013 </xref> ). The effects of their overexpression on tolerance to salt and osmotic stress and sensitivity to ABA were also negligible, or relatively small and transient ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw252/-/DC1"> Supplementary Fig. S3 </ext-link> ). On the other hand, transgenic plants overexpressing <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> flowered substantially earlier than control plants ( <xref ref-type="fig" rid="F8"> Fig. 8 </xref> ), a phenotype not observed in AtWRKY70- and AtWRKY54-overexpressing plants. Furthermore, silencing of <italic> <span class="gene">GmWRKY58</span> </italic> and <italic> <span class="gene">GmWRKY76</span> </italic> led to stunted plants characterized by greatly inhibited growth of expanding leaves and elongating stems ( <xref ref-type="fig" rid="F1"> Fig. 11 </xref> ). These results establish that these two closely related WRKY proteins play an important and specific role in plant growth and development.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Microsymbiont diversity and phylogeny of native bradyrhizobia associated with soybean ( <italic> Glycine max </italic> L. Merr.) nodulation in South African soils <sup> <xref ref-type="fn" rid="d32e50"> &#x02606; </xref> </sup></h2>
			<h3>PMID:27324571</h3>
				

				
					<h3>PMCID:4958686</h3>
					<h4>期刊:Systematic and Applied Microbiology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Naamala </surname>  <given-names> Judith </given-names>  </name>  <xref rid="aff0005" ref-type="aff"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jaiswal </surname>  <given-names> Sanjay K. </given-names>  </name>  <xref rid="aff0010" ref-type="aff"> b </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dakora </surname>  <given-names> Felix D. </given-names>  </name>  <email> dakorafd@tut.ac.za </email>  <xref rid="aff0010" ref-type="aff"> b </xref>  <xref rid="cor0005" ref-type="corresp"> &#x0204e; </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genetic diversity and identification of slow- and fast-growing soybean root nodule bacterial isolates from different agro-climatic regions in Mpumalanga, Limpopo and Gauteng Provinces of South Africa were evaluated. The 16S-rDNA-RFLP analysis of 100 rhizobial isolates and eight reference type strains placed the isolates into six major clusters, and revealed their site-dependent genomic diversity. Sequence analysis of single and concatenated housekeeping genes (atpD, <span class="gene">glnII</span> and gyrB), as well as the symbiotic gene <span class="gene">nifH</span> captured a considerably higher level of genetic diversity and indicated the dominance of Bradyrhizobium diazoefficiens and Bradyrhizobium japonicum in Mpumalanga, Limpopo and Gauteng Provinces. Gene sequence similarities of isolates with type strains of Bradyrhizobium ranged from 97.3 to 100% for the 16S rDNA, and 83.4 to 100% for the housekeeping genes. The <span class="gene">glnII</span> gene phylogeny showed discordance with the other genes, suggesting lateral gene transfer or recombination events. Concatenated gene sequence analysis showed that most of the isolates did not align with known type strains and might represent new species from South Africa. This underscores the <span class="trait">high</span> genetic variability associated with soybean Bradyrhizobium in South African soils, and the presence of an important reservoir of novel soybean-nodulating bradyrhizobia in the country. In this study, the grouping of isolates was influenced by site origin, with Group I isolates originating from Limpopo Province and Groups II and III from Mpumlanga Province in the 16S rDNA-RFLP analysis.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The genetic diversity and identification of slow- and fast-growing soybean root nodule bacterial isolates from different agro-climatic regions in Mpumalanga, Limpopo and Gauteng Provinces of South Africa were evaluated. The 16S-rDNA-RFLP analysis of 100 rhizobial isolates and eight reference type strains placed the isolates into six major clusters, and revealed their site-dependent genomic diversity. Sequence analysis of single and concatenated housekeeping genes ( <italic> <span class="gene">atpD</span> </italic> , <italic> <span class="gene">glnII</span> </italic> and <italic> <span class="gene">gyrB</span> </italic> ), as well as the symbiotic gene <italic> <span class="gene">nifH</span> </italic> captured a considerably higher level of genetic diversity and indicated the dominance of <italic> Bradyrhizobium diazoefficiens </italic> and <italic> Bradyrhizobium japonicum </italic> in Mpumalanga, Limpopo and Gauteng Provinces. Gene sequence similarities of isolates with type strains of <italic> Bradyrhizobium </italic> ranged from 97.3 to 100% for the 16S rDNA, and 83.4 to 100% for the housekeeping genes. The <italic> gln </italic> II gene phylogeny showed discordance with the other genes, suggesting lateral gene transfer or recombination events. Concatenated gene sequence analysis showed that most of the isolates did not align with known type strains and might represent new species from South Africa. This underscores the <span class="trait">high</span> genetic variability associated with soybean <italic> Bradyrhizobium </italic> in South African soils, and the presence of an important reservoir of novel soybean-nodulating bradyrhizobia in the country. In this study, the grouping of isolates was influenced by site origin, with Group I isolates originating from Limpopo Province and Groups II and III from Mpumlanga Province in the 16S rDNA-RFLP analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						A phylogenetic tree based on concatenated genes confirmed the presence of diverse and novel type of <italic> Bradyrhizobium </italic> isolates nodulating soybean in South Africa. With the discordance observed in test isolates from <italic> <span class="gene">glnII</span> </italic> and 16S rDNA phylograms in comparison to those of <italic> <span class="gene">atpD</span> </italic> and <italic> <span class="gene">gyrB</span> </italic> housekeeping genes, we constructed concatenated tree with <italic> <span class="gene">atpD</span> </italic> &#x000a0;+&#x000a0; <italic> <span class="gene">gyrB</span> </italic> sequences ( <xref rid="fig0010" ref-type="fig"> Fig. 2 </xref> ). The concatenated phylogeny was based on average consensus sequences of 843 analysed sites containing 509 (60.6%), 331 (39.4%), 170 (20.24%) and 161 (19.17%) of conserved, variable, parsimony-informative and singleton information sites, respectively ( <xref rid="sec0095" ref-type="sec"> Table S5 </xref> ). The constructed concatenated tree revealed four large clusters (Clusters I&#x02013;IV) with <italic> Bradyrhizobium </italic> species corresponding to <italic> B. diazoefficiens </italic> , <italic> Bradyrhizobium guangdongense </italic> and <italic> B. japonicum </italic> ( <xref rid="fig0010" ref-type="fig"> Fig. 2 </xref> ). Five test strains in Cluster I grouped with <italic> B. diazoefficiens </italic> with <span class="trait">high</span> bootstrap support (100%) and 100% sequence identity, while isolate TUTLl4.10.9 was linked to the Cluster I outgroup with 64% bootstrap support and 99.1% sequence identity ( <xref rid="fig0010" ref-type="fig"> Fig. 2 </xref> ; <xref rid="sec0095" ref-type="sec"> Table S7 </xref> ). In the <italic> <span class="gene">atpD</span> </italic> phylogram, the same isolate TUTLl4.10.9 grouped with <italic> B. japonicum </italic> in Cluster II.
					</p>
				
					<p>&nbsp;&nbsp;
						The reconstructed phylogenetic tree based on <italic> <span class="gene">nifH</span> </italic> was congruent with the tree built with <italic> <span class="gene">atpD</span> </italic> gene. Isolates TUTMP11.7, TUTMP14.20.2 and TUTMP14.10.3 were not included in the <italic> <span class="gene">nifH</span> </italic> study due to poor sequence results. In the <italic> <span class="gene">nifH</span> </italic> phylogenetic tree, all isolates from Limpopo Province, isolate TUTGP.10.10.2 from Gauteng, and some isolates from Mpumalanga were tightly clustered in one clade along with type strains such as <italic> Bradyrhizobium daqingense </italic> CCBAU 15774 <sup> T </sup> , <italic> Bradyrhizobium ottawaense </italic> OO99 <sup> T </sup> , <italic> Bradyrhizobium huanghuaihaiense </italic> CCBAU 23303 <sup> T </sup> , <italic> B. liaoningense </italic> LMG 18230 <sup> T </sup> , <italic> B. japonicum </italic> USDA 6 <sup> T </sup> and <italic> B. diazoefficiens </italic> USDA 110 <sup> T </sup> with a <span class="trait">high</span> 80 bootstrap support and 100% pair-wise sequence identity ( <xref rid="fig0015" ref-type="fig"> Fig. 3 </xref> ). Isolates TUTMP2.9 and TUTMP19.10 were located on the outgroups of Cluster I with bootstrap support 79 and 100, respectively, and pairwise sequence similarity of 91.6% and 98.3% with the <italic> Bradyrhizobium </italic> type strains present in Cluster I. Cluster II consisted of only Mpumalanga isolates TUTMP10.6, TUTMP9.7, TUTMP7.7 and TUTMP12.13 without any type or reference strains ( <xref rid="fig0015" ref-type="fig"> Fig. 3 </xref> ). In the <italic> <span class="gene">atpD</span> </italic> phylogram ( <xref rid="sec0095" ref-type="sec"> Fig. S2 </xref> ), isolate TUTMP10.6 was closely grouped with <italic> B. elkanii </italic> USDA 76 with a 56% bootstrap support, while TUTMP12.13, TUTMP9.7 and TUTMP7.7 clustered together without any reference strains.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:An Improved Variant of Soybean Type 1 Diacylglycerol Acyltransferase Increases the Oil Content and Decreases the Soluble Carbohydrate Content of Soybeans.</h2>
			<h3>PMID:27208257  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roesler K(1), Shen B(2), Bermudez E(2), Li C(2), Hunt J(2), Damude HG(2), Ripp KG(2), Everard JD(2), Booth JR(2), Castaneda L(2), Feng L(2), Meyer K(1).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Kinetically improved diacylglycerol acyltransferase (DGAT) variants were created to favorably alter carbon partitioning in soybean (Glycine max) seeds. Initially, variants of a type 1 <span class="gene">DGAT</span> from a high-oil, high-oleic acid plant seed, Corylus americana, were screened for <span class="trait">high</span> oil content in Saccharomyces cerevisiae Nearly all <span class="gene">DGAT</span> variants examined from high-oil strains had increased affinity for oleoyl-CoA, with S0.5 values decreased as much as 4.7-fold compared with the wild-type value of 0.94 µm Improved soybean <span class="gene">DGAT</span> variants were then designed to include amino acid substitutions observed in promising C. americana <span class="gene">DGAT</span> variants. The expression of soybean and C. americana <span class="gene">DGAT</span> variants in soybean somatic embryos resulted in oil contents as <span class="trait">high</span> as 10% and 12%, respectively, compared with only 5% and 7.6% oil achieved by overexpressing the corresponding wild-type DGATs. The affinity for oleoyl-CoA correlated strongly with oil content. The soybean <span class="gene">DGAT</span> variant that gave the greatest oil increase contained 14 amino acid substitutions out of a total of 504 (97% sequence identity with native). Seed-preferred expression of this soybean <span class="gene">DGAT1</span> variant increased oil content of soybean seeds by an average of 3% (16% relative increase) in highly replicated, single-location field trials. The <span class="gene">DGAT</span> transgenes significantly reduced the soluble carbohydrate content of mature seeds and increased the seed <span class="trait">protein</span> content of some events. This study demonstrated that engineering of the native <span class="gene">DGAT</span> enzyme is an effective strategy to improve the oil content and value of soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of calcium-dependent <span class="trait">protein</span> kinases and their expression patterns in response to herbivore and wounding stresses in soybean.</h2>
			<h3>PMID:27179522  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu H(1), Che Z(1), Zeng X(1), Zhou X(1), Sitoe HM(1), Wang H(1), Yu D(2).
				<br/> 
				年份：2016 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calcium-dependent <span class="trait">protein</span> kinases (CDPKs) play important roles in various aspects of plant physiology and involve in many cellular processes. However, genome-wide analysis of <span class="gene">CDPK</span> family in plant species is limited and few studies have been reported in soybean. In this study, a total of 39 genes encoding CDPKs were identified from the whole-genome sequence of soybean (Glycine max), which were denominated as <span class="gene">GmCPK1-GmCPK39.</span> These 39 <span class="gene">CDPK</span> genes could be classified into four subfamilies, and most genes showed tissue-specific expression patterns. Eight soybean CDPKs clustered together with the previously reported CDPKs related to pathogen, wounding, or herbivore stress were further analyzed. Differential gene expression analysis of these eight <span class="gene">CDPK</span> genes in response to herbivore and wounding stresses helps us identify <span class="gene">GmCPK3</span> and <span class="gene">GmCPK31</span> as the candidate genes for herbivore <span class="trait">resistance</span> in soybean, whose relative transcript abundance rapidly increased after wound and herbivore attacks. Sub-cellular localization revealed that <span class="gene">GmCPK3</span> and <span class="gene">GmCPK31</span> were localized in plasma membranes, which is consistent with previously reported plant defense related CDPKs. These results may suggest that <span class="gene">GmCPK3</span> and <span class="gene">GmCPK31</span> play important roles in the plant response to biotic stress. Simultaneously, our study will provide an important foundation for further functional characterization of the soybean <span class="gene">CDPK</span> gene family.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-Density Genetic Mapping Identifies New Major Loci for Tolerance to Low-Phosphorus Stress in Soybean.</h2>
			<h3>PMID:27065041</h3>
				

				
			<p>
				作者：Zhang D(1), Li H(1), Wang J(2), Zhang H(3), Hu Z(4), Chu S(1), Lv H(1), Yu D(5).
				<br/> 
				年份：2016 Mar 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus (P) is essential for all living cells and organisms, and low-P stress represents a major constraint on plant growth and <span class="trait">yield</span> worldwide. Soybean is an important economical resource of <span class="trait">protein</span> and oil for human and animals, and soybean is also a high-P demand species that is sensitive to low-P stress, which is considered a major constraint on soybean production. However, P efficiency is an important complex quantitative trait involving multiple genes, and the mechanisms underlying soybean P efficiency are largely unknown. Here, we reported the construction of a high-density genetic map using a specific-locus amplified fragment sequencing (SLAF-seq) strategy in soybean. This map, spanning 3020.59 cM in length, contained 6159 markers on 20 chromosomes, with an average distance of 0.49 cM between adjacent markers. Based on this map, 20 loci, including eight novel loci, associated with P efficiency-related traits were identified across multiple years and treatments. The confidence intervals of almost all QTLs were refined significantly, and the accuracy of this map was evidenced by coincident detections of the previously identified P efficiency-related genes <span class="gene">GmACP1</span> and <span class="gene">GmPT1.</span> Notably, a highly significant novel QTL located on chromosome 4, q4-2, was identified across traits, years and treatments. Several candidate genes, such as a pectin methylesterase-encoding gene <span class="gene">(Glyma.04G214000)</span> and a <span class="trait">protein</span> kinase gene <span class="gene">(Glyma.13G161900),</span> with significantly differential expression upon low-P stress were considered as promising candidates involved in regulating soybean P efficiency. Markers that tightly associated with P efficiency could be used for marker-assisted selection in a soybean P efficient breeding program. Further, dissection of these QTLs will facilitate gene cloning underlying P efficiency in soybean, and increase our understanding of efficient use of P in enhancing crop <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Determination of the Michaelis-Menten kinetics and the genes expression involved in phyto-degradation of cyanide and ferri-cyanide.</h2>
			<h3>PMID:26992391  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yu XZ(1), Zhang XH(2)(3).
				<br/> 
				年份：2016 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Hydroponic experiments were conducted with different species of plants (rice, maize, soybean and willow) exposed to ferri-cyanide to investigate the half-saturation constant (K M ) and the maximal metabolic capacity (v max ) involved in phyto-assimilation. Three varieties for each testing species were collected from different origins. Measured concentrations show that the uptake rates responded biphasically to ferri-cyanide treatments by showing increases linearly at low and almost constant at <span class="trait">high</span> concentrations from all treatments, indicating that phyto-assimilation of ferri-cyanide followed the Michaelis-Menten kinetics. Using non-linear regression, the highest v max was by rice, followed by willows. The lowest v max was found for soybean. All plants, except maize (DY26) and rice (XJ12), had a similar K M value, suggesting the same enzyme was active in phyto-assimilation of ferri-cyanide. Transcript level, by real-time quantitative PCR, of enzymes involved in degradation of cyanides showed that the analyzed genes were differently expressed during different cyanides exposure. The expression of <span class="gene">CAS</span> and ST genes responded positively to KCN exposure, suggesting that β-CAS and ST pathways were two possible pathways for cyanide detoxification in rice. The transcript level of NIT and ASPNASE genes also showed a remarkable up-regulation to KCN, implying the contribution to the pool of amino acid aspartate, which is an end product of CN metabolism. Up-regulation of <span class="gene">GS</span> genes suggests that acquisition of ammonium released from cyanide degradation may be an additional nitrogen source for plant nutrition. Results also revealed that the expressions of these genes, except for GS, were relatively constant during iron cyanide exposure, suggesting that they are likely metabolized by plants through a non-defined pathway rather than the β-CAS pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Ferritin Expression in Saccharomyces cerevisiae Modulates Iron Accumulation and <span class="trait">Resistance</span> to Elevated Iron Concentrations.</h2>
			<h3>PMID:26969708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：de Llanos R(1), Martínez-Garay CA(1), Fita-Torró J(1), Romero AM(1), Martínez-Pastor MT(2), Puig S(3).
				<br/> 
				年份：2016 May 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fungi, including the yeast Saccharomyces cerevisiae, lack ferritin and use vacuoles as iron storage organelles. This work explored how plant ferritin expression influenced baker's yeast iron metabolism. Soybean seed ferritin <span class="gene">H1</span> (SFerH1) and SFerH2 genes were cloned and expressed in yeast cells. Both soybean ferritins assembled as multimeric complexes, which bound yeast intracellular iron in vivo and, consequently, induced the activation of the genes expressed during iron scarcity. Soybean ferritin protected yeast cells that lacked the Ccc1 vacuolar iron detoxification transporter from toxic iron levels by reducing cellular oxidation, thus allowing growth at <span class="trait">high</span> iron concentrations. Interestingly, when simultaneously expressed in ccc1Δ cells, SFerH1 and SFerH2 assembled as heteropolymers, which further increased iron <span class="trait">resistance</span> and reduced the oxidative stress produced by excess iron compared to ferritin homopolymer complexes. Finally, soybean ferritin expression led to increased iron accumulation in both wild-type and ccc1Δ yeast cells at certain environmental iron concentrations.IMPORTANCE: Iron deficiency is a worldwide nutritional disorder to which women and children are especially vulnerable. A common strategy to combat iron deficiency consists of dietary supplementation with inorganic iron salts, whose bioavailability is very low. Iron-enriched yeasts and cereals are alternative strategies to diminish iron deficiency. Animals and plants possess large ferritin complexes that accumulate, detoxify, or buffer excess cellular iron. However, the yeast Saccharomyces cerevisiae lacks ferritin and uses vacuoles as iron storage organelles. Here, we explored how soybean ferritin expression influenced yeast iron metabolism, confirming that yeasts that express soybean seed ferritin could be explored as a novel strategy to increase dietary iron absorption.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The nuclear <span class="trait">protein</span> <span class="gene">GmbZIP110</span> has transcription activation activity and plays important roles in the response to salinity stress in soybean</h2>
			<h3>PMID:26837841</h3>
				

				
					<h3>PMCID:4738249</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhaolong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ali </surname>  <given-names> Zulfiqar </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiaolan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yihong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yi </surname>  <given-names> Jinxin </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shao </surname>  <given-names> Hongbo </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hongxiang </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dayong </given-names>  </name>  <xref ref-type="corresp" rid="c3"> c </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Provincial Key Laboratory of Agrobiology, Institute of Biotechnology, Jiangsu Academy of Agricultural Sciences </institution> , Nanjing 210014,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Department of Plant Breeding and Genetics, University of Agriculture </institution> , Faisalabad, Punjab,  <country> Pakistan </country>  </aff>
				<br/> 
				年份：<day> 31 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant basic-leucine zipper (bZIP) transcription factors play important roles in many biological processes and are involved in the regulation of salt stress tolerance. Previously, our lab generated digital gene expression profiling (DGEP) data to identify differentially expressed genes in a salt-tolerant genotype of Glycine soja (STGoGS) and a salt-sensitive genotype of Glycine max (SSGoGM). This DGEP data revealed that the expression (log2 ratio) of <span class="gene">GmbZIP110</span> was up-regulated 2.76-fold and 3.38-fold in SSGoGM and STGoGS, respectively. In the present study, the salt inducible gene <span class="gene">GmbZIP110</span> was cloned and characterized through phylogenetic analysis, subcellular localization and in silico transcript abundance analysis in different tissues. The functional role of this gene in salt tolerance was studied through transactivation analysis, DNA binding ability, expression in soybean composite seedlings and transgenic Arabidopsis, and the effect of <span class="gene">GmbZIP110</span> on the expression of stress-related genes in transgenic Arabidopsis was investigated. We found that <span class="gene">GmbZIP110</span> could bind to the ACGT motif, impact the expression of many stress-related genes and the accumulation of proline, Na(+) and K(+), and enhanced the salt tolerance of composite seedlings and transgenic Arabidopsis. Integrating all these results, we propose that <span class="gene">GmbZIP110</span> plays a critical role in the response to salinity stress in soybean and has <span class="trait">high</span> potential usefulness in crop improvement.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plant basic-leucine zipper (bZIP) transcription factors play important roles in many biological processes and are involved in the regulation of salt stress tolerance. Previously, our lab generated digital gene expression profiling (DGEP) data to identify differentially expressed genes in a salt-tolerant genotype of Glycine soja (STGoGS) and a salt-sensitive genotype of Glycine max (SSGoGM). This DGEP data revealed that the expression (log2 ratio) of <italic> <span class="gene">GmbZIP110</span> </italic> was up-regulated 2.76-fold and 3.38-fold in SSGoGM and STGoGS, respectively. In the present study, the salt inducible gene <italic> <span class="gene">GmbZIP110</span> </italic> was cloned and characterized through phylogenetic analysis, subcellular localization and in silico transcript abundance analysis in different tissues. The functional role of this gene in salt tolerance was studied through transactivation analysis, DNA binding ability, expression in soybean composite seedlings and transgenic <italic> Arabidopsis </italic> , and the effect of <span class="gene">GmbZIP110</span> on the expression of stress-related genes in transgenic <italic> Arabidopsis </italic> was investigated. We found that <span class="gene">GmbZIP110</span> could bind to the ACGT motif, impact the expression of many stress-related genes and the accumulation of proline, Na <sup> + </sup> and K <sup> + </sup> , and enhanced the salt tolerance of composite seedlings and transgenic <italic> Arabidopsis </italic> . Integrating all these results, we propose that <span class="gene">GmbZIP110</span> plays a critical role in the response to salinity stress in soybean and has <span class="trait">high</span> potential usefulness in crop improvement.
					</p>
				
					<p>&nbsp;&nbsp;
						The nucleotide sequence of <italic> <span class="gene">GmbZIP110</span> </italic> from <italic> Glycine max </italic> is identical to that from <italic> Glycine soja </italic> . The <span class="gene">GmbZIP110</span> amino acid sequence contains one conserved bZIP domain that is 64 residues in length ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ). The nuclear localization signal sequence RKRKRMLSNRESARRSRMRK (spanning amino acids 32&#x02013;51) was found in the conserved bZIP domain of <span class="gene">GmbZIP110.</span> A <span class="gene">GmbZIP110-green</span> fluorescent <span class="trait">protein</span> (GFP) fusion construct and a control construct containing only GFP were created using the pJIT166-GFP vector; transcription was driven by the CaMV 35S promoter in both constructs. Constructs were introduced into <italic> Arabidopsis </italic> protoplasts by a PEG4000-mediated method. The cells were inspected with a confocal laser-scanning microscope to visualize the subcellular localization of the proteins. The nucleus was stained with Hoechst 33342. The <span class="gene">GmbZIP110-GFP</span> <span class="trait">protein</span> was detected only in the nucleus; the control <span class="trait">protein</span> was observed throughout the cell ( <xref ref-type="fig" rid="f1"> Fig. 1c </xref> ). These results indicated that <span class="gene">GmbZIP110</span> was a nuclear <span class="trait">protein.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In silico transcript abundance of 99 <italic> <span class="gene">GmbZIPs</span> </italic> genes varied in the following samples: young leaves, flowers, one cm pods, pod shells at 10 days after flowering (DAF), pod shells at 14, seeds at 10 DAF, seeds at 14 DAF, seeds at 21 DAF, seeds at 25 DAF, seeds at 28 DAF, seeds at 35 DAF, seeds at 42 DAF, roots and nodules ( <xref ref-type="fig" rid="f2"> Fig. 2a </xref> ). The abundance of <italic> <span class="gene">GmbZIP110</span> </italic> transcripts was low in seeds and very <span class="trait">high</span> in nodule tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						bZIP proteins are known to bind to G-box elements <xref ref-type="bibr" rid="b13"> 13 </xref> . Therefore, the DNA binding ability of the <span class="gene">GmbZIP110</span> <span class="trait">protein</span> to a G-box element was examined using a yeast one-hybrid (Y1H) assay system (Clontech). Two tandem repeats of a G-box sequence (CACGTG) were synthesized and inserted into the pHIS2.1 vector, which contained the reporter gene <italic> <span class="gene">HIS3</span> </italic> . A minimal promoter was present downstream of the cis-elements and upstream of the <italic> <span class="gene">HIS3</span> </italic> gene. The reporter plasmid pHIS2.1, which harbored a double G-box motif plus the reporter gene <italic> <span class="gene">HIS3</span> </italic> , and the effector plasmid pGADT7-Rec2-GmbZIP110 were transformed together into the yeast strain Y187. The constructs pGADT7-Rec2&#x02013;53 and p53HIS2.1 were used as a positive control. Negative controls where transformed with pGADT7-Rec2 and p53HIS2.1 or pGADT7-Rec2 and pHIS2.1-P-motif. Screening was carried out on triple dropout medium (TDO, SD/-His/-Leu/-Trp) containing 10, 20, 40, 60, 80 or 100&#x02009;mM 3-amino-1, 2, 4-triazole (3-AT). The results showed that all of the transformants could grow on TDO medium containing a low concentration of 3-AT ( <xref ref-type="fig" rid="f3"> Fig. 3b </xref> ). Transformants harboring the effector plasmids pAD-GmbZIP110 and the reporter plasmid containing the G-box motif grew well on media with a <span class="trait">high</span> concentration of 3-AT medium. Growth on this medium was similar to that of the positive control, indicating that the <span class="gene">GmbZIP110</span> <span class="trait">protein</span> could bind to the G-box motif.
					</p>
				
					<p>&nbsp;&nbsp;
						The function of <span class="gene">GmbZIP110</span> in plant under salt stress was also investigated through generation of transgenic <italic> Arabidopsis </italic> plants. Under normal condition, all the transgenic lines showed no significant phenotypic and survival rate differences with the wild type (WT) plants ( <xref ref-type="fig" rid="f4"> Fig. 4c </xref> ). Four-week-old transgenic <italic> Arabidopsis </italic> seedlings were treated with 200&#x02009;mM NaCl for one week. WT seedlings were used as control. Seedlings from transgenic lines L-1, L-2 and L-4 showed improved growth compared to the WT ( <xref ref-type="fig" rid="f4"> Fig. 4c </xref> ). The survival rate, rosette diameter, relative electrolyte leakage and proline content of transgenic lines L-1, L-2 and L-4 were significantly higher than those of the WT, one exception, L-4, showed a non-significant increase in relative electrolyte leakage ( <xref ref-type="fig" rid="f4"> Fig. 4dA&#x02013;4dD </xref> ). Proline synthesis and accumulation is involved in the responses to <span class="trait">drought,</span> salt stress, and osmotic stress <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> <xref ref-type="bibr" rid="b24"> 24 </xref> . These results demonstrated that the overexpression of <italic> <span class="gene">GmbZIP110</span> </italic> could improve the response to salinity stress in plant.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">GmbZIP110</span> might have improved plant stress tolerance through regulation of downstream genes. We selected 13 stress responsive genes for further analysis. The relative expression levels of stress-related genes were determined by qPCR. cDNA from the leaves of two-week-old plants grown on &#x000bd; <span class="gene">MS</span> medium was used as templates. The transcript abundance of nine stress-related genes, <italic> UGT </italic> , <italic> <span class="gene">DREB2</span> </italic> , <italic> MYB2 </italic> , <italic> PAD3 </italic> , <italic> RCI3 </italic> , <italic> LTP3 </italic> , <italic> <span class="gene">LCL1</span> </italic> , <italic> NHX1 </italic> and <italic> <span class="gene">SOS1</span> </italic> , increased significantly in the three transgenic <italic> Arabidopsis </italic> lines compared with that in WT; the transcript abundance of two genes, <italic> <span class="gene">LHY</span> </italic> and <italic> RD29B </italic> , decreased significantly ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ). The relative expression of <italic> RCI3 </italic> in the three transgenic <italic> Arabidopsis </italic> lines was ~40. The relative expression of <italic> CCA </italic> and <italic> <span class="gene">P5CS</span> </italic> was similar in transgenic and WT plants. <italic> RCI3 </italic> encodes a peroxidase and its overexpression conferred dehydration and salt tolerance <xref ref-type="bibr" rid="b25"> 25 </xref> . <italic> UGT </italic> , <italic> PAD3 </italic> and <italic> LTP3 </italic> were involved in ABA responses <xref ref-type="bibr" rid="b26"> 26 </xref> <xref ref-type="bibr" rid="b27"> 27 </xref> <xref ref-type="bibr" rid="b28"> 28 </xref> . <italic> <span class="gene">DREB2</span> </italic> expression does not activate downstream genes under normal growth condition, however, its overexpression leads to <span class="trait">drought</span> stress tolerance and slight freezing tolerance <xref ref-type="bibr" rid="b29"> 29 </xref> . <italic> MYB2 </italic> and <italic> <span class="gene">LCL1</span> </italic> have been found to be responsive to stress and/or ABA <xref ref-type="bibr" rid="b30"> 30 </xref> . NHX1 and <span class="gene">SOS1</span> are Na <sup> + </sup> /H <sup> + </sup> exchanger in <italic> Arabidopsis </italic> and previous investigations have suggested them to play an important role in salt tolerance <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b32"> 32 </xref> . <italic> <span class="gene">LHY</span> </italic> and <italic> RD29B </italic> have been found to be responsive to stress and/or ABA <xref ref-type="bibr" rid="b33"> 33 </xref> . These results indicated that <span class="gene">GmbZIP110</span> regulated a common set of genes as well as specific sets of genes for NaCl stress tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants have the capability to respond and adapt to adverse environmental stressors, such as <span class="trait">drought,</span> cold, and salt, through physiological, biochemical, molecular and cellular processes <xref ref-type="bibr" rid="b34"> 34 </xref> <xref ref-type="bibr" rid="b35"> 35 </xref> . These environmental stressors alter the expression of a large number of genes in plants. The expression of <italic> bZIP </italic> gene family members was also altered; these proteins regulate important biological functions, including biotic and abiotic stress responses <xref ref-type="bibr" rid="b13"> 13 </xref> <xref ref-type="bibr" rid="b14"> 14 </xref> <xref ref-type="bibr" rid="b15"> 15 </xref> <xref ref-type="bibr" rid="b16"> 16 </xref> . These <italic> <span class="gene">GmbZIP</span> </italic> TFs showed differential expression in wild and cultivated soybean under normal and NaCl stress ( <xref ref-type="fig" rid="f2"> Fig. 2c </xref> ). <span class="gene">GmbZIP110</span> is one of 99 full-length soybean bZIPs ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ) that is strongly induced by salt ( <xref ref-type="fig" rid="f2"> Fig. 2b&#x02013;d </xref> ), and its transcript levels are highest in nodule tissue and second highest in roots ( <xref ref-type="fig" rid="f2"> Fig. 2a </xref> ). Previous studies identified 47 full-length <italic> <span class="gene">GmbZIP</span> </italic> genes, which were divided into ten groups based on cluster analysis and a comparison with <italic> Arabidopsis bZIP </italic> genes <xref ref-type="bibr" rid="b36"> 36 </xref> , Here, we identified 99 full-length <italic> <span class="gene">GmbZIP</span> </italic> genes that are divided into 13 groups; these findings are identical to those of Corr&#x000ea;a <italic> et al. </italic> (2008). Subgroups F and S are each divided into two groups; subgroup F and subgroup S contain groups VI and IX and groups XI and XII, respectively. A new subgroup, VIII, contains only one AtbZIP <span class="trait">protein.</span> This <span class="trait">protein,</span> AtbZIP62, does not belong to a group in Jakoby&#x02019;s study. Subgroup VIII also contains one <span class="gene">GmbZIP</span> <span class="trait">protein,</span> <span class="gene">GmbZIP39.</span> This change is dependent on genetic distance.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmbZIP110</span> </italic> showed a slightly greater salt tolerance compared to WT plants ( <xref ref-type="fig" rid="f4"> Fig. 4d </xref> ). This finding was consistent with the expression of <italic> OsbZIP23 </italic> , which was induced by dehydration, <span class="trait">high</span> salt stress and exogenous ABA application <xref ref-type="bibr" rid="b37"> 37 </xref> , Improved plant stress tolerance was reported in plants over-expressing <italic> OsbZIP23 </italic> . A recent study showed that the over-expression of <span class="gene">SCOF-1</span> (soybean cold-inducible factor-1), which is a cold-inducible zinc finger <span class="trait">protein</span> gene from soybean, induced the expression of cold-regulated genes and enhanced cold-tolerance in transgenic plants. However, <span class="gene">SCOF-1</span> did not bind directly to either CRT/DRE or ABRE. <span class="gene">SCOF-1</span> up-regulated ABRE-dependent gene expression and mediated cold-regulated gene expression via protein-protein interaction with SGBF-1 (soybean G-box binding factor 1), a G-box binding bZIP transcription factor from soybean <xref ref-type="bibr" rid="b38"> 38 </xref> . Similarly, expression of the intact <span class="gene">AREB1</span> gene on its own is insufficient for the expression of downstream genes under normal growth conditions. No alterations in growth phenotype were observed between the wild type and <span class="gene">AREB1</span> over-expressing plants. Plants over-expressing AREB1QT, an activated form of AREB1, showed ABA hypersensitivity and enhanced <span class="trait">drought</span> tolerance <xref ref-type="bibr" rid="b39"> 39 </xref> . The over-expression of <span class="gene">GmbZIP110</span> was sufficient for the regulation of downstream genes ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ). <span class="gene">GmbZIP110</span> activated reporter genes when used as bait with prey or empty AD plasmids, indicating the presence of a transcriptional activation domain in in this <span class="trait">protein</span> ( <xref ref-type="fig" rid="f3"> Fig. 3a </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic plants over-expressing <span class="gene">GmbZIP110</span> showed higher survival rate, increased rosette diameter, relative electrolyte leakage and proline content compared with those of WT plants ( <xref ref-type="fig" rid="f4"> Fig. 4d </xref> ). These results especially higher relative electrolyte leakage together with the fact that transgenic <italic> Arabidopsis </italic> plants have taken-up significantly less Na <sup> + </sup> ion while no significant change in K <sup> + </sup> ion uptake ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ), might resulted in enhancement of salt tolerance of transgenic <italic> Arabidopsis </italic> lines ( <xref ref-type="fig" rid="f4"> Fig. 4c </xref> ). This suggests that <span class="gene">GmbZIP110</span> maybe participates in other ion mediation, such as Ca <sup> 2+ </sup> , Amtmann <xref ref-type="bibr" rid="b40"> 40 </xref> reported increasing the concentration of Ca <sup> 2+ </sup> in intercellular leads to less absorption of Na <sup> + </sup> . So <span class="gene">GmbZIP110</span> might mediate some other harmless ion up-taking, thus to reduce the toxicity of <span class="trait">high</span> concentration of NaCl.
					</p>
				
					<p>&nbsp;&nbsp;
						The differential expression of stress-related genes in transgenic <italic> Arabidopsis </italic> plants in comparison with WT was evident in the present study ( <xref ref-type="fig" rid="f6"> Fig. 6 </xref> ). These genes included <italic> UGT </italic> , <italic> <span class="gene">DREB2</span> </italic> , <italic> MYB2 </italic> , <italic> PAD3 </italic> , <italic> RCI3 </italic> , <italic> LTP3 </italic> , <italic> <span class="gene">LCL1</span> </italic> , <italic> NHX1 </italic> and <italic> <span class="gene">SOS1</span> </italic> , which significantly increased in relative transcript abundance. <italic> <span class="gene">LHY</span> </italic> and <italic> RD29B </italic> significantly decreased in relative transcript abundance. <span class="gene">DREB2A</span> acts as a trans-acting factor in the signal transduction pathway under dehydration conditions; expression of this <span class="trait">protein</span> was induced by dehydration <xref ref-type="bibr" rid="b41"> 41 </xref> . <italic> RD29B </italic> is a well-studied dehydration-responsive gene; it contains at least one cis-acting element that is involved in the ABA response. The expression of <italic> RD29B </italic> was markedly increased under dehydration, <span class="trait">high</span> salt, and low temperature conditions <xref ref-type="bibr" rid="b42"> 42 </xref> . In the present study, the expression of <italic> RD29B </italic> was significantly reduced. The gene <italic> <span class="gene">P5CS</span> </italic> (delta 1-pyrroline-5-carboxylate synthase), which catalyzes the rate-limiting step in the proline biosynthesis pathway, is induced by salt stress, <span class="trait">drought,</span> and ABA <xref ref-type="bibr" rid="b43"> 43 </xref> . The expression levels of <italic> <span class="gene">P5CS</span> </italic> were similar in transgenic <italic> Arabidopsis </italic> and the WT. <italic> RCI3 </italic> encodes a peroxidase and its overexpression conferred dehydration and salt tolerance <xref ref-type="bibr" rid="b25"> 25 </xref> . <italic> UGT </italic> , <italic> PAD3 </italic> and <italic> LTP3 </italic> were involved in ABA responses <xref ref-type="bibr" rid="b26"> 26 </xref> <xref ref-type="bibr" rid="b27"> 27 </xref> <xref ref-type="bibr" rid="b28"> 28 </xref> . <italic> <span class="gene">DREB2</span> </italic> expression does not activate downstream genes under normal growth condition, however, its overexpression leads to <span class="trait">drought</span> stress tolerance and slight freezing tolerance <xref ref-type="bibr" rid="b29"> 29 </xref> . <italic> MYB2 </italic> and <italic> <span class="gene">LCL1</span> </italic> have been found to be responsive to stress and/or ABA <xref ref-type="bibr" rid="b30"> 30 </xref> . NHX1 and <span class="gene">SOS1</span> are Na <sup> + </sup> /H <sup> + </sup> exchanger in <italic> Arabidopsis </italic> and previous investigations have suggested them to play an important role in salt tolerance <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b32"> 32 </xref> . <italic> <span class="gene">LHY</span> </italic> and <italic> RD29B </italic> have been found to be responsive to stress and/or ABA <xref ref-type="bibr" rid="b33"> 33 </xref> . So we indicated that <span class="gene">GmbZIP110</span> might be involved in ABA dependent pathway and also related to ion transport.
					</p>
				
					<p>&nbsp;&nbsp;
						Young seedlings with unfolded cotyledons were infected at the cotyledonary node and/or the hypocotyl with <italic> Agrobacterium rhizogenes </italic> carrying the pCXSN-GmbZIP110 construct. The injection point was covered immediately after inoculation with a wet vermiculite/soil mixture. The infection sites were kept in a <span class="trait">high</span> humidity environment. Water was applied as required. The seedlings were grown in a greenhouse at 25&#x02009;&#x000b0;C with a 12&#x02009;h photoperiod and approximately 60% humidity. Hairy roots were induced at the inject point after one week; the roots had developed to 2&#x02013;3&#x02009;cm in length at one week after inoculation. In this way, the hairy roots developed at 3 weeks after sowing. The main roots were removed when the emerged hairy roots could support the plants <xref ref-type="bibr" rid="b51"> 51 </xref> . Composite seedlings with one hairy root were transferred to 100&#x02009;ml glass tubes containing &#x000bd; strength Hoagland medium <xref ref-type="bibr" rid="b52"> 52 </xref> . The glass tubes containing composite soybean seedlings were placed in a growth chamber set at 25/22&#x02009;&#x000b0;C day/night temperature with a 12&#x02009;h photoperiod for approximately one week. The Hoagland solution in the glass tubes was replaced every other day. Once the hairy roots had grown to the bottom of the tube, the transformation of hairy roots was verified through PCR analysis. PCR was performed using a forward primer that aligned to the CaMV35S promoter (5&#x02032;-CAATCCCACTATCCTTCGCAAGACC- 3&#x02032;), a <span class="gene">GmbZIP110</span> gene-specific reverse primer and genomic DNA from the hairy roots as a template. One set of soybean composite seedlings carrying the pCXSN empty vector was prepared for use as a control. Transgenic hairy roots of composite seedlings were immersed in &#x000bd; strength Hoagland solution containing 200&#x02009;mM NaCl and grown for one additional week in a growth chamber. The Hoagland solution containing salt was replaced every other day. The data were recorded after one week of stress treatment. The experiment was repeated 3 times.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">GmbZIP110</span> CDS was cloned into the plant expression vector pCXSN using TA cloning. The sequence of the construct was confirmed by PCR and sequencing. The gene was driven by a double CaMV35S promoter. The resulting plasmids were transformed into the <italic> Agrobacterium tumefaciens </italic> strain EHA105, which was used to transform <italic> Arabidopsis </italic> Columbia-0 plants via the flower dip method. Transformed EHA105 was cultured overnight at 28&#x02009;&#x000b0;C with shaking at 150&#x02009;rpm; LB medium containing rifampicin (50&#x02009;mg/L) and kanamycin (50&#x02009;mg/L) was used. When the OD <sub> 600 </sub> of the EHA105 culture reached 1.0, the culture was centrifuged at 5000&#x02009;rpm for 2&#x02009;min. The pellet was suspended and washed twice with the same volume of 5% sucrose. Finally, 0.1% SILWET L-77 was added. A drop of solution was placed on mature flower buds with unopened petals. The plants were kept in a <span class="trait">high</span> humidity environment in the dark for one day. Mature pods of treated Arabidopsis were harvested, and seeds were sown on selected medium containing 100&#x02009;mg/L hygromycin B. Five transformed resistant plants were transplanted into green house, and confirmed by PCR. Line 1, 2, 4 (L-1, L-2, L-4) were <span class="gene">GmbZIP110</span> overexpressing plants, these three lines were continued to develop and homozygous transgenic <italic> Arabidopsis </italic> lines were used for further analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> How to cite this article </bold> : Xu, Z. <italic> et al. </italic> The nuclear <span class="trait">protein</span> <span class="gene">GmbZIP110</span> has transcription activation activity plays important roles in the response to salinity stress in soybean. <italic> Sci. Rep. </italic> <bold> 6 </bold> , 20366; doi: 10.1038/srep20366 (2016).
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Phylogenetic tree containing full length soybean bZIPs, including <span class="gene">GmbZIP110</span> (indicated by an arrow) and <italic> Arabidopsis </italic> bZIP proteins. A multiple sequence alignment containing the amino acid sequences of all bZIP proteins was performed using ClustalX2. The phylogenetic tree was constructed using the Neighbor-Joining method with the MEGA software. The bZIP transcription factors were divided into 13 subfamilies titled I&#x02013;XIII. AtbZIP TFs were divided into 10 subfamilies titled A, B, C, D, E, F, G, H, I and S. <span class="gene">GmbZIP110</span> belongs to the XI(S) subfamily, which contains 13 additional <span class="gene">GmbZIPs</span> and 9 AtbZIPs. ( <bold> b </bold> ) Domain structure of the <span class="gene">GmbZIP110</span> subfamily XI(S): NLS, domain of putative nuclear localization signal under red bar, bZIP domain: basic region/leucine zipper motif of <span class="gene">GmbZIP110</span> <span class="trait">protein</span> under black bar, the different background color of amino acid meaning different conserved sequences. ( <bold> c) </bold> Subcellular localization of the <span class="gene">GmbZIP110</span> <span class="trait">protein</span> in <italic> Arabidopsis </italic> protoplasts. A construct encoding a <span class="gene">GmbZIP110-green</span> fluorescent <span class="trait">protein</span> fusion (p35S::GmbZIP110-GFP) was created using the pJIT166-GFP vector. The <span class="gene">GmbZIP110</span> was introduced without a termination codon, creating an in-frame fusion between the <span class="gene">GmbZIP110</span> CDS and GFP. The fusion construct and the GFP control plasmid (p35S::GFP) were transformed into <italic> Arabidopsis </italic> protoplasts using PEG4000. The transformed <italic> Arabidopsis </italic> protoplasts were incubated for 18&#x02013;24&#x02009;h at room temperature and observed under a confocal fluorescence microscope. The nucleic acid stain Hoechst 33342 was detected using an excitation wavelength of 350 nm and an emission wavelength of 461&#x02009;nm. Chloroplast auto-fluorescence was detected using an excitation wavelength of 480&#x02009;nm and an emission wavelength of 685&#x02009;nm. Scale bars&#x02009;=&#x02009;10000&#x02009;nm.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Transactivation activity of <span class="gene">GmbZIP110.</span> Mated yeast growing on double dropout medium (DDO, SD/-Leu/-Trp) and quadruple dropout medium (QDO, SD/-Ade/-His/-Leu/-Trp) with 125&#x02009;ng/ml Aureobasidin A and 40&#x02009;&#x003bc;g/ml X-&#x003b1;-gal. Growth of cells harboring the BD-53/AD-T pair (positive interaction control), the BD-Lam/AD-T pair (negative control), BD&#x02009;+&#x02009;AD-GmbZIP110, and BD-GmbZIP110&#x02009;+&#x02009;AD. Growth of the transformants on QDO/X/A; the blue color indicates that the corresponding gene has transactivation activity. ( <bold> b </bold> ) DNA-binding ability of <span class="gene">GmbZIP110</span> was analyzed using a yeast one-hybrid system. The growth of cells harboring the effector and reporter plasmids pGADT7-Rec2-53&#x02009;+&#x02009;p53HIS2.1 (positive control), pGADT7-Rec2&#x02009;+&#x02009;p53HIS2.1 (negative control), pGADT7-Rec2&#x02009;+&#x02009;pHIS2.1-P-motif (negative control) and pGADT7-Rec2-bZIP110&#x02009;+&#x02009;pHIS2.1-P-motif. Growth of the transformants on triple dropout medium (TDO, SD/-Trp/-Leu/-His) with 3-AT indicates that the <span class="gene">GmbZIP110</span> <span class="trait">protein</span> can bind to the corresponding cis-DNA element CACGTG.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Environmental Stability of Seed Carbohydrate Profiles in Soybeans Containing Different Alleles of the Raffinose Synthase 2 (RS2) Gene.</h2>
			<h3>PMID:26800264  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bilyeu KD(1), Wiebold WJ(2).
				<br/> 
				年份：2016 Feb 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is important for the <span class="trait">high</span> <span class="trait">protein</span> meal used for livestock feed formulations. Carbohydrates contribute positively or negatively to the potential metabolizable energy in soybean meal. The positive carbohydrate present in soybean meal consists primarily of sucrose, whereas the negative carbohydrate components are the raffinose family of oligosaccharides (RFOs), raffinose and stachyose. Increasing sucrose and decreasing raffinose and stachyose are critical targets to improve soybean. In three recently characterized lines, variant alleles of the soybean raffinose synthase 2 (RS2) gene were associated with increased sucrose and decreased RFOs. The objective of this research was to compare the environmental stability of seed carbohydrates in soybean lines containing wild-type or variant alleles of <span class="gene">RS2</span> utilizing a field location study and a date of planting study. The results define the carbohydrate variation in distinct regional and temporal environments using soybean lines with different alleles of the <span class="gene">RS2</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-Function Relationship of a Novel PR-5 <span class="trait">Protein</span> with Antimicrobial Activity from Soy Hulls.</h2>
			<h3>PMID:26753535  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu C(1), Cheng F(1), Sun Y(1), Ma H(2), Yang X(1)(3).
				<br/> 
				年份：2016 Feb 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An alkaline isoform of the PR-5 <span class="trait">protein</span> (designated <span class="gene">GmOLPc)</span> has been purified from soybean hulls and identified by MALDI-TOF/TOF-MS. <span class="gene">GmOLPc</span> effectively inhibited in vitro the growth of Phytophthora soja spore and Pseudomonas syringae pv glycinea. The antimicrobial activity of <span class="gene">GmOLPc</span> should be mainly ascribed to its <span class="trait">high</span> binding affinity with vesicles composed of DPPG, (1,3)-β-D-glucans, and weak endo-(1,3)-β-D-glucanase activity. From the 3D models, predicted by the homology modeling, <span class="gene">GmOLPc</span> contains an extended negatively charged cleft. The cleft was proved to be a prerequisite for endo-(1,3)-β-D-glucanase activity. Molecular docking revealed that the positioning of linear (1,3)-β-D-glucans in the cleft of <span class="gene">GmOLPc</span> allowed an interaction with Glu83 and Asp101 that were responsible for the hydrolytic cleavage of glucans. Interactions of <span class="gene">GmOLPc</span> with model membranes indicated that <span class="gene">GmOLPc</span> possesses good surface activity which could contribute to its antimicrobial activity, as proved by the behavior of perturbing the integrity of membranes through surface hydrophobic amino acid residues (Phe89 and Phe94).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of <span class="gene">"GmScream"</span> promoters that regulate highly expressing soybean (Glycine max Merr.) genes.</h2>
			<h3>PMID:26706070  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang N(1), McHale LK(2), Finer JJ(3).
				<br/> 
				年份：2015 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To increase our understanding of the regulatory components that control gene expression, it is important to identify, isolate and characterize new promoters. In this study, a group of highly expressed soybean (Glycine max Merr.) genes, which we have named <span class="gene">"GmScream",</span> were first identified from RNA-Seq data. The promoter regions were then identified, cloned and fused with the coding region of the green fluorescent <span class="trait">protein</span> (gfp) gene, for introduction and analysis in different tissues using 3 tools for validation. Approximately half of the <span class="gene">GmScream</span> promoters identified showed levels of GFP expression comparable to or higher than the Cauliflower Mosaic Virus 35S (35S) promoter. Using transient expression in lima bean cotyledonary tissues, the strongest <span class="gene">GmScream</span> promoters gave over 6-fold higher expression than the 35S promoter while several other <span class="gene">GmScream</span> promoters showed 2- to 3-fold higher expression. The two highest expressing promoters, <span class="gene">GmScreamM4</span> and <span class="gene">GmScreamM8,</span> regulated two different elongation factor 1A genes in soybean. In stably transformed soybean tissues, GFP driven by the <span class="gene">GmScreamM4</span> or <span class="gene">GmScreamM8</span> promoter exhibited constitutive <span class="trait">high</span> expression in most tissues with preferentially higher expression in proliferative embryogenic tissues, procambium, vascular tissues, root tips and young embryos. Using deletion analysis of the promoter, two proximal regions of the <span class="gene">GmScreamM8</span> promoter were identified as contributing significantly to <span class="trait">high</span> levels of gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cooperative <span class="trait">Protein</span> Folding by Two <span class="trait">Protein</span> Thiol Disulfide Oxidoreductases and 1 in Soybean.</h2>
			<h3>PMID:26645455  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matsusaki M(1), Okuda A(1), Masuda T(1), Koishihara K(1), Mita R(1), Iwasaki K(1), Hara K(1), Naruo Y(1), Hirose A(1), Tsuchi Y(1), Urade R(2).
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most proteins produced in the endoplasmic reticulum (ER) of eukaryotic cells fold via disulfide formation (oxidative folding). Oxidative folding is catalyzed by <span class="trait">protein</span> disulfide isomerase (PDI) and PDI-related ER <span class="trait">protein</span> thiol disulfide oxidoreductases (ER oxidoreductases). In yeast and mammals, ER oxidoreductin-1s (Ero1s) supply oxidizing equivalent to the active centers of PDI. In this study, we expressed recombinant soybean Ero1 <span class="gene">(GmERO1a)</span> and found that <span class="gene">GmERO1a</span> oxidized multiple soybean ER oxidoreductases, in contrast to mammalian Ero1s having a <span class="trait">high</span> specificity for PDI. One of these ER oxidoreductases, <span class="gene">GmPDIM,</span> associated in vivo and in vitro with <span class="gene">GmPDIL-2,</span> was unable to be oxidized by <span class="gene">GmERO1a.</span> We therefore pursued the possible cooperative oxidative folding by <span class="gene">GmPDIM,</span> <span class="gene">GmERO1a,</span> and <span class="gene">GmPDIL-2</span> in vitro and found that <span class="gene">GmPDIL-2</span> synergistically accelerated oxidative refolding. In this process, <span class="gene">GmERO1a</span> preferentially oxidized the active center in the A': domain among the A: , A': , and B: domains of <span class="gene">GmPDIM.</span> A disulfide bond introduced into the active center of the A': domain of <span class="gene">GmPDIM</span> was shown to be transferred to the active center of the A: domain of <span class="gene">GmPDIM</span> and the A: domain of <span class="gene">GmPDIM</span> directly oxidized the active centers of both the A: or A': domain of <span class="gene">GmPDIL-2.</span> Therefore, we propose that the relay of an oxidizing equivalent from one ER oxidoreductase to another may play an essential role in cooperative oxidative folding by multiple ER oxidoreductases in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel cis-acting element in the <span class="gene">GmERF3</span> promoter contributes to inducible gene expression in soybean and tobacco after wounding.</h2>
			<h3>PMID:26518427  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hernandez-Garcia CM(1)(2), Finer JJ(3).
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Using in silico and functional analyses, we cloned and validated the expression profile of an inducible soybean promoter <span class="gene">(GmERF3)</span> along with its novel wound-induced and delayed expression (WIDE) element. Promoters and their contributing promoter elements are the main regulators of gene expression at the transcriptional level. Although the Ethylene Response Factor (ERF) gene family is one of the most well-studied stress-responsive gene families in plants, their promoter regions have received little attention. In this study, we investigated the expression patterns driven by the soybean (Glycine max) <span class="gene">GmERF3</span> promoter and its cis-acting elements in soybean and tobacco. Transcriptomic data revealed that the native <span class="gene">GmERF3</span> gene was differentially expressed in organs and tissues of plants. In transgenic soybeans containing a 1.3 kb <span class="gene">GmERF3</span> promoter fused to the green fluorescent <span class="trait">protein</span> (gfp) gene, organ- and tissue-specificity was observed in untreated plants while mechanical wounding led to induction of GFP expression. Further in silico and in planta analyses of the <span class="gene">GmERF3</span> promoter sequence in soybean revealed different cis-acting elements, including a novel cis-acting element, which contributed to increased expression, 1-2 days after mechanical wounding. We have named this DNA motif the wound-induced and delayed expression element (GGATTCAAGTTTAACC). A synthetic promoter containing a tetrameric repeat of this element showed <span class="trait">high</span> but late wound-induced GFP expression in leaves of transgenic tobacco. Our study expands the toolbox of inducible promoters and promoter elements for potential use in basic and applied research.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mechanisms of Soybean Roots' Tolerances to Salinity Revealed by Proteomic and Phosphoproteomic Comparisons Between Two Cultivars.</h2>
			<h3>PMID:26407991  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pi E(1), Qu L(2), Hu J(3), Huang Y(2), Qiu L(4), Lu H(5), Jiang B(6), Liu C(2), Peng T(2), Zhao Y(2), Wang H(2), Tsai SN(7), Ngai S(8), Du L(1).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Understanding molecular mechanisms underlying plant salinity tolerance provides valuable knowledgebase for effective crop improvement through genetic engineering. Current proteomic technologies, which support reliable and high-throughput analyses, have been broadly used for exploring sophisticated molecular networks in plants. In the current study, we compared phosphoproteomic and proteomic changes in roots of different soybean seedlings of a salt-tolerant cultivar (Wenfeng07) and a salt-sensitive cultivar (Union85140) induced by salt stress. The root samples of Wenfeng07 and Union85140 at three-trifoliate stage were collected at 0 h, 0.5 h, 1 h, 4 h, 12 h, 24 h, and 48 h after been treated with 150 mm NaCl. LC-MS/MS based phosphoproteomic analysis of these samples identified a total of 2692 phosphoproteins and 5509 phosphorylation sites. Of these, 2344 phosphoproteins containing 3744 phosphorylation sites were quantitatively analyzed. Our results showed that 1163 phosphorylation sites were differentially phosphorylated in the two compared cultivars. Among them, 10 MYB/MYB transcription factor like proteins were identified with fluctuating phosphorylation modifications at different time points, indicating that their crucial roles in regulating flavonol accumulation might be mediated by phosphorylated modifications. In addition, the <span class="trait">protein</span> expression profiles of these two cultivars were compared using LC MS/MS based shotgun proteomic analysis, and expression pattern of all the 89 differentially expressed proteins were independently confirmed by qRT-PCR. Interestingly, the enzymes involved in chalcone metabolic pathway exhibited positive correlations with salt tolerance. We confirmed the functional relevance of chalcone synthase, chalcone isomerase, and cytochrome <span class="gene">P450</span> monooxygenase genes using soybean composites and Arabidopsis thaliana mutants, and found that their salt tolerance were positively regulated by chalcone synthase, but was negatively regulated by chalcone isomerase and cytochrome <span class="gene">P450</span> monooxygenase. A novel salt tolerance pathway involving chalcone metabolism, mostly mediated by phosphorylated MYB transcription factors, was proposed based on our findings. (The mass spectrometry raw data are available via ProteomeXchange with identifier PXD002856).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing cyclic electron flow is related to Na+ sequestration into vacuoles for salt tolerance in soybean.</h2>
			<h3>PMID:26276865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：He Y(1), Fu J(2), Yu C(2), Wang X(2), Jiang Q(2), Hong J(3), Lu K(4), Xue G(5), Yan C(6), James A(5), Xu L(2), Chen J(6), Jiang D(7).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In land plants, the NAD(P)H dehydrogenase (NDH) complex reduces plastoquinones and drives cyclic electron flow (CEF) around PSI. It also produces extra ATP for photosynthesis and improves plant fitness under conditions of abiotic environmental stress. To elucidate the role of CEF in salt tolerance of the photosynthetic apparatus, Na(+) concentration, chlorophyll fluorescence, and expression of NDH B and H subunits, as well as of genes related to cellular and vacuolar Na(+) transport, were monitored. The salt-tolerant Glycine max (soybean) variety S111-9 exhibited much higher CEF activity and ATP accumulation in light than did the salt-sensitive variety Melrose, but similar leaf Na(+) concentrations under salt stress. In S111-9 plants, <span class="gene">ndhB</span> and <span class="gene">ndhH</span> were highly up-regulated under salt stress and their corresponding proteins were maintained at <span class="trait">high</span> levels or increased significantly. Under salt stress, S111-9 plants accumulated Na(+) in the vacuole, but Melrose plants accumulated Na(+) in the chloroplast. Compared with Melrose, S111-9 plants also showed higher expression of some genes associated with Na(+) transport into the vacuole and/or cell, such as genes encoding components of the CBL10 (calcineurin B-like <span class="trait">protein</span> 10)-CIPK24 (CBL-interacting <span class="trait">protein</span> kinase 24)-NHX (Na(+)/H(+) antiporter) and CBL4 (calcineurin B-like <span class="trait">protein</span> 4)-CIPK24-SOS1 (salt overly sensitive 1) complexes. Based on the findings, it is proposed that enhanced NDH-dependent CEF supplies extra ATP used to sequester Na(+) in the vacuole. This reveals an important mechanism for salt tolerance in soybean and provides new insights into plant <span class="trait">resistance</span> to salt stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of different combinations of soybean-maize silage on its chemical composition, nutrient intake, degradability, and performance of Pelibuey lambs.</h2>
			<h3>PMID:26265017  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Martínez-García CG(1), Valencia-Núñez K(2), Bastida-López J(2), Estrada-Flores JG(1), Miranda-de la Lama GC(3), Cruz-Monterrosa RG(3), Rayas-Amor AA(4).
				<br/> 
				年份：2015 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sheep raising in the state of Guerrero, México, is a primary activity that is worth about US$3,251,931 annually. The objective of the present study was to evaluate the chemical composition, degradability, nutrient intake, and animal performance of Pelibuey lambs fed on different combinations of maize-soybean silages. Twenty-one combinations of maize silage (MS) and soybean silage (SS) were evaluated at day 45 post-ensiling; in each combination, <span class="gene">MS</span> was replaced by 5 % of SS. The 21 combinations were analysed for crude <span class="trait">protein</span> (CP) and chemical composition. In order to obtain a statistical criterion of potential treatments for the animal feeding test, a cluster analysis was performed based on the CP contents of all combinations at day 45 post-ensiling. From cluster analysis, four treatments were selected T1 = 100-0 % (MS/SS), T8 = 65-35 %, T12 = 45-55 %, and T16 = 25-75 %. Results indicated that cluster analysis was useful for identifying the potential treatments for animal feeding based on the crude <span class="trait">protein</span> content. The dry matter (DM), organic matter (OM), and acid detergent lignin (ADL) contents did not declined significantly (P &gt; 0.05) during the fermentation of silages but CP content decreased from day 0 to 45 post-ensiling. The treatment with the highest estimated microbial crude <span class="trait">protein</span> synthesis was T8 and it showed the highest metabolizable energy intake, <span class="trait">high</span> feed efficiency with a forage-concentrate ratio of 84:16.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSBH1, a homeobox transcription factor gene, relates to growth and development and involves in response to <span class="trait">high</span> temperature and humidity stress in soybean.</h2>
			<h3>PMID:26205508  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shu Y(1)(2), Tao Y(1), Wang S(1), Huang L(1), Yu X(1), Wang Z(1), Chen M(1), Gu W(3), Ma H(4).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: <span class="gene">GmSBH1</span> involves in response to <span class="trait">high</span> temperature and humidity stress. Homeobox transcription factors are key switches that control plant development processes. Glycine max <span class="gene">H1</span> Sbh1 <span class="gene">(GmSBH1)</span> was the first homeobox gene isolated from soybean. In the present study, the full ORF of <span class="gene">GmSBH1</span> was isolated, and the encoded <span class="trait">protein</span> was found to be a typical class I KNOX homeobox transcription factor. Subcellular localization and transcriptional activation assays showed that <span class="gene">GmSBH1</span> is a nuclear <span class="trait">protein</span> and possesses transcriptional activation activity in the homeodomain. The KNOX1 domain was found to play a clear role in suppressing the transcriptional activation activity of <span class="gene">GmSBH1.</span> <span class="gene">GmSBH1</span> showed different expression levels among different soybean tissues and was involved in response to <span class="trait">high</span> temperature and humidity (HTH) stress in developing soybean seeds. The overexpression of <span class="gene">GmSBH1</span> in Arabidopsis altered leaf and stoma phenotypes and enhanced seed tolerance to HTH stress. Overall, our results indicated that <span class="gene">GmSBH1</span> is involved in growth, development, and enhances tolerance to pre-harvest seed deterioration caused by HTH stress in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Oil Is More Obesogenic and Diabetogenic than Coconut Oil and Fructose in Mouse: Potential Role for the Liver.</h2>
			<h3>PMID:26200659  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deol P(1), Evans JR(1), Dhahbi J(2), Chellappa K(1), Han DS(1), Spindler S(2), Sladek FM(1).
				<br/> 
				年份：2015 Jul 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The obesity epidemic in the U.S. has led to extensive research into potential contributing dietary factors, especially fat and fructose. Recently, increased consumption of soybean oil, which is rich in polyunsaturated fatty acids (PUFAs), has been proposed to play a causal role in the epidemic. Here, we designed a series of four isocaloric diets (HFD, SO-HFD, F-HFD, F-SO-HFD) to investigate the effects of saturated versus unsaturated fat, as well as fructose, on obesity and diabetes. C57/BL6 male mice fed a diet moderately <span class="trait">high</span> in fat from coconut oil and soybean oil (SO-HFD, 40% kcal total fat) showed statistically significant increases in weight gain, adiposity, diabetes, glucose intolerance and insulin <span class="trait">resistance</span> compared to mice on a diet consisting primarily of coconut oil (HFD). They also had fatty livers with hepatocyte ballooning and very large lipid droplets as well as shorter colonic crypt length. While the <span class="trait">high</span> fructose diet (F-HFD) did not cause as much obesity or diabetes as SO-HFD, it did cause rectal prolapse and a very fatty liver, but no balloon injury. The coconut oil diet (with or without fructose) increased spleen weight while fructose in the presence of soybean oil increased kidney weight. Metabolomics analysis of the liver showed an increased accumulation of PUFAs and their metabolites as well as γ-tocopherol, but a decrease in cholesterol in SO-HFD. Liver transcriptomics analysis revealed a global dysregulation of cytochrome <span class="gene">P450</span> (Cyp) genes in SO-HFD versus HFD livers, most notably in the Cyp3a and Cyp2c families. Other genes involved in obesity (e.g., Cidec, Cd36), diabetes (Igfbp1), inflammation (Cd63), mitochondrial function (Pdk4) and cancer (H19) were also upregulated by the soybean oil diet. Taken together, our results indicate that in mice a diet <span class="trait">high</span> in soybean oil is more detrimental to metabolic health than a diet <span class="trait">high</span> in fructose or coconut oil.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Bradyrhizobium elkanii Genes Involved in Incompatibility with Soybean Plants Carrying the <span class="gene">Rj4</span> Allele.</h2>
			<h3>PMID:26187957  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Faruque OM(1), Miwa H(1), Yasuda M(1), Fujii Y(1), Kaneko T(2), Sato S(3), Okazaki S(4).
				<br/> 
				年份：2015 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Symbioses between leguminous plants and soil bacteria known as rhizobia are of great importance to agricultural production and nitrogen cycling. While these mutualistic symbioses can involve a wide range of rhizobia, some legumes exhibit incompatibility with specific strains, resulting in ineffective nodulation. The formation of nodules in soybean plants (Glycine max) is controlled by several host genes, which are referred to as Rj genes. The soybean cultivar BARC2 carries the <span class="gene">Rj4</span> gene, which restricts nodulation by specific strains, including Bradyrhizobium elkanii USDA61. Here we employed transposon mutagenesis to identify the genetic locus in USDA61 that determines incompatibility with soybean varieties carrying the <span class="gene">Rj4</span> allele. Introduction of the Tn5 transposon into USDA61 resulted in the formation of nitrogen fixation nodules on the roots of soybean cultivar BARC2 (Rj4 Rj4). Sequencing analysis of the sequence flanking the Tn5 insertion revealed that six genes encoding a putative histidine kinase, transcriptional regulator, DNA-binding transcriptional activator, helix-turn-helix-type transcriptional regulator, phage shock <span class="trait">protein,</span> and cysteine protease were disrupted. The cysteine protease mutant had a <span class="trait">high</span> degree of similarity with the type 3 effector <span class="trait">protein</span> XopD of Xanthomonas campestris. Our findings shed light on the diverse and complicated mechanisms that underlie these highly host-specific interactions and indicate the involvement of a type 3 effector in <span class="gene">Rj4</span> nodulation restriction, suggesting that <span class="gene">Rj4</span> incompatibility is partly mediated by effector-triggered immunity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of type of fiber, site of fermentation, and method of analysis on digestibility of soluble and insoluble fiber in rabbits.</h2>
			<h3>PMID:26115273  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abad-Guamán R, Carabaño R, Gómez-Conde MS, García J.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of type of fiber, site of fermentation, method for quantifying insoluble and soluble dietary fiber, and their correction for intestinal mucin on fiber digestibility were examined in rabbits. Three diets differing in soluble fiber were formulated (8.5% soluble fiber, on DM basis, in the low soluble fiber [LSF] diet; 10.2% in the medium soluble fiber [MSF] diet; and 14.5% in the <span class="trait">high</span> soluble fiber [HSF] diet). They were obtained by replacing half of the dehydrated alfalfa in the MSF diet with a mixture of beet and apple pulp (HSF diet) or with a mix of oat hulls and soybean <span class="trait">protein</span> (LSF diet). Thirty rabbits with ileal T-cannulas were used to determine ileal and fecal digestibility. Cecal digestibility was determined by difference between fecal and ileal digestibility. Insoluble fiber was measured as NDF, insoluble dietary fiber (IDF), and in vitro insoluble fiber, whereas soluble fiber was calculated as the difference between total dietary fiber (TDF) and NDF (TDF-NDF), IDF (TDF-IDF), and in vitro insoluble fiber (TDF-in vitro insoluble fiber). The intestinal mucin content was used to correct the TDF and soluble fiber digestibility. Ileal and fecal concentration of mucin increased from the LSF to the <span class="gene">HSF</span> diet group (P &lt; 0.01). Once corrected for intestinal mucin, ileal and fecal digestibility of TDF and soluble fiber increased whereas cecal digestibility decreased (P &lt; 0.01). Ileal digestibility of TDF increased from the LSF to the <span class="gene">HSF</span> diet group (12.0 vs. 28.1%; P &lt; 0.01), with no difference in the cecum (26.4%), resulting in a higher fecal digestibility from the LSF to the <span class="gene">HSF</span> diet group (P &lt; 0.01). Ileal digestibility of insoluble fiber increased from the LSF to the <span class="gene">HSF</span> diet group (11.3 vs. 21.0%; P &lt; 0.01), with no difference in the cecum (13.9%) and no effect of fiber method, resulting in a higher fecal digestibility for rabbits fed the <span class="gene">HSF</span> diet compared with the MSF and LSF diet groups (P &lt; 0.01). Fecal digestibility of NDF was higher compared with IDF or in vitro insoluble fiber (P &lt; 0.01). Ileal soluble fiber digestibility was higher for the <span class="gene">HSF</span> than for the LSF diet group (43.6 vs. 14.5%;P &lt; 0.01) and fiber method did not affect it. Cecal soluble fiber digestibility decreased from the LSF to the <span class="gene">HSF</span> diet group (72.1 vs. 49.2%; P &lt; 0.05). The lowest cecal and fecal soluble fiber digestibility was measured using TDF-NDF (P &lt; 0.01). In conclusion, a correction for intestinal mucin is necessary for ileal TDF and soluble fiber digestibility whereas the selection of the fiber method has a minor relevance. The inclusion of sugar beet and apple pulp increased the amount of TDF fermented in the small intestine.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physcomitrella patens auxin conjugate synthetase (GH3) double knockout mutants are more resistant to Pythium infection than wild type.</h2>
			<h3>PMID:26102574  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mittag J(1), Šola I(2), Rusak G(3), Ludwig-Müller J(4).
				<br/> 
				年份：2015 Jul 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Auxin homeostasis is involved in many different plant developmental and stress responses. The auxin amino acid conjugate synthetases belonging to the <span class="gene">GH3</span> family play major roles in the regulation of free indole-3-acetic acid (IAA) levels and the moss Physcomitrella patens has two <span class="gene">GH3</span> genes in its genome. A role for IAA in several angiosperm--pathogen interactions was reported, however, in a moss--oomycete pathosystem it had not been published so far. Using <span class="gene">GH3</span> double knockout lines we have investigated the role of auxin homeostasis during the infection of P. patens with the two oomycete species, Pythium debaryanum and Pythium irregulare. We show that infection with P. debaryanum caused stronger disease symptoms than with P. irregulare. Also, P. patens lines harboring fusion constructs of an auxin-inducible promoter from soybean <span class="gene">(GmGH3)</span> with a reporter (ß-glucuronidase) showed higher promoter induction after P. debaryanum infection than after P. irregulare, indicating a differential induction of the auxin response. Free IAA was induced upon P. debaryanum infection in wild type by 1.6-fold and in two <span class="gene">GH3</span> double knockout (GH3-doKO) mutants by 4- to 5-fold. All GH3-doKO lines showed a reduced disease symptom progression compared to wild type. Since P. debaryanum can be inhibited in growth on medium containing IAA, these data might indicate that endogenous <span class="trait">high</span> auxin levels in P. patens GH3-doKO mutants lead to higher <span class="trait">resistance</span> against the oomycete.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Linkage mapping, molecular cloning and functional analysis of soybean gene <italic> Fg3 </italic> encoding flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase</h2>
			<h3>PMID:26002063</h3>
				

				
					<h3>PMCID:4494776</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Di </surname>  <given-names> Shaokang </given-names>  </name>  <address>  <email> dishaokang@affrc.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Fan </given-names>  </name>  <address>  <email> yanfan@affrc.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodas </surname>  <given-names> Felipe Rojas </given-names>  </name>  <address>  <email> felipe-r@hotmail.es </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodriguez </surname>  <given-names> Tito O </given-names>  </name>  <address>  <email> rodtor@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Murai </surname>  <given-names> Yoshinori </given-names>  </name>  <address>  <email> murai@kahaku.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Iwashina </surname>  <given-names> Tsukasa </given-names>  </name>  <address>  <email> iwashina@kahaku.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sugawara </surname>  <given-names> Satoko </given-names>  </name>  <address>  <email> satoko.sugawra@riken.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mori </surname>  <given-names> Tetsuya </given-names>  </name>  <address>  <email> tetsuya.mori@riken.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakabayashi </surname>  <given-names> Ryo </given-names>  </name>  <address>  <email> ryo.nakabayashi@riken.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yonekura-Sakakibara </surname>  <given-names> Keiko </given-names>  </name>  <address>  <email> yskeiko@riken.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saito </surname>  <given-names> Kazuki </given-names>  </name>  <address>  <email> kazuki.saito@riken.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Takahashi </surname>  <given-names> Ryoji </given-names>  </name>  <address>  <email> masako@affrc.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <aff id="Aff1">  <label/> Graduate School of Life and Environmental Sciences, University of Tsukuba, Tsukuba, Ibaraki 305-8518 Japan  </aff>  <aff id="Aff2">  <label/> Department of Botany, National Museum of Nature and Science, Tsukuba, Ibaraki 305-0005 Japan  </aff>  <aff id="Aff3">  <label/> RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa 230-0045 Japan  </aff>  <aff id="Aff4">  <label/> National Institute of Crop Science, Tsukuba, Ibaraki 305-8518 Japan  </aff>  <aff id="Aff5">  <label/> Current address: Universidad Cat&#x000f3;lica de Oriente, Rionegro-Antioquia, Colombia  </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Flavonol glycosides (FGs) are major components of soybean leaves and there are substantial differences in FG composition among genotypes. The first objective of this study was to identify genes responsible for FG biosynthesis and to locate them in the soybean genome. The second objective was to clone the candidate genes and to verify their function. Recombinant inbred lines (RILs) were developed from a cross between cultivars Nezumisaya and Harosoy. RESULTS: HPLC comparison with authentic samples suggested that FGs having glucose at the 2″-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Nezumisaya, whereas FGs of Harosoy were devoid of 2″-glucose. Conversely, FGs having glucose at the 6″-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Harosoy, whereas these FGs were absent in Nezumisaya. Genetic analysis suggested that two genes control the pattern of attachment of these sugar moieties in FGs. One of the genes may be responsible for attachment of glucose to the 2″-position, probably encoding for a flavonol 3-O-glucoside/galactoside (1 → 2) glucosyltransferase. Nezumisaya may have a dominant whereas Harosoy may have a recessive allele of the gene. Based on SSR analysis, linkage mapping and genome database survey, we cloned a candidate gene designated as <span class="gene">GmF3G2″Gt</span> in the molecular linkage group <span class="gene">C2</span> (chromosome 6). The open reading frame of <span class="gene">GmF3G2″Gt</span> is 1380 bp long encoding 459 amino acids with four amino acid substitutions among the cultivars. The <span class="gene">GmF3G2″Gt</span> recombinant <span class="trait">protein</span> converted kaempferol 3-O-glucoside to kaempferol 3-O-sophoroside. <span class="gene">GmF3G2″Gt</span> of Nezumisaya showed a broad activity for kaempferol/quercetin 3-O-glucoside/galactoside derivatives but it did not glucosylate kaempferol 3-O-rhamnosyl-(1 → 4)-[rhamnosyl-(1 → 6)-glucoside] and 3-O-rhamnosyl-(1 → 4)-[glucosyl-(1 → 6)-glucoside]. CONCLUSION: <span class="gene">GmF3G2″Gt</span> encodes a flavonol 3-O-glucoside/galactoside (1 → 2) glucosyltransferase and corresponds to the Fg3 gene. <span class="gene">GmF3G2″Gt</span> was designated as UGT79B30 by the UGT Nomenclature Committee. Based on substrate specificity of <span class="gene">GmF3G2″Gt,</span> 2″-glucosylation of flavonol 3-O-glycoside may be irreconcilable with 4″-glycosylation in soybean leaves.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						HPLC comparison with authentic samples suggested that FGs having glucose at the 2&#x02033;-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Nezumisaya, whereas FGs of Harosoy were devoid of 2&#x02033;-glucose. Conversely, FGs having glucose at the 6&#x02033;-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Harosoy, whereas these FGs were absent in Nezumisaya. Genetic analysis suggested that two genes control the pattern of attachment of these sugar moieties in FGs. One of the genes may be responsible for attachment of glucose to the 2&#x02033;-position, probably encoding for a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase. Nezumisaya may have a dominant whereas Harosoy may have a recessive allele of the gene. Based on SSR analysis, linkage mapping and genome database survey, we cloned a candidate gene designated as <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> in the molecular linkage group <span class="gene">C2</span> (chromosome 6). The open reading frame of <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> is 1380 bp long encoding 459 amino acids with four amino acid substitutions among the cultivars. The <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> recombinant <span class="trait">protein</span> converted kaempferol 3- <italic> O </italic> -glucoside to kaempferol 3- <italic> O </italic> -sophoroside. <span class="gene">GmF3G2&#x02033;Gt</span> of Nezumisaya showed a broad activity for kaempferol/quercetin 3- <italic> O </italic> -glucoside/galactoside derivatives but it did not glucosylate kaempferol 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[rhamnosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside] and 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[glucosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside].
					</p>
				
					<p>&nbsp;&nbsp;
						Leaves of soybean ( <italic> Glycine max </italic> (L.) Merr.) contain a variety of flavonol glycosides (FGs) that are derivatives of quercetin and kaempferol [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. Buzzell and Buttery [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ] proposed four flavonol glycoside alleles, viz., <italic> Fg1 </italic> (&#x003b2;(1&#x02013;6)-glucoside present), <italic> Fg2 </italic> (&#x003b1;(1&#x02013;6)-rhamnoside present), <italic> Fg3 </italic> (&#x003b2;(1&#x02013;2)-glucoside present), and <italic> Fg4 </italic> (&#x003b1;(1&#x02013;2)-rhamnoside present). These alleles are defined by their ability to bind glucose or rhamnose at either position 2&#x02033; or 6&#x02033; to the glucose moiety that is bound to the 3-position of flavonols. Later, Buzzell and Buttery [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ] reported a new allele of the <italic> Fg2 </italic> locus, resulting in a series of alleles, <italic> Fg2-a </italic> , <italic> Fg2-b </italic> and <italic> fg2. Fg3 </italic> and <italic> Fg4 </italic> are linked with a recombination frequency of 12% in the molecular linkage group <span class="gene">C2</span> (chromosome 6) [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. Plants with the <italic> Fg1Fg3 </italic> alleles have a lower rate of photosynthesis, lower leaf chlorophyll concentration, lower leaf weight, and lower seed <span class="trait">yield</span> [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. Further, <italic> Fg1 </italic> and <italic> Fg3 </italic> control waviness of leaf margins in soybean [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Rojas Rodas et al. [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ] performed genetic analysis using RILs derived from a cross between Koganejiro and Kitakomachi which are soybean cultivars with gray pubescence. FGs of Koganejiro had rhamnose at the 6&#x02033;-position of glucose or galactose that is bound to the 3-position of kaempferol, whereas FGs of Kitakomachi were devoid of rhamnose. The presence of 6&#x02033;-rhamnose was controlled by a single gene. They cloned a candidate gene, <italic> <span class="gene">GmF3G6&#x02033;Rt</span> </italic> , in the molecular linkage group O (chromosome 10). The recombinant <italic> <span class="gene">GmF3G6&#x02033;Rt</span> </italic> <span class="trait">protein</span> converted UDP-rhamnose and kaempferol 3- <italic> O </italic> -glucoside or kaempferol 3- <italic> O </italic> -galactoside to kaempferol 3- <italic> O </italic> -rutinoside or kaempferol 3- <italic> O </italic> -robinobioside, proving that <italic> <span class="gene">GmF3G6&#x02033;Rt</span> </italic> encodes a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;6) rhamnosyltransferase and corresponds to the <italic> Fg2 </italic> gene. Thus, either glucose or galactose was attached to the 3-position of kaempferol partially contradicting Buttery and Buzzell [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ] who reported that only glucose was attached to the 3-position. In addition, FGs having rhamnose at the 4&#x02033;-position of 3- <italic> O </italic> -galactose, kaempferol 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[rhamnosyl-(1&#x02009;&#x02192;&#x02009;6)-galactoside] and kaempferol 3- <italic> O </italic> -rhamnosyl-(1&#x02009; <bold> &#x02192; </bold> &#x02009;4)-[glucosyl-(1&#x02009; <bold> &#x02192; </bold> &#x02009;6)-galactoside] have been identified in the leaves of soybean [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR11"> 11 </xref> ], suggesting the existence of flavonol 3- <italic> O </italic> -galactoside (1&#x02009;&#x02192;&#x02009;4) rhamnosyltransferase in soybean. Hence, the genetic control of FG biosynthesis proposed by Buttery and Buzzell [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ] should be revised. Overall, glucose or galactose is attached to the 3-position of kaempferol or quercetin in the biosynthesis of FGs in soybean leaves. Glucose can be attached to the 2&#x02033;- or 6&#x02033;-positions of glucose or galactose whereas rhamnose can be attached to the 2&#x02033;-, 4&#x02033;- or 6&#x02033;-positions, resulting in a wide variety of FGs [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Nucleotide sequences were determined with the BigDye terminator cycle method using an ABI3100 Genetic Analyzer (Applied Biosystems). Nucleotide sequences and the putative amino acid translations were analyzed with the BLAST program [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. Intron/exon structure of the gene was estimated based on the comparison between the cDNA sequence and the corresponding genome sequences of Williams 82 deposited in the soybean genome database. Sequence alignment was performed with ClustalW ( <ext-link ext-link-type="uri" xlink:href="http://clustalw.ddbj.nig.ac.jp/index.php?lang=ja"> http://clustalw.ddbj.nig.ac.jp/index.php?lang=ja </ext-link> ) using default settings. The amino acid sequences were aligned using ClustalW and the alignment was used to construct a phylogenetic tree using the neighbor-joining method with MEGA5 version 5.2.2 ( <ext-link ext-link-type="uri" xlink:href="http://www.megasoftware.net/"> http://www.megasoftware.net/ </ext-link> ) [ <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. Bootstrap test of 1000 replications was performed. Using the structure of the grape flavonoid 3- <italic> O- </italic> glucosyltransferase (VvGT1, <span class="trait">protein</span> ID: 2C1Z) [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] as a template, homology modeling was conducted using GASH (Genetic-algorithm ASH) [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] program at the <span class="trait">Protein</span> Data Bank Japan ( <ext-link ext-link-type="uri" xlink:href="http://sysimm.ifrec.osaka-u.ac.jp/ash_service/"> http://sysimm.ifrec.osaka-u.ac.jp/ash_service/ </ext-link> ). A model of <span class="gene">GmF3G2"Gt-b</span> was superimposed on 2C1Z using CueMol software ( <ext-link ext-link-type="uri" xlink:href="http://www.cuemol.org/en/"> http://www.cuemol.org/en/ </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The entire coding region of <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> was amplified from cDNA of Harosoy and Nezumisaya by PCR using the KOD -Plus- DNA polymerase (Toyobo) with <span class="trait">high</span> PCR fidelity and primers containing enzyme sites for <italic> Sac </italic> I and <italic> Xho </italic> I (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The PCR mixture contained 30 ng of genomic DNA, 10 pmol of each primer, 5 pmol of nucleotides, 2 mM of MgSO <sub> 4 </sub> and 0.5 unit of KOD &#x02013;Plus- in 1 &#x000d7; KOD &#x02013;Plus- Buffer supplied by the manufacturer in a total volume of 25 &#x003bc;l. PCR conditions were identical to those in a previous report [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. The PCR amplicon was digested with <italic> Sac </italic> I and <italic> Xho </italic> I and was cloned into the pCold ProS2 vector (Takara Bio). <span class="gene">GmF3G2&#x02033;Gt</span> proteins were expressed and purified as described previously [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">GmF3G2&#x02033;Gt</span> recombinant <span class="trait">protein</span> of Nezumisaya <span class="gene">(GmF3G2&#x02033;Gt-a)</span> and Harosoy <span class="gene">(GmF3G2&#x02033;Gt-b)</span> were expressed in <italic> E. coli </italic> as a His/ProS2 fusion and purified. The <span class="gene">GmF3G2&#x02033;Gt</span> proteins were used for enzymatic assays after cleavage of the His/ProS2 tag. <span class="gene">GmF3G2&#x02033;G-a</span> and <span class="gene">GmF3G2&#x02033;G-b</span> converted kaempferol 3- <italic> O </italic> -glucoside to kaempferol 3- <italic> O </italic> -sophoroside as confirmed by comparison of retention time, UV spectra and MS/MS ionization with the standard compound (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> , Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Figure S2). <span class="gene">GmF3G2&#x02033;Gt-a</span> showed a broad activity for kaempferol/quercetin 3- <italic> O </italic> -glucoside/galactoside derivatives (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). However, <span class="gene">GmF3G2&#x02033;Gt-a</span> did not glucosylate kaempferol 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[rhamnosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside] and 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[glucosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside]. <span class="gene">GmF3G2&#x02033;Gt-a</span> had a higher preference for UDP-glucose than UDP-galactose, with only 3% activity relative to that for UDP-galactose. No UGT activity was detected for UDP-arabinose and UDP-glucuronic acid. <span class="gene">GmF3G2&#x02033;Gt-b</span> also showed similar substrate specificity (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> : Table S1). Accordingly, <span class="gene">GmF3G2&#x02033;Gt-a</span> and <span class="gene">GmF3G2&#x02033;Gt-b</span> were defined as flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferases. <fig id="Fig6"> <label> Figure 6 </label> <caption> <p> Identification of reaction product of <span class="gene">GmF3G2&#x02033;Gt-a</span> (cultivar Nezumisaya). <bold> (A) </bold> Elution profiles of the standards (kaempferol 3- <italic> O </italic> -glucoside and kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-a</span> <span class="trait">protein.</span> <bold> (B) </bold> UV spectra of the standard (kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-a</span> <span class="trait">protein.</span> Mass spectra <bold> (C) </bold> and MS/MS spectra <bold> (D) </bold> of the standard (kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-a</span> <span class="trait">protein.</span> <bold> E </bold> , The MS/MS fragmentation for kaempferol 3- <italic> O </italic> -sophoroside. K3Glc, kaempferol 3- <italic> O </italic> -glucoside; K3Glc2&#x02033;Glc, kaempferol 3- <italic> O </italic> -sophoroside.
					</p>
				
					<p>&nbsp;&nbsp;
						The recombinant <italic> <span class="gene">GmF3G2&#x02033;Gt-a</span> </italic> <span class="trait">protein</span> converted UDP-glucose and kaempferol 3- <italic> O </italic> -glucoside to kaempferol 3- <italic> O </italic> -sophoroside, proving that <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> encodes a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase. The <span class="trait">protein</span> did not discriminate between glucose and galactose attached to the 3-position of kaempferol based on the results of HPLC analysis. Hence, it is similar to the soybean flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;6) rhamnosyltransferase [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. <span class="gene">GmF3G2&#x02033;Gt-a</span> <span class="trait">protein</span> showed a broad activity for kaempferol/quercetin 3- <italic> O </italic> -glucoside/galactoside derivatives, but it did not glucosylate kaempferol 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[rhamnosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside] and 3- <italic> O </italic> -rhamnosyl-(1&#x02009;&#x02192;&#x02009;4)-[glucosyl-(1&#x02009;&#x02192;&#x02009;6)-glucoside]. This suggests that 2&#x02033;-glucosylation of flavonol 3- <italic> O </italic> -glycoside is irreconcilable with 4&#x02033;-glycosylation. This hypothesis is supported by the flavonoid profiles of Nezumisaya and &#x02018;type 4&#x02019; in which FGs with 2&#x02033; and 4&#x02033;-double glycosylation are lacking. Further, leaves of seven soybean accessions contained FGs with double glycosylation of 2&#x02033; and 6&#x02033;, or 4&#x02033; and 6&#x02033;, but lacked FGs with double glycosylation of 2&#x02033; and 4&#x02033; [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Though four amino acids were substituted, the recombinant <italic> <span class="gene">GmF3G2&#x02033;Gt-b</span> </italic> <span class="trait">protein</span> also had a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase activity. A structural modeling of <span class="gene">GmF3G2&#x02033;Gt</span> using the 3D structure of a grape flavonoid 3- <italic> O </italic> -glucosyltransferase, VvGT1, as a template indicated that three substituted amino acids (V142, S149, T183 in <span class="gene">GmF3G2&#x02033;Gt-b)</span> were located far from the enzyme active sites. The other amino acid (T20) was relatively close to the uracil ring of UDP-2-deoxy-2-fluoro-glucose (6.55 &#x000c5;) but far from the active center. The threonine residues corresponding to T20 in <span class="gene">GmF3G2&#x02033;Gt-b</span> were also conserved in flavonoid 3- <italic> O </italic> -glucoside: (1-2) glucosyltransferase, such as AtF3G2&#x02033;Gt and IpA3G2&#x02033;Gt. These data suggest that four substituted amino acid residues in <span class="gene">GmF3G2&#x02033;Gt-b</span> are unlikely to affect enzyme activity. The transcript level of <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> in leaves of Harosoy was about 15% of that in Nezumisaya. <span class="gene">GmF3G2&#x02033;Gt</span> <span class="trait">protein</span> possibly have a threshold for catalysis and that of Harosoy is lower than the threshold. To confirm the hypothesis, further studies such as association analysis between gene expression and FG amounts among tissues or developmental stages may be necessary. Indels and substitutions in the 5&#x02032; upstream region may be responsible for the differences in expression level. Promoter assays may be necessary to determine which polymorphism is critical for gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						FGs having glucose at the 2&#x02033;-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Nezumisaya, whereas FGs of Harosoy were devoid of 2&#x02033;-glucose. Conversely, FGs having glucose at the 6&#x02033;-position of glucose or galactose that is bound to the 3-position of kaempferol were present in Harosoy, whereas these FGs were absent in Nezumisaya. Two genes control the FG pattern; the first is responsible for the attachment of glucose to the 2&#x02033;-position, and encodes a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase. Nezumisaya has a dominant whereas Harosoy has a recessive allele of the gene. A candidate gene <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> was located in the molecular linkage group <span class="gene">C2</span> (chromosome 6). The open reading frame of <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> is 1380 bp long encoding 459 amino acids with four amino acid substitutions among the cultivars. The <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> recombinant <span class="trait">protein</span> converted kaempferol 3- <italic> O </italic> -glucoside to kaempferol 3- <italic> O </italic> -sophoroside. Hence, <italic> <span class="gene">GmF3G2&#x02033;Gt</span> </italic> encodes a flavonol 3- <italic> O </italic> -glucoside/galactoside (1&#x02009;&#x02192;&#x02009;2) glucosyltransferase and corresponds to the <italic> Fg3 </italic> gene. Based on substrate specificity, 2&#x02033;-glucosylation of flavonol 3- <italic> O </italic> -glycoside may be irreconcilable with 4&#x02033;-glycosylation in soybean leaves.
					</p>
				
					<p>&nbsp;&nbsp;
						Identification of reaction product of <span class="gene">GmF3G2&#x02033;Gt-b</span> (cultivar Harosoy). (A) Elution profiles of the standards (kaempferol 3- <italic> O </italic> -glucoside and kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-b</span> <span class="trait">protein.</span> (B) UV spectra of the standard (kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-b</span> <span class="trait">protein.</span> Mass spectra (C) and MS/MS spectra (D) of the standard (kaempferol 3- <italic> O </italic> -sophoroside) and reaction product of <span class="gene">GmF3G2&#x02033;Gt-b</span> <span class="trait">protein.</span> K3Glc, kaempferol 3- <italic> O </italic> -glucoside; K3Glc2&#x02033;Glc, kaempferol 3- <italic> O </italic> -sophoroside.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:[Characterization and subcellular localization of two <span class="gene">SBP</span> genes and their response to abiotic stress in soybean (Glycine max (L.) Merr.)].</h2>
			<h3>PMID:25985522  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang Y, Wang S, Huang L, Ma H, Shu Y, He X, Ma H.
				<br/> 
				年份：2014 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> temperature and humidity stress during seed growth and development of spring soybean can result in seed deterioration in South China. We isolated two genes <span class="gene">(GmSBP</span> and <span class="gene">GmSBPL)</span> encoding putative <span class="gene">SBP</span> proteins from soybean (Glycine max (L.) Merr.) to study their biological functions and response to abiotic stress,. The two <span class="gene">SBP</span> proteins are hydrophilic and incomplete membrane ones. Real-time quantitative (RT-PCR) analysis reveals that the expression of the two genes in the developing seeds of the seed deterioration resistant cultivar Xiangdou No. 3 and sensitive cultivar Ningzhen No. 1 was significantly affected by <span class="trait">high</span> temperature and humidity treatment. Meanwhile, the levels of sucrose and soluble sugar in the developing seeds of both cultivars were also affected under <span class="trait">high</span> temperature and humidity stress. During seed growth and development, the expression of the two genes as well as the levels of sucrose and soluble sugar reached the highest at 30 days after flower. <span class="gene">GmSBP2</span> and <span class="gene">GmSBPL</span> were found to be differentially expressed in different soybean tissues. Sub-cellular localization indicated that two genes were located in cytoplasm and cell membrane. Our results indicate that <span class="gene">GmSBP2</span> and <span class="gene">GmSBPL</span> might be involved in the response to abiotic stress, which will enrich our understanding of pre-harvest seed deterioration and <span class="trait">resistance</span> in soybean from one side.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sequence and expression variations suggest an adaptive role for the <italic> DA1 </italic> -like gene family in the evolution of soybeans</h2>
			<h3>PMID:25975199</h3>
				

				
					<h3>PMCID:4432951</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Man </given-names>  </name>  <address>  <email> zhaoman09@sina.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gu </surname>  <given-names> Yongzhe </given-names>  </name>  <address>  <email> yongzhe@ibcas.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Lingli </given-names>  </name>  <address>  <email> linglihe@ibcas.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Qingshan </given-names>  </name>  <address>  <email> qshchen@126.com </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> He </surname>  <given-names> Chaoying </given-names>  </name>  <address>  <phone> +86-10-62836085 </phone>  <email> chaoying@ibcas.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> State Key Laboratory of Systematic and Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences, Nanxincun 20, Xiangshan, 100093 Beijing, China  </aff>  <aff id="Aff2">  <label/> University of Chinese Academy of Sciences, Yuquan Road 19, 100049 Beijing, China  </aff>  <aff id="Aff3">  <label/> College of Biological and Environmental Engineering, Zhejiang University of Technology, 310014 Hangzhou, China  </aff>  <aff id="Aff4">  <label/> College of Agriculture, Northeast Agricultural University, Harbin, Heilongjiang 150030 China  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The DA1 gene family is plant-specific and Arabidopsis DA1 regulates seed and organ size, but the functions in soybeans are unknown. The cultivated soybean (Glycine max) is believed to be domesticated from the annual wild soybeans (Glycine soja). To evaluate whether DA1-like genes were involved in the evolution of soybeans, we compared variation at both sequence and expression levels of DA1-like genes from G. max <span class="gene">(GmaDA1)</span> and G. soja (GsoDA1). RESULTS: Sequence identities were extremely <span class="trait">high</span> between the orthologous pairs between soybeans, while the paralogous copies in a soybean species showed a relatively <span class="trait">high</span> divergence. Moreover, the expression variation of DA1-like paralogous genes in soybean was much greater than the orthologous gene pairs between the wild and cultivated soybeans during development and challenging abiotic stresses such as salinity. We further found that overexpressing GsoDA1 genes did not affect seed size. Nevertheless, overexpressing them reduced transgenic Arabidopsis seed germination sensitivity to salt stress. Moreover, most of these genes could improve salt tolerance of the transgenic Arabidopsis plants, corroborated by a detection of expression variation of several key genes in the salt-tolerance pathways. CONCLUSIONS: Our work suggested that expression diversification of DA1-like genes is functionally associated with adaptive radiation of soybeans, reinforcing that the plant-specific DA1 gene family might have contributed to the successful adaption to complex environments and radiation of the plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Salt stress has posed a great threat to agricultural development around the world. Soil salinity can inhibit plant growth and decrease crop yields [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. More than 800 million hectares of lands have been affected by salinity per year in the world [ <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. In <italic> Arabidopsis thaliana </italic> , the ionic signaling pathway is transduced via the salt-overly-sensitive (SOS) pathway where a calcium-responsive SOS3-SOS2 <span class="trait">protein</span> kinase complex controls the expression and activity of ion transporters [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. The complex of serine/threonine <span class="trait">protein</span> kinase SOS2 and the myristoylated calcium-binding <span class="trait">protein</span> SOS3 can be activated by a salt-stress-elicited calcium signal, and then the activated <span class="trait">protein</span> kinase complex phosphorylates and activates various ion transporters, such as the plasma membrane Na <sup> + </sup> /H <sup> + </sup> antiporter <span class="gene">SOS1</span> [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ], which improves salt-tolerance. In response to osmotic stress caused by salt, plants can accumulate metabolites that act as compatible solutes to lower the cellular osmotic potential without affecting normal metabolic reactions [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. Delta 1-pyrroline-5-carboxylate synthetase gene ( <italic> P5CS1 </italic> ) encodes the rate-limiting enzyme involved in proline synthesis [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> ]; this enzyme can increase the rate of proline synthesis, which results in proline accumulation and increased osmotic pressure, thus increasing salt tolerance in plants [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Some other genes, such as the alcohol dehydrogenase gene ( <italic> ADH </italic> ) and <italic> FIERY1 </italic> ( <italic> FRY1 </italic> ) gene, that were involved in salt tolerance were related to the phytohormone abscisic acid (ABA) [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. Salt tolerance mechanisms in soybeans are unknown, but they have been proposed to consist of the maintenance of ion homeostasis, adjustment in response to osmotic stress, restoration of oxidative balance and other metabolic and structural adaptations [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis DA1 </italic> and <italic> DA1 </italic> -related 1 ( <italic> DAR1 </italic> ) genes, as representatives of the <italic> DA1 </italic> gene family, have a function in regulating organ and seed size [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. The <italic> DA1 </italic> expression is induced by ABA [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ], implying that the <italic> DA1 </italic> -like genes may be involved in the response to abiotic stresses. This assumption has been further corroborated by observations of another <italic> DAR </italic> gene; i.e., the <italic> chilling sensitive 3 </italic> ( <italic> <span class="gene">CHS3</span> </italic> ) gene, which plays a role in biotic and abiotic <span class="trait">resistance</span> responses [ <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Therefore, <italic> DA1 </italic> -like genes have multiple functions in plants. A genome-wide evolutionary study suggested that the <italic> DA1 </italic> gene family encodes a group of proteins belonging to the LIM (from lin-11, isl-1 and mec-3) domain superfamily, and that this gene family is plant-specific and may play a role in the successful radiation of plants in diverse environments during evolution [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Since wild soybeans are distributed in a variety of adverse natural environments, they may act as a source for improving phenological, agronomic traits, and the resistances and tolerances to biotic and abiotic stresses of cultivated soybeans [ <xref ref-type="bibr" rid="CR2"> 2 </xref> , <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Comparative study of wild and cultivated soybeans at gene, genomic and transcriptomic levels might provide multiple genetic resources to improve the <span class="trait">yield</span> and quality of cultivated soybeans. Such studies will also provide valuable insights into the molecular basis of soybean domestication, in turn further directing genetic improvement of modern soybean cultivars. Whether the <italic> DA1 </italic> -like genes were involved in the evolution / domestication of soybeans is unknown. In the present study, we evaluated this potentiality via comparison of sequences and expressions of the <italic> DA1 </italic> gene family between <italic> G. max </italic> ( <italic> <span class="gene">GmaDA1</span> </italic> ) and <italic> G. soja </italic> ( <italic> GsoDA1 </italic> ). Little sequence divergence was observed between the <span class="gene">Gma-GsoDA1</span> orthologous <span class="trait">protein</span> pairs while a relative <span class="trait">high</span> divergence was found among paralogous proteins in a species. The expression of their genes variously responded to diverse abiotic stresses. Moreover, overexpressing some <italic> GsoDA1 </italic> genes could improve salt-tolerance of the transgenic <italic> Arabidopsis </italic> plants. Therefore, our results suggested that expression variation of the <italic> DA1 </italic> -like genes may be largely attributed to the adaptive evolution of soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean genome encodes 11 <italic> DA1 </italic> -like genes [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. We first cloned their cDNAs in the cultivated soybean Suinong 14 and the wild accession ZYD00006. To show the orthology between the cultivated and the wild soybeans, an unrooted phylogenetic tree was constructed using the amino acid sequences. In line with our previous work [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ], the <italic> DA1 </italic> -like genes in soybeans diverged into Class I and Class II (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ). For each member, the two sequences from the cultivated and wild soybeans gathered together to form a pair (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ), suggesting their orthology, and their inheritance from common ancestors (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1b </xref> ). The sequence identities of the 11 orthologous pairs ranged from 95.8 to 100 % (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1), indicating a <span class="trait">high</span> conservation of these orthologous pairs. Four orthologous pairs were identical in Class I (DA1-1 and DA1-9) and Class II (DA1-7 and DA1-11), and the rest contained a few amino acid substitutions and indels (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ; Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). Nonetheless, sequence identities of DA1-like paralogs in a soybean showed a large difference ranging from 44.1 to 97.7 % (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). These results suggested that the 11 DA1-like paralogs in a soybean species originated in soybean common ancestors (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1b </xref> ) became more diversified than the orthologous pairs between the cultivated and wild soybeans during evolution. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> The phylogenetic relationship and sequence divergence of DA1-like proteins from <italic> G. soja </italic> (GsoDA1) and <italic> G. max </italic> <span class="gene">(GmaDA1).</span> <bold> a </bold> The ML (maximum likelihood) and NJ (neighbor-joining) trees were constructed using <span class="trait">protein</span> sequences. The green lines represent <span class="gene">GmaDA1</span> proteins, and the orange lines represent GsoDA1 proteins. The position of amino acid substitution sites (vertical black lines) and of indels (vertical blue lines) is shown (details in Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). <bold> b </bold> Schematic representation of the evolutionary process of <italic> DA1 </italic> -like genes during the speciation of soybeans
					</p>
				
					<p>&nbsp;&nbsp;
						To estimate the potential effects of amino acid substitutions on <span class="trait">protein</span> functions, we performed SNAP and PROVEAN analyses (see <xref rid="Sec13" ref-type="sec"> Methods </xref> ). The results of SNAP showed that all substitutions were neutral with an expected accuracy of more than 60 %, suggesting that these substitutions would be unlikely to change the <span class="trait">protein</span> functions (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). Furthermore, the PROVEAN analysis also predicted that almost all of the substitutions might not influence <span class="trait">protein</span> functions except for position 299 in DA1-8. The predicted score of substitution in DA1-8 was &#x02212;5.7, which was less than the cutoff value (&#x02212;2.5), implying that the substitution may be deleterious to the <span class="trait">protein</span> functions (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). The inconsistent prediction of DA1-8 in these two analyses may be caused by different referential databases of the two methods. Besides the amino acid substitutions, the indels could also affect the <span class="trait">protein</span> functions. Therefore, we used PROVEAN analysis to predict the functions of these indels, and showed that all the indels were neutral. Meanwhile, prediction of subcellular locations showed that there was no divergence in the distributions of these <span class="gene">Gma-GsoDA1</span> orthologous pairs. Nonetheless, subcellular locations of different DA1-like members (paralogs) in a soybean species were diverged (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). These predictions need to be further clarified experimentally, but they may reflect the putative functional variation resulting from the coding sequence variation in these <italic> DA1 </italic> -like genes. Besides this, the divergence of <italic> DA1 </italic> -like genes may occur in their expression levels.
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR was used to detect the relative expression levels of <italic> DA1 </italic> -like genes in various tissues of soybeans. Owing to <span class="trait">high</span> sequence identities, the degenerate primers of <italic> DA1-8 </italic> and <italic> DA1-10 </italic> were designed and named <italic> DA1-8/10 </italic> . The tissue-specific expression of <italic> <span class="gene">GmaDA1</span> </italic> genes in the cultivated soybean Suinong 14 was previously investigated [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. Here we investigated the expression of <italic> GsoDA1 </italic> genes in various organs in wild soybean ZYD00006 (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). <italic> GsoDA1 </italic> genes presented a higher expression level in roots, stems and leaves than the reproductive organs such as the flowers and the developing fruits. Moreover, the expression levels of these genes overall declined during fruit development with few fluctuations. The overall extensive expression of the <italic> DA1 </italic> -like genes in various organs implied that they may have universal roles in soybean development. We further compared the expression profile of <italic> <span class="gene">GmaDA1</span> </italic> [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ] and <italic> GsoDA1 </italic> (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ) orthologous gene pairs and we did not find a significant divergence ( <italic> P </italic> &#x02009;&#x0003e;&#x02009;0.09) between these 11 orthologous gene pairs, while <italic> <span class="gene">GmaDA1</span> </italic> ( <italic> P </italic> &#x02009;=&#x02009;0.009) or <italic> GsoDA1 </italic> ( <italic> P </italic> &#x02009;=&#x02009;0.04) paralogs in a species were different in all the detected organs, indicating an organ-biased role for each soybean <italic> DA1 </italic> -like gene. <fig id="Fig2"> <label> Fig. 2 </label> <caption> <p> Expression of the <italic> GsoDA1 </italic> genes during soybean development. ( <bold> a-j </bold> ) The expression of <italic> GsoDA1-1 </italic> ( <bold> a </bold> ), <italic> GsoDA1-2 </italic> ( <bold> b </bold> ), <italic> GsoDA1-3 </italic> ( <bold> c </bold> ), <italic> GsoDA1-4 </italic> ( <bold> d </bold> ), <italic> GsoDA1-5 </italic> ( <bold> e </bold> ), <italic> GsoDA1-6 </italic> ( <bold> f </bold> ), <italic> GsoDA1-7 </italic> ( <bold> g </bold> ), <italic> GsoDA1-8/10 </italic> ( <bold> h </bold> ), <italic> GsoDA1-9 </italic> ( <bold> i </bold> ) and <italic> GsoDA1-11 </italic> ( <bold> j </bold> ). The total RNAs were isolated from roots, stems, leaves of 14-day-old seedlings, unfertilized flower buds, flowers, and 2-, 4- and 6-day-old fruits after fertilization. The <italic> ACTIN </italic> gene was used as an internal control. The experiments were repeated using three independent biological samples. Error bar: standard deviation. The significance was tested in comparison with the expression of each gene in flower buds (black column). The * means significance at a <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05 level, and the ** represents the significance at a <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.01 level
					</p>
				
					<p>&nbsp;&nbsp;
						Wild and cultivated soybeans were regarded to have common progenitors, and the domestication has endowed the cultivated soybean with many advantages in morphological and physiological traits [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ], including retention of seed on the seed head, reduction in lateral <span class="trait">branching,</span> reduction in seed dormancy, an increase in seed size, and shifts in flowering time and grain composition [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. A growing number of major-effect domestication and crop improvement genes have been investigated, including the <italic> indeterminate </italic> gene <italic> 1 </italic> ( <italic> <span class="gene">Dt1</span> </italic> ) in soybean [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ], <italic> fascinated </italic> gene ( <italic> fas </italic> ) in tomato [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ], <italic> shattering </italic> gene <italic> 1 </italic> ( <italic> Sh1 </italic> ) in sorghum [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ] and <italic> FLOWERING LOCUS T </italic> gene ( <italic> HaFT1 </italic> ) in sunflower [ <xref ref-type="bibr" rid="CR37"> 37 </xref> ]. These studies revealed a diversity of underlying causative mutations affecting phenotypes important in plant domestication, including coding sequence substitutions, copy number variation, transposon activation leading to novel gene structures and expression patterns, diversification following gene duplication, and polyploidy leading to altered combinatorial capabilities [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. To exploit the role of <italic> DA1 </italic> -like genes in soybean domestication, we comparatively analyzed the sequences and expressions of these genes among cultivated and wild soybeans. Although <italic> Arabidopsis DA1 </italic> -like genes are involved in seed size control [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> ], we did not find any evidence supporting such a role for <italic> DA1 </italic> -like genes in soybeans. The differential expressions of <italic> DA1-6 </italic> , <italic> DA1-8/10 </italic> and <italic> DA1-11 </italic> between the cultivated and wild soybeans in the reproductive organs hinted at a potential role in seed development, but the transgenic <italic> Arabidopsis </italic> plants harboring <italic> GsoDA1 </italic> genes did not affect seed size. Manipulating <italic> <span class="gene">GmaDA1</span> </italic> genes in transgenic <italic> Arabidopsis </italic> might clarify this. Nonetheless, the role of <italic> DA1 </italic> -like genes in organ size control might be specific to the <italic> Brassica </italic> species or be a gain-of-function related to the <italic> da1-1 </italic> mutation (R358K) in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ], since all isolated soybean <italic> DA1 </italic> -like genes encoded putative proteins having a conserved 358 site (R).
					</p>
				
					<p>&nbsp;&nbsp;
						<media position="anchor" xlink:href="12870_2015_519_MOESM1_ESM.pdf" id="MOESM1"> <label> Additional file 1: Table S1. </label> <caption> <p> Sequence identities of GsoDA1 and <span class="gene">GmaDA1</span> proteins. <bold> Table S2. </bold> Divergence of GsoDA1 and <span class="gene">GmaDA1</span> proteins. Table S3. Primers used in the present work. <bold> Figure S1. </bold> Expression of the <italic> <span class="gene">GmaDA1</span> </italic> and <italic> GsoDA1 </italic> genes in response to <span class="trait">drought.</span> <bold> Figure S2. </bold> Expression of the <italic> <span class="gene">GmaDA1</span> </italic> and <italic> GsoDA1 </italic> genes in response to acid and base stresses. <bold> Figure S3. </bold> Expression of the <italic> <span class="gene">GmaDA1</span> </italic> and <italic> GsoDA1 </italic> genes in response to ABA. <bold> Figure S4. </bold> PCR-confirmation of <italic> 35S::GsoDA1 </italic> transgenic <italic> Arabidopsis. </italic> 1: <bold> Figure S5. </bold> Seed size of <italic> 35S::GsoDA1 </italic> transgenic <italic> Arabidopsis </italic> . <bold> Dataset S1. </bold> Soybean <italic> DA1- </italic> like expression in response to various abiotic stresses.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:SNP identification and marker assay development for high-throughput selection of soybean cyst nematode <span class="trait">resistance</span></h2>
			<h3>PMID:25903750</h3>
				

				
					<h3>PMCID:4407462</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Zi </given-names>  </name>  <address>  <email> shizi@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <address>  <email> smliu@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Noe </surname>  <given-names> James </given-names>  </name>  <address>  <email> jpnoe@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arelli </surname>  <given-names> Prakash </given-names>  </name>  <address>  <email> Prakash.Arelli@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <address>  <email> meksemk@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <address>  <email> zli@uga.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Center for Applied Genetic Technologies &#x00026; Dep. of Crop and Soil Sciences, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff2">  <label/> Department of Plant, Soil and Agriculture Systems, Southern Illinois University, Carbondale, IL 62901 USA  </aff>  <aff id="Aff3">  <label/> Department of Plant Pathology, University of Georgia, Athens, GA 30602 USA  </aff>  <aff id="Aff4">  <label/> USDA-ARS-SEA, Jackson, TN 38301 USA  </aff>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode (SCN) is the most economically devastating pathogen of soybean. Two <span class="trait">resistance</span> loci, <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> primarily contribute <span class="trait">resistance</span> to SCN race 3 in soybean. Peking and PI 88788 are the two major sources of SCN <span class="trait">resistance</span> with Peking requiring both <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> alleles and PI 88788 only the <span class="gene">Rhg1</span> allele. Although simple sequence repeat (SSR) markers have been reported for both loci, they are linked markers and limited to be applied in breeding programs due to accuracy, throughput and cost of detection methods. The objectives of this study were to develop robust functional marker assays for high-throughput selection of SCN <span class="trait">resistance</span> and to differentiate the sources of <span class="trait">resistance.</span> RESULTS: Based on the genomic DNA sequences of 27 soybean lines with known SCN phenotypes, we have developed Kompetitive Allele Specific PCR (KASP) assays for two Single nucleotide polymorphisms (SNPs) from <span class="gene">Glyma08g11490</span> for the selection of the <span class="gene">Rhg4</span> <span class="trait">resistance</span> allele. Moreover, the genomic DNA of <span class="gene">Glyma18g02590</span> at the <span class="gene">Rhg1</span> locus from 11 soybean lines and cDNA of Forrest, Essex, Williams 82 and PI 88788 were fully sequenced. Pairwise sequence alignment revealed seven SNPs/insertion/deletions (InDels), five in the 6th exon and two in the last exon. Using the same 27 soybean lines, we identified one SNP that can be used to select the <span class="gene">Rhg1</span> <span class="trait">resistance</span> allele and another SNP that can be employed to differentiate Peking and PI 88788-type <span class="trait">resistance.</span> These SNP markers have been validated and a strong correlation was observed between the SNP genotypes and reactions to SCN race 3 using a panel of 153 soybean lines, as well as a bi-parental population, F5-derived recombinant inbred lines (RILs) from G00-3213xLG04-6000. CONCLUSIONS: Three functional SNP markers (two for <span class="gene">Rhg1</span> locus and one for <span class="gene">Rhg4</span> locus) were identified that could provide genotype information for the selection of SCN <span class="trait">resistance</span> and differentiate Peking from PI 88788 source for most germplasm lines. The robust KASP SNP marker assays were developed. In most contexts, use of one or two of these markers is sufficient for high-throughput marker-assisted selection of plants that will exhibit SCN <span class="trait">resistance.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) is the most economically devastating pathogen of soybean. Two <span class="trait">resistance</span> loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> primarily contribute <span class="trait">resistance</span> to SCN race 3 in soybean. Peking and PI 88788 are the two major sources of SCN <span class="trait">resistance</span> with Peking requiring both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> alleles and PI 88788 only the <italic> <span class="gene">Rhg1</span> </italic> allele. Although simple sequence repeat (SSR) markers have been reported for both loci, they are linked markers and limited to be applied in breeding programs due to accuracy, throughput and cost of detection methods. The objectives of this study were to develop robust functional marker assays for high-throughput selection of SCN <span class="trait">resistance</span> and to differentiate the sources of <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the genomic DNA sequences of 27 soybean lines with known SCN phenotypes, we have developed Kompetitive Allele Specific PCR (KASP) assays for two Single nucleotide polymorphisms (SNPs) from <span class="gene">Glyma08g11490</span> for the selection of the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele. Moreover, the genomic DNA of <span class="gene">Glyma18g02590</span> at the <italic> <span class="gene">Rhg1</span> </italic> locus from 11 soybean lines and cDNA of Forrest, Essex, Williams 82 and PI 88788 were fully sequenced. Pairwise sequence alignment revealed seven SNPs/insertion/deletions (InDels), five in the 6th exon and two in the last exon. Using the same 27 soybean lines, we identified one SNP that can be used to select the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele and another SNP that can be employed to differentiate Peking and PI 88788-type <span class="trait">resistance.</span> These SNP markers have been validated and a strong correlation was observed between the SNP genotypes and reactions to SCN race 3 using a panel of 153 soybean lines, as well as a bi-parental population, F <sub> 5 </sub> &#x02013;derived recombinant inbred lines (RILs) from G00-3213 x LG04-6000.
					</p>
				
					<p>&nbsp;&nbsp;
						Three functional SNP markers (two for <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> locus) were identified that could provide genotype information for the selection of SCN <span class="trait">resistance</span> and differentiate Peking from PI 88788 source for most germplasm lines. The robust KASP SNP marker assays were developed. In most contexts, use of one or two of these markers is sufficient for high-throughput marker-assisted selection of plants that will exhibit SCN <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN, <italic> Heterodera glycines </italic> L.) is the most economically devastating pathogen of soybean and causes losses in soybean producing areas worldwide with no practical means of eradication [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. The annual losses exceed one billion dollars in the United States alone [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. Two <span class="trait">resistance</span> genes for SCN, <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">(Resistance</span> to <italic> H. glycines </italic> ) on Chromosome (Chr) 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr08, have been identified in the soybean germplasm, and have been shown to be the key sources of <span class="trait">resistance</span> in soybean cultivars [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. The <italic> <span class="gene">Rhg1</span> </italic> locus, which is crucial for broad spectrum <span class="trait">resistance</span> to SCN, has been identified in various <span class="trait">resistance</span> sources, such as PI 88788 and Peking [ <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						SCN field populations are genetically variable [ <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Sixteen SCN races have been described and the race designation was determined by comparing its reproduction on a set of four soybean differentials with a standard SCN-susceptible cultivar. To avoid the implication of genetic predictability, a revised classification scheme for field SCN population introduced the <italic> Heterodera glycines </italic> (HG) types determined by reactions to seven indicator lines [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. The distribution of SCN populations in the field varied among different geographic regions. However, race 3 is the most frequently found SCN populations in major soybean producing states in the USA [ <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. Soybean lines PI 88788 and Peking are the two main sources of <span class="trait">resistance</span> to SCN race 3 [ <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. Over 90% of the commercially available SCN-resistant soybean cultivars in the USA carry the <italic> rhg1-b </italic> allele, which is derived from PI 88788 due to its desirable agronomic characteristics [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. While PI 88788-type <span class="trait">resistance</span> only requires the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, Peking-type <span class="trait">resistance</span> needs both <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles for a resistant phenotype [ <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Although the molecular basis of SCN-resistance is still unknown, recent studies have shed light on the genes involved in the <span class="trait">resistance.</span> Map-based cloning showed that a single gene at <italic> <span class="gene">Rhg4</span> </italic> locus encoding a serine hydroxymethyltransferase (SHMT, <span class="gene">Glyma08g11490)</span> is responsible for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Two polymorphisms have been identified in the coding sequence of SHMT, 389&#x000a0;G/C in the first exon and 1,165 A/T in the second exon, which resulted in the amino acid change of Arginine (R) to Proline (P) and Tyrosine(Y) to Asparagine (N), respectively. These polymorphisms change the property of the enzyme and may correlate with <italic> <span class="gene">Rhg4</span> </italic> -dependent <span class="trait">resistance</span> and susceptibility [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. Genetic mapping and gene functional analysis of the <italic> rhg1-b </italic> locus exhibited a 31.2&#x000a0;kb genomic segment encoding a predicted amino acid transporter <span class="gene">(Glyma18g02580),</span> an &#x003b1;&#x02013;SNAP <span class="trait">protein</span> <span class="gene">(Glyma18g02590)</span> and a <span class="trait">protein</span> with wound-inducible <span class="trait">protein</span> 12 (WI12) <span class="gene">(Glyma18g02610)</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. Gene silencing and overexpression analysis showed that all three genes contribute to SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Interestingly, the phenotypic reaction to SCN is associated with the copy number variation of this 31.2&#x000a0;kb fragment. Only one copy is present in susceptible varieties, such as Williams 82, and multiple copies in SCN resistant lines. For example, Peking possesses three copies and seven to 10 copies were detected in PI 88788-derived resistant lines [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Sequencing of cDNA from several soybean lines revealed that multiple alleles of <span class="gene">Glyma18g02590</span> were present in the different multi-copy <italic> <span class="gene">Rhg1</span> </italic> categories [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Planting of resistant soybean cultivars combined with crop rotation formulates the best management strategy for reducing <span class="trait">yield</span> losses due to SCN. However, traditional breeding for resistant lines is challenging, not just because the phenotyping of SCN is time-consuming, but it is also affected by the different <span class="trait">resistance</span> genes [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and varied SCN populations in the field [ <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Therefore, the use of molecular markers to assist the selection of SCN <span class="trait">resistance</span> is a crucial improvement in breeding programs. Major efforts have been made to develop genetic markers linked to SCN <span class="trait">resistance</span> genes. Restriction fragment length polymorphisms (RFLPs) markers have been identified and reported to facilitate soybean breeding for both <italic> <span class="gene">Rhg1</span> </italic> [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> - <xref ref-type="bibr" rid="CR20"> 20 </xref> ] and <italic> <span class="gene">Rhg4</span> </italic> loci [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> - <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Simple sequence repeats (SSRs) have further been discovered and applied in the marker-assisted selection of SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> - <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. SNP/InDel markers have also been developed at the flanking region of both loci [ <xref ref-type="bibr" rid="CR11"> 11 </xref> , <xref ref-type="bibr" rid="CR28"> 28 </xref> - <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. However, they are not diagnostic markers, and both SSRs and RFLPs are labor/time-consuming and cost ineffective. Thus, they are very limited to be applied in breeding programs in a <span class="trait">high</span> throughput setting.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we report the development of three robust, functional SNP marker assays, two for the <italic> <span class="gene">Rhg1</span> </italic> locus and one for <italic> <span class="gene">Rhg4</span> </italic> . These functional SNP markers provide genotype information which can be used for the high-throughput selection of SCN <span class="trait">resistance,</span> as well as differentiating Peking and PI 88788 <span class="trait">resistance</span> sources for breeding of SCN <span class="trait">resistance</span> cultivars and germplasm introgression.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine SNP markers within the <italic> <span class="gene">Rhg1</span> </italic> locus, published sequences of the region were gathered which included the entire 31.2&#x000a0;kb block from PI 88788 and Williams 82 [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], but only the cDNA of <span class="gene">Glyma18g02590</span> from Peking was available [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. Moreover, we fully sequenced the genomic DNA of <span class="gene">Glyma18g02590</span> from 11 soybean lines and cDNA of <span class="gene">Glyma18g02590</span> from Forrest, Essex, Williams 82 and PI 88788 at the <italic> <span class="gene">Rhg1</span> </italic> locus. Sequence alignment was only carried out for <span class="gene">Glyma18g02580,</span> <span class="gene">Glyma18g02590</span> and <span class="gene">Glyma18g02610,</span> because other genes (partial <span class="gene">Glyma18g02570</span> and <span class="gene">Glyma18g02600)</span> in this 31.2&#x000a0;kb block are not functional according to the previous study [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Three SNPs were identified in <span class="gene">Glyma18g02580</span> from the comparison between Williams 82 and PI 88788, one synonymous SNP in the first exon, one in the intron and one in the 3&#x02019; untranslated region (UTR) (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> B). <span class="gene">Glyma18g02590</span> possesses more SNP/InDels. The alignment of both published sequences (alleles in black) and our sequencing result of 11 soybean lines (alleles in red) revealed five SNPs in the 6th exon and two SNPs in the last exon. All of them lead to amino acid changes in the <span class="trait">protein.</span> Three SNPs from the 5&#x02019;UTR of Glyma02610 have also been demonstrated between Williams 82 and PI 88788. <fig id="Fig1"> <label> Figure 1 </label> <caption> <p> Schematic graph showed the position of SNP/InDels at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci. <bold> A </bold> . Two SNPs identified at <span class="gene">Glyma08g11490</span> at <italic> <span class="gene">Rhg4</span> </italic> locus. <bold> B </bold> . 13 SNP/InDels identified in three genes within a 31.2&#x000a0;kb block at <italic> <span class="gene">Rhg1</span> </italic> locus. Blue squares represent exons and black lines represent UTRs and/or introns. The genome coordinates (Database: <span class="gene">Gmax_275_v2.0.softmasked)</span> are shown under each SNP in parenthesis. SNP alleles in black are from published sequences and SNP alleles in red from our sequencing results.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate which SNPs correlate with known SCN reactions of soybean lines, Kompetitive Allele Specific PCR (KASP) assays were developed and tested for all 13 SNPs at the <italic> <span class="gene">Rhg1</span> </italic> locus and two previously identified SNPs (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> A) at <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ] using 27 soybean lines with known SCN reactions. Two SNPs from <span class="gene">Glyma08g11490</span> are associated with the <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> allele, and they are completely linked (data not shown). Three SNPs in <span class="gene">Glyma18g02580</span> and two SNP/InDels in the last exon of <span class="gene">Glyma18g02590</span> are associated with the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele, and all five markers are completely linked. Five SNPs in the 6th exon of <span class="gene">Glyma18g02590</span> could distinguish the sources of Peking-type and PI88788-type <span class="trait">resistance.</span> However, three SNPs identified in the 5&#x02019;-UTR of <span class="gene">Glyma18g02610</span> showed no correlation with SCN <span class="trait">resistance.</span> Three SNPs were selected to provide genotype information for the selection of SCN <span class="trait">resistance</span> (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ), which were designated as GSM383, GSM381 and GSM191. GSM381 and GSM383 are located in the 6th and last exons of <span class="gene">Glyma18g02590</span> at the <italic> <span class="gene">Rhg1</span> </italic> locus, respectively, while GSM191 is in the first exon of <span class="gene">Glyma08g11490.</span> The KASP assays of three SNPs performed with 27 soybean lines exhibited a good clustering of the homozygous alleles, suggesting that it is feasible to employ those markers for selection. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> Positions of three SNPs in <span class="gene">Glyma18g02590</span> and <span class="gene">Glyma08g11490</span> at <italic> Rgh1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci, respectively. GSM numbers were used to designate SNP markers. Blue squares represent exons and black lines represent introns and UTRs. The genome coordinates (Database: <span class="gene">Gmax_275_v2.0.softmasked)</span> of each SNP are shown.
					</p>
				
					<p>&nbsp;&nbsp;
						Haplotype analysis was performed using the Soy50k SNP Infinium chip data of 27 lines and nine cultivars including Woodruff, Hartwig, Boggs, Gordon, Bryan, G00-3880, Prichard, Bedford and Carver. The <italic> <span class="gene">Rhg1</span> </italic> region was compared between Peking and the Peking-type lines: Forrest, Woodruff, Hartwig, Boggs, Gordon, Bryan, PI 89772, PI 90763 and PI 437654, revealing that these lines have same haplotypes to Peking at the <italic> <span class="gene">Rhg1</span> </italic> locus as large as 5.9 Mbp. A 788&#x000a0;kb haplotype window block was shown in Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A. Hartwig, PI 89772, PI 90763 and PI 437654 showed a relatively small haplotype window (710&#x000a0;kb) when compared to Peking. Woodruff had a larger haplotype block that is about 3.8Mbp and over 5.9Mbp was observed in Forrest, Gordon and Bryan. However, PI 88788 and other susceptible-type lines showed no introgression from Peking at the <italic> <span class="gene">Rhg1</span> </italic> locus. All eight PI 88788-type lines, G00-3880, Prichard, Bedford, Carver, PI 548316, PI 209332, PI 603372 and PI 603587A exhibited haplotype window blocks ranging from 1.6Mbp to 4.3Mbp, when compared to PI 88788 at the <italic> <span class="gene">Rhg1</span> </italic> region. Haplotype analysis of <italic> <span class="gene">Rhg4</span> </italic> locus on Chr08 indicated haplotype window blocks from Peking and nine Peking-type lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B), with a smaller region of 348&#x000a0;kb in Woodruff, Boggs and Bryan. However, the introgressed haplotype blocks are as large as 2.6Mbp in Forrest, Hartwig and Gordon. Examination of the pedigrees of these lines indicated that Peking is present somewhere in the extended pedigree of these six cultivars ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/npgs/acc/acc_queries.html"> http://www.ars-grin.gov/npgs/acc/acc_queries.html </ext-link> ) [ <xref ref-type="bibr" rid="CR32"> 32 </xref> - <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. These data are consistent with genotype indications of our three SNPs alleles and further substantiated the effectiveness of these SNP markers. <fig id="Fig3"> <label> Figure 3 </label> <caption> <p> Haplotype analysis at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci using Soy50k SNP Infinium chip data. Twenty-seven soybean plant introductions and nine cultivars, Woodruff, Hartwig, Boggs, Gordon, Bryan, G00-3880, Prichard, Bedford and Carver were used. <bold> A </bold> . Haplotype analysis at <italic> <span class="gene">Rhg1</span> </italic> locus. SNPs alleles at the <italic> <span class="gene">Rhg1</span> </italic> region on Chr18 were compared to Peking and PI 88788, respectively. A 788 kbp region (Coordinate of 1,266,210-2,054,952 on Chr18 based on <span class="gene">Gmax_275_v2.0.softmasked)</span> consisting of 82 SNPs is shown in the graph. <bold> B </bold> . Haplotype analysis at <italic> <span class="gene">Rhg4</span> </italic> locus. SNPs alleles at the <italic> <span class="gene">Rhg4</span> </italic> region on Chr08 were compared to Peking. A 1.5Mbp region (Coordinate of 6,903,359-8,399,514 on Chr08 based on <span class="gene">Gmax_275_v2.0.</span> softmasked) consisting of 82 SNPs is shown here. The reference line is shown in dark red. Red squares represent same alleles as the reference and green squares represent alleles that are different from the reference. The position of the locus and the <span class="trait">resistance</span> types are as indicated.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the SNP markers with more diverse lines, we explored the data from the previous three years&#x02019; USDA Uniform Soybean Tests for Southern States where the phenotypes of SCN race 3 are available for all the tested lines. We requested a total of 153 lines from eight institutions and performed KASP assays on these lines with SNP markers, GSM381, GSM383 and GSM191 (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> A). The SNP genotypes, SCN race 3 reactions and pedigrees were listed (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). The rating of SCN phenotype is on a scale of 1 to 5 based on the number of cysts, thus, lines with ratings smaller than 3 were considered as resistant and lines with ratings equal to or greater than 3 as susceptible. A strong accordance was observed between SNP genotypes and reactions to SCN race 3 for 145 germplasm lines with an exception of 10 lines (successful rate&#x02009;&#x0003e;&#x02009;93%). Due to the large discrepancy of ratings for same line between two years, eight lines were excluded for analysis. Single factor analysis using General Linear Model (GLM procedure, SAS 9.3) showed a significant correlation between SCN <span class="trait">resistance</span> phenotypes and genotypes consisted of three SNP markers (R <sup> 2 </sup> &#x02009;=&#x02009;0.69, P&#x02009;&#x0003c;&#x02009;0.001). Moreover, we have examined the extended pedigrees (data not shown) of these lines with <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> allele (GSM381&#x02009;=&#x02009;G) to identify the sources of <span class="trait">resistance</span> (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). The SNP genotypes are in complete agreement with the <span class="trait">resistance</span> sources based on extended pedigrees: Peking-source with a G allele at GSM383 and PI 88788-source with a C allele, indicating that GSM383 is a functional SNP to differentiate Peking and PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> <span class="trait">resistance</span> allele. <fig id="Fig4"> <label> Figure 4 </label> <caption> <p> KASP graphs of three SNPs for validation and precision genotyping test population. <bold> A </bold> . KASP SNP graphs of 153 soybean lines from 2011&#x02013;2013 USDA Southern Uniform Soybean Tests with known SCN race 3 reactions. <bold> B </bold> . KASP SNP graphs of F <sub> 5 </sub> -derived RILs from the population derived from G00-3213 x LG04-6000.
					</p>
				
					<p>&nbsp;&nbsp;
						We also performed a precision genotyping test on a bi-parental population of F <sub> 5 </sub> -derived Recombinant inbred lines (RILs) from G00-3213 x LG04-6000. The line G00-3213 carries a Peking-source <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> <span class="trait">resistance</span> alleles (GSM381&#x02009;=&#x02009;G, GSM383&#x02009;=&#x02009;G, and GSM191&#x02009;=&#x02009;G) and is resistant to SCN race 3, while LG04-6000 is a susceptible line with T, C and C alleles for GSM381, GSM383, and GSM191, respectively. Three replicates of 150 RILs with two parents were arranged in a randomized complete block design in greenhouse and each entry was genotyped with three SNP markers (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> B). Ten weeks after inoculation, phenotyping was carried out by counting the numbers of cysts on the roots of each individual plant. Resulting phenotypes demonstrated that combination of the <span class="trait">resistance</span> <italic> <span class="gene">Rhg1</span> </italic> (GSM381&#x02009;=&#x02009;G and GSM383&#x02009;=&#x02009;G) and <span class="trait">resistance</span> <italic> <span class="gene">Rhg4</span> </italic> alleles (GSM191&#x02009;=&#x02009;G) brought a resistant phenotype with only 0.4 cyst/plant and all the other combinations exhibited a more susceptible phenotype (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ), suggesting that these three markers can be employed for the selection of SCN race 3 <span class="trait">resistance</span> from a population derived from Peking source. Single factor analysis using a GLM procedure in SAS 9.3 was performed for three subsets individually and combined. All of them showed a significant correlation between the average number of cysts and genotypes of three SNPs, GSM381, GSM383, and GSM191 (P&#x02009;&#x0003c;&#x02009;0.0001), with an R-square of 0.64, 0.40, 0.24, and 0.3 for RIL sets 1, 2, 3 and combined, respectively. The different R-square values among sets might be due to the conditions at different inoculation times. <fig id="Fig5"> <label> Figure 5 </label> <caption> <p> Reactions of race3 to different combinations of SNP marker alleles for 150&#x000a0;F <sub> 5 </sub> families of G00-3213 x LG04-6000. <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> genotypes are represented as combinations of alleles of SNP markers, GSM381, GSM383 and GSM191. Boxes show median, 25% to 75% of the data and whiskers extend to minimum and maximum of the data.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic basis of <span class="trait">resistance</span> to SCN is complex in soybean, not only because <span class="trait">resistance</span> to SCN is multi-genic, but also because different sets of genes/ Quantitative trait loci (QTLs) confer <span class="trait">resistance</span> to the diverse populations of SCN due to their genetic heterogeneity. Field populations of soybean cyst nematode have been classified into 16 races based on their ability to develop on resistant cultivars [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ]. <span class="trait">Resistance</span> to SCN is race-cultivar specific [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ], for example, soybean cultivars derived from Peking source are commonly claimed to be resistant to race 1 and 3, however, the cultivars with SCN <span class="trait">resistance</span> from PI 88788 are usually shown to be resistant to race 3 and 14 [ <xref ref-type="bibr" rid="CR37"> 37 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. A growing body of evidence has shown different defense loci that underlie <span class="trait">resistance</span> to different races. The <italic> <span class="gene">Rhg1</span> </italic> locus on Chr18 is necessary for <span class="trait">resistance</span> to all SCN races [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Many other cultivar-race specific QTLs have been demonstrated in <span class="trait">resistance</span> sources, such as <italic> <span class="gene">Rhg4</span> </italic> for race 1 and 3, <italic> Rhg3 </italic> and <italic> Rhg5 </italic> for race 2 and <italic> Rhg2 </italic> for race 5 and 14 [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Although two functional SNPs have been identified for <italic> <span class="gene">Rhg4</span> </italic> locus [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], no high-throughput marker assays have been reported to date. Moreover, several SNPs have been found at <italic> <span class="gene">Rhg1</span> </italic> locus [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], but no specific functional SNPs as well as detection assays have been reported to associate with the <span class="trait">resistance</span> allele, nor the SNPs to differentiate sources of <span class="trait">resistance.</span> Built on the previous discoveries of copy number variation, gene dosage, and DNA methylation at the <italic> <span class="gene">Rhg1</span> </italic> locus by Cook et al. [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] and SNPs at the <italic> <span class="gene">Rhg4</span> </italic> locus by Liu et al. [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], in this study, we have developed SNP marker assays, GSM381, GSM383, and GSM191 at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> locus for the selection of <span class="trait">resistance</span> alleles, (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). The SNPs reside in the coding sequences of <span class="trait">resistance</span> genes <span class="gene">(Glyma18g02590</span> and <span class="gene">Glyma08g11490)</span> and also introduce changes in <span class="trait">protein</span> sequences, thus they can be considered as functional markers for SCN race 3 <span class="trait">resistance</span> and applied in <span class="trait">high</span> throughput marker-assisted selection with <span class="trait">high</span> accuracy. Similarly, the knowledge of additional loci along with the genome sequence of soybean may be used in the marker development for <span class="trait">resistance</span> to different SCN races.
					</p>
				
					<p>&nbsp;&nbsp;
						PI 88788 has been the primary choice in soybean cultivar development as a source for SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ], however, SCN populations have demonstrated the capability to mutate and recombine to new races, which can overcome the soybean <span class="trait">resistance</span> derived from PI 88788 [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Reports have shown that PI 88788 has not been providing sufficient protection against SCN in some fields [ <xref ref-type="bibr" rid="CR42"> 42 </xref> - <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. Thus, additional <span class="trait">resistance</span> sources are desired to be used for developing new resistant cultivars and Peking is believed to be an attractive option. Although SNP GSM381 discriminates resistant and susceptible <italic> <span class="gene">Rhg1</span> </italic> alleles, it does not distinguish PI 88788- and Peking-type <italic> <span class="gene">Rhg1</span> </italic> loci. Thus, to identify the sources of <span class="trait">resistance,</span> we demonstrated that one SNP marker at <italic> <span class="gene">Rhg1</span> </italic> locus, GSM383 was able to distinguish Peking from PI 88788 loci for most soybean germplasm (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). This SNP can be employed in differentiating Peking from PI 88788 sources for the <italic> <span class="gene">Rhg1</span> </italic> allele for breeding programs.
					</p>
				
					<p>&nbsp;&nbsp;
						The successful prediction rate for three SNP markers at <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci is over 93% in this study (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). A few lines (10 lines from USDA Uniform Test (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1) and two lines from 27 validation lines (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> )) did not show a correlation between the expected phenotypes and the <span class="trait">resistance</span> SNP alleles. A number of factors may result in the discrepancy between genotyping and phenotyping results, including 1) phenotyping/ or seed source errors: some of the lines have been tested for only one year, and one-time phenotyping of SCN <span class="trait">resistance</span> may not be definitive; 2) sources other than PI 88788 and Peking may be providing <span class="trait">resistance;</span> and 3) the complexity of the <italic> <span class="gene">Rhg1</span> </italic> region is not fully understood. A future study will be designed to understand why the lines with the <span class="trait">resistance</span> genotypes did not have a <span class="trait">resistance</span> reaction to SCN race 3. The 31.2&#x000a0;kb region at <italic> Rgh1 </italic> locus is repeated in resistant genotypes [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], and the sequences of the individual copies are also variant within a single genotype, which makes marker development challenging for the <italic> <span class="gene">Rhg1</span> </italic> locus. We also tested marker performance with artificial heterozygous DNA, where PI 88788/ Peking and Williams 82 DNA were mixed at 1:1 ratio. Although the artificial heterozygous allele cluster was located a little closer towards the resistant homozygous allele cluster in the KASP SNP graphs (data not shown) because of the gene repeats in resistant lines, the alleles can still be called clearly. Therefore, the functional SNP markers that we identified can be used not only to select the <span class="trait">resistance</span> allele, but also to differentiate <span class="trait">resistance</span> sources, which were also confirmed by analysis of the <italic> <span class="gene">Rhg1</span> </italic> region with Soy50k SNP Infinium chip data using 38 soybean lines (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						A recent study of the <italic> <span class="gene">Rhg1</span> </italic> locus revealed that copy number variation of the 31&#x000a0;kb DNA fragment, DNA sequence variation, and differentially methylated patterns were correlated with the <italic> <span class="gene">Rhg1</span> </italic> -mediated <span class="trait">resistance</span> to SCN [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. These discoveries may also offer additional selection strategies for the <italic> <span class="gene">Rhg1</span> </italic> allele, in addition to the functional SNP detection assays we developed. Cook et al. [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] explored the whole genome data for <italic> <span class="gene">Rhg1</span> </italic> locus in 42 soybean lines, and multiple lines are in common in both studies. In this study, we got the same C-terminal <span class="trait">protein</span> sequence of <span class="gene">Glyma18g02590</span> for both low-copy and high-copy <italic> <span class="gene">Rhg1</span> </italic> as described in the previous study ([ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B), indicating that GSM 381 maybe also exist in their DNA sequences. Interestingly, they identified a novel splice isoform of <span class="gene">Glyma18g02590</span> cDNA in Peking-type <italic> <span class="gene">Rhg1</span> </italic> lines, however, we did not detect this isoform in our cDNA sequencing, probably due to the low copy number of this isoform and use of a different sequencing method. Thus, more studies are needed to investigate the <italic> <span class="gene">Rhg1</span> </italic> locus in various soybean lines for new insights into the development of improved selection approaches.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic sequence of <span class="gene">Glyma08g11490</span> at <italic> <span class="gene">Rhg4</span> </italic> locus in 28 soybean lines and <span class="gene">Glyma18g02590</span> at <italic> <span class="gene">Rhg1</span> </italic> locus in 11 lines, along with the known reactions to soybean cyst nematode race 3 of these lines, allowed us to identify functional SNPs and develop robust KASP SNP assays for marker-assisted breeding. Using a panel of 153 soybean germplasm and a segregating bi-parent population, we have validated the usefulness of these three SNP markers, two at <italic> <span class="gene">Rhg1</span> </italic> locus and one at <italic> Rhg4. </italic> The results of this work demonstrated that these markers could be applied for <span class="trait">high</span> throughput selection of SCN <span class="trait">resistance</span> from PI 88788 and Peking sources and for accelerating breeding of SCN <span class="trait">resistance</span> in an accurate and efficient manner.
					</p>
				
					<p>&nbsp;&nbsp;
						To test the association of SNPs with the reactions to SCN race 3 and develop robust markers for <span class="trait">high</span> throughput selection, Kompetitive Allele Specific PCR (KASP) assays were developed and tested for all the identified SNPs using above 27 soybean lines with known SCN phenotypes. The assays were performed as previously described [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. Three SNPs, GSM381 (in <span class="gene">Glyma18g02590,</span> G/T), GSM383 (in <span class="gene">Glyma18g02590,</span> G/C) and GSM191 (in <span class="gene">Glyma08g11490,</span> G/C), were then selected for marker validation. Primer sequences are listed in Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> . KASP assays were run with 4&#x000a0;&#x003bc;L reaction system including 2&#x000a0;&#x003bc;L low rox KASP master mix (KBiosciences, Herts England), 0.106&#x000a0;&#x003bc;L of primer mix (0.318&#x000a0;&#x003bc;L of each primer at final concentration) and 2&#x000a0;&#x003bc;L of 10-25&#x000a0;ng/&#x003bc;l genomic DNA. The PCR conditions for KASP marker assay was 94&#x000b0;C for 15&#x000a0;min, followed by 10&#x000a0;cycles of touch down PCR from 68&#x000b0;C to 60&#x000b0;C with 0.8&#x000b0;C decrease per cycle, then followed by 30&#x000a0;cycles of 94&#x000b0;C for 20&#x000a0;s and 57&#x000b0;C for 1&#x000a0;min. . PCR fluorescent endpoint readings were performed using the Light Cycler&#x000ae; 480 Real-Time PCR System (Roche, Germany). <table-wrap id="Tab3"> <label> Table 3 </label> <caption> <p> <bold> KASP assay primer sequences of three SNPs, GSM381, GSM383 and GSM191 </bold>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Consuming Beef vs. Soy <span class="trait">Protein</span> Has Little Effect on Appetite, Satiety, and Food Intake in Healthy Adults.</h2>
			<h3>PMID:25809680  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Douglas SM(1), Lasley TR(1), Leidy HJ(2).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Although protein-rich plant foods have recently been touted as an equivalent to animal products, limited data exist regarding the effects of animal vs. plant proteins on appetite, satiety, and subsequent food intake. OBJECTIVE: The objective was to determine whether a high-protein beef lunch alters the appetitive, hormonal, and neural signals controlling food intake regulation vs. a macronutrient and fiber-matched (MF) or serving size-matched (SS) soy lunch in healthy adults. METHODS: A randomized, double-blind, crossover design study was completed in 21 adults (aged 23 ± 1 y; body mass index: 23.8 ± 0.6 kg/m²; mean ± SEM) to compare 400-kcal MF and <span class="gene">SS</span> lunches varying in <span class="trait">protein</span> quality. The MF lunches contained 24-g beef <span class="trait">protein</span> or 24-g soy <span class="trait">protein,</span> whereas the <span class="gene">SS</span> lunches contained 1 serving of beef (24-g protein/1-g fiber) or soy (14-g protein/5-g fiber). Pre- and postlunch appetite questionnaires and blood sampling were completed over an 8-h period until dinner was voluntarily requested. In addition, pre- and postlunch functional MRI brain scans were completed to assess neural activation in response to food stimuli. RESULTS: On average, dinner was requested at ∼250 ± 20 min postlunch with no differences between beef vs. soy within the MF and <span class="gene">SS</span> conditions. Furthermore, no differences in hunger, fullness, peptide YY, or glucagon-like peptide-1 responses were observed after the beef vs. soy lunches within the MF and <span class="gene">SS</span> conditions. The <span class="gene">SS</span> beef meal led to reductions in anterior cingulate activation (-22 ± 6%) vs. soy (+7 ± 9%; P &lt; 0.01) and greater reductions in insular activation (-30 ± 6%) vs. <span class="gene">SS</span> soy (-7 ± 10%; P &lt; 0.01). The participants consumed ∼945 ± 78 kcal after the MF meals and 910 ± 74 kcal after the <span class="gene">SS</span> meals with no differences between beef vs. soy. CONCLUSIONS: When comparing 2 high-quality <span class="trait">protein</span> sources, such as beef and soy, the type of <span class="trait">protein</span> consumed within a mixed meal elicited very little effect on appetite, satiety, and food intake in healthy adults.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a novel group of legume-specific, Golgi apparatus-localized WRKY and Exo70 proteins from soybean.</h2>
			<h3>PMID:25805717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chi Y(1), Yang Y(1), Li G(1), Wang F(2), Fan B(2), Chen Z(3).
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many plant genes belong to families that arise from extensive proliferation and diversification allowing the evolution of functionally new proteins. Here we report the characterization of a group of proteins evolved from WRKY and exocyst complex subunit Exo70 proteins through fusion with a novel transmembrane (TM) domain in soybean (Glycine max). From the soybean genome, we identified a novel WRKY-related <span class="trait">protein</span> <span class="gene">(GmWRP1)</span> that contains a WRKY domain with no binding activity for W-box sequences. GFP fusion revealed that <span class="gene">GmWRP1</span> was targeted to the Golgi apparatus through its N-terminal TM domain. Similar Golgi-targeting TM domains were also identified in members of a new subfamily of Exo70J proteins involved in vesicle trafficking. The novel TM domains are structurally most similar to the endosomal cytochrome b561 from birds and close homologues of <span class="gene">GmWRP1</span> and <span class="gene">GmEx070J</span> proteins with the novel TM domain have only been identified in legumes. Transient expression of some <span class="gene">GmExo70J</span> proteins or the Golgi-targeting TM domain in tobacco altered the subcellular structures labelled by a fluorescent Golgi marker. <span class="gene">GmWRP1</span> transcripts were detected at <span class="trait">high</span> levels in roots, flowers, pods, and seeds, and the expression levels of <span class="gene">GmWRP1</span> and <span class="gene">GmExo70J</span> genes were elevated with increased age in leaves. The legume-specific, Golgi apparatus-localized <span class="gene">GmWRP1</span> and <span class="gene">GmExo70J</span> proteins are probably involved in Golgi-mediated vesicle trafficking of biological molecules that are uniquely important to legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The genome sequence of <italic> Pseudoplusia includens single nucleopolyhedrovirus </italic> and an analysis of <italic> p26 </italic> gene evolution in the baculoviruses</h2>
			<h3>PMID:25765042</h3>
				

				
					<h3>PMCID:4346127</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Craveiro </surname>  <given-names> Saluana R </given-names>  </name>  <address>  <email> saluanarocha@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Inglis </surname>  <given-names> Peter W </given-names>  </name>  <address>  <email> peterwinglis@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Togawa </surname>  <given-names> Roberto C </given-names>  </name>  <address>  <email> roberto.togawa@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grynberg </surname>  <given-names> Priscila </given-names>  </name>  <address>  <email> priscila.grynberg@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Melo </surname>  <given-names> Fernando L </given-names>  </name>  <address>  <email> flucasmelo@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ribeiro </surname>  <given-names> Zilda Maria A </given-names>  </name>  <address>  <email> zilda.ribeiro@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ribeiro </surname>  <given-names> Bergmann M </given-names>  </name>  <address>  <email> bergmann.ribeiro@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000e1;o </surname>  <given-names> S&#x000f4;nia N </given-names>  </name>  <address>  <email> snbao2009@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Castro </surname>  <given-names> Maria Elita B </given-names>  </name>  <address>  <email> elita.castro@embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Embrapa Recursos Gen&#x000e9;ticos e Biotecnologia, Parque Esta&#x000e7;&#x000e3;o Biol&#x000f3;gica, W5 Norte Final, 70770-917, Bras&#x000ed;lia, DF Brazil  </aff>  <aff id="Aff2">  <label/> Departamento de Biologia Celular, Universidade de Bras&#x000ed;lia-UnB, Bras&#x000ed;lia, DF Brazil  </aff>
				<br/> 
				年份：<day> 4 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Pseudoplusia includens single nucleopolyhedrovirus (PsinSNPV-IE) is a baculovirus recently identified in our laboratory, with <span class="trait">high</span> pathogenicity to the soybean looper, Chrysodeixis includens (Lepidoptera: Noctuidae) (Walker, 1858). In Brazil, the C. includens caterpillar is an emerging pest and has caused significant losses in soybean and cotton crops. The PsinSNPV genome was determined and the phylogeny of the p26 gene within the family Baculoviridae was investigated. RESULTS: The complete genome of PsinSNPV was sequenced (Roche 454 <span class="gene">GS</span> FLX - Titanium platform), annotated and compared with other Alphabaculoviruses, displaying a genome apparently different from other baculoviruses so far sequenced. The circular double-stranded DNA genome is 139,132 bp in length, with a GC content of 39.3 % and contains 141 open reading frames (ORFs). PsinSNPV possesses the 37 conserved baculovirus core genes, 102 genes found in other baculoviruses and 2 unique ORFs. Two baculovirus repeat ORFs (bro) homologs, bro-a (Psin33) and bro-b (Psin69), were identified and compared with Chrysodeixis chalcites nucleopolyhedrovirus (ChchNPV) and Trichoplusia ni single nucleopolyhedrovirus (TnSNPV) bro genes and showed <span class="trait">high</span> similarity, suggesting that these genes may be derived from an ancestor common to these viruses. The homologous repeats (hrs) are absent from the PsinSNPV genome, which is also the case in ChchNPV and TnSNPV. Two p26 gene homologs (p26a and p26b) were found in the PsinSNPV genome. P26 is thought to be required for optimal virion occlusion in the occlusion bodies (OBs), but its function is not well characterized. The P26 phylogenetic tree suggests that this gene was obtained from three independent acquisition events within the Baculoviridae family. The presence of a signal peptide only in the PsinSNPV p26a/ORF-20 homolog indicates distinct function between the two P26 proteins. CONCLUSIONS: PsinSNPV has a genomic sequence apparently different from other baculoviruses sequenced so far. The complete genome sequence of PsinSNPV will provide a valuable resource, contributing to studies on its molecular biology and functional genomics, and will promote the development of this virus as an effective bioinsecticide.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The complete genome of PsinSNPV was sequenced (Roche 454 <span class="gene">GS</span> FLX &#x02013; Titanium platform), annotated and compared with other Alphabaculoviruses, displaying a genome apparently different from other baculoviruses so far sequenced. The circular double-stranded DNA genome is 139,132&#x000a0;bp in length, with a GC content of 39.3 % and contains 141 open reading frames (ORFs). PsinSNPV possesses the 37 conserved baculovirus core genes, 102 genes found in other baculoviruses and 2 unique ORFs. Two <italic> baculovirus repeat ORFs </italic> ( <italic> bro </italic> ) homologs, <italic> bro-a </italic> (Psin33) and <italic> bro-b </italic> (Psin69), were identified and compared with <italic> Chrysodeixis chalcites nucleopolyhedrovirus </italic> (ChchNPV) and <italic> Trichoplusia ni single nucleopolyhedrovirus </italic> (TnSNPV) <italic> bro </italic> genes and showed <span class="trait">high</span> similarity, suggesting that these genes may be derived from an ancestor common to these viruses. The homologous repeats ( <italic> hrs </italic> ) are absent from the PsinSNPV genome, which is also the case in ChchNPV and TnSNPV. Two <italic> p26 </italic> gene homologs ( <italic> p26a </italic> and <italic> p26b </italic> ) were found in the PsinSNPV genome. P26 is thought to be required for optimal virion occlusion in the occlusion bodies (OBs), but its function is not well characterized. The P26 phylogenetic tree suggests that this gene was obtained from three independent acquisition events within the <italic> Baculoviridae </italic> family. The presence of a signal peptide only in the PsinSNPV <italic> p26a </italic> /ORF-20 homolog indicates distinct function between the two P26 proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						The baculovirus genes are categorized based on their functions during the viral cycle as follows: DNA replication, RNA transcription, ODV and BV structural proteins or oral infectivity proteins [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Baculovirus genome replication mechanisms are still not fully understood. Several studies have been developed to <span class="gene">try</span> to identify the genes responsible for DNA replication and translation. The essential DNA replication factors <italic> late expression factor 1 </italic> ( <italic> lef-1 </italic> ), <italic> lef-2 </italic> , <italic> lef-3 </italic> , <italic> DNA polymerase </italic> ( <italic> dnapol </italic> ), <italic> p6.9 </italic> , <italic> 38&#x000a0;k </italic> , <italic> helicase </italic> ( <italic> hel </italic> ) and <italic> immediate early 1 </italic> ( <italic> ie-1 </italic> ) homologs are all present in the PsinSNPV genome. In addition, the PsinSNPV genome contains genes homologous to <italic> proliferating cell nuclear antigen </italic> ( <italic> pcna </italic> ), <italic> major early-transcribed <span class="trait">protein</span> 53 </italic> ( <italic> me53 </italic> ), <italic> DNA binding <span class="trait">protein</span> </italic> ( <italic> dbp </italic> ), <italic> alkaline exonuclease </italic> ( <italic> alkexo </italic> ) and <italic> exon-0/ie-0 </italic> , which were not identified in all baculoviruses but may influence viral DNA replication [ <xref ref-type="bibr" rid="CR11"> 11 </xref> , <xref ref-type="bibr" rid="CR27"> 27 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						PsinSNPV belongs to a group where the <italic> <span class="gene">phr</span> </italic> gene is conserved and, as expected, its genome encodes a CPD photolyase <span class="trait">protein</span> (Psin68). The complete nucleotide sequence of the PsinSNPV <italic> <span class="gene">phr</span> </italic> gene is of 1,512&#x000a0;bp with GC %&#x02009;=&#x02009;36.2%. The deduced <span class="gene">PHR</span> amino acid sequence of PsinSNPV, ChchNPV- PHR1, &#x02212;PHR2, and TnSNPV were aligned, revealing that the PsinSNPV photolyase possesses <span class="trait">high</span> identity to TnSNPV and ChchNPV- <span class="gene">PHR1</span> (Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ). Previous studies showed that ChchNPV - <span class="gene">PHR1</span> is not active when tested in an <italic> Escherichia coli </italic> photolyase deficient strain [ <xref ref-type="bibr" rid="CR40"> 40 </xref> ]. The active copy (PHR-2) is distinct in its possession of two conserved tryptophan residues, which may be involved in an electron transfer mechanism [ <xref ref-type="bibr" rid="CR40"> 40 </xref> ]. In the PsinSNPV photolyase <span class="trait">protein,</span> the tryptophan residues are replaced by histidine and tyrosine in positions 368 and 370 aa, respectively (Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> ). Therefore, both tryptophans are absent in the PsinSNPV photolyase, suggesting that this <span class="trait">protein</span> might not be active. The partial <span class="gene">PHR</span> amino acid sequences of PsinNPV &#x02013;GT1 (EU401912); &#x02212;GT2 [GenBank: EU682272], PsinNPV - USA [GenBank: EU401913] and PsinSNPV-IA to -IG isolates described in the literature [ <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR39"> 39 </xref> ] were aligned, where PsinNPV-GT1 and PsinNPV &#x02013; USA isolates showed <span class="trait">high</span> similarity to the PsinSNPV-IA to -IG isolates. Interestingly, in contrast to other PsinNPV isolates reported so far, PsinNPV - GT2 possesses a tryptophan residue at position 368, which is thought to be essential for enzyme activity (data not shown). Further studies are needed to confirm and investigate the activity of the PsinSNPV - GT2 photolyase.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification and expression analysis of the CPP-like gene family in soybean.</h2>
			<h3>PMID:25730064  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang L(1), Zhao HK(2), Wang YM(1), Yuan CP(1), Zhang YY(1), Li HY(1), Yan XF(3), Li QY(4), Dong YS(5).
				<br/> 
				年份：2015 Feb 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cysteine-rich polycomb-like <span class="trait">protein</span> (CPP-like) genes are a group of transcription factors with highly conserved cysteine-rich domains and are widely distributed in animals and plants, but do not present in yeast. Previous studies have shown that members of this family play important roles in the development of reproductive tissue and in the control of cell division in plants. In this study, whole genome identification of soybean CPP transcription factors was performed using bioinformatic methods. The results showed that there were 20 CPP transcription factors in the soybean genome, which encoded for 28 distinct CPP proteins. These transcription factors were distributed on 16 of 20 chromosomes. Phylogenetic relationship analysis showed that expression of CPP gene family members occurred before the differentiation of monocotyledons and dicotyledons. RNA-Seq analysis showed that 5 genes were highly expressed in all tissues, including <span class="gene">Glyma10g39080,</span> <span class="gene">Glyma01g44670,</span> Glyma101g66920, <span class="gene">Glyma02g01540,</span> and <span class="gene">Glyma20g28740.</span> One gene <span class="gene">(Glyma14g14750)</span> was specifically expressed in young leaves, while 2 genes <span class="gene">(Glyma02g01540</span> and <span class="gene">Glyma10g01580)</span> were highly expressed in root nodules. Quantitative reverse transcriptase-polymerase chain reaction analysis revealed that the expression levels of most genes increased in the roots under <span class="trait">high</span> temperature stress. Our findings indicate that these genes are not only involved in growth and development, but also in the responses to <span class="trait">high</span> temperature stress in soybean roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ectopic phytocystatin expression increases nodule numbers and influences the responses of soybean (Glycine max) to nitrogen deficiency.</h2>
			<h3>PMID:25659749  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Quain MD(1), Makgopa ME(2), Cooper JW(3), Kunert KJ(2), Foyer CH(4).
				<br/> 
				年份：2015 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cysteine proteases and cystatins have many functions that remain poorly characterised, particularly in crop plants. We therefore investigated the responses of these proteins to nitrogen deficiency in wild-type soybeans and in two independent transgenic soybean lines (OCI-1 and OCI-2) that express the rice cystatin, oryzacystatin-I (OCI). Plants were grown for four weeks under either a <span class="trait">high</span> (5 mM) nitrate (HN) regime or in the absence of added nitrate (LN) in the absence or presence of symbiotic rhizobial bacteria. Under the LN regime all lines showed similar classic symptoms of nitrogen deficiency including lower shoot biomass and leaf chlorophyll. However, the LN-induced decreases in leaf <span class="trait">protein</span> and increases in root <span class="trait">protein</span> tended to be smaller in the OCI-1 and OCI-2 lines than in the wild type. When LN-plants were grown with rhizobia, OCI-1 and OCI-2 roots had significantly more crown nodules than wild-type plants. The growth nitrogen regime had a significant effect on the abundance of transcripts encoding vacuolar processing enzymes (VPEs), LN-dependent increases in <span class="gene">VPE2</span> and VPE3 transcripts in all lines. However, the LN-dependent increases of <span class="gene">VPE2</span> and VPE3 transcripts were significantly lower in the leaves of OCI-1 and OCI-2 plants than in the wild type. These results show that nitrogen availability regulates the leaf and root cysteine protease, VPE and cystatin transcript profiles in a manner that is in some cases influenced by ectopic OCI expression. Moreover, the OCI-dependent inhibition of papain-like cysteine proteases favours increased nodulation and enhanced tolerance to nitrogen limitation, as shown by the smaller LN-dependent decreases in leaf <span class="trait">protein</span> observed in the OCI-1 and OCI-2 plants relative to the wild type.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Industrial potential of lipoxygenases.</h2>
			<h3>PMID:25641326  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heshof R(1), de Graaff LH(1), Villaverde JJ(2)(3), Silvestre AJ(2), Haarmann T(4), Dalsgaard TK(5), Buchert J(6).
				<br/> 
				年份：2016 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Lipoxygenases (LOXs) are iron- or manganese-containing oxidative enzymes found in plants, animals, bacteria and fungi. LOXs catalyze the oxidation of polyunsaturated fatty acids to the corresponding highly reactive hydroperoxides. Production of hydroperoxides by <span class="gene">LOX</span> can be exploited in different applications such as in bleaching of colored components, modification of lipids originating from different raw materials, production of lipid derived chemicals and production of aroma compounds. Most application research has been carried out using soybean LOX, but currently the use of microbial LOXs has also been reported. Development of <span class="gene">LOX</span> composition with <span class="trait">high</span> activity by heterologous expression in suitable production hosts would enable full exploitation of the potential of <span class="gene">LOX</span> derived reactions in different applications. Here, we review the biological role of LOXs, their heterologous production, as well as potential use in different applications. LOXs may fulfill an important role in the design of processes that are far more environmental friendly than currently used chemical reactions. Difficulties in screening for the optimal enzymes and producing <span class="gene">LOX</span> enzymes in sufficient amounts prevent large-scale application so far. With this review, we summarize current knowledge of <span class="gene">LOX</span> enzymes and the way in which they can be produced and applied.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:"Potential health benefits of lunasin: a multifaceted soy-derived bioactive peptide".</h2>
			<h3>PMID:25627564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lule VK(1), Garg S, Pophaly SD, Hitesh, Tomar SK.
				<br/> 
				年份：2015 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bioactive peptides are small <span class="trait">protein</span> fragments derived from enzymatic hydrolysis of food proteins, fermentation with proteolytic starter cultures, and gastrointestinal digestion. These peptides have positive impacts on a number of physiological functions in living beings. Lunasin, a soy-derived bioactive peptide, is one of the most promising among them. Lunasin encoded within 2S albumin <span class="gene">(GM2S-1)</span> gene, identified as a novel peptide extracted from soybean seed. It is composed of 43 amino acid residues with a molecular weight of 5.5 kDa. Extensive scientific studies have shown that lunasin possesses inherent antioxidative, anti-inflammatory, anticancerous properties and could also play a vital role in regulating of cholesterol biosynthesis in the body. Its <span class="trait">high</span> bioavailability and heat stable nature allow its potential use as dietary supplement. The present review summarizes some of the potential health and therapeutic benefits of lunasin reported hitherto.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of duplicated Symbiosis Receptor Kinase (SymRK) genes during nodulation and mycorrhizal infection in soybean (Glycine max).</h2>
			<h3>PMID:25617765  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Indrasumunar A(1), Wilde J(1), Hayashi S(1), Li D(1), Gresshoff PM(2).
				<br/> 
				年份：2015 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Association between legumes and rhizobia results in the formation of root nodules, where symbiotic nitrogen fixation occurs. The early stages of this association involve a complex of signalling events between the host and microsymbiont. Several genes dealing with early signal transduction have been cloned, and one of them encodes the leucine-rich repeat (LRR) receptor kinase (SymRK; also termed NORK). The Symbiosis Receptor Kinase gene is required by legumes to establish a root endosymbiosis with Rhizobium bacteria as well as mycorrhizal fungi. Using degenerate primer and BAC sequencing, we cloned duplicated SymRK homeologues in soybean called <span class="gene">GmSymRKα</span> and <span class="gene">GmSymRKβ.</span> These duplicated genes have <span class="trait">high</span> similarity of nucleotide (96%) and amino acid sequence (95%). Sequence analysis predicted a malectin-like domain within the extracellular domain of both genes. Several putative cis-acting elements were found in promoter regions of <span class="gene">GmSymRKα</span> and <span class="gene">GmSymRKβ,</span> suggesting a participation in lateral root development, cell division and peribacteroid membrane formation. The mutant of SymRK genes is not available in soybean; therefore, to know the functions of these genes, RNA interference (RNAi) of these duplicated genes was performed. For this purpose, RNAi construct of each gene was generated and introduced into the soybean genome by Agrobacterium rhizogenes-mediated hairy root transformation. RNAi of <span class="gene">GmSymRKβ</span> gene resulted in an increased reduction of nodulation and mycorrhizal infection than RNAi of <span class="gene">GmSymRKα,</span> suggesting it has the major activity of the duplicated gene pair. The results from the important crop legume soybean confirm the joint phenotypic action of <span class="gene">GmSymRK</span> genes in both mycorrhizal and rhizobial infection seen in model legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association between co-inhibitory molecule gene tagging single nucleotide polymorphisms and the risk of colorectal cancer in Chinese.</h2>
			<h3>PMID:25604582  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ge J(1), Zhu L, Zhou J, Li G, Li Y, Li S, Wu Z, Rong J, Yuan H, Liu Y, Chi Q, Piao D, Zhao Y, Cui B.
				<br/> 
				年份：2015 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						PURPOSE: T lymphocyte immune responses are controlled by both co-stimulatory and co-inhibitory signaling through T cell co-receptors. Cytotoxic T lymphocyte antigen-4 (CTLA-4), programmed death 1 (PD-1) and B and T lymphocyte attenuator (BTLA) are all co-inhibitory molecules that negatively regulate the activation of T cells. In this study, we investigated the relationship between ten tagging SNPs in three co-inhibitory molecule genes and colorectal cancer (CRC). METHODS: We conducted a hospital-based case-control study consisting of 601 cases with CRC and 627 CRC-free individuals from the Heilongjiang Province of China. RESULTS: The rs7421861 CT genotype was significantly associated with the risk of colorectal cancer compared to the wild-type TT genotype (adjusted OR 1.314, 95% <span class="gene">CI</span> 1.012-1.706, P = 0.041). The rs2705535 TT genotype was associated with the risk of rectal cancer [OR 1.819 (1.093-3.027), P = 0.021]. There was statistical interaction between the PD-1/rs2227982 (CT + TT) genotypes and <span class="trait">high</span> seafood intake (&gt;once/week), as well as the CTLA-4/rs231777 variant and <span class="trait">high</span> pungent food intake (&gt;3 times/week). The AG + AA genotypes of CTLA-4/rs3087243 statistically and antagonistically interacted with soybeans, pork and alcohol intake and were associated with CRC risk. Analogously, BTLA/rs1844089 interacted with pork intake, PD-1/rs7421861 with beef and lamb consumption and PD-1/rs6710479 with barbecue consumption. Haplotype G-C-G-A-T-T-A was significantly associated with CRC risk (OR 1.221 P = 0.034). CONCLUSIONS: These data indicate potential associations between BTLA and PD-1 polymorphisms and CRC susceptibility. Additionally, the three co-inhibitory molecule gene SNPs have environmental interactions associated with CRC risk.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel soybean (Glycine max) gene encoding a family 3 β-glucosidase has <span class="trait">high</span> isoflavone 7-O-glucoside-hydrolyzing activity in transgenic rice.</h2>
			<h3>PMID:25569564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hsu CC(1), Wu TM, Hsu YT, Wu CW, Hong CY, Su NW.
				<br/> 
				年份：2015 Jan 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A previous study demonstrated that purified Glycine max β-glucosidase <span class="gene">(GmBGL)</span> could hydrolyze glucosyl isoflavone to the aglyconic form. This study reports the cloning and functional characterization of a soybean cDNA encoding the β-glucosidase. <span class="gene">GmBGL</span> was isolated by use of a purified soybean N-terminal amino acid sequence and conserved sequences of β-glucosidase genes from other plants. Sequence analysis of <span class="gene">GmBGL</span> revealed an open reading frame of 1884 bp encoding a polypeptide of 627 amino acids with a calculated molecular mass of 69 kDa. Phylogenetic analysis classified the <span class="gene">GmBGL</span> into the glycosyl hydrolase 3 family. In soybean, the <span class="gene">GmBGL</span> transcript was predominantly accumulated in roots and leaves. To examine the enzymatic activity and substrate specificity, <span class="gene">GmBGL</span> was ectopically expressed in transgenic rice. Purified <span class="gene">GmBGL</span> <span class="trait">protein</span> from transgenic rice could catalyze the hydrolysis of genistin and daidzin to produce genistein and daidzein, respectively, which confirmed <span class="gene">GmBGL</span> as a functional β-glucosidase with isoflavone glucoside-hydrolyzing activity. This paper reveals that <span class="gene">GmBGL</span> is a key enzyme in transforming glucosyl isoflavones to aglycones in soybean, which may help in genetic manipulation of aglycone-rich soybean seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in the levels of off-flavor generation in soybean through biotic elicitor treatments.</h2>
			<h3>PMID:25552290  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kumari S(1), Dahuja A, Vinutha T, Lal SK, Kar A, Rai RD.
				<br/> 
				年份：2015 Jan 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nutritional benefits of soybean remain underutilized as the off-flavor present in it limits the consumption and acceptability among people. The aim of the present study was to unveil the effect of the phytohormones methyl jasmonate (MJ: 0, 50 μM, 1 mM, and 15 mM) and salicylic acid (SA: 0, 50 μM, 0.1 mM, and 10 mM) as elicitors on two contrasting off-flavor soybean varieties at different growth stages (1, bloom; 2, pod development; 3, seed development). The effects of two elicitors varied widely and were found to be dose dependent and growth stage independent. SA reduces the lipoxygenase (LOX) and hydroperoxide lyase (HPL) activity, which in turn resulted in reduction in the TBA number and carbonyl value in contrast to MJ. SA 0.1 mM is the most effective dose in reduction of off-flavor determining parameters and <span class="trait">protein</span> oxidation, and it reduces the <span class="gene">LOX</span> and <span class="gene">HPL</span> activity by 2.3- and 2.4-fold, respectively in <span class="trait">"high</span> off-flavor" cultivar 'Bragg' compared to "low off-flavor" cultivar 'DS 2706' which showed 1.4- and 2.1-fold, respectively. This reduction in <span class="trait">protein</span> oxidation is also supported by enhanced content of antioxidant enzymes. Thus, phytohormone SA can be used in reduction of off-flavor generation, more effectively than MJ treatments, in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean DREB1/CBF-type transcription factors function in heat and <span class="trait">drought</span> as well as cold stress-responsive gene expression.</h2>
			<h3>PMID:25495120  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kidokoro S(1), Watanabe K, Ohori T, Moriwaki T, Maruyama K, Mizoi J, Myint Phyu Sin Htwe N, Fujita Y, Sekita S, Shinozaki K, Yamaguchi-Shinozaki K.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is a globally important crop, and its growth and <span class="trait">yield</span> are severely reduced by abiotic stresses, such as <span class="trait">drought,</span> heat, and cold. The cis-acting element DRE (dehydration-responsive element)/CRT plays an important role in activating gene expression in response to these stresses. The Arabidopsis DREB1/CBF genes that encode DRE-binding proteins function as transcriptional activators in the cold stress responsive gene expression. In this study, we identified 14 DREB1-type transcription factors <span class="gene">(GmDREB1s)</span> from a soybean genome database. The expression of most <span class="gene">GmDREB1</span> genes in soybean was strongly induced by a variety of abiotic stresses, such as cold, <span class="trait">drought,</span> <span class="trait">high</span> salt, and heat. The <span class="gene">GmDREB1</span> proteins activated transcription via DREs (dehydration-responsive element) in Arabidopsis and soybean protoplasts. Transcriptome analyses using transgenic Arabidopsis plants overexpressing <span class="gene">GmDREB1s</span> indicated that many of the downstream genes are cold-inducible and overlap with those of Arabidopsis DREB1A. We then comprehensively analyzed the downstream genes of <span class="gene">GmDREB1B;1,</span> which is closely related to DREB1A, using a transient expression system in soybean protoplasts. The expression of numerous genes induced by various abiotic stresses were increased by overexpressing <span class="gene">GmDREB1B;1</span> in soybean, and DREs were the most conserved element in the promoters of these genes. The downstream genes of <span class="gene">GmDREB1B;1</span> included numerous soybean-specific stress-inducible genes that encode an ABA receptor family <span class="trait">protein,</span> <span class="gene">GmPYL21,</span> and translation-related genes, such as ribosomal proteins. We confirmed that <span class="gene">GmDREB1B;1</span> directly activates <span class="gene">GmPYL21</span> expression and enhances ABRE-mediated gene expression in an ABA-independent manner. These results suggest that <span class="gene">GmDREB1</span> proteins activate the expression of numerous soybean-specific stress-responsive genes under diverse abiotic stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Catalytic features and crystal structure of a tau class glutathione transferase from Glycine max specifically upregulated in response to soybean mosaic virus infections.</h2>
			<h3>PMID:25479053  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Skopelitou K(1), Muleta AW(2), Papageorgiou AC(2), Chronopoulou E(1), Labrou NE(3).
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant tau class glutathione transferases (GSTs) play important roles in biotic and abiotic stress tolerance in crops and weeds. In this study, we systematically examined the catalytic and structural features of a <span class="gene">GST</span> isoenzyme from Glycine max <span class="gene">(GmGSTU10-10).</span> <span class="gene">GmGSTU10-10</span> is a unique isoenzyme in soybean that is specifically expressed in response to biotic stress caused by soybean mosaic virus (SMV) infections. <span class="gene">GmGSTU10-10</span> was cloned, expressed in Escherichia coli, purified and characterized. The results showed that <span class="gene">GmGSTU10-10</span> catalyzes several different reactions and exhibits wide substrate specificity. Of particular importance is the finding that the enzyme shows <span class="trait">high</span> antioxidant catalytic function and acts as hydroperoxidase. In addition, its Km for GSH is significantly lower, compared to other plant GSTs, suggesting that <span class="gene">GmGSTU10-10</span> is able to perform efficient catalysis under conditions where the concentration of reduced glutathione is low (e.g. oxidative stress). The crystal structure of <span class="gene">GmGSTU10-10</span> was solved by molecular replacement at 1.6Å resolution in complex with glutathione sulfenic acid (GSOH). Structural analysis showed that <span class="gene">GmGSTU10-10</span> shares the same overall fold and domain organization as other plant cytosolic GSTs; however, major variations were identified in helix H9 and the upper part of helix H4 that affect the size of the active site pockets, substrate recognition and the catalytic mechanism. The results of the present study provide new information into <span class="gene">GST</span> diversity and give further insights into the complex regulation and enzymatic functions of this plant gene superfamily.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nitrate inhibits soybean nodulation by regulating expression of CLE genes.</h2>
			<h3>PMID:25443828  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lim CW(1), Lee YW(2), Lee SC(3), Hwang CH(4).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen compounds such as nitrate act as a potential inhibitor for legume nodulation. In this study, we isolated a new CLE gene, <span class="gene">GmNIC2,</span> from nitrate-treated roots, which shares <span class="trait">high</span> sequence homology with nitrate-induced CLE gene <span class="gene">GmNIC1.</span> Similar to <span class="gene">GmNIC1,</span> the expression level of <span class="gene">GmNIC2</span> was not significantly altered in roots by rhizobial inoculation and was much higher in young nodules than in roots. In addition, overexpression of <span class="gene">GmNIC2</span> led to similar nodulation inhibition of transgenic hairy roots to that of <span class="gene">GmNIC1,</span> which occurred in <span class="gene">GmNARK-dependent</span> manner and at the local level. By analyzing <span class="gene">GmNARK</span> loss-of-function mutant, SS2-2, it was found that expression levels of <span class="gene">GmNIC1</span> and <span class="gene">GmNIC2</span> in the SS2-2 roots were lower than in the wild type (WT) roots in response to nitrate. In contrast to <span class="gene">GmNIC1</span> and <span class="gene">GmNIC2,</span> expressions of <span class="gene">GmRIC1</span> and <span class="gene">GmRIC2</span> genes that are related to the autoregulation of nodulation (AON) were strongly suppressed both of the soybeans during all periods of nitrate treatment and even were not induced by additional inoculation with rhizobia. Taken together, the results of this study suggest that <span class="gene">GmNIC2,</span> as an active homologous gene located in chromosome 13, acts locally to suppress nodulation, like <span class="gene">GmNIC1,</span> and nitrate inhibition of nodulation is led by fine-tuned regulation of both nitrate-induced CLEs and rhizobia-induced CLEs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the Hsf family in soybean and functional identification of <span class="gene">GmHsf-34</span> involvement in <span class="trait">drought</span> and heat stresses.</h2>
			<h3>PMID:25416131  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li PS, Yu TF, He GH, Chen M, Zhou YB, Chai SC(1), Xu ZS, Ma YZ.
				<br/> 
				年份：2014 Nov 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">High</span> temperature affects organism growth and metabolic activity. Heat shock transcription factors (Hsfs) are key regulators in heat shock response in eukaryotes and prokaryotes. Under <span class="trait">high</span> temperature conditions, Hsfs activate heat shock proteins (Hsps) by combining with heat stress elements (HSEs) in their promoters, leading to defense of heat stress. Since the first plant Hsf gene was identified in tomato, several plant Hsf family genes have been thoroughly characterized. Although soybean (Glycine max), an important oilseed crops, genome sequences have been available, the Hsf family genes in soybean have not been characterized accurately. RESULT: We analyzed the Hsf genetic structures and <span class="trait">protein</span> function domains using the GSDS, Pfam, SMART, PredictNLS, and NetNES online tools. The genome scanning of dicots (soybean and Arabidopsis) and monocots (rice and maize) revealed that the whole-genome replication occurred twice in soybean evolution. The plant Hsfs were classified into 3 classes and 16 subclasses according to <span class="trait">protein</span> structure domains. The A8 and B3 subclasses existed only in dicots and the A9 and <span class="gene">C2</span> occurred only in monocots. Thirty eight soybean Hsfs were systematically identified and grouped into 3 classes and 12 subclasses, and located on 15 soybean chromosomes. The promoter regions of the soybean Hsfs contained cis-elements that likely participate in <span class="trait">drought,</span> low temperature, and ABA stress responses. There were large differences among Hsfs based on transcriptional levels under the stress conditions. The transcriptional levels of the A1 and A2 subclass genes were extraordinarily <span class="trait">high.</span> In addition, differences in the expression levels occurred for each gene in the different organs and at the different developmental stages. Several genes were chosen to determine their subcellular localizations and functions. The subcellular localization results revealed that <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> were located in the nucleus. Overexpression of the <span class="gene">GmHsf-34</span> gene improved the tolerances to <span class="trait">drought</span> and heat stresses in Arabidopsis plants. CONCLUSIONS: This present investigation of the quantity, structural features, expression characteristics, subcellular localizations, and functional roles provides a scientific basis for further research on soybean Hsf functions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Connecting Lipoxygenase Function to Structure by Electron Paramagnetic Resonance</h2>
			<h3>PMID:25341190</h3>
				

				
					<h3>PMCID:4270396</h3>
					<h4>期刊:Accounts of Chemical Research</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="ath1">  <name>  <surname> Gaffney </surname>  <given-names> Betty J. </given-names>  </name>  <xref rid="cor1" ref-type="other"> * </xref>  </contrib>  <aff id="aff1"> Department
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						CONSPECTUS: Lipoxygenase enzymes insert oxygen in a polyunsaturated lipid, yielding a hydroperoxide product. When the acyl chain is arachidonate, with three cis-pentadiene units, 12 positionally and stereochemically different products might result. The plant lipids, linoleate and linolenate, have, respectively, four and eight potential oxygen insertion sites. The puzzle of how specificity is achieved in these reactions grows as more and more <span class="trait">protein</span> structures confirm the conservation of a lipoxygenase <span class="trait">protein</span> fold in plants, animals, and bacteria. Lipoxygenases are large enough (60-100 kDa) that they provide a <span class="trait">protein</span> shell completely surrounding an active site cavity that has the shape of a long acyl chain and contains a catalytic metal (usually iron). This Account summarizes electron paramagnetic resonance (EPR) spectroscopic, and other, experiments designed to bridge the <span class="gene">gap</span> between lipid-lipoxygenase interactions in solution and crystal structures. Experiments with spin-labeled lipids give a picture of bound lipids tethered to <span class="trait">protein</span> by an acyl chain, but with a polar end emerging from the cavity to solvent exposure, where the headgroup is highly flexible. The location of a spin on the polar end of a lysolecithin was determined by pulsed, dipolar EPR measurements, by representing the <span class="trait">protein</span> structure as a five-point grid of spin-labels with coordinates derived from 10 distance determinations between spin pairs. Distances from the lipid spin to each grid site completed a six-point representation of the enzyme with a bound lipid. Insight into the dynamics that allow substrate/product to enter/exit the cavity was obtained with a different set of spin-labeled <span class="trait">protein</span> mutants. Once substrate enters the cavity, the rate-limiting step of catalysis involves redox cycling at the metal center. Here, a mononuclear iron cycles between ferric and ferrous (high-spin) forms. Two helices provide pairs of side-chain ligands to the iron, resulting in characteristic EPR signals. Quantitative comparison of EPR spectra of plant and bacterial lipoxygenases has suggested conservation of a unique geometry of lipoxygenase iron centers. <span class="trait">High</span> frequency (94 GHz) EPR is consistent with a similar metal center in a manganese version of lipoxygenase. Overall, established and emerging EPR experiments have been developed and applied to the lipoxygenase family of enzymes to elucidate changes in the solution structures that are related to function.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						This Account summarizes electron paramagnetic resonance (EPR) spectroscopic, and other, experiments designed to bridge the <span class="gene">gap</span> between lipid&#x02013;lipoxygenase interactions in solution and crystal structures. Experiments with spin-labeled lipids give a picture of bound lipids tethered to <span class="trait">protein</span> by an acyl chain, but with a polar end emerging from the cavity to solvent exposure, where the headgroup is highly flexible. The location of a spin on the polar end of a lysolecithin was determined by pulsed, dipolar EPR measurements, by representing the <span class="trait">protein</span> structure as a five-point grid of spin-labels with coordinates derived from 10 distance determinations between spin pairs. Distances from the lipid spin to each grid site completed a six-point representation of the enzyme with a bound lipid. Insight into the dynamics that allow substrate/product to enter/exit the cavity was obtained with a different set of spin-labeled <span class="trait">protein</span> mutants. Once substrate enters the cavity, the rate-limiting step of catalysis involves redox cycling at the metal center. Here, a mononuclear iron cycles between ferric and ferrous (high-spin) forms. Two helices provide pairs of side-chain ligands to the iron, resulting in characteristic EPR signals. Quantitative comparison of EPR spectra of plant and bacterial lipoxygenases has suggested conservation of a unique geometry of lipoxygenase iron centers. <span class="trait">High</span> frequency (94 GHz) EPR is consistent with a similar metal center in a manganese version of lipoxygenase. Overall, established and emerging EPR experiments have been developed and applied to the lipoxygenase family of enzymes to elucidate changes in the solution structures that are related to function.
					</p>
				
					<p>&nbsp;&nbsp;
						The author is deeply indebted to Ph.D. students whose dissertations and papers, over 1986&#x02013;2014, made this review possible. They are Dimitrios V. Mavrophilipos (initiated studies of SBL1), An-Suei Yang (computed EPR spectra of iron centers with distributions of zero-field splitting), Brendan C. Maguire (cloned lipoxygenase from soybeans), Jeffrey C. Boyington (solved lipoxygenase crystal structure with L. M. Amzel), Kutbuddin S. Doctor (simulated <span class="trait">high-</span> and low-frequency iron EPR), Fayi Wu (constructed no-cysteine SBL1 and determined affinity for doxyl stearates), and, currently, Miles D. Bradshaw (finding location of entry to the active site in SBL1). Earlier, Shaw T. Chen and Gary L. Willingham set the stage (membrane studies). The author&#x02019;s work has been supported by The National Institutes of Health grants, presently by <span class="gene">GM065268</span> to B.J.G.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic plasticity of HSP70s gene expression during diapause: signs of evolutionary responses to cold stress among Soybean Pod Borer populations (Leguminivora glycinivorella) in Northeast of China.</h2>
			<h3>PMID:25330365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang L(1), Yang S(1), Han L(1), Fan D(1), Zhao K(1).
				<br/> 
				年份：2014 Oct 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean pod borer (Leguminivora glycinivorella Matsumura) successfully survives the winter because of its <span class="trait">high</span> expression of 70-kDa heat shock proteins (HSP70s) during its overwintering diapause. The amount of HSP70s is different under different environmental stresses. In this study, inducible heat shock <span class="trait">protein</span> 70 and its constitutive heat shock cognate 70 were cloned by RT-PCR and RACE. These genes were named Lg-hsp70 and Lg-hsc70, respectively. Gene transcription and <span class="trait">protein</span> expression after cold stress treatment (5°C to -5°C) were analyzed by western blotting and by qRT-PCR for four populations that were sampled in the northeast region of China, including Shenyang, Gongzhuling, Harbin and Heihe, when the soybean pod borer was in diapause. As the cold shock temperature decreased, the levels of Lg-HSP70s were significantly up-regulated. The amount of cold-induced Lg-HSP70s was highest in the southernmost population (Shenyang, 41°50'N) and lowest in the northernmost population (Heihe, 50°22'N). These results support the hypothesis that the soybean pod borer in the northeast region of China displays phenotypic plasticity, and the accumulation of Lg-HSP70s is a strategy for overcoming environmental stress. These results also suggest that the induction of <span class="gene">HSP70</span> synthesis, which is a complex physiological adaptation, can evolve quickly and inherit stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The intron and 5' distal region of the soybean <span class="gene">Gmubi</span> promoter contribute to very <span class="trait">high</span> levels of gene expression in transiently and stably transformed tissues.</h2>
			<h3>PMID:25292438  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：De La Torre CM(1), Finer JJ.
				<br/> 
				年份：2015 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: An extended version of an intron-containing soybean polyubiquitin promoter gave very <span class="trait">high</span> levels of gene expression using three different validation tools. The intron-containing Glycine max polyubiquitin promoter <span class="gene">(Gmubi)</span> is able to regulate expression levels five times higher than the widely used CaMV35S promoter. In this study, eleven <span class="gene">Gmubi</span> derivatives were designed and evaluated to determine which regions contributed to the <span class="trait">high</span> levels of gene expression, observed with this promoter. Derivative constructs regulating GFP were evaluated using transient expression in lima bean cotyledons and stable expression in soybean hairy roots. With both expression systems, removal of the intron in the 5'UTR led to reduced levels of gene expression suggesting a role of the intron in promoter activity. Promoter constructs containing an internal intron duplication and upstream translocations of the intron resulted in higher and similar expression levels to <span class="gene">Gmubi,</span> respectively, indicating the presence of enhancers within the intron. Evaluation of 5' distal extensions of the <span class="gene">Gmubi</span> promoter resulted in significantly higher levels of GFP expression, suggesting the presence of upstream regulatory elements. A twofold increase in promoter strength was obtained when <span class="gene">Gmubi</span> was extended 1.5 kb upstream to generate <span class="gene">GmubiXL</span> (2.4 kb total length). In stably transformed soybean plants containing GFP regulated by CaMV35S, <span class="gene">Gmubi</span> and <span class="gene">GmubiXL,</span> the <span class="gene">GmubiXL</span> promoter clearly produced the highest levels of gene expression, with especially <span class="trait">high</span> GFP fluorescence in the vascular tissue and root tips. Use of <span class="gene">GmubiXL</span> leads to very <span class="trait">high</span> levels of gene expression in soybean and represents a native soybean promoter, which may be useful for regulating transgene expression for both basic and applied research.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Soybean C2H2-Type Zinc Finger Gene <italic> <span class="gene">GmZF1</span> </italic> Enhanced Cold Tolerance in Transgenic <italic> Arabidopsis </italic></h2>
			<h3>PMID:25286048</h3>
				

				
					<h3>PMCID:4186855</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Yu </surname>  <given-names> Guo-Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiang </surname>  <given-names> Lin-Lin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Xue-Feng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao-Shi </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Meng-Meng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Shu-Guang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Xian-Guo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 7 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Zinc finger proteins were involved in response to different environmental stresses in plant species. A typical Cys2/His2-type (C2H2-type) zinc finger gene <span class="gene">GmZF1</span> from soybean was isolated and was composed of 172 amino acids containing two conserved C2H2-type zinc finger domains. Phylogenetic analysis showed that <span class="gene">GmZF1</span> was clustered on the same branch with six C2H2-type ZFPs from dicotyledonous plants excepting for GsZFP1, and distinguished those from monocotyledon species. The <span class="gene">GmZF1</span> <span class="trait">protein</span> was localized at the nucleus, and has specific binding activity with EP1S core sequence, and nucleotide mutation in the core sequence of EPSPS promoter changed the binding ability between <span class="gene">GmZF1</span> <span class="trait">protein</span> and core DNA element, implying that two amino acid residues, G and C boxed in core sequence TGACAGTGTCA possibly play positive regulation role in recognizing DNA-binding sites in <span class="gene">GmZF1</span> proteins. <span class="trait">High</span> accumulation of <span class="gene">GmZF1</span> mRNA induced by exogenous ABA suggested that <span class="gene">GmZF1</span> was involved in an ABA-dependent signal transduction pathway. Over-expression of <span class="gene">GmZF1</span> significantly improved the contents of proline and soluble sugar and decreased the MDA contents in the transgenic lines exposed to cold stress, indicating that transgenic Arabidopsis carrying <span class="gene">GmZF1</span> gene have adaptive mechanisms to cold stress. Over-expression of <span class="gene">GmZF1</span> also increased the expression of cold-regulated cor6.6 gene by probably recognizing protein-DNA binding sites, suggesting that <span class="gene">GmZF1</span> from soybean could enhance the tolerance of Arabidopsis to cold stress by regulating expression of cold-regulation gene in the transgenic Arabidopsis.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Zinc finger proteins were involved in response to different environmental stresses in plant species. A typical Cys2/His2-type (C2H2-type) zinc finger gene <italic> <span class="gene">GmZF1</span> </italic> from soybean was isolated and was composed of 172 amino acids containing two conserved C2H2-type zinc finger domains. Phylogenetic analysis showed that <span class="gene">GmZF1</span> was clustered on the same branch with six C2H2-type ZFPs from dicotyledonous plants excepting for GsZFP1, and distinguished those from monocotyledon species. The <span class="gene">GmZF1</span> <span class="trait">protein</span> was localized at the nucleus, and has specific binding activity with EP1S core sequence, and nucleotide mutation in the core sequence of <italic> EPSPS </italic> promoter changed the binding ability between <span class="gene">GmZF1</span> <span class="trait">protein</span> and core DNA element, implying that two amino acid residues, G and C boxed in core sequence <named-content content-type="gene"> TGACAGTGTCA </named-content> possibly play positive regulation role in recognizing DNA-binding sites in <span class="gene">GmZF1</span> proteins. <span class="trait">High</span> accumulation of <italic> <span class="gene">GmZF1</span> </italic> mRNA induced by exogenous ABA suggested that <italic> <span class="gene">GmZF1</span> </italic> was involved in an ABA-dependent signal transduction pathway. Over-expression of <italic> <span class="gene">GmZF1</span> </italic> significantly improved the contents of proline and soluble sugar and decreased the MDA contents in the transgenic lines exposed to cold stress, indicating that transgenic <italic> Arabidopsis </italic> carrying <italic> <span class="gene">GmZF1</span> </italic> gene have adaptive mechanisms to cold stress. Over-expression of <italic> <span class="gene">GmZF1</span> </italic> also increased the expression of cold-regulated <italic> cor6.6 </italic> gene by probably recognizing protein-DNA binding sites, suggesting that <italic> <span class="gene">GmZF1</span> </italic> from soybean could enhance the tolerance of <italic> Arabidopsis </italic> to cold stress by regulating expression of cold-regulation gene in the transgenic <italic> Arabidopsis </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						To date, few zinc finger proteins from cultivar soybean ( <italic> Glycine max </italic> ) have been isolated and characterized. <italic> SCOF-1, </italic> a typical C2H2-type zinc finger gene, was isolated from soybean and was confirmed to have a positive role in regulating the expression of cold-regulated gene and enhancing cold tolerance in transgenic plants <xref rid="pone.0109399-Kim1" ref-type="bibr"> [21] </xref> . <italic> GsZFP1 </italic> , a zinc finger transcription factor lacking typical QALGGH motif was isolated from wild soybean ( <italic> Glycine soja </italic> L. G07256), and transformed into <italic> Arabidopsis </italic> , and confirmed to play a crucial role in withstanding cold and <span class="trait">drought</span> stresses <xref rid="pone.0109399-Luo1" ref-type="bibr"> [22] </xref> , indicating that the presence of QALGGH motifs in the C2H2 zinc finger <italic> GsZFP1 </italic> gene from wild soybean seems to be not a crucial element in functioning during adaptation to abitotic stress. In our study, a novel C2H2-type zinc finger gene, <italic> <span class="gene">GmZF1</span> </italic> was isolated from cultivar soybean, and identified to have two typical conserved QALGGH motifs, and is significantly different from <italic> GsZFP1 </italic> gene in wild soybean <xref rid="pone.0109399-Luo1" ref-type="bibr"> [22] </xref> . To our knowledge, the understanding on the function of zinc finger C2H2 from cultivar soybean in responses to cold stress is still limited. Therefore, based on the structure analysis and <italic> in </italic> <italic> vitro </italic> identification of <italic> <span class="gene">GmZF1</span> </italic> gene, we transformed the <italic> <span class="gene">GmZF1</span> </italic> gene into <italic> Arabidopsis </italic> plants and identified three homozygous lines which were subsequently used to observe the function effects of <italic> <span class="gene">GmZF1</span> </italic> gene in responses to cold stress and exogenous ABA. Our data showed that <italic> <span class="gene">GmZF1</span> </italic> from cultivar soybean was induced by ABA, and over-expression of <italic> <span class="gene">GmZF1</span> </italic> gene significantly enhanced the tolerance of the transgenic <italic> Arabidopsis </italic> to low temperature stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmZF1</span> </italic> cDNA is composed of 765-bp nucleotides, and eocodes a predicted <span class="trait">protein</span> of 172 amino acids with a calculated molecular mass of 19.2 kDa, and flanked by a 97 bp fragment at 5&#x02032; end and a 149 bp fragment at 3&#x02032; end at two untranslating regions, respectively (accession number DQ055134). Electrophoresis analysis showed that PCR product from the genomic DNA of soybean seedlings has the same size as that from the cDNA length generated by RT-PCR using total RNA as remplate, and sequencing also showed that the generated <italic> <span class="gene">GmZF1</span> </italic> clones from gDNA and cDNA have identical sequence, indicating that <italic> <span class="gene">GmZF1</span> </italic> gene has no intron in the genomic DNA. A homology search against the GenBank database showed that <span class="gene">GmZF1</span> is a homolog of C2H2-type zinc finger proteins (ZFPs), and has two typical C2/H2 type zinc finger domains (CX <sub> 2 </sub> CX <sub> 3 </sub> FX <sub> 3 </sub> QALGGHX <sub> 3 </sub> H) ( <xref ref-type="fig" rid="pone-0109399-g001"> Fig. 1a </xref> ). <span class="gene">GmZF1</span> has 57%, 48%, 47% and 46% homology with the C2H2-type ZFPs from <italic> Medicago truncatula </italic> , Petunia, <italic> Arabidopsis </italic> and rice, respectively, and no highly homology was observed between these genes from different plants, suggesting that <italic> <span class="gene">GmZF1</span> </italic> belongs to a novel subfamily member of C2H2 zinc finger. Like C2H2 zinc finger genes from four kinds of plants mentioned above, the <span class="gene">GmZF1</span> <span class="trait">protein</span> also has plant-specific QALGGH motifs as well as a conserved motif L-box with more Leu residues, and contains a short hydrophobic region with a highly conserved DLN box near the C-terminus of <span class="gene">GmZF1</span> ( <xref ref-type="fig" rid="pone-0109399-g001"> Fig. 1a </xref> ), which may function as a transcription repression domain <xref rid="pone.0109399-Sugano1" ref-type="bibr"> [16] </xref> , <xref rid="pone.0109399-Sakamoto1" ref-type="bibr"> [18] </xref> . It is noteworthy that <span class="gene">GmZF1</span> have two conserved domains with QALGGH motifs, and is significantly different from <italic> GsZFP1 </italic> isolated from wild soybean, the later only has one C2H2 domain lacking QALGGH motif, and the number of amino acid residues between two His (H) residues is composed of four amino acids, which is different from <span class="gene">GmZF1</span> having three amino acids between two His amino acids (as indicated by # and * respectively in <xref ref-type="fig" rid="pone-0109399-g001"> Fig. 1a </xref> ), and no DLN box was found at the C-terminus of GsZFP1 <xref rid="pone.0109399-Luo1" ref-type="bibr"> [22] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of SDS-PAGE indicated that the <span class="gene">GmZF1::GST</span> fusion <span class="trait">protein</span> with 45.2 kD molecular weight was successfully expressed in <italic> E. coli </italic> strain ( <xref ref-type="fig" rid="pone-0109399-g002"> Fig. 2a </xref> ), and the size of expressed <span class="trait">protein</span> is identical with the predicted molecular weight of target <span class="trait">protein</span> in the fusion vector vector pGEX-4 T-1. As a DNA-binding motif, a C2H2-type zinc-finger domain in the <span class="gene">GmZF1</span> has been identified in many transcription factors. In our study, we observed that the transcription factors containing the zinc finger motif could specifically bind to EP1S core sequence ( <named-content content-type="gene"> TGACAGTGTCA </named-content> ), which was originally identified as a <italic> cis </italic> -element within the <italic> EPSPS </italic> gene promoter in petunia <xref rid="pone.0109399-Takatsuji2" ref-type="bibr"> [13] </xref> , <xref rid="pone.0109399-Takatsuji3" ref-type="bibr"> [23] </xref> . For identifying DNA-binding ability of <span class="gene">GmZF1,</span> a gel-shift assay was performed <italic> in </italic> <italic> vitro </italic> using the procedure as described in the method. Data showed that <span class="gene">GmZF1</span> <span class="trait">protein</span> was expressed as fusion proteins with <span class="gene">GST</span> in <italic> E. coli. </italic> To better understand the molecular mechanism of specific-binding between the <span class="gene">GmZF1</span> proteins and EP1S core sequences, a series of synthesis probes including wild type and mutants was prepared as the following descriptions. Briefly, the EP1S core sequence (wild-type, E1) and two probes (E2: bases substitution, E3: with four bases substitution) in the repeated sequences were prepared ( <xref ref-type="fig" rid="pone-0109399-g002"> Fig. 2b </xref> ), and fusion <span class="trait">protein</span> was purified for gel shift assay. Assay showed that the <span class="gene">GmZF1::GST</span> fusion <span class="trait">protein</span> strongly bound to <span class="gene">E1</span> and E2 probes, but weakly binding to E3 probe ( <xref ref-type="fig" rid="pone-0109399-g002"> Fig. 2c </xref> ), no complex was found in reaction solution containing <span class="gene">GST</span> <span class="trait">protein</span> and probes, suggesting that <span class="gene">GmZF1</span> <span class="trait">protein</span> could bind specifically to the EP1S core sequence <italic> in </italic> <italic> vitro </italic> , and two amino acid residues, G and C boxed in core sequence of <named-content content-type="gene"> TGACAGTGTCA </named-content> probably play key role in recognizing DNA-binding sites in <span class="gene">GmZF1</span> proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						a: Induction expression and purification of target proteins, <span class="gene">GmZF1::GST</span> fusion <span class="trait">protein</span> and <span class="gene">GST</span> <span class="trait">protein.</span> Lane 1-uninduced strains containing pGEX4T-1 vector; Lane 2-induced strains containing pGEX4T-1; Lane-3-uninduced strains containing <span class="gene">GmZF1::GST</span> fusion; Lane 4-induced strains containing <span class="gene">GmZF1::GST</span> fusion; Lane 5-purified target <span class="trait">protein</span> as indicated by arrows. b: Probe design and composition. Nucleotide sequences of EP1S and mutated EP1S (E1, E2 and E3) probes. The nucleotide mutations in the EP1S core motif for each probe are boxed. c: The gel shift assay was performed in a solution containing 0.2 &#x000b5;g <span class="gene">GmZF1::GST</span> proteins or <span class="gene">GST</span> proteins and the probes <sup> 32 </sup> P-labeled EP1S (E1) or mutant EP1S (E2 and E3), respectively. The <span class="gene">GmZF1::EP1S</span> complex and free probes are indicated by arrows.
					</p>
				
					<p>&nbsp;&nbsp;
						To identify the subcellular localization of <span class="gene">GmZF1,</span> we fused the full length of <span class="gene">GmZF1</span> to GFP vector under control of the constitutive 35S promoter. Both the recombinant DNA constructs encoding a <span class="gene">GmZF1::GFP</span> fusion <span class="trait">protein</span> and a GFP <span class="trait">protein</span> were respectively introduced into <italic> Arabidopsis </italic> protoplast cells. Localization of the <span class="gene">GmZF1::GFP</span> fusion <span class="trait">protein</span> was visualized exclusively in the nucleus ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3b </xref> ), whereas the control GFP (35S::GFP) was distributed throughout the protoplast cells ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3a </xref> ), demonstrating that <span class="gene">GmZF1</span> is a nuclear <span class="trait">protein</span> possibly functioning as a transcriptional activator.
					</p>
				
					<p>&nbsp;&nbsp;
						a: Images expressing h16318-GFP in protoplast cells; b: images expressing the <span class="gene">GmZF1::GFP</span> fusion <span class="trait">protein,</span> fluorescent-field illumination was used to examine GFP signal (left); followed by bright-field illumination (middle) and confocal microscopy (right) for an merged image of bright and fluorescent illumination. c: the relative accumulation of <italic> <span class="gene">GmZF1</span> </italic> mRNA in soybean seedlings of 10-day-old exposed to 4&#x000b0;C at different time points; d: the accumulation of <span class="gene">GmZF1</span> mRNA in soybean seedlings of 10-day-old exposed to ABA of 200 &#x000b5;M at different time points.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine the transcription profile of <italic> <span class="gene">GmZF1</span> </italic> in soybean under cold stress, the expression of <italic> <span class="gene">GmZF1</span> </italic> in soybean was detected using semi-quantitative RT-PCR. The result showed that <italic> <span class="gene">GmZF1</span> </italic> was weakly expressed in young seedlings, and induced by low temperature and reached a maximum at 5 h after 4&#x000b0;C stress ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3c </xref> ). Data also showed that <italic> <span class="gene">GmZF1</span> </italic> gene could be induced by exogenous ABA, and seems to have similar expression pattern under cold stress, and maintained at a <span class="trait">high</span> expression level when the seedlings were kept at ABA treatment ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3d </xref> ), suggesting that <italic> <span class="gene">GmZF1</span> </italic> might be involved in plant responses to cold stress via an ABA-dependent pathway.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, as a novel C2H2 zinc finger <span class="trait">protein</span> gene, <italic> <span class="gene">GmZF1</span> </italic> from soybean was characterized. Sequence analysis revealed that the <span class="gene">GmZF1</span> had <span class="trait">high</span> identity with other C2H2-type ZFPs, and shared two zinc finger motifs containing a conserved plant-specific QALGGH amino acid sequence which was proved to be critical for DNA-binding activity <xref rid="pone.0109399-Kubo1" ref-type="bibr"> [11] </xref> . Based on the present data, we predicted that the binding activity of <span class="gene">GmZF1</span> to EP1S core elements was probably mediated by the QALGGH sequence. Additionally, the C-terminus of <italic> <span class="gene">GmZF1</span> </italic> gene contains typical Leu-rich L-box and DLN-box which play roles in <span class="trait">protein</span> interactions or in maintaining the folded structure <xref rid="pone.0109399-Sakamoto2" ref-type="bibr"> [19] </xref> . The DLN-box was thought to function in transcriptional repression. As reported previously, the DLN-boxes in the zinc finger proteins ZPT2&#x02013;3 from petunia <xref rid="pone.0109399-Sugano1" ref-type="bibr"> [16] </xref> and STZ/ZAT10 from <italic> Arabidopsis </italic> exhibited repression roles in transcription activities <xref rid="pone.0109399-Sakamoto1" ref-type="bibr"> [18] </xref> . However, some zinc finger proteins containing the DLN-box were involved in transcriptional activation, such as ThZF1 from <italic> Thellungiella halophila </italic> <xref rid="pone.0109399-Xu2" ref-type="bibr"> [25] </xref> and CaZF from chickpea <xref rid="pone.0109399-Jain1" ref-type="bibr"> [26] </xref> . Subcellular localization analysis revealed that <span class="gene">GmZF1</span> localized at nuclei ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3 </xref> ), implying that <span class="gene">GmZF1,</span> like other ZFPs from plant TFIIIA-type, functions as a transcription factor in plant cells and may play an important role in signaling pathway in soybean under abiotic stress.
					</p>
				
					<p>&nbsp;&nbsp;
						As a key regulator, ABA plays an important role in signaling pathways under stresses, such as <span class="trait">drought,</span> low temperature and osmotic stress, and induces the expressions of a number of genes that respond to abiotic stress <xref rid="pone.0109399-Xiong1" ref-type="bibr"> [30] </xref> , <xref rid="pone.0109399-Finkelstein1" ref-type="bibr"> [31] </xref> . For example, the expression of <italic> AZF2 </italic> gene from <italic> Arabidopsis </italic> was strongly induced following ABA treatment under <span class="trait">drought</span> and salt stress <xref rid="pone.0109399-Kodaira1" ref-type="bibr"> [1] </xref> , <xref rid="pone.0109399-Chen1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0109399-Sakamoto1" ref-type="bibr"> [18] </xref> . However, it was reported that some C2H2-type zinc finger genes are induced by dehydration and cold stress, but do not respond to exogenous ABA <xref rid="pone.0109399-YamaguchiShinozaki1" ref-type="bibr"> [2] </xref> , <xref rid="pone.0109399-Davletova1" ref-type="bibr"> [32] </xref> , <xref rid="pone.0109399-Zhu1" ref-type="bibr"> [33] </xref> , suggesting that two kinds of signaling transduction pathways, ABA-independent pathway and ABA-dependent pathway play different regulation roles in responding to abiotic stresses, repectively, and that the initial stress signal was converted into cellular responses <xref rid="pone.0109399-Shinozaki1" ref-type="bibr"> [34] </xref> . In this study, our data proved that <italic> <span class="gene">GmZF1</span> </italic> was involved in plant responses to cold stress through an ABA-dependent signal transduction pathway since the expression of <italic> <span class="gene">GmZF1</span> </italic> in soybean seedlings was clearly induced by ABA ( <xref ref-type="fig" rid="pone-0109399-g004"> Fig. 4 </xref> ). In addition, accumulation pattern of <italic> <span class="gene">GmZF1</span> </italic> gene in the transgenic <italic> Arabidopsis </italic> lines treated by exogenous ABA also showed that over-expression of <italic> <span class="gene">GmZF1</span> </italic> gene in <italic> Arabidopsis </italic> was strongly induced by ABA.
					</p>
				
					<p>&nbsp;&nbsp;
						Some of the C2H2-type ZFPs from different plant species are confirmed to play regulatory roles in stress responses, such as AZF2 and STZ in Arabidopsis <xref rid="pone.0109399-Chen1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0109399-Lippuner1" ref-type="bibr"> [17] </xref> , <xref rid="pone.0109399-Nakai1" ref-type="bibr"> [27] </xref> , <span class="gene">SCOF-1</span> in soybean <xref rid="pone.0109399-Kim1" ref-type="bibr"> [21] </xref> , GsZFP1 in wild soybean <xref rid="pone.0109399-Luo1" ref-type="bibr"> [22] </xref> , StZFP1 in potato <xref rid="pone.0109399-Tian1" ref-type="bibr"> [9] </xref> , DST in rice <xref rid="pone.0109399-Huang1" ref-type="bibr"> [28] </xref> and TaCHP in wheat <xref rid="pone.0109399-Li1" ref-type="bibr"> [29] </xref> . Expression analysis revealed that <span class="gene">GmZF1</span> was clearly induced by cold stress ( <xref ref-type="fig" rid="pone-0109399-g004"> Fig. 4 </xref> ), suggesting that <italic> <span class="gene">GmZF1</span> </italic> might be involved in plant responses to cold stress in soybean. Transgenic <italic> Arabidopsis </italic> plants over-expressing <italic> <span class="gene">GmZF1</span> </italic> were evaluated for the involvement of <italic> <span class="gene">GmZF1</span> </italic> gene in cold tolerance of plants. Although no morphological differences between the transgenic lines and the wild type was observed, the remarkable differences in accumulation of free proline, soluble sugar and MDA were confirmed after cold and ABA treatment ( <xref ref-type="fig" rid="pone-0109399-g005"> Fig. 5c&#x02013;e </xref> and <xref ref-type="fig" rid="pone-0109399-g006"> Fig. 6c&#x02013;e </xref> ). Accumulation of proline by stress-induced has been observed in many plant species, and functions as an osmo-protectant coping with stress <xref rid="pone.0109399-Davletova1" ref-type="bibr"> [32] </xref> , <xref rid="pone.0109399-Igarashi1" ref-type="bibr"> [35] </xref> , <xref rid="pone.0109399-Dobr1" ref-type="bibr"> [36] </xref> . For example, the transgenic tobacco plants over-expressing <italic> <span class="gene">GmDREB3</span> </italic> could accumulate much free proline than the wild-type plants after <span class="trait">drought</span> stress treatment for 16 d <xref rid="pone.0109399-Chen2" ref-type="bibr"> [37] </xref> . In plants, soluble sugar has been shown to fulfill a dual role as both metabolites and as signaling molecules <xref rid="pone.0109399-BolouriMoghaddam1" ref-type="bibr"> [38] </xref> , <xref rid="pone.0109399-Smeekens1" ref-type="bibr"> [39] </xref> that may play important roles in the mechanisms of plant responding to the stress <xref rid="pone.0109399-Sperdouli1" ref-type="bibr"> [40] </xref> , <xref rid="pone.0109399-Ramel1" ref-type="bibr"> [41] </xref> . In our study, the accumulations of proline and soluble sugar in the wild type and transgenic lines were improved, and increase times in the contents of proline and soluble sugar in the transgenic lines were higher than that in wild type during cold and ABA treatment ( <xref ref-type="fig" rid="pone-0109399-g005"> Fig. 5 </xref> and <xref ref-type="fig" rid="pone-0109399-g006"> Fig. 6 </xref> ), proving that the accumulation of soluble sugars has been occurred in many plant species during cold acclimation <xref rid="pone.0109399-Guy1" ref-type="bibr"> [42] </xref> . Based on our investigations, both wild type and transgenic <italic> Arabidopsis </italic> lines basically have no significant differences in germination rates. However, the survival rates from these two phenotypes was remarkable when these two seedlings were exposed to cold stress, the survival rates of transgenic <italic> Arabidopsis </italic> lines cultured at 4&#x000b0;C significantly increased after one week comparing to the wild types ( <xref ref-type="fig" rid="pone-0109399-g004"> Fig. 4 a,b </xref> ), indicating that <italic> <span class="gene">GmZF1</span> </italic> gene could play regulation role during the plant growth and development. Our study showed that over-expression of <italic> <span class="gene">GmZF1</span> </italic> gene significantly enhanced the transcription levels of <italic> cor6.6 </italic> gene responding to cold stress and ABA supply, and increased the accumulation of proline, soluble sugar in the transgenic <italic> Arabidopsis </italic> , and reduced the content of MDA in the transgenic lines comparing to the wild type, and significantly improved the tolerance of the transgenic lines exposed to cold stress and exogenous ABA ( <xref ref-type="fig" rid="pone-0109399-g005"> Fig. 5e </xref> ; <xref ref-type="fig" rid="pone-0109399-g006"> Fig. 6e </xref> ). In conclusion, over-expression of <italic> <span class="gene">GmZF1</span> </italic> resulted in an enhancement in cold tolerance of the transgenic <italic> Arabidopsis </italic> by activating transcription of <italic> cor6.6 </italic> and/or cold resistance-genes ( <xref ref-type="fig" rid="pone-0109399-g007"> Fig. 7 </xref> ) as well as accumulation changes of proline, soluble sugar and MDA <xref rid="pone.0109399-Giannakoula1" ref-type="bibr"> [43] </xref> &#x02013; <xref rid="pone.0109399-Xu3" ref-type="bibr"> [45] </xref> . Bioinformatics analysis fund that the promoter with 1955 bp of <italic> cor6.6 </italic> gene contains 7&#x000d7; <underline> G </underline> A <underline> C </underline> A and 6&#x000d7; <underline> G </underline> T <underline> C </underline> A repeat in core element <named-content content-type="gene"> TTGACAGTGTCAC </named-content> , respectively (NCBI Reference Sequence in <italic> Arbidopsis thaliana </italic> chromosome 5: NC_003076.8-AT5G15950/NM_121600.3/NP_197099.1), and these two core elements possibly play key role in recognizing DNA-binding sites in target proteins, because the mutation of G and C in the two boxed core elements could change the ability of <span class="gene">GmZf1</span> <span class="trait">protein</span> binding to DNA, especially the mutation of G and C in the GCTA box ( <xref ref-type="fig" rid="pone-0109399-g003"> Fig. 3c </xref> ). Therefore, <italic> <span class="gene">GmZF1</span> </italic> could play regulation role by activating the transcription of <italic> cor6.6 </italic> cold-regulated gene in <italic> Arabidopsis, </italic> and led to an enhancement in the tolerance of transgenic plants to cold stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The zinc finger gene <italic> <span class="gene">GmZF1</span> </italic> was cloned from soybean based on the expressed sequence tag (EST) database of soybean. Total RNA was extracted from soybean seedlings using RNA Prep. Pure plant kit (Tiangen Biological Company, Beijing), and cDNA synthesis was performed by a reverse transcription kit (TaKaRa Dalian BioCompany). The entire <italic> <span class="gene">GmZF1</span> </italic> cDNA of was obtained by PCR using the specific primers (F: <named-content content-type="gene"> 5&#x02032;-AGAGGAAACTAGCTAGGGCACTTC-3&#x02032; </named-content> and R: <named-content content-type="gene"> 5&#x02032;-CCCGAGAACTAAGAAGTTTCGTATT-3&#x02032; </named-content> ). The deduced amino acid sequences of soybean <span class="gene">GmZF1</span> zinc finger <span class="trait">protein</span> were matched by a <span class="trait">protein</span> blast procedure ( <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/BLAST/"> http://www.ncbi.nlm.nih.gov/BLAST/ </ext-link> ). BioEdit version 7.0 software was used to multiple sequence alignments. Phylogenetic analysis was carried out by MEGA version 4.0 with adopting position correction distance using a bootstrap replicate number of 1000.
					</p>
				
					<p>&nbsp;&nbsp;
						An entire 516-bp <italic> <span class="gene">GmZF1</span> </italic> fragment containing the DNA binding domain was amplified by PCR using a pair of primers F and R (F: <named-content content-type="gene"> 5&#x02032;-ACAACTCGAGATGAAGAGAGGCAGAGAA-3&#x02032; </named-content> and R: <named-content content-type="gene"> 5&#x02032;-AGACGAATTCAATGAAACAATTGAGCAC-3&#x02032; </named-content> ), and was subcloned onto the pGEX4 T-1 vector by inserting at two specific sites, <italic> EcoR </italic> I and <italic> Xho </italic> I (Amersham Biosciences), and the recombinant pGEX-4 T-1 plasmids were identified by sequencing, and transferred into <italic> Escherichia coli </italic> BL21 cells (Amersham Biosciences). The induction of target fusion proteins was performed by adding 0.8 mM IPTG (Isopropyl-&#x003b2;-d-thiogalactopyranoside) into the culture of <italic> E. coli </italic> strains, and the cultures carrying <span class="gene">GmZF1::GST</span> fusion proteins was incubated for 5 hours at 37&#x000b0;C with 250 rpm The bacterial cells were shattered by ultrasonication in phosphate-buffered saline (PBS) and centrifuged at 11000 <italic> g </italic> for 10 min to remove insoluble cell debris, and then the supernates were collected, and applied on the gel containing 12% ployacrylamide for electrophoresis detection of fusion <span class="trait">protein</span> by the method of SDS-PAGE described in the book <xref rid="pone.0109399-Davis1" ref-type="bibr"> [46] </xref> . The GST::GmZF1 fusion <span class="trait">protein</span> was purified using a glutathione-Sepharose 4B column (Amersham Biosciences) according to the manufacturer&#x02019;s instructions. The 52 bp DNA fragment containing four copies of wild-type or mutant EP1S core sequences were labeled by 25 &#x000b5;Ci&#x000b7;&#x000b5;L <sup> &#x02212;1 </sup> of &#x003b3;- <sup> 32 </sup> P-dATP (Amersham Biosciences). The DNA-binding reaction was performed in a 20 &#x000b5;L binding buffer containing <sup> 32 </sup> P-labelled probe, glycerine and purified fusion <span class="trait">protein</span> <xref rid="pone.0109399-Lu1" ref-type="bibr"> [24] </xref> , and were subjected to electrophoresis with the gel containing 0.53 Tris-borate-EDTA, 6% polyacrylamide, and the gels were dried and visualized by autoradiography.
					</p>
				
					<p>&nbsp;&nbsp;
						In brief, an entire <italic> <span class="gene">GmZF1</span> </italic> cDNA fragment was amplified by PCR procedure with a pair of specific primers (F: <named-content content-type="gene"> 5&#x02032;-AACACTGCAGATGAAGAGAGGCAGAGA-3&#x02032; </named-content> ), and R: 5&#x02032;-CGGGATCCAATGAAACAATTGAGCAC-3&#x02032;.). Based on the specific sites in the MCS of GFP vector, two specific sites, a <italic> Pst </italic> I and <italic> BamH </italic> I were respectively inserted into the 5&#x02032; ends of the primers for subcloning requirement. The digested PCR fragment was inserted into the multiple cloning sites (MCS) in the GFP vector carrying a GFP <span class="trait">protein</span> driven by a 35 S promoter, and this construct carrying 35 S::GFP or 35 S::GmZF1::GFP were introduced into <italic> Arabidopsis </italic> protoplast cells, which was prepared by young seedlings of <italic> Arabidopsis, </italic> and the transformation procedures were performed by the method as described previously <xref rid="pone.0109399-Han1" ref-type="bibr"> [47] </xref> , <xref rid="pone.0109399-Liu2" ref-type="bibr"> [48] </xref> . The images were captured using a Nikon Eclipse TE2000-U microscope (Nikon).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A comparison study of Agrobacterium-mediated transformation methods for root-specific promoter analysis in soybean.</h2>
			<h3>PMID:25097075  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li C(1), Zhang H, Wang X, Liao H.
				<br/> 
				年份：2014 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Both in vitro and in vivo hairy root transformation systems could not replace whole plant transformation for promoter analysis of root-specific and low-P induced genes in soybean. An efficient genetic transformation system is crucial for promoter analysis in plants. Agrobacterium-mediated transformation is the most popular method to produce transgenic hairy roots or plants. In the present study, first, we compared the two different Agrobacterium rhizogenes-mediated hairy root transformation methods using either constitutive CaMV35S or the promoters of root-preferential genes, <span class="gene">GmEXPB2</span> and <span class="gene">GmPAP21,</span> in soybean, and found the efficiency of in vitro hairy root transformation was significantly higher than that of in vivo transformation. We compared Agrobacterium rhizogenes-mediated hairy root and Agrobacterium tumefaciens-mediated whole plant transformation systems. The results showed that low-phosphorous (P) inducible <span class="gene">GmEXPB2</span> and <span class="gene">GmPAP21</span> promoters could not induce the increased expression of the GUS reporter gene under low P stress in both in vivo and in vitro transgenic hairy roots. Conversely, GUS activity of <span class="gene">GmPAP21</span> promoter was significantly higher at low P than <span class="trait">high</span> P in whole plant transformation. Therefore, both in vitro and in vivo hairy root transformation systems could not replace whole plant transformation for promoter analysis of root-specific and low-P induced genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-allergen detection in food by micro high-performance liquid chromatography coupled to a dual cell linear ion trap mass spectrometry.</h2>
			<h3>PMID:25042441  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Monaci L(1), Pilolli R(2), De Angelis E(2), Godula M(3), Visconti A(2).
				<br/> 
				年份：2014 Sep 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						There is a raising demand for sensitive and <span class="trait">high</span> throughput <span class="gene">MS</span> based methods for screening purposes especially tailored to the detection of allergen contaminants in different food commodities. A challenging issue is represented by complex food matrices where the antibody-based kits commercially available might encounter objective limitations consequently to epitope masking phenomena due to a multitude of interfering compounds arising from the matrix. The performance of a method duly optimized for the extraction and simultaneous detection of soy, egg and milk allergens in a cookie food matrix by microHPLC-ESI-MS/MS, is herein reported. Thanks to the innovative configuration and the versatility shown by the dual cell linear ion trap <span class="gene">MS</span> used, the most intense and reliable peptide markers were first identified by untargeted survey experiment, and subsequently employed to design an ad hoc multi-target SRM method, based on the most intense transitions recorded for each selected precursor peptide. A sample extraction and purification protocol was optimized also including an additional step based on sonication, which resulted in a considerable improvement in the detection of milk allergen peptides. Data Dependent™ Acquisition scheme allowed to fill out a tentative list of potential peptide markers, which were further filtered upon fulfilling specific requirements. A total of eleven peptides were monitored simultaneously for confirmation purposes of each allergenic contaminant and the two most sensitive peptide markers/protein were selected in order to retrieve quantitative information. Relevant LODs were found to range from 0.1μg/g for milk to 0.3μg/g for egg and 2μg/g for soy.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean (Glycine max) nodulation-suppressive CLE peptide, <span class="gene">GmRIC1,</span> functions interspecifically in common white bean (Phaseolus vulgaris), but not in a supernodulating line mutated in the receptor PvNARK.</h2>
			<h3>PMID:25040127  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ferguson BJ(1), Li D, Hastwell AH, Reid DE, Li Y, Jackson SA, Gresshoff PM.
				<br/> 
				年份：2014 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume plants regulate the number of nitrogen-fixing root nodules they form via a process called the Autoregulation of Nodulation (AON). Despite being one of the most economically important and abundantly consumed legumes, little is known about the AON pathway of common bean (Phaseolus vulgaris). We used comparative- and functional-genomic approaches to identify central components in the AON pathway of common bean. This includes identifying PvNARK, which encodes a LRR receptor kinase that acts to regulate root nodule numbers. A novel, truncated version of the gene was identified directly upstream of PvNARK, similar to Medicago truncatula, but not seen in Lotus japonicus or soybean. Two mutant alleles of PvNARK were identified that cause a classic shoot-controlled and nitrate-tolerant supernodulation phenotype. Homeologous over-expression of the nodulation-suppressive CLE peptide-encoding soybean gene, <span class="gene">GmRIC1,</span> abolished nodulation in wild-type bean, but had no discernible effect on PvNARK-mutant plants. This demonstrates that soybean <span class="gene">GmRIC1</span> can function interspecifically in bean, acting in a PvNARK-dependent manner. Identification of bean PvRIC1, PvRIC2 and PvNIC1, orthologues of the soybean nodulation-suppressive CLE peptides, revealed a <span class="trait">high</span> degree of conservation, particularly in the CLE domain. Overall, our work identified four new components of bean nodulation control and a truncated copy of PvNARK, discovered the mutation responsible for two supernodulating bean mutants and demonstrated that soybean <span class="gene">GmRIC1</span> can function in the AON pathway of bean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pharmacological evaluation and docking studies of α,β-unsaturated carbonyl based synthetic compounds as inhibitors of secretory phospholipase A₂, cyclooxygenases, lipoxygenase and proinflammatory cytokines.</h2>
			<h3>PMID:24938495  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bukhari SN(1), Lauro G(2), Jantan I(3), Bifulco G(2), Amjad MW(4).
				<br/> 
				年份：2014 Aug 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Arachidonic acid and its metabolites have generated <span class="trait">high</span> level of interest among researchers due to their vital role in inflammation. The inhibition of enzymes involved in arachidonic acid metabolism has been considered as synergistic anti-inflammatory effect. A series of novel α,β-unsaturated carbonyl based compounds were synthesized and evaluated for their inhibitory activity on secretory phospholipase A₂ (sPLA₂), cyclooxygenases (COX), soybean lipoxygenase (LOX) in addition to proinflammatory cytokines comprising IL-6 and TNF-α. Six α,β-unsaturated carbonyl based compounds (2, 3, 4, 12, 13 and 14) exhibited strong inhibition of sPLA₂ activity, with IC₅₀ values in the range of 2.19-8.76 μM. Nine compounds 1-4 and 10-14 displayed inhibition of COX-1 with IC₅₀ values ranging from 0.37 to 1.77 μM (lower than that of reference compound), whereas compounds 2, 10, 13 and 14 strongly inhibited the COX-2. The compounds 10-14 exhibited strong inhibitory activity against <span class="gene">LOX</span> enzyme. All compounds were evaluated for the inhibitory activities against LPS-induced TNF-α and IL-6 release in the macrophages. On the basis of screening results, five active compounds 3, 4, 12, 13 and 14 were found strong inhibitors of TNF-α and IL-6 release in a dose-dependent manner. Molecular docking experiments were performed to clarify the molecular aspects of the observed COX and <span class="gene">LOX</span> inhibitory activities of the investigated compounds. Present findings increases the possibility that these α,β-unsaturated carbonyl based compounds might serve as beneficial starting point for the design and development of improved anti-inflammatory agents.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture and evolution of the mating type locus in fusaria that cause soybean sudden death syndrome and bean root rot.</h2>
			<h3>PMID:24891421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hughes TJ(1), O'Donnell K(2), Sink S(3), Rooney AP(4), Scandiani MM(5), Luque A(5), Bhattacharyya MK(6), Huang X(7).
				<br/> 
				年份：2014 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium tucumaniae is the only known sexually reproducing species among the seven closely related fusaria that cause soybean sudden death syndrome (SDS) or bean root rot (BRR). In a previous study, laboratory mating of F. tucumaniae yielded recombinant ascospore progeny but required two mating-compatible strains, indicating that it is heterothallic. To assess the reproductive mode of the other SDS and BRR fusaria, and their potential for mating, whole-genome sequences of two SDS and one BRR pathogen were analyzed to characterize their mating type (MAT) loci. This bioinformatic approach identified a <span class="gene">MAT1-1</span> idiomorph in F. virguliforme NRRL 22292 and MAT1-2 idiomorphs in F. tucumaniae NRRL 34546 and F. azukicola NRRL 54364. Alignments of the MAT loci were used to design PCR primers within the conserved regions of the flanking genes <span class="gene">APN1</span> and SLA2, which enabled primer walking to obtain nearly complete sequences of the MAT region for six <span class="gene">MAT1-1</span> and five MAT1-2 SDS/BRR fusaria. As expected, sequences of the highly divergent 4.7 kb <span class="gene">MAT1-1</span> and 3.7 kb MAT1-2 idiomorphs were unalignable. However, sequences of the respective idiomorphs and those that flank <span class="gene">MAT1-1</span> and MAT1-2 were highly conserved. In addition to three genes at <span class="gene">MAT1-1</span> (MAT1-1-1, MAT1-1-2, MAT1-1-3) and two at MAT1-2 (MAT1-2-1, MAT1-2-3), the MAT loci of the SDS/BRR fusaria also include a putative gene predicted to encode for a 252 amino acid <span class="trait">protein</span> of unknown function. Alignments of the MAT1-1-3 and MAT1-2-1 sequences were used to design a multiplex PCR assay for the MAT loci. This assay was used to screen DNA from 439 SDS/BRR isolates, which revealed that each isolate possessed <span class="gene">MAT1-1</span> or MAT1-2, consistent with heterothallism. Both idiomorphs were represented among isolates of F. azukicola, F. brasiliense, F. phaseoli and F. tucumaniae, whereas isolates of F. virguliforme and F. cuneirostrum were only <span class="gene">MAT1-1</span> and F. crassistipitatum were only MAT1-2. Finally, nucleotide sequence data from the <span class="gene">RPB1</span> and <span class="gene">RPB2</span> genes were used to date the origin of the SDS/BRR group, which was estimated to have occurred about 0.75 Mya (95% HPD interval: 0.27, 1.68) in the mid-Pleistocene, long before the domestication of the common bean or soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improved soybean oil quality by targeted mutagenesis of the fatty acid desaturase 2 gene family.</h2>
			<h3>PMID:24851712  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Haun W(1), Coffman A, Clasen BM, Demorest ZL, Lowy A, Ray E, Retterath A, Stoddard T, Juillerat A, Cedrone F, Mathis L, Voytas DF, Zhang F.
				<br/> 
				年份：2014 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean oil is <span class="trait">high</span> in polyunsaturated fats and is often partially hydrogenated to increase its shelf life and improve oxidative stability. The trans-fatty acids produced through hydrogenation pose a health threat. Soybean lines that are low in polyunsaturated fats were generated by introducing mutations in two fatty acid desaturase 2 genes (FAD2-1A and FAD2-1B), which in the seed convert the monounsaturated fat, oleic acid, to the polyunsaturated fat, linoleic acid. Transcription activator-like effector nucleases (TALENs) were engineered to recognize and cleave conserved DNA sequences in both genes. In four of 19 transgenic soybean lines expressing the TALENs, mutations in <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> were observed in DNA extracted from leaf tissue; three of the four lines transmitted heritable <span class="gene">FAD2-1</span> mutations to the next generation. The fatty acid profile of the seed was dramatically changed in plants homozygous for mutations in both <span class="gene">FAD2-1A</span> and FAD2-1B: oleic acid increased from 20% to 80% and linoleic acid decreased from 50% to under 4%. Further, mutant plants were identified that lacked the TALEN transgene and only carried the targeted mutations. The ability to create a valuable trait in a single generation through targeted modification of a gene family demonstrates the power of TALENs for genome engineering and crop improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mutations in <span class="gene">SACPD-C</span> result in a range of elevated stearic acid concentration in soybean seed.</h2>
			<h3>PMID:24846334  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carrero-Colón M(1), Abshire N(2), Sweeney D(2), Gaskin E(2), Hudson K(1).
				<br/> 
				年份：2014 May 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean oil has a wide variety of uses, and stearic acid, which is a relatively minor component of soybean oil is increasingly desired for both industrial and food applications. New soybean mutants containing <span class="trait">high</span> levels of the saturated fatty acid stearate in seeds were recently identified from a chemically mutagenized population. Six mutants ranged in stearate content from 6-14% stearic acid, which is 1.5 to 3 times the levels contained in wild-type seed of the Williams 82 cultivar. Candidate gene sequencing revealed that all of these lines carried amino acid substitutions in the gene encoding the delta-9-stearoyl-acyl-carrier <span class="trait">protein</span> desaturase enzyme (SACPD-C) required for the conversion of stearic acid to oleic acid. Five of these missense mutations were in highly conserved residues clustered around the predicted di-iron center of the <span class="gene">SACPD-C</span> enzyme. Co-segregation analysis demonstrated a positive association of the elevated stearate trait with the <span class="gene">SACPD-C</span> mutation for three populations. These missense mutations may provide additional alleles that may be used in the development of new soybean cultivars with increased levels of stearic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of transcript polymorphisms in soybean lines varying in oil composition and content.</h2>
			<h3>PMID:24755115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Goettel W, Xia E, Upchurch R, Wang ML, Chen P, An YQ(1).
				<br/> 
				年份：2014 Apr 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Variation in seed oil composition and content among soybean varieties is largely attributed to differences in transcript sequences and/or transcript accumulation of oil production related genes in seeds. Discovery and analysis of sequence and expression variations in these genes will accelerate soybean oil quality improvement. RESULTS: In an effort to identify these variations, we sequenced the transcriptomes of soybean seeds from nine lines varying in oil composition and/or total oil content. Our results showed that 69,338 distinct transcripts from 32,885 annotated genes were expressed in seeds. A total of 8,037 transcript expression polymorphisms and 50,485 transcript sequence polymorphisms (48,792 SNPs and 1,693 small Indels) were identified among the lines. Effects of the transcript polymorphisms on their encoded <span class="trait">protein</span> sequences and functions were predicted. The studies also provided independent evidence that the lack of <span class="gene">FAD2-1A</span> gene activity and a non-synonymous SNP in the coding sequence of FAB2C caused elevated oleic acid and stearic acid levels in soybean lines M23 and FAM94-41, respectively. CONCLUSIONS: As a proof-of-concept, we developed an integrated RNA-seq and bioinformatics approach to identify and functionally annotate transcript polymorphisms, and demonstrated its <span class="trait">high</span> effectiveness for discovery of genetic and transcript variations that result in altered oil quality traits. The collection of transcript polymorphisms coupled with their predicted functional effects will be a valuable asset for further discovery of genes, gene variants, and functional markers to improve soybean oil quality.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A classification scheme for alternative oxidases reveals the taxonomic distribution and evolutionary history of the enzyme in angiosperms.</h2>
			<h3>PMID:24751423  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Costa JH(1), McDonald AE(2), Arnholdt-Schmitt B(3), Fernandes de Melo D(4).
				<br/> 
				年份：2014 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A classification scheme based on <span class="trait">protein</span> phylogenies and sequence harmony method was used to clarify the taxonomic distribution and evolutionary history of the alternative oxidase (AOX) in angiosperms. A large data set analyses showed that <span class="gene">AOX1</span> and <span class="gene">AOX2</span> subfamilies were distributed into 4 phylogenetic clades: AOX1a-c/1e, AOX1d, AOX2a-c and AOX2d. <span class="trait">High</span> diversity in AOX family compositions was found. While the <span class="gene">AOX2</span> subfamily was not detected in monocots, the <span class="gene">AOX1</span> subfamily has expanded (AOX1a-e) in the large majority of these plants. In addition, Poales AOX1b and 1d were orthologous to eudicots AOX1d and then renamed as AOX1d1 and 1d2. <span class="gene">AOX1</span> or <span class="gene">AOX2</span> losses were detected in some eudicot plants. Several <span class="gene">AOX2</span> duplications (AOX2a-c) were identified in eudicot species, mainly in the asterids. The AOX2b originally identified in eudicots in the Fabales order (soybean, cowpea) was divergent from AOX2a-c showing some specific amino acids with AOX1d and then it was renamed as AOX2d. AOX1d and AOX2d seem to be stress-responsive, facultative and mutually exclusive among species suggesting a complementary role with an AOX1(a) in stress conditions. Based on the data collected, we present a model for the evolutionary history of AOX in angiosperms and highlight specific areas where further research would be most beneficial.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a pathogenesis-related <span class="trait">protein</span> 10 gene <span class="gene">(GmPR10)</span> with induced expression in soybean (Glycine max) during infection with Phytophthora sojae.</h2>
			<h3>PMID:24737571  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu P(1), Jiang L, Wu J, Li W, Fan S, Zhang S.
				<br/> 
				年份：2014 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In previous study, a cDNA library enriched for mRNAs encoding ESTs that increased in abundance during infection with Phytophthora sojae was constructed by suppression subtractive hybridization from leaf tissues of a <span class="trait">high</span> resistant soybean, and an EST homologous to the class 10 of pathogenesis-related (PR) proteins was identified to be up-regulated by microarray and real-time PCR. Here, the full-length cDNA (termed <span class="gene">GmPR10,</span> GenBank accession number FJ960440; ADC31789.1) of the EST was isolated by rapid amplification of cDNA ends, and contains an open reading frame of 474 bp. The <span class="gene">GmPR10</span> <span class="trait">protein</span> included a "P-loop'' motif. The constitutive transcript abundance of <span class="gene">GmPR10</span> in soybean was the highest in leaves, followed by roots and stems. Further analysis showed that <span class="gene">GmPR10</span> mRNA abundance was increased during infection with P. sojae following leaf treatments with gibberellin (GA3), hydrogen peroxide (H2O2), salicylic acid (SA), and abscisic acid (ABA). The dialytically renatured <span class="gene">GmPR10</span> <span class="trait">protein</span> significantly inhibited P. sojae hyphal growth and exhibited RNase activity. Transgenic tobacco and soybean plants overexpressing <span class="gene">GmPR10</span> showed increased <span class="trait">resistance</span> to P. nicotianae Breda and P. sojae, respectively. These results suggest that the <span class="gene">GmPR10</span> <span class="trait">protein</span> plays an important role in host defense against P. sojae infection. To the best of our knowledge, this is the first report on the functional characterization of a PR10 <span class="trait">protein</span> from soybean in defense against P. sojae.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distinct Copy Number, Coding Sequence, and Locus Methylation Patterns Underlie Rhg1-Mediated Soybean <span class="trait">Resistance</span> to Soybean Cyst Nematode.</h2>
			<h3>PMID:24733883</h3>
				

				
			<p>
				作者：Cook DE(1), Bayless AM(1), Wang K(1), Guo X(1), Song Q(1), Jiang J(1), Bent AF(2).
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Copy number variation of kilobase-scale genomic DNA segments, beyond presence/absence polymorphisms, can be an important driver of adaptive traits. <span class="trait">Resistance</span> to Heterodera glycines (Rhg1) is a widely utilized quantitative trait locus that makes the strongest known contribution to <span class="trait">resistance</span> against soybean cyst nematode (SCN), Heterodera glycines, the most damaging pathogen of soybean (Glycine max). <span class="gene">Rhg1</span> was recently discovered to be a complex locus at which resistance-conferring haplotypes carry up to 10 tandem repeat copies of a 31-kb DNA segment, and three disparate genes present on each repeat contribute to SCN <span class="trait">resistance.</span> Here, we use whole-genome sequencing, fiber-FISH (fluorescence in situ hybridization), and other methods to discover the genetic variation at <span class="gene">Rhg1</span> across 41 diverse soybean accessions. Based on copy number variation, transcript abundance, nucleic acid polymorphisms, and differentially methylated DNA regions, we find that SCN <span class="trait">resistance</span> is associated with multicopy <span class="gene">Rhg1</span> haplotypes that form two distinct groups. The tested high-copy-number <span class="gene">Rhg1</span> accessions, including plant introduction (PI) 88788, contain a flexible number of copies (seven to 10) of the 31-kb <span class="gene">Rhg1</span> repeat. The identified low-copy-number <span class="gene">Rhg1</span> group, including PI 548402 (Peking) and PI 437654, contains three copies of the <span class="gene">Rhg1</span> repeat and a newly identified allele of <span class="gene">Glyma18g02590</span> (a predicted α-SNAP [α-soluble N-ethylmaleimide-sensitive factor attachment <span class="trait">protein]).</span> There is strong evidence for a shared origin of the two resistance-conferring multicopy <span class="gene">Rhg1</span> groups and subsequent independent evolution. Differentially methylated DNA regions also were identified within <span class="gene">Rhg1</span> that correlate with SCN <span class="trait">resistance.</span> These data provide insights into copy number variation of multigene segments, using as the example a disease <span class="trait">resistance</span> trait of <span class="trait">high</span> economic importance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of active VQ motif-containing genes and the expression patterns under low nitrogen treatment in soybean.</h2>
			<h3>PMID:24727126  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Zhang H(1), Sun G(2), Jin Y(1), Qiu L(3).
				<br/> 
				年份：2014 Jun 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant VQ motif-containing <span class="trait">protein</span> family plays crucial roles in plant growth, seed development, and defense responses in Arabidopsis. However, its function in soybean is still not well defined. We aim to identify the VQ gene family, and explore the genetic variation of active <span class="gene">GmVQ</span> genes in soybean and their expression patterns under low nitrogen stresses. A total of 74 VQ motif-containing genes were identified in soybean genome, and were clustered into five distinct subfamilies <span class="gene">(GmVQI-V)</span> with each gene having two or three copies except <span class="gene">GmVQ55</span> <span class="gene">(GmVQIV)</span> with single copy. Fourteen genes with relatively <span class="trait">high</span> expression level, at least in one tissue, were defined as active <span class="gene">GmVQ</span> genes. Most of these active <span class="gene">GmVQ</span> genes specifically expressed in soybean pod shell (7/74), root (9/74) and/or nodule (10/74) respectively. Single nucleotide polymorphism (SNP) analysis in cultivated and wild soybeans revealed there were selected site(s) in <span class="gene">GmVQ6,</span> <span class="gene">GmVQ7,</span> <span class="gene">GmVQ10,</span> <span class="gene">GmVQ26</span> and <span class="gene">GmVQ61,</span> which means that these genes have undergone artificial selection during soybean domestication. After low nitrogen treatment, enhanced expression of VQ genes was noticed in specific tissues, such as <span class="gene">GmVQ53,</span> <span class="gene">GmVQ26,</span> <span class="gene">GmVQ58,</span> <span class="gene">GmVQ61,</span> <span class="gene">GmVQ70</span> and <span class="gene">GmVQ6</span> in shoot, and <span class="gene">GmVQ53,</span> <span class="gene">GmVQ58,</span> <span class="gene">GmVQ48</span> in root. On the contrary, suppressed expression of <span class="gene">GmVQ57,</span> <span class="gene">GmVQ21</span> and <span class="gene">GmVQ1</span> genes was noticed in root after the treatment. Duplicated copy of the active <span class="gene">GmVQ</span> genes showed similar expression pattern, suggesting that these genes might be complete copies. The results suggested that soybean VQ-motif containing genes may act as positive or negative regulators in soybean growth, development and nitrogen metabolism. Taken together, our results provided useful information for functional characterization of soybean <span class="gene">GmVQ</span> genes to unravel their biological roles.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plastid transformation in soybean.</h2>
			<h3>PMID:24599865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dubald M(1), Tissot G, Pelissier B.
				<br/> 
				年份：2014
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The biotechnological potential of plastid genetic engineering has been illustrated in a limited number of higher plant species. We have developed a reproducible method to generate plastid transformants in soybean (Glycine max), a crop of major agronomic importance. The transformation vectors are delivered to embryogenic cultures by the particle gun method and selection performed using the <span class="gene">aadA</span> antibiotic <span class="trait">resistance</span> gene. Homoplasmy is established rapidly in the selected events without the need for further selection or regeneration cycles, and genes of interest can be expressed at a <span class="trait">high</span> level in green tissues. This is a significant step toward the commercial application of this technology.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="gene">GmRAV</span> ortholog is involved in photoperiod and sucrose control of flowering time in soybean.</h2>
			<h3>PMID:24551235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu Q(1), Zhao L(1), Li D(1), Hao D(1), Zhan Y(2), Li W(1).
				<br/> 
				年份：2014 Feb 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photoperiod and sucrose levels play a key role in the control of flowering. <span class="gene">GmRAV</span> reflected a diurnal rhythm with the highest expression at 4 h after the beginning of a dark period in soybean leaves, and was highly up-regulated under short-day (SD) conditions, despite of not following a diurnal pattern under long-day (LD) conditions. <span class="gene">GmRAV-i</span> <span class="gene">(GmRAV-inhibition)</span> transgenic soybean exhibited early flowering phenotype. Two of the FT Arabidopsis homologs, <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a,</span> were highly expressed in the leaves of soybeans with inhibition (-i) of <span class="gene">GmRAV</span> under SD conditions. Moreover, the transcript levels of the two FT homologs in <span class="gene">GmRAV-i</span> soybeans were more sensitive to SD conditions than LD conditions compared to the WT plant. <span class="gene">GmRAV-i</span> soybeans and Arabidopsis rav mutants showed more sensitive hypocotyl elongation responses when compared with wild-type seedlings, and <span class="gene">GmRAV-ox</span> overevpressed in tobacco revealed no sensitive changes in hypocotyl length. These indicated that <span class="gene">GmRAV</span> was a novel negative regulator of SD-mediated flowering and hypocotyl elongation. Although sucrose has been suggested to promote flowering induction in many plant species, <span class="trait">high</span> concentration of sucrose (4% [w/v]) applied into media defer flowering time in Arabidopsis wild-type and rav mutant. This delayed flowering stage might be caused by reduction of LEAFY expression. Furthermore, Arabidopsis rav mutants and <span class="gene">GmRAV-i</span> soybean plants were less sensitive to sucrose by the inhibition assays of hypocotyls and roots growth. In contrast, transgenic <span class="gene">GmRAV</span> overexpressing (-ox) tobacco plants displayed more sensitivity to sucrose. In conclusion, <span class="gene">GmRAV</span> was inferred to have a fundamental function in photoperiod, darkness, and sucrose signaling responses to regulate plant development and flowering induction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nonthermal inactivation of soy (Glycine max Sp.) lipoxygenase by pulsed ultraviolet light.</h2>
			<h3>PMID:24460768  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Janve BA(1), Yang W, Marshall MR, Reyes-De-Corcuera JI, Rababah TM.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This study investigated pulsed ultraviolet (PUV) illumination at different distances from the PUV source on soybean lipoxygenase (LOX) (0.4 mg/mL in 0.01 M Tris-HCl buffer, pH 9) activity. Samples (5 mL) were illuminated for 1, 2, 4, 8, and 16 s at 3 distances 6, 8.5, and 11 cm from the PUV lamp's quartz window. The temperature of 33.5 ± 1.8°C was observed for the highest treatment time of 16 s at the shortest distance of 6 cm, and resulted in a 3.5 log reduction (99.95%) in initial <span class="gene">LOX</span> activity. Illumination time and distance from the lamp significantly (P ≤ 0.05) affected <span class="gene">LOX</span> inactivation. Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) was performed on treated <span class="gene">LOX</span> samples and further <span class="trait">protein</span> profile for treated <span class="gene">LOX</span> filtrate (≤10 kDa), was analyzed by reverse phase high-performance liquid chromatography (RP-HPLC). The <span class="trait">protein</span> profile analysis revealed that <span class="gene">LOX</span> <span class="trait">protein</span> degradation was influenced significantly (P ≤ 0.05) by PUV illumination time.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ascorbic acid 6-palmitate: a potent inhibitor of human and soybean lipoxygenase-dependent lipid peroxidation.</h2>
			<h3>PMID:24359271  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mohamed R(1), Tarannum S, Yariswamy M, Vivek HK, Siddesha JM, Angaswamy N, Vishwanath BS.
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVES: Lipoxygenases (LOX) are the key enzymes involved in the biosynthesis of leukotrienes and reactive oxygen species, which are implicated in pathophysiology of inflammatory disorders. This study was conducted to evaluate the inhibitory effect of water-soluble antioxidant ascorbic acid and its lipophilic derivative, ascorbic acid 6-palmitate (Vcpal) on polymorphonuclear lymphocyte 5-LOX and soybean 15-LOX (sLOX) in vitro. METHODS: <span class="gene">LOX</span> activity was determined by measuring the end products, 5-hydroperoxy eicosatetraenoic acid (5-HETE) and lipid hydroperoxides, by spectrophotometric and <span class="trait">high</span> performance liquid chromatography methods. The substrate-dependent enzyme kinetics and docking studies were carried out to understand the nature of inhibition. KEY FINDINGS: Vcpal potently inhibited 5-LOX when compared with its inhibitory effect on sLOX (IC50; 2.5 and 10.3 μm respectively, P = 0.003). Further, Vcpal inhibited 5-LOX more strongly than the known synthetic drugs: phenidone and nordihydroguaiaretic acid (P = 0.0007). Enzyme kinetic studies demonstrated Vcpal as a non-competitive reversible inhibitor of 5-LOX. In-silico molecular docking revealed <span class="trait">high</span> MolDock and Rerank score for Vcpal than ascorbic acid, complementing in-vitro results. CONCLUSION: Both in-vitro and docking studies demonstrated Vcpal but not ascorbic acid as a non-competitive inhibitor of 5-LOX- and sLOX-induced lipid peroxidation, suggesting a key role for lipophilic nature in bringing about inhibition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of Δ9-stearoyl-ACP-desaturase-C mutants in a <span class="trait">high</span> oleic background on soybean seed oil composition.</h2>
			<h3>PMID:24193357  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ruddle P 2nd(1), Whetten R, Cardinal A, Upchurch RG, Miranda L.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Two new sources of elevated seed stearic acid were identified and the feasibility of an elevated stearic acid, <span class="trait">high</span> oleic acid germplasm was studied. Soybean [Glycine max (L.) Merr.] oil typically contains 2-4% stearic acid. Oil with at least 20% stearic acid is desirable because of its improved baking properties and health profile. This study identifies two new sources of <span class="trait">high</span> stearic acid and evaluates the interaction of <span class="trait">high</span> stearic and oleic acid alleles. TCHM08-1087 and TCHM08-755, <span class="trait">high</span> stearic acid 'Holladay' mutants, were crossed to FAM94-41-3, a line containing a point mutation in a seed-specific isoform of a Δ9-stearoyl-acyl carrier protein-desaturase (SACPD-C). F2-derived lines were evaluated for fatty acid content in four field environments. Sequencing of SACPDs in TCHM08-1087 and TCHM08-755 revealed distinct deletions of at least one megabase encompassing <span class="gene">SACPD-C</span> in both lines. After genotyping, the additive effect for stearic acid was estimated at +1.8% for the <span class="gene">SACPD-C</span> point mutation and +4.1% for the <span class="gene">SACPD-C</span> deletions. Average stearic acid in lines homozygous for the deletions was 12.2%. A FAM94-41-3-derived line and TCHM08-1087-11, a selection from TCHM08-1087, were crossed to S09-2902-145, a line containing missense mutations in two fatty acid desaturases (FAD2-1A and FAD2-1B). F1 plants were grown in a greenhouse and individual F2 seed were genotyped and phenotyped. No interaction was observed between either <span class="gene">FAD2-1A</span> or <span class="gene">FAD2-1B</span> and any of the <span class="gene">SACPD-C</span> mutant alleles. Seed homozygous mutant for SACPD-C/FAD2-1A/FAD2-1B contained 12.7% stearic acid and 65.5% oleic acid while seed homozygous for the <span class="gene">SACPD-C</span> deletion and mutant for <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> averaged 10.4% stearic acid and 75.9% oleic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of Als1, an acetohydroxyacid synthase mutation conferring <span class="trait">resistance</span> to sulfonylurea herbicides in soybean.</h2>
			<h3>PMID:24129392  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ghio C(1), Ramos ML, Altieri E, Bulos M, Sala CA.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The AHAS gene family in soybean was characterized. The locus Als1 for sulfonylurea <span class="trait">resistance</span> was mapped and the resistant allele was characterized at the molecular level. Sulfonylurea (SU) <span class="trait">resistance</span> in soybean is controlled by Als1, a semi-dominant allele obtained by EMS mutagenesis over the cultivar Williams 82. The overall objective of this research was to map Als1 in the soybean genome and to determine the nucleotidic changes conferring <span class="trait">resistance</span> to SU. Four nucleotide sequences <span class="gene">(GmAhas1-4)</span> showing <span class="trait">high</span> homology with the Arabidopsis thaliana acetohydroxyacid synthase (AHAS, EC 4.1.3.18) gene sequence were identified by in silico analysis, PCR-amplified from the SU-resistant line BTK323STS and sequenced. Expression analysis showed that <span class="gene">GmAhas1,</span> located on chromosome 4 by in silico analysis, is the most expressed sequence in true leaves. F2:3 families derived from the cross between susceptible and resistant lines were evaluated for SU <span class="trait">resistance.</span> Mapping results indicate that the locus als1 is located on chromosome 4. Sequence comparison of <span class="gene">GmAhas1</span> between BTK323STS and Williams 82 showed a single nucleotide change from cytosine to thymine at position 532. This transversion generates an amino acid change from proline to serine at position 197 (A. thaliana nomenclature) of the AHAS catalytic subunit. An allele-specific marker developed for the <span class="gene">GmAhas1</span> mutant sequence cosegregated with SU <span class="trait">resistance</span> in the F2 population. Taking together, the mapping, expression and sequencing results indicate that the <span class="gene">GmAhas1</span> sequence corresponds to the Als1 gene sequence controlling SU <span class="trait">resistance</span> in soybean. The molecular breeding tools described herein create the basis to speed up the identification of new mutations in soybean AHAS leading to enhanced levels of <span class="trait">resistance</span> to SU or to other families of AHAS inhibitor herbicides.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmCaM4</span> in soybean enhances <span class="trait">resistance</span> to pathogens and tolerance to salt stress.</h2>
			<h3>PMID:24118726  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rao SS(1), El-Habbak MH, Havens WM, Singh A, Zheng D, Vaughn L, Haudenshield JS, Hartman GL, Korban SS, Ghabrial SA.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant diseases inflict heavy losses on soybean <span class="trait">yield,</span> necessitating an understanding of the molecular mechanisms underlying biotic/abiotic stress responses. Ca(2) (+) is an important universal messenger, and <span class="trait">protein</span> sensors, prominently calmodulins (CaMs), recognize cellular changes in Ca(2) (+) in response to diverse signals. Because the development of stable transgenic soybeans is laborious and time consuming, we used the Bean pod mottle virus (BPMV)-based vector for rapid and efficient <span class="trait">protein</span> expression and gene silencing. The present study focuses on the functional roles of the gene encoding the soybean <span class="gene">CaM</span> isoform <span class="gene">GmCaM4.</span> Overexpression of <span class="gene">GmCaM4</span> in soybean resulted in enhanced <span class="trait">resistance</span> to three plant pathogens and increased tolerance to <span class="trait">high</span> salt conditions. To gain an understanding of the underlying mechanisms, we examined the potential defence pathways involved. Our studies revealed activation/increased expression levels of pathogenesis-related (PR) genes in <span class="gene">GmCaM4-overexpressing</span> plants and the accumulation of jasmonic acid (JA). Silencing of <span class="gene">GmCaM4,</span> however, markedly repressed the expression of PR genes. We confirmed the in vivo interaction between <span class="gene">GmCaM4</span> and the <span class="gene">CaM</span> binding transcription factor Myb2, which regulates the expression of salt-responsive genes, using the yeast two-hybrid (Y2H) system and bimolecular fluorescence complementation assays. <span class="gene">GmCaM4</span> and Glycine max <span class="gene">CaM</span> binding receptor-like kinase <span class="gene">(GmCBRLK)</span> did not interact in the Y2H assays, but the interaction between <span class="gene">GmCaM2</span> and <span class="gene">GmCBRLK</span> was confirmed. Thus, a <span class="gene">GmCaM2-GmCBRLK-mediated</span> salt tolerance mechanism, similar to that reported in Glycine soja, may also be functional in soybean. Confocal microscopy showed subcellular localization of the green fluorescent <span class="trait">protein</span> (GFP)-GmCaM4 fusion <span class="trait">protein</span> in the nucleus and cytoplasm.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Protective effect of soyasaponins on acute liver injury induced by D-galactosamine and lipopolysaccharide in mice].</h2>
			<h3>PMID:24079251  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu HX(1), Zhao WX, Quan JS, Yin XZ.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: To investigate the protective effect of soyasaponins on acute liver injury induced by D-galactosamine (GalN) and lipopolysaccharide (LPS) in mice. METHOD: The mice were randomly divided into five groups: the normal control, the model group, the silymarin (positive control) group, and soyasaponins <span class="trait">high</span> and low-dose groups. They were administered with drugs once every day for 7 days. At the end of the experiment, GalN and LPS were injected intraperitoneally to all of the groups except for the normal group to establish the acute liver injury model. The pathological changes were detected with hematoxylin & eosin (HE) staining, tumor necrosis factor-alpha (TNF-alpha) was detected by ELISA method, and the alanine aminotransferase (ALT), aspartate aminotransferase (AST), catalase (CAT), glutathione peroxidase (GPx), glutathione S-transferase (GST), reduced glutathione (GSH), malondialdehyde (MDA), nitric oxide (NO), and the activation of Caspase-3 and Caspase-8 were detected by the colorimetric method. RESULT: Soyasaponins could reduce the activities of serum ALT and AST, the acute hepatic injury induced by GalN/LPS, serum TNF-alpha level, hepatic NO and MDA contents, and the Caspase-3 and Caspase-8 activations of liver tissues, and increase the hepatic CAT, GPx, <span class="gene">GST</span> and GSH levels. CONCLUSION: Soyasaponins shows the protective effect on acute liver injury induced by GalN and LPS in mice, which may be related to its antioxidative ability and anti-liver apoptosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Linkage mapping, molecular cloning and functional analysis of soybean gene Fg2 encoding flavonol 3-O-glucoside (1 → 6) rhamnosyltransferase.</h2>
			<h3>PMID:24072327  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rojas Rodas F(1), Rodriguez TO, Murai Y, Iwashina T, Sugawara S, Suzuki M, Nakabayashi R, Yonekura-Sakakibara K, Saito K, Kitajima J, Toda K, Takahashi R.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						There are substantial genotypic differences in the levels of flavonol glycosides (FGs) in soybean leaves. The first objective of this study was to identify and locate genes responsible for FG biosynthesis in the soybean genome. The second objective was to clone and verify the function of these candidate genes. Recombinant inbred lines (RILs) were developed by crossing the Kitakomachi and Koganejiro cultivars. The FGs were separated by <span class="trait">high</span> performance liquid chromatography (HPLC) and identified. The FGs of Koganejiro had rhamnose at the 6″-position of the glucose or galactose bound to the 3-position of kaempferol, whereas FGs of Kitakomachi were devoid of rhamnose. Among the 94 RILs, 53 RILs had HPLC peaks classified as Koganejiro type, and 41 RILs had peaks classified as Kitakomachi type. The segregation fitted a 1:1 ratio, suggesting that a single gene controls FG composition. SSR analysis, linkage mapping and genome database survey revealed a candidate gene in the molecular linkage group O (chromosome 10). The coding region of the gene from Koganejiro, designated as <span class="gene">GmF3G6″Rt-a,</span> is 1,392 bp long and encodes 464 amino acids, whereas the gene of Kitakomachi, <span class="gene">GmF3G6″Rt-b,</span> has a two-base deletion resulting in a truncated polypeptide consisting of 314 amino acids. The recombinant <span class="gene">GmF3G6″Rt-a</span> <span class="trait">protein</span> converted kaempferol 3-O-glucoside to kaempferol 3-O-rutinoside and utilized 3-O-glucosylated/galactosylated flavonols and UDP-rhamnose as substrates. <span class="gene">GmF3G6″Rt-b</span> <span class="trait">protein</span> had no activity. These results indicate that <span class="gene">GmF3G6″Rt</span> encodes a flavonol 3-O-glucoside (1 → 6) rhamnosyltransferase and it probably corresponds to the Fg2 gene. <span class="gene">GmF3G6″Rt</span> was designated as UGT79A6 by the UGT Nomenclature Committee.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmNAC5, a NAC Transcription Factor, Is a Transient Response Regulator Induced by Abiotic Stress in Soybean</h2>
			<h3>PMID:23983646</h3>
				

				
					<h3>PMCID:3745988</h3>
					<h4>期刊:The Scientific World Journal</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> Hangxia </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Guangli </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meng </surname>  <given-names> Qingchang </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Fang </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  <xref ref-type="corresp" rid="cor2"> * </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC5</span> is a member of NAM subfamily belonging to NAC transcription factors in soybean (Glycine max (L.) Merr.). Studies on NAC transcription factors have shown that this family functioned in the regulation of shoot apical meristem (SAM), hormone signalling, and stress responses. In this study, we examined the expression levels of <span class="gene">GmNAC5.</span> <span class="gene">GmNAC5</span> was highly expressed in the roots and immature seeds, especially strongly in immature seeds of 40 days after flowering. In addition, we found that <span class="gene">GmNAC5</span> was induced by mechanical wounding, <span class="trait">high</span> salinity, and cold treatments but was not induced by abscisic acid (ABA). The subcellular localization assay suggested that <span class="gene">GmNAC5</span> was targeted at nucleus. Together, it was suggested that <span class="gene">GmNAC5</span> might be involved in seed development and abiotic stress responses in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC5</span> is a member of NAM subfamily belonging to NAC transcription factors in soybean ( <italic> Glycine max </italic> (L.) Merr.). Studies on NAC transcription factors have shown that this family functioned in the regulation of shoot apical meristem (SAM), hormone signalling, and stress responses. In this study, we examined the expression levels of <italic> <span class="gene">GmNAC5</span> </italic> . <italic> <span class="gene">GmNAC5</span> </italic> was highly expressed in the roots and immature seeds, especially strongly in immature seeds of 40 days after flowering. In addition, we found that <italic> <span class="gene">GmNAC5</span> </italic> was induced by mechanical wounding, <span class="trait">high</span> salinity, and cold treatments but was not induced by abscisic acid (ABA). The subcellular localization assay suggested that <span class="gene">GmNAC5</span> was targeted at nucleus. Together, it was suggested that <span class="gene">GmNAC5</span> might be involved in seed development and abiotic stress responses in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The NAC family consists of several subfamilies [ <xref rid="B15" ref-type="bibr"> 15 </xref> ]. The NAM subfamily is the best studied NAC subfamily. <italic> CUC1 </italic> and <italic> CUC2 </italic> , encoding NAM subfamily proteins, are a pair of functionally redundant genes, expressed in <italic> Arabidopsis </italic> meristem and organ primordia boundary [ <xref rid="B1" ref-type="bibr"> 1 </xref> , <xref rid="B16" ref-type="bibr"> 16 </xref> ]. The cotyledons of the transgenic seedlings overexpressing <italic> CUC1 </italic> ( <italic> 35S::CUC1 </italic> ) regularly had two basal lobes, small and round epidermal cells between the sinuses, and adventitious SAMs on the adaxial surface of this region [ <xref rid="B17" ref-type="bibr"> 17 </xref> ]. It has been reported that CUC2 is essential for dissecting the leaves of a wide range of lobed/serrated <italic> Arabidopsis </italic> lines. Inactivation of <italic> CUC3 </italic> leads to a partial suppression of the serrations, indicating a role for this gene in leaf shaping. Morphometric analysis of leaf development and genetic analysis provide evidences for different temporal contributions of CUC2 and CUC3 [ <xref rid="B18" ref-type="bibr"> 18 </xref> ]. The <italic> CUP </italic> played an important role in the lateral organ boundary forming snapdragon. Cupuliformis mutants are defective in shoot apical meristem formation, but cup plants overcome this early barrier to development to reach maturity. <italic> CUP </italic> encodes a NAM <span class="trait">protein,</span> homologous to the petunia NAM and <italic> Arabidopsis </italic> CUC proteins. The phenotype of <italic> cup </italic> mutants differs from the phenotype of <italic> NAM </italic> and <italic> CUC1 CUC2 </italic> in that dramatic organ fusion is observed throughout development [ <xref rid="B19" ref-type="bibr"> 19 </xref> ]. Phloem transport of <italic> CmNACP </italic> mRNA was proved directly by heterograft studies between pumpkin and cucumber plants, in which <italic> CmNACP </italic> transcripts were shown to accumulate in cucumber scion phloem and apical tissues [ <xref rid="B20" ref-type="bibr"> 20 </xref> ]. Petunia NAM proteins were mainly expressed in the meristem and primordia boundaries, which might be required by embryo and flower pattern formation [ <xref rid="B5" ref-type="bibr"> 5 </xref> ]. For abiotic stress, it was observed that <italic> Arabidopsis AtNAC2 </italic> expression was induced by salt stress and this induction was reduced in magnitude in the transgenic <italic> Arabidopsis </italic> plants overexpressing tobacco ethylene receptor gene <italic> NTHK1. AtNAC2 </italic> was localized in the nucleus and had transcriptional activation activity. It can form a homodimer in yeast. <italic> AtNAC2 </italic> was highly expressed in roots and flowers but less expressed in other organs examined. In addition to the salt induction, <italic> AtNAC2 </italic> can be induced by abscisic acid (ABA), ACC, and NAA [ <xref rid="B21" ref-type="bibr"> 21 </xref> ]. These showed that the NAM subfamily members not only play a regulatory role in plant development but also participate in stress responses. <italic> <span class="gene">GmNAC5</span> </italic> , which is a member of NAM subfamily belonging to NAC transcription factor in soybean, was cloned and analysed [ <xref rid="B22" ref-type="bibr"> 22 </xref> ]. In order to further study the physiological and biochemical processes that <italic> <span class="gene">GmNAC5</span> </italic> gene may be involved in, the soybean organ expression patterns of the gene and the relationship between <italic> <span class="gene">GmNAC5</span> </italic> gene and abiotic stress were examined.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmNAC5</span> </italic> encoding product is presumed to act as a transcription factor. If transcription factors achieve the precise adjustment of the target genes, this specific transcription factor should be located in the nucleus. Interestingly, <span class="gene">GmNAC5</span> lacks the traditional nuclear localization signal (NLS); even some researchers have found that some NAC domain proteins have the nuclear localization signals [ <xref rid="B16" ref-type="bibr"> 16 </xref> , <xref rid="B25" ref-type="bibr"> 25 </xref> , <xref rid="B26" ref-type="bibr"> 26 </xref> ]. To clarify whether soybean NAC <span class="trait">protein</span> <span class="gene">GmNAC5</span> is located in the nucleus, the subcellular localization assay was performed ( <xref ref-type="fig" rid="fig2"> Figure 2 </xref> ). Despite the transient expression in the onion epidermal cells, it was observed that the <span class="gene">GmNAC5-GFP</span> fusion <span class="trait">protein</span> was located predominantly in the nucleus whereas GFP alone was localized throughout the cells ( <xref ref-type="fig" rid="fig2"> Figure 2(b) </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmNAC5</span> </italic> was weakly expressed in leaves in soybean under normal growth condition. The real-time qRT-PCR was performed to detect the expression of <italic> <span class="gene">GmNAC5</span> </italic> in soybean under various stresses ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> ). For jasmonic acid treatment, <italic> <span class="gene">GmNAC5</span> </italic> was significantly induced after 3&#x02009;h of JA treatment ( <xref ref-type="fig" rid="fig4"> Figure 4(a) </xref> ). For mechanical wounding, expression of <italic> <span class="gene">GmNAC5</span> </italic> was sharply induced after 1&#x02009;h of treatment ( <xref ref-type="fig" rid="fig4"> Figure 4(b) </xref> ). For NaCl treatment, <italic> <span class="gene">GmNAC5</span> </italic> expression was markedly upregulated by 8-fold after 3&#x02009;h of treatment and then decreased ( <xref ref-type="fig" rid="fig4"> Figure 4(c) </xref> ). Under <span class="trait">drought</span> treatment, expression of <italic> <span class="gene">GmNAC5</span> </italic> showed a weak increase and then declined ( <xref ref-type="fig" rid="fig4"> Figure 4(d) </xref> ). For cold stress, it was found that <italic> <span class="gene">GmNAC5</span> </italic> expression was gradually increased and reached the maximum after 12&#x02009;h of treatment ( <xref ref-type="fig" rid="fig4"> Figure 4(e) </xref> ). In order to reveal whether stress responsive expression of <italic> <span class="gene">GmNAC5</span> </italic> was involved in ABA pathway, we studied expression of <italic> <span class="gene">GmNAC5</span> </italic> under ABA treatment ( <xref ref-type="fig" rid="fig4"> Figure 4(f) </xref> ). The qRT-PCR assay suggested that expression of <span class="gene">GmNAC5</span> was not markedly affected by ABA, suggesting that <span class="gene">GmNAC5</span> may participate in ABA-independent signaling pathway in soybean under abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						Subcellular localization of <span class="gene">GmNAC5.</span> (a) The structure of 35S:GmNAC5-GFP vector. (b) Subcellular localization of <span class="gene">GmNAC5-GFP</span> fusion <span class="trait">protein.</span> The arrow indicates the location of the nucleus. Bars: 40&#x02009; <italic> &#x003bc; </italic> m in 35S:GFP; 80&#x02009; <italic> &#x003bc; </italic> m in 35S:GmNAC5-GFP.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of <italic> <span class="gene">GmNAC5</span> </italic> in soybean seedlings under various stresses. The soybean seedlings were stressed with 100&#x02009; <italic> &#x003bc; </italic> M JA (a), wounding (b), 200&#x02009;mM NaCl (c), <span class="trait">drought</span> (d), 4&#x000b0;C (e), and 100&#x02009; <italic> &#x003bc; </italic> M ABA (f).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Antioxidant capacity and DNA-interaction studies of zinc complexes with a non-steroidal anti-inflammatory drug, mefenamic acid.</h2>
			<h3>PMID:23948577  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tarushi A(1), Karaflou Z, Kljun J, Turel I, Psomas G, Papadopoulos AN, Kessissoglou DP.
				<br/> 
				年份：2013 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Zinc(II) complexes of a non-steroidal anti-inflammatory drug, mefenamic acid(=Hmef) in the absence or presence of the nitrogen donor heterocyclic ligands 2,2'-bipyridine(=bipy), 2,2'-bipyridylamine(=bipyam), 2,2'-dipyridylketone oxime(=Hpko) or 1,10-phenanthroline(=phen) have been synthesized and characterized. The crystal structures of [Zn(mef-O,O')2(bipy)], 2, [Zn(mef-O)2(Hpko-N,N')2]·EtOH, 4 and [Zn(mef-O)(mef-O,O')(phen)(H2O)], 5, have been determined by X-ray crystallography showing distinct binding modes of mefenamato carboxylato group, bidentate in 2, monodentate in 4 or both in 5. Interaction studies of the complexes with calf-thymus DNA (CT DNA) have shown that complexes can bind to CT DNA with [Zn(mef-O)2(Hpko)2] exhibiting the highest binding constant to CT DNA (Kb = 1.93(±0.04) × 10(7) M(-1)). The complexes can bind to CT DNA via intercalation as concluded by DNA solution viscosity measurements. Competitive studies with ethidium bromide (EB) have shown that the complexes can displace the DNA-bound EB. The complexes exhibit good binding affinity to serum albumin proteins with [Zn(mef-O)2(H2O)4], 1 exhibiting the highest quenching ability (kq = 1.46 × 10(15) M(-1) s(-1) for human and 5.55 × 10(15) M(-1) s(-1) for bovine serum albumin). All compounds have been tested for their antioxidant and free radical scavenging activity as well as for their in vitro inhibitory activity against soybean lipoxygenase. The scavenging activity is low to moderate against 1,1-diphenyl-picrylhydrazyl (DPPH) radicals and <span class="trait">high</span> against hydroxyl and 2,2'-azinobis-(3-ethylbenzothiazoline-6-sulfonic acid) (ABTS(+·)) radicals, with [Zn(mef-O)2(H2O)4], 1 (ABTS%, 0.1 mM: 94.75(±1.06)%; (·)OH%, 0.1mM: 96.69(±0.27)%; LOX: IC50 = 27.34(±0.90) μM) exhibiting the highest scavenging activity of the ABTS radical cation among the complexes. Additionally, the complexes exhibit higher scavenging and <span class="gene">LOX</span> inhibitory activity than free mefenamic acid (ABTS%, 0.1 mM: 66.32(±0.38)%; (·)OH%, 0.1 mM: 92.51(±0.44)%; LOX: IC50 = 48.52(±0.88) μM).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmHsp90s,</span> a Heat Shock <span class="trait">Protein</span> 90 (Hsp90) Gene Family Cloning from Soybean, Decrease Damage of Abiotic Stresses in <italic> Arabidopsis thaliana </italic></h2>
			<h3>PMID:23936107</h3>
				

				
					<h3>PMCID:3723656</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xu </surname>  <given-names> Jinyan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xue </surname>  <given-names> Chenchen </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xue </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Jinming </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Na </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xing </surname>  <given-names> Han </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Hsp90 is one of the most conserved and abundant molecular chaperones and is an essential component of the protective stress response; however, its roles in abiotic stress responses in soybean (Glycine max) remain obscure. Here, 12 <span class="gene">GmHsp90</span> genes from soybean were identified and found to be expressed and to function differentially under abiotic stresses. The 12 <span class="gene">GmHsp90</span> genes were isolated and named <span class="gene">GmHsp90A1-GmHsp90A6,</span> <span class="gene">GmHsp90B1,</span> <span class="gene">GmHsp90B2,</span> <span class="gene">GmHsp90C1.1,</span> <span class="gene">GmHsp90C1.2,</span> <span class="gene">GmHsp90C2.1</span> and <span class="gene">GmHsp90C2.2</span> based on their characteristics and <span class="trait">high</span> homology to other Hsp90s according to a new nomenclature system. Quantitative real-time PCR expression data revealed that all the genes exhibited higher transcript levels in leaves and could be strongly induced under heat, osmotic and salt stress but not cold stress. Overexpression of five typical genes <span class="gene">(GmHsp90A2,</span> <span class="gene">GmHsp90A4,</span> <span class="gene">GmHsp90B1,</span> <span class="gene">GmHsp90C1.1</span> and <span class="gene">GmHsp90C2.1)</span> in Arabidopsis thaliana provided useful evidences that <span class="gene">GmHsp90</span> genes can decrease damage of abiotic stresses. In addition, an abnormal accumulation of proline was detected in some transgenic Arabidopsis plants suggested overexpressing <span class="gene">GmHsp90s</span> may affect the synthesis and response system of proline. Our work represents a systematic determination of soybean genes encoding Hsp90s, and provides useful evidence that <span class="gene">GmHsp90</span> genes function differently in response to abiotic stresses and may affect the synthesis and response system of proline.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Hsp90 is one of the most conserved and abundant molecular chaperones and is an essential component of the protective stress response; however, its roles in abiotic stress responses in soybean ( <italic> Glycine max </italic> ) remain obscure. Here, 12 <italic> <span class="gene">GmHsp90</span> </italic> genes from soybean were identified and found to be expressed and to function differentially under abiotic stresses. The 12 <span class="gene">GmHsp90</span> genes were isolated and named <italic> <span class="gene">GmHsp90A1&#x02013;GmHsp90A6</span> </italic> , <italic> <span class="gene">GmHsp90B1</span> </italic> , <italic> <span class="gene">GmHsp90B2</span> </italic> , <italic> <span class="gene">GmHsp90C1.1</span> </italic> , <italic> <span class="gene">GmHsp90C1</span> </italic> . <italic> 2 </italic> , <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 1 </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 2 </italic> based on their characteristics and <span class="trait">high</span> homology to other Hsp90s according to a new nomenclature system. Quantitative real-time PCR expression data revealed that all the genes exhibited higher transcript levels in leaves and could be strongly induced under heat, osmotic and salt stress but not cold stress. Overexpression of five typical genes ( <italic> <span class="gene">GmHsp90A2</span> </italic> , <italic> <span class="gene">GmHsp90A4</span> </italic> , <italic> <span class="gene">GmHsp90B1</span> </italic> , <italic> <span class="gene">GmHsp90C1.1</span> </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 1 </italic> ) in <italic> Arabidopsis thaliana </italic> provided useful evidences that <span class="gene">GmHsp90</span> genes can decrease damage of abiotic stresses. In addition, an abnormal accumulation of proline was detected in some transgenic Arabidopsis plants suggested overexpressing <span class="gene">GmHsp90s</span> may affect the synthesis and response system of proline. Our work represents a systematic determination of soybean genes encoding Hsp90s, and provides useful evidence that <span class="gene">GmHsp90</span> genes function differently in response to abiotic stresses and may affect the synthesis and response system of proline.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean [ <italic> Glycine max </italic> (L.) Merr.] is an important crop plant, and its <span class="trait">yield</span> is severely affected by various environmental conditions, such as heat stress or osmotic stress. In soybean, two Hsp90s ( <italic> <span class="gene">GmHSP90-1</span> </italic> and <italic> <span class="gene">GmHSP90-2</span> </italic> ) have been cloned by Fu <italic> et al </italic> . <xref ref-type="bibr" rid="pone.0069810-Fu1"> [22] </xref> ; However, their roles in abiotic stresses remain obscure. In this study, we report the identification of 12 Hsp90 genes from soybean. For ease of study, we use a new nomenclature system and name them <italic> <span class="gene">GmHsp90A1&#x02013;GmHsp90A6</span> </italic> , <italic> <span class="gene">GmHsp90B1</span> </italic> , <italic> <span class="gene">GmHsp90B2</span> </italic> , <italic> <span class="gene">GmHsp90C1</span> </italic> . <italic> 1 </italic> , <italic> <span class="gene">GmHsp90C1</span> </italic> . <italic> 2 </italic> , <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 1 </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 2 </italic> based on their characteristics and <span class="trait">high</span> homology to other Hsp90s. Their expression patterns in different tissues and abiotic stresses were characterized. Five typical genes ( <italic> <span class="gene">GmHsp90A2</span> </italic> , <italic> <span class="gene">GmHsp90A4</span> </italic> , <italic> <span class="gene">GmHsp90B1</span> </italic> , <italic> <span class="gene">GmHsp90C1</span> </italic> .1 and <italic> <span class="gene">GmHsp90C2</span> </italic> .1) whose homology and expression patterns differ were further characterized for their function in abiotic stress responses using an overexpression approach in model plant Arabidopsis. These studies are significant in elucidation of the role of <span class="gene">GmHsp90</span> in response to abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean <span class="gene">HSP90</span> family is distributed on different chromosomes and the sizes of their ORFs were in the range of 2097&#x02013;2447 bp, while MW was 80.13&#x02013;93.56 kDa ( <xref ref-type="table" rid="pone-0069810-t001"> Table 1 </xref> ). We compared the predicted amino acid sequence similarity among all 12 members of the Hsp90 family, and this indicated that they could be divided into four groups according to their similarity ( <xref ref-type="supplementary-material" rid="pone.0069810.s004"> Table S2 </xref> ). Members of the same group had <span class="trait">high</span> homology but lower similarity to other groups. The homology between <span class="gene">GmHsp90-7,</span> <span class="gene">GmHsp90-8,</span> <span class="gene">GmHsp90-9</span> and <span class="gene">GmHsp90-13</span> was almost 98.4%, while <span class="gene">GmHsp90-4</span> and <span class="gene">GmHsp90-12</span> had about 97.4% homology to each other but only about 50% to other groups. <span class="gene">GmHsp90-3</span> and <span class="gene">GmHsp90-11</span> had almost 97% homology but only 65% homology to <span class="gene">GmHsp90-6</span> and <span class="gene">GmHsp90-14,</span> and only had 44% similarity to other groups.
					</p>
				
					<p>&nbsp;&nbsp;
						The phylogenetic relationships of soybean Hsp90 genes with those of Arabidopsis were investigated. Like the Hsp90 genes from Arabidopsis, all genes from soybean formed three distinct clades, and genes in the same clade had <span class="trait">high</span> homology ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> ). Based on the results of multiple sequence alignments ( <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> ), phylogenetic relationships ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> , <xref ref-type="supplementary-material" rid="pone.0069810.s001"> S1 </xref> ), the new nomenclature system for Hsp90 <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> and subcellular localization prediction, as well as for ease of study of the <span class="gene">GmHsp90</span> genes, we renamed the 12 <span class="gene">GmHsp90s</span> ( <xref ref-type="table" rid="pone-0069810-t001"> Table 1 </xref> ). <span class="gene">GmHsp90</span> proteins included a C-terminal pentapeptide MEEVD ( <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> ), which is diagnostic of cytoplasmic Hsp90 proteins from both plants and animals <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0069810-Gupta1"> [53] </xref> were named <span class="gene">GmHsp90A1&#x02013;GmHsp90A6.</span> As <span class="gene">GmHsp90-1</span> and <span class="gene">GmHsp90-2</span> were similar to <span class="gene">GmHsp90-8</span> and <span class="gene">GmHsp90-9,</span> respectively, and we did not find more complete sequences in the database, we considered that they were the same genes and named them <span class="gene">GmHsp90A3</span> and <span class="gene">GmHsp90A4,</span> respectively. <span class="gene">GmHsp90-4</span> and <span class="gene">GmHsp90-12</span> had <span class="trait">high</span> homology to ER Hsp90s and contained a putative C-terminal KDEL ER-retention motif ( <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> , <xref ref-type="supplementary-material" rid="pone.0069810.s001"> S1a </xref> ), indicating they may be ER Hsp90 proteins <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0069810-Gupta1"> [53] </xref> . So we named them <span class="gene">GmHsp90B1</span> and <span class="gene">GmHsp90B2,</span> respectively. Chen <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> named the <span class="trait">high</span> homology chloroplast and mitochondrial Hsp90 sequence containing a DPW motif at their C-terminal as HSP90C1, and the copies with <span class="trait">high</span> similarity to HSP90C1 but lacking the DPW motif as HSP90C2&#x02013; thus we named the last four sequences in <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> <span class="gene">GmHsp90C1.1,</span> <span class="gene">GmHsp90C1.2,</span> <span class="gene">GmHsp90C2.1</span> and <span class="gene">GmHsp90C2.2</span> which had <span class="trait">high</span> homology to chloroplast AtHsp90-5 and mitochondrial AtHsp90-6 <xref ref-type="bibr" rid="pone.0069810-Krishna1"> [16] </xref> , <xref ref-type="bibr" rid="pone.0069810-Prassinos1"> [54] </xref> , respectively ( <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> , <xref ref-type="supplementary-material" rid="pone.0069810.s001"> S1b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The phylogenetic tree indicated that, in the first clade formed by HSP90A, there were two well- defined branches which separated <span class="gene">GmHsp90A1</span> and <span class="gene">GmHsp90A2</span> from other <span class="gene">GmHsp90A</span> ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> ). <span class="gene">GmHsp90A3&#x02013;GmHsp90A6</span> and AtHsp90-2&#x02013;AtHsp90-4 were so similar that we could not separate them further and showed that cytoplasmic <span class="gene">HSP90</span> had <span class="trait">high</span> homology, whether from soybean or Arabidopsis. <span class="gene">GmHsp90A1</span> and <span class="gene">GmHsp90A2</span> had lower homology with other <span class="gene">GmHSP90A&#x02013;</span> perhaps there is some functional difference between them. HSP90A and HSP90B had higher homology than HSP90C, as if they had originated from different ancestors <xref ref-type="bibr" rid="pone.0069810-Schmitz1"> [13] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To analyze the expression of soybean <span class="gene">HSP90</span> genes in different soybean tissues, quantitative real-time PCR was performed for 12 <span class="gene">GmHsp90</span> genes ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2b </xref> ). Gene-specific primers were also designed for each member ( <xref ref-type="supplementary-material" rid="pone.0069810.s005"> Table S3 </xref> ). Genes from the same branch ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> ) had more similar expression patterns, indicating that they may be functionally redundant. Although expression was diverse in different branches, there were significantly higher transcript levels in the leaves. It may be that the leaves are always exposed to <span class="trait">high</span> temperature during soybean cultivation, so <span class="gene">GmHsp90</span> genes accumulate in leaves for more protection. Some branches had the same expression pattern in various soybean tissues but different transcript levels, suggesting a major role of higher transcript level genes in soybean growth, such as <italic> <span class="gene">GmHsp90B1</span> </italic> and <italic> <span class="gene">GmHsp90B2</span> </italic> . <italic> <span class="gene">GmHsp90C1</span> </italic> were more highly transcribed in flowers, while <italic> <span class="gene">GmHsp90A3</span> </italic> , <italic> <span class="gene">GmHsp90A6</span> </italic> and <italic> <span class="gene">GmHsp90B</span> </italic> genes were expressed more highly in roots than other genes, indicating that they may play some different roles in soybean growth.
					</p>
				
					<p>&nbsp;&nbsp;
						To further understand the function of <span class="gene">GmHsp90</span> genes in soybean, expression of <span class="gene">GmHsp90</span> under several stresses was analyzed at the transcript level ( <xref ref-type="fig" rid="pone-0069810-g003"> Fig. 3 </xref> ). A significant up- regulation was observed as early as 30 min at 42&#x000b0;C among all <span class="gene">GmHsp90</span> genes and then decreased to the normal level at 24 h ( <xref ref-type="fig" rid="pone-0069810-g003"> Fig. 3a, c </xref> ). <italic> <span class="gene">GmHsp90A1</span> </italic> and <italic> <span class="gene">GmHsp90A2</span> </italic> were strongly induced upon heat stress and the relative expression of the two genes increased up to about 6000 or 4000 times after 1 h at 42&#x000b0;C, thereafter transcripts decreased but remain at elevated levels compared to other genes until 24 h. All the <span class="gene">GmHsp90A</span> genes had the highest transcript levels at 1 h, indicating their rapid response to heat stress. <span class="gene">GmHsp90B</span> and <span class="gene">GmHsp90C</span> genes were also strongly induced by heat stress but the highest transcript levels were delayed compared with <span class="gene">GmHsp90A.</span> Similar to tissue- specific expression, <italic> <span class="gene">GmHsp90B1</span> </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> .1 had relatively <span class="trait">high</span> and continuing transcript levels compared to <italic> <span class="gene">GmHsp90B2</span> </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> .2 during heat stress &#x02013; further suggesting their major role.
					</p>
				
					<p>&nbsp;&nbsp;
						Like heat and salt stresses, <span class="gene">GmHsp90</span> genes were also strongly induced by osmotic stress, although the transcript levels of <span class="gene">GmHsp90B</span> and <italic> <span class="gene">GmHsp90C1</span> </italic> were slightly lower ( <xref ref-type="fig" rid="pone-0069810-g003"> Fig. 3b, e </xref> ). Unlike heat and salt stress, the response of <italic> <span class="gene">GmHsp90A1</span> </italic> and <italic> <span class="gene">GmHsp90A2</span> </italic> to osmotic stress was delayed to 3 h, while the expression of other genes was slightly raised at 1 h and all peaks had appeared by 12 h (except for <span class="gene">GmHsp90C)</span> and decreased abruptly to normal levels at 24 h. The expression pattern differed between <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 1 </italic> and <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 2 </italic> , suggesting there may be some functional complementation under osmotic stress. The <span class="gene">GmHsp90</span> genes responded to osmotic stress more quickly than to salt stress but more slowly than to heat stress. However, we did not detect <span class="trait">high</span> up-regulation under cold stress but down- regulation at some time points ( <xref ref-type="fig" rid="pone-0069810-g003"> Fig. 3f </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Twelve members of the Hsp90 family were identified in soybean. Hsp90 from soybean are highly conserved ( <xref ref-type="fig" rid="pone-0069810-g001"> Fig. 1 </xref> , <xref ref-type="fig" rid="pone-0069810-g002"> 2a </xref> ) and for ease of study, we used a new nomenclature system ( <xref ref-type="table" rid="pone-0069810-t001"> Table 1 </xref> ) <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> . Like Hsp90 in Arabidopsis, soybean <span class="gene">GmHsp90A3&#x02013;GmHsp90A6</span> <span class="trait">protein</span> sequences were highly similar suggesting they may be functionally redundant. Although <span class="gene">GmHsp90A1</span> and <span class="gene">GmHsp90A2</span> were also highly homology to other <span class="gene">GmHsp90As,</span> they were a little different to others ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> ) and according to their expression under stresses; they may belong to an inducible form as already identified in mammals <xref ref-type="bibr" rid="pone.0069810-Chen1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0069810-Csermely1"> [56] </xref> . Thus, <span class="gene">GmHsp90A</span> can also be defined as inducible and relatively constitutively expressed.
					</p>
				
					<p>&nbsp;&nbsp;
						Almost all <span class="gene">GmHsp90</span> genes can rapidly and strongly respond to heat stress except <italic> <span class="gene">GmHsp90C1</span> </italic> ( <xref ref-type="fig" rid="pone-0069810-g003"> Fig. 3 </xref> c). As <span class="gene">GmHsp90s</span> can rapidly accumulate and be maintained for a long period, they may play an important role in protecting plants, being involved in signal transduction or maintaining cell viability over a long period of heat stress <xref ref-type="bibr" rid="pone.0069810-Taipale1"> [1] </xref> . The expressions of <italic> <span class="gene">GmHsp90A1</span> </italic> and <italic> <span class="gene">GmHsp90A2</span> </italic> were extremely <span class="trait">high</span> and remained at higher levels than other genes in the whole process, suggesting their inducible and crucial role in heat stress. The peak of <span class="gene">GmHsp90C1</span> genes was delayed and indicated a possible specific function or different control mode under heat stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, overexpression of <span class="gene">GmHsp90A</span> genes conferred higher germination rates and maintained the growth and development of Arabidopsis to abiotic stresses, especially <italic> <span class="gene">GmHsp90A2</span> </italic> , through decreasing chlorophyll loss and lipid peroxidation levels. As <span class="gene">GmHsp90As</span> had <span class="trait">high</span> homology to other cytoplasmic Hsp90 ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> , <xref ref-type="supplementary-material" rid="pone.0069810.s001"> S1a </xref> ), we considered they may have similar function. Cytoplasmic Hsp90 function is a key component underlying maintenance of cellular homeostasis in the face of environmental fluctuation <xref ref-type="bibr" rid="pone.0069810-Neckers1"> [57] </xref> and plays a role via cross-talk with other mechanisms and functions synergistically with other components to decrease cellular damage <xref ref-type="bibr" rid="pone.0069810-Wang1"> [7] </xref> . In addition, as chlorophyll content and MDA content were closely related to oxidative stress which caused by ROS transiently produced by abiotic stresses <xref ref-type="bibr" rid="pone.0069810-Hossain1"> [58] </xref> , we speculate <span class="gene">GmHsp90A</span> may also participate in decreasing oxidative stress damages under abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, overexpression of <italic> <span class="gene">GmHsp90C2</span> </italic> . <italic> 1 </italic> did not confer Arabidopsis as <span class="trait">high</span> tolerance (fresh weight) as other transgenic Arabidopsis plants under osmotics stress ( <xref ref-type="fig" rid="pone-0069810-g006"> Fig. 6i </xref> , <xref ref-type="fig" rid="pone-0069810-g007"> 7b </xref> ). As <span class="gene">GmHsp90C2</span> are highly homology to mitochondria Hsp90, they may have similar function but functions distinct from other Hsp90 <xref ref-type="bibr" rid="pone.0069810-Felts1"> [62] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, there was no significant difference in some performances of <italic> <span class="gene">GmHsp90B1</span> </italic> and control plants under abiotic stresses ( <xref ref-type="fig" rid="pone-0069810-g004"> Fig. 4b, c </xref> , <xref ref-type="fig" rid="pone-0069810-g007"> 7a </xref> ). <span class="gene">GmHsp90B</span> has <span class="trait">high</span> homology to other ER Hsp90 chaperone ( <xref ref-type="fig" rid="pone-0069810-g002"> Fig. 2a </xref> , <xref ref-type="supplementary-material" rid="pone.0069810.s001"> S1a </xref> ) which suggests it may function similar to ER Hsp90. Hsp90B as an ER chaperone has evolved to optimize the output of properly folded secretory and membrane proteins but its function is rather selective and interacts with late folding intermediates <xref ref-type="bibr" rid="pone.0069810-Eletto1"> [63] </xref> . This may partly explain the performances of <italic> <span class="gene">GmHsp90B1</span> </italic> transgenic Arabidopsis plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Phylogenetic relationships analysis of <span class="gene">HSP90</span> proteins in multiple species. </bold> Phylogenetic trees were constructed with <span class="gene">HSP90</span> <span class="trait">protein</span> sequences from <italic> Glycine max </italic> <span class="gene">(GmHsp90),</span> <italic> Arabidopsis thaliana </italic> (AtHsp90), <italic> Drosophila melanogaster </italic> (NP_523899.1, NP_651601.1), <italic> Caenorhabditis elegans </italic> (NP_506626.1), <italic> Gallus gallus </italic> (CAA30251.1, NP_989620.1), <italic> Cryptococcus neoformans </italic> (XP_568451.1), <italic> Saccharomyces cerevisiae </italic> (AAA02743.1), <italic> Danio rerio </italic> (AAH63951.1), <italic> Oryza sativa </italic> (BAA90487.1), <italic> Chlamydomonas reinhardtii </italic> (AAU10511.1), <italic> Bacillus subtilis </italic> (NP_391861.1), <italic> Escherichia coli </italic> (NP_415006.1), <italic> Streptomyces coelicolor </italic> (NP_631561.1),, <italic> Pinus taeda </italic> (ABV21762.1) and <italic> Medicago truncatula </italic> (XP_003589505.1). The trees were constructed using the neighbor-joining algorithm included in the MEGA4.0. software. (a) Phylogenetic relationships analysis of HSP90A and HSP90B proteins in multiple species. (b) Phylogenetic relationships analysis of HSP90C and HTPG proteins in multiple species.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A temporal regulatory mechanism controls the different contribution of endoplasmic reticulum and plastidial ω-3 desaturases to trienoic fatty acid content during leaf development in soybean (Glycine max cv Volania).</h2>
			<h3>PMID:23928132  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lagunas B(1), Román Á, Andreu V, Picorel R, Alfonso M.
				<br/> 
				年份：2013 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We analyzed the molecular mechanism controlling ω-3 fatty acid desaturases during seed germination and leaf development in soybean. During germination, soybean seeds were characterized by a <span class="trait">high</span> 18:2(Δ9,12) level (more than 50%) and reduced 18:3(Δ9,12,15) content (10%). Interestingly, transcripts from all endoplasmic reticulum <span class="gene">(GmFAD3A</span> and <span class="gene">GmFAD3B)</span> and plastidial <span class="gene">(GmFAD7-1/GmFAD7-2</span> or <span class="gene">GmFAD8-1/GmFAD8-2)</span> desaturase genes were detected during seed germination. Upon germination, soybean trifoliate leaf development was accompanied by an increase in linolenic acid (18:3(Δ9,12,15)). Our data showed that transcripts corresponding to the endoplasmic reticulum ω-3 desaturases <span class="gene">GmFAD3A</span> and <span class="gene">GmFAD3B</span> decreased with leaf development. No changes in the expression profile of the plastidial ω-3 desaturases <span class="gene">GmFAD7-1</span> and <span class="gene">GmFAD7-2</span> genes were detected. On the contrary, <span class="gene">GmFAD8-2</span> transcript levels increased while <span class="gene">GmFAD8-1</span> transcripts decreased during leaf development. Given this expression profile, our data suggested the existence of a temporal regulatory mechanism controlling ω-3 desaturases during leaf development in which the endoplasmic reticulum ω-3 desaturases would be more important in young leaves while plastidial ω-3 desaturases might contribute to 18:3(Δ9,12,15) production in mature leaves. Photosynthetic cell cultures showed 18:3(Δ9,12,15) levels similar to those from leaves. No changes in the 18:3(Δ9,12,15) content or expression of the ω-3 desaturase genes were detected along the cell culture cycle. A comparison of our data with those available in Arabidopsis or wheat suggested that the regulatory mechanism controlling the expression and activity of both endoplasmic reticulum and plastidial desaturases during leaf development might differ among plant species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rapid shoot-to-root signalling regulates root hydraulic conductance via aquaporins.</h2>
			<h3>PMID:23926961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vandeleur RK(1), Sullivan W, Athman A, Jordans C, Gilliham M, Kaiser BN, Tyerman SD.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We investigated how root hydraulic conductance (normalized to root dry weight, Lo ) is regulated by the shoot. Shoot topping (about 30% reduction in leaf area) reduced Lo of grapevine (Vitis vinifera L.), soybean (Glycine max L.) and maize (Zea mays L.) by 50 to 60%. More detailed investigations with soybean and grapevine showed that the reduction in Lo was not correlated with the reduction in leaf area, and shading or cutting single leaves had a similar effect. Percentage reduction in Lo was largest when initial Lo was <span class="trait">high</span> in soybean. Inhibition of Lo by weak acid (low pH) was smaller after shoot damage or leaf shading. The half time of reduction in Lo was approximately 5 min after total shoot decapitation. These characteristics indicate involvement of aquaporins. We excluded phloem-borne signals and auxin-mediated signals. Xylem-mediated hydraulic signals are possible since turgor rapidly decreased within root cortex cells after shoot topping. There was a significant reduction in the expression of several aquaporins in the plasma membrane intrinsic <span class="trait">protein</span> (PIP) family of both grapevine and soybean. In soybean, there was a five- to 10-fold reduction in <span class="gene">GmPIP1;6</span> expression over 0.5-1 h which was sustained over the period of reduced Lo .
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteogenomic analysis of Bradyrhizobium japonicum USDA110 using GenoSuite, an automated multi-algorithmic pipeline.</h2>
			<h3>PMID:23882027  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kumar D(1), Yadav AK, Kadimi PK, Nagaraj SH, Grimmond SM, Dash D.
				<br/> 
				年份：2013 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We present GenoSuite, an integrated proteogenomic pipeline to validate, refine and discover <span class="trait">protein</span> coding genes using high-throughput mass spectrometry (MS) data from prokaryotes. To demonstrate the effectiveness of GenoSuite, we analyzed proteomics data of Bradyrhizobium japonicum (USDA110), a model organism to study agriculturally important rhizobium-legume symbiosis. Our analysis confirmed 31% of known genes, refined 49 gene models for their translation initiation site (TIS) and discovered 59 novel <span class="trait">protein</span> coding genes. Notably, a novel <span class="trait">protein</span> which redefined the boundary of a crucial cytochrome <span class="gene">P450</span> system related operon was discovered, known to be highly expressed in the anaerobic symbiotic bacteroids. A focused analysis on N-terminally acetylated peptides indicated downstream TIS for gene blr0594. Finally, ortho-proteogenomic analysis revealed three novel genes in recently sequenced B. japonicum USDA6(T) genome. The discovery of large number of missing genes and correction of gene models have expanded the proteomic landscape of B. japonicum and presents an unparalleled utility of proteogenomic analyses and versatility of GenoSuite for annotating prokaryotic genomes including pathogens.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic variation in four maturity genes affects photoperiod insensitivity and PHYA-regulated post-flowering responses of soybean.</h2>
			<h3>PMID:23799885  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu M(1), Xu Z, Liu B, Kong F, Tsubokura Y, Watanabe S, Xia Z, Harada K, Kanazawa A, Yamada T, Abe J.
				<br/> 
				年份：2013 Jun 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Absence of or low sensitivity to photoperiod is necessary for short-day crops, such as rice and soybean, to adapt to <span class="trait">high</span> latitudes. Photoperiod insensitivity in soybeans is controlled by two genetic systems and involves three important maturity genes: E1, a repressor for two soybean orthologs of Arabidopsis FLOWERING LOCUS T <span class="gene">(GmFT2a</span> and <span class="gene">GmFT5a),</span> and E3 and E4, which are phytochrome A genes. To elucidate the diverse mechanisms underlying photoperiod insensitivity in soybean, we assessed the genotypes of four maturity genes (E1 through E4) in early-flowering photoperiod-insensitive cultivars and their association with post-flowering responses. RESULTS: We found two novel dysfunctional alleles in accessions originally considered to have a dominant E3 allele according to known DNA markers. The E3 locus, together with <span class="gene">E1</span> and E4, contained multiple dysfunctional alleles. We identified 15 multi-locus genotypes, which we subdivided into 6 genotypic groups by classifying their alleles by function. Of these, the e1-as/e3/E4 genotypic group required an additional novel gene (different from E1, E3, and E4) to condition photoperiod insensitivity. Despite their common pre-flowering photoperiod insensitivity, accessions with different multi-locus genotypes responded differently to the post-flowering photoperiod. Cultivars carrying E3 or E4 were sensitive to photoperiod for post-flowering characteristics, such as reproductive period and stem growth after flowering. The phytochrome A-regulated expression of the determinate growth habit gene Dt1, an ortholog of Arabidopsis TERMINAL FLOWER1, was involved in the persistence of the vegetative activity at the stem apical meristem of flower-induced plants under long-day conditions. CONCLUSIONS: Diverse genetic mechanisms underlie photoperiod insensitivity in soybean. At least three multi-locus genotypes consisting of various allelic combinations at E1, E3, and E4 conferred pre-flowering photoperiod insensitivity to soybean cultivars but led to different responses to photoperiod during post-flowering vegetative and reproductive development. The <span class="gene">phyA</span> genes E3 and E4 are major controllers underlying not only pre-flowering but also post-flowering photoperiod responses. The current findings improve our understanding of genetic diversity in pre-flowering photoperiod insensitivity and mechanisms of post-flowering photoperiod responses in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and functional characterization of silicon transporters in soybean using comparative genomics of major intrinsic proteins in Arabidopsis and rice.</h2>
			<h3>PMID:23771580  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deshmukh RK(1), Vivancos J, Guérin V, Sonah H, Labbé C, Belzile F, Bélanger RR.
				<br/> 
				年份：2013 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Silicon (Si) confers several benefits to many plant species when absorbed as silicic acid through nodulin 26-like intrinsic proteins (NIPs). The NIPs belong to major intrinsic <span class="trait">protein</span> (MIP) family, members of which form channels with <span class="trait">high</span> selectivity to control transport of water and different solutes. Here, comparative genomic analysis of the MIPs was performed to investigate the presence of Si transporter MIPs in soybean. Thorough analysis of phylogeny, gene organization, transcriptome profiling and <span class="trait">protein</span> modeling was performed to characterize MIPs in rice, Arabidopsis and soybean. Based on several attributes, two putative Si transporter genes, <span class="gene">GmNIP2-1</span> and <span class="gene">GmNIP2-2,</span> were identified, characterized and cloned from soybean. Expression of both genes was detected in shoot and root tissues, and decreased as Si increased. The <span class="trait">protein</span> encoded by <span class="gene">GmNIP2-2</span> showed functionality for Si transport when expressed in Xenopus oocytes, thus confirming the genetic capability of soybean to absorb the element. Comparative analysis of MIPs in plants provides opportunities to decipher gene evolution, functionality and selectivity of nutrient uptake mechanisms. Exploitation of this strategy has helped to uncover unique features of MIPs in soybean. The identification and functional characterization of Si transporters can be exploited to optimize the benefits that plants can derive from Si absorption.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of flavonoid 6-hydroxylase candidate genes in normal and mutant soybean genotypes for glycitein content.</h2>
			<h3>PMID:23695494  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Artigot MP(1), Baes M, Daydé J, Berger M.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean seeds accumulate large amounts of isoflavones (genistein, daidzein and glycitein), secondary metabolites known for their phytoestrogenic activities. Isoflavone composition depends on the seed part and glycitein is almost found exclusively in hypocotyls. Moreover, two major phenotypes are encountered in soybean cultivars, with either low (35 %) or <span class="trait">high</span> (55 %) levels of glycitein in their hypocotyls. This trait was under a quasi-mendelian heredity, implicating at most one or two genes. A <span class="gene">CYP71D9</span> cDNA displaying a flavonoid 6-hydroxylase (F6H) activity had previously been isolated from elicitor-induced soybean (Glycine max L.) cells. This enzyme allows the synthesis of the glycitein flavanone intermediate (6,7,4'- trihydroxyflavanone) by catalyzing the A-ring hydroxylation of liquiritigenin. In this study, the <span class="gene">CYP71D9</span> gene (F6H1) and two other candidates (F6H2 and F6H3) were studied using contrasted soybean cultivars for glycitein content (0, 35, 55 and 80 %). Their expression was observed in chitosan elicited leaves. They encode <span class="gene">P450</span> proteins of 496, 469 and 481 amino acids respectively and were expressed in leaves with or without elicitation. The expression patterns of these three genes were performed in cotyledons and hypocotyls at different developmental stages. <span class="gene">F6H1</span> and <span class="gene">F6H2</span> were not expressed in the developing seed. <span class="gene">F6H3</span> was only expressed in hypocotyls. Its expression levels did not correlate with hypocotyls glycitein content, but it was not expressed in the null mutant for glycitein. Thus, this <span class="gene">F6H3</span> gene is a good potential candidate for glycitein biosynthesis in soybean seed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmTMT2a from soybean elevates the α-tocopherol content in corn and Arabidopsis.</h2>
			<h3>PMID:23645501  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang L(1), Luo Y, Zhu Y, Zhang L, Zhang W, Chen R, Xu M, Fan Y, Wang L.
				<br/> 
				年份：2013 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Tocochromanol, or vitamin E, plays a crucial role in human and animal nutrition and is synthesized only by photosynthetic organisms. γ-Tocopherol methyltransferase (γ-TMT), one of the key enzymes in the tocopherol biosynthetic pathway in plants, converts γ, δ-tocopherols into α-, β-tocopherols. Tocopherol content was investigated in 15 soybean cultivars and <span class="gene">GmTMT2</span> was isolated from five varieties based on tocopherol content. <span class="gene">GmTMT2a</span> was expressed in E. coli and the purified <span class="trait">protein</span> effectively converted γ-tocopherol into α-tocopherol in vitro. Overexpression of <span class="gene">GmTMT2a</span> enhanced α-tocopherol content 4-6-fold in transgenic Arabidopsis, and α-tocopherol content increased 3-4.5-fold in transgenic maize seed, which correlated with the accumulation of <span class="gene">GmTMT2a.</span> Transgenic corn that is α-tocopherol-rich may be beneficial for animal health and growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean seed oil: II. QTL and genes that increase oil concentration without decreasing <span class="trait">protein</span> or with increased seed <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] seed oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as seed <span class="trait">yield</span> or <span class="trait">protein</span> concentration. The objectives of the present study were to study the relationship between seed oil concentration and important agronomic and seed quality traits, including seed <span class="trait">yield,</span> 100-seed weight, <span class="trait">protein</span> concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately <span class="trait">high</span> oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with seed oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for <span class="trait">protein</span> concentration, four for seed <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with seed <span class="trait">protein</span> concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with seed <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with seed oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for <span class="trait">protein</span> concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as seed <span class="trait">yield</span> and <span class="trait">protein</span> concentration, could be used in the soybean marker breeding programs aimed at developing either higher seed <span class="trait">yield</span> and oil concentration or higher seed <span class="trait">protein</span> and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmNFYA3, a target gene of miR169, is a positive regulator of plant tolerance to <span class="trait">drought</span> stress.</h2>
			<h3>PMID:23483290  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni Z(1), Hu Z, Jiang Q, Zhang H.
				<br/> 
				年份：2013 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nuclear factor Y (NF-Y) is a heterotrimeric transcription factor composed of NF-YA, NF-YB and NF-YC proteins. In this study, we identified and characterized a gene, <span class="gene">GmNFYA3,</span> which encodes the NF-YA subunit of the NF-Y complex in soybeans (Glycine max L.). Real time RT-PCR analysis indicated that <span class="gene">GmNFYA3</span> was induced by abscisic acid (ABA) and abiotic stresses, such as polyethylene glycol, NaCl and cold. Subcellular localization analysis suggested that <span class="gene">GmNFYA3</span> may activate its specific targets in the nucleus. Histochemical β-glucuronidase (GUS) staining revealed that the expression of the GUS gene driven by the <span class="gene">GmNFYA3</span> promoter occurred in various transgenic Arabidopsis tissues. Coexpression in Nicotiana benthamiana and 5' RACE assays indicated that miR169 directs <span class="gene">GmNFYA3</span> mRNA cleavage in vivo. Overexpression of <span class="gene">GmNFYA3</span> resulted in Arabidopsis with reduced leaf water loss and enhanced <span class="trait">drought</span> tolerance. In addition, the transgenic Arabidopsis exhibited increased sensitivity to <span class="trait">high</span> salinity and exogenous ABA. Moreover, the transcript levels of ABA biosynthesis (ABA1, ABA2), ABA signaling (ABI1, ABI2) and stress-responsive genes, including <span class="gene">RD29A</span> and CBF3, were generally higher in <span class="gene">GmNFYA3</span> plants than in wild-type controls under normal conditions. These results suggest that the <span class="gene">GmNFYA3</span> gene functions in positive modulation of <span class="trait">drought</span> stress tolerance and has potential applications in molecular breeding to enhance <span class="trait">drought</span> tolerance in crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmGBP1, a homolog of human ski interacting <span class="trait">protein</span> in soybean, regulates flowering and stress tolerance in Arabidopsis.</h2>
			<h3>PMID:23388059  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang Y(1), Zhao L, Li H, Gao Y, Li Y, Wu X, Teng W, Han Y, Zhao X, Li W.
				<br/> 
				年份：2013 Feb 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: SKIP is a transcription cofactor in many eukaryotes. It can regulate plant stress tolerance in rice and Arabidopsis. But the homolog of SKIP <span class="trait">protein</span> in soybean has been not reported up to now. RESULTS: In this study, the expression patterns of soybean GAMYB binding <span class="trait">protein</span> gene <span class="gene">(GmGBP1)</span> encoding a homolog of SKIP <span class="trait">protein</span> were analyzed in soybean under abiotic stresses and different day lengths. The expression of <span class="gene">GmGBP1</span> was induced by polyethyleneglycol 6000, NaCl, gibberellin, abscisic acid and heat stress. <span class="gene">GmGBP1</span> had transcriptional activity in C-terminal. <span class="gene">GmGBP1</span> could interact with R2R3 domain of <span class="gene">GmGAMYB1</span> in SKIP domain to take part in gibberellin flowering pathway. In long-day (16 h-light) condition, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> exhibited earlier flowering and less number of rosette leaves; Suppression of AtSKIP in Arabidopsis resulted in growth arrest, flowering delay and down-regulation of many flowering-related genes (CONSTANS, FLOWERING LOCUS T, LEAFY); Arabidopsis myb33 mutant plants with ectopic overexpression of <span class="gene">GmGBP1</span> showed the same flowering phenotype with wild type. In short-day (8 h-light) condition, transgenic Arabidopsis plants with <span class="gene">GmGBP1</span> flowered later and showed a higher level of FLOWERING LOCUS C compared with wild type. When treated with abiotic stresses, transgenic Arabidopsis with the ectopic overexpression of <span class="gene">GmGBP1</span> enhanced the tolerances to heat and <span class="trait">drought</span> stresses but reduced the tolerance to <span class="trait">high</span> salinity, and affected the expressions of several stress-related genes. CONCLUSIONS: In Arabidopsis, <span class="gene">GmGBP1</span> might positively regulate the flowering time by affecting CONSTANS, FLOWERING LOCUS T, LEAFY and GAMYB directly or indirectly in photoperiodic and gibberellin pathways in LDs, but <span class="gene">GmGBP1</span> might represse flowering by affecting FLOWERING LOCUS C and SHORT VEGETATIVE PHASE in autonomous pathway in SDs. <span class="gene">GmGBP1</span> might regulate the activity of ROS-eliminating to improve the <span class="trait">resistance</span> to heat and <span class="trait">drought</span> but reduce the high-salinity tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification of soybean microRNAs and their targets reveals their organ-specificity and responses to phosphate starvation</h2>
			<h3>PMID:23368765</h3>
				

				
					<h3>PMCID:3673897</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Xu </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> june92634@126.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Liu </surname>  <given-names> Qian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> lqqhdx@yahoo.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Chen </surname>  <given-names> Luying </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> cly0406@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kuang </surname>  <given-names> Jiebin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> binel168@163.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Walk </surname>  <given-names> Thomas </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> tcwalk@gmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Wang </surname>  <given-names> Jinxiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jinxwang@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Liao </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hliao@scau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Phosphorus (P) plays important roles in plant growth and development. MicroRNAs involved in P signaling have been identified in Arabidopsis and rice, but P-responsive microRNAs and their targets in soybean leaves and roots are poorly understood. RESULTS: Using high-throughput sequencing-by-synthesis (SBS) technology, we sequenced four small RNA libraries from leaves and roots grown under phosphate (Pi)-sufficient (+Pi) and Pi-depleted (-Pi) conditions, respectively, and one RNA degradome library from Pi-depleted roots at the genome-wide level. Each library generated ~21.45-28.63 million short sequences, resulting in ~20.56-27.08 million clean reads. From those sequences, a total of 126 miRNAs, with 154 gene targets were computationally predicted. This included 92 new miRNA candidates with 20-23 nucleotides that were perfectly matched to the Glycine max genome 1.0, 70 of which belong to 21 miRNA families and the remaining 22 miRNA unassigned into any existing miRNA family in miRBase 18.0. Under both +Pi and -Pi conditions, 112 of 126 total miRNAs (89%) were expressed in both leaves and roots. Under +Pi conditions, 12 leaf- and 2 root-specific miRNAs were detected; while under -Pi conditions, 10 leaf- and 4 root-specific miRNAs were identified. Collectively, 25 miRNAs were induced and 11 miRNAs were repressed by Pi starvation in soybean. Then, stem-loop real-time PCR confirmed expression of four selected P-responsive miRNAs, and RLM-5' RACE confirmed that a PHO2 and <span class="gene">GmPT5,</span> a kelch-domain containing <span class="trait">protein,</span> and a Myb transcription factor, respectively are targets of miR399, miR2111, and miR159e-3p. Finally, P-responsive cis-elements in the promoter regions of soybean miRNA genes were analyzed at the genome-wide scale. CONCLUSIONS: Leaf- and root-specific miRNAs, and P-responsive miRNAs in soybean were identified genome-wide. A total of 154 target genes of miRNAs were predicted via degradome sequencing and computational analyses. The targets of miR399, miR2111, and miR159e-3p were confirmed. Taken together, our study implies the important roles of miRNAs in P signaling and provides clues for deciphering the functions for microRNA/target modules in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Using high-throughput sequencing-by-synthesis (SBS) technology, we sequenced four small RNA libraries from leaves and roots grown under phosphate (Pi)-sufficient (+Pi) and Pi-depleted (-Pi) conditions, respectively, and one RNA degradome library from Pi-depleted roots at the genome-wide level. Each library generated &#x0223c;21.45&#x02212;28.63 million short sequences, resulting in &#x0223c;20.56&#x02212;27.08 million clean reads. From those sequences, a total of 126 miRNAs, with 154 gene targets were computationally predicted. This included 92 new miRNA candidates with 20-23 nucleotides that were perfectly matched to the <italic> Glycine max </italic> genome 1.0, 70 of which belong to 21 miRNA families and the remaining 22 miRNA unassigned into any existing miRNA family in miRBase 18.0. Under both +Pi and -Pi conditions, 112 of 126 total miRNAs (89%) were expressed in both leaves and roots. Under +Pi conditions, 12 leaf- and 2 root-specific miRNAs were detected; while under -Pi conditions, 10 leaf- and 4 root-specific miRNAs were identified. Collectively, 25 miRNAs were induced and 11 miRNAs were repressed by Pi starvation in soybean. Then, stem-loop real-time PCR confirmed expression of four selected P-responsive miRNAs, and RLM-5&#x02019; RACE confirmed that a PHO2 and <span class="gene">GmPT5,</span> a kelch-domain containing <span class="trait">protein,</span> and a Myb transcription factor, respectively are targets of miR399, miR2111, and miR159e-3p. Finally, P-responsive <italic> cis </italic> -elements in the promoter regions of soybean miRNA genes were analyzed at the genome-wide scale.
					</p>
				
					<p>&nbsp;&nbsp;
						Small RNAs containing 21, 22, or 24 nucleotides had <span class="trait">high</span> abundance relative to those of other lengths in all four libraries, while those containing 20 nucleotides were <span class="trait">high</span> in leaves (Figure <xref ref-type="fig" rid="F1"> 1 </xref> A; Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). The 39-45% abundance of 21 nt small RNAs in <span class="gene">HPL</span> and LPL library was dominant over 22 and 24-nt small RNAs (Figure <xref ref-type="fig" rid="F1"> 1 </xref> A). When only unique reads were considered, 24-nt small RNAs were most abundant, with 21 nt and 22 nt small RNA following in all libraries (Figure <xref ref-type="fig" rid="F1"> 1 </xref> B). Overall, these data are consistent with previous studies [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B20"> 20 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A number of miRNAs are notable for their expression patterns. This included most members of miR156, 164 and 167 families, along with 12 individual miRNAs (miR168, miR172b-3p, miR2118a, miR2118b, miR408c, miR1507a, miR1508d, miR1508e, miR1509a, miR1510b-5p, miR1510c, and miR1511) that were found in <span class="trait">high</span> abundance (&#x0003e;1000 TPM) in one or both of the <span class="gene">HPL</span> or LPL treatments (Tables <xref ref-type="table" rid="T3"> 3 </xref> , <xref ref-type="table" rid="T4"> 4 </xref> and <xref ref-type="table" rid="T5"> 5 </xref> ). Others appeared to be constitutively expressed in leaves and roots, including, for example, members of the conserved miR160, miR164, and miR2118 families (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Within families, expression can vary significantly among family members. For example, expression levels of miR156 family members ranged from 46 to 66744 TPM in leaves, and from 0.01 (actually 0, only for normalization) to 22303 TPM in roots, and expression levels of miR166 family members ranged from from 48 to 16705 TPM in leaves, and from 12 to 20693 TPM in roots (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Overall, a diverse range of responses was observed among treatments, tissues and family members, but variation between leaves and roots appeared to be more prevalent in less-conserved than in conserved miRNAs (Tables <xref ref-type="table" rid="T3"> 3 </xref> , <xref ref-type="table" rid="T4"> 4 </xref> and <xref ref-type="table" rid="T5"> 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						As for the conserved miRNAs, the number of targets for conserved miRNA (2.31 targets per miRNA) was higher than that of less-conserved miRNA (1.53 targets per miRNA) (Additional files <xref ref-type="supplementary-material" rid="S7"> 7 </xref> and <xref ref-type="supplementary-material" rid="S8"> 8 </xref> ). Among the conserved miRNAs, five (miR162c, miR166h-3p, miR166j-3p, miR390b,and miR408b-5p) were identified to have only one target, while the rest have two or more targets, most notably miR156d, miR167e, miR167g, miR167j, miR167k, miR172b-3p, and miR172h-5p (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). In contrast, 15 out of 25 less-conserved miRNAs had only one target. However, the previously unreported less-conserved miR1508e and miR3508 each have seven targets (Additional file <xref ref-type="supplementary-material" rid="S8"> 8 </xref> ). <italic> <span class="gene">Glyma08g21610</span> </italic> and <italic> <span class="gene">Glyma16g34300</span> </italic> , which encode a AGO proteinendoing and a HD-ZIP <span class="trait">protein</span> respectively, were predicted target of miR166g and miR168 (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). A NF-YA gene <italic> <span class="gene">Glyma10g10240</span> </italic> was the putative target of miR169c (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). Importantly, these predicted cleavage sites for the three targets are consistent with previous RLM-5&#x02019; RACE results [ <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ], supporting the reliability of our computational analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						In regards to nutrient stress, the targets of root P-responsive miRNAs and their cleavage sites are highlighted in Table <xref ref-type="table" rid="T6"> 6 </xref> . A PHO2, two Pi transporter transcripts ( <italic> <span class="gene">Glyma10g04230</span> </italic> and <italic> <span class="gene">Glyma14g36650</span> </italic> ), and an <span class="gene">AP2</span> <span class="trait">protein</span> gene ( <italic> <span class="gene">Glyma01g13410</span> </italic> ) were predicted to be targets of miR399a, b, c, d, and e (Table <xref ref-type="table" rid="T6"> 6 </xref> ). In fact, miR2111 was detected in small RNA libraries in this study, but the level of it was too low to be filtered out based on the strict criteria. The target of miR2111 was predicted. As in Arabidopsis, a kelch repeat-containing F-box <span class="trait">protein,</span> <italic> <span class="gene">Glyma16g01060</span> </italic> , was the putative target of miR2111. The targets of the Pi starvation-induced miR169o, miR397a, miR408c, miR4416a, miRnov_2, and miRnov_7 were detected in the degradome library or computationally predicted, as were the targets of the P limitation-repressed miRNAs miR159e-3p, miR169r, miR2109, miR4376, miRnov_5a, miRnov_5b, and miRnov_6a (Table <xref ref-type="table" rid="T6"> 6 </xref> ). It must be pointed out that most of the targets listed in Additional files <xref ref-type="supplementary-material" rid="S7"> 7 </xref> , <xref ref-type="supplementary-material" rid="S8"> 8 </xref> and <xref ref-type="supplementary-material" rid="S9"> 9 </xref> need to be experimentally confirmed with RLM-5&#x02019; RACE or transit expression analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						AP2, <span class="gene">AP2</span> domain <span class="trait">protein;</span> CDS: coding sequence; Cleavage site: nucleotide position in cDNA cleaved by miRNA from 5&#x02019; to 3&#x02019;; Myb, Myb family of transcription factors; NF-Y, NUCLEAR FACTOR Y (also CCAAT, CCAAT-binding transcription factor); TP10M: transcripts per 10 million reads; UTR: untranslated region.
					</p>
				
					<p>&nbsp;&nbsp;
						MiRNAs play crucial roles in leaf and root development [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. Figure <xref ref-type="fig" rid="F2"> 2 </xref> shows 112 miRNAs detected both in leaves and roots, along with leaf- and root-specific miRNAs extracted from +Pi or Pi treatments. Comparable to the reported very <span class="trait">high</span> levels of miR156f and miR156h in soybean leaves [ <xref ref-type="bibr" rid="B35"> 35 </xref> ] were the currently reported very <span class="trait">high</span> expression levels of miR156d, miR156h, and miR156o in soybean leaves (Table <xref ref-type="table" rid="T3"> 3 </xref> ). This implies crucial roles for all of these miRNAs in leaf development. The <span class="trait">high</span> expression of 11 miRNAs (gma-miR164, miR167, miR168b, miR319a, miR396a, miR482b, miR482b*, miR2118a, miR2118b, miR1508a, and miR1509a) in soybean leaves has been verified by microarray analysis, as were low expression levels of miR169a, miR390c, miR1507c, and miR1510a [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. In this study, the abundance of several miR169 family members (miR169c, miR169p, miR169q, and miR169r) was low yet still detectable in soybean leaves (Table <xref ref-type="table" rid="T3"> 3 </xref> ), which indicates some roles for these miRNAs in leaves, and stands in contrast to the report that gma-miR169 is abundant in soybean shoot apical meristem (SAM) and undetectable in mature soybean leaves [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. The higher expression of miR164 in mature soybean leaves than in SAM has also been reported [ <xref ref-type="bibr" rid="B35"> 35 </xref> ], and consistent with this, the levels of miR164e, f, g, h, and i in leaves were much higher than in roots (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Over-expression of rice miR172 results in loss of spikelet determinacy and flower abnormalities [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. Interestingly, miR172b-3p, miR172h-5p, and miR172k were present at higher levels in leaves than in roots (Table <xref ref-type="table" rid="T3"> 3 </xref> ), conversely, miR172b-3p was still found in significant numbers in roots. Transcripts of miR396k in leaves were higher than that in roots. This miRNA targets the cytokinin (CTK) oxidase/dehydrogenase encoded by <italic> <span class="gene">Glyma12g01390</span> </italic> (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). As CTK regulates meristem activity in leaves [ <xref ref-type="bibr" rid="B38"> 38 </xref> ], miR396k appears to regulate soybean leaf meristem activity. One interesting study that this work brings to mind is the exploration of the roles of the 10 and 12 leaf-specific miRNAs under +Pi and -Pi conditions, respectively (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ) in soybean compound leaf development.
					</p>
				
					<p>&nbsp;&nbsp;
						Several HD-ZIP class III genes such as <italic> PHV </italic> , <italic> REV </italic> , and <italic> AtHB15 </italic> are negatively regulated by miR165/166 [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. Consistently, four HD-ZIP transcription factors were predicted to be cleaved by miR166 family miRNAs (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). NF-YAs, the targets of miR169, participate in N and <span class="trait">drought</span> stress responses [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Soybean 5 NF-YA transcripts are potentially targeted by miR169 members (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). In addition, miR169c and miR169r were down-regulated by -Pi in leaves and roots respectively, and miR169q was up-regulated in leaves by low P (Table <xref ref-type="table" rid="T6"> 6 </xref> ). Hence, miR169s might act as integrators of P and N signaling. Ata-miR169s are down-regulated by low P, consistently,some of which have <span class="gene">PHR1</span> binding sites in the upstream region [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. PHO-like binding sites and/or W-box elements were detected in promoter regions of soybean 169c, 169o, and 169q (Additional file <xref ref-type="supplementary-material" rid="S12"> 12 </xref> ). <italic> BAM3 </italic> ( <italic> At4g20270 </italic> ) encodes a receptor kinase-like <span class="trait">protein</span> that functions in shoot and flower meristem development [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Soybean miR390b, d, e and f all target <italic> <span class="gene">Glyma02g45010</span> </italic> , which shows very <span class="trait">high</span> similarity with <italic> BAM3 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast to miR399 activity in Arabidopsis and rice [ <xref ref-type="bibr" rid="B10"> 10 </xref> ], soybean miR399 targets PHO2 in the 5&#x02019; UTR but not in the coding region (Figure <xref ref-type="fig" rid="F6"> 6 </xref> C). <span class="gene">GmPT5</span> <span class="gene">(Glyma10g04230)</span> was reported to be responsible for P homeostasis in nodule development [ <xref ref-type="bibr" rid="B52"> 52 </xref> ]. RLM-5&#x02019; RACE data verified predictions that miR399a/b/c/d/e cleaves <span class="gene">GmPT5</span> (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> , Figure <xref ref-type="fig" rid="F6"> 6 </xref> D), implying potential regulation of P nutrition in nodules by miR399. Moreover, <italic> <span class="gene">GmIPS1</span> </italic> might act as a RNA mimic to attenuate miR399 activity as <italic> AtIPS1/At4 </italic> does in Arabidopsis [ <xref ref-type="bibr" rid="B26"> 26 </xref> ] (Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> ). Then RNA mimcry might be a useful tool to decipher other miRNA functions in soybean. Among those with unknown functions is soybean miR2111, which attacks a kelch repeat-domain containing F-box <span class="trait">protein</span> gene (Figure <xref ref-type="fig" rid="F6"> 6 </xref> F), as it does in Arabidopsis [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. However, the function of miR2111 and its target in Arabidopsis is still unclear. Exploring the role of miR399 and miR2111 in soybean in the near future at genetic and biochemical levels promises to <span class="trait">yield</span> useful insights into how these miRNAs function in plants, and their effects on associated networks.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AtAGO1 </italic> ( <italic> <span class="gene">At1g48410</span> </italic> ) is cleaved by miR168a and miR168b in Arabidopsis [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. <italic> <span class="gene">Glyma16g34300</span> </italic> and <italic> <span class="gene">Glyma09g29720</span> </italic> , homologues of <italic> AtAGO1 </italic> are also targets of miR168 in soybean (Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> ). Pentatricopeptide repeat (PPR) proteins are predicted to be involved in RNA editing and metabolism in mitochondria and are essential for 5&#x02019;end processing of transcripts [ <xref ref-type="bibr" rid="B54"> 54 </xref> ]. MiR1508d targets one PPR transcript, while miR1508e targets 7 PPR transcripts (Additional file <xref ref-type="supplementary-material" rid="S8"> 8 </xref> ). In Arabidopsis, three PPR genes, <italic> <span class="gene">At1g06580</span> </italic> , <italic> <span class="gene">At1g62720</span> </italic> , and <italic> <span class="gene">At1g62670</span> </italic> , were predicted to be the targets of miR161 [ <xref ref-type="bibr" rid="B55"> 55 </xref> ]. These results indicate that regulation of PPR by miRNAs is conserved across plants, though it may be accomplished by different miRNAs among plant species. In Arabidopsis, SGS3 is involved in trans-acting siRNAs generation, and thus participates in the post-transcriptional gene silencing and natural virus <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]. Interestingly, miR2118a and miR2118b both target an X1-like transcription factor encoded by <italic> <span class="gene">Glyma05g33260</span> </italic> , which is a homologue of <italic> AtSGS3 </italic> . OsBIRH1, a DEAD box RNA helicase, regulates defense responses [ <xref ref-type="bibr" rid="B57"> 57 </xref> ]. Four DEAD helicase were predicted to be the targets of miRnov_6 (Table <xref ref-type="table" rid="T6"> 6 </xref> ). Taken together, these results suggest that RNA metabolism is also tightly regulated by miRNAs in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Accumulation of reactive oxygen species (ROS) is one strategy for plants to cope with early stages of abiotic stress. However, higher ROS levels will damage proteins, nucleic acids, and lipids. P, K and Cu deficiency, as well as, <span class="trait">drought</span> stress boost ROS levels [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. Under <span class="trait">drought</span> conditions, miR408 is strongly induced in photosynthetic tissues in <italic> Medicago </italic> [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. Table <xref ref-type="table" rid="T3"> 3 </xref> demonstrates that miR408c was highly expressed in leaves in +Pi and induced even further in -Pi. These results imply that miR408c might be an integrator of stresses. Superoxide dismutase (SOD), peroxidase (POD), glutathione S-transferase (GST), and cytochrome <span class="gene">P450</span> genes are strongly induced by long-term Pi starvation [ <xref ref-type="bibr" rid="B59"> 59 </xref> ], further supporting the notion that ROS participate in root responses to low P stress. Interestingly, POD, NADP oxidase, and cytochrome <span class="gene">P450</span> genes were targets of miR1507a, miRnov_1, and miRnov_9 (Table <xref ref-type="table" rid="T6"> 6 </xref> ; Additional files <xref ref-type="supplementary-material" rid="S8"> 8 </xref> and <xref ref-type="supplementary-material" rid="S9"> 9 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Glyma17g05970.1</span> </italic> was targeted by miR4376-5p, and miR4376-5p was undetectable in roots (Additional file <xref ref-type="supplementary-material" rid="S8"> 8 </xref> ; Table <xref ref-type="table" rid="T4"> 4 </xref> ). The homologue of <span class="gene">Glyma17g05970.1</span> in Arabidopsis regulates root hair growth, trichome development, and organelle trafficking [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. Loss of function of <italic> AtENO1 </italic> ( <italic> <span class="gene">At1g74030</span> </italic> ), an Arabidopsis phosphoenolpyruvate enolase, results in distorted trichomes and fewer root hairs [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. An enolase, <italic> <span class="gene">Glyma19g37520</span> </italic> was the target of gma-miRnov_7, which is also specifically expressed in leaves. In roots, the abundance of miR4416a is higher than in leaves, and its target is a nodulin-like <span class="trait">protein</span> gene, <italic> <span class="gene">Glyma10g06650</span> </italic> (Additional file <xref ref-type="supplementary-material" rid="S8"> 8 </xref> ). This implies that miR4416a is potentially involved in nodule development in interactions with rhizobial symbionts.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Differential gene expression in soybean leaf tissues at late developmental stages under <span class="trait">drought</span> stress revealed by genome-wide transcriptome analysis.</h2>
			<h3>PMID:23189148  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Le DT(1), Nishiyama R, Watanabe Y, Tanaka M, Seki M, Ham le H, Yamaguchi-Shinozaki K, Shinozaki K, Tran LS.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The availability of complete genome sequence of soybean has allowed research community to design the 66 K Affymetrix Soybean Array GeneChip for genome-wide expression profiling of soybean. In this study, we carried out microarray analysis of leaf tissues of soybean plants, which were subjected to <span class="trait">drought</span> stress from late vegetative V6 and from full bloom reproductive R2 stages. Our data analyses showed that out of 46,093 soybean genes, which were predicted with <span class="trait">high</span> confidence among approximately 66,000 putative genes, 41,059 genes could be assigned with a known function. Using the criteria of a ratio change &gt; = 2 and a q-value&lt;0.05, we identified 1458 and 1818 upregulated and 1582 and 1688 downregulated genes in drought-stressed V6 and R2 leaves, respectively. These datasets were classified into 19 most abundant biological categories with similar proportions. There were only 612 and 463 genes that were overlapped among the upregulated and downregulated genes, respectively, in both stages, suggesting that both conserved and unconserved pathways might be involved in regulation of <span class="trait">drought</span> response in different stages of plant development. A comparative expression analysis using our datasets and that of <span class="trait">drought</span> stressed Arabidopsis leaves revealed the existence of both conserved and species-specific mechanisms that regulate <span class="trait">drought</span> responses. Many upregulated genes encode either regulatory proteins, such as transcription factors, including those with <span class="trait">high</span> homology to Arabidopsis DREB, NAC, AREB and ZAT/STZ transcription factors, kinases and two-component system members, or functional proteins, e.g. late embryogenesis-abundant proteins, glycosyltransferases, glycoside hydrolases, defensins and glyoxalase I family proteins. A detailed analysis of the <span class="gene">GmNAC</span> family and the hormone-related gene category showed that expression of many <span class="gene">GmNAC</span> and hormone-related genes was altered by <span class="trait">drought</span> in V6 and/or R2 leaves. Additionally, the downregulation of many photosynthesis-related genes, which contribute to growth retardation under <span class="trait">drought</span> stress, may serve as an adaptive mechanism for plant survival. This study has identified excellent drought-responsive candidate genes for in-depth characterization and future development of improved drought-tolerant transgenic soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmDREB2A;2, a canonical DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2-type transcription factor in soybean, is posttranslationally regulated and mediates dehydration-responsive element-dependent gene expression.</h2>
			<h3>PMID:23151346  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mizoi J(1), Ohori T, Moriwaki T, Kidokoro S, Todaka D, Maruyama K, Kusakabe K, Osakabe Y, Shinozaki K, Yamaguchi-Shinozaki K.
				<br/> 
				年份：2013 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is an important crop around the world. Abiotic stress conditions, such as <span class="trait">drought</span> and heat, adversely affect its survival, growth, and production. The DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2 (DREB2) group includes transcription factors that contribute to <span class="trait">drought</span> and heat stress tolerance by activating transcription through the cis-element dehydration-responsive element (DRE) in response to these stress stimuli. Two modes of regulation, transcriptional and posttranslational, are important for the activation of gene expression by <span class="gene">DREB2A</span> in Arabidopsis (Arabidopsis thaliana). However, the regulatory system of <span class="gene">DREB2</span> in soybean is not clear. We identified a new soybean <span class="gene">DREB2</span> gene, <span class="gene">GmDREB2A;2,</span> that was highly induced not only by dehydration and heat but also by low temperature. <span class="gene">GmDREB2A;2</span> exhibited a <span class="trait">high</span> transactivation activity via DRE and has a serine/threonine-rich region, which corresponds to a negative regulatory domain of <span class="gene">DREB2A</span> that is involved in its posttranslational regulation, including destabilization. Despite the partial similarity between these sequences, the activity and stability of the <span class="gene">GmDREB2A;2</span> <span class="trait">protein</span> were enhanced by removal of the serine/threonine-rich region in both Arabidopsis and soybean protoplasts, suggestive of a conserved regulatory mechanism that involves the recognition of serine/threonine-rich sequences with a specific pattern. The heterologous expression of <span class="gene">GmDREB2A;2</span> in Arabidopsis induced DRE-regulated stress-inducible genes and improved stress tolerance. However, there were variations in the growth phenotypes of the transgenic Arabidopsis, the induced genes, and their induction ratios between <span class="gene">GmDREB2A;2</span> and DREB2A. Therefore, the basic function and regulatory machinery of <span class="gene">DREB2</span> have been maintained between Arabidopsis and soybean, although differentiation has also occurred.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The response of the different soybean metallothionein isoforms to cadmium intoxication.</h2>
			<h3>PMID:23073037  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pagani MA(1), Tomas M, Carrillo J, Bofill R, Capdevila M, Atrian S, Andreo CS.
				<br/> 
				年份：2012 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cadmium is a highly toxic heavy metal for both plants and animals. The presence of Cd in agricultural soils is of major concern regarding its entry into the food chain, since Cd compounds are readily taken up by plants, and accumulated in edible parts due to their <span class="trait">high</span> solubility. In this study, we first demonstrate the <span class="trait">high</span> capacity for Cd concentration of soybean grains. Consequently, we considered the study and characterization of the molecular determinants of Cd accumulation -such as metallothioneins (MT)- to be of major practical importance. We report here the first characterization of the soybean MT system, with the identification of nine genes (one of which is a truncated pseudogene), belonging to the four plant MT types. The most highly expressed of each type was chosen for further function analysis. All of them are expressed at <span class="trait">high</span> levels in soybean tissues: <span class="gene">GmMT1,</span> <span class="gene">GmMT2</span> and <span class="gene">GmMT3</span> in roots, shoots and seeds, and <span class="gene">GmMT4</span> only in seeds. The corresponding recombinant soybean MTs, synthesized in Escherichia coli cells cultured in metal supplemented media, exhibit greater cadmium than zinc binding capacity. These results suggest a definite role of <span class="gene">GmMTs</span> in Cd(II) accumulation as one of the main responses of soybean to an overload of this metal.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of soybean <span class="gene">GmCBL1</span> enhances abiotic stress tolerance and promotes hypocotyl elongation in Arabidopsis.</h2>
			<h3>PMID:23044418  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li ZY(1), Xu ZS, He GY, Yang GX, Chen M, Li LC, Ma YZ.
				<br/> 
				年份：2012 Nov 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Although extensive studies and remarkable progress have been made with Arabidopsis calcineurin B-like proteins (CBLs), knowledge of their functions in other plant species is still limited. Here we isolated gene <span class="gene">GmCBL1</span> from soybean, a homolog of AtCBL1 in Arabidopsis. <span class="gene">GmCBL1</span> was differentially induced by multiple abiotic stress and plant hormones, and its transcripts were abundant in seedlings and mature roots. We over-expressed <span class="gene">GmCBL1</span> in Arabidopsis and found that it enhanced tolerances to both <span class="trait">high</span> salt and <span class="trait">drought</span> stresses in the transgenic plants. Overexpression of <span class="gene">GmCBL1</span> also promoted hypocotyl elongation under light conditions. <span class="gene">GmCBL1</span> may regulate stress tolerance through activation of stress-related genes, and may control hypocotyl development by altering the expression of gibberellin biosynthesis-related genes. This study identifies a putative soybean CBL gene that functions in both stress tolerance and light-dependent hypocotyl development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional studies of soybean (Glycine max L.) seed LEA proteins <span class="gene">GmPM6,</span> <span class="gene">GmPM11,</span> and <span class="gene">GmPM30</span> by CD and FTIR spectroscopy.</h2>
			<h3>PMID:23017910  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shih MD(1), Hsieh TY, Jian WT, Wu MT, Yang SJ, Hoekstra FA, Hsing YI.
				<br/> 
				年份：2012 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">protein</span> and mRNA levels of late embryogenesis abundant (LEA) genes may be linked to osmotic stresses. Here, we characterized three soybean hydrophilic LEA proteins--GmPM11 (LEA I), <span class="gene">GmPM6</span> (LEA II), and <span class="gene">GmPM30</span> (LEA III)--by circular dichroism and Fourier transform infrared spectroscopy. Structural analysis revealed that the LEA proteins adopted <span class="trait">high</span> amounts of disordered conformations in solution and underwent conformational changes with hydrophobicity and desiccation induction. Macromolecular interaction studies revealed that the <span class="gene">GmPM</span> proteins interact with non-reducing sugars and phospholipids. <span class="gene">GmPM6</span> and <span class="gene">GmPM30</span> but not <span class="gene">GmPM11</span> could prevent beta-aggregation of poly-L-lysine after slow drying. We discuss the possible functions of hydrophilic LEA proteins in maturing seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of gma-MIR394a confers tolerance to <span class="trait">drought</span> in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:23000164  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni Z(1), Hu Z, Jiang Q, Zhang H.
				<br/> 
				年份：2012 Oct 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs, key posttranscriptional regulators of eukaryotic gene expression, play important roles in plant development and response to stress. In this study, a soybean gma-MIR394a gene was functionally characterized, especially with regard to its role in <span class="trait">drought</span> stress <span class="trait">resistance.</span> Expression analysis revealed that gma-MIR394a was expressed differentially in various soybean tissues and was induced by <span class="trait">drought,</span> <span class="trait">high</span> salinity, low temperature stress, and abscisic acid treatment in leaves. One target gene of gma-miR394a, <span class="gene">Glyma08g11030,</span> was predicted and verified using a modified 5' RLM-RACE (RNA ligase-mediated rapid amplification of 5' cDNA ends) assay. Overexpression of gma-MIR394a resulted in plants with lowered leaf water loss and enhanced <span class="trait">drought</span> tolerance. Furthermore, overexpression of gma-MIR394a in Arabidopsis reduced the transcript of an F-box gene <span class="gene">(At1g27340)</span> containing a miR394 complementary target site. These results suggest that the gma-MIR394a gene functions in positive modulation of <span class="trait">drought</span> stress tolerance and has potential applications in molecular breeding to enhance <span class="trait">drought</span> tolerance in crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The heat-induced <span class="trait">protein</span> aggregate correlated with trypsin inhibitor inactivation in soymilk processing.</h2>
			<h3>PMID:22838846  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu Z(1), Chen Y, Zhang C, Kong X, Hua Y.
				<br/> 
				年份：2012 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Kunitz trypsin inhibitor (KTI) and Bowman-Birk inhibitor (BBI) have trypsin inhibitor activities (TIA), which could cause pancreatic disease if at a <span class="trait">high</span> level. It is not clear why some <span class="gene">KTI</span> and BBI lose TIA and some does not in the soymilk processing. This would be examined in this study. TIA assay showed residual TIA was decreased with elevated temperature and TIA was decreased quickly in the beginning and then slowly in boiling water bath. Interestingly, ultracentrifugation showed low residual TIA soymilk had more precipitate than <span class="trait">high</span> residual TIA soymilk and soymilk TIA loss had a <span class="trait">high</span> correlation coefficient (R(2) &gt; 0.9) with precipitate amount. In addition, the TIAs of floating, supernatant, and precipitate obtained by ultracentrifugation were assayed and &gt;80% residual TIA was concentrated in the supernatant. Tricine-SDS-PAGE showed <span class="gene">KTI</span> in supernatant was mainly a noncovalent bound form which might exist as itself and/or incorporated into a small <span class="trait">protein</span> aggregate, while <span class="gene">KTI</span> in precipitate was incorporated into a <span class="trait">protein</span> aggregate by disulfide and/or noncovalent bonds. Chymotrypsin inhibitor activity (CIA) assay showed about 89% of the original CIA remained after 100 °C for 15 min. Ultracentrifugation showed that &gt;90% residual CIA was concentrated in supernatant. Tricine-SDS-PAGE showed soymilk (100 °C, 15 min) BBI mainly existed in supernatant but not in precipitate. It was considered that BBI tended to exist as itself with its natural conformation. Thus, it was suggested residual TIA was mainly from the free BBI and TIA inactivation was mainly from <span class="gene">KTI</span> incorporation into <span class="trait">protein</span> aggregate. This study is meaningful for a new strategy for low TIA soymilk manufacture based on the consideration of promoting <span class="trait">protein</span> aggregate formation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A simple, economical and reproducible <span class="trait">protein</span> extraction protocol for proteomics studies of soybean roots.</h2>
			<h3>PMID:22802720</h3>
				

				
			<p>
				作者：Rodrigues EP(1), Torres AR, da Silva Batista JS, Huergo L, Hungria M.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sample preparation is a critical step in two-dimensional gel electrophoresis (2-DE) of plant tissues. Here we describe a phenol/SDS procedure that, although greatly simplified, produced well-resolved and reproducible 2-DE profiles of <span class="trait">protein</span> extracts from soybean [Glycine max (L.) Merril] roots. Extractions were made in three replicates using both the original and simplified procedure. To evaluate the quality of the extracted proteins, ten spots were randomly selected and identified by mass spectrometry (MS). The 2-DE gels were equally well resolved, with no streaks or smears, and no significant differences were observed in <span class="trait">protein</span> <span class="trait">yield,</span> reproducibility, resolution or number of spots. Mass spectra of the ten selected spots were compared with database entries and allowed high-quality identification of proteins. The simplified protocol described here presents considerable savings of time and reagents without compromising the quality of 2-DE <span class="trait">protein</span> profiles and compatibility with <span class="gene">MS</span> analysis, and may facilitate the progress of proteomics studies of legume-rhizobia interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and expression analysis of the Glycine max CYP707A gene family in response to <span class="trait">drought</span> and salt stresses.</h2>
			<h3>PMID:22751653  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng Y(1), Huang Y, Xian W, Wang J, Liao H.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIMS: Abscisic acid (ABA) plays crucial roles in plants' responses to abiotic stresses. ABA 8'-hydroxylation controlled by CYP707A genes has been well studied in Arabidopsis and rice, but not in legumes. The aims of the present study were to identify and functionally analyse the soybean CYP707A gene family, and to explore their expression patterns under dehydration and salt stresses. METHODS: A complementation experiment was employed to verify the function of soybean CYP707A1a in ABA catabolism. Genomic and cDNA sequences of other soybean CYP707A genes were isolated from the Phytozome database based on soybean CYP707A1a. The structure and phylogenetic relationship of this gene family was further analysed. The expression patterns of soybean CYP707A genes under dehydration and salt stress were analysed via quantitative real-time PCR. KEY RESULTS: Over-expression of <span class="gene">GmCYP707A1a</span> in the atcyp707a2 T-DNA insertion mutant decreased its sensitivity to ABA, indicating that <span class="gene">GmCYP707A1a</span> indeed functions as an ABA 8'-hydroxylase in higher plants. The soybean genome contains ten CYP707A genes. Gene structure and phylogenetic analysis showed <span class="trait">high</span> conservation of ten <span class="gene">GmCYP707A</span> genes to the other CYP707A genes from monocots and dicots. Seed imbibition induced expression of A1a, A1b, A2a, A2b, A2c, A3a and A5 in embryo, and expression of A1a, A1b, A2a and A2b in cotyledon. Dehydration induced expression of A1a, A1b, A2b, A2c, A3a, A3b, A4a, A4b and A5 both in roots and in leaves, whereas rehydration stimulated transcription of A2a, A2b, A3b, A4a and A5 in roots, and only A3b and A5 in leaves. Expression of all soybean CYP707A genes was induced either by short- or by long-term salt stress. CONCLUSIONS: The first biological evidence is provided that <span class="gene">GmCYP7071a</span> encodes an ABA 8'-hydroxylase through transgenic studies. Ten soybean <span class="gene">GmCYP707A</span> genes were identified, most of them expressed in multiple soybean tissues, and were induced by imbibition, dehydration and salinity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rice heterotrimeric G-protein gamma subunits (RGG1 and RGG2) are differentially regulated under abiotic stress.</h2>
			<h3>PMID:22751322  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yadav DK(1), Islam SM, Tuteja N.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heterotrimeric G-proteins (α, β and γ subunits) are primarily involved in diverse signaling processes by transducing signals from an activated transmembrane G-protein coupled receptor (GPCR) to appropriate downstream effectors within cells. The role of α and β G-protein subunits in salinity and heat stress has been reported but the regulation of γ subunit of plant G-proteins in response to abiotic stress has not heretofore been described. In the present study we report the isolation of full-length cDNAs of two isoforms of Gγ [RGG1(I), 282 bp and RGG2(I), 453 bp] from rice (Oryza sativa cv Indica group Swarna) and described their transcript regulation in response to abiotic stresses. <span class="trait">Protein</span> sequence alignment and pairwise comparison of γ subunits of Indica rice [RGG(I)] with other known plant G-protein γ subunits demonstrated <span class="trait">high</span> homology to barley (HvGs) while soybean <span class="gene">(GmG2)</span> and Arabidopsis (AGG1) were least related. The numbers of the exons and introns were found to be similar between RGG1(I) and RGG2(I), but their sizes were different. Analyses of promoter sequences of RGG(I) confirmed the presence of stress-related cis-regulatory signature motifs suggesting their active and possible independent roles in abiotic stress signaling. The transcript levels of RGG1(I) and RGG2(I) were upregulated following NaCl, cold, heat and ABA treatments. However, in <span class="trait">drought</span> stress only RGG1(I) was upregulated. Strong support by transcript profiling suggests that γ subunits play a critical role via cross talk in signaling pathways. These findings provide first direct evidence for roles of Gγ subunits of rice G-proteins in regulation of abiotic stresses. These findings suggest the possible exploitation of γ subunits of G-protein machinery for promoting stress tolerance in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biological evaluation of mechlorethamine-Pt(II) complex, part II: antimicrobial screening and lox study of the complex and its ligand.</h2>
			<h3>PMID:22741794  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Radojević ID(1), Petrović ZD, Čomić LR, Simijonović D, Petrović VP, Hadjipavlou-Litina D.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The reaction of K(2)PtCl(4) with anticancer-alkylating agent mechlorethamine hydrochloride (CH(3)NH(C(2)H(4)Cl)(2) x HCl = HN2×HCl), in the molar ratio 1 : 2, affords the complex [H2N2](2)[PtCl(4)]. In vitro antimicrobial and lipoxygenase inhibitory activities of the complex and its precursor were evaluated. Antimicrobial activity of the HN2×HCl and [H2N2](2)[PtCl(4)] complex was investigated against 29 species of microorganisms. Testing is performed by microdilution method. Minimum inhibitory concentration (MIC) and minimum microbicidal concentration (MMC) have been determined. The difference between antimicrobial activity of precursor and corresponding platinum(II) complex is noticed and the activity of the precursor was higher. Tested compounds demonstrated the <span class="trait">high</span> and significant antifungal activity and low to moderate antibacterial activity. It was shown that the gram-positive bacteria were more sensitive than the gram-negative. UV absorbance-based enzyme assays were performed with HN2×HCl and [H2N2](2)[PtCl(4)] complex, in order to evaluate their in vitro inhibitory activity of soybean lipoxygenase (LOX), also. Assay with <span class="gene">LOX</span> showed significantly greater inhibitory activity of the complex, than the precursor.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cloning and functional analysis of <span class="gene">SCTF-1</span> encoding a C2H2-type Zinc finger <span class="trait">protein</span> from soybean].</h2>
			<h3>PMID:22698747  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song B(1), Wang PW, Fu YP, Fan XH, Xia HF, Gao W, Hong Y, Wang H, Zhang Z, Ma J.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The zinc finger <span class="trait">protein</span> is one of the proteins with finger-like domain. Some of them are transcription factors which play important role in plant growth and plant <span class="trait">resistance</span> to abiotic stresses. In this paper, a novel C2H2-type zinc finger <span class="trait">protein</span> gene <span class="gene">SCTF-1</span> (GenBank accession number JQ692081) was isolated from soybean (Glycine max (L.) Merr.) This gene has a 699 bp ORF (open reading frame) with no intron and encodes a 24.9 kDa <span class="trait">protein</span> with 233 amino acids. Its isoelectric point (pI) is 8.33. The <span class="gene">SCTF-1</span> <span class="trait">protein</span> contains two typical C2H2-type zinc finger domains. Both of them have highly conserved amino acid sequence-QALGGH which is a particular characteristic of plant. Transient expression of the GFP-SCTF-1 <span class="trait">protein</span> in onion epidermal cell showed that <span class="gene">SCTF-1</span> was localized in cell nuclei. RT-PCR results showed that <span class="gene">SCTF-1</span> gene was expressed with <span class="trait">high</span> levels in flowers and leaves in soybean, but low in roots and stems. The expression of <span class="gene">SCTF-1</span> gene was strongly induced by low temperature in the soybean seedlings. Overexpression of <span class="gene">SCTF-1</span> enhanced cold tolerance of transgenic tobacco (Nicotiana tabacum L.) compared to the control.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Positional cloning and characterization reveal the molecular basis for soybean maturity locus <span class="gene">E1</span> that regulates photoperiodic flowering.</h2>
			<h3>PMID:22619331  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xia Z(1), Watanabe S, Yamada T, Tsubokura Y, Nakashima H, Zhai H, Anai T, Sato S, Yamazaki T, Lü S, Wu H, Tabata S, Harada K.
				<br/> 
				年份：2012 Aug 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complex and coordinated regulation of flowering has <span class="trait">high</span> ecological and agricultural significance. The maturity locus <span class="gene">E1</span> has a large impact on flowering time in soybean, but the molecular basis for the <span class="gene">E1</span> locus is largely unknown. Through positional cloning, we delimited the <span class="gene">E1</span> locus to a 17.4-kb region containing an intron-free gene (E1). The <span class="gene">E1</span> <span class="trait">protein</span> contains a putative bipartite nuclear localization signal and a region distantly related to B3 domain. In the recessive allele, a nonsynonymous substitution occurred in the putative nuclear localization signal, leading to the loss of localization specificity of the <span class="gene">E1</span> <span class="trait">protein</span> and earlier flowering. The early-flowering phenotype was consistently observed in three ethylmethanesulfonate-induced mutants and two natural mutations that harbored a premature stop codon or a deletion of the entire <span class="gene">E1</span> gene. <span class="gene">E1</span> expression was significantly suppressed under short-day conditions and showed a bimodal diurnal pattern under long-day conditions, suggesting its response to photoperiod and its dominant effect induced by long day length. When a functional <span class="gene">E1</span> gene was transformed into the early-flowering cultivar Kariyutaka with low <span class="gene">E1</span> expression, transgenic plants carrying exogenous <span class="gene">E1</span> displayed late flowering. Furthermore, the transcript abundance of <span class="gene">E1</span> was negatively correlated with that of <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a,</span> homologues of FLOWERING LOCUS T that promote flowering. These findings demonstrated the key role of <span class="gene">E1</span> in repressing flowering and delaying maturity in soybean. The molecular identification of the maturity locus <span class="gene">E1</span> will contribute to our understanding of the molecular mechanisms by which a short-day plant regulates flowering time and maturity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative proteome analysis of <span class="trait">high</span> and low cadmium accumulating soybeans under cadmium stress.</h2>
			<h3>PMID:22588482  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hossain Z(1), Hajika M, Komatsu S.
				<br/> 
				年份：2012 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A comparative proteomic study was performed to unravel the <span class="trait">protein</span> networks involved in cadmium stress response in soybean. Ten-day-old seedlings of contrasting cadmium accumulating soybean cultivars-Harosoy <span class="trait">(high</span> cadmium accumulator), Fukuyutaka (low cadmium accumulator), and their recombinant inbred line CDH-80 <span class="trait">(high</span> cadmium accumulator) were exposed to 100 μM CdCl(2) treatment for 3 days. Root growth was found to be affected under cadmium stress in all. Varietal differences at root <span class="trait">protein</span> level were evaluated. NADP-dependent alkenal double bond reductase <span class="gene">P1</span> was found to be more abundant in low cadmium accumulating Fukuyutaka. Leaf proteome analysis revealed that differentially expressed proteins were primarily involved in metabolism and energy production. The results indicate that both <span class="trait">high</span> and low cadmium accumulating cultivars and CDH-80 share some common defense strategies to cope with the cadmium stress. <span class="trait">High</span> abundance of enzymes involved in glycolysis and TCA cycle might help cadmium challenged cells to produce more energy necessary to meet the <span class="trait">high</span> energy demand. Moreover, enhanced expressions of photosynthesis related proteins indicate quick utilization of photoassimilates in energy generation. Increased abundance of glutamine synthetase in all might be involved in phytochelatin mediated detoxification of cadmium ions. In addition, increased abundance of antioxidant enzymes, namely superoxide dismutase, ascorbate peroxidase, catalase, ensures cellular protection from reactive oxygen species mediated damages under cadmium stress. Enhanced expression of molecular chaperones in <span class="trait">high</span> cadmium accumulating cultivar might be another additional defense mechanism for refolding of misfolded proteins and to stabilize <span class="trait">protein</span> structure and function, thus maintain cellular homeostasis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsAPK, an ABA-activated and calcium-independent SnRK2-type kinase from G. soja, mediates the regulation of plant tolerance to salinity and ABA stress.</h2>
			<h3>PMID:22439004  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang L(1), Ji W, Gao P, Li Y, Cai H, Bai X, Chen Q, Zhu Y.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant <span class="gene">Snf1</span> (sucrose non-fermenting-1) related <span class="trait">protein</span> kinase (SnRK), a subfamily of serine/threonine kinases, has been implicated as a crucial upstream regulator of ABA and osmotic signaling as in many other signaling cascades. In this paper, we have isolated a novel plant specific ABA activated calcium independent <span class="trait">protein</span> kinase (GsAPK) from a highly salt tolerant plant, Glycine soja (50109), which is a member of the SnRK2 family. Subcellular localization studies using GFP fusion <span class="trait">protein</span> indicated that GsAPK is localized in the plasma membrane. We found that autophosphorylation and Myelin Basis <span class="trait">Protein</span> phosphorylation activity of GsAPK is only activated by ABA and the kinase activity also was observed when calcium was replaced by EGTA, suggesting its independence of calcium in enzyme activity. We also found that cold, salinity, <span class="trait">drought,</span> and ABA stress alter GsAPK gene transcripts and heterogonous overexpression of GsAPK in Arabidopsis alters plant tolerance to <span class="trait">high</span> salinity and ABA stress. In summary, we demonstrated that GsAPK is a Glycine soja ABA activated calcium independent SnRK-type kinase presumably involved in ABA mediated stress signal transduction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of nitrous oxide reductase from Pseudomonas stutzeri in transgenic tobacco roots using the root-specific rolD promoter from Agrobacterium rhizogenes.</h2>
			<h3>PMID:22423324</h3>
				

				
			<p>
				作者：Wan S(1), Johnson AM, Altosaar I.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nitrous oxide (N(2)O) reduction pathway from a soil bacterium, Pseudomonas stutzeri, was engineered in plants to reduce N(2)O emissions. As a proof of principle, transgenic plants expressing nitrous oxide reductase (N(2)OR) from P. stutzeri, encoded by the <span class="gene">nosZ</span> gene, and other transgenic plants expressing N(2)OR along with the more complete operon from P. stutzeri, encoded by nosFLZDY, were generated. Gene constructs were engineered under the control of a root-specific promoter and with a secretion signal peptide. Expression and rhizosecretion of the transgene <span class="trait">protein</span> were achieved, and N(2)OR from transgenic Nicotiana tabacum proved functional using the methyl viologen assay. Transgenic plant line 1.10 showed the highest specific activity of 16.7 µmol N(2)O reduced min(-1) g(-1) root <span class="trait">protein.</span> Another event, plant line 1.9, also demonstrated <span class="trait">high</span> specific activity of N(2)OR, 13.2 µmol N(2)O reduced min(-1) g(-1) root <span class="trait">protein.</span> The availability now of these transgenic seed stocks may enable canopy studies in field test plots to monitor whole rhizosphere N flux. By incorporating one bacterial gene into genetically modified organism <span class="gene">(GMO)</span> crops (e.g., cotton, corn, and soybean) in this way, it may be possible to reduce the atmospheric concentration of N(2)O that has continued to increase linearly (about 0.26% year(-1)) over the past half-century.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and evaluation of ω-3 fatty acid desaturase genes for hyperfortifying α-linolenic acid in transgenic rice seed.</h2>
			<h3>PMID:22378946  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu HL(1), Yin ZJ, Xiao L, Xu YN, Qu le Q.
				<br/> 
				年份：2012 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						α-Linolenic acid (ALA) deficiency and a skewed of ω6:ω3 fatty acid ratio in the diet are a major explanation for the prevalence of cardiovascular diseases and inflammatory/autoimmune diseases. There is a need to enhance the ALA content and to reduce the ratio of linoleic acid (LA) to ALA. Six ω-3 (Δ-15) fatty acid desaturase (FAD) genes were cloned from rice and soybean. The subcellular localizations of the proteins were identified. The FAD genes were introduced into rice under the control of an endosperm-specific promoter, GluC, or a Ubi-1 promoter to evaluate their potential in increasing the ALA content in seeds. The ALA contents in the seeds of endoplasmic reticulum (ER)-localized <span class="gene">GmFAD3-1</span> and OsFAD3 overexpression lines increased from 0.36 mg g⁻¹ to 8.57 mg g⁻¹ and 10.06 mg g⁻¹, respectively, which was 23.8- and 27.9-fold higher than that of non-transformants. The trait of <span class="trait">high</span> ALA content was stably inheritable over three generations. Homologous OsFAD3 is more active than <span class="gene">GmFAD3-1</span> in catalysing LA conversion to ALA in rice seeds. Overexpression of ER-localized <span class="gene">GmFAD3-2/3</span> and chloroplast-localized OsFAD7/8 had less effect on increasing the ALA content in rice seeds. The GluC promoter is advantageous compared with Ubi-1 in this experimental system. The enhanced ALA was preferentially located at the sn-2 position in triacylglycerols. A meal-size portion of <span class="trait">high</span> ALA rice would meet &gt;80% of the daily adult ALA requirement. The ALA-rich rice could be expected to ameliorate much of the global dietary ALA deficiency.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Comparative study of the consumption of virgin olive oil or seje on lipid profile and oxidation <span class="trait">resistance</span> of <span class="trait">high</span> density lipoprotein (HDL) of rat plasma].</h2>
			<h3>PMID:22308940  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Isabel Giacopini M(1), Guerrero O, Moya M, Bosch V.
				<br/> 
				年份：2011 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We compared the effect of the consumption of seje oil (Oenocarpus bataua), with that of olive oil, on plasma lipids and susceptibility in vitro to oxidation of <span class="trait">high</span> density lipoprotein (HDL) in the rat. Two groups often male Sprague Dawley rats were fed ad libitum, for a lapse of eight week, with a purified diets with 10g de seje oil or olive oil/100 g of diet (GS y GO respectively). The animals were exsanguinated at the end of the experimental after a 14 hour fast. Plasma was isolated by centrifugation, and the fractions of lipoproteins were separated from the plasma by sequential ultracentrifugation. Rats of GO had a statistically significant lower in concentration of TG (p &lt; 0.05) compared with <span class="gene">GS</span> group. HDL fractions in both groups were oxidatively modified by incubation with copper ions. Differences in the fractions susceptibilities to peroxidation were studied by measuring the formation of thiobarbituric acid reactive substance (TBARS) for 3 hours. HDL in <span class="gene">GS</span> had a statistically significant decrease in TBARS formation (p &lt; 0.05) relative to HDL of GO. This may be explained by the lower concentration of polyunsaturated fatty acids of HDL in <span class="gene">GS</span> compared with HDL in GO.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative proteomics analysis reveals the mechanism of pre-harvest seed deterioration of soybean under <span class="trait">high</span> temperature and humidity stress.</h2>
			<h3>PMID:22270011  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang L(1), Ma H, Song L, Shu Y, Gu W.
				<br/> 
				年份：2012 Apr 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> temperature and humidity (HTH) stress during soybean seed development and maturity in the field easily leads seed to pre-harvest deterioration. However, how proteins and their involved pathways in developing soybean seed systematically cause deterioration is still not largely understood. To reveal it, we compared the proteome composition of developing seed (R(7) period) of a pre-harvest seed deterioration sensitive soybean cultivar at different HTH stress time points (24, 96 and 168 h) with their corresponding controls by 2-DE. 42 <span class="trait">protein</span> spots were found to be differentially expressed and successfully identified by MALDI-TOF <span class="gene">MS</span> to match 31 diverse <span class="trait">protein</span> species. These proteins were involved in 13 cellular responses and metabolic processes including carbohydrate metabolism, signal transduction, <span class="trait">protein</span> biosynthesis, photosynthesis, <span class="trait">protein</span> folding and assembly, energy pathway, cell rescue and defense, cell cycle, nitrogen metabolism, lipid metabolism, amino acid metabolism, transcription regulation, and secondary metabolite biosynthesis. Based on these proteins' functions and involved pathways, together with ultrastructural, physical and chemical, and metabolomic data, a pre-harvest seed deterioration mechanism was proposed. Such a mechanism allows us to further understand the possible management strategy of cellular activities occurring in the HTH-stressed developing seeds and provides new insights into the HTH stress responses in crop developing seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pathobiochemical effect of acylated steryl-β-glucoside on aggregation and cytotoxicity of α-synuclein.</h2>
			<h3>PMID:22124781  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Usuki S(1), Kamitani T, Matsuo Y, Yu RK.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cycad seed consumption by the native islanders of Guam is frequently associated with <span class="trait">high</span> rates of amyotrophic lateral sclerosis-parkinsonism dementia complex (ALS/PDC); furthermore, accompanying pathological examination often exhibits α-synuclein inclusions in the neurons of the affected brain. Acylated steryl-β-glucoside (ASG) contained in cycad seeds is considered as causative environmental risk factor. We aimed to investigate whether ASG influences aggregation and cell toxicity of α-synuclein. To understand whether ASG is a causative factor in the development of ALS/PDC, soybean-derived ASG was tested for its effect on in vitro aggregation of α-synuclein using Thioflavin-T. ASG was also tested to determine whether it modulates α-synuclein cytotoxicity in yeast cells. In addition, we determined whether an interaction between ASG and α-synuclein occurs in the plasma membrane or cytoplasm using three factors: <span class="gene">GM1</span> ganglioside, small unilamellar vesicles, and ATP. In the present study, we found that ASG-mediated acceleration of α-synuclein aggregation is influenced by the presence of ATP, but not by the presence of <span class="gene">GM1.</span> ASG accelerated the α-synuclein aggregation in the cytoplasm. ASG also enhanced α-synuclein-induced cytotoxicity in yeast cells. This study demonstrated that ASG directly enhances aggregation and cytotoxicity of α-synuclein, which are often observed in patients with ALS/PDC. These results, using assays that replicate cytoplasmic conditions, are consistent with the molecular mechanism that cytotoxicity is caused by intracellular α-synuclein fibril formation in neuronal cells.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Vernonia DGATs can complement the disrupted oil and <span class="trait">protein</span> metabolism in epoxygenase-expressing soybean seeds.</h2>
			<h3>PMID:22107928  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li R(1), Yu K, Wu Y, Tateno M, Hatanaka T, Hildebrand DF.
				<br/> 
				年份：2012 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant oils can be useful chemical feedstocks such as a source of epoxy fatty acids. <span class="trait">High</span> seed-specific expression of a Stokesia laevis epoxygenase (SlEPX) in soybeans only results in 3-7% epoxide levels. SlEPX-transgenic soybean seeds also exhibited other phenotypic alterations, such as altered seed fatty acid profiles, reduced oil accumulation, and variable <span class="trait">protein</span> levels. SlEPX-transgenic seeds showed a 2-5% reduction in total oil content and <span class="trait">protein</span> levels of 30.9-51.4%. To address these pleiotrophic effects of SlEPX expression on other traits, transgenic soybeans were developed to co-express SlEPX and <span class="gene">DGAT</span> (diacylglycerol acyltransferase) genes (VgDGAT1 & 2) isolated from Vernonia galamensis, a <span class="trait">high</span> accumulator of epoxy fatty acids. These side effects of SlEPX expression were largely overcome in the <span class="gene">DGAT</span> co-expressing soybeans. Total oil and <span class="trait">protein</span> contents were restored to the levels in non-transgenic soybeans, indicating that both VgDGAT1 and VgDGAT2 could complement the disrupted phenotypes caused by over-expression of an epoxygenase in soybean seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An engineered pathway for glyoxylate metabolism in tobacco plants aimed to avoid the release of ammonia in photorespiration</h2>
			<h3>PMID:22104170</h3>
				

				
					<h3>PMCID:3252329</h3>
					<h4>期刊:BMC Biotechnology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Carvalho </surname>  <given-names> Josirley de FC </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> josie.carvalho@hotmail.co.uk </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Madgwick </surname>  <given-names> Pippa J </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> pippa.madgwick@rothamsted.ac.uk </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Powers </surname>  <given-names> Stephen J </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> stephen.powers@rothamsted.ac.uk </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Keys </surname>  <given-names> Alfred J </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> alfred.keys@rothamsted.ac.uk </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Lea </surname>  <given-names> Peter J </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> p.lea@lancaster.ac.uk </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Parry </surname>  <given-names> Martin AJ </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> martin.parry@rothamsted.ac.uk </email>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The photorespiratory nitrogen cycle in C₃ plants involves an extensive diversion of carbon and nitrogen away from the direct pathways of assimilation. The liberated ammonia is re-assimilated, but up to 25% of the carbon may be released into the atmosphere as CO₂. Because of the loss of CO₂ and <span class="trait">high</span> energy costs, there has been considerable interest in attempts to decrease the flux through the cycle in C₃ plants. Transgenic tobacco plants were generated that contained the genes <span class="gene">gcl</span> and <span class="gene">hyi</span> from E. coli encoding glyoxylate carboligase (EC 4.1.1.47) and hydroxypyruvate isomerase (EC 5.3.1.22) respectively, targeted to the peroxisomes. It was presumed that the two enzymes could work together and compete with the aminotransferases that convert glyoxylate to glycine, thus avoiding ammonia production in the photorespiratory nitrogen cycle. RESULTS: When grown in ambient air, but not in elevated CO₂, the transgenic tobacco lines had a distinctive phenotype of necrotic lesions on the leaves. Three of the six lines chosen for a detailed study contained single copies of the <span class="gene">gcl</span> gene, two contained single copies of both the <span class="gene">gcl</span> and <span class="gene">hyi</span> genes and one line contained multiple copies of both <span class="gene">gcl</span> and <span class="gene">hyi</span> genes. The <span class="gene">gcl</span> <span class="trait">protein</span> was detected in the five transgenic lines containing single copies of the <span class="gene">gcl</span> gene but <span class="gene">hyi</span> <span class="trait">protein</span> was not detected in any of the transgenic lines. The content of soluble amino acids including glycine and serine, was generally increased in the transgenic lines growing in air, when compared to the wild type. The content of soluble sugars, glucose, fructose and sucrose in the shoot was decreased in transgenic lines growing in air, consistent with decreased carbon assimilation. CONCLUSIONS: Tobacco plants have been generated that produce bacterial glyoxylate carboligase but not hydroxypyruvate isomerase. The transgenic plants exhibit a stress response when exposed to air, suggesting that some glyoxylate is diverted away from conversion to glycine in a deleterious short-circuit of the photorespiratory nitrogen cycle. This diversion in metabolism gave rise to increased concentrations of amino acids, in particular glutamine and asparagine in the leaves and a decrease of soluble sugars.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The photorespiratory nitrogen cycle in C <sub> 3 </sub> plants involves an extensive diversion of carbon and nitrogen away from the direct pathways of assimilation. The liberated ammonia is re-assimilated, but up to 25% of the carbon may be released into the atmosphere as CO <sub> 2 </sub> . Because of the loss of CO <sub> 2 </sub> and <span class="trait">high</span> energy costs, there has been considerable interest in attempts to decrease the flux through the cycle in C <sub> 3 </sub> plants. Transgenic tobacco plants were generated that contained the genes <italic> <span class="gene">gcl</span> </italic> and <italic> <span class="gene">hyi</span> </italic> from <italic> E. coli </italic> encoding glyoxylate carboligase (EC 4.1.1.47) and hydroxypyruvate isomerase (EC 5.3.1.22) respectively, targeted to the peroxisomes. It was presumed that the two enzymes could work together and compete with the aminotransferases that convert glyoxylate to glycine, thus avoiding ammonia production in the photorespiratory nitrogen cycle.
					</p>
				
					<p>&nbsp;&nbsp;
						When grown in ambient air, but not in elevated CO <sub> 2 </sub> , the transgenic tobacco lines had a distinctive phenotype of necrotic lesions on the leaves. Three of the six lines chosen for a detailed study contained single copies of the <italic> <span class="gene">gcl</span> </italic> gene, two contained single copies of both the <italic> <span class="gene">gcl</span> </italic> and <italic> <span class="gene">hyi</span> </italic> genes and one line contained multiple copies of both <italic> <span class="gene">gcl</span> </italic> and <italic> <span class="gene">hyi</span> </italic> genes. The <span class="gene">gcl</span> <span class="trait">protein</span> was detected in the five transgenic lines containing single copies of the <italic> <span class="gene">gcl</span> </italic> gene but <span class="gene">hyi</span> <span class="trait">protein</span> was not detected in any of the transgenic lines. The content of soluble amino acids including glycine and serine, was generally increased in the transgenic lines growing in air, when compared to the wild type. The content of soluble sugars, glucose, fructose and sucrose in the shoot was decreased in transgenic lines growing in air, consistent with decreased carbon assimilation.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Schematic representations of the <italic> <span class="gene">gcl</span> </italic> and <italic> gcl-hyi </italic> cassettes for transformation of tobacco </bold> . <bold> (A) </bold> .Cassette pBIN19 <italic> <span class="gene">gcl</span> </italic> . LB: left border; RB: right border; CaMV 35S: CaMV promoter; <italic> nos </italic> : nopaline synthase terminator; <italic> ocs </italic> octopine synthase terminator; <italic> <span class="gene">gcl</span> </italic> : gene encoding <italic> E. coli </italic> glyoxylate carboligase; <italic> Pnos: </italic> nopaline synthase promoter; <italic> NPTII </italic> : neophosphotransferase gene for kanamycin <span class="trait">resistance.</span> The arrowheads indicate the orientation of the promoter and terminator sequences. <bold> (B) </bold> . Cassette pBIN19 <italic> gcl-hyi </italic> . LB: left border; RB: right border; CaMV 35S: CaMV promoter; <italic> nos </italic> : nopaline synthase terminator; <italic> ocs </italic> octopine synthase terminator; <italic> <span class="gene">gcl</span> </italic> : gene encoding <italic> E. coli </italic> glyoxylate carboligase; <italic> <span class="gene">hyi</span> </italic> gene encoding <italic> E.coli </italic> hydroxypyruvate isomerase; <italic> Pnos: </italic> nopaline synthase promoter; <italic> NPTII </italic> : neophosphotransferase gene for kanamycin <span class="trait">resistance.</span> The arrowheads indicate the orientation of the promoter and terminator sequences.
					</p>
				
					<p>&nbsp;&nbsp;
						The phenotype, common to all transgenic individuals containing either <italic> <span class="gene">gcl</span> </italic> alone or together with <italic> <span class="gene">hyi</span> </italic> , was characterised by the presence of <span class="trait">chlorosis</span> of the leaves following growth in ambient air. Figure <xref ref-type="fig" rid="F3"> 3A </xref> shows the characteristic lesions on leaves of young T <sub> 2 </sub> transgenic plants compared to the wild type (wt), after the plants had been grown in <span class="trait">high</span> CO <sub> 2 </sub> and then transferred to ambient air for 12 days. The phenotype was exhibited far less strongly in lines 32 and 79. Figure <xref ref-type="fig" rid="F3"> 3B </xref> shows leaves from T <sub> 1 </sub> plants of line 37 grown at <span class="trait">high</span> CO <sub> 2 </sub> and then exposed to ambient air. The mature leaf on the left of Figure <xref ref-type="fig" rid="F3"> 3B </xref> shows chlorotic patches near to the lateral veins and characteristic necrosis along the upper surface of the mid-rib. Leaves from young seedlings shown on the right of Figure <xref ref-type="fig" rid="F3"> 3B </xref> were from plants grown in elevated CO <sub> 2 </sub> and exposed to ambient air for 7 days. Chlorotic areas were produced close to the veins with a characteristic deformation of the leaf tip and cessation of growth. Downward curvature of the leaf margins was also a characteristic of this phenotype. The lines studied all showed similar changes on exposure to air containing ambient concentrations of CO <sub> 2 </sub> although to varying extents. Also, the phenotypes described above showed more quickly at higher light intensities.
					</p>
				
					<p>&nbsp;&nbsp;
						Western blotting (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ) using antisera raised against <span class="gene">gcl</span> expressed in <italic> E. coli </italic> , showed that there was clear expression of a <span class="trait">protein</span> of approximate molecular mass of 66 kDa in lines 33, 37, 84 and 92, weak expression in line 32 and none in the wt and 79. Using antisera raised against a KLH peptide (see Materials and Methods), no evidence of expression of a <span class="gene">hyi</span> <span class="trait">protein</span> in any of the transgenic plants was obtained by western blotting, although a 29 kDa <span class="trait">protein</span> was detected in extracts of <italic> E.coli </italic> cells expressing <span class="gene">hyi</span> (result not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Western blot showing expression of the <italic> E.coli </italic> <span class="gene">gcl</span> <span class="trait">protein</span> in T <sub> 2 </sub> transgenic lines </bold> . The position and sizes in kDa of the pre-stained <span class="trait">protein</span> standard is indicated as M. <italic> E.co </italic> shows the position of <span class="gene">gcl</span> expressed in <italic> E. coli </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Autoradiograms of thin layer cellulose chromatograms showing the products of metabolism of [ <sup> 14 </sup> C]glycollate in leaf strips from wild type and T <sub> 2 </sub> transgenic plants </bold> . The plants were grown in <span class="trait">high</span> CO <sub> 2 </sub> for 35 days and were then transferred to ambient air for 3 days. Radioactivity was found in glycollate (1), ala (2), glycerate (3), gly (4), ser (5), sucrose (6), malate (7), <span class="gene">glu</span> (8) and asp (9) in the wild-type. There was a radioactive impurity (10) present in the sample of [ <sup> 14 </sup> C]glycolate, probably monochloroacetate, that appeared in all of the chromatograms. Chromatograms were developed using n-propanol/0.880 sg ammonia solution/H <sub> 2 </sub> O (6:3:1) in the first dimension (vertical in the Figure) and twice with n-propylacetate/90% formic acid/H <sub> 2 </sub> O (11:5:3) in the second dimension.
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic lines exhibited chlorotic lesions close to the veins when exposed to ambient air. In four of the six transgenic lines chosen for further study (lines 33 and 37 transformed with the <italic> <span class="gene">gcl</span> </italic> gene only, and 84 and 92 transformed with the combined <italic> gcl-hyi </italic> gene), the lesions were severe (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). The chlorotic phenotype was less visible in the leaves of line 32 and hardly at all in 79. Transgenic line 32 was not included in the final stages of the investigation as the <span class="gene">gcl</span> <span class="trait">protein</span> was only weakly expressed, possibly because the plants may not all have been homozygous. All transgenic lines appeared to grow normally in elevated CO <sub> 2 </sub> , except that germination of the seed was slower. The data indicated that the <italic> E.coli </italic> <italic> <span class="gene">gcl</span> </italic> gene had been transferred as one copy into lines 33, 37, 84 and 92 (Figure <xref ref-type="fig" rid="F4"> 4A </xref> ) and that the gene could be transcribed and translated to form a polypeptide of the correct molecular mass (Figures <xref ref-type="fig" rid="F5"> 5 </xref> and <xref ref-type="fig" rid="F6"> 6 </xref> ). Similarly, the data indicated that lines 84 and 92 had been transformed with one copy of the <italic> E.coli <span class="gene">hyi</span> </italic> gene and that the gene was transcribed to form mRNA. However, antibody raised against a short hydrophilic sequence of the <span class="gene">hyi</span> <span class="trait">protein</span> that recognised the native <italic> E. coli </italic> <span class="trait">protein,</span> did not cross react with any <span class="trait">protein</span> in extracts of the transgenic leaf. This would suggest that either the mRNA was not translated or that the <span class="gene">hyi</span> polypeptide was immediately subjected to proteolysis following synthesis. The Southern blots in Figure <xref ref-type="fig" rid="F4"> 4A </xref> and <xref ref-type="fig" rid="F4"> 4B </xref> indicate that four copies of the combined <italic> E.coli </italic> <italic> gcl-hyi </italic> gene had been transferred into line 79; however, there was no evidence of transcription of any mRNA or translation of any <span class="trait">protein.</span> Co-suppression and gene silencing phenomena of multiple genes that have been transferred into plants are now well established [ <xref ref-type="bibr" rid="B38"> 38 </xref> ] and this was presumably what had occurred here.
					</p>
				
					<p>&nbsp;&nbsp;
						The data in Figure <xref ref-type="fig" rid="F8"> 8 </xref> show that there is an increase in the majority of soluble amino acids, in particular the amides asparagine and glutamine, in the leaves, especially of lines 33, 37, 84 and 92, when compared to the wt, following exposure to air. However, in the leaves of line 79, where there is evidence of co-suppression, there is no major accumulation. Thus there is a correlation between expression of the <span class="gene">gcl</span> <span class="trait">protein,</span> leaf necrosis and amino acid accumulation. The question is whether the accumulation of amino acids and amides is a consequence of the <span class="trait">chlorosis,</span> or the cause. During the normal senescence of leaves of both tobacco and <italic> A. thaliana </italic> , the soluble amino acid concentration decreases with leaf age even though the <span class="trait">protein</span> is being hydrolysed [ <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. The result suggests that the necrosis in the vascular area of the transgenic plants described above prevents the efficient transport of proteolysis-liberated amino acids out of the senescing leaf, as happens in detached leaves [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. Alternatively, the increase in amino acids could be due to a general response to stress, cessation of <span class="trait">protein</span> synthesis or a decrease in the degradation of particular amino acids. That there was no comparable accumulation of sugars due to such a vascular dysfunction is reasonably explained by the decreased carbon assimilation due to destruction of the photosynthetic apparatus.
					</p>
				
					<p>&nbsp;&nbsp;
						It is worth considering what effects the insertion of a fully operative gcl/hyi pathway, as shown in Figure <xref ref-type="fig" rid="F1"> 1 </xref> , would have on photosynthetic metabolism as a whole. The first point is that there would be no release of ammonia and hence no need for the photorespiratory nitrogen cycle, and no need to recycle glutamate and 2-oxoglutarate. There would therefore be a saving of ATP and reduced ferredoxin in the chloroplast from the <span class="gene">GS</span> and GOGAT reactions, whilst NADH would not be generated in the mitochondria, following the glycine to serine conversion. The CO <sub> 2 </sub> liberated in the mitochondria would now be formed in the peroxisomes by the <span class="gene">gcl</span> reaction. However, as there is evidence that there is a specific <span class="trait">protein</span> (PEX10) on the single membrane of the peroxisome that allows attachment of peroxisomes to chloroplasts [ <xref ref-type="bibr" rid="B62"> 62 </xref> ], it is possible that the CO <sub> 2 </sub> would diffuse more readily to the site of Rubisco activity rather than if it had been generated in the mitochondria.
					</p>
				
					<p>&nbsp;&nbsp;
						The initial step in the conversion of glycine to serine requires the action of a four peptide-containing glycine decarboxylase enzyme. The methylene group remaining from glycine following the removal of CO <sub> 2 </sub> , ammonia and NADH synthesis is transferred to tetrahydrofolate (THF) to form 5, 10-methylene-THF, which is then transferred to a second molecule of glycine, catalysed by serine hydroxymethyltransferase. The role of 10-formyl THF deformylase during photorespiration has recently been demonstrated [ <xref ref-type="bibr" rid="B22"> 22 </xref> ]. One carbon metabolism using serine as a donor is important in the synthesis of a range of methylated compounds including nucleic acids, proteins and chlorophyll. [ <xref ref-type="bibr" rid="B63"> 63 </xref> - <xref ref-type="bibr" rid="B65"> 65 </xref> ]. A range of mutant lines of <italic> A. thaliana </italic> and barley have been isolated and characterized that have deficiencies in the double enzyme complex that is required to convert glycine to serine [ <xref ref-type="bibr" rid="B66"> 66 </xref> - <xref ref-type="bibr" rid="B69"> 69 </xref> ]. Initially it was demonstrated that such mutants were able to grow normally in elevated CO <sub> 2 </sub> , indicating the complex was only required for the photorespiratory nitrogen cycle and was not required for the synthesis of other C1-THF derivatives. Other pathways of <span class="gene">C1</span> metabolism were proposed, some involving formate [ <xref ref-type="bibr" rid="B64"> 64 </xref> , <xref ref-type="bibr" rid="B70"> 70 </xref> ]. However, in an elegant series of experiments, Engel et al. [ <xref ref-type="bibr" rid="B24"> 24 </xref> ] demonstrated that knocking out both genes encoding the P <span class="trait">protein</span> of glycine decarboxylase of <italic> A. thaliana </italic> produced a lethal mutant that was unable to grow past the seedling stage even at elevated concentrations of CO <sub> 2 </sub> . As the transgenic lines in these experiments contained <span class="trait">high</span> concentrations of both glycine and serine, it seems unlikely that C1-THF metabolism is a limiting factor.
					</p>
				
					<p>&nbsp;&nbsp;
						Extension of the transformation strategy described would require that the reasons for the lack of expression of the <span class="gene">hyi</span> <span class="trait">protein</span> must be investigated. Hydroxypyruvate and tartronic semialdehyde are labile and reactive compounds that are interconvertible under alkaline conditions without the presence of the <span class="gene">hyi</span> enzyme. Even with the expression of active enzymes of both <span class="gene">gcl</span> and hyi, it may be necessary to undertake further manipulation to down-regulate the aminotransferases that catalyse the amination of glyoxylate. This should then allow a rapid transfer of the glyoxylate to hydroxypyruvate within the peroxisome and negate the requirement for photorespiratory ammonia assimilation.
					</p>
				
					<p>&nbsp;&nbsp;
						The plasmid pBIN19 <italic> <span class="gene">gcl</span> </italic> (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), was constructed from pBIN19AR and pGCL3, encoding a <span class="gene">gcl</span> <span class="trait">protein</span> with a 3' extension sequence encoding the peroxisome targeting sequence RSKL. The <italic> <span class="gene">hyi</span> </italic> gene with a 3' extension sequence encoding the peroxisome targeting sequence was incorporated into pBIN19 <italic> <span class="gene">gcl</span> </italic> to form the pBIN19 <italic> gcl-hyi </italic> plasmid (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), so that both genes could be used for transformation of tobacco at the same time.
					</p>
				
					<p>&nbsp;&nbsp;
						The plasmids were introduced into the wt tobacco by <italic> Agrobacterium tumefaciens </italic> -mediated transformation of leaf explants. Plantlets were regenerated by organ-tissue culture and adapted <italic> ex-vitro </italic> in a controlled environment. The T <sub> 0 </sub> plants were screened by PCR with construct-specific primers to identify those containing the <italic> <span class="gene">gcl</span> </italic> or <italic> <span class="gene">gcl</span> </italic> and <italic> <span class="gene">hyi</span> </italic> genes. A phenotype for plants carrying the genes was recognised by <span class="trait">chlorosis</span> of leaves growing in normal light and in ambient CO <sub> 2 </sub> . The T <sub> 0 </sub> lines retained included line 4, transformed with an empty plasmid, lines <italic> <span class="gene">gcl</span> </italic> 6, <italic> <span class="gene">gcl</span> </italic> 15, <italic> <span class="gene">gcl</span> </italic> 32, <italic> <span class="gene">gcl</span> </italic> 33, <italic> <span class="gene">gcl</span> </italic> 37 and <italic> <span class="gene">gcl</span> </italic> 38, transformed with pBIN19 <italic> <span class="gene">gcl</span> </italic> and the lines <italic> gcl-hyi </italic> 79, <italic> gcl-hyi </italic> 83, <italic> gcl-hyi </italic> 84, <italic> gcl-hyi </italic> 89, <italic> gcl-hyi </italic> 92, transformed with pBIN19 <italic> gcl-hyi </italic> . Plants of these lines were propagated from stem cuttings in low light in a glasshouse. Lines <italic> <span class="gene">gcl</span> </italic> 32, <italic> <span class="gene">gcl</span> </italic> 33, <italic> <span class="gene">gcl</span> </italic> 37, <italic> gcl-hyi </italic> 79, <italic> gcl-hyi </italic> 84 and <italic> gcl-hyi </italic> 92 were retained for further study. For the sake of simplicity, the transgenic lines have only been referred to by their numbers in the text. T <sub> 1 </sub> seeds produced following self-pollination of the selected T <sub> 0 </sub> lines were germinated for 12 days, and seedlings were subsequently grown in 48 cm <sup> 3 </sup> pots in CO <sub> 2 </sub> -enriched (3000 &#x003bc;mol mol <sup> -1 </sup> ) CE cabinets. After 15 days, the seedlings were transferred to a glasshouse for 7 days in ambient CO <sub> 2 </sub> . Seedlings showing <span class="trait">chlorosis</span> were re-potted into 854 cm <sup> 3 </sup> pots and transferred back to CO <sub> 2 </sub> -enriched cabinets to flower and to produce seeds following self-pollination. Samples of T <sub> 2 </sub> seeds from the selected T <sub> 1 </sub> plants exhibiting the phenotype were germinated and the seedlings established in CO <sub> 2 </sub> -enriched cabinets and then subjected to growth in ambient CO <sub> 2 </sub> , in a glasshouse. Individual T <sub> 1 </sub> plants that produced 100% of T <sub> 2 </sub> seedlings showing <span class="trait">chlorosis</span> were selected. Seeds from these T <sub> 1 </sub> plants were used to produce the T <sub> 2 </sub> plants for the experimental studies.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">gcl</span> </italic> gene was transferred from pGCL3 to the Qiagen expression plasmid pQE31 to create an <italic> E.coli </italic> expression plasmid pQE31 <italic> <span class="gene">gcl</span> </italic> . The <italic> <span class="gene">hyi</span> </italic> gene was transferred from pUCP <italic> <span class="gene">hyi</span> </italic> T to the Qiagen expression plasmid pQE32, to create <italic> E.coli </italic> expression plasmid pQE32 <italic> <span class="gene">hyi</span> </italic> . 5 mL cultures of colonies transformed with these plasmids were grown for 3 hours in Luria-Bertani (LB) broth containing 5 &#x003bc;L of ampicillin (100 mg mL <sup> -1 </sup> ) and 3 &#x003bc;L kanamycin (50 mg mL <sup> -1 </sup> ), <span class="trait">protein</span> expression was induced by the addition of Isopropyl &#x003b2;-D-thiogalactosylpyranoside (IPTG) to a final concentration of 0.5 mM and the culture was grown overnight. Next morning, cells from 200 &#x003bc;L aliquots collected by centrifugation, were resuspended in 2&#x000d7; SDS loading buffer (0.06 M Tris-HCl, pH 6.8, 10% v/v glycerol, 2% v/v SDS, 0.1% v/v saturated bromophenol blue), and heated at 95&#x000b0;C for 3 min and analysed by denaturing polyacrylamide gel electrophoresis.
					</p>
				
					<p>&nbsp;&nbsp;
						The primary antibody to <span class="gene">gcl</span> was raised against <span class="trait">protein</span> purified from <italic> E.coli </italic> cells. <italic> E. coli </italic> M15[pREP4] cells were transformed with plasmid pQE30 <italic> <span class="gene">gcl</span> </italic> , and, following induction with 1 mM IPTG, expressed gcl. The <span class="trait">protein</span> was purified for antibody preparation by the method suggested by the manufacturer [ <xref ref-type="bibr" rid="B74"> 74 </xref> ]. Lysozyme was needed to release the <span class="trait">protein</span> from the bacterial cells; the <span class="trait">protein</span> was bound to a Ni-NTA agarose column and was recovered in the fraction eluting in buffer containing 229 mM imidazole. When the eluate was dialysed against phosphate buffered saline (PBS), the <span class="trait">protein</span> precipitated. However, precipitated <span class="trait">protein</span> was dissolved by raising the pH to 9.5 and the solution was freeze dried. This <span class="trait">protein</span> was used to generate rabbit antiserum.
					</p>
				
					<p>&nbsp;&nbsp;
						For hyi, the primary antibody used was raised to a sequence DNPHRGEPGTGEINY located towards the carboxy terminus of the <italic> E.coli </italic> <span class="gene">hyi</span> <span class="trait">protein</span> in rabbit, in conjunction with keyhole limpet haemocyanin (KLH).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Increased salt and <span class="trait">drought</span> tolerance by D-ononitol production in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:22079289  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ahn C(1), Park U, Park PB.
				<br/> 
				年份：2011 Dec 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The methylation of myo-inositol forms O-methyl inositol (D-ononitol) when plants are under abiotic stress in a reaction catalyzed by myo-inositol methyltransferase (IMT). D-Ononitol can serve as an osmoprotectant that prevents water loss in plants. We isolated the IMT cDNA from Glycine max and found by RT-PCR analysis that <span class="gene">GmIMT</span> transcripts are induced by <span class="trait">drought</span> and salinity stress treatments in the leaves of soybean seedlings. We confirmed the <span class="trait">protein</span> product of <span class="gene">GmIMT</span> and its substrate using a recombinant system in E. coli. Transgenic Arabidopsis plants over-expressing <span class="gene">GmIMT</span> displayed improved tolerance to dehydration stress treatment and to a lesser extent <span class="trait">high</span> salinity stress treatment. These results indicate that <span class="gene">GmIMT</span> is functional in heterologous Arabidopsis plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript profiling reveals expression differences in wild-type and glabrous soybean lines.</h2>
			<h3>PMID:22029708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hunt M(1), Kaur N, Stromvik M, Vodkin L.
				<br/> 
				年份：2011 Oct 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trichome hairs affect diverse agronomic characters such as seed weight and <span class="trait">yield,</span> prevent insect damage and reduce loss of water but their molecular control has not been extensively studied in soybean. Several detailed models for trichome development have been proposed for Arabidopsis thaliana, but their applicability to important crops such as cotton and soybean is not fully known. RESULTS: Two <span class="trait">high</span> throughput transcript sequencing methods, Digital Gene Expression (DGE) Tag Profiling and RNA-Seq, were used to compare the transcriptional profiles in wild-type (cv. Clark standard, CS) and a mutant (cv. Clark glabrous, i.e., trichomeless or hairless, CG) soybean isoline that carries the dominant <span class="gene">P1</span> allele. DGE data and RNA-Seq data were mapped to the cDNAs (Glyma models) predicted from the reference soybean genome, Williams 82. Extending the model length by 250 bp at both ends resulted in significantly more matches of authentic DGE tags indicating that many of the predicted gene models are prematurely truncated at the 5' and 3' UTRs. The genome-wide comparative study of the transcript profiles of the wild-type versus mutant line revealed a number of differentially expressed genes. One highly-expressed gene, <span class="gene">Glyma04g35130,</span> in wild-type soybean was of interest as it has <span class="trait">high</span> homology to the cotton gene GhRDL1 gene that has been identified as being involved in cotton fiber initiation and is a member of the BURP <span class="trait">protein</span> family. Sequence comparison of <span class="gene">Glyma04g35130</span> among Williams 82 with our sequences derived from CS and CG isolines revealed various SNPs and indels including addition of one nucleotide C in the CG and insertion of ~60 bp in the third exon of CS that causes a frameshift mutation and premature truncation of peptides in both lines as compared to Williams 82. CONCLUSION: Although not a candidate for the <span class="gene">P1</span> locus, a BURP family member <span class="gene">(Glyma04g35130)</span> from soybean has been shown to be abundantly expressed in the CS line and very weakly expressed in the glabrous CG line. RNA-Seq and DGE data are compared and provide experimental data on the expression of predicted soybean gene models as well as an overview of the genes expressed in young shoot tips of two closely related isolines.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Evaluation of the intake of a low daily amount of soybeans in oxidative stress, lipid and inflammatory profile, and insulin <span class="trait">resistance</span> in patients with metabolic syndrome].</h2>
			<h3>PMID:22011856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bahls LD(1), Venturini D, Scripes Nde A, Lozovoy MA, Simão TN, Simão AN, Dichi I, Morimoto HK.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: Studies show that regular consumption of soybeans reduces the risk of diabetes and cardiovascular diseases. However, most of these studies recommend daily intake of 25 g or more of soy <span class="trait">protein,</span> an amount considered <span class="trait">high</span> and not well tolerated by patients. The objective of this study was to assess the effect of low daily intake of soybeans in oxidative stress and in components of the metabolic syndrome (MS). SUBJECTS AND METHODS: Forty individuals with <span class="gene">MS</span> were selected and divided into two groups: control group (n = 20) and soybean-treated group (n = 20), which consumed 12.95 g of soy <span class="trait">protein</span> for 90 days. RESULTS: After the treatment, the soybean-treated group showed a decrease in fasting glucose and increase in serum HDL and adiponectin. CONCLUSION: Low intake of soy <span class="trait">protein</span> for 90 days, besides being well tolerated by the patients, was able to improve several parameters related to the pathophysiology of MS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Malus hupehensis NPR1 induces pathogenesis-related <span class="trait">protein</span> gene expression in transgenic tobacco.</h2>
			<h3>PMID:21973266  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang JY(1), Qiao YS, Lv D, Gao ZH, Qu SC, Zhang Z.
				<br/> 
				年份：2012 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most commercially grown apple cultivars are susceptible to fungal diseases. Malus hupehensis has <span class="trait">high</span> <span class="trait">resistance</span> to many diseases affecting apple cultivars. Understanding innate defence mechanisms would help to develop disease-resistant apple crops. Non-expressor of pathogenesis-related genes 1 (NPR1) plays a key role in regulating salicylic acid (SA)-mediated systemic acquired <span class="trait">resistance</span> (SAR). MhNPR1 cDNA, corresponding to genomic DNA and its 5' flanking sequences, was isolated from M. hupehensis. Sequence analysis showed that the regulatory mechanism for oligomer-monomer transition of the MhNPR1 <span class="trait">protein</span> in apple might be similar to that of <span class="gene">GmNPR1</span> in soybean, but different from that of AtNPR1 in Arabidopsis. No significant differences in MhNPR1 expression were found in M. hupehensis after infection with Botryosphaeria berengeriana, showing that MhNPR1 might be regulated by pathogens at the <span class="trait">protein</span> level, as described for Arabidopsis and grapevine. SA treatment significantly induced MhNPR1 expression in leaves, stems and roots, while methyl jasmonate (MeJA) treatment induced MhNPR1 expression in roots, but not in leaves or stems. The expression of MhNPR1 was highly increased in roots, moderately in leaves, and did not change in stems after treatment with 1-aminocyclopropane-1-carboxylic acid (ACC). SAR marker genes (MhPR1 and MhPR5) were induced by SA, MeJA and ACC in leaves, stems and roots. Overexpression of MhNPR1 significantly induced the expression of pathogenesis-related genes (NtPR1, NtPR3 and NtPR5) in transgenic tobacco plants and <span class="trait">resistance</span> to the fungus Botrytis cinerea, suggesting that MhNPR1 orthologues are a component of the SA defence signalling pathway and SAR is induced in M. hupehensis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development and validation of a method for the quantification of milk proteins in food products based on liquid chromatography with mass spectrometric detection.</h2>
			<h3>PMID:21919337  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lutter P(1), Parisod V, Weymuth H.
				<br/> 
				年份：2011 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The protection of allergic consumers is crucial to the food industry. Therefore, accurate methods for the detection of food allergens are required. Targeted detection of selected molecules by <span class="gene">MS</span> combines <span class="trait">high</span> selectivity with accurate quantification. A confirmatory method based on LC/selected reaction monitoring (SRM)-MS/MS was established and validated for the quantification of milk traces in food. Tryptic peptides of the major milk proteins beta-lactoglobulin, beta-casein, alphaS2-casein, and K-casein were selected as quantitative markers. Precise quantification was achieved using internal standard peptides containing isotopically labeled amino acids. For each peptide, qualifier and quantifier fragments were selected according to Commission Decision 2002/657/EC. A simple sample preparation method was established without immunoaffinity or SPE enrichment steps for food matrixes containing different amounts of <span class="trait">protein,</span> such as baby food, breakfast cereals, infant formula, and cereals. Intermediate reproducibility, repeatability, accuracy, and measurement uncertainty were determined for each matrix. LOD values of 0.2-0.5 mg/kg, e.g., for beta-lactoglobulin, were comparable to those obtained with ELISA kits. An LOQ of approximately 5 mg/kg, expressed as mass fraction skim milk powder, was validated in protein-rich infant cereals. The obtained validation data show that the described LC/SRM-MS/MS approach can serve as a confirmatory method for the determination of milk traces in selected food matrixes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Synthesis of green note aroma compounds by biotransformation of fatty acids using yeast cells coexpressing lipoxygenase and hydroperoxide lyase.</h2>
			<h3>PMID:21789493  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Buchhaupt M(1), Guder JC, Etschmann MM, Schrader J.
				<br/> 
				年份：2012 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Green notes are substances that characterize the aroma of freshly cut grass, cucumbers, green apples, and foliage. In plants, they are synthesized by conversion of linolenic or linoleic acid via the enzymes lipoxygenase (LOX) and hydroperoxide lyase (HPL) to short-chained aldehydes. Current processes for production of natural green notes rely on plant homogenates as enzyme sources but are limited by low enzyme concentration and low specificity. In an alternative approach, soybean <span class="gene">LOX2</span> and watermelon <span class="gene">HPL</span> were overexpressed in Saccharomyces cerevisiae. After optimization of the expression constructs, a yeast strain coexpressing <span class="gene">LOX</span> and <span class="gene">HPL</span> was applied in whole cell biotransformation experiments. Whereas addition of linolenic acid to growing cultures of this strain yielded no products, we were able to identify <span class="trait">high</span> green note concentrations when resting cells were used. The primary biotransformation product was 3(Z)-hexenal, a small amount of which isomerized to 2(E)-hexenal. Furthermore, both aldehydes were reduced to the corresponding green note alcohols by endogenous yeast alcohol dehydrogenase to some extent. As the cosolvent ethanol was the source of reducing equivalents for green note alcohol formation, the hexenal/hexenol ratio could be influenced by the use of alternative cosolvents. Further investigations to identify the underlying mechanism of the rather low biocatalyst stability revealed a <span class="trait">high</span> toxicity of linolenic acid to yeast cells. The whole cell catalyst containing <span class="gene">LOX</span> and <span class="gene">HPL</span> enzyme activity described here can be a promising approach towards a highly efficient microbial green note synthesis process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Snack bars with <span class="trait">high</span> soy <span class="trait">protein</span> and isoflavone content for use in diets to control dyslipidaemia.</h2>
			<h3>PMID:21728920  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lobato LP(1), Iakmiu Camargo Pereira AE, Lazaretti MM, Barbosa DS, Carreira CM, Mandarino JM, Grossmann MV.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objective was to develop a cereal <span class="gene">bar</span> product rich in isoflavones and soy <span class="trait">protein</span> to be used in diets for controlling dyslipidaemia. A soy snack <span class="gene">bar</span> with 39.88 g/100 g dietary fibre, 34.25 g/100 g <span class="trait">protein,</span> 100.39 mg/100 g isoflavones and 245.47 kcal/100 g was produced. The shelf life of the product was tested over a period of 6 months. The hardness, water activity and darkness of the snack bars increased with storage time. Clinical trial was carried out to evaluate the effectiveness of the soy snack <span class="gene">bar</span> on the lipid profiles of 22 dyslipidaemic subjects for a period of 45 days. There was a decrease in levels of triglycerides (TGs) (- 20%) and an increase in high-density lipoprotein cholesterol (HDL-c) (+8%) comparing with baseline values. Therefore, relatively <span class="trait">high</span> intakes of soy <span class="trait">protein</span> and isoflavones in moderately hypercholesterolaemic subjects (&gt;200 mg/dl) may have a moderately beneficial effect on plasma lipid profiles, as TG and HDL-c levels, without additional diet restrictions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel <span class="gene">FAD2-1</span> A allele in a soybean plant introduction offers an alternate means to produce soybean seed oil with 85% oleic acid content.</h2>
			<h3>PMID:21681491  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pham AT(1), Lee JD, Shannon JG, Bilyeu KD.
				<br/> 
				年份：2011 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alteration of fatty acid profiles in soybean to improve soybean oil quality has been a long-time goal of soybean researchers. Soybean oil with elevated oleic acid is desirable because this monounsaturated fatty acid improves the nutrition and oxidative stability of soybean oil compared to other oils. In the lipid biosynthetic pathway, the enzyme fatty acid desaturase 2 (FAD2) is responsible for the conversion of oleic acid precursors to linoleic acid precursors in developing soybean seeds. Two genes encoding <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> were identified to be expressed specifically in seeds during embryogenesis and have been considered to hold an important role in controlling the seed oleic acid content. A total of 22 soybean plant introduction (PI) lines identified to have an elevated oleic acid content were characterized for sequence mutations in the FAD 2-1A and <span class="gene">FAD2-1B</span> genes. PI 603452 was found to contain a deletion of a nucleotide in the second exon of FAD2-1A. These important SNPs were used in developing molecular marker genotyping assays. The assays appear to be a reliable and accurate tool to identify the FAD 2-1A and <span class="gene">FAD2-1B</span> genotype of wild-type and mutant plants. PI 603452 was subsequently crossed with PI 283327, a soybean line that has a mutation in FAD2-1B. Interestingly, soybean lines carrying both homozygous insertion/deletion mutation (indel) <span class="gene">FAD2-1A</span> alleles and mutant <span class="gene">FAD2-1B</span> alleles have an average of 82-86% oleic acid content, compared to 20% in conventional soybean, and low levels of linoleic and linolenic acids. The newly identified indel mutation in the <span class="gene">FAD2-1A</span> gene offers a simple method for the development of <span class="trait">high</span> oleic acid commercial soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of Candida antarctica lipase B gene open reading frame using automated PCR gene assembly protocol on robotic workcell and expression in an ethanologenic yeast for use as resin-bound biocatalyst in biodiesel production.</h2>
			<h3>PMID:21609683  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hughes SR(1), Moser BR, Harmsen AJ, Bischoff KM, Jones MA, Pinkelman R, Bang SS, Tasaki K, Doll KM, Qureshi N, Saha BC, Liu S, Jackson JS, Robinson S, Cotta MC, Rich JO, Caimi P.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A synthetic Candida antarctica lipase B (CALB) gene open reading frame (ORF) for expression in yeast was constructed, and the lycotoxin-1 (Lyt-1) <span class="gene">C3</span> variant gene ORF, potentially to improve the availability of the active enzyme at the surface of the yeast cell, was added in frame with the CALB ORF using an automated PCR assembly and DNA purification protocol on an integrated robotic workcell. Saccharomyces cerevisiae strains expressing CALB <span class="trait">protein</span> or CALB Lyt-1 fusion <span class="trait">protein</span> were first grown on 2% (w/v) glucose, producing 9.3 g/L ethanol during fermentation. The carbon source was switched to galactose for GAL1-driven expression, and the CALB and CALB Lyt-1 enzymes expressed were tested for fatty acid ethyl ester (biodiesel) production. The synthetic enzymes catalyzed the formation of fatty acid ethyl esters from ethanol and either corn or soybean oil. It was further demonstrated that a one-step-charging resin, specifically selected for binding to lipase, was capable of covalent attachment of the CALB Lyt-1 enzyme, and that the resin-bound enzyme catalyzed the production of biodiesel. High-level expression of lipase in an ethanologenic yeast strain has the potential to increase the profitability of an integrated biorefinery by combining bioethanol production with coproduction of a low-cost biocatalyst that converts corn oil to biodiesel.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean transcription factor <span class="gene">GmMYBZ2</span> represses catharanthine biosynthesis in hairy roots of Catharanthus roseus.</h2>
			<h3>PMID:21590290  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou ML(1), Hou HL, Zhu XM, Shao JR, Wu YM, Tang YX.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Catharanthus roseus (L.) G. Don is a plant species known for its production of a variety of terpenoid indole alkaloids, many of which have pharmacological activities. Production of catharanthine in cell cultures or in hairy roots established by transformation with Agrobacterium rhizogenes is of interest because catharanthine can be chemically coupled to the abundant leaf alkaloid vindoline to form the valuable anticancer drug vinblastine. Here, we observed a <span class="trait">high</span> amount of catharanthine in hairy roots of C. roseus, established by infecting leaf explants with the A. rhizogenes &gt;agropine-type A4 strain carrying plasmid pRi. T-DNA transfer from plasmid pRi into hairy roots was confirmed by PCR for the essential T-DNA genes rolA and rolB and the agropine synthesis gene ags. The results suggest that integration of T-DNA into the plant DNA plays a positive role on the catharanthine pathway in C. roseus hairy roots. Furthermore, co-transformation with the soybean transcription factor <span class="gene">GmMYBZ2</span> indicated that <span class="gene">GmMYBZ2</span> reduces the catharanthine production by alteration of expression of a number of genes linked to the pathway. Transcription levels of the zinc-finger transcription factor 1 gene ZCT1 were <span class="trait">high,</span> and the transcription levels of the anthranilate synthase gene ASα, the strictosidine synthase gene STR, and the key transcription factor gene octadecanoid-responsive Catharanthus APETALA2/ethylene response factor were low. In addition, <span class="gene">GmMYBZ2</span> had a negative effect on the gene expression levels of A-type cyclin CYSA and B-type cyclin CYSB, which was correlated with a reduced growth rate of the hairy roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of whole-crop pea (Pisum sativum L.) silages differing in condensed tannin content as a substitute for grass silage and soybean meal on the performance, metabolism, and carcass characteristics of lambs.</h2>
			<h3>PMID:21571892  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hart KJ(1), Sinclair LA, Wilkinson RG, Huntington JA.
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two experiments were conducted to investigate the effect of inclusion of whole-crop pea (WCP) silages, differing in condensed tannin content, as a substitute for grass silage (GS) and soybean meal on lamb metabolism, performance, plasma metabolites, digestibility, and carcass characteristics. In both experiments lambs were offered either solely <span class="gene">GS</span> or a 50:50 mix on a DM basis of <span class="gene">GS</span> with either low-tannin (LTPS) or high-tannin (HTPS) pea silage ad libitum. Each forage mix was fed with either 400 g/d of low-protein (LP) concentrate or 400 g/d of LP with an additional 200 g/d of pelletized soybean meal (HP), resulting in 6 dietary treatments. Experiment 1 examined the effects of the diets on metabolism, digestibility, and N balance using 6 lambs in 4 periods of 21 d in an incomplete crossover design. Experiment 2 used 48 lambs and examined the effects of the diets on ADG, plasma metabolites, and carcass characteristics over 56 d. Both experiments were analyzed using a 3 × 2 factorial arrangement of treatments. In Exp. 1, lambs offered the LTPS diets had a greater (P &lt; 0.05) digestibility of DM and OM than those offered the <span class="gene">GS</span> diets. Lambs offered the WCP silages had an increased (P &lt; 0.05) N intake, N output, and digestibility of GE compared with those offered GS. Mean N digestibility was greatest (P &lt; 0.05) in lambs offered LTPS. Lambs offered HP diets had increased (P &lt; 0.001) digestibility of DM, OM, GE and N, and N- intake, output, retention, and digestibility compared with those offered the LP diets. In Exp. 2, there was no effect (P &gt; 0.05) of forage type on intake, slaughter BW, or feed conversion efficiency (FCE). However, lambs offered the LTPS had a greater (P &lt; 0.05) ADG than those offered the <span class="gene">GS</span> diets. Feeding diets containing HP increased (P &lt; 0.001) total DMI, slaughter BW, ADG, and FCE. Lambs offered the WCP had a greater (P &lt; 0.05) plasma β-hydroxybutyrate and urea concentration compared with those offered the <span class="gene">GS</span> diets. Feeding lambs HP diets increased (P &lt; 0.05) plasma urea and total <span class="trait">protein.</span> Forage mix had no effect (P &gt; 0.05) on carcass composition except for fat depth, which was greater (P &lt; 0.05) in lambs offered WCP silage. Diets containing the HP increased (P &lt; 0.05) carcass weight, hind leg circumference, chop dimensions, and kidney weight. It was concluded that lambs offered LTPS performed better than those offered <span class="gene">GS</span> and that LTPS has a concentrate sparing effect. Additionally, the increased tannin concentration in HTPS did not increase performance over lambs offered either <span class="gene">GS</span> or LTPS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nonsense mutation of an MYB transcription factor is associated with purple-blue flower color in soybean.</h2>
			<h3>PMID:21566002  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Takahashi R(1), Benitez ER, Oyoo ME, Khan NA, Komatsu S.
				<br/> 
				年份：2011 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies revealed that the recessive allele of the <span class="gene">W2</span> locus generated purple-blue color and <span class="trait">high</span> vacuolar pH of flower petals in soybean. The location of <span class="gene">W2</span> gene was reportedly close to simple sequence repeat marker Satt318 in molecular linkage group B2. We used information from the soybean genome to clone a candidate gene for W2. An MYB transcription factor gene belonging to G20 group was found in the vicinity of Satt318. Full-length cDNAs were cloned from purple-flowered cultivar Harosoy (W2 allele) and purple-blue flowered cultivars, Nezumisaya and w2-20 (w2 allele), by reverse transcription-PCR and designated as <span class="gene">GmMYB-G20-1.</span> Its open reading frame was 1083 bp long that encoded 361 amino acids in Harosoy. <span class="gene">GmMYB-G20-1</span> had 53.7% similarity in amino acid sequence with the PH4 gene of petunia controlling blueness and vacuolar pH of flower petals. <span class="gene">GmMYB-G20-1</span> of Nezumisaya and w2-20 had 3 base substitutions compared with that of Harosoy. The first substitution generated a stop codon in the MYB domain, resulting in truncated polypeptides. Cleaved amplified polymorphic sequence (CAPS) marker was developed to detect the base substitution. The polymorphic CAPS marker co-segregated with alleles at the <span class="gene">W2</span> locus in the F(2) population. These results suggest that <span class="gene">GmMYB-G20-1</span> might correspond to the <span class="gene">W2</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fe binding properties of two soybean (Glycine max L.) LEA4 proteins associated with antioxidant activity.</h2>
			<h3>PMID:21531760  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu G(1), Xu H, Zhang L, Zheng Y.
				<br/> 
				年份：2011 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Late embryogenesis abundant (LEA) group 4 (LEA4) proteins play an important role in the water stress tolerance of plants. Although they have been hypothesized to stabilize macromolecules in stressed cells, the protective functions and mechanisms of LEA4 proteins are still not clear. In this study, the metal binding properties of two related soybean LEA4 proteins, <span class="gene">GmPM1</span> and <span class="gene">GmPM9,</span> were tested using immobilized metal ion affinity chromatography (IMAC). The metal ions Fe(3+), Ni(2+), Cu(2+) and Zn(2+) were observed to bind these two proteins, while Ca(2+), Mg(2+) or Mn(2+) did not. Results from isothermal titration calorimetry (ITC) indicated that the binding affinity of <span class="gene">GmPM1</span> for Fe(3+) was stronger than that of <span class="gene">GmPM9.</span> Hydroxyl radicals generated by the Fe(3+)/H(2)O(2) system were scavenged by both <span class="gene">GmPM1</span> and <span class="gene">GmPM9</span> in the absence or the presence of <span class="trait">high</span> ionic conditions (100 mM NaCl), although the scavenging activity of <span class="gene">GmPM1</span> was significantly greater than that of <span class="gene">GmPM9.</span> These results suggest that <span class="gene">GmPM1</span> and <span class="gene">GmPM9</span> are metal-binding proteins which may function in reducing oxidative damage induced by abiotic stress in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPep914, an eight-amino acid peptide isolated from soybean leaves, activates defense-related genes.</h2>
			<h3>PMID:21478368  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yamaguchi Y(1), Barona G, Ryan CA, Pearce G.
				<br/> 
				年份：2011 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Only a handful of endogenous peptide defense signals have been isolated from plants. Herein, we report a novel peptide from soybean (Glycine max) leaves that is capable of alkalinizing the media of soybean suspension cells, a response that is generally associated with defense peptides. The peptide, DHPRGGNY, was synthesized and found to be active at 0.25 nM and requiring only 5 to 10 min to obtain a maximal pH change. The peptide is located on the carboxy-terminal end of a 52-amino acid precursor <span class="trait">protein</span> <span class="gene">(Glyma12g00990)</span> deduced from the soybean genome project. A search of the soybean databank revealed a homolog <span class="gene">(Glyma09g36370)</span> that contained a similar peptide, DLPRGGNY, which was synthesized and shown to have identical activity. The peptides, designated <span class="gene">GmPep914</span> (DHPRGGNY) and <span class="gene">GmPep890</span> (DLPRGGNY), were capable of inducing the expression of both <span class="gene">Glyma12g00990</span> <span class="gene">(GmPROPEP914)</span> and <span class="gene">Glyma09g36370</span> <span class="gene">(GmPROPEP890)</span> in cultured soybean suspension cells within 1 h. Both peptides induced the expression of defense genes, including CYP93A1, a cytochrome <span class="gene">P450</span> gene involved in phytoalexin synthesis, chitinaseb1-1, a chitinase involved in pathogen defense, and Glycine max chalcone synthase1 <span class="gene">(Gmachs1),</span> chalcone synthase, involved in phytoalexin production. Both <span class="gene">GmPROPEP914</span> and <span class="gene">GmPROPEP890</span> were highly expressed in the roots, relative to the aerial portions of the plant. However, treatment of the aerial portion of soybean plants with hormones involved in elicitation of defense responses revealed a significant increase in expression levels of <span class="gene">GmPROPEP914</span> and <span class="gene">GmPROPEP890.</span> A search of gene databases revealed homologous sequences in other members of the Fabales and also in the closely related Cucurbitales but not in any other order of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Polyproline II structure is critical for the enzyme protective function of soybean <span class="gene">Em</span> (LEA1) conserved domains.</h2>
			<h3>PMID:21455835  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zou Y(1), Hong R, He S, Liu G, Huang Z, Zheng Y.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Group 1 late embryogenesis-abundant (LEA1) proteins protect enzyme activity from dehydration and are structurally conserved with three different 20 amino acid motifs in the N-terminal, middle and C-terminal domains. Three soybean <span class="gene">Em</span> (LEA1) domain peptides (Em-N, Em-2M and Em-C) covering these respective motifs were constructed and had differential protective ability on lactate dehydrogenase against freeze-thaw: Em-C &gt; Em-2M &gt; Em-N. CD spectroscopy revealed that Em-2M and Em-C contained both polyproline II (PII) helical structure and α-helix, while Em-N had a <span class="trait">high</span> potential to form α-helix but did not contain PII structure. The PII helical structure between the third and fifth glycine in the middle motif was shown, through site mutation, to be critical for the enzyme protective function of soybean <span class="gene">Em</span> (LEA1) conserved domain under freezing stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in the plant proteome resulting from salt stress: toward the creation of salt-tolerant crops?</h2>
			<h3>PMID:21440686  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sobhanian H(1), Aghaei K, Komatsu S.
				<br/> 
				年份：2011 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Salinity in agricultural land is a major problem worldwide, placing a severe constraint on crop growth and productivity in many regions, and increased salinization of arable land is expected to have devastating global effects. Though plants vary in their sensitivity to salt stress, <span class="trait">high</span> salinity causes water deficit and ion toxicity in many plant species. Considerable efforts have therefore been made to investigate how genes respond to salt stress in various plants by using several approaches, including proteomics. Proteomic approaches for identifying proteins that are regulated in response to salt stress are becoming common in the post-genomics <span class="gene">era</span> of crop research. In this review, we describe the physiological and biological changes in the proteomes of several important food crops under salt stress. We also provide a viewpoint into how proteomics-based research is likely to develop in this field.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and biochemical characterization of an endo-β-mannanase gene from soybean for soybean meal improvement.</h2>
			<h3>PMID:21425861  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Pantalone VR, Li G, Chen F.
				<br/> 
				年份：2011 May 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean meal is the most commonly used <span class="trait">protein</span> source in animal feeds. Among the undesirable attributes of soybean meal is the <span class="trait">high</span> level of β-mannan, which was determined to be detrimental to the growth performance of animals. β-Mannan is a type of hemicellulose in the plant cell wall and can be hydrolyzed by endo-β-mannanase. The goal of this study is to isolate and characterize an endo-β-mannanase gene from soybean that can be used for genetic improvement of soybean meal. From the sequenced soybean genome, 21 putative endo-β-mannanase genes were identified. On the basis of their relatedness to known functional plant endo-β-mannanases, four soybean endo-β-mannanase genes <span class="gene">(GmMAN1</span> to <span class="gene">GmMAN4)</span> were chosen for experimental analysis. <span class="gene">GmMAN1</span> and <span class="gene">GmMAN4</span> showed expression in the soybean tissue examined, and their cDNAs without the sequences for signal peptide were cloned and expressed in Escherichia coli to produce recombinant enzymes. Only <span class="gene">GmMAN1</span> showed endo-β-mannanase hydrolase activity. Further gene expression analysis showed that <span class="gene">GmMAN1</span> is specifically expressed in cotyledons of seedlings, suggesting a role of <span class="gene">GmMAN1</span> in degrading mannan-rich food reserves during soybean seedling establishment. Purified recombinant <span class="gene">GmMAN1</span> exhibited an apparent K(m) value of 34.9 mg/mL. The catalytic efficiency (k(cat)/K(m)) of <span class="gene">GmMAN1</span> was determined to be 0.7 mL/(mg·s). <span class="gene">GmMAN1</span> was also shown to be active in hydrolyzing the β-mannan-rich cell wall of soybean seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of water content on thermal behavior of freeze-dried soy whey and their isolated proteins.</h2>
			<h3>PMID:21413812  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sobral PA(1), Palazolo GG, Wagner JR.
				<br/> 
				年份：2011 Apr 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Thermal behavior of lyophilized soy whey (LSW) and whey soy proteins (WSP) at different water contents (WC) was studied by DSC. In anhydrous condition, Kunitz trypsin inhibitor (KTI) and lectin (L) were more heat stable for WSP with respect to LSW sample. The increase of WC destabilized both proteins but differently depending on the sample analyzed. Thermal stability inversion of <span class="gene">KTI</span> and L was observed for WSP and LSW at 50.0% and 17.0% WC, respectively, which correspond to the same water-protein content mass ratio (W/P ≈ 1.9). At W/P &lt; 1.9, <span class="gene">KTI</span> was more heat stable than L. Before the inversion point, WC strongly modified the peak temperatures (T(p)) of <span class="gene">KTI</span> and L for WSP, whereas this behavior was not observed for LSW. The <span class="trait">high</span> sugar content was responsible for the thermal behavior of <span class="gene">KTI</span> and L in LSW under anhydrous condition and low WC. These results have important implications for the soy whey processing and inactivation of antinutritional factors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Map-Based Cloning Strategy Employing a Residual Heterozygous Line Reveals that the <italic> GIGANTEA </italic> Gene Is Involved in Soybean Maturity and Flowering</h2>
			<h3>PMID:21406680</h3>
				

				
					<h3>PMCID:3122305</h3>
					<h4>期刊:Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Satoshi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Zhengjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> * </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hideshima </surname>  <given-names> Rumiko </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tsubokura </surname>  <given-names> Yasutaka </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sato </surname>  <given-names> Shusei </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> &#x000a7; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamanaka </surname>  <given-names> Naoki </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> ** </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takahashi </surname>  <given-names> Ryoji </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> &#x02020;&#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anai </surname>  <given-names> Toyoaki </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tabata </surname>  <given-names> Satoshi </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> &#x000a7; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kitamura </surname>  <given-names> Keisuke </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> &#x02021;&#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harada </surname>  <given-names> Kyuya </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> * </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan, </aff>  <aff id="aff2">  <label> &#x02020; </label> Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Harbin 150040, People's Republic of China, </aff>  <aff id="aff3">  <label> &#x02021; </label> Faculty of Agriculture, Saga University, Saga 840-8502, Japan, </aff>  <aff id="aff4">  <label> &#x000a7; </label> Kazusa DNA Research Institute, Kisarazu, Chiba 292-0812, Japan, </aff>  <aff id="aff5">  <label> ** </label> Japan International Research Center for Agricultural Sciences, Tsukuba, Ibaraki 305-8686, Japan, </aff>  <aff id="aff6">  <label> &#x02020;&#x02020; </label> National Institute of Crop Science, Tsukuba, Ibaraki 305-8518, Japan and </aff>  <aff id="aff7">  <label> &#x02021;&#x02021; </label> Graduate School of Agriculture, Hokkaido University, Sapporo, Hokkaido 060-8589, Japan </aff>
				<br/> 
				年份：<day> 3 </day>  <month> 3 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flowering is indicative of the transition from vegetative to reproductive phase, a critical event in the life cycle of plants. In soybean (Glycine max), a flowering quantitative trait locus, FT2, corresponding to the maturity locus E2, was detected in recombinant inbred lines (RILs) derived from the varieties "Misuzudaizu" (ft2/ft2; JP28856) and "Moshidou Gong 503" (FT2/FT2; JP27603). A map-based cloning strategy using the progeny of a residual heterozygous line (RHL) from the RIL was employed to isolate the gene responsible for this quantitative trait locus. A GIGANTEA ortholog, <span class="gene">GmGIa</span> <span class="gene">(Glyma10g36600),</span> was identified as a candidate gene. A common premature stop codon at the 10th exon was present in the Misuzudaizu allele and in other near isogenic lines (NILs) originating from Harosoy (e2/e2; PI548573). Furthermore, a mutant line harboring another premature stop codon showed an earlier flowering phenotype than the original variety, Bay (E2/E2; PI553043). The e2/e2 genotype exhibited elevated expression of <span class="gene">GmFT2a,</span> one of the florigen genes that leads to early flowering. The effects of the E2 allele on flowering time were similar among NILs and constant under <span class="trait">high</span> (43°N) and middle (36°N) latitudinal regions in Japan. These results indicate that <span class="gene">GmGIa</span> is the gene responsible for the E2 locus and that a null mutation in <span class="gene">GmGIa</span> may contribute to the geographic adaptation of soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flowering is indicative of the transition from vegetative to reproductive phase, a critical event in the life cycle of plants. In soybean ( <italic> Glycine max </italic> ), a flowering quantitative trait locus, <italic> FT2 </italic> , corresponding to the maturity locus <italic> E2 </italic> , was detected in recombinant inbred lines (RILs) derived from the varieties &#x0201c;Misuzudaizu&#x0201d; ( <italic> ft2/ft2 </italic> ; JP28856) and &#x0201c;Moshidou Gong 503&#x0201d; ( <italic> FT2/FT2 </italic> ; JP27603). A map-based cloning strategy using the progeny of a residual heterozygous line (RHL) from the RIL was employed to isolate the gene responsible for this quantitative trait locus. A <italic> GIGANTEA </italic> ortholog, <italic> <span class="gene">GmGIa</span> </italic> <span class="gene">(Glyma10g36600),</span> was identified as a candidate gene. A common premature stop codon at the 10th exon was present in the Misuzudaizu allele and in other near isogenic lines (NILs) originating from Harosoy ( <italic> e2/e2 </italic> ; PI548573). Furthermore, a mutant line harboring another premature stop codon showed an earlier flowering phenotype than the original variety, Bay ( <italic> E2/E2 </italic> ; PI553043). The <italic> e2/e2 </italic> genotype exhibited elevated expression of <italic> <span class="gene">GmFT2a</span> </italic> , one of the florigen genes that leads to early flowering. The effects of the <italic> E2 </italic> allele on flowering time were similar among NILs and constant under <span class="trait">high</span> (43&#x000b0;N) and middle (36&#x000b0;N) latitudinal regions in Japan. These results indicate that <italic> <span class="gene">GmGIa</span> </italic> is the gene responsible for the <italic> E2 </italic> locus and that a null mutation in <italic> <span class="gene">GmGIa</span> </italic> may contribute to the geographic adaptation of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In <italic> A. thaliana </italic> , flowering is promoted under long-day (LD) conditions through the <italic> CONSTANS </italic> ( <italic> CO </italic> )-mediated photoperiodic flowering pathway that regulates the expression of the <italic> FLOWERING LOCUS T </italic> ( <italic> FT </italic> ) encoding florigen ( <xref ref-type="bibr" rid="bib62"> S <sc> amach </sc> <italic> et al. </italic> 2000 </xref> ; <xref ref-type="bibr" rid="bib69"> S <sc> uarez </sc> -L <sc> opez </sc> <italic> et al. </italic> 2001 </xref> ; <xref ref-type="bibr" rid="bib2"> A <sc> be </sc> <italic> et al. </italic> 2005 </xref> ). The stability of <italic> CO </italic> mediated by phytochrome A and cryptochrome avoiding proteasome <span class="trait">protein</span> degradation increases under LD conditions and induces the expression of <italic> FT </italic> in leaves ( <xref ref-type="bibr" rid="bib89"> Y <sc> anovsky </sc> and K <sc> ay </sc> 2002 </xref> ; <xref ref-type="bibr" rid="bib77"> V <sc> alverde </sc> <italic> et al. </italic> 2004 </xref> ; <xref ref-type="bibr" rid="bib43"> L <sc> iu </sc> <italic> et al. </italic> 2008 </xref> ). The <italic> GIGANTEA </italic> ( <italic> GI </italic> ) mutant, discovered half a century ago, shows the late-flowering phenotype in LD conditions ( <xref ref-type="bibr" rid="bib59"> R <sc> edei </sc> 1962 </xref> ). <italic> GI </italic> encodes a nuclear-localized membrane <span class="trait">protein</span> that functions upstream of <italic> CO </italic> and <italic> FT </italic> ( <xref ref-type="bibr" rid="bib37"> K <sc> oornneef </sc> <italic> et al. </italic> 1998 </xref> ; <xref ref-type="bibr" rid="bib22"> F <sc> owler </sc> <italic> et al. </italic> 1999 </xref> ; <xref ref-type="bibr" rid="bib28"> H <sc> uq </sc> <italic> et al. </italic> 2000 </xref> ; <xref ref-type="bibr" rid="bib50"> M <sc> izoguchi </sc> <italic> et al. </italic> 2005 </xref> ). GI coupled with the blue light receptor <span class="trait">protein,</span> FLAVIN binding KELCH REPEAT F-BOX with PAS/LOV domains (FKF), forms a blue-light-dependent complex that degrades the CYCLING DOF FACTOR 1 (CDF) <span class="trait">protein</span> that binds the promoter region of <italic> CO </italic> and thereby induces <italic> CO </italic> expression ( <xref ref-type="bibr" rid="bib53"> N <sc> elson </sc> <italic> et al. </italic> 2000 </xref> ; <xref ref-type="bibr" rid="bib29"> I <sc> maizumi </sc> <italic> et al. </italic> 2003 </xref> , <xref ref-type="bibr" rid="bib30"> 2005 </xref> ; <xref ref-type="bibr" rid="bib64"> S <sc> awa </sc> <italic> et al. </italic> 2007 </xref> ). Moreover, GI regulates another pathway that controls <italic> FT </italic> expression via microRNAs that cooperate with another transcriptional factor ( <xref ref-type="bibr" rid="bib33"> J <sc> <span class="gene">ung</span> </sc> <italic> et al. </italic> 2007 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The complete sequence of the physical contig, assembled from the five BAC clones, corresponded to a 427-kbp sequence from 44,415 to 44,843 kbp in <span class="gene">Gm10</span> of the soybean genome sequence available in the public database. The 94-kbp sequence of MiB300H01 coincided with the sequence from 44,693 to 44,787 kbp in <span class="gene">Gm10.</span> The average similarity between the sequences of Misuzudaizu and Williams 82 in this region was 99.6%. Nine annotated genes <span class="gene">(Glyma10g36580&#x02013;36670)</span> were predicted in this region. One of these genes, <span class="gene">Glyma10g36600,</span> with a <span class="trait">high</span> level of similarity with the <italic> GI </italic> gene, was considered a strong candidate for the <italic> FT2 </italic> locus, since the loss of function of <italic> GI </italic> is known to cause drastic changes in the flowering phenotype of other plant species ( <xref ref-type="bibr" rid="bib22"> F <sc> owler </sc> <italic> et al. </italic> 1999 </xref> ; <xref ref-type="bibr" rid="bib26"> H <sc> echt </sc> <italic> et al. </italic> 2007 </xref> ). On the basis of this assumption, we isolated the complete predicted coding region using an RNA sample extracted from leaves of NILs 6&#x02013;8- <italic> FT2 </italic> . We hereafter refer to this gene as <italic> <span class="gene">GmGIa</span> </italic> , since another <italic> GI </italic> gene, <italic> <span class="gene">GmGIb</span> </italic> , was also obtained from the same RNA sample. <italic> <span class="gene">GmGI</span> </italic> genes isolated from the Mo late-flowering allele ( <italic> <span class="gene">GmGIa-Mo</span> </italic> and <italic> <span class="gene">GmGIb-Mo</span> </italic> ) were found to encode proteins consisting of 1170 and 1168 amino acids, respectively. Amino acid identity between the two <span class="gene">GmGI</span> proteins was close to 97.3%. The predicted amino acid sequences of <span class="gene">GmGIa</span> and <span class="gene">GmGIb</span> were completely identical to <span class="gene">Glyma10g36600</span> and <span class="gene">Glyma20g30980,</span> respectively. Phylogenic analysis showed that these soybean GI proteins displayed a <span class="trait">high</span> level of similarity (71&#x02013;91%) to GI proteins from dicots and monocots ( <xref ref-type="fig" rid="fig3"> Figure 3 </xref> ). The presence of paralog genes with <span class="trait">high</span> levels of similarity attracted our attention. However, as there were no experimental data for <italic> <span class="gene">GmGIb</span> </italic> , we focused on the <italic> <span class="gene">GmGIa</span> </italic> gene in the present study.
					</p>
				
					<p>&nbsp;&nbsp;
						The coding sequence of <italic> <span class="gene">GmGIa</span> </italic> was extended to a 20-kbp genomic region and contained 14 exons ( <xref ref-type="fig" rid="fig4"> Figure 4A </xref> ). Marker 4, which cosegregated with the QTL genotypes and originated from the AFLP marker, E60M38, was located in the 5th intron ( <xref ref-type="fig" rid="fig4"> Figure 4A </xref> ). Compared to <italic> <span class="gene">GmGIa</span> </italic> -Mo, the Misuzudaizu early-flowering allele, <italic> <span class="gene">GmGIa</span> </italic> -Mi, showed four single nucleotide polymorphisms (SNPs) in its coding sequence ( <xref ref-type="fig" rid="fig4"> Figure 4B </xref> ). One of these SNPs, detected in 10th exon, introduced a premature stop codon mutation that led to a truncated 521-aa GI <span class="trait">protein</span> in the <italic> <span class="gene">GmGIa</span> </italic> -Mi allele. This stop codon mutation was considered a candidate for a functional nucleotide polymorphism in <italic> <span class="gene">GmGIa</span> </italic> . A derived cleaved amplified polymorphic sequence (dCAPs) marker was developed to examine the identity of this stop codon mutation in other NILs for the <italic> E2 </italic> locus. <xref ref-type="bibr" rid="bib47"> M <sc> cBlain </sc> <italic> et al. </italic> (1987) </xref> performed testcross experiments using various combinations of NILs originating from Harosoy ( <italic> e2 </italic> / <italic> e2 </italic> ) and Clark ( <italic> E2 </italic> / <italic> E2 </italic> ). The genotypes of all isolines tested coincided well with the genotype of this diagnostic dCAPs marker ( <ext-link ext-link-type="uri" xlink:href="http://www.genetics.org/cgi/data/genetics.110.125062/DC1/4"> Figure S3 </ext-link> ). This result indicated that the gene responsible for the QTL of the <italic> FT2 </italic> was probably identical to the <italic> E2 </italic> gene and that a conserved mutation might have caused the early flowering phenotype in their recessive allele. To validate the significance of the mutation in the <italic> <span class="gene">GmGIa</span> </italic> gene, we screened a mutant line from the mutagen-treated libraries. The sequence of <italic> <span class="gene">GmGIa</span> </italic> in the wild-type Bay cultivar was completely identical to the <italic> E2 </italic> allele. One mutant line harboring a deletion in the 10th exon that caused a truncated <span class="trait">protein</span> (735 aa; <xref ref-type="fig" rid="fig4"> Figure 4B </xref> and <ext-link ext-link-type="uri" xlink:href="http://www.genetics.org/cgi/data/genetics.110.125062/DC1/5"> Figure S4 </ext-link> ) showed a significantly earlier flowering phenotype (39.8 &#x000b1; 1.16; <italic> n </italic> = 8) than the wild type (47.6 &#x000b1; 1.06; <italic> n </italic> = 29) under natural day-length conditions. These results indicated that the mutation in <italic> <span class="gene">GmGIa</span> </italic> causes the early flowering phenotype in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Gene structure of <italic> <span class="gene">GmGIa</span> </italic> in NIL 6-8- <italic> ft2 </italic> ( <italic> ft2 </italic> / <italic> ft2 </italic> ), NIL 6-8- <italic> FT2 </italic> ( <italic> FT2 </italic> / <italic> FT2 </italic> ), and the <italic> <span class="gene">GmGIa</span> </italic> mutant line ( <italic> E2 </italic> -mut). (A) Exons, a part of the 3&#x02032; UTR, and introns of the <italic> <span class="gene">GmGIa</span> </italic> gene in the 24- to 45-kbp region of MiB300H01 are indicated by solid boxes, an open box and lines, respectively. Units on the nucleotide sequence scale shown by the vertical lines represent 1 kbp. The location in the 5th intron of marker 4, originating from AFLP E60M38, is represented by the shaded bar. (B) The substitution of nucleotides and amino acid residues (in parentheses) in the coding sequence are displayed above the boxes that represent each exon. Nucleotide and amino acid changes in the case of nonsynonymous and nonsense mutations from the <italic> ft2 </italic> to the <italic> FT2 </italic> allele, on the basis of the start codon position, are shown. In the early flowering <italic> ft2 </italic> allele, a single nucleotide substitution causing a premature stop codon was discovered in the 10th exon. The mutant line, harboring a nucleotide deletion in the same exon as the <italic> ft2 </italic> allele, had a truncated GI <span class="trait">protein</span> that similarly caused the early flowering phenotype under natural day-length conditions. The truncated <span class="trait">protein</span> lengths of the three alleles are compared at the bottom.
					</p>
				
					<p>&nbsp;&nbsp;
						A redundancy in highly similar genes, which originate from genome duplication, can increase the chance of evolving subfunctionalized genes ( <xref ref-type="bibr" rid="bib3"> A <sc> dams </sc> and W <sc> endel </sc> 2005 </xref> ). Indeed, hexaploid wheat has a greater potential for successful adaptation to a wide range of environmental conditions than tetraploid wheat (D <sc> ubcovsky </sc> and <xref ref-type="bibr" rid="bib17"> D <sc> vorak </sc> 2007 </xref> ). The soybean genome is a typical paleopolyploid species, and the duplicated regions extend over the whole genome ( <xref ref-type="bibr" rid="bib68"> S <sc> hoemaker </sc> <italic> et al. </italic> 1996 </xref> ; <xref ref-type="bibr" rid="bib74"> T <sc> subokura </sc> <italic> et al. </italic> 2008 </xref> ; <xref ref-type="bibr" rid="bib66"> S <sc> chmutz </sc> <italic> et al. </italic> 2010 </xref> ). The soybean genome is considered to originate from two duplication events: the soybean lineage-specific paleotetraploidization [13&#x02013;15 million years ago (Mya)] and the early duplication in legumes that occurred near the origin of the papilionoid lineage (44&#x02013;59 Mya; <xref ref-type="bibr" rid="bib65"> S <sc> chlueter </sc> <italic> et al. </italic> 2004 </xref> ; <xref ref-type="bibr" rid="bib66"> S <sc> chmutz </sc> <italic> et al. </italic> 2010 </xref> ). Representative examples of such gene redundancy that affect the same agronomic trait are the two photoreceptor genes, <italic> <span class="gene">GmPhyA2</span> </italic> and - <italic> 3 </italic> , which affect the photoperiod-sensitive maturity loci, <italic> E4 </italic> and <italic> E3 </italic> , respectively ( <xref ref-type="bibr" rid="bib44"> L <sc> iu </sc> <italic> et al. </italic> 2008 </xref> ; <xref ref-type="bibr" rid="bib80"> W <sc> atanabe </sc> <italic> et al. </italic> 2009 </xref> ). In some cases, the <italic> E3 </italic> and <italic> E4 </italic> alleles are found to have overlapping functions in flowering responses depending on photo quality and length ( <xref ref-type="bibr" rid="bib13"> C <sc> ober </sc> <italic> et al. </italic> 1996 </xref> ; <xref ref-type="bibr" rid="bib1"> A <sc> be </sc> <italic> et al. </italic> 2003 </xref> ), and the geographical distribution of the recessive <italic> e4 </italic> allele is found to be restricted to the high-latitude regions of Japan ( <xref ref-type="bibr" rid="bib34"> K <sc> anazawa </sc> <italic> et al. </italic> 2009 </xref> ). These facts indicate that duplication and subfunctionalization of <italic> PhyA </italic> genes contribute to the divergence of flowering time and maturity in soybean. Such duplicated gene phenomena may well explain the situation of the <italic> <span class="gene">GmGI</span> </italic> s. Premature stop codon mutations in the <italic> GI </italic> genes, induced by various mutagens, were found to have severe effects on flowering as well as pleiotropic effects on other traits in Arabidopsis (A <sc> raki </sc> and <xref ref-type="bibr" rid="bib4"> K <sc> omeda </sc> 1993 </xref> ; <xref ref-type="bibr" rid="bib22"> F <sc> owler </sc> <italic> et al. </italic> 1999 </xref> ) and pea ( <xref ref-type="bibr" rid="bib26"> H <sc> echt </sc> <italic> et al. </italic> 2007 </xref> ). <italic> GI </italic> is associated not only with circadian rhythms ( <xref ref-type="bibr" rid="bib50"> M <sc> izoguchi </sc> <italic> et al. </italic> 2005 </xref> ; <xref ref-type="bibr" rid="bib45"> M <sc> artin </sc> -T <sc> ryon </sc> <italic> et al. </italic> 2007 </xref> ) but also with several phenotypes and physiological processes, including photomorphogenesis ( <xref ref-type="bibr" rid="bib58"> P <sc> altiel </sc> <italic> et al. </italic> 2006 </xref> ; <xref ref-type="bibr" rid="bib57"> O <sc> liverio </sc> <italic> et al. </italic> 2007 </xref> ), cold stress response ( <xref ref-type="bibr" rid="bib9"> C <sc> ao </sc> <italic> et al. </italic> 2005 </xref> ), oxidative stress tolerance ( <xref ref-type="bibr" rid="bib40"> K <sc> urepa </sc> <italic> et al. </italic> 1998 </xref> ; <xref ref-type="bibr" rid="bib10"> C <sc> ao </sc> <italic> et al. </italic> 2006 </xref> ), and starch accumulation ( <xref ref-type="bibr" rid="bib20"> E <sc> imert </sc> <italic> et al. </italic> 1995 </xref> ). Association analysis using the flowering gene in Arabidopsis has shown that the <italic> FRIGIDA </italic> and <italic> FLC </italic> genes, which are related to the vernalization pathway, contribute to geographic adaptation. However, SNPs in the <italic> GI </italic> gene, not related to the null allele, had only small effects on flowering time under LD conditions in a natural accession ( <xref ref-type="bibr" rid="bib19"> E <sc> hrenreich </sc> <italic> et al. </italic> 2009 </xref> ) or in inbred lines derived from intercrosses with multiple parental lines ( <xref ref-type="bibr" rid="bib38"> K <sc> over </sc> <italic> et al. </italic> 2009 </xref> ). Although <italic> GI </italic> overexpressing transgenic rice plants and specific RNA interference lines have been evaluated, a natural loss-of-function <italic> GI </italic> rice mutant has not yet been discovered. In barley, the correlation between the <italic> GI </italic> locus and some QTL related to flowering traits has also not yet been reported ( <xref ref-type="bibr" rid="bib18"> D <sc> unford </sc> <italic> et al. </italic> 2005 </xref> ). Compared with species that harbor a single <italic> GI </italic> gene in their genome, the loss of function of <italic> E2 </italic> in soybean has little influence on growth except for phenotypes directly associated with flowering. The <span class="trait">high</span> level of sequence similarity (97%) between <italic> <span class="gene">GmGIa</span> </italic> <span class="gene">(Glyma10g36600)</span> and <italic> <span class="gene">GmGIb</span> </italic> <span class="gene">(Glyma20g30980)</span> indicates that these genes originated from soybean lineage-specific duplication. Although we have no experimental data regarding the functions of <italic> <span class="gene">GmGIb</span> </italic> , it is likely that these duplicated soybean <italic> GI </italic> genes cooperate or functionally compensate for each other and that the loss of function of <italic> <span class="gene">GmGIa</span> </italic> in soybean varieties contributes to their geographic adaptation, as with the <italic> <span class="gene">GmPhyA</span> </italic> s. Studies of such duplicated genes, represented by <italic> <span class="gene">GmGI</span> </italic> s and <italic> <span class="gene">GmPhyA</span> </italic> s, especially in comparison with <italic> Vitis vinifera </italic> that contains no recent genome duplication ( <xref ref-type="bibr" rid="bib31"> J <sc> aillon </sc> <italic> et al. </italic> 2007 </xref> ) or with plants that have complex genomes, such as <italic> Zea mays </italic> , <italic> T. aestivum </italic> , and <italic> Populus trichocarpa </italic> , will provide further novel insights into the functions of these genes.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A novel Glycine soja tonoplast intrinsic <span class="trait">protein</span> gene responds to abiotic stress and depresses salt and dehydration tolerance in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:21397356  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Ji W, Bai X, Cai H, Zhu D, Sun XL, Chen LJ, Zhu YM.
				<br/> 
				年份：2011 Jul 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Tonoplast intrinsic <span class="trait">protein</span> (TIP) is a subfamily of the aquaporin (AQP), also known as major intrinsic <span class="trait">protein</span> (MIP) family, and regulates water movement across vacuolar membranes. Some reports have implied that <span class="gene">TIP</span> genes are associated with plant tolerance to some abiotic stresses that cause water loss, such as <span class="trait">drought</span> and <span class="trait">high</span> salinity. In our previous work, we found that an expressed sequence tag (EST) representing a <span class="gene">TIP</span> gene in our Glycine soja EST library was inducible by abiotic stresses. This <span class="gene">TIP</span> was subsequently isolated from G. soja with cDNA library screening, EST assembly and PCR, and named as GsTIP2;1. The expression patterns of GsTIP2;1 in G. soja under low temperature, salt and dehydration stress were different in leaves and roots. Though GsTIP2;1 is a stress-induced gene, overexpression of GsTIP2;1 in Arabidopsis thaliana depressed tolerance to salt and dehydration stress, but did not affect seedling growth under cold or favorable conditions. Higher dehydration speed was detected in Arabidopsis plants overexpressing GsTIP2;1, implying GsTIP2;1 might mediate stress sensitivity by enhancing water loss in the plant. Such a result is not identical to previous reports, providing some new information about the relationship between <span class="gene">TIP</span> and plant abiotic stress tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolutionary duplication of lipo-oligochitin-like receptor genes in soybean differentiates their function in cell division and cell invasion.</h2>
			<h3>PMID:21389773  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Indrasumunar A(1), Gresshoff PM.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Gene duplication in evolution has long been viewed as a mechanism for functional divergence. We recently cloned two related lipo-oligo-chitin receptor genes <span class="gene">(GmNFR1α</span> and <span class="gene">GmNFR1β)</span> in Glycine max(soybean) that allowed the distinction of two nodulation factor (NF) responses during early legume nodule ontogeny, namely invasion of the root hair and concomitant cortical cell divisions. Root-controlled <span class="gene">GmNFR1αmutants</span> nod49 and rj1 failed to form curled root hairs, infection threads and nodules but develop subepidermal cortical cell divisions (CCD) and mycorrhizal associations. In contrast <span class="gene">GmNFR1β</span> mutant PI437.654 had full symbiotic abilities. However, <span class="gene">GmNFR1α</span> mutants formed normal nodules at reduced frequency when inoculated with <span class="trait">high</span> Bradyrhizobium titers. The mutation was complemented in Agrobacterium rhizogenes K599 transformed roots using both CaMV 35S and the native <span class="gene">GmNFR1promoters.</span> <span class="gene">GmNFR1α</span> may encode a <span class="trait">high</span> affinity NF receptor responsible for the entire nodulation cascade while <span class="gene">GmNFR1β</span> with lower affinity to NF suffices to induce cell divisions but not early infection events.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization, gene cloning, and heterologous expression of β-mannanase from a thermophilic Bacillus subtilis.</h2>
			<h3>PMID:21369984  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Summpunn P(1), Chaijan S, Isarangkul D, Wiyakrutta S, Meevootisom V.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacillus subtilis BCC41051 producing a thermostable β-mannanase was isolated from soybean meal-enriched soil and was unexpectedly found to be thermophilic in nature. The extracellular β-mannanase (ManA) produced was hydrophilic, as it was not precipitated even with ammonium sulfate at 80% saturation. The estimated molecular weight of ManA was 38.0 kDa by SDS-PAGE with a <span class="gene">pi</span> value of 5.3. Optimal pH and temperature for mannan-hydrolyzing activity was 7.0 and 60°C, respectively. The enzyme was stable over a pH range of 5.0-11.5, and at temperatures of up to 60°C for 30 min, with more than 80% of its activity retained. ManA was strongly inhibited by Hg(2+) (1 mM), but was sensitive to other divalent ions to a lesser degree. The gene of ManA encoded a <span class="trait">protein</span> of 362 amino acid residues, with the first 26 residues identified as a signal peptide. <span class="trait">High</span> expression of recombinant ManA was achieved in both Escherichia coli BL21 (DE3) (415.18 U/ml) and B. megaterium UNcat (359 U/ml).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phylogeny of nodulation and nitrogen-fixation genes in Bradyrhizobium: supporting evidence for the theory of monophyletic origin, and spread and maintenance by both horizontal and vertical transfer.</h2>
			<h3>PMID:21357454  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Menna P(1), Hungria M.
				<br/> 
				年份：2011 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacteria belonging to the genus Bradyrhizobium are capable of establishing symbiotic relationships with a broad range of plants belonging to the three subfamilies of the family Leguminosae (=Fabaceae), with the formation of specialized structures on the roots called nodules, where fixation of atmospheric nitrogen takes place. Symbiosis is under the control of finely tuned expression of common and host-specific nodulation genes and also of genes related to the assembly and activity of the nitrogenase, which, in Bradyrhizobium strains investigated so far, are clustered in a symbiotic island. Information about the diversity of these genes is essential to improve our current poor understanding of their origin, spread and maintenance and, in this study, we provide information on 40 Bradyrhizobium strains, mostly of tropical origin. For the nodulation trait, common (nodA), Bradyrhizobium-specific (nodY/K) and host-specific (nodZ) nodulation genes were studied, whereas for fixation ability, the diversity of <span class="gene">nifH</span> was investigated. In general, clustering of strains in all nod and <span class="gene">nifH</span> trees was similar and the Bradyrhizobium group could be clearly separated from other rhizobial genera. However, the congruence of nod and nif genes with ribosomal and housekeeping genes was low. <span class="gene">nodA</span> and nodY/K were not detected in three strains by amplification or hybridization with probes using Bradyrhizobium japonicum and Bradyrhizobium elkanii type strains, indicating the <span class="trait">high</span> diversity of these genes or that strains other than photosynthetic Bradyrhizobium must have alternative mechanisms to initiate the process of nodulation. For a large group of strains, the <span class="trait">high</span> diversity of nod genes (with an emphasis on nodZ), the low relationship between nod genes and the host legume, and some evidence of horizontal gene transfer might indicate strategies to increase host range. On the other hand, in a group of five symbionts of Acacia mearnsii, the <span class="trait">high</span> congruence between nod and ribosomal/housekeeping genes, in addition to shorter nodY/K sequences and the absence of nodZ, highlights a co-evolution process. Additionally, in a group of B. japonicum strains that were symbionts of soybean, vertical transfer seemed to represent the main genetic event. In conclusion, clustering of <span class="gene">nodA</span> and <span class="gene">nifH</span> gives additional support to the theory of monophyletic origin of the symbiotic genes in Bradyrhizobium and, in addition to the analysis of nodY/K and nodZ, indicates spread and maintenance of nod and nif genes through both vertical and horizontal transmission, apparently with the dominance of one or other of these events in some groups of strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy <span class="trait">protein</span> retards the progression of non-alcoholic steatohepatitis via improvement of insulin <span class="trait">resistance</span> and steatosis.</h2>
			<h3>PMID:21333494  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang HY(1), Tzeng YH, Chai CY, Hsieh AT, Chen JR, Chang LS, Yang SS.
				<br/> 
				年份：2011 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: Non-alcoholic steatohepatitis (NASH) is a common cause of liver disease, and it may progress to fibrosis or cirrhosis. The aim of this study was to investigate the effects of soy <span class="trait">protein</span> on hepatic steatosis and insulin <span class="trait">resistance</span> in NASH. METHODS: Forty male Sprague-Dawley rats were fed a high-fat diet for 4 wk to induce NASH and then were allocated to one of four diets: a NASH-inducing diet, a standard diet, a NASH-inducing diet plus soy <span class="trait">protein,</span> and a standard diet plus soy <span class="trait">protein.</span> RESULTS: After the 10-wk experimental period, the results showed that soy <span class="trait">protein</span> significantly lowered plasma cholesterol concentrations and body fat accumulation. Soy <span class="trait">protein</span> intake also decreased the hepatic lipid depots of triacylglycerols and cholesterol and decreased the concentrations of lipid peroxides. In an analysis of antioxidative status, rats fed the soy <span class="trait">protein</span> diet showed improved antioxidative potential due to increases in superoxide dismutase and catalase activities and a decrease in the <span class="trait">protein</span> expression of cytochrome <span class="gene">P450</span> 2E1. CONCLUSION: Soy <span class="trait">protein</span> may improve the liver function in patients with NASH by lowering lipid levels in the blood and liver, increasing the antioxidative capacity, and improving insulin <span class="trait">resistance.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">GmbZIP1</span> transcription factor enhances multiple abiotic stress tolerances in transgenic plants.</h2>
			<h3>PMID:21331631  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao SQ(1), Chen M, Xu ZS, Zhao CP, Li L, Xu HJ, Tang YM, Zhao X, Ma YZ.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Abscisic acid (ABA)-responsive element binding proteins (AREBs) are basic domain/leucine zipper transcription factors that bind to the ABA-responsive element (ABRE) in the promoter regions of ABA-inducible genes in plants. A novel bZIP transcription factor gene, <span class="gene">GmbZIP1,</span> encoding 438 amino acids with a conserved bZIP domain composed of 60 amino acids was isolated from salt-tolerant soybean cv. Tiefeng 8. Southern blotting showed that only one copy was present in the soybean genome. Phylogenetic analyses showed that <span class="gene">GmbZIP1</span> belonged to the AREB subfamily of the bZIP family and was most closely related to AtABF2 and OsTRAB1. The expression of <span class="gene">GmbZIP1</span> was highly induced by ABA, <span class="trait">drought,</span> <span class="trait">high</span> salt and low temperature; and <span class="gene">GmbZIP1</span> was expressed in soybean roots, stems and leaves under different stress conditions. <span class="gene">GmbZIP1</span> was localized inside the nuclei of transformed onion epidermal cells. Overexpression of <span class="gene">GmbZIP1</span> enhanced the responses of transgenic plants to ABA and triggered stomatal closure under stresses, potentially leading to improved tolerances to several abiotic stresses such as <span class="trait">high</span> salt, low temperature and <span class="trait">drought</span> in transgenic plants. Furthermore, overexpression of <span class="gene">GmbZIP1</span> affected the expression of some ABA or stress-related genes involved in regulating stomatal closure in Arabidopsis under ABA, <span class="trait">drought</span> and <span class="trait">high</span> salt stress conditions. A few AREB elements were detected in the promoter region of those ABA or stress-related genes, suggesting that <span class="gene">GmbZIP1</span> regulates the ABA response or stomatal closure mediated by those downstream genes in transgenic Arabidopsis. Moreover, <span class="gene">GmbZIP1</span> was used to improve the <span class="trait">drought</span> tolerance trait of Chinese wheat varieties BS93. Functional analysis showed that overexpression of <span class="gene">GmbZIP1</span> enhanced the <span class="trait">drought</span> tolerance of transgenic wheat, and transcripts of <span class="gene">GmbZIP1</span> were detected in transgenic wheat using RT-PCR. In addition, <span class="gene">GmbZIP1</span> overexpression did not result in growth retardation in all transgenic plants, suggesting that <span class="gene">GmbZIP1</span> may be a valuable genetic resource for engineering stress tolerance of crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:pHairyRed: a novel binary vector containing the DsRed2 reporter gene for visual selection of transgenic hairy roots.</h2>
			<h3>PMID:21324970  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin MH(1), Gresshoff PM, Indrasumunar A, Ferguson BJ.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We developed a new plant transformation vector, pHairyRed, for enabling <span class="trait">high</span> throughput, non-destructive selection of Agrobacterium rhizogenes-mediated 'hairy-root' transformation. pHairyRed allows easy in planta visualization of transgenic tissue with minimal disturbance to the plant. The DsRed2 reporter gene, encoding a red fluorescent <span class="trait">protein,</span> was cloned to <span class="trait">yield</span> pHairyRed (harbouring a multiple cloning site), which was used with the highly efficient K599 A. rhizogenes strain to infect soybean (Glycine max L. Merrill) plants. DsRed2 fluorescence was easily detected in planta for the duration of a 5-week study with negligible levels of background autofluorescence. This enabled visual selection of transformed roots and subsequent excission of non-transformed roots. pHairyRed-transformed roots nodulated normally when inoculated with Bradyrhizobium japonicum. Within the nodule, DsRed2 fluorescence was plant-specific, being absent in the bacteroid-dominated nodule infected zone. To test the reliability of pHairyRed as a high-fidelity binary vector reporter system, the gene encoding the soybean Nod factor receptor, <span class="gene">GmNFR1α,</span> was cloned into the vector for use in a complementation study with a non-nodulating nfr1α mutant of soybean. Complementation was achieved and, without exception, DsRed2 fluorescence was detected in all hairy roots that successfully formed nodules (100%, n = 34). We anticipate broad application of this reporter system for the further analysis of root-related events in soybean and related legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic and genomic analyses of a fast neutron mutant population resource in soybean.</h2>
			<h3>PMID:21321255  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bolon YT(1), Haun WJ, Xu WW, Grant D, Stacey MG, Nelson RT, Gerhardt DJ, Jeddeloh JA, Stacey G, Muehlbauer GJ, Orf JH, Naeve SL, Stupar RM, Vance CP.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mutagenized populations have become indispensable resources for introducing variation and studying gene function in plant genomics research. In this study, fast neutron (FN) radiation was used to induce deletion mutations in the soybean (Glycine max) genome. Approximately 120,000 soybean seeds were exposed to FN radiation doses of up to 32 Gray units to develop over 23,000 independent M2 lines. Here, we demonstrate the utility of this population for phenotypic screening and associated genomic characterization of striking and agronomically important traits. Plant variation was cataloged for seed composition, maturity, morphology, pigmentation, and nodulation traits. Mutants that showed significant increases or decreases in seed <span class="trait">protein</span> and oil content across multiple generations and environments were identified. The application of comparative genomic hybridization (CGH) to lesion-induced mutants for deletion mapping was validated on a midoleate x-ray mutant, M23, with a known <span class="gene">FAD2-1A</span> (for fatty acid desaturase) gene deletion. Using CGH, a subset of mutants was characterized, revealing deletion regions and candidate genes associated with phenotypes of interest. Exome resequencing and sequencing of PCR products confirmed FN-induced deletions detected by CGH. Beyond characterization of soybean FN mutants, this study demonstrates the utility of CGH, exome sequence capture, and next-generation sequencing approaches for analyses of mutant plant genomes. We present this FN mutant soybean population as a valuable public resource for future genetic screens and functional genomics research.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the recombinant copper chaperone (CCS) from the plant Glycine (G.) max.</h2>
			<h3>PMID:21264427  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sagasti S(1), Yruela I, Bernal M, Lujan MA, Frago S, Medina M, Picorel R.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The goal of the present work was to characterize the recombinant copper chaperone (CCS) from soybean. Very little is known about plant copper chaperones, which makes this study of current interest, and allows for a comparison with the better known homologues from yeast and humans. To obtain sizeable amounts of pure <span class="trait">protein</span> suitable for spectroscopic characterization, we cloned and overexpressed the G. max <span class="gene">CCS</span> chaperone in E. coli in the presence of 0.5 mM CuSO(4) and 0.5 mM ZnSO(4) in the broth. A pure <span class="trait">protein</span> preparation was obtained by using two IMAC steps and pH gradient chromatography. Most of the proteins were obtained as apo-form, devoid of copper atoms. The chaperone showed a <span class="trait">high</span> content (i.e., over 40%) of loops, turns and random coil as determined both by circular dichroism and homology modelling. The homology 3-D structural model suggests the <span class="trait">protein</span> might fold in three structural <span class="trait">protein</span> domains. The 3-D model along with the primary structure and spectroscopic data may suggest that copper atoms occupy the two metal binding sites, MKCEGC and CTC, within the N-terminal domain I and C-terminal domain III, respectively. But only one Zn-binding site was obtained spectroscopically.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean β-expansin gene <span class="gene">GmEXPB2</span> intrinsically involved in root system architecture responses to abiotic stresses.</h2>
			<h3>PMID:21261763  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo W(1), Zhao J, Li X, Qin L, Yan X, Liao H.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Root system architecture responds plastically to some abiotic stresses, including phosphorus (P), iron (Fe) and water deficiency, but its response mechanism is still unclear. We cloned and characterized a vegetative β-expansin gene, <span class="gene">GmEXPB2,</span> from a Pi starvation-induced soybean cDNA library. Transient expression of 35S::GmEXPB2-GFP in onion epidermal cells verified that <span class="gene">GmEXPB2</span> is a secretory <span class="trait">protein</span> located on the cell wall. <span class="gene">GmEXPB2</span> was found to be primarily expressed in roots, and was highly induced by Pi starvation, and the induction pattern was confirmed by GUS staining in transgenic soybean hairy roots. Results from intact soybean composite plants either over-expressing <span class="gene">GmEXPB2</span> or containing knockdown constructs, showed that <span class="gene">GmEXPB2</span> is involved in hairy root elongation, and subsequently affects plant growth and P uptake, especially at low P levels. The results from a heterogeneous transformation system indicated that over-expressing <span class="gene">GmEXPB2</span> in Arabidopsis increased root cell division and elongation, and enhanced plant growth and P uptake at both low and <span class="trait">high</span> P levels. Furthermore, we found that, in addition to Pi starvation, <span class="gene">GmEXPB2</span> was also induced by Fe and mild water deficiencies. Taken together, our results suggest that <span class="gene">GmEXPB2</span> is a critical root β-expansin gene that is intrinsically involved in root system architecture responses to some abiotic stresses, including P, Fe and water deficiency. In the case of Pi starvation responses, <span class="gene">GmEXPB2</span> may enhance both P efficiency and P responsiveness by regulating adaptive changes of the root system architecture. This finding has great agricultural potential for improving crop P uptake on both low-P and P-fertilized soils.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protein and metabolite composition of xylem sap from field-grown soybeans (Glycine max).</h2>
			<h3>PMID:21246215  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Krishnan HB(1), Natarajan SS, Bennett JO, Sicher RC.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The xylem, in addition to transporting water, nutrients and metabolites, is also involved in long-distance signaling in response to pathogens, symbionts and environmental stresses. Xylem sap has been shown to contain a number of proteins including metabolic enzymes, stress-related proteins, signal transduction proteins and putative transcription factors. Previous studies on xylem sap have mostly utilized plants grown in controlled environmental chambers. However, plants in the field are subjected to <span class="trait">high</span> light and to environmental stress that is not normally found in growth chambers. In this study, we have examined the <span class="trait">protein</span> and metabolite composition of xylem sap from field-grown cultivated soybean plants. One-dimensional gel electrophoresis of xylem sap from determinate, indeterminate, nodulating and non-nodulating soybean cultivars revealed similar <span class="trait">protein</span> profiles consisting of about 8-10 prominent polypeptides. Two-dimensional gel electrophoresis of soybean xylem sap resulted in the visualization of about 60 distinct <span class="trait">protein</span> spots. A total of 38 <span class="trait">protein</span> spots were identified using MALDI-TOF <span class="gene">MS</span> and LC-MS/MS. The most abundant proteins present in the xylem sap were identified as 31 and 28 kDa vegetative storage proteins. In addition, several proteins that are conserved among different plant species were also identified. Diurnal changes in the metabolite profile of xylem sap collected during a 24-h cycle revealed that asparagine and aspartate were the two predominant amino acids irrespective of the time collected. Pinitol (D-3-O-methyl-chiro-inositol) was the most abundant carbohydrate present. The possible roles of xylem sap proteins and metabolites as nutrient reserves for sink tissue and as an indicator of biotic stress are also discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Subjects with elevated LDL cholesterol and metabolic syndrome benefit from supplementation with soy <span class="trait">protein,</span> phytosterols, hops <span class="gene">rho</span> iso-alpha acids, and Acacia nilotica proanthocyanidins.</h2>
			<h3>PMID:21122628  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lerman RH(1), Minich DM, Darland G, Lamb JJ, Chang JL, Hsi A, Bland JS, Tripp ML.
				<br/> 
				年份：2010 Jan-Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Metabolic syndrome is associated with increased cardiovascular disease (CVD) risk, a risk that is significantly increased when accompanied by elevated low-density lipoprotein cholesterol (LDL-C). Whereas lifestyle therapies are the initial intervention of choice for both of these risk factors, it has not been clearly determined that this approach is efficacious when they occur concomitantly. OBJECTIVE: To evaluate effects of supplementing a lifestyle program with a medical food and nutraceutical in individuals with metabolic syndrome and elevated LDL-C. METHODS: We conducted a subgroup analysis of a 12-week, randomized trial in adults with metabolic syndrome; data from those with LDL-C ≥ 160 mg/dL were analyzed. Control-arm subjects were instructed to consume a modified Mediterranean-style, low-glycemic-load diet (MED, n = 12). Treatment-arm subjects received a phytochemical-enhanced diet (PED, n = 12) consisting of the same low-glycemic-load diet plus a medical food containing soy <span class="trait">protein</span> and plant sterols and a nutraceutical containing hops <span class="gene">rho</span> iso-alpha acids and acacia proanthocyanidins. All subjects received identical aerobic exercise counseling. RESULTS: At 12 weeks, mean weight loss did not differ between arms. However, the PED arm exhibited greater improvement than the MED arm (P &lt; .05) in total cholesterol, LDL-C, non-high-density lipoprotein cholesterol (non-HDL-C), cholesterol/HDL-C, triglyceride/HDL-C, apolipoprotein (apo) B, apo B/apo A-1, homocysteine, total LDL particle number, and large HDL particle number. All individuals in the PED arm but only one third in the MED arm achieved LDL-C levels &lt; 160 mg/dL. CONCLUSION: Individuals at <span class="trait">high</span> CVD risk benefit from a soy/phytosterol containing medical food and phytochemical supplemented lifestyle program.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine genetic mapping of the genomic region controlling leaflet shape and number of seeds per pod in the soybean.</h2>
			<h3>PMID:21104397  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong N(1), Moon JK, Kim HS, Kim CG, Jeong SC.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Narrow leaflet cultivars tend to have more seeds per pod than broad leaflet cultivars in soybean [Glycine max (L.) Merr.], which suggests that the leaflet-shape trait locus is tightly linked to or cosegregates with the trait locus controlling the number of seeds per pod (NSPP). Here, we attempted to further elucidate the relationship between leaflet shape and NSPP. A BC(3)F(2) population from a cross between the 'Sowon' (narrow leaflets and <span class="trait">high</span> NSPP) and 'V94-5152' (broad leaflets and low NSPP) variants was used. The results of the molecular genetic analyses indicated that, although the NSPP characteristic, in particular, the occurrence of 4-seeded pods, is governed by additional modifying genes that are likely present in Sowon, the two traits cosegregate in the BC(3)F(2) population. The mapping results generated using public markers demonstrated that the narrow leaflet-determining gene in Sowon is an allele of the previously highly studied <span class="gene">ln</span> gene on chromosome 20. A high-resolution map delimited the genomic region controlling both the leaflet shape and NSPP traits to a sequence length of 66 kb, corresponding to 0.7 cM. Among the three genes annotated in this 66 kb region, <span class="gene">Glyma20g25000.1</span> appeared to be a good candidate for the Ln-encoding gene, owing to its 47.8% homology with the <span class="trait">protein</span> encoding for the JAGGED gene that regulates lateral organ development in Arabidopsis. Taken together, our results suggested that phenotypic variations for narrow leaflet and NSPP are predominantly from the pleiotropic effects of the <span class="gene">ln</span> gene. Thus, our results should provide a molecular framework for soybean breeding programs with the objective of improving soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich <span class="trait">protein)</span> gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genes coding for PGYRPs (proline-, glycine- and tyrosine-rich proteins) are widely distributed across eukaryotes and have been proposed to have critical role in plant development, especially in response to environmental stresses. In this study, total of 12 soybean PGYRPs <span class="gene">(GmPGYRP1-12)</span> were identified from the soybean genome database for the first time and full-length cDNA and DNA sequences of <span class="gene">GmPGYRP7</span> was cloned. <span class="gene">GmPGYRP1-12</span> genes encoded a set of small predicted proteins (&lt;120 aa) with molecular mass of 7.20-13.29 kDa and isoelectric point of 4.06-6.57. All <span class="gene">GmPGYRPs</span> contained three exons and two introns with fixed occurring sites within genomic DNA sequences. In the putative <span class="gene">GmPGYRP</span> sequences, 4 amino acids (proline, glycine, tyrosine, and glutamine) account for more than 39% of the total <span class="trait">protein</span> composition. <span class="gene">GmPGYRPs</span> had a relatively flexible GYPPX motif followed by a highly conserved cysteine-rich domain (GCLAAXCCCCXLXC) and showed <span class="trait">high</span> similarity to other known PGYRPs, especially in C-terminal region. Most of PGYRPs can be divided into five subgroups according to phylogenetic analysis. The transcripts of <span class="gene">GmPGYRP1,</span> 3, 5, and 7, representing different PGYRP subgroups, appeared in different organs including seedling leaves, stems, roots, flowers, and developing seeds, but mainly accumulated in seedling roots. Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage. Our data suggest that <span class="gene">GmPGYRP</span> genes encoding a class of conservative XYPPX-repeat proteins probably play an important role in plant development as well as in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of <span class="gene">GmCaMK1,</span> a member of a soybean calmodulin-binding receptor-like kinase family.</h2>
			<h3>PMID:21056039  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：DeFalco TA(1), Chiasson D, Munro K, Kaiser BN, Snedden WA.
				<br/> 
				年份：2010 Dec 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calmodulin(CaM)-regulated <span class="trait">protein</span> phosphorylation forms an important component of Ca(2+) signaling in animals but is less understood in plants. We have identified a CaM-binding receptor-like kinase from soybean nodules, <span class="gene">GmCaMK1,</span> a homolog of Arabidopsis CRLK1. We delineated the CaM-binding domain (CaMBD) of <span class="gene">GmCaMK1</span> to a 24-residue region near the C-terminus, which overlaps with the kinase domain. We have demonstrated that <span class="gene">GmCaMK1</span> binds <span class="gene">CaM</span> with <span class="trait">high</span> affinity in a Ca(2+)-dependent manner. We showed that <span class="gene">GmCaMK1</span> is expressed broadly across tissues and is enriched in roots and developing nodules. Finally, we examined the CaMBDs of the five-member <span class="gene">GmCaMK</span> family in soybean, and orthologs present across taxa.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RIN4-like proteins mediate <span class="trait">resistance</span> protein-derived soybean defense against Pseudomonas syringae.</h2>
			<h3>PMID:21051954  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Selote D(1), Kachroo A.
				<br/> 
				年份：2010 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> (R) <span class="trait">protein</span> mediated recognition of pathogen avirulence effectors triggers signaling that induces a very robust form of species-specific immunity in plants. The soybean Rpg1-b <span class="trait">protein</span> mediates this form of <span class="trait">resistance</span> against the bacterial blight pathogen, Pseudomonas syringae expressing AvrB Pgyrace4. Likewise, the Arabidopsis RPM1 <span class="trait">protein</span> also mediates species-specific <span class="trait">resistance</span> against AvrB expressing bacteria. RPM1 and Rpg1-b are non-orthologous and differ in their requirements for downstream signaling components. We recently showed that the activation of Rpg1-b derived <span class="trait">resistance</span> signaling requires two host proteins that directly interact with AvrB. These proteins share <span class="trait">high</span> sequence similarity with the Arabidopsis RPM1 interacting <span class="trait">protein</span> 4 (RIN4), which is essential for RPM1-derived <span class="trait">resistance.</span> The two soybean RIN4-like proteins <span class="gene">(GmRIN4a</span> and b) differ in their abilities to interact with Rpg1-b as well as to complement the Arabidopsis rin4 mutation. Because the two <span class="gene">GmRIN4</span> proteins interact with each other, we proposed that they might function as a heteromeric complex in mediating Rpg1-b-derived <span class="trait">resistance.</span> Absence of <span class="gene">GmRIN4a</span> or b enhanced basal <span class="trait">resistance</span> against bacterial and oomycete pathogens in soybean. Lack of <span class="gene">GmRIN4a</span> also enhanced the virulence of avrB bacteria in plants lacking Rpg1-b. Our studies suggest that multiple RIN4-like proteins proteins mediate R-mediated signaling, in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structural stability of soybean (Glycine max) α-amylase: properties of the unfolding transition studied with fluorescence and CD spectroscopy.</h2>
			<h3>PMID:20955173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kumari A(1), Rosenkranz T, Fitter J, Kayastha AM.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Stability and unfolding of mammalian and microbial α-amylases have been intensively investigated. However, there is only limited information available on the structural stability of plant α-amylases, namely of the two isoenzymes from barley <span class="gene">AMY1</span> and AMY2, of the α-amylase from mung bean (Vigna radiata), and of the α-amylase from malted sorghum (Sorghum bicolor). We report here the stability of soyabean α-amylase <span class="gene">(GMA),</span> against elevated temperatures and chemical denaturants (GndHCl) by employing circular dichroism and fluorescence spectroscopy. Since it is well-known that calcium ions play a crucial role for enzymatic activity and stability of a-amylases, we performed our studies with calcium bound and calcium free <span class="gene">GMA.</span> The thermal unfolding transition temperature decreased from 72°C for calcium saturated samples to 57°C for the case of calcium depleted <span class="gene">GMA.</span> Similarly, the GndHCl transition concentration was lowered from 0.70 M for calcium bound <span class="gene">GMA</span> to 0.41 M in the absence of calcium. Thermal unfolding of <span class="gene">GMA</span> irreversible due to aggregation of the unfolded state. <span class="gene">GMA</span> unfolded in 6 M GndHCl shows <span class="trait">high</span> degree of reversibility after diluting the unfolded enzyme in native buffer containing 7 M glycerol. Furthermore, the refolded enzyme showed 93% of activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression of three fatty acid desaturase genes from cold-sensitive lima bean (Phaseolus lunatus L.).</h2>
			<h3>PMID:20953666  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang YM(1), Wang CC, Hu HH, Yang L.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The critical chilling temperature of cold-sensitive lima bean (Phaseolus lunatus L.) seedlings was determined to be approx. 8°C. The full-length cDNAs of PlSAD, PlFAD2, and PlFAD3 encoding three fatty acid desaturases were isolated from lima bean leaves. The PlSAD open reading frame (ORF) had 1,317 bp and a single intron of 601 bp, encoding a polypeptide of 438 amino acids that showing 86% homology with the plastidial stearoyl-acyl carrier <span class="trait">protein</span> desaturase in soybean. The PlFAD2 ORF contained 1,059 bp and was uninterrupted, encoding a polypeptide of 352 amino acids having 85% identity with the microsomal omega-6 desaturase <span class="gene">GmFAD2-3</span> in soybean. PlFAD3 ORF had 1,116 bp and seven introns, encoding a polypeptide of 371 amino acids showing 91% homology with the microsomal omega-3 desaturase <span class="gene">FAD3</span> in cowpea. PlSAD and PlFAD2 were expressed highly in leaves but poorly in roots and stems, while PlFAD3 was expressed in three tissues. All three genes were significantly induced in leaves by <span class="trait">drought.</span> PlSAD and PlFAD3 in leaves were down-regulated by <span class="trait">high</span> temperature and salinity, whereas PlFAD2 was up-regulated by the two stresses. The expression patterns of the three genes in lima bean leaves under suboptimal temperature, 18°C, were different from those under 8°C. The down-regulation of PlSAD transcript at chilling temperature might be an important factor contributing to chilling susceptibility for lima bean plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:On the role of molecular oxygen in lipoxygenase activation: comparison and contrast of epidermal lipoxygenase-3 with soybean lipoxygenase-1.</h2>
			<h3>PMID:20923767  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng Y(1), Brash AR.
				<br/> 
				年份：2010 Dec 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The oxygenation of polyunsaturated fatty acids by lipoxygenases (LOX) is associated with a lag phase during which the resting ferrous enzyme is converted to the active ferric form by reaction with fatty acid hydroperoxide. Epidermal lipoxygenase-3 (eLOX3) is atypical in displaying hydroperoxide isomerase activity with fatty acid hydroperoxides through cycling of the ferrous enzyme. Yet eLOX3 is capable of dioxygenase activity, albeit with a long lag phase and need for <span class="trait">high</span> concentrations of hydroperoxide activator. Here, we show that higher O(2) concentration shortens the lag phase in eLOX3, although it reduces the rate of hydroperoxide consumption, effects also associated with an A451G mutation known to affect the disposition of molecular oxygen in the <span class="gene">LOX</span> active site. These observations are consistent with a role of O(2) in interrupting hydroperoxide isomerase cycling. Activation of eLOX3, A451G eLOX3, and soybean LOX-1 with 13-hydroperoxy-linoleic acid forms oxygenated end products, which we identified as 9R- and 9S-hydroperoxy-12S,13S-trans-epoxyoctadec-10E-enoic acids. We deduce that activation partly depends on reaction of O(2) with the intermediate of hydroperoxide cleavage, the epoxyallylic radical, giving an epoxyallylic peroxyl radical that does not further react with Fe(III)-OH; instead, it dissociates and leaves the enzyme in the activated free ferric state. eLOX3 differs from soybean LOX-1 in more tightly binding the epoxyallylic radical and having limited access to O(2) within the active site, leading to a deficiency in activation and a dominant hydroperoxide isomerase activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary soy <span class="trait">protein</span> benefit in experimental kidney disease is preserved after isoflavone depletion of diet.</h2>
			<h3>PMID:20921276  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ogborn MR(1), Nitschmann E, Bankovic-Calic N, Weiler HA, Aukema HM.
				<br/> 
				年份：2010 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soy diet ameliorates renal injury in the Han:SPRD-cy rat. The relative roles of <span class="trait">protein,</span> isoflavones and changes in polyunsaturated fatty acid (PUFA) status are not determined. We fed male Han:SPRD-cy heterozygotes casein (C), <span class="trait">high</span> isoflavone soy <span class="trait">protein</span> (HIS), alcohol-extracted low isoflavone soy <span class="trait">protein</span> (LIS) or mixed soy <span class="trait">protein</span> diet (MIS). LIS and MIS were associated with a small decrease in animal weight compared with <span class="gene">HIS</span> or C. Soy diets preserved normal renal function and reduced relative renal weight (10.9-14.6 g/kg, cf. 23.6, P &lt; 0.001), scores for cystic change (0.168-0.239, cf. 0.386, P &lt; 0.05), fibrosis (0.013-0.015, cf. 0.032, P &lt; 0.05), tissue oxidized LDL content (0.012-0.021, cf. 0.048, P &lt; 0.05), inflammation (8.5-12.9, cf. 31.2, P &lt; 0.05) and epithelial cell proliferation (6.5-13.8, cf. 26.3, P &lt; 0.05). In post hoc testing, LIS produced a greater reduction in relative renal weight, cystic change and epithelial proliferation, whereas <span class="gene">HIS</span> produced a significantly greater reduction in oxidized-LDL. Soy diets were associated with increased hepatic content of 18C PUFA (P &lt; 0.001). LIS and <span class="gene">HIS</span> diets were associated with a small increase in body fat content (P &lt; 0.001). Alcohol-extracted soy <span class="trait">protein</span> retains its major protective effects in this model with subtle differences attributable to isoflavones.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two coordinately regulated homologs of FLOWERING LOCUS T are involved in the control of photoperiodic flowering in soybean.</h2>
			<h3>PMID:20864544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kong F(1), Liu B, Xia Z, Sato S, Kim BM, Watanabe S, Yamada T, Tabata S, Kanazawa A, Harada K, Abe J.
				<br/> 
				年份：2010 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						FLOWERING LOCUS T (FT) is a key flowering integrator in Arabidopsis (Arabidopsis thaliana), with homologs that encode florigens in many plant species regardless of the type of photoperiodic response. We identified 10 FT homologs, which were arranged as five pairs of linked genes in different homoeologous chromosomal regions, in soybean (Glycine max), a paleopolyploid species. Two of the FT homologs, <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a,</span> were highly up-regulated under short-day (SD) conditions (inductive for flowering in soybean) and had diurnal expression patterns with the highest expression 4 h after dawn. Under long-day (LD) conditions, expression of <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a</span> was down-regulated and did not follow a diurnal pattern. Flowering took much longer to initiate under LD than under SD, and only the <span class="gene">GmFT5a</span> transcript accumulated late in development under LD. Ectopic expression analysis in Arabidopsis confirmed that both <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a</span> had the same function as Arabidopsis FT, but the effect of <span class="gene">GmFT5a</span> was more prominent. A double-mutant soybean line for two PHYTOCHROME A (PHYA) genes expressed <span class="trait">high</span> levels of <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a</span> under LD, and it flowered slightly earlier under LD than the wild type grown under SD. The expression levels of <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a</span> were regulated by the PHYA-mediated photoperiodic regulation system, and the <span class="gene">GmFT5a</span> expression was also regulated by a photoperiod-independent system in LD. Taken together, our results suggest that <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a</span> coordinately control flowering and enable the adaptation of soybean to a wide range of photoperiodic environments.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mutant alleles of <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> combine to produce soybeans with the <span class="trait">high</span> oleic acid seed oil trait</h2>
			<h3>PMID:20828382</h3>
				

				
					<h3>PMCID:2956544</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Pham </surname>  <given-names> Anh-Tung </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> tapgw7@mail.mizzou.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Lee </surname>  <given-names> Jeong-Dong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jdlee@knu.ac.kr </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Shannon </surname>  <given-names> J Grover </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> shannong@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Bilyeu </surname>  <given-names> Kristin D </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> Kristin.Bilyeu@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 9 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The alteration of fatty acid profiles in soybean [Glycine max (L.) Merr.] to improve soybean oil quality is an important and evolving theme in soybean research to meet nutritional needs and industrial criteria in the modern market. Soybean oil with elevated oleic acid is desirable because this monounsaturated fatty acid improves the nutrition and oxidative stability of the oil. Commodity soybean oil typically contains 20% oleic acid and the target for <span class="trait">high</span> oleic acid soybean oil is approximately 80% of the oil; previous conventional plant breeding research to raise the oleic acid level to just 50-60% of the oil was hindered by the genetic complexity and environmental instability of the trait. The objective of this work was to create the <span class="trait">high</span> oleic acid trait in soybeans by identifying and combining mutations in two delta-twelve fatty acid desaturase genes, <span class="gene">FAD2-1A</span> and FAD2-1B. RESULTS: Three polymorphisms found in the <span class="gene">FAD2-1B</span> alleles of two soybean lines resulted in missense mutations. For each of the two soybean lines, there was one unique amino acid change within a highly conserved region of the <span class="trait">protein.</span> The mutant <span class="gene">FAD2-1B</span> alleles were associated with an increase in oleic acid levels, although the <span class="gene">FAD2-1B</span> mutant alleles alone were not capable of producing a <span class="trait">high</span> oleic acid phenotype. When existing <span class="gene">FAD2-1A</span> mutations were combined with the novel mutant <span class="gene">FAD2-1B</span> alleles, a <span class="trait">high</span> oleic acid phenotype was recovered only for those lines which were homozygous for both of the mutant alleles. CONCLUSIONS: We were able to produce conventional soybean lines with 80% oleic acid in the oil in two different ways, each requiring the contribution of only two genes. The <span class="trait">high</span> oleic acid soybean germplasm developed contained a desirable fatty acid profile, and it was stable in two production environments. The presumed causative sequence polymorphisms in the <span class="gene">FAD2-1B</span> alleles were developed into highly efficient molecular markers for tracking the mutant alleles. The resources described here for the creation of <span class="trait">high</span> oleic acid soybeans provide a framework to efficiently develop soybean varieties to meet changing market demands.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The alteration of fatty acid profiles in soybean [ <italic> Glycine max </italic> (L.) Merr.] to improve soybean oil quality is an important and evolving theme in soybean research to meet nutritional needs and industrial criteria in the modern market. Soybean oil with elevated oleic acid is desirable because this monounsaturated fatty acid improves the nutrition and oxidative stability of the oil. Commodity soybean oil typically contains 20% oleic acid and the target for <span class="trait">high</span> oleic acid soybean oil is approximately 80% of the oil; previous conventional plant breeding research to raise the oleic acid level to just 50-60% of the oil was hindered by the genetic complexity and environmental instability of the trait. The objective of this work was to create the <span class="trait">high</span> oleic acid trait in soybeans by identifying and combining mutations in two delta-twelve fatty acid desaturase genes, <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Three polymorphisms found in the <italic> <span class="gene">FAD2-1B</span> </italic> alleles of two soybean lines resulted in missense mutations. For each of the two soybean lines, there was one unique amino acid change within a highly conserved region of the <span class="trait">protein.</span> The mutant <italic> <span class="gene">FAD2-1B</span> </italic> alleles were associated with an increase in oleic acid levels, although the <italic> <span class="gene">FAD2-1B</span> </italic> mutant alleles alone were not capable of producing a <span class="trait">high</span> oleic acid phenotype. When existing <italic> <span class="gene">FAD2-1A</span> </italic> mutations were combined with the novel mutant <italic> <span class="gene">FAD2-1B</span> </italic> alleles, a <span class="trait">high</span> oleic acid phenotype was recovered only for those lines which were homozygous for both of the mutant alleles.
					</p>
				
					<p>&nbsp;&nbsp;
						We were able to produce conventional soybean lines with 80% oleic acid in the oil in two different ways, each requiring the contribution of only two genes. The <span class="trait">high</span> oleic acid soybean germplasm developed contained a desirable fatty acid profile, and it was stable in two production environments. The presumed causative sequence polymorphisms in the <italic> <span class="gene">FAD2-1B</span> </italic> alleles were developed into highly efficient molecular markers for tracking the mutant alleles. The resources described here for the creation of <span class="trait">high</span> oleic acid soybeans provide a framework to efficiently develop soybean varieties to meet changing market demands.
					</p>
				
					<p>&nbsp;&nbsp;
						Selection for breeding and genetic engineering resulting in elevated oleic acid levels were reported in many oilseed crops: safflower[ <xref ref-type="bibr" rid="B11"> 11 </xref> ], sunflower [ <xref ref-type="bibr" rid="B12"> 12 </xref> ], peanut [ <xref ref-type="bibr" rid="B13"> 13 </xref> - <xref ref-type="bibr" rid="B15"> 15 </xref> ], canola [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B17"> 17 </xref> ], cotton [ <xref ref-type="bibr" rid="B12"> 12 </xref> ] and maize [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. While the elevated oleic acid phenotype was often observed after a single <italic> <span class="gene">FAD2</span> </italic> gene was mutated, the very highest oleic acid phenotypes (&#x0003e; 80% of the total oil) were achieved most frequently by silencing all copies of <italic> <span class="gene">FAD2</span> </italic> genes that were expressed in developing seeds, or in particular for peanut, by combining the mutation in the active site of <italic> ah </italic> FAD2A with the loss of transcription of <italic> ahFAD2B </italic> [ <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> ]. No soybean lines exist in the USDA National Plant Germplasm System collection with the <span class="trait">high</span> oleic acid trait (oleic acid content above 70% of the oil fraction), although multiple lines contain elevated oleic acid levels [ <xref ref-type="bibr" rid="B9"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Suppression of <italic> <span class="gene">FAD2-1</span> </italic> gene expression by means of genetic engineering has been successful in creating soybean lines with the <span class="trait">high</span> oleic acid trait, with oleic acid content above 80% of the oil fraction reported and very little environmental impact on the trait [ <xref ref-type="bibr" rid="B3"> 3 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Transgenic expression of ribozyme terminated sense or antisense <italic> <span class="gene">FAD2-1</span> </italic> constructs was successful in eliminating the <italic> <span class="gene">FAD2-1</span> </italic> mRNA expression signal in developing embryos and producing siRNAs for the <italic> <span class="gene">FAD2-1</span> </italic> genes [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. No attempt was made to distinguish between the <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes in the transgenic work, and it is also possible that the <italic> <span class="gene">FAD2-2</span> </italic> genes were targeted in the experiments [ <xref ref-type="bibr" rid="B25"> 25 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The objective of this work was to create the <span class="trait">high</span> oleic acid trait in soybeans using conventional plant breeding technology. We hypothesized that combinations of mutant alleles of the soybean <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes would greatly reduce the <span class="gene">FAD2</span> enzyme activity in developing seeds and thus result in an accumulation of oleic acid at the expense of linoleic and linolenic acid in the triacylglycerol fraction of the seed oil. We took a candidate gene approach with the <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes present in soybean germplasm accessions containing elevated levels of oleic acid in the oil. Two mutant alleles of <italic> <span class="gene">FAD2-1B</span> </italic> were identified that associated with elevated oleic acid content. In addition, mutant alleles of <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> were combined to create soybeans with the <span class="trait">high</span> oleic acid trait.
					</p>
				
					<p>&nbsp;&nbsp;
						To predict the potential effect of the amino acid changes to soybean <span class="gene">FAD2-1B</span> enzyme function, we used the program PolyPhen to analyze the potential severity of each amino acid change [ <xref ref-type="bibr" rid="B29"> 29 </xref> ]. In addition, the relative amino acid conservation for each position in the enzyme was evaluated visually using Weblogo after alignment of 100 different <span class="gene">FAD2</span> <span class="trait">protein</span> sequences present in the National Center for Biotechnology Information database [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. The two shared amino acid substitutions M126V and S86F each occur at a highly variable position according to the alignment in Weblogo and were classified by Polyphen as benign substitutions, indicating they are likely to have no phenotypic effect. For the M126V change, methionine and valine have similar chemical properties.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">FAD2-1B</span> </italic> P137R mutation present in PI 283327 represents a charge change for the substituted amino acid since proline is nonpolar while arginine is basic. At position 137 of wild-type FAD2-1B, the proline is perfectly conserved; a proline at that position is invariant for all of the tested <span class="gene">FAD2</span> sequences represented in the <span class="trait">protein</span> database (Figure <xref ref-type="fig" rid="F1"> 1C </xref> ). In PolyPhen, the amino acid change was classified as probably damaging for P137R, which means that this SNP is predicted to affect <span class="trait">protein</span> structure and/or function.
					</p>
				
					<p>&nbsp;&nbsp;
						For the isoleucine present at position 143 of wild-type FAD2-1B, other amino acids were observed in this position in the <span class="gene">FAD2</span> sequences present in the <span class="trait">protein</span> database, indicating that an isoleucine at position 143 is less conserved than the proline at the 137 position (Figure <xref ref-type="fig" rid="F1"> 1C </xref> ). However, the substitution of threonine for isoleucine at this position was not observed in the database analysis. Moreover, isoleucine and threonine also have contrasting chemical properties since isoleucine is nonpolar while threonine is uncharged polar. In PolyPhen, the I143T amino acid change was classified as probably damaging. The conservation of amino acids in the general region of the <italic> <span class="gene">FAD2-1B</span> </italic> P137R and I143T mutations combined with the chemical nature of the changes is suggestive of the potential deleterious effects of these mutations to the <span class="gene">FAD2-1B</span> enzyme's structure and function.
					</p>
				
					<p>&nbsp;&nbsp;
						Our initial investigation of both the <italic> <span class="gene">FAD2-1</span> </italic> genotype and fatty acid phenotype in F <sub> 2 </sub> seeds from Population 4 ( <italic> <span class="gene">FAD2-1A</span> </italic> S117N &#x000d7; <italic> <span class="gene">FAD2-1B</span> </italic> P137 cross) demonstrated the epistatic nature of the mutant alleles working in combination, and the results revealed that only homozygous combinations of both mutant <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> were capable of producing the <span class="trait">high</span> oleic acid phenotype. Of the 200 F <sub> 2 </sub> seeds that were phenotyped, there were 12 individual F <sub> 2 </sub> seeds with genotype <italic> <span class="gene">FAD2-1</span> </italic> aabb, and they had an average oleic acid content of 81%, ranging from 75.2% to 83.9% oleic acid (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). The next highest oleic acid phenotype in the set was 48.8%, and that seed had the <italic> <span class="gene">FAD2-1</span> </italic> Aabb genotype. For a two recessive gene model, one sixteenth of the individuals should inherit the phenotype; recovery of 12 individuals with the <span class="trait">high</span> oleic acid phenotype satisfies this expectation by Chi-Square test at the 0.05 probability level.
					</p>
				
					<p>&nbsp;&nbsp;
						The necessity of the homozygous <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> mutant combination requirement for the <span class="trait">high</span> oleic acid phenotype was confirmed in an independent analysis of <italic> <span class="gene">FAD2-1</span> </italic> genotype and fatty acid phenotype of field produced F <sub> 2 </sub> seeds that contained homozygous <italic> <span class="gene">FAD2-1A</span> </italic> &#x00394; alleles but which were segregating for <italic> <span class="gene">FAD2-1B</span> </italic> P137R alleles (Population 5). While the average oleic acid level of those seeds with the <italic> <span class="gene">FAD2-1</span> </italic> aabb genotype was 82.5%, <italic> <span class="gene">FAD2-1</span> </italic> aaBb seeds averaged 55.4%; <italic> <span class="gene">FAD2-1</span> </italic> aaBB seeds averaged 43.4% oleic acid in the seed oil. The presence of a single wild-type version of the <italic> <span class="gene">FAD2-1B</span> </italic> allele also prevented a <span class="trait">high</span> oleic acid content in the seed oil, although the magnitude of the difference was greater for the F <sub> 2 </sub> seeds from Population 4.
					</p>
				
					<p>&nbsp;&nbsp;
						Some of the <span class="trait">high</span> oleic acid soybean lines developed in this study demonstrated stability for the <span class="trait">high</span> oleic acid trait when grown in different environments (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Of the three environments, Costa Rica typically has the warmest temperatures during seed development, followed by the Portageville, MO environment; the Columbia, MO environment is the coolest of the three environments during seed development [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B10"> 10 </xref> ]. The differences in the oleic acid contents between environments when the <italic> <span class="gene">FAD2-1B</span> </italic> P137R alleles were present were minor. Soybean lines with genotype <italic> <span class="gene">FAD2-1</span> </italic> aabb of population 2 and 4 produced more than 80% oleic acid content in Costa Rica and Portageville, MO environments, and the oleic acid level was an average of 2-4% lower when grown in the Columbia, MO environment. It is notable that the variation in the phenotype was narrow in all of the environments. In contrast, the <italic> <span class="gene">FAD2-1</span> </italic> aabb soybean lines of population 3 containing the <italic> <span class="gene">FAD2-1B</span> </italic> I143T alleles had lower and more variable oleic acid content in the cooler environments and failed to produce a <span class="trait">high</span> oleic acid phenotype in either the Columbia, MO or Portageville, MO environments.
					</p>
				
					<p>&nbsp;&nbsp;
						The full fatty acid profiles of the seeds of contrasting <italic> <span class="gene">FAD2-1</span> </italic> genotypic classes produced from Populations 2, 3, and 4 in this study revealed additional alterations in palmitic acid, linoleic acid, and linolenic acid content (Table <xref ref-type="table" rid="T2"> 2 </xref> ). As expected for a major decrease in seed expressed <span class="gene">FAD2</span> enzyme activity that results in an accumulation of oleic acid, the <span class="gene">FAD2</span> reaction products linoleic acid and linolenic acid were dramatically reduced in the <span class="trait">high</span> oleic <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> homozygous mutant lines when either of the <italic> <span class="gene">FAD2-1A</span> </italic> mutations were present along with the <italic> <span class="gene">FAD2-1B</span> </italic> P137R or I143T alleles.
					</p>
				
					<p>&nbsp;&nbsp;
						Because of the concern that improvement in fatty acid profiles might have negative impacts on the total oil and <span class="trait">protein</span> profiles of the seeds, we also evaluated the <span class="trait">protein</span> and oil contents for the field produced F <sub> 2:3 </sub> seeds from Population 4. There were no significant differences in the <span class="trait">protein</span> or oil contents among the different homozygous <italic> <span class="gene">FAD2-1</span> </italic> genotypes, or with those lines compared to either Williams 82 or the 17D parental line. The <italic> <span class="gene">FAD2-1B</span> </italic> P137R allele donor parental line had a minor decrease in the average oil content and the highest mean <span class="trait">protein</span> content of all of the lines examined.
					</p>
				
					<p>&nbsp;&nbsp;
						Increasing the oleic acid content in soybean seed oil is one of the most effective and efficient ways to enhance the nutritional value and practical utilization of soybean oil. However, the previously characterized sources of elevated oleic acid soybean involved mutation of the <italic> <span class="gene">FAD2-1A</span> </italic> gene alone, which failed to achieve oleic acid levels stable above 60% of the oil [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> ] or utilized approaches that have proven to have limited usage in soybean breeding due to the complexity of the trait [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. A transgenic approach was successful in downregulation of the <italic> <span class="gene">FAD2-1</span> </italic> genes leading to <span class="trait">high</span> oleic acid levels in the oil [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> ]. We have demonstrated here that an allele of the <italic> <span class="gene">FAD2-1B</span> </italic> gene containing a conserved amino acid substitution is responsible for the elevated oleic acid content in PI 283327, since soybean lines inheriting the homozygous mutant alleles have higher levels of oleic acid compared to lines inheriting wild type <italic> <span class="gene">FAD2-1B</span> </italic> alleles. The PI 567189A <italic> <span class="gene">FAD2-1B</span> </italic> allele, which contains a very rare amino acid substitution, was also predicted to have a negative impact on the enzyme activity and function.
					</p>
				
					<p>&nbsp;&nbsp;
						Our study demonstrated that different types of <italic> <span class="gene">FAD2-1A</span> </italic> mutant alleles bring about a similar <span class="trait">high</span> level of oleic acid content in the oil of soybean seeds when combined with the P137R <italic> <span class="gene">FAD2-1B</span> </italic> allele from PI 283327. M23 has lost its <italic> <span class="gene">FAD2-1A</span> </italic> alleles during the X-ray treatment while 17D carries the alleles with a mutagenesis-induced mutation that appeared to be less effective in accumulating oleic acid in the seed oil [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> ]. Hence, it was assumed that 17D would be less useful than M23 in creating a <span class="trait">high</span> oleic acid trait. In contrast to what we expected, the <italic> <span class="gene">FAD2-1A</span> </italic> alleles from 17D were able to combine with mutant <italic> <span class="gene">FAD2-1B</span> </italic> to produce oleic acid levels similar to those when M23 was the source of the deleted <italic> <span class="gene">FAD2-1A</span> </italic> gene, and the phenotype was stable across the different environments evaluated. We conclude that for the <italic> <span class="gene">FAD2-1A</span> </italic> gene from 17D the encoded enzyme was unable to function appropriately, so when in combination with the mutant P137R <italic> <span class="gene">FAD2-1B</span> </italic> allele, the conversion of oleic acid precursors to linoleic acid precursors was almost completely blocked. Presumably there is some compensatory difference in the activity of the functional <span class="gene">FAD2-1B</span> when the S117N allele of <italic> <span class="gene">FAD2-1A</span> </italic> was present compared to the situation from M23 when <italic> <span class="gene">FAD2-1A</span> </italic> is deleted.
					</p>
				
					<p>&nbsp;&nbsp;
						The occurrence of soybean lines with the <span class="trait">high</span> oleic acid phenotype in population 4 fit a model of two independent recessive genes segregating with very little evidence for additional modifier genes, demonstrating that in this circumstance only <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> are contributing to the <span class="trait">high</span> oleic acid content in soybean seed oil. Based on our results, we hypothesize that any soybean line carrying a null or severely mutated <italic> <span class="gene">FAD2-1A</span> </italic> allele that is crossed with PI 283327 is likely to produce individual soybean seeds in the F <sub> 2 </sub> progeny with at least 80% oleic acid content of the seed oil.
					</p>
				
					<p>&nbsp;&nbsp;
						Although there is evidence of influence of temperature on the soybean seed oleic acid content [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B10"> 10 </xref> ], two of our three <span class="trait">high</span> oleic acid soybean genotypes proved to be capable of producing a <span class="trait">high</span> and stable oleic acid content in three environments. Moreover, there was no reduction in oil and <span class="trait">protein</span> content in the evaluated <span class="trait">high</span> oleic acid soybean lines. Soybean lines with the combination of <italic> <span class="gene">FAD2-1A</span> </italic> &#x00394; and <italic> <span class="gene">FAD2-1B</span> </italic> I143T alleles from population 3 failed to produce the <span class="trait">high</span> oleic acid phenotype when grown in the non-tropical environments. A possible explanation is the mutation in the <italic> <span class="gene">FAD2-1B</span> </italic> allele of PI 567189 A encodes at least nominal enzyme function. This explanation is supported by the fact that the I143T substitution is in a less conserved amino acid of the <span class="gene">FAD2</span> enzyme than the P137R substitution. Other than that, our <span class="trait">high</span> oleic acid soybean lines showed a reduction of 4% at most when they were grown in the cooler environment, with a small variation in the oleic acid content. It will be necessary to test the performance of these <span class="trait">high</span> oleic acid soybean lines in the main North American soybean growing locations in more northern latitudes. The mutant <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> alleles will have to be combined in soybean lines with the appropriate maturity for those experiments to be conducted. However, based on the stability of the trait that we have observed so far, any reduction of oleic acid content due to the environment is likely to be minor because very little <span class="gene">FAD2</span> enzyme activity remains in developing seeds in the mutant <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> lines. An additional factor is that the end use market has not matured sufficiently to define the exact oleic acid content desired for different oil uses. Another question that should be addressed is whether the trait will affect <span class="trait">yield</span> or other agronomic traits. It has been reported that the transgenic soybean lines with the <italic> <span class="gene">FAD2-1</span> </italic> genes being silenced did not show any <span class="trait">yield</span> drag or abnormal physiology characteristics [ <xref ref-type="bibr" rid="B26"> 26 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Traditional breeding has been used previously to produce soybean with up to 70% oleic acid content in the seed oil [ <xref ref-type="bibr" rid="B33"> 33 </xref> ]. However, the phenotype is not consistent across environments, and the genetics of the trait is not very well understood, which limits the usage of these soybean lines. Practices to boost oleic acid content in soybean to more than 80% have been achieved by means of suppression of the expression level of <italic> <span class="gene">FAD2</span> </italic> genes, and as a result, transgenic <span class="trait">high</span> oleic soybeans were produced [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Commercial release of transgenic plants still has to overcome regulatory hurdles, and production and importation of transgenic plants remain unacceptable in various countries.
					</p>
				
					<p>&nbsp;&nbsp;
						Our research results have demonstrated the capacity to develop soybeans containing more than 80% oleic acid in the oil based on very simple genetic manipulation, the combination of two recessive genes. As part of this research, we developed molecular marker assays that allow the selection of the desired mutant <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> alleles, even when they are present in the heterozygous state. Molecular marker selection thus eliminates the time necessary to produce an extra generation of plants that must be screened for the fatty acid phenotype. Simple genetics combined with perfect molecular marker assays will make it possible for soybean breeders to quickly incorporate the <span class="trait">high</span> oleic acid trait in their breeding programs. The resulting <span class="trait">high</span> oleic acid soybean lines can then be efficiently developed and released as cultivars to producers. Also, because only two genes control the vast majority of the <span class="trait">high</span> oleic acid phenotype, other genes may be added to enhance soybean oil quality such as low linolenic acid, low allergens, or a growing list of traits involved in soybean meal quality [ <xref ref-type="bibr" rid="B34"> 34 </xref> ]. In the short term, examination of the stability of the <span class="trait">high</span> oleic acid soybean lines across different environments is of particular interest. Also, the soybean lines with the <span class="trait">high</span> oleic acid trait should be used for development of soybean varieties with favorable agronomic traits including <span class="trait">high</span> <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Protein</span> and oil contents were determined for seeds of F <sub> 2:3 </sub> lines for population 4 using <span class="gene">NIR</span> spectroscopy [ <xref ref-type="bibr" rid="B38"> 38 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:AtCML8, a calmodulin-like <span class="trait">protein,</span> differentially activating CaM-dependent enzymes in Arabidopsis thaliana.</h2>
			<h3>PMID:20820784  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Park HC(1), Park CY, Koo SC, Cheong MS, Kim KE, Kim MC, Lim CO, Lee SY, Yun DJ, Chung WS.
				<br/> 
				年份：2010 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants express many calmodulins (CaMs) and calmodulin-like (CML) proteins that sense and transduce different Ca(2+) signals. Previously, we reported divergent soybean (Glycine max) <span class="gene">CaM</span> isoforms <span class="gene">(GmCaM4/5)</span> with differential abilities to activate CaM-dependent enzymes. To elucidate biological functions of divergent <span class="gene">CaM</span> proteins, we isolated a cDNA encoding a CML <span class="trait">protein,</span> AtCML8, from Arabidopsis. AtCML8 shows highest identity with <span class="gene">GmCaM4</span> at the <span class="trait">protein</span> sequence level. Expression of AtCML8 was <span class="trait">high</span> in roots, leaves, and flowers but low in stems. In addition, the expression of AtCML8 was induced by exposure to salicylic acid or NaCl. AtCML8 showed typical characteristics of <span class="gene">CaM</span> such as Ca(2+)-dependent electrophoretic mobility shift and Ca(2+) binding ability. In immunoblot analyses, AtCML8 was recognized only by antiserum against <span class="gene">GmCaM4</span> but not by <span class="gene">GmCaM1</span> antibodies. Interestingly, AtCML8 was able to activate phosphodiesterase (PDE) but did not activate NAD kinase. These results suggest that AtCML8 acts as a CML <span class="trait">protein</span> in Arabidopsis with characteristics similar to soybean divergent <span class="gene">GmCaM4</span> at the biochemical levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of H-1 and H-2 subunits of soybean seed ferritin in oxidative deposition of iron in <span class="trait">protein.</span></h2>
			<h3>PMID:20702403  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deng J(1), Liao X, Yang H, Zhang X, Hua Z, Masuda T, Goto F, Yoshihara T, Zhao G.
				<br/> 
				年份：2010 Oct 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Naturally occurring phytoferritin is a heteropolymer consisting of two different H-type subunits, H-1 and H-2. Prior to this study, however, the function of the two subunits in oxidative deposition of iron in ferritin was unknown. The data show that, upon aerobic addition of 48-200 Fe(2+)/shell to apoferritin, iron oxidation occurs only at the diiron ferroxidase center of recombinant <span class="gene">H1</span> (rH-1). In addition to the diiron ferroxidase mechanism, such oxidation is catalyzed by the extension peptide (a specific domain found in phytoferritin) of rH-2, because the H-1 subunit is able to remove Fe(3+) from the center to the inner cavity better than the H-2 subunit. These findings support the idea that the H-1 and H-2 subunits play different roles in iron mineralization in <span class="trait">protein.</span> Interestingly, at medium iron loading (200 irons/shell), wild-type (WT) soybean seed ferritin (SSF) exhibits a stronger activity in catalyzing iron oxidation (1.10 ± 0.13 μm iron/subunit/s) than rH-1 (0.59 ± 0.07 μm iron/subunit/s) and rH-2 (0.48 ± 0.04 μm iron/subunit/s), demonstrating that a synergistic interaction exists between the H-1 and H-2 subunits in SSF during iron mineralization. Such synergistic interaction becomes considerably stronger at <span class="trait">high</span> iron loading (400 irons/shell) as indicated by the observation that the iron oxidation activity of WT SSF is ∼10 times larger than those of rH-1 and rH-2. This helps elucidate the widespread occurrence of heteropolymeric ferritins in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of hydroperoxide lyase gene in Nicotiana benthamiana using a viral vector system.</h2>
			<h3>PMID:20691022  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huang FC(1), Studart-Witkowski C, Schwab W.
				<br/> 
				年份：2010 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						13-Lipoxygenase (13-LOX) and 13-hydroperoxide lyases (13-HPL) are the key enzymes for the production of the 'green note' compounds hexanal, (3Z)- and (2E)-hexenal in plant tissues. To produce <span class="trait">high</span> levels of 13-LOX and 13-HPL enzymatic activities for a biocatalytic process to generate C(6)-aldehydes on a large scale, soya bean 13-LOX <span class="gene">(GmVLXC)</span> and watermelon 13-HPL (ClHPL) genes were expressed in Nicotiana benthamiana using a viral vector system mediated by agroinfiltration. The N. benthamiana leaves produced <span class="trait">high</span> activity of watermelon HPL, but not <span class="gene">GmVLXC</span> 13-LOX. In addition, all leaves treated with bacterial suspension displayed a <span class="trait">high</span> activity of 9-LOX, indicating that the internal tobacco 9-LOX gene was highly induced through agroinfiltration because of wounding. <span class="gene">GmVLXC</span> and ClHPL transcripts could be detected in the corresponding transformed tobacco leaves by real-time RT-PCR analysis but the expression level of ClHPL was 24-fold higher than that of <span class="gene">GmVLXC.</span> Western blot analysis showed that <span class="gene">LOX</span> was present in all tobacco leaves which were treated with bacterial suspensions, but not in the untreated wild-type control. This result confirms that internal 9-LOX was highly induced by agroinfiltration. The highest levels of ClHPL activity under optimal infiltration conditions were 80 times the <span class="gene">HPL</span> activity of wild-type plants or plants transformed with control vector. A large amount of hexanal was formed when linoleic acid was incubated with extracts from N. benthamiana leaves over-expressing ClHPL in combination with <span class="gene">GmVLXC-expressing</span> yeast extracts. One gram of ClHPL-expressing N. benthamiana leaves (fresh weight) could produce 17 +/- 0.4 mg hexanal from 50 mg linoleic acid after 30 min.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant NAC-type transcription factor proteins contain a NARD domain for repression of transcriptional activation.</h2>
			<h3>PMID:20683728  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hao YJ(1), Song QX, Chen HW, Zou HF, Wei W, Kang XS, Ma B, Zhang WK, Zhang JS, Chen SY.
				<br/> 
				年份：2010 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-specific transcription factor NAC proteins play essential roles in many biological processes such as development, senescence, morphogenesis, and stress signal transduction pathways. In the NAC family, some members function as transcription activators while others act as repressors. In the present study we found that though the full-length <span class="gene">GmNAC20</span> from soybean did not have transcriptional activation activity, the carboxy-terminal activation domain of <span class="gene">GmNAC20</span> had <span class="trait">high</span> transcriptional activation activity in the yeast assay system. Deletion experiments revealed an active repression domain with 35 amino acids, named NARD (NAC Repression Domain), in the d subdomain of NAC DNA-binding domain. NARD can reduce the transcriptional activation ability of diverse transcription factors when fused to either the amino-terminal or the carboxy-terminal of the transcription factors. NARD-like sequences are also present in other NAC family members and they are functional repression domain when fused to VP16 in plant protoplast assay system. Mutation analysis of conserved amino acid residues in NARD showed that the hydrophobic LVFY motif may partially contribute to the repression function. It is hypothesized that the interactions between the repression domain NARD and the carboxy-terminal activation domain may finally determine the ability of NAC family proteins to regulate downstream gene expressions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional characterization of a UDP-glucose:flavonoid 3-O-glucosyltransferase from the seed coat of black soybean (Glycine max (L.) Merr.).</h2>
			<h3>PMID:20621794  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kovinich N(1), Saleem A, Arnason JT, Miki B.
				<br/> 
				年份：2010 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The seed coats of black soybean (Glycine max (L.) Merr.) accumulate red (cyanidin-), blue (delphinidin-), purple (petunidin-), and orange (pelargonidin-based) anthocyanins almost exclusively as 3-O-glucosides; however, the responsible enzyme has not been identified. In this study, the full-length cDNA which encodes the enzyme that catalyzes the final step in anthocyanin biosynthesis, namely UDP-glucose:flavonoid 3-O-glucosyltransferase (UGT78K1), was isolated from the seed coat tissue of black soybean using rapid amplification of cDNA ends (RACE). Of the 28 flavonoid substrates tested, the purified recombinant <span class="trait">protein</span> glucosylated only anthocyanidins and flavonols, and demonstrated strict 3-OH regiospecificity. Galactose could also be transferred with relatively low activity to the 3-position of cyanidin or delphinidin in vitro. These findings are consistent with previous reports of mainly 3-O-glucosylated and minor amounts of 3-O-galactosylated anthocyanins in the seed coat of black soybean. The recombinant enzyme exhibited pronounced substrate inhibition by cyanidin at 100 microM acceptor concentration. Transfer of <span class="gene">UGT78K1</span> into the Arabidopsis T-DNA mutant (ugt78d2) deficient in anthocyanidin and flavonol 3-O-glucosyltransferase activity, restored the accumulation of anthocyanins and flavonols, suggesting the in vivo function of the enzyme as a flavonoid 3-O-glucosyltransferase. Genomic and phylogenetic analyses suggest the existence of three additional soybean sequences with <span class="trait">high</span> similarity to UGT78K1. RT-PCR confirmed the co-expression of one of these genes <span class="gene">(Glyma08g07130)</span> with <span class="gene">UGT78K1</span> in the seed coat of black soybean, suggesting possible functional redundancies in anthocyanin biosynthesis in this tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2010 Elsevier Ltd. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Chromatographic separation and tandem <span class="gene">MS</span> identification of active peptides in potato <span class="trait">protein</span> hydrolysate that inhibit autoxidation of soybean oil-in-water emulsions.</h2>
			<h3>PMID:20614939  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cheng Y(1), Chen J, Xiong YL.
				<br/> 
				年份：2010 Aug 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A previously developed antioxidative potato <span class="trait">protein</span> hydrolysate was fractionated using gel filtration. The efficacy of different fractions for inhibiting lipid oxidation in soybean oil-in-water emulsions, neutralizing 2,2'-azinobis (3-ethylbenzothiazoline-6-sulfonic acid) (ABTS(+*)) radicals, and chelating prooxidative metal ions was investigated. Low-molecular-weight fraction Peak 3 (709 Da) exhibited the strongest antioxidant activity and radical scavenging activity. Active peptides based on the ABTS(+*) scavenging assay were isolated by RP-HPLC and purified by UPLC. The amino acid sequence determination by MS/MS identified eight prominent peptides in the antioxidative Peak 3 fraction, of which Thr-Tyr, Tyr-Phe-Glu, Tyr-Ser-Thr-Ala, and Asn-Tyr-Lys-Gln-Met matched the sequence of papatin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">GmFAD7</span> gene family from soybean: identification of novel genes and tissue-specific conformations of the <span class="gene">FAD7</span> enzyme involved in desaturase activity.</h2>
			<h3>PMID:20547564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Andreu V(1), Lagunas B, Collados R, Picorel R, Alfonso M.
				<br/> 
				年份：2010 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">FAD7</span> gene encodes a omega3 fatty acid desaturase which catalyses the production of trienoic fatty acids (TAs) in plant chloroplasts. A novel <span class="gene">GmFAD7</span> gene (named <span class="gene">GmFAD7-2)</span> has been identified in soybean, with <span class="trait">high</span> homology to the previously annotated <span class="gene">GmFAD7</span> gene. Genomic sequencing analysis together with searches at the soybean genome database further confirmed that both <span class="gene">GmFAD7</span> genes were located in two different loci within the soybean genome, suggesting that the soybean omega3 plastidial desaturase <span class="gene">FAD7</span> is encoded by two different paralogous genes. Both <span class="gene">GmFAD7-1</span> and <span class="gene">GmFAD7-2</span> genes were expressed in all soybean tissues examined, displaying their highest mRNA accumulation in leaves. This expression profile contrasted with <span class="gene">GmFAD3A</span> and <span class="gene">GmFAD3B</span> mRNA accumulation, which was very low in this tissue. These results suggested a concerted control of plastidial and reticular omega3 desaturase gene expression in soybean mature leaves. Analysis of <span class="gene">GmFAD7</span> <span class="trait">protein</span> distribution in different soybean tissues showed that, in mature leaves, two bands were detected, coincident with the higher expression level of both <span class="gene">GmFAD7</span> genes and the highest 18:3 fatty acid accumulation. By contrast, in seeds, where <span class="gene">FAD7</span> activity is low, specific <span class="gene">GmFAD7</span> <span class="trait">protein</span> conformations were observed. These <span class="gene">GmFAD7</span> <span class="trait">protein</span> conformations were affected in vitro by changes in the redox conditions of thiol groups and iron availability. These results suggest the existence of tissue-specific post-translational regulatory mechanisms affecting the distribution and conformation of the <span class="gene">FAD7</span> enzymes related with the control of its activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase propeptides are functional signal peptides and increase the <span class="trait">yield</span> of a foreign <span class="trait">protein.</span></h2>
			<h3>PMID:20535473  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schnell JA(1), Han S, Miki BL, Johnson DA.
				<br/> 
				年份：2010 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elements that contribute to the <span class="trait">high,</span> stable <span class="trait">yield</span> of soybean peroxidase (SBP) in soybean seed coats can be exploited in the development of this tissue as a <span class="trait">protein</span> production platform. <span class="gene">SBP</span> contains an N-terminal and a C-terminal propeptide that are predicted to direct vacuolar targeting; this may be one factor that contributes to its <span class="trait">high</span> <span class="trait">yield</span> and stability. We characterized the function of the <span class="gene">SBP</span> propeptides and investigated their ability to increase the <span class="trait">yield</span> of a foreign <span class="trait">protein</span> in a heterologous plant system. <span class="gene">SBP</span> propeptides are functional signal peptides capable of directing vacuolar transport in Arabidopsis. The use of these propeptides as well as an endoplasmic reticulum (ER)-retention signal to direct a foreign <span class="trait">protein</span> to the apoplast, ER, or vacuole can significantly increase <span class="trait">yield</span> and will therefore be useful for the development of the seed coat as a <span class="trait">protein</span> production platform. We also demonstrate that growth conditions may have a significant impact on the <span class="trait">yield</span> of a foreign <span class="trait">protein</span> and that this may be subcellular compartment-specific.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interaction of cytosolic glutamine synthetase of soybean root nodules with the C-terminal domain of the symbiosome membrane nodulin 26 aquaglyceroporin.</h2>
			<h3>PMID:20504761  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Masalkar P(1), Wallace IS, Hwang JH, Roberts DM.
				<br/> 
				年份：2010 Jul 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nodulin 26 (nod26) is a major intrinsic <span class="trait">protein</span> that constitutes the major <span class="trait">protein</span> component on the symbiosome membrane (SM) of N(2)-fixing soybean nodules. Functionally, nod26 forms a low energy transport pathway for water, osmolytes, and NH(3) across the SM. Besides their transport functions, emerging evidence suggests that <span class="trait">high</span> concentrations of major intrinsic proteins on membranes provide interaction and docking targets for various cytosolic proteins. Here it is shown that the C-terminal domain peptide of nod26 interacts with a 40-kDa <span class="trait">protein</span> from soybean nodule extracts, which was identified as soybean cytosolic glutamine synthetase GS(1)beta1 by mass spectrometry. Fluorescence spectroscopy assays show that recombinant soybean GS(1)beta1 binds the nod26 C-terminal domain with a 1:1 stoichiometry (K(d) = 266 nm). GS(1)beta1 also binds to isolated SMs, and this binding can be blocked by preincubation with the C-terminal peptide of nod26. In vivo experiments using either a split ubiquitin yeast two-hybrid system or bimolecular fluorescence complementation show that the four cytosolic <span class="gene">GS</span> isoforms expressed in soybean nodules interact with full-length nod26. The binding of GS, the principal ammonia assimilatory enzyme, to the conserved C-terminal domain of nod26, a transporter of NH(3), is proposed to promote efficient assimilation of fixed nitrogen, as well as prevent potential ammonia toxicity, by localizing the enzyme to the cytosolic side of the symbiosome membrane.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Disruption of the glycine cleavage system enables Sinorhizobium fredii USDA257 to form nitrogen-fixing nodules on agronomically improved North American soybean cultivars.</h2>
			<h3>PMID:20453144  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lorio JC(1), Kim WS, Krishnan AH, Krishnan HB.
				<br/> 
				年份：2010 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The symbiosis between Sinorhizobium fredii USDA257 and soybean [Glycine max (L.) Merr.] exhibits a <span class="trait">high</span> degree of cultivar specificity. USDA257 nodulates primitive soybean cultivars but fails to nodulate agronomically improved cultivars such as McCall. In this study we provide evidence for the involvement of a new genetic locus that controls soybean cultivar specificity. This locus was identified in USDA257 by Tn5 transposon mutagenesis followed by nodulation screening on McCall soybean. We have cloned the region corresponding to the site of Tn5 insertion and found that it lies within a 1.5-kb EcoRI fragment. DNA sequence analysis of this fragment and an adjacent 4.4-kb region identified an operon made up of three open reading frames encoding proteins of deduced molecular masses of 41, 13, and 104 kDa, respectively. These proteins revealed significant amino acid homology to glycine cleavage (gcv) system T, H, and P proteins of Escherichia coli and other organisms. Southern blot analysis revealed the presence of similar sequences in diverse rhizobia. Measurement of beta-galactosidase activity of a USDA257 strain containing a transcriptional fusion of <span class="gene">gcvT</span> promoter sequences to the lacZ gene revealed that the USDA257 gcvTHP operon was inducible by glycine. Inactivation of either <span class="gene">gcvT</span> or <span class="gene">gcvP</span> of USDA257 enabled the mutant to nodulate several agronomically improved North American soybean cultivars. These nodules revealed anatomical features typical of determinate nodules, with numerous bacteroids within the infected cells. Unlike for the previously characterized soybean cultivar specificity locus nolBTUVW, inactivation of the gcv locus had no discernible effect on the secretion of nodulation outer proteins of USDA257.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Application of multilocus sequence typing to study the genetic structure of megaplasmids in medicago-nodulating rhizobia.</h2>
			<h3>PMID:20418433  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：van Berkum P(1), Elia P, Eardly BD.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A multilocus sequence typing (MLST) analysis was used to examine the genetic structure and diversity within the two large extrachromosomal replicons in Medicago-nodulating rhizobia (Sinorhizobium meliloti and Sinorhizobium medicae). The allelic diversity within these replicons was <span class="trait">high</span> compared to the reported diversity within the corresponding chromosomes of the same strains (P. van Berkum et al., J. Bacteriol. 188:5570-5577, 2006). Also, there was strong localized linkage disequilibrium (LD) between certain pSymA loci: e.g., <span class="gene">nodC</span> and nifD. Although both of these observations could be explained by positive (or diversifying) selection by plant hosts, results of tests for positive selection did not provide consistent support for this hypothesis. The strong LD observed between the <span class="gene">nodC</span> and <span class="gene">nifD</span> genes could also be explained by their close proximity on the pSymA replicon. Evidence was obtained that some <span class="gene">nodC</span> alleles had a history of intragenic recombination, while other alleles of this locus had a history of intergenic recombination. Both types of recombination were associated with a decline in symbiotic competence with Medicago sativa as the host plant. The combined observations of LD between the <span class="gene">nodC</span> and <span class="gene">nifD</span> genes and intragenic recombination within one of these loci indicate that the symbiotic gene region on the pSymA plasmid has evolved as a clonal segment, which has been laterally transferred within the natural populations.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of alanine aminotransferase in various organs of soybean (Glycine max) and in dependence of different nitrogen fertilisers during hypoxic stress.</h2>
			<h3>PMID:20414691  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rocha M(1), Sodek L, Licausi F, Hameed MW, Dornelas MC, van Dongen JT.
				<br/> 
				年份：2010 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Alanine aminotransferase (AlaAT) catalyses the reversible conversion of pyruvate and glutamate into alanine and oxoglutarate. In soybean, two subclasses were identified, each represented by two highly similar members. To investigate the role of AlaAT during hypoxic stress in soybean, changes in transcript level of both subclasses were analysed together with the enzyme activity and alanine content of the tissue. Moreover, the dependency of AlaAT activity and gene expression was investigated in relation to the source of nitrogen supplied to the plants. Using semi-quantitative PCR, <span class="gene">GmAlaAT</span> genes were determined to be highest expressed in roots and nodules. Under normal growth conditions, enzyme activity of AlaAT was detected in all organs tested, with lowest activity in the roots. Upon waterlogging-induced hypoxia, AlaAT activity increased strongly. Concomitantly, alanine accumulated. During re-oxygenation, AlaAT activity remained <span class="trait">high,</span> but the transcript level and the alanine content decreased. Our results show a role for AlaAT in the catabolism of alanine during the initial period of re-oxygenation following hypoxia. <span class="gene">GmAlaAT</span> also responded to nitrogen availability in the solution during waterlogging. Ammonium as nitrogen source induced both gene expression and enzyme activity of AlaAT more than when nitrate was supplied in the nutrient solution. The work presented here indicates that AlaAT might not only be important during hypoxia, but also during the recovery phase after waterlogging, when oxygen is available to the tissue again.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of a glutathione S-transferase gene, GsGST, from wild soybean (Glycine soja) enhances <span class="trait">drought</span> and salt tolerance in transgenic tobacco.</h2>
			<h3>PMID:20383560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ji W(1), Zhu Y, Li Y, Yang L, Zhao X, Cai H, Bai X.
				<br/> 
				年份：2010 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine soja is a species of soybean that survives in adverse environments including <span class="trait">high</span> salt and <span class="trait">drought</span> conditions. We constructed a cDNA library from G. soja seedlings treated with NaCl and isolated a glutathione S-transferase gene (GsGST: GQ265911) from the library. The cDNA encoding GsGST contains an open reading frame of 660 bp and the predicted <span class="trait">protein</span> belongs to the tau class of <span class="gene">GST</span> family proteins. Tobacco plants over-expressing the GsGST gene showed sixfold higher <span class="gene">GST</span> activity than wild-type plants. Transgenic tobacco plants exhibited enhanced dehydration tolerance. T(2) transgenic tobacco plants showed higher tolerance at the seedling stage than wild-type plants to salt and mannitol as demonstrated by longer root length and less growth retardation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Size of AT(n) insertions in promoter region modulates <span class="gene">Gmhsp17.6-L</span> mRNA transcript levels.</h2>
			<h3>PMID:20300590  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fuganti R(1), Machado Mde F, Lopes VS, Silva JF, Arias CA, Marin SR, Binneck E, Abdelnoor RV, Marcelino FC, Nepomuceno AL.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						During earlier experiments, an SSR molecular marker (176 Soy HSP) showing <span class="trait">high</span> correlation (70%) with resistance/susceptibility to javanese root-knot nematode Meloidogyne javanica was identified in soybean. After being sequenced, results indicated that the SSR 176 Soy HSP marker was inserted in the promoter region of <span class="gene">Gmhsp17.6-L</span> gene. It was also detected in this region that resistant genotypes presented insertions between AT(31) and AT(33) in size and susceptible genotypes, AT(9). <span class="gene">Gmhsp17.6-L</span> gene coding region presented a perfect match in amino acid sequence in all soybean genotypes. A ribonuclease protection assay showed that <span class="gene">Gmhsp17.6-L</span> gene mRNA transcripts were present in all genotypes. A real-time relative quantification (qPCR) indicated in the resistant individuals higher mRNA transcripts levels, which presented in the sequencing more AT(n) insertions. These results suggest that the number of AT(n) insertions inside this promoter region could modulate up or down gene levels. Those findings can lead to the possibility of manipulating, between some limits, the mRNA transcripts levels using different sizes of AT(n) insertions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean stem growth habit gene <span class="gene">Dt1</span> is an ortholog of Arabidopsis TERMINAL FLOWER1.</h2>
			<h3>PMID:20219831  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu B(1), Watanabe S, Uchiyama T, Kong F, Kanazawa A, Xia Z, Nagamatsu A, Arai M, Yamada T, Kitamura K, Masuta C, Harada K, Abe J.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Classical genetic analysis has revealed that the determinate habit of soybean (Glycine max) is controlled by a recessive allele at the determinate stem (Dt1) locus. To dissect the molecular basis of the determinate habit, we isolated two orthologs of pea (Pisum sativum) TERMINAL FLOWER1a, <span class="gene">GmTFL1a</span> and <span class="gene">GmTFL1b,</span> from the soybean genome. Mapping analysis indicated that <span class="gene">GmTFL1b</span> is a candidate for Dt1. Despite their <span class="trait">high</span> amino acid identity, the two genes had different transcriptional profiles. <span class="gene">GmTFL1b</span> was expressed in the root and shoot apical meristems (SAMs), whereas <span class="gene">GmTFL1a</span> was mainly expressed in immature seed. The <span class="gene">GmTFL1b</span> transcript accumulated in the SAMs during early vegetative growth in both the determinate and indeterminate lines but thereafter was abruptly lost in the determinate line. Introduction of the genomic region of <span class="gene">GmTFL1b</span> from the indeterminate line complemented the stem growth habit in the determinate line: more nodes were produced, and flowering in the terminal raceme was delayed. The identity between <span class="gene">Dt1</span> and <span class="gene">GmTFL1b</span> was also confirmed with a virus-induced gene silencing experiment. Taken together, our data suggest that <span class="gene">Dt1</span> encodes the <span class="gene">GmTFL1b</span> <span class="trait">protein</span> and that the stem growth habit is determined by the variation of this gene. The dt1 allele may condition the determinate habit via the earlier loss in <span class="gene">GmTFL1b</span> expression concomitant with floral induction, although it functions normally under the noninductive phase of flowering. An association test of DNA polymorphisms with the stem growth habit among 16 cultivars suggested that a single amino acid substitution in exon 4 determines the fate of the SAM after floral induction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Formation of a cyclopropyl epoxide via a leukotriene A synthase-related pathway in an anaerobic reaction of soybean lipoxygenase-1 with 15S-hydroperoxyeicosatetraenoic acid: evidence that oxygen access is a determinant of secondary reactions with fatty acid hydroperoxides.</h2>
			<h3>PMID:20194505  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng Y(1), Brash AR.
				<br/> 
				年份：2010 Apr 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The further conversion of an arachidonic acid hydroperoxide to a leukotriene A (LTA) type epoxide by specific lipoxygenase (LOX) enzymes constitutes a key step in inflammatory mediator biosynthesis. Whereas mammalian 5-LOX transforms its primary product (5S-hydroperoxyeicosatetraenoic acid; 5S-HPETE) almost exclusively to LTA(4), the model enzyme, soybean LOX-1, normally produces no detectable leukotrienes and instead further oxygenates its primary product 15S-HPETE to 5,15- and 8,15-dihydroperoxides. Mammalian 15-LOX-1 displays both types of activity. We reasoned that availability of molecular oxygen within the <span class="gene">LOX</span> active site favors oxygenation, whereas lack of O(2) promotes LTA epoxide synthesis. To test this, we reacted 15S-HPETE with soybean LOX-1 under anaerobic conditions and identified the products by <span class="trait">high</span> pressure liquid chromatography, UV, mass spectrometry, and NMR. Among the products, we identified a pair of 8,15-dihydroxy diastereomers with all-trans-conjugated trienes that incorporated (18)O from H(2)(18)O at C-8, indicative of the formation of 14,15-LTA(4). A pair of 5,15-dihydroxy diastereomers containing two trans,trans-conjugated dienes (6E,8E,11E,13E) and that incorporated (18)O from H(2)(18)O at C-5 was deduced to arise from hydrolysis of a novel epoxide containing a cyclopropyl ring, 14,15-epoxy-[9,10,11-cyclopropyl]-eicosa-5Z,7E,13E-trienoic acid. Also identified was the delta-lactone of the 5,15-diol, a derivative that exhibited no (18)O incorporation due to its formation by intramolecular reaction of the carboxyl anion with the proposed epoxide intermediate. Our results support a model in which access to molecular oxygen within the active site directs the outcome from competing pathways in the secondary reactions of lipoxygenases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of two soybean (Glycine max L.) LEA IV proteins by circular dichroism and Fourier transform infrared spectrometry.</h2>
			<h3>PMID:20071374  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shih MD(1), Hsieh TY, Lin TP, Hsing YI, Hoekstra FA.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Late embryogenesis-abundant (LEA) proteins, accumulating to a <span class="trait">high</span> level during the late stages of seed development, may play a role as osmoprotectants. However, the functions and mechanisms of LEA proteins remained to be elucidated. Five major groups of LEA proteins have been described. In the present study, we report on the characterization of two members of soybean LEA IV proteins, basic <span class="gene">GmPM1</span> and acidic <span class="gene">GmPM28,</span> by circular dichroism and Fourier transform infrared spectroscopy. The spectra of both proteins revealed limited defined secondary structures in the fully hydrated state. Thus, the soybean LEA IV proteins are members of 'natively unfolded proteins'. <span class="gene">GmPM1</span> or <span class="gene">GmPM28</span> proteins showed a conformational change under hydrophobic or dry conditions. After fast or slow drying, the two proteins showed slightly increased proportions of defined secondary structures (alpha-helix and beta-sheet), from 30 to 49% and from 34 to 42% for <span class="gene">GmPM1</span> and <span class="gene">GmPm28,</span> respectively. In the dehydrated state, <span class="gene">GmPM1</span> and <span class="gene">GmPM28</span> interact with non-reducing sugars to improve the transition temperature of cellular glass, with poly-l-lysine to prevent dehydration-induced aggregation and with phospholipids to maintain the liquid crystal phase over a wide temperature range. Our work suggests that soybean LEA IV proteins are functional in the dry state. They are one of the important components in cellular glasses and may stabilize desiccation-sensitive proteins and plasma membranes during dehydration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative study of protein-protein interaction observed in PolyGalacturonase-Inhibiting Proteins from <italic> Phaseolus vulgaris </italic> and <italic> Glycine max </italic> and PolyGalacturonase from <italic> Fusarium moniliforme </italic></h2>
			<h3>PMID:19958482</h3>
				

				
					<h3>PMCID:2788371</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Maulik </surname>  <given-names> Aditi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> aditi_maulik05@yahoo.co.in </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Ghosh </surname>  <given-names> Hiren </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hirenbioinfo2006@gmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A3">  <name>  <surname> Basu </surname>  <given-names> Soumalee </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> soumalee@gmail.com </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The PolyGalacturonase-Inhibiting Proteins (PGIP) of plant cell wall limit the invasion of phytopathogenic organisms by interacting with the enzyme PolyGalacturonase (PG) they secrete to degrade pectin present in the cell walls. PGIPs from different or same plant differ in their inhibitory activity towards the same PG. PGIP2 from Phaseolus vulgaris (Pv) inhibits the PG from Fusarium moniliforme (Fm) although PGIP1, another member of the multigene family from the same plant sharing 99% sequence similarity, cannot. Interestingly, PGIP3 from Glycine max (Gm) which is a homologue of PGIP2 is capable of inhibiting the same PG although the extent of similarity is lower and is 88%. It therefore appears that subtle changes in the sequence of plant PGIPs give rise to different specificity for inhibiting pathogenic PGs and there exists no direct dependence of function on the extent of sequence similarity. RESULTS: Structural information for any PGIP-PG complex being absent, we resorted to molecular modelling to gain insight into the mechanism of recognition and discrimination of PGs by PGIPs. We have built homology models of PvPGIP1 and <span class="gene">GmPGIP3</span> using the crystal structure of PvPGIP2 (1OGQ) as template. These PGIPs were then docked individually to FmPG to elucidate the characteristics of their interactions. The mode of binding for PvPGIP1 to FmPG considerably differs from the mode observed for PvPGIP2-FmPG complex, regardless of the <span class="trait">high</span> sequence similarity the two PGIPs share. Both PvPGIP2 and <span class="gene">GmPGIP3</span> despite being relatively less similar, interact with residues of FmPG that are known from mutational studies to constitute the active site of the enzyme. PvPGIP1 tends to interact with residues not located at the active site of FmPG. Looking into the electrostatic potential surface for individual PGIPs, it was evident that a portion of the interacting surface for PvPGIP1 differs from the corresponding region of PvPGIP2 or <span class="gene">GmPGIP3.</span> CONCLUSION: van der Waals and electrostatic interactions play an active role in PGIPs for proper recognition and discrimination of PGs. Docking studies reveal that PvPGIP2 and <span class="gene">GmPGIP3</span> interact with the residues constituting the active site of FmPG with implications that the proteins bind/block FmPG at its active site and thereby inhibit the enzyme.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Bean PGIPs are encoded by a family of genes [ <xref ref-type="bibr" rid="B21"> 21 </xref> ], the products of which are <italic> Pv </italic> PGIP1, <italic> Pv </italic> PGIP2, <italic> Pv </italic> PGIP3 <italic> and Pv </italic> PGIP4. All of these show distinct regulation and specificity exhibiting different inhibitory capabilities against the PGs of <italic> Botrytis cinerea </italic> , <italic> Colletotrichum gloeosporioides, Stenocarpella maydis, Fusarium moniliforme and Aspergillus niger </italic> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]. The four mature products of the <span class="gene">pgip</span> genes have a very <span class="trait">high</span> extent of sequence similarity differing between 8 to 81 amino acids. Phylogenetic analysis indicates that <italic> Pvpgip1-Pvpgip2 </italic> groups with <italic> <span class="gene">Gmpgip3</span> </italic> of soybean ( <italic> Glycine max </italic> ), a species of the Phaseoleae tribe close to bean, which suggests that these two genes are probably closer to the ancestral gene than are <italic> Pvpgip3 </italic> and <italic> Pvpgip4 </italic> [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. Interestingly, although PGIP1 and PGIP2 from <italic> Phaseolus vulgaris </italic> share 99% sequence similarity, <italic> Pv </italic> PGIP2 has been reported to inhibit PG from <italic> Fusarium moniliforme </italic> ( <italic> Fm </italic> ) while <italic> Pv </italic> PGIP1 does not [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. Strikingly, <italic> Gm </italic> PGIP3 of soybean ( <italic> Glycine max </italic> ), shares a much lesser similarity of 88% with <italic> Pv </italic> PGIP2 and yet has the ability to inhibit <italic> FmPG </italic> with an inhibition kinetics similar to that of <italic> Pv </italic> PGIP2 [ <xref ref-type="bibr" rid="B22"> 22 </xref> ]. It is thus evident that the ability of PGIP molecules to defend plants against infection is not buried in the extent of similarity shared between the sequences alone. Probable implications could be the presence of key structural features important for the necessary protein-protein interaction that need to be conserved to preserve the function.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The solution structure of the Mg2+ form of soybean calmodulin isoform 4 reveals unique features of plant calmodulins in resting cells.</h2>
			<h3>PMID:20054830  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huang H(1), Ishida H, Vogel HJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean calmodulin isoform 4 (sCaM4) is a plant calcium-binding <span class="trait">protein,</span> regulating cellular responses to the second messenger Ca(2+). We have found that the metal ion free (apo-) form of sCaM4 possesses a half unfolded structure, with the N-terminal domain unfolded and the C-terminal domain folded. This result was unexpected as the apo-forms of both soybean calmodulin isoform 1 (sCaM1) and mammalian <span class="gene">CaM</span> (mCaM) are fully folded. Because of the fact that free Mg(2+) ions are always present at <span class="trait">high</span> concentrations in cells (0.5-2 mM), we suggest that Mg(2+) should be bound to sCaM4 in nonactivated cells. CD studies revealed that in the presence of Mg(2+) the initially unfolded N-terminal domain of sCaM4 folds into an alpha-helix-rich structure, similar to the Ca(2+) form. We have used the NMR backbone residual dipolar coupling restraints (1)D(NH), (1)D(C alpha H alpha), and (1)D(C'C alpha) to determine the solution structure of the N-terminal domain of Mg(2+)-sCaM4 (Mg(2+)-sCaM4-NT). Compared with the known structure of Ca(2+)-sCaM4, the structure of the Mg(2+)-sCaM4-NT does not fully open the hydrophobic pocket, which was further confirmed by the use of the fluorescent probe ANS. Tryptophan fluorescence experiments were used to study the interactions between Mg(2+)-sCaM4 and CaM-binding peptides derived from smooth muscle myosin light chain kinase and plant glutamate decarboxylase. These results suggest that Mg(2+)-sCaM4 does not bind to Ca(2+)-CaM target peptides and therefore is functionally similar to apo-mCaM. The Mg(2+)- and apo-structures of the sCaM4-NT provide unique insights into the structure and function of some plant calmodulins in resting cells.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of <span class="gene">CHS7</span> and CHS8 genes in soybean.</h2>
			<h3>PMID:20016991  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yi J(1), Derynck MR, Chen L, Dhaubhadel S.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chalcone synthase (CHS) catalyzes the first reaction specific for flavonoid and isoflavonoid biosynthesis. The soybean genome consists of nine copies of <span class="gene">CHS</span> genes (CHS1-CHS9) and a duplicate copy of CHS1. Even though the soybean <span class="gene">CHS</span> gene family members share a <span class="trait">high</span> degree of sequence similarity, they play different roles during plant development or in response to environmental stimuli. Our previous work on the comparison of a global gene expression in two soybean cultivars that differ in the level of total isoflavonoid accumulation has denoted the involvement of <span class="gene">CHS7</span> and CHS8 genes in isoflavonoid synthesis. We have extended our effort to understand expression patterns of these two genes in soybean and in transgenic Arabidopsis. Promoter regions of <span class="gene">CHS7</span> and CHS8 genes were isolated and in silico analysis performed to investigate potential transcription factor binding sites (TFBSs). The TFBSs were verified by DNase I footprint analysis. Some unique and several common TFBSs were identified in <span class="gene">CHS7</span> and CHS8 promoters. We cloned beta-glucuronidase (GUS) under <span class="gene">CHS7</span> and CHS8 promoters and monitored the tissue-specific GUS expression in transformed Arabidopsis. Differential GUS activity was observed in young leaves, roots, and mature pod walls of transgenic <span class="gene">CHS7</span> promoter-GUS and CHS8 promoter-GUS plants. The tissue-specific expression patterns of <span class="gene">CHS7</span> and CHS8 genes were determined in soybean by quantitative RT-PCR. Both <span class="gene">CHS7</span> and CHS8 genes were expressed at higher levels in roots; however, overall expression pattern of these genes varied in different tissues. The results suggest that the structural diversity within <span class="gene">CHS7</span> and CHS8 promoters may lead into differential activation of these genes by different inducers as well as developmental stage- and tissue-specific differences in gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Prostaglandin <span class="gene">E1</span> encapsulated into lipid nanoparticles improves its anti-inflammatory effect with low side-effect.</h2>
			<h3>PMID:20006690  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao Y(1), Xu P, Chen L, Li Y.
				<br/> 
				年份：2010 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Prostaglandin E(1) (PGE1) shows various pharmacological activities including anti-inflammation. However, the rapid metabolization and inactivation of the intravenously administered PGE1 during the first passage through the lungs result in significant non-compliance in clinical trials which greatly limits its application. The aim of this work was to prepare the lipid nanoparticles loading PGE1 to improve its anti-inflammatory effect with low side-effect. The experimental results showed that PGE1 loaded lipid nanoparticles (PLNs) could be successfully prepared by <span class="trait">high</span> pressure homogenization with particle size 68.1+/-4.7 nm, zeta potential -3.32+/-0.37 mV and entrapment efficiency 92.1+/-1.3%. PLNs exhibited a sustained release with low burst drug release. PLNs could improve the inhibition effects of PGE1 on lipopolysaccharides (LPS)-induced TNF-alpha expression on macrophage RAW264.7 cells, and improve the inhibition of lymphocyte to endothelial cell adhesion and ICAM-1 adhesion molecule expression on HUVEC and MDA-MB-468 cell membrane. No allergenicity, vascular and muscle irritation were induced in animals by PLNs even at double of the highest drug concentration of clinical infusion. As a result, PLNs could be a more potential delivery system for PGE1 in the treatment of inflammation-related diseases.
					</p>
				
					<p>&nbsp;&nbsp;
						2009 Elsevier B.V. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Stress-induced co-expression of two alternative oxidase (VuAox1 and 2b) genes in Vigna unguiculata.</h2>
			<h3>PMID:20005596  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Costa JH(1), Mota EF, Cambursano MV, Lauxmann MA, de Oliveira LM, Silva Lima Mda G, Orellano EG, Fernandes de Melo D.
				<br/> 
				年份：2010 May 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cowpea (Vigna unguiculata) alternative oxidase is encoded by a small multigene family (Aox1, 2a and 2b) that is orthologous to the soybean Aox family. Like most of the identified Aox genes in plants, VuAox1 and VuAox2 consist of 4 exons interrupted by 3 introns. Alignment of the orthologous Aox genes revealed <span class="trait">high</span> identity of exons and intron variability, which is more prevalent in Aox1. In order to determine Aox gene expression in V. unguiculata, a steady-state analysis of transcripts involved in seed development (flowers, pods and dry seeds) and germination (soaked seeds) was performed and systemic co-expression of VuAox1 and VuAox2b was observed during germination. The analysis of Aox transcripts in leaves from seedlings under different stress conditions (cold, PEG, salicylate and H2O2 revealed stress-induced co-expression of both VuAox genes. Transcripts of VuAox2a and 2b were detected in all control seedlings, which was not the case for VuAox1 mRNA. Estimation of the primary transcript lengths of V. unguiculata and soybean Aox genes showed an intron length reduction for VuAox1 and 2b, suggesting that the two genes have converged in transcribed sequence length. Indeed, a bioinformatics analysis of VuAox1 and 2b promoters revealed a conserved region related to a cis-element that is responsive to oxidative stress. Taken together, the data provide evidence for co-expression of <span class="gene">Aox1</span> and <span class="gene">Aox2b</span> in response to stress and also during the early phase of seed germination. The dual nature of VuAox2b expression (constitutive and induced) suggests that the constitutive <span class="gene">Aox2b</span> gene of V. unguiculata has acquired inducible regulatory elements.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2009 Elsevier <span class="gene">GmbH.</span> All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">PM2</span> <span class="trait">protein</span> (LEA3) confers the tolerance of Escherichia coli and stabilization of enzyme activity under diverse stresses.</h2>
			<h3>PMID:19949793  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu Y(1), Zheng Y, Zhang Y, Wang W, Li R.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Late embryogenesis abundant (LEA) proteins are closely associated with the tolerance of diverse stresses in organisms. To elucidate the function of group 3 LEA proteins, the soybean <span class="gene">PM2</span> <span class="trait">protein</span> (LEA3) was expressed in E. coli and the protective function of the <span class="gene">PM2</span> <span class="trait">protein</span> was assayed both in vivo and in vitro. The results of a spot assay and survival ratio demonstrated that the expression of the <span class="gene">PM2</span> <span class="trait">protein</span> conferred the tolerance to the E. coli recombinant for different temperature conditions (4, -20 or 50 degrees C) or high-salinity stresses (120 mmol/l MgCl(2) or 120 mmol/l CaCl(2)). In addition, it was demonstrated that the in vitro addition of the <span class="gene">PM2</span> <span class="trait">protein</span> could prevent the lactate dehydrogenase (LDH) inactivation normally induced by freeze-thaw. In the 62 degrees C condition, the <span class="gene">PM2</span> <span class="trait">protein</span> (1:5 mass ratio to LDH) effectively prevented the LDH thermo-denaturation by acting synergistically with trehalose (62.5 microg/ml), although the <span class="gene">PM2</span> <span class="trait">protein</span> alone at this concentration showed little protective effect on LDH activity. Furthermore, the results showed that the <span class="gene">PM2</span> <span class="trait">protein</span> could partially prevent the thermo-denaturation of the bacterial proteome after boiling for 2 min. Based on these results, we propose that the <span class="gene">PM2</span> <span class="trait">protein</span> itself, or together with trehalose, conferred the tolerance to the E. coli recombinant against diverse stresses by protecting proteins and enzyme activity under low- or <span class="trait">high-</span> temperature conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:1-Aminocyclopropane-1-carboxylic acid (ACC) concentration and ACC synthase expression in soybean roots, root tips, and soybean cyst nematode (Heterodera glycines)-infected roots.</h2>
			<h3>PMID:19861652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Xue P, Yang R.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Colonization of plant roots by root knot and cyst nematodes requires a functional ethylene response pathway. However, ethylene plays many roles in root development and whether its role in nematode colonization is direct or indirect, for example lateral root initiation or root hair growth, is not known. The temporal requirement for ethylene and localized synthesis of ethylene during the life span of soybean cyst nematode (SCN) on soybean roots was further investigated. Although a significant increase in ethylene evolution was not detected from SCN-colonized roots, the concentration of the immediate precursor to ethylene, 1-aminocyclopropane-1-carboxylic acid (ACC), was higher in SCN-colonized root pieces and root tips than in other parts of the root. Moreover, expression analysis of 17 ACC synthase (ACS) genes indicated that a select set of <span class="gene">ACS</span> genes is expressed in SCN-colonized root pieces that is clearly different from the set of genes expressed in non-colonized roots or root tips. Semi-quantitative real-time PCR indicated that <span class="gene">ACS</span> transcript accumulation correlates with the <span class="trait">high</span> concentration of ACC in root tips. In addition, an ACS-like sequence was found in the public SCN nucleotide database. Acquisition of a full-length sequence for this mRNA (accession GQ389647) and alignment with transcripts for other well-characterized <span class="gene">ACS</span> proteins indicated that the nematode sequence is missing a key element required for <span class="gene">ACS</span> activity and therefore probably is not a functional ACS. Moreover, no significant amount of ACC was found in any growth stage of SCN that was tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural and synthetic 2'-hydroxy-chalcones and aurones: synthesis, characterization and evaluation of the antioxidant and soybean lipoxygenase inhibitory activity.</h2>
			<h3>PMID:19853459  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Detsi A(1), Majdalani M, Kontogiorgis CA, Hadjipavlou-Litina D, Kefalas P.
				<br/> 
				年份：2009 Dec 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A series of 2'-hydroxy-chalcones and their oxidative cyclization products, aurones, have been synthesized and tested for their antioxidant and lipoxygenase inhibitory activity. The natural product aureusidin (31) was synthesized in <span class="trait">high</span> <span class="trait">yield</span> by a new approach. An extensive structure-relationship study was performed and revealed that several chalcones and aurones possess an appealing pharmacological profile combining <span class="trait">high</span> antioxidant and lipid peroxidation activity with potent soybean <span class="gene">LOX</span> inhibition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Absorption kinetics are a key factor regulating postprandial <span class="trait">protein</span> metabolism in response to qualitative and quantitative variations in <span class="trait">protein</span> intake.</h2>
			<h3>PMID:19812354  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fouillet H(1), Juillet B, Gaudichon C, Mariotti F, Tomé D, Bos C.
				<br/> 
				年份：2009 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have previously demonstrated that increasing the habitual <span class="trait">protein</span> intake widened the <span class="gene">gap</span> in nutritional quality between proteins through mechanisms that are not yet fully understood. We hypothesized that the differences in gastrointestinal kinetics between dietary proteins were an important factor affecting their differential response to an increased <span class="trait">protein</span> intake. To test this hypothesis, we built a 13-compartment model providing integrative insight into the sequential dynamics of meal nitrogen (Nm) absorption, splanchnic uptake, and metabolism, and subsequent peripheral transfer and deposition. The model was developed from data on postprandial Nm kinetics in certain accessible pools, obtained from subjects having ingested a (15)N-labeled milk or soy <span class="trait">protein</span> meal, after adaptation to normal (NP) or <span class="trait">high</span> (HP) <span class="trait">protein</span> diets. The faster absorption of Nm after soy vs. milk caused its earlier and stronger splanchnic delivery, which favored its local catabolic utilization (up to +30%) and limited its peripheral accretion (down to -20%). Nm absorption was also accelerated after HP vs. NP adaptation, and this kinetic effect accounted for most of the HP-induced increase (up to +20%) in splanchnic Nm catabolic use, and the decrease (down to -25%) in peripheral Nm anabolic utilization. The HP-induced acceleration in Nm absorption was more pronounced with soy than with milk, as were the HP effects on Nm regional metabolism. Our integrative approach identified Nm absorption kinetics, which exert a direct and lasting impact on Nm splanchnic catabolic use and peripheral delivery, as being critical in adaptation to both qualitative and quantitative changes in <span class="trait">protein</span> intake.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of a chickpea (Cicer arietinum L.) NAC family gene, CarNAC5, which is both developmentally- and stress-regulated.</h2>
			<h3>PMID:19800808  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Peng H(1), Cheng HY, Yu XW, Shi QH, Zhang H, Li JG, Ma H.
				<br/> 
				年份：2009 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been documented that the plant-specific NAC (for NAM, ATAF1,2 and CUC2) transcription factors play an important role in plant development and stress responses. In this study, a chickpea NAC gene CarNAC5 (for Cicer arietinum L. NAC gene 5) was isolated from a cDNA library from chickpea leaves treated by polyethylene glycol (PEG). CarNAC5, as a single/low copy gene, contained three exons and two introns within genomic DNA sequence and encoded a polypeptide with 291 amino acids. CarNAC5 <span class="trait">protein</span> had a conserved NAC domain in the N-terminus and showed <span class="trait">high</span> similarity to other NACs, especially ATAF subgroup members. The CarNAC5:GFP fusion <span class="trait">protein</span> was localized in the nucleus of onion epidermal cells. Furthermore, CarNAC5 <span class="trait">protein</span> activated the reporter genes LacZ and <span class="gene">HIS3</span> in yeast. The transactivation activity was mapped to the C-terminal region. The transcripts of CarNAC5 appeared in many chickpea tissues including seedling leaves, stems, roots, flowers, seeds and pods, but mostly accumulated in flowers. Meanwhile, CarNAC5 was strongly expressed during seed maturation and in embryos of the early germinating seeds. It was also significantly induced by <span class="trait">drought,</span> heat, wounding, salicylic acid (SA), and indole-3-acetic acid (IAA) treatments. Our results suggest that CarNAC5 encodes a novel NAC-domain <span class="trait">protein</span> and acts as a transcriptional activator involved in plant developmental regulation and various stress responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Systemic acquired <span class="trait">resistance</span> in soybean is regulated by two proteins, Orthologous to Arabidopsis NPR1.</h2>
			<h3>PMID:19656407  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sandhu D(1), Tasma IM, Frasch R, Bhattacharyya MK.
				<br/> 
				年份：2009 Aug 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Systemic acquired <span class="trait">resistance</span> (SAR) is induced in non-inoculated leaves following infection with certain pathogenic strains. SAR is effective against many pathogens. Salicylic acid (SA) is a signaling molecule of the SAR pathway. The development of SAR is associated with the induction of pathogenesis related (PR) genes. Arabidopsis non-expressor of PR1 (NPR1) is a regulatory gene of the SA signal pathway 123. SAR in soybean was first reported following infection with Colletotrichum trancatum that causes anthracnose disease. We investigated if SAR in soybean is regulated by a pathway, similar to the one characterized in Arabidopsis. RESULTS: Pathogenesis-related gene <span class="gene">GmPR1</span> is induced following treatment of soybean plants with the SAR inducer, 2,6-dichloroisonicotinic acid (INA) or infection with the oomycete pathogen, Phytophthora sojae. In P. sojae-infected plants, SAR was induced against the bacterial pathogen, Pseudomonas syringae pv. glycinea. Soybean <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> genes showed <span class="trait">high</span> identities to Arabidopsis NPR1. They showed similar expression patterns among the organs, studied in this investigation. <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> are the only soybean homologues of NPR1and are located in homoeologous regions. In <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> transformed Arabidopsis npr1-1 mutant plants, SAR markers: (i) PR-1 was induced following INA treatment and (ii) BGL2 following infection with Pseudomonas syringae pv. tomato (Pst), and SAR was induced following Pst infection. Of the five cysteine residues, Cys82, Cys150, Cys155, Cys160, and Cys216 involved in oligomer-monomer transition in NPR1, Cys216 in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> proteins was substituted to Ser and Leu, respectively. CONCLUSION: Complementation analyses in Arabidopsis npr1-1 mutants revealed that homoeologous <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> genes are orthologous to Arabidopsis NPR1. Therefore, SAR pathway in soybean is most likely regulated by <span class="gene">GmNPR1</span> genes. Substitution of Cys216 residue, essential for oligomer-monomer transition of Arabidopsis NPR1, with Ser and Leu residues in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2,</span> respectively, suggested that there may be differences between the regulatory mechanisms of <span class="gene">GmNPR1</span> and Arabidopsis NPR proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phylogeny and taxonomy of a diverse collection of Bradyrhizobium strains based on multilocus sequence analysis of the 16S rRNA gene, ITS region and glnII, recA, <span class="gene">atpD</span> and <span class="gene">dnaK</span> genes.</h2>
			<h3>PMID:19628593  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Menna P(1), Barcellos FG, Hungria M.
				<br/> 
				年份：2009 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genus Bradyrhizobium encompasses a variety of bacteria that can live in symbiotic and endophytic associations with legumes and non-legumes, and are characterized by physiological and symbiotic versatility and broad geographical distribution. However, despite indications of great genetic variability within the genus, only eight species have been described, mainly because of the highly conserved nature of the 16S rRNA gene. In this study, 169 strains isolated from 43 different legumes were analysed by rep-PCR with the BOX primer, by sequence analysis of the 16S rRNA gene and the 16S-23S rRNA intergenic transcribed spacer (ITS) and by multilocus sequence analysis (MLSA) of four housekeeping genes, glnII, recA, <span class="gene">atpD</span> and dnaK. Considering a cut-off at a level of 70 % similarity, 80 rep-PCR profiles were distinguished, which, together with type strains, were clustered at a very low level of similarity (24 %). In both single and concatenated analyses of the 16S rRNA gene and ITS sequences, two large groups were formed, with bootstrap support of 99 % in the concatenated analysis. The first group included the type and/or reference strains of Bradyrhizobium japonicum, B. betae, B. liaoningense, B. canariense and B. yuanmingense and B. japonicum USDA 110, and the second group included strains related to Bradyrhizobium elkanii USDA 76(T), B. pachyrhizi PAC48(T) and B. jicamae PAC68(T). Similar results were obtained with MLSA of glnII, recA, <span class="gene">atpD</span> and dnaK. Greatest variability was observed when the <span class="gene">atpD</span> gene was amplified, and five strains related to B. elkanii revealed a level of variability never reported before. Another important observation was that a group composed of strains USDA 110, SEMIA 5080 and SEMIA 6059, all isolated from soybean, clustered in all six trees with <span class="trait">high</span> bootstrap support and were quite distinct from the clusters that included B. japonicum USDA 6(T). The results confirm that MLSA is a rapid and reliable way of providing information on phylogenetic relationships and of identifying rhizobial strains potentially representative of novel species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the soybean <span class="gene">GmERF3</span> gene, an AP2/ERF type transcription factor for increased tolerances to salt, <span class="trait">drought,</span> and diseases in transgenic tobacco.</h2>
			<h3>PMID:19602544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Li L, Xu Z, Chen X, Guo J, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new member of the AP2/ERF transcription factor family, <span class="gene">GmERF3,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF3</span> contained an AP2/ERF domain of 58 amino acids and two putative nuclear localization signal (NLS) domains. It belonged to a group IV <span class="trait">protein</span> in the ERF (ethylene response factor) subfamily as typified by a conserved N-terminal motif [MCGGAI(I/L)]. Expression of <span class="gene">GmERF3</span> was induced by treatments with <span class="trait">high</span> salinity, <span class="trait">drought,</span> abscisic acid (ABA), salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and soybean mosaic virus (SMV), whereas there was no significant <span class="gene">GmERF3</span> mRNA accumulation under cold stress treatment. <span class="gene">GmERF3</span> could bind to the GCC box and DRE/CRT element, and was targeted to the nucleus when transiently expressed in onion epidermal cells. The <span class="gene">GmERF3</span> <span class="trait">protein</span> fused to the GAL4 DNA-binding domain to activate transcription of reporter genes in yeast. Ectopic expression of the <span class="gene">GmERF3</span> gene in transgenic tobacco plants induced the expression of some PR genes and enhanced <span class="trait">resistance</span> against infection by Ralstonia solanacearum, Alternaria alternata, and tobacco mosaic virus (TMV), and gave tolerance to <span class="trait">high</span> salinity and dehydration stresses. Furthermore, overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions. The overall results suggested that <span class="gene">GmERF3</span> as an AP2/ERF transcription factor may play dual roles in response to biotic and abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Expressing and functional analysis of <span class="gene">GmMYBJ6</span> from soybean].</h2>
			<h3>PMID:19586866  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang WJ(1), Wu YM, Tang YX.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB transcription factor is one of the largest families in plants, which plays an important role in regulating plant development and physiological metabolism. In this study, the expression and function of the new MYB transcription factor gene <span class="gene">GmMYBJ6</span> (GenBank No. DQ902863), isolated from soybean (Glycine max L.), were characterized. The expression pattern of <span class="gene">GmMYBJ6</span> in different organs was examined using Northern blotting analysis. The expression of <span class="gene">GmMYBJ6</span> was detected only in the leaves. The transcriptional activation ability of <span class="gene">GmMYBJ6</span> <span class="trait">protein</span> was confirmed by the yeast assay system and the activity of beta-galactosidase was 28.48 U/mL. The green fluorescent <span class="trait">protein</span> expression vector p163-GFP-GmMYBJ6 was constructed and transformed into the epidermal cells of onion via particle bombardmental method. The results of instantaneous expression showed that <span class="gene">GmMYBJ6</span> proteins were localized in cell nucleus. Semi-quantitative RT-PCR analysis indicated that <span class="gene">GmMYBJ6</span> improved the expression of certain flavonoid biosynthetic genes, such as <span class="gene">PAL</span> (Phenylalanine ammonia lyase), <span class="gene">C4H</span> (cinnamate-4-hydroxylase), <span class="gene">4CL</span> (4-coumaroyl-CoA ligase), <span class="gene">CHS</span> (Chalcone synthase), <span class="gene">CHI</span> (Chalcone isomerase), <span class="gene">F3H</span> (Flavanone 3-hydroxylase), and FLS (Flavonol synthase), resulting an increase of the total flavonoid levels in positive tobacco transformants. Additionally, the increasing expression of <span class="gene">GmMYBJ6</span> in soybean cultivar Zhongdou 27, induced by UV-B radiation, <span class="trait">drought,</span> and high-salt treatment, indicated that <span class="gene">GmMYBJ6</span> was associated with response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bradyrhizobium pachyrhizi sp. nov. and Bradyrhizobium jicamae sp. nov., isolated from effective nodules of Pachyrhizus erosus.</h2>
			<h3>PMID:19567584  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ramírez-Bahena MH(1), Peix A, Rivas R, Camacho M, Rodríguez-Navarro DN, Mateos PF, Martínez-Molina E, Willems A, Velázquez E.
				<br/> 
				年份：2009 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Several strains isolated from the legume Pachyrhizus erosus were characterized on the basis of diverse genetic, phenotypic and symbiotic approaches. These novel strains formed two groups closely related to Bradyrhizobium elkanii according to their 16S rRNA gene sequences. Strains PAC48T and PAC68T, designated as the type strains of these two groups, presented 99.8 and 99.1% similarity, respectively, in their 16S rRNA gene sequences with respect to B. elkanii USDA 76T. In spite of these <span class="trait">high</span> similarity values, the analysis of additional phylogenetic markers such as <span class="gene">atpD</span> and <span class="gene">glnII</span> genes and the 16S-23S intergenic spacer (ITS) showed that strains PAC48T and PAC68T represented two separate novel species of the genus Bradyrhizobium with B. elkanii as their closest relative. Phenotypic differences among the novel strains isolated from Pachyrhizus and B. elkanii were found regarding the assimilation of carbon sources and antibiotic <span class="trait">resistance.</span> All these differences were congruent with DNA-DNA hybridization analysis which revealed 21% genetic relatedness between strains PAC48T and PAC68T and 46% and 25%, respectively, between these strains and B. elkanii LMG 6134T. The <span class="gene">nodD</span> and <span class="gene">nifH</span> genes of strains PAC48T and PAC68T were phylogenetically divergent from those of bradyrhizobia species that nodulate soybean. Soybean was not nodulated by the novel Pachyrhizus isolates. Based on the genotypic and phenotypic data obtained in this study, the new strains represent two novel species for which the names Bradyrhizobium pachyrhizi sp. nov. (type strain PAC48T=LMG 24246T=CECT 7396T) and Bradyrhizobium jicamae sp. nov. (type strain PAC68T=LMG 24556T=CECT 7395T) are proposed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Map-based cloning of the gene associated with the soybean maturity locus E3.</h2>
			<h3>PMID:19474204  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Watanabe S(1), Hideshima R, Xia Z, Tsubokura Y, Sato S, Nakamoto Y, Yamanaka N, Takahashi R, Ishimoto M, Anai T, Tabata S, Harada K.
				<br/> 
				年份：2009 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosensitivity plays an essential role in the response of plants to their changing environments throughout their life cycle. In soybean [Glycine max (L.) Merrill], several associations between photosensitivity and maturity loci are known, but only limited information at the molecular level is available. The FT3 locus is one of the quantitative trait loci (QTL) for flowering time that corresponds to the maturity locus E3. To identify the gene responsible for this QTL, a map-based cloning strategy was undertaken. One phytochrome A gene <span class="gene">(GmPhyA3)</span> was considered a strong candidate for the FT3 locus. Allelism tests and gene sequence comparisons showed that alleles of Misuzudaizu (FT3/FT3; JP28856) and Harosoy (E3/E3; PI548573) were identical. The <span class="gene">GmPhyA3</span> alleles of Moshidou Gong 503 (ft3/ft3; JP27603) and L62-667 (e3/e3; PI547716) showed weak or complete loss of function, respectively. <span class="trait">High</span> red/far-red (R/FR) long-day conditions enhanced the effects of the E3/FT3 alleles in various genetic backgrounds. Moreover, a mutant line harboring the nonfunctional <span class="gene">GmPhyA3</span> flowered earlier than the original Bay (E3/E3; PI553043) under similar conditions. These results suggest that the variation in phytochrome A may contribute to the complex systems of soybean flowering response and geographic adaptation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and characterization of the soybean DEAD-box RNA helicase gene induced by low temperature and <span class="trait">high</span> salinity stress.</h2>
			<h3>PMID:19463922  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chung E(1), Cho CW, Yun BH, Choi HK, So HA, Lee SW, Lee JH.
				<br/> 
				年份：2009 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel gene encoding a DEAD-box RNA helicase designated as <span class="gene">GmRH</span> was isolated from soybean. Amino acid sequence alignment and phylogenetic tree analysis revealed a close relationship between <span class="gene">GmRH</span> and other orthologous DEAD-box RNA helicases from other plant species. Structural motif analysis revealed that the bipartite lysine rich nuclear localization signal (NLS) is present in the N-terminal variable region of <span class="gene">GmRH</span> and that there are ten conserved motifs found in DEAD-box RNA helicase proteins. Southern blot analysis revealed the presence of 2 copies of <span class="gene">GmRH</span> in the soybean genome. Northern blot analysis demonstrated that the RNA expression of the <span class="gene">GmRH</span> was induced during low temperature or <span class="trait">high</span> salinity stress, but not by the exogenous application of abscisic acid or <span class="trait">drought</span> stress. Subcellular localization studies showed that <span class="gene">GmRH((1-355))-GFP</span> is localized in the nucleus, whereas <span class="gene">GmRH((130-355))-GFP</span> is localized both in the cytoplasm and in the nucleus. This provides the evidence that the N-terminal region predicted as NLS is essential for nuclear targeting of the <span class="gene">GmRH</span> <span class="trait">protein</span> in the plant cell. Purified GST-GmRH recombinant <span class="trait">protein</span> was shown to unwind dsRNA independent of ATP in vitro. Here, we propose that <span class="gene">GmRH</span> plays an important role in RNA processing during low temperature and <span class="trait">high</span> salinity stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of dietary factors and the NAT2 acetylator status on gastric cancer in Koreans.</h2>
			<h3>PMID:19350634  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang YW(1), Eom SY, Kim YD, Song YJ, Yun HY, Park JS, Youn SJ, Kim BS, Kim H, Hein DW.
				<br/> 
				年份：2009 Jul 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Environmental dietary carcinogens and genetic polymorphisms in metabolic enzymes have been reported to be the risk factors for gastric cancer. This study was undertaken to investigate the effects of the diet, the N-acetyltransferase (NAT) 2 acetylation status and their interaction on gastric cancer risk. The study population consisted of 471 gastric cancer patients and 471 age- and sex-matched control subjects. NAT2 genotypes were identified using single-nucleotide primer extension reaction methods. Thirty-one alleles related to 12 polymorphism sites were assayed in this study. Significantly increased odds ratios were observed in former smokers (OR = 2.39, 95% <span class="gene">CI</span> = 1.57-3.62), heavy drinkers (OR = 1.28, 95% <span class="gene">CI</span> = 1.06-1.55) and individuals who eat well-done meat (OR = 1.24, 95% <span class="gene">CI</span> = 1.09-1.41). The odds ratios (95% CI) for <span class="trait">high</span> intake of kimchi, stews and soybean paste were 3.27 (2.44-4.37), 1.96 (1.50-2.58) and 1.63 (1.24-2.14), respectively. The NAT2 genotype alone was not associated with gastric cancer risk. A significant gene-environment interaction was observed between environmental carcinogens and NAT2 genotypes. The odds ratios for kimchi, stews and soybean paste were higher in slow/intermediate acetylators than in rapid acetylators. The odds ratios for slow/intermediate acetylators were 2.28 (95% CI: 1.29-4.04) for light smokers and 3.42 (95% CI: 2.06-5.68) for well-done meat intake. The NAT2 acetylator genotype may be an important modifier of the effects of environmental factors on gastric cancer risk.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two <span class="gene">MATE</span> proteins play a role in iron efficiency in soybean.</h2>
			<h3>PMID:19342121  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rogers EE(1), Wu X, Stacey G, Nguyen HT.
				<br/> 
				年份：2009 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Iron is a necessary but often limiting nutrient for plant growth and development. Soybeans grown on the high-pH calcareous soils are especially prone to developing iron deficiency <span class="trait">chlorosis</span> and suffering the resultant <span class="trait">yield</span> losses. Once iron is transported into the root, it must be translocated from the root to the shoot where it is needed for photosynthesis and other processes. Previous work has indicated that iron is likely to move through the xylem as ferric citrate. In Arabidopsis thaliana, citrate is effluxed into the xylem by the ferric reductase defective3 (FRD3) <span class="trait">protein.</span> Here, we present the identification and characterization of two soybean genes, <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b,</span> with similar sequence and function to AtFRD3. The expression of both <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b</span> is induced by iron deficiency in the iron-efficient reference cultivar Williams 82. <span class="gene">GmFRD3b,</span> but not <span class="gene">GmFRD3a,</span> is expressed at higher levels in the iron-efficient cultivar Clark than in its iron-inefficient near isogenic line iso-Clark (iso), likely accounting for the higher xylem citrate levels in Clark. Increased xylem citrate levels lead to increased solubility of ferric iron in Clark xylem exudate as compared to iso-Clark exudate. These results support the hypothesis that <span class="trait">high</span> xylem citrate levels are needed for efficient root to shoot translocation of iron. Along with efficient ferric chelate reductase activity and root iron uptake activity, <span class="trait">high</span> expression levels of FRD3 genes are also proposed as a target for future iron efficiency breeding projects.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of two low phytic acid mutations on seed quality and nutritional traits in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:19323582  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan FJ(1), Zhu DH, Deng B, Fu XJ, Dong DK, Zhu SL, Li BQ, Shu QY.
				<br/> 
				年份：2009 May 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reduction of phytic acid in soybean seeds has the potential to improve the nutritional value of soybean meal and lessen phosphorus pollution in large scale animal farming. The objective of this study was to assess the effect of two new low phytic acid (LPA) mutations on seed quality and nutritional traits. Multilocation/season comparative analyses showed that the two mutations did not affect the concentration of crude <span class="trait">protein,</span> any of the individual amino acids, crude oil, and individual saturated fatty acids. Among other traits, <span class="gene">Gm-lpa-TW75-1</span> had consistently higher sucrose contents (+47.4-86.1%) and lower raffinose contents (-74.2 to -84.3%) than those of wild type (WT) parent Taiwan 75; <span class="gene">Gm-lpa-ZC-2</span> had higher total isoflavone contents (3038.8-4305.4 microg/g) than its parent Zhechun # 3 (1583.6-2644.9 microg/g) in all environments. Further tests of homozygous F(3) progenies of the cross <span class="gene">Gm-lpa-ZC-2</span> x Wuxing # 4 (WT variety) showed that LPA lines had a mean content of total isoflavone significantly higher than WT lines. This study demonstrated that two LPA mutant genes have no negative effects on seed quality and nutritional traits; they instead have the potential to improve a few other properties. Therefore, these two mutant genes are valuable genetic resources for breeding <span class="trait">high</span> quality soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean product intake modifies the association between interleukin-10 genetic polymorphisms and gastric cancer risk.</h2>
			<h3>PMID:19321591  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ko KP(1), Park SK, Cho LY, Gwack J, Yang JJ, Shin A, Kim CS, Kim Y, Kang D, Chang SH, Shin HR, Yoo KY.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, our aim was to investigate the association of inflammation-related genetic polymorphisms and gastric cancer risk and to examine whether the combined effect of soybean product intake modified cancer risk. Eighty-four incident gastric cancer cases and 336 matched controls were selected from the Korean Multi-Center Cancer Cohort. We selected 14 single nucleotide polymorphisms (SNP) from 5 genes [interleukin (IL)-1beta, IL-2, IL-4, IL-8, and IL-10] and used unconditional logistic regression model to calculate the odds ratios (OR) and 95% <span class="gene">CI</span> adjusting for H. pylori seropositivity, smoking, age, sex, enrollment year, and residential area. The risk for gastric cancer in relation to genetic polymorphisms and haplotypes were assessed according to soybean product intake levels. Although no single SNP effect was found, the combined effect between IL-10 gene variants of -592 GG/GA, -819 TC/CC, or -1082 AG/GG and low intake of soybean products had an increased risk for gastric cancer compared with the group with no risk gene variants and a <span class="trait">high</span> intake of soybean products (OR [95% CI] = 2.82 [1.04-7.62], 2.75 [1.02-7.44], and 4.34 [1.51-12.5], respectively). Among the low-soybean product intake group, IL-10 CCG haplotype had an increased risk of gastric cancer (OR = 3.38 [1.40-8.13]) relative to the ATA haplotype. Our results suggest that the association between IL-10 genetic polymorphisms and gastric cancer risk was modified by soybean product intake.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">mntH</span> gene encodes the major Mn(2+) transporter in Bradyrhizobium japonicum and is regulated by manganese via the Fur <span class="trait">protein.</span></h2>
			<h3>PMID:19298371  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hohle TH(1), O'Brian MR.
				<br/> 
				年份：2009 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The bacterial Nramp family <span class="trait">protein</span> MntH is a divalent metal transporter, but <span class="gene">mntH</span> mutants have little or no phenotype in organisms where it has been studied. Here, we identify the <span class="gene">mntH</span> homologue of Bradyrhizobium japonicum, and demonstrate that it is essential for Mn(2+) transport and for maintenance of cellular manganese homeostasis. Transport activity was induced under manganese deficiency, and Fe(2+) did not compete with (54)Mn(2+) for uptake by cells. The steady-state level of <span class="gene">mntH</span> mRNA was negatively regulated by manganese, but was unaffected by iron. Control of <span class="gene">mntH</span> expression and Mn(2+) transport by manganese was lost in a fur strain, resulting in constitutively <span class="trait">high</span> activity. Fur protected a 35 bp region of the <span class="gene">mntH</span> promoter in DNase I footprinting analysis that includes three imperfect direct repeat hexamers that are needed for full occupancy. Mn(2+) increased the affinity of Fur for the <span class="gene">mntH</span> promoter by over 50-fold, with a K(d) value of 2.2 nM in the presence of metal. The findings identify MntH as the major Mn(2+) transporter in B. japonicum, and show that Fur is a manganese-responsive regulator in that organism. Furthermore, Fe(2+) is neither a substrate for MntH nor a regulator of <span class="gene">mntH</span> expression in vivo.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of stress-inducible <span class="gene">GmNAC</span> genes in soybean.</h2>
			<h3>PMID:19277718  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tran LS(1), Quach TN, Guttikonda SK, Aldrich DL, Kumar R, Neelakandan A, Valliyodan B, Nguyen HT.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is detrimental to plant growth and development, and often results in significant losses to the yields of economically important crops such as soybeans (Glycine max L.). NAC transcription factors (TFs), which consist of a large family of plant-specific TFs, have been reported to enhance <span class="trait">drought</span> tolerance in a number of plants. In this study, 31 unigenes that contain the complete open reading frames encoding <span class="gene">GmNAC</span> proteins were identified and cloned from soybean. Analysis of C-terminal regulatory domain using yeast one-hybrid system indicated that among 31 <span class="gene">GmNAC</span> proteins, 28 have transcriptional activation activity. Expression analysis of these <span class="gene">GmNAC</span> genes showed that they are differentially expressed in different organs, suggesting that they have diverse functions during plant growth and development. To search for the drought-inducible <span class="gene">GmNAC</span> genes, we prescreened and re-confirmed by quantitative real-time PCR analysis that nine <span class="gene">GmNAC</span> genes are induced by dehydration stress with differential induction levels in both shoot and root. The expression profiles of these nine <span class="gene">GmNAC</span> genes were also examined under other stresses such as <span class="trait">high</span> salinity, cold and with abscisic acid hormone treatments. Phylogenetic analysis of the <span class="gene">GmNAC</span> proteins with previously reported drought-inducible NAC proteins of Arabidopsis and rice revealed that the nine drought-inducible <span class="gene">GmNAC</span> proteins belong to the "stress-inducible" NAC group. The results of this systematic analysis of the <span class="gene">GmNAC</span> family will provide novel tools and resources for the development of improved <span class="trait">drought</span> tolerant transgenic soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lipoxygenase independent hexanal formation in isolated soy proteins induced by reducing agents.</h2>
			<h3>PMID:19241535  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lei Q(1), Boatright WL.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Compared to corresponding controls, 6.5 mM dithiothreitol (DTT) elevated headspace hexanal level over aqueous slurries of both commercial isolated soy proteins (ISP) and laboratory ISP prepared with 80 degrees C treatment. Further analysis revealed that lipoxygenase (LOX) activity was not detected from these ISP, indicating that <span class="gene">LOX</span> is not involved in the observed hexanal increase. Levels of the induced headspace hexanal over the ISP aqueous slurries were proportional to the amount of DTT added in the range of 0 to 65 mM. Subsequent systematic investigations with model systems revealed that iron was required for the reducing agent-induced hexanal formation from linoleic acid. Erythorbate, another reducing agent, can also induce hexanal formation in both ISP and model systems. As a comparison, the <span class="gene">LOX</span> activity and hexanal synthesis in defatted soy flour were examined. The corresponding results showed that defatted soy flour maintained <span class="trait">high</span> <span class="gene">LOX</span> activities and that hexanal synthesis in such sample was significantly inhibited by <span class="trait">high</span> concentration DTT (above 130 mM). Data from the current investigation demonstrate the existence of <span class="gene">LOX</span> independent hexanal formation induced by reducing agents in ISP and the potential requirement of iron as a catalyst.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy isoflavones have an antiestrogenic effect and alter mammary promoter hypermethylation in healthy premenopausal women.</h2>
			<h3>PMID:19235040  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Qin W(1), Zhu W, Shi H, Hewett JE, Ruhlen RL, MacDonald RS, Rottinghaus GE, Chen YC, Sauter ER.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We determined if soy isoflavones have dose-related estrogenic and methylation effects. Thirty-four healthy premenopausal women were randomized to 40 mg or 140 mg isoflavones daily through one menstrual cycle. Breast specific and systemic estrogenic effects were assessed measuring the estrogenic marker complement (C)3 and changes in cytology, whereas methylation assessment of 5 cancer related genes (p16, RASSF1A, RARbeta2, ER, and CCND2) was performed on intraductal specimens. Serum genistein significantly increased after consuming both isoflavone doses. Cytology did not significantly change at either isoflavone dose. Serum <span class="gene">C3</span> levels posttreatment were inversely related to change in serum genistein (r =-0.76, P = 0.0045) in women consuming low but not <span class="trait">high</span> dose isoflavones. The RAR beta 2 hypermethylation increase posttreatment correlated with the posttreatment genistein level considering the entire group (r = 0.67, P = 0.0017) and those receiving high-dose isoflavones (r = 0.68, P = 0.021). At the low but not the <span class="trait">high</span> isoflavone dose, CCND2 hypermethylation increase correlated with posttreatment genistein levels (r = 0.79, P = 0.011). In summary, the inverse correlation between <span class="gene">C3</span> and genistein suggests an antiestrogenic effect. Isoflavones induced dose-specific changes in RARbeta2 and CCND2 gene methylation, which correlated with genistein levels. This work provides novel insights into estrogenic and methylation effects of dietary isoflavones.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean (Glycine max) polyubiquitin promoter gives strong constitutive expression in transgenic soybean.</h2>
			<h3>PMID:19229538  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hernandez-Garcia CM(1), Martinelli AP, Bouchard RA, Finer JJ.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The success of plant genetic transformation relies greatly on the strength and specificity of the promoters used to drive genes of interest. In this study, we analyzed gfp gene expression mediated by a polyubiquitin promoter <span class="gene">(Gmubi)</span> from soybean (Glycine max) in stably transformed soybean tissues. Strong GFP expression was observed in stably transformed proliferative embryogenic tissues. In whole transgenic plants, GFP expression was observed in root tips, main and lateral roots, cotyledons and plumules in young plants as well as in leaf veins, petioles, flower petals, pollen, pods and developing seeds in mature plants. GFP expression was localized mainly in epidermal cells, leaf mesophyll, procambium and vascular tissues. Introduction of an intron-less version of the <span class="gene">Gmubi</span> promoter <span class="gene">(Gmupri)</span> displayed almost the same GFP expression pattern albeit at lower intensities. The <span class="gene">Gmubi</span> promoter showed <span class="trait">high</span> levels of constitutive expression and represents an alternative to viral promoters for driving gene expression in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of recombinant porcine colipase secreted by Pichia pastoris and its application to improve dietary fat digestion and growth of postweaning piglets.</h2>
			<h3>PMID:19194948  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu FC(1), Chen HL, Chong KY, Hsu AL, Chen CM.
				<br/> 
				年份：2008 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recombinant porcine pancreatic colipase (pCoL) was produced in the methylotrophic yeast Pichia pastoris. A synthetic yeast secretion cassette was constructed with the constitutive promoter of the glyceraldehyde-3-phosphate-dehydrogenase (GAPDH) gene and the yeast alpha-mating factor signal peptide. The pCoL cDNA corresponding in the coding sequence, excluding a 16-amino acid segment of the native signal sequence, was cloned into the pGAPZalphaB vector and integrated into the genome of P. pastoris. Yeast transformants were cultured and bioactive pCoL <span class="trait">protein</span> was detected in the supernatant at a high-level of 126.8 mg/L after 3 days of culture. The transformed yeast containing the highest recombinant colipase level (pCoL yeast) and native yeast <span class="gene">GS</span> 115 not containing pCoL (non-pCoL yeast, as a control group) were separately cultured and the supernatants were adsorbed by dried skim milk. In an animal trial, two concentrations of colipase activity (0 vs. 5,000 U/kg in the diet) were blended with the pig corn-soybean basal diet and fed to weaned piglets for 4 weeks. The pCoL-administrated test group gained significantly more weight than piglets in the control group when measured at Day 15 (11.84 +/- 0.70 vs. 10.59 +/- 0.39 kg, P &lt; 0.05), Day 22 (15.84 +/- 0.95 vs. 14.32 +/- 0.59 kg, P &lt; 0.01), and Day 28 (20.19 +/- 1.47 vs. 18.54 +/- 0.92 kg, P &lt; 0.01) after weaning. The blood triglyceride (TG) concentrations were significantly increased in the experimental test group that received recombinant colipase on the 28th day of postweaning when compared with that of the control group (32.50 vs. 16.37 mg/dL; P &lt; 0.0001). These experimental data suggest that the use of recombinant porcine colipase as a dietary supplement provides an alternative approach for improving fat digestion and enhancing growth in postweaning piglets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A breakfast with alpha-lactalbumin, gelatin, or gelatin + TRP lowers energy intake at lunch compared with a breakfast with casein, soy, whey, or whey-GMP.</h2>
			<h3>PMID:19185957  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Veldhorst MA(1), Nieuwenhuizen AG, Hochstenbach-Waelen A, Westerterp KR, Engelen MP, Brummer RJ, Deutz NE, Westerterp-Plantenga MS.
				<br/> 
				年份：2009 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND & AIMS: Dietary <span class="trait">protein</span> plays a role in body weight regulation, partly due to its effects on satiety. The objective was to compare the effects of casein-, soy-, whey-, whey without glycomacropeptide <span class="gene">(GMP)-,</span> alpha-lactalbumin-, gelatin-, or gelatin with tryptophan (TRP)-protein breakfasts at two concentrations on subsequent satiety and energy intake (EI). METHODS: Twenty-four healthy subjects (mean+/-SEM BMI: 24.8+/-0.5 kg/m(2); age: 25+/-2 years) received a breakfast; a custard with casein, soy, whey, whey-GMP, alpha-lactalbumin, gelatin, or gelatin+TRP as <span class="trait">protein</span> source with either 10/55/35 (normal) or 25/55/20 <span class="trait">(high)</span> En% protein/carbohydrate/fat in a randomized, single-blind design. At the precedingly determined time point for lunch, 180 min, subjects were offered an ad lib lunch. Appetite profile (Visual Analogue Scales, VAS) and EI were determined. RESULTS: Both at the level of 10 and 25 En% from <span class="trait">protein,</span> EI at lunch was approximately 20% lower after an alpha-lactalbumin or gelatin (+TRP) breakfast (2.5+/-0.2 MJ) compared with after a casein, soy, or whey-GMP breakfast (3.2+/-0.3 MJ, p&lt;0.05). Appetite ratings at 180 min differed 15-25 mm (approximately 40%, p&lt;0.05) between types of <span class="trait">protein.</span> Differences in EI were a function of differences in appetite ratings (R(2)=0.4, p&lt;0.001). CONCLUSIONS: Different proteins (alpha-lactalbumin, gelatin, gelatin+TRP) that are approximately 40% more satiating than other proteins (casein, soy, whey, whey-GMP) induce a related approximately 20% reduction of subsequent energy intake.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nutritional properties of potato <span class="trait">protein</span> concentrate compared with soybean meal as the main <span class="trait">protein</span> source in feed for the double-muscled Belgian Blue bull.</h2>
			<h3>PMID:22444222</h3>
				

				
			<p>
				作者：Froidmont E(1), Wathelet B, Oger R, Romnée JM, Colinet A, Cloet D, Didelez M, Pichon JC, Boudry C, Jean G, Bartiaux-Thill N.
				<br/> 
				年份：2009 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objective of this experiment was to compare the nutritional properties of potato <span class="trait">protein</span> concentrate, a by-product of the starch industry produced entirely in Europe, with that of soybean meal (SBM), for growing cattle. The experiment was conducted on double-muscled Belgian Blue bulls, fitted with rumen, duodenal and ileal cannulas, according to a 4 × 4 Latin square design. They were fed three different iso-N and iso-net energy diets formulated according to the Dutch feed evaluation system, differing in the nature of the main <span class="trait">protein</span> source, which was either SBM ('SBM' treatment), potato <span class="trait">protein</span> concentrate (PPC, 'PPC' treatment) or an iso-N mixture of these two <span class="trait">protein</span> sources ('mixed' treatment). A fourth treatment consisted of 'PPC' supplemented by 9.5% digestible proteins supplied by duodenal perfusion of sodium caseinate (CAS, 'PPC + CAS' treatment). No significant difference was observed in the ruminal fluid pH, whereas both 'PPC' and 'PPC + CAS' had the effect of reducing the ruminal ammonia nitrogen (N-NH3) concentration. No significant difference was observed in the apparent intestinal digestibility of the dry matter (DM), organic matter (OM) or N. Outflows of non-NH3-N, microbial proteins and dietary proteins from the rumen were similar for 'PPC', 'SBM' and 'mixed', and increased with <span class="gene">CAS</span> infusion by 20%, 17% and 27%, respectively. On the basis of in vivo observations, the degradability of SBM and PPC proteins was estimated at 0.60 and 0.43, respectively, corresponding to the values quoted in the literature. The supply of digestible essential amino acids (EAA) was significantly greater with 'PPC + CAS' and did not differ among 'SBM', 'mixed' and 'PPC'. This illustrates the difficulty of altering the amino acid (AA) pattern of digestible <span class="trait">protein</span> by the nature of the <span class="trait">protein</span> of dietary origin when an animal is fed a <span class="trait">high</span> nutritional value diet. N retention was not affected by replacing SBM with PPC, but increased by 10% with <span class="gene">CAS</span> infusion. On the basis of the plasma AA pattern, the supply of digestible Met was probably limiting with 'SBM', 'mixed' and 'PPC'. The <span class="gene">CAS</span> perfusion supplemented all AA, including Met, leading to increased N retention. This improvement was limited, however, and N utilisation remained unchanged between treatments. In conclusion, despite a more favourable EAA pattern, PPC offered no advantage compared with SBM for growing bulls when diets were formulated according to the Dutch feed evaluation system.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary soy isoflavones increase insulin secretion and prevent the development of diabetic cataracts in streptozotocin-induced diabetic rats.</h2>
			<h3>PMID:19083447  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu MP(1), Wang R, Song X, Chibbar R, Wang X, Wu L, Meng QH.
				<br/> 
				年份：2008 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soy isoflavone-containing diets have been reported to be beneficial in diabetes. This present study investigated the hypoglycemic effects of isoflavones in streptozotocin (STZ)-induced diabetes. Diabetes was induced in male Sprague-Dawley rats by intraperitoneal injection of 100 mg/kg STZ. Diabetic rats were then randomly divided into 3 groups and received a special diet supplemented with casein (control), low-isoflavone soy (LIS) <span class="trait">protein,</span> and high-isoflavone soy <span class="trait">protein</span> (HIS) for 8 weeks. Compared with the control or LIS groups, those rats on the <span class="gene">HIS</span> diet had significantly increased body weight and serum insulin levels and reduced serum glucose and methylglyoxal levels. Serum glutathione levels were also increased in rats given the <span class="gene">HIS</span> diet compared with those in the control or LIS (P &lt; .01). Serum high-density lipoprotein cholesterol level was significantly higher in HIS-fed rats than that of the control or LIS rats (P &lt; .05). More importantly, the death rate and incidence of cataracts in the diabetic rats were markedly decreased in the <span class="gene">HIS</span> group. In conclusion, ingestion of high-isoflavone soy <span class="trait">protein</span> not only lowers glucose levels but also reduces the incidence of cataracts in diabetic rats. The beneficial effects of soy isoflavones are attributed to increased insulin secretion, a better glycemic control, and antioxidant protection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interactive effects of elevated CO2 and ozone on leaf thermotolerance in field-grown Glycine max.</h2>
			<h3>PMID:19017127  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mishra S(1), Heckathorn SA, Barua D, Wang D, Joshi P, Hamilton Iii EW, Frantz J.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Humans are increasing atmospheric CO2, ground-level ozone (O3), and mean and acute <span class="trait">high</span> temperatures. Laboratory studies show that elevated CO2 can increase thermotolerance of photosynthesis in <span class="gene">C3</span> plants. O3-related oxidative stress may offset benefits of elevated CO2 during heat-waves. We determined effects of elevated CO2 and O3 on leaf thermotolerance of field-grown Glycine max (soybean, C3). Photosynthetic electron transport (et) was measured in attached leaves heated in situ and detached leaves heated under ambient CO2 and O3. Heating decreased et, which O3 exacerbated. Elevated CO2 prevented O3-related decreases during heating, but only increased et under ambient O3 in the field. Heating decreased chlorophyll and carotenoids, especially under elevated CO2. Neither CO2 nor O3 affected heat-shock proteins. Heating increased catalase (except in <span class="trait">high</span> O3) and Cu/Zn-superoxide dismutase (SOD), but not Mn-SOD; CO2 and O3 decreased catalase but neither SOD. Soluble carbohydrates were unaffected by heating, but increased in elevated CO2. Thus, protection of photosynthesis during heat stress by elevated CO2 occurs in field-grown soybean under ambient O3, as in the lab, and <span class="trait">high</span> CO2 limits heat damage under elevated O3, but this protection is likely from decreased photorespiration and stomatal conductance rather than production of heat-stress adaptations.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of a novel soybean gene encoding a neutral PR-5 <span class="trait">protein</span> induced by high-salt stress.</h2>
			<h3>PMID:19010689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tachi H(1), Fukuda-Yamada K, Kojima T, Shiraiwa M, Takahara H.
				<br/> 
				年份：2009 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, we characterized a novel soybean gene encoding a neutral PR-5 <span class="trait">protein</span> and compared it to two acidic isoforms of soybean PR-5 <span class="trait">protein.</span> This gene, designated as Glycine max osmotin-like <span class="trait">protein,</span> b isoform <span class="gene">(GmOLPb,</span> accession no. AB370233), encoded a putative <span class="trait">protein</span> having the greatest similarity to chickpea PR-5b (89% identity). Unlike the two acidic PR-5, <span class="gene">GmOLPa</span> and P21, the <span class="trait">protein</span> had a C-terminal elongation responsible for possible vacuolar targeting and after maturation showed a calculated molecular mass of 21.9kDa with pI 6.0. The 3D models, predicted by the homology modeling, contained four alpha-helixes and 16 beta-strands and formed three characteristic domains. The two acidic PR-5 proteins also showed a 3D structure very similar to <span class="gene">GmOLPb,</span> although the electrostatic potential on molecular surface of each PR-5 was significantly different. In the study of the gene expression under conditions of high-salt stress, <span class="gene">GmOLPb</span> was highly induced in the leaves of the soybean, particularly in the lower part of a leaf. The expression started at 2h after initiation of the stress and was highly induced between 18-72h. Gene expression of <span class="gene">P21e</span> <span class="trait">(protein</span> homologous to P21) was transiently induced by high-salt stress, but took place earlier than the gene expressions of <span class="gene">GmOLPa</span> and <span class="gene">GmOLPb.</span> Such differential expression was observed also under investigation with methyl jasmonate and salicylic acid. These results suggested that each soybean PR-5 might play a distinctive role in the defensive system protecting the soybean plant against high-salt stress, particularly in the leaves of the soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cold-induced modulation and functional analyses of the DRE-binding transcription factor gene, <span class="gene">GmDREB3,</span> in soybean (Glycine max L.).</h2>
			<h3>PMID:18988621  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen M(1), Xu Z, Xia L, Li L, Cheng X, Dong J, Wang Q, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DREB (dehydration-responsive element-binding <span class="trait">protein)</span> transcription factors have important roles in the stress-related regulation network in plants. A DREB orthologue, <span class="gene">GmDREB3,</span> belonging to the A-5 subgroup of the DREB subfamily, was isolated from soybean using the RACE (rapid amplification of cDNA ends) method. Northern blot analysis showed that expression of <span class="gene">GmDREB3</span> in soybean seedlings was induced following cold stress treatment for 0.5 h and was not detected after 3 h. However, it was not induced by <span class="trait">drought</span> and <span class="trait">high</span> salt stresses or by abscisic acid (ABA) treatment. This response was similar to those of members in the A-1 subgroup and different from those of other members in the A-5 subgroup, suggesting that the <span class="gene">GmDREB3</span> gene was involved in an ABA-independent cold stress-responsive signal pathway. Furthermore, analysis of the <span class="gene">GmDREB3</span> promoter elucidated its cold-induced modulation. A promoter fragment containing bases -1058 to -664 was involved in response to cold stress, and its effect was detected for 1 h after treatment, but a transcriptional repressor appeared to impair this response by binding to a cis-element in the region -1403 to -1058 at 24 h after the beginning of cold stress. Moreover, the <span class="gene">GmDREB3</span> <span class="trait">protein</span> could specifically bind to the DRE element in vitro, and activated expression of downstream reporter genes in yeast cells. In addition, overexpression of <span class="gene">GmDREB3</span> enhanced tolerance to cold, <span class="trait">drought,</span> and <span class="trait">high</span> salt stresses in transgenic Arabidopsis. Physiological analyses indicated that the fresh weight and osmolality of <span class="gene">GmDREB3</span> transgenic Arabidopsis under cold stress were higher than those of wild-type controls. <span class="gene">GmDREB3</span> transgenic tobacco accumulated higher levels of free proline under <span class="trait">drought</span> stress and retained higher leaf chlorophyll levels under <span class="trait">high</span> salt stress than wild-type tobacco. In addition, constitutive expression of <span class="gene">GmDREB3</span> in transgenic Arabidopsis caused growth retardation, whereas its expression under control of the stress-inducible Rd29A promoter minimized negative effects on plant growth under normal growth conditions, indicating that a combination of the Rd29A promoter and <span class="gene">GmDREB3</span> might be useful for improving tolerance to environmental stresses in crop plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protein recovery in soymilk and various soluble fractions as a function of genotype differences, changes during heating, and homogenization.</h2>
			<h3>PMID:18942846  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Malaki Nik A(1), Tosh SM, Poysa V, Woodrow L, Corredig M.
				<br/> 
				年份：2008 Nov 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Harovinton, a variety of tofu type soybean, and 11 derived null soybean genotypes lacking specific glycinin (11S) and beta-conglycinin (7S) <span class="trait">protein</span> subunits were investigated to determine whether changes in <span class="trait">protein</span> composition affected the <span class="trait">protein</span> recovery in soymilk and its soluble fractions after various centrifugation steps. As both heating and homogenization have a marked effect on the increase in <span class="trait">protein</span> solubility, the changes occurring during these processing steps were studied for each soybean genotype. Harovinton and 11S-null genotypes showed significantly higher <span class="trait">protein</span> yields than the other genotypes evaluated. Subunits of group I (A(1), A(2)) of glycinin had a negative impact on <span class="trait">protein</span> solubility in all treatments, but this effect was the greatest in unheated soymilk samples. Samples containing a <span class="trait">high</span> beta-conglycinin to glycinin ratio showed an effect of heating on the solubility of the <span class="trait">protein,</span> as beta-conglycinin subunits aggregate with heating. The presence of the <span class="gene">alpha'</span> subunit of beta-conglycinin aids in the recovery of <span class="trait">protein</span> in the supernatant prepared from lines containing group I (A(1,4) A(2)) glycinin. The results of this study will help determine which specific <span class="trait">protein</span> composition will confer an increased stability in soymilk and soymilk-derived products.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic potential for the regulation of variability in body lipid and <span class="trait">protein</span> content of European whitefish (Coregonus lavaretus).</h2>
			<h3>PMID:18826727  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kause A(1), Quinton CD, Ruohonen K, Koskela J.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Animals may have target levels for lipid and <span class="trait">protein</span> stores which they <span class="gene">try</span> to maintain by feedback mechanisms. Thus, variation in initial body composition may be related to subsequent feed utilisation, for animals to maintain body composition in homeostasis. We assessed whether such relationships are genetically determined within a farmed population of European whitefish (Coregonus lavaretus) grown either on fishmeal or soyabean-meal diets. Soyabean meal is an increasingly-used ingredient in aquaculture feeds. Fish from thirty-five paternal families were analysed for initial body lipid and <span class="trait">protein</span> content, and for subsequent daily weight gain, daily feed intake, feed efficiency and their lipid and <span class="trait">protein</span> components. The results showed that none of the correlations of initial body lipid percentage with subsequent growth and feed utilisation were statistically significant. In contrast, low initial <span class="trait">protein</span> percentage was related to increased subsequent weight gain, <span class="trait">protein</span> gain and <span class="trait">protein</span> retention efficiency. This led to reversed ranking of families during growth for body <span class="trait">protein</span> percentage. Thus, mechanisms maintaining stable body lipid percentage across the population were weak, whereas the mechanisms stabilising body <span class="trait">protein</span> percentage were strong and successful. This explains the observations that cascades of lipid deposition occur during fish growth, leading to <span class="trait">high</span> amounts of phenotypic and genetic variation for percentage body lipid. In contrast, <span class="trait">protein</span> percentage remains phenotypically and genetically more invariable, reducing the potential for selective breeding. The soyabean-meal diet, in turn, induced only weak genotype x diet interactions, aiding in the genetic improvement of farmed fish to adapt to future feeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hypocholesterolemic effects of dietary soybean vs. casein proteins in a crossed over diets in rat.</h2>
			<h3>PMID:18817248  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reza NM(1), Fatemeh BR, Fahimeh MT, Fatemeh ZN, Morteza BR.
				<br/> 
				年份：2008 Jun 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the effects of dietary proteins on the level of serum total cholesterol (TC), triglyceride (TG) and <span class="trait">high</span> density lipoprotein (HDL), 32 male Wistar rats were randomly divided in control and 3 experimental groups (El, E2 and E3). The feeding regimes of rats were as follow: control, standard diet; E1, a cholesterol free diet containing 20% soybean <span class="trait">protein;</span> E2, a cholesterol free diet containing 20% casein and E3, a cholesterol free diet containing 10% soybean <span class="trait">protein</span> and 10% casein. The experimental period was 11 weeks but at the end of 7th week the diets of <span class="gene">E1</span> and E2 groups were crossed over for the next 4 weeks. Blood samples were collected weekly, via the ophthalmic sinus and the serum levels TC, TG and HDL were measured. In comparison with control group, the results show that at the end of 7th week TC levels in <span class="gene">E1</span> and E2 groups were significantly (p&lt;0.05) increased while HDL level unchanged and the TC value of E2 was bigger (not significant) than E1. However by crossing over the diets, the TC level was significantly (p&lt;0.05) diminished in E2 while TG value remarkably (p&lt;0.05) increased. These results indicate that soybean <span class="trait">protein</span> may insert its hypocholesterolemic effect in hypercholestrolemic condition than in normolipidemic condition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and molecular characterization of two naturally occurring Soybean mosaic virus isolates that are closely related but differ in their ability to overcome Rsv4 <span class="trait">resistance.</span></h2>
			<h3>PMID:18793685  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gagarinova AG(1), Babu M, Poysa V, Hill JH, Wang A.
				<br/> 
				年份：2008 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A naturally occurring Rsv4 resistance-breaking isolate (L-RB) and a closely related non-resistance-breaking isolate (L) of Soybean mosaic virus (SMV) were identified in soybean fields in London, Ontario, Canada. The viral genomes of L and L-RB were completely sequenced. Each isolate has a 9585-nucleotide genome with a single open reading frame encoding a <span class="gene">polyprotein</span> of approximately 350 kDa. L-RB and L have a very <span class="trait">high</span> sequence similarity (99.6%) at both the nucleotide and amino acid levels. Phylogenetic analysis showed that the two isolates belong to the G2 pathotype. Pathogenicity predictions of all virus/soybean combinations, based on the phylogenetic profile, were confirmed by pathogenicity tests using L and L-RB isolates and soybeans carrying different <span class="trait">resistance</span> genes, with an exception that L-RB infected a soybean cultivar carrying Rsv4 <span class="trait">resistance.</span> The temporal and spatial proximity of L and L-RB and their <span class="trait">high</span> sequence similarity suggest L-RB was likely derived from the SMV-L quasispecies. Recombination analysis did not reveal the evidence of genetic recombination for the emergence of L-RB. Mutations introduced by virus-encoded RNA-dependent RNA polymerase during viral genome replication and selection pressure probably contributed to the occurrence of L-RB.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary intake of soy <span class="trait">protein</span> and tofu in association with breast cancer risk based on a case-control study.</h2>
			<h3>PMID:18791919  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim MK(1), Kim JH, Nam SJ, Ryu S, Kong G.
				<br/> 
				年份：2008
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soy food and its constituents may protect against breast cancer, but the association between soy intake and decreased breast cancer risk is inconsistent. We evaluated the relationship between breast cancer risk and the dietary intake of soy <span class="trait">protein</span> as measured by total soy food and tofu intake. Histologically confirmed cases (n = 362) were matched to controls by age (within 2 yr) and menopausal status. <span class="trait">High</span> soy <span class="trait">protein</span> intake was associated with reduced breast cancer risk in analyses adjusted for potential confounders including dietary factors among premenopausal women (odds ratio [OR] = 0.39 in the highest quintile, 95% confidence interval [CI] = 0.22-0.93, P for trend = 0.03) and postmenopausal women (OR = 0.22, 95% <span class="gene">CI</span> = 0.06-0.88, P for trend = 0.16). We also found an inverse association between total tofu intake and breast cancer risk among premenopausal women (for total tofu intake, OR = 0.23 in the highest quintile, 95% <span class="gene">CI</span> = 0.11-0.48, P for trend &lt; 0.01; for at least 1 serving of tofu as the main ingredient per day, OR = 0.26, 95% <span class="gene">CI</span> = 0.13-0.55, P for trend &lt; 0.01). We concluded that increased regular soy food intake at a level equivalent to traditional Korean consumption levels may be associated with a reduced risk of breast cancer, and this effect is more pronounced in premenopausal women.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy <span class="trait">protein</span> isolate reduces hepatosteatosis in yellow Avy/a mice without altering coat color phenotype.</h2>
			<h3>PMID:18791133  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Badger TM(1), Ronis MJ, Wolff G, Stanley S, Ferguson M, Shankar K, Simpson P, Jo CH.
				<br/> 
				年份：2008 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Agouti (A(vy)/a) mice fed an AIN-93G diet containing the soy isoflavone genistein (GEN) prior to and during pregnancy were reported to shift coat color and body composition phenotypes from obese-yellow towards lean pseudoagouti, suggesting epigenetic programming. Human consumption of purified GEN is rare and soy <span class="trait">protein</span> is the primary source of GEN. Virgin a/a female and A(vy)/a male mice were fed AIN-93G diets made with casein (CAS) or soy <span class="trait">protein</span> isolate (SPI) (the same approximate GEN levels as in the above mentioned study) for 2 wks prior to mating. A(vy)/a offspring were weaned to the same diets and studied at age 75 d. Coat color distribution did not differ among diets, but SPI-fed, obese A(vy)/a offspring had lower hepatosteatosis (P &lt; 0.05) and increased (P &lt; 0.05) expression of CYP4a 14, a PPARalpha-regulated gene compared to <span class="gene">CAS</span> controls. Similarly, weanling male Sprague-Dawley (SD) rats fed SPI had elevated hepatic Acyl Co-A Oxidase (ACO) mRNA levels and increased in vitro binding of PPARalpha to the PPRE promoter response element. In another hepatosteatosis model, adult SD rats fed a <span class="trait">high</span> fat/cholesterol diet, SPI reduced (P &lt; 0.05) steatosis. Thus, 1) consumption of diets made with SPI partially protected against hepatosteatosis in yellow mice and in SD rats, and this may involve induction of PPARalpha-regulated genes; and 2) the lifetime (in utero, neonatal and adult) exposure to dietary soy <span class="trait">protein</span> did not result in a shift in coat color phenotype of A(vy)/a mice. These findings, when compared with those of previously published studies of A(vy)/a mice, lead us to conclude that: 1) the effects of purified GEN differ from those of SPI when GEN equivalents are closely matched; 2) SPI does not epigenetically regulate the agouti locus to shift the coat color phenotype in the same fashion as GEN alone; and 3) SPI may be beneficial in management of non-alcoholic fatty liver disease.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multilocus sequence analysis for assessment of the biogeography and evolutionary genetics of four Bradyrhizobium species that nodulate soybeans on the asiatic continent.</h2>
			<h3>PMID:18791003  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vinuesa P(1), Rojas-Jiménez K, Contreras-Moreira B, Mahna SK, Prasad BN, Moe H, Selvaraju SB, Thierfelder H, Werner D.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A highly supported maximum-likelihood species phylogeny for the genus Bradyrhizobium was inferred from a supermatrix obtained from the concatenation of partial atpD, recA, glnII, and <span class="gene">rpoB</span> sequences corresponding to 33 reference strains and 76 bradyrhizobia isolated from the nodules of Glycine max (soybean) trap plants inoculated with soil samples from Myanmar, India, Nepal, and Vietnam. The power of the multigene approach using multiple strains per species was evaluated in terms of overall tree resolution and phylogenetic congruence, representing a practical and portable option for bacterial molecular systematics. Potential pitfalls of the approach are highlighted. Seventy-five of the isolates could be classified as B. japonicum type Ia (USDA110/USDA122-like), B. liaoningense, B. yuanmingense, or B. elkanii, whereas one represented a novel Bradyrhizobium lineage. Most Nepalese B. japonicum Ia isolates belong to a highly epidemic clone closely related to strain USDA110. Significant phylogenetic evidence against the monophyly of the of B. japonicum I and Ia lineages was found. Analysis of their DNA polymorphisms revealed <span class="trait">high</span> population distances, significant genetic differentiation, and contrasting population genetic structures, suggesting that the strains in the Ia lineage are misclassified as B. japonicum. The DNA polymorphism patterns of all species conformed to the expectations of the neutral mutation and population equilibrium models and, excluding the B. japonicum Ia lineage, were consistent with intermediate recombination levels. All species displayed epidemic clones and had broad geographic and environmental distribution ranges, as revealed by mapping climate types and geographic origins of the isolates on the species tree.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation of the oxidation state of verdoheme in the heme oxygenase reaction.</h2>
			<h3>PMID:18778686  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gohya T(1), Sato M, Zhang X, Migita CT.
				<br/> 
				年份：2008 Nov 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heme oxygenase (HO) converts hemin to biliverdin, CO, and iron applying molecular oxygen and electrons. During successive HO reactions, two intermediates, alpha-hydroxyhemin and verdoheme, have been generated. Here, oxidation state of the verdoheme-HO complexes is controversial. To clarify this, the heme conversion by soybean and rat HO isoform-1 <span class="gene">(GmHO-1</span> and rHO-1, respectively) was compared both under physiological conditions, with oxygen and NADPH coupled with ferredoxin reductase/ferredoxin for <span class="gene">GmHO-1</span> or with cytochrome <span class="gene">P450</span> reductase for rHO-1, and under a non-physiological condition with hydrogen peroxide. EPR measurements on the hemin-GmHO-1 reaction with oxygen detected a low-spin ferric intermediate, which was undetectable in the rHO-1 reaction, suggesting the verdoheme in the six-coordinate ferric state in <span class="gene">GmHO-1.</span> Optical absorption measurements on this reaction indicated that the heme degradation was extremely retarded at verdoheme though this reaction was not inhibited under high-CO concentrations, unlike the rHO-1 reaction. On the contrary, the Gm and rHO-1 reactions with hydrogen peroxide both provided ferric low-spin intermediates though their yields were different. The optical absorption spectra suggested that the ferric and ferrous verdoheme coexisted in reaction mixtures and were slowly converted to the ferric biliverdin complex. Consequently, in the physiological oxygen reactions, the verdoheme is found to be stabilized in the ferric state in <span class="gene">GmHO-1</span> probably guided by <span class="trait">protein</span> distal residues and in the ferrous state in rHO-1, whereas in the hydrogen peroxide reactions, hydrogen peroxide or hydroxide coordination stabilizes the ferric state of verdoheme in both HOs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of soybean <span class="gene">GmbZIP132</span> under abscisic acid and salt stresses.</h2>
			<h3>PMID:18713445  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liao Y(1), Zhang JS, Chen SY, Zhang WK.
				<br/> 
				年份：2008 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant basic-leucine zipper (bZIP) transcription factors play important roles in many biological processes. In the present study, a bZIP gene, <span class="gene">GmbZIP132,</span> was cloned from soybean and its biological function under abiotic stresses was studied. The transcription of <span class="gene">GmbZIP132</span> was induced by <span class="trait">drought</span> and <span class="trait">high</span> salt treatments. Among all of the organs analyzed, its expression was the highest in cotytledon and stems. <span class="gene">GmbZIP132</span> could weakly bind to the GCN4-like motif (GLM) (5'-GTGAGTCAT-3') in yeast one-hybrid assay. Compared with wild-type (WT) Arabidopsis plants, transgenic plants overexpressing <span class="gene">GmbZIP132</span> showed reduced abscisic acid sensitivity and increased water loss rate. At the stage of germination, transgenic plants were more tolerant to salt treatment than wild-type plants. The expression of some abiotic stress-related genes, such as rd29B, DREB2A, and P5CS, were upregulated in the transgenic plants. These results indicated that <span class="gene">GmbZIP132</span> was an abiotic stress-related gene, and its overexpression could increase the salt tolerance of transgenic Arabidopsis plants during germination, yet no significant difference of tolerance to abiotic stresses was found between transgenic and wild type plants at the seedling stage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hydrogen peroxide biosensor based on direct electrochemistry of soybean peroxidase immobilized on single-walled carbon nanohorn modified electrode.</h2>
			<h3>PMID:18703329  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shi L(1), Liu X, Niu W, Li H, Han S, Chen J, Xu G.
				<br/> 
				年份：2009 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Single-walled carbon nanohorns (SWCNHs) were used as a novel and biocompatible matrix for fabricating biosensing devices. The direct immobilization of acid-stable and thermostable soybean peroxidase (SBP) on SWCNH modified electrode surface can realize the direct electrochemistry of enzyme. Cyclic voltammogram of the adsorbed <span class="gene">SBP</span> displays a pair of redox peaks with a formal potential of -0.24 V in pH 5 phosphate buffer solution. The formal potential has a linear relationship with pH from 3 to 9 with a slope of -48.7 mV/pH, close to the value of -55.7 mV/pH expected at 18 degrees C for the reversible transfer of one proton and one electron. Bioactivity of <span class="gene">SBP</span> remains good in SWCNH microenvironment, along with effective catalysis of the reduction of H(2)O(2). In the absence of a mediator, this H(2)O(2) biosensor exhibited a <span class="trait">high</span> sensitivity (16.625 microAL/mmol), a linear range from 0.02 to 1.2 mmolL(-1), and a detection limit of 5.0 x 10(-7) mmolL(-1), as well as acceptable preparation reproducibility and excellent stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GH3::GUS reflects cell-specific developmental patterns and stress-induced changes in wood anatomy in the poplar stem.</h2>
			<h3>PMID:18595842  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Teichmann T(1), Bolu-Arianto WH, Olbrich A, Langenfeld-Heyser R, Göbel C, Grzeganek P, Feussner I, Hänsch R, Polle A.
				<br/> 
				年份：2008 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GH3</span> genes related to the auxin-inducible Glycine max (L.) Merr. <span class="gene">GmGH3</span> gene encode enzymes that conjugate amino acids to auxin. To investigate the role of <span class="gene">GH3</span> enzymes in stress responses and normal wood development, Populus x canescens (Ait.) was transformed with the promoter-reporter construct GH3::GUS containing a <span class="gene">GH3</span> promoter and the 5' UTR from soybean. beta-Glucuronidase (GUS) activity was present in the vascular tissues of leaves and in developing lateral roots and was inducible in silent tissues by external auxin application. A decrease in GUS activity from the stem apex to the bottom corresponded to decreases in auxin concentrations in these tissues. <span class="trait">High</span> auxin concentration and <span class="trait">high</span> GH3::GUS activity were present in the pith tissue, which may provide storage for auxin compounds. <span class="gene">GH3</span> reporter was active in ray cells, paratracheal parenchyma cells, maturing vessels and in cells surrounding maturing phloem fibers but not in the cambium and immature phloem, despite <span class="trait">high</span> auxin concentrations in the latter tissues. However, the <span class="gene">GH3</span> promoter in these tissues became active when the plants were exposed to abiotic stresses, like bending or salinity, causing changes in wood anatomy. We suggest that adjustment of the internal auxin balance in wood in response to environmental cues involves <span class="gene">GH3</span> auxin conjugate synthases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increased insulin sensitivity and changes in the expression profile of key insulin regulatory genes and beta cell transcription factors in diabetic KKAy-mice after feeding with a soy bean <span class="trait">protein</span> rich diet <span class="trait">high</span> in isoflavone content.</h2>
			<h3>PMID:18522411  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nordentoft I(1), Jeppesen PB, Hong J, Abudula R, Hermansen K.
				<br/> 
				年份：2008 Jun 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> content isoflavone soy <span class="trait">protein</span> (SBP) (Abalon) has been found in animal studies to possess beneficial effects on a number of the characteristic features of the insulin <span class="trait">resistance</span> syndrome. The aim of this study was to investigate whether <span class="gene">SBP</span> exerts beneficial effects on metabolism in the diabetic KKAy-mouse. Furthermore, we investigated the long-term in vivo effect of <span class="gene">SBP</span> on the expression profile in islets of key insulin regulatory genes. Twenty KKAy-mice, aged 5 weeks, were divided into 2 groups and treated for 9 weeks with either (A) standard chow diet (control) or (B) chow + 50% SBP. Twenty normal C57BL-mice fed with standard chow diet served as nondiabetic controls (C). Blood samples were collected and analyzed before and after intervention. Gene expression was determined in islets by quantitative real-time RT-PCR and Affymetrix microarray. It was demonstrated that long-term treatment with <span class="gene">SBP</span> improves glucose homeostasis, increases insulin sensitivity, and lowers plasma triglycerides in diabetic KKAy-mice. <span class="gene">SBP</span> reduces fasting plasma glucose, insulin, triglycerides, and total cholesterol. Furthermore, <span class="gene">SBP</span> markedly changes the gene expression profile of key insulin regulatory genes GLUT2, GLUT3, Ins1, Ins2, IGF1, Beta2/Neurod1, cholecystokinin, and LDLr, and proliferative genes in islets isolated from KKAy-mice. After 9 weeks of treatment with SBP, plasma glucose and insulin homeostasis was normalized compared to start levels. The results indicate that <span class="gene">SBP</span> improves glucose and insulin sensitivity and up-regulates the expression of key insulin regulatory genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nodulation gene regulation and quorum sensing control density-dependent suppression and restriction of nodulation in the Bradyrhizobium japonicum-soybean symbiosis.</h2>
			<h3>PMID:18441104  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jitacksorn S(1), Sadowsky MJ.
				<br/> 
				年份：2008 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nodulation of Glycine max cv. Lambert and the nodulation-restricting plant introduction (PI) genotype PI 417566 by wild-type Bradyrhizobium japonicum USDA110 is regulated in a population-density-dependent manner. Nodulation on both plant genotypes was suppressed (inhibited) when plants received a high-density inoculum (10(9) cells/ml) of strain USDA110 grown in complex medium, and more nodules were produced on plants receiving a low-cell-density inoculum (10(5) cells/ml). Since cell-free supernatants from strain USDA110 grown to <span class="trait">high</span> cell density in complex medium decreased the expression of an nodY-lacZ fusion, this phenomenon was attributed to bradyoxetin-induced repression of nod gene expression. Inoculation of either the permissive soybean genotype (cv. Lambert) or PI 417566 with 10(9) cells/ml of the nodD2, nolA, nodW, and nwsB mutants of USDA110 enhanced nodulation (up to 24%) relative to that seen with inoculations done with 10(5) cells/ml of the mutants or the wild-type strain, indicating that these genes are involved in population-density-dependent nodulation of soybeans. In contrast, the number of nodules produced by an <span class="gene">nodD1</span> mutant on either soybean genotype was less than those seen with the wild-type strain inoculated at a low inoculum density. The <span class="gene">nodD2</span> mutant outcompeted B. japonicum strain USDA123 for nodulation of G. max cv. Lambert at a <span class="trait">high</span> or low inoculum density, and the results of root-tip-marking and time-to-nodulate studies indicated that the <span class="gene">nolA</span> and <span class="gene">nodD2</span> mutants nodulated this soybean genotype faster than wild-type USDA110. Taken together, the results from these studies indicate that the <span class="gene">nodD2</span> mutant of B. japonicum may be useful to enhance soybean nodulation at <span class="trait">high</span> inoculum densities and that NodD2 is a key repressor influencing host-controlled restriction of nodulation, density-dependent suppression of nodulation, perception of bradyoxetin, and competitiveness in the soybean-B. japonicum symbiosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of class 1 DXS gene encoding 1-deoxy-D-xylulose-5-phosphate synthase, the first committed enzyme of the MEP pathway from soybean.</h2>
			<h3>PMID:18437529  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang M(1), Li K, Zhang C, Gai J, Yu D.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						1-Deoxy-D-xylulose-5-phosphate synthase (DXS) catalyses the first committed step of the 2C-methyl-D-erythritol-4-phosphate (MEP) pathway, which is an alternative isoprenoids biosynthetic route that has been recently discovered. In this work, a DXS1-like cDNA <span class="gene">(GmDXS1)</span> was isolated from soybean. The full-length cDNA of <span class="gene">GmDXS1</span> encoded 708 amino acid residues with a predicted molecular mass of 76.4 KD. Sequence alignment showed that <span class="gene">GmDXS1</span> had <span class="trait">high</span> homology to known DXS proteins from other plant species and contained the conserved N-terminal plastid transit peptide, the N-terminal thiamine binding domain and pyridine binding DRAG domain. Phylogenetic analysis indicated that <span class="gene">GmDXS1</span> belonged to the plant <span class="gene">DXS1</span> cluster. Southern blot analysis indicated that a single copy of <span class="gene">GmDXS1</span> gene existed in soybean genome. Tissue expression analysis revealed that <span class="gene">GmDXS1</span> expressed in all photosynthetic tissues except pod walls and roots. Green fluorescence analysis with the fusion <span class="trait">protein</span> 35S:GmDXS1:GFP suggested that <span class="gene">GmDXS1</span> was localized in plastid. The relatively higher photosynthetic pigment content in transgenic tobacco leaves compared to the control implied that <span class="gene">GmDXS1</span> catalyzed the first potential regulatory step in photosynthetic pigment biosynthesis via the MEP pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Down-regulation of flavonoid 3'-hydroxylase gene expression by virus-induced gene silencing in soybean reveals the presence of a threshold mRNA level associated with pigmentation in pubescence.</h2>
			<h3>PMID:18423787  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nagamatsu A(1), Masuta C, Matsuura H, Kitamura K, Abe J, Kanazawa A.
				<br/> 
				年份：2009 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Changes in flavonoid content are often manifested as altered pigmentation in plant tissues. Two loci have been identified as controlling pigmentation in soybean pubescence. Of these, the T locus appears to encode flavonoid 3'-hydroxylase (F3'H) <span class="trait">protein:</span> the T and t alleles are associated with tawny and gray colors, respectively, in pubescence. We previously down-regulated <span class="gene">F3'H</span> gene expression by virus-induced gene silencing (VIGS) in soybean. Despite this successful VIGS, the tawny pubescence pigmentation proved to be unchanged in greenhouse-grown plants. We hypothesized that the reduced mRNA level of the <span class="gene">F3'H</span> gene resulting from VIGS remained <span class="trait">high</span> enough to induce pigmentation. To verify this hypothesis, in the present study, we performed <span class="gene">F3'H</span> VIGS on plants grown under controlled conditions, in which the steady-state mRNA level of the <span class="gene">F3'H</span> gene was reduced to approximately 5% of that of greenhouse-grown plants. This VIGS treatment resulted in the loss of tawny pigmentation in pubescence, suggesting that the <span class="gene">sf3'h1</span> gene is involved in the control of pigmentation in pubescence. We detected a marked decrease in target mRNA, an accumulation of short interfering RNAs (siRNAs), and a decrease in quercetin content relative to kaempferol in leaf tissues, indicating that sequence-specific mRNA degradation of the <span class="gene">F3'H</span> gene was induced. These results suggest that leaf tissues have a threshold mRNA level of the <span class="gene">F3'H</span> gene, which is associated with the occurrence of tawny pigmentation in pubescence. The estimated threshold mRNA level for pigmentation in pubescence was approximately 3% of the steady-state mRNA level of the <span class="gene">F3'H</span> gene in greenhouse-grown plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The solution structures of two soybean calmodulin isoforms provide a structural basis for their selective target activation properties.</h2>
			<h3>PMID:18347016  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ishida H(1), Huang H, Yamniuk AP, Takaya Y, Vogel HJ.
				<br/> 
				年份：2008 May 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The intracellular calcium ion is one of the most important secondary messengers in eukaryotic cells. Ca(2+) signals are translated into physiological responses by EF-hand calcium-binding proteins such as calmodulin (CaM). Multiple <span class="gene">CaM</span> isoforms occur in plant cells, whereas only a single <span class="gene">CaM</span> <span class="trait">protein</span> is found in animals. Soybean <span class="gene">CaM</span> isoform 1 (sCaM1) shares 90% amino acid sequence identity with animal <span class="gene">CaM</span> (aCaM), whereas sCaM4 is only 78% identical. These two sCaM isoforms have distinct target-enzyme activation properties and physiological functions. sCaM4 is highly expressed during the self-defense reaction of the plant and activates the enzyme nitric-oxide synthase (NOS), whereas sCaM1 is incapable of activating NOS. The mechanism of selective target activation by plant <span class="gene">CaM</span> isoforms is poorly understood. We have determined <span class="trait">high</span> resolution NMR solution structures of Ca(2+)-sCaM1 and -sCaM4. These were compared with previously determined Ca(2+)-aCaM structures. For the N-lobe of the <span class="trait">protein,</span> the solution structures of Ca(2+)-sCaM1, -sCaM4, and -aCaM all closely resemble each other. However, despite the <span class="trait">high</span> sequence identity with aCaM, the C-lobe of Ca(2+)-sCaM1 has a more open conformation and consequently a larger hydrophobic target-protein binding pocket than Ca(2+)-aCaM or -sCaM4, the presence of which was further confirmed through biophysical measurements. The single Val-144 --&gt; Met substitution in the C-lobe of Ca(2+)-sCaM1, which restores its ability to activate NOS, alters the structure of the C-lobe to a more closed conformation resembling Ca(2+)-aCaM and -sCaM4. The relationships between the structural differences in the two Ca(2+)-sCaM isoforms and their selective target activation properties are discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Synthesis and biological evaluation of novel angular fused Pyrrolocoumarins.</h2>
			<h3>PMID:18341252  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kontogiorgis C(1), Litinas KE, Makri A, Nicolaides DN, Vronteli A, Hadjipavlou-Litina DJ, Pontiki E, Siohou A.
				<br/> 
				年份：2008 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Angular pyrrolocoumarins were synthesized from the reaction of 4-hydroxyindole or 5-hydroxyindole with DMAD and PPh(3) and were tested for anti-inflammatory and antioxidant activity. These compounds significantly inhibited the carrageenin-induced paw edema (60.5%-73.4%) and have important scavenging activity. Although their interaction with the free stable radical DPPH is not <span class="trait">high,</span> compound 9 is the most potent (73.4%) in the in vivo experiment. Compound 7 seems to be a potent <span class="gene">LOX</span> inhibitor. An attempt was made to correlate the biological results with their structural characteristics and physicochemical parameters.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a seed-specific isoform of microsomal omega-6 fatty acid desaturase gene (FAD2-1B) from soybean.</h2>
			<h3>PMID:18300159  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li L(1), Wang X, Gai J, Yu D.
				<br/> 
				年份：2008 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In plants, the endoplasmic reticulum (ER)-associated oleate desaturase (FAD2) is the key enzyme responsible for the production of linoleic acid in non-photosynthetic tissues. In this study, we report the characterization of a seed-specific isoform of microsomal omega-6 fatty acid desaturase gene (FAD2-1B) sharing <span class="trait">high</span> sequence similarity with <span class="gene">FAD2-1</span> from soybean. Several potential promoter elements including seed-specific motifs are found in the 5'-flanking region of <span class="gene">FAD2-1B</span> gene. The ORF of <span class="gene">FAD2-1B</span> is 1161 bp long and encodes a <span class="trait">protein</span> of 387 amino acids. This deduced <span class="trait">protein</span> holds three histidine boxes and four putative membrane-spanning helices, and possesses a signal for endoplasmic reticulum retention at C-terminal. Yeast cells transformed with the plasmid construct containing soybean <span class="gene">FAD2-1B</span> accumulate an appreciable amount of linoleic acid (18:2), normally not present in wild-type yeast cells, indicating that the cloned gene encodes a functional <span class="gene">FAD2</span> enzyme. Both semi-quantitative RT-PCR and in silico analysis show that <span class="gene">FAD2-1B</span> gene is specifically expressed in developing seeds of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Covalent attachment of functional <span class="trait">protein</span> to polymer surfaces: a novel one-step dry process.</h2>
			<h3>PMID:18285286  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：MacDonald C(1), Morrow R, Weiss AS, Bilek MM.
				<br/> 
				年份：2008 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The attachment of bioactive <span class="trait">protein</span> to surfaces underpins the development of biosensors and diagnostic microarrays. We present a surface treatment using plasma immersion ion implantation (PIII) to create stable covalent binding sites for the attachment of functional soya-bean peroxidase (SBP). Fourier transform infrared spectra of the surfaces show that <span class="trait">protein</span> is retained on the surface after boiling in sodium dodecyl sulphate and sodium hydroxide, which is indicative of covalent attachment. The activity of <span class="gene">SBP</span> on the treated surfaces remains <span class="trait">high</span> in comparison with <span class="gene">SBP</span> attached to control surfaces over the course of 11 days. Surface plasmon resonance was used to show that the surface coverage of the attached <span class="trait">protein</span> is close to a monolayer. We describe the potential of the PIII treatment method to be used as a one-step dry process to create surfaces for large-scale <span class="trait">protein</span> micro- or nanopatterning.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peanut gene expression profiling in developing seeds at different reproduction stages during Aspergillus parasiticus infection.</h2>
			<h3>PMID:18248674  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo B(1), Chen X, Dang P, Scully BT, Liang X, Holbrook CC, Yu J, Culbreath AK.
				<br/> 
				年份：2008 Feb 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Peanut (Arachis hypogaea L.) is an important crop economically and nutritionally, and is one of the most susceptible host crops to colonization of Aspergillus parasiticus and subsequent aflatoxin contamination. Knowledge from molecular genetic studies could help to devise strategies in alleviating this problem; however, few peanut DNA sequences are available in the public database. In order to understand the molecular basis of host <span class="trait">resistance</span> to aflatoxin contamination, a large-scale project was conducted to generate expressed sequence tags (ESTs) from developing seeds to identify resistance-related genes involved in defense response against Aspergillus infection and subsequent aflatoxin contamination. RESULTS: We constructed six different cDNA libraries derived from developing peanut seeds at three reproduction stages (R5, R6 and R7) from a resistant and a susceptible cultivated peanut genotypes, 'Tifrunner' (susceptible to Aspergillus infection with higher aflatoxin contamination and resistant to TSWV) and 'GT-C20' (resistant to Aspergillus with reduced aflatoxin contamination and susceptible to TSWV). The developing peanut seed tissues were challenged by A. parasiticus and <span class="trait">drought</span> stress in the field. A total of 24,192 randomly selected cDNA clones from six libraries were sequenced. After removing vector sequences and quality trimming, 21,777 high-quality EST sequences were generated. Sequence clustering and assembling resulted in 8,689 unique EST sequences with 1,741 tentative consensus EST sequences (TCs) and 6,948 singleton ESTs. Functional classification was performed according to <span class="gene">MIPS</span> functional catalogue criteria. The unique EST sequences were divided into twenty-two categories. A similarity search against the non-redundant <span class="trait">protein</span> database available from NCBI indicated that 84.78% of total ESTs showed significant similarity to known proteins, of which 165 genes had been previously reported in peanuts. There were differences in overall expression patterns in different libraries and genotypes. A number of sequences were expressed throughout all of the libraries, representing constitutive expressed sequences. In order to identify resistance-related genes with significantly differential expression, a statistical analysis to estimate the relative abundance (R) was used to compare the relative abundance of each gene transcripts in each cDNA library. Thirty six and forty seven unique EST sequences with threshold of R &gt; 4 from libraries of 'GT-C20' and 'Tifrunner', respectively, were selected for examination of temporal gene expression patterns according to EST frequencies. Nine and eight resistance-related genes with significant up-regulation were obtained in 'GT-C20' and 'Tifrunner' libraries, respectively. Among them, three genes were common in both genotypes. Furthermore, a comparison of our EST sequences with other plant sequences in the TIGR Gene Indices libraries showed that the percentage of peanut EST matched to Arabidopsis thaliana, maize (Zea mays), Medicago truncatula, rapeseed (Brassica napus), rice (Oryza sativa), soybean (Glycine max) and wheat (Triticum aestivum) ESTs ranged from 33.84% to 79.46% with the sequence identity &gt;/= 80%. These results revealed that peanut ESTs are more closely related to legume species than to cereal crops, and more homologous to dicot than to monocot plant species. CONCLUSION: The developed ESTs can be used to discover novel sequences or genes, to identify resistance-related genes and to detect the differences among alleles or markers between these resistant and susceptible peanut genotypes. Additionally, this large collection of cultivated peanut EST sequences will make it possible to construct microarrays for gene expression studies and for further characterization of host <span class="trait">resistance</span> mechanisms. It will be a valuable genomic resource for the peanut community. The 21,777 ESTs have been deposited to the NCBI GenBank database with accession numbers ES702769 to ES724546.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning, expression profiling and trans-activation property studies of a DREB2-like gene from chrysanthemum (Dendranthema vestitum).</h2>
			<h3>PMID:18224275  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu L(1), Zhu K, Yang Y, Wu J, Chen F, Yu D.
				<br/> 
				年份：2008 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Dehydration responsive element binding (DREB) proteins are important transcription factors in plant stress response and signal transduction. In this study, a DREB homolog gene, DvDREB2A, was isolated from chrysanthemum (Dendranthema vestitum) by reverse transcriptase-polymerase chain reaction (RT-PCR) and rapid amplification of cDNA ends (RACE) methods. It contained an open reading frame (ORF) of 1,471 bp encoding 366 amino acid residues and was classified as a <span class="gene">DREB2</span> subfamily member based on multiple sequence alignment. The predicted <span class="trait">protein</span> sequence contained a typical AP2/EREBP DNA-binding domain near the N-terminal region. In yeast one-hybrid analysis DvDREB2A <span class="trait">protein</span> was specifically bound to DRE elements (core sequence, A/GCCGAC) and activated the expression of the reporter <span class="gene">HIS3</span> and LacZ. Transient expression experiment suggested that DvDREB2A <span class="trait">protein</span> was localized to the nucleus of onion epidermis cells. Quantitative real-time PCR (QRT-PCR) experiments showed that expression level of DvDREB2A was significantly affected by heat, low temperature, <span class="trait">drought,</span> abscisic acid (ABA) and <span class="trait">high</span> salinity treatments. These results indicated that the DvDREB2A gene is a new member of the DREB transcription factors, which may play an important role in providing tolerance to environmental stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-level expression of the native barley alpha-amylase/subtilisin inhibitor in Pichia pastoris.</h2>
			<h3>PMID:18207271  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Micheelsen PO(1), Ostergaard PR, Lange L, Skjøt M.
				<br/> 
				年份：2008 Feb 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An expression system for high-level expression of the native Hordeum vulgare alpha-amylase/subtilisin inhibitor (BASI) has been developed in Pichia pastoris, using the methanol inducible alcohol oxidase 1 (AOX1) promoter. To optimize expression, two codon-optimized coding regions have been designed and expressed alongside the wild-type coding region. To ensure secretion of the native mature <span class="trait">protein,</span> a truncated version of the alpha mating factor secretion signal from Saccharomyces cerevisiae was used. In order to be able to compare expression levels from different clones, single insertion transformants generated by gene replacement of the <span class="gene">AOX1</span> gene was selected by PCR screening. Following methanol induction, expression levels reached 125 mgL(-1) from the wild-type coding region while expression from the two codon-optimized variants reached 65 and 125 mgL(-1), respectively. The <span class="trait">protein</span> was purified and characterized by Edman degradation, liquid chromatography mass spectrometry and insoluble blue starch assay, and was shown to possess the same characteristics as wild-type <span class="trait">protein</span> purified from barley grains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of a pepper <span class="gene">C2</span> domain-containing <span class="gene">SRC2</span> <span class="trait">protein</span> implicated in <span class="trait">resistance</span> against host and non-host pathogens and abiotic stresses.</h2>
			<h3>PMID:18204857  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim YC(1), Kim SY, Choi D, Ryu CM, Park JM.
				<br/> 
				年份：2008 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants guard themselves against pathogen attack using multi-layered defense mechanism. Calcium represents an important secondary messenger during such defense responses. Upon examination of a pepper cDNA library, we observed that the gene CaSRC2-1 (Capsicum annum SRC2-1) was upregulated significantly in response to infection with the type II non-host pathogen Xanthomonas axonopodis pv. glycines 8 ra, which elicits a hypersensitive response. CaSRC2-1 encodes a <span class="trait">protein</span> that contains a <span class="gene">C2</span> domain and it exhibits a <span class="trait">high</span> degree of homology to the <span class="trait">protein</span> Soybean genes regulated by cold 2 (SRC2). However, little is known about how <span class="gene">SRC2</span> expression is elicited by biotic stresses such as pathogen challenge. Further sequence analysis indicated that the CaSRC2-1 <span class="gene">C2</span> domain is unique and contain certain amino acids that are conserved within the <span class="gene">C2</span> domains of other plants and animals. CaSRC2-1 transcription was up-regulated under both biotic and abiotic stress conditions, including bacterial and viral pathogen infection, CaCl(2) and cold treatment, but unaffected by treatment with plant defense-related chemicals such as salicylic acid, methyl jasmonic acid, ethephone, and abscisic acid. Intriguingly, under steady state conditions, CaSRC2-1 was expressed only in the root system. A CaSRC2-1-GFP fusion <span class="trait">protein</span> was used to determine localization to the plasma membrane. A fusion <span class="trait">protein</span> lacking the <span class="gene">C2</span> domain failed to target the membrane but remained in the cytoplasm, indicating that the <span class="gene">C2</span> domain plays a critical role in localization. Thus, CaSRC2-1 encodes a novel <span class="gene">C2</span> domain-containing <span class="trait">protein</span> that targets the plasma membrane and plays a critical role in the abiotic stress and defense responses of pepper plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Modulation of methylglyoxal and glutathione by soybean isoflavones in mild streptozotocin-induced diabetic rats.</h2>
			<h3>PMID:18060748  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu MP(1), Wang R, Song X, Wang X, Wu L, Meng QH.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIMS: Evidence shows that methylglyoxal (MG), a very reactive metabolite of glucose, plays a critical role in the pathogenesis of diabetes and diabetic complications. Although soy isoflavones have beneficial effects in diabetes, the role of soy isoflavones in regulating MG levels is unknown. The present study investigates the effects of soy <span class="trait">protein</span> isoflavones on MG and reduced glutathione (GSH). METHODS AND RESULTS: Mild diabetes was induced in male Sprague-Dawley rats by intraperitoneal injection of 35 mg/kg streptozotocin (STZ). The diabetic rats were then randomly divided into three groups and received a special diet supplemented with casein (control), low-isoflavone soy <span class="trait">protein</span> (LIS), or high-isoflavone soy <span class="trait">protein</span> (HIS) for eight weeks, respectively. Compared to the control or LIS group, <span class="gene">HIS</span> diet significantly increased serum insulin levels (p&lt;0.01 or 0.05) and reduced serum glucose and MG levels (p&lt;0.05). Serum GSH levels were increased in HIS-fed rats compared with the control or LIS group (p&lt;0.01). Serum total cholesterol and homocysteine levels were significantly lower in <span class="gene">HIS</span> and LIS rats than those of the control rats. CONCLUSIONS: Both LIS and <span class="gene">HIS</span> diets can lower serum lipid and homocysteine levels in this mild diabetic model. <span class="gene">HIS</span> diet enhances insulin secretion and reduces glucose level. Moreover, the <span class="gene">HIS</span> diet has potential in reducing MG and increasing GSH levels. In addition to its hypoglycemic effect, the antioxidant protection may provide beneficial effects in preventing the development of diabetic complications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="trait">protein</span> hydrolysate improves plasma and liver lipid profiles in rats fed high-cholesterol diet.</h2>
			<h3>PMID:17914129  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang SC(1), Liu SM, Yang HY, Lin YH, Chen JR.
				<br/> 
				年份：2007 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: This investigation attempted to clarify the hypolipidemic effects of non-dialyzed soybean <span class="trait">protein</span> hydrolysate (NSPH), which is hydrolyzed by pepsin from soybean acid-precipitated <span class="trait">protein</span> (APP), in rats fed a cholesterol-rich diet. METHODS: Forty Sprague-Dawley rats were divided into four groups as the control group (19.7% casein), the <span class="gene">APP</span> group (14.7% casein + 5% APP), the NSPH group (14.7% casein + 5% NSPH), and the ISO group (19.7% casein + 0.0013% soy isoflavone). RESULTS: After 12-week experimental period, the <span class="gene">APP</span> and NSPH groups had a significant lower plasma total cholesterol, triglycerides, and LDL-cholesterol concentrations compared with the control group. Additionally, the atherosclerosis index in <span class="gene">APP</span> and NSPH group had also markedly decreased. Liver cholesterol and triglyceride contents of the <span class="gene">APP</span> and NSPH group were significantly lower than those of the control group. There were no different in plasma LDL-C, liver cholesterol and triglycerides between the ISO group and control group. Fecal excretion of neutral steroids and nitrogen compounds was significantly higher in the <span class="gene">APP</span> and NSPH groups than that in the control group. An in vitro study also showed that NSPH, compared with casein, obviously decreased cholesterol micellar solubility. CONCLUSION: These results suggested that NSPH may decrease lipid accumulation in the liver and have a hypolipidemic effect by enhancing excretion and inhibiting absorption of lipids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a multigene family encoding putative beta-glucan-binding proteins in Medicago truncatula.</h2>
			<h3>PMID:17728012  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Leclercq J(1), Fliegmann J, Tellström V, Niebel A, Cullimore JV, Niehaus K, Küster H, Ebel J, Mithöfer A.
				<br/> 
				年份：2008 May 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Branched 1,6-1,3-beta-glucans from Phytophthora sojae cell walls represent pathogen-associated molecular patterns (PAMPs) that have been shown to mediate the activation of plant defence reactions in many legumes. In soybean, a receptor <span class="trait">protein</span> complex containing a <span class="trait">high</span> affinity beta-glucan-binding <span class="trait">protein</span> (GBP) was identified and investigated in detail. In the model legume Medicago truncatula, used for functional genomic studies of various plant-microbe interactions, a high-affinity beta-glucan-binding site was characterized biochemically. However, to date, none of the genes encoding GBPs from M. truncatula have been described. Here, we report the identification of four full-length clones encoding putative beta-glucan-binding proteins from M. truncatula, MtGBP1, 2, 3, and 4, composing a multigene family encoding GBP-related proteins in this plant. Differences in expression patterns as well as in regulation on treatment with two different biotic elicitors are demonstrated for the members of the <span class="gene">GBP</span> family and for a selection of defence-related genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Promoters of orthologous Glycine max and Lotus japonicus nodulation autoregulation genes interchangeably drive phloem-specific expression in transgenic plants.</h2>
			<h3>PMID:17601165  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nontachaiyapoom S(1), Scott PT, Men AE, Kinkema M, Schenk PM, Gresshoff PM.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nodule autoregulation receptor kinase <span class="gene">(GmNARK)</span> of soybean (Glycine max) is essential for the systemic autoregulation of nodulation. Based on quantitative reverse-transcriptase polymerase chain reaction, <span class="gene">GmNARK</span> is ex-pressed to varying levels throughout the plant; the transcript was detected at <span class="trait">high</span> levels in mature leaves and roots but to a lesser extent in young leaves, shoot tips, and nodules. The transcript level was not significantly affected by Bradyrhizobium japonicum during the first week following inoculation. In addition, the activities of the promoters of <span class="gene">GmNARK</span> and Lotus japonicus HARI, driving a beta-glucuronidase (GUSPlus) reporter gene, were examined in stably transformed L. japonicus and transgenic hairy roots of soybean. Histochemical GUS activity in L. japonicus plants carrying either a 1.7-kb <span class="gene">GmNARKpr::GUS</span> or 2.0-kb LjHAR1pr::GUS construct was clearly localized to living cells within vascular bundles, especially phloem cells in leaves, stems, roots, and nodules. Phloem-specific expression also was detected in soybean hairy roots carrying these constructs. Our study suggests that regulatory elements required for the transcription of these orthologous genes are conserved. Moreover, rapid amplification of 5' cDNA ends (5' rapid amplification of cDNA ends) revealed two major transcripts of <span class="gene">GmNARK</span> potentially originating from two TATA boxes. Further analysis of the <span class="gene">GmNARK</span> promoter has confirmed that these two TATA boxes are functional. Deletion analysis also located a region controlling phloem-specific expression to a DNA sequence between 908 bp and 1.7 kb upstream of the translation start site of <span class="gene">GmNARK.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protease inhibitors from several classes work synergistically against Callosobruchus maculatus.</h2>
			<h3>PMID:17482206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Amirhusin B(1), Shade RE, Koiwa H, Hasegawa PM, Bressan RA, Murdock LL, Zhu-Salzman K.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Targeting multiple digestive proteases may be more effective in insect pest control than inhibition of a single enzyme class. We therefore explored possible interactions of three antimetabolic protease inhibitors fed to cowpea bruchids in artificial diets, using a recombinant soybean cysteine protease inhibitor scN, an aspartic protease inhibitor pepstatin A, and soybean Kunitz trypsin inhibitor KI. scN and pepstatin, inhibiting major digestive cysteine and aspartic proteases, respectively, significantly prolonged the developmental time of cowpea bruchids individually. When combined, the anti-insect effect was synergistic, i.e., the toxicity of the mixture was markedly greater than that of scN or pepstatin alone. <span class="gene">KI</span> alone did not impact insect development even at relatively <span class="trait">high</span> concentrations, but its anti-insect properties became apparent when acting jointly with scN or scN plus pepstatin. Incubating <span class="gene">KI</span> with bruchid midgut extract showed that it was partially degraded. This instability may explain its lack of anti-insect activity. However, this proteolytic degradation was inhibited by scN and/or pepstatin. Protection of <span class="gene">KI</span> from proteolysis in the insect digestive tract thus could be the basis for the synergistic effect. These observations support the concept that cowpea bruchid gut proteases play a dual role; digesting <span class="trait">protein</span> for nutrient needs and protecting insects by inactivating dietary proteins that may otherwise be toxic. Our results also suggest that transgenic <span class="trait">resistance</span> strategies that involve multigene products are likely to have enhanced efficacy and durability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of selective estrogen receptor modulators and genistein on the expression of ERalpha/beta and COX-1/2 in ovarectomized mouse uteri.</h2>
			<h3>PMID:17479667  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu Y(1), Niwa K, Onogi K, Tang L, Mori H, Tamaya T.
				<br/> 
				年份：2007
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This study was performed to examine the effects of selective estrogen receptor modulators [tamoxifen (TAM) and toremifene (TOR)] and pure anti-estrogen, ICI-182780 (ICI, Faslodex) and soybean isoflavone, genistein (GE) on the expression of estrogen-stimulated c-fos/jun, ERalpha/beta and COX-1/2 in the uteri of ovarectomized mice. TAM, TOR, ICI and GE treatment significantly decreased the levels of estradiol (E2)-induced c-fos. ICI and GE treatment significantly decreased the levels of E2-induced c-jun and ERalpha expressions. <span class="trait">High</span> doses of <span class="gene">TOR</span> treatment significantly increased the E2-induced ERbeta expression. In contrast, ICI and GE treatment significantly decreased the levels of E,-induced COX-2 expression, thus suggesting that <span class="gene">TOR</span> and GE might prevent E2-related endometrial carcinogenesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of a novel salt-inducible gene for an <span class="gene">OSBP</span> (oxysterol-binding protein)-homologue from soybean.</h2>
			<h3>PMID:17466467  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li DY(1), Inoue H, Takahashi M, Kojima T, Shiraiwa M, Takahara H.
				<br/> 
				年份：2008 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Oxysterol-binding <span class="trait">protein</span> (OSBP) and its homologues constitute a <span class="trait">protein</span> family in many eukaryotes from yeast to humans, which are involved in cellular lipid metabolism, vesicle transport and signal transduction. In this study, we characterized a novel salt-inducible gene for an OSBP-homologue from soybean (Glycine max [L.] Merr.). The soybean OSBP-homologous gene, denoted as G. max <span class="gene">OSBP</span> <span class="gene">(GmOSBP),</span> encoded a 789 aa putative <span class="trait">protein</span> with two characteristic domains; the pleckstrin homology (PH) domain and the ligand-binding (LB) domain, in the N- and C-terminus, respectively. The <span class="gene">GmOSBP-PH</span> domain showed localization into/around the nucleus in a transient subcellular localization assay. The phylogenetic relationship of the <span class="gene">GmOSBP-LB</span> domain to those in other OSBP-homologues suggested that <span class="gene">GmOSBP</span> might bind a lipid molecule(s) different from the ligand-candidates found for the human/yeast OSBP-homologues. The <span class="gene">GmOSBP</span> gene was constitutively transcribed in all of the soybean organs examined--root, stem and trifoliate leaf--at low levels and was highly induced in all these organs by high-salt stress (300 mM NaCl). Interestingly, gene expression of <span class="gene">GmOSBP</span> was also markedly induced in the senesced soybean cotyledon, which contains <span class="trait">high</span> levels of a variety of cellular lipids utilized for energy for germination and as membrane components. Therefore, we suggest that <span class="gene">GmOSBP</span> may be involved in some physiological reactions for stress-response and cotyledon senescence in the soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in photosynthetic electron transfer and state transitions in an herbicide-resistant D1 mutant from soybean cell cultures.</h2>
			<h3>PMID:17442261  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roncel M(1), Yruela I, Kirilovsky D, Guerrero F, Alfonso M, Picorel R, Ortega JM.
				<br/> 
				年份：2007 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Anomalies in photosynthetic activity of the soybean cell line STR7, carrying a single mutation (S268P) in the chloroplastic gene <span class="gene">psbA</span> that codes for the D1 <span class="trait">protein</span> of the photosystem II, have been examined using different spectroscopic techniques. Thermoluminescence emission experiments have shown important differences between STR7 mutant and wild type cells. The afterglow band induced by both white light flashes and far-red continuous illumination was downshifted by about 4 degrees C and the Q band was upshifted by 5 degrees C. <span class="trait">High</span> temperature thermoluminescence measurements suggested a higher level of lipid peroxidation in mutant thylakoid membranes. In addition, the reduction rate of P700(+) was significantly accelerated in STR7 suggesting that the mutation led to an activation of the photosystem I cyclic electron flow. Modulated fluorescence measurements performed at room temperature as well as fluorescence emission spectra at 77 K revealed that the STR7 mutant is defective in state transitions. Here, we discuss the hypothesis that activation of the cyclic electron flow in STR7 cells may be a mechanism to compensate the reduced activity of photosystem II caused by the mutation. We also propose that the impaired state transitions in the STR7 cells may be due to alterations in thylakoid membrane properties induced by a low content of unsaturated lipids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNAi silencing of genes for elicitation or biosynthesis of 5-deoxyisoflavonoids suppresses race-specific <span class="trait">resistance</span> and hypersensitive cell death in Phytophthora sojae infected tissues.</h2>
			<h3>PMID:17416637  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Graham TL(1), Graham MY, Subramanian S, Yu O.
				<br/> 
				年份：2007 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavonoids are thought to play an important role in soybean (Glycine max) <span class="trait">resistance</span> to Phytophthora sojae. This was addressed by silencing two genes for their biosynthesis and a third gene controlling their elicitation. Silencing of genes for isoflavone synthase (IFS) or chalcone reductase (CHR) was achieved in soybean roots through an Agrobacterium rhizogenes-mediated RNAi approach. Effectiveness of silencing was followed both by quantitative reverse transcriptase-polymerase chain reaction and high-performance liquid chromatography analyses. Silencing either <span class="gene">IFS</span> or CHR led to a breakdown of Rps-mediated <span class="trait">resistance</span> to race 1 of P. sojae in 'W79' (Rps 1c) or 'W82' (Rps 1k) soybean. Loss of <span class="trait">resistance</span> was accompanied by suppression of hypersensitive (HR) cell death in both cultivars and suppression of cell death-associated activation of hydrogen peroxide and peroxidase. The various results suggest that the 5-deoxyisoflavonoids play a critical role in the establishment of cell death and race-specific <span class="trait">resistance.</span> The P. sojae cell wall glucan elicitor, a potent elicitor of 5-deoxyisoflavonoids, triggered a cell death response in roots that was also suppressed by silencing either CHR or IFS. Furthermore, silencing of the elicitor-releasing endoglucanase (PR-2) led to a loss of HR cell death and race-specific <span class="trait">resistance</span> to P. sojae and also to a loss of isoflavone and cell death responses to cell wall glucan elicitor. Taken together, these results suggest that in situ release of active fragments from a general <span class="trait">resistance</span> elicitor (pathogen-associated molecular pattern) is necessary for HR cell death in soybean roots carrying <span class="trait">resistance</span> genes at the Rps 1 locus, and that this cell death response is mediated through accumulations of the 5-deoxyisoflavones.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hypocholesterolaemic effects of soya proteins: results of recent studies are predictable from the anderson meta-analysis data.</h2>
			<h3>PMID:17408521  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sirtori CR(1), Eberini I, Arnoldi A.
				<br/> 
				年份：2007 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In 1995, Anderson et al. published a meta-analysis, derived from most of the clinical studies on soya proteins given to individuals with varying levels of cholesterolaemia that had been reported up to that time. The meta-analysis clearly indicated that cholesterolaemias were generally reduced by diets with soya given as a partial or total substitution of animal proteins, with final mean total and LDL-cholesterol reductions of 23.2 mg/dl and 21.7 mg/dl, respectively. These findings were recently strongly criticised, based on the evaluation of later studies, frequently involving individuals with normal or moderately elevated cholesterolaemias. In the present paper, these more recent studies were re-evaluated using a 'nomogram' prepared on the basis of the quartiles of initial cholesterol concentrations in the Anderson meta-analysis and their corresponding <span class="gene">CI</span> for net cholesterol change. The five studies belonging to the first quartile and thirteen out of the fourteen belonging to the second quartile gave results perfectly in line with the nomogram. Out of the fourteen studies belonging to the third quartile, ten agreed with the nomogram and two gave lower cholesterol reductions, whereas two gave higher reductions. Unfortunately, none of the recent studies belonged to the fourth quartile as treatment with statins or other lipid-lowering drugs is now mandatory in the presence of very <span class="trait">high</span> cholesterol levels. The re-evaluation thus shows that the thirty-three studies published in the past 10 years are in agreement with the Anderson meta-analysis and confirm its validity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">NopM</span> and NopD are rhizobial nodulation outer proteins: identification using LC-MALDI and LC-ESI with a monolithic capillary column.</h2>
			<h3>PMID:17249710  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rodrigues JA(1), López-Baena FJ, Ollero FJ, Vinardell JM, Espuny Mdel R, Bellogín RA, Ruiz-Sainz JE, Thomas JR, Sumpton D, Ault J, Thomas-Oates J.
				<br/> 
				年份：2007 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have explored the potential of commercial polystyrene-divinylbenzene monolithic capillary nanoLC-MS/MS for identifying Sinorhizobium fredii HH103 nodulation outer proteins. Monolithic nanoLC with off-line MALDI-TOF/TOF and on-line ESI-q-oTOF is fast and robust, generating complementary data and offering high-confidence <span class="trait">protein</span> identifications from gel bands too weak for successful analysis using traditional approaches. This has allowed identification of two proteins not previously described as being type III-secreted in rhizobia, NopM and NopD.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the sucrose binding <span class="trait">protein</span> from soybean: renaturation and stability of the recombinant <span class="trait">protein.</span></h2>
			<h3>PMID:17222874  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rocha CS(1), Luz DF, Oliveira ML, Baracat-Pereira MC, Medrano FJ, Fontes EP.
				<br/> 
				年份：2007 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The sucrose binding <span class="trait">protein</span> (SBP) belongs to the cupin family of proteins and is structurally related to vicilin-like storage proteins. In this investigation, a <span class="gene">SBP</span> isoform <span class="gene">(GmSBP2/S64)</span> was expressed in E. coli and large amounts of the <span class="trait">protein</span> accumulated in the insoluble fraction as inclusion bodies. The renatured <span class="trait">protein</span> was studied by circular dichroism (CD), intrinsic fluorescence, and binding of the hydrophobic probes <span class="gene">ANS</span> and Bis-ANS. The estimated content of secondary structure of the renatured <span class="trait">protein</span> was consistent with that obtained by theoretical modeling with a large predominance of beta-strand structure (42%) over the alpha-helix (9.9%). The fluorescence emission maximum of 303 nm for <span class="gene">SBP2</span> indicated that the fluorescent tryptophan was completely buried within a highly hydrophobic environment. We also measured the equilibrium dissociation constant (K(d)) of sucrose binding by fluorescence titration using the refolded <span class="trait">protein.</span> The low sucrose binding affinity (K(d)=2.79+/-0.22 mM) of the renatured <span class="trait">protein</span> was similar to that of the native <span class="trait">protein</span> purified from soybean seeds. Collectively, these results indicate that the folded structure of the renatured <span class="trait">protein</span> was similar to the native <span class="gene">SBP</span> <span class="trait">protein.</span> As a member of the bicupin family of proteins, which includes highly stable seed storage proteins, <span class="gene">SBP2</span> was fairly stable at <span class="trait">high</span> temperatures. Likewise, it remained folded to a similar extent in the presence or absence of 7.6M urea or 6.7 M GdmHCl. The <span class="trait">high</span> stability of the renatured <span class="trait">protein</span> may be a reminiscent property of <span class="gene">SBP</span> from its evolutionary relatedness to the seed storage proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Alternative workflows for plant proteomic analysis.</h2>
			<h3>PMID:17216043  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lee J(1), Cooper B.
				<br/> 
				年份：2006 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High-throughput separations are intrinsic to the detection and analysis of peptides and proteins by mass spectrometry (MS). Together, efficient separation and <span class="gene">MS</span> can lead to the identification of thousands of proteins in a sample, cell or tissue and help build proteome maps that can be used to define a cell type or cellular state. Although 2D gels have been successfully used to separate proteins for subsequent <span class="gene">MS</span> analysis, alternative separation efficiencies and, consequently deeper results could be obtained with HPLC or other separation techniques that improve throughput. This highlight is aimed toward plant scientists who have special separation needs due to the nature of plant cells and who could benefit from knowing options and requirements for adopting alternative separation protocols. Through the various sample processing and <span class="trait">protein</span> separation strategies, plant biologists should be able to improve the quality of their proteomic reference maps and gain new information about the proteins that define plant cells.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmDREB2, a soybean DRE-binding transcription factor, conferred <span class="trait">drought</span> and high-salt tolerance in transgenic plants.</h2>
			<h3>PMID:17178106  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen M(1), Wang QY, Cheng XG, Xu ZS, Li LC, Ye XG, Xia LQ, Ma YZ.
				<br/> 
				年份：2007 Feb 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel DREB (dehydration responsive element binding <span class="trait">protein)</span> homologous gene, <span class="gene">GmDREB2,</span> was isolated from soybean. Based on its similarity with <span class="gene">AP2</span> domains, <span class="gene">GmDREB2</span> was classified into A-5 subgroup in DREB subfamily in AP2/EREBP family. Expression of <span class="gene">GmDREB2</span> gene was induced by <span class="trait">drought,</span> <span class="trait">high</span> salt, and low temperature stresses and abscisic acid treatment. The <span class="gene">GmDREB2</span> bound specifically to DRE element in vitro. Furthermore, the overexpression of <span class="gene">GmDREB2</span> activated expression of downstream genes in transgenic Arabidopsis, resulting in enhanced tolerance to <span class="trait">drought</span> and high-salt stresses and did not cause growth retardation. Analysis of free proline contents in transgenic tobacco indicated that the overexpression of <span class="gene">GmDREB2</span> accumulated higher level of free proline compared to the wild type plants under <span class="trait">drought</span> condition. The results from this study indicate that this novel soybean <span class="gene">GmDREB2</span> gene functions as an important transcriptional activator and may be useful in improving of plant tolerance to abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cloning of <span class="gene">GmHSFA1</span> gene and its overexpression leading to enhancement of heat tolerance in transgenic soybean].</h2>
			<h3>PMID:17098711  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen XJ(1), Ye CJ, Lü HY, Xu MX, Li W, Zhang LM, Wang C, Luo SP, Zhu BG.
				<br/> 
				年份：2006 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heat shock transcription factors (HSFs) are important in regulating heat stress response by mediating expression of heat shock <span class="trait">protein</span> (HSP) genes in various plant species. In the present study, a novel <span class="gene">GmHSFA1</span> with an ORF of 1,533 bp (full-length cDNA sequence of 1,781 bp) was cloned from soybean genome via comparative genomic approach and RACE (rapid amplification of cDNA ends). This gene encodes 510 amino acids consisting of a <span class="trait">protein</span> of 56.2 kDa (GenBank accession number: AY458843). Similar to other HSFs, <span class="gene">GmHSFA1</span> has the basic modular structure including DBD, OD, NLS, and CTAD. BLAST analysis revealed the identity of 52.46% between amino acid sequences between <span class="gene">GmHSFA1</span> and LpHSFA1 that has the highest similarity to <span class="gene">GmHSFA1</span> in all HSFA1s in various plant species. The results from RT-PCR, Northern blotting, and transformation showed: 1) <span class="gene">GmHsfA1</span> exhibited the constitutive expression patterns in different tissues of soybean; 2) The expression level of <span class="gene">GmHsfA1</span> in transgenic plants was notably higher than that in non-transgenic plants; 3) Overexpression of <span class="gene">GmHsfA1</span> activated transcription of <span class="gene">GmHSP22</span> in transgenic plants under normal conditions and enhanced obviously expressions of <span class="gene">GmHSP23</span> and <span class="gene">GmHSP70</span> in transgenic plants under heat stress conditions; 4) Heat tolerant temperature (as <span class="trait">high</span> as 52 degrees C) of transgenic plants was remarkably higher than that of non-transgenic plants. These results preliminarily proved that the overexpression of <span class="gene">GmHsfA1</span> possibly led to the notable enhancement of heat-tolerant level of transgenic plants by mediating the activation of transcription or improvement of expression of some <span class="gene">GmHSPs</span> in the <span class="gene">GmHsfA1's</span> downstream in transgenic plants, suggesting <span class="gene">GmHSFA1</span> is a novel and functional heat shock transcription factor of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Spectroscopic characterization of a higher plant heme oxygenase isoform-1 from Glycine max (soybean)--coordination structure of the heme complex and catabolism of heme.</h2>
			<h3>PMID:17076701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gohya T(1), Zhang X, Yoshida T, Migita CT.
				<br/> 
				年份：2006 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heme oxygenase converts heme into biliverdin, CO, and free iron. In plants, as well as in cyanobacteria, heme oxygenase plays a particular role in the biosynthesis of photoreceptive pigments, such as phytochromobilins and phycobilins, supplying biliverdin IX(alpha) as a direct synthetic resource. In this study, a higher plant heme oxygenase, <span class="gene">GmHO-1,</span> of Glycine max (soybean), was prepared to evaluate the molecular features of its heme complex, the enzymatic activity, and the mechanism of heme conversion. The similarity in the amino acid sequence between <span class="gene">GmHO-1</span> and heme oxygenases from other biological species is low, and <span class="gene">GmHO-1</span> binds heme with 1 : 1 stoichiometry at His30; this position does not correspond to the proximal histidine of other heme oxygenases in their sequence alignments. The heme bound to <span class="gene">GmHO-1,</span> in the ferric high-spin state, exhibits an acid-base transition and is converted to biliverdin IX(alpha) in the presence of NADPH/ferredoxin reductase/ferredoxin, or ascorbate. During the heme conversion, an intermediate with an absorption maximum different from that of typical verdoheme-heme oxygenase or CO-verdoheme-heme oxygenase complexes was observed and was extracted as a bis-imidazole complex; it was identified as verdoheme. A myoglobin mutant, H64L, with <span class="trait">high</span> CO affinity trapped CO produced during the heme degradation. Thus, the mechanism of heme degradation by <span class="gene">GmHO-1</span> appears to be similar to that of known heme oxygenases, despite the low sequence homology. The heme conversion by <span class="gene">GmHO-1</span> is as fast as that by SynHO-1 in the presence of NADPH/ferredoxin reductase/ferredoxin, thereby suggesting that the latter is the physiologic electron-donating system.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and characterization of a novel salt-inducible gene encoding an acidic isoform of PR-5 <span class="trait">protein</span> in soybean (Glycine max [L.] Merr.).</h2>
			<h3>PMID:17070691  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Onishi M(1), Tachi H, Kojima T, Shiraiwa M, Takahara H.
				<br/> 
				年份：2006 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We identified a novel salt-inducible soybean gene encoding an acidic-isoform of pathogenesis-related <span class="trait">protein</span> group 5 (PR-5 <span class="trait">protein).</span> The soybean PR-5-homologous gene, designated as Glycine max osmotin-like <span class="trait">protein,</span> acidic isoform <span class="gene">(GmOLPa)),</span> encodes a putative polypeptide having an N-terminal signal peptide. The mature <span class="gene">GmOLPa</span> <span class="trait">protein</span> without the signal peptide has a calculated molecular mass of 21.5 kDa and a pI value of 4.4, and was distinguishable from a known PR-5-homologous gene of soybean (namely P21 <span class="trait">protein)</span> through examination of the structural features. A comparison with two intracellular salt-inducible PR-5 proteins, tobacco osmotin and tomato NP24, revealed that <span class="gene">GmOLPa</span> did not have a C-terminal extension sequence functioning as a vacuole-targeting motif. The <span class="gene">GmOLPa</span> gene was transcribed constitutively in the soybean root and was induced almost exclusively in the root during 24 h of high-salt stress (300 mM NaCl). Interestingly, <span class="gene">GmOLPa</span> gene expression in the stem and leaf, not observed until 24 h, was markedly induced at 48 and 72 h after commencement of the high-salt stress. Abscisic acid (ABA) and dehydration also induced expression of the <span class="gene">GmOLPa</span> gene in the root; additionally, dehydration slightly induced expression in the stem and leaf. In fact, the 5'-upstream sequence of the <span class="gene">GmOLPa</span> gene contained several putative cis-elements known to be involved in responsiveness to ABA and dehydration, e.g. ABA-responsive element (ABRE), MYB/MYC, and low temperature-responsive element (LTRE). These results suggested that <span class="gene">GmOLPa</span> may function as a protective PR-5 <span class="trait">protein</span> in the extracellular space of the soybean root in response to high-salt stress and dehydration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:ESR spectroscopy investigation of the denaturation process of soybean peroxidase induced by guanidine hydrochloride, DMSO or heat.</h2>
			<h3>PMID:16967316  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Boscolo B(1), Laurenti E, Ghibaudi E.
				<br/> 
				年份：2006 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The involvement of <span class="trait">protein</span> denaturation and/or misfolding processes in the insurgence of several diseases raises the interest in structural dynamic studies of proteins. The use of nitroxide spin labels with electron paramagnetic resonance is a powerful tool for detecting structural changes in proteins. In the present study, we apply this strategy to soybean peroxidase (SBP), a <span class="trait">protein</span> characterised by <span class="trait">high</span> thermal and structural stability, and we propose a simple method to analyse the anisotropy changes of the <span class="trait">protein</span> system and to relate them with the structural changes induced by <span class="trait">protein</span> unfolding. We examined the effect of temperature, guanidine hydrochloride and dimethylsulfoxide on the stability of <span class="gene">SBP</span> and looked for correlations between the ESR results and the experimental findings obtained by other techniques, reported in the literature. The agreement between data obtained through different strategies supports the validity and reliability of the ESR approach to <span class="trait">protein</span> unfolding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of varying dietary ratios of alfalfa silage to corn silage on omasal flow and microbial <span class="trait">protein</span> synthesis in dairy cows.</h2>
			<h3>PMID:16960069  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brito AF(1), Broderick GA, Reynal SM.
				<br/> 
				年份：2006 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Eight ruminally cannulated multiparous Holstein cows that were part of a larger production trial were used to study the effects of varying dietary ratios of alfalfa silage (AS) to corn silage (CS) on omasal flow of nutrients and microbial <span class="trait">protein.</span> Cows were blocked by <span class="gene">DIM</span> and randomly assigned to 2 replicated 4 x 4 Latin squares (28-d periods). Diets fed contained (dry matter basis): A) 51% AS, 43% rolled high-moisture shelled corn (HMSC), and 3% solvent soybean meal (SSBM); B) 37% AS, 13% CS, 39% HMSC, and 7% SSBM; C) 24% AS, 27% CS, 35% HMSC, and 12% SSBM; or D) 10% AS, 40% CS, 31% HMSC, and 16% SSBM. Crude <span class="trait">protein</span> (CP) contents were 17.2, 16.9, 16.6, and 16.2% for diets A, B, C, and D. All 4 diets were <span class="trait">high</span> in energy, averaging 49% nonfiber carbohydrates and 24% neutral detergent fiber. Total microbial nonammonia nitrogen flow was lower on diet D (423 g/d) compared with diets A (465 g/d), B (479 g/d), and C (460 g/d). A significant quadratic effect indicated that microbial <span class="trait">protein</span> synthesis was maximal at 38% AS. Supply of rumen-degraded <span class="trait">protein</span> decreased linearly from 3,068 g/d (diet A) to 2,469 g/d (diet D). Omasal flow of rumen-undegraded <span class="trait">protein</span> did not differ among diets and averaged 1,528 g/d. However, when expressed as a percentage of dry matter intake, rumen-undegraded <span class="trait">protein</span> increased linearly from 5.59% (diet A) to 6.13% (diet D), probably because CP from SSBM was more resistant to degradation than CP from AS. Essential AA flow was lowest on diet D, and Lys flow tended to be lower on diet D, which may explain the lower milk and <span class="trait">protein</span> yields observed on that diet.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and characterization of a novel soybean receptor-like <span class="trait">protein</span> kinase gene <span class="gene">GmRLCK.</span></h2>
			<h3>PMID:16944594  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li PL(1), Ma YY, Gan R, Zhang LW, Li XP, Wang Y, Wang NN.
				<br/> 
				年份：2006 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Protein</span> kinases play key roles in transduction of both environmental and developmental signals in higher plants. Here reported full-length cDNA cloning and preliminary structural and functional analysis of a novel soybean receptor-like <span class="trait">protein</span> kinase (RLK) gene (GenBank Accession No.AY687390). The deduced <span class="trait">protein</span> product of this gene contains a transmembrane region, an intracellular catalytic domain possessing serine/threonine kinase activity, and an extracellular domain which lacks the N-terminus signal peptide. Bioinformatic analyses revealed highly similarity between this soybean gene and a group of the Arabidopsis <span class="gene">RLK</span> genes which all lack of N-terminus signal sequence and belong to RLCK subfamily. Thus, it was designated <span class="gene">GmRLCK.</span> Some residues of <span class="gene">GmRLCK</span> that showed <span class="trait">high</span> probability to be phosphorylated were also predicted. RT-PCR analysis revealed higher expression levels of <span class="gene">GmRLCK</span> in soybean cotyledons, roots, flowers and young pods; whereas lower expression levels were detected in radicles, stems and mature leaves at vegetative stage. Phylogeny analysis suggested that <span class="gene">GmRLCK</span> was more close to and might have a common ancestor with several senescence-associated RLKs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A multi-needle-assisted transformation of soybean cotyledonary node cells.</h2>
			<h3>PMID:16937246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xue RG(1), Xie HF, Zhang B.
				<br/> 
				年份：2006 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new and simple method for wounding cotyledonary node cells of soybean [Glycine max (L) Merrill] was developed for obtaining a <span class="trait">high</span> frequency of transformants. Soybean seeds were germinated for 1 day, and the cotyledonary node cells of half-seeds were wounded mechanically by using a multi-needle consisting of thin 30 fibers. The wounded half-seeds were inoculated with Agrobacterium tumefaciens cells harboring a recombinant DNA that contained the <span class="gene">bar</span> and sgfp genes conferring phosphinothricin (PPT)-resistance and green fluorescent <span class="trait">protein</span> (GFP) activity, respectively. The inoculated explants were selected on medium containing 5 or 3 mg PPT/l. The transformation efficiency of soybean was up to 12%. Polymerase chain reaction and genomic Southern blot analysis confirmed stable integration of the transgenes in the genome of the PPT-resistant plants. GFP analysis revealed that the transgenes were highly expressed in the plantlets. Adult plants were resistant to 100 mg PPT/l applied on the leaves, demonstrating their herbicide-resistance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of soybean D-LeciVita product on serum lipids and fatty acid composition in type 2 diabetic patients with hyperlipidemia.</h2>
			<h3>PMID:16935698  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ristić Medić D(1), Ristić V, Arsić A, Postić M, Ristić G, Blazencić Mladenović V, Tepsić J.
				<br/> 
				年份：2006 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIM: Hyperlipidemia is one of the major risk factors of cardiovascular complication in diabetes. <span class="trait">High</span> intake of soy product has been suggested to prevent cardiovascular disease. The purpose of this study was to evaluate if dietary supplement of soybean D-LeciVita product, rich in polyunsaturated phospholipids (with 12% lecithin, 35% soy <span class="trait">protein)</span> affects serum lipids and serum and erythrocyte phospholipid fatty acid composition in type 2 diabetic patients. METHODS AND RESULTS: Forty-seven patients (men and post-menopausal women) with isolated hypertriglyceridemia (IHTG) and combined hyperlipidemia (CHL), aged 43-70 years, were given 15g of D-LeciVita powder as a water suspension in a single evening dose during the follow-up period of 12 weeks. Patients kept their diabetic diet relatively constant. Treatment was associated with a significant (p &lt; or = 0.001) decrease in serum total cholesterol and triglyceride levels by 12% and 22%, respectively. LDL-cholesterol decreased by 16% and HDL-cholesterol increased by 11% (p &lt; or = 0.001). Our study shows a 27% decrease in LDL-cholesterol (p &lt; or = 0.001) and a 12% increase in HDL-cholesterol (p &lt; or = 0.01) in <span class="gene">CHL</span> type 2 diabetic patients. Triglyceride levels decreased in type 2 diabetic patients with IHTG and <span class="gene">CHL</span> by 29% and 13%, respectively (p &lt; or = 0.01 and p &lt; or = 0.05). Our results show decrease in SFA and increase in n-6 and n-3 PUFA in serum and erythrocyte phospholipids. SFA decreased and n-3 PUFA increased in serum and erythrocyte phospholipids in IHTG and <span class="gene">CHL</span> groups. CONCLUSION: The present study indicated that added to a regular diet, soybean D-LeciVita product (combination of soy <span class="trait">protein</span> and lecithin) is associated not only with lipid-lowering effects but also with more favorable serum phospholipids fatty acid profile in type 2 diabetic patients with hyperlipidemia.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A meta-analysis of the effect of soy <span class="trait">protein</span> supplementation on serum lipids.</h2>
			<h3>PMID:16923451  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reynolds K(1), Chin A, Lees KA, Nguyen A, Bujnowski D, He J.
				<br/> 
				年份：2006 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Hypercholesterolemia is a major modifiable risk factor for cardiovascular disease. Some, but not all, studies have shown that soy <span class="trait">protein</span> intake decreases total and low-density lipoprotein cholesterol and triglycerides and increases high-density lipoprotein cholesterol. The objective of this meta-analysis was to examine the effect of soy <span class="trait">protein</span> supplementation on serum lipid levels in adults. English language articles were retrieved by searching MEDLINE (1966 to February 2005) and the bibliographies of the retrieved articles. A total of 41 randomized controlled trials in which isolated soy <span class="trait">protein</span> supplementation was the only intervention and the net changes in serum lipids during intervention were reported. Information on study design, sample size, participant characteristics, intervention, follow-up duration, and treatment outcomes was independently abstracted using a standardized protocol. Using a random-effects model, data from each study were pooled and weighted by the inverse of their variance. Soy <span class="trait">protein</span> supplementation was associated with a significant reduction in mean serum total cholesterol (-5.26 mg/dl, 95% confidence interval [CI] -7.14 to -3.38), low-density lipoprotein cholesterol (-4.25 mg/dl, 95% <span class="gene">CI</span> -6.00 to -2.50), and triglycerides (-6.26 mg/dl, 95% <span class="gene">CI</span> -9.14 to -3.38) and a significant increase in high-density lipoprotein cholesterol (0.77 mg/dl, 95% <span class="gene">CI</span> 0.20 to 1.34). Meta-regression analyses showed a dose-response relation between soy <span class="trait">protein</span> and isoflavone supplementation and net changes in serum lipids. These results indicate that soy <span class="trait">protein</span> supplementation reduces serum lipids among adults with or without hypercholesterolemia. In conclusion, replacing foods <span class="trait">high</span> in saturated fat, trans-saturated fat, and cholesterol with soy <span class="trait">protein</span> may have a beneficial effect on coronary risk factors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Estimation of nitrogen maintenance requirements and potential for nitrogen deposition in fast-growing chickens depending on age and sex.</h2>
			<h3>PMID:16903472  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Samadi F(1), Liebert F.
				<br/> 
				年份：2006 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments were conducted to estimate daily N maintenance requirements (NMR) and the genetic potential for daily N deposition (ND(max)T) in fast-growing chickens depending on age and sex. In N-balance studies, 144 male and 144 female chickens (Cobb 500) were utilized in 4 consecutive age periods (I: 10 to 25 d; II: 30 to 45 d; III: 50 to 65 d; and IV: 70 to 85 d). The experimental diets contained high-protein soybean meal and crystalline amino acids as <span class="trait">protein</span> sources and 6 graded levels of <span class="trait">protein</span> supply (N1 = 6.6%; N2 = 13.0%; <span class="gene">N3</span> = 19.6%; N4 = 25.1%; N5 = 31.8%; and N6 = 37.6% CP in DM). The connection between N intake and total N excretion was fitted for NMR determination by an exponential function. The average NMR value (252 mg of N/BW(kg)0.67 per d) was applied for further calculation of ND(max)T as the threshold value of the function between N intake and daily N balance. For estimating the threshold value, the principle of the Levenberg-Marquardt algorithm within the SPSS program (Version 11.5) was applied. As a theoretical maximum for ND(max)T, 3,592, 2,723, 1,702, and 1,386 mg of N/BW(kg)0.67 per d for male and 3,452, 2,604, 1,501, and 1,286 mg of N/BW(kg)0.67 per d for female fast-growing chickens (corresponding to age periods I to IV) were obtained. The determined model parameters were the precondition for modeling of the amino acid requirement based on an exponential N-utilization model and depended on performance and dietary amino acid efficiency. This procedure will be further developed and applied in the subsequent paper.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lupin <span class="trait">protein</span> attenuates the development of hypertension and normalises the vascular function of NaCl-loaded Goto-Kakizaki rats.</h2>
			<h3>PMID:16845223  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pilvi TK(1), Jauhiainen T, Cheng ZJ, Mervaala EM, Vapaatalo H, Korpela R.
				<br/> 
				年份：2006 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The beneficial cardiovascular effects of soy <span class="trait">protein</span> have been studied intensively in recent years. Another protein-rich legume is lupin, which has been shown to have similar effects to those of soy in lowering serum cholesterol levels. In this study we compared the effects of lupin and soy <span class="trait">protein</span> on hypertension and vascular functions in spontaneously diabetic Goto-Kakizaki (GK) rat, which develop hypertension when fed a high-salt diet. The rats were fed with a 6% NaCl diet containing either lupin or soy <span class="trait">protein</span> isolate (20% weight/weight) for two weeks. In the end of the study the <span class="gene">SBP</span> was 18.6 mmHg lower (p&lt;0.001) in the lupin group, and 12.0 mmHg lower (p&lt;0.01) in the soy group than in the control group. Lupin and soy treatments normalised the decreased vasocontraction observed in the NaCl-fed control group, but only lupin treatment improved the impaired endothelium-dependent vasorelaxation. The attenuation of hypertension is likely to be mediated by the corrected vascular dysfunction, whose precise mechanism and the possible clinical relevance remains to be studied further.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Refolding, purification and characterization of replication-initiator <span class="trait">protein</span> from soybean-infecting geminivirus.</h2>
			<h3>PMID:16765456  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Girish KR(1), Palanivelu S, Kumar PD, Usha R.
				<br/> 
				年份：2006 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The replication-initiator <span class="trait">protein</span> (Rep) from a soybean-infecting geminivirus was overexpressed in E. coli as a fusion <span class="trait">protein</span> with maltose binding <span class="trait">protein</span> (MBP). In spite of the presence of the highly soluble MBP as the fusion partner, the overexpressed MBP-Rep fusion <span class="trait">protein</span> formed insoluble inclusion bodies. The <span class="trait">protein</span> was solubilized from the inclusion bodies and refolded. The refolded MBP-Rep <span class="trait">protein</span> was purified using ion exchange and amylose affinity chromatography. The activity of the purified MBP-Rep was assessed using an in vitro cleavage assay. Soluble and stable MBP-Rep <span class="trait">protein</span> was obtained in <span class="trait">high</span> abundance, providing the feasibility of large-scale production of active <span class="gene">Rep</span> <span class="trait">protein</span> for functional characterization and X-ray crystallographic structure determination.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel lipoxygenase in pea roots. Its function in wounding and biotic stress.</h2>
			<h3>PMID:16679421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Veronico P(1), Giannino D, Melillo MT, Leone A, Reyes A, Kennedy MW, Bleve-Zacheo T.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genome of pea (Pisum sativum) contains genes encoding a family of distinct lipoxygenases (LOX). Among these, LOXN2 showed eight exons encoding a 93.7-kD enzyme, harboring two C-terminal deletions and an unusual arginine/threonine-tyrosine motif in the domain considered to control the substrate specificity. LOXN2, when overexpressed in yeast, exhibited normal enzyme activity with an optimum at pH 4.5, and a dual positional specificity by releasing a 3:1 ratio of C-9 and C-13 oxidized products. The predicted LOXN2 structure lacked a loop present in soybean (Glycine max) LOX1, in a position consistent with control of the degree of substrate access to the catalytic site and for LOXN2's dual positional specificity. The LOXN2 gene was tightly conserved in the Progress 9 and MG103738 genotypes, respectively, susceptible and resistant to the root cyst nematode Heterodera goettingiana. LOXN2 transcription was monitored in roots after mechanical injury and during nematode infection. The message peaked at 3 and 24 h after wounding in both genotypes and was more abundant in the resistant than in the susceptible pea. In nematode-infected roots, transcription of several <span class="gene">LOX</span> genes was triggered except LOXN2, which was repressed in both genotypes. In situ hybridization revealed that LOXN2 message was widespread in the cortex and endodermis of healthy roots, but specifically localized at <span class="trait">high</span> level in the cells bordering the nematode-induced syncytia of infected roots. However, LOXN2 transcript signal was particularly intense in collapsing syncytia of MG103738 roots, suggesting LOXN2 involvement in late mechanisms of host <span class="trait">resistance.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic characterization of inter-alpha inhibitor proteins from human plasma.</h2>
			<h3>PMID:16596706  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Josic D(1), Brown MK, Huang F, Lim YP, Rucevic M, Clifton JG, Hixson DC.
				<br/> 
				年份：2006 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Inter-alpha inhibitor proteins (IaIp) are a family of structurally related serine protease inhibitors found in relatively <span class="trait">high</span> concentrations in human plasma. Recent studies have implicated a role for IaIp in sepsis, and have demonstrated their potential as biomarkers in sepsis and cancer. For characterization of isolated IaI proteins and contaminating proteins during the last steps of the purification process, SELDI-TOF <span class="gene">MS</span> and HPLC-ESI-MS/MS were used. After separation by SDS-PAGE or 2-DE, polypeptide bands of 80, 125 and 250 kDa were excised from gels and digested by trypsin. The tryptic peptides were analyzed by both <span class="gene">MS</span> methods. The main contamination during the purification process, a band of 80 kDa, contains mainly IaIp heavy chain (HC) H3. HC <span class="gene">H1</span> and H2 were also found in this band. In addition, some vitamin K-dependent clotting factors and inhibitors and other plasma proteins were identified. The 125-kDa band, representing the pre-alpha inhibitor, was found to contain both bikunin and HC H3. The presence of other HC H1, H2 and the recently described HC H4 was also detected by SELDI-TOF MS. The presence of HC H1, H2, and <span class="gene">H3</span> in the 125-kDa band was confirmed by ESI-MS/MS, but not the presence of the H4. Three polypeptides, <span class="gene">H1</span> and H2 together with bikunin, were identified in the 250-kDa band, representing the ITI, by both <span class="gene">MS</span> techniques. Once again, the presence of H4 was detected in this band only by SELDI-TOF MS, but the number of corresponding peptides was still not sufficient for final identification of this polypeptide. The importance of the application of proteomic methods for the proper evaluation of therapeutic drugs based on human plasma is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Functional expression of an omega-3 fatty acid desaturase gene from Glycine max in Saccharomyces cerevisiae].</h2>
			<h3>PMID:16572837  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang HT(1), Yang JS, Shan L, Bi YP.
				<br/> 
				年份：2006 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Alpha-linolenic acid(ALA, C18:3delta9,12,15 ) is an essential fatty acid which has many sanitary functions to human. However, its contents in diets are often not enough. In plants, omega-3 fatty acid desaturases(FAD) catalyze linoleic acid(LA, C18:2delta9,12) into ALA. The seed oil of Glycine max contains <span class="trait">high</span> level of ALA. To investigate the functions of Glycine max omega-3FAD, the cDNA of <span class="gene">GmFAD3</span> C was amplified by RT-PCR from immature seeds, then cloned into the shuttle expression vector p416 to generate the recombinant vector p4GFAD3C. The resulting vector was transformed into Saccharomyces cerevisiae K601 throuth LiAc method. The positive clones were screened on the CM(Ura-) medium and identified by PCR, and then cultured in CM (Ura-) liquid medium with exogenous LA in 20 degrees C for three days. The intracellular fatty acid composition of the engineering strain Kp416 and Kp4GFAD3C was analyzed by gas chromatography (GC). A novel peak in strain Kp4GFAD3C was detected,which was not detectable in control, Comparison of the retention times of the newly yielded peak with that of authentic standard indicated that the fatty acid is ALA. The content of ALA reached to 3.1% of the total fatty acid in recombinant strain, the content of LA correspondingly decreased from 22% to 16.2% by contrast. It was suggested that the <span class="trait">protein</span> encoded by <span class="gene">GmFAD3</span> C can specifically catalyze 18 carbon PUFA substrate of LA into ALA by taking off hydrogen atoms at delta15 location. In this study, we expressed a Glycine max omega-3 fatty acid desaturase gene in S. cerevisiae; An efficient and economical yeast expressing system(K601-p416 system) which is suitable for the expression of FAD was built.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a novel heat shock transcription factor gene, <span class="gene">GmHsfA1,</span> in soybeans (Glycine max).</h2>
			<h3>PMID:16570125  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhu B(1), Ye C, Lü H, Chen X, Chai G, Chen J, Wang C.
				<br/> 
				年份：2006 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants have a large family of HSFs with different roles in the heat shock response that mediate the expression of HSP regulated genes. The <span class="gene">HSF</span> encoding genes are easily identified by their highly conserved modular structure and motifs. In the present study, a putative <span class="gene">GmHsfA1</span> was identified and characterized from the soybean expressed sequence tag (EST) database by sequence comparison with the functionally well-characterized LpHsfA1 and rapid amplification of cDNA ends (RACE). Multiple alignment showed that the amino acid sequence of <span class="gene">GmHSFA1,</span> matching best with LpHSFA1 (52.2% similarity), was obviously different from that of each of several HSFA1s from other plant species. The <span class="gene">GmHsfA1</span> has a constitutive expression profile in the different tissues examined. The overexpression of <span class="gene">GmHsfA1</span> in transgenic soybean plants led to the activation of <span class="gene">GmHsp70</span> under normal temperature and the overexpression of <span class="gene">GmHsp70</span> under <span class="trait">high</span> temperature. Furthermore, transgenic soybean plants with <span class="gene">GmHsfA1</span> overexpression showed obvious enhancement of thermotolerance under heat stress in comparison with non-transgenic plants. The experimental results suggested that <span class="gene">GmHSFA1</span> is a novel and functional heat-shock transcription factor.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The 3' untranslated region of a soybean cytosolic glutamine synthetase (GS1) affects transcript stability and <span class="trait">protein</span> accumulation in transgenic alfalfa.</h2>
			<h3>PMID:16460515  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ortega JL(1), Moguel-Esponda S, Potenza C, Conklin CF, Quintana A, Sengupta-Gopalan C.
				<br/> 
				年份：2006 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Higher plants assimilate nitrogen in the form of ammonia through the concerted activity of glutamine synthetase (GS) and glutamate synthase (GOGAT). The <span class="gene">GS</span> enzyme is either located in the cytoplasm (GS1) or in the chloroplast (GS2). Glutamine synthetase 1 is regulated in different plants at the transcriptional level and there are some reports of regulation at the level of <span class="trait">protein</span> stability. Here we present data that clearly establish that GS1 in plants is also regulated at the level of transcript turnover and at the translational level. Using a Glycine max (soybean) GS1 transgene, with and without its 3' untranslated region (UTR), driven by the constitutive CaMV 35S promoter in Medicago sativa (alfalfa) and Nicotiana tabacum (tobacco), we show that the 3' UTR plays a major role in both transcript turnover and translation repression in both the leaves and the nodules. Our data suggest that the 3' UTR mediated turnover of the transcript is regulated by a nitrogen metabolite or carbon/nitrogen ratios. We also show that the 3' UTR of the gene for the soybean GS1 confers post-transcriptional regulation on a reporter gene. Our dissection of post-transcriptional and translational levels of regulation of <span class="gene">GS</span> in plants shows that the situation in plants strongly resembles that in other organisms where <span class="gene">GS</span> is regulated at almost all levels. Multistep regulation of <span class="gene">GS</span> shows the <span class="trait">high</span> priority given by organisms to regulating and ensuring optimal control of nitrogen substrates and preventing overproduction of glutamine and drainage of the glutamate pool.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cereal bars with soy <span class="trait">protein</span> and wheat germ, physicochemical characteristics and texture during the storage].</h2>
			<h3>PMID:16454057  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Castro Freitas DD(1).
				<br/> 
				年份：2005 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Studies analyzing cereal bars have reported on consumer characteristics and preferences in sensory analyses and on their market growth, however little has been published on their physicochemical data and texture properties. Thus the objective of this research was to provide information about the storage of a cereal <span class="gene">bar</span> formulation with <span class="trait">high</span> <span class="trait">protein</span> and vitamin levels based on soy <span class="trait">protein</span> and wheat germ, packaged in 3 different films (A: PET/PEBD; B: PETmet/PEBD; C: PET/PEBD/AL/PEBD), during 6 months under environmental conditions of temperature (25 +/- 2 degrees C) and relative humidity (56%). The moisture content, water activity, pH and total acidity of the cereal bars were determined. The textural measurements accompanied during storage were breaking strength, hardness and cohesiveness. The cereal bars presented variations in water activity (Aw), moisture content and total acidity during storage. The moisture content of the bars tended to increase, which led to a significant (p = 0.05) influence on the texture characteristics of breaking strength and hardness, in the different packaging films tested. The increase in the values for breaking strength (A: 4756,5N; B: 5093,0N; C: 5575,6N) at 45 days of storage was attributed to a possible crystallization of the agglutinating syrup used for the bars. The textured soy <span class="trait">protein</span> used in the formulation could also have contributed to this fact due to its hygroscopic character, also interfering in the decrease in the cohesiveness measurements (deformation) with time. The effect of the different barrier properties of the packaging films tested was significant (p &lt; 0.05) in the stability of the cereal bars during storage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Substrate specificity of insect trypsins and the role of their subsites in catalysis.</h2>
			<h3>PMID:16431280  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lopes AR(1), Juliano MA, Marana SR, Juliano L, Terra WR.
				<br/> 
				年份：2006 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Trypsins have <span class="trait">high</span> sequence similarity, although the responses of insect trypsins to chemical and natural inhibitors suggest they differ in specificities. Purified digestive trypsins from insects of four different orders were assayed with internally quenched fluorescent oligopeptides with two different amino acids at <span class="gene">P1</span> (Arg/Lys) and 15 amino acid replacements in positions P1', P2', P2, and P3. The binding energy (deltaG(s), calculated from Km values) and the activation energy (deltaG(T)(double dagger), determined from kcat/Km values) were calculated. Dictyoptera, Coleoptera and Diptera trypsins hydrolyze peptides with Arg at <span class="gene">P1</span> at least 3 times more efficiently than peptides with Lys at P1, whereas Lepidoptera trypsins have no preference between Arg and Lys at that position. The hydrophobicities of each subsite were calculated from the efficiency of hydrolysis of the different amino acid replacements at that subsite. The results suggested that insect trypsin subsites become progressively more hydrophobic along evolution. Apparently, this is an adaptation to resist plant <span class="trait">protein</span> inhibitors, which usually have polar residues at their reactive sites. Results also suggested that, at least in lepidopteran trypsins, S3, S2, S1', and S2' significantly bind the substrate ground state, whereas in the transition state only S1' and S2' do that, supporting aspects of the presently accepted mechanism of trypsin catalysis. Homology modeling showed differences among those trypsins that may account for the varied kinetic properties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Application of machine learning in SNP discovery</h2>
			<h3>PMID:16398931</h3>
				

				
					<h3>PMCID:1955739</h3>
					<h4>期刊:BMC Bioinformatics</h4>
				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Matukumalli </surname>  <given-names> Lakshmi K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> lmatukum@gmu.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Grefenstette </surname>  <given-names> John J </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jgrefens@gmu.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Hyten </surname>  <given-names> David L </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> hytend@ba.ars.usda.gov </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Choi </surname>  <given-names> Ik-Young </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> choii@ba.ars.usda.gov </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> creganp@ba.ars.usda.gov </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Van Tassell </surname>  <given-names> Curtis P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> curtvt@anri.barc.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 1 </month>  <year> 2006 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Single nucleotide polymorphisms (SNP) constitute more than 90% of the genetic variation, and hence can account for most trait differences among individuals in a given species. Polymorphism detection software PolyBayes and PolyPhred give <span class="trait">high</span> false positive SNP predictions even with stringent parameter values. We developed a machine learning (ML) method to augment PolyBayes to improve its prediction accuracy. ML methods have also been successfully applied to other bioinformatics problems in predicting genes, promoters, transcription factor binding sites and <span class="trait">protein</span> structures. RESULTS: The ML program C4.5 was applied to a set of features in order to build a SNP classifier from training data based on human expert decisions (True/False). The training data were 27,275 candidate SNP generated by sequencing 1973 <span class="gene">STS</span> (sequence tag sites) (12 Mb) in both directions from 6 diverse homozygous soybean cultivars and PolyBayes analysis. Test data of 18,390 candidate SNP were generated similarly from 1359 additional <span class="gene">STS</span> (8 Mb). SNP from both sets were classified by experts. After training the ML classifier, it agreed with the experts on 97.3% of test data compared with 7.8% agreement between PolyBayes and experts. The PolyBayes positive predictive values (PPV) (i.e., fraction of candidate SNP being real) were 7.8% for all predictions and 16.7% for those with 100% posterior probability of being real. Using ML improved the PPV to 84.8%, a 5- to 10-fold increase. While both ML and PolyBayes produced a similar number of true positives, the ML program generated only 249 false positives as compared to 16,955 for PolyBayes. The complexity of the soybean genome may have contributed to <span class="trait">high</span> false SNP predictions by PolyBayes and hence results may differ for other genomes. CONCLUSION: A machine learning (ML) method was developed as a supplementary feature to the polymorphism detection software for improving prediction accuracies. The results from this study indicate that a trained ML classifier can significantly reduce human intervention and in this case achieved a 5-10 fold enhanced productivity. The optimized feature set and ML framework can also be applied to all polymorphism discovery software. ML support software is written in Perl and can be easily integrated into an existing SNP discovery pipeline.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ML program C4.5 was applied to a set of features in order to build a SNP classifier from training data based on human expert decisions (True/False). The training data were 27,275 candidate SNP generated by sequencing 1973 <span class="gene">STS</span> (sequence tag sites) (12 Mb) in both directions from 6 diverse homozygous soybean cultivars and PolyBayes analysis. Test data of 18,390 candidate SNP were generated similarly from 1359 additional <span class="gene">STS</span> (8 Mb). SNP from both sets were classified by experts. After training the ML classifier, it agreed with the experts on 97.3% of test data compared with 7.8% agreement between PolyBayes and experts. The PolyBayes positive predictive values (PPV) (i.e., fraction of candidate SNP being real) were 7.8% for all predictions and 16.7% for those with 100% posterior probability of being real. Using ML improved the PPV to 84.8%, a 5- to 10-fold increase. While both ML and PolyBayes produced a similar number of true positives, the ML program generated only 249 false positives as compared to 16,955 for PolyBayes. The complexity of the soybean genome may have contributed to <span class="trait">high</span> false SNP predictions by PolyBayes and hence results may differ for other genomes.
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous report [ <xref ref-type="bibr" rid="B12"> 12 </xref> ], our group observed that in 500&#x02013;700 bp soybean <span class="gene">STS</span> containing two or more SNP a <span class="trait">high</span> level of linkage disequilibrium was present. Batley et al [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B19"> 19 </xref> ] observed that co-segregation of the SNP pattern between multiple SNP loci in an alignment as one of the important factors in SNP discovery. Because of the resulting haplotype structure, the SNP allele present at any one position in a fragment amplified from a particular genotype is highly predictive of other alleles in that fragment. Thus if variations are observed in the same sequence at different polymorphic sites and these polymorphisms correspond to one of the two or three haplotypes present in the <span class="gene">STS</span> then the polymorphisms detected in these positions are more likely to be true (Fig. <xref ref-type="fig" rid="F3"> 3 </xref> ). To capture this concept a feature called haplotype factor (feature # 12) was defined and the algorithm for the calculation is given in Table <xref ref-type="table" rid="T4"> 4 </xref> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Effects of soy <span class="trait">protein</span> and soy phytochemicals on mammary tumor development in female transgenic mice overexpressing human pituitary growth hormone.</h2>
			<h3>PMID:16379573  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hickey J(1), Bartke A, Winters T, Henry N, Banz W.
				<br/> 
				年份：2005 Winter
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to determine the effect of soy-based diets on mammary tumors in female cancer- prone mice. Transgenic virgin female mice expressing human pituitary growth hormone and their respective phenotypically normal littermates were fed a diet containing either casein (C), low-isoflavone soy <span class="trait">protein</span> (LIS), or high-isoflavone soy <span class="trait">protein</span> (HIS). Indices of tumor development were measured throughout the study. Both days from birth until death and days on diet until death were increased [by 20% (P = .01) and 19% (P = .02), respectively] in the LIS group when compared with the C group. Both intervals were increased also (by 16% and 17%, respectively; P &lt; .05) in the <span class="gene">HIS</span> group when compared with the C group. Days from birth to first tumor were increased by 7% (P &lt; .05), as was days on diet to first tumor by 5% (P &lt; .05), in the LIS group when compared with the C group. First-onset number of tumors was decreased (P = .02) by 41% and 34% in the LIS and C groups, respectively, when compared with the <span class="gene">HIS</span> group. Final onset of tumors was decreased (P &lt; .05) by 44% and 9% in the LIS and <span class="gene">HIS</span> groups, respectively, when compared with the C group. Total area of final tumors was decreased (P &lt; .05) by 30% in the LIS group when compared with the C group. Taken cumulatively, these data suggest that a diet rich in soy <span class="trait">protein</span> may provide protective benefits regarding tumor development in female cancer-prone mice. Furthermore, some bioactive compounds in the <span class="gene">HIS</span> diet appeared to confound the soy protein-induced beneficial effects.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of the solubility and emulsifying property of soybean proglycinin and rapeseed procruciferin in relation to structure modified by <span class="trait">protein</span> engineering.</h2>
			<h3>PMID:16248579  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tandang MR(1), Atsuta N, Maruyama N, Adachi M, Utsumi S.
				<br/> 
				年份：2005 Nov 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The presence or absence of a highly negatively charged extension region in beta-conglycinin (J. Agric. Food Chem. 1999, 47, 5278) and the length of a highly negatively charged variable region IV in glycinin (J. Agric. Food Chem. 2004, 52, 8197) are important determinants of solubility and emulsifying property. To examine the effects of the variable region IV from proglycinin A1aB1b and <span class="gene">A3B4</span> and of the extension region from beta-conglycinin <span class="gene">alpha'</span> (alpha'ext) on solubility and emulsifying properties in detail, several mutants of proglycinin, procruciferin, and beta-conglycinin were designed and prepared in Escherichia coli. Nine out of 10 mutants were expressed at <span class="trait">high</span> levels in E. coli and shown to be homotrimer similar to the wild types as assessed by gel filtration. The position of the introduced negatively charged region as well as the amino acid composition were demonstrated to affect solubility at mu = 0.08. All of the proglycinin, procruciferin, and beta-conglycinin mutants with the alpha'ext in the C-terminus, especially the proglycinin mutant, exhibited excellent emulsifying ability and emulsion stability. These indicate that improvement of emulsifying properties by insertion of the alpha'ext in the C-terminus may be generally applicable to seed globulins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Oleate desaturase enzymes of soybean: evidence of regulation through differential stability and phosphorylation.</h2>
			<h3>PMID:16236153  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tang GQ(1), Novitzky WP, Carol Griffin H, Huber SC, Dewey RE.
				<br/> 
				年份：2005 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The endoplasmic reticulum-associated oleate desaturase <span class="gene">FAD2</span> (1-acyl-2-oleoyl-sn-glycero-3-phosphocholine Delta12-desaturase) is the key enzyme responsible for the production of linoleic acid in non-photosynthetic tissues of plants. Little is known, however, concerning the post-transcriptional mechanisms that regulate the activity of this important enzyme. The soybean genome possesses two seed-specific isoforms of FAD2, designated <span class="gene">FAD2-1A</span> and FAD2-1B, which differ at only 24 amino acid residues. Expression studies in yeast revealed that the <span class="gene">FAD2-1A</span> isoform is more unstable than FAD2-1B, particularly when cultures were maintained at elevated growth temperatures. Analysis of chimeric <span class="gene">FAD2-1</span> constructs led to the identification of two domains that appear to be important in mediating the temperature-dependent instability of the <span class="gene">FAD2-1A</span> isoform. The enhanced degradation of <span class="gene">FAD2-1A</span> at <span class="trait">high</span> growth temperatures was partially abrogated by treating the cultures with the 26S proteasome-specific inhibitor MG132, and by expressing the <span class="gene">FAD2-1A</span> cDNA in yeast strains devoid of certain ubiquitin-conjugating activities, suggesting a role for ubiquitination and the 26S proteasome in <span class="trait">protein</span> turnover. In addition, phosphorylation state-specific antipeptide antibodies demonstrated that the Serine-185 of <span class="gene">FAD2-1</span> sequences is phosphorylated during soybean seed development. Expression studies of phosphopeptide mimic mutations in yeast suggest that phosphorylation may downregulate enzyme activity. Collectively, the results show that post-translational regulatory mechanisms are likely to play an important role in modulating <span class="gene">FAD2-1</span> enzyme activities.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of Bean pod mottle virus-based vectors for stable <span class="trait">protein</span> expression and sequence-specific virus-induced gene silencing in soybean.</h2>
			<h3>PMID:16226780  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Ghabrial SA.
				<br/> 
				年份：2006 Jan 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant virus-based vectors provide valuable tools for expression of foreign proteins in plants and for gene function studies. None of the presently available virus vectors is suitable for use in soybean. In the present study, we produced Bean pod mottle virus (BPMV)-based vectors that are appropriate for gene expression and virus-induced gene silencing (VIGS) in soybean. The genes of interest were inserted into the RNA2-encoded <span class="gene">polyprotein</span> open reading frame between the movement <span class="trait">protein</span> (MP) and the large coat <span class="trait">protein</span> (L-CP) coding regions. Additional proteinase cleavage sites were created to flank the foreign <span class="trait">protein</span> by duplicating the MP/L-CP cleavage site. To minimize the chances of homologous recombination and thus insert instability, we took advantage of the genetic code degeneracy and altered the nucleotide sequence of the duplicated regions without affecting amino acid sequences. The recombinant BPMV constructs were stable following several serial passages in soybean and relatively <span class="trait">high</span> levels of <span class="trait">protein</span> expression were attained. Successful expression of several proteins with different biological activities was demonstrated from the BPMV vector. These included the reporter proteins GFP and DsRed, phosphinothricin acetyltransferase (encoded by the herbicide <span class="trait">resistance</span> <span class="gene">bar</span> gene), and the RNA silencing suppressors encoded by Tomato bushy stunt virus, Turnip crinkle virus, Tobacco etch virus, and Soybean mosaic virus. The possible use of BPMV as a VIGS vector to study gene function in soybean was also demonstrated with the phytoene desaturase gene. Our results suggest that the BPMV-based vectors are suitable for expression of foreign proteins in soybean and for functional genomics applications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure and expression analysis of early auxin-responsive Aux/IAA gene family in rice (Oryza sativa).</h2>
			<h3>PMID:16200395  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jain M(1), Kaur N, Garg R, Thakur JK, Tyagi AK, Khurana JP.
				<br/> 
				年份：2006 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Auxin exerts pleiotropic effects on plant growth and development by regulating the expression of early auxin-responsive genes of auxin/indoleacetic acid (Aux/IAA), small auxin-up RNA, and <span class="gene">GH3</span> classes. These genes have been studied extensively in dicots like soybean and Arabidopsis. We had earlier characterized a cDNA of the first monocot member of Aux/IAA family from rice. The achievement of the large scale rice genome sequencing combined with the availability of full-length cDNA sequences from Knowledge-based Oryza Molecular Biological Encyclopedia provided us the opportunity to draw up the first comprehensive list of Aux/IAA genes in a monocot. By screening the available databases, we have identified 31 Aux/IAA genes having <span class="trait">high</span> sequence identity within the conserved domains I, II, III, and IV. The genomic organization as well as chromosomal location of all the Oryza sativa indoleacetic acid (OsIAA) genes is reported. The rice Aux/IAA proteins can be classified in two groups (A and B) on the basis of their phylogenetic relationship with Arabidopsis Aux/IAA proteins. An evolutionary pattern of the rice Aux/IAA genes has been discussed by analyzing their structure (exon/intron organization) and duplications. Interestingly, the duplication of rice Aux/IAA genes was found to be associated with chromosomal block duplication events in rice. The in-silico analysis has been complemented with real-time polymerase chain reaction analysis to quantify transcript levels of all Aux/IAA family members. OsIAA genes showed differential and overlapping organ-specific expression patterns in light- and dark-grown seedlings/plants. Although auxin enhanced the transcript abundance of most of the OsIAA genes, the effect was more pronounced on OsIAA9, 14, 19, 20, 24, and 31. These results provide a foundation for future studies on elucidating the precise role of rice Aux/IAA genes in early steps of auxin signal transduction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A simple and efficient system for green note compound biogenesis by use of certain lipoxygenase and hydroperoxide lyase sources.</h2>
			<h3>PMID:16104814  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fukushige H(1), Hildebrand DF.
				<br/> 
				年份：2005 Aug 24
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Six-carbon (C(6)) aldehydes and alcohols are important components of the aroma and flavor of fruits and vegetables. Soybean lipoxygenase (LOX) isozyme <span class="gene">LOX</span> 3 was reported not only to produce less 13-hydroperoxides, precursors of C(6) aldehydes, but also to convert them to ketodiene products. Here, we examined the effects of <span class="gene">LOX</span> 3 on hexenal formation from linolenic acid homogenized with watermelon 13-hydroperoxide lyase (HL)-overexpressing Nicotiana tabacum leaves and soybean acetone powder. Compared to the wild type, which contains LOXs 1, 2, and 3, the elimination of <span class="gene">LOX</span> 3 in <span class="gene">LOX</span> 1 + 2 facilitates greater production of hexenals. The use of <span class="gene">LOX</span> 2 alone yielded the highest hexenal production, while a two-step conversion was required for <span class="gene">LOX</span> 1 to produce hexenals at <span class="trait">high</span> levels due to different pH optima of the enzymes involved. These results clearly demonstrate that the soybeans lacking <span class="gene">LOX</span> 3 in combination with watermelon HL-overexpressing leaf tissues greatly enhance hexenal formation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-throughput double quantitative competitive polymerase chain reaction for determination of genetically modified organisms.</h2>
			<h3>PMID:16053289  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mavropoulou AK(1), Koraki T, Ioannou PC, Christopoulos TK.
				<br/> 
				年份：2005 Aug 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Quantitative competitive polymerase chain reaction (PCR), especially the double competitive PCR methods (DC-PCR), have evolved as reliable approaches to quantification of genetically modified organisms <span class="gene">(GMO)</span> in food. However, DC-PCR is a low-throughput method because it requires titration of each sample with various amounts of a competitive internal standard, a protocol that involves several PCRs per sample followed by electrophoresis and densitometry. To address this drawback, we have developed a new method for <span class="gene">GMO</span> quantification, namely, a high-throughput double quantitative competitive PCR (HT-DCPCR). In HT-DCPCR, electrophoresis and densitometry are replaced by a rapid, microtiter well-based bioluminometric hybridization assay and there is no need for titration of each sample. The determination of GM soya was chosen as a model. We have constructed internal standards (DNA competitors) both for the 35S promoter sequence and for a plant-specific reference gene (lectin). The competitors have identical size and share the same primer binding sites with the target sequences but differ in a 24-bp internal segment. Each target sequence (35S and lectin) is coamplified with a constant amount (1000 copies) of the respective competitor. The four amplified fragments are hybridized with specific probes and captured on a universal solid phase to achieve simplicity and <span class="trait">high</span> throughput. The hybrids are determined by using streptavidin conjugated to the photoprotein aequorin. The ratio of the luminescence values obtained for the target and the competitor is linearly related to the starting amount of target DNA. The limit of quantification for the 35S promoter is 24 copies. The proposed method was evaluated by determining the <span class="gene">GMO</span> content of soybean powder certified reference materials. Also HT-DCPCR was compared to real-time PCR in a variety of real samples.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification and identification of linoleic acid hydroperoxides generated by soybean seed lipoxygenases 2 and 3.</h2>
			<h3>PMID:15998134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fukushige H(1), Wang C, Simpson TD, Gardner HW, Hildebrand DF.
				<br/> 
				年份：2005 Jul 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been known that lipoxygenase (LOX) isozymes exhibit differences in product formation, but most product information to date is for <span class="gene">LOX</span> 1 among soybean (Glycine max) <span class="gene">LOX</span> isozymes. In this study, LOXs 2 and 3 were purified and used to generate hydroperoxide (HPOD) products in an in vitro system using linoleic acid as a substrate in the presence of either air or O2. The products were analyzed to determine their stereoisomeric characteristics. The control (no enzyme) showed only low levels of hydroperoxide production and no stereoisomeric specificity. <span class="gene">LOX</span> 2 shows <span class="trait">high</span> specificity in product formation, producing roughly 4 times more 13-HPOD than 9-HPOD, nearly all of which was 13-S(Z,E)-HPOD. <span class="gene">LOX</span> 3 produced more 9-HPOD than 13-HPOD at approximately a 2:1 ratio. No single stereoisomer was predominant among <span class="gene">LOX</span> 3 products. These results demonstrate that different isozymes of <span class="gene">LOX</span> have characteristic product profiles in in vitro reactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy <span class="trait">protein</span> influences the development of the metabolic syndrome in male obese ZDFxSHHF rats.</h2>
			<h3>PMID:15971156  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Davis J(1), Iqbal MJ, Steinle J, Oitker J, Higginbotham DA, Peterson RG, Banz WJ.
				<br/> 
				年份：2005 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous investigations have demonstrated a marked effect of soy <span class="trait">protein</span> on the metabolic syndrome (MS). The purpose of this preliminary study was to identify the effects of soy-based diets on male obese ZDFxSHHF (fa/ fa-cp/?) rats. Animals were randomly assigned to one of four diets: control, casein (C); low-isoflavone (LIS) soy <span class="trait">protein;</span> high-isoflavone (HIS) soy <span class="trait">protein;</span> or casein + rosiglitazone (CR). Physiological, biochemical, and molecular parameters were determined at sacrifice. Body weight (p &lt; 0.01) and food intake (p &lt; 0.05) were lower in LIS-fed rodents. Rosiglitazone-treated animals had higher body weight and adiposity (p &lt; 0.05). LIS and CR groups exhibited better glycemic control (p &lt; 0.05), but with a limited effect in rosiglitazone-treated animals. <span class="gene">HIS</span> fed rats had higher glucose and triacylglyceride levels (p &lt; 0.01), and lower plasma insulin (p &lt; 0.01). Renal function parameters with the exception of an increase in systolic blood pressure (p &lt; 0.05) were all suppressed in the LIS group (p &lt; 0.01). The CR group had twofold PPARalpha and PPARgamma mRNA abundance (p &lt; 0.01). LIS-fed animals also exhibited greater abundance of PPARgamma mRNA (p &lt; 0.001), and nearly threefold FAS and CPT-1 mRNA levels (p &lt; 0.05). HIS-fed rats also had higher abundance of CPT-1 mRNA, as well as a lower abundance of ACC mRNA (p &lt; 0.05). Soy-based diets, influenced by isoflavone content and distinct from rosiglitazone, improved several metabolic parameters in obese ZDFxSHHF rats.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Can organic and transgenic soy be used as a substitute for animal <span class="trait">protein</span> by rats?</h2>
			<h3>PMID:15962184  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Soares LL(1), Lucas AM, Boaventura GT.
				<br/> 
				年份：2005 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We evaluated the <span class="trait">protein</span> quality of organic and transgenic soy fed to rats throughout life. Thirty female Wistar rats were divided into three groups (N = 10): organic soy group (OSG) receiving organic soy-based diet, genetically modified soy group <span class="gene">(GMSG)</span> receiving transgenic soy-based diet, and a control group (CG) receiving casein-based diet. All animals received water and isocaloric diet (10% <span class="trait">protein),</span> ad libitum for 291 days. After this, the weight of <span class="gene">GMSG</span> animals (290.9 +/- 9.1 g) was significantly lower (P &lt;= 0.04) than CG (323.2 +/- 7.9 g). The weight of OSG (302.2 +/- 8.7 g) was between that of the <span class="gene">GMSG</span> and the CG. <span class="trait">Protein</span> intake was similar for OSG (308.4 +/- 6.8 g) and <span class="gene">GMSG</span> (301.5 +/- 2.5 g), and significantly lower (P &lt;= 0.0005) than the CG (358.4 +/- 8.1 g). Growth rate was similar for all groups: OSG (0.80 +/- 0.02 g), <span class="gene">GMSG</span> (0.81 +/- 0.03 g) and CG (0.75 +/- 0.02 g). In addition to providing a good <span class="trait">protein</span> intake and inducing less weight gain, both types of soy were utilized in a manner similar to that of casein, suggesting that the <span class="trait">protein</span> quality of soy is similar to that of the standard <span class="trait">protein</span> casein. The groups fed soy-based diet gained less weight, which may be considered to be beneficial for health. We conclude that organic and transgenic soy can be fed throughout life to rats in place of animal <span class="trait">protein,</span> because contain <span class="trait">high</span> quality <span class="trait">protein</span> and do not cause a marked increase in body weight.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Kimchi and soybean pastes are risk factors of gastric cancer.</h2>
			<h3>PMID:15929164  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nan HM(1), Park JW, Song YJ, Yun HY, Park JS, Hyun T, Youn SJ, Kim YD, Kang JW, Kim H.
				<br/> 
				年份：2005 Jun 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						AIM: This case-control study investigated the effects of kimchi, soybean paste, fresh vegetables, nonfermented alliums, nonfermented seafood, nonfermented soybean foods, and the genetic polymorphisms of some metabolic enzymes on the risk of gastric cancer in Koreans. METHODS: We studied 421 gastric cancer patients and 632 age- and sex-matched controls. Subjects completed a structured questionnaire regarding their food intake pattern. Polymorphisms of cytochrome <span class="gene">P450</span> 1A1 (CYP1A1), cytochrome <span class="gene">P450</span> 2E1 (CYP2E1), glutathione S-transferase mu 1 (GSTM1), glutathione S-transferase theta 1 (GSTT1) and aldehyde dehydrogenase 2 (ALDH2) were investigated. RESULTS: A decreased risk of gastric cancer was noted among people with <span class="trait">high</span> consumption of nonfermented alliums and nonfermented seafood. On the other hand, consumption of kimchi, and soybean pastes was associated with increased risk of gastric cancer. Individuals with the CYP1A1 Ile/Val or Val/Val genotype showed a significantly increased risk for gastric cancer. Increased intake of kimchi or soybean pastes was a significant risk factor for the CYP1A1 Ile/Ile, the CYP2E1 c1/c1, the GSTM1 non-null, the GSTT1 non-null, or the ALDH2 *1/*1 genotype. In addition, eating soybean pastes was associated with the increased risk of gastric cancer in individuals with the GSTM1 null type. Nonfermented alliums were significant in individuals with the CYP1A1 Ile/Ile, the CYP2E1 c1/c2 or c2/c2, the GSTT1 null, the GSTT1 non-null, or the ALDH2 *1/*2 or *2/*2 genotype, nonfermented seafood was those with the CYP1A1 Ile/Ile, the CYP2E1 c1/c1, the ALDH2 *1/*1 genotype or any type of GSTM1 or GSTT1. In homogeneity tests, the odds ratios of eating kimchi for gastric cancer according to the GSTM1 or GSTT1 genotype were not homogeneous. CONCLUSION: Kimchi, soybean pastes, and the CYP1A1 Ile/Val or Val/Val are risk factors, and nonfermented seafood and alliums are protective factors against gastric cancer in Koreans. Salt or some chemicals contained in kimchi and soybean pastes, which are increased by fermentation, would play important roles in the carcinogenesis of stomach cancer. Polymorphisms of the CYP1A1, CYP2E1, GSTM1, GSTT1, and ALDH2 genes could modify the effects of some environmental factors on the risk of gastric cancer.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Laboratory and field evaluations of transgenic soybean exhibiting high-dose expression of a synthetic Bacillus thuringiensis cry1A gene for control of Lepidoptera.</h2>
			<h3>PMID:15889751  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：MacRae TC(1), Baur ME, Boethel DJ, Fitzpatrick BJ, Gao AG, Gamundi JC, Harrison LA, Kabuye VT, McPherson RM, Miklos JA, Paradise MS, Toedebusch AS, Viegas A.
				<br/> 
				年份：2005 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transgenic lines of soybean, Glycine max (L.) Merrill, expressing a synthetic cry1A gene (tic107) from Bacillus thuringiensis (Bt), were evaluated in screenhouse and conventional field trials for efficacy against lepidopteran pests. In screenhouse trials, Bt soybean and negative checks (isogenic segregants and parental lines) were evaluated against Anticarsia gemmatalis Hübner and Pseudoplusia includens (Walker) in the United States and against A. gemmatalis, Epinotia aporema (Walsingham), Rachiplusia nu (Guenée), and Spilosoma virginica (F.) in Argentina. Bt soybean exhibited virtually complete efficacy against each of these pests, whereas negative checks suffered significant damage. Bt soybean and negative checks also were evaluated in conventional trials against native populations of A. gemmatalis and P. includens in the southeastern United States. Each of these insects caused significant damage to negative checks in one or more locations, whereas Bt soybean exhibited virtually complete efficacy against these pests. In the laboratory, lyophilized leaf tissues from Bt soybean incorporated in artificial diet at a concentration representing a 25-fold dilution of fresh tissue caused complete mortality of A. gemmatalis and near complete mortality of P. includens neonates after 11 d, whereas mortality on negative checks did not exceed 10% for either insect. Average TIC107 expression approached or exceeded 50 microg/g fresh weight at <span class="gene">V3</span> stage of growth and 200 microg/g by R6 stage of growth. These results demonstrate that expression of TIC107 in soybean can not only achieve highly efficacious control of several lepidopterans under field conditions but also provide a <span class="trait">high</span> dose for effective insect <span class="trait">resistance</span> management.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Novel reference gene, High-mobility-group <span class="trait">protein</span> I/Y, used in qualitative and real-time quantitative polymerase chain reaction detection of transgenic rapeseed cultivars.</h2>
			<h3>PMID:15859086  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Weng H(1), Yang L, Liu Z, Ding J, Pan A, Zhang D.
				<br/> 
				年份：2005 Mar-Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						With the development of transgenic crops, regulations to label the genetically modified organisms <span class="gene">(GMOs)</span> and their derived products have been issued in many countries. Polymerase chain reaction (PCR) methods are thought to be reliable and useful techniques for qualitative and quantitative detection of <span class="gene">GMOs.</span> These methods are generally needed to amplify the transgene and compare the amplified results with that of a corresponding reference gene to get the reliable results. Specific primers were developed for the rapeseed (Brassica napus), high-mobility-group <span class="trait">protein</span> I/Y(HMG-I/Y) single-copy gene and PCR cycling conditions suitable for the use of this sequence as an endogenous reference gene in both qualitative and quantitative PCR assays. Both methods were assayed with 15 different rapeseed varieties, and identical amplified products were obtained with all of them. No amplification was observed when templates were the DNA samples from the other species of Brassica genus or other species, such as broccoli, stem mustard, cauliflower, Chinese cabbage, cabbage, sprouts, Arabidopsis thaliana, carrot, tobacco, soybean, mung bean, tomato, pepper, eggplant, plum, wheat, maize, barley, rice, lupine, and sunflower. This system was specific for rapeseed. Limits of detection and quantitation in qualitative and quantitative PCR systems were about 13 pg DNA (about 10 haploid genomes) and about 1.3 pg DNA (about 1 haploid genome), respectively. To further test the feasibility of this HMG-I/Y gene as an endogenous reference gene, samples containing transgenic rapeseed GT73 with the inserted glyphosate oxidoreductase (GOX) gene were quantitated. These demonstrated that the endogenous PCR detection systems were applicable to the qualitative and quantitative detection of transgenic rapeseed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene cloning and characterization of a soybean (Glycine max L.) LEA <span class="trait">protein,</span> <span class="gene">GmPM16.</span></h2>
			<h3>PMID:15803408  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shih MD(1), Lin SC, Hsieh JS, Tsou CH, Chow TY, Lin TP, Hsing YI.
				<br/> 
				年份：2004 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Late embryogenesis abundant (LEA) proteins, present in abundance in seeds during the late stages of development, are associated with desiccation tolerance. In the present work, we characterize a soybean LEA <span class="trait">protein,</span> <span class="gene">GmPM16,</span> with low molecular weight, <span class="trait">high</span> pI value, and an unusual amino acid residue distribution along the <span class="trait">protein.</span> The transcripts were detected in cotyledon mesophyll cells but not in the vascular system of mature or pod-dried soybean seeds. Circular dichroism (CD) analysis and Fourier transfer infrared (FTIR) spectroscopy indicated that the <span class="gene">GmPM16</span> <span class="trait">protein</span> in solution was highly unordered, possessing only partial alpha-helical structures. However, the <span class="trait">protein</span> in sodium dodecyl sulfate (SDS) or trifluoroethanol (TFE) solution or in a dry state exhibited a conformation of abundant alpha-helical structures. As well, the <span class="gene">GmPM16</span> <span class="trait">protein</span> interacts with sugar and forms tightly glassy matrixes in the dry state. The <span class="trait">protein</span> may play a role in reducing cellular damage in drying seeds by changing the <span class="trait">protein</span> conformation and forming tight cellular glasses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Partial reconstruction of flavonoid and isoflavonoid biosynthesis in yeast using soybean type I and type II chalcone isomerases.</h2>
			<h3>PMID:15778463  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ralston L(1), Subramanian S, Matsuno M, Yu O.
				<br/> 
				年份：2005 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flavonoids and isoflavonoids are major plant secondary metabolites that mediate diverse biological functions and exert significant ecological impacts. These compounds play important roles in many essential physiological processes. In addition, flavonoids and isoflavonoids have direct but complex effects on human health, ranging from reducing cholesterol levels and preventing certain cancers to improving women's health. In this study, we cloned and functionally characterized five soybean (Glycine max) chalcone isomerases (CHIs), key enzymes in the phenylpropanoid pathway that produces flavonoids and isoflavonoids. Gene expression and kinetics analysis suggest that the soybean type I CHI, which uses naringenin chalcone as substrate, is coordinately regulated with other flavonoid-specific genes, while the type II CHIs, which use a variety of chalcone substrates, are coordinately regulated with an isoflavonoid-specific gene and specifically activated by nodulation signals. Furthermore, we found that some of the newly identified soybean CHIs do not require the 4'-hydroxy moiety on the substrate for <span class="trait">high</span> enzyme activity. We then engineered yeast (Saccharomyces cerevisiae) to produce flavonoid and isoflavonoid compounds. When one of the type II CHIs was coexpressed with an isoflavone synthase, the enzyme catalyzing the first committed step of isoflavonoid biosynthesis, various chalcone substrates added to the culture media were converted to an assortment of isoflavanones and isoflavones. We also reconstructed the flavonoid pathway by coexpressing <span class="gene">CHI</span> with either flavanone 3beta-hydroxylase or flavone synthase II. The in vivo reconstruction of the flavonoid and isoflavonoid pathways in yeast provides a unique platform to study enzyme interactions and metabolic flux.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Senescence-associated vacuoles with intense proteolytic activity develop in leaves of Arabidopsis and soybean.</h2>
			<h3>PMID:15743448  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Otegui MS(1), Noh YS, Martínez DE, Vila Petroff MG, Staehelin LA, Amasino RM, Guiamet JJ.
				<br/> 
				年份：2005 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Vacuolar compartments associated with leaf senescence and the subcellular localization of the senescence-specific cysteine-protease <span class="gene">SAG12</span> (senescence-associated gene 12) were studied using specific fluorescent markers, the expression of reporter genes, and the analysis of high-pressure frozen/freeze-substituted samples. Senescence-associated vacuoles (SAVs) with intense proteolytic activity develop in the peripheral cytoplasm of mesophyll and guard cells in Arabidopsis and soybean. The vacuolar identity of these compartments was confirmed by immunolabeling with specific antibody markers. SAVs and the central vacuole differ in their acidity and tonoplast composition: SAVs are more acidic than the central vacuole and, whereas the tonoplast of central vacuoles is highly enriched in gamma-TIP (tonoplast intrinsic <span class="trait">protein),</span> the tonoplast of SAVs lacks this aquaporin. The expression of a SAG12-GFP fusion <span class="trait">protein</span> in transgenic Arabidopsis plants shows that <span class="gene">SAG12</span> localizes to SAVs. The analysis of Pro(SAG12):GUS transgenic plants indicates that <span class="gene">SAG12</span> expression in senescing leaves is restricted to SAV-containing cells, for example, mesophyll and guard cells. A homozygous sag12 Arabidopsis mutant develops SAVs and does not show any visually detectable phenotypical alteration during senescence, indicating that <span class="gene">SAG12</span> is not required either for SAV formation or for progression of visual symptoms of senescence. The presence of two types of vacuoles in senescing leaves could provide different lytic compartments for the dismantling of specific cellular components. The possible origin and functions of SAVs during leaf senescence are discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inactivation of soybean trypsin inhibitors and lipoxygenase by high-pressure processing.</h2>
			<h3>PMID:15713024  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：van der Ven C(1), Matser AM, van den Berg RW.
				<br/> 
				年份：2005 Feb 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Trypsin inhibitors (TIA), one of the antinutritional factors of soy milk, are usually inactivated by heat treatment. In the current study, high-pressure processing (HPP) was evaluated as an alternative for the inactivation of TIA in soy milk. Moreover, the effect of HPP on lipoxygenase (LOX) in whole soybeans and soy milk was studied. For complete <span class="gene">LOX</span> inactivation either very <span class="trait">high</span> pressures (800 MPa) or a combined temperature/pressure treatment (60 degrees C/600 MPa) was needed. Pressure inactivation of TIA was possible only in combination with elevated temperatures. For TIA inactivation, three process parameters, temperature, time, and pressure, were optimized using experimental design and response surface methodology. A 90% TIA inactivation with treatment times of &lt;2 min can be reached at temperatures between 77 and 90 degrees C and pressures between 750 and 525 MPa.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene expression and adiposity are modified by soy <span class="trait">protein</span> in male Zucker diabetic fatty rats.</h2>
			<h3>PMID:15687389  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Banz WJ(1), Davis J, Peterson R, Iqbal MJ.
				<br/> 
				年份：2004 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has earlier been demonstrated that soy <span class="trait">protein</span> diets ameliorate the diabetic phenotype in obese Zucker rats. In this study, we further investigated physiological changes related to adiposity in male Zucker diabetic fatty rats consuming soy-based diets and compared these diets with the insulin-sensitizing drug, rosiglitazone. Transcript abundance of known genes was assessed in the livers to identify potential molecular connections between soy diets and adiposity. Male Zucker diabetic fatty rats were assigned to casein (C) <span class="trait">protein,</span> low-isoflavone soy (LIS) <span class="trait">protein,</span> high-isoflavone soy (HIS) <span class="trait">protein,</span> or C + rosiglitazone (CR) diets. Compared with the C diet, the LIS diet decreased plasma lipids and increased body weight, but did not change liver weight or carcass adiposity. <span class="gene">HIS</span> decreased plasma lipids, liver weight, and body weight. CR decreased plasma lipids and increased carcass adiposity and body weight with no effect on liver weight. In LIS livers, 15 genes involved in signaling and lipid metabolism were up-regulated 2-fold or higher. In <span class="gene">HIS</span> livers, seven genes had a 2-fold or higher change in abundance. However, in CR livers, none of the genes was significantly changed compared with the C diet. There appears to be a distinct change in gene expression associated with soy diets as compared with C-based diets and rosiglitazone treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Solution structure and backbone dynamics of the N-terminal region of the calcium regulatory domain from soybean calcium-dependent <span class="trait">protein</span> kinase alpha.</h2>
			<h3>PMID:15568805  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Weljie AM(1), Gagné SM, Vogel HJ.
				<br/> 
				年份：2004 Dec 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ca(2+)-dependent <span class="trait">protein</span> kinases (CDPKs) are vital Ca(2+)-signaling proteins in plants and protists which have both a kinase domain and a self-contained calcium regulatory calmodulin-like domain (CLD). Despite being very similar to <span class="gene">CaM</span> (&gt;40% identity) and sharing the same fold, recent biochemical and structural evidence suggests that the behavior of CLD is distinct from its namesake, calmodulin. In this study, NMR spectroscopy is employed to examine the structure and backbone dynamics of a 168 amino acid Ca(2+)-saturated construct of the CLD (NtH-CLD) in which almost the entire C-terminal domain is exchange broadened and not visible in the NMR spectra. Structural characterization of the N-terminal domain indicates that the first Ca(2+)-binding loop is significantly more open than in a recently reported structure of the CLD complexed with a putative intramolecular binding region (JD) in the CDPK. Backbone dynamics suggest that parts of the third helix exhibit unusually <span class="trait">high</span> mobility, and significant exchange, consistent with previous findings that this helix interacts with the C-terminal domain. Dynamics data also show that the "tether" region, consisting of the first 11 amino acids of CLD, is highly mobile and these residues exhibit distinctive beta-type secondary structure, which may help to position the JD and CLD. Finally, the unusual global dynamic behavior of the <span class="trait">protein</span> is rationalized on the basis of possible interdomain rearrangements and the highly variable environments of the C- and N-terminal domains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Selection on Glycine beta-1,3-endoglucanase genes differentially inhibited by a Phytophthora glucanase inhibitor <span class="trait">protein.</span></h2>
			<h3>PMID:15545660  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bishop JG(1), Ripoll DR, Bashir S, Damasceno CM, Seeds JD, Rose JK.
				<br/> 
				年份：2005 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant endo-beta-1,3-glucanases (EGases) degrade the cell wall polysaccharides of attacking pathogens and release elicitors of additional plant defenses. Isozymes EGaseA and <span class="gene">EGaseB</span> of soybean differ in susceptibility to a glucanase inhibitor <span class="trait">protein</span> (GIP1) produced by Phytophthora sojae, a major soybean pathogen. EGaseA, the major elicitor-releasing isozyme, is a high-affinity ligand for GIP1, which completely inhibits it, whereas <span class="gene">EGaseB</span> is unaffected by GIP1. We tested for departures from neutral evolution on the basis of partial sequences of EGaseA and <span class="gene">EGaseB</span> from 20 widespread accessions of Glycine soja (the wild progenitor of soybean), from 4 other Glycine species, and across dicotyledonous plants. G. soja exhibited little intraspecific variation at either locus. Phylogeny-based codon evolution models detected strong evidence of positive selection on Glycine EGaseA and weaker evidence for selection on dicot EGases and Glycine EGaseB. Positively selected peptide sites were identified and located on a structural model of EGase bound to GIP1. Positively selected sites and highly variable sites were found disproportionately within 4.5 angstroms of bound GIP1. Low variation within G. soja EGases, coupled with positive selection in both Glycine and dicot lineages and the proximity of rapidly evolving sites to GIP1, suggests an arms race involving repeated adaptation to pathogen attack and inhibition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of <span class="trait">protein</span> engineering of canola procruciferin on its physicochemical and functional properties.</h2>
			<h3>PMID:15506820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tandang MR(1), Adachi M, Inui N, Maruyama N, Utsumi S.
				<br/> 
				年份：2004 Nov 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The primary structure of Brassica napus procruciferin 2/3a was engineered to elucidate structure-function relationships and to improve the functionality of cruciferin. The following mutants were constructed: (1) C287T, (2) DeltaII, variable region II was deleted; (3) C287T/DeltaII, mutation involving (1) and (2); (4) DeltaIV + A1aIV; and (5) DeltaIV + A3IV, variable region IV was replaced with variable region IV containing many charged residues from soybean glycinin A1aB1b and <span class="gene">A3B4</span> subunits. Differential scanning calorimetry analysis revealed that the A1aIV region has a more favorable interaction with the procruciferin molecule than does A3IV as well as the original regions. On the basis of heat-induced precipitation analysis, it was concluded that replacement of the free cysteine residue with threonine (C287T) and insertion of charged regions (DeltaIV + A1aIV and DeltaIV + A3IV) could lead procruciferin to form soluble aggregates after heating. Low solubility was observed in mutants DeltaIV + A3IV, DeltaII, and C287T/DeltaII, especially between pH 4 and 6 at mu = 0.08, but not in DeltaIV + A1aIV, indicating that the number of acidic amino acid residues and the <span class="trait">high</span> number of glutamine residues are important factors for solubility at mu = 0.08. None of the mutants showed any improvements in emulsifying ability, indicating that destabilization and addition of the hydrophilic region are not effective for emulsification. The insertion of the A1aIV region in procruciferin made the molecule more susceptible to alpha-chymotrypsin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of genetically modified organisms by electrochemiluminescence PCR method.</h2>
			<h3>PMID:15494222  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu J(1), Xing D, Shen X, Zhu D.
				<br/> 
				年份：2004 Oct 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						With the development of biotechnology, more and more genetically modified organisms <span class="gene">(GMOs)</span> have entered commercial market. Because of the safety concerns, detection and characterization of <span class="gene">GMOs</span> have attracted much attention recently. In this study, electrochemiluminescence polymerase chain reaction (ECL-PCR) combined with hybridization technique was applied to detect the <span class="gene">GMOs</span> in genetically modified (GM) soybeans and papayas for the first time. Whether the soybeans and the papayas contain GM components was discriminated by detecting the Cauliflower mosaic virus 35S (CaMV35S) promoter. The experiment results show that the detection limit for CaMV35S promoter is 100 fmol, and the GM components can be clearly identified in GM soybeans and papayas. The technique may provide a new means in <span class="gene">GMOs</span> detection due to its simplicity and <span class="trait">high</span> efficiency.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-pressure refolding of bikunin: efficacy and thermodynamics.</h2>
			<h3>PMID:15388859  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seefeldt MB(1), Ouyang J, Froland WA, Carpenter JF, Randolph TW.
				<br/> 
				年份：2004 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bikunin is a glycosylated <span class="trait">protein</span> that aggregates extensively during mammalian cell culture, resulting in loss of activity, loss of native secondary structure, and the formation of nonnative disulfide bonds. We investigated the use of <span class="trait">high</span> hydrostatic pressure (1000-3000 bar) for the refolding of bikunin aggregates. The refolding <span class="trait">yield</span> obtained with pressure-modulated refolding at 2000 <span class="gene">bar</span> was 70 (+/-5%) by reverse-phase chromatography (RP-HPLC), significantly higher than the value of 55 (+/-6%) (RP-HPLC) obtained with traditional guanidine HCl "dilution-refolding." In addition, we determined the thermodynamics of pressure-modulated refolding. The change in volume for the transition of aggregate to monomer DeltaV(refolding) was calculated to be -28 (+/-5) mL/mole. Refolding was accompanied by a loss of hydrophobic exposure, resulting in a positive contribution to the DeltaV(refolding). These findings suggest that the disruption of electro-static interactions or the differences in size of solvent-free cavities between the aggregate and the monomer are the prevailing contributions to the negative DeltaV(refolding).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The nucleotide sequence of Watermelon mosaic virus (WMV, Potyvirus) reveals interspecific recombination between two related potyviruses in the 5' part of the genome.</h2>
			<h3>PMID:15290385  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Desbiez C(1), Lecoq H.
				<br/> 
				年份：2004 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Watermelon mosaic virus (WMV, Potyvirus) is a potyvirus with a worldwide distribution, mostly in temperate and mediterranean regions. According to the partial sequences that were available, WMV appeared to share <span class="trait">high</span> sequence similarity with Soybean mosaic virus (SMV), and it was almost considered as a strain of SMV in spite of its different and much broader host range. Like SMV, it was also related to legume-infecting potyviruses belonging to the " Bean common mosaic virus (BCMV) subgroup". In this paper we obtained the full-length sequence of WMV, and we confirmed that this virus is very closely related to SMV in most of its genome; however, there is evidence for an interspecific recombination in the <span class="gene">P1</span> <span class="trait">protein,</span> as the <span class="gene">P1</span> of WMV was 135 amino-acids longer than that of SMV, and the N-terminal half of the <span class="gene">P1</span> showed no relation to SMV but was 85% identical to BCMV. This suggests that WMV has emerged through an ancestral recombination event, and supports the distinction of WMV and SMV as separate taxonomic units.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of continuous phase <span class="trait">protein</span> on the oxidative stability of fish oil-in-water emulsions.</h2>
			<h3>PMID:15237967  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Faraji H(1), McClements DJ, Decker EA.
				<br/> 
				年份：2004 Jul 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Whey <span class="trait">protein</span> isolate (WPI), soy <span class="trait">protein</span> isolate (SPI), and sodium caseinate (CAS) can inhibit lipid oxidation when they produce a positive charge at the interface of emulsion droplets. However, when proteins are used to stabilize oil-in-water emulsions, only a fraction of them actually absorb to the emulsion droplets, with the rest remaining in the continuous phase. The impact of these continuous phase proteins on the oxidative stability of protein-stabilized emulsions is not well understood. WPI-stabilized menhaden oil-in-water emulsions were prepared by high-pressure homogenization. In some experiments WPI was removed from the continuous phase of the emulsions through repeated centrifugation and resuspension of the emulsion droplets (washed emulsion). Unwashed emulsions were more oxidatively stable than washed emulsions at pH 7.0, suggesting that continuous phase proteins were antioxidative. The oxidative stability of emulsions containing different kinds of <span class="trait">protein</span> in the continuous phase decreased in the order SPI &gt; <span class="gene">CAS</span> &gt; WPI, as determined by both hydroperoxide and headspace propanal formation. Iron-binding studies showed that the chelating ability of the proteins decreased in the order <span class="gene">CAS</span> &gt; SPI &gt; WPI. The free sulfhydryls of both WPI and SPI were involved in their antioxidant activity. This research shows that continuous phase proteins could be an effective means of protecting omega-3 fatty acids from oxidative deterioration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Large-scale overproduction, functional purification and ligand affinities of the His-tagged human histamine <span class="gene">H1</span> receptor.</h2>
			<h3>PMID:15206929  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ratnala VR(1), Swarts HG, VanOostrum J, Leurs R, DeGroot HJ, Bakker RA, DeGrip WJ.
				<br/> 
				年份：2004 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This report describes an efficient strategy for amplified functional purification of the human <span class="gene">H1</span> receptor after heterologous expression in Sf9 cells. The cDNA encoding a C-terminally histidine-tagged (10xHis) human histamine <span class="gene">H1</span> receptor was used to generate recombinant baculovirus in a Spodoptera frugiperda-derived cell line (IPLB-Sf9). As judged from its ligand affinity profile, functional receptor could be expressed at <span class="trait">high</span> levels (30-40 pmol per 10(6) cells). Rapid proteolysis in the cell culture led to limited fragmentation, without loss of ligand binding, but could be efficiently suppressed by including the protease inhibitor leupeptin during cell culture and all subsequent manipulations. Effective solubilization of functional receptor with optimal recovery and stability required the use of dodecylmaltoside as a detergent in the presence of a <span class="trait">high</span> concentration of NaCl and of a suitable inverse agonist. Efficient purification of solubilized receptor could be achieved by affinity chromatography over nickel(II) nitrilotriacetic acid resin. Functional membrane reconstitution of purified <span class="gene">H1</span> receptor was accomplished in mixed soybean lipids (asolectin). The final proteoliposomic <span class="gene">H1</span> receptor preparation has a purity greater than 90% on a <span class="trait">protein</span> basis and displays a ligand binding affinity profile very similar to the untagged receptor expressed in COS-7 cells. In conclusion, we are able to produce pharmacologically viable <span class="gene">H1</span> receptor in a stable membrane environment allowing economic large-batch operation. This opens the way to detailed studies of structure-function relationships of this medically and biologically important receptor <span class="trait">protein</span> by 3D-crystallography, FT-IR spectroscopy and solid-state NMR spectroscopy.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The galactolipid digalactosyldiacylglycerol accumulates in the peribacteroid membrane of nitrogen-fixing nodules of soybean and Lotus.</h2>
			<h3>PMID:15159398  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gaude N(1), Tippmann H, Flemetakis E, Katinakis P, Udvardi M, Dörmann P.
				<br/> 
				年份：2004 Aug 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The peribacteroid membrane (PBM) surrounding nitrogen fixing rhizobia in the nodules of legumes is crucial for the exchange of ammonium and nutrients between the bacteria and the host cell. Digalactosyldiacylglycerol (DGDG), a galactolipid abundant in chloroplasts, was detected in the PBM of soybean (Glycine max) and Lotus japonicus. Analyses of membrane marker proteins and of fatty acid composition confirmed that DGDG represents an authentic PBM lipid of plant origin and is not derived from the bacteria or from plastid contamination. In Arabidopsis, DGDG is known to accumulate in extraplastidic membranes during phosphate deprivation. However, the presence of DGDG in soybean PBM was not restricted to phosphate limiting conditions. Complementary DNA sequences corresponding to the two DGDG synthases, <span class="gene">DGD1</span> and <span class="gene">DGD2</span> from Arabidopsis, were isolated from soybean and Lotus. The two genes were expressed during later stages of nodule development in infected cells and in cortical tissue. Because nodule development depends on the presence of <span class="trait">high</span> amounts of phosphate in the growth medium, the accumulation of the non-phosphorus galactolipid DGDG in the PBM might be important to save phosphate for other essential processes, i.e. nucleic acid synthesis in bacteroids and host cells.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Unexpected structure of the Ca2+-regulatory region from soybean calcium-dependent <span class="trait">protein</span> kinase-alpha.</h2>
			<h3>PMID:15155727  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Weljie AM(1), Vogel HJ.
				<br/> 
				年份：2004 Aug 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calcium-dependent <span class="trait">protein</span> kinases (CDPKs) are an extensive class of multidomain Ca(2+)-regulated enzymes from plants and protozoa. In vivo the so-called calmodulin-like domain (CLD) of <span class="gene">CDPK</span> binds intramolecularly to the junction domain (JD), which exhibits both kinase-inhibitory and CLD binding properties. Here we report the <span class="trait">high</span> resolution solution structure of the calcium-regulatory region from soybean CDPK-alpha determined in the presence of a peptide encompassing the JD. The structure of both lobes of CLD resembles that of related helix-loop-helix Ca(2+)-binding proteins. NMR chemical shift mapping studies demonstrate that the JD induces significant structural changes in isolated Ca(2+)-CLD, particularly the C-terminal domain, although a stable complex is not formed. A CLD solution structure calculated on the basis of NMR data and long range fluorescence resonance energy transfer distances reveals an activated state with both lobes positioned side by side, similar to calcineurin B rather than calmodulin, highlighting the possible pitfall of assigning function purely from sequence information.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Covalent structure of soybean seed coat peroxidase.</h2>
			<h3>PMID:15063322  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Welinder KG(1), Larsen YB.
				<br/> 
				年份：2004 Apr 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peroxidase from soybean seed coat (SBP) is very stable at <span class="trait">high</span> temperature, extremes of pH, and in organic solvent. At the same time, it is highly reactive towards both organic and inorganic substrates, similar to horseradish peroxidase. <span class="gene">SBP</span> has a wide range of potential applications, and its structure is of particular interest for engineering purposes and as a model for stable heme peroxidases. The covalent structure of <span class="gene">SBP</span> has been determined by Edman sequencing and MALDI-TOF MS. <span class="gene">SBP</span> is a highly heterogeneous glycoprotein with <span class="gene">MS</span> determined masses from 39 to 41 kDa. The mature <span class="trait">protein</span> consists of 306 residues starting with pyrrolidone carboxylic acid. Seven glycosylation sites have been observed, although some sites were only partially glycosylated. No putative plant peroxidases were orthologous to SBP. However, <span class="gene">SBP</span> showed greater than 70% amino acid sequence identity to peroxidases from other legumes recruited in various defense responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[A study on urine microglobulin change after interventional therapy of renal artery in patients with coronary heart disease].</h2>
			<h3>PMID:14990016  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ni J(1), Shen WF, Zhang RY, Hu J, Zhang Q, Xie JY.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: To evaluate the urine microglobulin change after interventional therapy in patients with renal artery stenosis. METHODS: According to the results of renal artery angiography, 69 consecutive patients with coronary heart disease were divided into 2 groups, including 44 patients with severe renal artery stenosis RAS (luminal narrowing &gt; 70%, group I) and 25 patients with atherosclerotic lesion in renal artery (luminal narrowing &gt; 50% but &lt; 70%, group II). The urine alpha(1), beta(2)-microglobulin (alpha(1), beta(2)-MG) were measured respectively. The successful procedural rates, rates of complication, serum creatitine and the urine alpha(1), beta(2)-MG 3 months after procedure were also recorded. The acute results and long-term follow-up outcomes were compared between the 2 groups. RESULTS: Urine alpha(1)-MG [(5.2 +/- 2.5) microg/L vs (3.0 +/- 2.7) microg/L, P &gt; 0.001] and beta(2)-MG [(377 +/- 173) microg/L vs (202 +/- 184) microg/L, P &gt; 0.001] were significantly higher in group I than in group II. However, the urine alpha(1)-MG of the 2 groups did not exceed the normal range (&lt;6 microg/L). The urine beta(2)-MG [(237 +/- 187) microg/L vs (377 +/- 173) microg/L, P &gt; 0.01] 3 months after stenting procedure in group I were significantly decreased. There were no significance change in urine alpha(1)-MG in group I and both the microglobulins in group II during follow-up. There was significant difference in improvement of blood pressure between the 2 groups (62.5% vs 9.1%, P &gt; 0.01). As compared with group II, group I patients had more re-admission (22.5% vs 9.1%), renal failure (5% vs 0) and higher mortality (5% vs 0). However, these measurements did not differ significantly. Multivariate analysis revealed that persistent elevation of urine beta(2)-MG was an independent predictor of severe events (including re-admission and renal failure) after renal artery stenting (OR = 3.01, 95% <span class="gene">CI</span> 1.01 - 8.95, P = 0.036). CONCLUSIONS: Severe RAS causes damage of tubular reabsorption. Renal artery stenting improves tubular function, but in patients with persistent elevation of beta(2)-MG, the rates of re-admission and renal failure remain <span class="trait">high.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative expression analysis of GH3, a gene induced by plant growth regulator herbicides in soybean.</h2>
			<h3>PMID:14759135  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kelley KB(1), Lambert KN, Hager AG, Riechers DE.
				<br/> 
				年份：2004 Feb 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Symptoms resembling off-target plant growth regulator (PGR) herbicide injury are frequently found in soybean fields, but the causal agent is often difficult to identify. The expression of GH3, an auxin-regulated soybean gene, was quantified from soybean leaves injured by PGR herbicides using real-time RT-PCR. Expression of <span class="gene">GH3</span> was analyzed to ascertain its suitability for use in a diagnostic assay to determine whether PGR herbicides are the cause of injury. <span class="gene">GH3</span> was highly induced by dicamba within 3 days after treatment (DAT) and remained <span class="trait">high</span> at 7 DAT, but induction was much lower at 17 DAT. <span class="gene">GH3</span> was also highly induced at 7 DAT by dicamba + diflufenzopyr, and to a lesser extent by the other PGR herbicides clopyralid and 2,4-D. The non-PGR herbicides glyphosate, imazethapyr, and fomesafen did not significantly induce <span class="gene">GH3</span> expression above a low constitutive level. These results indicate that a diagnostic assay for PGR herbicide injury based on overexpression of auxin-responsive genes is feasible, and that <span class="gene">GH3</span> is a potential candidate from which a diagnostic assay could be developed. However, time course analysis of <span class="gene">GH3</span> expression indicates the assay would be effective for a limited time after exposure to the herbicide.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure of the core region of the soybean beta-conglycinin <span class="gene">alpha'</span> subunit.</h2>
			<h3>PMID:14747705  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Maruyama Y(1), Maruyama N, Mikami B, Utsumi S.
				<br/> 
				年份：2004 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The crystal structure of the core region of the <span class="gene">alpha'</span> subunit (alpha(c')) of soybean beta-conglycinin has been determined at 2.3 A resolution. alpha(c') was superimposed on the known crystal structure of the beta-conglycinin beta subunit with a small root-mean square deviation of 0.77 A, which is consistent with the <span class="trait">high</span> sequence identity of 75.5% between alpha(c') and the beta subunit. It is known that the thermal stability of the beta subunit is higher than that of the <span class="gene">alpha'</span> subunit and that their thermal stabilities are conferred by highly homologous core regions. Comparisons of the three-dimensional structures and primary sequences between alpha(c') and the beta subunit suggest that five factors account for this difference between subunits as regards the difference in thermal stability: (i) the total cavity volume is larger in alpha(c'), (ii) the cluster of charged residues at the intermonomer interface is smaller in alpha(c') and alpha(c') lacks the intermonomer salt bridge of the beta subunit, (iii) the solvent-accessible surface is more hydrophobic in alpha(c'), (iv) there are fewer proline residues in alpha(c') and (v) a loop region between helix 3 and strand J' in alpha(c') is more flexible owing to the insertion of five additional residues. Although more hydrogen bonds were found in alpha(c'), this difference should be more than compensated for by the combined contributions of these other factors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Introduction of DPR, an enterostatin fragment peptide, into soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis.</h2>
			<h3>PMID:14745196  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Takenaka Y(1), Doyama N, Maruyama N, Utsumi S, Yoshikawa M.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DPR, a fragment peptide of enterostatin (VPDPR) having hypocholesterolemic activity, was introduced into the three homologous sites, EPR, DYR, and DPI, in the soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis. The modified beta-conglycinin was expressed in Escherichia coli and recovered in the soluble fraction. After purification on ion-exchange HPLC, the modified beta-conglycinin was digested by trypsin to release integrated DPR. The <span class="trait">yield</span> of DPR from 1 mole of the modified beta-conglycinin was 1.2 mole.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of transgenic and endogenous plant DNA in rumen fluid, duodenal digesta, milk, blood, and feces of lactating dairy cows.</h2>
			<h3>PMID:14740846  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Phipps RH(1), Deaville ER, Maddison BC.
				<br/> 
				年份：2003 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objective was to determine the presence or absence of transgenic and endogenous plant DNA in ruminal fluid, duodenal digesta, milk, blood, and feces, and if found, to determine fragment size. Six multiparous lactating Holstein cows fitted with ruminal and duodenal cannulas received a total mixed ration. There were two treatments (T). In T1, the concentrate contained genetically modified (GM) soybean meal (cp4epsps gene) and GM corn grain (cry1a[b] gene), whereas T2 contained the near isogenic non-GM counterparts. Polymerase chain reaction analysis was used to determine the presence or absence of DNA sequences. Primers were selected to amplify small fragments from single-copy genes (soy lectin and corn high-mobility <span class="trait">protein</span> and <span class="gene">cp4epsps</span> and cry1a[b] genes from the GM crops) and multicopy genes (bovine mitochondrial cytochrome b and rubisco). Single-copy genes were only detected in the solid phase of rumen and duodenal digesta. In contrast, fragments of the rubisco gene were detected in the majority of samples analyzed in both the liquid and solid phases of ruminal and duodenal digesta, milk, and feces, but rarely in blood. The size of the rubisco gene fragments detected decreased from 1176 bp in ruminal and duodenal digesta to 351 bp in fecal samples.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic diversity of bradyrhizobial populations from diverse geographic origins that nodulate Lupinus spp. and Ornithopus spp.</h2>
			<h3>PMID:14666990  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jarabo-Lorenzo A(1), Pérez-Galdona R, Donate-Correa J, Rivas R, Velázquez E, Hernández M, Temprano F, Martínez-Molina E, Ruiz-Argüeso T, León-Barrios M.
				<br/> 
				年份：2003 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genetic diversity of 45 bradyrhizobial isolates that nodulate several Lupinus and Ornithopus species in different geographic locations was investigated by 16S rDNA PCR-RFLP and sequence analysis, 16S-23S rDNA intergenic spacer (IGS) PCR-RFLP analysis, and ERIC-PCR genomic fingerprinting. Reference strains of Bradyrhizobium japonicum, B. liaoningense and B. elkanii and some Canarian isolates from endemic woody legumes in the tribe Genisteae were also included. The 16S rDNA-RFLP analysis resolved 9 genotypes of lupin isolates, a group of fourteen isolates presented restriction-genotypes identical or very similar to B. japonicum, while another two main groups of isolates (69%) presented genotypes that clearly separated them from the reference species of soybean. 16S rDNA sequencing of representative strains largely agreed with restriction analysis, except for a group of six isolates, and showed that all the lupin isolates are relatives of B. japonicum, but different lineages were observed. The 16S-23S IGS-RFLP analysis showed a <span class="trait">high</span> resolution level, resolving 19 distinct genotypes among 30 strains analysed, and so demonstrating the heterogeneity of the 16S-RFLP groups. ERIC-PCR fingerprint analysis showed an enormous genetic diversity producing a different pattern for each but two of the isolates. Phylogeny of <span class="gene">nodC</span> gene was independent from the 16S rRNA phylogeny, and showed a tight relationship in the symbiotic region of the lupin isolates with isolates from Canarian genistoid woody legumes, and in concordance, cross-nodulation was found. We conclude that Lupinus is a promiscuous host legume that is nodulated by rhizobia with very different chromosomal genotypes, which could even belong to several species of Bradyrhizobium. No correlation among genomic background, original host plant and geographic location was found, so, different chromosomal genotypes could be detected at a single site and in a same plant species, on the contrary, an identical genotype was detected in very different geographical locations and plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Inhibitive effect of soybean isoflavone on prostate hyperplasia in rats].</h2>
			<h3>PMID:14653113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ren GF(1), Huang YM.
				<br/> 
				年份：2003 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: To explore the inhibitive effect of soybean isoflavone on benign prostatic hyperplasia in rats. METHODS: By subcutaneously injecting testosterone propionate to induce prostate hyperplasia in rats, the changes of prostate wet weight, prostatic index, morphological change, prostate-specific acid phosphatase (PAP), acid phosphatase in the control, the model, low, moderate, <span class="trait">high</span> dose of soybean isoflavone groups were observed. RESULTS: The ventral prostate wet weight, prostatic index, and <span class="gene">PAP</span> in the low, moderate, and <span class="trait">high</span> dose groups were significantly lower than those in the models (P &lt; 0.05). The ventral prostate wet weight, prostatic index, and <span class="gene">PAP</span> in the moderate, and <span class="trait">high</span> dose groups were significantly lower than those in the low dose group (P &lt; 0.05). CONCLUSION: Soybean isoflavone inhibits prostate hyperplasia and the increase of acid phosphatase and <span class="gene">PAP</span> in a dose-dependent manner in rats. Soybean isoflavone may serve as supplementary therapy and prevent benign prostatic hyperplasia.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean sucrose binding <span class="trait">protein</span> gene family: genomic organization, gene copy number and tissue-specific expression of the <span class="gene">SBP2</span> promoter.</h2>
			<h3>PMID:14585823  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contim LA(1), Waclawovsky AJ, Delú-Filho N, Pirovani CP, Clarindo WR, Loureiro ME, Carvalho CR, Fontes EP.
				<br/> 
				年份：2003 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The sucrose binding <span class="trait">protein</span> (SBP) from soybean has been implicated as an important component of the sucrose uptake system. Two <span class="gene">SBP</span> genomic clones, gsS641.1 and gsS641.2, which correspond to allelic forms of the <span class="gene">GmSBP2/S64</span> gene, have been isolated and characterized. As a member of the seed storage <span class="trait">protein</span> superfamily, it has been shown that the <span class="gene">SBP</span> gene structure is similar to vicilin genes with intron/exon boundaries at conserved positions. Fluores cence in situ hybridization (FISH) suggested that the soybean <span class="gene">SBP</span> gene family is represented by at least two non-allelic genes corresponding to the previously isolated <span class="gene">GmSBP1</span> and <span class="gene">GmSBP2/S64</span> cDNAs. These two cDNAs share extensive sequence similarity but are located at different loci in the soybean genome. To investigate transcriptional activation of the <span class="gene">GmSBP2</span> gene, 2 kb 5'-flanking sequences of gsS641.1 and gsS641.2 were fused to the beta-glucuronidase (GUS) reporter gene and to the green fluorescent <span class="trait">protein</span> (GFP) reporter gene and inde pendently introduced into Nicotiana tabacum by Agrobacterium tumefaciens-mediated transformation. The <span class="gene">SBP2</span> promoter directed expression of both GUS and GFP reporter genes with <span class="trait">high</span> specificity to the phloem of leaves, stems and roots. Thus, the overall pattern of SBP-GUS or SBP-GFP expression is consistent with the involvement of <span class="gene">SBP</span> in sucrose translocation-dependent physiological processes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An ancient enzyme domain hidden in the putative beta-glucan elicitor receptor of soybean may play an active part in the perception of pathogen-associated molecular patterns during broad host <span class="trait">resistance.</span></h2>
			<h3>PMID:14578352  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fliegmann J(1), Mithofer A, Wanner G, Ebel J.
				<br/> 
				年份：2004 Jan 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A successful defense against potential pathogens requires that a host organism is able to discriminate between self and nonself structures. Soybean (Glycine max L.) exploits a specific molecular pattern, a 1,6-beta-linked and 1,3-beta-branched heptaglucoside (HG), present in cell walls of the oomycetal pathogen Phytophthora sojae, as a signal compound eliciting the onset of defense reactions. The specific and <span class="trait">high</span> affinity HG-binding site is contained in the beta-glucan-binding <span class="trait">protein</span> (GBP), which in turn is part of a proposed receptor complex. The ability to perceive and respond to Phytophthora cell wall-derived beta-glucan elicitors is exclusive to plants that belong to the Fabaceae. However, we propose that the presence of the <span class="gene">GBP</span> is essential, but not sufficient for beta-glucan elicitor-dependent disease <span class="trait">resistance</span> because genes encoding GBP-related proteins can be retrieved from many plant species. Furthermore, we show that the <span class="gene">GBP</span> is composed of two different carbohydrateactive <span class="trait">protein</span> domains, one containing the beta-glucan-binding site, and the other related to glucan endoglucosidases of fungal origin. The glucan hydrolase displays most likely an endo-specific mode of action, cleaving only 1,3-beta-d-glucosidic linkages of oligoglucosides consisting of at least four moieties. Thus, the intrinsic endo-1,3-beta-glucanase activity of the <span class="gene">GBP</span> is perfectly suited during initial contact with Phytophthora to release oligoglucoside fragments enriched in motifs that constitute ligands for the <span class="trait">high</span> affinity binding site present in the same <span class="trait">protein.</span> The concept of innate immunity in plants receives substantial support by this highly sophisticated system using ancient enzyme modules as an active part of the recognition mechanism.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the 3' terminal region of the genome of Beet mosaic virus and its relation with other potyviruses.</h2>
			<h3>PMID:14505096  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Glasa M(1), Kúdela O, Subr Z.
				<br/> 
				年份：2003 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A1633 nucleotides long 3'-terminal sequence of genome of the Beet mosaic virus (BtMV) Ren1 isolate from Slovakia, spanning a part of the <span class="gene">NIb</span> gene, complete capsid <span class="trait">protein</span> (CP) gene and a 3'non-coding region was determined. Computer-assisted translation resulted in a polypeptide encompassing typical conserved potyviral amino acid (aa) motifs. Homology BLAST search and phylogenetic analysis revealed that BtMV was closest to Bean common mosaic virus, Soybean mosaic virus and Zucchini yellow mosaic virus. An unexpectedly <span class="trait">high,</span> 99.6% aa sequence identity was found between BtMV-Ren1 and a previously sequenced British BtMV isolate. Restriction analysis and sequencing of the CP gene of other Slovak BtMV isolates revealed their <span class="trait">high</span> homology. Developed reverse transcription-polymerase chain reaction protocols enabled sensitive, rapid and reliable detection of the virus.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:On-line microdialysis of proteins with high-salt buffers for direct coupling of electrospray ionization mass spectrometry and liquid chromatography.</h2>
			<h3>PMID:12831191  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Canarelli S(1), Fisch I, Freitag R.
				<br/> 
				年份：2002 Mar 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mass spectrometry (MS) is one of the most powerful instrumental techniques for <span class="trait">protein</span> analysis. The electrospray ionization (ESI) approach is known to be very gentle and at the same time compatible with liquid separation techniques such as HPLC and CE. However, ESI is known to be susceptible to salts and impurities, which often cause a dramatic decrease in sensitivity due to the suppression of the ionization of the product of interest. For this reason, LC-ESI-MS coupling has so far been largely limited to reversed-phase chromatography with its hydro-organic mobile phases. Other chromatographic techniques are typically "linked" to ESI-MS by time consuming, off-line desalting steps. On-line microdialysis has been proposed as a solution to this dilemma. In this paper, we introduce an improved microdialysis system, which enlarges the number of putative applications, thus allowing chromatographic separations of biological compounds to be directly coupled to <span class="gene">MS</span> detection with little to no loss in time or chromatographic resolution. Examples include separations by affinity, ion-exchange and size-exclusion chromatography, all of which were connected successfully to the ESI-MS detector via the on-line microdialyzer. We propose that, using this system, any kind of chromatography technique can be coupled to ESI-MS, thus enabling for example application in quality control or process monitoring of many bioproduction and downstream processes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Crystal structure of soybean 11S globulin: glycinin <span class="gene">A3B4</span> homohexamer.</h2>
			<h3>PMID:12771376  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Adachi M(1), Kanamori J, Masuda T, Yagasaki K, Kitamura K, Mikami B, Utsumi S.
				<br/> 
				年份：2003 Jun 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most plant seeds contain 11S globulins as major storage proteins for their nutrition. Soybean glycinin belongs to the 11S globulin family and consists of five kinds of subunits. We determined the crystal structure of a homohexamer of the glycinin <span class="gene">A3B4</span> subunit at 2.1-A resolution. The crystal structure shows that the hexamer has 32-point group symmetry formed by face-to-face stacking of two trimers. The interface buries the highly conserved interchain disulfide. Based on the structure, we propose that an ingenious face-to-face mechanism controls the hexamer formation of the 11S globulin by movement of a mobile disordered region to the side of the trimer after posttranslational processing. Electrostatic analysis of the faces suggests that the interchain disulfide-containing face has <span class="trait">high</span> positive potential at acidic pH, which induces dissociation of the hexamer into trimers that may be susceptible to proteinases after seed imbibition. This dissociation might result in the degradation and mobilization of 11S globulins as storage proteins in embryos during germination and seedling growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulatory regions and nuclear factors involved in nodule-enhanced expression of a soybean phosphoenolpyruvate carboxylase gene: implications for molecular evolution.</h2>
			<h3>PMID:12684874  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nakagawa T(1), Takane K, Sugimoto T, Izui K, Kouchi H, Hata S.
				<br/> 
				年份：2003 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have determined the genomic organization of two closely related phosphoenolpyruvate carboxylase genes in soybean, <span class="gene">GmPEPC7,</span> which is expressed at <span class="trait">high</span> levels in root nodules, and the housekeeping gene <span class="gene">GmPEPC15.</span> Their nucleotide sequences, including most introns and 5;-flanking regions within 600 bp upstream from the transcription start sites, are well conserved, suggesting that they were duplicated quite recently. To gain insights into the process of evolution of the tissue-specifically expressed <span class="gene">GmPEPC7gene,</span> we produced chimeric constructs carrying either the <span class="gene">GmPEPC7or</span> <span class="gene">GmPEPC15promoter</span> fused to the beta-glucuronidase gene. The expression patterns of the reporter observed in nodules that developed on transgenic hairy roots reflected the levels of mRNA levels produced by the genes in wild-type soybean plants, indicating that the <span class="gene">GmPEPC7promoter</span> directs nodule-specific expression. Loss-of-function experiments showed that the segment of <span class="gene">GmPEPC7between</span> -466 and -400, designated as the "switch region" (SR), was necessary for expression in nodules, although proteins that bind to SR were not detectable in a gel-retardation assay. Another gel-retardation assay indicated that putative nodule nuclear proteins bind specifically to the region of <span class="gene">GmPEPC7between</span> -400 and -318, designated as the "amplifier region" (AR). Both SR and AR have characteristic sequences that are not found in the <span class="gene">GmPEPC15promoter.</span> Furthermore, experiments using hybrid promoters derived from <span class="gene">GmPEPC15demonstrated</span> that AR confers high-level expression in nodules only in combination with SR. When wild-type soybean plants were subjected to prolonged darkness and subsequently illuminated, the level of <span class="gene">GmPEPC7mRNA</span> in nodules decreased and then recovered. This study suggests that the acquisition of two interdependent cis-acting elements resulted in molecular evolution of the nodule-enhanced <span class="gene">GmPEPC7gene.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification, properties and substrate specificity of a digestive trypsin from Periplaneta americana (Dictyoptera) adults.</h2>
			<h3>PMID:12650689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lopes AR(1), Terra WR.
				<br/> 
				年份：2003 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A digestive trypsin from the American cockroach (Periplaneta americana, Dictyoptera) males was purified by a combination of anionic chromatographies in low and <span class="trait">high</span> pressure systems. The <span class="trait">yield</span> was 70% with a final specific activity of 2,000 units per mg <span class="trait">protein</span> (substrate: benzoyl-Arg-p-nitroanilide, BRpNA). Chemical modification with TLCK (k(obs)=3.3 M(-1) s(-1); stoichiometry 1:1) and PMSF (k(obs)=0.18 M(-1) s(-1); stoichiometry 1:1) confirmed that this peptidase is a trypsin. This enzyme has a molecular weight of 29 kDa (SDS-PAGE), a pI of 6.0 and a pH optimum of 8.9. Kinetic parameters using different colorimetric, fluorimetric and internally-quenched substrates indicated that P. americana trypsin prefers to hydrolyze synthetic substrates containing more than one amino acid residue and with an arginine residue at <span class="gene">P1</span> position and a hydrophobic residue at P2. This enzyme presented a Km of 120 microM for BRpNA and is competitively inhibited by benzamidine (Ki=0.25 microM). Soybean trypsin inhibitor is a tight-binding inhibitor presenting a K(D) of 0.4 nM. Differences in substrate specificity and in the reactivity of the trypsin active site groups can be related to adaptation of insects to different hosts. P. americana trypsin is an excellent model for comparison as a basal group on evolutionary studies of insect trypsins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of an expansin gene is correlated with root elongation in soybean.</h2>
			<h3>PMID:12644651  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lee DK(1), Ahn JH, Song SK, Choi YD, Lee JS.
				<br/> 
				年份：2003 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Expansin is a family of proteins that catalyze long-term expansion of cell walls and has been considered a principal <span class="trait">protein</span> that affects cell expansion in plants. We have identified the first root-specific expansin gene in soybean (Glycine max), <span class="gene">GmEXP1,</span> which may be responsible for root elongation. Expression levels of <span class="gene">GmEXP1</span> were very <span class="trait">high</span> in the roots of 1- to 5-d-old seedlings, in which rapid root elongation takes place. Furthermore, <span class="gene">GmEXP1</span> mRNA was most abundant in the root tip region, where cell elongation occurs, but scarce in the region of maturation, where cell elongation ceases, implying that its expression is closely related to root development processes. In situ hybridization showed that <span class="gene">GmEXP1</span> transcripts were preferentially present in the epidermal cells and underlying cell layers in the root tip of the primary and secondary roots. Ectopic expression of <span class="gene">GmEXP1</span> accelerated the root growth of transgenic tobacco (Nicotiana tabacum) seedlings, and the roots showed insensitivity to obstacle-touching stress. These results imply that the <span class="gene">GmEXP1</span> gene plays an important role in root development in soybean, especially in the elongation and/or initiation of the primary and secondary roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-to-function relationship of mini-lipoxygenase, a 60-kDa fragment of soybean lipoxygenase-1 with lower stability but higher enzymatic activity.</h2>
			<h3>PMID:12626522  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Di Venere A(1), Salucci ML, van Zadelhoff G, Veldink G, Mei G, Rosato N, Finazzi-Agrò A, Maccarrone M.
				<br/> 
				年份：2003 May 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Lipoxygenase-1 (Lox-1) is a member of the lipoxygenase family, a class of dioxygenases that take part in the metabolism of polyunsatured fatty acids in eukaryotes. Tryptic digestion of soybean Lox-1 is known to produce a 60 kDa fragment, termed "mini-Lox," which shows enhanced catalytic efficiency and higher membrane-binding ability than the native enzyme (Maccarrone, M., Salucci, M. L., van Zadelhoff, G., Malatesta, F., Veldink, G. Vliegenthart, J. F. G., and Finazzi-Agrò, A. (2001) Biochemistry 40, 6819-6827). In this study, we have investigated the stability of mini-Lox in guanidinium hydrochloride and under <span class="trait">high</span> pressure by fluorescence and circular dichroism spectroscopy. Only a partial unfolding could be obtained at <span class="trait">high</span> pressure in the range 1-3000 <span class="gene">bar</span> at variance with guanidinium hydrochloride. However, in both cases a reversible denaturation was observed. The denaturation experiments demonstrate that mini-Lox is a rather unstable molecule, which undergoes a two-step unfolding transition at moderately low guanidinium hydrochloride concentration (0-4.5 m). Both chemical- and physical-induced denaturation suggest that mini-Lox is more hydrated than Lox-1, an observation also confirmed by 1-anilino-8-naphthalenesulfonate (ANS) binding studies. We have also investigated the occurrence of substrate-induced changes in the <span class="trait">protein</span> tertiary structure by dynamic fluorescence techniques. In particular, eicosatetraynoic acid, an irreversible inhibitor of lipoxygenase, has been used to mimic the effect of substrate binding. We demonstrated that mini-Lox is indeed characterized by much larger conformational changes than those occurring in the native Lox-1 upon binding of eicosatetraynoic acid. Finally, by both activity and fluorescence measurements we have found that 1-anilino-8-naphthalenesulfonate has access to the active site of mini-Lox but not to that of intact Lox-1. These findings strongly support the hypothesis that the larger hydration of mini-Lox renders this molecule more flexible, and therefore less stable.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biochemical evidence for a calcium-dependent <span class="trait">protein</span> kinase from Pharbitis nil and its involvement in photoperiodic flower induction.</h2>
			<h3>PMID:12591257  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jaworski K(1), Szmidt-Jaworska A, Tretyn A, Kopcewicz J.
				<br/> 
				年份：2003 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A soluble Ca(2+)-dependent <span class="trait">protein</span> kinase (CDPK) was isolated from seedlings of the short-day plant Pharbitis nil and purified to homogeneity. Activity of Pharbitis nil <span class="gene">CDPK</span> (PnCDPK) was strictly dependent on the presence of Ca(2+) (K(0,5)=4,9 microM). The enzyme was autophosphorylated on serine and threonine residues and phosphorylated a wide diversity of substrates only on serine residues. Histone III-S and syntide-2 were the best phosphate acceptors (K(m) for histone III-S=0,178 mg ml(-1)). Polyclonal antibodies directed to a regulatory region of the soybean <span class="gene">CDPK</span> recognized 54 and 62 kDa polypeptides from Pharbitis nil. However, only 54 kDa <span class="trait">protein</span> was able to catalyse autophosphorylation and phosphorylation of substrates in a Ca(2+)-dependent manner. <span class="gene">CDPK</span> autophosphorylation was <span class="trait">high</span> in 5-day-old Pharbitis nil seedlings grown under non-inductive continuous white light and was reduced to one-half of its original when plants were grown in the long inductive night. Also, the pattern of proteins phosphorylation has changed. After 16-h-long inductive night phosphorylation of endogenous target (specific band of 82 kDa) increased in the presence of calcium ions. It may suggest that Ca(2+)-dependent <span class="trait">protein</span> kinase is involved in this process and it is dependent on light/dark conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A physical map of a gene-dense region in soybean linkage group A2 near the black seed coat and Rhg (4) loci.</h2>
			<h3>PMID:12582695</h3>
				

				
			<p>
				作者：Lewers K(1), Heinz R, Beard H, Marek L, Matthews B.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( Glycine max L. Merrill) linkage group A2 contains a major <span class="trait">resistance</span> gene to the soybean cyst nematode ( Heterodera glycines Ichinohe) at the Rhg (4) locus near a gene encoding aspartokinase homoserine dehydrogenase (AK-HSDH) and also near the I locus affecting seed coat color. To identify clones related to this region of the genome, we used a PCR assay using primers designed from a gene encoding <span class="gene">AK-HSDH</span> to screen approximately 40,000 clones from a bacterial artificial chromosome (BAC) library constructed from genomic DNA of the susceptible cv. Williams 82. The identified BACs were screened with a second PCR assay using primers designed from DNA sequence associated with the I locus to confirm the location of the BACs. Only BAC <span class="gene">Gm_ISb001_056_G02</span> (56G2) was positive for both assays. BAC 56G2 contains several genes previously associated with stress or defense response including genes with <span class="trait">high</span> sequence similarity to those encoding chalcone synthase, glucosyl-transferase, a heat-shock transcription factor, a membrane-associated salt-inducible <span class="trait">protein,</span> adenosyl homocysteinase, a <span class="trait">protein</span> kinase, and a G10-like <span class="trait">protein.</span> The map contributes to our understanding of the organization of the soybean genome and to the completion of a physical map of the soybean genome. In addition, the genes identified provide landmarks to identify BAC clones near the Rhg (4) locus in resistant soybean genomic libraries and provide a foundation for comparison of soybean cyst nematode resistant and -susceptible DNA sequences in this region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary supplements of soya flour lower serum testosterone concentrations and improve markers of oxidative stress in men.</h2>
			<h3>PMID:12548304  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gardner-Thorpe D(1), O'Hagen C, Young I, Lewis SJ.
				<br/> 
				年份：2003 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: We examined the effects on serum sex steroids, lipids and markers of oxidative stress of supplementing the diets of healthy male volunteers with scones made with soya flour. DESIGN: A randomized placebo controlled cross-over trial. SETTING: University Hospital of Wales. SUBJECTS: Twenty volunteers recruited by advertisement. INTERVENTIONS: Male volunteers ate three scones a day in addition to their normal diet for a period of 6 weeks. The scones were made with either wheat or soya flour (containing 120 mg/day of isoflavones). Blood was analysed for sex steroids (testosterone, dihydro-testosterone, oestradiol, oestrone, sex hormone binding globulin, albumin and the concentration of non-protein bound sex steroids were calculated), lipid profile (total cholesterol, <span class="trait">high</span> density lipoprotein cholesterol and triglycerides) and measures of oxidative stress (hydroperoxides, susceptibility of LDL to oxidation with copper and myeloperoxidase). RESULTS: The volunteers' mean age was 35.6 (s.d. 11.2) y. Total serum testosterone fell in volunteers taking the soya scones (19.3-18.2 nmol/l; 95% <span class="gene">CI</span> 1.01, 1.12; P=0.03). No significant changes were seen in the concentrations of the other serum sex steroids, albumin or sex hormone binding globulin throughout the study. Significant improvements in two of the three markers of oxidative stress were seen in volunteers taking soya scones. Lag time for myeloperoxidase rose from 55.0 to 68.0 min (95% <span class="gene">CI</span> -16.0, -3.5; P=0.009) and the presence of hydroperoxides decreased from 2.69 to 2.34 micro mol/l (95% <span class="gene">CI</span> 0.12, 0.71; P=0.009). There were no changes seen in serum triglycerides or cholesterol. CONCLUSIONS: We have shown that soya supplements reduce serum testosterone and improve markers of oxidative stress. These findings provide a putative mechanism by which soya supplements could protect against prostatic disease and atherosclerosis. Further dietary studies with clinical end points are warranted.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the bile acid-binding region in the soy glycinin A1aB1b subunit.</h2>
			<h3>PMID:12506978  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choi SK(1), Adachi M, Utsumi S.
				<br/> 
				年份：2002 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soy glycinin has five major subunits which are classified into two groups according to their homology in amino acid sequences (group I, A1aB1b, A1bB2 and A2B1a; group II, <span class="gene">A3B4</span> and A5A4B3). It has been reported that the peptide fragments derived from the A1a and A2 chains of the A1aB1b and A2B1a subunits had bile acid-binding ability and that the region of 114-161 residues of the A1a chain was responsible for this bile acid-binding ability. In this study, we constructed A1a, A3 and 9 deletion mutants of A1a lacking various numbers of residues at the C-terminus, and evaluated their bile acid-binding ability by a cholic acid-conjugated column and fluorescence analysis. The bile acid-binding ability of A1a was higher than that of A3 and there was a remarkable decrease in the bile acid-binding ability between the delta[138-291] and delta[130-291] mutants. The 130-138 region is rich in hydrophobic residues. In this regard, when we constructed the delta[129-134] mutant lacking six contiguous hydrophobic residues (VAWWMY) and evaluated its bile acid-binding ability, a similar remarkable decrease in the bile acid-binding ability was observed. These results indicate that the 129-134 residue region (VAWWMY) with <span class="trait">high</span> hydrophobicity was important for bile acid-binding of A1a.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Alfalfa mosaic virus: coat protein-dependent initiation of infection.</h2>
			<h3>PMID:20569357</h3>
				

				
			<p>
				作者：Bol JF(1).
				<br/> 
				年份：2003 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SUMMARY Taxonomy: Alfalfa mosaic virus (AMV) is the type species of the genus Alfamovirus and belongs to the family Bromoviridae. In this family, the tripartite RNA genomes of bromo-, cucumo- and probably oleaviruses are infectious as such, whereas infection with the three genomic RNAs of alfamo- and ilarviruses requires addition to the inoculum of a few molecules of coat <span class="trait">protein</span> (CP) per RNA molecule. RNAs 1 and 2 encode the replicase proteins <span class="gene">P1</span> and P2, RNA 3 encodes the movement <span class="trait">protein</span> and CP. CP is translated from the subgenomic RNA 4. Physical properties: RNAs 1 (3.65 kb), 2 (2.6 kb) and 3 (2.2 kb) are separately encapsidated into bacilliform particles which are 19 nm wide and 35-56 nm long. In addition, the virus preparations contain spheroidal particles each containing two copies of RNA 4 (0.88 kb). Virus particles contain 16-17% RNA and are mainly stabilized by protein-RNA interactions. The 3'-termini of the viral RNAs contain a homologous sequence of 145 nucleotides that can adopt two alternative conformations: one represents a high-affinity binding site for CP, the other resembles a tRNA-like structure and is required for minus-strand promoter activity. Hosts: AMV mostly infects herbaceous plants, but several woody species are included in the natural host range. The experimental and natural host ranges include over 600 species in 70 families. At least 15 aphid species are known to transmit the virus in the stylet-borne or non-persistent manner. Economic importance: AMV is a significant pathogen in alfalfa and sweet clover and can spread from these forages to neighbouring crops like pepper, tobacco or soybean. The recent introduction of the soybean aphid (Aphis glycines) in the mid-west states of the USA has increased the incidence of AMV in soybean. AMV occurs world-wide in potato and is referred to as 'calico mosaic' because of its characteristic symptoms on the foliage. However, the economic importance of AMV in potato is limited.USEFUL WEBSITES: &lt;http://www.socgenmicrobiol.org.uk/JGV/080/1089/0801089A.PDF&gt; review paper; &lt;http://www.ncbi.nlm.nih.gov/ICTVdb/ICTVdB/10010001.htm&gt; host range and physical properties; &lt;http://mmtsb.scripps.edu/viper/1amv.html&gt; structural information.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a sucrose transporter isolated from the developing cotyledons of soybean.</h2>
			<h3>PMID:12504891  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Aldape MJ(1), Elmer AM, Chao WS, Grimes HD.
				<br/> 
				年份：2003 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reduced carbon produced in mature leaves is distributed throughout plants in the form of sucrose. Sucrose transporter proteins (SUT) play a crucial role in transporting sucrose. We isolated a cDNA encoding a sucrose transporter, <span class="gene">GmSUT1,</span> which is expressed in the developing cotyledons of soybean (Glycine max). [14C]sucrose uptake assays demonstrate that <span class="gene">GmSUT1</span> has a K(m) of 5.6mM and a V(max) of 5.8 nmol sucrose min(-1)(mg cells)(-1), which are similar to those of the low-affinity-high-capacity sucrose transporter family. <span class="gene">GmSUT1</span> <span class="trait">protein</span> accumulates gradually during cotyledon development, correlating with increasing sucrose levels in the maturing cotyledons. Collectively, these data suggest that <span class="gene">GmSUT1</span> plays an active role in the movement of sucrose into the developing seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bradyoxetin, a unique chemical signal involved in symbiotic gene regulation.</h2>
			<h3>PMID:12393811  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Loh J(1), Carlson RW, York WS, Stacey G.
				<br/> 
				年份：2002 Oct 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bradyrhizobium japonicum is a symbiotic bacterium that nodulates soybean. Critical for the infection and establishment of this symbiosis are the bacterial nodulation genes (nod, nol, noe), which are induced in the presence of plant produced isoflavones. Transcription of the nodulation genes is also controlled in a population density-dependent fashion. Expression of the nod genes is maximal at low population densities, and decreases significantly at higher culture densities. Population density control of the nodulation genes involves NolA and NodD2, both of which function in tandem to repress nod gene expression. An extracellular secreted factor (CDF) is known to mediate this repression. Here, we report that CDF is a novel signaling molecule, designated bradyoxetin, different from other Gram-negative quorum signals. The proposed structure of bradyoxetin is 2-[4-[[4-(3-aminooxetan-2-yl)phenyl](imino)methyl]phenyl]oxetan-3-ylamine. Interestingly, expression of bradyoxetin is iron-regulated, and is maximally produced under iron-starved conditions. Consistent with this, expression of the nodulation genes occurred in an iron-dependent fashion. Addition of iron to B. japonicum cultures at <span class="trait">high</span> optical densities resulted in decreased bradyoxetin production, and a concomitant reduction in <span class="gene">nolA</span> expression. A corresponding increase in nodY-lacZ expression was observed with iron treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of alternative oxidase gene expression in soybean.</h2>
			<h3>PMID:12374304  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Djajanegara I(1), Finnegan PM, Mathieu C, McCabe T, Whelan J, Day DA.
				<br/> 
				年份：2002 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max cv. Stevens) suspension cells were used to investigate the expression of the alternative oxidase (Aox) multigene family. Suspension cells displayed very <span class="trait">high</span> rates of cyanide-insensitive respiration, but <span class="gene">Aox3</span> was the only isoform detected in untreated cells. Incubation with antimycin A, citrate, salicylic acid or at low temperature (10 degrees C) specifically induced the accumulation of the <span class="gene">Aox1</span> isoform. Aox2 was not observed under any conditions in the cells. Increases in <span class="gene">Aox1</span> <span class="trait">protein</span> correlated with increases in <span class="gene">Aox1</span> mRNA. Treatment of soybean cotyledons with norflurazon also induced expression of Aox1. Reactive oxygen species (ROS) were detected upon incubation of cells with antimycin, salicylic acid or at low temperature, but not during incubation with citrate. <span class="gene">Aox1</span> induction by citrate, but not by antimycin, was prevented by including the <span class="trait">protein</span> kinase inhibitor staurosporine in the medium. The results suggest that multiple pathways exist in soybean to regulate expression of Aox genes and that <span class="gene">Aox1</span> specifically is induced by a variety of stress and metabolic conditions via at least two independent signal transduction pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aggregation of peptides during hydrolysis as a cause of reduced enzymatic extractability of soybean meal proteins.</h2>
			<h3>PMID:12137469  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fischer M(1), Gruppen H, Piersma SR, Kofod LV, Schols HA, Voragen AG.
				<br/> 
				年份：2002 Jul 31
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						With the purpose of analyzing the size and composition of enzyme-unextractable proteins in differently heat-treated soybean meals, a selection of extractants was screened for their ability to extract these proteins from enzyme-unextractable residues. The largest effects were obtained with urea, urea plus beta-mercaptoethanol, and dilute alkali; the latter extracted up to 87% of the enzyme-unextractable <span class="trait">protein.</span> Gel permeation chromatography indicated that a large proportion of the extracted material was of <span class="trait">high</span> molecular weight. However, the combined results from gel electrophoresis, LC-MS, and MALDI-ToF <span class="gene">MS</span> showed that the extracted <span class="trait">protein</span> material was composed of aggregated peptides. The largest aggregates were observed in the enzymatic residues originating from meals heat-treated at <span class="trait">high</span> humidity. Extracted aggregates were fully degraded upon subsequent proteolytic treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Thermal and conformational stability of seed coat soybean peroxidase.</h2>
			<h3>PMID:12119017  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kamal JK(1), Behere DV.
				<br/> 
				年份：2002 Jul 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean peroxidase (SBP) obtained from the soybean seed coats belongs to class III of the plant peroxidase superfamily. Detailed circular dichroism and steady state fluorescence studies have been carried out to monitor thermal as well as denaturant-induced unfolding of <span class="gene">SBP</span> and apo-SBP. Melting of secondary and tertiary structures of <span class="gene">SBP</span> occurs with characteristic transition midpoints, T(m), of 86 and 83.5 degrees C, respectively, at neutral pH. Removal of heme resulted in greatly decreased thermal stability of the <span class="trait">protein</span> (T(m) = 38 degrees C). The deltaG degrees (H2O) determined from guanidine hydrochloride-induced denaturation at 25 degrees C and at neutral pH is 43.3 kJ mol(-1) for <span class="gene">SBP</span> and 9.0 kJ mol(-1) for apo-SBP. Comparison with the reported unfolding data of the homologous enzyme, horseradish peroxidase (HRP-C), showed that <span class="gene">SBP</span> exhibits significantly <span class="trait">high</span> thermal and conformational stability. We show that this enhanced structural stability of <span class="gene">SBP</span> relative to HRP-C arises due to the unique nature of their heme binding. A stronger heme-apoprotein affinity probably due to the interaction between Met37 and the C8 heme vinyl substituent contributes to the unusually <span class="trait">high</span> structural stability of SBP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification and characterization of a putative virulence factor, serine protease, from Vibrio parahaemolyticus.</h2>
			<h3>PMID:12007650  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lee CY(1), Cheng MF, Yu MS, Pan MJ.
				<br/> 
				年份：2002 Mar 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A protease (protease A) was successfully purified from the extracellular proteins of Vibrio parahaemolyticus no. 93, a clinical strain carrying neither <span class="gene">tdh</span> nor trh genes, using phenyl-Sepharose CL-4B hydrophobic interaction chromatography. The molecular mass of protease A was 43 kDa using gel filtration, which was in agreement with the results obtained from SDS-PAGE, suggesting that protease A was a monomeric <span class="trait">protein.</span> Additionally, the isoelectric point of this <span class="trait">protein</span> was 5.0. The optimum temperature and pH of protease A ranged from 40 degrees C to 50 degrees C and pH 8, respectively. Protease A activity was inhibited by serine protease inhibitors, such as phenylmethylsulfonyl fluoride and soybean trypsin inhibitor; moreover, the activity could be blocked by treatment with 20 mM of 1,10-phenanthroline, but could not be restored by adding metal ions. These results indicated that protease A is a serine protease that requires metal. The 12 N-terminal residues of protease A showed a <span class="trait">high</span> degree of identity (81%) to the sequence of Vibrio metschnikovii VapT serine protease. The purified protease had significant effects on the growth of Chinese hamster ovary, HeLa, Vero and Caco-2 cells and its cytotoxic activity was not blocked by gangliosides. Protease A lysed erythrocytes well but its hemolytic activity was unstable after heat treatment, indicating that protease A is able to cause hemolysis but is a heat-labile <span class="trait">protein.</span> The purified protease caused tissue hemorrhage and death in mice when injected both intraperitoneally and intravenously. In conclusion, this is the first report of a serine protease purified directly from the supernatant of V. parahaemolyticus and identifying it as a potential virulence factor.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cleavage specificity of the subtilisin-like protease <span class="gene">C1</span> from soybean.</h2>
			<h3>PMID:12007608  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Boyd PM(1), Barnaby N, Tan-Wilson A, Wilson KA.
				<br/> 
				年份：2002 Apr 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cleavage specificity of protease C1, isolated from soybean (Glycine max (L.) Merrill) seedling cotyledons, was examined using oligopeptide substrates in an HPLC based assay. A series of peptides based on the sequence Ac-KVEKEESEEGE-NH2 was used, mimicking a natural cleavage site of protease <span class="gene">C1</span> in the alpha subunit of the storage <span class="trait">protein</span> beta-conglycinin. A study of substrate peptides truncated from either the N- or C-terminus indicates that the minimal requirements for cleavage by protease <span class="gene">C2</span> are three residues N-terminal to the cleaved bond, and two residues C-terminal (i.e. P3-P2'). The maximal rate of cleavage is reached with substrates containing four to five residues N-terminal to the cleaved bond and four residues C-terminal (i.e. P4 or <span class="gene">P5</span> to P4'). The importance of Glu residues at the P1, P1', and P4 positions was examined using a series of substituted nonapeptides (P5-P4') with a base sequence of Ac-KVEKEESEE-NH2. At the <span class="gene">P1</span> position, the relative ranking, based on kcat/Km, was E&gt;Q&gt;K&gt;A&gt;D&gt;F&gt;S. Substitutions at the P1' position <span class="trait">yield</span> the ranking E congruent withQ&gt;A&gt;S&gt;D&gt;K&gt;F, while those at P4' had less effect on kcat/Km, yielding the ranking F congruent with S congruent with E congruent withD&gt;K&gt;A congruent withQ. These data show that protease <span class="gene">C1</span> prefers to cleave at Glu-Glu and Glu-Gln bonds, and that the nature of the P4' position is less important. The fact that there is specificity in the cleavage of the oligopeptides suggests that the more limited specific cleavage of the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin by protease <span class="gene">C1</span> is due to a combination of the sequence cleavage specificity of the protease and the accessibility of appropriate scissile peptide bonds on the surface of the substrate <span class="trait">protein.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy in hypercholesterolaemia: a double-blind, placebo-controlled trial.</h2>
			<h3>PMID:11965512  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Puska P(1), Korpelainen V, Høie LH, Skovlund E, Lahti T, Smerud KT.
				<br/> 
				年份：2002 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: To study whether Abacor, a product based on isolated soy <span class="trait">protein</span> with <span class="trait">high</span> and standardised levels of isoflavones and cotyledon soy fibres, was more effective in lowering total and LDL cholesterol than placebo. DESIGN: Randomised, placebo-controlled, double-blind, parallel group, single centre study. SETTING: Primary care in Joensuu, North Karelia, Finland. SUBJECTS: Subjects were screened from the patient database of the health centre; 30 were randomised to the Abacor group and 30 subjects to placebo. Eight subjects were withdrawn, six from the active group, two from the placebo group. INTERVENTION: The preparations were given as two daily liquid supplements in addition to the subjects' regular diets for 6 weeks. RESULTS: Abacor showed a statistically significant lipid-lowering effect as compared to placebo, although an unexpected reduction was seen in the placebo group. The estimated difference between active treatment and placebo was 0.25 mmol/l (95% <span class="gene">CI</span> 0.01, 0.50; P=0.049) for total cholesterol, corresponding to reductions of 8.3 and 5.1%, respectively. The difference in reduction of LDL-cholesterol was 0.27 mmol/l (95% <span class="gene">CI</span> 0.06, 0.49; P=0.014) and corresponded to a reduction of 13.2% in the active treatment group, and 8.0% in the placebo group. Abacor showed a rapid onset of effect, as compared with placebo. During a wash-out period of 4 weeks after treatment, the subjects returned to pre-treatment cholesterol levels. CONCLUSION: Added to a regular diet, Abacor significantly reduced LDL-cholesterol and total cholesterol. These beneficial effects occurred within 6 weeks of treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Divergent members of a soybean (Glycine max L.) 4-coumarate:coenzyme A ligase gene family.</h2>
			<h3>PMID:11856365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lindermayr C(1), Möllers B, Fliegmann J, Uhlmann A, Lottspeich F, Meimberg H, Ebel J.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						4-Coumarate:CoA ligase (4CL) is involved in the formation of coenzyme A thioesters of hydroxycinnamic acids that are central substrates for subsequent condensation, reduction, and transfer reactions in the biosynthesis of plant phenylpropanoids. Previous studies of <span class="gene">4CL</span> appear to suggest that many isoenzymes are functionally equivalent in supplying substrates to various subsequent branches of phenylpropanoid biosyntheses. In contrast, divergent members of a <span class="gene">4CL</span> gene family were identified in soybean (Glycine max L.). We isolated three structurally and functionally distinct <span class="gene">4CL</span> cDNAs encoding 4CL1, 4CL2, and 4CL3 and the gene <span class="gene">Gm4CL3.</span> A fourth cDNA encoding <span class="gene">4CL4</span> had <span class="trait">high</span> similarity with 4CL3. The recombinant proteins expressed in Escherichia coli possessed highly divergent catalytic efficiency with various hydroxycinnamic acids. Remarkably, one isoenzyme (4CL1) was able to convert sinapate; thus the first cDNA encoding a <span class="gene">4CL</span> that accepts highly substituted cinnamic acids is available for further studies on branches of phenylpropanoid metabolism that probably lead to the precursors of lignin. Surprisingly, the activity levels of the four isoenzymes and steady-state levels of their transcripts were differently affected after elicitor treatment of soybean cell cultures with a beta-glucan elicitor of Phytophthora sojae, revealing the down-regulation of <span class="gene">4CL1</span> vs. up-regulation of 4CL3/4. A similar regulation of the transcript levels of the different <span class="gene">4CL</span> isoforms was observed in soybean seedlings after infection with Phytophthora sojae zoospores. Thus, partitioning of cinnamic acid building units between phenylpropanoid branch pathways in soybean could be regulated at the level of catalytic specificity and the level of expression of the <span class="gene">4CL</span> isoenzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">ENOD40</span> encodes two peptides that bind to sucrose synthase.</h2>
			<h3>PMID:11842184  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rohrig H(1), Schmidt J, Miklashevichs E, Schell J, John M.
				<br/> 
				年份：2002 Feb 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">ENOD40</span> is expressed at an early stage in root nodule organogenesis in legumes. Identification of <span class="gene">ENOD40</span> homologs in nonleguminous plants suggests that this gene may have a more general biological function. In vitro translation of soybean <span class="gene">ENOD40</span> mRNA in wheat germ extracts revealed that the conserved nucleotide sequence at the 5' end (region I) encodes two peptides of 12 and 24 aa residues (peptides A and B). These peptides are synthesized de novo from very short, overlapping ORFs. Appropriate ORFs are present in all legume ENOD40s studied thus far. In this case small peptides are directly translated from polycistronic eukaryotic mRNA. The 24-aa peptide B was detected in nodules by Western blotting. Both peptides specifically bind to the same 93-kDa <span class="trait">protein,</span> which was affinity purified from soybean nodules. Using peptide mass fingerprinting, we identified this binding <span class="trait">protein</span> as nodulin 100, which is a subunit of sucrose synthase. Based on our data we suggest that <span class="gene">ENOD40</span> peptides are involved in the control of sucrose use in nitrogen-fixing nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cytosolic glutamine synthetase in soybean is encoded by a multigene family, and the members are regulated in an organ-specific and developmental manner.</h2>
			<h3>PMID:11788764  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Morey KJ(1), Ortega JL, Sengupta-Gopalan C.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Gln synthetase (GS) is the key enzyme in N metabolism and it catalyzes the synthesis of Gln from glutamic acid, ATP, and NH4+. There are two major isoforms of <span class="gene">GS</span> in plants, a cytosolic form (GS1) and a chloroplastic form (GS2). In leaves, GS2 functions to assimilate ammonia produced by nitrate reduction and photorespiration, and GS1 is the major isoform assimilating NH3 produced by all other metabolic processes, including symbiotic N2 fixation in the nodules. GS1 is encoded by a small multigene family in soybean (Glycine max), and cDNA clones for the different members have been isolated. Based on sequence divergence in the 3'-untranslated region, three distinct classes of GS1 genes have been identified (alpha, beta, and gamma). Genomic Southern analysis and analysis of hybrid-select translation products suggest that each class has two distinct members. The alpha forms are the major isoforms in the cotyledons and young roots. The beta forms, although constitutive in their expression pattern, are ammonia inducible and show <span class="trait">high</span> expression in N2-fixing nodules. The gamma1 gene appears to be more nodule specific, whereas the gamma2 gene member, although nodule enhanced, is also expressed in the cotyledons and flowers. The two members of the alpha and beta class of GS1 genes show subtle differences in the expression pattern. Analysis of the promoter regions of the gamma1 and gamma2 genes show sequence conservation around the TATA box but complete divergence in the rest of the promoter region. We postulate that each member of the three GS1 gene classes may be derived from the two ancestral genomes from which the allotetraploid soybean was derived.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Activation of smooth muscle myosin light chain kinase by calmodulin. Role of LYS(30) and GLY(40).</h2>
			<h3>PMID:11748245  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Van Lierop JE(1), Wilson DP, Davis JP, Tikunova S, Sutherland C, Walsh MP, Johnson JD.
				<br/> 
				年份：2002 Feb 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Calmodulin (CaM)-dependent myosin light chain kinase (MLCK) plays a key role in activation of smooth muscle contraction. A soybean isoform of CaM, <span class="gene">SCaM-4</span> (77% identical to human CaM) fails to activate MLCK, whereas <span class="gene">SCaM-1</span> (90.5% identical to human CaM) is as effective as CaM. We exploited this difference to gain insights into the structural requirements in <span class="gene">CaM</span> for activation of MLCK. A chimera (domain I of <span class="gene">SCaM-4</span> and domains II-IV of SCaM-1) behaved like SCaM4, and analysis of site-specific mutants of <span class="gene">SCaM-1</span> indicated that K30E and G40D mutations were responsible for the reduction in activation of MLCK. Competition experiments showed that <span class="gene">SCaM-4</span> binds to the CaM-binding site of MLCK with <span class="trait">high</span> affinity. Replacement of <span class="gene">CaM</span> in skinned smooth muscle by exogenous <span class="gene">CaM</span> or SCaM-1, but not SCaM-4, restored Ca(2+)-dependent contraction. K30E/M36I/G40D <span class="gene">SCaM-1</span> was a poor activator of contraction, but site-specific mutants, K30E, M36I and G40D, each restored Ca(2+)-induced contraction to CaM-depleted skinned smooth muscle, consistent with their capacity to activate MLCK. Interpretation of these results in light of the high-resolution structures of (Ca(2+))(4)-CaM, free and complexed with the CaM-binding domain of MLCK, indicates that a surface domain containing Lys(30) and Gly(40) and residues from the C-terminal domain is created upon binding to MLCK, formation of which is required for activation of MLCK. Interactions between this activation domain and a region of MLCK distinct from the known CaM-binding domain are required for removal of the autoinhibitory domain from the active site, i.e., activation of MLCK, or this domain may be required to stabilize the conformation of (Ca(2+))(4)-CaM necessary for MLCK activation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of denaturants on the structural properties of soybean lipoxygenase-1.</h2>
			<h3>PMID:11741336  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Malvezzi-Campeggi F(1), Rosato N, Finazzi-Agrò A, Maccarrone M.
				<br/> 
				年份：2001 Dec 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of chemical (urea) and physical (temperature and <span class="trait">high</span> pressure) denaturation on the structural properties of soybean lipoxygenase-1 (LOX1) was analyzed through dynamic fluorescence spectroscopy and circular dichroism. We show that the fluorescence decay of the native <span class="trait">protein</span> could be fitted by two lorentzian distributions of lifetimes, centered at 1 and 4 ns. The analysis of the urea-denatured <span class="trait">protein</span> suggested that the shorter distribution is mostly due to the tryptophan residues located in the N-terminal domain of LOX1. We also show that a pressure of 2400 <span class="gene">bar</span> and a temperature of 55 degrees C brought LOX-1 to a state similar to a recently described stable intermediate "I." Analysis of circular dichroism spectra indicated a substantial decrease of alpha-helix compared with beta-structure under denaturing conditions, suggesting a higher stability of the N-terminal compared with the C-terminal domain in the denaturation process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Independent spontaneous mitochondrial malate dehydrogenase null mutants in soybean are the result of deletions.</h2>
			<h3>PMID:11535646  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Imsande J(1), Pittig J, Palmer RG, Wimmer C, Gietl C.
				<br/> 
				年份：2001 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The mitochondrial malate dehydrogenase-1 (Mdh1) gene of soybean [Glycine max (L.) Merr.] spontaneously mutates to a null phenotype at a relatively <span class="trait">high</span> rate. To determine the molecular basis for the instability of the <span class="gene">Mdh1</span> gene, the gene was cloned and sequenced. The null phenotype correlated with the deletion of specific genomic restriction fragments that encode the <span class="gene">Mdh1</span> gene. The composition of the <span class="gene">Mdh1</span> gene and its environs were compared with those of the more stable MDH2 gene. Several possible causes of the observed instability were found, including duplications, repeats, and two regions with similarity to a soybean catalase. The most likely cause of instability, however, appeared to be a 1233 bp region with 58.9% identity to the Cyclops retrotransposons. Translation of a 714 bp segment of this region produced a peptide composed of 238 amino acid residues that showed 35-40% identity and 55-60% similarity to several putative Cyclops <span class="gene">gag-pol</span> proteins (group-specific antigen polyprotein). This short peptide also contained a segment that corresponded to the protease active site of the <span class="gene">gag-pol</span> <span class="trait">protein.</span> Thus in an appropriate genetic background, a retrotransposon, whether whole or fractured, could promote genetic rearrangements.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel phytase with sequence similarity to purple acid phosphatases is expressed in cotyledons of germinating soybean seedlings.</h2>
			<h3>PMID:11500558  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hegeman CE(1), Grabau EA.
				<br/> 
				年份：2001 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (myo-inositol hexakisphosphate) is the major storage form of phosphorus in plant seeds. During germination, stored reserves are used as a source of nutrients by the plant seedling. Phytic acid is degraded by the activity of phytases to <span class="trait">yield</span> inositol and free phosphate. Due to the lack of phytases in the non-ruminant digestive tract, monogastric animals cannot utilize dietary phytic acid and it is excreted into manure. <span class="trait">High</span> phytic acid content in manure results in elevated phosphorus levels in soil and water and accompanying environmental concerns. The use of phytases to degrade seed phytic acid has potential for reducing the negative environmental impact of livestock production. A phytase was purified to electrophoretic homogeneity from cotyledons of germinated soybeans (Glycine max L. Merr.). Peptide sequence data generated from the purified enzyme facilitated the cloning of the phytase sequence <span class="gene">(GmPhy)</span> employing a polymerase chain reaction strategy. The introduction of <span class="gene">GmPhy</span> into soybean tissue culture resulted in increased phytase activity in transformed cells, which confirmed the identity of the phytase gene. It is surprising that the soybean phytase was unrelated to previously characterized microbial or maize (Zea mays) phytases, which were classified as histidine acid phosphatases. The soybean phytase sequence exhibited a <span class="trait">high</span> degree of similarity to purple acid phosphatases, a class of metallophosphoesterases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Developmental effects and health aspects of soy <span class="trait">protein</span> isolate, casein, and whey in male and female rats.</h2>
			<h3>PMID:11488559  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Badger TM(1), Ronis MJ, Hakkak R.
				<br/> 
				年份：2001 May-Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Dietary factors other than the traditional nutrients are found in the so-called functional foods. They are becoming increasingly recognized as potentially important for maintaining good health. Soybeans are rich in such factors thought to help prevent certain chronic diseases. Soy <span class="trait">protein</span> isolate (SPI) is one of the three major proteins used in infant formulas sold in the United States, with casein (CAS) and whey (WPH) proteins being the others. We have been studying the health effects of these proteins. Safety concerns have developed over the consumption of soy-based infant formula, partly because of the <span class="trait">high</span> circulating levels of the total isoflavones (phytoestrogens) during "critical periods of infant development." There is a paucity of data on developmental, physiological, neurophysiological, behavioral, metabolic, or molecular effects of soy phytochemicals in humans, especially during pregnancy and infancy. We have studied the effects of CAS, SPI, and WPH in short-term, long-term, and multigenerational studies in rats. Aside from minor differences in body weight gain profiles, CAS-, SPI- or WPH-fed rats did not differ in development, organ weights, in vitro hepatic metabolism of testosterone (T), or reproductive performance. However, some endocrine-related functions differed between rats fed these proteins. We found that SPI accelerated puberty in female rats (p &lt; .05) and WPH delayed puberty in males and females, as compared with <span class="gene">CAS</span> (p &lt; .05). Gender differences were also found in gonadectomy-induced steroid responses. Male rats had normal serum T levels, but female rats fed SPI had reduced serum 17beta-estradiol concentrations and a blunted 17beta-estradiol response to ovariectomy, as compared to rats fed <span class="gene">CAS</span> or WHP (p &lt; .05). Female rats fed SPI or WHP or treated with genistein had reduced incidence of chemically induced mammary cancers (p &lt; .05) compared to <span class="gene">CAS</span> controls, with WHP reducing tumor incidence by as much as 50%, findings that replicate previous results from our laboratory. Together, these results suggest gender-specific differences in development and certain endocrine responses among rats fed diets composed of a single <span class="trait">protein</span> source such as those used in infant formulas. Whether similar developmental effects occur in human infants is unknown, but unlikely because (1) most infants do not consume such diets throughout life as these rats did, and (2) no such effects have been reported in millions of American infants fed infant formula containing these proteins. The long-term health consequence implications of early diet exposure to SPI and WPH, such as reduced breast cancer incidence, are likely to be very positive.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Population-based case&#x02013;control study of soyfood intake and breast cancer risk in Shanghai</h2>
			<h3>PMID:11487268</h3>
				

				
					<h3>PMCID:2364075</h3>
					<h4>期刊:British Journal of Cancer</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dai </surname>  <given-names> Q </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  <xref ref-type="aff" rid="aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> X-O </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> F </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Potter </surname>  <given-names> J D </given-names>  </name>  <xref ref-type="aff" rid="aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kushi </surname>  <given-names> L H </given-names>  </name>  <xref ref-type="aff" rid="aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teas </surname>  <given-names> J </given-names>  </name>  <xref ref-type="aff" rid="aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Y-T </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> W </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 04 </month>  <year> 2001 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We evaluated the association of soyfood intake and breast cancer risk in a population-based case-control study among Chinese women in Shanghai. Included in the study were 1459 cases and 1556 age-matched controls, with respective response rates of 91.1% and 90.3%. Usual soyfood intake was assessed using a food frequency questionnaire (FFQ). Separate analyses were performed for all subjects and for the subset who reported no recent change in soyfood intake. The intake levels of soyfoods among women in Shanghai are <span class="trait">high,</span> with 96.6% women reporting soyfood consumption at least once a week. A statistically non-significant reduced risk (odds ratio (OR) = 0.78 95% <span class="gene">CI</span> = 0.52-1.16) of breast cancer was observed among those who reported eating soyfood at least once a week. Compared to those in the lowest decile intake group, women in the highest decile intake group had a 30% reduced risk of breast cancer (OR = 0.66, 95% <span class="gene">CI</span> = 0.46-0.95), but no monotonic dose-response relation was observed (P for trend, 0.28). Stratified analyses showed that the inverse association was restricted primarily among women who had a <span class="trait">high</span> body mass index (BMI), with an adjusted OR of 0.30 (95% <span class="gene">CI</span> = 0.10-0.94) observed for the highest intake group. The reduction in risk was stronger for breast cancer positive for both oestrogen receptor (ER) and progesterone receptor (PR) (OR = 0.44, 95% <span class="gene">CI</span> = 0.25-0.78) than those with other ER/PR status. More pronounced inverse associations were observed in analyses among those who reported no recent change in soyfood intake than those conducted in all subjects. A dose-response relation between soyfood intake and breast cancer risk was observed in this subset of women (P for trend, 0.02), with an OR of 0.46 (95%CI = 0.28-0.75) for those in the highest decile intake group. No clear monotonic dose-response relation was found between soyfood intake and breast cancer risk among regular soy eaters, but nevertheless the results suggest that regular soyfood consumption may reduce the risk of breast cancer, particularly for those positive for ER and PR; the effect may be modified by body mass index.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2001 Cancer Research Campaign.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We evaluated the association of soyfood intake and breast cancer risk in a population-based case&#x02013;control study among Chinese women in Shanghai. Included in the study were 1459 cases and 1556 age-matched controls, with respective response rates of 91.1&#x00025; and 90.3&#x00025;. Usual soyfood intake was assessed using a food frequency questionnaire (FFQ). Separate analyses were performed for all subjects and for the subset who reported no recent change in soyfood intake. The intake levels of soyfoods among women in Shanghai are <span class="trait">high,</span> with 96.6&#x00025; women reporting soyfood consumption at least once a week. A statistically non-significant reduced risk (odds ratio (OR) = 0.78 95&#x00025; <span class="gene">CI</span> = 0.52&#x02013;1.16) of breast cancer was observed among those who reported eating soyfood at least once a week. Compared to those in the lowest decile intake group, women in the highest decile intake group had a 30&#x00025; reduced risk of breast cancer (OR = 0.66, 95&#x00025; <span class="gene">CI</span> = 0.46&#x02013;0.95), but no monotonic dose&#x02013;response relation was observed ( <italic> P </italic> for trend, 0.28). Stratified analyses showed that the inverse association was restricted primarily among women who had a <span class="trait">high</span> body mass index (BMI), with an adjusted OR of 0.30 (95&#x00025; <span class="gene">CI</span> = 0.10&#x02013;0.94) observed for the highest intake group. The reduction in risk was stronger for breast cancer positive for both oestrogen receptor (ER) and progesterone receptor (PR) (OR = 0.44, 95&#x00025; <span class="gene">CI</span> = 0.25&#x02013;0.78) than those with other ER/PR status. More pronounced inverse associations were observed in analyses among those who reported no recent change in soyfood intake than those conducted in all subjects. A dose&#x02013;response relation between soyfood intake and breast cancer risk was observed in this subset of women ( <italic> P </italic> for trend, 0.02), with an OR of 0.46 (95&#x00025;CI = 0.28&#x02013;0.75) for those in the highest decile intake group. No clear monotonic dose&#x02013;response relation was found between soyfood intake and breast cancer risk among regular soy eaters, but nevertheless the results suggest that regular soyfood consumption may reduce the risk of breast cancer, particularly for those positive for ER and PR; the effect may be modified by body mass index. &#x000a9; 2001 Cancer Research Campaign <ext-link ext-link-type="uri" xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.bjcancer.com"> http://www.bjcancer.com </ext-link>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Pathological excretion patterns of urinary proteins in miners highly exposed to dinitrotoluene.</h2>
			<h3>PMID:11464391  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brüning T(1), Thier R, Mann H, Melzer H, Bröde P, Dallner G, Bolt HM.
				<br/> 
				年份：2001 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cohort of 161 underground miners who had been highly exposed to dinitrotoluene (DNT) in the copper-mining industry of the former German Democratic Republic was reinvestigated for signs of subclinical renal damage. The study included a screening of urinary proteins excreted by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), and quantitations of the specific urinary proteins alpha 1-microglobulin and glutathione-S-transferase alpha (GST alpha) as biomarkers for damage of the proximal tubule and glutathione-S-transferase <span class="gene">pi</span> (GST pi) for damage of the distal tubule. The exposures were categorized semiquantitatively (low, medium, <span class="trait">high,</span> and very <span class="trait">high),</span> according to the type and duration of professional contact with DNT. A straight dose-dependence of pathological <span class="trait">protein</span> excretion patterns with the semiquantitative ranking of DNT exposure was seen. Most of the previously reported cancer cases of the urinary tract, especially those in the higher exposed groups, were confined to pathological urinary <span class="trait">protein</span> excretion patterns. The damage from DNT was directed toward the tubular system. In many cases, the appearance of Tamm-Horsfall <span class="trait">protein,</span> a 105-kD <span class="trait">protein</span> marker, was noted. Data on the biomarkers alpha 1-microglobulin, <span class="gene">GST</span> alpha, and <span class="gene">GST</span> <span class="gene">pi</span> consistently demonstrated a dose-dependent increase in tubular damage, which confirmed the results of screening by SDS-PAGE and clearly indicated a nephrotoxic effect of DNT under the given conditions of exposure. Within the cluster of cancer patients observed among the DNT-exposed workers, only in exceptional cases were normal biomarker excretions found.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Design and production of genetically modified soybean <span class="trait">protein</span> with anti-hypertensive activity by incorporating potent analogue of ovokinin(2-7).</h2>
			<h3>PMID:11376661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matoba N(1), Doyama N, Yamada Y, Maruyama N, Utsumi S, Yoshikawa M.
				<br/> 
				年份：2001 May 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The potent anti-hypertensive peptide, RPLKPW, has been designed based on the structure of ovokinin(2-7). The sequence encoding this peptide was introduced into three homologous sites in the gene for soybean beta-conglycinin <span class="gene">alpha'</span> subunit. The native <span class="gene">alpha'</span> subunit as well as the modified, RPLKPW-containing <span class="gene">alpha'</span> subunit were expressed in Escherichia coli, recovered from the soluble fraction and then purified by ion-exchange chromatography. The RPLKPW peptide was released from recombinant RPLKPW-containing <span class="gene">alpha'</span> subunit after in vitro digestion by trypsin and chymotrypsin. Moreover, the undigested RPLKPW-containing <span class="gene">alpha'</span> subunit given orally at a dose of 10 mg/kg exerted an anti-hypertensive effect in spontaneously hypertensive rats, unlike the native <span class="gene">alpha'</span> subunit. These results provide evidence for the first time that a physiologically active peptide introduced into a food <span class="trait">protein</span> by site-directed mutagenesis could practically function in vivo even at a low dose.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Endothelium-dependent relaxation induced by cathepsin G in porcine pulmonary arteries.</h2>
			<h3>PMID:11375259  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Glusa E(1), Adam C.
				<br/> 
				年份：2001 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Serine proteinases elicit profound cellular effects in various tissues mediated by activation of proteinase-activated receptors (PAR). In the present study, we investigated the vascular effects of cathepsin G, a serine proteinase that is present in the azurophil granules of leukocytes and is known to activate several cells that express PARs. In prostaglandin F2alpha (3 microM)-precontracted rings from porcine pulmonary arteries with intact endothelium, cathepsin G caused concentration-dependent relaxant responses (pEC(50)=9.64+/-0.12). The endothelium-dependent relaxant effect of cathepsin G could also be demonstrated in porcine coronary arteries (pEC(50)=9.23+/-0.07). In pulmonary arteries the cathepsin G-induced relaxation was inhibited after blockade of nitric oxide synthesis by L-NAME (200 microM) and was absent in endothelium-denuded vessels. Bradykinin- and cathepsin G-induced relaxant effects were associated with a 5.7 fold and 2.4 fold increase in the concentration of cyclic <span class="gene">GMP,</span> respectively. Compared with thrombin and trypsin, which also produced an endothelium-dependent relaxation in pulmonary arteries, cathepsin G was 2.5 and four times more potent, respectively. Cathepsin G caused only small homologous desensitization. In cathepsin G-challenged vessels, thrombin was still able to elicit a relaxant effect. The effects of cathepsin G were blocked by soybean trypsin inhibitor (IC(50)=0.043 microg ml(-1)), suggesting that proteolytic activity is essential for induction of relaxation. Recombinant acetyl-eglin C proved to be a potent inhibitor (IC(50)=0.14 microg ml(-1)) of the cathepsin G effect, whereas neither indomethacin (3 microM) nor the thrombin inhibitor hirudin (5 ATU ml(-1)) elicited any inhibitory activity. Due to their polyanionic structure defibrotide (IC(50)=0.11 microg ml(-1)), heparin (IC(50)=0.48 microg ml(-1)) and suramin (IC(50)=1.85 microg ml(-1)) diminished significantly the relaxation in response to the basic <span class="trait">protein</span> cathepsin G. In conclusion, like thrombin and trypsin, cathepsin G is able to induce endothelium-dependent vascular relaxation. It can be released from activated leukocytes at sites of vascular injury and inflammation and, therefore, sufficiently <span class="trait">high</span> concentrations might be reached locally in the vascular space to induce vasodilatation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two expressed soybean genes with <span class="trait">high</span> sequence identity to tomato <span class="gene">Pti1</span> kinase lack autophosphorylation activity.</h2>
			<h3>PMID:11185558  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Staswick P(1).
				<br/> 
				年份：2000 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An important signaling pathway for disease <span class="trait">resistance</span> in tomato involves the R gene product Pto which phosphorylates Ptil, a downstream member of this signaling cascade. Both Pto and <span class="gene">Pti1</span> are Ser/Thr <span class="trait">protein</span> kinases capable of autophosphorylation in vitro. Two soybean (Glycine max L. Merr. var. Hobbit) cDNAs (sPti1a and sPti1b) were cloned and sequenced and found to each have 78% amino acid sequence identity with tomato Pti1. Glutathione S-transferase fusions of sPti1a and b expressed in Escherichia coli did not autophosphorylate in vitro, but were efficiently phosphorylated by tomato Pto. Replacement of Tyr197 with an Asp that is invariant at this position in other <span class="trait">protein</span> kinases did not restore autophosphorylation activity to sPti1a or b. Tyr197 was also present in the <span class="gene">Pti1</span> homologues of three distant relatives of G. max. Together these results suggest that soybean <span class="gene">Pti1</span> might function in a Pto-like signaling pathway that does not require <span class="gene">Pti1</span> kinase activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel cold-inducible zinc finger <span class="trait">protein</span> from soybean, SCOF-1, enhances cold tolerance in transgenic plants.</h2>
			<h3>PMID:11208017  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim JC(1), Lee SH, Cheong YH, Yoo CM, Lee SI, Chun HJ, Yun DJ, Hong JC, Lee SY, Lim CO, Cho MJ.
				<br/> 
				年份：2001 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cold stress on plants induces changes in the transcription of cold response genes. A cDNA clone encoding C2H2-type zinc finger <span class="trait">protein,</span> SCOF-1, was isolated from soybean. The transcription of <span class="gene">SCOF-1</span> is specifically induced by low temperature and abscisic acid (ABA) but not by dehydration or <span class="trait">high</span> salinity. Constitutive overexpression of <span class="gene">SCOF-1</span> induced cold-regulated (COR) gene expression and enhanced cold tolerance of non-acclimated transgenic Arabidopsis and tobacco plants. <span class="gene">SCOF-1</span> localized to the nucleus but did not bind directly to either C-repeat/dehydration (CRT/DRE) or ABA responsive element (ABRE), cis-acting DNA regulatory elements present in COR gene promoters. However, <span class="gene">SCOF-1</span> greatly enhanced the DNA binding activity of SGBF-1, a soybean G-box binding bZIP transcription factor, to ABRE in vitro. <span class="gene">SCOF-1</span> also interacted with SGBF-1 in a yeast two-hybrid system. The SGBF-1 transactivated the beta-glucuronidase reporter gene driven by the ABRE element in Arabidopsis leaf protoplasts. Furthermore, the <span class="gene">SCOF-1</span> enhanced ABRE-dependent gene expression mediated by SGBF-1. These results suggest that <span class="gene">SCOF-1</span> may function as a positive regulator of COR gene expression mediated by ABRE via protein-protein interaction, which in turn enhances cold tolerance of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ochratoxin A-induced tumor formation: is there a role of reactive ochratoxin A metabolites?</h2>
			<h3>PMID:11134544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zepnik H(1), Pähler A, Schauer U, Dekant W.
				<br/> 
				年份：2001 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ochratoxin A is a nephrotoxic and tumorigenic mycotoxin which contaminates a variety of food items, resulting in chronic human exposure. Biotransformation reactions have been implicated in the tumorigenicity of ochratoxin A. The biotransformation of ochratoxin A by cytochromes <span class="gene">P450</span> and other mammalian enzymes was investigated to optimize conditions for bacterial mutagenicity testing. Metabolite formation was assessed by HPLC with UV and fluorescence detection and by LC/MS/MS. When ochratoxin A was incubated with liver microsomes from rats and mice, formation of 4R- and 4S-hydroxyochratoxin A was observed at very low rates. However, oxidation of ochratoxin A was not observed using kidney microsomes from rats and mice. Significantly higher rates of oxidation were seen in liver microsomes from rats pretreated with 3-methylcholanthrene and dexamethasone. Other reported or postulated that ochratoxin A-metabolites were not formed in detectable concentrations. Human cytochromes <span class="gene">P450</span> (3A4, 1A2, and 2C9-1 Supersomes((R))) also showed very low activity with ochratoxin A (&lt;60 fmole/min x pmol P450). Other enzyme systems used to study possible biotransformation of ochratoxin A were rat and human liver and kidney S-9 fortified with NADPH and glutathione, semipurified glutathione S-transferases, horseradish peroxidase, and soybean lipoxygenase; none of these resulted in detectable biotransformation of ochratoxin A. Using rat liver microsomes with <span class="trait">high</span> activity for ochratoxin A oxidation and the other enzyme systems to activate ochratoxin A for mutagenicity testing in the Ames test, mutagenicity was not observed in Salmonella typhimurium TA 100 and TA 2638. The obtained results suggest that oxidative biotransformation of ochratoxin A occurs at low rates, is catalyzed by cytochromes P450, and is unlikely to form reactive intermediates capable of binding to DNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Studies on the topology of the <span class="trait">protein</span> import channel in relation to the plant mitochondrial processing peptidase integrated into the cytochrome <span class="gene">bc1</span> complex.</h2>
			<h3>PMID:11123802  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dessi P(1), Rudhe C, Glaser E.
				<br/> 
				年份：2000 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The mitochondrial processing peptidase (MPP) specifically cleaves N-terminal targeting signals from hundreds of nuclear-encoded, matrix-targeted precursor proteins. In contrast to yeast and mammals, the plant MPP is an integral component of the respiratory cytochrome <span class="gene">bc1</span> complex. The topology of the <span class="trait">protein</span> import channel in relation to MPP/bc1 in plants was studied using chimeric precursors containing truncated cytochrome b2 (cyt b2) proteins of 55-167 residues in length, fused to dihydrofolate reductase (DHFR). The DHFR domain could be tightly folded by methotrexate (MTX), generating translocation intermediates trapped in the import channel with only the cyt b2 pre-sequence/mature domain protruding into the matrix. Spinach and soybean mitochondria imported and processed unfolded precursors. MTX-folded intermediates were not processed in spinach but the longest (1-167) MTX-folded cyt b2-DHFR construct was processed in soybean, while yeast mitochondria successfully processed even shorter MTX-folded constructs. The MTX-folded precursors were cleaved with <span class="trait">high</span> efficiency by purified spinach MPP/bc1 complex. We interpret these results as indicating that the <span class="trait">protein</span> import channel is located distantly from the MPP/bc1 complex in plants, and that there is no link between <span class="trait">protein</span> translocation and <span class="trait">protein</span> processing.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy <span class="trait">protein</span> isolate reduces the oxidizability of LDL and the generation of oxidized LDL autoantibodies in rabbits with diet-induced atherosclerosis.</h2>
			<h3>PMID:11053500  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Damasceno NR(1), Goto H, Rodrigues FM, Dias CT, Okawabata FS, Abdalla DS, Gidlund M.
				<br/> 
				年份：2000 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The incidence of atherosclerosis can be modified by diet, and plant-derived proteins have a beneficial effect, but the underlying mechanisms remain unclear. It has been suggested that oxidized LDL (oxLDL) and autoantibodies against oxLDL are important in the development of atherosclerosis. We analyzed these factors in rabbits fed a nonpurified diet supplemented with <span class="trait">high</span> cholesterol (10.0 g/kg) containing either 270.0 g/kg casein (CAS, n = 10) or 270.0 g/kg soy <span class="trait">protein</span> isolate (SPI, n = 10) for 2 mo. Plasma and purified serum LDL from rabbits were analyzed at d 0, 15, 30, 45 and 60 of treatment, and the size of atherosclerotic lesions was evaluated at d 60 of treatment. CAS-fed rabbits had significantly higher plasma cholesterol at d 15-45 and LDL cholesterol levels at d 15 and 30. Levels of trilinolein and phosphatidylcholine hydroperoxides were higher in the LDL fraction of rabbits fed <span class="gene">CAS</span> than in those fed SPI. Also, CAS-fed rabbits had higher levels of highly oxidized LDL [monoclonal antibody (mAb) 24-reactive oxLDL] in plasma at d 60, whereas SPI-fed rabbits had higher levels of minimally oxidized LDL (mAb 28-reactive oxLDL) at d 45. These results were consistent with the earlier formation of anti-oxLDL antibodies and the presence of a larger area of atherosclerotic lesion in rabbits fed the <span class="gene">CAS</span> diet. These data indicate the importance of both the type of dietary <span class="trait">protein</span> used in the induction of atherosclerosis and the relevance of immunologic mechanisms in addition to biochemical and physiologic factors in the pathogenesis of atherosclerosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Flavonoid 6-hydroxylase from soybean (Glycine max L.), a novel plant P-450 monooxygenase.</h2>
			<h3>PMID:11027686  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Latunde-Dada AO(1), Cabello-Hurtado F, Czittrich N, Didierjean L, Schopfer C, Hertkorn N, Werck-Reichhart D, Ebel J.
				<br/> 
				年份：2001 Jan 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cytochrome P-450-dependent hydroxylases are typical enzymes for the modification of basic flavonoid skeletons. We show in this study that <span class="gene">CYP71D9</span> cDNA, previously isolated from elicitor-induced soybean (Glycine max L.) cells, codes for a <span class="trait">protein</span> with a novel hydroxylase activity. When heterologously expressed in yeast, this <span class="trait">protein</span> bound various flavonoids with <span class="trait">high</span> affinity (1.6 to 52 microm) and showed typical type I absorption spectra. These flavonoids were hydroxylated at position 6 of both resorcinol- and phloroglucinol-based A-rings. Flavonoid 6-hydroxylase (CYP71D9) catalyzed the conversion of flavanones more efficiently than flavones. Isoflavones were hardly hydroxylated. As soybean produces isoflavonoid constituents possessing 6,7-dihydroxy substitution patterns on ring A, the biosynthetic relationship of flavonoid 6-hydroxylase to isoflavonoid biosynthesis was investigated. Recombinant 2-hydroxyisoflavanone synthase (CYP93C1v2) efficiently used 6,7,4'-trihydroxyflavanone as substrate. For its structural identification, the chemically labile reaction product was converted to 6,7,4'-trihydroxyisoflavone by acid treatment. The structures of the final reaction products for both enzymes were confirmed by NMR and mass spectrometry. Our results strongly support the conclusion that, in soybean, the 6-hydroxylation of the A-ring occurs before the 1,2-aryl migration of the flavonoid B-ring during isoflavanone formation. This is the first identification of a flavonoid 6-hydroxylase cDNA from any plant species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Diets deficient in rumen undegraded <span class="trait">protein</span> did not depress milk production.</h2>
			<h3>PMID:10984157  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dunlap TF(1), Kohn RA, Douglass LW, Erdman RA.
				<br/> 
				年份：2000 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to evaluate the National Research Council's recommendations for feeding levels of rumen undegraded <span class="trait">protein</span> (RUP) for cows fed a one-group total mixed ration. Sixty Holstein cows were paired by parity (1 to 6) and <span class="gene">DIM</span> (23 to 315) and were randomly assigned to one of two treatment sequences. Diets contained alfalfa silage (30% diet DM) and corn silage (26% diet DM), and were isonitrogenous (16% CP) and isocaloric (1.71 Mcal/kg). Soybean meal, protected soybean meal (Soy Best), and urea were used to make ration <span class="trait">protein</span> fractions that were predicted to be 35 or 29% RUP. The 35% RUP diet was formulated to provide 98 and 105% of the average requirement for RUP and rumen degraded <span class="trait">protein</span> (RDP), respectively. The ration containing 29% RUP provided 79 and 117% of average required RUP and RDP, respectively. All cows were group-fed the <span class="trait">high</span> RUP diet during a 2-wk pretreatment period, and then were fed one ration for 4 wk followed by the other for 4 wk according to their assigned treatment sequence. Data were collected in the last wk of each period. Mean milk production, milk fat, and milk <span class="trait">protein</span> were 32.6 kg/d, 4.35%, and 3.36%, respectively, with no treatment differences. Treatment response was not affected by degree of predicted RUP deficiency. National Research Council requirements for RUP may be too <span class="trait">high</span> for cows fed diets similar in energy to a one-group total mixed ration. Alternatively, estimates of RUP content of feedstuffs may be low.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interpretation of DSC data on <span class="trait">protein</span> denaturation complicated by kinetic and irreversible effects.</h2>
			<h3>PMID:10867187  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grinberg VY(1), Burova TV, Haertlé T, Tolstoguzov VB.
				<br/> 
				年份：2000 May 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Thermal denaturation of Kunitz soybean trypsin inhibitor (KTI) and ribulose-1,5-biphosphate carboxylase (RBPC) from tobacco leafs was studied by the method of high-sensitivity differential scanning calorimetry (HS-DSC). The dependence of the denaturation temperature on the heating rate reveals in the case of both proteins a non-equilibrium character of the denaturation transition in applied conditions. Developed kinetic approach allows the determination of an equilibrium transition temperature as well as the rate constants of denaturation and renaturation from the complex data of HS-DSC. This method was applied to the analysis of the pH-induced change of the conformational stability of <span class="gene">KTI</span> within pH range from 2.0 to 11.0. It allowed the determination of the pH dependencies: of the excess free energy of denaturation, of the activation enthalpy and entropy of denaturation as well as of the denaturation rate constant. Conclusions have been made suggesting the contribution of side-chain hydrogen bonds in the stabilisation of the native and activated states of KTI.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of mammalian-like purple acid phosphatases in a wide range of plants.</h2>
			<h3>PMID:10854785  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schenk G(1), Guddat LW, Ge Y, Carrington LE, Hume DA, Hamilton S, de Jersey J.
				<br/> 
				年份：2000 May 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Purple acid phosphatases (PAPs) comprise a family of binuclear metal-containing hydrolases, members of which have been isolated from plants, mammals and fungi. Polypeptide chains differ in size (animal approximately 35kDa, plant approximately 55kDa) and exhibit low sequence homology between kingdoms but all residues involved in co-ordination of the metal ions are invariant. A search of genomic databases was undertaken using a sequence pattern which includes the conserved residues. Several novel potential <span class="gene">PAP</span> sequences were detected, including the first known examples from bacterial sources. Ten plant ESTs were also identified which, although possessing the conserved sequence pattern, were not homologous throughout their sequences to previously known plant PAPs. Based on these EST sequences, novel cDNAs from sweet potato, soybean, red kidney bean and Arabidopsis thaliana were cloned and sequenced. These sequences are more closely related to mammalian <span class="gene">PAP</span> than to previously characterized plant enzymes. Their predicted secondary structure is similar to that of the mammalian enzyme. A model of the sweet potato enzyme was generated based on the coordinates of pig PAP. These observations strongly suggest that the cloned cDNA sequences represent a second group of plant PAPs with properties more similar to the mammalian enzymes than to the <span class="trait">high</span> molecular weight plant enzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of low temperature on soybean peroxidase: spectroscopic characterization of the quantum-mechanically admixed spin state.</h2>
			<h3>PMID:10830877  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Indiani C(1), Feis A, Howes BD, Marzocchi MP, Smulevich G.
				<br/> 
				年份：2000 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A spectroscopic study of soybean peroxidase (SBP) has been carried out using electronic absorption, resonance Raman (RR) and electron paramagnetic resonance (EPR) spectroscopy in order to determine the effects of temperature on the heme spin state. Upon lowering the temperature a transition from <span class="trait">high</span> spin to low spin is induced in <span class="gene">SBP</span> resulting from conformational changes in the heme cavity, including a contraction of the heme core, the reorientation of the vinyl group in position 2 of the porphyrin macrocycle, and the binding of the distal His to the Fe atom. Moreover, the combined analysis of the data derived from the different techniques at both room and low temperatures demonstrates that at low temperature the quantum-mechanically admixed spin state (QS) of <span class="gene">SBP</span> has RR frequencies different from those observed for the QS species at room temperature.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Water-selective and multifunctional aquaporins from Lotus japonicus nodules.</h2>
			<h3>PMID:10805445  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guenther JF(1), Roberts DM.
				<br/> 
				年份：2000 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						By using reverse transcriptase-polymerase chain reaction, two cDNAs were isolated that encode major intrinsic membrane proteins (MIPs) that are expressed in nitrogen-fixing root nodules of Lotus japonicus. Lotus intrinsic membrane <span class="trait">protein</span> 1 (LIMP 1) is expressed at <span class="trait">high</span> levels in both nodule and root tissues and shows highest sequence similarity to members of the tonoplast intrinsic <span class="trait">protein</span> (TIP) subfamily of plant MIPs. Functional analysis of LIMP 1 by expression in Xenopus laevis oocytes show that it is a water-specific aquaporin. In contrast, LIMP 2 shows the highest sequence similarity to soybean nodulin 26 (67.8% amino acid sequence identity). LIMP 2 is also a nodulin, showing expression only in mature nitrogen fixing nodules of L. japonicus. LIMP 2 is a multifunctional aquaglyceroporin, and displays the ability to flux both water as well as glycerol upon expression in Xenopus oocytes. Additionally, the carboxyl terminal region of LIMP 2 has a conserved phosphorylation motif that is phosphorylated by a calmodulin-like domain <span class="trait">protein</span> kinase. Overall, the data show that L. japonicus nodules contain two structurally and functionally distinct MIP proteins: one (LIMP 2) which appears to be the nodulin 26 ortholog of L. japonicus and another (LIMP 1) which appears to be a member of the <span class="gene">TIP</span> subfamily.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural genetic competence in Bacillus subtilis natto OK2.</h2>
			<h3>PMID:10762239  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ashikaga S(1), Nanamiya H, Ohashi Y, Kawamura F.
				<br/> 
				年份：2000 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We isolated a Bacillus subtilis natto strain, designated OK2, from a lot of commercial fermented soybean natto and studied its ability to undergo natural competence development using a comG-lacZ fusion at the amyE locus. Although transcription of the late competence genes was not detected in the B. subtilis natto strain OK2 during competence development, these genes were constitutively transcribed in the OK2 strain carrying either the <span class="gene">mecA</span> or the <span class="gene">clpC</span> mutation derived from B. subtilis 168. In addition, both OK2 mutants exhibited <span class="trait">high</span> transformation frequencies, comparable with that observed for B. subtilis 168. Moreover, as expected from these results, overproduction of ComK derived from strain 168 in strain OK2 resulted in a <span class="trait">high</span> transformation frequency as well as in induction of the late competence genes. These results clearly indicated that ComK produced in both the <span class="gene">mecA</span> and <span class="gene">clpC</span> mutants of strain OK2 (ComK(OK2)) could activate the transcription of the whole set of late competence genes and suggested that ComK(OK2) was not activated in strain OK2 during competence development. We therefore sequenced the comS gene of OK2 and compared it with that of 168. The comS(OK2) had a single-base change, resulting in the replacement of Ser (strain 168) by Cys (strain OK2) at position 11.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two genes encoding a putative multidrug efflux pump of the RND/MFP family are cotranscribed with an <span class="gene">rpoH</span> gene in Bradyrhizobium japonicum.</h2>
			<h3>PMID:10675036  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Krummenacher P(1), Narberhaus F.
				<br/> 
				年份：2000 Jan 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The rpoH3 gene of Bradyrhizobium japonicum codes for one of three sigma32-type transcription factors in this organism and is flanked by rag (rpoH3-associated) genes comprising the chromosomal arrangement ragABrpoH3ragCD. The first genes in this cluster code for a classical two-component regulatory system with an unknown function (Narberhaus et al., 1997. Mol. Microbiol. 24, 93-104). The deduced proteins of the last two genes display a <span class="trait">high</span> sequence similarity to heavy metal or multidrug efflux pumps of the RND (Resistance/Nodulation/cell Division)-MFP (Membrane Fusion <span class="trait">Protein)</span> family. Reverse transcription and polymerase chain reaction (RT-PCR) analysis demonstrated that ragC is cotranscribed with rpoH3. Mutant strains carrying disrupted rag genes or an extented deletion of the rag locus exhibited neither an apparent growth defect nor a deficiency in the symbiotic interaction with soybean roots. The minimal inhibitory concentrations (MICs) of various metals and organic compounds were identical for the wild-type and mutant strains. Moreover, translational lacZ fusions to each of the first four genes of the rag cluster showed a very low expression under all conditions tested; hence, the substrate for the putative efflux pump has not been uncovered.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Carbon metabolism in developing soybean root nodules: the role of carbonic anhydrase.</h2>
			<h3>PMID:10656581  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kavroulakis N(1), Flemetakis E, Aivalakis G, Katinakis P.
				<br/> 
				年份：2000 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA clone encoding carbonic anhydrase (CA) was isolated from a soybean nodule cDNA library. In situ hybridization and immunolocalization were performed in order to assess the location of CA transcripts and <span class="trait">protein</span> in developing soybean nodules. CA transcripts and <span class="trait">protein</span> were present at <span class="trait">high</span> levels in all cell types of young nodules, whereas in mature nodules they were absent from the central tissue and were concentrated in cortical cells. The results suggested that, in the earlier stages of nodule development, CA might facilitate the recycling of CO2 while at later stages it may facilitate the diffusion of CO2 out of the nodule system. In parallel, sucrose metabolism was investigated by examination of the temporal and spatial transcript accumulation of sucrose synthase (SS) and phosphoenolpyruvate carboxylase (PEPC) genes, with in situ hybridization. In young nodules, <span class="trait">high</span> levels of <span class="gene">SS</span> gene transcripts were found in the central tissue as well as in the parenchymateous cells and the vascular bundles, while in mature nodules the levels of <span class="gene">SS</span> gene transcripts were much lower, with the majority of the transcripts located in the parenchyma and the pericycle cells of the vascular bundles. <span class="trait">High</span> levels of expression of PEPC gene transcripts were found in mature nodules, in almost all cell types, while in young nodules lower levels of transcripts were detected, with the majority of them located in parenchymateous cells as well as in the vascular bundles. These data suggest that breakdown of sucrose may take place in different sites during nodule development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">Eu3</span> gene encodes an Ni-binding <span class="trait">protein</span> necessary for urease activity.</h2>
			<h3>PMID:10652150  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Freyermuth SK(1), Bacanamwo M, Polacco JC.
				<br/> 
				年份：2000 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mutation in <span class="gene">Eu3</span> eliminates activity of both soybean ureases, the embryo-specific (encoded by Eu1) and the tissue-ubiquitous (encoded by Eu4). eu3-e1 is a completely recessive null allele. Eu3-e3 is a semi-dominant specifying 0.1% wild-type urease activity in the homozygous state and 5-10% as a heterozygote (Meyer-Bothling et al. 1987). Antibodies to plant UreG, a homologue of the bacterial urease accessory <span class="trait">protein,</span> revealed a 32 kDa <span class="trait">protein</span> (p32) in embryos of the Eu3/Eu3 precursor genotype. p32 is identical to UreG by the criteria of size, antigenicity, and its ability to bind Ni2+, a trait expected from the deduced histidine-rich N-terminus of UreG. UreG was absent in eu3-e1/eu3-e1, and lack of UreG co-segregated with eu3-e1. Eu3-e3 specified a UreG transcript which coded valine in place of alanine at residue 142 (A142V) confirming thatEu3 encodes UreG, which is renamed Eu3. <span class="gene">Eu3</span> (A142V) retained Ni-binding ability. <span class="gene">Eu3</span> is directly involved in urease activation, since anti-Eu3 (UreG) antibodies inhibited the in vitro activation of urease. Eu1 (embryo urease) and <span class="gene">Eu3</span> accumulated in parallel in the developing embryo. The presence of Eu1 was not necessary for the <span class="trait">high</span> embryonic level of Eu3. However, the presence of <span class="gene">Eu3</span> appeared to be important for accumulation of Eu1, perhaps by stabilizing it by Ni insertion. At the level of sensitivity employed <span class="gene">Eu3</span> was detected in crude extracts of embryos but not non-embryonic tissues which have 1/500th the embryo urease activity. Functional Eu3, however, is necessary for activation of the ubiquitous urease in non-embryonic tissues.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-physicochemical function relationships of soybean beta-conglycinin constituent subunits.</h2>
			<h3>PMID:10606609  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Maruyama N(1), Sato R, Wada Y, Matsumura Y, Goto H, Okuda E, Nakagawa S, Utsumi S.
				<br/> 
				年份：1999 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Beta-conglycinin, one of the dominant storage proteins of soybean, has a trimeric structure, being composed of three subunits alpha, alpha', and beta. The alpha and <span class="gene">alpha'</span> subunits contain the extension regions in addition to the core regions common to all subunits, which are N-glycosylated. Physicochemical functions of recombinant nonglycosylated individual subunits and deletion mutants (alpha(c) and alpha'(c)) lacking the extension regions of the alpha and <span class="gene">alpha'</span> subunits were examined at pH 7.6 and 3.7 at low (mu = 0.08) and <span class="trait">high</span> (mu = 0.5) ionic strengths. Although individual recombinant subunits exhibited different properties at all conditions, there were some consistencies. Surface hydrophobicities and thermal stabilities of the individual subunits were likely to be conferred by their core regions, and the carbohydrate moieties did not contribute to these properties at any conditions examined here. Solubility at mu = 0.08, heat-induced association, and emulsifying ability remarkably depended on the extension regions and the carbohydrate moieties in addition to the structural features of the core regions. These findings indicate that various end products could be produced by the selection of soybean varieties containing beta-conglycinin with different subunit compositions and suggest a direction for a principle of soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of disulfide bonds upon the structural stability of an amaranth globulin.</h2>
			<h3>PMID:10552600  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Castellani OF(1), Martínez EN, Añón MC.
				<br/> 
				年份：1999 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Analysis of globulin-P, the polymerized amaranth globulin, gave a low amount of free sulfhydryls (10.2 +/- 0.5 micromol/g) from which 7 +/- 1 micromol/g was buried inside the molecule. In addition, its disulfide content was <span class="trait">high</span> (51 +/- 1 micromol/g) and similar to soybean 11S globulin content. The more exposed disulfide bridges were found to be stabilizing polymers, whereas the less reactive bridges were either linking P(20) and P(30) polypeptides or forming intrachain linkages. It was found that the buried bonds participate in the stabilization of folded polypeptides and the quaternary structure of the globulin. In turn, the dissociation of polymers and disruption of the quaternary structure by the action of 2-mercaptoethanol reverted upon removal of the reducing agent. This demonstrates that the polymerized state and the quaternary structure of the molecules are most favorable from the thermodynamic point of view. The similar content of SH and <span class="gene">SS</span> in globulin-P and globulin-S found in this laboratory suggests that the differences between these proteins may be ascribed to other compositional differences.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel promoter from soybean that is active in a complex developmental pattern with and without its proximal 650 base pairs.</h2>
			<h3>PMID:10579489  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Strömvik MV(1), Sundararaman VP, Vodkin LO.
				<br/> 
				年份：1999 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We report the isolation of a novel soybean gene, Msg, which is highly expressed in developing soybean pods. The gene shows significant homology to a family of fruit- and flower-specific genes, designated the major latex <span class="trait">protein</span> (MLP) homologues, so far reported in only a few species and whose functions are unknown. The MLPs are more distantly related to a group of pathogenesis-related proteins (IPR or PR-10) whose functions are likewise unknown. This is the first report of a <span class="gene">MLP</span> homologue in a plant for which there is already an IPR-protein reported. We performed an analysis of the Msg promoter with 14 different promoter fragments ranging from 0.65 kb to 2.26 kb, fused to the uidA (GUS) gene. <span class="trait">High</span> transient expression was obtained with all the constructs upon particle bombardment in soybean and green bean pods. Stable Arabidopsis transformants were obtained with the Agrobacterium vacuum infiltration method. The promoter is fully active in Arabidopsis only in plants transformed with the 2.26 kb fragment promoter, expressing GUS in nectaries, nodes, short style and in guard cells of the silique, pedicel and stem but not in mature leaves. Surprisingly, the proximal 650 bp TATA-containing region cannot function on its own in Arabidopsis and can be deleted without a change in expression pattern in both Arabidopsis and soybean. Thus, tissue-specific regions of the complex Msg promoter reside in the distal 5' regions upstream of a dispensable TATA box in contrast to many examples of tissue-specific elements that reside much closer to the TATA box.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxynitrite-mediated modification of proteins at physiological carbon dioxide concentration: pH dependence of carbonyl formation, tyrosine nitration, and methionine oxidation.</h2>
			<h3>PMID:10393903  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tien M(1), Berlett BS, Levine RL, Chock PB, Stadtman ER.
				<br/> 
				年份：1999 Jul 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The ability of peroxynitrite to modify amino acid residues in glutamine synthetase (GS) and BSA is greatly influenced by pH and CO2. At physiological concentrations of CO2 (1.3 mM), the generation of carbonyl groups (0.2-0.4 equivalents/subunit) is little affected by pH over the range of 7.2-9.0, but, in the absence of CO2, carbonyl formation increases (from 0.1- 1.2 equivalents/subunit) as the pH is raised from 7.2 to 10.5. This increase is attributable, in part but not entirely, to the increase in peroxynitrite (PN) stability with increasing pH. Of several amino acid polymers tested, only those containing lysine residues yielded carbonyl derivatives. In contrast, the nitration of tyrosine residues of both <span class="gene">GS</span> and BSA at pH 7.5 almost completely depends on the presence of CO2. However, the pH profiles of tyrosine nitration in <span class="gene">GS</span> and BSA are not the same. With both proteins, nitration decreases approximately 65% with increasing pH over the range of 7.2-8.4, but, then in the case of <span class="gene">GS</span> only, there is a 3.4-fold increase in the level of nitration over the range pH 8.4-8.8. The oxidation of methionine residues in both proteins and in the tripeptide Ala-Met-Ala was inhibited by CO2 at both <span class="trait">high</span> and low pH values. These results emphasize the importance of controlling the pH and CO2 concentrations in studies involving PN and indicate that PN is not likely to contribute appreciably to carbonyl formation or oxidation of methionine residues of proteins at physiological pH and CO2 concentrations.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combined effect of vegetable <span class="trait">protein</span> (soy) and soluble fiber added to a standard cholesterol-lowering diet.</h2>
			<h3>PMID:10381159  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jenkins DJ(1), Kendall CW, Mehling CC, Parker T, Rao AV, Agarwal S, Novokmet R, Jones PJ, Raeini M, Story JA, Furumoto E, Vidgen E, Griffin LC, Cunnane SC, Ryan MA, Connelly PW.
				<br/> 
				年份：1999 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Dietary treatment of hyperlipidemia focuses on reducing saturated fat and dietary cholesterol. Other aspects of diet are not emphasized at present, despite growing evidence that a number of plant components decrease serum cholesterol. We therefore determined whether a combination of two plant components, vegetable <span class="trait">protein</span> and soluble fiber, further reduce serum lipids when incorporated into the currently advocated low-saturated-fat diet. Thirty-one hyperlipidemic men and women ate two 1-month low-fat (&lt;7% of total energy from saturated fat), low-cholesterol (&lt;80 mg cholesterol/d) metabolic diets in a randomized crossover study. The major differences between test and control diets were an increased amount of vegetable <span class="trait">protein</span> (93% v 23% of total <span class="trait">protein),</span> of which 33 g/d was soy, and a doubling of soluble fiber. Fasting blood samples were obtained at the start and end of each phase. On the last 3 days of each phase, fecal collections were obtained. Compared with the low-fat control diet, the test diet decreased total cholesterol (6.2% +/- 1.2%, P &lt; .001), low-density lipoprotein (LDL) cholesterol (6.7% +/- 1.7%, P &lt; .001), apolipoprotein B (8.2% +/- 1.2%, P &lt; .001), and the ratios of LDL to high-density lipoprotein (HDL) cholesterol (6.3% +/- 2.0%, P = .004) and apolipoprotein B to A-I (5.4% +/- 1.5%, P = .001). A combination of vegetable <span class="trait">protein</span> and soluble fiber significantly improved the lipid-lowering effect of a low-saturated-fat diet. The results support expanding the current dietary advice to include increased vegetable <span class="trait">protein</span> and soluble fiber intake so that the <span class="gene">gap</span> in effectiveness between a good diet and drug therapy is reduced.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of glycosylation in the functional expression of an Aspergillus niger phytase (phyA) in Pichia pastoris.</h2>
			<h3>PMID:10087168  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Han Y(1), Lei XG.
				<br/> 
				年份：1999 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Economical and thermostable phytase enzymes are needed to release phytate-phosphorus in plant foods for human and animal nutrition and to reduce phosphorus pollution of animal waste. Our objectives were to determine if a methylotrophic yeast, Pichia pastoris, was able to express a phytase gene (phyA) from Aspergillus niger efficiently and if suppression of glycosylation by tunicamycin affected its functional expression. The gene (1.4 kb) was inserted into an expression vector pPICZalphaA with a signal peptide alpha-factor, under the control of <span class="gene">AOX1</span> promoter. The resulting plasmid was transformed into two P. pastoris strains: KM71 (methanol utilization slow) and X33 (wild-type). Both host strains produced <span class="trait">high</span> levels of active phytase (25-65 units/ml of medium) that were largely secreted into the medium. The expressed enzyme was cross-reacted with the polyclonal antibody raised against the wild-type enzyme and showed two pH optima, 2.5 and 5.5, and an optimal temperature at 60 degrees C. Compared with the <span class="gene">phyA</span> phytase overexpressed by A. niger, this phytase had identical capacity in hydrolyzing phytate-phosphorus from soybean meal and slightly better thermostability. Deglycosylation of the secreted phytase resulted in reduction in the size from 95 to 55 kDa and in thermostability by 34%. Tunicamycin (20 microg/ml of medium) resulted in significant reductions of both intracellular and extracellular phytase activity expression. Because there was no accumulation of intracellular phytase <span class="trait">protein,</span> the impairment did not seem to occur at the level of translocation of phytase. In conclusion, glycosylation was vital to the biosynthesis of the <span class="gene">phyA</span> phytase in P. pastoris and the thermostability of the expressed enzyme.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 1999 Academic Press.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative analysis of Gly m Bd 28K in soybean products by a sandwich enzyme-linked immunosorbent assay.</h2>
			<h3>PMID:9919485  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bando N(1), Tsuji H, Hiemori M, Yoshizumi K, Yamanishi R, Kimoto M, Ogawa T.
				<br/> 
				年份：1998 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A sandwich enzyme-linked immunosorbent assay for the soybean allergen, which consists of a monoclonal antibody (D4) as the fixing (first) antibody and another peroxidase-conjugated monoclonal antibody (C5) as the second, has been developed. Both D4 and <span class="gene">C5</span> monoclonal antibodies strongly bound to the guanidine/HCl-denatured allergen, Gly m Bd 28K. Therefore the samples used in the present experiment were extracted with sodium phosphate buffer (pH 8.0) containing 6 M guanidine and 10 mM 2-mercaptoethanol, then completely dialyzed against phosphate-buffered saline (PBS). The dialyzed samples were subjected to the assay. Various soybean products were observed to contain the allergen at <span class="trait">high</span> concentrations, such as soybean <span class="trait">protein</span> isolate (SPI), tofu, kori-dofu, and yuba, but its content in soy milk and abura-age were found to be low. In fermented products such as natto, soy sauce, and miso, and even in the processed foods with soybean <span class="trait">protein</span> isolate (SPI), the allergen was not detected. These results were also confirmed by an immunoblotting technique with D4.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Electrophysiological study with oxonol VI of passive NO3- transport by isolated plant root plasma membrane.</h2>
			<h3>PMID:9876148  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pouliquin P(1), Grouzis J, Gibrat R.
				<br/> 
				年份：1999 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In contrast to animal cells, plant cells contain approximately 5-50 mM nitrate in cytosol and vacuole. The lack of specific spectroscopic probes, or suitable isotopes, impedes in vitro studies of NO3- transport. Reconstitution of root cell plasma membrane (PM) proteins in mixed soybean lipid:egg phosphatidylcholine allowed for the generation of large K+-valinomycin diffusion potentials (Em), monitored with the oxonol VI dye. Nevertheless, <span class="gene">Em</span> was restricted to approximately 130 mV by capacitor properties of biological membranes. This caused an increasing discrepancy at higher K+-Nernst potentials used for calibration. Therefore, <span class="gene">Em</span> was determined directly from the fluorescence of the dye free in buffer, bound at zero Em, and bound upon <span class="gene">Em</span> generation. Then, an electrophysiological analysis of the NO3--dependent dissipation rate of <span class="gene">Em</span> gave the net passive flux (JN) and the permeability coefficient to NO3- (PN). The plant root cell PM exhibited a strikingly large PN (higher than 10(-9) m s-1) at <span class="trait">high</span> <span class="gene">Em</span> (90-100 mV) and pH 6.5. At low <span class="gene">Em</span> (50-60 mV) and pH 7.4, PN decreased by 70-fold and became similar to that of the lipid bilayer. This agreed with the previous observation that 15 mM NO3- short-circuits the plant root PM H+-ATPase at its optimal pH of 6.5.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evidence for the formation of dinor isoprostanes <span class="gene">E1</span> from alpha-linolenic acid in plants.</h2>
			<h3>PMID:9830005  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Parchmann S(1), Mueller MJ.
				<br/> 
				年份：1998 Dec 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The free radical oxidation of arachidonic acid is known to generate complex metabolites, termed isoprostanes, that share structural features of prostaglandins and exert potent receptor-mediated biological activities. In the present study, we show that alpha-linolenic acid can undergo a similar oxidation process, resulting in a series of isomeric dinor isoprostanes E1. E-ring dinor isoprostane formation from linolenate was found to be catalyzed by soybean lipoxygenase. The main enzymatic products were 13- and 9-hydroperoxylinolenate but in addition, two dinor isoprostane <span class="gene">E1</span> regioisomers were formed with a <span class="trait">yield</span> of 0.31%. Identification and quantification of two dinor isoprostane <span class="gene">E1</span> regioisomers in plant cell cultures was achieved by a negative chemical ionization gas chromatography-mass spectrometry method using [18O]dinor isoprostanes <span class="gene">E1</span> as internal standards. Endogenous levels of these compounds were determined in four taxonomically distant plant species and found to be in the range of 4.5 to 60.9 ng/g of dry weight. Thus analogous pathways in animals and plants exist, each leading to a family of prostaglandin-like compounds derived from polyunsaturated fatty acids. It remains to be shown whether the dinor isoprostanes exert biological activities in plants as has been demonstrated for their C20 congeners in mammals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the 11,14,15- and 11,12, 15-trihydroxyeicosatrienoic acids as endothelium-derived relaxing factors of rabbit aorta.</h2>
			<h3>PMID:9812980  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pfister SL(1), Spitzbarth N, Nithipatikom K, Edgemond WS, Falck JR, Campbell WB.
				<br/> 
				年份：1998 Nov 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A number of endothelium-derived relaxing factors have been identified including nitric oxide, prostacyclin, and the epoxyeicosatrienoic acids. Previous work showed that in rabbit aortic endothelial cells, arachidonic acid was metabolized by a lipoxygenase to vasodilatory eicosanoids. The identity was determined by the present study. Aortic homogenates were incubated in the presence of [U-14C]arachidonic acid, [U-14C]arachidonic acid plus 15-lipoxygenase (soybean lipoxidase), or [U-14C]15-hydroxyeicosatetraenoic acid (15-HPETE) and analyzed by reverse phase <span class="trait">high</span> pressure liquid chromatography (RP-HPLC). Under both experimental conditions, there was a radioactive metabolite that migrated at 17.5-18.5 min on RP-HPLC. When the metabolite was isolated from aortic homogenates, it relaxed precontracted aortas in a concentration-dependent manner. Gas chromatography/mass spectrometry (GC/MS) of the derivatized metabolite indicated the presence of two products; 11,12,15-trihydroxyeicosatrienoic acid (THETA) and 11,14,15-THETA. A variety of chemical modifications of the metabolite supported these structures and confirmed the presence of a carboxyl group, double bonds, and hydroxyl groups. With the combination of 15-lipoxygenase, arachidonic acid, and aortic homogenate, an additional major radioactive peak was observed. This fraction was analyzed by GC/MS. The mass spectrum was consistent with this peak, containing both the 11-hydroxy-14, 15-epoxyeicosatrienoic acid (11-H-14,15-EETA) and 15-H-11,12-EETA. The hydroxyepoxyeicosatrienoic acid (HEETA) fraction also relaxed precontracted rabbit aorta. Microsomes derived from rabbit aortas also synthesized 11,12,15- and 11,14,15-THETAs from 15-HPETE, and pretreatment with the cyctochrome <span class="gene">P450</span> inhibitor, miconazole, blocked the formation of these products. The present studies suggest that arachidonic acid is metabolized by 15-lipoxygenase to 15-HPETE, which undergoes an enzymatic rearrangement to 11-H-14,15-EETA and 15-H-11,12-EETA. Hydrolysis of the epoxy group results in the formation of 11,14,15- and 11,12,15-THETA, which relaxed rabbit aorta. Thus, the 15-series THETAs join prostacyclin, nitric oxide, and epoxyeicosatrienoic acids as new members of the family of endothelium-derived relaxing factors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue- and stage-specific expression of a soybean (Glycine max L.) seed-maturation, biotinylated <span class="trait">protein.</span></h2>
			<h3>PMID:9747855  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hsing YC(1), Tsou CH, Hsu TF, Chen ZY, Hsieh KL, Hsieh JS, Chow TY.
				<br/> 
				年份：1998 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA clone <span class="gene">GmPM4</span> which encodes mRNA species in mature or dry soybean seeds was characterized. DNA sequence analysis shows that the deduced polypeptides have a molecular mass of 68 kDa. <span class="gene">GmPM4</span> proteins have a relatively <span class="trait">high</span> amino acid sequence homology with a major biotinylated <span class="trait">protein</span> isolated from pea seeds, SBP65, but both of these proteins differ markedly from that of presently known biotin enzymes. The accumulation of <span class="gene">GmPM4</span> mRNA is detectable in the leaf primodium and the vascular tissues of the hypocotyl-radicle axis of mature seeds, and the <span class="gene">GmPM4</span> proteins are present at <span class="trait">high</span> levels in dry and mature soybean seeds, but not in fresh immature seeds. It degrades rapidly at the early stage of seed germination. These proteins are boiling-soluble and biotinylated when they are present endogenously in soybean seeds; however, the same recombinant <span class="trait">protein</span> expressed in Escherichia coli is boiling-soluble, but it is not biotinylated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Exopolysaccharide (EPS) synthesis in Bradyrhizobium japonicum: sequence, operon structure and mutational analysis of an exo gene cluster.</h2>
			<h3>PMID:9747707  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Becker BU(1), Kosch K, Parniske M, Müller P.
				<br/> 
				年份：1998 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nucleotide sequence of a 8330-bp DNA fragment from Bradyrhizobium japonicum 110spc4 was determined. Sequence analysis revealed that six ORFs were present and the deduced amino acid sequences were homologous to enzymes involved in exopolysaccharide (EPS) biosynthesis. The genes appear to be organized into at least four different operons. One gene was found to be homologous to exoB, which encodes a UDP-galactose 4'-epimerase. Other ORFs were homologous to UDP-hexose transferases and one ORF showed similarity to Sinorhizobium (Rhizobium) meliloti ExoP, which has been suggested to be involved in EPS chain-length determination. A set of deletion and insertion mutants was constructed and the resulting B. japonicum strains were tested for their symbiotic traits. Deletion mutant deltaP22, which lacks the C-terminal part of ExoP, the UDP-hexose transferase ExoT and the N-terminal part of ExoB, shows a delayed nodulation phenotype and induces symptoms of plant defense reactions; its EPS does not contain galactose and no <span class="trait">high</span> molecular weight fraction is synthesized. In contrast, insertion mutant EH3, which expresses an <span class="gene">exoP</span> gene product that is truncated in its putative periplasmic domain, produced an EPS containing both HMW and LMW fractions. However, the interaction of EH3 with soybeans was severely perturbed. As a rule, only the initial steps of nodule formation were observed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization and functional expression of dihydroxypterocarpan 6a-hydroxylase, an enzyme specific for pterocarpanoid phytoalexin biosynthesis in soybean (Glycine max L.).</h2>
			<h3>PMID:9720921  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schopfer CR(1), Kochs G, Lottspeich F, Ebel J.
				<br/> 
				年份：1998 Aug 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Four cytochrome P450-dependent enzymes, among them dihydroxypterocarpan 6a-hydroxylase (D6aH), are specifically involved in the elicitor-inducible biosynthesis of glyceollins, the phytoalexins of soybean. Here we report that <span class="gene">CYP93A1</span> cDNA, which we isolated previously from elicitor-induced soybean cells, codes for a <span class="trait">protein</span> with D6aH activity. Analysis of the catalytic properties of recombinant <span class="gene">CYP93A1</span> expressed in yeast, its NADPH dependency, stereoselectivity and <span class="trait">high</span> substrate affinity confirmed that D6aH is the physiological function of CYP93A1. It thus represents the first isoflavonoid-specific CYP to be characterized at the molecular level. In elicitor-treated soybean cells producing phytoalexins, increases in D6aH activity were correlated with elevated transcript levels which indicates that expression of the enzyme is regulated at the level of transcription. Therefore, <span class="gene">CYP93A1</span> cDNA can be used as a specific molecular marker for the inducible defense response against pathogen attack.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Utilization of the <span class="gene">TEF1-alpha</span> gene (TEF1) promoter for expression of polygalacturonase genes, pgaA and pgaB, in Aspergillus oryzae.</h2>
			<h3>PMID:9720204  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kitamoto N(1), Matsui J, Kawai Y, Kato A, Yoshino S, Ohmiya K, Tsukagoshi N.
				<br/> 
				年份：1998 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For the development of an efficient gene expression system in a shoyu koji mold Aspergillus oryzae KBN616, the TEF1 gene, encoding translation-elongation factor 1 alpha, was cloned from the same strain and used for expression of polygalacturonase genes. The TEF1 gene comprised 1647 bp with three introns. The <span class="gene">TEF1-alpha</span> <span class="trait">protein</span> consisted of 460 amino acids possessing <span class="trait">high</span> identify to other fungal <span class="gene">TEF</span> proteins. Two nucleotide sequences homologous to the upstream activation sequence, characterized for the ribosomal <span class="trait">protein</span> genes in Saccharomyces cerevisiae, as well as the pyrimidine-rich sequences were present in the TEF1 gene promoter region, suggesting that the A, oryzae TEF1 gene has a strong promoter activity. Two expression vectors, pTFGA300 and pTFGB200 for production of polygalacturonases A and B respectively, were constructed by using the TEF1 gene promoter. A polygalacturonase (PGB) gene cloned from the same strain comprised 1226 bp with two introns and encoded a <span class="trait">protein</span> of 367 amino acids with <span class="trait">high</span> similarity to other fungal polygalacturonases. PGA and PGB were secreted at approximately 100 mg/l in glucose medium and purified to homogeneity. PGA had a molecular mass of 41 kDa, a pH optimum of 5.0 and temperature optimum of 45 degrees C. PGB had a molecular mass of 39 kDa, a pH optimum of 5.0 and temperature optimum of 55 degrees C.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of <span class="gene">GmNRT2</span> expression and nitrate transport activity in roots of soybean (Glycine max).</h2>
			<h3>PMID:9715532  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Amarasinghe BH(1), de Bruxelles GL, Braddon M, Onyeocha I, Forde BG, Udvardi MK.
				<br/> 
				年份：1998 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA, <span class="gene">GmNRT2,</span> encoding a putative high-affinity nitrate transporter was isolated from a Glycine max (L.) root cDNA library and sequenced. The deduced <span class="gene">GmNRT2</span> <span class="trait">protein</span> is 530 amino acids in length and contains 12 putative membrane-spanning domains and a long, hydrophilic C-terminal domain. <span class="gene">GmNRT2</span> is related to high-affinity nitrate transporters in the eukaryotes Chlamydomonas reinhardtii and Aspergillus nidulans, and to putative high-affinity nitrate transporters in barley and tobacco. Southern blot analysis indicated that <span class="gene">GmNRT2</span> is part of a small, multigene family in soybean. Expression of <span class="gene">GmNRT2</span> in roots was regulated by the type of nitrogen source provided to plants: <span class="gene">GmNRT2</span> mRNA levels were barely detectable in ammonium-grown plants, higher in nitrogen-deprived plants, and highest in nitrate-grown plants. Induction of <span class="gene">GmNRT2</span> mRNA levels in roots occurred within 1 h after exposure of plants to nitrate. Nitrate induction of <span class="gene">GmNRT2</span> mRNA levels was accompanied by a fourfold increase in net nitrate uptake by soybean roots at 100 microM external nitrate. The molecular and physiological evidence indicates that <span class="gene">GmNRT2</span> is probably a high-affinity nitrate transporter involved in nitrate uptake by soybean roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of a lipase fermentation process that uses a recombinant Pseudomonas alcaligenes strain.</h2>
			<h3>PMID:9647843  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gerritse G(1), Hommes RW, Quax WJ.
				<br/> 
				年份：1998 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pseudomonas alcaligenes M-1 secretes an alkaline lipase, which has excellent characteristics for the removal of fatty stains under modern washing conditions. A fed-batch fermentation process based on the secretion of the alkaline lipase from P. alcaligenes was developed. Due to the inability of P. alcaligenes to grow on glucose, citric acid and soybean oil were applied as substrates in the batch phase and feed phase, respectively. The gene encoding the high-alkaline lipase from P. alcaligenes was isolated and characterized. Amplification of lipase gene copies in P. alcaligenes with the aid of low- and high-copy-number plasmids resulted in an increase of lipase expression that was apparently colinear with the gene copy number. It was found that overexpression of the lipase helper gene, lipB, produced a stimulating effect in strains with <span class="trait">high</span> copy numbers (&gt; 20) of the lipase structural gene, lipA. In strains with <span class="gene">lipA</span> on a low-copy-number vector, the <span class="gene">lipB</span> gene did not show any effect, suggesting that LipB is required in a low ratio to LipA only. During scaling up of the fermentation process to 100 m3, severe losses in lipase productivity were observed. Simulations have identified an increased level of dissolved carbon dioxide as the most probable cause for the scale-up losses. A large-scale fermentation protocol with a reduced dissolved carbon dioxide concentration resulted in a substantial elimination of the scale-up loss.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Bradyrhizobium japonicum <span class="gene">noeD</span> gene: a negatively acting, genotype-specific nodulation gene for soybean.</h2>
			<h3>PMID:9612946  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lohrke SM(1), Day B, Kolli VS, Hancock R, Yuen JP, de Souza ML, Stacey G, Carlson R, Tong Z, Hur HG, Orf JH, Sadowsky MJ.
				<br/> 
				年份：1998 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bradyrhizobium japonicum strain USDA 110 is restricted for nodulation by soybean genotype PI 417566. We previously reported the identification of a USDA 110 Tn5 mutant, strain D4.2-5, that had the ability to overcome nodulation restriction conditioned by PI 417566 (S. M. Lohrke, J. H. Orf, E. Martínez-Romero, and M. J. Sadowsky, Appl. Environ. Microbiol. 61:2378-2383, 1995). In this study, we report the cloning and characterization of the negatively acting DNA region mutated in strain D4.2-5 that is involved in the genotype-specific nodulation of soybean. The Tn5 integration site was localized to a 5.2-kb EcoRI fragment isolated from wild-type USDA 110 genomic DNA. Saturation Tn5 mutagenesis of this 5.2-kb region and DNA homogenitization studies indicated that a 0.9-kb DNA region was involved in the genotype-specific nodulation of PI 417566. A single open reading frame (ORF) of 474 nucleotides, encoding a predicted <span class="trait">protein</span> of 158 amino acids, was identified within this region by DNA sequencing. This ORF was named noeD. Computer comparisons with available data bases revealed no significant similarities between the <span class="gene">noeD</span> DNA or predicted amino acid sequence and any known genes or their products. However, comparisons done with the region upstream of <span class="gene">noeD</span> revealed a <span class="trait">high</span> degree of similarity (about 76% similarity and 62% identity) to the N-terminal regions of the Rhizobium leguminosarum bv. viciae and R. meliloti <span class="gene">nodM</span> genes, which have been postulated to encode a glucosamine synthase. Southern hybridization analysis indicated that <span class="gene">noeD</span> is not closely linked to the main or auxiliary nodulation gene clusters in B. japonicum and that both nodulation-restricted and -unrestricted B. japonicum serogroup 110 strains contain a <span class="gene">noeD</span> homolog. High-performance liquid chromatography and fast atom bombardment-mass spectrometry analyses of the lipo-chitin oligosaccharide (LCO) nodulation signals produced by an <span class="gene">noeD</span> mutant showed a higher level of acetylation than that found with wild-type USDA 110. These results suggest that specific LCO signal molecules may be one of the factors influencing nodulation specificity in this symbiotic system.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression of the soybean <span class="gene">chlH</span> gene encoding a subunit of Mg-chelatase and localization of the Mg2+ concentration-dependent ChlH <span class="trait">protein</span> within the chloroplast.</h2>
			<h3>PMID:9588025  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nakayama M(1), Masuda T, Bando T, Yamagata H, Ohta H, Takamiya K.
				<br/> 
				年份：1998 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						From the soybean cDNA library, we isolated and analyzed the <span class="gene">chlH</span> gene encoding a subunit of Mg-chelatase. The subunit was a polypeptide of 1,383 amino acids with a molecular mass of 153,491 Da, which shared 90% identity with the olive gene from Antirrhinum majus. The regulation of the expression of <span class="gene">chlH</span> was investigated in photomix-otrophic soybean suspension cells (SB-P). The expression was light-inducible, and the induction was more rapid than those of chlI and cab2. Furthermore, the levels of the transcripts and products of <span class="gene">chlH</span> appeared to be regulated by a circadian oscillation. The subchloroplastic localization of ChlH was investigated by immunoblot analyses with antiserum against recombinant ChlH. Depending on the concentration of Mg2+ in the lysis buffer, the localization of ChlH <span class="trait">protein</span> migrated between the stroma and the envelope membrane; ChlH was localized on the envelope membrane, a major site of chlorophyll biosynthesis, when the Mg2+ concentration of the lysis buffer was <span class="trait">high</span> (above 5 mM). These results indicated that the activity of Mg-chelatase was regulated by modulation of the expression and subchloroplastic localization of ChlH <span class="trait">protein.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nitrogen utilization in pigs fed diets with soybean and rapeseed products leading to different ileal endogenous nitrogen losses.</h2>
			<h3>PMID:9498367  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grala W(1), Verstegen MW, Jansman AJ, Huisman J, Wasilewko J.
				<br/> 
				年份：1998 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen (N) balance was determined in 36 pigs (BW 24 to 30 kg) fed diets inducing different ileal endogenous N losses (ENL). We tested the hypothesis that enhanced <span class="gene">ENL</span> may be indicative of a higher recycling of endogenous proteins that will induce a greater urinary N loss and a lower efficiency of the dietary N utilization for retention. The cornstarch-based diets contained either soy concentrate (SC), soybean meal (SBM), a mixture of toasted and untoasted soybean meal (mSBM), dehulled-toasted rapeseed cake (RC1), non-dehulled-toasted rapeseed cake (RC2), or dehulled-untoasted rapeseed cake (RC3). The diets were balanced for their content of apparent ileal digestible (ID) CP (108 g/kg feed) and apparent ID of Lys, Thr, Met+Cys, Trp, and Ile. Feeding level was 2.7 times ME for maintenance per kilogram BWx75 and restricted to 88% of the requirements for ID Lys as the first-limiting amino acid. During a 5-d period, urine and feces were collected daily in metabolism cages. Compared with the SC diet (low ENL), the diets with SBM (medium ENL) and mSBM <span class="trait">(high</span> ENL) resulted in a greater (P &lt; .05) urinary N excretion. Nitrogen retention tended to be less (P = .08) in pigs fed diets that caused greater ENL. The utilization of ID N for retention in pigs fed the mSBM diet was lower (P &lt; .05) than for those fed the SC diet. There were no differences in urinary N excretion, N retention, and the utilization of ID N for retention in pigs fed the rapeseed diets of different fiber contents (hulls as the NDF source). We concluded that, at similar intakes of the first-limiting ID amino acid, N retention in pigs fed soybeans tended to be reduced by greater <span class="gene">ENL</span> as induced by antinutritional factors (e.g., trypsin inhibitors). Rapeseed hulls, as the predominant fiber source, do not affect N retention and the utilization of ID N for retention.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Kunitz-type soybean trypsin inhibitor revisited: refined structure of its complex with porcine trypsin reveals an insight into the interaction between a homologous inhibitor from Erythrina caffra and tissue-type plasminogen activator.</h2>
			<h3>PMID:9466914  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song HK(1), Suh SW.
				<br/> 
				年份：1998 Jan 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Kunitz-type trypsin inhibitor from soybean (STI) consists of 181 amino acid residues with two disulfide bridges. Its crystal structures have been determined in complex with porcine pancreatic trypsin in two crystal forms (an orthorhombic form at 1.75 A resolution and a tetragonal form at 1.9 A) and in the free state at 2.3 A resolution. They have been refined to crystallographic R-values of 18.9%, 21.6% and 19.8%, respectively. The three models of STI reported here represent a significant improvement over the partial inhibitor structure in the complex, which was previously determined at a nominal resolution of 2.6 A by the multiple isomorphous replacement method. This study provides the first high-resolution picture of the complex between a Kunitz-type proteinase inhibitor with its cognate proteinase. Many of the external loops of STI show <span class="trait">high</span> B-factors, both in the free and the complexed states, except the reactive site loop whose B-factors are dramatically reduced upon complexation. The reactive site loop of STI adopts a canonical conformation similar to those in other substrate-like inhibitors. The <span class="gene">P1</span> carbonyl group displays no out-of-plane displacement and thus retains a nominal trigonal planar geometry. Modeling studies on the complex between a homologous Kunitz-type trypsin inhibitor DE-3 from Erythrina caffra and the human tissue-type plasminogen activator reveal a new insight into the specific interactions which could play a crucial role in their binding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of two functional serine/threonine <span class="trait">protein</span> kinases from soybean that have an unusual acidic domain at the carboxy terminus.</h2>
			<h3>PMID:9267431  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yoon HW(1), Kim MC, Shin PG, Kim JS, Kim CY, Lee SY, Hwang I, Bahk JD, Hong JC, Han C, Cho MJ.
				<br/> 
				年份：1997 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two soybean cDNA clones, <span class="gene">SPK-3</span> and SPK-4, encoding putative <span class="trait">protein</span> kinases were isolated and characterized. Both cDNAs encoded approximately 40-kDa serine/threonine kinases with unusual stretches of acidic amino acids in their carboxy-terminal regions, which are highly homologous to PKABA1 from wheat and ASKs from Arabidopsis. These kinases are encoded by one- or two-copy genes in the soybean genome. Notably, <span class="gene">SPK-3</span> and -4 showed different patterns of expression in various soybean tissues. <span class="gene">SPK-3</span> is highly expressed in dividing and elongating tissues of young seedlings but relatively weakly in tissues of mature plants. In contrast, <span class="gene">SPK-4</span> showed relatively <span class="trait">high</span> and constitutive expression in all the tissues examined except for leaf tissues of mature plants. Although various stressors, such as dehydration and <span class="trait">high</span> salinity, increased the expression of both genes, the induction kinetics were different. The two genes also differed in their response to abscisic acid (ABA). <span class="gene">SPK-3</span> was induced but <span class="gene">SPK-4</span> was not affected by exogenously supplied abscisic acid. In accordance with these expression data analysis of the activity of a chimeric <span class="gene">SPK-3</span> promoter::beta-glucuronidase (GUS) reporter gene by transient expression in tobacco leaves confirmed the inducibility of <span class="gene">SPK-3</span> by salt and ABA. Polyclonal antibodies raised against a recombinant <span class="gene">SPK-4</span> <span class="trait">protein</span> produced in Escherichia coli specifically recognized both recombinant <span class="gene">SPK-3</span> and -4 proteins. Kinase assays using affinity-purified SPK-4/ antibody complexes with crude soybean extracts as substrate identified specific phosphorylation of two 41 and 170 kDa soybean proteins that were phosphorylated on serine residues. Taken together, our results suggest that SPK-3, and/or <span class="gene">SPK-4</span> are functional serine <span class="trait">protein</span> kinase(s). Furthermore, <span class="gene">SPK-3</span> and -4 may play different roles in the transduction of various environmental stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Parameters affecting the efficiency of affinity-based reversed micellar extraction and separation (ARMES) in glycoprotein purification.</h2>
			<h3>PMID:9265779  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choe J(1), VanderNoot VA, Linhardt RJ, Dordick JS.
				<br/> 
				年份：1997 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Affinity-based reversed micellar extraction and separation (ARMES) is an effective method for purifying both low and <span class="trait">high</span> molecular weight glycoproteins via liquid-liquid extraction. A range of extraction conditions were examined to gain insight into the mechanism of ARMES. Concanavalin A (Con A) was used as the model affinity ligand to bind soybean peroxidase (SBP) and beta-galactosidase as model glycoproteins. Factorial design was used to investigate the effect of various system variables on the extraction of <span class="gene">SBP</span> via ARMES. A quadratic model described the systems well, resulting in a standard deviation of 7% between calculated and experimental extraction efficiencies. Sensitivity analysis suggested that the key criteria in ARMES were the NaCl concentration and pH of the aqueous feed phase. Extraction of both glycoproteins decreased above pH 7 but fell to zero only at pH values significantly above the pI of the model glycoproteins and the Con A affinity ligand. It is proposed that the complex of the affinity lectin with the glycoprotein results in a sufficiently hydrophobic species that can be extracted into a reversed micellar organic phase even at pH's far above the pI's of the individual proteins that comprise the complex. This finding has practical considerations for the use of ARMES in the resolution and purification of <span class="trait">protein</span> glycoforms.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of human amyloid precursor <span class="trait">protein</span> ectodomains in Pichia pastoris: analysis of culture conditions, purification, and characterization.</h2>
			<h3>PMID:9226725  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Henry A(1), Masters CL, Beyreuther K, Cappai R.
				<br/> 
				年份：1997 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have examined the use of the yeast Pichia pastoris for expression of the human amyloid precursor <span class="trait">protein</span> (APP). The ectodomains of the isoforms APP695, APP751, and APP770 were expressed in both P. pastoris protease-deficient strain SMD1163 and wild-type strain GS115, using the secretion vector pHIL-S1. Expression of recombinant <span class="gene">APP</span> in each of these strains produced intact recombinant <span class="trait">protein,</span> together with a small number of breakdown products. The levels of these breakdown products were not significantly altered by expression in the protease-deficient strain compared with wild-type GS115. The effects of induction time and medium composition on recombinant <span class="gene">APP</span> stability were also examined. After optimization of expression and culture conditions, baffled shaker flask cultures of clones selected for <span class="trait">high</span> expression routinely yielded 13-24 mg/liter recombinant <span class="trait">protein</span> following a two-step purification procedure. The recombinant isoforms possessed the heparin binding, metal binding, and Kunitz-type protease inhibitor properties of human brain-derived APP. These data indicate that P. pastoris is an appropriate laboratory-scale expression system for production of sufficient quantities of recombinant <span class="gene">APP</span> for use in biological studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Feed intake and milk production in dairy cows fed whole fat soybeans].</h2>
			<h3>PMID:9735101  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preissinger W(1), Schwarz FJ, Kirchgessner M.
				<br/> 
				年份：1997
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In two experiments (E 1, E 2) two different basic diets were fed to 36 diary cows with addition of 1 or 2 kg of full fat soya beans (SB) or without SB for six weeks. The cows were fed individually. The basic diet consisted of 49% grass silage, 23% maize silage and 21% hay, supplemented with 7% manioc (% DM) in E 1 and of grass, supplemented with 2.0 kg maize silage, 3.0 kg hay and 1.5 kg manioc (kg DM/cow.d) in E 2. The basic diet used in E 1 and the grass in E 2 were offered ad libitum. Further a mixture of 1 kg SB, soya bean meal and manioc (Treatment 1), a mixture of 2 kg SB and manioc (Treatment 2) and a mixture of soya bean meal and manioc (Treatment 3, control), respectively, was supplemented to achieve diets with equivalent concentrations of energy and <span class="trait">protein.</span> Depending on the daily milk <span class="trait">yield</span> a concentrate was fed to <span class="trait">high</span> yielding cows (&gt; 21.5 kg milk in E 1 and &gt; 25.0 kg milk in E 2, respectively). The addition of 1 kg SB increased the forage intake in both experiments by 1.2-1.5 kg DM/cow.d significantly, while the addition of 2 kg SB increased the forage intake only in <span class="gene">E1</span> to the same extent. The mean total DMI was 19.2 kg in E 1 and 17.7 kg in E 2, respectively. The daily milk <span class="trait">yield</span> was not influenced by addition of SB. The average milk <span class="trait">yield</span> was 28.8 kg and 26.7 kg in E 1 and E 2, respectively. The milk fat content increased significantly in E 1 by 0.3-0.4% with no respect to the amount of the supplied SB (4.11% vs. 3.74%). In E 2 the milk fat content increased only slightly (3.84% vs. 3.74%). Milk <span class="trait">protein</span> content was not influenced in E 1, but decreased in E 2 after addition of 2 kg SB.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in Calcium-Dependent <span class="trait">Protein</span> Kinase Activity during in Vitro Tuberization in Potato.</h2>
			<h3>PMID:12226463</h3>
				

				
			<p>
				作者：MacIntosh GC(1), Ulloa RM, Raices M, Tellez-Inon MT.
				<br/> 
				年份：1996 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A soluble Ca2+-dependent <span class="trait">protein</span> kinase (CDPK) was purified to homogeneity in potato (Solanum tuberosum L.) plants. Potato <span class="gene">CDPK</span> was strictly dependent on Ca2+ (one-half maximal activation 0.6 [mu]M) and phosphorylated a wide diversity of substrates, in which Syntide 2 was the best phosphate acceptor (Michaelis constant = 30 [mu]M). The kinase was inhibited by Ca2+-chelating agents, phenotiazine derivatives, and N-(6-aminohexyl)-5-chloro-1-naphthalenesulfonamide (one-half maximal inhibition = 0.25 mM). Polyclonal antibodies directed against the regulatory region of the soybean <span class="gene">CDPK</span> recognized a 53-kD polypeptide. In an autophosphorylation assay, this same band was strongly labeled with [[gamma]-32P]ATP in the presence of Ca2+. <span class="gene">CDPK</span> activity was <span class="trait">high</span> in nontuberized plants, but increased 2.5-fold at the onset of tuber development and was reduced to one-half of its original activity when the tuber had completed formation. In the early stages of tuberization, Ca2+-dependent phosphorylation of endogenous targets (specific bands of 68, 51, and 46 kD) was observed. These polypeptides were not labeled in nontuberizing plants or in completely formed tubers, indicating that this phosphorylation is a stage-specific event. In addition, dephosphorylation of specific polypeptides was detected in tuberizing plants, suggesting the involvement of a phosphatase. Preincubation of crude extracts with phosphatase inhibitors rendered a 100% increase in <span class="gene">CDPK</span> activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An investigation of the binding of <span class="trait">protein</span> proteinase inhibitors to trypsin by electrospray ionization mass spectrometry.</h2>
			<h3>PMID:8906877  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kraunsoe JA(1), Aplin RT, Green B, Lowe G.
				<br/> 
				年份：1996 Oct 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The binding of BPTI and SBTI with trypsin has been investigated by ESI MS, using the mutant K15V-BPTI and the chemically modified RcamBPTI as controls. Although <span class="trait">high</span> cone voltages (+80 V) produce sharp spectra of BPTI, RcamBPTI, SBTI and trypsin alone, the complexes of BPTI, RcamBPTI and SBTI with trypsin undergo partial dissociation due to collisional activation. At lower cone voltages (+40 V) these non-covalent complexes are stable. The charge distribution on the trypsin and the inhibitors produced by gas phase dissociation of the complexes are markedly different from those of the components alone, indicating that ESI <span class="gene">MS</span> provides a novel probe for exploring the ionic interactions at the contact surface of proteins. Moreover, by determining the cone voltage at which the gas phase dissociation of complexes occurs it may be possible to use ESI <span class="gene">MS</span> to compare the binding energies of closely related complexes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybeans versus animal sources of rumen-undergradable <span class="trait">protein</span> and fat for early lactation dairy cows.</h2>
			<h3>PMID:8923252  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grummer RR(1), Slark K, Bertics SJ, Luck ML, Barmore JA.
				<br/> 
				年份：1996 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fifteen multiparous Holstein cows averaging 39 <span class="gene">DIM</span> were used in a replicated 5 x 5 Latin square design to compare roasted soybeans and animal by-products as supplements for increasing the RUP content of lactation diets based on alfalfa. The control diet contained roasted soybeans and was formulated to meet requirements for RUP. Rumen-undegradable <span class="trait">protein</span> in the diet was increased by including additional roasted soybeans or animal by-product proteins. To achieve diets with similar amounts of fat, we added tallow, hydrolyzed tallow fatty acids, or partially hydrogenated tallow to the control diet and to diets containing animal by-product proteins. Control diets and diets with <span class="trait">high</span> RUP were estimated to be 5.8 and 7.6% RUP (DM basis). The RDP of all diets was estimated to be approximately 11.3% of DM. Milk <span class="trait">yield</span> was increased by additional dietary RUP. <span class="trait">Yield</span> of FCM was greater for cows receiving animal byproducts than for those fed additional roasted soybeans. Cows fed additional RUP yielded more milk <span class="trait">protein,</span> but milk <span class="trait">protein</span> percentage was decreased. Milk fat percentage was reduced by additional dietary RUP, but fat <span class="trait">yield</span> was not affected. Nutrient digestibility was greatest for diets containing animal byproducts. Cows fed tallow yielded more milk, FCM, and <span class="trait">protein</span> that did those fed hydrolyzed tallow fatty acids or partially hydrogenated tallow. Lactation performance can be enhanced by supplementing animal by-products or roasted soybeans to diets based on alfalfa and formulated to meet NRC recommendations for RUP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic white clover. Studies with the auxin-responsive promoter, GH3, in root gravitropism and lateral root development.</h2>
			<h3>PMID:11539555  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Larkin PJ(1), Gibson JM, Mathesius U, Weinman JJ, Gartner E, Hall E, Tanner GJ, Rolfe BG, Djordjevic MA.
				<br/> 
				年份：1996 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We report improved method for white clover (Trifolium repens) transformation using Agrobacterium tumefaciens. <span class="trait">High</span> efficiencies of transgenic plant production were achieved using cotyledons of imbibed mature seed. Transgenic plants were recovered routinely from over 50% of treated cotyledons. The <span class="gene">bar</span> gene and phosphinothricin selection was shown to be a more effective selection system than nptII (kanamycin selection) or <span class="gene">aadA</span> (spectinomycin selection). White clover was transformed with the soybean auxin responsive promoter, GH3, fused to the GUS gene (beta-glucuronidase) to study the involvement of auxin in root development. Analysis of 12 independent transgenic plants showed that the location and pattern of GUS expression was consistent but the levels of expression varied. The level of GH3:GUS expression in untreated plants was enhanced specifically by auxin-treatment but the pattern of expression was not altered. Expression of the GH3:GUS fusion was not enhanced by other phytohormones. A consistent GUS expression pattern was evident in untreated plants presumably in response to endogenous auxin or to differences in auxin sensitivity in various clover tissues. In untreated plants, the pattern of GH3:GUS expression was consistent with physiological responses which are regarded as being auxin-mediated. For the first time it is shown that localised spots of GH3:GUS activity occurred in root cortical tissue opposite the sites where lateral roots subsequently were initiated. Newly formed lateral roots grew towards and through these islands of GH3:GUS expression, implying the importance of auxin in controlling lateral root development. Similarly, it is demonstrated for the first time that gravistimulated roots developed a rapid (within 1 h) induction of GH3:GUS activity in tissues on the non-elongating side of the responding root and this induction occurred concurrently with root curvature. These transgenic plants could be useful tools in determining the physiological and biochemical changes that occur during auxin-mediated responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel extensin gene encoding a hydroxyproline-rich glycoprotein requires sucrose for its wound-inducible expression in transgenic plants.</h2>
			<h3>PMID:8837503  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ahn JH(1), Choi Y, Kwon YM, Kim SG, Choi YD, Lee JS.
				<br/> 
				年份：1996 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel hydroxyproline-rich glycoprotein (SbHRGP3) that consists of two different domains is encoded by an extensin gene from soybean. The first domain (domain 1) located at the N terminus is composed of 11 repeats of Ser-Pro4-Lys-His-Ser-Pro4-Tyr3-His, whereas the second domain (domain 2) at the C terminus contains five repeats of Ser-Pro4-Val-Tyr-Lys-Tyr-Lys-Ser-Pro4-Tyr-Lys-Tyr-Pro-Ser-Pro5-Tyr-Lys-T yr- Pro-Ser-Pro4-Val-Tyr-Lys-Tyr-Lys. These two repeat motifs are organized in an extremely well-ordered pattern in each domain, which suggests that <span class="gene">SbHRGP3</span> belongs to a new group of proteins having the repeat motifs of two distinct groups of dicot extensins. The expression of the <span class="gene">SbHRGP3</span> gene increased with seedling maturation, and its expression was relatively <span class="trait">high</span> in the mature regions of the hypocotyl and in the root of soybean seedlings. An SbHRGP3-beta-glucuronidase (SbHRGP3-GUS) chimeric gene was constructed and expressed in transgenic tobacco plants. The expression of the SbHRGP3-GUS gene was not induced by wounding alone in transgenic tobacco plants; sucrose was also required. Expression was specific to phloem tissues and cambium cells of leaves and stems. In transgenic tobacco seedlings, SbHRGP3-GUS gene expression was activated by the maturation of the primary root and then inactivated; however, reactivation was specifically at the epidermis of the zone from which the lateral root was to be initiated. Its reactivation occurred just before the lateral root initiation. These results indicate that the <span class="gene">SbHRGP3</span> gene in different tissues responds to different signals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the early nodulin, ENOD40, in soybean roots in response to various lipo-chitin signal molecules.</h2>
			<h3>PMID:8758977  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Minami E(1), Kouchi H, Cohn JR, Ogawa T, Stacey G.
				<br/> 
				年份：1996 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The lipo-chitin (LCO) nodulation signal (nod signal) purified from Bradyrhizobium japonicum induced nodule primordia on soybean (i.e. Glycine soja) roots. These primordia were characterized by a bifurcated vascular connection, cortical cell division, and the accumulation of mRNA of the early nodulin gene, ENOD40. A chemically synthesized LCO identical in structure to the Nod signal purified from B. japonicum cultures showed the same activity when inoculated on to soybean roots. Surprisingly, synthetic LCO or chitin pentamer, inactive in inducing root hair curling (HAD) or cortical cell division (NOI) in G. soja, induced the transient accumulation of <span class="gene">ENOD40</span> mRNA. In roots inoculated with such LCO, <span class="gene">ENOD40</span> mRNA was abundant at 40 h after inoculation but decreased to the background levels 6 days after inoculation. In contrast, nod signals active in inducing HAD and NOI induced <span class="trait">high</span> levels of <span class="gene">ENOD40</span> accumulation at 40 h and 6 days after inoculation. In situ hybridization analysis showed that <span class="gene">ENOD40</span> mRNA accumulated in the pericycle of the vascular bundle at 24 h after root inoculation with nod signal. At 6 days post-inoculation with nod signal, <span class="gene">ENOD40</span> expression was seen in dividing subepidermal cortical cells. These results provide morphological and molecular evidence that nodule induction in soybean in response to purified or synthetic nod signal is similar, if not identical, to nodule formation induced by bacterial inoculation. Surprisingly, <span class="gene">ENOD40</span> mRNA accumulation occurs in response to non-specific chitin signals. This suggests that, in the case of ENOD40, nodulation specificity is not determined at the level of initial gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of ganglioside <span class="gene">GM3</span> on the activity and conformation of reconstituted Ca2+-ATPase.</h2>
			<h3>PMID:8690070  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang LH(1), Tu YP, Yang XY, Tsui ZC, Yang FY.
				<br/> 
				年份：1996 Jun 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Trace amounts of gangliosides were found in rabbit skeletal muscle sarcoplasmic reticulum and their main part was shown, by <span class="trait">high</span> performance thin layer chromatography. to be <span class="gene">GM3.</span> Addition of <span class="gene">GM3</span> to the soybean phospholipids used for reconstitution of proteoliposomes markedly increased ATP hydrolysis as well as Ca2+ uptake activity of sarcoplasmic reticulum Ca2+-ATPase incorporated into the proteoliposomes. Conformation changes of Ca2+-ATPase induced by <span class="gene">GM3</span> were also observed by intrinsic fluorescence and circular dichroism measurements.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning, expression and characterization of a blood group B active recombinant alpha-D-galactosidase from soybean (Glycine max).</h2>
			<h3>PMID:8828798  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Davis MO(1), Hata DJ, Johnson SA, Walker JC, Smith DS.
				<br/> 
				年份：1996 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA encoding soybean alpha-D-galactosidase [E.C. 3.2.1.22] was obtained by screening a soybean library with Phaseolus alpha-D-galactosidase cDNA. The Glycine max alpha-D-galactosidase cDNA is 1.75 kb long and contains untranslated 5' and 3' sequences. The deduced amino acid sequence of the soybean gene has a <span class="trait">high</span> degree of homology with other eucaryotic alpha-D-galactosidases. Recombinant alpha-D-galactosidase (rGal) was expressed in Pichia pastoris and purified by affinity chromatography. Purified rGal was homogeneous as judged by SDS-PAGE analysis with the relative molecular mass under reducing conditions of 39.8, and under nonreducing conditions 38.0 kDa. The expressed <span class="trait">protein</span> contained the sequence NGLGHTPPMG at the N-terminus, corresponding to the deduced amino acid sequence of the soybean gene. The relative native molecular mass by Sephacryl S-200 chromatography was determined to be 33.1 kDa. The specific activity was 295.6 mumoles of PNP-alpha-D-galactopyranoside hydrolyzed per mg pure rGal per min. rGal was highly specific for alpha-D-galactosyl residues. No detectable hemagglutinin or protease activity was present in the preparations. Furthermore, rGal was active against the blood group B antigen in native human erythrocyte cell suspension assays. The only detectable erythrocyte phenotypic change was loss of the B and <span class="gene">P1</span> epitopes. Consequently, recombinant Glycine max alpha-D-galactosidase may have useful biotechnical applications in the potential mass production of universally transfusable type O erythrocytes by enzymatic conversion.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A high-affinity cbb3-type cytochrome oxidase terminates the symbiosis-specific respiratory chain of Bradyrhizobium japonicum.</h2>
			<h3>PMID:8626278  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preisig O(1), Zufferey R, Thöny-Meyer L, Appleby CA, Hennecke H.
				<br/> 
				年份：1996 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been a long-standing hypothesis that the endosymbiotic rhizobia (bacteroids) cope with a concentration of 10 to 20 nM free O2 in legume root nodules by the use of a specialized respiratory electron transport chain terminating with an oxidase that ought to have a <span class="trait">high</span> affinity for O2. Previously, we suggested that the microaerobically and anaerobically induced fixNOQP operon of Bradyrhizobium japonicum might code for such a special oxidase. Here we report the biochemical characteristics of this terminal oxidase after a 27-fold enrichment from membranes of anaerobically grown B. japonicum wild-type cells. The purified oxidase has TMPD (N,N,N',N'-tetramethyl-p-phenylenediamine) oxidase activity as well as cytochrome c oxidase activity. N-terminal amino acid sequencing of its major constituent subunits confirmed that presence of the fixN,fixO, and <span class="gene">fixP</span> gene products. FixN is a highly hydrophobic, heme B-binding <span class="trait">protein.</span> FixO and FixP are membrane-anchored c-type cytochromes (apparent Mrs of 29,000 and 31,000, respectively), as shown by their peroxidase activities in sodium dodecyl sulfate-polyacrylamide gels. All oxidase properties are diagnostic for it to be a member of the cbb3-type subfamily of heme-copper oxidases. The FixP <span class="trait">protein</span> was immunologically detectable in membranes isolated from root nodule bacteroids, and 85% of the total cytochrome c oxidase activity in bacteroid membranes was contributed by the cbb3-type oxidase. The Km values for O2 of the purified enzyme and of membranes from different B. japonicum wild-type and mutant strains were determined by a spectrophotometric method with oxygenated soybean leghemoglobin as the sole O2 delivery system. The derived Km value for O2 of the cbb3-type oxidase in membranes was 7 nM, which is six- to eightfold lower than that determined for the aerobic aa3-type cytochrome c oxidase. We conclude that the cbb3-type oxidase supports microaerobic respiration in endosymbiotic bacteroids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induction and Regulation of Heat-Shock Gene Expression by an Amino Acid Analog in Soybean Seedlings.</h2>
			<h3>PMID:12226180</h3>
				

				
			<p>
				作者：Lee Y(1), Nagao RT, Lin CY, Key JL.
				<br/> 
				年份：1996 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of the proline analog azetidine-2-carboxylic acid (Aze) on the induction and the regulation of heat-shock (HS) mRNA accumulation and heat-shock <span class="trait">protein</span> (HSP) synthesis in soybean (Glycine max) seedlings was studied. Treatment with Aze elicited an HS-like response at the normal growth temperature, 28[deg]C, with seven of nine HS cDNA clones tested. Two cDNA clones, <span class="gene">Gm-Hsp22.5</span> and pFS2033, share 78% identity; however, transcripts hybridizing to <span class="gene">GmHsp22.5</span> but not pFS2033 accumulated with Aze treatment at 28[deg]C. Substantial incorporation of radioactive amino acid into <span class="trait">high</span> molecular weight HSPs but not low molecular weight HSPs was observed in vivo during Aze treatment at 28[deg]C. Low molecular weight HSPs were detected using antibodies raised against an abundant member of low molecular weight class I HSPs, indicating that low molecular weight HSPs were synthesized at normal growth temperatures during Aze treatment despite a lack of substantial in vivo radioactive amino acid incorporation. In summary, Aze treatment induced accumulation of most but not all HS mRNAs and HSPs in soybean seedlings; the observations presented here suggest differential regulation among various HS genes at the transcriptional and posttranscriptional levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The alternative oxidase is encoded in a multigene family in soybean.</h2>
			<h3>PMID:8580775  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Whelan J(1), Millar AH, Day DA.
				<br/> 
				年份：1996
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The copy number of the alternative oxidase gene, Aox, was investigated in soybean (Glycine max L.) using a Polymerase chain reaction (PCR) approach to amplify fragments from soybean genomic DNA. The primers used were based on absolutely conserved regions of Aox cDNA clones from a variety of plant species and the yeast Hansenula anomala. After subcloning of the 170-bp PCR products, 12 individual colonies were sequenced. Eleven plasmids yielded inserts representing three sequences in the ratio 4:3:4 (Aox1-3). The sequence of <span class="gene">Aox1</span> was 100% identical at the nucleic acid level to the published full-length cDNA from soybean. The other two sequences were 60-75% identical to <span class="gene">Aox1</span> and to each other at the nucleic acid and <span class="trait">protein</span> levels. Similar analysis of Nicotiana tabacum L. revealed an additional gene copy with <span class="trait">high</span> homology to the soybean Aox2 sequence. Genomic DNA from soybean cut with Hind III and probed with the full-length <span class="gene">Aox1</span> yielded a single positive band of 6.5 kb; when the same genomic blot was probed with a mixture of all three PCR fragments, three bands of 9 kb, 6.5 kb and 3 kb were detected. Reverse transcription-PCR performed on total RNA from various soybean tissues, followed by hybridisation with the three Aox sequences individually, revealed differential expression of the Aox genes between cotyledons and leaves. It is suggested that soybean contains a multigene Aox family. The implication of this for the understanding of alternative oxidase expression and regulation in plant tissues is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete nucleotide sequence and genome organization of bean common mosaic virus (NL3 strain).</h2>
			<h3>PMID:8607279  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fang GW(1), Allison RF, Zambolim EM, Maxwell DP, Gilbertson RL.
				<br/> 
				年份：1995 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nucleotide sequences of 3 cDNA clones corresponding to entire RNA genome of bean common mosaic virus NL3 strain have been determined. The RNA is 9612 nucleotides long, excluding a 3'-terminal poly(A) tail. A putative start codon located at nucleotide positions 170-172 initiates one large open reading frame that is terminated with a UAA codon at position 9368-9370. The predicted <span class="gene">polyprotein</span> has 3066 amino acids and an M(r) of 340.3 kDa. The positions of putative <span class="trait">protein</span> cleavage sites have been determined by analogy to consensus sequences in other potyviruses. The nucleotide sequences of the non-translated regions and the predicted amino acid sequences of BCMV NL3, were compared with those of other potyviruses. Comparison of the BCMV NL3 proteins with those of other potyviruses indicated a similar genomic organization, and <span class="trait">high</span> percentage of amino acid sequence identity in the cylindrical inclusion <span class="trait">protein,</span> nuclear inclusion 'b' <span class="trait">protein</span> and coat <span class="trait">protein.</span> BCMV NL3 displays the highest amino acid sequence identity with soybean mosaic virus.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of orally administered liposomes with soybean-derived sterols and their glucosides on rat body weight.</h2>
			<h3>PMID:8593478  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Maitani Y(1), Hazama M, Tojo H, Qi XR, Nagai T.
				<br/> 
				年份：1995 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects on Wistar rat body weight were examined after a single subcutaneous (s.c.) or oral (p.o.) administration of dipalmitoylphosphatidylcholine (DPPC) liposomes composed of soybean-derived sterols (SS) and their glucosides (SG) with or without entrapping recombinant human erythropoietin (Epo) for 1 week. Body weight increased significantly after both types of administration compared with the control groups irrespective of the existence of Epo. The neutral lipid concentration in plasma increased with the increase in body weight whereas the total contents of cholesterol and <span class="trait">high</span> density lipoprotein cholesterol in the plasma did not change significantly. The <span class="gene">SS</span> and SG suspensions following p.o. administration, however, did not alter the body weight. These findings suggest that liposomal <span class="gene">SS</span> and SG may be absorbed through the intestinal membrane and induce a change in the uptake of lipid, in contrast to the suspension state. <span class="gene">SS</span> in liposomes significantly increased body weight more than SG after p.o. administration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification, characterization and substrate specificity of rat pancreatic elastase II.</h2>
			<h3>PMID:7647093  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Szilagyi CM(1), Sarfati P, Pradayrol L, Morisset J.
				<br/> 
				年份：1995 Aug 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A somatostatin-14-degrading activity has been purified to homogeneity from rat pure pancreatic juice. This proteinase was concentrated more than 350-fold in a four-step procedure including ion-exchange and gel filtration. The final preparation contained a single <span class="trait">protein</span> with a molecular weight (M(r)) of approx. 29,000, as determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis. The determination of its NH2-terminal sequence led us to conclude that the purified proteinase corresponds to the rat pancreatic elastase II predicted from the cDNA clone isolated by MacDonald in 1982. This anionic proteinase exhibits an isoelectric point of 5.6 and does not contain any carbohydrate moieties in its structure. The proteinase is sensitive to the trypsin inhibitors soybean trypsin inhibitor and N alpha-tosyl-L-lysine-chloromethyl ketone and also to 3,4-dichloroisocoumarin, a general elastase inhibitor. The cleavage products obtained after hydrolysis of somatostatin-14 by the purified elastase, were separated by reversed phase <span class="trait">high</span> performance liquid chromatography and identified by amino-acid analysis. The primary hydrolysis was trypsin-like and consisted in an opening of the cyclic structure of somatostatin-14 after the Lys-9 residue leading to the formation of a Y-shaped peptide with the same amino-acid composition as the native peptide. The initial 'trypsin-like specificity' was not observed during the secondary hydrolysis of the Y-shaped peptide; indeed the proteinase seemed more specific for a certain motif in the native peptide rather than for a specific class of amino acid, this last kind of selectivity is commonly observed with trypsin and chymotrypsin. In order to establish that the proteinase possesses an extended recognition site on the substrate rather than a specificity for a class of amino acid, the substrate specificity of the rat pancreatic elastase II was investigated with a series of para-nitroanilide peptides. The proteinase exhibits a large specificity involving peptide chain of at least four amino acids with a preference for bulky residue in <span class="gene">P1</span> or P2. The Km values of 89 microM and 1567 microM obtained for somatostatin-14 and Suc-Ala-Ala-Pro-Met-pNA, respectively, indicate that elastase II has a greater affinity for the natural substrate than for synthetics. This last observation along with the substrate specificity of the proteinase leads us to propose that elastase II could be specifically involved in the regulation of biological functions of somatostatin-14 in the gastrointestinal tract.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of cDNA clones encoding valosin-containing <span class="trait">protein</span> and other plant plasma membrane-associated proteins by a general immunoscreening strategy.</h2>
			<h3>PMID:7753826  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shi J(1), Dixon RA, Gonzales RA, Kjellbom P, Bhattacharyya MK.
				<br/> 
				年份：1995 May 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An approach was developed for the isolation and characterization of soybean plasma membrane-associated proteins by immunoscreening of a cDNA expression library. An antiserum was raised against purified plasma membrane vesicles. In a differential screening of approximately 500,000 plaque-forming units with the anti-(plasma membrane) serum and DNA probes derived from highly abundant clones isolated in a preliminary screening, 261 clones were selected from approximately 1,200 antiserum-positive plaques. These clones were classified into 40 groups by hybridization analysis and 5'- and 3'-terminal sequencing. By searching nucleic acid and <span class="trait">protein</span> sequence data bases, 11 groups of cDNAs were identified, among which valosin-containing <span class="trait">protein</span> (VCP), clathrin heavy chain, phospholipase C, and S-adenosylmethionine:delta 24-sterol-C-methyltransferase have not to date been cloned from plants. The remaining 29 groups did not match any current data base entries and may, therefore, represent additional or yet uncharacterized genes. A full-length cDNA encoding the soybean VCP was sequenced. The <span class="trait">high</span> level of amino acid identity with vertebrate VCP and yeast <span class="gene">CDC48</span> <span class="trait">protein</span> indicates that the soybean <span class="trait">protein</span> is a plant homolog of vertebrate VCP and yeast <span class="gene">CDC48</span> <span class="trait">protein.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification and characterization of tissue kallikrein-like proteinases from the black sea bass (Centropristis striata) and the southern frog (Rana berlandieri).</h2>
			<h3>PMID:7656186  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Richards GP(1), Chao J, Chung P, Chao L.
				<br/> 
				年份：1995 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Serine proteinases were isolated from the pyloric caeca of the black sea bass (Centropristis striata) and the pancreas of the Southern frog (Rana berlandieri) and were purified to apparent homogeneity by aprotinin affinity column chromatography, reverse phase <span class="trait">high</span> performance liquid chromatography and gel filtration FPLC liquid chromatography to produce products with molecular masses of approximately 27,000 Da and isoelectric points from 4.2 to 5.0. Both enzymes were kallikrein-like and were bound by diisopropylfluorophosphate; had pH optima from 9 to 10; showed <span class="trait">high</span> specificity for the hydrolysis of arginine peptide bonds and low to moderate affinity for lysine bonds at the <span class="gene">P1</span> substrate recognition sites; were inhibited by aprotinin, benzamidine, leupeptin, and soybean trypsin inhibitor; generated kinin from kininogen and were highly stable at room temperature. Differences between the enzymes were observed relative to their hydrophobicities, substrate specificities, stabilities at acidic pHs in the presence and absence of calcium, and the amounts of kinin generated from kininogen. Many of the fish trypsins, previously identified as anionic trypsins, may actually be more kallikrein-like.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of gmsti, a stress-inducible gene from soybean (Glycine max) coding for a <span class="trait">protein</span> belonging to the TPR (tetratricopeptide repeats) family.</h2>
			<h3>PMID:7766904  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hernández Torres J(1), Chatellard P, Stutz E.
				<br/> 
				年份：1995 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In close vicinity of two <span class="gene">fus</span> nuclear genes (chloroplast-specific translation elongation factor cEF-G) of soybean (Glycine max) we localized a split nuclear gene coding for a <span class="trait">protein</span> with tetratricopeptide repeats (TPR). A full-length cDNA was sequenced (1871 nucleotides). It encodes a <span class="trait">protein</span> (569 amino acids) with <span class="trait">high</span> sequence identity to the yeast STI1 stress-inducible and the human transformation-sensitive IEF SSP 3521 <span class="trait">protein</span> which both carry TPR elements. The soybean gene is heat-inducible. This is the first evidence for the existence of plant genes coding for proteins which belong to the TPR family. We call the gene gmsti and the <span class="trait">protein</span> <span class="gene">GMSTI</span> in analogy to the yeast counterpart.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Duodenase, a new serine protease of unusual specificity from bovine duodenal mucosa. Purification and properties.</h2>
			<h3>PMID:7867648  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zamolodchikova TS(1), Vorotyntseva TI, Antonov VK.
				<br/> 
				年份：1995 Feb 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this paper, data are presented on purification and properties of a new serine endopeptidase (duodenase) isolated from bovine duodenum mucosa. The enzyme has been purified to homogeneity by combinations of ammonium sulphate fractionation, carboxymethyl-cellulose 52 chromatography, and affinity chromatography on Sepharose 4B with Kunitz soybean trypsin inhibitor as a ligand. Some physicochemical properties of this protease have been investigated. The molecular mass of the purified duodenase was determined to be 29 +/- 0.5 kDa by SDS/PAGE and G-2000 SW column chromatography. The enzyme molecule is a single chain and the native enzyme is a monomeric <span class="trait">protein.</span> Its isoelectric point was estimated to be 10 +/- 0.2. Duodenase has two forms (I and II) which possess similar properties but differ in their amino acid composition. The new protease is a glycoprotein and contains approximately 3.5% sugars. The enzyme displays trypsin-like and chymotrypsin-like activities and hydrolyzes the amide bonds of substrates having Lys, Arg, Tyr, Phe and Leu residues at the <span class="gene">P1</span> position. Duodenase is most active at pH 7.9-8.2. Duodenase was irreversibly inhibited by diisopropylphosphofluoridate and phenylmethanesulphonyl fluoride, indicative of an active-site serine in this protease. alpha-N-Tosyl-L-lysine chloromethane and alpha-N-tosyl-L-phenylalanine chloromethane, which react with an active His, caused marked inhibition of trypsin-like and chymotrypsin-like activities of duodenase. The enzyme activity was strongly suppressed by trypsin inhibitors from different sources (soybeans, bovine lungs and Lima beans). Chicken egg white ovomucoid had no effect on the duodenase activity. The N-terminal sequence of the native duodenase (24 amino acid residues) shows <span class="trait">high</span> similarity with those of human and murine cytotoxic T-lymphocyte granzymes, human leukocyte cathepsin G and rat mast cell chymases. The biological role of duodenase is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:cDNA cloning and sequence analysis of <span class="gene">NIb</span> gene of soybean mosaic virus.</h2>
			<h3>PMID:7755870  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu J(1), Peng X, Mang K.
				<br/> 
				年份：1995 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						cDNA of soybean mosaic virus (Beijing isolate, SMV-BJ) has been synthesized, using viral genomic RNA as a template and random hexanucleotides as primers. Based on the sequences of SMV-BJ coat <span class="trait">protein</span> (CP) gene as well as SMV- and WMV-II-related regions, oligonucleotides were made as primers for polymerase chain reaction (PCR). <span class="gene">NIb</span> gene of SMV-BJ was amplified by PCR, and cloned into pBluescript SK. The complete sequence was determined. The comparison of <span class="gene">NIb</span> genes between SMV-BJ and WMV-II (USA) shows that similarities for nucleotide sequence reach 80.3%; and the deduced amino acid sequence, 91.3%. In consideration of the <span class="trait">high</span> identities in between the CP gene and the 3'-non-coding region between them, WMV-II might be considered as a watermelon strain of SMV. Besides, some unexpected sequences were found in the 3'-region of 2 <span class="gene">NIb</span> gene clones. Following modification and splicing, a binary vector of <span class="gene">NIb</span> gene has been constructed for its expression in higher plant for the purpose of studying the possible replicase-mediated <span class="trait">resistance</span> in polyviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Alternative oxidase from mango (Mangifera indica, L.) is differentially regulated during fruit ripening.</h2>
			<h3>PMID:8555961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cruz-Hernández A(1), Gómez-Lim MA.
				<br/> 
				年份：1995
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Alternative oxidase is a respiratory-chain component of higher plants and fungi that catalyzes cyanide-resistant oxygen consumption. The activity of a alternative oxidase has been detected during ripening in several climacteric fruit including mango (Mangifera indica L.). Synthetic oligonucleotides, corresponding to conserved regions of the Sauromatum guttatum and Arabidopsis thaliana nucleotide sequences, were used as primers for polymerase chain reaction to amplify genomic DNA extracted from mango leaves. The 623-bp fragment was found to encode an open reading frame of 207 amino acids showing <span class="trait">high</span> identity to the S. guttatum enzyme. Using this fragment to screen a ripe mango mesocarp cDNA library, one full-length cDNA clone, designated pAOMI.1, was obtained that contained an open reading frame encoding a polypeptide of 318 amino acids. The predicted amino-acid sequence exhibited 62, 64 and 68% identity to the S. guttatum, soybean, and A. thaliana enzymes respectively, indicating that this cDNA encodes a mango homologue of the alternative oxidase. Gel blot hybridization showed that pAOMI.1 is likely to be encoded by a single-copy gene. The 1.6 kb-transcript was induced during mango fruit ripening although the transcript was clearly detectable in unripe and developing fruit. Antibodies raised against the S. guttatum enzyme recognized three bands of approximately 27, approximately 33 and approximately 36 kDa from mitochondrial mango proteins. Two of the bands were detectable before ripening and increase in ripe fruit, the other band (27 kDa) was barely present in unripe fruit but accumulated during ripening. The clone pAOMI.1 was able to complement an Escherichia coli <span class="gene">hemA</span> mutant deficient in cytochrome-mediated aerobic respiration. This is the first report on the analysis of alternative oxidase at the molecular level during the ripening of a climacteric fruit.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glyoxysomal malate dehydrogenase and malate synthase from soybean cotyledons (Glycine max L.): enzyme association, antibody production and cDNA cloning.</h2>
			<h3>PMID:8547819  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guex N(1), Henry H, Flach J, Richter H, Widmer F.
				<br/> 
				年份：1995
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In order to investigate a possible association between soybean malate synthase (MS; L-malate glyoxylate-lyase, CoA-acetylating, EC 4.1.3.2) and glyoxysomal malate dehydrogenase (gMDH; (S)-malate: NAD+ oxidoreductase, EC 1.1.1.37), two consecutive enzymes in the glyoxylate cycle, their elution profiles were analyzed on Superdex 200 HR fast <span class="trait">protein</span> liquid chromatography columns equilibrated in low- and high-ionic-strength buffers. Starting with soluble proteins extracted from the cotyledons of 5-d-old soybean seedlings and a 45% ammonium sulfate precipitation, <span class="gene">MS</span> and gMDH coeluted on Superdex 200 HR (low-ionic-strength buffer) as a complex with an approximate relative molecular mass (Mr) of 670,000. Dissociation was achieved in the presence of 50 mM KCl and 5 mM MgCl2, with the elution of <span class="gene">MS</span> as an octamer of M(r) 510,000 and of gMDH as a dimer of M(r) 73,000. Polyclonal antibodies raised to the native copurified enzymes recognized both denatured <span class="gene">MS</span> and gMDH on immunoblots, and their native forms after gel filtration. When these antibodies were used to screen a lambda ZAP II expression library containing cDNA from 3-d-old soybean cotyledons, they identified seven clones encoding gMDH, whereas ten clones encoding <span class="gene">MS</span> were identified using an antibody to SDS-PAGE-purified MS. Of these cDNA clones a 1.8 kb clone for <span class="gene">MS</span> and a 1.3-kb clone for gMDH were fully sequenced. While 88% identity was found between mature soybean gMDH and watermelon gMDH, the N-terminal transit peptides showed only 37% identity. Despite this low identity, the soybean gMDH transit peptide conserves the consensus R(X6)HL motif also found in plant and mammalian thiolases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:enod40, a gene expressed during nodule organogenesis, codes for a non-translatable RNA involved in plant growth.</h2>
			<h3>PMID:7957074  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Crespi MD(1), Jurkevitch E, Poiret M, d'Aubenton-Carafa Y, Petrovics G, Kondorosi E, Kondorosi A.
				<br/> 
				年份：1994 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizobium meliloti can interact symbiotically with Medicago plants, thereby inducing root nodules. However, certain Medicago plants can form nodules spontaneously, in the absence of rhizobia. A differential screening was performed using spontaneous nodule versus root cDNAs from Medicago sativa ssp. varia. Transcripts of a differentially expressed clone, Msenod40, were detected in all differentiating cells of nodule primordia and spontaneous nodules, but were absent in fully differentiated cells. Msenod40 showed homology to a soybean early nodulin gene, <span class="gene">Gmenod40,</span> although no significant open reading frame (ORF) or coding capacity was found in the Medicago sequence. Furthermore, in the sequences of cDNAs and a genomic clone (Mtenod40) isolated from Medicago truncatula, a species containing a unique copy of this gene, no ORFs were found either. In vitro translation of purified Mtenod40 transcripts did not reveal any <span class="trait">protein</span> product. Evaluation of the RNA secondary structure indicated that both msenod40 and <span class="gene">Gmenod40</span> transcripts showed a <span class="trait">high</span> degree of stability, a property shared with known non-coding RNAs. The Mtenod40 RNA was localized in the cytoplasm of cells in the nodule primordium. Infection with Agrobacterium tumefaciens strains bearing antisense constructs of Mtenod40 arrested callus growth of Medicago explants, while overexpressing Mtenod40 embryos developed into teratomas. These data suggest that the enod40 genes might have a role in plant development, acting as 'riboregulators', a novel class of untranslated RNAs associated with growth control and differentiation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">AAPT1</span> gene of soybean complements a cholinephosphotransferase-deficient mutant of yeast.</h2>
			<h3>PMID:7994181  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dewey RE(1), Wilson RF, Novitzky WP, Goode JH.
				<br/> 
				年份：1994 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aminoalcoholphosphotransferases (AAPTases) utilize diacylglycerols and cytidine diphosphate (CDP)-aminoalcohols as substrates in the synthesis of the abundant membrane lipids phosphatidylcholine and phosphatidylethanolamine. A soybean cDNA encoding an AAPTase that demonstrates <span class="trait">high</span> levels of CDP-choline:sn-1,2-diacylglycerol cholinephosphotransferase activity was isolated by complementation of a yeast strain deficient in this function and was designated AAPT1. The deduced amino acid sequence of the soybean cDNA showed nearly equal similarity to each of the two characterized AAPTase sequences from yeast, cholinephosphotransferase and ethanolaminephosphotransferase (CDP-ethanolamine:sn-1,2-diacylglycerol ethanolaminephosphotransferase). Moreover, assays of soybean AAPT1-encoded enzyme activity in yeast microsomal membranes revealed that the addition of CDP-ethanolamine to the reaction inhibited incorporation of 14C-CDP-choline into phosphatidylcholine in a manner very similar to that observed using unlabeled CDP-choline. Although DNA gel blot analysis suggested that AAPT1-like sequences are represented in soybean as a small multigene family, the same <span class="gene">AAPT1</span> isoform isolated from a young leaf cDNA library was also recovered from a developing seed cDNA library. Expression assays in yeast using soybean <span class="gene">AAPT1</span> cDNAs that differed only in length suggested that sequences in the 5'leader of the transcript were responsible for the negative regulation of gene activity in this heterologous system. The inhibition of translation mediated by a short open reading frame located 124 bp upstream of the <span class="gene">AAPT1</span> reading frame is one model proposed for the observed down-regulation of gene activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Novel N-terminal sequence of a glycine-rich <span class="trait">protein</span> in the aleurone layer of soybean seeds.</h2>
			<h3>PMID:7765520  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matsui M(1), Toyosawa I, Fukuda M.
				<br/> 
				年份：1994 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine-rich <span class="trait">protein</span> (GRP), a cell wall <span class="trait">protein,</span> was extracted with hot water from the aleurone layer of soybean seeds. <span class="gene">GRP</span> was purified by adsorption on DEAE-Sephadex, Sephadex G-100 gel chromatography, and anion exchange HPLC. The estimated molecular size of <span class="gene">GRP</span> was approximately 30 kDa and <span class="gene">GRP</span> contained 59% glycine and 15% serine. The N-terminal amino acid sequence was a novel Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-Gly-.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Kunitz domain inhibitors of tissue factor-factor VIIa. II. Potent and specific inhibitors by competitive phage selection.</h2>
			<h3>PMID:8071338  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dennis MS(1), Lazarus RA.
				<br/> 
				年份：1994 Sep 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Active-site inhibitors of the human tissue factor-Factor VIIa complex (TF.FVIIa) that are both potent and specific have been selected from Alzheimer's amyloid beta-protein precursor inhibitor (APPI) Kunitz domain libraries using a strategy termed competitive phage selection. Phage display Libraries I-III, previously sorted by direct selection, were sorted by competitive selection against immobilized TF.FVIIa in which increasing amounts of Factor XIa (FXIa) were added to the selection buffer to remove phage with <span class="trait">high</span> affinity for FXIa. The most striking difference in the selected Kunitz domain sequences was at the P4' position where Lys was highly preferred instead of Leu which was found here by direct selection (Dennis, M. S., and Lazarus, R. A. (1994) J. Biol. Chem. 269, 22129-22136). In addition, both Lys and Arg were selected in the <span class="gene">P1</span> position as opposed to Arg alone. A new library (Library IV) was constructed which contained all previously observed amino acids at 9 positions in the primary and secondary binding loops of APPI. Comparable results were obtained by sorting Library IV against immobilized TF.FVIIa in the presence of either FXIa alone or FXIa, plasma kallikrein, and plasmin. The Kunitz domains obtained by either competitive selection strategy potently inhibited TF.FVIIa, with Ki* values ranging from about 2 to 20 nM; the Ki* values were generally &gt; 1 microM for FXIa and plasma kallikrein and ranged from 4 to 200 nM for plasmin. Variants selective for TF.FVIIa were assayed for free FVIIa and FXa inhibitory activity and characterized in coagulation assays. A rationale for the selection of Lys at both the P4' and <span class="gene">P1</span> positions based upon comparison of sequences and structures of relevant serine proteases and inhibitors is presented.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two genes that encode Ca(2+)-dependent <span class="trait">protein</span> kinases are induced by <span class="trait">drought</span> and high-salt stresses in Arabidopsis thaliana.</h2>
			<h3>PMID:8078458  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Urao T(1), Katagiri T, Mizoguchi T, Yamaguchi-Shinozaki K, Hayashida N, Shinozaki K.
				<br/> 
				年份：1994 Aug 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two cDNA clones, cATCDPK1 and cATCDPK2, encoding Ca(2+)-dependent, calmodulin-independent <span class="trait">protein</span> kinases (CDPK) were cloned from Arabidopsis thaliana and their nucleotide sequences were determined. Northern blot analysis indicated that the mRNAs corresponding to the ATCDPK1 and ATCDPK2 genes are rapidly induced by <span class="trait">drought</span> and high-salt stress but not by low-temperature stress or heat stress. Treatment of Arabidopsis plants with exogenous abscisic acid (ABA) had no effect on the induction of ATCDPK1 or ATCDPK2. These findings suggest that a change in the osmotic potential of the environment can serve as a trigger for the induction of ATCDPK1 and ATCDPK2. Putative proteins encoded by ATCDPK1 and ATCDPK2 which contain open reading frames of 1479 and 1488 bp, respectively, are designated ATCDPK1 and ATCDPK2 and show 52% identity at the amino acid sequence level. ATCDPK1 and ATCDPK2 exhibit significant similarity to a soybean <span class="gene">CDPK</span> (51% and 73%, respectively). Both proteins contain a catalytic domain that is typical of serine/threonine <span class="trait">protein</span> kinases and a regulatory domain that is homologous to the Ca(2+)-binding sites of calmodulin. Genomic Southern blot analysis suggests the existence of a few additional genes that are related to ATCDPK1 and ATCDPK2 in the Arabidopsis genome. The ATCDPK2 <span class="trait">protein</span> expressed in Escherichia coli was found to phosphorylate casein and myelin basic <span class="trait">protein</span> preferentially, relative to a histone substrate, and required Ca2+ for activation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation of multiple isoforms of alpha-fibrinogenase from the Western diamondback rattlesnake, Crotalus atrox: N-terminal sequence homology with ancrod, an antithrombotic agent from Malayan viper.</h2>
			<h3>PMID:8024586  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hung CC(1), Chiou SH.
				<br/> 
				年份：1994 Jun 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Three fibrinogenolytic proteases from the rattlesnake (Crotalus atrox) venom were isolated and purified from multiple-step chromatographies including ion-exchange chromatography, gel permeation and reversed-phase HPLC. The fractions were shown to be homogeneous as judged by SDS-gel electrophoresis. In vitro, they also showed a <span class="trait">high</span> and specific proteolytic activity against alpha-chains of fibrinogen molecules. Further characterization of these purified fractions with fibrinogenase activity indicated that they are all single-chain proteins with similar molecular weights of about 33,000-35,000. Their stability at <span class="trait">high</span> temperatures was examined and the cleavage specificity of various proteases was studied using oxidized insulin B-chains as substrates. The reactivity toward fibrinogen molecules in the presence of varied protease inhibitors including EDTA, beta-mercaptoethanol, soybean trypsin inhibitor, aprotinin and phenylmethanesulfonyl fluoride was also investigated in order to classify these proteases based on their reaction mechanisms. Amino acid analysis indicates that these purified venom toxins possess closely-similar compositions with regard to most amino acids including cysteine, methionine and tyrosine. N-Terminal sequence analysis of these proteases revealed that they are similar to ancrod, an antithrombotic agent isolated from the Malayan <span class="gene">pit</span> viper. This study points to the existence of a family of novel ancrod-like fibrinogenases in crotalid rattlesnakes, which may be useful as effective venom-based anticoagulants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nucleotide sequence homology of cDNAs encoding soybean Bowman-Birk type proteinase inhibitor and its isoinhibitors.</h2>
			<h3>PMID:7764974  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Baek JM(1), Song JC, Choi YD, Kim SI.
				<br/> 
				年份：1994 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The sequences of cDNA clones encoding the Bowman-Birk proteinase inhibitor (BBPI) and its isoinhibitors (D-II and C-II) have been identified. Nucleotide sequence homologies among these clones and between the two reactive domains in each clone were very <span class="trait">high.</span> These homologies suggest that the BBPI and its isoinhibitor genes may have evolved from a common ancestral inhibitor. Also, clone pB2 (D-II) had two identical reactive sites that inhibit trypsin only, and three repetitive regions that are about 81 bp and initiated with ATG. It was assumed from these results that isoinhibitor <span class="gene">D-II</span> was the most primordial type of these inhibitors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure and light-regulated expression of the gsa gene encoding the chlorophyll biosynthetic enzyme, glutamate 1-semialdehyde aminotransferase, in Chlamydomonas reinhardtii.</h2>
			<h3>PMID:8155881  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matters GL(1), Beale SI.
				<br/> 
				年份：1994 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The gsa gene, which encodes glutamate 1-semialdehyde (GSA) aminotransferase (GSAT), an enzyme in the chlorophyll and heme biosynthetic pathway, has been cloned from Chlamydomonas reinhardtii by complementation of an Escherichia coli <span class="gene">hemL</span> mutant. The deduced C. reinhardtii GSAT amino acid sequence has a <span class="trait">high</span> degree of similarity to GSAT sequences from barley, tobacco, soybean and various prokaryotic sources. In vitro enzyme activity assays from E. coli transformed with the C. reinhardtii GSAT cDNA showed that higher levels of GSAT activity are associated with the expression of the cDNA insert. Analysis of changes in mRNA levels in light:dark synchronized C. reinhardtii cultures was done by northern blotting. The level of GSAT mRNA nearly doubled during the first 0.5 h in the light and increased over 26-fold after 2 h in the light. This increase is comparable to previously reported increases in GSAT activity in dark-grown cultures transferred to the light, and is the first report of induction by light of a gene encoding an ALA biosynthetic enzyme in plant or algal cells. The accumulation of GSAT mRNA follows the pattern of chlorophyll accumulation and the pattern of chlorophyll a/b-binding <span class="trait">protein</span> (cabII-1) mRNA accumulation in these cells, suggesting that the two genes may be regulated by light through a common mechanism. Additional evidence that the GSAT mRNA may be transcriptionally regulated by light is found in the genomic sequence of the gsa gene.(ABSTRACT TRUNCATED AT 250 WORDS)
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:PF1: an A-T hook-containing DNA binding <span class="trait">protein</span> from rice that interacts with a functionally defined d(AT)-rich element in the oat phytochrome A3 gene promoter.</h2>
			<h3>PMID:8148649  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nieto-Sotelo J(1), Ichida A, Quail PH.
				<br/> 
				年份：1994 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytochrome-imposed down-regulation of the expression of its own phytochrome A gene (PHYA) is one of the fastest light-induced effects on transcription reported in plants to date. Functional analysis of the oat PHYA3 promoter in a transfection assay has revealed two positive elements, PE1 and PE3, that function synergistically to support <span class="trait">high</span> levels of transcription in the absence of light. We have isolated a rice cDNA clone (pR4) encoding a DNA binding <span class="trait">protein</span> that binds to the AT-rich PE1 element. We tested the selectivity of the pR4-encoded DNA binding activity using linker substitution mutations of PE1 that are known to disrupt positive expression supported by the PHYA3 promoter in vivo. Binding to these linker substitution mutants was one to two orders of magnitude less than to the native PE1 element. Because this is the behavior expected of positive factor 1 (PF1), the presumptive nuclear transcription factor that acts in trans at the PE1 element in vivo, the data support the conclusion that the <span class="trait">protein</span> encoded by pR4 is in fact rice PF1. The PF1 polypeptide encoded by pR4 is 213 amino acids long and contains four repeats of the A-T hook DNA binding motif found in high-mobility group I-Y (HMGI-Y) proteins. In addition, PF1 contains an 11-amino acid-long hydrophobic region characteristic of HMG I proteins, its N-terminal region shows strong similarities to a pea <span class="gene">H1</span> histone sequence and a short peptide sequence from wheat HMGa, and it shows a <span class="trait">high</span> degree of similarity along its entire length to the HMG Y-like <span class="trait">protein</span> encoded by a soybean cDNA, SB16. In vitro footprinting and quantitative gel shift analyses showed that PF1 binds preferentially to the PE1 element but also at lower affinity to two other AT-rich regions upstream of PE1. This feature is consistent with the binding characteristics of HMG I-Y proteins that are known to bind to most runs of six or more AT base pairs. Taken together, the properties of PF1 suggest that it belongs to a newly described family of nuclear proteins containing both histone <span class="gene">H1</span> domains and A-T hook DNA binding domains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and characterization of cDNAs encoding oat PF1: a <span class="trait">protein</span> that binds to the PE1 region in the oat phytochrome A3 gene promoter.</h2>
			<h3>PMID:7639786  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nieto-Sotelo J(1), Quail PH.
				<br/> 
				年份：1994
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In monocotyledons, the expression of the oat phytochrome A gene (PHYA) is down-regulated by phytochrome itself. This autoregulatory repression is the most rapid light-induced effect on gene expression reported in plants to date. A functional analysis of the oat PHYA3 gene minimal promoter in a rice transient expression assay has identified two promoter elements, PE1 and PE3, that interact synergistically in positive regulation. We have isolated an oat cDNA clone (pO2) that encodes a DNA-binding <span class="trait">protein</span> that binds to the PE1 region of the oat PHYA3 gene promoter. The in vitro binding properties of the pO2-encoded <span class="trait">protein,</span> towards DNA probes containing either the PE1 sequence or linker-substitution mutations in PE1, correlate with the activity of these DNA elements in the rice transient expression assay. These mutations are known to abolish expression of a reporter gene in vivo. Binding of these linker-substitution mutants to the pO2-encoded <span class="trait">protein</span> in vitro was lower by one to two orders of magnitude than the binding of the native PE1 region. We suggest, therefore, that the pO2 clone may encode the putative nuclear factor, oat PF1, that is involved in positive regulation of PHYA3 by binding to PE1 in vivo. pO2 encodes a 170-amino-acid-long <span class="trait">protein</span> that contains three repeats of the 'AT-hook' DNA-binding motif found in <span class="trait">high</span> mobility group I-Y (HMGI-Y) proteins. Oat PF1 is highly similar to rice PF1 and to the <span class="trait">protein</span> encoded by soybean cDNA SB16. They all have a strong similarity in their N-terminus to the pea <span class="gene">H1</span> histone, and the presence of several AT-hook DNA-binding motifs in their C-terminal halves.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Seed Coat Peroxidase (A Comparison of High-Activity and Low-Activity Genotypes).</h2>
			<h3>PMID:12232000</h3>
				

				
			<p>
				作者：Gijzen M(1), Van Huystee R, Buzzell RI.
				<br/> 
				年份：1993 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peroxidase activity in the seed coats of soybean (Glycine max [L.] Merr.) is controlled by the <span class="gene">Ep</span> locus. We compared peroxidase activity in cell-free extracts from seed coat, root, and leaf tissues of three EpEp cultivars (Harosoy 63, Harovinton, and Coles) to three epep cultivars (Steele, Marathon, and Raiden). Extracts from the seed coats of EpEp cultivars were 100-fold higher in specific activity than those from epep cultivars, but there was no difference in specific activity in crude root or leaf extracts. Isoelectric focusing of root tissue extracts and staining for peroxidase activity showed that EpEp cultivars had a root peroxidase of identical isoelectric point to the seed coat peroxidase, whereas roots of the epep types were lacking that peroxidase, indicating that the <span class="gene">Ep</span> locus may also affect expression in the root. In seed coat extracts, peroxidase was the most abundant soluble <span class="trait">protein</span> in EpEp cultivars, whereas this enzyme was present only in trace amounts in epep genotypes, as determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis. Histochemical localization of peroxidase activity in seed coats of EpEp cultivars shows that the enzyme occurs predominately in the cytoplasm of hourglass cells of the subepidermis. No obvious difference in the gross or microscopic structure of the seed coat was observed to be associated with the <span class="gene">Ep</span> locus. These results suggest that soybean seed coat peroxidase may be involved in processes other than seed coat biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of an Arabidopsis lipoxygenase gene responsive to methyl jasmonate and wounding.</h2>
			<h3>PMID:8290626  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bell E(1), Mullet JE.
				<br/> 
				年份：1993 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA corresponding to the gene AtLox2 was isolated from an Arabidopsis thaliana library using a lipoxygenase (LOX) probe from soybean. AtLox2 encodes a 102-kD <span class="trait">protein,</span> AtLOX2, which has 42 to 45% amino acid sequence identity with other plant <span class="gene">LOX</span> sequences. The AtLOX2 sequence is more than 30 amino acids longer at the amino terminus than other plant <span class="gene">LOX</span> sequences, and this extension has features reminiscent of chloroplast transit peptides, suggesting that AtLOX2 may be chloroplast localized. AtLox2 mRNA levels are <span class="trait">high</span> in leaves and inflorescences but very low in seeds, roots, and stems. AtLox2 mRNA accumulation is rapidly induced in leaves in response to methyl jasmonate. Leaves that have been wounded and adjacent leaves on the same plant also accumulate AtLox2 mRNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the promoter of avirulence gene D from Pseudomonas syringae pv. tomato.</h2>
			<h3>PMID:8376338  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shen H(1), Keen NT.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The avirulence gene D (avrD) from Pseudomonas syringae pv. tomato comprises the first open reading frame (ORF) of a putative operon consisting of at least five tandem ORFs. The promoter of the avrD operon was localized to a 150-bp DNA fragment occurring 5' to the avrD gene by using the Tn7-lux and gus reporter systems. The avrD promoter in P. syringae pv. tomato and P. syringae pv. glycinea was poorly expressed when bacteria were grown in complex culture media but was activated during bacterial growth in plants. The timing and level of induction were similar in compatible and incompatible plant-pathogen interactions. When bacteria were grown in minimal culture medium, promoter activity was repressed by certain carbon sources, <span class="trait">high</span> concentrations of nitrogen compounds, and pH values above 6.5. Primer extension experiments on RNA from bacteria grown in minimal medium identified two transcription initiation sites 87 and 41 nucleotides upstream from the translational start site. Only the -41 transcriptional start site was identified in bacteria grown in soybean leaves. A sigma 54 promoter consensus sequence (GG-10 bp-GC) occurred 14 bp upstream of the -41 transcriptional start, and 3' deletions into this region completely abolished promoter activity. Little expression was observed when a gus fusion with the avrD promoter was introduced into an <span class="gene">ntrA</span> mutant strain of P. syringae pv. phaseolicola deficient in the sigma 54 cofactor. Expression from the avrD promoter also required the hrp regulatory genes, <span class="gene">hrpS</span> and hrpL. Deletions from the 5' end of the promoter region and base substitution analyses also identified two upstream elements important for expression. Sequence comparison of these elements with other cloned avirulence genes revealed the presence of a conserved consensus sequence elements with other cloned avirulence genes revealed the presence of a conserved consensus sequence (GGAACC-N15/16-CCAC) in the promoters of nine different avirulence genes from P. syringae pathovers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Does the ocs-element occur as a functional component of the promoters of plant genes?</h2>
			<h3>PMID:8220489  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ellis JG(1), Tokuhisa JG, Llewellyn DJ, Bouchez D, Singh K, Dennis ES, Peacock WJ.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The structural requirements of the ocs-element, a promoter element in several genes transferred to the host plant nucleus by Agrobacterium tumefaciens and certain DNA viruses, have been further characterized both in vitro and in vivo. Two adjacent and functionally identical protein-binding sites separated by an exact number of nucleotides are required for in vivo activity of the ocs-element. Plant pathogens have presumably recruited cellular transcription factors that interact with these binding sites to drive the high-level expression of their essential genes. Our functional analyses of the ocs-elements from two pathogen promoters define the structure of a sequence motif that might also be expected to occur in plant nuclear genes, and a search of the plant gene database has identified a number of plant gene promoters that contain sequences that resemble the ocs-element. These sequences were analysed for their ability both to bind the maize nuclear <span class="trait">protein</span> OCSTF and to activate transcription of an inactive promoter. A functional ocs-element was identified in only one of the plant genes, the soybean heat-shock gene, <span class="gene">Gmhsp26-A.</span> The apparent rarity of the ocs-element in plant genes contrasts with its frequent use by pathogens that transform the plant nucleus. Sequences resembling half of an ocs-element, on the other hand, are common in plant promoters and may form part of multi-element control motifs with a variety of regulatory functions. Plant pathogens may, therefore, have evolved to circumvent tight regulatory control of their promoters by the host by duplicating the half ocs-element promoter motifs to take advantage of the ubiquitous ocs-element-binding transcription factors in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of avirulence gene <span class="gene">avrRpt2</span> and identification of a putative regulatory sequence common to all known Pseudomonas syringae avirulence genes.</h2>
			<h3>PMID:8335641  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Innes RW(1), Bent AF, Kunkel BN, Bisgrove SR, Staskawicz BJ.
				<br/> 
				年份：1993 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">avrRpt2</span> locus from Pseudomonas syringae pv. tomato causes virulent strains of P. syringae to be avirulent on some, but not all, lines of Arabidopsis thaliana and Glycine max (soybean). We determined the DNA sequence of the <span class="gene">avrRpt2</span> locus and identified the <span class="gene">avrRpt2</span> gene as a 768-bp open reading frame encoding a putative 28.2-kDa <span class="trait">protein.</span> Deletion analysis and transcription studies provided further evidence that this open reading frame encodes AvrRpt2. We found that the <span class="gene">avrRpt2</span> gene also has avirulence activity in P. syringae pathogens of Phaseolus vulgaris (common bean), suggesting that disease <span class="trait">resistance</span> genes specific to <span class="gene">avrRpt2</span> are functionally conserved among diverse plant species. The predicted AvrRpt2 <span class="trait">protein</span> is hydrophilic and contains no obvious membrane-spanning domains or export signal sequences, and there was no significant similarity of AvrRpt2 to sequences in the GenBank, EMBL, or Swiss PIR data bases. A comparison of the <span class="gene">avrRpt2</span> DNA sequence to nine other P. syringae avirulence genes revealed a highly conserved sequence, GGAACCNA-N14-CCACNNA, upstream of the translation initiation codon. This motif is located 6 to 8 nucleotides upstream of the transcription start site in all four P. syringae avirulence genes for which a transcription start site has been determined, suggesting a role as a binding site for a novel form of RNA polymerase. Regulation of <span class="gene">avrRpt2</span> was similar to other P. syringae avirulence genes; expression was <span class="trait">high</span> in minimal medium and low in rich medium and depended on the hrpRS locus and an additional locus at the opposite end of the hrp cluster of P. syringae pv. tomato.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:One member of a gro-ESL-like chaperonin multigene family in Bradyrhizobium japonicum is co-regulated with symbiotic nitrogen fixation genes.</h2>
			<h3>PMID:8101485  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fischer HM(1), Babst M, Kaspar T, Acuña G, Arigoni F, Hennecke H.
				<br/> 
				年份：1993 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This report is concerned with the structural characterization and genetic regulation of new bacterial <span class="gene">groES</span> and <span class="gene">groEL</span> chaperonin genes, and presents two novelties. The first is the discovery that the nitrogen fixing soybean root nodule bacterium, Bradyrhizobium japonicum, unlike all other prokaryotes investigated so far, possesses a multigene family consisting of five very similar, though not identical, groESL-like genes. The second novelty relates to the finding that these five homologues are expressed to different degrees and, in particular, that one family member (namely groESL3) is induced by a mechanism that does not involve the well-known heat shock response. By contrast, the groESL3 genes are co-regulated together with symbiotic nitrogen fixation genes, in that they are activated by the nitrogen fixation regulatory <span class="trait">protein</span> NifA at low oxygen conditions and transcribed from a -24/-12 promoter by the sigma 54 RNA polymerase. Two other members of the groESL gene family are apparently expressed constitutively at different levels, and yet another one is strongly induced by <span class="trait">high</span> temperature. As an attractive hypothesis it follows that B. japonicum may modulate its cellular contents of GroES- and GroEL-like chaperonins in response to specific environmental conditions and physiological needs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of casein and soy <span class="trait">protein</span> on accumulation of cholesterol and dolichol in rat liver.</h2>
			<h3>PMID:8475141  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ishinaga M(1), Hamada M, Ohnaka K, Fukunaga K, Minato Y.
				<br/> 
				年份：1993 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A diet containing 15% (w/w) fat and 20% (w/w) of either casein (CAS) or soy <span class="trait">protein</span> (SOY) was fed to 4-week-old rats for a period of 18 months. The effects of these dietary proteins on the accumulation of cholesterol and dolichol in livers were studied. After 1 month, the amount of liver cholesterol was about 5 mg/g of liver. After an additional 5 months of feeding, there was a slight decrease in cholesterol per gram of liver (3.6 mg/g of liver in CAS-fed rats and 2.6 mg/g of liver in SOY-fed rats). However, after 18 months, there were a remarkable increase (7.5 mg/g of liver) in CAS-fed rats and only a slight increase in SOY-fed rats. The proportions of liver cholesterol ester in rats fed the <span class="gene">CAS</span> diet were 60-70% of the total cholesterol during the experimental period, but in the case of the SOY diet, only rats fed the diet for 1 month showed a <span class="trait">high</span> level, 70%, of cholesterol ester. The amounts of liver dolichol in rats fed the <span class="gene">CAS</span> and SOY diets after feeding for 18 months were 60 micrograms and 47 micrograms of liver, respectively. There was a 1.5-fold increase in both diets for a period of 18 months. The proportions of liver dolichyl fatty ester in rats fed the <span class="gene">CAS</span> diet were 35-40% of the total dolichol during the experimental period, but in the case of the SOY diet, only rats fed the diet for 1 month showed a <span class="trait">high</span> level, 36%, of dolichyl fatty ester. The proportions of dolichol ester in rats fed the SOY diet were 25-30% after 6 and 18 months of feeding. These observations indicated that the SOY diet depresses the accumulation of both liver dolichol and cholesterol.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete amino acid sequence of a Kunitz family trypsin inhibitor from seeds of Acacia confusa.</h2>
			<h3>PMID:8468333  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu HC(1), Lin JY.
				<br/> 
				年份：1993 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete amino acid sequence of a Kunitz-type two-chain trypsin inhibitor was determined for the first time. The sequence of the inhibitor from Acacia confusa (ACTI) was determined by analysis of peptides obtained from the reduced and S-carboxymethylated <span class="trait">protein</span> by digestion with endopeptidase Lys-C, endopeptidase Arg-C, and V8 endopeptidase. <span class="gene">ACTI</span> is comprised of two chains, namely A and B chains linked by the disulfide bridge between Cys(133) and Cys(141), and the inhibitor consists of 175 amino acid residues, 136 residues in the A-chain and 39 residues in the B-chain. The N-terminal amino acid sequence of <span class="gene">ACTI</span> shows extensive homology to the trypsin inhibitors from Acacia elata and Albizzia julibrissin, while the whole amino acid sequence of <span class="gene">ACTI</span> has a <span class="trait">high</span> degree of homology to the other Kunitz-type trypsin inhibitors from soybeans, winged bean seeds [Psophocarpus tetragonolobus (L) DC.], and seeds of Erythrina species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Direct isolation and expansion of human CD34+ hematopoietic stem cells.</h2>
			<h3>PMID:7522888  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lebkowski J(1), Schain L, Harvey M, Moseley A, Okarma TB.
				<br/> 
				年份：1993 Fall
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The isolation and expansion of CD34+ cells has numerous applications as supportive therapy for cytopenias that occur postchemotherapy. In this report, we describe a simple system to directly isolate and culture human CD34+ cells. Using this system, CD34+ cells isolated from bone marrow were cultured in the presence of <span class="gene">GM-CSF,</span> G-CSF, erythropoietin, stem cell factor, IL-1, IL-3, and IL-6. Cell expansions up to 100-fold were noted during the first 2 weeks of culture with <span class="trait">high</span> maintenance of CD34+ cells during the first week of culture. Maximal expansions of progenitors were observed at one week of culture, with, on average, 20-fold increases in CFU-GM number. The results support the feasibility of CD34+ cell expansion for clinical application.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed-specific repression of GUS activity in tobacco plants by antisense RNA.</h2>
			<h3>PMID:1463841  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fujiwara T(1), Lessard PA, Beachy RN.
				<br/> 
				年份：1992 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						beta-Conglycinin, the 7S storage <span class="trait">protein</span> of soybean, is expressed only in seeds and is regulated predominantly by gene transcription [5]. We applied an antisense strategy to modify expression of a beta-glucuronidase (uidA or gusA) gene in seeds using a promoter from a beta-conglycinin gene. Transgenic tobacco plants harboring the gusA gene under the control of the CaMV 35S promoter were retransformed with a gene construct comprising the beta-conglycinin promoter fused to the gusA gene in the antisense orientation. Double transformants were regenerated and transformation was confirmed by Southern blot hybridization. Seed-specific repression of GUS activity was observed in lines containing <span class="trait">high</span> copy numbers of the antisense gusA transgene. Suppression of GUS activity was correlated with the amounts of (-) sense gusA transcript detected and concomitantly with a decrease in gusA transcript levels. Furthermore, the amount of suppression of GUS activity was greatest during mid to late stages of seed development, when expression of the <span class="gene">alpha'</span> promoter is <span class="trait">high.</span> These results indicate that suppression of GUS activity is due to expression of the antisense gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Low density lipoprotein receptor activity is modulated by soybean globulins in cell culture.</h2>
			<h3>PMID:1527638  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lovati MR(1), Manzoni C, Corsini A, Granata A, Frattini R, Fumagalli R, Sirtori CR.
				<br/> 
				年份：1992 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of major storage globulins from soybean on cholesterol homeostasis were investigated in vitro and in vivo systems. The low density lipoprotein (LDL) uptake and degradation was studied both in human skin fibroblasts (HSF) and in a human hepatoma cell line (Hep G2). In Hep G2 cells a dose-dependent increase of both uptake and degradation of 125I-LDL was induced by the 7S globulin, whereas the 11S globulin exerted a lesser effect that was not dose-related. In <span class="gene">HSF</span> cells the 11S globulin increased the uptake of 125I-LDL to a greater extent than did 7S globulin; in this cell line, LDL degradation was not stimulated by either of the globulins. Rats fed a casein-cholesterol diet were treated daily with the 11S or 7S globulins for 2 wk. The administration of soybean globulins significantly reduced cholesterolemia (-35 and -34% with 7S and 11S globulins, respectively, vs. controls). Liver membrane preparations from the casein-cholesterol-fed rats showed a nonsignificant increase in the maximal binding of labeled cholesterol-rich lipoprotein fraction (beta-VLDL) to <span class="trait">high</span> affinity receptors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:AT-rich promoter elements of soybean heat shock gene <span class="gene">Gmhsp17.5E</span> bind two distinct sets of nuclear proteins in vitro.</h2>
			<h3>PMID:1511143  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Czarnecka E(1), Ingersoll JC, Gurley WB.
				<br/> 
				年份：1992 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A 33 bp double-stranded oligonucleotide homologous to two AT-rich sequences located upstream (-907 to -889 and -843 to -826) to the start of transcription of heat shock gene <span class="gene">Gmhsp17.5E</span> of soybean stimulated transcription when placed 5' to a truncated (-140) maize Adh1 promoter. The chimeric promoter was assayed in vivo utilizing anaerobically stressed sunflower tumors transformed by a pTi-based vector of Agrobacterium tumefaciens. Nuclear proteins extracted from soybean plumules were shown to bind double-stranded oligonucleotides homologous to AT-rich sequences in the 5' flanking regions of soybean beta-conglycinin, lectin, leghemoglobin and heat shock genes. These proteins were also shown to bind AT-rich probes homologous to homeobox <span class="trait">protein</span> binding sites from the Antennapedia and engrailed/fushi tarazu genes of Drosophila. Binding activity specific for AT-rich sequences showed a wide distribution among various plant organs and species. Preliminary characterization indicated that two sets of nuclear proteins from soybean bind AT-rich DNA sequences: a diverse high-molecular-weight (ca. 46-69 kDa) group, and a low-molecular-weight (23 and 32 kDa) group of proteins. The latter meets the operational criteria for high-mobility group proteins (HMGs).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Calcium-dependent <span class="trait">protein</span> kinase in the green alga Chara.</h2>
			<h3>PMID:24178199</h3>
				

				
			<p>
				作者：McCurdy DW(1), Harmon AC.
				<br/> 
				年份：1992 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cytoplasmic streaming in the characean algae is inhibited by micromolar rises in the level of cytosolic free Ca(2+), but both the mechanism of action and the molecular components involved in this process are unknown. We have used monoclonal antibodies against soybean Ca(2+)-dependent <span class="trait">protein</span> kinase (CDPK), a kinase that is activated by micromolar Ca(2+) and co-localizes with actin filaments in higher-plant cells (Putnam-Evans et al., 1989, Cell Motil. Cytoskel. 12, 12-22) to identify and localize its characean homologue. Immunoblot analysis revealed that <span class="gene">CDPK</span> in Chara corralina Klein ex. Wild shares the same relative molecular mass (51-55 kDa) as the kinase purified from soybean, and after electrophoresis in denaturing gels is capable of phosphorylating histone III-S in a Ca(2+)-dependent manner. Immunofluorescence microscopy localized <span class="gene">CDPK</span> in Chara to the subcortical actin bundles and the surface of small organelles and other membrane components of the streaming endoplasm. The endoplasmic sites carrying <span class="gene">CDPK</span> were extracted from internodal cells by vacuolar perfusion with 1 mM ATP or 10(-4) M Ca(2+). Both the localization of <span class="gene">CDPK</span> and its extraction from internodal cells by perfusion with ATP or <span class="trait">high</span> Ca(2+) are properties similar to that reported for the heavy chain of myosin in Chara (Grolig et al., 1988, Eur. J. Cell Biol. 47, 22-31). Based on its endoplasmic location and inferred enzymatic properties, we suggest that <span class="gene">CDPK</span> may be a putative element of the signal-transduction pathway that mediates the rapid Ca(2+)-induced inhibition of streaming that occurs in the characean algae.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sequencing and characterization of the soybean leaf metalloproteinase : structural and functional similarity to the matrix metalloproteinase family.</h2>
			<h3>PMID:16668986</h3>
				

				
			<p>
				作者：McGeehan G(1), Burkhart W, Anderegg R, Becherer JD, Gillikin JW, Graham JS.
				<br/> 
				年份：1992 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel zinc endoproteinase has been sequenced and characterized from soybean leaves (Glycine max var Williams 82) and has been designated as <span class="trait">Protein</span> Identification Resource accession No. A41820 <span class="gene">SMEP1</span> (soybean metalloendoproteinase 1). Comparison of the primary amino acid sequence with other zinc proteinases revealed the enzyme to be a new member of the matrix metalloproteinase (MMP) family of enzymes. SMEP was found to have MMP cleavage specificity toward peptide substrates and the enzyme is specifically inhibited by naturally occurring tissue inhibitors of MMPs through a high-affinity interaction (inhibitor concentration resulting in an approximate 50% decrease in enzyme activity = 23 x 10(-9) molar). Together, these results suggest that the origin of the MMP family of enzymes and their cognate inhibitors predates the divergence of plants and animals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of 5-oxo-15-hydroxy-6,8,11,13-eicosatetraenoic acid as a novel and potent human eosinophil chemotactic eicosanoid.</h2>
			<h3>PMID:1618754  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schwenk U(1), Morita E, Engel R, Schröder JM.
				<br/> 
				年份：1992 Jun 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Incubation of human eosinophils with arachidonic acid led to the formation of a novel and potent eosinophil chemotactic lipid (ECL) (Morita, E., Schröder, J.-M., and Christophers, E. (1990) J. Immunol. 144, 1893-1900). To test the working hypothesis of whether ECL could have been formed via eosinophil-arachidonic acid 15-lipoxygenase we investigated whether other arachidonic acid 15-lipoxygenases such as soybean lipoxygenase I catalyze formation of a similar ECL. In the presence of hemoproteins and soybean lipoxygenase I arachidonic acid is converted to an ECL, which has physicochemical properties similar to those found for the eosinophil-derived ECL. Purification of this ECL by <span class="trait">high</span> performance liquid chromatography revealed that ECL is structurally different from well known eosinophil chemotactic eicosanoids such as leukotriene B4, 5,15-(6E,8Z,11Z,13E)-dihydroxyeicosatetraenoic acid (5,15-diHETE), and (8S,15S)-(5Z,9E,11Z,13E)-dihydroxyeicosatetra <span class="gene">eno</span> ic acid ((8S,15S)-diHETE). UV spectra of this ECL with absorbance maxima at 230 and 278 nm revealed the presence of two independent chromophores such as a conjugated oxodiene and a conjugated diene. Catalytic hydrogenation of ECL methyl ester led to the formation of 5,15-dihydroxyarachidic acid methyl ester. Reduction of ECL with sodium borohydride produced a product which is identical with authentic (5S,15S)-(6E,8Z,11Z,13E)-diHETE. Formation of an ECL monomethoxime derivative supports the conclusion that this highly potent eosinophil chemotactic eicosanoid is structurally identical with 5-oxo-15-hydroxy-6,8,11,13-eicosatetraenoic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:DNA binding site preferences and transcriptional activation properties of the Arabidopsis transcription factor GBF1.</h2>
			<h3>PMID:1563344  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schindler U(1), Terzaghi W, Beckmann H, Kadesch T, Cashmore AR.
				<br/> 
				年份：1992 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The G-box is a cis-acting element found within the promoters of many plant genes where it mediates expression in response to a variety of different stimuli. This palindromic DNA motif (CCACGTGG) is composed of two identical half sites, the base pairs of which we have numbered -4 to +4 (numbering from 5' to 3'). Both half sites are involved in the binding of the bZIP <span class="trait">protein</span> GBF1, a member of the GBF family of Arabidopsis thaliana. Here we demonstrate using the random binding site selection method that <span class="gene">GBF1</span> interacts with, in addition to the palindromic G-box, other DNA motifs that fall into seven distinct groups. All groups share the ACGT core sequence, common to most DNA motifs bound by plant bZIP proteins so far characterized. Our studies demonstrate that a <span class="trait">high</span> affinity <span class="gene">GBF1</span> binding site is further defined by the following two parameters: first, all sites contain a G residue at position +3 (as in ACGTG) and secondly, only certain base pair combinations are allowed at positions -4, -3 and +4. Two of the identified groups (TGACGTGG and TGACGTGT) contain the base pairs TG at positions -4 and -3 and hence resemble the binding sites of another class of plant bZIP proteins (TGACGT/C binding proteins). However, <span class="gene">GBF1</span> only interacts with the TGACGT sequence if the two 3' distal nucleotides (positions +3 and +4) are occupied by GG or GT. These data define the differences between a G-box binding <span class="trait">protein</span> and TGACGT/C binding proteins. The N-terminal domain of <span class="gene">GBF1</span> is defined by a <span class="trait">high</span> proline content. Such regions were also identified in proteins related to GBF1. We demonstrate that this N-terminal proline-rich domain of GBF1, when fused to a heterologous DNA binding domain, stimulates transcription in both plant protoplasts and mammalian cells. These extensive DNA binding studies and the characterization of the <span class="gene">GBF1</span> activation domain will facilitate both the identification of regulatory elements and the in vivo function of GBF1.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Further characterization of cytochrome <span class="gene">P450</span> involved in phytoalexin synthesis in soybean: cytochrome <span class="gene">P450</span> cinnamate 4-hydroxylase and 3,9-dihydroxypterocarpan 6a-hydroxylase.</h2>
			<h3>PMID:1731635  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kochs G(1), Werck-Reichhart D, Grisebach H.
				<br/> 
				年份：1992 Feb 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two cytochrome <span class="gene">P450</span> enzymes, cinnamate 4-hydroxylase (C4H) and 3,9-dihydroxypterocarpan 6a-hydroxylase (D6aH), were isolated from elicitor-challenged soybean (Glycine max) cell cultures (G. Kochs and H. Grisebach, 1989, Arch. Biochem. Biophys. 273, 543-553). An earlier purification protocol was improved by the use of new chromatographic media, leading to a higher <span class="trait">yield</span> of enzymatic activity. After separation of <span class="gene">C4H</span> from D6aH on hydroxyapatite, the <span class="gene">C4H</span> was identified using anti-C4H antibody from Jerusalem artichoke (Helianthus tuberosus) (B. Gabriac et al., 1991, Arch. Biochem. Biophys. 288, 302-309). The two proteins show molecular weights of about 58,000 for <span class="gene">C4H</span> and about 55,000 for D6aH on sodium dodecyl sulfate-polyacrylamide gel electrophoresis. Both enzyme activities are dependent on NADPH:cytochrome <span class="gene">P450</span> reductase and cross-react with their respective antibodies. Both cytochrome <span class="gene">P450</span> subspecies show substrate binding and CO-difference spectra typical for cytochrome <span class="gene">P450</span> and were found to be glycoproteins by their cross-reaction with biotinylated lectins in Western blot. The N-terminal sequence of <span class="gene">C4H</span> from soybean shows <span class="trait">high</span> similarity to the N-terminus of <span class="gene">C4H</span> from Jerusalem artichoke.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The beta-tubulin gene family of pea: primary structures, genomic organization and intron-dependent evolution of genes.</h2>
			<h3>PMID:1558942  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liaud MF(1), Brinkmann H, Cerff R.
				<br/> 
				年份：1992 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						One gene and two cDNAs encoding three different beta-tubulins (TUB1, TUB2, TUB3) of pea have been cloned and sequenced. The derived amino acid sequences show between 92% and 96% identity relative to one another and to most other beta-tubulins of higher plants and green algae. Two notable extremes are the <span class="trait">high</span> similarity of 98% between pea TUB3 and maize beta-tubulin 2 and the relatively low similarity (90%) of the hypocotyl-specific beta-tubulin 1 of soybean to the pea sequences. These similarities do not reflect the molecular phylogeny but rather differences in evolutionary rate of beta-tubulins which are differentially regulated during plant development. Genomic Southern blots reveal a beta-tubulin gene family in pea with at least four separate members including two TUB1 genes, one <span class="gene">TUB2</span> gene and one TUB3 gene. This contradicts an earlier report by Raha et al. (Plant Mol Biol 9: 565-571, 1987) suggesting a tandem repeat organization of tubulin genes in pea. The pea TUB1 gene has two introns in identical positions compared to the beta-tubulin genes from Arabidopsis and soybean. In an attempt to reconstruct the universal ancestor of all present-day tubulin genes the intron positions in 38 different alpha- and beta-tubulin genes from plants, animals, fungi and protozoa were compared. This comparison shows that the primordial gene probably had many introns (more than 20) separating 'protoexons' of 15 to 20 codons in agreement with the 'exon theory of genes'. It also supports the view that, during the course of evolution, introns have shifted and were deleted preferentially in the 3' part of the genes. Similar observations have been made previously for other genes. They can be interpreted in terms of a homologous recombination of genes with their modified (incorrectly spliced) and reverse-transcribed pre-mRNAs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Existence of two histone <span class="gene">H3</span> variants in dicotyledonous plants and correlation between their acetylation and plant genome size.</h2>
			<h3>PMID:1731982  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Waterborg JH(1).
				<br/> 
				年份：1992 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Histone <span class="gene">H3</span> proteins were purified to near homogeneity from callus cultures of dicotyledonous plants alfalfa, soybean, Arabidopsis, carrot and tobacco to determine the number of histone <span class="gene">H3</span> variants. In every species two histone <span class="gene">H3</span> variants were identified by gradient gel electrophoresis and reversed-phase chromatography. They were named H3.1 and H3.2 in order of increasing mobility in acid-urea-Triton gels. Co-electrophoresis of histone H3.2 proteins of all species in this gel system and HPLC co-chromatography suggest that all histone H3.2 variants have a primary <span class="trait">protein</span> sequence identical to alfalfa H3.2. Two distinct H3.1 variant forms were identified, represented by alfalfa and Arabidopsis H3.1 proteins which differ only at residue 90. Soybean H3.1 resembles H3.1 of alfalfa. Carrot and tobacco H3.1 appear identical to the Arabidopsis H3.1 histone variant. All <span class="gene">H3</span> proteins were acetylated to multiple levels and in each plant the histone H3.2 forms were more highly acetylated. An inverse relationship was observed between plant genome size and the relative abundance of histone variant H3.2 and also with the level of acetylation of both histone <span class="gene">H3</span> variants. This correlation matches the general tendency that in plants with smaller genomes a larger fraction of the genome is transcriptionally active.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of dietary protein-energy interrelationships on Holstein steer performance and ruminal bacterial fermentation in continuous culture.</h2>
			<h3>PMID:1667012  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chester-Jones H(1), Stern MD, Metwally HM, Linn JG, Ziegler DM.
				<br/> 
				年份：1991 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In vivo and in vitro 3 x 2 factorial experiments were conducted concurrently to evaluate the incorporation of 0, 15, or 30% sugar beet pulp (SBP) as an energy source in diets fed to growing Holstein steers with either soybean meal (SBM) or alcohol-treated, defatted soybean flakes (ATSBF) as primary supplemental <span class="trait">protein</span> sources. Three groups of 42 Holstein steers each were fed six different diets from 54 kg initial BW to 320 kg in three experimental periods. There were no overall <span class="gene">SBP</span> level x <span class="trait">protein</span> source interactions (P greater than .05). Beet pulp level tended to decrease ADG (linear, P = .05) and increase feed/gain (linear, P less than .05) and DMI (quadratic, P less than .05). Each grower diet was used in a substrate for ruminal microbial metabolism in six dual-flow, continuous-culture fermenters. Organic matter and carbohydrate digestion were similar (P greater than .05) among diets. Increasing dietary levels of <span class="gene">SBP</span> caused a concomitant increase (P less than .05) in acetate and decrease (P less than .05) in butyrate and isobutyrate concentrations. Beet pulp level x <span class="trait">protein</span> source interactions (P less than .05) were observed for CP degradation, ammonia and nonammonia N, and dietary N flow. Crude <span class="trait">protein</span> degradation was higher (P less than .05) for the 0% <span class="gene">SBP</span> with SBM diet (81.3%) than for the 30% <span class="gene">SBP</span> with ATSBF diet (64.4%). Efficiency of bacterial synthesis was similar (P greater than .05) among diets. Results indicated that <span class="gene">SBP</span> is an effective dietary energy source for high-energy grower diets at 15 or 30% of the DM but may cause a decrease in some performance traits. There were no nutritional benefits of using ATSBF vs SBM as the supplemental N source.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete nucleotide sequence of pea seed-borne mosaic virus RNA.</h2>
			<h3>PMID:1940858  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Johansen E(1), Rasmussen OF, Heide M, Borkhardt B.
				<br/> 
				年份：1991 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete nucleotide sequence of the RNA genome of pea seed-borne mosaic virus (PSbMV) was determined from cloned cDNA and by direct sequencing of viral RNA. The PSbMV genomic sequence was determined to be 9924 nucleotides in length excluding the poly(A) tract. The RNA contained an open reading frame (ORF) of 9618 nucleotides with the potential to encode a <span class="gene">polyprotein</span> with a calculated Mr of 364000 (364K). The ORF was flanked by a 5' untranslated leader sequence of 143 nucleotides and a 3' untranslated region of 163 nucleotides. A comparison of the PSbMV <span class="gene">polyprotein</span> with the polyproteins of the potyviruses tobacco etch virus (TEV), tobacco vein mottling virus (TVMV), plum pox virus (PPV) and potato virus Y (PVY) showed that PSbMV had a similar genome organization. The polyproteins had a <span class="trait">high</span> level of amino acid identity except in the N-terminal region, which varied in both sequence and length. Putative proteolytic cleavage sites were identified in the <span class="gene">polyprotein</span> of PSbMV by comparison with those identified for other potyviruses. The cleavage site between the 6K <span class="trait">protein</span> and the 49K proteinase is proposed to occur at the C-terminal side of glutamic acid and not at the C-terminal side of glutamine as in other potyviruses. In addition to the five proteolytic cleavage sites for the 49K proteinase identified previously, a sixth putative cleavage site was identified internally in the 49K proteinase of PSbMV, as well as in the 49K proteinases of TEV, TVMV, PPV, PVY and soybean mosaic virus. Cleavage at this site in the 49K proteinases of TEV, TVMV and PPV would result in an N-terminal <span class="trait">protein</span> of 22K to 24K, which is similar in size to the size determined for their VPgs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Water deficit modulates gene expression in growing zones of soybean seedlings. Analysis of differentially expressed cDNAs, a new beta-tubulin gene, and expression of genes encoding cell wall proteins.</h2>
			<h3>PMID:1912487  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Creelman RA(1), Mullet JE.
				<br/> 
				年份：1991 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transfer of soybean seedlings to low-water-potential vermiculite (psi w = -0.3 MPa) results in a reversible decrease in hypocotyl growth and modulation of several polysomal mRNAs (Plant Physiol 92: 205-214). We report here the isolation of two cDNA clones (pGE16 and pGE95) which correspond to genes whose mRNA levels are increased, and one cDNA clone (pGE23) which corresponds to a gene whose mRNA level is decreased in the hypocotyl zone of cell elongation by water deficit. In well-watered seedlings mRNAs hybridizing to pGE16 and pGE95 are most abundant in mature regions of the seedling, but in water-deficient seedlings mRNA levels are reduced in mature regions and enhanced in elongating regions. RNA corresponding to soybean proline-rich <span class="trait">protein</span> 1 (sbPRP1) shows a similar tissue distribution and response to water deficit. In contrast, in well-watered seedlings, the gene corresponding to pGE23 was highly expressed in the hypocotyl and root growing zones. Transfer of seedlings to low-water-potential vermiculite caused a rapid decrease in mRNA hybridizing to pGE23. Sequence analysis revealed that pGE23 has <span class="trait">high</span> homology with beta-tubulin. Water deficit also reduced the level of mRNA hybridizing to JCW1, an auxin-modulated gene, although with different kinetics. Furthermore, mRNA encoding actin, glycine-rich proteins (GRPs), and hydroxyproline-rich glycoproteins (HRGPs) were down-regulated in the hypocotyl zone of elongation of seedlings exposed to water deficit. No effect of water deficit was observed on the expression of chalcone synthase. Decreased expression of beta-tubulin, actin, JCW1, HRGP and <span class="gene">GRP</span> and increased expression of sbPRP1, pGE95 and pGE16 in the hypocotyl zone of cell elongation could participate in the reversible growth inhibition observed in water-deficient soybean seedlings.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Response of soybean photosynthesis and chloroplast membrane function to canopy development and mutual shading.</h2>
			<h3>PMID:16668377</h3>
				

				
			<p>
				作者：Burkey KO(1), Wells R.
				<br/> 
				年份：1991 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of natural shading on photosynthetic capacity and chloroplast thylakoid membrane function was examined in soybean (Glycine max. cv Young) under field conditions using a randomized complete block design. Seedlings were thinned to 15 plants per square meter at 20 days after planting. Leaves destined to function in the shaded regions of the canopy were tagged during early expansion at 40 days after planting. To investigate the response of shaded leaves to an increase in available light, plants were removed from certain plots at 29 or 37 days after tagging to reduce the population from 15 to three plants per square meter and alter the irradiance and spectral quality of light. During the transition from a <span class="gene">sun</span> to a shade environment, maximum photosynthesis and chloroplast electron transport of control leaves decreased by two- to threefold over a period of 40 days followed by rapid senescence and abscission. Senescence and abscission of tagged leaves were delayed by more than 4 weeks in plots where plant populations were reduced to three plants per square meter. Maximum photosynthesis and chloroplast electron transport activity were stabilized or elevated in response to increased light when plant populations were reduced from 15 to three plants per square meter. Several chloroplast thylakoid membrane components were affected by light environment. Cytochrome f and coupling factor <span class="trait">protein</span> decreased by 40% and 80%, respectively, as control leaves became shaded and then increased when shaded leaves acclimated to <span class="trait">high</span> light. The concentrations of photosystem I (PSI) and photosystem II (PSII) reaction centers were not affected by light environment or leaf age in field grown plants, resulting in a constant PSII/PSI ratio of 1.6 +/- 0.3. Analysis of the chlorophyll-protein composition revealed a shift in chlorophyll from PSI to PSII as leaves became shaded and a reversal of this process when shaded leaves were provided with increased light. These results were in contrast to those of soybeans grown in a growth chamber where the PSII/PSI ratio as well as cytochrome f and coupling factor <span class="trait">protein</span> levels were dependent on growth irradiance. To summarize, light environment regulated both the photosynthetic characteristics and the timing of senescence in soybean leaves grown under field conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Estrogenic effect of dietary soya bean meal on vitellogenesis in cultured Siberian sturgeon Acipenser baeri.</h2>
			<h3>PMID:1936926  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pelissero C(1), Le Menn F, Kaushick S.
				<br/> 
				年份：1991 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The unusual presence of vitellogenin in the plasma of male and nonvitellogenic female Siberian sturgeon has been demonstrated previously (Pelissero and Le Menn, 1988; Pelissero et al., 1989a) and was attributed to dietary effects. The present study examines estrogenic effects of dietary soya bean meal and of commercial trout diet on vitellogenesis in sturgeon. The 4-month study compared three diets, one commercial (T) and two experimental synthetic diets, one containing casein alone (SC for synthetic diet made on casein), and the other casein and soya bean (SS for synthetic diet made with soya bean) as <span class="trait">protein</span> sources. The dietary soya bean meal contained plant isoflavonic compounds which are well known to mimic the effects of estrogens in mammals. The SC diet, free of estrogenic compounds, served as the reference diet. When fed with the SC diet, sturgeons showed significantly lower plasma vitellogenin levels (0.0045 +/- 0.0012 mg/ml) compared with those fed the commercial diet (1.24 +/- 0.37 mg/ml). The <span class="gene">SS</span> diet had a very pronounced effect on the plasma vitellogenin level, which at the end of the experiment had reached 6 mg vitellogenin/ml. In no case was estradiol detectable in the plasma. Plasma androgen levels were <span class="trait">high</span> in all the three groups throughout the study period, and not significantly different from one another. Sturgeon fed the T diet had larger livers, with enlarged hepatocytes, compared with those fed the <span class="gene">SS</span> and SC diets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of grain level and <span class="trait">protein</span> source on ruminal fermentation, degradability, and digestion in milking cows fed silage.</h2>
			<h3>PMID:1894816  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Petit HV(1), Veira DM.
				<br/> 
				年份：1991 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Five primiparous Holstein cows (483 kg BW) of low productivity were used in a 4 x 5 incomplete Latin square design to study the effects of feeding two levels of grain containing soybean meal and one level of grain containing fish meal on feed intake, milk production, digestibility, and rumen fermentation. Animals all were fed alfalfa silage for ad libitum intake. Collection of data was between wk 11 and 24 of lactation. The four treatments were an all silage diet (control), silage and medium concentrate fed at 1.8% BW containing soybean meal, and silage and low concentrate fed at 1.3% BW containing either soybean meal or fish meal. Treatments other than control were designed to give similar CP intake from the concentrate, which was based on <span class="trait">high</span> moisture corn and <span class="gene">cob</span> meal. Milk production and composition were similar among treatments except for fat percentage, which was significantly lower for low grain soybean meal. Intake of DM was higher on medium grain soybean meal compared with the other treatments. Average BW and change in BW (.11 kg/d) were not affected by treatments. Digestibility of DM was higher for cows fed grain (68 to 73%) than for those fed the control diet (60%). Generally, feeding concentrate at 1.3 or 1.8% BW and supplementing with soybean meal or fish meal resulted in pH and concentrations of NH3 N and VFA similar to those observed on the all silage diet. Feeding concentrate at 1.3 or 1.8% BW and supplying fish meal or soybean meal did not change feed utilization enough to increase milk production in low producing cows compared with an all silage diet.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Influence of oxygen on DNA binding, positive control, and stability of the Bradyrhizobium japonicum NifA regulatory <span class="trait">protein.</span></h2>
			<h3>PMID:2045367  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Morett E(1), Fischer HM, Hennecke H.
				<br/> 
				年份：1991 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Central to the genetic regulatory circuit that controls Bradyrhizobium japonicum nif and fix gene expression is the NifA <span class="trait">protein.</span> NifA activates transcription of several nif and fix genes and autoregulates its expression during symbiosis in soybean root nodules or in free-living microaerobic conditions. <span class="trait">High</span> O2 tensions result in the lack of nif expression, possibly by inactivation of NifA through oxidation of an essential metal cofactor. Several B. japonicum nif and fix promoters have upstream activator sequences (UAS) required for optimal activation. The UAS are located more than 100 bp from the -24/-12 promoter and have been proposed to be binding sites for NifA. We investigated the interaction of NifA with the <span class="gene">nifD</span> promoter region by using in vivo dimethyl sulfate footprinting. NifA-dependent protection from methylation of the two UAS of this promoter was detected. Footprinting experiments in the presence of rifampin showed that UAS-bound NifA led to the formation of an open <span class="gene">nifD</span> promoter-RNA polymerase sigma 54 complex. Shift to aerobic growth resulted in a rapid loss of protection of both the UAS and the promoter, indicating that the DNA-binding and the activation functions of NifA were controlled by the O2 status of the cell. After an almost complete inactivation by oxygen, the NifA <span class="trait">protein</span> began to degrade. Furthermore, metal deprivation also caused degradation of NifA. In this case, however, the rates of NifA inactivation and NifA degradation were not clearly distinguishable. The results are discussed in the light of a previously proposed model, according to which the oxidation state of a NifA-metal complex influences the conformation of NifA for both DNA-binding and positive control functions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of dietary energy level and <span class="trait">protein</span> source on nutrient digestion and ruminal nitrogen metabolism in steers.</h2>
			<h3>PMID:1648555  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cecava MJ(1), Merchen NR, Berger LL, Mackie RI, Fahey GC Jr.
				<br/> 
				年份：1991 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Four Simmental steers with ruminal, duodenal, and ileal cannulas were used to examine effects of dietary forage: concentrate ratio and supply of ruminally degradable true <span class="trait">protein</span> on site of nutrient digestion and net ruminal microbial <span class="trait">protein</span> synthesis. Steers (345 kg) were fed ammoniated corn <span class="gene">cob</span> <span class="trait">(high</span> forage; HF)- or corn cob/ground corn/cornstarch (low forage; LF)-based diets supplemented with soybean meal (SBM) or a combination of corn gluten meal and blood meal (CB). Diets were fed at 2-h intervals with average DM intake equal to 2.2% of BW. Feeding LF vs HF increased (P less than .05) OM digestion (percentage of intake) in the stomach, small intestine, and total tract. Efficiency of microbial CP synthesis (EMCP; g of N/kg of OM truly fermented) decreased (P less than .05) for LF vs HF (24.1 vs 26.8), but microbial N and total N flows to the small intestine were similar (P greater than .05) between energy levels (average 112 and 209 g/d, respectively). Total N flows to the small intestine were 13.1% greater (P less than .05) for CB than for SBM because of increased (P less than .05) passage of nonmicrobial N. Feeding SBM vs CB increased (P less than .05) EMCP (27.3 vs 23.3) and microbial N flow to the small intestine (127.5 vs 112.5 g/d), but these increases were not likely due to increased ruminal concentrations of ammonia N (NH3 N). Decreased (P less than .05) incorporation of NH3 N into bacterial N and slower turnover rates of ruminal NH3 N for SBM vs CB suggest that direct incorporation of preformed diet components into cell mass increased when SBM was fed. Results of this study suggest that the inclusion of ruminally degradable <span class="trait">protein</span> in the diet may increase the supply of products from proteolysis and that this can increase EMCP and microbial <span class="trait">protein</span> flow to the small intestine.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue-specific and organ-specific expression of soybean auxin-responsive transcripts <span class="gene">GH3</span> and SAURs.</h2>
			<h3>PMID:1840920  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gee MA(1), Hagen G, Guilfoyle TJ.
				<br/> 
				年份：1991 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We used in situ hybridization to localize two classes of auxin-regulated transcripts, <span class="gene">GH3</span> and SAURs, within organs and tissues of soybean seedlings and flowers. <span class="gene">GH3</span> transcripts occurred in the inner cortex and protoxylem ridges of roots and were expressed transiently during flower and pod development. SAUR transcripts were expressed in the epidermis, cortex, and starch sheath of epicotyls and immature hypocotyls. SAUR transcripts became more abundant on the bottom side of hypocotyls that were undergoing gravitropic curvature. SAURs were also expressed in developing xylem elements of the hypocotyl hook. When soybean organ sections were treated with 50 micromolar 2,4-dichlorophenoxyacetic acid (2,4-D), <span class="gene">GH3</span> transcripts became more abundant in the vascular regions of all organs analyzed. <span class="trait">High</span> levels of <span class="gene">GH3</span> transcripts were also found in developing palisade mesophyll cells of leaves, cotyledons, and flowers treated with 2,4-D. SAUR transcripts became more abundant in the epidermis, cortex, starch sheath, and pith of epicotyls and hypocotyls after 2,4-D treatment. Our results showed that a variety of tissues and cell types express auxin-responsive transcripts and that different tissues respond rapidly to exogenous auxin by expressing different hormone-responsive genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a soybean <span class="gene">hsp70</span> gene.</h2>
			<h3>PMID:1714321  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roberts JK(1), Key JL.
				<br/> 
				年份：1991 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean, like many other organisms, responds to an increase in growth temperature by producing a set of new proteins, heat shock proteins. The heat shock proteins have been classified into several categories according to their molecular weight. Data are presented on the isolation, sequence characterization, and expression of a 70 kDa heat shock <span class="trait">protein</span> gene from soybean. A cDNA clone was isolated using a Drosophila <span class="gene">hsp70</span> clone as a heterologous probe, and the cDNA was used for isolation of the soybean gene corresponding to the cDNA. The structure of this soybean is very similar to the <span class="gene">hsp70</span> genes from other organisms. It has several sequences in the 5' untranscribed region that are similar to the well characterized heat shock consensus element found in other organisms. These heat shock consensus elements have the expected position relative to the start of transcription. Unlike hsp70-like genes previously isolated from other plants, this gene does not have an intron. This <span class="trait">protein</span> shows <span class="trait">high</span> amino acid sequence similarity to other <span class="gene">hsp70</span> proteins from such diverse organisms as Drosophila, rat, and Xenopus. This soybean gene is only expressed during heat shock. In addition to the <span class="gene">hsp70</span> gene isolated here, there is evidence for many other hsp70-like genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multiple nuclear factors interact with upstream sequences of differentially regulated beta-conglycinin genes.</h2>
			<h3>PMID:1893110  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lessard PA(1), Allen RD, Bernier F, Crispino JD, Fujiwara T, Beachy RN.
				<br/> 
				年份：1991 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The expression of the <span class="gene">alpha'</span> and beta subunit genes of beta-conglycinin is differentially regulated during soybean embryo development. Although both are expressed solely in developing seeds during mid to late stages of embryo development, the <span class="gene">alpha'</span> subunit is expressed more highly on a per gene basis, and <span class="gene">alpha'</span> subunit mRNA begins to accumulate three to five days earlier than beta subunit mRNA. In cultured cotyledons, beta subunit gene(s) respond to changes in methionine or abscisic acid levels, whereas expression of the <span class="gene">alpha'</span> subunit gene(s) is unaffected by these changes. To investigate the mechanisms by which these genes are transcriptionally regulated, we examined the interactions of nuclear proteins with upstream sequences from the <span class="gene">alpha'</span> and beta subunit genes. Four distinct DNA binding factors were identified in nuclear extracts from developing soybean seeds. These factors are termed Soybean Embryo Factors (SEF) 1 through 4. SEF binding sites are distributed non-uniformly between the <span class="gene">alpha'</span> and beta subunit genes, and the amount of <span class="trait">protein</span> binding is modulated over the course of embryo development. DNA footprinting revealed the sequences recognized by three of these factors. Factors which behave in a manner similar to that of SEF3 were also identified in nuclear extracts from developing tobacco and sunflower seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:ELISA analysis of soybean trypsin inhibitors in processed foods.</h2>
			<h3>PMID:1716818  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brandon DL(1), Bates AH, Friedman M.
				<br/> 
				年份：1991
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean proteins are widely used in human foods in a variety of forms, including infant formulas, flour, <span class="trait">protein</span> concentrates, <span class="trait">protein</span> isolates, soy sauces, textured soy fibers, and tofu. The presence of inhibitors of digestive enzymes in soy proteins impairs the nutritional quality and possibly the safety of soybeans and other legumes. Processing, based on the use of heat or fractionation of <span class="trait">protein</span> isolates, does not completely inactivate or remove these inhibitors, so that residual amounts of inhibitors are consumed by animals and humans. New monoclonal antibody-based immunoassays can measure low levels of the soybean Kunitz trypsin inhibitor (KTI) and the Bowman-Birk trypsin and chymotrypsin inhibitor (BBI) and the Bowman-Birk foods. The enzyme-linked immunosorbent assay (ELISA) was used to measure the inhibitor content of soy concentrates, isolates, and flours, both heated and unheated; a commercial soy infant formula; <span class="gene">KTI</span> and BBI with rearranged disulfide bonds; browning products derived from heat-treatment of <span class="gene">KTI</span> with glucose and starch; and <span class="gene">KTI</span> exposed to <span class="trait">high</span> pH. The results indicate that even low inhibitor isolates contain significant amounts of specific inhibitors. Thus, infants on soy formula consume about 10 mg of <span class="gene">KTI</span> plus BBI per day. The immunoassays complement the established enzymatic assays of trypsin and chymotrypsin inhibitors, and have advantages in (a) measuring low levels of inhibitors in processed foods; and (b) differentiating between the Kunitz and Bowman-Birk inhibitors. The significance of our findings for food safety are discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:alpha2-Macroglobulin from hemolymph of the freshwater crayfish Astacus astacus.</h2>
			<h3>PMID:1714366  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Stöcker W(1), Breit S, Sottrup-Jensen L, Zwilling R.
				<br/> 
				年份：1991
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						1. A <span class="trait">high</span> mol. wt proteinase inhibitor has been purified from the haemolymph of the freshwater crayfish Astacus astacus. 2. The <span class="trait">protein</span> is a disulphide-bonded dimer (Mr 390,000) of two identical polypeptide chains (Mr 185,000). 3. The inhibitor displays a broad specificity and protects trypsin from inhibition by soybean trypsin inhibitor and thus is similar to vertebrate alpha 2-macroglobulin. 4. The alpha 2-macroglobulin-like inhibitor from Astacus interacts with bovine trypsin in an equimolar stoichiometry thereby decreasing tryptic hydrolysis of N-benzoyl-L-arginine-ethylester to 50% residual activity. In contrast, the activity of Astacus protease, a digestive zinc proteinase from crayfish toward succinyl-alanyl-alanyl-alanyl-4-nitroanilide is inhibited almost completely. 5. Sensitivity of the inhibitor to methylamine and autolytic cleavage suggests the presence of an internal thioester bond. 6. The N-terminal amino acid sequence of Astacus alpha 2-macroglobulin is strongly related to the alpha 2-macroglobulins from Pacifastacus leniusculus (91% identity) and from the lobster Homarus americanus (72% identity). In contrast, only 25% of the residues are identical with the alpha 2-macroglobulin from the horseshoe crab Limulus polyphemus. There is also a faint similarity to human complement <span class="trait">protein</span> <span class="gene">C3</span> and human alpha 2-macroglobulin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular genetic analyses of the soybean mosaic virus <span class="gene">NIa</span> proteinase.</h2>
			<h3>PMID:2212987  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ghabrial SA(1), Smith HA, Parks TD, Dougherty WG.
				<br/> 
				年份：1990 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recombinant DNA molecules containing cDNA to a soybean mosaic virus (SMV) RNA genome were constructed and partial nucleotide sequences determined for two cDNA inserts, pSMV-34 and pSMV-35. Comparison of the predicted amino acid sequence encoded by the pSMV-34 cDNA insert to other potyvirus <span class="trait">protein</span> sequences revealed extensive homology with the region of the genome encoding the <span class="gene">NIa</span> proteinase, with conservation of the amino acids proposed to form the catalytic triad of the active site. Cell-free transcription and translation of the cloned cDNA sequence containing the <span class="gene">NIa</span> open reading frame and flanking sequences revealed that <span class="gene">NIa</span> proteinase sequences, which were expressed as part of a <span class="trait">high</span> Mr precursor, were able to undergo proteolytic processing. Alteration of the codon for one of the putative active site residues by site-directed mutagenesis eliminated processing and resulted in the accumulation of a <span class="trait">high</span> Mr precursor. Based on predicted amino acid sequences at five putative cleavage sites within the SMV polyprotein, a consensus SMV NI a proteinase cleavage sequence of Glu/Asn-Xaa-Val-Xaa-Xaa-Gln decreases Gly/Ser was proposed. The SMV <span class="gene">NIa</span> proteinase and its putative cleavage sites maintained motifs found in other potyviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nutritive characteristics of pearl millet grain in beef cattle diets.</h2>
			<h3>PMID:2384397  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hill GM(1), Hanna WW.
				<br/> 
				年份：1990 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two trials were conducted to compare pearl millet grain with corn and grain sorghum in cattle diets. Grain portions of diets in metabolism and feedlot experiments contained 73% corn and 6% soybean meal (C); 76.2% grain sorghum and 2.8% soybean meal (GS); and 79% pearl millet (PM). In the metabolism trial (replicated 3 x 3 latin squares; six steers), apparent digestibilities of DM and OM were higher (P less than .05) for C than for <span class="gene">GS</span> or PM diets. Ether extract and CP digestibilities were higher (P less than .05) for C and PM than GS. Dietary TDN was higher (P less than .05) for C compared with <span class="gene">GS</span> or PM diets. Fecal N was higher (P less than .05) for <span class="gene">GS</span> than for C or PM, urinary N was higher (P less than .05) for PM than C or GS, but retained N was similar (P greater than .05) for C, <span class="gene">GS</span> and PM diets. In a 90-d feedlot trial (18 individually fed heifers), a trend was observed for higher (P less than .20) ADG on C compared with PM; however, feed/gain was similar for C, <span class="gene">GS</span> and PM diets (8.2, 9.1 and 8.5 kg feed/kg gain, respectively). Ruminal fluid acetate:propionate ratios were lower (P less than .05) at 3.5 and 7 h postfeeding for PM compared with C or <span class="gene">GS</span> diets on d 83. Pearl millet grain may be used as a feed grain for beef cattle, but diets should be formulated to efficiently utilize the <span class="trait">high</span> quantity of <span class="trait">protein</span> (about 14% of DM) in this grain as a substitute for supplemental <span class="trait">protein.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of a heat shock inducible expression cassette for plants: characterization of parameters for its use in transient expression assays.</h2>
			<h3>PMID:2102878  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ainley WM(1), Key JL.
				<br/> 
				年份：1990 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A heat-inducible expression cassette has been constructed to study the conditional expression of sense or antisense orientations of any sequence of interest in transgenic plants or plant tissues. The construct includes the promoter and all but 5 bases of the mRNA leader from the soybean <span class="gene">Gmhsp17.5-E</span> gene, the polylinker from pUC18 (modified to remove the ATG), and a fragment that contains the polyadenylation signal and site from the nopaline synthase gene. Analysis of transient expression of a construct containing the beta-glucuronidase (GUS) coding sequence cloned in the cassette and introduced into Nicotiana plumbaginifolia protoplasts by electroporation shows that the promoter has <span class="trait">high</span> expression at heat shock temperatures. This construct is expressed at a roughly 80-fold higher level per unit time than a cauliflower mosaic virus 35S gene promoter-GUS construction. The heat shock promoter is regulated positively by supercoiling in this transient assay system. The level of expression of HS-GUS constructions with the polyadenylation sites from either the nopaline synthase gene or the <span class="gene">Gmhsp17.5-E</span> gene was similar. Constructs with a perfect fusion at the 5' end had higher levels of expression than those with the corresponding nonperfect transcriptional fusion.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of structural modifications on the assembly of a glycinin subunit.</h2>
			<h3>PMID:2152166  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dickinson CD(1), Scott MP, Hussein EH, Argos P, Nielsen NC.
				<br/> 
				年份：1990 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A <span class="gene">Gy4</span> glycinin cDNA was modified and used to produce structurally altered 11S storage <span class="trait">protein</span> subunits. We evaluated these modified subunits for their ability to assemble into oligomers. Alterations made in the acidic polypeptide changed the subunit solubility characteristics but did not eliminate assembly. Modifications in the basic polypeptide usually eliminated assembly of subunits into trimers. A region exhibiting <span class="trait">high</span> natural variability located at the COOH terminus of the acidic polypeptide that we have designated the hypervariable region was also studied. Extensive deletions and insertions were tolerated in the hypervariable region without perturbing subunit assembly. Some of the insertions significantly increased the methionine content in the <span class="gene">Gy4</span> glycinin subunit. Together, our results indicated that the structure of the basic polypeptide was more critical for assembly of trimers than that of the acidic polypeptide, an observation that implies that the basic polypeptides direct trimer formation. The assembly assays described here will be useful in efforts to improve seed quality. Using them, the effects of modifications to the storage <span class="trait">protein</span> subunits can be rapidly evaluated before introducing the mutated genes into plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of soybean lectin gene deletions in tobacco.</h2>
			<h3>PMID:2379327  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lindstrom JT(1), Vodkin LO, Harding RW, Goeken RM.
				<br/> 
				年份：1990
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A series of constructs containing the developmentally regulated soybean lectin gene (Le1) were used to transform tobacco plants in order to assess developmental and quantitative regulation conferred by flanking sequences. The largest of the lectin constructs contained approximately 3,000 base pairs (bp) of <span class="gene">Le1</span> 5 flanking region and 1,500 bp of the 3 flanking region. The smallest construct contained no 5 flanking region and 194 bp of the 3 flanking region. ELISA assays of lectin in individual tobacco seeds and Southern blot analyses confirmed that most constructs were inherited as unique insertion events. Maximal expression of <span class="gene">Le1</span> required more than 338 bp of 5 sequence, indicating that far upstream factors are involved in quantitative control of lectin expression. Lectin expression declined more than 80% between deletions with 1,700 versus 338 bp of 5 flanking sequence. In contrast, developmental control of lectin expression was maintained by <span class="gene">Le1</span> inserts with only 190 bp of 5 sequence. The lectin promoter offers a potential means to target <span class="trait">high</span> levels of gene expression to the developing seeds of soybean or other dicotyledonous plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of dietary soybean meal on the microscopic anatomy of the small intestine in the early-weaned pig.</h2>
			<h3>PMID:2768128  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dunsford BR(1), Knabe DA, Haensly WE.
				<br/> 
				年份：1989 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two experiments were conducted to determine the effect of soybean meal in postweaning diets on villus height (VH), villus shape and lamina propria depth (LPD) in the small intestine. Intestinal samples were taken at 25, 50 and 75% of the small intestine's length. The first experiment determined the normal effects of age on these structures in nursing pigs from 9 to 36 d of age. Villus height decreased (P less than .01 at the 50 and 75% locations) and LPD increased (P less than .01 at the 25% location) in a quadratic fashion with increasing age. In the second experiment, pigs were weaned at 21 d of age and fed one of three 20% <span class="trait">protein</span> diets: 1) <span class="trait">protein</span> from hydrolyzed casein (CAS), 2) soybean meal (SBM) or 3) a conventional corn-soybean meal diet (CSBM). Intestinal samples were taken 0, 3, 6, 9, 12 and 15 d post-weaning. Villus height decreased dramatically the first 3 d postweaning, with a continuing decline until 12 d post-weaning. Averaged over the d 3 to 15 interval, VH were generally greater for pigs fed <span class="gene">CAS</span> than for pigs fed CSBM or SBM. Pigs fed SBM or CSBM had deformed villi, and pigs fed SBM had greater (P less than .05) LPD at all intestinal locations, suggesting less mature enterocytes on the villi. These data suggest that feeding <span class="trait">high</span> concentrations of soybean meal to the pig post-weaning has a detrimental effect on the small intestine, but this effect was minimized when soybean meal was fed as part of a corn-based diet.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Formation of the N-N bond from nitric oxide by a membrane-bound cytochrome bc complex of nitrate-respiring (denitrifying) Pseudomonas stutzeri.</h2>
			<h3>PMID:2542222  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heiss B(1), Frunzke K, Zumft WG.
				<br/> 
				年份：1989 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitric oxide (NO) reductase was solubilized by Triton X-100 from the membrane fraction of Pseudomonas stutzeri ZoBell and purified 100-fold to apparent electrophoretic homogeneity. The enzyme consisted of two polypeptides of Mr 38,000 and 17,000 associated with heme b and heme c, respectively. Absorption maxima of the reduced complex were at 420.5, 522.5, and 552.5 nm, with a shoulder at 560 nm. The electron paramagnetic resonance spectrum was characteristic of <span class="trait">high-</span> and low-spin ferric heme proteins; no signals typical for iron-sulfur proteins were found. Nitric oxide reductase stoichiometrically transformed NO to nitrous oxide in an ascorbate-phenazine methosulfate-dependent reaction with a specific activity of 11.8 mumols/min per mg of <span class="trait">protein.</span> The activity increased to 40 mumols upon the addition of soybean phospholipids, n-octyl-beta-D-glucopyranoside, or its thio derivative to the assay system. Apparent Km values for NO and phenazine methosulfate were 60 and 2 microM, respectively. The pH optimum of the reaction was at 4.8. Cytochrome co was purified from P. stutzeri to permit its distinction from NO reductase. Spectrophotometric binding assays and other criteria also differentiated NO reductase from the respiratory cytochrome <span class="gene">bc1</span> complex.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of <span class="gene">Gmhsp26-A,</span> a stress gene encoding a divergent heat shock <span class="trait">protein</span> of soybean: heavy-metal-induced inhibition of intron processing.</h2>
			<h3>PMID:2835661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Czarnecka E(1), Nagao RT, Key JL, Gurley WB.
				<br/> 
				年份：1988 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We determined the DNA sequence and mapped the corresponding transcripts of a genomic clone containing the <span class="gene">Gmhsp26-A</span> gene of soybean. This gene is homologous to the previously characterized cDNA clone pCE54 (E. Czarnecka, L. Edelman, F. Schöffl, and J. L. Key, Plant Mol. Biol. 3:45-58, 1984) and is expressed in response to a wide variety of physiological stresses including heat shock (HS). S1 nuclease mapping of transcripts and a comparison of the cDNA sequence with the genomic sequence indicated the presence of a soybean seedlings with either CdCl2 or CuSO4. Analysis of the 5' termini of transcripts indicated the presence of one major and at least two minor start sites. In each case, initiation occurred 27 to 30 base pairs downstream from a TATA-like motif, and thus each initiation site appears to be promoted by the activity of a separate subpromoter. The three subpromoters are all associated with sequences showing low homology to the HS consensus element of Drosophila melanogaster HS genes and are differentially induced in response to various stresses. Within the carboxyl-terminal half of the <span class="trait">protein,</span> hydropathy analysis of the deduced amino acid sequence indicated a <span class="trait">high</span> degree of relatedness to the small HS proteins. A comparison of the primary amino acid sequence of hsp26-A with sequences of the small HS proteins suggested that this stress <span class="trait">protein</span> is highly diverged and may therefore be specialized for stress adaptation in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nucleotide sequence analysis of soybean small heat shock <span class="trait">protein</span> genes belonging to two different multigene families.</h2>
			<h3>PMID:3351943  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Raschke E(1), Baumann G, Schöffl F.
				<br/> 
				年份：1988 Feb 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In soybean, the small heat shock proteins of 15 to 18 kDa are encoded in the nucleus by at least two different multigene families, designated class I and class VI. Genomic DNA sequences of two new heat shock genes and flanking regions were determined: <span class="gene">Gmhsp18.5-C,</span> a class I gene, and <span class="gene">Gmhsp17.9-D,</span> the first known class VI gene. Comparison of both genes revealed a moderate homology (approx. 38%) mainly within the 3' ends of their coding regions. Hydropathic characterizations and secondary-structure predictions of the deduced amino acid sequences revealed two conserved domains within the C-terminal halves of the polypeptides that are also present in related proteins of other organisms. The transcription of both genes is heat shock dependent and the mRNA start sites, as determined by S1 nuclease mapping, are located downstream from typical TATA box sequences and multiple heat shock promoter elements such as 5' CT-GAA--TTC-AG. The putative promoter regions of the genes are preceded by long tracts of repetitive sequences with a <span class="trait">high</span> A + T content of 79 to 89%, which are bordered by runs of "simple sequences" such as (A) 12/13, (T)10 and (TA)10. Similar characteristic features are present in the promoter and 5'-flanking regions of other soybean heat shock genes. The possible function of these distinct sequences is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of extra- and intracellular phytases from Rhizopus oligosporus used in tempeh production.</h2>
			<h3>PMID:2856346  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sutardi(1), Buckle KA.
				<br/> 
				年份：1988 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizopus oligosporus strain CT11K2, commonly used in tempeh (fermented soybean) production produced both extra- and intracellular phytases. The enzymes were isolated from growth media and the cultured mould and partially purified by acetone fractionation, gel filtration on Sephadex G-100 and DEAE-cellulose chromatography. Intracellular phytase activity was higher than that of the extracellular enzyme. Both enzymes showed maximum activity at pH 4.5 and 55 degrees C, suggesting relatively <span class="trait">high</span> thermostability. The enzymes were partially inhibited by <span class="trait">high</span> concentrations of substrate. The Km and Vmax values of the extracellular phytase were 0.15 mM and 0.076 mumol Pi per min per ml DEAE-cellulose purified enzyme, respectively, and for the intracellular phytase were 0.17 mM and 0.34 mumol Pi per min per ml enzyme, respectively. Extracellular phytases showed inactivation and activation energies for the hydrolysis of phytic acid of approximately 28,300 <span class="gene">cal</span> per mol and 6100 <span class="gene">cal</span> per mol, respectively, while inactivation and activation energies for the intracellular phytase were approximately 33,200 per mol and 9500 <span class="gene">cal</span> per mol, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effect of fasting on enzyme levels in the enlarged and involuting rat pancreas.</h2>
			<h3>PMID:3689745  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Crass RA(1), Oates PS, Morgan RG.
				<br/> 
				年份：1987 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						1. The effect on pancreatic digestive enzyme levels of fasting and changes from a diet containing trypsin inhibitor (raw soya-bean flour, RSF) to diets free of trypsin inhibitor (heated soya-bean flour, HSF, or commercial rat chow) was studied in rats for up to 7 d. 2. In RSF-fed rats killed without fasting, enzyme levels were low, but after fasting for 24 h before killing there was a marked increase in all enzyme levels. Histological studies showed that pancreatic acinar cells from RSF-fed rats killed without fasting were devoid of zymogen granules, but following a 24 h fast there was a marked accumulation of zymogen granules which extend into the basal cytoplasm. Fasting either produced no change or a fall in enzyme levels in rats fasted after feeding <span class="gene">HSF</span> or chow continuously. 3. If animals fed on RSF were changed to <span class="gene">HSF</span> and either fed or fasted for 24 h up to the time of killing there was an increase in amylase (EC 3.2.1.1), trypsin (EC 3.4.21.4), lipase (triacylglycerol lipase; EC 3.1.1.3) and <span class="trait">protein</span> 1 d after the change, followed by a fall over the next 6 d to levels similar to those seen in rats fed on <span class="gene">HSF</span> continuously. 4. Animals changed from RSF to chow showed similar effects as far as trypsin, lipase and <span class="trait">protein</span> were concerned, but amylase rose, to reach the level seen in rats fed on chow continuously (about ten times that seen in soya-bean-fed rats), after 2 d. 5. These results suggest that in the rats fed on RSF, pancreatic enzyme synthesis is rapid but secretion is equally rapid and intracellular enzyme levels are low. When these animals are fasted or changed to a diet free of trypsin inhibitor the rate of secretion falls but the <span class="trait">high</span> rate of synthesis continues for at least 24 h and enzymes accumulate in the pancreas. In studies of pancreatic enzyme levels in rats fed on trypsin inhibitor the extent of fasting before killing the animal is therefore an important variable. Such animals should probably not be fasted before study.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of cDNA for nodulin-75 of soybean: A gene product involved in early stages of root nodule development.</h2>
			<h3>PMID:16593857</h3>
				

				
			<p>
				作者：Franssen HJ(1), Nap JP, Gloudemans T, Stiekema W, Van Dam H, Govers F, Louwerse J, Van Kammen A, Bisseling T.
				<br/> 
				年份：1987 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Establishment of a nitrogen-fixing root nodule is accompanied by a developmentally regulated expression of nodulin genes, only some of which, the so-called early nodulin genes, are expressed in stages preceding actual nitrogen fixation. We have isolated soybean cDNA clones representing early nodulin genes and have studied clone pENOD2 in detail. The cDNA insert of this clone hybridizes to nodule-specific RNA of 1200 nucleotides in length. The RNA that was hybrid-selected by the cloned <span class="gene">ENOD2</span> DNA was in vitro translated to produce two nodulins with an apparent M(r) of 75,000, the N-75 nodulins. These two nodulins differ slightly in charge and one does not contain methionine. The amino acid sequence deduced from the DNA sequence shows that proline accounts for 45% of the 240 residues in these nodulins and the sequence contains at least 20 repeating heptapeptide units. The amino acid composition of none of the (hydroxy)proline-rich (glyco)proteins described in plants resembles the composition of the N-75 nodulins, especially with respect to the <span class="trait">high</span> glutamic acid and the low serine content. This suggests that the N-75 nodulins belong to a hitherto unidentified class of presumably structural proteins. The genes encoding the N-75 nodulins were found to be expressed in nodule-like structures devoid of intracellular bacteria and infection threads, indicating that these nodulins do not function in the infection process but more likely function in nodule morphogenesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Reconstitution of the rat liver vasopressin receptor coupled to guanine nucleotide-binding proteins.</h2>
			<h3>PMID:2954955  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dickey BF, Fishman JB, Fine RE, Navarro J.
				<br/> 
				年份：1987 Jun 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">V1</span> vasopressin receptor has been solubilized from rat liver membranes with the zwitterionic detergent 3-[(3-cholamidopropyl)dimethylammoniol]-1-propanesulfonate (CHAPS) and reconstituted into phospholipid vesicles. There is essentially complete solubilization of the receptor by 3% CHAPS at a <span class="trait">protein</span> concentration of 15 mg/ml. Reconstitution into soybean phospholipid vesicles is readily achieved either by gel filtration chromatography or by membrane dialysis. The binding of [3H]vasopressin to proteoliposomes is specific, saturable, reversible, and magnesium-dependent. In contrast, the detergent-soluble vasopressin receptor does not display specific binding. The apparent affinity of the reconstituted receptor for [3H]vasopressin is approximately 4-fold lower than that of the receptor in native membranes. In addition, the binding of [3H]vasopressin to reconstituted vesicles is not sensitive to 100 microM guanosine 5'-O-thiotriphosphate (GTP gamma S) as it is in native membranes. However, the apparent affinity of the reconstituted receptor for ligand approximates that of native membranes when membranes are prebound with vasopressin prior to solubilization and reconstitution into vesicles. Furthermore, vesicles reconstituted from membranes prebound with vasopressin show GTP gamma S sensitivity of [3H] vasopressin binding. This finding strongly suggests that vasopressin stabilizes a receptor-G-protein complex during solubilization. The rat liver vasopressin receptor is a glycoprotein, as shown by its specific binding to the lectin "wheat germ agglutinin." The vasopressin receptor can be reconstituted from the N-acetylglucosamine-eluted peak of a wheat germ agglutinin-Sepharose column, and [3H] vasopressin binding activity is purified 5-6-fold from membranes by this chromatographic procedure. The functionality of the partially purified receptor is indicated by its ability to bind ligand with <span class="trait">high</span> affinity and by its ability to functionally interact with a G-protein when vasopressin is bound prior to solubilization.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of a rat salivary sialoglycoprotein complex which agglutinates Streptococcus mutans.</h2>
			<h3>PMID:3570462  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Brack CM, Reynolds EC.
				<br/> 
				年份：1987 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rat saliva agglutinated Streptococcus mutans Ingbritt and NCTC 10449 and Streptococcus sanguis NCTC 7864 but not S. mutans NCTC 10921, <span class="gene">GS</span> 5, or LM 7, Streptococcus sobrinus 6715-13 or OMZ 65, or Streptococcus cricetus HS 6, as measured turbidometrically. The specificity of agglutination by rat saliva was the same as that by human saliva. Agglutination was associated with a mucin complex (rat salivary agglutinin complex [rS-A]) of sulfated sialoglycoproteins, with a trace of associated lipid and an apparent Mr of 1.6 X 10(6), isolated by gel-filtration Fast <span class="trait">Protein</span> Liquid Chromatography. The complex was dissociated in a high-ionic-strength buffer containing 6 M urea and then fractionated by gel filtration and anion-exchange Fast <span class="trait">Protein</span> Liquid Chromatography into four sulfated sialoglycoprotein components, designated rS-A-1Q1, rS-A-1Q2, rS-A-1Q3, and rS-A-2, with rS-A-1Q2 being polydisperse through differential glycosylation of the polypeptide backbone. The dissociation destroyed agglutination activity. The polypeptide backbones contained up to 42% serine plus threonine and up to 40% glycine plus alanine plus proline plus valine. The carbohydrate moiety of the rS-A sialoglycoproteins consisted of N-acetylgalactosamine, sialate, galactose, fucose, N-acetylglucosamine, and small amounts of mannose, with the predominant sugar being N-acetylgalactosamine. Agglutination was inhibited by 1 mM EDTA but was restored by 1.5 mM CaCl2. Agglutination was also inhibited by 5 mM CaCl2; nonimmune sera; cationic polymers; and wheat germ, lentil, soybean, and peanut lectins. However, agglutination was not affected by lipoteichoic acid, various anionic proteins, or various sugars. Neuraminidase treatment of rS-A did not affect activity, but tryptic digestion of S. mutans did prevent agglutination. The results are consistent with calcium bridging the negative groups within the rS-A complex and allowing the approach of rS-A to the bacterial cell surface to effect a specific conformational attachment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A calcium-dependent but calmodulin-independent <span class="trait">protein</span> kinase from soybean.</h2>
			<h3>PMID:16665348</h3>
				

				
			<p>
				作者：Harmon AC(1), Putnam-Evans C, Cormier MJ.
				<br/> 
				年份：1987 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A calcium-dependent <span class="trait">protein</span> kinase activity from suspension-cultured soybean cells (Glycine max L. Wayne) was shown to be dependent on calcium but not calmodulin. The concentrations of free calcium required for half-maximal histone <span class="gene">H1</span> phosphorylation and autophosphorylation were similar ( approximately 2 micromolar). The <span class="trait">protein</span> kinase activity was stimulated 100-fold by &gt;/=10 micromolar-free calcium. When exogenous soybean or bovine brain calmodulin was added in <span class="trait">high</span> concentration (1 micromolar) to the purified kinase, calcium-dependent and -independent activities were weakly stimulated (&lt;/=2-fold). Bovine serum albumin had a similar effect on both activities. The kinase was separated from a small amount of contaminating calmodulin by sodium dodecyl sulfate polyacrylamide gel electrophoresis. After renaturation the <span class="trait">protein</span> kinase autophosphorylated and phosphorylated histone <span class="gene">H1</span> in a calcium-dependent manner. Following electroblotting onto nitrocellulose, the kinase bound (45)Ca(2+) in the presence of KCl and MgCl(2), which indicates that the kinase itself is a high-affinity calcium-binding <span class="trait">protein.</span> Also, the mobility of one of two kinase bands in SDS gels was dependent on the presence of calcium. Autophosphorylation of the calmodulin-free kinase was inhibited by the calmodulin-binding compound N-(6-aminohexyl)-5-chloro-1-naphthalene sulfonamide (W-7), showing that the inhibition of activity by W-7 is independent of calmodulin. These results show that soybean calcium-dependent <span class="trait">protein</span> kinase represents a new class of <span class="trait">protein</span> kinase which requires calcium but not calmodulin for activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cytochrome oxidase subunit II gene is adjacent to an initiator methionine tRNA gene in soybean mitochondrial DNA.</h2>
			<h3>PMID:2834094  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grabau EA(1).
				<br/> 
				年份：1987
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nucleotide sequence of the cytochrome oxidase subunit II (COII) gene from soybean mitochondria reveals a <span class="trait">high</span> degree of sequence conservation to other plant <span class="gene">COII</span> sequences. The greatest homology is to pea (97.9%) and the homology extends for some distance both 5' and 3' from the gene. Upstream from the breakpoint in homology with the pea sequence there is a tRNA gene identified as the initiator methionine by homology to known plant tRNA genes. The <span class="gene">COII</span> coding sequence appears to be present at a single location in the soybean mitochondrial genome, however some portion of the 5' flanking region is repeated at other sites in the genome. The transcription from the cloned region is complex and is complicated by the presence of the repeated sequence.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of regulatory elements in a plant embryo-specific gene.</h2>
			<h3>PMID:2430294  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen ZL, Schuler MA, Beachy RN.
				<br/> 
				年份：1986 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previously we demonstrated the expression of a plant embryo-specific gene encoding the <span class="gene">alpha'</span> subunit of beta-conglycinin, a seed storage <span class="trait">protein</span> of soybean (Glycine max), in transgenic petunia plants. To examine the regulatory elements that control the expression of this embryo-specific gene <span class="gene">(Gmg17.1),</span> a series of deletion mutants was made that contain the alpha'-subunit gene flanked in the 5' direction from +14 nucleotides to -8.5 kilobases (kb) relative to the site of transcription initiation. Each of these deletion mutants was introduced into the genome of petunia cells with the help of Ti-plasmid-derived vectors. Petunia plants were regenerated from transformed cells and expression of the introduced soybean gene was examined. When the alpha'-subunit gene was flanked by 159 nucleotides upstream <span class="gene">(Gmg17.1</span> delta-159), the gene was expressed at a low level in immature embryos. When the gene was flanked by 257 nucleotides upstream of the site of transcription initiation <span class="gene">(Gmg17.1</span> delta-257), a <span class="trait">high</span> level of expression was obtained. An additional 8 kb of DNA sequence (which includes the sequence GTGGATAG at -560, which is identical to the core enhancer sequence of simian virus 40 and some animal genes) did not significantly increase the level of expression. The increase in expression level between the delta-159 and delta-257 mutants was at least 20-fold. Analysis of the nucleotides between delta-159 and delta-257 reveals four repeats of a 6-base-pair (G + C)-rich sequence (see formula in text). The deletion <span class="gene">Gmg17.1</span> delta-159 contains a single AACCCA sequence. We suggest that the (G + C)-rich repeats play a critical role in determining the level of expression of the transgenic plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of plane of nutrition from starting to and through the breeder period on reproductive performance of hen turkeys.</h2>
			<h3>PMID:3588479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ferket PR, Moran ET Jr.
				<br/> 
				年份：1986 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Large-type breeder hens were given <span class="trait">high</span> (HP) and low (LP) planes of nutrition from day of age to 31 weeks of age. Rearing treatments were then divided, and each was fed HP and LP breeder feeds to 57 weeks. <span class="trait">Protein</span> and metabolizable energy (ME) levels of the HP feeds approximated NRC (1977) recommendations, while LP feeds largely involved substitution of corn, fat, and soybean meal for wheat, barley, and oats, such that ME/protein ratio and relative content of essential amino acids were similar between treatments, respective of age. The LP rearing regimen led to reduced hen body weight at 31 weeks. Combining the LP rearing with LP breeder regimen resulted in continuously lighter weights than occurred with the other three treatments to 57 weeks. Carcass fat based on finish grade of sample hens was lower after LP than HP rearing at both 31 and 57 weeks, while the breeder treatments were without effect. Experimental variables neither influenced egg production nor egg weight, but a reduction in the proportion of yolk occurred that was attributable to LP rearing and to a lesser extent the LP breeder feed. Apparent <span class="trait">fertility</span> improved with hens that had been LP reared, while feeding the LP breeder led to reduced <span class="gene">pip</span> losses, but only if eggs were set within 7 days of being laid. The converse occurred with eggs set after 7 days storage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heme <span class="trait">protein</span> fluorescence versus pressure.</h2>
			<h3>PMID:3697474  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Marden MC, Hui Bon Hoa G, Stetzkowski-Marden F.
				<br/> 
				年份：1986 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fluorescence spectra of several ferric heme proteins have been measured vs. pressure to 6,000 bars. Sperm whale myoglobin (SW Mb), Aplysia myoglobin, leghemoglobin (Lb), and cytochrome <span class="gene">P450</span> all show excitation and emission spectra characteristic of tryptophan in proteins with peak emission at 330-340 nm. At one bar, the fluorescence is weak due to energy transfer to the heme group, which makes the <span class="trait">yield</span> a sensitive probe of <span class="trait">protein</span> unfolding at <span class="trait">high</span> pressure. After an initial decrease of a few percent per kbar, the <span class="trait">protein</span> shows a large increase in fluorescence at <span class="trait">high</span> pressure. The increase is pH dependent and the results indicate that several <span class="trait">high</span> pressure states occur. For SW Mb at 15 degrees C an increase of a factor of 20 occurs with midpoint at 2,000 bars at pH 5 and is only partially reversible, while the increase at pH 7 occurs at 4,000 bars and is only half as large and is completely reversible. Aplysia Mb and Lb show a similar effect, but unfold at a higher pressure than SW Mb. <span class="gene">P450</span> also shows a transition to a state of higher fluorescence, but the transition in this case is irreversible as a stable form, P420, is formed. The fluorescence intensity measurements permit an estimation of the increase in the TRY-heme distance in the <span class="trait">high</span> pressure state.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Activation of the 5-lipoxygenase pathway in E-mast cells by peanut agglutinin.</h2>
			<h3>PMID:3917471  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Razin E.
				<br/> 
				年份：1985 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mouse E-mast cells were differentiated and grown by culturing bone marrow cells in medium containing WEHI-3-conditioned medium. These cells possess surface receptors to the following agglutinins: peanut (PNA), concanavalin A (Con A), and soybean (Sb). One to 200 micrograms of PNA/10(6) E-mast cells selectively stimulated the generation of leukotriene <span class="gene">C4</span> (LTC4) in the absence of beta-hexosaminidase release. Exposure of 10(6) E-mast cells to 1 to 200 micrograms Con A or Sb had no effect either on preformed mediator release or on the generation of leukotrienes. LTC4 was quantitated by integrated UV absorbance after resolution by reverse phase <span class="trait">high</span> pressure liquid chromatography. The optimum release of LTC4 (13.2 ng/10(6) cells) was achieved by 50 micrograms of PNA/10(6) cells. The response is characterized by the inhibition by excess amounts of PNA. The amount of LTC4 generated during optimal PNA stimulation is lower than the amount produced after stimulation by IgE-antigen or by calcium ionophore A23187 (19.8 ng and 148 ng, respectively). The release of LTC4 began within 5 min after PNA stimulation, and reached a plateau within 45 to 60 min at 37 degrees C. This kinetic pattern is similar to that observed after calcium ionophore A23187 stimulation of these cells. The results suggest that PNA is capable of selectively activating the 5-lipoxygenation of arachidonic acid without affecting beta-hexosaminidase secretion. Apparently, separate biochemical events may serve to mobilize each class of mediators.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteolysis of cardiac <span class="gene">gap</span> junctions during their isolation from rat hearts.</h2>
			<h3>PMID:4009696  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Manjunath CK, Goings GE, Page E.
				<br/> 
				年份：1985
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Gap junctions (GJ) isolated from rat hearts in presence of the protease inhibitor phenylmethylsulfonylfluoride (PMSF) contain a Mr 44,000 to 47,000 major polypeptide and have a urea-resistant layer of fuzz on their cytoplasmic surfaces, whereas junctions isolated without PMSF are proteolyzed to a Mr 29,500 polypeptide by a serine protease and have smooth cytoplasmic surfaces (C.K. Manjunath, G.E. Goings & E. Page Am. J. Physiol. 246:H865-H875, 1984). Rat liver GJ isolated with or without PMSF contain a Mr 28,000 polypeptide and have smooth cytoplasmic surfaces. Here we examine the origin, type and inhibitor sensitivity of the heart protease; why similar proteolysis is absent during isolation of rat liver <span class="gene">gap</span> junctions; and whether the Mr 44,000 to 47,000 cardiac GJ polypeptide is the precursor of the Mr 29,500 subunit. We show that the Mr 44,000 to 47,000 polypeptide corresponds to the unproteolyzed connexon subunit; that proteolysis of this polypeptide occurs predominantly during exposure to <span class="trait">high</span> ionic strength solution (0.6 M KI) which releases serine protease from mast cell granules; that this protease is inhibitable with PMSF and (less completely) soybean trypsin inhibitor and chymostatin; and that in vivo degranulation of mast cells by injecting rats with compound 48/80 fails to prevent breakdown of cardiac GJ during isolation. The results support the concept that GJ from rat heart and liver differ in <span class="trait">protein</span> composition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and initial characterization of tumoricidal monokine(s) from the human monocytic leukemia cell line THP-1.</h2>
			<h3>PMID:3918210  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Armstrong CA, Klostergaard J, Granger GA.
				<br/> 
				年份：1985 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cloned subline of the human monocytic leukemia cell line, THP-1, was induced to produce <span class="trait">high</span> levels of cytotoxic activity following an 18-hour phorbol myristate acetate (CAS: 16561-29-8) stimulation in vitro. This activity, termed monocyte cell line cytotoxin(s) (MCCT), was tested in vitro on different human continuous cell lines (Chang, ESH-7, <span class="gene">GM3104A,</span> HeLa, HT-1080, K562, Mel, T-24) and on primary human fibroblasts <span class="gene">(GM3468,</span> Manz). The continuous cell lines exhibited a spectrum of sensitivity to MCCT-containing supernatants whereas the primary fibroblasts were resistant to lysis. Enzymatic degradation and heat denaturation studies indicate that MCCT is a <span class="trait">protein.</span> Its bioactivity can be resolved into three lytic peaks after molecular sieving on Ultrogel AcA 44. The major peak, designated alpha MCCT, eluted with a molecular weight of 100,000-140,000 daltons. A minor peak, beta MCCT, was seen at 60,000-80,000 daltons, and a third, unstable minor peak, gamma MCCT, eluted at less than 10,000 daltons. The alpha-lytic peak was examined further and was found to migrate as a single peak in 7% native polyacrylamide gel electrolysis tube gels with an rf of 0.25-0.30. None of the MCCT forms were immunologically cross-reactive with human alpha-lymphotoxin. Various protease inhibitors known to inhibit monokine- and macrophage-mediated direct cell lysis in vitro were tested for their inhibitory effects on alpha MCCT activity. The irreversible binding inhibitor N alpha-p-tosyl-L-lysyl chloromethyl ketone inhibited the biologic activity of alpha MCCT. The reversible binding inhibitors N alpha-p-tosyl-L-arginine methyl ester and soybean trypsin inhibitor were able to block in vitro lytic activity when added to alpha MCCT in the presence of the target cell. In contrast, the inhibitors phenylmethylsulfonyl fluoride, L-1-tosylamide, 2-phenylethyl chloromethyl ketone, and N alpha-acetyl-L-lysine methyl ester were not effective in blocking cytolysis. Finally, the hydrogen peroxide scavenger catalase inhibited alpha MCCT lytic activity in vitro; however, the hypochlorous acid scavengers alanine, serine, and valine were without effect.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Divergence and differential expression of soybean actin genes.</h2>
			<h3>PMID:2410251  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hightower RC, Meagher RB.
				<br/> 
				年份：1985 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DNA sequence analysis as well as genomic blotting experiments using cloned soybean actin DNA sequences as probes show that large sequence heterogeneity exists among members of the soybean actin multigene family. This heterogeneity suggested that the members of this family might be diverged in function and/or regulation. Five of the six soybean actin gene family members examined are shown to be significantly more diverged from one another than members of other known actin gene families. This <span class="trait">high</span> level of divergence was utilized in the preparation of actin gene-specific probes in the analysis of the complexity and expression of these members of the soybean actin gene family. Hybridization studies indicate that the six soybean actin genes fall into three classes with a pair of genes in each class. These six genes account for all but two actin gene fragments detected in the soybean genome. We have compared the relative steady state mRNA levels of these classes of soybean actin genes in three organs of soybean. We find that actin genes SAc6 and SAc7 are most highly expressed accounting for 80% of all actin mRNA with respect to the six soybean actin genes examined. Actin genes SAc3 and SAc1 are expressed at intermediate and low levels respectively; and SAc2 and <span class="gene">SAc4</span> are expressed at barely detectable levels. Four of the six soybean actin genes appear to be expressed at the same level in root, shoot and hypocotyl. SAc3 and SAc7 genes appear to be more highly expressed in shoot and 2,4-dichlorophenoxyacetic acid-induced hypocotyl than in root and hypocotyl.(ABSTRACT TRUNCATED AT 250 WORDS)
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Short-term effects of feeding raw soya flour on pancreatic cell turnover in the rat.</h2>
			<h3>PMID:6209986  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Oates PS, Morgan RG.
				<br/> 
				年份：1984 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pancreatic acinar cell turnover was studied after a 48-h fast and in rats fed raw soya flour (RSF) for up to 28 days. Feeding RSF for 2 days resulted in a significant increase in pancreatic weight and RNA content while <span class="trait">protein</span> was increased by the 4th day compared with rats fasted for 48 h. RSF also resulted in a significant increase in RNA by the 4th day and weight and <span class="trait">protein</span> by the 7th day compared with rats fed heated soya flour (HSF). This pancreatic hypertrophy was maintained for the rest of the study period. Two days after starting RSF, pancreatic DNA synthesis, measured by the rate of incorporation of [3H]thymidine into pancreatic DNA, had increased sixfold compared with that in animals fed <span class="gene">HSF</span> but returned to control values again by the 4th day on the diet. Autoradiography showed that this increase in DNA synthesis occurred in both acinar and duct cells, with turnover in acinar cells preceding that in duct cells. A second more gradual rise in DNA synthesis was seen from the 7th to 28th day. This peak in DNA synthesis was associated with an increased total pancreatic DNA content and occurred predominately in duct cells with a smaller contribution from acinar cells. Histological studies of the pancreas during the 1st wk showed cell damage and tissue necrosis, possibly due to exposure to <span class="trait">high</span> levels of cholecystokinin released by RSF. The first peak in DNA synthesis may be a regenerative response to this damage. The second more delayed peak appears to be hyperplasia in response to a trophic stimulus, again possibly mediated by cholecystokinin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Amidolytic assay of human factor XI in plasma: comparison with a coagulant assay and a new rapid radioimmunoassay.</h2>
			<h3>PMID:6606454  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Scott CF, Sinha D, Seaman FS, Walsh PN, Colman RW.
				<br/> 
				年份：1984 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The traditional coagulant assay for plasma factor XI suffers from a relatively <span class="trait">high</span> coefficient of variation, the need for rare congenitally deficient plasma, and a poor correlation between precision and sensitivity. We have developed a simple functional amidolytic assay for factor XI in plasma using the chromogenic substrate PyrGlu-Pro-Arg-p-nitroanilide (S-2366). After inactivation of alpha 1-antitrypsin, <span class="gene">CI</span> inhibitor, and other plasma protease inhibitors with CHCI3, plasma was incubated with kaolin, in the absence of added calcium, which limited the enzymes formed to those dependent on contact activation. Soybean trypsin inhibitor was used to minimize the action of kallikrein on the substrate. Once the reaction was complete, corn trypsin inhibitor was used to inactive factor XIIa, the enzyme generated by exposure of plasma to negatively charged surfaces, which had activated the factor XI. The assay is highly specific for factor XI, since plasma totally deficient in that zymogen yielded only 1%-3% of the enzymatic activity in normal plasma under identical conditions. The requirements for complete conversion of factor XI to XIa in plasma within 60 min were, respectively, factor XII, 0.6 U/ml, and <span class="trait">high</span> molecular weight kininogen, 0.2 U/ml. Prekallikrein was not an absolute requirement for complete activation but did accelerate the reaction. The intraassay coefficient of variation was 3.4%, and the mean of 35 normal plasmas was 1.00 U +/- 0.24 SD. In addition, a new rapid radioimmunoassay was devised using staphylococcal <span class="trait">protein</span> A as the precipitating agent for a complex of factor XI antigen with monospecific rabbit antibody. The mean was 1.01 U +/- 0.30 SD. The correlation coefficients for amidolytic versus coagulant and amidolytic versus radioimmunoassay were r = 0.95 for the former and 0.96 for the latter. Thus, a simple, accurate amidolytic assay and a radioimmunoassay have been devised for measuring factor XI in plasma that correlate well with the coagulant activity of factor XI, as determined in our laboratory.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glycine mottle virus, a possible member of the tombusvirus group.</h2>
			<h3>PMID:6202654  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Behncken GM, Dale JL.
				<br/> 
				年份：1984
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine mottle virus ( <span class="gene">GMeV</span> ) was isolated from Glycine tomentella and has a restricted distribution in north Queensland. It has a very narrow experimental host range, systemically infecting several Glycine spp., but not G. max (soybean). Local infection occurred in soybean and several other legumes, but none of six species of non-legumes became infected. <span class="gene">GMeV</span> is extremely stable and can be purified in <span class="trait">high</span> concentration from G. soja . Based on the following properties it is classified as a possible new member of the tombusvirus group: isometric virions with a diameter of 30 nm (25 nm in cell section) which sediment as a single component with S20,w = 138; single, major polypeptide of molecular weight 39,000; approximately equal to 20% RNA consisting of a single component of molecular weight 1.5 X 10(6); a <span class="trait">high</span> concentration of virions in the cytoplasm and nuclei of infected cells, especially in cytoplasmic extrusions into the vacuole; and the presence of vesicles in peroxisomes and chloroplasts in infected cells. <span class="gene">GMeV</span> differs from most tombusviruses in its extremely narrow host range, and its virions showed no serological relationship to those of known members of the tombusvirus group, nor those of any other isometric virus tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification and properties of ribulose 1,5-bisphosphate carboxylase from sunflower leaves.</h2>
			<h3>PMID:24271969</h3>
				

				
			<p>
				作者：Ranty B(1), Cavalie G.
				<br/> 
				年份：1982 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Extracts from sunflower leaves possess a <span class="trait">high</span> ribulose-1,5-bisphosphate (RuBP) carboxylase capacity but this enzyme activity is not stable. A purification procedure, developed with preservation of carboxylase activity by MgSO4, yielded purified RuBP carboxylase with <span class="trait">high</span> specific activity (40 nkat mg(-1) <span class="trait">protein).</span> Measurement of kinetic parameters showed <span class="trait">high</span> Km values (RuBP, HCO 3 (-) ) and <span class="trait">high</span> Vmax of the reaction catalyzed by this sunflower enzyme; the results are compared with those obtained for soybean carboxylase. Enzyme characteristics are discussed in relation to stabilization and activation procedures and to the <span class="trait">high</span> photosynthesis rates of this <span class="gene">C3</span> species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effect of dietary raw and autoclaved soya-bean <span class="trait">protein</span> fractions on growth, pancreatic enlargement and pancreatic enzymes in rats.</h2>
			<h3>PMID:7199936  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Naim M, Gertler A, Birk Y.
				<br/> 
				年份：1982 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						1. Raw soya-bean meal (RS) was fractionated into soya-bean lyophilized extract (SLE), soya-bean lyophilized residue (SLR), acid-precipitated proteins (APP) and whey proteins. 2. Trypsin (EC 3. 4. 21. 4)and chymotrypsin (EC 3. 4. 21.1) inhibitors (TI) were soluble at pH 8 and remained soluble after the extract was acidified to pH 4.4 Except for whey, heating abolished, almost totally, their inhibiting activity. 3. Feeding SLE diet <span class="trait">(high</span> TI content) and <span class="gene">APP</span> diet (low TI content) resulted in growth depression below the RS level. Feeding the SLR diet resulted in an optimal growth. Feeding diets containing heated fractions improved the growth rate though not to the level observed with heated RS (HS) diet. 4. RS, SLE, <span class="gene">APP</span> and whey diets produced similar pancreatic enlargement which could be totally (RS, whey) or partially (SLE, APP) abolished by heating. 5. Feeding the RS diet reduced pancreatic amylase content. The factor responsible for this effect cofractionated with SLE and whey proteins. 6. Two groups of factors in the various diets were probably responsible for the elevation in pancreatic proteases. The first group were the heat-labile factors present in RS, SLE and whey whereas the second group resisted the heat treatment and were found in <span class="gene">APP</span> and SLR. 7. The results suggest that for optimal growth rate of rats, heat treatment should be given to the unfractionated soya-bean proteins rather than to the isolated fractions. The results further indicated that TI are not the only factors that can lead to pancreatic enlargement and changes in pancreatic enzymes composition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a C3bi-specific membrane complement receptor that is expressed on lymphocytes, monocytes, neutrophils, and erythrocytes</h2>
			<h3>PMID:6915077</h3>
				

				
					<h3>PMCID:2186557</h3>
					<h4>期刊:The Journal of Experimental Medicine</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ross </surname>  <given-names> GD </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lambris </surname>  <given-names> JD </given-names>  </name>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cells expressing a membrane C receptor (CR(3)) specific for C3b-inactivator- cleaved C3b (C3bi) were identified by rosette assay with C3bi-coated sheep erythrocytes (EC3bi) or C3bi-coated fluorescent microspheres (C3bi-ms). C3bi- ms, probably because of their smaller size, bound to a higher proportion of cells than did EC3bi. C3bi-ms bound to greater than 90 percent of mature neutrophils, 85 percent of monocytes, 92 percent of erythrocytes, and 12 percent of peripheral blood lymphocytes. Binding of C3bi-ms to neutrophils, monocytes, and erythrocytes was inhibited by fluid-phase C3bi, Fab anti-C3c, or Fab anti-C3d but was not inhibited by F(ab')(2) anti-CR(1) (C3b receptor) or F(ab')(2) anti-CR(2) (C3d receptor) nor by fluid-phase C3b, C3c, or C3d. This indicated that monocytes, neutrophils, and erythrocytes expressed C3bi receptors (CR(3)) that were separate and distinct from CR(1) and CR(2) and specific for a site in the <span class="gene">C3</span> molecule that was only exposed subsequently to cleavage of C3b by C3b inactivator and that was either destroyed, covered, or liberated by cleavage of C3bi into C3c and C3d fragments. Lymphocytes differed from these other cell types in that they expressed CR2 in addition to CRa. Lymphocyte C3bi-ms rosettes were inhibited from 50 to 84 percent by F(ab')(2)-anti-CR(2) or fluid-phase C3d, whereas C3d-ms rosettes were inhibited completely by F(ab')(2) anti-CR(2), fluid-phase C3bi, or fluid- phase C3d. Thus, with lymphocytes, C3bi was bound to CR(3), and in addition was bound to CR(2) by way of the intact d region of the C3bi molecule. In studies of the acquisition of C receptors occurring during myeloid cell maturation, the ability to rosette with C3bi-coated particles was detected readily with immature low-density cells, whereas this ability was nearly undetectable with <span class="trait">high</span> density mature polymorphonuclear cells. This absence of C3bi binding to polymorphs was not due to a loss of the CR(3) but instead was due to the maturation-linked acquisition of the abiity to secrete elastase that cleaved reagent particle-bound C3bi into CR(3)-unreactive C3d. Neither neutrophils nor monocytes bound C3d-coated particles at any stage of maturation. Assay of CR(3) with mature neutrophils required inhibition of neutrophil elastase with either soybean trypsin inhibitor or anti-elastase antibodies, and the amounts of these elastase inhibitors required to allow EC3bi rosette formation increased with neutrophil maturation. Because lymphocytes bound C3bi to CR(2) as well as to CR(3), specific assay of lymphocyte CR(3) required saturation of membrane CR(2) with Fab' anti-CR(2) before assay for rosettes with C3bi-ms. Only 3.5 percent of anti-CR(2)- treated peripheral blood lymphocytes bound C3bi-ms. Therefore, among normal blood lymphocytes the majority of the 12 percent C3bi-ms-binding cells expressed only CR(2) (8.5 percent), and the small proportion of C3bi-ms- binding cells that expressed CR(3) (3.5 percent) represented a distinct subset from the CR2(+) cells. Double-label assay indicated that 3.0 percent out of 3.5 percent of these CR(3)-bearing lymphocytes were B cells because they expressed membrane immunoglobulins. Of the remaining CR(3)(+) cells, 0.2 percent expressed either Leu-1 or 3A1 T cell antigens, and 0.6 percent expressed the OKM-1 monocyte-null lymphocyte determinant.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Cells expressing a membrane C receptor (CR(3)) specific for C3b-inactivator- cleaved C3b (C3bi) were identified by rosette assay with C3bi-coated sheep erythrocytes (EC3bi) or C3bi-coated fluorescent microspheres (C3bi-ms). C3bi- ms, probably because of their smaller size, bound to a higher proportion of cells than did EC3bi. C3bi-ms bound to greater than 90 percent of mature neutrophils, 85 percent of monocytes, 92 percent of erythrocytes, and 12 percent of peripheral blood lymphocytes. Binding of C3bi-ms to neutrophils, monocytes, and erythrocytes was inhibited by fluid-phase C3bi, Fab anti-C3c, or Fab anti-C3d but was not inhibited by F(ab&#x02019;)(2) anti-CR(1) (C3b receptor) or F(ab&#x02019;)(2) anti-CR(2) (C3d receptor) nor by fluid-phase C3b, C3c, or C3d. This indicated that monocytes, neutrophils, and erythrocytes expressed C3bi receptors (CR(3)) that were separate and distinct from CR(1) and CR(2) and specific for a site in the <span class="gene">C3</span> molecule that was only exposed subsequently to cleavage of C3b by C3b inactivator and that was either destroyed, covered, or liberated by cleavage of C3bi into C3c and C3d fragments. Lymphocytes differed from these other cell types in that they expressed CR2 in addition to CRa. Lymphocyte C3bi-ms rosettes were inhibited from 50 to 84 percent by F(ab&#x02019;)(2)-anti-CR(2) or fluid-phase C3d, whereas C3d-ms rosettes were inhibited completely by F(ab&#x02019;)(2) anti-CR(2), fluid-phase C3bi, or fluid- phase C3d. Thus, with lymphocytes, C3bi was bound to CR(3), and in addition was bound to CR(2) by way of the intact d region of the C3bi molecule. In studies of the acquisition of C receptors occurring during myeloid cell maturation, the ability to rosette with C3bi-coated particles was detected readily with immature low-density cells, whereas this ability was nearly undetectable with <span class="trait">high</span> density mature polymorphonuclear cells. This absence of C3bi binding to polymorphs was not due to a loss of the CR(3) but instead was due to the maturation-linked acquisition of the abiity to secrete elastase that cleaved reagent particle-bound C3bi into CR(3)-unreactive C3d. Neither neutrophils nor monocytes bound C3d-coated particles at any stage of maturation. Assay of CR(3) with mature neutrophils required inhibition of neutrophil elastase with either soybean trypsin inhibitor or anti-elastase antibodies, and the amounts of these elastase inhibitors required to allow EC3bi rosette formation increased with neutrophil maturation. Because lymphocytes bound C3bi to CR(2) as well as to CR(3), specific assay of lymphocyte CR(3) required saturation of membrane CR(2) with Fab&#x02019; anti-CR(2) before assay for rosettes with C3bi-ms. Only 3.5 percent of anti-CR(2)- treated peripheral blood lymphocytes bound C3bi-ms. Therefore, among normal blood lymphocytes the majority of the 12 percent C3bi-ms-binding cells expressed only CR(2) (8.5 percent), and the small proportion of C3bi-ms- binding cells that expressed CR(3) (3.5 percent) represented a distinct subset from the CR2(+) cells. Double-label assay indicated that 3.0 percent out of 3.5 percent of these CR(3)-bearing lymphocytes were B cells because they expressed membrane immunoglobulins. Of the remaining CR(3)(+) cells, 0.2 percent expressed either Leu-1 or 3A1 T cell antigens, and 0.6 percent expressed the OKM-1 monocyte-null lymphocyte determinant.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Purification and characterization of mRNA from soybean seeds. Identification of glycinin and beta-conglycinin precursors.</h2>
			<h3>PMID:6167584  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tumer NE, Thanh VH, Nielsen NC.
				<br/> 
				年份：1981 Aug 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Poly(A)-rich RNA was isolated from developing soybean seeds (Glycine max (L.) Merr.) and fractionated on linear log sucrose gradients. Two major fractions sedimenting at 18 S and 20 S were separated and then purified by further sucrose gradient fractionation. Both fractions were active as messengers when added to a rabbit reticulocyte lysate <span class="trait">protein</span> synthesis system. The 18 S fraction caused proteins migrating primarily to the 60,000-dalton region of a sodium dodecyl sulfate gel to be produced, while translation of the 20 S fraction preferentially directed the synthesis of polypeptides similar in size to the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin. Evidence that many of the 60,000-dalton polypeptides were related to glycinin and the <span class="trait">high</span> molecular weight 20 S translation products were related to beta-conglycinin was obtained by immunoprecipitation using monospecific antibodies against glycinin and beta-conglycinin, respectively. Sodium dodecyl sulfate-polyacrylamide gel electrophoresis analysis of the immunoprecipitated products revealed that the glycinin precursor region contained at least three different size components and that the family of glycinin precursors had larger apparent molecular weight (58,000-63,000) than the disulfide-linked complexes between acidic and basic glycinin subunits (57,000). Unlike the disulfide-linked glycinin complexes which were cleaved by disulfide reduction, glycinin precursors were insensitive to reducing agents. The alpha and <span class="gene">alpha'</span> subunits synthesized in vitro also had slightly larger apparent molecular weights than purified alpha and <span class="gene">alpha'</span> standards.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In Vitro Synthesis of the alpha and <span class="gene">alpha'</span> Subunits of the 7S Storage Proteins (Conglycinin) of Soybean Seeds.</h2>
			<h3>PMID:16661321</h3>
				

				
			<p>
				作者：Beachy RN(1).
				<br/> 
				年份：1980 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Messenger RNAs (mRNAs), isolated from immature soybean (Glycine max L., Merr.) seeds, that bound to oligo(dT)-cellulose were fractionated by centrifugation in sucrose density gradients containing dimethyl sulfoxide. mRNAs with sedimentation values between 21S and 25S coded for the in vitro translation of polypeptides with electrophoretic mobilities similar to those of the <span class="gene">alpha'</span> and alpha subunits of the 7S seed storage <span class="trait">protein.</span> <span class="trait">High</span> pressure liquid chromatographic analyses of the trypsin-induced fragments ("column fingerprinting") verified that the polypeptides produced in vitro were closely related to authentic <span class="gene">alpha'</span> and alpha subunits.The fractions of RNA that coded for the in vitro synthesis of these subunits contained three major species of nonribosomal RNA (molecular weights of 1.1, 0.84, and 0.75 x 10(6)) sufficiently large to code for the synthesis of the <span class="gene">alpha'</span> and alpha subunits.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Studies on soybean trypsin inhibitors, XII. Linear sequences of two soybean double-headed trypsin inhibitors, <span class="gene">D-II</span> and E-I.</h2>
			<h3>PMID:641033  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Odani S, Ikenaka T.
				<br/> 
				年份：1978 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean inhibitor <span class="gene">D-II</span> is an inhibitor of bovine trypsin. Sequence analysis was carried out on the reduced and S-carboxymethylated <span class="trait">protein</span> by conventional methods to establish the complete amino acid sequence. The sequence of <span class="gene">D-II</span> indicated <span class="trait">high</span> homology with other legume inhibitors, but it was unique because of the occurrence of identical residues (arginine) at both of the reactive sites. This structure is thought to reflect that of a prototype double-headed inhibitor. The possible evolutionary process of the legume double-headed inhibitors is discussed on this basis. Comparison with another soybean inhibitor C-II suggested that a single methionine (C-II)-glutamine (D-II) replacement at the P2'position resulted in the loss of alpha-chymotrypsin inhibitory activity of D-II. The results of a hydrogen peroxide oxidation experiment on C-II supported this suggestion. The sequence of the amino-terminal 21 residues of inhibitor E-I was determined using a sequentor. It was shown that this inhibitor lacks the amino-terminal nine residues of D-II.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthesis and increased production of <span class="trait">protein.</span></h2>
			<h3>PMID:727014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Platt SG, Bassham JA.
				<br/> 
				年份：1978
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosynthesis, the use of light energy in the conversion of CO2 and inorganic nutrients into plant material, is the ultimate source of the food <span class="trait">protein</span> necessary to man's existence. Given certain assumptions, the overall maximal theoretical photosynthetic efficiency of agricultural plants can be calculated. Actual measured maximal growth rates of plants are equivalent to efficiency levels well below that theoretical maximum. In air, <span class="gene">C4</span> plants can some closer to the theoretical value than <span class="gene">C3</span> plants, perhaps because <span class="gene">C4</span> plants avoid the occurrence of measurable photorespiration and oxygen inhibition of photosynthesis. Alfalfa, a <span class="gene">C3</span> legume, is an extremely productive <span class="trait">protein</span> source. Its <span class="trait">protein</span> <span class="trait">yield</span> per acre can surpass that of commonly grown <span class="gene">C4</span> crops (corn, sorghum) and <span class="gene">C3</span> seed crops (soybean, wheat, rice). Alfalfa leaf <span class="trait">protein</span> is of <span class="trait">high</span> nutritional quality and can apparently be used directly in the human diet, eliminating the <span class="trait">protein</span> loss involved in animal production. Plant <span class="trait">protein</span> productivity can be raised as part of an increase in overall crop <span class="trait">yield.</span> The growth of plants in atmospheres with elevated CO2 levels can result in increased yields. In <span class="gene">C3</span> plants this is due, at least in part, to the suppression of photorespiration and oxygen inhibition of photosynthesis. We have investigated the effect of CO2 concentration on alfalfa photosynthetic metabolism. Our results support the contention that alfalfa productivity can be increased by an environment of elevated CO2. A second approach toward increased plant <span class="trait">protein</span> productivity is through regulation of carbon flow during photosynthesis so as to increase portein production relative to that of other plant constituents. In particular, we have investigated whether ammonia (the form in which plants first incorporate nitrogen) can act to regulate leaf carbon metabolism. Our results indicate that NH4+, in part through stimulation of pyruvate kinase, brings about increased production of amino acids at the expense of sucrose production in alfalfa. The effect may be of considerable importance in the regulation of green leaf <span class="trait">protein</span> synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Studies on soybean trypsin inhibitors. X. Isolation and partial characterization of four soybean double-headed proteinase inhibitors.</h2>
			<h3>PMID:599140  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Odani S, Ikenaka T.
				<br/> 
				年份：1977 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Four Bowman-Birk type double-headed inhibitors (B, C-II, D-II, and E-I) were isolated from soybeans. Inhibitor B was different from Bowman-Birk inhibitor only in chromatographic behavior. One mole of C-II inhibited one mole each of bovine trypsin and bovine alpha-chymotrypsin, probably at the same site, and porcine elastase at another reactive site. In the ordinary assay system <span class="gene">D-II</span> and E-I inhibited only trypsin activity at a non-stoichiometric inhibitor-enzyme ratio of 1:1.4, and the complexes had rather <span class="trait">high</span> dissociation constants. These inhibitors were all inactive toward subtilisin BPN'.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of isoflavones on breast density in pre- and post-menopausal women: a systematic review and meta-analysis of randomized controlled trials</h2>
			<h3>PMID:20511398</h3>
				
					<h3>PMCID:2953939</h3>
					<h4>期刊:Human Reproduction Update</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hooper </surname>  <given-names> Lee </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Madhavan </surname>  <given-names> Giri </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tice </surname>  <given-names> Jeffrey A. </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Leinster </surname>  <given-names> Sam J. </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cassidy </surname>  <given-names> Aed&#x000ed;n </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 4 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Increased circulating estradiol concentrations are associated with increased risk of breast cancer in post-menopausal women ( <xref ref-type="bibr" rid="DMQ011C18"> Endogenous Hormones Breast Cancer Collaborative Group, 2002 </xref> ), whereas higher soy isoflavone intakes are associated with lower breast cancer risk in epidemiological studies. Cross-cultural studies suggest 3-fold lower rates of breast cancer in Asian populations, where soy consumption is <span class="trait">high</span> ( <xref ref-type="bibr" rid="DMQ011C53"> Messina <italic> et al </italic> ., 2006 </xref> ; <xref ref-type="bibr" rid="DMQ011C71"> Trock <italic> et al </italic> ., 2006 </xref> ). A recent meta-analysis of cohort and case&#x02013;control data on soy intake and breast cancer showed soy to be associated with a small reduction in risk ( <xref ref-type="bibr" rid="DMQ011C71"> Trock <italic> et al </italic> ., 2006 </xref> ) whereas another found a significant trend of reduced breast cancer risk with increasing soy food intake ( <xref ref-type="bibr" rid="DMQ011C81"> Wu <italic> et al </italic> ., 2008 </xref> ). In a large recent analysis of breast cancer patients from the Shanghai Breast Cancer Survival Study, soy food consumption (with average intakes of 47 mg/day of isoflavones) was inversely associated with risk of death [hazard ratio (HR) 0.71, 95% confidence interval (CI) 0.54&#x02013;0.92 for the highest compared with the lowest quartiles of soy intake] and breast cancer recurrence (HR 0.68, 95% <span class="gene">CI</span> 0.54&#x02013;0.87; <xref ref-type="bibr" rid="DMQ011C67"> Shu <italic> et al </italic> ., 2009 </xref> ). Despite these epidemiological data, results from the available intervention trials have been equivocal. No large, long-term randomized controlled trials (RCTs), powered to assess breast cancer as an outcome, have been published. It remains unclear whether isoflavones from different sources increase or decrease breast cancer risk in humans. A recent meta-analysis of side effects of phytoestrogens observed only 16 cases of breast cancer diagnoses, insufficient to assess effects on this outcome ( <xref ref-type="bibr" rid="DMQ011C68"> Tempfer <italic> et al </italic> ., 2009 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of <span class="trait">Drought</span> Tolerance of the Vietnamese Soybean Cultivars Provides Potential Resources for Soybean Production and Genetic Engineering</h2>
			<h3>PMID:24804248</h3>
				
					<h3>PMCID:3997955</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-0210-1900 </contrib-id>  <name>  <surname> Thu </surname>  <given-names> Nguyen Binh Anh </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 3 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						During examination of root characteristics at <span class="gene">V3</span> stage under well-watered conditions, we found that there was a significant difference in taproot length among the cultivars ( <xref ref-type="fig" rid="fig1"> Figure 1(a) </xref> ). According to the results, DT51, DT22, DT96, and DT84 were classified into the long taproot group (&#x0003e;50&#x02009;cm). DT26, W82, MTD772, MTD765, MTD751, DT20, and MTD777-2 had medium taproot length (40&#x02013;50&#x02009;cm). The remaining varieties, including MTD775-2, MTD176, and MTD720, showed short taproot length (&#x0003c;40&#x02009;cm). Among 14 varieties, DT51 exhibited the longest taproot length (59.3&#x02009;cm), while MTD720 had the shortest root length (36.2&#x02009;cm). DT51 also had the highest shoot length, making it a member of the <span class="trait">high</span> shoot length group (&#x0003e;35&#x02009;cm) that also includes DT84 and MTD765. A number of cultivars, such as DT20, MTD751, MTD176, MTD772, and DT96, exhibited medium shoot length (30&#x02013;35&#x02009;cm), while MTD720, MTD775-2, DT26, DT22, and W82 fell into the low shoot length category (&#x0003c;30&#x02009;cm) ( <xref ref-type="fig" rid="fig1"> Figure 1(b) </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						These data together demonstrated that DT51, which had the longest root length and <span class="trait">high</span> shoot length, and MTD720, which displayed the shortest root length and short shoot length, at both seedling and <span class="gene">V3</span> stages, might be the two contrasting drought-responsive cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						With regard to root DM, our data indicated that there was a significant difference in root DM among the cultivars at both seedling and <span class="gene">V3</span> stages ( <xref ref-type="fig" rid="fig1"> Figure 1(c) </xref> ). At seedling stage, <span class="trait">high</span> root DM group included MTD777-2, MTD720, and DT22 (&#x0003e;0.045&#x02009;g). DT20, DT51, DT26, MTD751, DT84, and MTD176 had medium root DM (from 0.035 to 0.045&#x02009;g), while W82, MTD772, DT96, MTD765, and MTD775-2 showed low root DM (&#x0003c;0.035&#x02009;g). At <span class="gene">V3</span> stage, the <span class="trait">high</span> root DM group included DT26, MTD777-2, and DT84 (&#x0003e;0.15&#x02009;g). DT22, DT20, W82, MTD765, DT96, and MTD772 exhibited medium root DM (0.09&#x02013;0.15&#x02009;g), whereas MTD751, MTD720, DT51, MTD176, and MTD775-2 had low root DM (&#x0003c; 0.09&#x02009;g). There was a slight difference in shoot DM among all the cultivars at seedling stage. MTD720, DT20, DT26, MTD765, and MTD751 had higher shoot DM (&#x0003e;0.2&#x02009;g) than others, such as those belonging to medium shoot DM group, including DT51, MTD777-2, DT84, W82, and MTD176 (0.18&#x02013;0.2&#x02009;g), and those classified into low shoot DM group, including MTD772, DT96, DT22, and MTD775-2 (&#x0003c;0.18&#x02009;g). Significant differences were recorded at <span class="gene">V3</span> stage ( <xref ref-type="fig" rid="fig1"> Figure 1(d) </xref> ). All cultivars could be divided into <span class="trait">high</span> (&#x0003e;0.5&#x02009;g), medium (0.4&#x02013;0.5&#x02009;g), and low (&#x0003c;0.4&#x02009;g) groups. DT84, MTD777-2, DT26, DT22, and MTD765 had <span class="trait">high</span> shoot DM, whereas medium group included DT20, W82, MTD751, and MTD176. The remaining cultivars (DT96, MTD772, DT51, MTD720, and MTD772) displayed low value of shoot DM.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, we recorded DT51 and MTD720 as two cultivars having contrasting drought-tolerant features, of which DT51 was the highest drought-tolerant cultivar, whereas MTD720 was the lowest drought-tolerant cultivar. This finding was also supported by a differential expression analysis of a subset of <italic> <span class="gene">GmNAC</span> </italic> genes [ <xref rid="B34" ref-type="bibr"> 34 </xref> ], which are known as transcriptional factors involved in regulation of plant response to <span class="trait">drought</span> [ <xref rid="B35" ref-type="bibr"> 35 </xref> &#x02013; <xref rid="B37" ref-type="bibr"> 37 </xref> ]. The expression of drought-responsive <italic> <span class="gene">GmNAC</span> </italic> s in roots of DT51 and MTD720 was significantly different. The better drought-tolerant capacity of DT51 was shown to be related to the higher number of drought-inducible <italic> <span class="gene">GmNAC</span> </italic> genes, as well as the higher number of <italic> <span class="gene">GmNAC</span> </italic> genes with higher transcript accumulation in comparison with MTD720 [ <xref rid="B34" ref-type="bibr"> 34 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Genomics: Developments through the Use of Cultivar &#x0201c;Forrest&#x0201d;</h2>
			<h3>PMID:18483614</h3>
				
					<h3>PMCID:2376204</h3>
					<h4>期刊:International Journal of Plant Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lightfoot </surname>  <given-names> David A. </given-names>  </name>  <xref ref-type="aff" rid="I1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 12 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematodes ( <italic> Heterodera </italic> <italic> glycines </italic> I.) are the most damaging pests of soybean worldwide [ <xref ref-type="bibr" rid="B75"> 73 </xref> ]. Development of resistant cultivars is the only viable control measure [ <xref ref-type="bibr" rid="B77"> 75 </xref> ]. <span class="trait">Resistance</span> genes have been found to be located on 17 of the 20 chromosomes by 2007. A combination of recessive genes is necessary to provide <span class="trait">resistance</span> against SCN populations because many are known to be capable of overcoming all known single <span class="trait">resistance</span> genes. SCN populations can be classified into 16 broad races or up to 1024 biotypes (HG Types) [ <xref ref-type="bibr" rid="B78"> 76 </xref> ] based on the host responses of 8 weedy indicator lines. SCN <span class="trait">resistance</span> in many other adapted and weedy cultivars [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ] shared the same loci underlying bigeneic inheritance in E &#x000d7; F [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. The E &#x000d7; F population was used to isolate candidate genes for those two loci ( <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> ; <xref ref-type="table" rid="tab4"> Table 4 </xref> ) that control <span class="trait">resistance</span> against SCN race 3 (HG Type 0). Alleles of the candidate genes were identified in many PIs through association studies [ <xref ref-type="bibr" rid="B38"> 38 </xref> , <xref ref-type="bibr" rid="B79"> 77 </xref> ]. Paralogs of both these genes were found at new locations in BAC libraries and whole genome shotgun (WGS) sequences [ <xref ref-type="bibr" rid="B80"> 78 </xref> , <xref ref-type="bibr" rid="B81"> 79 </xref> ]. They appear to be part of multigene families showing homoeology and intragenomic conserved synteny.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isoflavone Content of Soybean Cultivars from Maturity Group 0 to VI Grown in Northern and Southern China</h2>
			<h3>PMID:24882872</h3>
				
					<h3>PMCID:4033804</h3>
					<h4>期刊:Journal of the American Oil Chemists' Society</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jingying </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ge </surname>  <given-names> Yinan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Fenxia </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Bin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Shurong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Sun </surname>  <given-names> Junming </given-names>  </name>  <address>  <phone> +86-10-82105805 </phone>  <fax> +86-10-82105805 </fax>  <email> sunjunming@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lianzheng </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> MOA Key Laboratory of Soybean Biology (Beijing), Institute of Crop Science, Chinese Academy of Agricultural Sciences, Beijing, 100081 People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavone content is highly variable and regulated by genetic and environmental factors, and the qualitative-quantitative genetic character is controlled by several major and minor genes [ <xref ref-type="bibr" rid="CR5"> 5 </xref> &#x02013; <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. Significant differences in isoflavone content of different soybean varieties indicated the existence of genetic differences. Even the isoflavone content of one genotype variety planted at the same location demonstrated a change up to threefold in different years [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. Therefore, for isoflavone content, the yearly difference is more important than location and other environmental factors [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. In addition to the year factor, other environmental factors are also involved in isoflavone contents and components. The malonyl-glucoside type is easily transformed to the corresponding aglycone form under <span class="trait">high</span> temperature conditions [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ], but higher isoflavone concentration is accumulated at the seed maturing stage at low temperature [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. Light can also affect the isoflavone distribution and accumulation by adjusting the content and activity of key enzymes (such as PAL, CHS, <span class="gene">CHI</span> and IFS) in the isoflavone biosynthetic pathway [ <xref ref-type="bibr" rid="CR14"> 14 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Isoflavone content decreased with increases in storage time and temperature, at the same time, the malonyl-glucoside components reach a <span class="trait">high</span> level [ <xref ref-type="bibr" rid="CR16"> 16 </xref> &#x02013; <xref ref-type="bibr" rid="CR18"> 18 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluating the Age-Dependent Potential for <span class="trait">Protein</span> Deposition in Naked Neck Meat Type Chicken</h2>
			<h3>PMID:26479137</h3>
				
					<h3>PMCID:4494337</h3>
					<h4>期刊:Animals : an Open Access Journal from MDPI</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Daulat R. </given-names>  </name>  <xref ref-type="aff" rid="af1-animals-05-00056"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wecke </surname>  <given-names> Christian </given-names>  </name>  <xref ref-type="aff" rid="af1-animals-05-00056"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharifi </surname>  <given-names> Ahmad R. </given-names>  </name>  <xref ref-type="aff" rid="af2-animals-05-00056"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liebert </surname>  <given-names> Frank </given-names>  </name>  <xref ref-type="aff" rid="af1-animals-05-00056"> 1 </xref>  <xref rid="c1-animals-05-00056" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 09 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Dietary ingredients, experimental diets and excreta were analyzed according to the German standards [ <xref rid="B17-animals-05-00056" ref-type="bibr"> 17 </xref> ]. The N content was quantified using to the Dumas method (Leco <sup> &#x000ae; </sup> LP-2000, Leco <sup> &#x000ae; </sup> Instrument <span class="gene">GmbH,</span> Kirchheim, Germany) and CP was calculated with factor 6.25. Amino acids (AA) of the <span class="trait">protein</span> sources were analyzed by ion-exchange chromatography (Biochrom <sup> &#x000ae; </sup> 30, Biochrom Ltd. Cambridge, UK) following acid hydrolysis with and without an oxidation step for quantitative determination of sulfur-containing amino acids. Ether extract was analyzed following HCl hydrolysis of the feed samples.
					</p>
				
					<p>&nbsp;&nbsp;
						The model parameter <italic> b </italic> as an indicator of the dietary <span class="trait">protein</span> quality was derived following logarithmization and transformation of Equation (1): <disp-formula> <italic> b </italic> = [ln NR <sub> max </sub> T &#x02212; <span class="gene">ln</span> (NR <sub> max </sub> T &#x02212; NR)]/NI <label> (3) </label> </disp-formula> where <italic> b </italic> = model parameter for dietary <span class="trait">protein</span> quality; <span class="gene">ln</span> = natural logarithm; NR <sub> max </sub> T = theoretical maximum for daily NR; NR = nitrogen retention; NI = nitrogen intake.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Regioselectivity and Lipid Class Specificity of Lipases on Transesterification, Exemplified by Biodiesel Production</h2>
			<h3>PMID:25045169</h3>
				
					<h3>PMCID:4070466</h3>
					<h4>期刊:Journal of the American Oil Chemists' Society</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> &#x00160;ink&#x0016b;nien&#x00117; </surname>  <given-names> Dovil&#x00117; </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Adlercreutz </surname>  <given-names> Patrick </given-names>  </name>  <address>  <phone> +46-46-2224842 </phone>  <email> Patrick.Adlercreutz@biotek.lu.se </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Biotechnology, Lund University, P.O. Box 124, SE-221 00 Lund, Sweden  </aff>  <aff id="Aff2">  <label/> Department of Biochemistry and Molecular Biology, Vilnius University, M. K. &#x0010c;iurlionio g. 21, 03101 Vilnius, Lithuania  </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Alcoholysis of triolein was used as a model for biodiesel production. Silylation of the samples followed by GC analysis allowed quantification of all the reaction products. Isomers of both monoolein and diolein were separated. A typical chromatogram is shown in Fig. S1 in the supplementary material. The alcoholysis reaction was carried out with the four immobilized lipases with and without silica gel as an acyl migration catalyst. The addition of silica gel led to an increase in oleic acid formation with all the enzymes studied, and an increase in ethyl oleate <span class="trait">yield</span> after 48&#x000a0;h when using RA, RM, and TL, but not with CA, as illustrated in Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> . The highest ethyl oleate <span class="trait">yield</span> was obtained using CA. The main side products were monoolein and diolein in all cases. <fig id="Fig3"> <label> Fig.&#x000a0;3 </label> <caption> <p> Product composition after 48 h of ethanolysis of triolein catalyzed by lipases with or without silica. <italic> black <span class="gene">bar</span> </italic> triolein, <italic> dotted <span class="gene">bar</span> </italic> diolein, <italic> light grey <span class="gene">bar</span> </italic> monoolein, <italic> white <span class="gene">bar</span> </italic> oleic acid, <italic> dark grey <span class="gene">bar</span> </italic> ethyl oleate
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey</h2>
			<h3>PMID:17524145</h3>
				
					<h3>PMCID:1894642</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" contrib-type="author">  <name>  <surname> Swaminathan </surname>  <given-names> Kankshita </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kank@uiuc.edu </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Varala </surname>  <given-names> Kranthi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kvarala2@uiuc.edu </email>  </contrib>  <contrib id="A3" corresp="yes" contrib-type="author">  <name>  <surname> Hudson </surname>  <given-names> Matthew E </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mhudson@uiuc.edu </email>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 5 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The average read size of our survey was 109.5 bp, giving a maximum average open reading frame size of 37 amino acids. Consequently, reads that are derived completely from exonic sequence are a potential source of partial <span class="trait">protein</span> sequence. The <span class="gene">GMGI</span> database v. 12.0 was used to estimate our survey's coverage of coding regions of the genome [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. This contains 63,676 sequences with an average length of 594 base pairs. A BLAST (blastn) search was performed with each <span class="gene">GMGI</span> sequence as a query and the survey reads as a database, with an expect value cutoff of 1E-6. Figure <xref ref-type="fig" rid="F3"> 3A </xref> shows the number of soybean ESTs with 95% or higher nucleotide level sequence matches to the raw reads, 23,389 of 63,676, or 37%. Since seven percent of the genome was covered with average 109.5 base pair reads, we expect approximately 37% of the ESTs known from soybean to have hits to the genomic reads. This concordance provides further evidence of the unbiased random sampling of genomic sequence by our sequencing method, and further evidence that the genome size estimate of 1,115 megabases of Arumaganthan and Earle [ <xref ref-type="bibr" rid="B15"> 15 </xref> ] is approximately correct.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to sequences that have hits to the <span class="gene">GMGI</span> EST collection, a number of reads contained open reading frames with BLAST hits to known proteins from other organisms, but no hits to soybean ESTs or other soybean sequences. Figure <xref ref-type="fig" rid="F3"> 3B </xref> shows the distribution of coding fragments with an open reading frame giving a 1E-6 or lower BLAST (blastp) e value to the nr database, and the taxonomy of the organism from which the closest sequence in that database was generated. This demonstrates the coverage of the existing EST collection, with over 50% of <span class="trait">protein</span> sequence derived from survey reads matching <italic> Glycine </italic> proteins that are already known. In total, 10,464 of the survey reads were identified as derived from likely conserved <span class="trait">protein</span> coding regions (using e &#x0003c;1E-6 BLAST (blastx) hits to the nr database); 41% of the identified <span class="trait">protein</span> fragments have no detectable similarity to known soybean <span class="trait">protein</span> sequences, giving over 4,000 potential novel soybean <span class="trait">protein</span> fragments with similar, conserved <span class="trait">protein</span> sequences known from other organisms.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Annotation of <span class="trait">protein</span> ORFs with hits to public database </bold> . A) Proportion of EST clones from the Glycine Max Gene Index <span class="gene">(GMGI)</span> matched by 454 reads at 95% and 100% sequence identity (using BLAST with e &#x0003c; 1E-6). The total number of sequences matching at 95% or higher identity is 37% of total EST clones. Note that few sequences match at 100% identity due to the error rate of the 454 pyrosequencing used for this study.B) Coding fragments discovered within the short reads (with e values to the GenBank <span class="trait">protein</span> (nr) database &#x0003c; 1E-6), and their closest protein-level sequence hit by taxonomy of the source organism of the database sequence.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Selective Isolation of Trypsin Inhibitor and Lectin from Soybean Whey by Chitosan/Tripolyphosphate/Genipin Co-Crosslinked Beads</h2>
			<h3>PMID:24901528</h3>
				
					<h3>PMCID:4100134</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Yu-Lung </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Tristan C. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tsai </surname>  <given-names> Min-Lang </given-names>  </name>  <xref rid="c1-ijms-15-09979" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 5 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The isoelectric point (pI) of soybean trypsin inhibitor is pH 4.5. Soybean trypsin inhibitor can be divided into Kunitz trypsin inhibitor (KTI) and Bowman-Birk inhibitor, and their molecular weights are 20 kDa [ <xref rid="B4-ijms-15-09979" ref-type="bibr"> 4 </xref> ] and 8 kDa [ <xref rid="B3-ijms-15-09979" ref-type="bibr"> 3 </xref> ], respectively. The <span class="gene">KTI</span> has two disulfide bridges and is very stable between pH 3 to 10, and has a relatively <span class="trait">high</span> thermal stability in low moisture conditions [ <xref rid="B5-ijms-15-09979" ref-type="bibr"> 5 </xref> ]. The trypsin inhibitor showed effects on carcinogenesis in a variety of <italic> in vivo </italic> and <italic> in vitro </italic> systems. This may be due to the dietary trypsin inhibitor inducing synthesis and distribution of endogenous trypsin inhibitor (acute-phase reactants), which have widespread effects on cell growth and behavior. Topical administration of trypsin inhibitor also demonstrated prominent anti-inflammatory effects [ <xref rid="B6-ijms-15-09979" ref-type="bibr"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Selective isolation of <span class="trait">protein</span> has more challenges than that of smaller molecules such as phytic acid, dyes and metals because proteins have diverse structures, and complex interactions between adsorbent and adsorbate occur. Furthermore, development of operative conditions will be required to scale-up the process for practical application. In this study, adsorption and desorption of <span class="gene">KTI</span> and lectin by different types of chitosan/TPP/genipin co-crosslinked beads (CBs) from soybean whey solutions at different pHs and temperatures were explored. Furthermore, the feasibility of selective isolation for <span class="gene">KTI</span> and lectin is assessed.
					</p>
				
					<p>&nbsp;&nbsp;
						The main driving force of adsorption of proteins to chitosan-based adsorbents is inferred to be electrostatic attraction, <italic> i.e. </italic> , chitosan contains positively charged groups and proteins contain negatively charged groups [ <xref rid="B20-ijms-15-09979" ref-type="bibr"> 20 </xref> , <xref rid="B21-ijms-15-09979" ref-type="bibr"> 21 </xref> , <xref rid="B22-ijms-15-09979" ref-type="bibr"> 22 </xref> ]. The system in our study, however, may be different. The adsorption of CBs proceeded at pH &#x0003c; 6.5. In a solution at a pH lower than its pI, the net charge of a <span class="trait">protein</span> is positive; thus electrostatic repulsion between proteins and the &#x02013;NH <sup> 3+ </sup> of CBs can occur, thus resulting in a low adsorption ratio. When a <span class="trait">protein</span> is in a solution at a pH higher than its pI, the net charge of the <span class="trait">protein</span> is negative; electrostatic repulsion between the <span class="trait">protein</span> and the P <sub> 3 </sub> O <sub> 10 </sub> <sup> 5&#x02212; </sup> of CBs can occur. This also results in a low adsorption ratio. The best adsorption ratios of CB to <span class="gene">KTI</span> and lectin were at pH 4.4 and 5.4, both slightly lower than their pIs (pH 4.5 and 5.8), respectively. This indicates that the main interaction between CBs and proteins could be H-bonding, while electrostatic attraction may be trivial. There are two hydroxyl groups each in the glucopyranose ring of chitosan and in the structure of genipin. These hydroxyl groups can interact with hydroxyl and carboxyl groups of proteins to form H-bonds, improving the adsorption of CBs to proteins. The electrostatic interaction includes attractive and repulsive forces. The electrostatic repulsion may play a more important role than electrostatic attraction in adsorbing CBs to proteins because the adsorption ratios decreased quickly as the pH of solution decreased below or increased above their pIs. Additionally, chitosan (80% degree of deacetylation) includes 20% hydrophobic <italic> N </italic> -acetyl groups. They could interact with hydrophobic groups on <span class="trait">protein</span> probably to form hydrophobic interactions and enhance the CBs&#x02019; ability to adsorb proteins. However, in general the hydrophobic groups are hidden in the protein&#x02019;s interior; consequently, the significance of hydrophobic interactions in CBs adsorbing proteins needs more in-depth study.
					</p>
				
					<p>&nbsp;&nbsp;
						Adsorption ratios of CBs to <span class="gene">KTI</span> or lectin in soybean whey solution (25 &#x000b0;C) were determined ( <xref ref-type="fig" rid="ijms-15-09979-f005"> Figure 5 </xref> ). Over a range of pH 2 to 6 of the soybean whey solution, the trends of adsorption ratios were similar for all three CBs (CB5, CB7 and CB9). The best adsorption ratios for <span class="gene">KTI</span> occurred at pH 4.4, and for lectin at pH 5.4. The adsorption ratios of CB5, CB7 and CB9 to <span class="gene">KTI</span> at pH 4.4 were 76.3%, 93.2% and 94.8%, and to lectin at pH 5.4 were 86.6%, 94.9% and 96.5%, respectively. The result may relate to crosslinking degree and type of CBs. The crosslinking degree of CB9 was 59.2%, leaving 40.8% of the amino groups free. In comparison, CB5 and CB7 were 74.6% and 62.3% crosslinked, with 25.4% and 37.7% of amino groups free, respectively. The ratios of TPP and genipin crosslinked with chitosan were CB5 &#x0003e; CB7 &#x0003e; CB9 and CB5 &#x0003c; CB7 &#x02264; CB9, respectively [ <xref rid="B26-ijms-15-09979" ref-type="bibr"> 26 </xref> ]. Interestingly, CB5 had the least amount of hydroxyl groups, while CB7 and CB9 were equivalent. Overall, the adsorption ratio of CBs to <span class="trait">protein</span> (KTI or lectin) was the highest for CB9, followed by CB7 then CB5 ( <xref ref-type="fig" rid="ijms-15-09979-f005"> Figure 5 </xref> ). Therefore, these results indicate that H-bonding was indeed the main interaction governing adsorption. In addition, in the ranges from pH 4.0 to 4.4 and from 5.0 to 5.4, the slope of the curves was CB5 &#x0003c; CB7 &#x02264; CB9. This may be due to CB5 having fewer free amino groups resulting in a weaker level of positive charge in the beads. Furthermore, as the pH decreases below the pI, the degree of positive charge of a <span class="trait">protein</span> increases Therefore, the electrostatic repulsion between proteins and beads were CB5 &#x0003c; CB7 &#x02264; CB9 at most pH conditions, which may explain the trend observed for the adsorption ratio (CB5 &#x0003c; CB7 &#x02264; CB9). These above results again illustrate that the main adsorption force between CBs and <span class="gene">KTI</span> or lectin was hydrogen bonding, while electrostatic repulsion suppressed adsorption of CBs to the proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						However, <span class="gene">KTI</span> and lectin desorb from CB7 by a mechanism different from that of phytic acid. At pH &#x0003e; 7, the amino group of chotosan does not protonize, and can interact with a <span class="trait">protein</span> to form a H-bond; this scenario is not conducive to desorption. When pH increases, the TPP becomes more negatively charged, and the net charge level of <span class="gene">KTI</span> and lectin is stronger. The low desorption ratios in pH 7 and 8 solutions were perhaps due to the electrostatic repulsions being weaker than the attractive force of the H-bonds forming between proteins and CB7. As the pH increased to 9, the stronger electrostatic repulsion surmounted the attraction of the H-bonding, and caused considerable quantities of proteins to desorb from CB7.
					</p>
				
					<p>&nbsp;&nbsp;
						The maximum adsorption ratios of CBs to <span class="gene">KTI</span> and lectin occurred at pH 4.4 and 5.4 respectively, close to their pIs. Fortunately, the selectivity was <span class="trait">high</span> at these pHs, <italic> i.e. </italic> , adsorption ratios of other proteins was very low. The adsorption ratios of CBs to <span class="gene">KTI</span> and lectin increased with increasing temperature. While CB9 produced the best adsorption ratio compared to CB7 or CB5, the proteins readily desorbed from CB7 (desorption ratio of 80.9% for <span class="gene">KTI</span> and 81.4% for lectin). Thus, selective isolation of <span class="gene">KTI</span> and lectin from soybean whey can be carried out due to the different pIs of the proteins, and by modulating pH and temperature of the solution.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:TILLING to detect induced mutations in soybean</h2>
			<h3>PMID:18218134</h3>
				
					<h3>PMCID:2266751</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" contrib-type="author">  <name>  <surname> Cooper </surname>  <given-names> Jennifer L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jlcooper@fhcrc.org </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Till </surname>  <given-names> Bradley J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> B.TILL@iaea.org </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Laport </surname>  <given-names> Robert G </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> rlaport@mail.rochester.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Darlow </surname>  <given-names> Margaret C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mdarlow@fhcrc.org </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Kleffner </surname>  <given-names> Justin M </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> kleffnerJ@missouri.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Jamai </surname>  <given-names> Aziz </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> aziz.jamai@dartmouth.edu </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> El-Mellouki </surname>  <given-names> Tarik </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> elmelloukit@umkc.edu </email>  </contrib>  <contrib id="A8" contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> smliu@siu.edu </email>  </contrib>  <contrib id="A9" contrib-type="author">  <name>  <surname> Ritchie </surname>  <given-names> Rae </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> rrecord1@purdue.edu </email>  </contrib>  <contrib id="A10" contrib-type="author">  <name>  <surname> Nielsen </surname>  <given-names> Niels </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> nnielsen@purdue.edu </email>  </contrib>  <contrib id="A11" contrib-type="author">  <name>  <surname> Bilyeu </surname>  <given-names> Kristin D </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> bilyeuk@missouri.edu </email>  </contrib>  <contrib id="A12" contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> meksemk@siu.edu </email>  </contrib>  <contrib id="A13" contrib-type="author">  <name>  <surname> Comai </surname>  <given-names> Luca </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> lcomai@ucdavis.edu </email>  </contrib>  <contrib id="A14" corresp="yes" contrib-type="author">  <name>  <surname> Henikoff </surname>  <given-names> Steven </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> steveh@fhcrc.org </email>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 1 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						KM, KB, NN, and RR planned and headed the development of the mutant populations. JK, AJ, and SL coordinated the experimental components of the A population development. RL, MD, TE, and SL isolated DNA. BT and JC oversaw the high-throughput laboratory during DNA preparation, arraying, and mutation detection. RL, JC, and MD designed and tested the primers. JC implemented methods for the elimination of multiple amplicons. JC, BT, SH, and LC designed experiments and interpreted the mutation detection data. SH and LC co-directed the <span class="trait">high</span> throughput <span class="gene">STP</span> laboratory. JC was primarily responsible for drafting and revising the manuscript with contributions from co-authors. All authors read and approved the final manuscript.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Diets <span class="trait">high</span> in selenium and isoflavones decrease androgen-regulated gene expression in healthy rat dorsolateral prostate</h2>
			<h3>PMID:19025659</h3>
				
					<h3>PMCID:2607280</h3>
					<h4>期刊:Reproductive Biology and Endocrinology : RB&#x00026;E</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Legg </surname>  <given-names> Russell L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> rj_legg@netzero.net </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Tolman </surname>  <given-names> Jessica R </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jredd3@gmail.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Lovinger </surname>  <given-names> Cameron T </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> cam0035@mac.com </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Lephart </surname>  <given-names> Edwin D </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> edwin_lephart@byu.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Setchell </surname>  <given-names> Kenneth DR </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> kenneth.setchell@cchmc.org </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Christensen </surname>  <given-names> Merrill J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> merrill_christensen@byu.edu </email>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 11 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Figures <xref ref-type="fig" rid="F1"> 1 </xref> and <xref ref-type="fig" rid="F2"> 2 </xref> show genes for whose expression the main effect of Se (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) or of isoflavones (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ) was statistically significant. Above each <span class="gene">bar</span> graph depicting steady state mRNA levels is shown an inset of a Western blot probed with an antibody to that <span class="trait">protein.</span> Results of densitometric scanning of band intensities are shown above each inset. As noted above, we were unable to find a commercial source for an antibody to rat Akr1c9. Stripping each blot and probing with an antibody to actin showed no significant differences in gel loading.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> isoflavone intake significantly decreased the expression of the Gucy1a3 gene (p &#x0003c; 0.001) (Figure <xref ref-type="fig" rid="F2"> 2A </xref> ). The product of this gene catalyzes conversion of GTP to the second messenger cyclic <span class="gene">GMP,</span> which regulates the activity of a variety of <span class="trait">protein</span> kinases, phosphodiesterases, and ion channels. The main effect of Se on Gucy1a3 gene expression was not significant (p = 0.472).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Trophic Discrimination Factors of Stable Carbon and Nitrogen Isotopes in Hair of Corn Fed Wild Boar</h2>
			<h3>PMID:25915400</h3>
				
					<h3>PMCID:4411150</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hol&#x000e1; </surname>  <given-names> Michaela </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Je&#x0017e;ek </surname>  <given-names> Milo&#x00161; </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ku&#x00161;ta </surname>  <given-names> Tom&#x000e1;&#x00161; </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ko&#x00161;atov&#x000e1; </surname>  <given-names> Michaela </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Our values of trophic discrimination factors for wild boar hair are different than those reported by Nardoto et al. [ <xref rid="pone.0125042.ref037" ref-type="bibr"> 37 </xref> ] for hair of domestic pig ( <italic> Sus scrofa </italic> , breed &#x0201c;Seghers&#x0201d;). Our value for &#x00394; <sup> 13 </sup> C is lower by 2.5 &#x02030; and higher by 0.8 &#x02030; for &#x00394; <sup> 15 </sup> N. There could be several explanations for such differences. Firstly, the differing diet compositions; their study individuals were fed with a mixed diet composed of 25% soybean, 65% corn, and 10% vitamin and mineral mix. The observed difference in &#x00394; <sup> 13 </sup> C and &#x00394; <sup> 15 </sup> N values is likely a result of different <span class="trait">protein</span> quality and amounts of essential amino acids of mixed C3- and C4- plant based diet vs. pure <span class="gene">C4</span> diet as used in the current study [ <xref rid="pone.0125042.ref063" ref-type="bibr"> 63 </xref> ]. Secondly, thousands of years of domestication has resulted in pronounced differences in anatomy and physiology between domestic pigs and their wild ancestors and hence differences in their metabolic pathways may have caused divergent isotopic routing [ <xref rid="pone.0125042.ref064" ref-type="bibr"> 64 </xref> &#x02013; <xref rid="pone.0125042.ref067" ref-type="bibr"> 67 </xref> ]. Thirdly, the age differences between the study animals; in our experiment individuals were 3 year old males whereas in the study of Nardoto et al. [ <xref rid="pone.0125042.ref037" ref-type="bibr"> 37 </xref> ], only 152 day-old females were used. We believe that the lower value of &#x00394; <sup> 15 </sup> N found in their study is likely an age effect, since 152 day-old individuals are still growing and will therefore have reduced &#x00394; <sup> 15 </sup> N [ <xref rid="pone.0125042.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0125042.ref034" ref-type="bibr"> 34 </xref> ]. Growing individuals are believed to retain more <sup> 14 </sup> N in their body pool as they use more dietary <span class="trait">protein</span> for tissue synthesis rather than removing N as waste, which results in lower &#x00394; <sup> 15 </sup> N values [ <xref rid="pone.0125042.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0125042.ref034" ref-type="bibr"> 34 </xref> ]. Lastly, they do not specify the type of hair used in their study and also sampled the hairs used for analysis by clipping (i.e. without root), which may perhaps also contribute to the observed differences in &#x00394; <sup> 13 </sup> C [ <xref rid="pone.0125042.ref054" ref-type="bibr"> 54 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:PCR-Free Detection of Genetically Modified Organisms Using Magnetic Capture Technology and Fluorescence Cross-Correlation Spectroscopy</h2>
			<h3>PMID:19956680</h3>
				
					<h3>PMCID:2778010</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xiaoming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xing </surname>  <given-names> Da </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Yonghong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Wei R. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 10 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Inspired by these FCCS applications, we hypothesize that this technique can be extended to direct identify <span class="gene">GMOs</span> from unamplified samples. However, the extremely <span class="trait">high</span> complexity of the unamplified target and sample media, such as fluorescence interference and enzyme interference, are often limiting factors for direct <span class="gene">GMO</span> identification. One strategy used in conventional southern blot for reduction in complexity is an electrophoresis based method. This procedure is time consuming, typically requiring the genomic DNA sample to be cleaved with a restriction enzyme, size separated by gel electrophoresis, and recovered from the gel to a nitrocellulose filter for subsequent hybridization and detection. Another challenge is that the genomic DNA consists of long chains of double-stranded DNA (dsDNA). Therefore, to achieve effective hybridization, problems such as cularization of the genomic DNA and supercoiling emergence during the heat denaturation step should be avoided <xref ref-type="bibr" rid="pone.0008074-Minunni1"> &#x0005b;6&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0008074-Hill1"> &#x0005b;25&#x0005d; </xref> . To solve these problems, it is highly desirable for detection probes to hybrid with the target in single-strand format and in clean environments. One strategy with increasing popularity in recent years is the use of surface-functionalized magnetic micro- or nano-particles to selectively bind and preconcentrate low-abundance target analytes (DNA, bacteria, <span class="trait">protein)</span> and to discard the sample matrix prior to the detection procedures. Magnetic beads have been proven valuable in immune and nucleic acid assays with improved sensitivity and selectivity <xref ref-type="bibr" rid="pone.0008074-Yan1"> &#x0005b;26&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0008074-Zhou1"> &#x0005b;28&#x0005d; </xref> . The possibility of magnetically confining and concentrating target analytes offers great potential for the detection of minute amounts of target DNA. Combining such a magnetic capture technique with FCCS could theoretically revolutionize the detection of low-level target DNA, particularly for the identification of unamplified <span class="gene">GMOs.</span> To the best of our knowledge, the feasibility of such a combination has not been demonstrated experimentally.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to determine the emission efficiency of the specially designed dye-labeled gene probes and their kinetics parameters, we measured the fluorescence auto-correlation functions, and the count rates, of Rhodamine Green-labeled and Cy5-labeled gene probes in separate solution using FCS ( <xref ref-type="fig" rid="pone-0008074-g003"> Figure 3 </xref> ). The photon count rates of the two gene probes, as shown in <xref ref-type="table" rid="pone-0008074-t002"> Table 2 </xref> , are <span class="trait">high</span> enough to ensure the reliable detection of <span class="gene">GMOs</span> at the single DNA molecule level by FCCS.
					</p>
				
					<p>&nbsp;&nbsp;
						After the initial evaluation of the probes and detection parameters of current FCCS system, experiments were conducted to reach the final aim of PCR-free identification of <span class="gene">GMOs.</span> Two genetically modified samples (GM soybean and GM tomato) and the control samples (Non-GM soybean and Non-GM tomato) are used for the assessment of the proposed method. Genomic DNA (5 &#x000b5;g) from GM soybean and GM tomato was fragmented and subjected to magnetic capture pretreatment and hybridized with the two dye labeled probes. Additional experiments were also executed for directly hybrid with these two dye labeled probes to the target without magnetic capture pretreatment. For negative control experiments, genomic DNA (5 &#x000b5;g) from non-GM soybean and non-GM tomato was handled through a magnetic capture pretreatment procedure which was identical to the positive GM samples assay procedure. An example of <span class="gene">GMO</span> detection is shown in <xref ref-type="fig" rid="pone-0008074-g007"> Figure 7 </xref> . It was found that the magnetic capture pretreatment applied to the negative samples (non-GM soybean and non-GM tomato control) resulted in correlation amplitudes close to 1. Their fluctuations at small correlation times were due to the wide emission spectra of RG and Cy5. However, these fluctuations were not <span class="trait">high</span> enough to affect the outcome of <span class="gene">GMO</span> detection. GM soybean and GM tomato samples with the absence of magnetic capture pretreatment resulted in signal levels comparable to the signals generated by the negative controls. This is probably because of the low target concentration, or the low hybridization efficiency due to cularization of the long double strand genomic DNA and supercoiling emergence during the denaturation. In comparison, high-levels of cross-correlation were observed for both GM soybean and GM tomato samples with the magnetic capture pretreatment ( <xref ref-type="fig" rid="pone-0008074-g007"> Figure 7 </xref> ). These results demonstrated the feasibility of FCCS-based sensing strategy combined with the magnetic capture technique in detecting 35S target directly in non-amplified genomic DNA.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study we provided the evidence as a proof-of-concept for the FCCS-magnetic capture combination. To further clarify the characteristics of the current PCR free method for determination of <span class="gene">GMOs,</span> the analytical performance of the proposed assay have been compared with those of other conventional techniques, such as, ELISA <xref ref-type="bibr" rid="pone.0008074-Hbner1"> &#x0005b;35&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0008074-Stave1"> &#x0005b;36&#x0005d; </xref> , southern blot <xref ref-type="bibr" rid="pone.0008074-Southern1"> &#x0005b;37&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0008074-Hill2"> &#x0005b;38&#x0005d; </xref> , PCR <xref ref-type="bibr" rid="pone.0008074-Vollenhofer1"> &#x0005b;39&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0008074-James1"> &#x0005b;40&#x0005d; </xref> , and real time PCR <xref ref-type="bibr" rid="pone.0008074-Vatilingom1"> &#x0005b;41&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0008074-Hbner2"> &#x0005b;43&#x0005d; </xref> . The major characteristics of different methods are summarized in <xref ref-type="table" rid="pone-0008074-t003"> table 3 </xref> . As shown in the table, the overall performance of the current assay is better than other conventional techniques, especially the specificity. Its <span class="trait">high</span> specificity is the result of the selective magnetic capture of the 35S target and the hybridization of two detection probes to the same 35S region. The sensitivity of current assay (&#x000b5;g genomic levels) is comparable with the other PCR-free technique, southern blot. However, in this study <span class="gene">GMOs</span> detection takes only hours (as opposed to days for conventional southern blot method). Although the <span class="trait">protein</span> based ELISA is a rapid method, one of the major disadvantages is the <span class="trait">protein</span> denaturation as a consequence of processing. Compared to conventional PCR based method (PCR and real time PCR), which may introduce ambiguities resulting from the potential false-positive identification caused by non-specific amplifications, this PCR free method represents a significant improvement in assay reliability. Although the sensitivity of the current assay was lower than the PCR-based method, it could be improved by several orders of magnitude using a simple probe quench method, as described in the literatures <xref ref-type="bibr" rid="pone.0008074-Nolan1"> &#x0005b;21&#x0005d; </xref> , <xref ref-type="bibr" rid="pone.0008074-Neel1"> &#x0005b;23&#x0005d; </xref> &#x02013; <xref ref-type="bibr" rid="pone.0008074-FldesPapp1"> &#x0005b;24&#x0005d; </xref> , or by enhancing the quantum <span class="trait">yield</span> with bi-labeling of the probe <xref ref-type="bibr" rid="pone.0008074-Korn1"> &#x0005b;22&#x0005d; </xref> , or by using the quantum dot label <xref ref-type="bibr" rid="pone.0008074-Fuji1"> &#x0005b;44&#x0005d; </xref> . FCCS can be easily combined with high-throughput screening, thus increasing the speed of detection as well as the number of analyzed samples <xref ref-type="bibr" rid="pone.0008074-Young1"> &#x0005b;45&#x0005d; </xref> . The combination of magnetic capture technology with FCCS presented in this study should not be restricted only to the detection of unamplified <span class="gene">GMOs</span> samples. The strategy and the procedures can be easily adapted for assaying other specific nucleic acids.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Fagopyrum tataricum </italic> (Buckwheat) Improved High-Glucose-Induced Insulin <span class="trait">Resistance</span> in Mouse Hepatocytes and Diabetes in Fructose-Rich Diet-Induced Mice</h2>
			<h3>PMID:22548048</h3>
				
					<h3>PMCID:3324901</h3>
					<h4>期刊:Experimental Diabetes Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Chia-Chen </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 11 </day>  <month> 1 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						FL83B cells were lysed in ice-cold lysis buffer containing 20&#x02009;mM of Tris-HCl (pH 7.4), 1% of Triton X-100, 0.1% of SDS, 2&#x02009;mM of EDTA, 10&#x02009;mM of NaF, 1&#x02009;mM of phenylmethylsulfonyl fluoride (PMSF), 500&#x02009; <italic> &#x003bc; </italic> M of sodium vanadate, and 10&#x02009; <italic> &#x003bc; </italic> g/mL of aprotinin overnight. And then the cell lysates were sonicated with ice cooling (four times each 15&#x02009;s) and then centrifuged (12,000&#x02009;&#x000d7;g, 10&#x02009;min) to recover the supernatant. The supernatant was taken as the cell extract. The <span class="trait">protein</span> concentration in the cell extract was determined using a Bio-Rad <span class="trait">protein</span> assay kit. The samples were subjected to 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). The <span class="trait">protein</span> spots were electrotransferred to a polyvinylidene difluoride (PVDF) membrane. The membrane was incubated with block buffer (PBS containing 0.05% of Tween-20 and 5% w/v nonfat dry milk) for 1&#x02009;h, washed with PBS containing 0.05% Tween-20 (PBST) three times, and then probed with anti-Akt antibody, anti-PTP1B, anti-GS, anti-p-PKC, anti-PKC, anti-AMPK, anti-p-Akt, anti-GLUT2, and anti-PPAR- <italic> &#x003b3; </italic> antibodies (Cell Signaling Technology, Beverly, MA, USA) overnight at 4&#x000b0;C. In addition, the intensity of the blots probed with 1&#x02009;:&#x02009;1000 diluted solution of mouse monoclonal antibody to bind <span class="gene">GAPDH</span> (Cell Signaling Technology) was used as the control to ensure that a constant amount of <span class="trait">protein</span> was loaded into each lane of the gel. The membrane was washed three times each for 5&#x02009;min in PBST, shaken in a solution of HRP-linked anti-rabbit IgG secondary antibody, and washed three more times each for 5&#x02009;min in PBST. The expressions of proteins were detected by enhanced chemiluminescent (ECL) reagent (Millipore, Billerica, MA, USA).
					</p>
				
					<p>&nbsp;&nbsp;
						Effects of EEB on AMPK (a), PTP1B (b), and <span class="gene">GS</span> expressions (c) of FL83B hepatocytes induced by <span class="trait">high</span> glucose. FL83B cells were incubated in serum-free F12K medium with glucose (33&#x02009;mM; final concentration) with or without EEB, quercetin, rutin, and quercetin + rutin for 48&#x02009;h. EEB: 75% ethanol extracts of buckwheat. <sup> # </sup> Significantly different ( <italic> P </italic> &#x0003c; 0.05) from normal; *significantly different ( <italic> P </italic> &#x0003c; 0.05) from high-glucose-treated group.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Methylation Affects Transposition and Splicing of a Large CACTA Transposon from a MYB Transcription Factor Regulating Anthocyanin Synthase Genes in Soybean Seed Coats</h2>
			<h3>PMID:25369033</h3>
				
					<h3>PMCID:4219821</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We determined the molecular basis of three soybean lines that vary in seed coat color at the <italic> R </italic> locus which is thought to encode a MYB transcription factor. RM55- <italic> r <sup> m </sup> </italic> is homozygous for a mutable allele ( <italic> r <sup> m </sup> </italic> ) that specifies black and brown striped seeds; RM30- <italic> R* </italic> is a stable black revertant isoline derived from the mutable line; and RM38- <italic> r </italic> has brown seed coats due to a recessive <italic> r </italic> allele shown to translate a truncated MYB <span class="trait">protein.</span> Using long range PCR, 454 sequencing of amplicons, and whole genome re-sequencing, we determined that the variegated RM55- <italic> r <sup> m </sup> </italic> line had a 13 kb CACTA subfamily transposon insertion (designated <italic> TgmR* </italic> ) at a position 110 bp from the beginning of Intron2 of the <italic> R </italic> locus, <span class="gene">Glyma09g36983.</span> Although the MYB encoded by <italic> R </italic> was expressed at only very low levels in older seed coats of the black revertant RM30- <italic> R* </italic> line, it upregulated expression of anthocyanidin synthase genes ( <italic> ANS2, <span class="gene">ANS3</span> </italic> ) to promote the synthesis of anthocyanins. Surprisingly, the RM30- <italic> R* </italic> revertant also carried the 13 kb <italic> TgmR* </italic> insertion in Intron2. Using RNA-Seq, we showed that intron splicing was accurate, albeit at lower levels, despite the presence of the 13 kb <italic> TgmR* </italic> element. As determined by whole genome methylation sequencing, we demonstrate that the <italic> TgmR* </italic> sequence was relatively more methylated in RM30- <italic> R* </italic> than in the mutable RM55- <italic> r <sup> m </sup> </italic> progenitor line. The stabilized and more methylated RM30- <italic> R* </italic> revertant line apparently lacks effective binding of a transposae to its subterminal repeats, thus allowing intron splicing to proceed resulting in sufficient MYB <span class="trait">protein</span> to stimulate anthocyanin production and thus black seed coats. In this regard, the <italic> TgmR* </italic> element in soybean resembles McClintock's <italic> Spm- </italic> suppressible and change-of-state alleles of maize. This comparison explains the opposite effects of the <italic> TgmR* </italic> element on intron splicing of the MYB gene in which it resides depending on the methylation state of the element.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently it has been proposed that <italic> R </italic> maps to a locus encoding a MYB transcription factor which may positively regulate the expression of a <italic> UDP-glucose: flavonoid 3-O-glucosyltransferase </italic> ( <italic> UF3GT </italic> ) gene that functions in the last step of anthocyanin synthesis <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> . These authors identified the recessive, brown <italic> r </italic> allele to affect the expression of a seed coat-specific R2R3 MYB transcription factor gene, <span class="gene">Glyma09g36983</span> from the sequenced soybean genome as a strong candidate for the <italic> R </italic> locus. In addition, their qRT-PCR analysis supported a correlation between the mRNA levels of the putative <italic> R </italic> gene and a UDP-glucose: flavonoid 3-O-glucosyltransferase ( <italic> UF3GT </italic> ) gene <span class="gene">(Glyma08g07130)</span> that catalyzes the glycosylation of cyanidins to anthocyanins. The recessive <italic> r </italic> (brown) allele at <span class="gene">Glyma09g36983</span> contained a single base deletion that would result in a truncated <span class="trait">protein</span> and did not upregulate this <italic> UF3GT </italic> gene. Based on a combined analysis of transcriptome and metabolome data, another recent study identified 20 anthocyanin, flavonoid and phenylpropanoid isogenes that were differentially expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) seeded soybean isolines <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> . This same study suggested that <italic> R </italic> locus candidate might be a transcription factor at the distal end of chromosome <span class="gene">Gm09</span> at or near <span class="gene">Glyma09g36840,</span> 120 kb from the <span class="gene">Glyma09g36983</span> MYB factor mapped by Gillman et al <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Using primers based on the <span class="gene">Glyma09g36983</span> as the <italic> R </italic> locus <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> , we here report finding a large 13 kb CACTA transposable element insertion in the second intron of the MYB factor in the RM55- <italic> r <sup> m </sup> </italic> mutable soybean line along with a 929 bp PCR product representing excision products that restore the MYB transcription factor sequence. The transposon represents a second CACTA subfamily in soybean based on its transposase and subterminal repeat sequences that most resemble the first CACTA defective element described in soybean, <italic> Tgm1 </italic> <xref rid="pone.0111959-Vodkin1" ref-type="bibr"> [7] </xref> , and is more distant to <italic> Tgmt </italic> <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> and <italic> Tgm9 </italic> <xref rid="pone.0111959-Xu1" ref-type="bibr"> [9] </xref> . Surprisingly the 13 kb CACTA transposon (designated <italic> TgmR* </italic> ) is retained in the <italic> R* </italic> allele of a stable black-seeded revertant isoline. Using quantitative RNA-Seq data from a developmental series of seed coats from the black-seeded RM30- <italic> R </italic> * revertant and the RM38- <italic> r </italic> brown-seeded isoline, we also demonstrate that the MYB factor encoded by the <italic> R </italic> gene appears to induce the expression of anthocyanidin synthase ( <italic> <span class="gene">ANS</span> </italic> ) genes at late stages of seed development in the black RM30- <italic> R* </italic> line. Thus, our data further support <span class="gene">Glyma09g3698</span> as the <italic> R </italic> locus in soybean and show that it regulates <italic> <span class="gene">ANS</span> </italic> genes and potentially other genes in the anthocyanin pathway despite a very low level of <italic> R </italic> gene transcripts. The <italic> R </italic> encoded MYB <span class="trait">protein</span> possesses four of the amino acids present in the subgroup five of MYB proteins that are involved in anthocyanin regulation by transcriptional activation <xref rid="pone.0111959-Stracke1" ref-type="bibr"> [10] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Schematics of the CACTA transposon ( <italic> TgmR* </italic> ) insertion in Intron2 of the <italic> R </italic> locus encoding a R2R3 MYB transcription factor. The <italic> R </italic> locus gene, <span class="gene">Glyma09g36983,</span> depicted in gray (exons) and blue (introns) has a 13 kb CACTA transposon insertion 110 bp from the beginning of Intron2 in the <italic> R </italic> locus of both the variegated RM55- <italic> r <sup> m </sup> </italic> and the black-seeded revertant RM30- <italic> R </italic> * soybean isolines. The ATG duplication is shown at the site of insertion. The locations of the two primers used to amplify the 14 kb fragment containing the transposon insertion are depicted by the two small arrows, FP1 at the 5&#x02032; UTR and RPB in Intron2 near the insertion site. The 13 kb <italic> TgmR* </italic> transposon is a CACTA element based on the terminal ends sequences and the <italic> Tnp2 </italic> transposase it encodes. Both the location of the transposase and the end sequences are marked. Sequences of <span class="trait">high</span> homology to this transposase region were found in chromosomes <span class="gene">Gm05</span> and <span class="gene">Gm15</span> and the Glyma models predicted for the Williams 82 genome are indicated. Likewise, the 3&#x02032; end of <italic> TgmR* </italic> has <span class="trait">high</span> homology to a sequence in chromosome <span class="gene">Gm08</span> and to <italic> Tgm1 </italic> , another smaller CACTA element of the soybean. Green and blue arrows represent the putative <span class="gene">ORF1</span> (Tnp2 transposase) and <span class="gene">ORF2</span> respectively. (B) Alignment of multiple CACTA transposon subterminal repeated motifs. The consensus sequences of the <italic> TgmR* </italic> subterminal direct repeats were aligned to other CACTA consensus sequences from other <italic> Glycine max </italic> transposons ( <italic> Tgm1, Tgmt*, Tgmw4m </italic> and <italic> Tgm-Express1 </italic> ) in addition to other subterminal repeats from CACTA transposons in other plants. Portions of the sequence domains for TNPA-like <span class="trait">protein</span> binding are boxed. One of these motifs are GC rich (left) and the second AT rich (right). Based on the similarities of these terminal repeat motifs, it is clear that <italic> TgmR* </italic> is more closely related to <italic> Tgm1 </italic> than to the other three characterized <italic> G. max </italic> transposons and all of the other CACTA transposon motifs in the lower panel.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic sequences of the alleles from the variegated ( <italic> r <sup> m </sup> </italic> ) and black ( <italic> R* </italic> ) seed lines reported here showed that both contained the <italic> TgmR* </italic> transposon insertion in Intron2. Could the instability of these alleles be dictated by their methylation levels in the two isolines? To answer this question we sequenced the genomes of three soybean lines, RM55- <italic> r <sup> m </sup> </italic> (variegated seed), RM30- <italic> R* </italic> (black seed) and UC44- <italic> R </italic> (black seed with no <italic> TgmR* </italic> insertion), using the bisulfite method (BS-seq) first described by Cokus et al, 2008 <xref rid="pone.0111959-Cokus1" ref-type="bibr"> [29] </xref> to <span class="trait">yield</span> 226, 209, and 235 million total reads respectively ( <xref ref-type="supplementary-material" rid="pone.0111959.s006"> Table S1 </xref> ). The resulting bisulfite sequences were processed through the three-letter aligner program, Bismark <xref rid="pone.0111959-Krueger1" ref-type="bibr"> [30] </xref> to determine the type and degree of &#x0201c;C&#x0201d;-methylation of the transposon and the upstream (35,513 bp) and the downstream (25,417 bp) regions at the <italic> R </italic> <span class="gene">(Glyma09g36983)</span> loci. The results were compiled in <xref ref-type="table" rid="pone-0111959-t004"> Table 4 </xref> and revealed a higher degree of methylation for the transposon ( <italic> TgmR* </italic> ) sequences, most notably, in the RM30- <italic> R* </italic> black-seeded revertant line with an almost double (5.8%) level of methylated C in the CHH context compared to 3% in the RM55- <italic> r <sup> m </sup> </italic> (variegated seed) and 2.9% in the UC44- <italic> R </italic> (black seed with no <italic> TgmR* </italic> insertion). The percentage of CpG methylation in sequences with similarity to <italic> TgmR* </italic> from both RM30- <italic> R* </italic> (86.2%) and UC44- <italic> R </italic> (84.7%) were substantially higher than in RM55- <italic> r <sup> m </sup> </italic> (77.6%). CHG methylation in sequences with similarity to <italic> TgmR* </italic> was also higher for RM30- <italic> R* </italic> (74.3%) compared to RM55- <italic> r <sup> m </sup> </italic> (63.1%). Although UC44- <italic> R </italic> does not contain <italic> TgmR* </italic> in the <italic> R </italic> locus, there are highly repeated regions with sequence similarity to the transposon elsewhere in the genome as previously discussed.
					</p>
				
					<p>&nbsp;&nbsp;
						The results we present here support the location of the <italic> R </italic> locus to be the MYB encoded by <span class="gene">Glyma09g36983</span> as determined recently by Gillman et al., 2011 <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> based on mapping polymorphisms associated with standard <italic> R </italic> and <italic> r </italic> alleles, one of which was a &#x0201c;C&#x0201d; deletion that truncates the MYB <span class="trait">protein</span> in the standard brown <italic> r </italic> allele. Using primers designed to <span class="gene">Glyma09g36983,</span> our analysis of two isolines including a mutable RM55- <italic> r <sup> m </sup> </italic> line with black and brown striped seed phenotype and RM30- <italic> R* </italic> , a stable black-seeded revertant isoline, revealed the existence of a 13 kb CACTA transposon insertion ( <italic> TgmR* </italic> ) at the beginning of the <span class="gene">Glyma09g36983</span> gene Intron2 in both of these lines ( <xref ref-type="fig" rid="pone-0111959-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g003"> 3 </xref> ). The transposon was absent in the brown-seeded RM38- <italic> r </italic> line. Thus, our data directly implicate the structural variations in <span class="gene">Glyma09g36983</span> as producing the phenotypic differences of the <italic> R </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on differences in the subterminal repeats, we previously predicted the existence of two CACTA subfamilies in soybean <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> . The novel <italic> TgmR </italic> * element is 13 kb in length and has all the characteristic features of a CACTA transposon based on the three base duplication (ATG) at the site of insertion, the CACTA ends and the direct and reverse repeats at the subterminal ends that can fold into complex secondary structures ( <xref ref-type="fig" rid="pone-0111959-g003"> Figure 3A </xref> , <xref ref-type="supplementary-material" rid="pone.0111959.s004"> Files S2 </xref> and <xref ref-type="supplementary-material" rid="pone.0111959.s005"> S3 </xref> ). <italic> TgmR* </italic> confirms the proposed existence of two subfamilies of CACTA transposons in soybean which was based on the two distinct types of repeated motif at the subterminal ends ( <xref ref-type="fig" rid="pone-0111959-g003"> Figure 3B </xref> ). The motif in <italic> TgmR* </italic> is more closely related to the one in <italic> Tgm1 </italic> , a defective element interrupting a soybean seed lectin gene <xref rid="pone.0111959-Vodkin1" ref-type="bibr"> [7] </xref> , <xref rid="pone.0111959-Rhodes1" ref-type="bibr"> [14] </xref> , than that of three other previously characterized elements ( <italic> Tgmt*, Tgmw4m, Tgm-Express1 </italic> ) <xref rid="pone.0111959-Zabala3" ref-type="bibr"> [32] </xref> , <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> . In addition, the <italic> TgmR* </italic> 13 kb element encodes two genes, <span class="gene">ORF1</span> (Tnp2-like transposase) and <span class="gene">ORF2</span> of unknown function. The transposase region of <italic> TgmR* </italic> is only 71% similar to the transposase in <italic> Tgmt* </italic> ( <italic> Tgm9 </italic> ). Thus, we propose that <italic> TgmR* </italic> is an active element of the <italic> Tgm1 </italic> type family of elements. Based on RNA-Seq data, both ORFs of <italic> TgmR* </italic> or related element are expressed at low levels in the seed coats of both black RM30- <italic> R* </italic> and RM38- <italic> r </italic> lines ( <xref ref-type="supplementary-material" rid="pone.0111959.s002"> Figure S2 </xref> ). Because DNA sequences with <span class="trait">high</span> similarity to the <span class="gene">ORF1</span> gene are highly repeated throughout the entire genome, the <span class="gene">ORF1</span> RNAs detected in both lines could be derived from sequences other than the one inserted in the <italic> R* </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Although many of the genes in the anthocyanin pathway examined showed differences in expression between the RM30- <italic> R* </italic> black revertant and the RM38- <italic> r </italic> brown isoline, the most significant differences detected were for the <italic> ANS2 </italic> and <italic> <span class="gene">ANS3</span> </italic> genes at the 400&#x02013;500 mg seed developmental stage ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6A </xref> ). In another study in which the transcript profiles of seed coats from a standard black and brown-seeded near-isogenic pair were compared at different stages of seed development, significant differences in <italic> ANS2/ANS3 </italic> and <italic> <span class="gene">UGT78K1</span> </italic> expression were also seen between the two isolines with higher levels in the black seed coats <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> . For <italic> ANS2/ANS3, </italic> the significant differential expression (P&#x0003c;0.01) manifested early in seed development, (75&#x02013;100 mg), while the <italic> <span class="gene">UGT78K1</span> </italic> differences were first detected at 200&#x02013;300 mg weight range and reaching maximum at 300&#x02013;400 mg seed size. Gillman et al, (2011) <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> measured <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression via qRT-PCR in seed coats of a brown- (PI 567115 B) and black- (PI 84970) seeded lines with similar results. As in those instances, our results show that the <italic> UFGT </italic> genes are expressed at very low level at early stages of seed development, while <italic> <span class="gene">ANS</span> </italic> expression is very <span class="trait">high</span> at those early stages and tapers down as seed development advances. This decrease is slowed down or reversed in the black-seeded phenotypes. The higher level of <italic> <span class="gene">ANS</span> </italic> gene expression at early stages of seed development can be explained by an additional role of the <span class="gene">ANS</span> enzyme in directing the synthesis towards the flavanol and proanthocyanidin branches of the flavonoid pathway ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ). In contrast, the UFGT enzyme function is limited to the last step in the synthesis of anthocyanins ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Involvement of methylation causing the variegated phenotype is consistent with the observation that there appear to be no structural changes in the MYB gene when the element excises from Intron2 since the 929-bp amplicon from the RM55- <italic> r <sup> m </sup> </italic> DNA aligned with the <span class="gene">Glyma09g36983</span> gene sequence in Phytozome (Williams 82 which has a standard <italic> R </italic> allele) with no rearrangement of Exon2 or the Exon2-Intron2 splice site junction. This suggests that when the element excises by transposition, a fully functional MYB <span class="trait">protein</span> is produced leading to the black sectors on the brown background in the RM55- <italic> r <sup> m </sup> </italic> line.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, our results confirm the soybean <italic> R </italic> locus to be <span class="gene">Glyma09g36983</span> and demonstrate that the transcription of the <italic> R </italic> gene in the black seed coats of the RM30- <italic> R </italic> * revertant line, may be hindered by the processing of the very large Intron2 containing the <italic> TgmR </italic> * (13kb) insertion, but it does not render the processed transcript non-functional. Instead, the translated MYB <span class="trait">protein</span> drives an increase in transcript accumulation of the <italic> ANS2/3 </italic> , <italic> UFGT2 </italic> and <italic> AOMT </italic> genes whose products function in the late steps of anthocyanin synthesis, particularly in production of the cyanidin-3-glycoside that accounts for the black pigmentation of the seed coats. In contrast, the observed higher level of transcripts found for the defective <italic> r </italic> allele in RM38- <italic> r </italic> seed coats may translate into truncated peptides that are non-functional and do not activate the transcription of the late anthocyanin pathway genes resulting in brown seeds which reflects accumulation of only proanthocyanidins. The <italic> TgmR* </italic> found in the striped RM55- <italic> r <sup> m </sup> </italic> and black-seeded RM30- <italic> R* </italic> isolines is a transposon of a second CACTA subfamily that encodes a full length <italic> Tnp2 </italic> transposase gene that may drive the observed instability of the isolines with the <italic> r <sup> m </sup> </italic> and <italic> R* </italic> alleles. The level of instability of the RM55- <italic> r <sup> m </sup> </italic> and RM30- <italic> R* </italic> isolines is influenced by the total level of C methylation and also methylated C in the CHH context, judging by the bisulfite sequencing results of three genomes. The <italic> TgmR* </italic> element in the variegated RM55- <italic> r <sup> m </sup> </italic> line was hypomethylated especially at the subterminal repeat ends compared to the RM30- <italic> R* </italic> element. Stress and other environmental factors may in turn affect the methylation level of the germinal and somatic tissues and consequently lead to the variegated phenotype and affect the stability of the mutable ( <italic> r <sup> m </sup> </italic> ) and revertant ( <italic> R* </italic> ) alleles by a mechanism similar to that elucidated for CACTA elements in maize including the <italic> Spm </italic> -suppressible alleles first described by Barbara McClintock (reviewed in <xref rid="pone.0111959-Wessler1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0111959-Fedoroff1" ref-type="bibr"> [13] </xref> ). This explains the opposite phenotypes resulting from the influence of the <italic> TgmR* </italic> transposase on intron processing of the MYB <span class="trait">protein</span> when the element is less methylated in the RM55- <italic> r <sup> m </sup> </italic> line (brown coat sectors) versus the more methylated element in the RM30- <italic> R* </italic> revertant line with black seed coats.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comprehensive Analysis of NAC Domain Transcription Factor Gene Family in <italic> Populus trichocarpa </italic></h2>
			<h3>PMID:20630103</h3>
				
					<h3>PMCID:3017804</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Hu </surname>  <given-names> Ruibo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hurb@qibebt.ac.cn </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Qi </surname>  <given-names> Guang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qiguang@qibebt.ac.cn </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A3">  <name>  <surname> Kong </surname>  <given-names> Yingzhen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yingzhenkong@hotmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kong </surname>  <given-names> Dejing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ky104@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Gao </surname>  <given-names> Qian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> gaoqian@qibebt.ac.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Zhou </surname>  <given-names> Gongke </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhougk@qibebt.ac.cn </email>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 7 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Multiple sequence alignments of the full-length <span class="trait">protein</span> sequences, including the highly conserved N-terminal NAM domain and the more divergent C-terminal activation domain, were performed by Clustal X (version 1.83) program [ <xref ref-type="bibr" rid="B90"> 90 </xref> ]. The unrooted phylogenetic trees were constructed with MEGA 4.0 using the Neighbor-Joining (NJ), Minimal Evolution (ME) and Maximum Parsimony (MP) methods and the bootstrap test carried out with 1000 iterations [ <xref ref-type="bibr" rid="B91"> 91 </xref> ]. Pairwise <span class="gene">gap</span> deletion mode was used to ensure that the more divergent C-terminal domains could contribute to the topology of the NJ tree.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Food protein-stabilized nanoemulsions as potential delivery systems for poorly water-soluble drugs: preparation, in vitro characterization, and pharmacokinetics in rats</h2>
			<h3>PMID:21468355</h3>
				
					<h3>PMCID:3065798</h3>
					<h4>期刊:International Journal of Nanomedicine</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="af1-ijn-6-521"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tan </surname>  <given-names> Yanan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijn-6-521"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Zhiqiang </given-names>  </name>  <xref ref-type="aff" rid="af1-ijn-6-521"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Lingyun </given-names>  </name>  <xref ref-type="aff" rid="af2-ijn-6-521"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Fuqiang </given-names>  </name>  <xref ref-type="aff" rid="af3-ijn-6-521"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="af1-ijn-6-521"> 1 </xref>  <xref ref-type="corresp" rid="c1-ijn-6-521"/>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> A </bold> ) Scheme of the process for preparing protein-stabilized nanoemulsions and photographs of nanoemulsions. <bold> B </bold> ) Transmission electron microphotography of food protein-stabilized nanoemulsions: soy <span class="trait">protein</span> isolate (SPI), whey <span class="trait">protein</span> isolate (WPI), and &#x003b2;-lactoglobulin (&#x003b2;-lg). The scale <span class="gene">bar</span> for all images represents 100 nm.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Antitumor and HIV-1 Reverse Transcriptase Inhibitory Activities of a Hemagglutinin and a Protease Inhibitor from Mini-Black Soybean</h2>
			<h3>PMID:21527979</h3>
				
					<h3>PMCID:3057713</h3>
					<h4>期刊:Evidence-based Complementary and Alternative Medicine : eCAM</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Xiu Juan </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 10 </day>  <month> 1 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Black soybean has long been used in Chinese traditional medicine, as a detoxifier, an anti-inflammatory drug and a blood nutrient and for promoting urination. It suppresses the growth of transplantable human bladder carcinoma and tumor angiogenesis in mice [ <xref rid="B25" ref-type="bibr"> 25 </xref> ], exhibits antioxidant activity, inhibits low density lipoprotein oxidation [ <xref rid="B26" ref-type="bibr"> 26 </xref> ], and can be used for treating an ophthalmic disorder associated with retinal pigment epithelium disturbance [ <xref rid="B27" ref-type="bibr"> 27 </xref> ]. The polysaccharides prepared from black soybeans [ <xref rid="B28" ref-type="bibr"> 28 </xref> ] can also be used to stimulate cellular immunity by augmenting the production of cytokines and in preventing or treating leukemia. It is beneficial to the health of patients who have been infected by pathogens such as HIV, by elevating the blood cell number and enhancing the immune system [ <xref rid="B29" ref-type="bibr"> 29 </xref> ]. Black soybean is more effective than yellow soybean in preventing menopausal symptoms. Ovariectomized menopausal rats fed on black soybeans demonstrated a significantly greater reduction in blood-cholesterol concentration, compared to rats fed on yellow soybeans. Consumption of black soybeans protects against bone loss in ovariectomized rats by inhibiting bone turnover and bone resorption [ <xref rid="B30" ref-type="bibr"> 30 </xref> ]. Black soybean peptides when applied in vivo and in vitro reduce endoplasmic reticulum stress and improve insulin <span class="trait">resistance</span> [ <xref rid="B31" ref-type="bibr"> 31 </xref> ], whilst anthocyanins from black soybean seed coats stimulate wound healing in fibroblasts and keratinocytes and prevent inflammation in endothelial cells [ <xref rid="B32" ref-type="bibr"> 32 </xref> ]. They preferentially inhibit tumor necrosis factor-alpha-mediated induction of vascular cell adhesion molecule-1 over intercellular adhesion molecule-1 through the regulation of transcription factor genes binding to DNA sequence <span class="gene">GATA</span> and interferon regulatory transcription factor-1 [ <xref rid="B33" ref-type="bibr"> 33 </xref> ]. The anthocyanins inhibit UVB-induced inflammatory cylooxygenase-2 gene expression and PGE <sub> 2 </sub> production through regulation of the nuclear factor-kappaB and phosphatidylinositol 3-kinase/Akt pathway [ <xref rid="B34" ref-type="bibr"> 34 </xref> ]. They also exert hypolipidemic and antiobesity actions [ <xref rid="B35" ref-type="bibr"> 35 </xref> ]. Red soybean anthocyanins demonstrate an antitumor action [ <xref rid="B36" ref-type="bibr"> 36 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Transcriptome Analysis between the Cytoplasmic Male Sterile Line NJCMS1A and Its Maintainer NJCMS1B in Soybean ( <italic> Glycine max </italic> (L.) Merr.)</h2>
			<h3>PMID:25985300</h3>
				
					<h3>PMCID:4436259</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Jiajia </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Shaohuai </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ding </surname>  <given-names> Xianlong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Tingting </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dai </surname>  <given-names> Jinying </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Shouping </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						It was suggested that aspartic protease acted as an anti-cell-death factor participating in programmed cell death (PCD) and the over-expression of this gene resulted in male sterility in <italic> Arabidopsis </italic> [ <xref rid="pone.0126771.ref051" ref-type="bibr"> 51 </xref> ]. In this study, we found 1 gene encoding aspartic proteinase A1 up-regulated in NJCMS1A, the higher expression level of this gene in the male sterile line NJCMS1A could lead to PCD of the pollen cell, ultimately causing male sterility. In addition, other DEGs were also found in our study, for example, there were 2 genes encoding cytochrome <span class="gene">P450</span> family <span class="trait">protein,</span> 1 gene encoding glutathione S-transferase tau 9, 1 gene encoding leucine-rich repeat transmembrane <span class="trait">protein</span> kinase and 1 gene encoding glyceraldehyde-3-phosphate dehydrogenase C subunit 1, etc., which were up-regulated in the male sterile line NJCMS1A compared to in the near-isogenic maintainer NJCMS1B; seven genes encoding major facilitator superfamily proteins, 6 genes encoding NAD(P)-binding Rossmann-fold superfamily proteins and 3 genes encoding tetratricopeptide repeat (TPR)-containing proteins, etc., which were down-regulated in the male sterile line NJCMS1A compared to in the near-isogenic maintainer NJCMS1B; and 20 DEGs with unknown functions. The above DEGs might be associated with the male sterility of NJCMS1A, but their specific functions needed to be further studied.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Screening of soy protein-derived hypotriglyceridemic di-peptides <italic> in vitro </italic> and <italic> in vivo </italic></h2>
			<h3>PMID:21600040</h3>
				
					<h3>PMCID:3116501</h3>
					<h4>期刊:Lipids in Health and Disease</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Inoue </surname>  <given-names> Nao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ninoue@biochem.tohoku.ac.jp </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Nagao </surname>  <given-names> Koji </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> knagao@cc.saga-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Sakata </surname>  <given-names> Kotaro </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sakata_k@wakunaga.co.jp </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Yamano </surname>  <given-names> Naomi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> 07552405@edu.cc.saga-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Gunawardena </surname>  <given-names> Pathma Elgoda Ranawakage </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> 03AJ52@edu.cc.saga-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Han </surname>  <given-names> Seo-Young </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> 9219@edu.cc.saga-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Matsui </surname>  <given-names> Toshiro </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> tmatsui@agr.kyushu-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Nakamori </surname>  <given-names> Toshihiro </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> nakamori.toshihiro@so.fujioil.co.jp </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Furuta </surname>  <given-names> Hitoshi </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> furuta.hitoshi@so.fujioil.co.jp </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Takamatsu </surname>  <given-names> Kiyoharu </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> takamatsu.kiyoharu@so.fujioil.co.jp </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Yanagita </surname>  <given-names> Teruyoshi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yanagitt@cc.saga-u.ac.jp </email>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 5 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several reports have suggested that the amino acid composition of dietary proteins and peptides influences their bioactivities [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. It has been reported that <span class="trait">high</span> amounts of histidine and hydrophobic amino acids contribute to antioxidant potency and that hydrophobic peptides can bind bile acids, thereby enhancing fecal steroid excretion [ <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> - <xref ref-type="bibr" rid="B50"> 50 </xref> ]. In the second part of the current study, we evaluated the effects of hydrophobic and hydrophilic fractions, separated from SCP-LD3 with hydrophobic synthetic absorbent, on TG synthesis in HepG2 cells. As shown in Table <xref ref-type="table" rid="T1"> 1 </xref> , the hydrophobic fraction was found to contain a higher amount of phenylalanine, whereas the hydrophilic fraction contained a higher amount of glutamine compared to SCP-LD3. In HepG2 cells, incorporation of [1- <sup> 14 </sup> C] acetate into the cellular TG fraction was significantly and dose-dependently lowered by hydrophilic fraction treatment (Figure <xref ref-type="fig" rid="F4"> 4A </xref> ). In the next <italic> in vivo </italic> experiment, we evaluated the effects of the hydrophilic fraction on lipid metabolism in OLETF rats. Though there was no significant alteration in growth parameters (Table <xref ref-type="table" rid="T2"> 2 </xref> ), feeding of the hydrophilic fraction alleviated hepatomegaly and hepatic TG accumulation in OLETF rats (Table <xref ref-type="table" rid="T2"> 2 </xref> , Figure <xref ref-type="fig" rid="F4"> 4B </xref> ). Moreover, despite the fact that the degree of supplementation with the hydrophilic fraction in the diet (substituted for 10% casein) was half that of SCP-LD3 (substituted for 20% casein), the TG-lowering effects of these 2 diets were almost the same. In agreement with the <italic> in vitro </italic> study, the TG-lowering effects were attributable to the suppression of fatty acid synthesis (represented by lowered activities of FAS, G6PDH, and malic enzyme) and TG synthesis (represented by lowered <span class="gene">PAP</span> activity) in the liver of OLETF rats.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Using Microsatellites to Understand the Physical Distribution of Recombination on Soybean Chromosomes</h2>
			<h3>PMID:21799819</h3>
				
					<h3>PMCID:3140510</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ott </surname>  <given-names> Alina </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Trautschold </surname>  <given-names> Brian </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sandhu </surname>  <given-names> Devinder </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 6 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To understand if the distribution of SSR markers on chromosomes represents a true picture of distribution of genes, we used the physical locations of 46,430 predicted soybean genes from soybean sequence database ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> www.phytozome.net </ext-link> ). A comparison of the percent markers in each section of the chromosomes to the percent genes in each section showed a <span class="trait">high</span> average correlation (R <sup> 2 </sup> &#x0200a;=&#x0200a;0.83) between marker and gene distribution ( <xref ref-type="fig" rid="pone-0022306-g002"> Figure 2 </xref> , <xref ref-type="table" rid="pone-0022306-t001"> Table 1 </xref> ). Chromosomes <span class="gene">Gm10,</span> <span class="gene">Gm14</span> and <span class="gene">Gm20</span> all had correlations above 0.9 with the highest R <sup> 2 </sup> value being 0.94 for <span class="gene">Gm20.</span> The lowest R <sup> 2 </sup> value of 0.57 was observed for <span class="gene">Gm13</span> ( <xref ref-type="table" rid="pone-0022306-t001"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						As expected based on <span class="trait">high</span> correlations between the distribution of genes and SSR markers, the genes showed similar patterns of distribution as the markers in the distal and proximal regions ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> ). The distal 25&#x00025; of each chromosome contained an average of 50.2&#x00025; of the genes while the proximal 25&#x00025; region of each chromosome contained an average of 6.7&#x00025; of the genes ( <xref ref-type="table" rid="pone-0022306-t001"> Table 1 </xref> ). The distal ends ranged from 31.4&#x00025; of the genes on chromosome <span class="gene">Gm13</span> to 63.6&#x00025; of the genes on <span class="gene">Gm01</span> ( <xref ref-type="table" rid="pone-0022306-t001"> Table 1 </xref> ). The proximal regions ranged from 3.5&#x00025; of the genes on chromosome <span class="gene">Gm06</span> to 13.4&#x00025; of the genes on <span class="gene">Gm13.</span> In addition, the correlation of gene density with distance from the centromere (R <sup> 2 </sup> &#x0200a;=&#x0200a;0.83) was higher than the correlation between marker density with distance from the centromere (R <sup> 2 </sup> &#x0200a;=&#x0200a;0.64) ( <xref ref-type="table" rid="pone-0022306-t001"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Comparison of sequence based physical maps and integrated genetic linkage maps using 2188 common markers provided a detailed and precise estimate of recombination at a sub-regional level ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> ). In general, recombination was higher in distal regions of the chromosomes ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> , <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). Due to lack of recombination in the proximal regions, a large number of markers appeared to be clustered in the centromeric regions of the linkage maps. Although most centromeric and pericentromeric regions were mostly devoid of recombination, a few chromosomes had spikes of very <span class="trait">high</span> recombination in the proximal regions ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> , <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). Besides the centromeric regions, there were a number of other locations throughout the chromosomes where no or very little recombination occurred ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> ). For example, <span class="gene">Gm01,</span> <span class="gene">Gm14,</span> and <span class="gene">Gm18</span> all showed low recombination toward the middle of the long chromosome arm. Crossing over frequency (CF) was used as a tool to compare distribution of recombination on soybean chromosomes. Among the 20 soybean chromosomes, <span class="gene">Gm16</span> had the lowest CF of 3.3 cM/MB and chromosome <span class="gene">Gm07</span> had the highest CF of 14.8 cM/MB, a greater than 4-fold difference ( <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). The genome average for CF was 7.2 cM/MB. Chromosome <span class="gene">Gm10</span> had the highest CF of 10.0 cM/MB in the distal 25&#x00025; and <span class="gene">Gm08</span> had the highest CF of 3.08 cM/MB in the proximal 25&#x00025; ( <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). Comparison of the average CF in the distal and proximal regions clearly showed higher recombination in the distal 25&#x00025; with the average CF value of 8.1 cM/MB compared to 0.4 cM/MB in the proximal 25&#x00025;. However, the correlation of CF with distance from the centromere was relatively weak with an R <sup> 2 </sup> value of 0.21. The highest correlation value of 0.40 was observed for chromosome <span class="gene">Gm04.</span> Chromosomes <span class="gene">Gm01,</span> <span class="gene">Gm07,</span> <span class="gene">Gm08,</span> <span class="gene">Gm13</span> and <span class="gene">Gm20</span> showed correlation values below 0.15 ( <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). The lowest correlation was observed for chromosome <span class="gene">Gm20.</span> The number of recombination hotspots, defined as having a CF in excess of 2-fold as compared to the genome average, ranged from 12 on <span class="gene">Gm11</span> to 1 on <span class="gene">Gm16,</span> with an average of 6 per chromosome ( <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Recombination is a vital component for crop breeding and genetics, as it results in new combinations of genes available for crop improvement. Understanding the distribution of recombination on chromosomes can be a powerful tool to characterize and clone genes. We integrated SSR-based genetic linkage maps to make a comprehensive linkage map. By comparing the linkage map with the sequence based physical map we identified the physical locations of the SSR markers on soybean chromosomes and studied physical distribution of recombination on chromosomes. Generally, <span class="trait">high</span> recombination was observed in the telomeric regions as compared to the centromeric regions ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> ). The average recombination of the distal 25&#x00025; of the chromosomes was 200-fold higher than the average recombination of the proximal 25&#x00025; of the chromosomes ( <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). These observations are in line with several previous studies, which showed suppressed recombination in centromeric regions <xref ref-type="bibr" rid="pone.0022306-Chen1"> [23] </xref> , <xref ref-type="bibr" rid="pone.0022306-Schnable1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0022306-Saintenac1"> [25] </xref> , <xref ref-type="bibr" rid="pone.0022306-Yu1"> [33] </xref> . In maize, the recombination rate was highest close to the telomeric ends of chromosomes and was highly suppressed around the centromeres <xref ref-type="bibr" rid="pone.0022306-Schnable1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0022306-Gore1"> [34] </xref> . Recombination varied greatly over the length of the soybean chromosomes, with a weak correlation with the distance from the centromere (R <sup> 2 </sup> &#x0200a;=&#x0200a;0.21). Chromosomes <span class="gene">Gm01,</span> <span class="gene">Gm07,</span> <span class="gene">Gm08,</span> <span class="gene">Gm13</span> and <span class="gene">Gm20</span> showed no significant correlation between recombination and distance from the centromere ( <xref ref-type="fig" rid="pone-0022306-g001"> Figure 1 </xref> , <xref ref-type="table" rid="pone-0022306-t002"> Table 2 </xref> ). Larger genomes like wheat and barley are known to show a higher correlation between rate of recombination and the distance from the centromere <xref ref-type="bibr" rid="pone.0022306-Kunzel1"> [35] </xref> , <xref ref-type="bibr" rid="pone.0022306-Akhunov2"> [36] </xref> . In wheat, recombination was seen to increase at an exponential rate relative to distance from the centromere <xref ref-type="bibr" rid="pone.0022306-Akhunov1"> [26] </xref> . In the model plant Arabidopsis, however, very little association was reported between recombination rates and distance from the centromere <xref ref-type="bibr" rid="pone.0022306-Drouaud1"> [37] </xref> . The soybean genome is about 8 times bigger than Arabidopsis but about 14 times smaller than wheat <xref ref-type="bibr" rid="pone.0022306-Arumuganathan1"> [38] </xref> . Differences in the distribution of recombination and gene distribution between soybean and large genome species like maize, barley, and wheat may be attributed to differences in genome sizes <xref ref-type="bibr" rid="pone.0022306-Saintenac1"> [25] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Soybean Agglutinin on Intestinal Barrier Permeability and Tight Junction <span class="trait">Protein</span> Expression in Weaned Piglets</h2>
			<h3>PMID:22272087</h3>
				
					<h3>PMCID:3257084</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yuan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-12-08502"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-12-08502"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qin </surname>  <given-names> Guixin </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-12-08502"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-12-08502"> 2 </xref>  <xref ref-type="corresp" rid="c1-ijms-12-08502"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Zewei </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-12-08502"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Che </surname>  <given-names> Dongsheng </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-12-08502"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-12-08502"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bao </surname>  <given-names> Nan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-12-08502"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-12-08502"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaodong </given-names>  </name>  <xref ref-type="aff" rid="af3-ijms-12-08502"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Injured intestinal barrier increased the epithelial permeability. Diamine oxidase (DAO), as a relatively stable marker of maturation and integrity of intestinal mucosa cells, is an intracellular enzyme with <span class="trait">high</span> activity existing in intestinal villous cells in mammalians, especially <span class="trait">high</span> in the jejunum and ileum [ <xref ref-type="bibr" rid="b15-ijms-12-08502"> 15 </xref> ]. The activity of <span class="gene">DAO</span> in intestinal mucosa decreases when the epithelium are injured, thus <span class="gene">DAO</span> activity of intestinal mucosa can indicate the changes in its cellular integrity. <sc> d </sc> -lactic acid is a metabolic product of bacteria [ <xref ref-type="bibr" rid="b16-ijms-12-08502"> 16 </xref> ], present in the intestinal lumen. The intact intestinal mucosa provides a barrier function to prevent <sc> d </sc> -lactic acid infiltrating the portal blood, as an index of increase in permeability of the intestinal wall [ <xref ref-type="bibr" rid="b17-ijms-12-08502"> 17 </xref> ]. In this study, ileal mucosa <span class="gene">DAO</span> in piglets fed 0.1, 0.15 or 0.2% SBA decreased, and plasma <sc> d </sc> -lactic acid content was enhanced, which indicated that the intestine injured by 0.1&#x02013;0.2% SBA increased the epithelial permeability. A previous study showed that SBA (90 &#x003bc;g/mL) slightly increased the values with quercetin glycosides or calcium ions [ <xref ref-type="bibr" rid="b18-ijms-12-08502"> 18 </xref> ], which is consistent with this study.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">high</span> dose SBA (0.1&#x02013;0.2%) could increase the intestinal permeability and reduce piglet intestinal epithelial tight junction <span class="trait">protein</span> occludin or ZO-1 expression, while the low dose of SBA (0.05% of total diet) had no effect. The contents of <span class="gene">DAO</span> or <sc> d </sc> -lactic acid and the expression of occludin or ZO-1 showed a linear relationship with SBA levels (0&#x02013;0.2%) in diets.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Universal Primer-Multiplex-PCR Method with Sequencing Gel Electrophoresis Analysis</h2>
			<h3>PMID:22272223</h3>
				
					<h3>PMCID:3260127</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xu </surname>  <given-names> Wentao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhai </surname>  <given-names> Zhifang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Kunlun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Nan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Yanfang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shang </surname>  <given-names> Ying </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Luo </surname>  <given-names> Yunbo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 8 </day>  <month> 7 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						With the rapid development of GM crops, more and more studies have recently described the use of multiplex PCR as a rapid and convenient screening assay for the detection of <span class="gene">GMOs.</span> In GM crops such as soybean, maize, and canola, a multiplex PCR system has been developed to detect multiple target sequences using simultaneous amplification profiling <xref ref-type="bibr" rid="pone.0022900-James1"> [11] </xref> . A sensitive and specific triplex nested PCR assay was developed for the detection of housekeeping gene (lectin) and inserted elements of Roundup Ready soybean, i.e., constitutively expressed <italic> CaMV 35S </italic> promoter, <italic> Cp4 epsps </italic> gene encoding for 5-enol-pyruvyl-shikimate-3-phosphate for herbicide tolerance, <italic> nos </italic> terminator, and a chloroplast transit peptide (ctp) facilitating transport of epsps <span class="trait">protein,</span> in highly processed products <xref ref-type="bibr" rid="pone.0022900-Zhang2"> [12] </xref> . Multiplex PCR simultaneously detecting eight lines of GM maize by employing sequence-specific primers and the maize endogenous <italic> Zein </italic> gene was developed <xref ref-type="bibr" rid="pone.0022900-Shrestha1"> [2] </xref> , which is also a most targets multiplex PCR system nowadays. Recently, multiplex PCR assays simultaneously amplifying the commonly used selectable marker genes, i.e., <italic> <span class="gene">aadA</span> </italic> , <italic> <span class="gene">bar</span> </italic> , <italic> <span class="gene">hpt</span> </italic> , <italic> nptII </italic> , <italic> <span class="gene">pat</span> </italic> , and a reporter gene <italic> uidA </italic> were developed as a reliable tool for qualitative screening of GM crops <xref ref-type="bibr" rid="pone.0022900-Randhawa1"> [13] </xref> . What's more, multiplex PCR-based assays have also been developed to simultaneously detect functional transgenes, control elements and housekeeping genes, such as <italic> cry1Ac </italic> gene for insect <span class="trait">resistance,</span> <italic> CaMV 35S </italic> promoter and endogenous <italic> SRK </italic> (S-locus Receptor Kinase) gene in Bt cauliflower <xref ref-type="bibr" rid="pone.0022900-Randhawa2"> [14] </xref> ; <italic> osmotin </italic> gene for salinity and <span class="trait">drought</span> tolerance, <italic> CaMV35S </italic> promoter and endogenous <italic> LAT52 </italic> (late anther tomato) gene in GM tomato <xref ref-type="bibr" rid="pone.0022900-Randhawa3"> [15] </xref> ; <italic> cry1Ab </italic> gene for insect <span class="trait">resistance,</span> <italic> CaMV 35S </italic> promoter; <italic> npt II </italic> marker gene and endogenous <italic> UGPase </italic> (uridine diphosphate glucose pyrophosphorylase) gene in Bt potato <xref ref-type="bibr" rid="pone.0022900-Randhawa4"> [16] </xref> . These studies demonstrate that the multiplex PCR system is also a convenient, cost-effective, and efficient assay for GM detection.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Common bean reaction to angular leaf spot comprises transcriptional modulation of genes in the ALS10.1 QTL</h2>
			<h3>PMID:25815001</h3>
				
					<h3>PMCID:4357252</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Oblessuc </surname>  <given-names> Paula R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/217627"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matiolli </surname>  <given-names> Cleverson C. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/217640"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chiorato </surname>  <given-names> Alisson F. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/215353"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Camargo </surname>  <given-names> Luis E. A. </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Benchimol-Reis </surname>  <given-names> Luciana L. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/213181"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Melotto </surname>  <given-names> Maeli </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/82741"/>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In addition to qualitative <span class="trait">resistance</span> genes, <span class="trait">resistance</span> to ALS was also assigned to quantitative trait loci (QTL) of major and minor effects on the phenotypic variation observed in segregating populations. Among the QTL controlling <span class="trait">resistance</span> to ALS, ALS10.1 is a major QTL that was mapped to linkage group Pv10 using the IAC-UNA &#x000d7; <span class="gene">CAL</span> 143 (UC) recombinant inbreed line (RIL) population (Oblessuc et al., <xref rid="B45" ref-type="bibr"> 2012 </xref> ). The ALS10.1 genome core was defined by integrating information on common bean genomic and genetic data using the six molecular markers that span the ALS10.1 region of 5.3 Mb (Oblessuc et al., <xref rid="B46" ref-type="bibr"> 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic background where the QTL are present influences the expression of <span class="trait">resistance</span> (L&#x000f3;pez et al., <xref rid="B32" ref-type="bibr"> 2003 </xref> ; Mahuku et al., <xref rid="B34" ref-type="bibr"> 2009 </xref> , <xref rid="B35" ref-type="bibr"> 2011 </xref> ; Oblessuc et al., <xref rid="B45" ref-type="bibr"> 2012 </xref> ). The bean line <span class="gene">CAL</span> 143, derived from AND 277, shows <span class="trait">resistance</span> to a great number of <italic> P. griseola </italic> isolates in the field (Aggarwal et al., <xref rid="B2" ref-type="bibr"> 2004 </xref> ; Oblessuc et al., <xref rid="B45" ref-type="bibr"> 2012 </xref> ), in addition to <span class="trait">resistance</span> against <span class="trait">rust</span> ( <italic> Uromyces appendiculatus </italic> ), leaf spot ( <italic> Alternaria alternata </italic> ), and anthracnose ( <italic> Colletotrichum lindemuthianum </italic> ) (Vieira et al., <xref rid="B66" ref-type="bibr"> 2002 </xref> ), highlighting the importance of this line as a source of broad disease <span class="trait">resistance.</span> Thus, the <span class="trait">resistance</span> locus ALS10.1 from <span class="gene">CAL</span> 143 was selected for the discovery of disease responsive genes. Here, we provide evidences that the ALS10.1 locus is enriched for genes involved in signal perception and transduction that includes two putative R gene clusters containing evolutionarily related genes, and putative negative regulators of immunity. We also showed that these defense-related genes on ALS10.1 are transcriptional regulated according to the plant's reaction to fungal infection. This study provides a significant step for functional characterization of this major common bean QTL and enhances our current understanding of the biological mechanisms that lead to bean <span class="trait">resistance</span> against angular leaf spot.
					</p>
				
					<p>&nbsp;&nbsp;
						The TIR-NB-ARC common bean <span class="trait">protein</span> Phvul.010G025700.1 was used as query for BLASTP (threshold <italic> E </italic> -value &#x02264; 1 &#x000d7; 10 <sup> &#x02212;20 </sup> ) against the common bean proteome (Phytozome v1.0; <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> http://www.phytozome.net/ </ext-link> ). The identified proteins and the Phvul.010G025700.1 sequences were aligned with CLUSTALW as part of the MEGA 6.06 software (Tamura et al., <xref rid="B59" ref-type="bibr"> 2013 </xref> ). The domains <span class="gene">TIR</span> and NB-ARC of the predicted Phvul.010G025700.1 were identified by searching this <span class="trait">protein</span> sequence against the NCBI <span class="trait">protein</span> conserved domain database (CDD) (Marchler-Bauer et al., <xref rid="B36" ref-type="bibr"> 2015 </xref> ). The proteins containing both conserved domains (TIR and NB-ARC) were used to obtain the phylogenetic tree with MEGA 6.06 using the neighbor-joining method with partial deletion (Tamura et al., <xref rid="B59" ref-type="bibr"> 2013 </xref> ). Bootstrap support values were obtained over 1000 replications.
					</p>
				
					<p>&nbsp;&nbsp;
						Seeds of the common bean genotypes <span class="gene">CAL</span> 143 (ALS <span class="trait">resistance</span> source) and the IAC-UNA (ALS susceptibility source) were placed on germination paper (Germilab, PR, Brazil) in a growth chamber at 25&#x000b0;C with 12 h photoperiod for 3 days. Seedlings were transplanted to pots filled with Dystrophic Red Latosol soil, fertilized with NPK 04-14-08 (400 kg/ha). Fungus culturing, inoculum preparation, and inoculation were performed as previously described (Oblessuc et al., <xref rid="B45" ref-type="bibr"> 2012 </xref> ). Inoculated plants of each genotype were maintained in the greenhouse for 17 days to verify disease symptoms. Mock-inoculated plants were used as control for gene expression analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the great number of TIR-NB-ARC coding genes in the ALS10.1 and the fact that 37 R genes in cluster one (C1, Figure <xref ref-type="fig" rid="F3"> 3B </xref> ) are similar to AT5G36930 and 8 R genes in cluster two (C2, Figure <xref ref-type="fig" rid="F3"> 3B </xref> ) are similar to AT5G17680, we performed a phylogenetic analysis to infer the evolutionary relationship among these genes, as well as with other genes coding for TIR-NB-ARC domains in the common bean genome. The Phvul.010G025700.1 predicted <span class="trait">protein</span> contains the complete TIR-NB-ARC domains as the great majority (75%) of the R proteins of the ALS10.1 locus (Table <xref ref-type="table" rid="T1"> 1 </xref> and Table S2) and it is regulated by fungal infection as explained below, we used this <span class="trait">protein</span> as query for BLASTP analysis (threshold <italic> E </italic> -value &#x0003c; 1 &#x000d7; 10 <sup> &#x02212;20 </sup> ) against the common bean proteome. We found 159 highly similar proteins in the bean proteome; however, only 108 of them contain both <span class="gene">TIR</span> (pfam01582) and NB-ARC (pfam00931) domains and were used for further <span class="trait">protein</span> analysis. Twenty-one of these proteins are translated from transcript splice variants, thus the phylogenetic tree was constructed with the query and the remainder 87 proteins predicted from the first splice variant of each gene (Figure <xref ref-type="supplementary-material" rid="SM4"> S1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						All TIR-NB-ARC proteins from the first R gene cluster at ALS10.1 (C1), with exception of Pvhul.010G025400.1 that contains an incomplete <span class="gene">TIR</span> domain, formed a unique clade along with two proteins that are just outside of the ALS0.1 region (Phvul.010G018400 and Phvul.010G023500) and the Phvul.L003500 located in the scaffold_40 of the bean genome reference (Figure <xref ref-type="fig" rid="F4"> 4A </xref> ). This major clade can be divided into two sub-clusters, in which the sub-cluster I was formed with 13 proteins including the query <span class="trait">protein</span> Phvul.010G025700. The query <span class="trait">protein</span> is closely related to the proteins Phvul.010G025000.1 and Phvul.010G028200.1, indicating these three genes may be close paralogs (Figure <xref ref-type="fig" rid="F4"> 4A </xref> ). Sub-cluster II contains 21 proteins including the ones outside the ALS10.1 locus on chromosome 10. These results indicate a complex evolution of these R genes on the ALS10.1 locus; and also suggest that the scaffold_40 could be part of the Pv10.
					</p>
				
					<p>&nbsp;&nbsp;
						The R genes and the <span class="gene">RLK</span> are examples of known gene families that perceive and transduce signals through the cell membrane. Many R genes have been cloned in different species and the majority is predicted to encode intracellular multi-domain proteins with a C-terminal leucine-rich repeat (LRR) domain fused to a central nucleotide-binding (NB) domain (NB-LRR) (Martin et al., <xref rid="B37" ref-type="bibr"> 2003 </xref> ; van Ooijen et al., <xref rid="B64" ref-type="bibr"> 2007 </xref> ). The NB domain is part of a larger <span class="trait">protein</span> domain family named NB-ARC, because of its presence in APAF-1 (apoptotic protease-activating factor-1), R proteins, and CED-4 ( <italic> Caenorhabditis elegans </italic> death-4 <span class="trait">protein)</span> (van der Biezen and Jones, <xref rid="B63" ref-type="bibr"> 1998 </xref> ). NB-ARC proteins form a subclass of the STAND (Signal Transduction ATPases with Numerous Domains) super family that function simultaneously as sensors, switch and response factors, and are involved in a variety of processes, including immunity and apoptosis (van Ooijen et al., <xref rid="B65" ref-type="bibr"> 2008 </xref> ). Among the probable defense related genes in the ALS10.1 locus, two clusters of putative R genes were identified. The Arabidopsis R gene AT5G36930 (TIR-NB-ARC) showed the highest similarity to the majority of the putative R genes in the ALS10.1 locus, and among them, the Phvul.010G025700 was the most similar ( <italic> E </italic> -value = 2 &#x000d7; 10 <sup> &#x02212;145 </sup> and 33.3% identity). Indeed, both genes code for TIR-NB-ARC proteins, i.e., a N-terminal <span class="gene">TIR</span> domain followed by the NB-ARC domain.
					</p>
				
					<p>&nbsp;&nbsp;
						Common bean R genes can be divided into <span class="gene">TIR</span> and non-TIR classes (Liu et al., <xref rid="B30" ref-type="bibr"> 2012 </xref> ; Garz&#x000f3;n et al., <xref rid="B20" ref-type="bibr"> 2013 </xref> ), where members of the non-TIR class are found mostly in the chromosomes Pv04 and Pv11, and members of the <span class="gene">TIR</span> class are found mainly in Pv10 (Schmutz et al., <xref rid="B54" ref-type="bibr"> 2014 </xref> ). Although TIR-NB-ARC proteins are found in all bean chromosomes, they are overrepresented on Pv10 and most of them (51.1%) are coded from genes in the ALS10.1 locus, highlighting the importance of this subfamily of R genes in the common bean response to <italic> P. griseola </italic> . Moreover, the <span class="trait">high</span> phylogenetic relationship among the R proteins from ALS10.1 indicates that the encoding genes underwent extensive duplication events, mainly in the first R gene cluster where all derived proteins formed a unique clade. Phylogeny analysis also indicated that cross-over may have occurred during the bean reference genotype (G19833) evolution between Pv10, Pv11, and Pv04 as putative TIR-NB-ARC paralogs from these chromosomes forms concise clades. Further molecular and biochemical analyses are needed to verify the function of these proteins in plant defense.
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis, the presence of either a coiled coil (CC) or <span class="gene">TIR</span> domain in the R <span class="trait">protein</span> typically determines whether an NB-LRR-mediated <span class="trait">resistance</span> response requires NDR1 (NON-RACE-SPECIFIC DISEASE <span class="trait">RESISTANCE)</span> or the EDS1 (ENHANCED DISEASE SUSCEPTIBILITY 1) signaling pathways, respectively (Aarts et al., <xref rid="B1" ref-type="bibr"> 1998 </xref> ; Feys et al., <xref rid="B18" ref-type="bibr"> 2005 </xref> ). Both pathway leads to SA-mediated PCD; however TIR-protein-induced PCD through EDS1 involves the autophagy machinery, while the CC-protein-mediated PCD is independent of autophagy (Hofius et al., <xref rid="B26" ref-type="bibr"> 2009 </xref> ). This difference in response could be important for plant immunity against hemibiotrophic phytopathogens, such as the <italic> P. griseola </italic> . Plant defense responses needs to be coordinated according to the life phase of the pathogen. The biotrophic phase of the pathogen is more sensitive to the host PCD as it needs a live host cell to infect, while the host PCD would benefit the fungus at the necrotrophic phase.
					</p>
				
					<p>&nbsp;&nbsp;
						Four genes had specific regulation direction (up- or down-regulation) depending on the time of fungus infection. During the probable fungus biotrophic phase, the two RLKs (Phvul.010G033800 and Phvul.010G031900) and the putative <italic> PvEDR3-like </italic> (Phvul.010G040900) were repressed in the <span class="trait">resistance</span> response of <span class="gene">CAL</span> 143 to <italic> P. griseola </italic> , whereas these genes were induced during the susceptible response of IAC-UNA. However, in the later timers of the resistant reaction, these genes were up-regulated reinforcing the hypothesis that after <span class="trait">resistance</span> have been achieved, the normal physiological balance of the cell needs to be re-established by discontinuing the, now unnecessary, immune response. Therefore, it is possible that Phvul.010G033800, Phvul.010G031900, and Phvul.010G040900 may act as negative regulator of immunity in common bean.
					</p>
				
					<p>&nbsp;&nbsp;
						The Phvul.010G040900 gene encodes for a putative PvEDR3-like <span class="trait">protein.</span> This annotation was based on the function of its closest ortholog in Arabidopsis, the <italic> AtEDR3 </italic> gene that encodes for a dynamin-related <span class="trait">protein</span> 1E (Tang et al., <xref rid="B60" ref-type="bibr"> 2006 </xref> ). Interestingly, the Arabidopsis loss-of-function mutant for this gene ( <italic> edr3 </italic> ) has increased disease <span class="trait">resistance</span> to powdery mildew ( <italic> Golovinomyces cichoracearum </italic> ), in a SA-dependent manner, whereas <italic> edr3 </italic> plants displays enhanced susceptibility to the necrotrophic fungal pathogen <italic> Botrytis cinerea </italic> (Tang et al., <xref rid="B60" ref-type="bibr"> 2006 </xref> ). The repression of the <italic> PvEDR3-like </italic> during the initial phase of the <italic> P. griseola </italic> in the resistant bean genotype <span class="gene">CAL</span> 143 suggests that this gene may have similar function as <italic> AtEDR3 </italic> in immunity. Therefore, it is possible that common bean <span class="trait">resistance</span> against <italic> P. griseola </italic> is achieved at the biotrophic developmental phase of the fungus, at least in part by down-regulating the expression of immunity repressors at the ALS10.1 locus during the biotrophich phase of the fungus life cycle.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Leaf morphology in Cowpea [ <italic> Vigna unguiculata </italic> (L.) Walp]: QTL analysis, physical mapping and identifying a candidate gene using synteny with model legume species</h2>
			<h3>PMID:22691139</h3>
				
					<h3>PMCID:3431217</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Pottorff </surname>  <given-names> Marti </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> martipottorff@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Ehlers </surname>  <given-names> Jeffrey D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jeff.ehlers@ucr.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Fatokun </surname>  <given-names> Christian </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> C.FATOKUN@cgiar.org </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Roberts </surname>  <given-names> Philip A </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> philip.roberts@ucr.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Close </surname>  <given-names> Timothy J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> timothy.close@ucr.edu </email>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> Hls </italic> region was examined for synteny with the soybean genome and was found to be highly co-linear with soybean chromosomes 3 and 19 (Figure <xref ref-type="fig" rid="F3"> 3 </xref> , Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). Eight Medicago genes orthologous to cowpea SNP markers identified the region from locus <span class="gene">Glyma03g34240</span> to <span class="gene">Glyma03g38550</span> as the <italic> Hls </italic> syntenic locus in soybean chromosome 3 (Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). The soybean syntenic locus corresponded to 27.60&#x02009;cM to 35.96&#x02009;cM region in the <italic> Hls </italic> locus and was also in the same general marker order as the cowpea consensus genetic map (Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). The region spanning between orthologous soybean genes to cowpea SNP markers 1_1013 and 1_0349 was examined for leaf morphology candidate genes on the soybean genome browser on the Phytozome webpage ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> http://www.phytozome.net </ext-link> ). Soybean locus <span class="gene">Glyma03g38320</span> was observed flanked by orthologous genes for cowpea SNP markers 1_1013 and 1_0417 and was annotated as an ortholog of EZA1/SWINGER (SWN) gene. <span class="gene">Glyma03g38320</span> has a SET domain <span class="trait">(protein</span> lysine methyltransferase enzyme) and two copies of a cysteine rich motif and is annotated as KOG: 1079; transcriptional repressor EZA1 ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> http://www.phytozome.net </ext-link> ) (accessed April 2012).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Hls </italic> syntenic region in soybean chromosome 19 was identified by thirteen out of fourteen SNP markers, spanning from <span class="gene">Glyma19g36180</span> to <span class="gene">Glyma19g41150</span> which corresponded to 24.10&#x02009;cM to 39.80&#x02009;cM on the cowpea consensus genetic map (Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). The syntenic region in soybean between orthologous cowpea SNP markers 1_0910 and 1_0349 was examined for known leaf development genes using the soybean genome browser on the Phytozome webpage ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> http://www.phytozome.net </ext-link> ). <span class="gene">Glyma19g40430</span> locus was observed flanked by soybean genes orthologous to SNP markers 1_0992 and 1_0417 and was annotated as an ortholog of the Arabidopsis EZA1/SWINGER (SWN) gene (Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). <span class="gene">Glyma19g40430</span> has a SET domain <span class="trait">(protein</span> lysine methyltransferase enzyme) and two copies of a cysteine rich motif and is annotated as KOG: 1079; transcriptional repressor EZA1 ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> http://www.phytozome.net </ext-link> ) (accessed April 2012).
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic sequences for Medtr7g133020, <span class="gene">Glyma03g38320,</span> <span class="gene">Glyma19g40430</span> and the Arabidopsis EZA1 gene (AT4G02020.1) were BLASTed to the cowpea genome vs. 02 ( <ext-link ext-link-type="uri" xlink:href="http://www.harvest-blast.org"> http://www.harvest-blast.org </ext-link> ) and HarvEST:Cowpea database ( <ext-link ext-link-type="uri" xlink:href="http://harvest.ucr.edu"> http://harvest.ucr.edu </ext-link> ) to identify orthologous cowpea sequences. The Medtr7g133020 and AT4G02020.1 genomic sequences returned a <span class="trait">high</span> BLAST alignment with contig C27495629 (Table&#x02009; <xref ref-type="table" rid="T6"> 6 </xref> ). The genomic sequences for <span class="gene">Glyma03g38320</span> and <span class="gene">Glyma19g40430</span> returned a <span class="trait">high</span> alignment with contig C27664167 and scaffold28398 (Table&#x02009; <xref ref-type="table" rid="T6"> 6 </xref> ). All genomic sequences when BLASTed to Harvest:Cowpea database returned the best alignment with cowpea unigene 21752 which was annotated as an EZA1 ortholog (Table&#x02009; <xref ref-type="table" rid="T6"> 6 </xref> ). Interestingly, unigene 21752 was obtained from leaf and shoot meristems used for a mature pre-flowering developmental stage cDNA library from cowpea varieties DanIla, Tvu11986, Tvu7778 and 12008D ( <ext-link ext-link-type="uri" xlink:href="http://harvest.ucr.edu"> http://harvest.ucr.edu </ext-link> ). The genomic and unigene sequences identified for the cowpea ortholog for EZA1 will enable future studies to clone and confirm the candidate gene.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary soyasaponin supplementation to pea <span class="trait">protein</span> concentrate reveals nutrigenomic interactions underlying enteropathy in Atlantic salmon ( <italic> Salmo salar </italic> )</h2>
			<h3>PMID:22748053</h3>
				
					<h3>PMCID:3424111</h3>
					<h4>期刊:BMC Veterinary Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" equal-contrib="yes" id="A1">  <name>  <surname> Kortner </surname>  <given-names> Trond M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Trond.Kortner@nvh.no </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Skugor </surname>  <given-names> Stanko </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> stanko.skugor@nvh.no </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Penn </surname>  <given-names> Michael H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Michael.Penn@nvh.no </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Mydland </surname>  <given-names> Liv Torunn </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> liv.mydland@umb.no </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Djordjevic </surname>  <given-names> Brankica </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> brankica.djordjevic1@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Hillestad </surname>  <given-names> Marie </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> marie.hillestad@biomar.no </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Krasnov </surname>  <given-names> Aleksei </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> aleksei.krasnov@nofima.no </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Krogdahl </surname>  <given-names> &#x000c5;shild </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Ashild.Krogdahl@nvh.no </email>  </contrib>
				<br/> 
				年份：<day> 2 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Salmon fed PPC&#x02009;+&#x02009;S showed increased distal intestinal transcription of genes involved in inflammation at different levels (Table&#x02009; <xref ref-type="table" rid="T3"> 3 </xref> ). Up-regulation was observed in several chemokines, cytokines, especially IL-22 (up-regulated 10-fold), as well as chemokine and cytokine receptors. Two genes for proteins of eicosanoid metabolism were induced 2- to 3-fold (arachidonate 5-lipoxygenase-activating <span class="trait">protein,</span> leukotriene b4 12-hydroxydehydrogenase). Annexins contribute to the intestinal <span class="trait">resistance</span> to injury, as they possess anti-inflammatory properties as well as gastroprotective properties. Microarray analyses did not show increase of IL-1 and TNFalpha. However, up-regulation of NFkB, several functionally related pro-inflammatory transcription factors and TNF-induced proteins was revealed and this suggested an acute character of inflammation. Interestingly, this was in parallel with the down-regulation of MHCI (down 4.3-fold) and several virus responsive genes that are dependent on IFN. Diverse immune effector mechanisms were activated. Up-regulation was shown for lectins and genes with reported induction in pathogen infected fish. Stimulation of different complement pathways was likely since 2- to 4-fold up-regulated levels were shown for genes associated with classical (C1Q-like proteins and IG receptors) and alternative (factors P and D) pathways. This was in parallel with decreased expression of negative complement regulators: <span class="gene">C1</span> inhibitor (5-fold down-regulated) and C4b-binding <span class="trait">protein</span> (1.8-fold down-regulated). Activation was shown by several extracellular matrix (ECM) degrading proteases including matrix metalloproteinases (MMPs) and their inhibitors that are commonly co-regulated. Inflammatory tissue damages were also suggested by up-regulation of several components of the respiratory burst complex that generates ROS, i.e. cytochromes b-245 and b558 (up-regulated 3- and 7-fold, respectively), myeloperoxidase (up 32-fold) and neutrophil cytosolic factor 1 (up 7-fold). Arginase 2 (ARG2, up 15-fold) and ornithine decarboxylase (ODC, up 1.8-fold) can direct the arginine flux away from nitric oxide synthase and nitrogen nitric oxide (NO) production, a free radical toxic to bacteria but also an important signalling molecule. The oxidative stress was probably reinforced by down-regulation of diverse ROS scavengers and proteins that bind iron and heme; both are potent catalysts of free radicals production.
					</p>
				
					<p>&nbsp;&nbsp;
						Up-regulation of several genes involved in exocytosis including annexins (Tables&#x02009; <xref ref-type="table" rid="T3"> 3 </xref> and <xref ref-type="table" rid="T5"> 5 </xref> ) was in accordance with increased production of mucus as indicated by increased numbers of goblet cells. Down-regulation was observed in several genes involved in epithelial cell-to-cell contacts including genes that encode tight junctions (TJs). The 2.5-fold down-regulation of the <span class="gene">gap</span> junction Cx32.2 could affect passage of inorganic ions and small water-soluble molecules in between cells. Suppression of the two cadherins (epithelial cadherin precursor and protocadherin 20, 3.1- and 2.2-fold down-regulated, respectively) is also worth mentioning as cadherin-mediated cell-cell and cell-matrix junctions have crucial roles both within normal and injured intestinal epithelium [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Surface glycans are essential for cell recognition and interactions. Three glycan modifying esterases were up-regulated (glucosamine 6-phosphate N-acetyltransferase, beta-1,3-N-acetylglucosaminyltransferase 7 (B3GN7) and alpha-1,3-fucosyltransferase), while several genes required for degradation and turnover of glycans were down-regulated, e.g. aspartylglucosaminidase (AGA, down 2.2-fold). Decreased expression was observed in growth factors and transcription regulators that control differentiation of various cell lineages including mesenchymal (class B basic helix-loop-helix proteins 2 and 3, down-regulated 1.7- and 5.9-fold, respectively) and endothelial (angiogenin and angiopoietin-related <span class="trait">protein</span> 4), fibroblasts and other elements of connective tissue. Furthermore, lactase-phlorizin hydrolase (LPH, 8.5-fold decrease) is exclusively expressed in the small intestine in mammals, and is often used as a marker for the differentiation of enterocytes, as LPH mRNA can first be detected in the transition zone between the crypt and the villus [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]. An interesting finding was the 2.4-fold up-regulation of putative guanylin, an intestinal peptide with an important role in regulation of water balance.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the MYB transcription factor superfamily in soybean</h2>
			<h3>PMID:22776508</h3>
				
					<h3>PMCID:3462118</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Du </surname>  <given-names> Hai </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> superduhai@yahoo.cn </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Yang </surname>  <given-names> Si-Si </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yang-ss@sohu.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Liang </surname>  <given-names> Zhe </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> zhe.liang@umb.no </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Feng </surname>  <given-names> Bo-Run </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> feng-brun@sohu.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Liu </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> liuleidh@126.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Huang </surname>  <given-names> Yu-Bi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yubihuang@sohu.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A7">  <name>  <surname> Tang </surname>  <given-names> Yi-Xiong </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> tangyx@mail.caas.net.cn </email>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Given the potential roles of MYB proteins in regulation of gene expression, secondary metabolism, and responses to environmental stresses, and that soybean is the first legume for which the complete genome has been sequenced, it is of interest to determine how many MYB genes exist in this species, as well as the characteristics of gene structures, phylogenetic relationships, chromosome locations, conserved motifs and expression patterns. Moreover, it is important to unravel the functionality of new and as yet uncharacterized soybean MYBs, especially those that might be involved in seed development and legume-specific nodulation. Within this framework, in the present study we identified the entire complement of MYBs in soybean and conducted a genome-wide comparative analysis of MYBs between soybean and <italic> Arabidopsis </italic> . We present a comprehensive classification, together with structural, expressional, and functional analyses, of the soybean MYB gene family. We identified a total of 252 MYBs, including 244 R2R3-MYB (2R-MYB) genes, six R1R2R3-MYB (3R-MYB) genes, and two R0R1R2R3-MYB (4R-MYB) genes, most of which have not been functionally characterized previously. On the basis of a phylogenetic analysis, we divided the MYBs in <italic> Arabidopsis </italic> and soybean into 48 subfamilies, which allowed identification of shared and unique subfamilies, and examination of functional relationships among subgroups of MYBs. We also estimated the number of MYBs in the most recent common ancestor of soybean and <italic> Arabidopsis </italic> . It is worth noting that the reliability of the subfamilies defined on the basis of the phylogenetic analysis was supported by additional criteria, such as the presence and position of introns, gene structure, and presence of common <span class="trait">protein</span> motifs outside the MYB domain. Chromosomal distribution analysis revealed that some of the duplication events are likely to have contributed to the expansion of this gene family. The particular clusters of paralogous and orthologous genes identified within each subfamily showed ancestral duplication and/or gene loss events. Furthermore, the expression profiles of different organs, tissues, and/or developmental stages were previously revealed by analyses of soybean whole-genome RNA-seq [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. Comparative analysis of the expression profiles of MYBs in soybean and <italic> Arabidopsis </italic> showed that AtMYBs and <span class="gene">GmMYBs</span> exhibit various expression patterns, and that MYBs in the same phylogenetic clades and subclades have conserved expression patterns. This finding suggests the existence of functional diversification and conservation between <span class="gene">GmMYBs</span> and their close homologs. Moreover, there are species-specific subfamilies, some of which are specifically expressed during nodulation. Our findings constitute the first step towards further investigations into the biological and molecular functions of MYB transcription factors in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to verify the reliability of our results, we performed PROSITE profile (PS50090) and simple modular architecture research tool (SMART) analysis to identify all of the putative MYB <span class="trait">protein</span> sequences in the soybean genome. We excluded from further analysis 26 putative MYB <span class="trait">protein</span> sequences that contained typical R2R3-MYB domains but lacked partial sequences in the N-terminal region ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). We also excluded <span class="gene">Glyma09g37340,</span> which contained a predicted MYB motif and intact open reading frame (OFR) with low confidence value. The putative novel MYB sequences, combined with the previous estimates of MYB families, finally yielded from the soybean genome a primary data set of 244 typical R2R3-MYB proteins (2R-MYB), six R1R2R3-MYB proteins (3R-MYB), and two 4R-like MYB proteins (4R-MYB) ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). It was defined that the MYB superfamily is the most abundant transcription factor family in plants, with 126 2R-MYB genes in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B4"> 4 </xref> ], 109 in rice[ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> ], 118 in grape[ <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> ], and 192 in Populus[ <xref ref-type="bibr" rid="B29"> 29 </xref> ]. In this study, the remaining 252 sequences (more than 4% of the 5671 predicted soybean transcription factor genes) represented an updated classification of the MYB family in soybean, and constituted one of the largest known plant MYB transcription factor gene families. The resulting sequences were named according to the generic system, and the correspondence of the sequence names with gene and <span class="trait">protein</span> identifiers from the corresponding genome browsers ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). The nomenclature system for <span class="gene">GmMYBs</span> used in the present study was provisionally applied to distinguish each of the MYB genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence similarity among the 244 <span class="gene">GmMYBs</span> at the cDNA level showed a <span class="trait">high</span> degree of sequence divergence ranging from 5.50% to 99% in the open reading frame (ORF). The highest similarity in the ORF was observed between <span class="gene">GmMYB100,</span> <span class="gene">GmMYB243,</span> and <span class="gene">GmMYB244,</span> whereas the lowest similarity was observed between <span class="gene">GmMYB029</span> and <span class="gene">GmMYB156.</span> Most of the genes had very short 5&#x02032;-untranslated regions (UTR) (Figure <xref ref-type="fig" rid="F2"> 2 </xref> b and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ), and showed the lowest similarity in that region, which indicated that the coding region was more conserved than the 5&#x02032;-UTR sequence. The 3&#x02032;-UTR sequences were similarly less conserved among the <span class="gene">GmMYBs.</span> In addition, we observed that some 5&#x02032;- and 3&#x02032;-UTR sequences of <span class="gene">GmMYBs</span> had introns. Interestingly, these genes were mainly clustered in the same subfamily (C40; <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). The predicted proteins encoded by the 244 R2R3-MYBs shared approximately 25&#x02013;100% identity. As with MYB proteins in other plants, the MYB proteins of soybean were highly conserved in the DNA-binding domain. The overall similarity in the MYB domain of the 244 <span class="gene">GmMYB</span> proteins appeared to be relatively <span class="trait">high.</span> About 11 pairs of <span class="gene">GmMYBs</span> proteins were almost identical in the MYB domains, whereas the similarity of more than 72 pairs of <span class="gene">GmMYBs</span> exceeded 97% (a difference was observed in fewer than three of the 108 residues in the R2R3 region). By contrast, there was little sequence similarity downstream of the MYB domain.
					</p>
				
					<p>&nbsp;&nbsp;
						In general, these AS events resulted in a variety of sequence insertions and/or deletions in the corresponding ORFs. For instance, a 33&#x02009;bp alternative 3&#x02032; splice site in <span class="gene">GmMYB028</span> allowed the lengthening or shortening of 11 amino acids in the R3 repeat. Similarly, a 21&#x02009;bp alternative 3&#x02032; splice site in <span class="gene">GmMYB140</span> resulted in a deletion of seven amino acids in the R3 repeat. Interestingly, we observed that some of the AS events changed the type of MYB <span class="trait">protein.</span> For example, a 189&#x02009;bp alternative promoter site of <span class="gene">GmMYB082</span> resulted in a frame shift, which changed the R2R3-MYB into a signal-repeat MYB type. Similarly, a 237&#x02009;bp alternative promoter site in <span class="gene">GmMYB152</span> that transposed the 5&#x02032; exons changed the typical R2R3-MYB gene into a signal-repeat MYB gene. In contrast, <span class="gene">GmMYB237</span> exhibited three alternative types of splicing by alternative promoters and alternative poly(A) sites. The first transcript resulted in a deletion of more than 2000&#x02009;bp at the 5&#x02032; terminus and a long insertion at the 3&#x02032; terminus; the second resulted in a long deletion at the 3&#x02032; terminus; and the third (3R-MYB <span class="trait">protein)</span> resulted in a deletion of the R1 and R3 repeats in the 3R-MYB <span class="trait">protein.</span> In <italic> Arabidopsis </italic> , 16 of the 18 AS events undergo similar AS events (data not shown), which further highlighted the relevance of AS to the MYB gene family.
					</p>
				
					<p>&nbsp;&nbsp;
						However, it is difficult to infer orthologs merely from tree topologies. In general, the gene functions of a clade appear highly but not absolutely conserved across plant species. Therefore, it is useful to accurate identify the true orthologs between plant species, based on syntenic relationship. To support the putative orthology of <span class="gene">GmMYBs</span> and AtMYBs, we examined the syntenic relationship based on gene content, order, and orientation between the genomes of soybean and <italic> Arabidopsis </italic> . Despite chromosomal rearrangements (e.g., inversion of gene orientation), gene content was similar between the different chromosome regions analyzed, which supported the deduction that these genes are more likely to be orthologous. The results demonstrated that extensive soybean regions exhibit synteny with <italic> Arabidopsis </italic> genome, and as much as 1610 syntenic blocks between soybean and <italic> Arabidopsis </italic> genome were identified in this study. Moreover, up to 100 <span class="trait">high</span> confidence syntenic blocks include at least one pair of MYB orthologous genes in these two species, accounting for about 36% of the <span class="gene">GmMYBs,</span> such as <span class="gene">GmMYB232/AtMYB103,</span> <span class="gene">GmMYB138/AtMYB124,</span> and <span class="gene">GmMYB214/AtMYB61</span> (Table&#x02009; <xref ref-type="table" rid="T1"> 1 </xref> , and <xref ref-type="supplementary-material" rid="S5"> Additional file 5 </xref> ). It is common to find that several <span class="gene">GmMYBs</span> located on different chromosomes are orthologs of a same AtMYB gene, which strongly supported the genome duplication event in soybean genome. By contrast, the observation that duplicated AtMYBs are orthologs of a single <span class="gene">GmMYB</span> gene suggested that the AtMYBs also experienced duplications after the divergence from the common ancestor.
					</p>
				
					<p>&nbsp;&nbsp;
						Remarkably, genes in the same subfamily generally showed the same intron pattern, with the position of the intron almost completely conserved within most subfamilies. This finding constitutes an independent criterion for testing the reliability of our phylogenetic analysis. For example, the subfamily C42 lacked introns 1 and 2; subfamily <span class="gene">C5</span> lacked intron 2; subfamily C30 lacked intron 1; and subfamily <span class="gene">C4</span> retained the typical splicing sites (Figure <xref ref-type="fig" rid="F2"> 2 </xref> b and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). Interestingly, in <italic> Arabidopsis </italic> , most MYBs contain no more than two introns in the MYB domain, except AtMYB88 and AtMYB124 in the &#x02018;guard cell division restriction&#x02019; clade, which have a complex exon/intron organization (10 and 11 introns, respectively) [ <xref ref-type="bibr" rid="B33"> 33 </xref> ]. However, in soybean, four homologous genes of AtMYB88 and AtMYB124 <span class="gene">(GmMYB051,</span> <span class="gene">GmMYB067,</span> <span class="gene">GmMYB091,</span> and <span class="gene">GmMYB138)</span> were detected and classified in subfamily C46. In common with AtMYB88 and AtMYB124, the <span class="gene">GmMYBs</span> in this subfamily also contained a complex exon/intron organization in the entire ORF (10&#x02013;12 introns). Members of this group contain up to five introns in the MYB domain, which not only supports their close evolutionary relationship, but also indicates the conservation of this intron pattern (pattern m) in evolution. Remarkably, despite the <span class="trait">high</span> number of introns in the MYB domain, the phase of the introns is almost conserved (Figure <xref ref-type="fig" rid="F3"> 3 </xref> and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). This indicates that it is not an occasional mutation event, but a conserved pattern in the evolution of plant MYB transcription factors.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the genomic distribution of the MYBs, we searched the soybean genome database using the DNA sequence of each soybean MYB gene and BLASTN. Genome chromosomal location analyses revealed that soybean MYBs were distributed on almost all chromosomes. Although each of the 20 soybean chromosomes contained MYBs, the distribution appeared to be uneven (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). In total, six <span class="gene">GmMYBs</span> were present on chromosome 14; seven on chromosome 16; 10 on each of chromosomes 1, 5, 9, 15 and 20; 11 on each of chromosomes 2, 3, 8, 11, and 17; 12 on each of chromosomes 4 and 18; 13 on chromosome 12; 14 on chromosome 10; 15 on each of chromosomes 6 and 7; 16 on chromosome 13; and 19 on chromosome 19. The gene density per chromosome was uneven, ranging from 0.3% to 0.7%, with the highest density observed on chromosome 19 and lowest density on chromosome 14 (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). On average, one R2R3-MYB gene was presented every 2.5&#x02009;Mb. Relatively <span class="trait">high</span> densities of MYBs were observed in some chromosomal regions, including the top and bottom of chromosomes 3, 4, 5, 6, 7, 8, 10, 12, 13, 18, and 19; the top of chromosomes 1, 9, and 20; and the bottom of chromosomes 2, 11, 14, 15, 16, and 17. By contrast, almost all central chromosomal regions lacked MYBs, including the centromere and the pericentromere regions (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Gene duplication has long been recognized to occur throughout plant evolution, thereby contributing to the establishment of new gene functions and underlying the origins of evolutionary novelty [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. The origin of multigene families is attributed to gene duplication that arose from region-specific duplication or genome-wide polyploidization. This mechanism has been proved to be a prominent feature of plant genome evolution. To detect a potential relationship between <span class="gene">GmMYBs</span> and genome duplications, we identified approximately 92 pairs of <span class="gene">GmMYBs</span> that were highly similar paralogs in the same subclades and shared a <span class="trait">high</span> degree of identity through their <span class="trait">protein</span> sequences. For example, the entire <span class="trait">protein</span> sequences of <span class="gene">GmMYB005</span> and <span class="gene">GmMYB042</span> shared 92% similarity, whereas those of <span class="gene">GmMYB058</span> and <span class="gene">GmMYB127</span> were 94% similar (Figure <xref ref-type="fig" rid="F4"> 4 </xref> , linked with red lines). These genes represent approximately 75% (184 of 244) of the soybean MYBs, which supported the hypothesis that they evolved from putative soybean genome duplication events. In addition, multiple pairs linked each of at least 24 potential chromosomal/segmental duplications (Figure <xref ref-type="fig" rid="F4"> 4 </xref> , pairs of bars with numbers 1&#x02013;24), such as the large sections of chromosomes 1 and 11. A genome duplication event in soybean occurred approximately 13 million years ago and resulted in a highly duplicated genome [ <xref ref-type="bibr" rid="B31"> 31 </xref> ]. Consistent with this, our results showed that many predicted <span class="gene">GmMYBs</span> had paralogous counterparts in syntenic regions of related chromosomes (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ), which indicated that segment duplication is a major driver of gene expansion in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, intrachromosomal duplication was also observed in the soybean genome. A series of tandem gene duplications was also indicated to contribute to increasing the number of R2R3-MYB family genes, as evidenced by the fact that some closely related <span class="gene">GmMYBs</span> in a single cluster were physically located near to each other on a given chromosome with no intervening annotated gene: for example, <span class="gene">GmMYB099/GmMYB226</span> at the bottom of chromosome 3, <span class="gene">GmMYB031/GmMYB112</span> at the top of chromosome 10, and <span class="gene">GmMYB026/GmMYB043</span> at the top of chromosome 19 (Figure <xref ref-type="fig" rid="F4"> 4 </xref> , marked with colored boxes). In total, about 5% (13 of 244) of <span class="gene">GmMYBs</span> were involved in tandem duplication. In general, the tandem repeat in soybean involved duplicate MYB-encoding genes, with the exception of a triplicate repeat (Figure <xref ref-type="fig" rid="F4"> 4 </xref> , chromosome 6; <span class="gene">GmMYB034/GmMYB062/GmMYB142).</span> Moreover, three specific genes <span class="gene">(GmMYB100,</span> <span class="gene">GmMYB243,</span> and <span class="gene">GmMYB244)</span> contained only the DNA-binding domain (with no C-terminal region), which suggested possible function loss for these genes. The amino acid sequences of <span class="gene">GmMYB100</span> and <span class="gene">GmMYB243</span> were identical; their corresponding cDNA and DNA sequences were also very similar (99.7% and 99.5%, respectively), except for a few discontinuous mutations in the C-terminal region. The amino acid sequence of <span class="gene">GmMYB244</span> was highly similar to those of <span class="gene">GmMYB100</span> and <span class="gene">GmMYB243,</span> except for a single-site mutation in the C-terminal region. Their <span class="trait">high</span> similarity suggested that these three genes arose from a recent duplication event, thereby providing strong evidence that gene duplication has made an important contribution to soybean MYB gene expansion.
					</p>
				
					<p>&nbsp;&nbsp;
						The schemes of <span class="trait">protein</span> motifs of individual members of the MYB family revealed the structural similarities among proteins within subfamilies (Figure <xref ref-type="fig" rid="F2"> 2 </xref> a and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). Although the functions of most of these conserved motifs have not been characterized, it is likely that some play important roles in the transcriptional regulation of target genes. For instance, motifs 7 and 8, which are conserved among members of subfamily C9 (Figure <xref ref-type="fig" rid="F2"> 2 </xref> c), have been characterized in AtMYBs as a <span class="gene">C2</span> repressor motif and zinc finger. Members of this group possess the <span class="gene">C1</span> and <span class="gene">C2</span> motifs, which are known to participate in bHLH interactions and promoter repression, respectively. In the present study, we identified three orthologous <span class="gene">GmMYBs</span> that contained the same conserved motifs in the C-terminal region, which have been shown to function as a repression domain. It is likely that these orthologous MYBs also function as repression factors in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In <italic> Arabidopsis </italic> , three atypical MYB genes (AtMYBCDC5, AtMYB3R-like, and AtMYB4R1), which encode MYB proteins with two or more repeats, are distantly related to the typical R2R3-MYB proteins [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B42"> 42 </xref> ]. AtMYBCDC5 contains an MYB domain, consisting of two repeats, that shows very low homology (approximately 31% identity) to the typical R2R3-type MYB domain, and a very long C-terminal region (more than three-fold longer than the typical R2R3-type MYB gene). In soybean, we observed two genes <span class="gene">(GmMYB147</span> and <span class="gene">GmMYB156)</span> with <span class="trait">high</span> homology to AtMYBCDC5 ( <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ; subfamily C47). Remarkably, although the length of the coding region and the intron pattern outside the MYB domain were different, the intron phase and splicing site within the MYB domain were almost identical (Figure <xref ref-type="fig" rid="F2"> 2 </xref> b). A MEME search using the amino acid sequences of these three atypical MYB proteins identified three unexpectedly large conserved motifs ( <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ; motifs 48, 49, and 50) in the C-terminal region of this subfamily. This finding may facilitate the identification of functional characteristics in these types of MYBs. However, their biological functions remain to be elucidated. We also observed two orthologous genes to AtMYB4R1, which is a putative MYB <span class="trait">protein</span> containing four R1R2-like repeats ( <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ; subfamily C48). Although the region before the MYB domain is likely to be longer than that of typical MYB genes, it appears not to be conserved in this subfamily. No subfamily-specific conserved motifs were detected at the N- or C-termini. Moreover, no homologous AtMYB3R-like gene was identified in the soybean genome; this may reflect gene loss during evolution of the soybean genome.
					</p>
				
					<p>&nbsp;&nbsp;
						In common with genes that encode transcription factors, many of the <span class="gene">GmMYBs</span> exhibited low transcript abundance levels, as determined by the RNA-Seq Atlas analysis. After integration of the two data sets, we observed that most of the genes had very broad expression spectra. However, five AtMYBs and 32 (13%) <span class="gene">GmMYBs</span> lacked expression information, which possibly indicated that these were pseudogenes or were expressed only at specific developmental stages or under special conditions. Next, we examined data from a second soybean transcriptome atlas [ <xref ref-type="bibr" rid="B49"> 49 </xref> ], which is an integrated transcriptome atlas of a variety of soybean tissues. The results showed that all of the 32 <span class="gene">GmMYBs</span> were predicted with <span class="trait">high</span> confidence in soybean tissues (data not shown), which demonstrated that these genes were expressed under highly restricted conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Although the expression pattern of MYBs varied between soybean and <italic> Arabidopsis </italic> , we also observed conserved expression profiles. In addition to groups of genes that exhibited similar transcript abundance profiles but were relatively phylogenetically distinct, several phylogenetic clades shared, to a large extent, the same transcript abundance profile. For example, in subfamily C1, most of the AtMYBs and <span class="gene">GmMYBs</span> were expressed in flower tissue, which indicated their conserved functional role in flower development. The expression of members of subfamily C20, in soybean and also in <italic> Arabidopsis </italic> , was detectable in root tissue, which suggested their conserved roles in root formation. Members of subfamily C23 showed similar expression patterns in flower and seed development, which supported the hypothesis that these genes function in soybean and <italic> Arabidopsis </italic> differentiation (Figure <xref ref-type="fig" rid="F2"> 2 </xref> d and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). By contrast, <span class="gene">GmMYBs</span> with <span class="trait">high</span> sequence similarity and shared expression profiles represent good candidates for evaluation of gene functions in soybean. Prominent among these clades are those that include MYBs related to the regulation of phenylpropanoid metabolism (subfamily C20).
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription factors are not only involved in the activation of plant defense systems, but also play key roles in the control of mutualistic interactions between plant roots and soil microorganisms. For example, nodulation involves the interaction between root and soil bacteria, leading to symbiotic nitrogen fixation, which is mainly restricted to legumes. The first transcription factor shown to be involved in nodulation was the <italic> Lotus japonicus </italic> NIN gene [ <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Subsequently, the <italic> Medicago truncatula </italic> ortholog of LjNIN was characterized by screening fast-neutron and Tnt1 transposon-tagged mutagenized populations [ <xref ref-type="bibr" rid="B52"> 52 </xref> , <xref ref-type="bibr" rid="B53"> 53 </xref> ]. Other transcription factors critical to the nodulation process have been identified, including GRAS family proteins in <italic> M.truncatula </italic> and <italic> L. japonicus </italic> [ <xref ref-type="bibr" rid="B54"> 54 </xref> , <xref ref-type="bibr" rid="B55"> 55 </xref> ], the bZIP transcription factor ASTRAY in <italic> L.japonicus </italic> [ <xref ref-type="bibr" rid="B56"> 56 </xref> ], the AP2-EREBP transcription factor MtERN in <italic> M.truncatula </italic> [ <xref ref-type="bibr" rid="B57"> 57 </xref> - <xref ref-type="bibr" rid="B59"> 59 </xref> ] and <italic> L.japonicus </italic> [ <xref ref-type="bibr" rid="B60"> 60 </xref> ], and the ARID transcription factor LjSIP1 in <italic> L. japonicus </italic> [ <xref ref-type="bibr" rid="B61"> 61 </xref> , <xref ref-type="bibr" rid="B62"> 62 </xref> ]. In the present study, we observed that 64 of 244 <span class="gene">GmMYBs</span> exhibited transcript accumulation during legume-specific nodulation, with 39 (16%) of them showing the highest expression levels in this tissue (Figure <xref ref-type="fig" rid="F2"> 2 </xref> d and <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). However, only three of the nodulation-expressed genes ( <italic> <span class="gene">GmMYB201</span> </italic> <italic> <span class="gene">GmMYB034</span> </italic> and <italic> <span class="gene">GmMYB035</span> </italic> ) are likely to be nodulation-specific, because no transcript accumulation was detected in other tissues. Moreover, <italic> <span class="gene">GmMYB201</span> </italic> has been proved to be nodulation-specific, recently [ <xref ref-type="bibr" rid="B63"> 63 </xref> ]. These results indicated that the MYB transcription factors might have a major role in regulation of legume-specific nodulation. It is interesting that what makes legumes special, such as the trait of symbiotic nitrogen fixation, and how these traits evolved in legumes. However, there are not significant differences in transcription factor gene distribution across transcription factor families in plants which suggest that species-specific traits may be likely dependent on transcription factor gene expression patterns and their functions. Interestingly, in our synteny analysis of <italic> Arabidopsis </italic> and soybean genomes, none of these three nodulation-specific-expressed <span class="gene">GmMYBs</span> has ortholog in <italic> Arabidopsis </italic> , which suggests that they may be new functionalization after the divergence of soybean and <italic> Arabidopsis. </italic> These results indicated that MYB genes may be important in the evolutional development of nodule in legumes. Although many of the <span class="gene">GmMYBs</span> showed <span class="trait">high</span> transcript accumulation during nodulation, the lack of corresponding orthologs involved in this process means that the functional characteristics of the <span class="gene">GmMYBs</span> in legume-specific nodulation remain to be elucidated.
					</p>
				
					<p>&nbsp;&nbsp;
						Currently, the most extensive annotative evaluation of plant MYBs was obtained from studies of <italic> Arabidopsis </italic> . In order to update functional clades that could also be present in soybean subfamilies, we constructed a phylogenetic tree to define MYB families of orthologs from different species, based on the well-characterized R2R3-MYB genes in <italic> Arabidopsis </italic> and some other plant species (Figure <xref ref-type="fig" rid="F2"> 2 </xref> a and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). Within each subfamily, we identified particular clusters of paralogous and orthologous genes, which were helpful for characterization of each subfamily. For example, subfamily <span class="gene">C1</span> consisted of four AtMYBs (AtMYB75, AtMYB90, AtMYB113, and AtMYB114,), 10 MYBs from other plant species (PhAn2; NtAn2; LeAn1; MdMYB10; VvMYBA1 and VvMYBA2; PyMYB10; ROSEA1; GhMYB10; and IbMYB1), and seven <span class="gene">GmMYBs</span> (Figure <xref ref-type="fig" rid="F2"> 2 </xref> a). These included gene members that contain motif 2 in the C-terminal region and are likely to have a consensus intron pattern. Although the functions of the <span class="gene">GmMYBs</span> in this subfamily are unknown, the remaining genes are known to be involved in anthocyanin biosynthesis. Thus, it is reasonable to hypothesize that the <span class="gene">GmMYBs</span> may also play a role in the anthocyanin biosynthesis pathway. Subfamily <span class="gene">C10</span> contains seven <span class="gene">GmMYBs,</span> three AtMYBs (AtMYB11, AtMYB12 and At MYB111) and five MYBs from other plant species (y1; P1; P2; VvMYBF1; and SlMYB12. Except for the <span class="gene">GmMYBs,</span> the other genes are well known to play crucial roles in the regulation of flavonol biosynthesis (Figure <xref ref-type="fig" rid="F2"> 2 </xref> a) [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. Thus, the <span class="gene">GmMYBs</span> in this subgroup might play important roles in the flavonol biosynthesis pathway. Similar results were observed for the MYBs in subfamily C38, among which the MYBs from other plant species (AtMYB2, AtMYB62, AtMYB78, AtMYB108, AtMYB112 and AtMYB116; TaPIMP1) are known to be involved in defense-related pathways, such as <span class="trait">drought,</span> salt, and pathogen response [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. The genes of subfamily C39 are involved in GA-induced responses (AtMYB33, AtMYB65, AtMYB81 and AtMYB101; LtGAMYB; HvGAMYB; and OsGAMYB), whereas subfamily C40 genes (AtMYB91; AmPHAN; LePHAN; ZmPHAN; and WRS2), including four soybean orthologs, are suggested to be involved in rough-sheath development (Figure <xref ref-type="fig" rid="F2"> 2 </xref> a and <xref ref-type="supplementary-material" rid="S3"> Additional file 3 </xref> ). Similarly, eight <span class="gene">GmMYBs</span> with homology to four AtMYBs (AtMYB44, AtMYB70, AtMYB73 and AtMYB77) in subfamily C42 were indicated to respond to abiotic stress [ <xref ref-type="bibr" rid="B42"> 42 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Combinatorial interactions between transcription factors are pivotal to the regulation of eukaryotic gene expression. The combinatorial regulation of the flavonoid biosynthetic pathway by the bHLH-MYB-WD40 complex is well characterized. These interactions can either modulate transcription factor activity or contribute to the biological specificity of factors with very similar DNA-interaction motifs. In maize, the MYB <span class="trait">protein</span> C1/Pl was reported to depend on the interaction with R/B (bHLH <span class="trait">protein)</span> for its regulatory function in anthocyanin biosynthesis [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. In <italic> Arabidopsis </italic> , TT2, TT8, and TTG1 (which encode MYB, bHLH, and WDR proteins, respectively) regulate expression of several flavonoid structural genes [ <xref ref-type="bibr" rid="B67"> 67 </xref> , <xref ref-type="bibr" rid="B68"> 68 </xref> ]. Similar regulatory factors that control phenylpropanoid biosynthesis have been isolated from other species [ <xref ref-type="bibr" rid="B69"> 69 </xref> , <xref ref-type="bibr" rid="B70"> 70 </xref> ]. Further investigations revealed that four residues in the first helix (all four residues must be present) and two additional residues in the second helix of the <span class="gene">C1</span> R3 repeat are necessary for the interaction with R in plant cells [ <xref ref-type="bibr" rid="B71"> 71 </xref> ]. Interestingly, in the present study, we observed that up to 10 <span class="gene">GmMYBs</span> had all six residues in the same location. Although these clustered in different clades within the MYB subfamily (C4 and C6), and appeared to have different target genes, we propose that these genes have a combinatorial regulatory mechanism to regulate expression of soybean target genes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic and Physical Mapping of Candidate Genes for <span class="trait">Resistance</span> to <italic> Fusarium oxysporum </italic> f.sp. <italic> tracheiphilum </italic> Race 3 in Cowpea [ <italic> Vigna unguiculata </italic> (L.) Walp]</h2>
			<h3>PMID:22860000</h3>
				
					<h3>PMCID:3409238</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Pottorff </surname>  <given-names> Marti </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wanamaker </surname>  <given-names> Steve </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Yaqin Q. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ehlers </surname>  <given-names> Jeffrey D. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Roberts </surname>  <given-names> Philip A. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Close </surname>  <given-names> Timothy J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The cowpea region carrying the <italic> Fot3-1 </italic> locus was compared with the soybean genome using HarvEST: Cowpea to determine if the gene order was conserved between species. <span class="trait">High</span> co-linearity with the <italic> Fot3-1 </italic> region in any of the sequenced genomes may enable identification of candidate genes. The <italic> Fot3-1 </italic> region was found to be highly co-linear with two regions of soybean, chromosome 9 and chromosome 15 ( <xref ref-type="fig" rid="pone-0041600-g003"> Figure 3 </xref> ). The syntenic region in soybean chromosome 9 extended from soybean locus <span class="gene">Glyma09g02100</span> to <span class="gene">Glyma09g02560</span> which corresponded to 17.14 cM to 19.04 cM of the <italic> Fot3-1 </italic> locus on the cowpea consensus genetic map ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). The syntenic region was scanned for known disease <span class="trait">resistance</span> genes on the soybean genome browser ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.org"> http://www.phytozome.org </ext-link> ) where two soybean disease <span class="trait">resistance</span> genes were observed. Soybean locus <span class="gene">Glyma09g02210</span> was flanked by orthologous soybean genes to EST-derived SNP markers 1_1211 and 1_1484 and was annotated as a leucine-rich repeat (LRR) serine/threonine <span class="trait">protein</span> kinase ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). <span class="gene">Glyma09g02420</span> was flanked by SNP markers 1_0860 and 1_1107 and was annotated as a disease <span class="trait">resistance</span> <span class="trait">protein</span> of the NBS-LRR class ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). The <italic> Fot3-1 </italic> syntenic locus in soybean chromosome 15 extended from soybean locus <span class="gene">Glyma15g12830</span> to <span class="gene">Glyma15g13470</span> which corresponded to 17.14 cM to 19.04 cM of the <italic> Fot3-1 </italic> locus on the cowpea consensus genetic map ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). The syntenic region of soybean chromosome 15 was scanned and four LRR genes were observed, <span class="gene">Glyma15g13100,</span> <span class="gene">Glyma15g13290,</span> <span class="gene">Glyma15g13300</span> and <span class="gene">Glyma15g13310</span> ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). <span class="gene">Glyma15g13100</span> was flanked by orthologous soybean genes to SNP markers 1_1077 and 1_1484 and was annotated as a LRR serine/threonine <span class="trait">protein</span> kinase ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). Soybean loci <span class="gene">Glyma15g13290,</span> <span class="gene">Glyma15g13300</span> and <span class="gene">Glyma15g13310</span> were identified between orthologous soybean genes to markers 1_0860 and 1_1212 ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). <span class="gene">Glyma15g13290</span> and <span class="gene">Glyma15g13300</span> were annotated as disease <span class="trait">resistance</span> proteins of the NBS-LRR class while <span class="gene">Glyma15g13310</span> was annotated as an LRR <span class="trait">protein</span> ( <xref ref-type="table" rid="pone-0041600-t003"> Table 3 </xref> ). Due to the <span class="trait">high</span> co-linearity of gene order between cowpea and soybean at the two syntenic loci, the observed soybean disease <span class="trait">resistance</span> genes were considered as orthologous candidate genes for the <italic> Fot3-1 </italic> locus. Soybean is the closest related legume model species to cowpea and both are members of the economically important warm season Phaseoleae clade <xref ref-type="bibr" rid="pone.0041600-Choi1"> [12] </xref> . The ability to use the sequenced soybean genome as a means to identify candidate genes within syntenic regions in cowpea exhibits the utility of these closely related legumes.
					</p>
				
					<p>&nbsp;&nbsp;
						Synteny was examined for the <italic> Fot3-1 </italic> locus between cowpea and <italic> G. max </italic> using EST-derived SNP markers previously BLASTed and aligned to the sequenced genome. The <italic> Fot3-1 </italic> locus on the cowpea consensus genetic map, linkage group 6 (17.88 cM to 19.04 cM), was determined to be syntenic with soybean chromosomes 9 and 15. The <italic> Fot3-1 </italic> syntenic locus in soybean chromosome 9 extended from soybean locus <span class="gene">Glyma09g02100</span> to <span class="gene">Glyma09g02560,</span> where two disease <span class="trait">resistance</span> genes, <span class="gene">Glyma09g02210</span> and <span class="gene">Glyma09g02420,</span> were observed. The <italic> Fot3-1 </italic> syntenic locus in soybean chromosome 15 extended from soybean locus <span class="gene">Glyma15g12830</span> to <span class="gene">Glyma15g13470</span> where four disease <span class="trait">resistance</span> genes were observed, <span class="gene">Glyma15g13100,</span> <span class="gene">Glyma15g13290,</span> <span class="gene">Glyma15g13300</span> and <span class="gene">Glyma15g13310.</span> The syntenic map was drawn using HarvEST:Cowpea database ( <ext-link ext-link-type="uri" xlink:href="http://harvest.ucr.edu"> http://harvest.ucr.edu </ext-link> ) using a cut-off e-score value of &#x02212;10 and a minimum number of 13 lines drawn per linkage group.
					</p>
				
					<p>&nbsp;&nbsp;
						The candidate BAC clone CH093L18 which harbors SNP markers 1_0860 and 1_1212 was also sequenced and the assembly resulted in 127 contigs with an estimated length of 184,856 bp (File S5). The EST sequences from which SNP markers 1_0860 and 1_1212 were derived were BLASTed to the assembled BAC clone CH093L18 to confirm their presence and the quality of assembly; both SNPs were identified (File S6). The six soybean candidate genes were BLASTed to CH093L18 to possibly identify orthologous cowpea candidate genes. <span class="gene">Glyma09g02210</span> was the only soybean gene which returned a <span class="trait">high</span> similarity with several nodes of the cowpea BAC clone ( <xref ref-type="table" rid="pone-0041600-t004"> Table 4 </xref> ). The assembled sequences of BAC clone CH093L18 were then BLASTed to the soybean genome to determine gene annotations for the entire clone. It appeared that there were twenty-five putative cowpea genes on BAC clone CH093L18 and that the only disease resistant-type genes were NODES 50, 57, 65 and 104 which were annotated as leucine-rich repeat serine/threonine <span class="trait">protein</span> kinases ( <xref ref-type="table" rid="pone-0041600-t004"> Table 4 </xref> ). The <italic> Fot3-1 </italic> <span class="trait">resistance</span> locus was narrowed to BAC clone CH093L18 and leucine-rich repeat serine/threonine <span class="trait">protein</span> kinases were identified as the cowpea candidate gene for <italic> Fot3-1 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean candidate disease <span class="trait">resistance</span> gene, <span class="gene">Glyma09g02210,</span> was BLASTed to the cowpea genome vs. 0.02 to identify candidate genomic sequences for <italic> Fot3-1 </italic> . The BLASTn search for the genomic and cDNA sequence of <span class="gene">Glyma09g02210</span> returned a <span class="trait">high</span> alignment with scaffold 17795 with e-score values of e-155 and e-147, respectively (File S7). The sequences for scaffold 17795 were then BLASTed back to BAC clone CH093L18 to determine which NODE of the assembly had the highest similarity; NODE 50 returned a perfect alignment with e-score value of 0.0 (File S8). We concluded that NODE 50 on BAC clone CH093L18 was the best candidate cowpea gene for the <italic> Fot3-1 </italic> locus and that scaffold 17795 may be the cowpea ortholog to soybean <span class="gene">Glyma09g02210.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Typically, <span class="trait">resistance</span> to Fusarium has been shown to be a dominant and monogenic trait <xref ref-type="bibr" rid="pone.0041600-Zink1"> [16] </xref> , <xref ref-type="bibr" rid="pone.0041600-Rubio1"> [17] </xref> , <xref ref-type="bibr" rid="pone.0041600-McGrath1"> [18] </xref> , <xref ref-type="bibr" rid="pone.0041600-Scott1"> [19] </xref> , <xref ref-type="bibr" rid="pone.0041600-Sarfatti1"> [20] </xref> which fits the gene-for-gene hypothesis whereby pathogen and host express complementary dominant genes <xref ref-type="bibr" rid="pone.0041600-Flor1"> [21] </xref> . The alteration or loss to either the host&#x02019;s <span class="trait">resistance</span> gene or pathogen&#x02019;s avirulence gene leads to disease <xref ref-type="bibr" rid="pone.0041600-Flor1"> [21] </xref> . The majority of disease <span class="trait">resistance</span> genes are classified as having an NBS-LRR motif which has been further sub-divided by their difference at the N- terminus, either having homology with the <span class="gene">TIR</span> domain (TIR-NBS-LRR) <xref ref-type="bibr" rid="pone.0041600-Meyers1"> [22] </xref> , <xref ref-type="bibr" rid="pone.0041600-Pan1"> [23] </xref> or a coiled-coil motif (CC-NBS-LRR or non TIR-NBS-LRR) <xref ref-type="bibr" rid="pone.0041600-Pan1"> [23] </xref> . Currently, two genes have been cloned which confer <span class="trait">resistance</span> to <italic> F </italic> . <italic> oxysporum </italic> , <italic> I-2 </italic> and <italic> Fom-2 </italic> <xref ref-type="bibr" rid="pone.0041600-Simons1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0041600-Joobeur1"> [25] </xref> . The <italic> I-2 </italic> locus, which confers <span class="trait">resistance</span> to <italic> F. oxysporum </italic> f.sp. <italic> lycopersici </italic> (Fol) race 2 in tomato was determined to be a CC-NBS-LRR disease <span class="trait">resistance</span> gene <xref ref-type="bibr" rid="pone.0041600-Simons1"> [24] </xref> . The <italic> Fom-2 </italic> locus, which confers <span class="trait">resistance</span> to <italic> F. oxysporum </italic> f.sp <italic> melonis </italic> (Fom) in melon was also identified as a CC-NBS-LRR gene <xref ref-type="bibr" rid="pone.0041600-Joobeur1"> [25] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Although the majority of cloned R genes have the conserved NBS-LRR structure, there are several disease <span class="trait">resistance</span> genes identified as belonging to the receptor-like kinase (RLK) family. RLKs are proteins that span the plasma membrane, recognizing and responding to extracellular signals <xref ref-type="bibr" rid="pone.0041600-Geer1"> [26] </xref> . The majority of <span class="gene">RLK</span> have serine/threonine kinases and LRR motifs <xref ref-type="bibr" rid="pone.0041600-Becraft1"> [27] </xref> . The receptor-like cytoplasmic kinase (RLCK) disease <span class="trait">resistance</span> genes include <italic> PBS1 </italic> , <italic> Pti </italic> and <italic> Pto </italic> <xref ref-type="bibr" rid="pone.0041600-Shiu1"> [28] </xref> . <italic> PBS1 </italic> confers <span class="trait">resistance</span> against <italic> Pseudomonas syringae </italic> pv <italic> phaseolicola </italic> in Arabidopsis <xref ref-type="bibr" rid="pone.0041600-Swiderski1"> [29] </xref> . <italic> Pti </italic> and <italic> Pto </italic> both confer <span class="trait">resistance</span> to the bacterium <italic> Pseudomonas syringae </italic> pv <italic> tomato </italic> <xref ref-type="bibr" rid="pone.0041600-Zhou1"> [30] </xref> , <xref ref-type="bibr" rid="pone.0041600-Martin1"> [31] </xref> . <italic> Xa21 </italic> is a LRR <span class="gene">RLK</span> and confers <span class="trait">resistance</span> against <italic> Xanthomonas campestris </italic> pv <italic> oryzae </italic> in rice <xref ref-type="bibr" rid="pone.0041600-Song1"> [32] </xref> . <italic> Lrk10 </italic> which confers <span class="trait">resistance</span> to the fungus, <italic> Puccinia recondite </italic> in wheat was also determined to be a serine/threonine <span class="trait">protein</span> kinase <xref ref-type="bibr" rid="pone.0041600-Feuillet1"> [33] </xref> . The <italic> I-3 </italic> locus which confers <span class="trait">resistance</span> to <italic> F. oxysporum </italic> f.sp. <italic> lycopersi </italic> race 3 in tomato, was determined to be positioned within a large cluster of S-locus receptor-like kinases (SRLK) <xref ref-type="bibr" rid="pone.0041600-Hemming1"> [34] </xref> . Interestingly, we recently identified TIR-NBS-LRR proteins and leucine-rich repeat serine/threonine <span class="trait">protein</span> kinases in the <italic> Fot4-1 </italic> and <italic> Fot4-2 </italic> syntenic regions of soybean (unpublished data). <italic> Fot4-1 </italic> and <italic> Fot4-2 </italic> confer <span class="trait">resistance</span> to Fot race 4 in cowpea (unpublished data). It may be possible that leucine-rich repeat serine/threonine <span class="trait">protein</span> kinases are the R genes conferring <span class="trait">resistance</span> in the cowpea-Fusarium pathovar system.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Deep Sequencing of the Transcriptomes of Soybean Aphid and Associated Endosymbionts</h2>
			<h3>PMID:22984624</h3>
				
					<h3>PMCID:3440339</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Sijun </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chougule </surname>  <given-names> Nanasaheb P. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vijayendran </surname>  <given-names> Diveena </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bonning </surname>  <given-names> Bryony C. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 8 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Gene annotation revealed that 43.4% of the <italic> Buchnera </italic> genes identified contain motifs that function in metabolic processes and 36% have a role in cellular processes ( <xref ref-type="fig" rid="pone-0045161-g002"> Figure 2 </xref> ). In molecular functions, 42% have catalytic activity and 42% are predicted to function in binding ( <xref ref-type="fig" rid="pone-0045161-g002"> Figure 2 </xref> ). The most highly expressed bacterial genes are the essential genes encoding ribosomal, cell division and chaperone/protease proteins <xref ref-type="bibr" rid="pone.0045161-Vinuelas1"> [39] </xref> , many of which were identified in the soybean aphid <italic> Buchnera </italic> transcripts. For instance, we identified the transcripts of 27 50S ribosomal <span class="trait">protein</span> L and 20 30S ribosomal <span class="trait">protein</span> S ( <xref ref-type="supplementary-material" rid="pone.0045161.s002"> Table S2 </xref> ), which were 69.2% of the annotated 50S ribosomal <span class="trait">protein</span> L and 74.1% of annotated 30 S ribosomal <span class="trait">protein</span> S from the <italic> Buchnera </italic> associated with the pea aphid (str. 5A and APS). We also identified eight transcripts related to cell division functions ( <italic> MInC </italic> , Dand <italic> E </italic> , <italic> FtsA, H, J W </italic> and <italic> Z </italic> ) and chaperone/heat shock proteins (e.g. <italic> dnaJ, dnak, groEL, groES, HtpX, htpG, hscA, <span class="gene">hslU</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						C002 is a 219 amino acid (aa) peptide, which was originally discovered from the pea aphid SG EST library. <italic> C002 </italic> was primarily expressed in the SG of the pea aphid, but transcripts of <italic> C002 </italic> were also detected in the gut at a level of 1% that in the SG <xref ref-type="bibr" rid="pone.0045161-Mutti1"> [24] </xref> . This <span class="trait">protein</span> was predicted from the pea aphid genome as a hypothetical <span class="trait">protein</span> (XP_001948358.2, <span class="gene">LOC100167863).</span> By conducting local BLAST analysis with the <italic> C002 </italic> sequence, we identified a full-length copy of the putative <italic> C002 </italic> homolog (see <xref ref-type="supplementary-material" rid="pone.0045161.s005"> Figure S1 </xref> ; [GenBank: JN135246]) from a single contig assembled from the WA reads with about 43-fold coverage, and a partial <italic> C002 </italic> sequence was assembled from the gut Illumina reads, reflecting the lower expression of <italic> C002 </italic> in the gut. The putative soybean aphid C002 is 214 aa, 5 aa shorter than that of the pea aphid C002. Alignment of the soybean C002 homolog with the pea aphid C002 showed less than 50% sequence identity at the <span class="trait">protein</span> level ( <xref ref-type="supplementary-material" rid="pone.0045161.s005"> Figure S1b </xref> ). C002 is secreted into the host plant and plays an important role in feeding, and hence may be involved in host plant selection <xref ref-type="bibr" rid="pone.0045161-Mutti1"> [24] </xref> . The lower identity between the soybean aphid and pea aphid C002 may reflect the differences in the host plant preferences of the two species and selection for divergent <span class="trait">protein</span> sequences to deal with some aspect of survival on the host plant. Functional analysis is required to confirm that silencing of this gene in the soybean aphid has similar effects to those reported for the pea aphid <xref ref-type="bibr" rid="pone.0045161-Mutti2"> [25] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Divergent patterns of endogenous small RNA populations from seed and vegetative tissues of <italic> Glycine max </italic></h2>
			<h3>PMID:23031057</h3>
				
					<h3>PMCID:3534067</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> g-zabala@life.illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Campos </surname>  <given-names> Edhilvia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> camposve@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Varala </surname>  <given-names> Kranthi K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kvarala2@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Bloomfield </surname>  <given-names> Sean </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> seanbloomfield@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Jones </surname>  <given-names> Sarah I </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sijones@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Win </surname>  <given-names> Hlaing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hwin2@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Tuteja </surname>  <given-names> Jigyasa H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jigyasa_tuteja@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Calla </surname>  <given-names> Bernarda </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> calla2@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Clough </surname>  <given-names> Steven J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sjclough@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Hudson </surname>  <given-names> Matthew </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mhudson@illinois.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Vodkin </surname>  <given-names> Lila O </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@illinois.edu </email>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">high</span> throughput sequence outcomes from different plant species have revealed the great abundance and complexity of these smRNA endogenous populations. The focus has been those sequences with <span class="trait">high</span> repetitions ranging in size between 21 to 24 nt. In turn, the smRNA sequence length has become an indicator of the DCL enzyme and the pathway generating them. The miRNAs are mostly 21 nt and the result of the <span class="gene">DCL1</span> containing complex. On the other hand, the majority of the siRNAs representing highly repetitive elements are processed from double stranded RNA (dsRNA) synthesized by the plant-specific RNA polymerase IVa (Pol IVa) followed by RNA-dependent RNA polymerase 2 (RDR2) reverse transcription and diced by DCL3 into 24-nt siRNAs. Other siRNAs, 21 nt in length, are derived from genomic regions with inverted repeats or inverted repeated genes. These siRNAs are most likely processed from single-stranded hairpin RNA precursors in a process resembling miRNA biogenesis. Some of these siRNAs might represent &#x0201c;evolving miRNAs&#x0201d; that target endogenous genes with sequence similarity. Recently, it was discovered that low abundance 22-nt miRNA/transacting siRNA products of DCL1/DCL2) bind to specific AGO/DCL/RDR2 complexes to produce dsRNAs and a second wave of phased secondary (or transitive) siRNAs from the regions surrounding their target sites [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. The 21-nt tasiRNAs are the product of miRNA-directed fracturing of <italic> TAS </italic> transcripts via the pathway involving the DCL4/RDR6 complex [ <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A group of signatures annotated as pentatricopeptide repeat-containing <span class="trait">protein</span> (PPR) had different origins, with some of the signatures originating from miRNAs and others being siRNAs. The most abundant signature (#22 in Table <xref ref-type="table" rid="T2"> 2 </xref> ) had similarity to gma-miR1508a and gma-miR1508b and was expressed in all tissues examined with the highest level (7892 occurrences) observed in germinating cotyledons. This miRNA maps to two loci of origin [chromosome 16 and 19 <span class="gene">(Gm16,</span> <span class="gene">Gm19)]</span> and potentially targets four different Glyma cDNAs annotated as PPR repeat containing <span class="trait">protein.</span> Also in this annotation group was a smRNA (#23, Table <xref ref-type="table" rid="T2"> 2 </xref> ) that matches an inverted repeat region on <span class="gene">Gm19</span> capable of forming a secondary structure with dG&#x02009;=&#x02009;-61.60 and similar to gma-miR4413. It is expressed at relatively low levels in all tissues and potentially targets six different Glyma gene models having the PPR motif. Further inspection using Bowtie [ <xref ref-type="bibr" rid="B39"> 39 </xref> ] (or <italic> MultAlin </italic> [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]) found a series of 115 unique siRNAs matching either the sense or antisense strand of the <span class="gene">Glyma09g07250</span> PPR gene&#x02019;s first exon (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> siRNAs aligned to soybean genomic sequences. </bold> Collections of smRNAs with the same annotation were aligned to specific genomic sequences with Bowtie. ( <bold> A </bold> ) Pentatricopeptide repeat-containing <span class="trait">protein</span> (PPR) siRNAs to <span class="gene">Glyma09g07250.1.</span> ( <bold> B </bold> ) Auxin response factor (ARF) siRNAs to <span class="gene">Glyma05g27580.1.</span> ( <bold> C </bold> ) <span class="gene">TIR</span> domain resistant gene (TIR-NBS-LRR) siRNAs to <span class="gene">Glyma12g27800.1.</span> ( <bold> D </bold> ) Aquaporin siRNAs aligned to an inverted repeat with similarity to an aquaporin gene NIP6-1, <span class="gene">Glyma07g03030.1</span> shown in (E). (F) Aquaporin siRNAs unrelated to those in D and E aligned to another aquaporin transporter, <span class="gene">Glyma19g37000.1.</span> Blue diamonds represent siRNAs aligning to the negative strand and red diamonds to the positive strand. Green arrow in (A) marks the binding site of gma-miR4413. Dashed lines connect the region of similarity in <span class="gene">Glyma07g03030.1</span> ( <bold> E </bold> ) to the aquaporin inverted repeat (D).
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, a second subgroup of 95 unique smRNAs annotated as Auxin response factor 6 mapped to a second gene, <span class="gene">Glyma08g10550,</span> with an auxin response factor motif. The smRNA signature with the most reads in this subgroup was found in the four seed coat libraries. The two gene models, <span class="gene">Glyma08g10550</span> and <span class="gene">Glyma05g27580,</span> represent members of a gene family since their sequences have <span class="trait">high</span> degree of similarity. It is interesting that only the exons at the 3&#x02019;-end of the two genes were the source of the two siRNA subgroups. A <italic> G. max </italic> BLAST search in Phytozome with the 2 kb region aligning to the siRNAs found two additional loci in <span class="gene">Gm13</span> <span class="gene">(Glyma13g29320)</span> and <span class="gene">Gm15</span> <span class="gene">(Glyma15g09750)</span> with an auxin response motif. However, only nine of the siRNAs in the auxin response factor subgroup aligned to <span class="gene">Glyma13g29320,</span> and one to <span class="gene">Glyma15g09750.</span> All four related genes map to four different chromosomes and it was not obvious where and how the primary inducer(s) of the auxin response factor siRNAs originated. Regardless of the place of origin, the pathway that generated them is tissue specific since all these siRNAs were found for the most part in the seed coat libraries. In sum, these two siRNA collections likely result from silencing at least two auxin response factor genes in a tissue specific manner in the seed coat.
					</p>
				
					<p>&nbsp;&nbsp;
						Many of the annotations of other Glyma models that matched siRNAs predict their involvement in transcription or physiological processes, such as <span class="gene">Glyma13g32810.1,</span> annotated as MADS-box transcription factor 27 which could potentially translate a transcription factor involved in DNA-dependent transcription regulation. Also, <span class="gene">Glyma09g02920.1,</span> annotated as a &#x0201c;dsRNA-specific nuclease Dicer and related ribonucleases&#x0201d;, aligned to 37 unique siRNAs. A larger group of 334 unique siRNAs mapped to <span class="gene">Glyma12g27800.1</span> annotated as a signal transduction transmembrane receptor with a <span class="gene">TIR</span> domain. <italic> MultAlin </italic> [ <xref ref-type="bibr" rid="B40"> 40 </xref> ] clearly revealed they were dispersed siRNAs, some with <span class="trait">high</span> counts, aligned to this Glyma model which might silence this putative <span class="gene">TIR</span> disease <span class="trait">resistance</span> gene (Figure <xref ref-type="fig" rid="F5"> 5C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						An additional search using a different <italic> G. max </italic> (Williams 82) Transposable Element Database [ <xref ref-type="bibr" rid="B42"> 42 </xref> ] was carried out to identify smRNAs from the entire collection that could be derived or linked to repeated elements and to a particular TE class. A total of 54,281 unique smRNA signatures matched sequences in this TE-database and of those, 27,731 were of the <italic> Gypsy </italic> family of elements, 11,397 <italic> Mutator </italic> , 11,165 <italic> Copia </italic> , 1,384 <italic> PIF-Harbinger </italic> , 1,377 L1, 553 <italic> CACTA </italic> , 311 unknown family, 170 <italic> Helitron </italic> , 76 <italic> hAT </italic> , 75 <italic> Tc1-Mariner </italic> and 42 <italic> PONG </italic> . Although the size range of the unique signatures associated with repeated elements varied from 15 to 35 nt, the 24-nt size class was the most abundant (17,206) followed by those with 22 nt in length (14,024) (Figure <xref ref-type="fig" rid="F7"> 7A </xref> ). When the total number of sequences per signature was taken into account, the 22-nt size class was the most abundant (1,107,944) followed by the 21-nt (849,187) and 24-nt class (597,566) (Figure <xref ref-type="fig" rid="F7"> 7B </xref> ) suggesting that the 24-nt signatures were more diverse than the 22 and 21-nt signatures. During the course of this analysis, it was observed that the extremely abundant soybean gma-miR3522 miRNA, previously mentioned to account for 3.9 million reads in the leaf library and 4.8 million total reads among all eight libraries, imprecisely matched a single position <span class="gene">(Gm02:21389789.21394849)</span> flanked by two LTRs and related to a <italic> Gypsy </italic> element. This 21-nt targeted site, with the 2-nt mismatches allowed in the Bowtie alignment, it is not repeated as such, in its entirety, elsewhere in the genome but rather in a degenerated form on ten other chromosomes. In these locations the imperfect 21-nt target site has been truncated separating the 16-nt at the 5&#x02019;end from the 5-nt at the 3&#x02019;end. If the <italic> Gypsy </italic> element on <span class="gene">Gm02</span> were expressed, it is possible that it could be silenced by miR3522 despite the 2-nt mismatch within the 21-nt miR3522 sequence. However, we found no small RNAs complementing the <span class="gene">Gm02</span> <italic> Gypsy </italic> element other than miR3522 in a bowtie search of all eight small RNA libraries indicating that the element is not likely expressed and silenced in the tissues examined. Regardless of the potential action of miR3522 on this family of retroelements, the <span class="trait">high</span> abundance of miR3522 is not tied to the repeated nature of the retroelement&#x02019;s LTR ends and it&#x02019;s sequence is even more degenerate in other locations and therefore, this extremely abundant miR3522 sequence was removed from the size distribution analysis of small RNAs matching to transposons shown in Figure <xref ref-type="fig" rid="F7"> 7 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						A comprehensive analysis was compiled on variation in the miRNAs present in a large population of 41 million smRNA sequences derived from eight libraries representing multiple tissues/organs of the soybean plant. At least 4,031 unique signatures corresponding to 213 miRNAs and representing 117 families were detailed for each of the sequences falling within each family ( <xref ref-type="supplementary-material" rid="S2"> Additional file 2 </xref> ). Examination of the number of reads of each signature revealed all the length or sequence variations found in the <span class="trait">high</span> throughput populations for all miRNA family members. Thus, the flux of expression patterns for each of the unique signatures in eight libraries representing both seed and vegetative tissues of the seedling are detailed. Preferential expression in the seed coats or in cotyledons of immature seeds as shown for certain family members by sequencing were verified by small RNA blots (Table <xref ref-type="table" rid="T2"> 2 </xref> , Figures <xref ref-type="fig" rid="F2"> 2 </xref> and <xref ref-type="fig" rid="F3"> 3 </xref> ). Some of the miRNAs were very abundant in certain tissues such as members of the miR167 family and gma-1512c, which were highly prevalent in the seed coats from multiple samples. Several others showed preferential expression in either of the germinating cotyledons, stems, or leaves. One single miRNA, gma-miR3522, accounted for 11% of the total reads in the immature cotyledons and an astonishing 65% of the total reads in the 1-month-old expanded leaf library. Promoter analysis of the gene <span class="gene">(Glyma15g06080)</span> encoding miR3522 found it contained cis-acting regulatory elements for tissue specific and enhanced gene expression ( <xref ref-type="supplementary-material" rid="S4"> Additional file 4 </xref> ). In the leaf sample, the diversity of unique small RNAs was reduced by four to five-fold compared to all the other libraries (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Whether this resulted from preferential amplification of the highly abundant gma-miR3522 during the cluster formation in the sequencing reaction is unknown. However, all of the other library samples yielded 27-38,000 diverse sequences and yet some individual miRNAs signatures repeatedly accounted 1-5% of the total counts in the multiple seed coat libraries. Thus, the <span class="trait">high</span> count miRNAs are likely biological in nature.
					</p>
				
					<p>&nbsp;&nbsp;
						Of particular interest was the distinct tissue expression of two subgroups of the same miR156 family. Although the sequences of the two subgroups differed only in one nucleotide, the gma-miR156c/d/e/g members were expressed at very low levels in the developing cotyledon (25 counts) while gma-miR156b/f members were expressed at <span class="trait">high</span> levels to 98,674 normalized counts which was nearly 10% of the same smRNA population of the cotyledon sample. The reverse is true for the stem and leaf expression (Table <xref ref-type="table" rid="T2"> 2 </xref> ). The gma-miR156c/d/e/g signature found matches to at least seven loci in six chromosomes with coding potential for miR156 genes. Given the low number of reads (25) in the developing cotyledon, one possibility is that many of these seven miRNA genes are suppressed in a tissue specific manner. The coordinated regulation of so many miRNA genes dispersed through at least six chromosomes could involve a transcription factor [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. In all, the miR156 gene family in soybean is composed of at least 18 putative member genes expressed differentially. The mature miR156 family members have been shown to target and silence genes of the <span class="gene">SBP</span> transcription factor family in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. Our results show the power of <span class="trait">high</span> throughput sequencing to quantitatively distinguish the differential expression of miRNA family members with single nucleotide polymorphisms in the mature miRNA sequences, which is not possible to do with blotting and difficult to achieve with quantitative RT-PCR.
					</p>
				
					<p>&nbsp;&nbsp;
						Two other potentially silenced genes, <span class="gene">Glyma05g27580</span> (Figure <xref ref-type="fig" rid="F5"> 5B </xref> ) and <span class="gene">Glyma08g10550</span> (not shown), both encoding putative auxin response factors, were revealed by two separate smRNA sets. Since additional related genomic sequences reside in separate chromosomes it was difficult to decipher the driving mechanism that could induce the production of siRNAs matching those two types of auxin response factor genes. However, it appears to be tissue specific since the majority of the siRNAs were found in seed coat libraries. This could be another example of silencing specific to the seed coat as that exhibited by the abundant <italic> <span class="gene">CHS</span> </italic> siRNAs [ <xref ref-type="bibr" rid="B24"> 24 </xref> ]. <span class="gene">Glyma12g27800,</span> encoding a disease <span class="trait">resistance</span> <span class="trait">protein</span> containing a TIR-motif, was identified as the source of a collection of 334 unique siRNAs (Figure <xref ref-type="fig" rid="F5"> 5C </xref> ). Other genes, <span class="gene">Glyma13g32810</span> and <span class="gene">Glyma15g06490,</span> annotated as &#x0201c;Mads-box transcription factor 27&#x0201d; were found to be the likely source of a collection of 77 unique siRNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						During investigation of the nature of smRNAs that match to more than 10 positions in the soybean genome, we discovered that many of these were siRNAs derived from ribosomal RNA genes (rDNA) representing the 26S rRNA sequence (AK286061.1) and 18S rRNA gene (X02623.1) although the annotations of these genomic regions were not shown as rDNA genes in Phytozome. Instead, Phytozome has assigned Glyma model numbers and peptide motifs to these coding regions including a serine-threonine <span class="trait">protein</span> kinase gene <span class="gene">(Glyma16g29220)</span> with similarity to only the 26S rRNA sequence (AK286061.1) and several predicted genes such as allatostatin <span class="gene">(Glyma13g11820),</span> polymerase Rpb1 C-terminal repeat <span class="gene">(Glyma13g11940)</span> and copper binding octapeptide <span class="gene">(Glyma13g11980).</span> These occur in up to 14 repeated sequence stretches containing the 26S rRNA (AK286061.1), the 18S rRNA (X02623.1) or 5.8S rRNA (FJ980442.1) genes. Recently, a study [ <xref ref-type="bibr" rid="B46"> 46 </xref> ] showed that misannotations of rDNA genes can generate false positive matches in many sequencing studies of both prokaryotes and eukaryotes as these misannotations have been propagated in databases such as GenBank and SwissProt. The more than 15 Glyma models in <span class="gene">Gm13</span> containing multiple rRNA repeats may constitute the nuclear organizer which may protect the ribosomal transcripts from silencing. The siRNAs that complement these ribosomal gene sequences may derive from similar sequences repeated in other chromosomes such as the above mentioned 26S rRNA insertion in <span class="gene">Glyma16g29220</span> (a serine-threonine <span class="trait">protein</span> kinase gene) also found in <span class="gene">Gm05,</span> <span class="gene">Gm15,</span> <span class="gene">Gm14</span> and several others.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of wild soybean miRNAs and their target genes responsive to aluminum stress</h2>
			<h3>PMID:23040172</h3>
				
					<h3>PMCID:3519564</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Zeng </surname>  <given-names> Qiao-Ying </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zengqiaoying123@163.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Yang </surname>  <given-names> Cun-Yi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ycy@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Ma </surname>  <given-names> Qi-Bin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mqb680210@sina.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Li </surname>  <given-names> Xiu-Ping </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lxpyx2008@163.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Dong </surname>  <given-names> Wen-Wen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dww.1122@163.co </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hnian@scau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In most cases, the identified miRNAs were predicted to cleave two or more different targets. For example, nine members of CCAAT-binding transcription factor family genes were predicted to be cleaved by PN-miR169 (identified in the Al-treated library), and PN-miR319 was predicted to slice eight genes belonging to the Myb and TCP families of transcription factors (T-plots of six of the targets are presented in Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). In contrast, only one target each was identified for gso-miR1509, gso-miR2109 and PN-miR399 (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). Interestingly, some transcripts were found to be regulated by pairs of miRNAs. For instance, PN-miR159 and PN-miR319 targeted two Myb family transcription factors <span class="gene">(Glyma13g04030</span> and <span class="gene">Glyma20g11040),</span> and PN-miR156 and PN-miR157 sliced five members of the <span class="gene">SBP</span> domain <span class="trait">protein</span> family, suggesting that pairs of miRNAs might have a combinatorial function in gene regulation networks (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Cross adaptation is a common phenomenon in plants exposed to different combinations of stresses [ <xref ref-type="bibr" rid="B64"> 64 </xref> ]. Researches have revealed that some conserved miRNAs that are responsive to biotic and abiotic stresses might play roles in cross adaptation [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. In this study, miR156 which were reported earlier to be down-regulated in response to cadmium treatment [ <xref ref-type="bibr" rid="B65"> 65 </xref> ] were found to be down-regulated under Al stress. In Arabidopsis, miR164 was reported to be induced by <span class="trait">drought</span> stress which cleaved the <span class="gene">NAC1</span> transcription factor gene leading to the down-regulation of auxin signals and resulting in reduction in lateral root emergence [ <xref ref-type="bibr" rid="B66"> 66 </xref> ]. We found that miR164 was down-regulated under Al stress (Table <xref ref-type="table" rid="T3"> 3 </xref> , Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). In plants, miR169 was found to be responsive to abiotic stresses such as cold, <span class="trait">drought,</span> and salinity [ <xref ref-type="bibr" rid="B67"> 67 </xref> - <xref ref-type="bibr" rid="B71"> 71 </xref> ]. Recently, miR169 were also found to be responsive to some biotic stresses such as a virulent form of the bacterium, <italic> P. syringae pv. tomato </italic> DC3000 in Arabidopsis [ <xref ref-type="bibr" rid="B72"> 72 </xref> ] and <italic> Fusarium virguliforme </italic> , the causal agent of sudden death syndrome, in soybean [ <xref ref-type="bibr" rid="B73"> 73 </xref> ]. Our high-throughput sequencing results showed that miR169 was down-regulated under Al stress (Table <xref ref-type="table" rid="T3"> 3 </xref> ). These findings suggest that the conserved miRNAs might take part in cross adaptation to regulate plant tolerance to biotic and abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						The target transcripts for which cleavage fragments were found only in the Al-treated library were involved in stress responses and included the mRNAs for NB-ARC domain proteins, LRR domain <span class="trait">protein,</span> NAM <span class="trait">protein,</span> Myb family transcription factors, auxin response factor, and cation transporting ATPase. This finding might be the result of the temporal expression of miRNAs and their target genes. The NB-ARC, LRR and <span class="gene">TIR</span> domains have been identified in plant <span class="trait">resistance</span> proteins involved in pathogen recognition and subsequent activation of innate immune responses [ <xref ref-type="bibr" rid="B79"> 79 </xref> - <xref ref-type="bibr" rid="B81"> 81 </xref> ]. The cleavage of TIR-NBS-LRR type transcripts were reported in <italic> M. truncatula </italic> under mercury stress [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. We found that NB-ARC domain, LRR domain and <span class="gene">TIR</span> domain transcripts (encoding the three domain TIR-NBS-LRR <span class="trait">protein)</span> were cleaved by the gso-miR1510 family of miRNAs. Interestingly, the cleavage fragments of NB-ARC domain, LRR <span class="trait">protein</span> were found only in the Al-treated library (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ), suggesting that the TIR-NBS-LRR <span class="trait">protein</span> might take part in a regulating pathway involved in the recognition of biotic and abiotic stresses. The cleavage of the TIR-NBS-LRR <span class="trait">protein</span> under Al stress might result in the disruption of the immune system which might increase the susceptibility of the plant to other stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						The perception and transmission of stress signals mediated by hormones could play important roles in Al tolerance in plants. Recent studies revealed that the inhibition of root elongation, a typical symptom of Al toxicity, might be was caused by disruption of auxin distribution in roots [ <xref ref-type="bibr" rid="B82"> 82 </xref> , <xref ref-type="bibr" rid="B83"> 83 </xref> ]. It was reported that cleavage of the auxin response factor (ARF) by miR160 regulated the development of the root cap. In Arabidopsis, when miR160 was over-expressed, three ARF genes were barely detectable and the root length of the transgenic seedlings was reduced [ <xref ref-type="bibr" rid="B84"> 84 </xref> ]. In the present study, seven genes belonging to the ARF family were found to be cleaved by miR160 only in the Al-treated library (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). Previously, <span class="gene">NAC1</span> was found to mediate auxin signaling to promote lateral root development [ <xref ref-type="bibr" rid="B85"> 85 </xref> , <xref ref-type="bibr" rid="B86"> 86 </xref> ]. Transgenic plants expressing antisense <span class="gene">NAC1</span> cDNA show a reduction of lateral roots. The no apical meristem <span class="trait">protein</span> is a member of the NAC domain superfamily. In the Al-treated library, no apical meristem was detected to be cleaved by PN-miR1514 (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ) and the expression of PN-miR1514 was found to be up-regulated under Al stress. These results suggest that the cleavage of ARF transcripts by miR160 and NAM transcripts by miR1514 might be involved in the plant&#x02019;s response to auxin which may regulate the inhibition of root development under Al toxicity. In <italic> G. max </italic> , ABA accumulation was induced by Al treatment, and the tendency of ABA to be distributed in the Al-exposed root was shown by a split-root experiment [ <xref ref-type="bibr" rid="B87"> 87 </xref> ]. Research revealed that the Myb transcription factors might take part in the response to ABA accumulation, and the cleavage of Myb transcription factors mediated by miR159 has been reported in Arabidopsis under <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="B88"> 88 </xref> ]. We also found that the cleavage of Myb transcription factors was mediated by miR159 in <italic> G. soja </italic> in this study (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). Together these results indicated that the cleavage of hormone-related genes might affect the response of wild soybean to hormones potentially affecting the plant&#x02019;s tolerance to Al stress and causing metabolic dysfunction.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The distribution of the transcription regulators cleaved by known miRNAs. </bold> The percentage indicates proportion of the different transcription regulators out of the total number of transcription regulators for the known miRNAs. CCAAT, CCAAT-binding transcription factor; Myb, Myb family of transcription factors; SBP, <span class="gene">SBP</span> domain proteins; TCP, TCP family of transcription factors; SPT6, transcription elongation factor SPT6; WRC, growth regulating factor; AP2, <span class="gene">AP2</span> domain <span class="trait">protein;</span> ARF, auxin response factor.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cytosolic Malic Enzyme 1 (ME1) Mediates <span class="trait">High</span> Fat Diet-Induced Adiposity, Endocrine Profile, and Gastrointestinal Tract Proliferation-Associated Biomarkers in Male Mice</h2>
			<h3>PMID:23056418</h3>
				
					<h3>PMCID:3464285</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Al-Dwairi </surname>  <given-names> Ahmed </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pabona </surname>  <given-names> John Mark P. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Simmen </surname>  <given-names> Rosalia C. M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Simmen </surname>  <given-names> Frank A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						RNA was extracted from individual distal colons and jejunums, livers and retroperitoneal fat depots (n&#x0200a;=&#x0200a;5&#x02013;7 animals/group/tissue) with TRIZol reagent (Life Technologies, Grand Island, NY). One ug of total RNA was reverse-transcribed to cDNA using the iScript cDNA synthesis kit (Bio-Rad; Hercules, CA). Expression of target genes was assayed by qRT-PCR using Bio-Rad iTaq SYBR Green Supermix. Primers used in qRT-PCR ( <xref ref-type="table" rid="pone-0046716-t002"> Table 2 </xref> ) were obtained from Integrated DNA Technologies, Inc. (Coralville, IA). The primers used to quantify ME1 mRNA were designed to amplify both murine ME1 RNA splice variants. Target mRNA abundance in distal colon was normalized to a factor derived from the geometric mean of expression values for 18S rRNA, &#x003b2;-actin, cyclophilin A and TATA box binding <span class="trait">protein</span> (TBP), calculated using GeNorm <xref ref-type="bibr" rid="pone.0046716-Vandesompele1"> [31] </xref> . Target mRNA abundance in jejunum was normalized to a factor derived from the geometric mean of expression values for &#x003b2;-actin, <span class="gene">GAPDH</span> and TBP. Target mRNA abundance in liver and fat was normalized to a factor derived from the geometric mean of expression values for &#x003b2;-actin, cyclophilin A and TBP. Two-way ANOVA and Student&#x02019;s <italic> t </italic> test were used to compare variables between groups (SigmaPlot 11; Systat Software, Inc.; Chicago, IL).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physical Activity, Nutrition, and Dyslipidemia in Middle-Aged Women</h2>
			<h3>PMID:23113107</h3>
				
					<h3>PMCID:3481733</h3>
					<h4>期刊:Iranian Journal of Public Health</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Delavar </surname>  <given-names> MA </given-names>  </name>  <xref ref-type="aff" rid="af1-ijph-40-89">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lye </surname>  <given-names> MS </given-names>  </name>  <xref ref-type="aff" rid="af2-ijph-40-89">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="c1-ijph-40-89">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hassan </surname>  <given-names> STBS </given-names>  </name>  <xref ref-type="aff" rid="af2-ijph-40-89">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khor </surname>  <given-names> GL </given-names>  </name>  <xref ref-type="aff" rid="af3-ijph-40-89">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hanachi </surname>  <given-names> P </given-names>  </name>  <xref ref-type="aff" rid="af4-ijph-40-89">  <sup> 4 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A higher proportion of women had moderate and <span class="trait">high</span> physical activity (24.8%, 74.5%, respectively). Only nine women (0.6%) reported being inactive (low physical activity level). The mean total kilocalorie consumed per day was 2,966&#x000b1;1,080. <xref ref-type="table" rid="t4-ijph-40-89"> Table 4 </xref> shows the adjusted odds ratio (with 95% CI) for different levels of carbohydrate, <span class="trait">protein,</span> fat intake, and physical activity. After controlling for age, total physical activity, years of school, income and body mass index, total energy intake, <span class="trait">protein,</span> fat, carbohydrate, the adjusted OR for dyslipidemia in women with moderate <span class="trait">protein</span> intake was significantly higher than in women with <span class="trait">high</span> and low intake (OR=2.31; 95% <span class="gene">CI</span> =1.61, 3.30). No significant associations were found between dyslipidemia and carbohydrate, fat intake or physical activity. Spearman correlation coefficient has shown soy bean <span class="trait">protein</span> was inversely associated with cholesterol ratio (rho= &#x02212;0.18, <italic> P </italic> &#x0003c; 0.001) and total physical activity did not correlate with total cholesterol: HDL cholesterol ratio.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Patterns and evolution of ACGT repeat <italic> cis </italic> -element landscape across four plant genomes</h2>
			<h3>PMID:23530833</h3>
				
					<h3>PMCID:3622567</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Mehrotra </surname>  <given-names> Rajesh </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> rmehrotra@pilanibits-pilani.ac.in </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Sethi </surname>  <given-names> Sachin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ssethi771@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A3">  <name>  <surname> Zutshi </surname>  <given-names> Ipshita </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ipshitazutshi@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Bhalothia </surname>  <given-names> Purva </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> purvalohan@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Mehrotra </surname>  <given-names> Sandhya </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sandhyamehrotrabits@gmail.com </email>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 3 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The frequency of occurrence of random 4&#x000a0;bp nucleotide sequences separated by varying spacer lengths in Arabidopsis, Soybean, Rice and Sorghum was determined, and compared within each species. A dot plot indicated that contrary to our hypothesis, ACGT elements are not drastically preferred over other elements across all species (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> ). In fact, ACGT elements usually lay around the mean of the frequencies of the control elements, with the exception of sorghum, for which the frequency of ACGT elements was lower. For Arabidopsis and Rice, spacer lengths 7 and 24 were present in higher frequencies, being surpassed only by ATGC, <span class="gene">GATC</span> and TGCA, which were predominantly <span class="trait">high</span> across all species (details in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> IDA </italic> -like gene expression in soybean and tomato leaf abscission and requirement for a diffusible stelar abscission signal</h2>
			<h3>PMID:23585923</h3>
				
					<h3>PMCID:3624929</h3>
					<h4>期刊:AoB Plants</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Tucker </surname>  <given-names> Mark L. </given-names>  </name>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Ronghui </given-names>  </name>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We identified in the soybean genomic sequence 12 <italic> IDA </italic> -like genes (henceforth referred to simply as <italic> IDA </italic> ). The soybean <italic> IDA </italic> genes were identified using a TBLASTN search (six-frame translation) of the <italic> G. max </italic> genomic sequence with the <italic> AtIDA </italic> (accession NP564941) <span class="trait">protein</span> sequence minus its N-terminal signal peptide. The 12 soybean <italic> IDA </italic> sequences were named <italic> <span class="gene">GmIDA</span> </italic> 1 through 6 with the letter a or b appended to denote highly similar paralogous genes. A similar approach was used to identify six <italic> IDA </italic> -like genes in bean ( <italic> Phaseolus vulgaris </italic> ) and five in tomato. The relatedness of the six <italic> Arabidopsis </italic> , 12 soybean, six bean and five tomato IDA proteins is displayed in the dendrogram shown in Fig.&#x000a0; <xref ref-type="fig" rid="PLS035F1"> 1 </xref> [nucleotide sequences available in <ext-link ext-link-type="uri" xlink:href="http://aobpla.oxfordjournals.org/lookup/suppl/doi:10.1093/aobpla/pls035/-/DC1"> Additional Information File AI1 </ext-link> ]. All the <italic> IDA </italic> genes include an uninterrupted open reading frame with no introns and encode a translation product with a predicted N-terminal signal peptide sequence. The amino acid sequence similarity between AtIDA and other IDA-like proteins minus the putative signal peptides ranges from 24 % with AtIDL3 to 53 % with <span class="gene">GmIDA1a.</span> All of the IDAs include a variable region immediately after the signal peptide, and all include the highly conserve EPIP domain ( <xref ref-type="bibr" rid="PLS035C25"> Stenvik <italic> et al. </italic> 2008 </xref> ) (Fig.&#x000a0; <xref ref-type="fig" rid="PLS035F2"> 2 </xref> ). <fig id="PLS035F1" position="float"> <label> Fig.&#x000a0;1 </label> <caption> <p> An unrooted dendrogram based on per cent amino acid identity among <italic> Arabidopsis </italic> , bean, soybean and tomato IDA and IDA-like <span class="trait">protein</span> sequences minus the N-terminal signal peptides.
					</p>
				
					<p>&nbsp;&nbsp;
						Eukaryotic genomes commonly include many <italic> <span class="gene">RLK</span> </italic> genes ( <xref ref-type="bibr" rid="PLS035C22"> Shiu and Bleecker 2001 </xref> ; <xref ref-type="bibr" rid="PLS035C14"> Morris and Walker 2003 </xref> ). Assuming that the translated open reading frames for the soybean genes most similar to the <italic> Arabidopsis </italic> HAESA (HAE) and HAESA-like2 (HSL2) peptides retained IDA-ligand-binding specificity and kinase signalling, we performed a TBLASTN search of the soybean genomic sequence with the AtHAESA and AtHAESA-like peptide sequences. Thirteen soybean <italic> HAESA </italic> -like genes (henceforth referred to simply as soybean <italic> HAE </italic> ) were identified. The relatedness of the 13 soybean HAE and the <italic> Arabidopsis </italic> HAE and HSL proteins is displayed in the dendrogram shown in Fig.&#x000a0; <xref ref-type="fig" rid="PLS035F3"> 3 </xref> [nucleotide sequences provided as <ext-link ext-link-type="uri" xlink:href="http://aobpla.oxfordjournals.org/lookup/suppl/doi:10.1093/aobpla/pls035/-/DC1"> Additional Information File AI1 </ext-link> ]. The soybean sequences identified and selected for study ranged from 82 % amino acid similarity between <span class="gene">GmHAE1a</span> and AtHSL1, and 49 % amino acid sequence similarity between <span class="gene">GmHAE7a</span> and AtHSL2. Within the <span class="trait">protein</span> sequences, the most highly conserved region was in the <span class="trait">protein</span> kinase domains in the C-terminal third of the <span class="trait">protein.</span> <fig id="PLS035F3" position="float"> <label> Fig.&#x000a0;3 </label> <caption> <p> An unrooted dendrogram based on per cent amino acid identity among <italic> Arabidopsis </italic> and soybean HAE and HSL <span class="trait">protein</span> sequences.
					</p>
				
					<p>&nbsp;&nbsp;
						We monitored the expression of 12 soybean <italic> IDA </italic> s and for most of the <italic> IDA </italic> genes expression increased late in abscission. However, by far the greatest and earliest increase occurred for <italic> <span class="gene">GmIDA2a</span> </italic> and <italic> <span class="gene">GmIDA2b</span> </italic> , each increasing by more than 100-fold in the AZ after 24 h of ethylene treatment or 96 h in air. The increase in <italic> <span class="gene">GmIDA2a</span> </italic> and <italic> <span class="gene">GmIDA2b</span> </italic> transcript was relatively specific to the AZ but somewhat less AZ specific than the genes for the CWMPs (Figs&#x000a0; <xref ref-type="fig" rid="PLS035F4"> 4 </xref> and <xref ref-type="fig" rid="PLS035F5"> 5 </xref> ). However, if you add up expression levels for all the <italic> IDA </italic> genes and assume that the secreted peptides are functionally redundant, there is a fairly <span class="trait">high</span> level of total <italic> IDA </italic> transcript in both AZ and petioles that could lead to synthesis and secretion of an active IDA peptide. However, the amount of IDA peptides secreted into the apoplast may not be directly proportional to the amount of <italic> IDA </italic> transcript in the AZ or petiole, and the receptor for these ligands may not be distributed in the same pattern as the secretion of the IDA peptide. Of interest in this regard is the finding that overexpression of <italic> IDA </italic> in <italic> Arabidopsis </italic> activated vestigial abscission of leaves ( <xref ref-type="bibr" rid="PLS035C24"> Stenvik <italic> et al. </italic> 2006 </xref> ). This suggests that a receptor for IDA is present in the vestigial leaf AZ that is capable of inducing an AZ-specific cell separation response. Thus, it may be the receptor that determines AZ-specific signalling.
					</p>
				
					<p>&nbsp;&nbsp;
						As a part of this project we examined the expression of <italic> IDA </italic> and <italic> HAE </italic> in ethylene-treated leaves. Unexpectedly, expression of several soybean <italic> IDA </italic> s increased late in senescing (yellowing) leaves exposed to ethylene for 96 h. Most notably, <italic> <span class="gene">GmIDA1a</span> </italic> and <italic> <span class="gene">GmIDA1b</span> </italic> transcripts accumulated to very <span class="trait">high</span> levels in senescent leaves (Figs&#x000a0; <xref ref-type="fig" rid="PLS035F4"> 4 </xref> and <xref ref-type="fig" rid="PLS035F5"> 5 </xref> ). Interestingly, there was a corresponding large increase in the expression of <italic> <span class="gene">GmHAE2a</span> </italic> in senescent soybean leaves (Figs&#x000a0; <xref ref-type="fig" rid="PLS035F4"> 4 </xref> and <xref ref-type="fig" rid="PLS035F5"> 5 </xref> ). What role IDA signalling might play in senescence is unknown and how much of the gene expression changes observed in the petiole and AZ of the soybean and tomato explants can be linked to a general senescence response is also unknown.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of Peanut Diacylglycerol Acyltransferase 2 in <italic> Escherichia coli </italic></h2>
			<h3>PMID:23593473</h3>
				
					<h3>PMCID:3623910</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Peng </surname>  <given-names> Zhenying </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Lan </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Lianqun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Bin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Gao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bi </surname>  <given-names> Yuping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 3 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Recombinant proteins can be used as alternatives to endogenous ones to study their structures and functions or to make high-titer antibodies that recognize them. Because most DGATs are integral membrane proteins, they are difficult to express and purify in heterologous expression systems <xref ref-type="bibr" rid="pone.0061363-Hobbs1"> [17] </xref> , <xref ref-type="bibr" rid="pone.0061363-Weselake1"> [18] </xref> ; thus far, only limited success has been achieved in this area <xref ref-type="bibr" rid="pone.0061363-Weselake1"> [18] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0061363-Cao1"> [20] </xref> . Weselake <italic> et al </italic> . expressed the N-terminal 116 amino acid residues of <italic> Brassica napus </italic> (oilseed rape) <span class="gene">DGAT1</span> as a His-tagged <span class="trait">protein</span> in <italic> Escherichia coli </italic> <xref ref-type="bibr" rid="pone.0061363-Banilas1"> [16] </xref> . The resulting recombinant BnDGAT1 <sub> (1&#x02013;116) </sub> His <sub> 6 </sub> interacted with long chain acyl-CoA and displayed enhanced affinity for erucoyl (22:1cis <sup> &#x00394;13 </sup> )-CoA over oleoyl (18:1cis <sup> &#x00394;9 </sup> )-CoA <xref ref-type="bibr" rid="pone.0061363-Weselake1"> [18] </xref> . Subsequently, the amino terminal 95 residues of mouse <span class="gene">DGAT1</span> were expressed in <italic> E. coli </italic> with similar results <xref ref-type="bibr" rid="pone.0061363-Siloto1"> [19] </xref> . Encouragingly, full-length <span class="gene">DGAT1</span> expression from the tung tree ( <italic> Vernicia fordii </italic> ) in <italic> E. coli </italic> has been achieved <xref ref-type="bibr" rid="pone.0061363-Cao1"> [20] </xref> . In this case, the recombinant <span class="trait">protein</span> was mostly targeted to the membranes, and there were insoluble fractions with extensive degradation from the carboxyl end as well as association with other proteins, lipids, and membranes.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arachis hypogaea </italic> (peanut, Fabaceae) is one of the most economically-important oil-producing crops, so the fact that peanut DGATs have not been extensively studied is surprising. Saha <italic> et al </italic> . identified a soluble DGAT3 from immature peanut cotyledons and expressed its full length in <italic> E. coli </italic> , where the recombinant <span class="trait">protein</span> had <span class="trait">high</span> levels of <span class="gene">DGAT</span> activity but no wax ester synthase activity <xref ref-type="bibr" rid="pone.0061363-Saha1"> [5] </xref> ; this is the only published report on peanut DGATs thus far. Here, we identified two isozymes of DGAT2 in peanut and expressed both of them as full-length recombinant proteins in <italic> E. coli. </italic> This is the first time that a full-length recombinant DGAT2 <span class="trait">protein</span> from peanut has been successfully expressed in <italic> E. coli </italic> , and the first evaluation of its effects on the growth and FA content of the transformed <italic> E. coli </italic> strains studied.
					</p>
				
					<p>&nbsp;&nbsp;
						We evaluated the time course of expression of the fusion proteins by inducing cells transformed with the empty vector or with a GST-tagged fusion <span class="trait">protein</span> with 1.0 mM IPTG at 25&#x000b0;C or 37&#x000b0;C. Cells were collected after 0, 2, 4, and 6 h by centrifuging (5,000&#x000d7;g, 10 min), and the pellets were sonicated for 10 min in homogenization buffer (3&#x02013;4 mL/g wet cells) containing 20 mM Tris-HCl, pH 7.4, 200 mM NaCl, 10 mM &#x003b2;-mercaptoethanol, 1 mM EDTA, 0.5 mM phenylmethylsulfonyl fluoride, and 1&#x02236;100&#x02013;1&#x02236;500 dilution of protease inhibitor cocktail (#P8340, Sigma&#x02013;Aldrich, St. Louis, MO, USA). Cell debris, inclusion bodies, and <span class="trait">protein</span> aggregates in the homogenate were removed by two consecutive centrifugations (2,000&#x000d7;g, 10 min, followed by 10,000&#x000d7;g, 10 min). Samples were treated with 0.5 M NaOH and <span class="trait">protein</span> concentrations were determined with the Bradford method <span class="trait">(Protein</span> Assay Dye Reagent Concentrate, Bio-Rad, Hercules, CA, USA). The supernatant and the pellet were evaluated by SDS-PAGE (10%) and visualized by staining with Coomassie brilliant blue. Furthermore, the expression locations of the recombinant proteins were determined by purifying cytoplasmic and inclusion body fractions using a B-PER <span class="gene">GST</span> Fusion <span class="trait">Protein</span> Purification Kit (Thermo Scientific, Rockford, IL, USA). <span class="trait">Protein</span> expression was verified by a western blot analysis using an anti-GST tag monoclonal antibody (EarthOx, San Francisco, CA, USA).
					</p>
				
					<p>&nbsp;&nbsp;
						M: <span class="trait">Protein</span> molecular weight marker. (A) Lanes 1, 3: AhDGAT2a&#x02013;GST and AhDGAT2b&#x02013;GST extracted from the cytoplasmic fraction; Lanes 2, 4: AhDGAT2a&#x02013;GST and AhDGAT2b&#x02013;GST extracted from inclusion bodies. (B) Western blot analysis of the AhDGAT2&#x02013;GST fusion proteins using anti-GST tag monoclonal antibody. Lane 1: Wild-type <italic> E. coli </italic> Rosetta (DE3) strain; Lane 2, <span class="gene">GST</span> expression from the empty-vector transformed strain; Lanes 3, 5: AhDGAT2a&#x02013;GST and AhDGAT2b&#x02013;GST from the cytoplasmic fraction; Lanes 4, 6: AhDGAT2a&#x02013;GST and AhDGAT2b&#x02013;GST from inclusion bodies.
					</p>
				
					<p>&nbsp;&nbsp;
						Increasing energy costs and environmental concerns have compelled the production of sustainable renewable fuels and chemicals. In recent years, biofuels have received significant attention and investment <xref ref-type="bibr" rid="pone.0061363-Lu1"> [27] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0061363-Zhang1"> [35] </xref> . Prokaryotic expression systems, particularly <italic> E. coli </italic> systems, remain an effective way of producing large quantities of a variety of fusion proteins. For example, Lu et al. introduced four distinct genetic changes into the <italic> E. coli </italic> genome to achieve a <span class="trait">high</span> level of FA production (2.5 g/L) <xref ref-type="bibr" rid="pone.0061363-Lu1"> [27] </xref> . Zhang et al. studied the effects of overexpression of acyl-ACP thioesterase genes from four different plant species using <italic> E. coli </italic> that lacked FA production, and found that the transformed <italic> E. coli </italic> strains synthesized approximately 0.2 g/L of free FAs <xref ref-type="bibr" rid="pone.0061363-Zhang1"> [35] </xref> . Jeon et al. cloned and overexpressed five genes (acetyl-CoA carboxylase A, acetyl-CoA carboxylase B, acetyl-CoA carboxylase C, malonyl-CoA-[acyl-carrier-protein] transacylase, and acyl-ACP thioesterase) in <italic> E. coli </italic> MG1655 and found that the FA (C16) levels from the recombinant strains were 1.23&#x02013;2.41 times higher than those from the wild type <xref ref-type="bibr" rid="pone.0061363-Jeon1"> [34] </xref> . Furthermore, Liu et al. engineered an <italic> E. coli </italic> cell line that produced 4.5 g/L/day FAs <xref ref-type="bibr" rid="pone.0061363-Liu1"> [33] </xref> . In addition, when a soluble DGAT3 from peanut was introduced into <italic> E. coli </italic> , the transformants showed <span class="trait">high</span> levels of <span class="gene">DGAT</span> activity and formation of labeled TAG <xref ref-type="bibr" rid="pone.0061363-Saha1"> [5] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, the transformed <italic> E. coli </italic> strains changed their general morphology from short rods to filamentous structures ( <xref ref-type="fig" rid="pone-0061363-g007"> Figure 7 </xref> ), a change similar to bacteria encountering nutritional stress <xref ref-type="bibr" rid="pone.0061363-Young1"> [39] </xref> . These changes occurred gradually over time (data not shown) and were not caused by IPTG addition alone, because IPTG induction over 6 h caused no such morphological changes in the WT strain ( <xref ref-type="fig" rid="pone-0061363-g007"> Figure 7A&#x02013;B </xref> ). Furthermore, when fresh medium with or without IPTG was added to the 6-h induced cultures, the cells neither increased nor decreased in length when IPTG was included in the fresh medium, but they gradually shortened over several hours when IPTG was absent from the fresh medium (data not shown), suggesting that nutritional stress did not cause the changes in morphology. Perhaps the rapid accumulation of over-expressed proteins or FAs altered the cell shape. To some extent, cell size was related to the size of exogenous proteins produced. For example, <span class="gene">GST</span> transformants that produced &#x0223c;27 kDa proteins had cell sizes about 1.5 times those of their uninduced counterparts ( <xref ref-type="table" rid="pone-0061363-t002"> Table 2 </xref> ). In contrast, AhDGAT2a&#x02013;GST and AhDGAT2b&#x02013;GST transformants (expressing 64 kDa AhDGAT2&#x02013;GST fusion proteins) increased their sizes by about 2.4&#x02013;2.5 times that of their uninduced counterparts ( <xref ref-type="fig" rid="pone-0061363-g007"> Figure 7E&#x02013;F, G&#x02013;H </xref> ). Apparently, the larger the size of the exogenous <span class="trait">protein,</span> the larger the transformed cell will become.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, the identification of these genes will help spur the creation of transgenic peanut germplasms with <span class="trait">high</span> oil content or other special characteristics. The <span class="gene">DGAT</span> antibody will be important for identifying <span class="gene">DGAT</span> <span class="trait">protein</span> expression levels in transgenic plants. Finally, the differences in enzyme activity <italic> in vitro </italic> will assist in identifying important motif sites or single nucleotide polymorphisms that can be used in molecular marker-assisted breeding.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Decreased IgA <sup> + </sup> B Cells Population and IgA, IgG, IgM Contents of the Cecal Tonsil Induced by Dietary <span class="trait">High</span> Fluorine in Broilers</h2>
			<h3>PMID:23644827</h3>
				
					<h3>PMCID:3709348</h3>
					<h4>期刊:International Journal of Environmental Research and Public Health</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Juan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Hengmin </given-names>  </name>  <xref rid="c1-ijerph-10-01775" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Xi </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fang </surname>  <given-names> Jing </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zuo </surname>  <given-names> Zhicai </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deng </surname>  <given-names> Junliang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Hesong </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Bangyuan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deng </surname>  <given-names> Yuanxin </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Kangping </given-names>  </name>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						IgA <sup> + </sup> B cells in the lamina propria of the cecal tonsil at 42 days of age. ( <bold> a </bold> ) The cecal tonsil in control group (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> b </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group I. The quantities of IgA <sup> + </sup> B cells were not significantly decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> c </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group II. The quantities of IgA <sup> + </sup> B cells were decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> d </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group III. The quantities IgA <sup> + </sup> B cells were significantly decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m).
					</p>
				
					<p>&nbsp;&nbsp;
						IgA <sup> + </sup> B cells in the diffuse lymphoid tissues and lymphoid follicles of cecal tonsil at 42 days of age. ( <bold> a </bold> ) The cecal tonsil in control group (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> b </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group I. The quantities of IgA <sup> + </sup> B cells were not significantly decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> c </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group II. The quantities of IgA <sup> + </sup> B cells were decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m); ( <bold> d </bold> ) The cecal tonsil in <span class="trait">high</span> fluorine group III. The quantities IgA <sup> + </sup> B cells were significantly decreased (400&#x000d7;, <span class="gene">bar</span> = 50 &#x003bc;m).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcription profile of soybean-root-knot nematode interaction reveals a key role of phythormones in the <span class="trait">resistance</span> reaction</h2>
			<h3>PMID:23663436</h3>
				
					<h3>PMCID:3701510</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Beneventi </surname>  <given-names> Magda Aparecida </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> magbeneventi@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> da Silva </surname>  <given-names> Orzenil Bonfim </given-names>  <suffix> Jr </suffix>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> orzenil.silva@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> de S&#x000e1; </surname>  <given-names> Maria Eug&#x000ea;nia Lisei </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> msa@cenargen.embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Firmino </surname>  <given-names> Alexandre Augusto Pereira </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> alexfirm@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> de Amorim </surname>  <given-names> Regina Maria Santos </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> rmsamorim@yahoo.com.br </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Albuquerque </surname>  <given-names> &#x000c9;rika Val&#x000e9;ria Saliba </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> erika.albuquerque@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> da Silva </surname>  <given-names> Maria Cristina Mattar </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> cristina.mattar@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> da Silva </surname>  <given-names> Joseane Padilha </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> joseane.padilha@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Campos </surname>  <given-names> Magn&#x000f3;lia de Ara&#x000fa;jo </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> magnoliaacp@ufcg.edu.br </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Lopes </surname>  <given-names> Marcus Jos&#x000e9; Concei&#x000e7;&#x000e3;o </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> marcusjclbr@yahoo.com.br </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Togawa </surname>  <given-names> Roberto Coiti </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> roberto.togawa@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Pappas </surname>  <given-names> Georgios Joanis </given-names>  <suffix> Jr </suffix>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> gpappas@unb.br </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A13">  <name>  <surname> Grossi&#x02013;de&#x02013;Sa </surname>  <given-names> Maria Fatima </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> fatima.grossi@embrapa.br </email>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 5 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We also find domains that suggest a role in plant defense or stress responses that are over-expressed in the inoculated root sample. These include gene sequences containing the Plant Lipid Transfer Proteins (PLTPs) domain, which are also referred as plant Pathogenesis-Related (PR-14) proteins, BURP domain proteins and Fasciclin-Like Arabinogalactan-proteins (FLA) (PF02469). It was also detected significant induction in genes encoding proteins that are responsive to the hormone auxin: containing Auxin-inducible, <span class="gene">GH3</span> and AUX_IAA domains and proteins regulated by the hormone plant gibberellins. Over-expression of genes coding for the APETALA2 transcription factors family, EREBP (Ethylene-responsive element binding <span class="trait">protein)</span> and Leafy Petiole (LEP) were also noted (Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						Apparently, <span class="trait">high</span> levels of auxins, mainly in the form of free indole-3-acetic acid (IAA), can arise in the interplay between IAA biosynthesis and conjugation in the maintenance of IAA gradients [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. In the inoculated sample we found an over-expressed gene ( <italic> GH3.6 </italic> ) which the encoded enzyme functions in the synthesis of IAA-amino acid conjugates. Accordingly, the over-expression of <italic> <span class="gene">GH3</span> </italic> might lead to the accumulation of IAA-Asp conjugates which is a potential mechanism for plant cells to cope with presence of auxin excess in its bioactive form [ <xref ref-type="bibr" rid="B47"> 47 </xref> ]. This <italic> <span class="gene">GH3</span> </italic> -mediated auxin homeostasis also was proposed as an essential constituent of auxin actions that regulate stress adaptation responses in plants [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. Furthermore, our study also found a number of up-regulated genes, encoding family domains associated with induction by the plant hormone auxin, such as AUX/IAA and small auxin-up RNA (SAUR).
					</p>
				
					<p>&nbsp;&nbsp;
						Our study also showed that transcriptional activity of glutathione S-transferases (GST) genes increased in the inoculated sample, as well as many PR <span class="trait">protein</span> encoding genes, mainly group 5 (PR-5; Pfam: PF00314) and LTPs (PR-14; Pfam: PF00234), and a phenylalanine ammonia-lyase (PAL) gene (Pfam:PF00221; EC:4.3.1.24). <span class="gene">GST</span> is known to function in hydroperoxide detoxification through reduction of peroxides with the help of GSH and PR. <span class="gene">PAL</span> proteins were identified as inhibitors of H <sub> 2 </sub> O <sub> 2 </sub> production [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. Taken together, these observations may suggest that the host plants have the ability to adapt very well to low activities of both catalase and ascorbate peroxidase by induction of other defense systems, probably by signaled induction.
					</p>
				
					<p>&nbsp;&nbsp;
						One interesting <span class="trait">protein</span> found in our study regards great similarity to <span class="gene">Glyma05g03490,</span> a gene that encodes a <span class="trait">protein</span> containing GRASS domain, and exhibits <span class="trait">high</span> expression level in the inoculated sample. Similarity analysis using tblastn program indicates that the <span class="gene">Glyma05g03490</span> gene shares suggestive homology (55% identity and 70% similarity) with the rice locus <span class="gene">Os06g0127800</span> gene encoded <span class="trait">protein.</span> Further investigation accounts that the <span class="gene">Glyma05g03490</span> encoded <span class="trait">protein</span> shares GRAS domains such as LHRI, VHIID, LHR, PFYRE and SAW motifs with the <span class="gene">Os06g0127800</span> encoded <span class="trait">protein,</span> but both of them lack DELLA and TVHYNP motifs found conserved in DELLA proteins. Recently, the locus <span class="gene">Os06g0127800</span> was amplified from <italic> dwarf 62(d62) </italic> rice mutant inducing the dwarf phenotype with increased gene expression levels of both GA biosynthetic and deactivating genes, OsGA20ox2 and OsGA2ox3, respectively [ <xref ref-type="bibr" rid="B68"> 68 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification, characterization, molecular cloning and extracellular production of a phospholipase A <sub> 1 </sub> from <italic> Streptomyces albidoflavus </italic> NA297</h2>
			<h3>PMID:23772365</h3>
				
					<h3>PMCID:3678127</h3>
					<h4>期刊:FEBS Open Bio</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sugimori </surname>  <given-names> Daisuke </given-names>  </name>  <email> sugimori@sss.fukushima-u.ac.jp </email>  <xref rid="cor0001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kano </surname>  <given-names> Kota </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matsumoto </surname>  <given-names> Yusaku </given-names>  </name>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Approximately 1500 strains were isolated from various soil samples of Fukushima, Japan using HV medium [ <xref rid="bib18" ref-type="bibr"> 18 </xref> ]. Among the isolates, strains exhibiting a clear halo on <span class="gene">TSB</span> plates containing lecithin were selected [ <xref rid="bib19" ref-type="bibr"> 19 </xref> ]. Strain NA297 exhibiting <span class="trait">high</span> enzyme activity and good reproducible production of enzymes was selected and was identified as a strain related to <italic> S. albidoflavus </italic> based on morphological and the 16S rDNA sequence (DDBJ database under accession number <ext-link ext-link-type="uri" xlink:href="ncbi-n:AB738935" id="interref3"> AB738935 </ext-link> ). <italic> S. albidoflavus </italic> NA297 was deposited as NITE BP-1014 in the NITE Patent Microorganisms Depositary (NPMD) (Chiba, Japan).
					</p>
				
					<p>&nbsp;&nbsp;
						The purified <span class="trait">protein</span> was resolved by SDS&#x02013;PAGE and then electroblotted onto a PVDF membrane (Immobilon-P <sup> SQ </sup> transfer membrane, Millipore Co., Billerica, MA). The PVDF membrane was stained with Coomassie brilliant blue R-250 (CBB), and the transferred 28-kDa band was excised and subjected to N-terminal amino acid sequence analysis (Procise 494 HT <span class="trait">Protein</span> Sequencing System; Applied Biosystems, Foster City, CA, USA). For internal terminal amino-acid sequencing, an SDS&#x02013;PAGE gel was stained using CBB, the 28-kDa band was excised and then decolorized with 30% (vol/vol) acetonitrile containing 25 mM (NH <sub> 4 </sub> ) <sub> 2 </sub> HCO <sub> 3 </sub> . The in-gel digestion was performed by the method described by Shevchenko et al. [ <xref rid="bib22" ref-type="bibr"> 22 </xref> ]. Briefly, the excised 28-kDa band was digested with Trypsin (Sequencing Grade Modified Trypsin, Promega Corporation, Madison, WI, USA) for 45 h at 4 &#x000b0;C. The fragments were analyzed with a nanoACQUITY UPLC Xevo QTof <span class="gene">MS</span> system (Waters Corp., Milford, MA, USA). The sample solution was transferred to an autosampler vial. One &#x003bc;l was chromatographed on a nanoAcquity column BEH130C18 (75 &#x003bc;m &#x000d7; 150 mm) using a nanoAcquity UPLC system (Waters Corp., Milford, MA, USA). The column was heated to 40 &#x000b0;C, and ultrapure water containing 0.1% (vol/vol) formic acid (A) and 0.1% (vol/vol) formic acid/acetonitrile (B) were employed. A typical 101-min sample run consisted of a gradient from 99% to 50% solvent A over 95 min, from 50% to 10% solvent A over 1 min, and maintaining 10% solvent A for 4 min followed by an increase of solvent A up to 99% over 1 min. A flow rate of 0.3 &#x003bc;L/min was used and the effluent was sprayed using Pre-cut PicoTip Emitter (Waters, 360 &#x003bc;m OD &#x000d7; 20 &#x003bc;m ID; 10 &#x003bc;m tip; 6.35 cm length). The UPLC system was interfaced by electrospray ionization (ESI <sup> + </sup> ) to a Waters Xevo QTOF-MS operated in data dependent acquisition (DDA) mode with positive ionization. The capillary and sampling cone voltages were set to 3000 and 24 V, respectively. Source and desolvation temperatures were set to 90 and 200 &#x000b0;C, respectively, and the cone, desolvation and nanoflow gas flows were set to 30, 800 and 0.3 l/h, respectively. The collision argon gas energy was optimized to monitor the product ions of interest. To maintain mass accuracy, [Glu <sup> 1 </sup> ]-Fibrinopeptide B human (Sigma-Aldrich Co. LLC., MO, USA) as a lock mass ( <italic> m </italic> / <italic> z </italic> 785.00 for positive ion mode) at a concentration of 500 fmol/&#x003bc;l in 0.1% (vol/vol) formic acid/50% (vol/vol) acetonitrile was used, and injected at a rate of 0.5 &#x003bc;l/min. Accurate mass LC-MS/MS DDA data were acquired in the centroid mode from 50 to 1990 <italic> m </italic> / <italic> z </italic> . Data acquisition was achieved with MassLynx version 4.1 SCN 712 (Waters Corp., Milford, MA, USA). De novo sequencing was performed with the ProteinLynx Global SERVER (Waters Corp., Milford, MA, USA).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Antiosteoclastic Activity of Milk Thistle Extract after Ovariectomy to Suppress Estrogen Deficiency-Induced Osteoporosis</h2>
			<h3>PMID:23781510</h3>
				
					<h3>PMCID:3678416</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Jung-Lye </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 30 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Effects of milk thistle extract (MTE) and soybean extract (SBE) on alkaline phosphatase (ALP) activity of osteoblasts ((a) and (b)) and tartrate-resistant acid phosphatase (TRAP) activity of osteoclasts ((c) and (d)). After differentiation of MC3T3-E1 cells to osteoblasts and of RAW264.7 cells to osteoclasts, MTE ((a) and (c)) and SBE ((b) and (d)) were treated individually or in combination at the doses indicated. To detect activity of ALP in MC3T3-E1 cell media on day 6, a colorimetric enzyme assay was employed. Absorbance was measured at <italic> &#x003bb; </italic> = 405&#x02009;nm and compared with <italic> p- </italic> nitrophenol (PNP) standard (mean &#x000b1; SEM, <italic> n </italic> = 4). The ALP enzyme activity was expressed as nmol PNP/min/mg <span class="trait">protein.</span> For the measurement of TRAP activity, RAW 264.7 cells were exposed to 50&#x02009;ng/mL RANKL for 5 days. After 5-day culture, TRAP-positive activity was determined at <italic> &#x003bb; </italic> = 405&#x02009;nm (mean &#x000b1; SEM, <italic> n </italic> = 3). Values in <span class="gene">bar</span> graphs not sharing a letter indicate significant difference at <italic> P </italic> &#x0003c; 0.05.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Elucidation of miRNAs-Mediated Responses to Low Nitrogen Stress by Deep Sequencing of Two Soybean Genotypes</h2>
			<h3>PMID:23861762</h3>
				
					<h3>PMCID:3704600</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yejian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chanjuan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qinnan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sha </surname>  <given-names> Aihua </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Longping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 5 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies showed that a single miRNA precursor could produce two or more distinct miRNAs <xref ref-type="bibr" rid="pone.0067423-Li2"> [53] </xref> . In this study, we also found that some pre-miRNAs, such as pre-miR159a, pre-miR319a, pre-miR394a, and pre-miR2118a could generate distinct abundant sequences on their 5&#x02032; arm or 3&#x02032; arms, the positions of which didn&#x02019;t overlap and the sequence frequencies were <span class="trait">high</span> enough. Most importantly, we found most of their corresponding miRNA*s sequences, indicating they were likely to be the products of <span class="gene">DCL1</span> processing. These distinct abundant sequences from the same precursor could be annotated as different miRNA variants of the same precursor, amplifying analyzed known miRNA variants to 362 in 16 libraries ( <xref ref-type="table" rid="pone-0067423-t002"> Table 2 </xref> , <xref ref-type="supplementary-material" rid="pone.0067423.s004"> Table S2 </xref> , <xref ref-type="supplementary-material" rid="pone.0067423.s001"> Figure S1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To elucidate the functions of low N-responsive miRNAs, target genes with functional annotations were analyzed. We found that the predicted targets were involved in a broad range of plant physiological and biochemical processes, including regulation of <span class="trait">protein</span> degradation (26S proteasome regulatory complex, Apoptotic ATPase, Vesicle coat complex COPI, Mitochondrial import inner membrane translocase etc), cellular Transport (multicopper oxidase, Amino acid transporters), hormone signaling pathways (AUX/IAA family, Auxin response factor, gibberellin 2-oxidase), Carbohydrate metabolism(Long-chain acyl-CoA synthetases, acetyl-CoA carboxylase carboxyl transferase subunit, pyruvate dehydrogenase <span class="gene">E1</span> component), nucleic acid metabolism(Asparaginyl-tRNA synthetase, CCAAT-binding factor, tRNA delta(2)-isopentenylpyrophosphate transferase, RNA recognition motif, Chromatin assembly factor-I). Interestingly, some target genes are also important transcription factors (Myb superfamily, TCP family transcription factor, GRAS family transcription factor, <span class="gene">AP2</span> domain-containing transcription factor).
					</p>
				
					<p>&nbsp;&nbsp;
						By comparing the target genes of differentially expressed miRNAs with previously discovered genes involved in low N stress response in other plants, we found that these responsive genes were indeed involved in various metabolic and regulatory pathways. For example, Peng found 21 genes involved in <span class="trait">protein</span> degradation through autophagy and ubiquitin-proteosome pathways were up-regulated by N limitation <xref ref-type="bibr" rid="pone.0067423-Peng1"> [75] </xref> . Some of our predicted target genes were determined to play roles in <span class="trait">protein</span> degradation, including <span class="gene">Glyma07g31580</span> (target of gma-miR156b/6f-5p), as well as <span class="gene">Glyma05g20930</span> and <span class="gene">Glyma06g18790</span> (target of gma-miR396b&#x0223c;g-5p), which were predicted to encode separately E3 ubiquitin ligase and Cathepsin L1. Metabolism of N is closely related to carbon metabolism, because nitrate assimilation and biosynthesis of nitrogenous macromolecules require abundant energy, reducing equivalents and organic carbon intermediates provided by carbon metabolism. Some studies found that genes involved in carbon metabolism were responsive to low N stress <xref ref-type="bibr" rid="pone.0067423-Peng1"> [75] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0067423-Bi2"> [76] </xref> . In the present study, we found that some targets of gma-miR159d-3p, gma-miR396b&#x0223c;g-5p, such as <span class="gene">Glyma05g23280,</span> <span class="gene">Glyma07g05550,</span> <span class="gene">Glyma16g02090,</span> <span class="gene">Glyma17g16750,</span> <span class="gene">Glyma19g44930,</span> <span class="gene">Glyma15g08010</span> and <span class="gene">Glyma19g01200</span> were related to carbon metabolism. Gene expression alterations caused by low N stress correspondingly activate signal transduction and gene transcription regulatory networks to coordinate all of these changes <xref ref-type="bibr" rid="pone.0067423-Lian1"> [73] </xref> , <xref ref-type="bibr" rid="pone.0067423-Peng1"> [75] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0067423-Bi2"> [76] </xref> . We observed that some target genes were transcription factors and participated in signal transduction, such as MYB, AP2, ARF, SPB, and zinc finger. Futhermore, MYB, ARF and <span class="gene">AP2</span> families have been known to be involved in plant stress responses <xref ref-type="bibr" rid="pone.0067423-Todd1"> [77] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0067423-Song3"> [79] </xref> . In addition, we noticed that gma-miR5372-5p showed distinct expression patterns between the shoots of the two genotypes under the short-term low N condition, and its target gene, <span class="gene">Glyma09g02600,</span> encodes peroxidase <span class="trait">protein</span> which could eliminate excess concentrations of reactive oxygen species produced under stress conditions. This phenomenon also was observed by Kulcheski et al who found that MIR-Seq11 had different expression behavior between the two contrasting soybean genotypes under the <span class="trait">drought</span> stress, and MIR-Seq11 was also predicted to target peroxidase <span class="trait">protein</span> <xref ref-type="bibr" rid="pone.0067423-Kulcheski2"> [80] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Meal Induces Intestinal Inflammation in Zebrafish Larvae</h2>
			<h3>PMID:23894568</h3>
				
					<h3>PMCID:3720926</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hedrera </surname>  <given-names> Manuel I. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Galdames </surname>  <given-names> Jorge A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jimenez-Reyes </surname>  <given-names> Maria F. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reyes </surname>  <given-names> Ariel E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Avenda&#x000f1;o-Herrera </surname>  <given-names> Ruben </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Romero </surname>  <given-names> Jaime </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Feij&#x000f3;o </surname>  <given-names> Carmen G. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Two compounds of soybean meal (50SBM diet) (C), soy <span class="trait">protein</span> (43SP diet) (D) and soy saponin (50SS diet) (E), were analyzed to determine their immunological effect. (G) An increase of neutrophils in the intestine (white arrow) was observed after 4 days of feeding in the soy saponin group (and soybean meal), but not in the soy <span class="trait">protein</span> group. (F) Scheme representing the intestine territory analyzed (red rectangle). At least 15 larvae per condition were analyzed in three different experiments. Statistical analysis was performed using one-way ANOVA. ns (no significance) *p value &#x0003c; 0.05; ***p value &#x0003c; 0.001. Scale <span class="gene">bar</span> = 200&#x000b5;m.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High conjugated linoleic acid enriched ghee (clarified butter) increases the antioxidant and antiatherogenic potency in female Wistar rats</h2>
			<h3>PMID:23923985</h3>
				
					<h3>PMCID:3766171</h3>
					<h4>期刊:Lipids in Health and Disease</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Chinnadurai </surname>  <given-names> Kathirvelan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kadhirc@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Kanwal </surname>  <given-names> Harpreet Kaur </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> harpreetmitthu@gmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A3">  <name>  <surname> Tyagi </surname>  <given-names> Amrish Kumar </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> amrishtyagi1963@yahoo.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Stanton </surname>  <given-names> Catherine </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> Catherine.Stanton@teagasc.ie </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Ross </surname>  <given-names> Paul </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> Paul.Ross@teagasc.ie </email>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 7 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						At the age of 55 <sup> th </sup> day, all the animals were sacrificed and CAT, SOD and <span class="gene">GST</span> activities were analyzed in liver. Liver CAT activity was the highest in <span class="trait">high</span> CLA ghee fed group followed by low CLA fed group and lowest in soybean oil fed group (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). CAT activity increased to 52.24, 37.82 and 20% in <span class="trait">high</span> CLA, low CLA ghee and soybean oil fed group, respectively, from that of its day zero and variation among the groups was significant (P&#x02009;&#x0003c;&#x02009;0.05). Similarly, highest liver SOD activity was determined in <span class="trait">high</span> CLA ghee fed group followed by low CLA fed group and lowest in soybean oil fed group. SOD activity was increased by 227, 179 and 42% in <span class="trait">high</span> CLA, low CLA ghee and soybean oil group, respectively, from zero day and variation among the groups was significant (P&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						The detoxifying, phase II <span class="gene">GST</span> enzyme was the highest in <span class="trait">high</span> CLA ghee fed group followed by low CLA fed group and lowest in soybean oil fed group (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). Significant variations were observed among the different dietary groups (P&#x02009;&#x0003c;&#x02009;0.01). <span class="gene">GST</span> activity increased significantly in <span class="trait">high</span> CLA, low CLA ghee and soybean oil fed group, respectively, compared to day zero. The extent of endogenous lipid peroxidation level of TBARS (Thiobarbituric acid reactive substances) was the highest in soybean oil fed group followed by low CLA fed group and lowest in <span class="trait">high</span> CLA ghee fed group (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). TBARS levels significantly changed at day 35 (P&#x02009;&#x0003c;&#x02009;0.01) as compared to controls (day zero) in different dietary groups.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, <span class="gene">GST</span> activity was the highest in <span class="trait">high</span> CLA ghee followed by low CLA ghee fed groups rather than in soybean oil fed group. Higher <span class="gene">GST</span> activity in response to <span class="trait">high</span> CLA are believed to modify the antioxidant system in rats [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The study shows that <span class="trait">high</span> CLA enriched ghee increases the antioxidant enzyme activities such as CAT, SOD and <span class="gene">GST</span> activities; which concludes its antioxidant nature. Feeding of <span class="trait">high</span> CLA enriched ghee also lead to a decrease in cholesterol, triglyceride and <span class="trait">high</span> density lipoproteins and inturn, decreased the low density lipoproteins, which finally lead to the reduction in atherogenic index, which proves its antiatherogenic property. The present study indicates that <span class="trait">high</span> CLA enriched ghee (clarified butter) has antioxidant and antiatherogenic activities suggesting that ghee can be as an important food component for decreasing the risk of cardiovascular diseases.
					</p>
				
					<p>&nbsp;&nbsp;
						All the rats were sacrificed after the experiment and entire liver was excised. Liver homogenate was prepared as described by Cohen et al. [ <xref ref-type="bibr" rid="B31"> 31 </xref> ]. For catalase assay, liver homogenate was further processed as described by Cohen et al. [ <xref ref-type="bibr" rid="B31"> 31 </xref> ] and the catalase activity and Superoxide dismutase were assayed. Glutathione-S-Transferase (GST) was determined by the method of Habig et al. [ <xref ref-type="bibr" rid="B32"> 32 </xref> ]. One international <span class="gene">GST</span> unit is expressed as mmoles of cDNB conjugated/minute/mg <span class="trait">protein.</span> For detection of lipid peroxides (malondialdehyde) in liver homogenate method of Uchiyama and Mihara [ <xref ref-type="bibr" rid="B33"> 33 </xref> ] was followed. <span class="trait">Protein</span> was estimated according to the method of Bradford [ <xref ref-type="bibr" rid="B34"> 34 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The large soybean ( <italic> Glycine max </italic> ) WRKY TF family expanded by segmental duplication events and subsequent divergent selection among subgroups</h2>
			<h3>PMID:24088323</h3>
				
					<h3>PMCID:3850935</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Yin </surname>  <given-names> Guangjun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yinguangjundejia@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Xu </surname>  <given-names> Hongliang </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xuxxx812@umn.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Xiao </surname>  <given-names> Shuyang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> susanshaw@Yeah.net </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Qin </surname>  <given-names> Yajuan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qyj718897021@126.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Li </surname>  <given-names> Yaxuan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lyx1006@sohu.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Yan </surname>  <given-names> Yueming </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yanym2004@163.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A7">  <name>  <surname> Hu </surname>  <given-names> Yingkao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yingkaohu@yahoo.com </email>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 10 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In plants, the dicot model organism <italic> Arabidopsis </italic> is commonly used to predict the function of a gene in a newly or partially sequenced organism that has a close phylogenetic relationship to <italic> Arabidopsis </italic> , such as soybean. Moreover, there are at least 72 WRKY family members in <italic> Arabidopsis </italic> , and most of these genes have been extensively studied and reported to be involved in many physiological and biochemical processes [ <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B22"> 22 </xref> ]. With the aim of defining the soybean protein-containing WRKY domains, we downloaded the 72 known <italic> Arabidopsis </italic> WRKY <span class="trait">protein</span> sequences from the <italic> Arabidopsis </italic> transcription factor database (AtTFDB; <ext-link ext-link-type="uri" xlink:href="http://arabidopsis.med.ohio-state.edu/"> http://arabidopsis.med.ohio-state.edu/ </ext-link> ). In order to examine the structural features of each <italic> At </italic> WRKY, we performed a multiple sequence alignment (data not shown). Two members, <italic> At4g12020 </italic> and <italic> At4g30930 </italic> , were excluded from the analysis due to incomplete zinc fingers and the lack of WRKY domains. Therefore, 70 <italic> Arabidopsis </italic> WRKY <span class="trait">protein</span> sequences were used to BLAST the completed soybean genome sequences for genes that encode proteins containing the WRKY domain. The WRKY domain of each predicted <span class="trait">protein</span> was identified by searching against the SMART database. After manually removing overlapping genes, a total of 133 non-redundant genes in the soybean genome were identified as members of the WRKY family (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ). Among these members, annotation (predicted) of 23 proteins revealed that they have two complete WRKY domains each, which all belong to group I. In addition, physical positions of WRKY genes were obtained from the Phytozome database, and were used to map these genes onto their corresponding soybean chromosomes (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). Results showed that WRKY genes in soybean could be mapped on all chromosomes, from chromosome 1 to 20. Chromosome 8 had the highest density of WRKY genes with 11 members, while in chromosomes 10, 11, 12, and 20, no more than three WRKY genes could be found, respectively. Examination of the location of each WRKY gene revealed that all <italic> Gm </italic> WRKY genes, except for <italic> <span class="gene">Glyma02g39870</span> </italic> , <italic> <span class="gene">Glyma03g25770</span> </italic> , <italic> <span class="gene">Glyma04g05700</span> </italic> , <italic> <span class="gene">Glyma04g12830</span> </italic> , <italic> <span class="gene">Glyma05g20710</span> </italic> , <italic> <span class="gene">Glyma06g06530</span> </italic> , <italic> <span class="gene">Glyma06g13090</span> </italic> , <italic> <span class="gene">Glyma06g27440</span> </italic> , <italic> <span class="gene">Glyma06g47880</span> </italic> , <italic> <span class="gene">Glyma08g01430</span> </italic> , <italic> <span class="gene">Glyma09g09400</span> </italic> , <italic> <span class="gene">Glyma09g24080</span> </italic> , <italic> <span class="gene">Glyma09g37930</span> </italic> , <italic> <span class="gene">Glyma11g29720</span> </italic> , <italic> <span class="gene">Glyma12g23950</span> </italic> , <italic> <span class="gene">Glyma13g34280</span> </italic> , <italic> <span class="gene">Glyma14g17330</span> </italic> , <italic> <span class="gene">Glyma14g36430</span> </italic> , <italic> <span class="gene">Glyma14g37960</span> </italic> , <italic> <span class="gene">Glyma15g20990</span> </italic> , <italic> <span class="gene">Glyma16g05880</span> </italic> , <italic> <span class="gene">Glyma17g29190</span> </italic> , <italic> <span class="gene">Glyma18g39970</span> </italic> , <italic> <span class="gene">Glyma18g49140</span> </italic> , <italic> <span class="gene">Glyma19g02440</span> </italic> , <italic> <span class="gene">Glyma19g26400</span> </italic> , and <italic> <span class="gene">Glyma20g03410</span> </italic> , originated from segmental duplications (102 of 133) or tandem duplications (18 of 133) (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). The 27 genes mentioned above might have been produced by retrotransposition instead.
					</p>
				
					<p>&nbsp;&nbsp;
						The phylogenetic relationship of <italic> Gm </italic> WRKY proteins was examined by multiple sequence alignment of their WRKY domains, which span across approximately 60 amino acids (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> ). A comparison with soybean WRKY domains and several homologous <italic> Arabidopsis </italic> proteins resulted in a separation of the different groups and subgroups. For each group or subgroup, one <italic> Arabidopsis </italic> <span class="trait">protein</span> was selected randomly, which included <italic> At2g04880C </italic> , with only one C-terminal WRKY domain, <italic> At4g26440N </italic> , with only one N-terminal WRKY domain, <italic> <span class="gene">At1g80840</span> </italic> , <italic> <span class="gene">At1g18860</span> </italic> , <italic> <span class="gene">At1g69310</span> </italic> , <italic> At2g30590 </italic> , <italic> <span class="gene">At1g29280</span> </italic> , and <italic> At2g46400 </italic> . As shown in Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> , the sequences of soybean WRKY were found to be highly conserved.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to better separate the groups and examine the evolutionary relationships of the WRKY family in soybean and <italic> Arabidopsis </italic> , an unrooted phylogenetic tree was constructed from alignments of their domain <span class="trait">protein</span> sequences, which resulted in the formation of three distinct clusters: group I, group II, and group III (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ). WRKY proteins of <italic> Arabidopsis </italic> and soybean were present in all clusters. This classification was consistent with results of Rushton et al. [ <xref ref-type="bibr" rid="B20"> 20 </xref> ], who suggested that WRKY domains could be classified into three large groups corresponding to groups I, II, and III of <italic> Arabidopsis </italic> . Notably, <italic> At </italic> WRKY members were more similar to those in the same class in divergent species than they were to other WRKY proteins in the same species. In order to examine the syntenic relationship of the WRKY gene family between the genomes of soybean and <italic> Arabidopsis </italic> , each WRKY gene within the family in <italic> Arabidopsis </italic> was searched in the PGDD ( <ext-link ext-link-type="uri" xlink:href="http://chibba.agtec.uga.edu/duplication/"> http://chibba.agtec.uga.edu/duplication/ </ext-link> ) (data no shown). During the course of this analysis we found that synteny was relatively well conserved between soybean and <italic> Arabidopsis </italic> proteins. For example, in subgroup II a (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ), several <italic> Gm </italic> WRKYs ( <italic> <span class="gene">Glyma06g06530</span> </italic> , <italic> <span class="gene">Glyma07g02630</span> </italic> , <italic> <span class="gene">Glyma08g23380</span> <span class="gene">Glyma13g44730</span> </italic> , and <italic> <span class="gene">Glyma15g00570</span> </italic> ) located on different chromosomes are orthologs of a same <italic> At </italic> WRKY gene ( <italic> <span class="gene">At1g80840</span> </italic> ). Additionally, it is worth noting that the structure and phylogenetic tree of the <italic> Gm </italic> WRKY domain clearly indicated that group II proteins could be divided into five distinct subgroups (II a-e).
					</p>
				
					<p>&nbsp;&nbsp;
						Since the transcriptome sequencing of soybean was completed, the availability of the soybean gene expression atlas facilitates additional studies on the basic biology of soybean [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. The recently developed RNA-Seq web-based tools, which include gene expression data across multiple tissues and organs, allow for characterization and comparisons of the gene transcriptome atlas in soybean. Consequently, distinct transcript abundance patterns are readily identifiable in the RNA-Seq atlas data set for all 133 <italic> Gm </italic> WRKYs. Similar to other genes that encode transcription factors, many of the <span class="gene">GmWRKYs</span> exhibited low transcript abundance levels, as determined by the RNA-Seq atlas analysis. Furthermore, we observed that most of the genes had very broad expression spectra. However, six <italic> Gm </italic> WRKYs, including <italic> <span class="gene">Glyma04g05700</span> </italic> , <italic> <span class="gene">Glyma04g39650</span> </italic> , <italic> <span class="gene">Glyma04g40120</span> </italic> , <italic> <span class="gene">Glyma08g01430</span> </italic> , <italic> <span class="gene">Glyma01g05050</span> </italic> , and <italic> <span class="gene">Glyma14g11440</span> </italic> , lacked expression information, which possibly indicated that these were pseudogenes or were expressed only at specific developmental stages or under special conditions. We observed that accumulation of WRKY gene transcripts was associated with different tissues, and expression patterns differed among each WRKY gene member (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> ). In soybean, 33.1% (44 of 133) of the analyzed WRKYs were constitutively expressed in all of the seven tissues tested, which suggested that <italic> Gm </italic> WRKYs play regulatory roles at multiple developmental stages. By contrast, most <italic> Gm </italic> WRKYs showed preferential expression. RNA-Seq atlas data revealed that the majority (92 of 133; 69.2%) of <italic> Gm </italic> WRKYs exhibited transcript abundance profiles with marked peaks in only a single tissue. This result suggests that these WRKY proteins function as tissue-specific regulators and are limited to discrete cells or organs. Approximately 45 of these 133 (33.8%) <italic> Gm </italic> WRKYs showed the highest transcript accumulation level in root tissue, 20 (15.0%) showed the highest transcript accumulation in flower tissue, 13 (9.8%) showed the highest transcript accumulation level in nodule tissue, and surprisingly, only one showed the highest transcript accumulation level in seed tissue. The wide expression of these genes suggests that WRKY genes from soybean are involved in the development of all organs or tissues under normal conditions. In addition to groups of genes that exhibited similar transcript abundance profiles but were relatively phylogenetically distinct, several phylogenetic clades shared the same transcript abundance profile to a large extent. For example, in subgroup II b, most of the <italic> Gm </italic> WRKYs were preferentially expressed in root tissue. Interestingly, in <italic> Arabidopsis </italic> , according to the experimental results of Dong et al. [ <xref ref-type="bibr" rid="B45"> 45 </xref> ], more than half of the members in subgroup II b showed similar preferential expression in the root tissue under normal conditions, which indicated the conserved functional role of subgroup II b in root development between the two species. The expression of members of group I in soybean was detectable in flower tissue, which suggested their conserved roles in flower formation. Members of group I also showed similar expression patterns in nodule and root development. Furthermore, <italic> Gm </italic> WRKYs with <span class="trait">high</span> sequence similarity and shared expression profiles represent good candidates for evaluation of gene functions in soybean. The transcriptome atlas indicted that differential expression was extended to all groups or subgroups of the soybean WRKY gene family, which was further verified by the promoter analysis. Because transcription factors bind to corresponding transcription factor binding sites (TFBSs) upstream of genes of interest, profiles of <italic> cis </italic> -acting elements may thus provide useful information related to the regulatory mechanism of gene expression. A computational tool, PlantCARE [ <xref ref-type="bibr" rid="B46"> 46 </xref> ], was adopted to identify putative TFBSs in the 1500-bp DNA sequence upstream of the translation initiation codon of WRKY genes in soybean. Four types of <italic> cis </italic> -acting elements were found to be significantly abundant in the promoter region of <italic> Gm </italic> WRKY genes (Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> ). The first type of <italic> cis </italic> -acting element enriched in the promoter region is the light responsive elements, such as G-box [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ], GAG-motif [ <xref ref-type="bibr" rid="B49"> 49 </xref> ], Box I [ <xref ref-type="bibr" rid="B50"> 50 </xref> ], and Box 4 [ <xref ref-type="bibr" rid="B51"> 51 </xref> ], etc. The G-box element appears to be more abundant in subgroup II a, in which each member contains at least two copies. In addition, the mean number of G-box copies was 3.625 in subgroup II a, which is higher than that in other subgroups or in the whole WRKY gene family. This result indicates that the G-box element seems to be enriched in subgroup II a. All but seven (94.7%; 126 of 133, Group I; 2, Group II b; 2, Group II c; 1, Group II d; 1, and <italic> <span class="gene">Glyma14g37960</span> </italic> ; 1) have at least one Box 4 element copy. Its mean number of copies (3.744) in the whole WRKY gene family was apparently higher than that of other types of <italic> cis </italic> -acting elements except for TATA-box, CAAT-box, and unnamed-4. This result suggests that the Box 4 element tends to be enriched in the soybean WRKY gene family. As one part of a conserved DNA module involved in light responsiveness, previous studies showed that the Box 4 element is frequently found in promoter regions of different genes from various species [ <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. However, it is noteworthy that the Box 4 element was found in <span class="trait">high</span> frequency in the soybean WRKY gene family, which suggests that the Box 4 element may be important for light-controlled transcriptional activity [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. Plant hormone responsive elements, such as ABRE [ <xref ref-type="bibr" rid="B54"> 54 </xref> ], P-box [ <xref ref-type="bibr" rid="B55"> 55 </xref> ], as well as the TCA-element [ <xref ref-type="bibr" rid="B56"> 56 </xref> ], constitute the second class. ABRE (51.9%; 69 of 133) appears to be one of the most abundant hormone-related <italic> cis </italic> -acting element in soybean, suggesting that abscisic acid (ABA) regulates the expression of some <italic> Gm </italic> WRKYs, whereas such elements were rarely detected in Group II d (20.0%; 3 of 15). By contrast, the salicylic acid responsive TCA-element was frequently found in groups or subgroups. These observations suggest that <italic> Gm </italic> WRKYs in different groups are likely to be significantly regulated by different types of hormones. The third most abundant <italic> cis </italic> -acting element class consisted of elements that respond to external environmental stresses. We observed that most of the <italic> Gm </italic> WRKYs examined appeared to contain MBS (72.2%; 96 of 133) [ <xref ref-type="bibr" rid="B57"> 57 </xref> ], heat shock element (HSE) (77.4%; 103 of 133) [ <xref ref-type="bibr" rid="B58"> 58 </xref> ], and TC-rich repeat elements (71.4%; 95 of 133) [ <xref ref-type="bibr" rid="B59"> 59 </xref> ]. MBS is an element involved in <span class="trait">drought</span> induction, and HSE is also enriched in the promoter. With a few exceptions, <italic> Gm </italic> WRKYs contain more than two copies of this element. Circadian elements, which are involved in circadian control [ <xref ref-type="bibr" rid="B60"> 60 </xref> ], is the fourth type of <italic> cis </italic> -acting element that was abundantly found in the promoter regions of soybean WRKY genes. PlantCARE identified 98 (73.7%; 98 of 133) <italic> Gm </italic> WRKY genes containing circadian elements, which may be responsible for its distinct diurnal expression pattern. The presence of a diversity of <italic> cis </italic> -acting elements in the upstream regions of <italic> Gm </italic> WRKYs indicates that <italic> Gm </italic> WRKYs may function in a relatively wide range of activities.
					</p>
				
					<p>&nbsp;&nbsp;
						Gene duplication events are important for gene family evolution, because duplicated genes provide the raw materials for the generation of new genes, which in turn facilitate the generation of new functions. Three principal evolutionary patterns were attributed to gene duplications: segmental duplication, tandem duplication, and transposition events such as retroposition and replicative transposition [ <xref ref-type="bibr" rid="B71"> 71 </xref> ]. Among these, segmental duplication occurs most frequently in plants because most plants are diploidized polyploids and retain numerous duplicated chromosomal blocks within their genomes [ <xref ref-type="bibr" rid="B72"> 72 </xref> ]. Previous studies reported several rounds of whole-genome duplication (WGD) in both the <italic> Arabidopsis </italic> and rice genomes [ <xref ref-type="bibr" rid="B73"> 73 </xref> , <xref ref-type="bibr" rid="B74"> 74 </xref> ]. The occurrence of large-scale gene duplication events was also demonstrated in soybean [ <xref ref-type="bibr" rid="B43"> 43 </xref> ]. For this analysis, we focused on the tandem and segmental duplication modes. Tandem duplications were characterized as multiple members of one family occurring within the same intergenic region or in neighboring intergenic regions, where genes were clustered together with a maximum of 10 extra genes between them [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. We searched for contiguous WRKY genes in both the sharing region and neighboring regions. Eighteen of the 133 genes (13.5%) in this family were found to be located as tandem repeats in soybean (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ), indicating that tandem duplications contributed to the expansion of this family. We also tested the hypothesis that segmental duplication events played a large role in the evolution of the WRKY gene family in soybean. For each WRKY gene, we tallied the number of flanking protein-coding genes with a best non-self match to a protein-coding gene neighboring its paralog. According to our results, 76.7% (102 of 133) of genes showed <span class="trait">high</span> conservation, indicating that these WRKY genes were formed through segmental duplication in soybean (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). Intriguingly, comparison of the 102 segmental duplicated genes in our study to the results of Du et al. [ <xref ref-type="bibr" rid="B75"> 75 </xref> ] suggested that 91 (89.2%; 91 of 102) WRKY genes originated from WGDs, and the duplication status of the remaining 11 (10.7%; 11 of 102) WRKY genes, including <italic> <span class="gene">Glyma01g05050</span> </italic> , <italic> <span class="gene">Glyma01g43420</span> </italic> , <italic> <span class="gene">Glyma02g15920</span> </italic> , <italic> <span class="gene">Glyma02g45530</span> </italic> , <italic> <span class="gene">Glyma03g00460</span> </italic> , <italic> <span class="gene">Glyma03g31630</span> </italic> , <italic> <span class="gene">Glyma08g15210</span> </italic> , <italic> <span class="gene">Glyma10g03820</span> </italic> , <italic> <span class="gene">Glyma13g38630</span> </italic> , <italic> <span class="gene">Glyma18g16170</span> </italic> , and <italic> <span class="gene">Glyma18g44030</span> </italic> , was singleton, which indicated that these segmental duplication genes may be derived from independent duplication events. These results indicated that most of the WRKY genes in soybean were retained after WGDs. Edger et al. [ <xref ref-type="bibr" rid="B76"> 76 </xref> ] stated that dosage-sensitive genes, including transcription factors, were preferentially retained following WGDs, which is compatible with the present study. We did not find evidence that other pairs of paralogous genes in soybean originated from segmental duplication. These results indicate that the soybean WRKY family arose mainly through segmental duplications.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Glyma14g37960</span> </italic> and <italic> <span class="gene">Glyma18g39970</span> </italic> were not assigned to any groups or subgroups. <italic> <span class="gene">Glyma14g37960</span> </italic> has one WRKY domain; however, it is phylogenetically closer to group I N. Thus, <italic> <span class="gene">Glyma14g37960</span> </italic> may have arisen from a two-domain WRKY <span class="trait">protein</span> that lost one of its WRKY domains during evolution, whereas in <italic> <span class="gene">Glyma18g39970</span> </italic> , a mutation in the sequence outside of the WRKY domain may have occurred before or after the domain loss.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Model building of the 3D structure of the soybean WRKY <span class="trait">protein</span> ( </bold> <bold> <italic> <span class="gene">Glyma13g00380</span> </italic> </bold> <bold> ) based on similarity to the </bold> <bold> <italic> At </italic> </bold> <bold> WRKY4-C domain <span class="trait">(Protein</span> Data Bank (PDB) code: 2lexA). </bold> The ensemble of the selected structures in stereo view <bold> (A) </bold> , <bold> (B) </bold> , <bold> (C) </bold> , and <bold> (D) </bold> positive selection sites detected by the branch-site model presented in group I, group II c, group II e, and group III, respectively. The sites with red color indicate amino acid residues under statistically significant (p&#x02009;&#x0003c;&#x02009;0.05) positive selection, as calculated by Bayes Empirical Bayes estimation methods. The presented region is Asp247&#x02013;Pro306, excluding the N-terminal region. The figure was produced using the Swiss-model and pyMOL programs.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Diets <span class="trait">High</span> in Heat-Treated Soybean Meal Reduce the Histamine-Induced Epithelial Response in the Colon of Weaned Piglets and Increase Epithelial Catabolism of Histamine</h2>
			<h3>PMID:24260435</h3>
				
					<h3>PMCID:3833947</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kr&#x000f6;ger </surname>  <given-names> Susan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pieper </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schwelberger </surname>  <given-names> Hubert G. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Villodre Tudela </surname>  <given-names> Carmen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Aschenbach </surname>  <given-names> J&#x000f6;rg R. </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Van Kessel </surname>  <given-names> Andrew G. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zentek </surname>  <given-names> J&#x000fc;rgen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 10 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The colon tissue was ground into fine powder over liquid nitrogen and 50 mg were used for extraction of the total RNA using the NucleoSpin&#x000ae; RNAII kit (Macherey-Nagel <span class="gene">GmbH</span> &#x00026; Co. KG, Germany). The expression of <italic> <span class="gene">DAO</span> </italic> and <italic> HMT </italic> , as indicators for the histamine metabolism, <italic> C-Kit </italic> and <italic> Fc&#x003b5;RI </italic> , as marker molecules expressed by mast cells and <italic> CFTR </italic> as a chloride transporting ion channel were determined, and 18S rRNA, 60S ribosomal <span class="trait">protein</span> L19 ( <italic> RPL19 </italic> ), hypoxanthine phosphoribosyltransferase I ( <italic> HPRTI </italic> ) and &#x003b2;-actin were used as housekeeping genes for data normalization. Primers ( <xref ref-type="supplementary-material" rid="pone.0080612.s002"> Table S2 </xref> ) were designed based on published sequences of the above mentioned target genes in the pig using the NCBI online primer design tool. Quantitative real-time PCR was performed using the one-step QRT-PCR master mix kit (Brilliant&#x000ae;II SYBR&#x000ae;Green, Agilent Technologies, USA) as described previously [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. The obtained Ct values were normalized and arbitrary values were calculated and used for statistical comparisons.
					</p>
				
					<p>&nbsp;&nbsp;
						For determination of <span class="gene">DAO</span> and HMT activities, frozen tissue samples (50 mg) were homogenised with a 5 mm steel bead in a TissueLyser II instrument (Qiagen, Germany) for 5 min at 30 Hz in 1 ml of 20 mM bis-Tris hydrochloride pH 7.0 containing 5 mM dithiothreitol and 1 x Complete Protease Inhibitor Cocktail (Roche, Germany). The homogenates were cleared by centrifugation for 10 min at 23,000 x <italic> g </italic> and the supernatants were used to determine <span class="gene">DAO</span> and HMT activities as well as <span class="trait">protein</span> concentrations.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene expression of <italic> HMT </italic> was higher ( <italic> P </italic> &#x0003c; 0.05) in the proximal colon tissue of weaning piglets fed with diets containing <span class="trait">high</span> amounts of fCP ( <xref ref-type="table" rid="pone-0080612-t003"> Table 3 </xref> ). In contrast, the <italic> <span class="gene">DAO</span> </italic> expression was not significantly altered by dietary treatments. The inclusion of fCHO did not affect the expression of the two enzymes.
					</p>
				
					<p>&nbsp;&nbsp;
						The activities of both enzymes, <span class="gene">DAO</span> and HMT, were increased ( <italic> P </italic> &#x0003c; 0.05) in the groups fed the <span class="trait">high</span> fCP diets, independent of the fCHO content ( <xref ref-type="table" rid="pone-0080612-t003"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Proceeding from increased luminal histamine levels of not defined origin, the present data suggest a counter-regulatory adaptation of the colonic tissue towards overall increased histamine concentrations in the <span class="trait">high</span> fCP groups. The activity of the histamine-degrading enzymes <span class="gene">DAO</span> and HMT was enhanced in these groups. The increased activity of HMT coincided with an upregulation of HMT gene transcription, while increased activity of <span class="gene">DAO</span> appeared to have a purely posttranscriptional nature. Both enzymes would limit the bioavailability of histamine upon entry from the lumen or release from mast cells [ <xref ref-type="bibr" rid="B22"> 22 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> ] with implications for the effects of histamine on the colonic tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> <span class="trait">protein</span> diets are often associated with changes in faecal consistency, possibly due to metabolites such as histamine or PGE <sub> 2 </sub> . Although they can induce the secretory response of the colonic epithelium in the pig, these effects were counter-regulated with diets containing <span class="trait">high</span> levels of fermentable <span class="trait">protein.</span> For histamine, this effect seemed to be mainly facilitated through a reduced efficacy of histamine to stimulate chloride secretion via the cAMP/CFTR pathway and an increased epithelial catabolism of histamine due to higher mRNA abundance of <italic> HMT </italic> and higher activities of <span class="gene">DAO</span> and HMT in the colonic tissue. In addition, these effects were independent of the inclusion of fCHO in the diet.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular and phenotypic characterization of <italic> Als1 </italic> and <italic> Als2 </italic> mutations conferring tolerance to acetolactate synthase herbicides in soybean</h2>
			<h3>PMID:24425499</h3>
				
					<h3>PMCID:4282486</h3>
					<h4>期刊:Pest Management Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Walter </surname>  <given-names> Kay L </given-names>  </name>  <xref ref-type="aff" rid="au1"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Strachan </surname>  <given-names> Stephen D </given-names>  </name>  <xref ref-type="aff" rid="au2"> b </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferry </surname>  <given-names> Nancy M </given-names>  </name>  <xref ref-type="aff" rid="au2"> b </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Albert </surname>  <given-names> Henrik H </given-names>  </name>  <xref ref-type="aff" rid="au1"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Castle </surname>  <given-names> Linda A </given-names>  </name>  <xref ref-type="aff" rid="au1"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sebastian </surname>  <given-names> Scott A </given-names>  </name>  <xref ref-type="aff" rid="au3"> c </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <aff id="au1">  <label> a </label>  <institution> DuPont Pioneer </institution>  <addr-line> Hayward, CA, USA </addr-line>  </aff>  <aff id="au2">  <label> b </label>  <institution> DuPont Crop Protection, DuPont Stine-Haskell Research Center </institution>  <addr-line> Newark, DE, USA </addr-line>  </aff>  <aff id="au3">  <label> c </label>  <institution> DuPont Pioneer </institution>  <addr-line> Johnston, IA, USA </addr-line>  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Sequencing results for the five ALS genes showed a mutation for two out of the five genes in Table <xref ref-type="table" rid="tbl1"> 1 </xref> , <span class="gene">Glyma04g37270.1</span> and <span class="gene">Glyma06g17790.1.</span> The mutant allele in <span class="gene">Glyma04g37270.1</span> is <italic> Als1 </italic> and the mutant allele in <span class="gene">Glyma06g17790.1</span> is proposed as <italic> Als2 </italic> . Alignments of the full-length wild-type <italic> als1 </italic> and <italic> als2 </italic> alleles with their <italic> Als1 </italic> and <italic> Als2 </italic> counterparts were made with the AlignX feature in Vector NTI software (Invitrogen). Results are shown in <xref ref-type="fig" rid="fig01"> Figs. 1 </xref> and <xref ref-type="fig" rid="fig02"> 2 </xref> . W4-4's <italic> Als1 </italic> allele resulted in a proline-to-serine substitution at position 178 of the full-length soybean <span class="trait">protein</span> (P197S in <italic> A. thaliana </italic> ), which confirms the mutation previously found in W20 (Falco SC, unpublished). W4-4's <italic> Als2 </italic> allele resulted in a tryptophan-to-leucine substitution at position 560 of the full-length soybean <span class="trait">protein</span> (W574L in <italic> A. thaliana </italic> ). These mutations are similar to the mutations in the highly herbicide-resistant ALS variant known as HRA, <xref rid="b21" ref-type="bibr"> 21 </xref> &#x02013; <xref rid="b23" ref-type="bibr"> 23 </xref> which, in soybean, has mutations P178A and W555L in the same ALS gene (P197A and W574L in <italic> A. thaliana </italic> ). Amino acid position 555 in <italic> Als1 </italic> is analogous to amino acid 560 in <italic> Als2 </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The first draft of the pigeonpea genome sequence</h2>
			<h3>PMID:24431589</h3>
				
					<h3>PMCID:3886394</h3>
					<h4>期刊:Journal of Plant Biochemistry and Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Singh </surname>  <given-names> Nagendra K. </given-names>  </name>  <address>  <phone> +91-11-25841186 </phone>  <email> nksingh@nrcpb.org </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gupta </surname>  <given-names> Deepak K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jayaswal </surname>  <given-names> Pawan K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mahato </surname>  <given-names> Ajay K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dutta </surname>  <given-names> Sutapa </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Sangeeta </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhutani </surname>  <given-names> Shefali </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dogra </surname>  <given-names> Vivek </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Bikram P. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumawat </surname>  <given-names> Giriraj </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pal </surname>  <given-names> Jitendra K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pandit </surname>  <given-names> Awadhesh </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Archana </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rawal </surname>  <given-names> Hukum </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Akhilesh </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rama Prashat </surname>  <given-names> G. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khare </surname>  <given-names> Ambika </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yadav </surname>  <given-names> Rekha </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Raje </surname>  <given-names> Ranjit S. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Mahendra N. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Datta </surname>  <given-names> Subhojit </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fakrudin </surname>  <given-names> Bashasab </given-names>  </name>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wanjari </surname>  <given-names> Keshav B. </given-names>  </name>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kansal </surname>  <given-names> Rekha </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dash </surname>  <given-names> Prasanta K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jain </surname>  <given-names> Pradeep K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhattacharya </surname>  <given-names> Ramcharan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gaikwad </surname>  <given-names> Kishor </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mohapatra </surname>  <given-names> Trilochan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Srinivasan </surname>  <given-names> R. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharma </surname>  <given-names> Tilak R. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> National Research Centre on Plant Biotechnology, Indian Agricultural Research Institute, New Delhi, 110 012 India  </aff>  <aff id="Aff2">  <label/> Division of Genetics, Indian Agricultural Research Institute, New Delhi, 110012 India  </aff>  <aff id="Aff3">  <label/> Institute of Agricultural Sciences, Banaras Hindu University, Varanasi, UP 221005 India  </aff>  <aff id="Aff4">  <label/> Indian Institute of Pulses Research, Kanpur, UP 208024 India  </aff>  <aff id="Aff5">  <label/> University of Agricultural Sciences, Dharwad, Karnataka 580005 India  </aff>  <aff id="Aff6">  <label/> Panjabrao Deshmukh Krishi Vidyapeeth, Krishinagar, Akola, Maharasthra 444 104 India  </aff>
				<br/> 
				年份：<day> 7 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">High</span> quality genomic DNA was isolated from the leaves of a single plant of variety &#x02018;Asha&#x02019; using CTAB method (Murray and Thompson <xref ref-type="bibr" rid="CR25"> 1980 </xref> ). Sequencing of 19 plates of whole genome shotgun libraries of short DNA fragments was carried out using GS-FLX Phase D chemistry, and 3 plates of paired end sequences from a library of 20 Kb long fragments of pigeonpea genomic DNA using GS-FLX Titanium chemistry (Margulies et al. <xref ref-type="bibr" rid="CR23"> 2005 </xref> ). Filtered <span class="trait">high</span> quality sequence reads were assembled using &#x0201c;Newbler <span class="gene">GS</span> <italic> De Novo </italic> assembler version 2.5.3&#x0201d; (Roeche Inc. Germany) with: Overlap minimum match length = 25&#x000a0;bp, Large genome = True, Number of CPU used = 0 (all), Exclude contigs of &#x0003c;500&#x000a0;bp. The <span class="gene">GS</span> Assembler is designed to compare all sequence reads in a pair wise fashion. Reads that overlap one another are joined into contigs. The consensus sequence for a contig is computed by taking an average of all aligned reads at a specific nucleotide position, the paired end reads were used for making scaffolds of sequence contigs. The large sequence contigs were quality checked and contaminating sequences were identified and removed. The quality check passed Fasta files containing 510,809,477&#x000a0;bp of pigeonpea genome sequence were further processed using command line software of NCBI to generate .sqn file ( <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/HTGS/tbl2asninfo.html"> http://www.ncbi.nlm.nih.gov/HTGS/tbl2asninfo.html </ext-link> ), which was submitted to GenBank as draft genome version 1 using Genomes Macro Send direct submission tool.
					</p>
				
					<p>&nbsp;&nbsp;
						The whole genome large sequence contigs were passed through FGENESH tool of MOLQUEST software ( <ext-link ext-link-type="uri" xlink:href="http://www.softberry.com"> www.softberry.com </ext-link> ) using <italic> Arabidopsis thaliana </italic> gene models as reference. From all predicted genes only those with size of &#x0003e;500&#x000a0;bp were taken for further analysis. The genes were BLAST searched against NCBI non-redundant database using optimized search parameters of <span class="gene">gap</span> opening penalty (G) = 4, <span class="gene">gap</span> extension penalty (E) = 1, mismatch score (q) = &#x02212;1, match score (r) = 1, word size (W) = 11 and e-value &#x0003c;e <sup> &#x02212;20 </sup> (Singh et al. <xref ref-type="bibr" rid="CR33"> 2004 </xref> ). Low complexity regions were included in the search. The BLAST search output was processed using BLAST Parser software ( <ext-link ext-link-type="uri" xlink:href="http://geneproject.altervista.org/"> http://geneproject.altervista.org/ </ext-link> ). All the hits having bit scores of &#x0003e;100 and e values of &#x0003c;e <sup> &#x02212;20 </sup> were tabulated in Microsoft Excel. Gene annotations were manually curated and categorized based on their functions. Details of pigeonpea transcriptome assemblies used for validation of predicted gene models is described earlier (Dutta et al. <xref ref-type="bibr" rid="CR5"> 2011 </xref> ). The predicted genes were manually curated with different keywords/phrases using auto filters to find R-like and defense response genes and categorize them into five main classes (Hulbert et al. <xref ref-type="bibr" rid="CR16"> 2001 </xref> ; The Rice Chromosomes 11 and 12 sequencing consortia <xref ref-type="bibr" rid="CR39"> 2005 </xref> ): (a) NBS-LRR (matching with NBS-LRR, but not with LZ-NBS-LRR and LRR, CC-NBS-LRR <italic> , Rp 1-d8 </italic> , <italic> Lr10 </italic> , <italic> Mla 1 </italic> and <span class="trait">rust</span> <span class="trait">resistance),</span> (b) LZ-NBS-LRR (matching with LZ-NBS-LRR, but not with NBS-LRR, CC-NBS-LRR, LRR and <italic> RPM1 </italic> ), (c) LRR-TM (matching with serine/threonine kinases and <italic> Cf2/Cf5 </italic> <span class="trait">resistance),</span> (d) miscellaneous category (matching with disease <span class="trait">resistance,</span> viral <span class="trait">resistance,</span> LRR, but not with NBS-LRR, CC-NBS-LRR, LZ-NBS-LRR), (e) defense response genes (matching with glucanases, chitinases and thaumatin like proteins). Similarly, genes for abiotic stress tolerance, lipid metabolism, sugar and starch biosynthesis, cellulose synthesis and transcription factors were also identified and categorized.
					</p>
				
					<p>&nbsp;&nbsp;
						Schmutz et al. ( <xref ref-type="bibr" rid="CR31"> 2010 </xref> ) identified 1,127 putative acyl lipid metabolism genes in the oilseed crop soybean. A similar analysis of genes for lipid metabolism in pigeonpea genome identified only 269 such genes, while soybean showed 536 genes (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> , Supplementary Table S <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Apart from the seed storage lipids these genes are involved in the metabolism of membrane lipids and various kinds of lipo-protein, glyco-lipid and mineral-lipid interactions. In contrast, pigeonpea genome has a higher number of 43 cellulose synthase genes as compared to only 37 genes in the soybean genome, which may be important for its woody plant architecture (Supplementary Table S <xref rid="MOESM4" ref-type="media"> 4 </xref> ). Pigeonpea genome has 108 genes for the synthesis of various kinds of sugars, sugar transporters and starches including granule bound starch synthase, soluble starch synthase, starch <span class="trait">branching</span> and debranching enzymes. These have important implications for the grain <span class="trait">yield</span> and biomass accumulation (Supplementary Table S <xref rid="MOESM5" ref-type="media"> 5 </xref> ). We identified 1,470 genes for different transcription factors and regulatory proteins in the pigeonpea genome (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> , Supplementary Table S <xref rid="MOESM6" ref-type="media"> 6 </xref> ). These transcription factors play pivotal roles in the developmental regulation of gene expression and response of plants to various biotic and abiotic stresses. Most predominant transcription factors in the pigeonpea genome were <span class="gene">AP2</span> domain-containing proteins, NAC domain containing proteins, WRKY transcription factors, Zinc finger proteins and MYB transcription factors.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary Proteins and Angiogenesis</h2>
			<h3>PMID:24445377</h3>
				
					<h3>PMCID:3916867</h3>
					<h4>期刊:Nutrients</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Medina </surname>  <given-names> Miguel &#x000c1;ngel </given-names>  </name>  <xref ref-type="aff" rid="af1-nutrients-06-00371"> 1 </xref>  <xref ref-type="aff" rid="af2-nutrients-06-00371"> 2 </xref>  <xref rid="c1-nutrients-06-00371" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Quesada </surname>  <given-names> Ana R. </given-names>  </name>  <xref ref-type="aff" rid="af1-nutrients-06-00371"> 1 </xref>  <xref ref-type="aff" rid="af2-nutrients-06-00371"> 2 </xref>  </contrib>
				<br/> 
				年份：<day> 09 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A third interesting bio-active <span class="trait">protein</span> present in milk is lactadherin. <xref ref-type="fig" rid="nutrients-06-00371-f004"> Figure 4 </xref> shows the crystal structure of the <span class="gene">C2</span> domain of bovine lactadherin. Also known as MFG-E8 (from Milk Fat Globule Epidermal growth factor-8), it was initially identified as one of the main components of milk fat globules, membrane-surrounded <span class="trait">protein-</span> and triacylglycerol-enriched lipid droplets that bud from the apical surface of mammary epithelia during lactation [ <xref rid="B34-nutrients-06-00371" ref-type="bibr"> 34 </xref> , <xref rid="B35-nutrients-06-00371" ref-type="bibr"> 35 </xref> ]. More recently, the acronym SED1 has been proposed as an alternative name for lactadherin, making reference to its nature of being a secreted <span class="trait">protein</span> with a multidomain structure, comprising an <italic> N </italic> -terminal domain with one (in human) or two (in mouse and rat) EGF-like domains (the second of which with an integrin-binding RGD motif) and a <italic> C </italic> -terminal domain with two discoidin/F5/8C domains that bind to anionic phospholipids and extracellular matrices. Behaving as an opsonin, lactadherin contributes to phagocytic removal of apoptotic cells in different tissue and facilitates a number of intercellular interactions, recently reviewed in [ <xref rid="B36-nutrients-06-00371" ref-type="bibr"> 36 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Schematic representation of the crystal structure of the <span class="gene">C2</span> domain of bovine lactadherin at a resolution of 1.67 &#x000c5;. The file with pdb ID 3bng was retrieved from <span class="trait">protein</span> data bank [ <xref rid="B16-nutrients-06-00371" ref-type="bibr"> 16 </xref> ] and visualized with Open Astex Viewer maintaining the <span class="trait">protein</span> on a black background and representing the <span class="trait">protein</span> surface in grey with a 30% of opacity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Substitution of dietary fish oil with plant oils is associated with shortened mid intestinal folds in Atlantic salmon ( <italic> Salmo salar </italic> )</h2>
			<h3>PMID:24606841</h3>
				
					<h3>PMCID:3973862</h3>
					<h4>期刊:BMC Veterinary Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Moldal </surname>  <given-names> Torfinn </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> torfinn.moldal@vetinst.no </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> L&#x000f8;kka </surname>  <given-names> Guro </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> guro.lokka@nvh.no </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wiik-Nielsen </surname>  <given-names> Jannicke </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jannicke.wiik@vetinst.no </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Austb&#x000f8; </surname>  <given-names> Lars </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> lars.austbo@nvh.no </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Torstensen </surname>  <given-names> Bente E </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> bente.torstensen@nifes.no </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Rosenlund </surname>  <given-names> Grethe </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> grethe.rosenlund@skretting.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Dale </surname>  <given-names> Ole Bendik </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ole.b.dale@vetinst.no </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Kaldhusdal </surname>  <given-names> Magne </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> magne.kaldhusdal@vetinst.no </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> Koppang </surname>  <given-names> Erling Olaf </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> erling.o.koppang@nvh.no </email>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several studies have addressed the effects of vegetable oils as lipid sources in the feed on Atlantic salmon intestinal absorption [ <xref ref-type="bibr" rid="B16"> 16 </xref> ], post-absorptive fates [ <xref ref-type="bibr" rid="B17"> 17 </xref> ], feed uptake, growth rate, metabolism and nutrient content of the fish filet [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> - <xref ref-type="bibr" rid="B22"> 22 </xref> ]. Whereas many studies have addressed the intestinal health of the fish when fish meal is replaced by different plant-derived proteins, and both soybean meal and pea <span class="trait">protein</span> concentrate have been shown to induce enteritis [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B24"> 24 </xref> ], there is a knowledge <span class="gene">gap</span> regarding the impact on intestinal health when fish oil is replaced by plant oils.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SFGD: a comprehensive platform for mining functional information from soybean transcriptome data and its use in identifying acyl-lipid metabolism pathways</h2>
			<h3>PMID:24712981</h3>
				
					<h3>PMCID:4051163</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Yu </surname>  <given-names> Juan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ake909@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Zhang </surname>  <given-names> Zhenhai </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhang.zhenhai552@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wei </surname>  <given-names> Jiangang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> weijiangang02@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Ling </surname>  <given-names> Yi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lingyi@cau.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Xu </surname>  <given-names> Wenying </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> x_wenying@yahoo.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Su </surname>  <given-names> Zhen </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhensu@cau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 31 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> L.) is a major leguminous seed crop providing an important source of oil, and ranks first in oil production among the major oil seed crops [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. In addition to its usage for human consumption, given its <span class="trait">high</span> content of essential fatty acids such as linoleic acid, soybean oil is an important renewable resource for chemical and biofuel production [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. Increasing the relative oil concentration in soybean seeds is many researchers&#x02019; goal, using breeding methods and biotechnological strategies [ <xref ref-type="bibr" rid="B3"> 3 </xref> - <xref ref-type="bibr" rid="B5"> 5 </xref> ]. The key governing genes of seed oil biosynthesis in higher plants are those involved in the synthesis pathway of triacylglycerol (TAG), plastid fatty acids, endomembrane lipids and the storage process, which constitute more than 274 genes in soybean [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. Several functional essays have shown that <span class="gene">DGAT1</span> and DGAT2 have roles in seed oil accumulation [ <xref ref-type="bibr" rid="B7"> 7 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ]. Mutant alleles of <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> were combined to create soybeans with the <span class="trait">high</span> oleic acid trait [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Some important transcription factors (TF) regulating lipid metabolism and seed maturation have been reported, including basic leucine zipper (bZIP) and DNA binding with one finger (DOF) TF families [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B12"> 12 </xref> ]. However, most of these genes&#x02019; functions are not clear and the biosynthesis of soybean oil has yet to be elucidated.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we selected the biosynthesis pathway of triacylglycerol (TAG) as an example to introduce the application of our database. Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> A shows the biosynthesis pathway of TAG and Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> B shows the tissue-specific expression of genes that encode the enzymes involved in this pathway&#x02013;all these data can be obtained from the SFGD platform. Additional file <xref ref-type="supplementary-material" rid="S4"> 4 </xref> : Figure S1 shows a screenshot of the web page with brief information (a coding gene, the corresponding probe set, <span class="gene">GmaAffx.88235.1.S1_at</span> and one microarray expression series, and ten deep sequence samples). Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> C shows the co-expression network of <span class="gene">WRI1</span> <span class="gene">(Glyma15g34770),</span> which has been reported to regulate the process of TAG biosynthesis by affecting the synthesis precursors in developing embryos [ <xref ref-type="bibr" rid="B77"> 77 </xref> , <xref ref-type="bibr" rid="B78"> 78 </xref> ]. Table&#x000a0; <xref ref-type="table" rid="T4"> 4 </xref> lists all soybean coding genes that appeared in Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> C and their annotations, and also gives these genes&#x02019; orthologs in <italic> Arabidopsis </italic> . Users can also do cis-element significance analysis, and a job ID is produced by the server&#x02013;the results are stored in the server for several weeks so that they can be traced back. The resulting web-page only shows motifs with P&#x02009;&#x0003c;&#x02009;0.05, and Table&#x000a0; <xref ref-type="table" rid="T5"> 5 </xref> is one example of the significance analysis results for all genes that appeared in Table&#x000a0; <xref ref-type="table" rid="T4"> 4 </xref> . Table&#x000a0; <xref ref-type="table" rid="T5"> 5 </xref> shows that some motifs may be closely related to seed development, such as &#x02018;ABREBNNAPA&#x02019; and &#x02018;ABRETAEM&#x02019;, with their annotations &#x02018;napA; storage <span class="trait">protein;</span> ABRE; napin; seed&#x02019; and &#x02018;ABA; ABRE; EMBP-1; seed&#x02019;, respectively.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a dominant gene in <italic> Medicago truncatula </italic> that restricts nodulation by <italic> Sinorhizobium meliloti </italic> strain Rm41</h2>
			<h3>PMID:24934080</h3>
				
					<h3>PMCID:4070093</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Liu </surname>  <given-names> Jinge </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jinge.liu@uky.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Yang </surname>  <given-names> Shengming </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> syang2@uky.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Zheng </surname>  <given-names> Qiaolin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qiaolinzheng@uky.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Zhu </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hzhu4@uky.edu </email>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We took advantage of the availability of the <italic> M. truncatula </italic> genome sequence [ <xref ref-type="bibr" rid="B46"> 46 </xref> ] to develop high-density SNP (single nucleotide polymorphism) markers for fine mapping of the <italic> Mt-NS1 </italic> locus. SNPs were genotyped either by converting to CAPS (cleaved amplified polymorphic sequences) markers or by direct sequencing. Phenotyping and genotyping a total of 3,900 F2 plants using the SNP markers allowed us to delimit the <italic> Mt-NS1 </italic> locus between SNP103 and SNP127 (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> B), which span ~50 kb based on the genome sequence of Jemalong A17 (Nod&#x02009;+&#x02009;Fix-). The 50-kb genomic sequence of Jemalong A17 contains at least six predicted genes based on the <italic> Medicago truncatula </italic> genome release version 4.0 ( <ext-link ext-link-type="uri" xlink:href="http://www.jcvi.org/medicago"> http://www.jcvi.org/medicago </ext-link> ) (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> C). These genes include Medtr8g028070, a ferric-chelate reductase-like <span class="trait">protein;</span> Medtr8g028080, an RNase T2 family <span class="trait">protein;</span> Medtr8g028085 and Medtr8g028100, two highly conserved proteins with unknown function; Medtr8g028105, an A/G-specific adenine DNA glycosylase-like <span class="trait">protein;</span> and Medtr8g028110, a leucine-rich repeat receptor-like kinase (LRR-RLK). In particular, the <span class="gene">LRR-RLK</span> (Medtr8g028110; here we call it as RLK-1) is structurally similar to the DMI2/SYMRK orthologs in legumes that are required for both rhizobial and mycorrhizal symbioses, which consists of an N-terminal signal peptide, an extracellular malectin-like domain comprising three leucine-rich repeats, a transmembrane domain, and an intracellular <span class="trait">protein</span> kinase domain [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. In addition, there exists a tandem duplication <italic> &#x02013; </italic> truncation of the RLK-1 gene (annotated as Medtr8g028115; here we call it as trRLK-2) (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> C). As described below, the presence of a large insertion/deletion polymorphism around the <italic> Mt-NS1 </italic> locus between F83005.5 and Jemalong A17 complicated the prediction of candidate genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Establishing a successful interaction between legumes and rhizobia requires signal recognition between the two symbiotic partners. Thus, the evolution of symbiosis specificity involves both rhizobial and host genes. From the bacterial side, specificity determinants include Nod factors, surface polysaccharides, and secreted proteins. However, we know relatively less from the host side. Perception of the Nod-factor signal is mediated by direct binding to the host Nod factor receptors (NFRs), which are plasma membrane-localized receptor kinases containing LysM motifs in their extracellular domains [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. The role of NFRs in regulating host specificity was demonstrated by transferring the <italic> L. japonicus </italic> versions of <italic> NFRs </italic> to <italic> M. truncatula </italic> , which enabled the transgenic <italic> M. truncatula </italic> roots to nodulate with the <italic> L. japonicus </italic> symbiont <italic> Mesorhizobium loti </italic> [ <xref ref-type="bibr" rid="B24"> 24 </xref> ]. However, natural variation in NFRs that causes changed specificity has rarely been documented. In contrast, numerous dominant genes have been identified in soybeans and other legumes that restrict nodulation with specific rhizobial strains [ <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ]. We recently cloned the two soybean genes <italic> <span class="gene">Rj2</span> </italic> and <italic> Rfg1 </italic> that restrict nodulation with specific strains of <italic> Bradyrhizobium japonicum </italic> and <italic> Sinorhizobium fredii </italic> , respectively [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. We demonstrated that <italic> <span class="gene">Rj2</span> </italic> and <italic> Rfg1 </italic> are allelic genes encoding a member of the Toll-interleukin receptor/nucleotide-binding site/leucine-rich repeat (TIR-NBS-LRR) class of plant <span class="trait">resistance</span> (R) proteins. Our discovery is consistent with recent reports that documented a large number of secreted effectors delivered into the host cell by rhizobial T3SS and suggests that establishment of a root nodule symbiosis requires the evasion of plant immune responses triggered by rhizobial effectors.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:&#x003c9;-Alkynyl Lipid Surrogates for Polyunsaturated Fatty Acids: Free Radical and Enzymatic Oxidations</h2>
			<h3>PMID:25034362</h3>
				
					<h3>PMCID:4140476</h3>
					<h4>期刊:Journal of the American Chemical Society</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="ath1">  <name>  <surname> Beavers </surname>  <given-names> William
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To a solution of <italic> a </italic> LA in benzene was added 2,2&#x02032;-azobis(4-methoxy-2,4-dimethylvaleronitrile) (MeOAMVN) in benzene, and the mixture was incubated at 37 &#x000b0;C. After 45 min, a solution of butylated hydroxytoluene (BHT) and triphenyl phosphine (PPh <sub> 3 </sub> ) in benzene was added, and the mixture was vortexed for 1 min. Benzene was evaporated under a stream of argon, and the residual material was reconstituted in 1.2% isopropanol in hexanes with 0.1% acetic acid for HPLC-UV analysis. For direct infusion <span class="gene">MS</span> studies of <italic> a </italic> HODE, eluted peaks from HPLC-UV analysis were collected, and solvents were evaporated under a stream of argon; then the residues were reconstituted in methanol. For NMR studies of <italic> a </italic> HODE, corresponding peaks collected from HPLC-UV analysis were combined, solvents were evaporated, and the residues were dried under <span class="trait">high</span> vacuum for 2 h. These dried materials were reconstituted in benzene- <italic> d </italic> <sub> 6 </sub> and placed into 1.7 mm OD sample tubes for NMR analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate the potential of <italic> a </italic> AA as a tool in cellular settings, its incorporation, release, and metabolism was tested in RAW264.7 macrophages. BSA/ <italic> a </italic> AA complexes were formed as previously described and then were added to serum-free cell culture medium for 24 h. <sup> <xref ref-type="bibr" rid="ref29"> 29 </xref> </sup> Cells were then washed with medium to remove any unincorporated <italic> a </italic> AA and activated with 100 ng/mL KLA for 24 h. Fatty acid metabolites were extracted from the combined media and cells. Figure <xref rid="fig7" ref-type="fig"> 7 </xref> shows the quantification of <italic> a </italic> 11-HETE, <italic> a </italic> 11-8,9-HEET, <italic> a </italic> PG, and PG in cells enriched with <italic> a </italic> AA and activated with KLA. The alkynylated products were only seen at <span class="trait">high</span> levels in the <italic> a </italic> AA-enriched and KLA-activated cells, which corresponds to the conditions where levels of fatty acid release and <span class="gene">COX2</span> expression are highest. Further correlating to the kinetic and <italic> in vitro </italic> experiments, <italic> a </italic> 11-HETE was more abundant than <italic> a </italic> PG and <italic> a </italic> 11-8,9-HEET in cells. The ratio of <italic> a </italic> 11-HETE to <italic> a </italic> PG and <italic> a </italic> 11-8,9-HEET is increased from the purified <span class="trait">protein</span> analyses indicating that cellular and purified enzyme metabolite profiles may have slight differences.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Complete Plastome Sequences from <italic> Glycine syndetika </italic> and Six Additional Perennial Wild Relatives of Soybean</h2>
			<h3>PMID:25155272</h3>
				
					<h3>PMCID:4199708</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sherman-Broyles </surname>  <given-names> Sue </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bombarely </surname>  <given-names> Aureliano </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grimwood </surname>  <given-names> Jane </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schmutz </surname>  <given-names> Jeremy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Doyle </surname>  <given-names> Jeff </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <aff id="aff1">  <label> * </label> Cornell University, Department of Plant Biology, Ithaca, New York 14853 </aff>  <aff id="aff2">  <label> &#x02020; </label> Hudson Alpha Institute for Biotechnology, Huntsville, Alabama 35806 </aff>
				<br/> 
				年份：<day> 19 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Glycine </italic> plastomes contain 111 unique genes, including 77 <span class="trait">protein</span> coding genes, 30 tRNA genes, and 4 rRNA genes. Six <span class="trait">protein</span> coding genes, seven tRNA genes, and all of the rRNA genes are completely duplicated in the IRs. The perennial <italic> Glycine </italic> plastomes are similar to the annual soybean plastome, with 19 genes containing introns, of which six are tRNA genes. Two genes have alternative start codons ( <italic> <span class="gene">psbL</span> </italic> and <italic> <span class="gene">ndhD</span> </italic> ) in all of the <italic> Glycine </italic> chloroplast sequences as found in other angiosperm plastomes ( <xref rid="bib68" ref-type="bibr"> Sugiura 2013 </xref> ). The GC content in <italic> G. max </italic> is 34% ( <xref rid="bib61" ref-type="bibr"> Saski <italic> et al. </italic> 2005 </xref> ), whereas in the perennial taxa the GC content of all species is 35% ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Codon bias is consistent with the A/T-rich aspect of angiosperm chloroplast genomes, with codons with A/T(U) in the third position being more common.
					</p>
				
					<p>&nbsp;&nbsp;
						The total number of genes reported above does not include <italic> <span class="gene">ycf4</span> </italic> , which in legumes is divergent from other angiosperm <italic> <span class="gene">ycf4</span> </italic> sequences ( <xref rid="bib67" ref-type="bibr"> Stefanovic <italic> et al. </italic> 2009 </xref> ) and was not annotated by Dogma ( <xref rid="bib76" ref-type="bibr"> Wyman <italic> et al. </italic> 2004 </xref> ) or GenBank for our submissions. <xref rid="bib67" ref-type="bibr"> Stefanovic <italic> et al. </italic> (2009) </xref> were the first to draw attention to the fact that the gene was present and highly divergent in legumes ( <xref rid="bib67" ref-type="bibr"> Stefanovic <italic> et al. </italic> 2009 </xref> ) rather than absent, as suggested by DNA hybridization surveys ( <xref rid="bib15" ref-type="bibr"> Doyle <italic> et al. </italic> 1995 </xref> ) and the <italic> G. max </italic> annotation ( <xref rid="bib61" ref-type="bibr"> Saski <italic> et al. </italic> 2005 </xref> ). This gene is thought to be a nonessential Photosystem I assembly factor in higher plants ( <xref rid="bib37" ref-type="bibr"> Krech <italic> et al. </italic> 2012 </xref> ), and this level of sequence divergence may indicate that if its function is retained in <italic> Glycine </italic> , it is replaced by a nuclear copy of a plastid gene (NuPT). This region flanks one of several NuPTs found in the soybean genome sequence (A. Bombarely, D. Robinson, S. Sherman-Broyles, and J. J. Doyle, unpublished data). Although <italic> <span class="gene">ycf4</span> </italic> is divergent from other legume <italic> <span class="gene">ycf4</span> </italic> sequences, there is no indication that it is a mutational hotspot in <italic> Glycine </italic> , as it is known to be in the legume genus <italic> Lathyrus </italic> ( <xref rid="bib41" ref-type="bibr"> Magee <italic> et al. </italic> 2010 </xref> ).There are <span class="trait">high</span> levels of sequence similarity in genes between <italic> <span class="gene">rps16</span> </italic> and <italic> <span class="gene">cemA</span> </italic> , the hypervariable region in <xref rid="bib41" ref-type="bibr"> Magee <italic> et al. </italic> (2010) </xref> ( <xref ref-type="fig" rid="fig3"> Figure 3 </xref> ). Pairwise Ka/Ks values for all <italic> Glycine </italic> species (data not shown) are misleading because there are so few differences that ratios were based on too few polymorphic sites.
					</p>
				
					<p>&nbsp;&nbsp;
						The genes that mark the beginning and end of the IR are only partially duplicated: 61 bp of <italic> <span class="gene">rps19</span> </italic> for all species except <italic> G. falcata </italic> (59 bp), and hypothetical chloroplast RF1 ( <italic> ycf </italic> 1), with 478 bp duplicated in <italic> G. max </italic> , 490 bp in <italic> G. dolichocarpa </italic> , and 463 bp in the remaining perennial <italic> Glycine </italic> taxa ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). The full-length <italic> <span class="gene">ycf1</span> </italic> sequence in <italic> G. cyrtoloba </italic> , spanning the IRb and SSC region, has a premature stop codon in the single copy region; further analyses are necessary to determine if this is a sequencing error or an edited base, or if <italic> <span class="gene">ycf1</span> </italic> is a pseudogene in <italic> G. cyrtoloba </italic> . The function of <italic> <span class="gene">ycf1</span> </italic> was recently determined to be a translocon <span class="trait">protein</span> of the inner chloroplast membrane and in <italic> Arabidiposis </italic> was renamed Tic214 ( <xref rid="bib35" ref-type="bibr"> Kikuchi <italic> et al. </italic> 2013 </xref> ). It has been found to have relaxed substitution rates except at its 5&#x02032; end, which may coincide with an origin of replication but might also be explained by its duplication in the IR regions, which have lower substitution rates than other chloroplast regions ( <xref rid="bib73" ref-type="bibr"> Wicke <italic> et al. </italic> 2011 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The &#x0201c;one-step&#x0201d; <italic> Bean pod mottle virus </italic> (BPMV)-derived vector is a functional genomics tool for efficient overexpression of heterologous <span class="trait">protein,</span> virus-induced gene silencing and genetic mapping of BPMV R-gene in common bean ( <italic> Phaseolus vulgaris </italic> L.)</h2>
			<h3>PMID:25168520</h3>
				
					<h3>PMCID:4163167</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Pflieger </surname>  <given-names> St&#x000e9;phanie </given-names>  </name>  <address>  <email> stephanie.pflieger@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Blanchet </surname>  <given-names> Sophie </given-names>  </name>  <address>  <email> sophie.blanchet@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meziadi </surname>  <given-names> Chouaib </given-names>  </name>  <address>  <email> chouaib.meziadi@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Richard </surname>  <given-names> Manon MS </given-names>  </name>  <address>  <email> manon.richard@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Thareau </surname>  <given-names> Vincent </given-names>  </name>  <address>  <email> vincent.thareau@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mary </surname>  <given-names> Fanny </given-names>  </name>  <address>  <email> fanny.mary@live.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mazoyer </surname>  <given-names> C&#x000e9;line </given-names>  </name>  <address>  <email> c.mazoyer@aol.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Geffroy </surname>  <given-names> Val&#x000e9;rie </given-names>  </name>  <address>  <email> valerie.geffroy@u-psud.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <aff id="Aff1">  <label/> CNRS, Institut de Biologie des Plantes, UMR 8618, Universit&#x000e9; Paris Sud, Saclay Plant Sciences (SPS), 91405 Orsay, France  </aff>  <aff id="Aff2">  <label/> Univ Paris Diderot, Sorbonne Paris Cit&#x000e9;, 75205 Paris, France  </aff>  <aff id="Aff3">  <label/> INRA, Unit&#x000e9; Mixte de Recherche de G&#x000e9;n&#x000e9;tique V&#x000e9;g&#x000e9;tale, Universit&#x000e9; Paris Sud, IDEEV FR3284, Ferme du Moulon, 91190 Gif-sur-Yvette, France  </aff>
				<br/> 
				年份：<day> 20 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The &#x02018;one-step&#x02019; BPMV vector has been shown to infect common bean cv. Black Valentine [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. Three weeks post-inoculation (wpi) of common bean plants with a BPMV- <italic> Green Fluorescent <span class="trait">Protein</span> </italic> (GFP) construct, extensive green fluorescence was visible in the upper systemic leaves and roots of infected plants [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. To date, only one VIGS study has been reported in common bean using the first generation BPMV vector [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. Common bean genes encoding nodulin 22 and stearoyl-acyl carrier <span class="trait">protein</span> desaturase were successfully silenced in cv. Black Valentine [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. However, use of this first generation BPMV vector is limited to low-throughput VIGS studies mainly because (i) foreign sequences must be cloned in-frame into the RNA2 <span class="gene">polyprotein</span> and (ii) delivery into plants is achieved by viral RNAs transcribed <italic> in vitro </italic> from the BPMV constructs.
					</p>
				
					<p>&nbsp;&nbsp;
						GFP fluorescence was detected in reproductive tissues of <italic> P. vulgaris </italic> cv. Black Valentine (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). Fluorescence was observed at 30 dpi in floral buds of BPMV-GFP infected plants. In petals, we observed stronger fluorescence in the standard (dorsal petal) compared with the lateral and ventral petals (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). At 8 wpi, pods of BPMV-0 and BPMV-GFP infected plants exhibited strong viral symptoms characterized by a curved shape and a bloated and mottled pod surface (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). When observed under UV light, infected pods from BPMV-GFP infected plants displayed extensive and homogenous GFP fluorescence (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). Notably, at 10 wpi, no GFP fluorescence was detected in the embryos of seeds harvested from BPMV-GFP-infected plants, while strong fluorescence was observed in the corresponding seed coats (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> <bold> <italic> Bean pod mottle virus </italic> </bold> <bold> (BPMV)-induced expression of the </bold> <bold> <italic> green fluorescent <span class="trait">protein</span> </italic> </bold> <bold> ( </bold> <bold> <italic> GFP </italic> </bold> <bold> ) gene in reproductive tissues after rub-inoculation of one primary leaf with leaf sap. </bold> Floral buds, flowers, pods and seeds of <italic> P. vulgaris </italic> cv. Black Valentine plants infected with mock buffer, BPMV empty vector (BPMV-0) and <italic> GFP </italic> -expressing vector (BPMV-GFP) were photographed at 30&#x000a0;days post-inoculation (pi), 30&#x000a0;days pi, 8&#x000a0;weeks <span class="gene">pi</span> and 10&#x000a0;weeks pi, respectively, under natural light (top panel) and UV light (bottom panel).
					</p>
				
					<p>&nbsp;&nbsp;
						The efficiency of endogenous gene silencing using the BPMV VIGS vector delivered through direct DNA rubbing in <italic> P. vulgaris </italic> cv. Black Valentine was investigated by targeting the <italic> PvPDS </italic> gene. <italic> PDS </italic> is routinely used as a marker gene for VIGS in plants as silencing this gene causes chlorophyll degradation resulting in a typical photobleached phenotype in emerging leaves. Initial tests were carried out with the BPMV-GmPDS-327&#x000a0;bp construct containing a 327&#x000a0;bp fragment of the <italic> PDS </italic> gene from <italic> Glycine max </italic> ( <italic> <span class="gene">GmPDS</span> </italic> ) (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ) as it was immediately available (supplied by C. Zhang) [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. Alignment of the 327&#x000a0;bp <italic> <span class="gene">GmPDS</span> </italic> fragment with <italic> PvPDS </italic> sequences from G19833 and BAT93 revealed a <span class="trait">high</span> level of sequence conservation with 5 DNA stretches of 23&#x000a0;nt or more (the minimal length for VIGS induction, [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ]) having 100% identity between the two <italic> PvPDS </italic> sequences and the <italic> <span class="gene">GmPDS</span> </italic> sequence (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S4). The BPMV-GmPDS-327&#x000a0;bp construct was delivered into <italic> P. vulgaris </italic> cv. Black Valentine seedlings by direct rub-inoculation. Infected leaves were used for secondary inoculations of healthy plants. The infected plants displayed photobleached leaves at 28 dpi, unlike plants infected with the empty BPMV-0 vector or mock buffer (Figures&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> A and B). <fig id="Fig5"> <label> Figure 5 </label> <caption> <p> <bold> Silencing of </bold> <bold> <italic> PDS </italic> </bold> <bold> in </bold> <bold> <italic> P. vulgaris </italic> </bold> <bold> cv. Black Valentine using the VIGS vector BPMV-GmPDS-327&#x000a0;bp. (A-B) </bold> Plants inoculated with mock buffer (left panel), BPMV-0 (middle panel), and BPMV-GmPDS-327&#x000a0;bp (right panel) were photographed under natural light at 28&#x000a0;days post-inoculation (dpi). <bold> (B) </bold> Trifoliate systemic leaves. <bold> (C) </bold> Semi-quantitative RT-PCR of <italic> PvPDS </italic> , BPMV RNA1 and RNA2 in systemic leaves of plants inoculated with mock, BPMV-0, and BPMV-GmPDS-327&#x000a0;bp. Ubiquitin ( <italic> PvUBI </italic> ) was used as an internal control. Total RNA was extracted at 21 dpi from the third trifoliate leaf of three different plants of Black Valentine.
					</p>
				
					<p>&nbsp;&nbsp;
						The duration of VIGS was estimated in <italic> P. vulgaris </italic> cv. Black Valentine plants inoculated with the BPMV-PvPDS-391&#x000a0;bp vector. Plants grown under normal light conditions showed recovery of silenced leaves more than 2&#x000a0;months pi, as characterized by an overall decline of white leaves over time. However, silenced plants placed under <span class="trait">high</span> intensity illumination (sodium lamp) displayed a photobleached phenotype for more than 3&#x000a0;months <span class="gene">pi</span> (data not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						Names of the BPMV constructs used in this study and the corresponding RNA1- and RNA2-derived plasmids are listed in Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> . The pBPMV-IA-R1M, pBPMV-IA-V1, pBPMV-GFP2 and pBPMV-PDS-3R DNA plasmids were kindly supplied by C. Zhang from Iowa State University, USA, and were described in Zhang <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. All others RNA2-derived plasmids were constructed in our laboratory by insertion of <italic> P. vulgaris </italic> target gene fragments in the <italic> Bam </italic> <span class="gene">H1</span> restriction site of pBPMV-IA-V1. For construction of BPMV-PvPDS-52&#x000a0;bp, we directly cloned a double-stranded synthetic oligonucleotide (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ) into the <italic> Bam </italic> <span class="gene">H1</span> restriction site. For all other BPMV-derived vectors, target gene fragments were first amplified by PCR using specific primers (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ), <italic> P. vulgaris </italic> cDNAs as template, and a <span class="trait">high</span> fidelity polymerase (Advantage HF2, Clontech). PCR products were digested with <italic> Bam </italic> <span class="gene">H1</span> and inserted into <italic> Bam </italic> H1-digested and dephosphorylated pBPMV-IA-V1 to generate the VIGS vectors listed in Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> . The orientation of the cloned inserts was determined by PCR using a combination of vector-specific and fragment-specific primers (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). Clones thought to contain inserted fragments in the antisense orientation were sequenced (GATC, Germany) to confirm their identity and were subsequently used for gene silencing. <table-wrap id="Tab3"> <label> Table 3 </label> <caption> <p> <bold> PCR primers used for construction of the BPMV VIGS vectors and RT-PCR analyses </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						<media position="anchor" xlink:href="12870_2014_232_MOESM1_ESM.pptx" id="MOESM1"> <label> Additional file 1: Figure S1. </label> <caption> <p> <italic> Bean pod mottle virus </italic> (BPMV)-induced expression of the <italic> green fluorescent <span class="trait">protein</span> </italic> ( <italic> GFP </italic> ) gene in roots after rub inoculation of one primary leaf with leaf sap. Roots of <italic> P. vulgaris </italic> cv. Black Valentine plants infected with mock buffer, BPMV empty vector (BPMV-0) and <italic> GFP </italic> -expressing vector (BPMV-GFP) were rinsed and photographed at 21&#x000a0;days post-inoculation (dpi) under natural light (top panel) and UV light (middle panel). Epifluorescence microscopy detection of GFP fluorescence in roots of common bean plants (bottom panel). Scale bars are 250&#x000a0;&#x003bc;m. <bold> Figure S2. </bold> Stability of <italic> green fluorescent <span class="trait">protein</span> </italic> ( <italic> GFP </italic> )-expression in <italic> P. vulgaris </italic> cv. Black Valentine after four serial inoculations. BPMV-GFP inoculated plants of a fourth serial inoculation were photographed under UV light, at 21&#x000a0;days post-inoculation (dpi). <bold> Figure S3. </bold> <italic> Bean pod mottle virus </italic> (BPMV)-induced expression of the <italic> green fluorescent <span class="trait">protein</span> </italic> ( <italic> GFP </italic> ) gene in leaves of <italic> P. vulgaris </italic> cultivars. BPMV-GFP inoculated plants were photographed under UV light, at 7&#x000a0;days post inoculation (dpi) for inoculated leaves (IL), and at 21 dpi and 28 dpi for systemic leaves (SL). <bold> Figure S4. </bold> Alignement of the 327-bp fragment of the <italic> Glycine max PDS </italic> ortholog ( <italic> <span class="gene">GmPDS</span> </italic> ) with the corresponding regions of <italic> PvPDS </italic> from <italic> P. vulgaris </italic> cv. G19833 ( <italic> PvaPDS </italic> ) and BAT93 ( <italic> PvmPDS </italic> ). The longest region presenting 100% nucleic identity has a length of 52&#x000a0;bp and is indicated with light grey characters on the alignment. Overall identity of the three sequences on the 327&#x000a0;bp region is 92%.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The kinase LYK5 is a major chitin receptor in <italic> Arabidopsis </italic> and forms a chitin-induced complex with related kinase CERK1</h2>
			<h3>PMID:25340959</h3>
				
					<h3>PMCID:4356144</h3>
					<h4>期刊:eLife</h4>
				

				
			<p>
				作者：<contrib id="author-15831" contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Yangrong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="aff" rid="aff2"/>  <xref ref-type="author-notes" rid="equal-contrib"> &#x02020; </xref>  <xref ref-type="fn" rid="con1"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-15832" contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="aff" rid="aff2"/>  <xref ref-type="author-notes" rid="equal-contrib"> &#x02020; </xref>  <xref ref-type="fn" rid="con2"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-15838" contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Kiwamu </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-5045-560X </contrib-id>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="aff" rid="aff2"/>  <xref ref-type="author-notes" rid="pa1"> &#x02021; </xref>  <xref ref-type="fn" rid="con3"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-15834" contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Cuong T </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="aff" rid="aff2"/>  <xref ref-type="fn" rid="con4"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-15835" contrib-type="author">  <name>  <surname> Jedrzejczak </surname>  <given-names> Robert P </given-names>  </name>  <xref ref-type="aff" rid="aff3"/>  <xref ref-type="fn" rid="con5"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-15836" contrib-type="author">  <name>  <surname> Joachimiak </surname>  <given-names> Andrzej </given-names>  </name>  <xref ref-type="aff" rid="aff3"/>  <xref ref-type="other" rid="par-3"/>  <xref ref-type="other" rid="par-4"/>  <xref ref-type="fn" rid="con6"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <contrib id="author-3525" contrib-type="author">  <name>  <surname> Stacey </surname>  <given-names> Gary </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="aff" rid="aff2"/>  <xref ref-type="corresp" rid="cor1"> * </xref>  <xref ref-type="other" rid="par-1"/>  <xref ref-type="other" rid="par-2"/>  <xref ref-type="fn" rid="con7"/>  <xref ref-type="fn" rid="conf1"/>  </contrib>  <aff id="aff1">  <institution content-type="dept"> Division of Plant Sciences, National Center for Soybean Biotechnology </institution> ,  <institution> University of Missouri </institution> ,  <addr-line> Columbia </addr-line> ,  <country> United States </country>  </aff>  <aff id="aff2">  <institution content-type="dept"> Department of Biochemistry </institution> ,  <institution> University of Missouri </institution> ,  <addr-line> Columbia </addr-line> ,  <country> United States </country>  </aff>  <aff id="aff3">  <institution content-type="dept"> Midwest Center for Structural Genomics </institution> ,  <institution> Argonne National Laboratory </institution> ,  <addr-line> Argonne </addr-line> ,  <country> United States </country>  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plants use cell-surface localized pattern-recognition receptors (PRRs) to detect MAMPs activating MTI ( <xref rid="bib5" ref-type="bibr"> Boller and Felix, 2009 </xref> ; <xref rid="bib34" ref-type="bibr"> Macho and Zipfel, 2014 </xref> ). In plants, several PRRs have been well characterized, including FLAGELLIN-SENSING 2 (FLS2) and elongation factor-TU (EF-Tu) RECEPTOR (EFR), which detect bacterial flagellin and EF-Tu, respectively ( <xref rid="bib14" ref-type="bibr"> Gomez-Gomez and Boller, 2000 </xref> , <xref rid="bib13" ref-type="bibr"> 2002 </xref> ; <xref rid="bib67" ref-type="bibr"> Zipfel et al., 2006 </xref> ). These two receptors belong to the leucine rich repeat receptor like <span class="trait">protein</span> kinase (LRR-RLK) family ( <xref rid="bib51" ref-type="bibr"> Shiu and Bleecker, 2001 </xref> ). Activation of these receptors by ligand binding induces the association with BRASSINOSTEROID INSENSITIVE 1 (BRI1)- Associated receptor Kinase 1 (BAK1) and phosphorylation of Botrytis-Induced Kinase (BIK1) ( <xref rid="bib9" ref-type="bibr"> Chinchilla et al., 2007 </xref> ; <xref rid="bib18" ref-type="bibr"> Heese et al., 2007 </xref> ; <xref rid="bib33" ref-type="bibr"> Lu et al., 2010 </xref> ; <xref rid="bib46" ref-type="bibr"> Roux et al., 2011 </xref> ; <xref rid="bib66" ref-type="bibr"> Zhang et al., 2010 </xref> ). Phosphorylated BIK1 dissociates from the receptor and subsequently phosphorylates the respiratory burst oxidase homolog D (RBOHD) <span class="trait">protein,</span> which controls ROS production in a calcium-independent manner ( <xref rid="bib22" ref-type="bibr"> Kadota et al., 2014 </xref> ; <xref rid="bib27" ref-type="bibr"> Li et al., 2014 </xref> ). Other MAMPs, such as the oligosaccharides bacterial peptidoglycan (PGN) and fungal chitin [degree of polymerization (dp) &#x02265; 6], are detected by lysin-motif (LysM) containing proteins. The chitin receptor was first reported in rice with the identification of the chitin-elicitor binding <span class="trait">protein</span> (CEBiP) ( <xref rid="bib23" ref-type="bibr"> Kaku et al., 2006 </xref> ), which contains an extracellular LysM motif and a transmembrane domain, but lacks an intracellular kinase domain. Data indicate that CEBiP forms a complex with the rice chitin-elicitor receptor kinase 1 (OsCERK1) to mediate MTI in response to chitin ( <xref rid="bib17" ref-type="bibr"> Hayafune et al., 2014 </xref> ; <xref rid="bib49" ref-type="bibr"> Shimizu et al., 2010 </xref> ). OsCERK1 has an active, intracellular kinase domain. The data suggest that OsCERK1 does not bind chitin but its intracellular kinase domain is activated by chitin binding to OsCEBiP ( <xref rid="bib17" ref-type="bibr"> Hayafune et al., 2014 </xref> ). In <italic> Arabidopsis thaliana </italic> , AtCERK1 was shown to be a key chitin receptor involved in chitin perception ( <xref rid="bib38" ref-type="bibr"> Miya et al., 2007 </xref> ; <xref rid="bib59" ref-type="bibr"> Wan et al., 2008 </xref> ). For example, <italic> Atcerk1 </italic> mutant plants completely lose the ability to respond to chitin elicitation. There are three homologs of CEBiP in <italic> Arabidopsis </italic> , but a triple knock-out mutant, <italic> Atlym1/Atlym2/Atlym3 </italic> lacking these three proteins was fully competent to respond to chitin treatment ( <xref rid="bib58" ref-type="bibr"> Wan et al., 2012 </xref> ). However, two CEBiP like proteins appear to function in conjunction with CERK1 to mediate recognition of bacterial <span class="gene">PGN</span> (OsLYP4 and OsLYP6 in rice, AtLYM1 and AtLYM3 in <italic> Arabidopsis </italic> ) ( <xref rid="bib31" ref-type="bibr"> Liu et al., 2012a </xref> ; <xref rid="bib63" ref-type="bibr"> Willmann et al., 2011 </xref> ). Another CEBiP-like <span class="trait">protein,</span> AtLYM2, was demonstrated to act independently of AtCERK1 to mediate chitin-induced suppression of intracellular flux through plasmodesmata ( <xref rid="bib12" ref-type="bibr"> Faulkner et al., 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A complete alignment based on the full-length sequences of each <span class="trait">protein</span> was used to draw the phylogenetic tree using Clustal X software. AtCERK1: At3g21630, AtLYK2: At3g01840, AtLYK3: <span class="gene">At1g51940,</span> AtLYK4: At2g23770, AtLYK5: At2g33580.
					</p>
				
					<p>&nbsp;&nbsp;
						Receptor homodimerization or oligomerization and subsequent phosphorylation are common mechanisms for ligand-mediate receptor activation. <italic> Arabidopsis </italic> examples include AtBRI1 homodimerization and heterodimerization with AtBAK1 ( <xref rid="bib26" ref-type="bibr"> Li et al., 2002 </xref> ; <xref rid="bib41" ref-type="bibr"> Nam and Li, 2002 </xref> ; <xref rid="bib48" ref-type="bibr"> Santiago, et al., 2013 </xref> ; <xref rid="bib54" ref-type="bibr"> Sun, et al., 2013a </xref> ; <xref rid="bib60" ref-type="bibr"> Wang et al., 2005 </xref> ). AtFLS2 appears to exist in the cell as a homodimer before and after ligand perception and then associates with AtBAK1 upon flagellin treatment ( <xref rid="bib1" ref-type="bibr"> Albert, et al., 2013 </xref> ; <xref rid="bib9" ref-type="bibr"> Chinchilla et al., 2007 </xref> ; <xref rid="bib18" ref-type="bibr"> Heese et al., 2007 </xref> ; <xref rid="bib53" ref-type="bibr"> Sun et al., 2012 </xref> ; <xref rid="bib55" ref-type="bibr"> Sun, et al., 2013b </xref> ). The suggested mechanism is that ligand perception triggers autophosphorylation between the homodimer or trans-autophosphorylation between/among hetero-interacting proteins to initiate cellular signaling. It appears that all LysM receptors may function as a <span class="trait">protein</span> complex. For example, in leguminous plants, the LysM domain, Nod factor receptors (e.g., <italic> Lotus japonicas </italic> Nod factor receptor 1 and 5; LjNFR1 and LjNFR5) function as a heterodimer to mediate <span class="trait">high</span> affinity binding to the lipo-chitooligosaccharide Nod factor ( <xref rid="bib6" ref-type="bibr"> Broghammer et al., 2012 </xref> ; <xref rid="bib15" ref-type="bibr"> Gust et al., 2012 </xref> ; <xref rid="bib35" ref-type="bibr"> Madsen et al., 2011 </xref> ; <xref rid="bib45" ref-type="bibr"> Radutoiu et al., 2003 </xref> ). It should be emphasized that this model for Nod factor binding is very similar to that of the <italic> Arabidopsis </italic> chitin receptor. In both cases, one LysM <span class="gene">RLK</span> (CERK1/NFR1) has an active intracellular kinase domain but interacts with a second LysM <span class="gene">RLK</span> (LYK5/NFR5) that lacks intracellular kinase activity. This similarity underlines the now well recognized evolutionary link between chitin, Nod factor and mycorrhizal (Myc) factor recognition ( <xref rid="bib29" ref-type="bibr"> Liang et al., 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Purification of AtCERK1 and AtLYK5 intracellular domains fused with N-terminal <span class="gene">GST</span> tag and in vitro kinase assay were performed as described by <xref rid="bib8" ref-type="bibr"> Cao et al. (2013) </xref> . Briefly, cultures of <italic> E. coli </italic> strains harboring plasmid were supplemented with 0.1 mM isopropyl b-D-thiogalactopyranoside (IPTG) at OD <sub> 600 </sub> 0.8 at 18&#x000b0;C for 12 hr. The cells were lysed in a buffer containing 1 &#x000d7; PBS (MP Biomedicals, France) supplemented with 1 mM EDTA, 0.1% Triton X-100, 1 mg/ml lysozyme and placed on ice for 30 min with slow shaking before sonication. After centrifuge at 4&#x000b0;C for 30 min, the supernatant was applied to a column containing glutathione Sepharose 4B (GE Healthcare, Milwaukee, WI) for <span class="trait">protein</span> purification. The column was washed five-times with 1 &#x000d7; PBS buffer. Recombinant proteins were eluted with 10 mM reduced glutathione. Proteins were dialyzed with a buffer containing 50 mM Tris (PH 7.6), 50 mM KCl, and 10% glycerol. For in vitro kinase assay, proteins were incubated in the buffer [50 mM Tris (PH 7.6), 50 mM KCl, 10 mM MnCl <sub> 2 </sub> , 10 mM MgCl <sub> 2 </sub> , 10 mM ATP, and 10 mCi &#x003b3;-[ <sup> 32 </sup> P]-ATP at room temperature for 30 min. Autoradiography was performed using a phosphor screen and a phosphorimager.
					</p>
				
					<p>&nbsp;&nbsp;
						However, the problem is that expression and purification of the LysM <span class="trait">protein</span> is very difficult. Based on our own experience, this is due to an inability to express the proteins in sufficient quantity for such studies. We consider ourselves fortunate that we were able to get enough CERK1 and LYK5 proteins to do the studies described in the paper. Unfortunately, the same was not true of LYK4. Using the same conditions we used for CERK1/LYK4, we tried to express the <span class="gene">LYK4</span> extracellular domain in <italic> E.coli </italic> but obtained poor expression. What <span class="trait">protein</span> that was produced was insoluble and efforts to solubilize and refold have, thus far, proven unsuccessful. Therefore, although we would like to address the reviewers and our own curiosity about LYK4, this is going to have to wait until we work out a condition to get sufficient <span class="trait">protein</span> for binding studies. Since our current paper focuses on LYK5, these experiments concerning <span class="gene">LYK4</span> are not really pertinent to the story being described. The data support the model that LYK5 is the major chitin binding <span class="trait">protein,</span> while <span class="gene">LYK4</span> plays a minor role. We certainly hope to overcome the technical problems with regard to <span class="gene">LYK4</span> expression and, if successful, will likely publish this as a separate story with the main focus on LYK4.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The IQD Gene Family in Soybean: Structure, Phylogeny, Evolution and Expression</h2>
			<h3>PMID:25343341</h3>
				
					<h3>PMCID:4208818</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Feng </surname>  <given-names> Lin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chen </surname>  <given-names> Zhu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yiyi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xiang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 9 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Members of the plant-specific IQ67-domain (IQD) <span class="trait">protein</span> family are involved in plant development and the basal defense response. Although systematic characterization of this family has been carried out in <italic> Arabidopsis </italic> , tomato ( <italic> Solanum lycopersicum </italic> ), <italic> Brachypodium distachyon </italic> and rice ( <italic> Oryza sativa </italic> ), systematic analysis and expression profiling of this gene family in soybean ( <italic> Glycine max </italic> ) have not previously been reported. In this study, we identified and structurally characterized IQD genes in the soybean genome. A complete set of 67 soybean IQD genes ( <italic> <span class="gene">GmIQD1</span> </italic> &#x02013; <italic> 67 </italic> ) was identified using Blast search tools, and the genes were clustered into four subfamilies (IQD I&#x02013;IV) based on phylogeny. These soybean IQD genes are distributed unevenly across all 20 chromosomes, with 30 segmental duplication events, suggesting that segmental duplication has played a major role in the expansion of the soybean IQD gene family. Analysis of the Ka/Ks ratios showed that the duplicated genes of the <span class="gene">GmIQD</span> family primarily underwent purifying selection. Microsynteny was detected in most pairs: genes in clade 1&#x02013;3 might be present in genome regions that were inverted, expanded or contracted after the divergence; most gene pairs in clade 4 showed <span class="trait">high</span> conservation with little rearrangement among these gene-residing regions. Of the soybean IQD genes examined, six were most highly expressed in young leaves, six in flowers, one in roots and two in nodules. Our qRT-PCR analysis of 24 soybean IQD III genes confirmed that these genes are regulated by MeJA stress. Our findings present a comprehensive overview of the soybean IQD gene family and provide insights into the evolution of this family. In addition, this work lays a solid foundation for further experiments aimed at determining the biological functions of soybean IQD genes in growth and development.
					</p>
				
					<p>&nbsp;&nbsp;
						Calmodulin (CaM) and calcineurin B-like (CBL) proteins, which lack catalytic activity, are sometimes referred to as &#x0201c;Ca <sup> 2+ </sup> sensor relays&#x0201d; <xref rid="pone.0110896-Zhang1" ref-type="bibr"> [15] </xref> , <xref rid="pone.0110896-Reddy2" ref-type="bibr"> [19] </xref> , <xref rid="pone.0110896-Batistic1" ref-type="bibr"> [20] </xref> . In contrast, <span class="gene">CDPK</span> proteins, which function as catalytic effectors, are referred to as &#x0201c;Ca <sup> 2+ </sup> sensor responders&#x0201d; <xref rid="pone.0110896-Zuo1" ref-type="bibr"> [18] </xref> . Among these Ca <sup> 2+ </sup> -binding proteins, calmodulin is the most extensively studied Ca <sup> 2+ </sup> sensor. Calmodulin is small, acid resistant, heat resistant and highly stable. This multifunctional <span class="trait">protein</span> is widespread in eukaryotic cells, highly conserved and has at least 30 multiple target proteins or enzymes <xref rid="pone.0110896-Boonburapong1" ref-type="bibr"> [21] </xref> &#x02013; <xref rid="pone.0110896-DeFalco1" ref-type="bibr"> [23] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The physicochemical parameters of each gene were calculated using ExPASy. Although all of the <span class="gene">GmIQD</span> genes encode the conserved IQ67 domains ( <xref ref-type="supplementary-material" rid="pone.0110896.s001"> Figure S1 </xref> ), their sequences are highly diverse with respect to size (141&#x02013;904 aa) and molecular mass (16.3&#x02013;99.2 kDa; <xref ref-type="table" rid="pone-0110896-t001"> Table 1 </xref> ). Almost all soybean IQD proteins (97%) have relatively <span class="trait">high</span> isoelectric points (pI&#x0003e;7.0 with an average of 10.1), except for <italic> <span class="gene">GmIQD9</span> </italic> (pI 5.4) and <italic> <span class="gene">GmIQD17</span> </italic> (pI 5.7; <xref ref-type="table" rid="pone-0110896-t001"> Table 1 </xref> ). All soybean IQD proteins were submitted to TargetP and Wolf PSORT to predict their subcellular localizations. Wolf PSORT revealed that fifty-seven soybean IQD proteins are localized to the nucleus, nine to the chloroplast and one to the endoplasmic reticulum. TargetP analysis revealed that fifteen soybean IQD proteins are located in the mitochondria, five in the chloroplast, one in the secretory pathway and forty-six in other compartments ( <xref ref-type="table" rid="pone-0110896-t001"> Table 1 </xref> ). The detailed parameters are provided in <xref ref-type="table" rid="pone-0110896-t001"> Table 1 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The schematic soybean IQD proteins were aligned relative to the IQ67 domain (motif 1, light blue box). The lengths of the proteins and motifs can be estimated using the scale. Motifs shared by at least four soybean IQD proteins are depicted at the reference <span class="gene">bar</span> on top of each alignment. The positions of putative calmodulin-binding sites predicted by the Calmodulin Target Database are indicated by vertical tick marks below each <span class="trait">protein</span> model. Subfamilies and subclasses (I&#x02013;IV) of IQD proteins are highlighted with colored backgrounds and colored vertical bars as in <xref ref-type="fig" rid="pone-0110896-g001"> Figure 1a </xref> to the right of the gene names.
					</p>
				
					<p>&nbsp;&nbsp;
						To further examine the structural diversity of the IQD genes in soybean, we deduced the exon/intron organization of individual <span class="gene">GmIQD</span> genes ( <xref ref-type="fig" rid="pone-0110896-g001"> Figure 1b </xref> ). A comparison of the 67 genomic loci with corresponding cDNA sequences revealed that most of the gene models predicted by GSDS are correct, except for one pair of genes ( <italic> <span class="gene">GmIQD9</span> </italic> /- <italic> 17 </italic> ). Both <italic> <span class="gene">GmIQD9</span> </italic> and <italic> <span class="gene">GmIQD17</span> </italic> encode six exons, but GSDS predicted that these genes contain only five exons. This unconformity is caused by the missing annotation of the fifth intron by GSDS. The schematic structures reveal that the coding sequence of each IQD gene contains 2&#x02013;6 translated exons ( <xref ref-type="fig" rid="pone-0110896-g001"> Figure 1b </xref> ), which is similar to that reported in <italic> Arabidopsis </italic> , rice and <italic> Brachypodium distachyon </italic> <xref rid="pone.0110896-Abel1" ref-type="bibr"> [30] </xref> , <xref rid="pone.0110896-Filiz1" ref-type="bibr"> [34] </xref> . More than three&#x02013;fifths of the soybean IQD gene family (41 members) contain five or six protein-coding exons, and one gene ( <italic> <span class="gene">GmIQD56</span> </italic> , encoding the smallest <span class="trait">protein,</span> comprising 141 aa) contains two exons ( <xref ref-type="fig" rid="pone-0110896-g001"> Figure 1b </xref> ). Most closely related soybean IQD members in the same subfamilies share similar intron numbers and exon lengths. Soybean IQD genes in subfamily II and IV possess five and six exons, respectively. Most members in subfamily III contain five exons, except for <italic> <span class="gene">GmIQD44</span> </italic> (four exons) and <italic> <span class="gene">GmIQD32</span> </italic> , <italic> <span class="gene">GmIQD41</span> </italic> , <italic> <span class="gene">GmIQD46</span> </italic> and <italic> <span class="gene">GmIQD47</span> </italic> (six exons). Subfamily I genes harbor 2&#x02013;5 exons. All introns of most IQD genes are in phase-0 (interrupting two triplet codons exactly); a phase-1 intron (separating the first and second nucleotides of a codon) was found in 15 remaining IQD genes, and no phase-2 intron (splitting the second and third nucleotides of a codon) was found ( <xref ref-type="fig" rid="pone-0110896-g001"> Figure 1b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 67 IQD genes from soybean were subjected to analysis with MEME to reveal conserved domains or motifs shared among related proteins. We identified 10 conserved motifs ( <xref ref-type="fig" rid="pone-0110896-g002"> Figure 2 </xref> and <xref ref-type="supplementary-material" rid="pone.0110896.s003"> Table S1 </xref> ). Each of the putative motifs was annotated by searching Pfam and SMART. Motif 1 was found to encode the IQ domain. Motif 2 and motif 7 were found to encode proteins of unknown function (DUF4005) and (DUF3982). While the other subfamily-specific motifs have not functional annotation. As expected, most of the closely related members had common motif compositions, suggesting functional similarities among IQD proteins within the same subfamily ( <xref ref-type="fig" rid="pone-0110896-g002"> Figure 2 </xref> ). The most common motif is motif 1, found in all sixty-seven soybean IQD genes ( <xref ref-type="fig" rid="pone-0110896-g002"> Figure 2 </xref> ). Motif 8 is mainly present in subfamily I besides one of <italic> <span class="gene">GmIQD14</span> </italic> exists in subfamily IV. Subfamily III members contain motif 1, motif 10, motif 4 and motif 3 in order, except for <italic> <span class="gene">GmIQD28</span> </italic> lacking motif 10. Motif 7 is peculiar to subfamily IV. To some extent, these subfamily-specific motifs may contribute to the functional divergence of IQD genes in soybean. The detailed information is shown in <xref ref-type="supplementary-material" rid="pone.0110896.s003"> table S1 </xref> . To predict calmodulin-binding sites, we searched the Calmodulin Target Database, which provides various structural and biophysical parameters for the 67 soybean IQD <span class="trait">protein</span> sequences. This analysis predicted that all soybean IQD proteins contain multiple IQ motifs and 1&#x02013;3 strings of high-scoring amino acid residues ( <xref ref-type="table" rid="pone-0110896-t003"> Table 3 </xref> ). These IQ motifs and amino acid residues indicate the locations of putative calmodulin interaction sites. Among the 67 IQD <span class="trait">protein</span> sequences, the predicted calmodulin binding sites of 50 sequences overlap with the IQ67 domain ( <xref ref-type="fig" rid="pone-0110896-g002"> Figure 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In clade 1 ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5a </xref> ), Map a shows a higher level of microsynteny, with both the same and opposite directions. <italic> SlSUN11 </italic> / <italic> <span class="gene">GmIQD21</span> </italic> and <italic> <span class="gene">GmIQD53</span> </italic> / <italic> <span class="gene">GmIQD48</span> </italic> exhibit remarkable opposite-direction microsynteny, while <italic> <span class="gene">GmIQD10</span> </italic> / <italic> <span class="gene">GmIQD18,</span> <span class="gene">GmIQD21</span> </italic> / <italic> <span class="gene">GmIQD28</span> </italic> , <italic> AtIQD7 </italic> / <italic> AtIQD8 </italic> and <italic> AtIQD8 </italic> / <italic> SlSUN11 </italic> are aligned with flanking gene pairs in the same order but discordant transcriptional orientation. In addition, genes in map a were divided into two groups ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5a </xref> ), i.e., one group with higher levels of microsynteny ( <italic> <span class="gene">GmIQD21</span> </italic> , <italic> <span class="gene">GmIQD28</span> </italic> , <italic> <span class="gene">GmIQD53</span> </italic> , <italic> <span class="gene">GmIQD48</span> </italic> , <italic> AtIQD7 </italic> , <italic> AtIQD8 </italic> and <italic> AtIQD8 </italic> and <italic> SlSUN11 </italic> ) and the other group with lower levels of microsynteny ( <italic> <span class="gene">GmIQD61</span> </italic> , <italic> <span class="gene">GmIQD50</span> </italic> , <italic> <span class="gene">GmIQD10</span> </italic> , <italic> <span class="gene">GmIQD18</span> </italic> , <italic> AtIQD6 </italic> and <italic> SlSUN22 </italic> ). These two groups were also detected in the phylogenetic tree of IQD III genes ( <xref ref-type="supplementary-material" rid="pone.0110896.s002"> Figure S2 </xref> ). In clade 2 ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5b </xref> ), two pairs from soybean and tomato, <italic> <span class="gene">GmIQD41</span> </italic> / <italic> SlSUN31 </italic> and <italic> <span class="gene">GmIQD47</span> </italic> / <italic> SlSUN31 </italic> , exhibited microsynteny. However, the predicted duplicated pair <italic> <span class="gene">GmIQD41</span> </italic> / <italic> <span class="gene">GmIQD47</span> </italic> had no detectable linkage with each other. <span class="trait">High</span> level of microsynteny exists in Clade 3, with most pairs in reverse order ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5c </xref> ), especially <italic> <span class="gene">GmIQD42</span> </italic> / <italic> <span class="gene">GmIQD46</span> </italic> , <italic> <span class="gene">GmIQD45</span> </italic> / <italic> <span class="gene">GmIQD54</span> </italic> and <italic> <span class="gene">GmIQD25</span> </italic> / <italic> <span class="gene">GmIQD44</span> </italic> . <italic> <span class="gene">GmIQD45</span> </italic> / <italic> <span class="gene">GmIQD44</span> </italic> and <italic> <span class="gene">GmIQD44</span> </italic> / <italic> SlSUN33 </italic> were identified as having same-direction microsynteny. In clade 4 ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5d </xref> ), we also observed a higher level of microsynteny. Except for <italic> AtIQD2/SlSUN29 </italic> , <italic> SlSUN30 </italic> / <italic> SlSUN29 </italic> and <italic> <span class="gene">GmIQD67</span> </italic> / <italic> <span class="gene">GmIQD36</span> </italic> , which are aligned in the opposite direction, most gene pairs in this clade have successive collinearity in order and the same orientation.
					</p>
				
					<p>&nbsp;&nbsp;
						Most soybean IQD genes exhibit broad expression patterns ( <xref ref-type="fig" rid="pone-0110896-g006"> Figure 6 </xref> ). Forty-four soybean IQDs are expressed in all of the seven tissues (young leaves, flowers, one-cm pod, pod-shell, roots, nodules and seed). The heat map also revealed that the majority of <span class="gene">GmIQDs</span> showed preferential expression. Based on a hierarchical clustering analysis, fifty-nine IQD genes were mainly clustered into four groups (A&#x02013;D) ( <xref ref-type="fig" rid="pone-0110896-g006"> Figure 6 </xref> ). Group A showed partial expression in young leaves, group B in roots, group C in nodules and group D in flowers. Eight <span class="gene">GmIQDs</span> ( <italic> <span class="gene">GmIQD8</span> </italic> , - <italic> 13 </italic> , - <italic> 19 </italic> , - <italic> 29 </italic> , - <italic> 36 </italic> , - <italic> 52 </italic> , - <italic> 64 </italic> and - <italic> 67 </italic> ) showed marked <span class="trait">high</span> transcript abundance profiles in only a single tissue. Among the fifty-nine soybean IQD genes examined, six showed the highest transcript accumulation in young leaves ( <italic> <span class="gene">GmIQD13</span> </italic> , - <italic> 18 </italic> , - <italic> 19 </italic> , - <italic> 29 </italic> , - <italic> 50 </italic> and <italic> -61 </italic> ), six in flowers ( <italic> <span class="gene">GmIQD11</span> </italic> , - <italic> 15 </italic> , - <italic> 38 </italic> , - <italic> 52 </italic> -6 <italic> 5 </italic> and - <italic> 67 </italic> ), one in roots ( <italic> <span class="gene">GmIQD26</span> </italic> ) and two in nodules ( <italic> <span class="gene">GmIQD8</span> </italic> and - <italic> 64 </italic> ; <xref ref-type="fig" rid="pone-0110896-g006"> Figure 6 </xref> ). Genes in different subfamilies have their primary abundant transcripts, for instance, <span class="gene">GmIQD</span> I in leaves, flowers and nodules, <span class="gene">GmIQD</span> II in flowers and roots, <span class="gene">GmIQD</span> III in young leaves, flowers and roots and <span class="gene">GmIQD</span> IV in roots and young leaves ( <xref ref-type="fig" rid="pone-0110896-g006"> Figure 6 </xref> ). These subfamily-specific tissue expression patterns may be closely related to gene functions. The expression patterns of the paralogous pairs were also revealed by heat maps; paralogous pairs with <span class="trait">high</span> sequence similarity have similar expression patterns. The best examples of this include <italic> <span class="gene">GmIQD8</span> </italic> /- <italic> 64 </italic> and <italic> <span class="gene">GmIQD6</span> </italic> /- <italic> 26 </italic> , which are strongly expressed in nodules and root respectively, with little or no expression in other tissues. Expression divergence was also found in paralogous pairs. For example, <italic> <span class="gene">GmIQD15</span> </italic> is highly expressed in flowers, while its paralog, <italic> <span class="gene">GmIQD59</span> </italic> , is highly expressed in nodules.
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 31 soybean IQD paralogs examined, 27 exhibit highly conserved exon-intron structures, which is consistent with the <span class="trait">high</span> degree of position and phase conservation broadly found across angiosperms <xref rid="pone.0110896-Schmutz1" ref-type="bibr"> [41] </xref> . In addition, the sizes of related introns between paralogs are also highly conserved, indicating that few insertions and deletions have accumulated within introns over the past 13 million years <xref rid="pone.0110896-Schmutz1" ref-type="bibr"> [41] </xref> . Most introns in <span class="gene">GmIQD</span> genes are in phase-0. This strong bias for phase-0 introns in soybean IQD genes is also found in IQD genes of <italic> Arabidopsis </italic> , rice and <italic> Brachypodium distachyon </italic> . The strong bias for one intron phase class, along with the variation in the number of exons (two to six) and the sizes of encoded proteins, suggests that exon shuffling has played a prominent role during the evolution and diversification of IQD genes <xref rid="pone.0110896-Abel1" ref-type="bibr"> [30] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The duplication of individual genes, chromosomal segments or entire genomes has been a major force in the evolution of plant genome structure and content during the process of genome evolution <xref rid="pone.0110896-Du2" ref-type="bibr"> [60] </xref> , <xref rid="pone.0110896-Ohno1" ref-type="bibr"> [61] </xref> . The soybean genome has undergone at least two round of duplication, resulting in the presence of significant features of remnants of a glycine-specific genome duplication that occurred &#x0223c;13 Mya and fainter remnants of older polyploidies prior to the divergence of the papilionoids (58&#x02013;60 Mya) that occurred &#x0223c;58 Mya <xref rid="pone.0110896-Schmutz1" ref-type="bibr"> [41] </xref> , <xref rid="pone.0110896-Cannon2" ref-type="bibr"> [51] </xref> . Thus, 75% of soybean genes are present in multiple copies <xref rid="pone.0110896-Schmutz1" ref-type="bibr"> [41] </xref> . Among 67 soybean IQD genes, <italic> <span class="gene">GmIQD3</span> </italic> , - <italic> 32 </italic> , - <italic> 56 </italic> and - <italic> 60 </italic> were found as single copies on duplication blocks. These results suggest that segmental duplication has occurred as a continuous process and dynamic changes may have occurred in a chromosomal segment that contained two ancestral IQD genes, leading to corresponding sister gene loss <xref rid="pone.0110896-Schlueter1" ref-type="bibr"> [62] </xref> . One paralogs ( <italic> <span class="gene">GmIQD11</span> </italic> /- <italic> 39 </italic> ) shares 91.2% identity and similar exon/intron organization, but exists outside of any duplicated blocks. this pair might have been produced by retrotransposition. A <span class="trait">high</span> proportion (approximately 96%) of soybean IQD genes reside preferentially in duplicated segments, suggesting that segmental duplications have played a prominent role in the expansion of the soybean IQD gene family. The duplicated IQD genes in soybean have been preferentially retained at the <span class="trait">high</span> rate of 92.5% (62/67), which is distinctly higher than the retention rate (67.3%) of duplicated paralogs in the 1.1-gigabase sequence of the soybean (cv. Williams 82) genome, in which 31,264 genes exist as 15,632 paralog pairs (out of the 46,430 predicted high-confidence genes that were duplicated and retained after the 13-Mya tetraploidy event) <xref rid="pone.0110896-Shoemaker1" ref-type="bibr"> [63] </xref> . The higher retention rate corroborates previous findings that genes involved in signal transduction are preferentially retained following duplications <xref rid="pone.0110896-Dreze1" ref-type="bibr"> [64] </xref> . Our calculation of the duplication dates of the 31 paralogous pairs revealed that all of the segmental duplication events in the soybean IQD family occurred during the recent whole genome duplication event.
					</p>
				
					<p>&nbsp;&nbsp;
						During evolution, eukaryotic genomes have retained genes on corresponding chromosomes (synteny) and in corresponding orders (collinearity) to various degrees. Synteny broadly refers to parallels in gene arrangement in dissimilar genomes. Collinearity, a specific form of synteny, requires genes to occur in largely corresponding orders along the chromosomes of respective genomes. According to the microsynteny analysis, microsynteny relationships among <italic> AtIQD3 </italic> , <italic> AtIQD4 </italic> or <italic> SlSUN13 </italic> with other IQD III members in these three dicot genomes were not observed indicating that either these genes are ancient genes without detectable linkage to other IQD genes or that they were formed through complete transposition and loss of their primogenitors. In addition, three different duplicated chromosomal segments (harboring <italic> AtIQD5 </italic> , <italic> <span class="gene">GmIQD32</span> </italic> and <italic> <span class="gene">GmIQD6</span> </italic> ) that lost their sister IQD genes lack detectable microsynteny relationships to all other IQD III genes in the soybean and <italic> Arabidopsis </italic> genome, respectively. In the four IQD III gene clades, genes from soybean, tomato and <italic> Arabidopsis </italic> exhibit <span class="trait">high</span> levels of microsynteny, which indicates the IQD III genes existed before the divergence of the three dicotyledons genomes (soybean, tomato and <italic> Arabidopsis </italic> ). Microsynteny was detected in most pairs, and alignment in clade 1&#x02013;3 was discordant, suggesting that these genes may all be present in genome regions that were inverted, expanded or contracted after the divergence. Notably, most gene pairs in clade 4 have successive collinearity in order and the same orientation, which indicates <span class="trait">high</span> conservation among these IQD III gene-residing regions, with little rearrangement. The low (18.41%) synteny quality of IQD III genes from the three dicotyledon genomes (soybean, tomato and <italic> Arabidopsis </italic> ) may have been due to the fact that these plants are not closely related; moreover, the gene density differs between <italic> Arabidopsis </italic> and the two other species. Significantly, the number of synteny blocks (31) within the soybean genome is much more than the number (3 or 4) of synteny blocks between tomato or <italic> Arabidopsis </italic> genomes, which suggests that soybean IQD III genes may have undergone large-scale duplication events and less rearrangement was followed ( <xref ref-type="fig" rid="pone-0110896-g005"> Figure 5 </xref> and <xref ref-type="supplementary-material" rid="pone.0110896.s007"> Table S5b </xref> ). The gene expansion pattern analysis of soybean paralogs indicates that most pairs were generated from a large-scale duplication, which supports the results of soybean gene duplication analysis, with the exception of <italic> <span class="gene">GmIQD41</span> </italic> /- <italic> 47 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Except for group C comprised of genes from <span class="gene">GmIQD</span> I, group A, B and D comprise genes from four subfamilies indicating these soybean IQD genes exhibit similar transcript abundance profiles but are relatively phylogenetically distinct. The analysis indicated that only some members within the same phylogenetic subgroup share a similar expression profile in soybean organs/tissues during development, excluding <italic> <span class="gene">GmIQD6</span> </italic> and <italic> <span class="gene">GmIQD26</span> </italic> belong to <span class="gene">GmIQD</span> IIb. For instance, <italic> <span class="gene">GmIQD4</span> </italic> , - <italic> 34 </italic> , - <italic> 38 </italic> and - <italic> 65 </italic> belong to <span class="gene">GmIQD</span> Ib clustered in group with <span class="trait">high</span> expression in flowers suggesting their potential roles in flower formation. While the other two <span class="gene">GmIQD</span> Ib members ( <italic> <span class="gene">GmIQD8</span> </italic> and - <italic> 64 </italic> ) were detected in nodules indicating they may involve in fixing atmospheric nitrogen.
					</p>
				
					<p>&nbsp;&nbsp;
						Members possessing similar sequences are clustered in the same subfamilies, which may have similar expression patterns or functions. In IQD subgroup Ia, <italic> Arabidopsis IQD22 </italic> is involved in the negative feedback regulation of GA-responsive DELLA genes <xref rid="pone.0110896-Zentella1" ref-type="bibr"> [36] </xref> . Subgroup Ia members of <italic> Arabidopsis IQD26 </italic> has higher expression level in parts with divided vigorous growth and microtubule organization of leaves, root and flowers <xref rid="pone.0110896-Su-Juan1" ref-type="bibr"> [65] </xref> . Eight of the of the thirteen soybean IQD Ia members ( <italic> <span class="gene">GmIQD3</span> </italic> , - <italic> 16 </italic> , - <italic> 22 </italic> ,- <italic> 27 </italic> , - <italic> 29 </italic> , - <italic> 31 </italic> - <italic> 37 </italic> and - <italic> 66 </italic> ) have <span class="trait">high</span> expression in young fleaves. And six of them ( <italic> <span class="gene">GmIQD15</span> </italic> , - <italic> 16 </italic> , - <italic> 27 </italic> , - <italic> 31 </italic> - <italic> 37 </italic> and - <italic> 66 </italic> ) have <span class="trait">high</span> expression in one-cm pod or pod-shell 10-DAF. <italic> <span class="gene">GmIQD22</span> </italic> have obviously higher expression at seed 21-DAF, the period of seed cell division (3, 4 weeks after flowering) <xref rid="pone.0110896-Hajduch1" ref-type="bibr"> [66] </xref> . These founding suggested that soybean IQD Ia members may function in transport of signaling molecules, nutrient transport and cell division.
					</p>
				
					<p>&nbsp;&nbsp;
						Mapping and positional cloning of the <italic> SUN </italic> locus revealed that this member of the IQD II subfamily was generated by duplication of a 24.7-kb region carrying the tomato <italic> IQD12 </italic> gene, a major gene involved in the control of fruit shape, particularly length, in tomato <xref rid="pone.0110896-Huang1" ref-type="bibr"> [35] </xref> . <italic> SUN </italic> is expressed the highest in hypocotyl and shoot apex. Overexpression of <italic> SUN </italic> causes root reduction when applied auxin and prostrate growth and twisted stems indicating that <italic> SUN </italic> can affect auxin transport or response <xref rid="pone.0110896-Wu1" ref-type="bibr"> [38] </xref> . <italic> SlSUN1 </italic> shows slightly higher expression in the hypocotyl, flower at anthesis and fruit at 10 and 20 DPA <xref rid="pone.0110896-Huang1" ref-type="bibr"> [35] </xref> . <italic> SlSUN12 </italic> and <italic> SlSUN21 </italic> highly are expressed in the hypocotyls and root respectively <xref rid="pone.0110896-Huang1" ref-type="bibr"> [35] </xref> . <italic> SlSUN17 </italic> evenly expressed in almost all tissues <xref rid="pone.0110896-Huang1" ref-type="bibr"> [35] </xref> . Soybean IQD II members showed the similar expression profile ( <italic> <span class="gene">GmIQD11</span> </italic> : highest in flowers; <italic> <span class="gene">GmIQD33</span> </italic> , - <italic> 39 </italic> and - <italic> 40 </italic> : slight <span class="trait">high</span> in flowers and <italic> <span class="gene">GmIQD6</span> </italic> and - <italic> 26 </italic> : highest in roots) indicating that <span class="gene">GmIQD</span> II members may play similar role in plant development.
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple sequence alignment of all predicted soybean IQD <span class="trait">protein</span> sequences was performed with Clustal X2.0 software using default parameters. Then, based on this alignment, phylogenetic trees were constructed using Clustal X2.0 with the Neighbor-Joining (NJ) method, and bootstrap analysis was conducted using 1,000 replicates <xref rid="pone.0110896-Hu1" ref-type="bibr"> [76] </xref> . An unrooted NJ tree of 184 the full-length IQD <span class="trait">protein</span> sequences from soybean, rice, <italic> Arabidopsis </italic> , tomato, <italic> Brachypodium </italic> was constructed using Clustal X2.0 with one moss IQD <span class="trait">protein</span> (Pp1s38230v6.1) as the outgroup. The <span class="gene">GmIQD</span> genes were classified according to their phylogenetic relationships with the corresponding <italic> Arabidopsis </italic> and rice IQD genes. For microsynteny analysis of IQD III genes across soybean, tomato and <italic> Arabidopsis </italic> , a phylogenetic tree was constructed using MEGA 5.0 with the NJ method and bootstrap analysis was conducted using 1,000 replicates.
					</p>
				
					<p>&nbsp;&nbsp;
						Online MEME (Multiple Expectation Maximization for Motif Elicitation) ( <ext-link ext-link-type="uri" xlink:href="http://meme.sdsc.edu/meme4_4_0/intro.html"> http://meme.sdsc.edu/meme4_4_0/intro.html </ext-link> ) was performed to identify the conserved motif structures encoded by <span class="gene">GmIQD</span> genes. The parameters were as followings: number of repetitions - any, maximum number of motifs -10, and the optimum motif width was constrained to between 6 and 200 residues. In addition, each structural motif was annotated using the Pfam ( <ext-link ext-link-type="uri" xlink:href="http://pfam.sanger.ac.uk/search"> http://pfam.sanger.ac.uk/search </ext-link> ) and SMART ( <ext-link ext-link-type="uri" xlink:href="http://smart.embl-heidelberg.de/"> http://smart.embl-heidelberg.de/ </ext-link> ) tools. All IQD <span class="trait">protein</span> sequences were examined against the Calmodulin Target Database ( <ext-link ext-link-type="uri" xlink:href="http://calcium.uhnres.utoronto.ca/ctdb/ctdb/home.html"> http://calcium.uhnres.utoronto.ca/ctdb/ctdb/home.html </ext-link> ) to predict putative calmodulin-binding sites.
					</p>
				
					<p>&nbsp;&nbsp;
						The chromosomal location image of <span class="gene">GmIQD</span> genes was generated by MapInspect ( <ext-link ext-link-type="uri" xlink:href="http://www.plantbreeding.wur.nl/uk/software_mapinspect.html"> http://www.plantbreeding.wur.nl/uk/software_mapinspect.html </ext-link> ) according to chromosomal position information provided in the Phytozome database. To identify tandem and segmental duplications, two genes in the same species located in the same clade of the phylogenetic tree were defined as coparalogs. The SoyBase browser ( <ext-link ext-link-type="uri" xlink:href="http://soybase.org/gb2/gbrowse/gmax1.01/"> http://soybase.org/gb2/gbrowse/gmax1.01/ </ext-link> ) <xref rid="pone.0110896-Grant1" ref-type="bibr"> [78] </xref> was queried to detect the segmental duplication coordinates of the target genes. Coparalogs were deemed to result from segmental duplication if they were located on duplicated chromosomal blocks <xref rid="pone.0110896-Wei1" ref-type="bibr"> [79] </xref> . Paralogs were deemed to be tandem duplicated genes if two genes were separated by five or fewer genes in a 100-kb region <xref rid="pone.0110896-Wang1" ref-type="bibr"> [80] </xref> . The local alignment of two <span class="trait">protein</span> sequences was calculated using the Smith-Waterman algorithm ( <ext-link ext-link-type="uri" xlink:href="http://www.ebi.ac.uk/Tools/psa/"> http://www.ebi.ac.uk/Tools/psa/ </ext-link> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comprehensive characterization and RNA-Seq profiling of the HD-Zip transcription factor family in soybean ( <italic> Glycine max </italic> ) during dehydration and salt stress</h2>
			<h3>PMID:25362847</h3>
				
					<h3>PMCID:4226900</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Belamkar </surname>  <given-names> Vikas </given-names>  </name>  <address>  <email> belamkar@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff24"/>  <xref ref-type="aff" rid="Aff25"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weeks </surname>  <given-names> Nathan T </given-names>  </name>  <address>  <email> nathan.weeks@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff26"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bharti </surname>  <given-names> Arvind K </given-names>  </name>  <address>  <email> arvindkbharti30@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff27"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farmer </surname>  <given-names> Andrew D </given-names>  </name>  <address>  <email> adf@ncgr.org </email>  </address>  <xref ref-type="aff" rid="Aff27"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Graham </surname>  <given-names> Michelle A </given-names>  </name>  <address>  <email> michelle.graham@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff25"/>  <xref ref-type="aff" rid="Aff26"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Cannon </surname>  <given-names> Steven B </given-names>  </name>  <address>  <email> steven.cannon@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff25"/>  <xref ref-type="aff" rid="Aff26"/>  </contrib>  <aff id="Aff24">  <label/> Interdepartmental Genetics, Iowa State University, Ames, IA 50011 USA  </aff>  <aff id="Aff25">  <label/> Department of Agronomy, Iowa State University, Ames, IA 50011 USA  </aff>  <aff id="Aff26">  <label/> United States Department of Agriculture - Agricultural Research Service, Corn Insects and Crop Genetics Research Unit, Ames, IA 50011 USA  </aff>  <aff id="Aff27">  <label/> National Center for Genome Resources, Santa Fe, NM 87505 USA  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Copy number expansion of the HD-Zip family in the soybean genome has primarily occurred through genome duplication events (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , Additional file <xref rid="MOESM13" ref-type="media"> 13 </xref> : Table S1). Each angiosperm clade in each of the four subfamilies (Figures&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , <xref rid="Fig2" ref-type="fig"> 2 </xref> , <xref rid="Fig3" ref-type="fig"> 3 </xref> and <xref rid="Fig4" ref-type="fig"> 4 </xref> ) contains two to four soybean gene copies that are a result of retention of genes after the legume WGD (~59 Mya) and/or the <italic> Glycine </italic> -specific WGD (~13 Mya). Retention of genes following these WGDs has been <span class="trait">high,</span> with retention of 32 of 36 HD-Zip I (88.9%), 20 of 24 HD-Zip II (83.3%), 10 of 11 HD-Zip III (90.9), and 26 of 30 (86.7%) HD-Zip IV genes (Additional file <xref rid="MOESM13" ref-type="media"> 13 </xref> : Table S1). There are two tandemly duplicated HD-Zip pairs in subfamily III, and another pair in subfamily IV. Phylogenetic patterns indicate that the tandemly duplicated genes in subfamily III further duplicated during a WGD event, giving rise to <span class="gene">Glyma07g01940</span> and <span class="gene">Glyma07g01950</span> on chromosome 07 and <span class="gene">Glyma08g21610</span> and <span class="gene">Glyma08g21620</span> on homoeologous chromosome 8. Genes <span class="gene">Glyma09g02990</span> and <span class="gene">Glyma09g03000,</span> in HD-Zip IV, are another pair of tandemly duplicated genes. The Glycine WGD event resulted in the homoeologous gene pair <span class="gene">Glyma09g03000</span> and Glyma15gg13950, whereas the homoeologous gene for <span class="gene">Glyma09g02990</span> has evidently either been lost following the WGD &#x02013; or the <span class="gene">Glyma09g02990</span> and <span class="gene">Glyma09g03000</span> duplication occurred after the Glycine WGD. Overall, 88 of the 101 HD-Zip genes are members of homoeologous gene pairs in the soybean genome.
					</p>
				
					<p>&nbsp;&nbsp;
						RNA-Seq analysis was utilized to investigate genes involved in dehydration and salt stress. The expression of all soybean genes including the 101 HD-Zip genes identified in this study was studied in the roots of soybean cv. Williams 82 at <span class="gene">V1</span> stage, at four different time points, and under dehydration and salt stress. The evaluation of plants at the <span class="gene">V1</span> stage may assist in identification of candidate genes involved in initiation of dehydration and salt stress. Recently, Chen et al. [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ] reported the influence of <span class="trait">drought</span> and salinity stress on HD-Zip genes using publicly available microarray data sets available at National Center for Biotechnology Information under accession numbers GSE41125 and GSE40627. The microarray datasets facilitated investigation of the expression of 55 of the 88 HD-Zip genes identified in their study. The microarray dataset GSE40627 reports expression of genes in the leaves under <span class="trait">drought</span> stress imposed at late developmental stages (V6 and R2), whereas the dataset GSE41125 describes expression of genes in 14 d seedlings utilizing pooled RNA samples from 0, 3, 6, 12 and 24&#x000a0;hr of mock and salinity stressed plants. Thus, in the current study, the utilization of root tissue at the <span class="gene">V1</span> stage, and investigation of gene expression separately at each of the four time points 0, 1, 6 and 12&#x000a0;hr provided clearly different and more precise insight into genes that are involved in dehydration and salt stress.
					</p>
				
					<p>&nbsp;&nbsp;
						All six genes identified as DE in the roots under dehydration stress in this study, were also, DE under <span class="trait">drought</span> stress in leaves [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Four of the five DE HD-Zip I genes belong to the angiosperm clade A5. This clade contains genes such as <italic> CPHB-5 </italic> from <italic> C. plantagineum </italic> , and <italic> Zmhdz1 </italic> , &#x02212; <italic> 2 </italic> , &#x02212; <italic> 3 </italic> from maize, that have previously been shown to have a role in water-stress response [ <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. Chen et al. [ <xref ref-type="bibr" rid="CR109"> 109 </xref> ] showed <span class="gene">Glyma06g20230</span> DE in this study was DE under dehydration stress, in the roots of drought-tolerant soybean genotype, &#x0201c;Jindou21&#x0201d;.
					</p>
				
					<p>&nbsp;&nbsp;
						The HD-Zip I gene <span class="gene">Glyma01g04890</span> was upregulated at 6 and 12&#x000a0;hr time points under both salt and dehydration stress. This gene was also upregulated under both <span class="trait">drought</span> and salt stress in the leaves and seedlings, respectively, in two microarray experiments [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. A BLASTP search with <span class="gene">Glyma01g04890</span> <span class="trait">protein</span> sequence against the patent database [ <xref ref-type="bibr" rid="CR113"> 113 </xref> , <xref ref-type="bibr" rid="CR114"> 114 </xref> ] found a match (E-value = 0; Similarity &#x0003e;99.4%; Coverage = 100%) with sequences in five &#x0201c;patent applications&#x0201d; (US_2012_0278947_A1; US_2012_0096584_A1; US_2007_0277269_A1; US_2012_0005773_A1; US_2009_0144847_A1) that described the role of this sequence in improving plant performance under abiotic stress.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Abscission of flowers and floral organs is closely associated with alkalization of the cytosol in abscission zone cells</h2>
			<h3>PMID:25504336</h3>
				
					<h3>PMCID:4339595</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sundaresan </surname>  <given-names> Srivignesh </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru483/-/DC1"> Figure S1. </ext-link> Fluorescence micrographs of BCECF images of flower organ AZ of <italic> Arabidopsis </italic> Col WT in <span class="gene">P5</span> flower and of a cross-section of tomato flower pedicel AZ excised 14h after flower removal, showing a <span class="trait">high</span> intensity of green fluorescence in the cytosol.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A draft genome of field pennycress ( <italic> Thlaspi arvense </italic> ) provides tools for the domestication of a new winter biofuel crop</h2>
			<h3>PMID:25632110</h3>
				
					<h3>PMCID:4401323</h3>
					<h4>期刊:DNA Research: An International Journal for Rapid Publication of Reports on Genes and Genomes</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dorn </surname>  <given-names> Kevin M. </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fankhauser </surname>  <given-names> Johnathon D. </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="author-notes" rid="AN1"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wyse </surname>  <given-names> Donald L. </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marks </surname>  <given-names> M. David </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <aff id="af1">  <label> 1 </label>  <addr-line> Department of Plant Biology </addr-line> ,  <institution> University of Minnesota </institution> ,  <addr-line> Saint Paul, MN 55108 </addr-line> ,  <country> USA </country>  </aff>  <aff id="af2">  <label> 2 </label>  <addr-line> Department of Agronomy and Plant Genetics </addr-line> ,  <institution> University of Minnesota </institution> ,  <addr-line> Saint Paul, MN 55108 </addr-line> ,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> A. thaliana </italic> , <italic> B. rapa </italic> , and <italic> Brassica oleraceae </italic> genes previously identified to be involved in glucosinolate biosynthesis, breakdown, and transport were derived from a previous study. <sup> <xref rid="DSU045C4" ref-type="bibr"> 4 </xref> </sup> Alignments of predicted peptide sequences for each gene were performed in CLC Genomics Workbench using the Create Alignment tool (Gap open cost = 10, Gap extension cost = 1, End <span class="gene">gap</span> cost = Free, Alignment mode = Very accurate). Neighbour Joining trees were created in CLC Genomics Workbench <span class="trait">(Protein</span> distance measure = Jukes-Cantor, Perform bootstrap analysis = Yes, Replicates = 100). To examine expression levels of predicted gene models, Illumina RNAseq reads from the previously published transcriptome assembly <sup> <xref rid="DSU045C19" ref-type="bibr"> 19 </xref> </sup> were trimmed and filtered (Illumina TruSeq Trim Adaptor 3, Ambiguous Trim = Yes, Ambiguous Limit = 2, Quality Trim = Yes, Quality Limit = 0.05, Also search on reversed sequence = Yes, Remove 5&#x02032; terminal nucleotides = Yes, Number of 5&#x02032; terminal nucleotides to remove = 10, Remove 3&#x02032; terminal nucleotides = no, Save broken pairs = Yes), and mapped to the annotated draft genome (Mapping Type = Map to gene regions only, Maximum number of hits for a read = 10, Strand Specific = Both, Count paired read as two = Yes, Expression Value = RPKM, Reference type = Genome annotated with genes and transcripts, Global alignment = no, Auto detect paired distances = Yes, Similarity fraction = 0.8, Length Fraction = 0.8, Mismatch cost = 2, Insertion cost = 3, Deletion cost = 3).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A systems biology approach toward understanding seed composition in soybean</h2>
			<h3>PMID:25708381</h3>
				
					<h3>PMCID:4331812</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Li </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> liling@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Hur </surname>  <given-names> Manhoi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lee </surname>  <given-names> Joon-Yong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Zhou </surname>  <given-names> Wenxu </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Song </surname>  <given-names> Zhihong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Ransom </surname>  <given-names> Nick </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Demirkale </surname>  <given-names> Cumhur Yusuf </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Nettleton </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Westgate </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Arendsee </surname>  <given-names> Zebulun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Iyer </surname>  <given-names> Vidya </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Shanks </surname>  <given-names> Jackie </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  </contrib>  <contrib contrib-type="author" id="A13">  <name>  <surname> Nikolau </surname>  <given-names> Basil </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A14">  <name>  <surname> Wurtele </surname>  <given-names> Eve Syrkin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> mash@iastate.edu </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The flux of metabolites was determined in isolated cotyledons in medium containing <sup> 13 </sup> C [ <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> ] (Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> : Tables S5a and S5b). The flux represents the pathway dynamics in the cotyledons at the quasi-steady-state developmental stage of early seed fill (21-26 DAF). The metabolic flux map representing central carbon metabolism is visualized in Figure <xref ref-type="fig" rid="F5"> 5A </xref> . The metabolic reaction network in early seed-fill soybean cotyledons consists of parallel pathways of glycolysis and oxidative pentose phosphate pathway (oxPPP) in the cytosol and the plastid. Flux through the &#x003b3;-aminobutyric acid (GABA) shunt was quite <span class="trait">high,</span> as was the flux through glycolysis, the TCA cycle, and oxPPP. The <span class="trait">high</span> flux through the GABA shunt may reflect the need to shuttle glutamine and &#x003b1;-ketoglutarate between mitochondria (where the TCA cycle is located), and the plastids (the site of glutamine assimilation). Our analyses of the transcriptomics and metabolomics data identified pathways that were significantly changed during seed development (Additional file <xref ref-type="supplementary-material" rid="S3"> 3 </xref> : Table S3) among those pathways identified as having a <span class="trait">high</span> flux. For example, in the transcriptomics data the TCA cycle and glycolysis were over-represented by Fisher's Exact test, with representative genes of these pathways decreasing during seed development (Clusters T5 and T6), and the MRPP test indicated these processes were significantly changed. Similarly, levels of metabolites in the TCA cycle and that of glutamate were altered. The flux through non-oxidative reactions of the oxPPP catalyzed by the transketolase ( <italic> <span class="gene">tkt</span> </italic> ) and transaldolase ( <italic> <span class="gene">tal</span> </italic> ) enzymes was much greater in the plastid as compared to the comparable cytosolic reactions. <italic> F16BPase </italic> , a key reaction of gluconeogenesis, was also higher in the plastid than the cytosol. The relative flux rate through a number of amino acid biosynthetic pathways (e.g., serine, glycine and cysteine) was comparatively low at this stage of seed development; this may indicate that these amino acids are formed predominantly via interconversion from other intermediates.
					</p>
				
					<p>&nbsp;&nbsp;
						We integrated these complex datasets to interpret metabolic changes associated with seed composition, highlighting as examples, the metabolism of asparagine, starch, and fatty acids. The amino acid asparagine is the major form of nitrogen imported from the vegetative soybean plant by the developing embryo and interconverted to other amino acids to support <span class="trait">protein</span> synthesis; it is therefore considered to be important to <span class="trait">protein</span> content in mature seeds [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. We evaluated this aspect of metabolism using PMR. Volcano plots comparing the metabolomics of seeds at 25 and 50 DAF indicate that free amino acids are more abundant in 25 DAF seeds (also reported for several amino acids in Collakova <italic> et al </italic> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]), while sterols are greatly increased in 50 DAF seeds (Figure <xref ref-type="fig" rid="F6"> 6A </xref> ). As indicated by Figure <xref ref-type="fig" rid="F2"> 2C </xref> , Asn was the most abundant free amino acid in the developing Evans seeds but decreased quickly after 30 DAF. Co-analysis of the metabolomics and transcriptomic data indicated that Asn concentration varied during seed development and that this pattern strongly correlated with one of the six transcripts computationally predicted to encode an asparaginase-like gene (soybean locus ID: <span class="gene">Glyma05g07420;</span> Figure <xref ref-type="fig" rid="F6"> 6B </xref> ). These data lead us to predict that the <span class="gene">Glyma05g07420</span> gene encodes the particular asparaginase enzyme that is associated with amino acid interconversion during seed fill.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Interactive analysis of metabolomics and transcriptomics data using PMR </bold> . <bold> A </bold> . Volcano plot (p-value vs. fold change) comparison of metabolites in 25 DAF versus 50 DAF seeds in PMR <ext-link ext-link-type="uri" xlink:href="http://metnetdb.org/PMR/"> http://metnetdb.org/PMR/ </ext-link> . Clicking or mousing-over a metabolite brings up its annotation information. The data indicate that almost every amino acid is more abundant in 25 DAF seeds than 50 DAF seeds, while almost all sterols are more abundant in 50 DAF seeds than in 25 DAF seeds. <bold> B </bold> . Transcript <span class="gene">Glyma05g07420.1</span> has a <span class="trait">high</span> correlation with asparagine accumulation during seed development (Pearson Correlation&#x0003e;0.86; p-value = 0.06). There are six genes in the soybean genome annotated as having sequence similarity to known asparaginase; of these, only <span class="gene">Glyma05g07420.1</span> has an expression pattern similar to that of asparagine. These data indicate that <span class="gene">Glyma05g07420.1</span> is a likely candidate for encoding the asparaginase activity associated with <span class="trait">protein</span> synthesis during seed fill. <bold> C </bold> . Other over-represented pathways in the set of genes that have a <span class="trait">high</span> correlation with asparagine accumulation. <bold> D </bold> . Asparaginase reaction in soybean visualized in PMR via Cytoscape [ <xref ref-type="bibr" rid="B62"> 62 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification and Evolution of HECT Genes in Soybean</h2>
			<h3>PMID:25894222</h3>
				
					<h3>PMCID:4425094</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Meng </surname>  <given-names> Xianwen </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Chen </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rahman </surname>  <given-names> Siddiq Ur </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yaxu </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ailan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tao </surname>  <given-names> Shiheng </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-16-08517"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-16-08517"> 2 </xref>  <xref rid="c1-ijms-16-08517" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ubiquitin-proteasome system (UPS) plays a crucial role in plant growth, development, and response to environmental stress [ <xref rid="B1-ijms-16-08517" ref-type="bibr"> 1 </xref> , <xref rid="B2-ijms-16-08517" ref-type="bibr"> 2 </xref> , <xref rid="B3-ijms-16-08517" ref-type="bibr"> 3 </xref> , <xref rid="B4-ijms-16-08517" ref-type="bibr"> 4 </xref> , <xref rid="B5-ijms-16-08517" ref-type="bibr"> 5 </xref> , <xref rid="B6-ijms-16-08517" ref-type="bibr"> 6 </xref> , <xref rid="B7-ijms-16-08517" ref-type="bibr"> 7 </xref> ]. The ubiquitination pathway consists of an enzymatic cascade mediated by three sequential enzymes: <span class="gene">E1</span> ubiquitin activating enzyme (E1), E2 ubiquitin conjugating enzyme (E2), and E3 ubiquitin ligase (E3) [ <xref rid="B8-ijms-16-08517" ref-type="bibr"> 8 </xref> , <xref rid="B9-ijms-16-08517" ref-type="bibr"> 9 </xref> , <xref rid="B10-ijms-16-08517" ref-type="bibr"> 10 </xref> , <xref rid="B11-ijms-16-08517" ref-type="bibr"> 11 </xref> ]. During the ubiquitination process, the specificity of the selective proteolysis by UPS is usually determined by E3s, which targets substrate proteins with different substrate recognition domains for ubiquitylation [ <xref rid="B4-ijms-16-08517" ref-type="bibr"> 4 </xref> , <xref rid="B12-ijms-16-08517" ref-type="bibr"> 12 </xref> ]. In plants, E3s can be classified into three main types according to differences in their action mechanisms, and the presence of specific domains [ <xref rid="B13-ijms-16-08517" ref-type="bibr"> 13 </xref> , <xref rid="B14-ijms-16-08517" ref-type="bibr"> 14 </xref> , <xref rid="B15-ijms-16-08517" ref-type="bibr"> 15 </xref> , <xref rid="B16-ijms-16-08517" ref-type="bibr"> 16 </xref> , <xref rid="B17-ijms-16-08517" ref-type="bibr"> 17 </xref> , <xref rid="B18-ijms-16-08517" ref-type="bibr"> 18 </xref> , <xref rid="B19-ijms-16-08517" ref-type="bibr"> 19 </xref> , <xref rid="B20-ijms-16-08517" ref-type="bibr"> 20 </xref> ]: homologous to the E6-associated <span class="trait">protein</span> (E6-AP) carboxyl terminus (HECT), really interesting new gene (RING), and U-box.
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of the expression patterns of these soybean genes in 14 tissues showed that most HECT genes were relatively highly expressed in flowers and roots. However, <italic> <span class="gene">Glyma06g00600</span> </italic> and <italic> <span class="gene">Glyma17g01210</span> </italic> were highly expressed in the nodules. From this, we inferred that the highly expressed HECT genes in the flowers may be involved in the degradation of genes relating to flowering, via ubiquitination during soybean flowering stage. Additionally, the results suggested that the highly expressed genes in roots and nodules may directly or indirectly control the expression of nitrogen-fixing genes during symbiotic nitrogen fixation. Previous studies have revealed that <italic> Arabidopsis thaliana </italic> <italic> AT4G38600/UPL3 </italic> restricts the rounds of genome endoreduplication and cell <span class="trait">branching</span> that occur during trichome development [ <xref rid="B7-ijms-16-08517" ref-type="bibr"> 7 </xref> ], and <italic> AT4G12570 </italic> / <italic> UPL5 </italic> regulates leaf senescence through the degradation of <italic> AT4G23810 </italic> / <italic> WRKY53 </italic> , a transcription factor that acts positively in leaf senescence [ <xref rid="B3-ijms-16-08517" ref-type="bibr"> 3 </xref> ]. In our analysis, the soybean genes orthologous to <italic> Arabidopsis thaliana </italic> <italic> AT4G38600 </italic> / <italic> UPL3 </italic> are four paralogous genes in group VI. These four genes were all expressed in soybean, but display different expression patterns in different tissues. Except for <italic> <span class="gene">Glyma06g00600</span> </italic> , which is expressed relatively highly in nodules, the other three genes are relatively highly expressed in roots and flowers. This may be caused by mutations accumulated after the two segmental duplication events, especially the latest duplication events. The soybean genes orthologous to <italic> Arabidopsis thaliana </italic> <italic> AT4G12570 </italic> / <italic> UPL5 </italic> are two paralogous genes in group III. <italic> <span class="gene">Glyma17g01210</span> </italic> was also highly expressed in nodules, while <italic> Glyma7g39546 </italic> was not expressed. The differential expression of paralogous genes of the same group indicates that the HECT genes in soybean may have the same or similar function as their orthologues in <italic> Arabidopsis thaliana </italic> ; however, they may have evolved functional differences.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of the Chinese Herbal Formulation ( <italic> Liu Wei Di Huang Wan </italic> ) on the Pharmacokinetics of Isoflavones in Postmenopausal Women</h2>
			<h3>PMID:26146635</h3>
				
					<h3>PMCID:4471402</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Limopasmanee </surname>  <given-names> Wirin </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 30 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The LWDHW used in this study was manufactured by the Faculty of Pharmacy, Chiang Mai University, Thailand. It consisted of 24% Radix Rehmannia Preparata, 12% each of Fructus Corni and Rhizoma Dioscoreae, 9% each of Rhizoma Alismatis, Cortex Moutan, and Poria as well as 25% honey. All six crude drugs were purchased from Vejpong Pharmacy Co., Ltd (Bangkok, Thailand). Raw materials used in this formula were evaluated following <italic> Chinese Pharmacopoeia 2005 </italic> [ <xref rid="B18" ref-type="bibr"> 18 </xref> ]. Chemical control of LWDHW was analyzed by <span class="trait">high</span> performance liquid chromatography coupled with photodiode array (HPLC-PDA) using loganin as the active marker. The standardized crude drugs of LWHDW were ground into a fine powder and mixed with heated honey to make boluses approximately 4&#x02009;mm in diameter. Quality control of the honey pill (i.e., assessment of weight variation, disintegration, moisture content, and microbial contamination) was conducted following WHO guidelines on good manufacturing practices <span class="gene">(GMP)</span> for herbal medicines [ <xref rid="B19" ref-type="bibr"> 19 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Comparison of transgenic and wild type soybean seeds: analysis of transcriptome profiles using RNA-Seq</h2>
			<h3>PMID:26427366</h3>
				
					<h3>PMCID:4591623</h3>
					<h4>期刊:BMC Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lambirth </surname>  <given-names> Kevin C. </given-names>  </name>  <address>  <email> kclambirth@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Whaley </surname>  <given-names> Adam M. </given-names>  </name>  <address>  <email> awhaley9@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Blakley </surname>  <given-names> Ivory C. </given-names>  </name>  <address>  <email> ieclabau@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schlueter </surname>  <given-names> Jessica A. </given-names>  </name>  <address>  <email> jschluet@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bost </surname>  <given-names> Kenneth L. </given-names>  </name>  <address>  <email> klbost@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Loraine </surname>  <given-names> Ann E. </given-names>  </name>  <address>  <email> Ann.Loraine@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Piller </surname>  <given-names> Kenneth J. </given-names>  </name>  <address>  <email> kjpiller@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Biological Sciences, University of North Carolina at Charlotte, Charlotte, NC 28223 USA  </aff>  <aff id="Aff2">  <label/> Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, Charlotte, NC 28223 USA  </aff>  <aff id="Aff3">  <label/> Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, North Carolina Research Campus, Kannapolis, NC 28081 USA  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seeds from each transgenic event as well as from wild type were germinated in moistened soil in 6-pack planting trays. Following germination, plants were propagated in Scott&#x02019;s 6-month nutrient Miracle Grow potting mix with 16&#x000a0;h light (26&#x000a0;&#x000b0;C) and 8&#x000a0;h night cycles (20&#x000a0;&#x000b0;C) in controlled growth chambers with ~50&#x000a0;% relative humidity. Plants were watered every other day or as needed if the soil was observed to be dry, and were transferred to 4-in. pots (Dillen Greenhouse, 4.00 Square Traditional) at 3&#x000a0;weeks of age and then 1.5 <span class="gene">gal</span> pots (Nursery Supplies Inc. C600) at 6&#x000a0;weeks of age. Light intensities were measured at ~500&#x02013;550 &#x003bc;E m <sup> &#x02212;2 </sup> s <sup> &#x02212;1 </sup> . Three plants were chosen from each genotype, which were all phenotypically identical to wild type plants with respect to overall size, leaf structure, and approximate seed <span class="trait">yield.</span> Dried pods were collected following senescence and fully matured dry seeds at the final R8 stage of development were removed and used for molecular characterization and transcriptome sequencing. Three seeds from each plant were collected and processed individually, generating three biological replicates from each plant, and three biological replicates from each construct. ST77, 764, and ST111 seeds were obtained from T7, T4, and T3 generation transgenic plants respectively, and were stored in individual seed bags at 23&#x000a0;&#x000b0;C and 50&#x000a0;% relative humidity until processing. In total, nine seeds were chosen from each transgenic event and from wild type for a total of 36 samples (See Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Cufflinks software was used in addition to edgeR to investigate differential expression and revealed similar differences in gene expression. Cufflinks reported 47 upregulated and 28 downregulated genes in ST77, 744 upregulated and 361 downregulated genes in ST111 and 1249 upregulated and 843 downregulated genes in 764. Volcano plots were constructed from the results and are shown in Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> . These plots show the relationship between fold change and statistical significance of differentially expressed genes. Note that there is &#x0003e;20-fold difference in the number of differentially expressed (DE) genes between ST77 and 764. Thus, it is clear from both the edgeR and cufflinks results that while there were significant differences in all three transgenic events, differences were the most substantial in line 764 relative to the wild type controls. The results of these two programs are illustrated in Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> . The Venn diagrams (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a </xref> &#x02013; <xref rid="Fig5" ref-type="fig"> c </xref> ) indicate the number of up and downregulated genes identified by each program separately and together, while the <span class="gene">bar</span> chart (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5d </xref> ) shows the total number of upregulated and downregulated genes as well as the portion of shared genes identified from each program. Five genes were differentially expressed in all three transgenic lines, including <span class="gene">Glyma.12G136600</span> <span class="trait">(protein</span> kinase), <span class="gene">Glyma.13G171200</span> (ribosomal RNA protein-7 related), <span class="gene">Glyma.01G103100</span> (branched chain alpha-keto acid decarboxylase <span class="gene">E1</span> beta subunit), and two genes with no functional annotation information <span class="gene">(Glyma.07G207000,</span> <span class="gene">Glyma.13G011800).</span> <span class="gene">Glyma.01G103100</span> and <span class="gene">Glyma.13G171200</span> showed no commonality between the three events in the direction of altered expression; however <span class="gene">Glyma.01G103100,</span> <span class="gene">Glyma.07G207000,</span> and Glyma 13.G011800 were upregulated in all three transgenics. Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5e </xref> shows the number of common DE genes shared between each of the three lines based on the edgeR results. A list of all shared differentially expressed genes between all events is available in the git repository file &#x0201c;Diffexpoverlap&#x0201d; under the &#x0201c;DiffExp&#x0201d; directory. <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> Cufflinks plots of differentially expressed genes. ( <bold> a </bold> - <bold> c </bold> ) Cufflinks volcano plots for each transgenic event showing variances in gene expression with respect to fold-change and significance. Each dot represents an individual gene. Black dots represent genes that are not significantly differentially expressed while red dots represent genes that are significantly differentially expressed
					</p>
				
					<p>&nbsp;&nbsp;
						We next performed a gene ontology (GO) enrichment analysis using GOseq [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ] which accounts for selection biases in RNA-Seq data in which larger, more highly expressed transcripts are preferentially detected as differentially expressed. In line 764 we detected at least one read sequence from ~42,000 of the 56,000 annotated soybean genes, and of these ~42,000 expressed genes, approximately 3800 (9&#x000a0;%) were differentially expressed. The input list consisted of approximately 1500 genes that were considered differentially expressed after combining the lists from both edgeR and cufflinks. Thus, on average, we expected that approximately 3.5&#x000a0;% of genes in any random sample of expressed genes would be differentially expressed. However, there were several GO categories that exceeded this 3.5&#x000a0;% threshold and are grouped according to their parent terms in Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> . A more detailed flowchart of all GO terms can be found in Additional file 1: Figure S3. Of 16 genes annotated to the term &#x0201c;nuclear pore&#x0201d;, nine were differentially expressed, and all were downregulated. Of the 490 genes annotated to the term &#x0201c;structural constituent of ribosome&#x0201d;, 47 were differentially expressed, and of these 94&#x000a0;% were upregulated. All DE genes annotated as protease inhibitors were upregulated, including 8 of 19 genes encoding serine-type endopeptidase inhibitors, and 10 of 60 genes encoding peptidase inhibitor and regulator activity. Intracellular transport also appeared affected in the 764 samples, as 8&#x000a0;% annotated to non-membrane intracellular organelles were differentially expressed and most (82&#x000a0;%) were upregulated. All 10 DE genes encoding mitochondrial function were also upregulated. In addition, several genes (5 of 8) annotated with the biological process term &#x0201c;response to wounding&#x0201d; were upregulated. Taken together, these results suggested that <span class="trait">protein</span> synthesis was more active in the 764 seeds as compared to the nontransgenic controls. These results also suggest that aspects of intracellular transport and nuclear pore structures may be altered. The annotation of upregulated genes involved in wounding responses and peptidase inhibitors suggests that some aspects of a physical stress response may have been activated. <fig id="Fig7"> <label> Fig. 7 </label> <caption> <p> GOseq results from the edgeR and cufflinks merged DE gene list. The numbers over each <span class="gene">bar</span> indicate the total number of differentially expressed (DE) genes in each category. The length and direction of the arrow in each <span class="gene">bar</span> indicate how many of the genes were up (arrows pointing up) or down (arrows pointing down) regulated in that category. The red dashed line indicates the percentage of all GO-annotated genes that are DE from edgeR and cufflinks in this line (~4&#x000a0;%). All terms shown have an FDR of 0.05 or smaller
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole Soy Flour Incorporated into a Muffin and Consumed at 2 Doses of Soy <span class="trait">Protein</span> Does Not Lower LDL Cholesterol in a Randomized, Double-Blind Controlled Trial of Hypercholesterolemic Adults <xref ref-type="author-notes" rid="fn1"> <sup> 1 </sup> </xref> <xref ref-type="author-notes" rid="fn2"> <sup> 2 </sup> </xref></h2>
			<h3>PMID:26446482</h3>
				
					<h3>PMCID:4656908</h3>
					<h4>期刊:The Journal of Nutrition</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Padhi </surname>  <given-names> Emily MT </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Blewett </surname>  <given-names> Heather J </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Duncan </surname>  <given-names> Alison M </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guzman </surname>  <given-names> Randolph P </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hawke </surname>  <given-names> Aileen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seetharaman </surname>  <given-names> Koushik </given-names>  </name>  <xref ref-type="author-notes" rid="fn3">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tsao </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wolever </surname>  <given-names> Thomas MS </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ramdath </surname>  <given-names> D Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <aff id="aff1">  <label> 3 </label> Guelph Food Research Centre, Agriculture and Agri-Food Canada, Guelph, Canada; </aff>  <aff id="aff2">  <label> 4 </label> Department of Human Health and Nutritional Sciences, University of Guelph, Guelph, Canada; </aff>  <aff id="aff3">  <label> 5 </label> Canadian Centre for Agri-Food Research in Health and Medicine, Agriculture and Agri-Food Canada, Winnipeg, Canada; and </aff>  <aff id="aff4">  <label> 6 </label> Glycemic Index Laboratories Inc., Toronto, Canada </aff>
				<br/> 
				年份：<day> 9 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						An aliquot of plasma was transported from participating clinical trial centers to the Guelph Food Research Centre for isoflavone analysis with the use of an established LC tandem <span class="gene">MS</span> method ( <xref rid="b30" ref-type="bibr"> 30 </xref> , <xref rid="b31" ref-type="bibr"> 31 </xref> ). Briefly, thawed samples were treated with &#x003b2;-glucuronidase type <span class="gene">H1</span> from <italic> Helix pomatia </italic> (Sigma-Aldrich), followed by <span class="trait">protein</span> precipitation with the use of acidified methanol, filtration, and analysis via LC tandem <span class="gene">MS</span> ( <xref rid="b32" ref-type="bibr"> 32 </xref> ). An equol producer was defined as a participant with plasma equol concentrations &#x0003e;20 nmol/L ( <xref rid="b33" ref-type="bibr"> 33 </xref> ). Currently, there are no available guidelines for the classification of an <italic> O </italic> -desmethylangolensin producer. Therefore, for the purpose of this study, an <italic> O </italic> -desmethylangolensin producer was considered to be anyone who produced detectable amounts of this metabolite in his or her plasma.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole-Genome Sequencing and Comparative Genome Analysis of <italic> Bacillus subtilis </italic> Strains Isolated from Non-Salted Fermented Soybean Foods</h2>
			<h3>PMID:26505996</h3>
				
					<h3>PMCID:4624242</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kamada </surname>  <given-names> Mayumi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hase </surname>  <given-names> Sumitaka </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujii </surname>  <given-names> Kazushi </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Miyake </surname>  <given-names> Masato </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sato </surname>  <given-names> Kengo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kimura </surname>  <given-names> Keitarou </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sakakibara </surname>  <given-names> Yasubumi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To analyze detected variants statistically, we calculated a genetic variant score for each gene based on the variant data with predicted impact effects. The genetic variant score of a gene <italic> i </italic> for a strain X is defined by <disp-formula id="pone.0141369.e001"> <alternatives> <graphic xlink:href="pone.0141369.e001.jpg" id="pone.0141369.e001g" mimetype="image" position="anchor" orientation="portrait"/> <mml:math id="M1"> <mml:mrow> <mml:msubsup> <mml:mrow> <mml:mtext> gene </mml:mtext> </mml:mrow> <mml:mrow> <mml:mi> i </mml:mi> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:mo> = </mml:mo> <mml:mo form="prefix"> log </mml:mo> <mml:mrow> <mml:mo> ( </mml:mo> <mml:msubsup> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> h </mml:mi> <mml:mi> i </mml:mi> <mml:mi> g </mml:mi> <mml:mi> h </mml:mi> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:msubsup> <mml:mi> N </mml:mi> <mml:mrow> <mml:mi> h </mml:mi> <mml:mi> i </mml:mi> <mml:mi> g </mml:mi> <mml:msub> <mml:mi> h </mml:mi> <mml:mi> i </mml:mi> </mml:msub> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:mo> + </mml:mo> <mml:msubsup> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> m </mml:mi> <mml:mi> o </mml:mi> <mml:mi> d </mml:mi> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:msubsup> <mml:mi> N </mml:mi> <mml:mrow> <mml:mi> m </mml:mi> <mml:mi> o </mml:mi> <mml:msub> <mml:mi> d </mml:mi> <mml:mi> i </mml:mi> </mml:msub> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:mo> + </mml:mo> <mml:msubsup> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> l </mml:mi> <mml:mi> o </mml:mi> <mml:mi> w </mml:mi> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:msubsup> <mml:mi> N </mml:mi> <mml:mrow> <mml:mi> l </mml:mi> <mml:mi> o </mml:mi> <mml:msub> <mml:mi> w </mml:mi> <mml:mi> i </mml:mi> </mml:msub> </mml:mrow> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mi> X </mml:mi> <mml:mo> ) </mml:mo> </mml:mrow> </mml:msubsup> <mml:mo> + </mml:mo> <mml:mn> 1 </mml:mn> <mml:mo> ) </mml:mo> </mml:mrow> <mml:mo> , </mml:mo> </mml:mrow> </mml:math> </alternatives> </disp-formula> where <italic> N </italic> <sub> <italic> <span class="trait">high</span> </italic> <sub> <italic> i </italic> </sub> </sub> , <italic> N </italic> <sub> <italic> <span class="gene">mod</span> </italic> <sub> <italic> i </italic> </sub> </sub> , and <italic> N </italic> <sub> <italic> low </italic> <sub> <italic> i </italic> </sub> </sub> are the number of variants in gene <italic> i </italic> annotated as <span class="trait">high-,</span> moderate- and low-effect impact, respectively, and <italic> C </italic> is a positive constant for each effect defined as follows: <disp-formula id="pone.0141369.e002"> <alternatives> <graphic xlink:href="pone.0141369.e002.jpg" id="pone.0141369.e002g" mimetype="image" position="anchor" orientation="portrait"/> <mml:math id="M2"> <mml:mrow> <mml:msub> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> l </mml:mi> <mml:mi> o </mml:mi> <mml:mi> w </mml:mi> </mml:mrow> </mml:msub> <mml:mo> = </mml:mo> <mml:mn> 1 </mml:mn> <mml:mo> , </mml:mo> <mml:msub> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> m </mml:mi> <mml:mi> o </mml:mi> <mml:mi> d </mml:mi> </mml:mrow> </mml:msub> <mml:mo> = </mml:mo> <mml:mi> M </mml:mi> <mml:mi> A </mml:mi> <mml:msub> <mml:mi> X </mml:mi> <mml:mrow> <mml:mi> l </mml:mi> <mml:mi> o </mml:mi> <mml:mi> w </mml:mi> </mml:mrow> </mml:msub> <mml:mo> / </mml:mo> <mml:mn> 2 </mml:mn> <mml:mo> , </mml:mo> <mml:msub> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> h </mml:mi> <mml:mi> i </mml:mi> <mml:mi> g </mml:mi> <mml:mi> h </mml:mi> </mml:mrow> </mml:msub> <mml:mo> = </mml:mo> <mml:mrow> <mml:mo> ( </mml:mo> <mml:msub> <mml:mi> C </mml:mi> <mml:mrow> <mml:mi> m </mml:mi> <mml:mi> o </mml:mi> <mml:mi> d </mml:mi> </mml:mrow> </mml:msub> <mml:mo> + </mml:mo> <mml:mi> M </mml:mi> <mml:mi> A </mml:mi> <mml:msub> <mml:mi> X </mml:mi> <mml:mrow> <mml:mi> m </mml:mi> <mml:mi> o </mml:mi> <mml:mi> d </mml:mi> </mml:mrow> </mml:msub> <mml:mo> ) </mml:mo> </mml:mrow> <mml:mo> / </mml:mo> <mml:mn> 2 </mml:mn> <mml:mo> , </mml:mo> </mml:mrow> </mml:math> </alternatives> </disp-formula> where <italic> MAX </italic> <sub> <italic> low </italic> </sub> and <italic> MAX </italic> <sub> <italic> <span class="gene">mod</span> </italic> </sub> are the maximum number of low and moderate effects in each strain.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone.0141369.g001"> Fig 1 </xref> shows aligned regions of the BEST195 genome with each draft genome using MUMmer. In our previous study about whole-genome resequencing of BEST195 [ <xref rid="pone.0141369.ref005" ref-type="bibr"> 5 </xref> ], we discussed <span class="gene">gap</span> (incomplete) regions in the first draft genome sequence that previously had been sequenced using very short reads generated from Illumina GAII [ <xref rid="pone.0141369.ref043" ref-type="bibr"> 43 </xref> ], and we showed that these regions were attributed to GC bias and repetitive sequences. It is known that these regions are difficult to assemble with only short reads. The previous <span class="gene">gap</span> regions are indicated as black lines in <xref ref-type="fig" rid="pone.0141369.g001"> Fig 1 </xref> . Many of the unaligned regions (white regions in <xref ref-type="fig" rid="pone.0141369.g001"> Fig 1 </xref> ) correspond to the previous <span class="gene">gap</span> regions, and regions including transposases of ISs and rRNA genes, which are also difficult to assemble because of the characteristics of repetitive sequences and patterns. When a sequence differs from the reference sequence or <italic> de novo </italic> assembly in the first step does not work well, the regions cannot be covered even though we used the reference-assisted process. Therefore, there is a <span class="trait">high</span> possibility that unaligned regions are regions of repetitive sequences that could not be assembled with short reads or deleted regions in the strains.
					</p>
				
					<p>&nbsp;&nbsp;
						To score and vectorize detected variants (see <xref ref-type="sec" rid="sec002"> Material and Methods </xref> ), we performed PCA and hierarchical clustering analysis, and the results are shown in <xref ref-type="fig" rid="pone.0141369.g003"> Fig 3 </xref> . The principal components obtained in PCA were transformations of variant score vectors by a linear combination that was chosen to maximize the variance of the score vectors of all eight strains. As shown in <xref ref-type="fig" rid="pone.0141369.g003"> Fig 3-A </xref> , the first principal component (contributing rate: 51%) indicates a feature of the non-Japanese strains, and the second principal component (contributing rate: 19%) can be regarded as a feature to distinguish strain LaoA1 from the other non-Japanese strains. The Japanese strains converged and formed a small cluster. For the first principal component, principal scores are <span class="trait">high</span> for genes BSNT_09336, BSNT_09102, and BSNT_09338, which mean that these genes contribute to the first principal component. Although they all are annotated as genes encoding hypothetical proteins and their function is not identified yet, BSNT_09337 and BSNT_09100, which are located near these genes, are annotated with GO terms related to membrane and DNA binding, respectively. Genes BSNT_08913, BSNT_06665, and BSNT_08139 highly contributed to the second principal component. The functions of these genes are also unclear, but BSNT_08139 is annotated with a GO term of motor activity. BSNT_06664, which is located upstream of BSNT_06665, is annotated with a GO term of transferase activity, <italic> <span class="gene">comGA</span> </italic> (BSNT_08912) encodes a competence <span class="trait">protein</span> GA, and BSNT_08914 is annotated with a GO term related to adenyl nucleotide binding.
					</p>
				
					<p>&nbsp;&nbsp;
						Biotin is synthesized via the <italic> bio </italic> operon ( <italic> bioWAFDBI </italic> ) in <italic> B. subtilis </italic> . In the natto-fermenting strains, the <italic> <span class="gene">bioB</span> </italic> gene is functionally expressed, but the <italic> <span class="gene">bioW</span> </italic> gene and <italic> <span class="gene">bioF</span> </italic> gene are defective because of a nonsense mutation and a large deletion, respectively [ <xref rid="pone.0141369.ref044" ref-type="bibr"> 44 </xref> ]. These defects result in biotin auxotrophy of the natto-fermenting strains. According to the experimental studies, the biotin depletion condition leads to an overproduction of L-glutamic acid that is a component of <italic> &#x003b3; </italic> PGA [ <xref rid="pone.0141369.ref045" ref-type="bibr"> 45 </xref> , <xref rid="pone.0141369.ref046" ref-type="bibr"> 46 </xref> ]. Therefore, biotin auxotrophy might be related to transcriptional regulation that is favorable for <italic> &#x003b3; </italic> PGA synthesis and natto fermentation [ <xref rid="pone.0141369.ref010" ref-type="bibr"> 10 </xref> ]. The multiple sequence alignment of <italic> <span class="gene">bioF</span> </italic> and <italic> <span class="gene">bioW</span> </italic> in the genomes of the eight strains, BEST195, and 168 showed that all eight strains had the same deletion in <italic> <span class="gene">bioF</span> </italic> as BEST195. However, strains LaoA1 and MyaA2 did not have the nonsense mutation in <italic> <span class="gene">bioW</span> </italic> that was also missing in strain 168 ( <xref ref-type="fig" rid="pone.0141369.g004"> Fig 4 </xref> ). ThaB had the nonsense mutation in <italic> <span class="gene">bioW</span> </italic> , but this strain is biotin prototrophic [ <xref rid="pone.0141369.ref011" ref-type="bibr"> 11 </xref> ]. Thus, it is assumed that biotin auxotrophy of the natto-fermenting strains depends mainly on the deletion in <italic> <span class="gene">bioF</span> </italic> . The non-Japanese strains excluding NepD5 have many nucleotide changes in the <italic> bio </italic> operon. Especially, MyaA2 has a variant causing frame shift in <italic> <span class="gene">bioD</span> </italic> that encodes a dethiobiotin synthetase. The positions and details of these variants are shown in Table E in <xref ref-type="supplementary-material" rid="pone.0141369.s001"> S1 File </xref> . Additionally, in the ThaB strain, no read was mapped to <italic> <span class="gene">bioI</span> </italic> and a portion of BSNT_09438 ( <xref ref-type="fig" rid="pone.0141369.g004"> Fig 4 </xref> ). <italic> <span class="gene">bioI</span> </italic> encodes a biotin biosynthesis cytochrome <span class="gene">P450</span> <span class="trait">protein,</span> and BSNT_09438 encodes a hypothetical <span class="trait">protein</span> and corresponds to <italic> ytbQ </italic> in strain 168. Pair-wise alignment of BEST195 and the draft genome of strain ThaB using Murasaki also showed the deletion of these regions ( <xref ref-type="fig" rid="pone.0141369.g004"> Fig 4 </xref> ). Thus, we inferred that biotin synthetic organization of strains LaoA1, MyaA2, and ThaB are different from those of other strains, and these changes might result in different biotin metabolism and growth conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Subtilisin NAT (formerly designated as nattokinase) is an extracellular enzyme secreted by <italic> B. subtilis </italic> (natto) [ <xref rid="pone.0141369.ref047" ref-type="bibr"> 47 </xref> ], and belongs to the alkaline serine protease family. Subtilisin NAT is considered to be the most important enzyme for the characteristic taste and flavor of natto, and a gene encoding subtilisin NAT was determined to be <italic> <span class="gene">aprN</span> </italic> [ <xref rid="pone.0141369.ref048" ref-type="bibr"> 48 </xref> ]. Focusing on <italic> <span class="gene">aprN</span> </italic> and neighboring genes, three nucleotide changes were found to be common in <italic> <span class="gene">aprN</span> </italic> in strains LaoA1, KorC1, and MyaA2. However, a thymine-to-cytosine nucleotide change in <italic> <span class="gene">aprN</span> </italic> that leads to a non-synonymous mutation (asparagine to serine) was found only in strain ThaB. Some nucleotide changes were also found in genes located upstream and downstream of <italic> <span class="gene">aprN</span> </italic> in strains KorC1, LaoA1, MyaA2, ThaB, and NepD5. The details of these variations are shown in Figure A and Table F in <xref ref-type="supplementary-material" rid="pone.0141369.s001"> S1 File </xref> . It has been reported that some <italic> B. subtilis </italic> strains isolated from &#x0201c;Tua nao&#x0201d; exhibited higher value of production of subtilisin NAT and PGA than the Japanese commercial strain used in natto production [ <xref rid="pone.0141369.ref049" ref-type="bibr"> 49 </xref> ]. Therefore, there is a possibility that the nucleotide changes uniquely found in strain ThaB are related to the <span class="trait">high</span> productivity of subtilisin NAT, and the experimental verifications of these variants may provide an insight into the difference on the subtilisin NAT production.
					</p>
				
					<p>&nbsp;&nbsp;
						Focusing on <italic> <span class="gene">fliF</span> </italic> , which encodes the flagellar MS-ring <span class="trait">protein</span> FliF. A 13-bp fragment encoding 5 amino acids in <italic> <span class="gene">fliF</span> </italic> was largely deleted, and <italic> <span class="gene">fliF</span> </italic> is separated into two genes, BSNT_08132 and BSNT_08133 in the BEST195 genome. This deletion is estimated to deactivate <italic> <span class="gene">fliF</span> </italic> in BEST195. It should be noted that the same deletions were found in strain ThaB and the three Japanese strains, but the other strains had the same sequence as strain 168 ( <xref ref-type="fig" rid="pone.0141369.g005"> Fig 5-B </xref> ). According to previous experimental data [ <xref rid="pone.0141369.ref011" ref-type="bibr"> 11 </xref> ], strains LaoA3, which was isolated from the same source as LaoA1, MyaA2, and NepD5 did not produce <italic> &#x003b3; </italic> PGA, but strain ThaB did. In other words, a common difference in the non-Japanese strains excluding strain ThaB might have an influence on the production of <italic> &#x003b3; </italic> PGA. Additionally, in the experiment in [ <xref rid="pone.0141369.ref056" ref-type="bibr"> 56 </xref> ], <italic> &#x003b3; </italic> PGA overproduction was observed in mutants defective in flagellar basal body assembly, including a mutation in <italic> <span class="gene">fliF</span> </italic> . Although the production of <italic> &#x003b3; </italic> PGA by KorC1 was not indicated in [ <xref rid="pone.0141369.ref011" ref-type="bibr"> 11 </xref> ], it is highly likely that the 13-bp deletion of <italic> <span class="gene">fliF</span> </italic> causes the strain to be nonmotile and improves <italic> &#x003b3; </italic> PGA yields. Moreover, this result strongly supports the suggestions from the experimental studies at the nucleotide sequence level. The common deletion in <italic> <span class="gene">fliF</span> </italic> among <italic> B. subtilis </italic> strains producing <italic> &#x003b3; </italic> PGA is a previously unreported variation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association of Processed Meat Intake with Hypertension Risk in Hemodialysis Patients: A Cross-Sectional Study</h2>
			<h3>PMID:26517837</h3>
				
					<h3>PMCID:4627724</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Pei-Yu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Shwu-Huey </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wong </surname>  <given-names> Te-Chih </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Tzen-Wen </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> His-Hsien </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Tso-Hsiao </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Yu-Tong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this cross-sectional and objective study, all data and information were collected in the same week. The medical chart was used to collect the data and information, excepting dietary history. The demographic data included gender, age, HD duration, medication and comorbidities, including HTN, diabetes, and CVD. Participants were considered with HTN history, if the diagnosis code from I10 to I15 in tenth revision of the international classification of diseases and health problems (ICD-10) was found in their medical chart. Manifestations of CVD included myocardial infarction (ICD 10 code I24.1), heart failure (ICD 10 code I50) and stroke (ICD 10 code I61-I64). In the anthropometric measurements, body mass index (BMI) was calculated from the dry body weight. The seated systolic blood pressure (SBP) and diastolic blood pressure (DBP) were measured using electronic BP monitors (Colin TP-8800 Series NIBP Monitor, Mexico or Terumo ES-P110, Japan). HD patients with a weekly average <span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg were considered hypertension [ <xref rid="pone.0141917.ref021" ref-type="bibr"> 21 </xref> , <xref rid="pone.0141917.ref022" ref-type="bibr"> 22 </xref> ]. Besides, 64% of the patients (n = 57) had <span class="gene">SBP</span> &#x0003e;140 mmHg, 16% (n = 17) had DBP &#x0003e;90 mmHg, and 13% (n = 13) had both <span class="trait">high</span> <span class="gene">SBP</span> and DBP. However, only 10 patients (10%) took at least one kind of antihypertens(ion medication, and most of them used diuretic (n = 7) to control their blood pressure. Diuretic was deposited in the loop of Henle of renal tubules and interrupts sodium and chlorine reabsorption. Other antihypertensive medication used included angiotensin-converting-enzyme inhibitor (n = 6), and beta-adrenergic blocking agents (n = 4). All participants maintained their antihypertensive medication for at least 3 month before participation in this study. All serum samples were collected after fasting at least 8-h period and were then sent to the Department of Laboratory Medicine, Taipei Medical University Hospital for analysis. The serum concentration of CRP was measured using a particle-enhanced immunoturbidimetric assay (IMMULITE; Diagnostic Products Corp., Los Angeles, California), and the coefficient of variation was &#x02264;5% at 0.20 mg/L of CRP. A previous study indicated that HD patients with a CRP concentration &#x0003e;3 mg/L (equal to 2.86 nM/L) had significantly higher mortality rate [ <xref rid="pone.0141917.ref023" ref-type="bibr"> 23 </xref> ]. Therefore, serum CRP concentration &#x0003e;3 mg/L was defined as inflammation, which was observed in 45% (n = 47) of the HD patients in this study.
					</p>
				
					<p>&nbsp;&nbsp;
						The continuous variables were expressed as the mean &#x000b1; standard error and mean (95% confidence intervals, CIs), and the categorical variables were expressed as the number of subjects (%). The normal distribution was assessed by the Kolmogorov&#x02013;Smirnoff test. Gender-linked differences between groups was tested using Student's t-test for normally distributed data, and otherwise using the simple linear rank test. Categorical variable proportions were compared between groups by using the chi squared test. Furthermore, multiple robust regression models were used for analyzing the association between the different <span class="trait">protein</span> foods and blood pressure and serum CRP concentrations. Multiple logistic regression models were used to determine the odds ratio for hypertension and inflammation among different <span class="trait">protein</span> foods. Model 1 was adjusted for dietary energy, and Model 2 was additionally adjusted for gender, age, HTN medication (only for <span class="gene">SBP</span> and DBP), and HD duration (only for serum CRP). Because HTN history and HTN medication had collinearity in the regression model for hypertension, and HTN medication had higher predictive power. Therefore, this study did not adjust for HTN history. In addition, because the BMI [ <xref rid="pone.0141917.ref016" ref-type="bibr"> 16 </xref> ] and sodium content in food [ <xref rid="pone.0141917.ref008" ref-type="bibr"> 8 </xref> ] may modify the association between different <span class="trait">protein</span> foods and the CVD risk factors: HTN and inflammation, Model 2 was additionally adjusted for BMI, sodium contents, and their interaction with the <span class="trait">protein</span> foods. Phosphorus and fatty acids components in processed meat may also affect the association of processed meat and hypertension [ <xref rid="pone.0141917.ref033" ref-type="bibr"> 33 </xref> &#x02013; <xref rid="pone.0141917.ref035" ref-type="bibr"> 35 </xref> ]. Besides, in this study, there was a slightly positive association between phosphorus and sodium contents in processed meat (r square = 0.03, <italic> P </italic> = 0.04). Therefore, both phosphorus and fatty acids components would be considered in the Model 2.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg were considered as HTN in this study. After adjustment for gender, age, HTN medication, and dietary energy intake, each additional serving of processed meat positively associated with the risk of HTN ( <xref rid="pone.0141917.t003" ref-type="table"> Table 3 </xref> ). The odds ratio for <span class="gene">SBP</span> &#x0003e;140 mmHg was 2.10 (95% CI, 1.03 to 4.30) and odds ratio for DBP &#x0003e;90 mmHg was 2.51 (95% CI, 1.15 to 5.47). Furthermore, an increasing intake of 1 serving size of red meat was negatively associated with the risk of HTN, and the odds ratio for <span class="gene">SBP</span> &#x0003e;140 mmHg was 0.48 (95% CI: 0. 27, 0. 88) and DBP &#x0003e;90 mmHg was 0. 38 (95% CI: 0. 20, 0. 74). After further adjustment of sodium contents or BMI, these associations with <span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg attenuated and were no longer significant. Both of white meats and soybeans were not significantly associated with blood pressure of HD patients. Patients with a serum CRP concentration &#x0003e;3 mg/L were considered to have inflammation, and no association was observed between inflammation and the 4 <span class="trait">protein</span> foods.
					</p>
				
					<p>&nbsp;&nbsp;
						The substitution effects were estimated according to exchanging one serving of processed meat intake with another <span class="trait">protein</span> food containing equal <span class="trait">protein.</span> Substituted one serving of processed meat with one serving of red meat or white meat was negatively associated with risk of <span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg in HD patients ( <xref rid="pone.0141917.t004" ref-type="table"> Table 4 </xref> ). These effects became attenuated to non-significance after additional adjustment of models for sodium contents or BMI. Moreover, after further adjusted for the saturated fatty acids (percentage of dietary energy), the ratio of unsaturated fatty acids to saturated fatty acids, serum phosphate, or the interaction between phosphorus and sodium contents in processed meat, substituting processed meat with other <span class="trait">protein</span> foods remained significantly associated with <span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg (data did not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						This study had several limitations. First, the sample size was small. However, the demographic data of the HD patients in this study was consistent with those reported by previous large studies, including a study conducted at 25 HD centers in Taiwan [ <xref rid="pone.0141917.ref062" ref-type="bibr"> 62 </xref> ], the Taiwan Renal Registry Data System [ <xref rid="pone.0141917.ref002" ref-type="bibr"> 2 </xref> ], the United States Renal Data System [ <xref rid="pone.0141917.ref063" ref-type="bibr"> 63 </xref> ], and the DOPPS [ <xref rid="pone.0141917.ref054" ref-type="bibr"> 54 </xref> ]. Furthermore, the average age of the patients was 60.1&#x02013;66.6 years, approximately 50% of the patients were male, and the pre-HD <span class="gene">SBP</span> was 145.9 &#x000b1; 23.2 mmHg. Medical history review revealed that 36.2%&#x02013;44.4% of the patients had diabetes, 71.1%&#x02013;79.6% had HTN, and at least 20.35% had CVD. Although the prevalence of HTN was lower in this study than in previous studies, the results were independent of the HTN medication. Moreover, the association between the intake of processed meat and risk of <span class="gene">SBP</span> &#x0003e;140 mmHg and DBP &#x0003e;90 mmHg was significant. Second, selection bias existed during enrollment, and few patients took HTN medication, and the average serum CRP concentration was lower than that reported by a previous study [ <xref rid="pone.0141917.ref054" ref-type="bibr"> 54 </xref> ]. Therefore, HD patients in this study may be healthier than those in a previous study [ <xref rid="pone.0141917.ref064" ref-type="bibr"> 64 </xref> ]. Although the association between <span class="trait">protein</span> foods and blood pressure was independent of HTN medication, the future study with more dialysis patients using HTN medication is needed. Third, because this was a cross-sectional study, it was difficult to determine causal relationships. However, in Nurses&#x02019; Health Study [ <xref rid="pone.0141917.ref017" ref-type="bibr"> 17 </xref> ], greater intake of processed meats is associated with higher risk of coronary heart disease, and substituting processed meats with white meats was associated with reduced risk of coronary heart disease. HTN is one major predictor of coronary heart disease [ <xref rid="pone.0141917.ref065" ref-type="bibr"> 65 </xref> ]. Fourth, although a dietary record over multiple days was used to evaluate the validity of other dietary assessment tool [ <xref rid="pone.0141917.ref066" ref-type="bibr"> 66 </xref> , <xref rid="pone.0141917.ref067" ref-type="bibr"> 67 </xref> ], dietary records may underestimate the actual food intake [ <xref rid="pone.0141917.ref068" ref-type="bibr"> 68 </xref> ]. However, the under-reported bias was lower in this study [ <xref rid="pone.0141917.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0141917.ref030" ref-type="bibr"> 30 </xref> ], more than 7% of participants under reported their dietary intake. Fifth, this study did not include all confounding factors for pre-HD blood pressure in HD patients, such as the concentration of sodium in the dialysis solution [ <xref rid="pone.0141917.ref069" ref-type="bibr"> 69 </xref> ]. However, the results regarding factors influencing pre-HD blood pressure were still inconsistent [ <xref rid="pone.0141917.ref070" ref-type="bibr"> 70 </xref> ], and future interventional studies are required to assess the effects of processed meat on this factor in HD patients.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Co-expression of <italic> G2-EPSPS </italic> and glyphosate acetyltransferase <italic> GAT </italic> genes conferring <span class="trait">high</span> tolerance to glyphosate in soybean</h2>
			<h3>PMID:26528311</h3>
				
					<h3>PMCID:4606067</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Bingfu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hong </surname>  <given-names> Huilong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> Longguo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Lijuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Ru-Zhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Min </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/212998/overview"/>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Up to now, two strategies for achieving glyphosate tolerance have been successfully applied in GM plants. The first one is overproduction of target enzyme EPSPS, such as expression of exogenous <italic> CP4-EPSPS </italic> ( <xref rid="B25" ref-type="bibr"> Meilan et al., 2002 </xref> ) or mutant <italic> EPSPS </italic> in transgenic plants ( <xref rid="B41" ref-type="bibr"> Tian et al., 2011 </xref> ). One of the disadvantages of this strategy is that glyphosate remains to be accumulated in plant tissues and decrease crop <span class="trait">yield</span> by interfering with the development of reproductive tissues ( <xref rid="B30" ref-type="bibr"> Pline et al., 2002 </xref> ). For example, although the labeled rate for glyphosate application in <span class="gene">GR</span> soybeans is varies from 600 to 1200 g a.e.ha <sup> &#x02013;1 </sup> and the tolerant level could reach to threefolds of labeled rate (2400 g a.e. ha <sup> &#x02013;1 </sup> ) at some growth stages, some physiological indexes including photosynthesis, nutrient accumulation, and nodulation in some cultivars was still reduced after receiving increasing glyphosate rates or applications at later growth stages ( <xref rid="B12" ref-type="bibr"> Gazziero et al., 2008 </xref> ; <xref rid="B51" ref-type="bibr"> Zobiole et al., 2012 </xref> ). Meanwhile, the typical symptom, known as &#x0201c;yellow flashing,&#x0201d; was observed in both GR1 and GR2 soybeans ( <xref rid="B51" ref-type="bibr"> Zobiole et al., 2012 </xref> ). This symptom is attributed to the decrease of chlorophyll content, accumulation of the primary phytotoxic metabolite or the formation of insoluble glyphosate-metal complexes ( <xref rid="B3" ref-type="bibr"> Campbell et al., 1976 </xref> ; <xref rid="B33" ref-type="bibr"> Reddy et al., 2004 </xref> ; <xref rid="B51" ref-type="bibr"> Zobiole et al., 2012 </xref> ). The second strategy is removal of herbicidal residue after detoxification it by glyphosate oxidoreductase ( <italic> GOX </italic> ) gene ( <xref rid="B29" ref-type="bibr"> Pedotti et al., 2009 </xref> ) or glyphosate N-acetyltransferase ( <italic> GAT </italic> ) gene ( <xref rid="B4" ref-type="bibr"> Castle et al., 2004 </xref> ; <xref rid="B37" ref-type="bibr"> Siehl et al., 2005 </xref> ). GOX is an enzyme which can degrade glyphosate and converting it to glyoxylate and aminomethylphosphonic acid (AMPA). Although AMPA is less phytotoxic than glyphosate to plant, it could still induce &#x0201c;yellow flashing&#x0201d; ( <xref rid="B6" ref-type="bibr"> Ding et al., 2011 </xref> ). Another enzyme GAT could degrade glyphosate residues by catalyzing acetylation of glyphosate. Therefore, overexpression of the <italic> GOX </italic> or <italic> GAT </italic> gene can also result in a relatively <span class="trait">high</span> level of glyphosate tolerance in different crops ( <xref rid="B15" ref-type="bibr"> Green et al., 2008 </xref> ; <xref rid="B18" ref-type="bibr"> Hadi et al., 2012 </xref> ). <xref rid="B7" ref-type="bibr"> Dun et al. (2014) </xref> compared the glyphosate tolerance of transgenic tobacco which expressed <italic> GAT </italic> and <italic> EPSPS </italic> alone or combination and the results suggested that co-expression of them showed the highest tolerance to glyphosate, providing a new method to develop <span class="trait">high</span> glyphosate tolerant crops by combination of different strategies.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induction of proto-oncogene BRF2 in breast cancer cells by the dietary soybean isoflavone daidzein</h2>
			<h3>PMID:26573593</h3>
				
					<h3>PMCID:4647806</h3>
					<h4>期刊:BMC Cancer</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Koo </surname>  <given-names> Jana </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cabarcas-Petroski </surname>  <given-names> Stephanie </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Petrie </surname>  <given-names> John L. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Diette </surname>  <given-names> Nicole </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> White </surname>  <given-names> Robert J. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Schramm </surname>  <given-names> Laura </given-names>  </name>  <address>  <email> schramml@stjohns.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Biological Sciences, St. John&#x02019;s University, Queens New York, 11439 USA  </aff>  <aff id="Aff2">  <label/> Pennsylvania State University, Beaver Campus, Monaca, PA 15061 USA  </aff>  <aff id="Aff3">  <label/> Department of Biology, University of York, Heslington, York, YO10 5DD UK  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The chemopreventive polyphenol EGCG, enriched in green tea, specifically decreases TFIIIB activity in cervical cancer cells [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. The polyphenols genistein and daidzein are isoflavone components of soybeans, a major crop in the United States and globally [ <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. These soy isoflavones are major food-derived phytoestrogens that are structurally similar to estrogen with the capacity to weakly bind to estrogen receptors (ERs) [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. The putative estrogenic effect of soy raises the concern that <span class="trait">high</span> consumption of soy foods by breast cancer patients and/or women at <span class="trait">high</span> risk for breast cancer may increase estrogen-dependent breast tumor growth [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. A recent study reported a significant increase in cell proliferation in breast cancer patients who used soy supplementation [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. Dietary soy isoflavones increase metastasis to lungs in an experimental model of breast cancer [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. These data prompted us to investigate if the soy isoflavone daidzein regulates TFIIIB. We found that 10 uM daidzein stimulates expression of the TFIIIB subunits BRF1 and BRF2 in ER-positive breast cancer cells, as well as <span class="gene">pol</span> III products U6 snRNA and tRNA <sub> i </sub> <sup> Met </sup> . Daidzein treatment stabilizes BRF2 and BRF1 mRNAs and raises levels of their <span class="trait">protein</span> products. It also triggers selective demethylation of the BRF2 promoter. These effects are not seen in an ER-negative breast cancer line. An isoflavone-enriched diet also induces BRF2 in female mice, but has the opposite effect in males. These <italic> in vitro </italic> and <italic> in vivo </italic> data suggest that dietary isoflavones differentially regulate TFIIIB expression, an important observation given the evidence that BRF2 can drive tumorigenesis and is predictive of poor prognosis.
					</p>
				
					<p>&nbsp;&nbsp;
						Human breast cancer cell lines were treated for 48&#x000a0;h with daidzein and qRT-PCRwas used to assay expression of mRNAs encoding the subunits of TFIIIB. The mRNAs encoding BRF1 and BRF2 were both found to be induced significantly in ER-positive MCF-7 cells ( <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.001 and <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01, respectively; Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1a </xref> ). This response is selective and not shown by the mRNAs encoding TBP and BDP1, the other TFIIIB subunits. Furthermore, none of the TFIIIB mRNAs showed significant responses to daidzein in ER-negative MDA-MB-231 cells, under these conditions (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1b </xref> ). The induction of BRF1 and BRF2 mRNAs in MCF-7 cells is translated into a corresponding increase in the <span class="trait">protein</span> products, as shown by western blotting (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2a </xref> ). As with the mRNA, <span class="trait">protein</span> expression does not respond in MDA-MB-231 cells (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2b </xref> ). Quantitative analysis by ELISA established that BRF2 <span class="trait">protein</span> induction in MCF-7 cells is 2.4-fold with 3&#x000a0;&#x003bc;M daidzein and 4.2-fold with 10&#x000a0;&#x003bc;M daidzein (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2c </xref> ). As observed by western, no significant change in BRF2 levels was detected by ELISA with MDA-MB-231 cells (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2d </xref> ). The uninduced level of BRF2 is higher in MDA-MB-231 than MCF-7 cells (1.4-fold elevated by quantitative ELISA), consistent with a previously published report [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ], which showed that BRF2 mRNA is expressed at higher levels in MDA-MB-231 cells than in MCF-7 cells. However, it is unlikely that BRF2 has reached a saturation point in untreated MDA-MB-231 cells that would preclude further induction. <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> Daidzein induces BRF1 and BRF2 mRNA specifically in MCF-7 breast cancer cells. MCF-7 ( <bold> a </bold> ) and MDA-MB-231 ( <bold> b </bold> ) cells were treated with 0, 3 or 10&#x000a0;&#x003bc;M daidzein for 48&#x000a0;h. BRF2, BRF1, BDP1 and TBP mRNA expression was then analysed by qRT-PCRusing the &#x00394;&#x00394;Ct method with <span class="gene">RPS13</span> expression levels as a reference for normalization. Meta-analysis of three independent experiments performed in triplicate was completed using one-way ANOVA with a Tukey&#x02019;s post-test with a 95&#x000a0;% confidence interval (Graphpad Prism 3.03); *&#x02009;=&#x02009; <italic> p </italic> &#x0003c;0.05; **&#x02009;=&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01; ***&#x02009;=&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.001
					</p>
				
					<p>&nbsp;&nbsp;
						The 3&#x02019;-untranslated regions (3&#x02019;-UTR) of BRF1 and BRF2 mRNAs are AU-rich, which may influence their stability [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. Various classes of polyphenols have been shown to post-transcriptionally regulate levels of mRNA containing AU-rich elements by raising expression of RNA binding proteins [ <xref ref-type="bibr" rid="CR46"> 46 </xref> &#x02013; <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. To test if daidzein stabilizes BRF mRNAs, as a potential mechanism of induction, we treated MCF-7 cells with the transcription inhibitor actinomycin D [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ] and monitored mRNA decay using qRT-PCR. Although 10&#x000a0;&#x003bc;M daidzein was unable to prevent the turnover of BRF mRNAs by 24&#x000a0;h, it prevented degradation of both BRF2 (p&#x02009;&#x0003c;&#x02009;0.01; Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6a </xref> ) and BRF1 ( <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05; Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6b </xref> ) for 12&#x000a0;h after transcription inhibition. This effect is specific, as mRNA encoding ribosomal <span class="trait">protein</span> S13 (RPS13) decayed with similar kinetics in the presence or absence of daidzein, when assayed in parallel (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6c </xref> ). The selective stabilization of BRF mRNAs offers a potential explanation for the increased expression seen when MCF-7 cells are exposed to 10&#x000a0;&#x003bc;M daidzein. Additional mechanisms may also be involved, such as demethylation of the BRF2 promoter (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4b </xref> ). <fig id="Fig6"> <label> Fig. 6 </label> <caption> <p> Daidzein increases stability of BRF1 and BRF2 mRNAs selectively in MCF-7 cells. MCF-7 cells were treated with 5&#x000a0;&#x003bc;g/ml of actinomycin D, either alone (white bars) or with 10&#x000a0;&#x003bc;M daidzein (black bars) for 24&#x000a0;h. The cells were then harvested at 4, 8, 12 and 24&#x000a0;h. The RNA was extracted and analyzed by qRT-PCRfor levels of ( <bold> a </bold> ) BRF2, ( <bold> b </bold> ) BRF1, ( <bold> c </bold> ) RPS13, and actin &#x003b2; mRNAs. &#x00394;&#x00394;Ct method with <span class="gene">RPS13</span> and actin &#x003b2; expression levels were calculated for assay normalization. Data presented are average of three independent experiments. Statistical analysis was performed using one-way ANOVA with a Tukey&#x02019;s post-test with a 95&#x000a0;% confidence interval (Graphpad Prism 3.03); *&#x02009;=&#x02009; <italic> p </italic> &#x0003c;0.05; **&#x02009;=&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01; ***&#x02009;=&#x02009; <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.001
					</p>
				
					<p>&nbsp;&nbsp;
						Purina 5001 is a chow that is commonly fed to laboratory rodents and contains <span class="trait">high</span> levels of both daidzein and genistein, the two major isoflavone components of soybeans [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Our observations with daidzein in cultured cells prompted us to investigate if BRF expression is elevated in mice fed a diet enriched in these isoflavones. RNA was isolated from the blood of mice maintained on either Purine 5001 or the isoflavone-free casein-based Purina 5&#x000a0;K96 chow. Significantly elevated BRF2 expression was found in female mice with the <span class="trait">high</span> isoflavone Purina 5001 diet (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7a </xref> ). In striking contrast, the opposite was seen in males, where BRF2 expression was significantly lower with <span class="trait">high</span> isoflavones relative to the isoflavone-free diet (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7b </xref> ). BRF1 was also suppressed in males fed the <span class="trait">high</span> isoflavone Purina 5001 chow, but was unaffected in females. No statistically significant changes were found in the other TFIIIB subunits, TBP and BDP1. These <italic> in vivo </italic> data suggest that male and female C57BL/6&#x000a0;J mice respond differently to dietary sources of isoflavones and these differences are measurable in the blood. They emphasize the importance of taking into account both gender and diet in mouse models of cancer. <fig id="Fig7"> <label> Fig. 7 </label> <caption> <p> Differential expression of TFIIIB subunits in female and male mice fed <span class="trait">high</span> or low isoflavone diets. Five week C57CBL/6&#x000a0;J ( <bold> a </bold> ) female and ( <bold> b </bold> ) male mice had free access to isoflavone-free, casein-based diet 5&#x000a0;K96 or regular Lab Diet 5001, with <span class="trait">high</span> isoflavone content. Post-treatments, total RNA was isolated from blood collected by cardiac puncture and analyzed by qRT-PCR for expression of BRF2, BRF1, BDP1, and TBP. The &#x00394;&#x00394;Ct method with <span class="gene">GAPDH</span> expression levels was used as a reference for normalization. Data presented are average of six female and six male mice from respective groups. Statistical analysis was performed using one-way ANOVA with a Tukey&#x02019;s post-test with a 95&#x000a0;% confidence interval (Graphpad Prism 3.03); *&#x02009;=&#x02009;p &#x0003c;0.05; **&#x02009;=&#x02009;p&#x02009;&#x0003c;&#x02009;0.01; ***&#x02009;=&#x02009;p&#x02009;&#x0003c;&#x02009;0.001
					</p>
				
					<p>&nbsp;&nbsp;
						It remains to be determined how BRF2 exerts oncogenic effects, although the obvious possibility is through transcriptomic changes, given its well-established function as a transcription factor. Genome-wide analyses showed that BRF2 has a highly restricted set of target genes [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ], all encoding short <span class="gene">pol</span> III-dependent noncoding RNAs. In addition to U6 snRNA, these include 7SK RNA, which regulates transcription of protein-coding genes by inhibiting P-TEFb [ <xref ref-type="bibr" rid="CR58"> 58 </xref> &#x02013; <xref ref-type="bibr" rid="CR60"> 60 </xref> ], and hY RNAs, which promote DNA replication [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. The hY RNAs are commonly overexpressed in tumors [ <xref ref-type="bibr" rid="CR62"> 62 </xref> , <xref ref-type="bibr" rid="CR63"> 63 </xref> ]. For example, hY1 is overexpressed ~13-fold in lung, prostate and other cancers ( <italic> P </italic> &#x02009;=&#x02009;9.7&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;26 </sup> ) and its knockdown inhibits proliferation of lung and prostate cell lines [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. As a key component of spliceosomes, U6 snRNA is an essential BRF2-dependent product. It was consistently detected at elevated levels in sera from 140 breast cancer patients, relative to 115 healthy age-matched controls [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. U6 was also the most strongly upregulated (P&#x02009;&#x0003c;&#x02009;0.0001; FDR &#x0003c;0.001) non-coding RNA in serum from 75 glioblastoma multiforme patients, relative to matched healthy controls [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. MicroRNAs are widely believed to have considerable potential as biomarkers, but U6 outperformed all 381 miRNAs tested in this study [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. It has also been suggested that BRF2 might serve as a biomarker, given its early induction in development of lung SqCC [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ], where <span class="trait">high</span> BRF2 is independently prognostic ( <italic> P </italic> &#x02009;=&#x02009;0.007) of unfavourable survival [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ], as also reported ( <italic> P </italic> &#x02009;=&#x02009;0.009) for oesophageal SqCC [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Our demonstration that BRF2 levels can be monitored in mouse serum provides initial evidence that a non-invasive test is practical for this potential biomarker.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Compositional regulation of poly(3-hydroxybutyrate- <italic> co </italic> -3-hydroxyhexanoate) by replacement of granule-associated <span class="trait">protein</span> in <italic> Ralstonia eutropha </italic></h2>
			<h3>PMID:26597300</h3>
				
					<h3>PMCID:4657207</h3>
					<h4>期刊:Microbial Cell Factories</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kawashima </surname>  <given-names> Yui </given-names>  </name>  <address>  <email> cogito.erugo.sum@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Orita </surname>  <given-names> Izumi </given-names>  </name>  <address>  <email> iorita@bio.titech.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakamura </surname>  <given-names> Satoshi </given-names>  </name>  <address>  <email> nakamura.s.af@m.titech.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Fukui </surname>  <given-names> Toshiaki </given-names>  </name>  <address>  <phone> +81-45-924-5766 </phone>  <email> tfukui@bio.titech.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Department of Bioengineering, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsuta, Midori-ku, Yokohama, 226-8501 Japan  </aff>
				<br/> 
				年份：<day> 4 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The most abundant PHA in nature is poly(3-hydroxybutyrate) [P(3HB)], which is generally synthesized from acetyl-CoA through three consecutive reactions by &#x003b2;-ketothiolase (PhaA), NADPH-dependent acetoacetyl-CoA reductase (PhaB) and PHA synthase (PhaC). <italic> Ralstonia eutropha </italic> ( <italic> Cupriavidus necator </italic> ) H16 is a well-studied P(3HB) producer [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. It has been known that the three genes for P(3HB) biosynthesis are clustered as <italic> phaC </italic> - <italic> A </italic> - <italic> B1 </italic> <sub> <italic> Re </italic> </sub> operon, and PhaC <sub> <italic> Re </italic> </sub> shows polymerization activity toward short-chain-length ( <italic> R </italic> )-3-hydroxyacyl (3HA)-CoAs of C <sub> 3 </sub> &#x02013;C <sub> 5 </sub> . However, the application range of P(3HB) is very limited due to the brittle and struggle properties attributed to the <span class="trait">high</span> crystallinity and <span class="trait">high</span> melting temperature. Poly(3-hydroxybutyrate- <italic> co </italic> -3-hydroxyhexanoate) [P(3HB- <italic> co </italic> -3HHx)] is a PHA copolyester naturally produced by some bacteria such as <italic> Aeromonas caviae </italic> from vegetable oils and fatty acids [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. This copolyester has been demonstrated to show more soft and flexible properties that are suitable for practical applications when compared to P(3HB) homopolymer [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. In <italic> A. caviae </italic> , ( <italic> R </italic> )-3HA-CoA monomers of C <sub> 4 </sub> and C <sub> 6 </sub> are provided from 2-enoyl-CoA intermediates in &#x003b2;-oxidation by the function of ( <italic> R </italic> )-specific enoyl-CoA hydratase (PhaJ <sub> <italic> Ac </italic> </sub> ), and successively polymerized by PhaC <sub> <italic> Ac </italic> </sub> having unique substrate specificity (C <sub> 4 </sub> -C <sub> 7 </sub> ) [ <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Based on these facts, efforts have been made to construct recombinant bacteria for efficient production of P(3HB- <italic> co </italic> -3HHx). Several previous studies engineered <italic> R. eutropha </italic> H16 by introduction of heterologous PHA synthase and ( <italic> R </italic> )-specific enoyl-CoA hydratase, resulting in biosynthesis of P(3HB- <italic> co </italic> -3HHx) composed of adequately <span class="trait">high</span> 3HHx fractions from vegetable oils (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ) [ <xref ref-type="bibr" rid="CR8"> 8 </xref> &#x02013; <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Tsuge et al. have reported that the Asn149Ser/Asp171Gly double mutant of <italic> A. caviae </italic> PHA synthase, named PhaC <sub> NSDG </sub> , could accept more 3HHx unit than wild type PhaC <sub> <italic> Ac </italic> </sub> in vivo [ <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. Insomphan et al. reported that deletion of one of <italic> <span class="gene">fadB</span> </italic> homologs in <italic> R. eutropha </italic> harboring <italic> phaC </italic> <sub> NSDG </sub> increased 3HHx fractions in the soybean oil-based P(3HB- <italic> co </italic> -3HHx) [ <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. Recently, an artificial pathway for synthesis of this copolyester from structurally unrelated fructose was developed in <italic> R. eutropha </italic> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. <fig id="Fig1"> <label> Fig.&#x000a0;1 </label> <caption> <p> Proposed P(3HB- <italic> co </italic> -3HHx) biosynthesis pathway from soybean oil through &#x003b2;-oxidation by <italic> R. eutropha </italic> NSDG-based recombinant strains. <italic> PhaA </italic> <sub> <italic> Re </italic> </sub> and <italic> BktB </italic> <sub> <italic> Re </italic> </sub> &#x003b2;-ketothiolases, <italic> PhaB1 </italic> <sub> <italic> Re </italic> </sub> NADPH-acetoacetyl-CoA reductase, <italic> PhaJ </italic> <sub> <italic> Ac </italic> </sub> and <italic> PhaJ4a </italic> <sub> <italic> Re </italic> </sub> ( <italic> R </italic> )-specific enoyl-CoA hydratases, <italic> FadB </italic> <sub> <italic> Re </italic> </sub> bifunctional ( <italic> S </italic> )-specific 2-enoyl-CoA hydratase/( <italic> S </italic> )-3-hydroxyacyl-CoA dehydrogenase, <italic> PhaC </italic> <sub> <italic> NSDG </italic> </sub> N143S/D171G mutant of PHA synthase from <italic> A. caviae </italic> , <italic> Re R. eutropha </italic> , <italic> Ac A. caviae </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						The accumulation of PHA granules within <italic> R. eutropha </italic> H16, NSDG, NSDG-P <sub> Ac </sub> , NSDG-P1 <sub> Re </sub> CJ, and NSDG-P <sub> Ac </sub> CJ grown on soybean oil were observed by transmission electron microscopy (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). The wild strain H16 accumulated 10&#x02013;16 of small PHA granules per one cell, whereas NSDG and NSDG-P <sub> Ac </sub> formed one to three of much larger PHA granule(s). In the cells of NSDG-P1 <sub> Re </sub> CJ and NSDG-P <sub> Ac </sub> CJ, granules with various sizes were existed together. The size and number of PHA granules were greatly affected by the kind of PHA synthase, but not by phasin on the granule surface. The reason for these observations has been unclear yet, but the formation of less larger granules in the strain NSDG was similar to phenotype of the <italic> phaP1 </italic> <sub> <italic> Re </italic> </sub> -disruptant [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ]. The hydrophilicity of cytosolic side of the PhaC <sub> NSDG </sub> <span class="trait">protein</span> on PHA granules may be lower than that of PhaC <sub> <italic> Re </italic> </sub> , leading to formation of large granules with reduced surface area. <fig id="Fig4"> <label> Fig.&#x000a0;4 </label> <caption> <p> Transmission electron micrographs of ultrathin sections of <italic> R. eutropha </italic> strains. Magnification: &#x000d7;10,000; <italic> scale <span class="gene">bar</span> </italic> 0.5&#x000a0;&#x003bc;m
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <italic> Pratylenchus penetrans </italic> Transcriptome as a Source for the Development of Alternative Control Strategies: Mining for Putative Genes Involved in Parasitism and Evaluation of <italic> in planta </italic> RNAi</h2>
			<h3>PMID:26658731</h3>
				
					<h3>PMCID:4684371</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vieira </surname>  <given-names> Paulo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Eves-van den Akker </surname>  <given-names> Sebastian </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Verma </surname>  <given-names> Ruchi </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wantoch </surname>  <given-names> Sarah </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Eisenback </surname>  <given-names> Jonathan D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kamo </surname>  <given-names> Kathryn </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						For nematode <span class="trait">resistance</span> tests, roots 3&#x02013;5 cm in length were excised from stock cultures and transferred to fresh <span class="gene">MS</span> plates without antibiotics. Three independent lines were challenged with RLN using nine hairy root systems per line in each of two independent biological tests for each dsRNA construct. Three control lines (containing the pBinJit empty vector) were selected using nine hairy roots per line. Two weeks later, each hairy root system was inoculated with a mixed population of approximately 300 sterile <italic> P </italic> . <italic> penetrans </italic> and maintained in the dark at 28&#x000b0;C. Approximately three months after nematode inoculation, infected soybean hairy roots were chopped into small pieces and both roots and media were placed into sterile glass bowls filled with sterile water containing 50 mg/l carbenicillin and 50 mg/l kanamycin. Nematodes were extracted 5 days later by sieving the water with a 500 &#x003bc;m mesh sieve. Data are expressed as the total mean number of nematodes &#x000b1; Standard Error of the Mean (SEM) collected for each line, from two independent biological experiments. All data were analysed using analysis of variance (ANOVA), and means were compared using Tukey&#x02019;s honestly significant difference (HSD) test at the 5% probability level.
					</p>
				
					<p>&nbsp;&nbsp;
						Other interesting transcripts similar to neuromuscular function related genes, such as the FMRFamide-like peptides ( <italic> flp-1 </italic> and <italic> <span class="gene">flp-3</span> </italic> ) and neuropeptide-like <span class="trait">protein</span> ( <italic> nlp-10 </italic> ), were also prominently represented in this set. Using homology-based Blast searches we detected transcripts encoding up to 12, 9 and 3 members of the FMRFamide-like, NLP-like and insulin-like peptides families in the full dataset, respectively ( <xref rid="pone.0144674.s004" ref-type="supplementary-material"> S2 Table </xref> ). Due to their broad functional scope, such as neuroendocrine and neuro-modulatory effects on locomotion, reproduction, feeding and behavior, there has been a long-standing interest in using the neuropeptidergic system and FLP-signaling pathways as alternative target for the development of novel methods to control animal- and plant-parasitic nematodes [ <xref rid="pone.0144674.ref044" ref-type="bibr"> 44 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Comparison of the Beneficial Effects of Live and Heat-Inactivated Baker&#x02019;s Yeast on Nile Tilapia: Suggestions on the Role and Function of the Secretory Metabolites Released from the Yeast</h2>
			<h3>PMID:26696403</h3>
				
					<h3>PMCID:4690590</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ran </surname>  <given-names> Chao </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Huang </surname>  <given-names> Lu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Zhi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Yalin </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tacon </surname>  <given-names> Philippe </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Auclair </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Zhigang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Yeast is frequently used as a probiotic in aquaculture with the potential to substitute for antibiotics. In this study, the involvement and extent to which the viability of yeast cells and thus the secretory metabolites released from the yeast contribute to effects of baker&#x02019;s yeast was investigated in Nile tilapia. No yeast, live yeast or heat-inactivated baker&#x02019;s yeast were added to basal diets <span class="trait">high</span> in fishmeal and low in soybean (diet A) or low in fishmeal and <span class="trait">high</span> in soybean (diet B), which were fed to fish for 8 weeks. Growth, feed utilization, gut microvilli morphology, and expressions of <italic> <span class="gene">hsp70</span> </italic> and inflammation-related cytokines in the intestine and head kidney were assessed. Intestinal microbiota was investigated using 16S rRNA gene pyrosequencing. Gut alkaline phosphatase (AKP) activity was measured after challenging the fish with <italic> Aeromonas hydrophila </italic> . Results showed that live yeast significantly improved FBW and WG ( <italic> P </italic> &#x0003c; 0.05), and tended to improve FCR ( <italic> P </italic> = 0.06) of fish compared to the control (no yeast). No significant differences were observed between inactivated yeast and control. Live yeast improved gut microvilli length ( <italic> P </italic> &#x0003c; 0.001) and density ( <italic> P </italic> &#x0003c; 0.05) while inactivated yeast did not. The <italic> <span class="gene">hsp70</span> </italic> expression level in both the intestine and head kidney of fish was significantly reduced by live yeast ( <italic> P </italic> &#x0003c; 0.05) but not inactivated yeast. Live yeast but not inactivated yeast reduced intestinal expression of <italic> tnf&#x003b1; </italic> ( <italic> P </italic> &#x0003c; 0.05), <italic> tgf&#x003b2; </italic> ( <italic> P </italic> &#x0003c; 0.05 under diet A) and <italic> il1&#x003b2; </italic> ( <italic> P </italic> = 0.08). Intestinal <italic> Lactococcus </italic> spp. numbers were enriched by both live and inactivated yeast. Lastly, both live and inactivated yeast reduced the gut AKP activity compared to the control ( <italic> P </italic> &#x0003c; 0.001), indicating protection of the host against infection by <italic> A </italic> . <italic> hydrophila </italic> . In conclusion, secretory metabolites did not play major roles in the growth promotion and disease protection effects of yeast. Nevertheless, secretory metabolites were the major contributing factor towards improved gut microvilli morphology, relieved stress status, and reduced intestinal inflammation of Nile tilapia fed diets supplemented with baker&#x02019;s yeast.
					</p>
				
					<p>&nbsp;&nbsp;
						Live yeast supplementation significantly reduced the expression of <italic> <span class="gene">hsp70</span> </italic> in both intestine and head kidney of tilapia. <span class="gene">HSP70</span> brings appropriate protection of <span class="trait">protein</span> structures, strengthens the immune system and stops apoptotic mechanisms [ <xref rid="pone.0145448.ref038" ref-type="bibr"> 38 </xref> ], and its expression has been found to be related to stressful conditions in aquaculture, such as temperature oscillations, crowding, insufficient water quality, and improper diet [ <xref rid="pone.0145448.ref039" ref-type="bibr"> 39 </xref> ]. Generally, <italic> <span class="gene">hsp70</span> </italic> expression decreases in fish given probiotics and coincides with improved welfare and growth [ <xref rid="pone.0145448.ref040" ref-type="bibr"> 40 </xref> &#x02013; <xref rid="pone.0145448.ref043" ref-type="bibr"> 43 </xref> ]. In contrast to the live yeast feeding group, only a marginal reduction in <italic> <span class="gene">hsp70</span> </italic> expression was observed in the inactivated yeast group, indicating the importance of viability of yeast in the relief of stress status in fish.
					</p>
				
					<p>&nbsp;&nbsp;
						The formulations of the two basal diets were both extensively used in the aquaculture practice. In both diets, the inclusion of fishmeal was relatively low, which was replaced by soybean meal and rapeseed meal. Although the level of soybean meal was higher in diet B, no intestinal inflammation was observed, which may be attributed to the low inclusion of soybean meal in this diet. Moreover, diet B generally gave improved growth, feed utilization, microvilli morphology, as well as intestinal stress status (lower <italic> <span class="gene">hsp70</span> </italic> expression), indicating a more balanced nutrition of diet B compared with diet A. Nevertheless, there were no difference in disease <span class="trait">resistance</span> between fish groups fed diet A and diet B, implying that the advantage of diet B mainly involves nutrition, with no effect on the host immunity. Apart from the <italic> tgf&#x003b2; </italic> expression in intestine, there were no interactions between yeast treatment and basal diets for all the parameters evaluated, indicating that the beneficial effects of yeast, as well as the advantage of live yeast, don&#x02019;t depend on the <span class="trait">protein</span> profiles of the basal diets and the different intestinal microbiota they induced.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, the present study showed the advantages of live yeast as a dietary supplement for Nile tilapia, as indicated by the improved gut microvilli morphology, reduced <italic> <span class="gene">hsp70</span> </italic> expression level, as well as the generally down-regulated intestinal inflammation status, which suggests a role for yeast secretory metabolites. However, the observed improvement at the microscopic/molecular level did not translate into evident improvement in growth performance and disease <span class="trait">resistance,</span> and only a marginal increase was observed. Therefore, the secretory metabolites might not play a major role in growth promotion and disease protection effects of yeast, as compared to the structural components, which might be due to the limited dose provided by the live yeast in the host or the duration of feeding. Nevertheless, the beneficial effects and modes of action of yeast secretory metabolites deserve further study, which may lead to the development of novel functional dietary supplements for the replacement of antibiotics.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomics and Metabolomics: Two Emerging Areas for Legume Improvement</h2>
			<h3>PMID:26734026</h3>
				
					<h3>PMCID:4689856</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ramalingam </surname>  <given-names> Abirami </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kudapa </surname>  <given-names> Himabindu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/134204/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pazhamala </surname>  <given-names> Lekha T. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/209488/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weckwerth </surname>  <given-names> Wolfram </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/13413/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/25772/overview"/>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Gel-based systems have been widely applied for <span class="trait">protein</span> differential expression analysis which involves the use of two-dimensional gel electrophoresis (2D-GE) or two-dimensional difference gel electrophoresis (2D-DIGE) for <span class="trait">protein</span> separation coupled with <span class="gene">MS</span> applications for the identification and quantitation of proteins (Kosov&#x000e1; et al., <xref rid="B105" ref-type="bibr"> 2011 </xref> ; Subramanian and Smith, <xref rid="B183" ref-type="bibr"> 2013 </xref> ). The 2D-GE method has an advantage of resolving and visualizing thousands of spots corresponding to different molecular forms of same proteins. However, it has low sensitivity and is not suited for <span class="trait">protein</span> quantitation (Marouga et al., <xref rid="B125" ref-type="bibr"> 2005 </xref> ; Abdallah et al., <xref rid="B1" ref-type="bibr"> 2012 </xref> ). It was found that the use of fluorescent dyes in place of Coomassie can considerably increase the sensitivity and compatibility for <span class="gene">MS</span> analyses (Vanderschuren et al., <xref rid="B190" ref-type="bibr"> 2013 </xref> ). These factors have led to the usage of a more refined gel electrophoresis method, called the 2D-DIGE which is based on pre-labeling of <span class="trait">protein</span> mixtures with fluorescence. Thus, 2D-DIGE is a much more sensitive method, which allows <span class="trait">protein</span> detection at sub-picomolar levels for highly precise quantitative proteomic studies (Marouga et al., <xref rid="B125" ref-type="bibr"> 2005 </xref> ). Few studies have been reported in legumes, which employed 2D-DIGE for monitoring early response to symbiosis and pathogenesis, such as in the root proteome in Medicago (Schenkluhn et al., <xref rid="B165" ref-type="bibr"> 2010 </xref> ) and parasitic infections in pea (Castillejo et al., <xref rid="B36" ref-type="bibr"> 2012 </xref> ). The advantages and disadvantages of 2D-GE and 2D-DIGE have been reviewed before (see Griffin and Aebersold, <xref rid="B67" ref-type="bibr"> 2001 </xref> ; Marouga et al., <xref rid="B125" ref-type="bibr"> 2005 </xref> ; Abdallah et al., <xref rid="B1" ref-type="bibr"> 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						For identification of proteins separated by 2D-GE, <span class="gene">MS</span> based techniques were found to have <span class="trait">high</span> accuracy, resolution and sensitivity but the sample preparation seemed to be laborious (van Wijk, <xref rid="B191" ref-type="bibr"> 2001 </xref> ; Subramanian and Smith, <xref rid="B183" ref-type="bibr"> 2013 </xref> ). Earlier, the usage of techniques such as matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) <span class="gene">MS</span> and electrospray ionization (ESI) <span class="gene">MS</span> with tandem <span class="gene">MS</span> (MS/MS) overcame the slower and less sensitive methods (Griffin and Aebersold, <xref rid="B67" ref-type="bibr"> 2001 </xref> ; Griffin et al., <xref rid="B68" ref-type="bibr"> 2001 </xref> ; van Wijk, <xref rid="B191" ref-type="bibr"> 2001 </xref> ). In particular, the application of tandem <span class="gene">MS</span> methods provided sequence information for more peptides accurately for the identification of proteins and PTMs (Pandey and Mann, <xref rid="B144" ref-type="bibr"> 2000 </xref> ; Bachi and Bonaldi, <xref rid="B13" ref-type="bibr"> 2008 </xref> ). The use of MALDI ionization with two TOF analysers, (MALDI TOF/TOF) and liquid chromatography tandem <span class="gene">MS</span> (LC-MS/MS) have been fairly new technologies for <span class="trait">protein</span> identification that enhanced the pipeline for <italic> de novo </italic> assembly. The availability of these tools, together with the genome sequences of more plants and accessibility of sequence databases, intensified the identification of <span class="trait">protein</span> spots in different plants (see Jorr&#x000ed;n-Novo et al., <xref rid="B95" ref-type="bibr"> 2015 </xref> ). The most commonly used pipeline for the identification of proteins involves comparing the MS/MS spectra to the reference databases (Romero-Rodr&#x000ed;guez et al., <xref rid="B154" ref-type="bibr"> 2014 </xref> ). Although SEQUEST, MASCOT and X!Tandem are the widely used database search programs for <span class="trait">protein</span> identification, MassMatrix is also suited for tandem <span class="gene">MS</span> data (Xu and Freitas, <xref rid="B223" ref-type="bibr"> 2009 </xref> ; Senkler and Braun, <xref rid="B172" ref-type="bibr"> 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Shot-gun proteomics is a gel-free &#x0201c;bottom-up&#x0201d; strategy, in which complex peptide fractions produced after proteolytic digestion of proteins are analyzed by LC-MS/MS. The <span class="trait">protein</span> identification rates are enhanced using different peptide or <span class="trait">protein</span> fractionation strategies (Abdallah et al., <xref rid="B1" ref-type="bibr"> 2012 </xref> ). This approach allows <span class="trait">high</span> throughput and comprehensive investigation, by providing an overview of the organelle or cell-type proteomes (Glinski and Weckwerth, <xref rid="B62" ref-type="bibr"> 2006 </xref> ). For large-scale proteomics, a multidimensional chromatography technique called multidimensional <span class="trait">protein</span> identification technology (MudPIT) has been used to separate complex <span class="trait">protein</span> samples prior to <span class="gene">MS</span> analyses to increase the proteome coverage and dynamic range (Washburn et al., <xref rid="B203" ref-type="bibr"> 2001 </xref> ; Bachi and Bonaldi, <xref rid="B13" ref-type="bibr"> 2008 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">MS</span> based <span class="trait">protein</span> quantitation strategies may include untargeted quantitation, (database dependent or database independent <span class="trait">protein</span> identification) or targeted absolute quantitation, which are important for biomarker discovery and <span class="trait">protein</span> stoichiometries in <span class="trait">protein</span> complexes (Wienkoop et al., <xref rid="B214" ref-type="bibr"> 2008b </xref> ; Weckwerth, <xref rid="B209" ref-type="bibr"> 2011b </xref> ; Schmidt and Urlaub, <xref rid="B169" ref-type="bibr"> 2012 </xref> ). Relative quantitation is an unbiased, large-scale screening strategy, which is useful to detect proteins involved in unknown regulatory processes, <span class="trait">protein</span> modifications and mechanisms in systems biology. MS-based methods for <span class="trait">protein</span> quantitation can be label (stable-isotope <italic> in vivo </italic> or <italic> in vitro </italic> label) or non-label based (see Bachi and Bonaldi, <xref rid="B13" ref-type="bibr"> 2008 </xref> ; Abdallah et al., <xref rid="B1" ref-type="bibr"> 2012 </xref> ). Label based proteomics for relative quantitation consists of chemical labeling and metabolic labeling. The iTRAQ (isobaric tags for relative and absolute quantitation), is a chemical labeling approach and has been applied to study salinity stress tolerance in the phloem sap proteome of <italic> Cucumis sativus </italic> (Fan et al., <xref rid="B55" ref-type="bibr"> 2015 </xref> ), leaf proteome of <italic> Vitis vinifera </italic> to understand heat stress response (Liu et al., <xref rid="B122" ref-type="bibr"> 2014 </xref> ) and the root proteome of rice to investigate aluminum stress response (Wang et al., <xref rid="B202" ref-type="bibr"> 2014 </xref> ). The platforms available for relative quantitation include Mass Accuracy Precursor Alignment (MAPA), ProtMax, MASS WESTERN, and PROMEX (Hoehenwarter et al., <xref rid="B79" ref-type="bibr"> 2008 </xref> ; Weckwerth et al., <xref rid="B212" ref-type="bibr"> 2014 </xref> ). Recently, a rapid shot-gun LC-MS approach for relative quantitation called full-scan (FS) selective peptide extraction (Selpex) was successfully used for generating a reference library of targeted peptides from leaf tissue of Medicago (Castillejo et al., <xref rid="B37" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Selective reaction monitoring (SRM) or multiple reaction monitoring (MRM), which involves the use of triple-quadrupole (QQQ) <span class="gene">MS</span> is a quantitative, targeted and label-free approach, that complements untargeted shotgun methods due to its reliable quantitation of low abundance proteins in complex mixtures (Lange et al., <xref rid="B113" ref-type="bibr"> 2008 </xref> ; Wienkoop et al., <xref rid="B217" ref-type="bibr"> 2010b </xref> ; Schmidt and Urlaub, <xref rid="B169" ref-type="bibr"> 2012 </xref> ). This approach has been useful to measure a predetermined set of proteins that could constitute cellular networks or candidate biomarkers across samples consistent and reproducibly (Wienkoop et al., <xref rid="B217" ref-type="bibr"> 2010b </xref> ; Picotti and Aebersold, <xref rid="B148" ref-type="bibr"> 2012 </xref> ). In Medicago, the MRM approach was applied for the absolute quantitation of sucrose synthase isoforms and N-metabolism enzymes in symbiotic root nodules as well as in the analysis of nodule metabolism under <span class="trait">drought</span> stress (Wienkoop et al., <xref rid="B214" ref-type="bibr"> 2008b </xref> ; Larrainzar et al., <xref rid="B115" ref-type="bibr"> 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						PTMs are required for the functionality of proteins that regulate processes and the subcellular localization that could be analyzed using proteomics approaches (Pandey and Mann, <xref rid="B144" ref-type="bibr"> 2000 </xref> ; Seo and Lee, <xref rid="B173" ref-type="bibr"> 2004 </xref> ; Downes and Vierstra, <xref rid="B53" ref-type="bibr"> 2005 </xref> ). Although mostly identified through <span class="gene">MS</span> approaches, PTM analyses are not straightforward as <span class="trait">protein</span> identifications, the reason being peptide analyses do not show the expected molecular mass and therefore more <span class="trait">protein</span> samples are required (Pandey and Mann, <xref rid="B144" ref-type="bibr"> 2000 </xref> ). However, advancement in phosphoproteomics procedures, e.g., the enrichment of phosphopeptides using immobilized metal affinity chromatography (IMAC) or proteins with aluminum hydroxide or titanium dioxide called metal oxide affinity chromatography (MOAC) (Wolschin et al., <xref rid="B218" ref-type="bibr"> 2005 </xref> ) have improved the identification efficiency (Ndassa et al., <xref rid="B137" ref-type="bibr"> 2006 </xref> ; Chen et al., <xref rid="B41" ref-type="bibr"> 2010 </xref> ; Hoehenwarter et al., <xref rid="B78" ref-type="bibr"> 2013 </xref> ; Beckers et al., <xref rid="B16" ref-type="bibr"> 2014 </xref> ). Among the PTMs, phosphorylation is the major post-translational regulatory processes in all eukaryotes followed by ubiquitin and SUMO (Small Ubiquitin-like MOdifier) conjugations (Mazzucotelli et al., <xref rid="B128" ref-type="bibr"> 2008 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						PPIs provide functional knowledge about proteins by analysing the interacting partners and interactomics, which is the large-scale study of PPI networks using <span class="trait">high</span> throughput methods. This provides valuable insights for understanding cellular function, metabolism and signaling mechanisms (Braun et al., <xref rid="B26" ref-type="bibr"> 2013 </xref> ; Stasi et al., <xref rid="B178" ref-type="bibr"> 2015 </xref> ). Interactomes (maps of PPI) are constructed on the basis of experimental data and computational prediction of interactions (Stasi et al., <xref rid="B178" ref-type="bibr"> 2015 </xref> ). Approaches used in PPI analyses and mapping can be genetic, biochemical or proteomics-based. The strategies involve direct interactions with binary methods such as yeast two-hybrid (Y2H) or split ubiquitin, analyses of <span class="trait">protein</span> complexes with co-immunoprecipitation or affinity purification followed by <span class="gene">MS</span> (AP-MS), imaging and database analyses (Pandey and Mann, <xref rid="B144" ref-type="bibr"> 2000 </xref> ; Fukao, <xref rid="B60" ref-type="bibr"> 2012 </xref> ; Braun et al., <xref rid="B26" ref-type="bibr"> 2013 </xref> ; Stasi et al., <xref rid="B178" ref-type="bibr"> 2015 </xref> ). The identification of minor PPIs are also essential for studying transient interactions and proteins of low-abundance, which may be functionally relevant (Fukao, <xref rid="B60" ref-type="bibr"> 2012 </xref> ). In recent years, many proteins have been identified with advances in <span class="gene">MS</span> technologies and large-scale proteomics, however, the consequence of this is the increased number of false positive <span class="trait">protein</span> identifications. Therefore, for the accurate appraisal of PPI identification, the inclusion of independent experimental validation is required (Fukao, <xref rid="B60" ref-type="bibr"> 2012 </xref> ). Y2H system is sensitive in detecting transient and unstable interactions, and is suitable for PPI mapping and generating high-throughput data with a fine resolution to understand cellular process at the systemic level. However, it has limitations with incidences of false-positives and false-negative interactions. The PPIs are usually validated with techniques such as pull-down assays, co-immunoprecipitation, <italic> in-situ </italic> hybridization (von Mering et al., <xref rid="B198" ref-type="bibr"> 2002 </xref> ; Parrish et al., <xref rid="B145" ref-type="bibr"> 2006 </xref> ; Br&#x000fc;ckner et al., <xref rid="B32" ref-type="bibr"> 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolomics is a rapidly developing technology and at present metabolic fingerprinting and metabolite profiling approaches are being used. For wide coverage of the vast range of metabolites present, several analytical techniques involving separation and detection are implemented (Scherling et al., <xref rid="B166" ref-type="bibr"> 2010 </xref> ; Doerfler et al., <xref rid="B50" ref-type="bibr"> 2013 </xref> , <xref rid="B51" ref-type="bibr"> 2014 </xref> ). The separation technique is selective for certain groups of metabolites which includes gas chromatography (GC) for volatile and primary metabolites such as sugars and amino acids (Weckwerth, <xref rid="B208" ref-type="bibr"> 2011a </xref> ), LC for mainly secondary metabolites (Scherling et al., <xref rid="B166" ref-type="bibr"> 2010 </xref> ; Weckwerth, <xref rid="B208" ref-type="bibr"> 2011a </xref> ), capillary electrophoresis (CE) for ionic metabolites to be separated (Soga et al., <xref rid="B177" ref-type="bibr"> 2003 </xref> ; Soga, <xref rid="B176" ref-type="bibr"> 2007 </xref> ) and ultra-performance liquid chromatography (UPLC). UPLC is a powerful technique which has <span class="trait">high</span> resolution, sensitivity and throughput than conventional <span class="trait">high</span> performance liquid chromatography (HPLC). The <span class="gene">MS</span> analysers have been commonly used for metabolite profiling, particularly those that provide accurate mass measurements such as FTICR_MS, Orbitrap-MS or TOF-MS due to its fast scan times with improved deconvolution, run times and <span class="trait">high</span> mass accuracy (Scherling et al., <xref rid="B166" ref-type="bibr"> 2010 </xref> ; Weckwerth, <xref rid="B207" ref-type="bibr"> 2010 </xref> , <xref rid="B208" ref-type="bibr"> 2011a </xref> ; Obata and Fernie, <xref rid="B139" ref-type="bibr"> 2012 </xref> ; Arbona et al., <xref rid="B12" ref-type="bibr"> 2013 </xref> ; Doerfler et al., <xref rid="B50" ref-type="bibr"> 2013 </xref> , <xref rid="B51" ref-type="bibr"> 2014 </xref> ). GC-MS has been widely used in plant metabolomics research and electron impact (EI) supports strong interfacing of GC with <span class="gene">MS</span> which allows fragmentation patterns to be highly reproducible. Major metabolomics approaches and their applications in legumes are described below and are summarized in Table <xref ref-type="table" rid="T3"> 3 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolite profiling is the simultaneous measurement of all or a set of metabolites in a given sample. Several analytical techniques such as nuclear magnetic resonance (NMR), GC-MS, LC-MS, capillary electrophoresis&#x02013;MS (CE-MS) and Fourier transform infrared (FT-IR) spectroscopy have been reported for analysing the data from metabolite profiling (Sumner et al., <xref rid="B184" ref-type="bibr"> 2003 </xref> ; Weckwerth, <xref rid="B206" ref-type="bibr"> 2003 </xref> ; Shulaev, <xref rid="B174" ref-type="bibr"> 2006 </xref> ). The advantages and disadvantages of each technique for metabolite profiling have been discussed previously (Fiehn et al., <xref rid="B59" ref-type="bibr"> 2000 </xref> ; Roessner et al., <xref rid="B153" ref-type="bibr"> 2000 </xref> ; Sumner et al., <xref rid="B184" ref-type="bibr"> 2003 </xref> ; Weckwerth, <xref rid="B206" ref-type="bibr"> 2003 </xref> ; Weckwerth et al., <xref rid="B210" ref-type="bibr"> 2004a </xref> , <xref rid="B211" ref-type="bibr"> b </xref> ; Shulaev, <xref rid="B174" ref-type="bibr"> 2006 </xref> ). The use of metabolic profiling has been limited in crop legumes but this approach has been successfully demonstrated in model legumes. For example, in Medicago, untargeted quantitative <span class="gene">MS</span> approach was used to profile metabolites treated with rhizobial Nod factors to study the metabolic changes between the symbionts (Zhang et al., <xref rid="B230" ref-type="bibr"> 2013 </xref> ). The study showed decrease in concentration of (9)-HODE class of oxylipins upon Nod factor treatment <italic> in planta </italic> and together with jasmonic acid inhibited Nod factor signaling. This suggests an important role for the oxylipin pathway in Nod factor signaling in symbiosis. In a different study, under early salt and <span class="trait">drought</span> stress conditions, the involvement of certain metabolites in nutritional priming through symbiotic interaction of nodulated plants and N-fertilized Medicago has been reported (Staudinger et al., <xref rid="B180" ref-type="bibr"> 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolic fingerprinting is mainly used to identify metabolic signatures, for example, finding patterns associated with a particular stress response without precise quantitation/identification of different metabolites in the given sample. Features specific to a fingerprint can be identified using a variety of pattern recognition and multivariate statistical analysis (e.g., principal component analysis (PCA), self-organizing maps (SOMs) and hierarchical clustering, discriminant function analysis (DFA), ANOVA etc.) on the data (Sumner et al., <xref rid="B184" ref-type="bibr"> 2003 </xref> ). Different analytical techniques, including NMR spectroscopy (Krishnan et al., <xref rid="B108" ref-type="bibr"> 2005 </xref> ), <span class="gene">MS</span> (Goodacre et al., <xref rid="B65" ref-type="bibr"> 2003 </xref> ), and FT-IR (Johnson et al., <xref rid="B93" ref-type="bibr"> 2003 </xref> ) may be used to perform metabolic finger printing. Of these, <span class="gene">MS</span> is advantages over NMR spectroscopy because of the low sensitivity of NMR, which makes it difficult to detect low abundance cellular metabolites. On contrary, <span class="gene">MS</span> has <span class="trait">high</span> resolving power compared to NMR, providing higher sensitivity and lower detection limit but generates more complex spectrum because of its results in the form of discriminant ions which remains as a challenge for data validation. Moreover, a larger subset of metabolites associated with the phenotype can be identified using MS.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Research protocol: effect of natural S-equol on blood pressure and vascular function- a six-month randomized controlled trial among equol non-producers of postmenopausal women with prehypertension or untreated stage 1 hypertension</h2>
			<h3>PMID:26928904</h3>
				
					<h3>PMCID:4772692</h3>
					<h4>期刊:BMC Complementary and Alternative Medicine</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Liu </surname>  <given-names> Zhao-min </given-names>  </name>  <address>  <phone> 852 22528790 </phone>  <email> liuzhaomin@cuhk.edu.hk </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ho </surname>  <given-names> Suzanne C. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Yu-ming </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Yao Jie </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Huang </surname>  <given-names> Zhi-guan </given-names>  </name>  <address>  <phone> 8620 38024600 </phone>  <email> zhiguan1980@163.com </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ling </surname>  <given-names> Wen-hua </given-names>  </name>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <aff id="Aff1">  <label/> Division of Family Medicine and Primary Care, The Jockey Club School of Public Health and Primary Care, the Chinese University of Hong Kong, Hong Kong, SAR  </aff>  <aff id="Aff2">  <label/> Division of Epidemiology, The Jockey Club School of Public Health and Primary Care, the Chinese University of Hong Kong, Hong Kong, SAR  </aff>  <aff id="Aff3">  <label/> Department of Medical Statistics and Epidemiology, School of Public Health, Sun Yat-sen University, Guangzhou, PR China  </aff>  <aff id="Aff4">  <label/> School of Nursing, Faculty of Health and Social Science, the Hong Kong Polytechnic University, Hong Kong, SAR  </aff>  <aff id="Aff5">  <label/> Department of Sport and Health, Guangzhou Sport University, Guangzhou, PR China  </aff>  <aff id="Aff6">  <label/> Department of Nutrition, School of Public Health, Sun Yat-sen University, Guangzhou, PR China  </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Our previous RCT [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ] on soy showed a 4.2&#x000a0;mmHg net reduction (SD of change 10&#x000a0;mmHg) in <span class="gene">SBP</span> after 6-month soy <span class="trait">protein</span> and isoflavones supplementation relative to placebo group among 130 pre- and hypertensive postmenopausal women [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. Assuming a reduction of 5&#x000a0;mmHg of treatment effect on <span class="gene">SBP</span> (main effect), 63 subjects per group will <span class="trait">yield</span> a power of at least 80&#x000a0;% at 5&#x000a0;% level of significance (for a 2-side t-test). Allowing for 10&#x000a0;% non-compliance or drop out from the study, 69 subjects per group and a total 207 subjects (63&#x02009;&#x000d7;&#x02009;3&#x02009;&#x000d7;&#x02009;110&#x000a0;%) will be recruited. Similar attrition rates (7.8&#x000a0;% and 7.0&#x000a0;%) have been reported in our previous soy trials [ <xref ref-type="bibr" rid="CR45"> 45 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Data are presented as mean&#x02009;&#x000b1;&#x02009;standard deviation for continuous variables or number (%) for categorical variables. FFQ indicates food frequency questionnaires; PA indicates physical activity; Habitual drinking means drinking alcohol, tea or coffee more than 1 time per week; METs are multiples of resting metabolic rates and a MET-minute is computed by multiplying the <span class="gene">MET</span> score of an activity by the minutes performed. Dietary nutrients intakes were calculated mainly based on the China Food Composition Table 2002 and 2004. <italic> <span class="gene">SBP</span> </italic> systolic blood pressure, <italic> DBP </italic> diastolic blood pressure, <italic> TG </italic> triglycerides, <italic> BMI </italic> body mass index, <italic> WHR </italic> waist to hip ratio, <italic> Hs-CRP </italic> <span class="trait">high</span> sensitivity c-reactive <span class="trait">protein</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Data in support of the detection of genetically modified organisms <span class="gene">(GMOs)</span> in food and feed samples</h2>
			<h3>PMID:26958644</h3>
				
					<h3>PMCID:4773575</h3>
					<h4>期刊:Data in Brief</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Alasaad </surname>  <given-names> Noor </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alzubi </surname>  <given-names> Hussein </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kader </surname>  <given-names> Ahmad Abdul </given-names>  </name>  <email> ahmadabdulkader2@gmail.com </email>  <xref rid="cor1" ref-type="corresp"> &#x0204e; </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The results revealed for the first time in Syria the presence of GM foods and feeds with glyphosate-resistant trait of <italic> P35S </italic> promoter and <italic> NOS </italic> terminator in the imported soybean samples with <span class="trait">high</span> frequency (5 out of the 6 imported soybean samples). While, tests showed negative results for the local samples. Also, tests revealed existence of <span class="gene">GMOs</span> in two imported maize samples detecting the presence of 35S promoter and <italic> nos </italic> terminator. Nested PCR results using two sets of primers confirmed our data.
					</p>
				
					<p>&nbsp;&nbsp;
						Most of the DNA extracted by CTAB methods in this study showed a <span class="trait">high</span> molecular weight and <span class="trait">high</span> purity with A260/A280 ratios ranging from 1.8 to 2.0. The purity of DNA extracted from samples was confirmed by PCR amplification using soybean-specific ( <italic> lectin </italic> gene) and maize-specific ( <italic> zein </italic> gene) primers for samples derived from soybean and maize, respectively. These tests also indicated whether the other tested samples contained either soybean or maize materials, and if there was any contamination between DNA samples tested (15). The primer pair <span class="gene">GM03/GM04</span> with amplicon size of (118&#x000a0;bp) is specific for the single copy of <italic> lectin </italic> gene LE1was used. On the other hand, the primer pair zein3/zein4 was used which is specific for the native maize <italic> zein </italic> gene (Ze1, coding for a 10&#x000a0;kD <span class="trait">protein)</span> and yields a PCR product of 277&#x000a0;bp size (13).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Landscape of genomic diversity and trait discovery in soybean</h2>
			<h3>PMID:27029319</h3>
				
					<h3>PMCID:4814817</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dan Qiu </surname>  <given-names/>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zeng </surname>  <given-names> Peng </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Jiaying </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dai </surname>  <given-names> Lu </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Chengxuan </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yanjun </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Musket </surname>  <given-names> Theresa A. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shifeng </surname>  <given-names> Cheng </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xin </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Division of Plant Sciences and National Center for Soybean Biotechnology (NCSB), University of Missouri </institution> , Columbia 65211,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Beijing Genomics Institute-Shenzhen </institution> , Shenzhen, 518083,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Department of Molecular Microbiology and Immunology and Medical Research Office, School of Medicine, University of Missouri </institution> , Columbia, 65212 </aff>  <aff id="a4">  <label> 4 </label>  <institution> Department of Computer Science, Informatics Institute and Christopher S. Bond Life Sciences Center, University of Missouri </institution> , Columbia, 65211,  <country> USA </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Division of Plant Sciences and NCSB, University of Missouri-Fisher Delta Research Center </institution> , Portageville, MO, 63873,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 02 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean QTL information available in Soybase ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> ) was gathered to develop a gene list containing 15,502 genes associated with 96 QTL regions for the most important soybean agronomic traits. These genes were used for the diversity analysis. Approximately 22 sweep regions (496 genes underlying 26 QTL) were overlapped in the 15,502 gene regions ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 9 </xref> ). We identified the region that had the highest &#x003c0; <sub> w </sub> /&#x003c0; <sub> c </sub> value. Interestingly, 20 genes with the highest &#x003c0; <sub> w </sub> /&#x003c0; <sub> c </sub> values from the 27.82&#x02009;Mbp to 28.55&#x02009;Mbp region in chromosome <span class="gene">Gm01</span> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 8 </xref> ) were located in the QTL region for seed weight <xref ref-type="bibr" rid="b30"> 30 </xref> <xref ref-type="bibr" rid="b31"> 31 </xref> . Moreover, 44 genes that have the second highest &#x003c0; <sub> w </sub> /&#x003c0; <sub> c </sub> value from the 22.31&#x02009;Mbp to 24.26&#x02009;Mbp region in <span class="gene">Gm12</span> were reported in the QTL region for soybean <span class="trait">yield</span> <xref ref-type="bibr" rid="b32"> 32 </xref> . Overall, these sweep regions will provide key information for future soybean breeding and selection.
					</p>
				
					<p>&nbsp;&nbsp;
						We have selected a subset of soybean lines based on their contrasting total seed oil content, and they showed a wide range of oil content from 5&#x02013;24%. Lines, based on oil content, were categorized as low oil lines (5 to 16.9%), moderate oil lines (17 to 19.9%), and <span class="trait">high</span> oil lines (20 to 24%), comprising 26, 38, and 37 lines respectively. All the US PIs from elite backgrounds contain more than 19% oil. In contract, most of the wild relatives contain less than 14% oil, except for 2 lines ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 13 </xref> ). Candidate genes involved in acyl lipid metabolism were selected from diverse data sets such as the soybean acyl-lipid dataset <xref ref-type="bibr" rid="b38"> 38 </xref> , the Meta-QTL dataset, and the RNAseq dataset. We focused our subsequent analysis to identify the allelic variation in acyl-lipid genes and found a wide variation in non-syn-SNPs, indels, and CNVs. A total of 1,686 non-syn-SNPs were identified; out of those, 150 were associated with <span class="trait">high</span> and low oil lines that were significantly enriched for biological process of wax biosynthesis (WBT), fatty acid synthesis (FAS), and tri acyl glycerol biosynthesis and degradation (TAG_FAD, TAGB). To better define the allelic variation, we further narrowed down the search in 15 PIs lines with <span class="trait">high</span> and low oil content. We also found 40 SNPs in the candidate genes leading to premature stop codons (large effect SNPs). Analysis of the copy number variation suggested that approximately 23% of the predicted acyl-lipid genes in soybean had CNVs ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 14 </xref> ). The comparative analysis between <span class="trait">high</span> and low oil lines showed an increased copy number of lipid transfer <span class="trait">protein</span> (LPT; <italic> <span class="gene">Glyma16g31780,</span> <span class="gene">Glyma16g31840</span> </italic> , and <italic> <span class="gene">Glyma16g31540</span> </italic> ) in <span class="trait">high</span> oil lines, whereas the negative regulators (ABC transporter, <italic> <span class="gene">Glyma03g36310</span> </italic> ; Lipase 3, and <italic> <span class="gene">Glyma13g04561</span> </italic> ) of lipid biosynthesis genes showed more copies in low oil lines ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 11a </xref> ). The differential variation in SNP and copy number for lipid biosynthesis and lipid degradation genes between <span class="trait">high</span> oil and low oil lines might be associated with the differences in oil content within the PI lines. Recently, Wang <italic> et al </italic> . <xref ref-type="bibr" rid="b39"> 39 </xref> compared the sesame (55% oil) acyl-lipid genes with soybean and concluded that in families encoding lipid transfer proteins (LTP), midchain alkane hydroxylase was expanded by tandem duplication in sesame, whereas soybean has fewer numbers of those gene families.
					</p>
				
					<p>&nbsp;&nbsp;
						Similar to acyl-lipid, the allelic variation in seed <span class="trait">protein</span> related genes was investigated. Seed <span class="trait">protein</span> related genes were selected from the known QTL that is associated with <span class="trait">protein</span> content reported in Soybase.org and from the molecular dissection of QTL I <xref ref-type="bibr" rid="b40"> 40 </xref> and GWAS studies <xref ref-type="bibr" rid="b41"> 41 </xref> . We found 42 SNPs in 21 genes that were associated with <span class="trait">protein</span> content. The genomic block on <span class="gene">Gm20</span> (24.5 to 32.9 Mbp, 8.1 Mbp) is defined as a major <span class="trait">protein</span> QTL for <span class="trait">protein</span> content <xref ref-type="bibr" rid="b40"> 40 </xref> <xref ref-type="bibr" rid="b41"> 41 </xref> , and we found that the copy number variations in candidate genes (HSP, <italic> <span class="gene">Glyma20g19680</span> </italic> ; ammonium transporter, <italic> <span class="gene">Glyma20g21030</span> </italic> ; ethylene receptor, and <italic> <span class="gene">Glyma20g21780</span> </italic> ) were associated with <span class="trait">protein</span> content ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table 15 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. 11b </xref> ). The 2.4&#x02009;Mbp genomic region (within 8.1&#x02009;Mbp) defined in the GWAS study <xref ref-type="bibr" rid="b41"> 41 </xref> was further studied to determine haplotype variation, and we found 3 major clusters that were significantly associated with <span class="trait">protein</span> content.
					</p>
				
					<p>&nbsp;&nbsp;
						Diversity pattern in <span class="gene">Gm15</span> for <italic> G. soja </italic> , landrace and elite cultivars associated with <span class="trait">yield</span> and flower number related QTLs <xref ref-type="bibr" rid="b48"> 48 </xref> (4.5&#x02013;4.9&#x02009;Mb). Previously this region was identified as a large QTL (3.1&#x02013;6.7&#x02009;MB) and contained 47 genes. In the present study this region narrow down to 4.5&#x02013;4.9&#x02009;Mb and contains 5 genes, in which <italic> <span class="gene">Glyma15g00750</span> </italic> showed relatively lower &#x003c0; in cultivated soybean, and functionally this gene is annotated to bHLH/Circadian-protein, suggesting that this gene might be involved in yield-related traits in cultivated soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Potential for Engineering Enhanced Functional-Feed Soybeans for Sustainable Aquaculture Feed</h2>
			<h3>PMID:27092158</h3>
				
					<h3>PMCID:4820450</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Herman </surname>  <given-names> Eliot M. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/46361/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schmidt </surname>  <given-names> Monica A. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/337084/overview"/>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybeans null for bioactive seed proteins that have been isolated from the USDA soybean collection include Kunitz trypsin inhibitor (KTI) null ( <xref rid="B40" ref-type="bibr"> Orf and Hymowitz, 1979 </xref> ; <xref rid="B23" ref-type="bibr"> Hymowitz, 1986 </xref> ), soybean agglutinin (SBA) null ( <xref rid="B41" ref-type="bibr"> Orf et al., 1978 </xref> ; <xref rid="B15" ref-type="bibr"> Goldberg et al., 1983 </xref> ) and immunodominant soybean allergen P34 <span class="trait">protein</span> nulls ( <xref rid="B26" ref-type="bibr"> Josephs et al., 2006 </xref> ). Each of these nulls has the potential to partially address concerns of soybean feed/food consumption and stacked together these traits can form a platform for engineering enhanced soybean varieties. A <italic> Triple Null </italic> stack of the recessive nulls of KTI, SBA, and P34 in the standard cultivar <italic> Williams </italic> (the archetype used to elucidate the soybean genome) has been produced ( <xref rid="B50" ref-type="bibr"> Schmidt et al., 2015a </xref> ). The parental components of the <italic> Triple Null (p34,kti,le </italic> ) stack include frame shift mutants of P34 ( <xref rid="B26" ref-type="bibr"> Josephs et al., 2006 </xref> ; <xref rid="B3" ref-type="bibr"> Bilyeu et al., 2009 </xref> ) and <span class="gene">KTI</span> ( <xref rid="B25" ref-type="bibr"> Jofuku et al., 1989 </xref> ) that still accumulate a small amount of authentic <span class="trait">protein</span> due to a shift of the start methionine inward in the open reading frame <xref rid="B50" ref-type="bibr"> Schmidt et al. (2015a) </xref> . The <italic> Lectin </italic> (LE) null results from a large transposon and does not produce a functional transcript or lectin <span class="trait">protein</span> accumulation ( <xref rid="B41" ref-type="bibr"> Orf et al., 1978 </xref> ; <xref rid="B15" ref-type="bibr"> Goldberg et al., 1983 </xref> ). Proteomic analysis of <italic> Triple Null </italic> shows that the line lacks this trio of bioactive proteins while retaining the full complement of other proteome constituents without any other collateral bioactive <span class="trait">protein</span> alterations ( <xref rid="B50" ref-type="bibr"> Schmidt et al., 2015a </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Variations in the amount of astaxanthin produced in different plant systems can be attributed to the inherent ability of certain plant organs to accumulate carotenoids, leaf and flower tissue more than seeds, and the catalytic attributes of the carotenoid biosynthetic enzymes. Enzymes in the carotenoid pathway have been reported to act upon an assortment of various substrates, often resulting in a mixture of carotenoids produced rather than a majority of a specific sought metabolite. Following on research showing that enhanced &#x003b2;-carotene levels could be achieved by the overexpression of a bacterial <italic> Pantoea </italic> phytoene synthase gene in soybean seeds ( <bold> Figure <xref ref-type="fig" rid="F2"> 2 </xref> </bold> ) ( <xref rid="B51" ref-type="bibr"> Schmidt et al., 2015b </xref> ), this construct was then combined individually with two ketolase genes from different sources: one from bacteria <italic> Brevundimonas </italic> <span class="gene">spp</span> and the other algae known for its <span class="trait">high</span> concentration of astaxanthin <italic> Haematococcus pluvialis. </italic> A mixture of carotenoids resulted with only one line using the bacterial ketolase producing detectable levels of astaxanthin ( <xref rid="B42" ref-type="bibr"> Pierce et al., 2015 </xref> ). Further emphasizing the importance of enzyme choice, three &#x003b2;-carotene ketolase enzymes were used from different sources in <italic> Arabidopsis </italic> leaves and found a wide variety of successful production of the desired astaxanthin carotenoid ( <xref rid="B59" ref-type="bibr"> Zhong et al., 2011 </xref> ). Up to 2 mg/g was produced when the ketolase gene from <italic> Chlamydomonas </italic> , compared to 0.24 mg/g when <italic> Chlorella </italic> was the gene source and non-detectable levels when <italic> Haematococcus </italic> ketolase gene was used. These findings emphasis the importance of enzyme selection in the biosynthesis of carotenoids giving promise to the future production of sustainable and commercially viable levels of this carotenoid pigment in a suitable plant delivery system.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:New insights into the post-translational modification of multiple phosphoenolpyruvate carboxylase isoenzymes by phosphorylation and monoubiquitination during sorghum seed development and germination</h2>
			<h3>PMID:27194739</h3>
				
					<h3>PMCID:4892742</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ruiz-Ballesta </surname>  <given-names> Isabel </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Peptide mass fingerprinting by MALDI-TOF <span class="gene">MS</span> of the p110 and p107 polypeptides immunoprecipitated using anti-C <sub> 4 </sub> PTPC revealed that different PTPC isoenzymes co-exist in developing and germinating sorghum seeds ( <xref ref-type="table" rid="T2"> Table 2 </xref> ; asterisks denote confirmation of the identification via LC MS/MS sequencing of tryptic peptides). SbPPC3 was the most abundant during the course of the life cycle of the seeds in both p107 and p110. SbPPC3 was highly abundant in the embryo ( <xref ref-type="table" rid="T2"> Table 2 </xref> ) and less abundant in the aleurone/endosperm of the germinating seed; however, SbPPC3 was the only isoenzyme detected in this tissue ( <xref ref-type="table" rid="T2"> Table 2 </xref> ). The housekeeping SbPPC2 ( <xref rid="CIT0013" ref-type="bibr"> Lepiniec <italic> et al. </italic> , 1993 </xref> , <xref rid="CIT0014" ref-type="bibr"> 1994 </xref> ) was present in early development only as a p107 (i.e. probably non-monoubiquitinated) subunit ( <xref ref-type="table" rid="T2"> Table 2 </xref> , stages I and III) and was identified as a p110 and p107 subunit in stage VI ( <xref ref-type="table" rid="T2"> Table 2 </xref> ). SbPPC2 was also present as a monoubiquitinated p110 subunit in the embryo of dry and 48-h post-imbibition seeds, but was not identified in the aleurone ( <xref ref-type="table" rid="T2"> Table 2 </xref> ). Finally, the <italic> SbPPC4 </italic> isogene was shown to be translated into the corresponding p107 PTPC subunits in early development ( <xref ref-type="table" rid="T2"> Table 2 </xref> , stages I and III), in agreement with the transcript profiling results of <xref ref-type="fig" rid="F3"> Fig. 3E </xref> . In addition, SbPPC4 was also detected in p110, indicating the probable monoubiquitination of this PTPC isoenzyme. Although the precise physiological function of SbPPC4 remains unclear, our results support a role for this isoenzyme during the early stages of seed development. During the phase of cellularization SbPPC4 is hypothesized to generate the C-skeletons needed as precursors for synthesis of new components such as amino acids/proteins, and membrane or storage lipids. Finally, SbPPC5 was detected only as p110 polypeptides in stage I ( <xref ref-type="table" rid="T2"> Table 2 </xref> ). SbPPC6 polypeptides were never detected by MS/MS analysis, in agreement with the absence of <italic> SbPPC6 </italic> expression during sorghum seed development and germination. Nor was a <span class="trait">high</span> molecular mass Class-2 PEPC hetero-octamer evident when clarified extracts from different stages of development and germination were subjected to non-denaturing PAGE followed by in-gel PEPC activity staining (see <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw186/-/DC1"> Supplementary Fig. S3 </ext-link> ). However, tetrameric and dimeric Class-1 PEPC oligomers were observed, with the dimeric form being the most abundant <italic> in vitro </italic> ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw186/-/DC1"> Supplementary Fig. S3A </ext-link> ). This dimeric form was not altered by de-ubiquitination of the enzyme after incubation with 2 &#x000b5;M of the catalytic subunit of ubiquitin specific protease-2 (USP- <sub> 2c </sub> ) ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erw186/-/DC1"> Supplementary Fig. S3B </ext-link> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Proteomic Analysis of Soybean Leaves and Roots by iTRAQ Provides Insights into Response Mechanisms to Short-Term Salt Stress</h2>
			<h3>PMID:27200046</h3>
				
					<h3>PMCID:4850148</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ji </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/312223/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cong </surname>  <given-names> Ru </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/343941/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Sheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/335890/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Rui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/343932/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qin </surname>  <given-names> Zhiwei </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yanjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/343931/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xiaolin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/343938/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Sixue </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/40384/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/343943/overview"/>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The raw MS/MS data files acquired from the Orbitrap were processed by a thorough database searching considering biological modification and amino acid substitution against the Uniprot Soybean database with 71,042 entries (downloaded on May 16, 2013), using Proteome Discoverer 1.4 (Thermo Scientific Inc., Bremen, Germany) with the SEQUEST algorithm. The following parameters were used for searching: lowest and highest charge: +2 and +5, respectively; minimum and maximum precursor mass: 300 and 6000 Da, respectively; minimum S/N ratio: 3; enzyme: trypsin; maximum missed cleavages: 1; FDR &#x02266; 0.01; mass tolerance: 10 ppm for precursor ions and 0.5 Da for fragment ions; dynamic modifications: phosphorylation (+79.966 (S,T,Y)), carbamidomethyl (+57.021 Da (C)), oxidation (+15.995 Da (M)), carbamidomethyl (+57.021 Da (C)). The N-terminal modification was set for iTRAQ8plex (+304.205 Da). The Proteome Discoverer results files (.msf) were uploaded to ProteoIQ 2.6 (NuSep) software for further filtering. Peptide probability was applied to filter peptide assignments obtained from MS/MS database searching results using predictable false identification error rate. <span class="trait">Protein</span> probability was used to filter proteins with the null hypothesis that the database matching is random and taking into account of the peptide probability for all the peptides apportioned to that <span class="trait">protein</span> (Koh et al., <xref rid="B23" ref-type="bibr"> 2015 </xref> ). Proteins detected with at least three spectral counts, FDR &#x02264; 5%, 95% probability and listed as top scoring proteins are considered as <span class="trait">high</span> confidence matches and are presented in the results. To be identified as being significantly differentially expressed, a <span class="trait">protein</span> should be quantified with at least three peptides in each experimental replicate, a <italic> p </italic> -value smaller than 0.05 and fold change greater than 1.3 or less than 0.7. The <span class="gene">MS</span> proteomics data have been deposited in the ProteomeXchange Consortium via the PRIDE partner repository (Vizcaino et al., <xref rid="B52" ref-type="bibr"> 2013 </xref> ) with the data set identifier PXD002851. The functional annotation of proteins found was determined by Blast2GO (Bioinformatics Department, CIPF, Valencia, Spain), and then grouped on the basis of their biological functions from Gene Ontology (GO) terms combined with information from the literature (Zhang et al., <xref rid="B62" ref-type="bibr"> 2012 </xref> ; Zhao et al., <xref rid="B64" ref-type="bibr"> 2013 </xref> ). The protein-protein interaction network was constructed using the String program ( <ext-link ext-link-type="uri" xlink:href="http://string-db.org"> http://string-db.org </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To predict the relationship among all these identified DEPs in soybean leaves and roots, a protein-protein interaction (PPI) networks were generated using the web-tool STRING 9.1. A total of 104 differentially abundant proteins represented by 72 unique proteins from soybean were shown in the PPI network (Figure <xref ref-type="fig" rid="F5"> 5 </xref> , Table <xref ref-type="supplementary-material" rid="SM6"> S6 </xref> ) based on the published literature and other experimental evidence (Zhao et al., <xref rid="B63" ref-type="bibr"> 2015 </xref> ). Four functional modules forming tightly-connected clusters were illuminated in the network (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Nodes in different colors belong to four main groups. Stronger associations are represented by thicker lines. In Module 1 (blue nodes), six <span class="trait">protein</span> synthesis related proteins, five amino acid metabolism related proteins, two ATP synthases, a TCP as well as a MAPK appeared closely linked. This implied that amino acid metabolism, <span class="trait">protein</span> synthesis and energy supply were active and cooperated closely in soybean seedlings under salt stress. Module 2 (red nodes) included multiple enzymes involved in the TCA cycle, glycolysis, fatty acid biosynthesis and nitrogen metabolism. These linked proteins indicated that a synergistic system for carbon and nitrogen metabolism may play important roles in salt response. Moreover, <span class="gene">PPC1</span> and PPC16, two key enzymes in organic acid metabolism were linked with module 3 (yellow nodes), which included ten proteins functioning in photosynthesis, carbohydrate and energy metabolism. Furthermore, proteins involved in <span class="trait">protein</span> folding, transporting, ROS scavenging, as well as some signaling components were assigned in Model 4 (green nodes). This indicated that proteins in this network played important functions in redox homeostasis, response to stress, signal transduction and <span class="trait">protein</span> metabolism.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, three <span class="trait">protein</span> kinases and two <span class="trait">protein</span> phosphatases were identified, which showed enhanced expression levels under salt stress (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Specifically, mitogen-activated <span class="trait">protein</span> kinase homolog mmk2-like was up-regulated more than 15-fold under salt stress. The MAPK family is reported to play various roles in intra- and extra-cellular signaling in plants by transferring the information from sensors to responsers, which act as points of convergence in abiotic stress signaling (Sinha et al., <xref rid="B44" ref-type="bibr"> 2011 </xref> ). The serine threonine <span class="trait">protein</span> phosphatase <span class="gene">PP1</span> also showed enhanced expression level under salt stress in soybean roots. Although there is no direct evidence in support of the significant role of <span class="gene">PP1</span> in plant salt tolerance, the <span class="gene">PP1</span> regulatory <span class="trait">protein</span> RICE SALT SENSITIVE 1 (RSS1) was identified recently through a combined approach of genetic screening for salt tolerance in rice and yeast two-hybrid screening, and the loss of RSS1 results in short root and dwarf phenotypes under <span class="trait">high</span> salt (Ogawa et al., <xref rid="B35" ref-type="bibr"> 2011 </xref> ). Similar to the expression patterns at the translational level, up-regulation of MMK2 and <span class="gene">PP1</span> were also validated by semi-quantitative RT-PCR analysis, which indicated that these proteins were regulated at the transcriptional level as well (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Finally, we identified 15 vesicle trafficking-related proteins that exhibited differential expression patterns in soybean seedlings under salt stress (Table <xref ref-type="table" rid="T2"> 2 </xref> ). Aquaporin (AQP) proteins function in transporting water and other small neutral solutes or gasses through the biological membranes, which is crucial for plants to survive in <span class="trait">drought</span> or salt stress conditions (Sade et al., <xref rid="B42" ref-type="bibr"> 2010 </xref> ). AQP contains two subfamilies, the plasma membrane intrinsic proteins (PIPs) and the tonoplast intrinsic proteins (TIPs), which are most abundant in the plasma membrane and vacuolar membrane, respectively. Many genes encoding PIPs have been identified from different plant species, and overexpression of these genes have been reported to enhance plants salt tolerance (Sade et al., <xref rid="B42" ref-type="bibr"> 2010 </xref> ; Hu et al., <xref rid="B20" ref-type="bibr"> 2012 </xref> ; Liu et al., <xref rid="B27" ref-type="bibr"> 2013 </xref> ; Xu et al., <xref rid="B57" ref-type="bibr"> 2014 </xref> ; Sreedharan et al., <xref rid="B47" ref-type="bibr"> 2015 </xref> ). Here, two isoforms of <span class="gene">PIP</span> showed different expression patterns in soybean leaves and roots in response to salt stress (Table <xref ref-type="table" rid="T2"> 2 </xref> ). This may be attributed to tissue- and time-specific expression manners of different <span class="gene">PIP</span> isoforms under salt stress. In addition, an annexin-like <span class="trait">protein</span> was up-regulated by salt stimulus in soybean leaves in this study. Annexin functions as a Ca <sup> 2+ </sup> -permeable channel in the plasma membrane to form a ROS-stimulated passive Ca <sup> 2+ </sup> transport pathway (Laohavisit et al., <xref rid="B24" ref-type="bibr"> 2010 </xref> ), which was reported to be induced by salinity in variety of plant species based on proteomics data (Lee et al., <xref rid="B25" ref-type="bibr"> 2004 </xref> ; Manaa et al., <xref rid="B33" ref-type="bibr"> 2013 </xref> ), indicating the significance of this <span class="trait">protein</span> in plant salt stress tolerance. In summary, the identified membrane and transport related proteins were consistent with the physiological processes of maintaining ion homeostasis and membrane stability, which are crucial for plant growth during salt stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the redox related proteins, plants have developed cross-tolerance mechanisms to cope with different stresses (Zhang et al., <xref rid="B62" ref-type="bibr"> 2012 </xref> ). From our iTRAQ data, some biotic stress-related proteins were induced under salt stress conditions, such as disease <span class="trait">resistance</span> <span class="trait">protein</span> rpp13 and pathogenesis-related <span class="trait">protein</span> class 10 (PR10), which mediates tolerance to heavy metals (Wang Z. Q. et al., <xref rid="B55" ref-type="bibr"> 2014 </xref> ) and pathogen attack (Coumans et al., <xref rid="B8" ref-type="bibr"> 2009 </xref> ). Interestingly, several major latex proteins (MLPs) were up-regulated in soybean roots but down-regulated in soybean leaves under salt stress (Tables <xref ref-type="supplementary-material" rid="SM2"> S2 </xref> , <xref ref-type="supplementary-material" rid="SM3"> S3 </xref> ), which was in agreement with the proteomic findings of soybean root tips under flooding (Yin X. et al., <xref rid="B59" ref-type="bibr"> 2014 </xref> ). MLPs were found only in plants and associated with fruit and flower development and in pathogen defense responses. So far, only one study reported that transcription of the <italic> mlp </italic> gene in cotton was rapidly induced by NaCl and overexpression of this gene enhanced salt tolerance in <italic> Arabidopsis </italic> (Chen and Dai, <xref rid="B6" ref-type="bibr"> 2010 </xref> ). However, the specific biological function of <span class="gene">MLP</span> and whether up-regulation of <span class="gene">MLP</span> correlates with enhanced salt tolerance in soybean plants are unknown and to the best of our knowledge, it may represent a novel salt-stress-responsive <span class="trait">protein</span> in soybean plants. Furthermore, some <span class="trait">drought</span> stress-related proteins, e.g., dehydrin and desiccation protectant <span class="trait">protein</span> lea14 homolog, also responded to salt stress in our study (Tables <xref ref-type="supplementary-material" rid="SM4"> S4 </xref> , <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). These proteins provide novel insights into the understanding of the cross-tolerance mechanisms in soybean seedlings in response to biotic and abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						A large number of DEPs were found to be involved in nitrogen and amino acid metabolism. Glutamine synthetase, the key enzyme in plant NH <sup> 4+ </sup> metabolism, has been reported to play an important role in enhancing rice tolerance to salt and chilling stresses (Hoshida et al., <xref rid="B18" ref-type="bibr"> 2000 </xref> ). We also found it up-regulated in soybean leaves under salt stress. Cysteine synthase is responsible for the final step in cysteine biosynthesis, the key limiting step in producing glutathione (GSH), which is involved in <span class="trait">resistance</span> to adverse stresses. Liu et al. (Liu et al., <xref rid="B28" ref-type="bibr"> 2014 </xref> ) used comparative proteomic methods to show that cysteine synthase was induced in a salt-tolerant rice cultivar but down regulated in salt-sensitive rice leaves. In this study, four cysteine synthases were identified to be up-regulated in soybean roots (Table <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ), and appeared tightly linked with other proteins in the soybean PPI network (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Aspartate aminotransferase (AAT) catalyzes the conversion of &#x003b1;-ketoglutarate and aspartate to glutamate and oxaloacetate (Hodges, <xref rid="B17" ref-type="bibr"> 2002 </xref> ). Proteomic analysis has found the salt also induced <span class="gene">AAT</span> in rice roots (Nam et al., <xref rid="B34" ref-type="bibr"> 2012 </xref> ), which is consistent with the results in this study. There were many other salt-responsive proteins related to amino acid metabolism identified only in soybean roots, such as aminotransferase-like, glutamate decarboxylase, alanine aminotransferase, serine hydroxymethyltransferase, phosphoserine aminotransferase, asparagine synthetase and serine hydroxymethyltransferase (Table <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). These results indicated that amino acid and nitrogen metabolism were enhanced in soybean seedling leaves and roots under salt stress. Significantly, 14 lipoxygenases showed enhanced levels both in soybean leaves and roots under salt stress, which suggested that lipid metabolism changes under salt stress and may play important roles in soybean seedling growth.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential Contribution of Malic Enzymes during Soybean and Castor Seeds Maturation</h2>
			<h3>PMID:27347875</h3>
				
					<h3>PMCID:4922584</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Gerrard Wheeler </surname>  <given-names> Mariel Claudia </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Arias </surname>  <given-names> Cintia Luc&#x000ed;a </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Righini </surname>  <given-names> Silvana </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Badia </surname>  <given-names> Mariana Beatriz </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Andreo </surname>  <given-names> Carlos Santiago </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Drincovich </surname>  <given-names> Mar&#x000ed;a Fabiana </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saigo </surname>  <given-names> Mariana </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						(A). <span class="trait">Protein</span> sequences were aligned using Mega 5.10 and the phylogenetic tree was constructed by the neighbor joining method. The numbers indicate the statistical significance of each branch obtained by bootstrap analysis. Relative level of the transcripts of <italic> NADP-ME </italic> genes in maturing soybean embryos (B, D and F) and castor bean endosperms (C, E and G) determined by qRT-PCR. The functionality of primers for <span class="gene">GmNADP-ME1.2</span> and 1.3 was verified using soybean genomic DNA. The primers for <span class="gene">GmNADP-ME2.2</span> also anneals with the splicing version <span class="gene">GmNADP-ME2.2spl.</span> The actin genes <span class="gene">Glyma04g39380</span> and Rco30206.m000761 were used as reference in soybean and castor, respectively. The values are the average of at least two independent experiments &#x000b1; SD. For each transcript, values with the same letter are not significantly different (p &#x0003c; 0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						(A). <span class="trait">Protein</span> sequences were aligned using Mega 5.10 and the phylogenetic tree was constructed by the neighbor joining method. The numbers indicate the statistical significance of each branch obtained by bootstrap analysis. Relative level of the transcripts of <italic> NAD-ME </italic> genes in maturing soybean embryos (B and D) and castor bean endosperms (C and E) determined by qRT-PCR. The actin genes <span class="gene">Glyma04g39380</span> and Rco30206.m000761 were used as reference in soybean and castor, respectively. The values are the average of at least two independent experiments &#x000b1; SD. For each transcript, values with the same letter are not significantly different (p &#x0003c; 0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">high</span> number of NADP-ME and NAD-ME genes in soybean could be indicating two contrasting situations: functional redundancy or functional differentiation. Then, we extended the expression analysis to other non-seed samples in order to determine if any of the genes is specifically expressed in seeds and if any of the non-expressed genes is actually active in other organs. The samples analyzed included germinating seeds and leaves, roots and stems from young plants. The putative transcripts <span class="gene">GmNADP-ME1.2</span> and <span class="gene">GmNADP-ME1.3,</span> absent in soybean embryos, were not detected in neither of the new samples tested probably indicating inactive genes or an expression pattern very confined to temporal or spatial signals. Otherwise, <span class="gene">GmNADP-ME2.2,</span> <span class="gene">GmNADP-ME2.2spl</span> and <span class="gene">GmNADP-ME3.3</span> were only found in maturing embryos raising the possibility that they are embryo specific transcripts. NAD-ME genes and <span class="gene">GmNADP-ME3.1</span> were detected in similar levels in the other samples compared to embryo samples ( <xref ref-type="supplementary-material" rid="pone.0158040.s001"> S1 Fig </xref> ). <span class="gene">GmNADP-ME1.1</span> transcript was more abundant in embryos but <span class="gene">GmNADP-ME2.1</span> and <span class="gene">GmNADP-ME3.2</span> were detected at higher levels in non-embryo samples ( <xref ref-type="supplementary-material" rid="pone.0158040.s001"> S1 Fig </xref> ). Overall, these observations show that genes of lineages 2 and 3 may have some functional specialization. Nevertheless, a broader analysis is needed to arrive to a certain conclusion about the physiological significance of each lineage in different organs.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome Profile of Near-Isogenic Soybean Lines for &#x003b2;-Conglycinin &#x003b1;-Subunit Deficiency during Seed Maturation</h2>
			<h3>PMID:27532666</h3>
				
					<h3>PMCID:4988716</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> An </surname>  <given-names> Lixin </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Yanjing </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Hongxiu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ren </surname>  <given-names> Hongbo </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Xiaoshuang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Krishnan </surname>  <given-names> Hari B. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0001-7273-735X </contrib-id>  <name>  <surname> Liu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The top 20 genes that showed high-level differential expression related to the &#x003b1;-null mutation were ranked and are shown in <xref ref-type="table" rid="pone.0159723.t004"> Table 4 </xref> . <span class="gene">Glyma17g34220</span> (encoding alpha crystalline) and another six genes <span class="gene">(Glyma13g11840,</span> <span class="gene">Glyma13g1189,</span> <span class="gene">Glyma13g11961,</span> <span class="gene">Glyma13g12033,</span> Novel00815, and Novel01348), which were not annotated, were all downregulated at 18 DAF. Expression of <span class="gene">Glyma13g11840</span> (no annotation) was downregulated by 12.16-fold, which was the most highly downregulated of all the DEGs identified in our data. <span class="gene">Glyma20g28660</span> showed the highest differential expression related to &#x003b1;-null at 25, 35, and 50 DAF, and was downregulated by 9.32-fold at 25 DAF, 9.49-fold at 35 DAF, and 8.8-fold at 50 DAF. At 55 DAF, two genes <span class="gene">(Glyma06g02690</span> and <span class="gene">Glyma04g02660)</span> encoding a Gibberellin-regulated <span class="trait">protein</span> and two genes <span class="gene">(Glyma10g39760</span> and <span class="gene">Glyma07g40110)</span> encoding Concanavalin A-like lectins were downregulated by 9.03-fold and 7.91-fold, and by 8.47-fold and 6.93-fold, respectively. In addition, another four genes (Novel01985, Novel02415, <span class="gene">Glyma15g10450,</span> and <span class="gene">Glyma16g03600)</span> were upregulated at 55 DAF. Expression of <span class="gene">Glyma15g10450,</span> encoding a <span class="trait">protein</span> arginine N-methyltransferase, was upregulated by 8.05-fold; and <span class="gene">Glyma16g03600,</span> encoding an aminotransferase that takes part in cysteine and methionine metabolism, was upregulated by 7.56-fold. We hypothesized that these genes are putatively&#x003b1;-null-related transcripts. Based on obtained DEGs information and bioinformatics, we will conduct further studies focused on gene function identification of the above-mentioned DEGs.
					</p>
				
					<p>&nbsp;&nbsp;
						The &#x003b1;-subunit is one of the major components of soybean seed storage proteins; therefore, the complete deficiency of the &#x003b1;-subunit should change the gene&#x02019; expression profiles and metabolic pathways during seed maturation. NILs are valuable genetic resources to identify genomic regions and alleles responsible for trait variation [ <xref rid="pone.0159723.ref045" ref-type="bibr"> 45 </xref> ], and are also particularly suitable for genetic analyses of transcriptome and proteome variations. To further understand the potential mechanisms involved in the regulation of the &#x003b1;-null mutation, we have used RNA-seq to investigate global gene expression changes over five stages of soybean cotyledon development in seeds of &#x003b1;-subunit-deficient NIL lines ( <italic> cgy-2- </italic> NIL). We have identified several critical genes that were possibly associated with the &#x003b1;-null mutation. Only <span class="gene">Glyma20G28660</span> was annotated as the &#x003b1;-subunit gene of &#x003b2;-conglycinin, and appeared to be significantly downregulated throughout the three green stages (25, 35, and 50 DAF) of development studied here. Surprisingly, at 55 DAF (the desiccating stage of development), the number of DEGs was the highest. This observation is consistent with the results of a previous report [ <xref rid="pone.0159723.ref026" ref-type="bibr"> 26 </xref> ], in which many genes showed peak expression at the latter stages of seed maturation. These genes were annotated as being TFs or related to <span class="trait">protein</span> degradation [ <xref rid="pone.0159723.ref026" ref-type="bibr"> 26 </xref> ]. Our analyses have also resulted in the identification of interesting late expressed DEGs (at 55 DAF). In particular, <span class="gene">Glyma16G03600,</span> which is involved in cysteine and methionine metabolism, was upregulated by 7.56-fold in &#x02018; <italic> cgy-2- </italic> NIL&#x02019;, and <span class="gene">Glyma04G02660</span> and <span class="gene">Glyma06G02690,</span> which were annotated as gibberellin-regulated <span class="trait">protein</span> genes. We also predicted many novel candidate genes that were associated with the &#x003b1;-null mutation, which provide a strong basis for future research on determining the molecular mechanism of &#x003b1;-subunit-null deficiency. To determine whether the differential expression of genes such as <span class="gene">Glyma16G03600,</span> <span class="gene">Glyma04G02660,</span> and <span class="gene">Glyma06G02690</span> have a direct relation to &#x003b1;-subunit-null mutation, the function of these DEGS will be studied by RNA interference or by overexpression in transgenic plants in the future. This could lead to a better understanding of the molecular regulation of storage <span class="trait">protein</span> subunit accumulation in the &#x003b1;-null mutant.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean seeds contain between 35 and 45% <span class="trait">protein</span> on a dry weight basis, of which about 70% consists of the two major storage proteins, 7S globulin (&#x003b2;-conglycinin) and 11S (glycinin). Development changes in the synthesis of &#x003b2;-conglycinin and glycinin have been described previously [ <xref rid="pone.0159723.ref050" ref-type="bibr"> 50 </xref> , <xref rid="pone.0159723.ref025" ref-type="bibr"> 25 </xref> , <xref rid="pone.0159723.ref026" ref-type="bibr"> 26 </xref> ]. In the present study, the expression of various subunit gens of &#x003b2;-conglycinin and glycinin in both <italic> cgy-2 </italic> -NIL and DN47 showed similar developmental expression patterns: they presented a bell-shaped pattern of expression that started at 18&#x02013;25 DAF, reached a maximum at 35 DAF or 50 DAF, and declined rapidly thereafter. The &#x003b1;&#x02032;-and &#x003b1;-subunit genes of &#x003b2;-conglycinin reached their expression peaks (at 35 DAF) before the &#x003b2;-subunit genes of &#x003b2;-conglycinin and five glycinin, <italic> <span class="gene">Gy1</span> </italic> &#x02013; <italic> <span class="gene">Gy5</span> </italic> subunit genes (at 50 DAF). These results were similar to earlier observations [ <xref rid="pone.0159723.ref050" ref-type="bibr"> 50 </xref> , <xref rid="pone.0159723.ref025" ref-type="bibr"> 25 </xref> , <xref rid="pone.0159723.ref026" ref-type="bibr"> 26 </xref> ]. However, the expression levels of 18 cupin genes in <italic> cgy-2 </italic> -NIL were significantly different to those in DN47. The &#x003b1;-null mutation caused almost all the &#x003b2;-conglycinin (&#x003b1;&#x02032;-, &#x003b1;-, and &#x003b2;-subunit) genes and glycinin (Gy1-, Gy2-, Gy3-, Gy4-, Gy5-, -subunits) genes to show downregulated expression in at least two stages of development studied here. The expressions of various &#x003b2;-conglycinin and glycinin subunit genes were regulated coordinately in the <italic> cgy-2 </italic> -NIL, which might be responsible for the altered amino acid composition and improved <span class="trait">protein</span> quality.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous analysis of &#x003b2;-conglycinin-deficient lines revealed that the loss of &#x003b2;-conglycinin was compensated for by an increase in the abundance of glycinin [ <xref rid="pone.0159723.ref001" ref-type="bibr"> 1 </xref> ]. Glycinin, an 11S globulin, is the predominant seed storage <span class="trait">protein</span> in soybean, and makes an important contribution to the nutritional quality of soy <span class="trait">protein.</span> In the present study, compared with DN47, the &#x003b1;-null mutation caused glycinin <italic> Gy3 </italic> to be upregulated at 35 DAF and <italic> <span class="gene">Gy7</span> </italic> was upregulated throughout all five stages. To date, five glycinin genes, <italic> <span class="gene">Gy1</span> </italic> &#x02013; <italic> <span class="gene">Gy5</span> </italic> have been described in detail. <italic> <span class="gene">Gy4</span> </italic> and <italic> <span class="gene">Gy5</span> </italic> encode proteins that have lower concentration of sulfur amino acids than the proteins derived from <italic> <span class="gene">Gy1</span> </italic> , <italic> <span class="gene">Gy2</span> </italic> , and <italic> Gy3 </italic> [ <xref rid="pone.0159723.ref051" ref-type="bibr"> 51 </xref> ]. Furthermore, Belinson et al. [ <xref rid="pone.0159723.ref044" ref-type="bibr"> 44 </xref> ] identified and mapped a new functional glycinin gene, <italic> <span class="gene">Gy7</span> </italic> , which encodes the sixth glycinin subunit Gy7. Their data revealed that the steady-state amount of mRNA encoding <span class="gene">Gy7</span> at seed mid-maturation is an order of magnitude less than the mRNA encoding the five other glycinin subunits [ <xref rid="pone.0159723.ref044" ref-type="bibr"> 44 </xref> ]. Similar results were obtained in our study, which further confirmed that the <italic> GY7 </italic> gene has a lower expression level than the five other glycinin subunits from 25 to 55 DAF, both in <italic> cgy-2- </italic> NIL and DN47. To date, little is known about the effect of the <span class="gene">Gy7</span> subunit on <span class="trait">protein</span> nutritional quality, tofu-making quality, and its health benefits. Different from the other five glycinin genes, <italic> GY7 </italic> expression in <italic> cgy-2- </italic> NIL slightly exceeded that of DN47 throughout the five stages identified in the present study, and showed a unique developmental expression pattern in both <italic> cgy-2- </italic> NIL and DN47, i.e., increased from the 18 DAF until reaching a peak at 55 DAF. The upregulated expressions of <italic> Gy3 </italic> and <italic> <span class="gene">Gy7</span> </italic> might, at least in part, contribute to the modified final seed <span class="trait">protein</span> content in <italic> cgy-2 </italic> -NIL.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:BG-4, a novel anticancer peptide from bitter gourd ( <italic> Momordica charantia </italic> ), promotes apoptosis in human colon cancer cells</h2>
			<h3>PMID:27628414</h3>
				
					<h3>PMCID:5024301</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dia </surname>  <given-names> Vermont P. </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Krishnan </surname>  <given-names> Hari B. </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Department of Food Science and Technology, The University of Tennessee Institute of Agriculture </institution> , Knoxville TN 37996  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> USDA-ARS Plant Genetics Resources Unit, University of Missouri </institution> , Columbia, MO 65211  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 08 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Momordica charantia </italic> , commonly known as bitter gourd (BG) or bitter melon, belongs to the family of Cucurbitaceae and is cultivated throughout the world, traditionally been used in folk medicines in cases of malaria, peptic ulcers, kidney stone, eczema <xref ref-type="bibr" rid="b14"> 14 </xref> to name a few and is commonly used in the treatment of type II diabetes <xref ref-type="bibr" rid="b15"> 15 </xref> <xref ref-type="bibr" rid="b16"> 16 </xref> . In addition, extracts from different parts of <span class="gene">BG</span> possessed anticancer properties. MAP30, momordica anti-HIV <span class="trait">protein,</span> inhibited proliferation of MDA-MB-231 breast cancer cells and increased survival in MDA-MB-231 breast cancer mice xenografts <xref ref-type="bibr" rid="b17"> 17 </xref> , methanolic extract of <span class="gene">BG</span> led to reduced proliferation and activated autophagy in CRC <xref ref-type="bibr" rid="b18"> 18 </xref> and sensitized CRC to the cytotoxic effect of doxorubicin <xref ref-type="bibr" rid="b19"> 19 </xref> while <span class="gene">BG</span> lectin has been shown to induce G2/M cell cycle arrest, autophagy and apoptosis in hepatocellular cancer <xref ref-type="bibr" rid="b20"> 20 </xref> . These previous studies support the idea on the potential therapeutic use of <span class="gene">BG</span> and its biologically active constituents.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Protein</span> profile of <span class="gene">BG</span> seed extracted with an increasing concentration of ethanol is shown in <xref ref-type="fig" rid="f1"> Fig. 1a </xref> Several polypeptides ranging from 70 to 4&#x02009;kDa was seen in 0 to 20% ethanol seed extracts. Prominent among them were a 35&#x02009;kDa, 24&#x02009;kDa and 4&#x02009;kDa proteins. These three abundant proteins accounted for the bulk of the <span class="gene">BG</span> seed proteins. Increasing the concentration of ethanol drastically reduced the abundance of the 35&#x02009;kDa and 24&#x02009;kDa proteins. Interestingly, the 4&#x02009;kDa peptide was extractable with all the tested concentration of ethanol. In 60 and 70% ethanol extracts only the 4&#x02009;kDa and a 14&#x02009;kDa peptides were present. Our observation suggests that aqueous ethanol could be exploited as a fast and easy way of purification of the 4&#x02009;kDa peptide.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies have established the presence of low molecular weight protease inhibitors in <span class="gene">BG</span> seeds. Two novel trypsin inhibitors, MCTI-II&#x02019; and BGIT, have been purified from <span class="gene">BG</span> and their amino acid sequence have been elucidated <xref ref-type="bibr" rid="b21"> 21 </xref> . MCTI-II&#x02019; consists of 27 amino acid residues while the BGIT is made up of 68 amino acid residues and is capable of inhibiting not only trypsin but also subtilisin Carlsberg. We measured the trypsin inhibitor activity of the aqueous ethanol extracts to verify if they exhibit any protease inhibitor activity ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ). The aqueous ethanol extracts, which predominantly contain the 4&#x02009;kDa peptide, revealed relatively <span class="trait">high</span> levels of trypsin inhibitor activity when compared to that of soybean seed ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ). Additionally, MALDI-TOF-MS analysis of the 4&#x02009;kDa <span class="trait">protein</span> followed by peptide mass searches indicated that significant peptide matches with <italic> Momordica charantia </italic> protease inhibitor ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). These observations indicate that the abundant 4&#x02009;kDa peptide, which is preferentially soluble in aqueous ethanol, is likely a trypsin inhibitor.
					</p>
				
					<p>&nbsp;&nbsp;
						We report for the first time the biological activity of a newly isolated and identified bioactive peptide from <italic> Momordica charantia </italic> we called BG-4. BG-4 was easily extracted using different concentrations of ethanol and can be purified from the rest of <span class="gene">BG</span> seed proteins by 70% ethanol as evidenced by electrophoresis profile of different <span class="gene">BG</span> seed aqueous ethanol extracts ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ), a method that can be easily performed for the potential application of BG-4 as an anticancer agent. Amino acid sequencing of the purified peptide indicated that BG-4 amino acid sequences matched with the sequences of a protease inhibitor found in <italic> Momordica charantia </italic> ( <xref ref-type="table" rid="t1"> Table 1 </xref> ), interestingly all ethanolic <span class="trait">protein</span> extracts from <italic> Momordica charantia </italic> seed inhibited trypsin activity higher than purified soybean Kunitz trypsin inhibitor. The 70% ethanol extract containing purified BG-4 peptide inhibited trypsin activity 8.6 times higher than purified soybean trypsin inhibitor. Another interesting observation showed that increasing ethanol concentrations which is equivalent to increasing purity of BG-4 resulted in increased trypsin inhibitory activity ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ) indicating that BG-4 is the responsible constituent for trypsin inhibition. Previous reports have shown that trypsin inhibitors from food sources possessed anticancer potential, the most studied are the Bowman-Birk inhibitor and Kunitz trypsin inhibitor from soybean <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> . Hence we tested BG-4 for its potential anticancer activities using two different human colon cancer cells.
					</p>
				
					<p>&nbsp;&nbsp;
						HCT-116 and HT-29 cells were seeded in 6-well plate at a density of 2&#x02009;&#x000d7;&#x02009;10 <sup> 5 </sup> cells/mL, allowed to attach overnight, treated with BG-4 as mentioned previously and lysed with RIPA buffer. The cell lysates were vortexed for 5&#x02009;min and centrifuged at 14,000&#x02009;&#x000d7;&#x02009;g for 10&#x02009;min at 4&#x02009;&#x000b0;C. The supernatant was collected, mixed with Laemmli buffer containing 5% &#x003b2;-mercaptoethanol, boiled for 5&#x02009;min and stored at &#x02212;20&#x02009;&#x000b0;C until analysis. A portion of supernatant was diluted with TBS to determine <span class="trait">protein</span> concentration as measured using Bicinchoninic acid assay (ThermoFisher Scientific, Waltham MA). The whole cell lysates were used for Bax, Bcl-2, XIAP, caspase-3, p21 and CDK2 (Proteintech Group, Inc. Rosemont IL) and <span class="gene">GAPDH</span> (Santa Cruz Biotechnology, Santa Cruz CA) expressions. Western blot analysis was conducted as follows: whole cell lysates (25&#x02009;&#x003bc;g <span class="trait">protein)</span> were loaded in 4&#x02013;20% Tris-HCl gel (Bio-Rad Laboratories), run at 200&#x02009;V for 35&#x02009;min at room temperature. The gels were transferred to PVDF membrane (GE Healthcare Life Sciences, Pittsburgh PA) at 110&#x02009;V for 1&#x02009;h at 4&#x02009;&#x000b0;C. The blots were blocked with 5% non-fat dry milk for 1&#x02009;h at room temperature, washed three times with TBS containing 0.1% Tween 20 (TBST) for 5&#x02009;min and incubated with primary antibody overnight at 4&#x02009;&#x000b0;C at 1:1000 dilution. The blots were washed with TBST five times for 5&#x02009;min and incubated with anti-rabbit or anti-mouse secondary antibody at 1:5000 dilution for 2&#x02009;h at room temperature. After washing with TBST, the blots were imaged with C-Digit Blot scanner (Li-Cor Biosciences, Lincoln, NE) using WesternSure ECL Substrate (Li-Cor Biosciences, Lincoln, NE). The intensity of bands was quantified using Image Studio Software (Li-Cor Biosciences, Lincoln, NE). Each experiment was performed for in three independent replicates.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and differential regulation of microRNAs in response to methyl jasmonate treatment in <italic> Lycoris aurea </italic> by deep sequencing</h2>
			<h3>PMID:27724902</h3>
				
					<h3>PMCID:5057397</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xu </surname>  <given-names> Sheng </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiang </surname>  <given-names> Yumei </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ning </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Bing </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Yilong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiaodan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhengzhi </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yikui </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Wang </surname>  <given-names> Ren </given-names>  </name>  <address>  <email> jswangren@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Institute of Botany, Jiangsu Province and Chinese Academy of Sciences, Nanjing, 210014 China  </aff>  <aff id="Aff2">  <label> 2 </label> National Center for Soybean Improvement/Key Laboratory of Biology and Genetic Improvement of Soybean (General, Ministry of Agriculture), Nanjing Agricultural University, Nanjing, 210095 China  </aff>  <aff id="Aff3">  <label> 3 </label> National Center for Soybean Biotechnology and Division of Plant Sciences, University of Missouri, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Jasmonic acid (JA) system is a key component in the complex plant hormone signaling systems. It is biosynthesized from &#x003b1;-linolenic acid by the octadecanoid pathway and oxygenation of &#x003b1;-linolenic acid is the initial step. The oxygen has to be inserted in the C-13 position by a lipoxygenase (LOX) to form (13S)-hydroperoxy-octadecatrienoic acid (13-HPOT) [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. 13-HPOT is then utilized as substrates in the following reaction initiated by allene oxide synthase (AOS) and further converted to (9S, 13S)-12-oxo-phytodienoic acid [(9S, 13S)-OPDA] by allene oxide cyclase (AOC) [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. For further conversion, OPDA is translocated from chloroplasts to peroxisomes, where JA biosynthesis continues and a series of enzymes including OPDA reductase 3 (OPR3), acyl-CoA oxidase (ACX), multifunctional <span class="trait">protein,</span> and L-3-ketoacyl CoA thiolase are involved in. JA and its derivatives, collectively referred to as jasmonates, are important signal molecules detected in a wide spectrum of plant species and function on a lot of biological processes including growth inhibition, senescence, wound response, plant defense and secondary mechanism [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. For example, it was first demonstrated that an endogenous rise in JA levels upon elicitation with a rough yeast elicitor was associated with the induction of alkaloid synthesis in plant cell cultures [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. Later, by inducing taxoid biosynthesis with MJ in <italic> Taxus cuspidata </italic> cells, two cytochrome <span class="gene">P450</span> cDNA clones that encoded hydroxylases of the taxol biosynthetic pathway were identified [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Additionally, exogenous MJ elicits massive accumulation of caffeoylputrescine in tomato and this introduction is tightly controlled by the constitutive activation of the jasmonate signaling pathway [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. Broadly, three major classes of plant secondary metabolites defined as the terpenoids, alkaloids and phenylpropanoids were induced by jasmonates [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR23"> 23 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular Mechanism of MicroRNA-200c Regulating Transforming Growth Factor-&#x003b2; (TGF-&#x003b2;)/SMAD Family Member 3 (SMAD3) Pathway by Targeting Zinc Finger E-Box Binding Homeobox 1 (ZEB1) in Hypospadias in Rats</h2>
			<h3>PMID:27794206</h3>
				
					<h3>PMCID:5091215</h3>
					<h4>期刊:Medical Science Monitor : International Medical Journal of Experimental and Clinical Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Qian </surname>  <given-names> Chong </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn2-medscimonit-22-4073"> B </xref>  <xref ref-type="author-notes" rid="fn3-medscimonit-22-4073"> C </xref>  <xref ref-type="author-notes" rid="fn4-medscimonit-22-4073"> D </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dang </surname>  <given-names> Xiangyang </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn2-medscimonit-22-4073"> B </xref>  <xref ref-type="author-notes" rid="fn3-medscimonit-22-4073"> C </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xianglin </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn2-medscimonit-22-4073"> B </xref>  <xref ref-type="author-notes" rid="fn5-medscimonit-22-4073"> E </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn2-medscimonit-22-4073"> B </xref>  <xref ref-type="author-notes" rid="fn3-medscimonit-22-4073"> C </xref>  <xref ref-type="author-notes" rid="fn4-medscimonit-22-4073"> D </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pang </surname>  <given-names> Guijian </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn5-medscimonit-22-4073"> E </xref>  <xref ref-type="author-notes" rid="fn6-medscimonit-22-4073"> F </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Yifeng </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn4-medscimonit-22-4073"> D </xref>  <xref ref-type="author-notes" rid="fn5-medscimonit-22-4073"> E </xref>  <xref ref-type="author-notes" rid="fn6-medscimonit-22-4073"> F </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Chengbei </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-medscimonit-22-4073"> A </xref>  <xref ref-type="author-notes" rid="fn5-medscimonit-22-4073"> E </xref>  <xref ref-type="corresp" rid="c1-medscimonit-22-4073"/>  </contrib>  <aff id="af1-medscimonit-22-4073"> Department of Urology, The First People&#x02019;s Hospital of Yulin, Yulin, Guangxi, P.R. China </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 1 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						After 72-h <italic> in vitro </italic> incubation of penis tissues, expressions of Zeb1, TGF-&#x003b2;, and Smad3 mRNAs were quantified ( <xref ref-type="fig" rid="f6-medscimonit-22-4073"> Figure 6A </xref> ), showing that the largest expression quantity was in the control group, but the expression quantity in the experiment group was close to that in the blank group. Western blot analysis was performed to detect the expression of Zeb1, TGF-&#x003b2;, and Smad3 proteins. Taking <span class="gene">GAPDH</span> as an internal reference, Zeb1, TGF-&#x003b2;, and Smad3 expressions were highly expressed in the control group and the expression results in the experiment group were similar to that in the blank group ( <xref ref-type="fig" rid="f6-medscimonit-22-4073"> Figure 6B </xref> ). The results indicated that the <span class="trait">high</span> dose of miRNA-200c inhibited the expressions of Zeb 1, TGF-&#x003b2;, and Smad3 in penis tissues and reduced the occurrence and severity of hypospadias. The results also identified that Zeb1, TGF-&#x003b2;, and Smad3 were highly expressed in hypospadias penis tissues.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The unique genomic landscape surrounding the <italic> EPSPS </italic> gene in glyphosate resistant <italic> Amaranthus palmeri </italic> : a repetitive path to <span class="trait">resistance</span></h2>
			<h3>PMID:28095770</h3>
				
					<h3>PMCID:5240378</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Molin </surname>  <given-names> William T. </given-names>  </name>  <address>  <email> William.Molin@ARS.USDA.GOV </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wright </surname>  <given-names> Alice A. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lawton-Rauh </surname>  <given-names> Amy </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Saski </surname>  <given-names> Christopher A. </given-names>  </name>  <address>  <email> saski@clemson.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> United States Department of Agriculture, Crop Production Systems Research Unit, Stoneville, MS USA  </aff>  <aff id="Aff2">  <label> 2 </label> Department of Genetics and Biochemistry, Clemson University, Clemson, SC USA  </aff>  <aff id="Aff3">  <label> 3 </label> Institute of Translational Genomics, Genomics and Computational Biology Lab, Clemson University, Clemson, SC USA  </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Weedy species are a continuous threat to crop and pasture land, requiring constant mitigation to minimize <span class="trait">yield</span> losses, protect commodity quality, and prevent infestations in subsequent years. Estimates of crop losses due to weeds worldwide in wheat, maize and cotton production for 2001 to 2003 were 7.7, 10.5, and 8.6%, respectively [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. Although numerous management practices are available to farmers, such as tillage and cultivation, the use of herbicides remains the most effective and preferred method in minimizing weed interference. In past decades, herbicide-based strategies have performed well in weed management. However, in recent years several weeds have evolved <span class="trait">resistance</span> to some of the most widely used herbicides, such as glyphosate, and have become a serious threat to crop production in the last decade. Glyphosate resistant Palmer amaranth ( <italic> Amaranthus palmeri </italic> S. Wats) alone has increased production costs of corn by $2 to $35 per acre, $0 to $100 per acre in cotton, and $6 to $42 per acre in soybean [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. <italic> A. palmeri </italic> is a fast growing, highly competitive, yield-reducing weed of row crops that must be controlled throughout the crop development cycle to minimize losses. With the introduction of glyphosate resistant crops in 1996 (GR cropping systems), farmers gained an effective tool to control seedling <italic> A. palmeri. </italic> <span class="gene">GR</span> cropping systems permitted repeated applications throughout the crop season with excellent crop safety while avoiding tilling practices that reduce soil longevity and cause erosion. The broad species spectrum and efficacy of control with glyphosate contributed to the expansion of no- and reduced-tillage production systems [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The first reports of glyphosate resistant <italic> A. palmeri </italic> emerged nearly ten years after the introduction of <span class="gene">GR</span> cropping systems, in 2005, in Georgia and North Carolina [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. Since then, reports of glyphosate resistant <italic> A. palmeri </italic> have been confirmed in 25 states [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ]; it continues to spread across the southern states, into the Ohio Valley and into the northeastern states as far north as Pennsylvania and New Jersey. It is not clear whether glyphosate <span class="trait">resistance</span> across all of these regions is derived from single or independent adaptive events. Moreover, several of these glyphosate resistant <italic> A. palmeri </italic> populations also have <span class="trait">resistance</span> to other herbicides, most commonly acetolactate synthase inhibitors [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						BAC libraries are critical tools for genomic dissection and they serve as the primary templates for targeted sequencing to facilitate accurate reconstruction of the genomic architecture around genes, especially in large repetitive plant genomes [ <xref ref-type="bibr" rid="CR22"> 22 </xref> &#x02013; <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. Here, we constructed the first <span class="trait">high</span> quality BAC library from a glyphosate resistant biotype of <italic> A. palmeri </italic> that consists of 36,864 clones with an average insert size of 110&#x000a0;kb. Based on an estimated genome size of 938 Mbp (2C) for the <italic> A. palmeri </italic> genome [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ], the BAC resources represent approximately 8.6 genome equivalents. The BAC library was screened for clones containing the <italic> EPSPS </italic> locus. A total of 680 clones (~2%) were identified as positive to the <italic> EPSPS </italic> probe, suggesting <span class="trait">high</span> representation in the genome and the BAC library (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S1A). Because the large number of positively identified BACs likely correlates with multiple genomic copies of <italic> EPSPS </italic> , we fingerprinted 192 BACs to assess overlap (see Methods) and determine a tile path of representative BACs. Under stringent fingerprint assembly parameters, 186 BACs clustered into a single contig, and a tile path of 4 representative BAC clones was selected for individual sequencing on the Illumina MiSeq platform and as a pool on the Pacific Biosystems (PacBio) RSII platform. The Illumina-only assemblies resulted in hundreds to thousands of small-unordered contigs, suggesting a <span class="trait">high</span> repetitive content. We were unable to produce an assembly with contiguity, and efforts were redirected to the long read single molecule PacBio dataset. A total of 65,733 raw reads were collected and filtered for length that resulted in a total of 57,957 reads with lengths between 1,000&#x000a0;bp and 31&#x000a0;kb, with a mean read length of 6.1&#x000a0;kb (Accession <span class="gene">SAMN06007219).</span> A <italic> de novo </italic> assembly was performed that resulted in a single consensus sequence (see Methods) that amassed 228,387&#x000a0;bp. A subsequent round of DNA hybridization was performed with overgo probes that were designed from the terminal sequences that were not represented elsewhere within the 228&#x000a0;kb genome segment, which resulted in approximately 100 hits per BAC filter (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S1B). Four additional extension BACs were sequenced to produce a final assembly length of 297,445&#x000a0;bp (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). The <italic> A. palmeri </italic> BAC library (APC_Ba) and filter set is publically available at <ext-link ext-link-type="uri" xlink:href="http://www.genome.clemson.edu/"> www.genome.clemson.edu </ext-link> . <fig id="Fig1"> <label> Fig. 1 </label> <caption> <p> The glyphosate resistant <italic> EPSPS </italic> locus and surrounding genomic landscape; highlighted in yellow is the <italic> EPSPS </italic> gene. The outer ideograms ( <italic> blue </italic> is glyphosate resistant biotype and orange is glyphosate sensitive) are situated to depict the assembled 297,445&#x000a0;bp as a self-alignment. The outermost track is the gene annotation where the gene histograms that are transcribed in the clockwise direction are plotted in green in the outward direction, and gene histograms transcribed counterclockwise are plotted in <italic> yellow </italic> on the internal axis. The line graphs ( <italic> orange and blue </italic> ) represent aligned resistant and susceptible WGS reads reported as coverage depths. The light blue and light orange colors represent regions covered with up to 2,000 reads, and dark blue and dark orange mark regions covered by greater than 2,001 reads. Dark red glyphs indicate bases with &#x0201c;0&#x0201d; coverage. The inner track with red only histograms depicts predicted helitron sequences with LCV values of at least &#x02018;3&#x02019;, and the innermost histogram track displays predicted repetitive elements. The internal links connect repetitive regions by self-alignment of the molecule; black links are direct repeats and green are inverted in orientation
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Possible health impacts of Bt toxins and residues from spraying with complementary herbicides in genetically engineered soybeans and risk assessment as performed by the European Food Safety Authority EFSA</h2>
			<h3>PMID:28133586</h3>
				
					<h3>PMCID:5236067</h3>
					<h4>期刊:Environmental Sciences Europe</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Then </surname>  <given-names> Christoph </given-names>  </name>  <address>  <email> christoph.then@testbiotech.org </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bauer-Panskus </surname>  <given-names> Andreas </given-names>  </name>  <address>  <email> panskus@testbiotech.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Testbiotech, Institute for Independent Impact Assessment in Biotechnology, Frohschammerstr. 14, 80807 Munich, Germany  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In general, the risk assessment of genetically engineered herbicide-resistant plants currently performed by EFSA is divided into the assessment of the plant performed by the <span class="gene">GMO-panel,</span> and assessment of the pesticide performed by the pesticide panel. However, this does not mean that if a pesticide is authorised for use in the EU, that no further investigation of the residues from spraying with the complementary herbicide is needed. Due to the specific agricultural practices that go along with the cultivation of these herbicide-resistant plants, there are, for example, specific patterns of applications, exposure and occurrence of specific metabolites and an emergence of combinatorial effects that require special attention. For example, large-scale commercial cultivation of these plants results in a strong selective pressure on weeds to develop <span class="trait">resistance</span> to the herbicides [ <xref ref-type="bibr" rid="CR75"> 75 </xref> ]. This problem is also relevant for health risk assessment, since this has led to increasing amounts of glyphosate being sprayed [ <xref ref-type="bibr" rid="CR76"> 76 </xref> ] and subsequently more residues in the harvest [ <xref ref-type="bibr" rid="CR77"> 77 </xref> ]. Furthermore, herbicide-tolerant plants are engineered to survive the application of the complementary herbicide, while most other plants will die after a short time. Thus, for example, residues of glyphosate, its metabolites and additives to the formulated product might accumulate and interact in the plants, also changing the plant&#x02019;s composition [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ]. Finally, in stacked events, such as <italic> Intacta </italic> , a combination of specific plant constituents is present in the genetically engineered plants. The combination of the residues from spraying and of insecticidal proteins (as in the case of <italic> Intacta </italic> ) leads to a unique pattern of combinatorial exposure in the feed and food chain.
					</p>
				
					<p>&nbsp;&nbsp;
						Whatever the case may be, both the EU pesticide regulation and the <span class="gene">GMO</span> Regulation require a <span class="trait">high</span> level of protection for health and the environment. Thus, in regard to herbicide-resistant plants, the specific assessment of residues from spraying with complementary herbicides must be considered to be a prerequisite for granting authorisation. In addition, additive or synergistic effects need to be investigated if a plant contains or produces other compounds with potential toxicity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conservation and diversification of the miR166 family in soybean and potential roles of newly identified miR166s</h2>
			<h3>PMID:28143404</h3>
				
					<h3>PMCID:5286673</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xuyan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Xin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ji </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Yuhai </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hou </surname>  <given-names> Yanming </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhai </surname>  <given-names> Lulu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiao </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fu </surname>  <given-names> Yanli </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Ranran </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Bian </surname>  <given-names> Shaomin </given-names>  </name>  <address>  <phone> +86-431-86758762 </phone>  <email> shmbian@jlu.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 5735 </institution-id>  <institution-id institution-id-type="GRID"> grid.64924.3d </institution-id>  <institution> College of Plant Science,  </institution>  <institution> Jilin University,  </institution>  </institution-wrap> Changchun, Jilin China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 1302 4958 </institution-id>  <institution-id institution-id-type="GRID"> grid.55614.33 </institution-id>  <institution/>  <institution> Agriculture and Agri-Food Canada, London Research and Development Centre,  </institution>  </institution-wrap> London, ON Canada  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 8884 </institution-id>  <institution-id institution-id-type="GRID"> grid.39381.30 </institution-id>  <institution> Department of Biology,  </institution>  <institution> Western University,  </institution>  </institution-wrap> London, ON Canada  </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs (miRNAs) are a class of endogenous, single-stranded, small regulatory RNA molecules, which broadly exist in diverse eukaryotes. In plants, miRNAs range from 20 to 24&#x000a0;nt in length and regulate the expression of target genes mainly at post-transcriptional levels [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. Plant miRNAs are derived from their congate <italic> MIRNA </italic> genes that are mostly located in intergenic regions, sometimes in intragenic regions, of the plant genome [ <xref ref-type="bibr" rid="CR2"> 2 </xref> &#x02013; <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. Generally, a <italic> MIR </italic> gene is transcribed by Polymerase II into a capped and polyadenylated primary miRNA (pri-miRNA) [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR6"> 6 </xref> ]. Subsequently, DCL <span class="trait">protein</span> carries out the cleavage of pri-miRNA into the stem-loop precursor (pre-miRNA), which is further processed by <span class="gene">DCL1</span> to generate mature miRNA and miRNA*. Mature miRNA is then loaded into an Argonaute <span class="trait">protein</span> to form the miRNA-induced silencing complex (miRISC). Finally, miRISCs can bind to the transcripts of target genes through perfect or near-perfect pairing with its mature miRNA, and guild mRNA cleavage or translational inhibition in plants [ <xref ref-type="bibr" rid="CR7"> 7 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. Thus, plant miRNAs are thought to mediate almost all plant cellular and metabolic processes via regulating posttranscriptional gene silencing [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To explore the regulatory function of miR166s in soybean, the expression patterns of 12 target genes with annotation in Phytozome were also analyzed by qRT-PCR during seed development and in response to abiotic stresses. As shown in Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10a </xref> , the expressions of all the <italic> HD-ZIP III </italic> genes except <span class="gene">Glyma05G166400</span> were gradually increased as seeds developed, and reached the peak at 65 DAF. Their fold changes ranged from 4.39 to 59.16, compared with the ones at 15 DAF. On the contrary, <italic> PSY </italic> was gradually decreased in transcript accumulation as seeds developed, while <italic> AMT2-LIKE </italic> was dramatically decreased by 2.34 folds at 30 DAF. <fig id="Fig10"> <label> Fig. 10 </label> <caption> <p> Expression analysis of the 12 target genes in response to seed development and abiotic stresses. Expression patterns of target genes during seed development ( <bold> a </bold> ), under cold stress ( <bold> b </bold> ), <span class="trait">drought</span> stress ( <bold> c </bold> ) and salinity stress ( <bold> d </bold> ). Stress treatments and seed development stage were used as same as the ones in Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> . <italic> PSY </italic> , <italic> PHYTOENE SYNTHASE </italic> ; <italic> AMT2-LIKE </italic> , <italic> AMMONIUM TRANSPORTER 2-LIKE </italic> . Error bars indicate SE of two biological and three technical replicates. Values were normalized against the <italic> SUBI3 </italic> gene
					</p>
				
					<p>&nbsp;&nbsp;
						Four <italic> ATHB14-LIKE </italic> , <italic> REV-LIKE </italic> <span class="gene">(Glyma11G145800)</span> and <italic> PSY </italic> displayed gradual decrease in transcript accumulation as <span class="trait">drought</span> stress prolonged (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10c </xref> ), and their maximal fold changes ranged from 2.02 to 38.17. Although <italic> ATM2-LIKE </italic> displayed similar expression level to the control at early stage (2&#x02013;4 days after <span class="trait">drought</span> treatment), the transcript accumulation was dramatically increased from 6&#x000a0;days after treatment until 10&#x000a0;days with 9.49 fold changes, suggesting that <italic> ATM2-LIKE </italic> might respond to <span class="trait">drought</span> stress at middle and late stages. The decreased transcript accumulation was also observed for the rest of target genes on one or more treatment points as compared to control.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analyzing pepsin degradation assay conditions used for allergenicity assessments to ensure that pepsin susceptible and pepsin resistant dietary proteins are distinguishable</h2>
			<h3>PMID:28207780</h3>
				
					<h3>PMCID:5312868</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-7198-7784 </contrib-id>  <name>  <surname> Wang </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Edrington </surname>  <given-names> Thomas C. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Storrs </surname>  <given-names> S. Bradley </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Crowley </surname>  <given-names> Kathleen S. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ward </surname>  <given-names> Jason M. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Thomas C. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Zi L. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Bin </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Glenn </surname>  <given-names> Kevin C. </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To further compare the effect of pepsin degradation of <span class="gene">LTP</span> and STI with the three pepsin susceptible proteins (Rubisco, HRP and Hb), a time course of experiments was conducted. When assayed under a standardized pepsin degradation condition of pH 1.2 and 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein,</span> Rubisco LS, HRP and Hb were all degraded by &#x0003e; 98% within 0.5 minute. In contrast, the intact forms of <span class="gene">LTP</span> and STI were relatively stable for over 60 minutes ( <xref ref-type="fig" rid="pone.0171926.g005"> Fig 5A </xref> , <xref ref-type="supplementary-material" rid="pone.0171926.s004"> S4 File </xref> ). At pH 4.0 and 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein,</span> the amount of Rubisco LS and Hb degradation was reduced, and by 60 minutes the intact form of each <span class="trait">protein</span> was fully degraded ( <xref ref-type="fig" rid="pone.0171926.g005"> Fig 5B </xref> , <xref ref-type="supplementary-material" rid="pone.0171926.s004"> S4 File </xref> ). However, &#x02265; 96% of the intact HRP <span class="trait">protein</span> persisted throughout the experiment incubation time under these sub-optimal pepsin degradation conditions ( <xref ref-type="fig" rid="pone.0171926.g005"> Fig 5B </xref> , <xref ref-type="supplementary-material" rid="pone.0171926.s004"> S4 File </xref> ). Under both enzymatic reaction conditions, low pH/high amount of pepsin and <span class="trait">high</span> pH/low amount of pepsin, &#x0003e; 67% of <span class="gene">LTP</span> and STI remained intact over 60 minutes ( <xref ref-type="fig" rid="pone.0171926.g005"> Fig 5A and 5B </xref> , <xref ref-type="supplementary-material" rid="pone.0171926.s004"> S4 File </xref> ). Further analysis by SDS-PAGE of the <span class="gene">LTP</span> and STI reaction mixtures at pH 1.2 and 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein</span> revealed no potential degradation fragments for up to 60 minutes ( <xref ref-type="fig" rid="pone.0171926.g005"> Fig 5C and 5D </xref> , respectively).
					</p>
				
					<p>&nbsp;&nbsp;
						On Panel A and Panel B, the amount of Coomassie Blue stained intact <span class="trait">protein</span> at each condition was quantified and is shown as a percentage relative to the amount of starting material. Panel A is at 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein</span> and pH 1.2. Panel B is at 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein</span> and pH 4. Panels C and D are SDS-PAGE analysis of pepsin degradation of <span class="gene">LTP</span> and STI, respectively, at pH 1.2 and 10 U pepsin to 1 &#x003bc;g substrate <span class="trait">protein</span> over time (0.5 to 60 minute(s)). The gel lanes are: 1: MW, 2: Pepsin Only, 3: Substrate <span class="trait">Protein</span> Only, 4: 0 minute: Pepsin + Substrate <span class="trait">Protein,</span> 5&#x02013;11: 0.5, 2, 5, 10, 20, 30, 60 minute(s), respectively, of Pepsin + Substrate <span class="trait">Protein.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The proteins selected for this analysis included three proteins known to be susceptible to pepsin degradation [ribulose 1,5-bisphosphate carboxylase-oxygenase large subunit (Rubisco LS), horseradish peroxidase (HRP), hemoglobin (Hb)] and two known to be pepsin-resistant [lipid transfer <span class="trait">protein</span> (LTP) and soybean trypsin inhibitor (STI)]. All five of these substrate proteins have been well characterized [ <xref rid="pone.0171926.ref030" ref-type="bibr"> 30 </xref> &#x02013; <xref rid="pone.0171926.ref035" ref-type="bibr"> 35 </xref> ]. Selection of the substrate proteins focused on identifying a set of substrates with a diversity of physiochemical properties that could affect pepsin hydrolysis. For candidate proteins, amino acid sequences from the National Center for Biotechnology Information (NCBI) were used to calculate the theoretical <italic> pI </italic> , molecular weight, intramolecular disulfide bonds and number of aromatic amino acids ( <xref ref-type="table" rid="pone.0171926.t001"> Table 1 </xref> ). The theoretical <italic> pIs </italic> ranged from a low of 4.91 for STI [ <xref rid="pone.0171926.ref036" ref-type="bibr"> 36 </xref> ] to a <span class="trait">high</span> of 9.47 for <span class="gene">LTP</span> [ <xref rid="pone.0171926.ref037" ref-type="bibr"> 37 </xref> ], and molecular weights ranged from 9 kDa for <span class="gene">LTP</span> to ~500 kDa for Rubisco (taking into account that Rubisco contains 8 large subunits at ~ 52 kDa each and 8 small subunits at ~13 kDa each) [ <xref rid="pone.0171926.ref038" ref-type="bibr"> 38 </xref> ]. All substrate proteins ( <xref ref-type="table" rid="pone.0171926.t001"> Table 1 </xref> ) contained different levels of disulfide bonds from zero for Hb [ <xref rid="pone.0171926.ref039" ref-type="bibr"> 39 </xref> ] to four for <span class="gene">LTP</span> and HRP [ <xref rid="pone.0171926.ref037" ref-type="bibr"> 37 </xref> , <xref rid="pone.0171926.ref040" ref-type="bibr"> 40 </xref> ]. The number of aromatic amino acids ranged from 2% for <span class="gene">LTP</span> (2 out of the 93 amino acids) to 10% for Rubisco LS (47 out of 475 amino acids). The prevalence of aromatic amino acids is critical because pepsin is a broad-spectrum protease that preferentially hydrolyzes peptide bonds between the aromatic amino acids Phe, Trp and Tyr [ <xref rid="pone.0171926.ref003" ref-type="bibr"> 3 </xref> ]. The percentage of pepsin cleavage sites also varied for the substrate proteins. For example, the Rubisco LS contains 474 peptide bonds, with the maximum number of theoretical cleavage sites for pepsin calculated to be 122 ( <ext-link ext-link-type="uri" xlink:href="http://web.expasy.org/peptide_cutter/"> http://web.expasy.org/peptide_cutter/ </ext-link> ). These cleavage sites represent approximately 26% of the total peptide bonds found in the Rubisco LS. In contrast, the maximum theoretical cleavages of lipid transfer <span class="trait">protein</span> (LTP) are only eight out of a total of 92 peptide bonds ( <ext-link ext-link-type="uri" xlink:href="http://web.expasy.org/peptide_cutter/"> http://web.expasy.org/peptide_cutter/ </ext-link> ), which account for approximately 9% of total peptide bonds.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous reports have indicated that, under conditions that attempt to be more physiologically relevant, degradation of substrate proteins is slower or less complete because those conditions are sub-optimal for pepsin (e.g., higher pH and/or lower ratios of pepsin-to-substrate <span class="trait">protein)</span> [ <xref rid="pone.0171926.ref010" ref-type="bibr"> 10 </xref> , <xref rid="pone.0171926.ref044" ref-type="bibr"> 44 </xref> ]. To further characterize the effect of the ratio of pepsin-to-substrate <span class="trait">protein</span> on degradation of these same three substrate proteins (Rubisco, HRP, and Hb), three ratios were used: 10 U:1 &#x003bc;g, 1 U:1 &#x003bc;g and 0.1 U:1 &#x003bc;g. Each of these three ratios was tested at pH ranging from 1.2&#x02013;4.0 with multiple incubation times or at pH 5.0 and 6.0 with a single incubation time. Given that the specific activity of pepsin can vary from preparation to preparation, the enzyme to substrate ratio was based on the units of pepsin activity and not the amount of pepsin <span class="trait">protein</span> being used. The results indicated that the extent of degradation of substrate proteins was directly proportional to the ratio of units of pepsin activity to substrate <span class="trait">protein</span> (Figs <xref ref-type="fig" rid="pone.0171926.g002"> 2 </xref> to <xref ref-type="fig" rid="pone.0171926.g005"> 5 </xref> ). The higher pepsin-to-substrate <span class="trait">protein</span> ratio (10 U:1 &#x003bc;g) resulted in a lower percentage of intact <span class="trait">protein</span> at all incubation times and pH values, compared with the middle (1 U:1 &#x003bc;g) or low ratio (0.1 U:1 &#x003bc;g). Nearly complete degradation of the three substrate proteins after two minutes was limited to an assay condition at the 10 U:1 &#x003bc;g ratio, which is the standardized and commonly applied condition previously established [ <xref rid="pone.0171926.ref023" ref-type="bibr"> 23 </xref> ]. In contrast, two substrate proteins that are known to be resistant to pepsin degradation, <span class="gene">LTP</span> and STI, exhibited pepsin-resistance at the higher pepsin-to-substrate <span class="trait">protein</span> ratio (10 U:1 &#x003bc;g) regardless of the test pH (Figs <xref ref-type="fig" rid="pone.0171926.g004"> 4 </xref> and <xref ref-type="fig" rid="pone.0171926.g005"> 5 </xref> ). Thus, not only does the pH of an assay condition greatly affect the ability of pepsin to degrade substrate proteins, but also the ratio of pepsin-to-substrate <span class="trait">protein</span> can substantially influence the assessment of pepsin&#x02019;s ability to degrade a substrate <span class="trait">protein.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An Induced Chromosomal Translocation in Soybean Disrupts a <italic> KASI </italic> Ortholog and Is Associated with a High-Sucrose and Low-Oil Seed Phenotype</h2>
			<h3>PMID:28235823</h3>
				
					<h3>PMCID:5386870</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dobbels </surname>  <given-names> Austin A. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Michno </surname>  <given-names> Jean-Michel </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-3723-2246 </contrib-id>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Campbell </surname>  <given-names> Benjamin W. </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-5510-8583 </contrib-id>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Virdi </surname>  <given-names> Kamaldeep S. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stec </surname>  <given-names> Adrian O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Muehlbauer </surname>  <given-names> Gary J. </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-9320-2629 </contrib-id>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Naeve </surname>  <given-names> Seth L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stupar </surname>  <given-names> Robert M. </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-8836-2924 </contrib-id>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108 </aff>  <aff id="aff2">  <label> &#x02020; </label> Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108 </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seed composition profiles of the high-sucrose/low-oil mutant, the wild-type line (&#x0201c;M92-220&#x0201d;), and the progeny from both the outcross and backcross populations were assessed using multiple methods. Seed composition profiles of all samples in this study were determined using <span class="gene">NIR</span> spectroscopy of whole seeds (described above). <span class="gene">NIR</span> was considered sufficient for quantifying total <span class="trait">protein</span> and oil composition.
					</p>
				
					<p>&nbsp;&nbsp;
						An initial screen of several thousand soybean FN plants by <span class="gene">NIR</span> ( <xref rid="bib5" ref-type="bibr"> Bolon <italic> et al. </italic> 2011 </xref> ) identified plant FN0176450 as a seed composition mutant. This line was advanced for its high-sucrose and low-oil phenotype, along with other lines that were at least 2 SDs above or below the mean of the population for seed composition traits. The phenotype of this mutant was measured over 6 yr and consistently showed an elevated sucrose and reduced oil seed composition phenotype compared to the wild-type &#x0201c;M92-220&#x0201d; line. The <span class="trait">protein</span> and oil measurements were determined by <span class="gene">NIR</span> analysis, while the sucrose measurements were determined by a combination of <span class="gene">NIR</span> and GLC. Seed sucrose and oil levels were significantly different every year between FN0176450 and M92-220, while seed <span class="trait">protein</span> levels were not determined to be statistically different. These differences observed for sucrose and oil content over the 6 year period indicated a heritable change induced by the FN mutagen. FN0176450 also exhibited a visible phenotype in the seed, including a whiter coloration and a more wrinkled seed coat than wild-type seeds (Figure S1 in <ext-link ext-link-type="uri" xlink:href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038596/-/DC1/FileS1.doc"> File S1 </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Mapping was conducted to identify the structural variant associated with the high-sucrose/low-oil seed composition phenotype. FN0176450 was crossed to genotype &#x0201c;Noir 1&#x0201d; to generate a segregating F <sub> 2 </sub> population for mapping. The distribution of seed sucrose content for the F <sub> 2 </sub> population, as measured by a colorimetric assay, is displayed in <xref ref-type="fig" rid="fig2"> Figure 2A </xref> , with shaded bars indicating the individuals selected for the <span class="trait">high-</span> and low-sucrose bulks for BSA. Across the genome, each bulk is expected to have approximately equal ratios of DNA from each parent. However, at SNPs surrounding the causative QTL, a spread in allele frequencies is expected between the <span class="trait">high-</span> and low-sucrose bulks. Visual analysis was done on all 20 chromosomes for this spread in allele frequency, and a 9 Mb interval on chromosome 8 between positions &#x0223c;3 and &#x0223c;12 Mb was found to be of interest due to its conspicuous spread in allele frequencies between the two bulked samples ( <xref ref-type="fig" rid="fig2"> Figure 2B </xref> ). A closer look at the aCGH data in this interval revealed a single probe <span class="gene">(Gm_CUST_P11209911)</span> with a log <sub> 2 </sub> ratio of &#x02212;6.8 in FN0176450 at position 6,358,158 bp, between the fourth and fifth exon of a single gene model <span class="gene">Glyma.08G084300</span> ( <xref ref-type="fig" rid="fig2"> Figure 2, C and D </xref> ). Spurious data points can occur in aCGH analyses; however, this particular probe was consistently observed to indicate a deletion at this locus across three replicated aCGH hybridizations.
					</p>
				
					<p>&nbsp;&nbsp;
						A flow chart of trait mapping. Bulked Segregant Analysis (BSA) was performed on an F <sub> 2 </sub> outcross population to identify a structural variant in a &#x003b2;-ketoacyl-[acyl carrier <span class="trait">protein]</span> synthase 1 (KASI) ortholog associated with a high-sucrose/low-oil phenotype. (A) A histogram of sucrose content, as measured by the colorimetric assay, for the 113 individuals in the F <sub> 2 </sub> population (&#x0201c;Noir 1&#x0201d; &#x000d7; FN0176450). Shaded histogram bars indicate those individuals that were selected for BSA and the vertical red and blue dashed lines indicate the parental phenotypes. Two bulks of DNA were formed (high-sucrose individuals and low-sucrose individuals) for whole-genome sequencing. (B) The allelic ratio for each single nucleotide polymorphism (SNP) along chromosome 8 in the high-sucrose (red data points) and low-sucrose bulks (blue data points). (C) The array comparative genomic hybridization (aCGH) graph displays the log <sub> 2 </sub> ratio of the mutant genotype <italic> vs. </italic> the &#x0201c;M92-220&#x0201d; reference, where each dot represents a single aCGH probe. If the log <sub> 2 </sub> ratio is &#x0003c;0, that indicates that the probe had a stronger signal intensity in wild type than in mutant, while a log <sub> 2 </sub> ratio &#x0003e;0 would indicate a stronger signal intensity in mutant than wild type. A black box was drawn around a single probe with a low log <sub> 2 </sub> ratio (log <sub> 2 </sub> ratio = &#x02212;6.5) highlighting that this probe was likely present in the wild-type line and absent in the high-sucrose/low-oil mutant line. (D) A structural variant between the fourth and fifth exon of <span class="gene">Glyma.08g084300,</span> indicated by the red lightning bolt.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nattokinase: An Oral Antithrombotic Agent for the Prevention of Cardiovascular Disease</h2>
			<h3>PMID:28264497</h3>
				
					<h3>PMCID:5372539</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Weng </surname>  <given-names> Yunqi </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-00523"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yao </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-00523"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sparks </surname>  <given-names> Sawyer </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-00523"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Kevin Yueju </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-00523"> 2 </xref>  <xref rid="c1-ijms-18-00523" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Nattokinase is encoded by the <italic> <span class="gene">aprN</span> </italic> gene, which was first cloned and sequenced from <italic> B. subtilis </italic> (natto) [ <xref rid="B36-ijms-18-00523" ref-type="bibr"> 36 </xref> ]. The full length polypeptide contains a 29-residue signal peptide which directs <span class="trait">protein</span> secretion through the cell membrane, and a 77-residue propeptide which plays a crucial role as an intramolecular chaperone during the <span class="trait">protein</span> folding process; resulting in a 275-residue, mature and functional NK peptide. The partial length of <italic> <span class="gene">aprN</span> </italic> gene is 0.8 kb in length ( <xref ref-type="fig" rid="ijms-18-00523-f005"> Figure 5 </xref> A) and the mature peptide has a molecular weight of 27.7 kDa ( <xref ref-type="fig" rid="ijms-18-00523-f005"> Figure 5 </xref> B). Since the amino acid sequence of NK is almost identical (99.3% homologous) to Subtilisin E, NK was also considered as a member of the subtilisin serine protease family. Unlike subtilisin, however, NK has a very specific affinity for fibrin degradation [ <xref rid="B37-ijms-18-00523" ref-type="bibr"> 37 </xref> ]. The D <sup> 32 </sup> -H <sup> 64 </sup> -S <sup> 221 </sup> motif and N <sup> 155 </sup> function as the catalytic triad and oxyanion hole of NK, respectively [ <xref rid="B38-ijms-18-00523" ref-type="bibr"> 38 </xref> , <xref rid="B39-ijms-18-00523" ref-type="bibr"> 39 </xref> ]. These sites are critical for the protease hydrolysis process. Based on the crystal structure of Subtilisin E, a 3D structural model of NK was constructed [ <xref rid="B38-ijms-18-00523" ref-type="bibr"> 38 </xref> ]. It indicated that the overall active centers are negatively charged, suggesting that NK is more specific for positive-charged substrates. 3D model predicted that the mechanism of NK was induced by attacking of hydroxyl rich in catalytic environment and locating of S <sup> 221 </sup> [ <xref rid="B38-ijms-18-00523" ref-type="bibr"> 38 </xref> ]. Site-directed mutagenesis and molecular dynamics simulation of a 3D model revealed that hydrogen bonds formed between residues, S <sup> 33 </sup> , D <sup> 60 </sup> , S <sup> 62 </sup> , and T <sup> 220 </sup> stabilize the transition state of the hydrolysis reaction. S <sup> 125 </sup> , L <sup> 126 </sup> , and G <sup> 127 </sup> serve as substrate binding sites. Three residues of S3 binding sites, G <sup> 100 </sup> , S <sup> 101 </sup> , and L <sup> 126 </sup> are responsible for fibrinolytic activity while they moderately affected substrate specificity [ <xref rid="B40-ijms-18-00523" ref-type="bibr"> 40 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-quality permanent draft genome sequence of the <italic> Bradyrhizobium elkanii </italic> type strain USDA 76 <sup> T </sup> , isolated from <italic> Glycine max </italic> (L.) Merr</h2>
			<h3>PMID:28270909</h3>
				
					<h3>PMCID:5336687</h3>
					<h4>期刊:Standards in Genomic Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Reeve </surname>  <given-names> Wayne </given-names>  </name>  <address>  <email> W.Reeve@murdoch.edu.au </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> van Berkum </surname>  <given-names> Peter </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-9938-606X </contrib-id>  <name>  <surname> Ardley </surname>  <given-names> Julie </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Rui </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gollagher </surname>  <given-names> Margaret </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marinova </surname>  <given-names> Dora </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Elia </surname>  <given-names> Patrick </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> T. B. K. </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pillay </surname>  <given-names> Manoj </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Varghese </surname>  <given-names> Neha </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seshadri </surname>  <given-names> Rekha </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ivanova </surname>  <given-names> Natalia </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Woyke </surname>  <given-names> Tanja </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baeshen </surname>  <given-names> Mohamed N. </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baeshen </surname>  <given-names> Nabih A. </given-names>  </name>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kyrpides </surname>  <given-names> Nikos </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0436 6763 </institution-id>  <institution-id institution-id-type="GRID"> grid.1025.6 </institution-id>  <institution> School of Veterinary and Life Sciences,  </institution>  <institution> Murdoch University,  </institution>  </institution-wrap> Murdoch, Australia  </aff>  <aff id="Aff2">  <label> 2 </label> U.S. Department of Agriculture, Soybean Genomics and Improvement Laboratory, Beltsville Agricultural Research Center, 10300 Baltimore Avenue, Bldg. 006, Beltsville, MD 20705 USA  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0375 4078 </institution-id>  <institution-id institution-id-type="GRID"> grid.1032.0 </institution-id>  <institution/>  <institution> Curtin University Sustainability Policy Institute, Curtin University,  </institution>  </institution-wrap> Bentley, WA Australia  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0449 479X </institution-id>  <institution-id institution-id-type="GRID"> grid.451309.a </institution-id>  <institution/>  <institution> DOE Joint Genome Institute,  </institution>  </institution-wrap> Walnut Creek, CA USA  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2231 4551 </institution-id>  <institution-id institution-id-type="GRID"> grid.184769.5 </institution-id>  <institution/>  <institution> Biological Data Management and Technology Center, Lawrence Berkeley National Laboratory,  </institution>  </institution-wrap> Berkeley, CA USA  </aff>  <aff id="Aff6">  <label> 6 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.460099.2 </institution-id>  <institution> Department of Biology,  </institution>  <institution> Faculty of Science, University of Jeddah,  </institution>  </institution-wrap> Jeddah, Saudi Arabia  </aff>  <aff id="Aff7">  <label> 7 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0619 1117 </institution-id>  <institution-id institution-id-type="GRID"> grid.412125.1 </institution-id>  <institution> Department of Biological Sciences,  </institution>  <institution> Faculty of Science, King Abdulaziz University,  </institution>  </institution-wrap> Jeddah, Saudi Arabia  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Using the Phylogenetic Profiler tool, 7556 genes were found to be conserved in five <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1461"> <italic> B. elkanii </italic> </ext-link> strains (587, <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DCCBAU+43297"> CCBAU43297 </ext-link> , <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DCCBAU+05737"> CCBAU05737 </ext-link> , <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DUSDA+76"> USDA 76 <sup> T </sup> </ext-link> , <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DUSDA+94"> USDA 94 </ext-link> ), including genes encoding a general secretion pathway and type II, III, IV and VI secretion system proteins. The Type III secretion system (T3SS) [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ] can either promote or impair the establishment of symbiosis, depending on the legume host [ <xref ref-type="bibr" rid="CR56"> 56 </xref> ], and has been characterized as a host determinant for <italic> rj1, Rfg1, <span class="gene">Rj2</span> </italic> and <italic> <span class="gene">Rj4</span> </italic> soybean cultivars [ <xref ref-type="bibr" rid="CR33"> 33 </xref> , <xref ref-type="bibr" rid="CR57"> 57 </xref> , <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. The dominant soybean genes <italic> <span class="gene">Rj2</span> </italic> and <italic> <span class="gene">Rj4</span> </italic> restrict nodulation with specific strains of <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1459"> <italic> Bradyrhizobium </italic> </ext-link> [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. Most investigations of soybean host genes controlling the symbiosis have focused on the <italic> <span class="gene">Rj4</span> </italic> soybean line that was originally identified by its inability to nodulate with <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DUSDA+61"> USDA 61 </ext-link> ( <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1461"> <italic> B. elkanii </italic> </ext-link> , serogroup 31) [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ]. The predicted <italic> <span class="gene">Rj4</span> </italic> thaumatin-like <span class="trait">protein</span> is thought to be involved in conferring <span class="trait">resistance</span> to <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1459"> <italic> Bradyrhizobium </italic> </ext-link> strains producing specific T3SS effector proteins [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. However, <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DUSDA+76"> USDA 76 <sup> T </sup> </ext-link> was reported to nodulate and form an effective nitrogen-fixing symbiosis with the isogenic lines <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DBARC+2"> BARC-2 </ext-link> ( <italic> <span class="gene">Rj4</span> </italic> ) and <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DBARC+3"> BARC-3 </ext-link> ( <italic> <span class="gene">rj4</span> </italic> ) [ <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR61"> 61 </xref> ], suggesting that this strain does not produce the interacting T3SS effector protein(s). Conversely, the recessive soybean gene <italic> rj1rj1 </italic> [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ], encoding a putative truncated Nod factor receptor <span class="trait">protein</span> [ <xref ref-type="bibr" rid="CR63"> 63 </xref> ], restricts nodulation by many <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1459"> <italic> Bradyrhizobium </italic> </ext-link> and <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1328"> <italic> Ensifer </italic> </ext-link> strains, although specific strains of <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/nm.1461"> <italic> B. elkanii </italic> </ext-link> , including <ext-link ext-link-type="uri" xlink:href="http://doi.org/10.1601/strainfinder?urlappend=%3Fid%3DUSDA+76"> USDA 76 <sup> T </sup> </ext-link> , can form a limited number of nodules when tested with plants in Leonard jars containing sterilized vermiculite or sand [ <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR59"> 59 </xref> , <xref ref-type="bibr" rid="CR61"> 61 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Exploring structural variation and gene family architecture with <italic> De Novo </italic> assemblies of 15 <italic> Medicago </italic> genomes</h2>
			<h3>PMID:28347275</h3>
				
					<h3>PMCID:5369179</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Peng </given-names>  </name>  <address>  <email> zhoux379@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silverstein </surname>  <given-names> Kevin A. T. </given-names>  </name>  <address>  <email> kats@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ramaraj </surname>  <given-names> Thiruvarangan </given-names>  </name>  <address>  <email> tr@ncgr.org </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guhlin </surname>  <given-names> Joseph </given-names>  </name>  <address>  <email> guhli007@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Denny </surname>  <given-names> Roxanne </given-names>  </name>  <address>  <email> denny004@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Junqi </given-names>  </name>  <address>  <email> liuxx162@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farmer </surname>  <given-names> Andrew D. </given-names>  </name>  <address>  <email> adf@ncgr.org </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Steele </surname>  <given-names> Kelly P. </given-names>  </name>  <address>  <email> Kelly.Steele@asu.edu </email>  </address>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stupar </surname>  <given-names> Robert M. </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Miller </surname>  <given-names> Jason R. </given-names>  </name>  <address>  <email> jmiller@jcvi.org </email>  </address>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tiffin </surname>  <given-names> Peter </given-names>  </name>  <address>  <email> ptiffin@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mudge </surname>  <given-names> Joann </given-names>  </name>  <address>  <email> jm@ncgr.org </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Young </surname>  <given-names> Nevin D. </given-names>  </name>  <address>  <email> neviny@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000419368657 </institution-id>  <institution-id institution-id-type="GRID"> grid.17635.36 </institution-id>  <institution> Department of Plant Pathology,  </institution>  <institution> University of Minnesota,  </institution>  </institution-wrap> St. Paul, MN USA  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000419368657 </institution-id>  <institution-id institution-id-type="GRID"> grid.17635.36 </institution-id>  <institution/>  <institution> Supercomputing Institute for Advanced Computational Research, University of Minnesota,  </institution>  </institution-wrap> Minneapolis, MN USA  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2219 756X </institution-id>  <institution-id institution-id-type="GRID"> grid.419253.8 </institution-id>  <institution/>  <institution> National Center for Genome Resources,  </institution>  </institution-wrap> Santa Fe, NM USA  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000419368657 </institution-id>  <institution-id institution-id-type="GRID"> grid.17635.36 </institution-id>  <institution/>  <institution> Department of Plant Biology, University of Minnesota,  </institution>  </institution-wrap> St. Paul, MN USA  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000419368657 </institution-id>  <institution-id institution-id-type="GRID"> grid.17635.36 </institution-id>  <institution> Department of Agronomy and Plant Genetics,  </institution>  <institution> University of Minnesota,  </institution>  </institution-wrap> St. Paul, MN USA  </aff>  <aff id="Aff6">  <label> 6 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2151 2636 </institution-id>  <institution-id institution-id-type="GRID"> grid.215654.1 </institution-id>  <institution/>  <institution> Science and Mathematics Faculty, Arizona State University,  </institution>  </institution-wrap> Mesa, AZ USA  </aff>  <aff id="Aff7">  <label> 7 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.469946.0 </institution-id>  <institution/>  <institution> J. Craig Venter Institute,  </institution>  </institution-wrap> Rockville, MD USA  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						AUGUSTUS [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ] was used to make <italic> ab initio </italic> gene predictions for each assembly using both RNA-Seq expression evidence and <italic> M. truncatula </italic> HM101 reference sequence (Mt4.0) [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ] homology evidence. RNA-Seq data came from transcript sequencing of four diverse accessions, HM034, HM056, HM101 and HM340. Reads from HM034, HM056 and HM340 were directly mapped to their <italic> de novo </italic> assemblies using Tophat [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ] to generate intron hints for AUGUSTUS. For the remaining 12 accessions, RNA-Seq reads from the closest available accession were mapped to the corresponding assembly to generate intron hints. Predicted <span class="trait">protein</span> sequences were scanned for PFAM domains (Pfam-A.hmm) [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ] using HMMER [ <xref ref-type="bibr" rid="CR56"> 56 </xref> ] and processed using custom scripts. Domain categories were then assigned according to the most significant Pfam hits. Among the resulting Pfam domains, 160 were associated with transposable elements and grouped into a large &#x0201c;TE" category. NBS-LRR and <span class="gene">RLK</span> genes were scanned using sub-family alignments from previous work [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ] with 37 NBS-LRR sub-family identifiers (TNL0100-TNL0850, CNL0100-CNL1600) and 35 <span class="gene">RLK</span> sub-family identifiers (LRR_I-LRR_XIII, RLCK_I-RLCK_XI) created in consistent with previous research. NCRs and the broader CRP super-family were annotated by running the SPADA pipeline [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ] with group identifiers exactly following previous literature [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]: sub-family CRP0000-CRP1030 representing defensing-like genes (DEFLs), CRP1040-CRP1530 representing NCRs, and CRP1600-CRP6250 representing other types of CRPs.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of <italic> Bacillus subtilis </italic> and <italic> Bacillus licheniformis </italic> supplementation in diets with low- and high-protein content on ileal crude <span class="trait">protein</span> and amino acid digestibility and intestinal microbiota composition of growing pigs</h2>
			<h3>PMID:28469845</h3>
				
					<h3>PMCID:5410705</h3>
					<h4>期刊:Journal of Animal Science and Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kaewtapee </surname>  <given-names> Chanwit </given-names>  </name>  <address>  <email> agrcwk@ku.ac.th </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Burbach </surname>  <given-names> Katharina </given-names>  </name>  <address>  <email> katharina.burbach@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tomforde </surname>  <given-names> Georgina </given-names>  </name>  <address>  <email> g.tomforde@web.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hartinger </surname>  <given-names> Thomas </given-names>  </name>  <address>  <email> t.hartinger@uni-bonn.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Camarinha-Silva </surname>  <given-names> Am&#x000e9;lia </given-names>  </name>  <address>  <email> amelia.silva@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heinritz </surname>  <given-names> Sonja </given-names>  </name>  <address>  <email> sonja.heinritz@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seifert </surname>  <given-names> Jana </given-names>  </name>  <address>  <email> jseifert@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wiltafsky </surname>  <given-names> Markus </given-names>  </name>  <address>  <email> markus.wiltafsky@evonik.com </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mosenthin </surname>  <given-names> Rainer </given-names>  </name>  <address>  <email> rainer.mosenthin@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Rosenfelder-Kuon </surname>  <given-names> Pia </given-names>  </name>  <address>  <email> pia.rosenfelder@uni-hohenheim.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2290 1502 </institution-id>  <institution-id institution-id-type="GRID"> grid.9464.f </institution-id>  <institution/>  <institution> University of Hohenheim, Institute of Animal Science,  </institution>  </institution-wrap> Emil-Wolff-Strasse 10, 70599 Stuttgart, Germany  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0944 049X </institution-id>  <institution-id institution-id-type="GRID"> grid.9723.f </institution-id>  <institution> Present address: Department of Animal Science,  </institution>  <institution> Faculty of Agriculture, Kasetsart University,  </institution>  </institution-wrap> 50 Ngam Wong Wan Rd, Chatuchak, Bangkok 10900 Thailand  </aff>  <aff id="Aff3">  <label> 3 </label> Present address: University of Bonn, Institute of Animal Science, Endenicher Allee 15, 53115 Bonn, Germany  </aff>  <aff id="Aff4">  <label> 4 </label> Evonik Nutrition &#x00026; Care GmbH, Rodenbacher Chaussee 4, 63457 Hanau-Wolfgang, Germany  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The assay diets were based on wheat, barley, and soybean meal with 2 <span class="trait">protein</span> levels resulting in a low-protein (14% CP, as-fed; LP) and a high-protein diet (18% CP, as-fed; HP). The LP diet was accomplished by blending the HP diet with 25% of native cornstarch. The contents of oil, minerals, vitamins, and titanium dioxide were the same for all diets. The <italic> Bacillus </italic> spp. product is comprised of a mixture of spray-dried spores of <italic> B. licheniformis </italic> and <italic> B. subtilis </italic> . The LP and HP diets were supplemented with (+) or without (-) <italic> Bacillus </italic> spp. at a level of 0.04% (as-fed). All assay diets were formulated (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ) to meet or exceed the dietary threshold levels for CP and AA according to <span class="gene">Fan</span> et al. [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ] and NRC [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ] nutrient recommendations for pigs from 25 to 50&#x000a0;kg BW. Vitamins and minerals were supplemented to all diets to meet or exceed NRC [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ] standard, and all diets contained titanium dioxide at a level of 0.4% (as-fed basis) as an indigestible marker. <table-wrap id="Tab1"> <label> Table 1 </label> <caption> <p> Ingredient composition of assay diets, % as-fed basis
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Lactobacillus </italic> spp. and other bacteria groups of interest were quantified in ileal digesta by qPCR (Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ). Mantel test showed a significant correlation between the two approaches, sequencing and qPCR (Rho&#x02009;=&#x02009;0.852, <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.01), thus confirming that both methodological approaches resulted in comparable results. The HP diets increased abundance of <italic> Lactobacillus </italic> <span class="gene">spp</span> <italic> . </italic> and <italic> Bifidobacterium </italic> spp. ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05). No effects of CP content on ileal gene copy numbers of total bacteria, <italic> Roseburia </italic> spp., <italic> Enterobacteriaceae </italic> , <italic> Bacteroides </italic> - <italic> Prevotella </italic> - <italic> Porphyromonas </italic> , <italic> Clostridium </italic> cluster IV and <italic> Bacillus </italic> spp. were found. Likewise, no significant effect of supplementation of <italic> Bacillus </italic> spp. was observed for ileal gene copy numbers of total bacteria, <italic> Lactobacillus </italic> spp., <italic> Bifidobacterium </italic> spp., <italic> Enterobacteriaceae </italic> , <italic> Clostridium </italic> cluster IV and <italic> Bacillus </italic> spp.. However, dietary supplementation of <italic> Bacillus </italic> spp. increased ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05) abundance of <italic> Roseburia </italic> spp., while it tended ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.10) to promote <italic> Bacillus </italic> spp. and total bacteria. Furthermore, there was an interaction ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05) of <span class="trait">protein</span> level and <italic> Bacillus </italic> spp. supplementation for ileal gene copy numbers of <italic> Bacteroides </italic> - <italic> Prevotella </italic> - <italic> Porphyromonas </italic> . The LP&#x02009;+&#x02009;resulted in higher ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05) abundance of <italic> Bacteroides </italic> - <italic> Prevotella </italic> - <italic> Porphyromonas </italic> than the LP -, but did not differ from the HP diets. <table-wrap id="Tab7"> <label> Table 7 </label> <caption> <p> Ileal gene copy numbers <sup> a </sup> in ileal digesta of growing pigs
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dietary macronutrient composition on lipid metabolism-associated factor gene expression in the adipose tissue of chickens are influenced by fasting and refeeding</h2>
			<h3>PMID:28496985</h3>
				
					<h3>PMCID:5424285</h3>
					<h4>期刊:BMC obesity</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Guoqing </given-names>  </name>  <address>  <email> guoqingw@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McConn </surname>  <given-names> Betty R. </given-names>  </name>  <address>  <email> betty12@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongmin </given-names>  </name>  <address>  <email> doliu@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cline </surname>  <given-names> Mark A. </given-names>  </name>  <address>  <email> macline2@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Gilbert </surname>  <given-names> Elizabeth R. </given-names>  </name>  <address>  <phone> 1-540-231-4750 </phone>  <email> egilbert@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0694 4940 </institution-id>  <institution-id institution-id-type="GRID"> grid.438526.e </institution-id>  <institution/>  <institution> Department of Animal and Poultry Sciences Virginia Polytechnic Institute and State University,  </institution>  </institution-wrap> Blacksburg, VA 24061 USA  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0694 4940 </institution-id>  <institution-id institution-id-type="GRID"> grid.438526.e </institution-id>  <institution/>  <institution> Department of Human Nutrition, Foods and Exercise Virginia Polytechnic Institute and State University,  </institution>  </institution-wrap> Blacksburg, VA 24061 USA  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<sup> a </sup> Gene abbreviations: Peroxisome proliferator-activated receptor gamma: <italic> PPAR&#x003b3; </italic> ; CCAAT/enhancer-binding <span class="trait">protein</span> alpha and beta: <italic> C/EBP&#x003b1; </italic> and <italic> C/EBP&#x003b2; </italic> , respectively; Fatty acid binding <span class="trait">protein</span> 4: <italic> FABP4 </italic> ; Sterol regulatory element-binding transcription factor 1: <italic> SREBP1 </italic> ; Kruppel-Like Factor 7: <italic> KLF7 </italic> ; <span class="gene">GATA</span> binding <span class="trait">protein</span> 2: <italic> GATA2 </italic> ; 1-acylglycerol-3-phosphate O-acyltransferase 9: <italic> AGPAT9 </italic> ; Monoglyceride lipase: <italic> MGLL </italic> ; Adipose triglyceride lipase: <italic> ATGL </italic> ; Comparative gene identification-58: <italic> CGI-58 </italic> ; Perilipin 1: <italic> PLN1 </italic> ; Acyl-CoA dehydrogenase, long chain: <italic> ACADL </italic> ; Lipoprotein lipase: <italic> LPL </italic> ; Neuropeptide Y: <italic> NPY </italic> ; <italic> NPY </italic> receptor 2: <italic> NPYR2 </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GATA</span> binding <span class="trait">protein</span> 2
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Meta-Analyses of QTLs Associated with <span class="trait">Protein</span> and Oil Contents and Compositions in Soybean [ <italic> Glycine max </italic> (L.) Merr.] Seed</h2>
			<h3>PMID:28587169</h3>
				
					<h3>PMCID:5486003</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Van </surname>  <given-names> Kyujung </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-01180"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McHale </surname>  <given-names> Leah K. </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-01180"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-18-01180"> 2 </xref>  <xref ref-type="aff" rid="af3-ijms-18-01180"> 3 </xref>  <xref rid="c1-ijms-18-01180" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 55 meta-QTLs (43 <span class="trait">high</span> confidence meta-QTLs) were identified for all traits and trait combinations ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ). The meta-QTLs represented 284 projected QTLs (263 <span class="trait">high</span> confidence projected QTLs). While the majority of meta-QTLs were identified only when the meta-analysis was carried out across traits and included both <span class="trait">protein</span> and oil QTLs ( <xref ref-type="fig" rid="ijms-18-01180-f001"> Figure 1 </xref> and <xref ref-type="app" rid="app1-ijms-18-01180"> Table S4 </xref> ), Chr 20 possessed <span class="trait">high</span> confidence meta-QTLs for <span class="trait">protein,</span> oil, protein+oil and protein+Cys+Met (designated as mP20-#, mPO20-#, and mPCM20-#, respectively) ( <xref ref-type="fig" rid="ijms-18-01180-f002"> Figure 2 </xref> and <xref ref-type="app" rid="app1-ijms-18-01180"> Table S4 </xref> ). Twelve meta-QTLs on Chrs 9, 14, and 15 were identified only when QTLs with LOD scores &#x0003c; 2.0 were included in the analysis ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ). Meta-analysis reduced the <span class="gene">CI</span> of meta-QTLs relative to the projected QTLs by a wide margin, with the projected QTLs spanning an average of 21.77 cM (30.07 cM for <span class="trait">high</span> confidence QTLs) and meta-QTLs spanning an average of only 3.88 cM (4.30 cM for <span class="trait">high</span> confidence meta-QTLs).
					</p>
				
					<p>&nbsp;&nbsp;
						In order to identify positional candidate genes potentially contributing to the meta-QTLs for each trait, the physical and genetic positions of the left and right markers of each meta-QTL&#x02019;s <span class="gene">CI</span> were obtained for those markers found on both the consensus map 4.0 [ <xref rid="B44-ijms-18-01180" ref-type="bibr"> 44 </xref> ] and <italic> G. max </italic> genome assembly, version Glyma.Wm82.a2.v1 <span class="gene">(Gmax2.0)</span> (available online: <uri xlink:href="http://soybase.org"> http://soybase.org </uri> ). For those markers not positioned on either the soybean consensus map 4.0 or <span class="gene">Gmax2.0,</span> the next nearest marker located on both the consensus map and <span class="gene">Gmax2.0</span> was selected. The meta-QTLs encompassed a total of 7412 positional candidate genes (5440 positional candidate genes for <span class="trait">high</span> confidence meta-QTLs), with each meta-QTL encompassing an average of 135 positional candidate genes (127 positional candidate genes for <span class="trait">high</span> confidence meta-QTLs) ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ). Meta-QTL mPO20-2 contained the smallest number of positional candidate genes, with 14 genes within the <span class="gene">CI</span> of 1.83 cM (24.07&#x02013;25.90 cM on Chr 20, <xref ref-type="app" rid="app1-ijms-18-01180"> Figure S2 </xref> ), whereas there were an immense 590 positional candidate genes encompassed by mPO7-6 with a <span class="gene">CI</span> of 17.41 cM (57.99&#x02013;75.40 cM on Chr 7). While, generally, a small number of positional candidate genes were detected if the <span class="gene">CI</span> of the meta-QTL was small, the CIs of mP20-3, mO20-4, mPO20-4 and mPCM20-3 were only 0.61, 0.01, 0.00 and 0.14 cM, respectively; however, 49 positional candidate genes were identified within each of these meta-QTL. This was largely due to the closest marker being located at least 4 cM away on the reference map from the <span class="gene">CI</span> boundaries ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ). Thus, while the CIs of meta-QTL are generally narrower than the projected QTL from which they are synthesized, the advantage of this narrowed region in identifying positional candidate genes can only be observed when the markers can be easily translated to the genome sequences <span class="gene">(Gmax2.0).</span>
					</p>
				
					<p>&nbsp;&nbsp;
						No meta-QTLs were synthesized for fatty acids or amino acids when analyzed as independent traits. This may be due to the limited number of studies as compared to <span class="trait">protein</span> and oil as well as the minimal overlap of QTLs detected among those studies. However, genes which contribute to environmentally stable changes in fatty acid composition in multiple genetic backgrounds have been cloned, such as 3-keto-acyl-ACP synthase II gene ( <italic> KAS II </italic> ) [ <xref rid="B50-ijms-18-01180" ref-type="bibr"> 50 </xref> ], &#x003c9;-3 fatty-acid desaturase gene ( <italic> <span class="gene">FAD3</span> </italic> ) [ <xref rid="B51-ijms-18-01180" ref-type="bibr"> 51 </xref> ], microsomal oleate desaturase gene ( <italic> <span class="gene">FAD2</span> </italic> ) [ <xref rid="B52-ijms-18-01180" ref-type="bibr"> 52 </xref> ] and &#x00394; <sup> 9 </sup> -stearoyl-ACP-desaturase gene ( <italic> <span class="gene">SACPD-C</span> </italic> ) gene [ <xref rid="B53-ijms-18-01180" ref-type="bibr"> 53 </xref> ]. Thus, it is known that at least a portion of the QTL for fatty acid composition are robust.
					</p>
				
					<p>&nbsp;&nbsp;
						In Chr 20, meta-QTLs consistently possessed narrower CIs than the projected QTLs from which they were synthesized. Meta-QTLs for <span class="trait">protein,</span> oil, protein+oil, and protein+Cys+Met decreased <span class="gene">CI</span> by 13.56 (73.3%), 21.69 (81.1%), 28.63 (89.6%) and 18.67 (88.7%) cM, respectively, as compared to the projected QTL. The translation of genetic distances to the identification of positional candidate genes on a physical map is dependent on the ability to locate the physical positions of flanking genetic markers from the reference map. Accordingly, we found that the decreased confidence intervals often, but not always (e.g., mP20-3, mO20-4, mPO20-4 and mPCM20-3), correlated with a decrease in the number of positional candidate genes encompassed by the meta-QTLs in comparison to the projected QTLs. Meta-QTLs for <span class="trait">protein,</span> oil, protein+oil, and protein+Cys+Met decreased the average number of positional candidate genes by 367 (480 to 113), 686 (815 to 129), 526 (702 to 176) and 461 (527 to 66), respectively, as compared to the projected QTLs.
					</p>
				
					<p>&nbsp;&nbsp;
						Traditionally, QTL positions are refined through the tedious process of fine-mapping which can include selecting for recombination in targeted regions and evaluating those individuals (e.g., [ <xref rid="B35-ijms-18-01180" ref-type="bibr"> 35 </xref> ]), increasing marker density (e.g., [ <xref rid="B54-ijms-18-01180" ref-type="bibr"> 54 </xref> ]), and/or &#x0201c;mendelizing&#x0201d; the QTL by generating near-isogenic lines [ <xref rid="B35-ijms-18-01180" ref-type="bibr"> 35 </xref> , <xref rid="B55-ijms-18-01180" ref-type="bibr"> 55 </xref> ]. In some cases, meta-analysis of QTLs may replace or strengthen these methods. For example, mPO15-2, formed from eight projected QTL is coincident with the confirmed seed <span class="trait">protein</span> QTL cqSeed protein-001 (available online: <uri xlink:href="http://soybase.org"> http://soybase.org </uri> ) [ <xref rid="B56-ijms-18-01180" ref-type="bibr"> 56 </xref> ]. Satt384 (at 19.62 cM on the soybean consensus map) is positioned within the mPO15-2 CI. Satt384 was the key marker in the fine-mapping of cqSeed protein-001 and fine-mapping had previously served to decrease the interval to 535 kb based on <span class="gene">Gmax2.0</span> [ <xref rid="B35-ijms-18-01180" ref-type="bibr"> 35 </xref> ]. Though the positioning of the mPO15-2 flanking markers on the reference genome are inverted relative to the reference map, making a direct interpretation of results difficult, mPO15-2 coincided with the 535 kb interval identified by fine-mapping and even further narrowed this region ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 49 positional candidate genes encompassed by mPO5-1, 11 candidate genes had predicted functions related to <span class="trait">protein</span> and oil accumulation and/or biosynthesis ( <xref ref-type="fig" rid="ijms-18-01180-f002"> Figure 2 </xref> and <xref ref-type="table" rid="ijms-18-01180-t002"> Table 2 </xref> ). Of potential functional interest were the candidate genes annotated as transporters of either fatty acids <span class="gene">(Glyma.05g028500)</span> or nitrate. Fatty acids are transported across most membrane systems for modification and lipid assembly following synthesis in plastids [ <xref rid="B78-ijms-18-01180" ref-type="bibr"> 78 </xref> ]. <span class="gene">Glyma.05g028500</span> may contribute to the transportation of chloroplast produced fatty acids into the peroxisome, where they can be incorporated into &#x003b2;-oxidation. Nitrate uptake from the soil is transcriptionally controlled by both nitrate and photosynthate availability. De Jong et al. [ <xref rid="B79-ijms-18-01180" ref-type="bibr"> 79 </xref> ] showed nitrate uptake and assimilation was closely coordinated with glucose to supply amino acids and <span class="trait">protein</span> for plant growth. Thus, the five candidate genes annotated as nitrate transporters may play a role in production of amino acids and total <span class="trait">protein.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Meta-QTL mPO15-1 encompassed 49 positional candidate genes. Seven of these candidate genes putatively encode contributing factors to the biosynthesis of jasmonic acid, threonine, cell wall and alkaloid as well as degradation of <span class="trait">protein</span> and sucrose ( <xref ref-type="table" rid="ijms-18-01180-t002"> Table 2 </xref> and <xref ref-type="fig" rid="ijms-18-01180-f002"> Figure 2 </xref> ). <span class="gene">Glyma.15g028200</span> annotated as a peroxidase is involved in active oxygen species-scavenging systems [ <xref rid="B85-ijms-18-01180" ref-type="bibr"> 85 </xref> ] and methylglyoxal degradation by <span class="gene">Glyma.15g028900</span> may serve to increase the level of pyruvate [ <xref rid="B86-ijms-18-01180" ref-type="bibr"> 86 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Meta-QTL mPO15-2 encompassed positional 14 candidate genes, including <span class="gene">Glyma.15g049100</span> and <italic> <span class="gene">Glyma.15g050100</span> </italic> ( <xref ref-type="table" rid="ijms-18-01180-t002"> Table 2 </xref> and <xref ref-type="fig" rid="ijms-18-01180-f002"> Figure 2 </xref> ). <italic> <span class="gene">Glyma.15g049100</span> </italic> putatively encodes a vinorine synthase, playing a role in alkaloid biosynthesis. <italic> <span class="gene">Glyma.15g050100</span> </italic> putatively encodes a fructose-bisphosphatase, a key enzyme in glycolysis/sucrose biosynthesis [ <xref rid="B87-ijms-18-01180" ref-type="bibr"> 87 </xref> , <xref rid="B88-ijms-18-01180" ref-type="bibr"> 88 </xref> ]. Thus, this gene might be involved in carbon partitioning and further regulating <span class="trait">protein</span> and oil content in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 12 positional candidate genes encompassed by mPO15-5, notable functional candidates are <italic> <span class="gene">Glyma.15g142500</span> </italic> and <italic> <span class="gene">Glyma.15g143100</span> </italic> , which are putatively involved in cell wall degradation and acetyl-CoA biosynthesis, respectively ( <xref ref-type="table" rid="ijms-18-01180-t002"> Table 2 </xref> and <xref ref-type="fig" rid="ijms-18-01180-f002"> Figure 2 </xref> ). Enzymes in cell wall degradation of cell wall including glucan endo-1,3-&#x003b2;- <sc> d </sc> -glucosidase <span class="gene">(Glyma.15g142500)</span> involved in the complete hydrolysis of polysaccharides to glucose can lead to <span class="trait">protein</span> and oil accumulation [ <xref rid="B89-ijms-18-01180" ref-type="bibr"> 89 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Literature including scientific journals and available theses and dissertations were mined for QTL mapping studies conducted on soybean seed contents and compositions of <span class="trait">protein</span> and oil. Recent literature published up to and including 2016 were mined for relevant QTL mapping studies by search of Google Scholar using key words. In addition, mapping studies identified from SoyBase (available online: <uri xlink:href="http://soybase.org"> http://soybase.org </uri> ) were also included in the study. Parents, size, crossing type and generation of population and genetic map information of each population were collected. For each QTL, name, trait, experiment location, year of experiment, names of chromosome and linkage group, LOD score, <italic> R <sup> 2 </sup> </italic> value (proportion of phenotypic variance explained), most likely position of QTL (in cM) and confidence interval (CI, in cM) were collected. If the <span class="gene">CI</span> of a QTL was not provided, the formula proposed by [ <xref rid="B93-ijms-18-01180" ref-type="bibr"> 93 </xref> ] was used for calculation of a 95% of CI, <disp-formula> <span class="gene">CI</span> = 530/( <italic> N </italic> &#x000d7; <italic> R <sup> 2 </sup> </italic> ) <label> (1) </label> </disp-formula> <disp-formula> <span class="gene">CI</span> = 163/( <italic> N </italic> &#x000d7; <italic> R <sup> 2 </sup> </italic> ) <label> (2) </label> </disp-formula> where <italic> N </italic> is the size of population. <span class="gene">CI</span> of both backcross and F <sub> 2 </sub> populations were calculated using equation (1), equation (2) was applied to QTL studies carried out using recombinant inbred line populations. The formula derived from [ <xref rid="B94-ijms-18-01180" ref-type="bibr"> 94 </xref> ] was used for the estimation of LOD, which is the explanatory power of a QTL, if the LOD value was not reported, <disp-formula> <italic> R </italic> <sup> 2 </sup> = 1 &#x02212; 10 <sup> (&#x02212;2 LOD/N) </sup> <label> (3) </label> </disp-formula> All information was arranged by the name of the originating map. QTL information were discarded if <italic> R </italic> <sup> 2 </sup> value is unavailable from the original study.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparisons among <span class="trait">protein,</span> oil, protein+oil and protein+Cys+Met meta-quantitative trait loci (meta-QTLs) on Chromosome (Chr) 20 (Linkage group I; LG I). Projected QTL are displayed to the left of Chr 20 for each trait or trait combination. A 95% of confidence interval of each meta-QTL is represented as filled colors on the chromosome arm, with four, five, four and four meta-QTLs identified for <span class="trait">protein,</span> oil, protein+oil and protein+Cys+Met, respectively. A detailed view of the projected QTL contributing to each meta-QTL cluster #2 of Chr 20 ( <xref ref-type="table" rid="ijms-18-01180-t001"> Table 1 </xref> ) are also shown on the right of the chromosome. In the detailed view, each <span class="gene">bar</span> represents a projected QTL and is color-coded for each trait <span class="trait">(protein,</span> khaki; oil, red; cysteine, green; and methionine, blue).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide comparative analysis of DNA methylation between soybean cytoplasmic male-sterile line NJCMS5A and its maintainer NJCMS5B</h2>
			<h3>PMID:28806912</h3>
				
					<h3>PMCID:5557475</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yanwei </given-names>  </name>  <address>  <email> wwyixiao2013@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ding </surname>  <given-names> Xianlong </given-names>  </name>  <address>  <email> xlding2012@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xuan </given-names>  </name>  <address>  <email> sink0828@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Tingting </given-names>  </name>  <address>  <email> ht142857@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hao </given-names>  </name>  <address>  <email> haozhang1223@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Longshu </given-names>  </name>  <address>  <email> dashu6090@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Tanliu </given-names>  </name>  <address>  <email> tanliuWTL@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Linfeng </given-names>  </name>  <address>  <email> linfeng_chen2015@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <address>  <email> sri@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Yang </surname>  <given-names> Shouping </given-names>  </name>  <address>  <email> spyung@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9750 7019 </institution-id>  <institution-id institution-id-type="GRID"> grid.27871.3b </institution-id>  <institution> Soybean Research Institute, National Center for Soybean Improvement, Key Laboratory of Biology and Genetic Improvement of Soybean (General, Ministry of Agriculture), State Key Laboratory of Crop Genetics and Germplasm Enhancement, Jiangsu Collaborative Innovation Center for Modern Crop Production,  </institution>  <institution> Nanjing Agricultural University,  </institution>  </institution-wrap> Nanjing, 210095 China  </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 7 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						DNA methylation patterns of four differentially methylated genes (DMGs) validated by bisulfite treatment. <bold> a </bold> <span class="gene">Glyma.04G206000</span> (oxysterol binding <span class="trait">protein);</span> <bold> b </bold> <span class="gene">Glyma.19G144200</span> (basic helix-loop-helix DNA-binding <span class="trait">protein);</span> <bold> c </bold> <span class="gene">Glyma.16G081800</span> (lipase); <bold> d </bold> <span class="gene">Glyma.06G266900</span> (galacturonosyl transferase). The red, blue and green columns in the histograms refer to the collective methylation levels (in percentage) in CG, CHG and CHH contexts, respectively
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Soybean <italic> Rfg1 </italic> Gene Restricts Nodulation by <italic> Sinorhizobium fredii </italic> USDA193</h2>
			<h3>PMID:28936222</h3>
				
					<h3>PMCID:5594104</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Yinglun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/456516/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jinge </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/456419/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lyu </surname>  <given-names> Shanhua </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/472783/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Qi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/456420/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Shengming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/456411/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/432702/overview"/>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We have cloned several dominant genes in soybeans (e.g., <italic> <span class="gene">Rj2</span> </italic> , <italic> Rfg1 </italic> , and <italic> <span class="gene">Rj4</span> </italic> ) that restrict nodulation with specific rhizobial strains ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ; <xref rid="B41" ref-type="bibr"> Tang et al., 2016 </xref> ). In these cases, symbiosis incompatibility is controlled in a similar manner as &#x02018;gene-for-gene&#x02019; <span class="trait">resistance</span> against plant pathogens ( <xref rid="B39" ref-type="bibr"> Sadowsky et al., 1990 </xref> ; <xref rid="B9" ref-type="bibr"> Devine and Kuykendall, 1996 </xref> ). <italic> <span class="gene">Rj2</span> </italic> and <italic> Rfg1 </italic> are allelic genes, each encoding a typical Toll-interleukin receptor/nucleotide-binding site/leucine-rich repeat (TIR-NBS-LRR) <span class="trait">resistance</span> <span class="trait">protein</span> that confers <span class="trait">resistance</span> to nodulation by specific strains of <italic> Bradyrhizobium japonicum </italic> and <italic> Sinorhizobium fredii </italic> , respectively ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ), while <italic> <span class="gene">Rj4</span> </italic> encodes a thaumatin-like pathogenesis-related <span class="trait">protein</span> that restricts nodulation by specific strains of <italic> B. elkanii </italic> ( <xref rid="B41" ref-type="bibr"> Tang et al., 2016 </xref> ). Moreover, the function of these nodulation-restrictive genes is dependent on the bacterial type III secretion system ( <xref rid="B26" ref-type="bibr"> Krishnan et al., 2003 </xref> ; <xref rid="B34" ref-type="bibr"> Okazaki et al., 2009 </xref> ; <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ; <xref rid="B43" ref-type="bibr"> Tsukui et al., 2013 </xref> ; <xref rid="B44" ref-type="bibr"> Tsurumaru et al., 2015 </xref> ; <xref rid="B41" ref-type="bibr"> Tang et al., 2016 </xref> ; <xref rid="B50" ref-type="bibr"> Yasuda et al., 2016 </xref> ). These studies revealed an important role of effector-triggered plant immunity in the regulation of nodulation specificity in soybeans ( <xref rid="B45" ref-type="bibr"> Wang et al., 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Here we describe the study of symbiotic incompatibility of soybean with <italic> S. fredii </italic> USDA193. A previous report suggested that restriction of nodulation by this strain is associated with the <italic> <span class="gene">Rj2</span> </italic> and/or <italic> Rj3 </italic> loci ( <xref rid="B33" ref-type="bibr"> Nakano et al., 1997 </xref> ). Our study revealed a single dominant gene responsible for this incompatibility. Genetic mapping in an F2 population showed co-segregation of the underlying locus with the previously cloned <italic> Rfg1 </italic> gene that confers <span class="trait">resistance</span> to nodulation by <italic> S. fredii </italic> strains USDA257 and USDA205 ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ). Through complementation tests and CRISPR/Cas9-mediated gene disruption, we demonstrate that the <italic> Rfg1 </italic> allele also is responsible for restricting nodulation by <italic> S. fredii </italic> USDA193. Our study suggests that the <italic> Rfg1 </italic> locus is involved in the determination of nodulation specificity with multiple <italic> S. fredii </italic> and <italic> B. japonicum </italic> strains in soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						Williams 82 and Peking showed the same phenotype when inoculated with <italic> S. fredii </italic> strains USDA257 and USDA193, and we previously reported that the <span class="trait">resistance</span> to nodulation with USDA257 in Williams 82 is controlled by the dominant <italic> Rfg1 </italic> allele ( <italic> <span class="gene">Glyma16g33780</span> </italic> ) that encodes a TIR-NBS-LRR <span class="trait">protein</span> ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ). We therefore suspected that <italic> Rfg1 </italic> possibly also confers <span class="trait">resistance</span> to <italic> S. fredii </italic> USDA193. To test this possibility, we started the mapping experiment by using a polymorphic DNA marker developed from <italic> Rfg1 </italic> . Consistent with our hypothesis, linkage analysis in the aforementioned F2 population revealed co-segregation between the nodulation phenotypes and the marker genotypes. Thus, we considered <italic> Rfg1 </italic> as a candidate gene that restricts nodulation by <italic> S. fredii </italic> USDA193.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Rfg1 </italic> and <italic> <span class="gene">Rj2</span> </italic> are allelic genes, each encoding a TIR-NBS-LRR <span class="trait">protein</span> of 1052 amino acids ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ). A survey of a group of soybean lines identified three types of naturally occurring alleles, namely <italic> <span class="gene">Rj2</span> (rfg1) </italic> , <italic> <span class="gene">rj2</span> </italic> ( <italic> Rfg1 </italic> ), and <italic> <span class="gene">rj2</span> </italic> ( <italic> rfg1 </italic> ) ( <xref rid="B48" ref-type="bibr"> Yang et al., 2010 </xref> ). The <italic> <span class="gene">Rj2</span> (rfg1 </italic> ) allele restricts nodulation with USDA122 but not with USDA257; the <italic> <span class="gene">rj2</span> </italic> ( <italic> Rfg1 </italic> ) allele restricts nodulation with USDA257 but not with USDA122; and the <italic> <span class="gene">rj2</span> </italic> ( <italic> rfg1 </italic> ) allele allows nodulation with both strains. These allelic specificities are determined by seven amino acid substitutions occurring around the C-terminus of the NBS domain and the sixth LRR repeat ( <bold> Figures <xref ref-type="fig" rid="F4"> 4A,B </xref> </bold> ). Comparing between <span class="gene">Rj2</span> (rfg1) and <span class="gene">rj2</span> (rfg1) and between <span class="gene">rj2</span> (Rfg1), and <span class="gene">rj2</span> (rfg1) suggests that E452 and I490 are required for the <italic> <span class="gene">Rj2</span> </italic> -mediated nodulation restriction against USDA122, while E731, N736, S743, D756, and S758 are essential for <italic> Rfg1 </italic> -mediated <span class="trait">resistance</span> against USDA257 and USDA193. If this inference is true, then the chimeric gene of <italic> <span class="gene">Rj2</span> </italic> and <italic> Rfg1 </italic> , called <italic> <span class="gene">Rj2</span> </italic> ( <italic> Rfg1 </italic> ), encoding a <span class="trait">protein</span> with E452, I490, E731, N736, S743, D756, and S758, would prevent nodulation with both strains. We generated multiple transgenic roots (from &#x0003e;100 independent plants) expressing the chimeric gene in Peking that carries an <italic> <span class="gene">rj2</span> </italic> ( <italic> rfg1 </italic> ) allele and forms nodules with USDA257, USDA193, and USDA122. In consistence with our hypothesis, the transgenic roots restrict nodulation with both USDA257 ( <bold> Figure <xref ref-type="fig" rid="F4"> 4C </xref> </bold> ), USDA193 ( <bold> Figure <xref ref-type="fig" rid="F4"> 4D </xref> </bold> ), and USDA122 ( <bold> Figure <xref ref-type="fig" rid="F4"> 4E </xref> </bold> ). The transgenic roots retained their ability to nodulate with <italic> B. japonicum </italic> USDA110, a strain that nodulates both <italic> <span class="gene">Rj2</span> </italic> and <italic> Rfg1 </italic> genotypes ( <bold> Figure <xref ref-type="fig" rid="F4"> 4F </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Functional analysis of a chimeric gene of <italic> Rfg1 </italic> and <italic> <span class="gene">Rj2</span> </italic> . <bold> (A,B) </bold> Domain structure of the TIR-NBS-LRR <span class="trait">protein</span> <bold> (A) </bold> showing the seven substitution sites and amino acid polymorphisms that distinguish between the <span class="gene">Rj2</span> (rfg1), <span class="gene">rj2</span> (rfg1), and <span class="gene">rj2</span> (Rfg1) <span class="trait">protein</span> isoforms <bold> (B) </bold> . <span class="gene">Rj2</span> (Rfg1) represents a <span class="trait">protein</span> isoform resulting from expressing the chimeric gene. <bold> (C&#x02013;F) </bold> Transgenic roots expressing the chimeric gene (blue) in the Peking background restricted nodulation by USDA257 <bold> (C) </bold> , USDA193 <bold> (D) </bold> , and USDA122 <bold> (E) </bold> but allowed nodulation with <italic> B. japonicum </italic> USDA110 <bold> (F) </bold> . In all cases, the nodulation was normal on the non-transgenic roots (white).
					</p>
				
					<p>&nbsp;&nbsp;
						The polymorphic <italic> <span class="gene">Rj2</span> </italic> / <italic> Rfg1 </italic> locus encodes a TIR-NB-LRR gene with three allelic variants that show differing specificities to yet unknown bacterial effectors. E452 and I490 in or near the C-terminal NBS domain of <span class="gene">Rj2</span> are required for the recognition of the cognate effector from <italic> B. japonicum </italic> USDA122, while E731, N736, S743, D756, and S758 in the LRR domain of Rfg1 are essential for the recognition of the corresponding effectors secreted by USDA257 and USDA193. This situation is similar to the previous studies showing that both TIR, NBS, and LRR domains could be involved in the determination of <span class="trait">resistance</span> specificity ( <xref rid="B15" ref-type="bibr"> Ellis et al., 1999 </xref> ; <xref rid="B11" ref-type="bibr"> Dodds et al., 2001 </xref> ; <xref rid="B31" ref-type="bibr"> Mondrag&#x000f3;n-Palomino et al., 2002 </xref> ; <xref rid="B14" ref-type="bibr"> Ellis et al., 2007 </xref> ). The chimeric <italic> Rj2(Rfg1) </italic> allele composed of E452, I490, E731, N736, S743, D756, and S758 could recognize corresponding effectors of all these strains. Such an allele could exist in natural population if intragenic recombination events occur between the <italic> Rj2(rfg1) </italic> and <italic> rj2(Rfg1) </italic> alleles. It is possible that these polymorphic sites play a role in direct effector binding or intra- or intermolecular interactions in <span class="trait">protein</span> complexes during recognition and signaling. Further analysis of the mechanisms underlying the recognitions mediated by the <italic> Rj2/Rfg1 </italic> locus awaits the identification of the cognate bacterial effectors.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-quality draft genome sequence of <italic> Ensifer meliloti </italic> Mlalz-1, a microsymbiont of <italic> Medicago laciniata </italic> (L.) miller collected in Lanzarote, Canary Islands, Spain</h2>
			<h3>PMID:28975015</h3>
				
					<h3>PMCID:5613336</h3>
					<h4>期刊:Standards in Genomic Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Osman </surname>  <given-names> Wan Adnawani Meor </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> van Berkum </surname>  <given-names> Peter </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Le&#x000f3;n-Barrios </surname>  <given-names> Milagros </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vel&#x000e1;zquez </surname>  <given-names> Encarna </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Elia </surname>  <given-names> Patrick </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Rui </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-9938-606X </contrib-id>  <name>  <surname> Ardley </surname>  <given-names> Julie </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gollagher </surname>  <given-names> Margaret </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seshadri </surname>  <given-names> Rekha </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> T. B. K. </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ivanova </surname>  <given-names> Natalia </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Woyke </surname>  <given-names> Tanja </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pati </surname>  <given-names> Amrita </given-names>  </name>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Markowitz </surname>  <given-names> Victor </given-names>  </name>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baeshen </surname>  <given-names> Mohamed N. </given-names>  </name>  <xref ref-type="aff" rid="Aff8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Baeshen </surname>  <given-names> Naseebh Nabeeh </given-names>  </name>  <xref ref-type="aff" rid="Aff8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kyrpides </surname>  <given-names> Nikos </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Reeve </surname>  <given-names> Wayne </given-names>  </name>  <address>  <email> W.Reeve@murdoch.edu.au </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0436 6763 </institution-id>  <institution-id institution-id-type="GRID"> grid.1025.6 </institution-id>  <institution> School of Veterinary and Life Sciences,  </institution>  <institution> Murdoch University,  </institution>  </institution-wrap> Murdoch, WA Australia  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0478 6311 </institution-id>  <institution-id institution-id-type="GRID"> grid.417548.b </institution-id>  <institution> U.S. Department of Agriculture, Soybean Genomics and Improvement Laboratory,  </institution>  <institution> Beltsville Agricultural Research Center,  </institution>  </institution-wrap> 10300 Baltimore Avenue, Bldg. 006, Beltsville, MD 20705 USA  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000121060879 </institution-id>  <institution-id institution-id-type="GRID"> grid.10041.34 </institution-id>  <institution> Departamento de Bioqu&#x000ed;mica, Microbiolog&#x000ed;a, Biolog&#x000ed;a Celular y Gen&#x000e9;tica,  </institution>  <institution> Universidad de La Laguna,  </institution>  </institution-wrap> Tenerife, Spain  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2180 1817 </institution-id>  <institution-id institution-id-type="GRID"> grid.11762.33 </institution-id>  <institution> Departamento de Microbiolog&#x000ed;a y Genetica and Instituto Hispanoluso de Investigaciones Agrarias (CIALE),  </institution>  <institution> Universidad de Salamanca,  </institution>  </institution-wrap> Salamanca, Spain  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0375 4078 </institution-id>  <institution-id institution-id-type="GRID"> grid.1032.0 </institution-id>  <institution> Curtin University Sustainability Policy Institute,  </institution>  <institution> Curtin University,  </institution>  </institution-wrap> Bentley, WA Australia  </aff>  <aff id="Aff6">  <label> 6 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0449 479X </institution-id>  <institution-id institution-id-type="GRID"> grid.451309.a </institution-id>  <institution> DOE Joint Genome Institute,  </institution>  </institution-wrap> Walnut Creek, CA USA  </aff>  <aff id="Aff7">  <label> 7 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2231 4551 </institution-id>  <institution-id institution-id-type="GRID"> grid.184769.5 </institution-id>  <institution> Biological Data Management and Technology Center, Lawrence Berkeley National Laboratory,  </institution>  </institution-wrap> Berkeley, CA USA  </aff>  <aff id="Aff8">  <label> 8 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.460099.2 </institution-id>  <institution> Department of Biology, Faculty of Science,  </institution>  <institution> University of Jeddah,  </institution>  </institution-wrap> Jeddah, Saudi Arabia  </aff>
				<br/> 
				年份：<day> 14 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> a </bold> Map of 10.1601/nm.1335 Mlalz-1: A3CADRAFT_scaffold_ 5.6. <bold> b </bold> Genetic organization of the T2SS clusters in 10.1601/nm.1335 Mlalz-1 ( <italic> Emel </italic> Mlalz-1), 10.1601/nm.1331 HH103 ( <italic> Efre </italic> HH103) and 10.1601/nm.1331 10.1601/strainfinder?urlappend=%3Fid%3DNGR+234 ( <italic> Efre </italic> 10.1601/strainfinder?urlappend=%3Fid%3DNGR+234). J, <italic> gspJ </italic> ; I, <italic> gspI </italic> ; H, <italic> <span class="gene">gspH</span> </italic> ; F, <italic> <span class="gene">gspF</span> </italic> ; E, <italic> <span class="gene">gspE</span> </italic> ; K, <italic> <span class="gene">gspK</span> </italic> ; C, <italic> gspC </italic> ; M, <italic> gspM </italic> ; L, <italic> gspL </italic> ; G, <italic> <span class="gene">gspG</span> </italic> ; O, <italic> gspO </italic> ; D, <italic> <span class="gene">gspD</span> </italic> . <bold> c </bold> Schematics of the T2SS of Gram-negative bacteria [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. The secretin, GspD (A3CADRAFT_01569); the polytopic <span class="trait">protein,</span> GspF (A3CADRAFT_01561); the cytoplasmic ATPase, GspE (A3CADRAFT_01562); the major pseudopilin component, GspG (A3CADRAFT_01567); the minor pseudopilins, GspH (A3CADRAFT_01560), GspI (A3CADRAFT_01559), GspJ (A3CADRAFT_01558) and GspK (A3CADRAFT_01563); the bitopic proteins, GspL (A3CADRAFT_01566), GspC (A3CADRAFT_01564) and GspM (A3CADRAFT_01565); the peptidase, GspO (A3CADRAFT_01568)(GspS is absent from Mlalz-1); OM, outer membrane; PG, peptidoglycan; IM, inner membrane
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An excess dietary vitamin E concentration does not influence Nrf2 signaling in the liver of rats fed either soybean oil or salmon oil</h2>
			<h3>PMID:29176993</h3>
				
					<h3>PMCID:5693465</h3>
					<h4>期刊:Nutrition &#x00026; Metabolism</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Eder </surname>  <given-names> Klaus </given-names>  </name>  <address>  <email> klaus.eder@ernaehrung.uni-giessen.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Siebers </surname>  <given-names> Marina </given-names>  </name>  <address>  <email> marina.siebers@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Most </surname>  <given-names> Erika </given-names>  </name>  <address>  <email> erika.most@ernaehrung.uni-giessen.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Scheibe </surname>  <given-names> Susan </given-names>  </name>  <address>  <email> susan.scheibe@innere.med.uni-giessen.de </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weissmann </surname>  <given-names> Norbert </given-names>  </name>  <address>  <email> norbert.weissmann@innere.med.uni-giessen.de </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Gessner </surname>  <given-names> Denise K. </given-names>  </name>  <address>  <email> denise.gessner@ernaehrung.uni-giessen.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2165 8627 </institution-id>  <institution-id institution-id-type="GRID"> grid.8664.c </institution-id>  <institution> Institute of Animal Nutrition and Nutrition Physiology,  </institution>  <institution> Justus-Liebig-Universit&#x000e4;t Gie&#x000df;en,  </institution>  </institution-wrap> Heinrich-Buff-Ring 26-32, 35392 Gie&#x000df;en, Germany  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2165 8627 </institution-id>  <institution-id institution-id-type="GRID"> grid.8664.c </institution-id>  <institution> Excellence Cluster Cardio-Pulmonary System (ECCPS),  </institution>  <institution> Justus-Liebig-Universit&#x000e4;t Gie&#x000df;en,  </institution>  </institution-wrap> Aulweg 130, 35392 Gie&#x000df;en, Germany  </aff>
				<br/> 
				年份：<day> 31 </day>  <month> 10 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<sup> a </sup> <italic> ATP5B </italic> ATP synthase, <italic> H+ </italic> transporting, mitochondrial F1 complex, beta polypeptide, <italic> ATF4 </italic> activating transcription factor 4, <italic> CANX </italic> calnexin, <italic> CAT </italic> catalase, <italic> DDIT3 </italic> DNA damage inducible transcript&#x000a0;3, <italic> DNAJC3 </italic> <span class="gene">DnaJ</span> heat shock <span class="trait">protein</span> family (Hsp40) member C3, <italic> EDEM1 </italic> ER degradation enhancing alpha-mannosidase like <span class="trait">protein</span> 1, <italic> GCLM </italic> glutamate cysteine ligase, modifier subunit, <italic> GPX1 </italic> glutathione peroxidase 1, <italic> GSTA1 </italic> glutathione S-transferase alpha 1, <italic> HERPUD1 </italic> homocysteine inducible ER <span class="trait">protein</span> with ubiquitin like domain 1, <italic> HMOX1 </italic> heme oxygenase 1, <italic> HSPA5 </italic> heat shock <span class="trait">protein</span> family A member 5, <italic> KEAP1 </italic> kelch like ECH associated <span class="trait">protein</span> 1, <italic> MDH1 </italic> malate dehydrogenase 1, <italic> NFE2L2 </italic> nuclear factor, erythroid 2 like 2, <italic> NQO1 </italic> NAD(P)H quinone dehydrogenase 1, <italic> PDIA4 </italic> <span class="trait">protein</span> disulfide isomerase family A, member 4, <italic> <span class="gene">SOD1</span> </italic> superoxide dismutase 1, <italic> SULT1B1 </italic> sulfotransferase family 1B member 1, <italic> UGT1A6 </italic> UDP glucuronosyltransferase family 1 member A6
					</p>
				
					<p>&nbsp;&nbsp;
						<sup> a </sup> <italic> Abbreviations </italic> : <italic> CAT </italic> catalase, <italic> GCLM </italic> glutamate-cysteine ligase modifier subunit, <italic> GPX1 </italic> glutathione peroxidase 1, <italic> GSTA1 </italic> glutathione S-transferase alpha 1, <italic> HMOX1 </italic> heme oxygenase 1, <italic> KEAP1 </italic> kelch like ECH associated <span class="trait">protein</span> 1, <italic> NFE2L2 </italic> nuclear factor, erythroid 2 like 2, <italic> NQO1 </italic> NAD(P)H quinone dehydrogenase 1, <italic> <span class="gene">SOD1</span> </italic> superoxide dismutase 1, <italic> SULT1B1 </italic> sulfotransferase family 1B member 1, <italic> UGT1A6 </italic> UDP glucuronosyltransferase family 1 member A6
					</p>
				
					<p>&nbsp;&nbsp;
						<sup> a </sup> <italic> ATF4 </italic> activating transcription factor 4, <italic> DDIT3 </italic> DNA damage inducible transcript&#x000a0;3, <italic> DNAJC3 </italic> <span class="gene">DnaJ</span> heat shock <span class="trait">protein</span> family (Hsp40) member C3, <italic> EDEM1 </italic> ER degradation enhancing alpha-mannosidase like <span class="trait">protein</span> 1, <italic> HERPUD1 </italic> homocysteine inducible ER <span class="trait">protein</span> with ubiquitin like domain 1, <italic> HSPA5 </italic> heat shock <span class="trait">protein</span> family A member 5, <italic> PDIA4 </italic> <span class="trait">protein</span> disulfide isomerase family A, member 4
					</p>
				
					<p>&nbsp;&nbsp;
						Effect on <span class="trait">protein</span> expression of UPR targets. <bold> a </bold> Relative <span class="trait">protein</span> expression of <span class="gene">BIP</span> and ( <bold> b </bold> ) ratio of <span class="trait">protein</span> expression of p-elF2&#x003b1; and total elf2&#x003b1; in the liver of rats fed diets with either soybean oil or salmon oil with various vitamin E concentrations. Bars represent relative <span class="trait">protein</span> level expressed as fold of control (diet with soybean oil and 25&#x000a0;mg/kg vitamin E) and are means &#x000b1; SD from 12 animals per group. Representative immunoblots specific to <span class="gene">BIP</span> ( <bold> a </bold> ), p-elF2&#x003b1; and total elf2&#x003b1; ( <bold> b </bold> ) and &#x000df;-actin and &#x003b1;-tubulin as respective loading controls are shown for one animal per group
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Three deaf mice: mouse models for TECTA-based human hereditary deafness reveal domain-specific structural phenotypes in the tectorial membrane</h2>
			<h3>PMID:24363064</h3>
				
					<h3>PMCID:3990158</h3>
					<h4>期刊:Human Molecular Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Legan </surname>  <given-names> P. Kevin </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="author-notes" rid="AN1"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Goodyear </surname>  <given-names> Richard J. </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="author-notes" rid="AN1"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mor&#x000ed;n </surname>  <given-names> Mat&#x000ed;as </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  <xref ref-type="author-notes" rid="AN1"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mencia </surname>  <given-names> Angeles </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pollard </surname>  <given-names> Hilary </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Olavarrieta </surname>  <given-names> Leticia </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Korchagina </surname>  <given-names> Julia </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Modamio-Hoybjor </surname>  <given-names> Silvia </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mayo </surname>  <given-names> Fernando </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moreno </surname>  <given-names> Felipe </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moreno-Pelayo </surname>  <given-names> Miguel-Angel </given-names>  </name>  <xref ref-type="aff" rid="af2"> 2 </xref>  <xref ref-type="aff" rid="af3"> 3 </xref>  <xref ref-type="author-notes" rid="AN2"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Richardson </surname>  <given-names> Guy P. </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="author-notes" rid="AN2"> &#x02021; </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Light microscopy was used to examine the location and gross structure of the TM in mice homozygous for the three Tecta mutations (Fig.&#x000a0; <xref ref-type="fig" rid="DDT646F11"> 11 </xref> ). In the Tecta <sup> C1619S/C1619S </sup> TM mouse, the TM has a &#x02018;hump-backed&#x02019; cross-sectional profile and a much reduced limbal attachment zone (Fig.&#x000a0; <xref ref-type="fig" rid="DDT646F11"> 11 </xref> A), a phenotype resembling that seen in mice heterozygous for the ZP domain mutations. The disruption of the marginal band seen in Tecta <sup> C1619S/+ </sup> mice is exacerbated in homozygous animals. TMs of both Tecta <sup> L1820F,G1824D/L1820F,G1824D </sup> and Tecta <sup> C1837G/C1837G </sup> animals are completely detached from the spiral limbus and are associated instead with Reissner's membrane (Fig.&#x000a0; <xref ref-type="fig" rid="DDT646F11"> 11 </xref> B and C). <span class="trait">High</span> levels of Tecta are detected by immunofluorescence in the TM of Tecta <sup> C1619S/C1619S </sup> cochleae, and Tecta is also readily detectable in the detached TM of the Tecta <sup> L1820F,G1824D/L1820F,G1824D </sup> mice (Fig.&#x000a0; <xref ref-type="fig" rid="DDT646F11"> 11 </xref> D and E). In the TM of the Tecta <sup> C1837G/C1837G </sup> mouse, Tecta is virtually undetectable (Fig.&#x000a0; <xref ref-type="fig" rid="DDT646F11"> 11 </xref> F), although some weak labelling is seen at the extreme apical end (not shown). Similarly, Tectb staining is detected throughout the TM of the Tecta <sup> C1619S/C1619S </sup> mouse, is present along the length of the TM of the Tecta <sup> L1820F,G1824D/L1820F,G1824D </sup> mouse, but is restricted to the extreme apical end of the TM in the Tecta <sup> C1837G/C1837G </sup> mouse (not shown). <fig id="DDT646F11" position="float"> <label> Figure&#x000a0;11. </label> <caption> <p> Structure of the organ of Corti in homozygous <italic> Tecta </italic> mutants. Toluidine blue stained resin sections ( <bold> A </bold> &#x02013; <bold> C </bold> ) and anti-Tecta (red) and phalloidin (green) labelled cryosections ( <bold> D </bold> &#x02013; <bold> F </bold> ) from basal cochlear coil of homozygous Tecta <sup> C1619S/C1619S </sup> (A, D) Tecta <sup> L1820F,G1824D/L1820F,G1824D </sup> (B, E) and Tecta <sup> C1837G/C1837G </sup> (C, F) mutant mice. The tectorial membrane in the Tecta <sup> C1619S/C1619S </sup> mouse (A, D) has reduced attachment to the spiral limbus (SL), a highly fragmented marginal band (arrow) but retains strong labelling for Tecta (D). The tectorial membranes of Tecta <sup> L1820F,G1824D/L1820F,G1824D </sup> and Tecta <sup> C1837G/C1837G </sup> mice are completely detached (arrows in B&#x02013;C, E&#x02013;F) from the spiral limbus and instead are associated with Reissner's membrane (arrowhead in B and C). Although the matrix of the tectorial membrane in the Tecta <sup> L1820,G1824D/L1820F,G1824D </sup> mouse stains positively for Tecta (E) that in Tecta <sup> C1837G/C1837G </sup> mutant does not (F). This may be reflected in the relative density of the tectorial membranes in the Toluidine blue sections (B, C). Scale <span class="gene">bar</span> in <bold> C </bold> = 50 &#x003bc;m and applies to A&#x02013;C, scale <span class="gene">bar</span> in F = 50 &#x003bc;m and applies to D&#x02013;F.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of the hybrid proline-rich <span class="trait">protein</span> families from seven plant species suggests rapid diversification of their sequences and expression patterns</h2>
			<h3>PMID:17997832</h3>
				
					<h3>PMCID:2216038</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Dvo&#x00159;&#x000e1;kov&#x000e1; </surname>  <given-names> Lenka </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lenkadvorakova@centrum.cz </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Cvr&#x0010d;kov&#x000e1; </surname>  <given-names> Fatima </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> fatima@natur.cuni.cz </email>  </contrib>  <contrib id="A3" corresp="yes" contrib-type="author">  <name>  <surname> Fischer </surname>  <given-names> Luk&#x000e1;&#x00161; </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lukasf@natur.cuni.cz </email>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 11 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						HyPRPs are characterized by the presence of two different domains: a proline-rich N-terminal repetitive domain and a hydrophobic C-terminal domain. Based on the proline-rich domain and a secretory signal, HyPRPs belong to the group of secreted structural cell wall proline-rich proteins [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. The repetitive character and <span class="trait">high</span> proline content of N-terminal domains resemble other proline-rich proteins, though the repeated amino acid motifs vary [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. The hydrophobic C-terminal domain classifies HyPRPs into the group of proteins containing the 8 CM domain [ <xref ref-type="bibr" rid="B3"> 3 </xref> ], together with lipid transfer proteins (LTPs), amylase inhibitors, 2S albumins and some other subgroups. The characteristic 8 CM domain usually consists of less than 100 amino acid residues and contains eight cysteine residues in a specific order [ <xref ref-type="bibr" rid="B3"> 3 </xref> ]. Tertiary structure formed by four hydrophobic helices stabilized by the conserved cysteines has been determined for several proteins of the family since the first reports on crystallographic analysis of the soybean ( <italic> Glycine max </italic> ) hydrophobic seed <span class="trait">protein</span> [ <xref ref-type="bibr" rid="B4"> 4 </xref> ]) and NMR analysis of wheat <span class="gene">LTP</span> in solution [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Although no structural data are available for proteins of the HyPRP subgroup, which has been, so far, analyzed only on the sequence level, we can assume that the tertiary structure of their C-terminal domains resembles other 8 CM proteins. However, the conserved structure does not allow any functional predictions. It apparently serves only as a scaffold carrying specific functional elements in various subgroups of the 8 CM family [ <xref ref-type="bibr" rid="B2"> 2 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Hybrid proline-rich proteins (HyPRPs) represent a group of secreted cell wall proteins specific to seed plants. We could find <italic> bona fide </italic> homologues of <italic> HyPRP </italic> genes only in database sequences derived from angio- and gymnosperm plants, while no representatives of this family were identified in non-seed plant species. In contrast, putative members of the lipid transfer <span class="trait">protein</span> (LTP) family, which share the conserved 8 CM domain with HyPRPs, were found even in a green alga ( <italic> Chlamydomonas </italic> ) and in the moss <italic> Physcomitrella patens </italic> (LD and LF, unpublished). We thus hypothesize that the first <italic> HyPRP </italic> gene may have been a <italic> <span class="gene">LTP</span> </italic> derivative that acquired a sequence encoding proline-rich N-terminal domain either by means of gene fusion or by introduction of an in-frame repetitive element. This event may have represented one of the evolutionary innovations of seed plants. This view is consistent with the previously described <span class="trait">high</span> flexibility and adaptability of the plant 8 CM domains (including LTPs and HyPRPs) to diverse functional requirements and sequence contexts [ <xref ref-type="bibr" rid="B2"> 2 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Sequence variability, similar to that observed for HyPRPs, was documented also for the related family of LTPs, where only the eight cysteines of the 8 CM motif were absolutely conserved [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Using <italic> in vitro </italic> assays, only loose substrate-binding specificity was found among different <span class="gene">LTP</span> family members; LTPs analyzed in detail bound a wide spectrum of lipidic ligands, from <span class="gene">C10</span> to C18 [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Therefore, we assume that, similar to the LTPs, <span class="trait">high</span> variability of both C- and N- terminal domains of HyPRPs is rather a consequence of low selection pressure, possible functional redundancy, limited degree of functional specialization and <span class="trait">high</span> flexibility of the 8 CM domain.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>