<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>lodging</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>lodging</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:[Relationship between cellulose synthesis metabolism and <span class="trait">lodging</span> <span class="trait">resistance</span> in intercropping soybean at seedling stage].</h2>
			<h3>PMID:27396119  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deng YC, Liu WG, Yuan XQ, Yuan J, Zou JL, Du JB, Yang WY.
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Physical characteristics of stem are closely relative to the crop <span class="trait">lodging.</span> Increase of stem strength is conducive to resolve the problem of <span class="trait">lodging.</span> Three soybean cultivars with different shade tolerance were planted under maize-soybean intercropping and soybean monocropping, respectively. Physiological and biochemical indices including cellulose, soluble sugar, sucrose, starch contents and enzyme activity were investigated to assess the snapping <span class="trait">resistance</span> and <span class="trait">lodging</span> <span class="trait">resistance</span> of the stems of soybean seedling, and snapping- and lodging-resistance indices were calculated for further verification. Furthermore, relationship analyses between these factors and the <span class="trait">lodging</span> of inter-cropped soybean showed that the intercropping soybean lodged seriously, the snapping <span class="trait">resistance,</span> <span class="trait">lodging</span> <span class="trait">resistance</span> index, contents of cellulose, soluble sugar, sucrose, starch and activities of the related enzymes were significantly lower than monocropping soybean at seedling stage. The three soybean cultivars showed different phenotypes in intercropping condition. The snapping-resistant Nandou12 with strong shade-tolerant traits was the most lodging-resistant phenotype, and it also harbored high contents of cellulose, soluble sugar, sucrose, starch and active enzymes. The <span class="trait">lodging</span> <span class="trait">resistance</span> index, cellulose content of the stems of intercropped soybean seedling were significantly positively correlated with the snapping <span class="trait">resistance,</span> and were significantly negatively correlated with the actual <span class="trait">lodging</span> percentage. The activities of sucrose phosphate synthase (SPS) , sucrose synthase (SS) and neutral invertase (NI) were positively correlated with sucrose is content, but not the acid invertase (AI). The activities of SPS, NI and <span class="gene">SS</span> were positively correlated with cellulose content, but not Al. In a word, the high activities of SPS and <span class="gene">SS</span> in the soybean stem were the enzymatic basis to maintain relatively higher cellulose and sucrose content, which is conducive to improve the stem-sfrength and enhance the <span class="trait">lodging</span> <span class="trait">resistance</span> ability in intercropping condition. Effects of different light conditions on cellulose metabolic mechanism of soybean seedling stems, <span class="trait">lodging</span> resistant characteristics of soybean seedlings studied in the corn-soybean intercropping system provided a basis for screening more shade-tolerant soybean variety.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, Heterodera glycines Ichinohe). Although this approach successfully mapped a large number of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse set of 553 soybean plant introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as seed coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dual functions of <span class="gene">GmTOE4a</span> in the regulation of photoperiod-mediated flowering and plant morphology in soybean.</h2>
			<h3>PMID:25991035  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao X(1), Cao D, Huang Z, Wang J, Lu S, Xu Y, Liu B, Kong F, Yuan X.
				<br/> 
				年份：2015 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flowering time, maturity, and plant morphology have considerable effects on the adaptation and grain <span class="trait">yield</span> of soybean (Glycine max). The identification of novel genes and an understanding of their molecular basis are critical to improve soybean productivity. In this study, we cloned a flowering time related APETALA2-like gene <span class="gene">GmTOE4a</span> and generated <span class="gene">GmTOE4a-overexpressing</span> lines in the cultivar Williams 82. The transgenic lines exhibited late flowering both under long day and short day conditions, and repressed the flowering-related genes, including <span class="gene">GmFT2a,</span> <span class="gene">GmFT5a,</span> <span class="gene">GmAP1,</span> and <span class="gene">GmLFY,</span> whereas the flowering repressors <span class="gene">GmFT4</span> and miR156 were upregulated. Interestingly, <span class="gene">GmTOE4a</span> was also mediated by photoperiod via maturity genes E3 and E4, which encode photoreceptors in soybean. Further, miR172-mediated <span class="gene">GmTOE4a,</span> which regulates flowering in soybean, is different in Arabidopsis in that it is dependent on the CONSTANS-like gene <span class="gene">GmCOL1a.</span> In addition to its effect on flowering time, <span class="gene">GmTOE4a</span> regulated plant morphology, increased stem thickness, and reduced plant height, internode length and leaf size, which are important agronomic traits that enhance the capacity to resist <span class="trait">lodging</span> and increase soybean <span class="trait">yield.</span> This is useful information to understand the molecular mechanism of flowering time and plant morphology in soybean and will greatly influence soybean <span class="trait">yield</span> improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of seed <span class="trait">yield</span> and agronomic characters by photoperiod sensitivity and growth habit genes in soybean.</h2>
			<h3>PMID:20012856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cober ER(1), Morrison MJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes are adapted to narrow bands of latitude due to photoperiod sensitivity. There are several photoperiod-sensitive loci (E1, E2, E3, E4, E5, E6, E7, E8). Determinate and indeterminate growth habits are controlled by a single locus. The objective of our research was to examine the effects of photoperiod sensitivity and growth habit alleles on seed <span class="trait">yield</span> and other agronomic characters using isogenic lines. Twenty 'Harosoy' isolines with 11 photoperiod-sensitive genotypes many with both indeterminate and determinate growth habits were grown in the field at Ottawa, ON, from 2003 to 2007. Maturity ranged from 97 to 127 days, and seed <span class="trait">yield</span> increased linearly with maturity until about 112 days when it plateaued. Determinate lines were always shorter than indeterminate lines of equivalent maturity. Seed <span class="trait">yield</span> was associated with plant height, maturity, seed sugar concentration, seed weight and <span class="trait">lodging.</span> Effects of alleles at individual loci, and additive and epistatic effects across multiple loci were examined. At a single locus, photoperiod-insensitive alleles produced isolines that matured 8-11 days earlier, yielded less, and had shorter plants with reduced <span class="trait">lodging.</span> In multiple loci analyses, additive effects explained most of the variation in agronomic characters since additive models with E1, E3, E4, E7 and <span class="gene">Dt1</span> loci included compared well to additive plus epistatic models and genotype-based models. Variation in photoperiod sensitivity and growth habit alleles results in a range of maturity, with pleiotropic effects on seed <span class="trait">yield</span> and agronomic characteristics, and play an important role in providing adaptation across latitudes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being seed oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of quantitative trait loci for plant height, <span class="trait">lodging,</span> and maturity in a soybean population segregating for growth habit.</h2>
			<h3>PMID:24166318</h3>
				

				
			<p>
				作者：Lee SH(1), Bailey MA, Mian MA, Shipe ER, Ashley DA, Parrott WA, Hussey RS, Boerma HR.
				<br/> 
				年份：1996 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The use of molecular markers to identify quantitative trait loci (QTLs) has the potential to enhance the efficiency of trait selection in plant breeding. The purpose of the present study was to identify additional QTLs for plant height, <span class="trait">lodging,</span> and maturity in a soybean, Glycine max (L.) Merr., population segregating for growth habit. In this study, 153 restriction fragment length polymorphisms (RFLP) and one morphological marker (Dt1) were used to identify QTLs associated with plant height, <span class="trait">lodging,</span> and maturity in 111 F2-derived lines from a cross of PI 97100 and 'Coker 237'. The F2-derived lines and two parents were grown at Athens, Ga., and Blackville, S.C., in 1994 and evaluated for phenotypic traits. The genetic linkage map of these 143 loci covered about 1600 cM and converged into 23 linkage groups. Eleven markers remained unlinked. Using interval-mapping analysis for linked markers and single-factor analysis of variance (ANOVA), loci were tested for association with phenotypic data taken at each location as well as mean values over the two locations. In the combined analysis over locations, the major locus associated with plant height was identified as <span class="gene">Dt1</span> on linkage group (LG) L. The <span class="gene">Dt1</span> locus was also associated with <span class="trait">lodging.</span> This locus explained 67.7% of the total variation for plant height, and 56.4% for <span class="trait">lodging.</span> In addition, two QTLs for plant height (K007 on LG H and A516b on LG N) and one QTL for <span class="trait">lodging</span> (cr517 on LG J) were identified. For maturity, two independent QTLs were identified in intervals between R051 and N100, and between B032 and CpTI, on LG K. These QTLs explained 31.2% and 26.2% of the total variation for maturity, respectively. The same QTLs were identified for all traits at each location. This consistency of QTLs may be related to a few QTLs with large effects conditioning plant height, <span class="trait">lodging,</span> and maturity in this population.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed <span class="trait">yield</span> and its components of indeterminate and determinate lines in recombinant inbred lines of soybean</h2>
			<h3>PMID:26069445</h3>
				
					<h3>PMCID:4430510</h3>
					<h4>期刊:Breeding Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kato </surname>  <given-names> Shin </given-names>  </name>  <xref ref-type="aff" rid="af1-65_154"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujii </surname>  <given-names> Kenichiro </given-names>  </name>  <xref ref-type="aff" rid="af2-65_154"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yumoto </surname>  <given-names> Setsuzo </given-names>  </name>  <xref ref-type="aff" rid="af3-65_154"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ishimoto </surname>  <given-names> Masao </given-names>  </name>  <xref ref-type="aff" rid="af2-65_154"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shiraiwa </surname>  <given-names> Tatsuhiko </given-names>  </name>  <xref ref-type="aff" rid="af4-65_154"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sayama </surname>  <given-names> Takashi </given-names>  </name>  <xref ref-type="aff" rid="af2-65_154"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kikuchi </surname>  <given-names> Akio </given-names>  </name>  <xref ref-type="aff" rid="af1-65_154"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishio </surname>  <given-names> Takeshi </given-names>  </name>  <xref rid="c1-65_154" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="af5-65_154"> 5 </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Stem growth habit is an important key character affecting <span class="trait">yield</span> in soybean. Based on stem growth habits, soybeans can be classified into indeterminate growth plants (IND), semi-determinate growth plants and determinate growth plants (DET), these categories having been demonstrated to be primarily controlled by <italic> <span class="gene">Dt1</span> </italic> and <italic> <span class="gene">Dt2</span> </italic> loci ( <xref rid="b1-65_154" ref-type="bibr"> Bernard 1972 </xref> ). The stem length and leaf formation in IND continue to grow for a long period after flowering. In DET, stem growth is terminated when flowering begins or soon afterward, and semi-determinate plants show stem termination intermediate between that of IND and that of DET ( <xref rid="b1-65_154" ref-type="bibr"> Bernard 1972 </xref> ). <italic> TERMINAL FLOWER1 </italic> ( <italic> <span class="gene">GmTFL1</span> </italic> ) on chromosome 19 ( <italic> <span class="gene">Glyma19g37890.1</span> </italic> ) has been identified as <italic> <span class="gene">Dt1</span> </italic> using transformation and virus-induced gene silencing approaches in previous reports ( <xref rid="b15-65_154" ref-type="bibr"> Liu <italic> et al. </italic> 2010 </xref> , <xref rid="b29-65_154" ref-type="bibr"> Tian <italic> et al. </italic> 2010 </xref> ). <italic> <span class="gene">GmTFL1</span> </italic> transcripts have been shown to accumulate in shoot apical meristems during early vegetative growth in both DET and IND, but <italic> <span class="gene">GmTFL1</span> </italic> transcripts are abruptly lost after flowering in DET, while remaining in IND ( <xref rid="b15-65_154" ref-type="bibr"> Liu <italic> et al. </italic> 2010 </xref> ). This difference results in a difference of main stem nodes and flowering periods between IND and DET.
					</p>
				
					<p>&nbsp;&nbsp;
						The evaluation of stem termination is sometimes difficult depending on cultivation condition (i.e., <span class="trait">lodging</span> and shading by neighboring lines). For judging stem termination precisely, we developed a DNA marker of <italic> <span class="gene">GmTFL1</span> </italic> for predicting stem termination beforehand and confirmed it by visual inspection. Because some alleles in the <italic> <span class="gene">GmTFL1</span> </italic> locus, i.e., <italic> <span class="gene">GmTFL1-a</span> </italic> ( <italic> <span class="gene">Dt1</span> </italic> ), <italic> <span class="gene">GmTFL1-b</span> </italic> ( <italic> <span class="gene">Dt1</span> </italic> ), <italic> <span class="gene">Gmtfl1-ta</span> </italic> ( <italic> dt1 </italic> ), <italic> <span class="gene">Gmtfl1-ab</span> </italic> ( <italic> dt1 </italic> ), <italic> <span class="gene">Gmtfl1-bb</span> </italic> ( <italic> dt1 </italic> ) and <italic> <span class="gene">Gmtfl1-tb</span> </italic> ( <italic> dt1 </italic> ), have been reported previously, we identified the alleles of <italic> <span class="gene">GmTFL1</span> </italic> of the parents of RILs by sequencing the <italic> <span class="gene">GmTFL1</span> </italic> locus from &#x02212;740 to +1440 (nucleotide positions from the start codon) ( <xref rid="b29-65_154" ref-type="bibr"> Tian <italic> et al. </italic> 2010 </xref> ). There were four sequence polymorphisms between the DET cultivars, i.e., &#x02018;Ohsuzu&#x02019;, &#x02018;Tachingaha&#x02019;, and the IND cultivars, i.e., &#x02018;Athow&#x02019; and &#x02018;Stressland&#x02019;. These cultivars were classified into two groups according to the sequence polymorphism of <italic> <span class="gene">GmTFL1</span> </italic> . The <italic> <span class="gene">GmTFL1</span> </italic> sequences of &#x02018;Athow&#x02019; and &#x02018;Stressland&#x02019; were completely the same as the sequence of <italic> <span class="gene">GmTFL1-a</span> </italic> ( <italic> <span class="gene">Dt1</span> </italic> ) ( <xref rid="b29-65_154" ref-type="bibr"> Tian <italic> et al. </italic> 2010 </xref> ). &#x02018;Ohsuzu&#x02019; and &#x02018;Tachinagaha&#x02019; had AAAAA[T]TTCTT in 5&#x02032; UTR (&#x02212;499), TATGA[G]CAGCG in 5&#x02032; UTR (&#x02212;410), AAA[GAGAAGATTTTTTT]TTG in the 1 <sup> st </sup> intron (+279&#x02013;+292) and CACAG[T]GGGAA in the 4 <sup> th </sup> exon (+1283), which were completely the same as the sequence of <italic> <span class="gene">GmTFL1-ab</span> </italic> ( <italic> dt1 </italic> ) ( <xref rid="b29-65_154" ref-type="bibr"> Tian <italic> et al. </italic> 2010 </xref> ). In the 1 <sup> st </sup> intron (+279&#x02013;+292), a length polymorphism of 6 bp was identified ( <xref rid="f1-65_154" ref-type="fig"> Fig. 1A </xref> ), as shown in a previous study ( <xref rid="b29-65_154" ref-type="bibr"> Tian <italic> et al. </italic> 2010 </xref> ). The <italic> <span class="gene">GmTFL1</span> </italic> genotypes of the parents and the lines in OA-RILs and ST-RILs were analyzed by distinguishing the 6-bp length polymorphism in the 1 <sup> st </sup> intron within PCR products amplified by a primer pair of Dt1Int ( <xref rid="f1-65_154" ref-type="fig"> Fig. 1B </xref> ). The PCR products of &#x02018;Ohsuzu&#x02019; and &#x02018;Tachinagaha&#x02019; ( <italic> dt1 </italic> genotype) produced bands of 123 bp, while the PCR products of &#x02018;Athow&#x02019; and &#x02018;Stressland&#x02019; ( <italic> <span class="gene">Dt1</span> </italic> genotype) produced bands of 129 bp ( <xref rid="f1-65_154" ref-type="fig"> Fig. 1B </xref> ). All the lines showing 129-bp bands were IND as &#x02018;Athow&#x02019; and &#x02018;Stressland&#x02019;, whereas all the lines showing 123-bp bands were DET as &#x02018;Ohsuzu&#x02019; and &#x02018;Tachinagaha&#x02019; ( <xref rid="f1-65_154" ref-type="fig"> Fig. 1B </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>