<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>main stem nodes</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>main stem nodes</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of <span class="trait">nodes</span> on <span class="trait">main</span> <span class="trait">stem,</span> <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being seed oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and <span class="trait">nodes</span> on the <span class="trait">main</span> <span class="trait">stem,</span> respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Validation of internal control for gene expression study in soybean by quantitative real-time PCR</h2>
			<h3>PMID:18573215</h3>
				

				
					<h3>PMCID:2443375</h3>
					<h4>期刊:BMC Molecular Biology</h4>
				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Jian </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jianbo1007@yahoo.com </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Bin </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> bin.liu@bio.ntnu.no </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Bi </surname>  <given-names> Yurong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yrbi@lzu.edu.cn </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Hou </surname>  <given-names> Wensheng </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> houwsh@caas.net.cn </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> wucx@mail.caas.net.cn </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hantf@mail.caas.net.cn </email>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 6 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Normalizing to housekeeping gene (HKG) can make results from quantitative real-time PCR (qRT-PCR) more reliable. Recent studies have shown that no single HKG is universal for all experiments. Thus, a suitable HKG should be selected before its use. Only a few studies on HKGs have been done in plants, and none in soybean, an economically important crop. Therefore, the present study was conducted to identify suitable HKG(s) for normalization of gene expression in soybean. RESULTS: All ten HKGs displayed a wide range of Ct values in 21 sample pools, confirming that they were variably expressed. GeNorm was used to determine the expression stability of the HGKs in seven series sets. For all the sample pools analyzed, the stability rank was ELF1B, CYP2 &gt; ACT11 &gt; TUA &gt; ELF1A &gt; UBC2 &gt; ACT2/7 &gt; <span class="gene">TUB</span> &gt; G6PD &gt; UBQ10. For different tissues under the same developmental stage, the rank was ELF1B, CYP2 &gt; ACT2/7 &gt; UBC2 &gt; TUA &gt; ELF1A &gt; ACT11 &gt; <span class="gene">TUB</span> &gt; G6PD &gt; UBQ10. For the developmental stage series, the stability rank was ACT2/7, TUA &gt; ELF1A &gt; UBC2 &gt; ELF1B &gt; <span class="gene">TUB</span> &gt; CYP2 &gt; ACT11 &gt; G6PD &gt; UBQ10. For photoperiodic treatments, the rank was ACT11, ELF1B &gt; CYP2 &gt; TUA &gt; ELF1A &gt; UBC2 &gt; ACT2/7 &gt; <span class="gene">TUB</span> &gt; G6PD &gt; UBQ10. For different times of the day, the rank was ELF1A, TUA &gt; ELF1B &gt; G6PD &gt; CYP2 &gt; ACT11 &gt; ACT2/7 &gt; <span class="gene">TUB</span> &gt; UBC2 &gt; UBQ10. For different cultivars and leaves on different <span class="trait">nodes</span> of the <span class="trait">main</span> <span class="trait">stem,</span> the ten HKGs' stability did not differ significantly. Delta Ct approach and 'Stability index' were also used to analyze the expression stability in all 21 sample pools. Results from Delta Ct approach and geNorm indicated that ELF1B and CYP2 were the most stable HKGs, and UBQ10 and G6PD the most variable ones. Results from 'Stability index' analysis were different, with ACT11 and CYP2 being the most stable HKGs, and ELF1A and TUA the most variable ones. CONCLUSION: Our data suggests that HKGs are expressed variably in soybean. Based on the results from geNorm and Delta Ct analysis, ELF1B and CYP2 could be used as internal controls to normalize gene expression in soybean, while UBQ10 and G6PD should be avoided. To achieve accurate results, some conditions may require more than one HKG to be used for normalization.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						All ten HKGs displayed a wide range of Ct values in 21 sample pools, confirming that they were variably expressed. GeNorm was used to determine the expression stability of the HGKs in seven series sets. For all the sample pools analyzed, the stability rank was <italic> ELF1B </italic> , <italic> CYP2 </italic> &#x0003e; <italic> ACT11 </italic> &#x0003e; <italic> TUA </italic> &#x0003e; <italic> ELF1A </italic> &#x0003e; <italic> UBC2 </italic> &#x0003e; <italic> ACT2/7 </italic> &#x0003e; <italic> <span class="gene">TUB</span> </italic> &#x0003e; <italic> G6PD </italic> &#x0003e; <italic> UBQ10 </italic> . For different tissues under the same developmental stage, the rank was <italic> ELF1B </italic> , <italic> CYP2 </italic> &#x0003e; <italic> ACT2/7 </italic> &#x0003e; <italic> UBC2 </italic> &#x0003e; <italic> TUA </italic> &#x0003e; <italic> ELF1A </italic> &#x0003e; <italic> ACT11 </italic> &#x0003e; <italic> <span class="gene">TUB</span> </italic> &#x0003e; <italic> G6PD </italic> &#x0003e; <italic> UBQ10 </italic> . For the developmental stage series, the stability rank was <italic> ACT2/7 </italic> , <italic> TUA </italic> &#x0003e; <italic> ELF1A </italic> &#x0003e; <italic> UBC2 </italic> &#x0003e; <italic> ELF1B </italic> &#x0003e; <italic> <span class="gene">TUB</span> </italic> &#x0003e; <italic> CYP2 </italic> &#x0003e; <italic> ACT11 </italic> &#x0003e; <italic> G6PD </italic> &#x0003e; <italic> UBQ10 </italic> . For photoperiodic treatments, the rank was <italic> ACT11 </italic> , <italic> ELF1B </italic> &#x0003e; <italic> CYP2 </italic> &#x0003e; <italic> TUA </italic> &#x0003e; <italic> ELF1A </italic> &#x0003e; <italic> UBC2 </italic> &#x0003e; <italic> ACT2/7 </italic> &#x0003e; <italic> <span class="gene">TUB</span> </italic> &#x0003e; <italic> G6PD </italic> &#x0003e; <italic> UBQ10 </italic> . For different times of the day, the rank was <italic> ELF1A </italic> , <italic> TUA </italic> &#x0003e; <italic> ELF1B </italic> &#x0003e; <italic> G6PD </italic> &#x0003e; <italic> CYP2 </italic> &#x0003e; <italic> ACT11 </italic> &#x0003e; <italic> ACT2/7 </italic> &#x0003e; <italic> <span class="gene">TUB</span> </italic> &#x0003e; <italic> UBC2 </italic> &#x0003e; <italic> UBQ10 </italic> . For different cultivars and leaves on different <span class="trait">nodes</span> of the <span class="trait">main</span> <span class="trait">stem,</span> the ten HKGs' stability did not differ significantly. &#x00394;Ct approach and 'Stability index' were also used to analyze the expression stability in all 21 sample pools. Results from &#x00394;Ct approach and geNorm indicated that <italic> ELF1B </italic> and <italic> CYP2 </italic> were the most stable HKGs, and <italic> UBQ10 </italic> and <italic> G6PD </italic> the most variable ones. Results from 'Stability index' analysis were different, with <italic> ACT11 </italic> and <italic> CYP2 </italic> being the most stable HKGs, and <italic> ELF1A </italic> and <italic> TUA </italic> the most variable ones.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>