<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Matsumuraeses phaseoli</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Matsumuraeses phaseoli</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Diversity of sporadic symbionts and nonsymbiotic endophytic bacteria isolated from nodules of woody, shrub, and food legumes in Ethiopia.</h2>
			<h3>PMID:24196581  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Aserse AA(1), Räsänen LA, Aseffa F, Hailemariam A, Lindström K.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fifty-five bacterial isolates were obtained from surface-sterilized nodules of woody and shrub legumes growing in Ethiopia: Crotalaria spp., Indigofera spp., and Erythrina brucei, and the food legumes soybean and common bean. Based on partial 16S rRNA gene sequence analysis, the majority of the isolates were identified as Gram-negative bacteria belonging to the genera Achromobacter, Agrobacterium, Burkholderia, Cronobacter, Enterobacter, Mesorhizobium, Novosphingobium, Pantoea, Pseudomonas, Rahnella, Rhizobium, Serratia, and Variovorax. Seven isolates were Gram-positive bacteria belonging to the genera Bacillus, Paenibacillus, Planomicrobium, and Rhodococcus. Amplified fragment length polymorphism (AFLP) fingerprinting showed that each strain was genetically distinct. According to phylogenetic analysis of recA, glnII, rpoB, and 16S rRNA gene sequences, Rhizobium, Mesorhizobium, and Agrobacterium were further classified into six different genospecies: Agrobacterium spp., Agrobacterium radiobacter, Rhizobium sp., Rhizobium <span class="trait">phaseoli,</span> Mesorhizobium sp., and putative new Rhizobium species. The strains from R. <span class="trait">phaseoli,</span> Rhizobium sp. IAR30, and Mesorhizobium sp. ERR6 induced nodules on their host plants. The other strains did not form nodules on their original host. Nine endophytic bacterial strains representing seven genera, Agrobacterium, Burkholderia, Paenibacillus, Pantoea, Pseudomonas, Rhizobium, and Serratia, were found to colonize nodules of Crotalaria incana and common bean on co-inoculation with symbiotic rhizobia. Four endophytic Rhizobium and two Agrobacterium strains had identical <span class="gene">nifH</span> gene sequences with symbiotic Rhizobium strains, suggesting horizontal gene transfer. Most symbiotic and nonsymbiotic endophytic bacteria showed plant growth-promoting properties in vitro, which indicate their potential role in the promotion of plant growth when colonizing plant roots and the rhizosphere.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Different species and symbiotic genotypes of field rhizobia can nodulate Phaseolus vulgaris in Tunisian soils.</h2>
			<h3>PMID:19709241</h3>
				

				
			<p>
				作者：Mhamdi R(1), Laguerre G, Aouani ME, Mars M, Amarger N.
				<br/> 
				年份：2002 Jul 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Abstract A collection of 160 isolates of rhizobia nodulating Phaseolus vulgaris in three geographical regions in Tunisia was characterized by restriction fragment length polymorphism analysis of polymerase chain reaction (PCR)-amplified 16S rDNA, <span class="gene">nifH</span> and <span class="gene">nodC</span> genes. Nine groups of rhizobia were delineated: Rhizobium gallicum biovar (bv.) gallicum, Rhizobium leguminosarum bv. <span class="trait">phaseoli</span> and bv. viciae, Rhizobium etli bv. <span class="trait">phaseoli,</span> Rhizobium giardinii bv. giardinii, and four groups related to species of the genus Sinorhizobium, Sinorhizobium meliloti, Sinorhizobium medicae and Sinorhizobium fredii. The most abundant rhizobial species were R. gallicum, R. etli, and R. leguminosarum encompassing 29-20% of the isolates each. Among the isolates assigned to R. leguminosarum, two-thirds were ineffective in nitrogen fixation with P. vulgaris and harbored a symbiotic gene typical of the biovar viciae. The S. fredii-like isolates did not nodulate soybean plants but formed numerous effective nodules on P. vulgaris. Comparison of <span class="gene">nodC</span> gene sequences showed that their symbiotic genotype was not related to that of S. fredii, but to that of the S. fredii-like reference strain GR-06, which was isolated from a bean plant grown in a Spanish soil. An additional genotype including 16% of isolates was found to be closely related to species of the genus Agrobacterium. However, when re-examined, these isolates did not nodulate their original host.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The conjugative plasmid of a bean-nodulating <italic> Sinorhizobium fredii </italic> strain is assembled from sequences of two <italic> Rhizobium </italic> plasmids and the chromosome of a <italic> Sinorhizobium </italic> strain</h2>
			<h3>PMID:21702991</h3>
				
					<h3>PMCID:3224233</h3>
					<h4>期刊:BMC Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Cervantes </surname>  <given-names> Laura </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lcervant@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Bustos </surname>  <given-names> Patricia </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> paty@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Girard </surname>  <given-names> Lourdes </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> girard@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Santamar&#x000ed;a </surname>  <given-names> Rosa Isela </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> rosa@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> D&#x000e1;vila </surname>  <given-names> Guillermo </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> davila@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Vinuesa </surname>  <given-names> Pablo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> vinuesa@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Romero </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dromero@ccg.unam.mx </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Brom </surname>  <given-names> Susana </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sbrom@ccg.unam.mx </email>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 6 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Conjugative transfer is thought to be the most relevant mechanism that contributes to the dissemination and diversification of genetic information, particularly that localized on plasmids. Conjugation systems are constituted by a DNA transfer and replication (Dtr) component, encoded by <italic> tra </italic> genes and a <italic> cis </italic> -acting <italic> oriT </italic> site, and a mating pair formation (Mpf) component, encoded by <italic> trb </italic> genes [ <xref ref-type="bibr" rid="B3"> 3 </xref> ]. Information on the conjugative transfer mechanisms of rhizobial plasmids is still scarce. However, two groups of plasmids containing a functional set of conjugative transfer genes have been described: Type I, which are regulated by quorum-sensing [ <xref ref-type="bibr" rid="B3"> 3 </xref> - <xref ref-type="bibr" rid="B5"> 5 </xref> ], and Type II, which have permanently RctA-repressed transfer genes [ <xref ref-type="bibr" rid="B3"> 3 </xref> , <xref ref-type="bibr" rid="B6"> 6 </xref> ]. A third type of conjugative plasmids has been recently proposed, represented by the largest plasmids of <italic> R. leguminosarum </italic> bv viciae strains [ <xref ref-type="bibr" rid="B3"> 3 </xref> ]. Some plasmids are mobilizable in the presence of transmissible plasmids, either by cointegration (conduction) [ <xref ref-type="bibr" rid="B7"> 7 </xref> ], or by classical (trans) helper mechanisms [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. Specifically in the bean nodulating type strain <italic> Rhizobium etli </italic> CFN42, we have previously shown that it contains a quorum-sensing regulated self-transmissible plasmid (pRet42a) [ <xref ref-type="bibr" rid="B5"> 5 </xref> ], and that transfer of the symbiotic plasmid (pRet42d) occurs only in the presence of pRet42a. The event requires cointegration of both replicons. This may be achieved through IntA-dependent site-specific recombination between <italic> attA </italic> and <italic> attD </italic> sites, or through RecA-dependent homologous recombination among large sequence segments shared between the replicons. The cointegrate is able to transfer, using the pRet42a-encoded machinery. In the transconjugants, the cointegrate is usually resolved to regenerate the wild-type plasmids, but in a few cases, resolution of the cointegrate leads to the formation of recombinant plasmids that contain segments of each plasmid, pRet42a and pRet42d [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. Mesoamerica has been identified as the place of origin of bean plants and <italic> Rhizobium etli </italic> bacteria [ <xref ref-type="bibr" rid="B10"> 10 </xref> ], while soybean and its nodulating bacteria ( <italic> Sinorhizobium fredii </italic> ) originated in East Asia [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. In the early XVI <sup> th </sup> century, common beans and their symbionts were transported to Europe and other parts of the world. A survey of bean-nodulating strains in Granada, Spain, showed the presence of strains belonging to five different species: <italic> R. etli </italic> , <italic> R. gallicum </italic> , <italic> R. giardinii </italic> , <italic> R. leguminosarum </italic> and <italic> S. fredii </italic> [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. The usual host of <italic> Sinorhizobium fredii </italic> strains is soybean ( <italic> Glycine soja </italic> ), not common bean ( <italic> Phaseolus vulgaris </italic> ). Nevertheless, the bean-nodulating strains classified as <italic> S. fredii </italic> , were unable to nodulate cvs. Williams or Peking of <italic> Glycine max </italic> . Hybridization of digested genomic DNA with <italic> <span class="gene">nodB</span> </italic> and <italic> <span class="gene">nifH</span> </italic> genes from <italic> R. etli </italic> , showed a very weak signal [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. <italic> R. etli </italic> bv <span class="trait">phaseoli</span> symbiotic plasmids (pSyms) are characterized by the presence of three copies of <italic> <span class="gene">nifH</span> </italic> . The bean-nodulating <italic> S. fredii </italic> strains showed only one copy of this gene [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. While conjugative transfer may explain the acquisition of new symbiotic features by strains belonging to diverse species, the relationship between <italic> R. etli </italic> and bean-nodulating <italic> S. fredii </italic> is not so easily established. In order to gain further insight into the mechanisms and pathways leading to the generation of new rhizobial strains, in this work we present the analysis of the bean-nodulating <italic> S. fredii </italic> strain GR64, isolated from the soil in Granada. The results indicate that the plasmids present in GR64 likely derived from conjugative transfer and rearrangement events among sequences localized in at least three different replicons, including two different plasmids and a chromosome.
					</p>
				
					<p>&nbsp;&nbsp;
						A derivative of GR64 carrying a Tn <italic> 5mob </italic> -labeled pSym was constructed by mating GR64 with strain S-17/pSUP5011 and selecting for <span class="trait">resistance</span> to Nal and Nm. Tagged plasmids were mobilized to <italic> A. tumefaciens </italic> <span class="gene">GMI9023</span> [ <xref ref-type="bibr" rid="B35"> 35 </xref> ] in triparental crosses, using pRK2013 [ <xref ref-type="bibr" rid="B36"> 36 </xref> ] as helper, and selecting for Rif <sup> R </sup> Nm <sup> R </sup> transconjugants. Transconjugants carrying the tagged pSym (pSfr64b) were identified using Eckhardt type gels.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture and evolution of the mating type locus in fusaria that cause soybean sudden death syndrome and bean root rot.</h2>
			<h3>PMID:24891421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hughes TJ(1), O'Donnell K(2), Sink S(3), Rooney AP(4), Scandiani MM(5), Luque A(5), Bhattacharyya MK(6), Huang X(7).
				<br/> 
				年份：2014 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium tucumaniae is the only known sexually reproducing species among the seven closely related fusaria that cause soybean sudden death syndrome (SDS) or bean root rot (BRR). In a previous study, laboratory mating of F. tucumaniae yielded recombinant ascospore progeny but required two mating-compatible strains, indicating that it is heterothallic. To assess the reproductive mode of the other SDS and BRR fusaria, and their potential for mating, whole-genome sequences of two SDS and one BRR pathogen were analyzed to characterize their mating type (MAT) loci. This bioinformatic approach identified a <span class="gene">MAT1-1</span> idiomorph in F. virguliforme NRRL 22292 and MAT1-2 idiomorphs in F. tucumaniae NRRL 34546 and F. azukicola NRRL 54364. Alignments of the MAT loci were used to design PCR primers within the conserved regions of the flanking genes <span class="gene">APN1</span> and SLA2, which enabled primer walking to obtain nearly complete sequences of the MAT region for six <span class="gene">MAT1-1</span> and five MAT1-2 SDS/BRR fusaria. As expected, sequences of the highly divergent 4.7 kb <span class="gene">MAT1-1</span> and 3.7 kb MAT1-2 idiomorphs were unalignable. However, sequences of the respective idiomorphs and those that flank <span class="gene">MAT1-1</span> and MAT1-2 were highly conserved. In addition to three genes at <span class="gene">MAT1-1</span> (MAT1-1-1, MAT1-1-2, MAT1-1-3) and two at MAT1-2 (MAT1-2-1, MAT1-2-3), the MAT loci of the SDS/BRR fusaria also include a putative gene predicted to encode for a 252 amino acid protein of unknown function. Alignments of the MAT1-1-3 and MAT1-2-1 sequences were used to design a multiplex PCR assay for the MAT loci. This assay was used to screen DNA from 439 SDS/BRR isolates, which revealed that each isolate possessed <span class="gene">MAT1-1</span> or MAT1-2, consistent with heterothallism. Both idiomorphs were represented among isolates of F. azukicola, F. brasiliense, F. <span class="trait">phaseoli</span> and F. tucumaniae, whereas isolates of F. virguliforme and F. cuneirostrum were only <span class="gene">MAT1-1</span> and F. crassistipitatum were only MAT1-2. Finally, nucleotide sequence data from the <span class="gene">RPB1</span> and <span class="gene">RPB2</span> genes were used to date the origin of the SDS/BRR group, which was estimated to have occurred about 0.75 Mya (95% HPD interval: 0.27, 1.68) in the mid-Pleistocene, long before the domestication of the common bean or soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association of Effector <span class="gene">Six6</span> with Vascular Wilt Symptoms Caused by Fusarium oxysporum on Soybean.</h2>
			<h3>PMID:27349740  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lanubile A(1), Ellis ML(1), Marocco A(1), Munkvold GP(1).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Fusarium oxysporum species complex (FOSC) is a widely distributed group of fungi that includes both pathogenic and nonpathogenic isolates. In a previous study, isolates within the FOSC collected primarily from soybean were assessed for the presence of 12 fungal effector genes. Although none of the assayed genes was significantly associated with wilt symptoms on soybean, the secreted in xylem 6 (Six6) gene was present only in three isolates, which all produced high levels of vascular wilt on soybean. In the current study, a collection of F. oxysporum isolates from soybean roots and F. oxysporum f. sp. <span class="trait">phaseoli</span> isolates from common bean was screened for the presence of the <span class="gene">Six6</span> gene. Interestingly, all isolates for which the <span class="gene">Six6</span> amplicon was generated caused wilt symptoms on soybean, and two-thirds of the isolates showed high levels of aggressiveness, indicating a positive association between the presence of the effector gene <span class="gene">Six6</span> and induction of wilt symptoms. The expression profile of the <span class="gene">Six6</span> gene analyzed by quantitative reverse-transcription polymerase chain reaction revealed an enhanced expression for the isolates that caused more severe wilt symptoms on soybean, as established by the greenhouse assay. These findings suggest the suitability of the <span class="gene">Six6</span> gene as a possible locus for pathogenicity-based molecular diagnostics across the various formae speciales.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>