<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Meloidogyne javanica</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Meloidogyne javanica</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Size of AT <sub> ( <italic> n </italic> ) </sub> Insertions in Promoter Region Modulates <italic> <span class="gene">Gmhsp17.6-L</span> </italic> mRNA Transcript Levels</h2>
			<h3>PMID:20300590</h3>
				

				
					<h3>PMCID:2837908</h3>
					<h4>期刊:Journal of Biomedicine and Biotechnology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fuganti </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="I1">  <sup> 1, 2 </sup>  </xref>  <xref ref-type="aff" rid="I2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> da Silva Machado </surname>  <given-names> Maria de F&#x000e1;tima Pires </given-names>  </name>  <xref ref-type="aff" rid="I1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lopes </surname>  <given-names> Val&#x000e9;ria Stefania </given-names>  </name>  <xref ref-type="aff" rid="I3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silva </surname>  <given-names> Jo&#x000e3;o Fl&#x000e1;vio Veloso </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arias </surname>  <given-names> Carlos Alberto Arrabal </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marin </surname>  <given-names> Silvana Regina Rockenbach </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Binneck </surname>  <given-names> Eliseu </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo Vilela </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino </surname>  <given-names> Francismar C&#x000f4;rrea </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="I2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 12 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						During earlier experiments, an SSR molecular marker (176 Soy HSP) showing high correlation (70%) with resistance/susceptibility to javanese root-knot nematode <span class="trait">Meloidogyne</span> <span class="trait">javanica</span> was identified in soybean. After being sequenced, results indicated that the SSR 176 Soy HSP marker was inserted in the promoter region of <span class="gene">Gmhsp17.6-L</span> gene. It was also detected in this region that resistant genotypes presented insertions between AT(31) and AT(33) in size and susceptible genotypes, AT(9). <span class="gene">Gmhsp17.6-L</span> gene coding region presented a perfect match in amino acid sequence in all soybean genotypes. A ribonuclease protection assay showed that <span class="gene">Gmhsp17.6-L</span> gene mRNA transcripts were present in all genotypes. A real-time relative quantification (qPCR) indicated in the resistant individuals higher mRNA transcripts levels, which presented in the sequencing more AT(n) insertions. These results suggest that the number of AT(n) insertions inside this promoter region could modulate up or down gene levels. Those findings can lead to the possibility of manipulating, between some limits, the mRNA transcripts levels using different sizes of AT(n) insertions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Based on these previous results obtained by Fuganti and collaborators [ <xref ref-type="bibr" rid="B5"> 5 </xref> ], the 176 Soy HSP microsatellite molecular marker was selected to be further studied in soybean plants, resistant and susceptible to <italic> M. <span class="trait">javanica</span> </italic> when inoculated and noninoculated with the pathogen. The resulting sequence for this molecular marker showed high similarity with a promoter region for a gene encoding a small heat-shock-protein (sHSP) found in soybean, <italic> <span class="gene">Gmhsp17.6-L</span> </italic> (Genbank accession no. M11317).
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, considering the biological function of these AT-rich elements in heat-shock promoters and the significant differences found in this region, between the genotypes tested, our objective was to analyze <italic> <span class="gene">Gmhsp17.6-L</span> </italic> (Genbank accession no. M11317) mRNA expression level, using the ribonuclease protection assay (RPA) and real-time quantitative PCR (RT-qPCR), in an attempt to elucidate how genetic differences the samples may influence in the <span class="trait">resistance</span> and/or susceptibility responses in plants to the javanese root-knot nematode, <italic> M. <span class="trait">javanica.</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Since heat shock proteins have been known to be involved in responses to stress, including biotic stresses such as nematode infection, it seemed reasonable to assume that genetic differences in the promoter of <italic> <span class="gene">Gmhsp17.6-L</span> </italic> may have an impact on nematode <span class="trait">resistance</span> associated with this SSR. As a result, a set of primers (pSoyHSP_F 5&#x02032;GGG&#x02009;&#x02009;CTG&#x02009;&#x02009;CAG&#x02009;&#x02009;GAA&#x02009;&#x02009;TTC&#x02009;&#x02009;TGA&#x02009;&#x02009;AAT&#x02009;&#x02009;TGG&#x02009;&#x02009;GTC&#x02009;&#x02009;TTT&#x02009;&#x02009;TTG3&#x02032;and SoyHSPCL_R 5&#x02032;CCC&#x02009;&#x02009;CCC&#x02009;&#x02009;GGG&#x02009;&#x02009;TTA&#x02009;&#x02009;ACC&#x02009;&#x02009;AGA&#x02009;&#x02009;GAT&#x02009;&#x02009;TTC&#x02009;&#x02009;TAT&#x02009;&#x02009;AGC&#x02009;&#x02009;CT3&#x02032;), with sites for restriction enzymes to facilitate the cloning process, was designed to amplify, clone and sequence the entire <italic> <span class="gene">Gmhsp17.6-L</span> </italic> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the significant sequence differences detected among resistant and susceptible soybean genotypes, an RPA was performed with the objective of detecting whether these differences caused inactivation of <italic> <span class="gene">Gmhsp17.6-L</span> </italic> expression in some genotypes. Thus, total RNA was extracted using Trizol reagent (Life Technologies) from resistant PI595099 and susceptible BRS133 parental genotypes. Half of the samples were treated with <italic> M. <span class="trait">javanica</span> </italic> eggs (designated &#x0201c;inoculated&#x0201d;) and the other half was not (designated &#x0201c;noninoculated&#x0201d;).
					</p>
				
					<p>&nbsp;&nbsp;
						Also, it was observed that when susceptible-population sample 256-S, which presented the smallest AT <sub> (n) </sub> insertion on the promoter region, was used as calibrator (value 1X), all resistant individuals presented higher gene-transcript levels, even when non-inoculated, which implies an evolutionary mechanism in nematode <span class="trait">resistance;</span> even in the absence of the pathogen, resistant genotypes express higher level of <italic> <span class="gene">Gmhsp17.6-L</span> </italic> mRNA transcripts. Similarly, the highest level of expression of gene <italic> <span class="gene">Gmhsp17.6-L</span> </italic> was detected in resistant individual JF7002, which also showed the longest AT <sub> (33) </sub> repetition, although, in general, all resistant samples showed high <italic> <span class="gene">Gmhsp17.6-L</span> </italic> transcript levels.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the <italic> Hsp </italic> 20 gene family in soybean: comprehensive sequence, genomic organization and expression profile analysis under abiotic and biotic stresses</h2>
			<h3>PMID:23985061</h3>
				

				
					<h3>PMCID:3852298</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Lopes-Caitar </surname>  <given-names> Val&#x000e9;ria S </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> valopes@cnpso.embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> de Carvalho </surname>  <given-names> Mayra CCG </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mayra@uenp.edu.br </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Darben </surname>  <given-names> Luana M </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> luana.darben@cnpso.embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kuwahara </surname>  <given-names> Marcia K </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> marcia.kuwahara@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre L </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> alexandre.nepomuceno@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Dias </surname>  <given-names> Waldir P </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> waldir.dias@embrapa.br </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo V </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ricardo.abdelnoor@embrapa.br </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar C </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> francismar.marcelino@embrapa.br </email>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The Hsp20 genes are associated with stress caused by HS and other abiotic factors, but have recently been found to be associated with the response to biotic stresses. These genes represent the most abundant class among the HSPs in plants, but little is known about this gene family in soybean. Because of their apparent multifunctionality, these proteins are promising targets for developing crop varieties that are better adapted to biotic and abiotic stresses. Thus, in the present study an in silico identification of <span class="gene">GmHsp20</span> gene family members was performed, and the genes were characterized and subjected to in vivo expression analysis under biotic and abiotic stresses. RESULTS: A search of the available soybean genome databases revealed 51 gene models as potential <span class="gene">GmHsp20</span> candidates. The 51 <span class="gene">GmHsp20</span> genes were distributed across a total of 15 subfamilies where a specific predicted secondary structure was identified. Based on in vivo analysis, only 47 soybean Hsp20 genes were responsive to heat shock stress. Among the <span class="gene">GmHsp20</span> genes that were potentials HSR, five were also cold-induced, and another five, in addition to one <span class="gene">GmAcd</span> gene, were responsive to <span class="trait">Meloidogyne</span> <span class="trait">javanica</span> infection. Furthermore, one predicted <span class="gene">GmHsp20</span> was shown to be responsive only to nematode infection; no expression change was detected under other stress conditions. Some of the biotic stress-responsive <span class="gene">GmHsp20</span> genes exhibited a divergent expression pattern between resistant and susceptible soybean genotypes under M. <span class="trait">javanica</span> infection. The putative regulatory elements presenting some conservation level in the <span class="gene">GmHsp20</span> promoters included HSE, W-box, CAAT box, and TA-rich elements. Some of these putative elements showed a unique occurrence pattern among genes responsive to nematode infection. CONCLUSIONS: The evolution of Hsp20 family in soybean genome has most likely involved a total of 23 gene duplications. The obtained expression profiles revealed that the majority of the 51 <span class="gene">GmHsp20</span> candidates are induced under HT, but other members of this family could also be involved in normal cellular functions, unrelated to HT. Some of the <span class="gene">GmHsp20</span> genes might be specialized to respond to nematode stress, and the predicted promoter structure of these genes seems to have a particular conserved pattern related to their biological function.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Considering the concept of parsimony, the conservation of this pattern of <italic> Hsp </italic> 20 gene duplication within the same chromosome observed in the genomes of rice, <italic> Arabidopsis </italic> and soybean most likely originated through segmental duplication events that occurred before the divergence of monocots and dicots, in the ancestral species, followed by chromosomal duplications in both the ancestral species and within each species [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B42"> 42 </xref> ]. Still, it is notable that three of the six Gm <italic> Hsp </italic> 20 genes that are responsive to <italic> M. <span class="trait">javanica</span> </italic> and <italic> H. glycines </italic> infection (Gm <italic> Hsp </italic> 17.4A, Gm <italic> Hsp </italic> 17.9B [ <xref ref-type="bibr" rid="B6"> 6 </xref> ] and Gm <italic> Hsp </italic> 17.6B [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]) are organized in blocks of segmental duplications in the genome. In soybean, expansion of the segmental gene families associated with the basal <span class="trait">resistance</span> response is common and has been observed in families including NBS-LRR, F-box and auxin-responsive genes [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Such duplications may contribute to the diversification of relevant alleles during plant-pathogen interactions or to the maintenance of similar levels of gene expression within the block, as observed in rice [ <xref ref-type="bibr" rid="B30"> 30 </xref> , <xref ref-type="bibr" rid="B43"> 43 </xref> ]. However, unlike the results reported by Ouyang et al. [ <xref ref-type="bibr" rid="B30"> 30 </xref> ], the expression pattern of the tandem duplicated genes, under the stress conditions tested here, was observed to be highly heterogeneous for <span class="gene">GmHsp20.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In <italic> Arabidopsis </italic> mutants for <italic> Hsp </italic> 20 genes involved in the responses to nematode infection, the TATAbox element should be preferentially located between 12 and 21 bp upstream of the transcription start site, followed by an HSE at around &#x02212;83 bp and a CCAAT box between 84 and 141 bp upstream of the transcription start site [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. The promoter of one <italic> Hsp </italic> 20, a CAAT box element was previously reported in the promoter region of <span class="gene">Hs1</span> pro-1, a gene conferring complete <span class="trait">resistance</span> to <italic> H. glycines </italic> , and appears to be essential for site-specific regulation [ <xref ref-type="bibr" rid="B29"> 29 </xref> ]. The promoters of all <italic> Hsp </italic> 20, which are responsive to nematode infection, also show putative CAAT elements. Moreover, the Gm <italic> Hsp </italic> 20 biotic stress-responsive genes followed the same pattern observed in <italic> Arabidopsis </italic> and sunflower and not observed in the others <span class="gene">Gmhsp20,</span> where the CAAT box always occurs either between the HSEs or immediately upstream of them, while the W-box, when present, is further upstream of the HSE. However, previous studies have shown the function of these cis-elements in the <italic> Hsp </italic> 20 regulation in <italic> Arabidopsis </italic> and sunflower, the involvement of them in soybean responses to nematodes need to be checked by <italic> in vivo </italic> experiments [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						This study makes a relevant contribution by identifying 51 potential genes that we suggest compose the soybean <italic> Hsp </italic> 20 gene family. The combination of <italic> in silico </italic> prediction strategies and <italic> in vivo </italic> expression analyses showed that the applied bioinformatic tools were very efficient in precisely identifying Gm <italic> Hsp </italic> 20 family members. In addition, the Gm <italic> Hsp </italic> 20 genes were divided among 13 of the 16 known plant <italic> Hsp </italic> 20 subfamilies and two additional unknown subfamilies that showed unique secondary structures and phylogenetic relationships between the soybean subfamily members and with members of the <italic> Hsp </italic> 20 gene families identified in other species. We have presented evidence of the genomic complexity and diversity of the expression of the soybean <italic> Hsp </italic> 20 gene family. The soybean <italic> Hsp </italic> 20 genes are distributed across 17 chromosomes, where gene duplication events have most likely resulted in expansion of the family, most notably for the <span class="gene">CI</span> subfamily. The vast majority of the <italic> Hsp </italic> 20 genes analyzed <italic> in vivo </italic> (47 genes) were found to be strongly induced under heat shock, but other members of this family could be involved in normal cellular functions, which are unrelated to heat shock. Among the Gm <italic> Hsp </italic> 20 genes that were HSR, five were also identified as being involved in the soybean response to cold, and five others were responsive to <italic> <span class="trait">Meloidogyne</span> <span class="trait">javanica</span> </italic> infection. Furthermore, one predicted Gm <italic> Hsp </italic> 20 was shown to be responsive to nematode infection, while no change in expression was observed under other conditions. These genes showed a divergent expression pattern between the examined resistant and susceptible soybean genotypes. The promoter region of the Gm <italic> Hsp </italic> 20 members is minimally defined by the presence of a putative TATAbox and one to three putative HSEs, but results obtained elsewhere suggest that other regulatory elements found in this study are also likely to be important, such as W-box, CCAAT box sequences and TA-rich regions. The promoters that were responsive to biotic stress followed the same <italic> in silico </italic> predicted <italic> cis </italic> -element composition and distribution patterns that have been described for other species following nematode infection. Moreover, further investigation is required to obtain clues regarding the functions of the individual genes identified in this study. The results presented here can be further analyzed to reveal candidate genes and promoter structures that will be useful in developing technologies that generate genotypes that are more resistant to the various stresses that affect soybean crops.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean seeds ( <italic> G. max </italic> L, cv BRS 133 and genotype PI 595099, which are susceptible and resistant, respectively, to infection with <italic> <span class="trait">Meloidogyne</span> <span class="trait">javanica</span> </italic> obtained from the Embrapa Soja Active Germplasm Bank (AGB] were soaked for three days in sand. After stage <span class="gene">V3</span> was reached, the plants were subjected to abiotic or biotic stress in two independent experiments. In the abiotic stress experiments, BR133 genotype plants were exposed to a temperature of 42&#x02009;&#x000b1;&#x02009;2&#x000b0;C (heat stress), or 4&#x02009;&#x000b1;&#x02009;2&#x000b0;C for 3 hours (cold stress), or were maintained at 25&#x02009;&#x000b1;&#x02009;2&#x000b0;C for 3 hours (control plants). After being subjected to these stresses, the leaves were immediately collected in liquid nitrogen and transferred to &#x02212;80&#x000b0;C. In the biotic stress experiments, the BRS 133 (susceptible) and PI 595099 (resistant) genotypes were inoculated with 500 infective second-stage juvenile (J2) <italic> M. <span class="trait">javanica</span> </italic> or not inoculated (control plants). The <italic> M. <span class="trait">javanica</span> </italic> eggs were collected as described by Hussey and Barker [ <xref ref-type="bibr" rid="B54"> 54 </xref> ]. The roots were collected at 4 and 8 days post-infection.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>