<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>mosaic virus</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>mosaic virus</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:The interaction of soybean reticulon homology domain protein <span class="gene">(GmRHP)</span> with Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> encoded P3 contributes to the viral infection.</h2>
			<h3>PMID:29229386</h3>
				

				
			<p>
				作者：Cui X(1), Lu L(1), Wang Y(2), Yuan X(1), Chen X(3).
				<br/> 
				年份：2018 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), a member of the Potyvirus genus, is a prevalent and devastating viral pathogen in soybean-growing regions worldwide. Potyvirus replication occurs in the 6K2-induced viral replication complex at endoplasmic reticulum exit sites. Potyvirus-encoded P3 is also associated with the endoplasmic reticulum and is as an essential component of the viral replication complex, playing a key role in viral replication. This study provides evidence that the soybean (Glycine max) reticulon homology domain protein (designated as <span class="gene">GmRHP)</span> interacts with SMV-P3 by using a two-hybrid yeast system to screen a soybean cDNA library. A bimolecular fluorescence complementation assay further confirmed the interaction, which occurred on the cytomembrane, endoplasmic reticulum and cytoskeleton in Nicotiana benthamiana cells. The transient expression of <span class="gene">GmRHP</span> can promote the coupling of Turnip <span class="trait">mosaic</span> <span class="trait">virus</span> replication and cell-to-cell movement in N. benthamiana. The interaction between the membrane protein SMV-P3 and <span class="gene">GmRHP</span> may contribute to the potyvirus infection, and <span class="gene">GmRHP</span> may be an essential host factor for P3's involvement in potyvirus replication.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine-mapping and identifying candidate genes conferring <span class="trait">resistance</span> to Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> strain SC20 in soybean.</h2>
			<h3>PMID:29181547</h3>
				

				
			<p>
				作者：Karthikeyan A(1)(2)(3)(4), Li K(1)(2)(3)(4)(5), Li C(1)(2)(3)(4), Yin J(1)(2)(3)(4), Li N(1)(2)(3)(4), Yang Y(1)(2)(3)(4), Song Y(1)(2)(3)(4), Ren R(1)(2)(3)(4), Zhi H(1)(2)(3)(4)(5), Gai J(6)(7)(8)(9)(10).
				<br/> 
				年份：2017 Nov 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The Mendelian gene conferring <span class="trait">resistance</span> to Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> Strain SC20 in soybean was fine-mapped onto a 79-kb segment on Chr.13 where two closely linked candidate genes were identified and qRT-PCR verified. Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) threatens the world soybean production, particularly in China. A country-wide SMV strain system composed of 22 strains was established in China, among which SC20 is a dominant strain in five provinces in Southern China. <span class="trait">Resistance</span> to SC20 was evaluated in parents, F1, F2 and the F2:7 RIL (recombinant inbred line) population derived from a cross between Qihuang-1 (resistant) and NN1138-2 (susceptible). The segregation ratio of resistant to susceptible in the populations suggested a single dominant gene involved in the <span class="trait">resistance</span> to SC20 in Qihuang-1. A "partial genome mapping strategy" was used to map the <span class="trait">resistance</span> gene on Chromosome 13. Linkage analysis between 178 RILs and genetic markers showed that the SC20-resistance gene located at 3.9 and 3.8 cM to the flanking markers BARCSOYSSR_13_1099 and BARCSOYSSR_13_1185 on Chromosome 13. Subsequently, a residual heterozygote segregating population with 346 individuals was developed by selfing four plants heterozygous at markers adjacent to the tentative SC20-resistance gene; then, the candidate region was delimited to a genomic interval of approximately 79 kb flanked by the new markers gm-ssr_13-14 and gm-indel_13-3. Among the seven annotated candidate genes in this region, two genes, <span class="gene">Glyma.13G194700</span> and <span class="gene">Glyma.13G195100,</span> encoding Toll Interleukin Receptor-nucleotide-binding-leucine-rich repeat <span class="trait">resistance</span> proteins were identified as candidate <span class="trait">resistance</span> genes by quantitative real-time polymerase chain reaction and sequence analysis. The two closely linked genes work together to cause the phenotypic segregation as a single Mendelian gene. These results will facilitate marker-assisted selection, gene cloning and breeding for the <span class="trait">resistance</span> to SC20.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in nitric oxide levels and their relationship with callose deposition during the interaction between soybean and Soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:29125664</h3>
				

				
			<p>
				作者：Xiao D(1), Duan X(1)(2), Zhang M(1), Sun T(1), Sun X(1), Li F(1), Liu N(1), Zhang J(1), Hou C(1), Wang D(1).
				<br/> 
				年份：2017 Nov 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The present study aimed to investigate changes in nitric oxide (NO) level and its relationship with callose deposition during the interaction between soybean and Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV). Soybean cv. 'Jidou 7' and SMV strains <span class="gene">N3</span> and SC-8 were used to constitute incompatible and compatible combinations. Intracellular NO was labelled with the NO-specific fluorescence probe DAF-FM DA. Confocal laser scanning microscopy (CLSM) was then used to observe changes in NO production during SMV infection-induced defence responses in soybean. The results showed NO fluorescence increased rapidly at 2-72 h post-inoculation, peaked at 72 h and then decreased in the incompatible combination. However, in the compatible combination, extremely weak NO fluorescence appeared in the early stage (2-24 h) post-inoculation, but was not observed thereafter. Injections of the NO scavenger c-PTIO prior to inoculation postponed the onset of NO production to 48 or 72 h post-inoculation. The same occurred when injections of NR or NOS inhibitors were applied prior to inoculation. The observation of callose fluorescence in the incompatible combination revealed that either the elimination or reduction of NO in the early stage led to a delay in callose formation, enabling the <span class="trait">virus</span> to cause systemic infection. Together with our previous findings, this study indicates that viral infection could induce NO production and callose deposition during the incompatible interaction between soybean and SMV. The production of NO involves NR and NOS enzymatic pathways, and NO mediates the process of callose deposition at plasmodesmata.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and validation of a candidate Rsv3 gene from a soybean genotype that confers strain-specific <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:29080441  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tran PT(1), Widyasari K(2), Seo JK(3), Kim KH(4).
				<br/> 
				年份：2018 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), a member of the genus Potyvirus, significantly reduces soybean production worldwide. Rsv3, which confers strain-specific <span class="trait">resistance</span> to SMV, was previously mapped between the markers A519F/R and M3Satt in chromosome 14 of the soybean [Glycine max (L.) Merr.] genotype L29. Analysis of the soybean genome database revealed that five different NBS-LRR sequences exist between the flanking markers. Among these candidate Rsv3 genes, the full-length cDNA of the <span class="gene">Glyma.14g204700</span> was successfully cloned from L29. Over-expression of <span class="gene">Glyma.14g204700</span> in leaves inoculated with SMV inhibited viral infection in a soybean genotype lacking Rsv3. In addition, the transient silencing of the candidate gene caused a high accumulation of an avirulent strain in L29 carrying Rsv3. Our results therefore provide additional line of evidence to support that <span class="gene">Glyma.14g204700</span> is likely Rsv3 gene that confers strain-specific <span class="trait">resistance</span> to SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Robust RNAi-mediated <span class="trait">resistance</span> to infection of seven potyvirids in soybean expressing an intron hairpin <span class="gene">NIb</span> RNA.</h2>
			<h3>PMID:28840434</h3>
				

				
			<p>
				作者：Yang X(1), Niu L(1), Zhang W(1), He H(1), Yang J(1), Xing G(1), Guo D(1), Du Q(1), Qian X(1), Yao Y(1), Li Q(2), Dong Y(3).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Viral pathogens, such as soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), are a major constraint in soybean production and often cause significant <span class="trait">yield</span> loss and quality deterioration. Engineering <span class="trait">resistance</span> by RNAi-mediated gene silencing is a powerful strategy for controlling viral diseases. In this study, a 248-bp inverted repeat of the replicase (nuclear inclusion b, NIb) gene was isolated from the SMV SC3 strain, driven by the leaf-specific rbcS2 promoter from Phaseolus vulgaris, and introduced into soybean. The transgenic lines had significantly lower average disease indices (ranging from 2.14 to 12.35) than did the non-transformed (NT) control plants in three consecutive generations, exhibiting a stable and significantly enhanced <span class="trait">resistance</span> to the SMV SC3 strain under field conditions. Furthermore, seed mottling did not occur in transgenic seeds, whereas the NT plants produced ~90% mottled seeds. <span class="trait">Virus</span> <span class="trait">resistance</span> spectrum screening showed that the greenhouse-grown transgenic lines exhibited robust <span class="trait">resistance</span> to five SMV strains (SC3, SC7, SC15, SC18, and a recombinant SMV), bean common <span class="trait">mosaic</span> <span class="trait">virus,</span> and watermelon <span class="trait">mosaic</span> <span class="trait">virus.</span> Nevertheless, no significantly enhanced <span class="trait">resistance</span> to bean pod mottle <span class="trait">virus</span> (BPMV, Comovirus) was observed in the transgenic lines relative to their NT counterparts. Consistent with the results of <span class="trait">resistance</span> evaluation, the accumulation of each potyvirid (but not of BPMV) was significantly inhibited in the transgenic plants relative to the NT controls as confirmed by quantitative real-time (qRT-PCR) and double antibody sandwich enzyme-linked immunosorbent assay (DAS-ELISA). These results demonstrate that robust RNAi-mediated <span class="trait">resistance</span> to multiple potyvirids in soybean was conferred by expressing an intron hairpin SMV <span class="gene">NIb</span> RNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine-mapping and identification of a novel locus Rsc15 underlying soybean <span class="trait">resistance</span> to Soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:28825113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rui R(1)(2), Liu S(1)(2), Karthikeyan A(1)(2), Wang T(1)(2), Niu H(1)(2), Yin J(1)(2), Yang Y(1)(2), Wang L(1)(2), Yang Q(1)(2), Zhi H(3)(4), Li K(5)(6).
				<br/> 
				年份：2017 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Rsc15, a novel locus underlying soybean <span class="trait">resistance</span> to SMV, was fine mapped to a 95-kb region on chromosome 6. The Rsc15- mediated <span class="trait">resistance</span> is likely attributed to the gene <span class="gene">GmPEX14</span> , the relative expression of which was highly correlated with the accumulation of H 2 O 2 along with the activities of POD and CAT during the early stages of SMV infection in RN-9. Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) causes severe <span class="trait">yield</span> losses and seed quality deterioration in soybean [Glycine max (L.) Merr.] worldwide. A series of single dominant SMV <span class="trait">resistance</span> genes have been identified on respective soybean chromosomes 2, 13 and 14, while one novel locus, Rsc15, underlying <span class="trait">resistance</span> to the virulent SMV strain SC15 from soybean cultivar RN-9 has been recently mapped to a 14.6-cM region on chromosome 6. However, candidate gene has not yet been identified within this region. In the present study, we aimed to fine map the Rsc15 region and identify candidate gene(s) for this invaluable locus. High-resolution fine-mapping revealed that the Rsc15 gene was located in a 95-kb genomic region which was flanked by the two simple sequence repeat (SSR) markers SSR_06_17 and BARCSOYSSR_06_0835. Allelic sequence comparison and expression profile analysis of candidate genes inferred that the gene <span class="gene">Glyma.06g182600</span> (designated as <span class="gene">GmPEX14)</span> was the best candidate gene attributing for the <span class="trait">resistance</span> of Rsc15, and that genes encoding receptor-like kinase (RLK) (i.e., <span class="gene">Glyma.06g175100</span> and <span class="gene">Glyma.06g184400)</span> and serine/threonine kinase (STK) (i.e., <span class="gene">Glyma.06g182900</span> and <span class="gene">Glyma.06g183500)</span> were also potential candidates. High correlations were established between the relative expression level of <span class="gene">GmPEX14</span> and the hydrogen peroxide (H2O2) concentration and activities of catalase (CAT) and peroxidase (POD) during the early stages of SMV-SC15 infection in RN-9. The results of the present study will be useful in marker-assisted breeding for SMV <span class="trait">resistance</span> and will lead to further understanding of the molecular mechanisms of host <span class="trait">resistance</span> against SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Down-regulation of genes coding for core RNAi components and disease <span class="trait">resistance</span> proteins via corresponding microRNAs might be correlated with successful Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> infection in soybean.</h2>
			<h3>PMID:28695996</h3>
				

				
			<p>
				作者：Bao D(1), Ganbaatar O(1), Cui X(1), Yu R(1), Bao W(1), Falk BW(2), Wuriyanghan H(1).
				<br/> 
				年份：2017 Jul 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants protect themselves from <span class="trait">virus</span> infections by several different defence mechanisms. RNA interference (RNAi) is one prominent antiviral mechanism, which requires the participation of AGO (Argonaute) and Dicer/DCL (Dicer-like) proteins. Effector-triggered immunity (ETI) is an antiviral mechanism mediated by <span class="trait">resistance</span> (R) genes, most of which encode nucleotide-binding site-leucine-rich repeat (NBS-LRR) family proteins. MicroRNAs (miRNAs) play important regulatory roles in plants, including the regulation of host defences. Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is the most common <span class="trait">virus</span> in soybean and, in this work, we identified dozens of SMV-responsive miRNAs by microarray analysis in an SMV-susceptible soybean line. Amongst the up-regulated miRNAs, miR168a, miR403a, miR162b and miR1515a predictively regulate the expression of AGO1, AGO2, <span class="gene">DCL1</span> and DCL2, respectively, and miR1507a, miR1507c and miR482a putatively regulate the expression of several NBS-LRR family disease <span class="trait">resistance</span> genes. The regulation of target gene expression by these seven miRNAs was validated by both transient expression assays and RNA ligase-mediated rapid amplification of cDNA ends (RLM-RACE) experiments. Transcript levels for AGO1, DCL1, <span class="gene">DCL2</span> and five NBS-LRR family genes were repressed at different time points after SMV infection, whereas the corresponding miRNA levels were up-regulated at these same time points. Furthermore, inhibition of miR1507a, miR1507c, miR482a, miR168a and miR1515a by short tandem target mimic (STTM) technology compromised SMV infection efficiency in soybean. Our results imply that SMV can counteract soybean defence responses by the down-regulation of several RNAi pathway genes and NBS-LRR family <span class="trait">resistance</span> genes via the induction of the accumulation of their corresponding miRNA levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of <span class="gene">GmSN1</span> enhances <span class="trait">virus</span> <span class="trait">resistance</span> in Arabidopsis and soybean.</h2>
			<h3>PMID:28656325</h3>
				

				
			<p>
				作者：He H(1), Yang X(2), Xun H(1), Lou X(1), Li S(1), Zhang Z(1), Jiang L(1), Dong Y(2), Wang S(3), Pang J(4), Liu B(1).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: <span class="gene">GmSN1</span> enhances <span class="trait">virus</span> <span class="trait">resistance</span> in plants most likely by affecting the expression of signal transduction and immune response genes. Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) infection causes severe symptom and leads to massive <span class="trait">yield</span> loss in soybean (Glycine max). By comparative analyzing gene expression in the SMV-resistant soybean cultivar Rsmv1 and the susceptible cultivar Ssmv1 at a transcriptome level, we found that a subgroup of Gibberellic Acid Stimulated Transcript (GAST) genes were down-regulated in SMV inoculated Ssmv1 plants, but not Rsmv1 plants. Sequence alignment and phylogenetic analysis indicated that one of the GAST genes, <span class="gene">GmSN1,</span> was closely related to Snakin-1, a well-characterized potato microbial disease <span class="trait">resistance</span> gene. When over-expressed in Arabidopsis and soybean, respectively, under the control of the 35S promoter, <span class="gene">GmSN1</span> enhanced turnip <span class="trait">mosaic</span> <span class="trait">virus</span> <span class="trait">resistance</span> in the transgenic Arabidopsis plants, and SMV <span class="trait">resistance</span> in the transgenic soybean plants, respectively. Transcriptome analysis results showed that the up-regulated genes in the 35S:GmSN1 transgenic Arabidopsis plants were largely enriched in functional terms including "signal transduction" and "immune response". Real-time PCR assay indicated that the expression of <span class="gene">GmAKT2,</span> a potassium channel gene known to enhance SMV <span class="trait">resistance</span> when over-expressed in soybean, was elevated in the 35S:GmSN1 transgenic soybean plants. Taken together, our results suggest that <span class="gene">GmSN1</span> enhances <span class="trait">virus</span> <span class="trait">resistance</span> in plants most likely by affecting the expression of signal transduction and immune response genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inheritance, fine-mapping, and candidate gene analyses of <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> strain SC5 in soybean.</h2>
			<h3>PMID:28352959  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Karthikeyan A(1)(2)(3)(4)(5), Li K(1)(2)(3)(4)(5), Jiang H(1)(2)(3)(4)(5), Ren R(1)(2)(3)(4)(5), Li C(1)(2)(3)(4)(5), Zhi H(1)(2)(3)(4)(5), Chen S(6), Gai J(7)(8)(9)(10)(11).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is one of the most devastating pathogens for soybeans in China. Among the country-wide 22 strains, SC5 dominates in Huang-Huai and Changjiang valleys. For controlling its damage, the <span class="trait">resistance</span> gene was searched through Mendelian inheritance study, gene fine-mapping, and candidate gene analysis combined with qRT-PCR (quantitative real-time polymerase chain reaction) analysis. The parents F1, F2, and RILs (recombinant inbred lines) of the cross Kefeng-1 <span class="trait">(Resistance,</span> R) × NN1138-2 (Susceptible, S) were used to examine the inheritance of SC5-resistance. The F1 was resistant and the F2 and RILs segregated in a 3R:1S and 1R:1S ratio, respectively, indicating a single dominant gene conferring the Kefeng-1 <span class="trait">resistance.</span> Subsequently, the genomic region conferring the <span class="trait">resistance</span> was found in "Bin 352-Bin353 with 500 kb" on Chromosome 2 using the phenotyping data of the 427 RILs and a high-density genetic map with 4703 bin markers. In the 500 kb genomic region, 38 putative genes are contained. The association analysis between the SNPs in a putative gene and the <span class="trait">resistance</span> phenotype for the 427 RILs prioritized 11 candidate genes using Chi-square criterion. The expression levels of these genes were tested by qRT-PCR. On infection with SC5, 7 out of the 11 genes had differential expression in Kefeng-1 and NN1138-2. Furthermore, integrating SNP-phenotype association analysis with qRT-PCR expression profiling analysis, <span class="gene">Glyma02g13495</span> was found the most possible candidate gene for SC5-resistance. This finding can facilitate the breeding for SC5-resistance through marker-assisted selection and provide a platform to gain a better understanding of SMV-resistance gene system in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of a soybean <span class="trait">mosaic</span> <span class="trait">virus</span> variant causing different diseases in Glycine max and Nicotiana benthamiana.</h2>
			<h3>PMID:27804020  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jiang H(1)(2)(3)(4)(5), Li K(1)(2)(3)(4)(5), Dou D(6), Gai J(7)(8)(9)(10)(11).
				<br/> 
				年份：2017 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We discovered a soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) variant (4278-1) that caused systemic infections in Nicotiana benthamiana plants, resulting in stem stunting and leaf shriveling. The <span class="trait">virus</span> had a particle morphology and incubation period similar to those of other SMV isolates but differed from them in the leaf symptoms it caused when infecting soybean and N. benthamiana. The genome of this variant consisted of a 9994-nt single-stranded RNA, which was different from most of the other known SMV isolates (approximately 9600 nt). Interestingly, we found evidence that two recombination events (nt 1-476 and nt 1145-1349) had occurred between 4278-1 and a watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> analogue (WMV analogue), in the 5' untranslated region and the <span class="gene">P1</span> cistron.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping of the Rsv1-h gene in the soybean cultivar Suweon 97 that confers <span class="trait">resistance</span> to two Chinese strains of the soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:27544525  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ma FF(1), Wu XY(1), Chen YX(1), Liu YN(1), Shao ZQ(1), Wu P(1), Wu M(1), Liu CC(1), Wu WP(1), Yang JY(2), Li DX(3), Chen JQ(4), Wang B(5).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The Rsv1 - h gene in cultivar Suweon 97, which confers <span class="trait">resistance</span> to SMVs, was mapped to a 97.5-kb location (29,815,195-29,912,667 bp on chromosome 13) in the Rsv1 locus, thereby providing additional insights into the molecular nature underlying variations in <span class="trait">resistance</span> alleles in this particular locus. Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is a well-known devastating pathogen of soybean (Glycine max (L.) Merrill.) causing significant <span class="trait">yield</span> losses and seed quality deterioration. A single dominant allele, Rsv1-h, which confers <span class="trait">resistance</span> to multiple SMV strains, was previously reported in the cultivar Suweon 97, but its exact location is unknown. In the present study, Suweon 97 was crossed with a SMV-sensitive cultivar, Williams 82. Inoculating 267 F 2 individuals with two Chinese SMV strains (SC6-N and SC7-N) demonstrated that one single dominant gene confers SMV <span class="trait">resistance.</span> Another 1,150 F 2 individuals were then screened for two simple sequence repeat (SSR) markers (BARCSOYSSR_13_1103 and BARCSOYSSR_13_1187) that flank the Rsv1 locus. Seventy-four recombinants were identified and 20 additional polymorphic SSR markers within the Rsv1 region were then employed in genotyping these recombinants. F 2:3 and F 3:4 recombinant lines were also inoculated with SC6-N and SC7-N to determine their phenotypes. The final data revealed that in Suweon 97, the Rsv1-h gene that confers <span class="trait">resistance</span> to SC6-N and SC7-N was flanked by BARCSOYSSR_13_1114 and BARCSOYSSR_13_1115, two markers that delimit a 97.5-kb region in the reference Williams 82 genome. In such region, eight genes were present, of which two, <span class="gene">Glyma13g184800</span> and <span class="gene">Glyma13g184900,</span> encode the characteristic CC-NBS-LRR type of <span class="trait">resistance</span> gene and were considered potential candidates for Rsv1-h.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Potyviral P3 Protein Targets Eukaryotic Elongation Factor 1A to Promote the Unfolded Protein Response and Viral Pathogenesis.</h2>
			<h3>PMID:27356973  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Luan H(1), Shine MB(1), Cui X(1), Chen X(1), Ma N(1), Kachroo P(1), Zhi H(2), Kachroo A(2).
				<br/> 
				年份：2016 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The biochemical function of the potyviral P3 protein is not known, although it is known to regulate <span class="trait">virus</span> replication, movement, and pathogenesis. We show that P3, the putative virulence determinant of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), targets a component of the translation elongation complex in soybean. Eukaryotic elongation factor 1A (eEF1A), a well-known host factor in viral pathogenesis, is essential for SMV virulence and the associated unfolded protein response (UPR). Silencing <span class="gene">GmEF1A</span> inhibits accumulation of SMV and another ER-associated <span class="trait">virus</span> in soybean. Conversely, endoplasmic reticulum (ER) stress-inducing chemicals promote SMV accumulation in wild-type, but not <span class="gene">GmEF1A-knockdown,</span> plants. Knockdown of genes encoding the eEF1B isoform, which is important for eEF1A function in translation elongation, has similar effects on UPR and SMV <span class="trait">resistance,</span> suggesting a link to translation elongation. P3 and <span class="gene">GmEF1A</span> promote each other's nuclear localization, similar to the nuclear-cytoplasmic transport of eEF1A by the Human immunodeficiency <span class="trait">virus</span> 1 Nef protein. Our results suggest that P3 targets host elongation factors resulting in UPR, which in turn facilitates SMV replication and place eEF1A upstream of BiP in the ER stress response during pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Multiple Phytotoxins Produced by Fusarium virguliforme Including a Phytotoxic Effector (FvNIS1) Associated With Sudden Death Syndrome Foliar Symptoms.</h2>
			<h3>PMID:26646532  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chang HX(1), Domier LL(1)(2), Radwan O(1), Yendrek CR(1)(3), Hudson ME(1), Hartman GL(1)(2).
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sudden death syndrome (SDS) of soybean is caused by a soilborne pathogen, Fusarium virguliforme. Phytotoxins produced by F. virguliforme are translocated from infected roots to leaves, in which they cause SDS foliar symptoms. In this study, additional putative phytotoxins of F. virguliforme were identified, including three secondary metabolites and 11 effectors. While citrinin, fusaric acid, and radicicol induced foliar <span class="trait">chlorosis</span> and wilting, Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV)-mediated overexpression of F. virguliforme necrosis-inducing secreted protein 1 (FvNIS1) induced SDS foliar symptoms that mimicked the development of foliar symptoms in the field. The expression level of fvnis1 remained steady over time, although foliar symptoms were delayed compared with the expression levels. SMV::FvNIS1 also displayed genotype-specific toxicity to which 75 of 80 soybean cultivars were susceptible. Genome-wide association mapping further identified three single nucleotide polymorphisms at two loci, where three leucine-rich repeat receptor-like protein kinase (LRR-RLK) genes were found. Culture filtrates of fvnis1 knockout mutants displayed a mild reduction in phytotoxicity, indicating that FvNIS1 is one of the phytotoxins responsible for SDS foliar symptoms and may contribute to the quantitative susceptibility of soybean by interacting with the <span class="gene">LRR-RLK</span> genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a <span class="gene">GmCnx1</span> gene enhanced activity of nitrate reductase and aldehyde oxidase, and boosted <span class="trait">mosaic</span> <span class="trait">virus</span> <span class="trait">resistance</span> in soybean.</h2>
			<h3>PMID:25886067  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou Z(1), He H(2), Ma L(1), Yu X(1), Mi Q(1), Pang J(2), Tang G(1), Liu B(2).
				<br/> 
				年份：2015 Apr 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Molybdenum cofactor (Moco) is required for the activities of Moco-dependant enzymes. Cofactor for nitrate reductase and xanthine dehydrogenase (Cnx1) is known to be involved in the biosynthesis of Moco in plants. In this work, a soybean (Glycine max L.) Cnx1 gene <span class="gene">(GmCnx1)</span> was transferred into soybean using Agrobacterium tumefaciens-mediated transformation method. Twenty seven positive transgenic soybean plants were identified by coating leaves with phosphinothricin, <span class="gene">bar</span> protein quick dip stick and PCR analysis. Moreover, Southern blot analysis was carried out to confirm the insertion of <span class="gene">GmCnx1</span> gene. Furthermore, expression of <span class="gene">GmCnx1</span> gene in leaf and root of all transgenic lines increased 1.04-2.12 and 1.55-3.89 folds, respectively, as compared to wild type with <span class="gene">GmCnx1</span> gene and in line 10 , 22 showing the highest expression. The activities of Moco-related enzymes viz nitrate reductase (NR) and aldehydeoxidase (AO) of T1 generation plants revealed that the best line among the <span class="gene">GmCnx1</span> transgenic plants accumulated 4.25 μg g(-1) h(-1) and 30 pmol L(-1), respectively (approximately 2.6-fold and 3.9-fold higher than non-transgenic control plants).In addition, overexpression ofGmCnx1boosted the <span class="trait">resistance</span> to various strains of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV). DAS-ELISA analysis further revealed that infection rate of <span class="gene">GmCnx1</span> transgenic plants were generally lower than those of non-transgenic plants among two different <span class="trait">virus</span> strains tested. Taken together, this study showed that overexpression of a <span class="gene">GmCnx1</span> gene enhanced NR and AO activities and SMV <span class="trait">resistance,</span> suggesting its important role in soybean genetic improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean actin-depolymerizing factor 2 interacts with Soybean <span class="trait">mosaic</span> virus-encoded P3 protein.</h2>
			<h3>PMID:25537947  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu L(1), Wu G, Xu X, Luan H, Zhi H, Cui J, Cui X, Chen X.
				<br/> 
				年份：2015 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), a member of the Potyvirus genus, is one of the most prevalent and devastating viral pathogens in soybean-growing regions worldwide. It is generally accepted that symptom development of a viral plant disease results from molecular interactions between the <span class="trait">virus</span> and its host plant. P3 protein is the most variable <span class="gene">polyprotein</span> in potyviruses, which potentially plays an important role in the process of the evolution of <span class="trait">virus</span> type specialization. However, P3 not only plays a major role in <span class="trait">virus</span> replication and movement, but it is also responsible for symptom development in SMV-infected plants. This study provides evidence that actin-depolymerizing factor 2 (designated as ADF2) of soybean interacts with SMV P3 via a two-hybrid yeast system by screening a soybean cDNA library. Bimolecular fluorescence complementation assay further confirmed the interaction, which occurred in both the cytomembrane and cytoskeleton of Nicotiana benthamiana cells. The results support the hypothesis that SMV P3 might have a role in <span class="trait">virus</span> movement within cells.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Catalytic features and crystal structure of a tau class glutathione transferase from Glycine max specifically upregulated in response to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> infections.</h2>
			<h3>PMID:25479053  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Skopelitou K(1), Muleta AW(2), Papageorgiou AC(2), Chronopoulou E(1), Labrou NE(3).
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant tau class glutathione transferases (GSTs) play important roles in biotic and abiotic stress tolerance in crops and weeds. In this study, we systematically examined the catalytic and structural features of a <span class="gene">GST</span> isoenzyme from Glycine max <span class="gene">(GmGSTU10-10).</span> <span class="gene">GmGSTU10-10</span> is a unique isoenzyme in soybean that is specifically expressed in response to biotic stress caused by soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) infections. <span class="gene">GmGSTU10-10</span> was cloned, expressed in Escherichia coli, purified and characterized. The results showed that <span class="gene">GmGSTU10-10</span> catalyzes several different reactions and exhibits wide substrate specificity. Of particular importance is the finding that the enzyme shows high antioxidant catalytic function and acts as hydroperoxidase. In addition, its Km for GSH is significantly lower, compared to other plant GSTs, suggesting that <span class="gene">GmGSTU10-10</span> is able to perform efficient catalysis under conditions where the concentration of reduced glutathione is low (e.g. oxidative stress). The crystal structure of <span class="gene">GmGSTU10-10</span> was solved by molecular replacement at 1.6Å resolution in complex with glutathione sulfenic acid (GSOH). Structural analysis showed that <span class="gene">GmGSTU10-10</span> shares the same overall fold and domain organization as other plant cytosolic GSTs; however, major variations were identified in helix H9 and the upper part of helix H4 that affect the size of the active site pockets, substrate recognition and the catalytic mechanism. The results of the present study provide new information into <span class="gene">GST</span> diversity and give further insights into the complex regulation and enzymatic functions of this plant gene superfamily.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Type 2C protein phosphatase is a key regulator of antiviral extreme <span class="trait">resistance</span> limiting <span class="trait">virus</span> spread.</h2>
			<h3>PMID:25082428  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo JK(1), Kwon SJ(2), Cho WK(3), Choi HS(1), Kim KH(3).
				<br/> 
				年份：2014 Jul 31
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Effector-triggered immunity (ETI) is an active immune response triggered by interactions between host <span class="trait">resistance</span> proteins and their cognate effectors. Although ETI is often associated with the hypersensitive response (HR), various R genes mediate an HR-independent process known as extreme <span class="trait">resistance</span> (ER). In the soybean-Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) pathosystem, the strain-specific <span class="gene">CI</span> protein of SMV functions as an effector of Rsv3-mediated ER. In this study, we used the soybean (Rsv3)-SMV (CI) pathosystem to gain insight into the molecular signaling pathway involved in ER. We used genome-wide transcriptome analysis to identify a subset of the type 2C protein phophatase (PP2C) genes that are specifically up-regulated in Rsv3-mediated ER. Gain-of-function analysis of the most significantly expressed soybean PP2C gene, <span class="gene">GmPP2C3a,</span> showed that ABA-induced <span class="gene">GmPP2C3a</span> functions as a key regulator of Rsv3-mediated ER. Our results further suggest that the primary mechanism of ER against viruses is the inhibition of viral cell-to-cell movement by callose deposition in an ABA signaling-dependent manner.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> potassium channel enhances <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus</span></h2>
			<h3>PMID:24893844</h3>
				

				
					<h3>PMCID:4074861</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Zhou </surname>  <given-names> Lian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhoulian@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> He </surname>  <given-names> Hongli </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> hehl914@nenu.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Liu </surname>  <given-names> Ruifang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> liuruifang@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Han </surname>  <given-names> Qiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> han-qiang@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Shou </surname>  <given-names> Huixia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> huixia@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Liu </surname>  <given-names> Bao </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> baoliu@nenu.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 5 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The addition of K <sup> + </sup> fertilizer significantly reduced SMV incidence. Analysis of K <sup> + </sup> channel gene expression indicated that <italic> <span class="gene">GmAKT2,</span> </italic> the ortholog of <italic> Arabidopsis </italic> K <sup> + </sup> weak channel encoding gene <italic> AKT2 </italic> , was significantly induced by SMV inoculation in the SMV highly-resistant genotype Rsmv1, but not in the susceptible genotype Ssmv1. Transgenic soybean plants overexpressing <italic> <span class="gene">GmAKT2</span> </italic> were produced and verified by Southern blot and RT-PCR analysis. Analysis of K <sup> + </sup> concentrations on different leaves of both the transgenic and the wildtype (Williams 82) plants revealed that overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly increased K <sup> + </sup> concentrations in young leaves of plants. In contrast, K <sup> + </sup> concentrations in the old leaves of the <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants were significantly lower than those in WT plants. These results indicated that <italic> <span class="gene">GmAKT2</span> </italic> acted as a K <sup> + </sup> transporter and affected the distribution of K <sup> + </sup> in soybean plants. Starting from 14&#x000a0;days after inoculation (DAI) of SMV G7, severe <span class="trait">mosaic</span> symptoms were observed on the WT leaves. In contrast, the <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants showed no symptom of SMV infection. At 14 and 28 DAI, the amount of SMV RNA in WT plants increased 200- and 260- fold relative to <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants at each time point. Thus, SMV development was significantly retarded in <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly enhanced SMV <span class="trait">resistance</span> in transgenic soybean. Thus, alteration of K <sup> + </sup> transporter expression is a novel molecular approach for enhancing SMV <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						We observed that the incidence of SMV can be significantly reduced by application of K <sup> + </sup> fertilizer. Analysis of K <sup> + </sup> channel gene expression in SMV-resistant and SMV-susceptible cultivars showed that the expression of <italic> <span class="gene">GmAKT2</span> </italic> , the soybean inner K <sup> + </sup> transporter gene, was induced in the resistant variety, but not in the susceptible cultivar. Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly increased SMV <span class="trait">resistance</span> in the SMV susceptible cultivar Williams 82. Our results suggest that alterlation of the expression of the K <sup> + </sup> transporter <italic> AKT2 </italic> is a novel molecular approach to genetically enhance SMV <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> SMV <span class="trait">resistance</span> and K </bold> <sup> <bold> + </bold> </sup> <bold> concentrations of Williams 82 soybean plants grown under different K supplies. (A) </bold> SMV symptoms at 14 or 28&#x000a0;days after inoculation (DAI) with SMV in soybean plants grown in soil pots with low (36.5&#x000a0;mg/kg) or high (200&#x000a0;mg/kg) levels of K <sup> + </sup> . Unrolled unifoliate leaves of 10-day-old soybean plants were mechanically inoculated with SMV strain G7 or buffer (Mock). Photos were taken on the newest leaf of the plants, which were 2 <sup> nd </sup> trifoliate leaf at 14 DAI and 7 <sup> th </sup> trifoliate leaf at 28 DAI, respectively. <bold> (B) </bold> Amount of SMV RNA detected by quantitative RT-PCR (qRT-PCR). The middle leaflets of the leaves of Williams 82 plants which grown in low or high K soil were sampled at 14 and 28 DAI to extract total RNA for qRT-PCR analysis of SMV. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . <bold> (C) </bold> K <sup> + </sup> concentrations in individual leaves of plants grown in either high- or low-K soil. The first through seventh trifoliate leaves from 45-day-old soybean plants were sampled. Data represent means of three biological replicates with error bars indicating SD. Asterisks indicate a significant difference between high- and low-K soil (*P&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> SMV <span class="trait">resistance</span> and </bold> <bold> <italic> <span class="gene">GmAKT2</span> </italic> </bold> <bold> expression in three soybean genotypes. (A) </bold> Symptoms of three soybean genotypes at 14 and 28 DAI. Rsmv1, SMV highly-resistant; Williams 82, susceptible; Ssmv1, highly-susceptible. Ten-day-old soybean plants in low-K soil with unrolled unifoliate leaves were mechanically inoculated with SMV G7 or buffer (Mock). <bold> (B) </bold> Amount of SMV RNA detected by qRT-PCR. Leaves of three soybean genotypes were sampled at 14 and 28 DAI to extract total RNA for qRT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD. <bold> (C) </bold> Relative <italic> <span class="gene">GmAKT2</span> </italic> expression in three soybean genotypes at 28 DAI. Leaves of three soybean genotypes were sampled at 28 DAI to extract total RNA for qRT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmAKT2</span> </italic> full-length cDNA was amplified from Rsmv1, Williams 82, and Ssmv1, respectively. Sequence analysis showed that there is no variation on the sequence of <italic> <span class="gene">GmAKT2</span> </italic> cDNA (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S1). To assay the effect of <italic> <span class="gene">GmAKT2</span> </italic> on SMV <span class="trait">resistance,</span> the obtained <italic> <span class="gene">GmAKT2</span> </italic> cDNA driven by a modified CaMV 35S promoter (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> A) was introduced into Williams 82 via <italic> Agrobacterium </italic> -mediated transformation. Four independent T <sub> 1 </sub> transgenic lines overexpressing <italic> <span class="gene">GmAKT2</span> </italic> ( <italic> <span class="gene">GmAKT2</span> </italic> -Oe1, 2, 3, and 4) were verified by Southern blot analysis using the phosphinothricin acetyl transferase gene ( <italic> <span class="gene">bar</span> </italic> ) as the probe. Oe1, 2, 3, and 4 were four independent transgenic lines each containing a single copy (Oe1, 2, and 3) or two copies (Oe4) of the transgene (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> B). qRT-PCR analysis of the T <sub> 2 </sub> transgenic soybean plants showed that the transgene <italic> <span class="gene">GmAKT2</span> </italic> was highly expressed in all transgenic lines (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> C). The two transgenic events, Oe1 and Oe2, showing the highest levels of <italic> <span class="gene">GmAKT2</span> </italic> gene expression, were selected for further experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the effect of <italic> <span class="gene">GmAKT2</span> </italic> overexpression on SMV <span class="trait">resistance,</span> <italic> <span class="gene">GmAKT2</span> </italic> -Oe and WT plants were grown in pots with soil without the addition of K fertilizer in greenhouses. At least five plants were inoculated with SMV G7 or buffer in their unrolled unifoliate leaves at the VC stage [ <xref ref-type="bibr" rid="B35"> 35 </xref> ] when the unifoliate leaves were fully expanded. Starting from 14 DAI, while severe <span class="trait">mosaic</span> symptoms were observed on the leaves of the WT plants, the <italic> <span class="gene">GmAKT2</span> </italic> -overexpression lines Oe1 and Oe2 showed no symptom of SMV infection (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A, <xref ref-type="fig" rid="F6"> 6 </xref> B).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Overexpression of </bold> <bold> <italic> <span class="gene">GmAKT2</span> </italic> </bold> <bold> enhanced SMV <span class="trait">resistance.</span> (A) </bold> SMV symptoms in the WT and <span class="gene">GmAKT2</span> overexpression plants at 35 DAI. Ten-day-old soybean plants in low-K soil with unrolled unifoliate leaves were mechanically inoculated with SMV or buffer (Mock). <bold> (B) </bold> SMV symptoms on the newest trifoliate leaves of Williams 82 and <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic lines Oe1 and Oe2 at 14, 28, and 35 DAI. <bold> (C) </bold> Amount of SMV RNA detected by qRT-PCR. The middle leaflets of the leaves of Williams 82, and <italic> <span class="gene">GmAKT2</span> </italic> -ovexpression lines Oe1 and Oe2 were sampled at 14 and 28 DAI to extract total RNA for quantitative RT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD.
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm the results, qRT-PCR was used to determine SMV <span class="trait">virus</span> accumulation in these soybean plants. Compared to those in the Mock treatment, SMV inoculation significantly increased the amount of SMV <span class="trait">virus</span> in both WT and <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants. At 14 DAI, the level of SMV RNA in WT plants were about 240 and 160 times greater than that in the <italic> <span class="gene">GmAKT2</span> </italic> - Oe1 and Oe2 lines, respectively (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> C). With the increase of the infection process, at 28 DAI, the amount of SMV RNA in the WT was further increased. It was about 260 times as much as that of the average level of SMV RNA in <italic> <span class="gene">GmAKT2</span> </italic> -overexpression lines.
					</p>
				
					<p>&nbsp;&nbsp;
						To verify whether the SMV <span class="trait">resistance</span> conferred by <italic> <span class="gene">GmAKT2</span> </italic> overexpression is also effective on the other SMV stain, we tested the <span class="trait">resistance</span> of the transgenic plants to SMV strain G3. Results showed that the transgenic lines <italic> <span class="gene">GmAKT2</span> </italic> -Oe1 and Oe2 were also resistant to the infection of G3. Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly reduced the amount of G3 RNA in plants (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S2).
					</p>
				
					<p>&nbsp;&nbsp;
						The increased <span class="trait">virus</span> <span class="trait">resistance</span> in the transgenic lines <italic> <span class="gene">GmAKT2</span> </italic> -Oe1 and Oe2 significantly increased their growth performance and <span class="trait">yield</span> when the plants inoculated SMV strain G7. As shown in Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> , the SMV infection significantly reduced the plant height, numbers of node, pod, and seed, and the seed weight per plant of the WT soybean. In contrast, the infection of SMV <span class="trait">virus</span> slightly affected those agronomic traits. The difference in those traits between the infected plants and Mock was not significant.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the enhancement of K <sup> + </sup> transport, overexpression of <italic> <span class="gene">GmAKT2</span> </italic> on a SMV-susceptible genotype Williams 82 significantly retarded the development of SMV and resulted in higher <span class="trait">resistance</span> to SMV (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A, B, C). Plant <span class="trait">resistance</span> to viral disease can be achieved by inhibiting viral long-distance movement. In a typical <span class="trait">virus</span> infection process, <span class="trait">virus</span> moves from the source leaves to the sink leaves via the phloem transport system [ <xref ref-type="bibr" rid="B37"> 37 </xref> ], similar to the direction of K <sup> + </sup> movement. A mutation in <italic> AtAKT2 </italic> was found to inhibit sugar loading into the phloem of Arabidopsis [ <xref ref-type="bibr" rid="B28"> 28 </xref> ]. It is possible that overexpression of the soybean <italic> AKT2 </italic> resulted in changes in phloem environment to be unfavorable to <span class="trait">virus</span> loading, movement and unloading. Identification of the factor(s) to link the K <sup> + </sup> transportation and <span class="trait">virus</span> movement will help elucidate the mechanism of how the <italic> <span class="gene">GmAKT2</span> </italic> overexpression in soybean results in greater <span class="trait">resistance</span> to SMV.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant <span class="trait">resistance</span> to <span class="trait">virus</span> disease could also be achieved by inhibition of the viral multiplication. As expected, plant nutritional status affects plant health and, consequently, influences plant <span class="trait">resistance</span> to pathogen infection. In this study, the application of K <sup> + </sup> fertilizer significantly reduced the SMV accumulation in the SMV-susceptible cultivar Williams 82. Thus, the sufficient K <sup> + </sup> concentrations in soybean plant cells should be important for suppression of SMV spread. On the other hand, in the SMV-resistant <italic> <span class="gene">GmAKT2</span> </italic> overexpression plants, the increase of K <sup> + </sup> concentrations only occurred in younger leaves, but not in the older leaves. The results suggest that maintenance sufficient K <sup> + </sup> content in young leaves should be important for soybean <span class="trait">resistance</span> to <span class="trait">virus</span> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean potassium channel gene, <italic> <span class="gene">GmAKT2</span> </italic> was significantly induced by SMV inoculation in the SMV highly-resistant genotype Rsmv1, but not in the highly-susceptible genotype Ssmv1. The role of <italic> <span class="gene">GmAKT2</span> </italic> on SMV <span class="trait">resistance</span> was evaluated in the study. Results showed that overexpression of <italic> <span class="gene">GmAKT2</span> </italic> on a SMV-susceptible genotype Williams 82 enhanced the reallocation of K <sup> + </sup> in the soybean plants and increased K <sup> + </sup> concentrations in young leaves. Furthermore, SMV development was significantly retarded in <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic soybean plants following SMV inoculation. These data provide a novel route for initiating molecular breeding to improve SMV <span class="trait">resistance</span> in soybean, an important crop world wide.
					</p>
				
					<p>&nbsp;&nbsp;
						The full-length cDNA of soybean <italic> <span class="gene">GmAKT2</span> </italic> (Glym08g20030.1) was amplified by RT-PCR and ligated into a TOPO vector (Invitrogen, Carlsbad, CA, USA). <italic> <span class="gene">GmAKT2</span> </italic> was cloned into the gateway plasmid pB7RWG2,0 (Invitrogen) under the control of the 35S promoter of cauliflower <span class="trait">mosaic</span> <span class="trait">virus</span> (CaMV). The resulting construct was introduced into the Agrobacterium strain EHA101. Transgenic soybean plants were generated using the <italic> Agrobacterium tumefaciens </italic> -mediated cotyledon node transformation system [ <xref ref-type="bibr" rid="B39"> 39 </xref> ]. The Williams 82 genotype was used as the recipient of the transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						First-strand cDNA was synthesized from total RNA using Moloney Murine Leukemia <span class="trait">Virus</span> Reverse Transcriptase (M-MLV RT, Promega, San Luis Obispo, CA, USA). qRT-PCR was performed using SYBR Green I on a Light Cycler 480 (Roche Applied Science, Indianapolis, IN, USA), according to the manufacturer&#x02019;s instructions. The amplification program for SYBR Green I was performed as 94&#x000b0;C for 10&#x000a0;sec, 58&#x000b0;C for 10&#x000a0;sec, and 72&#x000b0;C for 10&#x000a0;sec. Triplicate quantitative assays were performed for each cDNA sample. <italic> <span class="gene">GmACTIN</span> </italic> was used as an internal control. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for relative expression or 2 <sup> -&#x00394;&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . <span class="trait">Virus</span> accumulation in soybean plants was estimated by qRT-PCR analysis of the SMV <span class="gene">polyprotein</span> gene in the SMV strain G7 or G3 genome. All primers used for RT-PCR are provided in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S1.
					</p>
				
					<p>&nbsp;&nbsp;
						The addition of K <sup> + </sup> fertilizer significantly reduced SMV incidence. Analysis of K <sup> + </sup> channel gene expression indicated that <italic> <span class="gene">GmAKT2,</span> </italic> the ortholog of <italic> Arabidopsis </italic> K <sup> + </sup> weak channel encoding gene <italic> AKT2 </italic> , was significantly induced by SMV inoculation in the SMV highly-resistant genotype Rsmv1, but not in the susceptible genotype Ssmv1. Transgenic soybean plants overexpressing <italic> <span class="gene">GmAKT2</span> </italic> were produced and verified by Southern blot and RT-PCR analysis. Analysis of K <sup> + </sup> concentrations on different leaves of both the transgenic and the wildtype (Williams 82) plants revealed that overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly increased K <sup> + </sup> concentrations in young leaves of plants. In contrast, K <sup> + </sup> concentrations in the old leaves of the <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants were significantly lower than those in WT plants. These results indicated that <italic> <span class="gene">GmAKT2</span> </italic> acted as a K <sup> + </sup> transporter and affected the distribution of K <sup> + </sup> in soybean plants. Starting from 14&#x000a0;days after inoculation (DAI) of SMV G7, severe <span class="trait">mosaic</span> symptoms were observed on the WT leaves. In contrast, the <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants showed no symptom of SMV infection. At 14 and 28 DAI, the amount of SMV RNA in WT plants increased 200- and 260- fold relative to <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants at each time point. Thus, SMV development was significantly retarded in <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly enhanced SMV <span class="trait">resistance</span> in transgenic soybean. Thus, alteration of K <sup> + </sup> transporter expression is a novel molecular approach for enhancing SMV <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						We observed that the incidence of SMV can be significantly reduced by application of K <sup> + </sup> fertilizer. Analysis of K <sup> + </sup> channel gene expression in SMV-resistant and SMV-susceptible cultivars showed that the expression of <italic> <span class="gene">GmAKT2</span> </italic> , the soybean inner K <sup> + </sup> transporter gene, was induced in the resistant variety, but not in the susceptible cultivar. Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly increased SMV <span class="trait">resistance</span> in the SMV susceptible cultivar Williams 82. Our results suggest that alterlation of the expression of the K <sup> + </sup> transporter <italic> AKT2 </italic> is a novel molecular approach to genetically enhance SMV <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> SMV <span class="trait">resistance</span> and K </bold> <sup> <bold> + </bold> </sup> <bold> concentrations of Williams 82 soybean plants grown under different K supplies. (A) </bold> SMV symptoms at 14 or 28&#x000a0;days after inoculation (DAI) with SMV in soybean plants grown in soil pots with low (36.5&#x000a0;mg/kg) or high (200&#x000a0;mg/kg) levels of K <sup> + </sup> . Unrolled unifoliate leaves of 10-day-old soybean plants were mechanically inoculated with SMV strain G7 or buffer (Mock). Photos were taken on the newest leaf of the plants, which were 2 <sup> nd </sup> trifoliate leaf at 14 DAI and 7 <sup> th </sup> trifoliate leaf at 28 DAI, respectively. <bold> (B) </bold> Amount of SMV RNA detected by quantitative RT-PCR (qRT-PCR). The middle leaflets of the leaves of Williams 82 plants which grown in low or high K soil were sampled at 14 and 28 DAI to extract total RNA for qRT-PCR analysis of SMV. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . <bold> (C) </bold> K <sup> + </sup> concentrations in individual leaves of plants grown in either high- or low-K soil. The first through seventh trifoliate leaves from 45-day-old soybean plants were sampled. Data represent means of three biological replicates with error bars indicating SD. Asterisks indicate a significant difference between high- and low-K soil (*P&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> SMV <span class="trait">resistance</span> and </bold> <bold> <italic> <span class="gene">GmAKT2</span> </italic> </bold> <bold> expression in three soybean genotypes. (A) </bold> Symptoms of three soybean genotypes at 14 and 28 DAI. Rsmv1, SMV highly-resistant; Williams 82, susceptible; Ssmv1, highly-susceptible. Ten-day-old soybean plants in low-K soil with unrolled unifoliate leaves were mechanically inoculated with SMV G7 or buffer (Mock). <bold> (B) </bold> Amount of SMV RNA detected by qRT-PCR. Leaves of three soybean genotypes were sampled at 14 and 28 DAI to extract total RNA for qRT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD. <bold> (C) </bold> Relative <italic> <span class="gene">GmAKT2</span> </italic> expression in three soybean genotypes at 28 DAI. Leaves of three soybean genotypes were sampled at 28 DAI to extract total RNA for qRT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmAKT2</span> </italic> full-length cDNA was amplified from Rsmv1, Williams 82, and Ssmv1, respectively. Sequence analysis showed that there is no variation on the sequence of <italic> <span class="gene">GmAKT2</span> </italic> cDNA (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S1). To assay the effect of <italic> <span class="gene">GmAKT2</span> </italic> on SMV <span class="trait">resistance,</span> the obtained <italic> <span class="gene">GmAKT2</span> </italic> cDNA driven by a modified CaMV 35S promoter (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> A) was introduced into Williams 82 via <italic> Agrobacterium </italic> -mediated transformation. Four independent T <sub> 1 </sub> transgenic lines overexpressing <italic> <span class="gene">GmAKT2</span> </italic> ( <italic> <span class="gene">GmAKT2</span> </italic> -Oe1, 2, 3, and 4) were verified by Southern blot analysis using the phosphinothricin acetyl transferase gene ( <italic> <span class="gene">bar</span> </italic> ) as the probe. Oe1, 2, 3, and 4 were four independent transgenic lines each containing a single copy (Oe1, 2, and 3) or two copies (Oe4) of the transgene (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> B). qRT-PCR analysis of the T <sub> 2 </sub> transgenic soybean plants showed that the transgene <italic> <span class="gene">GmAKT2</span> </italic> was highly expressed in all transgenic lines (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> C). The two transgenic events, Oe1 and Oe2, showing the highest levels of <italic> <span class="gene">GmAKT2</span> </italic> gene expression, were selected for further experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the effect of <italic> <span class="gene">GmAKT2</span> </italic> overexpression on SMV <span class="trait">resistance,</span> <italic> <span class="gene">GmAKT2</span> </italic> -Oe and WT plants were grown in pots with soil without the addition of K fertilizer in greenhouses. At least five plants were inoculated with SMV G7 or buffer in their unrolled unifoliate leaves at the VC stage [ <xref ref-type="bibr" rid="B35"> 35 </xref> ] when the unifoliate leaves were fully expanded. Starting from 14 DAI, while severe <span class="trait">mosaic</span> symptoms were observed on the leaves of the WT plants, the <italic> <span class="gene">GmAKT2</span> </italic> -overexpression lines Oe1 and Oe2 showed no symptom of SMV infection (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A, <xref ref-type="fig" rid="F6"> 6 </xref> B).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Overexpression of </bold> <bold> <italic> <span class="gene">GmAKT2</span> </italic> </bold> <bold> enhanced SMV <span class="trait">resistance.</span> (A) </bold> SMV symptoms in the WT and <span class="gene">GmAKT2</span> overexpression plants at 35 DAI. Ten-day-old soybean plants in low-K soil with unrolled unifoliate leaves were mechanically inoculated with SMV or buffer (Mock). <bold> (B) </bold> SMV symptoms on the newest trifoliate leaves of Williams 82 and <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic lines Oe1 and Oe2 at 14, 28, and 35 DAI. <bold> (C) </bold> Amount of SMV RNA detected by qRT-PCR. The middle leaflets of the leaves of Williams 82, and <italic> <span class="gene">GmAKT2</span> </italic> -ovexpression lines Oe1 and Oe2 were sampled at 14 and 28 DAI to extract total RNA for quantitative RT-PCR analysis. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . Data represent means of three biological replicates with error bars indicating SD.
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm the results, qRT-PCR was used to determine SMV <span class="trait">virus</span> accumulation in these soybean plants. Compared to those in the Mock treatment, SMV inoculation significantly increased the amount of SMV <span class="trait">virus</span> in both WT and <italic> <span class="gene">GmAKT2</span> </italic> -Oe plants. At 14 DAI, the level of SMV RNA in WT plants were about 240 and 160 times greater than that in the <italic> <span class="gene">GmAKT2</span> </italic> - Oe1 and Oe2 lines, respectively (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> C). With the increase of the infection process, at 28 DAI, the amount of SMV RNA in the WT was further increased. It was about 260 times as much as that of the average level of SMV RNA in <italic> <span class="gene">GmAKT2</span> </italic> -overexpression lines.
					</p>
				
					<p>&nbsp;&nbsp;
						To verify whether the SMV <span class="trait">resistance</span> conferred by <italic> <span class="gene">GmAKT2</span> </italic> overexpression is also effective on the other SMV stain, we tested the <span class="trait">resistance</span> of the transgenic plants to SMV strain G3. Results showed that the transgenic lines <italic> <span class="gene">GmAKT2</span> </italic> -Oe1 and Oe2 were also resistant to the infection of G3. Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> significantly reduced the amount of G3 RNA in plants (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S2).
					</p>
				
					<p>&nbsp;&nbsp;
						The increased <span class="trait">virus</span> <span class="trait">resistance</span> in the transgenic lines <italic> <span class="gene">GmAKT2</span> </italic> -Oe1 and Oe2 significantly increased their growth performance and <span class="trait">yield</span> when the plants inoculated SMV strain G7. As shown in Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> , the SMV infection significantly reduced the plant height, numbers of node, pod, and seed, and the seed weight per plant of the WT soybean. In contrast, the infection of SMV <span class="trait">virus</span> slightly affected those agronomic traits. The difference in those traits between the infected plants and Mock was not significant.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the enhancement of K <sup> + </sup> transport, overexpression of <italic> <span class="gene">GmAKT2</span> </italic> on a SMV-susceptible genotype Williams 82 significantly retarded the development of SMV and resulted in higher <span class="trait">resistance</span> to SMV (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A, B, C). Plant <span class="trait">resistance</span> to viral disease can be achieved by inhibiting viral long-distance movement. In a typical <span class="trait">virus</span> infection process, <span class="trait">virus</span> moves from the source leaves to the sink leaves via the phloem transport system [ <xref ref-type="bibr" rid="B37"> 37 </xref> ], similar to the direction of K <sup> + </sup> movement. A mutation in <italic> AtAKT2 </italic> was found to inhibit sugar loading into the phloem of Arabidopsis [ <xref ref-type="bibr" rid="B28"> 28 </xref> ]. It is possible that overexpression of the soybean <italic> AKT2 </italic> resulted in changes in phloem environment to be unfavorable to <span class="trait">virus</span> loading, movement and unloading. Identification of the factor(s) to link the K <sup> + </sup> transportation and <span class="trait">virus</span> movement will help elucidate the mechanism of how the <italic> <span class="gene">GmAKT2</span> </italic> overexpression in soybean results in greater <span class="trait">resistance</span> to SMV.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant <span class="trait">resistance</span> to <span class="trait">virus</span> disease could also be achieved by inhibition of the viral multiplication. As expected, plant nutritional status affects plant health and, consequently, influences plant <span class="trait">resistance</span> to pathogen infection. In this study, the application of K <sup> + </sup> fertilizer significantly reduced the SMV accumulation in the SMV-susceptible cultivar Williams 82. Thus, the sufficient K <sup> + </sup> concentrations in soybean plant cells should be important for suppression of SMV spread. On the other hand, in the SMV-resistant <italic> <span class="gene">GmAKT2</span> </italic> overexpression plants, the increase of K <sup> + </sup> concentrations only occurred in younger leaves, but not in the older leaves. The results suggest that maintenance sufficient K <sup> + </sup> content in young leaves should be important for soybean <span class="trait">resistance</span> to <span class="trait">virus</span> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean potassium channel gene, <italic> <span class="gene">GmAKT2</span> </italic> was significantly induced by SMV inoculation in the SMV highly-resistant genotype Rsmv1, but not in the highly-susceptible genotype Ssmv1. The role of <italic> <span class="gene">GmAKT2</span> </italic> on SMV <span class="trait">resistance</span> was evaluated in the study. Results showed that overexpression of <italic> <span class="gene">GmAKT2</span> </italic> on a SMV-susceptible genotype Williams 82 enhanced the reallocation of K <sup> + </sup> in the soybean plants and increased K <sup> + </sup> concentrations in young leaves. Furthermore, SMV development was significantly retarded in <italic> <span class="gene">GmAKT2</span> </italic> -overexpressing transgenic soybean plants following SMV inoculation. These data provide a novel route for initiating molecular breeding to improve SMV <span class="trait">resistance</span> in soybean, an important crop world wide.
					</p>
				
					<p>&nbsp;&nbsp;
						The full-length cDNA of soybean <italic> <span class="gene">GmAKT2</span> </italic> (Glym08g20030.1) was amplified by RT-PCR and ligated into a TOPO vector (Invitrogen, Carlsbad, CA, USA). <italic> <span class="gene">GmAKT2</span> </italic> was cloned into the gateway plasmid pB7RWG2,0 (Invitrogen) under the control of the 35S promoter of cauliflower <span class="trait">mosaic</span> <span class="trait">virus</span> (CaMV). The resulting construct was introduced into the Agrobacterium strain EHA101. Transgenic soybean plants were generated using the <italic> Agrobacterium tumefaciens </italic> -mediated cotyledon node transformation system [ <xref ref-type="bibr" rid="B39"> 39 </xref> ]. The Williams 82 genotype was used as the recipient of the transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						First-strand cDNA was synthesized from total RNA using Moloney Murine Leukemia <span class="trait">Virus</span> Reverse Transcriptase (M-MLV RT, Promega, San Luis Obispo, CA, USA). qRT-PCR was performed using SYBR Green I on a Light Cycler 480 (Roche Applied Science, Indianapolis, IN, USA), according to the manufacturer&#x02019;s instructions. The amplification program for SYBR Green I was performed as 94&#x000b0;C for 10&#x000a0;sec, 58&#x000b0;C for 10&#x000a0;sec, and 72&#x000b0;C for 10&#x000a0;sec. Triplicate quantitative assays were performed for each cDNA sample. <italic> <span class="gene">GmACTIN</span> </italic> was used as an internal control. Transcript levels were calculated using the formula 2 <sup> -&#x00394;Ct </sup> for relative expression or 2 <sup> -&#x00394;&#x00394;Ct </sup> for the expression levels relative to <italic> <span class="gene">GmACTIN</span> </italic> . <span class="trait">Virus</span> accumulation in soybean plants was estimated by qRT-PCR analysis of the SMV <span class="gene">polyprotein</span> gene in the SMV strain G7 or G3 genome. All primers used for RT-PCR are provided in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S1.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Role of soybean <span class="trait">mosaic</span> virus-encoded proteins in seed and aphid transmission in soybean.</h2>
			<h3>PMID:23927427  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jossey S(1), Hobbs HA, Domier LL.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is seed and aphid transmitted and can cause significant reductions in <span class="trait">yield</span> and seed quality in soybean (Glycine max). The roles in seed and aphid transmission of selected SMV-encoded proteins were investigated by constructing mutants in and chimeric recombinants between SMV 413 (efficiently aphid and seed transmitted) and an isolate of SMV G2 (not aphid or seed transmitted). As previously reported, the DAG amino acid sequence motif near the amino terminus of the coat protein (CP) was the major determinant in differences in aphid transmissibility of the two SMV isolates, and helper component proteinase (HC-Pro) played a secondary role. Seed transmission of SMV was influenced by P1, HC-Pro, and CP. Replacement of the <span class="gene">P1</span> coding region of SMV 413 with that of SMV G2 significantly enhanced seed transmissibility of SMV 413. Substitution in SMV 413 of the two amino acids that varied in the CPs of the two isolates with those from SMV G2, G to D in the DAG motif and Q to P near the carboxyl terminus, significantly reduced seed transmission. The Q-to-P substitution in SMV 413 also abolished virus-induced seed-coat mottling in plant introduction 68671. This is the first report associating P1, CP, and the DAG motif with seed transmission of a potyvirus and suggests that HC-Pro interactions with CP are important for multiple functions in the <span class="trait">virus</span> infection cycle.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean nuclear localized type-III <span class="gene">DnaJ</span> domain-containing HSP40 reveals its roles in cell death and disease <span class="trait">resistance.</span></h2>
			<h3>PMID:23289813  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu JZ(1), Whitham SA.
				<br/> 
				年份：2013 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heat-shock proteins such as <span class="gene">HSP70</span> and <span class="gene">HSP90</span> are important molecular chaperones that play critical roles in biotic and abiotic stress responses; however, the involvement of their co-chaperones in stress biology remains largely uninvestigated. In a screen for candidate genes stimulating cell death in Glycine max (soybean), we transiently overexpressed full-length cDNAs of soybean genes that are highly induced during soybean <span class="trait">rust</span> infection in Nicotiana benthamiana leaves. Overexpression of a type-III <span class="gene">DnaJ</span> domain-containing HSP40 <span class="gene">(GmHSP40.1),</span> a co-chaperone of HSP70, caused hypersensitive response (HR)-like cell death. The HR-like cell death was dependent on MAPKKKα and WIPK, because silencing each of these genes suppressed the HR. Consistent with the presence of a nuclear localization signal (NLS) motif within the <span class="gene">GmHSP40.1</span> coding sequence, GFP-GmHSP40.1 was exclusively present in nuclear bodies or speckles. Nuclear localization of <span class="gene">GmHSP40.1</span> was necessary for its function, because deletion of the NLS or addition of a nuclear export signal abolished its HR-inducing ability. <span class="gene">GmHSP40.1</span> co-localized with HcRed-SE, a protein involved in pri-miRNA processing, which has been shown to be co-localized with SR33-YFP, a protein involved in pre-mRNA splicing, suggesting a possible role for <span class="gene">GmHSP40.1</span> in mRNA splicing or miRNA processing, and a link between these processes and cell death. Silencing <span class="gene">GmHSP40.1</span> enhanced the susceptibility of soybean plants to Soybean <span class="trait">mosaic</span> <span class="trait">virus,</span> confirming its positive role in pathogen defense. Together, the results demonstrate a critical role of a nuclear-localized <span class="gene">DnaJ</span> domain-containing <span class="gene">GmHSP40.1</span> in cell death and disease <span class="trait">resistance</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The requirement of multiple defense genes in soybean Rsv1-mediated extreme <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:22712511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Grosic S, Whitham SA, Hill JH.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is a major viral pathogen of soybean. Among the three SMV <span class="trait">resistance</span> genes, Rsv1 mediates extreme <span class="trait">resistance</span> (ER) against most SMV strains, including the β-glucuronidase-tagged G2 isolate that was previously used in studies of Rsv1. Using virus-induced gene silencing (VIGS), we screened 82 VIGS constructs to identify genes that play a role in Rsv1-mediated ER to SMV infection. The target genes included putative Rsv1 candidate genes, soybean orthologs to known defense-signaling genes, and 62 WRKY transcription factors. We identified eight VIGS constructs that compromised Rsv1-mediated <span class="trait">resistance</span> when the target genes were silenced, including <span class="gene">GmEDR1,</span> <span class="gene">GmEDS1,</span> <span class="gene">GmHSP90,</span> <span class="gene">GmJAR1,</span> <span class="gene">GmPAD4,</span> and two WRKY transcription factors. Together, our results provide new insight into the soybean signaling network required for ER against SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Simultaneous Mutations in Multi-Viral Proteins Are Required for <italic> Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> to Gain Virulence on Soybean Genotypes Carrying Different <italic> R </italic> Genes</h2>
			<h3>PMID:22140577</h3>
				

				
					<h3>PMCID:3227670</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chowda-Reddy </surname>  <given-names> R. V. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Haiyue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hill </surname>  <given-names> John H. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Poysa </surname>  <given-names> Vaino </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Aiming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Genetic <span class="trait">resistance</span> is the most effective and sustainable approach to the control of plant pathogens that are a major constraint to agriculture worldwide. In soybean, three dominant R genes, i.e., Rsv1, Rsv3 and Rsv4, have been identified and deployed against Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) with strain-specificities. Molecular identification of virulent determinants of SMV on these <span class="trait">resistance</span> genes will provide essential information for the proper utilization of these <span class="trait">resistance</span> genes to protect soybean against SMV, and advance knowledge of virus-host interactions in general. METHODOLOGY/PRINCIPAL FINDINGS: To study the gain and loss of SMV virulence on all the three <span class="trait">resistance</span> loci, SMV strains G7 and two G2 isolates L and LRB were used as parental viruses. SMV chimeras and mutants were created by partial genome swapping and point mutagenesis and then assessed for virulence on soybean cultivars PI96983 (Rsv1), L-29 (Rsv3), V94-5152 (Rsv4) and Williams 82 (rsv). It was found that P3 played an essential role in virulence determination on all three <span class="trait">resistance</span> loci and <span class="gene">CI</span> was required for virulence on Rsv1- and Rsv3-genotype soybeans. In addition, essential mutations in HC-Pro were also required for the gain of virulence on Rsv1-genotype soybean. To our best knowledge, this is the first report that <span class="gene">CI</span> and P3 are involved in virulence on Rsv1- and Rsv3-mediated <span class="trait">resistance,</span> respectively. CONCLUSIONS/SIGNIFICANCE: Multiple viral proteins, i.e., HC-Pro, P3 and CI, are involved in virulence on the three <span class="trait">resistance</span> loci and simultaneous mutations at essential positions of different viral proteins are required for an avirulent SMV strain to gain virulence on all three <span class="trait">resistance</span> loci. The likelihood of such mutations occurring naturally and concurrently on multiple viral proteins is low. Thus, incorporation of all three <span class="trait">resistance</span> genes in a soybean cultivar through gene pyramiding may provide durable <span class="trait">resistance</span> to SMV.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To study the gain and loss of SMV virulence on all the three <span class="trait">resistance</span> loci, SMV strains G7 and two G2 isolates L and LRB were used as parental viruses. SMV chimeras and mutants were created by partial genome swapping and point mutagenesis and then assessed for virulence on soybean cultivars PI96983 ( <italic> Rsv1 </italic> ), L-29 ( <italic> Rsv3 </italic> ), V94-5152 ( <italic> Rsv4 </italic> ) and Williams 82 ( <italic> rsv </italic> ). It was found that P3 played an essential role in virulence determination on all three <span class="trait">resistance</span> loci and <span class="gene">CI</span> was required for virulence on <italic> Rsv1 </italic> - and <italic> Rsv3 </italic> -genotype soybeans. In addition, essential mutations in HC-Pro were also required for the gain of virulence on <italic> Rsv1 </italic> -genotype soybean. To our best knowledge, this is the first report that <span class="gene">CI</span> and P3 are involved in virulence on <italic> Rsv1- </italic> and <italic> Rsv3 </italic> -mediated <span class="trait">resistance,</span> respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> (SMV), a member of the family <italic> Potyviridae </italic> , is the most common viral pathogen of soybean <xref ref-type="bibr" rid="pone.0028342-Gunduz1"> [12] </xref> . SMV is a seed-borne, aphid-transmitted <span class="trait">virus</span> that causes severe <span class="trait">yield</span> loss and reduction in seed quality worldwide <xref ref-type="bibr" rid="pone.0028342-Cui1"> [13] </xref> . Similar to other potyviruses, SMV has a positive-sense, single-stranded RNA molecule as its genome, which is approximately 9,600 nucleotides in length encoding a large <span class="gene">polyprotein</span> of &#x0223c;350 kDa and a short <span class="gene">polyprotein</span> as a result of translational frameshift in the P3 coding region ( <xref ref-type="fig" rid="pone-0028342-g001"> Fig. 1A </xref> ). These two polyproteins are processed by three viral proteases (P1 and HC-Pro responsible for autocleavage at their N-terminus and NIA-Pro for all other cleavages) to release 11 mature proteins, from the N-terminus: P1, HC-Pro, P3, P3N-PIPO (resulting from translational slippage or frameshift in P3), 6K1, CI, 6K2 (or 6K), NIa-VPg, NIa-Pro, NIb, and CP <xref ref-type="bibr" rid="pone.0028342-Chung1"> [14] </xref> , <xref ref-type="bibr" rid="pone.0028342-UrcuquiInchima1"> [15] </xref> . To date, numerous SMV isolates have been reported. In North America, SMV isolates are classified into seven distinct strains, G1 through G7, based on their differential responses on susceptible and resistant soybean cultivars <xref ref-type="bibr" rid="pone.0028342-Cho1"> [16] </xref> . Screening for resistant soybean germplasm to SMV has identified three independent <span class="trait">resistance</span> loci, <italic> Rsv1 </italic> , <italic> Rsv3 </italic> and <italic> Rsv4 </italic> <xref ref-type="bibr" rid="pone.0028342-Gunduz2"> [17] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Zheng1"> [20] </xref> . These three loci are all dominant <italic> R </italic> genes <xref ref-type="bibr" rid="pone.0028342-Kang1"> [5] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hayes1"> [18] </xref> , <xref ref-type="bibr" rid="pone.0028342-Gore1"> [21] </xref> , <xref ref-type="bibr" rid="pone.0028342-Buss1"> [22] </xref> . <italic> Rsv1 </italic> , found in PI96983 confers extreme <span class="trait">resistance</span> to most SMV strains but not to G7, whereas <italic> Rsv3 </italic> -genotype soybean is resistant to higher numbered strain groups including G5 through G7 but susceptible to lower numbered strain groups (G1 through G4) <xref ref-type="bibr" rid="pone.0028342-Jeong1"> [23] </xref> . <italic> Rsv4 </italic> is the only gene that confers <span class="trait">resistance</span> to all the seven strains <xref ref-type="bibr" rid="pone.0028342-Gunduz1"> [12] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the past several years, many naturally and experimentally evolved SMV resistance-breaking isolates (all the three <span class="trait">resistance</span> loci) were documented <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0028342-Gagarinova1"> [8] </xref> , <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad3"> [27] </xref> , <xref ref-type="bibr" rid="pone.0028342-Kim1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0028342-Choi1"> [30] </xref> . As there are only three naturally-occurring resistant sources deployed for soybean breeding programs worldwide, concerns have been raised about the durability of these <span class="trait">resistance</span> genes <xref ref-type="bibr" rid="pone.0028342-Cui1"> [13] </xref> . The great majority of recent studies have focused on the genetic basis of SMV virulence on <span class="trait">resistance</span> mediated by each individual <span class="trait">resistance</span> gene. Through comparative genomic analyses, virulence proteins responsible for breaking down <span class="trait">resistance</span> have been mapped to HC-Pro, P3 and <span class="gene">CI</span> depending on the type of <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad4"> [28] </xref> , <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Wen1"> [33] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To study genetic determinants of SMV virulence on all three identified <span class="trait">resistance</span> loci in soybean, we used SMV strain G7 and two isolates of the G2 strain, L and LRB ( <xref ref-type="fig" rid="pone-0028342-g001"> Fig. 1 </xref> ). G7 is avirulent on <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans and virulent on <italic> Rsv1 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Gunduz1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0028342-Buss1"> [22] </xref> . In contrast, L infects <italic> Rsv3 </italic> - but not <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> -genotype soybeans, and LRB, a naturally evolved isolate of L differentiated from L by overcoming <italic> Rsv4 </italic> -mediated <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0028342-Gagarinova1"> [8] </xref> . The full-length cDNA infectious clones derived from these three SMVs shared pathogenicity similar to their respective parental viruses ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ). Since P3 is a virulence determinant for <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> <span class="trait">-resistance</span> <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad4"> [28] </xref> and <span class="gene">CI</span> is critical for <italic> Rsv3 </italic> <span class="trait">-resistance</span> <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0028342-Zhang1"> [32] </xref> , we first constructed hybrid SMVs by swapping the genomic fragment M (encoding the C-terminal 30 amino acids of HC-Pro, P3, P3-PIPO, 6K1 and the N-terminal two thirds of CI) ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ). The resulting hybrid SMVs were subjected to pathogenicity assays. Reciprocal exchange of the M fragment of L and LRB did not change their avirulence on <italic> Rsv1 </italic> -genotype and virulence on <italic> Rsv3 </italic> -genotype soybean but resulted in the gain or loss of virulence on <italic> Rsv4 </italic> -genotype soybean, consistent with our published data that P3 of LRB is responsible for breaking down <italic> Rsv4 </italic> -mediated <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> . When the M fragment of L or LRB was replaced with that of G7, the resulting chimeras were avirulent on <italic> Rsv1 </italic> -, <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ). In agreement with previous observations, both the N-terminal P3 (overlapping with P3N-PIPO) and HC-Pro of G7 are required for a G2 isolate to gain virulence on <italic> Rsv1 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Hajimorad4"> [28] </xref> , <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> and the <span class="gene">CI</span> of G2 is required for G7 to gain virulence on <italic> Rsv3 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Zhang1"> [32] </xref> . Chimeric SMVs derived from G7 whose M fragment was substituted with the homologous region of L or LRB lost virulence on <italic> Rsv1 </italic> -genotype soybean, gained virulence on <italic> Rsv3 </italic> -genotype soybean and were avirulent on <italic> Rsv4 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3A </xref> ). The loss of the G7 P3 may account for losing virulence on <italic> Rsv1 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Hajimorad4"> [28] </xref> , <xref ref-type="bibr" rid="pone.0028342-Zhang1"> [32] </xref> . The gain of virulence on <italic> Rsv3 </italic> -genotype soybean was not expected as these hybrid SMVs did not contain <italic> Rsv3 </italic> pathogenetic determinants identified previously, i.e., the C-terminal <span class="gene">CI</span> of G7H <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> or both the N- and C-terminal <span class="gene">CI</span> of G2 <xref ref-type="bibr" rid="pone.0028342-Zhang1"> [32] </xref> . Since the LRB P3 is responsible for the gain of virulence on <italic> Rsv4 </italic> soybean <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , it was surprising that G7(LRB 2339-4624), a G7 derivative containing the entire LRB P3, was unable to infect <italic> Rsv4 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3A </xref> ). Thus, it is reasonable to suggest that, in addition to P3, other viral protein(s) or domain(s) of G2 are also required for virulence on <italic> Rsv4 </italic> -conferred <span class="trait">resistance.</span> Taken together these results suggest multi-viral proteins constitute virulence determinants for each of the three <span class="trait">resistance</span> loci in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The amino acid sequence of HC-Pro, P3, P3N-PIPO, 6K1 and <span class="gene">CI</span> was compared among SMV isolates. The very C-terminal HC-Pro amino-acid sequence (downstream of the KpnI site) consisting of 30 amino acids, is identical among all the SMV isolates analyzed including G2 and G7 isolates ( <xref ref-type="fig" rid="pone-0028342-g004"> Fig. 4 </xref> ). The 6K1 sequence is also highly conserved among SMV isolates with only one substitution (A to V) concerning two similar amino acids for two isolates ( <xref ref-type="supplementary-material" rid="pone.0028342.s001"> Fig. S1 </xref> ). Therefore, these two regions are unlikely to be virulence determinants on the three <span class="trait">resistance</span> loci. To further determine the virulence role of P3 (consisting of the embedded P3N-PIPO) ( <xref ref-type="fig" rid="pone-0028342-g005"> Fig. 5 </xref> ; <xref ref-type="supplementary-material" rid="pone.0028342.s002"> Fig. S2 </xref> ) and the N-terminal <span class="gene">CI</span> ( <xref ref-type="supplementary-material" rid="pone.0028342.s003"> Figs. S3 </xref> and <xref ref-type="supplementary-material" rid="pone.0028342.s004"> S4 </xref> ), two more hybrid SMVs, G7(L 3234-4624) and G7(L 2339-3234) were created ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ). Both of them lost virulence on <italic> Rsv3 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ), indicating neither the N-terminal P3 (including P3N-PIPO) of G2 nor the C-terminal P3/N-terminal <span class="gene">CI</span> of G2 was sufficient for virulence on <italic> Rsv3 </italic> -genotype soybean. Thus, both P3 and <span class="gene">CI</span> were involved in virulence on <italic> Rsv3 </italic> -genotype soybean. Neither of these two viruses restored virulence on <italic> Rsv1 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ), indicating the N-terminal P3 of G7 as well as the C-terminal P3/N-terminal <span class="gene">CI</span> of G7 was essential for G7 to maintain virulence on <italic> Rsv1 </italic> -genotype soybean. Previously, a single mutation (G1054R), downstream of the KpnI site in the C-terminal P3 of isolate NPL, an L derivative ( <xref ref-type="fig" rid="pone-0028342-g005"> Fig. 5 </xref> ) was sufficient to make isolate L virulent on <italic> Rsv4 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> . This mutation was introduced into the hybrid <span class="trait">virus</span> G7(L 3234-4624). The resulting <span class="trait">virus</span> G7(L 3234-4624)(G1054R) was unable to infect <italic> Rsv4 </italic> -genotype soybean or other resistant soybeans ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ). This result again supports that the G2 P3 must function with other viral determinants including <span class="gene">CI</span> (see below) for virulence on <italic> Rsv4 </italic> <span class="trait">-resistance.</span> To further clarify the virulence role of the N-terminal CI, a chimeric infectious clone G7(L 3625-4624) was constructed. In comparison with wild-type G7, this recombinant <span class="trait">virus</span> lost virulence on <italic> Rsv1 </italic> -genotype soybean but gained infectivity on <italic> Rsv3 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3B </xref> ), indicating an essential role of <span class="gene">CI</span> for breaking down both <italic> Rsv1 </italic> - and <italic> Rsv3 </italic> -resistances. The fact that this chimeric <span class="trait">virus</span> infected <italic> Rsv3 </italic> -genotype soybean, in comparison with recombinant clones G7(L 3234-4624) and G7(L 2339-4624), confirmed the involvement of P3 in breaking down Rsv3-resistance ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3 </xref> ). Taken together these data suggest P3 is involved in virulence on all three <span class="trait">resistance</span> loci and <span class="gene">CI</span> is essential at least for virulence on <italic> Rsv-1 </italic> and <italic> Rsv3 </italic> -genotype soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						The C-terminal moiety (the BglII and KpnI fragment) of HC-Pro of L(G7 2342-3237) and L(G7 2342-3237)(G1054R) was further replaced with the corresponding region of G7 to generate chimeric SMVs L(G7 1608-3237) and L(G7 1608-3237)(G1054R) ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g004"> 4 </xref> ). Pathogenicity tests showed that L(G7 1608-3237) retained virulence on <italic> Rsv3 </italic> -genotype soybean, remained avirulent on <italic> Rsv4 </italic> -genotype soybean and gained virulence on <italic> Rsv1 </italic> -genotype soybean. L(G7 1608-3237)(G1054R) retained virulence on <italic> Rsv3 </italic> -genotype soybean and gained virulence on <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> -genotype soybeans ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g007"> 7 </xref> ). To test if the N-terminal virus-encoded <span class="gene">polyprotein</span> (upstream of BglII) and the C-terminal <span class="gene">polyprotein</span> (downstream of AgeI) affect virulence on <italic> Rsv1 </italic> -, <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans, L(G7 1608-3237)(G1054R) was used as a parental clone to produce recombinant viruses L(G7 1-3237)(G1054R) and G7(L 1-1605, 3234-4624)(G1054R). These two chimeric viruses retained infectivity on <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> -genotype soybeans ( <xref ref-type="fig" rid="pone-0028342-g008"> Fig. 8 </xref> ), indicating either N- (upstream of BglII) or C-termini (downstream of AgeI) of the virus-encoded <span class="gene">polyprotein</span> between G2 and G7 does not contain avirulent determinant(s) on <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> -resistances. However, replacement of the C-terminal <span class="gene">polyprotein</span> (downstream of AgeI) did affected virulence on <italic> Rsv3 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g008"> Fig. 8B </xref> ), suggesting the involvement of this region in breaking down <italic> Rsv3 </italic> <span class="trait">-resistance.</span> The symptoms in <italic> Rsv1 </italic> -genotype soybean plants induced by L(G7 1608-3237), L(G7 1608-3237)(G1054R), L(G7 1-3237)(G1054R) and G7(L 1-1605, 3234-4624)(G1054R) were typical of LSHR ( <xref ref-type="fig" rid="pone-0028342-g007"> Figs. 7 </xref> , <xref ref-type="fig" rid="pone-0028342-g008"> 8 </xref> and <xref ref-type="fig" rid="pone-0028342-g009"> 9 </xref> ), similar to those induced by G7 ( <xref ref-type="fig" rid="pone-0028342-g009"> Fig. 9 </xref> ). These results demonstrate that the C-terminal moiety of HC-Pro of G7 and the N-terminal P3 of G7 co-determine virulence on <italic> Rsv1 </italic> -genotype soybean but the corresponding regions of G2 are not required for virulence on <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans. As mentioned earlier, L(G7 2342-4627) or LRB(G7 2342-4624) lost virulence on <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans and the G2 P3 must function with other viral determinant(s) to maintain virulence on <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans. Taken together, these data suggest the G2 <span class="gene">CI</span> is critical for virulence on <italic> Rsv3 </italic> -genotype soybean and likely on <italic> Rsv4 </italic> -genotype soybean as well. Indeed, SMVs, i.e., G7(L3234-4624)(G1054R) and G7(L 2339-3234) consisting of <span class="gene">CI</span> hybrids (G2/G7) or the entire G7 <span class="gene">CI</span> abolished virulence on <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans ( <xref ref-type="fig" rid="pone-0028342-g002"> Fig. 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, three SMV isolates of two strains with different responses on <italic> Rsv1 </italic> -, <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans were employed to study virulence determinants required for virulence on soybean genotypes carrying different <span class="trait">resistance</span> genes. Through comparative genomic analyses, we have provided evidence that P3 is involved in virulence on these resistant cultivars. In recent studies, the potyviral P3 protein (not the embedded PIPO) has been shown as a major determinant for the loss and gain of virulence in a number of potyviral pathosystems. For instance, a mutation (A1047V) in the C-terminus (downstream of PIPO) of the <italic> Tobacco etch <span class="trait">virus</span> </italic> (TEV) P3 protein was found to be associated with the adaptation of TEV to <italic> Arabidopsis thaliana </italic> <xref ref-type="bibr" rid="pone.0028342-AgudeloRomero1"> [34] </xref> . Multiple determinants in the N-terminal P3 (upstream of PIPO) of <italic> Pea seed-borne <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> were identified to determine the gain and loss of virulence in <italic> Pisum sativum </italic> carrying the recessive <span class="trait">resistance</span> gene <italic> sbm-2 </italic> <xref ref-type="bibr" rid="pone.0028342-Hjulsager1"> [35] </xref> . The virulence determinants of <italic> Turnip <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> (TuMV) on <italic> Brassica napus </italic> cultivars carrying <span class="trait">resistance</span> genes <italic> TuRB03 </italic> or <italic> TuRB04 </italic> were mapped to both the N- and C-termini of P3 (outside PIPO) <xref ref-type="bibr" rid="pone.0028342-Jenner1"> [36] </xref> , <xref ref-type="bibr" rid="pone.0028342-Jenner2"> [37] </xref> . The N-terminal P3 (upstream of PIPO) of TuMV strain TuR1 was also shown to determine the systemic necrosis in <italic> Arabidopsis </italic> ecotype Ler carrying the dominant gene <italic> TuNI </italic> through the protein-protein interaction between TuNI and P3 <xref ref-type="bibr" rid="pone.0028342-Kim2"> [38] </xref> . In the case of SMV, the N-terminal P3 (before PIPO) of G7 was shown to be essential for its virulence on <italic> Rsv1 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Hajimorad5"> [39] </xref> . When the P3 or the N-terminal moiety of P3 of G7 was replaced with the corresponding region from isolates N, L and LRB (three avirulent G2 isolates), the resulting SMVs lost virulence on <italic> Rsv1 </italic> -genotype soybean <xref ref-type="bibr" rid="pone.0028342-Hajimorad5"> [39] </xref> . We have also reported that the C-terminal P3 (after PIPO) of isolate LRB, a naturally evolved G2 isolate, is responsible for breaking down <italic> Rsv4 </italic> -mediated <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> . A point mutation in the C-terminal P3 (after PIPO) in the isolate L (Q1033K or G1054R) <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> or recombinant SMVs L(G7 2342-3237) (G1054R) and L(G7 1608-3237)(G1054R) in this study is sufficient to alter pathogenicity on <italic> Rsv4 </italic> -genotype soybean. In this report, we have shown that a recombinant G7 containing the entire P3 as well as the N-terminal <span class="gene">CI</span> of L, a G2 isolate, gained virulence on <italic> Rsv3 </italic> -genotype soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3A </xref> ). When the N-terminal P3 was reversely swapped back to the G7 type, the resulting recombinant SMV G7(L 3234-4624) lost the ability to infect <italic> Rsv3 </italic> -genotype soybean. Such virulence on <italic> Rsv3 </italic> -genotype soybean could be restored by the chimeric <span class="trait">virus</span> G7(L 3625-4624) where the entire P3 was from G7 ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3B </xref> ). These data suggest P3 is not only a virulence determinant on <italic> Rsv1 </italic> - and <italic> Rsv4 </italic> -genotype soybeans <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad5"> [39] </xref> but is also required for virulence on <italic> Rsv3 </italic> -genotype soybean. Thus, P3 is a virulence determinant for all three <span class="trait">resistance</span> genes in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In recent studies, the <span class="gene">CI</span> protein of G7H and G5H has been shown to be a virulence and avirulence determinant on <italic> Rsv3 </italic> -genotype soybean, respectively <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> . A single amino acid change in the C-terminal of <span class="gene">CI</span> was sufficient for G5H to gain virulence on <italic> Rsv3 </italic> -genotype soybean. For G2 and G7 pathotypes, both the N and C termini of <span class="gene">CI</span> of the N isolate were required for G7 to break down <italic> Rsv3 </italic> <span class="trait">-resistance</span> <xref ref-type="bibr" rid="pone.0028342-Zhang1"> [32] </xref> . In this report, we not only confirmed that <span class="gene">CI</span> is involved in virulence on <italic> Rsv3 </italic> -genotype soybean but also provided evidence that <span class="gene">CI</span> is essential in virulence on <italic> Rsv1 </italic> -soybean ( <xref ref-type="fig" rid="pone-0028342-g002"> Figs. 2 </xref> and <xref ref-type="fig" rid="pone-0028342-g003"> 3B </xref> ). Previously, <span class="gene">CI</span> was suggested to be a virulence determinant for several other potyviruses. For instance, the <span class="gene">CI</span> protein of TuMV was shown to be responsible for overcoming <italic> TuRB01 </italic> - and <italic> TuRB05 </italic> -mediated <span class="trait">resistance</span> in <italic> Brassica napus </italic> <xref ref-type="bibr" rid="pone.0028342-Jenner3"> [40] </xref> , <xref ref-type="bibr" rid="pone.0028342-AbdulRazzak1"> [41] </xref> . In the pepper- <italic> Lettuce <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> (LMV) pathosystem, mutations in the C-terminal region of <span class="gene">CI</span> granted an avirulent LMV isolate the ability to infect lettuce carrying either recessive <span class="trait">resistance</span> genes <italic> mol1 </italic> or <italic> mol2 </italic> <xref ref-type="bibr" rid="pone.0028342-AbdulRazzak1"> [41] </xref> . Interestingly, a hybrid <span class="gene">CI</span> significantly increased the capacity of a <italic> Potato <span class="trait">virus</span> Y </italic> (PVY) isolate to break down <italic> pvr2 </italic> -mediated <span class="trait">resistance</span> in pepper <xref ref-type="bibr" rid="pone.0028342-Montarry1"> [42] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Multi-viral proteins including HC-Pro, P3 and <span class="gene">CI</span> are responsible for potyvirus virulence or avirulence on <italic> Rsv1 </italic> -, <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans ( <xref ref-type="bibr" rid="pone.0028342-Seo1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad4"> [28] </xref> , <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Wen1"> [33] </xref> , this study) and other <italic> R </italic> -genotype plant species <xref ref-type="bibr" rid="pone.0028342-Moury1"> [2] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hjulsager1"> [35] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Kim2"> [38] </xref> , <xref ref-type="bibr" rid="pone.0028342-Jenner3"> [40] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Montarry1"> [42] </xref> . Whether these viral proteins function independently or cooperatively as virulence determinants remains to be elucidated. The function of the potyviral P3 protein is poorly characterized <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> . In addition to its function as a virulence determinant, it has been assumed to play a role in several steps of the potyviral infection cycle such as <span class="trait">virus</span> replication, systemic infection, pathogenicity and movement <xref ref-type="bibr" rid="pone.0028342-ChowdaReddy1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> , <xref ref-type="bibr" rid="pone.0028342-Jenner2"> [37] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad6"> [43] </xref> , <xref ref-type="bibr" rid="pone.0028342-Klein1"> [49] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Cui2"> [52] </xref> . Both <span class="gene">CI</span> and HC-Pro are multifunctional proteins <xref ref-type="bibr" rid="pone.0028342-UrcuquiInchima1"> [15] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad6"> [43] </xref> , <xref ref-type="bibr" rid="pone.0028342-AlaPoikela1"> [53] </xref> . CI, having RNA binding, RNA helicase and ATPase activities, has been shown to be essential in <span class="trait">virus</span> intra- and intercellular movement and <span class="trait">virus</span> replication <xref ref-type="bibr" rid="pone.0028342-Carrington1"> [54] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Wei1"> [56] </xref> . HC-Pro has auto-catalytic proteinase and RNA silencing suppression activities and participates in <span class="gene">polyprotein</span> processing, aphid transmission, long-distance movement and viral genome amplification <xref ref-type="bibr" rid="pone.0028342-Blanc1"> [57] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0028342-Kasschau2"> [59] </xref> . Since HC-Pro, P3, 6K1 and <span class="gene">CI</span> result from catalytic processing of the large potyviral polyprotein, it is possible that an intermediate precursor protein containing these proteins acts as an elicitor in <italic> Rsv1 </italic> -, <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> -genotype soybeans. As HC-Pro is a cysteine proteinase that efficiently autocleaves the junction between itself and P3, it is more likely that protein complexes formed through protein-protein interactions rather than a single polypeptide play the elicitor role. Indeed, the <span class="gene">CI</span> has been shown to bind to other viral proteins including HC-Pro <xref ref-type="bibr" rid="pone.0028342-Blanc1"> [57] </xref> , P3 <xref ref-type="bibr" rid="pone.0028342-Kasschau1"> [58] </xref> and P3N-PIPO <xref ref-type="bibr" rid="pone.0028342-Kasschau2"> [59] </xref> . The essential components of the protein complex may vary depending on the type of <span class="trait">resistance.</span> For instance, HC-Pro and P3 are essential for <italic> Rsv1 </italic> <span class="trait">-resistance</span> <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad6"> [43] </xref> whereas P3 and <span class="gene">CI</span> are required for <italic> Rsv3 </italic> - and <italic> Rsv4 </italic> <span class="trait">-resistance</span> (this study). As suggested previously <xref ref-type="bibr" rid="pone.0028342-Culver1"> [60] </xref> , overcoming <italic> R </italic> -mediated <span class="trait">resistance</span> may be the outcome of a temporal race of the replication and intercellular movement of the invading <span class="trait">virus</span> against the host defense response. Therefore, it is also possible that HC-Pro, P3 and <span class="gene">CI</span> may operate separately or as a complex with distinct roles for each of them: one as an elicitor to interact with the <italic> R </italic> product and the other(s) as a conditioner to regulate <span class="trait">virus</span> replication and intercellular movement. This may explain why the absence of the avirulent elicitor is insufficient and a complementary virulence factor is required for the gain of virulence ( <xref ref-type="bibr" rid="pone.0028342-Eggenberger1"> [31] </xref> , <xref ref-type="bibr" rid="pone.0028342-Hajimorad6"> [43] </xref> , this study). The functional roles of these viral proteins are beyond current understanding.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean homologs of MPK4 negatively regulate defense responses and positively regulate growth and development.</h2>
			<h3>PMID:21878550  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu JZ(1), Horstman HD, Braun E, Graham MA, Zhang C, Navarre D, Qiu WL, Lee Y, Nettleton D, Hill JH, Whitham SA.
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mitogen-activated protein kinase (MAPK) cascades play important roles in disease <span class="trait">resistance</span> in model plant species such as Arabidopsis (Arabidopsis thaliana) and tobacco (Nicotiana tabacum). However, the importance of MAPK signaling pathways in the disease <span class="trait">resistance</span> of crops is still largely uninvestigated. To better understand the role of MAPK signaling pathways in disease <span class="trait">resistance</span> in soybean (Glycine max), 13, nine, and 10 genes encoding distinct MAPKs, MAPKKs, and MAPKKKs, respectively, were silenced using virus-induced gene silencing mediated by Bean pod mottle <span class="trait">virus.</span> Among the plants silenced for various MAPKs, MAPKKs, and MAPKKKs, those in which <span class="gene">GmMAPK4</span> homologs <span class="gene">(GmMPK4s)</span> were silenced displayed strong phenotypes including stunted stature and spontaneous cell death on the leaves and stems, the characteristic hallmarks of activated defense responses. Microarray analysis showed that genes involved in defense responses, such as those in salicylic acid (SA) signaling pathways, were significantly up-regulated in <span class="gene">GmMPK4-silenced</span> plants, whereas genes involved in growth and development, such as those in auxin signaling pathways and in cell cycle and proliferation, were significantly down-regulated. As expected, SA and hydrogen peroxide accumulation was significantly increased in <span class="gene">GmMPK4-silenced</span> plants. Accordingly, <span class="gene">GmMPK4-silenced</span> plants were more resistant to downy mildew and Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> compared with vector control plants. Using bimolecular fluorescence complementation analysis and in vitro kinase assays, we determined that <span class="gene">GmMKK1</span> and <span class="gene">GmMKK2</span> might function upstream of <span class="gene">GmMPK4.</span> Taken together, our results indicate that <span class="gene">GmMPK4s</span> negatively regulate SA accumulation and defense response but positively regulate plant growth and development, and their functions are conserved across plant species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Amino acid changes in P3, and not the overlapping pipo-encoded protein, determine virulence of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> on functionally immune Rsv1-genotype soybean.</h2>
			<h3>PMID:21726381  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wen RH(1), Maroof MA, Hajimorad MR.
				<br/> 
				年份：2011 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A small open reading frame, termed 'pipo', is embedded in the P3 cistron of potyviruses. Currently, knowledge on pipo and its role(s) in the life cycle of potyviruses is limited. The P3 and helper-component proteinase (HC-Pro) cistrons of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) harbour determinants affecting virulence on functionally immune Rsv1-genotype soybeans. Interestingly, a key virulence determinant of SMV on Rsv1-genotype soybeans (i.e. soybeans containing the Rsv1 <span class="trait">resistance</span> gene) that resides at <span class="gene">polyprotein</span> codon 947 overlaps both P3 and a pipo-encoded codon. This raises the question of whether PIPO or P3 is the virulence factor. To answer this question, the corresponding pipo of an avirulent and two virulent strains of SMV were studied by comparative genomics, followed by syntheses and analyses of site-directed mutants. Our data demonstrate that the virulence of SMV on Rsv1-genotype soybeans is affected by P3 and not the overlapping pipo-encoded protein.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multiple loci condition seed transmission of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) and SMV-induced seed coat mottling in soybean.</h2>
			<h3>PMID:21561316  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Domier LL(1), Hobbs HA, McCoppin NK, Bowen CR, Steinlage TA, Chang S, Wang Y, Hartman GL.
				<br/> 
				年份：2011 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Infection of soybean plants with Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), which is transmitted by aphids and through seed, can cause significant reductions in seed production and quality. Because seedborne infections are the primary sources of inoculum for SMV infections in North America, host-plant <span class="trait">resistance</span> to seed transmission can limit the pool of plants that can serve as sources of inoculum. To examine the inheritance of SMV seed transmission in soybean, crosses were made between plant introductions (PIs) with high (PI88799), moderate (PI60279), and low (PI548391) rates of transmission of SMV through seed. In four F(2) populations, SMV seed transmission segregated as if conditioned by two or more genes. Consequently, a recombinant inbred line population was derived from a cross between PIs 88799 and 548391 and evaluated for segregation of SMV seed transmission, seed coat mottling, and simple sequence repeat markers. Chromosomal regions on linkage groups <span class="gene">C1</span> and <span class="gene">C2</span> were significantly associated with both transmission of isolate SMV 413 through seed and SMV-induced seed coat mottling, and explained ≈42.8 and 46.4% of the variability in these two traits, respectively. Chromosomal regions associated with seed transmission and seed coat mottling contained homologues of Arabidopsis genes DCL3 and RDR6, which encode enzymes involved in RNA-mediated transcriptional and posttranscriptional gene silencing.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interaction between potyvirus P3 and ribulose-1,5-bisphosphate carboxylase/oxygenase (RubisCO) of host plants.</h2>
			<h3>PMID:21400205  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin L(1), Luo Z, Yan F, Lu Y, Zheng H, Chen J.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The P3 protein encoded by Shallot yellow stripe <span class="trait">virus</span> onion isolate (SYSV-O) interacted in the Yeast Two-hybrid (Y2H) system and in co-immunoprecipitation (Co-IP) assays with the large subunit of the ribulose-1,5-bisphosphate carboxylase/oxygenase (RubisCO) protein that is encoded by the <span class="gene">rbcL</span> gene of its onion host. Dissection analysis by Y2H showed that the main part of SYSV P3 (amino acids 1-390) and onion RbcL (amino acids 1-137) were responsible for the interaction. The P3 proteins encoded by Onion yellow dwarf <span class="trait">virus</span> (OYDV), Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> Pinellia isolate (SMV-P), and Turnip <span class="trait">mosaic</span> <span class="trait">virus</span> (TuMV) also interacted with RbcL, suggesting that a P3/RbcL interaction might exist generally for potyviruses. An interaction between P3 of these potyviruses and the small subunit of RubisCO (RbcS) was also demonstrated. Moreover, the P3N-PIPO protein encoded by a newly identified open reading frame embedded within the P3 cistron also interacted with both RbcL and RbcS. It is possible that the potyvirus P3 protein affects the normal functions of RubisCO which thus contributes to symptom development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping and analyses of R ( SC8 ) <span class="trait">resistance</span> candidate genes to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> in soybean.</h2>
			<h3>PMID:20981404  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang D(1), Ma Y, Yang Y, Liu N, Li C, Song Y, Zhi H.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) in soybean [Glycine max (L.) Merr.] is a destructive foliar disease in soybean-producing countries worldwide. In this study, F(2), F(2:3), and F(7:11) recombinant inbred lines populations derived from Kefeng No.1 × Nannong 1138-2 were used to study inheritance and linkage mapping of the SMV strain SC8 <span class="trait">resistance</span> gene in Kefeng No.1. Results indicated that a single dominant gene (designated R(SC8)) controls <span class="trait">resistance,</span> which is located on chromosome 2 (MLG D1b). A mixed segregating population was developed by selfing two heterozygous plants (RHL153-1 and RHL153-2) at four markers adjacent to the locus and used in fine mapping R(SC8). In addition, two genomic-simple sequence repeats (SSR) markers BARCSOYSSR_02_0610 and BARCSOYSSR_02_0616 were identified that flank the two sides of R(SC8). Sequence analysis of the soybean genome indicated that the interval between the two genomic-SSR markers is 200 kb. QRT-PCR analysis of the candidate genes determined that five genes <span class="gene">(Glyma02g13310,</span> 13320, 13400, 13460, and 13470) are likely involved in soybean SMV <span class="trait">resistance.</span> These results will have utility in cloning, transferring, and pyramiding of the R(SC8) through marker-assisted selection in soybean breeding programs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A single amino acid change in HC-Pro of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> alters symptom expression in a soybean cultivar carrying Rsv1 and Rsv3.</h2>
			<h3>PMID:20938695  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo JK(1), Sohn SH, Kim KH.
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is generally believed that infidelity of RNA <span class="trait">virus</span> replication combined with R-gene-driven selection is one of the major evolutionary forces in overcoming host <span class="trait">resistance.</span> In this study, we utilized an avirulent soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) mutant to examine the possibility of emergence of mutant viruses capable of overcoming R-gene-mediated <span class="trait">resistance</span> during serial passages. Interestingly, we found that the emerged progeny <span class="trait">virus</span> induced severe rugosity and local necrotic lesions in Jinpumkong-2 (Rsv1 + Rsv3) plants, while SMV-G7H provoked a lethal systemic hypersensitive response. Genome sequence analysis of the emerged progeny <span class="trait">virus</span> revealed that the mutation in <span class="gene">CI</span> that had caused SMV-G7H to lose its virulence was restored to the original sequence, and a single amino acid was newly introduced into HC-Pro, which means that the symptom alteration was due to this single amino acid mutation in HC-Pro. Our results suggest that SMV HC-Pro functions as a symptom determinant in the SMV-soybean pathosystem.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multiplex single nucleotide polymorphism (SNP) assay for detection of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> <span class="trait">resistance</span> genes in soybean.</h2>
			<h3>PMID:20931170  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shi A(1), Chen P, Vierling R, Zheng C, Li D, Dong D, Shakiba E, Cervantez I.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) is one of the most destructive viral diseases in soybean (Glycine max). Three independent loci for SMV <span class="trait">resistance</span> have been identified in soybean germplasm. The use of genetic <span class="trait">resistance</span> is the most effective method of controlling this disease. Marker assisted selection (MAS) has become very important and useful in the effort of selecting genes for SMV <span class="trait">resistance.</span> Single nucleotide polymorphism (SNP), because of its abundance and high-throughput potential, is a powerful tool in genome mapping, association studies, diversity analysis, and tagging of important genes in plant genomics. In this study, a 10 SNPs plus one insert/deletion (InDel) multiplex assay was developed for SMV <span class="trait">resistance:</span> two SNPs were developed from the candidate gene <span class="gene">3gG2</span> at Rsv1 locus, two SNPs selected from the clone N11PF linked to Rsv1, one 'BARC' SNP screened from soybean chromosome 13 [linkage group (LG) F] near Rsv1, two 'BARC' SNPs from probe A519 linked to Rsv3, one 'BARC' SNP from chromosome 14 (LG B2) near Rsv3, and two 'BARC' SNPs from chromosome 2 (LG D1b) near Rsv4, plus one InDel marker from expressed sequence tag (EST) AW307114 linked to Rsv4. This 11 SNP/InDel multiplex assay showed polymorphism among 47 diverse soybean germplasm, indicating this assay can be used to investigate the mode of inheritance in a SMV resistant soybean line carrying Rsv1, Rsv3, and/or Rsv4 through a segregating population with phenotypic data, and to select a specific gene or pyramid two or three genes for SMV <span class="trait">resistance</span> through MAS in soybean breeding program. The presence of two SMV <span class="trait">resistance</span> genes (Rsv1 and Rsv3) in J05 soybean was confirmed by the SNP assay.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Near infrared spectroscopy and aquaphotomics: Novel approach for rapid in vivo diagnosis of <span class="trait">virus</span> infected soybean.</h2>
			<h3>PMID:20570650  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jinendra B(1), Tamaki K, Kuroki S, Vassileva M, Yoshida S, Tsenkova R.
				<br/> 
				年份：2010 Jul 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Near infrared spectroscopy with aquaphotomics as a novel approach was assessed for the diagnosis of soybean plants (Glycine max) infected with soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) at latent symptomless stage of the disease. Near infrared (NIR) leaf spectra (in the range of 730-1025nm) acquired from soybean plants with and without the inoculation of SMV were used. Leaf samples from all plants were assayed with enzyme-linked immunosorbent assay (ELISA) to confirm the infection. Previously reported <span class="gene">NIR</span> band for water at 970nm and two new bands at 910nm and 936nm in the water specific region of <span class="gene">NIR</span> were found to be markedly sensitive to the SMV infection 2weeks prior to the appearance of visual symptoms on infected leaves. The spectral calibration model soft independent modeling of class analogy (SIMCA), predicted the disease with 91.6% sensitivity and 95.8% specificity when the second order derivative of the individual plant averaged spectra were used. The study shows the potential of <span class="gene">NIR</span> spectroscopy with its novel approach to elucidate latent biochemical and biophysical information of an infection as it allowed successful discrimination of SMV infected plant from healthy at the early symptomless stage of the disease.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2010 Elsevier Inc. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mutational analysis of the putative pipo of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> suggests disruption of PIPO protein impedes movement.</h2>
			<h3>PMID:20170935  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wen RH(1), Hajimorad MR.
				<br/> 
				年份：2010 Apr 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The presence of a small open reading frame embedded in the P3 cistron of potyvirus turnip <span class="trait">mosaic</span> <span class="trait">virus,</span> termed "pipo," was recently discovered. We have now studied the putative pipo of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV). Introduction of single, or multiple, stop codon mutations at different locations within pipo, without substitution in <span class="gene">polyprotein</span> amino acids, did not abolish replication, but restricted the <span class="trait">virus</span> to small cluster of cells within the inoculated leaves. Furthermore, extensive mutagenesis of the conserved GA(6) motif at the 5' end of pipo also generated two out of five mutants that remained restricted to small foci of infected cells within the inoculated leaves. Long-distance movement function of the movement-defective PIPO-mutants was not restored following co-inoculation with competent SMV strains. Taken together, the data suggest that the putative pipo of SMV is essential for the <span class="trait">virus</span> movement; however, knock out of its expression does not abolish replication.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2010 Elsevier Inc. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular variability and genetic structure of the population of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> based on the analysis of complete genome sequences.</h2>
			<h3>PMID:19716150  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo JK(1), Ohshima K, Lee HG, Son M, Choi HS, Lee SH, Sohn SH, Kim KH.
				<br/> 
				年份：2009 Oct 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete genomes of 30 Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) isolates and strains were sequenced in this study. Together with fourteen previously reported sequences, we analyzed the genetic structure of the SMV population. Analyses of genetic diversity showed that different genomic regions of SMV are under different evolutionary constraints and that there was no significant genetic differentiation between East Asian and North American populations of SMV. Phylogenetic analyses revealed a significant correlation between phylogeny of the cylindrical inclusion (CI) gene of SMV and SMV <span class="trait">resistance</span> gene 3 (Rsv3)-relating pathogenicity of SMV, suggesting <span class="gene">CI</span> might be a pathogenic determinant in Rsv3-mediated disease response. Interestingly, recombination analyses identified 19 'clear' recombination events in the SMV population. Furthermore, as several resistance-breaking strains were identified as recombinants, it appears that recombination might contribute to overcome host <span class="trait">resistance</span> in SMV-soybean pathosystem. Our finding suggests that recombination as well as mutation is an important evolutionary process in the genetic diversification of SMV population.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Strain-specific cylindrical inclusion protein of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> elicits extreme <span class="trait">resistance</span> and a lethal systemic hypersensitive response in two resistant soybean cultivars.</h2>
			<h3>PMID:19656049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo JK(1), Lee SH, Kim KH.
				<br/> 
				年份：2009 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In the Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV)-soybean pathosystem, three independent genes (Rsv1, Rsv3, and Rsv4) conferring <span class="trait">resistance</span> to SMV have been identified. Recently, we constructed infectious cDNA clones of SMV G7H and G5H strains and found that these two strains differ in their ability to infect soybean genotypes possessing different SMV <span class="trait">resistance</span> genes despite a difference of only 33 amino acids. In particular, pSMV-G7H induced <span class="trait">mosaic</span> symptoms systemically in L29 (Rsv3) and provoked a lethal systemic hypersensitive response (LSHR) in Jinpumkong-2, whereas pSMV-G5H could not infect these soybean genotypes. To identify the responsible pathogenic determinants of SMV, we exploited the differential responses of pSMV-G7H- and pSMV-G5H-derived chimeric viruses and amino acid substitution mutant viruses in several soybean genotypes and demonstrated that cylindrical inclusion (CI) protein is the elicitor of Rsv3-mediated extreme <span class="trait">resistance</span> and a pathogenic determinant provoking LSHR in Jinpumkong-2. A single amino acid substitution in <span class="gene">CI</span> was found to be responsible for gain or loss of elicitor function of CI. Our finding provides a role for <span class="gene">CI</span> as a pathogenic determinant in the SMV-soybean pathosystem, and increases the understanding of the basis of the different disease responses of SMV strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the soybean <span class="gene">GmERF3</span> gene, an AP2/ERF type transcription factor for increased tolerances to salt, <span class="trait">drought,</span> and diseases in transgenic tobacco.</h2>
			<h3>PMID:19602544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Li L, Xu Z, Chen X, Guo J, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new member of the AP2/ERF transcription factor family, <span class="gene">GmERF3,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF3</span> contained an AP2/ERF domain of 58 amino acids and two putative nuclear localization signal (NLS) domains. It belonged to a group IV protein in the ERF (ethylene response factor) subfamily as typified by a conserved N-terminal motif [MCGGAI(I/L)]. Expression of <span class="gene">GmERF3</span> was induced by treatments with high salinity, <span class="trait">drought,</span> abscisic acid (ABA), salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), whereas there was no significant <span class="gene">GmERF3</span> mRNA accumulation under cold stress treatment. <span class="gene">GmERF3</span> could bind to the GCC box and DRE/CRT element, and was targeted to the nucleus when transiently expressed in onion epidermal cells. The <span class="gene">GmERF3</span> protein fused to the GAL4 DNA-binding domain to activate transcription of reporter genes in yeast. Ectopic expression of the <span class="gene">GmERF3</span> gene in transgenic tobacco plants induced the expression of some PR genes and enhanced <span class="trait">resistance</span> against infection by Ralstonia solanacearum, Alternaria alternata, and tobacco <span class="trait">mosaic</span> <span class="trait">virus</span> (TMV), and gave tolerance to high salinity and dehydration stresses. Furthermore, overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions. The overall results suggested that <span class="gene">GmERF3</span> as an AP2/ERF transcription factor may play dual roles in response to biotic and abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel EAR-motif-containing gene <span class="gene">GmERF4</span> from soybean (Glycine max L.).</h2>
			<h3>PMID:19597961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Chen X, Xu Z, Li L, Guo J, Ma Y.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptional repressors are emerging as central regulators of development and stress responses in different organisms. The ERF-associated amphiphilic repression (EAR) motif was identified as essential for transcriptional repression. To gain a better understanding of this type of protein, we reported here a novel <span class="gene">GmERF4</span> protein from soybean. Sequence alignment showed that <span class="gene">GmERF4</span> contains one AP2/ERF domain, two putative nuclear localization signal regions and one EAR motif. The <span class="gene">GmERF4</span> protein was preferentially localized to the nucleus of onion epidermis cells and bound specifically to the GCC box and DRE/CRT element in vitro. Furthermore, the expression of <span class="gene">GmERF4</span> was induced by ethylene, JA, SA, cold, salt, <span class="trait">drought,</span> and soybean <span class="trait">mosaic</span> <span class="trait">virus,</span> and repressed by ABA. Constitutive expression of <span class="gene">GmERF4</span> in transgenic tobacco plants increased tolerance to salt and <span class="trait">drought</span> stresses compared with wild-type plants, but did not exhibit detectable <span class="trait">resistance</span> against bacterial infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cytoplasmic inclusion cistron of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> serves as a virulence determinant on Rsv3-genotype soybean and a symptom determinant.</h2>
			<h3>PMID:19595406  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Hajimorad MR, Eggenberger AL, Tsang S, Whitham SA, Hill JH.
				<br/> 
				年份：2009 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV; Potyvirus, Potyviridae) is one of the most widespread viruses of soybean globally. Three dominant <span class="trait">resistance</span> genes (Rsv1, Rsv3 and Rsv4) differentially confer <span class="trait">resistance</span> against SMV. Rsv1 confers extreme <span class="trait">resistance</span> and the <span class="trait">resistance</span> mechanism of Rsv4 is associated with late susceptibility. Here, we show that Rsv3 restricts the accumulation of SMV strain G7 to the inoculated leaves, whereas, SMV-N, an isolate of SMV strain G2, establishes systemic infection. This observation suggests that the <span class="trait">resistance</span> mechanism of Rsv3 differs phenotypically from those of Rsv1 and Rsv4. To identify virulence determinant(s) of SMV on an Rsv3-genotype soybean, chimeras were constructed by exchanging fragments between avirulent SMV-G7 and the virulent SMV-N. Analyses of the chimeras showed that both the N- and C-terminal regions of the cytoplasmic inclusion (CI) cistron are required for Rsv3-mediated <span class="trait">resistance.</span> Interestingly, the N-terminal region of <span class="gene">CI</span> is also involved in severe symptom induction in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protein-protein interactions in two potyviruses using the yeast two-hybrid system.</h2>
			<h3>PMID:19189854  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin L(1), Shi Y, Luo Z, Lu Y, Zheng H, Yan F, Chen J, Chen J, Adams MJ, Wu Y.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Interactions between all ten mature proteins of the potyviruses Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (Pinellia ternata isolate) and Shallot yellow stripe <span class="trait">virus</span> were investigated using yeast two-hybrid (Y2H) assays. Consistently strong self-interactions were found between the pairs of HC-Pro, VPg, NIa-Pro, <span class="gene">NIb</span> and CP in both viruses. Apart from the NIb, such interactions have been previously reported for some other potyviruses. The 6K1/NIa-Pro combination gave a consistently moderate to strong interaction in both directions for both viruses. This interaction occurred even when the 6K1 of SMV-P was truncated to eliminate the C-terminal motif that acts as a recognition site for cleavage by the NIa-Pro. Many other interactions occurred only in one direction or only for one of the two viruses. When taken together with other published reports, the data suggest that interactions detected by Y2H should be regarded as only preliminary indications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmRAR1 and <span class="gene">GmSGT1</span> are required for basal, R gene-mediated and systemic acquired <span class="trait">resistance</span> in soybean.</h2>
			<h3>PMID:19061405  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fu DQ(1), Ghabrial S, Kachroo A.
				<br/> 
				年份：2009 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						RAR1, SGT1, and <span class="gene">HSP90</span> are important components of effector-triggered immunity (ETI) in diverse plants, where RAR1 and SGT1 are thought to serve as <span class="gene">HSP90</span> co-chaperones. We show that ETI in soybean requires RAR1 and SGT1 but not HSP90. Rsv1-mediated extreme <span class="trait">resistance</span> to Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) and Rpg-1b-mediated <span class="trait">resistance</span> to Pseudomonas syringae were compromised in plants silenced for <span class="gene">GmRAR1</span> and <span class="gene">GmSGT1-2</span> but not <span class="gene">GmHSP90.</span> This suggests that RAR1- or SGT1-dependant signaling is not always associated with a dependence on HSP90. Unlike in Arabidopsis, SGT1 in soybean also mediates ETI against the bacterial pathogen P. syringae. Similar to Arabidopsis, soybean RAR1 and SGT1 proteins interact with each other and two related <span class="gene">HSP90</span> proteins. Plants silenced for <span class="gene">GmHSP90</span> genes or <span class="gene">GmRAR1</span> exhibited altered morphology, suggesting that these proteins also contribute to developmental processes. Silencing <span class="gene">GmRAR1</span> and <span class="gene">GmSGT1-2</span> impaired <span class="trait">resistance</span> to virulent bacteria and systemic acquired <span class="trait">resistance</span> (SAR) in soybean as well. Because the Arabidopsis rar1 mutant also showed a defect in SAR, we conclude that RAR1 and SGT1 serve as a point of convergence for basal <span class="trait">resistance,</span> ETI, and SAR. We demonstrate that, although soybean defense signaling pathways recruit structurally conserved components, they have distinct requirements for specific proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Diversity Among Isolates of Bean pod mottle <span class="trait">virus.</span></h2>
			<h3>PMID:18942958</h3>
				

				
			<p>
				作者：Gu H, Clark AJ, de Sá PB, Pfeiffer TW, Tolin S, Ghabrial SA.
				<br/> 
				年份：2002 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						ABSTRACT Isolates of Bean pod mottle <span class="trait">virus</span> (BPMV), a member of the genus Comovirus, collected from soybean fields in Kentucky, Virginia, Arkansas, and Iowa were classified into two distinct subgroups, I and II, based on nucleic acid hybridization analysis using cloned cDNA probes to RNA-1 from BPMV strains K-G7 and K-Ha1. Slot blot hybridization analysis using cloned cDNA probes to RNA-2 from the same two strains (K-G7 and K-Ha1), however, revealed that some of the isolates, initially classified as belonging to subgroup I after analysis with RNA-1 probes, are in fact natural reassortants between the two strain subgroups. This was corroborated by nucleotide sequence analysis of full-length cDNA clones of both RNA-1 and RNA-2 from a putative reassortant strain (K-Ho1). These results indicate that BPMV strain diversity is more complex than initially anticipated, and that the use of cloned probes to both genomic RNAs during nucleic acid hybridization analysis is required to unravel the extent of such diversity. In a field plot experiment, BPMV isolates that belong to distinct strain subgroups induced symptoms that varied in severity and in the level of <span class="trait">yield</span> losses. In this regard, the reassortant strain K-Ho1 caused the most serious damage compared with four other BPMV isolates tested. Furthermore, the soybean alleles Rsv(1) and Rsv(4), known to confer <span class="trait">resistance</span> against Soybean <span class="trait">mosaic</span> <span class="trait">virus,</span> a member of the genus Potyvirus, did not provide any protection against BPMV. Additionally, we developed a reverse transcription-polymerase chain reaction procedure based on the sequence of a highly conserved region in the capsid <span class="gene">polyprotein</span> coding sequence that provides efficient and highly sensitive detection of all BPMV isolates tested, regardless of their strain classification.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and molecular characterization of two naturally occurring Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> isolates that are closely related but differ in their ability to overcome Rsv4 <span class="trait">resistance.</span></h2>
			<h3>PMID:18793685  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gagarinova AG(1), Babu M, Poysa V, Hill JH, Wang A.
				<br/> 
				年份：2008 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A naturally occurring Rsv4 resistance-breaking isolate (L-RB) and a closely related non-resistance-breaking isolate (L) of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) were identified in soybean fields in London, Ontario, Canada. The viral genomes of L and L-RB were completely sequenced. Each isolate has a 9585-nucleotide genome with a single open reading frame encoding a <span class="gene">polyprotein</span> of approximately 350 kDa. L-RB and L have a very high sequence similarity (99.6%) at both the nucleotide and amino acid levels. Phylogenetic analysis showed that the two isolates belong to the G2 pathotype. Pathogenicity predictions of all virus/soybean combinations, based on the phylogenetic profile, were confirmed by pathogenicity tests using L and L-RB isolates and soybeans carrying different <span class="trait">resistance</span> genes, with an exception that L-RB infected a soybean cultivar carrying Rsv4 <span class="trait">resistance.</span> The temporal and spatial proximity of L and L-RB and their high sequence similarity suggest L-RB was likely derived from the SMV-L quasispecies. Recombination analysis did not reveal the evidence of genetic recombination for the emergence of L-RB. Mutations introduced by virus-encoded RNA-dependent RNA polymerase during viral genome replication and selection pressure probably contributed to the occurrence of L-RB.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A potyvirus <span class="gene">P1</span> protein interacts with the Rieske Fe/S protein of its host.</h2>
			<h3>PMID:20507538</h3>
				

				
			<p>
				作者：Shi Y(1), Chen J, Hong X, Chen J, Adams MJ.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						ABSTRACT Yeast two-hybrid (Y2H) screens were used to test for interactions between the <span class="gene">P1</span> protein of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> Pinellia isolate (SMV-P) and a cDNA expression library of its host, the aroid Pinellia ternata. Of the 13 independent interacting clones identified, ten were identical and had an open reading frame predicted to encode a 23.7-kDa protein closely related to the cytochrome b6/f complex Rieske Fe/S genes of plants. The interaction between SMV-P-P1 and the mature Rieske Fe/S protein (without transit peptide) of the host was confirmed by in vitro co-immunoprecipitation of the two proteins. Y2H assays using different parts of the two proteins showed that only the N-terminal part (amino acids 1-82) of SMV-P <span class="gene">P1</span> was responsible for the interaction with the Rieske Fe/S protein and that amino acids 1-33 interacted only with the transit peptide, while amino acids 34-82 could interact with the entire Rieske Fe/S protein. SMV-P <span class="gene">P1</span> also interacted moderately with the Rieske Fe/S protein of its other hosts, soybean and Zantedeschia aethiopica, but weakly with that of the non-host Arabidopsis thaliana. The P1-Rieske Fe/S protein interactions are likely to be involved in symptom development, and the very variable N-terminus of <span class="gene">P1</span> may play an important role in host adaptation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inheritance analysis of herbicide-resistant transgenic soybean lines.</h2>
			<h3>PMID:17185170  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang Y(1), Yang BY, Chen SY.
				<br/> 
				年份：2006 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Four transgenic soybean lines generated via Agrobacterium-mediated transformation were used to analyze inheritance of the transgenes. Seed chip GUS assay and herbicide leaf painting and spraying assays were applied to test the gus reporter gene and the herbicide resistant <span class="gene">bar</span> selectable marker gene, respectively. Three of the four transgenic soybean lines were stably inherited in a Mendelian fashion with co-segregation of both transgenes in a 3:1 segregation ratio in the T(1) progeny, indicating that both transgenes were integrated into the same locus of the soybean genome. Homozygous transgenic progeny plants were obtained in the T(2) generation of these lines, and the transgenes were inherited in five successive generations. However, in one transgenic line, all the T(1) progeny plants showed GUS negative and herbicide sensitive. Southern blotting analysis confirmed that the transgenes were passed into the T(1) progeny, indicating that the transgenes were both silenced. To test if the transgene silencing was due to transcriptional or post-transcriptional level, Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) was inoculated on leaf tissues of the T(1) plants to test possible reverse effects on transgene silencing. Infection with SMV did not suppress transgene silencing, suggesting that transgene silencing in this transgenic line may not be due to post-transcriptional gene silencing.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete nucleotide sequence of a Pakistani isolate of Watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> provides further insights into the taxonomic status in the Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> subgroup.</h2>
			<h3>PMID:16732483  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ali A(1), Natsuaki T, Okuda S.
				<br/> 
				年份：2006 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> (WMV) is a potyvirus with a worldwide distribution, but is mostly found in temperate and Mediterranean regions. The complete nucleotide (nt) sequence of a Pakistani isolate of WMV (WMV-Pk) was determined and compared with French isolate (WMV-Fr) and other closely related potyviruses. WMV-Pk showed overall identities of 94.4% (nt) and 96% (amino acid; aa) with the WMV-Fr. However, variability was observed in the 5' UTR and <span class="gene">P1</span> region. Although sequence identities over most of the genome were well above 90% at both the nt and aa levels, reaching 99.6% (aa) in the CP and 100% (aa) in the 6K1 and 6K2, thereby suggesting that WMV-Pk and WMV-Fr are identical strains, but the sequence identities in the <span class="gene">P1</span> region were only 80.6% (aa) and 82.8% (nt), while that in the 5' UTR was 82%. These differences may be due to different mutation phenomena of a common ancestor <span class="trait">virus</span> or mutations caused by different selection pressures in two different agro-ecological zones. The sequence of WMV-Pk is very close to that of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) over most of the genome, except for the N-terminal region, which is subject to recombination between SMV and Peanut stripe <span class="trait">virus</span> (PSV)/Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> (BCMV), as revealed by Simplot and phylogenetic analyses of N- and C-terminal P1, HC-Pro, and 5' UTR regions of the genome.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The genomic sequence of Wisteria vein <span class="trait">mosaic</span> <span class="trait">virus</span> and its similarities with other potyviruses.</h2>
			<h3>PMID:16732423  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liang WX(1), Song LM, Tian GZ, Li HF, Fan ZF.
				<br/> 
				年份：2006 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete nucleotide sequence of a Beijing isolate of Wisteria vein <span class="trait">mosaic</span> <span class="trait">virus</span> was determined to be 9695 nucleotides in length excluding the poly(A) tail. Sequence analysis predicted a single large open reading frame of 9279 nucleotides potentially encodes a <span class="gene">polyprotein</span> of 3092 amino acids. Phylogenetic analysis based on the genomic and deduced amino acid sequences support the current status of Wisteria vein <span class="trait">mosaic</span> <span class="trait">virus</span> (WVMV) as a distinct <span class="trait">virus</span> of the genus Potyvirus and a member of the Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> (BCMV) subgroup. Sequence comparisons of WVMV and other members of the BCMV subgroup showed that WVMV is most closely related to both soybean <span class="trait">mosaic</span> <span class="trait">virus</span> and watermelon <span class="trait">mosaic</span> <span class="trait">virus.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pseudomonas syringae effector avrB confers soybean cultivar-specific avirulence on Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> adapted for transgene expression but effector avrPto does not.</h2>
			<h3>PMID:16570660  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang L(1), Eggenberger A, Hill J, Bogdanove AJ.
				<br/> 
				年份：2006 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) was adapted for transgene expression in soybean and used to examine the function of avirulence genes avrB and avrPto of Pseudomonas syringae pvs. glycinea and tomato, respectively. A cloning site was introduced between the <span class="gene">P1</span> and HC-Pro genes in 35S-driven infectious cDNAs of strains SMV-N and SMV-G7. Insertion of the uidA gene or the green fluorescent protein gene into either modified cDNA and bombardment into primary leaves resulted in systemic expression that reflected the pattern of viral movement into uninoculated leaves. Insertion of avrB blocked symptom development and detectable viral movement in cv. Harosoy, which carries the Rpg1-b <span class="trait">resistance</span> gene corresponding to avrB, but not in cvs. Keburi or Hurrelbrink, which lack Rpg1-b. In Keburi and Hurrelbrink, symptoms caused by SMV carrying avrB appeared more quickly and were more severe than those caused by the <span class="trait">virus</span> without avrB. Insertion of avrPto enhanced symptoms in Harosoy, Hurrelbrink, and Keburi. This result was unexpected because avrPto was reported to confer avirulence on P. syringae pv. glycinea inoculated to Harosoy. We inoculated Harosoy with P syringae pv. glycinea expressing avrPto, but observed no hypersensitive reaction, avrPto-dependent induction of pathogenesis-related protein la, or limitation of bacterial population growth. In Hurrelbrink, avrPto enhanced bacterial multiplication and exacerbated symptoms. Our results establish SMV as an expression vector for soybean. They demonstrate that <span class="trait">resistance</span> triggered by avrB is effective against SMV, and that avrB and avrPto have general virulence effects in soybean. The results also led to a reevaluation of the reported avirulence activity of avrPto in this plant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection and characterisation of a second potyvirus from Thunberg fritillary in China.</h2>
			<h3>PMID:16328133  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen J(1), Zheng HY, Shi YH, Adams MJ, Wei CB, Lin L, Chen JP.
				<br/> 
				年份：2006 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants of Thunberg fritillary (Fritillaria thunbergii Miq.) from Zhejiang Province, were found to be co-infected with two distinct potyviruses. One was an isolate of the recently reported Thunberg fritillary <span class="trait">mosaic</span> <span class="trait">virus</span> (TFMV; Wei et al., (2005) Arch Virol 150: 1271-1280), while the other was a distinct <span class="trait">virus</span> that did not react with TFMV antiserum nor with antisera to 17 other potyviruses, except for a weak reaction with antibodies produced to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) Pinellia strain. Both viruses could be transmitted mechanically to their original host but not to any of a range of commonly used indicator plants. No local lesion host was identified that would enable the viruses to be propagated independently. The complete sequences of both viruses were determined; that of the new <span class="trait">virus</span> (9656 nt) had the typical genome organisation and recognised sequence motifs of a potyvirus, encoding a putative <span class="gene">polyprotein</span> of 351 kDa. Phylogenetic analysis, sequence comparisons, and the pattern of <span class="gene">polyprotein</span> cleavage sites all indicated that it was a member of the Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> subgroup. The most closely related species are Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> and Wisteria vein <span class="trait">mosaic</span> <span class="trait">virus,</span> with 68-69% amino acid identity between their polyproteins. This is sufficiently different for the new <span class="trait">virus</span> to be regarded as a distinct species, which we have tentatively named Fritillary <span class="trait">virus</span> Y.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> helper component-protease enhances somatic embryo production and stabilizes transgene expression in soybean.</h2>
			<h3>PMID:16316753  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lim HS(1), Ko TS, Lambert KN, Kim HG, Korban SS, Hartman GL, Domier LL.
				<br/> 
				年份：2005 Oct-Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) helper component protease (HC-Pro), a suppressor of post-transcriptional gene silencing, was evaluated for its ability to enhance production of soybean hygromycin-resistant somatic embryos (HR-SEs), and stabilize transgene expression. Immature soybean cotyledonary explants were co-cultured with Agrobacterium tumefaciens strain KYRT1 harboring either pCAMBIA1302, carrying a hygromycin phosphotransferase gene (hpt) and a gene encoding green fluorescent protein; pCAMBIA1305.1, carrying <span class="gene">hpt</span> and beta-glucuronidase (uidA) genes; pG2-HC-Pro, a derivative of pCAMBIA1305.1 containing SMV G2 HC-Pro; or pG5-HC-Pro, a derivative of pCAMBIA1305.1 containing SMV G5 HC-Pro, but lacking uidA. Significantly (rho&lt;0.02) higher numbers of HR-SEs were obtained from explants transformed with Agrobacterium harboring either pG2-HC-Pro or pG5-HC-Pro than with either of the vector controls (pCAMBIA1302 or pCAMBIA1305.1). Beta-glucuronidase (GUS) expression was significantly (rho&lt;0.003) higher in 50-day-old transgenic plants expressing GUS along with SMV-HC-Pro and in SMV-infected GUS transgenic plants than in transgenic plants expressing GUS alone. Together, these data suggest that SMV-HC-Pro enhanced recovery of HR-SEs by suppressing silencing of the hygromycin phosphotransferase gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of Bean pod mottle virus-based vectors for stable protein expression and sequence-specific virus-induced gene silencing in soybean.</h2>
			<h3>PMID:16226780  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Ghabrial SA.
				<br/> 
				年份：2006 Jan 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant virus-based vectors provide valuable tools for expression of foreign proteins in plants and for gene function studies. None of the presently available <span class="trait">virus</span> vectors is suitable for use in soybean. In the present study, we produced Bean pod mottle <span class="trait">virus</span> (BPMV)-based vectors that are appropriate for gene expression and virus-induced gene silencing (VIGS) in soybean. The genes of interest were inserted into the RNA2-encoded <span class="gene">polyprotein</span> open reading frame between the movement protein (MP) and the large coat protein (L-CP) coding regions. Additional proteinase cleavage sites were created to flank the foreign protein by duplicating the MP/L-CP cleavage site. To minimize the chances of homologous recombination and thus insert instability, we took advantage of the genetic code degeneracy and altered the nucleotide sequence of the duplicated regions without affecting amino acid sequences. The recombinant BPMV constructs were stable following several serial passages in soybean and relatively high levels of protein expression were attained. Successful expression of several proteins with different biological activities was demonstrated from the BPMV vector. These included the reporter proteins GFP and DsRed, phosphinothricin acetyltransferase (encoded by the herbicide <span class="trait">resistance</span> <span class="gene">bar</span> gene), and the RNA silencing suppressors encoded by Tomato bushy stunt <span class="trait">virus,</span> Turnip crinkle <span class="trait">virus,</span> Tobacco etch <span class="trait">virus,</span> and Soybean <span class="trait">mosaic</span> <span class="trait">virus.</span> The possible use of BPMV as a VIGS vector to study gene function in soybean was also demonstrated with the phytoene desaturase gene. Our results suggest that the BPMV-based vectors are suitable for expression of foreign proteins in soybean and for functional genomics applications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Emergence of Rsv-resistance breaking Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> isolates from Korean soybean cultivars.</h2>
			<h3>PMID:15869819  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choi BK(1), Koo JM, Ahn HJ, Yum HJ, Choi CW, Ryu KH, Chen P, Tolin SA.
				<br/> 
				年份：2005 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Twelve Rsv resistance-breaking (RB) isolates of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) were obtained from field-grown soybean plants showing <span class="trait">mosaic</span> symptoms and subsequently examined biologically and molecularly. All of these RB isolates were identified as SMV based on serological and infectivity assays, and the amplification of <span class="gene">P1</span> gene products by reverse transcription-polymerase chain reaction (RT-PCR). Differential soybean cultivars, lines or accessions Lee 68 (rsv), PI 96983, York, Marshall, Ogden, Kwanggyo, Suweon 97 (Rsv1 alleles), L29 (Rsv3), and V94-5152 (Rsv4), following inoculation with each RB isolate, showed similar systemic symptoms suggesting that these RB isolates can overcome Rsv <span class="trait">resistance</span> at three loci. To differentiate the 12 RB isolates molecularly, the <span class="gene">P1</span> coding region for each isolate was amplified, cloned, sequenced and compared to known SMV strains. The <span class="gene">P1</span> region from the RB isolates shared 86-90% and 90-99% similarities in amino acid (aa) and nucleotide sequence, respectively, with known SMV strains. Comparison of aa sequences indicated that these RB isolates are newly emerging isolates capable of breaking Rsv <span class="trait">resistance.</span> Phylogenetic analysis further suggested that the RB isolates can be classified as three major types. However, recombination was not observed within the coding region of <span class="gene">P1</span> protein among the types. This is the first report on the emergence of SMV isolates capable of overcoming all of the known <span class="trait">resistance</span> alleles at the Rsv1 locus, as well as distinct <span class="trait">resistance</span> genes at Rsv3 and Rsv4.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and characterization of an HDZip I gene <span class="gene">GmHZ1</span> from soybean.</h2>
			<h3>PMID:15754189  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang YJ(1), Li YD, Luo GZ, Tian AG, Wang HW, Zhang JS, Chen SY.
				<br/> 
				年份：2005 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						By using cDNA-AFLP, we analyzed a recombinant inbred line population of soybean that was derived from a soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) resistant cultivar Kefeng No.1 and a susceptible cultivar Nannong 1138-2. One hundred and eight fragments showing polymorphism between SMV resistant and susceptible pools were identified. One fragment w27 was 96 bp in length and showed homology to homeobox ggth with a coding region of 738 bp, encoding a protein of 245 amino acids. The genomic sequence analysis defined an intron of 521 bp in the coding region. <span class="gene">GmHZ1</span> was characterized by the presence of a homeodomain (HD) with a closely linked leucine zipper motif (Zip). Southern blot analysis indicated that there was a single copy of <span class="gene">GmHZ1</span> in the soybean genome. When inoculated with SMV strain N3, resistant and susceptible varieties showed reduced and increased expression of the <span class="gene">GmHZ1,</span> respectively. The fusion protein of <span class="gene">GmHZ1</span> with GFP was targeted only in nucleus. Yeast two hybrid studies revealed that the <span class="gene">GmHZ1</span> had transcriptional activation activity and can form homodimer. <span class="gene">GmHZ1</span> can bind two 9-bp pseudopalindromic elements (CAAT(A/T)ATTG and CAAT(C/G)ATTG) with different affinity. Using GUS as a reporter gene, <span class="gene">GmHZ1</span> was proved to be a transcriptional activator and enhanced GUS expression by binding with the two elements in plant cells. These results indicate that the <span class="gene">GmHZ1</span> may have a transcriptional activator function in plant response to SMV infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Further molecular characterisation of potyviruses infecting aroid plants for medicinal use in China.</h2>
			<h3>PMID:15449140  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shi YH(1), Hong XY, Chen J, Adams MJ, Zheng HY, Lin L, Qin BX, Chen JP.
				<br/> 
				年份：2005 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Degenerate primers were used to detect and amplify 3'-terminal genome fragments of potyviruses from medicinal aroid plants growing at 16 sites in China. <span class="trait">Virus</span> was detected in 7 samples of which six, all of Pinellia ternata, contained a strain of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) similar to that previously reported from this host in China. The complete sequence of one isolate and the <span class="gene">P1</span> protein coding region of the other isolates were also sequenced. In all cases, the <span class="gene">P1</span> proteins resembled isolates of Dasheen <span class="trait">mosaic</span> <span class="trait">virus</span> (DsMV) more closely than SMV, confirming earlier suggestions of recombination in this region. In a phylogenetic analysis of SMV, DsMV and related sequences, the aroid sequences of SMV formed a distinct group which also included a sequence published as Zantedeschia symptomless <span class="trait">virus</span> (AF469171). One of the P. ternata samples was also infected with a second potyvirus, the 3'-terminal sequence of which was similar to DsMV and to some sequences published as Vanilla <span class="trait">mosaic</span> <span class="trait">virus.</span> The seventh infected sample was Typhonium flagelliforme and the <span class="trait">virus</span> from it was identified from its sequence as zantedeschia <span class="trait">mosaic</span> <span class="trait">virus</span> (ZaMV), providing the first report of this <span class="trait">virus</span> from mainland China.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A protein interaction map of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> strain G7H based on the yeast two-hybrid system.</h2>
			<h3>PMID:15359133  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kang SH(1), Lim WS, Kim KH.
				<br/> 
				年份：2004 Aug 31
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A protein interaction map of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) strain G7H was generated by the yeast two-hybrid system. Clones encoding the genes P1, HC-Pro, P3, 6K1, CI, 6K2, VPg, NIa, NIb, and CP were fused downstream of the GAL4 binding domain (GAL4-BD) and of the GAL4 activation domain (GAL4-AD). The GAL4-BD and GAL4-AD fusion derivatives of each gene were co-transformed into yeast and transformants in which interaction took place were identified on selective media. Interacting fusion proteins were extracted from the yeast cells, run on SDS-PAGE gels and finally checked by Western blotting with GAL4 polyclonal antibodies. Strong interactions were detected between the pairs CP/CP, HC-Pro/HC-Pro, NIa/NIa, and CP/HC-Pro. Relatively weak but significant interaction was detected between <span class="gene">VPg</span> and NIa. Although not all of the protein-protein interactions previously reported in other potyviruses were detected, the interactions revealed here were, in general, similar to those reported previously.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The nucleotide sequence of Watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> (WMV, Potyvirus) reveals interspecific recombination between two related potyviruses in the 5' part of the genome.</h2>
			<h3>PMID:15290385  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Desbiez C(1), Lecoq H.
				<br/> 
				年份：2004 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> (WMV, Potyvirus) is a potyvirus with a worldwide distribution, mostly in temperate and mediterranean regions. According to the partial sequences that were available, WMV appeared to share high sequence similarity with Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV), and it was almost considered as a strain of SMV in spite of its different and much broader host range. Like SMV, it was also related to legume-infecting potyviruses belonging to the " Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> (BCMV) subgroup". In this paper we obtained the full-length sequence of WMV, and we confirmed that this <span class="trait">virus</span> is very closely related to SMV in most of its genome; however, there is evidence for an interspecific recombination in the <span class="gene">P1</span> protein, as the <span class="gene">P1</span> of WMV was 135 amino-acids longer than that of SMV, and the N-terminal half of the <span class="gene">P1</span> showed no relation to SMV but was 85% identical to BCMV. This suggests that WMV has emerged through an ancestral recombination event, and supports the distinction of WMV and SMV as separate taxonomic units.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Recombination within a nucleotide-binding-site/leucine-rich-repeat gene cluster produces new variants conditioning <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> in soybeans.</h2>
			<h3>PMID:15020438  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hayes AJ(1), Jeong SC, Gore MA, Yu YG, Buss GR, Tolin SA, Maroof MA.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean Rsv1 gene for <span class="trait">resistance</span> to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV; Potyvirus) has previously been described as a single-locus multi-allelic gene mapping to molecular linkage group (MLG) F. Various Rsv1 alleles condition different responses to the seven (G1-G7) described strains of SMV, including extreme <span class="trait">resistance,</span> localized and systemic necrosis, and <span class="trait">mosaic</span> symptoms. We describe the cloning of a cluster of NBS-LRR <span class="trait">resistance</span> gene candidates from MLG F of the virus-resistant soybean line PI96983 and demonstrate that multiple genes within this cluster interact to condition unique responses to SMV strains. In addition to cloning 3gG2, a strong candidate for the major Rsv1 <span class="trait">resistance</span> gene from PI96983, we describe various unique resistant and necrotic reactions coincident with the presence or absence of other members of this gene cluster. Responses of recombinant lines from a high-resolution mapping population of PI96983 (resistant) x Lee 68 (susceptible) demonstrate that more than one gene in this region of the PI96983 chromosome conditions <span class="trait">resistance</span> and/or necrosis to SMV. In addition, the soybean cultivars Marshall and Ogden, which carry other previously described Rsv1 alleles, are shown to possess the <span class="gene">3gG2</span> gene in a NBS-LRR gene cluster background distinct from PI96983. These observations suggest that two or more related non-TIR-NBS-LRR gene products are likely involved in the allelic response of several Rsv1-containing lines to SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="trait">virus</span> related to soybean <span class="trait">mosaic</span> <span class="trait">virus</span> from Pinellia ternata in China and its comparison with local soybean SMV isolates.</h2>
			<h3>PMID:14745600  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen J(1), Zheng HY, Lin L, Adams MJ, Antoniw JF, Zhao MF, Shang YF, Chen JP.
				<br/> 
				年份：2004 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A potyvirus isolated from Pinellia ternata in China was characterised and shown to be related to Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV). The <span class="trait">virus</span> was pathogenic on P. ternata and some soybean cultivars, whereas the local soybean SMV isolate HH5 did not infect P. ternata. Western blot experiments demonstrated a serological relationship between the <span class="trait">virus</span> from Pinellia, SMV and Watermelon <span class="trait">mosaic</span> <span class="trait">virus</span> (WMV). The complete nucleotide sequences of the Pinellia <span class="trait">virus</span> (isolate P-1, 9735 nt) and of the Chinese soybean SMV isolates HH5 (9585 nt) and HZ (9588 nt) were determined. A 1733 nt sequence at the 3'-terminus of a second isolate from Pinellia (isolate P-2) was also determined. The predicted <span class="gene">polyprotein</span> of isolate P-1 has 83% amino acid (aa) identity with those of published SMV sequences. In many parts of the genome, aa identity was about 90% but it was much lower in the <span class="gene">P1</span> protein region (24-29%), where it more closely resembled Dasheen <span class="trait">mosaic</span> <span class="trait">virus</span> (62%). The partial sequence of isolate P-2 had 91% nt identity to P-1 and both isolates resembled a recent sequence in the public databases (AF469171) wrongly named Zantedeschia <span class="trait">mosaic</span> <span class="trait">virus.</span> The two complete SMV soybean sequences had 93-95% nt identity with those of the previously sequenced isolates and &gt;97% amino acid identity. Phylogenetic analysis and comparisons of coat proteins suggest that the Pinellia, WMV and SMV potyviruses should probably be treated as strains of the same species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution of Soybean <span class="trait">mosaic</span> virus-G7 molecularly cloned genome in Rsv1-genotype soybean results in emergence of a mutant capable of evading Rsv1-mediated recognition.</h2>
			<h3>PMID:14554079  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hajimorad MR(1), Eggenberger AL, Hill JH.
				<br/> 
				年份：2003 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant <span class="trait">resistance</span> (R) genes direct recognition of pathogens harboring matching avirluent signals leading to activation of defense responses. It has long been hypothesized that under selection pressure the infidelity of RNA <span class="trait">virus</span> replication together with large population size and short generation times results in emergence of mutants capable of evading R-mediated recognition. In this study, the Rsv1/Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) pathosystem was used to investigate this hypothesis. In soybean line PI 96983 (Rsv1), the progeny of molecularly cloned SMV strain G7 (pSMV-G7) provokes a lethal systemic hypersensitive response (LSHR) with up regulation of a defense-associated gene transcript (PR-1). Serial passages of a large population of the progeny in PI 96983 resulted in emergence of a mutant population (vSMV-G7d), incapable of provoking either Rsv1-mediated LSHR or PR-1 protein gene transcript up regulation. An infectious clone of the mutant (pSMV-G7d) was synthesized whose sequences were very similar but not identical to the vSMV-G7d population; however, it displayed a similar phenotype. The genome of pSMV-G7d differs from parental pSMV-G7 by 17 substitutions, of which 10 are translationally silent. The seven amino acid substitutions in deduced sequences of pSMV-G7d differ from that of pSMV-G7 by one each in <span class="gene">P1</span> proteinase, helper component-proteinase, and coat protein, respectively, and by four in P3. To the best of our knowledge, this is the first demonstration in which experimental evolution of a molecularly cloned plant RNA <span class="trait">virus</span> resulted in emergence of a mutant capable of evading an R-mediated recognition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variability and transmission by Aphis glycines of North American and Asian Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> isolates.</h2>
			<h3>PMID:14551816  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Domier LL(1), Latorre IJ, Steinlage TA, McCoppin N, Hartman GL.
				<br/> 
				年份：2003 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The variability of North American and Asian strains and isolates of Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> was investigated. First, polymerase chain reaction (PCR) products representing the coat protein (CP)-coding regions of 38 SMVs were analyzed for restriction fragment length polymorphisms (RFLP). Second, the nucleotide and predicted amino acid sequence variability of the P1-coding region of 18 SMVs and the helper component/protease (HC/Pro) and CP-coding regions of 25 SMVs were assessed. The CP nucleotide and predicted amino acid sequences were the most similar and predicted phylogenetic relationships similar to those obtained from RFLP analysis. Neither RFLP nor sequence analyses of the CP-coding regions grouped the SMVs by geographical origin. The <span class="gene">P1</span> and HC/Pro sequences were more variable and separated the North American and Asian SMV isolates into two groups similar to previously reported differences in pathogenic diversity of the two sets of SMV isolates. The <span class="gene">P1</span> region was the most informative of the three regions analyzed. To assess the biological relevance of the sequence differences in the HC/Pro and CP coding regions, the transmissibility of 14 SMV isolates by Aphis glycines was tested. All field isolates of SMV were transmitted efficiently by A. glycines, but the laboratory isolates analyzed were transmitted poorly. The amino acid sequences from most, but not all, of the poorly transmitted isolates contained mutations in the aphid transmission-associated DAG and/or KLSC amino acid sequence motifs of CP and HC/Pro, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the 3' terminal region of the genome of Beet <span class="trait">mosaic</span> <span class="trait">virus</span> and its relation with other potyviruses.</h2>
			<h3>PMID:14505096  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Glasa M(1), Kúdela O, Subr Z.
				<br/> 
				年份：2003 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A1633 nucleotides long 3'-terminal sequence of genome of the Beet <span class="trait">mosaic</span> <span class="trait">virus</span> (BtMV) Ren1 isolate from Slovakia, spanning a part of the <span class="gene">NIb</span> gene, complete capsid protein (CP) gene and a 3'non-coding region was determined. Computer-assisted translation resulted in a polypeptide encompassing typical conserved potyviral amino acid (aa) motifs. Homology BLAST search and phylogenetic analysis revealed that BtMV was closest to Bean common <span class="trait">mosaic</span> <span class="trait">virus,</span> Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> and Zucchini yellow <span class="trait">mosaic</span> <span class="trait">virus.</span> An unexpectedly high, 99.6% aa sequence identity was found between BtMV-Ren1 and a previously sequenced British BtMV isolate. Restriction analysis and sequencing of the CP gene of other Slovak BtMV isolates revealed their high homology. Developed reverse transcription-polymerase chain reaction protocols enabled sensitive, rapid and reliable detection of the <span class="trait">virus.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a full-length <span class="trait">resistance</span> gene homolog from soybean.</h2>
			<h3>PMID:12647051  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：He CY(1), Tian AG, Zhang JS, Zhang ZY, Gai JY, Chen SY.
				<br/> 
				年份：2003 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Using mixed <span class="trait">resistance</span> gene analogs as probes, a putative <span class="trait">resistance</span> gene (KR1) was isolated from soybean and characterized further. The <span class="gene">KR1</span> protein consists of a Toll/interleukin receptor (TIR) domain, a nucleotide binding site (NBS) domain, an imperfect leucine-rich repeat (LRR) domain and two C-terminal transmembrane segments. Due to these features, <span class="gene">KR1</span> represents a distinct member in the TIR-NBS-LRR class of <span class="trait">resistance</span> genes. Southern-blot analysis indicated that there were several KR1-related sequences within the soybean genome, and two polymorphic loci were mapped onto linkage group L. <span class="gene">KR1</span> was induced by SA treatment and soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) infection in the resistant line (Kefeng 1). An orthologue (NR1) and a homologue (NR2) of the <span class="gene">KR1</span> gene were also identified in the SMV susceptible-line Nannong1138-2. Sequencing analysis revealed that NR2 was highly homologous to <span class="gene">KR1</span> and NR1, but had a 21-bp deletion. Moreover, the NR1, NR2 transcription and the ratio of NR1/ NR2 was up-regulated by viral infection in Nannong1138-2. These results indicated the complexity of the regulatory mechanism in the plant responses to SMV infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family.</h2>
			<h3>PMID:11402861  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen J(1), Chen J, Adams MJ.
				<br/> 
				年份：2001
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A universal primer (Sprimer: 5'-GGX AAY AAY AGY GGX CAZ CC-3', X = A, G, C or T; Y = T or C; Z = A or G), designed from the consensus sequences that code for the conserved sequence GNNSGQP in the <span class="gene">NIb</span> region of members of the family Potyviridae, was used to amplify by RT-PCR the 3'-terminal genome regions from infected plant samples representing 21 different viruses in the family. Sequencing of some of the fragments (c. 1.7 kb) showed that the type strain (ATTC PV-107) of Oat necrotic mottle <span class="trait">virus</span> is not a distinct species in the genus Rymovirus, but is synonymous with Brome streak <span class="trait">mosaic</span> <span class="trait">virus</span> (genus Tritimovirus) and that Celery <span class="trait">mosaic</span> <span class="trait">virus</span> is a distinct member of the genus Potyvirus not closely related to any other sequenced species. Potyviruses infecting crops in China were also investigated, showing that viruses on cowpea and maize in Hangzhou, Zhejiang province were respectively Bean common <span class="trait">mosaic</span> <span class="trait">virus</span> and Sugarcane <span class="trait">mosaic</span> <span class="trait">virus</span> and that one on garlic in Nanjing, Jiangsu province was Onion yellow dwarf <span class="trait">virus.</span> Fragments were also sequenced from Chinese isolates of Lettuce <span class="trait">mosaic</span> <span class="trait">virus</span> and Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (from Hangzhou), Turnip <span class="trait">mosaic</span> <span class="trait">virus</span> (2 different isolates from Zhejiang province) and RNA1 of Wheat yellow <span class="trait">mosaic</span> <span class="trait">virus</span> (from Rongcheng, Shandong province).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete nucleotide sequence and genome organization of bean common <span class="trait">mosaic</span> <span class="trait">virus</span> (NL3 strain).</h2>
			<h3>PMID:8607279  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fang GW(1), Allison RF, Zambolim EM, Maxwell DP, Gilbertson RL.
				<br/> 
				年份：1995 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nucleotide sequences of 3 cDNA clones corresponding to entire RNA genome of bean common <span class="trait">mosaic</span> <span class="trait">virus</span> NL3 strain have been determined. The RNA is 9612 nucleotides long, excluding a 3'-terminal poly(A) tail. A putative start codon located at nucleotide positions 170-172 initiates one large open reading frame that is terminated with a UAA codon at position 9368-9370. The predicted <span class="gene">polyprotein</span> has 3066 amino acids and an M(r) of 340.3 kDa. The positions of putative protein cleavage sites have been determined by analogy to consensus sequences in other potyviruses. The nucleotide sequences of the non-translated regions and the predicted amino acid sequences of BCMV NL3, were compared with those of other potyviruses. Comparison of the BCMV NL3 proteins with those of other potyviruses indicated a similar genomic organization, and high percentage of amino acid sequence identity in the cylindrical inclusion protein, nuclear inclusion 'b' protein and coat protein. BCMV NL3 displays the highest amino acid sequence identity with soybean <span class="trait">mosaic</span> <span class="trait">virus.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:cDNA cloning and sequence analysis of <span class="gene">NIb</span> gene of soybean <span class="trait">mosaic</span> <span class="trait">virus.</span></h2>
			<h3>PMID:7755870  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu J(1), Peng X, Mang K.
				<br/> 
				年份：1995 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						cDNA of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (Beijing isolate, SMV-BJ) has been synthesized, using viral genomic RNA as a template and random hexanucleotides as primers. Based on the sequences of SMV-BJ coat protein (CP) gene as well as SMV- and WMV-II-related regions, oligonucleotides were made as primers for polymerase chain reaction (PCR). <span class="gene">NIb</span> gene of SMV-BJ was amplified by PCR, and cloned into pBluescript SK. The complete sequence was determined. The comparison of <span class="gene">NIb</span> genes between SMV-BJ and WMV-II (USA) shows that similarities for nucleotide sequence reach 80.3%; and the deduced amino acid sequence, 91.3%. In consideration of the high identities in between the CP gene and the 3'-non-coding region between them, WMV-II might be considered as a watermelon strain of SMV. Besides, some unexpected sequences were found in the 3'-region of 2 <span class="gene">NIb</span> gene clones. Following modification and splicing, a binary vector of <span class="gene">NIb</span> gene has been constructed for its expression in higher plant for the purpose of studying the possible replicase-mediated <span class="trait">resistance</span> in polyviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nucleotide sequence and genome organization of peanut stripe potyvirus.</h2>
			<h3>PMID:8077957  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gunasinghe UB(1), Flasinski S, Nelson RS, Cassidy BG.
				<br/> 
				年份：1994 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A blotch isolate of peanut stripe <span class="trait">virus</span> (PStV) was cloned, sequenced and compared with other full-length potyvirus sequences. The viral genome was 10059 nucleotides (nt) in length excluding the poly(A) tail. Two potential AUG start codons were identified in the 5' non-translated region. Analysis of in vitro translation products from transcripts containing the first 600 nt of the PStV genome indicated that the first AUG (nt 134 to 136) was preferred over the second AUG (nt 146 to 148) for initiation of translation. Within the single large open reading frame, eight processed proteins were predicted. In general, motifs conserved in other potyviral sequences were also found in the PStV genome. The presence of a 6K protein between the P3 and <span class="gene">CI</span> proteins was predicted. An altered amino acid motif, from FI(V)VRG to FMIIRG, within the carboxyl terminus of the <span class="gene">P1</span> protein, separates the PStV sequence from the majority of potyvirus sequences. Based on comparisons with available full-length potyvirus genome sequences, PStV was found to be most closely related to soybean <span class="trait">mosaic</span> <span class="trait">virus.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation of proline rich cell wall proteins in soybean lines with anthocyanin mutations.</h2>
			<h3>PMID:8425044  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nicholas CD(1), Lindstrom JT, Vodkin LO.
				<br/> 
				年份：1993 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The I locus controls inhibition of anthocyanin accumulation in the epidermal cells of the soybean seed coat and affects abundance of PRP1, a proline-rich cell wall protein in the seed coat. Saline-soluble <span class="gene">PRP1</span> is abundant in the developing seed coats of cultivar Richland (homozygous I, yellow), while it is significantly decreased in the pigmented isogenic mutant T157 (homozygous i, imperfect black). In this report, we examined soluble <span class="gene">PRP1</span> in several cultivars containing alleles of the I locus which affect spatial distribution of pigmentation in the seed coat. We also characterized <span class="gene">PRP1</span> in isolines with allelic variants of several other loci involved in seed coat pigmentation, including T and Im. The T gene is pleiotropic and affects both pubesence color and seed coat pigmentation and structure. Soluble <span class="gene">PRP1</span> was abundant in the developing seed coats of lines with yellow seed (I or ii alleles) regardless of pubescence color, just as in Richland. Likewise, soluble <span class="gene">PRP1</span> was decreased in pigmented seed coats (ik or i alleles) with grey (t) pubescence, as in T157. However, the total seed coat proteins were not extractable from pigmented seed coats with tawny pubescence (i, T genotypes) because they have proanthocyanidins that exhibit tannin properties. The dominant Im allele inhibits seed coat mottling (irregular patches of pigmentation) that occurs if plants are infected with soybean <span class="trait">mosaic</span> <span class="trait">virus.</span> <span class="gene">PRP1</span> was 35 kDa in mottled (im) isolines and 34 kDa in non-mottled (Im) isolines. PRP2, which is expressed later in seed coat development and in the hypocotyl hooks of soybean seedlings, was also smaller in Im isolines. In summary, some of the anthocyanin mutations affect the quantity of soluble <span class="gene">PRP1</span> polypeptides. while others correlate with structural changes in developmentally regulated proline-rich proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Complete nucleotide sequences of two soybean <span class="trait">mosaic</span> <span class="trait">virus</span> strains differentiated by response of soybean containing the Rsv <span class="trait">resistance</span> gene.</h2>
			<h3>PMID:1645142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jayaram C(1), Hill JH, Miller WA.
				<br/> 
				年份：1992 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete nucleotide sequence of the genomic RNAs of strains G2 and G7 of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> were determined. In both cases, the genome is 9588 nucleotides long, excluding the 3'-terminal poly(A) sequence. A large open reading frame (nucleotides 132 to 9329) encodes a <span class="gene">polyprotein</span> of 3066 amino acids with a predicted M(r) of either 349542 (strain G2) or 349741 (strain G7). Based on comparison with the proposed locations of cleavage sites of other potyvirus polyproteins, nine mature proteins are predicted. The mature proteins of the two strains share 94 to 100% amino acid identity, with the greatest variability occurring in the 35K and 42K proteins. Differences in local net charge in portions of these proteins as well as differences in amino acid sequence throughout the genome are discussed in relation to <span class="trait">resistance</span> and susceptibility of host plants to strains G2 and G7. Comparison with other potyviruses may be useful for taxonomic clarification of viruses and strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete nucleotide sequence of pea seed-borne <span class="trait">mosaic</span> <span class="trait">virus</span> RNA.</h2>
			<h3>PMID:1940858  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Johansen E(1), Rasmussen OF, Heide M, Borkhardt B.
				<br/> 
				年份：1991 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete nucleotide sequence of the RNA genome of pea seed-borne <span class="trait">mosaic</span> <span class="trait">virus</span> (PSbMV) was determined from cloned cDNA and by direct sequencing of viral RNA. The PSbMV genomic sequence was determined to be 9924 nucleotides in length excluding the poly(A) tract. The RNA contained an open reading frame (ORF) of 9618 nucleotides with the potential to encode a <span class="gene">polyprotein</span> with a calculated Mr of 364000 (364K). The ORF was flanked by a 5' untranslated leader sequence of 143 nucleotides and a 3' untranslated region of 163 nucleotides. A comparison of the PSbMV <span class="gene">polyprotein</span> with the polyproteins of the potyviruses tobacco etch <span class="trait">virus</span> (TEV), tobacco vein mottling <span class="trait">virus</span> (TVMV), plum pox <span class="trait">virus</span> (PPV) and potato <span class="trait">virus</span> Y (PVY) showed that PSbMV had a similar genome organization. The polyproteins had a high level of amino acid identity except in the N-terminal region, which varied in both sequence and length. Putative proteolytic cleavage sites were identified in the <span class="gene">polyprotein</span> of PSbMV by comparison with those identified for other potyviruses. The cleavage site between the 6K protein and the 49K proteinase is proposed to occur at the C-terminal side of glutamic acid and not at the C-terminal side of glutamine as in other potyviruses. In addition to the five proteolytic cleavage sites for the 49K proteinase identified previously, a sixth putative cleavage site was identified internally in the 49K proteinase of PSbMV, as well as in the 49K proteinases of TEV, TVMV, PPV, PVY and soybean <span class="trait">mosaic</span> <span class="trait">virus.</span> Cleavage at this site in the 49K proteinases of TEV, TVMV and PPV would result in an N-terminal protein of 22K to 24K, which is similar in size to the size determined for their VPgs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular genetic analyses of the soybean <span class="trait">mosaic</span> <span class="trait">virus</span> <span class="gene">NIa</span> proteinase.</h2>
			<h3>PMID:2212987  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ghabrial SA(1), Smith HA, Parks TD, Dougherty WG.
				<br/> 
				年份：1990 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recombinant DNA molecules containing cDNA to a soybean <span class="trait">mosaic</span> <span class="trait">virus</span> (SMV) RNA genome were constructed and partial nucleotide sequences determined for two cDNA inserts, pSMV-34 and pSMV-35. Comparison of the predicted amino acid sequence encoded by the pSMV-34 cDNA insert to other potyvirus protein sequences revealed extensive homology with the region of the genome encoding the <span class="gene">NIa</span> proteinase, with conservation of the amino acids proposed to form the catalytic triad of the active site. Cell-free transcription and translation of the cloned cDNA sequence containing the <span class="gene">NIa</span> open reading frame and flanking sequences revealed that <span class="gene">NIa</span> proteinase sequences, which were expressed as part of a high Mr precursor, were able to undergo proteolytic processing. Alteration of the codon for one of the putative active site residues by site-directed mutagenesis eliminated processing and resulted in the accumulation of a high Mr precursor. Based on predicted amino acid sequences at five putative cleavage sites within the SMV polyprotein, a consensus SMV NI a proteinase cleavage sequence of Glu/Asn-Xaa-Val-Xaa-Xaa-Gln decreases Gly/Ser was proposed. The SMV <span class="gene">NIa</span> proteinase and its putative cleavage sites maintained motifs found in other potyviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Recombination analysis of <italic> Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> sequences reveals evidence of RNA recombination between distinct pathotypes</h2>
			<h3>PMID:19036160</h3>
				
					<h3>PMCID:2627826</h3>
					<h4>期刊:Virology Journal</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Gagarinova </surname>  <given-names> Alla G </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> alla.gagarinova@utoronto.ca </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Babu </surname>  <given-names> Mohan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mohanbabu_r@yahoo.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Str&#x000f6;mvik </surname>  <given-names> Martina V </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> martina.stromvik@mcgill.ca </email>  </contrib>  <contrib id="A4" corresp="yes" contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Aiming </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> wanga@agr.gc.ca </email>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 11 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Soybean <span class="trait">mosaic</span> <span class="trait">virus</span> </italic> (SMV) is a member of the genus <italic> Potyvirus </italic> , the family <italic> Potyviridae </italic> . It is one of the most devastating viral pathogens of soybean crops causing severe symptoms such as <span class="trait">mosaic,</span> mottling, <span class="trait">chlorosis</span> and rugosity in leaves, as well as reductions in plant growth with <span class="trait">yield</span> losses of up to 100% [ <xref ref-type="bibr" rid="B1"> 1 </xref> - <xref ref-type="bibr" rid="B4"> 4 </xref> ]. Like all potyviral genomes, the SMV genome is a single-stranded, positive-sense RNA molecule that is approximately 10 kb in length and contains a single open reading frame [ <xref ref-type="bibr" rid="B4"> 4 </xref> - <xref ref-type="bibr" rid="B6"> 6 </xref> ]. It encodes a large <span class="gene">polyprotein</span> that is co- and post-translationally cleaved into 11 final protein products [ <xref ref-type="bibr" rid="B4"> 4 </xref> - <xref ref-type="bibr" rid="B6"> 6 </xref> ]. SMV is found in all soybean-growing regions of the world. In the United States, at least 98 SMV isolates have been documented [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Based on their differential interactions with SMV resistant cultivars, these isolates are classified into seven distinct strain groups, G1 through G7 [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Similarly, five (A to E) and eight (Sa to Sh) SMV strains have been reported in Japan and China, respectively [ <xref ref-type="bibr" rid="B9"> 9 </xref> - <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Though the pathotypic relationships between the SMV groups in the United States and the strains in China and Japan are not clear, these data clearly suggest a high genetic diversity of SMV.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Engineering of soybean <span class="trait">mosaic</span> <span class="trait">virus</span> as a versatile tool for studying protein&#x02013;protein interactions in soybean</h2>
			<h3>PMID:26926710</h3>
				
					<h3>PMCID:4772626</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Seo </surname>  <given-names> Jang-Kyun </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Choi </surname>  <given-names> Hong-Soo </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Kook-Hyung </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Crop Protection Division, National Academy of Agricultural Science, Rural Development Administration </institution> , Wanju 565-851,  <country> Republic of Korea </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Department of Agricultural Biotechnology and Plant Genomics and Breeding Institute, Seoul National University </institution> , Seoul 151-921,  <country> Republic of Korea </country>  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 02 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the previous study, we showed that DNA-mediated rub-inoculation of the SMV infectious cDNA construct yielded highly efficient infection on soybean plants <xref ref-type="bibr" rid="b7"> 7 </xref> . Thus, in the present study, we sought to examine whether the additional insertion of the gene cassette between the <span class="gene">NIb</span> and CP cistrons affects the infectivity of the pSMV-Dual plasmid. To this end, soybean seedlings (cv. Lee74, used throughout this study) were rub-inoculated with different quantities (10&#x02009;&#x003bc;g, 5&#x02009;&#x003bc;g, or 1&#x02009;&#x003bc;g) of plasmid DNA of pSMV-Dual. Experiments were carried out three times independently, using 45 plants in total ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Infection of the inoculated plants with SMV was investigated by observing the appearance of symptoms on systemic leaves and by RT-PCR analysis using SMV-specific primers as described previously <xref ref-type="bibr" rid="b7"> 7 </xref> . At 15 days post inoculation (dpi), all of the soybean plants inoculated with 10&#x02009;&#x003bc;g, 5&#x02009;&#x003bc;g, or 1&#x02009;&#x003bc;g of plasmid DNA showed typical mild <span class="trait">mosaic</span> symptoms in upper uninoculated leaves. RT-PCR analysis further confirmed that all the inoculated plants were systemically infected ( <xref ref-type="table" rid="t1"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm these known interactions by co-IP and to identify additional cellular interacting partner proteins, we decided to transiently express the SMV HC-Pro tagged with the Flag epitope (Asp-Tyr-Lys-Asp-Asp-Asp-Asp-Lys) at the N-terminus using the SMV-based dual vector. To this end, we generated a recombinant SMV construct, pSMV-Dual-fHC-Pro, that expresses Flag-tagged HC-Pro ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ). In parallel, an additional SMV construct, pSMV-Dual-fGFP, that expresses Flag-tagged GFP, was generated and used as a negative control in co-IP experiments. Each construct was rub-inoculated on the leaves of soybean seedlings. At 15&#x02009;dpi, crude plant extracts were prepared by homogenizing the upper symptomatic leaves in the extraction buffer. After removing cell debris by centrifugation, the resulting extracts were incubated with anti-Flag antibody-conjugated agarose beads followed by precipitation by centrifugation. The resulting co-immunoprecipitated products were analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and stained with Coomassie Blue ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ). After staining, the specific dominant bands in each lane were excised from the gel and subjected to LC-MS/MS analysis. The identified proteins and MS/MS spectral information are shown in <xref ref-type="fig" rid="f4"> Fig. 4 </xref> . As expected, one dominant band in the Flag-GFP co-IP sample was identified as GFP. A total of six bands were analyzed in the Flag-HC-Pro co-IP sample. The bands at position 220, 110, and 55&#x02009;kDa were identified as HC-Pro whereas the band at position 35&#x02009;kDa was identified as SMV CP. Therefore, our co-IP approach confirmed the previous findings of the HC-Pro oligomerization and the HC-Pro&#x02013;CP interaction. The identification of HC-Pro from the bands at positions 220, 110, and 55&#x02009;kDa indicates tetrameric, dimeric, and monomeric forms of HC-Pro, respectively. The HC-Pro self-interaction is likely quite strong because the oligomeric forms were detected under the denaturing conditions of SDS-PAGE. In addition, we identified two novel cellular interacting partners of HC-Pro, glyceraldehyde-3-phosphate dehydrogenase (GAPDH) at position 43&#x02009;kDa and cytochrome b6/f complex subunit IV (PetD) at position 17&#x02009;kDa. <span class="gene">GAPDH</span> is an important enzyme that plays a pivotal role in energy metabolism <xref ref-type="bibr" rid="b26"> 26 </xref> . Recent studies have shown that <span class="gene">GAPDH</span> has multiple functions in DNA replication/repair, RNA transport, apoptosis, oxidative stress, membrane fusion, and cytoskeleton assembly <xref ref-type="bibr" rid="b27"> 27 </xref> <xref ref-type="bibr" rid="b28"> 28 </xref> <xref ref-type="bibr" rid="b29"> 29 </xref> <xref ref-type="bibr" rid="b30"> 30 </xref> . In addition, a few studies have suggested that <span class="gene">GAPDH</span> may play a role in RNA <span class="trait">virus</span> replication <xref ref-type="bibr" rid="b31"> 31 </xref> <xref ref-type="bibr" rid="b32"> 32 </xref> <xref ref-type="bibr" rid="b33"> 33 </xref> . On the other hand, PetD is required for photosynthetic electron transport <xref ref-type="bibr" rid="b34"> 34 </xref> . The involvement of PetD in <span class="trait">virus</span> infection cycle has not been studied yet. We are currently performing a separate study to further characterize the interactions between HC-Pro and either <span class="gene">GAPDH</span> or PetD and to examine the roles of <span class="gene">GAPDH</span> and PetD in HC-Pro functions and SMV infection.
					</p>
				
					<p>&nbsp;&nbsp;
						Potyviruses constitute the largest genus of plant viruses and adopt the same gene expression strategy of proteolytic processing of <span class="gene">polyprotein</span> precursors, making heterologous technologies broadly applicable for manipulation of potyvirus genomes <xref ref-type="bibr" rid="b10"> 10 </xref> <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b46"> 46 </xref> . So far, various potyviruses including tobacco etch <span class="trait">virus</span> (TEV), turnip <span class="trait">mosaic</span> <span class="trait">virus</span> (TuMV), and zucchini yellow <span class="trait">mosaic</span> <span class="trait">virus</span> (ZYMV) that have a broad host range have been engineered to express genes of interest <xref ref-type="bibr" rid="b46"> 46 </xref> <xref ref-type="bibr" rid="b47"> 47 </xref> <xref ref-type="bibr" rid="b48"> 48 </xref> . Therefore, our approaches shown in this study with the SMV-dual vector can be applied to other potyvirus-host plant systems for simultaneous expression of multiple genes, visualization of protein subcellular localization and identification and characterization of protein-protein interactions in plants in which agroinfiltration is unsuccessful. We expect that our method will become a versatile and powerful tool for studying protein&#x02013;protein interactions and for rapid analysis of gene function in various plant species.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Current Status of the Soybean- <italic> Soybean <span class="trait">Mosaic</span> <span class="trait">Virus</span> </italic> (SMV) Pathosystem</h2>
			<h3>PMID:27965641</h3>
				
					<h3>PMCID:5127794</h3>
					<h4>期刊:Frontiers in Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jian-Zhong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/268513/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fang </surname>  <given-names> Yuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/389900/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pang </surname>  <given-names> Hongxi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Rsv1 </italic> was originally identified in the soybean line PI 96983 ( <xref rid="B42" ref-type="bibr"> Kiihl and Hartwig, 1979 </xref> ), and it confers extreme <span class="trait">resistance</span> (ER) to SMV-G1 through G6 but not to SMV-G7 ( <xref rid="B6" ref-type="bibr"> Chen et al., 1991 </xref> ; <xref rid="B31" ref-type="bibr"> Hajimorad and Hill, 2001 </xref> ; <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ). Multiple <italic> Rsv1 </italic> alleles including <italic> Rsv1-y, Rsv1-m, Rsv1-t, Rsv1 </italic> - <italic> k </italic> , and <italic> Rsv1-r </italic> have been identified from different soybean cultivars with differential reactions to SMV G1&#x02013;G7 strains ( <xref rid="B8" ref-type="bibr"> Chen et al., 2001 </xref> ). <italic> Rsv1 </italic> was initially mapped to soybean linkage group F on chromosome 13 ( <xref rid="B78" ref-type="bibr"> Yu et al., 1994 </xref> ) and two classes of NBS-LRR sequences (classes b and j) were identified in this <span class="trait">resistance</span> gene cluster ( <xref rid="B77" ref-type="bibr"> Yu et al., 1996 </xref> ). A large family of homologous sequences of the class j Nucleotide biinding site-leucine rich repeat (NBS-LRRs) clustered at or near the <italic> Rsv1 </italic> locus ( <xref rid="B38" ref-type="bibr"> Jeong et al., 2001 </xref> ; <xref rid="B23" ref-type="bibr"> Gore et al., 2002 </xref> ; <xref rid="B53" ref-type="bibr"> Pe&#x000f1;uela et al., 2002 </xref> ). Six candidate genes (1eG30, 5gG3, 3gG2, 1eG15, 6gG9, and 1gG4) in PI96983 were mapped to a tightly clustered region near <italic> Rsv1 </italic> , three of them (3gG2, 5gG3, and 6gG9) were completely cloned and sequenced (GenBank accession no. AY518517&#x02013;AY518519). Among the three genes, <span class="gene">3gG2</span> was found to be a strong candidate for <italic> Rsv1 </italic> ( <xref rid="B32" ref-type="bibr"> Hayes et al., 2004 </xref> ). When 3gG2, 5gG3, and 6gG93 were simultaneously silenced using <italic> Bean pod mottle <span class="trait">virus</span> </italic> -induced gene silencing (BPMV-VIGS), the <italic> Rsv1 </italic> -mediated <span class="trait">resistance</span> was compromised, confirming that one or more of these three genes is indeed the <italic> Rsv1 </italic> ( <xref rid="B80" ref-type="bibr"> Zhang et al., 2012 </xref> ). Because, the sequence identities of these three <italic> R </italic> genes are extremely high along the entire cDNAs, it is impossible to differentiate which one(s) is <italic> Rsv1. </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Many <italic> Rsc </italic> loci have been identified. The <span class="trait">resistance</span> genes <italic> Rsc-8 </italic> and <italic> Rsc-9 </italic> , which confer <span class="trait">resistance</span> to strains <italic> SC-8 </italic> and <italic> SC-9 </italic> respectively, have been mapped to the soybean chromosomes 2 (MLG D1b+W) ( <xref rid="B68" ref-type="bibr"> Wang et al., 2004 </xref> ). The interval of <italic> Rsc-8 </italic> was estimated to be 200 kb and contains 17 putative genes and five of them, <span class="gene">Glyma02g13310,</span> 13320, 13400, 13460, and 13470 could be the candidates of <italic> Rsc-8 </italic> based on their predicted functions and expression patterns ( <xref rid="B63" ref-type="bibr"> Wang et al., 2011 </xref> ). The <italic> Rsc-15 </italic> resistant gene was mapped between Sat_213 and Sat_286 with distances of 8.0 and 6.6 cM to the respective flanking markers on chromosome 6 ( <xref rid="B74" ref-type="bibr"> Yang and Gai, 2011 </xref> ). The <span class="trait">resistance</span> gene <italic> Rsc-7 </italic> in the soybean cultivar Kefeng No.1 was mapped to a 2.65 mega-base (Mb) region on soybean chromosome 2 ( <xref rid="B20" ref-type="bibr"> Fu et al., 2006 </xref> ) and was subsequently narrowed down to a 158 kilo-base (Kb) region ( <xref rid="B73" ref-type="bibr"> Yan et al., 2015 </xref> ). Within 15 candidate genes in the region, one NBS-LRR type gene <span class="gene">(Glyma02g13600),</span> one HSP40 gene <span class="gene">(Glyma02g13520)</span> and one serine carboxypeptidase-type gene <span class="gene">(Glyma02g13620)</span> could be the candidates for <italic> Rsc-7 </italic> . The allelic relationship between the <italic> Rsv </italic> loci and the <italic> Rsc </italic> loci has yet to be determined.
					</p>
				
					<p>&nbsp;&nbsp;
						Despite numerous efforts, none of the SMV resistant genes has been cloned and their identities remain to be identified. This reflects the complex nature of the resistant genes in palaeopolyploid soybean, in which 75% of the genes are present in multiple copies ( <xref rid="B56" ref-type="bibr"> Schmutz et al., 2010 </xref> ). This statement is reinforced by a recent finding that the soybean cyst nematode (SCN) <span class="trait">resistance</span> mediated by the <italic> <span class="gene">Rhg1</span> </italic> is conditioned by copy number variation of a 31-kilobase segment, in which three different novel genes are present ( <xref rid="B16" ref-type="bibr"> Cook et al., 2012 </xref> ). There are 1&#x02013;3 copies of the 31-kilobase segment per haploid genome in susceptible varieties, but 10 tandem copies in resistant varieties ( <xref rid="B16" ref-type="bibr"> Cook et al., 2012 </xref> ). The presence of more copies of the 31-kb segment in resistant varieties increases the expressions of this set of the 3 genes and thus conferes the <span class="trait">resistance</span> ( <xref rid="B16" ref-type="bibr"> Cook et al., 2012 </xref> , <xref rid="B15" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Rsv1 </italic> , a single dominant <span class="trait">resistance</span> gene in soybean PI 96983 ( <italic> Rsv1 </italic> ), confers ER against SMV-G1 through G6 but not to SMV-G7 ( <xref rid="B6" ref-type="bibr"> Chen et al., 1991 </xref> ; <xref rid="B31" ref-type="bibr"> Hajimorad and Hill, 2001 </xref> ; <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ). SMV-N (an avirulent isolate of strain G2) elicits ER whereas strain SMV-G7 provokes a lethal systemic hypersensitive response (LSHR) ( <xref rid="B27" ref-type="bibr"> Hajimorad et al., 2003 </xref> ; <xref rid="B32" ref-type="bibr"> Hayes et al., 2004 </xref> ). SMV-G7d, an evolved variant of SMV-G7 from lab, induces systemic <span class="trait">mosaic</span> ( <xref rid="B27" ref-type="bibr"> Hajimorad et al., 2003 </xref> ). Serial passages of a large population of the progeny in PI 96983 resulted in emergence of a mutant population (vSMV-G7d), which can evade <italic> Rsv1 </italic> -mediated recognition and the putative amino acid changes that potentially responsible for the mutant phenotype is initially tentatively narrowed down to HC-Pro, coat protein, PI proteinase or P3 ( <xref rid="B27" ref-type="bibr"> Hajimorad et al., 2003 </xref> ; <xref rid="B59" ref-type="bibr"> Seo et al., 2011 </xref> ) and was later mapped to P3 through domain swapping between the pSMV-G7 and pSMV-G7d ( <xref rid="B28" ref-type="bibr"> Hajimorad et al., 2005 </xref> ). The amino acids 823, 953, and 1112 of the SMV-G7d are critical in evading of <italic> Rsv1 </italic> -mediated recognition ( <xref rid="B28" ref-type="bibr"> Hajimorad et al., 2005 </xref> , <xref rid="B29" ref-type="bibr"> 2006 </xref> ). By generating a series of chimeras between SMV-G7 and SMV-N in combination with site-directed mutagenesis, <xref rid="B17" ref-type="bibr"> Eggenberger et al. (2008) </xref> and <xref rid="B30" ref-type="bibr"> Hajimorad et al. (2008) </xref> independently showed that gain of virulence on <italic> Rsv1 </italic> -genotype soybean by an avirulent SMV strains requires concurrent mutations in both P3 and HC-Pro and HC-Pro complementation of P3 is essential for SMV virulence on <italic> Rsv1 </italic> -genotype soybean ( <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ). A key virulence determinant of SMV on <italic> Rsv1 </italic> -genotype soybeans that resides at <span class="gene">polyprotein</span> codon 947 overlaps both P3 and a PIPO-encoded codon. This raises the question of whether PIPO or P3 is the virulence factor. <xref rid="B71" ref-type="bibr"> Wen et al. (2011) </xref> confirmed that amino acid changes in P3, and not the overlapping PIPO-encoded protein, which is embedded in the P3 cistron, determine virulence of SMV on <italic> Rsv1 </italic> -genotype soybean. <xref rid="B13" ref-type="bibr"> Chowda-Reddy et al. (2011b) </xref> constructed a chimeric infectious clone of G7, in which the N-ternimal part of <span class="gene">CI</span> was swapped with the corresponding part of G2. Compared with wildtype G7, this chimeric strain lost virulence on <italic> Rsv1 </italic> -genotype plant but gained infectivity on <italic> Rsv3 </italic> -genotype plant, indicating an essential role of <span class="gene">CI</span> for breaking down both <italic> Rsv1 </italic> - and <italic> Rsv3 </italic> -mediated <span class="trait">resistance</span> ( <xref rid="B13" ref-type="bibr"> Chowda-Reddy et al., 2011b </xref> ). Together, it appears that P3, HC-Pro and possibly <span class="gene">CI</span> are virulent determinants for <italic> Rsv1 </italic> -mediated <span class="trait">resistance</span> ( <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						It has been proven that cytoplasmic inclusion cistron (CI) of SMV serves as a virulence and symptom determinant on <italic> Rsv3 </italic> -genotype soybean and a single amino acid substitution in <span class="gene">CI</span> was found to be responsible for gain or loss of elicitor function of <span class="gene">CI</span> ( <xref rid="B58" ref-type="bibr"> Seo et al., 2009 </xref> ; <xref rid="B81" ref-type="bibr"> Zhang et al., 2009a </xref> ). Analyses of the chimeras by exchanging fragments between avirulent SMV-G7 and the virulent SMV-N showed that both the N- and C-terminal regions of the <span class="gene">CI</span> cistron are required for <italic> Rsv3 </italic> -mediated <span class="trait">resistance</span> and the N-terminal region of <span class="gene">CI</span> is also involved in severe symptom induction in soybean ( <xref rid="B81" ref-type="bibr"> Zhang et al., 2009a </xref> ). In addition to CI, P3 has also been reported to play an essential role in virulence determination on <italic> Rsv3 </italic> -mediated <span class="trait">resistance</span> ( <xref rid="B12" ref-type="bibr"> Chowda-Reddy et al., 2011a </xref> , <xref rid="B13" ref-type="bibr"> b </xref> ; <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear now that P3 plays essential roles in virulence determination on <italic> Rsv1, Rsv3 </italic> , and <italic> Rsv4 </italic> resistant loci, while <span class="gene">CI</span> is required for virolence on <italic> Rsv1 </italic> and <italic> Rsv3 </italic> genotype soybean plants ( <xref rid="B12" ref-type="bibr"> Chowda-Reddy et al., 2011a </xref> , <xref rid="B13" ref-type="bibr"> b </xref> ). These results imply that avirulent proteins from SMV might interact with the soybean <italic> R </italic> gene products at a converged point. This evolved interactions sometimes could give SMV advantage in breaking <span class="trait">resistance</span> conferred by different SMV resistant genes simply by mutations within a single viral protein. On the other hand, since multiple proteins are involved in virulence on different resistant loci, concurrent mutantions in multiple proteins of SMV are required to evade the <span class="trait">resistance</span> conferred by different SMV resistant genes. The likelihood of such naturally occuurred concurrent mutations in different viral proteins is low. Therefore, integration of all three SMV resistant genes in a single elite soybean cultivar may provide long-lasting <span class="trait">resistance</span> to SMV in soybean breeding practice ( <xref rid="B13" ref-type="bibr"> Chowda-Reddy et al., 2011b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The key components in <italic> R </italic> gene mediated disease resistant signaling pathway have been identified in model plant <italic> Arabidopsis </italic> , among which, RAR1 ( <italic> Required for Mla 12 <span class="trait">Resistance</span> </italic> ), SGT1( <italic> Suppressor of G2 Allele of Skp1 </italic> ) and <span class="gene">HSP90</span> ( <italic> Heat Shock Protein 90 </italic> ) are the most important ones ( <xref rid="B1" ref-type="bibr"> Belkhadir et al., 2004 </xref> ). Using BPMV-VIGS, it has been shown that <italic> Rsv1 </italic> -mediated ER against SMV in soybean requires RAR1 and SGT1 but not <span class="gene">GmHSP90,</span> suggesting although soybean defense signaling pathways recruit structurally conserved components, they have distinct requirements for specific proteins ( <xref rid="B19" ref-type="bibr"> Fu et al., 2009 </xref> ). However, <xref rid="B80" ref-type="bibr"> Zhang et al. (2012) </xref> showed that silencing <italic> <span class="gene">GmHSP90</span> </italic> using BPMV-VIGS compromised <italic> Rsv1 </italic> -mediated <span class="trait">resistance.</span> In addition, silencing <italic> <span class="gene">GmEDR1</span> </italic> ( <italic> Enhanced Disease <span class="trait">Resistance</span> 1 </italic> ), <italic> <span class="gene">GmEDS1</span> </italic> ( <italic> Enhanced Disease Susceptibility 1 </italic> ), <italic> <span class="gene">GmHSP90,</span> <span class="gene">GmJAR1</span> </italic> ( <italic> Jasmonic Responsive 1 </italic> ), <italic> <span class="gene">GmPAD4</span> </italic> ( <italic> Phytoalexin Deficient </italic> 4), and two genes encoding <italic> WRKY </italic> transcription factors (WRKY6 and WRKY 30), all of which are involved in defense pathways in model plant <italic> Arabidopsis, Rsv1 </italic> -mediated ER was also compromised ( <bold> Table <xref ref-type="table" rid="T2"> 2 </xref> </bold> ). These results suggest that the host components required for <italic> R </italic> gene-mediated resistant signaling pathways are conserved across plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						Mitogen-activated protein kinase (MAPK) cascades play important roles in disease <span class="trait">resistance</span> ( <xref rid="B52" ref-type="bibr"> Meng and Zhang, 2013 </xref> ). The function of MAPK signaling pathways in disease <span class="trait">resistance</span> was investigated in soybean using BPMV-VIGS ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> , <xref rid="B45" ref-type="bibr"> 2014 </xref> , <xref rid="B46" ref-type="bibr"> 2015 </xref> ). Among the plants silenced for multiple genes in MAPK pathway, the plants silenced for the <italic> <span class="gene">GmMAPK4</span> and <span class="gene">GmMAPK6</span> </italic> homologs displayed strong phenotypes of activated defense responses ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> , <xref rid="B45" ref-type="bibr"> 2014 </xref> ). Consistent with the activated defense response phenotypes, these plants were more resistant to SMV compared with vector control plants ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> , <xref rid="B45" ref-type="bibr"> 2014 </xref> ), indicating that both genes play critical negative roles in basal <span class="trait">resistance</span> or PAMP-triggered immunity (PTI) in soybean. The constitutively activated defense responses has been reported for <italic> mpk4 </italic> mutant in <italic> Arabidopsis </italic> ( <xref rid="B54" ref-type="bibr"> Petersen et al., 2000 </xref> ) and the positive role of MPK6 in defense responses is well-documented ( <xref rid="B52" ref-type="bibr"> Meng and Zhang, 2013 </xref> ). However, the negative role of MAKP6 homologs has not been reported previously ( <xref rid="B45" ref-type="bibr"> Liu et al., 2014 </xref> ), indicating that both conserved and distinct functions of MAPK signaling pathways in immunity are observed between <italic> Arabidopsis </italic> and soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Numerous host factors participate in defense responses in plants. Identification of these factors may facilitate rationale design of novel resistant strategies. Recently, it has been shown that silencing <italic> <span class="gene">GmHSP40.1</span> </italic> , a soybean nuclear-localized type-III <span class="gene">DnaJ</span> domain-containing HSP40, results in increased infectivity of SMV, indicating a positive role of <span class="gene">GmHSP40.1</span> in basal <span class="trait">resistance</span> ( <xref rid="B48" ref-type="bibr"> Liu and Whitham, 2013 </xref> ). A subset of type 2C protein phophatase (PP2C) gene family, which participate ABA signaling pathway, is specifically up-regulated during <italic> Rsv3 </italic> -mediated <span class="trait">resistance</span> ( <xref rid="B57" ref-type="bibr"> Seo et al., 2014 </xref> ). Synchronized overexpression of <span class="gene">GmPP2C3a</span> using SMV-G7H vector inhibits <span class="trait">virus</span> cell-to-cell movement mediated by callose deposition in an ABA signaling-dependent manner, indicating that <span class="gene">GmPP2C3a</span> functions as a key regulator of <italic> Rsv3 </italic> -mediated <span class="trait">resistance</span> ( <xref rid="B57" ref-type="bibr"> Seo et al., 2014 </xref> ). An ortholog of <italic> Arabidopsis </italic> K <sup> + </sup> weak channel encoding gene AKT2, was significantly induced by SMV inoculation in the SMV highly resistant genotype, but not in the susceptible genotype ( <xref rid="B85" ref-type="bibr"> Zhou et al., 2014 </xref> ). Overexpression of <italic> <span class="gene">GmAKT2</span> </italic> not only significantly increased K <sup> + </sup> concentrations in young leaves but also significantly enhanced the <span class="trait">resistance</span> against SMV, indicating alteration of K <sup> + </sup> transporter expression could be a novel molecular approach for enhancing SMV <span class="trait">resistance</span> in soybean ( <xref rid="B85" ref-type="bibr"> Zhou et al., 2014 </xref> ). Molybdenum cofactor (Moco) is required for the activities of Moco-dependant enzymes. Cofactor for nitrate reductase and xanthine dehydrogenase (Cnx1) is known to be involved in the biosynthesis of Moco in plants. Soybean plants transformed with <italic> Cnx1 </italic> enhanced the enzyme activities of nitrate reductase (NR) and aldehydeoxidase (AO) and resulted in an enhanced <span class="trait">resistance</span> against various strains of SMV ( <xref rid="B86" ref-type="bibr"> Zhou et al., 2015 </xref> ). The differentially expressed genes in <italic> Rsv1 </italic> genotype in response to G7 infection have been identified ( <xref rid="B3" ref-type="bibr"> Chen et al., 2016a </xref> ). Knocking down one of the identified genes, the eukaryotic translation initiation factor 5A ( <italic> eIF5A </italic> ), diminished the LSHR and enhanced viral accumulation, suggesting an essential role of eIF5A in the <italic> Rsv1 </italic> -mediated LSHR signaling pathway. Eukaryotic elongation factor 1A (eEF1A) is a well-known host factor in viral pathogenesis. Recently, <xref rid="B49" ref-type="bibr"> Luan et al. (2016) </xref> showed that silencing <italic> <span class="gene">GmeEF1A</span> </italic> inhibits accumulation of SMV and P3 protein of SMV interacts with <span class="gene">GmeEF1A</span> to facilitate its nuclear localization and therefore, promotes SMV pathogenicity.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Bean pod mottle <span class="trait">virus</span> </italic> -induced gene silencing system has been proven successful in gene function studies in soybean ( <xref rid="B82" ref-type="bibr"> Zhang et al., 2009b </xref> , <xref rid="B79" ref-type="bibr"> 2010 </xref> ; <xref rid="B46" ref-type="bibr"> Liu et al., 2015 </xref> ). There are four <italic> <span class="gene">GmMAPK4</span> </italic> homologs that can be divided into two paralogous groups ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> ). The sequence identities of ORFs within the groups are greater than 96%, whereas the identities between the groups are 88.7% ( <xref rid="B46" ref-type="bibr"> Liu et al., 2015 </xref> ). The BPMV-VIGS construct used for silencing <italic> <span class="gene">GmMAPK4</span> </italic> by <xref rid="B47" ref-type="bibr"> Liu et al. (2011) </xref> actually can silence all four of the isoforms simultaneously. When only one parologous group was silenced by using construct targeted for the 3&#x02032; UTR (the sequence identity of the 3&#x02032; UTRs between the two parologous groups is less than 50%), the activated defense response was not observed, indicating that silencing the four <italic> <span class="gene">GmMAPK4</span> </italic> isoforms simultaneously is necessary for activating defense responses in soybean. Using the same approach, it has been differentiated that <italic> <span class="gene">GmSGT1-2</span> </italic> but not <italic> <span class="gene">GmSGT1-1</span> </italic> is required for the <italic> Rsv1 </italic> -mediated ER against SMV ( <xref rid="B19" ref-type="bibr"> Fu et al., 2009 </xref> ). Thus, VIGS is currently the most powerful tool in overcoming the gene redundancy in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						As none of the SMV resistant gene has been cloned so far, it is not possible to generating resistant soybean plants simply by transforming the resistant genes. In addition, due to the rapid evolution in avirulence/effector genes, the <span class="trait">resistance</span> conditioned by <italic> R </italic> genes will be overcome quickly ( <xref rid="B11" ref-type="bibr"> Choi et al., 2005 </xref> ; <xref rid="B21" ref-type="bibr"> Gagarinova et al., 2008 </xref> ). Therefore, there is urgent need for a better solution in generating long-lasting SMV <span class="trait">resistance</span> with wide spectrums. As the first step, the identities of different <italic> Rsv </italic> genes need to be revealed and the key components in SMV resistant signaling pathway need to be identified. Cutting edge functional genomics tools and technologies have been proven successful in cloning of SCN resistant genes <italic> <span class="gene">Rhg4</span> </italic> ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> ). TILLING coupled with VIGS and RNA interference confirmed that a mutation in the <italic> <span class="gene">Rhg4</span> </italic> , a serine hydroxymethyltransferase ( <italic> <span class="gene">SHMT</span> </italic> ) gene, is responsible for <italic> <span class="gene">Rhg4</span> </italic> mediated <span class="trait">resistance</span> to SCN ( <xref rid="B47" ref-type="bibr"> Liu et al., 2011 </xref> ). VIGS has been proven useful in interrogating gene functions and can overcome gene redundancy in soybean ( <xref rid="B46" ref-type="bibr"> Liu et al., 2015 </xref> ). It has been shown recently that knocking out all three <italic> TaMLO </italic> homoeologs simultaneously in hexaploid bread wheat using TALEN and CRISPR-CAS9 resulted in heritable broad-spectrum <span class="trait">resistance</span> to powdery mildew ( <xref rid="B65" ref-type="bibr"> Wang et al., 2014 </xref> ). We believe that the same strategy can be applied to soybean in the near future. These new functional genomics approaches and genome editing tools will greatly facilitate the cloning of SMV resistant genes and elucidating the SMV resistant signaling pathways. Marker assisted selection (MAS) has become very useful in the effort of tagging genes for SMV <span class="trait">resistance.</span> Single nucleotide polymorphism (SNP) is a powerful tool in genome mapping, association studies, and cloning of important genes ( <xref rid="B14" ref-type="bibr"> Clevenger et al., 2015 </xref> ) and the increasingly saturated SNPs are being established in soybean ( <xref rid="B72" ref-type="bibr"> Wu et al., 2010 </xref> ; <xref rid="B43" ref-type="bibr"> Lee et al., 2015 </xref> ). With all these tools and resources available, pyramiding multiple SMV <span class="trait">resistance</span> genes in elite soybean cultivars to generate durable <span class="trait">resistance</span> with broad spectrum is more realistic than ever.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Stability evaluation of reference genes for gene expression analysis by RT-qPCR in soybean under different conditions</h2>
			<h3>PMID:29236756</h3>
				
					<h3>PMCID:5728501</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0001-9674-4933 </contrib-id>  <name>  <surname> Wan </surname>  <given-names> Qiao </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chen </surname>  <given-names> Shuilian </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Zhihui </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Zhonglu </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Limiao </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chanjuan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Songli </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qinnan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaojuan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Dezhen </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-9670-9895 </contrib-id>  <name>  <surname> Chen </surname>  <given-names> Haifeng </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001"/>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff001"/>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Real-time quantitative reverse transcription PCR is a sensitive and widely used technique to quantify gene expression. To achieve a reliable result, appropriate reference genes are highly required for normalization of transcripts in different samples. In this study, 9 previously published reference genes ( <italic> 60S </italic> , <italic> Fbox </italic> , <italic> ELF1A </italic> , <italic> ELF1B </italic> , <italic> ACT11 </italic> , <italic> TUA5 </italic> , <italic> <span class="gene">UBC4</span> </italic> , <italic> G6PD </italic> , <italic> CYP2 </italic> ) of soybean [ <italic> Glycine max </italic> (L.) Merr.] were selected. The expression stability of the 9 genes was evaluated under conditions of biotic stress caused by infection with soybean <span class="trait">mosaic</span> <span class="trait">virus,</span> nitrogen stress, across different cultivars and developmental stages. &#x00394;Ct and geNorm algorithms were used to evaluate and rank the expression stability of the 9 reference genes. Results obtained from two algorithms showed high consistency. Moreover, results of pairwise variation showed that two reference genes were sufficient to normalize the expression levels of target genes under each experimental setting. For <span class="trait">virus</span> infection, <italic> ELF1A </italic> and <italic> ELF1B </italic> were the most stable reference genes for accurate normalization. For different developmental stages, <italic> Fbox </italic> and <italic> G6PD </italic> had the highest expression stability between two soybean cultivars (Tanlong No. 1 and Tanlong No. 2). <italic> ELF1B </italic> and <italic> ACT11 </italic> were identified as the most stably expressed reference genes both under nitrogen stress and among different cultivars. The results showed that none of the candidate reference genes were uniformly expressed at different conditions, and selecting appropriate reference genes was pivotal for gene expression studies with particular condition and tissue. The most stable combination of genes identified in this study will help to achieve more accurate and reliable results in a wide variety of samples in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Common reference genes used in RT-qPCR for normalization are housekeeping genes related to basic metabolism pathway, such as <italic> ACTIN </italic> (essential for cytoskeleton structuring and kinetics), <italic> elongation factors </italic> ( <italic> ELF1&#x003b1; </italic> and <italic> ELF1&#x003b2; </italic> ), <italic> 60s </italic> (60s Ribosomal protein L30), <italic> tubulin </italic> (&#x003b1;- and &#x003b2;-tubulin, TUA and TUB, essential for cytoskeleton structuring and kinetic), <italic> Fbox </italic> (Fbox protein family), <italic> Ubiquitin </italic> -conjugating enzyme E2 (family UBC4-enzyme involved in abnormal and short-lived proteins degradation), <italic> glucose-6-phosphate dehydrogenase </italic> (G6PD-important enzyme of the glycolysis pathway) and <italic> cyclophylin </italic> (CYP-central to protein unfolding and protein interaction) [ <xref rid="pone.0189405.ref002" ref-type="bibr"> 2 </xref> , <xref rid="pone.0189405.ref003" ref-type="bibr"> 3 </xref> , <xref rid="pone.0189405.ref012" ref-type="bibr"> 12 </xref> &#x02013; <xref rid="pone.0189405.ref015" ref-type="bibr"> 15 </xref> ]. As a consensus, it is assumed that these housekeeping genes have constant expression levels among control and treated samples regardless of experimental conditions, developmental stages, tissues and organs or stress treatments [ <xref rid="pone.0189405.ref012" ref-type="bibr"> 12 </xref> , <xref rid="pone.0189405.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0189405.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0189405.ref017" ref-type="bibr"> 17 </xref> ]. However, a number of studies reported that transcription of housekeeping genes can fluctuate considerably under certain stress conditions, like pathogen infection, cold temperature and <span class="trait">drought</span> [ <xref rid="pone.0189405.ref018" ref-type="bibr"> 18 </xref> &#x02013; <xref rid="pone.0189405.ref020" ref-type="bibr"> 20 </xref> ]. For example, when maize seeds were infected by fungi, the expression level of some genes involved in metabolism, protein synthesis were down-regulated, including housekeeping genes like <italic> <span class="gene">GAPDH</span> </italic> [ <xref rid="pone.0189405.ref018" ref-type="bibr"> 18 </xref> , <xref rid="pone.0189405.ref020" ref-type="bibr"> 20 </xref> ]. Thus, there are no universal reference genes under all experimental conditions [ <xref rid="pone.0189405.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0189405.ref021" ref-type="bibr"> 21 </xref> , <xref rid="pone.0189405.ref022" ref-type="bibr"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In RT-qPCR analysis, it is assumed that reference genes have constant expression levels among different samples. However, evidences showed that transcripts levels of housekeeping genes may vary considerably under experimental conditions and/or in tissues types [ <xref rid="pone.0189405.ref002" ref-type="bibr"> 2 </xref> ]. To obtain high accuracy, it is necessary to validate reference genes for each plant species being studied and for each specific experimental condition [ <xref rid="pone.0189405.ref003" ref-type="bibr"> 3 </xref> ]. The target gene expression was evaluated according to reference gene expression level, thus unstable reference genes can result in inaccurate evaluation of target gene expression. As reported by Dung et al. in the previous study, the expression of four known dehydration-inducible genes <italic> <span class="gene">GmNACs</span> </italic> in dehydrate treatment was hard to detect with the least stable reference gene of <italic> SUBI2 </italic> , while the relative transcript abundance of <italic> <span class="gene">GmNACs</span> </italic> was induced by 3- to 4-fold with the stable reference genes of <italic> 60S </italic> and <italic> Fbox </italic> [ <xref rid="pone.0189405.ref019" ref-type="bibr"> 19 </xref> ]. A previous report also described that when the two most stable genes <italic> <span class="gene">GAPDH1</span> </italic> and <italic> EF </italic> were used as reference genes for pistillate flower, the expression level of <italic> AGAMOUS </italic> gene increased gradually in stage 1 to stage 4 and then declined at stage 5. However, when the least stable reference gene <italic> PLA </italic> was used, the expression level of <italic> AGAMOUS </italic> showed fluctuations and failed to achieve a consistent expression pattern [ <xref rid="pone.0189405.ref001" ref-type="bibr"> 1 </xref> ]. Under salinity or <span class="trait">drought</span> stress, the expression of salinity and <span class="trait">drought</span> response gene <italic> FaWRKY </italic> in roots of tall fescue peaked at 3 h post salt stress with the most stable reference genes ( <italic> SAND </italic> and <italic> <span class="gene">TUB</span> </italic> ) used, however, the expression of <italic> FaWRKY1 </italic> did not show a consistent pattern with the least stable gene ( <italic> EF1&#x003b1; </italic> ) used [ <xref rid="pone.0189405.ref035" ref-type="bibr"> 35 </xref> ]. These data indicated that use of suitable internal controls could reveal a more reliable result and is critical for RT-qPCR analysis. Thus, the selection of suitable reference genes in RT-qPCR analysis is pivotal to normalize the transcript expression of target genes.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>