<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Nitrogen Efficiency</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Nitrogen Efficiency</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Heterologous expression of ATG8c from soybean confers tolerance to <span class="trait">nitrogen</span> deficiency and increases <span class="trait">yield</span> in Arabidopsis.</h2>
			<h3>PMID:22629371  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xia T(1), Xiao D, Liu D, Chai W, Gong Q, Wang NN.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Nitrogen</span> is an essential element for plant growth and <span class="trait">yield.</span> Improving <span class="trait">Nitrogen</span> Use <span class="trait">Efficiency</span> (NUE) of crops could potentially reduce the application of chemical fertilizer and alleviate environmental damage. To identify new NUE genes is therefore an important task in molecular breeding. Macroautophagy (autophagy) is an intracellular process in which damaged or obsolete cytoplasmic components are encapsulated in double membraned vesicles termed autophagosomes, then delivered to the vacuole for degradation and nutrient recycling. One of the core components of autophagosome formation, ATG8, has been shown to directly mediate autophagosome expansion, and the transcript of which is highly inducible upon starvation. Therefore, we postulated that certain homologs of Saccharomyces cerevisiae ATG8 (ScATG8) from crop species could have potential for NUE crop breeding. A soybean (Glycine max, cv. Zhonghuang-13) ATG8, <span class="gene">GmATG8c,</span> was selected from the 11 family members based on transcript analysis upon <span class="trait">nitrogen</span> deprivation. <span class="gene">GmATG8c</span> could partially complement the yeast atg8 mutant. Constitutive expression of <span class="gene">GmATG8c</span> in soybean callus cells not only enhanced <span class="trait">nitrogen</span> starvation tolerance of the cells but accelerated the growth of the calli. Transgenic Arabidopsis over-expressing <span class="gene">GmATG8c</span> performed better under extended <span class="trait">nitrogen</span> and carbon starvation conditions. Meanwhile, under optimum growth conditions, the transgenic plants grew faster, bolted earlier, produced larger primary and axillary inflorescences, eventually produced more seeds than the wild-type. In average, the <span class="trait">yield</span> was improved by 12.9%. We conclude that <span class="gene">GmATG8c</span> may serve as an excellent candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Comparative Analysis of CBS Domain-Containing Proteins in Soybean (Glycine max) and the Primary Function of <span class="gene">GmCBS21</span> in Enhanced Tolerance to Low <span class="trait">Nitrogen</span> Stress.</h2>
			<h3>PMID:27128900  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hao Q(1)(2), Shang W(3)(4), Zhang C(5)(6), Chen H(7)(8), Chen L(9)(10), Yuan S(11)(12), Chen S(13), Zhang X(14), Zhou X(15)(16).
				<br/> 
				年份：2016 Apr 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Nitrogen</span> is an important macronutrient required for plant growth, and is a limiting factor for crop productivity. Improving the <span class="trait">nitrogen</span> use <span class="trait">efficiency</span> (NUE) is therefore crucial. At present, the NUE mechanism is unclear and information on the genes associated with NUE in soybeans is lacking. cystathionine beta synthase (CBS) domain-containing proteins (CDCPs) may be implicated in abiotic stress tolerance in plants. We identified and classified a CBS domain-containing protein superfamily in soybean. A candidate gene for NUE, <span class="gene">GmCBS21,</span> was identified. <span class="gene">GmCBS21</span> gene characteristics, the temporal expression pattern of the <span class="gene">GmCBS21</span> gene, and the phenotype of <span class="gene">GmCBS21</span> overexpression in transgenic Arabidopsis thaliana under low <span class="trait">nitrogen</span> stress were analyzed. The phenotypes suggested that the transgenic Arabidopsis thaliana seedlings performed better under the nitrogen-deficient condition. <span class="gene">GmCBS21-overexpressing</span> transgenic plants exhibit higher low <span class="trait">nitrogen</span> stress tolerance than WT plants, and this suggests its role in low <span class="trait">nitrogen</span> stress tolerance in plants. We conclude that <span class="gene">GmCBS21</span> may serve as an excellent candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:MicroRNA167-Directed Regulation of the Auxin Response Factors <span class="gene">GmARF8a</span> and <span class="gene">GmARF8b</span> Is Required for Soybean Nodulation and Lateral Root Development.</h2>
			<h3>PMID:25941314  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang Y(1), Li K(1), Chen L(1), Zou Y(1), Liu H(1), Tian Y(1), Li D(1), Wang R(1), Zhao F(1), Ferguson BJ(1), Gresshoff PM(1), Li X(2).
				<br/> 
				年份：2015 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume root nodules convert atmospheric <span class="trait">nitrogen</span> gas into ammonium through symbiosis with a prokaryotic microsymbiont broadly called rhizobia. Auxin signaling is required for determinant nodule development; however, the molecular mechanism of auxin-mediated nodule formation remains largely unknown. Here, we show in soybean (Glycine max) that the microRNA miR167 acts as a positive regulator of lateral root organs, namely nodules and lateral roots. miR167c expression was up-regulated in the vasculature, pericycle, and cortex of soybean roots following inoculation with Bradyrhizobium japonicum strain USDA110 (the microsymbiont). It was found to positively regulate nodule numbers directly by repressing the target genes <span class="gene">GmARF8a</span> and <span class="gene">GmARF8b</span> (homologous genes of Arabidopsis [Arabidopsis thaliana] AtARF8 that encode auxin response factors). Moreover, the expression of miR167 and its targets was up- and down-regulated by auxin, respectively. The miR167-GmARF8 module also positively regulated nodulation <span class="trait">efficiency</span> under low microsymbiont density, a condition often associated with environmental stress. The regulatory role of miR167 on nodule initiation was dependent on the Nod factor receptor <span class="gene">GmNFR1α,</span> and it acts upstream of the nodulation-associated genes nodule inception, nodulation signaling pathway1, early nodulin40-1, <span class="gene">NF-YA1</span> (previously known as HAEM activator protein2-1), and NF-YA2. miR167 also promoted lateral root numbers. Collectively, our findings establish a key role for the miR167-GmARF8 module in auxin-mediated nodule and lateral root formation in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Influence of elevated atmospheric carbon dioxide on transcriptional responses of Bradyrhizobium japonicum in the soybean rhizoplane.</h2>
			<h3>PMID:23666536  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sugawara M(1), Sadowsky MJ.
				<br/> 
				年份：2013
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elevated atmospheric CO2 can influence the structure and function of rhizoplane and rhizosphere microorganisms by altering root growth and the quality and quantity of compounds released into the rhizoplane and rhizosphere via root exudation. In these studies we investigated the transcriptional responses of Bradyrhizobium japonicum cells growing in the rhizoplane of soybean plants exposed to elevated atmospheric CO2. The results of microarray analyses indicated that elevated atmospheric CO2 concentration indirectly influenced the expression of a large number of genes in Bradyrhizobium attached to soybean roots. In addition, relative to plants and bacteria grown under ambient CO2 growth conditions, genes involved in <span class="gene">C1</span> metabolism, denitrification and FixK2-associated genes, including those involved in <span class="trait">nitrogen</span> fixation, microaerobic respiration, respiratory nitrite reductase, and heme biosynthesis, were significantly up-regulated under conditions of elevated CO2 in the rhizosphere. The expression profile of genes involved in lipochitooligosaccharide Nod factor biosynthesis and negative transcriptional regulators of nodulation genes, <span class="gene">nolA</span> and nodD2, were also influenced by plant growth under conditions of elevated CO2. Taken together, the results of these studies indicate that the growth of soybeans under conditions of elevated atmospheric CO2 influences gene expressions in B. japonicum in the soybean rhizoplane, resulting in changes to carbon/nitrogen metabolism, respiration, and nodulation <span class="trait">efficiency.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nitrogen utilization in pigs fed diets with soybean and rapeseed products leading to different ileal endogenous <span class="trait">nitrogen</span> losses.</h2>
			<h3>PMID:9498367  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grala W(1), Verstegen MW, Jansman AJ, Huisman J, Wasilewko J.
				<br/> 
				年份：1998 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Nitrogen</span> (N) balance was determined in 36 pigs (BW 24 to 30 kg) fed diets inducing different ileal endogenous N losses (ENL). We tested the hypothesis that enhanced <span class="gene">ENL</span> may be indicative of a higher recycling of endogenous proteins that will induce a greater urinary N loss and a lower <span class="trait">efficiency</span> of the dietary N utilization for retention. The cornstarch-based diets contained either soy concentrate (SC), soybean meal (SBM), a mixture of toasted and untoasted soybean meal (mSBM), dehulled-toasted rapeseed cake (RC1), non-dehulled-toasted rapeseed cake (RC2), or dehulled-untoasted rapeseed cake (RC3). The diets were balanced for their content of apparent ileal digestible (ID) CP (108 g/kg feed) and apparent ID of Lys, Thr, Met+Cys, Trp, and Ile. Feeding level was 2.7 times ME for maintenance per kilogram BWx75 and restricted to 88% of the requirements for ID Lys as the first-limiting amino acid. During a 5-d period, urine and feces were collected daily in metabolism cages. Compared with the SC diet (low ENL), the diets with SBM (medium ENL) and mSBM (high ENL) resulted in a greater (P &lt; .05) urinary N excretion. <span class="trait">Nitrogen</span> retention tended to be less (P = .08) in pigs fed diets that caused greater ENL. The utilization of ID N for retention in pigs fed the mSBM diet was lower (P &lt; .05) than for those fed the SC diet. There were no differences in urinary N excretion, N retention, and the utilization of ID N for retention in pigs fed the rapeseed diets of different fiber contents (hulls as the NDF source). We concluded that, at similar intakes of the first-limiting ID amino acid, N retention in pigs fed soybeans tended to be reduced by greater <span class="gene">ENL</span> as induced by antinutritional factors (e.g., trypsin inhibitors). Rapeseed hulls, as the predominant fiber source, do not affect N retention and the utilization of ID N for retention.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rj4, a Gene Controlling Nodulation Specificity in Soybeans, Encodes a Thaumatin-Like Protein But Not the One Previously Reported.</h2>
			<h3>PMID:26582727  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tang F(1), Yang S(1), Liu J(1), Zhu H(2).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Rj4</span> is a dominant gene in soybeans (Glycine max) that restricts nodulation by many strains of Bradyrhizobium elkanii. The soybean-B. elkanii symbiosis has a low nitrogen-fixation <span class="trait">efficiency,</span> but B. elkanii strains are highly competitive for nodulation; thus, cultivars harboring an <span class="gene">Rj4</span> allele are considered favorable. Cloning the <span class="gene">Rj4</span> gene is the first step in understanding the molecular basis of Rj4-mediated nodulation restriction and facilitates the development of molecular tools for genetic improvement of <span class="trait">nitrogen</span> fixation in soybeans. We finely mapped the <span class="gene">Rj4</span> locus within a small genomic region on soybean chromosome 1, and validated one of the candidate genes as <span class="gene">Rj4</span> using both complementation tests and CRISPR/Cas9-based gene knockout experiments. We demonstrated that <span class="gene">Rj4</span> encodes a thaumatin-like protein, for which a corresponding allele is not present in the surveyed rj4 genotypes, including the reference genome Williams 82. Our conclusion disagrees with the previous report that <span class="gene">Rj4</span> is the <span class="gene">Glyma.01G165800</span> gene (previously annotated as <span class="gene">Glyma01g37060).</span> Instead, we provide convincing evidence that <span class="gene">Rj4</span> is <span class="gene">Glyma.01g165800-D,</span> a duplicated and unique version of <span class="gene">Glyma.01g165800,</span> that has evolved the ability to control symbiotic specificity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Search for Nodulation and Nodule Development-Related Cystatin Genes in the Genome of Soybean ( <italic> Glycine max </italic> )</h2>
			<h3>PMID:27826313</h3>
				

				
					<h3>PMCID:5078837</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Songli </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/308749/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/349967/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Haifeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chanjuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Limiao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qingnan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/301356/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Zhihui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaojuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shuilian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Zhonglu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Dezhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/355169/overview"/>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 10 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nodulation, nodule development and senescence directly affects <span class="trait">nitrogen</span> fixation <span class="trait">efficiency,</span> and previous studies have shown that inhibition of some cysteine proteases delay nodule senescence, so their nature inhibitors, cystatin genes, are very important in nodulation, nodule development, and senescence. Although several cystatins are actively transcribed in soybean nodules, their exact roles and functional diversities in legume have not been well explored in genome-wide survey studies. In this report, we performed a genome-wide survey of cystatin family genes to explore their relationship to nodulation and nodule development in soybean and identified 20 cystatin genes that encode peptides with 97-245 amino acid residues, different isoelectric points (pI) and structure characteristics, and various putative plant regulatory elements in 3000 bp putative promoter fragments upstream of the 20 soybean cystatins in response to different abiotic/biotic stresses, hormone signals, and symbiosis signals. The expression profiles of these cystatin genes in soybean symbiosis with rhizobium strain Bradyrhizobium japonicum strain 113-2 revealed that 7 cystatin family genes play different roles in nodulation as well as nodule development and senescence. However, these genes were not root nodule symbiosis (RNS)-specific and did not encode special clade cystatin protein with structures related to nodulation and nodule development. Besides, only two of these soybean cystatins were not upregulated in symbiosis after ABA treatment. The functional analysis showed that a candidate gene <span class="gene">Glyma.15G227500</span> <span class="gene">(GmCYS16)</span> was likely to play a positive role in soybean nodulation. Besides, evolutionary relationships analysis divided the cystatin genes from Arabidopsis thaliana, Nicotiana tabacum, rice, barley and four legume plants into three groups. Interestingly, Group A cystatins are special in legume plants, but only include one of the above-mentioned 7 cystatin genes related to nodulation and nodule development. Overall, our results provide useful information or clues for our understanding of the functional diversity of legume cystatin family proteins in soybean nodulation and nodule development and for finding nodule-specific cysteine proteases in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						As Table <xref ref-type="table" rid="T2"> 2 </xref> described, 80% soybean cystatin genes have abscisic acid responsive elements (ABRE or CE3), suggesting that their expressions are induced by ABA, a negative regulator of symbiotic <span class="trait">nitrogen</span> fixation (Tominaga et al., <xref rid="B49" ref-type="bibr"> 2009 </xref> , <xref rid="B50" ref-type="bibr"> 2010 </xref> ) that decreases nodule number (Phillips, <xref rid="B34" ref-type="bibr"> 1971 </xref> ; Suzuki et al., <xref rid="B45" ref-type="bibr"> 2004 </xref> ; Ding et al., <xref rid="B11" ref-type="bibr"> 2008 </xref> ; Biswas et al., <xref rid="B5" ref-type="bibr"> 2009 </xref> ) and negatively regulates <span class="trait">nitrogen</span> fixation (Gonz&#x000e1;lez et al., <xref rid="B13" ref-type="bibr"> 2001 </xref> ). To confirm the response of soybean cystatins to ABA in symbiosis, the transcriptional levels of 10 soybean cystatins that contain ABRE and/or CE3 elements, and are expressed in inoculated roots (Tables <xref ref-type="table" rid="T2"> 2 </xref> , <xref ref-type="table" rid="T3"> 3 </xref> ) at 6, 33, and 62 d of post-inoculation were compared using <italic> t </italic> -tests to show statistical difference among different treatment groups (with nodules, ddH <sub> 2 </sub> O and ABA treatment) (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). Three soybean cystatins ( <italic> <span class="gene">GmCYS2,</span> <span class="gene">GmCYS9</span> </italic> and <italic> <span class="gene">GmCYS17</span> </italic> ) were upregulated (&#x0003e;2 fold) after ABA treatment in inoculated roots at 6, 33, and 62 d of post-inoculation (Figures <xref ref-type="fig" rid="F6"> 6A,C,I </xref> ). The expression of <italic> <span class="gene">GmCYS16</span> </italic> was increased (&#x0003e;2 fold) after ABA treatment in inoculated roots at 6 and 62 d of post-inoculation (Figure <xref ref-type="fig" rid="F6"> 6H </xref> ), the expression level of <italic> <span class="gene">GmCYS8,</span> <span class="gene">GmCYS14</span> </italic> and <italic> <span class="gene">GmCYS15</span> </italic> was increased (&#x0003e;2 fold) after ABA treatment in inoculated roots at 6 d of post-inoculation (Figures <xref ref-type="fig" rid="F6"> 6B,F,G </xref> ), and the expression level of <italic> <span class="gene">GmCYS12</span> </italic> was increased (&#x0003e;2 fold) after ABA treatment in inoculated roots at 33 d of post-inoculation (Figure <xref ref-type="fig" rid="F6"> 6E </xref> ). The expression of <italic> <span class="gene">GmCYS10</span> </italic> was not changes (&#x0003c;2 fold) at the transcription level in soybean inoculated roots after ABA treatment (Figure <xref ref-type="fig" rid="F6"> 6D </xref> ), suggesting that its expressions were not induced by ABA in soybean symbiosis with <italic> B. japonicum </italic> strain 113-2. Whether its expressions were induced by ABA in non-inoculated roots and/or other tissues need to be further explored.
					</p>
				
					<p>&nbsp;&nbsp;
						As described above, when inoculated with <italic> B. japonicum </italic> strain 113-2, the expression of <italic> <span class="gene">GmCYS16</span> </italic> in soybean roots was significantly increased (Figure <xref ref-type="fig" rid="F4"> 4J </xref> ), suggesting that <italic> <span class="gene">GmCYS16</span> </italic> may plays a role in nodulation. To confirm this result, <italic> <span class="gene">Glyma.15G227500</span> </italic> ( <italic> <span class="gene">GmCYS16</span> </italic> ) was expressed under the control of the maize ( <italic> Zea mays </italic> ) ubiquitin promoter <span class="gene">(Glyma.15G227500-OX)</span> in transgenic hairy roots of <italic> L. japonicus </italic> (Kumagai and Kouchi, <xref rid="B22" ref-type="bibr"> 2003 </xref> ; Yuan et al., <xref rid="B56" ref-type="bibr"> 2012 </xref> ). The phenotypes of nodulation were scored at 32 days of post-inoculation with <italic> M. loti </italic> MAFF303099, which expresses &#x003b2;-galactosidase (lacZ) as a constitutive marker for the presence of rhizobial cells (Tansengco et al., <xref rid="B48" ref-type="bibr"> 2003 </xref> ). As shown in Figures <xref ref-type="fig" rid="F7"> 7A,B </xref> , more nodules were produced in <span class="gene">Glyma.15G227500-OX</span> hairy roots. The nodule number per root system was increased from 5.0976 in the control to 8.7 in <span class="gene">Glyma.15G227500-OX</span> hairy roots (Figure <xref ref-type="fig" rid="F7"> 7Ba </xref> ). Analysis of the distribution of nodule number per plant revealed that 40% of <span class="gene">Glyma.15G227500-OX</span> hairy roots developed more than 10 nodules. In contrast, only 15% of the control hairy roots produced more than 10 nodules, and no hairy roots produced more than 20 nodules (Figure <xref ref-type="fig" rid="F7"> 7Bb </xref> ). Semi-quantitative RT-PCR results showed that the expression level of <italic> <span class="gene">Glyma.15G227500</span> </italic> was abundant in <span class="gene">Glyma.15G227500-OX</span> hairy roots (Figure <xref ref-type="fig" rid="F7"> 7C </xref> ). qPCR analyses of expressions of <italic> NIN </italic> and <italic> <span class="gene">ENOD40</span> </italic> , two early nodulin genes implicated in the processes of rhizobial entrance, nodule initiation, and subsequent organogenesis (Schauser et al., <xref rid="B38" ref-type="bibr"> 1999 </xref> ; Kumagai et al., <xref rid="B21" ref-type="bibr"> 2006 </xref> ), as well as <italic> Lb </italic> (leghemoglobin), a typical nodulin gene required for <span class="trait">nitrogen</span> fixation (Ott et al., <xref rid="B31" ref-type="bibr"> 2005 </xref> ) showed that the expression levels of these three nodulin genes, especially <italic> NIN </italic> and <italic> Lb </italic> , were increased in <span class="gene">Glyma.15G227500-OX</span> hairy roots, as compared with those in the control hairy roots, (Figure <xref ref-type="fig" rid="F7"> 7D </xref> ). These results suggest that <span class="gene">Glyma.15G227500</span> plays a positive role in nodulation of <italic> L. japonicus </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Previous reports demonstrated that cystatin could inhibit activity of cysteine proteases in the process of plant seed germination (Martinez et al., <xref rid="B27" ref-type="bibr"> 2009 </xref> ) and host immunity (Koiwa et al., <xref rid="B20" ref-type="bibr"> 2000 </xref> ; van der Linde et al., <xref rid="B51" ref-type="bibr"> 2012 </xref> ), and also play a key role in hypersensitive cell death and the regulation of defense against pathogens and insects (Bobek and Levine, <xref rid="B7" ref-type="bibr"> 1992 </xref> ; Belenghi et al., <xref rid="B2" ref-type="bibr"> 2003 </xref> ; Gholizadeh et al., <xref rid="B12" ref-type="bibr"> 2005 </xref> ; Pog&#x000e1;ny et al., <xref rid="B35" ref-type="bibr"> 2015 </xref> ). Besides, their expression is also responsive to abiotic stresses and hormone signals (Zhang et al., <xref rid="B57" ref-type="bibr"> 2008 </xref> ; Wang et al., <xref rid="B54" ref-type="bibr"> 2015 </xref> ). Similar to this study, various putative plant regulatory elements in response to different abiotic/biotic stresses (various environmental stresses) and hormone signals were shown in 3000 bp putative promoter fragments upstream of the 20 soybean cystatins (Table <xref ref-type="table" rid="T2"> 2 </xref> ). The 7 nodulation and nodule development-related cystatin proteins have inconsistent plant regulatory elements in their promoters (Table <xref ref-type="table" rid="T2"> 2 </xref> ). For hormones responsive elements, <italic> <span class="gene">GmCYS2</span> </italic> has elements in response to 5 different hormones, including abscisic acid, gibberellin acid, salicylic acid, auxin and jasmonic acid, <italic> <span class="gene">GmCYS12,</span> <span class="gene">GmCYS15,</span> <span class="gene">GmCYS16</span> </italic> , and <italic> <span class="gene">GmCYS17</span> </italic> have no auxin responsive elements, <italic> <span class="gene">GmCYS15</span> </italic> has no jasmonic acid responsive element, <italic> <span class="gene">GmCYS18</span> </italic> has no abscisic acid and gibberellin acid responsive elements, and <italic> <span class="gene">GmCYS20</span> </italic> has no gibberellin acid responsive elements. For environmental responsive elements, <italic> <span class="gene">GmCYS15,</span> <span class="gene">GmCYS16</span> </italic> , and <italic> <span class="gene">GmCYS18</span> </italic> have no MBS element, <italic> <span class="gene">GmCYS12</span> </italic> has no TC-rich repeats element, and <italic> <span class="gene">GmCYS15</span> </italic> has no circadian-regulated elements. Besides, <italic> <span class="gene">GmCYS2,</span> <span class="gene">GmCYS15,</span> <span class="gene">GmCYS17</span> </italic> , and <italic> <span class="gene">GmCYS18</span> </italic> have fungal elicitor responsive element, <italic> <span class="gene">GmCYS15</span> </italic> and <italic> <span class="gene">GmCYS17</span> </italic> have wound-responsive element, and <italic> <span class="gene">GmCYS15,</span> <span class="gene">GmCYS17</span> </italic> , and <italic> <span class="gene">GmCYS20</span> </italic> have ethylene-responsive element. Interesting, <italic> <span class="gene">GmCYS2</span> </italic> has a nodule specific factor binding site element (Nodule-site2, Table <xref ref-type="table" rid="T2"> 2 </xref> ), which also exists in the promoter fragment of soybean leghemoglobin ( <italic> LBC3 </italic> ), a typical nodulin gene required for <span class="trait">nitrogen</span> fixation (She et al., <xref rid="B41" ref-type="bibr"> 1993 </xref> ). However, whether <italic> <span class="gene">GmCYS2</span> </italic> plays an important role in the soybean <span class="trait">nitrogen</span> fixation needs to be further studied. These results indicated that these 7 soybean cystatin proteins are not root nodule symbiosis (RNS)&#x02014;specific cystatins, and have different putative biological functions.
					</p>
				
					<p>&nbsp;&nbsp;
						As described in the evolutionary relationships among the cystatin family in rice, barley, <italic> A. thaliana, N. tabacum </italic> and four legume plants, Group A cystatins are special branch for legume plant, Group B cystatins only contain non-legume plant cystatins (non-legume plant dicotyledon and monocotyledon), and Group C cystatins are both legume plants and non-legume plant related (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). Of the 7 cystatins related to nodulation and nodule development, only <italic> <span class="gene">GmCYS15</span> </italic> fall into Group A, the other 6 cystatins fall into Group C (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). These results indicated that cystatins in the special branch are not special for legume property-nodulation and <span class="trait">nitrogen</span> fixation and may possess other biological functions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">NopC</span> Is a <italic> Rhizobium </italic> -Specific Type 3 Secretion System Effector Secreted by <italic> Sinorhizobium </italic> ( <italic> Ensifer </italic> ) <italic> fredii </italic> HH103</h2>
			<h3>PMID:26569401</h3>
				

				
					<h3>PMCID:4646503</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jim&#x000e9;nez-Guerrero </surname>  <given-names> Irene </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> P&#x000e9;rez-Monta&#x000f1;o </surname>  <given-names> Francisco </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Medina </surname>  <given-names> Carlos </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ollero </surname>  <given-names> Francisco Javier </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> L&#x000f3;pez-Baena </surname>  <given-names> Francisco Javier </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sinorhizobium (Ensifer) fredii HH103 is a broad host-range nitrogen-fixing bacterium able to nodulate many legumes, including soybean. In several rhizobia, root nodulation is influenced by proteins secreted through the type 3 secretion system (T3SS). This specialized secretion apparatus is a common virulence mechanism of many plant and animal pathogenic bacteria that delivers proteins, called effectors, directly into the eukaryotic host cells where they interfere with signal transduction pathways and promote infection by suppressing host defenses. In rhizobia, secreted proteins, called nodulation outer proteins (Nops), are involved in host-range determination and symbiotic <span class="trait">efficiency.</span> S. fredii HH103 secretes at least eight Nops through the T3SS. Interestingly, there are Rhizobium-specific Nops, such as NopC, which do not have homologues in pathogenic bacteria. In this work we studied the S. fredii HH103 <span class="gene">nopC</span> gene and confirmed that its expression was regulated in a flavonoid-, NodD1- and TtsI-dependent manner. Besides, in vivo bioluminescent studies indicated that the S. fredii HH103 T3SS was expressed in young soybean nodules and adenylate cyclase assays confirmed that NopC was delivered directly into soybean root cells by means of the T3SS machinery. Finally, nodulation assays showed that NopC exerted a positive effect on symbiosis with Glycine max cv. Williams 82 and Vigna unguiculata. All these results indicate that NopC can be considered a Rhizobium-specific effector secreted by S. fredii HH103.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Sinorhizobium </italic> ( <italic> Ensifer </italic> ) <italic> fredii </italic> HH103 is a broad host-range nitrogen-fixing bacterium able to nodulate many legumes, including soybean. In several rhizobia, root nodulation is influenced by proteins secreted through the type 3 secretion system (T3SS). This specialized secretion apparatus is a common virulence mechanism of many plant and animal pathogenic bacteria that delivers proteins, called effectors, directly into the eukaryotic host cells where they interfere with signal transduction pathways and promote infection by suppressing host defenses. In rhizobia, secreted proteins, called nodulation outer proteins (Nops), are involved in host-range determination and symbiotic <span class="trait">efficiency.</span> <italic> S </italic> . <italic> fredii </italic> HH103 secretes at least eight Nops through the T3SS. Interestingly, there are <italic> Rhizobium </italic> -specific Nops, such as NopC, which do not have homologues in pathogenic bacteria. In this work we studied the <italic> S </italic> . <italic> fredii </italic> HH103 <italic> <span class="gene">nopC</span> </italic> gene and confirmed that its expression was regulated in a flavonoid-, NodD1- and TtsI-dependent manner. Besides, <italic> in vivo </italic> bioluminescent studies indicated that the <italic> S </italic> . <italic> fredii </italic> HH103 T3SS was expressed in young soybean nodules and adenylate cyclase assays confirmed that NopC was delivered directly into soybean root cells by means of the T3SS machinery. Finally, nodulation assays showed that NopC exerted a positive effect on symbiosis with <italic> Glycine max </italic> cv. Williams 82 and <italic> Vigna unguiculata </italic> . All these results indicate that NopC can be considered a <italic> Rhizobium </italic> -specific effector secreted by <italic> S </italic> . <italic> fredii </italic> HH103.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine whether <italic> S </italic> . <italic> fredii </italic> HH103 T3SS translocates the NopC-Cya fusion protein into soybean root cells, the protocol described by [ <xref rid="pone.0142866.ref024" ref-type="bibr"> 24 </xref> ] was used with some modifications. Eighteen pre-germinated soybean seeds were aseptically transferred to a recipient containing vermiculite and 150 ml of a F&#x000e5;hraeus solution 1M pH 6.8 and grown in a controlled environment chamber with a 16 h day/8 h night cycle and a relative humidity of 70%. Growth temperatures were set to 26&#x000b0;C during the day period and to 18&#x000b0;C during the night. The system was inoculated at the time of transferring the pre-germinated seeds with bacterial cultures of about 10 <sup> 8 </sup> c.f.u.ml <sup> -1 </sup> of the HH103 Rif <sup> R </sup> or the <italic> <span class="gene">ttsI</span> </italic> &#x02126; mutant strains, both containing a chromosomal integration of the <italic> nopC-cya </italic> fusion. Cyclic AMP (cAMP) accumulation was measured in nodules harvested 18 d.p.i. Nodules were frozen in liquid <span class="trait">nitrogen,</span> ground to a fine powder and resuspended in a 0.1 M hydrochloric acid solution. The suspension was centrifuged and the supernatant was used for cAMP measurement using the cyclic AMP (direct) EIA kit (Cayman Chemical Company, USA) according to the manufacturer&#x02019;s instructions. Each sample was diluted for quantification to measure cAMP concentration in the detection range of the assay. The HH103 Rif <sup> R </sup> strain without the <italic> <span class="gene">nopC</span> </italic> - <italic> cya </italic> fusion was used as a control for quantification.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription of the T3SS-genes are down-regulated in soybean bacteroids [ <xref rid="pone.0142866.ref044" ref-type="bibr"> 44 </xref> ] and previous reports detected <italic> S </italic> . <italic> fredii </italic> USDA257 NopX (previously called NolX) in infection threads but not in mature nodules of soybean [ <xref rid="pone.0142866.ref040" ref-type="bibr"> 40 </xref> ], suggesting that the T3SS was induced early in the symbiosis with soybean but repressed at later stages. However, results shown in this work of the <italic> in vivo </italic> monitoring of the activity of the <italic> tts </italic> box upstream <italic> <span class="gene">nopC</span> </italic> fused to <italic> lux </italic> genes showed that the HH103 T3SS was expressed in some but not all young soybean nodules ( <xref rid="pone.0142866.g005" ref-type="fig"> Fig 5A </xref> ). The fact that bioluminescence was not detected in all nodules could have some explanations. First, plasmid pMUS1207 (derived from the high-copy plasmid pMP92) could be lost by part of the rhizobial population. Both bioluminescence and <span class="trait">nitrogen</span> fixation require high energy consumption and the promoter cloned shows very high activity with inducer flavonoids. Therefore, the presence of the plasmid and the production of bioluminescence since the early stages of the infection process could have a negative impact on bacterial fitness. In addition, luciferase needs oxygen and ATP for its activity and the atmosphere in the symbiosome, necessary for <span class="trait">nitrogen</span> fixation, is deficient in oxygen. Therefore, bioluminescence would not be detected in mature nodules, making this method only suitable for instant detection of gene expression in the early stages of the symbiosis, time period that coincides with the moment in which the T3SS shows its highest activity. Nevertheless, the objective of the use of this reporter was the selection of a particular root region/tissue with the best characteristics to further measure the production of cAMP in the Cya assay.
					</p>
				
					<p>&nbsp;&nbsp;
						Rhizobial proteins secreted through the T3SS are involved in host-range determination and symbiotic <span class="trait">efficiency</span> [ <xref rid="pone.0142866.ref013" ref-type="bibr"> 13 </xref> ]. The positive, negative or neutral effects on symbiosis of each effector are determined by the host plant. For instance, in the symbioses between <italic> S </italic> . <italic> fredii </italic> HH103 and its host legumes, the inactivation of the T3SS is beneficial for the symbioses with several cultivars of <italic> G </italic> . <italic> max </italic> (including Williams 82) and <italic> Glycyrrhiza uralensis </italic> and detrimental on <italic> Erythrina variegata </italic> [ <xref rid="pone.0142866.ref017" ref-type="bibr"> 17 </xref> ]. Results shown in this work indicated that NopC had a positive role in the symbiosis between HH103 and both soybean cv. Williams 82 and <italic> V </italic> . <italic> unguiculata </italic> , since the mutation of this gene caused a decrease in the number and the fresh mass of the nodules formed in both plants (Tables <xref rid="pone.0142866.t003" ref-type="table"> 3 </xref> and <xref rid="pone.0142866.t004" ref-type="table"> 4 </xref> ). This positive effect on soybean symbiosis was also observed in plants inoculated with the <italic> <span class="gene">nopA</span> </italic> mutant strain ( <xref rid="pone.0142866.t003" ref-type="table"> Table 3 </xref> ). Interestingly, the symbiotic effect in <italic> V </italic> . <italic> unguiculata </italic> of the mutation of <italic> <span class="gene">nopA</span> </italic> was opposite to that observed with the <italic> <span class="gene">nopC</span> </italic> mutant ( <xref rid="pone.0142866.t004" ref-type="table"> Table 4 </xref> ), indicating that both Nops are playing different roles in the symbiotic process. Besides, results suggest that NopC is contributing, together with still unknown effectors, to an effective symbiosis with both plants.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Phylogenetic relationship of Lotus uliginosus symbionts with bradyrhizobia nodulating genistoid legumes.</h2>
			<h3>PMID:22092879  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lorite MJ(1), Videira e Castro I, Muñoz S, Sanjuán J.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Lotus species are legumes with potential for pastures in soils with low-fertility and environmental constraints. The aim of this work was to characterize bacteria that establish efficient nitrogen-fixing symbiosis with the forage species Lotus uliginosus. A total of 39 isolates were obtained from nodules of L. uliginosus naturally growing in two different locations of Portugal. Molecular identification of the isolates plus the commercial inoculant strain NZP2039 was performed by REP-PCR, 16S rRNA RFLP, and 16S rRNA, <span class="gene">glnII</span> and <span class="gene">recA</span> sequence analyses. Limited genetic diversity was found among the L. uliginosus symbionts, which showed a close phylogenetic relationship with the species Bradyrhizobium japonicum. The symbiotic nifH, <span class="gene">nodA</span> and <span class="gene">nodC</span> gene sequences were closely related with the corresponding genes of various Bradyrhizobium strains isolated from Lupinus and other genistoid legumes and therefore were phylogenetically separated from other Lotus spp. rhizobia. The L. uliginosus bradyrhizobia were able to nodulate and fix <span class="trait">nitrogen</span> in association with L. uliginosus, could nodulate Lotus corniculatus with generally poor nitrogen-fixing <span class="trait">efficiency,</span> formed nonfixing nodules in Lotus tenuis and Lupinus luteus roots and were unable to nodulate Glycine soja or Glycine max. Thus, L. uliginosus rhizobia seem closely related to B. japonicum biovar genistearum strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[A study of the effects on the symbiotic <span class="trait">nitrogen</span> fixation of Sinorhizobium fredii with the introduction of dctABD and <span class="gene">nifA</span> genes].</h2>
			<h3>PMID:11902003  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li YG(1), Zhou JC.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A recombinant plasmid pHN307 containing C4-dicarboxylic acid transport genes (dctABD) from Sinorhizobium meliloti, <span class="gene">nifA</span> genes from Klebsiella pneumoniae and reporter genes luxAB from pDB30 was constructed by using pTR102 as the vector. The pHN307 was then introduced into the S. fredii HN01, YC4 and GR3 by tri-parental mating, and the stability of pHN307 in the transconjugants under free-living and symbiotic condition was also investigated. The results of plant pot experiment indicated that the introduction of pHN307 in the transconjugants could significantly increase the nodule fresh weight, shoot dry weight (biomass) and total <span class="trait">nitrogen</span> content of the symbiotic plants with soybean variety of Heilong 33. When the transconjugants were in symbiotic with soybean variety of Chuanzao No. 1, HN01 (pHN307) could significantly increase its root nodule number and fresh weight; GR3 (pHN307) could significantly increase its root nodule number, nodule fresh weight, shoot dry weight and total <span class="trait">nitrogen</span> content, but YC4(pHN307) demonstrated negative effect under the same condition. The results of this study suggested that the introduction of dctABD and <span class="gene">nifA</span> could improve the symbiotic <span class="trait">nitrogen</span> fixation <span class="trait">efficiency</span> and nodulation ability of the rhizobia tested, respectively, and its effect was relevant to the characteristics of recipient rhizobia and soybean variety.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of whole-crop pea (Pisum sativum L.) silages differing in condensed tannin content as a substitute for grass silage and soybean meal on the performance, metabolism, and carcass characteristics of lambs.</h2>
			<h3>PMID:21571892  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hart KJ(1), Sinclair LA, Wilkinson RG, Huntington JA.
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two experiments were conducted to investigate the effect of inclusion of whole-crop pea (WCP) silages, differing in condensed tannin content, as a substitute for grass silage (GS) and soybean meal on lamb metabolism, performance, plasma metabolites, digestibility, and carcass characteristics. In both experiments lambs were offered either solely <span class="gene">GS</span> or a 50:50 mix on a DM basis of <span class="gene">GS</span> with either low-tannin (LTPS) or high-tannin (HTPS) pea silage ad libitum. Each forage mix was fed with either 400 g/d of low-protein (LP) concentrate or 400 g/d of LP with an additional 200 g/d of pelletized soybean meal (HP), resulting in 6 dietary treatments. Experiment 1 examined the effects of the diets on metabolism, digestibility, and N balance using 6 lambs in 4 periods of 21 d in an incomplete crossover design. Experiment 2 used 48 lambs and examined the effects of the diets on ADG, plasma metabolites, and carcass characteristics over 56 d. Both experiments were analyzed using a 3 × 2 factorial arrangement of treatments. In Exp. 1, lambs offered the LTPS diets had a greater (P &lt; 0.05) digestibility of DM and OM than those offered the <span class="gene">GS</span> diets. Lambs offered the WCP silages had an increased (P &lt; 0.05) N intake, N output, and digestibility of GE compared with those offered GS. Mean N digestibility was greatest (P &lt; 0.05) in lambs offered LTPS. Lambs offered HP diets had increased (P &lt; 0.001) digestibility of DM, OM, GE and N, and N- intake, output, retention, and digestibility compared with those offered the LP diets. In Exp. 2, there was no effect (P &gt; 0.05) of forage type on intake, slaughter BW, or feed conversion <span class="trait">efficiency</span> (FCE). However, lambs offered the LTPS had a greater (P &lt; 0.05) ADG than those offered the <span class="gene">GS</span> diets. Feeding diets containing HP increased (P &lt; 0.001) total DMI, slaughter BW, ADG, and FCE. Lambs offered the WCP had a greater (P &lt; 0.05) plasma β-hydroxybutyrate and urea concentration compared with those offered the <span class="gene">GS</span> diets. Feeding lambs HP diets increased (P &lt; 0.05) plasma urea and total protein. Forage mix had no effect (P &gt; 0.05) on carcass composition except for fat depth, which was greater (P &lt; 0.05) in lambs offered WCP silage. Diets containing the HP increased (P &lt; 0.05) carcass weight, hind leg circumference, chop dimensions, and kidney weight. It was concluded that lambs offered LTPS performed better than those offered <span class="gene">GS</span> and that LTPS has a concentrate sparing effect. Additionally, the increased tannin concentration in HTPS did not increase performance over lambs offered either <span class="gene">GS</span> or LTPS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Estimation of <span class="trait">nitrogen</span> maintenance requirements and potential for <span class="trait">nitrogen</span> deposition in fast-growing chickens depending on age and sex.</h2>
			<h3>PMID:16903472  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Samadi F(1), Liebert F.
				<br/> 
				年份：2006 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments were conducted to estimate daily N maintenance requirements (NMR) and the genetic potential for daily N deposition (ND(max)T) in fast-growing chickens depending on age and sex. In N-balance studies, 144 male and 144 female chickens (Cobb 500) were utilized in 4 consecutive age periods (I: 10 to 25 d; II: 30 to 45 d; III: 50 to 65 d; and IV: 70 to 85 d). The experimental diets contained high-protein soybean meal and crystalline amino acids as protein sources and 6 graded levels of protein supply (N1 = 6.6%; N2 = 13.0%; <span class="gene">N3</span> = 19.6%; N4 = 25.1%; N5 = 31.8%; and N6 = 37.6% CP in DM). The connection between N intake and total N excretion was fitted for NMR determination by an exponential function. The average NMR value (252 mg of N/BW(kg)0.67 per d) was applied for further calculation of ND(max)T as the threshold value of the function between N intake and daily N balance. For estimating the threshold value, the principle of the Levenberg-Marquardt algorithm within the SPSS program (Version 11.5) was applied. As a theoretical maximum for ND(max)T, 3,592, 2,723, 1,702, and 1,386 mg of N/BW(kg)0.67 per d for male and 3,452, 2,604, 1,501, and 1,286 mg of N/BW(kg)0.67 per d for female fast-growing chickens (corresponding to age periods I to IV) were obtained. The determined model parameters were the precondition for modeling of the amino acid requirement based on an exponential N-utilization model and depended on performance and dietary amino acid <span class="trait">efficiency.</span> This procedure will be further developed and applied in the subsequent paper.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">Rj4</span> allele in soybean represses nodulation by chlorosis-inducing bradyrhizobia classified as DNA homology group II by antibiotic <span class="trait">resistance</span> profiles.</h2>
			<h3>PMID:24220807</h3>
				

				
			<p>
				作者：Devine TE(1), Kuykendall LD, O'Neill JJ.
				<br/> 
				年份：1990 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To determine the relationship between nodulation restriction by the <span class="gene">Rj4</span> allele of soybean, rhizobitoxine-induced <span class="trait">chlorosis,</span> and taxonomic grouping of bradyrhizobia, 119 bradyrhizobial isolates were tested in Leonard jar culture for nodulation response and <span class="trait">chlorosis</span> induction. In addition to strain USDA 61, the strain originally reported as defining the <span class="gene">Rj4</span> response, eight other isolates (i.e., USDA 62, 83, 94, 238, 252, 259, 260, and 340) were discovered to elicit the nodulation interdiction of the <span class="gene">Rj4</span> allele. Only 16% of all the bradyrhizobial strains tested induced <span class="trait">chlorosis,</span> but seven of the nine strains (78%) interdicted by the <span class="gene">Rj4</span> allele were chlorosis-inducing strains. Furthermore, in tests for antibiotic <span class="trait">resistance</span> profile, eight of the nine interdicted strains (89%) were classed in DNA homology group II. This evidence suggests that the <span class="gene">Rj4</span> allele has a positive value to the host plant in shielding it from nodulation by certain chlorosis-inducing bradyrhizobia of a DNA homology group with impaired <span class="trait">efficiency</span> of <span class="trait">nitrogen</span> fixation with soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of alanine dehydrogenase and its role in mixed secretion of ammonium and alanine by pea bacteroids.</h2>
			<h3>PMID:10792736  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Allaway D(1), Lodwig EM, Crompton LA, Wood M, Parsons R, Wheeler TR, Poole PS.
				<br/> 
				年份：2000 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						N2-fixation by Rhizobium-legume symbionts is of major ecological and agricultural importance, responsible for producing a substantial fraction of the biosphere's <span class="trait">nitrogen.</span> On the basis of 15N-labelling studies, it had been generally accepted that ammonium is the sole secretion product of N2-fixation by the bacteroid and that the plant is responsible for assimilating it into amino acids. However, this paradigm has been challenged in a recent 15N-labelling study showing that soybean bacteroids only secrete alanine. Hitherto, <span class="trait">nitrogen</span> secretion has only been assessed from in vitro 15N-labelling studies of isolated bacteroids. We show that both ammonium and alanine are secreted by pea bacteroids. The in vitro partitioning between them will depend on whether the system is open or closed, as well as the ammonium concentration and bacteroid density. To overcome these limitations we identified and mutated the gene for alanine dehydrogenase (aldA) and demonstrate that AldA is the primary route for alanine synthesis in isolated bacteroids. Bacteroids of the <span class="gene">aldA</span> mutant fix <span class="trait">nitrogen</span> but only secrete ammonium at a significant rate, resulting in lower total <span class="trait">nitrogen</span> secretion. Peas inoculated with the <span class="gene">aldA</span> mutant are green and healthy, demonstrating that ammonium secretion by bacteroids can provide sufficient <span class="trait">nitrogen</span> for plant growth. However, plants inoculated with the mutant are reduced in biomass compared with those inoculated with the wild type. The labelling and plant growth studies suggest that alanine synthesis and secretion contributes to the <span class="trait">efficiency</span> of N2-fixation and therefore biomass accumulation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of broiler chicken performance when fed diets containing meals of Bollgard II hybrid cotton containing Cry-X gene (Cry1Ac and Cry2ab gene), parental line or commercial cotton.</h2>
			<h3>PMID:15623220  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mandal AB(1), Elangovan AV, Shrivastav AK, Johri AK, Kaur S, Johri TS.
				<br/> 
				年份：2004 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						(1) Total and free gossypol contents were 6.2 and 0.8, 5.4 and 0.5, and 6.1 and 0.7 g/kg in meals processed (solvent extracted) from Bollgard (BG) II, non-BG II or commercial cottonseeds, respectively. (2) Broiler chicks were given one of 7 dietary treatments (iso-nitrogenous, 220 and 195 g crude protein/ kg diet at 0 to 21 and 21 to 42 d, respectively, at a metabolisable energy concentration of 12.15 MJ/kg). The treatments were: D1 (control, soybean meal [SBM] based), D2 and D3 (commercial CSM at 100 g/kg of diet with and without additional iron), D4 and D5 (BG II CSM with and without additional iron), and D6 and D7 (non-BG II parental CSM with or without additional iron). (3) Body weight gain, feed intake, feed conversion <span class="trait">efficiency,</span> nutrient utilisation, certain blood constituents and carcase traits were not significantly affected by dietary treatments. (4) Weights of bursa and thymus were significantly higher in groups given diets containing <span class="gene">BG</span> II or non-BG diets containing added iron. (5) The results suggest that low free gossypol content cottonseed meals, for example, <span class="gene">BG</span> II, non-BG II and commercial solvent-extracted CSM could be included at 100 g/kg in broiler diets, safely replacing soybean meal without additional iron.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthesis and increased production of protein.</h2>
			<h3>PMID:727014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Platt SG, Bassham JA.
				<br/> 
				年份：1978
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosynthesis, the use of light energy in the conversion of CO2 and inorganic nutrients into plant material, is the ultimate source of the food protein necessary to man's existence. Given certain assumptions, the overall maximal theoretical photosynthetic <span class="trait">efficiency</span> of agricultural plants can be calculated. Actual measured maximal growth rates of plants are equivalent to <span class="trait">efficiency</span> levels well below that theoretical maximum. In air, <span class="gene">C4</span> plants can some closer to the theoretical value than <span class="gene">C3</span> plants, perhaps because <span class="gene">C4</span> plants avoid the occurrence of measurable photorespiration and oxygen inhibition of photosynthesis. Alfalfa, a <span class="gene">C3</span> legume, is an extremely productive protein source. Its protein <span class="trait">yield</span> per acre can surpass that of commonly grown <span class="gene">C4</span> crops (corn, sorghum) and <span class="gene">C3</span> seed crops (soybean, wheat, rice). Alfalfa leaf protein is of high nutritional quality and can apparently be used directly in the human diet, eliminating the protein loss involved in animal production. Plant protein productivity can be raised as part of an increase in overall crop <span class="trait">yield.</span> The growth of plants in atmospheres with elevated CO2 levels can result in increased yields. In <span class="gene">C3</span> plants this is due, at least in part, to the suppression of photorespiration and oxygen inhibition of photosynthesis. We have investigated the effect of CO2 concentration on alfalfa photosynthetic metabolism. Our results support the contention that alfalfa productivity can be increased by an environment of elevated CO2. A second approach toward increased plant protein productivity is through regulation of carbon flow during photosynthesis so as to increase portein production relative to that of other plant constituents. In particular, we have investigated whether ammonia (the form in which plants first incorporate <span class="trait">nitrogen)</span> can act to regulate leaf carbon metabolism. Our results indicate that NH4+, in part through stimulation of pyruvate kinase, brings about increased production of amino acids at the expense of sucrose production in alfalfa. The effect may be of considerable importance in the regulation of green leaf protein synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of dietary supplementation of chitosan and galacto-mannan-oligosaccharide on serum parameters and the insulin-like growth factor-I mRNA expression in early-weaned piglets.</h2>
			<h3>PMID:15826777  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tang ZR(1), Yin YL, Nyachoti CM, Huang RL, Li TJ, Yang C, Yang XJ, Gong J, Peng J, Qi DS, Xing JJ, Sun ZH, Fan MZ.
				<br/> 
				年份：2005 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The study was to determine effects of dietary supplementation of chitosan (COS) and galacto-mannan-oligosaccharides <span class="gene">(GMOS)</span> on some serum biochemical indices, serum growth hormone (GH) and insulin-like growth factor-I (IGF-I) levels, and hepatic and long gissimus muscle IGF-I mRNA expression in early-weaned piglets. Twenty six Duroc x Landrace x Yorkshire piglets at the age of 15 days were used. The piglets had access to creep feed during the suckling. Six piglets were sacrificed for sampling at the beginning of the study. The other 20 piglets were individually housed in metabolic cages and randomly allotted to four corn and soybean meal-based diets including the control group, the antibiotic group with 110 mg lincomycin/kg diet, the COS group containing 0.025% COS, and the <span class="gene">GMOS</span> group with 0.20% <span class="gene">GMOS,</span> respectively, in a 2-week feeding experiment. Blood urea <span class="trait">nitrogen</span> (BUN) level was reduced whereas serum total protein concentration was increased (P&lt;0.05) in responses to the COS and <span class="gene">GMOS</span> supplementation. Dietary supplementation of COS and <span class="gene">GMOS</span> also increased (P&lt;0.05) the serum GH and IGF-I levels along with enhanced hepatic and the muscle IGF-I mRNA abundance. Dietary supplementation of oligosaccharides such as COS and <span class="gene">GMOS</span> may improve growth and feed conversion <span class="trait">efficiency</span> by increasing plasma GH and IGF-I levels, in the early-weaned piglets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq analysis of nodule development at five different developmental stages of soybean (Glycine max) inoculated with Bradyrhizobium japonicum strain 113-2.</h2>
			<h3>PMID:28169364</h3>
				

				
			<p>
				作者：Yuan SL(1)(2), Li R(1)(2), Chen HF(1)(2), Zhang CJ(1)(2), Chen LM(1)(2), Hao QN(1)(2), Chen SL(1)(2), Shan ZH(1)(2), Yang ZL(1)(2), Zhang XJ(1)(2), Qiu Z(1)(2), Zhou XA(1)(2).
				<br/> 
				年份：2017 Feb 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nodule development directly affects <span class="trait">nitrogen</span> fixation <span class="trait">efficiency</span> during soybean growth. Although abundant genome-based information related to nodule development has been released and some studies have reported the molecular mechanisms that regulate nodule development, information on the way nodule genes operate in nodule development at different developmental stages of soybean is limited. In this report, notably different nodulation phenotypes in soybean roots inoculated with Bradyrhizobium japonicum strain 113-2 at five developmental stages <span class="trait">(branching</span> stage, flowering stage, fruiting stage, pod stage and harvest stage) were shown, and the expression of nodule genes at these five stages was assessed quantitatively using RNA-Seq. Ten comparisons were made between these developmental periods, and their differentially expressed genes were analysed. Some important genes were identified, primarily encoding symbiotic <span class="trait">nitrogen</span> fixation-related proteins, cysteine proteases, cystatins and cysteine-rich proteins, as well as proteins involving plant-pathogen interactions. There were no significant shifts in the distribution of most GO functional annotation terms and KEGG pathway enrichment terms between these five development stages. A cystatin <span class="gene">Glyma18g12240</span> was firstly identified from our RNA-seq, and was likely to promote nodulation and delay nodule senescence. This study provides molecular material for further investigations into the mechanisms of <span class="trait">nitrogen</span> fixation at different soybean developmental stages.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nonspecific Symbiosis Between Sophora flavescens and Different Rhizobia.</h2>
			<h3>PMID:29173048</h3>
				

				
			<p>
				作者：Liu YH(1), Jiao YS(1), Liu LX(1), Wang D(1), Tian CF(1), Wang ET(1)(2), Wang L(1), Chen WX(1), Wu SY(3), Guo BL(4), Guan ZG(5), Poinsot V(6), Chen WF(1).
				<br/> 
				年份：2018 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We explored the genetic basis of the promiscuous symbiosis of Sophora flavescens with diverse rhizobia. To determine the impact of Nod factors (NFs) on the symbiosis of S. flavescens, nodulation-related gene mutants of representative rhizobial strains were generated. Strains with mutations in common nodulation genes (nodC, nodM, and nodE) failed to nodulate S. flavescens, indicating that the promiscuous nodulation of this plant is strictly dependent on the basic NF structure. Mutations of the NF decoration genes nodH, nodS, nodZ, and <span class="gene">noeI</span> did not affect the nodulation of S. flavescens, but these mutations affected the nitrogen-fixation <span class="trait">efficiency</span> of nodules. Wild-type Bradyrhizobium diazoefficiens USDA110 cannot nodulate S. flavescens, but we obtained 14 Tn5 mutants of B. diazoefficiens that nodulated S. flavescens. This suggested that the mutations had disrupted a negative regulator that prevents nodulation of S. flavescens, leading to nonspecific nodulation. For Ensifer fredii CCBAU 45436 mutants, the minimal NF structure was sufficient for nodulation of soybean and S. flavescens. In summary, the mechanism of promiscuous symbiosis of S. flavescens with rhizobia might be related to its nonspecific recognition of NF structures, and the host specificity of rhizobia may also be controlled by currently unknown nodulation-related genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Dynamic Regulation of microRNAs Involved in Salt Stress Responses in Functional Soybean Nodules by High-Throughput Sequencing</h2>
			<h3>PMID:23358256</h3>
				
					<h3>PMCID:3588011</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Zhanghui </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-14-02717"> 2 </xref>  <xref ref-type="aff" rid="af3-ijms-14-02717"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yanwei </given-names>  </name>  <xref ref-type="aff" rid="af4-ijms-14-02717"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Liang </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cai </surname>  <given-names> Zhaoming </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-14-02717"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Youning </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> Jingbo </given-names>  </name>  <xref ref-type="aff" rid="af5-ijms-14-02717"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xia </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-14-02717"> 1 </xref>  <xref ref-type="corresp" rid="c1-ijms-14-02717"> * </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The previous microarray analysis of young nodules in soybean suggests that nodulation and functions of symbiosis are regulated by complex regulatory networks at multiple levels [ <xref ref-type="bibr" rid="b49-ijms-14-02717"> 49 </xref> ]. Among 770 predicated targets, there were 79 transcription factors (TFs), such as B-box zinc finger, homeodomain protein, <span class="gene">AP2</span> TFs etc. In particular, among 26 putative targets for the top 9 highly expressed miRNAs, 14 targets were TFs, including 1 zinc finger protein and 11 <span class="gene">SBP</span> TFs. In addition, the predicated targets included genes in protein biosynthesis/degradation ( <italic> i.e. </italic> , Ca <sup> 2+ </sup> /calmodulin-dependent protein kinase, F-box proteins, ubiquitin-conjugating enzymes, proteases <italic> etc. </italic> ), suggesting that nodule functions and stress response are regulated at multiple levels. Importantly, we found 2 putative targets annotated as an inorganic pyrophosphatase/nucleosome remodeling factor by a novel miRNA gly_20 and a symovial sarcoma-associated SS18 protein which interacts with SWI/SNF protein in nucleus for miR168, suggesting the chromatin remodeling plays a critical role in functioning of SNF. Strikingly, a hemoglobinase family protein involving hemoglobin catabolism was predicated as a putative target for a novel miRNA, gly_32, suggesting that gly_32 may directly regulate the function of <span class="trait">nitrogen</span> fixation in mature soybean nodules. Furthermore, we found that many putative target genes might play important roles in various biological processes, such as cell cycle (Translin and translin associated protein, CDC2-related kinase), cellular skeleton (tubulin-specific chaperone, tubulin targeted by a miR156 variant), lipid/carbohydrate metabolism and ion/anion/sugar/amino transport. Twenty genes targeted by seven miRNAs are likely involved in auxin synthesis, auxin response and auxin regulated morphorgenesis. Many genes annotated as TFs, phosphotases, and kinases modulate plant responses to abiotic and biotic stresses. Moreover, several targets act in antioxidant defense, such as peroxidase, glutaredoxin, NADH dehydrogenase, <italic> etc. </italic> ). Notably, glutamine amidotransferase class-I, a key enzyme involving glutamate synthesis from glutamine was also identified as the target of miR1511. Expression in high levels of gma-miR1511 in functionally very different tissues suggests that conversion of glutamine to glutamate occurs in all tissues and that miR1511 is essential for regulation of <span class="trait">nitrogen</span> metabolism in soybean. This makes sense because <span class="trait">nitrogen</span> demands are higher during soybean growth and development than in non-legume crops [ <xref ref-type="bibr" rid="b50-ijms-14-02717"> 50 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the majority of the miRNAs highly expressed in nodules were also responsive to salt stress ( <xref ref-type="supplementary-material" rid="ijms-14-02717-s003"> Table S2 </xref> or <xref ref-type="fig" rid="f3-ijms-14-02717"> Figure 3c </xref> ). Among them, gma-miR166a,b showed the highest induction by salt among all the differentially expressed known-miRNAs. <span class="gene">Gma-miR159c,</span> gma-miR159b, gma-miR169c, gma-miR319a,b and gma-miR1517 exhibited a more significant downregulation between stressed- and non-stressed nodules ( <xref ref-type="supplementary-material" rid="ijms-14-02717-s003"> Table S2 </xref> ). These miRNAs and their targets such as MYB TFs and NFY-A have previously been linked to abiotic stress and ABA signaling [ <xref ref-type="bibr" rid="b55-ijms-14-02717"> 55 </xref> , <xref ref-type="bibr" rid="b56-ijms-14-02717"> 56 </xref> ]. Further analysis also revealed the association of several novel miRNAs with nodule response to salt stress. <span class="gene">Gma-miR4416d,</span> gly_1 and gly_15 and gma-miR5559 are respectively the most prominently upregulated and downregulated by salt stress ( <xref ref-type="supplementary-material" rid="ijms-14-02717-s004"> Table S3 </xref> ). Their predicted targets zinc finger TFs, GRAS, serine-threonine protein kinase, and calmodulin-dependent protein kinases have been linked to plant adaptation to abiotic stress in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="b57-ijms-14-02717"> 57 </xref> &#x02013; <xref ref-type="bibr" rid="b59-ijms-14-02717"> 59 </xref> ]. Further characterization of these miRNAs and their targets in modulating nodule response to salinity is needed. Moreover, novel miRNA gly_32 was detected only in salt-stressed nodules ( <xref ref-type="supplementary-material" rid="ijms-14-02717-s005"> Table S4 </xref> ) and one target of gly_32 is a hemoglobinase family member which catalyzes the degradation of hemoglobin, which is essential in functional symbiosis between legumes and rhizobia [ <xref ref-type="bibr" rid="b60-ijms-14-02717"> 60 </xref> ]. In particular, hemoglobin was previously shown to be an adaptive outcome in legumes, and to play a significant role when nodules are exposed to adverse environmental conditions [ <xref ref-type="bibr" rid="b61-ijms-14-02717"> 61 </xref> ]. Therefore, gly_32 may inhibit activity of hemoglobinase in mature soybean nodules under salt stress, thus maintaining both high levels of hemoglobin in stressed nodules and the subsequent functioning symbiosis and <span class="trait">nitrogen</span> fixation activity. This is consistent with the evolutionary role of hemoglobin in adaptation to adverse environments. Further studies are needed to better elucidate causal relationships between gly_32 and hemoglobin catabolism as well as between gly_32 and nodule adaptation to high salinity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Recent Achievement in Gene Cloning and Functional Genomics in Soybean</h2>
			<h3>PMID:24311973</h3>
				
					<h3>PMCID:3842071</h3>
					<h4>期刊:The Scientific World Journal</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0003-4819-0184 </contrib-id>  <name>  <surname> Xia </surname>  <given-names> Zhengjun </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 18 </day>  <month> 9 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flowering is one of the most important ecological and agronomical traits since it is related to the domestication, latitudinal and ecological adaption, and <span class="trait">yield</span> directly. About ten major quantitative trait loci (QTLs) for flowering time have been reported in soybean [ <xref rid="B8" ref-type="bibr"> 8 </xref> &#x02013; <xref rid="B15" ref-type="bibr"> 15 </xref> ]. The interactions between major QTLs have been studied intensively among different environments and geographical locations. The <italic> E </italic> serials ( <italic> <span class="gene">E1</span> </italic> to <italic> E8 </italic> ) are controlling flowering time, duration of the reproductive phase (DRP) [ <xref rid="B16" ref-type="bibr"> 16 </xref> ], and other physiological or agronomical traits, such as <span class="trait">branching</span> [ <xref rid="B17" ref-type="bibr"> 17 </xref> ], <span class="trait">yield</span> [ <xref rid="B18" ref-type="bibr"> 18 </xref> ], and chilling <span class="trait">resistance</span> [ <xref rid="B19" ref-type="bibr"> 19 </xref> , <xref rid="B20" ref-type="bibr"> 20 </xref> ]. Many researchers were involved in the identification of molecular basis for <italic> E </italic> locus in soybean [ <xref rid="B21" ref-type="bibr"> 21 </xref> , <xref rid="B22" ref-type="bibr"> 22 </xref> ]. In particular, cooperative researches from Japan and China have cloned <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> , and <italic> E4 </italic> genes. In 1998, <italic> E4 </italic> gene was identified to encode phytochrome A2 protein, by the candidate gene approach based on the QTL position on the map [ <xref rid="B23" ref-type="bibr"> 23 </xref> ]. In the following year, the <italic> E3 </italic> gene was successfully cloned by positional cloning using residual heterozygous line (RHL) [ <xref rid="B24" ref-type="bibr"> 24 </xref> ]. Both <italic> E3 </italic> and <italic> E4 </italic> are involved in response to the light quality (red to far-red quantum (R&#x02009;:&#x02009;FR) ratios); however, their function pathways are different but overlapping [ <xref rid="B25" ref-type="bibr"> 25 </xref> , <xref rid="B26" ref-type="bibr"> 26 </xref> ]. In addition, <italic> E3 </italic> gene has a dominant effect over <italic> E4 </italic> gene since <italic> E4 </italic> genotype only showed its own phenotype under <italic> E3 </italic> genetic background. The cloning result showed that molecular basis of <italic> E3 </italic> gene is a copy of the phytochrome, <italic> <span class="gene">GmPhyA3</span> </italic> . In soybean, there is a third <italic> <span class="gene">GmphyA</span> </italic> gene, <italic> <span class="gene">GmPhyA1</span> </italic> , whose function needs further characterization [ <xref rid="B23" ref-type="bibr"> 23 </xref> ]. Molecular basis for <italic> E2 </italic> locus was identified with the same strategy as the one used in cloning of <italic> E3 </italic> [ <xref rid="B27" ref-type="bibr"> 27 </xref> ]. <italic> <span class="gene">GmGIa</span> </italic> ( <italic> <span class="gene">Glyma10g36600</span> </italic> ) has been proven to be the genetic factor underlying the <italic> E2 </italic> locus [ <xref rid="B27" ref-type="bibr"> 27 </xref> ]. The <italic> GIGANTEA </italic> ( <italic> GI </italic> ) gene in <italic> Arabidopsis </italic> has been proven to play an important role in GI-CO-FT mediated photoperiodic flowering. However, in soybean, the flowering time phenotype difference between dominant and recessive alleles of <italic> E2 </italic> appeared to be independent of daylength, inferring that this locus is not significantly associated with photoperiodic response.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode ( <italic> Heterodera glycines </italic> Ichinohe) is a major constraint to soybean production worldwide. This nematode disease causes more than US$1 billion in <span class="trait">yield</span> losses annually in the United States [ <xref rid="B40" ref-type="bibr"> 40 </xref> ]. Recently, two important genes, <italic> <span class="gene">Rhg4</span> </italic> and <italic> Rhg1, </italic> have been cloned and functionally characterized. <italic> <span class="gene">Rhg4</span> </italic> (for <span class="trait">resistance</span> to <italic> Heterodera glycines 4 </italic> ) locus is a major quantitative trait locus contributing to <span class="trait">resistance</span> to this pathogen. Positional cloning reveals that the corresponding gene encodes a serine hydroxymethyltransferase, an enzyme (SHMT) that is ubiquitous in nature and structurally conserved across plant kingdoms. The enzyme functions as interconversion of serine and glycine, involved in cellular one-carbon metabolism [ <xref rid="B40" ref-type="bibr"> 40 </xref> ]. Various function methods, such as mutation analysis, gene silencing, and transgenic complementation, all confirmed that this gene confers <span class="trait">resistance.</span> On the other hand, most SCN-resistant soybeans in the Midwest, USA, are bred to contain <italic> <span class="gene">Rhg1</span> </italic> ( <italic> rhg1-b </italic> ). After positional cloning and <italic> Rhg2-b </italic> gene silencing, genes in a 31-kilobase segment at <italic> rhg1-b </italic> encode three types of functional proteins, an amino acid transporter, an <italic> &#x003b1; </italic> -SNAP protein, and a WI12 (wound-inducible domain) protein, each contributes to <span class="trait">resistance</span> [ <xref rid="B41" ref-type="bibr"> 41 </xref> ]. Ten tandem copies are present in an <italic> rhg1-b </italic> haplotype; in comparison, only one copy of the 31-kilobase segment per haploid genome in susceptible varieties is existing. Overexpression of individual genes in roots is not sufficient; only overexpression of these genes together can gain enhanced SCN <span class="trait">resistance.</span> This result showed an interesting new insight into our understanding of disease <span class="trait">resistance</span> that copy number variation increases the expression of a set of dissimilar genes in a repeated multigene segment [ <xref rid="B41" ref-type="bibr"> 41 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improving the Reliability of Optimal In-Feed Amino Acid Ratios Based on Individual Amino Acid <span class="trait">Efficiency</span> Data from N Balance Studies in Growing Chicken</h2>
			<h3>PMID:26479521</h3>
				
					<h3>PMCID:4494439</h3>
					<h4>期刊:Animals : an Open Access Journal from MDPI</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wecke </surname>  <given-names> Christian </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liebert </surname>  <given-names> Frank </given-names>  </name>  <xref rid="c1-animals-03-00558" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Making use of the feed optimization program Fumi for Windows 3.1 (HYBRIMIN <sup> &#x000ae; </sup> Computer+Programme <span class="gene">GmbH</span> &#x00026; Co. KG, Hessisch Oldendorf, Germany), diets based on corn, wheat, fishmeal, field peas and wheat gluten, including small quantities of supplemented methionine, cystine and arginine, were created ( <xref ref-type="table" rid="animals-03-00558-t003"> Table 3 </xref> ). According to optimization, the final ingredient composition of all formulated CD fulfilled the postulated optimal AA ratios (values in parentheses of <xref ref-type="table" rid="animals-03-00558-t004"> Table 4 </xref> , compared to data in <xref ref-type="table" rid="animals-03-00558-t001"> Table 1 </xref> ), except for histidine, leucine and phenylalanine, respectively. Consequently, these AAs remained in relative excess and had to be deleted from the current data evaluation based on dietary AA <span class="trait">efficiency,</span> because their limiting position was doubtful. Due to reduced demands of broiler chickens on dietary protein and AA concentrations with increasing body weight (BW), the total protein and AA contents in the CD for grower periods were diminished by diet dilution with wheat starch ( <xref ref-type="table" rid="animals-03-00558-t003"> Table 3 </xref> ). However, the protein quality between corresponding starter and grower diets was kept constant, according to an equal proportion between all feed protein sources.
					</p>
				
					<p>&nbsp;&nbsp;
						Chemical analyses were conducted according to the German standard methods [ <xref rid="B24-animals-03-00558" ref-type="bibr"> 24 </xref> ]. Feed ingredients and experimental diets were analyzed for crude nutrients, starch and total sugars. Ether extracts of the feed were analyzed following HCl-hydrolysis. <span class="trait">Nitrogen</span> determination utilized the DUMAS-method (LECO <sup> &#x000ae; </sup> LP-2000, LECO Instruments <span class="gene">GmbH,</span> Kirchheim, Germany) and crude protein calculation by a factor of 6.25. Amino acids (except tryptophan) were analyzed by ion-exchange chromatography (LC 3000; Biotronik, Eppendorf-Netheler-Hinz <span class="gene">GmbH,</span> Hamburg, Germany) following acid hydrolysis without and with an oxidation step for quantitative determination of sulfur-containing amino acids. Tryptophan was quantified by reverse phase high performance liquid chromatography and fluorometric detection following alkaline hydrolysis with barium hydroxide during autoclaving (16 h, 120 &#x000b0;C). Additionally, apparent metabolizable energy (ME) content of balanced CD was calculated according to the World&#x02019;s Poultry Science Association (WPSA) [ <xref rid="B25-animals-03-00558" ref-type="bibr"> 25 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq Analysis of Differential Gene Expression Responding to Different Rhizobium Strains in Soybean ( <italic> Glycine max </italic> ) Roots</h2>
			<h3>PMID:27303417</h3>
				
					<h3>PMCID:4885319</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Songli </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/308749/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/349967/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shuilian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Haifeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chanjuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Limiao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qingnan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/301356/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Zhihui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Zhonglu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Dezhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaojuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean can establish nitrogen-fixing symbiosis with different species of rhizobium strains (Hayashi et al., <xref rid="B13" ref-type="bibr"> 2012a </xref> ). The best studied rhizobium-soybean symbiotic models are <italic> B. japonicum </italic> &#x02014;soybean and <italic> S. fredii </italic> &#x02014;soybean (Israel et al., <xref rid="B18" ref-type="bibr"> 1986 </xref> ; Annapurna and Krishnan, <xref rid="B1" ref-type="bibr"> 2003 </xref> ; Sanz-Saez et al., <xref rid="B44" ref-type="bibr"> 2015 </xref> ). The symbiotic phenotype was different between soybean Tian long No.1 inoculated with slow-growing rhizobium strains <italic> B. japonicum </italic> 113-2 and fast-growing rhizobium strains <italic> S. fredii </italic> USDA205 (Figure <xref ref-type="fig" rid="F1"> 1 </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). At 12 dpi, the control (CK, inoculated with media only) had slightly better growth and higher chlorophyll content than the two symbionts (Figures <xref ref-type="fig" rid="F1"> 1A,B </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ), mainly because the symbiotic process needed more energy from the host plant and the <span class="trait">nitrogen</span> fixation function was weak during this period. At 30 and 42 dpi (Figures <xref ref-type="fig" rid="F1"> 1C,D </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ), the control had no additional <span class="trait">nitrogen</span> supply, thus its growth and chlorophyll synthesis was limited by insufficient <span class="trait">nitrogen</span> in cotyledon. Comparison of the two symbionts showed that <italic> S. fredii </italic> USDA205-soybean symbiont had much less nodule numbers per root system (0 vs. 13) at 12 dpi (Figure <xref ref-type="fig" rid="F1"> 1F </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ), although they had similar growth and chlorophyll content (Figures <xref ref-type="fig" rid="F1"> 1A,B </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). In addition, compared with <italic> B. japonicum </italic> 113-2-soybean symbiont, <italic> S. fredii </italic> USDA205-soybean symbiont had (1) obviously more yellow leaves and less chlorophyll content at 30 dpi (Figures <xref ref-type="fig" rid="F1"> 1C,D </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ); (2) worse growth (Figure <xref ref-type="fig" rid="F1"> 1E </xref> ), less chlorophyll content (Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ), fewer nodule per root system (16.64 vs. 51.52, Figure <xref ref-type="fig" rid="F1"> 1G </xref> ) and larger nodules (3-fold) (Figure <xref ref-type="fig" rid="F1"> 1H </xref> , Supplemental Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ) at 42 dpi, and (3) lower expression of <italic> Gm NIN-like </italic> genes <span class="gene">(Glyma02g48080</span> and <span class="gene">Glyma04g00210),</span> which are required for nodulation (Borisov et al., <xref rid="B5" ref-type="bibr"> 2003 </xref> ), in soybean roots at almost all the tested time points (Figures <xref ref-type="fig" rid="F1"> 1I,J </xref> ). These results suggest that the soybean roots of these two symbiotic systems have differential cellular responses.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic loci of soybean, namely Rj (s) or rj (s), have been identified responsive to the nodule formation (Hayashi et al., <xref rid="B13" ref-type="bibr"> 2012a </xref> ), and several of these genes (e.g., Rj2, Rj4, and Rfg1) are found involving in restrict nodulation with specific rhizobial strains (Yang et al., <xref rid="B51" ref-type="bibr"> 2010 </xref> ). To investigate whether <span class="trait">resistance</span> (R) proteins are involved in control of the nodulation phenotypes in soybean inoculated with rhizobium strains 113-2 or USDA205, DEGs encoding R proteins in soybean roots were analyzed in more detail (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). Figure <xref ref-type="fig" rid="F6"> 6A </xref> shows the numbers of DEGs encoding R proteins in each set. A total of 24 such genes were only found in one group (7, 7, and 10 in Groups 1&#x02013;3, respectively), 22 genes were found in two groups (5 in Group 1 and 2; 4 in Group 2 and 3; 13 in Group 1 and 3) and 3 genes <span class="gene">(Glyma05g17470,</span> <span class="gene">Glyma12g01420,</span> and <span class="gene">Glyma19g35270)</span> were found in all three groups. Figure <xref ref-type="fig" rid="F6"> 6B </xref> shows the number of R genes differentially expressed at the five different time points in soybean roots in the three groups. It can be seen that more R genes were found in Group 1 than in Group 2 at each time point, especially at 16 dpi and not all differentially expressed R genes in Group 1 and/or Group 2 are also found in Group 3 (Figures <xref ref-type="fig" rid="F6"> 6A,B </xref> ), indicating that they participate in nodulation but not in restriction of specific rhizobial strains. Besides, some differentially expressed R genes in Group 3 were not found in Group 1 and Group 2, indicating that these R genes may be associated with T3SS of rhizobium and/or defense responses in soybean roots.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>