<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Photoperiod sensitivity</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Photoperiod sensitivity</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:GsSNAP33, a novel Glycine soja SNAP25-type protein gene: Improvement of plant salt and <span class="trait">drought</span> tolerances in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:28841544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nisa ZU(1), Mallano AI(2), Yu Y(3), Chen C(4), Duan X(5), Amanullah S(6), Kousar A(7), Baloch AW(8), Sun X(9), Tabys D(10), Zhu Y(11).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The N-ethylmaleimide-sensitive factor attachment protein receptor (SNARE) superfamily, specifically the SNAP25-type proteins and t-SNAREs, have been proposed to regulate cellular processes and plant <span class="trait">resistance</span> mechanisms. However, little is known about the role of SNAP25-type proteins in combating abiotic stresses, specifically in wild soybean. In the current study, the isolation and functional characterization of the putative synaptosomal-associated SNAP25-type protein gene GsSNAP33 from wild soybean (Glycine soja) were performed. GsSNAP33 has a molecular weight of 33,311 Da and comprises 300 amino acid residues along with Qb-Qc SNARE domains. Multiple sequence alignment revealed the highest similarity of the GsSNAP33 protein to <span class="gene">GmSNAP33</span> (91%), VrSNAP33 (89%), PvSNAP33 (86%) and AtSNAP33 (63%). Phylogenetic studies revealed the abundance of SNAP33 proteins mostly in dicotyledons. Quantitative real-time PCR assays confirmed that GsSNAP33 expression can be induced by salt, alkali, ABA and PEG treatments and that GsSNAP33 is highly expressed in the pods, seeds and roots of Glycine soja. Furthermore, the overexpression of the GsSNAP33 gene in WT Arabidopsis thaliana resulted in increased germination rates, greater root lengths, improved photosynthesis, lower electrolyte leakage, higher biomass production and up-regulated expression levels of various stress-responsive marker genes, including KINI, COR15A, P5Cs, RAB18, <span class="gene">RD29A</span> and COR47 in transgenic lines compared with those in WT lines. Subcellular localization studies revealed that the GsSNAP33-eGFP fusion protein was localized to the plasma membrane, while eGFP was distributed throughout whole cytoplasm of onion epidermal cells. Collectively, our findings suggest that GsSNAP33, a novel plasma membrane protein gene of Glycine soja, might be involved in improving plant responses to salt and <span class="trait">drought</span> stresses in Arabidopsis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmFDL19</span> enhances tolerance to <span class="trait">drought</span> and salt stresses in soybean.</h2>
			<h3>PMID:28640834  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li Y(1)(2), Chen Q(2), Nan H(2), Li X(2)(3), Lu S(2)(4), Zhao X(2)(4), Liu B(2)(4), Guo C(1), Kong F(2)(4), Cao D(2).
				<br/> 
				年份：2017 Jun 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The basic leucine zipper (bZIP) family of transcription factors plays an important role in the growth and developmental process as well as responds to various abiotic stresses, such as <span class="trait">drought</span> and high salinity. Our previous work identified <span class="gene">GmFDL19,</span> a bZIP transcription factor, as a flowering promoter in soybean, and the overexpression of <span class="gene">GmFDL19</span> caused early flowering in transgenic soybean plants. Here, we report that <span class="gene">GmFDL19</span> also enhances tolerance to <span class="trait">drought</span> and salt stress in soybean. <span class="gene">GmFDL19</span> was determined to be a group A member, and its transcription expression was highly induced by abscisic acid (ABA), polyethylene glycol (PEG 6000) and high salt stresses. Overexpression of <span class="gene">GmFDL19</span> in soybean enhanced <span class="trait">drought</span> and salt tolerance at the seedling stage. The relative plant height (RPH) and relative shoot dry weight (RSDW) of transgenic plants were significantly higher than those of the WT after PEG and salt treatments. In addition, the germination rate and plant height of the transgenic soybean were also significantly higher than that of WT plants after various salt treatments. Furthermore, we also found that <span class="gene">GmFDL19</span> could reduce the accumulation of Na+ ion content and up-regulate the expression of several ABA/stress-responsive genes in transgenic soybean. We also found that <span class="gene">GmFDL19</span> overexpression increased the activities of several antioxidative enzyme and chlorophyll content but reduced malondialdehyde content. These results suggested that <span class="gene">GmFDL19</span> is involved in soybean abiotic stress responses and has potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Genome-Wide Association Study for Agronomic Traits in Soybean Using SNP Markers and SNP-Based Haplotype Analysis.</h2>
			<h3>PMID:28152092  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contreras-Soto RI(1), Mora F(2), de Oliveira MA(3), Higashi W(3), Scapim CA(1), Schuster I(4).
				<br/> 
				年份：2017 Feb 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mapping quantitative trait loci through the use of linkage disequilibrium (LD) in populations of unrelated individuals provides a valuable approach for dissecting the genetic basis of complex traits in soybean (Glycine max). The haplotype-based genome-wide association study (GWAS) has now been proposed as a complementary approach to intensify benefits from LD, which enable to assess the genetic determinants of agronomic traits. In this study a GWAS was undertaken to identify genomic regions that control 100-seed weight (SW), plant height (PH) and seed <span class="trait">yield</span> (SY) in a soybean association mapping panel using single nucleotide polymorphism (SNP) markers and haplotype information. The soybean cultivars (N = 169) were field-evaluated across four locations of southern Brazil. The genome-wide haplotype association analysis (941 haplotypes) identified eleven, seventeen and fifty-nine SNP-based haplotypes significantly associated with SY, SW and PH, respectively. Although most marker-trait associations were environment and trait specific, stable haplotype associations were identified for SY and SW across environments (i.e., haplotypes <span class="gene">Gm12_Hap12).</span> The haplotype block 42 on Chr19 <span class="gene">(Gm19_Hap42)</span> was confirmed to be associated with PH in two environments. These findings enable us to refine the breeding strategy for tropical soybean, which confirm that haplotype-based GWAS can provide new insights on the genetic determinants that are not captured by the single-marker approach.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean seed set and size using RNA sequencing analysis.</h2>
			<h3>PMID:28087653</h3>
				

				
			<p>
				作者：Du J(1), Wang S(1), He C(1), Zhou B(2), Ruan YL(3), Shou H(1).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To understand the gene expression networks controlling soybean seed set and size, transcriptome analyses were performed in three early seed developmental stages, using two genotypes with contrasting seed size. The two-dimensional data set provides a comprehensive and systems-level view on dynamic gene expression networks underpinning soybean seed set and subsequent development. Using pairwise comparisons and weighted gene coexpression network analyses, we identified modules of coexpressed genes and hub genes for each module. Of particular importance are the discoveries of specific modules for the large seed size variety and for seed developmental stages. A large number of candidate regulators for seed size, including those involved in hormonal signaling pathways and transcription factors, were transiently and specifically induced in the early developmental stages. The soybean homologs of a brassinosteroid signaling receptor kinase, a brassinosteroid-signaling kinase, were identified as hub genes operating in the seed coat network in the early seed maturation stage. Overexpression of a candidate seed size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased seed size and seed weight. Together, these analyses identified a large number of potential key regulators controlling soybean seed set, seed size, and, consequently, <span class="trait">yield</span> potential, thereby providing new insights into the molecular networks underlying soybean seed development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptomic analysis reveals the flooding tolerant mechanism in flooding tolerant line and abscisic acid treated soybean.</h2>
			<h3>PMID:28012053  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin X(1)(2), Hiraga S(2), Hajika M(2), Nishimura M(3), Komatsu S(4)(5).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is highly sensitive to flooding stress and exhibits markedly reduced plant growth and grain <span class="trait">yield</span> under flooding conditions. To explore the mechanisms underlying initial flooding tolerance in soybean, RNA sequencing-based transcriptomic analysis was performed using a flooding-tolerant line and ABA-treated soybean. A total of 31 genes included 12 genes that exhibited similar temporal patterns were commonly changed in these plant groups in response to flooding and they were mainly involved in RNA regulation and protein metabolism. The mRNA expression of matrix metalloproteinase, glucose-6-phosphate isomerase, ATPase family <span class="gene">AAA</span> domain-containing protein 1, and cytochrome <span class="gene">P450</span> 77A1 was up-regulated in wild-type soybean under flooding conditions; however, no changes were detected in the flooding-tolerant line or ABA-treated soybean. The mRNA expression of cytochrome <span class="gene">P450</span> 77A1 was specifically up-regulated in root tips by flooding stress, but returned to the level found in control plants following treatment with the <span class="gene">P450</span> inhibitor uniconazole. The survival ratio and root fresh weight of plants were markedly improved by 3-h uniconazole treatment under flooding stress. Taken together, these results suggest that cytochrome <span class="gene">P450</span> 77A1 is suppressed by uniconazole treatment and that this inhibition may enhance soybean tolerance to flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing seed size and quality by manipulating BIG SEEDS1 in legume species.</h2>
			<h3>PMID:27791139</h3>
				

				
			<p>
				作者：Ge L(1), Yu J(2), Wang H(1), Luth D(3)(4), Bai G(2)(5), Wang K(3)(4), Chen R(6).
				<br/> 
				年份：2016 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant organs, such as seeds, are primary sources of food for both humans and animals. Seed size is one of the major agronomic traits that have been selected in crop plants during their domestication. Legume seeds are a major source of dietary proteins and oils. Here, we report a conserved role for the BIG SEEDS1 (BS1) gene in the control of seed size and weight in the model legume Medicago truncatula and the grain legume soybean (Glycine max). <span class="gene">BS1</span> encodes a plant-specific transcription regulator and plays a key role in the control of the size of plant organs, including seeds, seed pods, and leaves, through a regulatory module that targets primary cell proliferation. Importantly, down-regulation of <span class="gene">BS1</span> orthologs in soybean by an artificial microRNA significantly increased soybean seed size, weight, and amino acid content. Our results provide a strategy for the increase in <span class="trait">yield</span> and seed quality in legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T set was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less pod wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant seed weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher seed weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Clustering of soybean genotypes via Ward-MLM and ANNs associated with mixed models.</h2>
			<h3>PMID:27525912  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Teodoro PE(1), Torres FE(2), Corrêa AM(2).
				<br/> 
				年份：2016 Aug 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objectives of this study were to use mixed models to confirm the presence of genetic variability in 16 soybean genotypes, to compare clusters generated by artificial neural networks (ANNs) with those created by the Ward modified location model (MLM) technique, and to indicate parental combinations that hold promise for obtaining superior segregating populations of soybean. A field trial was conducted between November 2014 and February 2015 at Universidade Estadual de Mato Grosso do Sul, Aquidauana, MS. The experimental design consisted of four replications of randomized blocks, each containing 16 treatments. We assessed the following agronomic traits: plant height, first pod height, number of branches per plant, number of pods per plant, number of grains per pod, hundred-grain weight, and grain <span class="trait">yield.</span> Mixed models were used to estimate variance components and genetic parameters, and obtain genotypic values for each trait. After verifying the presence of genetic variability for all traits, genotypic values were submitted to both a Ward-MLM procedure and ANNs to estimate genetic divergence among genotypes. The number of groups formed by both methods was the same, but there were differences in group constitutions. <span class="gene">ANN</span> analysis improved soybean genotypes clustering patterns compared to Ward-MLM procedure. Based on these methods, divergent crosses may be made between genotype 97R73 with genotypes AS3797 and SYN9070, whereas convergent crosses may be made between genotypes AS3797 and SYN9070.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development and application of a general plasmid reference material for <span class="gene">GMO</span> screening.</h2>
			<h3>PMID:27497661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu Y(1), Li J(1), Wang Y(1), Li X(1), Li Y(1), Zhu L(1), Li J(1), Wu G(2).
				<br/> 
				年份：2016 Sep - Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The use of analytical controls is essential when performing <span class="gene">GMO</span> detection through screening tests. Additionally, the presence of taxon-specific sequences is analyzed mostly for quality control during <span class="gene">GMO</span> detection. In this study, 11 commonly used genetic elements involving three promoters (P-35S, P-FMV35S and P-NOS), four marker genes (Bar, NPTII, HPT and Pmi), and four terminators (T-NOS, T-35S, T-g7 and T-e9), together with the reference gene fragments from six major crops of maize, soybean, rapeseed, rice, cotton and wheat, were co-integrated into the same single plasmid to construct a general reference plasmid pBI121-Screening. The suitability test of pBI121-Screening plasmid as reference material indicated that the non-target sequence on the pBI121-Screening plasmid did not affect the PCR amplification efficiencies of screening methods and taxon-specific methods. The <span class="trait">sensitivity</span> of screening and taxon-specific assays ranged from 5 to 10 copies of pBI121-Screening plasmid, meeting the <span class="trait">sensitivity</span> requirement of <span class="gene">GMO</span> detection. The construction of pBI121-Screening solves the lack of a general positive control for screening tests, thereby reducing the workload and cost of preparing a plurality of the positive control.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Black soldier fly as dietary protein source for broiler quails: apparent digestibility, excreta microbial load, feed choice, performance, carcass and meat traits.</h2>
			<h3>PMID:27339654  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cullere M(1), Tasoniero G(1), Giaccone V(1), Miotti-Scapin R(1), Claeys E(2), De Smet S(2), Dalle Zotte A(1).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In order to expand with validated scientific data the limited knowledge regarding the potential application of insects as innovative feed ingredients for poultry, the present study tested a partial substitution of soya bean meal and soya bean oil with defatted black soldier fly (Hermetia illucens) larvae meal (H) in the diet for growing broiler quails (Coturnix coturnix japonica) on growth performance, mortality, nutrients apparent digestibility, microbiological composition of excreta, feed choice, carcass and meat traits. With this purpose, a total of 450 10-day-old birds were allocated to 15 cages (30 birds/cage) and received three dietary treatments: a Control diet (C) and two diets (H1 and H2) corresponding to 10% and 15% H inclusion levels, respectively (H substituted 28.4% soya bean oil and 16.1% soya bean meal for H1, and 100% soya bean oil and 24.8% soya bean meal for H2, respectively). At 28 days of age, quails were slaughtered, carcasses were weighed, breast muscles were then excised from 50 quails/treatment, weighed, and ultimate pH (pHu) and L*, a*, b* colour values were measured. Breast muscles were then cooked to assess cooking loss and meat toughness. For the digestibility trial, a total of 15 28-day-old quails were assigned to the three feeding groups. The excreta samples were subjected to chemical and microbiological analysis. The same 15 quails were then simultaneously provided with C and H2 diets for a 10-day feed choice trial. Productive performance, mortality and carcass traits were in line with commercial standards and similar in all experimental groups. With the exception of ether extract digestibility, which was lower in <span class="gene">H1</span> group compared with C and H2 (P=0.0001), apparent digestibility of dry matter, CP, starch and energy did not differ among treatments. Microbial composition of excreta was also comparable among the three groups. Feed choice trial showed that quails did not express a preference toward C or H2 diets. Breast meat weight and <span class="trait">yield</span> did not differ among C, <span class="gene">H1</span> and H2 quails. Differently, the inclusion of H meal reduced meat pHu compared with C. In conclusion, this study demonstrated that H. illucens larvae meal can partially replace conventional soya bean meal and soya bean oil in the diet for growing broiler quails, thus confirming to be a promising insect protein source for the feed industry. Further research to assess the impact of H meal on intestinal morphology as well as on meat quality and sensory profile would be of utmost importance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and functional characterization of the sulfate transporter gene <span class="gene">GmSULTR1;2b</span> in soybean.</h2>
			<h3>PMID:27206527  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ding Y(1), Zhou X(1), Zuo L(1), Wang H(2), Yu D(3).
				<br/> 
				年份：2016 May 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is a major source of oil and protein in the human diet and in animal feed. However, as soybean is deficient in sulfur-containing amino acids, its nutritional value is limited. Increasing sulfate assimilation and utilization efficiency is a valuable approach to augment the concentration of sulfur-containing amino acids in soybean seeds, and sulfate transporters play important roles in both sulfate uptake and translocation within plants. RESULTS: In this study, we isolated and characterized a soybean sulfate transporter gene: <span class="gene">GmSULTR1;2b.</span> The gene was found to be specifically expressed in root tissues and induced by low-sulfur stress. In addition, <span class="gene">GmSULTR1;2b</span> expression in yeast could complement deficiency in the sulfate transporter genes SUL1 and SUL2. Under +S conditions, <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco plants accumulated higher levels of organic matter and exhibited enhanced biomass and seed weight compared to control plants. Under -S conditions, acclimation of <span class="gene">GmSULTR1;2b-overexpressing</span> plants was much better than control plants. <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, but no significant difference was observed under salt stress. Transcriptome analysis revealed 515 genes with at least a 2-fold change in expression in the leaves of tobacco plants overexpressing <span class="gene">GmSULTR1;2b.</span> Of these, 227 gene annotations were classified into 12 functional categories associated with 123 relevant pathways, including biosynthesis and metabolism-related proteins, stress-related proteins, and transporters. CONCLUSIONS: The findings reported here indicate that the increased biomass and seed <span class="trait">yield</span> observed in transgenic tobacco plants could have resulted from greater nutrient uptake and transport capability as well as enhanced development and accumulation of organic matter. Taken together, our results indicate that <span class="gene">GmSULTR1;2b</span> plays an important role in sulfur uptake and could alter the sulfur status of plants. Our study suggests that overexpressing <span class="gene">GmSULTR1;2b</span> may enhance plant <span class="trait">yield</span> under +S conditions, reduce plant production loss under -S conditions, and improve plant tolerance to sulfur deficiency stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soy Protein Supplementation Reduces Clinical Indices in Type 2 Diabetes and Metabolic Syndrome.</h2>
			<h3>PMID:26996569  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang XM(1), Zhang YB(2), Chi MH(3).
				<br/> 
				年份：2016 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						PURPOSE: Clinical trials have studied the use of soy protein for treating type 2 diabetes (T2D) and metabolic syndrome (MS). The purpose of this study was to outline evidence on the effects of soy protein supplementation on clinical indices in T2D and <span class="gene">MS</span> subjects by performing a meta-analysis of randomized controlled trials (RCTs). MATERIALS AND METHODS: We searched PubMed, EMBASE, and Cochrane databases up to March 2015 for RCTs. Pooled estimates and 95% confidence intervals (CIs) were calculated by the fixed-and-random-effects model. A total of eleven studies with eleven clinical variables met the inclusion criteria. RESULTS: The meta-analysis showed that fasting plasma glucose (FPG) [weighted mean difference (WMD), -0.207; 95% CI, -0.374 to -0.040; p=0.015], fasting serum insulin (FSI) (WMD, -0.292; 95% CI, -0.496 to -0.088; p=0.005), homeostasis model of assessment for insulin <span class="trait">resistance</span> index (HOMA-IR) (WMD, -0.346; 95% CI, -0.570 to -0.123; p=0.002), diastolic blood pressure (DBP) (WMD, -0.230; 95% CI, -0.441 to -0.019; p=0.033), low-density lipoprotein cholesterol (LDL-C) (WMD, -0.304; 95% CI, -0.461 to -0.148; p=0.000), total cholesterol (TC) (WMD, -0.386; 95% CI, -0.548 to -0.225; p=0.000), and C-reactive protein (CRP) (WMD, -0.510; 95% CI, -0.722 to -0.299; p=0.000) are significant reduced with soy protein supplementation, compared with a placebo control group, in T2D and <span class="gene">MS</span> patients. Furthermore, soy protein supplementation for longer duration (≥6 mo) significantly reduced FPG, LDL-C, and CRP, while that for a shorter duration (&lt;6 mo) significantly reduced FSI and HOMA-IR. CONCLUSION: Soy protein supplementation could be beneficial for FPG, FSI, HOMA-IR, DBP, LDL-C, TC, and CRP control in plasma.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association study, genomic prediction and marker-assisted selection for seed weight in soybean (Glycine max).</h2>
			<h3>PMID:26518570  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang J(1), Song Q(2), Cregan PB(2), Jiang GL(3).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Twenty-two loci for soybean SW and candidate genes conditioning seed development were identified; and prediction accuracies of <span class="gene">GS</span> and MAS were estimated through cross-validation and validation with unrelated populations. Soybean (Glycine max) is a major crop for plant protein and oil production, and seed weight (SW) is important for <span class="trait">yield</span> and quality in food/vegetable uses of soybean. However, our knowledge of genes controlling SW remains limited. To better understand the molecular mechanism underlying the trait and explore marker-based breeding approaches, we conducted a genome-wide association study in a population of 309 soybean germplasm accessions using 31,045 single nucleotide polymorphisms (SNPs), and estimated the prediction accuracy of genomic selection (GS) and marker-assisted selection (MAS) for SW. Twenty-two loci of minor effect associated with SW were identified, including hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> The mixed model containing these loci explained 83.4% of phenotypic variation. Candidate genes with Arabidopsis orthologs conditioning SW were also proposed. The prediction accuracies of <span class="gene">GS</span> and MAS by cross-validation were 0.75-0.87 and 0.62-0.75, respectively, depending on the number of SNPs used and the size of training population. <span class="gene">GS</span> also outperformed MAS when the validation was performed using unrelated panels across a wide range of maturities, with an average prediction accuracy of 0.74 versus 0.53. This study convincingly demonstrated that soybean SW is controlled by numerous minor-effect loci. It greatly enhances our understanding of the genetic basis of SW in soybean and facilitates the identification of genes controlling the trait. It also suggests that <span class="gene">GS</span> holds promise for accelerating soybean breeding progress. The results are helpful for genetic improvement and genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Arabidopsis KLU homologue <span class="gene">GmCYP78A72</span> regulates seed size in soybean.</h2>
			<h3>PMID:26482479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao B(1)(2), Dai A(2), Wei H(2), Yang S(3), Wang B(1), Jiang N(4), Feng X(5)(6).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is one of the most important crops in the world, and its <span class="trait">yield</span> is largely determined by grain weight and grain size. However, the genes that regulate soybean seed size have not been identified. CYP78A, which is highly conserved within terrestrial plants, regulates organ development. In Arabidopsis, AtCYP78A5/KLU has been shown to determine seed size. In the present study, soybean CYP78A72 <span class="gene">(GmCYP78A72),</span> one of the orthologs of KLU, was over-expressed in both Arabidopsis and soybean to examine its function in plant development. <span class="gene">GmCYP78A72</span> heterologous expression in Arabidopsis resulted in enlarged sepals, petals, seeds and carpel. Over-expression of <span class="gene">GmCYP78A72</span> in soybean resulted in increased pea size, which is an extremely desirable trait for enhancing productivity. Moreover, knock-down of <span class="gene">GmCYP78A72</span> does not reduce grain size. However, silencing of <span class="gene">GmCYP78A57,</span> <span class="gene">GmCYP78A70</span> and <span class="gene">GmCYP78A72</span> genes in triplet reduces the seed size significantly indicating functional redundancy of these three <span class="gene">GmCYP78A</span> genes. In conclusion, we investigated the role of CYP78A in soybean seed regulation, and our strategy can be effectively used to engineer large seed traits in soybean varieties as well as other crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Oil Is More Obesogenic and Diabetogenic than Coconut Oil and Fructose in Mouse: Potential Role for the Liver.</h2>
			<h3>PMID:26200659  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deol P(1), Evans JR(1), Dhahbi J(2), Chellappa K(1), Han DS(1), Spindler S(2), Sladek FM(1).
				<br/> 
				年份：2015 Jul 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The obesity epidemic in the U.S. has led to extensive research into potential contributing dietary factors, especially fat and fructose. Recently, increased consumption of soybean oil, which is rich in polyunsaturated fatty acids (PUFAs), has been proposed to play a causal role in the epidemic. Here, we designed a series of four isocaloric diets (HFD, SO-HFD, F-HFD, F-SO-HFD) to investigate the effects of saturated versus unsaturated fat, as well as fructose, on obesity and diabetes. C57/BL6 male mice fed a diet moderately high in fat from coconut oil and soybean oil (SO-HFD, 40% kcal total fat) showed statistically significant increases in weight gain, adiposity, diabetes, glucose intolerance and insulin <span class="trait">resistance</span> compared to mice on a diet consisting primarily of coconut oil (HFD). They also had fatty livers with hepatocyte ballooning and very large lipid droplets as well as shorter colonic crypt length. While the high fructose diet (F-HFD) did not cause as much obesity or diabetes as SO-HFD, it did cause rectal prolapse and a very fatty liver, but no balloon injury. The coconut oil diet (with or without fructose) increased spleen weight while fructose in the presence of soybean oil increased kidney weight. Metabolomics analysis of the liver showed an increased accumulation of PUFAs and their metabolites as well as γ-tocopherol, but a decrease in cholesterol in SO-HFD. Liver transcriptomics analysis revealed a global dysregulation of cytochrome <span class="gene">P450</span> (Cyp) genes in SO-HFD versus HFD livers, most notably in the Cyp3a and Cyp2c families. Other genes involved in obesity (e.g., Cidec, Cd36), diabetes (Igfbp1), inflammation (Cd63), mitochondrial function (Pdk4) and cancer (H19) were also upregulated by the soybean oil diet. Taken together, our results indicate that in mice a diet high in soybean oil is more detrimental to metabolic health than a diet high in fructose or coconut oil.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dietary soybean isoflavones on growth, innate immune responses, hepatic antioxidant abilities and disease <span class="trait">resistance</span> of juvenile golden pompano Trachinotus ovatus.</h2>
			<h3>PMID:25541076  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou C(1), Lin H(2), Ge X(3), Niu J(4), Wang J(4), Wang Y(4), Chen L(4), Huang Z(4), Yu W(4), Tan X(4).
				<br/> 
				年份：2015 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The present study was conducted to investigate the effects of dietary soybean isoflavones (SI) supplementation on growth performance, innate immune responses, hepatic antioxidant abilities, heat shock protein 70 (HSP70) gene expression and <span class="trait">resistance</span> to the pathogen Vibrio harveyi in Trachinotus ovatus. A basal diet was supplemented with SI at 0, 10, 20, 40, 60, 80 mg kg(-1) feed for 8 weeks. Significantly maximum weight gain (WG) and specific growth rate (SGR) were observed in treatment with 40 mg kg(-1) SI supplement (P &lt; 0.05). Feed conversion ratio (FCR), feeding rate (FR) and survival rate were not significantly different among treatments. Fish fed a diet with 40 mg kg(-1) SI showed significant increase in plasma total protein content, complement 3 content, lysozyme activity as well as respiratory burst activity, but decrease in alanine aminotransferase and aspartate aminotransferase activities (P &lt; 0.05). Increased plasma alkaline phosphatase activity, hepatic total antioxidative capacity, catalase activity and superoxide dismutase activity were also noticed in fish fed SI at 40 or 60 mg kg(-1) (P &lt; 0.05). On the contrary, the lowest hepatic malondialdehyde (MDA) content was observed in fish fed SI at 40 mg kg(-1) (P &lt; 0.05). Compared with the control, the relative level of <span class="gene">HSP70</span> mRNA in fish fed SI at 40-80 mg kg(-1) were significantly increased, respectively (P &lt; 0.05). After challenge with V. harveyi, significant higher post-challenge survival was observed in fish fed diets with 40-80 mg kg(-1) SI supplement than that in control group (P &lt; 0.05). These results indicated that dietary intake containing SI could enhance the immune ability of fish and improve its <span class="trait">resistance</span> to infection by V. harveyi. Especially supplementation with 40 mg kg(-1) SI to the fish for 8 weeks showed remarkable improvement in the growth, non-specific immune responses, hepatic antioxidant abilities and <span class="gene">HSP70</span> gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the Hsf family in soybean and functional identification of <span class="gene">GmHsf-34</span> involvement in <span class="trait">drought</span> and heat stresses.</h2>
			<h3>PMID:25416131  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li PS, Yu TF, He GH, Chen M, Zhou YB, Chai SC(1), Xu ZS, Ma YZ.
				<br/> 
				年份：2014 Nov 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: High temperature affects organism growth and metabolic activity. Heat shock transcription factors (Hsfs) are key regulators in heat shock response in eukaryotes and prokaryotes. Under high temperature conditions, Hsfs activate heat shock proteins (Hsps) by combining with heat stress elements (HSEs) in their promoters, leading to defense of heat stress. Since the first plant Hsf gene was identified in tomato, several plant Hsf family genes have been thoroughly characterized. Although soybean (Glycine max), an important oilseed crops, genome sequences have been available, the Hsf family genes in soybean have not been characterized accurately. RESULT: We analyzed the Hsf genetic structures and protein function domains using the GSDS, Pfam, SMART, PredictNLS, and NetNES online tools. The genome scanning of dicots (soybean and Arabidopsis) and monocots (rice and maize) revealed that the whole-genome replication occurred twice in soybean evolution. The plant Hsfs were classified into 3 classes and 16 subclasses according to protein structure domains. The A8 and B3 subclasses existed only in dicots and the A9 and <span class="gene">C2</span> occurred only in monocots. Thirty eight soybean Hsfs were systematically identified and grouped into 3 classes and 12 subclasses, and located on 15 soybean chromosomes. The promoter regions of the soybean Hsfs contained cis-elements that likely participate in <span class="trait">drought,</span> low temperature, and ABA stress responses. There were large differences among Hsfs based on transcriptional levels under the stress conditions. The transcriptional levels of the A1 and A2 subclass genes were extraordinarily high. In addition, differences in the expression levels occurred for each gene in the different organs and at the different developmental stages. Several genes were chosen to determine their subcellular localizations and functions. The subcellular localization results revealed that <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> were located in the nucleus. Overexpression of the <span class="gene">GmHsf-34</span> gene improved the tolerances to <span class="trait">drought</span> and heat stresses in Arabidopsis plants. CONCLUSIONS: This present investigation of the quantity, structural features, expression characteristics, subcellular localizations, and functional roles provides a scientific basis for further research on soybean Hsf functions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution and association analysis of <span class="gene">GmCYP78A10</span> gene with seed size/weight and pod number in soybean.</h2>
			<h3>PMID:25324172  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Zhang H, Sun G, Zhang W, Qiu L.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed-size/weight traits, controlled by multiple genes in soybean, play an important role in determining seed <span class="trait">yield.</span> However, the molecular mechanisms controlling the seed size and weight in soybean remain unclear. In Arabidopsis, P450/CYP78A gene family has been proved extremely relevant to seed size (such as AtCYP78A5, AtCYP78A6 and AtCYP78A9). We found that a soybean <span class="gene">GmCYP78A10</span> gene underwent artificial selection during soybean breeding. The <span class="gene">GmCYP78A10a</span> allele mainly distributed in wild soybean (Glycine soja), but has been eliminated in the cultivars during early stage of soybean breeding, while the <span class="gene">GmCYP78A10b</span> allele has been accumulated and become the predominant allele in cultivated soybean (G. max). ANOVA analysis showed that the mean seed weight, seed width and seed thickness of soybean varieties with <span class="gene">GmCYP78A10b</span> allele was significantly heavier/bigger than those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01). The allele could explain 7.2 % variation in seed weight. The pod number of the soybeans with <span class="gene">GmCYP78A10b</span> allele significantly decreased compared to those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01, R(2) = 5.8 %), while other agronomic traits including seed weight/plant were not significantly affected by these two alleles. We speculated that during the early stage of soybean breeding, breeders selected big seed carrying <span class="gene">GmCYP78A10b</span> allele, but lowered pod number simultaneously. Overall, the selection did not cause the significantly change in soybean seed <span class="trait">yield.</span> Our results suggests that the soybean <span class="gene">GmCYP78A10</span> gene may have a similar function to those genes belonging to P450/CYP78A subfamily in Arabidopsis and provides new information for the genetic control of seed size in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A rapid test for soy aeroallergens exposure assessment.</h2>
			<h3>PMID:24533134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Alvarez-Simon D(1), Cruz MJ(2), Untoria MD(3), Muñoz X(4), Villalbí JR(5), Morell F(2), Gómez-Ollés S(2).
				<br/> 
				年份：2014 Feb 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Determining soy aeroallergens levels is extremely important in the assessment of health risks due to these airborne substances. Currently, soy aeroallergens exposure in the environment is monitored using enzyme immunoassays (EIA) which must be evaluated in a specialized laboratory by skilled personnel. OBJECTIVE: To describe the development and performance of a rapid immunochromatography assay for the detection of soy aeroallergens in environmental samples. METHODS: A test strip using gold labeled anti-soy hull low molecular weight extract (SHLMWE) antibody for the rapid detection of soy aeroallergens in environmental samples was developed. One hundred nineteen airborne samples were analysed in parallel by the strip assay and the anti-SHLMWE sandwich EIA. The assay results were visually analysed by three independent observers who ranked samples as: -, + or ++. Strips were also scanned and analysed by densitometry. RESULTS: The rapid test detected a range of concentrations from 6.25 to 25 ng/mL. Agreement in strip assay interpretations between evaluators was substantial (Kappa = 0.63; <span class="gene">CI</span> 0.544-0.715). Visual interpretation also gave a good concordance with EIA results, with <span class="trait">sensitivity</span> ranging from 77.3 to 100 and specificity from 65 to 83.5 depending on the observer. Furthermore, a strong correlation was observed between densitometry results of strip assay and EIA determinations. CONCLUSIONS: The strip assay developed is rapid, simple, and sensitive and does not require expensive equipment or specific skills. It has considerable potential in the environmental monitoring field for screening soy aeroallergens levels in port cities where allergen measurements are not currently performed. Due to its simplicity, the test will improve the management of soy allergic patients by controlling environmental allergen exposure without the need for apparatus or skilled personnel.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional properties and expression quantitative trait loci for phosphate transporter <span class="gene">GmPT1</span> in soybean.</h2>
			<h3>PMID:23889314  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song H(1), Yin Z, Chao M, Ning L, Zhang D, Yu D.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphate (Pi) remobilization within a plant is critical for plant survival under Pi-limiting conditions. In this paper, a soybean Pi transporter gene, <span class="gene">GmPT1,</span> was characterized. A marked induction of <span class="gene">GmPT1</span> transcript was observed in young leaves, mature leaves and lateral roots during long-term Pi starvation. Transgenic tobacco plants containing the <span class="gene">GmPT1</span> gene were obtained using an Agrobacterium-mediated transformation system. Compared with wild-type plants, transgenic plants showed significant increases in phosphorus-use efficiency (PUE), photosystem II (PSII) function, total dry weight and seed weight under Pi-deficient conditions. <span class="gene">GmPT1</span> expression levels and PUE were determined in a soybean recombinant inbred line population during a pot experiment that was conducted to measure chlorophyll fluorescence parameters, photosynthetic rate (PN ) and seed <span class="trait">yield.</span> Correlation analysis revealed that <span class="gene">GmPT1</span> expression levels had significantly positive correlations with seed <span class="trait">yield,</span> PUE, PN and the quantum <span class="trait">yield</span> of PSII primary photochemistry (ΦPSII ). Expression quantitative trait loci (eQTL) mapping for <span class="gene">GmPT1</span> revealed two eQTLs, one of which coincided with both the physical location of <span class="gene">GmPT1</span> and a QTL associated with seed <span class="trait">yield.</span> These results suggest that <span class="gene">GmPT1</span> plays a role in Pi remobilization, and it may be possible to improve soybean seed yields under Pi-limiting conditions by modulating <span class="gene">GmPT1</span> expression levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dopamine on root growth and enzyme activity in soybean seedlings.</h2>
			<h3>PMID:23838960  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guidotti BB(1), Gomes BR, Siqueira-Soares Rde C, Soares AR, Ferrarese-Filho O.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, we investigated the effects of dopamine, an allelochemical exuded from the velvetbean (Mucuna pruriens L DC. var utilis), on the growth and cell viability of soybean (Glycine max L. Merrill) roots. We analyzed the effects of dopamine on superoxide dismutase, phenylalanine ammonia-lyase and cell wall-bound peroxidase activities as well as its effects on lignin contents in the roots. Three-day-old seedlings were cultivated in half-strength Hoagland nutrient solution (pH 6.0), without or with 0.25 to 1.0 mM dopamine, in a growth chamber (25°C, 12L:12D <span class="trait">photoperiod,</span> irradiance of 280 μmol m(-2) s(-1)) for 24 h. In general, the length, fresh weight and dry weight of roots, cell viability, <span class="gene">PAL</span> and POD activities decreased, while SOD activities increased after dopamine treatment. The content of lignin was not altered. The data demonstrate the susceptibility of soybean to dopamine and reinforce the role of this catecholamine as a strong allelochemical. The results also suggest that dopamine-induced inhibition in soybean roots is not related to the production of lignin, but may be related to damage caused by reactive oxygen species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean seed oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased seed <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] seed oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as seed <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between seed oil concentration and important agronomic and seed quality traits, including seed <span class="trait">yield,</span> 100-seed weight, protein concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with seed oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for protein concentration, four for seed <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with seed protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with seed <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with seed oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for protein concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as seed <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher seed <span class="trait">yield</span> and oil concentration or higher seed protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In vitro analysis of the monolignol coupling mechanism using dehydrogenative polymerization in the presence of peroxidases and controlled feeding ratios of coniferyl and sinapyl alcohol.</h2>
			<h3>PMID:22884779  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Moon SJ(1), Kwon M, Choi D, Won K, Kim YH, Choi IG, Choi JW.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, dehydrogenative polymers (DHP) were synthesized in vitro through dehydrogenative polymerization using different ratios of coniferyl alcohol (CA) and sinapyl alcohol (SA) (10:0, 8:2, 6:4, 2:8, 0:10), in order to investigate the monolignol coupling mechanism in the presence of horseradish peroxidase (HRP), Coprinus cinereus peroxidase (CiP) or soybean peroxidase (SBP) with H(2)O(2), respectively. The turnover capacities of HRP, CiP and <span class="gene">SBP</span> were also measured for coniferyl alcohol (CA) and sinapyl alcohol (SA), and CiP and <span class="gene">SBP</span> were found to have the highest turnover capacity for CA and SA, respectively. The yields of HRP-catalyzed DHP (DHP-H) and CiP-catalyzed DHP (DHP-C) were estimated between ca. 7% and 72% based on the original weights of CA/SA in these synthetic conditions. However, a much lower <span class="trait">yield</span> of SBP-catalyzed DHP (DHP-S) was produced compared to that of DHP-H and DHP-C. In general, the DHP yields gradually increased as the ratio of CA/SA increased. The average molecular weight of DHP-H also increased with increasing CA/SA ratios, while those of DHP-C and DHP-S were not influenced by the ratios of monolignols. The frequency of β-O-4 linkages in the DHPs decreased with increasing CA/SA ratios, indicating that the formation of β-O-4 linkages during DHP synthesis was influenced by peroxidase type.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-environment mapping and meta-analysis of 100-seed weight in soybean.</h2>
			<h3>PMID:22740134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun YN(1), Pan JB, Shi XL, Du XY, Wu Q, Qi ZM, Jiang HW, Xin DW, Liu CY, Hu GH, Chen QS.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						100-Seed weight (100-SW) of soybean is an important but complicated quantitative trait to <span class="trait">yield.</span> This study was focus on the quantitative trait loci (QTLs) of soybean 100-SW from 2006 to 2010, using recombination inbred lines population that was derived from a cross between Charleston and Dongnong 594. A total of 23 QTLs for 100-SW were detected in the linkage group C2, D1a, F, G and O. Nine QTLs were identified by composite interval mapping including one QTL with the minimum confidence interval (CI) of 1.3 cM, while 14 QTLs by multiple interval mapping. Furthermore, 94 reported QTLs of 100-SW were integrated with our QTL mapping results using BioMercator. As a result, 15 consensus QTLs and their corresponding markers were identified. The minimum <span class="gene">CI</span> was reduced to 1.52 cM by the combination of meta-analysis. These findings may merit fine-mapping of these QTL in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detecting un-authorized genetically modified organisms <span class="gene">(GMOs)</span> and derived materials.</h2>
			<h3>PMID:22333321  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Holst-Jensen A(1), Bertheau Y, de Loose M, Grohmann L, Hamels S, Hougs L, Morisset D, Pecoraro S, Pla M, Van den Bulcke M, Wulff D.
				<br/> 
				年份：2012 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genetically modified plants, in the following referred to as genetically modified organisms or <span class="gene">GMOs,</span> have been commercially grown for almost two decades. In 2010 approximately 10% of the total global crop acreage was planted with <span class="gene">GMOs</span> (James, 2011). More than 30 countries have been growing commercial <span class="gene">GMOs,</span> and many more have performed field trials. Although the majority of commercial <span class="gene">GMOs</span> both in terms of acreage and specific events belong to the four species: soybean, maize, cotton and rapeseed, there are another 20+ species where <span class="gene">GMOs</span> are commercialized or in the pipeline for commercialization. The number of <span class="gene">GMOs</span> cultivated in field trials or for commercial production has constantly increased during this time period. So have the number of species, the number of countries involved, the diversity of novel (added) genetic elements and the global trade. All of these factors contribute to the increasing complexity of detecting and correctly identifying <span class="gene">GMO</span> derived material. Many jurisdictions, including the European Union (EU), legally distinguish between authorized (and therefore legal) and un-authorized (and therefore illegal) <span class="gene">GMOs.</span> Information about the developments, field trials, authorizations, cultivation, trade and observations made in the official <span class="gene">GMO</span> control laboratories in different countries around the world is often limited, despite several attempts such as the OECD BioTrack for voluntary dissemination of data. This lack of information inevitably makes it challenging to detect and identify <span class="gene">GMOs,</span> especially the un-authorized <span class="gene">GMOs.</span> The present paper reviews the state of the art technologies and approaches in light of coverage, practicability, <span class="trait">sensitivity</span> and limitations. Emphasis is put on exemplifying practical detection of un-authorized <span class="gene">GMOs.</span> Although this paper has a European (EU) bias when examples are given, the contents have global relevance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript profiling reveals expression differences in wild-type and glabrous soybean lines.</h2>
			<h3>PMID:22029708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hunt M(1), Kaur N, Stromvik M, Vodkin L.
				<br/> 
				年份：2011 Oct 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trichome hairs affect diverse agronomic characters such as seed weight and <span class="trait">yield,</span> prevent insect damage and reduce loss of water but their molecular control has not been extensively studied in soybean. Several detailed models for trichome development have been proposed for Arabidopsis thaliana, but their applicability to important crops such as cotton and soybean is not fully known. RESULTS: Two high throughput transcript sequencing methods, Digital Gene Expression (DGE) Tag Profiling and RNA-Seq, were used to compare the transcriptional profiles in wild-type (cv. Clark standard, CS) and a mutant (cv. Clark glabrous, i.e., trichomeless or hairless, CG) soybean isoline that carries the dominant <span class="gene">P1</span> allele. DGE data and RNA-Seq data were mapped to the cDNAs (Glyma models) predicted from the reference soybean genome, Williams 82. Extending the model length by 250 bp at both ends resulted in significantly more matches of authentic DGE tags indicating that many of the predicted gene models are prematurely truncated at the 5' and 3' UTRs. The genome-wide comparative study of the transcript profiles of the wild-type versus mutant line revealed a number of differentially expressed genes. One highly-expressed gene, <span class="gene">Glyma04g35130,</span> in wild-type soybean was of interest as it has high homology to the cotton gene GhRDL1 gene that has been identified as being involved in cotton fiber initiation and is a member of the BURP protein family. Sequence comparison of <span class="gene">Glyma04g35130</span> among Williams 82 with our sequences derived from CS and CG isolines revealed various SNPs and indels including addition of one nucleotide C in the CG and insertion of ~60 bp in the third exon of CS that causes a frameshift mutation and premature truncation of peptides in both lines as compared to Williams 82. CONCLUSION: Although not a candidate for the <span class="gene">P1</span> locus, a BURP family member <span class="gene">(Glyma04g35130)</span> from soybean has been shown to be abundantly expressed in the CS line and very weakly expressed in the glabrous CG line. RNA-Seq and DGE data are compared and provide experimental data on the expression of predicted soybean gene models as well as an overview of the genes expressed in young shoot tips of two closely related isolines.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Refuge or reservoir? The potential impacts of the biofuel crop Miscanthus x giganteus on a major pest of maize.</h2>
			<h3>PMID:20016814  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Spencer JL(1), Raghu S.
				<br/> 
				年份：2009 Dec 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Interest in the cultivation of biomass crops like the <span class="gene">C4</span> grass Miscanthus x giganteus (Miscanthus) is increasing as global demand for biofuel grows. In the US, Miscanthus is promoted as a crop well-suited to the Corn Belt where it could be cultivated on marginal land interposed with maize and soybean. Interactions (direct and indirect) of Miscanthus, maize, and the major Corn Belt pest of maize, the western corn rootworm, (Diabrotica virgifera virgifera LeConte, WCR) are unknown. Adding a perennial grass/biomass crop to this system is concerning since WCR is adapted to the continuous availability of its grass host, maize (Zea mays). METHODOLOGY/PRINCIPAL FINDINGS: In a greenhouse and field study, we investigated WCR development and oviposition on Miscanthus. The suitability of Miscanthus for WCR development varied across different WCR populations. Data trends indicate that WCR populations that express behavioural <span class="trait">resistance</span> to crop rotation performed as well on Miscanthus as on maize. Over the entire study, total adult WCR emergence from Miscanthus (212 WCR) was 29.6% of that from maize (717 WCR). Adult dry weight was 75-80% that of WCR from maize; female emergence patterns on Miscanthus were similar to females developing on maize. There was no difference in the mean no. of WCR eggs laid at the base of Miscanthus and maize in the field. CONCLUSIONS/SIGNIFICANCE: Field oviposition and significant WCR emergence from Miscanthus raises many questions about the nature of likely interactions between Miscanthus, maize and WCR and the potential for Miscanthus to act as a refuge or reservoir for Corn Belt WCR. Responsible consideration of the benefits and risks associated with Corn Belt Miscanthus are critical to protecting an agroecosystem that we depend on for food, feed, and increasingly, fuel. Implications for European agroecosystems in which Miscanthus is being proposed are also discussed in light of the WCR's recent invasion into Europe.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of seed <span class="trait">yield</span> and agronomic characters by <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> and growth habit genes in soybean.</h2>
			<h3>PMID:20012856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cober ER(1), Morrison MJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes are adapted to narrow bands of latitude due to <span class="trait">photoperiod</span> <span class="trait">sensitivity.</span> There are several photoperiod-sensitive loci (E1, E2, E3, E4, E5, E6, E7, E8). Determinate and indeterminate growth habits are controlled by a single locus. The objective of our research was to examine the effects of <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> and growth habit alleles on seed <span class="trait">yield</span> and other agronomic characters using isogenic lines. Twenty 'Harosoy' isolines with 11 photoperiod-sensitive genotypes many with both indeterminate and determinate growth habits were grown in the field at Ottawa, ON, from 2003 to 2007. Maturity ranged from 97 to 127 days, and seed <span class="trait">yield</span> increased linearly with maturity until about 112 days when it plateaued. Determinate lines were always shorter than indeterminate lines of equivalent maturity. Seed <span class="trait">yield</span> was associated with plant height, maturity, seed sugar concentration, seed weight and <span class="trait">lodging.</span> Effects of alleles at individual loci, and additive and epistatic effects across multiple loci were examined. At a single locus, photoperiod-insensitive alleles produced isolines that matured 8-11 days earlier, yielded less, and had shorter plants with reduced <span class="trait">lodging.</span> In multiple loci analyses, additive effects explained most of the variation in agronomic characters since additive models with E1, E3, E4, E7 and <span class="gene">Dt1</span> loci included compared well to additive plus epistatic models and genotype-based models. Variation in <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> and growth habit alleles results in a range of maturity, with pleiotropic effects on seed <span class="trait">yield</span> and agronomic characteristics, and play an important role in providing adaptation across latitudes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of putative reference genes for gene expression normalization in soybean by quantitative real-time RT-PCR.</h2>
			<h3>PMID:19785741  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hu R(1), Fan C, Li H, Zhang Q, Fu YF.
				<br/> 
				年份：2009 Sep 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Real-time quantitative reverse transcription PCR (RT-qPCR) data needs to be normalized for its proper interpretation. Housekeeping genes are routinely employed for this purpose, but their expression level cannot be assumed to remain constant under all possible experimental conditions. Thus, a systematic validation of reference genes is required to ensure proper normalization. For soybean, only a small number of validated reference genes are available to date. RESULTS: A systematic comparison of 14 potential reference genes for soybean is presented. These included seven commonly used (ACT2, ACT11, TUB4, TUA5, CYP, UBQ10, EF1b) and seven new candidates (SKIP16, MTP, PEPKR1, HDC, TIP41, UKN1, UKN2). Expression stability was examined by RT-qPCR across 116 biological samples, representing tissues at various developmental stages, varied photoperiodic treatments, and a range of soybean cultivars. Expression of all 14 genes was variable to some extent, but that of SKIP16, UKN1 and UKN2 was overall the most stable. A combination of ACT11, UKN1 and UKN2 would be appropriate as a reference panel for normalizing gene expression data among different tissues, whereas the combination SKIP16, UKN1 and MTP was most suitable for developmental stages. ACT11, TUA5 and TIP41 were the most stably expressed when the <span class="trait">photoperiod</span> was altered, and TIP41, UKN1 and UKN2 when the light quality was changed. For six different cultivars in long day (LD) and short day (SD), their expression stability did not vary significantly with ACT11, UKN2 and TUB4 being the most stable genes. The relative gene expression level of <span class="gene">GmFTL3,</span> an ortholog of Arabidopsis FT (FLOWERING LOCUS T) was detected to validate the reference genes selected in this study. CONCLUSION: None of the candidate reference genes was uniformly expressed across all experimental conditions, and the most suitable reference genes are conditional-, tissue-specific-, developmental-, and cultivar-dependent. Most of the new reference genes performed better than the conventional housekeeping genes. These results should guide the selection of reference genes for gene expression studies in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sinorhizobium fredii HH103 cgs mutants are unable to nodulate determinate- and indeterminate nodule-forming legumes and overproduce an altered EPS.</h2>
			<h3>PMID:19348575  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Crespo-Rivas JC(1), Margaret I, Hidalgo A, Buendía-Clavería AM, Ollero FJ, López-Baena FJ, del Socorro Murdoch P, Rodríguez-Carvajal MA, Soria-Díaz ME, Reguera M, Lloret J, Sumpton DP, Mosely JA, Thomas-Oates JE, van Brussel AA, Gil-Serrano A, Vinardell JM, Ruiz-Sainz JE.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sinorhizobium fredii HH103 produces cyclic beta glucans (CG) composed of 18 to 24 glucose residues without or with 1-phosphoglycerol as the only substituent. The S. fredii HH103-Rifr cgs gene (formerly known as ndvB) was sequenced and mutated with the lacZ-gentamicin <span class="trait">resistance</span> cassette. Mutant SVQ562 did not produce CG, was immobile, and grew more slowly in the hypoosmotic GYM medium, but its survival in distilled water was equal to that of HH103-Rifr. Lipopolysaccharides and K-antigen polysaccharides produced by SVQ562 were not apparently altered. SVQ562 overproduced exopolysaccharides (EPS) and its <span class="gene">exoA</span> gene was transcribed at higher levels than in HH103-Rifr. In GYM medium, the EPS produced by SVQ562 was of higher molecular weight and carried higher levels of substituents than that produced by HH103-Rifr. The expression of the SVQ562 cgsColon, two colonslacZ fusion was influenced by the pH and the osmolarity of the growth medium. The S. fredii cgs mutants SVQ561 (carrying cgs::Omega) and SVQ562 only formed pseudonodules on Glycine max (determinate nodules) and on Glycyrrhiza uralensis (indeterminate nodules). Although nodulation factors were detected in SVQ561 cultures, none of the cgs mutants induced any macroscopic response in Vigna unguiculata roots. Thus, the nodulation process induced by S. fredii cgs mutants is aborted at earlier stages in V. unguiculata than in Glycine max.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of broiler performance when fed diets containing event DP-3O5423-1, nontransgenic near-isoline control, or commercial reference soybean meal, hulls, and oil.</h2>
			<h3>PMID:19038811  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：McNaughton J(1), Roberts M, Smith B, Rice D, Hinds M, Sanders C, Layton R, Lamb I, Delaney B.
				<br/> 
				年份：2008 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DP-3Ø5423-1 (305423) is a genetically modified soybean that was produced by biolistic insertion of the gm-fad2-1 gene fragment and gm-hra genes into the germline of soybean seeds. Expression of gm-fad2-1 results in greater concentrations of oleic acid (18:1) by suppressing expression of the endogenous <span class="gene">FAD2-1</span> gene, which encodes an n-6 fatty acid desaturase enzyme that catalyzes desaturation of 18:1 to linoleic acid (18:2). The <span class="gene">GM-HRA</span> protein expressed by the gm-hra gene is a modified version of the soybean acetolactate synthase enzyme that is used as a selectable marker during transformation. A 42-d feeding trial was conducted with broiler chickens to compare the nutritional performance of 305423 soybeans with nontransgenic soybeans. Diets were prepared using processed fractions (meal, hulls, and oil) from 305423 soybean plants. For comparison, additional diets were produced with soybean fractions obtained from a nontransgenic near-isoline (control) and nontransgenic commercial Pioneer brand varieties (93B86, 93B15, and 93M40). Diets were fed to Ross x Cobb broilers (n = 120/group, 50% male and 50% female) in 3 phases. Starter, grower, and finisher diets contained 26.5, 23, and 21.5% soybean meal, respectively. Soybean hulls and oil were added at 1.0 and 0.5%, respectively, across all diets in each phase. No statistically significant differences were observed in growth performance (BW, mortality, feed efficiency), organ <span class="trait">yield</span> (liver and kidney), or carcass <span class="trait">yield</span> (breast, thigh, leg, wing, and abdominal fat) variables between broilers consuming diets prepared with isolated fractions from 305423 or near-isoline control soybean. Additionally, all performance and carcass variables from control and 305423 soybean treatment groups fell within tolerance intervals constructed for each response variable using data from broilers fed diets prepared with reference soybean fractions. Based on the results from this study, it was concluded that 305423 soybeans were nutritionally equivalent to non-transgenic control soybeans with a comparable genetic background.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cold-induced modulation and functional analyses of the DRE-binding transcription factor gene, <span class="gene">GmDREB3,</span> in soybean (Glycine max L.).</h2>
			<h3>PMID:18988621  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen M(1), Xu Z, Xia L, Li L, Cheng X, Dong J, Wang Q, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DREB (dehydration-responsive element-binding protein) transcription factors have important roles in the stress-related regulation network in plants. A DREB orthologue, <span class="gene">GmDREB3,</span> belonging to the A-5 subgroup of the DREB subfamily, was isolated from soybean using the RACE (rapid amplification of cDNA ends) method. Northern blot analysis showed that expression of <span class="gene">GmDREB3</span> in soybean seedlings was induced following cold stress treatment for 0.5 h and was not detected after 3 h. However, it was not induced by <span class="trait">drought</span> and high salt stresses or by abscisic acid (ABA) treatment. This response was similar to those of members in the A-1 subgroup and different from those of other members in the A-5 subgroup, suggesting that the <span class="gene">GmDREB3</span> gene was involved in an ABA-independent cold stress-responsive signal pathway. Furthermore, analysis of the <span class="gene">GmDREB3</span> promoter elucidated its cold-induced modulation. A promoter fragment containing bases -1058 to -664 was involved in response to cold stress, and its effect was detected for 1 h after treatment, but a transcriptional repressor appeared to impair this response by binding to a cis-element in the region -1403 to -1058 at 24 h after the beginning of cold stress. Moreover, the <span class="gene">GmDREB3</span> protein could specifically bind to the DRE element in vitro, and activated expression of downstream reporter genes in yeast cells. In addition, overexpression of <span class="gene">GmDREB3</span> enhanced tolerance to cold, <span class="trait">drought,</span> and high salt stresses in transgenic Arabidopsis. Physiological analyses indicated that the fresh weight and osmolality of <span class="gene">GmDREB3</span> transgenic Arabidopsis under cold stress were higher than those of wild-type controls. <span class="gene">GmDREB3</span> transgenic tobacco accumulated higher levels of free proline under <span class="trait">drought</span> stress and retained higher leaf chlorophyll levels under high salt stress than wild-type tobacco. In addition, constitutive expression of <span class="gene">GmDREB3</span> in transgenic Arabidopsis caused growth retardation, whereas its expression under control of the stress-inducible Rd29A promoter minimized negative effects on plant growth under normal growth conditions, indicating that a combination of the Rd29A promoter and <span class="gene">GmDREB3</span> might be useful for improving tolerance to environmental stresses in crop plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glyphosate-resistant and -susceptible soybean (Glycine max) and canola (Brassica napus) dose response and metabolism relationships with glyphosate.</h2>
			<h3>PMID:17417871  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nandula VK(1), Reddy KN, Rimando AM, Duke SO, Poston DH.
				<br/> 
				年份：2007 May 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments were conducted to determine (1) dose response of glyphosate-resistant (GR) and -susceptible (non-GR) soybean [Glycine max (L.) Merr.] and canola (Brassica napus L.) to glyphosate, (2) if differential metabolism of glyphosate to aminomethyl phosphonic acid (AMPA) is the underlying mechanism for differential <span class="trait">resistance</span> to glyphosate among <span class="gene">GR</span> soybean varieties, and (3) the extent of metabolism of glyphosate to AMPA in <span class="gene">GR</span> canola and to correlate metabolism to injury from AMPA. GR50 (glyphosate dose required to cause a 50% reduction in plant dry weight) values for <span class="gene">GR</span> (Asgrow 4603RR) and non-GR (HBKC 5025) soybean were 22.8 kg ae ha-1 and 0.47 kg ha-1, respectively, with <span class="gene">GR</span> soybean exhibiting a 49-fold level of <span class="trait">resistance</span> to glyphosate as compared to non-GR soybean. Differential reduction in chlorophyll by glyphosate was observed between <span class="gene">GR</span> soybean varieties, but there were no differences in shoot fresh weight reduction. No significant differences were found between <span class="gene">GR</span> varieties in metabolism of glyphosate to AMPA, and in shikimate levels. These results indicate that <span class="gene">GR</span> soybean varieties were able to outgrow the initial injury from glyphosate, which was previously caused at least in part by AMPA. GR50 values for <span class="gene">GR</span> (Hyola 514RR) and non-GR (Hyola 440) canola were 14.1 and 0.30 kg ha-1, respectively, with <span class="gene">GR</span> canola exhibiting a 47-fold level of <span class="trait">resistance</span> to glyphosate when compared to non-GR canola. Glyphosate did not cause reduction in chlorophyll content and shoot fresh weight in <span class="gene">GR</span> canola, unlike <span class="gene">GR</span> soybean. Less glyphosate (per unit leaf weight) was recovered in glyphosate-treated <span class="gene">GR</span> canola as compared to glyphosate-treated <span class="gene">GR</span> soybean. External application of AMPA caused similar injury in both <span class="gene">GR</span> and non-GR canola. The presence of a bacterial glyphosate oxidoreductase gene in <span class="gene">GR</span> canola contributes to breakdown of glyphosate to AMPA. However, the AMPA from glyphosate breakdown could have been metabolized to nonphytotoxic metabolites before causing injury to <span class="gene">GR</span> canola. Injury in <span class="gene">GR</span> and non-GR canola from exogenous application of AMPA was similar.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Influence of decomposition on chemical properties of plant- and manure-derived dissolved organic matter and sorption to goethite.</h2>
			<h3>PMID:17215221  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hunt JF(1), Ohno T, He Z, Honeycutt CW, Dail DB.
				<br/> 
				年份：2007 Jan 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sorption of dissolved organic matter (DOM) plays an important role in maintaining the <span class="trait">fertility</span> and quality of soils in agricultural ecosystems. Few studies have examined the effects of decomposition on DOM sorption and chemical characteristics. This study investigated the sorption to goethite (alpha-FeOOH) of fresh and decomposed hydrophilic (HPL) and hydrophobic (HPB) DOM fractions extracted from the shoots and roots of crimson clover (Trifolium incarnatum L.), corn (Zea mays L.), soybean [Glycine max (L.) Merr.], hairy vetch (Vicia villosa L.), and dairy and poultry manures. Sorption was positively related to apparent molecular weight (MWAP), aromaticity as measured by absorptivity at 280 nm, and phenolic acid content. A 10-d laboratory microbial decomposition of the source organic matter generally increased the sorption of the extracted DOM onto goethite. The decomposition effect on sorption was greater for the <span class="gene">HPL</span> fractions than for the HPB fractions. There was a decrease in the MWAP values of the DOM samples following sorption to goethite. In many cases the reduction in MWAP was large, indicating a strong preference by goethite for the higher MWAP DOM fractions. The results of this laboratory-based research demonstrate that microbial processes affect the chemical characteristics of DOM which may affect the distribution of soil organic C pools.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:L-DOPA increases lignification associated with Glycine max root growth-inhibition.</h2>
			<h3>PMID:17195115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Soares AR(1), Ferrarese Mde L, Siqueira Rde C, Böhm FM, Ferrarese-Filho O.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						L-3,4-dihydroxyphenylalanine (L: -DOPA), an allelochemical exuded from the roots of velvet bean [Mucuna pruriens (L.) DC. var. utilis], presents a highly inhibitory action to plant growth. The effects of L-DOPA on phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) and peroxidase (POD, EC 1.11.1.7) activities, and phenolic compound and lignin content in soybean [Glycine max (L.) Merr.] roots were investigated to determine the possible phytotoxic mechanism. Three-day-old seedlings were cultivated in half-strength Hoagland nutrient solution (pH 6.0), without or with 0.1 to 1.0 mM L-DOPA in a growth chamber (25 degrees C, 12-hr light to 12-hr darkness <span class="trait">photoperiod,</span> irradiance of 280 micromol m-2 s-1) for 24 hr. In general, the length, fresh weight, and dry weight of the roots decreased, whereas <span class="gene">PAL</span> and POD activities and phenolic compound and lignin content increased after L-DOPA treatments. Results showed the susceptibility of soybean to L-DOPA and reinforce the role of this nonprotein amino acid as a strong allelochemical. The present findings also suggest that L-DOPA-induced inhibition in soybean roots may be because of a cell wall stiffening process related to the formation of cross-linking between cell wall polymers linked to lignin production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of palm kernel meal and corn distillers grains in corn silage-based diets for lactating dairy cows.</h2>
			<h3>PMID:16772589  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carvalho LP(1), Cabrita AR, Dewhurst RJ, Vicente TE, Lopes ZM, Fonseca AJ.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of increasing levels of solvent-extracted palm kernel meal (SPKM) and corn distillers dried grains (CDG) in corn silage-based diets on feed intake and milk production were examined in 2 experiments. In Experiment 1, 20 Holstein cows averaging 100 d in milk (DIM) (SD = 61.5) at the start of the experiment were utilized in an 11-wk randomized complete block design with 4 treatments in 5 blocks to study effects of increasing levels of SPKM in the diet. During a 3-wk preliminary period, cows were fed a standard diet. At the end of the preliminary period, cows were blocked by 4% fat-corrected milk <span class="trait">yield,</span> parity number (primiparous and multiparous), and DIM, and were assigned randomly to 1 of 4 experimental diets. The total mixed ration (TMR) consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of SPKM were achieved by replacing protein sources and citrus pulp with SPKM and urea (0, 5, 10, and 15% SPKM and 0.06, 0.22, 0.38, and 0.55% urea for SPKM0, SPKM5, SPKM10, and SPKM15, respectively). In Experiment 2, 18 Holstein cows averaging 93 <span class="gene">DIM</span> (SD = 49.1) at the start of the experiment were utilized in an 11-wk randomized complete block design with 3 treatments in 6 blocks to study effects of increasing levels of CDG in the diet. The preliminary period lasted for 2 wk. Assignment of cows to treatments was the same as in Experiment 1. The TMR consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of CDG were achieved by replacing soybean meal and citrus pulp with CDG and urea (0, 7, and 14% CDG and 0, 0.22, and 0.49% urea for CDG0, CDG7, and CDG14, respectively). There were no significant treatment effects on dry matter intake, milk <span class="trait">yield,</span> or milk composition in Experiment 1. Inclusion of SPKM tended to increase protein and lactose contents of milk. The SPKM0 diet promoted body weight loss. There were no treatment effects in Experiment 2, except for milk protein content, which was decreased by CDG. Plasma Lys concentration tended to be affected by SPKM and CDG inclusions. Plasma concentrations of 3-methylhistidine and Leu seemed to be related to body protein degradation/synthesis. The feeding of SPKM up to 15% in the diet decreased feed costs without detrimental effects on productive responses and tended to increase milk protein content. The inclusion of CDG in diets based on corn silage and corn byproducts might decrease milk protein content due to an unbalanced supply of AA, particularly Lys.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Laboratory and field evaluations of transgenic soybean exhibiting high-dose expression of a synthetic Bacillus thuringiensis cry1A gene for control of Lepidoptera.</h2>
			<h3>PMID:15889751  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：MacRae TC(1), Baur ME, Boethel DJ, Fitzpatrick BJ, Gao AG, Gamundi JC, Harrison LA, Kabuye VT, McPherson RM, Miklos JA, Paradise MS, Toedebusch AS, Viegas A.
				<br/> 
				年份：2005 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transgenic lines of soybean, Glycine max (L.) Merrill, expressing a synthetic cry1A gene (tic107) from Bacillus thuringiensis (Bt), were evaluated in screenhouse and conventional field trials for efficacy against lepidopteran pests. In screenhouse trials, Bt soybean and negative checks (isogenic segregants and parental lines) were evaluated against Anticarsia gemmatalis Hübner and Pseudoplusia includens (Walker) in the United States and against A. gemmatalis, Epinotia aporema (Walsingham), Rachiplusia nu (Guenée), and Spilosoma virginica (F.) in Argentina. Bt soybean exhibited virtually complete efficacy against each of these pests, whereas negative checks suffered significant damage. Bt soybean and negative checks also were evaluated in conventional trials against native populations of A. gemmatalis and P. includens in the southeastern United States. Each of these insects caused significant damage to negative checks in one or more locations, whereas Bt soybean exhibited virtually complete efficacy against these pests. In the laboratory, lyophilized leaf tissues from Bt soybean incorporated in artificial diet at a concentration representing a 25-fold dilution of fresh tissue caused complete mortality of A. gemmatalis and near complete mortality of P. includens neonates after 11 d, whereas mortality on negative checks did not exceed 10% for either insect. Average TIC107 expression approached or exceeded 50 microg/g fresh weight at <span class="gene">V3</span> stage of growth and 200 microg/g by R6 stage of growth. These results demonstrate that expression of TIC107 in soybean can not only achieve highly efficacious control of several lepidopterans under field conditions but also provide a high dose for effective insect <span class="trait">resistance</span> management.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being seed oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:On-line microdialysis of proteins with high-salt buffers for direct coupling of electrospray ionization mass spectrometry and liquid chromatography.</h2>
			<h3>PMID:12831191  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Canarelli S(1), Fisch I, Freitag R.
				<br/> 
				年份：2002 Mar 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mass spectrometry (MS) is one of the most powerful instrumental techniques for protein analysis. The electrospray ionization (ESI) approach is known to be very gentle and at the same time compatible with liquid separation techniques such as HPLC and CE. However, ESI is known to be susceptible to salts and impurities, which often cause a dramatic decrease in <span class="trait">sensitivity</span> due to the suppression of the ionization of the product of interest. For this reason, LC-ESI-MS coupling has so far been largely limited to reversed-phase chromatography with its hydro-organic mobile phases. Other chromatographic techniques are typically "linked" to ESI-MS by time consuming, off-line desalting steps. On-line microdialysis has been proposed as a solution to this dilemma. In this paper, we introduce an improved microdialysis system, which enlarges the number of putative applications, thus allowing chromatographic separations of biological compounds to be directly coupled to <span class="gene">MS</span> detection with little to no loss in time or chromatographic resolution. Examples include separations by affinity, ion-exchange and size-exclusion chromatography, all of which were connected successfully to the ESI-MS detector via the on-line microdialyzer. We propose that, using this system, any kind of chromatography technique can be coupled to ESI-MS, thus enabling for example application in quality control or process monitoring of many bioproduction and downstream processes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-throughput quantification of soy isoflavones in human and rodent blood using liquid chromatography with electrospray mass spectrometry and tandem mass spectrometry detection.</h2>
			<h3>PMID:12270207  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Twaddle NC(1), Churchwell MI, Doerge DR.
				<br/> 
				年份：2002 Sep 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soy-containing foods and dietary supplements are widely consumed for putative health benefits (e.g., cancer chemoprevention, beneficial effects on serum lipids associated with cardiovascular health, reduction of osteoporosis, relief of menopausal symptoms). However, studies of soy isoflavones in experimental animals suggest possible adverse effects as well (e.g., enhancement of reproductive organ cancer, modulation of endocrine function, anti-thyroid effects). This paper describes the development and validation of a sensitive high throughput method for quantifying isoflavones in blood from experimental animal and human studies. Serum samples containing genistein, daidzein, and equol were processed using reverse phase solid-phase extraction in the 96-well format for subsequent LC-ES/MS/MS or LC-ES/MS analysis using isotope dilution in conjunction with labeled internal standards. The method was validated by repetitive analysis of spiked blank serum and the intra-day and inter-day accuracy (88-99%) and precision (relative standard deviations from 3 to 13%) of measurement determined. The lower limit of quantification for all isoflavones was approximately 0.005 micro M using MS/MS detection, and 0.03 micro M using <span class="gene">MS</span> for genistein and daidzein. The degree of method performance, with respect to throughput, <span class="trait">sensitivity</span> and selectivity, makes this approach practical for analysis of large sample sets generated from mechanistic animal studies and human clinical trials of soy isoflavones.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conversion of AFLP bands into high-throughput DNA markers.</h2>
			<h3>PMID:11361330  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Meksem K(1), Ruben E, Hyten D, Triwitayakorn K, Lightfoot DA.
				<br/> 
				年份：2001 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The conversion of AFLP bands into polymorphic sequence-tagged-site (STS) markers is necessary for high-throughput genotype scoring. Technical hurdles that must be overcome arise from genome complexity (particularly sequence duplication), from the low-molecular-weight nature of the AFLP bands and from the location of the polymorphism within the AFLP band. We generated six <span class="gene">STS</span> markers from ten AFLP bands (four AFLPs were from co-dominant pairs of bands) in soybean (Glycine max). The markers were all linked to one of two loci, rhg1 on linkage group G and <span class="gene">Rhg4</span> on linkage group A2, that confer <span class="trait">resistance</span> to the soybean cyst nematode (Heterodera glycines I.). When the polymorphic AFLP band sequence contained a duplicated sequence or could not be converted to a locus-specific <span class="gene">STS</span> marker, direct sequencing of BAC clones anchored to a physical map generated locus-specific flanking sequences at the polymorphic locus. When the polymorphism was adjacent to the restriction site used in the AFLP analysis, single primer extension was performed to reconstruct the polymorphism. The six converted AFLP markers represented 996 bp of sequence from alleles of each of two cultivars and identified eight insertions or deletions, two microsatellites and eight single-nucleotide polymorphisms (SNPs). The polymorphic sequences were used to design a non-electrophoretic, fluorometric assay (based on the TaqMan technology) and/or develop electrophoretic <span class="gene">STS</span> markers for high-throughput genotype determination during marker-assisted breeding for <span class="trait">resistance</span> to cyst nematode. We conclude that the converted AFLP markers contained polymorphism at a 10- to 20-fold higher frequency than expected for adapted soybean cultivars and that the efficiency of AFLP band conversion to <span class="gene">STS</span> can be improved using BAC libraries and physical maps. The method provides an efficient tool for SNP and <span class="gene">STS</span> discovery suitable for marker-assisted breeding and genomics.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Abscisic acid induces a decline in nitrogen fixation that involves leghaemoglobin, but is independent of sucrose synthase activity.</h2>
			<h3>PMID:11283173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：González EM(1), Gálvez L, Arrese-Igor C.
				<br/> 
				年份：2001 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS) activity has been suggested to be a key point of regulation in nodule metabolism since this enzyme is down-regulated in response to different stresses which lead to decreased nitrogen fixation. In soybean, a dramatic decline of <span class="gene">SS</span> transcripts has been observed within 1 d from the onset of <span class="trait">drought.</span> Such a quick response suggests mediation by a signal transduction molecule. Abscisic acid (ABA) is a likely candidate to act as such a molecule as it mediates in a significant number of plant responses to environmental constraints. The hypothesis of ABA controlling nodule metabolism was approached in this work by assessing nodule responses to exogenous ABA supply in pea. Under the experimental conditions, ABA did not affect plant biomass, nodule numbers or dry weight. However, nitrogen fixation rate was reduced by 70% within 5 d and by 80% after 9 d leading to a reduced plant organic nitrogen content. Leghaemoglobin (Lb) content declined in parallel with that of nitrogen fixation. <span class="gene">SS</span> activity, however, was not affected by ABA treatment, and neither were the activities of the enzymes aspartate amino transferase, alkaline invertase, malate dehydrogenase, glutamate synthase, uridine diphosphoglucose pyrophosphorylase, isocitrate dehydrogenase, and glutamine synthetase. Nodule bacteroid-soluble protein content was reduced in nodules only after 9 d of ABA treatment. These results do not support the hypothesis that ABA directly regulates <span class="gene">SS</span> activity. However, they do suggest the occurrence of at least two different control pathways in nodules under environmental constraints, which include ABA being involved in a Lb/oxygen-related control of nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fumaric acid: an overlooked form of fixed carbon in Arabidopsis and other plant species.</h2>
			<h3>PMID:11089689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chia DW(1), Yoder TJ, Reiter WD, Gibson SI.
				<br/> 
				年份：2000 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photoassimilates are used by plants for production of energy, as carbon skeletons and in transport of fixed carbon between different plant organs. Many studies have been devoted to characterizing the factors that regulate photoassimilate concentrations in different plant species. Most studies examining photoassimilate concentrations in <span class="gene">C3</span> plants have focused on analyzing starch and soluble sugars. However, work presented here demonstrates that a number of <span class="gene">C3</span> plants, including the popular model organism Arabidopsis thaliana (L.) Heynh., and agriculturally important plants, such as soybean, Glycine max (L.) Merr., contain significant quantities of fumaric acid. In fact, fumaric acid can accumulate to levels of several milligrams per gram fresh weight in Arabidopsis leaves, often exceeding those of starch and soluble sugars. Fumaric acid is a component of the tricarboxylic acid cycle and, like starch and soluble sugars, can be metabolized to <span class="trait">yield</span> energy and carbon skeletons for production of other compounds. Fumaric acid concentrations increase with plant age and light intensity in Arabidopsis leaves. Moreover, Arabidopsis phloem exudates contain significant quantities of fumaric acid, raising the possibility that fumaric acid may function in carbon transport.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distribution of two isoforms of NADP-dependent isocitrate dehydrogenase in soybean (Glycine max).</h2>
			<h3>PMID:10394941  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Park KS(1), Kahn ML.
				<br/> 
				年份：1999 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two different cDNAs that encode NADP-specific isocitrate dehydrogenase (NADP-IDH) isozymes of soybean (Glycine max) were characterized. The nucleotide sequences of the coding regions of these cDNAs have 74% identity to each other and give predicted amino acid sequences that have 83% identity to each other. Using PCR techniques, their coding regions were subcloned into a protein overexpression vector, pQE32, to <span class="trait">yield</span> pIDH4 and pIDH1, respectively. Both IDH4 and <span class="gene">IDH1</span> enzymes were expressed in Escherichia coli as catalytically active His6 tagged proteins, purified to homogeneity by affinity chromatography on nickel chelate resin and rabbit polyclonal antibodies to each were generated. Surprisingly, antiserum to IDH4 did not react with <span class="gene">IDH1</span> protein and <span class="gene">IDH1</span> antiserum reacted only very weakly with IDH4 protein. IDH4 antibody reacts with a protein of expected molecular weight in cotyledon, young leaf, young root, mature root and nodules but the reaction with mature leaf tissue was low compared to other tissues. Western blot results show that <span class="gene">IDH1</span> was not expressed in young roots but a protein that reacts with the <span class="gene">IDH1</span> antibody was highly expressed in leaves, showing that there was tissue-specific accumulation of NADP-IDH isozymes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular strategies to improve the nutritional quality of legume proteins.</h2>
			<h3>PMID:10335390  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：de Lumen BO(1), Galvez AF, Revilleza MJ, Krenz DC.
				<br/> 
				年份：1999
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume proteins are relatively deficient in the sulfur amino acids. Among the different strategies to increase the sulfur amino acid contents of legumes, we have chosen to increase the biosynthesis of endogenous, non-abundant sulfur-rich proteins we have identified in soybean seed. We identified and isolated an 8 kDa sulfur-rich protein from the albumin fraction of soybean seed. Subsequently, a cDNA designated <span class="gene">Gm2S-1,</span> has been isolated and characterized. The <span class="gene">Gm2S-1</span> cDNA encodes a pre-proprotein that contains a signal peptide and a precursor protein that undergoes post-translational processing to <span class="trait">yield</span> the mature 8 kDa protein containing 7.8% methionine and 7.8% cysteine. The <span class="gene">Gm2S-1</span> gene is an ideal candidate for overexpression to improve the nutritional quality of soybean and other legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evidence for the formation of dinor isoprostanes <span class="gene">E1</span> from alpha-linolenic acid in plants.</h2>
			<h3>PMID:9830005  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Parchmann S(1), Mueller MJ.
				<br/> 
				年份：1998 Dec 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The free radical oxidation of arachidonic acid is known to generate complex metabolites, termed isoprostanes, that share structural features of prostaglandins and exert potent receptor-mediated biological activities. In the present study, we show that alpha-linolenic acid can undergo a similar oxidation process, resulting in a series of isomeric dinor isoprostanes E1. E-ring dinor isoprostane formation from linolenate was found to be catalyzed by soybean lipoxygenase. The main enzymatic products were 13- and 9-hydroperoxylinolenate but in addition, two dinor isoprostane <span class="gene">E1</span> regioisomers were formed with a <span class="trait">yield</span> of 0.31%. Identification and quantification of two dinor isoprostane <span class="gene">E1</span> regioisomers in plant cell cultures was achieved by a negative chemical ionization gas chromatography-mass spectrometry method using [18O]dinor isoprostanes <span class="gene">E1</span> as internal standards. Endogenous levels of these compounds were determined in four taxonomically distant plant species and found to be in the range of 4.5 to 60.9 ng/g of dry weight. Thus analogous pathways in animals and plants exist, each leading to a family of prostaglandin-like compounds derived from polyunsaturated fatty acids. It remains to be shown whether the dinor isoprostanes exert biological activities in plants as has been demonstrated for their C20 congeners in mammals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Supplemental protein plus ruminally protected methionine and lysine for primiparous beef cattle consuming annual rye hay.</h2>
			<h3>PMID:9690631  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hess BW(1), Scholljegerdes EJ, Coleman SA, Williams JE.
				<br/> 
				年份：1998 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Primiparous beef cows (n = 35 Bos taurus, average initial BW of 498 kg) were allotted to treatments in a split-plot designed experiment to determine the effects of supplemental ruminally protected amino acids on cow and calf productivity and metabolic changes in the cows. Cows were fed chopped annual rye hay at 1.5% of BW. The following treatments were used: 1) .8 kg soybean hulls, 1.4 kg ground corn, .6 kg soybean meal (CON); 2) 1.4 kg ground corn, 1.4 kg soybean meal (PRO); 3) PRO plus ruminally protected methionine and lysine (supplied 5 and 10 g, respectively; PRO1); and 4) PRO with twice the level of ruminally protected amino acids in <span class="gene">PRO1</span> (PRO2). Cow weight gain was not different (P &gt; .26) among treatments and averaged 1.2 kg/d for the 45 + 6 d before parturition. After parturition, cow weight gain did not differ (P &gt; .47) between CON and PRO treatments, but it decreased quadratically (P &lt; .01) with increasing level of ruminally protected amino acids. Total milk <span class="trait">yield,</span> protein, and fat (4 h) were greater (P &lt; .05) for cows consuming PRO supplements than for CON, whereas CON cattle tended (P = .11) to lose less body condition. Total milk protein showed a quadratic increase (P &lt; .05) in response to level of ruminally protected amino acids that was the inverse of the quadratic response noted for cow weight gain. Serum urea N concentration was greater (P = .07) for cattle consuming additional protein. Metabolic hormones were not affected (P &gt; .18) by dietary treatment, but they responded (P &lt; .05) to changes in physiological state. Supplements with additional protein supported greater (P = .0001) milk urea N concentration and output. Milk urea N concentration increased (P &lt; .05) and milk IGF-I decreased (P &lt; .05) as the lactation period progressed. The measurement of CON and PRO diets revealed that supplements with additional soybean meal had greater (P &lt; .05) DM and N degradation; the extent of forage DM and NDF degradation was similar (P &gt; .05) among treatments. Production shifted away from body weight gain to increased milk protein production when daily supplementation of ruminally protected methionine and lysine increased from 5 and 10 to 10 and 20 g, respectively. This shift in production was not reflective of changes in the metabolic regulators measured.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phosphoenolpyruvate carboxylase protein kinase from soybean root nodules: partial purification, characterization, and up/down-regulation by photosynthate supply from the shoots.</h2>
			<h3>PMID:9224739  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang XQ(1), Chollet R.
				<br/> 
				年份：1997 Jul 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphoenolpyruvate carboxylase (PEPC) kinase was partially purified about 3000-fold from soybean root nodules by a fast-protein liquid chromatography protocol. This protein-serine kinase has an apparent native molecular mass of about 30,000 as estimated by size-exclusion chromatography. Following electrophoresis of this partially purified PEPC-kinase preparation in a denaturing gel containing dephospho maize leaf PEPC as substrate, the in situ renaturation and assay of protein kinase activity revealed two, PEPC-dependent kinase polypeptides with molecular masses of about 32 and 37 kDa. The approximately 32-kDa polypeptide was significantly more active than the approximately 37-kDa catalytic subunit. The activity of this partially purified PEPC kinase, and a less purified sample, was Ca2+-insensitive. This protein kinase preparation was able to phosphorylate purified PEPCs from soybean nodules, maize leaves, and a sorghum recombinant <span class="gene">C4</span> PEPC. In contrast, this PEPC kinase was unable to phosphorylate a phosphorylation-site mutant form of sorghum <span class="gene">C4</span> PEPC (S8Y), two other soybean nodule phosphoproteins [nodulin-26 and nodulin-100 (sucrose synthase)], bovine serum albumin, and histone III-S. Following in vitro phosphorylation of purified dephospho soybean nodule PEPC from stem-girdled plants by the partially purified nodule PEPC kinase, the former's activity and <span class="trait">sensitivity</span> to L-malate inhibition increased and decreased, respectively. Notably, the Ca2+-independent PEPC kinase activity in nodules from illuminated plants was markedly greater than that in nodules harvested from plants subjected to stem girdling or prolonged darkness. Furthermore, the kinase activity in nodules was controlled reversibly by illumination and extended darkness pretreatments of the parent plants, suggesting that photosynthate supply from the shoots is likely responsible for these striking changes in PEPC kinase activity observed in planta in the legume nodule.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induced <span class="trait">resistance</span> in soybean toHelicoverpa zea: Role of plant protein quality.</h2>
			<h3>PMID:24241708</h3>
				

				
			<p>
				作者：Bi JL(1), Felton GW, Mueller AJ.
				<br/> 
				年份：1994 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> in soybean toHelicoverpa zea is comprised of both constitutive and inducible factors. In this study, we investigated the induction of <span class="trait">resistance</span> byH. zea in both greenhouse and field studies. In a greenhouse experiment, fourth-instarH. zea growth rates were reduced by 39% after 24 hr feeding and by 27% after 48 hr when larvae fed on previously wounded <span class="gene">V3</span> foliage (cv. Forrest) compared with undamaged foliage. In a field study, the weight gain by larvae was more than 52% greater when larvae fed for 72 hr on undamaged R2/R3 soybean plants (cv. Braxton) compared to those that fed on previously wounded plants. A significant component of the induced <span class="trait">resistance</span> is due to a decline in the nutritional quality of foliar protein following foliar damage byH. zea. Foliar protein was extracted from damaged and undamaged foliage and incorporated into artificial diets. Larval growth was reduced 26% after four days and 49% after seven days on diets containing protein from damaged plants compared to larvae feeding on foliar protein from undamaged plants. Chemical analyses of protein quality also indicated a decline in quality in damaged plants compared to unwounded plants. Increases in lipoxygenase activity (53%), lipid peroxidation products (20%), and trypsin inhibitor content (34%) were observed in protein from wounded plants. Moreover, a 5.9% loss in free amines and 19% loss in total thiols occurred in protein from wounded plants. Larval feeding causes a significant increase in foliar lipoxygenase activity that varied among genotypes. Lipoxygenase isozymes were measured at pH 5.5, pH 7.0, and pH 8.5 in <span class="gene">V3</span> stage plants of Forrest, Hark, D75-1069, and PI 417061 genotypes. Lipoxygenase activity in each genotype was significantly increased after 72 hr of larval feeding at each pH level tested, with the exception of lipoxygenase isozymes at pH 5.5 in genotype PI 417061. Larval feeding on R2/R3 stage plants (field-grown cv. Braxton) for six days also increased foliar lipoxygenase activity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biochemical alterations in Bacillus megaterium as produced by aflatoxin B1.</h2>
			<h3>PMID:4993227  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Beuchat LR, Lechowich RV.
				<br/> 
				年份：1971 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacillus megaterium NRRL B-1368 cells and spores were produced on Trypticase Soy Broth (TSB) and Agar (TSA) containing 3.8 mug of aflatoxin B(1) per ml, analyzed for selected chemical constituents, and compared to cells and spores of B. megaterium produced on nontoxic Trypticase Soy Media. There was an initial 30% kill of cells after inoculation into toxic <span class="gene">TSB</span> and during the first 3.5 hr of incubation followed by a logarithmic growth phase in which the generation time was 75 min as compared to 20 min for the control culture. Chemical analyses revealed an increase in protein, deoxyribonucleic acid (DNA), and ribonucleic acid (RNA) on both a per cell basis and a per cent dry weight basis when B. megaterium was grown in toxic TSB. There was a concurrent decrease in the total amounts of cellular protein, DNA, and RNA synthesized in toxic TSB. Amino acid analyses of control and test cell walls showed little, if any, difference in cell wall composition. About 97% sporulation of B. megaterium occurred after 3 days on nontoxic <span class="gene">TSA</span> although 6 days were required to attain 65% sporulation on toxic TSA. Germination of spores was not inhibited by 4.0 mug of aflatoxin per ml but outgrowth was. No significant differences were observed in the heat <span class="trait">resistance,</span> protein, DNA, RNA, or dipicolinic acid content of spores formed on toxic <span class="gene">TSA</span> and nontoxic TSA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional and Evolutionary Characterization of the <italic> CONSTANS </italic> Gene Family in Short-Day Photoperiodic Flowering in Soybean</h2>
			<h3>PMID:24465684</h3>
				
					<h3>PMCID:3897488</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wu </surname>  <given-names> Faqiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Price </surname>  <given-names> Brian William </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Haider </surname>  <given-names> Waseem </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seufferheld </surname>  <given-names> Gabriela </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> Randall </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hanzawa </surname>  <given-names> Yoshie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean, one of the most important sources of protein and oil in the world, flowers in response to a <span class="trait">photoperiod</span> change from long day to short day at the summer solstice <xref ref-type="bibr" rid="pone.0085754-Borthwick1"> [1] </xref> . Despite this crop&#x02019;s agronomic importance, the molecular basis of the photoperiodic flowering response in soybean remains only partially characterized. Several studies have identified eight Maturity Loci, <span class="gene">E1</span> to E8, that affect flowering and seed maturity time <xref ref-type="bibr" rid="pone.0085754-Cober1"> [2] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0085754-Liu1"> [6] </xref> . The causal genes for four of these maturity loci, E1&#x02013; E4, were recently identified. The photoreceptor gene <italic> PHYTOCHROME A </italic> ( <italic> <span class="gene">PHYA</span> </italic> ) was isolated as the causal gene of E3 and E4 <xref ref-type="bibr" rid="pone.0085754-Liu2"> [7] </xref> , <xref ref-type="bibr" rid="pone.0085754-Watanabe1"> [8] </xref> , and a homolog of the key flowering gene <italic> GIGANTEA </italic> ( <italic> GI </italic> ) was isolated as the causal gene of E2 <xref ref-type="bibr" rid="pone.0085754-Watanabe2"> [9] </xref> . A transcription factor containing a plant-specific B3 domain was also recently cloned as the causal gene of <span class="gene">E1</span> <xref ref-type="bibr" rid="pone.0085754-Xia1"> [10] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						No significant variation in the mRNA abundance of <italic> <span class="gene">GmCOL</span> </italic> s appeared among the seven genotypes at the representative time points based on the RNA sequencing data under SD and LD ( <xref ref-type="fig" rid="pone-0085754-g002"> Figure 2 </xref> ). In contrast, striking variation appeared among the genotypes in the shift experiment. <italic> <span class="gene">GmCOL1a,</span> <span class="gene">GmCOL1b</span> </italic> and <italic> <span class="gene">GmCOL3b</span> </italic> in Clade I showed elevated mRNA abundance at all three time points in Clark and Williams 82 and at T1 (6&#x02236;30) in a NIL, L65-3366 (E1E2E3E4e5E7), in response to the <span class="trait">photoperiod</span> shift. A very similar response in mRNA abundance to the <span class="trait">photoperiod</span> shift was observed in <italic> <span class="gene">GmCOL5a</span> </italic> in Clade II. Another notable variation among genotypes was the elevated mRNA abundance of <italic> <span class="gene">GmCOL7a,</span> <span class="gene">GmCOL7b</span> </italic> and many <italic> <span class="gene">GmCOL</span> </italic> s in Clade III at T3 (22&#x02236;30) in L74-441 (e1E2e3E4e5E7) and at all three time points in L92-1195 (e1E2e3E4E5E7) and <italic> G. soja </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						A hierarchical clustering analysis of the mRNA expression patterns of <italic> <span class="gene">GmCOL</span> </italic> s under LD and SD revealed rapid divergence of regulatory mechanisms among <italic> <span class="gene">GmCOL</span> </italic> s as well as high conservation among recently derived homeologs. Two main branches appeared: a cluster of <italic> <span class="gene">GmCOL</span> </italic> s from Clades I and II, and a cluster of <italic> <span class="gene">GmCOL</span> </italic> s from Clade III ( <xref ref-type="fig" rid="pone-0085754-g003"> Figure 3 </xref> ), in agreement with the coding sequence similarity noted in the phylogenetic analysis ( <xref ref-type="fig" rid="pone-0085754-g001"> Figure 1A </xref> ). An exception was <italic> <span class="gene">GmCOL7a</span> </italic> and <italic> <span class="gene">GmCOL7b</span> </italic> from Clade II clustering together with <italic> <span class="gene">GmCOL8a</span> </italic> / <italic> <span class="gene">GmCOL8b</span> </italic> and <italic> <span class="gene">GmCOL9a</span> </italic> / <italic> <span class="gene">GmCOL9b</span> </italic> from Clade III, creating a sub-clade nested under the cluster of Clade III genes ( <xref ref-type="fig" rid="pone-0085754-g003"> Figure 3 </xref> ). This observation is consistent with the similarity in their mRNA abundance in response to the <span class="trait">photoperiod</span> shift ( <xref ref-type="fig" rid="pone-0085754-g002"> Figure 2 </xref> ). All pairs of homeologous genes derived from the last genome duplication clustered together, except the <italic> <span class="gene">GmCOL7a/mCOL7b,</span> <span class="gene">GmCOL8a/GmCOL8b</span> </italic> and <italic> <span class="gene">GmCOL9a/GmCOL9b</span> </italic> pairs ( <xref ref-type="fig" rid="pone-0085754-g003"> Figure 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We identified 26 COL homologs from the soybean genome and characterized their <span class="trait">photoperiod</span> response in mRNA accumulation, sequence divergence and protein function. The <italic> CO </italic> gene family participates in a wide range of events in plant development in response to photoperiodic signaling, including flowering <xref ref-type="bibr" rid="pone.0085754-Valverde1"> [50] </xref> , seedling growth <xref ref-type="bibr" rid="pone.0085754-Datta1"> [51] </xref> , <xref ref-type="bibr" rid="pone.0085754-Datta2"> [52] </xref> , dormancy <xref ref-type="bibr" rid="pone.0085754-Bohlenius2"> [53] </xref> and tuberization <xref ref-type="bibr" rid="pone.0085754-GonzlezSchain1"> [54] </xref> in vascular plants, as well as cell growth in the green algae <italic> Chlamidomonas </italic> <xref ref-type="bibr" rid="pone.0085754-Serrano1"> [18] </xref> . Functional divergence of the <italic> CO </italic> gene family in Arabidopsis has been reported. For instance, despite their high sequence similarity with CO, COL1 and COL2 cause no changes in flowering time when ectopically expressed <xref ref-type="bibr" rid="pone.0085754-Ledger1"> [55] </xref> . COL3 influences root growth and lateral root formation <xref ref-type="bibr" rid="pone.0085754-Datta1"> [51] </xref> and acts also as a repressor of flowering, unlike CO. Similar to COL3, <span class="gene">COL9</span> represses flowering <xref ref-type="bibr" rid="pone.0085754-Cheng1"> [56] </xref> . We observed that the soybean <italic> CO </italic> gene family, like its counterpart in Arabidopsis, showed highly diverse expression patterns while responding to photoperiods, strongly suggesting <span class="gene">GmCOLs&#x02019;</span> functional divergence in multiple aspects of soybean photoperiodic response, including flowering.
					</p>
				
					<p>&nbsp;&nbsp;
						Offering more support for their role in flowering, <italic> <span class="gene">GmCOL1a</span> </italic> and <italic> <span class="gene">GmCOL1b</span> </italic> display strong <span class="trait">photoperiod</span> responsiveness in two ways. First, the mRNA abundance level and patterns of <italic> <span class="gene">GmCOL1a</span> </italic> and <italic> <span class="gene">GmCOL1b</span> </italic> showed striking discrepancies between SD and LD, with a high peak under flowering-inductive SD ( <xref ref-type="fig" rid="pone-0085754-g004"> Figure 4 </xref> ). Second, <italic> <span class="gene">GmCOL1a</span> </italic> and <italic> <span class="gene">GmCOL1b,</span> </italic> as well as <italic> <span class="gene">GmCOL3a</span> </italic> and <italic> <span class="gene">GmCOL5a,</span> </italic> exhibited a strong response to the <span class="trait">photoperiod</span> shift ( <xref ref-type="fig" rid="pone-0085754-g002"> Figure 2 </xref> ). Taken together, our results suggest that <italic> <span class="gene">GmCOL1a</span> </italic> and <italic> <span class="gene">GmCOL1b</span> </italic> are inducers of photoperiodic flowering. Our work may also indicate that photoperiodic flowering regulation by the <italic> CO </italic> gene family in flowering plants stems from an ancestral gene of Clade I that harbors Arabidopsis CO and rice Hd1 as well as <span class="gene">GmCOL1a</span> and <span class="gene">GmCOL1b.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In the SD flowering plants rice and Pharbitis, expression of <italic> FT </italic> homologs peaks at the end of the night under SD, consistent with SD flowering plants&#x02019; requirement of exposure to long nights. Our results and published findings show that <italic> <span class="gene">GmCOL1a</span> </italic> and <italic> <span class="gene">GmCOL1b</span> </italic> expression peaks at the end of the night when <italic> <span class="gene">GmFT5a</span> </italic> expression rises ( <xref ref-type="fig" rid="pone-0085754-g004"> Figures 4 </xref> and <xref ref-type="fig" rid="pone-0085754-g005"> 5 </xref> ), indicating that a mechanism similar to the one in rice and Pharbitis measures <span class="trait">photoperiod</span> to regulate flowering in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SSR and AFLP based genetic diversity of soybean germplasm differing in <span class="trait">photoperiod</span> <span class="trait">sensitivity</span></h2>
			<h3>PMID:21637488</h3>
				
					<h3>PMCID:3036845</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Singh </surname>  <given-names> Ram K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1,3 </sup>  </xref>  <xref ref-type="author-notes" rid="fnaux1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Bhatia </surname>  <given-names> Virendra S. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Bhat </surname>  <given-names> K. V. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Mohapatra </surname>  <given-names> Trilochan </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Singh </surname>  <given-names> Nagendra K. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Bansal </surname>  <given-names> Kailash C. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Koundal </surname>  <given-names> K. R. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 12 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						*I = <span class="trait">Photoperiod</span> insensitive; LS = Low <span class="trait">sensitivity;</span> <span class="gene">MS</span> = moderate <span class="trait">sensitivity;</span> HS = high <span class="trait">sensitivity.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic and molecular bases of <span class="trait">photoperiod</span> responses of flowering in soybean</h2>
			<h3>PMID:23136492</h3>
				
					<h3>PMCID:3406791</h3>
					<h4>期刊:Breeding Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Satoshi </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-531"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harada </surname>  <given-names> Kyuya </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-531"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abe </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-531"> 2 </xref>  <xref rid="c1-bs-61-531" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 03 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> (L.) Merr.) is grown over a wide range of latitudes, from equatorial to at least 50 degrees north and 35 degrees south. However, the cultivation area of each cultivar is restricted to a very narrow range of latitudes. The wide adaptability of soybean has been created by natural variation in the major genes and quantitative trait loci (QTLs) controlling flowering. At present, nine major genes have been reported to control time to flowering and maturity in soybean: <italic> <span class="gene">E1</span> </italic> and <italic> E2 </italic> ( <xref rid="b3-bs-61-531" ref-type="bibr"> Bernard 1971 </xref> ), <italic> E3 </italic> ( <xref rid="b6-bs-61-531" ref-type="bibr"> Buzzell 1971 </xref> ), <italic> E4 </italic> ( <xref rid="b7-bs-61-531" ref-type="bibr"> Buzzell and Voldeng 1980 </xref> ), <italic> E5 </italic> ( <xref rid="b60-bs-61-531" ref-type="bibr"> McBlain and Bernard 1987 </xref> ), <italic> E6 </italic> ( <xref rid="b5-bs-61-531" ref-type="bibr"> Bonato and Vello 1999 </xref> ), <italic> E7 </italic> ( <xref rid="b15-bs-61-531" ref-type="bibr"> Cober and Voldeng 2001a </xref> ), <italic> E8 </italic> ( <xref rid="b19-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 2010 </xref> ), and <italic> J </italic> ( <xref rid="b70-bs-61-531" ref-type="bibr"> Ray <italic> et al. </italic> 1995 </xref> ). Linkage analyses have located these genes to molecular linkage groups (MLGs) <span class="gene">C1</span> <span class="gene">(Gm04)</span> for <italic> E8 </italic> ( <xref rid="b19-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 2010 </xref> ), <span class="gene">C2</span> <span class="gene">(Gm06)</span> for <italic> <span class="gene">E1</span> </italic> and <italic> E7 </italic> ( <xref rid="b15-bs-61-531" ref-type="bibr"> Cober and Voldeng 2001a </xref> , <xref rid="b63-bs-61-531" ref-type="bibr"> Molnar <italic> et al. </italic> 2003 </xref> ), I <span class="gene">(Gm20)</span> for <italic> E4 </italic> ( <xref rid="b1-bs-61-531" ref-type="bibr"> Abe <italic> et al. </italic> 2003 </xref> , <xref rid="b63-bs-61-531" ref-type="bibr"> Molnar <italic> et al. </italic> 2003 </xref> ), L <span class="gene">(Gm19)</span> for <italic> E3 </italic> ( <xref rid="b63-bs-61-531" ref-type="bibr"> Molnar <italic> et al. </italic> 2003 </xref> ) and O <span class="gene">(Gm10)</span> for <italic> E2 </italic> ( <xref rid="b2-bs-61-531" ref-type="bibr"> Akkaya <italic> et al. </italic> 1995 </xref> , <xref rid="b21-bs-61-531" ref-type="bibr"> Cregan <italic> et al. </italic> 1999 </xref> ). At all of the loci except for <italic> E6 </italic> and <italic> J </italic> , dominant alleles delay time to flowering to different extents, interacting with the environment and with genotypes at other loci. The recessive alleles <italic> e6 </italic> and <italic> j </italic> were identified in crosses with late-flowering cultivars carrying a long-juvenile trait to condition later flowering ( <xref rid="b5-bs-61-531" ref-type="bibr"> Bonato and Vello 1999 </xref> , <xref rid="b70-bs-61-531" ref-type="bibr"> Ray <italic> et al. </italic> 1995 </xref> ). In addition to these major genes, many QTLs controlling time to flowering have been reported ( <xref rid="b10-bs-61-531" ref-type="bibr"> Chapman <italic> et al. </italic> 2003 </xref> , <xref rid="b11-bs-61-531" ref-type="bibr"> Cheng <italic> et al. </italic> 2011 </xref> , <xref rid="b27-bs-61-531" ref-type="bibr"> Funatsuki <italic> et al. </italic> 2005 </xref> , <xref rid="b28-bs-61-531" ref-type="bibr"> Githiri <italic> et al. </italic> 2007 </xref> , <xref rid="b43-bs-61-531" ref-type="bibr"> Keim <italic> et al. </italic> 1990 </xref> , <xref rid="b44-bs-61-531" ref-type="bibr"> Khan <italic> et al. </italic> 2008 </xref> , <xref rid="b47-bs-61-531" ref-type="bibr"> Komatsu <italic> et al. </italic> 2007 </xref> , <xref rid="b50-bs-61-531" ref-type="bibr"> Lee <italic> et al. </italic> 1996 </xref> , <xref rid="b52-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> 2007 </xref> , <xref rid="b56-bs-61-531" ref-type="bibr"> 2011 </xref> , <xref rid="b54-bs-61-531" ref-type="bibr"> Liu and Abe 2010 </xref> , <xref rid="b57-bs-61-531" ref-type="bibr"> Mansur <italic> et al. </italic> 1993 </xref> , <xref rid="b67-bs-61-531" ref-type="bibr"> Orf <italic> et al. </italic> 1999 </xref> , <xref rid="b68-bs-61-531" ref-type="bibr"> Poopronpan <italic> et al. </italic> 2006 </xref> , <xref rid="b82-bs-61-531" ref-type="bibr"> Tasma <italic> et al. </italic> 2001 </xref> , <xref rid="b89-bs-61-531" ref-type="bibr"> Wang <italic> et al. </italic> 2004 </xref> , <xref rid="b90-bs-61-531" ref-type="bibr"> Watanabe <italic> et al. </italic> 2004 </xref> , <xref rid="b97-bs-61-531" ref-type="bibr"> Yamanaka <italic> et al. </italic> 2001 </xref> , <xref rid="b103-bs-61-531" ref-type="bibr"> Zhang <italic> et al. </italic> 2004 </xref> ). Some of these QTLs most likely correspond to one of the known major genes, such as <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> , <italic> E4 </italic> , or <italic> E8 </italic> ( <xref rid="b11-bs-61-531" ref-type="bibr"> Cheng <italic> et al. </italic> 2011 </xref> , <xref rid="b27-bs-61-531" ref-type="bibr"> Funatsuki <italic> et al. </italic> 2005 </xref> , <xref rid="b28-bs-61-531" ref-type="bibr"> Githiri <italic> et al. </italic> 2007 </xref> , <xref rid="b44-bs-61-531" ref-type="bibr"> Khan <italic> et al. </italic> 2008 </xref> , <xref rid="b54-bs-61-531" ref-type="bibr"> Liu and Abe 2010 </xref> , <xref rid="b90-bs-61-531" ref-type="bibr"> Watanabe <italic> et al. </italic> 2004 </xref> , <xref rid="b98-bs-61-531" ref-type="bibr"> Yamanaka <italic> et al. </italic> 2005 </xref> ). Some of these QTLs are described in the Soybase database ( <ext-link ext-link-type="uri" xlink:href="http://soybase.org/"> http://soybase.org/ </ext-link> ). The major genes and QTLs for flowering often influence agronomic traits other than flowering and maturity, such as plant height and <span class="trait">yield</span> ( <xref rid="b10-bs-61-531" ref-type="bibr"> Chapman <italic> et al. </italic> 2003 </xref> , <xref rid="b19-bs-61-531" ref-type="bibr"> Cober and Morrison 2010 </xref> , <xref rid="b50-bs-61-531" ref-type="bibr"> Lee <italic> et al. </italic> 1996 </xref> , <xref rid="b57-bs-61-531" ref-type="bibr"> Mansur <italic> et al. </italic> 1993 </xref> , <xref rid="b89-bs-61-531" ref-type="bibr"> Wang <italic> et al. </italic> 2004 </xref> , <xref rid="b103-bs-61-531" ref-type="bibr"> Zhang <italic> et al. </italic> 2004 </xref> ), degree of cleistogamy ( <xref rid="b44-bs-61-531" ref-type="bibr"> Khan <italic> et al. </italic> 2008 </xref> , <xref rid="b77-bs-61-531" ref-type="bibr"> Takahashi and Abe 1994 </xref> ), and seed coat pigmentation and cracking caused by chilling stress ( <xref rid="b28-bs-61-531" ref-type="bibr"> Githiri <italic> et al. </italic> 2007 </xref> , <xref rid="b78-bs-61-531" ref-type="bibr"> Takahashi and Abe 1999 </xref> ). Understanding of their molecular bases and their interactions with the environment may therefore be necessary to determine genotypic combinations that will lead to a higher or more stable <span class="trait">yield</span> in the cropping season of a particular region. In this review, we summarize the results obtained from previous studies of major genes and QTLs for flowering and those from recent molecular dissections of the maturity genes <italic> E2 </italic> , <italic> E3 </italic> and <italic> E4 </italic> and of soybean orthologs of the <italic> Arabidopsis FLOWERING LOCUS T </italic> gene. We also describe soybean orthologs for <italic> Arabidopsis </italic> flowering genes deposited in the Williams 82 genome database, and we discuss the resolution of QTL mapping and the positioning of these orthologs on genetic and physical maps. The information on soybean orthologs of <italic> Arabidopsis </italic> flowering genes should be helpful in the search for candidate genes for targeted major loci and QTLs for flowering in soybean, and in the development of functional DNA markers for use in breeding programs.
					</p>
				
					<p>&nbsp;&nbsp;
						Most soybean cultivars have a short-daylength (SD) requirement for floral induction. Flowering is usually suppressed under long-daylength (LD) conditions but induced when the daylength is shorter than a critical length. This <span class="trait">sensitivity</span> to <span class="trait">photoperiod</span> varies among cultivars: in particular, it is weak or absent in soybean cultivars adapted to high latitudes. Four major genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , <italic> E4 </italic> and <italic> E7 </italic> ) have been well characterized for their responses to LD artificially induced by fluorescent and incandescent lamps with different red to far-red quantum (R : FR) ratios ( <xref rid="b6-bs-61-531" ref-type="bibr"> Buzzell 1971 </xref> , <xref rid="b7-bs-61-531" ref-type="bibr"> Buzzell and Voldeng 1980 </xref> , <xref rid="b13-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996a </xref> , <xref rid="b14-bs-61-531" ref-type="bibr"> 1996b </xref> , <xref rid="b17-bs-61-531" ref-type="bibr"> 2001 </xref> , <xref rid="b15-bs-61-531" ref-type="bibr"> Cober and Voldeng 2001a </xref> , <xref rid="b16-bs-61-531" ref-type="bibr"> 2001b </xref> , <xref rid="b45-bs-61-531" ref-type="bibr"> Kilen and Hartwig 1971 </xref> , <xref rid="b71-bs-61-531" ref-type="bibr"> Saindon <italic> et al. </italic> 1989a </xref> , <xref rid="b72-bs-61-531" ref-type="bibr"> 1989b </xref> ). The <italic> E3 </italic> locus was first identified by extending natural daylength to 20 h with the use of cool-white fluorescent lamps with a high R : FR ratio; the <italic> e3e3 </italic> recessive homozygote alone can initiate flowering under fluorescence-induced LD (FLD) ( <xref rid="b6-bs-61-531" ref-type="bibr"> Buzzell 1971 </xref> , <xref rid="b45-bs-61-531" ref-type="bibr"> Kilen and Hartwig 1971 </xref> ). The <italic> E4 </italic> locus was identified by extending the natural daylength to 20 h with incandescent lamps with a low R : FR ratio ( <xref rid="b7-bs-61-531" ref-type="bibr"> Buzzell and Voldeng 1980 </xref> ). A homozygous recessive <italic> e4e4 </italic> genotype is necessary for plants homozygous for the <italic> e3 </italic> allele to flower under incandescence-induced LD (ILD) without any marked delay in flowering ( <xref rid="b7-bs-61-531" ref-type="bibr"> Buzzell and Voldeng 1980 </xref> , <xref rid="b71-bs-61-531" ref-type="bibr"> Saindon <italic> et al. </italic> 1989a </xref> ). However, the <italic> e4e4 </italic> genotype cannot on its own confer insensitivity to FLD. <italic> E3 </italic> and <italic> E4 </italic> most likely control flowering under LD conditions with a wide range of R : FR ratios in a non-additive manner.
					</p>
				
					<p>&nbsp;&nbsp;
						The other known flowering loci, <italic> <span class="gene">E1</span> </italic> , <italic> E7 </italic> and <italic> E8 </italic> , are also involved in the control of <span class="trait">sensitivity</span> to ILD, particularly in the double-recessive <italic> e3e3e4e4 </italic> genetic background ( <xref rid="b14-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996b </xref> , <xref rid="b17-bs-61-531" ref-type="bibr"> 2001 </xref> , <xref rid="b19-bs-61-531" ref-type="bibr"> 2010 </xref> , <xref rid="b15-bs-61-531" ref-type="bibr"> Cober and Voldeng 2001a </xref> , <xref rid="b16-bs-61-531" ref-type="bibr"> 2001b </xref> ). <italic> <span class="gene">E1</span> </italic> has the largest effect on flowering ( <xref rid="b3-bs-61-531" ref-type="bibr"> Bernard 1971 </xref> , <xref rid="b61-bs-61-531" ref-type="bibr"> McBlain <italic> et al. </italic> 1987 </xref> , <xref rid="b87-bs-61-531" ref-type="bibr"> Upadhyay <italic> et al. </italic> 1994 </xref> ). However, a near-isogenic line (NIL) of cv. Harosoy homozygous for <italic> <span class="gene">E1</span> </italic> , <italic> e3 </italic> and <italic> e4 </italic> (OT93-28) initiated flowering at the same time as the NIL homozygous for <italic> e1 </italic> , <italic> e3 </italic> and <italic> e4 </italic> (OT85-9) under FLD, suggesting that <italic> <span class="gene">E1</span> </italic> does not influence time to flowering under LD with a high R : FR ratio ( <xref rid="b14-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996b </xref> ). In contrast, <italic> <span class="gene">E1</span> </italic> exhibits a marked inhibitory effect on flowering under ILD with a R : FR ratio of less than 1.0 ( <xref rid="b14-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996b </xref> , <xref rid="b84-bs-61-531" ref-type="bibr"> Thakare <italic> et al. </italic> 2010 </xref> ). A homozygous recessive <italic> e7e7 </italic> genotype further weakens the response of plants homozygous for <italic> e1 </italic> , <italic> e3 </italic> and <italic> e4 </italic> to ILD ( <xref rid="b15-bs-61-531" ref-type="bibr"> Cober and Voldeng 2001a </xref> , <xref rid="b17-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 2001 </xref> ). However, the <italic> e7e7 </italic> genotype does not confer a complete loss of <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> ( <xref rid="b17-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 2001 </xref> ). Another recessive allele, <italic> e8 </italic> , is involved in the genetic difference in flowering time observed between breeding lines of genotype <italic> e1e1e3e3e4e4e7e7 </italic> ( <xref rid="b19-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The different responses of <italic> E </italic> genes to LD conditions with different R : FR ratios have suggested that some of these genes are involved in phytochrome A (phyA)-regulated floral induction in soybean ( <xref rid="b14-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996b </xref> , <xref rid="b17-bs-61-531" ref-type="bibr"> 2001 </xref> ). <xref rid="b53-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> (2008) </xref> analyzed the sequence variation in a <italic> <span class="gene">phyA</span> </italic> homolog ( <italic> <span class="gene">GmphyA2</span> </italic> ) between NILs that were <span class="trait">photoperiod</span> sensitive and insensitive for <italic> E4 </italic> . They found that a <italic> Ty1 </italic> / <italic> copia </italic> -like retrotransposon designated <italic> SORE-1 </italic> was inserted in the first exon of the <italic> <span class="gene">GmphyA2</span> </italic> gene of photoperiod-insensitive lines carrying the recessive <italic> e4 </italic> allele ( <xref rid="b42-bs-61-531" ref-type="bibr"> Kanazawa <italic> et al. </italic> 2009 </xref> , <xref rid="b53-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> 2008 </xref> ). This insertion resulted in a premature stop codon causing a truncated and dysfunctional protein. Genetic mapping analysis confirmed that <italic> <span class="gene">GmphyA2</span> </italic> cosegregated with <italic> E4 </italic> on MLG I <span class="gene">(Gm20)</span> ( <xref rid="b1-bs-61-531" ref-type="bibr"> Abe <italic> et al. </italic> 2003 </xref> , <xref rid="b53-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> 2008 </xref> ). Furthermore, the NIL for <italic> e4 </italic> showed an impaired deetiolation (greening) response under continuous FR-light conditions, as found in <italic> <span class="gene">phyA</span> </italic> null mutants of <italic> Arabidopsis </italic> ( <xref rid="b64-bs-61-531" ref-type="bibr"> Neff and Chory 1998 </xref> ), rice ( <xref rid="b79-bs-61-531" ref-type="bibr"> Takano <italic> et al. </italic> 2001 </xref> , <xref rid="b80-bs-61-531" ref-type="bibr"> 2005 </xref> ), and pea ( <xref rid="b93-bs-61-531" ref-type="bibr"> Weller <italic> et al. </italic> 1997 </xref> , <xref rid="b94-bs-61-531" ref-type="bibr"> 2001 </xref> ). Taking these findings together, <xref rid="b53-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> (2008) </xref> concluded that the <italic> E4 </italic> gene encodes the <span class="gene">GmphyA2</span> protein and that the recessive <italic> e4 </italic> allele is a loss-of function allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean possesses a homoeologous copy of <italic> <span class="gene">GmphyA2</span> </italic> , namely <italic> <span class="gene">GmphyA1</span> </italic> , in MLG O <span class="gene">(Gm10)</span> ( <xref rid="b12-bs-61-531" ref-type="bibr"> Choi <italic> et al. </italic> 2007 </xref> , <xref rid="b53-bs-61-531" ref-type="bibr"> Liu <italic> et al. </italic> 2008 </xref> ). The function of <italic> <span class="gene">GmphyA1</span> </italic> remains undetermined, because no genetic variant is available yet at this locus. However, two findings may indicate that <italic> <span class="gene">GmphyA1</span> </italic> , like <italic> E4 </italic> , is involved in both de-etiolation response and flowering under FR-enriched LD conditions. First, the <span class="gene">phyA</span> function of the <italic> e4 </italic> allele in the de-etiolation response was not completely lost, whereas the <italic> <span class="gene">phyA</span> </italic> null mutants of <italic> Arabidopsis </italic> , pea, and rice all showed a complete loss of the de-etiolation response under continuous FR light ( <xref rid="b64-bs-61-531" ref-type="bibr"> Neff and Chory 1998 </xref> , <xref rid="b79-bs-61-531" ref-type="bibr"> Takano <italic> et al. </italic> 2001 </xref> , <xref rid="b80-bs-61-531" ref-type="bibr"> 2005 </xref> , <xref rid="b93-bs-61-531" ref-type="bibr"> Weller <italic> et al. </italic> 1997 </xref> , <xref rid="b94-bs-61-531" ref-type="bibr"> 2001 </xref> ), suggesting that another <italic> <span class="gene">phyA</span> </italic> copy has a redundant function to <italic> E4 </italic> . Second, a Harosoy NIL for double-recessive alleles at <italic> E3 </italic> and <italic> E4 </italic> ( <italic> e3e3e4e4 </italic> ) did not respond to LD with a relatively high R : FR ratio (1.0&#x02013;5.0) but showed delayed flowering under LD with a low R : FR ratio (&#x0003c;1.0) ( <xref rid="b14-bs-61-531" ref-type="bibr"> Cober <italic> et al. </italic> 1996b </xref> , <xref rid="b17-bs-61-531" ref-type="bibr"> 2001 </xref> ). These redundant functions for de-etiolation and flowering suggest that <italic> <span class="gene">GmphyA1</span> </italic> itself functions redundantly with <italic> E4 </italic> in both de-etiolation responses and <span class="trait">photoperiod</span> responses under FR-enriched light. However, sequence analyses of both homoeologs in wild ( <italic> G. soja </italic> ) and cultivated soybeans revealed that the nucleotide diversity at non-synonymous sites was lower in <italic> <span class="gene">GmphyA1</span> </italic> than in <italic> <span class="gene">GmphyA2</span> </italic> , although the diversity at synonymous sites and non-coding regions was almost the same in the two loci, suggesting that <italic> <span class="gene">GmphyA1</span> </italic> has been subject to more intense purifying selection than <italic> <span class="gene">GmphyA2</span> </italic> (our unpublished data). Some degree of subfunctionalization may thus have occurred between the two <italic> <span class="gene">phyA</span> </italic> homoeologs. Further studies using dysfunctional mutants will be needed to determine the function of <italic> <span class="gene">GmphyA1</span> </italic> in photoperiodic responses of flowering.
					</p>
				
					<p>&nbsp;&nbsp;
						An SD-to-LD transfer experiment further demonstrated the difference in response to <span class="trait">photoperiod</span> between <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> . Expression of <italic> <span class="gene">GmFT2a</span> </italic> was strictly regulated by photoperiodic changes from SD to LD, whereas the response of <italic> <span class="gene">GmFT5a</span> </italic> to photoperiodic changes was gradual, and its expression was retained at low levels even after the plants were transferred to LD ( <xref rid="b48-bs-61-531" ref-type="bibr"> Kong <italic> et al. </italic> 2010 </xref> ). These findings suggest that, in addition to the phyA-mediated photo-period response, a second regulatory mechanism may also be involved in the differences in expression pattern between <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> . Under the phyA-mediated photo-periodic regulation system, <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> may redundantly and strongly induce flowering under shorter daylengths, but under longer daylengths <italic> <span class="gene">GmFT5a</span> </italic> alone may promote flowering in a photoperiod-independent manner. These two <italic> FT </italic> homologs may therefore coordinately control flowering in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Information on the physical position of orthologs to known <italic> Arabidopsis </italic> genes ( <xref rid="SD2" ref-type="supplementary-material"> Supplemental Table 1 </xref> and <xref rid="SD1" ref-type="supplementary-material"> Supplemental Fig. 1 </xref> ) may help to identify candidate genes for targeted major loci and QTLs. <xref rid="f2-bs-61-531" ref-type="fig"> Fig. 2 </xref> is an example showing the usefulness of physical map information in identifying candidate genes responsible for three flowering QTLs detected in a cross between Misuzudaizu and Moshido Gong 503 ( <xref rid="b90-bs-61-531" ref-type="bibr"> Watanabe <italic> et al. </italic> 2004 </xref> , <xref rid="b91-bs-61-531" ref-type="bibr"> 2009 </xref> , <xref rid="b92-bs-61-531" ref-type="bibr"> 2011 </xref> , <xref rid="b97-bs-61-531" ref-type="bibr"> Yamanaka <italic> et al. </italic> 2001 </xref> , <xref rid="b98-bs-61-531" ref-type="bibr"> 2005 </xref> ). The positions of DNA markers tagging the three QTLs, which were originally detected in an RIL population derived from these two parents, delineated their genomic positions in specific regions of <span class="gene">Gm06</span> (MLG C2), <span class="gene">Gm10</span> (MLG O) and <span class="gene">Gm19</span> (MLG L). Fine-mapping and QTL analysis detected two DNA markers separated by a genetic distance of 2 cM, Satt365 and Satt489, for <italic> qFT1 </italic> ( <xref rid="b98-bs-61-531" ref-type="bibr"> Yamanaka <italic> et al. </italic> 2005 </xref> ). The region flanked by the two markers corresponds to a physical distance of approximately 3 Mbp in a pericentromeric region where repetitive sequences are very rich and recombination is severely inhibited ( <xref rid="b8-bs-61-531" ref-type="bibr"> Cannon and Shoemaker 2012 </xref> , <xref rid="b75-bs-61-531" ref-type="bibr"> Schmutz <italic> et al. </italic> 2010 </xref> ). The genome sequence information predicted only one ortholog of an <italic> Arabidopsis </italic> flowering gene, <italic> REPRESSOR OF GA1-3 </italic> <span class="gene">(Glyma06g23940)</span> in the region ( <xref rid="f2-bs-61-531" ref-type="fig"> Fig. 2 </xref> ). Considering the involvement of <italic> qFT1 </italic> ( <italic> <span class="gene">E1</span> </italic> ) in <span class="trait">photoperiod</span> <span class="trait">sensitivity,</span> however, this ortholog is not likely to be the responsible gene, although further studies are needed to confirm its identity. Similarly, QTL mapping placed <italic> qFT2 </italic> ( <italic> E2 </italic> ) and <italic> qFT </italic> 3 ( <italic> E3 </italic> ) in regions on <span class="gene">Gm10</span> (MLG O) and <span class="gene">Gm19</span> (MLG L), respectively; each QTL is flanked by two SSR markers, which are separated by ca. 11 cM and 15 cM, respectively ( <xref rid="b90-bs-61-531" ref-type="bibr"> Watanabe <italic> et al. </italic> 2004 </xref> ). These regions contain one and three orthologs for <italic> qFT2 </italic> and <italic> qFT3 </italic> , respectively, although the physical distances between the markers are still over 1.0 Mbp ( <xref rid="f1-bs-61-531" ref-type="fig"> Fig. 1 </xref> ). Fine-mapping studies further narrowed these regions into regions of less than 100 Kbp within single genomic DNA clones, and finally a single ortholog could be evaluated and identified as a candidate gene for each <italic> qFT </italic> ( <xref rid="b91-bs-61-531" ref-type="bibr"> Watanabe <italic> et al. </italic> 2009 </xref> , <xref rid="b92-bs-61-531" ref-type="bibr"> 2011 </xref> ). Hence, fine-mapping subsequent to QTL analysis, together with a candidate gene approach based on the physical positions of <italic> Arabidopsis </italic> orthologs, may facilitate identification and characterization of molecular major genes and QTLs for flowering in soybean, although novel genes that have no corresponding <italic> Arabidopsis </italic> flowering gene should not be excluded from consideration as candidate genes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic variation in four maturity genes affects <span class="trait">photoperiod</span> insensitivity and PHYA-regulated post-flowering responses of soybean</h2>
			<h3>PMID:23799885</h3>
				
					<h3>PMCID:3698206</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Xu </surname>  <given-names> Meilan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> xumeilan_1984@yahoo.co.jp </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Xu </surname>  <given-names> Zeheng </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> ls_xuzh@163.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> liubh@neigaehrb.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> kongfj@neigaehrb.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Tsubokura </surname>  <given-names> Yasutaka </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Yasutaka.Tsubokura@snowseed.co.jp </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Watanabe </surname>  <given-names> Satoshi </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> nabemame@cc.saga-u.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Xia </surname>  <given-names> Zhengjun </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> xiazhj@neigaehrb.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Harada </surname>  <given-names> Kyuya </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> haradaq@nifty.com </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Kanazawa </surname>  <given-names> Akira </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kanazawa@res.agr.hokudai.ac.jp </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Yamada </surname>  <given-names> Testuya </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> tetsuyay@res.agr.hokudai.ac.jp </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Abe </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jabe@res.agr.hokudai.ac.jp </email>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Absence of or low <span class="trait">sensitivity</span> to <span class="trait">photoperiod</span> is necessary for short-day crops, such as rice and soybean, to adapt to high latitudes. <span class="trait">Photoperiod</span> insensitivity in soybeans is controlled by two genetic systems and involves three important maturity genes: <italic> <span class="gene">E1</span> </italic> , a repressor for two soybean orthologs of <italic> Arabidopsis FLOWERING LOCUS T </italic> ( <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> ), and <italic> E3 </italic> and <italic> E4 </italic> , which are phytochrome A genes. To elucidate the diverse mechanisms underlying <span class="trait">photoperiod</span> insensitivity in soybean, we assessed the genotypes of four maturity genes ( <italic> <span class="gene">E1</span> </italic> through <italic> E4 </italic> ) in early-flowering photoperiod-insensitive cultivars and their association with post-flowering responses.
					</p>
				
					<p>&nbsp;&nbsp;
						We found two novel dysfunctional alleles in accessions originally considered to have a dominant <italic> E3 </italic> allele according to known DNA markers. The <italic> E3 </italic> locus, together with <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> , contained multiple dysfunctional alleles. We identified 15 multi-locus genotypes, which we subdivided into 6 genotypic groups by classifying their alleles by function. Of these, the <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> genotypic group required an additional novel gene (different from <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> ) to condition <span class="trait">photoperiod</span> insensitivity. Despite their common pre-flowering <span class="trait">photoperiod</span> insensitivity, accessions with different multi-locus genotypes responded differently to the post-flowering <span class="trait">photoperiod.</span> Cultivars carrying <italic> E3 </italic> or <italic> E4 </italic> were sensitive to <span class="trait">photoperiod</span> for post-flowering characteristics, such as reproductive period and stem growth after flowering. The phytochrome A&#x02013;regulated expression of the determinate growth habit gene <italic> <span class="gene">Dt1</span> </italic> , an ortholog of <italic> Arabidopsis TERMINAL FLOWER1 </italic> , was involved in the persistence of the vegetative activity at the stem apical meristem of flower-induced plants under long-day conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Diverse genetic mechanisms underlie <span class="trait">photoperiod</span> insensitivity in soybean. At least three multi-locus genotypes consisting of various allelic combinations at <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> conferred pre-flowering <span class="trait">photoperiod</span> insensitivity to soybean cultivars but led to different responses to <span class="trait">photoperiod</span> during post-flowering vegetative and reproductive development. The <italic> <span class="gene">phyA</span> </italic> genes <italic> E3 </italic> and <italic> E4 </italic> are major controllers underlying not only pre-flowering but also post-flowering <span class="trait">photoperiod</span> responses. The current findings improve our understanding of genetic diversity in pre-flowering <span class="trait">photoperiod</span> insensitivity and mechanisms of post-flowering <span class="trait">photoperiod</span> responses in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Photoperiod</span> <span class="trait">sensitivity</span> is an important trait that enables crops to adapt to diverse latitudinal environments. In particular, absence of or low <span class="trait">sensitivity</span> to <span class="trait">photoperiod</span> is necessary for short-day (SD) crops, such as rice and soybean, to adapt to high latitudes. In soybean ( <italic> Glycine max </italic> (L.) Merr.), nine major genes, <italic> <span class="gene">E1</span> </italic> through <italic> E8 </italic> and <italic> J </italic> , have so far been reported to control time to flowering and maturity (reviewed in [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]). <span class="trait">Photoperiod</span> insensitivity is controlled by at least two genetic systems, in which dysfunctional alleles at three maturity loci&#x02014; <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> &#x02014;are involved. One control system is ascribed to the double-recessive genotype for <italic> E3 </italic> and <italic> E4 </italic> [ <xref ref-type="bibr" rid="B2"> 2 </xref> - <xref ref-type="bibr" rid="B6"> 6 </xref> ], which encode the phytochrome A (PHYA) proteins <span class="gene">GmPHYA3</span> and <span class="gene">GmPHYA2,</span> respectively [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Together with <span class="gene">GmPHYA1,</span> which is encoded by a homoeologous copy of <italic> E4 </italic> , these two <span class="gene">PHYA</span> proteins redundantly or complementarily function in floral induction and de-etiolation responses under various light conditions [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ]. <italic> E3 </italic> and <italic> E4 </italic> direct different flowering responses to long-day (LD) conditions with different red-to-far&#x02013;red (R:FR) quantum ratios [ <xref ref-type="bibr" rid="B2"> 2 </xref> - <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. <italic> E3 </italic> controls the response to light with a high or low R:FR ratio; plants homozygous for the recessive <italic> e3 </italic> allele can initiate flowering under the LD conditions generated by fluorescent lamps with a high R:FR ratio [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. <italic> E4 </italic> is involved in the response to light with a low R:FR ratio; plants homozygous for the <italic> e3 </italic> allele need a recessive <italic> e4 </italic> allele to flower under LD generated by incandescent lamps with a low R:FR ratio [ <xref ref-type="bibr" rid="B3"> 3 </xref> , <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The <italic> e4 </italic> allele itself cannot confer the insensitivity to LD conditions induced by both fluorescent and incandescent lamps under the <italic> E3 </italic> genetic background [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ]. The <span class="gene">PHYA</span> protein is not only an effective FR sensor but also acts as an R-light photoreceptor under R light with high-photon irradiance [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ], which is involved, either directly or via interactions with other photoreceptors, in various developmental processes (reviewed in [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]). The <italic> E3 </italic> and <italic> E4 </italic> genes therefore may participate in a non-additive manner in different aspects of <span class="gene">PHYA</span> functions, which are controlled by a single <italic> <span class="gene">phyA</span> </italic> gene in <italic> Arabidopsis </italic> . Another <italic> <span class="gene">phyA</span> </italic> gene in soybean, <italic> <span class="gene">GmphyA1</span> </italic> , has been suggested to function redundantly with <italic> E4 </italic> in the de-etiolation response of hypocotyls and floral induction under FR light [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. Owing to the lack of genetic variants causing phenotypic differences, the function of <italic> <span class="gene">GmphyA1</span> </italic> has not yet been determined.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the major genes and QTLs that have been reported so far, the <italic> <span class="gene">E1</span> </italic> gene has the most prominent effect on flowering time in soybean (reviewed in [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]). Cober et al. [ <xref ref-type="bibr" rid="B9"> 9 </xref> ] used an early-maturing cultivar Harosoy and its near-isogenic lines (NILs) for <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> and <italic> E4 </italic> loci to reveal their <span class="trait">photoperiod</span> responses to LDs with different R:FR ratios. They found that a NIL with <italic> <span class="gene">E1</span> </italic> / <italic> e3 </italic> / <italic> e4 </italic> was insensitive to R-enriched LD conditions but still retained <span class="trait">sensitivity</span> to FR-enriched LD conditions with the low R:FR ratio of 0.9, although the NIL with <italic> e1-as </italic> (originally designed as <italic> e1 </italic> , but renamed after [ <xref ref-type="bibr" rid="B13"> 13 </xref> ])/ <italic> e3 </italic> / <italic> e4 </italic> lost the <span class="trait">sensitivity</span> across the wide range of R:FR ratios [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. This result indicates that <italic> <span class="gene">E1</span> </italic> has a marked inhibitory effect on flowering, particularly under LD conditions with the low R:FR ratio.
					</p>
				
					<p>&nbsp;&nbsp;
						Positional cloning revealed that <italic> <span class="gene">E1</span> </italic> encodes a protein that contains a putative bipartite nuclear localization signal and a region related to the B3 domain, a highly conserved domain found in transcription factors in plants [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. The abundance of <italic> <span class="gene">E1</span> </italic> transcripts is under the photoperiodic control regulated by the <italic> E3 </italic> and <italic> E4 </italic> genes and is negatively correlated with that of two soybean orthologs of <italic> Arabidopsis FLOWERING LOCUS T </italic> , <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. <italic> <span class="gene">E1</span> </italic> expression was suppressed under SD conditions regardless of genotype and induced under LD conditions in plants containing either <italic> E3 </italic> or <italic> E4 </italic> but not in those with the double-recessive <italic> e3e3e4e4 </italic> genotype [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Furthermore, <italic> <span class="gene">E1</span> </italic> over-expression in transgenic T2 soybean plants suppressed the expression of two <italic> <span class="gene">GmFT</span> </italic> genes, thereby markedly delaying flowering, suggesting that <span class="gene">E1</span> is a direct repressor of <span class="gene">GmFTs</span> [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Taken together, the weakened function of <span class="gene">PHYA</span> caused by the double-recessive genotype at the <italic> E3 </italic> and <italic> E4 </italic> loci may ablate or weaken <span class="trait">sensitivity</span> to <span class="trait">photoperiod,</span> at least partly, through the down-regulation of <italic> <span class="gene">E1</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						The other system controlling <span class="trait">photoperiod</span> insensitivity in soybean acts through involvement of dysfunctional alleles at the <italic> <span class="gene">E1</span> </italic> locus itself. <span class="trait">Photoperiod</span> insensitivity in the Japanese landrace &#x02018;Sakamotowase&#x02019; has been suggested to be controlled by a different genetic mechanism from <span class="gene">PHYA</span> dysfunction, because this landrace has a dominant functional allele at the <italic> E4 </italic> locus [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. Quantitative trait locus (QTL) mapping using testcrosses with a photoperiod-sensitive Harosoy NIL whose genotype was <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> indicated that the <span class="trait">photoperiod</span> insensitivity in Sakamotowase was controlled mainly by an allele at or a gene tightly linked to the <italic> <span class="gene">E1</span> </italic> locus, although a minor QTL was detected in linkage group L (Glyma19) [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. Analysis of the <italic> <span class="gene">E1</span> </italic> sequence of Sakamotowase revealed that it contained a dysfunctional allele, <italic> <span class="gene">e1-fs</span> </italic> , that produced a loss-of-function truncated protein due to a premature stop codon, which arose as a consequence of a single-base deletion [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. In contrast, the recessive allele <italic> e1-as </italic> possessed a nonsynonymous substitution in the putative nuclear localization signal, leading to reduced localization specificity of the <span class="gene">E1</span> protein in nucleus and thereby reducing the ability of <span class="gene">E1</span> to suppress expression of the <italic> <span class="gene">GmFT</span> </italic> genes [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. These findings strongly suggest that the molecular basis of the <span class="trait">photoperiod</span> insensitivity of Sakamotowase is due, at least in part, to a complete lack of <span class="gene">E1</span> function caused by a dysfunctional allele at the <italic> <span class="gene">E1</span> </italic> locus itself. The dysfunctional allele <italic> <span class="gene">e1-fs</span> </italic> therefore may provide, singly or together with other unknown genes, another mechanism in the control of <span class="trait">photoperiod</span> insensitivity under the presence of <italic> E4 </italic> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						To survey the diverse mechanisms underlying <span class="trait">photoperiod</span> insensitivity in soybean and to search for novel genetic factors involved in this process, we first used allele-specific DNA markers to catalog the genotypes of four maturity genes ( <italic> <span class="gene">E1</span> </italic> through <italic> E4 </italic> ) in photoperiod-insensitive cultivars and breeding lines introduced from various geographic regions. We also tested the association of these various genotypes with pre- and post-flowering <span class="trait">photoperiod</span> responses. Here we report that soybean has gained the trait of <span class="trait">photoperiod</span> insensitivity independently and repeatedly through diverse mechanisms. In addition, <italic> E3 </italic> and <italic> E4 </italic> , together with <italic> <span class="gene">E1</span> </italic> , play important roles in not only floral initiation but also post-flowering <span class="trait">photoperiod</span> responses, such as maturation and stem termination, in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						As in our previous studies [ <xref ref-type="bibr" rid="B6"> 6 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> ], we evaluated <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> according to the difference in the time to flowering (stage R1 [ <xref ref-type="bibr" rid="B28"> 28 </xref> ]) between artificially induced LD and natural daylength (ND) conditions. Harosoy is a photoperiod-sensitive, early-maturing indeterminate cultivar: it possesses a maturity genotype of <italic> e1-as </italic> / <italic> e2 </italic> / <italic> E3 </italic> / <italic> E4 </italic> , and a dominant <italic> <span class="gene">Dt1</span> </italic> gene [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. During the 2-year study period, this cultivar flowered at an average of 57 days after sowing (DAS) under ND conditions of Sapporo, Japan (43&#x000ba;06&#x000b4;N, 141&#x000ba;35&#x000b4;E) but did not produce any flower buds during LD until the end of the artificially induced condition (62 to 65 DAS). Similarly, Harosoy NILs for <italic> e3 </italic> (H- <italic> e3 </italic> ) or <italic> e4 </italic> (H- <italic> e4 </italic> ) did not form any flower buds until the end of the artificially induced LD condition, and they flowered slightly earlier during ND conditions than did Harosoy; the average flowering time was 46 DAS for H- <italic> e3 </italic> and 53 DAS for H- <italic> e4 </italic> . In contrast, the Harosoy NIL for both <italic> e3 </italic> and <italic> e4 </italic> (H- <italic> e3 </italic> / <italic> e4 </italic> ) flowered at 40 DAS under both ND and LD, and this line produced pods of 2 cm or longer (stages R4 to R5 [ <xref ref-type="bibr" rid="B28"> 28 </xref> ]) by the end of the artificially induced LD condition. All 53 photoperiod-insensitive accessions used in the current study flowered under LD conditions within 5 days after the date that they flowered under ND conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Genotyping of the four maturity genes by using allele-specific markers indicated that, despite their <span class="trait">photoperiod</span> insensitivity, 31 accessions had dominant alleles at either or both of the <italic> E3 </italic> and <italic> E4 </italic> loci. This result suggests that a novel allele or gene may be involved in the control of this trait. We then sequenced the accessions having at least one dominant <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , or <italic> E4 </italic> allele to examine whether these alleles were truly functional. No sequence variation was detected in the coding region of <italic> <span class="gene">E1</span> </italic> in the eight accessions of the <italic> <span class="gene">E1</span> </italic> / <italic> e3 </italic> / <italic> e4 </italic> genotype; all showed the same sequence as the published dominant <italic> <span class="gene">E1</span> </italic> allele [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. Sequence analysis of the <italic> E4 </italic> gene similarly revealed no deviations from the published sequence [ <xref ref-type="bibr" rid="B7"> 7 </xref> ] in the 15 accessions tested. However, we found two novel dysfunctional alleles for the <italic> E3 </italic> gene among the 18 accessions initially scored as having the <italic> e1 </italic> / <italic> E3 </italic> / <italic> E4 </italic> or <italic> e1 </italic> / <italic> E3 </italic> / <italic> e4 </italic> genotype (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> A and <xref ref-type="fig" rid="F1"> 1 </xref> B). One of these novel alleles was a nonsense mutation, in which a single-nucleotide substitution from C to T at position 3139 from the adenine of the first codon in exon 3 created a stop codon in place of a codon encoding glutamine. The other novel allele had a premature stop codon in exon 1 that was generated by frameshifting due to the insertion of T at position 1275 in exon 1. We designated these dysfunctional alleles <italic> e3-ns </italic> and <italic> e3 </italic> - <italic> fs </italic> , respectively, and renamed the conventional <italic> e3 </italic> allele as <italic> e3-tr </italic> (truncated), because it lacks the 3&#x02032; region of the gene, including exon 4 [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Determinate cultivars typically terminate stem growth shortly after flowering, whereas indeterminate ones continue stem growth until vegetative growth of the shoot apical meristem (SAM) ends [ <xref ref-type="bibr" rid="B33"> 33 </xref> ]. As expected, our 12 determinate accessions that had the <italic> dt1-bb </italic> or <italic> dt1-tb </italic> allele produced almost the same number of nodes on the main stem during ND and LD (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ; indicated by pink bars with small red or open bar). In contrast, the 41 indeterminate <italic> <span class="gene">Dt1</span> </italic> accessions produced more nodes during LD than during ND, and the differences in node number on the main stem between ND and LD varied with the multi-locus genotype (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ; indicated by blue bars with small red or open bar). The difference was less than 2.4 (average, 0.8) nodes among indeterminate accessions with the <italic> e3 </italic> / <italic> e4 </italic> genotype and was independent of the <italic> <span class="gene">E1</span> </italic> genotype (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ); that is, stem growth terminated shortly after flowering under both ND and LD conditions. In contrast, accessions with the <italic> E4 </italic> allele tended to produce more nodes during LD than during ND, and this response appeared to be dependent on the genotype ( <italic> e1 </italic> or <italic> e1-as </italic> ) at the <italic> <span class="gene">E1</span> </italic> locus (Figures&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> and <xref ref-type="fig" rid="F4"> 4 </xref> ). Accessions having the <italic> e1 </italic> ( <italic> e1-nl </italic> ) allele ( <italic> e1 </italic> / <italic> e3 </italic> / <italic> E4 </italic> ) terminated stem growth similarly between ND and LD; the difference in node number between ND and LD was, on average, 1.5 nodes across accessions and years (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> ). In contrast, accessions having the <italic> e1-as </italic> allele ( <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> ) produced at least 3.4 (average, 5.7) more nodes during LD than during ND. Therefore, post-flowering <span class="trait">photoperiod</span> growth in soybean, assessed as maturation and stem termination, varied among the accessions tested and with the maturity genotype.
					</p>
				
					<p>&nbsp;&nbsp;
						Han et al. [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] revealed that the post-flowering growth of a photoperiod-sensitive cultivar is influenced by the photoreaction that can be induced by exposure to R light and reversed by subsequent FR light exposure during the dark period. This pattern implies that phytochromes are involved in the <span class="trait">photoperiod</span> responses for post-flowering development in soybean. To confirm the different responses of post-flowering growth to <span class="trait">photoperiod</span> that we observed among the maturity genotypes in the field experiment, we analyzed pod development, stem termination, and the expression pattern of <italic> <span class="gene">Dt1</span> </italic> in stem tips under LD for Harosoy ( <italic> E3 </italic> / <italic> E4 </italic> / <italic> <span class="gene">Dt1</span> </italic> ) and its NILs for <italic> e3 </italic> , <italic> e4 </italic> , and <italic> dt1 </italic> . We induced flowering in these lines as described in the Methods section; SD treatment until 12 days after emergence (DAE) is sufficient to initiate and maintain flowering of Harosoy under non-inductive LD conditions [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. The soybean maturity genes, <italic> E3 </italic> and <italic> E4 </italic> , have been characterized in detail for the responses to long days with different R:FR ratios [ <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. We thus used two LD conditions with different light sources, R-light-enriched and FR-light-enriched, to discriminate the functions of <italic> E3 </italic> and <italic> E4 </italic> . Harosoy and its NILs for <italic> e3 </italic> , <italic> e4 </italic> , and <italic> dt1 </italic> ( <italic> dt1-bb </italic> ) flowered at almost the same time (11 to 13 days after the conversion to LD conditions; Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> C, D) under two lighting conditions. Stem growth terminated at the 6th to 8th node in the Harosoy NIL for <italic> dt1 </italic> (H- <italic> dt1 </italic> ); indeterminate lines produced more nodes than did H- <italic> dt1 </italic> , and the number of nodes produced until the end of experiment (30 days after flowering) varied with the maturity genotype (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> A). Compared with H- <italic> dt1 </italic> , Harosoy and H- <italic> e4 </italic> produced, on average, 7.5 and 6.4 more nodes under the R-light&#x02013;enriched condition and 5.9 to 6.5 nodes more under the FR-light&#x02013;enriched condition, respectively. H- <italic> e3 </italic> / <italic> e4 </italic> and H- <italic> e3 </italic> terminated stem growth earlier than did Harosoy and H- <italic> e4 </italic> under both conditions. In addition, stem growth terminated earlier in H- <italic> e3 </italic> / <italic> e4 </italic> than in H- <italic> e3 </italic> under the FR-light&#x02013;enriched condition, and H- <italic> e3 </italic> / <italic> e4 </italic> produced almost the same number of nodes as did H- <italic> dt1 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Different post-flowering <span class="trait">photoperiod</span> responses among Harosoy and its NILs for a determinate growth habit allele ( </bold> <bold> <italic> dt1 </italic> </bold> <bold> ) and the maturity alleles </bold> <bold> <italic> e3 </italic> </bold> <bold> and </bold> <bold> <italic> e4 </italic> </bold> <bold> in two independent experiments using different light sources, R-light&#x02013;enriched [blue] and FR-light&#x02013;enriched [red] conditions. </bold> In the R-light&#x02013;enriched condition, a combination of fluorescent and incandescent lamps with an R:FR ratio of 1.2 was used as the light source for 16 hours after dawn followed by lighting with the same light source for an additional 4 hours. The FR-light&#x02013;enriched condition involved the use of both fluorescent and incandescent lamps with an R:FR ratio of 1.2 for 16 hours after dawn followed by lighting with incandescent lamps only for an additional 4 hours. <bold> A </bold> ) Number of nodes (average&#x02009;&#x000b1;&#x02009;1 standard deviation) on the main stem at 30 days after flowering. <bold> B </bold> ) Number of pods (average&#x02009;&#x000b1;&#x02009;1 standard deviation) more than 3.0 cm long at 30 days after flowering. Transcripts levels of <italic> <span class="gene">Dt1</span> </italic> ( <italic> <span class="gene">GmTFL1b</span> </italic> ) at stem tips in different growth stages under ( <bold> C </bold> ) R-light&#x02013;enriched and ( <bold> D </bold> ) FR-light&#x02013;enriched conditions. Relative transcript levels (mean and standard error; <italic> n </italic> &#x02009;=&#x02009;3) were analyzed by quantitative RT-PCR and normalized to <italic> &#x003b2;-tubulin </italic> ( <italic> <span class="gene">TUB</span> </italic> ). FL indicates time of flowering.
					</p>
				
					<p>&nbsp;&nbsp;
						In the current study, we assessed the genotypes of 53 photoperiod-insensitive soybean cultivars at four maturity genes and a determinate growth-habit gene by using allele-specific DNA markers that distinguished each of the recessive alleles from the others. These accessions have been introduced from various high-latitude geographic regions. Our first goal was to determine whether there was any association between the genotypes at four major maturity loci ( <italic> <span class="gene">E1</span> </italic> through <italic> E4 </italic> ) and <span class="trait">photoperiod</span> <span class="trait">sensitivity.</span> Our second goal was to reveal any novel genetic factors that controlled <span class="trait">photoperiod</span> insensitivity in these diverse genetic resources. The photoperiod-insensitive cultivars and breeding lines we tested were classified into three genotypic groups according to the functions of the alleles at the respective loci: the <italic> e3 </italic> / <italic> e4 </italic> group; the group containing <italic> e1 </italic> ( <italic> e1 </italic> - <italic> nl </italic> or <italic> <span class="gene">e1-fs</span> </italic> ) and either <italic> E3 </italic> or <italic> E4 </italic> ; and the <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> group.
					</p>
				
					<p>&nbsp;&nbsp;
						One of the three genotypic groups consisted of accessions that possessed the double-recessive genotype ( <italic> e3 </italic> / <italic> e4 </italic> ) at the <italic> E3 </italic> and <italic> E4 </italic> loci. These accessions accounted for approximately 70% (36 of 53) of the photoperiod-insensitive accessions tested, suggesting that dysfunction of <span class="gene">PHYA</span> is the most common mechanism underlying <span class="trait">photoperiod</span> insensitivity in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Another genotypic group comprised the accessions having the dysfunctional <italic> e1 </italic> alleles ( <italic> e1-nl </italic> or <italic> <span class="gene">e1-fs</span> </italic> ) in combination with either a dominant <italic> E3 </italic> or <italic> E4 </italic> allele. This group contained Sakamotowase, which has the <italic> <span class="gene">e1-fs</span> </italic> allele that yields a nonfunctional <span class="gene">E1</span> protein, as has been revealed through transient expression assays [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. In addition, the major QTL for <span class="trait">photoperiod</span> insensitivity in this line is tightly linked to the <italic> <span class="gene">E1</span> </italic> locus [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. These observations suggest that the <span class="trait">photoperiod</span> insensitivity of Sakamotowase can be ascribed to the complete lack of <span class="gene">E1</span> function. Another dysfunctional allele, <italic> e1-nl </italic> , which lacks the entire <italic> <span class="gene">E1</span> </italic> gene [ <xref ref-type="bibr" rid="B13"> 13 </xref> ], occurred fairly frequently (11 of 53 lines) among the photoperiod-insensitive accessions we tested. The dysfunctional alleles <italic> <span class="gene">e1-fs</span> </italic> and <italic> e1-nl </italic> may have similar roles in abolishing or weakening the <span class="trait">photoperiod</span> responses regulated by the <italic> E3 </italic> or <italic> E4 </italic> allele. Additional research is needed to determine whether these dysfunctional alleles at the <italic> <span class="gene">E1</span> </italic> locus can singly lead to <span class="trait">photoperiod</span> insensitivity in the presence of the <italic> E3 </italic> or <italic> E4 </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						The remaining genotypic group consisted of the accessions with the allelic combination of <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> . The Harosoy NIL for <italic> e3 </italic> also has the same allelic combination. However, this NIL is sensitive to the LD conditions generated by using an FR-enriched light source (like that we generated by using incandescent lamps in the current study), although it did not respond to the R-enriched LD condition [ <xref ref-type="bibr" rid="B4"> 4 </xref> - <xref ref-type="bibr" rid="B6"> 6 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. Therefore, the allelic combination of <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> is not sufficient to confer <span class="trait">photoperiod</span> insensitivity. It is conceivable that a novel gene&#x02014;different from the dysfunctional alleles at the <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> loci&#x02014;may be involved in the control of the <span class="trait">photoperiod</span> insensitivity of accessions of this genotype under the FR-enriched LD condition. <italic> <span class="gene">GmphyA1</span> </italic> , a homoeolog of <italic> E4 </italic> , is a possible candidate for this photoperiod-insensitivity controller: <italic> <span class="gene">GmphyA1</span> </italic> has been suggested to function in a redundant manner to the <italic> E4 </italic> allele in the de-etiolation responses of hypocotyls and in flowering under the FR-enriched LD condition [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The dysfunctional allele at the <italic> E4 </italic> locus has adverse effects on plant morphogenesis, such as an impaired de-etiolation response under FR light [ <xref ref-type="bibr" rid="B7"> 7 </xref> ] and the production of longer internodes under LD conditions relative to the <italic> E4 </italic> allele [ <xref ref-type="bibr" rid="B43"> 43 </xref> ], making the plant susceptible to <span class="trait">lodging.</span> Accordingly, the use of the dominant <italic> E4 </italic> allele may contribute to the <span class="trait">lodging</span> tolerance of photoperiod-insensitive cultivars if another factor conditions the insensitivity. An appropriate combination of alleles should therefore be selected from among those representing diverse genetic mechanisms to adapt soybean cultivars to the conditions of the target environment.
					</p>
				
					<p>&nbsp;&nbsp;
						Multiple dysfunctional alleles have been detected for all of the <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> loci [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ], including two novel alleles at the <italic> E3 </italic> locus [this study]. Our previous studies indicated that five loss-of-function alleles at the <italic> E4 </italic> locus originated recently and independently in different soybean landraces from East Asia [ <xref ref-type="bibr" rid="B31"> 31 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> ]. These results suggest that <span class="trait">photoperiod</span> insensitivity in soybean has arisen redundantly through multiple combinations of independently generated alleles at the <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> loci.
					</p>
				
					<p>&nbsp;&nbsp;
						Another finding of the current study is that vegetative and reproductive growth characteristics after flowering, such as maturation and stem termination, were influenced by <span class="trait">photoperiod</span> responses regulated by the <italic> E3 </italic> and <italic> E4 </italic> loci. Our findings are therefore in good agreement with previous observations regarding an association between <span class="trait">photoperiod</span> sensing and post-flowering responses [ <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B18"> 18 </xref> ] and involvement of phytochrome(s) in post-flowering responses [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. We found that soybean accessions with different photoperiod-insensitive multi-locus genotypes responded differently to the <span class="trait">photoperiod</span> after flowering. Despite their <span class="trait">photoperiod</span> insensitivity in regard to time to flowering, accessions that had either the <italic> E3 </italic> or <italic> E4 </italic> allele matured at least 10 days later under LD than ND conditions (Figures&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> and <xref ref-type="fig" rid="F4"> 4 </xref> ). Furthermore, vegetative activity of the SAM during LD persisted through late growth stages in the indeterminate accessions with allelic combinations of <italic> e1 </italic> / <italic> E3 </italic> or <italic> e1 </italic> - <italic> as </italic> / <italic> E4 </italic> to produce more nodes than did those grown under ND condition (Figures&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> and <xref ref-type="fig" rid="F4"> 4 </xref> ). These behaviors are in sharp contrast to those of the accessions of the <italic> e3 </italic> / <italic> e4 </italic> group, which, independent of the type of allele or genotype at the <italic> <span class="gene">E1</span> </italic> locus, matured without notable delays during LD relative to ND and terminated stem growth similarly under these two conditions (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We have summarized our current results as a gene network involving three maturity genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> ), a determinate habit gene ( <italic> <span class="gene">Dt1</span> </italic> ), and two <italic> <span class="gene">GmFT</span> </italic> s ( <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> ), that regulates the pre-flowering and post-flowering <span class="trait">photoperiod</span> responses of soybean under LD (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> ). In photoperiod-sensitive plants having the <italic> <span class="gene">E1</span> </italic> or <italic> e1 </italic> - <italic> as </italic> allele and a dominant allele at either the <italic> E3 </italic> or <italic> E4 </italic> locus (or both) (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A), we propose that flowering is not induced under FR-enriched LD [ <xref ref-type="bibr" rid="B3"> 3 </xref> - <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ] because the <italic> <span class="gene">E1</span> </italic> and <italic> e1 </italic> - <italic> as </italic> alleles inhibit the gene expression of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. When those plants are exposed to a short period of SD, flowering is induced and persists even after the transfer to non-inductive LD, but subsequent seed maturation is delayed through the influence of an as yet unknown factor (Y), and vegetative activity at SAM is retained to produce more nodes due to <italic> <span class="gene">Dt1</span> </italic> expression. Both factor Y and <italic> <span class="gene">Dt1</span> </italic> are under the control of <span class="gene">PHYA</span> encoded at the <italic> E3 </italic> or <italic> E4 </italic> locus (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Proposed model of the genetic network comprising three maturity genes and one growth-habit gene affecting pre- and post-flowering <span class="trait">photoperiod</span> responses in soybean under LD. A </bold> ) Photoperiod-sensitive plants with <italic> <span class="gene">E1</span> </italic> or <italic> e1 </italic> - <italic> as </italic> and either <italic> E3 </italic> or <italic> E4 </italic> . PHYA-regulated <italic> <span class="gene">E1</span> </italic> expression inhibits the expression of soybean orthologs of <italic> FT </italic> ( <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> ). Once induced by brief exposure to SD, flowering persists after the transfer to LD, but seed maturation and stem termination are inhibited under the control of <span class="gene">PHYA</span> encoded by the <italic> E3 </italic> and <italic> E4 </italic> alleles through activation of an unknown factor (Y) and upregulation of <italic> <span class="gene">Dt1</span> </italic> expression, respectively. <bold> B </bold> ) Photoperiod-insensitive plants of the <italic> e3 </italic> / <italic> e4 </italic> group. Owing to the dysfunction of PHYA, flowering is induced under LD, which is followed by normal seed maturation and stem termination. <bold> C </bold> and <bold> D </bold> ) Photoperiod-insensitive plants of <bold> C </bold> ) the <italic> e1 </italic> / <italic> E3 </italic> / <italic> e4 </italic> or <italic> e1 </italic> / <italic> e3 </italic> / <italic> E4 </italic> group and D) the <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> group. In the <italic> e1 </italic> / <italic> E3 </italic> / <italic> e4 </italic> or <italic> e1 </italic> / <italic> e3 </italic> / <italic> E4 </italic> group, flowering is induced by the dysfunction of the <italic> <span class="gene">E1</span> </italic> gene (panel <bold> C </bold> ); whereas that in the <italic> e1-as </italic> / <italic> e3 </italic> / <italic> E4 </italic> group is induced through an unknown factor (X) (panel <bold> D </bold> ). Seed maturation of plants in these two groups is delayed similarly because of the presence of the <italic> E3 </italic> or <italic> E4 </italic> allele. However, stem growth after LD terminates earlier in the <italic> e1 </italic> / <italic> E3 </italic> / <italic> e4 </italic> or <italic> e1 </italic> / <italic> e3 </italic> / <italic> E4 </italic> group than in the <italic> e1 </italic> - <italic> as </italic> / <italic> e3 </italic> / <italic> E4 </italic> group: PHYA-mediated <italic> <span class="gene">Dt1</span> </italic> expression under LD likely preserves the vegetative activity of the SAM to produce more nodes in the <italic> e1 </italic> - <italic> as </italic> / <italic> e3 </italic> / <italic> E4 </italic> group. The designations <italic> e1 </italic> , <italic> e3 </italic> , and <italic> e4 </italic> refer to all dysfunctional alleles at these loci. Solid and dotted arrows indicate activation and lack of activation, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						These models prompt two questions regarding post-flowering growth in soybean. First, how does the <italic> <span class="gene">E1</span> </italic> gene influence stem growth after flowering? Apart from its effect on flowering and maturity, the role of the <italic> <span class="gene">E1</span> </italic> gene in morphogenesis has not been determined, although it indeed influences various morphologic traits, <span class="trait">yield,</span> and other traits such as tolerance to chilling temperature [ <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> ]. The <italic> <span class="gene">E1</span> </italic> and <italic> e1 </italic> - <italic> as </italic> alleles inhibit the expression of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> , which are under the control of <italic> <span class="gene">phyA</span> </italic> genes [ <xref ref-type="bibr" rid="B13"> 13 </xref> ], whereas the loss-of-function alleles <italic> e1 </italic> - <italic> fs </italic> and <italic> e1 </italic> - <italic> nl </italic> do not. In <italic> Arabidopsis </italic> , FT protein binds to FLOWERING LOCUS D (FD) and promotes the expression of <italic> APETALA1 </italic> and <italic> LEAFY </italic> , flower meristem genes [ <xref ref-type="bibr" rid="B47"> 47 </xref> - <xref ref-type="bibr" rid="B49"> 49 </xref> ]; reviewed in [ <xref ref-type="bibr" rid="B50"> 50 </xref> ], which, in turn, suppresses <italic> TFL1 </italic> transcription and terminates stem growth [ <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. Therefore, the different effects of the alleles at the <italic> <span class="gene">E1</span> </italic> locus on post-flowering stem growth might be ascribed not to a direct effect but rather to an indirect effect through suppression of <italic> <span class="gene">GmFT</span> </italic> expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Another question is whether flowering itself is a direct trigger for seed development in soybean. The findings we obtained in the current study indicate that PHYA-mediated <span class="trait">photoperiod</span> responses may regulate seed maturation directly or indirectly via an as yet unknown factor: only when two <italic> <span class="gene">phyA</span> </italic> genes were dysfunctional did seed maturation after flowering under LD proceed without any marked delay (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> ). Additional research therefore should explore key factors (such as Y in our model) in promoting seed development. Comparing expression profiles during early seed development between SD-grown plants and those whose reproductive growth has reverted to a vegetative pattern may advance our understanding of the molecular bases of seed development in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The present study revealed diverse genetic mechanisms underlying <span class="trait">photoperiod</span> insensitivity in soybean. At least three multi-locus genotypes comprising various allelic combinations at the <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> loci can confer pre-flowering <span class="trait">photoperiod</span> insensitivity. These genotypes responded differently to <span class="trait">photoperiod</span> during post-flowering reproductive development, indicating involvement of the <italic> <span class="gene">phyA</span> </italic> genes <italic> E3 </italic> and <italic> E4 </italic> . Our results further indicate that flowering itself may not necessarily be a direct trigger for seed development. <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> control the <span class="trait">photoperiod</span> responses for pre- and post-flowering development, which directly influences final seed <span class="trait">yield</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript levels of <italic> <span class="gene">Dt1</span> </italic> were determined by quantitative real-time PCR (qRT-PCR). The qRT-PCR mixture was prepared by mixing 1 &#x003bc;L of the cDNA synthesis reaction, 5 &#x003bc;L 1.2 &#x003bc;M primer premix, 10 &#x003bc;L SYBR Premix <italic> ExTaq </italic> Perfect Real Time (TaKaRa, Otsu, Japan), and water to <span class="trait">yield</span> a final volume of 20 &#x003bc;L. The analysis was done by using the CFX96 Real-Time System (BIO-RAD Laboratories Japan, Tokyo, Japan). The primers used were 5&#x02032;-AGGCACAACAGATGCCACAT-3&#x02032; and 5&#x02032;-GGCAAAACCAGCAGCTACTT-3&#x02032; for <italic> <span class="gene">GmTFL1b</span> </italic> ( <italic> <span class="gene">Dt1</span> </italic> ) and 5&#x02032;-GAGAAGAGTATCCGGATAGG-3&#x02032; and 5&#x02032;-GAGCTTGAGTGTTCGGAAAC-3&#x02032; for <italic> &#x003b2;-tubulin </italic> . The PCR cycling conditions were 95&#x000b0;C for 3 min followed by 40 cycles of 95&#x000b0;C for 10 sec, 58&#x000b0;C for 20 sec, 72&#x000b0;C for 20 sec, and 78&#x000b0;C for 2 sec. Fluorescence was quantified before and after incubation at 78&#x000b0;C to monitor the formation of primer dimers. A reaction mixture without reverse transcriptase was included as a control to confirm that no amplification resulted from genomic DNA contaminants in the RNA sample. In all PCR experiments, amplification of a single DNA species was confirmed by both melting curve analysis of qRT-PCR and gel electrophoresis of PCR products. The mRNA level of <italic> <span class="gene">GmTFL1b</span> </italic> was normalized to that of <italic> &#x003b2;-tubulin </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> <span class="gene">GmFT2a</span> </italic> Polymorphism and Maturity Diversity in Soybeans</h2>
			<h3>PMID:24155962</h3>
				
					<h3>PMCID:3796496</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiang </surname>  <given-names> Bingjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Yue </surname>  <given-names> Yanlei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Youfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Liming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Shi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hou </surname>  <given-names> Wensheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lam </surname>  <given-names> Hon-Ming </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 9 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a short-day crop of agricultural, ecological, and economic importance. The sensitive <span class="trait">photoperiod</span> responses significantly limit its breeding and adaptation. <italic> <span class="gene">GmFT2a</span> </italic> , a putative florigen gene with different transcription profiles in two cultivars (late-maturing Zigongdongdou and early-maturing Heihe 27) with different maturity profiles, is key to flowering and maturation. However, up to now, its role in the diverse patterns of maturation in soybeans has been poorly understood.
					</p>
				
					<p>&nbsp;&nbsp;
						Eighty varieties, including 19 wild accessions, covering 11 of all 13 maturity groups, were collected. They were planted in pots and maintained under different photoperiodicity conditions (SD, short day; LD, long day; and ND, natural day). The day to first flowering was recorded and the <span class="trait">sensitivity</span> to <span class="trait">photoperiod</span> was investigated. Polymorphisms in the <italic> <span class="gene">GmFT2a</span> </italic> coding sequence were explored by searching the known SNP database (NCBI dbSNP). The <italic> <span class="gene">GmFT2a</span> </italic> promoter regions were then cloned from these varieties and sequenced. Further polymorphism and association analyses were conducted.
					</p>
				
					<p>&nbsp;&nbsp;
						These varieties varied greatly in time to first flowering under ND and exhibited a consecutive distribution of <span class="trait">photoperiod</span> <span class="trait">sensitivity,</span> which suggested that there is rich diversity in flowering time. Furthermore, although <italic> <span class="gene">GmFT2a</span> </italic> had only one known synonymous SNP in the coding sequence, there were 17 haplotypes of the <italic> <span class="gene">GmFT2a</span> </italic> promoter region, HT06 of which was extremely abundant. Further association analysis found some SNPs that might be associated with day to first flowering and <span class="trait">photoperiod</span> <span class="trait">sensitivity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Photoperiod</span> responses and maturity patterns in soybean are quantitative traits controlled by multiple genes or loci. Up till now, nine maturity loci have been reported, including, E1-E8, and J [ <xref ref-type="bibr" rid="B5"> 5 </xref> - <xref ref-type="bibr" rid="B13"> 13 </xref> ]. These loci have been comprehensively reviewed by Xia et al. [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. They play different roles under different photoperiods with stronger effects under long-day and weaker effects under short-day conditions [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. Four of these loci were characterized at the molecular level, using map-based or candidate-based cloning. <italic> <span class="gene">E1</span> </italic> encodes a soybean-specific potential transcription factor <span class="gene">Glyma06g23040</span> [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]; <italic> E2 </italic> encodes a GIGANTEA homologue, <span class="gene">GmGIa</span> [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]; and <italic> E3 </italic> and <italic> E4 </italic> encode the phytochromes <span class="gene">GmPhyA3</span> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] and <span class="gene">GmPhyA2</span> [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. However, the genes corresponding to the five remaining loci have not been identified and the exact functions of the four identified loci remain unclear.
					</p>
				
					<p>&nbsp;&nbsp;
						The key flowering gene <italic> Flowering Locus T </italic> ( <italic> FT </italic> ) in <italic> Arabidopsis thaliana </italic> encodes a putative florigen that is an integrating factor of the flowering regulation network [ <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> ]. Soybean has at least 10 <italic> FT </italic> -like genes [ <xref ref-type="bibr" rid="B22"> 22 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> ], among which, <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> could functionally promote flowering in <italic> A. thaliana </italic> [ <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B24"> 24 </xref> ]. Furthermore, it was observed that <italic> <span class="gene">GmFT2a</span> </italic> overexpression could induce early flowering in transgenic soybean [ <xref ref-type="bibr" rid="B24"> 24 </xref> ]. Therefore, it is surprising that these two important flowering genes have not been considered as candidate genes for the five unidentified maturity loci. <italic> <span class="gene">GmFT2a</span> </italic> is regulated by <span class="trait">photoperiod</span> differentially in two cultivars exhibiting different <span class="trait">photoperiod</span> sensitivities (photoperiod-sensitive Zigongdongdou and photoperiod-insensitive Heihe 27) [ <xref ref-type="bibr" rid="B24"> 24 </xref> ], suggesting that the function of <italic> <span class="gene">GmFT2a</span> </italic> might be related to the regulation of maturity. While it is speculated that the expression of <italic> <span class="gene">GmFT2a</span> </italic> may be developmentally regulated [ <xref ref-type="bibr" rid="B24"> 24 </xref> ], the promoter region of <italic> <span class="gene">GmFT2a</span> </italic> has not been thoroughly analyzed.
					</p>
				
					<p>&nbsp;&nbsp;
						Further association analysis with GLM did not show a significant association between <italic> <span class="gene">GmFT2a</span> </italic> polymorphism and maturity diversity, which is consistent with the idea that <italic> <span class="gene">GmFT2a</span> </italic> is under strong selection. At the level of p&#x0003c;0.01, SNP S17 showed a relationship with the day to first flowering under SD and ND while SNPs S162 and S1849 showed such a relationship only under SD. The PLACE program ( <ext-link ext-link-type="uri" xlink:href="http://www.dna.affrc.go.jp/place/"> <underline> http://www.dna.affrc.go.jp/PLACE/ </underline> </ext-link> ) identified a CIACADIANLELHC element ( <named-content content-type="gene"> CAANNNNATC </named-content> , dark letter means SNP location) near SNP S17 (G /A). Near SNP S1849 (T/C), however, the program found an IBOXCORENT element ( <named-content content-type="gene"> GATAAGR </named-content> ) [ <xref ref-type="bibr" rid="B34"> 34 </xref> , <xref ref-type="bibr" rid="B35"> 35 </xref> ]. Whereas the CIACADIANLELHC element is associated with circadian expression, the IBOXCORENT element is associated with light-responsive regulation; both are related to <span class="trait">photoperiod</span> responses in soybeans. However, more evidence is needed to elucidate how these SNPs function to regulate <span class="trait">photoperiod</span> reaction. Considering that <italic> <span class="gene">GmFT2a</span> </italic> is under high selection pressure, polymorphism in this gene does not appear to be responsible for maturity diversity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> <span class="gene">GmFT4,</span> </italic> a Homolog of <italic> FLOWERING LOCUS T </italic> , Is Positively Regulated by <italic> <span class="gene">E1</span> </italic> and Functions as a Flowering Repressor in Soybean</h2>
			<h3>PMID:24586488</h3>
				
					<h3>PMCID:3929636</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhai </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> L&#x000fc; </surname>  <given-names> Shixiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Shuang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xingzheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Xiaohui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Zhengjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The major maturity gene <italic> <span class="gene">E1</span> </italic> has the most prominent effect on flowering time and <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> of soybean, but the pathway mediated by <italic> <span class="gene">E1</span> </italic> is largely unknown. Here, we found the expression of <italic> <span class="gene">GmFT4</span> </italic> , a homolog of <italic> Flowering Locus T, </italic> was strongly up-regulated in transgenic soybean overexpressing <italic> <span class="gene">E1</span> </italic> , whereas expression of flowering activators, <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> , was suppressed. <italic> <span class="gene">GmFT4</span> </italic> expression was strongly up-regulated by long days exhibiting a diurnal rhythm, but down-regulated by short days. Notably, the basal expression level of <italic> <span class="gene">GmFT4</span> </italic> was elevated when transferred to continous light, whereas repressed when transferred to continuous dark. <italic> <span class="gene">GmFT4</span> </italic> was primarily expressed in fully expanded leaves. Transcript abundance of <italic> <span class="gene">GmFT4</span> </italic> was significantly correlated with that of functional <italic> E1, </italic> as well as flowering time phenotype in different cultivars. Overexpression of <italic> <span class="gene">GmFT4</span> </italic> delayed the flowering time in transgenic <italic> Arabidopsis </italic> . Taken together, we propose that <italic> <span class="gene">GmFT4</span> </italic> acts downstream of <italic> <span class="gene">E1</span> </italic> and functions as a flowering repressor, and the balance of two antagonistic factors ( <italic> <span class="gene">GmFT4</span> </italic> vs <italic> <span class="gene">GmFT2a</span> </italic> / <italic> 5a </italic> ) determines the flowering time of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						It is generally accepted that the clock-controlled <italic> CO </italic> - <italic> FT </italic> external coincidence mechanism is conserved in higher plants. However, each plant species has evolved its own unique mechanisms to induce flowering under optimal conditions. In rice, <italic> Hd1 </italic> , an ortholog of <italic> CO </italic> , promotes flowering under SD conditions, while another rice <italic> CO </italic> -like gene, <italic> Ghd7 </italic> , acts as a floral repressor under LD conditions and suppresses the transcription of <italic> Ehd1 </italic> , a floral activator of multiple flowering signals <xref rid="pone.0089030-Komiya2" ref-type="bibr"> [20] </xref> , <xref rid="pone.0089030-Doi1" ref-type="bibr"> [31] </xref> &#x02013; <xref rid="pone.0089030-Xue1" ref-type="bibr"> [32] </xref> . In barley, <italic> HvCO9 </italic> , an ortholog of <italic> CO </italic> , acts as a negative regulator of flowering under non-inductive SD conditions. In Pharbitis, <italic> PnFT </italic> mRNA abundance was not related to <italic> PnCO </italic> expression <xref rid="pone.0089030-Hayama1" ref-type="bibr"> [28] </xref> . It was reported <italic> Lotus japonicas </italic> (a model legume) might lack the upstream positive regulator CO <xref rid="pone.0089030-Yamashino1" ref-type="bibr"> [33] </xref> . In pea ( <italic> Pisum sativum </italic> ), <italic> COLa </italic> is the most homologous gene, but is not the ortholog to <italic> AtCO </italic> in terms of the function. The diurnal expression rhythm of <italic> COLa </italic> under long days is more similar to <italic> Arabidopsis COL1 </italic> and <italic> COL2 </italic> <xref rid="pone.0089030-Hecht1" ref-type="bibr"> [34] </xref> , which have little effect on flowering time <xref rid="pone.0089030-Ledger1" ref-type="bibr"> [35] </xref> , and the expression of <italic> COLa </italic> is not obviously altered in <italic> late1 </italic> mutants ( <italic> LATE1 </italic> is an ortholog of <italic> Arabidopsis GIGANTEA </italic> ) <xref rid="pone.0089030-Hecht1" ref-type="bibr"> [34] </xref> , while in <italic> Arabidopsis AtCO </italic> expression is constantly dampened in <italic> gi-2 </italic> mutants <xref rid="pone.0089030-Sawa1" ref-type="bibr"> [36] </xref> . In soybean, the maturity gene <italic> <span class="gene">E1</span> </italic> , which has the most prominent effect on flowering time and <span class="trait">photoperiod</span> <span class="trait">sensitivity,</span> is a legume-specific gene <xref rid="pone.0089030-Xia1" ref-type="bibr"> [37] </xref> . Hence, it is speculated that there might be significant differences in the mechanisms of flowering time regulation between legume and the model species <italic> Arabidopsis/ </italic> rice.
					</p>
				
					<p>&nbsp;&nbsp;
						To date, eight flowering time or maturity loci, designated <italic> <span class="gene">E1</span> </italic> to <italic> E8 </italic> <xref rid="pone.0089030-Bernard1" ref-type="bibr"> [38] </xref> &#x02013; <xref rid="pone.0089030-Saindon1" ref-type="bibr"> [45] </xref> , along with the <italic> J </italic> locus for the long juvenile period trait, with which soybean flowers late even under short days (SDs) <xref rid="pone.0089030-Ray1" ref-type="bibr"> [46] </xref> , have been characterized genetically. Of these, <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> are involved in <span class="trait">photoperiod</span> responses <xref rid="pone.0089030-Buzzell1" ref-type="bibr"> [40] </xref> &#x02013; <xref rid="pone.0089030-Buzzell2" ref-type="bibr"> [41] </xref> , <xref rid="pone.0089030-McBlain1" ref-type="bibr"> [44] </xref> , <xref rid="pone.0089030-Abe2" ref-type="bibr"> [47] </xref> &#x02013; <xref rid="pone.0089030-Cober4" ref-type="bibr"> [49] </xref> . <italic> E3 </italic> and <italic> E4 </italic> encode homologs of the photoreceptor phytochrome A (PHYA) <xref rid="pone.0089030-Franklin1" ref-type="bibr"> [50] </xref> &#x02013; <xref rid="pone.0089030-Watanabe1" ref-type="bibr"> [52] </xref> . <italic> E2 </italic> encodes a homolog of GIGANTEA (GI) <xref rid="pone.0089030-Watanabe2" ref-type="bibr"> [53] </xref> , a key regulator of photoperiodic flowering in <italic> Arabidopsis </italic> that functions upstream of <italic> CO </italic> and <italic> FT </italic> <xref rid="pone.0089030-Sawa1" ref-type="bibr"> [36] </xref> , <xref rid="pone.0089030-David1" ref-type="bibr"> [54] </xref> . <italic> <span class="gene">E1</span> </italic> encodes a legume-specific protein, which contains a putative bipartite nuclear localization signal, a region distantly related to DNA-binding B3 domain and a helix&#x02013;turn&#x02013;helix structure, and might function as a transcription factor <xref rid="pone.0089030-Xia1" ref-type="bibr"> [37] </xref> . In addition, two of the <italic> FT </italic> homologs, <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> are responsible for inducing flowering under short-day conditions <xref rid="pone.0089030-Sun1" ref-type="bibr"> [22] </xref> &#x02013; <xref rid="pone.0089030-Lazakis1" ref-type="bibr"> [23] </xref> . <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> are regulated by PHYA: Functional <italic> E3 </italic> and <italic> E4 </italic> genotypes suppressed the expression of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> under long-day conditions and delayed flowering, whereas double-recessive <italic> e3e4 </italic> genotypes induced <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> expression and promoted early flowering regardless of day length <xref rid="pone.0089030-Sun1" ref-type="bibr"> [22] </xref> &#x02013; <xref rid="pone.0089030-Lazakis1" ref-type="bibr"> [23] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In order to assess whether the expression of <italic> <span class="gene">GmFT4</span> </italic> is involved in the <span class="trait">photoperiod</span> pathway, transcript levels of <italic> <span class="gene">GmFT4</span> </italic> in different soybean cultivars under both LDs and SDs were investigated ( <xref ref-type="fig" rid="pone-0089030-g003"> Figure 3A, B </xref> ). Generally, <italic> <span class="gene">GmFT4</span> </italic> was highly induced under LDs, and repressed under SDs in most soybean cultivars, especially in photoperiod-sensitive cultivars, e.g. HN112, HN89, HX3, Harosoy- <italic> <span class="gene">E1</span> </italic> , Harosoy- <italic> e1, </italic> Jack, Jilin 35 and Sidou 11. Whereas, in photoperiod-insensitive cultivars, such as Kariyutaka, Heinong 48, Mufeng 7 and Sakamotowase, <italic> <span class="gene">GmFT4</span> </italic> expression level was very low under both SDs and LDs. Relatively higher <italic> <span class="gene">GmFT4</span> </italic> expression levels under SDs than under LDs were observed in Kariyutaka and Sakamotowase, however, it may not be meaningful to compare them since both were at very low levels. <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , and <italic> E4 </italic> were reported to be involved in <span class="trait">photoperiod</span> responses. Kariyutaka has the <italic> e3e4 </italic> genotype (double recessive <italic> E3 </italic> , <italic> E4 </italic> ) and showed a suppressed <italic> <span class="gene">E1</span> </italic> expression. Both Heinong 48 and Mufeng 7 carry the <italic> e3 </italic> genotype ( <italic> E3 </italic> recessive). Sakamotowase carries the <italic> <span class="gene">e1-fs</span> </italic> genotype (a frame shift mutation of <italic> <span class="gene">E1</span> </italic> ). Therefore, we might be able to hypothesize <italic> E3 </italic> , <italic> E4 </italic> and <italic> <span class="gene">E1</span> </italic> regulate the <span class="trait">photoperiod</span> response of soybean via <italic> <span class="gene">GmFT4</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Each soybean cultivar is agronomically adapted to a narrow range of latitude for their maximal <span class="trait">yield</span> due to <span class="trait">photoperiod</span> <span class="trait">sensitivity,</span> thus limiting widely distribution of elite soybean cultivars. Among the major maturity genes or QTLs that have been reported so far, the <italic> <span class="gene">E1</span> </italic> gene has the most prominent effect on flowering time and <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> in soybean <xref rid="pone.0089030-Cober3" ref-type="bibr"> [48] </xref> , <xref rid="pone.0089030-Watanabe3" ref-type="bibr"> [60] </xref> &#x02013; <xref rid="pone.0089030-Yamanaka1" ref-type="bibr"> [62] </xref> , suggesting <italic> <span class="gene">E1</span> </italic> is a key regulator of flowering in soybean. Phylogenetic analysis on protein sequence indicated <italic> <span class="gene">E1</span> </italic> is legume-specific <xref rid="pone.0089030-Xia1" ref-type="bibr"> [37] </xref> , implying that the major photoperiodic pathway in soybean might be different from that in <italic> Arabidopsis </italic> and rice. Negative correlation between <italic> <span class="gene">E1</span> </italic> and <italic> <span class="gene">GmFT2a/GmFT5a</span> </italic> expression was observed in our previous study <xref rid="pone.0089030-Xia1" ref-type="bibr"> [37] </xref> . Here, we found that a <italic> FT </italic> ortholog, <italic> <span class="gene">GmFT4</span> </italic> , was positively associated with <italic> <span class="gene">E1</span> </italic> in <italic> <span class="gene">E1</span> </italic> overexpressing transgenic soybean ( <xref ref-type="fig" rid="pone-0089030-g001"> Figure 1 </xref> ), <italic> <span class="gene">E1</span> </italic> NILs and different soybean cultivars ( <xref ref-type="fig" rid="pone-0089030-g003"> Figure 3B </xref> ). Interestingly, unlike most <italic> FT </italic> -like genes, <italic> <span class="gene">GmFT4</span> </italic> was characterized as a flowering repressor ( <xref ref-type="fig" rid="pone-0089030-g006"> Figure 6 </xref> ). These results indicated soybean has evolved a different strategy to regulate flowering time, and the <italic> <span class="gene">E1</span> </italic> - <italic> <span class="gene">GmFT4</span> </italic> pathway is valuable for understanding the molecular mechanisms of flowering time regulation in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In plants, the regulation of <italic> FTs </italic> in different species is highly diversified. Most <italic> FT </italic> -like genes are flowering activators and their induction occurs only in daylength that induce flowering. Not surprisingly, high <italic> <span class="gene">GmFT4</span> </italic> expression level in non-inductive LDs is consistent with its flowering repressing function. In <italic> Arabidopsis </italic> and rice, response to <span class="trait">photoperiod</span> is mediated by transcriptional regulation of <italic> FTs </italic> through an intersection between clock regulation and daylength. The expression of <italic> FT </italic> requires the activation of the clock output gene <italic> CO </italic> in the presence of light in <italic> Arabidopsis </italic> <xref rid="pone.0089030-Yanovsky1" ref-type="bibr"> [10] </xref> . <italic> FT </italic> transcription is activated by <italic> CO </italic> under LDs but not under SDs, because under LDs, <italic> CO </italic> mRNA expression coincides with exposure of plants to light leading to stabilization of CO protein <xref rid="pone.0089030-SurezLpez1" ref-type="bibr"> [9] </xref> &#x02013; <xref rid="pone.0089030-Yanovsky1" ref-type="bibr"> [10] </xref> . Here, we found that induction of <italic> <span class="gene">GmFT4</span> </italic> occurred in non-inductive daylength, and the diurnal oscillation expression pattern could be retained to some extent when transferred to subsequent 24 h continuous light or continuous dark, indicating that <italic> <span class="gene">GmFT4</span> </italic> was partly impacted by the circadian clock, and moreover that the induction of <italic> <span class="gene">GmFT4</span> </italic> required the exposure of plants to light, which indicated, just like other <italic> FT </italic> -like genes, transcriptional regulation of <italic> <span class="gene">GmFT4</span> </italic> also through an intersection between clock regulation and daylength.
					</p>
				
					<p>&nbsp;&nbsp;
						Most soybean cultivars have a short-day requirement for floral induction, so under long days, flowering and maturing are delayed and differ greatly among different cultivars. Here, we found the transcriptional abundance of <italic> <span class="gene">GmFT4</span> </italic> is significantly correlated with the flowering time of different soybean cultivars under LDs, indicating that <italic> <span class="gene">GmFT4</span> </italic> might be related to the flowering time regulation under LDs. Soybean cultivars grown at high latitudes are often <span class="trait">photoperiod</span> insensitive, because soybean plants planted in spring are required to flower under LDs during early summer and complete seed production in the limited frost-free season. <span class="trait">Photoperiod</span> insensitive cultivars grown at high latitude, such as Kariyutaka, Heinong 48, MuFeng 7 and Sakamotowase displayed low levels of <italic> <span class="gene">GmFT4</span> </italic> expression. Even under SDs, transcriptional abundance of <italic> <span class="gene">GmFT4</span> </italic> was also significantly correlated with the flowering time of different soybean cultivars. At low latitudes, soybean cultivars with the classic response to <span class="trait">photoperiod</span> flower early resulting in short plants and low grain <span class="trait">yield</span> <xref rid="pone.0089030-CarpentieriPpolo1" ref-type="bibr"> [64] </xref> , so soybean cultivars bred at low latitudes often needs the long juvenile period trait that featured as delayed flowering under SDs <xref rid="pone.0089030-Cober5" ref-type="bibr"> [65] </xref> &#x02013; <xref rid="pone.0089030-Tomkins1" ref-type="bibr"> [67] </xref> . In this study, cultivar HX3, known for the Brazilian long juvenile period trait exhibited a delayed flowering phenotype under SDs. Generally, cultivars with the long juvenile period trait need a SD regime of 8 h/16 h (light/dark) to promote flowering. Here, we found cultivar HX3 also showed a relatively high <italic> <span class="gene">GmFT4</span> </italic> expression even under SDs, indicating that <italic> <span class="gene">GmFT4</span> </italic> might be an important determinant for flowering time regardless of day-length conditions. We assume that <italic> <span class="gene">GmFT4</span> </italic> might contribute greatly to the wide adaptability of soybean to wide range of latitudes. Hence, we proposed a model for the flowering time regulation in soybean ( <xref ref-type="fig" rid="pone-0089030-g007"> Figure 7 </xref> ). In this model, all flowering promoters <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a,</span> </italic> and repressor <italic> <span class="gene">GmFT4</span> </italic> , function downstream of <italic> <span class="gene">E1</span> </italic> , and the balance between the antagonistic <italic> FTs </italic> ( <italic> <span class="gene">GmFT4</span> </italic> vs <italic> <span class="gene">GmFT2a/GmFT5a</span> </italic> ) determines soybean flowering time. Under SDs or in cultivars carrying nonfunctional <italic> <span class="gene">E1</span> </italic> alleles or devoid of <italic> <span class="gene">E1</span> </italic> expression, the expression of the flowering repressor <italic> <span class="gene">GmFT4</span> </italic> is very low except for cultivars carrying long juvenile trait, and the expression of flowering promoters <italic> <span class="gene">GmFT2a/5a</span> </italic> are high, leading to an initiation of the flowering process. Under LDs, the expression of flowering repressor <italic> <span class="gene">GmFT4</span> </italic> is high, while flowering promoters <italic> <span class="gene">GmFT2a/5a</span> </italic> are repressed, so the flowering process is delayed.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Allelic Combinations of Soybean Maturity Loci <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> and <italic> E4 </italic> Result in Diversity of Maturity and Adaptation to Different Latitudes</h2>
			<h3>PMID:25162675</h3>
				
					<h3>PMCID:4146597</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Jiang </surname>  <given-names> Bingjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Nan </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Gao </surname>  <given-names> Youfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Lili </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yue </surname>  <given-names> Yanlei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Sijia </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Liming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Shi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jialin </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Xiaohui </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hou </surname>  <given-names> Wensheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although these four known loci <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> and <italic> E4 </italic> provide an important key to learn the mechanism of flowering and maturity, we mainly got knowledge based on few cultivars but not on a population level. Population-level knowledge will provide another different view of these four loci&#x02019;s role on maturity and adaptation. Therefore, in this study we selected a set of soybean cultivars which cover 12 maturity groups from MG000 to MGVIII and MGX plus some cultivars with wide range of latitude from N 18&#x000b0; to N 53&#x000b0;. These cultivars were subjected to different <span class="trait">photoperiod</span> treatments. Traits of beginning bloom (R1), physiological maturity (R7) and full maturity (R8) were investigated <xref rid="pone.0106042-Fehr1" ref-type="bibr"> [20] </xref> . Maturity loci <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> and <italic> E4 </italic> were genotyped in the population. Further association analysis was done. The results showed that allelic combinations of these four <italic> E </italic> genes significantly determine the ecological-economical adaption of cultivars although they have different impacts on maturity. In addition, the genetic mechanisms underlying <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> and adaptation in wild soybean seemed unique from those in cultivated soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The tested soybean cultivars were selected from North America, China and Russia. Some of them covered from MG000 to MGVIII and MGX, almost all of the total 13 MG <xref rid="pone.0106042-Hartwig1" ref-type="bibr"> [2] </xref> and the others were collected from N 18&#x000b0; to N 53&#x000b0; to cover the wide range of latitude, which represents the main soybean producing area in China. In addition, some wild soybean accessions were also included. Thus, the population of lines used here should exhibit the diversity of maturity not only in phenotype but also in genotype. The field experiment under ND provided strong evidence. Some cultivars failed to flower (R1), some could not reach pod yellowing (R7) and some could not reach full maturity (R8) ( <xref ref-type="table" rid="pone-0106042-t003"> Table 3 </xref> ). Even for those cultivars that flowered and matured, the days to first flowering, physiological maturity and full maturity varied significantly ( <xref ref-type="table" rid="pone-0106042-t003"> Table 3 </xref> ). What&#x02019;s more, in the experiment of <span class="trait">photoperiod</span> treatments, the population also showed diversity of <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> as expected. The days to first flowering varied between different <span class="trait">photoperiod</span> treatments and between different cultivars. The <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> also varied from 10% to 80% ( <xref ref-type="table" rid="pone-0106042-t004"> Table 4 </xref> ). Six cultivars were classified as <span class="trait">photoperiod</span> insensitive: CAAE001 (PI548594, MG000, <italic> e1/e2/e3/e4 </italic> ), CAAE002 (PI567787, MG000, <italic> e1/e2/e3/e4 </italic> ), CAAE005 (PI592523, MG00, <italic> e1-as/e2/e3/E4 </italic> ), CAAE007 (PI596541, MG0, <italic> e1-as/e2/e3/E4 </italic> ), CAAE042 (Heihe 3, <italic> e1-as/e2/e3/E4 </italic> ) and CAAE091 (Mohe 1, <italic> e1/e2/e3/e4 </italic> ). These results were consistent with previous report that soybean <span class="trait">photoperiod</span> insensitivity was at least conditioned by three genetic mechanisms according to allelic combinations of <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> and <italic> E4 </italic> : <italic> e3/e4 </italic> ; <italic> e1/e3 or e1/e4 </italic> and <italic> e1-as/e3/E4 </italic> . In the genetic mechanism of <italic> e1-as/e3/E4, </italic> novel unidentified gene/genes participated in <span class="trait">photoperiod</span> insensitivity <xref rid="pone.0106042-Xu1" ref-type="bibr"> [26] </xref> , <xref rid="pone.0106042-Tsubokura1" ref-type="bibr"> [27] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean, as a short-day crop, has many cultivars with diversified maturity structure. <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> and <italic> E3 </italic> are involved with different impacts. In the field experiment under ND, the recessive allele <italic> e1 </italic> , <italic> el-as </italic> and <italic> e2 </italic> significantly narrowed the variation of VE-R1 more than that of VE-R7 and R1&#x02013;R7 ( <xref ref-type="fig" rid="pone-0106042-g003"> Figure 3 </xref> ), suggesting that <italic> <span class="gene">E1</span> </italic> and <italic> E2 </italic> genes have significant impact on pre-flowering development other than post-flowering responses. However, the loci <italic> E3 </italic> and <italic> E4 </italic> might function not only in pre-flowering development but also in post-flowering development, indicated by the narrow variation in VE-R1, VE-R7 and R1&#x02013;R7 of cultivars with <italic> e3 </italic> and e4 alleles ( <xref ref-type="fig" rid="pone-0106042-g003"> Figure 3 </xref> ). This result is consistent with Xu et al <xref rid="pone.0106042-Xu1" ref-type="bibr"> [26] </xref> . that <italic> E3 </italic> and <italic> E4 </italic> respond not only to pre-flowering but also to post-flowering by increasing pod filling duration, number of nods and pod numbers by up-regulating the expression of growth habit gene <italic> <span class="gene">Dt1</span> </italic> . This result implies the significance of <italic> E3 </italic> and <italic> E4 </italic> loci for molecular genetic breeding to increase soybean productivity. Moreover, the outliers suggested that the disfunction of <italic> E3 </italic> and <italic> E4 </italic> might be interrupted by other genes. Similarly, the loci <italic> <span class="gene">E1</span> </italic> and <italic> E3 </italic> are related with <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> ( <xref ref-type="fig" rid="pone-0106042-g004"> Figure 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						For the genotype of <italic> E1/E2/E3/E4 </italic> , the <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> was mostly above 70%. Compared with genotypes <italic> E1/E2/E3/E4 </italic> , <italic> e1-as/E2/E3/E4 </italic> , <italic> e1-as/e2/E3/E4 </italic> and <italic> e1-as/e2/e3/E4 </italic> ( <xref ref-type="fig" rid="pone-0106042-g005"> Figure 5 </xref> ), the <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> decreased with the numbers of recessive alleles. While comparing <italic> E1/e2/E3/E4 </italic> and <italic> e1-as/E2/E3/E4 </italic> with <italic> E1E2E3E4 </italic> , it was suggested that <italic> <span class="gene">E1</span> </italic> plays a more important role than <italic> E2 </italic> because <italic> e1 </italic> decreased the <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> more significantly and narrowed its range. These results further proved that soybean <span class="trait">photoperiod</span> insensitivity was involved by four maturity loci <italic> E1, E3, E4 </italic> and <italic> E7 </italic> while <italic> E2 </italic> locus was not involved <xref rid="pone.0106042-Cober3" ref-type="bibr"> [28] </xref> , <xref rid="pone.0106042-Cober4" ref-type="bibr"> [29] </xref> . In these tested cultivars of China, <italic> E1/e2/E3/E4 </italic> is much more abundant. It ranged from N 18&#x000b0; -N 42&#x000b0; while <italic> E1/E2/E3/E4 </italic> was distributed south of N 39&#x000b0; except for the four wild soybeans (CAAE087, Heiheyesheng; CAAE088, Bayanyesheng; CAAE089, Baiyangdianyesheng; and CAAE090, Guangxiyesheng). For <italic> e1 </italic> or <italic> el-as </italic> alleles, the associated cultivars where mostly located above N 45&#x000b0; except for CAAE053 (Tiefeng 31, N 35&#x000b0;&#x02013;40&#x000b0;). Thus, to improve the ecological adaptability of cultivars, the <italic> <span class="gene">E1</span> </italic> gene must function less because of its most strong impact on delaying maturity while other <italic> E </italic> genes might become important in adaptation. Unlike cultivated soybean, two wild soybean accessions, CAAE087 (Heiheyesheng) and CAAE088 (Bayanyesheng) adapted north of N 46&#x000b0; in China where cultivars generally had less <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> where genotyped as <italic> E1/E2/E3/E4 </italic> . It is greatly important for wild soybean to adapt and survive during season alteration. The genetic mechanisms underlying <span class="trait">photoperiod</span> insensitivity and adaptation in wild soybean were therefore unique from those in cultivated soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> E </italic> genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> and <italic> E4 </italic> ) have different roles in maturity and <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> and their allelic combinations determine maturity group and adaptation to different latitude.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantifying the Effects of <span class="trait">Photoperiod,</span> Temperature and Daily Irradiance on Flowering Time of Soybean Isolines</h2>
			<h3>PMID:27135515</h3>
				
					<h3>PMCID:4844283</h3>
					<h4>期刊:Plants</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cober </surname>  <given-names> Elroy R. </given-names>  </name>  <xref rid="c1-plants-03-00476" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Curtis </surname>  <given-names> Daniel F. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stewart </surname>  <given-names> Douglas W. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Morrison </surname>  <given-names> Malcolm J. </given-names>  </name>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Eight genes, which influence time from planting to first flower, have been identified in soybean [ <italic> Glycine max </italic> (L.) Merr.] to date: <span class="gene">E1</span> and E2 [ <xref rid="B1-plants-03-00476" ref-type="bibr"> 1 </xref> ], E3 [ <xref rid="B2-plants-03-00476" ref-type="bibr"> 2 </xref> ], E4 [ <xref rid="B3-plants-03-00476" ref-type="bibr"> 3 </xref> , <xref rid="B4-plants-03-00476" ref-type="bibr"> 4 </xref> ], E5 [ <xref rid="B5-plants-03-00476" ref-type="bibr"> 5 </xref> ], E6 [ <xref rid="B6-plants-03-00476" ref-type="bibr"> 6 </xref> ], E7 [ <xref rid="B7-plants-03-00476" ref-type="bibr"> 7 </xref> ] and E8 [ <xref rid="B8-plants-03-00476" ref-type="bibr"> 8 </xref> ]. Two of these, E3 [ <xref rid="B9-plants-03-00476" ref-type="bibr"> 9 </xref> ] and E4 [ <xref rid="B10-plants-03-00476" ref-type="bibr"> 10 </xref> ] have been identified as phytochrome A genes while E2 has been identified as a GIGANTEA homolog [ <xref rid="B11-plants-03-00476" ref-type="bibr"> 11 </xref> ] and <span class="gene">E1</span> as an inhibitor of FT [ <xref rid="B12-plants-03-00476" ref-type="bibr"> 12 </xref> ]. Alleles at these loci, in conjunction with <span class="trait">photoperiod</span> [ <xref rid="B13-plants-03-00476" ref-type="bibr"> 13 </xref> ] and temperature [ <xref rid="B14-plants-03-00476" ref-type="bibr"> 14 </xref> ], regulate the timing of flowering and maturity of soybean lines. Soybean is a facultative short day plant, so under non-inductive long days (&#x0003e;14 h) and high temperatures (25 to 30 &#x000b0;C) dominant E alleles delay flowering, with the exception of the E6 locus [ <xref rid="B14-plants-03-00476" ref-type="bibr"> 14 </xref> ]. Lower temperatures reduced the delaying effect of E alleles under non-inductive photoperiods [ <xref rid="B14-plants-03-00476" ref-type="bibr"> 14 </xref> ]. Isolines with dominant E alleles at two or three loci flowered earlier under low temperature (18 C) and non-inductive photoperiods compared to higher temperature (28 &#x000b0;C) with non-inductive photoperiods, but the time to flowering in both instances was still greater than the time to flower under inductive short photoperiods [ <xref rid="B14-plants-03-00476" ref-type="bibr"> 14 </xref> ]. This result underscored the importance of the E genes in the adaptation of soybean lines to specific climates and highlighted the requirement for a thorough understanding of their function in controlling flowering.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean quantitative trait locus that promotes flowering under long days is identified as <italic> FT5a </italic> , a <italic> FLOWERING LOCUS T </italic> ortholog</h2>
			<h3>PMID:27422993</h3>
				
					<h3>PMCID:5014162</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Takeshima </surname>  <given-names> Ryoma </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hayashi </surname>  <given-names> Takafumi </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Jianghui </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Chen </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Meilan </given-names>  </name>  <xref ref-type="aff" rid="AF0002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi </surname>  <given-names> Naoya </given-names>  </name>  <xref ref-type="aff" rid="AF0003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sayama </surname>  <given-names> Takashi </given-names>  </name>  <xref ref-type="aff" rid="AF0004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ishimoto </surname>  <given-names> Masao </given-names>  </name>  <xref ref-type="aff" rid="AF0004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Lingping </given-names>  </name>  <xref ref-type="aff" rid="AF0002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Xinyi </given-names>  </name>  <xref ref-type="aff" rid="AF0002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="AF0002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Zhixi </given-names>  </name>  <xref ref-type="aff" rid="AF0005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamada </surname>  <given-names> Tetsuya </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="AF0002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="c1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abe </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="AF0001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="c1"> * </xref>  </contrib>  <aff id="AF0001">  <sup> 1 </sup>  <institution> Research Faculty of Agriculture, Hokkaido University </institution> ,  <addr-line> Sapporo, Hokkaido 060-8589 </addr-line> ,  <country> Japan </country>  </aff>  <aff id="AF0002">  <sup> 2 </sup>  <institution> The Key Laboratory of Soybean Molecular Design Breeding, Northeast Institute of Geography and Agroecology </institution> ,  <addr-line> Chinese Academy of Sciences, Harbin 150081 </addr-line> ,  <country> China </country>  </aff>  <aff id="AF0003">  <sup> 3 </sup>  <institution> Hokkaido Research Organization Tokachi Agricultural Experiment Station </institution> ,  <addr-line> Memuro, Hokkaido 082-0081 </addr-line> ,  <country> Japan </country>  </aff>  <aff id="AF0004">  <sup> 4 </sup>  <institution> National Institute of Agrobiological Sciences </institution> ,  <addr-line> Kannondai, Ibaraki 305-8602 </addr-line> ,  <country> Japan </country>  </aff>  <aff id="AF0005">  <sup> 5 </sup>  <institution> State Key Laboratory of Plant Cell and Chromosome Engineering, Institute of Genetics and Developmental Biology </institution> ,  <addr-line> Chinese Academy of Sciences, Beijing 1001014 </addr-line> ,  <country> China </country>  </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Segregating populations of two soybean crosses, Toyoharuka (TH) &#x000d7; 1532-1 (cross A), and a near-isogenic line (NIL) of Harosoy for <italic> e3 </italic> (H- <italic> e3 </italic> , PI547716) &#x000d7; Jiagedaqi-02 (J02) (cross B), were used in this study. TH and 1532-1 have the same maturity genotype at the <italic> E2 </italic> , <italic> E3 </italic> , and <italic> E9 </italic> loci ( <italic> e2 </italic> / <italic> E3 </italic> / <italic> e9 </italic> ), but differ at the <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> loci: TH has the <italic> e1-nl </italic> allele, which lacks the entire <italic> <span class="gene">E1</span> </italic> genomic region, and the <italic> e4 </italic> allele, whereas 1532-1 has functional <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> alleles ( <xref rid="CIT0077" ref-type="bibr"> Yamaguchi <italic> et al. </italic> , 2014 </xref> ). J02 is a breeding line developed at the Agricultural Experimental Station at Jiagedaqi, Heilongjiang Province, China. It has the same maturity genotype as H- <italic> e3 </italic> at all of the five maturity loci ( <italic> e1-as </italic> / <italic> e2 </italic> / <italic> e3 </italic> / <italic> E4 </italic> / <italic> E9 </italic> ), but unlike H- <italic> e3 </italic> it exhibits a reduced <span class="trait">photoperiod</span> <span class="trait">sensitivity</span> of flowering under incandescent LD conditions (ILD; <xref rid="CIT0051" ref-type="bibr"> Saindon <italic> et al. </italic> , 1989 </xref> ), where natural daylength is extended to 20h by using incandescent lamps with low R:FR ratios ( <xref rid="CIT0075" ref-type="bibr"> Xu <italic> et al. </italic> , 2013 </xref> ). Recombinant inbred lines (RILs) for each of the two crosses were developed with the single seed descent method. The RIL population consisted of 99 lines homozygous for <italic> e1-nl </italic> and 62 lines homozygous for <italic> <span class="gene">E1</span> </italic> in cross A and 79 lines in cross B. NILs for the early-flowering ( <italic> ef </italic> ) and late-flowering ( <italic> lf </italic> ) alleles at a QTL were developed from the progenies of F <sub> 6 </sub> or F <sub> 7 </sub> plants used for fine-mapping in cross A (NILs #46 and #64) and from the progeny of an F <sub> 6 </sub> plant in cross B (NIL #8).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>