<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Plant weight</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Plant weight</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Development and application of a novel genome-wide SNP array reveals domestication history in soybean</h2>
			<h3>PMID:26856884</h3>
				
					<h3>PMCID:4746597</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jiao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Huairen </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Ying </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Hao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> National Center for Soybean Improvement, National Key Laboratory of Crop Genetics and Germplasm Enhancement, Nanjing Agricultural University </institution> , Nanjing, 210095,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 01 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Increased seed <span class="trait">weight</span> is a major trait associated with the domestication of soybeans <xref ref-type="bibr" rid="b6"> 6 </xref> . Among the candidate domestication genes, 16 genes were found to be associated with seed <span class="trait">weight.</span> We then compared the location of these genes with mapped seed <span class="trait">weight</span> related QTLs reported in Soybase ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://soybase.org/"> http://soybase.org/ </ext-link> ). The results showed that six of these genes had overlapping regions with mapped seed <span class="trait">weight</span> related QTLs ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S5 </xref> ). For example, <italic> Glyma06g 12800 </italic> , a homologous gene of <italic> GS5, </italic> which encodes a putative serine carboxypeptidase positively regulating grain size, genetically maps to two QTLs related to seed <span class="trait">weight</span> (sw 29&#x02013;3 and sw 30&#x02013;1) in rice <xref ref-type="bibr" rid="b24"> 24 </xref> . Furthermore, three non-synonymous substitutions with significant differences of allele frequency between wild and cultivated soybeans were detected among these six genes. For instance, <italic> <span class="gene">Glyma01g41990</span> </italic> is an ortholog of rice GIF1 (Grain Incomplete Filling 1), an enzyme involved in the carbohydrate metabolism pathway that controls seed <span class="trait">weight</span> during grain development <xref ref-type="bibr" rid="b25"> 25 </xref> . We detected one non-synonymous substitution (T to C) in this gene which resulted in the V326A amino acid mutation. In wild soybeans only 20.48% of the individuals maintained the amino acid V, while in cultivated soybeans almost all of the individuals (96.18%) maintained the amino acid V. This mutation, located in the glycosyl hydrolases family 32 N-terminal domain of GIF1, may affect gene function.
					</p>
				
					<p>&nbsp;&nbsp;
						The 100-seed <span class="trait">weight</span> of the wild soybeans in four environments and cultivated soybeans in nine environments were assayed and showed that there were broad variations in seed <span class="trait">weight</span> for either cultivated soybeans or wild soybeans ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S5 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Table S6 and S7 </xref> ). We found that heritability is high for both cultivated and wild soybeans. Most of the signals identified by GWAS were in regions previously reported to contain seed-weight QTLs ( <xref ref-type="table" rid="t2"> Table 2 </xref> ). Thirteen significant SNPs were detected in cultivated soybeans. One of these 13 SNPs, located on chromosome 11, was observed in two environments and showed a significant marker-trait association, with a <italic> P </italic> -value as low as 2.479&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;6 </sup> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S6b </xref> &#x00026; d; <xref ref-type="table" rid="t2"> Table 2 </xref> ). There were 17 genes detected in the 130&#x02009;kb flanking region (LD decay distance of cultivated soybeans) of this SNP. Among these genes, <italic> <span class="gene">Glyma11g03360</span> </italic> is homologous to <italic> Os04g33740 </italic> ( <italic> GIF1 </italic> ) that regulates grain filling and size in rice <xref ref-type="bibr" rid="b25"> 25 </xref> , and <italic> <span class="gene">Glyma11g03430</span> </italic> is homologous to <italic> Os11g12740 </italic> ( <italic> sp1 </italic> ), a gene that participates in panicle elongation and grain size <xref ref-type="bibr" rid="b26"> 26 </xref> . Thus, these two genes in soybean could be considered as candidate genes for seed <span class="trait">weight.</span> The other 12 SNPs were detected in the E5 environment under the threshold of <italic> P </italic> &#x02009;&#x0003c;&#x02009;4.82&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;6 </sup> , and were organized as a cluster on chromosome 20 (26.9&#x02013;27.8&#x02009;Mb) ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ). Under the threshold of <italic> P&#x02009; </italic> &#x0003c;&#x02009;1&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;4 </sup> , all these 12 SNPs were repetitively detected in the best linear unbiased prediction (BLUP) data set. Strong and extensive linkage disequilibrium was observed in this region, an indication of human selection. Furthermore, considering the LD decay distance, we extended this region from 26.8&#x02013;27.9&#x02009;Mb. QTLs associated with seed <span class="trait">weight</span> have been reported multiple times in this region ( <xref ref-type="table" rid="t2"> Table 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, we performed GWAS in wild soybeans and identified 12 SNPs significantly related to seed <span class="trait">weight,</span> which are located on chromosomes 11, 13, and 18 ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S7 </xref> ; <xref ref-type="table" rid="t2"> Table 2 </xref> ). These SNPs are anchored to nine regions ( <xref ref-type="table" rid="t2"> Table 2 </xref> ). Within these regions, three genes, <italic> <span class="gene">Glyma11g05760</span> </italic> , <italic> <span class="gene">Glyma18g05240</span> </italic> , and <italic> <span class="gene">Glyma18g43500</span> </italic> , are considered to be candidate genes for seed <span class="trait">weight,</span> since their homologous genes ( <italic> SWN, IKU2 </italic> in <italic> Arabidopsis, </italic> and <italic> LRK1 </italic> in rice, respectively) participate in the regulation of seed size or grain <span class="trait">weight</span> <xref ref-type="bibr" rid="b27"> 27 </xref> <xref ref-type="bibr" rid="b28"> 28 </xref> <xref ref-type="bibr" rid="b29"> 29 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						A subset of 211 accessions from 262 cultivated soybeans was used for association analysis, and these accessions were grown from 2011&#x02013;2013 in Nanjing, Nantong, and Yangzhou, China. Each location within each certain year is considered to be a single environment; thus, this study contained nine environments, which were designated as <span class="gene">E1</span> to E9 (E1: Jiangpu_2011, E2: Jiangpu_2012, E3: Jiangpu_2013, E4: Nantong_2011, E5: Nantong_2012 and E6: Nantong_2013, E7: Yangzhou_2011, E8: Yangzhou_2012, E9: Yangzhou_2013). All accessions were arranged in a completely randomized block design with three replicates in each test location. For each accession, the replication of 80 plants was arranged in each of a four square meter matrix plot in the field (four rows, ten hills for each row with 20&#x02009;cm space, and thinning to two plants left for each hill after two weeks). After maturity, four individuals for each accession were randomly screened for 100-seed <span class="trait">weight</span> measurement with four technical repeats.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Suppression of extracellular invertase inhibitor gene expression improves seed <span class="trait">weight</span> in soybean ( <italic> Glycine max </italic> )</h2>
			<h3>PMID:28204559</h3>
				
					<h3>PMCID:5441900</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Xiaofei </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The roles of <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF1</span> as invertase inhibitors were examined <italic> in vitro </italic> in soybean ( <italic> Glycine max </italic> ), and post-translational elevation of extracellular invertase activity by silencing <italic> <span class="gene">GmCIF1</span> </italic> expression was found to increase seed <span class="trait">weight.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Cell wall invertase (CWI) and vacuolar invertase (VI) play multiple functions in <span class="trait">plant</span> growth. As well as depending on transcriptional and post-transcriptional regulation, there is growing evidence that CWI and VI are also subject to post-translational control by small inhibitory proteins. Despite the significance of this, genes encoding inhibitors, their molecular and biochemical properties, and their potential roles in regulating seed production have not been well documented in soybean ( <italic> Glycine max </italic> ). In this study, two invertase inhibitor isoforms, <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2,</span> were characterized to possess inhibitory activities <italic> in vitro </italic> via heterologous expression. Transcript analyses showed that they were predominantly expressed in developing seeds and in response to ABA. In accordance with this, surveys of primary targets showed subcellular localizations to the apoplast in tobacco epidermis after expressing YFP-fusion constructs. Investigations using RNAi transgenic plants demonstrated marked elevations of CWI activities and improvements in seed <span class="trait">weight</span> in conjunction with higher accumulations of hexoses, starch, and protein in mature seeds. Further co-expression analyses of <italic> <span class="gene">GmCIF1</span> </italic> with several putative CWI genes corroborated the notion that <span class="gene">GmCIF1</span> modulation of CWI that affects seed <span class="trait">weight</span> is mainly contingent on post-translational mechanisms. Overall, the results suggest that post-translational elevation of CWI by silencing of <italic> <span class="gene">GmCIF1</span> </italic> expression orchestrates the process of seed maturation through fine-tuning sucrose metabolism and sink strength.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Glycine max </italic> (soybean), a legume crop, is an important source of edible proteins and oils that are widely used throughout the world ( <xref rid="CIT0032" ref-type="bibr"> Song <italic> et al. </italic> , 2013 </xref> ). There has been ongoing interest in improving soybean performance, although significantly enhancing seed production remains a major challenge for modern agriculture. In spite of the advances made in commonly studied plants, little is known about invertase inhibitory proteins and their physiological significance in soybean. To our knowledge, there have been no reports on the targets of invertase inhibitors, on the post-translational regulation of CWI, or on the phenotypes that arise from modification of the expression of inhibitors in soybean plants. In a bid to rectify this situation, two putative soybean invertase inhibitors, <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2,</span> were functionally characterized by the use of recombinant proteins. Their subcellular targets were examined by ectopic expression with fusions of fluorescent protein. The functional role of <span class="gene">GmCIF1</span> was further explored in RNAi transgenic plants via the silencing of its expression. Based on transcript and metabolic analysis, our work demonstrates the physiological roles of <span class="gene">GmCIF1</span> in controlling seed maturation through specifically depressing CWI activities. The substantially improved seed <span class="trait">weight</span> reported here provides a promising strategy for future development, and in the long term, it may facilitate increases in crop production via manipulation of the post-translational modulation of CWI.
					</p>
				
					<p>&nbsp;&nbsp;
						Systematic BLAST searches within the Genbank database retrieved a large number of <italic> C/VIF </italic> -like genes in the soybean genome. Two of them, <italic> <span class="gene">GmCIF1</span> </italic> (BT090960) and <italic> <span class="gene">GmC/VIF2</span> </italic> (BT091584), encoding putative invertase inhibitors were identified with high homologies (&#x0003e;40% amino acid identity) to <italic> AtCIF1 </italic> from Arabidopsis ( <xref rid="CIT0030" ref-type="bibr"> Link <italic> et al. </italic> , 2004 </xref> ; <xref rid="CIT0052" ref-type="bibr"> Su <italic> et al. </italic> , 2016 </xref> ). A further survey of protein sequences by performing a BLAST search against the SoyBase database ( <uri xlink:href="http://www.soybase.org/"> http://www.soybase.org/ </uri> ) confirmed that <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> were encoded by <span class="gene">Glyma.17G036300</span> and <span class="gene">Glyma.17G036400,</span> respectively. These two isoforms have a similar genomic structure and protein size, sharing 54% amino acid identity ( <xref ref-type="fig" rid="F1"> Fig. 1A </xref> , <xref ref-type="fig" rid="F1"> B </xref> ). After removal of the N-terminal signal peptides, the deduced mature proteins of <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> are comprised of 155 and 157 amino acid residues, respectively. The predicted molecular <span class="trait">weight</span> is 16.77 kDa for <span class="gene">GmCIF1,</span> a pI of 6.06, and 16.72 kDa for <span class="gene">GmC/VIF2,</span> a pI of 7.59. Protein sequence alignment with other known C/VIFs ( <xref ref-type="fig" rid="F1"> Fig.1A </xref> ) showed that <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> contain typical hallmarks, namely four conserved cysteine (C) residues that form two disulfide bridges to sustain protein stability, and a conserved motif (PKF) that has been demonstrated by crystallographic analysis to interact for inhibitory functions with the invertase substrate cleft ( <xref rid="CIT0020" ref-type="bibr"> Hothorn <italic> et al. </italic> , 2010 </xref> ). Phylogenetic analyses showed that <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> are clustered into a clade together with an experimentally determined AtCIF1 from Arabidopsis, whereas their locations were far away from the C/VIFs in tomato, potato, and tobacco (see Supplementary Fig. S1).
					</p>
				
					<p>&nbsp;&nbsp;
						To verify their inhibitory functions <italic> in vitro </italic> , the coding sequences of <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> were amplified with cDNA prepared from flowers. Sequences encoding mature proteins (omitting the signal peptides) were cloned into the pETG-20A vector (see Supplementary Fig. S2A). Heterologous expression in the <italic> E. coli </italic> strain Rosetta-gami&#x02122; (DE3) yielded N-terminal TrxA fusion proteins for <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2.</span> Under native conditions, the fusion proteins were released by the cleavage of the TEV protease. The purified <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> were recovered by Ni-TED affinity chromatography to remove the His-tagged TrxA and TEV protease (Supplementary Fig. S2B). Processes during the protein purification suggested that recombinant <span class="gene">GmCIF1</span> and <span class="gene">GmC/VIF2</span> were soluble. The sizes of both the finally purified proteins were close to the predicted molecular <span class="trait">weight.</span> Further analysis of their mobility on SDS-PAGE under non-reducing conditions revealed increased shifts, reflecting the presence of active intramolecular disulfide bridges in the recombinant proteins (data not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						Given that the silencing of the <italic> <span class="gene">GmCIF1</span> </italic> transcript increased CWI activities, we then explored whether there were any potential phenotypic changes in <span class="trait">plant</span> growth and development. Interestingly, the morphological analysis showed significant differences in seed <span class="trait">weight</span> between RNAi plants and the wild-type plants. RNAi plants produced larger seeds than the wild-type control ( <xref ref-type="fig" rid="F6"> Fig. 6A </xref> , <xref ref-type="fig" rid="F6"> B </xref> ; see Supplementary Fig. S5A). In the same backgrounds, analyses of the seed production per individual <span class="trait">plant</span> and the total across all plants showed a statistical improvement ( <xref ref-type="fig" rid="F6"> Fig. 6C </xref> , <xref ref-type="fig" rid="F6"> D </xref> ). However, even if very large increases in CWI activities were detected at the beginning of seed germination (24 h) of RNAi plants, the germination rate was not statistically affected (Supplementary Fig. S5B). RNAi plants appeared to grow as normally as the wild-type plants and no significant differences were recorded for the <span class="trait">plant</span> height in RNAi plants (data not shown). ABA perturbation caused marked depressions of activities of both CWI and VI in leaves, whereas CWI activities of RNAi plants remained at higher levels than the wild-type control plants (Supplementary Fig. S4D, E). However, RNAi plants and wild-type plants showed no differences in ABA-mediated leaf ageing (Supplementary Fig. S4B, C). Interestingly, in comparison with the wild-type control at 125 d, RNAi plants showed a prolonged <span class="trait">plant</span> greening period of approximately 4&#x02013;6 d, which was statistically significant (Supplementary Fig. S5).
					</p>
				
					<p>&nbsp;&nbsp;
						A large amount of evidence has indicated that CWIs are responsible for sugar partitioning in the apoplast of sinks and that they are involved in the regulation of seed and fruit development in a variety of species ( <xref rid="CIT0045" ref-type="bibr"> Ruan <italic> et al. </italic> , 2010 </xref> ; <xref rid="CIT0044" ref-type="bibr"> Ruan, 2014 </xref> ). Improved seed production has been postulated as a result of elevated extracellular invertase activities in Arabidopsis ( <xref rid="CIT0018" ref-type="bibr"> Heyer <italic> et al. </italic> 2004 </xref> ). Investigation of a rice extracellular invertase gene, <italic> OsGIF </italic> , revealed that it functions as a determinant of grain filling ( <xref rid="CIT0056" ref-type="bibr"> Wang <italic> et al. </italic> , 2008 </xref> ). Constant expression of CWIs in maize substantially improved the grain <span class="trait">weight</span> and starch content ( <xref rid="CIT0029" ref-type="bibr"> Li <italic> et al. </italic> , 2013 </xref> ). Similarly, it is not surprising that elevation of CWI activities through silencing or knock-out of CIFs has been shown to bring about improvements in seed <span class="trait">weight</span> and production ( <xref rid="CIT0021" ref-type="bibr"> Jin <italic> et al. </italic> , 2009 </xref> ; <xref rid="CIT0052" ref-type="bibr"> Su <italic> et al. </italic> , 2016 </xref> ). These reports suggest an association between functional CWI and sink strength. In agreement with this, the specific suppression of <italic> <span class="gene">GmCIF1</span> </italic> expression significantly improved seed <span class="trait">weight</span> and production concurrent with markedly elevated CWI activities ( <xref ref-type="fig" rid="F5"> Figs 5C </xref> and <xref ref-type="fig" rid="F6"> 6A </xref> ). CWI activities were observed to be higher than VI activities at the mid- and late stages of seed maturity, indicating the fundamental role of CWI, but not VI, in controlling seed development in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						CWI typically catalyzes hydrolysis of sucrose and releases hexoses into the apoplast; hence, the hexoses released are spatially and temporally associated with CWI activities ( <xref rid="CIT0040" ref-type="bibr"> Roitsch, 1999 </xref> ; <xref rid="CIT0060" ref-type="bibr"> Weschke <italic> et al. </italic> , 2003 </xref> ). Starch serves as the major storage form of carbohydrate in developing seeds ( <xref rid="CIT0029" ref-type="bibr"> Li <italic> et al. </italic> , 2013 </xref> ). After seed germination, the reserved starch degrades into soluble sugars, which fuel cellular respiration to produce energy for seedling growth ( <xref rid="CIT0065" ref-type="bibr"> Zeeman <italic> et al. </italic> , 2010 </xref> ). Soluble sugars were quantified in mature seeds of the RNAi plants; however, only glucose and fructose were positively affected, reflecting a correlation with the increased CWI activities. Starch content significantly increased in RNAi backgrounds ( <xref ref-type="table" rid="T1"> Table 1 </xref> ). These results suggested that silencing of <span class="gene">GmCIF1</span> may liberate extra CWI activities, which most likely enhance sucrose hydrolysis and starch storage in sinks. Seed protein content statistically increased, whereas the oil content was barely affected in several lines of the RNAi plants ( <xref ref-type="table" rid="T1"> Table 1 </xref> ). These data are in accordance with the fact that seed protein is negatively associated with seed oil content in soybean ( <xref rid="CIT0037" ref-type="bibr"> Proulx and Naeve, 2009 </xref> ; <xref rid="CIT0012" ref-type="bibr"> Eskandari <italic> et al. </italic> , 2013 </xref> ). The improved seed <span class="trait">weight</span> and protein content result from the over-accumulation of hexoses, which have been shown to serve not only as nutrients but also as signals to induce storage-associated cell differentiation in seed development ( <xref rid="CIT0061" ref-type="bibr"> Wobus and Weber, 1999 </xref> ). Moreover, sugar transport from source to sink is one of the major determinants of <span class="trait">plant</span> growth ( <xref rid="CIT0027" ref-type="bibr"> Lemoine <italic> et al. </italic> , 2013 </xref> ), and therefore we hypothesize that the modified sink strength may initially benefit from the improved efficiency of sugar production in source leaves. This notion was supported by the finding of increased CWI activities in sources leaves of RNAi plants (Supplementary Fig. S4D). The increased CWI activities correlate with the increase of photosynthetic capacity in the sources ( <xref rid="CIT0007" ref-type="bibr"> Dai <italic> et al. </italic> , 1999 </xref> ; <xref rid="CIT0021" ref-type="bibr"> Jin <italic> et al. </italic> , 2009 </xref> ). Taken together, the elevation of CWI by the silencing of its inhibitor may facilitate sucrose unloading and release more hexoses, which are required for starch and protein synthesis during seed maturation.
					</p>
				
					<p>&nbsp;&nbsp;
						The search for the primary target of <span class="gene">GmCIF1</span> in combination with enzyme assays and RNAi analysis revealed that <span class="gene">GmCIF1</span> specifically inhibits CWI. We provide evidence that <span class="gene">GmCIF1</span> plays a fundamental role in the post-translational control of extracellular invertase, which results in an impact on seed development and sucrose hydrolysis at a tissue-specific level. Silencing of <span class="gene">GmCIF1</span> significantly elevated CWI activities in soybean plants. Our results are instrumental in helping to decipher how invertase inhibitors regulate CWI during seed maturation. Moreover, unravelling the mechanisms underlying the post-translational modulation of extracellular invertase in soybean has implications for <span class="trait">plant</span> biotechnology and may provide new cues for improving crop performance through manipulating the interaction of CWI and its inhibitory proteins. The potential roles of <span class="gene">GmCIF1</span> in <span class="trait">resistance</span> to biotic and abiotic stresses as well as its impact on nutrient use efficiency will be the subject of future investigations.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dietary Patterns during Pregnancy Are Associated with Risk of Gestational Diabetes Mellitus</h2>
			<h3>PMID:26569302</h3>
				
					<h3>PMCID:4663600</h3>
					<h4>期刊:Nutrients</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shin </surname>  <given-names> Dayeon </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Kyung Won </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Won O. </given-names>  </name>  <xref rid="c1-nutrients-07-05472" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this cross-sectional study, three dietary patterns during pregnancy were identified with the choice of response variables including prepregnancy BMI, ratio of poly- and monounsaturated fatty acids to saturated fatty acids and dietary fiber: &#x0201c;high refined grains, fats, oils and fruit juice&#x0201d; pattern, &#x0201c;high nuts, seeds, fat and soybean; low milk and cheese&#x0201d; pattern, and &#x0201c;high added sugar and organ meats; low fruits, vegetables and seafood&#x0201d; pattern. Despite small differences, all three dietary patterns were associated with increased risks for GDM. Among three dietary patterns, the strongest relationship to the GDM risk was found for &#x0201c;high added sugar and organ meats; low fruits, vegetables and seafood&#x0201d; pattern. The positive association of the &#x0201c;high added sugar and organ meats; low fruits, vegetables and seafood&#x0201d; pattern with GDM, was largely explained by the high consumption of added sugars and low consumption of fruits and vegetables. Sugar-sweetened beverages are one of the leading sources of added sugars in the American diet [ <xref rid="B36-nutrients-07-05472" ref-type="bibr"> 36 </xref> ]. In the Nurses&#x02019; Health Study II, intake of sugar-sweetened coke before pregnancy was positively associated with the risk of GDM [ <xref rid="B37-nutrients-07-05472" ref-type="bibr"> 37 </xref> ]. Compared to women who consumed one serving/month, those women who consumed &#x02265;5 servings/week of sugar sweetened coke had a 22% greater risk for GDM (relative risk (RR) 1.22; 95% <span class="gene">CI</span> 1.01&#x02013;1.47). Epidemiologic studies demonstrate that high consumption of sugar-sweetened beverages was associated with increased risk for type 2 diabetes among general adult populations [ <xref rid="B38-nutrients-07-05472" ref-type="bibr"> 38 </xref> , <xref rid="B39-nutrients-07-05472" ref-type="bibr"> 39 </xref> , <xref rid="B40-nutrients-07-05472" ref-type="bibr"> 40 </xref> ]. High sugar intake is also associated with high energy intake and hence obesity which is associated with risk of GDM. The high levels of rapidly absorbable carbohydrates in the form of added sugars of sugar sweetened beverages [ <xref rid="B40-nutrients-07-05472" ref-type="bibr"> 40 </xref> ] may increase the levels of fasting blood glucose levels and insulin <span class="trait">resistance.</span> In our study, low intake of fruits and vegetables pattern was associated with an increased risk for GDM. Although the biological mechanisms for the inverse associations of fruits and vegetable intake and GDM risk are not clear, Bazzano <italic> et al. </italic> [ <xref rid="B41-nutrients-07-05472" ref-type="bibr"> 41 </xref> ] explained that fruit and green leafy vegetables may contribute to a decreased incidence of type 2 diabetes through their low energy density, low glycemic load and high fiber. This mechanism may partially explain the association of low intake of fruits and vegetables in relation to decreased risk for GDM. Our findings are further supported by the findings from the Nurses&#x02019; Health Study II [ <xref rid="B8-nutrients-07-05472" ref-type="bibr"> 8 </xref> ]. Women in the lowest quintile of the prudent pattern characterized by a high intake of fruit, vegetables, and green leafy vegetables (lowest adherence) were associated with increased risks for GDM compared to those women in the highest quintile (highest adherence) (RR 1.39; 95% <span class="gene">CI</span> 1.08&#x02013;1.80). In the same prospective cohort of Nurses&#x02019; Health Study II, intake of whole fruits and green leafy green vegetables was inversely associated with incidence of type 2 diabetes in the middle-aged U.S. women [ <xref rid="B42-nutrients-07-05472" ref-type="bibr"> 42 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effectiveness of food supplements in increasing fat-free tissue accretion in children with moderate acute malnutrition: A randomised 2 &#x000d7; 2 &#x000d7; 3 factorial trial in Burkina Faso</h2>
			<h3>PMID:28892496</h3>
				
					<h3>PMCID:5593178</h3>
					<h4>期刊:PLoS Medicine</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-1987-781X </contrib-id>  <name>  <surname> Fabiansen </surname>  <given-names> Christian </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yam&#x000e9;ogo </surname>  <given-names> Charles W. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Iuel-Brockdorf </surname>  <given-names> Ann-Sophie </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-7558-7032 </contrib-id>  <name>  <surname> Cichon </surname>  <given-names> Bernardette </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-9490-9586 </contrib-id>  <name>  <surname> Rytter </surname>  <given-names> Maren J. H. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kurpad </surname>  <given-names> Anura </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0003-0411-8025 </contrib-id>  <name>  <surname> Wells </surname>  <given-names> Jonathan C. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-5095-0624 </contrib-id>  <name>  <surname> Ritz </surname>  <given-names> Christian </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0003-2311-2593 </contrib-id>  <name>  <surname> Ashorn </surname>  <given-names> Per </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Filteau </surname>  <given-names> Suzanne </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff007">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0001-9390-8541 </contrib-id>  <name>  <surname> Briend </surname>  <given-names> Andr&#x000e9; </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0001-9515-4333 </contrib-id>  <name>  <surname> Shepherd </surname>  <given-names> Susan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff008">  <sup> 8 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-0890-3832 </contrib-id>  <name>  <surname> Christensen </surname>  <given-names> Vibeke B. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff009">  <sup> 9 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0003-0449-0839 </contrib-id>  <name>  <surname> Michaelsen </surname>  <given-names> Kim F. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0002-2848-2940 </contrib-id>  <name>  <surname> Friis </surname>  <given-names> Henrik </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Between September 9, 2013, and August 29, 2014, a randomised 2 &#x000d7; 2 &#x000d7; 3 factorial trial recruited 6- to 23-month-old children with MAM in Burkina Faso. The intervention comprised 12 weeks of food supplementation providing 500 kcal/day as LNS or CSB, each containing SI or DS, and 0%, 20%, or 50% of protein from milk. Fat-free mass (FFM) was assessed by deuterium dilution technique. By dividing FFM by length squared, the primary outcome was expressed independent of length as FFM index (FFMI) accretion over 12 weeks. Other outcomes comprised recovery rate and additional anthropometric measures. Of 1,609 children, 4 died, 61 were lost to follow-up, and 119 were transferred out due to supplementation being switched to non-experimental products. No children developed allergic reaction. At inclusion, 95% were breastfed, mean (SD) <span class="trait">weight</span> was 6.91 kg (0.93), with 83.5% (5.5) FFM. In the whole cohort, <span class="trait">weight</span> increased 0.90 kg (95% <span class="gene">CI</span> 0.88, 0.93; <italic> p &#x0003c; </italic> 0.01) comprising 93.5% (95% <span class="gene">CI</span> 89.5, 97.3) FFM. As compared to children who received CSB, FFMI accretion was increased by 0.083 kg/m <sup> 2 </sup> (95% <span class="gene">CI</span> 0.003, 0.163; <italic> p </italic> = 0.042) in those who received LNS. In contrast, SI did not increase FFMI compared to DS (mean difference 0.038 kg/m <sup> 2 </sup> ; 95% <span class="gene">CI</span> &#x02212;0.041, 0.118; <italic> p </italic> = 0.35), irrespective of matrix. Having 20% milk protein was associated with 0.097 kg/m <sup> 2 </sup> (95% <span class="gene">CI</span> &#x02212;0.002, 0.196) greater FFMI accretion than having 0% milk protein, although this difference was not significant ( <italic> p </italic> = 0.055), and there was no effect of 50% milk protein (0.049 kg/m <sup> 2 </sup> ; 95% <span class="gene">CI</span> &#x02212;0.047, 0.146; <italic> p </italic> = 0.32). There was no effect modification by season, admission criteria, or baseline FFMI, stunting, inflammation, or breastfeeding ( <italic> p &#x0003e; </italic> 0.05). LNS compared to CSB resulted in 128 g (95% <span class="gene">CI</span> 67, 190; <italic> p &#x0003c; </italic> 0.01) greater <span class="trait">weight</span> gain if both contained SI, but there was no difference between LNS and CSB if both contained DS (mean difference 22 g; 95% <span class="gene">CI</span> &#x02212;40, 84; <italic> p </italic> = 0.49) (interaction <italic> p </italic> = 0.017). Accordingly, SI compared to DS increased <span class="trait">weight</span> by 89 g (95% <span class="gene">CI</span> 27, 150; <italic> p </italic> = 0.005) when combined with LNS, but not when combined with CSB. A limitation of this and other food supplementation trials is that it is not possible to collect reliable data on individual adherence.
					</p>
				
					<p>&nbsp;&nbsp;
						Baseline <span class="trait">weight</span> mean (SD) was 6.91 kg (0.93). <span class="trait">Weight</span> at baseline comprised 83.5% (5.5) FFM. We estimated, based on regression of baseline data on age, that <span class="trait">weight</span> would increase by 469 g (95% <span class="gene">CI</span> 453, 485) and FFM by 434 g (95% <span class="gene">CI</span> 416, 453) over 3 months in the absence of supplementation. During the intervention, <span class="trait">weight</span> increased 0.90 kg (95% <span class="gene">CI</span> 0.88, 0.93), and 93.5% (95% <span class="gene">CI</span> 89.6, 97.4) of the <span class="trait">weight</span> gain comprised FFM. WHZ increased from &#x02212;2.22 (0.51) by 0.68 (95% <span class="gene">CI</span> 0.65, 0.72), while the height-for-age <italic> z </italic> -score, which was &#x02212;1.70 (1.1) at baseline, declined by 0.17 (95% <span class="gene">CI</span> 0.15, 0.19) ( <xref ref-type="table" rid="pmed.1002387.t003"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						There was no effect modification by season, admission criteria, or baseline FFMI, stunting, inflammation, or breastfeeding ( <italic> p &#x0003e; </italic> 0.05). For <span class="trait">weight</span> gain, an interaction between matrix and soy quality was found ( <italic> p </italic> = 0.017). Hence, the 75 g (95% <span class="gene">CI</span> 32, 119) main effect of LNS shown in <xref ref-type="table" rid="pmed.1002387.t004"> Table 4 </xref> reflects a 128 g (95% <span class="gene">CI</span> 67, 190) larger gain when giving LNS compared to CSB if both contained SI, but no difference between LNS and CSB if both contained DS (22 g; 95% <span class="gene">CI</span> &#x02212;40, 84). Similarly, the main 36 g (95% <span class="gene">CI</span> &#x02212;8, 79) effect of SI compared to DS reflects that SI increased <span class="trait">weight</span> by 89 g (95% <span class="gene">CI</span> 27, 150) compared to DS when combined with LNS, but not when combined with CSB (&#x02212;18 g; 95% <span class="gene">CI</span> &#x02212;80, 44). LNS compared to CSB had an effect on both MUAC (mean difference 1 mm; 95% <span class="gene">CI</span> 0.5, 1.6) and triceps skinfold (mean difference 0.16 mm; 95% <span class="gene">CI</span> 0.06, 0.25). None of the factors had effects on length and knee-heel length, although 50% milk protein was associated with a 0.07 cm (95% <span class="gene">CI</span> &#x02212;0.01, 0.14) knee-heel length gain ( <italic> p </italic> = 0.07) compared to 0% milk protein.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome Regions Associated with Functional Performance of Soybean Stem Fibers in Polypropylene Thermoplastic Composites</h2>
			<h3>PMID:26167917</h3>
				
					<h3>PMCID:4500502</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Reinprecht </surname>  <given-names> Yarmilla </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arif </surname>  <given-names> Muhammad </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Simon </surname>  <given-names> Leonardo C. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pauls </surname>  <given-names> K. Peter </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Currently, 487 QTL for maturity, <span class="trait">plant</span> height, <span class="trait">lodging</span> and height per unit of <span class="trait">lodging</span> [159 maturity QTL (20 chromosomes, 32 studies), 187 <span class="trait">plant</span> height QTL (20 chromosomes, 35 studies), 80 <span class="trait">lodging</span> QTL (19 chromosomes, 27 studies) and 11 height per unit of <span class="trait">lodging</span> QTL (four chromosomes, four studies)] have been identified in different populations and are deposited in SoyBase (available at <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> ; accessed 14 Mar 2015). Current work identified 82 QTL for these traits using a set (50 RILs) of the RG10 x OX948 population (n = 169) in three locations over two years. Twenty five QTL were detected for <span class="trait">plant</span> height on 12 chromosomes and explained up to 36% of the phenotypic variability for the trait. Twenty three QTL were identified for <span class="trait">plant</span> height on seven chromosomes. These QTL explained up to 44% of total phenotypic variability for the trait. Sixteen QTL for <span class="trait">lodging</span> were mapped on nine chromosomes and explained 11% to 44% variability for the trait. Using the set of 50 RILs, this study confirmed several maturity QTL (on chromosomes <span class="gene">Gm02,</span> <span class="gene">Gm08,</span> <span class="gene">Gm10,</span> <span class="gene">Gm11</span> and <span class="gene">Gm13),</span> <span class="trait">plant</span> height QTL (chromosomes <span class="gene">Gm03,</span> <span class="gene">Gm06,</span> <span class="gene">Gm10</span> and <span class="gene">Gm18)</span> and <span class="trait">lodging</span> QTL (on chromosomes <span class="gene">Gm11</span> and <span class="gene">Gm19)</span> identified previously with the whole population {n = 169, [ <xref rid="pone.0130371.ref051" ref-type="bibr"> 51 </xref> ]}. Eighteen QTL for the <span class="trait">plant</span> height to <span class="trait">lodging</span> ratio (H/L) selection trait were mapped on nine chromosomes and linkage group X and explained up to 32% phenotypic variability for the trait. Eleven QTL for this trait were identified in soybean previously {SoyBase, available at www://soybase.org; accessed 14 Mar 2015; [ <xref rid="pone.0130371.ref077" ref-type="bibr"> 77 </xref> ]} and were mapped on four chromosomes <span class="gene">(Gm07,</span> <span class="gene">Gm13,</span> <span class="gene">Gm18</span> and <span class="gene">Gm19)</span> in four independent mapping populations [ <xref rid="pone.0130371.ref078" ref-type="bibr"> 78 </xref> , <xref rid="pone.0130371.ref079" ref-type="bibr"> 79 </xref> ]. Although some H/L QTL identified in the current study mapped to the same regions as QTL identified earlier (eg. QTL on chromosomes <span class="gene">Gm07,</span> <span class="gene">Gm13,</span> <span class="gene">Gm18</span> and <span class="gene">Gm19),</span> the majority of the H/L QTL mapped to new genomic regions on six different chromosomes <span class="gene">(Gm01,</span> <span class="gene">Gm02,</span> <span class="gene">Gm10,</span> <span class="gene">Gm11,</span> <span class="gene">Gm14</span> and <span class="gene">Gm17)</span> and linkage group X ( <xref rid="pone.0130371.g004" ref-type="fig"> Fig 4 </xref> ; <xref rid="pone.0130371.t004" ref-type="table"> Table 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Nevertheless, several fiber traits were associated with specific cell wall biosynthetic genes mapped in the RG10 x OX948 population. For example, impact strength QTL IS8W on chromosome <span class="gene">Gm12</span> ( <xref rid="pone.0130371.g004" ref-type="fig"> Fig 4 </xref> ; Fig G in <xref rid="pone.0130371.s001" ref-type="supplementary-material"> S1 File </xref> ) co-localized with fiber QTL HC9H (hemicellulose) and both were associated with the gene coding for glycine-rich protein (GRP). This gene-spercific marker explained 13% phenotypic variability for the impact strength in Woodstock in 2008. The superfamily of <span class="trait">plant</span> glycine-rich proteins (GRPs) is characterized by the presence of semi-repetitive glycine-rich motifs and diversity in structure, expression and localization. GRPs may act as scaffolds for the deposition of cell wall constituents, during protoxylem development and in cell wall fortification by connecting lignin rings [ <xref rid="pone.0130371.ref083" ref-type="bibr"> 83 </xref> , <xref rid="pone.0130371.ref084" ref-type="bibr"> 84 </xref> ]. In addition, it was shown that Arabidopsis <italic> AtGRP9 </italic> interacts with <italic> AtCAD5 </italic> (which catalyzes the last step in monolignol biosynthesis [ <xref rid="pone.0130371.ref085" ref-type="bibr"> 85 </xref> ]&#x02013;the reduction of cinnamaldehydes into cinnamyl alcohols) suggesting the involvement of GRPs in lignin biosynthesis/deposition [ <xref rid="pone.0130371.ref086" ref-type="bibr"> 86 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						An example of overlapping fiber composite performance QTL occurred on chromosome <span class="gene">Gm06</span> ( <xref rid="pone.0130371.g004" ref-type="fig"> Fig 4 </xref> ; Fig G in <xref rid="pone.0130371.s001" ref-type="supplementary-material"> S1 File </xref> ) between markers UBC463-600 and Satt357 (27 cM) where a QTL for flexural modulus FM8W overlapped with several other composite performance QTL, including: two ultimate tensile stregth QTL (UTS9H and UTS8W), a flexural sterngth QTL (FS8W), a flexural modulus QTL (FM8W) and a tensile strength QTL (TS8W). In addition, it co-localized with two fiber composition QTL [C8R (cellulose) and FPH8R (free phenolics)] and a <span class="trait">lodging</span> QTL (LG9H). Because of low polymorphism in the RG10 x OX948 population, we were not able to map any of the candidate cell wall synthesis genes directly in this region. However, the <italic> in silico </italic> map (Fig G in <xref rid="pone.0130371.s001" ref-type="supplementary-material"> S1 File </xref> ) contains numerous potential cell wall candidate genes in this region, including: glycosyl hydrolase 5 (Cell.GlyH5), cell wall protein (CWall), xyloglucan endo-transglycosylase (XET), cytochrome <span class="gene">P450</span> (P450), peroxidase (Pox), cellulose synthase (CesA) and a number of transcription factors.
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, two QTL for tensile strength (TS8W and TS9H) on chromosome <span class="gene">Gm06</span> overlaped with a QTL for impact strength (IS8H), a QTL for tensile modulus (TM8W) and two QTL for ultimate tensile sterngth (UTS9H and UTS8W), and these co-localized with a number of fiber composition QTL for lignin (L8W and L8R), cellulose (C9H) and free phenolics (FPH9H)) and three <span class="trait">plant</span> height QTL (PH9W, PH8W and PH8R). The <italic> in silico </italic> map for this region contains cell wall-related genes, such as genes coding for expansin (EXP), Myb305-like transcription factor (Myb305), glycosyl-phosphatidyl inositol anchored protein [COBRA(TC216062)], flavonoid 3'-hydroxylase (F3'H), peroxidase (Pox), ABC transporter (ABCtr) or cellulose synthase (CesA). A gene coding for a member of a COBRA-like protein family (COBL4) was mapped on chromosome <span class="gene">Gm18</span> ( <xref rid="pone.0130371.g004" ref-type="fig"> Fig 4 </xref> ; Fig G in <xref rid="pone.0130371.s001" ref-type="supplementary-material"> S1 File </xref> ). Maize brittle stalk2 ( <italic> bk2 </italic> ) encodes a COBRA-like protein (similar to phytochelatin synthetase), which is involved in secondary cell wall biosynthesis [ <xref rid="pone.0130371.ref087" ref-type="bibr"> 87 </xref> ]. It is expressed in early organ development but, it is also required for tissue flexibility at maturity. Mutants make smaller plants with reduced levels of cellulose and cell wall sugars. In general, cell wall biosynthetic (structural and regulatory) genes are encoded by gene families. In addition, extensive duplication exists in the soybean genome [ <xref rid="pone.0130371.ref088" ref-type="bibr"> 88 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Several genomic regions contained clusters of linked and/or peiotropic loci that affected a number of traits ( <xref rid="pone.0130371.g004" ref-type="fig"> Fig 4 </xref> ; Tables <xref rid="pone.0130371.t004" ref-type="table"> 4 </xref> and <xref rid="pone.0130371.t005" ref-type="table"> 5 </xref> ). For example, SND2-Sat_120 marker interval on chromosome <span class="gene">Gm13</span> (F) was associated with QTL for days to maturity (DM9R), height per unit of <span class="trait">lodging</span> (HL08H, HL08R, HL08W), ultimate tensile strength (UTS9H) and tensile strength (TS9H). Similarly, marker interval (UBC122 <sub> -1500 </sub> -Satt070-Satt534) on chromosome <span class="gene">Gm14</span> (B2) was associated with QTL for several traits, inclusing days to maturity (DM9R), <span class="trait">lodging</span> (LG8H, LG8R, LG8W, LG9W), height per unit of <span class="trait">lodging</span> (HL8H, HL8R, HL8W), cellulose (C9R), flexural modulus (FM9H), ultimate tensile strength (UTS8H), tensile strength (TS8H, TS9W) and tensile modulus (TM8H). Pleoitropy is usually associated with major gene effects [ <xref rid="pone.0130371.ref089" ref-type="bibr"> 89 </xref> ]. A common genetic basis might explain some correlations between agronomic, fiber compositional and composite traits. Alternatively, clustering of genes for different traits might be the basis for overlapping QTL. Mansur et al [ <xref rid="pone.0130371.ref090" ref-type="bibr"> 90 </xref> ] and Orf et al [ <xref rid="pone.0130371.ref079" ref-type="bibr"> 79 </xref> ] observed clustering of QTL with strong effects on maturity, <span class="trait">plant</span> height and <span class="trait">lodging.</span> However, the marker density in the current map was not sufficient to determine if the regions significant for more than one trait were the result of pleiotropy or gene linkage. Currently, a work to add microarray-based single feature polymorphism (SFP) markers to the map is underway. A more saturated map would help to resolve this ambiguity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Oil Is More Obesogenic and Diabetogenic than Coconut Oil and Fructose in Mouse: Potential Role for the Liver</h2>
			<h3>PMID:26200659</h3>
				
					<h3>PMCID:4511588</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Deol </surname>  <given-names> Poonamjot </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Evans </surname>  <given-names> Jane R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dhahbi </surname>  <given-names> Joseph </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chellappa </surname>  <given-names> Karthikeyani </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Diana S. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Spindler </surname>  <given-names> Stephen </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sladek </surname>  <given-names> Frances M. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The obesity epidemic in the U.S. has led to extensive research into potential contributing dietary factors, especially fat and fructose. Recently, increased consumption of soybean oil, which is rich in polyunsaturated fatty acids (PUFAs), has been proposed to play a causal role in the epidemic. Here, we designed a series of four isocaloric diets (HFD, SO-HFD, F-HFD, F-SO-HFD) to investigate the effects of saturated versus unsaturated fat, as well as fructose, on obesity and diabetes. C57/BL6 male mice fed a diet moderately high in fat from coconut oil and soybean oil (SO-HFD, 40% kcal total fat) showed statistically significant increases in <span class="trait">weight</span> gain, adiposity, diabetes, glucose intolerance and insulin <span class="trait">resistance</span> compared to mice on a diet consisting primarily of coconut oil (HFD). They also had fatty livers with hepatocyte ballooning and very large lipid droplets as well as shorter colonic crypt length. While the high fructose diet (F-HFD) did not cause as much obesity or diabetes as SO-HFD, it did cause rectal prolapse and a very fatty liver, but no balloon injury. The coconut oil diet (with or without fructose) increased spleen <span class="trait">weight</span> while fructose in the presence of soybean oil increased kidney <span class="trait">weight.</span> Metabolomics analysis of the liver showed an increased accumulation of PUFAs and their metabolites as well as &#x003b3;-tocopherol, but a decrease in cholesterol in SO-HFD. Liver transcriptomics analysis revealed a global dysregulation of cytochrome <span class="gene">P450</span> ( <italic> Cyp </italic> ) genes in SO-HFD versus HFD livers, most notably in the <italic> Cyp3a </italic> and <italic> Cyp2c </italic> families. Other genes involved in obesity (e.g., <italic> Cidec </italic> , <italic> Cd36 </italic> ), diabetes ( <italic> Igfbp1 </italic> ), inflammation ( <italic> Cd63 </italic> ), mitochondrial function ( <italic> Pdk4 </italic> ) and cancer ( <italic> H19 </italic> ) were also upregulated by the soybean oil diet. Taken together, our results indicate that in mice a diet high in soybean oil is more detrimental to metabolic health than a diet high in fructose or coconut oil.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we examined the effect of both soybean oil and fructose on the development of obesity and its associated co-morbidities in C57/BL6 male mice, in the context of a diet moderately high in total fat. We formulated diets with 40% kcal fat from coconut oil (HFD) or coconut oil plus soybean oil (SO-HFD) and then supplemented those with fructose (F-HFD and F-SO-HFD). The total amount of fat is comparable to current American consumption [ <xref rid="pone.0132672.ref035" ref-type="bibr"> 35 </xref> ] as is the soybean oil [ <xref rid="pone.0132672.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0132672.ref036" ref-type="bibr"> 36 </xref> ] and fructose [ <xref rid="pone.0132672.ref037" ref-type="bibr"> 37 </xref> ]. Mice on the high soybean oil diet (SO-HFD) showed increased <span class="trait">weight</span> gain, adiposity, fatty liver with balloon injury, diabetes and IR, compared to mice on the HFD. Fructose in the diet (F-HFD and F-SO-HFD) had less severe metabolic effects than soybean oil but caused rectal prolapse and seemed to synergize with soybean oil to increase kidney <span class="trait">weight.</span> The SO-HFD produced distinct changes in the liver transcriptome from the HFD, most notable of which was a global dysregulation of cytochrome <span class="gene">P450</span> ( <italic> Cyp </italic> ) genes as well as genes involved in obesity, diabetes, lipid metabolism and cancer. Metabolomic analyses of the HFD and SO-HFD livers revealed a greater accumulation of PUFAs and their metabolites but also increased antioxidant capacity and decreased cholesterol in SO-HFD. Taken together, these results indicate that while there may be some health benefits to a diet rich in soybean oil, and while fructose induced some negative effects in the gut and kidney, overall soybean oil induced more obesity, diabetes, IR and liver injury than either fructose or saturated fat from coconut oil in mice.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the Hsf family in soybean and functional identification of <italic> <span class="gene">GmHsf-34</span> </italic> involvement in <span class="trait">drought</span> and heat stresses</h2>
			<h3>PMID:25416131</h3>
				
					<h3>PMCID:4253008</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Pan-Song </given-names>  </name>  <address>  <email> pasonlee@sohu.com </email>  </address>  <xref ref-type="aff" rid="Aff20"/>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Yu </surname>  <given-names> Tai-Fei </given-names>  </name>  <address>  <email> yutaifei824@163.com </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Guan-Hua </given-names>  </name>  <address>  <email> hghqwe1024@sina.cn </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ming </given-names>  </name>  <address>  <email> chenming@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Yong-Bin </given-names>  </name>  <address>  <email> zhouyongbin@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Chai </surname>  <given-names> Shou-Cheng </given-names>  </name>  <address>  <email> chaishoucheng@126.com </email>  </address>  <xref ref-type="aff" rid="Aff20"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Xu </surname>  <given-names> Zhao-Shi </given-names>  </name>  <address>  <email> xuzhaoshi@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> You-Zhi </given-names>  </name>  <address>  <email> mayouzhi@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff21"/>  </contrib>  <aff id="Aff20">  <label/> College of Agronomy, Northwest A &#x00026; F University, Yangling, 712100 China  </aff>  <aff id="Aff21">  <label/> Chinese Academy of Agricultural Sciences (CAAS)/National Key Facility for Crop Gene Resources and Genetic Improvement, Key Laboratory of Biology and Genetic Improvement of Triticeae Crops, Ministry of Agriculture, Institute of Crop Science, Beijing, 100081 China  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We analyzed the Hsf genetic structures and protein function domains using the GSDS, Pfam, SMART, PredictNLS, and NetNES online tools. The genome scanning of dicots (soybean and <italic> Arabidopsis </italic> ) and monocots (rice and maize) revealed that the whole-genome replication occurred twice in soybean evolution. The <span class="trait">plant</span> Hsfs were classified into 3 classes and 16 subclasses according to protein structure domains. The A8 and B3 subclasses existed only in dicots and the A9 and <span class="gene">C2</span> occurred only in monocots. Thirty eight soybean Hsfs were systematically identified and grouped into 3 classes and 12 subclasses, and located on 15 soybean chromosomes. The promoter regions of the soybean Hsfs contained <italic> cis </italic> -elements that likely participate in <span class="trait">drought,</span> low temperature, and ABA stress responses. There were large differences among Hsfs based on transcriptional levels under the stress conditions. The transcriptional levels of the A1 and A2 subclass genes were extraordinarily high. In addition, differences in the expression levels occurred for each gene in the different organs and at the different developmental stages. Several genes were chosen to determine their subcellular localizations and functions. The subcellular localization results revealed that <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> were located in the nucleus. Overexpression of the <italic> <span class="gene">GmHsf-34</span> </italic> gene improved the tolerances to <span class="trait">drought</span> and heat stresses in <italic> Arabidopsis </italic> plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Since the first <span class="trait">plant</span> Hsf gene was identified in tomato [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], the Hsf family genes have been thoroughly characterized, and 21, 25, 25, and 27 Hsf genes were found in <italic> Arabidopsis </italic> , rice, maize, and tomato, respectively [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> &#x02013; <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. In the present study, we scanned for and integrated all the nonredundant sets of the soybean Hsf genes, determined their chromosomal locations, predicted their protein structures by available software and network stations, analyzed the expression levels of the soybean Hsf genes by qRT-PCR and identified the function of <italic> <span class="gene">GmHsf-34</span> </italic> in the tolerance to <span class="trait">drought</span> and heat stresses. This study provides a version on the structures and evolutionary history of the soybean Hsfs, and a candidate gene to the crop molecular breeding.
					</p>
				
					<p>&nbsp;&nbsp;
						Gene structure analysis revealed the existence of introns in the soybean Hsf genes. Four introns were found in <italic> <span class="gene">GmHsf-20</span> </italic> , 3 in <italic> <span class="gene">GmHsf-23</span> </italic> , and 2 in <italic> <span class="gene">GmHsf-02</span> </italic> , <italic> <span class="gene">GmHsf-12</span> </italic> , <italic> <span class="gene">GmHsf-17</span> </italic> , and <italic> <span class="gene">GmHsf-18</span> </italic> respectively (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). <italic> Cis </italic> -element analysis demonstrated that every soybean Hsf member carried one or more MYB and MYC elements in their promoters. In addition, 52.6% of the members contained an ABA-responsive element (ABRE), 31.6% contained a dehydration-responsive element (DRE), and 42.1% contained a low-temperature responsive element (LTRE) (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). It was reported that the above-mentioned 5 elements play different significant roles in stress responses in plants. For example, MYB is involved in stress-induced <span class="trait">drought,</span> low temperature, salt, ABA, and GA responses [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. ABRE responds to <span class="trait">drought</span> and ABA <italic> via </italic> combination with ABRE binding proteins (AREB) [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. DRE combining with DRE binding proteins (DREB) participate in <span class="trait">drought,</span> salt, low temperature, and ABA responses [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. LTRE contributes primarily to low temperature response and regulation [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Analyses of <italic> cis </italic> -elements in the promoters suggest that Hsfs are significantly related to stress response. <fig id="Fig3"> <label> Figure 3 </label> <caption> <p> <bold> Intron-exon structures of soybean Hsf genes. </bold> The Intron-exon structures were produced by the GSDS online tool. The exons, introns and untranslated regions (UTRs) were indicated by the white boxes, black lines and gray lines respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Nineteen soybean Hsf genes which expressed highly in different tissues were selected to further confirm their responses to <span class="trait">drought</span> stress and heat stress. qRT-PCR was carried out using soybean plants exposed to <span class="trait">drought</span> (0, 6, and 12&#x000a0;h) and high temperature (0, 6, and 12&#x000a0;h). These genes expressed diversely under both stresses (Figure&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> A, B); 14 genes were up-regulated (&#x0003e;2-fold) by <span class="trait">drought</span> stress, and 13 were up-regulated by heat stress. Notably, 10 soybean Hsf genes ( <italic> <span class="gene">GmHsf-04</span> </italic> , <italic> <span class="gene">GmHsf-08</span> </italic> , <italic> <span class="gene">GmHsf-09</span> </italic> , <italic> <span class="gene">GmHsf-10</span> </italic> , <italic> <span class="gene">GmHsf-11</span> </italic> , <italic> <span class="gene">GmHsf-16</span> </italic> , <italic> <span class="gene">GmHsf-17</span> </italic> , <italic> <span class="gene">GmHsf-25</span> </italic> , <italic> <span class="gene">GmHsf-33</span> </italic> , and <italic> <span class="gene">GmHsf-34</span> </italic> ) showed up-regulation under both <span class="trait">drought</span> and heat stress conditions. Two soybean Hsf genes ( <italic> <span class="gene">GmHsf-03</span> </italic> and <italic> <span class="gene">GmHsf-07</span> </italic> ) were greatly down-regulated (&#x0003c;0.5-fold) during the heat stress treatment. Moreover, the transcript level of <italic> <span class="gene">GmHsf-38</span> </italic> , was unchanged by either stress. <fig id="Fig7"> <label> Figure 7 </label> <caption> <p> <bold> Relative expressions of soybean Hsf genes under <span class="trait">drought</span> and heat conditions. </bold> Analyses were carried out by qRT-PCR under <span class="trait">drought</span> <bold> (A) </bold> and heat stress <bold> (B) </bold> treatment. qRT-PCR data were normalized using soybean <italic> Actin </italic> (U60506) gene and shown relative to 0&#x000a0;h. X-axes showed soybean Hsf genes and y-axes are scales of relative expression level (error bars indicate SD).
					</p>
				
					<p>&nbsp;&nbsp;
						Three genes ( <italic> <span class="gene">GmHsf-04</span> </italic> , <italic> <span class="gene">GmHsf-33</span> </italic> , and <italic> <span class="gene">GmHsf-34</span> </italic> ) up-regulated strongly by both heat and <span class="trait">drought</span> were selected for subcellular localization. Expression vectors with green fluorescent protein (GFP) tags were constructed for subcellular localization analysis. The coding regions of <italic> <span class="gene">GmHsf-04</span> </italic> , <italic> <span class="gene">GmHsf-33</span> </italic> , and <italic> <span class="gene">GmHsf-34</span> </italic> were amplified from the soybean cDNA by PCR with specific primers and fused to the N-terminal of GFP under control of the CaMV 35S promoter. Subcellular localization of GFP expression in mesophyll cell protoplasts of <italic> Arabidopsis </italic> was monitored by confocal microscopy 16&#x000a0;h after transformation mediated by PEG; 35S::GFP vector was transformed as the control. As shown in Figure&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> , control hGFP was uniformly distributed throughout the mesophyll cell protoplast, whereas <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> fusion proteins were exclusively localized in the nucleus. These results suggest that <span class="gene">GmHsf-04,</span> <span class="gene">GmHsf-33,</span> and <span class="gene">GmHsf-34</span> are nuclear proteins, possibly serving as transcription factors. <fig id="Fig8"> <label> Figure 8 </label> <caption> <p> <bold> Subcellular localization of <span class="gene">GmHsf</span> proteins. </bold> The 35S::GmHsf-04-GFP, 35S::GmHsf-33-GFP, 35S::GmHsf-34-GFP and 35S::GFP control vectors were transiently expressed in <italic> Arabidopsis </italic> protoplasts. Results were visualized by a confocal microscopy 16&#x000a0;h after transformation. Bars =10&#x000a0;&#x003bc;m.
					</p>
				
					<p>&nbsp;&nbsp;
						According to the expression analysis, <italic> <span class="gene">GmHsf-34</span> </italic> was strongly induced by <span class="trait">drought</span> and heat stresses. To confirm the functions of <italic> <span class="gene">GmHsf-34</span> </italic> in abiotic stress response, three lines of <italic> Arabidopsis </italic> overexpressing <italic> <span class="gene">GmHsf-34</span> </italic> were tested under <span class="trait">drought</span> and high temperature conditions, respectively. Seed germination and root growth of transgenic <italic> Arabidopsi </italic> s were tested in the presence of 4% PEG (Figure&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> A to D). Under standard culture conditions, no significant differences in germination rate or morphology between transgenic and wild-type (Col-0) plants were observed. However, germination percentage of transgenic plants was enhanced by nearly 15% compared to wild-type after 2-3 days (Figure&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> A and B). In the presence of 4% PEG, roots of transgenic lines were longer than those of wild-type plants (Figure&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> C and D), showing that overexpression of <italic> <span class="gene">GmHsf-34</span> </italic> improved tolerance to the imposed <span class="trait">drought</span> treatment in <italic> Arabidopsis </italic> . After heat stress treatment, survival rates of <italic> Arabidopsis </italic> seedlings overexpressing <italic> <span class="gene">GmHsf-34</span> </italic> and wild-type were recorded (Figure&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> E and F). The survival rate of wild-type <italic> Arabidopsis </italic> seedlings was about 11.5%, whereas that of <italic> Arabidopsis </italic> seedlings overexpressing <italic> <span class="gene">GmHsf-34</span> </italic> was improved to 58.5-62.5%. Obviously, the transgenic seedlings displayed higher tolerance to high temperature compared to wild-type plants. <fig id="Fig9"> <label> Figure 9 </label> <caption> <p> <bold> Responses of </bold> <bold> <italic> <span class="gene">GmHsf-34</span> </italic> </bold> <bold> overexpressing </bold> <bold> <italic> Arabdopsis </italic> </bold> <bold> to <span class="trait">drought</span> and heat stresses </bold> <bold> <italic> . </italic> </bold> <bold> (A) </bold> Seed germinations of wild-type (Col-0) and <italic> <span class="gene">GmHsf-34</span> </italic> overexpressing <italic> Arabdopsis </italic> seeds. Seeds from three independent transgenetic lines of <italic> <span class="gene">GmHsf-34</span> </italic> were grown on &#x000bd; <span class="gene">MS</span> medium with or without 4% PEG. Wild-type seeds were grown in the same condition as a control. Representative images of wild-type and one line (OE-1) of the transgenic plants were taken after treated by 4% PEG for 4 days. <bold> (B) </bold> Germination rates of wild-type and <italic> <span class="gene">GmHsf-34</span> </italic> overexpressing <italic> Arabdopsis </italic> seeds. Germination rates were determined daily after 2&#x000a0;days stratification. Data represent means&#x02009;&#x000b1;&#x02009;SD (n =90). <bold> (C) </bold> and <bold> (D) </bold> Root lengths of wild-type and <italic> <span class="gene">GmHsf-34</span> </italic> overexpressing plants. Root length was measured by the ImageJ 2X software. Data represent means&#x02009;&#x000b1;&#x02009;SD (n =90). <bold> (E) </bold> Representative images of wild-type and transgenic plants after treated under high temperature conditions. Four-week-old wild-type and <italic> <span class="gene">GmHsf-34</span> </italic> overexpressing seedlings (OE-1, OE-2, and OE-3) were exposed to 42&#x000b0;C for 6&#x000a0;hours heat stress, and then transferred to normal condition. After one week, the photographs were taken. Three independent experiments were conducted. <bold> (F) </bold> Survival rates of wild-type and <italic> <span class="gene">GmHsf-34</span> </italic> overexpressing plants after heat stress. One week after heat treated, survival rate of wild-type and transgenetic plants. Data represent means&#x02009;&#x000b1;&#x02009;SD (n =90).
					</p>
				
					<p>&nbsp;&nbsp;
						In recent years, research on the role of introns has made significant progress. Studies in mammals, nematodes, insects, fungi, and plants suggest that the introns not only regulate the gene expression, but also participate in the gene evolution [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. Analysis of gene structures revealed that soybean Hsf genes contain a single intron except <italic> <span class="gene">GmHsf-12</span> </italic> containing two introns. Combined with analysis of the gene chip expression results (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ), the soybean Hsf gene <italic> <span class="gene">GmHsf-12</span> </italic> was not expressed at lower levels than others under the normal conditions. Seemingly, an intron does not affect gene expressions. Combined with the analysis to phylogenetic evolution (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ), the number and location of <italic> Hsf </italic> intron in the same subclass are conserved (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). For example, <italic> <span class="gene">GmHsf-30</span> </italic> and <italic> <span class="gene">GmHsf-34</span> </italic> in subclass A2 respectively at 307&#x000a0;bp to 2110&#x000a0;bp and 304&#x000a0;bp to 2113&#x000a0;bp section contains an intron, which indicated that introns could be a reference to the evolution of <span class="trait">plant</span> genes.
					</p>
				
					<p>&nbsp;&nbsp;
						The signal transduction pathways are complicated networks where components work together to control the <span class="trait">plant</span> physiological and biochemical process. <span class="gene">AREB1</span> (ABRE-BINDING PROTEIN 1), AREB2, and ABF3, members of class A bZIP transcription co-factors of the ABRE elements, regulate the response to osmotic stress through combining with the element in the <italic> <span class="gene">DREB2A</span> </italic> promoter region [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. The analysis of <italic> cis </italic> -acting elements in their promoter regions revealed that the soybean Hsf genes contain the MYB/MYC elements and some contain the ABRE, DRE, and/or LTRE elements, demonstrating that the soybean Hsfs play significant roles in the regulation of stress responses (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). The MYB elements basically participate in the <span class="trait">drought,</span> low temperature, salt, ABA, and GA stress responses [ <xref ref-type="bibr" rid="CR34"> 34 </xref> , <xref ref-type="bibr" rid="CR35"> 35 </xref> ] and the MYC elements participate in the <span class="trait">drought,</span> salt, and ABA stress responses. We show that the <italic> Arabidopsis </italic> , rice, and maize Hsf gene promoters contain the MYB/MYC elements (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ) and we concluded that Hsfs are involved in the responses to <span class="trait">drought,</span> salt, and ABA in plants. However, the expression results from the gene chips did not agree perfectly with the conclusion. For instance, <italic> <span class="gene">GmHsf-25</span> </italic> contained 15 <italic> cis </italic> -acting elements and its expression value was up to 1090, whereas the value for <italic> <span class="gene">GmHsf-10</span> </italic> carrying 85 <italic> cis </italic> -acting elements was only 228 (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). One explanation may be that, several elements lost their activities or performed in a negative way.
					</p>
				
					<p>&nbsp;&nbsp;
						In consideration of putative similar functions of Hsf genes in the same subclass, 19 genes containing all subclass were selected to investigate responses to <span class="trait">drought</span> and heat stresses using qRT-PCR. The results showed that the soybean Hsf genes were differently expressed under the <span class="trait">drought</span> stress and heat stress conditions. We detected that three soybean Hsfs ( <italic> <span class="gene">GmHsf-04</span> </italic> , <italic> <span class="gene">GmHsf-17</span> </italic> , and <italic> <span class="gene">GmHsf-33</span> </italic> ) belonged to the subclass A1 were expressed at significantly high levels under the <span class="trait">drought</span> stress and heat stress (Figure&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ). Moreover, <italic> <span class="gene">GmHsf-08</span> </italic> and <italic> <span class="gene">GmHsf-34</span> </italic> (subclass A2), <italic> <span class="gene">GmHsf-11</span> </italic> (subclass A4), <italic> <span class="gene">GmHsf-09</span> </italic> (subclass A5), <italic> <span class="gene">GmHsf-10</span> </italic> (subclass A8), <italic> <span class="gene">GmHsf-25</span> </italic> (subclass B1), and <italic> <span class="gene">GmHsf-16</span> </italic> (subclass C1) were up-regulated under both <span class="trait">drought</span> stress and heat stress conditions. It was reported that tomato HsfA1a and <italic> Arabidopsis </italic> HsfA2 function as master regulators for acquired thermo-tolerance [ <xref ref-type="bibr" rid="CR36"> 36 </xref> , <xref ref-type="bibr" rid="CR37"> 37 </xref> ], and tomato HsfB1 was a co-regulator with HsfA1a [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. These results indicate that A1, A2, and several other subclass members may be involved in the <span class="trait">drought</span> stress and heat stress responses in plants. According to the soybean gene chip data, <italic> <span class="gene">GmHsf-02</span> </italic> , <italic> <span class="gene">GmHsf-07</span> </italic> , <italic> <span class="gene">GmHsf-19</span> </italic> , <italic> <span class="gene">GmHsf-20</span> </italic> , <italic> <span class="gene">GmHsf-28</span> </italic> , <italic> <span class="gene">GmHsf-35</span> </italic> , and <italic> <span class="gene">GmHsf-37</span> </italic> were expressed at very low levels. Among them, <italic> <span class="gene">GmHsf-07</span> </italic> , <italic> <span class="gene">GmHsf-20</span> </italic> , and <italic> <span class="gene">GmHsf-28</span> </italic> belong to subclass A3. It is likely that the functions of these soybean Hsf genes, especially subclass A3, are not related to <span class="trait">drought</span> or heat stress responses. Most of the soybean Hsf genes expressed in roots were regulated by <span class="trait">drought</span> while those expressed in young leaves were regulated by heat. This is consistent with the fact that the root is the first organ sensing <span class="trait">drought</span> stress whereas leaf is first to experience heat stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In the former publication [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ], expression analyses of 5 soybean Hsf genes ( <italic> <span class="gene">GmHsf12</span> </italic> , <italic> <span class="gene">GmHsf28</span> </italic> , <italic> <span class="gene">GmHsf34</span> </italic> , <italic> <span class="gene">GmHsf35</span> </italic> , and <italic> <span class="gene">GmHsf47</span> </italic> ) were performed under heat, low-temperature, NaCl, and <span class="trait">drought</span> stresses, respectively. These 5 genes were named in our study as <italic> <span class="gene">GmHsf-08</span> </italic> , <italic> <span class="gene">GmHsf-21</span> </italic> , <italic> <span class="gene">GmHsf-26</span> </italic> , <italic> <span class="gene">GmHsf-25</span> </italic> , and <italic> <span class="gene">GmHsf-32</span> </italic> respectively. We founded that <italic> <span class="gene">GmHsf-08</span> </italic> , <italic> <span class="gene">GmHsf-21</span> </italic> , and <italic> <span class="gene">GmHsf-25</span> </italic> showed markedly up-regulation by heat, which was consistent with the former works, while <italic> <span class="gene">GmHsf-26</span> </italic> showed no detectable alteration. Under <span class="trait">drought</span> condition, <italic> <span class="gene">GmHsf-08</span> </italic> , <italic> <span class="gene">GmHsf-25</span> </italic> , and <italic> <span class="gene">GmHsf-</span> </italic> 26 showed to be up-regulated strongly while <italic> <span class="gene">GmHsf-21</span> </italic> expression was not influenced. <italic> <span class="gene">GmHsf-32</span> </italic> expressions under abotic conditions were not surveyed in our study. <italic> <span class="gene">GmHsf-34</span> </italic> was strongly induced by <span class="trait">drought</span> and heat stresses and its overexpression improved survival rate and/or root development in <italic> Arabidopsis </italic> under simulated <span class="trait">drought</span> and heat conditions (Figure&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ). Similarly, overexpression of <italic> Arabidopsis AtHsfA2 </italic> , one of the most strongly induced genes, lead to enhanced tolerance to heat stress [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Given the close phylogenetic relationship of <italic> <span class="gene">GmHsf-34</span> </italic> with <italic> AtHsfA2 </italic> , it is speculated that <span class="gene">GmHsf-34</span> functions as a typical transcription factor due to the existence of Hsf-type DBD, OD, NES, NLS, and AHA motifs, and participates in heat and <span class="trait">drought</span> responses.
					</p>
				
					<p>&nbsp;&nbsp;
						Thirty eight soybean Hsf genes were initially identified and classified after scanning for the soybean genome data base. Their locations and duplications, intron-exon structures, DBD structures and HR-A/B, distribution of <italic> cis </italic> -acting elements in the soybean Hsf promoters, and the expression patterns were determined. Based on the expression analysis, we inferred that soybean Hsf subclasses A1 and A2 may be the primary regulators of the heat stress response in soybean. <italic> <span class="gene">GmHsf-34</span> </italic> , a member of subclass A2, played an important role in the response to the <span class="trait">drought</span> and heat stress treatments imposed in this study.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmHsf-34</span> </italic> gene, which is induced by the <span class="trait">drought</span> and heat stresses, was selected to confirm gene functions. Expression vector pBI121::GmHsf-34 containing <italic> <span class="gene">GmHsf-34</span> </italic> under control of the CaMV35S promoter was built. Three <italic> Arabidopsis </italic> lines overexpressing <italic> <span class="gene">GmHsf-34</span> </italic> were obtained after the transformation mediated by agrobacterium ( <italic> Agrobacterium tumefaciens </italic> ). For the germination assays, the seeds of Col-0 and transgenic plants were placed on &#x000bd; <span class="gene">MS</span> medium containing no or 4% (w/v) PEG for 7&#x000a0;days. For the root growth assays, 4-day-old seedlings grown on &#x000bd; <span class="gene">MS</span> medium were transferred to &#x000bd; <span class="gene">MS</span> medium containing no or 4% (w/v) PEG for 4&#x000a0;days after which the root lengths were measured by a root system scanner. For the heat stress tolerance assays, the 21-day-old seedlings grown in the soil transferred from &#x000bd; <span class="gene">MS</span> medium were treated at 42&#x000b0;C for 6&#x000a0;h and then grown under the normal condition for several days.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A rice LSD1-like-type ZFP gene <italic> OsLOL5 </italic> enhances saline-alkaline tolerance in transgenic <italic> Arabidopsis thaliana </italic> , yeast and rice</h2>
			<h3>PMID:26920613</h3>
				
					<h3>PMCID:4769587</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> QJ </given-names>  </name>  <address>  <email> guanqingjie@nefu.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> HY </given-names>  </name>  <address>  <email> 849021046@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> ZJ </given-names>  </name>  <address>  <email> wangzhenjuan0205@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> ZY </given-names>  </name>  <address>  <email> wangzhenyu@neigaehrb.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bu </surname>  <given-names> QY </given-names>  </name>  <address>  <email> buqingyun111@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Liu </surname>  <given-names> SK </given-names>  </name>  <address>  <email> shenkuiliu@nefu.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Key Laboratory of Saline-alkali Vegetation Ecology Restoration in Oil Field (SAVER), Ministry of Education, Alkali Soil Natural Environmental Science Center (ASNESC), Northeast Forestry University, No.26 Hexing Road, Nangang District, Harbin City, Heilongjiang 150040 China  </aff>  <aff id="Aff2">  <label/> Lab of Soybean Molecular Biology and Molecular Breeding, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, No.138 Haping Road, Nangang District, Harbin City, Heilongjiang 150081 China  </aff>
				<br/> 
				年份：<day> 10 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Using primer pair OsLOL5-P4, the <italic> OsLOL5 </italic> PCR product was ligated into the expression vector pCXSN after XcmI digestion [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. For <italic> Arabidopsis </italic> transformation, the <italic> pCXSN::OsLOL5 </italic> (Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ) vectors were first introduced into <italic> Agrobacterium tumefaciens </italic> GV3101 by electroporation. <italic> Arabidopsis </italic> cv Col-0 plants were transformed via floral dip method as previously described [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. <italic> Arabidopsis </italic> transgenic seeds were plated on half-strength Murashige and Skoog (MS) medium containing 25&#x000a0;mg&#x02009;&#x000b7;&#x02009;L <sup> &#x02212;1 </sup> hygromycin for selection. Resistant plants were used for molecular identification. To study the function of OsLOL5 in the abiotic stress response, the transgenic T3 generation encoding <italic> OsLOL5 </italic> driven by the cauliflower mosaic virus (CaMV35S) promoter were tested with primer pair OsLOL5-P5 and then used for the following studies. The T3 (#1&#x02013;#3) and WT seeds were sterilized and sowed in 1/2 <span class="gene">MS</span> medium for germination for 10 d. The seedlings were then transferred to 1/2 <span class="gene">MS</span> medium containing 0 (0&#x000a0;mmol&#x000a0;L <sup> -1 </sup> as control group), 100, 125, or 150&#x000a0;mmol&#x000a0;L <sup> -1 </sup> NaCl or 0, 2, 4, and 6&#x000a0;mmol&#x000a0;L <sup> -1 </sup> NaHCO <sub> 3 </sub> . After 30 d, the growth phenotype, <span class="trait">plant</span> height, fresh <span class="trait">weight,</span> and MDA content of the seedlings were measured.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean &#x003b2;-Conglycinin Induces Inflammation and Oxidation and Causes Dysfunction of Intestinal Digestion and Absorption in Fish</h2>
			<h3>PMID:23520488</h3>
				
					<h3>PMCID:3592885</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jin-Xiu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Lin-Ying </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Feng </surname>  <given-names> Lin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Wei-Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kuang </surname>  <given-names> Sheng-Yao </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Kai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shu-Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xiao-Qiu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> in vivo </italic> experiment investigated the hepatopancreas <span class="trait">weight</span> (HW), protein content (HPC), hepatopancreas somatic index (HSI) and intestinal <span class="trait">weight</span> (IW), protein content (IPC), length (IL), intestinal length index (ILI), intestinal somatic index (ISI) and folds height. In addition, digestive ability (intestinal trypsin, chymotrypsin, lipase and amylase activities) and absorptive function (creatinkinase, Na <sup> + </sup> ,K <sup> + </sup> -ATPase, alkaline phosphatase and gamma-glutamyl transpeptidase (&#x003b3;-GT) in the proximal (PI), mid (MI) and distal intestine (DI)) were studied. Furthermore, malondialdehyde (MDA), protein carbonyls (PC), superoxide dismutase (SOD), catalase (CAT), glutathione- <italic> S </italic> -transferase (GST), glutathione peroxidase (GPx) and glutathione reductase (GR) activities and glutathione (GSH) content in the hepatopancreas and intestine were also investigated. Finally, gene expression of <span class="gene">TOR</span> and <span class="gene">4E-BP</span> in the PI, MI and DI and interleukin-1 (IL-1), IL-8, tumor necrosis factor-&#x003b1; (TNF-&#x003b1;) and TGF-&#x003b2; genes in the distal intestine (DI) as well as the whole intestinal GPx1a, GPx1b, GR, CuZnSOD, MnSOD and CAT were investigated.
					</p>
				
					<p>&nbsp;&nbsp;
						Gln is a precursor of nucleotides, glucose, other amino acids and serves as a major fuel for intestinal epithelial cells in mammals <xref ref-type="bibr" rid="pone.0058115-Windmueller1"> [75] </xref> . Recent studies have demonstrated that dietary Gln supplementation could improve growth performance, intestinal structure and function in carp <xref ref-type="bibr" rid="pone.0058115-Lin2"> [24] </xref> , as well as enhance the antioxidant defense in hybrid sturgeon ( <italic> Acipenser schrenckii &#x02640;&#x000d7;Huso dauricus &#x02642; </italic> ) <xref ref-type="bibr" rid="pone.0058115-Zhu1"> [76] </xref> . According to these findings, we investigated the potential protection of Gln against &#x003b2;-conglycinin toxicity in the present study. As results, compared with the &#x003b2;-conglycinin alone group, dietary supplementation with Gln significantly increased SGR, feed intake, feed efficiency, hepatopancreas protein, hepatopancreas <span class="trait">weight,</span> intestinal protein, intestinal <span class="trait">weight,</span> intestinal length, intestinal length index and foregut folds height, and also significantly increased the activities of intestinal digestive enzymes including trypsin, lipase and chymotrypsin, absorptive enzymes including creatinkinase, Na <sup> + </sup> , K <sup> + </sup> -ATPase and alkaline phosphatase in the PI, MI and DI, as well as antioxidant enzymes including CAT and <span class="gene">GR</span> in hepatopancreas and SOD, CAT, GPx in the intestine. Moreover, SGR, intestinal length index, activities of intestinal trypsin, lipase, chymotrypsin, creatinkinase, Na <sup> + </sup> ,K <sup> + </sup> -ATPase, alkaline phosphatase and GPx were restored to have no significant difference compared with the control. These results indicated that dietary Gln exerted a protective role against the growth suppression, intestinal integrity damage and function disruption induced by &#x003b2;-conglycinin.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of Safety of Iron-Fortified Soybean Sprouts, a Potential Component of Functional Food, in Rat</h2>
			<h3>PMID:26880214</h3>
				
					<h3>PMCID:4786607</h3>
					<h4>期刊:Plant Foods for Human Nutrition (Dordrecht, Netherlands)</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kujawska </surname>  <given-names> Ma&#x00142;gorzata </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ewertowska </surname>  <given-names> Ma&#x00142;gorzata </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ignatowicz </surname>  <given-names> Ewa </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Adamska </surname>  <given-names> Teresa </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Szaefer </surname>  <given-names> Hanna </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zieli&#x00144;ska-Dawidziak </surname>  <given-names> Magdalena </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Piasecka-Kwiatkowska </surname>  <given-names> Dorota </given-names>  </name>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Jodynis-Liebert </surname>  <given-names> Jadwiga </given-names>  </name>  <address>  <email> liebert@ump.edu.pl </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Toxicology, Poznan University of Medical Sciences, 30 Dojazd Str., 60-631 Pozna&#x00144;, Poland  </aff>  <aff id="Aff2">  <label/> Department of Pharmaceutical Biochemistry, Poznan University of Medical Sciences, 4 &#x0015a;wi&#x00119;cicki Str., 60-781 Pozna&#x00144;, Poland  </aff>  <aff id="Aff3">  <label/> Department of Food Biochemistry and Analysis, Poznan University of Life Sciences, 48 Mazowiecka Str., 60-623 Pozna&#x00144;, Poland  </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The difference between the heme-iron and ionic iron systems of the <span class="trait">plant</span> ferritin absorption suggests that the protein may be an abundant source of this element for individuals with disturbed uptake mechanisms [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ]. Legume seeds are the richest in ferritin among edible parts of plants. We found that the cultivation of soybean seeds in a solution of FeSO <sub> 4 </sub> permits obtaining material with extremely high iron content of 560.6&#x000a0;mg/100&#x000a0;g of dry matter, while the ferritin iron content was ~420.5&#x000a0;mg/100&#x000a0;g d.m. It means that the ferritin iron represented ~75&#x000a0;% of total iron in the obtained preparation. In sprouts cultured in distilled water the total iron or ferritin iron content was distinctly lower: 11.0&#x000a0;mg/100&#x000a0;g d.m. and 1.8&#x000a0;mg/100&#x000a0;g d.m., respectively. Moreover, the powdered soybean sprouts have been found not only a source of iron, but also various bioactive compounds. Abiotic stress influence reactive oxygen species formation, both as a consequence of hydrolytic processes and oxidative conditions. Apart from ferritin other compounds with strong antioxidant properties are formed while sprouting in the described conditions [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. TRAP determined in the material tested was 5.57&#x000a0;&#x000b1;&#x000a0;0.16&#x000a0;mM TrxE/kg dry matter and this value corresponds to the relatively high concentration of both total phenolic and flavonoid compounds: 3.55&#x000a0;&#x000b1;&#x000a0;0.02&#x000a0;mg GAE/g d.m. and 0.185&#x000a0;&#x000b1;&#x000a0;0.02&#x000a0;mg QE/g d.m., respectively. The TRAP value for sprouts cultured in water was slightly higher, 7.16&#x000a0;&#x000b1;&#x000a0;2.21, whereas the concentration of total phenolic or flavonoid compounds was similar to that found in iron-fortified sprouts: 4.01&#x000a0;&#x000b1;&#x000a0;0.03&#x000a0;mg GAE/g d.m. and 0.222&#x000a0;&#x000b1;&#x000a0;0.06&#x000a0;mg QE/g d.m., respectively. Thus, it could be expected that potential pro-oxidant effects associated to the high content of iron would be cancelled by the presence of antioxidant compounds in the material tested. To assess the potential adverse effects of the preparation containing such a high concentration of iron, soybean sprouts powder was administered to rats for 90&#x000a0;days via diet. On the basis of food consumption measurements and the nominal dietary concentration of the material tested, the calculated mean daily intake of soybean sprouts was 0.63, 1.96, 3.90&#x000a0;g/kg body weight/day in males and 0.50, 1.25, 4.10&#x000a0;g/kg body weight/day in females. There were no statistically significant differences in final body <span class="trait">weight</span> and mean food consumption between controls and rats administered sprouts (data not shown). No statistically significant differences in haematology and clinical chemistry parameters were found between controls and treated rats (Tables I and II, supplementary material). Microscopic examination of 22 tissues did not reveal any pathology due to soybean sprouts intake (data not shown). A fundamental role of iron in generation of reactive oxygen species, prompted us to investigate the markers of oxidative damage to hepatic lipids, protein and DNA. No changes in the levels of carbonyl groups (marker of protein oxidation) as well as DNA damage (determined by comet assay) were found. The level of lipid peroxidation, expressed as TBARS concentration, was slightly elevated, by 22&#x02013;28&#x000a0;%, only in females. This increase was statistically significant in mid-dose and high-dose group (Table <xref rid="Tab1" ref-type="table"> 1 </xref> ). Our results showed that the basal level of TBARS was greater in females than in males which has been noted also by other authors [ <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. Hence, it could be hypothesized that the slightly weaker antioxidant status of females rendered them more susceptible to ferric ions ingested with soybean sprouts. However, it cannot be translated to humans since there are no data concerning the sex-depended differences in antioxidant status. No&#x000a0;substantial difference in the content of reduced glutathione (GSH), an essential cell antioxidant, between controls and treated animals was observed (Table <xref rid="Tab1" ref-type="table"> 1 </xref> ). The results of the antioxidant enzymes activity determination are depicted in Table <xref rid="Tab2" ref-type="table"> 2 </xref> . The most consistent changes were observed for two glutathione related enzymes, glutathione peroxidase (GPx) and glutathione reductase (GR). GPx activity was increased in all (except for high-dose males) groups by 13&#x02013;58&#x000a0;%. In all groups of rats a decrease in <span class="gene">GR</span> activity by 22&#x02013;34&#x000a0;% was found. These changes, although consistent, were not dose related. The activity of superoxide dismutase (SOD) was increased in all female groups by 27&#x02013;54&#x000a0;% and in one low-dose male group. Changes in catalase activity were diversified: 19&#x000a0;% decrease in low-dose male and female group, 20&#x000a0;% increase in mid-dose females and no changes in high-dose groups. Similarly, S-glutathione transferase activity was changed only in some animals, showing 32 and 53&#x000a0;% increase in mid- and high-dose female groups, respectively. A decrease in <span class="gene">GR</span> activity observed in all treated groups of rats can be ascribed to the inhibition of the enzyme by some flavonoids, components of soybean sprouts. Also other authors have reported a decrease in the activity of <span class="gene">GR</span> in the liver of rats treated with quercetin and genistein [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] or in rat liver cells incubated with delphinidin, (&#x02212;)epicatechin, kaempferol, quercetin, luteolin, naringenin and apigenin [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. <table-wrap id="Tab1"> <label> Table 1 </label> <caption> <p> Effect of soybean sprouts on oxidative stress parameters in the liver of rats
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed Coat Cracking in Soybean Isolines for Pubescence Color and Maturity.</h2>
			<h3>PMID:11756256</h3>
				

				
			<p>
				作者：Yang D(1), Nakamura T, Ohtsubo N, Takahashi K, Abe J, Takahashi R.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed coats of soybean crack under various stress conditions. Cracking of seed coats degrades the external appearance of soybean seeds and reduces their commercial value. Previous studies revealed that the T gene responsible for pubescence color, and the maturity genes, <span class="gene">E1</span> and E5, had inhibitory effects on low-temperature induced seed coat cracking. The objective of this study was to evaluate the effects of the T gene and five maturity genes (E1 to E5) on the intensity of seed coat cracking induced by pod-removal treatments. Soybean cv. Harosoy (te1e2E3E4e5) and its near-isogenic lines for T and <span class="gene">E1</span> to E5 loci were used in the experiment. Cracking was induced by removing the upper 50% of pods on the <span class="trait">plant</span> 40 d after anthesis. Frequency and degree of cracking were not different among the isolines in the control group. In contrast, there were significant differences among isolines subjected to the pod-removal treatment. Frequency and degree of cracking was low in Harosoy, Harosoy-E1, e3, and e4, and high in Harosoy-T and E2. The results suggest that genotypes at T and E2 loci were associated with severity of seed coat cracking induced by pod-removal. There was a positive correlation (r = 0.90**) between individual seed <span class="trait">weight</span> and frequency of cracking among isolines in the pod-removal treatment. Seed coat cracking was probably exacerbated in part by the genes that allow enlargement of individual seeds in response to pod-removal. The differences among isolines suggest that the mechanism of seed coat cracking induced by pod removal may differ from that induced by low temperature treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hypolipidemic and Antioxidant Activity of Enoki Mushrooms ( <italic> Flammulina velutipes </italic> )</h2>
			<h3>PMID:25250317</h3>
				
					<h3>PMCID:4164374</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-6290-2702 </contrib-id>  <name>  <surname> Yeh </surname>  <given-names> Ming-Yei </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 7 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The animal diet is detailed in <xref ref-type="table" rid="tab1"> Table 1 </xref> . Selected 5-6-week old male Syrian hamsters were used in this study and housed individually in stainless steel cages. The temperature was 24 &#x000b1; 1&#x000b0;C, the humidity was 40 ~ 60%, and the daily light cycle was 12&#x02009;h. After one week of ambience adaptation, they were randomly divided into 8 groups of 6 hamsters each. One group of hamsters was given a normal diet (N) and 7 groups of hamsters were given a high fat high cholesterol diet (H). After two weeks of high fat inducement, either 1, 2, and 3% FVP or 1, 2, and 3 % <span class="gene">FVE</span> was added to the diet of one of the high fat groups and no addition was made in one high fat group (see <xref ref-type="table" rid="tab1"> Table 1 </xref> ). The experiment lasted for 8 weeks. Animal body <span class="trait">weight</span> was recorded daily, whereas the intake of water and food was recorded every two days ( <xref ref-type="table" rid="tab5"> Table 5 </xref> ). The animals fasted overnight at the end of the experiment, and samples were collected for analysis, including feces, liver, and blood. The animals were etherized and dissected; the blood samples were extracted from abdominal aorta to analyze total triglyceride (TG), total cholesterol, low density lipoprotein (LDL), and high density lipoprotein (HDL) content. All samples were stored at &#x02212;80&#x000b0;C so that they were ready for the measurement of the blood lipid.
					</p>
				
					<p>&nbsp;&nbsp;
						Effect of dried FVP and <span class="gene">FVE</span> on food intake, body <span class="trait">weight,</span> and feed efficiency of hamster <sup> a </sup> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential Degradation and Detoxification of an Aromatic Pollutant by Two Different Peroxidases</h2>
			<h3>PMID:28335468</h3>
				
					<h3>PMCID:5372743</h3>
					<h4>期刊:Biomolecules</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Alneyadi </surname>  <given-names> Aysha Hamad </given-names>  </name>  <xref ref-type="aff" rid="af1-biomolecules-07-00031"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shah </surname>  <given-names> Iltaf </given-names>  </name>  <xref ref-type="aff" rid="af2-biomolecules-07-00031"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> AbuQamar </surname>  <given-names> Synan F. </given-names>  </name>  <xref ref-type="aff" rid="af1-biomolecules-07-00031"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ashraf </surname>  <given-names> Syed Salman </given-names>  </name>  <xref ref-type="aff" rid="af2-biomolecules-07-00031"> 2 </xref>  <xref rid="c1-biomolecules-07-00031" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Enzymatic degradation of organic pollutants is a new and promising remediation approach. Peroxidases are one of the most commonly used classes of enzymes to degrade organic pollutants. However, it is generally assumed that all peroxidases behave similarly and produce similar degradation products. In this study, we conducted detailed studies of the degradation of a model aromatic pollutant, Sulforhodamine B dye (SRB dye), using two peroxidases&#x02014;soybean peroxidase (SBP) and chloroperoxidase (CPO). Our results show that these two related enzymes had different optimum conditions (pH, temperature, H <sub> 2 </sub> O <sub> 2 </sub> concentration, etc.) for efficiently degrading SRB dye. High-performance liquid chromatography and liquid chromatography &#x02013;mass spectrometry analyses confirmed that both <span class="gene">SBP</span> and CPO transformed the SRB dye into low molecular <span class="trait">weight</span> intermediates. While most of the intermediates produced by the two enzymes were the same, the CPO treatment produced at least one different intermediate. Furthermore, toxicological evaluation using lettuce ( <italic> Lactuca sativa </italic> ) seeds demonstrated that the SBP-based treatment was able to eliminate the phytotoxicity of SRB dye, but the CPO-based treatment did not. Our results show, for the first time, that while both of these related enzymes can be used to efficiently degrade organic pollutants, they have different optimum reaction conditions and may not be equally efficient in detoxification of organic pollutants.
					</p>
				
					<p>&nbsp;&nbsp;
						Our data showed that SRB dye was toxic to the <span class="trait">plant</span> and significantly affected its growth and development. Surprisingly, there were no significant toxic effects of SRB dye on the actual germination of the seeds ( <xref ref-type="fig" rid="biomolecules-07-00031-f008"> Figure 8 </xref> ). <xref ref-type="fig" rid="biomolecules-07-00031-f008"> Figure 8 </xref> B also shows that after the SBP/H <sub> 2 </sub> O <sub> 2 </sub> treatment, the toxicity of SRB dye was reduced. Surprisingly, treatment of SRB dye with CPO/H <sub> 2 </sub> O <sub> 2 </sub> failed to eliminate the toxicity of the dye. Statistical analyses showed that there was no significant difference in the root length of seeds treated with SBP-degraded SRB dye (1.64 &#x000b1; 0.66 cm) compared with the control (2.36 &#x000b1; 1.12 cm), suggesting almost complete elimination of SRB dye toxicity to <italic> L. sativa </italic> seeds. Conversely, the mean root length of the seeds treated with CPO-degraded SRB dye was only 0.14 &#x000b1; 0.17 cm compared to those of seeds treated with SBP-degraded SRB dye ( <xref ref-type="fig" rid="biomolecules-07-00031-f008"> Figure 8 </xref> B). This difference between SBP- and CPO-treated SRB dye shows the importance of phytotoxicity analyses of remediated wastewater samples. One possible reason for the difference could be due to the way the two enzymes degraded SRB dye. <xref ref-type="fig" rid="biomolecules-07-00031-f007"> Figure 7 </xref> and <xref ref-type="table" rid="biomolecules-07-00031-t002"> Table 2 </xref> show that, although most of the intermediates produced by both enzymes were the same (compounds <bold> A </bold> , <bold> B </bold> , and <bold> C </bold> ), the concentrations of intermediates A and B after CPO-based degradation were significantly higher than those produced in the SBP-based treatment. This difference might explain the toxicity of the CPO-treated samples. Additionally, the 168 <italic> m </italic> / <italic> z </italic> intermediate (compound <bold> D </bold> ) was only observed during the CPO treatment, so it is possible that the toxicity of the CPO-treated SRB dye could also be partly due to this compound. Interestingly, other researchers have also reported that enzymatic degradation of dyes could result in increased toxicity in some cases [ <xref rid="B53-biomolecules-07-00031" ref-type="bibr"> 53 </xref> , <xref rid="B54-biomolecules-07-00031" ref-type="bibr"> 54 </xref> ]. These results highlight the dramatic differences between <span class="gene">SBP</span> and CPO enzymes and also stress the importance of toxicological evaluation of peroxidase-degraded enzymatic pollutants.
					</p>
				
					<p>&nbsp;&nbsp;
						Sulforhodamine B at 97% purity (molecular formula = C <sub> 27 </sub> H <sub> 30 </sub> N <sub> 2 </sub> O <sub> 7 </sub> S <sub> 2 </sub> , formula <span class="trait">weight</span> = 558.6 g&#x000b7;mol <sup> &#x02212;1 </sup> ) was purchased from AnaSpec (Fremont, CA, USA). <span class="gene">SBP</span> and CPO were purchased from Bio-Research Products (North Liberty, IA, USA). The specific activities of the enzymes were as follows: <span class="gene">SBP</span> was 2700 IU/mg (1 mg/mL), 26 &#x003bc;M, while CPO activity was 1296 IU/mg (17 mg/mL), 405 &#x003bc;M. The enzymes were diluted in water to the appropriate concentrations as needed. Hydrogen peroxide (30% <italic> w </italic> / <italic> v </italic> ) was purchased from Sigma-Aldrich (St. Louis, MO, USA). Universal buffers were prepared using 0.1 M citric acid and 0.2 M K <sub> 2 </sub> HPO <sub> 4 </sub> . LC&#x02013;MS grade acetonitrile, methanol, and formic acid were obtained from Sigma-Aldrich Chemical Co.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of an efficient acyl-CoA: diacylglycerol acyltransferase 1 ( <italic> <span class="gene">DGAT1</span> </italic> ) gene from the microalga <italic> Chlorella ellipsoidea </italic></h2>
			<h3>PMID:28222675</h3>
				
					<h3>PMCID:5319178</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Xuejie </given-names>  </name>  <address>  <email> xuejie2011@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Chengming </given-names>  </name>  <address>  <email> cmfan@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Yuhong </given-names>  </name>  <address>  <email> yhchen@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jingqiao </given-names>  </name>  <address>  <email> jingqiao_wang@126.com </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yin </surname>  <given-names> Weibo </given-names>  </name>  <address>  <email> wbyin@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Richard R. C. </given-names>  </name>  <address>  <email> Richard.Wang@ARS.USDA.GOV </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Hu </surname>  <given-names> Zanmin </given-names>  </name>  <address>  <phone> 86-10-64807626 </phone>  <email> zmhu@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0596 2989 </institution-id>  <institution-id institution-id-type="GRID"> grid.418558.5 </institution-id>  <institution/>  <institution> Institute of Genetics and Developmental Biology, Chinese Academy of Sciences,  </institution>  </institution-wrap> Beijing, 100101 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1797 8419 </institution-id>  <institution-id institution-id-type="GRID"> grid.410726.6 </institution-id>  <institution/>  <institution> University of Chinese Academy of Sciences,  </institution>  </institution-wrap> Beijing, 100049 China  </aff>  <aff id="Aff3">  <label> 3 </label> Institute of Economical Crops, Yunnan Agricultural Academy, Kunming, 65023 China  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2185 8768 </institution-id>  <institution-id institution-id-type="GRID"> grid.53857.3c </institution-id>  <institution> United States Department of Agriculture, Agricultural Research Service, Forage and Range Research Laboratory,  </institution>  <institution> Utah State University,  </institution>  </institution-wrap> Logan, UT 84322-6300 USA  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000119573309 </institution-id>  <institution-id institution-id-type="GRID"> grid.9227.e </institution-id>  <institution/>  <institution> Present address: Institute of Genetics and Developmental Biology, Chinese Academy of Sciences,  </institution>  </institution-wrap> Datun Road, Chaoyang District, Beijing, 100101 China  </aff>
				<br/> 
				年份：<day> 2 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA encoding a putative diacylglycerol acyltransferase 1 (DGAT1, EC 2.3.1.20) was obtained from <italic> Chlorella ellipsoidea </italic> . The 2,142&#x000a0;bp open reading frame of this cDNA, designated <italic> CeDGAT1 </italic> , encodes a protein of 713 amino acids showing no more than 40% identity with DGAT1s of higher plants. Transcript analysis showed that the expression level of <italic> CeDGAT1 </italic> markedly increased under nitrogen starvation, which led to significant triacylglycerol (TAG) accumulation. CeDGAT1 activity was confirmed in the yeast quadruple mutant strain H1246 by restoring its ability to produce TAG. Upon expression of <italic> CeDGAT1 </italic> , the total fatty acid content in wild-type yeast (INVSc1) increased by 142%, significantly higher than that transformed with <italic> DGAT1s </italic> from higher plants, including even the oil crop soybean. The over-expression of Ce <italic> <span class="gene">DGAT1</span> </italic> under the NOS promoter in wild-type <italic> Arabidopsis thaliana </italic> and <italic> Brassica napus </italic> var. Westar significantly increased the oil content by 8&#x02013;37% and 12&#x02013;18% and the average 1,000-seed <span class="trait">weight</span> by 9&#x02013;15% and 6&#x02013;29%, respectively, but did not alter the fatty acid composition of the seed oil. The net increase in the 1,000-seed total lipid content was up to 25&#x02013;50% in both transgenic <italic> Arabidopsis </italic> and <italic> B. napus </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In eukaryotes, three types of DGATs have been reported: the endoplasmic reticulum (ER)- localized DGAT1, DGAT2 and the soluble cytosolic DGAT3. Among them, <span class="gene">DGAT1</span> and DGAT2 are responsible for the bulk of TAG synthesis in most organisms [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. It has been proposed that these two enzymes have no redundant functions in TAG biosynthesis [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. <span class="gene">DGAT1</span> plays a dominating role in the determination of oil accumulation and fatty acid composition in seed oils [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ], and DGAT2 may influence the content and composition of some <span class="trait">plant</span> seed oils containing unusual fatty acids (e.g., epoxy and hydroxyl) [ <xref ref-type="bibr" rid="CR11"> 11 </xref> &#x02013; <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. The role of the cytosolic DGAT3 has not yet been determined.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, we isolated a <italic> <span class="gene">DGAT1</span> </italic> gene ( <italic> CeDGAT1 </italic> ) from <italic> C. ellipsoidea </italic> , a unicellular eukaryotic green alga that can be easily cultured under either autotrophic or heterotrophic conditions, and characterized its function in yeast and higher plants ( <italic> Arabidopsis </italic> and <italic> B. napus </italic> ). Compared with DGAT1s of higher plants, such as <italic> Glycine max </italic> , <italic> Arabidopsis </italic> and <italic> Brassica oleracea </italic> , CeDGAT1 could more effectively enhance fatty acid accumulation in the wild-type yeast (INVSc1). The over-expression of <italic> CeDGAT1 </italic> can significantly enhance the seed oil content and seed <span class="trait">weight</span> in <italic> Arabidopsis </italic> and <italic> B. napus </italic> . Furthermore, the expression pattern of the isolated <italic> <span class="gene">DGAT1</span> </italic> gene was investigated. This study would be helpful for understanding the function of <span class="gene">DGAT</span> from microalgae and for improving oil production in <italic> B. napus </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In higher plants, the expression of <italic> <span class="gene">DGAT</span> </italic> generally correlates with oil deposition in developing seeds [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. For soybeans, a stronger expression of <italic> <span class="gene">DGAT1</span> </italic> was found in developing seeds than in other tissues [ <xref ref-type="bibr" rid="CR13"> 13 </xref> ]. However, <span class="gene">DGAT1</span> transcripts were also detected in other <span class="trait">plant</span> tissues, e.g., <italic> AtDGAT </italic> was expressed in a wide range of tissues but most strongly in developing embryos and flower petals [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. <span class="gene">DGAT1</span> is also highly expressed during pollen development, presumably contributing to TAG accumulation in the pollen grain [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. These findings suggested that these <span class="gene">DGAT</span> enzymes may be related to other physiological processes in addition to seed oil synthesis [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. For unicellular eukaryotic green algae, all physiological processes take place within a cell; thus, the expression of <italic> <span class="gene">DGAT1</span> </italic> can directly reflect the dynamics of TAG accumulation.
					</p>
				
					<p>&nbsp;&nbsp;
						Several previous studies have reported that the genetic manipulation of <span class="gene">DGAT</span> can lead to increased oil biosynthesis in <italic> Arabidopsis </italic> and <italic> B. napus </italic> . For instance, the seed-specific over-expression of <italic> A. thaliana <span class="gene">DGAT1</span> </italic> in wild-type <italic> Arabidopsis </italic> can increase the seed oil content by 11&#x02013;28% and the seed <span class="trait">weight</span> by 2.5&#x02013;32.3% [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Similarly, the seed-specific expression of <italic> TmDGAT1 </italic> in wild-type <italic> Arabidopsis </italic> resulted in a 10&#x02013;33% net increase in the seed oil content and a 15% increase in the 1,000-seed <span class="trait">weight</span> in transgenic <italic> Arabidopsis </italic> . Furthermore, the seed-specific expression of <italic> TmDGAT1 </italic> in high-erucic acid <italic> B. napus </italic> led to a net increase of 11&#x02013;15% in the seed oil content of transgenic plants [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ]. In addition, the over-expression of <italic> AtDGAT1 </italic> and <italic> BnDGAT1 </italic> in canola under the control of the napin promoter led to an increase of 2.5&#x02013;7% in the oil content [ <xref ref-type="bibr" rid="CR56"> 56 </xref> ]. The over-expression of <italic> JcDGAT1 </italic> in <italic> Arabidopsis </italic> under both CaMV35S promoter and a seed specific promoter resulted in a 30&#x02013;41% increase in the seed oil content [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. Our studies showed that the expression of <italic> CeDGAT1 </italic> in <italic> Arabidopsis </italic> and <italic> B. napus </italic> under the NOS promoter does indeed increase oil biosynthesis in transgenic seeds by approximately 8&#x02013;37% and 12&#x02013;18% over that of the control. In addition, neither the <italic> Arabidopsis </italic> nor the <italic> B. napus CeDGAT1 </italic> transformants showed significant changes in fatty acid composition. In some studies, however, there were alterations in the oil composition through <italic> <span class="gene">DGAT</span> </italic> expression. The over-expression of <italic> JcDGAT1 </italic> in <italic> Arabidopsis </italic> resulted in a significant decrease in oleic acid (C18:1) and an increase in linolenic acid (C18:3) [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ], and the transgenic expression of <italic> Sesamum indicum <span class="gene">DGAT1</span> </italic> in <italic> Arabidopsis </italic> led to an increase in eicosenoic acid (C20:1) and a reduction in oleic acid (C18:1) in seed oil [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. More importantly, the expression of <italic> CeDGAT1 </italic> in <italic> Arabidopsis </italic> and <italic> B. napus </italic> under the NOS promoter also led to a significant increase in the average 1,000-seed <span class="trait">weight</span> in <italic> CeDGAT1 </italic> transgenic lines, by 9&#x02013;15% and 6&#x02013;29% in <italic> Arabidopsis </italic> and <italic> B. napus </italic> , respectively, and thus there was no decrease in the 1,000-seed <span class="trait">weight</span> caused by the oil content increase. Considering the constitutive expression of <italic> CeDGAT1 </italic> under the NOS promoter, larger increments in seed oil biosynthesis and seed <span class="trait">weight</span> can probably be expected when using a seed specific promoter. Furthermore, there was a difference between <italic> CeDGAT1 </italic> transgenic <italic> Arabidopsis </italic> and <italic> B. napus </italic> , with respect to effects on seed oil biosynthesis and seed <span class="trait">weight.</span> The seed oil content increased more in transgenic <italic> Arabidopsis </italic> than in <italic> B. napus </italic> , but the average seed <span class="trait">weight</span> increase was greater in <italic> B. napus </italic> . Interestingly, the increase in the oil content on a per-1,000-seed basis was similar between transgenic <italic> Arabidopsis </italic> and <italic> B. napus </italic> , at approximately 25&#x02013;50%. To date, there has been no report that the over-expression of <italic> <span class="gene">DGAT1</span> </italic> can significantly increase the seed <span class="trait">weight</span> in the oil <span class="trait">plant</span> <italic> B. napus </italic> , although this effect has been reported in <italic> Arabidopsis </italic> . In addition, transgenic plants showed no other phenotypic differences. Therefore, CeDGAT1 should have great potential for increasing the net oil production of the oil <span class="trait">plant</span> oilseed rape.
					</p>
				
					<p>&nbsp;&nbsp;
						The complete <italic> CeDGAT1 </italic> was cloned into the <span class="trait">plant</span> expression vector pCAMBIA2301 under the control of the nopaline synthase (NOS) promoter and nos terminator, yielding pCAMBIA2301- <italic> CeDGAT1 </italic> (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> : Figure S5). The final binary vector was verified and then transferred into <italic> Agrobacterium tumefaciens </italic> strain GV3101 by the freeze-thaw method [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ]. <italic> Arabidopsis </italic> plants were transformed by vacuum infiltration [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. <italic> Brassica napus </italic> var. Westar was transformed using hypocotyl explants and the modified method of DeBlock et al. [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. T1 generation seeds were selected on kanamycin (50&#x000a0;mg/L), and then the selected transformed plants were transferred to soil. T3 transgenic <italic> B. napus </italic> lines and homozygous T4 transgenic <italic> Arabidopsis </italic> lines were used for seed and oil analyses. Genomic DNA was isolated from <italic> B. napus </italic> var. Westar leaf material. The stable integration of the NOS: CeDGAT1: nos cassette into the genome of transgenic <italic> B. napus </italic> was checked by PCR amplification using the specific primers P21 and P22 (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> : Table S2). GUS histochemical staining of the leaves from the transgenic lines was also conducted as described by Jefferson et al. [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. In the meantime, the expression of <italic> CeDGAT1 </italic> in <italic> Arabidopsi </italic> s and <italic> B. napus </italic> was detected by RT-PCR using the <italic> CeDGAT1- </italic> specific primers P11 and P12 (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> : Table S2). The <italic> Arabidopsis </italic> housekeeping gene <italic> actin </italic> (primers P23 and P24) and the <italic> B. napus </italic> housekeeping gene <italic> <span class="gene">GAPDH</span> </italic> (primers P25 and P26) were used as internal controls.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ketocarotenoid Production in Soybean Seeds through Metabolic Engineering</h2>
			<h3>PMID:26376481</h3>
				
					<h3>PMCID:4574205</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Pierce </surname>  <given-names> Emily C. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> LaFayette </surname>  <given-names> Peter R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ortega </surname>  <given-names> Mar&#x000ed;a A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joyce </surname>  <given-names> Blake L. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kopsell </surname>  <given-names> Dean A. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Parrott </surname>  <given-names> Wayne A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The pink or red ketocarotenoids, canthaxanthin and astaxanthin, are used as feed additives in the poultry and aquaculture industries as a source of egg yolk and flesh pigmentation, as farmed animals do not have access to the carotenoid sources of their wild counterparts. Because soybean is already an important component in animal feed, production of these carotenoids in soybean could be a cost-effective means of delivery. In order to characterize the ability of soybean seed to produce carotenoids, soybean cv. Jack was transformed with the <italic> <span class="gene">crtB</span> </italic> gene from <italic> Pantoea ananatis </italic> , which codes for phytoene synthase, an enzyme which catalyzes the first committed step in the carotenoid pathway. The <italic> <span class="gene">crtB</span> </italic> gene was engineered together in combinations with ketolase genes ( <italic> crtW </italic> from <italic> Brevundimonas </italic> sp. strain SD212 and <italic> bkt1 </italic> from <italic> Haematococcus pluvialis </italic> ) to produce ketocarotenoids; all genes were placed under the control of seed-specific promoters. HPLC results showed that canthaxanthin is present in the transgenic seeds at levels up to 52 &#x003bc;g/g dry <span class="trait">weight.</span> Transgenic seeds also accumulated other compounds in the carotenoid pathway, such as astaxanthin, lutein, &#x003b2;-carotene, phytoene, &#x003b1;-carotene, lycopene, and &#x003b2;-cryptoxanthin, whereas lutein was the only one of these detected in non-transgenic seeds. The accumulation of astaxanthin, which requires a &#x003b2;-carotene hydroxylase in addition to a &#x003b2;-carotene ketolase, in the transgenic seeds suggests that an endogenous soybean enzyme is able to work in combination with the ketolase transgene. Soybean seeds that accumulate ketocarotenoids could potentially be used in animal feed to reduce or eliminate the need for the costly addition of these compounds.
					</p>
				
					<p>&nbsp;&nbsp;
						Two genes are necessary for ketocarotenoid formation. First, phytoene synthase is needed to obtain &#x003b2;-carotene, and a &#x003b2;-carotene ketolase is needed to produce canthaxanthin. The <italic> Brevundimonas </italic> sp. strain SD212 <italic> crtW </italic> coding sequence was synthesized from GenBank (AB181388.1). GeneArt (Invitrogen) codon-optimized this sequence for expression in soybean (KC954630). To obtain the <italic> bkt1 </italic> coding sequence, RNA was extracted from astaxanthin-producing <italic> H </italic> . <italic> pluvialis </italic> using the Spectrum <span class="trait">Plant</span> Total RNA Kit (Sigma-Aldrich). This RNA was then used as the template for the synthesis of cDNA by SuperScript III Reverse Transcriptase (Invitrogen) using an oligo-dT primer. Polymerase chain reaction was used to amplify the cDNA from <italic> bkt1 </italic> (X86782.1) using gene-specific primers ( <monospace> 5&#x02019;-GTTTGTGCGCCTCGAC-3&#x02019;/5&#x02019;CCAGCTAGGCAGGAACCA-3&#x02019; </monospace> ). The <italic> bkt1 </italic> and <italic> crtW </italic> coding sequences were fused to the pea ribulose 1,5-biphosphate carboxylase small subunit transit peptide [ <xref rid="pone.0138196.ref037" ref-type="bibr"> 37 </xref> ] sequence (which in the case of the <italic> bkt1 </italic> construct contained a native intron) and cloned in between the soybean seed-specific &#x003b2;-conglycinin promoter [ <xref rid="pone.0138196.ref038" ref-type="bibr"> 38 </xref> ] and terminator sequences in the previously constructed pBeta vector [ <xref rid="pone.0138196.ref035" ref-type="bibr"> 35 </xref> ]. The BconP:TP:codingsequence:BconT cassettes were then each cloned into pCEH1B, a pSMART HCKan (Lucigen) based vector previously modified to contain <italic> <span class="gene">crtB</span> </italic> ( <italic> P </italic> . <italic> ananatis) </italic> regulated by the soybean lectin promoter and terminator as well as a hygromycin <span class="trait">resistance</span> gene under control of the potato ( <italic> Solanum tuberosum </italic> ) ubiquitin promoter and terminator for use in selection of transgenic events ( <xref rid="pone.0138196.g002" ref-type="fig"> Fig 2 </xref> ) [ <xref rid="pone.0138196.ref039" ref-type="bibr"> 39 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						RT-PCR was done to verify that the transgenes were expressed in the events producing ketocarotenoids (as determined by HPLC). Tissue was not available for the crtw-2-60, bkt-4-16, and CW-1-12B events. T1 seeds were planted and total RNA was isolated from immature orange-colored cotyledons (40&#x02013;90 mg fresh <span class="trait">weight)</span> from these events using the Quick-RNA Mini Prep kit (Zymo Research) with DNaseI digestion. The cDNA synthesis for RT-PCR was done using the GoScript Reverse Transcription System with oligo (dT) priming (Promega). Orange cotyledons were selected for this analysis except in the case of the non-transformed Jack sample. RT-PCR was done for <italic> <span class="gene">crtB</span> </italic> (primers: <monospace> 5&#x02019;-CTCAATCATGCGGTCGAAAC-3&#x02019;/5&#x02019;-CAATGACAATCACTAGCG-3&#x02019; </monospace> ) and for either <italic> crtW </italic> (primers: <monospace> 5&#x02019;-TGACTGCTGCTGTTGCTGAAC-3&#x02019;/5-&#x02019;CGCTCTTTCTTTCCGGATTCGT-3&#x02019; </monospace> ), or <italic> bkt1 </italic> (primers: <monospace> 5&#x02019;-AATCAAGCTTCCGACCTCCTTGG-3&#x02019;/5-&#x02019;CGCTCTTTCTTTCCGGATTCGT-3&#x02019; </monospace> ) as applicable. An RT-PCR reaction was also done with a non-transgenic cv. Jack control sample for <italic> <span class="gene">crtB</span> </italic> . Control reactions without reverse transcriptase were also done for all samples.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">crtB</span> </italic> transgene encoding a phytoene synthase gene was the only transgene used in this study which encodes an enzyme that acts upstream of &#x003b2;-carotene production in the carotenoid biosynthetic pathway. Nevertheless, &#x003b2;-carotene accumulation in the analyzed dried seeds reached levels between 60&#x02013;741 &#x003bc;g g <sup> -1 </sup> . In contrast, Kim et al. [ <xref rid="pone.0138196.ref048" ref-type="bibr"> 48 </xref> ] used a bicistronic recombinant gene to simultaneously express phytoene synthase from bell pepper ( <italic> Capsicum annum </italic> ) and <italic> <span class="gene">crtI</span> </italic> from <italic> P </italic> . <italic> ananatis </italic> under the control of the &#x003b2;-conglycinin promoter. Their highest level of &#x003b2;-carotene accumulation was 112 &#x003bc;g g <sup> -1 </sup> , even though they used two genes for this purpose. In canola, seed-specific expression of <italic> <span class="gene">crtB</span> </italic> produced 314&#x02013;949 &#x003bc;g g <sup> -1 </sup> fresh <span class="trait">weight</span> of &#x003b2;-carotene [ <xref rid="pone.0138196.ref022" ref-type="bibr"> 22 </xref> ]. When additional transgenes coding for enzymes in the pathway between phytoene and &#x003b2;-carotene were used in conjunction with <italic> <span class="gene">crtB</span> </italic> , carotenoid ratios were altered compared to the use of <italic> <span class="gene">crtB</span> </italic> alone, but the overall levels remained the same and did not lead to higher levels of &#x003b2;-carotene [ <xref rid="pone.0138196.ref034" ref-type="bibr"> 34 </xref> ]. Additionally, a previous study in which soybean seeds were engineered with <italic> <span class="gene">crtB</span> </italic> alone shows that <italic> <span class="gene">crtB</span> </italic> by itself leads to large increases in &#x003b2;-carotene in soybean seeds [ <xref rid="pone.0138196.ref035" ref-type="bibr"> 35 </xref> ]. The accumulation of &#x003b2;-carotene in the transgenic seeds in this study therefore supports the findings of these two previous studies. The transgenic seeds in this study also accumulated phytoene and &#x003b1;-carotene, which were not detected in the non-transgenic control seeds. Similar results were seen in soybean seeds engineered with <italic> <span class="gene">crtB</span> </italic> alone; these seeds accumulated measurable amounts of phytoene and &#x003b1;-carotene while non-transgenic control seeds did not [ <xref rid="pone.0138196.ref035" ref-type="bibr"> 35 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A comparative analysis of transcriptomic, biochemical, and physiological responses to elevated ozone identifies species-specific mechanisms of resilience in legume crops</h2>
			<h3>PMID:26324463</h3>
				
					<h3>PMCID:4765784</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yendrek </surname>  <given-names> Craig R. </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Species comparison of phenolic-related transcript and content changes in response to elevated [O <sub> 3 </sub> ]. (A) Heat map of transcript abundance changes for <italic> PHENYLANLANINE AMMONIA LYASE </italic> ( <italic> <span class="gene">PAL</span> </italic> ), <italic> CHALCONE SYNTHASE </italic> ( <italic> <span class="gene">CHS</span> </italic> ), <italic> ISOFLAVONE REDUCTASE </italic> ( <italic> IFR </italic> ), and <italic> DIHYDROFLAVONOL 4-REDUCTASE </italic> ( <italic> DFR </italic> ) in leaf tissue collected from the youngest fully expanded leaf at midday after plants were grown in elevated [O <sub> 3 </sub> ] (150 ppb; 8h d <sup> &#x02212;1 </sup> ) for one month. Each coloured block represents the log <sub> 2 </sub> fold change of a separate homologue that was determined to be differentially expressed. (B) Midday total foliar phenolic content of the youngest fully expanded leaf after plants were grown in elevated [O <sub> 3 </sub> ] for one month. Absolute values represent gallic acid equivalents based on a comparison to a standard curve and were normalized to tissue dry <span class="trait">weight.</span> The mean (&#x000b1; SD; n = 3) is presented with letters representing significant differences ( <italic> P </italic> &#x0003c; 0.05).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nodulation Characterization and Proteomic Profiling of <italic> Bradyrhizobium liaoningense </italic> CCBAU05525 in Response to Water-Soluble Humic Materials</h2>
			<h3>PMID:26054030</h3>
				
					<h3>PMCID:4650689</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guo Gao </surname>  <given-names> Tong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan Xu </surname>  <given-names> Yuan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhen Li </surname>  <given-names> Bao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shui Yang </surname>  <given-names> Jin </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tao Wang </surname>  <given-names> En </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li Yuan </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> State Key Laboratory of AgroBiotechnology and MOA Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University </institution> , Beijing 100193,  <country> P. R. China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> College of Life Science, Agricultural University of Hebei </institution> , Baoding 071001,  <country> P. R. China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Escuela Nacional de Ciencias Biol&#x000f3;gicas, Instituto Polit&#x000e9;cnico Nacional </institution> , M&#x000e9;xico D.F. 11340,  <country> Mexico </country>  </aff>
				<br/> 
				年份：<day> 14 </day>  <month> 04 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flavonoids secreted by host <span class="trait">plant</span> induce expression of <italic> <span class="gene">nodD</span> </italic> gene which then regulated the transcription of the structure genes (such as <italic> nodABC </italic> ) to synthesis Nod factor <xref ref-type="bibr" rid="b30"> 30 </xref> . Therefore, they were essential to the symbiosis between the legumes and rhizobia <xref ref-type="bibr" rid="b31"> 31 </xref> . It was shown in this study that WSHM increased <italic> nodD1, nodD2, <span class="gene">nodA</span> </italic> gene expression of <italic> B. liaoningense </italic> CCBAU05525, which might be the reason for the increase in the nodule number and nitrogenase activity of soybean in the greenhouse experiments. The enhanced expression of <italic> nod </italic> genes by humic acids in <italic> B. liaoningense </italic> had not been reported previously. As expected, genistein also increased the expression of <italic> nodD1, nodD2, <span class="gene">nodA</span> </italic> gene. WSHM contained flavonoid analogue, which might account for its contribution to <italic> nodD1, nodD2, <span class="gene">nodA</span> </italic> gene expression. However, WSHM treatment caused a greater increase in the expression of these genes than genistein did, similar to the tendency of the proteins in category 1 shown in proteomics assay, indicating that the effects of WSHM were stronger than those of genistein. However, the higher concentration of WSHM used in the experiment might partially account for the greater effects. In addition, another explanation might be the complex mixture of components present in the WSHM treatment ( <xref ref-type="table" rid="t2"> Table 2 </xref> ), in which there might be some strong inducers for particular metabolic procedures; since <italic> B. liaoningense </italic> CCBAU05525 responded to WSHM and genistein differently in category 2 and 3 in proteomics assay. These strong inducers in the WSHM compounds had great potential in the development of new materials to stimulate the symbiotic nitrogen fixation between rhizobium and legume.
					</p>
				
					<p>&nbsp;&nbsp;
						Conclusively, WSHM treatment increased the number of soybean nodules, the nodule fresh <span class="trait">weight</span> and the nitrogenase activity, as well as enhanced the cell density of <italic> B. liaoningense </italic> CCBAU05525. WSHM contained flavonoid analogues and showed similar bioactivities with genistein but had wider effects than genistein such as inducing the expression of <italic> nodD1, nodD2, <span class="gene">nodA</span> </italic> , and up-regulating a range of proteins involved in a variety of metabolic pathways. Furthermore, several nitrogen fixation-associated proteins such as molybdenum ABC transporter were also affected. Therefore, WSHM might prepare the free-living rhizobia for better bacteroid development. This was the first study about the effects of humic acids on rhizobia growth, <italic> nod </italic> gene expression and proteomics. These results shed new light on the effects of humic material on legumes and demonstrated great application value in improving soybean production.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Bt </italic> -maize (MON810) and Non-GM Soybean Meal in Diets for Atlantic Salmon ( <italic> Salmo salar </italic> L.) Juveniles &#x02013; Impact on Survival, Growth Performance, Development, Digestive Function, and Transcriptional Expression of Intestinal Immune and Stress Responses</h2>
			<h3>PMID:24923786</h3>
				
					<h3>PMCID:4055707</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gu </surname>  <given-names> Jinni </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bakke </surname>  <given-names> Anne Marie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valen </surname>  <given-names> Elin C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lein </surname>  <given-names> Ingrid </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Krogdahl </surname>  <given-names> &#x000c5;shild </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 5 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Inclusion of <span class="trait">plant</span> crops in commercial diets for farmed fish is increasing in an effort to reduce dependence on limited marine resources such as fishmeal and fish oil, and to improve cost efficiency and sustainability of the aquaculture industry. This includes piscivorous species such as Atlantic salmon ( <italic> Salmo salar </italic> L.). A large share of the global market of some <span class="trait">plant</span> crops, such as maize and soybeans, are now genetically modified (GM). To assess its safety, feeding trials with GM <italic> Bt </italic> -maize in post-smolt <xref rid="pone.0099932-Sissener2" ref-type="bibr"> [13] </xref> , <xref rid="pone.0099932-Hemre1" ref-type="bibr"> [25] </xref> &#x02013; <xref rid="pone.0099932-Gu1" ref-type="bibr"> [28] </xref> and juvenile <xref rid="pone.0099932-Sanden1" ref-type="bibr"> [10] </xref> &#x02013; <xref rid="pone.0099932-BakkeMcKellep1" ref-type="bibr"> [11] </xref> , <xref rid="pone.0099932-Sanden3" ref-type="bibr"> [29] </xref> Atlantic salmon diets, as well as in diets for a model fish species, zebrafish ( <italic> Danio rerio </italic> ) <xref rid="pone.0099932-Sissener1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0099932-Sanden2" ref-type="bibr"> [14] </xref> , have been conducted. Although no clearly negative health effects were observed in these studies, the salmon fed <italic> Bt </italic> -maize-containing diets displayed some differences in performance parameters and functional responses compared to control non-GM maize diets, including reduced feed intake and growth despite elevated activity of maltase in mid and distal intestine <xref rid="pone.0099932-Hemre1" ref-type="bibr"> [25] </xref> , and elevated gene expression of stress-related heat shock protein (HSP) 70 and superoxide dismutase (SOD) in the liver <xref rid="pone.0099932-Sagstad1" ref-type="bibr"> [26] </xref> . More recently and within the <span class="gene">GMSAFOOD</span> project, post-smolt (seawater stage) Atlantic salmon fed a 20% inclusion level of <italic> Bt </italic> -maize for 90 days displayed reductions in digestibility of protein and mineral, retention efficiency of lipid and energy, as well as activity of leucine aminopeptidase (LAP) in proximal intestine. Gene expression of the T cell marker CD4 and the cytokine interferon-&#x003b3; in the distal intestine was increased <xref rid="pone.0099932-Gu1" ref-type="bibr"> [28] </xref> . In an effort to explore whether a pre-existing hypersensitivity response would alter responses to <italic> Bt </italic> -maize, experimental groups were included that were simultaneously exposed to extracted soybean meal (SBM) in their diets, which causes an apparently T-cell-mediated type IV hypersensitivity response in the distal intestine of salmon <xref rid="pone.0099932-Baeverfjord1" ref-type="bibr"> [30] </xref> - <xref rid="pone.0099932-Marjara1" ref-type="bibr"> [36] </xref> . The results suggested that <italic> Bt </italic> -maize potentiated oxidative cellular stress in the inflamed intestine, as indicated by increased transcription level of superoxide dismutase (SOD) and heat shock protein 70 (HSP70) <xref rid="pone.0099932-Gu1" ref-type="bibr"> [28] </xref> . However, no specific biomarkers for GM exposure or responses to exposure to use for PMM were identified.
					</p>
				
					<p>&nbsp;&nbsp;
						During the feeding trial, the SBM-fed Atlantic salmon juvenile generally displayed lower cumulative mortality and enhanced growth performance compared to the other diet groups ( <xref ref-type="table" rid="pone-0099932-t003"> Table 3 </xref> ). Total cumulative mortality, as well as mortalities during the periods day 16&#x02013;36 and day 37&#x02013;48 were significantly lower in SBM-fed fish. Condition factor was significantly higher on day 15 and 36 and tended ( <italic> p </italic> &#x0003c;0.10) to be higher at day 48 and 99 compared with non-SBM fed fish. Moreover, significantly higher body <span class="trait">weight,</span> fork length and SGR were observed in SBM-fed fish on day 99. When differences were observed in LAP ( <xref ref-type="fig" rid="pone-0099932-g001"> Fig. 1A&#x02013;C </xref> ) and maltase ( <xref ref-type="fig" rid="pone-0099932-g001"> Fig. 1D&#x02013;F </xref> ) activities in either whole fish or dissected intestinal sections, the SBM-fed groups usually exhibited significantly lower or a trend ( <italic> p </italic> &#x0003c;0.10) towards lower activities, with an exception of significantly higher LAP activity in PI on day 48 ( <italic> p </italic> &#x0200a;=&#x0200a;0.0107; <xref ref-type="fig" rid="pone-0099932-g001"> Fig. 1B </xref> ). Trypsin activity ( <xref ref-type="fig" rid="pone-0099932-g002"> Fig. 2A&#x02013;B </xref> ) in whole fish was significantly higher in fish fed SBM diets on day 36 ( <italic> p </italic> &#x0200a;=&#x0200a;0.0374; <xref ref-type="fig" rid="pone-0099932-g002"> Fig. 2A </xref> ), whereas amylase activity ( <xref ref-type="fig" rid="pone-0099932-g002"> Fig. 2C&#x02013;D </xref> ) was not affected. Bile salt concentration ( <xref ref-type="fig" rid="pone-0099932-g002"> Fig. 2E&#x02013;F </xref> ) tended to be higher in PI on day 48 ( <italic> p </italic> &#x0200a;=&#x0200a;0.0955) but was significantly lower in PI+MI on day 99 ( <italic> p </italic> &#x0200a;=&#x0200a;0.0169) in SBM-fed fish ( <xref ref-type="fig" rid="pone-0099932-g002"> Fig. 2F </xref> ). Histological examination revealed that SBM did not induce inflammation in the DI of Atlantic salmon juveniles. The SBM-fed fish displayed normal morphological structures in DI ( <xref ref-type="fig" rid="pone-0099932-g003"> Fig. 3 </xref> ), with the exception of one of the 24 sampled SBM-fed fish that exhibited mild, focal, inflammatory lesions in the submucosa. No changes were observed in the liver or PI sections, in skeletal morphology ( <xref ref-type="fig" rid="pone-0099932-g004"> Fig. 4 </xref> ), or in transcriptional expression of CD4, IL17a, IFN&#x003b3;, PCNA or <span class="gene">HSP70</span> in the DI ( <xref ref-type="fig" rid="pone-0099932-g005"> Fig. 5 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <italic> <span class="gene">GmMYB73</span> </italic> promotes lipid accumulation in transgenic plants</h2>
			<h3>PMID:24655684</h3>
				
					<h3>PMCID:3998039</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Liu </surname>  <given-names> Yun-Feng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yunfengliu5@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Li </surname>  <given-names> Qing-Tian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qtli@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lu </surname>  <given-names> Xiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qingxinsong@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Song </surname>  <given-names> Qing-Xin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qxsong@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Lam </surname>  <given-names> Sin-Man </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> smlam@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Zhang </surname>  <given-names> Wan-Ke </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wkzhang@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Ma </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mabiao@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Lin </surname>  <given-names> Qing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qlin@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Man </surname>  <given-names> Wei-Qun </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> manweiqun@163.com </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Du </surname>  <given-names> Wei-Guang </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> weiguangdu@126.com </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Shui </surname>  <given-names> Guang-Hou </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ghshui@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A12">  <name>  <surname> Chen </surname>  <given-names> Shou-Yi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sychen@genetics.ac.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A13">  <name>  <surname> Zhang </surname>  <given-names> Jin-Song </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jszhang@genetics.ac.cn </email>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						One MYB-type gene <italic> <span class="gene">GmMYB73</span> </italic> was found to display differential expression in soybean seeds of different developing stages by microarray analysis and was further investigated for its functions in lipid accumulation. <span class="gene">GmMYB73</span> is a small protein with single MYB repeat and has similarity to CPC-like MYB proteins from Arabidopsis. <span class="gene">GmMYB73</span> interacted with GL3 and EGL3, and then suppressed <italic> GL2 </italic> , a negative regulator of oil accumulation. <italic> <span class="gene">GmMYB73</span> </italic> overexpression enhanced lipid contents in both seeds and leaves of transgenic Arabidopsis plants. Seed length and thousand-seed <span class="trait">weight</span> were also promoted. <italic> <span class="gene">GmMYB73</span> </italic> introduction into the Arabidopsis <italic> try cpc </italic> double mutant rescued the total lipids, seed size and thousand-seed <span class="trait">weight.</span> <italic> <span class="gene">GmMYB73</span> </italic> also elevated lipid levels in seeds and leaves of transgenic Lotus, and in transgenic hairy roots of soybean plants. <span class="gene">GmMYB73</span> promoted <italic> PLD&#x003b1;1 </italic> expression, whose promoter can be bound and inhibited by GL2. <italic> PLD&#x003b1;1 </italic> mutation reduced triacylglycerol levels mildly in seeds but significantly in leaves of Arabidopsis plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> <span class="gene">GmMYB73</span> </italic> </bold> <bold> expression. (a) </bold> <italic> <span class="gene">GmMYB73</span> </italic> &#x02009;expression at different stages of developing soybean seeds. Left panel: different stages of soybean seeds; percentages indicate relative seed <span class="trait">weight</span> compared to the full-sized seed without desiccation. Right panel: <italic> <span class="gene">GmMYB73</span> </italic> &#x02009;relative expression in seeds at different stages. <bold> (b) </bold> <italic> <span class="gene">GmMYB73</span> </italic> &#x02009;expression in different organs of soybean plants. Stage five pods were used. Three-week-old seedlings were used for harvest of stem, root and leaf. For gene expression in <bold> (a) </bold> &#x02009;and <bold> (b) </bold> , error bars indicate SD (n&#x02009;=&#x02009;4). <bold> (c) </bold> &#x02009;Fatty acid contents in seeds at different developmental stages. Total contents are derived from the content of five individual fatty acids. The values are in dry <span class="trait">weight.</span> Error bars indicate SD (n&#x02009;=&#x02009;4).
					</p>
				
					<p>&nbsp;&nbsp;
						To study <italic> <span class="gene">GmMYB73</span> </italic> functions, we generated the construct harboring <italic> <span class="gene">GmMYB73</span> </italic> controlled by CaMV 35S promoter in pPROK II vector and transformed this gene into Arabidopsis plants using Agrobacterium-mediated floral dip transformation method. <italic> <span class="gene">GmMYB73</span> </italic> expression was examined (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ) in homozygous transgenic lines (OE-1, 2, 5, 7, and 10) and <span class="trait">plant</span> phenotypes were investigated.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine whether <span class="gene">GmMYB73</span> affects <italic> GL2 </italic> expression, transgenic Arabidopsis plants harboring <italic> GL2 promoter::GUS </italic> ( <italic> GL2 </italic> <sub> <italic> p </italic> </sub> <italic> ::GUS </italic> ) [ <xref ref-type="bibr" rid="B48"> 48 </xref> ] were crossed with the transgenic plants overexpressing <italic> <span class="gene">GmMYB73</span> </italic> (OE-5) plants, and GUS staining was examined in homozygous <italic> GL2 </italic> <sub> <italic> p </italic> </sub> <italic> ::GUS/GmMYB73 </italic> transgenic plants. In the <italic> GL2 </italic> <sub> <italic> p </italic> </sub> <italic> ::GUS </italic> <span class="trait">plant</span> <italic> , </italic> GUS staining was observed in hypocotyl, root, shoot apex and trichomes of flower buds, leaf and stem (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> c, d). GUS staining was strongly inhibited by <span class="gene">GmMYB73</span> in hypocotyl and shoot apex in <italic> GL2 </italic> <sub> <italic> p </italic> </sub> <italic> ::GUS/GmMYB73 </italic> plants (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> c). Trichomes and GUS staining were barely detectable in flower buds, leaf and stem of these plants (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> d). GUS staining was only partially suppressed in roots of <italic> GL2 </italic> <sub> <italic> p </italic> </sub> <italic> ::GUS/GmMYB73 </italic> plants (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> c). GUS activity was also quantified in leaves, hypocotyls and roots of above transgenic plants, and the levels were consistent with the GUS staining results (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> c, d, e). These results indicate that <span class="gene">GmMYB73</span> negatively regulates trichome formation by interacting with GL3 and EGL3 to repress <italic> GL2 </italic> transcription in transgenic Arabidopsis plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> <span class="gene">GmMYB73</span> </italic> </bold> <bold> overexpression controls seed size and thousand-seed <span class="trait">weight</span> of transgenic Arabidopsis plants. (a) </bold> &#x02009;Morphology of Arabidopsis seeds under scanning electron microscope. Seeds from <italic> <span class="gene">GmMYB73</span> </italic> -overexpressing Arabidopsis plants (OE), <italic> gl2-2 </italic> , <italic> try cpc </italic> &#x02009;and <italic> try cpc/GmMYB73 </italic> &#x02009;lines were used. <bold> (b) </bold> &#x02009;Comparison of seed length from various <span class="trait">plant</span> seeds. Error bars indicate SD (n&#x02009;=&#x02009;20&#x02009;~&#x02009;30). The values from <italic> try cpc/GmMYB73 </italic> &#x02009;line are only compared with those from the <italic> try cpc </italic> mutant. &#x02018;**&#x02019; and &#x02018;*&#x02019; above the columns indicate a significant difference from Col-0 or between the compared pairs at P&#x02009;&#x0003c;&#x02009;0.01 and P&#x02009;&#x0003c;&#x02009;0.05, respectively. <bold> (c) </bold> Comparison of seed width from various A. thaliana lines. Others are as in <bold> (b) </bold> . <bold> (d) </bold> &#x02009;Comparison of the ratio of seed length to seed width. Others are as in <bold> (b) </bold> . <bold> (e) </bold> &#x02009;Comparison of thousand-seed <span class="trait">weight</span> in various A. thaliana lines. Error bars indicate SD (n&#x02009;=&#x02009;4). Others are as in <bold> (b) </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						Thousand-seed weights were also measured and the five <italic> <span class="gene">GmMYB73</span> </italic> -overexpressing lines had significantly higher thousand-seed weights than Col-0 (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> e). The <italic> gl2 </italic> seeds also had slightly but significantly higher levels of the parameter (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> e). In contrast, <italic> try cpc </italic> double mutant had significantly lower thousand-seed <span class="trait">weight</span> than Col and introduction of <italic> <span class="gene">GmMYB73</span> </italic> recovered the level in <italic> try cpc/GmMYB73 </italic> plants (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> e). These results indicate that <span class="gene">GmMYB73,</span> its homologues TRY and CPC, and GL2 regulate seed development.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> <span class="gene">GmMYB73</span> </italic> </bold> <bold> increases lipid contents in seeds and leaves of transgenic Arabidopsis plants. (a) </bold> &#x02009;Total lipid contents in seeds of Col, <italic> <span class="gene">GmMYB73</span> </italic> -transgenic plants (OE-1, 2, 5, 7, 10) <italic> gl2-2 </italic> , <italic> try cpc </italic> , and <italic> try cpc/GmMYB73 </italic> . Error bars indicate SD (n&#x02009;=&#x02009;4) and the values are in dry <span class="trait">weight</span> for seeds. The values from <italic> try cpc/GmMYB73 </italic> line are only compared with those from the <italic> try cpc </italic> &#x02009;mutant. Asterisks indicate a significant difference from Col-0 or between the compared pairs (*P&#x02009;&#x0003c;&#x02009;0.05 and **P&#x02009;&#x0003c;&#x02009;0.01). <bold> (b) </bold> &#x02009;Contents of total fatty acids in seeds of various plants. Error bars indicate SD (n&#x02009;=&#x02009;4). Others are as in <bold> (a) </bold> . <bold> (c) </bold> &#x02009;Compositions of fatty acids in seeds of various plants. Error bars indicate SD (n&#x02009;=&#x02009;4). Others are as in <bold> (a) </bold> . <bold> (d) </bold> &#x02009;Total fatty acids in <span class="trait">plant</span> leaves. Error bars indicate SD (n&#x02009;=&#x02009;4) and the values are in dry <span class="trait">weight.</span> Others are as in <bold> (a) </bold> . <bold> (e) </bold> Compositions of fatty acids in <span class="trait">plant</span> leaves. Error bars indicate SD (n&#x02009;=&#x02009;4) and the values are in dry <span class="trait">weight.</span> Others are as in <bold> (a) </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						Fatty acid levels were measured in leaves of various <span class="trait">plant</span> lines. <italic> <span class="gene">GmMYB73</span> </italic> -overexpressing lines had significantly higher total fatty acid contents in leaf compared to Col-0 (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> d). Level of each fatty acid compostion, except 18:0, was also significantly or slightly increased in the transgenic lines compared to the levels in Col-0 (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> e). These results indicate that <italic> <span class="gene">GmMYB73</span> </italic> increased contents of total lipids and total fatty acids in seeds and leaves of transgenic Arabidopsis plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is a legume <span class="trait">plant</span> and we further transformed the <italic> <span class="gene">GmMYB73</span> </italic> into the legume Lotus japonicus (Leo) plants. Two transgenic lines were identified and both displayed <italic> <span class="gene">GmMYB73</span> </italic> expressions compared to WT (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> a). Total lipids and total fatty acids in seeds of the two lines were significantly increased compared to WT plants (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> b, c). As for the fatty acid composition, only two fatty acids (18:2 and 18:3) showed significant increase in the transgenic seeds (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> d). Fatty acid levels in leaves of the transgenic plants were also determined and we found that total fatty acids and three fatty acids (16:0, 18:2 and 18:3) were apparently enhanced (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> e, f). These results indicate that <italic> <span class="gene">GmMYB73</span> </italic> increases total lipid and total fatty acid contents in seeds and leaves of Lotus transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> <span class="gene">GmMYB73</span> </italic> </bold> <bold> enhances lipid contents in seeds and leaves of transgenic Lotus plants, and in transgenic hairy roots of soybean plants. (a) </bold> <italic> <span class="gene">GmMYB73</span> </italic> &#x02009;expression in leaves of transgenic Lotus plants. Two lines OE-7 and OE-21 were used. Tubulin gene was amplified as a control. <bold> (b) </bold> Contents of total lipids in transgenic seeds compared to WT. Error bars indicate SD (n&#x02009;=&#x02009;4) and the values are in dry <span class="trait">weight.</span> Asterisks indicate significant difference compared to WT (**P&#x02009;&#x0003c;&#x02009;0.01, *P&#x02009;&#x0003c;&#x02009;0.05). <bold> (c) </bold> &#x02009;Contents of total fatty acids in transgenic seeds. Error bars indicate SD (n&#x02009;=&#x02009;4) and others are as in <bold> (b) </bold> . <bold> (d) </bold> &#x02009;Contents of each fatty acid composition in transgenic seeds. Others are as in <bold> (b) </bold> . <bold> (e) </bold> &#x02009;Contents of total fatty acids in leaves of transgenic Lotus plants. The values are in fresh <span class="trait">weight.</span> Others are as in (b). <bold> (f) </bold> &#x02009;Contents of each fatty acid in leaves of transgenic Lotus plants. The values are in fresh <span class="trait">weight.</span> Others are as in <bold> (b) </bold> . <bold> (g) </bold> <italic> <span class="gene">GmMYB73</span> </italic> expression in transgenic hairy roots of soybean plants. K599: control roots. <span class="gene">GmMYB73:</span> <italic> <span class="gene">GmMYB73</span> </italic> -transgenic hairy roots. Error bars indicate SD (n&#x02009;=&#x02009;4). Asterisk indicates significant difference compared to control K599 (*P&#x02009;&#x0003c;&#x02009;0.05). <bold> (h) </bold> &#x02009;Total fatty acid levels in <italic> <span class="gene">GmMYB73</span> </italic> -transgenic hairy roots. K599: control roots. <span class="gene">GmMYB73:</span> <italic> <span class="gene">GmMYB73</span> </italic> -transgenic hairy roots. <span class="gene">GmDof4:</span> <italic> <span class="gene">GmDof4</span> </italic> -transgenic hairy roots as a positive control for lipid accumulation. Error bars indicate SD (n&#x02009;=&#x02009;4). The values are in fresh <span class="trait">weight.</span> Asterisks indicate significant difference compared to control K599 (**P&#x02009;&#x0003c;&#x02009;0.01, *P&#x02009;&#x0003c;&#x02009;0.05). <bold> (i) </bold> Levels of each fatty acid composition in <italic> <span class="gene">GmMYB73</span> </italic> -transgenic hairy roots. The values are in fresh <span class="trait">weight.</span> Other indications are as in <bold> (h) </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">GmMYB73</span> </italic> enhanced seed size and thousand-seed-weight of Arabidopsis transgenic plants (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> e). The <italic> <span class="gene">GmMYB73</span> </italic> also rescued the seed phenotype and thousand-seed <span class="trait">weight</span> in <italic> try cpc </italic> double mutant (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ). These results indicate that <italic> <span class="gene">GmMYB73</span> </italic> or its homologues in Arabidopsis regulates seed development in addition to control of trichome formation. Several other genes involved in regulation of trichome formation also affect seed development. These include <italic> TRANSPARENT TESTA GLABRA2 </italic> [ <xref ref-type="bibr" rid="B53"> 53 </xref> ] and <italic> TRANSPARENT TESTA GLABRA1 </italic> [ <xref ref-type="bibr" rid="B54"> 54 </xref> ] in seed coat differentiation; <italic> gl2 </italic> mutants do not produce releasable mucilage [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]; <italic> GL3 </italic> , <italic> EGL3 </italic> , and <italic> TTG1 </italic> regulate seed coat proanthocyanidin biosynthesis [ <xref ref-type="bibr" rid="B55"> 55 </xref> , <xref ref-type="bibr" rid="B56"> 56 </xref> ]. Some of these genes also play roles in seed size control and thousand-seed <span class="trait">weight</span> regulation [ <xref ref-type="bibr" rid="B57"> 57 </xref> ]. It should be mentioned that <italic> gl2-1 </italic> seeds (Ler ecotype) show no change in thousand-seed <span class="trait">weight</span> when compared to WT [ <xref ref-type="bibr" rid="B57"> 57 </xref> ]. In contrast, the present <italic> gl2-2 </italic> (Col ecotype) exhibited slight but significant increase in thousand-seed <span class="trait">weight</span> when compared to control (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> e). This discrepancy may be derived from different ecotypes used and/or different storage time for seeds. There are other genes that regulate both seed development and aspects of <span class="trait">plant</span> growth and development. Inhibition of ethylene receptor gene <italic> OsETR2 </italic> expression in rice leads to increased thousand-grain <span class="trait">weight</span> and early flowering [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. Mutation of <italic> MHZ7/OsEIN2 </italic> affects grain shape and senescence [ <xref ref-type="bibr" rid="B59"> 59 </xref> ]. Ethylene and salt-induced NIMA-related kinase NEK6 enhances <span class="trait">plant</span> growth and seed <span class="trait">yield</span> in Arabidopsis; however, thousand-seed <span class="trait">weight</span> and seed width are reduced [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. Recently, two <italic> DOF </italic> genes <italic> DOF4.2 </italic> and <italic> DOF4.4 </italic> have been found to promote shoot <span class="trait">branching</span> but affect seed/silique development in Arabidopsis [ <xref ref-type="bibr" rid="B61"> 61 </xref> ]. Other genes involved in regulating seed size and/or development are reviewed in [ <xref ref-type="bibr" rid="B62"> 62 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmMYB73</span> </italic> overexpression promotes fatty acid accumulation in transgenic Arabidopsis and transgenic Lotus plants (Figures&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> and <xref ref-type="fig" rid="F5"> 5 </xref> ). The gene also fully rescued the total lipid level and partially rescued the total fatty acid level in <italic> try cpc </italic> double mutant (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> ), indicating that the <italic> <span class="gene">GmMYB73</span> </italic> is involved in upregulation of fatty acid accumulation. It should be noted that the increase of total fatty acid levels in seeds of the <italic> <span class="gene">GmMYB73</span> </italic> -transgenic Arabidopsis plants was likely not due to the increase in any specific fatty acid but rather due to the overall increase in each fatty acid level (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> b, c). However, in leaves of the transgenic plants, the increase of the total fatty acids was due to an increase in linolenic acid (18:3) (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> d, e). The difference in the accumulation patterns of fatty acids in seed and leaf tissues may be due the physiological and metabolic difference of these organs, where seed is a sink organ, while the leaf is a source organ. In <italic> <span class="gene">GmMYB73</span> </italic> -transgenic Lotus plants, the increase of total fatty acids in leaves is mainly due to increase of linolenic acid (18:3) (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> f), similar to the case in leaves of Arabidopsis transgenic plants (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> e). However, the seeds of transgenic Lotus plants showed a different change in fatty acid compositions compared to seeds of transgenic Arabidopsis, and the increase of total fatty acids was attributed to an increase in linolic acid (18:2) and/or linolenic acid (18:3) (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> ). The different accumulation patterns of fatty acid compositions in seeds of transgenic Arabidopsis and Lotus plants suggest that some different mechanisms may be affected in lipid accumulation in these <span class="trait">plant</span> species.
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the difficulty of soybean <span class="trait">plant</span> transformation, we adopted a method for generating transgenic hairy roots in soybean. <italic> <span class="gene">GmMYB73</span> </italic> overexpression in soybean transgenic hairy roots increased the total fatty acid levels and this elevation was likely due to the increase in linolic acid (18:2) and linolenic acid (18:3) levels (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> h, i). It should be mentioned that the changes in fatty acid composition in soybean transgenic hairy roots were very similar to those in leaves of transgenic Arabidopsis plants (Figures&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> e and <xref ref-type="fig" rid="F5"> 5 </xref> i), suggesting that similar mechanisms may have been affected in vegetative organs of Arabidopsis and soybean. It is possible that <italic> <span class="gene">GmMYB73</span> </italic> would promote lipid accumulation in soybean seeds as it did in Arabidopsis seeds although further studies are needed to confirm this.
					</p>
				
					<p>&nbsp;&nbsp;
						PLD hydrolyses the P-O bond of phosphatidylcholine (PC) to produce phosphatidic acid (PA) and choline. The PA is converted to 1,2-sn-diacylglycerol (DAG) [ <xref ref-type="bibr" rid="B64"> 64 </xref> ] by the action of PA phosphatase and then DAG can be acylated to produce triacylglycerol (TAG). In developing soybean seeds, the major pathway for TAG formation is through conversion of PC to DAG and acylation of DAG to produce TAG [ <xref ref-type="bibr" rid="B65"> 65 </xref> ]. Recently, Lee <italic> et al </italic> . [ <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ] reported that the lipid profile was changed by suppression of <italic> PLD&#x003b1; </italic> in the soybean seeds and the total lipids and TAG signals tended to decrease in fresh seeds of <italic> PLD&#x003b1;- </italic> knockdown soybean. Presently, we find that <italic> PLD&#x003b1;1 </italic> mutation in Arabidopsis substantially reduced the TAG levels in both seeds and leaves (Tables&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> and <xref ref-type="table" rid="T2"> 2 </xref> ). In contrast, <italic> <span class="gene">GmMYB73</span> </italic> -transgenic Arabidopsis plants substantially had more TAG in seeds and leaves compared to Col-0 (Tables&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> and <xref ref-type="table" rid="T2"> 2 </xref> ). It is noted that in seeds, most of the lipid are TAG, whereas in leaves, the major lipid species are PC and PA (Tables&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> and <xref ref-type="table" rid="T2"> 2 </xref> ). The roughly negative correlation between PA and PC contents in leaves of both <italic> <span class="gene">GmMYB73</span> </italic> -transgenic plants and <italic> pld&#x003b1;1 </italic> mutant (Tables&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> and <xref ref-type="table" rid="T2"> 2 </xref> ) suggests that <span class="gene">GmMYB73</span> may finally promote PA and possible DAG and TAG accumulation through PC conversion by PLD&#x003b1;1 function. Similar case may happen in <italic> <span class="gene">GmMYB73</span> </italic> -transgenic seeds although PA levels were not high, possibly due to an efficient conversion to DAG and TAG. All these data suggest that <span class="gene">GmMYB73</span> may enhance lipid accumulation at least partially through repression of <italic> GL2 </italic> and promotion of <italic> PLD&#x003b1;1 </italic> . Other downstream genes (e.g., <italic> MUM4 </italic> ) may also be involved in this process [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. It should be mentioned that the <italic> <span class="gene">GmMYB73</span> </italic> -overexpresing Arabidopsis plants have no or very little trichomes. Considering that trichome contained a multitude of wax components and can secrete lipids [ <xref ref-type="bibr" rid="B66"> 66 </xref> ], the lack of trichome may save total energy and resources for lipid biosynthesis in other organs such as seeds and leaves of transgenic plants. Recently, <italic> PLD&#x003b1;1 </italic> overexpression has been found to improve <span class="trait">drought</span> tolerance and increase seed <span class="trait">yield</span> [ <xref ref-type="bibr" rid="B67"> 67 </xref> ], lending support to our present study.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Using RNA-Seq to Profile Soybean Seed Development from Fertilization to Maturity</h2>
			<h3>PMID:23555009</h3>
				
					<h3>PMCID:3598657</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jones </surname>  <given-names> Sarah I. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Likewise, about 30% of the genes in <xref ref-type="fig" rid="pone-0059270-g001"> Figure 1C </xref> are annotated as proline-rich proteins, which are involved in cell wall structure in plants. The high expression of these genes in early stages may be due to increased cell growth and development at this time. This annotation is also highly over-represented in this cluster, with &#x0003c;1% of the 78,773 soybean gene models described as encoding proline-rich proteins. Other studies have shown soybean proline-rich protein gene <italic> <span class="gene">PRP1</span> </italic> to be highly expressed in developing seed coats up to 50 mg fresh <span class="trait">weight</span> <xref ref-type="bibr" rid="pone.0059270-Hong1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0059270-Lindstrom1"> [12] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of Candidate Reference Genes for Normalization of Quantitative RT-PCR in Soybean Tissues under Various Abiotic Stress Conditions</h2>
			<h3>PMID:23029532</h3>
				
					<h3>PMCID:3460875</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Le </surname>  <given-names> Dung Tien </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Aldrich </surname>  <given-names> Donavan L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Yasuko </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ha </surname>  <given-names> Chien Van </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishiyama </surname>  <given-names> Rie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guttikonda </surname>  <given-names> Satish K. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Quach </surname>  <given-names> Truyen N. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gutierrez-Gonzalez </surname>  <given-names> Juan J. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn3">  <sup> &#x000a4;c </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 2 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						ABA has been established as a key hormone involved in the regulation of <span class="trait">plant</span> responses to various abiotic stresses, such as <span class="trait">drought</span> and high salinity. Under these stresses, endogenous ABA level is increased, leading to up-regulation of many stress-responsive genes in ABA-dependent manner <xref ref-type="bibr" rid="pone.0046487-Fujita1"> [62] </xref> . Thus, ABA treatment is often included in expression studies together with other stress treatments to determine whether the change in gene expression in response to stresses is ABA-dependent or ABA-independent <xref ref-type="bibr" rid="pone.0046487-Tran8"> [39] </xref> , <xref ref-type="bibr" rid="pone.0046487-Gao1"> [51] </xref> , <xref ref-type="bibr" rid="pone.0046487-Nishiyama2"> [52] </xref> . Hence, a good reference gene for expression studies of stress-responsive genes under abiotic stresses should also stably express under ABA treatment. This prompted us to rank the performance of the selected 13 candidate reference genes to obtain the most stably expressed genes in roots and shoots that had been subjected to exogenous ABA treatment. The five top-listed genes in the roots and shoots under ABA treatment as revealed by delta CT analysis were <italic> 60s </italic> &#x0003e; <italic> ELF1b </italic> &#x0003e; <italic> ABC </italic> &#x0003e; <italic> ELF1a </italic> &#x0003e; <italic> CYP2 </italic> and <italic> ELF1b </italic> &#x0003e; <italic> Fbox </italic> &#x0003e; <italic> ABC </italic> &#x0003e; <italic> Act11 </italic> &#x0003e; <italic> <span class="gene">CDPK</span> </italic> , respectively ( <xref ref-type="table" rid="pone-0046487-t003"> Table 3 </xref> ). Only two genes, namely the <italic> ABC </italic> and <italic> ELF1b </italic> , were overlapping in the lists of the top five most stably expressed genes in each tissue, suggesting that the most stable reference genes are not only stress/hormone-dependent but also tissue-dependent. When geNorm was use to analyze the expression stability, we found that all the top five genes listed for the roots ( <italic> ABC </italic> / <italic> 60s </italic> &#x0003e; <italic> ELF1a </italic> &#x0003e; <italic> ELF1b </italic> &#x0003e; <italic> CYP2 </italic> ) were also ranked as the top five by a geNorm analysis although the order was different ( <xref ref-type="fig" rid="pone-0046487-g002"> Figure 2D </xref> , <xref ref-type="table" rid="pone-0046487-t003"> Table 3 </xref> ). As for the shoots, with the exception of the <italic> ABC </italic> , the other four top-listed genes determined by delta CT analysis were found among the top five genes as suggested by a geNorm analysis ( <italic> Act11 </italic> / <italic> Fbox </italic> &#x0003e; <italic> ELF1b </italic> &#x0003e; <italic> IDE </italic> &#x0003e; <italic> <span class="gene">CDPK</span> </italic> ) ( <xref ref-type="fig" rid="pone-0046487-g003"> Figure 3D </xref> , <xref ref-type="table" rid="pone-0046487-t003"> Table 3 </xref> ). In addition, we also searched for reference genes that can be used to compare gene expression across tissues under ABA treatment. The data for roots and shoots were combined and analyzed using the delta CT approach. Results on <xref ref-type="table" rid="pone-0046487-t003"> Table 3 </xref> demonstrated that the top five most stably expressed genes in both roots and shoots of soybean seedlings were <italic> 60s </italic> &#x0003e; <italic> ELF1b </italic> &#x0003e; <italic> ABC </italic> &#x0003e; <italic> Fbox </italic> &#x0003e; <italic> ELF1a </italic> . The same five genes were also identified by geNorm, although the exact order was different ( <xref ref-type="fig" rid="pone-0046487-g004"> Figure 4D </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Upon exposure to stresses, soybean plants activate numerous signaling pathways to respond to the adverse environmental stimuli <xref ref-type="bibr" rid="pone.0046487-Tran1"> [1] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0046487-Thao1"> [3] </xref> . The first step toward adapting to stresses is to induce or repress the expression of various genes. To accurately understand the mechanisms regulating stress responses at transcriptional level and to identify the appropriate stress-responsive genes, stably expressed reference genes are needed if qRT-PCR is used for expression profiling. Before this study was conducted, there have been no comprehensive reports on the search for reference genes for soybean under various abiotic stress conditions, except a study which surveyed for reference genes for expression study under <span class="trait">drought</span> stress in soybean. However, this study used only four candidate genes, namely the <italic> <span class="gene">Gm&#x003b2;-actin</span> </italic> , <italic> <span class="gene">GmGADPH</span> </italic> , <italic> <span class="gene">GmLectin</span> </italic> and <italic> <span class="gene">GmRNA18s</span> </italic> , among which the <italic> <span class="gene">Gm&#x003b2;-actin</span> </italic> and <italic> <span class="gene">GmRNA18s</span> </italic> were found to be the best reference genes for soybean under their experimental conditions <xref ref-type="bibr" rid="pone.0046487-StolfMoreira2"> [47] </xref> . This prompted us to choose 13 candidate reference genes from literature and ranked their performance as the best reference genes under various abiotic stress and hormonal treatments, including dehydration, salt, cold and ABA treatments. Because most of the recent studies showed the preferable use of root and shoot tissues of soybean seedlings for gene expression profiling under abiotic stresses using qRT-PCR <xref ref-type="bibr" rid="pone.0046487-Tran8"> [39] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0046487-Le3"> [41] </xref> , <xref ref-type="bibr" rid="pone.0046487-StolfMoreira3"> [55] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0046487-Wei1"> [58] </xref> ; we tested the expression stability of the 13 selected candidate genes in these tissues of 12-day old soybean plants. Our results revealed that there is no single reference gene that can be best for all conditions and/or both the tissues. Instead, we found that the best reference genes are tissue- and/or stress-dependent. In addition, although there is high agreement in overall among the top five genes ranked by either delta CT analysis or geNorm, the best reference genes inferred by one method may not be exactly the same ones as determined by another approach; a phenomenon which has also been observed earlier <xref ref-type="bibr" rid="pone.0046487-Kortner1"> [63] </xref> , <xref ref-type="bibr" rid="pone.0046487-MehdiKhanlou1"> [64] </xref> . As geNorm has been known to rank co-regulated genes as the best reference genes, our conclusions were made based primarily on the result of delta CT analysis. The result obtained by geNorm is provided to readers as additional information source. The most stably expressed reference genes identified in this study were shown to help detect subtle differential rates of gene expression as well as avoid the underestimation of the induction/repression levels.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant Products Affect Growth and Digestive Efficiency of Cultured Florida Pompano ( <italic> Trachinotus carolinus </italic> ) Fed Compounded Diets</h2>
			<h3>PMID:22536344</h3>
				
					<h3>PMCID:3335036</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lech </surname>  <given-names> Gregory P. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reigh </surname>  <given-names> Robert C. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 3 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The quantity of nitrogen-free extract (carbohydrates) and fiber in a prepared diet often increases with the amount of <span class="trait">plant</span> products in the formulation. Diet digestibility can be negatively affected as <span class="trait">plant</span> products replace ingredients of animal origin and the ratio of <span class="trait">plant</span> products to animal products increases. <xref ref-type="fig" rid="pone-0034981-g002"> Figure 2 </xref> illustrates the relationship between the calculated apparent crude protein digestibility (ACPD) of canola meal (CM) and corn gluten meal (CGM) in studies with Florida pompano and other carnivorous marine fishes ( <xref ref-type="supplementary-material" rid="pone.0034981.s001"> Table S1 </xref> ). Curves represent a second order polynomial. A high ratio of <span class="trait">plant</span> protein to animal protein (i.e., higher quantity of <span class="trait">plant</span> products) in the reference diet is associated with reduced <span class="gene">ACPD</span> in the test ingredient. This suggests that reference diet composition can affect digestibility measurements and should be considered when evaluating ingredients to provide the most useful results. Accuracy of calculated digestibility coefficients may be improved by using a reference diet with composition similar to the practical formulation in which the test ingredient will be used.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">ACPD</span> of CM and CGM in this study &#x02014; 39 percent and 57 percent, respectively &#x02014; were considerably lower than <span class="gene">ACPD</span> coefficients previously reported for these ingredients fed to Florida pompano and other marine species ( <xref ref-type="fig" rid="pone-0034981-g002"> Figure 2 </xref> ). Reference diets reported by Glencross et al. <xref ref-type="bibr" rid="pone.0034981-Glencross1"> [20] </xref> , Riche and Williams <xref ref-type="bibr" rid="pone.0034981-Riche2"> [33] </xref> , Burel et al. <xref ref-type="bibr" rid="pone.0034981-Burel2"> [48] </xref> , Lee <xref ref-type="bibr" rid="pone.0034981-Lee1"> [49] </xref> , Tibbetts et al. <xref ref-type="bibr" rid="pone.0034981-Tibbetts1"> [50] </xref> , <xref ref-type="bibr" rid="pone.0034981-Tibbetts2"> [51] </xref> , and Zhou et al. <xref ref-type="bibr" rid="pone.0034981-Zhou2"> [52] </xref> were formulated to contain high (greater than 60 percent) concentrations of dietary protein from animal sources and therefore low levels of crude protein from <span class="trait">plant</span> materials. In the current study, however, animal (fish meal) protein and <span class="trait">plant</span> (soybean and wheat) protein sources provided similar quantities of dietary protein. This suggests that the quantity of plant-protein supplements in a reference diet may affect the apparent digestibility of protein in a test ingredient mixed with that diet in a 70/30 ratio. Thus, it is possible that reference diets containing high levels of crude protein from <span class="trait">plant</span> ingredients may <span class="trait">yield</span> lower <span class="gene">ACPD</span> coefficients for test ingredients combined with them than reference diets that contain high levels of animal protein products. It is not possible to prove this hypothesis with the data collected in the current study, but a relationship between reference diet composition and digestibility of protein in <span class="trait">plant</span> products fed to Florida pompano is indicated by the results of the current study.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Use of pJANUS&#x02122;-02-001 as a calibrator plasmid for Roundup Ready soybean event GTS-40-3-2 detection: an interlaboratory trial assessment</h2>
			<h3>PMID:20016879</h3>
				
					<h3>PMCID:2836459</h3>
					<h4>期刊:Analytical and Bioanalytical Chemistry</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lievens </surname>  <given-names> A. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bellocchi </surname>  <given-names> G. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Bernardi </surname>  <given-names> D. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moens </surname>  <given-names> W. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Savini </surname>  <given-names> C. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mazzara </surname>  <given-names> M. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Van den Eede </surname>  <given-names> G. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Van den Bulcke </surname>  <given-names> M. </given-names>  </name>  <address>  <email> marc.van-den-bulcke@ec.europa.eu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Division of Biosafety and Biotechnology, Scientific Institute of Public Health, J. Wytsmanstreet 14, 1050 Brussels, Belgium  </aff>  <aff id="Aff2">  <label> 2 </label> Molecular Biology and Genomics Unit, Institute for Health and Consumer Protection, European Commission Joint Research Centre, Via E. Fermi 2749, 21027 Ispra (VA), Italy  </aff>
				<br/> 
				年份：<day> 24 </day>  <month> 11 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Most commonly used calibrators for quantitative <span class="gene">GMO</span> analysis are DNA solutions extracted from certified reference powders obtained from admixed GM grain samples. These powders are available (e.g. from the Institute of Reference Materials and Measurements, European Commission Joint Research Centre, Geel, Belgium [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]), and are certified for a certain <span class="trait">weight</span> percentage of a specific GM event, hence their name &#x0201c;certified reference materials&#x0201d; (CRMs). A number of drawbacks are, however, connected with the use of such CRMs. Firstly, their production from raw materials (mostly grains) necessitates a guaranteed permanent supply of adequate <span class="trait">plant</span> material from the <span class="gene">GMO</span> developer/notifier. Secondly, their manufacturing is tedious and expensive. Guaranteed supplies may be difficult for newly authorized <span class="gene">GMOs</span> and for <span class="gene">GMOs</span> that are being withdrawn from the commercial market. To date, only a limited working range (0&#x02013;5%) of CRMs is available for only a restricted number of GM events [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Another critical issue connected with grain-derived CRM powders is the weight-percentage-based certification. In hybrid crops (such as maize and cotton), the application of the recommended <span class="gene">GMO</span> unit, the so-called haploid genome equivalent (being the percentage of GM target sequences per taxon specific sequence) (EC decision 2004/787 [ <xref ref-type="bibr" rid="CR13"> 13 </xref> ]), may be biased in ploidy of the transgene according to the parental donor of the transgene copies (of either male or female origin). Finally, the use of CRMs as a source of calibrator standards represents a considerable cost in quantitative <span class="gene">GMO</span> analysis for the laboratories.
					</p>
				
					<p>&nbsp;&nbsp;
						To limit bias in the Q-PCR determination, the reference material must fulfil the following characteristics: (1) the zygosity of the GM material and the wild-type material with respect to the applied targets has to be well established or homozygous by nature, (2) the GM material and the wild-type material should preferentially be isogenic and (3) the DNA should not be degraded to prevent bias by extensive loss of target copies. In most plants, leaf tissue obtained from isogenic GM and wild-type soybean plants would fulfil the first two requirements. For this, plants from both isogenic wild-type and Roundup Ready soybean were grown and leaf tissue was used to isolate the genomic DNA. CTAB-based methods are generally recognized as suitable for extracting soybean DNA for <span class="gene">GMO</span> quantification purposes [ <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. Indeed, agarose gel electrophoresis showed that the CTAB-purified genomic DNA from both wild-type and Roundup Ready soybean leaves migrates as a high molecular <span class="trait">weight</span> band (estimated at about 20&#x000a0;kb), without the presence of major degradation (visible as background smears) (data not shown). The homozygosis of the GM soybean material with respect to the transgene was tested by Q-PCR analysis with both plasmid and genomic DNA as a template (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). At an expected 1:1 ratio and assuming equimolar binding of SYBR&#x000ae; Green molecules to both amplicons, the &#x02206;Ct between endogene and transgene in plasmid and genomic DNA templates should be similar, taking into account PCR efficiencies for the different systems. The results of the Q-PCR analysis with the Q-PCR system of Terry et al. [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] point out that there is a 1:1 ratio between transgene and endogene targets in the Roundup Ready soybean leaf tissue genomic DNA.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of putative reference genes for gene expression normalization in soybean by quantitative real-time RT-PCR</h2>
			<h3>PMID:19785741</h3>
				
					<h3>PMCID:2761916</h3>
					<h4>期刊:BMC Molecular Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Ruibo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ruibohu@yahoo.com.cn </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Chengming </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> fanchengming@gmail.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hongyu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lihongyu7001@yahoo.com.cn </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Qingzhu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> qzhzhang@yahoo.com.cn </email>  </contrib>  <contrib id="A5" corresp="yes" contrib-type="author">  <name>  <surname> Fu </surname>  <given-names> Yong-Fu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> fuyf@caas.net.cn </email>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 9 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is the leading legume crop, and has been used as a model <span class="trait">plant</span> in the context of the flowering response to photoperiod. Many of these studies have used <italic> <span class="gene">TUB</span> </italic> and/or <italic> <span class="gene">ACT</span> </italic> as a reference gene (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ). A literature search based on the keywords "soybean" and "gene expression" produced 54 hits in PubMed (publication period 2001 to 2009). In 23 of these studies (43%), <italic> <span class="gene">TUB</span> </italic> was the reference gene, in 15 of them (28%) <italic> <span class="gene">ACT</span> </italic> , and in six (11%) <italic> 18SrRNA </italic> . All of the studies surveyed used one single reference gene and no preliminary validations were performed (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ). To date, only a limited number of statistically validated reference genes have been identified in soybean. A comparison of the performance of ten conventional housekeeping genes across 21 soybean samples allowed the identification of a panel of genes suitable for gene expression normalization [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. However, the limited number of samples tested meant that a full representation of developmental stages and tissues/organs could not be achieved; instead, a set of new reference genes, chosen to exhibit constancy of expression over a range of experimental conditions, was mined from multiple soybean microarray datasets [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. In the present report, we compare the performance of seven commonly used housekeeping genes and seven of these new reference genes across a large set of biological samples representing various developmental stages, tissues, photoperiod treatments and cultivars of soybean. The recently released soybean whole genome sequence [ <xref ref-type="bibr" rid="B42"> 42 </xref> ] has facilitated genome-wide mining for reference genes in soybean. Based on sequence homology, soybean orthologs of the best three A. thaliana reference genes have been identified. A further four genes have been selected, which have shown stable expression on a micro-array platform [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. Our data indicate that many of these newer reference genes indeed have greater expression stability than the conventionally used housekeeping genes. As a result, the use of combinations of these reference genes should provide a more reliable means of normalizing gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Reference genes are routinely used as a means of quantifying gene expression. The ideal reference genes should be expressed at a constant level throughout the <span class="trait">plant</span> and not be influenced by exogenous treatment [ <xref ref-type="bibr" rid="B1"> 1 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Housekeeping genes, such as those involved in basic cellular processes ( <italic> EF1&#x003b1;, UBQ </italic> and <italic> CYP </italic> ) or cell structure maintenance ( <italic> ACT, <span class="gene">TUB</span> </italic> ), have been extensively used, but increasingly it has become apparent that their expression level is not as independent of experimental conditions as had been expected [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. This implies a need to test in advance the expression stability of any proposed reference gene(s), a procedure which is often not followed in the literature. Normalization based on several reference genes has begun to become the standard, supported by the development of software such as geNorm and Normfinder [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> ]. However, the prior validation of reference genes remains uncommon in <span class="trait">plant</span> research, although it is the norm in human and animal research [ <xref ref-type="bibr" rid="B22"> 22 </xref> - <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> - <xref ref-type="bibr" rid="B54"> 54 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean has been used as a model <span class="trait">plant</span> for the study of photoperiod-induced floral induction [ <xref ref-type="bibr" rid="B45"> 45 </xref> ], but the molecular mechanism underlying this induction remains poorly understood. In soybean, <italic> ACT, <span class="gene">TUB</span> </italic> and <italic> UBQ </italic> are the most frequently used reference genes (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ), but there is increasing evidence that their expression is not particularly stable under certain conditions. More recently, some alternative reference genes have emerged [ <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> ]. Although four of these ( <italic> SKIP16, MTP, PEPKR1 </italic> and <italic> UKN2 </italic> ) have been shown by RT-qPCR to be stably expressed under certain limited experimental conditions, no detailed validation has to date been carried out to test their suitability in experiments involving photoperiodic treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						The weakness of <italic> ACT2 </italic> in soybean, rice, potato and sugarcane has been noted previously [ <xref ref-type="bibr" rid="B32"> 32 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B39"> 39 </xref> , <xref ref-type="bibr" rid="B40"> 40 </xref> ], while <italic> ACT2/7 </italic> was seen to be rather variable in <italic> A. thaliana </italic> [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. However, <italic> ACT2/7 </italic> was judged to be the most stable of a set of ten conventional housekeeping genes across 21 soybean samples, covering a range of developmental stages [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. Similarly, <italic> <span class="gene">TUB</span> </italic> performed poorly as a reference gene in grape, potato and soybean [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B39"> 39 </xref> ]. <italic> UBQ10 </italic> , which ranked poorly in the present experiments, was previously deemed unsatisfactory as a reference in soybean [ <xref ref-type="bibr" rid="B36"> 36 </xref> ] and in grape [ <xref ref-type="bibr" rid="B16"> 16 </xref> ], but enjoyed very stable expression in <italic> A. thaliana </italic> and <italic> Brachypodium sp </italic> . [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> ]. <italic> EF1b </italic> was among the most stable genes both in this study and in a previous study of soybean [ <xref ref-type="bibr" rid="B36"> 36 </xref> ], while in both potato and rice, <italic> EF1&#x003b1; </italic> was very stably expressed under conditions of biotic and abiotic stress [ <xref ref-type="bibr" rid="B39"> 39 </xref> ]. The same gene was also identified as being highly stable in its expression across tissues of rice [ <xref ref-type="bibr" rid="B31"> 31 </xref> ], but was unstable across tissues and organs of tomato at various developmental stages [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. <italic> TUA5 </italic> was identified as being highly stable across development in soybean [ <xref ref-type="bibr" rid="B36"> 36 </xref> ], while in poplar, <italic> TUA </italic> was very stably expressed across different tissues [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. Here, <italic> TUA5 </italic> expression was hardly affected by changes in photoperiod. Globally, the best-performing genes were <italic> SKIP16, UKN1, UKN2 </italic> and <italic> TIP41 </italic> , while the worst were <italic> PEPKR1 </italic> and <italic> HDC </italic> . <italic> TIP41 </italic> and <italic> UKN2 </italic> have been noted as showing stable expression across tissues and development in both tomato [ <xref ref-type="bibr" rid="B38"> 38 </xref> ] and aspen [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. However, <italic> TIP41 </italic> performed poorly during grape berry development [ <xref ref-type="bibr" rid="B16"> 16 </xref> ], and in the roots and leaves of <italic> A. thaliana </italic> plants suffering cadmium or copper stress [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. In aspen cambial cells, <italic> UKN2 </italic> expression was too unstable for the gene to be used for normalization [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. Thus, overall, while certain reference genes are stably expressed in one <span class="trait">plant</span> species, they may not be well suited for use in others. As a consequence, prior validation of reference genes needs to be carried out under the specific experimental conditions to be applied in gene expression studies.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of an isoflavones-containing red clover preparation and alkaline supplementation on bone metabolism in ovariectomized rats</h2>
			<h3>PMID:19503771</h3>
				
					<h3>PMCID:2685230</h3>
					<h4>期刊:Clinical Interventions in Aging</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kawakita </surname>  <given-names> S </given-names>  </name>  <xref ref-type="aff" rid="af1-cia-4-091"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marotta </surname>  <given-names> F </given-names>  </name>  <xref ref-type="aff" rid="af2-cia-4-091"> 2 </xref>  <xref ref-type="corresp" rid="c1-cia-4-091"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Naito </surname>  <given-names> Y </given-names>  </name>  <xref ref-type="aff" rid="af3-cia-4-091"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gumaste </surname>  <given-names> U </given-names>  </name>  <xref ref-type="aff" rid="af4-cia-4-091"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jain </surname>  <given-names> S </given-names>  </name>  <xref ref-type="aff" rid="af5-cia-4-091"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tsuchiya </surname>  <given-names> J </given-names>  </name>  <xref ref-type="aff" rid="af1-cia-4-091"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Minelli </surname>  <given-names> E </given-names>  </name>  <xref ref-type="aff" rid="af2-cia-4-091"> 2 </xref>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Bones were kept in NaCl (9 g/l) at 4 &#x000b0;C until whole left femoral breaking force was determined 24 h afterwards with three-point loading. Prior to mechanical testing, the left femurs were slightly thawed and held at room temperature on the day of test, then connective tissues were carefully removed, the length of the femurs was measured with a micrometer. The load was applied perpendicularly to the long axis of the femur in the mid-length of the bone supported on its epiphyses. The load increased at a rate of 100 N/min. The load was measured by the sensor with a strain gauge; the deformation was measured by an inductive sensor. The signals sent by the sensors were amplified and registered. The load-deformation curves, obtained for each bone, representing the relationships between load applied to the bone and deformation in response to the load, were analyzed. The load-deformation curve can be divided into the elastic deformation region and the plastic deformation region. Within the elastic deformation region, the slope of the linear part of the load-deformation curve, representing the extrinsic stiffness of bone, was tested. Within the plastic deformation region, the ultimate load and the breaking load were determined. The ultimate load is the maximum load sustained by the bone. The breaking load is the load at which the bone actually breaks. The force and energy (area under the curve, <span class="gene">N3</span> mm) necessary for the break at the center of the femur shaft were measured. Deformation caused by the applied loads was also measured. The mechanical strength of the left femoral neck was determined using a compression test with the same instrument. In femur shaft mechanical test, all femoral specimens were broken into two pieces, the proximal part was used and embedded in resin up to the lesser trochanter and a cylinder was shaped. The sample was inserted into a steel support in a vertical orientation. The mechanical <span class="trait">resistance</span> to failure was tested by applying a vertical load on the femoral head at a speed of 2 mm/min. All mechanic tests data were expressed as Newton (N).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improving the Reliability of Optimal In-Feed Amino Acid Ratios Based on Individual Amino Acid Efficiency Data from N Balance Studies in Growing Chicken</h2>
			<h3>PMID:26479521</h3>
				
					<h3>PMCID:4494439</h3>
					<h4>期刊:Animals : an Open Access Journal from MDPI</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wecke </surname>  <given-names> Christian </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liebert </surname>  <given-names> Frank </given-names>  </name>  <xref rid="c1-animals-03-00558" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Making use of the feed optimization program Fumi for Windows 3.1 (HYBRIMIN <sup> &#x000ae; </sup> Computer+Programme <span class="gene">GmbH</span> &#x00026; Co. KG, Hessisch Oldendorf, Germany), diets based on corn, wheat, fishmeal, field peas and wheat gluten, including small quantities of supplemented methionine, cystine and arginine, were created ( <xref ref-type="table" rid="animals-03-00558-t003"> Table 3 </xref> ). According to optimization, the final ingredient composition of all formulated CD fulfilled the postulated optimal AA ratios (values in parentheses of <xref ref-type="table" rid="animals-03-00558-t004"> Table 4 </xref> , compared to data in <xref ref-type="table" rid="animals-03-00558-t001"> Table 1 </xref> ), except for histidine, leucine and phenylalanine, respectively. Consequently, these AAs remained in relative excess and had to be deleted from the current data evaluation based on dietary AA efficiency, because their limiting position was doubtful. Due to reduced demands of broiler chickens on dietary protein and AA concentrations with increasing body <span class="trait">weight</span> (BW), the total protein and AA contents in the CD for grower periods were diminished by diet dilution with wheat starch ( <xref ref-type="table" rid="animals-03-00558-t003"> Table 3 </xref> ). However, the protein quality between corresponding starter and grower diets was kept constant, according to an equal proportion between all feed protein sources.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bacteriocins from the rhizosphere microbiome &#x02013; from an agriculture perspective</h2>
			<h3>PMID:26579159</h3>
				
					<h3>PMCID:4626563</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Subramanian </surname>  <given-names> Sowmyalakshmi </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/181089/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> Donald L. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/127867/overview"/>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Cerein 8A isolated from <italic> B. cereus </italic> 8A, interferes with cell membrane integrity and causes cell wall damage ( <xref rid="B9" ref-type="bibr"> Bizani et al., 2005 </xref> ), which is seen to be the mode of action of many bacteriocins. (Please refer to <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> for bacteriocins from <italic> B. thuringiensis </italic> and examples of other bacteria). Bacteriocin Bac-GM17 from the rhizosphere bacteria <italic> Bacillus clausii </italic> strain <span class="gene">GM17</span> of <italic> Ononis angustissima </italic> Lam. is a 5.158 kDa monomer protein with a unique sequence and having bactericidal effect on <italic> Agrobacterium tumefaciens </italic> C58 and fungistatic effect on <italic> Candida tropicalis </italic> R2 CIP203 ( <xref rid="B43" ref-type="bibr"> Mouloud et al., 2013 </xref> ). The bacteriocin putidacin, produced by <italic> Pseudomonas putida </italic> strain BW11M1, isolated from banana root, is very similar to mannose-binding <span class="trait">plant</span> lectins ( <xref rid="B49" ref-type="bibr"> Parret et al., 2003 </xref> ). Amylocyclicin, a 6.381 kDa peptide from <italic> Bacillus amyloliquefaciens </italic> FZB42, is a novel circular, ribosomally synthesized bacteriocin with high antibacterial activity to closely related Gram-positive bacteria ( <xref rid="B57" ref-type="bibr"> Scholz et al., 2014 </xref> ). <italic> B. subtilis </italic> strain 14B produces Bac 14B, a 21 kDa bacteriocin that is effective against crown gall disease caused by <italic> A. tumefaciens </italic> ( <xref rid="B27" ref-type="bibr"> Hammami et al., 2009 </xref> ). <italic> B. subtilis </italic> strain IH7 produces a bacteriocin Bac IH7 which is reported to be a <span class="trait">plant</span> growth promotor. Tomato and muskmelon treated with Bac IH7 showed enhanced germination percentage and increased shoot <span class="trait">weight</span> and height and root lengths; it also served as a biocontrol for <italic> Alternaria solani </italic> and other seed borne pathogens ( <xref rid="B28" ref-type="bibr"> Hammami et al., 2011 </xref> ). A 25&#x02013;35 kDa bacteriocin from <italic> Lysinibacillus </italic> jx416856, a bacteria isolated from fruit and vegetable waste, was observed to inhibit food borne pathogens such as <italic> Staphylococcus aureus </italic> , <italic> Staphylococcus epidermidis </italic> , and <italic> B. cereus </italic> ( <xref rid="B3" ref-type="bibr"> Ahmad et al., 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Optimizing medium composition, incubation and agitation speed can result in enhancement of the production of some bacteriocins. For example, <italic> B. thuringiensis </italic> subsp. <italic> kurstaki </italic> strain, producing the bacteriocin Bacthuricin F4 when grown in <span class="gene">TSB</span> medium with an optimal carbon&#x02013;nitrogen ratio of 9 increases the bacteriocin production fourfold ( <xref rid="B38" ref-type="bibr"> Kamoun et al., 2009 </xref> ). Bacteriocin <italic> Bacillus </italic> sp. YAS 1 could be increased 1.6-fold by this method. This bacteriocin had a wide pH range (1&#x02013;13) as well as temperature (45&#x02013;80&#x000b0;C) with antimicrobial activity to human pathogens such as <italic> Clostridium </italic> , <italic> Staphylococcus </italic> , <italic> Enterococcus </italic> , and <italic> Salmonella </italic> , and <span class="trait">plant</span> pathogens such as <italic> E. amylovora </italic> , and showing no effect on lactic acid bacteria ( <xref rid="B22" ref-type="bibr"> Embaby et al., 2014 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, Heterodera glycines Ichinohe). Although this approach successfully mapped a large number of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse set of 553 soybean <span class="trait">plant</span> introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as seed coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>