<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Plant Height</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Plant Height</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Expanding Omics Resources for Improvement of Soybean Seed Composition Traits</h2>
			<h3>PMID:26635846</h3>
				
					<h3>PMCID:4657443</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chaudhary </surname>  <given-names> Juhi </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/262086/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant B. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/104588/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sonah </surname>  <given-names> Humira </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/162853/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh K. </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103657/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/162861/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>
				<br/> 
				年份：<day> 05 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Association studies and MAS have been used in <span class="trait">plant</span> improvement programs for several years. Nevertheless, they have some limitations, such as long selection cycles and the identification of significant marker-QTL associations which is unable to capture &#x0201c;minor&#x0201d; gene effects (Desta and Ortiz, <xref rid="B34" ref-type="bibr"> 2014 </xref> ). These limitations can be effectively addressed by using a promising approach, known as genomic selection (GS) (Meuwissen et al., <xref rid="B89" ref-type="bibr"> 2001 </xref> ). Unlike MAS, <span class="gene">GS</span> utilizes the entire set of markers to predict genomic estimated breeding values (GEBVs) for all genotyped individuals within a breeding population by capturing the total additive genetic variance for a particular trait of interest (Heffner et al., <xref rid="B55" ref-type="bibr"> 2009 </xref> , <xref rid="B54" ref-type="bibr"> 2011 </xref> ). <span class="gene">GS</span> requires a training population (both genotyped and phenotyped) to calculate breeding values by using all marker information and avoiding biased marker effects simultaneously (Heffner et al., <xref rid="B55" ref-type="bibr"> 2009 </xref> ). The primary benefit of <span class="gene">GS</span> is that selection can be imposed at a very early stage in the breeding process, thus accelerating the breeding cycles without phenotyping. However, optimizing the constituents of a training population is very challenging and is influenced by a number of parameters, such as choice of model, size of training data, trait heritability, span of linkage disequilibrium (LD), marker density, and strength of genetic relationships between training and validation populations (Bentley et al., <xref rid="B15" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the advantages of <span class="gene">GS</span> over conventional methods, it has been successfully applied to a variety of crops, such as wheat and maize (Crossa et al., <xref rid="B31" ref-type="bibr"> 2010 </xref> ), pear (Iwata et al., <xref rid="B60" ref-type="bibr"> 2013 </xref> ), sugar beet (W&#x000fc;rschum and Kraft, <xref rid="B155" ref-type="bibr"> 2014 </xref> ), and is constantly being used for other crop species. There are only a few reports for the application of <span class="gene">GS</span> in soybean (Hu et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> ; Shu et al., <xref rid="B129" ref-type="bibr"> 2012 </xref> ; Bao et al., <xref rid="B9" ref-type="bibr"> 2014 </xref> ; Jarqu&#x000ed;n et al., <xref rid="B62" ref-type="bibr"> 2014 </xref> ). Hu et al. ( <xref rid="B57" ref-type="bibr"> 2011 </xref> ) studied primary embryogenesis capacity in soybean using 126 RILs and 80 SSRs and reported a strong correlation ( <italic> r </italic> <sup> 2 </sup> = 0.78) between GEBVs and phenotypic data (Hu et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> ). A study was performed on soybean HSW using 79 sequence-characterized amplified region (SCAR) markers for 288 varieties. This study reported a correlation coefficient of 0.904 amongst the GEBVs and phenotypic values (Shu et al., <xref rid="B129" ref-type="bibr"> 2012 </xref> ). Recently, Bao et al. ( <xref rid="B9" ref-type="bibr"> 2014 </xref> ) investigated soybean cyst nematode <span class="trait">resistance</span> via genotyping of 282 accessions employing the 1536 SNP array and demonstrated a significant prediction accuracy (0.59&#x02013;0.67) for soybean cyst nematode <span class="trait">resistance</span> in soybean. Another study reported a prediction accuracy of 0.64 for grain <span class="trait">yield,</span> which indicates a good potential of <span class="gene">GS</span> utilization strategy in soybean (Jarqu&#x000ed;n et al., <xref rid="B62" ref-type="bibr"> 2014 </xref> ). It is noticeable that implementation of a <span class="gene">GS</span> approach is needed in soybean seed related traits. Moreover, with the declining genotyping costs and increasing phenotyping costs, <span class="gene">GS</span> can be helpful to mitigate many of the selections associated with phenotyping and ultimately speed up the breeding cycles. Hence, the <span class="gene">GS</span> statistical method is more feasible and more emphasis should be given to employ this approach in the improvement of seed composition with available genotypic and phenotypic data.
					</p>
				
					<p>&nbsp;&nbsp;
						The availability of the sequence information provides new avenues for the engineering of soybean seed composition. The new &#x0201c;omics&#x0201d; approaches offers a potential resource to comprehend the metabolic regulatory network governing seed storage compound accumulation. To accommodate the global food demand and growing population needs, several genetic engineering and genome editing strategies need to be employed to obtain the next-generation crop at faster rate than the conventional breeding. Studies have reported improved soybean seed oil content via X-ray, and EMS-induced mutations in the desired genes (Dierking and Bilyeu, <xref rid="B37" ref-type="bibr"> 2009 </xref> ; Pham et al., <xref rid="B106" ref-type="bibr"> 2010 </xref> , <xref rid="B107" ref-type="bibr"> 2011 </xref> ). For example, deletion of a 100 kb sequence on Chr. 10 in a mutant line M23 increased oleic acid content but caused <span class="trait">yield</span> drag (Pham et al., <xref rid="B107" ref-type="bibr"> 2011 </xref> ). Similarly, the <span class="gene">SACPD-C</span> deletion in an A6 mutant line showed elevated stearic acid content; however, simultaneously a reduced seed <span class="trait">yield</span> was also observed (Gillman et al., <xref rid="B45" ref-type="bibr"> 2014 </xref> ). Nevertheless, conventional methods require labor intensive screening procedure to identify germplasm accessions with desired phenotypes and stable performance in multiple generations and environments.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetically modified organisms <span class="gene">(GMOs)</span> have quite a few limitations, such as social acceptance, cost-related concerns (regulatory and licensing), limited resources, and time. To overcome these drawbacks, genome editing has arisen as a modern approach for specifically targeting and modifying DNA sequences for crop improvement and is considered a non-GM approach (Voytas and Gao, <xref rid="B147" ref-type="bibr"> 2014 </xref> ). Several methodologies are available for genome engineering, such as engineered homing endonucleases or meganucleases, zinc finger nucleases (ZFNs) and transcription activator-like effector nucleases (TALENs). These are being used to mutagenize genomes at precise locations (Cermak et al., <xref rid="B22" ref-type="bibr"> 2011 </xref> ; Sander et al., <xref rid="B121" ref-type="bibr"> 2011 </xref> ). An optimized mutagenesis approach is required for the simultaneous recovery of plants with single or multiple mutations in a gene of interest without interfering with the rest of the genetic background. Therefore, site-directed mutagenesis could be a powerful method for producing the desired results. Moreover, CRISPR (clustered regularly interspaced short palindromic repeats)/Cas (CRISPR-associated) has emerged as a simpler method that permits introduction of desired SNP into a gene of interest without a need for extensive design and time-consuming assembly of individual DNA-binding proteins (Belhaj et al., <xref rid="B13" ref-type="bibr"> 2013 </xref> ; Bortesi and Fischer, <xref rid="B17" ref-type="bibr"> 2014 </xref> ; Voytas and Gao, <xref rid="B147" ref-type="bibr"> 2014 </xref> ). In addition, Cas9 has been reported as a successful method for genome editing in tobacco (Nekrasov et al., <xref rid="B96" ref-type="bibr"> 2013 </xref> ), rice, wheat, Arabidopsis, tobacco, and sorghum (Jiang et al., <xref rid="B65" ref-type="bibr"> 2013 </xref> ), and sweet orange (Jia and Wang, <xref rid="B64" ref-type="bibr"> 2014 </xref> ). Targeted genome editing has been tested for soybean seed composition improvement. The context-dependent assembly (CoDA) was used to create ZFNs for target specific mutagenesis ( <italic> DICER-LIKE </italic> and other genes involved in RNA silencing) in soybean. This approach showed successful heritable transmission of the ZFN-induced mutation in subsequent generations and could be useful for making mutations in duplicated genes efficiently (Curtin et al., <xref rid="B32" ref-type="bibr"> 2011 </xref> ). While in another study, mutations were induced in two fatty acid desaturase-2 genes ( <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> ) using TALENs. The changed fatty acid profile of the seed with increased oleic acid (20&#x02013;80%) and decreased linoleic acid (from 50% to less than 4%) was reported for <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> mutations (Haun et al., <xref rid="B53" ref-type="bibr"> 2014 </xref> ). In comparison to other transgenic technologies, the CRISPR system is considered a non-GM technology, since the CAS9 can be deleted from the host <span class="trait">plant</span> via backcrossing in subsequent generations, once the mutation is accomplished. The usefulness of the CRISPR system being a simple and cost effective technique for genome editing in soybean can't be denied. This approach can be applied to confirm candidate genes, novel alleles/phenotypes, and engineer soybeans with high quality seed traits.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolutionary Quantitative Genomics of <italic> Populus trichocarpa </italic></h2>
			<h3>PMID:26599762</h3>
				
					<h3>PMCID:4658102</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Porth </surname>  <given-names> Ilga </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Kl&#x000e1;p&#x00161;t&#x0011b; </surname>  <given-names> Jaroslav </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McKown </surname>  <given-names> Athena D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> La Mantia </surname>  <given-names> Jonathan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guy </surname>  <given-names> Robert D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ingvarsson </surname>  <given-names> P&#x000e4;r K. </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hamelin </surname>  <given-names> Richard </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mansfield </surname>  <given-names> Shawn D. </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ehlting </surname>  <given-names> J&#x000fc;rgen </given-names>  </name>  <xref ref-type="aff" rid="aff007">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Douglas </surname>  <given-names> Carl J. </given-names>  </name>  <xref ref-type="aff" rid="aff008">  <sup> 8 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> El-Kassaby </surname>  <given-names> Yousry A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						SNP allelic frequencies within both nitrate transporter genes <italic> ATNRT2 </italic> : <italic> 4 </italic> and <italic> ATNRT2 </italic> :1 were strongly aligned with temperature variation (R <sup> 2 </sup> ~90%) in <italic> P </italic> . <italic> trichocarpa </italic> . Moreover, these SNPs were pleiotropic for multiple autumn phenology traits, <span class="trait">height,</span> and leaf <span class="trait">rust</span> <span class="trait">resistance</span> ( <xref rid="pone.0142864.s005" ref-type="supplementary-material"> S3 Table </xref> ). Nitrate transporters are generally important in plants, as nitrate is the main nitrogen source required for synthesis of nucleic and amino acids. Therefore, a regulation of nitrate distribution is crucial to modulate growth (biomass acquisition) in response to temperature or light conditions ([ <xref rid="pone.0142864.ref061" ref-type="bibr"> 61 </xref> , <xref rid="pone.0142864.ref062" ref-type="bibr"> 62 </xref> ]). Interestingly, there are only two poplar representatives within a phylogenetic sub-clade of <span class="gene">NRT2</span> that is populated by as many as five Arabidopsis sequences ( <italic> ATNRT2 </italic> . <italic> 1 </italic> / <italic> 2 </italic> . <italic> 2 </italic> / <italic> 2 </italic> . <italic> 3/2 </italic> . <italic> 4 </italic> / <italic> 2 </italic> . <italic> 6 </italic> ). This implies that a deletion event occurred in this clade whose functional significance remains elusive to date [ <xref rid="pone.0142864.ref062" ref-type="bibr"> 62 </xref> ]. Phylogenetic reconstruction coupled with gene expression analysis point at neo/subfunctionalisation of the two poplar nitrate transporters for long distance nitrate transport from roots, wood to leaves [ <xref rid="pone.0142864.ref062" ref-type="bibr"> 62 </xref> ]. This acquisition of novel expression pattern and loss of the ancestral expression pattern demonstrates the signature of adaptive evolution in functional diversification in paralogous gene pairs [ <xref rid="pone.0142864.ref063" ref-type="bibr"> 63 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, our results revealed that adaptive genetic variants within both poplar nitrate transporters were also associated with leaf <span class="trait">rust</span> <span class="trait">resistance</span> ([ <xref rid="pone.0142864.ref038" ref-type="bibr"> 38 </xref> ]; <xref rid="pone.0142864.s005" ref-type="supplementary-material"> S3 Table </xref> ). In <italic> Arabidopsis </italic> , loss of function of <italic> ATNRT2 </italic> . <italic> 1 </italic> primes salicylic acid signaling and <italic> PR1 </italic> up-regulation [ <xref rid="pone.0142864.ref064" ref-type="bibr"> 64 </xref> ]. In poplar leaf <span class="trait">rust</span> inoculations, both <span class="gene">PT</span> <italic> <span class="gene">NRT2</span> </italic> . <italic> 4 </italic> and <span class="gene">PT</span> <italic> <span class="gene">NRT2</span> </italic> . <italic> 1 </italic> are strongly down-regulated in incompatible interactions, while no expression change is apparent in compatible interactions (J. La Mantia, personal observation). The identified nitrogen transporters might be important in nitrogen storage and nitrogen remobilization to recycle nutrients during the progression of leaf senescence [ <xref rid="pone.0142864.ref065" ref-type="bibr"> 65 </xref> ]. They may also function in down-regulation of nitrogen assimilation during seasonal remodeling of tree phenology related to growth cessation induced by short photoperiods ([ <xref rid="pone.0142864.ref066" ref-type="bibr"> 66 </xref> , <xref rid="pone.0142864.ref067" ref-type="bibr"> 67 </xref> ]) and/or temperature [ <xref rid="pone.0142864.ref058" ref-type="bibr"> 58 </xref> ]. The effect of temperature on <span class="trait">rust</span> aggressiveness is noted [ <xref rid="pone.0142864.ref068" ref-type="bibr"> 68 </xref> ] and the climatic conditions which form a conducive environment for <span class="trait">rust</span> infection and disease duration likely provide a strong adaptive selection toward <span class="trait">resistance.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional conservation and diversification of the soybean maturity gene <italic> <span class="gene">E1</span> </italic> and its homologs in legumes</h2>
			<h3>PMID:27405888</h3>
				
					<h3>PMCID:4942777</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xingzheng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhai </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yaying </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Xiaojie </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yupeng </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> L&#x000fc; </surname>  <given-names> Shixiang </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Guang </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yuqiu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bu </surname>  <given-names> Qingyun </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Zhengjun </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Key Laboratory of Soybean Molecular Design Breeding, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences </institution> , Harbin 150081,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> University of Chinese Academy of Sciences </institution> , No. 19A Yuquan Road, Beijing 100049,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Information Center of Heilongjiang Academy of Agricultural Sciences </institution> , Harbin 150086,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 03 </day>  <month> 06 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Gene regulatory networks involved in flowering time and photoperiodic responses in legumes remain unknown. Although the major maturity gene <italic> <span class="gene">E1</span> </italic> has been successfully deciphered in soybean, knowledge on the functional conservation of this gene is limited to a certain extent to <italic> <span class="gene">E1</span> </italic> homologs in legumes. The ectopic expression of <italic> Phvul.009G204600 </italic> ( <italic> PvE1L </italic> ), an <italic> <span class="gene">E1</span> </italic> homolog from common bean, delayed the onset of flowering in soybean. By contrast, the ectopic expression of <italic> Medtr2g058520 </italic> ( <italic> MtE1L </italic> ) from <italic> Medicago truncatula </italic> did not affect the flowering of soybean. Characterization of the late-flowering <italic> mte1l </italic> mutant indicated that <italic> MtE1L </italic> promoted flowering in <italic> Medicago truncatula </italic> . Moreover, all transgenic <italic> <span class="gene">E1</span> </italic> , <italic> PvE1L </italic> and <italic> MtE1L </italic> soybean lines exhibited phenotypic changes in terms of <span class="trait">plant</span> <span class="trait">height.</span> Transgenic <italic> <span class="gene">E1</span> </italic> or <italic> PvE1L </italic> plants were taller than the wild-type, whereas transgenic <italic> MtE1L </italic> plants produced dwarf phenotype with few nodes and short internode. Thus, functional conservation and diversification of <italic> <span class="gene">E1</span> </italic> family genes from legumes in the regulation of flowering and <span class="trait">plant</span> growth may be associated with lineage specification and genomic duplication.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is an important model <span class="trait">plant</span> used to elucidate photoperiodism in legumes <xref ref-type="bibr" rid="b36"> 36 </xref> . <italic> <span class="gene">E1</span> </italic> is a major gene associated with flowering time and maturity and is located at the pericentromeric region; <italic> <span class="gene">E1</span> </italic> is intron-free and encodes a protein containing a putative bipartite nuclear localizing signal (NLS) and a domain distantly related to the plant-specific B3 domain (B3-like domain) <xref ref-type="bibr" rid="b37"> 37 </xref> . Several members of the B3 superfamily directly or indirectly regulate flowering. <italic> AtRAV1 </italic> in <italic> Arabidopsis </italic> also influences flowering time <xref ref-type="bibr" rid="b38"> 38 </xref> . <italic> TEMPRANILLO1 </italic> and <italic> TEMPRANILLO2 </italic> genes can negatively regulate <italic> FT </italic> expression and floral transition in <italic> Arabidopsis </italic> through direct binding of the B3 domain to a consensus bipartite sequence element in the <italic> FT </italic> 5&#x02032;-untranslated region <xref ref-type="bibr" rid="b39"> 39 </xref> . In addition, several genes containing the B3 domain, such as <italic> AUXIN RESPONSE FACTOR 1 </italic> <xref ref-type="bibr" rid="b40"> 40 </xref> and <italic> LEAFY COTYLEDON2 </italic> , regulate <span class="trait">plant</span> growth and development <xref ref-type="bibr" rid="b41"> 41 </xref> . The four major recessive alleles of <italic> <span class="gene">E1</span> </italic> identified in soybean are <italic> e1-as </italic> (a single missense point mutation), <italic> <span class="gene">e1-fs</span> </italic> (a 1 bp deletion leading to frame-shift), <italic> e1-nl </italic> (~130 kb deletion comprising the <italic> <span class="gene">E1</span> </italic> gene), and <italic> e1-b3a </italic> (three SNPs and 2 bp deletion in the middle of the B3 domain) <xref ref-type="bibr" rid="b42"> 42 </xref> . <italic> e1-as </italic> is apparently a leaky allele and partially suppresses flowering in soybean <xref ref-type="bibr" rid="b37"> 37 </xref> <xref ref-type="bibr" rid="b42"> 42 </xref> , contrary to the functional <italic> <span class="gene">e1-fs</span> </italic> and <italic> e1-nl </italic> . <italic> <span class="gene">E1</span> </italic> is expressed in a bimodal pattern, with higher expression in long-day (LD) conditions than in short-day (SD) conditions. <italic> <span class="gene">E1</span> </italic> is a putative transcription factor (TF) that negatively controls <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> to delay flowering under the background with functional <italic> <span class="gene">PHYA</span> </italic> genes ( <italic> E3E4 </italic> ) and LD conditions. Zhai <italic> et al. </italic> <xref ref-type="bibr" rid="b42"> 42 </xref> proved that <italic> <span class="gene">E1</span> </italic> can positively regulate <italic> <span class="gene">GmFT4</span> </italic> to control flowering in soybean. Xu <italic> et al. </italic> <xref ref-type="bibr" rid="b43"> 43 </xref> revealed that, similar to <italic> <span class="gene">E1</span> </italic> , two <italic> <span class="gene">E1</span> </italic> orthologs, namely, <italic> <span class="gene">E1La</span> </italic> ( <italic> <span class="gene">Glyma.04G156400.1</span> </italic> ) and <italic> <span class="gene">E1Lb</span> </italic> ( <italic> <span class="gene">Glyma.04G143300.1</span> </italic> ), can control the onset of flowering. That <italic> LjE1L </italic> controls flowering was proposed based on the analysis of the polymorphism of <italic> Lj5g3v2221340 </italic> ( <italic> LjE1L </italic> , an ortholog of <italic> <span class="gene">E1</span> </italic> ) among different wild <italic> Lotus japonicus </italic> species from various geographic locations in Japan <xref ref-type="bibr" rid="b44"> 44 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we analyzed the protein structure and phylogenetic relationships among <italic> <span class="gene">E1</span> </italic> family genes identified in legumes. We also retrieved sequences of <italic> <span class="gene">E1</span> </italic> homologs in common bean and Medicago and performed functional analysis by overexpressing the homologs in soybean, <italic> Arabidopsis </italic> , and rice. Functional conservation and diversification among <italic> <span class="gene">E1</span> </italic> family genes of leguminous species were consistent with the divergence time of genes and lineage species in this <span class="trait">plant</span> group.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">E1</span> </italic> family genes, except for <italic> CcE1L1 </italic> and <italic> CcE1L2 </italic> , were annotated on six chromosomes in different genomes ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S1 </xref> ). <italic> <span class="gene">E1La</span> </italic> and <italic> <span class="gene">E1Lb</span> </italic> possibly originated from segmental duplication because both genes were annotated on chromosome 4 and located at approximately 10&#x02009;Mb apart ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S1 </xref> ). To examine whether <italic> <span class="gene">E1</span> </italic> family genes are involved in segmental duplication, we retrieved the syntenic blocks of individual <italic> <span class="gene">E1</span> </italic> homologs from the <span class="trait">Plant</span> Genome Duplication Database (PGDD). Nonsynonymous ( <italic> Ka </italic> ) and synonymous ( <italic> Ks </italic> ) substitution rates were then used to estimate date of divergence. The <italic> Ka/Ks </italic> ratio of the segmental pairs in soybean varied from 0.1736 to 0.2089 (ratio&#x02009;&#x0003c;&#x02009;1) ( <xref ref-type="table" rid="t1"> Table 1 </xref> ), indicating that <italic> <span class="gene">E1</span> </italic> , <italic> <span class="gene">E1La</span> </italic> , and <italic> <span class="gene">E1Lb</span> </italic> underwent negative or purifying selection. The soybean genome underwent two WGD rounds, which occurred ~58 and ~13 Mya <xref ref-type="bibr" rid="b47"> 47 </xref> . The divergence time between <italic> <span class="gene">E1</span> </italic> and <italic> <span class="gene">E1La</span> </italic> / <italic> <span class="gene">E1Lb</span> </italic> can be traced back to 10.6 and 8.8 Mya, respectively ( <xref ref-type="table" rid="t1"> Table 1 </xref> ), which is possibly associated with the recent WGD in soybean. The divergence time between <italic> <span class="gene">E1</span> </italic> and <italic> PvE1L </italic> was also estimated to have occurred 14.1 Mya. By contrast, the divergence time between <italic> <span class="gene">E1</span> </italic> and <italic> MtE1L </italic> could be traced back to 62.9 Mya ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Among the <italic> <span class="gene">E1</span> </italic> family members in non-soybean legumes, <italic> PvE1L </italic> displayed the closest genetic relationship with <italic> <span class="gene">E1</span> </italic> . In addition, estimated divergence times between <italic> <span class="gene">E1</span> </italic> and <italic> CcE1L1 </italic> / <italic> CcE1L2 </italic> are 28.3 and 28.1 Mya, respectively. Similarly, the divergence time between <italic> <span class="gene">E1</span> </italic> and <italic> LjE1L </italic> / <italic> CaE1L </italic> are 60.8 and 88.5 Mya, respectively ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). The evolution rates may not be inaccurate because of differences among species or even among the distinct locations of genes in a chromosome. Nevertheless, these predicted results revealed the sequential evolutionary relationship of these genes. The <italic> <span class="gene">E1</span> </italic> family genes can be divided into two main groups, Group I and Group II, based on the estimated divergence times ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S1 </xref> ), corresponding to the Millettiod/Phaseoloid and Hologalegina branch, respectively <xref ref-type="bibr" rid="b51"> 51 </xref> . More conserved syntenic blocks were also observed between the region of the <italic> <span class="gene">E1</span> </italic> and that of <italic> <span class="gene">E1La</span> </italic> or <italic> <span class="gene">E1Lb</span> </italic> than that between the region of <italic> <span class="gene">E1</span> </italic> and that of <italic> PvE1L </italic> , <italic> MtE1L </italic> , or <italic> CaE1L </italic> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S2 </xref> ). These results are consistent with expected decline in conservation with taxonomic distance. We could not retrieve any <italic> <span class="gene">E1</span> </italic> homolog in <italic> L. japonicus </italic> and <italic> C. cajan </italic> from the CoGe server for syntenic analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Two independent <italic> <span class="gene">E1</span> </italic> complementation lines ( <italic> <span class="gene">E1</span> </italic> #L16 and <italic> <span class="gene">E1</span> </italic> #L18) and one <italic> PvE1L </italic> overexpression line ( <italic> PvE1L </italic> #L2) were obtained, We painted the leaves with 160&#x02009;mg L <sup> &#x02212;1 </sup> glufosinate to test herbicide <span class="trait">resistance,</span> and the resistant individuals were segregated from the non-resistant individuals in an approximately 3:1 ratio ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S2 </xref> ). Herbicide-resistant T <sub> 1 </sub> and T <sub> 2 </sub> plants of transgenic <italic> <span class="gene">E1</span> </italic> and <italic> PvE1L </italic> lines exhibited late-flowering phenotype, Statistical analysis of flowering time data of herbicide-resistant T <sub> 3 </sub> plants of transgenic lines revealed that two <italic> <span class="gene">E1</span> </italic> transgenic lines and <italic> PvE1L </italic> #L2 flowered significantly later than the WT-DN plants in both LD ( <xref ref-type="fig" rid="f3"> Figs 3a </xref> and <xref ref-type="fig" rid="f4"> 4a&#x02013;c </xref> ) and SD ( <xref ref-type="fig" rid="f3"> Figs 3b </xref> and <xref ref-type="fig" rid="f4"> 4d&#x02013;f </xref> ) conditions. The WT-DN plants flowered at 35.3 days after emergence (DAE) under LD conditions and 24.2 DAE under SD conditions. In addition, <italic> <span class="gene">E1</span> </italic> #L16 and <italic> <span class="gene">E1</span> </italic> #L18, which are two <italic> <span class="gene">E1</span> </italic> transgenic lines, flowered at 48.7 and 43.2 DAE under LD conditions and at 29.7 and 27.8 DAE under SD conditions. These results indicate that <italic> <span class="gene">E1</span> </italic> overexpression delays the onset of flowering. The <italic> PvE1L </italic> transgenic line also flowered at a significantly later time (44.2 DAE and 33.2 DAE under LD and SD conditions, respectively) compared with the WT-DN plants (35.3 and 24.2 DAE under LD and SD conditions, respectively) ( <xref ref-type="fig" rid="f3"> Figs 3c </xref> and <xref ref-type="fig" rid="f4"> 4g </xref> ). <italic> PvE1L </italic> overexpression had no effect on the expression of native <italic> e1-as </italic> in <italic> PvE1L </italic> #L2 ( <xref ref-type="fig" rid="f4"> Fig. 4h </xref> ). Moreover, the expression levels of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> were significantly reduced, whereas <italic> <span class="gene">GmFT4</span> </italic> expression was significantly induced ( <xref ref-type="fig" rid="f3"> Figs 3d,e </xref> and <xref ref-type="fig" rid="f4"> 4i,j </xref> ) in transgenic lines when <italic> <span class="gene">E1</span> </italic> ( <xref ref-type="fig" rid="f3"> Fig. 3f </xref> ) and <italic> PvE1L </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4k </xref> ) were overexpressed under LD and SD conditions. These results are consistent with previous findings <xref ref-type="bibr" rid="b25"> 25 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> . <italic> <span class="gene">E1</span> </italic> complementation test results further proved that <italic> <span class="gene">E1</span> </italic> is a flowering repressor in soybean. <italic> PvE1L </italic> also demonstrated a conserved physiological function as a flowering repressor. Interestingly, the <italic> PvE1L </italic> gene was annotated in a physical position, where two most significant QTLs ( <italic> df9.1 </italic> and <italic> df9. </italic> 2) for flowering time were mapped <xref ref-type="bibr" rid="b52"> 52 </xref> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S1 </xref> ); Thus, <italic> PvE1L </italic> possibly suppresses flowering by regulating the expression of <italic> FT </italic> -like genes in common bean. In this study, we propose that <italic> PvE1L </italic> plays a key role in modulating flowering time in common bean, although this assumption requires more actual evidence confirmation.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, overexpression of <italic> <span class="gene">E1</span> </italic> and <italic> PvE1L </italic> in soybean also resulted in several morphological changes, e.g., <span class="trait">plant</span> <span class="trait">height</span> and number of nodes ( <xref ref-type="supplementary-material" rid="S1"> Supplemental Figs S3a&#x02013;d </xref> and <xref ref-type="supplementary-material" rid="S1"> S4a&#x02013;d </xref> ). WT-DN plants in LDs were 66.3&#x02013;73.1&#x02009;cm tall with 18.3&#x02013;19.0 nodes, whereas the <italic> <span class="gene">E1</span> </italic> #L16, <italic> <span class="gene">E1</span> </italic> #L18, and <italic> PvE1L </italic> #L2 were significantly taller (27%, 24% and 15% increase, respectively) than WT-DN plants. This characteristic was due to the increase in the number of nodes (3.7, 3.3 and 2.4 nodes more, respectively) because internode length was not significantly different from WT-DN <span class="trait">plant</span> ( <xref ref-type="supplementary-material" rid="S1"> Supplemental Figs S3c&#x02013;e </xref> and <xref ref-type="supplementary-material" rid="S1"> S4c&#x02013;e </xref> ). Although all lines appeared shorter than under LD conditions, similar trends in <span class="trait">plant</span> <span class="trait">height</span> and node number between transgenic <italic> <span class="gene">E1</span> </italic> / <italic> PvE1L </italic> lines and WT-DN were observed under SD conditions ( <xref ref-type="supplementary-material" rid="S1"> Supplemental Figs S3 </xref> and <xref ref-type="supplementary-material" rid="S1"> S4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						These results suggested that <italic> MtE1L </italic> may regulate flowering in Medicago. To investigate whether ectopic overexpression of <italic> MtE1L </italic> in soybean affects flowering time, we transformed <italic> MtE1L </italic> into Dongnong 50. Two soybean transgenic lines ( <italic> MtE1L </italic> #L6 and <italic> MtE1L </italic> #L14) showed high <italic> MtE1L </italic> expression, whereas no <italic> MtE1L </italic> expression was detected in WT-DN lines ( <xref ref-type="fig" rid="f6"> Fig. 6a </xref> ). Meanwhile, progenies of both transgenic lines were segregated when tested for herbicide <span class="trait">resistance.</span> The resistant to non-resistant individuals were consistent with 3:1 of Mendelian inheritance ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S2 </xref> ). Two transgenic <italic> MtE1L </italic> soybean lines flowered at 36.3&#x02013;37.2 and 24.0&#x02013;24.3 DAE under LD and SD conditions not significantly different from that of the WT-DN plants (36.5 and 24.2 DAE under LD and SD conditions) ( <xref ref-type="fig" rid="f6"> Fig. 6b&#x02013;d </xref> ). In addition, the expression levels of native <italic> e1 </italic> -as, <italic> <span class="gene">GmFT2a</span> </italic> , <italic> <span class="gene">GmFT5a</span> </italic> , and <italic> <span class="gene">GmFT4</span> </italic> in transgenic plants were consistent with those in WT-DN plants both under LD and SD conditions ( <xref ref-type="fig" rid="f6"> Fig. 6e&#x02013;g </xref> ). The significant difference of WT-DN plants in flowering time between LD or SD conditions revealed that the soybean cultivar Dongnong 50 was sensitive to photoperiod. Previous study showed that <italic> e1-as </italic> represents a partial functional allele that controls flowering in soybean <xref ref-type="bibr" rid="b42"> 42 </xref> . Thus, the ectopic overexpression of <italic> MtE1L </italic> in soybean did not affect flowering in soybean. Taken mutant analysis and transgene in soybean together, we propose the <italic> MtE1L </italic> gene is a flowering promoter in Medicago, contrary to the <italic> <span class="gene">E1</span> </italic> or <italic> PvE1L </italic> genes as flowering repressor.
					</p>
				
					<p>&nbsp;&nbsp;
						The legume family consists of crops that serve as primary sources of <span class="trait">plant</span> proteins for food and feed. Flowering is one of the major agronomic traits of legumes and has been the focus of fundamental and applied studies. Flowering regulation pathway in soybean is distinct from that in model plants, such as <italic> Arabidopsis </italic> and rice. Moreover, <italic> <span class="gene">E1</span> </italic> is a key gene in the gene regulatory network of flowering in soybean. The product of this gene is a negative regulator of the photoperiod response pathway. However, the exact molecular function and evolutionary history of the <italic> <span class="gene">E1</span> </italic> gene remains to be elucidated.
					</p>
				
					<p>&nbsp;&nbsp;
						These genes apparently perform different functions in flowering, although <italic> <span class="gene">E1</span> </italic> family genes in legume seem to have the same origin. <italic> E1, <span class="gene">E1La</span> and <span class="gene">E1Lb</span> </italic> <xref ref-type="bibr" rid="b43"> 43 </xref> in soybean suppress flowering, whereas <italic> MtE1L </italic> in Medicago appears to promote flowering. The distinct function was probably due to the presence of an extra helix in the proteins ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). Interestingly, the extra helix was present in MtE1L from <italic> M. truncatula </italic> , but not in E1. The association between <span class="gene">E1</span> structure and flowering physiology might be an interesting topic for further studies. From a parsimony perspective, the evolution of different forms of TFs from the same ancestral protein is efficient for &#x0201c;turning on&#x0201d; or &#x0201c;turning off&#x0201d; the same set of target genes to fulfill the photoperiod requirement in different plants or in different seasons of the same <span class="trait">plant.</span> The two <italic> <span class="gene">E1</span> </italic> copies in pigeon pea that only diverged for ~3.9 Mya support this notion. Interestingly, the predicted product from <italic> CcE1L2 </italic> contained the extra helix, whereas that from <italic> CcE1L1 </italic> was devoid of the extra helix. Moreover, <italic> K </italic> <sub> a </sub> / <italic> K </italic> <sub> s </sub> value between the gene pair was almost 1 (0.94, <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S1 </xref> ), implying the lack of purifying selection between two genes or evolution of different functions. We infer that both copies may exert different functions considering the distinct structures between the two concurrent copies of <italic> <span class="gene">E1</span> </italic> homologs in pigeon pea. Nevertheless, this result provides a good example to investigate the functional diversity in relation to structural variation.
					</p>
				
					<p>&nbsp;&nbsp;
						The transition from vegetative to reproductive growth is a complex process that requires coordination among numerous genes, as well as the redistribution of resources between reproductive and vegetative organs. Consequently, late flowering is logically associated with increased <span class="trait">plant</span> <span class="trait">height</span> (see <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs S3 </xref> and <xref ref-type="supplementary-material" rid="S1"> S4 </xref> ). Whether the reduced <span class="trait">plant</span> size accompanying early flowering is a result of reduced available resources for growth or an intentional inhibition of growth occurs to ensure flower development is a key question in the relationship of flowering development and growth. No flower phenotype was observed when <italic> MtE1L </italic> was overexpressed in soybean. This finding is unexpected because the relevant gene seemed to function in Medicago. More surprisingly, the dwarf phenotype was observed in the overexpression lines. This observation clearly indicates that the retarded growth was not a consequence of early flowering. Instead, this type of growth might be a part of the <italic> MtE1L </italic> regulation. The role and mechanism in flowering and <span class="trait">plant</span> development of <italic> MtE1L </italic> , and other members of the <italic> <span class="gene">E1</span> </italic> family merit further studies. Taken together, putative molecular networks of photoperiodic flowering regulation are proposed in legumes according to variations in photoperiodic response ( <xref ref-type="fig" rid="f7"> Fig. 7 </xref> ). Networks of photoperiodic flowering regulation are conserved in short-day plants soybean and common bean, in which <italic> <span class="gene">E1</span> </italic> or <italic> PvE1L </italic> is highly expressed and suppresses the expression of the florigen genes <italic> <span class="gene">GmFT2a/5a</span> </italic> or <italic> PvFTs </italic> under LD conditions. However, under SD conditions, the expression of both genes is suppressed, and the florigens are upregulated. On the other hand, in long-day <span class="trait">plant</span> Medicago, the function of <italic> MtE1L </italic> is shifted to flowering promoting. However, in the proposed flowering networks in legume, some putative interactions need to be tested or confirmed in the future study, e.g. how <italic> PvE1L </italic> and <italic> MtE1L </italic> genes regulate florigen genes in their native species. Nevertheless, the functional diversification of <italic> <span class="gene">E1</span> </italic> family genes revealed in this study and further characterization of flowering networks will provide insights in natural selection, domestication of important traits as well as genome evolution in legume.
					</p>
				
					<p>&nbsp;&nbsp;
						A series of <span class="trait">plant</span> overexpression and expression vectors were constructed for genetic transformation. All vectors contained the <italic> bar </italic> gene for selection. According to the nature of the intron-free <italic> <span class="gene">E1</span> </italic> family genes, <italic> <span class="gene">E1</span> </italic> , <italic> PvE1L </italic> , and <italic> MtE1L </italic> were amplified from Harosoy (soybean), Furongdidou (common bean) and R108 (Medicago), respectively, using gene-specific primers to which <italic> Xba </italic> I and <italic> Sac </italic> I cloning sites were manually introduced. The fragments of PCR products and <italic> pTF101.1-GmFT2a </italic> vector <xref ref-type="bibr" rid="b23"> 23 </xref> containing a cauliflower mosaic virus <italic> 35S </italic> promoter ( <italic> CaMV 35S </italic> ) digested by <italic> Xba </italic> I <italic> /Sac </italic> I were assembled together to construct <italic> pTF101.1 </italic> - <italic> <span class="gene">E1</span> </italic> , <italic> pTF101.1 </italic> - <italic> PvE1L </italic> , and <italic> pTF101.1 </italic> - <italic> MtE1L </italic> . These vectors were transferred into <italic> Agrobacterium tumefaciens EHA101 </italic> . Meanwhile, another <span class="trait">plant</span> expression vector <italic> pMDC123-E1 </italic> <xref ref-type="bibr" rid="b37"> 37 </xref> , in which <italic> <span class="gene">E1</span> </italic> was driven by the <italic> <span class="gene">E1</span> </italic> native promoter, was transferred into <italic> A. tumefaciens EHA105 </italic> and subsequently used for rice genetic transformation. <italic> Agrobacterium </italic> -mediated cotyledon-node explant method was used for soybean genetic transformation <xref ref-type="bibr" rid="b56"> 56 </xref> , whereas floral dip method was used for <italic> Arabidopsis </italic> genetic transformation <xref ref-type="bibr" rid="b57"> 57 </xref> , Moreover, <italic> Agrobacterium </italic> -mediated co-cultivation method was used for rice genetic transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						Seeds from transgenic rice lines were harvested and dried at 37&#x02009;&#x000b0;C. Then, the seeds were germinated in water under SD (10&#x02009;h light/14&#x02009;h dark) conditions. The plants were transplanted into the soil under SD conditions when seedling shoots were about 5&#x02013;6&#x02009;cm <span class="trait">height.</span> T <sub> 0 </sub> , T <sub> 1 </sub> , and T <sub> 2 </sub> transgenic lines were verified by semi-quantity RT-PCR with <italic> <span class="gene">E1</span> </italic> -specific primers (see <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Flowering phenotype between WT-DN lines and the <italic> PvE1L </italic> ( <italic> PvE1L </italic> #L2) transgenic issue under LD ( <bold> a&#x02013;c) </bold> and SD ( <bold> d&#x02013;f) </bold> conditions, respectively. ( <bold> a </bold> , <bold> d) </bold> WT-DN plants exhibiting flowers and pods at the axils of trifoliate leaves at approximately 35 and 24 DAE under LD and SD conditions, respectively. Close-up images of <bold> (b) </bold> WT-DN <span class="trait">plant</span> with flowers, and ( <bold> c) </bold> one <italic> PvE1L </italic> transgenic <span class="trait">plant</span> with flower buds under LD contidions. Close-up images of ( <bold> e) </bold> WT-DN plants with a pod (approximately 8&#x02009;mm in length), and ( <bold> f) </bold> <italic> PvE1L </italic> #L2 with flower buds at the axils of trifoliate leaves under SD conditions. ( <bold> g) </bold> Days to flowering in WT-DN line and <italic> PvE1L </italic> #L2. Herbicide-resistant T <sub> 3 </sub> plants of <italic> PvE1L </italic> #L2 were grown for flowering time investigation. Values represent the average of six replicates&#x02009;+&#x02009;s.d. P-value was determined using two-tailed Students&#x02019; <italic> t </italic> -test. Similar results were observed in three separate experiments. ( <bold> h) </bold> Expression of native <italic> e1-as </italic> in WT-DN and <italic> PvE1L </italic> overexpression lines under LD and SD conditions. ( <bold> i </bold> , <bold> j) </bold> Expression of <italic> <span class="gene">GmFTs</span> </italic> ( <italic> <span class="gene">GmFT2a</span> </italic> , <italic> <span class="gene">GmFT5a</span> </italic> , and <italic> <span class="gene">GmFT4</span> </italic> ) in the third fully expanded trifoliate leaves of WT-DN and <italic> PvE1L </italic> transgenic lines under LD and SD conditions, respectively. ( <bold> k) </bold> Expression of <italic> PvE1L </italic> in WT-DN and transgenic plants. Relative expression levels were analyzed by quantitative RT-PCR and normalized to <italic> TUA5 </italic> . Values represent the average of three biological replicates +s.d. *, **, and ***indicate significant differences between transgenic lines and WT-DN plants at P&#x02009;&#x0003c;&#x02009;0.05, P&#x02009;&#x0003c;&#x02009;0.01, and P&#x02009;&#x0003c;&#x02009;0.001, respectively.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hydrogen peroxide spraying alleviates <span class="trait">drought</span> stress in soybean plants.</h2>
			<h3>PMID:21377755  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ishibashi Y(1), Yamaguchi H, Yuasa T, Iwaya-Inoue M, Arima S, Zheng SH.
				<br/> 
				年份：2011 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To ascertain the effect of exogenously applied hydrogen peroxide (H2O2) on <span class="trait">drought</span> stress, we examined whether the spraying of soybean leaves with H2O2 would alleviate the symptoms of <span class="trait">drought</span> stress. Pre-treatment by spraying leaves with H2O2 delayed foliar wilting caused by <span class="trait">drought</span> stress compared to leaves sprayed with distilled water (DW). Additionally, the relative water content of drought-stressed leaves pre-treated with H2O2 was higher than that of leaves pre-treated with DW. Therefore, we analyzed the effect of H2O2 spraying on photosynthetic parameters and on the biosynthesis of oligosaccharides related to water retention in leaves during <span class="trait">drought</span> stress. Under conditions of <span class="trait">drought</span> stress, the net photosynthetic rate and stomatal conductance of leaves pre-treated with H2O2 were higher than those of leaves pre-treated with DW. In contrast to DW spraying, H2O2 spraying immediately caused an increase in the mRNA levels of d-myo-inositol 3-phosphate synthase 2 <span class="gene">(GmMIPS2)</span> and galactinol synthase (GolS), which encode key enzymes for the biosynthesis of oligosaccharides known to help plants tolerate <span class="trait">drought</span> stress. In addition, the levels of myo-inositol and galactinol were higher in H2O2-treated leaves than in DW-treated leaves. These results indicated that H2O2 spraying enabled the soybean <span class="trait">plant</span> to avoid <span class="trait">drought</span> stress through the maintenance of leaf water content, and that this water retention was caused by the promotion of oligosaccharide biosynthesis rather than by rapid stomatal closure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Candidate gene analysis for determinacy in pigeonpea ( <italic> Cajanus </italic> spp.)</h2>
			<h3>PMID:25331300</h3>
				
					<h3>PMCID:4236620</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mir </surname>  <given-names> Reyazul Rouf </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kudapa </surname>  <given-names> Himabindu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Srikanth </surname>  <given-names> Sandhya </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saxena </surname>  <given-names> Rachit K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharma </surname>  <given-names> Ashutosh </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Azam </surname>  <given-names> Sarwar </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saxena </surname>  <given-names> Kulbhushan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Varma Penmetsa </surname>  <given-names> R. </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K. </given-names>  </name>  <address>  <email> r.k.varshney@cgiar.org </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <aff id="Aff1">  <label/> International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru, 502 324 Hyderabad India  </aff>  <aff id="Aff2">  <label/> Division of Plant Breeding and Genetics, Shere-Kashmir University of Agricultural Sciences and Technology of Jammu (SKUAST-J), Chatha, 180 009 Jammu India  </aff>  <aff id="Aff3">  <label/> Institute of Molecular, Cell, and Systems Biology, College of Medical, Veterinary, and Life Sciences, University of Glasgow, Glasgow, G12 8QQ UK  </aff>  <aff id="Aff4">  <label/> Department of Plant Pathology, University of California-Davis, Davis, CA 95616 USA  </aff>  <aff id="Aff5">  <label/> School of Plant Biology and Institute of Agriculture, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009 Australia  </aff>
				<br/> 
				年份：<day> 1 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several approaches like genetic linkage analysis, candidate gene association analysis, and heterologous transformation have been used to test for the candidacy of homologous genes from <italic> Arabidopsis </italic> into other crop species like soybean (Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ). These studies revealed that flowering time/flowering pattern/determinacy has been selected long ago by breeders in combination with photoperiod insensitivity to obtain varieties with shorter flowering period, earlier maturation and ease of mechanized harvest (Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ). Genetic mechanism responsible for these traits has been uncovered in model <span class="trait">plant</span> <italic> Arabidopsis </italic> ( <italic> Arabidopsis </italic> <italic> thaliana </italic> ), pea ( <italic> Pisum sativum </italic> ), soybean ( <italic> Glycine max </italic> ), common bean ( <italic> Phaseolus vulgaris </italic> ) etc. (Foucher et al. <xref ref-type="bibr" rid="CR6"> 2003 </xref> ; Hecht et al. <xref ref-type="bibr" rid="CR11"> 2005 </xref> ; Kwak et al. <xref ref-type="bibr" rid="CR17"> 2008 </xref> ; Liu et al. <xref ref-type="bibr" rid="CR20"> 2010 </xref> ; Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ). In some cases it was proved that determinacy is controlled by a single gene, whereas in other studies more than one gene was found responsible for the transition of different growth habits (Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ). In pea, it was shown that the determinate mutant ( <italic> det </italic> ) is caused by mutations in a homologue of the <italic> Arabidopsis </italic> <italic> TFL1 </italic> gene (Foucher et al. <xref ref-type="bibr" rid="CR6"> 2003 </xref> ). In soybean, the gene responsible for determinacy &#x0201c; <italic> <span class="gene">GmTfl1</span> </italic> &#x0201d; was isolated and found to complement the functions of <italic> TFL1 </italic> in <italic> Arabidopsis </italic> (Liu et al. <xref ref-type="bibr" rid="CR20"> 2010 </xref> ; Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ). Similarly, in common bean, it was proved that gene &#x0201c; <italic> PvTFL1y </italic> &#x0201d; co-segregated with the determinacy locus &#x0201c; <italic> fin </italic> &#x0201d; (Kwak et al. <xref ref-type="bibr" rid="CR17"> 2008 </xref> ) and later the same was validated and found as a functional homolog of <italic> Arabidopsis </italic> <italic> TFL1 </italic> gene (Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ). In pigeonpea, both indeterminate (IDT) and determinate (DT) type flowering pattern exist (Mir et al. <xref ref-type="bibr" rid="CR24"> 2012b </xref> ). Wild relatives and most of the cultivars have indeterminate growth habit and therefore, it is believed that determinate forms of pigeonpea were selected by farmers or breeders during pigeonpea domestication process or breeding. The availability of determinate growth habit genotypes having initial vigor and tolerance to <span class="trait">drought</span> and water logging have been found advantageous over indeterminate types for environments with moderate growth (5&#x02013;6&#x000a0;t&#x000a0;ha <sup> &#x02212;1 </sup> ), while as IDT type lines have been found suitable for environments with high (7&#x02013;8&#x000a0;t&#x000a0;ha <sup> &#x02212;1 </sup> ) growth potential (Singh and Oswalt <xref ref-type="bibr" rid="CR36"> 1992 </xref> ). However, only some linked markers associated with flowering pattern/determinacy have been reported recently in pigeonpea (Mir et al. <xref ref-type="bibr" rid="CR24"> 2012b </xref> ). The present study reports the isolation of seven genes and identification of likely candidate gene &#x0201c; <italic> CcTFL1 </italic> &#x0201d; for determinacy in pigeonpea using candidate gene sequencing, linkage mapping based association analysis, comparative genomics and differential gene expression approaches.
					</p>
				
					<p>&nbsp;&nbsp;
						Determinacy is one of the most important and widely studied domesticated traits in flowering plants. In order to obtain early maturing varieties with shorter flowering period, determinacy trait has been selected via domestication process together with photoperiod insensitivity (Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ). Several studies have been conducted in the past in model <span class="trait">plant</span> <italic> Arabidopsis </italic> , pea, soybean, common bean, etc. to identify the genetic mechanism that is responsible for different forms of growth habit (Foucher et al. <xref ref-type="bibr" rid="CR6"> 2003 </xref> ; Hecht et al. <xref ref-type="bibr" rid="CR11"> 2005 </xref> ; Kwak et al. <xref ref-type="bibr" rid="CR17"> 2008 </xref> ; Liu et al. <xref ref-type="bibr" rid="CR20"> 2010 </xref> ; Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ; Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ). In some cases it has been proved that determinacy is controlled by single gene, whereas in other studies more than one gene have been found responsible for the transition from vegetative growth to reproductive growth (Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ). In pea, it has been shown recently that the determinate mutant ( <italic> det </italic> ) is caused by mutations in a homologue of the <italic> Arabidopsis </italic> <italic> TFL1 </italic> gene. These mutations are synonymous or non-synonymous substitutions at the junction between an exon and an intron resulting in splicing failure (Foucher et al. <xref ref-type="bibr" rid="CR6"> 2003 </xref> ). In soybean, the gene responsible for determinacy &#x0201c; <italic> <span class="gene">GmTfl1</span> </italic> &#x0201d; has been isolated and found to complement the functions of <italic> TFL1 </italic> in <italic> Arabidopsis </italic> (Liu et al. <xref ref-type="bibr" rid="CR20"> 2010 </xref> ; Tian et al. <xref ref-type="bibr" rid="CR40"> 2010 </xref> ). Similarly, in common bean, it has been proved that gene &#x0201c; <italic> PvTFL1y </italic> &#x0201d; co-segregated with the determinacy locus &#x0201c; <italic> fin </italic> &#x0201d; (Kwak et al. <xref ref-type="bibr" rid="CR17"> 2008 </xref> ) and later the same has been validated and found as a functional homolog of <italic> Arabidopsis </italic> <italic> TFL1 </italic> gene (Repinski et al. <xref ref-type="bibr" rid="CR35"> 2012 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>