<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Pod dehiscence</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Pod dehiscence</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Divergence of flowering genes in soybean.</h2>
			<h3>PMID:23107921  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim MY(1), Shin JH, Kang YJ, Shim SR, Lee SH.
				<br/> 
				年份：2012 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genome sequences were blasted with Arabidopsis thaliana regulatory genes involved in photoperioddependent flowering. This approach enabled the identification of 118 genes involved in the flowering pathway. Two genome sequences of cultivated (Williams 82) and wild (IT182932) soybeans were employed to survey functional DNA variations in the flowering-related homologs. Forty genes exhibiting nonsynonymous substitutions between G. max and G. soja were catalogued. In addition, 22 genes were found to co-localize with QTLs for six traits including flowering time, first flower, <span class="trait">pod</span> maturity, beginning of <span class="trait">pod,</span> reproductive period, and seed filling period. Among the genes overlapping the QTL regions, two LHY/CCA1 genes, GI and SFR6 contained amino acid changes. The recently duplicated sequence regions of the soybean genome were used as additional criteria for the speculation of the putative function of the homologs. Two duplicated regions showed redundancy of both flowering-related genes and QTLs. ID 12398025, which contains the homeologous regions between chr 7 and chr 16, was redundant for the LHY/CCA1 and SPA1 homologs and the QTLs. Retaining of the CRY1 gene and the <span class="trait">pod</span> maturity QTLs were observed in the duplicated region of ID 23546507 on chr 4 and chr 6. Functional DNA variation of the LHY/CCA1 gene <span class="gene">(Glyma07g05410)</span> was present in a counterpart of the duplicated region on chr 7, while the gene <span class="gene">(Glyma16g01980)</span> present in the other portion of the duplicated region on chr 16 did not show a functional sequence change. The gene list catalogued in this study provides primary insight for understanding the regulation of flowering time and maturity in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping and use of QTLs controlling <span class="trait">pod</span> <span class="trait">dehiscence</span> in soybean</h2>
			<h3>PMID:23136494</h3>
				
					<h3>PMCID:3406785</h3>
					<h4>期刊:Breeding Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Funatsuki </surname>  <given-names> Hideyuki </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-554"> 1 </xref>  <xref rid="c1-bs-61-554" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hajika </surname>  <given-names> Makita </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-554"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamada </surname>  <given-names> Tetsuya </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-554"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Suzuki </surname>  <given-names> Masaya </given-names>  </name>  <xref ref-type="aff" rid="af3-bs-61-554"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hagihara </surname>  <given-names> Seiji </given-names>  </name>  <xref ref-type="aff" rid="af4-bs-61-554"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Yoshinori </given-names>  </name>  <xref ref-type="aff" rid="af4-bs-61-554"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujita </surname>  <given-names> Shohei </given-names>  </name>  <xref ref-type="aff" rid="af5-bs-61-554"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ishimoto </surname>  <given-names> Masao </given-names>  </name>  <xref ref-type="aff" rid="af6-bs-61-554"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujino </surname>  <given-names> Kaien </given-names>  </name>  <xref ref-type="aff" rid="af3-bs-61-554"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 9 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A major QTL was repeatedly identified on the linkage group (LG) J (chromosome 16) in independent studies using segregating populations derived from a cross between shattering-resistant (SR) and -susceptible (SS) parents. The first evidence was provided by <xref rid="b1-bs-61-554" ref-type="bibr"> Bailey <italic> et al. </italic> (1997) </xref> , who used a population of recombinant inbred lines (RILs) created from a cross between a shattering-resistant, North American cultivar Young and a shattering-susceptible accession introduced from Japan (PI 416937). They mapped a major QTL on LG J ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> ), although the position was inaccurate because of the sparseness of RFLP markers used and no use of interval mapping. Subsequently, a QTL with a large effect was identified also on LG J in populations of RILs and F <sub> 2 </sub> plants, derived from a cross between <span class="gene">SS</span> and SR Japanese cultivars, Toyomusume and Hayahikari ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> , <xref rid="b6-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> 2005 </xref> , <xref rid="b7-bs-61-554" ref-type="bibr"> 2006 </xref> ). The shattering <span class="trait">resistance</span> of Hayahikari was derived from a Thai cultivar, SJ2. Since all of the SSR markers available were used in that study, the position of the QTL could be narrowed down to &#x0003c;5 cM genetic distance, which was included in the region of the major QTL suggested by <xref rid="b1-bs-61-554" ref-type="bibr"> Bailey <italic> et al. </italic> (1997) </xref> . Furthermore, <xref rid="b13-bs-61-554" ref-type="bibr"> Kang <italic> et al. </italic> (2009) </xref> found a major QTL on LG J in a population involving Korean cultivars, Keunolkong (SS) and Sinpaldalkong (SR) ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> ). The candidate position overlapped those identified previously by <xref rid="b1-bs-61-554" ref-type="bibr"> Bailey <italic> et al. </italic> (1997) </xref> and <xref rid="b7-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> (2006) </xref> . Finally, <xref rid="b33-bs-61-554" ref-type="bibr"> Yamada <italic> et al. </italic> (2009) </xref> conducted allelism tests using two crosses between SR and <span class="gene">SS</span> cultivars, Kariyutaka (SS) &#x000d7; Hayahikari (SR) and Wasekogane (SR) &#x000d7; Yukihomare (SS). The two resistant cultivars had no common ancestors. Major QTLs were found in almost the same position of LG J as the previously-detected QTLs ( <xref rid="b1-bs-61-554" ref-type="bibr"> Bailey <italic> et al. </italic> 1997 </xref> , <xref rid="b7-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> 2006 </xref> ), suggesting that the <span class="trait">resistance</span> in the two parents was controlled by the same QTLs. These findings suggest a QTL on LG J (chromosome 16) in cultivated soybean that roughly determines whether the cultivar is SR. This locus was designated <italic> qPDH1 </italic> by <xref rid="b8-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> (2008) </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, relatively little commonality was found among the positions for minor QTLs ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> ). <xref rid="b1-bs-61-554" ref-type="bibr"> Bailey <italic> et al. </italic> (1997) </xref> found QTLs on LGs D1b, E, L and N in a cross between Young and PI416937. <xref rid="b13-bs-61-554" ref-type="bibr"> Kang <italic> et al. </italic> (2009) </xref> identified QTLs on LGs A1, B2, D1b, L and O in crosses between Keunolkong (SS) and Sinpaldalkong (SR) and between Keunolkong (SS) and Iksan 10 (SR). <xref rid="b33-bs-61-554" ref-type="bibr"> Yamada <italic> et al. </italic> (2009) </xref> showed the possible presence of a QTL on LG A2 in a cross between Toyomusume (SS) and Harosoy (SR). Since these studies differed in cross combination, growth conditions and method for evaluating shattering <span class="trait">resistance,</span> the minor QTLs might be dependent on the genetic background and/or the environment for plant growth and drying the pods. These QTLs may be useful for fine-tuning shattering <span class="trait">resistance,</span> since in some cases moderate shattering <span class="trait">resistance</span> is required. In addition, in the segregating population derived from an <span class="gene">SS</span> cultivar, Keunolkong, and an SR cultivar, Iksan 10, only minor QTLs were identified; no major QTL was detected either on LG J, or on other LGs ( <xref rid="b13-bs-61-554" ref-type="bibr"> Kang <italic> et al. </italic> 2009 </xref> ). This suggests that an SR cultivar could be developed by pyramiding shattering <span class="trait">resistance</span> alleles at minor QTLs.
					</p>
				
					<p>&nbsp;&nbsp;
						As mentioned above, elimination or reduction of natural seed dispersal is regarded as the single most important domestication trait ( <xref rid="b5-bs-61-554" ref-type="bibr"> Fuller 2007 </xref> ); therefore, to study the domestication process of soybean, it is of importance to dissect the trait of <span class="trait">pod</span> <span class="trait">dehiscence</span> genetically by comparing cultivated and wild soybeans. <xref rid="b23-bs-61-554" ref-type="bibr"> Saxe <italic> et al. </italic> (1996) </xref> found three QTLs, two on LG J and one on LG D1b ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> ). Both QTLs on LG J were mapped quite far from <italic> qPDH1 </italic> ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> and <xref rid="f1-bs-61-554" ref-type="fig"> Fig. 1 </xref> ). These QTLs are likely to differ from <italic> qPDH </italic> , although mapping accuracy might not be very high because of the limited numbers of markers used. A QTL was also detected on LG J in another study ( <xref rid="b18-bs-61-554" ref-type="bibr"> Liu <italic> et al. </italic> 2007 </xref> ). The locus was, in contrast, mapped near <italic> qPDH1 </italic> ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> and <xref rid="f1-bs-61-554" ref-type="fig"> Fig. 1 </xref> ), indicating that it is possibly identical to <italic> qPDH1 </italic> ; however, its effect was relatively small ( <xref rid="t1-bs-61-554" ref-type="table"> Table 1 </xref> ). The line used in the latter study, Tokei 780, is <span class="gene">SS</span> in common cultivated soybean lines (Y. Tanaka unpublished data) and is presumed to have an <span class="gene">SS</span> allele at <italic> qPDH1 </italic> , although the genotype of this line at the locus on LG J was determined to be SR in the <italic> G. max </italic> &#x000d7; G. soja population. These findings suggest the presence of multiple alleles at <italic> qPDH1 </italic> or the presence of another QTL near <italic> qPDH1 </italic> . No other significant QTLs for <span class="trait">pod</span> <span class="trait">dehiscence</span> were detected in the study by <xref rid="b18-bs-61-554" ref-type="bibr"> Liu <italic> et al. </italic> (2007) </xref> , although the difference between cultivated and wild soybeans seen in shattering <span class="trait">resistance</span> was considerably large. This suggests the presence of a number of QTLs for <span class="trait">pod</span> <span class="trait">dehiscence.</span> Taken together, several loci are likely to be involved in the domestication of wild soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Since the major QTL, <italic> qPDH1 </italic> , had a marked effect and a recessive allele controlled shattering <span class="trait">resistance</span> ( <xref rid="b7-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> 2006 </xref> ), the development of useful DNA markers linked with <italic> qPDH1 </italic> was desired for the establishment of MAS for shattering <span class="trait">resistance.</span> Using a residual heterozygous line (RHL), which was first defined and used for fine mapping of a QTL for flowering time by <xref rid="b36-bs-61-554" ref-type="bibr"> Yamanaka <italic> et al. </italic> (2005) </xref> , <xref rid="b8-bs-61-554" ref-type="bibr"> Funatsuki <italic> et al. </italic> (2008) </xref> confirmed that the major QTL was located between SSR markers Sat_093 and Sat_366 on LG J. Furthermore, a large segregating population was screened for recombinants between Sat_093 and Sat.366 ( <xref rid="b28-bs-61-554" ref-type="bibr"> Suzuki <italic> et al. </italic> 2010 </xref> ). The genotypes of the recombinants were determined with SSR markers designed using the Williams 82 genome sequence (Phytozome). Analysis of the genotype and the phenotype of each recombinant narrowed down the candidate region of <italic> qPDH1 </italic> to 134 kb ( <xref rid="b28-bs-61-554" ref-type="bibr"> Suzuki <italic> et al. </italic> 2010 </xref> ). In this region, <xref rid="b28-bs-61-554" ref-type="bibr"> Suzuki <italic> et al. </italic> (2010) </xref> found several insertion/deletion (In/Del) variations between SR and <span class="gene">SS</span> parents, which were used to develop three DNA markers (SRM0, SRM1 and SRM2) for <italic> qPDH1 </italic> . A survey of the genotype of various SR and <span class="gene">SS</span> accessions at these markers revealed that all shattering-resistant accessions had an identical genotype at these markers, which was distinct to those of <span class="gene">SS</span> accessions with a few exceptions at SRM2. These findings suggest that the markers developed can be used to select progeny derived from various cross combinations between SR and <span class="gene">SS</span> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						The eleven SR lines developed possess genetic backgrounds of representative cultivars grown on the main island, and Kyushu and Shikoku regions of Japan ( <xref rid="b34-bs-61-554" ref-type="bibr"> Yamada <italic> et al. </italic> 2010 </xref> ). These cultivars are frequently used as parents for crossing, indicating that we now have a variety of breeding materials with shattering <span class="trait">resistance</span> in Japan. Since the shattering <span class="trait">resistance</span> of these lines is basically granted by the <italic> qPDH1 </italic> locus, half of the progeny in advanced generations should be shattering-resistant in the case of the cross combination of SR and <span class="gene">SS</span> lines, suggesting the relatively straightforward selection of SR lines with other favorable traits even without MAS. In addition, rapid and efficient MAS systems are available in soybean (e.g., <xref rid="b24-bs-61-554" ref-type="bibr"> Sayama <italic> et al. </italic> 2011 </xref> ), enabling us to treat shattering <span class="trait">resistance</span> as a target trait of MAS along with other important traits such as disease <span class="trait">resistance.</span> We expect that many SR cultivars will be released in the near future using these lines and markers, resulting in a steep increase in the growing area of SR cultivars and a marked reduction of <span class="trait">yield</span> loss due to <span class="trait">pod</span> <span class="trait">dehiscence</span> in Japan.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic region in which <italic> qPDH1 </italic> is located has been narrowed down to 134 kb, where no more than 10 candidate genes are presumably present ( <xref rid="b28-bs-61-554" ref-type="bibr"> Suzuki <italic> et al. </italic> 2010 </xref> ). A large segregating population and a complementation test may allow us to identify the gene underlying <italic> qPDH1 </italic> . As mentioned above, no sequence showed any homology to the genes associated with <span class="trait">pod</span> <span class="trait">dehiscence</span> that have been revealed in <italic> Arabidopsis </italic> . Characterization of <italic> qPDH1 </italic> at the molecular level will lead to an understanding of the mechanism of <span class="trait">pod</span> <span class="trait">dehiscence</span> specific to soybean or legumes. This information may also be useful for breeding other legumes since <span class="trait">pod</span> <span class="trait">dehiscence</span> is also a problem in the breeding of legume crops such as cowpea and lupin, especially when using <span class="gene">SS</span> wild relatives (e.g., <xref rid="b15-bs-61-554" ref-type="bibr"> Li <italic> et al. </italic> 2010 </xref> , <xref rid="b20-bs-61-554" ref-type="bibr"> Mohammed <italic> et al. </italic> 2010 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>