<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Pod number</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Pod number</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, under saline field conditions the NIL-T lines had significantly greater plant seed weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher seed weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dopamine on root growth and enzyme activity in soybean seedlings.</h2>
			<h3>PMID:23838960  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guidotti BB(1), Gomes BR, Siqueira-Soares Rde C, Soares AR, Ferrarese-Filho O.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In general, the length, fresh weight and dry weight of roots, cell viability, <span class="gene">PAL</span> and <span class="trait">POD</span> activities decreased, while SOD activities increased after dopamine treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:L-DOPA increases lignification associated with Glycine max root growth-inhibition.</h2>
			<h3>PMID:17195115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Soares AR(1), Ferrarese Mde L, Siqueira Rde C, Böhm FM, Ferrarese-Filho O.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In general, the length, fresh weight, and dry weight of the roots decreased, whereas <span class="gene">PAL</span> and <span class="trait">POD</span> activities and phenolic compound and lignin content increased after L-DOPA treatments.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lignification and related enzymes in Glycine max root growth-inhibition by ferulic acid.</h2>
			<h3>PMID:15303323  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：dos Santos WD(1), Ferrarese Mde L, Finger A, Teixeira AC, Ferrarese-Filho O.
				<br/> 
				年份：2004 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Length, fresh weight, and dry weight of roots decreased, while soluble and cell wall bound <span class="trait">POD</span> activity, <span class="gene">PAL</span> activity, and lignin content increased after ferulic acid treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxidase and phenylalanine ammonia-lyase activities, phenolic acid contents, and allelochemicals-inhibited root growth of soybean.</h2>
			<h3>PMID:12125206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Herrig V(1), Ferrarese Mde L, Suzuki LS, Rodrigues JD, Ferrarese-Filho O.
				<br/> 
				年份：2002
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The results are discussed on the basis of the role of these compounds on phenylpropanoid metabolism and root growth and suggest that the effects caused on <span class="trait">POD</span> and <span class="gene">PAL</span> activities are some of the many mechanisms by which allelochemicals influence plant growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTLomics in Soybean: A Way Forward for Translational Genomics and Breeding</h2>
			<h3>PMID:28066449</h3>
				

				
					<h3>PMCID:5174554</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kumawat </surname>  <given-names> Giriraj </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/311123/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gupta </surname>  <given-names> Sanjay </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374813/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ratnaparkhe </surname>  <given-names> Milind B. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/55589/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maranna </surname>  <given-names> Shivakumar </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374396/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Satpute </surname>  <given-names> Gyanesh K. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374178/overview"/>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						1 </italic> ( <italic> <span class="gene">Pdh1</span> </italic> ), which encodes a dirigent-like protein known to be involved in the lignifications which increases dehiscing forces by increasing torsion of dried <span class="trait">pod</span> walls
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Pdh1</span> </italic> is a novel gene identified for <span class="trait">pod</span> dehiscence and selection of nonsense mutation in this gene has played an important role in soybean domestication
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B152" ref-type="bibr"> 2016 </xref> ) identified that the copy <span class="trait">number</span> variations in candidate genes (HSP, <italic> <span class="gene">Glyma20g19680</span> </italic> ; ammonium transporter, <italic> <span class="gene">Glyma20g21030</span> </italic> ; ethylene receptor, <italic> <span class="gene">Glyma20g21780</span> </italic> ) were associated with protein content
					</p>
				
					<p>&nbsp;&nbsp;
						The comparative analysis shows increased copy <span class="trait">number</span> of lipid transfer protein (LPT; <italic> <span class="gene">Glyma16g31780,</span> <span class="gene">Glyma16g31840</span> </italic> , and <italic> <span class="gene">Glyma16g31540</span> </italic> ) in high oil lines, whereas the negative regulators of lipid biosynthesis genes (ABC transporter, <italic> <span class="gene">Glyma03g36310</span> </italic> , Lipase 3, and <italic> <span class="gene">Glyma13g04561</span> </italic> ) showed more copies in low oil lines
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B175" ref-type="bibr"> 2013 </xref> ) identified two candidate genes underlying a major QTL for root not nematode <span class="trait">resistance</span> mapped to bin 10 of <span class="gene">Gm10</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In resistant genotype Peking, a dominant <span class="trait">resistance</span> gene ( <italic> <span class="gene">Rhg4</span> </italic> ) was identified that was linked to <italic> I </italic> locus underlying seed coat color (Matson and Williams, <xref rid="B98" ref-type="bibr"> 1965 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B94" ref-type="bibr"> 2012 </xref> ) identified a gene underlying <italic> <span class="gene">Rhg4</span> </italic> locus on <span class="gene">Gm08,</span> a major QTL contributing to <span class="trait">resistance</span> to SCN
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized the <italic> <span class="gene">GmRLK18-1</span> </italic> , the gene underlying <italic> <span class="gene">Rhg1</span> </italic> locus, in transgenic and heterozygous plants and demonstrated that the dominant allele confers reported pleiotropic <span class="trait">resistance</span> to SCN and sudden death syndrome (SDS) caused by <italic> Fusarium virguliforme </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						In the absence of <italic> <span class="gene">Rhg4</span> </italic> , the <italic> <span class="gene">GmRLK18-1</span> </italic> had provided partial <span class="trait">resistance</span> to SCN and conferred nearly complete <span class="trait">resistance</span> to SDS, whereas, in the presence of <italic> <span class="gene">Rhg4</span> </italic> , the plants with the <italic> <span class="gene">GmRLK18-1</span> </italic> transgene were fully resistant to SCN as well as SDS (Srour et al
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B26" ref-type="bibr"> 2012 </xref> ) identified that SCN <span class="trait">resistance</span> mediated by the soybean QTL <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 is conferred by copy <span class="trait">number</span> variation of a 31-kilobase segment encoding an amino acid transporter ( <italic> <span class="gene">Glyma18g02580</span> </italic> ), an &#x003b1;-SNAP (&#x003b1;-soluble N-ethylmaleimide-sensitive factor attachment protein) protein ( <italic> <span class="gene">Glyma18g02590</span> </italic> ), and a WI12 (wound-inducible domain) protein ( <italic> <span class="gene">Glyma18g02610</span> </italic> )
					</p>
				
					<p>&nbsp;&nbsp;
						Further, characterization of genetic variation at <italic> <span class="gene">Rhg1</span> </italic> across 41 diverse soybean accessions discovered that SCN <span class="trait">resistance</span> is associated with multicopy <italic> <span class="gene">Rhg1</span> </italic> haplotypes that form two distinct groups (Cook et al
					</p>
				
					<p>&nbsp;&nbsp;
						The high-copy-number <italic> <span class="gene">Rhg1</span> </italic> accessions reportedly contain a flexible <span class="trait">number</span> of copies (seven to 10) of the 31-kb <italic> <span class="gene">Rhg1</span> </italic> repeat whereas the low-copy-number <italic> <span class="gene">Rhg1</span> </italic> group contains three copies of the <italic> <span class="gene">Rhg1</span> </italic> repeat and a newly identified allele of &#x003b1;-SNAP (Cook et al
					</p>
				
					<p>&nbsp;&nbsp;
						A Taqman assay was also developed from the conserved region of the <italic> <span class="gene">Rhg1</span> </italic> locus for detection of copy <span class="trait">number</span> variation at <italic> <span class="gene">Rhg1</span> </italic> locus (Kadam et al
					</p>
				
					<p>&nbsp;&nbsp;
						QTLs analysis for BLP <span class="trait">resistance</span> revealed that Satt372 and Satt486 on <span class="gene">Gm17</span> were strongly associated with BLP <span class="trait">resistance</span> (Narvel et al
					</p>
				
					<p>&nbsp;&nbsp;
						A locus <italic> Rfs2/Rhg1 </italic> on <span class="gene">Gm18</span> underlay partial <span class="trait">resistance</span> to the spread of root infections by <italic> F
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized <italic> Rfs2 </italic> / <italic> <span class="gene">Rhg1</span> </italic> QTL on <span class="gene">Gm18</span> identifying a receptor like kinase ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) responsible for <span class="trait">resistance</span> to SDS and SCN
					</p>
				
					<p>&nbsp;&nbsp;
						Several of SNPs significantly associated with SDS <span class="trait">resistance</span> were within or adjacent to sequences annotated as plant disease <span class="trait">resistance</span> genes including the previously identified SDS <span class="trait">resistance</span> gene <italic> <span class="gene">GmRLK18-1</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						The peak SNP locus associated with SDS <span class="trait">resistance</span> was present in the coding region of the <italic> <span class="gene">SIK1</span> </italic> resulting in a non-synonymous mutation
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B86" ref-type="bibr"> 2016b </xref> ) conducted GWAS in 279 soybean accessions and identified seven candidate genes within 500-kb regions on <span class="gene">Gm13</span> that are probably involved in natural variations in partial <span class="trait">resistance</span> to <italic> P
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B97" ref-type="bibr"> 2015 </xref> ) identified a major QTL on <span class="gene">Gm08</span> that governed root traits (tap root length and lateral root <span class="trait">number)</span> and shoot length, and identified six transcription factors (MYB HD, TPR, C2H2 Zn, bZIP, GRAS, and Ring finger) and two key cell wall expansion-related genes which encode xyloglucan endo-transglycosylases as candidate genes in the confidence interval of the QTL
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">Pdh1</span> </italic> is a novel gene identified for <span class="trait">pod</span> dehiscence and selection of nonsense mutation in this gene has played an important role in soybean domestication.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B152" ref-type="bibr"> 2016 </xref> ) identified that the copy <span class="trait">number</span> variations in candidate genes (HSP, <italic> <span class="gene">Glyma20g19680</span> </italic> ; ammonium transporter, <italic> <span class="gene">Glyma20g21030</span> </italic> ; ethylene receptor, <italic> <span class="gene">Glyma20g21780</span> </italic> ) were associated with protein content.
					</p>
				
					<p>&nbsp;&nbsp;
						The comparative analysis shows increased copy <span class="trait">number</span> of lipid transfer protein (LPT; <italic> <span class="gene">Glyma16g31780,</span> <span class="gene">Glyma16g31840</span> </italic> , and <italic> <span class="gene">Glyma16g31540</span> </italic> ) in high oil lines, whereas the negative regulators of lipid biosynthesis genes (ABC transporter, <italic> <span class="gene">Glyma03g36310</span> </italic> , Lipase 3, and <italic> <span class="gene">Glyma13g04561</span> </italic> ) showed more copies in low oil lines.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B175" ref-type="bibr"> 2013 </xref> ) identified two candidate genes underlying a major QTL for root not nematode <span class="trait">resistance</span> mapped to bin 10 of <span class="gene">Gm10</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In resistant genotype Peking, a dominant <span class="trait">resistance</span> gene ( <italic> <span class="gene">Rhg4</span> </italic> ) was identified that was linked to <italic> I </italic> locus underlying seed coat color (Matson and Williams, <xref rid="B98" ref-type="bibr"> 1965 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B94" ref-type="bibr"> 2012 </xref> ) identified a gene underlying <italic> <span class="gene">Rhg4</span> </italic> locus on <span class="gene">Gm08,</span> a major QTL contributing to <span class="trait">resistance</span> to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized the <italic> <span class="gene">GmRLK18-1</span> </italic> , the gene underlying <italic> <span class="gene">Rhg1</span> </italic> locus, in transgenic and heterozygous plants and demonstrated that the dominant allele confers reported pleiotropic <span class="trait">resistance</span> to SCN and sudden death syndrome (SDS) caused by <italic> Fusarium virguliforme </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the absence of <italic> <span class="gene">Rhg4</span> </italic> , the <italic> <span class="gene">GmRLK18-1</span> </italic> had provided partial <span class="trait">resistance</span> to SCN and conferred nearly complete <span class="trait">resistance</span> to SDS, whereas, in the presence of <italic> <span class="gene">Rhg4</span> </italic> , the plants with the <italic> <span class="gene">GmRLK18-1</span> </italic> transgene were fully resistant to SCN as well as SDS (Srour et al.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B26" ref-type="bibr"> 2012 </xref> ) identified that SCN <span class="trait">resistance</span> mediated by the soybean QTL <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 is conferred by copy <span class="trait">number</span> variation of a 31-kilobase segment encoding an amino acid transporter ( <italic> <span class="gene">Glyma18g02580</span> </italic> ), an &#x003b1;-SNAP (&#x003b1;-soluble N-ethylmaleimide-sensitive factor attachment protein) protein ( <italic> <span class="gene">Glyma18g02590</span> </italic> ), and a WI12 (wound-inducible domain) protein ( <italic> <span class="gene">Glyma18g02610</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Further, characterization of genetic variation at <italic> <span class="gene">Rhg1</span> </italic> across 41 diverse soybean accessions discovered that SCN <span class="trait">resistance</span> is associated with multicopy <italic> <span class="gene">Rhg1</span> </italic> haplotypes that form two distinct groups (Cook et al.
					</p>
				
					<p>&nbsp;&nbsp;
						The high-copy-number <italic> <span class="gene">Rhg1</span> </italic> accessions reportedly contain a flexible <span class="trait">number</span> of copies (seven to 10) of the 31-kb <italic> <span class="gene">Rhg1</span> </italic> repeat whereas the low-copy-number <italic> <span class="gene">Rhg1</span> </italic> group contains three copies of the <italic> <span class="gene">Rhg1</span> </italic> repeat and a newly identified allele of &#x003b1;-SNAP (Cook et al.
					</p>
				
					<p>&nbsp;&nbsp;
						A Taqman assay was also developed from the conserved region of the <italic> <span class="gene">Rhg1</span> </italic> locus for detection of copy <span class="trait">number</span> variation at <italic> <span class="gene">Rhg1</span> </italic> locus (Kadam et al.
					</p>
				
					<p>&nbsp;&nbsp;
						QTLs analysis for BLP <span class="trait">resistance</span> revealed that Satt372 and Satt486 on <span class="gene">Gm17</span> were strongly associated with BLP <span class="trait">resistance</span> (Narvel et al.
					</p>
				
					<p>&nbsp;&nbsp;
						A locus <italic> Rfs2/Rhg1 </italic> on <span class="gene">Gm18</span> underlay partial <span class="trait">resistance</span> to the spread of root infections by <italic> F.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized <italic> Rfs2 </italic> / <italic> <span class="gene">Rhg1</span> </italic> QTL on <span class="gene">Gm18</span> identifying a receptor like kinase ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) responsible for <span class="trait">resistance</span> to SDS and SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						Several of SNPs significantly associated with SDS <span class="trait">resistance</span> were within or adjacent to sequences annotated as plant disease <span class="trait">resistance</span> genes including the previously identified SDS <span class="trait">resistance</span> gene <italic> <span class="gene">GmRLK18-1</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						The peak SNP locus associated with SDS <span class="trait">resistance</span> was present in the coding region of the <italic> <span class="gene">SIK1</span> </italic> resulting in a non-synonymous mutation.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B86" ref-type="bibr"> 2016b </xref> ) conducted GWAS in 279 soybean accessions and identified seven candidate genes within 500-kb regions on <span class="gene">Gm13</span> that are probably involved in natural variations in partial <span class="trait">resistance</span> to <italic> P.
					</p>
				
					<p>&nbsp;&nbsp;
						( <xref rid="B97" ref-type="bibr"> 2015 </xref> ) identified a major QTL on <span class="gene">Gm08</span> that governed root traits (tap root length and lateral root <span class="trait">number)</span> and shoot length, and identified six transcription factors (MYB HD, TPR, C2H2 Zn, bZIP, GRAS, and Ring finger) and two key cell wall expansion-related genes which encode xyloglucan endo-transglycosylases as candidate genes in the confidence interval of the QTL.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Neodiversification of homeologous <italic> CLAVATA1 </italic> -like receptor kinase genes in soybean leads to distinct developmental outcomes</h2>
			<h3>PMID:28827708</h3>
				

				
					<h3>PMCID:5566472</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mirzaei </surname>  <given-names> Saeid </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Batley </surname>  <given-names> Jacqueline </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> El-Mellouki </surname>  <given-names> Tarik </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferguson </surname>  <given-names> Brett J. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Gresshoff </surname>  <given-names> Peter M. </given-names>  </name>  <address>  <email> p.gresshoff@uq.edu.au </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9320 7537 </institution-id>  <institution-id institution-id-type="GRID"> grid.1003.2 </institution-id>  <institution/>  <institution> Centre for Integrative Legume Research, School of Agriculture and Food Sciences, The University of Queensland,  </institution>  </institution-wrap> St Lucia, Brisbane, QLD 4072 Australia  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 7910 </institution-id>  <institution-id institution-id-type="GRID"> grid.1012.2 </institution-id>  <institution/>  <institution> School of Biological Sciences, University of Western Australia,  </institution>  </institution-wrap> Crawley, WA 6009 Australia  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 1090 2313 </institution-id>  <institution-id institution-id-type="GRID"> grid.411026.0 </institution-id>  <institution/>  <institution> Department of Plant, Soil and Agricultural Systems, Southern Illinois University,  </institution>  </institution-wrap> Carbondale, IL 62901 USA  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.448905.4 </institution-id>  <institution/>  <institution> Department of Biotechnology, Institute of Science, High Technology and Environmental Sciences, Graduate University of Advanced Technology,  </institution>  </institution-wrap> Kerman, Iran  </aff>
				<br/> 
				年份：<day> 29 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phenotypes of <span class="trait">pod,</span> stem (as demonstrated by cotyledonary node <span class="trait">branching)</span> and nodulated root systems of the soybean wild type Forrest, and its TILLING-derived mutants <italic> , <span class="gene">Gmclv1a</span> </italic> ( <italic> S562L) </italic> and <italic> <span class="gene">Gmnark</span> </italic> ( <italic> W677* </italic> )
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		
	</div>
</body>

</html>