<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Pod number</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Pod number</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T set was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less <span class="trait">pod</span> wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant seed weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher seed weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dopamine on root growth and enzyme activity in soybean seedlings.</h2>
			<h3>PMID:23838960  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guidotti BB(1), Gomes BR, Siqueira-Soares Rde C, Soares AR, Ferrarese-Filho O.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, we investigated the effects of dopamine, an allelochemical exuded from the velvetbean (Mucuna pruriens L DC. var utilis), on the growth and cell viability of soybean (Glycine max L. Merrill) roots. We analyzed the effects of dopamine on superoxide dismutase, phenylalanine ammonia-lyase and cell wall-bound peroxidase activities as well as its effects on lignin contents in the roots. Three-day-old seedlings were cultivated in half-strength Hoagland nutrient solution (pH 6.0), without or with 0.25 to 1.0 mM dopamine, in a growth chamber (25°C, 12L:12D photoperiod, irradiance of 280 μmol m(-2) s(-1)) for 24 h. In general, the length, fresh weight and dry weight of roots, cell viability, <span class="gene">PAL</span> and <span class="trait">POD</span> activities decreased, while SOD activities increased after dopamine treatment. The content of lignin was not altered. The data demonstrate the susceptibility of soybean to dopamine and reinforce the role of this catecholamine as a strong allelochemical. The results also suggest that dopamine-induced inhibition in soybean roots is not related to the production of lignin, but may be related to damage caused by reactive oxygen species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:L-DOPA increases lignification associated with Glycine max root growth-inhibition.</h2>
			<h3>PMID:17195115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Soares AR(1), Ferrarese Mde L, Siqueira Rde C, Böhm FM, Ferrarese-Filho O.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						L-3,4-dihydroxyphenylalanine (L: -DOPA), an allelochemical exuded from the roots of velvet bean [Mucuna pruriens (L.) DC. var. utilis], presents a highly inhibitory action to plant growth. The effects of L-DOPA on phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) and peroxidase <span class="trait">(POD,</span> EC 1.11.1.7) activities, and phenolic compound and lignin content in soybean [Glycine max (L.) Merr.] roots were investigated to determine the possible phytotoxic mechanism. Three-day-old seedlings were cultivated in half-strength Hoagland nutrient solution (pH 6.0), without or with 0.1 to 1.0 mM L-DOPA in a growth chamber (25 degrees C, 12-hr light to 12-hr darkness photoperiod, irradiance of 280 micromol m-2 s-1) for 24 hr. In general, the length, fresh weight, and dry weight of the roots decreased, whereas <span class="gene">PAL</span> and <span class="trait">POD</span> activities and phenolic compound and lignin content increased after L-DOPA treatments. Results showed the susceptibility of soybean to L-DOPA and reinforce the role of this nonprotein amino acid as a strong allelochemical. The present findings also suggest that L-DOPA-induced inhibition in soybean roots may be because of a cell wall stiffening process related to the formation of cross-linking between cell wall polymers linked to lignin production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lignification and related enzymes in Glycine max root growth-inhibition by ferulic acid.</h2>
			<h3>PMID:15303323  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：dos Santos WD(1), Ferrarese Mde L, Finger A, Teixeira AC, Ferrarese-Filho O.
				<br/> 
				年份：2004 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Changes in soluble and cell wall bound peroxidase <span class="trait">(POD,</span> EC 1.11.1.7) activity, phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) activity, and lignin content in roots of ferulic acid-stressed soybean (Glycine max (L.) Merr.) seedlings and their relationships with root growth were investigated. Three-day-old soybean seedlings were cultivated in half-strength Hoagland nutrient solution containing 1.0 mM ferulic acid for 24-72 hr. Length, fresh weight, and dry weight of roots decreased, while soluble and cell wall bound <span class="trait">POD</span> activity, <span class="gene">PAL</span> activity, and lignin content increased after ferulic acid treatment. These enzymes probably participate in root growth reduction in association with cell wall stiffening related to the formation of cross-linking among cell wall polymers and lignin production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxidase and phenylalanine ammonia-lyase activities, phenolic acid contents, and allelochemicals-inhibited root growth of soybean.</h2>
			<h3>PMID:12125206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Herrig V(1), Ferrarese Mde L, Suzuki LS, Rodrigues JD, Ferrarese-Filho O.
				<br/> 
				年份：2002
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The influence of the allelochemicals ferulic (FA) and vanillic (VA) acids on peroxidase <span class="trait">(POD,</span> EC 1.11.1.7) and phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) activities and their relationships with phenolic acid (PhAs) contents and root growth of soybean (Glycine max (L.) Merr.) were examined. Three-day-old seedlings were cultivated in nutrient solution containing FA or VA (0.1 to 1 mM) for 48 h. Both compounds (at 0.5 and 1 mM) decreased root length (RL), fresh weight (FW) and dry weight (DW) and increased PhAs contents. At 0.5 and 1 mM, FA increased soluble <span class="trait">POD</span> activity (18% and 47%, respectively) and cell wall (CW)-bound <span class="trait">POD</span> activity (61% and 34%), while VA increased soluble <span class="trait">POD</span> activity (33% and 17%) but did not affect CW-bound <span class="trait">POD</span> activity. At 1 mM, FA increased (82%) while VA reduced (32%) <span class="gene">PAL</span> activities. The results are discussed on the basis of the role of these compounds on phenylpropanoid metabolism and root growth and suggest that the effects caused on <span class="trait">POD</span> and <span class="gene">PAL</span> activities are some of the many mechanisms by which allelochemicals influence plant growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Single feature polymorphisms (SFPs) for <span class="trait">drought</span> tolerance in pigeonpea ( <italic> Cajanus </italic> spp.)</h2>
			<h3>PMID:21547435</h3>
				
					<h3>PMCID:3220819</h3>
					<h4>期刊:Functional &#x00026; Integrative Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Saxena </surname>  <given-names> Rachit K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Xinping </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Thakur </surname>  <given-names> Vivek </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Walter </surname>  <given-names> Barbara </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Close </surname>  <given-names> Timothy J. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K. </given-names>  </name>  <address>  <fax> +91-40-30713074 </fax>  <email> r.k.varshney@cgiar.org </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru, 502324 India  </aff>  <aff id="Aff2">  <label> 2 </label> University of California, Riverside, CA 92521 USA  </aff>  <aff id="Aff3">  <label> 3 </label> CGIAR- Generation Challenge Programme (GCP), c/o CIMMYT, 06600 Mexico, DF Mexico  </aff>
				<br/> 
				年份：<day> 17 </day>  <month> 4 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						By using stringent criteria and RPP analysis, a total of 5,692 potential SFPs were discovered across the six genotypes (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). As an example, Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> shows detection of SFP between ICPL 8755 and ICPL 227 genotypes for the probe #5 of the SFP probe set <span class="gene">GmaAffx.5953.1.A1_at.</span> As the six genotypes analyzed in this study represents the parents of three mapping populations, an effort was made to identify the SFPs for different parental combinations. In this context, 850 SFPs were identified from 5,405 &#x0201c;present&#x0201d; probe sets in the ICP 28&#x02009;&#x000d7;&#x02009;ICPW 94 (cross 1), 854 SFPs out of 5,455 &#x0201c;present&#x0201d; probe sets in the ICPL 151&#x02009;&#x000d7;&#x02009;ICPL 87 (cross 2), and 780 SFPs out of 4,989 in the ICPL 8755&#x02009;&#x000d7;&#x02009;ICPL 227 (cross 3). However, many SFPs that were detected were unique to one parental combination. The <span class="trait">number</span> of SFPs in common between cross 1 and 2 was 17, between cross 1 and 3 it was 14, and between cross 2 and 3 a total of 19 SFPs were in common. A total of 10 SFPs were in common for all the three crosses used. These results reconfirm earlier observations on occurrence of low level of genetic diversity based on other marker systems such as amplified fragment length polymorphisms (Panguluri et al. <xref ref-type="bibr" rid="CR15"> 2006 </xref> ), Diversity Array Technology (Yang et al. <xref ref-type="bibr" rid="CR27"> 2011 </xref> ) and SSRs (Saxena et al. <xref ref-type="bibr" rid="CR19"> 2010a </xref> ). Nevertheless, the present study adds a set of about 1,000 novel markers (SFPs) for genetics and breeding analysis in pigeonpea. <fig id="Fig1"> <label> Fig.&#x000a0;1 </label> <caption> <p> A snapshot of SFP analysis in pigeonpea for the probe set <span class="gene">GmaAffx.5953.1.A1_at</span> of Affymetrix Soybean Genome Array. Two replicates of genotype ICPL 8755 and genotype ICPL 227 has been shown in <italic> dark </italic> and <italic> dotted lines </italic> , respectively. The <italic> top panel </italic> shows log intensities ( <italic> left side </italic> ) and hybridization affinities ( <italic> right side </italic> ) for all probes of the probe set for two genotypes. The <italic> bottom panel </italic> shows the difference in hybridization affinities ( <italic> left side </italic> ) and the individual outlying scores ( <italic> right side </italic> ) for each probe between two genotypes
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTLomics in Soybean: A Way Forward for Translational Genomics and Breeding</h2>
			<h3>PMID:28066449</h3>
				
					<h3>PMCID:5174554</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kumawat </surname>  <given-names> Giriraj </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/311123/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gupta </surname>  <given-names> Sanjay </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374813/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ratnaparkhe </surname>  <given-names> Milind B. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/55589/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maranna </surname>  <given-names> Shivakumar </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374396/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Satpute </surname>  <given-names> Gyanesh K. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374178/overview"/>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Like phosphorus, soils are abundant in iron but most of it is present in the form of Fe <sup> 3+ </sup> which is not accessible to plants. The Fe <sup> 2+ </sup> form is physiologically more significant for plants but it readily oxidizes to sparingly accessible form (Fe <sup> 3+ </sup> ). Fe <sup> 3+ </sup> is insoluble in neutral and high pH, making iron unavailable to plants in alkaline and in calcareous soils. Furthermore, in these types of soil, iron readily combines with phosphates, carbonates, calcium, magnesium, and hydroxide ions and results in poor iron availability to plants. These conditions cause iron deficiency <span class="trait">chlorosis</span> (IDC) in soybean, in which photosynthetic pigments, electron transport potential, and thylakoid system are reduced as plants fail to utilize iron from the soil (Spiller and Terry, <xref rid="B134" ref-type="bibr"> 1980 </xref> ; Taylor et al., <xref rid="B143" ref-type="bibr"> 1982 </xref> ). The best way to manage IDC is the use of iron-efficient varieties in commercial crops (Peiffer et al., <xref rid="B107" ref-type="bibr"> 2012 </xref> ). Lin et al. ( <xref rid="B91" ref-type="bibr"> 1997 </xref> ) identified a major QTL for iron efficiency on <span class="gene">Gm03,</span> explaining more than 70% of the phenotypic variation in the Anoka &#x000d7; A7 mapping population. Using introgression mapping, transcriptome sequencing and gene expression analysis, Peiffer et al. ( <xref rid="B107" ref-type="bibr"> 2012 </xref> ) characterized this QTL and identified two candidate genes <italic> <span class="gene">Glyma03g28610</span> </italic> and <italic> <span class="gene">Glyma03g28630</span> </italic> , encoding transcription factors. These two transcription factor encoding genes, were identified as homologs of the subgroup <italic> Ib basic helix-loop-helix (bHLH) </italic> genes that are known to regulate the strategy I response adaptation in Arabidopsis (Wang et al., <xref rid="B164" ref-type="bibr"> 2007 </xref> ; Peiffer et al., <xref rid="B107" ref-type="bibr"> 2012 </xref> ). These two genes were significantly induced in soybean roots under iron limiting conditions. Re-sequencing of these two induced genes identified a significant deletion of 12-bp within a predicted dimerization domain of <italic> <span class="gene">Glyma03g28610</span> </italic> in iron-inefficient lines (Peiffer et al., <xref rid="B107" ref-type="bibr"> 2012 </xref> ). Further experiments for functional validation and identifying the target genes for DNA binding of the two <italic> basic helix-loop-helix (bHLH) </italic> genes may help in identification of iron efficiency molecular network in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Pod</span> dehiscence is a critical step in the seed dispersal of legumes and especially in wild species of crop plants. However, it can cause significant <span class="trait">yield</span> losses in soybean and shattering resistant genotype is desirable. The cultivated soybean has been domesticated from its annual wild relative, <italic> Glycine soja </italic> (Sieb et Zucc), in China ~5000 years ago (Hymowitz and Singh, <xref rid="B56" ref-type="bibr"> 1987 </xref> ) resulting in a multitude of soybean landraces that have adapted to various climate environments. Although, the cultivated soybean is more shattering resistant than the wild soybean, genetic variation in the degree of <span class="trait">pod</span> dehiscence is also present among cultivated soybean (Tsuchiya, <xref rid="B149" ref-type="bibr"> 1987 </xref> ). Bailey et al. ( <xref rid="B6" ref-type="bibr"> 1997 </xref> ) identified a major QTL <italic> qPdh1 </italic> on linkage group J, with a phenotypic variance contribution of 50% for <span class="trait">pod</span> dehiscence in cultivated soybean. QTL <italic> qPdh1 </italic> was later confirmed in other studies and fine-mapped to a 47-kb genomic region (Funatsuki et al., <xref rid="B40" ref-type="bibr"> 2006 </xref> , <xref rid="B39" ref-type="bibr"> 2008 </xref> ; Suzuki et al., <xref rid="B141" ref-type="bibr"> 2010 </xref> ; Gao and Zhu, <xref rid="B43" ref-type="bibr"> 2013 </xref> ). Using fine mapping and complementation test, Funatsuki et al. ( <xref rid="B41" ref-type="bibr"> 2014 </xref> ) identified an underlying gene <italic> <span class="gene">Glyma16g25580.1</span> </italic> ( <italic> <span class="gene">Pdh1</span> </italic> ), which encodes a dirigent-like protein known to be involved in the lignifications which increases dehiscing forces by increasing torsion of dried <span class="trait">pod</span> walls. In the shattering-resistant genotype Hayahikari, the gene <italic> pdh1 </italic> was defective, having a premature stop codon due to the presence of a non-sense SNP in the coding sequence (Funatsuki et al., <xref rid="B41" ref-type="bibr"> 2014 </xref> ). <italic> <span class="gene">Pdh1</span> </italic> is a novel gene identified for <span class="trait">pod</span> dehiscence and selection of nonsense mutation in this gene has played an important role in soybean domestication. Functional markers developed based on this nonsense SNP could be utilized for development of shattering resistant soybean genotypes through marker-assisted selection (MAS).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is a rich source of protein therefore it is imperative to investigate genes responsible for seed protein accumulation. &#x003b2;-conglycinin and glycinin are the most prominent seed proteins in soybean (Roberts and Briggs, <xref rid="B119" ref-type="bibr"> 1965 </xref> ; Thanh and Shibasaki, <xref rid="B145" ref-type="bibr"> 1976 </xref> ). Several seed protein QTLs have been identified in soybean and among them QTLs of LG I <span class="gene">(Gm20)</span> are important as they have been reported in many of the mapping populations (Diers et al., <xref rid="B32" ref-type="bibr"> 1992 </xref> ; Seboldt et al., <xref rid="B125" ref-type="bibr"> 2000 </xref> ; Csan&#x000e1;di et al., <xref rid="B28" ref-type="bibr"> 2001 </xref> ). Bolon et al. ( <xref rid="B12" ref-type="bibr"> 2010 </xref> ) could also identify QTL for seed protein at this linkage group and characterized the underlying genes by using a near-isogenic line (NIL) pair that contrasted in seed protein content. This NIL pair differed in protein content because of introgression of a genomic segment containing the high protein QTL of <italic> G. soja </italic> (PI468916) into <italic> G. max </italic> A81-356022. Using microarray and high-throughput whole transcriptome sequencing approaches, 13 genes displaying significant seed transcript expression between NILs were identified which included a potential regulatory protein of the <span class="gene">Mov34-1</span> family, a heat shock protein Hsp22.5, and an ATP synthase (Bolon et al., <xref rid="B12" ref-type="bibr"> 2010 </xref> ). In a regional association study of genomic block on <span class="gene">Gm20,</span> Valliyodan et al. ( <xref rid="B152" ref-type="bibr"> 2016 </xref> ) identified that the copy <span class="trait">number</span> variations in candidate genes (HSP, <italic> <span class="gene">Glyma20g19680</span> </italic> ; ammonium transporter, <italic> <span class="gene">Glyma20g21030</span> </italic> ; ethylene receptor, <italic> <span class="gene">Glyma20g21780</span> </italic> ) were associated with protein content. Lestari et al. ( <xref rid="B82" ref-type="bibr"> 2013 </xref> ) reported that parts of <span class="gene">Gm20</span> and <span class="gene">Gm10</span> showed synteny, however QTLs for seed protein content were detected only on <span class="gene">Gm20.</span> Comparative sequence analysis identified that four tandem duplicates of the putative homeobox protein 22 (HB22) were present specifically on <span class="gene">Gm20</span> (Lestari et al., <xref rid="B82" ref-type="bibr"> 2013 </xref> ). This <italic> Medicago truncatula </italic> homolog has been reported to be expressed in endosperm and hypothesized to regulate seed filling stage (Verdier et al., <xref rid="B158" ref-type="bibr"> 2008 </xref> ). Like <italic> Medicago truncatula </italic> , these tandem duplicates could be regulating seed filling stage in soybean as well and therefore contribute to the seed protein QTL of <span class="gene">Gm20.</span> Further, functional studies are required to identify and validate the role of these candidate genes in seed protein accumulation.
					</p>
				
					<p>&nbsp;&nbsp;
						Isoflavones are health beneficial molecules in human nutrition (Tsukamoto et al., <xref rid="B150" ref-type="bibr"> 1995 </xref> ; Rochfort and Panozzo, <xref rid="B120" ref-type="bibr"> 2007 </xref> ). In soybean, isoflavones are critical factor in defense against pathogens (Benhamou and Nicole, <xref rid="B9" ref-type="bibr"> 1999 </xref> ; Subramanian et al., <xref rid="B137" ref-type="bibr"> 2004 </xref> ), in promoting nodulation by rhizobia (Subramanian et al., <xref rid="B138" ref-type="bibr"> 2006 </xref> , <xref rid="B139" ref-type="bibr"> 2007 </xref> ) and in changing or adjusting the microbial population around plant roots (Lozovaya et al., <xref rid="B95" ref-type="bibr"> 2004 </xref> ). Several QTLs have been mapped for three species of isoflavones i.e., genistein ( <italic> GEN </italic> ), daidzein ( <italic> DAI </italic> ), glycitein ( <italic> GLY </italic> ), and total of isoflavones ( <italic> TOT </italic> ) (Primomo et al., <xref rid="B112" ref-type="bibr"> 2005 </xref> ; Zeng et al., <xref rid="B178" ref-type="bibr"> 2009 </xref> ; Gutierrez-Gonzalez et al., <xref rid="B50" ref-type="bibr"> 2010 </xref> , <xref rid="B49" ref-type="bibr"> 2011 </xref> ; Wang et al., <xref rid="B168" ref-type="bibr"> 2014 </xref> ). Gutierrez-Gonzalez et al. ( <xref rid="B50" ref-type="bibr"> 2010 </xref> , <xref rid="B49" ref-type="bibr"> 2011 </xref> ) identified and validated a major QTL in two RILs derived from low isoflavone lines Magellan and Essex, and high isoflavone line PI 437654. This QTL was detected on <span class="gene">Gm05</span> in both of the studies and had significant effect on individual isoflavone and total seed isoflavone accumulation (named <italic> qGEN5, qDAI5, qGLY5 </italic> , and <italic> qTOT5 </italic> , respectively). This locus accounted for about 36, 35, 12, and 37% of the variance for genistein, daidzein, glycitein, and total isoflavone concentration, respectively. Another major QTL has been identified on <span class="gene">Gm08</span> by Yang et al. ( <xref rid="B177" ref-type="bibr"> 2011 </xref> ). Gutierrez-Gonzalez et al. ( <xref rid="B50" ref-type="bibr"> 2010 </xref> ) used candidate gene approach to identify homologous sequences in the soybean genome with assigned or putative functions in tocopherol biosynthesis pathway, and placed them on the map at their approximate positions, along with all reported QTLs using their interval of confidence. The study identified <span class="trait">number</span> of putative candidate genes underlying several of the main-effect and epistatic QTLs. Since there is possibility of trans-acting regulatory elements responsible for the phenotype, Wang et al. ( <xref rid="B168" ref-type="bibr"> 2014 </xref> ) used expression QTL (eQTL) analysis combined with phenotypic QTL analyses identifying a total of 33 eQTLs, of them five eQTL intervals were overlapped with phenotypic QTLs (pQTLs). A total of 11 candidate genes were identified within the overlapped eQTL and pQTL genomic regions (Table <xref ref-type="table" rid="T2"> 2 </xref> ; Wang et al., <xref rid="B168" ref-type="bibr"> 2014 </xref> ). Identification of causative genetic variants within these genes and structural variations in flanking regions is required for development of haplotype based molecular markers facilitating manipulation of isoflavone content in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Oil content of soybean is a valuable quantitative trait controlled by multiple genes. At least 110 QTLs for soybean oil content have been mapped but the utilization of these QTLs for soybean breeding has some limitations. Recently, Valliyodan et al. ( <xref rid="B152" ref-type="bibr"> 2016 </xref> ) conducted comparative sequence analysis between high and low oil lines and showed differential variation in SNP and copy <span class="trait">number</span> for lipid biosynthesis and lipid degradation genes between high oil and low oil lines. The comparative analysis shows increased copy <span class="trait">number</span> of lipid transfer protein (LPT; <italic> <span class="gene">Glyma16g31780,</span> <span class="gene">Glyma16g31840</span> </italic> , and <italic> <span class="gene">Glyma16g31540</span> </italic> ) in high oil lines, whereas the negative regulators of lipid biosynthesis genes (ABC transporter, <italic> <span class="gene">Glyma03g36310</span> </italic> , Lipase 3, and <italic> <span class="gene">Glyma13g04561</span> </italic> ) showed more copies in low oil lines. The differential variation for lipid biosynthesis and lipid degradation genes between high oil and low oil lines might be associated with the differences in oil content.
					</p>
				
					<p>&nbsp;&nbsp;
						Nematode causes severe <span class="trait">yield</span> losses in soybean in the regions of warm climate and sandy soils (Walters and Baker, <xref rid="B161" ref-type="bibr"> 1994 </xref> ). The root not nematode and soybean cyst nematode are two major pests of soybean in United States. Using whole genome re-sequencing of recombinant inbred lines (RIL) derived from a cross between Magellan (susceptible) and PI 438489B (resistant), Xu et al. ( <xref rid="B175" ref-type="bibr"> 2013 </xref> ) identified two candidate genes underlying a major QTL for root not nematode <span class="trait">resistance</span> mapped to bin 10 of <span class="gene">Gm10.</span> The two candidate genes <italic> <span class="gene">Glyma10g02150</span> </italic> and <italic> <span class="gene">Glyma10g02160</span> </italic> , encodes a pectin methylesterase inhibitor and a pectin methylesterase inhibitor-pectin methylesterase, respectively (Xu et al., <xref rid="B175" ref-type="bibr"> 2013 </xref> ). The protein encoded by <italic> <span class="gene">Glyma10g02160</span> </italic> was found similar to the <italic> Arabidopsis </italic> protein pectin methylesterase, PME3, which was shown to be important in nematode parasitism (Hewezi et al., <xref rid="B55" ref-type="bibr"> 2008 </xref> ). Although, SNPs and InDels were reported between parental lines for both genes but functional characterization and haplotype analysis is required for marker development and utilization.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN) ( <italic> Heterodera glycines </italic> Ichinohe) feeds on the roots of soybean and is a major constraint to soybean production in United States. In resistant genotype Peking, a dominant <span class="trait">resistance</span> gene ( <italic> <span class="gene">Rhg4</span> </italic> ) was identified that was linked to <italic> I </italic> locus underlying seed coat color (Matson and Williams, <xref rid="B98" ref-type="bibr"> 1965 </xref> ). Through positional cloning from soybean cultivar Forrest, Liu et al. ( <xref rid="B94" ref-type="bibr"> 2012 </xref> ) identified a gene underlying <italic> <span class="gene">Rhg4</span> </italic> locus on <span class="gene">Gm08,</span> a major QTL contributing to <span class="trait">resistance</span> to SCN. The gene underlying <italic> <span class="gene">Rhg4</span> </italic> encodes a serine hydroxymethyltransferase (SHMT) enzyme, a ubiquitous enzyme in nature with a key role in one-carbon folate metabolism. Alternative alleles of <italic> <span class="gene">Rhg4</span> </italic> differ by two genetic polymorphisms that alter a key regulatory property of the enzyme (Liu et al., <xref rid="B94" ref-type="bibr"> 2012 </xref> ). Haplotype survey of <italic> <span class="gene">SHMT</span> </italic> sequence identified eight haplotypes, where soybean lines with haplotypes H1&#x02013;H3 carry resistant alleles at <italic> <span class="gene">SHMT</span> </italic> (Liu et al., <xref rid="B94" ref-type="bibr"> 2012 </xref> ). The identified <span class="trait">resistance</span> mechanism can be readily exploited to improve <span class="trait">resistance</span> against SCN in susceptible cultivars. A recessive and codominant locus <italic> rhg1 </italic> which is necessary, but not sufficient, for <span class="trait">resistance</span> to all known Hg-biotypes of SCN, was also mapped (Concibido et al., <xref rid="B24" ref-type="bibr"> 1997 </xref> ; Ruben et al., <xref rid="B123" ref-type="bibr"> 2006 </xref> ). Ruben et al. ( <xref rid="B123" ref-type="bibr"> 2006 </xref> ) fine mapped the <italic> rhg1 </italic> locus and identified a single <italic> receptor like kinase </italic> ( <italic> RLK; leucine rich repeat transmembrane-protein kinase </italic> ) as candidate <span class="trait">resistance</span> gene. Srour et al. ( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized the <italic> <span class="gene">GmRLK18-1</span> </italic> , the gene underlying <italic> <span class="gene">Rhg1</span> </italic> locus, in transgenic and heterozygous plants and demonstrated that the dominant allele confers reported pleiotropic <span class="trait">resistance</span> to SCN and sudden death syndrome (SDS) caused by <italic> Fusarium virguliforme </italic> . In the absence of <italic> <span class="gene">Rhg4</span> </italic> , the <italic> <span class="gene">GmRLK18-1</span> </italic> had provided partial <span class="trait">resistance</span> to SCN and conferred nearly complete <span class="trait">resistance</span> to SDS, whereas, in the presence of <italic> <span class="gene">Rhg4</span> </italic> , the plants with the <italic> <span class="gene">GmRLK18-1</span> </italic> transgene were fully resistant to SCN as well as SDS (Srour et al., <xref rid="B135" ref-type="bibr"> 2012 </xref> ). <italic> In vitro </italic> , the purified LRR domain of <italic> <span class="gene">GmRLK18-1</span> </italic> has been shown to bind with the CLE peptides of plant origin known to involve in tracheary element inhibition (Afzal et al., <xref rid="B2" ref-type="bibr"> 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						About 90% of the commercially cultivated SCN-resistant soybean varieties in the central United States use the <italic> rhg1-b </italic> allele (haplotype) as the main SCN <span class="trait">resistance</span> locus derived from the genotype PI 88788 (Cook et al., <xref rid="B26" ref-type="bibr"> 2012 </xref> ). Fine mapping of <italic> rhg1-b </italic> haplotype in PI88788 has delineated <italic> rhg1-b </italic> in an interval that corresponds to a 67-kb segment carrying 11 predicted genes in the genome of Williams 82 (Kim M. et al., <xref rid="B72" ref-type="bibr"> 2010 </xref> ). By using gene silencing approach, Cook et al. ( <xref rid="B26" ref-type="bibr"> 2012 </xref> ) identified that SCN <span class="trait">resistance</span> mediated by the soybean QTL <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 is conferred by copy <span class="trait">number</span> variation of a 31-kilobase segment encoding an amino acid transporter ( <italic> <span class="gene">Glyma18g02580</span> </italic> ), an &#x003b1;-SNAP (&#x003b1;-soluble N-ethylmaleimide-sensitive factor attachment protein) protein ( <italic> <span class="gene">Glyma18g02590</span> </italic> ), and a WI12 (wound-inducible domain) protein ( <italic> <span class="gene">Glyma18g02610</span> </italic> ). Further, characterization of genetic variation at <italic> <span class="gene">Rhg1</span> </italic> across 41 diverse soybean accessions discovered that SCN <span class="trait">resistance</span> is associated with multicopy <italic> <span class="gene">Rhg1</span> </italic> haplotypes that form two distinct groups (Cook et al., <xref rid="B25" ref-type="bibr"> 2014 </xref> ). The high-copy-number <italic> <span class="gene">Rhg1</span> </italic> accessions reportedly contain a flexible <span class="trait">number</span> of copies (seven to 10) of the 31-kb <italic> <span class="gene">Rhg1</span> </italic> repeat whereas the low-copy-number <italic> <span class="gene">Rhg1</span> </italic> group contains three copies of the <italic> <span class="gene">Rhg1</span> </italic> repeat and a newly identified allele of &#x003b1;-SNAP (Cook et al., <xref rid="B25" ref-type="bibr"> 2014 </xref> ). Kadam et al. ( <xref rid="B66" ref-type="bibr"> 2016 </xref> ) selected SNPs from the <italic> <span class="gene">Rhg1</span> </italic> gene ( <italic> <span class="gene">Glyma18g02590</span> </italic> ) and <italic> <span class="gene">Rhg4</span> </italic> gene ( <italic> <span class="gene">Glyma08g11490</span> </italic> ) to develop KASPar assays. Two SNP markers specific to <italic> <span class="gene">Rhg1</span> </italic> locus and two SNP markers specific to <italic> <span class="gene">Rhg4</span> </italic> locus were validated in 95 germplasm lines and a subset of RILs. A Taqman assay was also developed from the conserved region of the <italic> <span class="gene">Rhg1</span> </italic> locus for detection of copy <span class="trait">number</span> variation at <italic> <span class="gene">Rhg1</span> </italic> locus (Kadam et al., <xref rid="B66" ref-type="bibr"> 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean bacterial leaf pustule (BLP) is caused by <italic> Xanthomonas axonopodis pv. glycines </italic> . BLP <span class="trait">resistance</span> was reported to be controlled by a single recessive gene ( <italic> rxp </italic> ) in BLP resistant source, CNS (PI 548445) (Hartwig and Lehman, <xref rid="B54" ref-type="bibr"> 1951 </xref> ). QTLs analysis for BLP <span class="trait">resistance</span> revealed that Satt372 and Satt486 on <span class="gene">Gm17</span> were strongly associated with BLP <span class="trait">resistance</span> (Narvel et al., <xref rid="B102" ref-type="bibr"> 2001 </xref> ; Van et al., <xref rid="B154" ref-type="bibr"> 2004 </xref> ). Fine mapping in RILs derived from a cross between &#x0201c;Taekwangkong&#x0201d; (BLP susceptible) and &#x0201c;Danbaekkong&#x0201d; (BLP resistant) and two pair of NILs, Kim D. et al. ( <xref rid="B71" ref-type="bibr"> 2010 </xref> ) narrowed down the BLP <span class="trait">resistance</span> locus to 33 Kb. Annotation of this 33 Kb sequence identified two putative candidate genes, a membrane protein gene ( <italic> <span class="gene">Glyma17g09780</span> </italic> ) and a Zinc finger family protein gene ( <italic> <span class="gene">Glyma17g09790</span> </italic> ). The candidate genes showed high similarity with their paralogous genes, which were located on the duplicated regions reported for bacterial leaf pustule <span class="trait">resistance</span> QTLs (Kim D. et al., <xref rid="B71" ref-type="bibr"> 2010 </xref> ). Functional role to these genes has not been assigned.
					</p>
				
					<p>&nbsp;&nbsp;
						White mold, caused by <italic> Sclerotinia sclerotiorum </italic> [(Lib.) W. Phillips.] de Bary, is reported as a devastating disease of soybean and many other crops (Boland and Hall, <xref rid="B11" ref-type="bibr"> 1994 </xref> ). Currently, due to the lack of immune type <span class="trait">resistance</span> genotypes, only partially resistant cultivars have been used for genetic mapping. Recently, Zhao et al. ( <xref rid="B187" ref-type="bibr"> 2015 </xref> ) identified a major QTL on <span class="gene">Gm13</span> by linkage and association mapping. Four potential candidate genes known to be involved in disease response and anthocyanin biosynthesis were identified at the locus near the peak SNPs. The candidate genes identified were a <italic> PR5-like receptor kinase, RINT-1/TIP-1 family, Myb domain protein 33 </italic> , and a <italic> C2H2-type zinc finger family protein </italic> gene (Zhao et al., <xref rid="B187" ref-type="bibr"> 2015 </xref> ). These candidate genes will be useful for functional validation and soybean breeding for improving <span class="trait">resistance</span> to white mold.
					</p>
				
					<p>&nbsp;&nbsp;
						Sudden death syndrome of soybean caused by <italic> Fusarium virguliforme </italic> (Roy et al., <xref rid="B122" ref-type="bibr"> 1997 </xref> ), is a serious threat to soybean production (Wrather et al., <xref rid="B173" ref-type="bibr"> 2010 </xref> ) which can be managed with host plant <span class="trait">resistance</span> (Wen et al., <xref rid="B172" ref-type="bibr"> 2014 </xref> ). <span class="trait">Resistance</span> to SDS in soybean is multi-geneic and reportedly had two components; a partial <span class="trait">resistance</span> to leaf scorch caused indirectly by translocated fungal toxins; and a partial <span class="trait">resistance</span> to root infection and rot caused directly at the site of infection by the fungus (Njiti et al., <xref rid="B104" ref-type="bibr"> 1998 </xref> ; Kazi et al., <xref rid="B68" ref-type="bibr"> 2008 </xref> ). Linkage mapping identified several QTLs associated with SDS <span class="trait">resistance</span> across 12 of the 20 soybean chromosomes ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org/"> http://www.soybase.org/ </ext-link> ). A locus <italic> Rfs2/Rhg1 </italic> on <span class="gene">Gm18</span> underlay partial <span class="trait">resistance</span> to the spread of root infections by <italic> F. virguliforme </italic> and SCN (Njiti et al., <xref rid="B104" ref-type="bibr"> 1998 </xref> ; Triwitayakorn et al., <xref rid="B146" ref-type="bibr"> 2005 </xref> ). Using NILs, Srour et al. ( <xref rid="B135" ref-type="bibr"> 2012 </xref> ) characterized <italic> Rfs2 </italic> / <italic> <span class="gene">Rhg1</span> </italic> QTL on <span class="gene">Gm18</span> identifying a receptor like kinase ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) responsible for <span class="trait">resistance</span> to SDS and SCN. In a GWAS study using two sets of 392 and 300 unique soybean genotypes as independent association panels, Wen et al. ( <xref rid="B172" ref-type="bibr"> 2014 </xref> ) identified a total of 20 loci underlying SDS <span class="trait">resistance.</span> Several of SNPs significantly associated with SDS <span class="trait">resistance</span> were within or adjacent to sequences annotated as plant disease <span class="trait">resistance</span> genes including the previously identified SDS <span class="trait">resistance</span> gene <italic> <span class="gene">GmRLK18-1</span> </italic> . In a GWAS study using 214 germplasm accessions Zhang et al. ( <xref rid="B182" ref-type="bibr"> 2015b </xref> ) identified a potential candidate gene on <span class="gene">Gm02</span> encoding <italic> stress induced receptor-like kinase gene 1 </italic> ( <italic> <span class="gene">SIK1</span> </italic> ). The peak SNP locus associated with SDS <span class="trait">resistance</span> was present in the coding region of the <italic> <span class="gene">SIK1</span> </italic> resulting in a non-synonymous mutation.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Phytophthora </italic> root rot is one of the important diseases of soybean and is caused by <italic> Phytophthora sojae </italic> . Tolerance to <italic> Phytophthora </italic> root rot is multi-genic due to the presence of several races of pathogen and so far 20 <italic> Rps </italic> <span class="trait">(resistance</span> to <italic> P. sojae </italic> ) loci including 26 alleles have been detected on four different chromosomes i.e., <span class="gene">Gm03,</span> <span class="gene">Gm13,</span> <span class="gene">Gm16,</span> and <span class="gene">Gm18</span> (Li et al., <xref rid="B85" ref-type="bibr"> 2016a </xref> ). Lin et al. ( <xref rid="B90" ref-type="bibr"> 2013 </xref> ) have identified two <italic> Rps </italic> genes, <italic> RpsUN1 </italic> on chromosome 3 and <italic> RpsUN2 </italic> on chromosome 16 from a soybean landrace PI 567139B, which together confer complete <span class="trait">resistance</span> to 16 <italic> P. sojae </italic> races/isolates. Li et al. ( <xref rid="B85" ref-type="bibr"> 2016a </xref> ), further narrowed <italic> RpsUN1 </italic> and <italic> RpsUN2 </italic> , to a 151 kb region that harbors five genes including three disease <span class="trait">resistance</span> (R)-like genes, and a 36 kb region that contains four genes, all of which are of the typical R-like genes, respectively. RNA-seq analysis of these nine genes before and after inoculation with the pathogen suggest that <italic> <span class="gene">Glyma.03g034600</span> </italic> in the <italic> RpsUN1 </italic> region, and <italic> <span class="gene">Glyma.16g215200</span> </italic> and <italic> <span class="gene">Glyma.16g214900</span> </italic> in the <italic> RpsUN2 </italic> region of PI 567139B may be associated with the <span class="trait">resistance</span> to <italic> P. sojae </italic> . Li et al. ( <xref rid="B86" ref-type="bibr"> 2016b </xref> ) conducted GWAS in 279 soybean accessions and identified seven candidate genes within 500-kb regions on <span class="gene">Gm13</span> that are probably involved in natural variations in partial <span class="trait">resistance</span> to <italic> P. sojae </italic> . These genes encode a COPI ( <italic> <span class="gene">Glyma13g32980</span> </italic> ), a 2OG-Fe(II) protein ( <italic> <span class="gene">Glyma13g33900</span> </italic> ), a PPR protein ( <italic> <span class="gene">Glyma13g33512</span> </italic> ), LRR domain proteins ( <italic> <span class="gene">Glyma13g33536,</span> <span class="gene">Glyma13g33740</span> </italic> ), a Gpi16 subunit ( <italic> <span class="gene">Glyma13g33243</span> </italic> ), and a Zn-finger protein ( <italic> <span class="gene">Glyma13g33260</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Identification of genes underlying root system architecture and canopy characteristics is critical to develop soybean that is suited to water-limited environments. Prince et al. ( <xref rid="B113" ref-type="bibr"> 2015 </xref> ) identified four significant QTLs associated with different root architectural traits on <span class="gene">Gm06</span> and <span class="gene">Gm07</span> in an interspecific RILs population of <italic> G. max </italic> (V71-370) &#x000d7; <italic> G. soja </italic> (PI 407162). A <span class="trait">number</span> of candidate genes were identified for root system architecture traits. Notable among them are one encoding triglyceride lipase mapped to QTLs for root distribution traits, two genes encoding an apoptosis inhibitory 5 family protein and an oxidoreductase/transition metal ion binding protein mapped to the taproot length QTLs, a cell cycle associated <italic> D6 type cyclin gene </italic> and a key hormone auxin-associated gene <italic> auxin efflux carrier protein gene </italic> (Prince et al., <xref rid="B113" ref-type="bibr"> 2015 </xref> ). In an another study, Manavalan et al. ( <xref rid="B97" ref-type="bibr"> 2015 </xref> ) identified a major QTL on <span class="gene">Gm08</span> that governed root traits (tap root length and lateral root <span class="trait">number)</span> and shoot length, and identified six transcription factors (MYB HD, TPR, C2H2 Zn, bZIP, GRAS, and Ring finger) and two key cell wall expansion-related genes which encode xyloglucan endo-transglycosylases as candidate genes in the confidence interval of the QTL. These are key candidate genes for validation and to develop a better root ideotype in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Traditionally molecular breeding is performed using linked molecular markers which suffered from limitation of marker polymorphism and genetic recombination, so even if parental genotypes possesses beneficial alleles the selection efficiency may still be &#x0003c;100%. The first example of MAS in soybean was of soybean cyst nematode <span class="trait">resistance</span> QTL conditioned by <italic> rhg1 </italic> locus (Cregan et al., <xref rid="B27" ref-type="bibr"> 1999 </xref> ). The two SSR markers linked with SCN <span class="trait">resistance</span> <italic> rhg1 </italic> locus were further validated by Silva et al. ( <xref rid="B130" ref-type="bibr"> 2007 </xref> ) and found that only a combination of two markers provide selection efficiency of 100%. Shivakumar et al. ( <xref rid="B129" ref-type="bibr"> 2016 </xref> ) employed MABC for conversion of two high Kunitz trypsin inhibitor (KTI) content soybean varieties (DS 9712 and DS 9814) into low <span class="gene">KTI</span> soybean genotypes using three SSR markers. Rosso et al. ( <xref rid="B121" ref-type="bibr"> 2011 </xref> ) has developed breeder friendly markers for <italic> D-myo-inositol 3-phosphate synthase 1 </italic> gene ( <italic> <span class="gene">MIPS1</span> </italic> ) which causes modifications to seed phosphorus and carbohydrate content in soybean. In this study, they clearly demonstrated that an allele specific SNP marker developed from causal mutation of <italic> <span class="gene">MIPS1</span> </italic> was able to select 100% mutant individuals as compared to four linked polymorphic SSR markers which showed selection efficiency of 42&#x02013;93%. Therefore, it is necessary to develop functional markers in soybean for selection of QTLs with 100% efficiency. In soybean, functional markers have been developed for several QTLs and a summary of these functional markers is presented in Table <xref ref-type="table" rid="T4"> 4 </xref> , which will be very useful for soybean molecular breeders for carrying out translational breeding in soybean. Pham et al. ( <xref rid="B108" ref-type="bibr"> 2010 </xref> , <xref rid="B109" ref-type="bibr"> 2011 </xref> ) used allele specific functional markers for <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes to combine mutant alleles in a single background. They identified soybean lines carrying both homozygous mutant <italic> <span class="gene">FAD2-1A</span> </italic> alleles and mutant <italic> <span class="gene">FAD2-1B</span> </italic> alleles producing &#x0003e;80% oleic acid content, compared to 20% in conventional soybean. Kumar et al. ( <xref rid="B76" ref-type="bibr"> 2013 </xref> ) successfully used gene specific functional marker for elimination of kunitz trypsin inhibitor from Indian soybean cultivar JS 97-52 through MABC. These are some examples which demonstrate the utility of functional markers developed by characterization of underlying genes of specific traits, to harness the true benefits of translational genomics and breeding in soybean. Although, the functional markers used in the above cited examples were not developed by QTL mapping and characterization, however several functional markers have been recently developed based on characterization of genes underlying QTLs and currently has been utilized in ongoing soybean breeding programs (Tsubokura et al., <xref rid="B147" ref-type="bibr"> 2013 </xref> ; Xu et al., <xref rid="B175" ref-type="bibr"> 2013 </xref> ; Tardivel et al., <xref rid="B142" ref-type="bibr"> 2014 </xref> ; Kadam et al., <xref rid="B66" ref-type="bibr"> 2016 </xref> ; Patil et al., <xref rid="B106" ref-type="bibr"> 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Several factors influence QTL detection in a mapping population i.e., the QTL allele effect, the existence of other linked QTL, the mapping population size, extent of genetic recombination, phenotypic assessment accuracy, genotyping errors, missing data, and environmental effects (Collard et al., <xref rid="B23" ref-type="bibr"> 2005 </xref> ; Francia et al., <xref rid="B37" ref-type="bibr"> 2005 </xref> ). Further, independent populations chosen from different gene pools may express different phenotypic values in different environments. Another difficulty for translational genomics is un-expected expression of introgressed QTLs in a new genetic background. The genomic region associated with a QTL may hold not just one but several genes, and recombination between those genes would then modify the effect of the introgressed segments. Since, biochemical pathways and enzyme protein sequences are known for some of seed composition traits, candidate gene identification is much easier for these traits resulting in the development of several functional markers in soybean i.e., oleic acid content, isoflavone content, and raffinose content etc. (Dierking and Bilyeu, <xref rid="B31" ref-type="bibr"> 2010 </xref> ; Pham et al., <xref rid="B108" ref-type="bibr"> 2010 </xref> , <xref rid="B109" ref-type="bibr"> 2011 </xref> ; Wang et al., <xref rid="B168" ref-type="bibr"> 2014 </xref> ). However, for agronomic and <span class="trait">yield</span> related traits, biochemical pathways are not well-characterized and therefore QTL mapping is the major strategy for identification of underlying genes. Although several genes are involved in the biological network of a quantitative trait, we are only able to detect the variation present in the parental genotypes and gene pool utilized for QTL mapping. This is further complicated by epistatic gene interactions and the target phenotypic environment. So, the major challenge in soybean QTLomics is to capture the genetic variation present among whole soybean gene pool for specific quantitative trait in target environment. The recent advances in NGS technologies and high-throughput marker genotyping enabled genotyping of whole germplasm accessions allowing GWAS for mapping of QTLs in whole gene pool. The large data-sets generated from NGS and high density genotyping requires sound computational algorithms for detection of minor QTLs as well as rare alleles with major effect phenotype. The biggest challenge for QTLomics is to integrate data from genome sequencing, transcriptomics, the epigenome and phenotyping, to make sense of it all. To overcome genetic background effect functional markers used for MAS can be combined with <span class="gene">GS</span> models (Deshmukh et al., <xref rid="B30" ref-type="bibr"> 2014 </xref> ). The major challenges for translational genomics in legumes includes: (1) limited availability of predictive and functional markers for target traits, (2) limited access to high-throughput genotyping technologies, (3) limited availability of high-throughput, cost-effective and precise phenotyping platforms, (4) availability of automated data management, analyses, and interpretation systems (Varshney et al., <xref rid="B157" ref-type="bibr"> 2012 </xref> , <xref rid="B156" ref-type="bibr"> 2015 </xref> ). Genotyping errors, false discovery rate, over-estimation of QTL effect and non-detection of superior but rare alleles are potential errors which QTLomics and translational genomics frequently encounter.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Neodiversification of homeologous <italic> CLAVATA1 </italic> -like receptor kinase genes in soybean leads to distinct developmental outcomes</h2>
			<h3>PMID:28827708</h3>
				
					<h3>PMCID:5566472</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mirzaei </surname>  <given-names> Saeid </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Batley </surname>  <given-names> Jacqueline </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> El-Mellouki </surname>  <given-names> Tarik </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferguson </surname>  <given-names> Brett J. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Gresshoff </surname>  <given-names> Peter M. </given-names>  </name>  <address>  <email> p.gresshoff@uq.edu.au </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9320 7537 </institution-id>  <institution-id institution-id-type="GRID"> grid.1003.2 </institution-id>  <institution/>  <institution> Centre for Integrative Legume Research, School of Agriculture and Food Sciences, The University of Queensland,  </institution>  </institution-wrap> St Lucia, Brisbane, QLD 4072 Australia  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 7910 </institution-id>  <institution-id institution-id-type="GRID"> grid.1012.2 </institution-id>  <institution/>  <institution> School of Biological Sciences, University of Western Australia,  </institution>  </institution-wrap> Crawley, WA 6009 Australia  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 1090 2313 </institution-id>  <institution-id institution-id-type="GRID"> grid.411026.0 </institution-id>  <institution/>  <institution> Department of Plant, Soil and Agricultural Systems, Southern Illinois University,  </institution>  </institution-wrap> Carbondale, IL 62901 USA  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.448905.4 </institution-id>  <institution/>  <institution> Department of Biotechnology, Institute of Science, High Technology and Environmental Sciences, Graduate University of Advanced Technology,  </institution>  </institution-wrap> Kerman, Iran  </aff>
				<br/> 
				年份：<day> 29 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Structure and genomic environments of <italic> CLAVATA1 </italic> and AON-related genes. ( <bold> A) </bold> Intron and exon positions and sizes of <italic> AtCLV1 </italic> , <italic> <span class="gene">GmCLV1A,</span> <span class="gene">GmNARK,</span> MtSUNN, LjHAR1, PsSYM29 </italic> and <italic> PvNARK </italic> . ( <bold> B </bold> ) TILLed regions of <italic> <span class="gene">GmNARK</span> </italic> and <italic> <span class="gene">GmCLV1A</span> </italic> . ( <bold> C </bold> ) Genomic environment of <italic> AtCLV1A </italic> , <italic> <span class="gene">GmCLV1A</span> </italic> , <italic> <span class="gene">GmNARK</span> </italic> , <italic> LiHAR1 </italic> , <italic> MtSUNN </italic> and <italic> PvNARK </italic> ; approximately 100&#x02009;kb is shown. The same <span class="trait">number</span> and colour indicates similar genes. The CLV1 and its orthologs in legumes are in grey. The <span class="trait">number</span> &#x02018;1&#x02019; represents a truncated gene. ( <bold> D </bold> ) Positioning and size of <italic> <span class="gene">GmCLV1A</span> </italic> with <italic> <span class="gene">GmTrCLV1A,</span> PvNARK </italic> with <italic> PvTrNARK </italic> and <italic> MtSUNN </italic> with <italic> MtRLP1 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Phenotypes of <span class="trait">pod,</span> stem (as demonstrated by cotyledonary node <span class="trait">branching)</span> and nodulated root systems of the soybean wild type Forrest, and its TILLING-derived mutants <italic> , <span class="gene">Gmclv1a</span> </italic> ( <italic> S562L) </italic> and <italic> <span class="gene">Gmnark</span> </italic> ( <italic> W677* </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Macro- and microscopic phenotypes of the soybean wild type Forrest, and its mutant <italic> S562L </italic> . ( <bold> A) </bold> Stem thickness of 5 month-old plants (plants were intentionally defoliated to enhance visibility of stem architecture); ( <bold> B) </bold> First trifoliate node showing fasciation and excessive flowering in the mutant; ( <bold> C) </bold> Vein-bladed leaf structures on the underside of <italic> <span class="gene">Gmclv1a</span> </italic> mutant leaves. ( <bold> D) </bold> Young <span class="trait">pod</span> morphology (dashed line indicates the position of the cross-section seen in ( <bold> F </bold> ). ( <bold> E) </bold> Stem section at node 4 of Forrest and <italic> S562L </italic> mutant plants (4 month-old). ( <bold> F) </bold> Young <span class="trait">pod</span> cross-sections. Note the bifurcated, deformed <span class="trait">pod</span> of the <italic> S562L </italic> mutant. VB&#x02009;=&#x02009;Vascular bundle; Ep&#x02009;=&#x02009;Epidermis; IS&#x02009;=&#x02009;Inner sclerenchyma.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Branching</span> phenotype of 4 week-old soybean cv. Forrest, its mutant <italic> <span class="gene">Gmclv1a</span> </italic> ( <italic> S562L </italic> ), and F <sub> 2 </sub> segregants from a cross between them. CC&#x02009;=&#x02009;wild-type segregants; cc&#x02009;=&#x02009; <italic> <span class="gene">Gmclv1a</span> </italic> segregants. Forrest n&#x02009;=&#x02009;10, <italic> S562L </italic> n&#x02009;=&#x02009;9, CC n&#x02009;=&#x02009;11 and cc n&#x02009;=&#x02009;14. Error bars indicate SE. Different letters above bars represent statistically significant differences (Student&#x02019;s <italic> t </italic> test; <italic> P </italic> &#x02264; 0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						Phenotypes of reciprocally grafted (scion/rootstock) plants between wild-type soybean cv. Forrest and its mutants <italic> <span class="gene">Gmclv1a</span> </italic> ( <italic> S562L </italic> ) <italic> <span class="gene">Gmnark</span> </italic> ( <italic> W677* </italic> ). Plants were grafted 12 days after sowing. Data were collected 45 days later. ( <bold> A) </bold> Nodule <span class="trait">number</span> per plant; ( <bold> B) </bold> lateral root <span class="trait">number</span> per plant (in the 5&#x02013;15&#x02009;cm region below the crown); ( <bold> C) </bold> average nodule weight; and ( <bold> D) </bold> nodulation index ( <italic> i.e </italic> ., % of root nodulated). Different letters above the <span class="gene">bar</span> represent statistically significant differences (Duncan test; <italic> P </italic> &#x02009;&#x02264;&#x02009;0.05). Error bars indicate SE.
					</p>
				
					<p>&nbsp;&nbsp;
						Temperature influence on phenotypes of wild-type soybean cv. Forrest, and its mutants <italic> <span class="gene">Gmclv1a</span> </italic> ( <italic> S562L </italic> ), <italic> <span class="gene">Gmnark</span> </italic> ( <italic> W677* </italic> ) and the <italic> <span class="gene">Gmclv1a</span> <span class="gene">Gmnark</span> </italic> double mutant (DM). Plants were grown at 28/25&#x02009;&#x000b0;C or 20/17&#x02009;&#x000b0;C. ( <bold> A) </bold> Plant height. ( <bold> B) </bold> Node <span class="trait">number.</span> ( <bold> C) </bold> Leaf <span class="trait">number</span> at node 3. ( <bold> D) </bold> Percentage of plants having at least one vein-bladed leaf; Vein-bladed phenotype were scored 4 weeks after flowering. ( <bold> E) </bold> <span class="trait">Pod</span> <span class="trait">number</span> (including both developing and mature pods). ( <bold> F) </bold> Nodule <span class="trait">number</span> per plant. ( <bold> G) </bold> Shoot dry weight. ( <bold> H) </bold> Root dry weight. Plant height, node <span class="trait">number</span> and leaf <span class="trait">number</span> at node 3 were measured 4 weeks after sowing; n&#x02009;=&#x02009;9&#x02013;13. Nodule <span class="trait">number,</span> shoot and root dry weight were measured 3 weeks after sowing; n&#x02009;=&#x02009;6. Error bars indicate SE. Nd&#x02009;=&#x02009;&#x02018;not detected&#x02019;. Different letters above the <span class="gene">bar</span> represent statistically significant differences (Duncan test; <italic> P </italic> &#x02264; 0.05).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>