<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Powdery mildew</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Powdery mildew</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Comparative analysis of genome-wide Mlo gene family in Cajanus cajan and Phaseolus vulgaris.</h2>
			<h3>PMID:26961357  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deshmukh R(1), Singh VK(2), Singh BD(3).
				<br/> 
				年份：2016 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Mlo gene was discovered in barley because the mutant 'mlo' allele conferred broad-spectrum, non-race-specific <span class="trait">resistance</span> to <span class="trait">powdery</span> <span class="trait">mildew</span> caused by Blumeria graminis f. sp. hordei. The Mlo genes also play important roles in growth and development of plants, and in responses to biotic and abiotic stresses. The Mlo gene family has been characterized in several crop species, but only a single legume species, soybean (Glycine max L.), has been investigated so far. The present report describes in silico identification of 18 CcMlo and 20 PvMlo genes in the important legume crops Cajanus cajan (L.) Millsp. and Phaseolus vulgaris L., respectively. In silico analysis of gene organization, protein properties and conserved domains revealed that the C. cajan and P. vulgaris Mlo gene paralogs are more divergent from each other than from their orthologous pairs. The comparative phylogenetic analysis classified CcMlo and PvMlo genes into three major clades. A comparative analysis of CcMlo and PvMlo proteins with the G. max Mlo proteins indicated close association of one CcMlo, one PvMlo with two <span class="gene">GmMlo</span> genes, indicating that there was no further expansion of the Mlo gene family after the separation of these species. Thus, most of the diploid species of eudicots might be expected to contain 15-20 Mlo genes. The genes CcMlo12 and 14, and PvMlo11 and 12 are predicted to participate in <span class="trait">powdery</span> <span class="trait">mildew</span> <span class="trait">resistance.</span> If this prediction were verified, these genes could be targeted by TILLING or CRISPR to isolate <span class="trait">powdery</span> <span class="trait">mildew</span> resistant mutants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative phylogenetic analysis of genome-wide Mlo gene family members from Glycine max and Arabidopsis thaliana.</h2>
			<h3>PMID:24469270  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deshmukh R(1), Singh VK, Singh BD.
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Powdery</span> <span class="trait">mildew</span> locus O (Mlo) gene family is one of the largest seven transmembrane protein-encoding gene families. The Mlo proteins act as negative regulators of <span class="trait">powdery</span> <span class="trait">mildew</span> <span class="trait">resistance</span> and a loss-of-function mutation in Mlo is known to confer broad-spectrum <span class="trait">resistance</span> to <span class="trait">powdery</span> <span class="trait">mildew.</span> In addition, the Mlo gene family members are known to participate in various developmental and biotic and abiotic stress response-related pathways. Therefore, a genome-wide similarity search using the characterized Mlo protein sequences of Arabidopsis thaliana was carried out to identify putative Mlo genes in soybean (Glycine max) genome. This search identified 39 Mlo domain containing protein-encoding genes that were distributed on 15 of the 20 G. max chromosomes. The putative promoter regions of these Mlo genes contained response elements for different external stimuli, including different hormones and abiotic stresses. Of the 39 <span class="gene">GmMlo</span> proteins, 35 were rich (8.7-13.1 %) in leucine, while five were serine-rich (9.2-11.9 %). Furthermore, all the <span class="gene">GmMlo</span> members were localized in the plasma membrane. Phylogenetic analysis of the <span class="gene">GmMlo</span> and the AtMlo proteins classified them into three main clusters, and the cluster I comprised two sub-clusters. Multiple sequence alignment visualized the location of seven transmembrane domains, and a conserved CaM-binding domain. Some of the <span class="gene">GmMlo</span> proteins <span class="gene">(GmMlo10,</span> 20, 22, 23, 32, 36, 37) contained less than seven transmembrane domains. The motif analysis yielded 27 motifs; out of these, motif 2, the only motif present in all the <span class="gene">GmMlos,</span> was highly conserved and three amino acid residues were essentially invariant. Five of the <span class="gene">GmMlo</span> members were much smaller in size; presumably they originated through deletion following a gene duplication event. The presence of a large number of <span class="gene">GmMlo</span> members in the G. max genome may be due to its paleopolyploid nature and the large genome size as compared to that of Arabidopsis. The findings of this study may further help in characterization and isolation of individual <span class="gene">GmMlo</span> members.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nod factor induces soybean <span class="trait">resistance</span> to <span class="trait">powdery</span> <span class="trait">mildew.</span></h2>
			<h3>PMID:16310369  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Duzan HM(1), Mabood F, Zhou X, Souleimanov A, Smith DL.
				<br/> 
				年份：2005 Oct-Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants possess highly sensitive perception systems by which microbial signal molecules are recognized. In the Bradyrhizobium-soybean (Glycine max (L.) Merr.) symbiosis, recognition is initiated through exchange of signal molecules, generally flavonoids from soybean and lipo-chitooligosaccharides (Nod factors) from the microsymbiont. Application of the Nod factor Nod Bj-V (C18:1, MeFuc) induced soybean <span class="trait">resistance</span> to <span class="trait">powdery</span> <span class="trait">mildew</span> caused by Microsphaera diffusa. Addition of Nod factor (concentrations ranging from 10(-6) to 10(-10) M) to soybean root systems led to reductions in disease incidence. The lowest disease incidence was caused by Nod factor treatment at 10(-6) M. The effect of Nod factor application on fungal growth and development was measured at 4, 12, 48, and 96 h after inoculation. Colony diameter and number of germ tubes per conidium were decreased by 10(-6) M Nod factor. Phenylalanine ammonia lyase (PAL, EC.4.3.1.1.) is the first enzyme of the phenyl propanoid pathway, and is commonly activated as part of plant responses to disease. Treatment of soybean seedlings with Nod factor, through stem wounds, induced <span class="gene">PAL</span> activity; the most rapid increase followed treatment with 10(-6) M Nod factor. These data show that soybean plants are able to detect root applied LCO and respond by increased disease <span class="trait">resistance.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:PCR Sampling of disease resistance-like sequences from a disease <span class="trait">resistance</span> gene cluster in soybean.</h2>
			<h3>PMID:12582561</h3>
				

				
			<p>
				作者：Graham A(1), Marek F, Shoemaker C.
				<br/> 
				年份：2002 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Clusters of Resistance-like genes (RLGs) have been identified from a variety of plant species. In soybean, RLG-specific primers and BAC-fingerprinting were used to develop a contig of overlapping BACs for a cluster of RLGs on soybean linkage group J. The <span class="trait">resistance</span> genes Rps2 ( Phytophthora stem and root rot) and Rmd-c <span class="trait">(powdery</span> <span class="trait">mildew)</span> and the ineffective nodulation gene <span class="gene">Rj2</span> were previously mapped to this region of linkage group J. PCR hybridization was used to place two TIR/NBD/LRR cDNAs on overlapping BACs from this contig. Both of the cDNAs were present on BAC 34P7. Fingerprinting of this BAC suggested as many as twelve different RLGs were present. Given the high nucleotide identity shared between cDNAs LM6 and MG13 (&gt;90%), direct sequencing of this region would be difficult. More sequence information was needed about the RLGs within this region before sequencing could be undertaken. By comparing the genomic sequences of cDNAs LM6 and MG13 we identified conserved regions from which oligonucleotide primers specific to BAC 34P7 RLGs could be designed. The nine primer pairs spanned the genomic sequence of LM6 and produced overlapping RLG products upon amplification of BAC 34P7. Amplification products from 12 different RLGs were identified. On average, nucleotide identity between RLG sequences was greater than 95%. Examination of RLG sequences also revealed evidence of additions, deletions and duplications within targeted regions of these genes. Using previously mapped cDNAs we were able to quickly and inexpensively access multiple RLGs within a single specific cluster.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Organization, expression and evolution of a disease <span class="trait">resistance</span> gene cluster in soybean.</h2>
			<h3>PMID:12524363  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Graham MA(1), Marek LF, Shoemaker RC.
				<br/> 
				年份：2002 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						PCR amplification was previously used to identify a cluster of <span class="trait">resistance</span> gene analogues (RGAs) on soybean linkage group J. <span class="trait">Resistance</span> to <span class="trait">powdery</span> <span class="trait">mildew</span> (Rmd-c), Phytophthora stem and root rot (Rps2), and an ineffective nodulation gene (Rj2) map within this cluster. BAC fingerprinting and RGA-specific primers were used to develop a contig of BAC clones spanning this region in cultivar "Williams 82" [rps2, Rmd (adult onset), rj2]. Two cDNAs with homology to the TIR/NBD/LRR family of R-genes have also been mapped to opposite ends of a BAC in the contig <span class="gene">Gm_Isb001_091F11</span> (BAC 91F11). Sequence analyses of BAC 91F11 identified 16 different resistance-like gene (RLG) sequences with homology to the TIR/NBD/LRR family of disease <span class="trait">resistance</span> genes. Four of these RLGs represent two potentially novel classes of disease <span class="trait">resistance</span> genes: TIR/NBD domains fused inframe to a putative defense-related protein (NtPRp27-like) and <span class="gene">TIR</span> domains fused inframe to soybean calmodulin Ca(2+)-binding domains. RT-PCR analyses using gene-specific primers allowed us to monitor the expression of individual genes in different tissues and developmental stages. Three genes appeared to be constitutively expressed, while three were differentially expressed. Analyses of the R-genes within this BAC suggest that R-gene evolution in soybean is a complex and dynamic process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Genomics of Non-TNL Disease <span class="trait">Resistance</span> Genes from Six Plant Species</h2>
			<h3>PMID:28973974</h3>
				
					<h3>PMCID:5664099</h3>
					<h4>期刊:Genes</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="true"> https://orcid.org/0000-0001-5565-523X </contrib-id>  <name>  <surname> Nepal </surname>  <given-names> Madhav P. </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-08-00249"> 1 </xref>  <xref rid="c1-genes-08-00249" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Andersen </surname>  <given-names> Ethan J. </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-08-00249"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neupane </surname>  <given-names> Surendra </given-names>  </name>  <xref ref-type="aff" rid="af1-genes-08-00249"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Benson </surname>  <given-names> Benjamin V. </given-names>  </name>  <xref ref-type="aff" rid="af2-genes-08-00249"> 2 </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Expression values of the CNL-A gene members were high in all species as shown in <xref ref-type="app" rid="app1-genes-08-00249"> Figure S18 </xref> . Relative to three other clades, CNL-A had a disproportionate number (12 of the 30) of highly expressed genes in three species <span class="gene">(Glyma01g39000,</span> Potri.002G129300, Potri.007G039000, Potri.014G035700, Medtr1g021100, Medtr1g021110, Medtr5g018060, Medtr5g018210, Medtr5g036240, Medtr8g079280, Medtr8g079350 and Medtr8g079360), data not available for <italic> P. vulgaris </italic> . The expression values were below average for the CNL-A orthologs in <italic> Arabidopsis </italic> , and the CNL-A clade was absent in rice. CNL-A clade had the highest expression values in <italic> Arabidopsis </italic> , <italic> Medicago </italic> , and poplar, but second highest in soybean. CNL-B had 12 highly expressed genes <span class="gene">(Glyma01g10254,</span> <span class="gene">Glyma07g06914,</span> <span class="gene">Glyma14g01231,</span> <span class="gene">Glyma18g46050,</span> <span class="gene">Glyma18g51533,</span> Potri.006G147100, Medtr4g091380, At4G10780, At1G12280, At1G63350, At1G61180, and At1G62630) ( <xref ref-type="app" rid="app1-genes-08-00249"> Figure S18 </xref> ). The CNL-B orthologs did not have an as easily discernible expression patterns as the CNL-A genes had in terms of gene homology associated with specific functions. The only gene homologies in this clade were RPS5 <span class="gene">(At1g12220)</span> and RPS2 (At4g26090), which did not appear to be among the highly expressed genes. The proportion of highly expressed genes in clade CNL-C corresponded to the clade size ( <xref ref-type="app" rid="app1-genes-08-00249"> Figure S19 </xref> ), with 70 highly expressed sequences (see <xref ref-type="table" rid="genes-08-00249-t006"> Table 6 </xref> ), while clade CNL-D had only two highly expressed genes (Potri.018G138500 and At1G10920; see <xref ref-type="app" rid="app1-genes-08-00249"> Figure S18 </xref> ). CNL-A genes were nested into three subclades, two of which were similar to a previous study by Collier (2011) [ <xref rid="B55-genes-08-00249" ref-type="bibr"> 55 </xref> ]. The highly expressed sequences could represent those genes that are necessary for pathways other than the disease signaling. The clade of Activated Disease <span class="trait">Resistance</span> 1-Like 1 ( <italic> ADR1-L1 </italic> ; AT4G33300) orthologs does not contain a member from common bean, while the clade of <italic> NRG1 </italic> (N requirement gene 1) orthologs includes one member from common bean genome. Both <italic> NRG1 </italic> and <italic> ADR1 </italic> proteins are possible downstream targets in the defense signaling pathway, which could explain why the CNL-A group contains proportionally higher number of highly expressed genes [ <xref rid="B54-genes-08-00249" ref-type="bibr"> 54 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The following are available online at <uri xlink:href="www.mdpi.com/2073-4425/8/10/249/s1"> www.mdpi.com/2073-4425/8/10/249/s1 </uri> . Figure S1: CNL-A clade with gene members nested into three subclades, Figure S2: CNL-B clade with gene members nested into three subclades, Figure S3: CNL-C clade with gene members nested into 13 subclades, Figure S4: CNL-D clade with gene members, including RPP8 and RPP13 orthologs, Figure S5: MEME analysis and predicted domain structure of nTNL genes in <italic> A. thaliana </italic> CNL. Conserved domain structures as predicted by MEME analysis of CNL genes in <italic> Arabidopsis </italic> . Sequences are grouped by CNL subgroup. The first three sequences appearing at the top (AT4G1905 (NL-A), AT5G66630 (CN-A), AT3g15700 (CN-B)) are without CNL designation. On the right are shown twenty conserved domains searched and weblogo visualization of residue frequency, Figure S6A&#x02013;C: MEME analysis predicted domain structure of <italic> M. truncatula </italic> nTNL genes. Twenty different sites were analyzed for prediction of conserved domain structures. Sequences are grouped by CNL subgroup. (A) Contains the motif WebLogo and legend along with sequences belonging to CNL-A, CNL-B and CNL-D. (B) Contains sequences that belong to CNL-C that appearing on chromosome 1 through chromosome 4. (C) contains sequences that from Clad CNL-C appearing on chromosome 5 through Chromosome 8, Figure S7A&#x02013;B: MEME analysis and predicted domain structure of <italic> O. sativa </italic> nTNL genes. Twenty different sites were searched for conserved domain structures. Sequences are grouped by their CNL subgroup. Three CNL-B sequences are located at the top of column one and one sequence that did not lie in any of the other CNL-sub groups identified as unsettled is listed at the bottom of column two. All other sequences belong to CNL-C of rice. (A) Contains the Motif weblogo and legend along with the sequences belonging to CNL-B the single unsettled sequence and CNL-C sequences appearing on chromosome 1 through chromosome 5. (B) Contains CNL-C sequences appearing on chromosome 6 through chromosome 12, Figure S8: MEME analysis and predicted domain structure of <italic> P. vulgaris </italic> nTNL genes. Twenty different sites were searched for conserved domain structures. Sequences are grouped by their CNL subgroup, Figure S9A&#x02013;B: MEME analysis and predicted domain structure of <italic> P. trichocarpa </italic> nTNL genes. Twenty different sites were searched for conserved domain structures. Sequences are grouped by their CNL subgroup. (A) Contains accessions from CNL-A, B, and D. (B) Contains accessions from CNL-C. Figure S10: Chromosomal distribution of the nTNL genes of <italic> A. thaliana </italic> (2N = 10). Black lines represent chromosomes, black ovals represent centromeres, and arrows indicate the location and orientation of CNL genes. Gene clades CNL-A, CNL-B, CNL-C and CNL-D were color-coded in blue, pink, red and green, respectively. Figure S11: Chromosomal distribution of the nTNL genes of <italic> M. truncatula </italic> (2N = 16). Black lines represent chromosomes, black ovals represent centromeres, and arrows indicate the location and orientation of CNL genes. Gene clades CNL-A, CNL-B, CNL-C and CNL-D were color-coded in blue, pink, red and green, respectively. Figure S12: Chromosomal distribution of the nTNL genes of <italic> O. sativa </italic> (2N = 24). Black lines represent chromosomes, black ovals represent centromeres, and arrows indicate the location and orientation of CNL genes. Gene clades CNL-A, CNL-B, CNL-C and CNL-D were color-coded in blue, pink, red and green, respectively. Figure S13: Chromosomal distribution of the nTNL genes of <italic> P. vulgaris </italic> (2N = 22). Black lines represents chromosomes, black ovals represent centromeres, yellow ovals represent the pericentromeric regions, and arrows indicate the location and orientation of CNL genes. Gene clades CNL-A, CNL-B, CNL-C and CNL-D were color-coded in blue, pink, red and green, respectively. Figure S14: Chromosomal distribution of the nTNL genes of <italic> P. trichocarpa </italic> (2N = 36). Black lines represent chromosomes, black ovals represent centromere, and arrows indicate the location and orientation of CNL genes. Gene clades CNL-A, CNL-B, CNL-C and CNL-D were color-coded in blue, pink, red and green, respectively. Figure S15: Syntenic map of chromosomes with high populations of nTNL genes in the six plant species, Figure S16: Syntenic map of chromsomes showing synteny with chromosome 11 in rice (Os11), Figure S17: Phylogeny of nTNL accessions from the short arm of chromosome Mt03 (left panel), along with the major block of synteny with this region, found on chromosome Pv05 (right panel), Figure S18: Abundance of number of expressed CNL-A, CNL-B, and CNL-D in available plant genomes (At = <italic> A. thaliana </italic> , Gm = <italic> Glycine max </italic> , Mt = <italic> Medicago truncatula </italic> , Os = <italic> Oryza sativa </italic> , and Pt = <italic> Populus trichocarpa </italic> ). The expression level are color coded: abundence of CNL gene members with expression level above average are shown in red while those below average are in blue. Figure S19: Abundance of number of expressed CNL-C genes in five plant genomes (At = <italic> A. thaliana </italic> , Gm = <italic> Glycine max </italic> , Mt = <italic> Medicago truncatula </italic> , Os = <italic> Oryza sativa </italic> , and Pt = <italic> Populus trichocarpa </italic> ). The expression level are color coded: abundence of CNL gene members with expression level above average are shown in red while those below average are in blue. Figure S20: Gene expression data for <italic> A. thaliana </italic> nTNL gene members are visualized as heatmaps. Log 2-base value was employed to construct heatmaps. Samples were collected 24 h after inoculation. Control was a mock inoculation. Avrpstdc3000, represents data collected from pathogenic leaf bacterium ( <italic> P. syringae </italic> pv. <italic> abrassicicola </italic> ), was a pathogenic leaf fungus ( <italic> Alternaria brassicicola </italic> ), <italic> P. rapae </italic> was a tissue-chewing caterpillar ( <italic> Pieris rapae </italic> ) and F-occidentalis, cell-content-feeding thrips ( <italic> Frankliniella occidentalis </italic> ). Figure S21: Gene expression data for <italic> G. max </italic> nTNL gene members are visualized as heatmaps. Log 2-base value was employed to construct heatmaps. All samples were collected 24 h after inoculation. Control (Moc) was the mock inoculation data at 24 h. Tw80-2 samples are representations of virulent soybean <span class="trait">rust</span> <italic> Phakopsora pachyrhizi </italic> (tw80-2). Hw94-1 represents an avirulent soybean <span class="trait">rust</span> challenge (hw94-1) from the same pathogen. Figure S22: Gene expression data for <italic> M. truncatula </italic> nTNL gene members are visualized as heatmaps. All experimental samples were collected 72 h after inoculation (hai). Each of the three conditions had its own control (identified with moc) taken 0 hai. <italic> Ralstonia solanacearum </italic> is the causal agent of the devastating bacterial wilt disease. <italic> Ralstonia solanacearum </italic> was analyzed in the pathosystems A17 (identified as A17 72 hai) and F83005.5 (identified as f83005.5 72hai) resistant <italic> M. truncatula </italic> lines infected with the pathogenic strain <span class="gene">GMI1000.</span> The mutant A17 line, Sickle (identified as sickle 72 hai), which showed a resistant phenotype was also part of the experiment. Figure S23: Gene expression data for <italic> O. sativa </italic> nTNL gene members are visualized as heatmaps. All experimental samples were collected at 24 h after inoculation. Xoo represents samples collected from leaves inoculated with Bacterial blight <italic> Xanthomonas oryzae </italic> pathovar (pv.) <italic> oryzae </italic> and Xoc bacterial leaf streak <italic> X. oryzae </italic> pv. <italic> oryzicola. </italic> The control for this experiment was a mock inoculated sample examined 24 h after inoculation. Figure S24: Gene expression data for <italic> P. trichocarpa </italic> nTNL gene members are visualized as heatmaps. Control samples (identified as Moc) were taken at 0 h after inoculation. Experimental samples (MARSSONINABRUNNEAIN 2DAI) were taken 48 h after inoculation of fungal pathogen <italic> Marssonia brunneain. </italic> Table S1: Exon locations within the nTNL genes of the six species. Each gene accession is listed along with its corresponding exon locations, described by chromosome location, nucleotide length within gene, and exon rank, Table S2: Protein domain structure of RNL (RNL, RN, and RCNL) proteins, as annotated in InterProScan. All 26 sequences nested within the CNL-A clade ( <xref ref-type="fig" rid="genes-08-00249-f001"> Figure 1 </xref> ). In terms of statistical significance, all &#x0201c; <italic> Arabidopsis </italic> broad-spectrum <span class="trait">mildew</span> <span class="trait">resistance</span> protein RPW8&#x0201d; domains had E-values between 2.3 &#x000d7; 10 <sup> &#x02212;7 </sup> and 2.8 &#x000d7; 10 <sup> &#x02212;45 </sup> , whereas RPW8 domain profiles were all between 8.6 and and 31.5.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>