<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Pseudomonas syringae</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Pseudomonas syringae</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Enhanced Disease Susceptibility1 Mediates Pathogen <span class="trait">Resistance</span> and Virulence Function of a Bacterial Effector in Soybean.</h2>
			<h3>PMID:24872380</h3>
				

				
			<p>
				作者：Wang J(1), Shine MB(1), Gao QM(1), Navarre D(1), Jiang W(1), Liu C(1), Chen Q(1), Hu G(2), Kachroo A(2).
				<br/> 
				年份：2014 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Enhanced disease susceptibility1 (EDS1) and phytoalexin deficient4 (PAD4) are well-known regulators of both basal and <span class="trait">resistance</span> (R) protein-mediated plant defense. We identified two EDS1-like <span class="gene">(GmEDS1a/GmEDS1b)</span> proteins and one PAD4-like <span class="gene">(GmPAD4)</span> protein that are required for <span class="trait">resistance</span> signaling in soybean (Glycine max). Consistent with their significant structural conservation to Arabidopsis (Arabidopsis thaliana) counterparts, constitutive expression of <span class="gene">GmEDS1</span> or <span class="gene">GmPAD4</span> complemented the pathogen <span class="trait">resistance</span> defects of Arabidopsis eds1 and pad4 mutants, respectively. Interestingly, however, the <span class="gene">GmEDS1</span> and <span class="gene">GmPAD4</span> did not complement pathogen-inducible salicylic acid accumulation in the eds1/pad4 mutants. Furthermore, the <span class="gene">GmEDS1a/GmEDS1b</span> proteins were unable to complement the turnip crinkle virus coat protein-mediated activation of the Arabidopsis R protein Hypersensitive reaction to Turnip crinkle virus (HRT), even though both interacted with HRT. Silencing <span class="gene">GmEDS1a/GmEDS1b</span> or <span class="gene">GmPAD4</span> reduced basal and pathogen-inducible salicylic acid accumulation and enhanced soybean susceptibility to virulent pathogens. The <span class="gene">GmEDS1a/GmEDS1b</span> and <span class="gene">GmPAD4</span> genes were also required for <span class="trait">Resistance</span> to <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv glycinea2 (Rpg2)-mediated <span class="trait">resistance</span> to <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span> Notably, the <span class="gene">GmEDS1a/GmEDS1b</span> proteins interacted with the cognate bacterial effector AvrA1 and were required for its virulence function in rpg2 plants. Together, these results show that despite significant structural similarities, conserved defense signaling components from diverse plants can differ in their functionalities. In addition, we demonstrate a role for <span class="gene">GmEDS1</span> in regulating the virulence function of a bacterial effector.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RPG1-B-derived <span class="trait">resistance</span> to AvrB-expressing <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> requires RIN4-like proteins in soybean.</h2>
			<h3>PMID:20484023  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Selote D(1), Kachroo A.
				<br/> 
				年份：2010 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) RPG1-B (for <span class="trait">resistance</span> to <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv glycinea) mediates species-specific <span class="trait">resistance</span> to P. <span class="trait">syringae</span> expressing the avirulence protein AvrB, similar to the nonorthologous RPM1 in Arabidopsis (Arabidopsis thaliana). RPM1-derived signaling is presumably induced upon AvrB-derived modification of the RPM1-interacting protein, RIN4 (for RPM1-interacting 4). We show that, similar to RPM1, RPG1-B does not directly interact with AvrB but associates with RIN4-like proteins from soybean. Unlike Arabidopsis, soybean contains at least four RIN4-like proteins <span class="gene">(GmRIN4a</span> to <span class="gene">GmRIN4d).</span> <span class="gene">GmRIN4b,</span> but not <span class="gene">GmRIN4a,</span> complements the Arabidopsis rin4 mutation. Both <span class="gene">GmRIN4a</span> and <span class="gene">GmRIN4b</span> bind AvrB, but only <span class="gene">GmRIN4b</span> binds RPG1-B. Silencing either <span class="gene">GmRIN4a</span> or <span class="gene">GmRIN4b</span> abrogates RPG1-B-derived <span class="trait">resistance</span> to P. <span class="trait">syringae</span> expressing AvrB. Binding studies show that <span class="gene">GmRIN4b</span> interacts with <span class="gene">GmRIN4a</span> as well as with two other AvrB/RPG1-B-interacting isoforms, <span class="gene">GmRIN4c</span> and <span class="gene">GmRIN4d.</span> The lack of functional redundancy among <span class="gene">GmRIN4a</span> and <span class="gene">GmRIN4b</span> and their abilities to interact with each other suggest that the two proteins might function as a heteromeric complex in mediating RPG1-B-derived <span class="trait">resistance.</span> Silencing <span class="gene">GmRIN4a</span> or <span class="gene">GmRIN4b</span> in rpg1-b plants enhances basal <span class="trait">resistance</span> to virulent strains of P. <span class="trait">syringae</span> and the oomycete Phytophthora sojae. Interestingly, <span class="gene">GmRIN4a-</span> or <span class="gene">GmRIN4b-silenced</span> rpg1-b plants respond differently to AvrB-expressing bacteria. Although both <span class="gene">GmRIN4a</span> and <span class="gene">GmRIN4b</span> function to monitor AvrB in the presence of RPG1-B, <span class="gene">GmRIN4a,</span> but not <span class="gene">GmRIN4b,</span> negatively regulates AvrB virulence activity in the absence of RPG1-B.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the <span class="gene">hrpC</span> and hrpRS operons of <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathovars <span class="trait">syringae,</span> tomato, and glycinea and analysis of the ability of hrpF, hrpG, hrcC, hrpT, and hrpV mutants to elicit the hypersensitive response and disease in plants.</h2>
			<h3>PMID:9721291  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deng WL(1), Preston G, Collmer A, Chang CJ, Huang HC.
				<br/> 
				年份：1998 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The species <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> encompasses plant pathogens with differing host specificities and corresponding pathovar designations. P. <span class="trait">syringae</span> requires the Hrp (type III protein secretion) system, encoded by a 25-kb cluster of hrp and hrc genes, in order to elicit the hypersensitive response (HR) in nonhosts or to be pathogenic in hosts. DNA sequence analysis of the <span class="gene">hrpC</span> and hrpRS operons of P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> 61 (brown spot of beans), P. <span class="trait">syringae</span> pv. glycinea U1 (bacterial blight of soybeans), and P. <span class="trait">syringae</span> pv. tomato DC3000 (bacterial speck of tomatos) revealed that the 13 genes comprising the right half of the hrp cluster (including those in the previously sequenced <span class="gene">hrpZ</span> operon) are conserved and identically arranged. The <span class="gene">hrpC</span> operon is comprised of hrpF, hrpG, hrcC, hrpT, and hrpV. hrcC encodes a putative outer membrane protein that is conserved in all type III secretion systems. The other four genes appear to be characteristic of group I Hrp systems, such as those possessed by P. <span class="trait">syringae</span> and Erwinia amylovora. The predicted products of these four genes in P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> 61 are HrpF (8 kDa), HrpG (15.4 kDa), HrpT (7.5 kDa), and HrpV (13.4 kDa). HrpT is a putative outer membrane lipoprotein. HrpF, HrpG, and HrpV are all hydrophilic proteins lacking N-terminal signal peptides. The HrpG, HrcC, HrpT, and HrpV proteins of P. <span class="trait">syringae</span> pathovars <span class="trait">syringae</span> and tomato (the two most divergent pathovars) had at least 76% amino acid identity with each other, whereas the HrpF proteins of these two pathovars had only 36% amino acid identity. The HrpF proteins of P. <span class="trait">syringae</span> pathovars <span class="trait">syringae</span> and glycinea also showed significant similarity to the HrpA pilin protein of P. <span class="trait">syringae</span> pathovar tomato. Functionally nonpolar mutations were introduced into each of the genes in the <span class="gene">hrpC</span> operon of P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> 61 by insertion of an nptII cartridge lacking a transcription terminator. The mutants were assayed for their ability to elicit the HR in nonhost tobacco leaves or to multiply and cause disease in host bean leaves. Mutations in hrpF, hrcC, and hrpT abolished or greatly reduced the ability of P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> 61 to elicit the HR in tobacco. The <span class="gene">hrpG</span> mutant had only weakly reduced HR activity, and the activity of the hrpV mutant was indistinguishable from that of the wild type. Each of the mutations could be complemented, but surprisingly, the hrpV subclone caused a reduction in the HR elicitation ability of the DeltahrpV::nptII mutant. The <span class="gene">hrpF</span> and hrcC mutants caused no disease in beans, whereas the hrpG, hrpT, and hrpV mutants had reduced virulence. Similarly, the hrcC mutant grew little in beans, whereas the other mutants grew to intermediate levels in comparison with the wild type. These results indicate that HrpC and HrpF have essential functions in the Hrp system, that HrpG and HrpT contribute quantitatively but are not essential, and that HrpV is a candidate negative regulator of the Hrp system.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the promoter of avirulence gene D from <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. tomato.</h2>
			<h3>PMID:8376338  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shen H(1), Keen NT.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The avirulence gene D (avrD) from <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. tomato comprises the first open reading frame (ORF) of a putative operon consisting of at least five tandem ORFs. The promoter of the avrD operon was localized to a 150-bp DNA fragment occurring 5' to the avrD gene by using the Tn7-lux and gus reporter systems. The avrD promoter in P. <span class="trait">syringae</span> pv. tomato and P. <span class="trait">syringae</span> pv. glycinea was poorly expressed when bacteria were grown in complex culture media but was activated during bacterial growth in plants. The timing and level of induction were similar in compatible and incompatible plant-pathogen interactions. When bacteria were grown in minimal culture medium, promoter activity was repressed by certain carbon sources, high concentrations of nitrogen compounds, and pH values above 6.5. Primer extension experiments on RNA from bacteria grown in minimal medium identified two transcription initiation sites 87 and 41 nucleotides upstream from the translational start site. Only the -41 transcriptional start site was identified in bacteria grown in soybean leaves. A sigma 54 promoter consensus sequence (GG-10 bp-GC) occurred 14 bp upstream of the -41 transcriptional start, and 3' deletions into this region completely abolished promoter activity. Little expression was observed when a gus fusion with the avrD promoter was introduced into an <span class="gene">ntrA</span> mutant strain of P. <span class="trait">syringae</span> pv. phaseolicola deficient in the sigma 54 cofactor. Expression from the avrD promoter also required the hrp regulatory genes, <span class="gene">hrpS</span> and hrpL. Deletions from the 5' end of the promoter region and base substitution analyses also identified two upstream elements important for expression. Sequence comparison of these elements with other cloned avirulence genes revealed the presence of a conserved consensus sequence elements with other cloned avirulence genes revealed the presence of a conserved consensus sequence (GGAACC-N15/16-CCAC) in the promoters of nine different avirulence genes from P. <span class="trait">syringae</span> pathovers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean NDR1-like proteins bind pathogen effectors and regulate <span class="trait">resistance</span> signaling.</h2>
			<h3>PMID:24372490  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Selote D(1), Shine MB, Robin GP, Kachroo A.
				<br/> 
				年份：2014 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nonrace specific disease <span class="trait">resistance</span> 1 (NDR1) is a conserved downstream regulator of <span class="trait">resistance</span> (R) protein-derived signaling. We identified two NDR1-like sequences <span class="gene">(GmNDR1a,</span> b) from soybean, and investigated their roles in R-mediated <span class="trait">resistance</span> and pathogen effector detection. Silencing <span class="gene">GmNDR1a</span> and b in soybean shows that these genes are required for <span class="trait">resistance</span> derived from the Rpg1-b, Rpg3, and Rpg4 loci, against <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> (Psg) expressing avrB, avrB2 and avrD1, respectively. Immunoprecipitation assays show that the <span class="gene">GmNDR1</span> proteins interact with the AvrB2 and AvrD1 Psg effectors. This correlates with the enhanced virulence of Psg avrB2 and Psg avrD1 in <span class="gene">GmNDR1-silenced</span> rpg3 rpg4 plants, even though these strains are not normally more virulent on plants lacking cognate R loci. The <span class="gene">GmNDR1</span> proteins interact with <span class="gene">GmRIN4</span> proteins, but not with AvrB, or its cognate R protein Rpg1-b. However, the <span class="gene">GmNDR1</span> proteins promote AvrB-independent activation of Rpg1-b when coexpressed with a phosphomimic derivative of <span class="gene">GmRIN4b.</span> The role of <span class="gene">GmNDR1</span> proteins in Rpg1-b activation, their direct interactions with AvrB2/AvrD1, and a putative role in the virulence activities of Avr effectors, provides the first experimental evidence in support of the proposed role for NDR1 in transducing extracellular pathogen-derived signals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmRIN4 protein family members function nonredundantly in soybean race-specific <span class="trait">resistance</span> against <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span></h2>
			<h3>PMID:23278373  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Selote D(1), Robin GP, Kachroo A.
				<br/> 
				年份：2013 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> effector AvrB interacts with four related soybean (Glycine max) proteins <span class="gene">(GmRIN4a-d),</span> three <span class="gene">(GmRIN4b,</span> c, d) of which also interact with the cognate <span class="trait">resistance</span> (R) protein, Rpg1-b. Here, we investigated the specific requirements for the <span class="gene">GmRIN4</span> proteins in R-mediated <span class="trait">resistance</span> and examined the mechanism of Rpg1-b activation. Using virus-induced gene silencing, we show that only <span class="gene">GmRIN4a</span> and b are required for Rpg1-b-mediated <span class="trait">resistance.</span> In planta binding assays show that <span class="gene">GmRIN4a</span> can associate with Rpg1-b indirectly via <span class="gene">GmRIN4b.</span> Pathogen-delivered AvrB induces the phosphorylation of <span class="gene">GmRIN4b</span> alone, and prevents interactions between <span class="gene">GmRIN4b</span> and Rpg1-b or <span class="gene">GmRIN4a.</span> Consistent with this result, a phosphomimic derivative of <span class="gene">GmRIN4b</span> (pm4b) fails to bind Rpg1-b and <span class="gene">GmRIN4a.</span> Conversely, a phosphodeficient derivative of <span class="gene">GmRIN4b</span> (pd4b) continues to bind the R protein and <span class="gene">GmRIN4a,</span> in the presence of AvrB. Coexpression of Rpg1-b with pm4b, but not <span class="gene">GmRIN4b</span> or pd4b, induces cell death and ion leakage in the heterologous Nicotiana benthamiana. Our data suggest that the AvrB-induced phosphorylation of <span class="gene">GmRIN4b,</span> and the subsequent inhibition of interaction among <span class="gene">GmRIN4b,</span> <span class="gene">GmRIN4a</span> and Rpg1-b, might activate the R protein. Furthermore, even though <span class="gene">GmRIN4c</span> and d are not required for Rpg1-b-derived <span class="trait">resistance,</span> they do function in <span class="trait">resistance</span> derived from other R loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of avirulence gene <span class="gene">avrRpt2</span> and identification of a putative regulatory sequence common to all known <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> avirulence genes.</h2>
			<h3>PMID:8335641  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Innes RW(1), Bent AF, Kunkel BN, Bisgrove SR, Staskawicz BJ.
				<br/> 
				年份：1993 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">avrRpt2</span> locus from <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. tomato causes virulent strains of P. <span class="trait">syringae</span> to be avirulent on some, but not all, lines of Arabidopsis thaliana and Glycine max (soybean). We determined the DNA sequence of the <span class="gene">avrRpt2</span> locus and identified the <span class="gene">avrRpt2</span> gene as a 768-bp open reading frame encoding a putative 28.2-kDa protein. Deletion analysis and transcription studies provided further evidence that this open reading frame encodes AvrRpt2. We found that the <span class="gene">avrRpt2</span> gene also has avirulence activity in P. <span class="trait">syringae</span> pathogens of Phaseolus vulgaris (common bean), suggesting that disease <span class="trait">resistance</span> genes specific to <span class="gene">avrRpt2</span> are functionally conserved among diverse plant species. The predicted AvrRpt2 protein is hydrophilic and contains no obvious membrane-spanning domains or export signal sequences, and there was no significant similarity of AvrRpt2 to sequences in the GenBank, EMBL, or Swiss PIR data bases. A comparison of the <span class="gene">avrRpt2</span> DNA sequence to nine other P. <span class="trait">syringae</span> avirulence genes revealed a highly conserved sequence, GGAACCNA-N14-CCACNNA, upstream of the translation initiation codon. This motif is located 6 to 8 nucleotides upstream of the transcription start site in all four P. <span class="trait">syringae</span> avirulence genes for which a transcription start site has been determined, suggesting a role as a binding site for a novel form of RNA polymerase. Regulation of <span class="gene">avrRpt2</span> was similar to other P. <span class="trait">syringae</span> avirulence genes; expression was high in minimal medium and low in rich medium and depended on the hrpRS locus and an additional locus at the opposite end of the hrp cluster of P. <span class="trait">syringae</span> pv. tomato.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation of a novel ABC-transporter gene from soybean induced by salicylic acid.</h2>
			<h3>PMID:16720608  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eichhorn H(1), Klinghammer M, Becht P, Tenhaken R.
				<br/> 
				年份：2006
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This paper reports on the identification and characterization of a new ATP-binding cassette (ABC) transporter which was identified as a salicylic acid-induced gene from soybean (Glycine max cv. Williams 82) in a subtractive suppression hybridization approach. A fragment of an ABC-transporter gene was used to isolate a full-length cDNA clone for this gene with a length of 4750 bp. The encoded protein has a length of 1447 amino acids and is composed of two similar repeat units typical of full-size ABC transporters. The sequence displays a close relationship to plant pleiotropic drug <span class="trait">resistance</span> (PDR)-type transporters and, on a homology basis, clusters together with the Arabidopsis thaliana <span class="gene">PDR12</span> gene, suggesting <span class="gene">GmPDR12</span> as a name for the gene isolated from soybean. <span class="gene">GmPDR12</span> is rapidly responsive to salicylic acid and methyl jasmonate. The mRNA starts to accumulate 30 min after the addition of the signalling compounds. Salicylic acid is required for the execution of the hypersensitive reaction in soybean cell suspension cultures inoculated with <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea. It has been demonstrated previously that salicylic acid can be substituted by a variety of functional analogues of salicylic acid. All of these compounds lead to a strong and rapid transcriptional activation of <span class="gene">GmPDR12,</span> suggesting a shared signalling pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">RpoN</span> (sigma(54)) is required for plasmid-encoded coronatine biosynthesis in <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span></h2>
			<h3>PMID:12726764  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Alarcón-Chaidez FJ(1), Keith L, Zhao Y, Bender CL.
				<br/> 
				年份：2003 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea PG4180 produces coronatine (COR), a phytotoxin which functions as a virulence factor in bacterial blight of soybeans. The COR biosynthetic gene cluster in PG4180 is borne on a 90-kb plasmid named p4180A. Although pathway-specific regulatory genes for COR have been identified, global regulatory genes for COR production in PG4180 remain undefined. In the present study, we evaluated the role of rpoN, which encodes sigma(54), in the virulence of strain PG4180. A <span class="gene">rpoN</span> mutant of PG4180, designated PG4180.K2, was unable to grow in M9 minimal medium; however, the addition of exogenous glutamate, glutamine or aspartate to M9 medium enabled PG4180.K2 to grow in vitro. PG4180.K2 could not induce disease symptoms or multiply in soybean plants and was defective in COR production and cor gene expression. Furthermore, PG4180.K2 was impaired in transcription of hrpL, an alternate sigma factor that mediates expression of genes in the type III secretion system of P. <span class="trait">syringae.</span> PG4180.K2 transconjugants with a wild-type copy of <span class="gene">rpoN</span> were complemented for hrpL and cor gene expression, COR biosynthesis, and growth in vitro. Our results indicate that <span class="gene">rpoN</span> is required for growth and the expression of both chromosomal and plasmid-encoded virulence factors in P. <span class="trait">syringae</span> pv. glycinea PG4180.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The matrix metalloproteinase gene <span class="gene">GmMMP2</span> is activated in response to pathogenic infections in soybean.</h2>
			<h3>PMID:11743122  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu Y(1), Dammann C, Bhattacharyya MK.
				<br/> 
				年份：2001 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Matrix metalloproteinases (MMPs) play an important role in host defense responses against pathogens in mammals where their activities lead to the production of antimicrobial peptides. We have identified a novel soybean (Glycine max) metalloproteinase gene, <span class="gene">GmMMP2,</span> that is transcriptionally up-regulated in infected tissues. The deduced amino acid sequence indicates that this gene belongs to the MMP family. It is a preproprotein containing an N-terminal signal peptide, a cysteine switch, a zinc-binding catalytic motif, and a C-terminal transmembrane domain. The <span class="gene">GmMMP2</span> expressed in and purified from Escherichia coli exhibited an in vitro enzymatic activity in digesting myelin basic protein. All plant metalloproteinases reported so far have no known functions. However, they have been suggested to be involved in extracellular cell matrix degradation during development or senescence. Our investigations demonstrate that the <span class="gene">GmMMP2</span> transcript levels were rapidly increased in compatible and incompatible interactions of soybean tissues with the oomycete pathogen Phytophthora sojae or the bacterial pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea. In agreement with the <span class="gene">GmMMP2</span> activation, a metalloproteinase activity was gradually increased in suspension-cultured cells following the bacterial infection. <span class="gene">GmMMP2</span> was also activated in response to wounding and dehydration. However, <span class="gene">GmMMP2</span> activation did not correlate with the oxidative burst leading to the hypersensitive response cell death or the tissue senescence progress that involves programmed cell death. Our investigations suggest that <span class="gene">GmMMP2</span> may be involved in a novel defense response of soybean against pathogenic infections.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The HrpZ proteins of <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pvs. <span class="trait">syringae,</span> glycinea, and tomato are encoded by an operon containing Yersinia ysc homologs and elicit the hypersensitive response in tomato but not soybean.</h2>
			<h3>PMID:7579616  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preston G(1), Huang HC, He SY, Collmer A.
				<br/> 
				年份：1995 Sep-Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathovars are composed of host-specific plant pathogens that characteristically elicit the defense-associated hypersensitive response (HR) in nonhost plants. P. s. pv. <span class="trait">syringae</span> 61 secretes an HR elicitor, harpinPss (HrpZPss), in a hrp-dependent manner. An internal fragment of the P. s. pv. <span class="trait">syringae</span> 61 <span class="gene">hrpZ</span> gene was used to clone the <span class="gene">hrpZ</span> locus from P. s. pv. glycinea race 4 (bacterial blight of soybean) and P. s. pv. tomato DC3000 (bacterial speck of tomato). DNA sequence analysis revealed that <span class="gene">hrpZ</span> is the second ORF in a polycistronic operon. The amino acid sequence identities of HrpZPss/HrpZPsg and HrpZPss/HrpZPst were 79 and 63%, respectively. Although none of the HrpZ proteins showed significant overall sequence similarity with other known proteins, HrpZPst contained a 24-amino acid sequence that is homologous with a region of the PopA1 elicitor protein of the tomato pathogen, <span class="trait">Pseudomonas</span> solanacearum <span class="gene">GMI1000.</span> hrpA, the upstream ORF, was highly divergent: The amino acid sequence identities of HrpAPss/HrpAPsg and HrpAPss/HrpAPst were 91 and 28%, respectively, and no HrpA sequence showed similarity to known proteins. In contrast, the predicted products of the downstream ORFs in P. s. pv. <span class="trait">syringae</span> and P. s. pv. tomato, hrpB, hrpC, hrpD, and <span class="gene">hrpE</span> showed varying levels of similarity to those of yscI, yscJ, yscK, and yscL. These are colinearly arranged genes in the virC locus of Yersinia spp., which are involved in the secretion of the Yop virulence proteins via the type III pathway. The similarity of the Ysc proteins was generally stronger in comparisons with the P. s. pv. tomato Hrp proteins.(ABSTRACT TRUNCATED AT 250 WORDS)
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cooperative functioning between phenylalanine ammonia lyase and isochorismate synthase activities contributes to salicylic acid biosynthesis in soybean.</h2>
			<h3>PMID:27411159</h3>
				

				
			<p>
				作者：Shine MB(1), Yang JW(1), El-Habbak M(1), Nagyabhyru P(1), Fu DQ(1), Navarre D(2), Ghabrial S(1), Kachroo P(1), Kachroo A(3).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Salicylic acid (SA), an essential regulator of plant defense, is derived from chorismate via either the phenylalanine ammonia lyase (PAL) or the isochorismate synthase (ICS) catalyzed steps. The ICS pathway is thought to be the primary contributor of defense-related SA, at least in Arabidopsis. We investigated the relative contributions of <span class="gene">PAL</span> and ICS to defense-related SA accumulation in soybean (Glycine max). Soybean plants silenced for five <span class="gene">PAL</span> isoforms or two ICS isoforms were analyzed for SA concentrations and SA-derived defense responses to the hemibiotrophic pathogens <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> and Phytophthora sojae. We show that, unlike in Arabidopsis, <span class="gene">PAL</span> and ICS pathways are equally important for pathogen-induced SA biosynthesis in soybean. Knock-down of either pathway shuts down SA biosynthesis and abrogates pathogen <span class="trait">resistance.</span> Moreover, unlike in Arabidopsis, pathogen infection is associated with the suppression of ICS gene expression. Pathogen-induced biosynthesis of SA via the <span class="gene">PAL</span> pathway correlates inversely with phenylalanine concentrations. Although infections with either virulent or avirulent strains of the pathogens increase SA concentrations, <span class="trait">resistance</span> protein-mediated response to avirulent P. sojae strains may function in an SA-independent manner. These results show that PAL- and ICS-catalyzed reactions function cooperatively in soybean defense and highlight the importance of <span class="gene">PAL</span> in pathogen-induced SA biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-Function Relationship of a Novel PR-5 Protein with Antimicrobial Activity from Soy Hulls.</h2>
			<h3>PMID:26753535  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu C(1), Cheng F(1), Sun Y(1), Ma H(2), Yang X(1)(3).
				<br/> 
				年份：2016 Feb 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An alkaline isoform of the PR-5 protein (designated <span class="gene">GmOLPc)</span> has been purified from soybean hulls and identified by MALDI-TOF/TOF-MS. <span class="gene">GmOLPc</span> effectively inhibited in vitro the growth of Phytophthora soja spore and <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv glycinea. The antimicrobial activity of <span class="gene">GmOLPc</span> should be mainly ascribed to its high binding affinity with vesicles composed of DPPG, (1,3)-β-D-glucans, and weak endo-(1,3)-β-D-glucanase activity. From the 3D models, predicted by the homology modeling, <span class="gene">GmOLPc</span> contains an extended negatively charged cleft. The cleft was proved to be a prerequisite for endo-(1,3)-β-D-glucanase activity. Molecular docking revealed that the positioning of linear (1,3)-β-D-glucans in the cleft of <span class="gene">GmOLPc</span> allowed an interaction with Glu83 and Asp101 that were responsible for the hydrolytic cleavage of glucans. Interactions of <span class="gene">GmOLPc</span> with model membranes indicated that <span class="gene">GmOLPc</span> possesses good surface activity which could contribute to its antimicrobial activity, as proved by the behavior of perturbing the integrity of membranes through surface hydrophobic amino acid residues (Phe89 and Phe94).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmRAR1 and <span class="gene">GmSGT1</span> are required for basal, R gene-mediated and systemic acquired <span class="trait">resistance</span> in soybean.</h2>
			<h3>PMID:19061405  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fu DQ(1), Ghabrial S, Kachroo A.
				<br/> 
				年份：2009 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						RAR1, SGT1, and <span class="gene">HSP90</span> are important components of effector-triggered immunity (ETI) in diverse plants, where RAR1 and SGT1 are thought to serve as <span class="gene">HSP90</span> co-chaperones. We show that ETI in soybean requires RAR1 and SGT1 but not HSP90. Rsv1-mediated extreme <span class="trait">resistance</span> to Soybean mosaic virus (SMV) and Rpg-1b-mediated <span class="trait">resistance</span> to <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> were compromised in plants silenced for <span class="gene">GmRAR1</span> and <span class="gene">GmSGT1-2</span> but not <span class="gene">GmHSP90.</span> This suggests that RAR1- or SGT1-dependant signaling is not always associated with a dependence on HSP90. Unlike in Arabidopsis, SGT1 in soybean also mediates ETI against the bacterial pathogen P. <span class="trait">syringae.</span> Similar to Arabidopsis, soybean RAR1 and SGT1 proteins interact with each other and two related <span class="gene">HSP90</span> proteins. Plants silenced for <span class="gene">GmHSP90</span> genes or <span class="gene">GmRAR1</span> exhibited altered morphology, suggesting that these proteins also contribute to developmental processes. Silencing <span class="gene">GmRAR1</span> and <span class="gene">GmSGT1-2</span> impaired <span class="trait">resistance</span> to virulent bacteria and systemic acquired <span class="trait">resistance</span> (SAR) in soybean as well. Because the Arabidopsis rar1 mutant also showed a defect in SAR, we conclude that RAR1 and SGT1 serve as a point of convergence for basal <span class="trait">resistance,</span> ETI, and SAR. We demonstrate that, although soybean defense signaling pathways recruit structurally conserved components, they have distinct requirements for specific proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:nolC, a Rhizobium fredii gene involved in cultivar-specific nodulation of soybean, shares homology with a heat-shock gene.</h2>
			<h3>PMID:1646377  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Krishnan HB(1), Pueppke SG.
				<br/> 
				年份：1991 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizobium fredii strain USDA257 does not nodulate soybean (Glycine max (L.) Merr.) cultivar McCall. Mutant 257DH5, which contains a Tn5 insert in the bacterial chromosome, forms nodules on this cultivar, but acetylene-reduction activity is absent. We have sequenced the region corresponding to the site of Tn5 insertion in this mutant and find that it lies within a 1176bp open reading frame that we designate nolC. <span class="gene">nolC</span> encodes a protein of deduced molecular weight 43564. Nucleotide sequences homologous to <span class="gene">nolC</span> are present in several other Rhizobium strains, as well as Agrobacterium tumefaciens, but not in <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathovar glycinea. <span class="gene">nolC</span> lacks significant sequence homology with known genes that function in nodulation, but is 61% homologous to dnaJ, an Escherichia coli gene that encodes a 41 kDa heat-shock protein. Both R. fredii USDA257 and mutant 257DH5 produce heat-shock proteins of 78, 70, 22, and 16kDa. A 4.3kb EcoRI-HindIII subclone containing <span class="gene">nolC</span> expresses a single 43kDa polypeptide in mini-cells. A longer, 9.4kb EcoRI fragment expresses both the 43kDa polypeptide and a 78kDa polypeptide that corresponds in size to that of the largest heat-shock protein. Thus, although <span class="gene">nolC</span> has strong sequence homology to <span class="gene">dnaJ</span> and appears to be linked to another heat-shock gene, it does not directly function in the heat-shock response.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathogens of Arabidopsis and a bacterial locus determining avirulence on both Arabidopsis and soybean.</h2>
			<h3>PMID:1824334  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Whalen MC(1), Innes RW, Bent AF, Staskawicz BJ.
				<br/> 
				年份：1991 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To develop a model system for molecular genetic analysis of plant-pathogen interactions, we studied the interaction between Arabidopsis thaliana and the bacterial pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv tomato (Pst). Pst strains were found to be virulent or avirulent on specific Arabidopsis ecotypes, and single ecotypes were resistant to some Pst strains and susceptible to others. In many plant-pathogen interactions, disease <span class="trait">resistance</span> is controlled by the simultaneous presence of single plant <span class="trait">resistance</span> genes and single pathogen avirulence genes. Therefore, we tested whether avirulence genes in Pst controlled induction of <span class="trait">resistance</span> in Arabidopsis. Cosmids that determine avirulence were isolated from Pst genomic libraries, and the Pst avirulence locus <span class="gene">avrRpt2</span> was defined. This allowed us to construct pathogens that differed only by the presence or absence of a single putative avirulence gene. We found that Arabidopsis ecotype Col-0 was susceptible to Pst strain DC3000 but resistant to the same strain carrying avrRpt2, suggesting that a single locus in Col-0 determines <span class="trait">resistance.</span> As a first step toward genetically mapping the postulated <span class="trait">resistance</span> locus, an ecotype susceptible to infection by DC3000 carrying <span class="gene">avrRpt2</span> was identified. The <span class="gene">avrRpt2</span> locus from Pst was also moved into virulent strains of the soybean pathogen P. <span class="trait">syringae</span> pv glycinea to test whether this locus could determine avirulence on soybean. The resulting strains induced a resistant response in a cultivar-specific manner, suggesting that similar <span class="trait">resistance</span> mechanisms may function in Arabidopsis and soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Metabolic Adaptation of <italic> Ralstonia solanacearum </italic> during Plant Infection: A Methionine Biosynthesis Case Study</h2>
			<h3>PMID:22615832</h3>
				
					<h3>PMCID:3353975</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Plener </surname>  <given-names> Laure </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boistard </surname>  <given-names> Pierre </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gonz&#x000e1;lez </surname>  <given-names> Adriana </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boucher </surname>  <given-names> Christian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Genin </surname>  <given-names> St&#x000e9;phane </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In addition to <italic> R. solanacearum </italic> , <italic> met </italic> auxotroph mutants have been described in other plant pathogens such as <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> <xref ref-type="bibr" rid="pone.0036877-Andersen1"> [32] </xref> , <xref ref-type="bibr" rid="pone.0036877-Somylai1"> [33] </xref> and <italic> Agrobacterium tumefaciens </italic> <xref ref-type="bibr" rid="pone.0036877-Lippincott1"> [34] </xref> . Whereas <italic> A. tumefaciens </italic> and <italic> P. <span class="trait">syringae</span> </italic> pv <italic> phaseolicola met </italic> mutants were avirulent, <italic> P. <span class="trait">syringae</span> </italic> pv <italic> <span class="trait">syringae</span> met </italic> genes were not tested for virulence but were shown to be required for optimum epiphytic fitness <xref ref-type="bibr" rid="pone.0036877-Beattie1"> [35] </xref> . Compared to these former studies, a major difference is that the <italic> <span class="gene">metE</span> </italic> or <italic> <span class="gene">metH</span> </italic> single mutants are not methionine auxotrophs and our observations incite to think that the methionine biosynthetic pathway contributes to <italic> R. solanacearum </italic> aggressiveness not only by promoting bacterial growth. The strongest argument supporting this view is that at 5 dpi, the <italic> <span class="gene">metE</span> </italic> and <italic> metR </italic> mutants reach population levels similar to the wild-type parent, whereas significant differences in the rate of symptom development are already visible at this time point. This indicates that bacterial multiplication <italic> in planta </italic> and wilt symptom production can be dissociated in <italic> metER </italic> mutants in the experimental conditions used. Future investigations should determine if this behavior is variable among various host plants and whether bacterial multiplication of <italic> metER </italic> mutants is directly linked to the methionine concentration inside these hosts. Finally, methionine biosynthesis appears to be an essential requirement for pathogenicity of phylogenetically distant <italic> R. solanacearum </italic> strains differing in their host range spectrum: a recent study led on strain IPO1609, which carries a large genomic deletion comprising the <italic> metER </italic> genes, was shown to be hypovirulent on potato and this reduced virulence was partly dependent on <italic> metER </italic> genes <xref ref-type="bibr" rid="pone.0036877-Gonzalez1"> [36] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">metE</span> </italic> and <italic> metR </italic> genes were disrupted by insertion of an &#x003a9; cassette <xref ref-type="bibr" rid="pone.0036877-Prentki1"> [42] </xref> conferring <span class="trait">resistance</span> to spectinomycin. A 960 bp fragment upstream <italic> <span class="gene">metE</span> </italic> was cloned after PCR amplification using primers metE-Xba <named-content content-type="gene"> 5&#x02032;-TCTAGAGGATCTCGAAGCGGAACAG-3&#x02032; </named-content> and metE-HindUP <named-content content-type="gene"> 5&#x02032;-AAGCTTGGTCATTTCCATGGCC-3&#x02032; </named-content> . A 980 bp fragment downstream of <italic> <span class="gene">metE</span> </italic> was cloned downstream of the first fragment using primers metE-HindDW <named-content content-type="gene"> 5&#x02032;-AAGCTTCCGCGAGGGCCTGCCG -3&#x02032; </named-content> and metE-Not <named-content content-type="gene"> 5&#x02032;-GCGGCCGCCGGCGACATCCACG-3&#x02032; </named-content> . The &#x003a9; cassette was inserted between the two fragments using <italic> Hin </italic> dIII restriction site, resulting in plasmid pLP45. A 1.6 kb fragment of <italic> metR </italic> gene was cloned after PCR amplification using primers metR-F <named-content content-type="gene"> 5&#x02032;- TTCGCGCACCTCGTCGAACA-3&#x02032; </named-content> and metR-R <named-content content-type="gene"> 5&#x02032;- GCGCCCCGCGTTCTGGAC-3&#x02032; </named-content> . The &#x003a9; cassette was inserted at position 225 of the gene using <italic> Eco </italic> RV restriction site resulting in plasmid pLP71. Both constructions were linearized by <italic> Xba </italic> I and introduced into <italic> R. solanacearum </italic> wild type and mutant strains by natural transformation. Double recombination events were selected using spectinomycin <span class="trait">resistance</span> and checked by PCR.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> metE::lacZ </italic> and <italic> metH::lacZ </italic> fusion were created by using integration plasmid pCZ367 <xref ref-type="bibr" rid="pone.0036877-Cunnac1"> [44] </xref> . An internal fragment of the <italic> <span class="gene">metE</span> </italic> gene was PCR amplified by using primers MetE-5 ( <named-content content-type="gene"> 5&#x02032;-AAGCTTATGAGCTCAAGGGCTGGCTG-3&#x02032; </named-content> ) and MetE-3 ( <named-content content-type="gene"> 5&#x02032;-TCTAGACTCCGAATAGCACATGTGCG-3&#x02032; </named-content> ) and cloned as a 938-bp <italic> Hin </italic> dIII- <italic> Xba </italic> I fragment into pCZ367. An internal fragment of the <italic> metHa </italic> gene was PCR amplified by using primers MetH-5 ( <named-content content-type="gene"> 5&#x02032;-AAGCTTCCTGGTGAACCTGGTCGG-3&#x02032; </named-content> ) and MetH-3 ( <named-content content-type="gene"> 5&#x02032;-ACCGGATCCAAGGCGTTC-3&#x02032; </named-content> ). These plasmids were used to transform <italic> R. solanacearum </italic> <span class="gene">GMI1000</span> and a single integrative event was selected by using pCZ367 gentamicin <span class="trait">resistance.</span> Recombinant clones were then checked by PCR.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:LYK4, a lysin motif receptor-like kinase, is important for chitin signaling and plant innate immunity in Arabidopsis.</h2>
			<h3>PMID:22744984  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wan J(1), Tanaka K, Zhang XC, Son GH, Brechenmacher L, Nguyen TH, Stacey G.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chitin is commonly found in fungal cell walls and is one of the well-studied microbe/pathogen-associated molecular patterns. Previous studies showed that lysin motif (LysM)-containing proteins are essential for plant recognition of chitin, leading to the activation of plant innate immunity. In Arabidopsis (Arabidopsis thaliana), the LYK1/CERK1 (for LysM-containing receptor-like kinase1/chitin elicitor receptor kinase1) was shown to be essential for chitin recognition, whereas in rice (Oryza sativa), the LysM-containing protein, CEBiP (for chitin elicitor-binding protein), was shown to be involved in chitin recognition. Unlike LYK1/CERK1, CEBiP lacks an intracellular kinase domain. Arabidopsis possesses three CEBiP-like genes. Our data show that mutations in these genes, either singly or in combination, did not compromise the response to chitin treatment. Arabidopsis also contains five LYK genes. Analysis of mutations in LYK2, -3, -4, or -5 showed that <span class="gene">LYK4</span> is also involved in chitin signaling. The lyk4 mutants showed reduced induction of chitin-responsive genes and diminished chitin-induced cytosolic calcium elevation as well as enhanced susceptibility to both the bacterial pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv tomato DC3000 and the fungal pathogen Alternaria brassicicola, although these phenotypes were not as dramatic as that seen in the lyk1/cerk1 mutants. Similar to LYK1/CERK1, the <span class="gene">LYK4</span> protein was also localized to the plasma membrane. Therefore, <span class="gene">LYK4</span> may play a role in the chitin recognition receptor complex to assist chitin signal transduction and plant innate immunity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Silencing genes encoding omega-3 fatty acid desaturase alters seed size and accumulation of Bean pod mottle virus in soybean.</h2>
			<h3>PMID:21117867  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Singh AK(1), Fu DQ, El-Habbak M, Navarre D, Ghabrial S, Kachroo A.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Omega-3 fatty acid desaturase (FAD3)-catalyzed conversion of linoleic acid to linolenic acid (18:3) is an important step for the biosynthesis of fatty acids as well as the phytohormone jasmonic acid (JA) in plants. We report that silencing three microsomal isoforms of <span class="gene">GmFAD3</span> enhanced the accumulation of Bean pod mottle virus (BPMV) in soybean. The <span class="gene">GmFAD3-silenced</span> plants also accumulated higher levels of JA, even though they contained slightly reduced levels of 18:3. Consequently, the <span class="gene">GmFAD3-silenced</span> plants expressed JA-responsive pathogenesis-related genes constitutively and exhibited enhanced susceptibility to virulent <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span> Increased accumulation of BPMV in <span class="gene">GmFAD3-silenced</span> plants was likely associated with their JA levels, because exogenous JA application also increased BPMV accumulation. The JA-derived increase in BPMV levels was likely not due to repression of salicylic acid (SA)-derived signaling because the <span class="gene">GmFAD3-silenced</span> plants were enhanced in SA-dependent defenses. Furthermore, neither exogenous SA application nor silencing the SA-synthesizing phenylalanine ammonia lyase gene altered BPMV levels in soybean. In addition to the altered defense responses, the <span class="gene">GmFAD3-silenced</span> plants also produced significantly larger and heavier seed. Our results indicate that loss of <span class="gene">GmFAD3</span> enhances JA accumulation and, thereby, susceptibility to BPMV in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RIN4-like proteins mediate <span class="trait">resistance</span> protein-derived soybean defense against <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span></h2>
			<h3>PMID:21051954  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Selote D(1), Kachroo A.
				<br/> 
				年份：2010 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> (R) protein mediated recognition of pathogen avirulence effectors triggers signaling that induces a very robust form of species-specific immunity in plants. The soybean Rpg1-b protein mediates this form of <span class="trait">resistance</span> against the bacterial blight pathogen, <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> expressing AvrB Pgyrace4. Likewise, the Arabidopsis RPM1 protein also mediates species-specific <span class="trait">resistance</span> against AvrB expressing bacteria. RPM1 and Rpg1-b are non-orthologous and differ in their requirements for downstream signaling components. We recently showed that the activation of Rpg1-b derived <span class="trait">resistance</span> signaling requires two host proteins that directly interact with AvrB. These proteins share high sequence similarity with the Arabidopsis RPM1 interacting protein 4 (RIN4), which is essential for RPM1-derived <span class="trait">resistance.</span> The two soybean RIN4-like proteins <span class="gene">(GmRIN4a</span> and b) differ in their abilities to interact with Rpg1-b as well as to complement the Arabidopsis rin4 mutation. Because the two <span class="gene">GmRIN4</span> proteins interact with each other, we proposed that they might function as a heteromeric complex in mediating Rpg1-b-derived <span class="trait">resistance.</span> Absence of <span class="gene">GmRIN4a</span> or b enhanced basal <span class="trait">resistance</span> against bacterial and oomycete pathogens in soybean. Lack of <span class="gene">GmRIN4a</span> also enhanced the virulence of avrB bacteria in plants lacking Rpg1-b. Our studies suggest that multiple RIN4-like proteins proteins mediate R-mediated signaling, in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Systemic acquired <span class="trait">resistance</span> in soybean is regulated by two proteins, Orthologous to Arabidopsis NPR1</h2>
			<h3>PMID:19656407</h3>
				

				
					<h3>PMCID:2738679</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" contrib-type="author">  <name>  <surname> Sandhu </surname>  <given-names> Devinder </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> dsandhu@uwsp.edu </email>  </contrib>  <contrib id="A2" equal-contrib="yes" contrib-type="author">  <name>  <surname> Tasma </surname>  <given-names> I Made </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> tasma12@yahoo.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Frasch </surname>  <given-names> Ryan </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Ryan.M.Frasch@uwsp.edu </email>  </contrib>  <contrib id="A4" corresp="yes" contrib-type="author">  <name>  <surname> Bhattacharyya </surname>  <given-names> Madan K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mbhattac@iastate.edu </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 8 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Systemic acquired <span class="trait">resistance</span> (SAR) is induced in non-inoculated leaves following infection with certain pathogenic strains. SAR is effective against many pathogens. Salicylic acid (SA) is a signaling molecule of the SAR pathway. The development of SAR is associated with the induction of pathogenesis related (PR) genes. Arabidopsis non-expressor of PR1 (NPR1) is a regulatory gene of the SA signal pathway 123. SAR in soybean was first reported following infection with Colletotrichum trancatum that causes anthracnose disease. We investigated if SAR in soybean is regulated by a pathway, similar to the one characterized in Arabidopsis. RESULTS: Pathogenesis-related gene <span class="gene">GmPR1</span> is induced following treatment of soybean plants with the SAR inducer, 2,6-dichloroisonicotinic acid (INA) or infection with the oomycete pathogen, Phytophthora sojae. In P. sojae-infected plants, SAR was induced against the bacterial pathogen, <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea. Soybean <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> genes showed high identities to Arabidopsis NPR1. They showed similar expression patterns among the organs, studied in this investigation. <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> are the only soybean homologues of NPR1and are located in homoeologous regions. In <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> transformed Arabidopsis npr1-1 mutant plants, SAR markers: (i) PR-1 was induced following INA treatment and (ii) BGL2 following infection with <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. tomato (Pst), and SAR was induced following Pst infection. Of the five cysteine residues, Cys82, Cys150, Cys155, Cys160, and Cys216 involved in oligomer-monomer transition in NPR1, Cys216 in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> proteins was substituted to Ser and Leu, respectively. CONCLUSION: Complementation analyses in Arabidopsis npr1-1 mutants revealed that homoeologous <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> genes are orthologous to Arabidopsis NPR1. Therefore, SAR pathway in soybean is most likely regulated by <span class="gene">GmNPR1</span> genes. Substitution of Cys216 residue, essential for oligomer-monomer transition of Arabidopsis NPR1, with Ser and Leu residues in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2,</span> respectively, suggested that there may be differences between the regulatory mechanisms of <span class="gene">GmNPR1</span> and Arabidopsis NPR proteins.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Pathogenesis-related gene <italic> <span class="gene">GmPR1</span> </italic> is induced following treatment of soybean plants with the SAR inducer, 2,6-dichloroisonicotinic acid (INA) or infection with the oomycete pathogen, <italic> Phytophthora sojae </italic> . In <italic> P. sojae </italic> -infected plants, SAR was induced against the bacterial pathogen, <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. glycinea. Soybean <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> <span class="gene">GmNPR1-2</span> </italic> genes showed high identities to Arabidopsis <italic> NPR1 </italic> . They showed similar expression patterns among the organs, studied in this investigation. <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> <span class="gene">GmNPR1-2</span> </italic> are the only soybean homologues of <italic> NPR1 </italic> and are located in homoeologous regions. In <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> <span class="gene">GmNPR1-2</span> </italic> transformed Arabidopsis <italic> npr1-1 </italic> mutant plants, SAR markers: (i) <italic> PR-1 </italic> was induced following INA treatment and (ii) <italic> BGL2 </italic> following infection with <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. tomato ( <italic> Pst </italic> ), and SAR was induced following <italic> Pst </italic> infection. Of the five cysteine residues, Cys <sup> 82 </sup> , Cys <sup> 150 </sup> , Cys <sup> 155 </sup> , Cys <sup> 160 </sup> , and Cys <sup> 216 </sup> involved in oligomer-monomer transition in NPR1, Cys <sup> 216 </sup> in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2</span> proteins was substituted to Ser and Leu, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						The development of SAR is associated with the induction of pathogenesis related ( <italic> PR </italic> ) gene expression [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. Increases in the endogenous SA levels in the pathogen-inoculated plants coincide with the increased levels of the <italic> PR </italic> gene expression and enhanced disease <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Transgenic plants expressing the bacterial salicylate hydroxylase ( <italic> <span class="gene">nahG</span> </italic> ) gene cannot accumulate SA and fail to express SAR development [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ]. The <italic> PR </italic> genes, known as the SAR markers, have been identified from several plant species including tobacco and Arabidopsis [ <xref ref-type="bibr" rid="B4"> 4 </xref> ]. A soybean <italic> PR1 </italic> homolog, <italic> <span class="gene">GmPR1</span> </italic> is induced by both SA treatment and infection of soybean leaves with <italic> soybean mosaic virus </italic> (SMV) [ <xref ref-type="bibr" rid="B12"> 12 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In the United States, soybean suffers annual <span class="trait">yield</span> losses valued at more than 2.6 billion dollars from various pathogenic diseases [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. SAR in soybean was first reported following infection with <italic> Colletotrichum trancatum </italic> that causes anthracnose disease [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. A significant reduction in lesion sizes following <italic> C. trancatum </italic> infection was noted in epicotyls, when cotyledons were pre-injected with <italic> C. trancatum </italic> and <italic> C. lagenarium </italic> spore suspensions [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. We investigated if SAR in soybean is regulated by a pathway, similar to the one characterized in Arabidopsis. We have shown that there are two orthologous <italic> NPR1 </italic> copies in soybean. Non conservation of the Arabidopsis Cys <sup> 216 </sup> residue in <span class="gene">GmNPR1s</span> suggests that either conserved Cys <sup> 82 </sup> , Cys <sup> 150 </sup> , Cys <sup> 155 </sup> , Cys <sup> 160 </sup> residues are sufficient for <span class="gene">GmNPR1s'</span> monomerization or some other soybean cysteine residue(s) complements the Arabidopsis Cys <sup> 216 </sup> function.
					</p>
				
					<p>&nbsp;&nbsp;
						Although it was demonstrated earlier that <italic> <span class="gene">GmPR1</span> </italic> , a SAR marker, was induced in response to SMV infection, it has not been shown if <italic> <span class="gene">GmPR1</span> </italic> is systemically induced in non-inoculated systemic leaves [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. For SAR, induction of <italic> <span class="gene">GmPR1</span> </italic> gene, a SAR marker, is needed in non-inoculated systemic tissue to provide <span class="trait">resistance</span> against secondary infection. To determine if pathogenic infection can also lead to <italic> <span class="gene">GmPR1</span> </italic> expression in systemic tissues, hypocotyls of young soybean seedlings were inoculated with an avirulent <italic> P. sojae </italic> race and <italic> <span class="gene">GmPR1</span> </italic> expression was monitored at the site of infection and in non-inoculated systemic leaves. Induction of <italic> <span class="gene">GmPR1</span> </italic> at infection sites was observed as early as on day 1 with a peak on day 2 post inoculation; and thereafter, induction continued until day 9 following inoculation (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). In the systemic leaves, induction of <italic> <span class="gene">GmPR1</span> </italic> was clearly observed by day 9 following inoculation (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). No systemic induction of <italic> <span class="gene">GmPR1</span> </italic> was observed when only agar medium with no <italic> P. sojae </italic> mycelia was used to inoculate the wounded hypocotyls (data not shown). These results suggested that SAR pathway is active in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine if <span class="gene">GmNPR1</span> proteins can induce SAR in non-inoculated leaves, we infected one transformant containing either <italic> <span class="gene">GmNPR1-1</span> </italic> or <italic> <span class="gene">GmNPR1-2</span> </italic> with the bacterial pathogen <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. tomato ( <italic> Pst </italic> ) DC3000 containing <italic> AvrRpt2 </italic> . Three days after inoculation, we inoculated two young non-inoculated leaves with a virulent strain, <italic> Pst </italic> DC3000 and extent of SAR induction in these leaves was determined. Arabidopsis transformants carrying either of the <italic> <span class="gene">GmNPR1</span> </italic> genes showed induction of SAR in response to infection with the avirulent strain, <italic> Pst </italic> DC3000 carrying <italic> AvrRpt2 </italic> . There was about 9.5-fold reduction in the number of colony forming units (cfu) of <italic> Pst </italic> in <italic> <span class="gene">GmNPR1-1</span> </italic> -complemented plants, when preinoculated with the avirulent strain as compared to the MgCl <sub> 2 </sub> control (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ). <italic> <span class="gene">GmNPR1-2</span> </italic> , however, resulted in 3.3-fold reduction in numbers of cfu in transformants, preinoculated with the avirulent strain as compared to that in the control (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ). In the avirulent <italic> Pst </italic> strain infected Columbia, <span class="gene">GmNPR1-1-</span> and <span class="gene">GmNPR1-2-complemented</span> <italic> npr1-1 </italic> plants, significant reduction in cfu of <italic> Pst </italic> was observed when compared to their corresponding MgCl <sub> 2 </sub> controls (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean suffers estimated annual <span class="trait">yield</span> loss valued at 2.6 billion dollars from attack of various pathogens [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Broad-spectrum SAR has the potentiality to reduce the crop losses from diverse pathogens in soybean. Here we have presented molecular evidence suggesting that the SAR pathway is conserved in soybean. We have isolated soybean genes encoding the SAR regulatory protein, NPR1. Results from Southern blot analysis, gene cloning experiments and soybean genome analyses strongly suggested that there are two <italic> NPR1 </italic> -like sequences in soybean. We have also shown that in soybean, SAR marker <italic> <span class="gene">GmPR1</span> </italic> is induced in response to both (i) SAR inducer, INA and (ii) <italic> P. sojae </italic> infection (Figures <xref ref-type="fig" rid="F1"> 1 </xref> and <xref ref-type="fig" rid="F2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We have shown that SAR marker, <italic> <span class="gene">GmPR1</span> </italic> is expressed in response to both INA treatment and <italic> P. sojae </italic> infection in soybean, and soybean <italic> NPR1 </italic> orthologues are functional. In soybean, it has recently been demonstrated that RAR1 and SGT1 are required for SAR and are functional [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. Together, these data strongly suggest that SAR is induced in soybean. Therefore, overexpression of <italic> <span class="gene">GmNPR1</span> </italic> genes will most likely enhance broad-spectrum <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Complementation analyses in the Arabidopsis <italic> npr1-1 </italic> mutant suggested that homoeologous <italic> <span class="gene">GmNPR1-1</span> </italic> and <italic> <span class="gene">GmNPR1-2</span> </italic> genes are orthologous to Arabidopsis <italic> NPR1 </italic> . Therefore, SAR pathway in soybean is most likely regulated by <italic> <span class="gene">GmNPR1</span> </italic> genes. Substitution of essential Cys <sup> 216 </sup> residue for oligomer-monomer transition of Arabidopsis NPR1 with Ser and Leu residues in <span class="gene">GmNPR1-1</span> and <span class="gene">GmNPR1-2,</span> respectively suggested that there may be differences between the regulatory mechanisms of <span class="gene">GmNPR1</span> and Arabidopsis NPR proteins. Soybean plants showed expression of the SAR marker <italic> PR1 </italic> gene and SAR following infection, and carry functional <italic> <span class="gene">GmNPR1</span> </italic> genes suggesting that overexpression of <italic> <span class="gene">GmNPR1s</span> </italic> in transgenic soybean plants may enhance <span class="trait">resistance</span> against many pathogens.
					</p>
				
					<p>&nbsp;&nbsp;
						Ecotype Columbia, <italic> npr1-1 </italic> mutant, and <italic> npr1-1 </italic> mutant transformed with either <italic> <span class="gene">GmNPR1-1</span> </italic> or <italic> <span class="gene">GmNPR1-2</span> </italic> were selected for investigating the complementation of NPR1 function for SAR activity in the <italic> npr1-1 </italic> mutant background. Arabidopsis plants were sown in Sunshine LC1 mix (Sun Grow Horticulture Inc., Bellevue, WA) under 9 h light and 15 h dark regimen at 22&#x000b0;C with 55% humidity. After two weeks, seedlings were transplanted. Four weeks following planting fully developed two leaves (leaf number 3 and 4) were inoculated with 10 mM MgCl <sub> 2 </sub> or an avirulent strain <italic> Pst DC3000 </italic> containing <italic> AvrRpt2 </italic> . Leaves were inoculated with a syringe containing bacterial cells grown for 48 h in NYG medium containing rifampicin (50 &#x003bc;g/ml) and kanamycin (25 &#x003bc;g/ml) [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. Bacterial cells for inoculation were collected by centrifugation and then resuspended in 10 mM MgCl <sub> 2 </sub> to an optical density 0.2 at A <sub> 600 </sub> , which corresponds to ~10 <sup> 8 </sup> cfu/ml. Bacterial suspensions were diluted to 10 <sup> 7 </sup> cfu/ml in 10 mM MgCl <sub> 2 </sub> [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Two leaves per plant were infiltrated with this bacterial suspension using a 1-ml syringe. About 40 &#x003bc;l bacterial suspension (10 <sup> 7 </sup> cfu/ml) was infiltrated in each leaf. Three days after inoculation, two younger systemic leaves (leaf number 5 and 6) were inoculated with the virulent strain <italic> Pst </italic> DC3000. <italic> Pst DC3000 </italic> contains empty vector with the kanamycin <span class="trait">resistance</span> gene. Bacterial cells for inoculation were collected by centrifugation and then resuspended in 10 mM MgCl <sub> 2 </sub> to an optical density 0.001 at A <sub> 600 </sub> . After incubation for 3 days at 22&#x000b0;C, the inoculated leaves were harvested. Same size leaf disc was taken from each leaf and was washed twice in sterile water and homogenized in 1 ml 0.9% NaCl. The samples were vortexed and serial dilutions prepared in 0.9% NaCl were plated on NYGA solid medium containing rifampicin (50 &#x003bc;g/ml) and kanamycin (25 &#x003bc;g/ml), and viable colonies were counted after 2 d of growth at 28&#x000b0;C. The study was conducted with four biological replications. Two factor ANOVA was used to compare different treatments. To determine which of the eight treatments differ from each other, Fisher's least significant difference (LSD) comparisons were performed at P value of 0.05.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:An oleic acid-mediated pathway induces constitutive defense signaling and enhanced <span class="trait">resistance</span> to multiple pathogens in soybean.</h2>
			<h3>PMID:18393616  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kachroo A(1), Fu DQ, Havens W, Navarre D, Kachroo P, Ghabrial SA.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Stearoyl-acyl carrier protein-desaturase (SACPD)-catalyzed synthesis of oleic acid (18:1) is an essential step in fatty acid biosynthesis. Arabidopsis mutants (ssi2) with reduced <span class="gene">SACPD</span> activity accumulate salicylic acid (SA) and exhibit enhanced <span class="trait">resistance</span> to multiple pathogens. We show that reduced levels of 18:1 induce similar defense-related phenotypes in soybean. A Bean pod mottle virus (BPMV)-based vector was employed to effectively silence soybean SACPDs. The silenced plants contained reduced 18:1 and increased stearic acid, developed spontaneous cell death lesions, increased SA accumulation, and constitutively expressed pathogenesis-related genes. These plants also expressed elevated levels of resistance-like genes and showed <span class="trait">resistance</span> to bacterial and oomycete pathogens. Exogenous application of glycerol induced similar phenotypes, mimicking the effect of silencing SACPDs in healthy soybean plants. Overexpression of a soybean <span class="gene">SACPD</span> increased 18:1 levels in ssi2 but not in wild-type Arabidopsis plants, suggesting that the soybean enzyme was under feedback regulation similar to that of the Arabidopsis isozymes. These results suggest that soybean and Arabidopsis respond similarly to 18:1-derived cues by inducing a novel broad-spectrum resistance-conferring pathway, even though they differ significantly in their lipid biosynthetic pathways. We also demonstrate the efficacy of BPMV-induced gene silencing as a tool for functional studies in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome changes in the phenylpropanoid pathway of <italic> Glycine max </italic> in response to <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> infection</h2>
			<h3>PMID:17083738</h3>
				

				
					<h3>PMCID:1636052</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> g-zabala@uiuc.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Zou </surname>  <given-names> Jijun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jijun.zou@pioneer.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Tuteja </surname>  <given-names> Jigyasa </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> tuteja@uiuc.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Gonzalez </surname>  <given-names> Delkin O </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dogonzal@uiuc.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Clough </surname>  <given-names> Steven J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> sjclough@uiuc.edu </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 11 </month>  <year> 2006 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Reports of plant molecular responses to pathogenic infections have pinpointed increases in activity of several genes of the phenylpropanoid pathway leading to the synthesis of lignin and flavonoids. The majority of those findings were derived from single gene studies and more recently from several global gene expression analyses. We undertook a global transcriptional analysis focused on the response of genes of the multiple branches of the phenylpropanoid pathway to infection by the <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea with or without the avirulence gene avrB to characterize more broadly the contribution of the multiple branches of the pathway to the <span class="trait">resistance</span> response in soybean. Transcript abundance in leaves was determined from analysis of soybean cDNA microarray data and hybridizations to RNA blots with specific gene probes. RESULTS: The majority of the genes surveyed presented patterns of increased transcript accumulation. Some increased rapidly, 2 and 4 hours after inoculation, while others started to accumulate slowly by 8-12 hours. In contrast, transcripts of a few genes decreased in abundance 2 hours post inoculation. Most interestingly was the opposite temporal fluctuation in transcript abundance between early responsive genes in defense (CHS and IFS1) and F3H, the gene encoding a pivotal enzyme in the synthesis of anthocyanins, proanthocyanidins and flavonols. <span class="gene">F3H</span> transcripts decreased rapidly 2 hours post inoculation and increased during periods when <span class="gene">CHS</span> and <span class="gene">IFS</span> transcripts decreased. It was also determined that all but one (CHS4) family member genes (CHS1, CHS2, CHS3, CHS5, <span class="gene">CHS6</span> and CHS7/8) accumulated higher transcript levels during the defense response provoked by the avirulent pathogen challenge. CONCLUSION: Based on the mRNA profiles, these results show the strong bias that soybean has towards increasing the synthesis of isoflavonoid phytoalexins concomitant with the down regulation of genes required for the synthesis of anthocyanins and proanthocyanins. Although proanthocyanins are known to be toxic compounds, the cells in the soybean leaves seem to be programmed to prioritize the synthesis and accumulation of isoflavonoid and pterocarpan phytoalexins during the <span class="trait">resistance</span> response. It was known that <span class="gene">CHS</span> transcripts accumulate in great abundance rapidly after inoculation of the soybean plants but our results have demonstrated that all but one (CHS4) member of the gene family member genes accumulated higher transcript levels during the defense response.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Based on the mRNA profiles, these results show the strong bias that soybean has towards increasing the synthesis of isoflavonoid phytoalexins concomitant with the down regulation of genes required for the synthesis of anthocyanins and proanthocyanins. Although proanthocyanins are known to be toxic compounds, the cells in the soybean leaves seem to be programmed to prioritize the synthesis and accumulation of isoflavonoid and pterocarpan phytoalexins during the <span class="trait">resistance</span> response. It was known that <span class="gene">CHS</span> transcripts accumulate in great abundance rapidly after inoculation of the soybean plants but our results have demonstrated that all but one ( <italic> CHS4 </italic> ) member of the gene family member genes accumulated higher transcript levels during the defense response.
					</p>
				
					<p>&nbsp;&nbsp;
						Consequently, many secondary metabolites derived from multiple branches of the phenylpropanoid pathway, including lignins, isoflavonoid-phytoalexins, other phenolic compounds and SA are instrumental in the plant's ability to mount successful defenses to invading pathogens (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). Most studies of the phenylpropanoid pathway to date have investigated the molecular response of individual genes of the pathway. No major systematic or global analysis focused on the many genes from the multiple branches of the pathway (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) has been reported. Although general, global transcriptome changes during defense responses of various plants ( <italic> Arabidopsis </italic> , tomato, <italic> Medicago truncatula </italic> , soybean) to several pathogens ( <italic> <span class="trait">Pseudomonas</span> spp </italic> , <italic> Alternaria brassicicola </italic> , <italic> Xanthomonas </italic> , <italic> Phytophthora </italic> ) have been examined [ <xref ref-type="bibr" rid="B7"> 7 </xref> - <xref ref-type="bibr" rid="B11"> 11 </xref> ], none of the analyses have thoroughly mined the data with specific emphasis on the overall response of the phenylpropanoid pathway genes. Nevertheless, those global studies have shown several groups of up regulated ESTs representing three main branches of the phenylpropanoid pathway and it appears that all plants examined respond to infection with the induction of phenylalanine ammonia-lyase ( <italic> <span class="gene">PAL</span> </italic> ) coumarate CoA-ligase ( <italic> <span class="gene">C4H</span> </italic> ) and cynamyl alcohol dehydrogenase ( <italic> CAD </italic> ) genes. Chalcone synthase (CHS) a central enzyme in the pathway is consistently induced in all plants examined with the exception of <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. In addition, four isoflavonoid pathway ESTs were up-regulated in <italic> Medicago truncatula </italic> and soybean [ <xref ref-type="bibr" rid="B9"> 9 </xref> , <xref ref-type="bibr" rid="B10"> 10 </xref> ] and four putative anthocyanin pathway ESTs in tomato [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Here we report our findings on the levels of transcript abundance of 19 phenylpropanoid pathway genes and the identification of stress-responsive <italic> <span class="gene">CHS</span> </italic> gene family member(s) in leaves of soybean (cultivar Williams 82) challenged with <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> glycinea </italic> with or without the <italic> avrB </italic> gene. Transcript abundance patterns obtained using RNA from infected leaves in hybridizations to soybean cDNA microarrays [ <xref ref-type="bibr" rid="B11"> 11 </xref> ] were, supported and extended further by single gene RNA gel blots for five genes of key phenylpropanoid pathway enzymes (CAD, CHS, IFS, <span class="gene">F3'H</span> and F3H). Real time quantitative RT-PCR was also used to measure the relative transcript levels of the individual <italic> <span class="gene">CHS</span> </italic> gene family members in leaf tissues 8 hrs after inoculation.
					</p>
				
					<p>&nbsp;&nbsp;
						A delayed induction of <italic> <span class="gene">CHS</span> </italic> genes with transcript accumulation that peaked at about 4 hrs after fungal elicitor addition, compared to the more rapid induction of a <italic> CAD </italic> gene showing maximum transcript accumulation at 2 hrs, had been observed also in <italic> Phaseolus vulgaris </italic> cell cultures [ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ]. The faster induction of the <italic> CAD </italic> gene seems to be due to a transient stress response to the vacuum infiltration <italic> per se </italic> as was described earlier (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). On the other hand, we observed a difference in the sustained high levels of <span class="gene">CHS</span> transcript accumulation up to 53 hrs when compared to what had been observed in alfalfa leaves infected with <italic> P. <span class="trait">syringae</span> </italic> pv. <italic> pisi </italic> , where the levels of <span class="gene">CHS</span> transcripts peaked at 6 hr and declined rapidly to about 20% of peak value by 48 hr post inoculation [ <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript levels of some of these genes have been examined independently one at a time and more recently in global microarray analysis of <italic> Arabidopsis </italic> , tomato, <italic> Medicago truncatula </italic> and soybean plants responding to bacterial and fungal infections. These studies have provided some hints as to which branches of the phenylpropanoid pathway may be activated during the <span class="trait">resistance</span> response in those and other plants [ <xref ref-type="bibr" rid="B7"> 7 </xref> - <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Based on the up-regulation of a few ESTs reported from those microarray studies it appears that the lignin/suberin branch of the phenylpropanoid pathway is up-regulated in all the plants examined but that branches downstream of <span class="gene">CHS</span> (flavonoids and isoflavonoids) appear not to be induced in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. In contrast, <span class="gene">CHS</span> and anthocyanin pathway ESTs were up-regulated in tomato plants [ <xref ref-type="bibr" rid="B8"> 8 </xref> ] and isoflavonoid ESTs in <italic> Medicago truncatula </italic> and soybean [ <xref ref-type="bibr" rid="B9"> 9 </xref> - <xref ref-type="bibr" rid="B11"> 11 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Immediately following CAD was the induction of the isoflavonoid branch of the pathway as reflected by higher accumulation of CHS, CHR, CHI, IFS, IOMT, and IFR transcripts 4 hrs post inoculation (Figure <xref ref-type="fig" rid="F8"> 8 </xref> and Table <xref ref-type="table" rid="T1"> 1 </xref> ). The induction of the isoflavonoid pathway was further enhanced during the <span class="trait">resistance</span> response elicited by the <italic> Psg-avrB </italic> infection as compared to the induction in response to the virulent strain (Figures <xref ref-type="fig" rid="F3"> 3 </xref> and <xref ref-type="fig" rid="F5"> 5 </xref> and Table <xref ref-type="table" rid="T1"> 1 </xref> ). These results support previous observations enlisting <italic> CAD </italic> , <italic> CH </italic> S and <italic> <span class="gene">IFS</span> </italic> as early responders [ <xref ref-type="bibr" rid="B9"> 9 </xref> - <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> - <xref ref-type="bibr" rid="B48"> 48 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A novel finding was the more delayed increase in <span class="gene">F3'H</span> transcripts starting at approximately 12 hr after infection with <italic> Psg-avrB </italic> (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ) and clearly distinct from the kinetics of transcript accumulation observed upon infection with the virulent <italic> Psg </italic> (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). Based on this finding we could speculate that the flavones branch of the flavonoid pathway also participates in the cascade of reactions elicited during defense, although at a later time relative to the isoflavone/phytoalexin branch of the pathway. <span class="gene">F3'H</span> hydroxylates naringenin to eriodictyol a substrate for the synthesis of flavones, flavonols, cyaniding anthocyanidins and proanthocyanidins (Figure <xref ref-type="fig" rid="F1"> 1 </xref> , [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]). Since transcripts of enzymes leading to the synthesis of flavonols, anthocyanidins and proanthocyanidins decreased in abundance in the course of the <span class="trait">resistance</span> response, the increase in <span class="gene">F3'H</span> transcripts during this same time suggest a potential increase in the synthesis of flavones (i.e. luteolin) and therefore an involvement of these secondary metabolites 8 hrs into the defense response. No soybean or any other plant EST has been annotated as flavone synthase 1 or 2 (FS1 and FS2) and it is not currently possible to determine if transcripts from these genes accumulate in the same fashion as <span class="gene">F3'H</span> to further support an involvement of flavones in defense.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, three other flavonoid branches leading to the synthesis of flavonols, anthocyanins, and proanthocyanidins were slowed down judging by the lower amounts of transcripts hybridizing to the F3H, DFR, LDOX and FLS cDNAs as early as 2 hrs after infection (Figure <xref ref-type="fig" rid="F8"> 8 </xref> and Table <xref ref-type="table" rid="T1"> 1 </xref> ). The suppression of these branches of the flavonoid pathway seems to be lifted somehow in the soybean plant response to infection with the virulent <italic> Psg </italic> . Transcripts of the <italic> <span class="gene">F3H</span> </italic> gene encoding the key enzyme feeding all three branches (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) accumulated slightly higher amounts in plants infected with <italic> Psg </italic> lacking <italic> avr-B </italic> (Figure <xref ref-type="fig" rid="F7"> 7 </xref> ). In tomato, microarray data from a one time point (8 hr) after infection with <italic> P. <span class="trait">syringae</span> </italic> pv. <italic> tomato </italic> has shown up-regulation of four putative anthocyanin ESTs but no activation of isoflavonoid ESTs [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Even though there have been previous reports describing increases in <span class="gene">CHS</span> and <span class="gene">IFS</span> transcripts in tissues of infected soybean plants, the cyclic nature of the increments, possibly associated with the diurnal cycle, observed in our RNA gel blot analysis, had not been documented before (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ). Likewise, the increases in <span class="gene">F3H</span> transcripts at the times when CSH and <span class="gene">IFS</span> transcripts decrease in abundance is a novel finding showing that the synthesis of <span class="gene">F3H</span> transcripts is opposite correlated to the activity of the <italic> <span class="gene">IFS</span> </italic> genes during the pathogen stress response. This observation allows us to speculate that the biosynthesis of anthocyanins, proanthocyanidins and flavonols is relegated to second place compared to that of the isoflavones/isoflavanones-phytoalexins during the soybean <span class="trait">resistance</span> response to <italic> Psg-AvrB </italic> infection. We further infer from the data showing that <span class="gene">F3H</span> transcript levels are reciprocally correlated to those of <span class="gene">CHS</span> and IFS, that the entrance into the flavonol/anthocyanin/proanthocyanin branch of the pathway is possibly restricted by the low levels of the <span class="gene">F3H</span> and transcripts of genes downstream of F3H, thus giving preference to the isoflavone/phytoalexin branch of the pathway during the response to infection. The critical nature of partitioning the naringenin substrate between <span class="gene">F3H</span> and <span class="gene">IFS</span> enzymes has been noted previously in that <span class="gene">F3H</span> must be non-functional in order to permit accumulation of isoflavones in <italic> Arabidopsis </italic> plants transformed with an <italic> <span class="gene">IFS</span> </italic> gene [ <xref ref-type="bibr" rid="B15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The transcriptional analysis of genes from the multiple branches of the phenylpropanoid pathway has shown that in soybean synthesis of lignin/suberin is one of the early responses to <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> glycinea </italic> infection as deduced by the high accumulation of CAD transcripts 2 hrs after <italic> Psg-avrB </italic> inoculation. Immediately following the CAD response was the induction of the isoflavonoid biosynthetic pathway judging by the high accumulation of CHS, CHR, CHI, IFS, IOMT, and IFR transcripts 4 hrs post inoculation (Figure <xref ref-type="fig" rid="F8"> 8 </xref> and Table <xref ref-type="table" rid="T1"> 1 </xref> ). The data also allowed us to predict that the synthesis of flavones may participate in the R gene-mediated defensive response, as suggested by the higher levels of <span class="gene">F3'H</span> transcripts accumulated 24 hours post infiltration. In contrast, transcripts of genes involved in the synthesis of flavonols, anthocyanins and proanthocyanidins, also pathogen deterring compounds, seem to be down regulated during the periods of time when the genes involved in the synthesis of isoflavonoids are up-regulated. However, it appears that at times when the activity of <italic> <span class="gene">IFS</span> </italic> genes decrease, synthesis of flavonols, anthocyanins and proanthocyanins may resume based on the concomitant increases of <span class="gene">F3H</span> transcripts which encode an enzyme required for biosynthesis of those secondary metabolites. Lastly, we conclude that the large transcript increases measured for <italic> <span class="gene">CHS</span> </italic> , the gene encoding the central enzyme of the phenylpropanoid pathway, is brought about by the additive up-regulation of seven <italic> <span class="gene">CHS</span> </italic> family member genes.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Cynnamyl alcohol dehydrogenase (CAD) RNA (1.4 kb) profile </bold> . Measurements made at 8 intervals through a 53 hr period in response to infection by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv <italic> glycinea </italic> with <italic> avrB </italic> ( <italic> avrB </italic> ) or without (vir) (First and second panels respectively). Third panel is the CAD RNA profile in response to infiltration with control medium (MgCl <sub> 2 </sub> ). Dark background sub-panels are the 25 S ribosomal RNAs from corresponding ethidium bromide-stained gels prior to membrane transfer and shown to compare sample loading. <span class="gene">Gm-r1030-4089</span> soybean cDNA clone was used as probe.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Chalcone synthase (CHS) RNA (1.4 kb) profile </bold> . Measurements made at 8 intervals through a 53 hr period in response to infection by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv <italic> glycinea </italic> with <italic> avrB </italic> ( <italic> avrB </italic> ) or without (vir) (First and second panels respectively). Third panel is the <span class="gene">CHS</span> RNA profile in response to infiltration with control medium (MgCl <sub> 2 </sub> ). Dark background sub-panels are the 25 S ribosomal RNAs from corresponding ethidium bromide-stained gels prior to membrane transfer and shown to compare sample loading. The soybean <span class="gene">CHS</span> genomic clone (pC2H2.0, <span class="gene">Gm-b10BB-5)</span> was used as probe.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Isoflavone synthase (IFS) RNA (1.7 kb) profile </bold> . Measurements made at 8 intervals through a 53 hr period in response to infection by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv <italic> glycinea </italic> with <italic> avrB </italic> ( <italic> avrB </italic> ) or without (vir) (First and second panels respectively). Dark background sub-panels are the 25 S ribosomal RNAs from corresponding ethidium bromide-stained gels prior to membrane transfer and shown to compare sample loading. The <span class="gene">Gm-c1059-264</span> soybean cDNA clone was used as probe.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Flavonoid 3'-hydroxylase (F3'H) RNA (1.8 kb) profile </bold> . Measurements made at 8 intervals through a 53 hr period in response to infection by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv <italic> glycinea </italic> with <italic> avrB </italic> ( <italic> avrB </italic> ) or without (vir) (First and second panels respectively). Third panel is the <span class="gene">F3'H</span> RNA profile in response to infiltration with control medium (MgCl <sub> 2 </sub> ). Dark background sub-panels are the 25 S ribosomal RNAs from corresponding ethidium bromide-stained gels prior to membrane transfer and shown to compare sample loading. <span class="gene">Gm-c1019-10961</span> soybean cDNA clone was used as probe.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Flavanone 3-hydroxylase (F3H) RNA (1.4 kb) profile </bold> . Measurements made at 8 intervals through a 53 hr period in response to infection by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv <italic> glycinea </italic> with <italic> avrB </italic> ( <italic> avrB </italic> ) or without (vir) (First and second panels respectively). Dark background sub-panels are the 25 S ribosomal RNAs from corresponding ethidium bromide-stained gels prior to membrane transfer and shown to compare sample loading. <span class="gene">Gm-c1012-683</span> soybean cDNA clone was used as probe.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Alignment of CAD, CHS, IFS, <span class="gene">F3'H</span> and F3H, RNA gel blot profiles </bold> . The alignment of the RNA profiles from previous five figures revealed the opposite fluctuation in RNA accumulation of the <span class="gene">F3H</span> RNAs during the plant's response to <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> glycinea </italic> with <italic> avrB </italic> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression profile of the <italic> <span class="gene">CHS</span> </italic> gene family members as determined by TaqMan <sup> &#x000ae; </sup> RT-PCR </bold> . Total RNA from leaves of <italic> Glycine max </italic> cv. Williams infiltrated with MgCl <sub> 2 </sub> (gray boxes) or the avirulent (black boxes) strains of <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> glycinea </italic> was isolated after eight hours of infection, reverse transcribed and subjected to real-time PCR. Relative amounts were calculated and normalized with respect to <italic> PEPC </italic> transcript levels (=100%). Data shown represents mean values obtained from three independent amplification reactions and the error bars indicate the S.E. (standard error) of the mean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning of a soybean class III beta-1,3-glucanase gene that is regulated both developmentally and in response to pathogen infection.</h2>
			<h3>PMID:10725560</h3>
				

				
			<p>
				作者：Cheong YH(1), Kim1 CY, Chun HJ, Moon BC, Park HC, Kim JK, Lee S, Han C, Lee SY, Cho MJ.
				<br/> 
				年份：2000 May 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We isolated and characterized a soybean gene (SGN1) encoding a basic beta-1,3-glucanase that is a plant class III isoform of beta-1,3-glucanase. The deduced amino acid sequence of the <span class="gene">SGN1</span> gene is similar to that of the PR-Q'b gene, the basic class III beta-1,3-glucanase of tomato. Based on RNA blot hybridization, <span class="gene">SGN1</span> gene expression was detected in all tissues of 4-day old seedlings, but it was present only in root tissue of 30-day old plants. GUS expression analysis carried out in transgenic tobacco plants harboring a SGN1::GUS reporter gene revealed the same expression pattern. Furthermore, the expression of <span class="gene">SGN1</span> was strongly induced by a variety of defense-related signals, such as treatment with H(2)O(2), wounding, or treatment with fungal elicitor prepared from Phytophthora spp as well as inoculation with <span class="trait">Pseudomonas</span> <span class="trait">syringae.</span> However, the expression level of <span class="gene">SGN1</span> was hardly induced with jasmonate, ethephon and salicylate. Overall the results suggest that the <span class="gene">SGN1</span> may play a role in both plant development and plant defense against pathogen attack.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rapid stimulation of a soybean protein-serine kinase that phosphorylates a novel bZIP DNA-binding protein, G/HBF-1, during the induction of early transcription-dependent defenses.</h2>
			<h3>PMID:9049302  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dröge-Laser W(1), Kaiser A, Lindsay WP, Halkier BA, Loake GJ, Doerner P, Dixon RA, Lamb C.
				<br/> 
				年份：1997 Feb 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The G-box (CACGTG) and H-box (CCTACC) cis elements function in the activation of phenylpropanoid biosynthetic genes involved in the elaboration of lignin precursors, phytoalexins and the secondary signal salicylic acid as early responses to pathogen attack. We have isolated a soybean cDNA encoding a novel bZIP protein, G/HBF-1, which binds to both the G-box and adjacent H-box in the proximal region of the chalcone synthase chs15 promoter. While <span class="gene">G/HBF-1</span> transcript and protein levels do not increase during the induction of phenylpropanoid biosynthetic genes, <span class="gene">G/HBF-1</span> is phosphorylated rapidly in elicited soybean cells, almost exclusively on serine residues. Using recombinant <span class="gene">G/HBF-1</span> as a substrate, we identified a cytosolic protein-serine kinase that is rapidly and transiently stimulated in cells elicited with either glutathione or an avirulent strain of the soybean pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea. Phosphorylation of <span class="gene">G/HBF-1</span> in vitro enhances binding to the chs15 promoter and we conclude that stimulation of <span class="gene">G/HBF-1</span> kinase activity and <span class="gene">G/HBF-1</span> phosphorylation are terminal events in a signal pathway for activation of early transcription-dependent plant defense responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increase of chalcone synthase mRNA in pathogen-inoculated soybeans with race-specific <span class="trait">resistance</span> is different in leaves and roots.</h2>
			<h3>PMID:16667156</h3>
				

				
			<p>
				作者：Dhawale S(1), Souciet G, Kuhn DN.
				<br/> 
				年份：1989 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybeans (Glycine max [L.] Merr.) respond to pathogens by producing isoflavonoid-derived phytoalexins. Chalcone synthase (CHS) is the first enzyme of the flavonoid/isoflavonoid biosynthetic pathway. We investigated changes in the steady state levels of <span class="gene">CHS</span> mRNA and other specific mRNAs at increasing times after inoculation in two different race-specific interactions, one between leaves and the bacterium <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv glycinea (Psg), and one between roots and the fungus, Phytophthora megasperma f. sp. glycinea (Pmg). The amount of <span class="gene">CHS</span> mRNA increases significantly in soybean leaves inoculated with an avirulent race of Psg but not with a virulent race or water. In contrast, the increase in <span class="gene">CHS</span> mRNA is similar in roots inoculated with zoospores of either an avirulent or virulent race of Pmg. <span class="gene">CHS</span> mRNA increases significantly in pathogen inoculated roots but not in uninoculated controls. Hydroxyproline-rich glycoprotein (HRGP) has been observed by others to increase in wounded or pathogen-inoculated plants. We report here that HRGP mRNA levels are greater in roots inoculated with an avirulent Pmg race than with a virulent race, but inoculation with either race causes a significant increase in HRGP mRNA with respect to controls. Calmodulin or ubiquitin mRNA do not increase in either uninoculated or inoculated roots and leaves. The possibility that race-specific <span class="trait">resistance</span> in soybeans is expressed differently in different organs of the plant is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:LIK1, a CERK1-interacting kinase, regulates plant immune responses in Arabidopsis.</h2>
			<h3>PMID:25036661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Le MH(1), Cao Y(1), Zhang XC(1), Stacey G(1).
				<br/> 
				年份：2014 Jul 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chitin, an integral component of the fungal cell wall, is one of the best-studied microbe-associated molecular patterns. Previous work identified a LysM receptor-like kinase (LysM-RLK1/CERK1) as the primary chitin receptor in Arabidopsis. In order to identify proteins that interact with CERK1, we conducted a yeast two-hybrid screen using the intracellular kinase domain of CERK1 as the bait. This screen identified 54 putative CERK1-interactors. Screening mutants defective in 43 of these interacting proteins identified only two, a calmodulin like protein (At3g10190) and a leucine-rich repeat receptor like kinase (At3g14840), which differed in their response to pathogen challenge. In the present work, we focused on characterizing the <span class="gene">LRR-RLK</span> gene where mutations altered responses to chitin elicitation. This <span class="gene">LRR-RLK</span> was named LysM RLK1-interacting kinase 1 (LIK1). The interaction between CERK1 and LIK1 was confirmed by co-immunoprecipitation using protoplasts and transgenic plants. In vitro experiments showed that LIK1 was directly phosphorylated by CERK1. In vivo phosphorylation assays showed that Col-0 wild-type plants have more phosphorylated LIK1 than cerk1 mutant plants, suggesting that LIK1 may be directly phosphorylated by CERK1. Lik1 mutant plants showed an enhanced response to both chitin and flagellin elicitors. In comparison to the wild-type plants, lik1 mutant plants were more resistant to the hemibiotrophic pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae,</span> but more susceptible to the necrotrophic pathogen Sclerotinia sclerotiorum. Consistent with the enhanced susceptibility to necrotrophs, lik1 mutants showed reduced expression of genes involved in jasmonic acid and ethylene signaling pathways. These data suggest that LIK1 directly interacts with CERK1 and regulates MAMP-triggered innate immunity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pseudomonas asturiensis sp. nov., isolated from soybean and weeds.</h2>
			<h3>PMID:23727430  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：González AJ(1), Cleenwerck I, De Vos P, Fernández-Sanz AM.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Five strains of gram negative bacteria, isolated from soybean (LPPA 221(T), 222 and 223) and weeds (LPPA 816 and 1442), were analyzed by a polyphasic approach. The isolates showed variation in their phenotypic traits and were placed in the <span class="trait">Pseudomonas</span> fluorescens lineage, based on 16S rRNA gene sequence phylogeny, as a single but well separated cluster. MLSA analysis based on <span class="gene">gyrB</span> and <span class="gene">rpoD</span> sequences clustered the strains in a single branch in the <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> group, and revealed P. viridiflava as closest relative. DNA-DNA hybridizations showed medium levels of DNA-DNA relatedness with the type strain of P. viridiflava (50%) and lower levels (&lt;32%) with other type strains of the P. <span class="trait">syringae</span> group, supporting classification within a novel species of the genus <span class="trait">Pseudomonas.</span> The strains can be distinguished from species of the P. <span class="trait">syringae</span> group by the fatty acid C17:0 cyclo that is present in a low amount (2.5%) and from P. viridiflava by their inability to assimilate d-tartrate and d-sorbitol, and by the formation of red colonies on TTC medium. For this new species, the name <span class="trait">Pseudomonas</span> asturiensis sp. nov. is proposed. The type strain is LPPA 221(T) (=LMG 26898(T)=CECT 8095(T)).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inhibition of phosphoinositide-specific phospholipase C results in the induction of pathogenesis-related genes in soybean.</h2>
			<h3>PMID:15570470  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chou WM(1), Shigaki T, Dammann C, Liu YQ, Bhattacharyya MK.
				<br/> 
				年份：2004 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The inositol 1,4,5-trisphosphate (IP3) content is decreased in soybean cells following infection with <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea (Psg). In this investigation, a differential display approach was applied to isolate soybean genes that are transcriptionally up-regulated by the inhibition of phosphoinositide-specific phospholipase C (PI-PLC) activity and to study if the transcription of those genes is altered following Psg infection. Four genes, transcriptionally activated following treatment with the PI-PLC-specific inhibitor U-73122, were cloned. Three of the four genes were induced following infection with Psg. The transcripts of a hydrolase homologue <span class="gene">(GmHy)</span> were induced in the incompatible but not compatible soybean-Psg interaction. The transcripts of a putative ascorbate oxidase gene <span class="gene">(GmAO)</span> were induced in both compatible and incompatible interactions. <span class="gene">GmHy</span> and <span class="gene">GmAO</span> may represent new classes of pathogenesis-related genes. In addition to these two novel genes, homologues of PR-10 and polygalacturonase inhibitor protein <span class="gene">(GmPR10</span> and <span class="gene">GmPGIP,</span> respectively) were identified. These two genes have previously been reported as pathogenesis-related. Transcripts of <span class="gene">GmPR-10,</span> but not <span class="gene">GmPGIP,</span> were induced in both compatible and incompatible soybean-Psg interactions. Induction of these genes, except for <span class="gene">GmPGIP,</span> following inhibition of PI-PLC by either the U-73122 treatment or bacterial infection suggests that PI-PLC may negatively regulate the expression of defence genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant-bacterium interactions analyzed by proteomics</h2>
			<h3>PMID:23424014</h3>
				
					<h3>PMCID:3573209</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Afroz </surname>  <given-names> Amber </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zahur </surname>  <given-names> Muzna </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zeeshan </surname>  <given-names> Nadia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Komatsu </surname>  <given-names> Setsuko </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flagellin, elongation factor (EF) Tu, peptidoglycan, lipopolysaccharide, and bacterial cold shock proteins are important PAMPS and plant responses induced by them are referred to as &#x0201c;basal&#x0201d; defenses (Newton et al., <xref ref-type="bibr" rid="B72"> 2010 </xref> ). Upon recognition of highly conserved amino terminus of flagellin (flg22), flagellin sensing 2 (FLS2) induces a suite of defense responses, including MAP kinase signaling, transcriptional activation, and deposition of callose, a putative physical barrier at the site of infection (Gomez-Gomez et al., <xref ref-type="bibr" rid="B30"> 1999 </xref> ). EF Tu potent bacterial PAMP in <italic> Arabidopsis </italic> and other members of the Brassicaceae family, serves as an adhesion factor at the bacterial surface, in addition to its primary role in translation (Boller and Felix, <xref ref-type="bibr" rid="B8"> 2009 </xref> ). Asparate oxidase is required for PAMP-triggered RBOHD-dependent (responsible for stomatal closure) ROS burst and stomatal immunity against the <italic> P. <span class="trait">syringae</span> </italic> (Macho et al., <xref ref-type="bibr" rid="B59"> 2012 </xref> ). The LRR receptor kinases, <span class="gene">EF-Tu</span> receptor and FLS2 are PRRs, contributing to disease <span class="trait">resistance</span> against the hemibiotrophic bacterium <italic> P. <span class="trait">syringae</span> </italic> (Roux et al., <xref ref-type="bibr" rid="B76"> 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> X. fastidiosa </italic> , whose genome was fully sequenced leads to a clearer understanding of the biology of phytopathogenic organism at both the genomic and proteome levels (Simpson et al., <xref ref-type="bibr" rid="B82"> 2000 </xref> ). The cellular and secreted protein profiling of pathogenic bacteria <italic> X. fastidiosa </italic> , led to the identification of proteins involved in cellular adhesion systems, proteases, antioxidant, and toxins (Smolka et al., <xref ref-type="bibr" rid="B83"> 2003 </xref> ). These proteins can be the candidates for understanding of molecular mechanism of disease cycle of citrus variegated <span class="trait">chlorosis</span> and Pierce's disease in grapevine caused by <italic> X. fastidiosa </italic> . In non-virulent strain of <italic> E. chrysanthemi </italic> ( <italic> opg </italic> mutant), differential expression of protein is not restricted to the envelope, but affects general metabolism such as membrane lipid composition, protein folding, carbohydrate catabolism, and protein degradation (Bouchart et al., <xref ref-type="bibr" rid="B11"> 2007 </xref> ). The secretome of <italic> X. campestris </italic> pv. <italic> campestris </italic> using 2-DE and matrix-assisted laser desorption/ionization time of flight (MALDI-TOF) <span class="gene">MS</span> resulted in the identification of 87 proteins known to be involved in degradative activities and important for the infection of susceptible plant hosts (Watt et al., <xref ref-type="bibr" rid="B89"> 2005 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The co-regulation of the synthesis of enzymes secreted via the T2SS and T3SS has been reported frequently (Figures <xref ref-type="fig" rid="F1"> 1 </xref> , <xref ref-type="fig" rid="F2"> 2 </xref> ). One reasonable explanation for this co-regulation is based on the observation that substrates that are secreted via the T2SS are not only associated with bacterial vir but can also induce plant defense responses. The concurrent induction of the T2SS and the T3SS probably allows the pathogen to counteract basal plant defense responses that are elicited by proteins released via the T2SS with the effector proteins transported by the T3SS (Jha et al., <xref ref-type="bibr" rid="B39"> 2007 </xref> ). T3SS are used to inject different effector proteins, formerly known as avirulence proteins, directly into the host cell via an extracellular pilus that acts as channel (McCann and Guttman, <xref ref-type="bibr" rid="B67"> 2008 </xref> ). T3SS is required for pathogenicity in host plants and for eliciting a rapid hypersensitive <span class="trait">resistance</span> response in non-host plants, and the genes encoding T3SS were named HR and pathogenicity (hrp) genes (Jacobs et al., <xref ref-type="bibr" rid="B38"> 2012 </xref> ). hrp genes were identified in most Gram-negative plant pathogenic bacteria with the exception of <italic> A. tumefaciens </italic> and <italic> Xylella fastidiosa </italic> (Buttner and Bonas, <xref ref-type="bibr" rid="B14"> 2010 </xref> ). Homologues of the hrp genes were also found in different rhizobia (Jones and Dangl, <xref ref-type="bibr" rid="B44"> 2006 </xref> ). To identify the <span class="gene">SS</span> that is responsible for the transport of distinct vir factors, the secretome of a <span class="gene">SS</span> mutant strain and the wild-type strain is compared (Kazemi-Pour et al., <xref ref-type="bibr" rid="B50"> 2004 </xref> ; Watt et al., <xref ref-type="bibr" rid="B90"> 2009 </xref> ). Joshi et al. ( <xref ref-type="bibr" rid="B45"> 2010 </xref> ) discussed twin arginine transport (Tat) dependent translocation of vir factor in <italic> S. scabies </italic> under T3SSs. Tat-secreted vir proteins includes lipoproteins, ABC transporters, phospholipases/phosphoesterases, &#x003b2;-lactamase, and proteins involved in Fe homeostasis and Tat dep vir factors (Table <xref ref-type="table" rid="T1"> 1 </xref> ). T3SS-secreted proteins responsible for vir process usually silenced in plant-pathogenic bacteria and were only induced during the plant infection process (Buttner and Bonas, <xref ref-type="bibr" rid="B13"> 2002 </xref> ). Another vir factor Svx (secreted vir factor from <italic> Xanthomonas </italic> ), in a secrotomic study of <italic> P. atrosepticum </italic> , was analyzed (Corbett et al., <xref ref-type="bibr" rid="B20"> 2005 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Chen et al. ( <xref ref-type="bibr" rid="B16"> 2007 </xref> ) analyzed proteins from rice plasma membrane to study the early defense responses involved in <italic> Xa21 </italic> mediated <span class="trait">resistance.</span> <italic> Xa21 </italic> is a rice receptor kinase, which is predicted to perceive the <italic> X. oryzae </italic> pv. oryzae signal at the cell surface, leading to the &#x0201c;gene-for-gene&#x0201d; <span class="trait">resistance</span> response (Song et al., <xref ref-type="bibr" rid="B85"> 1995 </xref> ). Twenty proteins that were differentially expressed had potential functions in rice defense. Proteins expressed in partially resistant lines and a susceptible tomato species that are regulated in response to <italic> C. michiganensis </italic> ssp. michiganensis were identified 72 and 144 h post inoculation. Using 2-DE and MS, 26 differentially regulated tomato proteins were identified; 12 of which were directly related to defense such as PR-3, PR-9, and stress responses (Coaker et al., <xref ref-type="bibr" rid="B19"> 2004 </xref> ). PR proteins seem to be good candidates to develop the incompatible interactions with bacteria along with SAR in plants. Proteomic analysis was also used to detect the responses of the model legume <italic> M. truncatula </italic> to the pathogenic bacterium <italic> P. aeruginosa </italic> (Mathesius et al., <xref ref-type="bibr" rid="B65"> 2003 </xref> ). There is accumulation of 154 proteins, among which 21 are related to defense and stress responses. Molecular chaperones, protein related to defense, destination and storage were differentially expressed in resistant tomato cvs (Afroz et al., <xref ref-type="bibr" rid="B1"> 2009 </xref> ). Apical membrane antigen was found to be the novel protein expressed in susceptible cultivar in SA cascade (Afroz et al., <xref ref-type="bibr" rid="B2"> 2010 </xref> ). Kandasamy et al. ( <xref ref-type="bibr" rid="B47"> 2009 </xref> ) reported differential proteins in response to <italic> P. fluorescens </italic> in rice leaf sheath (Table <xref ref-type="table" rid="T2"> 2 </xref> ). Along with protein degradation and antioxidation, photosynthetic proteins thioredoxin was upregulated. Defense proteins can be the other target protein for the disease <span class="trait">resistance</span> against bacterial pathogens. Tomato seed treatment with rhizobacteria <italic> P. fluorescens </italic> exhibited growth promotion along with protection from infection by <italic> Fusarium oxysporum </italic> (Ramamoorthy et al., <xref ref-type="bibr" rid="B74"> 2002 </xref> ). PR-2, PR-3, and PR-5 were found to be induced in <italic> P. fluorescens </italic> treated plants challenged with <italic> F. oxysporum </italic> . Similarly, the expression of glutathione S-transferase (GST) is known to be involved in tagging toxic endogenous substrates with glutathione conjugation to transport toxic substrates into the vacuole through a glutathione pump (Ishikawa, <xref ref-type="bibr" rid="B37"> 1992 </xref> ). <span class="gene">GST</span> is reported to be induced in response to <italic> P. <span class="trait">syringae</span> </italic> in <italic> Arabidopsis </italic> , and has an important role in plant defense from oxidative damages caused by various biotic or abiotic stresses such as heavy metal, wounding, ethylene, ozone, and pathogen attack (Marrs, <xref ref-type="bibr" rid="B61"> 1996 </xref> ; Jones et al., <xref ref-type="bibr" rid="B43"> 2004 </xref> , <xref ref-type="bibr" rid="B41"> 2006a </xref> , <xref ref-type="bibr" rid="B42"> b </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <italic> Nictaba </italic> -Like Lectin Genes from <italic> Glycine max </italic> Confers Tolerance toward <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> Infection, Aphid Infestation and Salt Stress in Transgenic <italic> Arabidopsis </italic> Plants</h2>
			<h3>PMID:27826309</h3>
				

				
					<h3>PMCID:5078610</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Van Holle </surname>  <given-names> Sofie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/369756/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smagghe </surname>  <given-names> Guy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/21020/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Van Damme </surname>  <given-names> Els J. M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/115932/overview"/>  </contrib>
				<br/> 
				年份：<day> 07 </day>  <month> 10 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants have evolved a sophisticated immune system that allows them to recognize invading pathogens by specialized receptors. Carbohydrate-binding proteins or lectins are part of this immune system and especially the lectins that reside in the nucleocytoplasmic compartment are known to be implicated in biotic and abiotic stress responses. The class of Nictaba-like lectins (NLL) groups all proteins with homology to the tobacco (Nicotiana tabacum) lectin, known as a stress-inducible lectin. Here we focus on two Nictaba homologs from soybean (Glycine max), referred to as <span class="gene">GmNLL1</span> and <span class="gene">GmNLL2.</span> Confocal laser scanning microscopy of fusion constructs with the green fluorescent protein either transiently expressed in Nicotiana benthamiana leaves or stably transformed in tobacco BY-2 suspension cells revealed a nucleocytoplasmic localization for the <span class="gene">GmNLLs</span> under study. RT-qPCR analysis of the transcript levels for the Nictaba-like lectins in soybean demonstrated that the genes are expressed in several tissues throughout the development of the plant. Furthermore, it was shown that salt treatment, Phytophthora sojae infection and Aphis glycines infestation trigger the expression of particular NLL genes. Stress experiments with Arabidopsis lines overexpressing the NLLs from soybean yielded an enhanced tolerance of the plant toward bacterial infection <span class="trait">(Pseudomonas</span> <span class="trait">syringae),</span> insect infestation (Myzus persicae) and salinity. Our data showed a better performance of the transgenic lines compared to wild type plants, indicating that the NLLs from soybean are implicated in the stress response. These data can help to further elucidate the physiological importance of the Nictaba-like lectins from soybean, which can ultimately lead to the design of crop plants with a better tolerance to changing environmental conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						An extensive survey of genome databases revealed that Nictaba-like lectins (NLLs) are widespread in plants (Delporte et al., <xref rid="B13" ref-type="bibr"> 2015 </xref> ). Thus, far, functional characterization has been focused on the tobacco lectin and one F-box Nictaba homolog from <italic> Arabidopsis </italic> (Stefanowicz et al., <xref rid="B53" ref-type="bibr"> 2012 </xref> ; Delporte et al., <xref rid="B13" ref-type="bibr"> 2015 </xref> ). Lectin expression in tobacco is enhanced after caterpillar attack, suggesting a role for Nictaba in plant defense. Furthermore, experiments using transgenic tobacco plants overexpressing the lectin gene or plants with reduced expression indicated that Nictaba exerts insecticidal activity toward Lepidopteran pest insects (Vandenborre et al., <xref rid="B58" ref-type="bibr"> 2010 </xref> ). The <italic> Arabidopsis </italic> F-box-Nictaba homolog is upregulated after treatment with salicylic acid and upon <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> infection and overexpression of the gene in <italic> Arabidopsis </italic> plants confers increased tolerance to the pathogen (Stefanowicz et al., <xref rid="B54" ref-type="bibr"> 2016 </xref> ). In order to refine our understanding of this specific group of nucleocytoplasmic lectins, we focus here on some Nictaba-like lectins from soybean. Soybean presents an exciting opportunity to investigate the stress inducibility of these proteins in an important crop species. Several <italic> <span class="gene">GmNictaba</span> </italic> -related genes have recently been identified in the soybean genome. Of the 31 identified <italic> <span class="gene">GmNLL</span> </italic> genes, 25 encode chimerolectins, consisting of one Nictaba lectin domain combined with an N-terminal F-box protein domain. The remaining six genes encode Nictaba orthologs containing one or two Nictaba domains as building blocks (Van Holle and Van Damme, <xref rid="B61" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Since <italic> <span class="gene">GmNLL1</span> </italic> and <italic> <span class="gene">GmNLL2</span> </italic> gene expression in soybean was significantly upregulated upon infection with <italic> P. sojae </italic> (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ), the hypothesis was put forward that <italic> <span class="gene">GmNLLs</span> </italic> play a role in plant defense responses. The <italic> Arabidopsis </italic> lines overexpressing <italic> <span class="gene">GmNLL1</span> </italic> or <italic> <span class="gene">GmNLL2</span> </italic> and wild type plants were challenged with <italic> P. brassicae </italic> using mycelium plugs or zoospore drop inoculation to investigate the effect of <italic> <span class="gene">GmNLL</span> </italic> overexpression on the plant's <span class="trait">resistance</span> to pathogen infection. However, no differences in disease progression were observed between wild type plants and the <italic> <span class="gene">GmNLLs</span> </italic> overexpression lines. All plants became heavily colonized by <italic> P. brassicae </italic> as confirmed by staining of callose deposition in infected leaves (Results not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						Wild type <italic> Arabidopsis </italic> plants and transgenic 35S:: <italic> <span class="gene">GmNLL1</span> </italic> and 35S:: <italic> <span class="gene">GmNLL2</span> </italic> plants were subjected to bacterial infection with <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> tomato </italic> to further investigate the role of <italic> <span class="gene">GmNLLs</span> </italic> in plant defense. Disease symptoms, bacterial growth and cell death were monitored daily. The first 2 days after the infection, no visible signs of bacterial infection were observed. Starting from 3 days post-infection, lesions were observed on the leaves and reduced disease symptoms were clear 4 days post-infection for the overexpression lines compared to the wild plants (Figure <xref ref-type="fig" rid="F9"> 9A </xref> and Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> 2 </xref> ). In wild type plants, around 70% of the leaf is constituted of discolored lesions caused by the pathogen infection, while for all overexpression lines, the percentage of leaf damage ranged between 16 and 42% 4 days post-infection. The lesion area of mock infected plants was also measured for all time points but the calculated lesion area was never higher than 2%.
					</p>
				
					<p>&nbsp;&nbsp;
						Infection assays with <italic> P. brassicae </italic> did not show an enhanced disease <span class="trait">resistance</span> for the tested overexpression lines compared to wild type <italic> Arabidopsis </italic> plants. Bacterial blight of soybean is caused by <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> glycinea </italic> and can cause significant <span class="trait">yield</span> losses. <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNLLs</span> </italic> were used in an infection assay with <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> tomato </italic> , an <italic> Arabidopsis </italic> compatible pathogen (Katagiri et al., <xref rid="B26" ref-type="bibr"> 2002 </xref> ) and demonstrated that less disease symptoms were observed on the transgenic lines compared to wild type plants. These observations could be explained by reduced bacterial biomass ratios for some of the overexpression lines. It was demonstrated that <italic> <span class="trait">Pseudomonas</span> syringe </italic> induces both SA and JA pathways (Spoel et al., <xref rid="B52" ref-type="bibr"> 2003 </xref> ) but RT-qPCR analysis demonstrated that these pathways are not perturbed in the <italic> <span class="trait">Pseudomonas</span> </italic> infected <italic> <span class="gene">GmNLL</span> </italic> overexpression lines (data not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						All previous research from NLLs focused on the model species <italic> Arabidopsis </italic> and tobacco. Hence, this is the first study that focusses on NLLs in a crop species. Our data show that similar to Nictaba in tobacco, the NLLs from soybean can also be considered as stress inducible proteins. Nevertheless, the <italic> Nictaba </italic> -like genes in both species act differently. The expression of <italic> Nictaba </italic> from tobacco is increased after treatment with jasmonates whereas this is not the case for the soybean <italic> NLLs </italic> under study. <italic> Nictaba </italic> expression in tobacco was enhanced after insect herbivory by caterpillars but not by aphids. For soybean, our data clearly show that <italic> A. glycines </italic> infestation triggers the expression of particular <italic> NLL </italic> genes. Furthermore, <italic> <span class="gene">GmNLL</span> </italic> overexpression lines in <italic> Arabidopsis </italic> reduced the growth and development of <italic> M. persicae </italic> . In addition, these transgenic lines also enhanced tolerance to salt stress at the seedling stage, and showed less disease symptoms upon <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> infection. The data strongly suggest the involvement of <italic> Gm </italic> NLLs in plant defense responses not only against pest or pathogens, but also in abiotic stress. These results propose that <italic> <span class="gene">GmNLLs</span> </italic> are controlled by a complex regulatory network. <italic> Gm </italic> NLL1 and <italic> Gm </italic> NLL2 are two possible candidates to further elucidate the physiological importance of the Nictaba-like lectins from soybean, which can ultimately lead to novel strategies and design of crop plants with improved tolerance to changing environmental conditions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Pseudomonas <span class="trait">syringae</span> Effector AvrPphB Suppresses AvrB-Induced Activation of RPM1 but Not AvrRpm1-Induced Activation.</h2>
			<h3>PMID:25625821  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Russell AR(1), Ashfield T(1), Innes RW(1).
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> effector AvrB triggers a hypersensitive <span class="trait">resistance</span> response in Arabidopsis and soybean plants expressing the disease <span class="trait">resistance</span> (R) proteins RPM1 and Rpg1b, respectively. In Arabidopsis, AvrB induces RPM1-interacting protein kinase (RIPK) to phosphorylate a disease regulator known as RIN4, which subsequently activates RPM1-mediated defenses. Here, we show that AvrPphB can suppress activation of RPM1 by AvrB and this suppression is correlated with the cleavage of RIPK by AvrPphB. Significantly, AvrPphB does not suppress activation of RPM1 by AvrRpm1, suggesting that RIPK is not required for AvrRpm1-induced modification of RIN4. This observation indicates that AvrB and AvrRpm1 recognition is mediated by different mechanisms in Arabidopsis, despite their recognition being determined by a single R protein. Moreover, AvrB recognition but not AvrRpm1 recognition is suppressed by AvrPphB in soybean, suggesting that AvrB recognition requires a similar molecular mechanism in soybean and Arabidopsis. In support of this, we found that phosphodeficient mutations in the soybean <span class="gene">GmRIN4a</span> and <span class="gene">GmRIN4b</span> proteins are sufficient to block Rpg1b-mediated hypersensitive response in transient assays in Nicotiana glutinosa. Taken together, our results indicate that AvrB and AvrPphB target a conserved defense signaling pathway in Arabidopsis and soybean that includes RIPK and RIN4.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The endoplasmic reticulum binding protein BiP displays dual function in modulating cell death events.</h2>
			<h3>PMID:24319082  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carvalho HH(1), Silva PA, Mendes GC, Brustolini OJ, Pimenta MR, Gouveia BC, Valente MA, Ramos HJ, Soares-Ramos JR, Fontes EP.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The binding protein (BiP) has been demonstrated to participate in innate immunity and attenuate endoplasmic reticulum- and osmotic stress-induced cell death. Here, we employed transgenic plants with manipulated levels of BiP to assess whether BiP also controlled developmental and hypersensitive programmed cell death (PCD). Under normal conditions, the BiP-induced transcriptome revealed a robust down-regulation of developmental PCD genes and an up-regulation of the genes involved in hypersensitive PCD triggered by nonhost-pathogen interactions. Accordingly, the BiP-overexpressing line displayed delayed leaf senescence under normal conditions and accelerated hypersensitive response triggered by <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv tomato in soybean (Glycine max) and tobacco (Nicotiana tabacum), as monitored by measuring hallmarks of PCD in plants. The BiP-mediated delay of leaf senescence correlated with the attenuation of N-rich protein (NRP)-mediated cell death signaling and the inhibition of the senescence-associated activation of the unfolded protein response (UPR). By contrast, under biological activation of salicylic acid (SA) signaling and hypersensitive PCD, BiP overexpression further induced NRP-mediated cell death signaling and antagonistically inhibited the UPR. Thus, the SA-mediated induction of <span class="gene">NRP</span> cell death signaling occurs via a pathway distinct from UPR. Our data indicate that during the hypersensitive PCD, BiP positively regulates the <span class="gene">NRP</span> cell death signaling through a yet undefined mechanism that is activated by SA signaling and related to ER functioning. By contrast, BiP's negative regulation of leaf senescence may be linked to its capacity to attenuate the UPR activation and <span class="gene">NRP</span> cell death signaling. Therefore, BiP can function either as a negative or positive modulator of PCD events.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Suppression of the ribosomal L2 gene reveals a novel mechanism for stress adaptation in soybean.</h2>
			<h3>PMID:11346953  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ludwig A(1), Tenhaken R.
				<br/> 
				年份：2001 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea bacteria or zoospores of the fungus Phytophthora sojae were used to trigger a hypersensitive reaction (HR) in cell cultures of soybean (Glycine max [L.] Merr. cv. Williams 82). During a screen for genes that show an altered expression as a response to dying neighbour cells we have identified a gene fragment that is specifically but transiently down-regulated in an HR. The corresponding cDNA codes for the ribosomal protein L2 (rpL2) of 80S ribosomes, which is essential for the peptidyl-transferase activity. Two gene copies of <span class="gene">rpL2</span> exist in soybean and both genes are transcribed. The temporary down-regulation of the <span class="gene">rpL2</span> genes is followed by a transient block in the synthesis of new proteins as visualised by pulse-labelling experiments using 35S-amino acids. The same basic phenomenon was also found after treatment of soybean cells with other stress-causing compounds such as elicitors or heavy metals. It is suggested that the transient block in protein synthesis allows a more rapid depletion of, for example, signal molecules with a short half-life time and thus leads to a faster adaptation of the cellular protein inventory to the new environmental conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> N </italic> &#x02010;acylhomoserine lactone&#x02010;regulation of genes mediating motility and pathogenicity in <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pathovar <italic> tabaci </italic> 11528</h2>
			<h3>PMID:28133926</h3>
				
					<h3>PMCID:5458577</h3>
					<h4>期刊:MicrobiologyOpen</h4>
				

				
			<p>
				作者：<contrib id="mbo3440-cr-0001" contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Feifei </given-names>  </name>  <xref ref-type="aff" rid="mbo3440-aff-0001">
				<br/> 
				年份：<day> 07 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The authors thank Prof. H. S. Baik (Dept. of Microbiology, Pusan National University) for providing <italic> P. <span class="trait">syringae</span> </italic> 11528 &#x00394; <italic> psyI </italic> mutant. This study was funded by the National Natural Science Foundation of China (21177145 and 21377157), the Strategic Priority Research Program of the <span class="gene">CAS</span> (XDB15030101 and XDB14020206), the Youth Innovation Promotion Association <span class="gene">CAS</span> (2016039), and Project 135 of the Chinese Academy of Sciences (YSW2013B06).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the trehalose biosynthetic loci of <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> and their contribution to fitness in the phyllosphere.</h2>
			<h3>PMID:20192963  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Freeman BC(1), Chen C, Beattie GA.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Surprisingly little is known of the trehalose biosynthetic pathways in pseudomonads, despite the importance of trehalose to protecting cells from environmental stresses such as low water availability. The genome of the foliar pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. tomato strain DC3000 contains genes for two trehalose biosynthetic pathways, TreS and TreYZ, and lacks genes for the more common OtsAB pathway. Deletion of either the <span class="gene">treS</span> (PSPTO_2760-2762) or treY/treZ (PSPTO_3125-3134) locus eliminated trehalose accumulation and reduced bacterial growth under hyperosmotic conditions. In evaluating the role of trehalose in P. <span class="trait">syringae</span> fitness on leaves, we found that a double deletion mutant lacking these loci exhibited poorer survival than the wild type on tomato leaves over a 2-week period in a growth chamber. Similarly, this mutant exhibited reduced survival on leaves of susceptible and resistant cultivars of the host plant tomato and of the non-host plant soybean over a 10-day period in field plots. Thus, the trehalose biosynthetic loci in P. <span class="trait">syringae,</span> which are highly conserved among pseudomonads, contributed to DC3000 fitness on leaves, supporting a role for trehalose in P. <span class="trait">syringae</span> survival and population maintenance in the phyllosphere.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Determining the <span class="gene">GmRIN4</span> Requirements of the Soybean Disease <span class="trait">Resistance</span> Proteins Rpg1b and Rpg1r Using a <italic> Nicotiana glutinosa </italic> -Based Agroinfiltration System</h2>
			<h3>PMID:25244054</h3>
				

				
					<h3>PMCID:4171518</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kessens </surname>  <given-names> Ryan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ashfield </surname>  <given-names> Tom </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Sang Hee </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Innes </surname>  <given-names> Roger W. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rpg1b and Rpg1r are soybean disease <span class="trait">resistance</span> (R) genes responsible for conferring <span class="trait">resistance</span> to <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> strains expressing the effectors AvrB and AvrRpm1, respectively. The study of these cloned genes would be greatly facilitated by the availability of a suitable transient expression system. The commonly used Niciotiana benthamiana-based system is not suitable for studying Rpg1b and Rpg1r function, however, because expression of AvrB or AvrRpm1 alone induces a hypersensitive response (HR), indicating that N. benthamiana contains endogenous R genes that recognize these effectors. To identify a suitable alternative host for transient expression assays, we screened 13 species of Nicotiana along with 11 accessions of N. tabacum for lack of response to transient expression of AvrB and AvrRpm1. We found that N. glutinosa did not respond to either effector and was readily transformable as determined by transient expression of β-glucuronidase. Using this system, we determined that Rpg1b-mediated HR in N. glutinosa required co-expression of avrB and a soybean ortholog of the Arabidopsis RIN4 gene. All four soybean RIN4 orthologs tested worked in the assay. In contrast, Rpg1r did not require co-expression of a soybean RIN4 ortholog to recognize AvrRpm1, but recognition was suppressed by co-expression with AvrRpt2. These observations suggest that an endogenous RIN4 gene in N. glutinosa can substitute for the soybean RIN4 ortholog in the recognition of AvrRpm1 by Rpg1r.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						While recognition of AvrB and AvrRpm1 in soybean likely has similarities to that in Arabidopsis, there are at least two key differences. Unlike Arabidopsis, which requires just one <italic> R </italic> gene to confer <span class="trait">resistance</span> to both AvrB and AvrRpm1, soybean requires two <italic> R </italic> genes. These <italic> R </italic> genes are <italic> Rpg1b </italic> and <italic> Rpg1r </italic> , which confer <span class="trait">resistance</span> to AvrB and AvrRpm1, respectively <xref rid="pone.0108159-Ashfield1" ref-type="bibr"> [10] </xref> . Another difference is that the Arabidopsis genome only encodes one <italic> RIN4 </italic> gene while soybean contains four <italic> RIN4 </italic> -homologues ( <italic> <span class="gene">GmRIN4a,</span> <span class="gene">GmRIN4b,</span> <span class="gene">GmRIN4c,</span> </italic> and Gm <italic> <span class="gene">RIN4d</span> </italic> ) <xref rid="pone.0108159-Chen1" ref-type="bibr"> [11] </xref> . At least two of these family members, <italic> <span class="gene">GmRIN4a</span> </italic> and <italic> <span class="gene">GmRIN4b</span> </italic> , appear to be necessary for <italic> Rpg1b </italic> to confer <span class="trait">resistance</span> to AvrB <xref rid="pone.0108159-Selote1" ref-type="bibr"> [12] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The goal of this study was to identify an alternative species to <italic> N. benthamiana </italic> for transient expression studies involving soybean Rpg1b and Rpg1r and to determine the <italic> Gm </italic> RIN4 requirements of each of these R proteins. An ideal species would have many of the characteristics that make <italic> N. benthamiana </italic> a good system such as ease of injection and high-transformation efficiency, but would not display signs of HR when <italic> AvrB </italic> and <italic> AvrRpm1 </italic> were expressed alone. To achieve this goal, we screened <italic> Nicotiana </italic> germplasm, including 13 distinct species and 11 accessions of <italic> N. tabacum </italic> , for their response to transiently expressed <italic> AvrB </italic> and <italic> AvrRpm1 </italic> . The transformation efficiency of each species was also assessed using a <italic> GUS </italic> reporter gene to ensure that lack of HR was not simply due to low levels of gene expression. The well-studied <italic> RPS5 </italic> -mediated HR pathway was reconstituted in the most promising genotype ( <italic> N. glutinosa </italic> ) to test its efficacy for reconstructing an <italic> R </italic> gene pathway. This was accomplished by co-expressing the <italic> P. <span class="trait">syringae</span> </italic> effector <italic> AvrPphB </italic> , its target <italic> PBS1 </italic> , and the <italic> R </italic> gene <italic> RPS5 </italic> <xref rid="pone.0108159-Simonich1" ref-type="bibr"> [17] </xref> . The final step was to co-express each of the soybean <italic> R </italic> genes with their corresponding effectors and one or more of the <italic> <span class="gene">GmRIN4</span> </italic> s to determine which, if any, <italic> Gm </italic> RIN4s were required by either R protein to detect its corresponding effector.
					</p>
				
					<p>&nbsp;&nbsp;
						Since Rpg1r did not require co-expression of a <span class="gene">GmRIN4</span> to recognize AvrRpm1 in <italic> N. glutinosa </italic> , we hypothesized that it may be using an endogenous RIN4 protein for this purpose. To test this hypothesis, we employed the <italic> P. <span class="trait">syringae</span> </italic> effector AvrRpt2, which is a cysteine protease that has been shown to cleave Arabidopsis RIN4, leading to its degradation <xref rid="pone.0108159-Axtell2" ref-type="bibr"> [22] </xref> . A BLAST search of the <italic> N. benthamiana </italic> genome using the Arabidopsis RIN4 amino acid sequence as the query revealed two predicted full length proteins with high sequence similarity and conserved AvrRpt2 cleavage sites <xref rid="pone.0108159-Chisholm1" ref-type="bibr"> [23] </xref> ( <xref ref-type="supplementary-material" rid="pone.0108159.s003"> Fig. S3 </xref> ). By expressing <italic> AvrRpt2 </italic> under the constitutively active CaMV 35S promoter, we hoped to eliminate any endogenous RIN4 homologues in <italic> N. glutinosa </italic> before inducing the expression of <italic> Rpg1r </italic> and <italic> AvrRpm1 </italic> , both of which were under the control of a DEX-inducible promoter. Co-expression of <italic> AvrRpt2 </italic> with <italic> Rpg1r </italic> and <italic> AvrRpm1 </italic> led to a reduction in the severity of HR, as indicated by reduced leaf collapse, compared to co-expression of the proteolytically inactive <italic> AvrRpt2 (C122A) </italic> mutant with <italic> Rpg1r </italic> and <italic> AvrRpm1 </italic> . The images in <xref ref-type="fig" rid="pone-0108159-g003"> Fig. 3a </xref> are representative of the typical responses displayed by leaves expressing each combination while <xref ref-type="fig" rid="pone-0108159-g003"> Figure 3b </xref> is an assessment of the strength of HR for each combination. These data suggest that Rpg1r is employing an endogenous copy of RIN4 for recognition of AvrRpm1.
					</p>
				
					<p>&nbsp;&nbsp;
						Through the use of virus-induced gene silencing (VIGS) of soybean <italic> RIN4 </italic> genes, Selote and colleagues have previously determined that both <italic> <span class="gene">GmRIN4A</span> </italic> and <italic> <span class="gene">GmRIN4B</span> </italic> are required for <italic> Rpg1b </italic> -mediated <span class="trait">resistance</span> to <italic> P. <span class="trait">syringae</span> </italic> strains expressing <italic> AvrB </italic> <xref rid="pone.0108159-Selote1" ref-type="bibr"> [12] </xref> , while <italic> <span class="gene">GmRIN4C</span> </italic> and <italic> <span class="gene">GmRIN4D</span> </italic> are not <xref rid="pone.0108159-Selote2" ref-type="bibr"> [16] </xref> . Contrary to their findings, our findings suggest that each of the individual <italic> Gm </italic> RIN4 proteins can be used by Rpg1b to recognize AvrB, as co-expression of any <italic> <span class="gene">GmRIN4</span> </italic> with <italic> Rpg1b </italic> and <italic> AvrB </italic> in <italic> N. glutinosa </italic> resulted in an HR. However, a major difference between Selote&#x02019;s work and ours is that they used VIGS to silence native genes while we used transient expression to over-express foreign genes. If there is a <span class="gene">GmRIN4</span> expression level threshold in soybean required for Rpg1b function, then transient overexpression would likely exceed this threshold. Therefore, it is possible that <italic> <span class="gene">GmRIN4C</span> </italic> and <italic> <span class="gene">GmRIN4D</span> </italic> could be used for <italic> Rpg1b </italic> -mediated <span class="trait">resistance</span> in soybean if they were expressed at a high enough level.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> P. <span class="trait">syringae</span> </italic> effector genes <italic> AvrPphB </italic> , <italic> AvrB </italic> and <italic> AvrRpm1 </italic> were cloned in the pTA7002 plasmid, which places the transgene under control of a dexamethasone (DEX) inducible promoter <xref rid="pone.0108159-Aoyama1" ref-type="bibr"> [28] </xref> . The empty vector, <italic> RPS5 </italic> , <italic> PBS1 </italic> , <italic> Rpg1b </italic> , <italic> Rpg1r </italic> and <italic> <span class="gene">GmRIN4</span> </italic> constructs also employed the pTA7002 vector. The <italic> RPS5 </italic> and <italic> PBS1 </italic> constructs contained C-terminal 5x-Myc and 3x-HA tags, respectively. The <italic> <span class="gene">GmRIN4</span> </italic> constructs contained an N-terminal 5x-Myc tag, while the <italic> Rpg1r </italic> construct contained a C-terminal sYFP tag. The <italic> GUS </italic> reporter gene was in the pCNL65 plasmid, which places the transgene under control of the cauliflower mosaic virus 35S promoter <xref rid="pone.0108159-Liu2" ref-type="bibr"> [29] </xref> . The effector genes <italic> AvrRpt2 </italic> and <italic> AvrRpt2 (C122A) </italic> were also under CaMV 35S control in the pMD1 vector and each had a C-terminal 3x-HA tag <xref rid="pone.0108159-Axtell2" ref-type="bibr"> [22] </xref> . Expression of DEX-inducible constructs was achieved by spraying transiently transformed plants with a solution of 50 &#x000b5;M dexamethasone (Sigma-Aldrich) and 0.02% Silwet-L77 (Momentive, Albany, NY) approximately 40 hours post-infiltration.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping of the major <italic> Soybean dwarf virus </italic> <span class="trait">resistance</span> gene <italic> Rsdv1 </italic> of the soybean cultivar &#x02018;Wilis&#x02019;</h2>
			<h3>PMID:24399914</h3>
				
					<h3>PMCID:3859353</h3>
					<h4>期刊:Breeding Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yamashita </surname>  <given-names> Yoko </given-names>  </name>  <xref rid="c1-63_417" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="af1-63_417"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takeuchi </surname>  <given-names> Toru </given-names>  </name>  <xref ref-type="aff" rid="af1-63_417"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ohnishi </surname>  <given-names> Shizen </given-names>  </name>  <xref ref-type="aff" rid="af2-63_417"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sasaki </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="af2-63_417"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tazawa </surname>  <given-names> Akiko </given-names>  </name>  <xref ref-type="aff" rid="af3-63_417"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 10 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Another <italic> Rsdv1 </italic> candidate gene, <italic> <span class="gene">Glyma05g34360,</span> </italic> is predicted to encode the mitochondrial import receptor subunit TOM40. TOM40 is required for replication of <italic> African swine fever virus </italic> in cultured human cells ( <xref rid="b5-63_417" ref-type="bibr"> Chang <italic> et al. </italic> 2006 </xref> ). No specific relation has been suggested between TOM40 and virus replication in plants. The other four genes have not been reported to be specifically associated with <span class="trait">resistance</span> to viruses.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Defects in D-rhamnosyl residue biosynthetic genes affect lipopolysaccharide structure, motility, and cell-surface hydrophobicity in <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathovar glycinea race 4.</h2>
			<h3>PMID:23470736  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chiku K(1), Tsunemi K, Yamamoto M, Ohnishi-Kameyama M, Yoshida M, Ishii T, Taguchi F, Iwaki M, Ichinose Y, Ono H.
				<br/> 
				年份：2013
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						D-rhamnose (D-Rha) residue is a major component of lipopolysaccharides (LPSs) in strains of the phytopathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pathovar glycinea. To investigate the effects of a deficiency in GDP-D-rhamnose biosynthetic genes on LPS structure and pathogenicity, we generated three mutants defective in D-Rha biosynthetic genes, encoding proteins GDP-D-mannose 4,6-dehydratase <span class="gene">(GMD),</span> GDP-4-keto-6-deoxy-D-mannose reductase (RMD), and a putative α-D-rhamnosyltransferase (WbpZ) in P. <span class="trait">syringae</span> pv. glycinea race 4. The Δgmd, Δrmd, and ΔwbpZ mutants had a reduced O-antigen polysaccharide consisting of D-Rha residues as compared with the wild type (WT). The swarming motility of the Δgmd, Δrmd, and ΔwbpZ mutant strains decreased and hydrophobicity and adhesion ability increased as compared with WT. Although the mutants had truncated O-antigen polysaccharides, and altered surface properties, they showed virulence to soybean, as WT did.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phosphatidylcholine synthesis is essential for HrpZ harpin secretion in plant pathogenic <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> and non-pathogenic <span class="trait">Pseudomonas</span> sp. 593.</h2>
			<h3>PMID:23886927  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xiong M(1), Long D(1), He H(1), Li Y(1), Li Y(1), Wang X(2).
				<br/> 
				年份：2014 Feb-Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. <span class="trait">syringae</span> van Hall is important phytopathogenic bacterium of stone fruit trees, and able to elicit hypersensitive response (HR) in nonhost plants. The HrpZ, secreted via type III secretion system (T3SS) to the extracellular space of the plant, is a T3SS-dependent protein and a sole T3SS effector able to induce the host defense response outside host cells. We deleted the phosphatidylcholine synthase gene (pcs) of P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> van Hall CFCC 1336, and found that the 1336 pcs(-) mutant was unable to synthesize phosphatidylcholine and elicit a typical HR in soybean. Further studies showed that the 1336 pcs(-) mutant was unable to secrete HrpZ harpin but could express HrpZ protein in cytoplasm as effectively as the wild type. To confirm if phosphatidylcholine affects HrpZ harpin secretion, we introduced the <span class="gene">hrpZ</span> gene into the soil-dwelling bacterium <span class="trait">Pseudomonas</span> sp. 593 and the 593 pcs(-) mutant, which were unable to express HrpZ harpin and elicit HR in tobacco or soybean. Western blotting and HR assay showed that the 593H not only secreted HrpZ harpin but also caused a strong HR in tobacco and soybean. In contrast, the 593 pcs(-)H only expressed HrpZ protein in its cytoplasm at the wild type level, but did not secrete HrpZ harpin or elicit HR reaction. Our results demonstrate that phosphatidylcholine is essential for the secretion of HrpZ harpin in P. <span class="trait">syringae</span> pv. <span class="trait">syringae</span> van Hall and other <span class="trait">Pseudomonas</span> strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ectopic expression of <italic> AtPAD4 </italic> broadens <span class="trait">resistance</span> of soybean to soybean cyst and root-knot nematodes</h2>
			<h3>PMID:23617694</h3>
				

				
					<h3>PMCID:3648381</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Youssef </surname>  <given-names> Reham M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> Reham.Youssef@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> MacDonald </surname>  <given-names> Margaret H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Margaret.MacDonald@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Brewer </surname>  <given-names> Eric P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Eric.Brewer@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Bauchan </surname>  <given-names> Gary R </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Gary.Bauchan@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Kim </surname>  <given-names> Kyung-Hwan </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> kimhwan@rda.go.kr </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ben.matthews@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The gene encoding <span class="gene">PAD4</span> (PHYTOALEXIN-DEFICIENT4) is required in Arabidopsis for expression of several genes involved in the defense response to <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. maculicola. AtPAD4 (Arabidopsis thaliana PAD4) encodes a lipase-like protein that plays a regulatory role mediating salicylic acid signaling. RESULTS: We expressed the gene encoding AtPAD4 in soybean roots of composite plants to test the ability of AtPAD4 to deter plant parasitic nematode development. The transformed roots were challenged with two different plant parasitic nematode genera represented by soybean cyst nematode (SCN; Heterodera glycines) and root-knot nematode (RKN; Meloidogyne incognita). Expression of AtPAD4 in soybean roots decreased the number of mature SCN females 35 days after inoculation by 68 percent. Similarly, soybean roots expressing AtPAD4 exhibited 77 percent fewer galls when challenged with RKN. CONCLUSIONS: Our experiments show that AtPAD4 can be used in an economically important crop, soybean, to provide a measure of <span class="trait">resistance</span> to two different genera of nematodes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The gene encoding <italic> <span class="gene">PAD4</span> </italic> (PHYTOALEXIN-DEFICIENT4) is required in <italic> Arabidopsis </italic> for expression of several genes involved in the defense response to <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pv. <italic> maculicola. AtPAD4 </italic> ( <italic> Arabidopsis thaliana <span class="gene">PAD4</span> </italic> ) encodes a lipase-like protein that plays a regulatory role mediating salicylic acid signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						Inducible defense responses are activated when plants respond to pathogen attack [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. The gene-for-gene defense response is a strong form of plant <span class="trait">resistance</span> against pathogens. This type of <span class="trait">resistance</span> is turned on when the plants have a specific <span class="trait">resistance</span> ( <italic> R </italic> ) gene that recognizes the product of a corresponding pathogen gene known as the avirulence ( <italic> avr </italic> ) gene. This interaction between an <italic> R </italic> -gene and an <italic> avr </italic> -gene triggers the hypersensitive response (HR) and rapid expression of defense responses that result in programmed cell death within 24 h of infection [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. Another type of defense response occurs after attack by virulent pathogens that do not have an <italic> avr </italic> -gene recognized by the plant. In this case, the plant responds more slowly than in gene-for-gene <span class="trait">resistance,</span> allowing the pathogen to multiply. <span class="trait">Resistance</span> to virulent pathogens can occur through a phenomenon called systemic acquired <span class="trait">resistance</span> (SAR) [ <xref ref-type="bibr" rid="B3"> 3 </xref> ], which occurs after the hypersensitive response. SAR reduces symptoms produced by a variety of pathogens, but not by all pathogens [ <xref ref-type="bibr" rid="B4"> 4 </xref> ]. Salicylic acid (SA) plays a central signaling role in both gene-for-gene <span class="trait">resistance</span> and SAR. This role for SA was proven by construction of transgenic plants expressing a bacterial salicylate hydroxylase gene ( <italic> <span class="gene">nahG</span> </italic> ) that converts SA to catechol [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. During gene-for-gene <span class="trait">resistance</span> or infection with virulence pathogens, the <italic> <span class="gene">nahG</span> </italic> plants failed to express pathogenesis related ( <italic> PR) </italic> genes, and their susceptibility to both virulent and avirulent pathogens was greatly enhanced [ <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B6"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Mutants of the model plant <italic> Arabidopsis thaliana </italic> have been used for studying SA-dependent regulation of plant defense responses. Production of certain defense signals is controlled by the <italic> Arabidopsis <span class="gene">PAD4</span> </italic> ( <italic> AtPAD4 </italic> ) gene. Plants carrying <italic> <span class="gene">PAD4</span> </italic> mutations displayed reduced levels of SA, decreased expression of the defense gene <italic> PR1 </italic> , and reduced synthesis of the indole derivative, camalexin, after infection with a virulent strain of <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> [ <xref ref-type="bibr" rid="B7"> 7 </xref> - <xref ref-type="bibr" rid="B10"> 10 </xref> ]. In contrast, the role of <italic> <span class="gene">PAD4</span> </italic> in defense against the green peach aphid <italic> Myzus persicae </italic> Sulzer is independent of SA and camalexin [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Reported that <italic> <span class="gene">PAD4</span> </italic> encodes a nucleo-cytoplasmic protein which has similarity to triacyl glycerol lipases and other esterases. In defense signaling, <italic> <span class="gene">PAD4</span> </italic> acts in conjunction with the <italic> EDS1 </italic> gene (ENHANCED DISEASE SUSCEPTIBILITY1), which encodes a structurally related protein also found in the nucleus and cytoplasm [ <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B13"> 13 </xref> ]. EDS1 is required for accumulation of <span class="gene">PAD4</span> protein [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. EDS1 also interacts with another lipase-like protein, <span class="gene">SAG101</span> (SENESCENCE-ASSOCIATED GENE101), which accumulates in the nucleus [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. The occurrence of EDS1-PAD4 and EDS1-SAG101 complexes inside plant cells suggests that EDS1 works as an adaptor for both <span class="gene">PAD4</span> and <span class="gene">SAG101</span> in defense signaling [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although <italic> <span class="gene">PAD4</span> </italic> has been extensively studied in <italic> Arabidopsis </italic> , less is known about its role in conferring <span class="trait">resistance</span> to nematodes, and it is not known if <italic> AtPAD4 </italic> can function in economically important crops, such as soybean, to provide <span class="trait">resistance</span> to nematodes. The soybean cyst nematode (SCN; <italic> Heterodera glycines </italic> ) and the root-knot nematode (RKN; <italic> Melidogyne incognita </italic> ) are obligatory plant parasites that are responsible for more than 100 billion U. S. dollars in <span class="trait">yield</span> losses annually of economic crops worldwide [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. Both nematode species establish complex feeding sites within their host plants. At infection, the pre-parasitic second stage juveniles (J2) penetrate the roots and migrate towards the vascular cylinder where they induce the growth of a multinuclear feeding site, termed a &#x02018;syncytium&#x02019; produced by SCN and a &#x02018;giant cell&#x02019; produced by RKN [ <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although, there have been extensive studies on the defense response of <italic> Arabidopsis </italic> to fungal and bacterial pathogens (1&#x02013;6), little of this work in <italic> Arabidopsis </italic> has been directly translated to economically important food crops, such as soybean, and particularly in respect to plant parasitic nematodes. The <italic> Arabidopsis </italic> lipase-like protein PHYTOALEXIN DEFICIENT4 (PAD4) was identified by several genetic studies as essential component for plant immunity against virulent pathogens for promoting the defense signaling hormone salicylic acid (SA) and the phytoalexin camalexin [ <xref ref-type="bibr" rid="B8"> 8 </xref> - <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> ]. Our experiments show that <italic> AtPAD4 </italic> can be used in an economically important crop, soybean, to provide a measure of <span class="trait">resistance</span> to two different genera of nematodes. In <italic> Arabidopsis, </italic> the <italic> <span class="gene">PAD4</span> </italic> gene functions upstream of the defense responses triggered by SA [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. <span class="gene">PAD4</span> can work in combination with EDS1 to trigger aspects of the defense response, but EDS1 can also interact with <span class="gene">SAG101</span> independent of <span class="gene">PAD4</span> [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. In our experiments, overexpression of <italic> AtPAD4 </italic> in soybean roots did not greatly influence levels of transcripts of <italic> <span class="gene">GmEDS1,</span> </italic> but there was an increase in <italic> Gm PR1 </italic> transcripts. <italic> PR1 </italic> transcription is responsive to increased SA levels and is downstream of SA [ <xref ref-type="bibr" rid="B46"> 46 </xref> - <xref ref-type="bibr" rid="B48"> 48 </xref> ]. Overexpression expression of <italic> AtPAD4 </italic> in transgenic <italic> Arabidopsis </italic> conferred <span class="trait">resistance</span> to green peach aphid [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. This <span class="trait">resistance</span> did not require EDS1. Our data, extend understanding of <italic> AtPAD4 </italic> by showing that this <italic> Arabidopsis </italic> gene can be overexpressed in an economically important crop to confer <span class="trait">resistance</span> to two distinct genera of nematodes.
					</p>
				
					<p>&nbsp;&nbsp;
						The Arabidopsis <italic> <span class="gene">PAD4</span> </italic> gene (accession No. NM_115103) was amplified from cDNA from <italic> A. thaliana </italic> leaves using gene specific primers (Table&#x000a0; <xref ref-type="table" rid="T4"> 4 </xref> ) to <span class="trait">yield</span> a 1626-bp product. We added a CACC sequence to the 5&#x02032; end of forward primer to enable insertion of the amplicon into the pENTR vector (Invitrogen). The PCR product was purified using the E-Gel&#x000ae; Electrophoresis System (Invitrogen).
					</p>
				
					<p>&nbsp;&nbsp;
						The pENTR&#x02122; Directional TOPO Cloning Kit (Invitrogen, Carlsbad, CA) was used to clone <italic> AtPAD4 </italic> into the pENTR cloning vector. The resulting construct was transformed into competent <italic> Escherichia coli </italic> cells using One Shot Mach1&#x02122; T1R chemically competent <italic> E. coli </italic> (Invitrogen, Carlsbad, CA), and the plasmid was harvested using the QIAprep Spin Miniprep Kit (Qiagen, Valencia, CA). Presence and orientation of <italic> AtPAD4 </italic> were confirmed by DNA sequencing the positive samples using a 3130XL Genetic Analyzer (Applied Biosystems, Foster City, CA). <italic> AtPAD4 </italic> was moved from the pENTR vector to the plant overexpression vector pRAP15 (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ) using Invitrogen&#x02019;s Gateway technology. The pRAP15 vector has a tetracycline <span class="trait">resistance</span> gene (TetR)for bacterial selection engineered into a BstEII site that lies outside the left and right borders and the enhanced green fluorescent protein ( <italic> eGFP </italic> ) gene [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]driven by the rolD root promoter [ <xref ref-type="bibr" rid="B52"> 52 </xref> , <xref ref-type="bibr" rid="B53"> 53 </xref> ]for visual screening of transformed roots [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. The inserted <italic> AtPAD4 </italic> gene was driven by the figwort mosaic virus subgenomic transcript (FMV-sgt) promoter [ <xref ref-type="bibr" rid="B51"> 51 </xref> ], which exhibits strong, constitutive root expression. The cloning reaction was mediated by the Gateway LR Clonase&#x02122; II Enzyme Mix (Invitrogen,Carlsbad, CA)and involves crossing over between <italic> att </italic> R sites on pRAP15 and <italic> att </italic> L sites on pENTR, with <italic> AtPAD4 </italic> replacing the lethal <italic> ccd </italic> B gene that is used for bacterial selection. Colony PCR was used to confirm the presence and/or orientation of <italic> AtPAD4 </italic> and <italic> eGFP </italic> using the following primers sets: (1) FMV-F + <italic> <span class="gene">PAD4</span> </italic> -R; (2) <italic> eGFP </italic> -F + <italic> eGFP </italic> -R (Table&#x000a0; <xref ref-type="table" rid="T4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						RNA was extracted from three individual roots (100 mg each), roots transformed with the empty pRAP15 (as a control) having the strongest <italic> eGFP </italic> expression and representing independent transformation events using the Ultra Clean Plant RNA Isolation Kit (MOBIO, Carlsbad, CA). The RNA was treated with DNase I to remove genomic DNA. The RNA was utilized to synthesize single-stranded cDNA using reverse transcriptase (Invitrogen, Carlsbad, CA) and oligo dT primers, according to the manufacturer&#x02019;s instructions. All the primer sets were designed to flank a region that contains one intron to make sure that the expected size product was amplified from cDNA and not from genomic DNA. Primers (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ) were designed to be specific to the flanking region of the Arabidopsis <italic> <span class="gene">PAD4</span> </italic> ( <italic> AtPAD4 </italic> ) and to <span class="trait">yield</span> PCR-amplified fragments of approximately 150 bp. Also, the soybean ubiquitin-3 gene, GenBank accession D28123 used as a positive RT-PCR control for the experiment to confirm that the soybean RNA was present in all samples. And the soybean genes ( <italic> <span class="gene">GmPAD4</span> </italic> ; <italic> <span class="gene">GmEDS1</span> </italic> ) Phytozome accession <span class="gene">Glyma06g16290.1;</span> <span class="gene">Glyma06g19920.1</span> and ( <italic> <span class="gene">GmPR1</span> </italic> ) GenBank accession XM_003545723.1as related defense genes.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning of a defense-response-related cytochrome <span class="gene">P450</span> gene from tobacco.</h2>
			<h3>PMID:10682345  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Takemoto D(1), Hayashi M, Doke N, Nishimura M, Kawakita K.
				<br/> 
				年份：1999 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant defenses against pathogen attack involve a series of inducible responses that contribute to <span class="trait">resistance.</span> Tobacco leaves injected with HWC (hyphal wall components prepared from Phytophthora infestans) elicitor showed typical defense responses, including the induction of localized necrosis and the accumulation of pathogenesis-related proteins. In order to elucidate the molecular mechanisms by which plant defense systems are activated, we screened tobacco plants for genes differentially expressed in response to HWC. We performed differential screening by RT-PCR with random primers and obtained PCR products specific to HWC-treated leaf RNA. Northern hybridization using the PCR products as probes confirmed that one transcript was actually induced by HWC treatment. As the deduced amino acid sequence of this clone showed the highest degree of similarity to elicitor-induced soybean cytochrome <span class="gene">P450</span> CYP82A4, it was designated CYP82E1. The expression of CYP82E1 was strongly induced in tobacco by the soybean pathogen <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pv. glycinea (nonpathogenic on tobacco), but it was activated only slightly and in a delayed fashion by the tobacco pathogen P. <span class="trait">syringae</span> pv. tabaci (pathogenic on tobacco), implying that the product of CYP82E1 may be involved in disease <span class="trait">resistance</span> in tobacco.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A draft genome sequence and functional screen reveals the repertoire of type III secreted proteins of <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic> pathovar <italic> tabaci </italic> 11528</h2>
			<h3>PMID:19703286</h3>
				
					<h3>PMCID:2745422</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Studholme </surname>  <given-names> David J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> david.studholme@tsl.ac.uk </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Ibanez </surname>  <given-names> Selena Gimenez </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> selena.gimenez-ibanez@tsl.ac.uk </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> MacLean </surname>  <given-names> Daniel </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dan.maclean@tsl.ac.uk </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Dangl </surname>  <given-names> Jeffery L </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> dangl@email.unc.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Jeff H </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> changj@cgrb.oregonstate.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Rathjen </surname>  <given-names> John P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> john.rathjen@tsl.ac.uk </email>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 8 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In sequenced strains of <italic> P. <span class="trait">syringae</span> </italic> , the gene cluster encoding the T3SS apparatus is flanked by collections of effector genes termed the exchangeable effector locus (EEL) and the conserved effector locus (CEL). Together, these three genetic components comprise the <italic> Hrp </italic> pathogenicity island [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. A core set of <italic> hop </italic> genes is located in the <italic> Hrp </italic> pathogenicity island [ <xref ref-type="bibr" rid="B44"> 44 </xref> ], which is highly conserved between <italic> Pta </italic> 11528 and <italic> Pph </italic> 1448A (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), except that in <italic> Pta </italic> 11528 there is a deletion in <italic> hrpZ1 </italic> and an insertion in the <italic> hrpV </italic> - <italic> <span class="gene">hrcU</span> </italic> intergenic region. The core <italic> hop </italic> genes <italic> avrE1 </italic> , <italic> hopAH2 </italic> , <italic> hopAJ2 </italic> , <italic> hopAK1 </italic> , <italic> hopAN1 </italic> , <italic> hopI1 </italic> , <italic> hopJ1, hopX1 </italic> and <italic> hrpK1 </italic> are conserved in <italic> Pta </italic> 11528 and encode intact full-length proteins. <italic> Pta </italic> 11528 encodes a full-length HrpW1 protein, albeit with insertions of 69 and 12 nucleotides relative to the <italic> Pph </italic> 1448A sequence. However, there is a large deletion in <italic> hrpZ1 </italic> that likely renders it non-functional and <italic> hopAF1 </italic> is completely absent.
					</p>
				
					<p>&nbsp;&nbsp;
						In plasmid pMA4326B from <italic> P. <span class="trait">syringae</span> </italic> pathovar <italic> maculicola </italic> ( <italic> Pma </italic> ), the <italic> hopW1 </italic> effector gene is immediately adjacent to a three-gene cassette comprising a resolvase, an integrase and <italic> <span class="gene">exeA</span> </italic> . This cassette is also found in plasmids and chromosomes of several human-pathogenic Gram-negative bacteria [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. We found a homologue of this cassette along with a <italic> hopW1 </italic> homologue on supercontig 955 of the <italic> Pta </italic> 11528 genome assembly. Stavrinides and Guttman [ <xref ref-type="bibr" rid="B58"> 58 </xref> ] proposed that the boundaries of the cassette lay upstream of the resolvase and upstream of <italic> hopW1 </italic> . The presence of this four-gene unit in a completely different location in <italic> Pta </italic> 11528 is indeed consistent with the hypothesis that it represents a discrete mobile unit.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The conserved upstream region of <italic> lscB/C </italic> determines expression of different levansucrase genes in plant pathogen <italic> <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> </italic></h2>
			<h3>PMID:24670199</h3>
				
					<h3>PMCID:3973379</h3>
					<h4>期刊:BMC Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Khandekar </surname>  <given-names> Shaunak </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> skhandekar@jacobs-alumni.de </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Srivastava </surname>  <given-names> Abhishek </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> abhishek.srivastava@igb-berlin.de </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Pletzer </surname>  <given-names> Daniel </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> d.pletzer@jacobs-university.de </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Stahl </surname>  <given-names> Antje </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> a.stahl@jacobs-university.de </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Ullrich </surname>  <given-names> Matthias S </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> m.ullrich@jacobs-university.de </email>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Total RNA was isolated by acid phenol/chloroform extraction as described previously [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. The <span class="trait">yield</span> and the purity of RNA were determined by measuring absorption at 260&#x000a0;nm. Total mRNA samples were treated with TURBO DNA-free (Applied Biosystems, Darmstadt, Germany) to remove remaining traces of genomic DNA as described by the manufacturer&#x02019;s recommendation. SYBR-green based qRT-PCR was performed with 5&#x000a0;ng RNA template and 100&#x000a0;&#x003bc;M primer with QuantiTect SYBR Green one-step RT-PCR Kit (Qiagen, Hilden, Germany) according to the manufacturer&#x02019;s instructions. The thermocycler program comprised an initial step of 95&#x000b0;C for 15&#x000a0;min followed by 40&#x000a0;cycles of 95&#x000b0;C for 30&#x000a0;s, 58&#x000b0;C for 30&#x000a0;s, 72&#x000b0;C for 30&#x000a0;s. Reactions were performed with biological triplicates in a Mastercycler ep realplex2 real-time PCR system (Eppendorf, Hamburg, Germany) as described by the manufacturer using their universal program. Reactions with no addition of reverse transcriptase served as negative control and proved the absence of DNA contamination. Specificity of amplification was assessed by analyzing the melting curve of the amplification product. Primers to amplify <italic> lscB </italic> were used for constructs <italic> lscB </italic> and <italic> lscA </italic> <sub> <italic> Up </italic> </sub> <italic> B </italic> while primers to amplify <italic> lscA </italic> were used for constructs <italic> lscA </italic> , <italic> lscB </italic> <sub> <italic> UpN </italic> </sub> <italic> A </italic> and <italic> lscB </italic> <sub> <italic> Up </italic> </sub> <italic> A. </italic> All the results were normalized to amplification of the cDNA of <italic> <span class="gene">gyrA</span> </italic> (PSPPH3667) as described previously [ <xref ref-type="bibr" rid="B43"> 43 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pseudomonas <span class="trait">syringae</span> effector avrB confers soybean cultivar-specific avirulence on Soybean mosaic virus adapted for transgene expression but effector avrPto does not.</h2>
			<h3>PMID:16570660  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang L(1), Eggenberger A, Hill J, Bogdanove AJ.
				<br/> 
				年份：2006 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean mosaic virus (SMV) was adapted for transgene expression in soybean and used to examine the function of avirulence genes avrB and avrPto of <span class="trait">Pseudomonas</span> <span class="trait">syringae</span> pvs. glycinea and tomato, respectively. A cloning site was introduced between the <span class="gene">P1</span> and HC-Pro genes in 35S-driven infectious cDNAs of strains SMV-N and SMV-G7. Insertion of the uidA gene or the green fluorescent protein gene into either modified cDNA and bombardment into primary leaves resulted in systemic expression that reflected the pattern of viral movement into uninoculated leaves. Insertion of avrB blocked symptom development and detectable viral movement in cv. Harosoy, which carries the Rpg1-b <span class="trait">resistance</span> gene corresponding to avrB, but not in cvs. Keburi or Hurrelbrink, which lack Rpg1-b. In Keburi and Hurrelbrink, symptoms caused by SMV carrying avrB appeared more quickly and were more severe than those caused by the virus without avrB. Insertion of avrPto enhanced symptoms in Harosoy, Hurrelbrink, and Keburi. This result was unexpected because avrPto was reported to confer avirulence on P. <span class="trait">syringae</span> pv. glycinea inoculated to Harosoy. We inoculated Harosoy with P <span class="trait">syringae</span> pv. glycinea expressing avrPto, but observed no hypersensitive reaction, avrPto-dependent induction of pathogenesis-related protein la, or limitation of bacterial population growth. In Hurrelbrink, avrPto enhanced bacterial multiplication and exacerbated symptoms. Our results establish SMV as an expression vector for soybean. They demonstrate that <span class="trait">resistance</span> triggered by avrB is effective against SMV, and that avrB and avrPto have general virulence effects in soybean. The results also led to a reevaluation of the reported avirulence activity of avrPto in this plant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome Sequence Analyses of <italic> <span class="trait">Pseudomonas</span> savastanoi </italic> pv. <italic> glycinea </italic> and Subtractive Hybridization-Based Comparative Genomics with Nine Pseudomonads</h2>
			<h3>PMID:21304594</h3>
				
					<h3>PMCID:3029378</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Qi </surname>  <given-names> Mingsheng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dongping </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bradley </surname>  <given-names> Carl A. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Youfu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 12 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genome comparison among genomes of <italic> P. savastanoi </italic> pv. <italic> glycinea </italic> and other sequenced <italic> P. <span class="trait">syringae</span> </italic> strains was carried out using mGenomeSubstractor web server <xref ref-type="bibr" rid="pone.0016451-Shao1"> &#x0005b;72&#x0005d; </xref> . Genomes were also aligned using a genome-wide Blast comparison and visualized through <span class="gene">ACT</span> <xref ref-type="bibr" rid="pone.0016451-Carver1"> &#x0005b;85&#x0005d; </xref> . Graphical views of genome alignments were generated using CGView <xref ref-type="bibr" rid="pone.0016451-Grant2"> &#x0005b;86&#x0005d; </xref> . Protein sequences of known effectors from Hop Database ( <ext-link ext-link-type="uri" xlink:href="http://pseudomonas-syringae.org/"> http://pseudomonas-syringae.org/ </ext-link> ) were used for effectors prediction <xref ref-type="bibr" rid="pone.0016451-Lindeberg3"> &#x0005b;87&#x0005d; </xref> . Effector prediction was also performed using the EffectiveT3 program ( <ext-link ext-link-type="uri" xlink:href="http://www.effectors.org/"> http://www.effectors.org/ </ext-link> ) <xref ref-type="bibr" rid="pone.0016451-Arnold1"> &#x0005b;88&#x0005d; </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of extra-cellular levansucrase in <italic> <span class="trait">Pseudomonas</span> </italic> <span class="trait">syringae</span> is controlled by the <italic> in planta </italic> fitness-promoting metabolic repressor HexR</h2>
			<h3>PMID:25886911</h3>
				
					<h3>PMCID:4357207</h3>
					<h4>期刊:BMC Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Mehmood </surname>  <given-names> Amna </given-names>  </name>  <address>  <email> a.mehmood@jacobs-university.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Abdallah </surname>  <given-names> Khaled </given-names>  </name>  <address>  <email> k.abdallah@jacobs-university.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khandekar </surname>  <given-names> Shaunak </given-names>  </name>  <address>  <email> skhandekar@jacobs-alumni.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhurina </surname>  <given-names> Daria </given-names>  </name>  <address>  <email> dzhurina@jacobs-alumni.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Srivastava </surname>  <given-names> Abhishek </given-names>  </name>  <address>  <email> asrivastava@jacobs-alumni.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Al-Karablieh </surname>  <given-names> Nehaya </given-names>  </name>  <address>  <email> n.alKarablieh@ju.edu.jo </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alfaro-Espinoza </surname>  <given-names> Gabriela </given-names>  </name>  <address>  <email> m.alfaroespinoza@jacobs-university.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pletzer </surname>  <given-names> Daniel </given-names>  </name>  <address>  <email> d.pletzer@jacobs-university.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Ullrich </surname>  <given-names> Matthias S </given-names>  </name>  <address>  <email> m.ullrich@jacobs-university.de </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Molecular Life Science Research Center, Jacobs University Bremen, Campus Ring 1, Bremen, 28759 Germany  </aff>  <aff id="Aff2">  <label/> Hamdi Mango Center for Scientific Research, The University of Jordan, P.O. Box 13507, Amman, 11942 Jordan  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose is the most abundant plant storage sugar [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. Bacterial <italic> in planta </italic> and <italic> in vitro </italic> growth analyses indicated that the substrate of Lsc, sucrose, and in consequence its enzymatic product, glucose, seem to be major nutrient sources for <italic> P. <span class="trait">syringae</span> </italic> during <italic> in planta </italic> growth. This is in line with previous findings reporting high molecular abundances of sucrose in bean plant&#x02019;s apoplastic fluids [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Consequently, expression and secretion of Lsc might be a fitness factor for the <italic> in planta </italic> life of <italic> P. <span class="trait">syringae</span> </italic> . Along the course of the experiment, the total populations of PG4180 wild type and its <italic> hexR </italic> mutant varied but were not significantly different from each other. This variation reflects a feature that is usually observed during assessment of bacterial populations [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. LL Kinkel, M Wilson and SE Lindow [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ] showed that two leaves of the same plant species might vary by over 10-fold in their total epiphytic bacteria. Moreover, a study by <span class="gene">SS</span> Hirano and CD Upper [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ] also demonstrated that regardless of the geographic area, plant species, or time scale tested, variations in population sizes of <italic> P. <span class="trait">syringae</span> </italic> are common. Many factors such as availability of nutrients, bacterial immigration [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ], and ability of bacteria to tolerate environmental stresses on leaf surfaces [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ] contribute to the observed variations.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>