<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Reaction to Phytophthora sojae infection</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Reaction to Phytophthora sojae infection</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Isolation and characterization of a pathogenesis-related protein 10 gene <span class="gene">(GmPR10)</span> with induced expression in soybean (Glycine max) during <span class="trait">infection</span> with <span class="trait">Phytophthora</span> <span class="trait">sojae.</span></h2>
			<h3>PMID:24737571  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu P(1), Jiang L, Wu J, Li W, Fan S, Zhang S.
				<br/> 
				年份：2014 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In previous study, a cDNA library enriched for mRNAs encoding ESTs that increased in abundance during <span class="trait">infection</span> with <span class="trait">Phytophthora</span> <span class="trait">sojae</span> was constructed by suppression subtractive hybridization from leaf tissues of a high resistant soybean, and an EST homologous <span class="trait">to</span> the class 10 of pathogenesis-related (PR) proteins was identified <span class="trait">to</span> be up-regulated by microarray and real-time PCR. Here, the full-length cDNA (termed <span class="gene">GmPR10,</span> GenBank accession number FJ960440; ADC31789.1) of the EST was isolated by rapid amplification of cDNA ends, and contains an open reading frame of 474 bp. The <span class="gene">GmPR10</span> protein included a "P-loop'' motif. The constitutive transcript abundance of <span class="gene">GmPR10</span> in soybean was the highest in leaves, followed by roots and stems. Further analysis showed that <span class="gene">GmPR10</span> mRNA abundance was increased during <span class="trait">infection</span> with P. <span class="trait">sojae</span> following leaf treatments with gibberellin (GA3), hydrogen peroxide (H2O2), salicylic acid (SA), and abscisic acid (ABA). The dialytically renatured <span class="gene">GmPR10</span> protein significantly inhibited P. <span class="trait">sojae</span> hyphal growth and exhibited RNase activity. Transgenic tobacco and soybean plants overexpressing <span class="gene">GmPR10</span> showed increased <span class="trait">resistance</span> <span class="trait">to</span> P. nicotianae Breda and P. <span class="trait">sojae,</span> respectively. These results suggest that the <span class="gene">GmPR10</span> protein plays an important role in host defense against P. <span class="trait">sojae</span> <span class="trait">infection.</span> <span class="trait">To</span> the best of our knowledge, this is the first report on the functional characterization of a PR10 protein from soybean in defense against P. <span class="trait">sojae.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Myb Transcription Factor of <italic> <span class="trait">Phytophthora</span> <span class="trait">sojae</span> </italic> , Regulated by MAP Kinase PsSAK1, Is Required for Zoospore Development</h2>
			<h3>PMID:22768262</h3>
				
					<h3>PMCID:3386981</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Meng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tao </surname>  <given-names> Kai </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Wenwu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Aining </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiaoyun </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Liang </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Suomeng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Xiaobo </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yuanchao </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						( <bold> A </bold> ) The nuclei distributions in sporangium of early stage of silenced-lines or controls, take parental control P6497 as an example; and the zoosporogenesis stages of P6497, <italic> PsMYB1 </italic> -silenced transformant T54, and <italic> PsSAK1 </italic> -silenced line T31 were observed with DAPI staining of the nuclei. ( <bold> B </bold> ) Cleavage-system degeneration during germ tube formation in test strains was seen with FM4-64 <span class="trait">to</span> selectively visualize the plasma membrane. The bars at the bottom of (A-B) represent 50 &#x000b5;m. ( <bold> C </bold> ) Nuclear distribution in zoospores. The <span class="gene">bar</span> at the base of the panel represents 20 &#x000b5;m.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An Oomycete CRN Effector Reprograms Expression of Plant <italic> HSP </italic> Genes by Targeting their Promoters</h2>
			<h3>PMID:26714171</h3>
				
					<h3>PMCID:4695088</h3>
					<h4>期刊:PLoS Pathogens</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Tianqiao </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Zhenchuan </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Danyu </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Qi </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wanlin </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Su </surname>  <given-names> Liming </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Tingyue </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Meixiang </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yuanchao </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dou </surname>  <given-names> Daolong </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Some oomycete effectors also act inside host cell nuclei. An RxLR effector from <italic> H </italic> . <italic> arabidopsidis </italic> (HaRxL44) localizes <span class="trait">to</span> the nucleus of plant cells and interacts with the host mediator subunit MED19a, an important regulator of gene transcription, <span class="trait">to</span> reduce expression of genes regulated by SA [ <xref rid="ppat.1005348.ref021" ref-type="bibr"> 21 </xref> ]. Bioinformatics and localization studies have showed that a number of oomycete CRN effectors target host cell nuclei <span class="trait">to</span> cause cell death and/or mediate virulence [ <xref rid="ppat.1005348.ref006" ref-type="bibr"> 6 </xref> , <xref rid="ppat.1005348.ref022" ref-type="bibr"> 22 </xref> ]. CRN effectors were initially identified based on their cell death-inducing activity in plants [ <xref rid="ppat.1005348.ref023" ref-type="bibr"> 23 </xref> ]. They form highly expanded gene families in oomycete genomes and diversify via gene duplication and chimeric recombination [ <xref rid="ppat.1005348.ref024" ref-type="bibr"> 24 </xref> , <xref rid="ppat.1005348.ref025" ref-type="bibr"> 25 </xref> ]. The functional mechanisms of only three CRN effectors have been discovered <span class="trait">to</span> date. The <italic> P </italic> . <italic> infestans </italic> effector CRN8, which has high sequence similarity with serine/threonine kinases, may suppress plant defense and cause cell death [ <xref rid="ppat.1005348.ref026" ref-type="bibr"> 26 </xref> ]. <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> effector pair, <span class="gene">CRN63</span> and CRN115, manipulate host H <sub> 2 </sub> O <sub> 2 </sub> homeostasis and promote pathogenicity by interacting with plant catalases [ <xref rid="ppat.1005348.ref027" ref-type="bibr"> 27 </xref> ]. All three CRN effectors localize <span class="trait">to</span> plant cell nuclei <span class="trait">to</span> exert their functions [ <xref rid="ppat.1005348.ref026" ref-type="bibr"> 26 </xref> , <xref rid="ppat.1005348.ref027" ref-type="bibr"> 27 </xref> ]. This study focuses on another <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> CRN effector that reduces expression of plant heat shock protein (Hsp)-encoding genes by directly targeting their promoters.
					</p>
				
					<p>&nbsp;&nbsp;
						Although PsCRN108 was predicted <span class="trait">to</span> contain a signal peptide by the SignalP3.0 HMM algorithm, the SignalP3.0 NN algorithm and the SignalP4.0 algorithms did not agree with this prediction. Therefore, <span class="trait">to</span> functionally validate the predicted signal peptide, we used a yeast genetic assay that is based on the requirement for secretion of invertase for yeast growth on raffinose medium [ <xref rid="ppat.1005348.ref042" ref-type="bibr"> 42 </xref> ]. The N-terminal regions of PsCRN108 (1&#x02013;21 amino acids [aa]) and <span class="gene">Avr1b</span> (1&#x02013;23 aa, positive control) were fused in-frame <span class="trait">to</span> yeast invertase within the vector pSUC2 [ <xref rid="ppat.1005348.ref042" ref-type="bibr"> 42 </xref> ]. Both constructs enabled the invertase mutant yeast strain YTK12 <span class="trait">to</span> grow on YPRAA medium and <span class="trait">to</span> catalyze conversion of 2, 3, 5-triphenyltetrazolium chloride (TTC) <span class="trait">to</span> the insoluble red-colored triphenylformazan ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1A </xref> ). In contrast, the negative controls (yeast strain YTK12 and YTK12 carrying the pSUC2 vector) could not grow on YPRAA plates, and TTC-treated cultures remained colorless ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1A </xref> ). Thus, it was concluded that PsCRN108 carries a functional secretory signal peptide.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Yeast invertase secretion assay of the predicted signal peptide of PsCRN108. The predicted signal peptide sequences and the following two amino acids (1&#x02013;23) of PsAvr1b was used as the positive control <span class="trait">to</span> assay the predicted signal peptide sequences and the following four amino acids (1&#x02013;21) of PsCRN108. CMD-W (minus Trp) plates were used <span class="trait">to</span> select yeast strain YTK12 carrying the pSUC2 vector. YPRAA media was used <span class="trait">to</span> indicate invertase secretion. An enzymatic activity test based on reduction of TTC <span class="trait">to</span> red-colored formazan was used <span class="trait">to</span> confirm invertase secretion. <bold> (B) </bold> Western blot analysis of proteins in culture supernatants of the <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transgenic lines. PsCRN108:Avr1bCt indicates the full length of PsCRN108 fused <span class="trait">to</span> the <span class="gene">Avr1b</span> C-terminal regions (Avr1bCt aa 66&#x02013;138, lacking the predicted signal peptide and RxLR host translocation domain). A <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> line expressing Avr1b:mRFP was used as the positive control. The His tag was attached <span class="trait">to</span> the C-terminus of all the fusion proteins. The non-transformed <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> P6497 line was used as the antibody-specificity control. Proteins extracted from mycelia (M) and culture supernatants (S) were analyzed by Western blotting using anti-His or anti-Actin antibodies. The protein sizes are expressed in kDa. (C) Western blot analysis of proteins isolated from <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transgenic lines during <span class="trait">infection.</span> Avr1bCt:mRFP indicates <span class="gene">Avr1b</span> C-terminal regions fused <span class="trait">to</span> mRFP and is used as a control. The infected hypocotyl tissues (HARO13) including mycelia of <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transgenic lines were collected at 12 hpi for protein extraction, and then Western blotting was performed using anti-His antibodies. The protein sizes are expressed in kDa. PS, Ponceau S staining. (D) Hypocotyl-inoculation assay using two soybean cultivars. Two soybean cultivars, HARO13 ( <italic> Rps1b </italic> ) and Williams ( <italic> rps </italic> ), were inoculated with the indicated <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transgenic lines. Photographs were taken 2 dpi. Data (e.g. 2/83) indicate the number (2) of dead plants out of the total number (83) of plants tested. (E) Western blot analysis of GFP fusion proteins transiently expressed in <italic> N </italic> . <italic> benthamiana </italic> . The indicated proteins were extracted from leaves at 48 h after agoinfiltration and detected using anti-GFP antibodies. GFP:PsCRN108-NLSm and GFP:PsCRN108-HhHm are described in Fig 3. Dotted line indicates lanes not adjacent on the gel. The protein sizes are expressed in kDa. PS, Ponceau S staining (F) Nuclear localization of PsCRN108 in <italic> N </italic> . <italic> benthamiana </italic> . Epidermal cells of <italic> N </italic> . <italic> benthamiana </italic> leaves transiently expressing GFP alone or GFP:PsCRN108 were observed using confocal microscopy at 2 days post infiltration. DAPI staining was used <span class="trait">to</span> visualize the nuclei. Scale bars = 50 &#x003bc;m (upper panel) and 10 &#x003bc;m (lower panel).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">To</span> confirm directly whether this protein is secreted from <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> and also whether it is translocated into host cells during <span class="trait">infection,</span> we replaced the N-terminus of <span class="gene">Avr1b</span> (1&#x02013;65 aa, including signal peptide and RxLR-dEER domain) with full-length PsCRN108 <span class="trait">to</span> generate the His-tagged fusion protein (PsCRN108:Avr1bCt), which was then expressed in <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transformants. First, PsCRN108:Avr1bCt secretion from <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> was demonstrated by detecting the protein in culture supernatants of the transgenic lines by western blotting with an anti-His-tag antibody; an Avr1b:mRFP fusion line was used as a positive control and wild type (P6497) as a negative control. PsCRN108:Avr1bCt and Avr1b:mRFP proteins were detected in culture supernatants, as well as in the hyphae of the respective transformants ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1B </xref> ). However, the control cytoplasmic protein, actin, was found only in the hyphae ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1B </xref> ). The results support that PsCRN108 is secreted from <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> . Second, the translocation into host cells of the PsCRN108 effector was evaluated by analyzing the avirulence/virulence phenotype of <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transformants on a soybean cultivar (HARO13) carrying the <italic> Rps1b </italic> <span class="trait">resistance</span> gene. Successful translocation into the cells of Rps1b plants is expected <span class="trait">to</span> prevent <span class="trait">infection,</span> as recognition of <span class="gene">Avr1b</span> by Rps1b inside the cell results in a successful defense response [ <xref rid="ppat.1005348.ref005" ref-type="bibr"> 5 </xref> ]. This method has previously been used successfully <span class="trait">to</span> determine host translocation by oomycete effectors [ <xref rid="ppat.1005348.ref006" ref-type="bibr"> 6 </xref> , <xref rid="ppat.1005348.ref009" ref-type="bibr"> 9 </xref> ]. The fusion proteins were expressed at the expected size in the transgenic <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> lines ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1B </xref> ) and were also detected during <span class="trait">infection</span> ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1C </xref> ). The transformants expressing PsCRN108:Avr1bCt were unable <span class="trait">to</span> infect hypocotyls of soybean cultivar HARO13, but could infect soybean cultivar Williams which lacks the <italic> Rps1b </italic> gene ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1D </xref> ). Similarly, the positive control line expressing Avr1b:mRFP fusion proteins [ <xref rid="ppat.1005348.ref009" ref-type="bibr"> 9 </xref> ] was avirulent on HARO13 but virulent on Williams, while the recipient WT strain (P6497) and the transgenic lines expressing Avr1bCt:mRFP (as negative controls) were virulent ( <xref rid="ppat.1005348.g001" ref-type="fig"> Fig 1D </xref> ). The results indicate that PsCRN108 could functionally replace the host cell entry domain of Avr1b. Taken together, these results indicate that PsCRN108 can be secreted from <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> and enter host cells during <span class="trait">infection.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Relative transcript levels of <italic> AtHSP90 </italic> . <italic> 1 </italic> in transgenic <italic> Arabidopsis </italic> leaves. The transcript levels of <italic> AtHSP90 </italic> . <italic> 1 </italic> ( <italic> AT5G52640 </italic> ) in stable transgenic <italic> Arabidopsis </italic> leaves expressing <italic> GFP </italic> or <italic> PsCRN108 </italic> were measured by qRT-PCR using RNA samples extracted from <italic> P </italic> . <italic> capsici </italic> inoculated leaves (10 hpi) and mock-treated leaves. The relative transcript levels during <span class="trait">infection</span> were normalized <span class="trait">to</span> those of mock-treated leaves expressing <italic> GFP </italic> using <italic> Actin2 </italic> as an internal reference. Values are means &#x000b1; SEM of three independent biological replicates ( <italic> ** </italic> , P&#x0003c;0.01; <italic> t </italic> -test). (B) Relative transcript levels of <italic> NbHsp90-1 </italic> . The transcript levels of <italic> NbHsp90-1 </italic> in <italic> N </italic> . <italic> benthamiana </italic> leaves expressing <italic> GFP </italic> , <italic> PsCRN108 </italic> or the indicated <italic> PsCRN108 </italic> variants were measured by qRT-PCR using RNA samples extracted from <italic> P </italic> . <italic> capsici </italic> inoculated leaves (10 hpi) and mock-treated leaves. The relative transcript levels during <span class="trait">infection</span> were normalized <span class="trait">to</span> those of mock-treated leaves expressing GFP using <italic> NbEF1&#x003b1; </italic> gene as an internal reference. Values are means &#x000b1; SEM of three independent biological replicates ( <italic> ** </italic> , P&#x0003c;0.01 compared with GFP; Dunnett's test). (C) Relative transcript levels of <italic> <span class="gene">GmHsp90A1</span> </italic> . Hypocotyls of etiolated soybean cultivar (Williams) were challenged with zoospores of the indicated <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> lines, and RNA was extracted at 10 hpi. The transcript levels of <italic> <span class="gene">GmHsp90A1</span> </italic> in soybean inoculated with transgenic <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> lines were normalized <span class="trait">to</span> levels of those of mock-treated soybean using <italic> ACT20 </italic> gene as an internal reference. Values are the means &#x000b1; SEM of three independent biological replicates (**, P&#x0003c;0.01; *, P&#x0003c;0.05 compared with the WT; Dunnett's test). (D) Schematic of the constructs used in the GUS activity assay. A -2000 bp promoter region of <italic> AtHSP90 </italic> . <italic> 1 </italic> were fused <span class="trait">to</span> <italic> GUS </italic> . &#x0201c;ATG&#x0201d; is the translational start site for the <italic> GUS </italic> gene. The 35S min promoter fused <span class="trait">to</span> <italic> GUS </italic> (p35S min:GUS) was used for normalization of the relative GUS activity. (E) Relative GUS enzyme activity in <italic> N </italic> . <italic> benthamiana </italic> . The constructs of pAtHSP90.1:GUS and p35S min:GUS were transiently introduced into <italic> N </italic> . <italic> benthamiana </italic> leaves together with constructs encoding GFP, PsCRN108 or the indicated PsCRN108 variants. The leaves were immersed in a solution containing <italic> P </italic> . <italic> capsici </italic> zoospores 48 h post infiltration, and the protein was extracted at 3 hpi. GUS activity levels were measured using a fluorometric assay. Each column represents the ratio from the pAtHSP90.1:GUS construct relative <span class="trait">to</span> that from the p35S min:GUS construct. Values are the mean &#x000b1; SEM of three independent biological replicates with three technological replicates each ( <italic> ** </italic> , P&#x0003c;0.01 compared with GFP; Dunnett's test).
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 4 <italic> HSP </italic> gene families, HSP90&#x02019;s have been shown <span class="trait">to</span> have a direct involvement in plant defense [ <xref rid="ppat.1005348.ref032" ref-type="bibr"> 32 </xref> &#x02013; <xref rid="ppat.1005348.ref035" ref-type="bibr"> 35 </xref> ]. <span class="trait">To</span> further confirm PsCRN108-mediated suppression of <italic> <span class="gene">HSP90</span> </italic> expression, we tested the effect of PsCRN108 on the expression of the <italic> NbHsp90-1 </italic> gene in <italic> N </italic> . <italic> benthamiana </italic> and of <italic> <span class="gene">GmHsp90A1</span> </italic> in soybean, respectively. After <italic> P </italic> . <italic> capsici </italic> <span class="trait">infection,</span> <italic> N </italic> . <italic> benthamiana </italic> expressing GFP:PsCRN108 showed lower <italic> NbHsp90-1 </italic> gene transcript levels compared with <italic> N </italic> . <italic> benthamiana </italic> expressing GFP ( <xref rid="ppat.1005348.g004" ref-type="fig"> Fig 4B </xref> ). In the case of <italic> <span class="gene">GmHsp90A1</span> </italic> , we compared the transcript levels of the gene in soybean infected with <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> WT, T17 or <italic> PsCRN108 </italic> -silenced lines (ST18 and ST19). As shown in <xref rid="ppat.1005348.g004" ref-type="fig"> Fig 4C </xref> , soybean plants infected with ST18 and ST19 showed markedly higher <italic> <span class="gene">GmHsp90-A1</span> </italic> transcript levels compared with plants infected with WT and T17, indicating that silencing of <italic> PsCRN108 </italic> in <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> may reduce its ability <span class="trait">to</span> suppress host <italic> <span class="gene">GmHsp90-A1</span> </italic> gene transcription. Collectively, the results suggest that PsCRN108 can suppress transcription of <italic> HSP </italic> genes in <italic> Arabidopsis </italic> , <italic> N </italic> . <italic> benthamiana </italic> and soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Since PsCRN108 appeared <span class="trait">to</span> suppress <italic> AtHSP90 </italic> . <italic> 1 </italic> promoter-mediated GUS expression ( <xref rid="ppat.1005348.g004" ref-type="fig"> Fig 4E </xref> ) and contains a functionally essential DNA-binding motif ( <xref rid="ppat.1005348.s001" ref-type="supplementary-material"> S1A </xref> and <xref rid="ppat.1005348.s002" ref-type="supplementary-material"> S2 </xref> Figs), we speculated that PsCRN108 might directly target <italic> HSP </italic> promoters. As one test of this hypothesis, we carried out chromatin immunoprecipitation (ChIP) on <italic> PsCRN108 </italic> - transgenic <italic> Arabidopsis </italic> plants and <italic> GFP </italic> - transgenic plants with and without pathogen <span class="trait">infection</span> using an anti-GFP antibody. The enrichment of four independent promoter segments each of four <italic> AtHSP </italic> genes ( <italic> AtHSP20 </italic> , <italic> AT1G53540 </italic> ; <italic> AtHSP70 </italic> , <italic> AT3G12580 </italic> ; <italic> AtHSP90 </italic> . <italic> 1 </italic> , <italic> AT5G52640 </italic> ; <italic> AtHSP101 </italic> , <italic> AT1G74310 </italic> ) in immunoprecipitated DNA was measured by qPCR, using four primer pairs <span class="trait">to</span> target each promoter segment ( <xref rid="ppat.1005348.s007" ref-type="supplementary-material"> S7 Fig </xref> ). There was no apparent enrichment of any segments in the <italic> GFP </italic> -transgenic lines with or without pathogen <span class="trait">infection</span> ( <xref rid="ppat.1005348.g005" ref-type="fig"> Fig 5A </xref> ). In contrast, the S1 and S2 segments of <italic> AtHSP20 </italic> and <italic> AtHSP70 </italic> , and the S1 segment of <italic> AtHSP90 </italic> . <italic> 1 </italic> and <italic> AtHSP101 </italic> were highly enriched (&#x0003e;5-fold) in seedlings expressing GFP:PsCRN108. Also the S3 segment of <italic> AtHSP20 </italic> , and the S2 and S3 segments of <italic> AtHSP90 </italic> . <italic> 1 </italic> and <italic> AtHSP101 </italic> were moderately enriched (&#x0003e; 3-fold). However, the S4 segments of all <italic> HSP </italic> genes and all four segments of a negative gene ( <italic> AT5G56010 </italic> , an <italic> <span class="gene">HSP90</span> </italic> gene which is not significantly down-regulated by PsCRN108 and contains no HSE) were not enriched in precipitated chromatin from <italic> PsCRN108 </italic> plants ( <xref rid="ppat.1005348.g005" ref-type="fig"> Fig 5A </xref> ). Thus, PsCRN108 binds <span class="trait">to</span> the chromatin of the four PsCRN108-regulated <italic> HSP </italic> promoters evaluated, with highest enrichment of regions closest <span class="trait">to</span> HSE&#x02019;s.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Enrichment of <italic> AtHSP </italic> gene promoters as determined by ChIP assays. Transgenic <italic> Arabidopsis </italic> plants expressing GFP or GFP:PsCRN108 infected with <italic> P </italic> . <italic> capsici </italic> at 10 hpi (I) or without (N) <span class="trait">infection</span> were used for ChIP assays. Specific primers were designed for four fragments (S1-4 are described in <xref rid="ppat.1005348.s007" ref-type="supplementary-material"> S7 Fig </xref> ) in the five selected <italic> Arabidopsis HSP </italic> promoters ( <italic> AtHSP20 </italic> , <italic> AT1G53540 </italic> ; <italic> AtHSP70 </italic> , <italic> AT3G12580 </italic> ; <italic> AtHSP90 </italic> , <italic> AT5G52640 </italic> ; <italic> AtHSP101 </italic> , <italic> AT1G74310 </italic> ; and <italic> AT5G56010 </italic> , another <italic> <span class="gene">HSP90</span> </italic> gene used as a negative control). qPCR analysis was performed on immunoprecipitated DNA using anti-GFP antibodies. Values for the ChIP samples were firstly normalized <span class="trait">to</span> the input control and then divided by the no-antibody control <span class="trait">to</span> obtain the fold enrichment values. Values are the means &#x000b1; SEM of three independent biological replicates (**, P&#x0003c;0.01; *, P&#x0003c;0.05 compared with GFP without <span class="trait">infection;</span> Dunnett&#x02019;s test). (B) Yeast one-hybrid (Y1H) assay of PsCRN108 with the <italic> AtHSP90 </italic> . <italic> 1 </italic> promoter region and HSE. A 500-bp (-1 <span class="trait">to</span> -500) fragment of the <italic> AtHSP90 </italic> . <italic> 1 </italic> promoter, a 19 bp HSE (CCAGAAGCTTCCAGAAGCC), or a 19 bp HSE mutant (HSEm, CCAtAAGCTTaCAtAAGCC) were integrated into the genome of yeast upstream of the <italic> AUR1-C </italic> gene <span class="trait">to</span> produce yeast bait strains. The lowest concentrations of Aureobasidin A (AbA) that limited the growth of yeast bait strains were determined before transformation with the pGAD plasmid ( <italic> AtHSP90 </italic> . <italic> 1 </italic> &#x02013;500, 200 ng ml <sup> -1 </sup> ; HSE, 500 ng ml <sup> -1 </sup> ; HSEm, 500 ng ml <sup> -1 </sup> ) and were used <span class="trait">to</span> assess the pGAD transformants. Yeast growth on selective medium (-Leu +AbA) was recorded on day 3 as an indicator of protein&#x02014;DNA interactions. pGAD:AtHsfA1a served as the positive control and pGAD as the negative control.
					</p>
				
					<p>&nbsp;&nbsp;
						We demonstrated that PsCRN108 could be translocated into plant cells, where it was targeted <span class="trait">to</span> the nuclei. It is a modular protein that contains a conserved N-terminal domain that may mediate translocation of its C-terminal effector domains into host cells [ <xref rid="ppat.1005348.ref006" ref-type="bibr"> 6 </xref> ]. A yeast secretion assay indicated that its predicted signal peptide was functional, and detection of this protein in culture supernatants of <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> lines using Western blotting confirmed that PsCRN108 was secreted by the pathogen into the culture supernatnants, and presumably therefore into apoplastic space during <span class="trait">infection.</span> Furthermore, the protein could deliver the C-terminal region of <span class="gene">Avr1b</span> into soybean cells, demonstrating that PsCRN108 is translocated into host cells during <span class="trait">infection.</span> In this respect, it is similar <span class="trait">to</span> other characterized host intracellular effectors from <italic> <span class="trait">Phytophthora</span> </italic> pathogens, including <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> <span class="gene">Avr1b</span> [ <xref rid="ppat.1005348.ref005" ref-type="bibr"> 5 </xref> ] and Isc1 [ <xref rid="ppat.1005348.ref009" ref-type="bibr"> 9 </xref> ]; and <italic> P </italic> . <italic> infestans </italic> <span class="gene">Avr3a</span> [ <xref rid="ppat.1005348.ref004" ref-type="bibr"> 4 </xref> ] and CRN2/8/16 [ <xref rid="ppat.1005348.ref006" ref-type="bibr"> 6 </xref> ]. However, the mechanisms of CRN effector translocation into host cells remain unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						PsCRN108 contributes <span class="trait">to</span> <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> virulence. Its expression was induced during the early stages of <span class="trait">infection,</span> whereas the paralogs <italic> PsCRN112a/b/c </italic> and <italic> PsCRN114 </italic> were not. Furthermore, silencing of <italic> PsCRN108 </italic> (together with <italic> PsCRN112a/b/c </italic> ) led <span class="trait">to</span> a dramatic reduction in <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> virulence towards soybean. It has been reported that joint silencing of <italic> PsCRN63/115 </italic> also reduced virulence [ <xref rid="ppat.1005348.ref050" ref-type="bibr"> 50 </xref> ]; and that silencing of <italic> <span class="gene">Avr3a</span> </italic> or <italic> PITG_03192 </italic> compromised the pathogenicity of <italic> P </italic> . <italic> infestans </italic> [ <xref rid="ppat.1005348.ref007" ref-type="bibr"> 7 </xref> , <xref rid="ppat.1005348.ref008" ref-type="bibr"> 8 </xref> ]. Despite the fact that the genome of this pathogen contains 100 <italic> CRN </italic> genes and 102 pseudogenes [ <xref rid="ppat.1005348.ref025" ref-type="bibr"> 25 </xref> ], silencing of PsCRN108 and <italic> PsCRN112a/b/c </italic> indicates that these closely-related effectors likely play an essential role during <span class="trait">infection.</span> Thus, future screening of diverse soybean cultivars <span class="trait">to</span> find <italic> R </italic> genes that can detect this effector <span class="trait">to</span> trigger ETI might result in generation of durable resistant cultivars for breeding applications.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on both <italic> in vivo </italic> and <italic> in vitro </italic> assays, the interaction of PsCRN108 with HSE&#x02019;s and HSE-containing promoters could suppress the interaction between HSE&#x02019;s and a heat shock transcription factor (AtHsfA1a), which is an important mediator of abiotic stress responses and a major transcriptional activator of <italic> HSP </italic> genes [ <xref rid="ppat.1005348.ref044" ref-type="bibr"> 44 </xref> , <xref rid="ppat.1005348.ref045" ref-type="bibr"> 45 </xref> , <xref rid="ppat.1005348.ref057" ref-type="bibr"> 57 </xref> &#x02013; <xref rid="ppat.1005348.ref059" ref-type="bibr"> 59 </xref> ]. The DNA-binding domain (DBD) of heat shock transcription factors is responsible for specific binding of the HSE&#x02019;s in <italic> HSP </italic> promoters, which is essential for induction of the expression of plant <italic> HSP </italic> genes [ <xref rid="ppat.1005348.ref038" ref-type="bibr"> 38 </xref> ]. <italic> AtHsfA1a </italic> is usually expressed constitutively at low levels; however, its activity is repressed under normal conditions but is activated during stress at multiple levels, including oligomerization, localization, phosphorylation, HSE binding and transcriptional activation during stresses [ <xref rid="ppat.1005348.ref058" ref-type="bibr"> 58 </xref> ]. Both human and plant HSP70s may interact directly with HSE: Hsf1 complexes, resulting in negative feedback regulation of Hsf activity [ <xref rid="ppat.1005348.ref060" ref-type="bibr"> 60 </xref> , <xref rid="ppat.1005348.ref061" ref-type="bibr"> 61 </xref> ]. The mechanism by which <span class="gene">HSP70</span> interferes with transcriptional activation of AtHsfA1a is unclear. Here, we found that PsCRN108 might interfere with the regulation of AtHsfA1a in plant cells by suppressing its binding <span class="trait">to</span> HSE&#x02019;s and/or by inhibiting the action of HSE-bound AtHsfA1a. This ability of a pathogen effector <span class="trait">to</span> prevent host proteins from binding <span class="trait">to</span> their target gene promoters has been reported in only a few bacterial pathogens [ <xref rid="ppat.1005348.ref016" ref-type="bibr"> 16 </xref> ]. For example, the human pathogen <italic> Shigella flexneri </italic> produces outer-membrane protein F (OspF), which interferes with binding of NF-&#x003ba;B <span class="trait">to</span> its target gene promoters [ <xref rid="ppat.1005348.ref062" ref-type="bibr"> 62 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The yeast signal sequence trap system assay was performed following the described method [ <xref rid="ppat.1005348.ref042" ref-type="bibr"> 42 </xref> ]. We used the pSUC2T7M13ORI (pSUC2) vector, which carries a truncated invertase gene ( <italic> SUC2 </italic> ) lacking both the initiation Met and signal peptide. The N-terminal regions of PsCRN108 and <span class="gene">Avr1b</span> were introduced into pSUC2 <span class="trait">to</span> create in-frame fusions <span class="trait">to</span> the invertase (all primers and constructs used in this study are listed in <xref rid="ppat.1005348.s012" ref-type="supplementary-material"> S3 </xref> and <xref rid="ppat.1005348.s013" ref-type="supplementary-material"> S4 </xref> Tables). Briefly, pSUC2-derived plasmids were transformed into yeast, which was plated on CMD-W (minus Trp) plates. The positive clones were then transferred <span class="trait">to</span> YPRAA plates <span class="trait">to</span> assay invertase secretion. Invertase activity was determined by monitoring the reduction of TTC <span class="trait">to</span> the insoluble, red-colored triphenylformazan.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">To</span> evaluate functional expression of <span class="gene">Avr1b</span> in <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transformants, seedlings of the soybean cultivars Williams ( <italic> rps </italic> ) and HARO13 (Harosoy background, <italic> Rps1b </italic> ) grown in the light for 7 days. Then the hypocotyls were inoculated with small clumps of mycelia. The number of surviving and dead plants were counted after 2 days <span class="trait">to</span> evaluate the ability of the transformants <span class="trait">to</span> infect the plants containing <italic> Rps1b </italic> [ <xref rid="ppat.1005348.ref005" ref-type="bibr"> 5 </xref> , <xref rid="ppat.1005348.ref009" ref-type="bibr"> 9 </xref> ]. <span class="trait">To</span> quantitatively evaluate the overall virulence of <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transformants, etiolated soybean seedlings of cv. Williams were grown for 4 days, then the hypocotyls were inoculated with approximately 100 <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> zoospores [ <xref rid="ppat.1005348.ref009" ref-type="bibr"> 9 </xref> ]. Lesion lengths were measured after 36 hours, and significant differences compared with the WT were identified using Dunnett&#x02019;s test. <span class="trait">To</span> test the protein expression of <italic> P </italic> . <italic> <span class="trait">sojae</span> </italic> transformants over-expressing <span class="gene">Avr1b</span> and Avr1bCt fusion proteins during <span class="trait">infection,</span> mycelium was used <span class="trait">to</span> infect the wounded hypocotyls of soybean plants. The infected hypocotyl tissue samples including mycelium were collected 12 hpi.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>