<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Rhizoctonia solani</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Rhizoctonia solani</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:A Metabolic Profiling Strategy for the Dissection of Plant Defense against Fungal Pathogens</h2>
			<h3>PMID:25369450</h3>
				
					<h3>PMCID:4219818</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Aliferis </surname>  <given-names> Konstantinos A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Faubert </surname>  <given-names> Denis </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jabaji </surname>  <given-names> Suha </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The complexity of plants' metabolome makes their deconvolution challenging, requiring often the utilization of more than one analyzer. For DIMS-based metabolomics, ion suppression can impact the validity of analysis, however, information on its dynamics is yet fragmented. Here, analysis of samples with similar metabolite profiles resulted in consistent ion suppression as revealed by the tight clustering among biological replications performing multivariate analyses ( <xref ref-type="supplementary-material" rid="pone.0111930.s001"> Fig. S1 </xref> ). The latter confirms the potential of DIMS for high-throughput metabolomics applications in line with recent studies <xref rid="pone.0111930-Draper1" ref-type="bibr"> [19] </xref> , <xref rid="pone.0111930-Kirwan1" ref-type="bibr"> [20] </xref> . The developed protocol ( <xref ref-type="supplementary-material" rid="pone.0111930.s002"> Fig. S2 </xref> ) enabled the in-depth deconvolution of DIMS data, as confirmed by the large number of obtained frames using the software SIEVE ( <xref ref-type="supplementary-material" rid="pone.0111930.s006"> Table S1 </xref> ). On the other hand, GC/MS analyses facilitated the construction of a matrix composed of 135 features, reproducibly detected across treatments. In total, 377 putatively or absolutely identified metabolites were statistically significant different between controls and infected soybean sprouts ( <xref ref-type="supplementary-material" rid="pone.0111930.s007"> Data Set S1 </xref> ). <span class="gene">MS</span> spectra of identified metabolites of biological origin from GC/MS analysis, and MS/MS spectra from DIMS Orbitrap analysis provided in the <xref ref-type="supplementary-material" rid="pone.0111930.s008"> Data Set S2 </xref> and <xref ref-type="supplementary-material" rid="pone.0111930.s009"> Data Sets S3 </xref> and <xref ref-type="supplementary-material" rid="pone.0111930.s010"> S4 </xref> , respectively. Sets of original GC/MS and DIMS Orbitrap data can be found at the public repository of Metabolights ( <ext-link ext-link-type="uri" xlink:href="http://www.ebi.ac.uk/metabolights/"> http://www.ebi.ac.uk/metabolights/ </ext-link> ) (Accession # MTBLS118 and MTBLS117, respectively). The complexity of undergoing biochemical events during soybean- <italic> <span class="trait">Rhizoctonia</span> </italic> interaction ( <xref ref-type="supplementary-material" rid="pone.0111930.s003"> Fig. S3 </xref> ) is indicated by the diversity of chemical groups and biosynthetic pathways involved ( <xref ref-type="fig" rid="pone-0111930-g001"> Figs. 1 </xref> and <xref ref-type="fig" rid="pone-0111930-g002"> 2 </xref> , <xref ref-type="supplementary-material" rid="pone.0111930.s004"> Fig. S4 </xref> ). Up-regulated metabolites also detected in fungal profiles, which could have leverage on data interpretation, were omitted from analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						Samples (100 mg of fresh weight, fw) were harvested from the edges of necrotic lesions at 24 h and 48 h post-inoculation and corresponding portions of control seedlings ( <xref ref-type="supplementary-material" rid="pone.0111930.s003"> Fig. S3 </xref> ). Additionally, mycelia (50 mg, fw) from 3-day-old <italic> R. <span class="trait">solani</span> </italic> cultures were harvested. Samples were pulverized to a fine powder in a mortar with a pestle under liquid nitrogen and kept at &#x02212;80&#x000b0;C in glass autosampler vials (2 ml) until further processing. Metabolite extraction was performed as previously described <xref rid="pone.0111930-Aliferis1" ref-type="bibr"> [4] </xref> . Briefly, 1 ml of a mixture of methanol:ethyl acetate (50&#x02236;50, v/v) was added to the pulverized samples followed by sonication in an ultrasonic bath Branson 5510 (Branson, Connecticut, USA) for 25 min. Samples were then transferred to an orbital shaker (Infors AG, Bottmingen, Switzerland) for extraction under continuous agitation (150 rpm) for 2 h at 24&#x000b0;C. Extracts were filtered through 0.2 &#x000b5;m filters (Millex-FG, Millipore, MA, USA) to remove debris and the volume of samples was adjusted to 1.0 ml. Consecutively, extracts were divided into two equal portions (0.5 ml) in glass autosampler vials for GC/MS and direct infusion Orbitrap <span class="gene">MS</span> analysis (DIMS). The latter were sub-divided into equal portions of 0.25 ml for analysis in positive (ESI <sup> + </sup> ) and negative (ESI <sup> &#x02212; </sup> ) electrospray modes, respectively. Finally, extracts were dried using a Labconco CentriVap refrigerated vacuum concentrator (Labconco, Kansas City, MO, USA) equipped with a cold trap.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Partial least squares-discriminant analyses (PLS-DA) PC1/PC2 score plots of direct infusion Orbitrap <span class="gene">MS</span> and GC/MS metabolite profiles of control (&#x025aa;) and </bold> <bold> <italic> <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> </italic> </bold> <bold> -infected (&#x02022;) soybean seedlings, at 24 h and 48 h post-inoculation. </bold> The ellipse represents the Hotelling T <sup> 2 </sup> with 95% confidence interval. Five (5) biological replications were used per treatment and one quality control sample (QC) [Q <sup> 2 </sup> <sub> (cum) </sub> ; cumulative fraction of the total variation of the X's that can be predicted by the extracted components, <italic> R <sup> 2 </sup> X </italic> and <italic> R <sup> 2 </sup> Y </italic> ; the fraction of the sum of squares of all <italic> X </italic> 's and <italic> Y </italic> 's explained by the current component, respectively, <italic> PCs </italic> ; principal components].
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Representative total ion chromatograms (TIC) of control and </bold> <bold> <italic> <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> </italic> </bold> <bold> -infected soybean seedlings at 24 h and 48 h post-inoculation performing direct infusion Orbitrap <span class="gene">MS</span> analysis. </bold> Mass spectra that correspond to the area between the dashed lines were used in metabolomics analyses.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mass-spectrometry data for <italic> <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> </italic> proteins produced during infection of wheat and vegetative growth</h2>
			<h3>PMID:27331100</h3>
				
					<h3>PMCID:4906030</h3>
					<h4>期刊:Data in Brief</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> Jonathan P. </given-names>  </name>  <email> Jonathan.Anderson@csiro.au </email>  <xref rid="aff0005" ref-type="aff"> a </xref>  <xref rid="aff0010" ref-type="aff"> b </xref>  <xref rid="cor1" ref-type="corresp"> &#x0204e; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hane </surname>  <given-names> James K. </given-names>  </name>  <xref rid="aff0005" ref-type="aff"> a </xref>  <xref rid="fn1" ref-type="fn"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stoll </surname>  <given-names> Thomas </given-names>  </name>  <xref rid="aff0015" ref-type="aff"> c </xref>  <xref rid="fn2" ref-type="fn"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pain </surname>  <given-names> Nicholas </given-names>  </name>  <xref rid="aff0005" ref-type="aff"> a </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hastie </surname>  <given-names> Marcus L. </given-names>  </name>  <xref rid="aff0015" ref-type="aff"> c </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kaur </surname>  <given-names> Parwinder </given-names>  </name>  <xref rid="aff0005" ref-type="aff"> a </xref>  <xref rid="fn3" ref-type="fn"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoogland </surname>  <given-names> Christine </given-names>  </name>  <xref rid="aff0015" ref-type="aff"> c </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gorman </surname>  <given-names> Jeffrey J. </given-names>  </name>  <xref rid="aff0015" ref-type="aff"> c </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Karam B. </given-names>  </name>  <xref rid="aff0005" ref-type="aff"> a </xref>  <xref rid="aff0010" ref-type="aff"> b </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						This proteomics dataset comprises <span class="gene">MS</span> RAW files and identification files (mzIdentML and Scaffold files). MS/MS raw files were mapped to the <italic> R. <span class="trait">solani</span> </italic> AG8 WAC10335 gene models (GenBank assembly accession: GCA_000695385.1) and a six-frame translation of the genome using Mascot V2.4.1. Samples are obtained from either <italic> R. <span class="trait">solani</span> </italic> mycelium undergoing vegetative growth or <italic> R. <span class="trait">solani</span> </italic> infecting wheat seedlings at either early or late time points ( <xref rid="f0005" ref-type="fig"> Fig. 1 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The evolution and pathogenic mechanisms of the rice sheath blight pathogen.</h2>
			<h3>PMID:23361014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng A(1), Lin R, Zhang D, Qin P, Xu L, Ai P, Ding L, Wang Y, Chen Y, Liu Y, Sun Z, Feng H, Liang X, Fu R, Tang C, Li Q, Zhang J, Xie Z, Deng Q, Li S, Wang S, Zhu J, Wang L, Liu H, Li P.
				<br/> 
				年份：2013
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Rhizoctonia</span> <span class="trait">solani</span> is a major fungal pathogen of rice (Oryza sativa L.) that causes great <span class="trait">yield</span> losses in all rice-growing regions of the world. Here we report the draft genome sequence of the rice sheath blight disease pathogen, R. <span class="trait">solani</span> <span class="gene">AG1</span> IA, assembled using next-generation Illumina Genome Analyser sequencing technologies. The genome encodes a large and diverse set of secreted proteins, enzymes of primary and secondary metabolism, carbohydrate-active enzymes, and transporters, which probably reflect an exclusive necrotrophic lifestyle. We find few repetitive elements, a closer relationship to Agaricomycotina among Basidiomycetes, and expand protein domains and families. Among the 25 candidate pathogen effectors identified according to their functionality and evolution, we validate 3 that trigger crop defence responses; hence we reveal the exclusive expression patterns of the pathogenic determinants during host infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic structure of populations of <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> anastomosis group-1 IA from soybean in Brazil.</h2>
			<h3>PMID:18943212  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ciampi MB(1), Meyer MC, Costa MJ, Zala M, McDonald BA, Ceresini PC.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Basidiomycete fungus <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> anastomosis group (AG)-1 IA is a major pathogen of soybean in Brazil, where the average <span class="trait">yield</span> losses have reached 30 to 60% in some states in Northern Brazil. No information is currently available concerning levels of genetic diversity and population structure for this pathogen in Brazil. A total of 232 isolates of R. <span class="trait">solani</span> <span class="gene">AG1</span> IA were collected from five soybean fields in the most important soybean production areas in central-western, northern, and northeastern Brazil. These isolates were genotyped using 10 microsatellite loci. Most of the multilocus genotypes (MLGTs) were site-specific, with few MLGTs shared among populations. Significant population subdivision was evident. High levels of admixture were observed for populations from Mato Grosso and Tocantins. After removing admixed genotypes, three out of five field populations (Maranhao, Mato Grosso, and Tocantins), were in Hardy-Weinberg (HW) equilibrium, consistent with sexual recombination. HW and gametic disequilibrium were found for the remaining soybean-infecting populations. The findings of low genotypic diversity, departures from HW equilibrium, gametic disequilibrium, and high degree of population subdivision in these R. <span class="trait">solani</span> AG-1 IA populations from Brazil are consistent with predominantly asexual reproduction, short-distance dispersal of vegetative propagules (mycelium or sclerotia), and limited long-distance dispersal, possibly via contaminated seed. None of the soybean-infecting populations showed a reduction in population size (bottleneck effect). We detected asymmetric historical migration among the soybean-infecting populations, which could explain the observed levels of subdivision.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome analysis reveals the host selection fitness mechanisms of the <italic> <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> </italic> AG1IA pathogen</h2>
			<h3>PMID:28860554</h3>
				
					<h3>PMCID:5579035</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Yuan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fei </surname>  <given-names> Binghong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Jiayu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Menglin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Danhua </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Linxiu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shuangcheng </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Yueyang </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lingxia </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Jianqing </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ping </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Aiping </given-names>  </name>  <address>  <email> apzh0602@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0185 3134 </institution-id>  <institution-id institution-id-type="GRID"> grid.80510.3c </institution-id>  <institution> Rice Research Institute, Sichuan Agricultural University,  </institution>  </institution-wrap> Chengdu, 611130 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0185 3134 </institution-id>  <institution-id institution-id-type="GRID"> grid.80510.3c </institution-id>  <institution> Key Laboratory of Sichuan Crop Major Diseases, Sichuan Agricultural University,  </institution>  </institution-wrap> Chengdu, 611130 China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0185 3134 </institution-id>  <institution-id institution-id-type="GRID"> grid.80510.3c </institution-id>  <institution> Key Laboratory of Southwest Corp Gene Resource and Genetic Improvement of Ministry of Education, Sichuan Agricultural University,  </institution>  </institution-wrap> Ya&#x02019;an, 625014 China  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Differential expression of secondary metabolite-associated genes in <italic> R. <span class="trait">solani</span> </italic> AG1IA during infection of various hosts. ( <bold> A </bold> ) Expression profiles of up-regulated cytochrome <span class="gene">P450</span> during infection of various hosts. ( <bold> B </bold> ) Expression profiles of differentially expressed candidate ABC transporters during infection of various hosts. A hierarchical clustering statistic method based on Pearson&#x02019;s correlation (correlation&#x02009;&#x02265;&#x02009;98%) and average linkage was used. Red indicates a significant increase in gene expression, black indicates no change in expression and green indicates a significant decrease in gene expression (|log2FC|&#x02009;&#x02265;&#x02009;1). Colour scale represents log <sub> 2 </sub> fold change values from &#x02212;3 to 3.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation of ethylene-insensitive soybean mutants that are altered in pathogen susceptibility and gene-for-gene disease <span class="trait">resistance</span></h2>
			<h3>PMID:10069832</h3>
				

				
			<p>
				作者：Hoffman T(1), Schmidt JS, Zheng X, Bent AF.
				<br/> 
				年份：1999 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants commonly respond to pathogen infection by increasing ethylene production, but it is not clear if this ethylene does more to promote disease susceptibility or disease <span class="trait">resistance.</span> Ethylene production and/or responsiveness can be altered by genetic manipulation. The present study used mutagenesis to identify soybean (Glycine max L. Merr.) lines with reduced sensitivity to ethylene. Two new genetic loci were identified, Etr1 and Etr2. Mutants were compared with isogenic wild-type parents for their response to different soybean pathogens. Plant lines with reduced ethylene sensitivity developed similar or less-severe disease symptoms in response to virulent Pseudomonas syringae pv glycinea and Phytophthora sojae, but some of the mutants developed similar or more-severe symptoms in response to Septoria glycines and <span class="trait">Rhizoctonia</span> <span class="trait">solani.</span> Gene-for-gene <span class="trait">resistance</span> against P. syringae expressing <span class="gene">avrRpt2</span> remained effective, but Rps1-k-mediated <span class="trait">resistance</span> against P. sojae races 4 and 7 was disrupted in the strong ethylene-insensitive etr1-1 mutant. Rps1-k-mediated <span class="trait">resistance</span> against P. sojae race 1 remained effective, suggesting that the Rps1-k locus may encode more than one gene for disease <span class="trait">resistance.</span> Overall, our results suggest that reduced ethylene sensitivity can be beneficial against some pathogens but deleterious to <span class="trait">resistance</span> against other pathogens.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ubiquitous urease affects soybean susceptibility to fungi</h2>
			<h3>PMID:22382992</h3>
				

				
					<h3>PMCID:3332383</h3>
					<h4>期刊:Plant Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wiebke-Strohm </surname>  <given-names> Beatriz </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pasquali </surname>  <given-names> Giancarlo </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Margis-Pinheiro </surname>  <given-names> M&#x000e1;rcia </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bencke </surname>  <given-names> Marta </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000fc;cker-Neto </surname>  <given-names> Lauro </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Becker-Ritt </surname>  <given-names> Arlete B. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Martinelli </surname>  <given-names> Anne H. S. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rechenmacher </surname>  <given-names> Ciliana </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Polacco </surname>  <given-names> Joseph C. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stolf </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino </surname>  <given-names> Francismar C. </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo V. </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Homrich </surname>  <given-names> Milena S. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Del Ponte </surname>  <given-names> Emerson M. </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Carlini </surname>  <given-names> Celia R. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Carvalho </surname>  <given-names> Mayra C. C. G. </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Bodanese-Zanettini </surname>  <given-names> Maria Helena </given-names>  </name>  <address>  <phone> +5551-330-86725 </phone>  <fax> +5551-3308-7311 </fax>  <email> mhbzanettini@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Programa de P&#x000f3;s-Gradua&#x000e7;&#x000e3;o em Gen&#x000e9;tica e Biologia Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Porto Alegre, Brazil  </aff>  <aff id="Aff2">  <label> 2 </label> Programa de P&#x000f3;s-Gradua&#x000e7;&#x000e3;o em Biologia Celular e Molecular, Centro de Biotecnologia, UFRGS, Porto Alegre, Brazil  </aff>  <aff id="Aff3">  <label> 3 </label> Biochemistry Department, University of Missouri, Columbia, MO USA  </aff>  <aff id="Aff4">  <label> 4 </label> Departamento de Fitossanidade, Faculdade de Agronomia, UFRGS, Porto Alegre, Brazil  </aff>  <aff id="Aff5">  <label> 5 </label> Empresa Brasileira de Pesquisa Agropecu&#x000e1;ria, Embrapa Soja, Londrina, Brazil  </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 2 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean ubiquitous urease (encoded by <span class="gene">GmEu4)</span> is responsible for recycling metabolically derived urea. Additional biological roles have been demonstrated for plant ureases, notably in toxicity to other organisms. However, urease enzymatic activity is not related to its toxicity. The role of <span class="gene">GmEu4</span> in soybean susceptibility to fungi was investigated in this study. A differential expression pattern of <span class="gene">GmEu4</span> was observed in susceptible and resistant genotypes of soybeans over the course of a Phakopsora pachyrhizi infection, especially 24 h after infection. Twenty-nine adult, transgenic soybean plants, representing six independently transformed lines, were obtained. Although the initial aim of this study was to overexpress <span class="gene">GmEu4,</span> the transgenic plants exhibited <span class="gene">GmEu4</span> co-suppression and decreased ureolytic activity. The growth of <span class="trait">Rhizoctonia</span> <span class="trait">solani,</span> Phomopsis sp., and Penicillium herguei in media containing a crude protein extract from either transgenic or non-transgenic leaves was evaluated. The fungal growth was higher in the protein extracts from transgenic urease-deprived plants than in extracts from non-transgenic controls. When infected by P. pachyrhizi uredospores, detached leaves of urease-deprived plants developed a significantly higher number of lesions, pustules and erupted pustules than leaves of non-transgenic plants containing normal levels of the enzyme. The results of the present work show that the soybean plants were more susceptible to fungi in the absence of urease. It was not possible to overexpress active <span class="gene">GmEu4.</span> For future work, overexpression of urease fungitoxic peptides could be attempted as an alternative approach.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The soybean ubiquitous urease (encoded by <italic> <span class="gene">GmEu</span> </italic> 4) is responsible for recycling metabolically derived urea. Additional biological roles have been demonstrated for plant ureases, notably in toxicity to other organisms. However, urease enzymatic activity is not related to its toxicity. The role of <italic> <span class="gene">GmEu</span> </italic> 4 in soybean susceptibility to fungi was investigated in this study. A differential expression pattern of <italic> <span class="gene">GmEu</span> </italic> 4 was observed in susceptible and resistant genotypes of soybeans over the course of a <italic> Phakopsora pachyrhizi </italic> infection, especially 24&#x000a0;h after infection. Twenty-nine adult, transgenic soybean plants, representing six independently transformed lines, were obtained. Although the initial aim of this study was to overexpress <italic> <span class="gene">GmEu</span> </italic> 4, the transgenic plants exhibited <italic> <span class="gene">GmEu</span> </italic> 4 co-suppression and decreased ureolytic activity. The growth of <italic> <span class="trait">Rhizoctonia</span> <span class="trait">solani</span> </italic> , <italic> Phomopsis </italic> sp., and <italic> Penicillium herguei </italic> in media containing a crude protein extract from either transgenic or non-transgenic leaves was evaluated. The fungal growth was higher in the protein extracts from transgenic urease-deprived plants than in extracts from non-transgenic controls. When infected by <italic> P. pachyrhizi </italic> uredospores, detached leaves of urease-deprived plants developed a significantly higher number of lesions, pustules and erupted pustules than leaves of non-transgenic plants containing normal levels of the enzyme. The results of the present work show that the soybean plants were more susceptible to fungi in the absence of urease. It was not possible to overexpress active <italic> <span class="gene">GmEu</span> </italic> 4. For future work, overexpression of urease fungitoxic peptides could be attempted as an alternative approach.
					</p>
				
					<p>&nbsp;&nbsp;
						The reaction of soybean plants to <span class="trait">rust</span> infection was assessed by the inoculation of <italic> P. pachyrhizi </italic> spores collected in the field into plants maintained under greenhouse conditions at Embrapa Soja, Londrina, PR, Brazil. The soybean plants were grown in a pot-based system and maintained in a greenhouse at 28&#x000a0;&#x000b1;&#x000a0;1&#x000b0;C with 16/8&#x000a0;h light/dark at a light intensity of 22.5&#x000a0;&#x003bc;Em <sup> &#x02212;2 </sup> &#x000a0;s <sup> &#x02212;1 </sup> . The Embrapa-48 genotype, which develops a Tan lesion (van de Mortel et al. <xref ref-type="bibr" rid="CR52"> 2007 </xref> ), was used as the susceptible standard and the PI561356 genotype, which carries the <span class="trait">resistance</span> to soybean <span class="trait">rust</span> mapped to linkage group G, was used as the resistant standard (Camargo <xref ref-type="bibr" rid="CR4"> 2010 </xref> ). Uredospores were harvested from leaves exhibiting sporulating uredia and diluted in distilled water with 0.05% Tween-20 to a final concentration of 3&#x000a0;&#x000d7;&#x000a0;10 <sup> 5 </sup> spores/mL. The spore suspension was sprayed onto plantlets at the <span class="gene">V2</span> developmental stage. The same solution lacking spores was used for mock inoculations. Following fungal or mock inoculations, water-misted bags were placed over all plants for 1&#x000a0;day to promote infection and to prevent cross-contamination of the mock-infected plants. One trifoliate leaf from each plant was collected at 1, 12, 24, 48, 96 and 192&#x000a0;h after inoculation, frozen in liquid nitrogen, and stored at &#x02212;80&#x000b0;C. Three biological replicates from each genotype were analysed for both treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						Total DNA was extracted from plant leaves as previously described (Doyle and Doyle <xref ref-type="bibr" rid="CR9"> 1987 </xref> ). Putative transgenic plants were PCR-screened for the presence of the hygromycin <span class="trait">resistance</span> gene ( <italic> hpt </italic> ) and a chimeric gene (P <italic> 35S </italic> - <italic> <span class="gene">GmEu</span> </italic> 4) formed by the CaMV <italic> 35S </italic> promoter (P <italic> 35S </italic> ) and the ubiquitous urease ORF ( <italic> <span class="gene">GmEu</span> </italic> 4). The following primer pairs were used in the PCR assays: 5&#x02032;-GAGCCTGACCTATTGCATCTCC-3&#x02032; and 5&#x02032;-GGCCTCCAGAAGAAGATGTTGG-3&#x02032; ( <italic> hpt </italic> ); 5&#x02032;-CGCACAATCCCACTATCCTT-3&#x02032; and 5&#x02032;-ATGCTAGTTCAAGGTTTCCATTCT-3&#x02032; (P <italic> 35S </italic> - <italic> <span class="gene">GmEu</span> </italic> 4). The PCR mixture consisted of 200&#x000a0;ng of template DNA, 0.4&#x000a0;mM of dNTPs, 0.4 &#x003bc;M of each <italic> primer </italic> , 2.5&#x000a0;mM of MgCl <sub> 2 </sub> , 1&#x000d7; Taq Buffer, 1 U of Taq DNA Polymerase (Invitrogen, S&#x000e3;o Paulo, Brazil), and autoclaved distilled water in a final volume of 25&#x000a0;&#x003bc;L. The reactions were heated at the beginning (5&#x000a0;min at 94&#x000b0;C) and subjected to 30 cycles as follows: 45&#x000a0;s at 94&#x000b0;C, 45&#x000a0;s at 42&#x000b0;C and 45&#x000a0;s at 72&#x000b0;C. After electrophoresis in a 1% agarose gel containing ethidium bromide (0.01&#x000a0;mg/L), the PCR products were visualised under ultraviolet light.
					</p>
				
					<p>&nbsp;&nbsp;
						The transcript levels of <italic> <span class="gene">GmEu</span> </italic> 4 in soybeans inoculated with <italic> P. pachyrhizi </italic> were determined by RT-qPCR. The leaves of two soybean genotypes exhibiting contrasting responses to <span class="trait">rust</span> were assayed: Embrapa-48 is highly susceptible, while PI561356 is more resistant (van de Mortel et al. <xref ref-type="bibr" rid="CR52"> 2007 </xref> ; Camargo <xref ref-type="bibr" rid="CR4"> 2010 </xref> ). Differences in interactions among genotypes, different points in the time-course and the presence or absence of pathogens were highly significant ( <italic> p </italic> &#x000a0;&#x0003c;&#x000a0;0.0001). <italic> <span class="gene">GmEu</span> </italic> 4 expression in non-infected (mock) plants was continuous during the infection for both genotypes. As shown in Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , in the susceptible host, <italic> <span class="gene">GmEu</span> </italic> 4 transcripts were significantly upregulated (2.26-fold) at 1&#x000a0;h, followed by a strong downregulation (25-fold) at 24&#x000a0;h and another upregulation peak at 192&#x000a0;h after pathogen inoculation. In contrast, in the resistant genotype, <italic> <span class="gene">GmEu</span> </italic> 4 expression was strongly upregulated (4.07-fold) 24&#x000a0;h after the inoculation. <fig id="Fig2"> <label> Fig.&#x000a0;2 </label> <caption> <p> <italic> <span class="gene">GmEu</span> </italic> 4 and <italic> <span class="gene">GmPR</span> </italic> 4 gene expression in response to <italic> P. pachyrhizi </italic> infection in susceptible and resistant soybean genotypes. The relative expression levels of <italic> <span class="gene">GmEu</span> </italic> 4 and <italic> <span class="gene">GmPR</span> </italic> 4 in the soybean leaves of susceptible (Embrapa-48) and resistant (PI561356) genotypes were determined by RT-qPCR 1, 12, 24, 48, 92 and 196&#x000a0;h after <italic> P. pachyrhizi </italic> (infected) or mock (non-infected) inoculation. In these <italic> panels </italic> , the <italic> y </italic> -axes indicate that the scale is log2 (fold-change) to allow a better comparison of groups of genes presenting large differences in their expression levels. The values are the means of three biological replicates with four technical replicates each. The means followed by equal letters in the same cultivar do not differ significantly (Bonferroni multiple comparison test, <italic> p </italic> &#x000a0;&#x0003c;&#x000a0;0.05). The F-Box protein and Metalloprotease reference genes were used as internal controls to normalise the amount of mRNA present in each sample. The transcript levels of <italic> <span class="gene">GmEu</span> </italic> 4 and <italic> <span class="gene">GmPR</span> </italic> 4 in mock-inoculated plants were used to normalise the transcript accumulation in inoculated plants
					</p>
				
					<p>&nbsp;&nbsp;
						To determine whether <italic> <span class="gene">GmEu</span> </italic> 4 overexpression would enhance <span class="trait">resistance</span> to fungi, soybean somatic embryos were transformed with the pH7WG2D- <italic> <span class="gene">GmEu</span> </italic> 4 vector. The efficiency of plant recovery via somatic embryogenesis is genotype dependent (Droste et al. <xref ref-type="bibr" rid="CR10"> 2002 </xref> ); therefore, the cultivars IAS5 and Bragg were chosen for this experiment. A total of twenty-nine well-developed plants, representing six independent transgenic lines, were regenerated from two transformation experiments. Twenty-six plants from cultivar IAS5, derived from four independent transgenic lines, were recovered from tissue submitted to particle bombardment. Three plants were recovered from embryos transformed by bombardment/ <italic> Agrobacterium </italic> . These plants correspond to two independent transgenic lines, one from cultivar IAS5 and one from Bragg (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). All of the plants flowered, and 21 set seeds. <table-wrap id="Tab2"> <label> Table&#x000a0;2 </label> <caption> <p> Number of transgenic soybean adult plants and independent lines
					</p>
				
					<p>&nbsp;&nbsp;
						Increasing turbidity of the medium, as the result of hyphal development, was observed over time (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). H <sub> 2 </sub> O <sub> 2 </sub> and protein extraction buffer were used as positive and negative controls for the experimental conditions, respectively. However, to evaluate the biological role of urease on fungal growth, the results obtained from transgenic and non-transgenic plants of the same cultivar must be compared because urease content is supposed to be the unique distinctive characteristic between these plants. <italic> P. herguei </italic> , <italic> Phomopsis </italic> sp. and <italic> R. <span class="trait">solani</span> </italic> growth in protein extracts from two independent transgenic co-suppressed plants was significantly higher than in that from non-transgenic plants. This result suggests that the urease-deprived plants were more susceptible to fungal growth than the plants with normal levels of urease. <fig id="Fig5"> <label> Fig.&#x000a0;5 </label> <caption> <p> Fungal growth after 60&#x000a0;h on soybean crude extracts of <italic> <span class="gene">GmEu</span> </italic> 4 co-suppressed plants. Spores (1,000 spores in 10&#x000a0;&#x003bc;L) were inoculated into 110&#x000a0;&#x003bc;L PDB, incubated at 28&#x000b0;C for 16&#x000a0;h and subsequently, crude leaf protein extract (15&#x000a0;&#x003bc;g in 50&#x000a0;&#x003bc;L) was added to the fungal culture. The samples were incubated at 28&#x000b0;C and the absorbance (430&#x000a0;nm) was recorded every 12&#x000a0;h. The experiment was carried out with four T <sub> 0 </sub> co-suppressed plants (two from each A3 and A8 <italic> lines </italic> ) and three non-transformed plants (IAS5 NT). Hydrogen peroxide (H <sub> 2 </sub> O <sub> 2 </sub> ) and protein extraction buffer were taken as positive and negative controls of the experimental conditions, respectively. For biological interpretations, transgenic and non-transgenic plants of the same cultivar must be compared. Technical triplicates were analysed per sample. The data (mean&#x000a0;&#x000b1;&#x000a0;SD) are proportional to the 0&#x000a0;h absorbance, which was considered to be one. The means followed by different letters in the same fungus/experiment are significantly different (ANOVA, Bonferroni multiple comparison test, <italic> p </italic> &#x000a0;&#x0003c;&#x000a0;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						Tan lesions were observed on all detached leaves 12&#x000a0;days after <italic> P. pachyrhizi </italic> inoculation (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). However, the number of lesions, number of lesions with pustules, number of pustules and number of opened pustules were significantly higher in <italic> <span class="gene">GmEu</span> </italic> 4 co-suppressed lines (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). The susceptibility of the detached leaves to <italic> P. pachyrhizi </italic> is in agreement with previous results using intact plants (Twizeyimana et al. <xref ref-type="bibr" rid="CR51"> 2006 </xref> ). It is important to note that no visible difference was observed in the timing of lesion and pustule formation or pustule eruption (data not shown). <fig id="Fig6"> <label> Fig.&#x000a0;6 </label> <caption> <p> Soybean <span class="trait">rust</span> ( <italic> P. Pachyrhizi </italic> ) development on detached soybean leaves of <italic> <span class="gene">GmEu</span> </italic> 4 co-suppressed plants 12&#x000a0;days after inoculation. Detached leaves were inoculated with 10 <sup> 5 </sup> /mL of a uredospore suspension and incubated at 20&#x000b0;C. In the <italic> upper panel </italic> , <italic> Tan-coloured </italic> lesions and pustules as observed under stereomicroscope are shown. In the <italic> lower panel </italic> , the evaluation of the four infection parameters were evaluated in the seven T <sub> 1 </sub> co-suppressed plants (four from A3 <italic> line </italic> and three from A8 <italic> line </italic> ) and three non-transgenic plants is shown. The <italic> symbol </italic> * indicates that the means are significantly different between leaves of transformed and non-transformed plants (Student&#x02019;s <italic> t </italic> test, <italic> p </italic> &#x000a0;&#x0003c;&#x000a0;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						It has been previously demonstrated that purified ureases from soybean (embryo-specific urease), jackbean and cotton seeds display antifungal properties (Becker-Ritt et al. <xref ref-type="bibr" rid="CR2"> 2007 </xref> ; Menegassi et al. <xref ref-type="bibr" rid="CR29"> 2008 </xref> ). The results of the present study consistently demonstrated that the absence of the <italic> <span class="gene">GmEu</span> </italic> 4 product enhanced susceptibility of the soybean plant to necrotrophic pathogens, <italic> P. herguei </italic> , <italic> Phomopsis </italic> sp. and <italic> R. <span class="trait">solani</span> </italic> , and a biotrophic pathogen, <italic> P. pachyrhizi </italic> . As observed for other ureases, the soybean ubiquitous urease affected the development of a broad spectrum of fungi. In addition, it was confirmed that the in vivo urease levels influenced the plant susceptibility to necrotrophic and biotrophic fungal pathogens.
					</p>
				
					<p>&nbsp;&nbsp;
						Although there is insufficient biochemical and molecular information currently available to predict the precise role of <italic> <span class="gene">GmEu</span> </italic> 4 during fungal infection, the contribution(s) of urease to plant susceptibility to a broad spectrum of fungal pathogens is demonstrated in this study. The transfer of the <italic> <span class="gene">GmEu</span> </italic> 4 co-suppression characteristic into resistant genotypes and further bioassays may provide new insights into the role of the ubiquitous urease in plant-fungi interaction. Because we assume that urease overexpression, at least in its intact form, may be impossible, the identification and overexpression of the urease fungitoxic peptide is an alternative approach that could reduce the frequency of the co-suppressed phenotype and thereby achieve the desirable increase in the soybean plant <span class="trait">resistance</span> to fungi.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of plant immunity through modulation of phytoalexin synthesis.</h2>
			<h3>PMID:24914895  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zernova OV(1), Lygin AV(2), Pawlowski ML(3), Hill CB(4), Hartman GL(5), Widholm JM(6), Lozovaya VV(7).
				<br/> 
				年份：2014 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean hairy roots transformed with the resveratrol synthase and resveratrol oxymethyl transferase genes driven by constitutive Arabidopsis actin and CsVMV promoters were characterized. Transformed hairy roots accumulated glycoside conjugates of the stilbenic compound resveratrol and the related compound pterostilbene, which are normally not synthesized by soybean plants. Expression of the non-native stilbenic phytoalexin synthesis in soybean hairy roots increased their <span class="trait">resistance</span> to the soybean pathogen <span class="trait">Rhizoctonia</span> <span class="trait">solani.</span> The expression of the AhRS3 gene resulted in 20% to 50% decreased root necrosis compared to that of untransformed hairy roots. The expression of two genes, the AhRS3 and ROMT, required for pterostilbene synthesis in soybean, resulted in significantly lower root necrosis (ranging from 0% to 7%) in transgenic roots than in untransformed hairy roots that had about 84% necrosis. Overexpression of the soybean prenyltransferase (dimethylallyltransferase) <span class="gene">G4DT</span> gene in soybean hairy roots increased accumulation of the native phytoalexin glyceollin resulting in decreased root necrosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>