<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Root rot</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Root rot</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Genome-wide association mapping of quantitative <span class="trait">resistance</span> to sudden death syndrome in soybean</h2>
			<h3>PMID:25249039</h3>
				
					<h3>PMCID:4189206</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wen </surname>  <given-names> Zixiang </given-names>  </name>  <address>  <email> wzxsoy@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tan </surname>  <given-names> Ruijuan </given-names>  </name>  <address>  <email> tanruij1@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Jiazheng </given-names>  </name>  <address>  <email> yuanjz@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bales </surname>  <given-names> Carmille </given-names>  </name>  <address>  <email> balesca1@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Wenyan </given-names>  </name>  <address>  <email> duwenyan@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shichen </given-names>  </name>  <address>  <email> zhangs55@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chilvers </surname>  <given-names> Martin I </given-names>  </name>  <address>  <email> chilvers@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schmidt </surname>  <given-names> Cathy </given-names>  </name>  <address>  <email> wheee@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <address>  <email> qijian.song@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B </given-names>  </name>  <address>  <email> perry.cregan@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Wang </surname>  <given-names> Dechun </given-names>  </name>  <address>  <email> wangdech@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Plant, Soil and Microbial Sciences, Michigan State University, 1066 Bogue St., Rm. A384-E, East Lansing, MI 48824 USA  </aff>  <aff id="Aff2">  <label/> Agronomy Research Center, Southern Illinois University Carbondale, Carbondale, Illinois 62903-7002 USA  </aff>  <aff id="Aff3">  <label/> USDA, Agricultural Research Service, Soybean Genomics and Improvement Laboratory, Beltsville, Maryland 20705 USA  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 20 loci underlying SDS <span class="trait">resistance</span> were identified in the two independent studies, including 7 loci localized in previously mapped QTL intervals and 13 novel loci. One strong peak of association on chromosome 18, associated with all disease assessment criteria across the two panels, spanned a physical region of 1.2&#x000a0;Mb around a previously cloned SDS <span class="trait">resistance</span> gene ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) in locus <italic> Rfs2 </italic> . An additional variant independently associated with SDS <span class="trait">resistance</span> was also found in this genomic region. Other peaks were within, or close to, sequences annotated as homologous to genes previously shown to be involved in plant disease <span class="trait">resistance.</span> The identified loci explained an average of 54.5% of the phenotypic variance measured by different disease assessment criteria.
					</p>
				
					<p>&nbsp;&nbsp;
						Host plant <span class="trait">resistance</span> is believed to be the most effective control measure for SDS [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. Since no soybean genotypes confer complete immunity to this disease, soybean breeders still rely on quantitative <span class="trait">resistance</span> to SDS [ <xref ref-type="bibr" rid="CR7"> 7 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. The wide range of variation of susceptibility to both leaf scorch and <span class="trait">root</span> <span class="trait">rot</span> also provides a great opportunity to improve SDS <span class="trait">resistance</span> through genetic manipulation [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ]. However, most of what we know about the genetic architecture of SDS <span class="trait">resistance</span> is based on traditional quantitative trait locus (QTL) linkage mapping using bi-parental populations. Fourteen QTLs dispersed throughout the genome, underlying <span class="trait">resistance</span> to <span class="trait">root</span> infection, leaf scorch or both, have been confirmed in several bi-parental populations [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. However, the large confidence intervals for those QTLs impair the precise identification of causative genes. To date, only one <span class="trait">resistance</span> gene ( <italic> <span class="gene">GmRLK18-1</span> </italic> ) has been tagged and cloned [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. This gene is at 1.071 kbp on chromosome 18, within the major <span class="trait">resistance</span> QTL ( <italic> Rfs2 </italic> ). Association mapping, which exploits historical recombination events at the population level, has become a powerful alternative to linkage mapping in the dissection of complex trait variation at the sequence level [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. A more specific strategy, genome-wide association (GWA) mapping, is a powerful complementary strategy for classical bi-parental linkage mapping to dissect complex traits and has been used with success in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="CR13"> 13 </xref> ], rice [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ] and maize [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. The use of association mapping in soybean was therefore desirable to improve the mapping of important traits in soybean. So far, only a few association mapping studies, with limited numbers of markers, have been reported for dissecting agronomic traits in soybean [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. To the best of our knowledge, GWA mapping has not yet been employed to study any traits related to soybean disease <span class="trait">resistance.</span> Recently, the availability of a soybean reference genome sequence and the development of high throughput SNP assays has enabled GWA mapping in soybean [ <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. A previous study reported that approximately 1% of the 6,037 Plant Introductions (PIs) from the USDA Soybean Germplasm Collection were partially resistant to SDS [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Therefore, conducting an association study in assembled PI collections might not be feasible. Furthermore, previous research indicated that most of the PIs showing <span class="trait">resistance</span> to soybean cyst nematode (SCN) were also partially resistant to SDS [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. Therefore, association mapping with released elite cultivars is more likely to identify superior <span class="trait">resistance</span> alleles that have been captured and accumulated by SCN or SDS breeding practices.
					</p>
				
					<p>&nbsp;&nbsp;
						The association panel <span class="gene">P1</span> was evaluated for SDS <span class="trait">resistance</span> in a naturally infested SDS disease nursery at Decatur, Michigan during the growing season (May-October) in 2011 and 2012, where consistent, natural and heavy SDS disease symptoms was observed on susceptible checks. Four replications per year were grown in a lattice design with four-row plots 6 meters long. The association panel P2 was divided into four groups based on the maturity group I to IV, and were evaluated for SDS <span class="trait">resistance</span> during the summers of 2011 and 2012 in 14 locations including Michigan (Decatur), Iowa (Kanawha and Ames), Minnesota (Waseca and Rosemount), Illinois (Manito, Streator, Fairbury, Beardstown, Urbana, Shawnee town and Valmeyer), Missouri (Sikeston) and Ontario, Canada (Harrow). Four replications per year were grown in a lattice design. SDS was evaluated by scoring disease incidence (DI) and disease severity (DS) at the R6 growth stage, the stage at which pods contain full-size green beans at one of the four uppermost nodes with a completely unrolled leaf [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. SDS leaf scorch DI was rated from 0% (no disease) to 100% (all plants symptomatic), and DS was measured on a scale from 1 to 9 as described in Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> (after Bond, J. <italic> unpublished </italic> ). The SDS disease index (DX, 0&#x02013;100) was calculated as DI&#x02009;&#x000d7;&#x02009;DS/9. In panel P1, mean values of DI, DS and DX across replicates and years were used in association analysis throughout the study. The trait distribution for DX and DI was slightly skewed towards susceptible, thus a square <span class="trait">root</span> transformation was used to normalize the trait distribution prior to further analysis. The association panel P2 was phenotyped in multiple environments; best linear unbiased predictors (BLUPs) were used for the overall association analysis in panel P2. The BLUPs for each line were calculated with the R package, lme4, using the equation <italic> Y </italic> <sub> <italic> ijk </italic> </sub> &#x02009;=&#x02009;Line <sub> <italic> k </italic> </sub> &#x02009;+&#x02009;Environment <sub> <italic> i </italic> </sub> &#x02009;+&#x02009;Replicate (Environment) <sub> <italic> ij </italic> </sub> &#x02009;+&#x02009;(Line&#x02009;&#x000d7;&#x02009;Environment) <sub> <italic> ik </italic> </sub> &#x02009;+&#x02009;&#x003f5; <sub> <italic> ijk </italic> </sub> , where <italic> Y </italic> <sub> <italic> ijk </italic> </sub> is the observed phenotype for the <italic> k </italic> <sup> th </sup> line in the <italic> j </italic> <sup> th </sup> replicate of the <italic> i </italic> <sup> th </sup> environment; Line <sub> k </sub> is the random effect of the <italic> k </italic> <sup> th </sup> line; Environment <sub> <italic> i </italic> </sub> is the random effect of the <italic> i </italic> <sup> th </sup> environment; Replicate (Environment) <sub> <italic> ij </italic> </sub> is the random effect of the <italic> j </italic> <sup> th </sup> replicate in the <italic> i </italic> <sup> th </sup> environment; (Environment&#x02009;&#x000d7;&#x02009;Line) <sub> <italic> ik </italic> </sub> is the random interaction effect of the <italic> i </italic> <sup> th </sup> environment and the <italic> k </italic> <sup> th </sup> line, and &#x003f5; <sub> <italic> ijk </italic> </sub> is the error term. Analysis of variance (ANOVA) for the phenotypic data was performed with the R package, lm(stats) and anova.lm(stats). The heritability estimates were calculated using variance components obtained by ANOVA [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In both association panels, we observed abundant phenotypic variation in SDS <span class="trait">resistance</span> measured by disease incidence (DI), disease severity (DS) and calculated disease index (DX, see Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> ). The mean DX distribution ranged from 0 to 96.3 in panel <span class="gene">P1</span> and 0 to 82.0 in panel P2. The broad-sense heritability of DX was higher within two environments (two years in one location) in panel <span class="gene">P1</span> (83%) than that across multiple environments (two years across 14 locations) in panel P2 (average 65%) [see Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To characterize the mapping resolution for genome scans and GWA mapping, we quantified the average extent of genome-wide LD decay distance in panel <span class="gene">P1</span> and P2. These estimates were approximately 270&#x000a0;kb and 460&#x000a0;kb, respectively, where the <italic> r </italic> <sup> <italic> 2 </italic> </sup> drops to half its maximum value (0.24 and 0.19, respectively). Given that our average inter-marker distance (density) is 35&#x000a0;kb and 241&#x000a0;kb for the panels <span class="gene">P1</span> and the P2 respectively, we expect to have reasonable power to identify common large effect variants associated with SDS <span class="trait">resistance</span> in both association panels. Overall LD decay distance in panel <span class="gene">P1</span> was smaller than that in the panel P2 (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). LD decay distance in panel <span class="gene">P1</span> was also smaller than previously published values in soybean [ <xref ref-type="bibr" rid="CR33"> 33 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. This difference may be attributed to smaller sample size and lower genome coverage of markers in P2 and previous studies. Since panel <span class="gene">P1</span> had larger sample size and was genotyped with more SNP markers than panel P2, estimation of LD in panel <span class="gene">P1</span> is more reliable. Linkage disequilibrium decay distance varies over different chromosomes, with 410&#x000a0;kb in chromosome 19, 100&#x02013;200&#x000a0;kb in chromosomes 3, 4, 9, 10, 11 and 16, and 200&#x02013;300&#x000a0;kb in the remaining 13 chromosomes in panel P1. These LD decay estimates are slightly higher than that in rice (75-150&#x000a0;kb) [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ], but much greater than in maize (1.5-10&#x000a0;kb) [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. This result is consistent with earlier estimate that LD extends to a much longer distance in self-pollinated species than in cross-pollinated species [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. <fig id="Fig1"> <label> Figure 1 </label> <caption> <p> <bold> Genome-wide average LD decay estimated in association panels <span class="gene">P1</span> and P2. </bold> Decay of LD (measured as genotypic <italic> r </italic> <sup> 2 </sup> ) as a function of distance between SNPs.
					</p>
				
					<p>&nbsp;&nbsp;
						Notably, one of the overlaps is the QTL <italic> Rfs2/Rhg1 </italic> on chromosome 18. This locus consistently contributes more effective coinheritance of <span class="trait">resistance</span> to SDS and reduces infestation by SCN [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. Previous fine map development did not resolve <italic> Rfs2 </italic> from <italic> <span class="gene">Rhg1</span> </italic> , suggesting that the underlying genes were either very closely linked or pleiotropic [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. In this study, we did detect a cluster of associations spanning a physical region of 1.2&#x000a0;Mb (1.2-2.4&#x000a0;Mb) around three <italic> <span class="gene">Rhg1</span> </italic> genes that were found to contribute to SCN <span class="trait">resistance</span> (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> a, Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> , <xref rid="MOESM10" ref-type="media"> 10 </xref> , <xref rid="MOESM11" ref-type="media"> 11 </xref> , <xref rid="MOESM12" ref-type="media"> 12 </xref> ) [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. The cluster of associations also explained a major part of phenotypic variation of SDS <span class="trait">resistance</span> in both panels (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> , <xref rid="MOESM10" ref-type="media"> 10 </xref> , <xref rid="MOESM11" ref-type="media"> 11 </xref> , <xref rid="MOESM12" ref-type="media"> 12 </xref> ). If the three <italic> <span class="gene">Rhg1</span> </italic> genes were pleiotropic, the peak SNP for SDS <span class="trait">resistance</span> should be located either within or in the same LD block with the three <italic> <span class="gene">Rhg1</span> </italic> genes. However, the peak SNP <span class="gene">(GM18-1709751)</span> was not only located outside of, but also belonged to a different LD block than the three <italic> <span class="gene">Rhg1</span> </italic> genes (Additional file <xref rid="MOESM15" ref-type="media"> 15 </xref> ). One possible explanation for this is that SDS <span class="trait">resistance</span> mediated by <italic> <span class="gene">Rhg1</span> </italic> is also conferred by copy number variation (CNV) that increases the expression of a set of dissimilar genes. Alternatively, there exist other gene/s mediating SDS <span class="trait">resistance</span> that are closely linked with <italic> <span class="gene">Rhg1</span> </italic> . In fact, we found that the peak SNP <span class="gene">(GM18-1709751)</span> was located at approximately 2.2&#x000a0;kb upstream of <italic> <span class="gene">GmRLK18-1</span> </italic> , a gene that encodes a receptor-like kinase, and its <span class="trait">resistance</span> allele is sufficient to confer nearly complete <span class="trait">resistance</span> to both <span class="trait">root</span> and leaf symptoms of SDS [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. Moreover, there was another significant SNP <span class="gene">(GM18-1712832</span> with <italic> P </italic> -value of 1.2&#x02009;&#x000d7;&#x02009;10 <sup> -8 </sup> for DX) located within an exon of <italic> <span class="gene">GmRLK18-1</span> </italic> . These results support previous studies with regard to the key role of <italic> <span class="gene">GmRLK18-1</span> </italic> . However, we cannot exclude the possibility that structural variation in the form of CNV may have functional importance and thus contribute to SDS <span class="trait">resistance</span> that is not captured by our SNPs. <fig id="Fig5"> <label> Figure 5 </label> <caption> <p> <bold> Regional plots showing association mapping results for SNPs located around </bold> <bold> <italic> Rfs2 </italic> </bold> <bold> / </bold> <bold> <italic> <span class="gene">Rhg1</span> </italic> </bold> <bold> on chromosome 18. </bold> Before <bold> (a) </bold> and after <bold> (b) </bold> controlling for the effects of peak SNP, negative log10-transformed <italic> P- </italic> values from the MLM are plotted on the left vertical axis for panel P1; Negative log10-transformed <italic> P- </italic> values from the MLM are plotted on the right vertical axis for association panel P2. Blue horizontal dashed lines indicate the genome-wide significance threshold in <span class="gene">P1</span> association panel. Previously identified genes controlling the SDS <span class="trait">resistance</span> are labeled.
					</p>
				
					<p>&nbsp;&nbsp;
						We searched for additional independently associated SNP variants near the <italic> Rfs2/Rhg1 </italic> locus by conditioning on the peak SNP <span class="gene">(GM18-1709751)</span> at the <italic> Rfs2/Rhg1 </italic> loci. At 519&#x000a0;kb downstream of <italic> Rfs2/Rhg1 </italic> , we found an independently associated SNP variant, <span class="gene">Gm18-2228646</span> with <italic> P </italic> -value of 6.5&#x02009;&#x000d7;&#x02009;10 <sup> -5 </sup> in the conditional model, as measured by DS. This SNP is located at 44&#x000a0;bp downstream of a gene encoding an aquaporin transporter (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> b). Therefore, our results provide evidence for the presence of a regulating gene other than <italic> <span class="gene">GmRLK18-</span> </italic> 1 that is associated with SDS <span class="trait">resistance</span> on chromosome 18.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic evaluation and genetic dissection of <span class="trait">resistance</span> to <italic> Phytophthora sojae </italic> in the Chinese soybean mini core collection</h2>
			<h3>PMID:27316671</h3>
				
					<h3>PMCID:4912746</h3>
					<h4>期刊:BMC Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Na </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Jutao </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Guanjun </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Haipeng </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yanfei </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xing </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Jinming </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Xing </surname>  <given-names> Han </given-names>  </name>  <address>  <email> hanx@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> National Center for Soybean Improvement/National Key laboratory of Crop Genetics and Germplasm enhancement, Key laboratory of Biology and Genetics and Breeding for Soybean, Ministry of Agriculture, Nanjing Agricultural University, Nanjing, 210095 People&#x02019;s Republic of China  </aff>  <aff id="Aff2">  <label/> The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The most significant association with <span class="trait">resistance</span> to the HLJ08-35 isolate was on <span class="gene">Gm15.</span> The <span class="trait">resistance</span> trait for the P7063 isolate had associations in the same region with two SCN-QTLs [ <xref ref-type="bibr" rid="CR43"> 43 </xref> , <xref ref-type="bibr" rid="CR44"> 44 </xref> ], on <span class="gene">Gm03</span> and <span class="gene">Gm15,</span> respectively. One <span class="trait">resistance</span> association of the H15 isolate was on <span class="gene">Gm16,</span> within a brown stem <span class="trait">rot</span> (BSR) <span class="trait">resistance</span> QTL [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ] interval. For the HeN08-35 isolate, two significant associations were on <span class="gene">Gm16</span> and <span class="gene">Gm20,</span> respectively. The <span class="trait">resistance</span> to the PNJ1 isolate showed associations with genome regions on <span class="gene">Gm06</span> and <span class="gene">Gm16,</span> which were co-located with SCN-QTL [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ] and PRR-QTL [ <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. For the Pmg isolate, one marker-trait association on <span class="gene">Gm11</span> overlapped with a sclerotinia stem <span class="trait">rot</span> (SSR) <span class="trait">resistance</span> locus [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ], and one on <span class="gene">Gm16</span> in the same <italic> Rps </italic> Un2 interval [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ] was identified. Two associations with <span class="trait">resistance</span> to the Pm28 isolate were located on <span class="gene">Gm03</span> and <span class="gene">Gm15</span> in the same regions of <italic> Rps </italic> 1 [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ] and PRR-QTL [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. <span class="trait">Resistance</span> to the Pm31 isolate had two associations, on <span class="gene">Gm04</span> and <span class="gene">Gm09.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						We chose genes where the most significantly associated SNP markers were located as candidate genes for associations. <italic> <span class="gene">Glyma15g41680</span> </italic> is a member of the LEM3 (ligand-effect modulator 3) family/CDC50 protein family, and it may be required for phospholipid translocation across the plasma membrane [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. <italic> <span class="gene">Glyma03g28660</span> </italic> is an ADP-ribosylation factor gene. <italic> ADP-ribosylation factor 1 </italic> ( <italic> RARF1 </italic> ) induced pathogenesis-related gene expression and pathogen <span class="trait">resistance</span> when it was over-expressed in tobacco plants [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ], and it acted as a positive regulator of cell death [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. <italic> <span class="gene">Glyma16g30140</span> </italic> is a predicted lipase class 3 gene that may have essential functions in plant defence or priming. Ectopic expression of <italic> AtPAD4 </italic> possessing a lipase 3 motif broadened <span class="trait">resistance</span> to SCN and root-knot nematode (RKN) diseases in the soybean [ <xref ref-type="bibr" rid="CR63"> 63 </xref> ]. <italic> <span class="gene">Glyma16g04700</span> </italic> encodes a thioredoxin protein. Two transcript variants of the <italic> Pi-ta </italic> protein, which coupled the original NBS-LRR domain with a C-terminal thioredoxin domain, showed the highest level of expression in comparison to other transcript variants in rice [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. <italic> <span class="gene">Glyma20g39240</span> </italic> is a DEAD/DEAH box helicase, which may take part in splicing the RNA precursor [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ] and in transcription initiation [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. <italic> <span class="gene">Glyma06g01080</span> </italic> belongs to the 2OG-Fe (II) oxygenase superfamily and may catalyse the formation of plant hormones, such as ethylene and gibberellins. In <italic> Arabidopsis </italic> , the <italic> DMR6 </italic> gene, which has the same domain, is defence-associated and required for susceptibility to downy mildew [ <xref ref-type="bibr" rid="CR67"> 67 </xref> ]. <italic> <span class="gene">Glyma16g14080</span> </italic> and <italic> <span class="gene">Glyma04g40800</span> </italic> encode a serine/threonine protein kinase, which not only has signal transduction activity but also inhibits the release of spores and the germination of the germ tube in <italic> Phytophthora infestans </italic> [ <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. <italic> <span class="gene">Glyma11g11100</span> </italic> encodes an NPH3 family protein, which activates signal transduction and regulates auxin signalling during plant growth [ <xref ref-type="bibr" rid="CR69"> 69 </xref> , <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. <italic> <span class="gene">Glyma16g31930</span> </italic> contains a zinc finger domain, which has been shown to negatively regulate transcription [ <xref ref-type="bibr" rid="CR71"> 71 </xref> , <xref ref-type="bibr" rid="CR72"> 72 </xref> ]. Many NBS-LRR domain genes contain zinc finger domains [ <xref ref-type="bibr" rid="CR73"> 73 </xref> ], and some have been reported to play a role in <span class="trait">resistance</span> to PRR [ <xref ref-type="bibr" rid="CR74"> 74 </xref> ]. <italic> <span class="gene">Glyma03g04960</span> </italic> encodes a lipid transport protein (LTP); lipid signals are essential for the activation of plant defence responses. Many studies have shown that over-expression of LTPs in plants can enhance their <span class="trait">resistance</span> to pathogens [ <xref ref-type="bibr" rid="CR75"> 75 </xref> &#x02013; <xref ref-type="bibr" rid="CR77"> 77 </xref> ], possibly through inhibition of germination of their spores [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ]. <italic> <span class="gene">Glyma09g04310</span> </italic> has an ankyrin repeat and a calmodulin-binding motif. The ankyrin repeat participates in the regulation of transcription [ <xref ref-type="bibr" rid="CR79"> 79 </xref> ]; for example, the <italic> AKP2 </italic> gene in <italic> Arabidopsis </italic> is a negative regulator of <italic> PR-1 </italic> ( <italic> pathogen-related protein 1 </italic> ) expression [ <xref ref-type="bibr" rid="CR80"> 80 </xref> ]. Calmodulin-binding proteins can detect external stimuli and regulate systemic acquired <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR81"> 81 </xref> , <xref ref-type="bibr" rid="CR82"> 82 </xref> ]. Over-expression of <italic> <span class="gene">GmCaM4</span> </italic> in the soybean resulted in enhanced <span class="trait">resistance</span> to three pathogens through increased expression of pathogenesis-related (PR) genes [ <xref ref-type="bibr" rid="CR83"> 83 </xref> ]. In future studies, the functions of these genes in the <span class="trait">resistance</span> process will be examined.
					</p>
				
					<p>&nbsp;&nbsp;
						Some studies have inferred that some <italic> Rps </italic> genes contain NBS-LRR domains [ <xref ref-type="bibr" rid="CR13"> 13 </xref> , <xref ref-type="bibr" rid="CR58"> 58 </xref> , <xref ref-type="bibr" rid="CR59"> 59 </xref> ]. In our study, no such NBS-LRR-domain gene was found among the candidates, which may due to the complex mechanism of soybean <span class="trait">resistance.</span> This situation has been found in other disease studies. For instance, soybean <span class="trait">resistance</span> to SCN through the <italic> Rgh1 </italic> locus is conferred through copy number variation, but the <span class="gene">Glyma18g02680.1</span> gene in the <italic> <span class="gene">Rhg1</span> </italic> locus, which encodes an LRR-kinase, does not contribute to SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR84"> 84 </xref> , <xref ref-type="bibr" rid="CR85"> 85 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Deletion of the Phytophthora sojae avirulence gene <span class="gene">Avr1d</span> causes gain of virulence on Rps1d.</h2>
			<h3>PMID:23550527  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Na R(1), Yu D, Qutob D, Zhao J, Gijzen M.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora sojae is an oomycete and a pathogen of soybean that causes <span class="trait">root</span> <span class="trait">rot.</span> During infection P. sojae delivers effector proteins into host cells to foster disease. However, effector-triggered immunity (ETI) results when pathogen factors are recognized by host <span class="trait">resistance</span> (R) proteins. We have now identified the P. sojae <span class="gene">Avr1d</span> gene, which encodes a predicted effector protein with the amino acid motif Arg-X-Leu-Arg (RXLR). Genetic mapping of 16 different P. sojae isolates and of a segregating F2 population of 40 individuals shows that the predicted RXLR effector gene Avh6 precisely cosegregates with the <span class="gene">Avr1d</span> phenotype. Transient expression assays confirm that <span class="gene">Avr1d</span> triggers cell death specifically in Rps1d soybean plants. The <span class="gene">Avr1d</span> gene is present in P. sojae strains that are avirulent on Rps1d, whereas the gene is deleted from the genome of virulent strains. Two sequence variants of the <span class="gene">Avr1d</span> gene encoding different protein products occur in P. sojae strains, but both are recognized by Rps1d and cause ETI. Liposome binding assays show that <span class="gene">Avr1d</span> has affinity for phosphatidylinositol 4-phosphate and that binding can be disrupted by mutation of lysine residues in the carboxy-terminal effector domain of the protein. The identification of <span class="gene">Avr1d</span> aids pathogen diagnostics and soybean cultivar development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Different domains of Phytophthora sojae effector Avr4/6 are recognized by soybean <span class="trait">resistance</span> genes Rps4 and Rps6.</h2>
			<h3>PMID:20192830  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dou D(1), Kale SD, Liu T, Tang Q, Wang X, Arredondo FD, Basnayake S, Whisson S, Drenth A, Maclean D, Tyler BM.
				<br/> 
				年份：2010 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						At least 12 avirulence genes have been genetically identified and mapped in Phytophthora sojae, an oomycete pathogen causing <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean. Previously, the <span class="gene">Avr4</span> and Avr6 genes of P. sojae were genetically mapped within a 24 kb interval of the genome. Here, we identify <span class="gene">Avr4</span> and Avr6 and show that they are actually a single gene, Avr4/6, located near the 24-kb region. Avr4/6 encodes a secreted protein of 123 amino acids with an RXLR-dEER protein translocation motif. Transient expression of Avr4/6 in soybean leaves revealed that its gene product could trigger a hypersensitive response (HR) in the presence of either Rps4 or Rps6. Silencing Avr4/6 in P. sojae stable transformants abolished the avirulence phenotype exhibited on both Rps4 and Rps6 soybean cultivars. The N terminus of Avr4/6, including the dEER motif, is sufficient to trigger Rps4-dependent HR while its C terminus is sufficient to trigger Rps6-mediated HR. Compared with alleles from avirulent races, alleles of Avr4/6 from virulent races possess nucleotide substitutions in the 5' untranslated region of the gene but not in the protein-coding region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmWRKY31 and <span class="gene">GmHDL56</span> Enhances <span class="trait">Resistance</span> to <italic> Phytophthora sojae </italic> by Regulating Defense-Related Gene Expression in Soybean</h2>
			<h3>PMID:28553307</h3>
				

				
					<h3>PMCID:5427154</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/417953/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Junjiang </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Rongpeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Wencheng </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meng </surname>  <given-names> Fanshan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257714/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/418977/overview"/>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean [Glycine max (L.) Merr.] caused by the oomycete Phytophthora sojae, is a destructive disease worldwide. The molecular mechanism of the soybean response to P. sojae is largely unclear. We report a novel WRKY transcription factor (TF) in soybean, <span class="gene">GmWRKY31,</span> in the host response to P. sojae. Overexpression and RNA interference analysis demonstrated that <span class="gene">GmWRKY31</span> enhanced <span class="trait">resistance</span> to P. sojae in transgenic soybean plants. <span class="gene">GmWRKY31</span> was targeted to the nucleus, where it bound to the W-box and acted as an activator of gene transcription. Moreover, we determined that <span class="gene">GmWRKY31</span> physically interacted with <span class="gene">GmHDL56,</span> which improved <span class="trait">resistance</span> to P. sojae in transgenic soybean roots. <span class="gene">GmWRKY31</span> and <span class="gene">GmHDL56</span> shared a common target <span class="gene">GmNPR1</span> which was induced by P. sojae. Overexpression and RNA interference analysis demonstrated that <span class="gene">GmNPR1</span> enhanced <span class="trait">resistance</span> to P. sojae in transgenic soybean plants. Several pathogenesis-related (PR) genes were constitutively activated, including <span class="gene">GmPR1a,</span> <span class="gene">GmPR2,</span> <span class="gene">GmPR3,</span> <span class="gene">GmPR4,</span> <span class="gene">GmPR5a,</span> and <span class="gene">GmPR10,</span> in soybean plants overexpressing <span class="gene">GmNPR1</span> transcripts. By contrast, the induction of PR genes was compromised in transgenic <span class="gene">GmNPR1-RNAi</span> lines. Taken together, these findings suggested that the interaction between <span class="gene">GmWRKY31</span> and <span class="gene">GmHDL56</span> enhances <span class="trait">resistance</span> to P. sojae by regulating defense-related gene expression in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seventy-four WRKY TFs have been reported in Arabidopsis ( <xref rid="B65" ref-type="bibr"> Pandey and Somssich, 2009 </xref> ; <xref rid="B6" ref-type="bibr"> Bakshi and Oelm&#x000fc;ller, 2014 </xref> ), and 109 WRKY TFs have been identified in rice, including four with incomplete WRKY domains ( <xref rid="B106" ref-type="bibr"> Zhang and Wang, 2005 </xref> ). In Arabidopsis, WRKY TFs play a crucial role in the response to pathogen infection ( <xref rid="B3" ref-type="bibr"> Abuqamar et al., 2006 </xref> ; <xref rid="B41" ref-type="bibr"> Journot-Catalino et al., 2006 </xref> ; <xref rid="B49" ref-type="bibr"> Li et al., 2006 </xref> ; <xref rid="B99" ref-type="bibr"> Xu et al., 2006 </xref> ; <xref rid="B45" ref-type="bibr"> Knoth et al., 2007 </xref> ; <xref rid="B54" ref-type="bibr"> Lippok et al., 2007 </xref> ; <xref rid="B64" ref-type="bibr"> Murray et al., 2007 </xref> ; <xref rid="B107" ref-type="bibr"> Zheng et al., 2007 </xref> ; <xref rid="B34" ref-type="bibr"> Hu et al., 2008 </xref> ; <xref rid="B62" ref-type="bibr"> Mao et al., 2011 </xref> ). Overexpression of <italic> AtWRKY70 </italic> leads to the up-regulation of <italic> PR </italic> genes and the down-regulation of <italic> PDF1.2 </italic> , leading to enhanced <span class="trait">resistance</span> against biotrophic pathogens and increased susceptibility to constitutive necrotrophic pathogens ( <xref rid="B50" ref-type="bibr"> Li et al., 2004 </xref> ). In addition, mutations of the Arabidopsis <italic> WRKY33 </italic> gene enhance susceptibility to the necrotrophic fungal pathogens <italic> Botrytis cinerea </italic> and <italic> Alternaria brassicicola </italic> . By contrast, ectopic over-expression of <italic> WRKY33 </italic> increases <span class="trait">resistance</span> to these two necrotrophic fungal pathogens ( <xref rid="B108" ref-type="bibr"> Zheng et al., 2006 </xref> ). Arabidopsis plants overexpressing <italic> AtWRKY28 </italic> and <italic> AtWRK75 </italic> exhibit enhanced <span class="trait">resistance</span> to <italic> Sclerotinia sclerotiorum </italic> ( <xref rid="B15" ref-type="bibr"> Chen et al., 2013 </xref> ). Several studies have also suggested the importance of specific <italic> OsWRKY </italic> s in the transcriptional regulation of defense-related genes in the rice response to pathogens ( <xref rid="B35" ref-type="bibr"> Hwang et al., 2011 </xref> ; <xref rid="B48" ref-type="bibr"> Lee et al., 2013 </xref> ; <xref rid="B17" ref-type="bibr"> Choi et al., 2015 </xref> ). For example, <italic> OsWRKY55 </italic> ( <italic> OsWRKY31 </italic> according to the nomenclature of <xref rid="B96" ref-type="bibr"> Wu et al. (2005) </xref> is induced by the rice blast fungus, and the transgenic rice plants over-expressing <italic> OsWRKY22 </italic> are more resistant to <italic> M. oryzae </italic> ( <xref rid="B103" ref-type="bibr"> Zhang et al., 2008 </xref> ). <italic> OsWRKY6 </italic> plays a role in the pathogen- or salicylic acid (SA)-inducible expression of <italic> OsPR1a </italic> . Indeed, <italic> OsWRKY6 </italic> is induced by <italic> Xanthomonas oryzae </italic> pv. <italic> oryzae </italic> ( <italic> Xoo </italic> ) and SA and activates the <italic> OsPR1a </italic> promoter in rice ( <xref rid="B35" ref-type="bibr"> Hwang et al., 2011 </xref> ). <italic> OsWRKY22 </italic> is a regulator of rice <span class="trait">resistance</span> to <italic> M. oryzae </italic> and is also involved in the rice response to non-host barley powdery mildew ( <xref rid="B1" ref-type="bibr"> Abbruscato et al., 2012 </xref> ). A total of 197 WRKY genes have been identified in soybean [ <italic> G. max </italic> (L.) Merr.] ( <xref rid="B77" ref-type="bibr"> Schmutz et al., 2010 </xref> ), but only four members of the WRKY family in this species have been functionally characterized. Over-expressing <italic> <span class="gene">GmWRKY21</span> </italic> or <italic> <span class="gene">GmWRKY54</span> </italic> enhances cold tolerance and salt and <span class="trait">drought</span> tolerance in Arabidopsis. By contrast, over- expression of <italic> <span class="gene">GmWRKY13</span> </italic> leads to increased sensitivity to salt and mannitol stress ( <xref rid="B111" ref-type="bibr"> Zhou et al., 2008 </xref> ). Transgenic soybean hairy roots overexpressing the <italic> <span class="gene">GmWRKY27</span> </italic> gene exhibit enhanced tolerance to <span class="trait">drought</span> and salt stresses, whereas <italic> <span class="gene">GmWRKY27</span> </italic> RNAi roots exhibit hypersensitivity to these stresses ( <xref rid="B93" ref-type="bibr"> Wang F. et al., 2015 </xref> ). Although a role of WRKY TFs in abiotic stress responses in soybean has been identified, the potential function of these TFs in biotic stress responses remains unclear.
					</p>
				
					<p>&nbsp;&nbsp;
						Homeodomain leucine-zipper (HD-ZIP) TFs, several are known to be rapidly induced in response to altered environmental conditions and to integrate hormonal signals ( <xref rid="B7" ref-type="bibr"> Brandt et al., 2014 </xref> ). Members of the HD-ZIP proteins play roles related to <span class="trait">drought</span> stress and ABA-signaling in different plant species ( <xref rid="B22" ref-type="bibr"> Dezar et al., 2005 </xref> ; <xref rid="B21" ref-type="bibr"> Deng et al., 2006 </xref> ; <xref rid="B4" ref-type="bibr"> Agalou et al., 2008 </xref> ). Expression of the HD-ZIP genes AtHB6 and AtHB7 is induced by ABA application or water deficiency ( <xref rid="B82" ref-type="bibr"> S&#x000f6;derman et al., 1996 </xref> ; <xref rid="B33" ref-type="bibr"> Himmelbach et al., 2002 </xref> ; <xref rid="B47" ref-type="bibr"> Lechner et al., 2011 </xref> ). Recently, it was shown that both AtHB7 and AtHB12 act to repress the transcription of genes encoding the ABA receptors PYL5 and PYL8 in response to an ABA stimulus ( <xref rid="B90" ref-type="bibr"> Vald&#x000e9;s et al., 2012 </xref> ). In soybean, it has been demonstrated that <span class="gene">GmHDL56</span> and <span class="gene">GmHDL57</span> could interact with the -611 to -451 portion of the <italic> VspB </italic> promoter ( <xref rid="B86" ref-type="bibr"> Tang et al., 2001 </xref> ). However, the molecular mechanism of HD-ZIP proteins in response to pathogen infection still remains elusive.
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous study, a cDNA library of mRNAs encoding expressed sequence tags (ESTs) exhibiting increased expression during <italic> P. sojae </italic> infection was constructed using suppression subtractive hybridization (SSH) from the leaf tissues of the highly resistant soybean &#x02018;Suinong 10&#x02019;. A novel EST homologous to the WRKY TF <italic> AtWRKY31 </italic> (GenBank accession no. NM_118328.3) was significantly induced after infection with <italic> P. sojae </italic> ( <xref rid="B97" ref-type="bibr"> Xu et al., 2012 </xref> ). In this study, we isolated this WRKY TF, designated <italic> <span class="gene">GmWRKY31</span> </italic> (GenBank accession no. XM_003546112.3), from soybean &#x02018;Suinong 10&#x02019; and studied the expression and function of this gene. Transgenic soybean plants overexpressing <italic> <span class="gene">GmWRKY31</span> </italic> exhibited enhanced <span class="trait">resistance</span> to <italic> P. sojae </italic> , whereas RNA interference (RNAi) in transgenic soybean plants increased susceptibility to <italic> P. sojae </italic> . Further analysis indicated that <span class="gene">GmWRKY31</span> physically interacted with <span class="gene">GmHDL56</span> which improved <span class="trait">resistance</span> to <italic> P. sojae </italic> in transgenic soybean roots. <italic> <span class="gene">GmWRKY31</span> </italic> and <italic> <span class="gene">GmHDL56</span> </italic> shared a common target <italic> <span class="gene">GmNPR1</span> </italic> (GenBank accession no. NM_001251745.1). <italic> <span class="gene">GmWRKY31</span> </italic> and <italic> <span class="gene">GmHDL56</span> </italic> coregulated the expression of <italic> <span class="gene">GmNPR1</span> </italic> gene, leading to improved soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> . These data suggested that <span class="gene">GmWRKY31</span> interacted with <span class="gene">GmHDL56</span> in response to <italic> P. sojae </italic> infection via the activation of <italic> <span class="gene">GmNPR1</span> </italic> expression.
					</p>
				
					<p>&nbsp;&nbsp;
						The living cotyledons of those transgenic soybean plants were selected to investigate <span class="trait">resistance</span> to <italic> P. sojae </italic> . After 2 days of incubation with zoospores of <italic> P. sojae </italic> , a remarkable difference in the development of disease symptoms was observed ( <bold> Figure <xref ref-type="fig" rid="F2"> 2C </xref> </bold> ). In <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi lines, the cotyledons became soft, emitted a foul odor and exhibited clear water-soaked lesions compared with those of the <italic> <span class="gene">GmWRKY31</span> </italic> -OE lines, and there were nearly no visible lesions in <italic> <span class="gene">GmWRKY31</span> </italic> -OE lines and the cotyledons keep hard compared with those of the <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi lines or WT. Meanwhile, tthe relative biomass of <italic> P. sojae </italic> in infected living cotyledons after 2 days of incubation with zoospores of <italic> P. sojae </italic> was also analyzed. The biomass of <italic> P. sojae </italic> based on the transcript level of the <italic> P. sojae TEF1 </italic> gene (GenBank accession no. EU079791) was significantly ( <italic> P </italic> &#x0003c; 0.01) lower in <italic> <span class="gene">GmWRKY31</span> </italic> -OE lines than that in WT, and it was higher in <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi lines than that in WT but did not reach a significant level ( <bold> Figure <xref ref-type="fig" rid="F2"> 2D </xref> </bold> ). Similar results were obtained after 5 days of incubation with <italic> P. sojae </italic> , and the living leaves of the WT and <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi soybean plants exhibited clear, large lesions compared with those of the <italic> <span class="gene">GmWRKY31</span> </italic> -OE lines ( <bold> Figure <xref ref-type="fig" rid="F2"> 2E </xref> </bold> ). The lesion area of the <italic> <span class="gene">GmWRKY31</span> </italic> -OE lines was significantly ( <italic> P </italic> &#x0003c; 0.01) smaller than that of WT soybean plants after 5 days of incubation with <italic> P. sojae </italic> ( <bold> Figure <xref ref-type="fig" rid="F2"> 2F </xref> </bold> ). These results indicate that overexpression of <italic> <span class="gene">GmWRKY31</span> </italic> in soybean plants improves <span class="trait">resistance</span> to <italic> P. sojae </italic> and that <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi transgenic soybean plants have increased susceptibility to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						The interaction between <span class="gene">GmWRKY31</span> and <span class="gene">GmHDL56</span> raised the question of whether <span class="gene">GmHDL56</span> was also required for <italic> P. sojae </italic> responses in soybean. To this end, we examined the phenotype of <span class="gene">GmHDL56</span> in response to <italic> P. sojae </italic> infection. The plant over-expressing vector <italic> 35S:GmHDL56 </italic> was constructed and transformed into soybean hairy roots by high-efficiency <italic> Agrobacterium rhizogenes </italic> -mediated transformation ( <xref rid="B42" ref-type="bibr"> Kereszt et al., 2007 </xref> ). The transgenic soybean hairy roots were tested using LibertyLink strips ( <bold> Figure <xref ref-type="fig" rid="F4"> 4A </xref> </bold> ) and qRT-PCR ( <bold> Figure <xref ref-type="fig" rid="F4"> 4B </xref> </bold> ). Three overexpression hairy roots (OE101, OE106, and OE108) were selected to investigate <span class="trait">resistance</span> to <italic> P. sojae </italic> . After 5 days of incubation with zoospores of <italic> P. sojae </italic> , three <italic> <span class="gene">GmHDL56</span> </italic> -overexpressing lines (OE101, OE106, and OE108) displayed significantly enhanced <span class="trait">resistance</span> compared with the wild-type plants ( <bold> Figure <xref ref-type="fig" rid="F4"> 4C </xref> </bold> ). Moreover, we analyzed the relative biomass of <italic> P. sojae </italic> in soybean <span class="trait">root</span> based on the transcript level of the <italic> P. sojae TEF1 </italic> gene. The results indicated that the <italic> P. sojae </italic> biomass was lower in <italic> <span class="gene">GmHDL56</span> </italic> -overexpressing lines than that in wild-type plants ( <bold> Figure <xref ref-type="fig" rid="F4"> 4D </xref> </bold> ). These data suggest that <span class="gene">GmHDL56</span> positively regulates the soybean defense in response to <italic> P. sojae </italic> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="gene">GmHDL56</span> enhances <span class="trait">resistance</span> to <italic> P. sojae </italic> in transgenic soybean hairy roots. </bold> <bold> (A) </bold> Transgenic soybean plants were tested using Liberty Link strips. <bold> (B) </bold> Quantitative real-time PCR of transgenic soybean hairy roots overexpressing <italic> <span class="gene">GmHDL56</span> </italic> and WT. <italic> <span class="gene">GmEF1b</span> </italic> gene (NM_001248778) was used as an internal control to normalize all the data. The experiments were performed on three biological replicates with their respective three technical replicates and statistically analyzed using Student&#x02019;s <italic> t </italic> -test ( <sup> &#x02217; </sup> <italic> P </italic> &#x0003c; 0.05, <sup> &#x02217;&#x02217; </sup> <italic> P </italic> &#x0003c; 0.01). Bars indicate standard error of the mean. <bold> (C) </bold> Disease symptoms on the hairy roots of the transgenic lines and WT treated with zoospores of <italic> P. sojae </italic> at 0 and 5 days. <bold> (D) </bold> Accumulation of <italic> P. sojae </italic> biomass in the hairy roots of the transgenic soybean lines and WT. Transcript levels of <italic> P. sojae TEF1 </italic> (EU079791) in infected hairy roots (5 days) were plotted relative to soybean <italic> <span class="gene">GmEF1b</span> </italic> expression levels as determined by qRT-PCR. The experiments were performed on three biological replicates with their respective three technical replicates and statistically analyzed using Student&#x02019;s <italic> t </italic> -test ( <sup> &#x02217; </sup> <italic> P </italic> &#x0003c; 0.05, <sup> &#x02217;&#x02217; </sup> <italic> P </italic> &#x0003c; 0.01). Bars indicated standard error of the mean.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine whether the <span class="trait">resistance</span> to <italic> P. sojae </italic> phenotypes in <italic> <span class="gene">GmWRKY31</span> </italic> transgenic soybean plants is related to <italic> <span class="gene">GmNPR1</span> </italic> expression, we produced <italic> <span class="gene">GmNPR1</span> </italic> -overexpressing ( <italic> <span class="gene">GmNPR1</span> </italic> -OE) and RNAi ( <italic> <span class="gene">GmNPR1</span> </italic> -RNAi) transgenic plants. Southern blot results showed that three T2 <italic> <span class="gene">GmNPR1</span> </italic> -overexpressing transgenic lines ( <italic> <span class="gene">GmNPR1</span> </italic> -OE) and three T2 <italic> <span class="gene">GmNPR1</span> </italic> -RNAi transgenic soybean lines were integrated into the genomes of the three transgenic lines with one copy, respectively ( <bold> Figure <xref ref-type="fig" rid="F7"> 7A </xref> </bold> ). These transgenic lines were developed into T3 transgenic soybean plants. Expression analysis of the transgenic population led to the identification of lines with remarkably increased <italic> <span class="gene">GmNPR1</span> </italic> mRNA levels, and the expression analysis of T3 <italic> <span class="gene">GmWRKY31</span> </italic> -RNAi transgenic soybean plants led to the identification of lines with significantly reduced <italic> <span class="gene">GmNPR1</span> </italic> mRNA levels ( <bold> Figure <xref ref-type="fig" rid="F7"> 7B </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The living cotyledons of the <italic> <span class="gene">GmNPR1</span> </italic> -OE and <italic> <span class="gene">GmNPR1</span> </italic> -RNAi transgenic soybean plants were selected to investigate the <span class="trait">resistance</span> to <italic> P. sojae </italic> . After 2 days of incubation with zoospores of <italic> P. sojae </italic> , the remarkable differences in the development of disease symptoms were observed ( <bold> Figure <xref ref-type="fig" rid="F7"> 7C </xref> </bold> ). In <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines, the cotyledons became soft, emitted a foul odor and exhibited clear water-soaked lesions compared with those of the <italic> <span class="gene">GmNPR1</span> </italic> -OE lines. However, there were nearly no visible lesions in <italic> <span class="gene">GmNPR1</span> </italic> -OE lines, and the cotyledons keep hard compared with those of the <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines or WT. Meanwhile, the relative biomass of <italic> P. sojae </italic> in infected living cotyledons after 2 days of incubation with zoospores of <italic> P. sojae </italic> was also analyzed. The biomass of <italic> P. sojae </italic> based on the transcript level of the <italic> P. sojae TEF1 </italic> gene was significantly ( <italic> P </italic> &#x0003c; 0.01) lower in <italic> <span class="gene">GmNPR1</span> </italic> -OE lines than that in WT, and it was higher in <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines than that in WT but did not reach a significant level ( <bold> Figure <xref ref-type="fig" rid="F7"> 7D </xref> </bold> ). Similar results were obtained after 5 days of incubation with <italic> P. sojae </italic> , and the ability of living leaves to resist <italic> P. sojae </italic> significantly increased in the <italic> <span class="gene">GmNPR1</span> </italic> -OE lines and decreased in the <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines compared with WT ( <bold> Figure <xref ref-type="fig" rid="F7"> 7E </xref> </bold> ). The lesion area of the <italic> <span class="gene">GmNPR1</span> </italic> -OE lines was significantly ( <italic> P </italic> &#x0003c; 0.01) smaller and the lesion area of the <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines was larger than that of WT after 5 days infection with <italic> P. sojae </italic> ( <bold> Figure <xref ref-type="fig" rid="F7"> 7F </xref> </bold> ). These results indicate that the overexpression of <italic> <span class="gene">GmNPR1</span> </italic> in soybean plants improved <span class="trait">resistance</span> to <italic> P. sojae </italic> , whereas susceptibility to <italic> P. sojae </italic> was enhanced in <italic> <span class="gene">GmNPR1</span> </italic> -RNAi transgenic soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Since the first WRKY protein (SPF1) was identified in sweet potato ( <xref rid="B74" ref-type="bibr"> Rushton and Somssich, 1998 </xref> ), much progress has been made in studying the functions of WRKY family TFs. In model plants, the functional roles and underlying mechanisms of many WRKY TFs in pathogen responses have been studied ( <xref rid="B12" ref-type="bibr"> Chen and Chen, 2002 </xref> ; <xref rid="B43" ref-type="bibr"> Kim et al., 2006 </xref> ; <xref rid="B13" ref-type="bibr"> Chen et al., 2010 </xref> ; <xref rid="B1" ref-type="bibr"> Abbruscato et al., 2012 </xref> ), but little is known about the functional roles of WRKYs and their mechanisms in soybean. In this study, we determined that <italic> <span class="gene">GmWRKY31</span> </italic> , a new member of the soybean WRKY family of TFs, plays a crucial role in soybean during infection with <italic> P. sojae </italic> . To cope with pathogenic challenge, plants rapidly activate defense responses regulated by the major signaling molecules SA, JA, and ET ( <xref rid="B71" ref-type="bibr"> Reymond and Farmer, 1998 </xref> ; <xref rid="B46" ref-type="bibr"> Kunkel and Brooks, 2002 </xref> ; <xref rid="B84" ref-type="bibr"> Spoel et al., 2003 </xref> ; <xref rid="B72" ref-type="bibr"> Robert-Seilaniantz et al., 2011 </xref> ). <italic> WRKY </italic> genes are rapidly induced by pathogens, pathogen elicitors, or SA treatment in a number of plant species ( <xref rid="B27" ref-type="bibr"> Eulgem et al., 1999 </xref> ; <xref rid="B11" ref-type="bibr"> Chen and Chen, 2000 </xref> ). Arabidopsis WRKY70 is a common regulatory component of SA- and JA- dependent defense responses and is also required for <italic> R </italic> gene-mediated <span class="trait">resistance</span> ( <xref rid="B50" ref-type="bibr"> Li et al., 2004 </xref> , <xref rid="B49" ref-type="bibr"> 2006 </xref> ; <xref rid="B45" ref-type="bibr"> Knoth et al., 2007 </xref> ). In rice, OsWRKY13 functions as an activator of the SA-dependent defense response and a suppressor of the JA-dependent response, which mediates disease <span class="trait">resistance</span> to bacterial blight and fungal blast ( <xref rid="B68" ref-type="bibr"> Qiu et al., 2007 </xref> ). Here, we demonstrated that mRNA transcripts of <italic> <span class="gene">GmWRKY31</span> </italic> are remarkably increased by <italic> P. sojae </italic> , suggesting an important role of <italic> <span class="gene">GmWRKY31</span> </italic> in soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> . To understand the molecular basis of <span class="gene">GmWRKY31</span> in response to <italic> P. sojae </italic> , we further demonstrated that overexpression of <italic> <span class="gene">GmWRKY31</span> </italic> significantly increases <span class="trait">resistance</span> to <italic> P. sojae </italic> in transgenic soybean plants, whereas <span class="trait">resistance</span> to <italic> P. sojae </italic> was compromised in <italic> <span class="gene">GmWRKY31</span> </italic> RNA interference transgenic soybean plants. These results indicate that <span class="gene">GmWRKY31</span> is required for soybean defense responses to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						WRKY TFs are mainly involved in the response to biotic and abiotic stresses by binding to the W-box present in the promoter of defense-related genes ( <xref rid="B60" ref-type="bibr"> Maleck et al., 2000 </xref> ; <xref rid="B94" ref-type="bibr"> Wang H.H. et al., 2015 </xref> ). OsWRKY4 specifically binds the promoter regions of <italic> OsPR1b </italic> and <italic> OsPR5 </italic> , which contain W-box or W-box-like <italic> cis </italic> -elements that mediate <span class="trait">resistance</span> to rice sheath blight fungus ( <xref rid="B94" ref-type="bibr"> Wang H.H. et al., 2015 </xref> ). In this study, we demonstrated that <span class="gene">GmWRKY31</span> is localized in the nucleus. We also demonstrated that <span class="gene">GmWRKY31</span> activates basal transcription levels of a reporter gene in Arabidopsis cells. These findings suggest that <span class="gene">GmWRKY31</span> acts as a W-box-mediated transcriptional activator. <italic> WRKY </italic> genes act upstream of <italic> NPR1 </italic> and positively regulate its expression during the activation of plant defense responses ( <xref rid="B101" ref-type="bibr"> Yu et al., 2001 </xref> ). <xref rid="B55" ref-type="bibr"> Liu et al. (2005) </xref> demonstrated that <italic> OsNPR1 </italic> expression is constitutively induced in OsWRKY03 transgenic plants but not in control plants, suggesting that OsWRKY03 functions upstream of <italic> OsNPR1 </italic> in the defense signal pathway. Accordingly, <xref rid="B17" ref-type="bibr"> Choi et al. (2015) </xref> determined that OsWRKY6 positively regulates <italic> OsNPR1 </italic> expression in transient expression assays but did not detect any amplified PCR bands by ChIP-PCR assay in <italic> OsWRKY6 </italic> -ox lines, suggesting that OsWRKY6 does not directly bind to the <italic> OsNPR1 </italic> promoter. We previously reported differential abundance of <italic> <span class="gene">GmNPR1</span> </italic> in response to the oomycete <italic> P. sojae </italic> by SSH coupled with cDNA microarrays ( <xref rid="B97" ref-type="bibr"> Xu et al., 2012 </xref> ). In this work, <italic> <span class="gene">GmWRKY31</span> </italic> overexpression activated <italic> <span class="gene">GmNPR1</span> </italic> gene expression, and <italic> <span class="gene">GmWRKY31</span> </italic> RNA interference transgenic soybean plants reduced <italic> <span class="gene">GmNPR1</span> </italic> gene expression. In addition, we also demonstrated that <span class="gene">GmWRKY31</span> directly binds to the W-box within the -766 to -638 region of the <italic> <span class="gene">GmNPR1</span> </italic> promoter using EMSA and yeast-one-hybrid assays. Taken together, these results suggest that <span class="gene">GmWRKY31</span> promotes <italic> <span class="gene">GmNPR1</span> </italic> expression by directly binding to the W-box within the -766 to -638 region of the <italic> <span class="gene">GmNPR1</span> </italic> promoter <italic> in vivo </italic> and <italic> in vitro </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						WRKY TFs perform their diverse functions in various stress signaling pathways through physical interactions with different proteins, such as the VQ protein, MAP kinases, His deacetylases, and calmodulin ( <xref rid="B66" ref-type="bibr"> Park et al., 2005 </xref> ; <xref rid="B44" ref-type="bibr"> Kim et al., 2008 </xref> ; <xref rid="B69" ref-type="bibr"> Qiu et al., 2008 </xref> ; <xref rid="B36" ref-type="bibr"> Ishihama et al., 2011 </xref> ). In Arabidopsis, AtWRKY33, which interacts with multiple VQ proteins, is critical not only for <span class="trait">resistance</span> to necrotrophic pathogens but also for tolerance to important abiotic stresses ( <xref rid="B108" ref-type="bibr"> Zheng et al., 2006 </xref> ; <xref rid="B39" ref-type="bibr"> Jiang and Deyholos, 2009 </xref> ; <xref rid="B51" ref-type="bibr"> Li et al., 2011 </xref> ). In rice, OsWRKY30 interacts with several rice MAPKs and is phosphorylated by some interacting MAPKs ( <xref rid="B80" ref-type="bibr"> Shen et al., 2012 </xref> ). More recent research indicates that <span class="gene">GmWRKY27</span> interacts with <span class="gene">GmMYB174,</span> which cooperatively suppresses <span class="gene">GmNAC29</span> expression and enhances <span class="trait">drought</span> stress tolerance in soybean plants ( <xref rid="B93" ref-type="bibr"> Wang F. et al., 2015 </xref> ). The present data demonstrate that <span class="gene">GmWRKY31</span> interacts with <span class="gene">GmHDL56</span> <italic> in vivo </italic> and <italic> in vitro </italic> and responds to <italic> P. sojae </italic> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						NPR1 was first identified by screening for mutants blocked in SA-mediated <italic> PR </italic> gene expression and <span class="trait">resistance</span> ( <xref rid="B9" ref-type="bibr"> Cao et al., 1994 </xref> ; <xref rid="B19" ref-type="bibr"> Delaney et al., 1995 </xref> ; <xref rid="B92" ref-type="bibr"> Wang et al., 2006 </xref> ). <italic> NPR1 </italic> is constitutively expressed in plants, and its expression level is elevated upon SA treatment or pathogen infection ( <xref rid="B61" ref-type="bibr"> Malnoy et al., 2007 </xref> ; <xref rid="B81" ref-type="bibr"> Shi et al., 2010 </xref> ; <xref rid="B104" ref-type="bibr"> Zhang et al., 2012 </xref> ). In addition, the overexpression of the <italic> AtNPR1 </italic> gene or its orthologs enhances <span class="trait">resistance</span> to biotrophic and necrotrophic fungal, viral and bacterial pathogens in a number of plant species, including rice, wheat, tomato and tobacco ( <xref rid="B53" ref-type="bibr"> Lin et al., 2004 </xref> ; <xref rid="B16" ref-type="bibr"> Chern et al., 2005 </xref> ; <xref rid="B59" ref-type="bibr"> Makandar et al., 2006 </xref> ; <xref rid="B102" ref-type="bibr"> Yuan et al., 2007 </xref> ; <xref rid="B63" ref-type="bibr"> Meur et al., 2008 </xref> ). In the present study, the expression of <italic> <span class="gene">GmNPR1</span> </italic> was significantly induced by <italic> P. sojae </italic> . Further transgenic analysis revealed that the overexpression of <italic> <span class="gene">GmNPR1</span> </italic> in soybean improved <span class="trait">resistance</span> to <italic> P. sojae </italic> and enhanced susceptibility to <italic> P. sojae </italic> infection in <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines, and the accumulation of SA in the <italic> <span class="gene">GmNPR1</span> </italic> -OE lines was significantly higher than that of WT and <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines. In Arabidopsis, the TF NPR1 is essential for the induction of defense genes such as PR genes ( <xref rid="B109" ref-type="bibr"> Zhou et al., 2000 </xref> ; <xref rid="B29" ref-type="bibr"> <span class="gene">Fan</span> and Dong, 2002 </xref> ). Here, we observed that <italic> <span class="gene">GmPR1a</span> </italic> , <italic> <span class="gene">GmPR2</span> </italic> , <italic> <span class="gene">GmPR3</span> </italic> , <italic> <span class="gene">GmPR4</span> </italic> , <italic> <span class="gene">GmPR5a</span> </italic> , and <italic> <span class="gene">GmPR10</span> </italic> were highly induced in <italic> <span class="gene">GmNPR1</span> </italic> -OE lines and that <italic> <span class="gene">GmPR1a</span> </italic> , <italic> <span class="gene">GmPR3</span> </italic> , <italic> <span class="gene">GmPR4</span> </italic> , <italic> <span class="gene">GmPR5a</span> </italic> and <italic> <span class="gene">GmPR10</span> </italic> were significantly repressed in <italic> <span class="gene">GmNPR1</span> </italic> -RNAi lines. Taken together, these results indicated that overexpression of <italic> <span class="gene">GmNPR1</span> </italic> in soybean leads to higher SA level. Previous studies with SA in other plants showed that SA-induced defense responses are mediated by NPR1 ( <xref rid="B10" ref-type="bibr"> Cao et al., 1997 </xref> ; <xref rid="B84" ref-type="bibr"> Spoel et al., 2003 </xref> ) and the accumulation of SA leads to up-regulation of defense-related genes including the PR genes <italic> PR1 </italic> , <italic> PR2 </italic> and <italic> PR5 </italic> , and results in enhanced disease <span class="trait">resistance</span> against biotrophic pathogens ( <xref rid="B31" ref-type="bibr"> Gaffney et al., 1993 </xref> ; <xref rid="B20" ref-type="bibr"> Delaney et al., 1994 </xref> ). So, we speculate a model that <span class="gene">GmWRKY31</span> and <span class="gene">GmHDL56</span> interact with each other and directly activate the expression of <italic> <span class="gene">GmNPR1</span> </italic> , then <span class="gene">GmNPR1</span> positively regulates the content of the SA, and accumulation of SA leads to up-regulation of <italic> <span class="gene">GmPR1a</span> </italic> , <italic> <span class="gene">GmPR2</span> </italic> , <italic> <span class="gene">GmPR3</span> </italic> , <italic> <span class="gene">GmPR4</span> </italic> , <italic> <span class="gene">GmPR5a</span> </italic> , and <italic> <span class="gene">GmPR10</span> </italic> genes, which ultimately leads to enhanced <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean ( <bold> Figure <xref ref-type="fig" rid="F8"> 8 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Proposed model of <span class="gene">GmWRKY31</span> function in response to <italic> Phytophthora sojae </italic> in soybean. </bold> Expression of the <italic> <span class="gene">GmWRKY31</span> </italic> gene is induced by <italic> P. sojae </italic> . <span class="gene">GmWRKY31</span> interacts with <span class="gene">GmHDL56,</span> and cooperatively activate the expression of <italic> <span class="gene">GmNPR1</span> </italic> by binding to the core sequences in its promoter. <italic> <span class="gene">GmNPR1</span> </italic> positively regulates the expression of the <italic> <span class="gene">GmPR1a</span> </italic> (AF136636), <italic> <span class="gene">GmPR2</span> </italic> (M37753), <italic> <span class="gene">GmPR3</span> </italic> (AF202731), <italic> <span class="gene">GmPR4</span> </italic> (BT090788), <italic> <span class="gene">GmPR5a</span> </italic> (M21297), <italic> <span class="gene">GmPR10</span> </italic> (FJ960440) genes, finally leading to enhance <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, the <italic> <span class="gene">GmNPR1</span> </italic> -OE lines show a 15&#x02013;20 fold increase in transcripts for <italic> <span class="gene">GmNPR1</span> </italic> , but the <span class="gene">GmWRKY31</span> or <span class="gene">GmHDL56</span> transactivation assays show a rather weak increase in <italic> <span class="gene">GmNPR1</span> </italic> (&#x0003c;2-fold). In previous studies, there were reports that a few of genes were induced by <italic> P. sojae </italic> , such as N-rich protein ( <xref rid="B58" ref-type="bibr"> Ludwig and Tenhaken, 2001 </xref> ), <italic> <span class="gene">GmPR10</span> </italic> ( <xref rid="B98" ref-type="bibr"> Xu et al., 2014 </xref> ), <italic> <span class="gene">GmPRP</span> </italic> ( <xref rid="B38" ref-type="bibr"> Jiang et al., 2015 </xref> ), and <italic> Gly m 4l </italic> ( <xref rid="B28" ref-type="bibr"> <span class="gene">Fan</span> et al., 2015 </xref> ), then we speculate that there maybe <italic> P. sojae </italic> induced genes changed in their expression when the expression of <span class="gene">GmWRKY31</span> or <span class="gene">GmHDL56</span> is modified, which might affect the expression of <italic> <span class="gene">GmNPR1</span> </italic> . So, in the follow-up work, it is valuable to make a deeper research on other <italic> P. sojae </italic> induced genes, defensive barriers and phytoalexins in transgenic soybean plants, and it would improve the interpretation of the findings that the mechanism of the <span class="gene">GmWRKY31</span> and <span class="gene">GmHDL56</span> enhances <span class="trait">resistance.</span> This study provides new insights into the mechanism by which WRKY proteins regulate biotic stress responses in soybean and presents <span class="gene">GmWRKY31</span> as an ideal candidate target to improve <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean and other crops.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmERF5,</span> a new member of the soybean EAR motif-containing ERF transcription factor, enhances <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:25779701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dong L(1), Cheng Y(1), Wu J(2), Cheng Q(1), Li W(1), Fan S(1), Jiang L(1), Xu Z(3), Kong F(4), Zhang D(3), Xu P(5), Zhang S(5).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean [Glycine max (L.) Merr.], caused by Phytophthora sojae Kaufmann and Gerdemann, is a destructive disease throughout the soybean planting regions in the world. Here, we report insights into the function and underlying mechanisms of a novel ethylene response factor (ERF) in soybean, namely <span class="gene">GmERF5,</span> in host responses to P. sojae. <span class="gene">GmERF5-overexpressing</span> transgenic soybean exhibited significantly enhanced <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the PR10, PR1-1, and PR10-1 genes. Sequence analysis suggested that <span class="gene">GmERF5</span> contains an AP2/ERF domain of 58 aa and a conserved ERF-associated amphiphilic repression (EAR) motif in its C-terminal region. Following stress treatments, <span class="gene">GmERF5</span> was significantly induced by P. sojae, ethylene (ET), abscisic acid (ABA), and salicylic acid (SA). The activity of the <span class="gene">GmERF5</span> promoter <span class="gene">(GmERF5P)</span> was upregulated in tobacco leaves with ET, ABA, Phytophthora nicotianae, salt, and <span class="trait">drought</span> treatments, suggesting that <span class="gene">GmERF5</span> could be involved not only in the induced defence response but also in the ABA-mediated pathway of salt and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> could bind to the GCC-box element and act as a repressor of gene transcription. It was targeted to the nucleus when transiently expressed in Arabidopsis protoplasts. <span class="gene">GmERF5</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> and eukaryotic translation initiation factor <span class="gene">(GmEIF)</span> both in yeast cells and in planta. To the best of our knowledge, <span class="gene">GmERF5</span> is the first soybean EAR motif-containing ERF transcription factor demonstrated to be involved in the response to pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Phytophthora sojae <span class="gene">Avr1d</span> gene encodes an RxLR-dEER effector with presence and absence polymorphisms among pathogen strains.</h2>
			<h3>PMID:23594349  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin W(1), Dong S, Zhai L, Lin Y, Zheng X, Wang Y.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">root</span> and stem <span class="trait">rot</span> is caused by the oomycete pathogen Phytophthora sojae. The interaction between P. sojae and soybean fits the "gene-for-gene" hypothesis. Although more than 10 P. sojae avirulence (Avr) effectors have been genetically identified, nearly half of genetically defined avr genes have been cloned. In a previous bioinformatic and global transcriptional analysis, we identified a P. sojae RxLR effector, Avr1d, which was 125 amino acids in length. Mapping data demonstrated that <span class="gene">Avr1d</span> presence or absence in the genome was co-segregated with the <span class="gene">Avr1d</span> avirulence phenotype in F2 populations. Transient expression of the <span class="gene">Avr1d</span> gene using co-bombardment in soybean isogenic lines revealed that this gene triggered a hypersensitive response (HR) in the presence of Rps1d. Sequencing of <span class="gene">Avr1d</span> genes in different P. sojae strains revealed two <span class="gene">Avr1d</span> alleles. Although polymorphic, the two <span class="gene">Avr1d</span> alleles could trigger Rps1d-mediated HR. P. sojae strains carrying either of the alleles were avirulent on Rps1d soybean lines. <span class="gene">Avr1d</span> was upregulated during the germinating cyst and early infection stages. Furthermore, transient expression of <span class="gene">Avr1d</span> in Nicotiana benthamiana suppressed BAX-induced cell death and enhanced P. capsici infection. <span class="gene">Avr1d</span> also suppressed effector-triggered immunity induction by associating with <span class="gene">Avr1b</span> and Rps1b, suggestive of a role in suppressing plant immunity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two RxLR avirulence genes in Phytophthora sojae determine soybean Rps1k-mediated disease <span class="trait">resistance.</span></h2>
			<h3>PMID:23530601  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song T(1), Kale SD, Arredondo FD, Shen D, Su L, Liu L, Wu Y, Wang Y, Dou D, Tyler BM.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> to Phytophthora sojae (Rps) genes have been widely used in soybean against <span class="trait">root</span> and stem <span class="trait">rot</span> diseases caused by this oomycete. Among 15 known soybean Rps genes, Rps1k has been the most widely used in the past four decades. Here, we show that the products of two distinct but closely linked RxLR effector genes are detected by Rps1k-containing plants, resulting in disease <span class="trait">resistance.</span> One of the genes is Avr1b-1, that confers avirulence in the presence of Rps1b. Three lines of evidence, including overexpression and gene silencing of <span class="gene">Avr1b-1</span> in stable P. sojae transformants, as well as transient expression of this gene in soybean, indicated that <span class="gene">Avr1b</span> could trigger an Rps1k-mediated defense response. Some isolates of P. sojae that do not express <span class="gene">Avr1b</span> are nevertheless unable to infect Rps1k plants. In those isolates, we identified a second RxLR effector gene (designated Avr1k), located 5 kb away from Avr1b-1. Silencing or overexpression of Avr1k in P. sojae stable transformants resulted in the loss or gain, respectively, of the avirulence phenotype in the presence of Rps1k. Only isolates of P. sojae with mutant alleles of both <span class="gene">Avr1b-1</span> and Avr1k could evade perception by the soybean plants carrying Rps1k.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inheritance and mapping of 11 avirulence genes in Phytophthora sojae.</h2>
			<h3>PMID:12223184  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：May KJ(1), Whisson SC, Zwart RS, Searle IR, Irwin JA, Maclean DJ, Carroll BJ, Drenth A.
				<br/> 
				年份：2002 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two new crosses involving four races (races 7, 16, 17, and 25) of the soybean <span class="trait">root</span> and stem <span class="trait">rot</span> pathogen Phytophthora sojae were established (7/16 cross; 17/25 cross). An F2 population derived from each cross was used to determine the genetic basis of avirulence towards 11 different <span class="trait">resistance</span> genes in soybean. Avirulence was found to be dominant and determined by a single locus for Avr1b, 1d, 1k, 3b, 4, and 6, as expected for a simple gene-for-gene model. We also observed several cases of segregation, inconsistent with a single dominant gene being solely responsible for avirulence, which suggests that the genetic background of the different crosses can affect avirulence. <span class="gene">Avr4</span> and 6 cosegregated in both the 7/16 and 17/25 crosses and, in the 7/16 cross, <span class="gene">Avr1b</span> and 1k were closely linked. Information from segregating RAPD, RFLP, and AFLP markers screened on F2 progeny from the two new crosses and two crosses described previously (a total of 212 F2 individuals, 53 from each cross) were used to construct an integrated genetic linkage map of P. sojae. This revised genetic linkage map consists of 386 markers comprising 35 RFLP, 236 RAPD, and 105 AFLP markers, as well as 10 avirulence genes. The map is composed of 21 major linkage groups and seven minor linkage groups covering a total map distance of 1640.4cM.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Functional Characterization of the Soybean <italic> <span class="gene">GmaPPO12</span> </italic> Promoter Conferring <italic> Phytophthora sojae </italic> Induced Expression</h2>
			<h3>PMID:23840763</h3>
				
					<h3>PMCID:3695865</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Chunyue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Yanling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Danyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Yuren </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hongjuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dou </surname>  <given-names> Daolong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 5 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Identification of pathogen-inducible promoters largely lags behind cloning of the genes for disease <span class="trait">resistance.</span> Here, we cloned the soybean <italic> <span class="gene">GmaPPO12</span> </italic> gene and found that it was rapidly and strongly induced by <named-content content-type="genus-species"> <named-content content-type="genus"> Phytophthora </named-content> <italic> <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection. Computational analysis revealed that its promoter contained many known <italic> cis </italic> -elements, including several defense related transcriptional factor-binding boxes. We showed that the promoter could mediate induction of GUS expression upon infection in both transient expression assays in <named-content content-type="genus-species"> <named-content content-type="genus"> Nicotiana </named-content> <italic> <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> and stable transgenic soybean hairy roots. Importantly, we demonstrated that pathogen-induced expression of the <italic> <span class="gene">GmaPPO12</span> </italic> promoter was higher than that of the soybean <italic> <span class="gene">GmaPR1a</span> </italic> promoter. A progressive 5&#x02019; and 3&#x02019; deletion analysis revealed two fragments that were essential for promoter activity. Thus, the cloned promoter could be used in transgenic plants to enhance <span class="trait">resistance</span> to <italic> phytophthora </italic> pathogens, and the identified fragment could serve as a candidate to produce synthetic pathogen-induced promoters.
					</p>
				
					<p>&nbsp;&nbsp;
						We are unaware of pathogen-inducible promoters have been identified and characterized in soybeans. Here, we screened the published microarray data [ <xref ref-type="bibr" rid="B25"> 25 </xref> ] and found that the <italic> <span class="gene">GmaPPO</span> </italic> 12 gene is strongly induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> . We elucidated its expression pattern by quantitative RT-PCR analysis in soybeans, transient expression in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> , and stable expression in soybean hairy roots. We found that the gene had three expressional characterizations. First, it is an immediate-early pathogen response gene because it can be induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection at 0.5 h in soybean, and its promoter also can mediate fast induction activity in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> . Second, its induction is strong. The expression levels upon infection are higher than that of the <italic> <span class="gene">GmaPR1a</span> </italic> gene in soybean. Third, the expression is specific to infection. It cannot be induced by the tested hormones in soybean, and its background expression is almost undetectable in soybean and <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> . This promoter could be used in transgenic plants to enhance their <span class="trait">resistance</span> to <italic> Phytophthora </italic> and perhaps other oomycete diseases. Finally, we identified a small responsive region by progressive 5&#x02019; and 3&#x02019; deletions. The small functional segment could serve as a candidate to further identify novel pathogen responsive <italic> cis </italic> -elements or to produce synthetic pathogen-inducible promoters.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene is predicted to encode a member of the polyphenol oxidase (PPO) family that catalyzes the oxidation of phenolics to quinones [ <xref ref-type="bibr" rid="B26"> 26 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> ]. In green plants, the <italic> PPO </italic> gene family is highly variable in both gene number and structure, except in the genus <named-content content-type="genus-species"> Arabidopsis </named-content> [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. The enzymes play many divergent roles in a variety of processes, including defense processes [ <xref ref-type="bibr" rid="B46"> 46 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]. In tomatoes, for example, PPO overexpression increased <span class="trait">resistance</span> to <italic> <named-content content-type="genus-species"> Pseudomonas syringae </named-content> </italic> and several insects [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. PPOs are also implicated in latex coagulation [ <xref ref-type="bibr" rid="B48"> 48 </xref> ], and the biosynthesis of aurones and lignans [ <xref ref-type="bibr" rid="B49"> 49 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Soybean contains 11 PPO isoforms [ <xref ref-type="bibr" rid="B26"> 26 </xref> ], which correlate with its abundant flavonoids. Here, we showed that <italic> <span class="gene">GmaPPO12</span> </italic> was rapidly and strongly induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection. However, the role of <italic> <span class="gene">GmaPPO12</span> </italic> in disease <span class="trait">resistance</span> still needs to be addressed.
					</p>
				
					<p>&nbsp;&nbsp;
						Patterns of gene expression in soybean upon <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection have been investigated in several independent studies [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. Only 22 soybean genes were identified as upregulated early in infection (3 hrs). The genes included <italic> <span class="gene">GmaPR1a</span> </italic> and a gene encoding flavin adenine dinucleotide (FAD)-linked oxidoreductase [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Here, we identified <italic> <span class="gene">GmaPPO12</span> </italic> as being strongly inducing at 0.5 h postinfection. Its fold change was higher than that of <italic> <span class="gene">GmaPR1a</span> </italic> . Characterization of the <italic> <span class="gene">GmaPR1a</span> </italic> promoter in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves and soybean hairy roots also supported the idea that the gene was rapidly and highly induced by <italic> Phytophthora </italic> pathogens. Moreover, the fold change in <italic> <span class="gene">GmaPPO12</span> </italic> promoter driven GUS expression was lower than that of the endogenous gene in soybean. Quantitative RT-PCR analysis showed that the gene was upregulated 53-fold at 0.5 hpi in soybean, but the <italic> GUS </italic> gene was only induced 6.1-fold in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> and 4.8-fold in soybean hairy roots, respectively. This difference may be due to that the stress-induced expression of the endogenous gene driven by the native promoter not being similar to the heterologous <italic> GUS </italic> gene fused with the same promoter, as observed in previous studies [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. We have noted that the promoter driving GUS expression exhibited unwanted background expression, although the levels were low. One possible reason is that the agroinfiltration associated wounding induced gene expression. Further studies will be required to test the utilization of the cloned promoter to drive functional disease <span class="trait">resistance</span> genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Genomic DNA was extracted from 6&#x02013;7-day-old soybean roots, following the cetyltrimethylammonium bromide (CTAB) isolation procedure [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. The <italic> <span class="gene">GmaPPO12</span> </italic> and <italic> <span class="gene">GmaPR1a</span> </italic> promoters were isolated with the designed primers (Table. S1). The obtained promoters cover a sequence 1,500-bp in length upstream of the transcription start site. Five 5&#x02019; deletions, five 3&#x02019; deletions, and two 5&#x02019; and 3&#x02019; deletions of the <italic> <span class="gene">GmaPPO12</span> </italic> promoter region were generated by PCR, with forward primers containing a <italic> Kpn </italic> I restriction site and a reverse primer containing a <italic> Bgl </italic> II restriction site. The sequences of promoters used for the amplification of the deletion promoter fragments are summarized in <xref ref-type="supplementary-material" rid="pone.0067670.s004"> Figure S3 </xref> . All of the promoter fragments were digested with <italic> Kpn </italic> I and <italic> Bgl </italic> II and cloned into the <italic> Kpn </italic> I- <italic> Bgl </italic> II sites of pMDC162 [ <xref ref-type="bibr" rid="B59"> 59 </xref> ], a plant expression vector fused with the CaMV 35S minimal promoter [ <xref ref-type="bibr" rid="B60"> 60 </xref> ] and <italic> GUS </italic> reporter gene. A full description of plasmid construction can be found in <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3A </xref> . The pMDC162 vector containing the CaMV 35S minimal promoter lacking <italic> cis </italic> -acting elements was constructed and used as a negative control. PBI121 contains the <italic> GUS </italic> (or <italic> uidA </italic> ) gene under the control of the CaMV 35S promoter as the positive control. Both plasmids contain the selectable marker gene <italic> nptII </italic> , which confers <span class="trait">resistance</span> to kanamycin. All primers used in this study are listed in Supplemental Table. 1. All plasmids containing expression cassettes were confirmed by sequence analysis and introduced into <named-content content-type="genus-species"> Agrobacterium </named-content> strains by electroporation for transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						The introduction of expression constructs into soybean hairy roots using the <named-content content-type="genus-species"> <named-content content-type="genus"> Agrobacterium </named-content> <italic> <named-content content-type="species"> rhizogene </named-content> </italic> </named-content> K599 cucumopine strain [ <xref ref-type="bibr" rid="B63"> 63 </xref> ] was by electroporation as described by Ainsworth et al [ <xref ref-type="bibr" rid="B61"> 61 </xref> ]. Individual colonies were picked and verified by enzyme restriction analysis of DNA extracted from randomly selected clones. Plant inoculation was conducted according to Savka [ <xref ref-type="bibr" rid="B63"> 63 </xref> ] and modified as follows: soybean seeds were surface-sterilized with chlorine gas and germinated on Murashige and Skoog (MS) culture medium for 5-6 days in a 16: 8-h light/dark cycle incubator at 25&#x000b0;C. Cotyledons were wounded on the abaxial side with scalpel blades dipped in a culture of <named-content content-type="genus-species"> <named-content content-type="genus"> Agrobacterium </named-content> <italic> <named-content content-type="species"> rhizogenes </named-content> </italic> </named-content> K599 harboring the different constructs and cultured abaxial-side-up on <span class="gene">MS</span> culture medium. Carbenicillin and Cefotetan 250 &#x000b5;g/mL were added to inhibit the growth of <named-content content-type="genus-species"> <named-content content-type="genus"> A </named-content> <italic> . <named-content content-type="species"> rhizogenes </named-content> </italic> </named-content> and incubated at 25&#x000b0;C in the dark. The putative transformed hairy roots about 18-20 days after <span class="trait">root</span> emergence and at lengths of 4- to 6-cm were used for inoculations.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean ERF Transcription Factor, <italic> <span class="gene">GmERF113</span> </italic> , Increases <span class="trait">Resistance</span> to <italic> Phytophthora sojae </italic> Infection in Soybean</h2>
			<h3>PMID:28326092</h3>
				

				
					<h3>PMCID:5339286</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yuanling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Xin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qi </surname>  <given-names> Dongyue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Guangjin </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/418977/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257714/overview"/>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean caused by the oomycete Phytophthora sojae, is a destructive disease worldwide. Ethylene response factors (ERFs) play important roles in regulating plant biotic and abiotic stress tolerance. In this study, a new ERF gene, <span class="gene">GmERF113,</span> was isolated from the highly resistant soybean 'Suinong 10.' Sequence analysis suggested that the protein encoded by <span class="gene">GmERF113</span> contained a conserved AP2/ERF domain of 58 amino acid and belonged to the B-4 subgroup of the ERF subfamily. Expression of <span class="gene">GmERF113</span> was significantly induced by P. sojae, ethylene, and methyl jasmonate. <span class="gene">GmERF113</span> protein localized to the nucleus when transiently expressed in Arabidopsis protoplasts, could bind to the GCC-box, and acted as a transcription activator. In addition, a region of the full-length <span class="gene">GmERF113,</span> <span class="gene">GmERF113-II,</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> in yeast cells. Full-length <span class="gene">GmERF113</span> also interacted with <span class="gene">GmbHLH</span> in planta. <span class="gene">GmERF113-overexpressing</span> transgenic plants in susceptible cultivar 'Dongnong 50' soybean exhibited increased <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the pathogenesis-related genes, PR1 and PR10-1. These results indicate that <span class="gene">GmERF113</span> may play a crucial role in the defense of soybean against P. sojae infection.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Some ERF subfamily members also respond to phytohormones and abiotic stress in plants ( <xref rid="B54" ref-type="bibr"> Shinozaki et al., 2003 </xref> ; <xref rid="B46" ref-type="bibr"> Pieterse et al., 2009 </xref> ; <xref rid="B58" ref-type="bibr"> Sugano et al., 2013 </xref> ). Transcription of the <italic> TaERF1 </italic> gene can be induced by exogenous abscisic acid (ABA), ethylene (ET), and salicylic acid (SA). Furthermore, overexpression of <italic> TaERF1 </italic> activates stress-related genes, improving abiotic stress tolerance in transgenic plants ( <xref rid="B69" ref-type="bibr"> Xu et al., 2007 </xref> ). Expression of <italic> <span class="gene">GmERF7</span> </italic> is induced by treatment with methyl jasmonate (MeJA), ET, and ABA, and its overexpression enhances salt tolerance in transgenic tobacco plants ( <xref rid="B74" ref-type="bibr"> Zhai et al., 2013b </xref> ). Moreover, overexpression of <italic> AhERF019 </italic> , a peanut ERF gene, enhances tolerance to <span class="trait">drought,</span> heat, and salt stress in transgenic <italic> Arabidopsis </italic> ( <xref rid="B62" ref-type="bibr"> Wan et al., 2014 </xref> ), and mutation of ERF-associated amphiphilic repression (EAR) motif in <italic> OsERF3 </italic> represses ET biosynthesis and <span class="trait">drought</span> tolerance in rice ( <xref rid="B79" ref-type="bibr"> Zhang et al., 2013 </xref> ). Thus, ERF proteins have important roles, not only in pathogen defense responses, but also in tolerance to various abiotic stress factors.
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous study, we constructed a cDNA library from soybean &#x02018;Suinong 10&#x02019; (which has high <span class="trait">resistance</span> to <italic> P. sojae </italic> ) by suppression subtractive hybridization (SSH). The library was enriched for mRNAs encoding expressed sequence tags (ESTs) which increased in abundance during infection with <italic> P. sojae </italic> . In this study, an EST homologous to the AP2/ERF transcription factor, <italic> <span class="gene">GmERF113</span> </italic> (GenBank accession no. XM_003548806, NCBI protein no. XP_003548854), was isolated and its characteristics and expression patterns in response to different stress conditions analyzed, demonstrating that <italic> <span class="gene">GmERF113</span> </italic> has a role in the defense response of soybean to <italic> P. sojae. </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Fully expanded leaves of T <sub> 3 </sub> transgenic soybean plants, derived from T <sub> 2 </sub> plants identified by PCR and Southern blot hybridization, were tested by qPCR using the primer pair, <italic> <span class="gene">GmERF113-qF</span> </italic> and <italic> <span class="gene">GmERF113-qR</span> </italic> (Supplementary Table <xref ref-type="supplementary-material" rid="SM1"> 1 </xref> ) and screened for <span class="trait">resistance</span> to <italic> P. sojae </italic> as described by <xref rid="B30" ref-type="bibr"> Kim et al. (2014) </xref> , with some modifications. Live leaves inoculated with <italic> P. sojae </italic> were covered with polythene bags to maintain relative humidity levels; culture conditions were 25&#x000b0;C, 90% &#x000b1; 10% relative humidity, 16 h photoperiod, and 350 &#x003bc;mol m <sup> -2 </sup> s <sup> -1 </sup> light intensity. Soybean &#x02018;Dongnong 50&#x02019; plants were used as controls. After 2 and 4 days, disease symptoms on each leaf were observed and photographed using a Canon IXUS 860IS camera.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to confirm which proteins interact with <span class="gene">GmERF113-II,</span> four fusion genes from among the seventeen candidate genes ( <italic> <span class="gene">GmbHLH</span> </italic> , <italic> <span class="gene">GmPRP</span> </italic> , <italic> <span class="gene">GmHAT5</span> </italic> , and <italic> <span class="gene">GmFAO</span> </italic> ), with predicted functions related to pathogenesis and disease <span class="trait">resistance,</span> were selected for further investigation. Full-length cDNAs of these four genes were cloned and constructed in pGADT7. Analysis of whether these proteins had transcription activation and interaction with <span class="gene">GmERF113</span> in yeast cells was performed. Our results showed that the four proteins could not activate transcription in yeast cells, and only <span class="gene">GmbHLH</span> interacted with <span class="gene">GmERF113-II</span> ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ), while the other three candidate proteins could not interact with <span class="gene">GmERF113-II</span> in yeast.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine whether overexpression of <italic> <span class="gene">GmERF113</span> </italic> can improve <span class="trait">resistance</span> to <italic> P. sojae </italic> in transgenic soybean plants, T <sub> 1 </sub> and T <sub> 2 </sub> transgenic plants, screened by PCR amplification and Southern hybridization, were used to generate T <sub> 3 </sub> plants, constituting three independent <italic> <span class="gene">GmERF113</span> </italic> -overexpressing transgenic lines. The overexpression of <italic> <span class="gene">GmERF113</span> </italic> in the T <sub> 3 </sub> transgenic soybean lines (G1, G2, and G3) was detected by qRT-PCR to investigate the response of the plants to <italic> P. sojae </italic> ( <bold> Figure <xref ref-type="fig" rid="F8"> 8A </xref> </bold> ). Four days after inoculation with <italic> P. sojae </italic> , the leaves of non-transgenic soybean plants showed clear, large lesions compared with those of transgenic soybean plants ( <bold> Figure <xref ref-type="fig" rid="F8"> 8B </xref> </bold> ); the lesion areas in transgenic soybean lines were significantly smaller than those in non-transgenic plants ( <italic> P </italic> &#x0003c; 0.01) ( <bold> Figure <xref ref-type="fig" rid="F8"> 8C </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, T <sub> 4 </sub> transgenic soybean plants (G1-1, G2-1, and G3-1) were identified by qRT-PCR ( <bold> Figure <xref ref-type="fig" rid="F8"> 8D </xref> </bold> ) and the relative biomass of <italic> P. sojae </italic> in infected live cotyledons after 48 h incubation with <italic> P. sojae </italic> zoospore suspensions was tested. As shown in <bold> Figure <xref ref-type="fig" rid="F8"> 8E </xref> </bold> , after 48 h of incubation with <italic> P. sojae </italic> , the cotyledons of transgenic soybean plants exhibited smaller lesions compared with those of non-transgenic plants. The biomass of <italic> P. sojae </italic> , based on the transcript levels of the <italic> P. sojae TEF1 </italic> gene, was significantly lower in transgenic <italic> <span class="gene">GmERF113</span> </italic> -overexpressing plants than that in non-transgenics ( <italic> P </italic> &#x0003c; 0.01) ( <bold> Figure <xref ref-type="fig" rid="F8"> 8F </xref> </bold> ). These findings demonstrated that overexpression of <italic> <span class="gene">GmERF113</span> </italic> in soybean plants increased their <span class="trait">resistance</span> to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, <italic> <span class="gene">GmERF113</span> </italic> , a new member of the ERF subfamily identified in soybean, was demonstrated to increase soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> . ERF transcription factors have been identified in numerous plant species, including <italic> Arabidopsis thaliana </italic> ( <xref rid="B35" ref-type="bibr"> Liu et al., 1998 </xref> ; <xref rid="B42" ref-type="bibr"> Nakano et al., 2006 </xref> ; <xref rid="B56" ref-type="bibr"> Son et al., 2012 </xref> ), rice ( <xref rid="B8" ref-type="bibr"> Cao et al., 2006 </xref> ; <xref rid="B79" ref-type="bibr"> Zhang et al., 2013 </xref> ), wheat ( <xref rid="B69" ref-type="bibr"> Xu et al., 2007 </xref> ), cotton ( <xref rid="B26" ref-type="bibr"> Huang et al., 2007 </xref> ; <xref rid="B28" ref-type="bibr"> Jin and Liu, 2008 </xref> ), tomato ( <xref rid="B53" ref-type="bibr"> Sharma et al., 2010 </xref> ), cucumber ( <xref rid="B25" ref-type="bibr"> Hu and Liu, 2011 </xref> ), tobacco ( <xref rid="B19" ref-type="bibr"> Fischer and Droge-Laser, 2004 </xref> ), Chinese wild grapevine ( <xref rid="B83" ref-type="bibr"> Zhu et al., 2013 </xref> ), and peanut ( <xref rid="B62" ref-type="bibr"> Wan et al., 2014 </xref> ), among others. To our knowledge, only six members of the ERF subfamily have been functionally characterized in soybean ( <xref rid="B39" ref-type="bibr"> Mazarei et al., 2002 </xref> ; <xref rid="B75" ref-type="bibr"> Zhang et al., 2008 </xref> , <xref rid="B77" ref-type="bibr"> 2009 </xref> ; <xref rid="B76" ref-type="bibr"> Zhang G.Y. et al., 2010 </xref> ; <xref rid="B73" ref-type="bibr"> Zhai et al., 2013a </xref> , <xref rid="B74" ref-type="bibr"> b </xref> ; <xref rid="B15" ref-type="bibr"> Dong et al., 2015 </xref> ). Among these, <span class="gene">GmEREBP1</span> is assigned to the B-3 subgroup ( <xref rid="B39" ref-type="bibr"> Mazarei et al., 2002 </xref> ), while <span class="gene">GmERF3</span> and <span class="gene">GmERF7</span> ( <xref rid="B77" ref-type="bibr"> Zhang et al., 2009 </xref> ; <xref rid="B74" ref-type="bibr"> Zhai et al., 2013b </xref> ) are in the B-2 subgroup. These ERF proteins act as transcriptional activators and increase salt stress tolerance in tobacco. <span class="gene">GmERF4,</span> <span class="gene">GmERF5,</span> and <span class="gene">GmERF6</span> are assigned to the B-1 subgroup, and contain EAR motifs, which are characteristic of ERF repressors, and can repress negative regulators of plant defense responses; thereby, constitutive expression of <span class="gene">GmERF4</span> in transgenic tobacco plants increases tolerance to salt and <span class="trait">drought</span> stress ( <xref rid="B76" ref-type="bibr"> Zhang G.Y. et al., 2010 </xref> ), overexpression of <span class="gene">GmERF6</span> in transgenic <italic> Arabidopsis </italic> enhances <span class="trait">resistance</span> to <span class="trait">drought</span> stress ( <xref rid="B73" ref-type="bibr"> Zhai et al., 2013a </xref> ), and overexpression of <italic> <span class="gene">GmERF5</span> </italic> in tobacco and soybean plants improves <span class="trait">resistance</span> to <italic> P. nicotianae </italic> and <italic> P. sojae </italic> , respectively ( <xref rid="B15" ref-type="bibr"> Dong et al., 2015 </xref> ). There are three <italic> Arabidopsis </italic> ERFs in the B-4 subgroup, <italic> ABR1 </italic> , <italic> AtRAP2.6L </italic> , and <italic> AtRAP2.6 </italic> , which have been reported to respond to various biotic and abiotic stresses ( <xref rid="B3" ref-type="bibr"> Asahina et al., 2011 </xref> ; <xref rid="B13" ref-type="bibr"> Choi and Hwang, 2011 </xref> ; <xref rid="B2" ref-type="bibr"> Ali et al., 2013 </xref> ). As the only known soybean ERF transcription factor in the B-4 subgroup, <italic> <span class="gene">GmERF113</span> </italic> may positively regulate the expression of PR genes and enhance <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Ethylene response factor subfamily genes have crucial roles in the responses of plants to biotic stress ( <xref rid="B44" ref-type="bibr"> Okamuro et al., 1997 </xref> ; <xref rid="B55" ref-type="bibr"> Singh et al., 2002 </xref> ; <xref rid="B8" ref-type="bibr"> Cao et al., 2006 </xref> ; <xref rid="B5" ref-type="bibr"> Barah et al., 2013 </xref> ): overexpression of soybean <italic> <span class="gene">GmERF3</span> </italic> in transgenic tobacco led to increased <span class="trait">resistance</span> to <italic> Ralstonia solanacearum </italic> , <italic> Alternaria alternata </italic> , and TMV ( <xref rid="B77" ref-type="bibr"> Zhang et al., 2009 </xref> ); of <italic> AtERF6 </italic> in <italic> Arabidopsis </italic> enhanced <span class="trait">resistance</span> to the fungal pathogen, <italic> Botrytis cinerea </italic> ( <xref rid="B40" ref-type="bibr"> Moffat et al., 2012 </xref> ); and of <italic> RAP2.6 </italic> enhanced <span class="trait">resistance</span> against the beet cyst nematode, <italic> Heterodera schachtii </italic> , in <italic> Arabidopsis </italic> roots ( <xref rid="B2" ref-type="bibr"> Ali et al., 2013 </xref> ). In the present study, we demonstrated that overexpression of <italic> <span class="gene">GmERF113</span> </italic> increased soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Ethylene response factors regulate the expression of ET-inducible PR genes containing GCC-box sequences in their promoter regions ( <xref rid="B43" ref-type="bibr"> Ohme-Takagi and Shinshi, 1995 </xref> ). For example, <span class="gene">GmERF3</span> can bind to the GCC-box, and overexpression of <italic> <span class="gene">GmERF3</span> </italic> in transgenic tobacco activates the expression of several PR genes, including <italic> PR1 </italic> , <italic> PR2 </italic> , and <italic> PR4 </italic> ( <xref rid="B77" ref-type="bibr"> Zhang et al., 2009 </xref> ). Our results demonstrating significantly increased transcript levels of <italic> <span class="gene">GmPR1</span> </italic> and <italic> <span class="gene">GmPR10-1</span> </italic> in <italic> <span class="gene">GmERF113</span> </italic> transgenic soybean plants ( <bold> Figure <xref ref-type="fig" rid="F7"> 7 </xref> </bold> ) are consistent with these published data. Our previous research confirmed that transgenic soybean plants over-expressing <italic> <span class="gene">GmPR10</span> </italic> had increased <span class="trait">resistance</span> to <italic> P. sojae </italic> ( <xref rid="B27" ref-type="bibr"> Jiang et al., 2015 </xref> ). We speculated that <italic> <span class="gene">GmPR1</span> </italic> and <italic> <span class="gene">GmPR10-1</span> </italic> could be direct or indirect targets of <italic> <span class="gene">GmERF113</span> </italic> and the results presented here prove that <span class="gene">GmERF113</span> can positively regulate the expression of PR genes, thus improving soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Some <italic> ERFs </italic> are involved in the regulation of gene expression through interactions with other proteins or transcription factors ( <xref rid="B22" ref-type="bibr"> Gu et al., 2000 </xref> ; <xref rid="B68" ref-type="bibr"> Xu et al., 2011 </xref> ). For example, the AP2/ERF factor, NtORC1, interacts with NtbHLH and commonly regulates the expression of genes containing G-box and GCC motifs in their promoter regions ( <xref rid="B14" ref-type="bibr"> De Boer et al., 2011 </xref> ). A transcriptional repressor recently identified in banana fruit, MaERF10, interacts with MaJAZ3 proteins to enforce the repression of jasmonate (JA) biosynthesis-related genes involved in MeJA-mediated cold tolerance ( <xref rid="B47" ref-type="bibr"> Qi et al., 2016 </xref> ). In the present study, <span class="gene">GmbHLH</span> was found to interact with <span class="gene">GmERF113</span> using yeast two-hybrid and BiFC assays. Some bHLH proteins participate in regulation of PR gene expression ( <xref rid="B1" ref-type="bibr"> Abe et al., 1997 </xref> ; <xref rid="B20" ref-type="bibr"> Friedrichsen et al., 2002 </xref> ); for example, the bHLH transcription factor, AtHBI1, mediates pathogen-associated molecular pattern-triggered immunity in <italic> A. thaliana </italic> ( <xref rid="B17" ref-type="bibr"> <span class="gene">Fan</span> et al., 2014 </xref> ). We therefore speculate that <span class="gene">GmERF113</span> and <span class="gene">GmbHLH</span> proteins may cooperatively regulate <span class="trait">resistance</span> to <italic> P. sojae </italic> infection. Our previous research demonstrated that <span class="gene">GmERF5,</span> acting as a GCC-mediated transcriptional repressor, also interacted with <span class="gene">GmbHLH</span> and that overexpression of <italic> <span class="gene">GmERF5</span> </italic> could improve soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> ( <xref rid="B15" ref-type="bibr"> Dong et al., 2015 </xref> ). Thus, we hypothesize that <span class="gene">GmbHLH</span> may play a crucial role in modulating EFR transcription factors in defense against <italic> P. sojae </italic> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						The phytohormones ET, JA, SA, and ABA are important for the regulation of defense responses in plants ( <xref rid="B82" ref-type="bibr"> Zhou et al., 1997 </xref> ; <xref rid="B46" ref-type="bibr"> Pieterse et al., 2009 </xref> ; <xref rid="B52" ref-type="bibr"> Seo et al., 2011 </xref> ; <xref rid="B83" ref-type="bibr"> Zhu et al., 2013 </xref> ). Plant stress-tolerance is regulated through a network of signal transduction pathways, some of which may converge on ERF proteins through complex interactions ( <xref rid="B78" ref-type="bibr"> Zhang et al., 2004 </xref> ; <xref rid="B49" ref-type="bibr"> Rehman and Mahmood, 2015 </xref> ). For example, AtERF4 is thought to be a key factor in the regulation of ET/ABA-dependent defense pathways and could modulate the transcription of many ET/ABA-dependent defense genes ( <xref rid="B71" ref-type="bibr"> Yang et al., 2005 </xref> ); <span class="gene">GmERF3</span> may connect the ET, JA, and SA signaling pathways, which mediate biotic and abiotic stress responses ( <xref rid="B77" ref-type="bibr"> Zhang et al., 2009 </xref> ); <xref rid="B16" ref-type="bibr"> Dong (2001) </xref> determined that two well-defined signaling pathways involved in pathogen-defense responses make use of the plant hormones SA or ET/JA, respectively; <xref rid="B29" ref-type="bibr"> Kamsv&#x000e5;gmagnusson et al. (2014) </xref> also reported that the expression of genes encoding ERFs is regulated in both an ET-dependent and -independent manner; while jasmonates (JAs) also play central signaling roles, using MeJA as an elicitor, in a wide range of plant <span class="trait">resistance</span> responses ( <xref rid="B59" ref-type="bibr"> Thagun et al., 2016 </xref> ). Our present study demonstrated that levels of <italic> <span class="gene">GmERF113</span> </italic> mRNA transcripts were significantly enhanced by <italic> P. sojae </italic> , ET, and MeJA stress; however, the observed changes elicited in response to SA and ABA stress were relatively minor. We deduce that <span class="gene">GmERF113</span> may depend primarily on ET and MeJA signaling pathways, which mediate soybean responses to <italic> P. sojae </italic> infection.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on our data, we propose a model to explain the potential role of <span class="gene">GmERF113</span> as a positive regulator of soybean responses to <italic> P. sojae </italic> infection ( <bold> Figure <xref ref-type="fig" rid="F9"> 9 </xref> </bold> ). Expression of the <italic> <span class="gene">GmERF113</span> </italic> gene is activated by <italic> P. sojae </italic> infection, while exogenous ET and MeJA also induce moderate accumulation of <italic> <span class="gene">GmERF113</span> </italic> mRNA. <span class="gene">GmERF113</span> is involved in the integration of signals to activate the expression of PR genes through binding to GCC-box motifs, and thereby enhances soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> . Meanwhile, <span class="gene">GmERF113</span> and <span class="gene">GmbHLH</span> may interact to cooperatively regulate the <italic> P. sojae </italic> <span class="trait">resistance</span> response. As members of the same subfamily, <span class="gene">GmERF5</span> and <span class="gene">GmERF113</span> have similar functions, as described above. Furthermore, expression of <span class="gene">GmERF5</span> is significantly induced by ABA and SA, suggesting that this protein may be involved in ABA-mediated salt and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> is the soybean EAR motif-containing ERF transcription repressor demonstrated as involved in the response to pathogen infection ( <xref rid="B15" ref-type="bibr"> Dong et al., 2015 </xref> ), while <span class="gene">GmERF113</span> is the soybean ERF transcription activator with a crucial role in the defense of soybean against <italic> P. sojae </italic> infection. This study provides new insights into the mechanism by which the <span class="gene">GmERF113</span> protein regulates biotic stress responses in soybean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture and evolution of the mating type locus in fusaria that cause soybean sudden death syndrome and bean <span class="trait">root</span> <span class="trait">rot.</span></h2>
			<h3>PMID:24891421  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hughes TJ(1), O'Donnell K(2), Sink S(3), Rooney AP(4), Scandiani MM(5), Luque A(5), Bhattacharyya MK(6), Huang X(7).
				<br/> 
				年份：2014 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium tucumaniae is the only known sexually reproducing species among the seven closely related fusaria that cause soybean sudden death syndrome (SDS) or bean <span class="trait">root</span> <span class="trait">rot</span> (BRR). In a previous study, laboratory mating of F. tucumaniae yielded recombinant ascospore progeny but required two mating-compatible strains, indicating that it is heterothallic. To assess the reproductive mode of the other SDS and BRR fusaria, and their potential for mating, whole-genome sequences of two SDS and one BRR pathogen were analyzed to characterize their mating type (MAT) loci. This bioinformatic approach identified a <span class="gene">MAT1-1</span> idiomorph in F. virguliforme NRRL 22292 and MAT1-2 idiomorphs in F. tucumaniae NRRL 34546 and F. azukicola NRRL 54364. Alignments of the MAT loci were used to design PCR primers within the conserved regions of the flanking genes <span class="gene">APN1</span> and SLA2, which enabled primer walking to obtain nearly complete sequences of the MAT region for six <span class="gene">MAT1-1</span> and five MAT1-2 SDS/BRR fusaria. As expected, sequences of the highly divergent 4.7 kb <span class="gene">MAT1-1</span> and 3.7 kb MAT1-2 idiomorphs were unalignable. However, sequences of the respective idiomorphs and those that flank <span class="gene">MAT1-1</span> and MAT1-2 were highly conserved. In addition to three genes at <span class="gene">MAT1-1</span> (MAT1-1-1, MAT1-1-2, MAT1-1-3) and two at MAT1-2 (MAT1-2-1, MAT1-2-3), the MAT loci of the SDS/BRR fusaria also include a putative gene predicted to encode for a 252 amino acid protein of unknown function. Alignments of the MAT1-1-3 and MAT1-2-1 sequences were used to design a multiplex PCR assay for the MAT loci. This assay was used to screen DNA from 439 SDS/BRR isolates, which revealed that each isolate possessed <span class="gene">MAT1-1</span> or MAT1-2, consistent with heterothallism. Both idiomorphs were represented among isolates of F. azukicola, F. brasiliense, F. phaseoli and F. tucumaniae, whereas isolates of F. virguliforme and F. cuneirostrum were only <span class="gene">MAT1-1</span> and F. crassistipitatum were only MAT1-2. Finally, nucleotide sequence data from the <span class="gene">RPB1</span> and <span class="gene">RPB2</span> genes were used to date the origin of the SDS/BRR group, which was estimated to have occurred about 0.75 Mya (95% HPD interval: 0.27, 1.68) in the mid-Pleistocene, long before the domestication of the common bean or soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSGT1 is differently required for soybean Rps genes-mediated and basal <span class="trait">resistance</span> to Phytophthora sojae.</h2>
			<h3>PMID:24763608  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Q(1), Cui X, Su L, Xu N, Guo N, Xing H, Dou D.
				<br/> 
				年份：2014 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Using RNAi approach, we demonstrate that <span class="gene">GmSGT1</span> is an essential component in soybean against Phytophthora sojae, but not required for Rps 2 or Rps 3a-mediated <span class="trait">resistance.</span> Utilization of disease <span class="trait">resistance</span> in soybean is a major approach to combat <span class="trait">root</span> and stem <span class="trait">rot</span> disease, which is caused by Phytophthora sojae and poses a growing threat to soybean safety production. The SGT1 protein is essential for disease <span class="trait">resistance</span> in many plant species. Here, we analyzed and characterized functions of <span class="gene">GmSGT1</span> gene family in R protein-mediated <span class="trait">resistance</span> and basal defense in this important crop. Five candidate genes of <span class="gene">GmSGT1</span> were identified and they were grouped into three clades. Transcriptional levels of all the tested genes were highly induced upon P. sojae infection in four soybean cultivars that confer different resistant levels. Using a gene silencing system in soybean cotyledons, we demonstrated that silencing <span class="gene">GmSGT1</span> genes comprised race-specific <span class="trait">resistance</span> in soybean lines carrying genes at the following loci for race-specific <span class="trait">resistance</span> to P. sojae: Rps1a, Rps1c, Rps1d, Rps1k, and Rps8. In contrast, the <span class="trait">resistance</span> mediated by Rps2 or Rps3a was not affected. Silencing <span class="gene">GmSGT1</span> genes in cotyledons also reduced <span class="trait">resistance</span> to this pathogen in a moderately partial resistant cultivar. We further showed that transient overexpression of <span class="gene">GmSGT1-1</span> in Nicotiana benthamiana could enhance the <span class="trait">resistance</span> to P. capsici. These results suggest that <span class="gene">GmSGT1</span> is an essential component for soybean in resisting the pathogen and pathways of Rps-mediated disease <span class="trait">resistance</span> are diverse in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> In silico </italic> comparison of transcript abundances during <italic> Arabidopsis thaliana </italic> and <italic> Glycine max </italic> <span class="trait">resistance</span> to <italic> Fusarium virguliforme </italic></h2>
			<h3>PMID:18831797</h3>
				
					<h3>PMCID:2559896</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Jiazheng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> jyuan@siu.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Mengxia </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> mzhu@cs.siu.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Lightfoot </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ga4082@siu.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Iqbal </surname>  <given-names> M Javed </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mjiqbal@siu.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Jack Y </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> jyang@bwh.harvard.edu </email>  </contrib>  <contrib id="A6" corresp="yes" contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> meksemk@siu.edu </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Phytoalexins, phytoanticipins and signal molecules are three major natural products involved in plant defense with common precursors [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]. Phenylalanine ammonia-lyase (PAL; EC 4.3.1.5) expression has been associated with <span class="trait">resistance</span> to fungal pathogens in many plant species [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B19"> 19 </xref> ]. <span class="gene">PAL</span> catalyzes the deamination of phenylalanine to produce trans-cinnamic acid, the first step in the phenylpropanoid pathway leading to phytoalexins, lignins or coumarins. Multiple isoforms of the <italic> pal </italic> gene were identified in plants [ <xref ref-type="bibr" rid="B20"> 20 </xref> ]. Manipulation of PAL, the first enzyme of the phenylpropanoid pathway together with the downstream enzymes such as cinnamate 4-hydroxylase (C4H; EC 1.14.13.11), diphenol oxidase (laccase; EC 1.10.3.2) and 4-hydroxycinnamoyl CoA ligase (4CL; EC 6.2.1.12), revealed an association with <span class="trait">resistance</span> to viral and fungal infection [ <xref ref-type="bibr" rid="B21"> 21 </xref> - <xref ref-type="bibr" rid="B23"> 23 </xref> ]. Reduction of phenylpropanoid biosynthesis in tobacco via down-regulation of <span class="gene">PAL</span> reduced local and systemic acquired <span class="trait">resistance</span> to fungal or viral infection [ <xref ref-type="bibr" rid="B24"> 24 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Phenylpropanoid derived polymers like lignin also play an important role as a physical barrier against pathogen invasion [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Lignin, a complex racemic aromatic heteropolymer is the second most abundant cell wall polymer (after cellulose) and provides rigidity for the cell wall and a physical barrier against pathogens [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. Lignin is synthesized from the phenylpropanoid metabolism reactions. These series of hydroxylation and O-methylation and conversion of side-chain carboxyl to an alcohol result in the building blocks of lignin, which is initiated by deamination of phenylalanine by the enzyme <span class="gene">PAL</span> where hydroxycinnamic acid esters play a central role [ <xref ref-type="bibr" rid="B28"> 28 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> ]. <span class="gene">4CL</span> is responsible for the CoA esterification of p-coumaric acid. Down-regulation of isozymes of 4CLs may alter metabolite concentrations other than those involved in lignin production, with a secondary effect on growth as a consequence [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. Laccase was the first enzyme demonstrated to be able to perform lignin polymerization <italic> in vitro </italic> . Over-expressed laccase in the roots caused cell wall lignin deposition increases in the developing xylem [ <xref ref-type="bibr" rid="B31"> 31 </xref> , <xref ref-type="bibr" rid="B32"> 32 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> ]. Therefore lignin might be involved in the disease <span class="trait">resistance</span> mechanism of plant cells where wall fortification occurred in response to many pathogens and would be especially useful against <span class="trait">root</span> rots.
					</p>
				
					<p>&nbsp;&nbsp;
						In plant, SnRKs possess a catalytic domain similar to that of sucrose non-fermenting-1 (SNF1) of yeast ( <italic> Saccharomyces cerevisiae </italic> ) and AMP-activated protein kinase (AMPK) of animals. SnRK1, SnRK2 and SnRK3 are the key members of SnRKs family but the SnRK1 subfamily appears to share direct structural and functional homology with the SNF1/AMPK family. The SnRK1 protein complex can be further divided into 3 subunits: the &#x003b1; subunit which is SNF1-like protein; the &#x003b2; subunit that is composed of SIP1-, SIP2- and GAL83-like proteins; and the &#x003b3; subunit that is SNF4-like protein based on sequence structure and expression patterns [ <xref ref-type="bibr" rid="B33"> 33 </xref> , <xref ref-type="bibr" rid="B34"> 34 </xref> ]. Homologues of SnRKs occur in all kingdoms and they appear to be highly conserved among yeasts, animals and plants suggesting that they may play very similar roles across species [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. <span class="gene">SNF1</span> modulates the phosphorylation state of a number of metabolic enzymes whilst SnRK1 regulates several enzymes involved in sugar metabolism and cell energy metabolism [ <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B33"> 33 </xref> ]. In yeast, the function of <span class="gene">SNF1</span> is to coordinate about 600 genes to respond to lower cellular glucose concentrations. Hong et al. [ <xref ref-type="bibr" rid="B33"> 33 </xref> ] identified GAL83 as mediating carbon partitioning during the plant response to herbivore <italic> Manduca sexta </italic> attack. GAL83, a &#x003b2;-subunit of a heterotrimeric SnRK1, showed a decrease in source leaves whereas the abundance of the catalytic &#x003b1;-subunit of SNF remained unaltered. The herbivore-induced changes in sink-source relations in <italic> Nicotiana attenuata </italic> was regulated by the &#x003b2;-subunit of SnRK1 (SNF1-related kinase) protein kinase, GAL83. GAL83 silenced plants were unable to enhance <span class="trait">root</span> reserves, delay senescence or prolonged flowering following herbivore attack during early stages of development. In turn, SnRK1 can be used to alter resource allocation thereby plants may be equipped to better tolerate the pest attack [ <xref ref-type="bibr" rid="B37"> 37 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The transcriptional analysis of genes from the multiple branches of the phenylpropanoid pathway showed that 6 of the 31 genes were significantly altered in TA (Table <xref ref-type="table" rid="T2"> 2 </xref> ). Further, TA changes inferred that synthesis of cinnamic acid in Arabidopsis was one of the early responses to <italic> F. viguliforme </italic> infection. Phytoalexins and phytoanticipins serve both structural and metabolic functions in disease <span class="trait">resistance.</span> Our microarray data showed that the <span class="trait">resistance</span> response of the phenylpropanoid pathway was different in Arabidopsis and soybean [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The microarray data showed that the phenylpropanoid pathway was activated during the <span class="trait">resistance</span> response in the Arabidopsis. In contrast to soybean [ <xref ref-type="bibr" rid="B9"> 9 </xref> ], despite the fact that the <span class="gene">PAL</span> was induced in this metabolic pathway, other major genes such as cinnamic acid 4-hydroxylase (C4L), chalcone synthase (CHS) and cinnamyl-alcohol dehydrogenase (CAD) in the phenylpropanoid pathway were all suppressed (Table <xref ref-type="table" rid="T2"> 2 </xref> ), indicating that soybean and Arabidopsis did not share similar strategies in the specific pathway to <span class="trait">resistance</span> <italic> F. virguliforme </italic> . Based on the up-regulation of ESTs, total six genes in the phenylpropanoid pathway had more than 1-fold increase in abundances (Log2 scale). However, the rest of genes involved in major branches in the pathway were either down-regulated or no significant increase in TAs (Table <xref ref-type="table" rid="T2"> 2 </xref> ). The suppression of these branches of the phenylpropanoid pathway seemed somehow different from the soybean response to the pathogen [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. Since transcripts of those enzymes leading to the synthesis of flavonols, terpenes and proanthocyanidins, decrease in abundances of down-stream genes in the procedure of the <span class="trait">resistance</span> response in the pathway but increase in <span class="gene">PAL</span> and cinnamoyl-CoA reductase transcripts during this same time suggesting the existence of a potential bypass to synthesize secondary metabolites in the defense response. After the 192 soybean genes have been converted into 158 functional orthologs of Arabidopsis genes, comparison of the protein-protein interaction network in both Arabidopsis and soybean demonstrated a high specificity trend in gene regulation of the two species (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). There were 12 hubs identified in the 158 functional orthologous soybean genes whilst more than 15 network hubs were observed to mediate the <span class="trait">resistance</span> in the up-regulated Arabidopsis genes. Unfortunately, no <span class="trait">resistance</span> pathway was generated from our Arabidopsis microarray data by the Arabidopsis Interactions Viewer program in the bin16 and no significant interlog among genes in bin16 of the data obtained.
					</p>
				
					<p>&nbsp;&nbsp;
						It is well-known that over-expression <span class="gene">PAL</span> in tobacco plant can effectively be used against a virulent fungal pathogen <italic> Cercospora nicotiana </italic> infection and also altered the lesion phenotype inoculated with TMV [ <xref ref-type="bibr" rid="B57"> 57 </xref> ]. Reduction of phenylpropanoid biosynthesis in tobacco via down-regulation of <span class="gene">PAL</span> reduces local and systemic acquired <span class="trait">resistance</span> to fungal or viral infection [ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ]. The major function of <span class="gene">PAL</span> is to catalyze phytoalexins and phytoanticipins production and thereby reduce disease severity [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. Constitutive over-expression of the <span class="gene">PAL</span> gene from the tropical pasture legume <italic> Stylosanthes humilis </italic> in tobacco plants provided <span class="trait">resistance</span> to <italic> Cercospora nicotianae </italic> and to pathogen <italic> Phytophthora parasitica </italic> pv. <italic> Nicotianae </italic> [ <xref ref-type="bibr" rid="B59"> 59 </xref> ]. The key enzymes of phenylpropanoid pathway, PAL, C4L and <span class="gene">CHS</span> were all induced in soybean <span class="trait">resistance</span> to <italic> F. virguliforme </italic> [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The global expression analysis of the Arabidopsis plant challenged by <italic> F. virguliforme </italic> appeared to be different. The fungal pathogen affected the secondary metabolism not only in the phenylpropanoid pathway but also via pathways leading to other cellular functions (terpenes, phenolics, special N compounds metabolism). Transcripts of the <span class="gene">PAL</span> gene encode the key enzyme feeding all these branches. Therefore, the decreases in most key transcripts at the times when <span class="gene">PAL</span> and cinnamoyl-CoA reductase transcripts increases in abundance is a novel finding showing that the synthesis of <span class="gene">PAL</span> and cinnamoyl-CoA reductase transcripts is with negative correlation to the activity of the other genes during the pathogen stress response. Perhaps in <italic> A. thaliana </italic> , the phenylpropanoid pathway did not directly participate in the cascade of reactions elicited during the defense and probably, the biosynthesis of phytoalexins and phytoanticipins was regulated in different fashions in soybean and Arabidopsis. Transcripts of genes involved in the synthesis of lignin, flavonols, anthocyanins and proanthocyanidins (data not shown) seemed to be down-regulated during the periods of time when the genes involved in the synthesis of cinnamic acid was up-regulated. Arabidopsis lacks isoflavonoid and phytoalexins and produces mainly camalexin rather than a cocktail of isoflavonoids [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. Therefore, in Arabidopsis the flavonoids (anthocyanins, proanthocyanidins, flavones and flavonols) may indeed not play a significant role in defense. In contrast, there was evidence that cinnamoyl alcohol dehydrogenase (CAD) was induced rapidly in Arabidopsis infected with <italic> Xanthomonas </italic> [ <xref ref-type="bibr" rid="B61"> 61 </xref> ] and CAD is in the pathway that leads to lignin biosynthesis. In the phenylpropanoid pathway, key genes like <span class="gene">PAL</span> are encoded by a group of genes showing a great degree of sequence diversity. Significantly up-regulated <span class="gene">PAL</span> transcripts support for the hypothesis that in Arabidopsis expression of the <span class="gene">PAL</span> is significantly associated with pathogen infection. However, the <span class="gene">PAL</span> over-expression plants caused a significant reduction in growth and delayed flowering. These phenotypes may also be related to energy status or amino acid pool availability being altered by PAL.
					</p>
				
					<p>&nbsp;&nbsp;
						Orthologous genes are defined by direct evolutionary descent and should play similar developmental or physiological roles. Several gene groups were identified according to the outcome of functional orthologous analysis in this study, showing same functional orthologous relationships in these two species. In our primary results, the 1 to 1 ratio of the orthologous relationship was found among those several paired groups, indicating these gene groups were descended from a common ancestor and corresponded to well-conserved functions. These 1 to 1 ortholog classes are presumed to represent conserved functions in Arabidopsis and soybean, but they shared diverse bootstrapping value. Diversification following gene duplication may have occurred but the degree is still unknown. Twenty-eight pairs of the genes from both species were taxoned together and each pair possessed unique bootstrapping value and Kimura phylogenetic distance (data not shown). Based on phylogenetic analysis, the organization of soybean phenylalanine ammonia-lyase (PAL) proteins were very similar to that of the Arabidopsis <span class="gene">PAL</span> proteins, implying that the soybean and Arabidopsis <span class="gene">PAL</span> proteins analyzed here were all derived from a common ancestor. The fact that these clades were formed by both Arabidopsis and soybean <span class="gene">PAL</span> genes gave a suggestion that these <span class="gene">PAL</span> genes existed before the divergence of monocots and dicots. Interestingly, fair numbers of EST sequences were not found in the other organism, even in closely related gene families that were associated with the fungal <span class="trait">resistance.</span> However, detailed screening phylogenetic relationship among those <span class="trait">resistance</span> genes is necessary for detection of the gene evolution.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenylalanine ammonia-lyase2.1 contributes to the soybean response towards Phytophthora sojae infection.</h2>
			<h3>PMID:28775360</h3>
				

				
			<p>
				作者：Zhang C(1), Wang X(1)(2), Zhang F(3), Dong L(1), Wu J(4), Cheng Q(1), Qi D(1), Yan X(1), Jiang L(1), Fan S(1), Li N(1)(5), Li D(1), Xu P(6), Zhang S(7).
				<br/> 
				年份：2017 Aug 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean [Glycine max (L.) Merr.] caused by Phytophthora sojae is a destructive disease worldwide. Phenylalanine ammonia-lyase (PAL) is one of the most extensively studied enzymes related to plant responses to biotic and abiotic stresses. However, the molecular mechanism of <span class="gene">PAL</span> in soybean in response to P. sojae is largely unclear. Here, we characterize a novel member of the soybean <span class="gene">PAL</span> gene family, <span class="gene">GmPAL2.1,</span> which is significantly induced by P. sojae. Overexpression and RNA interference analysis demonstrates that <span class="gene">GmPAL2.1</span> enhances <span class="trait">resistance</span> to P. sojae in transgenic soybean plants. In addition, the <span class="gene">PAL</span> activity in <span class="gene">GmPAL2.1-OX</span> transgenic soybean is significantly higher than that of non-transgenic plants after infection with P. sojae, while that in <span class="gene">GmPAL2.1-RNAi</span> soybean plants is lower. Further analyses show that the daidzein, genistein and salicylic acid (SA) levels and the relative content of glyceollins are markedly increased in <span class="gene">GmPAL2.1-OX</span> transgenic soybean. Taken together, these results suggest the important role of <span class="gene">GmPAL2.1</span> functioning as a positive regulator in the soybean response to P. sojae infection, possibly by enhancing the content of glyceollins, daidzein, genistein and SA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean Dirigent Gene <italic> <span class="gene">GmDIR22</span> </italic> Contributes to Promotion of Lignan Biosynthesis and Enhances <span class="trait">Resistance</span> to <italic> Phytophthora sojae </italic></h2>
			<h3>PMID:28725237</h3>
				

				
					<h3>PMCID:5495835</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ninghui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Ming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Tengfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Junjiang </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Le </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chuanzhong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Wencheng </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/418977/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257714/overview"/>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> caused by the oomycete pathogen Phytophthora sojae is a destructive disease of soybean worldwide. Plant dirigent proteins (DIR) are proposed to have roles in biosynthesis of either lignan or lignin-like molecules, and are important for defense responses, secondary metabolism, and pathogen <span class="trait">resistance.</span> In the present work, a novel DIR gene expressed sequence tag is identified as up-regulated in the highly resistant soybean cultivar 'Suinong 10' inoculated with P. sojae. The full length cDNA is isolated using rapid amplification of cDNA ends, and designated <span class="gene">GmDIR22</span> (GenBank accession no. HQ_993047). The full length <span class="gene">GmDIR22</span> is 789 bp and contains a 567 bp open reading frame encoding a polypeptide of 188 amino acids. The sequence analysis indicated that <span class="gene">GmDIR22</span> contains a conserved dirigent domain at amino acid residues 43-187. The quantitative real-time reverse transcription PCR demonstrated that soybean <span class="gene">GmDIR22</span> mRNA is expressed most highly in stems, followed by roots and leaves. The treatments with stresses demonstrated that <span class="gene">GmDIR22</span> is significantly induced by P. sojae and gibberellic acid (GA3), and also responds to salicylic acid, methyl jasmonic acid, and abscisic acid. The <span class="gene">GmDIR22</span> is targeted to the cytomembrane when transiently expressed in Arabidopsis protoplasts. Moreover, The <span class="gene">GmDIR22</span> recombinant protein purified from Escherichia coli could effectively direct E-coniferyl alcohol coupling into lignan (+)-pinoresinol. Accordingly, the overexpression of <span class="gene">GmDIR22</span> in transgenic soybean increased total lignan accumulation. Moreover, the lignan extracts from <span class="gene">GmDIR22</span> transgenic plants effectively inhibits P. sojae hyphal growth. Furthermore, the transgenic overexpression of <span class="gene">GmDIR22</span> in the susceptible soybean cultivar 'Dongnong 50' enhances its <span class="trait">resistance</span> to P. sojae. Collectively, these data suggested that the primary role of <span class="gene">GmDIR22</span> is probably involved in the regulation of lignan biosynthesis, and which contributes to <span class="trait">resistance</span> to P. sojae.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> of soybean (PRR), caused by the oomycete pathogen <italic> Phytophthora sojae </italic> , is a destructive soil-borne disease of soybean all around the world ( <xref rid="B66" ref-type="bibr"> Wrather et al., 1997 </xref> ; <xref rid="B62" ref-type="bibr"> Tyler, 2007 </xref> ). In our previous study, a cDNA library was constructed from the leaf tissues of the highly resistant soybean &#x02018;Suinong 10,&#x02019; using suppression subtractive hybridization, and the mRNAs encoding expressed sequence tags (ESTs) showed increased expression during <italic> P. sojae </italic> infection ( <xref rid="B68" ref-type="bibr"> Xu et al., 2012 </xref> ). In this study, a full-length cDNA corresponding to an up-regulated EST encoding a conservative dirigent domain was isolated from soybean &#x02018;Suinong 10&#x02019; using rapid amplification of cDNA ends (RACE). The full-length cDNA belonged to a new member of the soybean DIR family and was designated <italic> <span class="gene">GmDIR22</span> </italic> (GenBank accession no. HQ_993047). The expression patterns of the <italic> <span class="gene">GmDIR22</span> </italic> gene were characterized in response to abiotic and biotic stresses, and its expression levels were determined in different soybean tissues. <span class="gene">GmDIR22</span> protein expressed in <italic> Escherichia coli </italic> was purified, and its function was investigated. Furthermore, the content of lignan and lignin in transgenic plants overexpressing <italic> <span class="gene">GmDIR22</span> </italic> was established and the antimicrobial activity of lignan extracts against <italic> P. sojae </italic> hyphal growth was evaluated. Moreover, the transgenic soybean plants over-expressing <italic> <span class="gene">GmDIR22</span> </italic> gene were produced, and their antimicrobial properties were also investigated.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate whether overexpression of <italic> <span class="gene">GmDIR22</span> </italic> could enhance <span class="trait">resistance</span> to <italic> P. sojae </italic> in transgenic plants, the T <sub> 5 </sub> transgenic soybean plants were selected by qRT-PCR (Supplementary Table <xref ref-type="supplementary-material" rid="SM14"> S11 </xref> ) to assay the pathogen response ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). The living cotyledons and detached leaves were treated with a <italic> P. sojae </italic> inoculum. As shown in <bold> Figure <xref ref-type="fig" rid="F6"> 6B </xref> </bold> , the cotyledons of the non-transgenic soybean plants exhibits clearer and larger water-soaked lesions compared with those of the transgenic plants after 72 h of incubation with <italic> P. sojae </italic> , and the lesion areas are significantly smaller in the transgenic lines than those of non-transgenic soybean plants 72 h after inoculation ( <italic> P </italic> &#x0003c; 0.01) ( <bold> Figure <xref ref-type="fig" rid="F6"> 6C </xref> </bold> ). After 96 h incubation with <italic> P. sojae </italic> , the similar results are observed in the detached leaves ( <bold> Figure <xref ref-type="fig" rid="F6"> 6E </xref> </bold> ), and the lesion areas in transgenic soybean lines were significantly smaller than those in non-transgenic plants ( <italic> P </italic> &#x0003c; 0.01) ( <bold> Figure <xref ref-type="fig" rid="F6"> 6D </xref> </bold> ). The biomass of <italic> P. sojae </italic> , based on the transcript levels of the <italic> P. sojae TEF1 </italic> gene (Supplementary Table <xref ref-type="supplementary-material" rid="SM13"> S10 </xref> ), was significantly lower in transgenic <italic> <span class="gene">GmDIR22</span> </italic> -overexpressing plants than that in non-transgenics ( <italic> P </italic> &#x0003c; 0.01) ( <bold> Figure <xref ref-type="fig" rid="F6"> 6F </xref> </bold> ). These data indicated that the over-expression of <italic> <span class="gene">GmDIR22</span> </italic> in soybean plants in a certain extent improves their <span class="trait">resistance</span> to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Resistance</span> analysis of <italic> <span class="gene">GmDIR22</span> </italic> transgenic soybean plants to <italic> P. sojae </italic> infection. <bold> (A) </bold> The qRT-PCR results demonstrating the relative abundance of <italic> <span class="gene">GmDIR22</span> </italic> in transgenic soybean plants. <bold> (B,E) </bold> The disease symptoms on the living cotyledons and detached leaves of transgenic lines treated with <italic> P. sojae </italic> inoculum at 72 h and 96 h, respectively. <bold> (C) </bold> The relative lesion area of transgenic soybean cotyledons infected with <italic> P. sojae </italic> after 72 h of inoculation. <bold> (D) </bold> The relative lesion area of transgenic soybean leaves infection with <italic> P. sojae </italic> after 96 h of inoculation. <bold> (F) </bold> qRT-PCR analysis of <italic> P. sojae </italic> relative biomass based on the transcript level of the <italic> P. sojae TEF1 </italic> gene in infected living cotyledons after 72 h of inoculation. Experiments were performed using three biological replicates with three technical replicates each, and statistically analyzed using Student&#x02019;s <italic> t </italic> -tests ( <sup> &#x02217; </sup> <italic> P </italic> &#x0003c; 0.05, <sup> &#x02217;&#x02217; </sup> <italic> P </italic> &#x0003c; 0.01). Bars indicated standard errors of the mean.
					</p>
				
					<p>&nbsp;&nbsp;
						Dirigent proteins were first identified in Forsythia ( <xref rid="B17" ref-type="bibr"> Davin et al., 1997 </xref> ) and are considered to be responsible for an important enzymatic reaction in the production of lignan and lignin, which evolved during the adaptation of aquatic plants to terrestrial environments ( <xref rid="B31" ref-type="bibr"> Kim et al., 2002 </xref> ). In addition, this gene family has primary roles in defense responses, secondary metabolism, and fiber biosynthesis ( <xref rid="B14" ref-type="bibr"> Davin and Lewis, 2000 </xref> ; <xref rid="B7" ref-type="bibr"> Burlat et al., 2001 </xref> ; <xref rid="B55" ref-type="bibr"> Ralph et al., 2007 </xref> ). However, little is known about dirigent and dirigent-like proteins and their biological functions in soybean, especially the likely effect on <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean. In this study, a novel dirigent gene, <italic> <span class="gene">GmDIR22</span> </italic> , that plays a positive role in soybean during infection with <italic> P. sojae </italic> . Here, we isolated and characterized the <italic> <span class="gene">GmDIR22</span> </italic> gene and our findings contributed to improved understanding of the function of dirigent genes in defense responses to <italic> P. sojae </italic> and secondary metabolism.
					</p>
				
					<p>&nbsp;&nbsp;
						Dirigent proteins are members of a large family, initially subdivided into five subfamilies (DIR-a, DIR-b, DIR-c, DIR-d, and DIR-e) ( <xref rid="B54" ref-type="bibr"> Ralph et al., 2006 </xref> ). With the increasing numbers of DIR proteins, the DIR-b and DIR-d subfamilies are combined together with the appearance of the DIR-f and DIR-g subfamilies ( <xref rid="B55" ref-type="bibr"> Ralph et al., 2007 </xref> ). The phylogenetic analysis demonstrated that <italic> <span class="gene">GmDIR22</span> </italic> is close to five <italic> AtDIR </italic> members that had been confirmed belonging to DIR-b/d subfamily ( <xref rid="B55" ref-type="bibr"> Ralph et al., 2007 </xref> ), indicating that <italic> <span class="gene">GmDIR22</span> </italic> might display similar functions with DIR-b/d. Many members of which are indicated to play a role in plants responses to biotic and abiotic stresses, including H <sub> 2 </sub> O <sub> 2 </sub> , NaCl, and PEG ( <xref rid="B23" ref-type="bibr"> Guo et al., 2012 </xref> ), SA, GA, and MeJA ( <xref rid="B12" ref-type="bibr"> Damaj et al., 2010 </xref> ). SA and JA are signaling molecules associated with pathogens, insect and mechanical damage (wound) <span class="trait">resistance.</span> The apparent upregulation of many DIR-b/d genes in response to attacks by fungi ( <xref rid="B65" ref-type="bibr"> Williams et al., 2002 </xref> ; <xref rid="B71" ref-type="bibr"> Zhu et al., 2007 </xref> ) and insects ( <xref rid="B55" ref-type="bibr"> Ralph et al., 2007 </xref> ) are of particular interest. Moreover, they have important roles in plant secondary metabolism, especially contribute to lignin formation ( <xref rid="B71" ref-type="bibr"> Zhu et al., 2007 </xref> ; <xref rid="B41" ref-type="bibr"> Ma and Liu, 2015 </xref> ). The expression pattern determined that the transcript levels of <italic> <span class="gene">GmDIR22</span> </italic> was higher in resistant cultivars than that in susceptible cultivars with <italic> P. sojae </italic> infection. The result indicated that <italic> <span class="gene">GmDIR22</span> </italic> may contribute to naturally occurring <span class="trait">resistance</span> to the pathogen. Moreover, <italic> <span class="gene">GmDIR22</span> </italic> mRNA transcripts are strongly induced by GA <sub> 3 </sub> stress; therefore, we speculate that <italic> <span class="gene">GmDIR22</span> </italic> may have an important role in soybean plant <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> , primarily through GA <sub> 3 </sub> signaling, which is an important signal transduction component involved in activation of plant defense responses against pathogen attack ( <xref rid="B45" ref-type="bibr"> Mellersh et al., 2002 </xref> ). Future studies are required to determine how this process may be regulated by GA <sub> 3 </sub> signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						Dirigent proteins proteins are involved in plant defenses against pathogens and are proposed to mediate the free radical coupling of monolignol plant phenols to <span class="trait">yield</span> the cell wall polymers, lignans and/or lignin-like compounds ( <xref rid="B20" ref-type="bibr"> Espi&#x000f1;eira et al., 2011 </xref> ; <xref rid="B2" ref-type="bibr"> Alvarez et al., 2016 </xref> ). In this study, the soybean plants overexpressing the <italic> <span class="gene">GmDIR22</span> </italic> gene exhibit increased lignan accumulation, compared with controls. In contrast, the levels of acid-insoluble lignin remain almost unchanged in transgenic plants relative to those in controls. We propose two possible explanations for this result: first, <span class="gene">GmDIR22</span> protein may direct the formation of lignan precursors, and is mainly, if not completely, involved in lignan biosynthesis; second, in the defense against pathogen invasion, the <span class="gene">GmDIR22</span> dirigent protein may induce production of a lignin-like defense barrier, consisting of suberized or lignified metabolites associated with specialized phloem parenchyma cells or resin ducts, rather than being directly involved in the production of lignin. This is conceivable when spatial separation of lignan and lignin synthesis systems is considered, and the former occurs in the cytoplasm, while the latter takes place in the cell wall compartment ( <xref rid="B13" ref-type="bibr"> Davin et al., 2008 </xref> ; <xref rid="B6" ref-type="bibr"> Bonawitz and Chapple, 2010 </xref> ). The subcellular localization of the <span class="gene">GmDIR22</span> protein was investigated, and the results indicated that <span class="gene">GmDIR22</span> is a membrane protein, suggesting that the expression of this protein in the membrane would probably be conducive when receiving stimulus signals and generate corresponding responses.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, lignan extracts from <italic> <span class="gene">GmDIR22</span> </italic> transgenic plants exhibit a certain effect inhibition on <italic> P. sojae </italic> hyphal growth. Furthermore, we determined that the recombinant <span class="gene">GmDIR22</span> protein could effectively direct <italic> E </italic> -coniferyl alcohol coupling into (+)-pinoresinol, suggesting that overexpression of soybean <italic> <span class="gene">GmDIR22</span> </italic> is likely to promote the synthesis of lignans, leading to enhanced <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean to a certain degree. The results of analysis of lignan extracts on the inhibition of <italic> P. sojae </italic> hyphal growth and the living cotyledons and detached leaves of transgenic soybean plants enhancing <span class="trait">resistance</span> to <italic> P. sojae </italic> also support this hypothesis.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Organization, expression and evolution of a disease <span class="trait">resistance</span> gene cluster in soybean.</h2>
			<h3>PMID:12524363  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Graham MA(1), Marek LF, Shoemaker RC.
				<br/> 
				年份：2002 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						PCR amplification was previously used to identify a cluster of <span class="trait">resistance</span> gene analogues (RGAs) on soybean linkage group J. <span class="trait">Resistance</span> to powdery mildew (Rmd-c), Phytophthora stem and <span class="trait">root</span> <span class="trait">rot</span> (Rps2), and an ineffective nodulation gene (Rj2) map within this cluster. BAC fingerprinting and RGA-specific primers were used to develop a contig of BAC clones spanning this region in cultivar "Williams 82" [rps2, Rmd (adult onset), rj2]. Two cDNAs with homology to the TIR/NBD/LRR family of R-genes have also been mapped to opposite ends of a BAC in the contig <span class="gene">Gm_Isb001_091F11</span> (BAC 91F11). Sequence analyses of BAC 91F11 identified 16 different resistance-like gene (RLG) sequences with homology to the TIR/NBD/LRR family of disease <span class="trait">resistance</span> genes. Four of these RLGs represent two potentially novel classes of disease <span class="trait">resistance</span> genes: TIR/NBD domains fused inframe to a putative defense-related protein (NtPRp27-like) and <span class="gene">TIR</span> domains fused inframe to soybean calmodulin Ca(2+)-binding domains. RT-PCR analyses using gene-specific primers allowed us to monitor the expression of individual genes in different tissues and developmental stages. Three genes appeared to be constitutively expressed, while three were differentially expressed. Analyses of the R-genes within this BAC suggest that R-gene evolution in soybean is a complex and dynamic process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular response to the pathogen <italic> Phytophthora sojae </italic> among ten soybean near isogenic lines revealed by comparative transcriptomics</h2>
			<h3>PMID:24410936</h3>
				

				
					<h3>PMCID:3893405</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Lin </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> feng.lin@pioneer.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Zhao </surname>  <given-names> Meixia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhao185@purdue.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Baumann </surname>  <given-names> Douglas D </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> dbaumann@uwlax.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Ping </surname>  <given-names> Jieqing </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ping@purdue.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Sun </surname>  <given-names> Lianjun </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sun259@purdue.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Liu </surname>  <given-names> Yunfeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> liu969@purdue.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhang </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zb380345709@126.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Tang </surname>  <given-names> Zongxiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zxtang@sicau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Hughes </surname>  <given-names> Elisa </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> elisadh@umich.edu </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Doerge </surname>  <given-names> Rebecca W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> doerge@purdue.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Hughes </surname>  <given-names> Teresa J </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> hughestj@purdue.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A12">  <name>  <surname> Ma </surname>  <given-names> Jianxin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> maj@purdue.edu </email>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The recognition of pathogen effectors triggers an innate immunity response that is mediated by two distinct types of NB-LRR proteins, the toll-interleukin-1 receptor (TIR)-NB-LRR proteins and the coiled-coil (CC)-NB-LRR proteins. The former requires EDS1 (ENHANCED DISEASE SUSCEPTIBILITY 1), a central regulator of effector-triggered immunity (ETI), which functions together with <span class="gene">PAD4</span> (PHYTOALEXIN DEFICIENT 4). In contrast, the latter NB-LRR proteins are independent of EDS1 but require NDR1 (NON-RACE-SPECIFIC DISEASE <span class="trait">RESISTANCE</span> 1) [ <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B21"> 21 </xref> ]. The interaction among these intracellular proteins results in a regulation network of phytohormone signaling, which is mainly mediated by salicylic acid (SA) for biotrophic and hemibiotrophic pathogens, and jasmonic acid (JA) and ethylene (ET) for necrotrophic pathogens [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B22"> 22 </xref> ]. In addition to SA, JA and ET, other phytohormones such as brassinosteroids (BR), abscisic acid (ABA), auxin, gibberellins (GA) and cytokinin (CK) also contribute to plant immune responses, with complex crosstalk between one another [ <xref ref-type="bibr" rid="B23"> 23 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies indicated that the <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean was mediated by SA signaling [ <xref ref-type="bibr" rid="B24"> 24 </xref> - <xref ref-type="bibr" rid="B26"> 26 </xref> ], with the <italic> NPR1 </italic> as the key component of SA-mediated signaling [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. In this study, two <italic> NPR1 </italic> -like IIGs ( <italic> <span class="gene">Glyma02g45260</span> </italic> and <italic> <span class="gene">Glyma14g03510</span> </italic> ) were identified (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> ). However, these two genes differ from the two <italic> NPR1 </italic> -like genes reported by Sandhu et al [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. In the later study, they studied soybean leaves four dpi with <italic> P. sojae </italic> race 4 while, in the present study, we studied stems 24 hpi with <italic> P. sojae </italic> race 1. As such, these two studies are less comparable.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the three major phytohormones described above, BR have also been found to enhance disease <span class="trait">resistance</span> against viral and bacterial pathogens in tobacco and rice [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. <italic> BAK1 </italic> is a crucial component of BR signaling, and can interact with <span class="gene">LRR-RLK</span> (leucine-rich repeat receptor-like protein kinase) genes such as <italic> FLS2 </italic> to promote their function in plant defense responses [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. As shown in Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , three <italic> BAK1 </italic> homologs were identified as IIGs and all were up-regulated upon inoculation with the pathogen, suggesting a positive role of BR signaling in defense response to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						It is documented that <span class="trait">resistance</span> genes recognize pathogens either directly or indirectly by guarding a protein or using a decoy [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. In <italic> Arabidopsis </italic> , <italic> RIN4 </italic> is &#x02018;guarded&#x02019; by <span class="trait">resistance</span> genes <italic> RPM1 </italic> and <italic> RPS2 </italic> , which trigger the immune response [ <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> ]. In soybean, two of four <italic> RIN4 </italic> - homologous genes appear to function as a heteromeric complex in mediating RPG1-B and RPM1-derived <span class="trait">resistance</span> to <italic> Pseudomonas syringae </italic> , and silencing <italic> <span class="gene">GmRIN4a</span> </italic> or <italic> <span class="gene">GmRIN4b</span> </italic> in <italic> rpg1-b </italic> plants enhances basal <span class="trait">resistance</span> to virulent strains of <italic> P. syringae </italic> and the oomycete <italic> P. sojae </italic> [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. In our study, a single <italic> RIN4 </italic> homolog was found to be up-regulated in C-III NILs, but not in NILs within C-I and C-II. If this <italic> RIN4 </italic> homolog did encode the &#x0201c;guard&#x0201d; protein that was recognized by the <italic> Rps </italic> genes in the C-III NILs, then the relatively low numbers of IIGs identified among NILs in C-I and C-II, as compared with the C-III NILs, may be explained by the distinct &#x0201c;recognition&#x0201d; processes needed for triggering the immune response, which may result in variations in speed, timing, and magnitude of molecular responses to the pathogen [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ].
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies indicated that the <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean was mediated by SA signaling [ <xref ref-type="bibr" rid="B24"> 24 </xref> - <xref ref-type="bibr" rid="B26"> 26 </xref> ], with the <italic> NPR1 </italic> as the key component of SA-mediated signaling [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. In this study, two <italic> NPR1 </italic> -like IIGs ( <italic> <span class="gene">Glyma02g45260</span> </italic> and <italic> <span class="gene">Glyma14g03510</span> </italic> ) were identified (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> ). However, these two genes differ from the two <italic> NPR1 </italic> -like genes reported by Sandhu et al [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. In the later study, they studied soybean leaves four dpi with <italic> P. sojae </italic> race 4 while, in the present study, we studied stems 24 hpi with <italic> P. sojae </italic> race 1. As such, these two studies are less comparable.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the three major phytohormones described above, BR have also been found to enhance disease <span class="trait">resistance</span> against viral and bacterial pathogens in tobacco and rice [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. <italic> BAK1 </italic> is a crucial component of BR signaling, and can interact with <span class="gene">LRR-RLK</span> (leucine-rich repeat receptor-like protein kinase) genes such as <italic> FLS2 </italic> to promote their function in plant defense responses [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. As shown in Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , three <italic> BAK1 </italic> homologs were identified as IIGs and all were up-regulated upon inoculation with the pathogen, suggesting a positive role of BR signaling in defense response to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						It is documented that <span class="trait">resistance</span> genes recognize pathogens either directly or indirectly by guarding a protein or using a decoy [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]. In <italic> Arabidopsis </italic> , <italic> RIN4 </italic> is &#x02018;guarded&#x02019; by <span class="trait">resistance</span> genes <italic> RPM1 </italic> and <italic> RPS2 </italic> , which trigger the immune response [ <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B15"> 15 </xref> ]. In soybean, two of four <italic> RIN4 </italic> - homologous genes appear to function as a heteromeric complex in mediating RPG1-B and RPM1-derived <span class="trait">resistance</span> to <italic> Pseudomonas syringae </italic> , and silencing <italic> <span class="gene">GmRIN4a</span> </italic> or <italic> <span class="gene">GmRIN4b</span> </italic> in <italic> rpg1-b </italic> plants enhances basal <span class="trait">resistance</span> to virulent strains of <italic> P. syringae </italic> and the oomycete <italic> P. sojae </italic> [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. In our study, a single <italic> RIN4 </italic> homolog was found to be up-regulated in C-III NILs, but not in NILs within C-I and C-II. If this <italic> RIN4 </italic> homolog did encode the &#x0201c;guard&#x0201d; protein that was recognized by the <italic> Rps </italic> genes in the C-III NILs, then the relatively low numbers of IIGs identified among NILs in C-I and C-II, as compared with the C-III NILs, may be explained by the distinct &#x0201c;recognition&#x0201d; processes needed for triggering the immune response, which may result in variations in speed, timing, and magnitude of molecular responses to the pathogen [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Candidate Gene Analysis of a Novel Phytophthora <span class="trait">Resistance</span> Gene <italic> Rps10 </italic> in a Chinese Soybean Cultivar</h2>
			<h3>PMID:23936102</h3>
				
					<h3>PMCID:3723638</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jiqing </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Changjian </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Duan </surname>  <given-names> Canxing </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Suli </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiaoming </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Xiaofei </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Zhendong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To date, 19 <italic> Rps </italic> genes/alleles have been reported, but only three <italic> Rps </italic> genes, <italic> Rps1 </italic> k, <italic> Rps2 </italic> and <italic> Rps4 </italic> , have been cloned and characterized as NBS-LRR genes, which involved recognizing the presence of pathogens and ultimately confer <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0069799-Gao1"> [35] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0069799-Sandhu1"> [37] </xref> . In this study, eight putative genes were found in the mapped region and the four un-annotated genes out of the eight genes were analyzed by BLAST searching the nucleotide sequence at the NCBI and the putative protein sequence at the EMBL-EBI databases. <span class="gene">Glyma17g28740.1</span> might function as a DNA/RNA polymerase or in nucleic acid binding. <span class="gene">Glyma17g28980.1</span> shared 100% identity with a gene (GenBank No. XM_003551110) encoding the predicted <italic> Glycine max </italic> DNA topoisomerase 1-like. <span class="gene">Glyma17g29050.1</span> might encode the 60S ribosomal protein L30-like predicted by the gene (GenBank No. XM_003552594). However, <span class="gene">Glyma17g29040.1</span> only shared 82% identity in nucleotide sequence with <italic> Glycine max </italic> clone <span class="gene">GM_WBc0124O13</span> (GenBank No. AC236224). There was no similar level of correspondence among the proteins in the EMBL-EBI database. We suggested that <span class="gene">Glyma17g28950.1</span> and <span class="gene">Glyma17g28970.1,</span> encoding plant-type Ser/Thr protein kinases, were the candidate genes of <italic> Rps10 </italic> . The two candidate genes, <italic> Rps10 </italic> - <italic> 1 </italic> and <italic> Rps10 </italic> - <italic> 2 </italic> , about 14 kb apart, were identified at this region. Using the InterProScan program analysis, their predicted domain structure was: 1) a hydrophobic N-terminal putative signal peptide, and 2) a C-terminal Ser/Thr protein kinase catalytic domain. Additionally, Rps10-1 contained a putative transmembrane domain followed by a membrane transfer-stop signal. These analyses indicated that Rps10-1 and Rps10-2 might be the Ser/Thr protein kinases. The difference in structures between the Ser/Thr genes representing <italic> Rps10 </italic> and the NBS-LRR genes <italic> Rps1 </italic> k, <italic> Rps2 </italic> , and <italic> Rps4 </italic> indicated that the mechanism of <span class="trait">resistance</span> in the soybean- <italic> P </italic> . <italic> sojae </italic> interaction was likely to be different.
					</p>
				
					<p>&nbsp;&nbsp;
						Some <span class="trait">resistance</span> genes belonging to the <span class="gene">RLK</span> families have been characterized as an intracellular Ser/Thr protein kinase, such as <italic> Pto </italic> , <italic> Xa21 </italic> , <italic> Xa26 </italic> , <italic> ZmPto </italic> and <italic> NgRLK1 </italic> <xref ref-type="bibr" rid="pone.0069799-Kim2"> [38] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0069799-Zou1"> [42] </xref> . Based on the structure of the putative extracellular domain, the plant RLKs were divided into seven classes: the S-locus glycoprotein like (SLG-like), Leucine-Rich Repeats (LRRs), epidermal growth factor-like (EGF-like) repeats, putative carbohydrate-binding lectin (Lectin-like), the tumor necrosis factor receptor (TNFR), pathogenesis-related (PR5) and N-glycosylation site type of RLKs <xref ref-type="bibr" rid="pone.0069799-Satterlee1"> [43] </xref> . Interestingly, the N-terminal 116 aa of predicted Rps10-1, and 302 aa of predicted Rps10-2 have no similarities to the extracellular regions of the known RLKs. The kinase domain of Rps10-1 and Rps10-2 showed 22&#x02013;41% sequence similarity to other plant RLKs (data not shown). In addition, Rps10-1 and Rps10-2 showed a low level of sequence identity (4&#x02013;5%) with the calcium-dependent protein kinase SK5-like proteins in soybean (data not shown). The above analyses suggested that <italic> Rps10-1 </italic> and <italic> Rps10-2 </italic> might encode a novel type of plant RLK.
					</p>
				
					<p>&nbsp;&nbsp;
						RLKs are unusual membrane-associated plant protein kinases, some of which have important roles in pathogen <span class="trait">resistance.</span> For example, the Ser/Thr protein kinase Pto, which confers <span class="trait">resistance</span> to <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> , is autophosphorylated at the Thr38 and Ser198 sites after monitoring the signal of the interaction between effector AvrPto and tomato Prf. Prf is further phosphorylated by another Ser/Thr kinase <span class="gene">Pti1</span> to induce a series of mitogen-activated protein kinase cascades, resulting in a hypersensitive response (HR) <xref ref-type="bibr" rid="pone.0069799-Salmero1"> [44] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0069799-Zhou2"> [47] </xref> . The rice Xa21 kinase domain interacts with the protein XB3 via its ankyrin repeat domain and trans-phosphorylation occurs. The physical interaction with XB3 is thought to stabilize the Xa21 protein, thereby maintaining R protein levels and hence the capacity to fully activate the defense response.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification of Chalcone Reductase Gene Family in Soybean: Insight into Root-Specific <span class="gene">GmCHRs</span> and Phytophthora sojae <span class="trait">Resistance.</span></h2>
			<h3>PMID:29270182</h3>
				

				
			<p>
				作者：Sepiol CJ(1)(2), Yu J(1), Dhaubhadel S(1)(2).
				<br/> 
				年份：2017 Dec 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max [L.] Merr) is one of the main grain legumes worldwide. Soybean farmers lose billions of dollars' worth of <span class="trait">yield</span> annually due to <span class="trait">root</span> and stem <span class="trait">rot</span> disease caused by the oomycete Phytophthora sojae. Many strategies have been developed to combat the disease, however, these methods have proven ineffective in the long term. A more cost effective and durable approach is to select a trait naturally found in soybean that can increase <span class="trait">resistance.</span> One such trait is the increased production of phytoalexin glyceollins in soybean. Glyceollins are isoflavonoids, synthesized via the legume-specific branch of general phenylpropanoid pathway. The first key enzyme exclusively involved in glyceollin synthesis is chalcone reductase (CHR) which coacts with chalcone synthase for the production of isoliquiritigenin, the precursor for glyceollin biosynthesis. Here we report the identification of 14 putative CHR genes in soybean where 11 of them are predicted to be functional. Our results show that <span class="gene">GmCHRs</span> display tissue-specific gene expression, and that only root-specific <span class="gene">GmCHRs</span> are induced upon P. sojae infection. Among 4 root-specific <span class="gene">GmCHRs,</span> <span class="gene">GmCHR2A</span> is located near a QTL that is linked to P. sojae <span class="trait">resistance</span> suggesting <span class="gene">GmCHR2A</span> as a novel locus for partial <span class="trait">resistance</span> that can be utilized for <span class="trait">resistance</span> breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPGIP3 enhanced <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> <span class="trait">rot</span> diseases in transgenic wheat.</h2>
			<h3>PMID:25487419  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang A(1), Wei X, Rong W, Dang L, Du LP, Qi L, Xu HJ, Shao Y, Zhang Z.
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Take-all (caused by the fungal pathogen Gaeumannomyces graminis var. tritici, Ggt) and common <span class="trait">root</span> <span class="trait">rot</span> (caused by Bipolaris sorokiniana) are devastating <span class="trait">root</span> diseases of wheat (Triticum aestivum L.). Development of resistant wheat cultivars has been a challenge since no resistant wheat accession is available. <span class="gene">GmPGIP3,</span> one member of polygalacturonase-inhibiting protein (PGIP) family in soybean (Glycine max), exhibited inhibition activity against fungal endopolygalacturonases (PGs) in vitro. In this study, the <span class="gene">GmPGIP3</span> transgenic wheat plants were generated and used to assess the effectiveness of <span class="gene">GmPGIP3</span> in protecting wheat from the infection of Ggt and B. sorokiniana. Four independent transgenic lines were identified by genomic PCR, Southern blot, and reverse transcription PCR (RT-PCR). The introduced <span class="gene">GmPGIP3</span> was integrated into the genomes of these transgenic lines and could be expressed. The expressing <span class="gene">GmPGIP3</span> protein in these transgenic wheat lines could inhibit the PGs produced by Ggt and B. sorokiniana. The disease response assessments postinoculation showed that the <span class="gene">GmPGIP3-expressing</span> transgenic wheat lines displayed significantly enhanced <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> <span class="trait">rot</span> diseases caused by the infection of Ggt and B. sorokiniana. These data suggested that <span class="gene">GmPGIP3</span> is an attractive gene resource in improving <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> <span class="trait">rot</span> diseases in wheat.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Arabidopsis nonhost <span class="trait">resistance</span> gene <italic> PSS1 </italic> confers immunity against an oomycete and a fungal pathogen but not a bacterial pathogen that cause diseases in soybean</h2>
			<h3>PMID:22694952</h3>
				
					<h3>PMCID:3507847</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Sumit </surname>  <given-names> Rishi </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> rsumit@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Sahu </surname>  <given-names> Binod B </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sbinod@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Xu </surname>  <given-names> Min </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> xumin@nature.berkeley.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Sandhu </surname>  <given-names> Devinder </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> dsandhu@uwsp.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Bhattacharyya </surname>  <given-names> Madan K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mbhattac@iastate.edu </email>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						NHR acts in two layers against the biotrophic fungal pathogens [ <xref ref-type="bibr" rid="B6"> 6 </xref> , <xref ref-type="bibr" rid="B7"> 7 </xref> ]. The first layer of NHR suppresses the invasion by non adapted pathogens at the pre-haustorial level. Three NHR genes, <italic> PEN1, PEN2 </italic> and <italic> PEN3 </italic> , required for penetration <span class="trait">resistance</span> of Arabidopsis against the non adapted barley biotrophic fungal pathogen, <italic> Blumeria graminis </italic> f. sp. <italic> hordei </italic> have been isolated [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ]. These genes act at the prehaustorial stage of the pathogen invasion [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. <italic> PEN1 </italic> encodes a soluble N-ethylmalemide sensitive attached receptor (SNARE) protein, which is involved in vesicle fusion and exocytosis of toxic compounds to the pathogen infection sites [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. <italic> PEN2 </italic> encodes a glycosyl hydrolase, which has been localized to the peroxisomes [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. <italic> PEN3 </italic> encodes an ATP-binding cassette (ABC) protein of the plasma membrane [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. Cytological studies have demonstrated that <italic> PEN2 </italic> and <italic> PEN3 </italic> work together to generate and transport toxic chemicals into the infection sites [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. The first layer of NHR prevents the biotrophic fungal pathogens from penetration and development of feeding structures, haustoria. Fungal pathogens that overcome the first layer of NHR encounter a post-haustorial defense mechanism. Some of the genes involved in the second layer of NHR in Arabidopsis are <italic> EDS1, <span class="gene">PAD4</span> </italic> and <italic> <span class="gene">SAG101</span> </italic> that are involved in plant defenses [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. Downstream antagonistic defense pathways regulated by salicylic acid (SA) and the jasmonic acid (JA) are activated upon infection with biotrophic and necrotrophic pathogens, respectively [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. SA and JA pathways are shown to be involved in the expression of nonhost <span class="trait">resistance</span> against the cowpea <span class="trait">rust,</span> <italic> Uromyces vignae </italic> , in Arabidopsis [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. Similarly, studies of mutants lacking <italic> PEN1, 2, </italic> and <italic> 3 </italic> established that SA and JA pathways are also involved in the expression of nonhost <span class="trait">resistance</span> in Arabidopsis against the soybean pathogen <italic> Phakopsora pachyrhizi </italic> that causes the Asian soybean <span class="trait">rust</span> [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						It has been shown that lack of either of a functional pathway, the PEN1/SNAP33/VAMP721/722 or the indole- glucosinolates/metabolites pathway, involving the PEN2/PEN3 activity is sufficient to allow a non-adapted fungal pathogen to enter Arabidopsis mutant plants at a rate similar to that in an adapted host [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. However, a complete loss of the subsequent post-invasion <span class="trait">resistance</span> mechanism encoded by plant defense genes <italic> <span class="gene">PAD4</span> </italic> and <italic> <span class="gene">SAG101</span> </italic> is necessary for a nonhost plant species to become a host for such non-adapted fungal pathogens [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]. In light of the critical role of the post-invasion genes as determinants of the nonhost status of Arabidopsis against non-adapted fungal pathogens, <italic> PSS1 </italic> &#x02019;s role at both pre- and post-haustorial levels in conferring NHR of Arabidopsis against <italic> P. sojae </italic> is novel.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping of a <italic> Phytophthora </italic> <span class="trait">-resistance</span> gene <italic> RpsWY </italic> in soybean ( <italic> Glycine max </italic> L.) by high-throughput genome-wide sequencing</h2>
			<h3>PMID:28246754</h3>
				

				
					<h3>PMCID:5395582</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cheng </surname>  <given-names> Yanbo </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ma </surname>  <given-names> Qibin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ren </surname>  <given-names> Hailong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Qiuju </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Enliang </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tan </surname>  <given-names> Zhiyuan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shuxian </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Gengyun </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <address>  <phone> 86-20-85288024 </phone>  <email> hnian@scau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> The State Key Laboratory for Conservation and Utilization of Subtropical Agro-bioresources,  </institution>  <institution> South China Agricultural University,  </institution>  </institution-wrap> Guangzhou, 510642 Guangdong People&#x02019;s Republic of China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> The Key Laboratory of Plant Molecular Breeding of Guangdong Province, College of Agriculture,  </institution>  <institution> South China Agricultural University,  </institution>  </institution-wrap> Guangzhou, 510642 Guangdong People&#x02019;s Republic of China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2034 1839 </institution-id>  <institution-id institution-id-type="GRID"> grid.21155.32 </institution-id>  <institution/>  <institution> Beijing Genomics Institute (BGI)-Shenzhen,  </institution>  </institution-wrap> Shenzhen, 518086 People&#x02019;s Republic of China  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0478 6311 </institution-id>  <institution-id institution-id-type="GRID"> grid.417548.b </institution-id>  <institution> Agricultural Research Service, Crop Genetics Research Unit,  </institution>  <institution> United States Department of Agriculture,  </institution>  </institution-wrap> Stoneville, MS 38776 USA  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Using a combination of phenotypic screening, genetic and statistical analyses, and high-throughput genome-wide sequencing, we have finely mapped a dominant Phytophthora <span class="trait">resistance</span> gene in soybean cultivar Wayao. Phytophthora <span class="trait">root</span> <span class="trait">rot</span> (PRR) caused by Phytophthora sojae is one of the most important soil-borne diseases in many soybean-production regions in the world. Identification of resistant gene(s) and incorporating them into elite varieties are an effective way for breeding to prevent soybean from being harmed by this disease. Two soybean populations of 191 F2 individuals and 196 F7:8 recombinant inbred lines (RILs) were developed to map Rps gene by crossing a susceptible cultivar Huachun 2 with the resistant cultivar Wayao. Genetic analysis of the F2 population indicated that PRR <span class="trait">resistance</span> in Wayao was controlled by a single dominant gene, temporarily named RpsWY, which was mapped on chromosome 3. A high-density genetic linkage bin map was constructed using 3469 recombination bins of the RILs to explore the candidate genes by the high-throughput genome-wide sequencing. The results of genotypic analysis showed that the RpsWY gene was located in bin 401 between 4466230 and 4502773 bp on chromosome 3 through line 71 and 100 of the RILs. Four predicted genes <span class="gene">(Glyma03g04350,</span> <span class="gene">Glyma03g04360,</span> <span class="gene">Glyma03g04370,</span> and <span class="gene">Glyma03g04380)</span> were found at the narrowed region of 36.5 kb in bin 401. These results suggest that the high-throughput genome-wide resequencing is an effective method to fine map PRR candidate genes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Phytophthora </italic> <span class="trait">root</span> <span class="trait">rot</span> (PRR) caused by <italic> Phytophthora sojae </italic> is one of the most important soil-borne diseases in many soybean-production regions in the world. Identification of resistant gene(s) and incorporating them into elite varieties are an effective way for breeding to prevent soybean from being harmed by this disease. Two soybean populations of 191 F <sub> 2 </sub> individuals and 196 F <sub> 7:8 </sub> recombinant inbred lines (RILs) were developed to map <italic> Rps </italic> gene by crossing a susceptible cultivar Huachun 2 with the resistant cultivar Wayao. Genetic analysis of the F <sub> 2 </sub> population indicated that PRR <span class="trait">resistance</span> in Wayao was controlled by a single dominant gene, temporarily named <italic> RpsWY </italic> , which was mapped on chromosome 3. A high-density genetic linkage bin map was constructed using 3469 recombination bins of the RILs to explore the candidate genes by the high-throughput genome-wide sequencing. The results of genotypic analysis showed that the <italic> RpsWY </italic> gene was located in bin 401 between 4466230 and 4502773&#x000a0;bp on chromosome 3 through line 71 and 100 of the RILs. Four predicted genes ( <italic> <span class="gene">Glyma03g04350,</span> <span class="gene">Glyma03g04360,</span> <span class="gene">Glyma03g04370</span> </italic> , and <italic> <span class="gene">Glyma03g04380</span> </italic> ) were found at the narrowed region of 36.5&#x000a0;kb in bin 401. These results suggest that the high-throughput genome-wide resequencing is an effective method to fine map PRR candidate genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">resistance</span> to <italic> P. sojae </italic> is controlled by two mechanisms, partial or complete <span class="trait">resistance</span> genes (Sugimoto et al. <xref ref-type="bibr" rid="CR54"> 2012 </xref> ). The mechanism of partial <span class="trait">resistance</span> involves multiple genes and limits damage to the plant (Schmitthenner <xref ref-type="bibr" rid="CR47"> 1985 </xref> ; Dorrance et al. <xref ref-type="bibr" rid="CR12"> 2003 </xref> ). The other mechanism is related to a single dominant gene <span class="trait">resistance</span> (Hartwig et al. <xref ref-type="bibr" rid="CR19"> 1968 </xref> ; Burnham et al. <xref ref-type="bibr" rid="CR5"> 2003a </xref> ; <span class="gene">Fan</span> et al. <xref ref-type="bibr" rid="CR14"> 2009 </xref> ; Yao et al. <xref ref-type="bibr" rid="CR75"> 2010 </xref> ; Yu et al. <xref ref-type="bibr" rid="CR76"> 2010 </xref> ; Sugimoto et al. <xref ref-type="bibr" rid="CR53"> 2011 </xref> ; Sun et al. <xref ref-type="bibr" rid="CR56"> 2011 </xref> , <xref ref-type="bibr" rid="CR57"> 2014 </xref> ; Wu et al. <xref ref-type="bibr" rid="CR72"> 2011b </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR81"> 2013b </xref> ), in which <italic> P. sojae </italic> interacts with <italic> Rps </italic> genes in a gene-for-gene system preventing disease development in plants (Schmitthenner <xref ref-type="bibr" rid="CR48"> 1999 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, <italic> RpsWY </italic> is flanked by Satt631 (2.1&#x000a0;cM) and Satt152 (3.9&#x000a0;cM), Satt009 (5.0&#x000a0;cM), Sat_084 (12.7&#x000a0;cM) on chromosome 3 using SSR marker analysis, and mapped to <italic> Rps </italic> gene rich regions on soybean chromosome 3. The results of SNP markers analysis indicated that <italic> RpsWY </italic> gene is located within the interval from 4,466,230 to 4,502,773&#x000a0;bp (36.5&#x000a0;kb) on chromosome 3 falling in the area of the <italic> Rps1k </italic> interval from 4,457,810 to 4,641,921&#x000a0;bp (Gao and Bhattacharyya <xref ref-type="bibr" rid="CR15"> 2008 </xref> ). Twelve known <italic> Rps </italic> genes were identified and mapped to chromosome 3 before <italic> RpsWY </italic> , including five alleles of <italic> Rps1 </italic> (Bernard et al. <xref ref-type="bibr" rid="CR3"> 1957 </xref> ; Mueller et al. <xref ref-type="bibr" rid="CR39"> 1978 </xref> ; Anderson and Buzzell <xref ref-type="bibr" rid="CR2"> 1992 </xref> ), <italic> Rps7 </italic> (Weng et al. <xref ref-type="bibr" rid="CR68"> 2001 </xref> ), <italic> Rps9 </italic> (Wu et al. <xref ref-type="bibr" rid="CR71"> 2011a </xref> ), <italic> RpsYu25 </italic> (Sun et al. <xref ref-type="bibr" rid="CR56"> 2011 </xref> ), <italic> RpsYD25 </italic> (Fan et al. <xref ref-type="bibr" rid="CR14"> 2009 </xref> ), <italic> RpsYD29 </italic> (Zhang et al. <xref ref-type="bibr" rid="CR80"> 2013a </xref> ), <italic> RpsUN1 </italic> (Lin et al. <xref ref-type="bibr" rid="CR35"> 2013 </xref> ), a <italic> Rps </italic> gene in Waseshiroge (Sugimoto et al. <xref ref-type="bibr" rid="CR53"> 2011 </xref> ) and a <italic> Rps </italic> gene in soybean E00003 within the <italic> Rps1k </italic> interval (Zhang et al. <xref ref-type="bibr" rid="CR83"> 2014 </xref> ). <italic> RpsWY </italic> is a distinct gene from the <italic> Rps1 </italic> alleles, because five differentials carrying <italic> Rps1 </italic> ( <italic> 1a, 1b, 1c, 1d </italic> , and <italic> 1k </italic> ) were PRR-susceptible to <italic> P. sojae </italic> Pm14 in our current study. The position information below SSR marker Satt152 (Fig.S5) suggests that <italic> RpsWY </italic> is distinct from <italic> RpsUN1, RpsYD29 </italic> and <italic> Rps7 </italic> (Weng et al. <xref ref-type="bibr" rid="CR68"> 2001 </xref> ; Lin et al. <xref ref-type="bibr" rid="CR35"> 2013 </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR80"> 2013a </xref> ). Otherwise, <italic> Rps9 </italic> was flanked by Satt152 (10.1&#x000a0;cM) and Satt631 (7.5&#x000a0;cM) (Fig.S5) (Wu et al. <xref ref-type="bibr" rid="CR71"> 2011a </xref> ). <italic> RpsYu25 </italic> is located between 3,338,620 and 3,465,436&#x000a0;bp by converting the flanking markers of Satt152 and Sat_186 from the interval 30.1 to 32.8&#x000a0;cM (Fig.S5) (Song et al. <xref ref-type="bibr" rid="CR51"> 2010 </xref> ; Sun et al. <xref ref-type="bibr" rid="CR56"> 2011 </xref> ). The locus analysis of SSR markers and sequence interval indicates that <italic> RpsWY </italic> is a distinct gene from <italic> Rps9 </italic> and <italic> RpsYu25 </italic> . In addition, the <italic> Rps </italic> gene in Waseshiroge is located between Satt009 and T003044871 and may reside in the nucleotide region between 3,919,203 and 4,486,048 (567&#x000a0;kb) on chromosome 3 (Sugimoto et al. <xref ref-type="bibr" rid="CR53"> 2011 </xref> ). The <italic> Rps </italic> gene in E00003 was located within interval 4,475,877&#x02013;4,563,799&#x000a0;bp (87.9&#x000a0;kb) on chromosome 3 (Zhang et al. <xref ref-type="bibr" rid="CR83"> 2014 </xref> ). <italic> RpsWY </italic> and the <italic> Rps </italic> gene from Waseshiroge or E00003 may be allelic, but not <italic> Rps1k </italic> . Therefore, <italic> RpsWY </italic> could be a new allele locus or a new gene. To confirm this finding, additional crosses between Wayao and Waseshiroge or E00003 should be made and genetically analyzed. In previous studies, more than 20 <italic> Rps </italic> genes/alleles have been mapped and designated to ten loci on four soybean chromosomes (13, 18, 16, and 3) since <italic> Rps1 </italic> was identified for the first <italic> P. sojae </italic> <span class="trait">resistance</span> genes (Lin et al. <xref ref-type="bibr" rid="CR35"> 2013 </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR83"> 2014 </xref> ). However, none of them was finely mapped by using low density genetic maps. <italic> Rps1 </italic> was one of the <italic> Rps </italic> genes which has been mapped to chromosome 3 of the USDA-ARS soybean Williams isolines (containing the genes <italic> <span class="gene">Rj2</span> Rmd-c Rps2 <span class="gene">Ti-a</span> </italic> ) approximately 2&#x000a0;cM from locus A071-1 by the restriction fragment length polymorphism (Polzin et al. <xref ref-type="bibr" rid="CR42"> 1994 </xref> ). Two <italic> Rps1 </italic> -k genes were predicted by the sequence analysis using a shotgun sequencing strategy applied in sequencing the BAC contig (Gao et al. <xref ref-type="bibr" rid="CR16"> 2005 </xref> ). However, more <italic> Rps </italic> genes were mapped on different chromosomes using F <sub> 2 </sub> , F <sub> 2:3 </sub> and/or RIL populations by SSR markers (Demirbas et al. <xref ref-type="bibr" rid="CR11"> 2001 </xref> ; Weng et al. <xref ref-type="bibr" rid="CR68"> 2001 </xref> ; <span class="gene">Fan</span> et al. <xref ref-type="bibr" rid="CR14"> 2009 </xref> ; Sugimoto et al. <xref ref-type="bibr" rid="CR53"> 2011 </xref> ; Sun et al. <xref ref-type="bibr" rid="CR56"> 2011 </xref> ; Wu et al. <xref ref-type="bibr" rid="CR71"> 2011a </xref> ; Lin et al. <xref ref-type="bibr" rid="CR35"> 2013 </xref> ; Zhang et al. <xref ref-type="bibr" rid="CR81"> 2013b </xref> , <xref ref-type="bibr" rid="CR82"> c </xref> , <xref ref-type="bibr" rid="CR83"> 2014 </xref> ). In our study, <italic> RpsWY </italic> was mapped on chromosome 3 with 191 F <sub> 2 </sub> individuals, which was located in the <italic> Rps </italic> -rich region, by genetic and physical analysis (Fig. S5). The candidate gene of <italic> RpsWY </italic> was further investigated using the 196 RIL population. Fortunately, we found that <italic> RpsWY </italic> was located within interval 4,466,230&#x02013;4,502,773&#x000a0;bp (36.5&#x000a0;kb) on chromosome 3 covering the sites of four potential candidate genes of <italic> RpsWY </italic> . Analyzing the overlapping region of line 71 and line 100 indicates that the discovery of recombinant monoclonal line 71 and line 100 was crucial for the fine mapping of the candidate genes (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping and candidate gene analysis of two loci conferring <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:27591777  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li L(1)(2), Lin F(2)(3), Wang W(2), Ping J(2)(4), Fitzgerald JC(2), Zhao M(2)(5), Li S(1), Sun L(2), Cai C(6), Ma J(7).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: RpsUN1 and RpsUN2 were fine mapped to two genomic regions harboring disease resistance-like genes. The haplotypes and instability of the regions and candidate genes for the two <span class="trait">resistance</span> loci were characterized. Phytophthora <span class="trait">root</span> and stem <span class="trait">rot</span> caused by Phytophthora sojae, is one of the most destructive diseases of soybean. Deploying soybean cultivars carrying race-specific <span class="trait">resistance</span> conferred by Rps genes is the most practical approach to managing this disease. Previously, two Rps genes, RpsUN1 and RpsUN2 were identified in a landrace PI 567139B and mapped to a 6.5 cM region on chromosome 3 and a 3.0 cM region on chromosome 16, corresponding to 1387 and 423 kb of the soybean reference genome sequences. By analyzing recombinants defined by genotypic and phenotypic screening of the 826 F2:3 families derived from two reciprocal crosses between the two parental lines, RpsUN1 and RpsUN2, were further narrowed to a 151 kb region that harbors five genes including three disease <span class="trait">resistance</span> (R)-like genes, and a 36 kb region that contains four genes including five R-like genes, respectively, according to the reference genome. Expressional changes of these nine genes before and after inoculation with the pathogen, as revealed by RNA-seq, suggest that <span class="gene">Glyma.03g034600</span> in the RpsUN1 region and <span class="gene">Glyma.16g215200</span> and <span class="gene">Glyma.16g214900</span> in the RpsUN2 region of PI 567139B may be associated with the <span class="trait">resistance</span> to P. sojae. It is also suggested that unequal recombination between/among R-like genes may have occurred, resulting in the formation of two recombinants with inconsistent genotypic and phenotypic observations. The haplotype variation of genomic regions where RpsUN1 and RpsUN2 reside in the entire soybean germplasm deposited in the US soybean germplasm collection suggests that RpsUN1 and RpsUN2 are most likely novel genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association of Putative Fungal Effectors in Fusarium oxysporum with Wilt Symptoms in Soybean.</h2>
			<h3>PMID:27146104  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ellis ML(1), Lanubile A(1), Garcia C(1), Munkvold GP(1).
				<br/> 
				年份：2016 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fungi within the Fusarium oxysporum species complex can cause <span class="trait">root</span> <span class="trait">rot,</span> seedling blight, and wilt of soybean. Isolates recovered from soybean vary in aggressiveness and also the type of symptoms they produce. The aim of this study was to identify genetic markers to detect aggressive soybean wilt isolates. Eighty isolates collected primarily from soybean were tested in the greenhouse for their ability to produce wilt symptoms using susceptible 'Jack' soybean. The same 80 isolates were assessed for the presence of fungal effector genes Fmk1, Fow1, Pda1, PelA, PelD, Pep1, Prt1, Rho1, Sge1, Six1, Six6, and Snf1. All polymerase chain reaction amplicons were sequenced, phylogenies were inferred, and analysis of molecular variance (AMOVA) was performed for 10 of the 12 genes. High incidence of vascular discoloration of roots or stems was observed with 3 isolates, while moderate to low levels of incidence were observed for 25 isolates. Fungal effector genes Fmk1, Fow1, PelA, Rho1, Sge1, and <span class="gene">Snf1</span> were present in all isolates screened, while Pda1, PelD, Pep1, Prt1, Six1, and <span class="gene">Six6</span> were dispersed among isolates. The Bayesian and AMOVA analyses found that the genes Fmk1, Fow1, Pda1, PelA, Rho1, Sge1, and <span class="gene">Snf1</span> corresponded to previously designated clades based on tef1α and mitochondrial small subunit sequences. None of the genes had a significant association with wilt symptoms on soybean. Interestingly, the <span class="gene">Six6</span> gene was only present in three previously known wilt isolates from soybean, common bean, and tomato; of these, the soybean and common bean isolates produced high levels of vascular wilt in our study.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loci and candidate gene identification for <span class="trait">resistance</span> to Phytophthora sojae via association analysis in soybean [Glycine max (L.) Merr].</h2>
			<h3>PMID:26758588  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li L(1), Guo N(1), Niu J(1), Wang Z(1), Cui X(1), Sun J(1), Zhao T(2), Xing H(3).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora sojae is an oomycete soil-borne plant pathogen that causes the serious disease Phytophthora <span class="trait">root</span> <span class="trait">rot</span> in soybean, leading to great loss of soybean production every year. Understanding the genetic basis of this plant-pathogen interaction is important to improve soybean disease <span class="trait">resistance.</span> To discover genes or QTLs underlying naturally occurring variations in soybean P.sojae <span class="trait">resistance,</span> we performed a genome-wide association study using 59,845 single-nucleotide polymorphisms identified from re-sequencing of 279 accessions from Yangtze-Huai soybean breeding germplasm. We used two models for association analysis. The same strong peak was detected by both two models on chromosome 13. Within the 500-kb flanking regions, three candidate genes <span class="gene">(Glyma13g32980,</span> <span class="gene">Glyma13g33900,</span> <span class="gene">Glyma13g33512)</span> had SNPs in their exon regions. Four other genes were located in this region, two of which contained a leucine-rich repeat domain, which is an important characteristic of R genes in plants. These candidate genes could be potentially useful for improving the <span class="trait">resistance</span> of cultivated soybean to P.sojae in future soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis and fine mapping of RpsJS, a novel <span class="trait">resistance</span> gene to Phytophthora sojae in soybean [Glycine max (L.) Merr].</h2>
			<h3>PMID:24419901  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun J(1), Li L, Zhao J, Huang J, Yan Q, Xing H, Guo N.
				<br/> 
				年份：2014 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: We finely map a novel <span class="trait">resistance</span> gene ( RpsJS ) to Phytophthora sojae in soybean. RpsJS was mapped in 138.9 kb region, including three NBS-LRR type predicted genes, on chromosome 18. Phytophthora <span class="trait">root</span> <span class="trait">rot</span> (PRR) caused by Phytophthora sojae (P. sojae) has been reported in most soybean-growing regions throughout the world. Development of PRR <span class="trait">resistance</span> varieties is the most economical and environmentally safe method for controlling this disease. Chinese soybean line Nannong 10-1 is resistant to many P. sojae isolates, and shows different reaction types to P. sojae isolates as compared with those with known Rps <span class="trait">(Resistance</span> to P. sojae) genes, which suggests that the line may carry novel Rps genes or alleles. A mapping population of 231 F(2) individuals from the cross of Nannong 10-1 (Resistant, R) and 06-070583 (Susceptible, S) was used to map the Rps gene. The segregation fits a ratio of 3R:1S within F(2) plants, indicating that <span class="trait">resistance</span> in Nannong 10-1 is controlled by a single dominant gene (designated as RpsJS). The results showed that RpsJS was mapped on soybean chromosome 18 (molecular linkage group G, MLG G) flanked by SSR (simple repeat sequences) markers BARCSOYSSR_18_1859 and SSRG60752K at a distance of 0.9 and 0.4 cm, respectively. Among the 14 genes annotated in this 138.9 kb region between the two markers, three genes <span class="gene">(Glyma18g51930,</span> <span class="gene">Glyma18g51950</span> and <span class="gene">Glyma18g51960)</span> are the nucleotide-binding site and a leucine-rich repeat (NBS-LRR) type gene, which may be involved in recognizing the presence of pathogens and ultimately conferring <span class="trait">resistance.</span> Based on marker-assisted <span class="trait">resistance</span> spectrum analyses of RpsJS and the mapping results, we inferred that RpsJS was a novel gene or a new allele at the Rps4, Rps5 or Rps6 loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean-Phytophthora <span class="trait">resistance</span> locus Rps1-k encompasses coiled coil-nucleotide binding-leucine rich repeat-like genes and repetitive sequences.</h2>
			<h3>PMID:18366691  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao H(1), Bhattacharyya MK.
				<br/> 
				年份：2008 Mar 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: A series of Rps <span class="trait">(resistance</span> to Pytophthora sojae) genes have been protecting soybean from the <span class="trait">root</span> and stem <span class="trait">rot</span> disease caused by the Oomycete pathogen, Phytophthora sojae. Five Rps genes were mapped to the Rps1 locus located near the 28 cM map position on molecular linkage group N of the composite genetic soybean map. Among these five genes, Rps1-k was introgressed from the cultivar, Kingwa. Rps1-k has been providing stable and broad-spectrum Phytophthora <span class="trait">resistance</span> in the major soybean-producing regions of the United States. Rps1-k has been mapped and isolated. More than one functional Rps1-k gene was identified from the Rps1-k locus. The clustering feature at the Rps1-k locus might have facilitated the expansion of Rps1-k gene numbers and the generation of new recognition specificities. The Rps1-k region was sequenced to understand the possible evolutionary steps that shaped the generation of Phytophthora <span class="trait">resistance</span> genes in soybean. RESULTS: Here the analyses of sequences of three overlapping BAC clones containing the 184,111 bp Rps1-k region are reported. A shotgun sequencing strategy was applied in sequencing the BAC contig. Sequence analysis predicted a few full-length genes including two Rps1-k genes, <span class="gene">Rps1-k-1</span> and Rps1-k-2. Previously reported Rps1-k-3 from this genomic region 1 was evolved through intramolecular recombination between <span class="gene">Rps1-k-1</span> and <span class="gene">Rps1-k-2</span> in Escherichia coli. The majority of the predicted genes are truncated and therefore most likely they are nonfunctional. A member of a highly abundant retroelement, SIRE1, was identified from the Rps1-k region. The Rps1-k region is primarily composed of repetitive sequences. Sixteen simple repeat and 63 tandem repeat sequences were identified from the locus. CONCLUSION: These data indicate that the Rps1 locus is located in a gene-poor region. The abundance of repetitive sequences in the Rps1-k region suggested that the location of this locus is in or near a heterochromatic region. Poor recombination frequencies combined with presence of two functional Rps genes at this locus has been providing stable Phytophthora <span class="trait">resistance</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phytophthora sojae avirulence genes <span class="gene">Avr4</span> and Avr6 are located in a 24kb, recombination-rich region of genomic DNA.</h2>
			<h3>PMID:14643260  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Whisson SC(1), Basnayake S, Maclean DJ, Irwin JA, Drenth A.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cross between two different races (race 7xrace 25) of the soybean <span class="trait">root</span> and stem <span class="trait">rot</span> pathogen Phytophthora sojae was analyzed to characterize the genomic region flanking two cosegregating avirulence genes, <span class="gene">Avr4</span> and Avr6. Both genes cosegregated in the ratio of 82:17 (avirulent:virulent) in an F(2) population, suggestive of a single locus controlling both phenotypes. A chromosome walk was commenced from RAPD marker OPE7.1C, 2.0cM distant from the Avr4/6 locus. Three overlapping cosmids were isolated which included genetic markers that flank the Avr4/6 locus. The chromosome walk spanned a physical distance of 67kb which represented a genetic map distance of 22.3cM, an average recombination frequency of 3.0kb/cM and 11.7-fold greater than the predicted average recombination frequency of 35.3kb/cM for the entire P. sojae genome. Six genes (cDNA clones) expressed from the Avr4/6 genomic region encompassed by the cosmid contig were identified. Single nucleotide polymorphisms and restriction fragment length polymorphisms showed these six genes were closely linked to the Avr4/6 locus. Physical mapping of the cDNA clones within the cosmid contig made it possible to deduce the precise linkage order of the cDNAs. None of the six cDNA clones appear to be candidates for Avr4/6. We conclude that two of these cDNA clones flank a physical region of approximately 24kb and 4.3cM that appears to include the Avr4/6 locus.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotypic and Phenotypic Characterization of Fungi in the Fusarium oxysporum Species Complex from Soybean Roots.</h2>
			<h3>PMID:24983844  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ellis ML, Cruz Jimenez DR, Leandro LF, Munkvold GP.
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isolates in the Fusarium oxysporum species complex (FOSC) from soybean range from nonpathogenic to aggressive pathogens causing seedling damping-off, wilt, and <span class="trait">root</span> <span class="trait">rot.</span> The objective of this research was to characterize the genotype and phenotype of isolates within the FOSC recovered predominantly from soybean roots and seedlings. Sequence analyses of the translation elongation factor (tef1α) gene and the mitochondrial small subunit (mtSSU), polymerase chain reaction restriction fragment length polymorphism (PCR-RFLP) analysis of the intergenic spacer (IGS) region, and identification of the mating type loci were conducted for 170 isolates. Vegetative compatibility (VC) tests were conducted for 114 isolates. Isolate aggressiveness was tested using a rolled towel assay for 159 isolates. Phylogenetic analysis of the tef1α and mtSSU and PCR-RFLP analysis of the IGS region separated the FOSC isolates into five clades, including F. commune. Both mating type loci, <span class="gene">MAT1-1</span> or MAT1-2, were present in isolates from all clades. The VC tests were not informative, because most VC groups consisted of a single isolate. Isolate aggressiveness varied within and among clades; isolates in clade 2 were significantly less aggressive (P &lt; 0.0001) when compared with isolates from the other clades and F. commune. The results from this study demonstrate the high levels of genotypic and phenotypic diversity within the FOSC from soybean but further work is needed to identify characteristics associated with pathogenic capabilities.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and mapping of quantitative trait loci (QTL) conferring <span class="trait">resistance</span> to <italic> Fusarium graminearum </italic> from soybean PI 567301B</h2>
			<h3>PMID:25690715</h3>
				

				
					<h3>PMCID:4544499</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Acharya </surname>  <given-names> Bhupendra </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Lee </surname>  <given-names> Sungwoo </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rouf Mian </surname>  <given-names> M. A. </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jun </surname>  <given-names> Tae-Hwan </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McHale </surname>  <given-names> Leah K. </given-names>  </name>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Michel </surname>  <given-names> Andrew P. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Dorrance </surname>  <given-names> Anne E. </given-names>  </name>  <address>  <email> dorrance.1@osu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Plant Pathology, The Ohio State University, 1680 Madison Avenue, Wooster, OH 44691 USA  </aff>  <aff id="Aff2">  <label/> Department of Entomology, The Ohio State University, 1680 Madison Avenue, Wooster, OH 44691 USA  </aff>  <aff id="Aff3">  <label/> Department of Crop Science, North Carolina State University, 3127 Ligon Street, Raleigh, NC 27607 USA  </aff>  <aff id="Aff4">  <label/> USDA-ARS and Department of Horticulture and Crop Science, The Ohio State University, 1680 Madison Avenue, Wooster, OH 44691 USA  </aff>  <aff id="Aff5">  <label/> USDA-ARS, Soybean Nitrogen Fixation Unit, Raleigh, NC 27606 USA  </aff>  <aff id="Aff6">  <label/> Department of Plant Bioscience, Pusan National University, Busan, 609-735 South Korea  </aff>  <aff id="Aff7">  <label/> Department of Horticulture and Crop Science, The Ohio State University, 2021 Coffey Road, Columbus, OH 43210 USA  </aff>
				<br/> 
				年份：<day> 31 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: A major novel QTL was identified in a recombinant inbred line population derived from a cross of 'Wyandot' × PI 567301B for Fusarium graminearum, a seed and seedling pathogen of soybean. Fusarium graminearum is now recognized as a primary pathogen of soybean, causing <span class="trait">root,</span> seed <span class="trait">rot</span> and seedling damping-off in North America. In a preliminary screen, 'Wyandot' and PI 567301B were identified with medium and high levels of partial <span class="trait">resistance</span> to F. graminearum, respectively. The objective of this study was to characterise <span class="trait">resistance</span> towards F. graminearum using 184 recombinant inbred lines (RILs) derived from a cross of 'Wyandot' × PI 567301B. The parents and the RILs of the mapping population were evaluated for <span class="trait">resistance</span> towards F. graminearum using the rolled towel assay in a randomized incomplete block design. A genetic map was constructed from 2545 SNP markers and 2 SSR markers by composite interval mapping. One major and one minor QTL were identified on chromosomes 8 and 6, respectively, which explained 38.5 and 8.1 % of the phenotypic variance. The major QTL on chromosome 8 was mapped to a 300 kb size genomic region of the Williams 82 sequence. Annotation of this region indicates that there are 39 genes including the <span class="gene">Rhg4</span> locus for soybean cyst nematode (SCN) <span class="trait">resistance.</span> Based on previous screens, PI 567301B is susceptible to SCN. Fine mapping of this locus will assist in cloning these candidate genes as well as identifying DNA markers flanking the QTL that can be used in marker-assisted breeding to develop cultivars with high levels of <span class="trait">resistance</span> to F. graminearum.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Fusarium graminearum </italic> is now recognized as a primary pathogen of soybean, causing <span class="trait">root,</span> seed <span class="trait">rot</span> and seedling damping-off in North America. In a preliminary screen, &#x02018;Wyandot&#x02019; and PI 567301B were identified with medium and high levels of partial <span class="trait">resistance</span> to <italic> F. graminearum </italic> , respectively. The objective of this study was to characterise <span class="trait">resistance</span> towards <italic> F. graminearum </italic> using 184 recombinant inbred lines (RILs) derived from a cross of &#x02018;Wyandot&#x02019; &#x000d7; PI 567301B. The parents and the RILs of the mapping population were evaluated for <span class="trait">resistance</span> towards <italic> F. graminearum </italic> using the rolled towel assay in a randomized incomplete block design. A genetic map was constructed from 2545 SNP markers and 2 SSR markers by composite interval mapping. One major and one minor QTL were identified on chromosomes 8 and 6, respectively, which explained 38.5 and 8.1&#x000a0;% of the phenotypic variance. The major QTL on chromosome 8 was mapped to a 300&#x000a0;kb size genomic region of the Williams 82 sequence. Annotation of this region indicates that there are 39 genes including the <italic> <span class="gene">Rhg4</span> </italic> locus for soybean cyst nematode (SCN) <span class="trait">resistance.</span> Based on previous screens, PI 567301B is susceptible to SCN. Fine mapping of this locus will assist in cloning these candidate genes as well as identifying DNA markers flanking the QTL that can be used in marker-assisted breeding to develop cultivars with high levels of <span class="trait">resistance</span> <italic> to F. graminearum </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Two QTL conferring <span class="trait">resistance</span> to <italic> F. graminearum </italic> were identified in the Wyandot &#x000d7; PI 567301B population by CIM, one major QTL on chromosome 8 and one minor QTL on chromosome 6 (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). Both QTL were highly significant with respective LOD scores of 23.9 and 6.3 compared to the genome-wide LOD threshold of 3.4 calculated by a 1000-permutation test. The major QTL on chromosome 8 was flanked by markers Sat_157 and ss715602786 and accounted for 38.5&#x000a0;% of the phenotypic variance (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> a). A minor QTL, which explained 8.1&#x000a0;% of the phenotypic variance, was closely linked to marker ss715593768 (Glyma.Wm82.a2, <span class="gene">Gm06</span> at 17672411&#x000a0;bp) on chromosome 6 (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> b). The <span class="trait">resistance</span> alleles for both QTL were contributed by PI 567301B (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). <table-wrap id="Tab2"> <label> Table&#x000a0;2 </label> <caption> <p> Quantitative trait loci conferring <span class="trait">resistance</span> to <italic> Fusarium graminearum </italic> identified via composite interval mapping in the &#x02018;Wyandot&#x02019; &#x000d7; PI 567301B population
					</p>
				
					<p>&nbsp;&nbsp;
						We characterized the <span class="trait">resistance</span> towards <italic> F. graminearum </italic> in PI 567301B and identified a major QTL using a population with over 2,500 molecular markers. The major QTL detected on chromosome 8 is a novel QTL associated with <span class="trait">resistance</span> to <italic> F. graminearum </italic> and, due to the marker density, was narrowed to a 305-kb region on chromosome 8 using a RIL population derived from a cross of Wyandot &#x000d7; PI 567301B. The QTL which contributed a major effect (38.5&#x000a0;% of phenotypic variance) for <span class="trait">resistance,</span> potentially overlaps the <italic> <span class="gene">Rhg4</span> </italic> locus ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> , accessed on July, 2014) located on chromosome 8 in the Soybean Consensus map&#x000a0;4.0 (Ariagada et al. <xref ref-type="bibr" rid="CR4"> 2012 </xref> ; Ferreira et al. <xref ref-type="bibr" rid="CR20"> 2011 </xref> ; Guo et al. <xref ref-type="bibr" rid="CR25"> 2006 </xref> ; Mahalingam and Skorupska <xref ref-type="bibr" rid="CR39"> 1995 </xref> ; Meksem et al. <xref ref-type="bibr" rid="CR42"> 2001 </xref> ; Vuong et al. <xref ref-type="bibr" rid="CR72"> 2011 </xref> ; Webb et al. <xref ref-type="bibr" rid="CR75"> 1995 </xref> ; Wu et al. <xref ref-type="bibr" rid="CR77"> 2009 </xref> ). Significant association of two molecular markers with <span class="trait">resistance</span> to three different isolates of <italic> P. sojae </italic> was also identified in this region on chromosome 8 (Wang et al. <xref ref-type="bibr" rid="CR73"> 2012 </xref> ). Interestingly, one of the minor <span class="trait">resistance</span> QTL for <italic> F. graminearum </italic> from the cultivar Conrad mapped near 102&#x000a0;cM (BARC-051847-11270) on chromosome 8 (Ellis et al. <xref ref-type="bibr" rid="CR19"> 2012 </xref> ), which is approximately, 56&#x000a0;cM (28&#x000a0;Mb) distant from the QTL identified in the present study.
					</p>
				
					<p>&nbsp;&nbsp;
						This QTL also contains a gene putatively encoding a serine hydromethyl transferase (SHMT), which was reported to function in photorespiratory pathway influencing <span class="trait">resistance</span> to biotic and abiotic stresses in Arabidopsis (Moreno et al. <xref ref-type="bibr" rid="CR46"> 2005 </xref> ). A study by Liu et al. ( <xref ref-type="bibr" rid="CR37"> 2012 </xref> ) confirmed that <span class="gene">SHMT</span> is the <italic> <span class="gene">Rhg4</span> </italic> gene using mutation analysis, gene silencing, and transgenic complementation techniques and also cloned the gene from soybean cultivar &#x02018;Forrest&#x02019;. Interestingly, Forrest was found to be susceptible to the same isolate of <italic> F. graminearum </italic> used in this study with a disease severity index of 70.7&#x000a0;% and mean disease score of 3.9 (Acharya <xref ref-type="bibr" rid="CR1"> 2014 </xref> ). PI 567301B is reportedly moderately susceptible and susceptible to SCN races 3 and 4, respectively (Germplasm Resources Information Network, <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov"> www.ars-grin.gov </ext-link> , accessed July 2014). This indicates that while <span class="trait">resistance</span> to <italic> F. graminearum </italic> is potentially linked to <italic> <span class="gene">Rhg4</span> </italic> , the <span class="gene">SHMT</span> allele contributing to SCN <span class="trait">resistance</span> is unlikely to be involved in the <span class="trait">resistance</span> response to <italic> F. graminearum </italic> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Co-Inoculation with Rhizobia and AMF Inhibited Soybean Red Crown <span class="trait">Rot:</span> From Field Study to Plant Defense-Related Gene Expression Analysis</h2>
			<h3>PMID:22442737</h3>
				
					<h3>PMCID:3307780</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Xiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Xing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Man </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Haiyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Ruqian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Jiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shuxian </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 2 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In addition, activation of specific plant defense mechanisms in response to symbiont colonization is also an obvious basis for the protective behavior of rhizobia and AMF <xref ref-type="bibr" rid="pone.0033977-Dar1"> [20] </xref> , <xref ref-type="bibr" rid="pone.0033977-Parnike1"> [43] </xref> , <xref ref-type="bibr" rid="pone.0033977-Arfaoui3"> [44] </xref> . Compounds associated with plant defense included, but not limited to, phytoalexins, chitinases, &#x003b2;-1, 3-glucanases, pathogenesis related proteins, callose and phenolics <xref ref-type="bibr" rid="pone.0033977-GianinazziPearson1"> [17] </xref> , <xref ref-type="bibr" rid="pone.0033977-Ismail1"> [45] </xref> , <xref ref-type="bibr" rid="pone.0033977-Song1"> [46] </xref> . In legumes, a number of genes encoding pathogenesis related proteins have been showed to be locally and systemically up-regulated after inoculation with pathogens <xref ref-type="bibr" rid="pone.0033977-Robert1"> [47] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0033977-Liu1"> [49] </xref> . Furthermore, the increase of some specific plant defense-related (PR) gene expressions to rhizobium and/or AMF colonization has been reported to against fungal pathogens <xref ref-type="bibr" rid="pone.0033977-Cordier1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0033977-Arfaoui2"> [26] </xref> . For example, inoculation with rhizobium could induce the PR genes involved in phytoalexin synthesis and made plants primed for against the fungal pathogen, and then reduce the severity of disease <xref ref-type="bibr" rid="pone.0033977-Arfaoui2"> [26] </xref> . Phenolics were observed in rhizobium treated seedlings against the pathogen infection, which might be controlled by <italic> <span class="gene">PAL</span> </italic> <xref ref-type="bibr" rid="pone.0033977-Mishra1"> [50] </xref> . Mycorrhization also amplified the accumulation of the PR genes and phytoalexins synthesis in fungal pathogen infected <xref ref-type="bibr" rid="pone.0033977-Pozo2"> [51] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0033977-Zhao1"> [53] </xref> . Our results through q-PCR analysis found that inoculation with rhizobia and/or AMF could significantly increase the expression of most selected PR genes in both roots and leaves at 1 and 3 DAI ( <xref ref-type="fig" rid="pone-0033977-g005"> Figure 5 </xref> ), especially under co-inoculation conditions. This indicated that inoculation with rhizobia and/or AMF in our study activated both local and systemic defense systems in a short- and long-term way in soybean plants. The similar elicitation via AMF and rhizobia symbiosis to activate specific defense reactions and predispose an early response when attacked by a <span class="trait">root</span> pathogen has been found in legumes <xref ref-type="bibr" rid="pone.0033977-GianinazziPearson1"> [17] </xref> , <xref ref-type="bibr" rid="pone.0033977-Zhao1"> [53] </xref> . Interestingly, we also found that inoculation with rhizobia up-regulated more PR genes in roots, but AMF up-regulated more in leaves ( <xref ref-type="fig" rid="pone-0033977-g005"> Figure 5 </xref> and <xref ref-type="supplementary-material" rid="pone.0033977.s003"> Figure S3 </xref> ), suggesting that the plant responds to rhizobium and AMF inoculation through different defense systems. Plants prefer to activate local defense systems to rhizobium inoculation, but systemic defense systems to AMF inoculation. Co-inoculation could activate both defense systems and result in better pathogen inhibition.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sequence composition of BAC clones and SSR markers mapped to Upland cotton chromosomes 11 and 21 targeting <span class="trait">resistance</span> to soil-borne pathogens</h2>
			<h3>PMID:26483808</h3>
				
					<h3>PMCID:4591483</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Congli </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/238894/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ulloa </surname>  <given-names> Mauricio </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/276977/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Xinyi </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/277381/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Xiaohui </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/269155/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saski </surname>  <given-names> Christopher </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/240581/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> John Z. </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/240485/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Roberts </surname>  <given-names> Philip A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/202942/overview"/>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Integrating disease <span class="trait">resistance</span> phenotypes into high-yielding, high-fiber quality cultivars is one of the most important objectives in cotton breeding programs (Ulloa et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> ). To further elucidate and expedite the discovery of R and/or PR genes; herein, we provide new DNA sequence information of large genomic segments (e.g., BAC clones) from cv. Acala Maxxa ( <italic> G. hirsutum </italic> L.) for which MUSB-derived single sequence repeat (SSR) markers were previously mapped to chromosomes (Chr) 11 and 21 (Frelichowski et al., <xref rid="B18" ref-type="bibr"> 2006 </xref> ; Ulloa et al., <xref rid="B56" ref-type="bibr"> 2008 </xref> ; Yu et al., <xref rid="B71" ref-type="bibr"> 2012 </xref> ). These markers reportedly underlie QTLs involved in disease <span class="trait">resistance;</span> therefore, capturing and sequencing BAC-sized genomic segments tightly linked to these SSRs will help to resolve local content and genome structure of RKN (Shen et al., <xref rid="B49" ref-type="bibr"> 2006 </xref> ; Wang et al., <xref rid="B62" ref-type="bibr"> 2006 </xref> ; Ynturi et al., <xref rid="B70" ref-type="bibr"> 2006 </xref> ; Ulloa et al., <xref rid="B58" ref-type="bibr"> 2010 </xref> ), <span class="gene">REN</span> (Dighe et al., <xref rid="B13" ref-type="bibr"> 2009 </xref> ; Guti&#x000e9;rrez et al., <xref rid="B26" ref-type="bibr"> 2011 </xref> ); FOV (Ulloa et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> , <xref rid="B55" ref-type="bibr"> 2013 </xref> ), VW (Bolek et al., <xref rid="B5" ref-type="bibr"> 2005 </xref> ; Fang et al., <xref rid="B16" ref-type="bibr"> 2014 </xref> ; Zhao et al., <xref rid="B73" ref-type="bibr"> 2014 </xref> ), and BRR (Niu et al., <xref rid="B36" ref-type="bibr"> 2008 </xref> ) <span class="trait">resistance.</span> The Maxxa BAC clone and marker sequence data were also compared to the whole genome sequence assemblies of the <italic> G. raimondii </italic> D <sub> 5 </sub> and <italic> G. arboreum </italic> A <sub> 2 </sub> ancestral diploid genomes (Paterson et al., <xref rid="B40" ref-type="bibr"> 2012 </xref> ; Wang et al., <xref rid="B64" ref-type="bibr"> 2012b </xref> ; Li et al., <xref rid="B33" ref-type="bibr"> 2014 </xref> ) and domesticated tetraploid TM-1 AD <sub> 1 </sub> ( <italic> G. hirsutum </italic> ) genome which are now publicly available (Li et al., <xref rid="B32" ref-type="bibr"> 2015 </xref> ; Zhang et al., <xref rid="B72" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We targeted all SSR markers previously mapped on Upland cotton Chr 11 and Chr 21 (CMD: <ext-link ext-link-type="uri" xlink:href="http://www.cottonmarker.org/"> http://www.cottonmarker.org/ </ext-link> ), especially those underlying QTLs determining <span class="trait">resistance</span> to RKN (Shen et al., <xref rid="B49" ref-type="bibr"> 2006 </xref> ; Wang et al., <xref rid="B62" ref-type="bibr"> 2006 </xref> , <xref rid="B61" ref-type="bibr"> 2012a </xref> ; Ynturi et al., <xref rid="B70" ref-type="bibr"> 2006 </xref> ; Guti&#x000e9;rrez et al., <xref rid="B25" ref-type="bibr"> 2010 </xref> ; Ulloa et al., <xref rid="B58" ref-type="bibr"> 2010 </xref> ), <span class="gene">REN</span> (Dighe et al., <xref rid="B13" ref-type="bibr"> 2009 </xref> ; Romano et al., <xref rid="B45" ref-type="bibr"> 2009 </xref> ; Guti&#x000e9;rrez et al., <xref rid="B26" ref-type="bibr"> 2011 </xref> ), FOV (Ulloa et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> , <xref rid="B55" ref-type="bibr"> 2013 </xref> ), VW (Bolek et al., <xref rid="B5" ref-type="bibr"> 2005 </xref> ; Fang et al., <xref rid="B16" ref-type="bibr"> 2014 </xref> ), and BRR (Niu et al., <xref rid="B36" ref-type="bibr"> 2008 </xref> ). QTL analyses of marker-resistance associations for RKN, REN, FOV, VW, and BRR on these chromosomes were reported from previous publications (Table <xref ref-type="table" rid="T2"> 2 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Highly Variable Supernumerary Chromosome Segments in an Asexual Pathogen</h2>
			<h3>PMID:27341103</h3>
				

				
					<h3>PMCID:4920403</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Xiaoqiu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Das </surname>  <given-names> Anindya </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sahu </surname>  <given-names> Binod B. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Srivastava </surname>  <given-names> Subodh K. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Leandro </surname>  <given-names> Leonor F. </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> O&#x02019;Donnell </surname>  <given-names> Kerry </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhattacharyya </surname>  <given-names> Madan K. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Supernumerary chromosome segments are known to harbor different transposons from their essential counterparts. The aim of this study was to investigate the role of transposons in the origin and evolution of supernumerary segments in the asexual fungal pathogen Fusarium virguliforme. We compared the genomes of 11 isolates comprising six Fusarium species that cause soybean sudden death syndrome (SDS) or bean <span class="trait">root</span> <span class="trait">rot</span> (BRR), and identified significant levels of genetic variation in A+T-rich repeat blocks of the essential chromosomes and in A+T-neutral regions of the supernumerary segments. The A+T-rich repeat blocks in the essential chromosomes were highly variable between F. virguliforme and non-F. virguliforme isolates, but were scarcely variable between F. virguliforme isolates. The A+T-neutral regions in the supernumerary segments, however, were highly variable between F. virguliforme isolates, with a statistically significant number (21 standard deviations above the mean) of single nucleotide polymorphisms (SNPs). And supernumerary sequence types and rearrangement patterns of some F. virguliforme isolates were present in an isolate of F. cuneirostrum but not in the other F. virguliforme isolates. The most variable and highly expressed region in the supernumerary segments contained an active DNA transposon that was a most conserved match between F. virguliforme and the unrelated fungus Tolypocladium inflatum. This transposon was absent from two of the F. virguliforme isolates. Furthermore, transposons in the supernumerary segments of some F. virguliforme isolates were present in non-F. virguliforme isolates, but were absent from the other F. virguliforme isolates. Two supernumerary <span class="gene">P450</span> enzymes were 43% and 57% identical to their essential counterparts. This study has raised the possibility that transposons generate genetic variation in supernumerary chromosome segments by frequent horizontal transfer within and between closely related species.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Aggressiveness of Fusarium species and impact of <span class="trait">root</span> infection on growth and <span class="trait">yield</span> of soybeans.</h2>
			<h3>PMID:23514263  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Arias MM(1), Leandro LF, Munkvold GP.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium spp. are commonly isolated from soybean roots but the pathogenic activity of most species is poorly documented. Aggressiveness and <span class="trait">yield</span> impact of nine species of Fusarium were determined on soybean in greenhouse (50 isolates) and field microplot (19 isolates) experiments. <span class="trait">Root</span> <span class="trait">rot</span> severity and shoot and <span class="trait">root</span> dry weights were compared at growth stages <span class="gene">V3</span> or R1. <span class="trait">Root</span> systems were scanned and digital image analysis was conducted; <span class="trait">yield</span> was measured in microplots. Disease severity and <span class="trait">root</span> morphology impacts varied among and within species. Fusarium graminearum was highly aggressive <span class="trait">(root</span> <span class="trait">rot</span> severity &gt;90%), followed by F. proliferatum and F. virguliforme. Significant variation in damping-off (20 to 75%) and <span class="trait">root</span> <span class="trait">rot</span> severity (&lt;20 to &gt;60%) was observed among F. oxysporum isolates. In artificially-infested microplots, <span class="trait">root</span> <span class="trait">rot</span> severity was low (&lt;25%) and mean <span class="trait">yield</span> was not significantly reduced. However, there were significant linear relationships between <span class="trait">yield</span> and <span class="trait">root</span> symptoms for some isolates. <span class="trait">Root</span> morphological characteristics were more consistent indicators of <span class="trait">yield</span> loss than <span class="trait">root</span> <span class="trait">rot</span> severity. This study provides the first characterization of aggressiveness and <span class="trait">yield</span> impact of Fusarium <span class="trait">root</span> <span class="trait">rot</span> species on soybean at different plant stages and introduces <span class="trait">root</span> image analysis to assess the impact of <span class="trait">root</span> pathogens on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Separate loci underlie <span class="trait">resistance</span> to <span class="trait">root</span> infection and leaf scorch during soybean sudden death syndrome.</h2>
			<h3>PMID:18324383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kazi S(1), Shultz J, Afzal J, Johnson J, Njiti VN, Lightfoot DA.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] cultivars show differences in their <span class="trait">resistance</span> to both the leaf scorch and <span class="trait">root</span> <span class="trait">rot</span> of sudden death syndrome (SDS). The syndrome is caused by <span class="trait">root</span> colonization by Fusarium virguliforme (ex. F. solani f. sp. glycines). <span class="trait">Root</span> susceptibility combined with reduced leaf scorch <span class="trait">resistance</span> has been associated with <span class="trait">resistance</span> to Heterodera glycines HG Type 1.3.6.7 (race 14) of the soybean cyst nematode (SCN). In contrast, the rhg1 locus underlying <span class="trait">resistance</span> to Hg Type 0 was found clustered with three loci for <span class="trait">resistance</span> to SDS leaf scorch and one for <span class="trait">root</span> infection. The aims of this study were to compare the inheritance of <span class="trait">resistance</span> to leaf scorch and <span class="trait">root</span> infection in a population that segregated for <span class="trait">resistance</span> to SCN and to identify the underlying quantitative trait loci (QTL). "Hartwig", a cultivar partially resistant to SDS leaf scorch, F. virguliforme <span class="trait">root</span> infection and SCN HG Type 1.3.6.7 was crossed with the partially susceptible cultivar "Flyer". Ninety-two F5-derived recombinant inbred lines and 144 markers were used for map development. Four QTL found in earlier studies were confirmed. One contributed <span class="trait">resistance</span> to leaf scorch on linkage group (LG) <span class="gene">C2</span> (Satt277; P = 0.004, R2 = 15%). Two on LG G underlay <span class="trait">root</span> infection at R8 (Satt038; P = 0.0001 R2 = 28.1%; Satt115; P = 0.003, R2 = 12.9%). The marker Satt038 was linked to rhg1 underlying <span class="trait">resistance</span> to SCN Hg Type 0. The fourth QTL was on LG D2 underlying <span class="trait">resistance</span> to <span class="trait">root</span> infection at R6 (Satt574; P = 0.001, R2 = 10%). That QTL was in an interval previously associated with <span class="trait">resistance</span> to both SDS leaf scorch and SCN Hg Type 1.3.6.7. The QTL showed repulsion linkage with <span class="trait">resistance</span> to SCN that may explain the relative susceptibility to SDS of some SCN resistant cultivars. One additional QTL was discovered on LG G underlying <span class="trait">resistance</span> to SDS leaf scorch measured by disease index (Satt130; P = 0.003, R2 = 13%). The loci and markers will provide tagged alleles with which to improve the breeding of cultivars combining resistances to SDS leaf scorch, <span class="trait">root</span> infection and SCN HG Type 1.3.6.7.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:PCR Sampling of disease resistance-like sequences from a disease <span class="trait">resistance</span> gene cluster in soybean.</h2>
			<h3>PMID:12582561</h3>
				

				
			<p>
				作者：Graham A(1), Marek F, Shoemaker C.
				<br/> 
				年份：2002 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Clusters of Resistance-like genes (RLGs) have been identified from a variety of plant species. In soybean, RLG-specific primers and BAC-fingerprinting were used to develop a contig of overlapping BACs for a cluster of RLGs on soybean linkage group J. The <span class="trait">resistance</span> genes Rps2 ( Phytophthora stem and <span class="trait">root</span> <span class="trait">rot)</span> and Rmd-c (powdery mildew) and the ineffective nodulation gene <span class="gene">Rj2</span> were previously mapped to this region of linkage group J. PCR hybridization was used to place two TIR/NBD/LRR cDNAs on overlapping BACs from this contig. Both of the cDNAs were present on BAC 34P7. Fingerprinting of this BAC suggested as many as twelve different RLGs were present. Given the high nucleotide identity shared between cDNAs LM6 and MG13 (&gt;90%), direct sequencing of this region would be difficult. More sequence information was needed about the RLGs within this region before sequencing could be undertaken. By comparing the genomic sequences of cDNAs LM6 and MG13 we identified conserved regions from which oligonucleotide primers specific to BAC 34P7 RLGs could be designed. The nine primer pairs spanned the genomic sequence of LM6 and produced overlapping RLG products upon amplification of BAC 34P7. Amplification products from 12 different RLGs were identified. On average, nucleotide identity between RLG sequences was greater than 95%. Examination of RLG sequences also revealed evidence of additions, deletions and duplications within targeted regions of these genes. Using previously mapped cDNAs we were able to quickly and inexpensively access multiple RLGs within a single specific cluster.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Transcriptome Analysis of Two Races of <italic> Heterodera glycines </italic> at Different Developmental Stages</h2>
			<h3>PMID:24662955</h3>
				
					<h3>PMCID:3963861</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Gaofeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Deliang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Bingli </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Wenkun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Lingan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Long </surname>  <given-names> Haibo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peng </surname>  <given-names> Huan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jian </surname>  <given-names> Heng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The soybean cyst nematode (SCN), <italic> Heterodera glycines </italic> , causes significant damage to soybean, resulting in annual <span class="trait">yield</span> losses between 1.9 and 3.5 million tons in the USA <xref rid="pone.0091634-Wrather1" ref-type="bibr"> [1] </xref> . Developing resistant cultivars to SCN has proven to be a highly effective strategy to control this nematode <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> . Several <span class="trait">resistance</span> genes and QTLs conferring <span class="trait">resistance</span> to <italic> H. glycines </italic> have been identified <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> &#x02013; <xref rid="pone.0091634-Liu1" ref-type="bibr"> [8] </xref> . However, few of these have been introgressed into commercially viable cultivars <xref rid="pone.0091634-Concibido1" ref-type="bibr"> [2] </xref> . Field populations of <italic> H. glycines </italic> have been shown to have high variability <xref rid="pone.0091634-Zheng1" ref-type="bibr"> [9] </xref> , and the significant diversity of field populations combined with lack of applicable <span class="trait">resistance</span> sources could potentially lead to population shifts over time, resulting in virulent populations. For example, repeated growth of potato cultivars carrying the <span class="gene">H1</span> <span class="trait">resistance</span> gene against <italic> Globodera rostochiensis </italic> has led to selection for <italic> G. pallida </italic> in parts of Europe <xref rid="pone.0091634-Turner1" ref-type="bibr"> [10] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>