<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Root length development</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Root length development</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Functional characterization of soybean strigolactone biosynthesis and signaling genes in Arabidopsis MAX mutants and <span class="gene">GmMAX3</span> in soybean nodulation</h2>
			<h3>PMID:29268717</h3>
				

				
					<h3>PMCID:5740752</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Haq </surname>  <given-names> Basir UI </given-names>  </name>  <address>  <email> basirulhaq3star@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ahmad </surname>  <given-names> Muhammad Zulfiqar </given-names>  </name>  <address>  <email> zulfiqar_nuas@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> ur Rehman </surname>  <given-names> Naveed </given-names>  </name>  <address>  <email> naveed.urrehman@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Junjie </given-names>  </name>  <address>  <email> junjiewang88@sina.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Penghui </given-names>  </name>  <address>  <email> lphui2012@126.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongqin </given-names>  </name>  <address>  <email> ldq@mail.hzau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-4416-7334 </contrib-id>  <name>  <surname> Zhao </surname>  <given-names> Jian </given-names>  </name>  <address>  <phone> +86 27 87385199 </phone>  <email> jzhao2@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1790 4137 </institution-id>  <institution-id institution-id-type="GRID"> grid.35155.37 </institution-id>  <institution> National Key Laboratory of Crop Genetic Improvement,  </institution>  <institution> Huazhong Agricultural University,  </institution>  </institution-wrap> Wuhan, 430075 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 4804 </institution-id>  <institution-id institution-id-type="GRID"> grid.411389.6 </institution-id>  <institution> State Key Laboratory of Tea Plant Biology and Utilization, College of Tea and Food Science and Technology,  </institution>  <institution> Anhui Agricultural University,  </institution>  </institution-wrap> Hefei, 230036 China  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Strigolactones (SLs) play important roles in controlling <span class="trait">root</span> growth, shoot <span class="trait">branching,</span> and plant-symbionts interaction. Despite the importance, the components of SL biosynthesis and signaling have not been unequivocally explored in soybean. RESULTS: Here we identified the putative components of SL synthetic enzymes and signaling proteins in soybean genome. Soybean genome contains conserved MORE AXILLARY <span class="trait">BRANCHING</span> (MAX) orthologs, <span class="gene">GmMAX1s,</span> <span class="gene">GmMAX2s,</span> <span class="gene">GmMAX3s,</span> and <span class="gene">GmMAX4s.</span> The tissue expression patterns are coincident with SL synthesis in roots and signaling in other tissues under normal conditions. <span class="gene">GmMAX1a,</span> <span class="gene">GmMAX2a,</span> <span class="gene">GmMAX3b,</span> and <span class="gene">GmMAX4a</span> expression in their Arabidopsis orthologs' mutants not only restored most characteristic phenotypes, such as shoot <span class="trait">branching</span> and shoot height, leaf shape, primary <span class="trait">root</span> <span class="trait">length,</span> and <span class="trait">root</span> hair growth, but also restored the significantly changed hormone contents, such as reduced JA and ABA contents in all mutant leaves, but increased auxin levels in atmax1, atmax3 and atmax4 mutants. Overexpression of these <span class="gene">GmMAXs</span> also altered the hormone contents in wild-type Arabidopsis. <span class="gene">GmMAX3b</span> was further characterized in soybean nodulation with overexpression and knockdown transgenic hairy roots. <span class="gene">GmMAX3b</span> overexpression <span class="gene">(GmMAX3b-OE)</span> lines exhibited increased nodule number while <span class="gene">GmMAX3b</span> knockdown <span class="gene">(GmMAX3b-KD)</span> decreased the nodule number in transgenic hairy roots. The expression levels of several key nodulation genes were also altered in <span class="gene">GmMAX3b</span> transgenic hairy roots. <span class="gene">GmMAX3b</span> overexpression hairy roots had reduced ABA, but increased JA levels, with no significantly changed auxin content, while the contrast changes were observed in <span class="gene">GmMAX3b-KD</span> lines. Global gene expression in <span class="gene">GmMAX3b-OE</span> or <span class="gene">GmMAX3b-KD</span> hairy roots also revealed that altered expression of <span class="gene">GmMAX3b</span> in soybean hairy roots changed several subsets of genes involved in hormone biosynthesis and signaling and transcriptional regulation of nodulation processes. CONCLUSIONS: This study not only revealed the conservation of SL biosynthesis and signaling in soybean, but also showed possible interactions between SL and other hormone synthesis and signaling during controlling plant <span class="trait">development</span> and soybean nodulation. <span class="gene">GmMAX3b-mediated</span> SL biosynthesis and signaling may be involved in soybean nodulation by affecting both <span class="trait">root</span> hair formation and its interaction with rhizobia.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Tissue-specific expression patterns of <italic> <span class="gene">GmMAX1a,</span> 1b, 2a, 3b, </italic> and <italic> 4a </italic> . Relative expression of these <italic> <span class="gene">GmMAXs</span> </italic> to <italic> <span class="gene">GmACTIN</span> </italic> was measured with samples from soybean cultivar &#x0201c;Tianlong 1&#x0201d; at different <span class="trait">development</span> stages by using qRT-PCR. Data are expressed as means &#x000b1; s.d from 3 independent experiments with biological replicates
					</p>
				
					<p>&nbsp;&nbsp;
						Function analysis of <italic> <span class="gene">GmMAX1a</span> </italic> expressed in Arabidopsis. Complementation of <italic> <span class="gene">GmMAX1a</span> </italic> in <italic> atmax1&#x02013;1 </italic> mutant and overexpression in the wild-type Col-0 were done to compare the functions of <span class="gene">GmMAX1a</span> with AtMAX1. <bold> a </bold> The seedlings of <italic> atmax1&#x02013;1, </italic> Col-0, and <italic> atmax1&#x02013;1 </italic> complementation plant with <italic> <span class="gene">GmMAX1a</span> </italic> ( <italic> atmax1/GmMAX1a </italic> ) with different primary <span class="trait">root</span> lengths (left panel). Quantification of shoot height was conducted and calculated on at least three independent lines of more than 20 plants (right panel). <bold> b </bold> Mature plants of Col-0, <italic> atmax1&#x02013;1 </italic> mutant, complementation ( <italic> atmax1/GmMAX1a </italic> ), and <italic> <span class="gene">GmMAX1a</span> </italic> overexpression lines (left panel). The numbers of branches were determined and calculated on at least three independent lines of more than 20 plants (right panel). <bold> c </bold> The differences between seedlings (top panel) and leaf shapes (bottom panel) of Col-0, <italic> atmax1&#x02013;1, </italic> complementation of ( <italic> atmax1&#x02013;1/GmMAX1a </italic> ) and <italic> <span class="gene">GmMAX1a-OE</span> </italic> plants (left panel). Ratios of leaf <span class="trait">length</span> to leaf width were collected on 2-week-old seedlings (right panel). Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 plants for each genotyping). <bold> d </bold> Silique lengths of Col-0, <italic> atmax1&#x02013;1, </italic> and complementation of ( <italic> atmax1/GmMAX1a </italic> ) plants (left panel). Quantification of silique <span class="trait">length</span> (cm) of Col-0, <italic> atmax1&#x02013;1, </italic> complementation of ( <italic> atmax1/GmMAX1a </italic> ) and <italic> <span class="gene">GmMAX1a-OE</span> </italic> plants (right panel). Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 plants for each genotyping). Data are expressed as means &#x000b1; s.d. from 3 independent experiments with biological replicates. Asterisks indicate significant difference according to a Student&#x02019;s <italic> t </italic> -test (** <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.001, * <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						Functions of <italic> <span class="gene">GmMAX2a</span> </italic> expressed in Arabidopsis plants. <bold> a </bold> The differences between seedlings (top panel) and leaf shapes (bottom panel) of Col-0, <italic> atmax2, </italic> complementation of ( <italic> atmax2/GmMAX2a </italic> ) and <italic> <span class="gene">GmMAX2a-OE</span> </italic> plants (left panel). Ratios of leaf <span class="trait">length</span> to leaf width were collected on 2-week-old seedlings (right panel). Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 plants for each genotyping). <bold> b </bold> Hypocotyl lengths of wild type (Col-0), <italic> atmax2, </italic> mutant complementation ( <italic> atmax2/GmMAX2a </italic> ) and <italic> <span class="gene">GmMAX2a-OE</span> </italic> plants. Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;30 seedlings for each genotyping). <bold> c </bold> Mature plants of Col-0, <italic> atmax2 </italic> , mutant complementation ( <italic> atmax2/GmMAX2a </italic> ), and <italic> <span class="gene">GmMAX2a</span> </italic> overexpression lines. <bold> d </bold> Quantification of the number of branches/plant (left panel) and shoot height (cm) (right panel) was conducted and calculated on at least three independent lines of more than 20 plants. Data are expressed as means &#x000b1; s.d. from 3 independent experiments with biological replicates. Asterisks indicate significant difference according to a Student&#x02019;s <italic> t </italic> -test (** <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.001, * <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						Functions of <span class="gene">GmMAX3b</span> when expressed in Arabidopsis plants. <bold> a </bold> Mature plants of Col-0, <italic> atmax3&#x02013;9, </italic> mutant complementation ( <italic> atmax3/GmMAX3b </italic> ), and <italic> <span class="gene">GmMAX3b</span> </italic> overexpression lines. Plants were photographed at 35&#x000a0;days after sowing (left panel). Quantification of the number of branches of each plant (right panel) was conducted and calculated on at least three independent lines of more than 20 plants. <bold> b </bold> The differences between seedlings (top panel) and leaf shapes (bottom panel) of <italic> Col-0 </italic> , <italic> atmax3, </italic> mutant complementation plant ( <italic> atmax3/GmMAX3b </italic> ), and <italic> <span class="gene">GmMAX3b-OE</span> </italic> plants (left panel). Ratios of leaf <span class="trait">length</span> to leaf width (right panel) were collected on 2-week-old seedlings. Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 plants for each genotyping). <bold> c </bold> The different primary <span class="trait">root</span> lengths seedlings of <italic> atmax3/GmMAX3b </italic> complementation <italic> , atmax3, </italic> and Col-0 plant with <italic> <span class="gene">GmMAX3b</span> </italic> ( <italic> atmax3/GmMAX3b </italic> ). <bold> d </bold> Quantification of shoot height was conducted and calculated on at least three independent lines of more than 20 plants. Data represent means &#x000b1; s.d from three independent experiments. Asterisks indicate significant difference according to a Student&#x02019;s <italic> t </italic> -test (** <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.001, * <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						Functions of <span class="gene">GmMAX4a</span> when expressed in Arabidopsis. <bold> a </bold> Mature plants of Col-0, <italic> atmax4 </italic> mutant, complementation ( <italic> atmax4/GmMAX4a </italic> ), and <italic> <span class="gene">GmMAX4a-OE</span> </italic> lines. Plants were photographed at 25&#x000a0;days after sowing (left panel). Quantification of the number of branches of each plant (right panel) was conducted and calculated on at least three independent lines of more than 20 plants. <bold> b </bold> The different primary <span class="trait">root</span> lengths seedlings (left panel) of <italic> atmax4 </italic> complementation plant with <italic> <span class="gene">GmMAX4a</span> </italic> ( <italic> atmax4/GmMAX4a </italic> ), <italic> atmax4 </italic> and Col-0 plants. Quantification of the shoot height (right panel) were conducted and calculated on at least three independent lines of more than 20 plants. <bold> c </bold> Leaf shapes of Col-0, <italic> atmax4, </italic> complementation ( <italic> atmax4/GmMAX4a </italic> ), and <italic> <span class="gene">GmMAX4a-OE</span> </italic> plants (left panel). Ratios of leaf <span class="trait">length</span> to leaf width were collected on 2-week-old seedlings (right panel). Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 plants for each genotyping). Data represent means &#x000b1; s.d from three independent experiments. Asterisks indicate significant difference according to a Student&#x02019;s <italic> t </italic> -test (** <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.001, * <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05)
					</p>
				
					<p>&nbsp;&nbsp;
						Effects of <italic> <span class="gene">GmMAX3b-</span> </italic> overexpression on morphology of hairy roots in mutant Arabidopsis and Soybean. <bold> a </bold> <span class="trait">Root</span> hairs of Col-0, <italic> atmax3&#x02013;9 </italic> , and complementation ( <italic> atmax3/GmMAX3b </italic> ) plants. Bar&#x02009;=&#x02009;1&#x000a0;mm. <bold> b </bold> Quantification of <span class="trait">root</span> hairy <span class="trait">length</span> of Col-0, <italic> atmax3&#x02013;9 </italic> , and complementation ( <italic> atmax3/GmMAX3b </italic> ) plants. <bold> c </bold> Soybean hairy roots expressing <span class="gene">GmMAX3b</span> and <italic> GUS </italic> (control). <bold> d </bold> <span class="trait">Root</span> hair patterns in the soybean hairy roots overexpressing <italic> <span class="gene">GmMAX3b</span> </italic> or <italic> GUS </italic> . Data represent means &#x000b1; s.d ( <italic> n </italic> &#x02009;=&#x02009;20 roots for each genotyping). 20 <span class="trait">root</span> hairs of each <span class="trait">root</span> were used for determination of <span class="trait">root</span> hair <span class="trait">length</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Effect of <italic> <span class="gene">GmMAX3b</span> </italic> overexpression and <italic> <span class="gene">GmMAX3b</span> </italic> knockdown in hairy roots on soybean nodulation. Chimerical soybean plants were generated by transformation with K599 harboring, <italic> <span class="gene">GmMAX3b</span> </italic> -overexpression ( <italic> <span class="gene">GmMAX3b-OE)</span> </italic> , <italic> <span class="gene">GmMAX3b</span> </italic> -knockdown ( <italic> <span class="gene">GmMAX3b-KD)</span> </italic> or <italic> GUS </italic> vector. Plants with wild-type shoots and transgenic hairy roots were inoculated with <italic> Bradyrhizobium japonicum </italic> strain USDA110. Nodule numbers from the hairy roots were counted and the roots were sampled for gene expression analysis at 28&#x000a0;days post inoculation. <bold> a </bold> Chimerical soybean plants with wild-type shoots but transgenic hairy roots. <italic> <span class="gene">GmMAX3b-OE</span> </italic> plants developed more nodules as compared with <italic> GUS </italic> control. <bold> b </bold> qRT-PCR confirmation of <italic> <span class="gene">GmMAX3b</span> </italic> - <italic> OE </italic> and <italic> <span class="gene">GmMAX3b-KD</span> </italic> in transgenic hairy roots as compared to GUS control. <bold> c </bold> Hairy <span class="trait">root</span> fresh weight (g) and nodule numbers ratio in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> control lines. <bold> d </bold> Expression levels of nodulation genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> transgenic hairy roots as control. <bold> e </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> as control hairy <span class="trait">root</span> lines. Gene expression was determined by qRT-PCR with <italic> <span class="gene">GmACTIN</span> </italic> as an internal control. Data are expressed as means &#x000b1; s.d. from at least 3 independent experiments with biological replicates. Differences were analyzed, * <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05; ** <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01 in student&#x02019;s <italic> t </italic> -test
					</p>
				
					<p>&nbsp;&nbsp;
						Effects of <italic> <span class="gene">GmMAX3b</span> </italic> -overexpression and <italic> <span class="gene">GmMAX3b</span> </italic> knockdown on hormone levels in hairy roots. Chimerical soybean plants with wild-type shoots and transgenic hairy roots overexpressing <italic> <span class="gene">GmMAX3b</span> <span class="gene">(GmMAX3b-OE),</span> <span class="gene">GmMAX3b</span> </italic> -knockdown ( <italic> <span class="gene">GmMAX3b-KD)</span> </italic> and <italic> GUS </italic> vector control were inoculated with <italic> Bradirhizobium japonicum </italic> strain USDA110. Hairy roots were sampled for hormones and gene expression analysis at 28&#x000a0;days post inoculation. <bold> a </bold> Hormone contents in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy roots. <bold> b </bold> Validation of auxin biosynthetic genes <italic> TAA </italic> s and <italic> YUC </italic> s in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines. <bold> c </bold> Heat map analysis of hormone biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines. <bold> d </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b</span> </italic> -OE, <italic> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines. Data are expressed as means &#x000b1; s.d from at least three independent experiments with duplicates. Differences are analyzed with student&#x02019;s <italic> t </italic> test, * <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.05, ** <italic> p </italic> &#x02009;&#x0003c;&#x02009;0.01
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors (MYB TFs) play diverse roles in plant <span class="trait">development</span> and stress responses. However, the mechanisms underlying the actions of MYB TFs during stress response remain unclear. In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance.</span> Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> salt stress, H2O2 and ABA. Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary <span class="trait">root</span> <span class="trait">length,</span> greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content. We also found that ROS could induce SOD/POD/CAT activity in OE lines. In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary <span class="trait">root</span> elongation. Some ROS-related genes were up-regulated under abiotic stress in <span class="gene">GmMYB84</span> transgenic plants compared with the WT. Furthermore, electrophoretic mobility shift assay and luciferase reporter analysis demonstrated that <span class="gene">GmMYB84</span> binds directly to the promoter of <span class="gene">GmRBOHB-1</span> and <span class="gene">GmRBOHB-2</span> genes. Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control <span class="trait">root</span> growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich protein) gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genes coding for PGYRPs (proline-, glycine- and tyrosine-rich proteins) are widely distributed across eukaryotes and have been proposed to have critical role in plant <span class="trait">development,</span> especially in response to environmental stresses. In this study, total of 12 soybean PGYRPs <span class="gene">(GmPGYRP1-12)</span> were identified from the soybean genome database for the first time and full-length cDNA and DNA sequences of <span class="gene">GmPGYRP7</span> was cloned. <span class="gene">GmPGYRP1-12</span> genes encoded a set of small predicted proteins (&lt;120 aa) with molecular mass of 7.20-13.29 kDa and isoelectric point of 4.06-6.57. All <span class="gene">GmPGYRPs</span> contained three exons and two introns with fixed occurring sites within genomic DNA sequences. In the putative <span class="gene">GmPGYRP</span> sequences, 4 amino acids (proline, glycine, tyrosine, and glutamine) account for more than 39% of the total protein composition. <span class="gene">GmPGYRPs</span> had a relatively flexible GYPPX motif followed by a highly conserved cysteine-rich domain (GCLAAXCCCCXLXC) and showed high similarity to other known PGYRPs, especially in C-terminal region. Most of PGYRPs can be divided into five subgroups according to phylogenetic analysis. The transcripts of <span class="gene">GmPGYRP1,</span> 3, 5, and 7, representing different PGYRP subgroups, appeared in different organs including seedling leaves, stems, roots, flowers, and developing seeds, but mainly accumulated in seedling roots. Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage. Our data suggest that <span class="gene">GmPGYRP</span> genes encoding a class of conservative XYPPX-repeat proteins probably play an important role in plant <span class="trait">development</span> as well as in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Novel QTL Governing <span class="trait">Root</span> Architectural Traits in an Interspecific Soybean Population</h2>
			<h3>PMID:25756528</h3>
				
					<h3>PMCID:4355624</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Manavalan </surname>  <given-names> Lakshmi P. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Prince </surname>  <given-names> Silvas J. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Musket </surname>  <given-names> Theresa A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chaky </surname>  <given-names> Julian </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> James C. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Specht </surname>  <given-names> James E. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						a) Variation in <span class="trait">root</span> architecture in soybean accessions 12 days after sowing (das); b) Seed coat color of parents (Dunbar, <span class="gene">P1</span> and PI 326582A, P2) and RILs c) Cone system for <span class="trait">root</span> screening d) Dunbar 12 das e) PI 326582A 12 das.
					</p>
				
					<p>&nbsp;&nbsp;
						The genotypes were grown using the &#x0201c;cone" system [ <xref rid="pone.0120490.ref021" ref-type="bibr"> 21 </xref> ] developed to study seedling <span class="trait">root</span> traits. The screening of the parental lines and 251 BILS were replicated six times in a randomized complete block design. The entire population was screened at the same time in a walk-in growth chamber that accommodated 76 racks (Stuewe and Sons, Oregon, USA) set on wooden pallets to facilitate support and water drainage ( <xref rid="pone.0120490.g001" ref-type="fig"> Fig. 1c </xref> ). As <italic> G </italic> . <italic> soja </italic> exhibits considerable seed dormancy, the parental <italic> G </italic> . <italic> soja </italic> seed coat was scarified before planting by making a slanted cut in the seed coat at the opposite end of the embryo from the hilum to facilitate the germination process. A turface:sand (2:1 v/v) mixture ( <ext-link ext-link-type="uri" xlink:href="http://www.hummert.com"> www.Hummert.com </ext-link> ) was used as the growth medium [ <xref rid="pone.0120490.ref021" ref-type="bibr"> 21 </xref> ]. The mixture is a growth medium similar to field soil and facilitates easy collection of the entire <span class="trait">root</span> system without damage [ <xref rid="pone.0120490.ref042" ref-type="bibr"> 42 </xref> ]. Turface is calcined clay and has cation exchange capacity, so it is likely to have nutrients associated with its exchange sites. It adds water and air-holding pore space, prevents soil compaction, and allows better drainage. Sand facilitates the optimization of the air&#x02014;water balance in the <span class="trait">root</span> zone. The media has no mechanical <span class="trait">resistance</span> to <span class="trait">root</span> penetration under well-watered conditions [ <xref rid="pone.0120490.ref021" ref-type="bibr"> 21 </xref> ]. The seedlings were grown in cones to the <span class="gene">V1</span> stage in a growth chamber with controlled conditions of 27/21&#x000b0;C day/night temperature, photoperiod of 16/8 day/night, 65% relative humidity, and an average light intensity of ~262 &#x003bc;mol m <sup> -2 </sup> s <sup> -1 </sup> measured at canopy level. At 12 days after sowing, intact seedlings were separated by cutting the cones longitudinally. The tap <span class="trait">root</span> <span class="trait">length</span> was measured with a ruler from the portion of the shoot that appeared white to the tip of the tap <span class="trait">root.</span> The <span class="trait">length</span> of the shoot was determined as that of the region above the white portion (either purple or green) up to the shoot apical meristem. The <span class="trait">root</span> system of each seedling was clipped and immediately washed in a tray of water, labeled, wrapped individually in wet paper towels and stored at 4&#x000b0;C for image analysis. Shoot dry weight was determined after shoot tissue (with cotyledons and leaves) had been dried in an oven at 65&#x000b0;C for 48 h.
					</p>
				
					<p>&nbsp;&nbsp;
						The selected BILs and parental lines were grown to the <span class="gene">V1</span> stage and the <span class="trait">root</span> tissues were collected for RNA isolation. RNA was extracted from <span class="trait">root</span> tissues (100 mg tissues) using RNeasy Plant mini kit (Qiagen, CA, USA) according to manufacturer&#x02019;s protocol. On-column DNA digestion was performed using RNase-Free DNase Set (Qiagen, CA, USA) according to the manufacturer&#x02019;s protocol. Each sample (2&#x003bc;g of total RNA) was reverse-transcribed to cDNA in a 20&#x003bc;L reaction volume using RNA to cDNA EcopryTM Premix (Double p <underline> r </underline> imed) cDNA Synthesis Kit (Clontech, CA, USA). Quantitative RT-PCR (qRT-PCR) was performed using the cDNA product corresponding to 25 ng of total RNA in a 10&#x003bc;L reaction volume using Maxima SYBR Green/ROX qPCR Master Mix (2X) (Thermo, USA) on a ABI7900HT detection system (Applied BioSystems, Foster City, CA, USA). The expression data for each sample was generated from three biological and two technical replicates. The relative expression of the selected genes were expressed as mean standard deviation, in comparison to transcript abundance levels of ubiquitin, a housekeeping gene and analyzed using Delta Ct method [ <xref rid="pone.0120490.ref050" ref-type="bibr"> 50 </xref> ]. The PCR conditions were as follows: 50&#x000b0;C for 2min, 95&#x000b0;C for 10 min, then 40 cycles of 95&#x000b0;C for 15 s, and 60&#x000b0;C for 1 min. To normalize the gene expression, ubiquitin ( <italic> <span class="gene">Glyma20g27950</span> </italic> ) was used as an internal control. All primers were designed using Primer3 web-interface ( <ext-link ext-link-type="uri" xlink:href="http://frodo.wi.mit.edu/primer3/"> http://frodo.wi.mit.edu/primer3/ </ext-link> input.htm) [ <xref rid="pone.0120490.ref051" ref-type="bibr"> 51 </xref> ] and the primer sequence information is given in <xref rid="pone.0120490.s006" ref-type="supplementary-material"> S2 Text </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Based on comparison of the genetic map with the sequence map of Williams82 ( <italic> Glyma </italic> 1.0 version) in SoyBase the marker flanking the QTL on chromosome 8, Satt315 and Satt424 were located between 6751690 and 6751725 and between 10721723 to 10721881 Mb, respectively. Within this genomic interval region, 504 genes were identified from Williams82 and sorted based on the expression pattern levels in soybean <span class="trait">root</span> and nodule tissues ( <xref rid="pone.0120490.s002" ref-type="supplementary-material"> S2 Dataset </xref> ). Based on Soybase transcriptome data only 75 out of 504 genes showed higher and tissue specific expressions in soybean <span class="trait">root</span> and nodules ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ). The nuclear transport factor 2 ( <italic> <span class="gene">Glyma08g11070</span> </italic> ) showed more than a 300-fold increased expression in <span class="trait">root</span> tissues. Although 12 transcription factor (TF) families were identified in this QTL interval, only a few TFs including Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ), GRAS ( <italic> <span class="gene">Glyma08g10140</span> </italic> ), WRKY ( <italic> <span class="gene">Glyma08g12460</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed more than a 50- fold expression in soybean <span class="trait">root</span> tissues ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ). Based on Affymetrix gene chip data, six Transcription factors (MYB HD; <italic> <span class="gene">Glyma08g12320</span> </italic> , TPR: <italic> <span class="gene">Glyma08g09550</span> </italic> , C2H2 Zn: <italic> <span class="gene">Glyma08g11800</span> </italic> , BZIP: <italic> <span class="gene">Glyma08g12170</span> </italic> , GRAS: <italic> <span class="gene">Glyma08g10140</span> </italic> and Ring finger: <italic> <span class="gene">Glyma08g13900</span> </italic> ) only showed higher expression on soybean <span class="trait">root</span> tissues ( <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ). All six TFs showed a higher expression in the wild soybean parental line, PI326582A than the Dunbar ( <xref rid="pone.0120490.g007" ref-type="fig"> Fig. 7 </xref> ), except for TPR ( <italic> <span class="gene">Glyma08g09550</span> </italic> ) transcription factor. Even the BIL126 with wild soybean parental allele in the QTL region did not showed similar expression as wild soybean parent. All the selected BILs showed expression level of TFs within the parental values except in BIL 217 for Ring finger TF ( <xref rid="pone.0120490.g007" ref-type="fig"> Fig. 7 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Based on public soybean Affymetrix expression data, two TFs, AP2-EREBP ( <italic> <span class="gene">Glyma08g14600</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed higher expression in <span class="trait">root</span> tissues ( <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ). This <span class="gene">AP2</span> TF also showed higher expression in <span class="trait">root</span> tissues (37- fold) and in nodules (13- fold) within the transcriptome data ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ). Key cell wall expansion-related genes ( <italic> <span class="gene">Glyma08g11300</span> </italic> and <italic> <span class="gene">Glyma08g09940</span> </italic> ) encoding xyloglucan endo-transglycosylases (XET) with high expression in both <span class="trait">root</span> and shoot tissues were identified. However, the gene <italic> <span class="gene">Glyma08g11300</span> </italic> showed a higher expression in <span class="trait">root</span> tissues than <italic> <span class="gene">Glyma08g09940</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						For the genomic comparison of the QTL region of chromosome 8, the DNA sequence data of Williams 82 (cultivated soybean) was compared with 17 and 2 diverse wild soybean accessions from China and Korea, respectively. These accessions showed conserved single nucleotide polymorphic (SNP) variation within them ( <xref rid="pone.0120490.s003" ref-type="supplementary-material"> S3 Dataset </xref> ) in comparison to the cultivated soybean reference genome, Williams 82. Among the two Korea wild soybean accessions, PI 407162 sequenced at 15X depth by our lab also enabled us to identify non-synonymous SNPs in genes located within the QTL interval. Among the 504 genes identified within this QTL interval, 43 genes had non-synonymous SNPs that altered the amino acid ( <xref rid="pone.0120490.s004" ref-type="supplementary-material"> S4 Dataset </xref> ), resulting in changing the translated protein. Most of these genes showed conserved SNP variation among diverse wild soybeans accessions from China and Korea. 12 genes identified with a non-synonymous SNP also showed higher expression in soybean <span class="trait">root</span> and nodule tissues ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ). In particular, two transcription factors like Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ) and C3H type 1 ( <italic> <span class="gene">Glyma08g09630</span> </italic> ) showed high expression in <span class="trait">root</span> tissues in both Affymetrix ( <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ) and transcriptome ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ) public datasets.
					</p>
				
					<p>&nbsp;&nbsp;
						The QTLs identified in this study on chromosome 8 influenced both shoot and <span class="trait">root</span> growth. It is of interest to compare the markers associated with our QTL with QTLs that have been reported earlier. For example, SSR marker Satt424 associated with TRL and SL in this study was, in a prior study, associated with a QTL that explained 46% of variation for internode <span class="trait">length</span> [ <xref rid="pone.0120490.ref058" ref-type="bibr"> 58 </xref> ], and in another study was associated with a large-effect QTL conferring flooding tolerance [ <xref rid="pone.0120490.ref059" ref-type="bibr"> 59 </xref> ]. In addition, this region confers <span class="trait">resistance</span> to biotic stresses such as soybean cyst nematode (QTL SCN30&#x02013;3 [ <xref rid="pone.0120490.ref060" ref-type="bibr"> 60 </xref> ]; SCN29&#x02013;5 [ <xref rid="pone.0120490.ref061" ref-type="bibr"> 61 </xref> ], and <italic> Sclerotinia </italic> <span class="trait">root</span> rot (QTL Sclero 2&#x02013;2, 3&#x02013;2, 5&#x02013;1, 6&#x02013;2) [ <xref rid="pone.0120490.ref062" ref-type="bibr"> 62 </xref> ]. The QTL map location for <span class="trait">root</span> and shoot traits ( <xref rid="pone.0120490.t004" ref-type="table"> Table 4 </xref> ) near the class I locus, which controls seed coat pigmentation [ <xref rid="pone.0120490.ref063" ref-type="bibr"> 63 </xref> ] on chromosome 8, is closely linked to <span class="gene">Rhg4</span> gene [ <xref rid="pone.0120490.ref064" ref-type="bibr"> 64 </xref> ] that encodes serine hydroxylmethyltransferase and confers <span class="trait">resistance</span> to soybean cyst nematode [ <xref rid="pone.0120490.ref065" ref-type="bibr"> 65 </xref> ]. A QTL for soybean seed <span class="trait">length</span> was also flanked by this marker [ <xref rid="pone.0120490.ref066" ref-type="bibr"> 66 </xref> ]. Collectively, the marker interval Satt315&#x02013;Satt424 contains QTLs for growth and <span class="trait">yield</span> components including leaf width, leaf shape [ <xref rid="pone.0120490.ref067" ref-type="bibr"> 67 </xref> ]; days to flowering [ <xref rid="pone.0120490.ref068" ref-type="bibr"> 68 </xref> ]; seed weight [ <xref rid="pone.0120490.ref069" ref-type="bibr"> 69 </xref> , <xref rid="pone.0120490.ref070" ref-type="bibr"> 70 </xref> ]; pod maturity and seed protein content [ <xref rid="pone.0120490.ref071" ref-type="bibr"> 71 </xref> ] in Soybase [ <xref rid="pone.0120490.ref037" ref-type="bibr"> 37 </xref> ]. These loci may be investigated as candidates for utilization in marker assisted breeding (MAB) programs to improve <span class="trait">root</span> architecture and other traits.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the eleven TFs identified within the QTL interval, the MYB TF ( <italic> <span class="gene">Glyma08g11640</span> </italic> ) showed higher expression in shoot tissues (shoot apical meristem and trifoliate leaves) ( <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ). But the MYB-HD TF showed a higher expression in <span class="trait">root</span> tissues <span class="trait">(root,</span> <span class="trait">root</span> tip, and pericycle) in both the Affymetrix gene chip expression data ( <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ) and the Soybase RNA sequencing data ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ). This TF ( <italic> <span class="gene">Glyma08g12320</span> </italic> ) also showed a higher expression in wild soybean accession PI407162 and none of the interspecific BILs showed an expression pattern as high as that of the parental lines ( <xref rid="pone.0120490.g007" ref-type="fig"> Fig. 7 </xref> ). The allelic interaction between cultivated and wild soybean may have affected the gene expression in BILs. Similar distinct expression patterns of <span class="trait">root</span> related genes were found in a soybean inter-specific mapping population [ <xref rid="pone.0120490.ref073" ref-type="bibr"> 73 </xref> ]. Role of MYB TFs in <span class="trait">root</span> formation were reported earlier in soybean and <italic> Arabidopsis </italic> [ <xref rid="pone.0120490.ref074" ref-type="bibr"> 74 </xref> ]. In these datasets, ( <xref rid="pone.0120490.s007" ref-type="supplementary-material"> S3 Text </xref> ; <xref rid="pone.0120490.g006" ref-type="fig"> Fig. 6 </xref> ) the <span class="gene">AP2</span> TF as being highly expressed in shoot, <span class="trait">root,</span> and nodule tissues. The role of this TF in nodulation and association with the Nod factor signaling pathway was shown earlier in <italic> Medicago truncatula </italic> [ <xref rid="pone.0120490.ref075" ref-type="bibr"> 75 </xref> ] and its over-expression also confers <span class="trait">drought</span> tolerance in <italic> Arabidopsis </italic> [ <xref rid="pone.0120490.ref076" ref-type="bibr"> 76 </xref> ]. Interestingly, the bZIP TF showed a higher expression in <span class="trait">root</span> pericycle and nodules in both datasets. The pericycle cells are key cell types that form lateral <span class="trait">root</span> primordium, which decides the lateral <span class="trait">root</span> number. A similar root-specific bZIP TF has been reported to be responsive during water stress and involved in intracellular signaling in both tepary and common beans [ <xref rid="pone.0120490.ref077" ref-type="bibr"> 77 </xref> ]. This TF was also reported to be involved in controlling nodule number by early initiation of nodulation in <italic> Lotus japonicus </italic> [ <xref rid="pone.0120490.ref078" ref-type="bibr"> 78 </xref> ]. Drought-tolerant soybean also showed high nodule number and size under water-deficit conditions [ <xref rid="pone.0120490.ref079" ref-type="bibr"> 79 </xref> , <xref rid="pone.0120490.ref080" ref-type="bibr"> 80 </xref> ], resulting in better nitrogen fixation ability that translated into higher <span class="trait">yield</span> in <span class="trait">drought</span> stress conditions [ <xref rid="pone.0120490.ref081" ref-type="bibr"> 81 </xref> ]. However, more research is needed to illuminate the role of this TF in increasing nodule number and sustaining nitrogen fixation capacity under <span class="trait">drought</span> conditions. Among the six TFs selected based on their Affymetrix gene chip expression pattern, TPR transcription factor showed a higher expression in cultivated soybean, Dunbar. Similar expression pattern for a TPR TF in the same gene family was reported in another cultivated soybean, V71&#x02013;370 with non-synonymous SNPs in comparison to a wild soybean, PI407162 [ <xref rid="pone.0120490.ref073" ref-type="bibr"> 73 </xref> ]. A similar class of transcription factor was expressed in soybean roots as an early response to iron availability [ <xref rid="pone.0120490.ref082" ref-type="bibr"> 82 </xref> ]. Among the 42 genes identified with non-synonymous SNPs ( <xref rid="pone.0120490.s004" ref-type="supplementary-material"> S4 Dataset </xref> ) in wild soybean accessions, only two TFs (C3H type 1 and Homeobox Domain) that showed higher expression in both Affymetrix and trancriptome datasets. These TFs might be the possible candidates for downstream genes associated with <span class="trait">root</span> system architecture in soybeans. An association of SNP variants with the <span class="trait">root</span> phenotype must be established in the future to identify genes that regulate <span class="trait">root</span> <span class="trait">development</span> in soybean. The conserved SNPs among diverse wild soybean accessions ( <xref rid="pone.0120490.s003" ref-type="supplementary-material"> S3 Dataset </xref> and <xref rid="pone.0120490.s004" ref-type="supplementary-material"> S4 Dataset </xref> ) might be best choices for studying the evolution of <span class="trait">root</span> traits in cultivated soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <italic> <span class="gene">GmRAV</span> </italic> Ortholog Is Involved in Photoperiod and Sucrose Control of Flowering Time in Soybean</h2>
			<h3>PMID:24551235</h3>
				

				
					<h3>PMCID:3925180</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Qingyao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Lin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Diqiu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhan </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photoperiod and sucrose levels play a key role in the control of flowering. <span class="gene">GmRAV</span> reflected a diurnal rhythm with the highest expression at 4 h after the beginning of a dark period in soybean leaves, and was highly up-regulated under short-day (SD) conditions, despite of not following a diurnal pattern under long-day (LD) conditions. <span class="gene">GmRAV-i</span> <span class="gene">(GmRAV-inhibition)</span> transgenic soybean exhibited early flowering phenotype. Two of the FT Arabidopsis homologs, <span class="gene">GmFT2a</span> and <span class="gene">GmFT5a,</span> were highly expressed in the leaves of soybeans with inhibition (-i) of <span class="gene">GmRAV</span> under SD conditions. Moreover, the transcript levels of the two FT homologs in <span class="gene">GmRAV-i</span> soybeans were more sensitive to SD conditions than LD conditions compared to the WT plant. <span class="gene">GmRAV-i</span> soybeans and Arabidopsis rav mutants showed more sensitive hypocotyl elongation responses when compared with wild-type seedlings, and <span class="gene">GmRAV-ox</span> overevpressed in tobacco revealed no sensitive changes in hypocotyl <span class="trait">length.</span> These indicated that <span class="gene">GmRAV</span> was a novel negative regulator of SD-mediated flowering and hypocotyl elongation. Although sucrose has been suggested to promote flowering induction in many plant species, high concentration of sucrose (4% [w/v]) applied into media defer flowering time in Arabidopsis wild-type and rav mutant. This delayed flowering stage might be caused by reduction of LEAFY expression. Furthermore, Arabidopsis rav mutants and <span class="gene">GmRAV-i</span> soybean plants were less sensitive to sucrose by the inhibition assays of hypocotyls and roots growth. In contrast, transgenic <span class="gene">GmRAV</span> overexpressing (-ox) tobacco plants displayed more sensitivity to sucrose. In conclusion, <span class="gene">GmRAV</span> was inferred to have a fundamental function in photoperiod, darkness, and sucrose signaling responses to regulate plant <span class="trait">development</span> and flowering induction.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this study, <italic> <span class="gene">GmRAV</span> </italic> was overexpressed in transgenic tobacco and inhibited in transgenic soybean, which showed that <italic> <span class="gene">GmRAV</span> </italic> was a responder in the photoperiodic control of flowering time and sugar signaling. We found that <italic> <span class="gene">GmRAV</span> </italic> transcript exhibited a circadian rhythm under SDs and decreased significantly in leaves by exogenous sucrose application. A detailed phenotypic characterization, along with genetic and physiological analysis, indicated that <italic> <span class="gene">GmRAV</span> </italic> was inferred to be a signaling component involved in regulation of plant <span class="trait">development</span> and flowering time.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription profiles of <italic> <span class="gene">GmRAV</span> </italic> were analyzed in various tissues of &#x02018;Dong Nong 42&#x02019; soybean grown under inductive SDs at 12 h after dawn by quantitative real-time RT-PCR. <italic> <span class="gene">GmRAV</span> </italic> mRNA was present in all organs examined, including leaf (Tl), trifoliate leaf (TL), stem (St), <span class="trait">root</span> (RO), pod (PO), flower bud (FB), and immature seed (IS). In SDs, the mRNA abundance of <italic> <span class="gene">GmRAV</span> </italic> was the highest in trifoliate leaves and the lowest in pods ( <xref ref-type="fig" rid="pone-0089145-g001"> Fig. 1A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A, Tissue-specific expression of soybean <italic> <span class="gene">GmRAV</span> </italic> in SDs. Tissues tested are leaf (Tl), trifoliate leaf (TL), stem (St), <span class="trait">root</span> (RO), pod (PO), flower bud (FB), and immature seed (IS) (plants aged 21 d). B, Relative transcript levels of <italic> <span class="gene">GmRAV</span> </italic> mRNA in soybean leaves under SDs and LDs. Soybean leaves were harvested every 4 h for 48 h at 25-day-old under LDs and SDs. <italic> Open </italic> and <italic> closed boxes </italic> indicate days and nights. C, Time course-dependent expression in LDs. Soybean &#x02018;Dong Nong 47&#x02019; plants were grown under LDs for 10 d and were transferred to LDs or SDs before sampling. Relative transcript levels were analyzed by qRT-PCR. D, Histochemical detection of <italic> <span class="gene">GmRAV</span> </italic> &#x02013;GUS promoter activity in transgenic <italic> Arabidopsis </italic> seedlings. 4-day-old seedlings were grown on <span class="gene">MS</span> medium.
					</p>
				
					<p>&nbsp;&nbsp;
						The transcript abundance of <italic> <span class="gene">GmRAV</span> </italic> was affected by day <span class="trait">length</span> in soybean leaves, the diurnal phase of <italic> <span class="gene">GmRAV</span> </italic> mRNA expression was regulated by the circadian clock, and higher levels of <italic> <span class="gene">GmRAV</span> </italic> mRNA were accumulated under SDs than under LDs. To examine whether <italic> <span class="gene">GmRAV</span> </italic> acted in a photoperiod functional context, the flowering time in WT and <italic> <span class="gene">GmRAV-i</span> </italic> soybeans under SDs and LDs were analyzed. The early-flowering phenotype soybean mutant <italic> <span class="gene">GmRAV-i</span> </italic> was observed despite of the day <span class="trait">length</span> in both LDs and SDs compared to the WT ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig. 2A </xref> ). Apart from their flowering-time phenotype, <italic> <span class="gene">GmRAV-i</span> </italic> soybeans also displayed complex pleiotropic alterations of vegetative <span class="trait">development.</span> Earlier emergence, reduced numbers of branches and leaves, longer petioles, larger leaves, increased apical dominance, and earlier flowering and maturity were the major phenotypic effects of <italic> <span class="gene">GmRAV-i</span> </italic> in all T6 generation plants in both SDs and LDs ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig. 2A </xref> , <xref ref-type="table" rid="pone-0089145-t003"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis, <italic> FT </italic> transcript levels oscillated with distinct circadian rhythms (Su&#x000e1;rez-L&#x000f3;pez et al., 2001). To check <italic> FT </italic> transcription levels in soybeans, the diurnal circadian rhythm of <italic> FT </italic> gene expression was analyzed by quantitative real-time RT-PCR for <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> in trifoliate leaves sampled at 15 d after emergence (DAE) in <italic> <span class="gene">GmRAV-i</span> </italic> soybean. The expression level of both <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> reached a peak 4 h after the beginning of the dark period and decreased toward dawn in <italic> <span class="gene">GmRAV-i</span> </italic> soybean plants under SDs. The amplitude and overall level of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> mRNA were much higher in <italic> <span class="gene">GmRAV-i</span> </italic> soybean plants than in WT plants under SDs ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig. 2C, D </xref> ). Moreover, the transcript abundance of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> was highly affected in <italic> <span class="gene">GmRAV-i</span> </italic> transgenic soybean leaves compared to the wild-type seedlings under SDs than under LDs ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig. 2E, F </xref> ). The results indicated that <italic> <span class="gene">GmRAV</span> </italic> was a SD-inducible flowering repressor in the flowering response of SD-induced soybeans by repressing positive regulator <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> gene expression. This work therefore described the conservation of components and sequence order of a pathway controlling flowering in response to day <span class="trait">length.</span> It revealed that the promotion of flowering in short days in <italic> <span class="gene">GmRAV-i</span> </italic> soybean resulted from the repression of <italic> <span class="gene">GmFT2a</span> </italic> and <italic> <span class="gene">GmFT5a</span> </italic> by <italic> <span class="gene">GmRAV</span> </italic> ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig. 2G </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Masa-aki et al. reported that 2 weeks culture was enough to observe the negative effects of high levels of sucrose on floral transition <xref rid="pone.0089145-Ohto1" ref-type="bibr"> [24] </xref> . Therefore, wild-type plants and <italic> Arabidopsis rav </italic> mutant seedlings were grown in culture medium containing 2% (w/v) and 4% (w/v) sucrose for 2 weeks, respectively and then were transferred to soil. Flowering time of Arabidopsis WT and <italic> rav </italic> mutants were examined on <span class="gene">MS</span> medium containing 4% sucrose in LDs. In comparison to the plants that were grown on 2% sucrose plates, <italic> rav </italic> mutants showed a 2-day delay in flowering time, whereas WT plants showed 4-day delay ( <xref ref-type="fig" rid="pone-0089145-g005"> Fig. 5A </xref> ). These results supported our hypothesis that Arabidopsis <italic> rav </italic> mutants responded to sucrose signaling in plant growth and <span class="trait">development.</span> Sucrose affecting flowering time was also observed in the developmental phenotypes ( <xref ref-type="fig" rid="pone-0089145-g005"> Fig. 5B </xref> ). Overall, these results indicated that <italic> AtRAV </italic> participated in the regulation of high levels of sucrose-dependent flowering response.
					</p>
				
					<p>&nbsp;&nbsp;
						A, Arabidopsis <italic> rav </italic> mutants and WT seedlings were grown on media with various concentrations of sucrose for 2 weeks, and then transferred to soil under LDs. The flowering time of seedlings with 2% sucrose was WT (24&#x000b1;0.5) and Arabidopsis <italic> rav </italic> mutant (21&#x000b1;0.8). WT plants are the control for Arabidopsis <italic> rav </italic> mutants. Values are the average of 30 to 45 plants. The error bars indicate one SE of the mean. Similar results were obtained in two independent experiments. *Differences in comparison to the non-transgenic lines at 0.01&#x0003c;P&#x0003c;0.05, **Significant differences in comparison to the non-transgenic lines at P&#x0003c;0.01 (Student&#x02019;s t test). B, Phenotypes of the Arabidopsis <italic> rav </italic> mutant. 23-day old seedlings of WT and Arabidopsis <italic> rav </italic> mutant under natural day <span class="trait">length</span> (LD) with treated by 2% and 4% sucrose. C, Quantitative real-time RT&#x02013;PCR analysis of <italic> <span class="gene">LFY</span> </italic> expression in Arabidopsis <italic> rav </italic> mutants. Control amplification of <italic> 18 s rRNA </italic> transcript indicated equal amounts of cDNA.
					</p>
				
					<p>&nbsp;&nbsp;
						To analyze the possible function of <italic> <span class="gene">GmRAV</span> </italic> in response to sucrose stress, we studied the responses to exogenous sucrose application using Arabidopsis <italic> rav </italic> , <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco, and <italic> <span class="gene">GmRAV-i</span> </italic> soybean plants. Sensitivity of <italic> Arabidopsis rav </italic> mutants and <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco seedlings in response to sucrose was tested in <span class="trait">root</span> and hypocotyl by growth inhibition assays. The hypocotyl and <span class="trait">root</span> lengths of <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and WT plants were inhibited by both concentrations of sucrose (2% and 4%) ( <xref ref-type="fig" rid="pone-0089145-g006"> Fig. 6A </xref> ), but <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco were more remarkably reduced compared to WT plants ( <xref ref-type="fig" rid="pone-0089145-g006"> Fig. 6B, C </xref> ). These results suggested that <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco were more sensitive to sucrose than WT plants in the assays of hypocotyls and roots growth inhibition. <xref ref-type="fig" rid="pone-0089145-g007"> Fig. 7A </xref> showed that the hypocotyl and <span class="trait">root</span> lengths of Arabidopsis <italic> rav </italic> mutants and WT plants were inhibited by both concentrations of sucrose. However, inhibited extent of <italic> rav </italic> mutants and WT plants by sucrose in hypocotyl and <span class="trait">root</span> inhibition assays was identical ( <xref ref-type="fig" rid="pone-0089145-g007"> Fig. 7B, C </xref> ). Likewise, the growth in soybean <italic> <span class="gene">GmRAV-i</span> </italic> and WT plants in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots was all inhibited by both concentrations of sucrose ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig. 8A </xref> ). Hypocotyl, <span class="trait">root</span> <span class="trait">length,</span> and the number of lateral roots of <italic> <span class="gene">GmRAV-i</span> </italic> soybeans were less inhibited by sucrose than in WT plants ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig. 8 </xref> B&#x02013;D). Therefore, <italic> <span class="gene">GmRAV-i</span> </italic> seedlings exhibited insensitive phenotypes, as compared with the WT plant, during sucrose-mediated <span class="trait">root</span> and hypocotyl growth inhibition. Overall, in agreement with the <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco results, the Arabidopsis <italic> rav </italic> mutants and the <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings showed a significantly decreased sensitivity to sucrose by <span class="trait">root</span> and hypocotyl growth inhibition assays.
					</p>
				
					<p>&nbsp;&nbsp;
						A and B, Relative hypocotyl and <span class="trait">root</span> growth in response to various concentrations of sucrose. The <span class="trait">length</span> of hypocotyl of tobacco seedlings grown 2% (w/v) sucrose was 4.91&#x000b1;0.75 mm for the WT and 5.85&#x000b1;0.76 mm for <italic> <span class="gene">GmRAV-ox</span> </italic> . <span class="trait">Root</span> <span class="trait">length,</span> of seedlings grown 2% (w/v) sucrose was 15.8&#x000b1;1.21 mm in the WT and 18.8&#x000b1;1.94 mm in <italic> <span class="gene">GmRAV-ox</span> </italic> . The hypocotyl and <span class="trait">root</span> <span class="trait">length</span> of 20&#x02013;30 seedlings were measured for each treatment. Error bars represent the SE. C, Phenotype of 7-day-old WT seedlings and T3 generation <italic> <span class="gene">GmRAV-ox</span> </italic> tobaccos on <span class="gene">MS</span> medium containing 2% and 4% sucrose. *Differences in comparison to the non-transgenic lines at 0.01&#x0003c;P&#x0003c;0.05 (Student&#x02019;s t test).
					</p>
				
					<p>&nbsp;&nbsp;
						A and B, Relative hypocotyl and <span class="trait">root</span> growth in response to various concentrations of sucrose. Hypocotyl <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on sucrose. The <span class="trait">length</span> of hypocotyl of Arabidopsis seedlings grown 2% (w/v) sucrose was 4.26&#x000b1;0.24 mm for the WT and 4.06&#x000b1;0.46 mm for Arabidopsis <italic> rav </italic> mutant. <span class="trait">Root</span> <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on sucrose. <span class="trait">Root</span> <span class="trait">length,</span> of seedlings grown without sucrose was 26.70&#x000b1;1.16 mm in the WT and 28.24&#x000b1;1.33 mm in Arabidopsis <italic> rav </italic> mutant. The hypocotyl and <span class="trait">root</span> <span class="trait">length</span> of 20&#x02013;30 seedlings were measured for each treatment. Error bars represent the SE. C, Phenotype of 7-day-old WT seedlings and Arabidopsis <italic> rav </italic> mutants on <span class="gene">MS</span> medium containing 2% and 4% sucrose.
					</p>
				
					<p>&nbsp;&nbsp;
						A and B, Relative hypocotyl and <span class="trait">root</span> growth in response to various concentrations of sucrose. Hypocotyl <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on sucrose. The <span class="trait">length</span> of hypocotyl of soybean seedlings grown without sucrose or glucose was 57.17&#x000b1;1.30 mm for the WT and 61.17&#x000b1;1.97 mm for <italic> <span class="gene">GmRAV-i</span> </italic> soybean. <span class="trait">Root</span> <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on sucrose. <span class="trait">Root</span> <span class="trait">length,</span> of seedlings grown without sucrose or glucose was 60.33&#x000b1;1.30 mm in the WT and 65.67&#x000b1;1.97 mm in <italic> <span class="gene">GmRAV-i</span> </italic> . Error bars represent the SE. C, Relative number of lateral roots in response to various concentrations of sucrose. The number of lateral roots, as a percentage of the untreated control, of seedlings grown on sucrose. Error bars represent the SE. The number of lateral roots of seedlings grown without sucrose and glucose was 17.83&#x000b1;0.75 for the WT and 21.17&#x000b1;0.98 for <italic> <span class="gene">GmRAV-i</span> </italic> soybean. The number of lateral roots of 20&#x02013;30 seedlings was scored for each treatment. *Differences in comparison to the non-transgenic lines at 0.01&#x0003c;P&#x0003c;0.05. D, Phenotypes of 7-day-old WT and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings on <span class="gene">MS</span> medium containing 2% and 4% sucrose. *Differences in comparison to the non-transgenic lines at 0.01&#x0003c;P&#x0003c;0.05 (Student&#x02019;s t test).
					</p>
				
					<p>&nbsp;&nbsp;
						The addition of chemicals to a medium changes both the chemical composition and the osmotic potential of medium. To identify whether the negative effects of sucrose on <span class="trait">development</span> and flowering were due to metabolic or osmotic factors, we examined the effects of both mannitol and sorbitol. These two sugar alcohols were widely used as the osmotic controls in plant. Therefore, a combined osmotic control value was calculated from both mannitol- and sorbitol-grown plants. Sugar effects were compared to the combined osmotic control value. Plants grown in the absence of any supplemental carbon (sugar alcohol or sucrose) were also measured for each parameter in order to observe the overall effect of increasing osmolarity. Increasing osmolarity of the growth media by sugar alcohols inhibited the plant growth when compared to untreated media. The behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth inhibition assay ( <xref ref-type="fig" rid="pone-0089145-g009"> Fig. 9 </xref> A&#x02013;E). However, <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> that exhibited insensitive phenotypes were less inhibited by 200 mM sugars (6.8%) ( <xref ref-type="fig" rid="pone-0089145-g009"> Fig. 9A, C&#x02013;E </xref> ), which indicated <italic> <span class="gene">RAV</span> </italic> gene played a negative role in sucrose-mediated regulation of hypocotyl and <span class="trait">root</span> elongation due to metabolic effects rather than osmotic effects.
					</p>
				
					<p>&nbsp;&nbsp;
						A and B, Relative <span class="trait">root</span> <span class="trait">length</span> in response to 200-day-old Arabidopsis and tobaccos. <span class="trait">Root</span> <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on carbon source. Error bars represent the SE. <span class="trait">Root</span> <span class="trait">length,</span> of seedlings grown without carbon sucrose was WT (15.96&#x000b1;1.2 mm) and Arabidopsis <italic> rav </italic> mutant (19.25&#x000b1;0.8 mm), and for tobaccos without carbon sucrose was WT (15.29&#x000b1;0.9 mm) and <italic> <span class="gene">GmRAV-ox</span> </italic> (18.4&#x000b1;1.2 mm). <span class="trait">Root</span> <span class="trait">length</span> of 20&#x02013;30 seedlings was measured for each treatment. C and D, Relative hypocotyl and <span class="trait">root</span> <span class="trait">length</span> in response to 200 mM carbon source in 8-day-old soybeans. Hypocotyl and <span class="trait">root</span> <span class="trait">length,</span> as a percentage of the untreated control, of seedlings grown on carbon source. Error bars represent the SE. Hypocotyl <span class="trait">length</span> without carbon source was WT (50.67&#x000b1;1.2 mm) and <italic> <span class="gene">GmRAV-i</span> </italic> (54.83&#x000b1;0.9 mm), and main <span class="trait">root</span> <span class="trait">length</span> was WT (67&#x000b1;1.2 mm) and <italic> <span class="gene">GmRAV-i</span> </italic> (79.83&#x000b1;0.7 mm). E, Relative number of lateral roots in response to 200 mM carbon source. Number of lateral roots, as a percentage of the untreated control, of seedlings grown on carbon source. Error bars represent the SE. The number of lateral roots, of seedlings grown without carbon source was WT (16.33&#x000b1;0.5) and <italic> <span class="gene">GmRAV-i</span> </italic> (21&#x000b1;1.0). Number of lateral roots of 20&#x02013;30 seedlings was scored for each treatment. *differences in comparison to the non-transgenic lines at 0.01&#x0003c;P&#x0003c;0.05, **Significant differences in comparison to the non-transgenic lines at P&#x0003c;0.01 (Student&#x02019;s t test).
					</p>
				
					<p>&nbsp;&nbsp;
						Carbohydrates are thought to play a crucial role in the regulation of flowering. The relation between sugar metabolism/signaling and floral transition received extensive attention lately <xref rid="pone.0089145-King1" ref-type="bibr"> [25] </xref> . Sugar signaling was of great importance in flowering time control, which directly affected <span class="trait">yield</span> <xref rid="pone.0089145-Amasino2" ref-type="bibr"> [38] </xref> , <xref rid="pone.0089145-Huang1" ref-type="bibr"> [39] </xref> . The work of Heyer et al. already provided clear evidence that flowering time control is strongly influenced by modifying sugar balances in the apex <xref rid="pone.0089145-Heyer1" ref-type="bibr"> [40] </xref> . Several sucrose signaling insensitive Arabidopsis mutants have been identified based on the effect of high levels of external sugars on seedling growth and <span class="trait">development</span> such as <italic> cai </italic> (carbohydrate insensitive) <xref rid="pone.0089145-Boxall1" ref-type="bibr"> [41] </xref> , <italic> isi </italic> (impaired sugar induction) <xref rid="pone.0089145-Rook1" ref-type="bibr"> [42] </xref> , <italic> lba </italic> (low levels of &#x003b2;-amylase) <xref rid="pone.0089145-Mita1" ref-type="bibr"> [43] </xref> , <italic> rsr </italic> (reduced sugar response) <xref rid="pone.0089145-Martin1" ref-type="bibr"> [44] </xref> , <italic> sis </italic> (sugar insensitive) <xref rid="pone.0089145-Laby1" ref-type="bibr"> [45] </xref> , <italic> sun </italic> (sucrose uncoupled) mutants <xref rid="pone.0089145-Dijkwel1" ref-type="bibr"> [46] </xref> . For example, <italic> sig </italic> (sucrose insensitive growth) mutant was selected on media containing 350 mM sucrose <xref rid="pone.0089145-Pego1" ref-type="bibr"> [47] </xref> . Similarly, Arabidopsis <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> which exhibited insensitive phenotypes were less inhibited by 200 mM sugars.
					</p>
				
					<p>&nbsp;&nbsp;
						To ensure uniform sugar responses, the initial mutant isolation screens were mostly performed on media containing high sucrose concentrations, raising concerns about the physiological relevance of sugar regulation. Moreover, the phenotypes could also be influenced by osmotic stress in the media. In this study, we treated Arabidopsis <italic> rav </italic> mutants, <italic> <span class="gene">GmRAV-ox</span> </italic> tobaccos and <italic> <span class="gene">GmRAV-i</span> </italic> soybeans with 200 mM supplemental carbon sources, and observed that transgenic plants and WT were almost uniformly inhibited by only supplemental sugar alcohols. A high concentration of sucrose added into basal medium also inhibited <span class="trait">development</span> both WT and transgenic plants. But we found the behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth. Thus, the specific effects of sucrose treatment could be attributed to the chemistry of the sucrose itself rather than to osmotic effects of the sucrose, which indicated <italic> <span class="gene">RAV</span> </italic> gene played a negative role in sucrose signaling due to metabolic effects rather than osmotic effects.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Protocol for augmented shoot organogenesis in selected variety of soybean [Glycine max L. (Merr.)].</h2>
			<h3>PMID:23214267  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Devi MK(1), Sakthivelu G, Giridhar P, Ravishankar GA.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Development</span> of a reproducible, versatile and efficient in vitro plant regeneration system is highly warranted for Indian soybean varieties for their mass multiplication in view of their commercial significance. Accordingly a protocol for direct shoot organogenesis in soybean variety JS 335 has been developed. Using cotyledonary node explants significant organogenic responses, mean shoot number and shoot <span class="trait">length</span> were observed when these were incubated on <span class="gene">MS</span> medium supplemented with 0.89 microM Benzyladenine (BA) and 5 microg/L triacontanol (TRIA) where in 9.3 +/- 0.5 shoots were obtained. TRIA at 5 microg/L able to produce 6.8 +/- 0.5 shoot buds in presence of 0.98 microM IBA and 0.89 microM BA. Highest mean shoot buds (14.0 +/- 0.5 and 9.0 +/- 0.5) and mean shoot <span class="trait">length</span> (4.6 +/- 0.3 and 10.0 +/- 0.7) were obtained when cotyledonary node and shoot tip explants were cultured on <span class="gene">MS</span> medium containing 0.14 microM gibberellic acid (GA3), 0.89 microM BA and 5 microg/L TRIA. Moreover, TRIA supported highest mean <span class="trait">root</span> number (6.3 +/- 0.5) and <span class="trait">root</span> <span class="trait">length</span> (21.5 +/- 0.57 cm). Field survival of in vitro derived plants of TRIA treatment was 70% and the overall growth and seed <span class="trait">yield</span> was also significantly better than control plants. This protocol may be used for improving the in vitro regeneration of soybean variety JS 335 for transformation studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome co-expression analysis of soybean cytochrome <span class="gene">P450</span> genes identifies nodulation-specific <span class="gene">P450</span> monooxygenases</h2>
			<h3>PMID:21062474</h3>
				

				
					<h3>PMCID:3095325</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Guttikonda </surname>  <given-names> Satish K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> SKGuttikonda@dow.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Trupti </surname>  <given-names> Joshi </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> joshitr@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Bisht </surname>  <given-names> Naveen C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> NBisht@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Chen </surname>  <given-names> Hui </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> huchen@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> An </surname>  <given-names> Yong-Qiang C </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> yong-qiang.an@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Pandey </surname>  <given-names> Sona </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> SPandey@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xudong@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Yu </surname>  <given-names> Oliver </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> oyu@danforthcenter.org </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 11 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Cytochrome <span class="gene">P450</span> monooxygenases (P450s) catalyze oxidation of various substrates using oxygen and NAD(P)H. Plant P450s are involved in the biosynthesis of primary and secondary metabolites performing diverse biological functions. The recent availability of the soybean genome sequence allows us to identify and analyze soybean putative P450s at a genome scale. Co-expression analysis using an available soybean microarray and Illumina sequencing data provides clues for functional annotation of these enzymes. This approach is based on the assumption that genes that have similar expression patterns across a set of conditions may have a functional relationship. RESULTS: We have identified a total number of 332 full-length <span class="gene">P450</span> genes and 378 pseudogenes from the soybean genome. From the full-length sequences, 195 genes belong to A-type, which could be further divided into 20 families. The remaining 137 genes belong to non-A type P450s and are classified into 28 families. A total of 178 probe sets were found to correspond to <span class="gene">P450</span> genes on the Affymetrix soybean array. Out of these probe sets, 108 represented single genes. Using the 28 publicly available microarray libraries that contain organ-specific information, some tissue-specific P450s were identified. Similarly, stress responsive soybean P450s were retrieved from 99 microarray soybean libraries. We also utilized Illumina transcriptome sequencing technology to analyze the expressions of all 332 soybean <span class="gene">P450</span> genes. This dataset contains total RNAs isolated from nodules, roots, <span class="trait">root</span> tips, leaves, flowers, green pods, apical meristem, mock-inoculated and Bradyrhizobium japonicum-infected <span class="trait">root</span> hair cells. The tissue-specific expression patterns of these <span class="gene">P450</span> genes were analyzed and the expression of a representative set of genes were confirmed by qRT-PCR. We performed the co-expression analysis on many of the 108 <span class="gene">P450</span> genes on the Affymetrix arrays. First we confirmed that CYP93C5 (an isoflavone synthase gene) is co-expressed with several genes encoding isoflavonoid-related metabolic enzymes. We then focused on nodulation-induced P450s and found that CYP728H1 was co-expressed with the genes involved in phenylpropanoid metabolism. Similarly, CYP736A34 was highly co-expressed with lipoxygenase, lectin and CYP83D1, all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development.</span> CONCLUSIONS: The genome scale analysis of P450s in soybean reveals many unique features of these important enzymes in this crop although the functions of most of them are largely unknown. Gene co-expression analysis proves to be a useful tool to infer the function of uncharacterized genes. Our work presented here could provide important leads toward functional genomics studies of soybean P450s and their regulatory network through the integration of reverse genetics, biochemistry, and metabolic profiling tools. The identification of nodule-specific P450s and their further exploitation may help us to better understand the intriguing process of soybean and rhizobium interaction.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We have identified a total number of 332 full-length <span class="gene">P450</span> genes and 378 pseudogenes from the soybean genome. From the full-length sequences, 195 genes belong to A-type, which could be further divided into 20 families. The remaining 137 genes belong to non-A type P450s and are classified into 28 families. A total of 178 probe sets were found to correspond to <span class="gene">P450</span> genes on the Affymetrix soybean array. Out of these probe sets, 108 represented single genes. Using the 28 publicly available microarray libraries that contain organ-specific information, some tissue-specific P450s were identified. Similarly, stress responsive soybean P450s were retrieved from 99 microarray soybean libraries. We also utilized Illumina transcriptome sequencing technology to analyze the expressions of all 332 soybean <span class="gene">P450</span> genes. This dataset contains total RNAs isolated from nodules, roots, <span class="trait">root</span> tips, leaves, flowers, green pods, apical meristem, mock-inoculated and <italic> Bradyrhizobium japonicum </italic> -infected <span class="trait">root</span> hair cells. The tissue-specific expression patterns of these <span class="gene">P450</span> genes were analyzed and the expression of a representative set of genes were confirmed by qRT-PCR. We performed the co-expression analysis on many of the 108 <span class="gene">P450</span> genes on the Affymetrix arrays. First we confirmed that <italic> CYP93C5 </italic> (an isoflavone synthase gene) is co-expressed with several genes encoding isoflavonoid-related metabolic enzymes. We then focused on nodulation-induced P450s and found that <italic> CYP728H1 </italic> was co-expressed with the genes involved in phenylpropanoid metabolism. Similarly, <italic> CYP736A34 </italic> was highly co-expressed with lipoxygenase, lectin and <italic> <span class="gene">CYP83D1</span> </italic> , all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Since membrane-bound enzymes are traditionally more difficult to study, few P450s have been functionally characterized. In the <italic> Arabidopsis </italic> genome, only 41 of the 245 coding sequences have been associated with a specific biochemical function [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Recently, a large-scale <span class="gene">P450</span> co-expression analysis with functional annotation of the 245 <italic> Arabidopsis </italic> <span class="gene">P450</span> genes was performed to predict the function of unknown <span class="gene">P450</span> genes [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. This co-expression analysis revealed expression patterns of the majority of <italic> Arabidopsis </italic> P450s, and provided novel clues on individual <span class="gene">P450</span> functions, pathways, and their regulatory networks. A novel phenolic pathway in pollen <span class="trait">development</span> was identified based on this co-expression analysis [ <xref ref-type="bibr" rid="B12"> 12 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, using <italic> in silico </italic> resources and bioinformatics tools, we identified and annotated putative functional <span class="gene">P450</span> encoding sequences in the soybean genome. Phylogenetic analysis using amino acid sequences allowed us to identify gene orthologs and clusters of orthologous groups for further characterizations. By analyzing large-scale microarray and Illumina sequencing data, we also analyzed the co-expression of <span class="gene">P450</span> genes in soybean, which could provide important clues to their function. We were able to identify tissue-specific <span class="gene">P450</span> genes that may play roles in biological process like nodulation, floral <span class="trait">development,</span> and seed maturation. The large-scale expression analyses were confirmed for selected <span class="gene">P450</span> genes using quantitative real-time reverse-transcription polymerase chain reaction (qRT-PCR).
					</p>
				
					<p>&nbsp;&nbsp;
						To obtain expression profiles of soybean <span class="gene">P450</span> genes, we first utilized the extensive Affymetrix array data publically available at the NCBI database. To identify <span class="gene">P450</span> probe sets on the Affymetrix soybean array (Part #900526), we performed BLASTN searches using the sequences of each probe set against the predicted soybean cDNAs at the TIGR gene index and Phytozome 4.0 genomic sequences. A total of 178 probe sets were found to correspond to <span class="gene">P450</span> genes. Out of these probe sets, 108 represented single genes (at least 100 bp in matching <span class="trait">length);</span> the rest matched with more than one gene (55 of them matched with more than two genes). Since multiple targets complicate interpretation of the expression profiles, we focused on the 108 single-target genes for further analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						There are 28 microarray libraries that contain organ-specific information in the public database. Following standard microarray data analysis, we identified highly tissue-specific soybean <span class="gene">P450</span> genes (with a cut-off &#x0003e; 3 for up-regulated genes and &#x0003c; &#x02212;3 for down regulated genes). The data are summarized in Figures <xref ref-type="fig" rid="F2"> 2 </xref> , <xref ref-type="fig" rid="F3"> 3 </xref> , <xref ref-type="fig" rid="F4"> 4 </xref> and Additional File <xref ref-type="supplementary-material" rid="S3"> 3 </xref> , Table S2. The <span class="gene">P450</span> genes highly induced in leaves, roots, hypocotyl, seeds and axillary meristem are apparent (Figures <xref ref-type="fig" rid="F2"> 2 </xref> and <xref ref-type="fig" rid="F3"> 3 </xref> ). For example, out of 108 <span class="gene">P450</span> genes, seven are highly induced and one significantly repressed in the axillary meristem tissues when all genes expressed in all tissues were averaged as control. Each tissue has its own unique set of differentially expressed P450s. Interestingly, <italic> <span class="gene">CYP93C1</span> (IFS1) </italic> and <italic> CYP93C5 (IFS2) </italic> were highly expressed in roots and seeds. More specifically, <italic> <span class="gene">IFS1</span> </italic> is mainly expressed in the <span class="trait">root</span> and seed coat; while <italic> <span class="gene">IFS2</span> </italic> is mainly expressed in embryos and pods. These results agree with previous publications using Northern blot analysis, qRT-PCR, and promoter:GUS transgene assays [ <xref ref-type="bibr" rid="B33"> 33 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						There are 99 microarray libraries in the public database from various stress treatments of soybean. As summarized in Figure <xref ref-type="fig" rid="F4"> 4 </xref> , the genes <italic> CYP736A28, CYP93E1, CYP82A4, CYP94C18 </italic> and <italic> <span class="gene">CYP81E11</span> </italic> are highly induced in soybean cyst nematode infection. Similarly, <italic> CYP71D9, CYP736A33, CYP72A128, CYP81E28 </italic> , and <italic> CYP81E21 </italic> showed strong induction in hypocotyls infected with <italic> Phytophthora Sojae </italic> . Interestingly, <italic> CYP707A51 </italic> and <italic> CYP710A23 </italic> are highly expressed in soybean leaves treated with Nod factor from <italic> B. japonicum </italic> compared to untreated leaves, suggesting they may be involved in the auto-regulation process in which shoots synthesize a mobile signal to suppress additional nodulation in the <span class="trait">root</span> after initial rhizobial infection. Under abiotic stress conditions, <italic> <span class="gene">CYP71D9</span> </italic> and <italic> <span class="gene">CYP83D1</span> </italic> were highly expressed under iron deficiency compared to plants grown under sufficient iron. The exact functions of these two genes have not been reported before.
					</p>
				
					<p>&nbsp;&nbsp;
						Given the limitations of the current soybean Affymetrix DNA microarray platform, we also utilized Illumina transcriptome sequencing technology to analyze all 332 soybean <span class="gene">P450</span> gene expressions, using published data from Libault <italic> et al </italic> .[ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. These data sets were generated from total RNAs isolated from nodules, roots, <span class="trait">root</span> tips, leaves, flowers, green pods, apical meristem, mock-inoculated and <italic> B. japonicum </italic> -infected <span class="trait">root</span> hair cells harvested at 12, 24 and 48 h after inoculation [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. For comparison, total RNA extracted from stripped roots (i.e. roots devoid of <span class="trait">root</span> hairs) were also harvested and sequenced at 48 h after inoculation with <italic> B. japonicum </italic> . Illumina transcriptome analysis allowed a more thorough assessment of all of the 332 soybean <span class="gene">P450</span> genes. Additionally, since Affymetrix array analysis is considered to be less sensitive when comparing different experimental set ups, especially for low abundant RNAs, this deep transcriptome sequence analysis allowed a clear distinction between low-abundance and undetectable transcripts.
					</p>
				
					<p>&nbsp;&nbsp;
						From these Illumina data, the <span class="gene">P450</span> genes highly expressed in different tissue were hierarchically clustered using the MeV v4.5.1 software. The entire expression profile in all tissues and treatments is shown in Additional File <xref ref-type="supplementary-material" rid="S5"> 5 </xref> , Figure S2. To simplify the figure, the expression of P450s involved in isoflavone biosynthesis, CYP93 family genes and <span class="gene">P450</span> families present only in soybean and not in <italic> Arabidopsis </italic> , including CYP92, CYP733, CYP735 and CYP736 have been highlighted (Figures <xref ref-type="fig" rid="F5"> 5 </xref> and <xref ref-type="fig" rid="F6"> 6 </xref> ). In Figure <xref ref-type="fig" rid="F5"> 5 </xref> , very different expression profiles in apical meristem, green pods, nodules, roots, flowers, leaves, and <span class="trait">root</span> tips can be seen among selected P450s. For example, CYP733A members were not detected in any of the tissues, suggesting a more narrow distribution beyond our collection of cDNA libraries. In Figure <xref ref-type="fig" rid="F6"> 6 </xref> , <span class="trait">root</span> hairs and stripped roots were profiled with or without rhizobia induction. Isoflavone biosynthetic enzymes were expressed in all tissues and treatments, in contrast with CYP733A and CYP83 family where no expression could be detected in any tissues. Interestingly, <italic> CYP88A3 </italic> appeared to be expressed only in the <span class="trait">root</span> hairs, not in the roots stripped of <span class="trait">root</span> hairs. Similarly, <italic> CYP736A31 </italic> and <italic> CYP736A32 </italic> exhibit much higher expression in the <span class="trait">root</span> hairs than in the stripped roots. In contrast, the same family <italic> CYP736A47 </italic> and <italic> CYP736A48 </italic> had an opposite expression pattern, with significantly higher expression in roots than in <span class="trait">root</span> hairs.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of soybean <span class="gene">P450</span> genes in <span class="trait">root</span> hair infection process according to the analysis of Illumina transcriptome dataset. </bold> The labels are 24HA1_UN_RH, <span class="trait">root</span> hair sample 24 hours after mock treatment; 12HA1_IN_RH, <span class="trait">root</span> hair sample 12 hours after infection; 24HA1_IN_RH, <span class="trait">root</span> hair sample 24 hours after infection; 12HA1_UN_RH, <span class="trait">root</span> hair sample 12 hours after mock treatment; 48HA1_UN_RH, <span class="trait">root</span> hair sample 48 hours after mock treatment 48HA1_IN_RH, <span class="trait">root</span> hair sample 48 hours after infection; 48HA1_Script_Root, stripped <span class="trait">root</span> sample 48 hours after infection.
					</p>
				
					<p>&nbsp;&nbsp;
						Of all <span class="gene">P450</span> genes analyzed, the ones highly expressed in nodules, roots, leaves and flowers are summarized in Table <xref ref-type="table" rid="T3"> 3 </xref> . The most abundant <span class="gene">P450</span> genes in each tissue, as shown by the calculated tag sequence numbers, are <italic> CYP83G4 </italic> (nodules), <italic> CYP76X6 </italic> (apical meristem, green pods and leaves), <italic> CYP83E12 </italic> (flowers), and <italic> CYP93C5 </italic> (roots and <span class="trait">root</span> tips). Clearly, <italic> CYP76X6 </italic> is the most abundant <span class="gene">P450</span> in green tissues (function unknown); while the isoflavone synthetic enzyme <span class="gene">IFS</span> is the major <span class="gene">P450</span> in roots.
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm these findings from Illumina sequencing, we carried out qRT-PCR analysis on the four nodule-specific genes and a set of selected other <span class="gene">P450</span> genes. Total RNAs were isolated from hypocotyls, roots, leaves, flowers, seed (stage R8), and dissected nodules. The expression levels of the above-mentioned <italic> CYP78A71, CYP83G4, CYP71A44 </italic> , and <italic> CYP82D30 </italic> were measured. All four genes showed dominant nodule-specific expression, consistent with the Illumina data. Two additional genes, <italic> CYP81E12 </italic> and <italic> CYP736A34 </italic> , showed strong <span class="trait">root</span> and nodule expression, also fitting the Illumina data very well. All six genes on Figure <xref ref-type="fig" rid="F7"> 7 </xref> had lower expression in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> qRT-PCR confirmation of soybean <span class="gene">P450</span> genes highly expressed in <span class="trait">root</span> and nodule as revealed by the analysis of Microarray and Solexa transcriptome datasets. </bold> Error bars represent SE (Standard error) of three replications.
					</p>
				
					<p>&nbsp;&nbsp;
						We then performed the co-expression analysis on many of the 108 <span class="gene">P450</span> genes on the Affymetrix arrays (Table <xref ref-type="table" rid="T4"> 4 </xref> and Additional File <xref ref-type="supplementary-material" rid="S6"> 6 </xref> , Table S4). We selected three examples and highlight them in Table <xref ref-type="table" rid="T4"> 4 </xref> . The <italic> CYP728H1 </italic> showed high expression in roots and nodules, and was co-expressed with 4-coumarate:CoA ligase isoenzyme 2, cytochrome <span class="gene">P450</span> 98A2 and 4- coumarate:coenzyme A ligase genes which are involved in phenylpropanoid metabolism. <italic> CYP728H1 </italic> is also co-expressed with endo-xyloglucan transferase, NAC transcription factor, 2'-hydroxy isoflavone and early nodulin genes. Gene expression and co-expression analysis suggests that <italic> CYP728H1 </italic> may play an important role in isoflavone metabolism, as well as during <span class="trait">root</span> and nodule <span class="trait">development.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The CYP736 family is present in soybean but not in <italic> Arabidopsis </italic> . <italic> CYP736A34 </italic> showed high expression in roots and nodules (Figure <xref ref-type="fig" rid="F7"> 7 </xref> ). Co-expression analysis showed that the expression of this gene is highly correlated with lipoxygenase, lectin and <italic> <span class="gene">CYP83D1</span> </italic> , all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> ). Some of the co-expressed genes are defense related genes such as cysteine proteinase. Since defense response is one of the early nodulation events, <italic> CYP736A34 </italic> may be functioning at the early stages of this symbiotic process.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is an important leguminous crop. With the advent of soybean genome sequencing, it is possible to study large gene families of soybean. We targeted one of the most challenging families in plants, the cytochrome <span class="gene">P450</span> genes, and investigated their genetic make-up, gene distributions, expression profiles, and co-expressed associates. Cytochrome P450s are indispensable for soybean growth, <span class="trait">development,</span> and defense against pathogens. They may play important roles for soybean symbiotic interaction with rhizobacteria. Overall, we identified 332 full-length <span class="gene">P450</span> genes and 378 pseudogenes in the genome. We used publicly available microarray libraries and identified few tissue-specific and stress responsive soybean P450s. The tissue-specific expression patterns of some <span class="gene">P450</span> genes were confirmed by qRT-PCR. In addition, the expressions patterns of all 332 soybean <span class="gene">P450</span> genes were obtained through the analysis of Illumina transcriptome datasets. The co-expression analysis on some of the <span class="gene">P450</span> genes was performed using the Affymetrix array datasets. We demonstrate that gene co-expression analysis is a useful tool to guide our further study on the function of uncharacterized genes. Importantly, the identification of nodule-specific P450s and their further exploitation may help us to uncover the intriguing process of soybean and rhizobium interaction.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Figure S2 Gene expression of 332 soybean <span class="gene">P450</span> genes according to the analysis of Illumina transcriptome dataset </bold> . Gene expression in different organ and <span class="trait">root</span> hair inoculation process.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The soybean root-specific protein kinase <span class="gene">GmWNK1</span> regulates stress-responsive ABA signaling on the <span class="trait">root</span> system architecture.</h2>
			<h3>PMID:20735771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang Y(1), Suo H, Zheng Y, Liu K, Zhuang C, Kahle KT, Ma H, Yan X.
				<br/> 
				年份：2010 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In humans, members of the WNK protein kinase family are osmosensitive regulators of cell volume homeostasis and epithelial ion transport, and mutation of these proteins causes a rare inherited form of hypertension due to increased renal NaCl re-absorption. A related class of kinases was recently discovered in plants, but their functions are largely unknown. We have identified a root-specific WNK kinase homolog, <span class="gene">GmWNK1,</span> in soybean (Glycine max). <span class="gene">GmWNK1</span> expression was detected in the <span class="trait">root,</span> specifically in <span class="trait">root</span> cells associated with lateral <span class="trait">root</span> formation, and was down-regulated by abscisic acid (ABA), as well as by mannitol, sucrose, polyethylene glycol and NaCl. In vitro and in vivo experiments showed that <span class="gene">GmWNK1</span> interacts with another soybean protein, <span class="gene">GmCYP707A1,</span> which is a key ABA 8'-hydroxylase that functions in ABA catabolism. Furthermore, 35S-GmWNK1 transgenic soybean plants had reduced lateral <span class="trait">root</span> number and <span class="trait">length</span> compared with wild-type, suggesting a role of <span class="gene">GmWNK1</span> in the regulation of <span class="trait">root</span> system architecture. We propose that <span class="gene">GmWNK1</span> functions to fine-tune ABA-dependent ABA homeostasis, thereby mediating the regulation of the <span class="trait">root</span> system architecture by ABA and osmotic signals. The study has revealed a new function of a plant <span class="gene">WNK1</span> gene from the important staple crop soybean, and has identified a new component of a regulatory pathway that is involved not only in ABA signaling, but also in the repression of lateral <span class="trait">root</span> formation by an ABA-dependent mechanism distinct from known ABA signaling pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-seq analyses of multiple meristems of soybean: novel and alternative transcripts, evolutionary and functional implications</h2>
			<h3>PMID:24939556</h3>
				
					<h3>PMCID:4070088</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Wang </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wangleibio@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Cao </surname>  <given-names> Chenlong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> 11210700144@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Ma </surname>  <given-names> Qibin </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> maqibin@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Zeng </surname>  <given-names> Qiaoying </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> zengqiaoying123@163.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Wang </surname>  <given-names> Haifeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> whfwind@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Cheng </surname>  <given-names> Zhihao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhihaocheng1@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhu </surname>  <given-names> Genfeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> gfzhu@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Qi </surname>  <given-names> Ji </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> qij@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Ma </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> hongma@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> hnian@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Wang </surname>  <given-names> Yingxiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yx_wang@fudan.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We also summarized the possible existence of 11 AS types in soybean, including four common types of intron retention (32.2%), ES (26.3%), A3SS (20.8%), A5SS (11.2%) (Table&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> ). Unlike the major type of ES in animals [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ], intron retention was the major type of AS in soybean, consistent with the observations in <italic> Arabidopsis </italic> , rice, maize and soybean [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B20"> 20 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B40"> 40 </xref> ]. Our result and those from others suggest that the mechanism for regulation of IR in plants is conserved. The higher proportion of ES events (26.3%) in soybean is also in agreement with that in rice and maize [ <xref ref-type="bibr" rid="B19"> 19 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> ], but significantly higher than that in <italic> Arabidopsis </italic> . ES has been reported to be involved in regulating tissue-specific functions [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. To investigate the tissue-specific expression of AS, we performed a MISO program analysis [ <xref ref-type="bibr" rid="B41"> 41 </xref> ] to identify 202 tissue-bias exon skipped events, including 2 paralogs (in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S11). Most of them encode enzymes and transcription factors that are enriched for protein degradation, RNA regulation, signaling and transport. We also found that several exons are recognized predominantly as exons in one tissue and also as introns in another tissue. For example, as shown in Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> a, the 7th exon of <italic> <span class="gene">Gm15g15960</span> </italic> , showed &#x003a8; with 88% in <span class="trait">root</span> tip and 6% in cotyledon (in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S11), suggesting divergent functions between <span class="trait">root</span> tip and cotyledon. In addition, 1,834 AS events changed greatly during SAM and flower <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> b and in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S12), GO analysis indicated that many genes encoding proteins participate in the reproductive <span class="trait">development</span> process. In addition to known flowering genes exhibiting AS changes, many uncharacterized genes were also observed to have significant AS changes, as exemplified by <italic> <span class="gene">Gm05g28120</span> </italic> , a gene with three sets of exons with mutually exclusive expression patterns (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> c).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Tissue-bias and developmentally regulated splicing events. (a) </bold> Scatter plot shows Y values of skipped exons, and switch score was determined on the basis of comparison of <span class="trait">root</span> tip (x-axis) with a second tissue (y-axis). Exons with a switch score 0.7 were shown as filled symbols; others were shown as small grey dots. <bold> (b) </bold> Heatmap showed the regulated alternative splicing events during SAM <span class="trait">development.</span> The scale bar indicates Z-scores of &#x003a8;. <bold> (c) </bold> Alternative splicing in <italic> <span class="gene">Gm05g28120</span> </italic> was regulated during SAM <span class="trait">development.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Characterization of tissue or cell-specific genes provides a foundation for unraveling their molecular mechanisms. Previous studies in multiple plants demonstrated that each organ or tissue has its specific transcripts [ <xref ref-type="bibr" rid="B18"> 18 </xref> , <xref ref-type="bibr" rid="B21"> 21 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> ], including soybean [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. These genes expressed more highly in one tissue (or a closely related set of tissues) than all other tissues examined are referred to as preferentially expressed genes (PEGs). To investigate the tissue PEGs, we first compared the transcriptomes among 11 tissues and found 6,557 tissue PEGs (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> ). Among these genes, <span class="trait">root</span> tips had 769 PEGs, including 65 paralogs; GO annotation showed that they were related to translational elongation, hormone signaling, cytokinin stimulus, stem cell maintenance and post-embryonic <span class="trait">root</span> <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S15 and S16). In <italic> Arabidopsis </italic> , <italic> PIN2 </italic> is specifically required for auxin transport during <span class="trait">root</span> <span class="trait">development</span> [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Two paralogs similar to <italic> PIN2 </italic> were found in soybean and showed similarly specific expression in <span class="trait">root</span> tips, suggesting that they have possibly redundant function in <span class="trait">root</span> <span class="trait">development</span> similar to that of <italic> PIN2 </italic> in <italic> Arabidopsis </italic> . In contrast, 1,053 PEGs identified in callus, including 102 transcription factors and 48 paralogs, were mainly involved in biotic and abiotic responses, such as defense, oxidative stress, vitamin, inorganic substance and cytokinin stimulus (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S17 and S18). The 762 cotyledon PEGs (49 paralogs) were enriched in photosynthesis, energy, transmitting tissue <span class="trait">development</span> and glucose metabolism (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S19 and S20). Auxin is a crucial regulator of cotyledon <span class="trait">development</span> [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. We detected several other auxin-related genes, including the pair of <italic> <span class="gene">Gm09g38700</span> </italic> and <italic> <span class="gene">Glyma18g47630</span> </italic> paralogs that are homologs of <italic> Arabidopsis PIN-FORMED 5 (PIN5) </italic> , which is required for auxin homoeostasis and gametophyte <span class="trait">development</span> [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. However, both genes were found with highest expression in cotyledon, but nearly undetectable in reproductive tissues, suggesting <italic> <span class="gene">PIN5</span> </italic> may have a divergent role in soybean. In contrast, the 539 hypocotyl PEGs (27 paralogs) were enriched for an auxin-mediated signaling pathway, and/or photo morphogenesis (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S21 and S22), including homologs of the <italic> Arabidopsis NON-PHOTOTROPIC HYPOCOTYL 3 (NPH3) </italic> gene [ <xref ref-type="bibr" rid="B49"> 49 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> , AM was highly similar to both SAM38D and IBM, pairwise comparison would probably miss many genes active in meristems. To identify PEGs in these meristems (but not specifically in one of them), we grouped similar meristems together and detected 821 genes (20 paralogs). GO annotation indicated that the most enriched categories were associated with flower <span class="trait">development</span> and regulation, floral transition from vegetative to reproductive phase, or meristematic phase transition and transcription regulation (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S23 and S24), which is in good agreement with previous reports in soybean [ <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ]. For instance, the PEGs included several homologs of <italic> SHORT VEGETATIVE PHASE </italic> ( <italic> SVP </italic> ) that specify the reproductive organ identity and control flowering time in <italic> Arabidopsis </italic> and rice [ <xref ref-type="bibr" rid="B52"> 52 </xref> , <xref ref-type="bibr" rid="B53"> 53 </xref> ] and genes involvement in <italic> WUSCHEL </italic> ( <italic> WUS </italic> ) regulatory network essential for SAM maintenance [ <xref ref-type="bibr" rid="B54"> 54 </xref> ]. We also found homologs ( <italic> <span class="gene">Gm14g15820</span> </italic> and <italic> <span class="gene">Gm7g30920</span> </italic> ) of genes for auxin synthesis and response, such as <italic> YUCCA4 </italic> , in accordance with the fact that the <italic> Arabidopsis YUCCA4 </italic> expression is restricted to the SAM and flower meristems or young floral primordia [ <xref ref-type="bibr" rid="B55"> 55 </xref> ], as well as 20 genes related to auxin-responsive genes regulating SAM <span class="trait">development</span> [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]. These good agreements between our GO enrichment results and known functions in meristem suggested the reliability of the collected samples for SAM and conservation of molecular mechanisms for controlling SAM between <italic> Arabidopsis </italic> and soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Accordingly, AM, IBM and IAM together had 1,325 PEGs (60 paralogs) (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> , in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S25), which were mainly involved in reproductive processes, such as floral organ determination and <span class="trait">development,</span> stamen <span class="trait">development,</span> tapetal layer <span class="trait">development,</span> pollen <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F5"> 5 </xref> ). For instance, in addition to the identification of several flower organ identity genes from ABC model (in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S26), we also found genes specifically for meiosis, such as <italic> MS5 </italic> ( <italic> <span class="gene">Gm08g47070</span> </italic> and <italic> <span class="gene">Gm18g38060</span> </italic> ) and <italic> MMD1 </italic> ( <italic> <span class="gene">Gm14g39310</span> </italic> and <italic> <span class="gene">Gm02g41020</span> </italic> ) [ <xref ref-type="bibr" rid="B57"> 57 </xref> ] <italic> . </italic> Unlike the expression of <italic> Arabidopsis MS5 </italic> and <italic> MMD1 </italic> genes restricted in meiocytes, the soybean homologs showed high expression in AM, suggesting a possible unknown function in soybean. Interestingly, the <italic> Arabidopsis DREB1B </italic> is one of the critical regulators for cold responses, and is also widely expressed [ <xref ref-type="bibr" rid="B58"> 58 </xref> ], whereas the soybean homologs ( <italic> <span class="gene">Gm11g19340</span> </italic> and <italic> <span class="gene">Gm12g09130</span> </italic> ) showed special expression in AM, IBM and IAM, but not in other vegetative tissues, suggesting it might have gained novel functions in reproductive <span class="trait">development</span> in soybean. In addition, one homolog of <italic> DREB1A </italic> ( <italic> <span class="gene">Gm17g14110</span> </italic> ) was also identified, consistent with a recent novel discovery that the <italic> Arabidopsis DREB1A </italic> gene is important for flower <span class="trait">development</span> especially under unfavorable conditions [ <xref ref-type="bibr" rid="B59"> 59 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Further classification of family-preferential expression showed that G1 mainly includes families of HMG, FHA, ZF-HD, SBP, TCP, C2C2 (Zn) <span class="gene">GATA</span> and PHD (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), indicating that early SAM <span class="trait">development</span> largely requires those transcription factor families. For example, <italic> SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) </italic> proteins are a family of plant-specific TFs having a conserved <span class="gene">SBP</span> ( <italic> SQUAMOSA </italic> promoter binding protein) domain, and play multiple roles in plant growth and <span class="trait">development</span> [ <xref ref-type="bibr" rid="B63"> 63 </xref> ]. 16 and 48 <italic> SPLs </italic> are found in <italic> Arabidopsis </italic> and soybean, respectively [ <xref ref-type="bibr" rid="B64"> 64 </xref> ], and were divided into eight clades (in Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S5 and in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Tables S34-S35). 23 <italic> SPLs </italic> from 7 clades were differentially expressed during soybean SAM and flower <span class="trait">development.</span> Available data from <italic> Arabidopsis </italic> , rice and tomato support the idea that the function of genes from some different clades might still be conserved, but genes from other clades might have diverged [ <xref ref-type="bibr" rid="B63"> 63 </xref> - <xref ref-type="bibr" rid="B65"> 65 </xref> ]. For instance, 10 of 16 <italic> Arabidopsis SPLs </italic> ( <italic> SPL2-6, SPL9-11, SPL13, </italic> and <italic> SPL15 </italic> ) from 5 clades are <italic> miR156/157 </italic> targets [ <xref ref-type="bibr" rid="B66"> 66 </xref> ], and play a similar role in phase transition [ <xref ref-type="bibr" rid="B65"> 65 </xref> ], whereas the clade I-, II- and III-containing genes lack <italic> miR156 </italic> and <italic> miR157 </italic> -binding sites. The clade I contains only <italic> SPL7 </italic> with ubiquitous expression and distinct function in regulating copper homeostasis [ <xref ref-type="bibr" rid="B67"> 67 </xref> ]. Consistently, two soybean <italic> SPL7 </italic> paralogs are also widely expressed with similar patterns, suggesting a conserved role in soybean (Figure&#x000a0; <xref ref-type="fig" rid="F8"> 8 </xref> ). Clade II has four members of <italic> SPL1, SPL12, SPL14 </italic> and <italic> SPL16 </italic> with wide expression in <italic> Arabidopsis </italic> (Figure&#x000a0; <xref ref-type="fig" rid="F8"> 8 </xref> ), but only <italic> AtSPL14 </italic> has been shown to participate in vegetative to reproductive transition [ <xref ref-type="bibr" rid="B63"> 63 </xref> ]. This clade includes eight members in soybean. Seven of them showed similar expression patterns to that of <italic> Arabidopsis </italic> (Figure&#x000a0; <xref ref-type="fig" rid="F8"> 8 </xref> and in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S36), but only <italic> <span class="gene">Gm17g04400</span> </italic> was differentially expressed in SAM, suggesting a function different from that of its counterpart. The clade III has only <italic> AtSPL8 </italic> with a function in <span class="trait">root</span> growth and microsporogenesis [ <xref ref-type="bibr" rid="B68"> 68 </xref> ]. Four soybean genes were found in this clade, and two of them had no detectable expression, suggesting possible non-functionalization. By contrast, the other two genes were specifically expressed in SAM and reproductive tissues. However, unlike <italic> AtSPL8 </italic> being functional in roots, the soybean homologs were not expressed in roots, resembling that of tomato <italic> SPL8 </italic> homologs [ <xref ref-type="bibr" rid="B64"> 64 </xref> ]. The clade IV contains <italic> AtSPL6 </italic> with constitutive expression and unknown function <italic> . </italic> However, a <italic> Physcomitrella patens </italic> homolog has been reported to repress reproductive <span class="trait">development</span> [ <xref ref-type="bibr" rid="B69"> 69 </xref> ], somehow similar to <italic> AtSPL14 </italic> [ <xref ref-type="bibr" rid="B70"> 70 </xref> ]. This clade had six soybean homologs; except for the undetectable expression of <italic> <span class="gene">Gm06g22450</span> </italic> , the other five genes were highly expressed in SAM and reproductive tissues. In comparison to the other 7 clades having more genes from soybean than <italic> Arabidopsis </italic> , the clade V has three Arabidopsis genes, but only two paralogs from soybean (Figure&#x000a0; <xref ref-type="fig" rid="F8"> 8 </xref> ). Therefore, it would be interesting to investigate possible reasons to cause gene lost on soybean in this clade.
					</p>
				
					<p>&nbsp;&nbsp;
						Different from G1, G2 mainly contained MADS, AS, BTB/POZ, WRKY, C2C2 (Zn) YABBY (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b). It has been reported that MADS-box gene family is not only key repressors or activators for flowering transition, but also as master regulators of reproductive organ identities [ <xref ref-type="bibr" rid="B74"> 74 </xref> ]. Our data detected 101 MADS-box genes during flower <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), such as <italic> <span class="gene">Gm01g08150,</span> <span class="gene">Gm04g42420,</span> <span class="gene">Gm08g12730</span> </italic> and <italic> <span class="gene">Gm08g27670</span> </italic> , which are homologs of <italic> AP1, PI, AG </italic> and <italic> SEP2 </italic> , respectively, consistent with their known function in floral organ identity. Therefore, the functions of the MADS-box gene family for regulation of flower <span class="trait">development</span> are likely conserved between soybean and <italic> Arabidopsis </italic> . In contrast, BZIP, C3H-type1 (Zn), C2H2 (Zn) Dof, AUX-IAA-ARF, LIM and CCAAT gene families were preferentially expressed in OF (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b). Many studies showed that auxin-dependent transcriptional regulation requires the auxin/indole-3-acetic acid (Aux/IAA) and auxin response factor (ARF) families of TFs [ <xref ref-type="bibr" rid="B75"> 75 </xref> ] and formation of Aux/IAA-ARFs heterodimers repress auxin signaling [ <xref ref-type="bibr" rid="B75"> 75 </xref> ]. In addition to the known role of auxin in <italic> Arabidopsis </italic> pollen <span class="trait">development,</span> pollination and fertilization also seem to need increased auxin levels [ <xref ref-type="bibr" rid="B76"> 76 </xref> ]. Indeed, we detected 33 differentially expressed members in OF, suggesting Aux/IAA-ARF regulatory pathway for later reproductive <span class="trait">development</span> is also conserved. However, the function of other enriched TFs in OF is still largely unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Table contents for S1 to S37. Table S1. </bold> Summary of mapped reads. <bold> Table S2. </bold> Distribution of reads among genome. <bold> Table S3. </bold> The <italic> t </italic> -test of 8768 paralogs. <bold> Table S4. </bold> List of 1509 nTRs in the 5&#x02019;UTR upstream of genes. <bold> Table S5. </bold> Distribution of 4949 nTUs on chromosomes. <bold> Table S6. </bold> List of 2326 nTUs and annotated genes in NCBI. <bold> Table S7. </bold> Go annotation of 698 nTUs. <bold> Table S8. </bold> The 47 nTUs in TE. <bold> Table S9. </bold> List of 40 nTUs. <bold> Table S10. </bold> List of AS events. <bold> Table S11. </bold> The description of 202 skipped exon gene. <bold> Table S12. </bold> The Psi value of 1834 transcripts. <bold> Table S13. </bold> Correlation matrix of the whole dataset. <bold> Table S14. </bold> GO annotation of overlapped genes between AM and IBM. <bold> Table S15. </bold> List of <span class="trait">root</span> tip PEGs. <bold> Table S16. </bold> GO annotation of <span class="trait">root</span> tip PEGs. <bold> Tables S17 and S18. </bold> List and GO annotation of callus PEGs. <bold> Tables S19 and S20. </bold> List and GO annotation of cotyledon PEGs. <bold> Tables S21 and S22. </bold> List and GO annotation of hypocotyl PEGs. <bold> Table S23. </bold> List of PEGs in multiple meristems. <bold> Table S24. </bold> GO annotation of PEGs in multiple meristems. <bold> Table S25. </bold> List of the PEGs in AM, IBM and IAM. <bold> Table S26. </bold> GO annotation of the PEGs in AM, IBM and IAM. <bold> Table S27. </bold> List of the OF PEGs. <bold> Table S28. </bold> GO annotation of the OF PEGs. <bold> Table S29. </bold> List of the PEGs among 6 SAM samples. <bold> Table S30. </bold> Correlation of mRNA-seq and qRT-PCR. <bold> Table S31. </bold> RPKM value and function description of eight clusters. <bold> Table S32. </bold> P value of each GO term. <bold> Table S33. </bold> Distribution of TF families among G1, G2 and G3. <bold> Table S34. </bold> Phylogenetic analysis of <italic> SBP-box </italic> family. <bold> Table S35. </bold> SBP-domain sequences and accession numbers of plant <italic> <span class="gene">SBP</span> </italic> s. <bold> Table S36. </bold> Expression of 48 soybean <italic> <span class="gene">SBP</span> </italic> s. <bold> Table S37. </bold> List of RT-PCR primers.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in high salt and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						YABBY family is a plant specific transcription factor family, with the typical N-terminal C2C2 type zinc finger domain and the C-terminal YABBY conservative structure domain, which plays important biological roles in plant growth, <span class="trait">development</span> and morphogenesis. In this study, a total of 17 YABBY genes were identified in the soybean genome. The results of this research showed that 17 soybean YABBY genes were located on 11 chromosomes. Analysis of putative cis-acting elements showed that soybean YABBY genes contained lots of MYB and MYC elements. Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses. And the transient expression in Arabidopsis protoplasts showed that <span class="gene">GmYABBY3</span> protein distributed uniformly the whole cells, while <span class="gene">GmYABBY10</span> protein was mainly localized in the membranes and cytoplasm and <span class="gene">GmYABBY16</span> protein was localized the nucleus and membranes. To further identify the function of <span class="gene">GmYABBY10,</span> we obtained the transgenic Arabidopsis overexpression <span class="gene">GmYABBY10.</span> Based on germination and seedling <span class="trait">root</span> arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment. While the <span class="trait">root</span> <span class="trait">length</span> and <span class="trait">root</span> surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings. When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and salt stresses. This study provided valuable information regarding the classification and functions of YABBY genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean Intrinsic Protein Gene, <span class="gene">GmTIP2;3,</span> Involved in Responding to Osmotic Stress.</h2>
			<h3>PMID:26779248</h3>
				

				
			<p>
				作者：Zhang D(1), Tong J(2), He X(1), Xu Z(1), Xu L(1), Wei P(1), Huang Y(1), Brestic M(3), Ma H(1), Shao H(4).
				<br/> 
				年份：2016 Jan 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water is essential for plant growth and <span class="trait">development.</span> Water deficiency leads to loss of <span class="trait">yield</span> and decreased crop quality. To understand water transport mechanisms in plants, we cloned and characterized a novel tonoplast intrinsic protein (TIP) gene from soybean with the highest similarity to TIP2-type from other plants, and thus designated <span class="gene">GmTIP2;3.</span> The protein sequence contains two conserved NPA motifs and six transmembrane domains. The expression analysis indicated that this gene was constitutively expressed in all detected tissues, with higher levels in the <span class="trait">root,</span> stem and pod, and the accumulation of <span class="gene">GmTIP2;3</span> transcript showed a significant response to osmotic stresses, including 20% PEG6000 (polyethylene glycol) and 100 μM ABA (abscisic acid) treatments. The promoter-GUS (glucuronidase) activity analysis suggested that <span class="gene">GmTIP2;3</span> was also expressed in the <span class="trait">root,</span> stem, and leaf, and preferentially expressed in the stele of <span class="trait">root</span> and stem, and the core promoter region was 1000 bp in <span class="trait">length,</span> located upstream of the ATG start codon. The GUS tissue and induced expression observations were consistent with the findings in soybean. In addition, subcellular localization showed that <span class="gene">GmTIP2;3</span> was a plasma membrane-localized protein. Yeast heterologous expression revealed that <span class="gene">GmTIP2;3</span> could improve tolerance to osmotic stress in yeast cells. Integrating these results, <span class="gene">GmTIP2;3</span> might play an important role in response to osmotic stress in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPTF1, a novel transcription factor gene, is involved in conferring soybean tolerance to phosphate starvation.</h2>
			<h3>PMID:24634113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li XH(1), Wu B(1), Kong YB(1), Zhang CY(2).
				<br/> 
				年份：2014 Feb 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus plays a pivotal role in plant growth and <span class="trait">development.</span> In this study, we isolated and characterized <span class="gene">GmPTF1,</span> a basic helix-loop-helix (bHLH) transcription factor (TF) gene from soybean (Glycine max) with tolerance to inorganic phosphate (Pi) starvation. Alignment analysis indicated that <span class="gene">GmPTF1</span> and other reported bHLH TFs share significant similarity in the region of the bHLH domain. As with OsPTF1 and other homologous Pi starvation-related bHLH TFs (His-5, Glu-9, Arg-12, and Arg-13), all recognition motifs for the G-box (CACGTG) were present in the <span class="gene">GmPTF1</span> domain. Prokaryotic expression in Escherichia coli strain BL21 (DE3) plysS showed that a novel 40-kDa polypeptide was expressed when cells containing <span class="gene">GmPTF1</span> were induced. The subcellular localization in cells from onion epidermis and Arabidopsis roots demonstrated that the <span class="gene">GmPTF1</span> protein was found in the nucleus. Furthermore, analysis of transcription activity in yeast revealed that full-length <span class="gene">GmPTF1</span> and its N-terminal and C-terminal domains could activate the histidine, adenine, and uracil reporter genes. This suggested that the N-terminal and C-terminal peptides of <span class="gene">GmPTF1</span> act as transcriptional activators. When real-time quantitative polymerase chain reaction was performed, the expression of <span class="gene">GmPTF1</span> under conditions of phosphate starvation was significantly induced in soybean roots of the low-Pi-tolerant variety ZH15. Moreover, the relative level of expression was much higher there than in roots of the sensitive variety NMH from days 7 to 56 of low-Pi stress. These results imply that <span class="gene">GmPTF1</span> is involved in conferring tolerance to phosphate starvation in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression analysis of a stress-induced <span class="gene">GmIMT1</span> gene in soybean (Glycine max).</h2>
			<h3>PMID:24615045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang HT(1), Guo N(1), Zhao JM(1), Karthikeyan A(1), Xue D(1), Xue CC(1), Xu JY(1), Xu ZH(1), Gai JY(1), Xing H(1).
				<br/> 
				年份：2014 Feb 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Here, we aimed to clone and identify the <span class="gene">GmIMT1</span> gene related to the salt stress response in soybean. The full-length cDNA sequence of the <span class="gene">GmIMT1</span> gene was amplified in soybean using degenerate primers of Mesembrythmum crystallium. To understand the stress response, the <span class="gene">GmIMT1</span> gene was cloned and sequenced. Then, the expression vectors of the gene were constructed, and introduced into the model plant Arabidopsis thaliana through Agrobacterium mediated transformation, and the salt tolerance was analyzed in the transgenic plants. In addition, the expression patterns of <span class="gene">GmIMT1</span> gene in soybean were analyzed. The expression was examined in different organs (roots, leaves, flower seeds, and stem) and under different stress conditions <span class="trait">(drought,</span> high salt, low temperature, salicylic acid, ethane, abscisic acid, and methyl jasmonate) by real-time fluorescent quantitative polymerase chain reaction analysis. The results showed that the <span class="trait">root,</span> leaves, and stems exhibited high level of <span class="gene">GmIMT1</span> gene expression, whereas there was no expression in the seeds. In addition, the <span class="gene">GmIMT1</span> gene expression was upregulated under all stress conditions. Overall, the results clearly indicate that <span class="gene">GmIMT1</span> might be involved in multiple plant response pathways to the different environmental conditions. Furthermore transgenic plants exhibited higher salt-tolerance compared to wild type plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional Analysis of Water Stress-Responsive Soybean <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> Transcription Factors in Lateral <span class="trait">Root</span> <span class="trait">Development</span> in <italic> Arabidopsis </italic></h2>
			<h3>PMID:24465446</h3>
				

				
					<h3>PMCID:3900428</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Quach </surname>  <given-names> Truyen N. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Hanh TM. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Rajesh </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn3">  <sup> &#x000a4;c </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neelakandan </surname>  <given-names> Anjanasree K. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn4">  <sup> &#x000a4;d </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guttikonda </surname>  <given-names> Satish Kumar </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn5">  <sup> &#x000a4;e </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharp </surname>  <given-names> Robert E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis, NAC (NAM, ATAF and CUC) transcription factors have been found to promote lateral <span class="trait">root</span> number through the auxin signaling pathway. In the present study, the role of water stress-inducible soybean <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> genes in the enhancement of lateral <span class="trait">root</span> <span class="trait">development</span> under water deficit conditions was investigated. Both genes were highly expressed in roots, leaves and flowers of soybean and were strongly induced by water stress and moderately induced by a treatment with abscisic acid (ABA). They showed a slight response to treatment with 2,4-dichlorophenoxyacetic acid (2,4-D). The transgenic Arabidopsis plants overexpressing <span class="gene">GmNAC004</span> showed an increase in lateral <span class="trait">root</span> number and <span class="trait">length</span> under non-stress conditions and maintained higher lateral <span class="trait">root</span> number and <span class="trait">length</span> under mild water stress conditions compared to the wild-type (WT), while the transgenic plants overexpressing <span class="gene">GmNAC003</span> did not show any response. However, LR <span class="trait">development</span> of <span class="gene">GmNAC004</span> transgenic Arabidopsis plants was not enhanced in the water-stressed compared to the well-watered treatment. In the treatment with ABA, LR density of the <span class="gene">GmNAC004</span> transgenic Arabidopsis was less suppressed than that of the WT, suggesting that <span class="gene">GmNAC004</span> counteracts ABA-induced inhibition of lateral <span class="trait">root</span> <span class="trait">development.</span> In the treatment with 2,4-D, lateral <span class="trait">root</span> density was enhanced in both <span class="gene">GmNAC004</span> transgenic Arabidopsis and WT plants but the promotion was higher in the transgenic plants. Conversely, in the treatment with naphthylphthalamic acid (NPA), lateral <span class="trait">root</span> density was inhibited and there was no difference in the phenotype of the <span class="gene">GmNAC004</span> transgenic Arabidopsis and WT plants, indicating that auxin is required for the action of <span class="gene">GmNAC004.</span> Transcript analysis for a number of known auxin and ABA related genes showed that <span class="gene">GmNAC004's</span> role may suppress ABA signaling but promote auxin signaling to increase lateral <span class="trait">root</span> <span class="trait">development</span> in the Arabidopsis heterologous system.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In <italic> Arabidopsis </italic> , NAC (NAM, ATAF and CUC) transcription factors have been found to promote lateral <span class="trait">root</span> number through the auxin signaling pathway. In the present study, the role of water stress&#x02013;inducible soybean <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the enhancement of lateral <span class="trait">root</span> <span class="trait">development</span> under water deficit conditions was investigated. Both genes were highly expressed in roots, leaves and flowers of soybean and were strongly induced by water stress and moderately induced by a treatment with abscisic acid (ABA). They showed a slight response to treatment with 2,4-dichlorophenoxyacetic acid (2,4-D). The transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> showed an increase in lateral <span class="trait">root</span> number and <span class="trait">length</span> under non-stress conditions and maintained higher lateral <span class="trait">root</span> number and <span class="trait">length</span> under mild water stress conditions compared to the wild-type (WT), while the transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> did not show any response. However, LR <span class="trait">development</span> of <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment. In the treatment with ABA, LR density of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> was less suppressed than that of the WT, suggesting that <italic> <span class="gene">GmNAC004</span> </italic> counteracts ABA-induced inhibition of lateral <span class="trait">root</span> <span class="trait">development.</span> In the treatment with 2,4-D, lateral <span class="trait">root</span> density was enhanced in both <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants but the promotion was higher in the transgenic plants. Conversely, in the treatment with naphthylphthalamic acid (NPA), lateral <span class="trait">root</span> density was inhibited and there was no difference in the phenotype of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants, indicating that auxin is required for the action of <italic> <span class="gene">GmNAC004</span> </italic> . Transcript analysis for a number of known auxin and ABA related genes showed that <italic> <span class="gene">GmNAC004</span> </italic> 's role may suppress ABA signaling but promote auxin signaling to increase lateral <span class="trait">root</span> <span class="trait">development</span> in the <italic> Arabidopsis </italic> heterologous system.
					</p>
				
					<p>&nbsp;&nbsp;
						The NAC ( <bold> N </bold> AM-no apical meristem, <bold> A </bold> TAF- <italic> Arabidopsis </italic> transcription activation factor, and <bold> C </bold> UC-cup-shaped cotyledon) transcription factor (TF) family is a large plant-specific TF family. NAC TFs are involved in diverse processes including <span class="trait">development,</span> defense and abiotic stress responses <xref ref-type="bibr" rid="pone.0084886-Tran1"> [29] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-Hadiarto1"> [33] </xref> . Overexpression of several dehydration-inducible <italic> NAC </italic> genes resulted in improved <span class="trait">drought</span> tolerance in rice <xref ref-type="bibr" rid="pone.0084886-Hu1"> [34] </xref> , <xref ref-type="bibr" rid="pone.0084886-Nakashima1"> [35] </xref> and <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0084886-Tran3"> [36] </xref> . A number of NAC TFs can also act as positive regulators of LR <span class="trait">development</span> through regulation of auxin signaling pathways <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . <italic> Arabidopsis </italic> <span class="gene">NAC1</span> acts downstream of an auxin receptor, TIR1, to regulate several proteins in the auxin signaling pathway. Overexpression of the <italic> <span class="gene">NAC1</span> </italic> gene increased LR number and was able to restore the phenotype of the <italic> tir1 </italic> mutant. Consistently, down-regulation of <italic> <span class="gene">NAC1</span> </italic> reduced LR <span class="trait">development</span> in transgenic <italic> Arabidopsis </italic> overexpressing the <italic> TIR1 </italic> gene <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . Another <italic> Arabidopsis </italic> NAC TF encoding gene, <italic> AtNAC2 </italic> , which is induced by dehydration, ABA and salt stresses, was shown to increase LR number in transgenic plants <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> . Recently, soybean <italic> <span class="gene">GmNAC20</span> </italic> was reported to stimulate LR number in transgenic <italic> Arabidopsis </italic> and its action also appears to be involved in genes related to auxin signaling <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean contains a large family of NAC TFs, with more than 150 members <xref ref-type="bibr" rid="pone.0084886-Wang1"> [40] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-Tran5"> [45] </xref> , among which at least 38 <italic> <span class="gene">GmNAC</span> </italic> genes are dehydration-inducible <xref ref-type="bibr" rid="pone.0084886-Le1"> [46] </xref> . In our previous report, we characterized 31 <italic> <span class="gene">GmNAC</span> </italic> genes and identified nine dehydration-inducible <italic> <span class="gene">GmNAC</span> </italic> genes <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> . Among these nine genes, only <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> showed differential expression patterns in response to ABA treatment, indicating that these two genes may be important regulators for soybean adaptation to water stress in an ABA-dependent manner. <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> were also shown to specifically express in roots, suggesting that they might play a role in the regulation of growth and <span class="trait">development</span> of the <span class="trait">root</span> system in response to water deficit conditions. These two TFs were localized in the nucleus and showed the ability to activate transcription in a yeast assay <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> , <xref ref-type="bibr" rid="pone.0084886-Pinheiro1"> [48] </xref> . With the hypothesis that water stress-inducible <italic> <span class="gene">GmNAC</span> </italic> genes might have a role in the enhanced <span class="trait">development</span> of LRs in soybean under mild water stress, the objective of the present study was to investigate the functions of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in LR <span class="trait">development.</span> The heterologous <italic> Arabidopsis </italic> system was selected for the study because of ease of genetic manipulation and available downstream hormone signaling information. In fact, the <italic> Arabidopsis </italic> model has been used successfully for functional characterization of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 and soybean <span class="gene">GmNAC20</span> in stimulating lateral <span class="trait">root</span> <span class="trait">development</span> <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . In the present study, we found that <italic> <span class="gene">GmNAC004</span> </italic> and <italic> <span class="gene">GmNAC003</span> </italic> had an increased expression level under auxin treatment and promoted LR number and total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants under well-watered conditions. Transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> maintained a higher LR number and greater total LR <span class="trait">length</span> compared to the WT under mild water deficit, but there was no promotion of LR <span class="trait">development</span> under water deficit conditions when compared to the well-watered control. <span class="gene">GmNAC004</span> appears to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic> under normal growth conditions via suppression of ABA signaling and promotion of auxin signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmNAC003</span> </italic> (DQ028771) and <italic> <span class="gene">GmNAC004</span> </italic> (DQ028772) gene specific primers were designed using Primer 3 software <xref ref-type="bibr" rid="pone.0084886-Rozen1"> [49] </xref> and are listed in <xref ref-type="supplementary-material" rid="pone.0084886.s003"> Table S2 </xref> . The primers were verified for binding specificity by blasting each primer sequence against the soybean genome sequence information at Phytozome ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> http://www.phytozome.net </ext-link> ). For transcript analysis of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in response to <span class="trait">drought,</span> abiotic stresses and hormonal treatments, nine soybean internal controls ( <xref ref-type="supplementary-material" rid="pone.0084886.s003"> Table S2 </xref> ) were included for the qRT-PCR reactions and the most stable genes were selected to calculate the normalization factors using GeNorm <xref ref-type="bibr" rid="pone.0084886-Vandesompele1"> [50] </xref> . RNA isolation, cDNA synthesis and qRT-PCR were performed as previously described <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> , <xref ref-type="bibr" rid="pone.0084886-GutierrezGonzalez1"> [51] </xref> . The qRT-PCR data were analyzed with the SDS 2.3 software package and a common signal threshold was set to 0.1. PCR amplification efficiency was obtained using the assumption-free PCR efficiency calculated by LinRegPCR <xref ref-type="bibr" rid="pone.0084886-Ruijter1"> [52] </xref> . Relative RNA abundance of each gene was calculated as GOI/N, where GOI is the expression quantity of the gene of interest and normalization factor. N was calculated by GeNorm as the geometric mean of the selected internal reference genes. In the situation where multiple controls were present, the treatment with the lowest expression was transformed to a value of 1 for convenient comparisons. For quantification of auxin and ABA related genes in <italic> Arabidopsis </italic> , ubiquitin was used as a reference gene ( <xref ref-type="supplementary-material" rid="pone.0084886.s003"> Table S2 </xref> ) and transcript profiling was performed for <span class="trait">root</span> samples collected from plants grown on &#x000bc;-strength <span class="gene">MS</span> nutrient agar plates.
					</p>
				
					<p>&nbsp;&nbsp;
						The open reading frames of soybean <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes were cloned by Tran <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> . The two genes were subsequently inserted into a pGreen plasmid that had been pre-inserted with the 35S promoter, to produce pGreen-P35S-GmNAC003 and pGreen-P35S-GmNAC004. These binary vectors containing the soybean <italic> <span class="gene">GmNAC</span> </italic> genes were verified by sequencing and transformed into <italic> Agrobacterium tumefaciens </italic> strain C58, which had been transformed with the helper vector pSoup. All binary plasmid constructs were transformed into <italic> Arabidopsis </italic> using the floral dip method <xref ref-type="bibr" rid="pone.0084886-Clough1"> [53] </xref> . Transgenic plants at T1, T2, and T3 generations were screened in &#x000bc;-strength <span class="gene">MS</span> medium containing either 25 mg/L hygromycin. Transgenic plants were verified for T-DNA insertion by PCR and the expression of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> was determined by qRT-PCR. Two events having single insertion copy and comparable plant growth to the WT were selected for phenotypic assays of <span class="trait">root</span> <span class="trait">development.</span> Transgene expression in the transgenic <italic> Arabidopsis </italic> plants was quantified from 2-week old seedlings grown on &#x000bc;-strength <span class="gene">MS</span> nutrient agar plates. The <italic> Arabidopsis </italic> ubiquitin gene (AT3G62250) was used as the reference gene for the qRT-PCR with the primers listed in <xref ref-type="supplementary-material" rid="pone.0084886.s003"> Table S2 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> seeds were surface-sterilized with 70% ethanol for 2 minutes followed by 10% bleach for 2 minutes and then rinsed four times with distilled water. After stratification for four days at 4&#x000b0;C in darkness, the seeds were transferred to a nursery plate and treated with red light for 1 hour and then kept in the dark for 24 hours at 22&#x000b0;C to promote germination. The plates were then placed in a growth chamber in a vertical position to allow downward <span class="trait">root</span> growth for three days. At day 4 (4 days after sowing), seedlings with a primary <span class="trait">root</span> <span class="trait">length</span> between 9 and 11 mm were transferred to bioassay plates with designated PEG concentrations as described above. Each bioassay plate contained twenty-five seedlings from five genotypes (two transgenic <span class="gene">GmNAC003,</span> two transgenic <span class="gene">GmNAC004,</span> and the WT lines). The plates were sealed by two layers of micropore tape (Micropore, 3M Company, St Paul, MN) and placed vertically in a growth chamber (Conviron model A1000TC) at a constant relative humidity of &#x0223c;70%, 22/18&#x000b0;C day/night temperatures, 16/8 hour day-length, and &#x0223c;200 &#x000b5;mol m <sup> &#x02212;2 </sup> s <sup> &#x02212;1 </sup> photosynthetic photon flux. At day 12, the plates were measured for LR number, total <span class="trait">root</span> <span class="trait">length</span> and primary <span class="trait">root</span> <span class="trait">length.</span> LR number was counted visually using a magnifying glass. Total <span class="trait">root</span> <span class="trait">length</span> was measured using the WinRHIZO program (Regent Instruments Inc., QC, Canada). Primary <span class="trait">root</span> <span class="trait">length</span> was measured with a ruler from the root/shoot junction to the <span class="trait">root</span> tip. Total LR <span class="trait">length</span> was calculated by subtracting the primary <span class="trait">root</span> <span class="trait">length</span> from the total <span class="trait">root</span> <span class="trait">length.</span> The experiment was repeated three times with six replicated plates to check for consistent results. At the end of the experiment, water potentials of the agar plates were measured using isopiestic thermocouple psychrometry <xref ref-type="bibr" rid="pone.0084886-Boyer1"> [56] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Seedlings of <span class="gene">GmNAC004</span> transgenic plants were grown as described above for the PEG treatments. At day 4 (4 days after sowing), they were transferred to bioassay agar plates containing the indicated hormone(s) with a density of 25 seedlings per plate. Each hormone treatment contained the basal nutrient medium (described above) with added single or combination of hormones to final concentrations of 5 &#x000b5;M ABA, 20 nM 2,4-D, and 2 &#x000b5;M NPA. The experiment was repeated two times with six replicated plates. LR number and <span class="trait">length</span> were measured at day 12 as described for the water stress treatments. NPA and 2,4-D have been reported to inhibit primary <span class="trait">root</span> elongation mainly by suppression of cell production <xref ref-type="bibr" rid="pone.0084886-Rahman1"> [57] </xref> . Therefore, we used LR density rather than total LR number to compare LR <span class="trait">development</span> among treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the response of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> to water deficit in various soybean organs, a greenhouse experiment was conducted in which water was withheld to allow progressive water stress <span class="trait">development</span> to mimic slow soil-drying under <span class="trait">drought</span> conditions in the field. Tissues were collected at three water stress levels of &#x0223c;&#x02212;0.5, &#x02212;1.0 and &#x02212;1.5 MPa stem water potentials, corresponding to 4, 8 and 20 days after water deficit treatment started ( <xref ref-type="supplementary-material" rid="pone.0084886.s002"> Table S1 </xref> ). These stress levels represent mild, intermediate and severe stress, respectively. The control soybean plants grew at a rate of about one new leaf per 3 days, and the developmental stages of the controls were different from the stressed plants, which almost ceased their growth as the stress intensified and had only 2 or 3 trifoliolate leaves by the end of the experiment. To avoid this difference in <span class="trait">development,</span> two controls were required: a developmental control having the same growth stage as the stressed treatments and temporal controls with the same durations of growth as the stressed treatments. The control <span class="gene">V2</span> stage was considered a developmental control because the stressed plants did not produce any more leaves. For the temporal comparison, <span class="gene">V2</span> was the control for &#x02212;0.5 MPa, and similarly, <span class="gene">V3</span> for &#x02212;1.0 MPa and V6 for &#x02212;1.5 MPa ( <xref ref-type="supplementary-material" rid="pone.0084886.s002"> Table S1 </xref> ). We define a water stress-responsive gene as a gene whose expression changed under water stress relative to both the developmental and temporal controls.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0084886-g001"> Figure 1A </xref> shows that the expression of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> was strongly induced by water stress in all the tissues examined and that the inducibility of <italic> <span class="gene">GmNAC004</span> </italic> was stronger than that of <italic> <span class="gene">GmNAC003</span> </italic> . In <span class="trait">root</span> and leaf tissues, both genes responded significantly even at the mild water stress level (&#x02212;0.5 MPa), while in the stem they were induced to a lesser extent and only at the intermediate and severe water stress levels.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant growth and <span class="trait">development</span> are regulated by various hormones and other environmental stresses, such as salinity and cold. In the abiotic signaling network, several <italic> NAC </italic> genes were reported to have ABA-dependent regulation of stress responses <xref ref-type="bibr" rid="pone.0084886-Hu1"> [34] </xref> , <xref ref-type="bibr" rid="pone.0084886-Tran3"> [36] </xref> , <xref ref-type="bibr" rid="pone.0084886-Shinozaki1"> [58] </xref> . Although the expression of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> under ABA, salinity and cold treatments was known in soybean cv. Maverick <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> , we wanted to re-examine the expression of the <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in W82, which was reported to have a greater LR number increase than cv. Maverick in response to mild water stress <xref ref-type="bibr" rid="pone.0084886-He1"> [4] </xref> . <xref ref-type="fig" rid="pone-0084886-g002"> Figure 2 </xref> shows that both <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes showed higher expression under the NaCl treatment, with a similar trend observed in response to dehydration ( <xref ref-type="fig" rid="pone-0084886-g001"> Figure 1B </xref> ). ABA treatment resulted in a moderate increase in gene expression of both genes. In contrast, cold stress had a slight increased expression of <italic> <span class="gene">GmNAC003</span> </italic> while suppressed expression of <italic> <span class="gene">GmNAC004</span> </italic> at early time points. The trend of responses of the <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in W82 is consistent with our previous study which was conducted for Maverick, although the inducibility of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes appeared to be higher in W82 than in Maverick in the salt stress treatment ( <xref ref-type="fig" rid="pone-0084886-g002"> Figure 2 </xref> , <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> ). The response to NaCl and dehydration treatments was detected as early as after 1 h of treatment. On the other hand, significant accumulation of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> under ABA treatment was observed only after 2 h. Taken together, these data suggest that these <italic> <span class="gene">GmNAC</span> </italic> genes do not require ABA accumulation for their early responses to either NaCl or dehydration, and that they regulate responses to water deficit and salt stress in both an ABA-dependent and an ABA-independent manners.
					</p>
				
					<p>&nbsp;&nbsp;
						NAC TFs have been shown to associate with auxin signaling in regulation of LR <span class="trait">development</span> <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> , <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . In our previous study, <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> had preferential expression in roots <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> suggesting that they may be involved in <span class="trait">root</span> <span class="trait">development.</span> To determine whether <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> act downstream of the auxin signaling pathway, we investigated their expression in response to 2,4-D treatment. <xref ref-type="fig" rid="pone-0084886-g002"> Figure 2 </xref> shows that expression of the <italic> <span class="gene">GmNAC003</span> </italic> gene in response to 2,4-D was slightly increased, while there was a moderate increase in expression of the <italic> <span class="gene">GmNAC004</span> </italic> gene after 1 h of 2,4-D treatment. The response of <italic> <span class="gene">GmNAC004</span> </italic> was rapid but transient for the first few hours, and then the expression was regained after a longer time of treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						To understand sequence relationship among <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> and <span class="gene">GmNAC20</span> <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> we reconstructed a phylogenetic tree of 152-soybean NAC transcription factor family ( <xref ref-type="supplementary-material" rid="pone.0084886.s001"> Figure S1 </xref> ) using recently updated data <xref ref-type="bibr" rid="pone.0084886-Le1"> [46] </xref> . <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> have high sequence similarity (67% identical amino acid sequences for full <span class="trait">length</span> and 92% for the 170-amino acid conserved NAC domain). However, these two proteins are relatively different at carboxyl terminals which acted as an activating domain <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> . These two proteins are sub-grouped to the same branch with <span class="gene">GmNAC20</span> and NAC domain sequence similarity between <span class="gene">GmNAC004</span> and <span class="gene">GmNAC20</span> is 66%. <span class="gene">GmNAC003,</span> <span class="gene">GmNAC004</span> and <span class="gene">GmNAC20,</span> however, has low sequence similarity with <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2, which function to stimulate LR number in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> , <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To explore the function of <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> in LR <span class="trait">development</span> we overexpressed the two genes in the <italic> Arabidopsis </italic> model. We used the <italic> Arabidopsis </italic> model for our work because of the ease of genetic manipulation and the previously successful characterizations of the <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 TFs and the soybean <span class="gene">GmNAC20</span> in the regulation of LR numbers <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> , <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . In addition, because <italic> <span class="gene">GmNAC004</span> </italic> was induced by water stress and by auxin treatment, we wanted to investigate whether this gene is involved in the enhancement of LR <span class="trait">development</span> under water stress. For this purpose, we developed transgenic <italic> Arabidopsis </italic> overexpressing <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes under control of the constitutive 35S promoter. Transgenic plants with single locus insertion copy were selected based on a 3&#x02236;1 segregation ratio at the T2 generation and used to develop T3 homozygous lines. Seven <italic> <span class="gene">GmNAC003</span> </italic> and six <italic> <span class="gene">GmNAC004</span> </italic> homozygous transgenic lines at the T3 generation having a single transgene insertion were selected for verification of transgenic expression and growth observation. Among the lines expressing the transgenes ( <xref ref-type="fig" rid="pone-0084886-g003"> Figure 3A </xref> ), two homozygous lines at the T3 generation from each gene construct (lines 9 and 10 for <italic> 35S:GmNAC003 </italic> and lines 1 and 3 for <italic> 35S:GmNAC004 </italic> , <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> ) that showed shoot growth equivalent to the WT ( <xref ref-type="fig" rid="pone-0084886-g003"> Figure 3B </xref> ) were selected for <span class="trait">root</span> growth assays. We generated several mild water deficit conditions within a water potential range from &#x02212;0.09 to &#x02212;0.24 MPa by diffusing PEG into the agar medium ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4A </xref> ). The water potentials of PEG-diffused agar plates were reproducible and constant throughout five days in growth chamber conditions, allowing accurate measurement of the stress level at the end of the experiments <xref ref-type="bibr" rid="pone.0084886-vanderWeele1"> [59] </xref> . The range of water deficit in this study was comparable to the stress level in which He <xref ref-type="bibr" rid="pone.0084886-He1"> [4] </xref> reported promotion of LR number and <span class="trait">length</span> in soybean. This water deficit range, therefore, was considered suitable for the investigation of the roles of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the regulation of LR growth and <span class="trait">development</span> under water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Water potentials of PEG-diffused agar plates. Water potentials were measured at the end of the experiment, 8 days from the date of stress exposure. (B) Lateral <span class="trait">root</span> number, (C) total lateral <span class="trait">root</span> <span class="trait">length,</span> and (D) primary <span class="trait">root</span> <span class="trait">length</span> of transgenic plants. Transgenic <italic> <span class="gene">GmNAC003</span> </italic> (events N3.09 and N3.10), <italic> <span class="gene">GmNAC004</span> </italic> (events N4.01 and N4.03), and WT <italic> Arabidopsis </italic> lines were grown in nutrient agar plates diffused with different concentrations of PEG. The transgenic plants were at the T4 generation and homozygous for the transgenes. Data were recorded at 12 days after sowing (or 8 days of stress exposure). (*) denotes significant difference at 95% confidence level using Duncan's multiple range test from six replications. Error bars are standard errors of the means from 6 replications.
					</p>
				
					<p>&nbsp;&nbsp;
						Constitutive overexpression of <italic> <span class="gene">GmNAC004</span> </italic> resulted in an increase in the number and total <span class="trait">length</span> of LRs in transgenic <italic> Arabidopsis </italic> plants compared to the WT ( <xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> and <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> ), while that of <italic> <span class="gene">GmNAC003</span> </italic> had no effect on these traits ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> ). Under well-watered conditions (&#x02212;0.09 MPa water potential), transgenic <italic> <span class="gene">GmNAC004</span> </italic> plants exhibited an increase in LR number of 37% over the WT. Under the mild water deficit condition, a significant reduction in LR number was observed in both the transgenic and the WT plants; however, the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants still had 20&#x02013;25% more LRs than the WT. Further lowering the water potential to &#x02212;0.24 MPa strongly suppressed the LR number in both transgenic and WT plants. These data demonstrate that overexpression of <italic> <span class="gene">GmNAC004</span> </italic> increased LR number under non-stress conditions and maintained higher LR number under mild water stress conditions in comparison with the WT. Similar to LR number, total LR <span class="trait">length</span> was also increased in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants compared to the WT under both well-watered (increased by 112%) and mild water stress (increased by 87&#x02013;123%) conditions ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4C </xref> ). Although having similar expression profiles and a conserved DNA binding domain with those of <italic> <span class="gene">GmNAC004</span> </italic> ( <xref ref-type="fig" rid="pone-0084886-g001"> Figures 1 </xref> &#x00026; <xref ref-type="fig" rid="pone-0084886-g002"> 2 </xref> and <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> ), overexpression of <italic> <span class="gene">GmNAC003</span> </italic> did not result in promotion of LR number or total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants in comparison with the WT.
					</p>
				
					<p>&nbsp;&nbsp;
						Within the range of mild water stress conditions from &#x02212;0.12 to &#x02212;0.24 MPa, primary <span class="trait">root</span> <span class="trait">length</span> was only slightly affected in both transgenic and WT <italic> Arabidopsis </italic> plants. Among the <italic> Arabidopsis </italic> lines, transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> did not show significant changes in primary <span class="trait">root</span> <span class="trait">length</span> when compared to the WT at any tested water potentials ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4D </xref> ). Lowering the water potential to &#x02212;0.24 MPa slightly decreased primary <span class="trait">root</span> <span class="trait">length.</span> This finding is consistent with the result reported by van der Weele <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-vanderWeele1"> [59] </xref> where the authors found only a small change in primary <span class="trait">root</span> <span class="trait">length</span> in response to mild water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Because the <italic> <span class="gene">GmNAC004</span> </italic> gene was induced by both ABA and water stress ( <xref ref-type="fig" rid="pone-0084886-g001"> Figures 1 </xref> and <xref ref-type="fig" rid="pone-0084886-g002"> 2 </xref> ), <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> via an ABA-signaling pathway. ABA is known to inhibit LR <span class="trait">development</span> <xref ref-type="bibr" rid="pone.0084886-DeSmet1"> [26] </xref> and suppress seed germination <xref ref-type="bibr" rid="pone.0084886-Bewley1"> [60] </xref> . To investigate the involvement of <span class="gene">GmNAC004</span> in the ABA-signaling network, we examined the response of seed germination of two transgenic events for each gene to a treatment of 1 &#x000b5;M ABA. <xref ref-type="fig" rid="pone-0084886-g006"> Figure 6 </xref> showed that seed germination of the WT was inhibited by nearly 25% in the ABA treatment. The <italic> <span class="gene">GmNAC003</span> </italic> transgenic plants showed significantly higher inhibition, suggesting that <span class="gene">GmNAC003</span> is a positive regulator of ABA signaling. By contrast, the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants displayed less inhibition in the presence of ABA. These data suggest that <span class="gene">GmNAC004</span> counteracts ABA signaling, at least in seed germination.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of <italic> <span class="gene">GmNAC004</span> </italic> in counteracting the inhibition of seed germination by ABA ( <xref ref-type="fig" rid="pone-0084886-g006"> Figure 6 </xref> ) raises the possibility that the <span class="gene">GmNAC004</span> protein may also counteract ABA inhibition of LR <span class="trait">development,</span> which has been reported under both normal and stress conditions <xref ref-type="bibr" rid="pone.0084886-DeSmet1"> [26] </xref> , <xref ref-type="bibr" rid="pone.0084886-Deak1"> [28] </xref> , <xref ref-type="bibr" rid="pone.0084886-Guo1"> [61] </xref> . This hypothesis was tested by examining the effects of applied ABA on LR <span class="trait">development</span> in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants under well-watered conditions. <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7A </xref> shows that ABA suppressed LR density of both transgenic and WT plants, but the suppression was lower for the transgenic plants. This result suggests that <span class="gene">GmNAC004</span> partially represses ABA-induced inhibition of LR number.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of <italic> <span class="gene">GmNAC004</span> </italic> was induced by 2,4-D ( <xref ref-type="fig" rid="pone-0084886-g002"> Figure 2 </xref> ); therefore, it is possible that the role of <span class="gene">GmNAC004</span> in regulation of LR number also involves auxin. To study the interaction of <span class="gene">GmNAC004</span> and auxin signaling on LR <span class="trait">development,</span> we applied the synthetic auxin 2,4-D and the polar auxin transport inhibitor NPA to the growing media and measured <span class="trait">root</span> growth traits. We observed that under the control treatment, LR density was higher in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants compared to the WT ( <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7A </xref> ). In the NPA treatment, LR density was significantly reduced in both the transgenic and WT plants. The NPA treatment totally suppressed the action of <span class="gene">GmNAC004</span> in LR promotion so that there was no difference in LR density between the transgenic and WT plants. Furthermore, in the treatments with 2,4-D, either alone or together with ABA, overexpression of <italic> <span class="gene">GmNAC004</span> </italic> resulted in greater promotion of LR density in the transgenic plants in comparison with the WT. Taken together, these results demonstrate that the stimulation of LR <span class="trait">development</span> by <span class="gene">GmNAC004</span> requires auxin and that <span class="gene">GmNAC004</span> stimulates LR <span class="trait">development</span> via ABA-dependent and auxin-dependent signaling pathways.
					</p>
				
					<p>&nbsp;&nbsp;
						To further elucidate the involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development,</span> expression of a number of representative genes in ABA and auxin signaling pathways were quantified. The genes regulating auxin signaling include <italic> AIR3 </italic> , <italic> ARF2 </italic> , <italic> ARF7 </italic> , <italic> ARF19, AXR1 </italic> , <italic> AXR3 </italic> , and <italic> LBD12 </italic> , and ABA signaling include <italic> ABI1 </italic> , <italic> ABI2 </italic> , <italic> ABI3 </italic> , <italic> ABI5 </italic> , <italic> ABI8 </italic> and <italic> ERA1 </italic> . <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7B </xref> shows that <italic> Arabidopsis </italic> transgenic <span class="gene">GmNAC004</span> plants had elevated expression of a number of auxin signaling genes including <italic> AIR3 </italic> , <italic> ARF7 </italic> , <italic> ARF19 </italic> , and <italic> LBD12 </italic> , while reduced expression of <italic> AXR1 </italic> . Expression of a number of genes in ABA signaling was also altered, with increased expression of a negative regulator of ABA response gene <italic> ERA1 </italic> . <span class="gene">GmNAC003</span> overexpression, in contrast, suppressed expression of <italic> ERA1 </italic> while stimulation of <italic> ABI3 </italic> and <italic> ABI5 </italic> , indicating a positive regulator of ABA signaling. Alteration of auxin and ABA related genes indicates that <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> through auxin and ABA signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						Significant advances in understanding of molecular networks regulating LR <span class="trait">development</span> in the past few years enabled us to further investigate the molecular regulation of <span class="trait">root</span> architecture under water deficit. The promotion of numbers and <span class="trait">length</span> of LRs in lower soil profiles in response to water deficit, which has been reported previously <xref ref-type="bibr" rid="pone.0084886-Read1"> [2] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-He1"> [4] </xref> , <xref ref-type="bibr" rid="pone.0084886-Huck1"> [13] </xref> , is considered an avoidance mechanism of soybean plants to water stress conditions <xref ref-type="bibr" rid="pone.0084886-Manavalan1"> [9] </xref> . This occurrence must be regulated by the activation of responsive proteins that control growth and optimize <span class="trait">development</span> to adapt to the stress. Using transcriptional profile analysis, we characterized two soybean genes, <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> , which were expressed strongly in roots and leaves of soybean in response to water stress. Strong lines of evidence have indicated that NAC TFs play an important role in water stress responses and in plant <span class="trait">development</span> <xref ref-type="bibr" rid="pone.0084886-Tran1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0084886-Nakashima2"> [62] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0084886-Nishiyama2"> [65] </xref> . In <italic> Arabidopsis </italic> , individual overexpression of the dehydration-inducible genes <italic> ANAC019 </italic> , <italic> ANAC055 </italic> and <italic> ANAC072 </italic> improved water stress tolerance <xref ref-type="bibr" rid="pone.0084886-Razem1"> [27] </xref> . In rice, Hu <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-Hu1"> [34] </xref> reported that transgenic plants overexpressing a drought-inducible <italic> SNAC1 </italic> gene could maintain leaf turgor and better spikelet <span class="trait">fertility</span> under <span class="trait">drought</span> stress. In regulation of LR <span class="trait">development,</span> NAC TFs are promoters of LR numbers in <italic> Arabidopsis </italic> and soybean. <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 transcription factors stimulate LR <span class="trait">development</span> through auxin signaling <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> , <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . In addition, <italic> AtNAC2 </italic> was up-regulated by ABA and salinity treatments, suggesting the interaction of <span class="trait">root</span> <span class="trait">development</span> and osmotic stress. In soybean, Hao <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> recently reported that a stress inducible <span class="gene">GmNAC20</span> stimulated LR <span class="trait">development</span> and stress tolerance in the <italic> Arabidopsis </italic> heterologous system. Molecular analysis of transgenic plants overexpressing <italic> <span class="gene">GmNAC20</span> </italic> revealed that the gene regulated several genes in the auxin signaling network including auxin response factor genes <italic> ARF7 </italic> and <italic> ARF19 </italic> , and a lateral boundaries domain gene <italic> LBD12 </italic> . <span class="gene">GmNAC20</span> TF also induced a number of cold-responsive genes. In the present study, we demonstrated that <italic> <span class="gene">GmNAC004</span> </italic> was induced by water stress, ABA and auxin. These results suggest that <span class="gene">GmNAC004</span> transcription factor have a role in regulation of plant <span class="trait">development</span> in response to environmental stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Using <italic> Arabidopsis in planta </italic> study, a role of <span class="gene">GmNAC004</span> in governing LR number and <span class="trait">development</span> was apparent. <xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> &#x00026; <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> show that <span class="gene">GmNAC004</span> significantly promoted LR number under non-stress conditions and that the transgenic plants were able to maintain higher LR numbers and total LR <span class="trait">length</span> than the WT under mild water deficit conditions. In contrast, although having a relative conserved DNA binding domain and transcription activation activity <xref ref-type="bibr" rid="pone.0084886-Tran6"> [47] </xref> when compared with <span class="gene">GmNAC004,</span> <span class="gene">GmNAC003</span> did not promote LR number. The difference in phenotypes of <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> transgenic plants might be due to the highly divergence of the sequences of the transcription activation domains at the C-terminal of the proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						The involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> through the ABA- and auxin-signaling pathways was also evident. The suppression of ABA inhibition of seed germination by <span class="gene">GmNAC004</span> ( <xref ref-type="fig" rid="pone-0084886-g006"> Figure 6 </xref> ) indicates that <span class="gene">GmNAC004</span> counteracts ABA signaling. Experiments using ABA treatments of transgenic <span class="gene">GmNAC004</span> and WT <italic> Arabidopsis </italic> further demonstrated that the protein is involved in suppression of ABA inhibition of LR number. ABA treatments (ABA, ABA &#x00026; 2,4-D) dramatically inhibited LR numbers in both the transgenic and WT lines, but the inhibition was not as strong as in the transgenic plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> ( <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7A </xref> ). The role of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> is supported by the report of De Smet <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-DeSmet1"> [26] </xref> , wherein they proposed a model of ABA-auxin interaction in the regulation of LR <span class="trait">development:</span> ABA inhibits LR <span class="trait">development</span> while auxin promotes it. This function of <span class="gene">GmNAC004</span> might work at least under normal growing conditions in which the inhibitory effect of ABA on LR number has been reported. Deak and Malamy <xref ref-type="bibr" rid="pone.0084886-Deak1"> [28] </xref> found that the ABA-deficient mutants of <italic> Arabidopsis aba2-1 </italic> and <italic> aba3-1 </italic> showed higher numbers of LRs than the WT under both well-watered and water-stressed (artificially induced by mannitol treatment) conditions. A similar observation was reported in an <italic> Arabidopsis </italic> ABA-deficient mutant in which the ABA biosynthesis enzyme 9- <italic> cis </italic> -epoxycarotenoid dioxygenase (NCED) was impaired. The <italic> nced3 </italic> mutant showed an increased LR number under well-watered conditions <xref ref-type="bibr" rid="pone.0084886-Guo1"> [61] </xref> . In contrast to ABA, auxin is a positive regulator of all stages of LR <span class="trait">development</span> from pre-initiation to emergence <xref ref-type="bibr" rid="pone.0084886-Overvoorde1"> [17] </xref> . In our experiments, <italic> <span class="gene">GmNAC004</span> </italic> was up-regulated by 2,4-D treatment in soybean seedlings ( <xref ref-type="fig" rid="pone-0084886-g002"> Figure 2 </xref> ), and the transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> showed a greater promotion of LR density when compared with the WT treated with 2,4-D ( <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7A </xref> ). In addition, <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants showed no promotion of LR number when auxin transport was blocked by NPA treatment, indicating that an auxin gradient is required for the action of <span class="gene">GmNAC004</span> in the regulation of LR number. In supporting for this response, molecular analysis showed that <span class="gene">GmNAC004</span> appears to regulate downstream molecules in both ABA and auxin signaling pathways. <span class="gene">GmNAC004</span> promoted expression of <italic> AIR3 </italic> , <italic> ARF7 </italic> , <italic> ARF19 </italic> and <italic> LBD12 </italic> and reduced expression of AXR1 ( <xref ref-type="fig" rid="pone-0084886-g007"> Figure 7B </xref> ). <italic> AIR3 </italic> was an auxin responsive gene which acts downstream of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and soybean <span class="gene">GmNAC20</span> in promotion of LR <span class="trait">development</span> in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0084886-Hao1"> [37] </xref> , <xref ref-type="bibr" rid="pone.0084886-Xie1"> [39] </xref> . ARF7 and ARF19 belong to the ARF family of transcriptional factors which bind to auxin response elements in the promoters of auxin-responsive genes. Double mutant arf7/arf19 severely reduced LR formation in <italic> Arabidopsis </italic> and significantly impaired global expression of auxin responsive genes <xref ref-type="bibr" rid="pone.0084886-Okushima1"> [66] </xref> . LBD12 is involved in the formation of lateral organs and differentially expressed in roots <xref ref-type="bibr" rid="pone.0084886-Shuai1"> [67] </xref> . AXR1 encodes an <span class="gene">E1</span> ubiquitin-activating enzyme targeting degradation of the AUX/IAA transcriptional repressors in response to auxin and might have a role in regulating LR in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0084886-Xie2"> [68] </xref> . In contrast to the positive role in regulation of auxin signaling pathway, <span class="gene">GmNAC004</span> down-regulates expression of <italic> ABI8 </italic> and <italic> ERA1 </italic> in the ABA signaling pathway. ABI8 is differentially expressed in <span class="trait">root</span> elongation zone of <italic> Arabidopsis </italic> and has a function in ABA-regulated seed dormancy and maintenance of <span class="trait">root</span> meristem <xref ref-type="bibr" rid="pone.0084886-BrocardGifford1"> [69] </xref> , <xref ref-type="bibr" rid="pone.0084886-Bassel1"> [70] </xref> . ERA1 is a farnesyl transferase which acts as a negative regulator of ABA response in regulation of seed germination and stomata opening, and the knockout mutant <italic> era1 </italic> resulted in increase in lateral <span class="trait">root</span> number <xref ref-type="bibr" rid="pone.0084886-Brady1"> [71] </xref> . In contrast, <span class="gene">GmNAC003</span> suppressed expression of <italic> ERA1 </italic> while promoted <italic> ABI3 </italic> and <italic> ABI5 </italic> to enhance ABA signaling <xref ref-type="bibr" rid="pone.0084886-Brady1"> [71] </xref> . Collectively, these data indicate that <span class="gene">GmNAC004</span> may enhance auxin signaling but suppress ABA signaling to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic> . The results show an additional crosstalk between the auxin and ABA signaling pathways and are in agreement with the views of De Smet <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0084886-DeSmet1"> [26] </xref> on the antagonistic roles of auxin and ABA in the regulation of LR <span class="trait">development.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Although transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> had greater LR number and <span class="trait">length</span> than the WT under both well-watered and mild water stress conditions, they did not show promotion of LR in response to water stress when compared to the well-watered condition ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> ). Thus, the results of this study do not support a role of <span class="gene">GmNAC004</span> in the enhanced LR number under water stress in soybean that has been reported previously <xref ref-type="bibr" rid="pone.0084886-Read1"> [2] </xref> , <xref ref-type="bibr" rid="pone.0084886-He1"> [4] </xref> . There are two possibilities: (i) the soybean <span class="gene">GmNAC004</span> protein does not have a function in increased LR <span class="trait">development</span> in soybean under water stress; or (ii) it has a function, but it is unable to increase LR number in <italic> Arabidopsis </italic> under water stress. The first possibility is supported by a similar study in <italic> Arabidopsis </italic> under salt stress <xref ref-type="bibr" rid="pone.0084886-He2"> [38] </xref> . The authors found that NaCl stress increased LR numbers of <italic> Arabidopsis </italic> . Under both normal and mild salinity stress conditions, transgenic plants overexpressing <italic> AtNAC2 </italic> , a salt stress-responsive gene, had greater LR numbers than the WT. However, LR number of <italic> AtNAC2 </italic> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the NaCl stress compared to the non-stress condition. In contrast, the <italic> nac2 </italic> mutant showed a similar response to the WT. This suggests that AtNAC2, although a promoter of LR number under normal conditions and induced by saline stress, was not the protein that regulated the enhanced LR number in <italic> Arabidopsis </italic> in response to salt stress. The second possibility is that <italic> Arabidopsis </italic> lacks of the regulatory system which supports a gene, either native or heterologous as <span class="gene">GmNAC004</span> to act on promotion of LR number under water deficit stress. This is supported by the fact that LR <span class="trait">development</span> in <italic> Arabidopsis </italic> is significantly inhibited in response to low water potential conditions induced by mannitol <xref ref-type="bibr" rid="pone.0084886-Deak1"> [28] </xref> or by PEG <xref ref-type="bibr" rid="pone.0084886-vanderWeele1"> [59] </xref> . In contrast to <italic> Arabidopsis </italic> , soybean shows stimulation of LR <span class="trait">development</span> under water deficit conditions <xref ref-type="bibr" rid="pone.0084886-Read1"> [2] </xref> , <xref ref-type="bibr" rid="pone.0084886-He1"> [4] </xref> . Therefore, in soybean, there may be a regulatory system that supports gene action to promote LR number under mild water deficit. To investigate this possibility, it would be useful to study the function of <span class="gene">GmNAC004</span> in the enhancement of LR number using transgenic soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, the findings of this study indicate that the soybean <span class="gene">GmNAC004</span> TF has a role in stimulating LR number and total LR <span class="trait">length</span> under non-stress conditions and maintains greater LR <span class="trait">development</span> under mild water stress, compared to the WT. However, LR <span class="trait">development</span> of <span class="gene">GmNAC004</span> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment. Nevertheless, <italic> <span class="gene">GmNAC004</span> </italic> can potentially be used for genetic engineering for a higher lateral <span class="trait">root</span> number, at least under normal growth conditions. Plants with early <span class="trait">development</span> of a large and extensive <span class="trait">root</span> system may benefit during later growth stages, where water deficit may occur. These plants can tolerate to <span class="trait">drought</span> by the dehydration avoidance mechanism to maintain tissue water content <xref ref-type="bibr" rid="pone.0084886-Manavalan1"> [9] </xref> , <xref ref-type="bibr" rid="pone.0084886-Turner1"> [72] </xref> that helps the plant survive and reproduce under <span class="trait">drought</span> conditions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean salicylic acid methyltransferase gene confers <span class="trait">resistance</span> to soybean cyst nematode.</h2>
			<h3>PMID:24034273  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M, Zhao N, Zhu JJ, Zhuang X, Liu W, Pantalone VR, Arelli PR, Stewart CN Jr, Chen F.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Salicylic acid plays a critical role in activating plant defence responses after pathogen attack. Salicylic acid methyltransferase (SAMT) modulates the level of salicylic acid by converting salicylic acid to methyl salicylate. Here, we report that a SAMT gene from soybean <span class="gene">(GmSAMT1)</span> plays a role in soybean defence against soybean cyst nematode (Heterodera glycines Ichinohe, SCN). <span class="gene">GmSAMT1</span> was identified as a candidate SCN defence-related gene in our previous analysis of soybean defence against SCN using GeneChip microarray experiments. The current study started with the isolation of the full-length cDNAs of <span class="gene">GmSAMT1</span> from a SCN-resistant soybean line and from a SCN-susceptible soybean line. The two cDNAs encode proteins of identical sequences. The <span class="gene">GmSAMT1</span> cDNA was expressed in Escherichia coli. Using in vitro enzyme assays, E. coli-expressed <span class="gene">GmSAMT1</span> was confirmed to function as salicylic acid methyltransferase. The apparent Km value of <span class="gene">GmSAMT1</span> for salicylic acid was approximately 46 μM. To determine the role of <span class="gene">GmSAMT1</span> in soybean defence against SCN, transgenic hairy roots overexpressing <span class="gene">GmSAMT1</span> were produced and tested for SCN <span class="trait">resistance.</span> Overexpression of <span class="gene">GmSAMT1</span> in SCN-susceptible backgrounds significantly reduced the <span class="trait">development</span> of SCN, indicating that overexpression of <span class="gene">GmSAMT1</span> in the transgenic hairy <span class="trait">root</span> system could confer <span class="trait">resistance</span> to SCN. Overexpression of <span class="gene">GmSAMT1</span> in transgenic hairy roots was also found to affect the expression of selected genes involved in salicylic acid biosynthesis and salicylic acid signal transduction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cloning and characterization of an aldehyde dehydrogenase gene <span class="gene">GmALDH3-1</span> from soybean].</h2>
			<h3>PMID:20466639  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huang F(1), Chi YJ, He H, Yu DY.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To research the mechanism of soybean reproductive <span class="trait">development,</span> we identified a number of flower <span class="trait">development</span> related genes in soybean by microarray hybridization. A gene predominately expressed in soybean flowers was chosen for further analysis. Through bioinformatic and RT-PCR approaches, the full-length gene was cloned from soybean flowers. The results of BLAST searching indicated that this gene encoded for an aldehyde dehydrogenase and was named as <span class="gene">GmALDH3-1.</span> <span class="gene">GmALDH3-1</span> contains a complete open reading frame of 1485 bp in <span class="trait">length,</span> which encodes for a peptide of 494 amino acids. The product encoded by <span class="gene">GmALDH3-1</span> shows 83% similarity and 68% identity with Populus tomentosa PtALDH3, respectively, and 39% and 59% with human ALDH3B. Phylogenetic analysis shows that <span class="gene">GmALDH3-1</span> and other ALDH3 subfamily members are grouped into the same branch and <span class="gene">GmALDH3-1</span> is close to PtALDH3 and Arabidopsis AtALDH3F1. Real-time RT-PCR analysis demonstrated that the highest expression level of <span class="gene">GmALDH3-1</span> occurred in flowers, but the expression of this gene was almost undetectable in leaves and roots. We further analyzed <span class="gene">GmALDH3-1</span> expression during the course of seed <span class="trait">development</span> based on publicly available microarray data and found that <span class="gene">GmALDH3-1</span> was highly expressed in the seed endothelium, epidermis, outer integument, and hilum.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:1-Aminocyclopropane-1-carboxylic acid (ACC) concentration and ACC synthase expression in soybean roots, <span class="trait">root</span> tips, and soybean cyst nematode (Heterodera glycines)-infected roots.</h2>
			<h3>PMID:19861652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Xue P, Yang R.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Colonization of plant roots by <span class="trait">root</span> knot and cyst nematodes requires a functional ethylene response pathway. However, ethylene plays many roles in <span class="trait">root</span> <span class="trait">development</span> and whether its role in nematode colonization is direct or indirect, for example lateral <span class="trait">root</span> initiation or <span class="trait">root</span> hair growth, is not known. The temporal requirement for ethylene and localized synthesis of ethylene during the life span of soybean cyst nematode (SCN) on soybean roots was further investigated. Although a significant increase in ethylene evolution was not detected from SCN-colonized roots, the concentration of the immediate precursor to ethylene, 1-aminocyclopropane-1-carboxylic acid (ACC), was higher in SCN-colonized <span class="trait">root</span> pieces and <span class="trait">root</span> tips than in other parts of the <span class="trait">root.</span> Moreover, expression analysis of 17 ACC synthase (ACS) genes indicated that a select set of <span class="gene">ACS</span> genes is expressed in SCN-colonized <span class="trait">root</span> pieces that is clearly different from the set of genes expressed in non-colonized roots or <span class="trait">root</span> tips. Semi-quantitative real-time PCR indicated that <span class="gene">ACS</span> transcript accumulation correlates with the high concentration of ACC in <span class="trait">root</span> tips. In addition, an ACS-like sequence was found in the public SCN nucleotide database. Acquisition of a full-length sequence for this mRNA (accession GQ389647) and alignment with transcripts for other well-characterized <span class="gene">ACS</span> proteins indicated that the nematode sequence is missing a key element required for <span class="gene">ACS</span> activity and therefore probably is not a functional ACS. Moreover, no significant amount of ACC was found in any growth stage of SCN that was tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean-Phytophthora <span class="trait">resistance</span> locus Rps1-k encompasses coiled coil-nucleotide binding-leucine rich repeat-like genes and repetitive sequences.</h2>
			<h3>PMID:18366691  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao H(1), Bhattacharyya MK.
				<br/> 
				年份：2008 Mar 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: A series of Rps <span class="trait">(resistance</span> to Pytophthora sojae) genes have been protecting soybean from the <span class="trait">root</span> and stem rot disease caused by the Oomycete pathogen, Phytophthora sojae. Five Rps genes were mapped to the Rps1 locus located near the 28 cM map position on molecular linkage group N of the composite genetic soybean map. Among these five genes, Rps1-k was introgressed from the cultivar, Kingwa. Rps1-k has been providing stable and broad-spectrum Phytophthora <span class="trait">resistance</span> in the major soybean-producing regions of the United States. Rps1-k has been mapped and isolated. More than one functional Rps1-k gene was identified from the Rps1-k locus. The clustering feature at the Rps1-k locus might have facilitated the expansion of Rps1-k gene numbers and the generation of new recognition specificities. The Rps1-k region was sequenced to understand the possible evolutionary steps that shaped the generation of Phytophthora <span class="trait">resistance</span> genes in soybean. RESULTS: Here the analyses of sequences of three overlapping BAC clones containing the 184,111 bp Rps1-k region are reported. A shotgun sequencing strategy was applied in sequencing the BAC contig. Sequence analysis predicted a few full-length genes including two Rps1-k genes, <span class="gene">Rps1-k-1</span> and Rps1-k-2. Previously reported Rps1-k-3 from this genomic region 1 was evolved through intramolecular recombination between <span class="gene">Rps1-k-1</span> and <span class="gene">Rps1-k-2</span> in Escherichia coli. The majority of the predicted genes are truncated and therefore most likely they are nonfunctional. A member of a highly abundant retroelement, SIRE1, was identified from the Rps1-k region. The Rps1-k region is primarily composed of repetitive sequences. Sixteen simple repeat and 63 tandem repeat sequences were identified from the locus. CONCLUSION: These data indicate that the Rps1 locus is located in a gene-poor region. The abundance of repetitive sequences in the Rps1-k region suggested that the location of this locus is in or near a heterochromatic region. Poor recombination frequencies combined with presence of two functional Rps genes at this locus has been providing stable Phytophthora <span class="trait">resistance</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A RAV-like transcription factor controls photosynthesis and senescence in soybean.</h2>
			<h3>PMID:18297307  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao L(1), Luo Q, Yang C, Han Y, Li W.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA library enriched for mRNAs encoding ESTs that increased in abundance during short days was constructed by SSH from leaf tissues of a photoperiod sensitive soybean. The proteins predicted to be encoded by the mRNAs were inferred to be involved in diverse functions. A full-length mRNA that encoded a soybean ortholog of the transcription factor <span class="gene">RAV</span> was isolated by RACE, containing an open reading frame of 1,056 bp. The <span class="gene">GmRAV</span> protein included an AP2/ERF domain and a B3 domain. <span class="gene">GmRAV</span> mRNA abundance was increased in SDs following leaf treatments with ABA and decreased following BR treatment. Transgenic tobacco overexpressing <span class="gene">GmRAV</span> showed morphological and physiological alterations such as slower plant growth rate (dwarfing), reduced <span class="trait">root</span> elongation, delayed flowering time and reduced photosynthetic rate, reduced chlorophyll contents in leaves. Therefore <span class="gene">GmRAV</span> may be a negative regulator acting on both photosynthesis and growth. Transgenic tobacco also showed accelerated senescence with both dark and ABA treatments versus the longer longevity compared to the wild type in LDs. The analyses of soybean leaf, <span class="trait">root</span> and stem organs showed that <span class="gene">GmRAV</span> mRNA abundances were higher in SDs than in LDs. Therefore, the enhanced expression of <span class="gene">GmRAV</span> in SDs compared to LDs may have caused the inhibited growth of soybean leaf, <span class="trait">root</span> and stem.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure, expression, and mapping of two nodule-specific genes identified by mining public soybean EST databases.</h2>
			<h3>PMID:16973305  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong SC(1), Yang K, Park JY, Han KS, Yu S, Hwang TY, Hur CG, Kim SH, Park PB, Kim HM, Park YI, Liu JR.
				<br/> 
				年份：2006 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Numerous nodule-specific genes, which are involved in the <span class="trait">root</span> nodule <span class="trait">development</span> and function, have been known and are still being discovered. Here, we reported the structure, expression, and genetic map location of two novel nodule-specific genes. First, two EST groups, one obtained from a nodule library and the other from all aboveground tissue libraries, were clustered with regard to in silico expression profiles. We compiled a pool of 103 putative nodule-specific sequence clusters. Then, two representative ESTs were selected for further experimental analyses. According to the full-length cDNA sequences, one was an EST of a novel nodule-specific polygalacturonase gene, <span class="gene">GmPGN,</span> and the other was an EST of a new short nodule-specific gene, <span class="gene">GmEKN.</span> The results of expression analyses of the <span class="gene">GmPGN</span> cDNAs indicated that <span class="gene">GmPGN</span> expression was not detectable in any of the soybean tissues except in the nodule tissue and may be regulated via alternative splicing. <span class="gene">GmEKN</span> expression was the most strongly detected in the nodule. The predicted <span class="gene">GmEKN</span> protein is both glutamic acid- and lysine-rich, and is also highly hydrophilic. Genetic mapping located <span class="gene">GmPGN</span> near the known quantitative trait locus conferring <span class="trait">resistance</span> to soybean cyst nematode on soybean molecular linkage group (MLG) B1, and <span class="gene">GmEKN</span> on MLG A2. These results provide useful information for the use of these genes in research on the orchestration of numerous genes in nodule <span class="trait">development</span> and function.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of an Arabidopsis phosphoglycerate mutase homologue is localized to apical meristems, regulated by hormones, and induced by sedentary plant-parasitic nematodes.</h2>
			<h3>PMID:15010616  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mazarei M(1), Lennon KA, Puthoff DP, Rodermel SR, Baum TJ.
				<br/> 
				年份：2003 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We previously isolated a partial soybean cDNA clone whose transcript abundance is increased upon infection by the sedentary, endoparasitic soybean cyst nematode Heterodera glycines. We now isolated the corresponding full-length cDNA and determined that the predicted gene product was similar to the group of cofactor-dependent phosphoglycerate mutase/bisphosphoglycerate mutase enzymes (PGM/bPGM; EC 5.4.2.1/5.4.2.4). We designated the corresponding soybean gene <span class="gene">GmPGM.</span> <span class="gene">PGM</span> and bPGM are key catalysts of glycolysis that have been well characterized in animals but not plants. Using the <span class="gene">GmPGM</span> cDNA sequence, we identified a homologous Arabidopsis thaliana gene, which we designated AtPGM. Histochemical GUS analyses of transgenic Arabidopsis plants containing the AtPGM promoter ::GUS construct revealed that the AtPGM promoter directs GUS expression in uninfected plants only to the shoot and <span class="trait">root</span> apical meristems. In infected plants, GUS staining also is evident in the nematode feeding structures induced by the cyst nematode Heterodera schachtii and by the root-knot nematode Meloidogyne incognita. Furthermore, we discovered that the AtPGM promoter was down-regulated by abscisic acid and hydroxyurea, whereas it was induced by sucrose, oryzalin, and auxin, thereby revealing expression characteristics typical of genes with roles in meristematic cells. Assessment of the auxin-inducible AUX1 gene promoter (a gene coding for a polar auxin transport protein) similarly revealed feeding cell and meristem expression, suggesting that auxin may be responsible for the observed tissue specificity of the AtPGM promoter. These results provide first insight into the possible roles of PGM/bPGM in plant physiology and in plant-pathogen interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterogeneity of sucrose synthase genes in bean (Phaseolus vulgaris L.): evidence for a nodule-enhanced sucrose synthase gene.</h2>
			<h3>PMID:12554718  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Silvente S(1), Camas A, Lara M.
				<br/> 
				年份：2003 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS), the key sucrose hydrolytic enzyme (EC 2.4.1.13), plays an important role in N(2)-fixing nodule metabolism. It has also been proposed that N(2) fixation in soybean nodules could be mediated by the potential to metabolize sucrose. The isolation and characterization of a nodule-enhanced <span class="gene">SS</span> full-length cDNA clone from the bean Phaseolus vulgaris is reported here. Southern blot analysis indicated that there are at least two <span class="gene">SS</span> genes in beans. Using a 3' specific probe from this <span class="gene">SS</span> cDNA clone, it was possible to identify a nodule-enhanced <span class="gene">SS</span> gene (PvSSn), which is expressed almost exclusively in nodules. A second gene (PvSS), which is expressed in all tissues tested, was detected using a coding region probe. Nodule-enhanced PvSSn transcript levels, but not the enzyme activity or protein amount, is reduced during nodule <span class="trait">development.</span> These data indicated that this reduction could be due to a limitation on the carbon availability in the nodule. PvSSn expression is reduced in the asparagine-treated nodules. By contrast, PvSSn transcript levels in nodules increased in the presence of glutamine, allantoin and allopurinol. This result suggests a relationship between ureide transport and <span class="gene">SS</span> regulation and could help in understanding why the ureide transport mechanism is activated during nitrogen fixation in bean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Carbon metabolism in developing soybean <span class="trait">root</span> nodules: the role of carbonic anhydrase.</h2>
			<h3>PMID:10656581  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kavroulakis N(1), Flemetakis E, Aivalakis G, Katinakis P.
				<br/> 
				年份：2000 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA clone encoding carbonic anhydrase (CA) was isolated from a soybean nodule cDNA library. In situ hybridization and immunolocalization were performed in order to assess the location of CA transcripts and protein in developing soybean nodules. CA transcripts and protein were present at high levels in all cell types of young nodules, whereas in mature nodules they were absent from the central tissue and were concentrated in cortical cells. The results suggested that, in the earlier stages of nodule <span class="trait">development,</span> CA might facilitate the recycling of CO2 while at later stages it may facilitate the diffusion of CO2 out of the nodule system. In parallel, sucrose metabolism was investigated by examination of the temporal and spatial transcript accumulation of sucrose synthase (SS) and phosphoenolpyruvate carboxylase (PEPC) genes, with in situ hybridization. In young nodules, high levels of <span class="gene">SS</span> gene transcripts were found in the central tissue as well as in the parenchymateous cells and the vascular bundles, while in mature nodules the levels of <span class="gene">SS</span> gene transcripts were much lower, with the majority of the transcripts located in the parenchyma and the pericycle cells of the vascular bundles. High levels of expression of PEPC gene transcripts were found in mature nodules, in almost all cell types, while in young nodules lower levels of transcripts were detected, with the majority of them located in parenchymateous cells as well as in the vascular bundles. These data suggest that breakdown of sucrose may take place in different sites during nodule <span class="trait">development.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of flower <span class="trait">development</span> related gene GsLFY from Glycine soja.</h2>
			<h3>PMID:28115306  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo WY(1), Cui YM(1), Wang TT(1), Yu DY(1), Huang F(1).
				<br/> 
				年份：2017 Jan 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						LEAFY/FLORICAULA (LFY/FLO) is a family of plant-specific transcription factors, which plays an important role(s) in the regulation of floral organ formation and <span class="trait">development.</span> So far, <span class="gene">LFY</span> regulation on floral <span class="trait">development</span> in wild soybean has not been reported in the literature. In this study, the <span class="gene">LFY</span> gene, GsLFY, has been isolated from Glycine soja, and characterized with molecular and transgenic techniques. The cDNA for GsLFY gene is 1224 bp in <span class="trait">length</span> and contains an open reading frame encoding a polypeptide of 407 amino acids. Quantitative RT-PCR analysis shows that GsLFY is prominently expressed in various tissues, including roots, flowers and seeds. Among the four floral organs, GsLFY is expressed highly in sepals and stamens while weakly in the petals and carpels. Yeast two-hybrid experiments show that GsLFY possesses transactivation activity while transient expression analysis with Arabidopsis thaliana protoplasts shows that GsLFY protein is localized in the nucleus, supporting the notion that GsLFY is a transcription factor. The GsLFY transgenic tobacco plants flower about 29 days earlier than the wild-type plants, thereby providing a potential rationale for developing new soybean varieties with altered flowering time.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Pure culture of Metarhizium anisopliae LHL07 reprograms soybean to higher growth and mitigates salt stress.</h2>
			<h3>PMID:22805930  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Khan AL(1), Hamayun M, Khan SA, Kang SM, Shinwari ZK, Kamran M, Ur Rehman S, Kim JG, Lee IJ.
				<br/> 
				年份：2012 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Little is known about the role of endophytic fungi against abiotic stresses and isoflavonoids (IF) contents of soybean. In current study, we investigated the role of fungal endophytes on the growth of soybean under salt stress conditions. Pure cultures of nine endophytic fungi were isolated from the roots of field-grown soybean plants, and their culture filtrates were screened on Waito-C and Dongjin-byeo rice cultivars; for identification of plant growth promoting fungal strains. It was observed that fungal isolate <span class="gene">GMC-2B</span> significantly promoted the growth of both Waito-C and Dongjin-byeo. <span class="gene">GMC-2B</span> was later identified as a new strain of Metarhizium anisopliae LHL07 on the basis of 18S rDNA sequences and phylogenetic analysis. Metarhizium anisopliae LHL07 inoculated soybean plants recorded significantly higher shoot <span class="trait">length,</span> shoot fresh and dry biomass, chlorophyll contents, transpiration rate, photosynthetic rate and leaf area; under sodium chloride induced salt stress as compared to non-inoculated control plants. An elevated proline and reduced superoxide dismutase and malondialdehyde contents in M. anisopliae LHL07 inoculated soybean plants demonstrated mitigation of salt induced oxidative stress. Furthermore, reduced abscisic acid and elevated jasmonic acid contents in soybean plants confirmed that lesser stress was convened to M. anisopliae inoculated-plants under salinity stress. We also assessed the role of M. anisopliae interaction on IF biosynthesis of soybean, and found significantly higher IF contents in M. anisopliae inoculated soybean plants. In conclusion, endophytic fungal interactions with soybean can be beneficial to improve soybean quality and quantity under salt affected agricultural systems.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Salinity stress <span class="trait">resistance</span> offered by endophytic fungal interaction between Penicillium minioluteum LHL09 and glycine max. L.</h2>
			<h3>PMID:21952365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Khan AL(1), Hamayun M, Ahmad N, Hussain J, Kang SM, Kim YH, Adnan M, Tang DS, Waqas M, Radhakrishnan R, Hwang YH, Lee IJ.
				<br/> 
				年份：2011 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Endophytic fungi are little known for their role in gibberellins (GAs) synthesis and abiotic stress <span class="trait">resistance</span> in crop plants. We isolated 10 endophytes from the roots of field-grown soybean and screened their culture filtrates (CF) on the GAs biosynthesis mutant rice line - Waito-C. CF bioassay showed that endophyte <span class="gene">GMH-1B</span> significantly promoted the growth of Waito-C compared with controls. <span class="gene">GMH-1B</span> was identified as Penicillium minioluteum LHL09 on the basis of ITS regions rDNA sequence homology and phylogenetic analyses. GC/MS-SIM analysis of CF of P. minioluteum revealed the presence of bioactive GA(4) and GA(7). In endophyte-soybean plant interaction, P. minioluteum association significantly promoted growth characteristics (shoot <span class="trait">length,</span> shoot fresh and dry biomasses, chlorophyll content, and leaf area) and nitrogen assimilation, with and without sodium chloride (NaCl)-induced salinity (70 and 140 mM) stress, as compared with control. Field-emission scanning electron microcopy showed active colonization of endophyte with host plants before and after stress treatments. In response to salinity stress, low endogenous abscisic acid and high salicylic acid accumulation in endophyte-associated plants elucidated the stress mitigation by P. minioluteum. The endophytic fungal symbiosis of P. minioluteum also increased the daidzein and genistein contents in the soybean as compared with control plants, under salt stress. Thus, P. minioluteum ameliorated the adverse effects of abiotic salinity stress and rescued soybean plant growth by influencing biosynthesis of the plant's hormones and flavonoids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide survey and expression analysis of the plant-specific NAC transcription factor family in soybean during <span class="trait">development</span> and dehydration stress.</h2>
			<h3>PMID:21685489  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Le DT(1), Nishiyama R, Watanabe Y, Mochida K, Yamaguchi-Shinozaki K, Shinozaki K, Tran LS.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-specific NAC transcription factors (TFs) play important roles in regulating diverse biological processes, including <span class="trait">development,</span> senescence, growth, cell division and responses to environmental stress stimuli. Within the soybean genome, we identified 152 full-length <span class="gene">GmNAC</span> TFs, including 11 membrane-bound members. In silico analysis of the <span class="gene">GmNACs,</span> together with their Arabidopsis and rice counterparts, revealed similar NAC architecture. Next, we explored the soybean Affymetrix array and Illumina transcriptome sequence data to analyse tissue-specific expression profiles of <span class="gene">GmNAC</span> genes. Phylogenetic analysis using stress-related NAC TFs from Arabidopsis and rice as seeding sequences identified 58 of the 152 <span class="gene">GmNACs</span> as putative stress-responsive genes, including eight previously reported dehydration-responsive <span class="gene">GmNACs.</span> We could design gene-specific primers for quantitative real-time PCR verification of 38 out of 50 newly predicted stress-related genes. Twenty-five and six <span class="gene">GmNACs</span> were found to be induced and repressed 2-fold or more, respectively, in soybean roots and/or shoots in response to dehydration. <span class="gene">GmNAC085,</span> whose amino acid sequence was 39%; identical to that of well-known SNAC1/ONAC2, was the most induced gene upon dehydration, showing 390-fold and 20-fold induction in shoots and roots, respectively. Our systematic analysis has identified excellent tissue-specific and/or dehydration-responsive candidate <span class="gene">GmNAC</span> genes for in-depth characterization and future <span class="trait">development</span> of improved drought-tolerant transgenic soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of a glutathione S-transferase gene, GsGST, from wild soybean (Glycine soja) enhances <span class="trait">drought</span> and salt tolerance in transgenic tobacco.</h2>
			<h3>PMID:20383560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ji W(1), Zhu Y, Li Y, Yang L, Zhao X, Cai H, Bai X.
				<br/> 
				年份：2010 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine soja is a species of soybean that survives in adverse environments including high salt and <span class="trait">drought</span> conditions. We constructed a cDNA library from G. soja seedlings treated with NaCl and isolated a glutathione S-transferase gene (GsGST: GQ265911) from the library. The cDNA encoding GsGST contains an open reading frame of 660 bp and the predicted protein belongs to the tau class of <span class="gene">GST</span> family proteins. Tobacco plants over-expressing the GsGST gene showed sixfold higher <span class="gene">GST</span> activity than wild-type plants. Transgenic tobacco plants exhibited enhanced dehydration tolerance. T(2) transgenic tobacco plants showed higher tolerance at the seedling stage than wild-type plants to salt and mannitol as demonstrated by longer <span class="trait">root</span> <span class="trait">length</span> and less growth retardation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homologous soybean and Arabidopsis genes share responsiveness to cyst nematode infection.</h2>
			<h3>PMID:20565617</h3>
				

				
			<p>
				作者：Mazarei M(1), Lennon KA, Puthoff DP, Rodermel SR, Baum TJ.
				<br/> 
				年份：2004 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SUMMARY We previously isolated a partial soybean cDNA clone (D17.1) whose corresponding transcript increases in susceptible roots 1 day post inoculation (dpi) with the soybean cyst nematode, Heterodera glycines. Here we isolated the corresponding full-length cDNA from a soybean cDNA library and designated this gene of unknown function <span class="gene">Gm17.1.</span> Time course RNA gel blot analyses revealed that <span class="gene">Gm17.1</span> mRNA steady-state levels were elevated in soybean roots following H. glycines infection up to at least 6 dpi. For further in-depth study we identified a homologous Arabidopsis thaliana gene and designated this gene At17.1. Arabidopsis is successfully infected by the sugar beet cyst nematode (H. schachtii), a close relative of H. glycines. We isolated the At17.1 promoter, fused it to the beta-glucuronidase (GUS) reporter gene, and transformed this construct into Arabidopsis plants as well as soybean hairy roots. Histochemical analysis of plant materials containing the At17.1::GUS construct revealed that the At17.1 promoter is functional in Arabidopsis as well as in soybean and that during normal plant <span class="trait">development</span> the At17.1 promoter directs GUS expression predominantly to the vascular tissues and <span class="trait">root</span> tips of both plant species. When At17.1::GUS Arabidopsis plants and soybean hairy roots were inoculated with cyst nematodes, strong GUS activity was detected within the cyst nematode-induced feeding structures. Further tests of At17.1 promoter activity in Arabidopsis revealed that this promoter was induced by auxin, jasmonic acid, mannitol and dehydration. Quantitative real-time reverse transcription-polymerase chain reaction assays of At17.1 expression confirmed the observed promoter characteristics. Based on our expression data and the observation that both the soybean and the Arabidopsis homologues behaved in a similar fashion following cyst nematode infection, it is likely that these genes are closely associated with cyst nematode parasitism of plants, potentially with hormone and osmotic changes occurring in the developing nematode feeding cells. Furthermore, these data provide additional insights into the strengths of the Arabidopsis-H. schachtii pathosystem to study cyst nematode-plant interactions in lieu of less tractable pathosystems. This finding is supported by the fact that the Arabidopsis promoter tested here produced similar results in Arabidopsis and soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthetic responses of corn and soybean to foliar application of salicylates.</h2>
			<h3>PMID:12806776  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Khan W(1), Prithiviraj B, Smith DL.
				<br/> 
				年份：2003 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Salicylic acid (SA) and related phenolic compounds can exert control over stomatal opening and previous work in our laboratory has shown that chronic injection of SA increases the photosynthetic rate of corn. The work reported in this paper investigated the effects of foliar applied SA, acetyl salicylic acid (ASA) and gentisic acid (GTA) on photosynthetic rates and growth of soybean (a <span class="gene">C3</span> plant) and corn (a <span class="gene">C4</span> plant) under greenhouse conditions. In general, the tested compounds enhanced photosynthetic rates in both soybean and corn. Stomatal conductance and transpiration were also increased. These compounds do not alter chlorophyll content. In some cases treatment with these compounds resulted in increased leaf areas and plant dry mass, however, plant height and <span class="trait">root</span> <span class="trait">length</span> were not affected.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxidase and phenylalanine ammonia-lyase activities, phenolic acid contents, and allelochemicals-inhibited <span class="trait">root</span> growth of soybean.</h2>
			<h3>PMID:12125206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Herrig V(1), Ferrarese Mde L, Suzuki LS, Rodrigues JD, Ferrarese-Filho O.
				<br/> 
				年份：2002
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The influence of the allelochemicals ferulic (FA) and vanillic (VA) acids on peroxidase (POD, EC 1.11.1.7) and phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) activities and their relationships with phenolic acid (PhAs) contents and <span class="trait">root</span> growth of soybean (Glycine max (L.) Merr.) were examined. Three-day-old seedlings were cultivated in nutrient solution containing FA or VA (0.1 to 1 mM) for 48 h. Both compounds (at 0.5 and 1 mM) decreased <span class="trait">root</span> <span class="trait">length</span> (RL), fresh weight (FW) and dry weight (DW) and increased PhAs contents. At 0.5 and 1 mM, FA increased soluble POD activity (18% and 47%, respectively) and cell wall (CW)-bound POD activity (61% and 34%), while VA increased soluble POD activity (33% and 17%) but did not affect CW-bound POD activity. At 1 mM, FA increased (82%) while VA reduced (32%) <span class="gene">PAL</span> activities. The results are discussed on the basis of the role of these compounds on phenylpropanoid metabolism and <span class="trait">root</span> growth and suggest that the effects caused on POD and <span class="gene">PAL</span> activities are some of the many mechanisms by which allelochemicals influence plant growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Resistance and resilience of <span class="trait">root</span> fungal communities to water limitation in a temperate agroecosystem</h2>
			<h3>PMID:28515880</h3>
				
					<h3>PMCID:5433968</h3>
					<h4>期刊:Ecology and Evolution</h4>
				

				
			<p>
				作者：<contrib id="ece32900-cr-0001" contrib-type="author">  <name>  <surname> Furze </surname>  <given-names> Jessie R. </given-names>  </name>  <xref ref-type="aff" rid="ece32900-aff-0001">
				<br/> 
				年份：<day> 21 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						There is considerable uncertainty regarding how climate change and associated expected shifts in temperature and precipitation regimes will influence agricultural crop production worldwide (Lobell &#x00026; Field, <xref rid="ece32900-bib-0043" ref-type="ref"> 2007 </xref> ; Schmidhuber &#x00026; Tubiello, <xref rid="ece32900-bib-0068" ref-type="ref"> 2007 </xref> ; Ziervogel &#x00026; Ericksen, <xref rid="ece32900-bib-0077" ref-type="ref"> 2010 </xref> ). On the one hand, certain models predicting crop growth and <span class="trait">yield</span> from physiological traits suggest that increased growing season temperatures, coupled with more variable and less evenly distributed precipitation, may result in net decreases in crop yields of 1%&#x02013;13% (Nelson et&#x000a0;al., <xref rid="ece32900-bib-0053" ref-type="ref"> 2010 </xref> ). Alternatively, other models predict increased crop yields in certain regions in response to warmer conditions and extended growing seasons (Smith et&#x000a0;al., <xref rid="ece32900-bib-0070" ref-type="ref"> 2013 </xref> ), coupled with elevated CO <sub> 2 </sub> concentrations that may increase crop water use efficiency and biomass accumulation; however, these responses may differ widely (at least) between <span class="gene">C3</span> and <span class="gene">C4</span> crops (McGrath &#x00026; Lobell, <xref rid="ece32900-bib-0049" ref-type="ref"> 2013 </xref> ; Yang et&#x000a0;al., <xref rid="ece32900-bib-0076" ref-type="ref"> 2014 </xref> ). Although a consensus understanding of how climate change will influence crop growth and <span class="trait">yield</span> remains elusive, there is considerable interest and urgency for understanding the mechanisms by which crops will respond to changing climate, in order to develop agricultural adaptation strategies (Beebe et&#x000a0;al., <xref rid="ece32900-bib-0006" ref-type="ref"> 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Primers designed by Lee, Lee, and Young ( <xref rid="ece32900-bib-0039" ref-type="ref"> 2008 </xref> ) to amplify the small ribosomal subunit (SSU) (AML1 5&#x02032;&#x02010;ATC AAC TTT CGA TGG TAG GAT AGA&#x02010;3&#x02032; forward and AML2 5&#x02032;&#x02010;GAA CCC <span class="gene">AAA</span> CAC TTT GGT TTC C&#x02010;3&#x02032; reverse) were used for amplifying AMF exclusively (Phylum: Glomeromycota). All primers were labeled with fluorescent dyes phosphoramidite, 6&#x02010;FAM and HEX (forward and reverse labeled at the 5&#x02032; end, respectively; Invitrogen, CA). Twenty &#x003bc;l amplification reactions consisted of 10&#x000a0;&#x003bc;l of HotStar Taq Plus Master Mix (Qiagen, CA), 1&#x000a0;&#x003bc;l of each primer at 10&#x000a0;&#x003bc;molL <sup> &#x02010;1 </sup> , 7&#x000a0;&#x003bc;l of RNA&#x02010;free water, and 1&#x000a0;&#x003bc;l of DNA template. Reactions were incubated in a PTC&#x02010;100 thermal cycler (MJ Research Inc., Waltham, Massachusetts, USA) under the following conditions: DNA polymerase initialization at 95&#x000b0;C for 50&#x000a0;s, followed by 34 cycles at 95&#x000b0;C for 50&#x000a0;s, 55.5&#x000b0;C for 50&#x000a0;s, 72&#x000b0;C for 60&#x000a0;s, ending with a final extension at 72&#x000b0;C for 10&#x000a0;min. 20&#x000a0;&#x003bc;l RNA&#x02010;free water was run as a negative control. Amplicons were digested using the restriction enzymes <italic> Alu </italic> I and <italic> Hinf </italic> I for 2&#x000a0;hr at 37&#x000b0;C (Fermentas Canada Inc., Burlington, Ontario, CA). Digests contained 15&#x000a0;&#x003bc;l of PCR product, 2U of each <italic> Alu </italic> I and <italic> Hinf </italic> I, 2&#x000a0;&#x003bc;l Tango10X buffer, and 2.6&#x000a0;&#x003bc;l RNA&#x02010;free water. AML amplicons of approximately 60&#x02013;812&#x000a0;bp in <span class="trait">length</span> were successfully obtained from <span class="trait">root</span> DNA.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of cDNA for nodulin-75 of soybean: A gene product involved in early stages of <span class="trait">root</span> nodule <span class="trait">development.</span></h2>
			<h3>PMID:16593857</h3>
				

				
			<p>
				作者：Franssen HJ(1), Nap JP, Gloudemans T, Stiekema W, Van Dam H, Govers F, Louwerse J, Van Kammen A, Bisseling T.
				<br/> 
				年份：1987 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Establishment of a nitrogen-fixing <span class="trait">root</span> nodule is accompanied by a developmentally regulated expression of nodulin genes, only some of which, the so-called early nodulin genes, are expressed in stages preceding actual nitrogen fixation. We have isolated soybean cDNA clones representing early nodulin genes and have studied clone pENOD2 in detail. The cDNA insert of this clone hybridizes to nodule-specific RNA of 1200 nucleotides in <span class="trait">length.</span> The RNA that was hybrid-selected by the cloned <span class="gene">ENOD2</span> DNA was in vitro translated to produce two nodulins with an apparent M(r) of 75,000, the N-75 nodulins. These two nodulins differ slightly in charge and one does not contain methionine. The amino acid sequence deduced from the DNA sequence shows that proline accounts for 45% of the 240 residues in these nodulins and the sequence contains at least 20 repeating heptapeptide units. The amino acid composition of none of the (hydroxy)proline-rich (glyco)proteins described in plants resembles the composition of the N-75 nodulins, especially with respect to the high glutamic acid and the low serine content. This suggests that the N-75 nodulins belong to a hitherto unidentified class of presumably structural proteins. The genes encoding the N-75 nodulins were found to be expressed in nodule-like structures devoid of intracellular bacteria and infection threads, indicating that these nodulins do not function in the infection process but more likely function in nodule morphogenesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lignification and related enzymes in Glycine max <span class="trait">root</span> growth-inhibition by ferulic acid.</h2>
			<h3>PMID:15303323  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：dos Santos WD(1), Ferrarese Mde L, Finger A, Teixeira AC, Ferrarese-Filho O.
				<br/> 
				年份：2004 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Changes in soluble and cell wall bound peroxidase (POD, EC 1.11.1.7) activity, phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) activity, and lignin content in roots of ferulic acid-stressed soybean (Glycine max (L.) Merr.) seedlings and their relationships with <span class="trait">root</span> growth were investigated. Three-day-old soybean seedlings were cultivated in half-strength Hoagland nutrient solution containing 1.0 mM ferulic acid for 24-72 hr. <span class="trait">Length,</span> fresh weight, and dry weight of roots decreased, while soluble and cell wall bound POD activity, <span class="gene">PAL</span> activity, and lignin content increased after ferulic acid treatment. These enzymes probably participate in <span class="trait">root</span> growth reduction in association with cell wall stiffening related to the formation of cross-linking among cell wall polymers and lignin production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Characterization of Major Intrinsic Proteins in Four Grass Plants and Their Non-Aqua Transport Selectivity Profiles with Comparative Perspective</h2>
			<h3>PMID:27327960</h3>
				

				
					<h3>PMCID:4915720</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Azad </surname>  <given-names> Abul Kalam </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ahmed </surname>  <given-names> Jahed </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alum </surname>  <given-names> Md. Asraful </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hasan </surname>  <given-names> Md. Mahbub </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ishikawa </surname>  <given-names> Takahiro </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sawa </surname>  <given-names> Yoshihiro </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Katsuhara </surname>  <given-names> Maki </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Major intrinsic proteins (MIPs), commonly known as aquaporins, transport not only water in plants but also other substrates of physiological significance and heavy metals. In most of the higher plants, MIPs are divided into five subfamilies (PIPs, TIPs, NIPs, SIPs and XIPs). Herein, we identified 68, 42, 38 and 28 full-length MIPs, respectively in the genomes of four monocot grass plants, specifically Panicum virgatum, Setaria italica, Sorghum bicolor and Brachypodium distachyon. Phylogenetic analysis showed that the grass plants had only four MIP subfamilies including PIPs, TIPs, NIPs and SIPs without XIPs. Based on structural analysis of the homology models and comparing the primary selectivity-related motifs [two NPA regions, aromatic/arginine (ar/R) selectivity filter and Froger's positions (FPs)] of all plant MIPs that have been experimentally proven to transport non-aqua substrates, we predicted the transport profiles of all MIPs in the four grass plants and also in eight other plants. Groups of MIP subfamilies based on ar/R selectivity filter and FPs were linked to the non-aqua transport profiles. We further deciphered the substrate selectivity profiles of the MIPs in the four grass plants and compared them with their counterparts in rice, maize, soybean, poplar, cotton, Arabidopsis thaliana, Physcomitrella patens and Selaginella moellendorffii. In addition to two NPA regions, ar/R filter and FPs, certain residues, especially in loops B and C, contribute to the functional distinctiveness of MIP groups. Expression analysis of transcripts in different organs indicated that non-aqua transport was related to expression of MIPs since most of the unexpressed MIPs were not predicted to facilitate the transport of non-aqua molecules. Among all MIPs in every plant, <span class="gene">TIP</span> (BdTIP1;1, SiTIP1;2, SbTIP2;1 and PvTIP1;2) had the overall highest mean expression. Our study generates significant information for understanding the diversity, evolution, non-aqua transport profiles and insight into comparative transport selectivity of plant MIPs, and provides tools for the <span class="trait">development</span> of transgenic plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Identification and characterization of the <italic> MIP </italic> gene family is the first step in investigating the role of MIPs in plant water relationships or transporting physiologically important small molecules. Grasses, plants of the Poaceae family, the largest plant family in the world, afford the bulk of human nutrition, and highly productive grasses are potential sources of sustainable biofuels [ <xref rid="pone.0157735.ref039" ref-type="bibr"> 39 </xref> , <xref rid="pone.0157735.ref040" ref-type="bibr"> 40 </xref> ]. Phytozome ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/"> www.phytozome.net </ext-link> ), which facilitates comparative genomic studies among green plants, provides access to six grass plants. The MIPs in rice and maize, among these six grass plants, have been reported [ <xref rid="pone.0157735.ref012" ref-type="bibr"> 12 </xref> , <xref rid="pone.0157735.ref014" ref-type="bibr"> 14 </xref> , <xref rid="pone.0157735.ref017" ref-type="bibr"> 17 </xref> ]. There has been no study for MIPs in the remaining four grass plants namely switchgrass ( <italic> Panicum virgatum </italic> ), foxtail millet ( <italic> Setaria italica </italic> ), sorghum ( <italic> Sorghum bicolor </italic> ) and <italic> Brachypodium distachyon </italic> . <italic> P </italic> . <italic> virgatum </italic> , which exists at multiple ploidies, is a <span class="trait">drought</span> tolerant plant and has been intensively studied as a source of lignocellulosic biomass to produce renewable energy [ <xref rid="pone.0157735.ref041" ref-type="bibr"> 41 </xref> , <xref rid="pone.0157735.ref042" ref-type="bibr"> 42 </xref> ]. <italic> S </italic> . <italic> italica </italic> is closely related to <italic> P </italic> . <italic> virgatum </italic> . It is a small diploid <span class="gene">C4</span> panicoid crop species and a more tractable experimental model because of its small genome [ <xref rid="pone.0157735.ref043" ref-type="bibr"> 43 </xref> ]. <italic> S </italic> . <italic> bicolor </italic> , related to sugar cane and maize, is grown for food, feed, fibre and biofuels [ <xref rid="pone.0157735.ref044" ref-type="bibr"> 44 </xref> ]. <italic> B </italic> . <italic> distachyon </italic> , related to rice, maize, wheat, barley, sorghum and millet, has several advantages as an experimental model organism for understanding genetic, cellular and molecular biology of temperate grasses [ <xref rid="pone.0157735.ref040" ref-type="bibr"> 40 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Inheritance and QTL mapping of related <span class="trait">root</span> traits in soybean at the seedling stage.</h2>
			<h3>PMID:25145446  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liang H(1), Yu Y, Yang H, Xu L, Dong W, Du H, Cui W, Zhang H.
				<br/> 
				年份：2014 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: This study provides a foundation for further research on <span class="trait">root</span> genetic regulation and molecular breeding with emphasis on correlations among <span class="trait">root</span> traits to ensure robust <span class="trait">root</span> growth and well-developed <span class="trait">root</span> systems. A set of 447 recombinant inbred lines (RILs) derived from a cross between Jingdou23 (cultivar, female parent) and ZDD2315 (semi-wild, male parent) were used to analyze inheritance and detect QTLs related to <span class="trait">root</span> traits at the seedling stage using major gene plus polygene mixed inheritance analysis and composite interval mapping. The results showed that maximum <span class="trait">root</span> <span class="trait">length</span> (MRL) was controlled by three equivalent major genes, lateral <span class="trait">root</span> number (LRN) was controlled by two overlapping major genes, <span class="trait">root</span> weight (RW) and volume (RV) were controlled by four equivalent major genes. Hypocotyl <span class="trait">length</span> (HL) was controlled by four additive main genes, and hypocotyl weight (HW) was controlled by four additive and additive × additive epistatic, major genes; however, polygene effects were not detected in these traits. Shoot weight (SW) was controlled by multi-gene effects, but major gene effects were not detected. Twenty-four QTLs for MRL, LRN, RW, RV, SW, HL, HW were mapped on LG A1 (chromosome 5), LG A2 (chromosome 8), LG B1 (chromosome 11), LG B2 (chromosome 14), LG <span class="gene">C2</span> (chromosome 6), LG D1b (chromosome 2), LG F_1 (chromosome 13), LG G (chromosome 18), LG H_1 (chromosome 12), LG H_2 (chromosome 12), LG I (chromosome 20), LG K_2 (chromosome 9), LG L (chromosome 19), LG M (chromosome 7), LG N (chromosome 3), LG O (chromosome 10), separately. <span class="trait">Root</span> traits were shown to have complex genetic mechanisms at the seedling stage, SW was controlled by multi-gene effects, and the other six traits were controlled by major gene effects. It is concluded that correlations among <span class="trait">root</span> traits must be considered to improve the <span class="trait">development</span> of beneficial <span class="trait">root</span> traits.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>