<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>root activity</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>root activity</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Heterologous Expression of <italic> Panax ginseng PgTIP1 </italic> Confers Enhanced Salt Tolerance of Soybean Cotyledon Hairy Roots, Composite, and Whole Plants</h2>
			<h3>PMID:28769947</h3>
				

				
					<h3>PMCID:5512343</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> An </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/437409/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Zhenmin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Che </surname>  <given-names> Benning </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Haiying </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Bingjun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/353643/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cai </surname>  <given-names> Weiming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Panax ginseng <span class="gene">TIP</span> gene PgTIP1 was previously demonstrated to have high water channel <span class="trait">activity</span> by its heterologous expression in Xenopus laevis oocytes and in yeast; it also plays a significant role in growth of PgTIP1-transgenic Arabidopsis plants under favorable conditions and has enhanced tolerance toward salt and <span class="trait">drought</span> treatment. In this work, we first investigated the physiological effects of heterologous PgTIP1 expression in soybean cotyledon hairy roots or composite plants mediated by Agrobacterium rhizogenes toward enhanced salt tolerance. The PgTIP1-transgenic soybean plants mediated by the pollen tube pathway, represented by the lines N and J11, were analyzed at the physiological and molecular levels for enhanced salt tolerance. The results showed that in terms of root-specific heterologous expression, the PgTIP1-transformed soybean cotyledon hairy roots or composite plants displayed superior salt tolerance compared to the empty vector-transformed ones according to the mitigatory effects of hairy <span class="trait">root</span> growth reduction, drop in leaf RWC, and rise in REL under salt stress. Additionally, declines in K+ content, increases in Na+ content and Na+/K+ ratios in the hairy roots, stems, or leaves were effectively alleviated by PgTIP1-transformation, particularly the stems and leaves of composite soybean plants. At the whole plant level, PgTIP1-trasgenic soybean lines were found to possess stronger <span class="trait">root</span> vigor, reduced <span class="trait">root</span> and leaf cell membrane damage, increased SOD, POD, CAT, and APX activities, steadily increased leaf Tr, RWC, and Pn values, and smaller declines in chlorophyll and carotenoid content when exposed to salt stress compared to wild type. Moreover, the distribution patterns of Na+, K+, and Cl- in the roots, stems, and leaves of salt-stressed transgenic plants were readjusted, in that the absorbed Na+ and Cl- were mainly restricted to the roots to reduce their transport to the shoots, and the transport of root-absorbed K+ to the shoots was simultaneously promoted. PgTIP1 transformation into soybean plants enhanced the expression of some stress-related genes <span class="gene">(GmPOD,</span> <span class="gene">GmAPX1,</span> <span class="gene">GmSOS1,</span> and <span class="gene">GmCLC1)</span> in the roots and leaves under salt treatment. This indicates that the causes of enhanced salt tolerance of heterologous PgTIP1-transformed soybean are associated with the positive regulation on water relations, ion homeostasis, and ROS scavenging under salt stress both at root-specific and whole plant levels.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> Panax ginseng <span class="gene">TIP</span> </italic> gene <italic> PgTIP1 </italic> was previously demonstrated to have high water channel <span class="trait">activity</span> by its heterologous expression in <italic> Xenopus laevis </italic> oocytes and in yeast; it also plays a significant role in growth of <italic> PgTIP1- </italic> transgenic <italic> Arabidopsis </italic> plants under favorable conditions and has enhanced tolerance toward salt and <span class="trait">drought</span> treatment. In this work, we first investigated the physiological effects of heterologous <italic> PgTIP1 </italic> expression in soybean cotyledon hairy roots or composite plants mediated by <italic> Agrobacterium rhizogenes </italic> toward enhanced salt tolerance. The <italic> PgTIP1- </italic> transgenic soybean plants mediated by the pollen tube pathway, represented by the lines N and J11, were analyzed at the physiological and molecular levels for enhanced salt tolerance. The results showed that in terms of root-specific heterologous expression, the <italic> PgTIP1 </italic> -transformed soybean cotyledon hairy roots or composite plants displayed superior salt tolerance compared to the empty vector-transformed ones according to the mitigatory effects of hairy <span class="trait">root</span> growth reduction, drop in leaf RWC, and rise in REL under salt stress. Additionally, declines in K <sup> + </sup> content, increases in Na <sup> + </sup> content and Na <sup> + </sup> /K <sup> + </sup> ratios in the hairy roots, stems, or leaves were effectively alleviated by <italic> PgTIP1- </italic> transformation, particularly the stems and leaves of composite soybean plants. At the whole plant level, <italic> PgTIP1 </italic> -trasgenic soybean lines were found to possess stronger <span class="trait">root</span> vigor, reduced <span class="trait">root</span> and leaf cell membrane damage, increased SOD, POD, CAT, and APX activities, steadily increased leaf Tr, RWC, and Pn values, and smaller declines in chlorophyll and carotenoid content when exposed to salt stress compared to wild type. Moreover, the distribution patterns of Na <sup> + </sup> , K <sup> + </sup> , and Cl <sup> - </sup> in the roots, stems, and leaves of salt-stressed transgenic plants were readjusted, in that the absorbed Na <sup> + </sup> and Cl <sup> - </sup> were mainly restricted to the roots to reduce their transport to the shoots, and the transport of root-absorbed K <sup> + </sup> to the shoots was simultaneously promoted. <italic> PgTIP1 </italic> transformation into soybean plants enhanced the expression of some stress-related genes ( <italic> <span class="gene">GmPOD</span> </italic> , <italic> <span class="gene">GmAPX1</span> </italic> , <italic> <span class="gene">GmSOS1</span> </italic> , and <italic> <span class="gene">GmCLC1</span> </italic> ) in the roots and leaves under salt treatment. This indicates that the causes of enhanced salt tolerance of heterologous <italic> PgTIP1- </italic> transformed soybean are associated with the positive regulation on water relations, ion homeostasis, and ROS scavenging under salt stress both at root-specific and whole plant levels.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">TIP</span> </italic> gene <italic> PgTIP1 </italic> , which was found to be highly and specifically expressed in hormone-autotrophic ginseng cells, was demonstrated to have high water channel <span class="trait">activity</span> by its heterologous expression in <italic> Xenopus laevis </italic> oocytes with regards to the cell volume-changing rate, as well as in yeast by means of a protoplast bursting assay. It was also shown to have a significant role in the growth and development of <italic> PgTIP1- </italic> transgenic <italic> Arabidopsis </italic> plants under favorable conditions, including enhanced seed size and mass, and higher fatty acid content than the wild-type control ( <xref rid="B18" ref-type="bibr"> Lin et al., 2007 </xref> ; <xref rid="B16" ref-type="bibr"> Li and Cai, 2015 </xref> ). Another related study by <xref rid="B24" ref-type="bibr"> Peng et al. (2007) </xref> reported that the overexpression of <italic> PgTIP1 </italic> in transgenic <italic> Arabidopsis </italic> plants led to enhanced tolerance against salt and <span class="trait">drought</span> stress, but lowered cold acclimation ability. Further work by <xref rid="B16" ref-type="bibr"> Li and Cai (2015) </xref> suggested that the conferred faster growth and enhanced salt tolerance in transgenic <italic> Arabidopsis </italic> plants was related to its water channel <span class="trait">activity</span> of the Ser <sup> 128 </sup> residue of <italic> PgTIP1 </italic> . Functional analyses using the plant <italic> PgTIP1- </italic> transgenic technique have predominantly focused on <italic> Xenopus laevis </italic> oocytes, yeast, or non-crop <italic> Arabidopsis </italic> , whereas related studies on crops have rarely been reported. Of these crops, soybean ( <italic> Glycine max </italic> L.) is the world&#x02019;s leading economic oilseed crop and is also a major source of human food and animal feed ( <xref rid="B19" ref-type="bibr"> Manavalan et al., 2009 </xref> ). Although soybean is classified as a moderately NaCl-tolerant plant, the conventional breeding of soybean cultivars for improved salt tolerance is challenging due to its narrow basis of genetic germplasm and relatively limited salt tolerance ( <xref rid="B17" ref-type="bibr"> Li et al., 2006 </xref> ; <xref rid="B45" ref-type="bibr"> Zhang et al., 2011 </xref> ). To date, about 70 AQP members have been discovered across the entire soybean genome, of which 23 are TIPs ( <xref rid="B44" ref-type="bibr"> Zhang et al., 2013 </xref> ; <xref rid="B7" ref-type="bibr"> Deshmukh et al., 2016 </xref> ; <xref rid="B32" ref-type="bibr"> Song et al., 2016 </xref> ). <xref rid="B39" ref-type="bibr"> Wang et al. (2011) </xref> reported <italic> Arabidopsis </italic> plants overexpressing <italic> GsTIP2;1 </italic> from <italic> G. soja </italic> displayed reduced tolerance to salt and <span class="trait">drought</span> stress when dehydration speed or water loss in the plant was enhanced. <xref rid="B32" ref-type="bibr"> Song et al. (2016) </xref> studied the solute transport function of two soybean <italic> TIPs </italic> ( <italic> <span class="gene">GmTIP1;5</span> </italic> and <italic> <span class="gene">GmTIP2;5</span> </italic> ) by expression in <italic> Xenopus laevis </italic> oocytes, and found that <span class="gene">GmTIP1;5</span> facilitated the rapid movement of water across the oocyte membrane, while <span class="gene">GmTIP2;5</span> facilitated the movement of water and boric acid. Thus, different types of <italic> <span class="gene">TIP</span> </italic> -encoded proteins in plants display diverse selectivity for substrate and various stress responses to salt and <span class="trait">drought.</span> Consequently, research on the effects and physiological and molecular mechanisms of <italic> PgTIP1 </italic> expression in widely cultivated crops such as soybean to improve tolerance of saline conditions, as well as the exploration of <italic> PgTIP1 </italic> heterologous expression for obtaining new transgenic plant germplasms and salt-tolerant crop varieties, seems very necessary. The theoretical potential and practical application should be explored.
					</p>
				
					<p>&nbsp;&nbsp;
						Tonoplast intrinsic proteins, as a major and plant-specific subfamily of AQPs, can regulate water exchange between the cytoplasm and vacuole and control cell water homeostasis involved in cellular storage, osmotic adjustment and turgor control, cell signaling and digestion, and abiotic stress tolerance ( <xref rid="B24" ref-type="bibr"> Peng et al., 2007 </xref> ; <xref rid="B39" ref-type="bibr"> Wang et al., 2011 </xref> ; <xref rid="B43" ref-type="bibr"> Xin et al., 2014 </xref> ; <xref rid="B32" ref-type="bibr"> Song et al., 2016 </xref> ). Therefore, plants or crops may be conferred with increased tolerance toward different abiotic stresses through a suitable <italic> <span class="gene">TIP</span> </italic> -transgenic approach ( <xref rid="B33" ref-type="bibr"> Srivastava et al., 2014 </xref> ). For example, ectopic overexpression of <italic> TsTIP1;2 </italic> from the halophyte <italic> Thellungiella salsuginea </italic> in <italic> Arabidopsis </italic> notably increased tolerance for <span class="trait">drought,</span> salt, and oxidative stresses ( <xref rid="B37" ref-type="bibr"> Wang et al., 2014 </xref> ). Transgenic <italic> Arabidopsis </italic> plants expressing <italic> SlTIP2;2 </italic> from <italic> Solanum lycopersicum </italic> exhibited enhanced salt stress tolerance by regulating ion (including Na <sup> + </sup> and K <sup> + </sup> ) homeostasis and antioxidant enzyme activities under salt treatment ( <xref rid="B43" ref-type="bibr"> Xin et al., 2014 </xref> ). A high water channel <span class="trait">activity</span> was displayed by <italic> PgTIP1- </italic> heterologous expression in <italic> Xenopus laevis </italic> oocytes and in yeast, and a significant role in growth and development under favorable conditions and enhanced salt and <span class="trait">drought</span> tolerance, together with upregulated expression of the stress-related genes, was demonstrated in <italic> PgTIP1- </italic> transgenic <italic> Arabidopsis </italic> plants ( <xref rid="B18" ref-type="bibr"> Lin et al., 2007 </xref> ; <xref rid="B24" ref-type="bibr"> Peng et al., 2007 </xref> ; <xref rid="B16" ref-type="bibr"> Li and Cai, 2015 </xref> ). In our work, we successfully obtained <italic> PgTIP1 </italic> -transformed soybean cotyledon hairy roots and composite plants mediated by <italic> A. rhizogenes </italic> , and <italic> PgTIP1 </italic> -transgenic plants (lines N and J11) through the pollen tube pathway without a marker gene ( <bold> Figures <xref ref-type="fig" rid="F1"> 1A </xref> </bold> , <bold> <xref ref-type="fig" rid="F2"> 2A </xref> </bold> , <bold> <xref ref-type="fig" rid="F4"> 4B </xref> </bold> ). However, as a technical problem occurred as a result of the large and complex soybean genome and sharing of exogenous <italic> PgTIP1 </italic> with high homology of soybean <italic> <span class="gene">GmTIPs</span> </italic> family itself, we thus adopted nested PCR (with 0.005% sensitivity) instead of standard PCR (with 0.5% sensitivity; <xref rid="B2" ref-type="bibr"> Ao et al., 2011 </xref> ) for positive identification of <italic> PgTIP1- </italic> transformed soybean materials, and showed fine effectiveness ( <bold> Figures <xref ref-type="fig" rid="F1"> 1A-c </xref> </bold> , <bold> <xref ref-type="fig" rid="F2"> 2B </xref> </bold> , <bold> <xref ref-type="fig" rid="F4"> 4A </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Under salt stress, and compared to empty vector-transformed cotyledon hairy roots or composite plants, the <italic> PgTIP1 </italic> -transformed cotyledon hairy roots or composite plants suffered far lighter salt injury symptoms, and this could be well reflected from the biomass reduction (fresh and dry weight, or maximal length of hairy roots) ( <bold> Figures <xref ref-type="fig" rid="F1"> 1B,C </xref> </bold> , <bold> <xref ref-type="fig" rid="F2"> 2C </xref> </bold> ) or withering degree ( <bold> Figure <xref ref-type="fig" rid="F2"> 2A-c </xref> </bold> ), leaf RWC drop and REL rise ( <bold> Figure <xref ref-type="fig" rid="F2"> 2D </xref> </bold> ) of composite plants. Of course, more important explanations for the differences in salt injury between <italic> PgTIP1 </italic> -transformed and empty vector-transformed cotyledon hairy roots or composite plants exist in that Na <sup> + </sup> contents in <italic> PgTIP1 </italic> -transformed cotyledon hairy roots or roots, stems, and leaves of <italic> PgTIP1 </italic> -transformed composite plants lowered significantly under salt stress in comparison with the empty vector-transformed, while K <sup> + </sup> levels were relatively better maintained at the same time, eventually leading to notably lower Na <sup> + </sup> /K <sup> + </sup> ratios than the empty vector-transformed ( <bold> Figures <xref ref-type="fig" rid="F1"> 1D,E </xref> </bold> , <bold> <xref ref-type="fig" rid="F3"> 3A&#x02013;C </xref> </bold> ). This is the first study on the aspect of root-specificity and heterologous overexpression of <italic> PgTIP1 </italic> that demonstrated that water relations and ion homeostasis of <italic> PgTIP1 </italic> -transformed soybean cotyledon hairy roots or composite plants are well adjusted under salt stress, and thus confer enhanced salt tolerance. On this basis, our further investigation using <italic> PgTIP1 </italic> -transgenic soybean lines (N and J11) showed that at the whole plant level, apart from a similar ability to adjust water relations ( <bold> Figure <xref ref-type="fig" rid="F4"> 4C </xref> </bold> ) as well as less salt damage (expressed as a REL) ( <bold> Figure <xref ref-type="fig" rid="F4"> 4E </xref> </bold> ) as the above-mentioned soybean hairy roots test, plant <span class="trait">root</span> vigor ( <bold> Figure <xref ref-type="fig" rid="F4"> 4D </xref> </bold> ) and a series of photosynthesis and fluorescence parameters, including leaf chlorophyll and carotenoid contents, chla/b ratio, and Pn, Tr, and Fv/Fm values ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ), were well maintained under salt stress in comparison to WT. With regards to Na <sup> + </sup> and K <sup> + </sup> contents and Na <sup> + </sup> /K <sup> + </sup> ratio, together with the Cl <sup> - </sup> contents in the roots, stems, and leaves of salt-stressed <italic> PgTIP1 </italic> -transgenic soybean plants exposed to salt stress, similar regulating effects were also observed as the above-mentioned <italic> PgTIP1 </italic> -transformed soybean hairy roots test, particularly line J11 which performed best ( <bold> Figures <xref ref-type="fig" rid="F6"> 6A </xref> &#x02013; <xref ref-type="fig" rid="F6"> D </xref> </bold> ). <italic> Arabidopsis </italic> plants overexpressing <italic> PgTIP1 </italic> displayed faster growth and enhanced salt stress tolerance through the enhancement of tonoplast permeability and increased water movement from the cytosol to vacuole, and greater accumulation of Na <sup> + </sup> due to the larger vacuole ( <xref rid="B24" ref-type="bibr"> Peng et al., 2007 </xref> ; <xref rid="B16" ref-type="bibr"> Li and Cai, 2015 </xref> ). AQPs may participate in ion homeostasis at the whole plant level by regulating the ratio of apoplastic/symplastic water flow, thus directing solute flux through different plant tissues, and furthermore, it has been reported that AQPs together with K <sup> + </sup> channels can function as plant osmo-regulators to maintain cytosolic osmolarity and increase tolerance to <span class="trait">drought</span> or other stresses ( <xref rid="B38" ref-type="bibr"> Wang et al., 2016 </xref> ). This might explain the retention of Na <sup> + </sup> and Cl <sup> - </sup> in the roots and maintenance of water content in the leaves, which conferred favorable regulation of water relation and ion homeostasis of <italic> PgTIP1 </italic> -transgenic soybean plants under salt stress. In addition, oxidative damage resulting from excessive ROS is the main secondary factor, apart from osmotic and ionic stresses, influencing the plant response under saline conditions ( <xref rid="B22" ref-type="bibr"> Munns and Tester, 2008 </xref> ; <xref rid="B46" ref-type="bibr"> Zhao et al., 2017 </xref> ). <xref rid="B16" ref-type="bibr"> Li and Cai (2015) </xref> further showed that the leaves of <italic> PgTIP1 </italic> -transgenic <italic> Arabidopsis </italic> plants accumulated less H <sub> 2 </sub> O <sub> 2 </sub> than WT under salt stress, which suggests that <italic> PgTIP1 </italic> transformation is also beneficial for relieving oxidative stress. In our study, when compared with WT, H <sub> 2 </sub> O <sub> 2 </sub> contents in the roots and leaves of the salt-stressed <italic> PgTIP1 </italic> -transgenic soybean lines N and J11 did not show the increase that WT did, but rather exhibited an obvious decrease. Additionally, <inline-formula> <mml:math id="M9"> <mml:mrow> <mml:msubsup> <mml:mrow> <mml:mi mathvariant="normal"> O </mml:mi> </mml:mrow> <mml:mrow> <mml:mn mathvariant="normal"> 2 </mml:mn> </mml:mrow> <mml:mrow> <mml:mo mathvariant="normal"> &#x02022; </mml:mo> <mml:mo mathvariant="normal"> &#x02212; </mml:mo> </mml:mrow> </mml:msubsup> </mml:mrow> </mml:math> </inline-formula> production rates rose slightly in the roots but dropped in the leaves ( <bold> Figures <xref ref-type="fig" rid="F7"> 7A,B </xref> </bold> ), which was consistent with enhanced activities of SOD, POD, CAT, and APX in the roots or leaves under salt stress ( <bold> Figures <xref ref-type="fig" rid="F7"> 7C,D </xref> </bold> ). <xref rid="B16" ref-type="bibr"> Li and Cai (2015) </xref> suggested that the heterologous expression of <italic> PgTIP1 </italic> led to the upregulated expression of stress-related genes, such as <italic> CBF3, MYB15, COR47, <span class="gene">ICE1</span> </italic> , etc., which could contribute to the salt tolerance of the transgenic <italic> Arabidopsis </italic> plant. In this work, we also found that the expression levels of <italic> <span class="gene">GmPOD</span> </italic> , <italic> <span class="gene">GmAPX1</span> </italic> , <italic> <span class="gene">GmSOS1</span> </italic> , and <italic> <span class="gene">GmCLC1</span> </italic> in the roots and leaves of <italic> PgTIP </italic> -transgenic lines were also reinforced in comparison to WT, but there were no differences in the expression of endogenous soybean <italic> <span class="gene">GmTIP1;1</span> </italic> and <italic> <span class="gene">GmTIP1;3</span> </italic> , which shared the highest homology with <italic> PgTIP1 </italic> ( <bold> Figure <xref ref-type="fig" rid="F8"> 8 </xref> </bold> ). This indicates that heterologous <italic> PgTIP1 </italic> transformation into soybean has an obvious positive contribution toward salt tolerance at the transcriptional level of stress-related genes coding ion transporters and antioxidant enzymes. <xref rid="B43" ref-type="bibr"> Xin et al. (2014) </xref> reported that, tomato <italic> SlTIP2;2 </italic> -transformed <italic> Arabidopsis </italic> plants exhibited enhanced salt stress tolerance, which has close links with higher K <sup> + </sup> /Na <sup> + </sup> ratio (for ion homeostasis) and increased SOD, CAT, and POD activities (for ROS scavenging) than the wild-type control under salt stress. <xref rid="B37" ref-type="bibr"> Wang et al. (2014) </xref> also demonstrated that, <italic> TsTIP1;2- </italic> overexpressed <italic> Arabidopsis </italic> plants displayed strong tolerance against salt or oxidative stress through the capability of direct conducting H <sub> 2 </sub> O, and the indirect facilitation of Na <sup> + </sup> influx or H <sub> 2 </sub> O <sub> 2 </sub> transport into the vacuoles. We also further need to seek more evidence at the subcellular level of vacuoles in future.
					</p>
				
					<p>&nbsp;&nbsp;
						Firstly, the heterologous overexpression of <italic> PgTIP1 </italic> resulted in the transformed soybean cotyledon hairy roots or composite plants mediated by A. <italic> rhizogenes </italic> displaying superior salt tolerance to the empty vector-transformant according to the ameliorative roles in hairy <span class="trait">root</span> growth reduction, leaf RWC drop, and REL rise of composite plants under salt stress. Additionally, a decline in K <sup> + </sup> contents, increase in Na <sup> + </sup> contents and Na <sup> + </sup> /K <sup> + </sup> ratios in the hairy roots, stems, or leaves were effectively mitigated by <italic> PgTIP1 </italic> transformation, especially with respect to the stems and leaves of composite soybean plants. At whole plant level, pollen-tube transformation pathway <italic> PgTIP1 </italic> -transgenic soybean lines possessed stronger <span class="trait">root</span> vigor, lighter cell membrane damage to the roots and leaves, improved SOD, POD, CAT, and APX activities along with the enhanced expression of <italic> POD </italic> and <italic> APX </italic> genes under salt treatment, steadily increased values of leaf RWC, Tr, and Pn, and smaller declines in chlorophyll and carotenoid contents when exposed to salt stress compared to WT. With regards to transport or distribution of salty ions in salt-stressed <italic> PgTIP1 </italic> -trasgenic soybean lines, the heterologous transformation of <italic> PgTIP1 </italic> into soybean plants could readjust the distribution pattern of Na <sup> + </sup> , K <sup> + </sup> , and Cl <sup> - </sup> in the roots and shoots (stems and leaves), i.e., the absorbed Na <sup> + </sup> and Cl <sup> - </sup> mainly accumulated in the roots so as to reduce their transport to the shoots, and the root-absorbed K <sup> + </sup> was simultaneously promoted to be transported toward the shoots. This may be related to <italic> PgTIP1 </italic> transformation- and salt stress-induced enhancement of the expression of <italic> <span class="gene">GmSOS1</span> </italic> and <italic> <span class="gene">GmCLC1</span> </italic> genes, which control Na <sup> + </sup> and Cl <sup> - </sup> into and out of the cell, and the transport and intracellular compartmentation of soybean plants. Finally, these results reveal that the cause of enhanced salt tolerance of heterologous <italic> PgTIP1- </italic> transformed soybeans lies in its positive regulation of water relations, ion homeostasis, and ROS scavenging under salt stress both at the root-specific and whole plant levels. Therefore, <italic> PgTIP1 </italic> can be considered as a good candidate gene for the production of new salt tolerant plant germplasms and crop cultivars in the future.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Sequence Characterization and Expression Analysis of Major Intrinsic Proteins in Soybean ( <italic> Glycine max </italic> L.)</h2>
			<h3>PMID:23437113</h3>
				
					<h3>PMCID:3577755</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Da Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ali </surname>  <given-names> Zulfiqar </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Chang Biao </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yi </surname>  <given-names> Jin Xin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao Long </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiao Qing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiao Lan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yi Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Iqrar Ahmad </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Trethowan </surname>  <given-names> Richard M. </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hong Xiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 8 </day>  <month> 1 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Water is essential for all living organisms. Aquaporin proteins are the major facilitator of water transport <span class="trait">activity</span> through cell membranes of plants including soybean. These proteins are diverse in plants and belong to a large major intrinsic (MIP) protein family. In higher plants, MIPs are classified into five subfamilies including plasma membrane intrinsic proteins (PIP), tonoplast intrinsic proteins (TIP), NOD26-like intrinsic proteins (NIP), small basic intrinsic proteins (SIP), and the recently discovered X intrinsic proteins (XIP). This paper reports genome wide assembly of soybean MIPs, their functional prediction and expression analysis. Using a bioinformatic homology search, 66 <span class="gene">GmMIPs</span> were identified in the soybean genome. Phylogenetic analysis of amino acid sequences of <span class="gene">GmMIPs</span> divided the large and highly similar multi-gene family into 5 subfamilies: <span class="gene">GmPIPs,</span> <span class="gene">GmTIPs,</span> <span class="gene">GmNIPs,</span> <span class="gene">GmSIPs</span> and <span class="gene">GmXIPs.</span> <span class="gene">GmPIPs</span> consisted of 22 genes and <span class="gene">GmTIPs</span> 23, which showed high sequence similarity within subfamilies. <span class="gene">GmNIPs</span> contained 13 and <span class="gene">GmSIPs</span> 6 members which were diverse. In addition, we also identified a two member <span class="gene">GmXIP,</span> a distinct 5 <sup> th </sup> subfamily. <span class="gene">GmMIPs</span> were further classified into twelve subgroups based on substrate selectivity filter analysis. Expression analyses were performed for a selected set of <span class="gene">GmMIPs</span> using semi-quantitative reverse transcription (semi-RT-qPCR) and qPCR. Our results suggested that many <span class="gene">GmMIPs</span> have high sequence similarity but diverse roles as evidenced by analysis of sequences and their expression. It can be speculated that <span class="gene">GmMIPs</span> contains true aquaporins, glyceroporins, aquaglyceroporins and mixed transport facilitators.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> in silico </italic> expression of 23 <span class="gene">GmMIPs</span> in roots was &#x0003e;50 ( <xref ref-type="table" rid="pone-0056312-t001"> Table 1 </xref> ). Based on <italic> in silico </italic> <span class="trait">root</span> specific expression and phylogenetic relationship ( <xref ref-type="fig" rid="pone-0056312-g001"> Fig. 1 </xref> ), 24 <italic> <span class="gene">GmMIPs</span> </italic> were selected for tissue specific expression analysis by semi-qPCR ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ). The expression patterns of <span class="gene">GmMIPs</span> in various organs of soybean were identified. <span class="gene">GmPIP1;3</span> strongly expressed in roots, stems, leaves and pods. Other strongly expressed genes included <span class="gene">GmPIP1;4</span> in stems and pods, <span class="gene">GmPIP2;3</span> in roots, flowers and pods, <span class="gene">GmTIP1;4</span> in flowers, <span class="gene">GmTIP1;7</span> in stems, leaves, flowers and pods, <span class="gene">GmTIP1;9</span> in roots, <span class="gene">GmTIP2;1</span> in stems and pods, <span class="gene">GmTIP2;2</span> in roots and pods, <span class="gene">GmTIP2;6</span> in stems, leaves, flowers and pods and <span class="gene">GmXIP1;2</span> in leaves. <span class="gene">GmPIP2;6</span> and <span class="gene">GmTIP4;1expressed</span> marginally while <span class="gene">GmPIP2;11</span> was marginally lower in all plant parts. The rest of the <span class="gene">GmMIPs</span> gave weak, marginal, marginal low or null expression.
					</p>
				
					<p>&nbsp;&nbsp;
						These <italic> <span class="gene">GmMIPs</span> </italic> were selected for tissue specific expression analysis by semi-qPCR on the basis of <italic> in silico </italic> <span class="trait">root</span> specific expression (&#x0003e;50, see <xref ref-type="table" rid="pone-0056312-t001"> Table 1 </xref> ) and phylogenetic relationship (see <xref ref-type="fig" rid="pone-0056312-g001"> Fig. 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The expression patterns of selective <italic> <span class="gene">GmMIPs</span> </italic> , expressing in <span class="trait">root</span> ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) and distributed on chromosomes 11, 12 and 13 ( <xref ref-type="table" rid="pone-0056312-t001"> Table 1 </xref> ), were further analyzed at 0, 7, 14 and 21 d of water stress using semi-qPCR ( <xref ref-type="fig" rid="pone-0056312-g006"> Fig. 6 </xref> ). Tubulin was used as the internal control. As <span class="gene">GmTIP1;7</span> is an isoform of AtTIP1;1 or AtTIP1;2 which are known as salt induced tonoplast intrinsic protein (SITIP), thus it was specifically included for semi-qPCR analysis under <span class="trait">drought.</span> Most of the <italic> <span class="gene">GmMIPs</span> </italic> expressed stably in roots with slight deviations at various times of water stress. The expression of <span class="gene">GmPIP1;8</span> peaked significantly at 7 d of stress whereas <span class="gene">GmTIP1;7</span> only expressed at 7 d without watering and <span class="gene">GmPIP2;4</span> only after 21 d of water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						The six MIPs were selected from 25 <span class="gene">GmMIPs</span> expressing in <span class="trait">root</span> (see <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) and distributed on chromosomes 11, 12 and 13 (see <xref ref-type="table" rid="pone-0056312-t001"> Table 1 </xref> ) as each of these chromosomes contain highest number of MIPs. The expression pattern of control plants (water was applied after three-leaf stage till 21 d) at various time points was similar to those at 0 d of no watering and thus picture not presented.
					</p>
				
					<p>&nbsp;&nbsp;
						All <span class="gene">GmNIPs</span> were classified into four subgroups. <span class="gene">GmNIP2</span> and <span class="gene">GmNIP6</span> subfamily members grouped in subgroup III and IV respectively appeared novel based on ar/R filter. NIPs have previously emerged as an interesting AQP subclass in terms of transport specificity and they are subdivided in two subgroups based on the predicted structure of their selectivity filter <xref ref-type="bibr" rid="pone.0056312-Wallace1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0056312-Wallace2"> [38] </xref> . It has been reported that phosphorylation of Nodulin-26 on Ser 262 enhanced water permeability and that phosphorylation is stimulated by <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0056312-Guenther2"> [39] </xref> . AtNIPs have been reported to be classified in two subgroups <xref ref-type="bibr" rid="pone.0056312-Wallace1"> [33] </xref> . Mixed transport activities of NIPs have been observed in different organisms, for example, <span class="gene">GmNOD26</span> can form a functional water channel and produce glycerol permease in <italic> X. laevis </italic> oocytes <xref ref-type="bibr" rid="pone.0056312-Rivers1"> [40] </xref> . The AtNIP1;1 was predicted as an AQP when expressed in <italic> X. laevis </italic> oocytes <xref ref-type="bibr" rid="pone.0056312-Weig1"> [41] </xref> . The amino acids for ar/R filter in subgroup III consisted of Gly, Ser, Gly, and Arg (GSGR), compared with Ala, Ile, Gly, and Arg (AIGR) in subgroup IIA ( <xref ref-type="fig" rid="pone-0056312-g004"> Fig. 4 </xref> ), which are comparable with OsLsi1 and AtNIP5;1 respectively. The residue at the H5 position of the ar/R filter of both OsLsi1 and AtNIP5;1 were revealed to play a key role in membrane permeability to silicic acid (Si) and boric acid (B), although there is a relatively low selectivity for arsenite (As) <xref ref-type="bibr" rid="pone.0056312-MitaniUeno1"> [34] </xref> . Previous reports show that G/A substitution at H2 (GSGR to ASGR) does not affect the transport <span class="trait">activity</span> of OsLsi1 for Si, As, B and water. The S/I substitution (GSGR to GIGR or AIGR) at H5 resulted in loss of Si, B and water transport <span class="trait">activity.</span> However, as transport was by 60% instead of total loss. AtNIP5;1, with AIGR residues for ar/R filter, was able to transport boric acid, water, and arsenite, but not silicic acid <xref ref-type="bibr" rid="pone.0056312-MitaniUeno1"> [34] </xref> . A single or double mutation at H2 and/or H5 did not result in any Si transport <span class="trait">activity..</span> In contrast, both single and double mutations at the H2 and/or H5 positions showed As transport <span class="trait">activity</span> and H5 I/S substitution led to decreased B transport <span class="trait">activity</span> while water permeability remained unaffected. The <span class="gene">GmNIPs</span> in subgroup III can be speculated to transport Si, As, B and water while those in subgroup IIA may transport As, B and water.
					</p>
				
					<p>&nbsp;&nbsp;
						Specific tools such as cDNA hybridization, fine mapping based on <italic> in situ </italic> RT-PCR, qPCR etc. have recently been used to monitor expression of the whole aquaporin gene family. Hybridization of cDNAs to arrays carrying aquaporin gene specific tags have revealed a coordinated down-regulation of aquaporin genes in response to water and nutrient stresses <xref ref-type="bibr" rid="pone.0056312-Boursiac1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0056312-Alexandersson1"> [19] </xref> . Quantitative PCR analyses have been used to monitor expression of aquaporin transcripts in various tissues, organs or stress conditions <xref ref-type="bibr" rid="pone.0056312-Boursiac1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0056312-Sakurai1"> [14] </xref> , <xref ref-type="bibr" rid="pone.0056312-Alexandersson1"> [19] </xref> . In the present study, we used semi-qPCR and qPCR analyses to establish the relationship of specific MIP abundance and <span class="trait">drought</span> tolerance in soybean roots. Among 24 soybean aquaporin genes selected on the basis of <italic> in silico </italic> expression in roots and phylogenetic relationship, our results indicated that at least ten genes expressed in roots and the expression of four genes encoding <span class="gene">PIP</span> and three genes encoding <span class="gene">TIP</span> was significantly greater than the internal control ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ). Most of these TIPs and PIPs are located on chromosomes 11 and 12. In Arabidopsis, a similar pattern has been observed in roots, leaves and flowers, where three PIPs, five TIPs, seven NIPs and one SIPs were expressed in roots <xref ref-type="bibr" rid="pone.0056312-Alexandersson1"> [19] </xref> . Orthologous MIPs in rice also expressed in roots, however the expression of OsTIP, OsNIP1;1, OsNIP1;4, OsNIP2;2, OsNIP3;3, OsNIP4;1 and OsSIP2;1 was weaker than the control <xref ref-type="bibr" rid="pone.0056312-Sakurai1"> [14] </xref> . This study showed organ specificity of <span class="gene">GmMIPs</span> as reported earlier in Arabidopsis <xref ref-type="bibr" rid="pone.0056312-Alexandersson1"> [19] </xref> and rice <xref ref-type="bibr" rid="pone.0056312-Sakurai1"> [14] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The genes expressing in roots and those which are located on chromosomes 11 and 12 were selected for subsequent testing water stress response. <span class="gene">GmPIP2;4</span> didn't express in any organ ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) until 21 d after cessation of watering ( <xref ref-type="fig" rid="pone-0056312-g006"> Fig. 6 </xref> ). However, it expressed in roots 12 h after growth in 20% PEG ( <xref ref-type="fig" rid="pone-0056312-g007"> Fig. 7 </xref> ). <span class="gene">GmTIP1;7</span> also didn't express in roots ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) until 7 d without watering ( <xref ref-type="fig" rid="pone-0056312-g006"> Fig. 6 </xref> ). Such expression patterns were also reported for OsTIP4;2 in rice, where its expression is very low in roots growing in normal conditions and significantly increased in roots after 10 h growth in 15% PEG and 8 h growth in 150 mM NaCl stress <xref ref-type="bibr" rid="pone.0056312-Li1"> [43] </xref> . It can be speculated that <span class="gene">GmPIP2;4</span> and <span class="gene">GmTIP1;7</span> are responsive to water stress as they only expressed after a period of water stress and are thus putatively useful. The qPCR analysis of nine <span class="gene">GmPIPs</span> and five TIPs further established the relationship of these <span class="gene">GmMIPs</span> with <span class="trait">drought</span> tolerance ( <xref ref-type="fig" rid="pone-0056312-g007"> Fig. 7 </xref> ). Three patterns were evident, down-regulation followed by up-regulation of <span class="gene">GmMIPs</span> and vice versa at different times after initiation of <span class="trait">drought</span> stress. The third type of expression pattern was revealed by qPCR analysis of <span class="gene">GmPIP2;5</span> and <span class="gene">GmTIP2;6.</span> The former did not express in any plant organ ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) but expressed after 12 h of <span class="trait">drought</span> stress in <span class="trait">root</span> ( <xref ref-type="fig" rid="pone-0056312-g007"> Fig. 7 </xref> ), while the later didn't express in roots ( <xref ref-type="fig" rid="pone-0056312-g005"> Fig. 5 </xref> ) but expressed 2 and 4 h after application of <span class="trait">drought</span> stress ( <xref ref-type="fig" rid="pone-0056312-g007"> Fig. 7 </xref> ). Similar expression patterns of various genes have been studied in Arabidopsis for urea transport <xref ref-type="bibr" rid="pone.0056312-Liu1"> [44] </xref> and for <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0056312-Alexandersson1"> [19] </xref> , and in rice for chilling and light reception <xref ref-type="bibr" rid="pone.0056312-Sakurai1"> [14] </xref> and for <span class="trait">drought</span> and salinity stress <xref ref-type="bibr" rid="pone.0056312-Li1"> [43] </xref> . The expression pattern of <span class="gene">GmMIPs</span> after application of <span class="trait">drought</span> treatment reflected a coordinated regulation of MIP isoforms that collectively contribute to the whole <span class="trait">root</span> water transport capacity. These <span class="gene">GmMIPs</span> are a potential resource for the genetic improvement of soybean <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						This study identified and characterized 66 soybean aquaporin genes. Phylogenetic analysis of amino acid sequences divided the large and highly similar multi-gene family into 5 subfamilies. These genes were further classified into twelve subgroups: <span class="gene">GmPIPs</span> located in one subgroup, <span class="gene">GmTIPs</span> in three, <span class="gene">GmNIPs</span> in four, <span class="gene">GmSIPs</span> in three and <span class="gene">GmXIPs</span> in one. It can be speculated that <span class="gene">GmMIPs</span> contains true aquaporins, glyceroporins, aquaglyceroporins and mixed transport facilitators. However, their functionality remains to be properly validated. Our results indicate that the genes identified in this study represent an important genetic resource for the improvement of water use efficiency and/or <span class="trait">drought</span> tolerance as well as for transport of non-aqua substrate in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Small RNA Analysis of Soybean Reveals Auxin-Responsive microRNAs that are Differentially Expressed in Response to Salt Stress in <span class="trait">Root</span> Apex</h2>
			<h3>PMID:26834773</h3>
				
					<h3>PMCID:4716665</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Zhengxi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/306882/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Youning </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mou </surname>  <given-names> Fupeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289023/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Yinping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Liang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Senlei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Qiong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xia </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/110076/overview"/>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a major economic crop worldwide; however, the <span class="trait">yield</span> is greatly affected by soil salinity (Yasuta and Kokubun, <xref rid="B64" ref-type="bibr"> 2014 </xref> ). Previous studies have shown that soybean copes with salt stress using physiological mechanisms that are similar to those in <italic> Arabidopsis </italic> , including the maintenance of ion homeostasis, adjustment of cellular responses to osmotic stress, and metabolic responses (Im et al., <xref rid="B31" ref-type="bibr"> 2012 </xref> ; Guan et al., <xref rid="B26" ref-type="bibr"> 2014 </xref> ; Rao et al., <xref rid="B53" ref-type="bibr"> 2014 </xref> ). Soybean is a legume that possesses the ability to form <span class="trait">root</span> nodules that can fix N <sub> 2 </sub> from the atmosphere through a symbiotic interaction with rhizobia (Ferguson et al., <xref rid="B20" ref-type="bibr"> 2010 </xref> ). Therefore, studies of <span class="trait">root</span> growth and development in response to salt stress are of the utmost significance in soybean genetics. Despite great effort to identify the relevant genes and/or proteins using various approaches (Aghaei et al., <xref rid="B1" ref-type="bibr"> 2009 </xref> ; Ge et al., <xref rid="B24" ref-type="bibr"> 2010 </xref> ; Sobhanian et al., <xref rid="B55" ref-type="bibr"> 2010 </xref> ), only a few genes, including that encoding respiratory alternative oxidase and <italic> <span class="gene">GmSALT3</span> </italic> , have been shown to mediate <span class="trait">root</span> responses to salt and salt tolerance in soybean (Guan et al., <xref rid="B26" ref-type="bibr"> 2014 </xref> ). Knowledge of the gene regulatory network underlying the response of the RAM in soybean to salt, especially the environmental plasticity of the miRNAome, is lacking.
					</p>
				
					<p>&nbsp;&nbsp;
						To prepare longitudinal sections of the roots, <span class="trait">root</span> tips were collected from the medium with or without 75 mM NaCl and then fixed in formalin/acetic acid/alcohol. The <span class="trait">root</span> segments were dehydrated at room temperature in a gradient series of ethanol solutions, followed by two changes of Clear-Rite 3 (Sigma-Aldrich Corp., St. Louis, MO) for 1 h each, and finally embedded in Paraplast (Leica Biosystems Nussloch <span class="gene">Gmbh,</span> Nu&#x000df;loch, Germany) at 62&#x000b0;C. Sectioning was performed using a microtome. Sections were cut every 20 &#x003bc;m and then stained with 0.1% (w/v) Eosin Y. The sections were observed under an Olympus CX31 biological microscope (Tokyo, Japan).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The expression of salt stress responsive miRNAs was affected by exogenous auxin </bold> . Five-day-old seedlings were transferred to liquid B5 medium supplemented with or without 1 &#x003bc;M 2,4-D. After 3 days, <span class="trait">root</span> tips were collected and used for RT-qPCR analysis. <span class="gene">Gma-miR1515a</span> was used as an internal control. The results shown are the averages &#x000b1; SD of three biological replicates. Error bars indicate the standard deviation. Student's <italic> t </italic> -tests were performed; statistically significant results are marked with <sup> * </sup> ( <italic> P </italic> &#x0003c; 0.05); <sup> ** </sup> ( <italic> P </italic> &#x0003c; 0.01); <sup> *** </sup> ( <italic> P </italic> &#x0003c; 0.001).
					</p>
				
					<p>&nbsp;&nbsp;
						Notably, we found that several known soybean miRNAs may target a greater number of genes than their homologs in <italic> Arabidopsis </italic> . In <italic> Arabidopsis </italic> , miR399 directly targets the protein PHO2 (UBC), which is an ubiquitin E2-conjugating enzyme (Bari et al., <xref rid="B4" ref-type="bibr"> 2006 </xref> ). However, in addition to <italic> <span class="gene">Glyma.13G239100</span> </italic> ( <italic> PHO2 </italic> ) and <italic> <span class="gene">Glyma.10G036800</span> </italic> ( <italic> <span class="gene">GmPT5</span> </italic> ), which have been validated as the targets of gma-miR399 (Xu et al., <xref rid="B63" ref-type="bibr"> 2013 </xref> ), <italic> <span class="gene">Glyma.14G188000</span> </italic> and <italic> <span class="gene">Glyma.15G074200</span> </italic> annotated as Inorganic phosphate transporter and Ubiquitin-conjugatine enzyme E2 respectively, were also predicted to be the target genes of gma-miR399. In soybean, <italic> <span class="gene">Glyma.08G359400</span> </italic> and <italic> <span class="gene">Glyma18G177400</span> </italic> encoding multicopper oxidases and <italic> <span class="gene">Glyma03G021900</span> </italic> encoding a growth-regulating factor were also predicted to be putative targets of gma-miR399a/b (Table <xref ref-type="supplementary-material" rid="SM7"> S4 </xref> ). We have detected the expression of these predicted target genes of gma-miR399a/b, as shown in Figure <xref ref-type="supplementary-material" rid="SM3"> S3 </xref> , <italic> <span class="gene">Glyma.03G021900,</span> <span class="gene">Glyma.08G359400,</span> <span class="gene">Glyma.14G188000</span> </italic> , and <italic> <span class="gene">Glyma.15G074200</span> </italic> were significantly induced by salt stress. Another known miRNA, gma-miR390e, was also predicted to target two additional genes ( <italic> <span class="gene">Glyma.02G281100</span> </italic> and <italic> <span class="gene">Glyma.14G033500</span> </italic> encoding LRR receptor-like kinases in addition to <italic> TAS3 </italic> ( <italic> <span class="gene">Glyma.15G137400</span> </italic> ) in soybean. Furthermore, we found that several novel miRNAs target genes that are involved in various biological processes. For example, Gly13 was predicted to target seven genes, including those encoding cysteine synthase A, ubiquitin-like protease protein 2, and a class I glutamine amidotransferase. Our results suggest that these soybean miRNAs modulate RAM <span class="trait">activity</span> by coordinating complicated regulatory networks.
					</p>
				
					<p>&nbsp;&nbsp;
						One efficient way to elucidate miRNA-mediated biological processes and plant stress responses is to find their target genes. Based on computational predictions, we identified putative target genes for all salt-responsive miRNAs in the soybean RAM. As expected, some salt-responsive miRNAs identified in this study, including gma-miR156 and gma-miR172, target the same family of genes encoding various proteins (e.g., <span class="gene">AP2</span> transcription factors; Table <xref ref-type="supplementary-material" rid="SM7"> S4 </xref> ). Unexpectedly, we found that some miRNAs may target more diverse gene families in soybean. For example, miR390 directly targets the non-coding TAS3 precursor RNA to trigger the biogenesis of tasiRNAs-ARF, which cleave the transcripts of ARF2, ARF3, and ARF4, thereby regulating lateral <span class="trait">root</span> development (Chen and Xiong, <xref rid="B10" ref-type="bibr"> 2010 </xref> ; Marin et al., <xref rid="B49" ref-type="bibr"> 2010 </xref> ). In soybean, we found that miR390 has additional putative target genes with diverse functions, except the <italic> Arabidopsis </italic> TAS homolog. One target gene encodes a protein with strong similarity to AtBAM3, a receptor kinase-like protein that regulates shoot and floral meristem development (Depuydt et al., <xref rid="B13" ref-type="bibr"> 2013 </xref> ). Taking into account the role of AtBAM3 in meristems, it is likely that gma-miR390 modulates RAM <span class="trait">activity</span> and <span class="trait">root</span> developmental plasticity through an <italic> AtBAM3 </italic> homolog in soybean. <span class="gene">Gma-miR399</span> also has additional putative target genes with different biochemical features beyond the well-known target gene <italic> PHO2 </italic> , which is involved in plant responses to phosphate starvation. Based on our results, <italic> <span class="gene">GmPHO2</span> </italic> has no significantly change in response to salt stress but another homologous gene <italic> <span class="gene">Glyma.15G074200</span> </italic> was obviously induced under salt stress. In <italic> Arabidopsis </italic> , the homologous gene of <italic> <span class="gene">Glyma.03G021900</span> </italic> function as GROWTH-REGULATING FACTOR 9, regulates the plant growth and development process (Horiguchi et al., <xref rid="B29" ref-type="bibr"> 2005 </xref> ). <italic> <span class="gene">Glyma.08G359400</span> </italic> was annotated as a multicopper oxidase which might play roles in plants development to response abiotic stress and phosphate sensing in the <span class="trait">root</span> tip (Svistoonoff et al., <xref rid="B57" ref-type="bibr"> 2007 </xref> ; Desnos, <xref rid="B14" ref-type="bibr"> 2008 </xref> ). We found both of these genes were induced by salt stress in the <span class="trait">root</span> tip of soybean (Figure <xref ref-type="supplementary-material" rid="SM3"> S3 </xref> ). These data suggest that gma-miR399a/b may participate in the plastic development of RAM under salt stress through target one or more genes. These findings suggest that miRNAs modulate plant growth and stress responses via a complicated network in soybean that includes the negative regulation of functionally diverse target genes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPAP4, a novel purple acid phosphatase gene isolated from soybean (Glycine max), enhanced extracellular phytate utilization in Arabidopsis thaliana.</h2>
			<h3>PMID:24595918  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kong Y(1), Li X, Ma J, Li W, Yan G, Zhang C.
				<br/> 
				年份：2014 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: <span class="gene">GmPAP4</span> , a novel plant <span class="gene">PAP</span> gene in soybean, has phytase <span class="trait">activity.</span> Over-expressing <span class="gene">GmPAP4</span> can enhance Arabidopsis growth when phytate is the sole P source in culture. Phosphorus (P) is an important macronutrient for plant growth and development. However, most of the total P in soils is fixed into organic phosphate (Po). Purple acid phosphatase (PAP) can hydrolyze Po in the soil to liberate inorganic phosphate and enhance plant P utilization. We isolated a novel <span class="gene">PAP</span> gene, <span class="gene">GmPAP4,</span> from soybean (Glycine max). It had an open reading frame of 1,329 bp, encoding 442 amino acid residues. Sequence alignment and phylogenetics analysis indicated that <span class="gene">GmPAP4</span> was similar to other plant PAPs with large molecular masses. Quantitative real-time PCR analysis showed that the induced expression of <span class="gene">GmPAP4</span> was greater in P-efficient genotype Zhonghuang15 (ZH15) than in P-inefficient genotype Niumaohuang (NMH) during the periods of flowering (28-35 days post phytate stress; DPP) and pod formation (49-63 DPP). Moreover, peak expression, at 63 DPP, was about 3-fold higher in 'ZH15' than in 'NMH'. Sub-cellular localization showed that <span class="gene">GmPAP4</span> might be on plasma membrane or in cytoplasm. Over-expressing <span class="gene">GmPAP4</span> in Arabidopsis resulted in significant rises in P acquisition and utilization compared with the wild-type (WT). Under phytate condition, transgenic Arabidopsis plants showed increases of approximately 132.7 % in dry weight and 162.6 % in shoot P content compared with the WT. Furthermore, when phytate was added as the sole P source in cultures, the <span class="trait">activity</span> of acid phosphatase was significantly higher in transgenic plants. Therefore, <span class="gene">GmPAP4</span> is a novel <span class="gene">PAP</span> gene that functions in plant's utilization of organic phosphate especially under phytate condition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPep914, an eight-amino acid peptide isolated from soybean leaves, activates defense-related genes.</h2>
			<h3>PMID:21478368  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yamaguchi Y(1), Barona G, Ryan CA, Pearce G.
				<br/> 
				年份：2011 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Only a handful of endogenous peptide defense signals have been isolated from plants. Herein, we report a novel peptide from soybean (Glycine max) leaves that is capable of alkalinizing the media of soybean suspension cells, a response that is generally associated with defense peptides. The peptide, DHPRGGNY, was synthesized and found to be active at 0.25 nM and requiring only 5 to 10 min to obtain a maximal pH change. The peptide is located on the carboxy-terminal end of a 52-amino acid precursor protein <span class="gene">(Glyma12g00990)</span> deduced from the soybean genome project. A search of the soybean databank revealed a homolog <span class="gene">(Glyma09g36370)</span> that contained a similar peptide, DLPRGGNY, which was synthesized and shown to have identical <span class="trait">activity.</span> The peptides, designated <span class="gene">GmPep914</span> (DHPRGGNY) and <span class="gene">GmPep890</span> (DLPRGGNY), were capable of inducing the expression of both <span class="gene">Glyma12g00990</span> <span class="gene">(GmPROPEP914)</span> and <span class="gene">Glyma09g36370</span> <span class="gene">(GmPROPEP890)</span> in cultured soybean suspension cells within 1 h. Both peptides induced the expression of defense genes, including CYP93A1, a cytochrome <span class="gene">P450</span> gene involved in phytoalexin synthesis, chitinaseb1-1, a chitinase involved in pathogen defense, and Glycine max chalcone synthase1 <span class="gene">(Gmachs1),</span> chalcone synthase, involved in phytoalexin production. Both <span class="gene">GmPROPEP914</span> and <span class="gene">GmPROPEP890</span> were highly expressed in the roots, relative to the aerial portions of the plant. However, treatment of the aerial portion of soybean plants with hormones involved in elicitation of defense responses revealed a significant increase in expression levels of <span class="gene">GmPROPEP914</span> and <span class="gene">GmPROPEP890.</span> A search of gene databases revealed homologous sequences in other members of the Fabales and also in the closely related Cucurbitales but not in any other order of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-function analysis of the <span class="gene">GmRIC1</span> signal peptide and CLE domain required for nodulation control in soybean.</h2>
			<h3>PMID:23386683  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reid DE(1), Li D, Ferguson BJ, Gresshoff PM.
				<br/> 
				年份：2013 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legumes control the nitrogen-fixing <span class="trait">root</span> nodule symbiosis in response to external and internal stimuli, such as nitrate, and via systemic autoregulation of nodulation (AON). Overexpression of the CLV3/ESR-related (CLE) pre-propeptide-encoding genes <span class="gene">GmNIC1</span> (nitrate-induced and acting locally) and <span class="gene">GmRIC1</span> (Bradyrhizobium-induced and acting systemically) suppresses soybean nodulation dependent on the <span class="trait">activity</span> of the nodulation autoregulation receptor kinase <span class="gene">(GmNARK).</span> This nodule inhibition response was used to assess the relative importance of key structural components within and around the CLE domain sequences of these genes. Using a site-directed mutagenesis approach, mutants were produced at each amino acid within the CLE domain (RLAPEGPDPHHN) of <span class="gene">GmRIC1.</span> This approach identified the Arg1, Ala3, Pro4, Gly6, Pro7, Asp8, His11, and Asn12 residues as critical to <span class="gene">GmRIC1</span> nodulation suppression <span class="trait">activity</span> (NSA). In contrast, none of the mutations in conserved residues outside of the CLE domain showed compromised NSA. Chimeric genes derived from combinations of <span class="gene">GmRIC1</span> and <span class="gene">GmNIC1</span> domains were used to determine the role of each pre-propeptide domain in NSA differences that exist between the two peptides. It was found that the transit peptide and CLE peptide regions of <span class="gene">GmRIC1</span> significantly enhanced <span class="trait">activity</span> of <span class="gene">GmNIC1.</span> In contrast, the comparable <span class="gene">GmNIC1</span> domains reduced the NSA of <span class="gene">GmRIC1.</span> Identification of these critical residues and domains provides a better understanding of how these hormone-like peptides function in plant development and regulation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification of Soybean U-Box E3 Ubiquitin Ligases and Roles of <span class="gene">GmPUB8</span> in Negative Regulation of <span class="trait">Drought</span> Stress Response in Arabidopsis.</h2>
			<h3>PMID:27057003  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang N(1), Liu Y(1), Cong Y(1), Wang T(1), Zhong X(1), Yang S(1), Li Y(2), Gai J(2).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant U-box (PUB) E3 ubiquitin ligases play important roles in hormone signaling pathways and response to abiotic stresses, but little is known about them in soybean, Glycine max. Here, we identified and characterized 125 PUB genes from the soybean genome, which were classified into eight groups according to their protein domains. Soybean PUB genes <span class="gene">(GmPUB</span> genes) are broadly expressed in many tissues and are a little more abundant in the roots than in the other tissues. Nine <span class="gene">GmPUB</span> genes, <span class="gene">GmPUB1-GmPUB9,</span> showed induced expression patterns by <span class="trait">drought,</span> and the expression of <span class="gene">GmPUB8</span> was also induced by exogenous ABA and NaCl. <span class="gene">GmPUB8</span> was localized to post-Golgi compartments, interacting with <span class="gene">GmE2</span> protein as demonstrated by yeast two-hybrid (Y2H) and bimolecular fluorescence complementation (BiFC) experiments, and showed E3 ubiquitin ligase <span class="trait">activity</span> by in vitro ubiquitination assay. Heterogeneous overexpression of <span class="gene">GmPUB8</span> in Arabidopsis showed decreased <span class="trait">drought</span> tolerance, enhanced sensitivity with respect to osmotic and salt stress inhibition of seed germination and seedling growth, and inhibited ABA- and mannitol-mediated stomatal closure. Eight <span class="trait">drought</span> stress-related genes were less induced in <span class="gene">GmPUB8-overexpressing</span> Arabidopsis after <span class="trait">drought</span> treatment compared with the wild type and the pub23 mutant. Taken together, our results suggested that <span class="gene">GmPUB8</span> might negatively regulate plant response to <span class="trait">drought</span> stress. In addition, Y2H and BiFC showed that <span class="gene">GmPUB8</span> interacted with soybean COL (CONSTANS LIKE) protein. <span class="gene">GmPUB8-overexpressing</span> Arabidopsis flowered earlier under middle- and short-day conditions but later under long-day conditions, indicating that <span class="gene">GmPUB8</span> might regulate flowering time in the photoperiod pathway. This study helps us to understand the functions of PUB E3 ubiquitin ligases in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmWRKY27 interacts with <span class="gene">GmMYB174</span> to reduce expression of <span class="gene">GmNAC29</span> for stress tolerance in soybean plants.</h2>
			<h3>PMID:25990284  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang F(1), Chen HW(1), Li QT(1), Wei W(1), Li W(2), Zhang WK(1), Ma B(1), Bi YD(2), Lai YC(2), Liu XL(3), Man WQ(3), Zhang JS(1), Chen SY(1).
				<br/> 
				年份：2015 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is an important crop for oil and protein resources worldwide. The molecular mechanism of the abiotic stress response in soybean is largely unclear. We previously identified multiple stress-responsive WRKY genes from soybean. Here, we further characterized the roles of one of these genes, <span class="gene">GmWRKY27,</span> in abiotic stress tolerance using a transgenic hairy <span class="trait">root</span> assay. <span class="gene">GmWRKY27</span> expression was increased by various abiotic stresses. Over-expression and RNAi analysis demonstrated that <span class="gene">GmWRKY27</span> improves salt and <span class="trait">drought</span> tolerance in transgenic soybean hairy roots. Measurement of physiological parameters, including reactive oxygen species and proline contents, supported this conclusion. <span class="gene">GmWRKY27</span> inhibits expression of a downstream gene <span class="gene">GmNAC29</span> by binding to the W-boxes in its promoter region. The <span class="gene">GmNAC29</span> is a negative factor of stress tolerance as indicated by the performance of transgenic hairy roots under stress. <span class="gene">GmWRKY27</span> interacts with <span class="gene">GmMYB174,</span> which also suppresses <span class="gene">GmNAC29</span> expression and enhances <span class="trait">drought</span> stress tolerance. The <span class="gene">GmWRKY27</span> and <span class="gene">GmMYB174</span> may have evolved to bind to neighbouring cis elements in the <span class="gene">GmNAC29</span> promoter to co-reduce promoter <span class="trait">activity</span> and gene expression. Our study discloses a valuable mechanism in soybean for regulation of the stress response by two associated transcription factors. Manipulation of these genes should facilitate improvements in stress tolerance in soybean and other crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> <span class="gene">GmPRP2</span> </italic> promoter drives root-preferential expression in transgenic <italic> Arabidopsis </italic> and soybean hairy roots</h2>
			<h3>PMID:25224536</h3>
				

				
					<h3>PMCID:4172956</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Li </given-names>  </name>  <address>  <email> chenli421@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Bingjun </given-names>  </name>  <address>  <email> jiangbingjun@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <address>  <email> wucunxiang@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Shi </given-names>  </name>  <address>  <email> sunshi@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Hou </surname>  <given-names> Wensheng </given-names>  </name>  <address>  <email> houwensheng@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <address>  <email> hantianfu@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> MOA Key Laboratory of Soybean Biology (Beijing), Institute of Crop Sciences, The Chinese Academy of Agricultural Sciences, Beijing, China  </aff>
				<br/> 
				年份：<day> 9 </day>  <month> 9 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Promoters play important roles in gene expression and function. There are three basic types of promoters: constitutive, specific, and inducible. Constitutive promoters are widely used in genetic engineering, but these promoters have limitations. Inducible promoters are activated by specific inducers. Tissue-specific promoters are a type of specific promoters that drive gene expression in specific tissues or organs. Here, we cloned and characterized the <span class="gene">GmPRP2</span> promoter from soybean. The expression pattern indicated that this promoter is root-preferential in transgenic Arabidopsis and the hairy roots of soybean. It can be used to improve the <span class="trait">root</span> <span class="trait">resistance</span> or tolerance to pathogens, pests, malnutrition and other abiotic stresses which cause extensive annual losses in soybean production. RESULTS: The <span class="gene">GmPRP2</span> promoter <span class="gene">(GmPRP2p-1062)</span> was isolated from soybean cv. Williams 82. Sequence analysis revealed that this promoter contains many cis-acting elements, including root-specific motifs. The <span class="gene">GmPRP2p-1062</span> and its 5'-deletion fragments were fused with the GUS reporter gene and introduced into Arabidopsis and the hairy roots of soybean to further determine promoter <span class="trait">activity.</span> Histochemical analysis in transgenic Arabidopsis showed that GUS <span class="trait">activity</span> was mainly detected in roots and hypocotyls in all deletion fragments except <span class="gene">GmPRP2p-471</span> (a 5'-deletion fragment of <span class="gene">GmPRP2p-1062</span> with 471 bp length). GUS <span class="trait">activity</span> was higher in transgenic Arabidopsis and hairy roots with <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> (a 5'-deletion fragment of <span class="gene">GmPRP2p-1062</span> with 852 bp length) constructs than the other two constructs. GUS <span class="trait">activity</span> was enhanced by NaCl, PEG, IAA and JM treatments and decreased by SA, ABA and GA treatments in transgenic Arabidopsis. CONCLUSIONS: <span class="gene">GmPRP2p-1062</span> is a root-preferential promoter, and its core fragment for root-preferential expression might lie between -369 and +1. <span class="gene">GmPRP2p-852</span> may be useful in the genetic engineering of novel soybean cultivars in the future.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Promoters play important roles in gene expression and function. There are three basic types of promoters: constitutive, specific, and inducible. Constitutive promoters are widely used in genetic engineering, but these promoters have limitations. Inducible promoters are activated by specific inducers. Tissue-specific promoters are a type of specific promoters that drive gene expression in specific tissues or organs. Here, we cloned and characterized the <italic> <span class="gene">GmPRP2</span> </italic> promoter from soybean. The expression pattern indicated that this promoter is root-preferential in transgenic <italic> Arabidopsis </italic> and the hairy roots of soybean. It can be used to improve the <span class="trait">root</span> <span class="trait">resistance</span> or tolerance to pathogens, pests, malnutrition and other abiotic stresses which cause extensive annual losses in soybean production.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmPRP2</span> </italic> promoter <span class="gene">(GmPRP2p-1062)</span> was isolated from soybean cv. Williams 82. Sequence analysis revealed that this promoter contains many <italic> cis </italic> -acting elements, including root-specific motifs. The <span class="gene">GmPRP2p-1062</span> and its 5&#x02019;-deletion fragments were fused with the <italic> GUS </italic> reporter gene and introduced into <italic> Arabidopsis </italic> and the hairy roots of soybean to further determine promoter <span class="trait">activity.</span> Histochemical analysis in transgenic <italic> Arabidopsis </italic> showed that GUS <span class="trait">activity</span> was mainly detected in roots and hypocotyls in all deletion fragments except <span class="gene">GmPRP2p-471</span> (a 5&#x02019;-deletion fragment of <span class="gene">GmPRP2p-1062</span> with 471&#x000a0;bp length). GUS <span class="trait">activity</span> was higher in transgenic <italic> Arabidopsis </italic> and hairy roots with <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> (a 5&#x02019;-deletion fragment of <span class="gene">GmPRP2p-1062</span> with 852&#x000a0;bp length) constructs than the other two constructs. GUS <span class="trait">activity</span> was enhanced by NaCl, PEG, IAA and JM treatments and decreased by SA, ABA and GA treatments in transgenic <italic> Arabidopsis </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Alfalfa <italic> A9 </italic> , rice <italic> Rcc2 </italic> and <italic> Rcc3 </italic> , carrot <italic> <span class="gene">PRP1</span> </italic> , maize <italic> ZRP3 </italic> and bean <italic> PVR5 </italic> , which all encode a proline-rich protein (PRP), are expressed preferentially in <span class="trait">root</span> [ <xref ref-type="bibr" rid="CR25"> 25 </xref> - <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. Three members of the <italic> PRP </italic> family are expressed in soybean with distinct, individual patterns of expression in different organs and at different development stages [ <xref ref-type="bibr" rid="CR29"> 29 </xref> - <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. <italic> <span class="gene">GmPRP1</span> </italic> and <italic> <span class="gene">GmPRP2</span> </italic> exhibit root-specific expression [ <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. Soybean <italic> <span class="gene">PRP1</span> </italic> mRNA is highly abundant in the elongating and mature region of the hypocotyls epidermal cells of seedlings. Soybean <italic> <span class="gene">PRP2</span> </italic> mRNA accumulates in phloem cells, and <italic> <span class="gene">PRP3</span> </italic> mRNA is specifically localized to the endodermoid layer of cells in the hypocotyl-elongating region [ <xref ref-type="bibr" rid="CR32"> 32 </xref> , <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. <italic> PRPs </italic> are expressed with spatiotemporal specificity [ <xref ref-type="bibr" rid="CR34"> 34 </xref> , <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. Moreover, factors associated with biotic and abiotic stresses also influence the expression of <italic> PRPs </italic> [ <xref ref-type="bibr" rid="CR36"> 36 </xref> - <xref ref-type="bibr" rid="CR39"> 39 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Some genes encoding PRPs have been isolated from soybean, but the function of the <italic> PRP </italic> promoters is not well characterized. Here, we cloned the <italic> <span class="gene">GmPRP2</span> </italic> promoter and studied its expression <span class="trait">activity</span> in transgenic <italic> Arabidopsis </italic> and soybean hairy roots.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmPRP2</span> </italic> expression was investigated by real-time PCR from <span class="trait">root,</span> stem, leaf, flower, seed and hypocotyl. The expression level was highest in the <span class="trait">root,</span> and second in the hypocotyl, seed and stem. The level in the leaf and flower were much lower than in the <span class="trait">root</span> (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Therefore, <italic> <span class="gene">GmPRP2</span> </italic> gene showed a root-preferential expression. <fig id="Fig1"> <label> Figure 1 </label> <caption> <p> <bold> The expression level of <span class="gene">GmPRP2</span> by qRT-PCR in different tissues. </bold> The relative expression level of <span class="gene">GmPRP2</span> from real-time PCR was in different tissues. R, <span class="trait">root;</span> S, stem; L, leaf; F, flower; Sd, seed; H, hypocotyl. Data are the means of three replicates with SE shown by vertical bars. The capitals differ significantly by one-sided paired <italic> t </italic> test at P&#x02009;&#x0003c;&#x02009;0.01.
					</p>
				
					<p>&nbsp;&nbsp;
						A 1,236 bp 5&#x02019; flanking region fragment of the <italic> <span class="gene">PRP2</span> </italic> gene was amplified using first-round PCR. This fragment contained a 174 bp <italic> <span class="gene">GmPRP2</span> </italic> gene partial coding sequence. The second PCR product contained a 1,062&#x000a0;bp flanking sequence upstream of the translated initiation codon. The first nucleotide of <italic> <span class="gene">GmPRP2</span> </italic> cDNA was designated as +1 to orient the sequence numbers. The promoter sequence was analyzed using the PLACE and PlantCARE web tools. Several putative <italic> cis </italic> -regulatory elements were deciphered from the promoter sequence of <italic> <span class="gene">GmPRP2</span> </italic> (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> A). TATA box sequence elements, which were required for critical and precise transcription initiation, were found in the &#x02212;317 region of the sequence. CAAT BOX sequences, which were responsible for the tissue-specific promoter <span class="trait">activity,</span> were found at numerous positions: &#x02212;776, &#x02212;762, &#x02212;449, &#x02212;409, &#x02212;347, &#x02212;342, &#x02212;227, and &#x02212;167. OSE2ROOTNODULE sequences, which were responsible for an organ-specific promoter <span class="trait">activity</span> in infected cells of <span class="trait">root</span> nodules, were found at &#x02212;1000 and &#x02212;430. ROOTMOTIFTAPOX1 sequence elements, which were required for organ specificity, were identified at the &#x02212;251 position. OSE2ROOTNODULE elements and ROOTMOTIFTAPOX1 element are critical for root-specific expression. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> <bold> Sequence of the <span class="gene">GmPRP2p-1062</span> denoting the </bold> <bold> <italic> cis </italic> </bold> <bold> -elements predicted by the PLACE and PlantCARE databases and schematic diagrams of truncated <span class="gene">GmPRP2p-1062</span> constructs. A </bold> A 1,236&#x000a0;bp 5&#x02019;-flanking region fragment of the <italic> <span class="gene">PRP2</span> </italic> gene, containing 1062&#x000a0;bp promoter sequence. The translated start site is defined as +1. The TATA box, partial CAAT box, Wbox, ABRE, MYB, MYC, root-specific elements are underlined with different colors. <bold> B </bold> Schematic diagrams of truncated <span class="gene">GmPRP2p-1062</span> constructs. The numbers on the left indicate 5&#x02019;-deletion fragments of the promoter. Some <italic> cis </italic> -elements are also marked with the colored columns.
					</p>
				
					<p>&nbsp;&nbsp;
						The 5&#x02019;-deletion fragments were also fused to the GUS gene and transformed into <italic> Arabidopsis </italic> . GUS staining showed that the <span class="gene">GmPRP2p-852</span> and <span class="gene">GmPRP2p-369</span> constructs presented similar expression patterns as the <span class="gene">GmPRP2p-1062</span> construct, which was high in roots and hypocotyls, except leaves. GUS staining was not visible with the <span class="gene">GmPRP2p-471</span> construct (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). Fluorometric GUS assays revealed that <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> constructs drove strong GUS <span class="trait">activity</span> in roots, and GUS <span class="trait">activity</span> was significantly reduced in <span class="gene">GmPRP2p-471</span> and <span class="gene">GmPRP2p-369</span> constructs compared with <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> constructs (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). The 5&#x02019;-deletions of the <span class="gene">GmPRP2p-1062</span> from &#x02212;1062 to &#x02212;852 slightly increased GUS <span class="trait">activity.</span> A further deletion to &#x02212;471 obviously decreased GUS <span class="trait">activity,</span> and a deletion to &#x02212;369 was slightly better than &#x02212;471. However, the GUS expression level in these plants was much lower than in the transgenic plants with <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> constructs. We speculated that the promoter fragment from &#x02212;471 to &#x02212;369 may contain the suppressor which induced the expression level increased in <span class="gene">GmPRP2p-369</span> compared with <span class="gene">GmPRP2p-471.</span> The GUS <span class="trait">activity</span> in roots was obviously higher than in leaves, regardless of the promoter fragment except <span class="gene">GmPRP2-471.</span> The results of GUS staining and fluorometric GUS assays were uniform. <fig id="Fig4"> <label> Figure 4 </label> <caption> <p> <bold> GUS <span class="trait">activity</span> assays in T </bold> <sub> <bold> 3 </bold> </sub> <bold> transgenic </bold> <bold> <italic> Arabidopsis </italic> </bold> <bold> roots and leaves carrying <span class="gene">GmPRP2p-1062,</span> <span class="gene">GmPRP2p-852,</span> <span class="gene">GmPRP2p-471</span> and <span class="gene">GmPRP2p-369</span> constructs. </bold> GUS <span class="trait">activity</span> assays in T <sub> 3 </sub> transgenic <italic> Arabidopsis </italic> roots and leaves carrying different deletion fragments of the <span class="gene">GmPRP2p-1062.</span> Each promoter fragment had three transgenic lines. P1062-9, P1062-14, and P1062-16 are the three transgenic lines carrying the <span class="gene">GmPRP2p-1062</span> construct; P852-5, P852-9, and P852-30 are the three transgenic lines carrying the <span class="gene">GmPRP2p-852</span> construct; P471-4, P471-6, and P471-7 are the three transgenic lines carrying the <span class="gene">GmPRP2p-471</span> construct; P369-2, P369-3, and P369-6 are the three transgenic lines carrying the <span class="gene">GmPRP2p-369</span> construct. Data are the means of three replicates with SE shown by vertical bars. **differ significantly by one-sided paired <italic> t </italic> test at P&#x02009;&#x0003c;&#x02009;0.01.
					</p>
				
					<p>&nbsp;&nbsp;
						We tested the GUS <span class="trait">activity</span> in the roots of 20-day-old transgenic seedlings treated with 1&#x000a0;mM SA, 100&#x000a0;&#x003bc;M MeJA, 100&#x000a0;&#x003bc;M IAA, 100&#x000a0;&#x003bc;M GA, and 100&#x000a0;&#x003bc;M ABA for 24&#x000a0;h to assess the response of the <span class="gene">GmPRP2p-1062</span> and 5&#x02019;-deletion fragments to various hormones. Seedlings treated with H <sub> 2 </sub> O and wild type seedlings treated with the above stressors were used as controls. MeJA enhanced the GUS <span class="trait">activity</span> in transgenic seedlings containing the <span class="gene">GmPRP2p-1062</span> construct compared to H <sub> 2 </sub> O control samples (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). The GUS <span class="trait">activity</span> was increased by 2.22-fold (P1062-9), 2.86-fold (P1062-14) and 2.02-fold (P1062-16) in the three transgenic lines. Similar results were observed in seedlings transformed with the <span class="gene">GmPRP2p-852,</span> <span class="gene">GmPRP2p-471</span> and <span class="gene">GmPRP2p-369</span> constructs. The GUS <span class="trait">activity</span> was increased one to three times in transgenic lines. IAA increased the GUS <span class="trait">activity</span> in most transgenic seedlings with different constructs by one to two times (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). <fig id="Fig6"> <label> Figure 6 </label> <caption> <p> <bold> GUS <span class="trait">activity</span> of transgenic </bold> <bold> <italic> Arabidopsis </italic> </bold> <bold> with different constructs <span class="gene">(GmPRP2p-1062,</span> <span class="gene">GmPRP2p-852,</span> <span class="gene">GmPRP2p-471,</span> <span class="gene">GmPRP2p-369)</span> in roots in responses to MeJA, IAA, SA, GA, ABA, NaCl, PEG. </bold> The GUS <span class="trait">activity</span> for each treatment was measured in 20-day-old seedlings from each of three independent transgenic lines. Transgenic seedlings were treated for 24&#x000a0;h. Control seedlings were treated with water. P1062-9, P1062-14, and P1062-16 are the three transgenic lines carrying the <span class="gene">GmPRP2p-1062</span> construct; P852-5, P852-9, and P852-30 are the three transgenic lines carrying the <span class="gene">GmPRP2p-852</span> construct; P471-4, P471-6, and P471-7 are the three transgenic lines carrying the <span class="gene">GmPRP2p-471</span> construct; P369-2, P369-3, and P369-6 are the three transgenic lines carrying the <span class="gene">GmPRP2p-369</span> construct. Data are presented as the means of three replicates with SE shown by vertical bars. *and** differ significantly by one-sided paired <italic> t </italic> test at P&#x02009;&#x0003c;&#x02009;0.05 and P&#x02009;&#x0003c;&#x02009;0.01, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						SA and GA treatments decreased the GUS <span class="trait">activity</span> approximately 40-70% in transgenic seedlings with the <span class="gene">GmPRP2p-1062</span> construct and approximately 70-80% in transgenic seedlings with the <span class="gene">GmPRP2p-852</span> construct. The GUS <span class="trait">activity</span> was decreased 70-90% in transgenic seedlings with the <span class="gene">GmPRP2p-471</span> and <span class="gene">GmPRP2p-369</span> constructs. ABA treatment slightly decreased the GUS <span class="trait">activity</span> approximately 30-60% in most transgenic seedlings compared to SA and GA treatments (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We tested GUS <span class="trait">activity</span> in the roots of 20-day-old transgenic seedlings treated with 200&#x000a0;mM NaCl and 20% PEG6000 for 24&#x000a0;h to identify the response of <span class="gene">GmPRP2p-1062</span> and 5&#x02019;-deletion fragments to environmental stresses. The seedlings treated with H <sub> 2 </sub> O and wild type seedlings treated with the above stressor were used as controls. NaCl and PEG treatment enhanced GUS <span class="trait">activity.</span> NaCl treatment increased GUS <span class="trait">activity</span> by 3.19-fold (P1062-9), 1.05-fold (P1062-14) and 1.05-fold (P1062-16) in the three <span class="gene">GmPRP2p-1062</span> transgenic lines. The GUS <span class="trait">activity</span> was increased one to two times in most of the transgenic lines with the other constructs (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). PEG treatment increased the GUS <span class="trait">activity</span> in all transgenic lines carrying different constructs, and greatly increased the GUS <span class="trait">activity</span> by 2.29-fold (P471-4), 3.24-fold (P471-6) and 4.08-fold (P471-7) in the <span class="gene">GmPRP2p-471</span> transgenic lines (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). However, the GUS levels were much lower in the <span class="gene">GmPRP2p-471</span> transgenic lines than in the <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> transgenic lines (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We used GUS staining to select positive roots because the hairy roots from cotyledon were not all positive. For each promoter construct, twenty soybean cotyledons as explants were used to transform. Thirty to fifty hairy roots from each transgenic promoter cotyledons were examined for GUS staining. The hairy roots of transgenic <span class="gene">GmPRP2p-471</span> constructs were also detected using PCR. Ten GUS-positive hairy roots of each transgenic construct were used to measure GUS <span class="trait">activity.</span> The GUS <span class="trait">activity</span> results were used for statistical analysis. The 35S promoter drove the highest GUS expression level compared to the maximum expression level of each transgenic construct (Figure&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ). The GUS expression levels in all transgenic <span class="gene">GmPRP2p-1062</span> and 5&#x02019;-deletion fragments were lower than that in the 50 <sup> th </sup> percentile of the 35S promoter. The GUS expression levels for <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> were higher than those of the lowest GUS expression level of the 35S promoter. The GUS expression levels of most hairy roots for <span class="gene">GmPRP2p-369</span> were less than the minimum of the 35S promoter, <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852.</span> The expression <span class="trait">activity</span> of <span class="gene">GmPRP2p-471</span> was extremely weak, and GUS staining was not detectable by eyes. The maximum GUS expression of <span class="gene">GmPRP2p-1062</span> was higher than that of <span class="gene">GmPRP2p-852.</span> But the GUS expression in the 50 <sup> th </sup> percentile of <span class="gene">GmPRP2p-1062</span> lines was lower than those of <span class="gene">GmPRP2p-852</span> (Figure&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ). <fig id="Fig8"> <label> Figure 8 </label> <caption> <p> <bold> GUS <span class="trait">activity</span> of transgenic soybean hairy roots with different constructs <span class="gene">(GmPRP2p-1062,</span> <span class="gene">GmPRP2p-852,</span> <span class="gene">GmPRP2p-471,</span> <span class="gene">GmPRP2p-369,</span> 35S). </bold> Box plot analysis of GUS activities obtained in transgenic soybean hairy roots with different constructs <span class="gene">(GmPRP2-1062,</span> <span class="gene">GmPRP2-852,</span> <span class="gene">GmPRP2-471,</span> <span class="gene">GmPRP2-369,</span> 35S). GUS <span class="trait">activity</span> is given in nmol 4-MU min <sup> &#x02212;1 </sup> mg <sup> &#x02212;1 </sup> total protein. The lower boundary of each box denotes the 25 <sup> th </sup> percentile for each promoter, the solid line within each box denotes the 50 <sup> th </sup> percentile, the upper boundary of each box denotes the 75 <sup> th </sup> percentile. Two ends of the vertical line denote the maximum and minimum. Outlying data are indicated by empty black circles.
					</p>
				
					<p>&nbsp;&nbsp;
						GUS staining showed a root-preferential expression in the vegetative stage in the model plant <italic> Arabidopsis </italic> (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). Therefore, <span class="gene">GmPRP2p-1062</span> can harbor some <italic> cis </italic> -regulatory elements and drive a root-preferential expression. Three 5&#x02019;-deletion fragments <span class="gene">(GmPRP2p-852,</span> <span class="gene">GmPRP2p-471,</span> <span class="gene">GmPRP2p-369)</span> also exhibited root-preferential expression, but the expression levels were significantly different. In our study, we detected that expression level in <span class="gene">GmPRP2p-471</span> was very low, then when a further deletion the expression level in <span class="gene">GmPRP2p-369</span> was increased, so we speculated that the suppressor may exist in the promoter fragment from &#x02212;471 to &#x02212;369. The elements in these deletion regions, which might be responsible for the <span class="trait">root</span> specificity and expression, require further research. The study of <span class="gene">GmPRP2p-1062</span> and 5&#x02019;-deletion fragments in soybean hairy roots showed that the expression patterns were similar to the transgenic <italic> Arabidopsis </italic> (Figure&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ), and GUS <span class="trait">activity</span> was much higher with the <span class="gene">GmPRP2p-1062</span> and <span class="gene">GmPRP2p-852</span> constructs than the other two constructs. The homogeneity in soybean and <italic> Arabidopsis </italic> inferred that the <italic> <span class="gene">PRP2</span> </italic> promoter has a similar expression pattern in dicotyledonous plants. The expression level was highest in transgenic <italic> Arabidopsis </italic> with the <span class="gene">GmPRP2p-852</span> construct, and the range of GUS <span class="trait">activity</span> varied little between the ten transgenic hairy roots with the <span class="gene">GmPRP2p-852</span> construct compared with the other constructs (Figure&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ). Therefore, the <span class="gene">GmPRP2p-852</span> may be suitable for application.
					</p>
				
					<p>&nbsp;&nbsp;
						Histochemical staining of GUS <span class="trait">activity</span> during reproductive growth was also observed in the petiole of green leaves and the split margins in siliques in all promoter constructs except the <span class="gene">GmPRP2p-471,</span> but the expression level was much lower than in roots. GUS staining was not observed in flowers and seeds (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). GUS staining was also detected in the main veins in transgenic lines with the <span class="gene">GmPRP2P-852</span> construct, which exhibited high expression levels. This result indicated that <italic> <span class="gene">GmPRP2</span> </italic> may be expressed in only a few tissues during the reproductive stage and suggested that the promoter may be suitable for genetic engineering with little concern about food safety.
					</p>
				
					<p>&nbsp;&nbsp;
						Biotic and abiotic stresses also influenced the expression of <italic> PRPs </italic> . Some evidence indicated that many internal and external factors, such as wounding, fungal infection, circadian rhythm, salt stress, <span class="trait">drought</span> stress, and plant regulators, up or down regulates the expression of <italic> PRPs </italic> [ <xref ref-type="bibr" rid="CR36"> 36 </xref> , <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. For example, Alfalfa <italic> MsPRP2 </italic> is salt-inducible [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. Low temperature induced transcripts of <italic> Brassica napus BnPRP </italic> . The bean <italic> PvPRP1 </italic> mRNA initially decreases then increases in wounded hypocotyls and decreases in the elicitor [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Cotton <italic> GhHyPRP3 </italic> transcription in roots was up-regulated by salt stress, cold stress, and osmotic stress and down-regulated by GA <sub> 3 </sub> [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. <italic> PRP </italic> gene expression is complicated with positive or negative regulation by stresses. The diversity of the promoter may regulate gene expression in many biological processes. Our results showed that the <span class="trait">activity</span> of the <span class="gene">GmPRP2p-1062</span> promoter was differentially regulated in response to various abiotic factors. NaCl, PEG, JM and IAA up-regulated the <span class="trait">activity,</span> and ABA, GA, and SA down-regulated the <span class="trait">activity</span> (Figure&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ). The deletion fragments contained stress elements. The &#x02212;1062 to &#x02212;853 fragment contained a copper- and salt-responsive element, and &#x02212;852 to &#x02212;472 contained light- and salt-responsive elements. The three 5&#x02019;-deletion fragments showed a similar response to the stress factors treated with the <span class="gene">GmPRP2p-1062.</span> The region from &#x02212;369 to +1 is rich in various biotic and abiotic stress-related <italic> cis- </italic> elements, including four CAATBOX, two MYB and three MYC <italic> cis </italic> -elements, two ARR1 <italic> cis </italic> -elements, one ABRE <italic> cis </italic> -element, one WBOX and one T/GBOX (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ), and this region is a crucial part of the promoter. MYB and MYC recognition sites may contribute to the activation of <span class="trait">drought-</span> and ABA-regulated gene expression. ABRE was identified as an ABA-responsive element. WBOX was involved in the activation of the ERF3 gene by wounding, and the T/GBOXATPIN2 element at &#x02212;13 was involved in jasmonate signaling (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). The above studies demonstrated that the <italic> <span class="gene">PRP2</span> </italic> promoter may regulate gene expression on the transcriptional level.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmPRP2p-1062</span> is a root-preferential promoter, and the core fragment for <span class="trait">root</span> expression might reside between &#x02212;369 and +1. The expression <span class="trait">activity</span> for this promoter was lower with shorter promoter sequences from &#x02212;471 to &#x02212;369&#x000a0;bp, and environmental stresses altered expression <span class="trait">activity.</span> <span class="gene">GmPRP2p-852</span> may be used in soybean genetic engineering to improve <span class="trait">root</span> tolerance and <span class="trait">resistance</span> in the future.
					</p>
				
					<p>&nbsp;&nbsp;
						Seeds of the soybean cultivar Williams 82 were grown in pots filled with vermiculite under 12&#x000a0;h light/12&#x000a0;h dark cycles at 28&#x000b0;C. The <span class="trait">root</span> tissues were cut for genomic DNA extraction after the first true leaf opening. Seeds of <italic> Arabidopsis thaliana </italic> (Columbia) were surface-sterilized with 10% NaClO for 15&#x000a0;min and washed four times with sterile water. The sterilized <italic> Arabidopsis </italic> seeds were placed on 1/2 <span class="gene">MS</span> medium, and the plates were transferred to a plant growth incubator for seed germination under 16&#x000a0;h light/ 8&#x000a0;h dark at 22&#x000b0;C after stratification at 4&#x000b0;C for 3 days.
					</p>
				
					<p>&nbsp;&nbsp;
						The total RNA was isolated from soybean different tissues of <span class="trait">root,</span> stem, leaf, flower, seed, and hypocotyl using Trizol agent (Trans). cDNAs were synthesized from 800 ng total RNA using the M-MLV reverse transcriptase and oligodT primer according to the manufacturer&#x02019;s instructions (Promega). qRT-PCR reactions (20 &#x003bc;l volume containing 2 &#x003bc;l cDNA as the template) were performed using the StepOne real-time PCR system (Applied Biosystems 7500) in standard mode with the KAPA SYBR FAST Universal qRT- PCR kit (KAPA Biosystems). Each tissue has three samples, and each sample of one tissue was performed in triplicate. The soybean <italic> Actin </italic> gene was used as the internal control. The primers for <italic> <span class="gene">GmPRP2</span> </italic> were 5&#x02032;- GCTCCTTAGTGCTGCTCCTT-3&#x02032; and 5&#x02032;-TCAGTGGGAGGCTTGTACA-3&#x02032;. The primers for <italic> Actin </italic> were 5&#x02032;-CGTTTCATGAATTCCAGTAGC-3&#x02032; and 5&#x02032;- GAGCTATGAATTGCCTGATGG-3&#x02032;.
					</p>
				
					<p>&nbsp;&nbsp;
						The plasmids of <span class="gene">GmPRP2p-1062</span> and 5&#x02019;-deletion fragments vectors were introduced into the <italic> Agrobacterium rhizogenes </italic> strain K599 to induce expression in hairy roots [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Soybean cv. Zigongdongdou was screened for transformation. Seeds were surface sterilized for 16&#x000a0;h using chlorine gas, which was produced by mixing 3.5&#x000a0;mL of 12&#x000a0;N HCl and 100&#x000a0;mL commercial bleach in a tightly sealed dessicator. Sterilized seeds were germinated in B5 medium, and entire cotyledons from 5-day-old seedlings as explant were harvested and wounded with a scalpel, which was previously dipped into an overnight culture of <italic> A. rhizogenes </italic> strain K599 carrying <span class="gene">GmPRP2p-1062::GUS</span> vector, <span class="gene">GmPRP2p-852::GUS</span> vector, <span class="gene">GmPRP2p-471::GUS</span> vector, <span class="gene">GmPRP2p-369::GUS</span> vector, 35S::GUS vector and PC13P1 vector. Subsequently, the entire cotyledons were immersed into <italic> A. rhizogenes </italic> culture and shaken (50&#x000a0;rpm) at 25&#x000b0;Cfor 30&#x000a0;min. Explants were dried on sterile filter paper and transferred to co-cultivation medium (CCM) containing 10% MS, 3.9&#x000a0;g/L morpholino ethanesulfonic acid, 150&#x000a0;mg/L cysteine and 150&#x000a0;mg/L dithiothreitol. The CCM was covered with sterile filter paper and incubated under a 16&#x000a0;h light/8&#x000a0;h dark cycle condition at 24&#x000b0;C. Entire cotyledons were cultured 5&#x000a0;days later on 1/2 MSB medium (1/2 Murashige and Skoog basal nutrient salts, B5 vitamins, 3% Suc and 3&#x000a0;g/L phytagel, pH&#x000a0;5.7) [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Approximately 10&#x02013;12 d after <span class="trait">root</span> emergence, 1&#x000a0;cm-long <span class="trait">root</span> tips or the bottoms of each <span class="trait">root</span> were cut for GUS staining. We selected GUS-positive roots for measurement of GUS <span class="trait">activity.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmPTF1, a novel transcription factor gene, is involved in conferring soybean tolerance to phosphate starvation.</h2>
			<h3>PMID:24634113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li XH(1), Wu B(1), Kong YB(1), Zhang CY(2).
				<br/> 
				年份：2014 Feb 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus plays a pivotal role in plant growth and development. In this study, we isolated and characterized <span class="gene">GmPTF1,</span> a basic helix-loop-helix (bHLH) transcription factor (TF) gene from soybean (Glycine max) with tolerance to inorganic phosphate (Pi) starvation. Alignment analysis indicated that <span class="gene">GmPTF1</span> and other reported bHLH TFs share significant similarity in the region of the bHLH domain. As with OsPTF1 and other homologous Pi starvation-related bHLH TFs (His-5, Glu-9, Arg-12, and Arg-13), all recognition motifs for the G-box (CACGTG) were present in the <span class="gene">GmPTF1</span> domain. Prokaryotic expression in Escherichia coli strain BL21 (DE3) plysS showed that a novel 40-kDa polypeptide was expressed when cells containing <span class="gene">GmPTF1</span> were induced. The subcellular localization in cells from onion epidermis and Arabidopsis roots demonstrated that the <span class="gene">GmPTF1</span> protein was found in the nucleus. Furthermore, analysis of transcription <span class="trait">activity</span> in yeast revealed that full-length <span class="gene">GmPTF1</span> and its N-terminal and C-terminal domains could activate the histidine, adenine, and uracil reporter genes. This suggested that the N-terminal and C-terminal peptides of <span class="gene">GmPTF1</span> act as transcriptional activators. When real-time quantitative polymerase chain reaction was performed, the expression of <span class="gene">GmPTF1</span> under conditions of phosphate starvation was significantly induced in soybean roots of the low-Pi-tolerant variety ZH15. Moreover, the relative level of expression was much higher there than in roots of the sensitive variety NMH from days 7 to 56 of low-Pi stress. These results imply that <span class="gene">GmPTF1</span> is involved in conferring tolerance to phosphate starvation in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and Characterization of a Novel Pathogenesis-Related Protein Gene ( <italic> <span class="gene">GmPRP</span> </italic> ) with Induced Expression in Soybean ( <italic> Glycine max </italic> ) during Infection with <italic> Phytophthora sojae </italic></h2>
			<h3>PMID:26114301</h3>
				

				
					<h3>PMCID:4482714</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Junjiang </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pathogenesis-related proteins (PR proteins) play crucial roles in the plant defense system. A novel PRP gene was isolated from highly resistant soybean infected with Phytophthora sojae (P. sojae) and was named <span class="gene">GmPRP</span> (GenBank accession number: KM506762). The amino acid sequences of <span class="gene">GmPRP</span> showed identities of 74%, 73%, 72% and 69% with PRP proteins from Vitis vinifera, Populus trichocarpa, Citrus sinensis and Theobroma cacao, respectively. Quantitative real-time reverse transcription PCR (qRT-PCR) data showed that the expression of <span class="gene">GmPRP</span> was highest in roots, followed by the stems and leaves. <span class="gene">GmPRP</span> expression was upregulated in soybean leaves infected with P. sojae. Similarly, <span class="gene">GmPRP</span> expression also responded to defense/stress signaling molecules, including salicylic acid (SA), ethylene (ET), abscisic acid (ABA) and jasmonic acid (JA). <span class="gene">GmPRP</span> was localized in the cell plasma membrane and cytoplasm. Recombinant <span class="gene">GmPRP</span> protein exhibited ribonuclease <span class="trait">activity</span> and significant inhibition of hyphal growth of P. sojae 1 in vitro. Overexpression of the <span class="gene">GmPRP</span> gene in T2 transgenic tobacco and T2 soybean plants resulted in enhanced <span class="trait">resistance</span> to Phytophthora nicotianae (P. nicotianae) and P. sojae race 1, respectively. These results indicated that the <span class="gene">GmPRP</span> protein played an important role in the defense of soybean against P. sojae infection.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although the biological and biochemical functions of PR proteins have been studied for several decades, the molecular mechanisms of many PR proteins remain unknown [ <xref rid="pone.0129932.ref008" ref-type="bibr"> 8 </xref> , <xref rid="pone.0129932.ref048" ref-type="bibr"> 48 </xref> , <xref rid="pone.0129932.ref049" ref-type="bibr"> 49 </xref> ]. Thus far, PR proteins have been identified in numerous plants, including hot pepper [ <xref rid="pone.0129932.ref012" ref-type="bibr"> 12 </xref> ], corn [ <xref rid="pone.0129932.ref013" ref-type="bibr"> 13 </xref> ], potato [ <xref rid="pone.0129932.ref050" ref-type="bibr"> 50 </xref> ], wheat [ <xref rid="pone.0129932.ref051" ref-type="bibr"> 51 </xref> ], lily [ <xref rid="pone.0129932.ref052" ref-type="bibr"> 52 </xref> ], rice [ <xref rid="pone.0129932.ref053" ref-type="bibr"> 53 </xref> ], and soybean [ <xref rid="pone.0129932.ref054" ref-type="bibr"> 54 </xref> ]. However, some PR proteins were not grouped into the 17 PR protein families and were simply named pathogenesis-related proteins (PRPs). Phytophthora <span class="trait">root</span> and stem rot of soybean ( <italic> Glycine max </italic> (L.) Merr.), caused by <italic> Phytophthora sojae </italic> Kaufmann and Gerdemann, is a destructive disease throughout the soybean-growing regions worldwide [ <xref rid="pone.0129932.ref055" ref-type="bibr"> 55 </xref> ]. We previously reported, using SSH and cDNA microarrays, that a highly upregulated <italic> PRP </italic> gene (termed <italic> <span class="gene">GmPRP</span> </italic> ) was induced by <italic> P </italic> . <italic> sojae </italic> in the highly resistant soybean cultivar &#x02018;Suinong 10&#x02019; [ <xref rid="pone.0129932.ref056" ref-type="bibr"> 56 </xref> ]. However, no further studies have been conducted to examine the expression, localization and biochemical <span class="trait">activity</span> of this PRP. The objective of the present study was to conduct a functional analysis of <italic> <span class="gene">GmPRP</span> </italic> in the defense against <italic> P </italic> . <italic> sojae </italic> for its possible use as a new tool for the management of Phytophthora <span class="trait">root</span> and stem rot in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						To elucidate the ribonuclease <span class="trait">activity</span> of the <span class="gene">GmPRP</span> protein, RNase <span class="trait">activity</span> assays were performed according to the method described by Bantignies et al. (2000) [ <xref rid="pone.0129932.ref063" ref-type="bibr"> 63 </xref> ] with minor modifications. Briefly, 2 &#x003bc;g, 4 &#x003bc;g, or 6 &#x003bc;g of purified recombinant protein and 10 &#x003bc;g of total RNA extracted from &#x02018;Suinong 10&#x02019; soybean leaves were mixed in separate 20 &#x003bc;L reaction mixtures and incubated at 37&#x000b0;C for 2 h. The amount of solubilized RNA in the supernatant was determined by UV absorbance at 260 nm (OD <sub> 260 </sub> ) with a NanoDrop ND-1000 spectrophotometer (Thermo). The water containing only dissolved RNA was used as a negative control.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate whether the <italic> <span class="gene">GmPRP</span> </italic> gene could enhance <span class="trait">resistance,</span> inoculum preparation and artificial inoculation procedures were modified from those described by Dou et al. (2003) [ <xref rid="pone.0129932.ref067" ref-type="bibr"> 67 </xref> ]. The leaves of transgenic tobacco plants that were tested through qRT-PCR were inoculated with a <italic> P </italic> . <italic> nicotianae </italic> Breda inoculum, and those of the soybean plants that were tested through qRT-PCR and Southern analysis were inoculated with a <italic> P </italic> . <italic> sojae </italic> race 1 inoculum. The leaves were incubated for 72 h in a mist chamber at 25&#x000b0;C and 90% relative humidity under a 14-h photoperiod with light intensity of 350 molm <sup> -2 </sup> s <sup> -1 </sup> . Non-transgenic leaves were used as controls. The disease symptoms on each leaf were observed and photographed using a Canon IXUS 860IS camera at 24, 48 and 72h after the inoculation.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine the antimicrobial <span class="trait">activity</span> effect of the recombinant <span class="gene">GmPRP</span> protein on the growth of <italic> P </italic> . <italic> sojae </italic> 1, filter-paper discs containing recombinant <span class="gene">GmPR10</span> protein (15, 25, 35 &#x003bc;g) were placed near the front of the growing hyphae of <italic> P </italic> . <italic> sojae </italic> 1. After incubation, a 2- to 3-mm zone with inhibited hyphal growth was detected when 25 &#x003bc;g of recombinant protein was applied to the filter-paper discs, and the bacteriostatic effect was enhanced by the application of 35 &#x003bc;g of the protein ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5A </xref> ). A control filter-paper disc containing boiled recombinant protein did not show an inhibitory effect.
					</p>
				
					<p>&nbsp;&nbsp;
						The total RNA isolated from the leaves of &#x02018;Suinong 10&#x02019; soybean was incubated with or without recombinant <span class="gene">GmPRP</span> protein ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5B </xref> ). The elution buffer did not degrade the total RNA ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5B </xref> , lane 1). The control sample, which was incubated with the boiled dialytically renatured <span class="gene">GmPRP</span> protein, did not show significant RNase <span class="trait">activity</span> against soybean RNA ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5B </xref> , lane 2). However, when incubated with the non-boiled dialytically renatured <span class="gene">GmPRP</span> protein, RNase <span class="trait">activity</span> was clearly visible through the migration of the degradation products in the agarose gel ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5B </xref> , lane 3, 4, 5).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmPRP</span> </italic> was overexpressed in tobacco and soybean to evaluate the antimicrobial <span class="trait">activity</span> of this protein <italic> in vivo </italic> . Two transgenic tobacco plants (T2-3 and T2-5) and two transgenic soybean plants (numbered S2-1 and S2-2) were selected to investigate the susceptibility or <span class="trait">resistance</span> to <italic> P </italic> . <italic> nicotianae </italic> and <italic> P </italic> . <italic> sojae </italic> race 1, respectively. After 72 h of incubation with <italic> P </italic> . <italic> nicotianae </italic> or <italic> P </italic> . <italic> sojae </italic> race 1, remarkable differences in the development of disease symptoms were observed between the transgenic and non-transgenic tobacco and soybean plants. After 72 h of incubation with <italic> P </italic> . <italic> nicotianae </italic> , severe symptoms (necrosis and <span class="trait">chlorosis)</span> around the infection areas were observed in non-transgenic tobacco plants ( <xref rid="pone.0129932.g007" ref-type="fig"> Fig 7A </xref> , Lane CK), but the transgenic tobacco plants showed only slight lesions ( <xref rid="pone.0129932.g007" ref-type="fig"> Fig 7A </xref> , Lane T2-3, T2-5). The lesion area of the inoculated CK (1.62 cm <sup> 2 </sup> ) is significantly different from the lesion area of transgenic lines T2-3 and T2-5 (only 1.36 and 0.27 cm <sup> 2 </sup> , respectively) ( <xref rid="pone.0129932.g007" ref-type="fig"> Fig 7B </xref> ). After 72 h of incubation with <italic> P </italic> . <italic> sojae </italic> race 1, the leaves of the non-transgenic soybean plants exhibited clear and large water-soaked lesions compared with those of the transgenic plants ( <xref rid="pone.0129932.g008" ref-type="fig"> Fig 8A </xref> ). The lesion area of the inoculated CK (0.34 cm <sup> 2 </sup> ) is significantly different from the lesion area of transgenic lines T2-3 and T2-5 (only 0.19 and 0.16 cm <sup> 2 </sup> , respectively) ( <xref rid="pone.0129932.g008" ref-type="fig"> Fig 8B </xref> )These results indicate that the overexpression of the <italic> <span class="gene">GmPRP</span> </italic> gene in tobacco and soybean plants improved the <span class="trait">resistance</span> to <italic> P </italic> . <italic> nicotianae </italic> and <italic> P </italic> . <italic> sojae </italic> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						In the interaction between soybean and <italic> P </italic> . <italic> sojae </italic> , certain plant PR proteins are accumulated upon <italic> P </italic> . <italic> sojae </italic> infection, and these proteins may be associated with <italic> P </italic> . <italic> sojae </italic> <span class="trait">resistance.</span> Moy et al. (2004) [ <xref rid="pone.0129932.ref068" ref-type="bibr"> 68 </xref> ] reported that PRa in a sensitive soybean was found to be upregulated at 3 h after infection with <italic> P </italic> . <italic> sojae </italic> and to maintain active expression until the last sampling time (48 h). Narayanan et al. (2009) [ <xref rid="pone.0129932.ref069" ref-type="bibr"> 69 </xref> ] found two defense-related <italic> PR </italic> genes that were upregulated in resistant soybean upon infection by <italic> P </italic> . <italic> sojae </italic> . Xu et al. (2012) [ <xref rid="pone.0129932.ref056" ref-type="bibr"> 56 </xref> ] reported microarray analysis showing four PR protein genes that were upregulated during infection by <italic> P </italic> . <italic> sojae </italic> and then confirmed that result using real-time PCR. Among these PR proteins, <italic> <span class="gene">GmPR10</span> </italic> protein was demonstrated to play an important role in the host defense against <italic> P </italic> . <italic> sojae </italic> infection [ <xref rid="pone.0129932.ref050" ref-type="bibr"> 50 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous study, a novel upregulated cDNA encoding a PRP was screened in the highly resistant soybean cultivar &#x02018;Suinong 10&#x02019; [ <xref rid="pone.0129932.ref056" ref-type="bibr"> 56 </xref> ]. Here, the novel <italic> PRP </italic> gene (termed <italic> <span class="gene">GmPRP</span> </italic> ) and corresponding gene products from soybean ( <italic> Glycine max </italic> ) were isolated and characterized to better understand the function of this protein in the defense against <italic> P </italic> . <italic> sojae </italic> . To the best of our knowledge, this study is the first report on the biological <span class="trait">activity</span> of <span class="gene">GmPRP</span> protein from soybean in the defense against a pathogen. Sequence analysis indicating that <span class="gene">GmPRP</span> contained no signal peptide suggested that it may be an intracellular protein located in the cell membrane and cytoplasm, and this was verified with subcellular localization of the <span class="gene">GmPRP</span> protein in Arabidopsis protoplast cells ( <xref rid="pone.0129932.g004" ref-type="fig"> Fig 4 </xref> ). Subcellular localization of the <span class="gene">GmPRP</span> protein was similar to other intracellular PR proteins, it may be secreted into cell plasma membrane to resist pathogen after being made in the cytoplasm [ <xref rid="pone.0129932.ref070" ref-type="bibr"> 70 </xref> ]. Most of the intracellular PR genes possess introns and exons, and <italic> <span class="gene">GmPRP</span> </italic> also contained introns and exons. The prediction of the three-dimensional (3D) structure of <span class="gene">GmPRP</span> ( <xref rid="pone.0129932.g001" ref-type="fig"> Fig 1B </xref> ) was very similar to those of certain other PR proteins, including <span class="gene">GmPR10</span> [ <xref rid="pone.0129932.ref050" ref-type="bibr"> 50 </xref> ]. However, the predicted <span class="gene">GmPRP</span> structure contained a particularly conserved NTF2-like motif that belongs to the Nuclear transport factor 2 (NTF2) superfamily ( <xref rid="pone.0129932.g001" ref-type="fig"> Fig 1C </xref> ). This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, scytalone dehydratases, and the beta subunit of Ring hydroxylating dioxygenases [ <xref rid="pone.0129932.ref071" ref-type="bibr"> 71 </xref> ]. Some reports have provided direct evidence that NTF2 is required for the nuclear import of RanGDP, which is associated with cell proliferation and gene regulation [ <xref rid="pone.0129932.ref072" ref-type="bibr"> 72 </xref> , <xref rid="pone.0129932.ref073" ref-type="bibr"> 73 </xref> ]. Hence, how <span class="gene">GmPRP</span> protein mediate NTF2 domain against <italic> P </italic> . <italic> sojae </italic> still need further study in the plant defense.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> P </italic> . <italic> sojae </italic> is a soil-borne pathogen that can survive for many years in soil and depends on sporangia and zoospore formation, as well as the direct penetration of hyphae between the cell walls of the epidermis, to infect soybean [ <xref rid="pone.0129932.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0129932.ref077" ref-type="bibr"> 77 </xref> ]. The inhibition of sporangia and zoospore formation or hyphal development will be useful for the host&#x02019;s <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> . In the present study, recombinant <span class="gene">GmPRP</span> protein significantly inhibited the hyphae growth of <italic> P </italic> . <italic> sojae in vitro </italic> ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5A </xref> ), but whether this protein inhibits hyphal development <italic> in vivo </italic> requires further research.
					</p>
				
					<p>&nbsp;&nbsp;
						In the RNA degradation assay, the recombinant <span class="gene">GmPRP</span> protein showed ribonucleolytic <span class="trait">activity,</span> where part of the RNA was degraded within 2 h of incubation ( <xref rid="pone.0129932.g005" ref-type="fig"> Fig 5B </xref> ), indicating that ribonucleolytic <span class="trait">activity</span> may be one of the important roles of this protein in the plant defense response to pathogen attack. Furthermore, the increased expression of <italic> <span class="gene">GmPRP</span> </italic> in transgenic tobacco and soybean plants may contribute to enhanced <span class="trait">resistance</span> against pathogens. In further experiments, <italic> <span class="gene">GmPRP</span> </italic> was successfully transformed into tobacco and soybean plants, and the antimicrobial activities of <italic> <span class="gene">GmPRP</span> </italic> were evaluated through the inoculation of transgenic tobacco and soybean plants overexpressing <italic> <span class="gene">GmPRP</span> </italic> . These transgenic tobacco and soybean plants showed enhanced levels of <span class="trait">resistance</span> to <italic> P </italic> . <italic> nicotianae </italic> and <italic> P </italic> . <italic> sojae </italic> , respectively. These results suggest that the enhanced <span class="trait">resistance</span> to pathogens in tobacco and soybean plants may be associated with the overexpression of <italic> <span class="gene">GmPRP</span> </italic> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean NAC transcription factors promote abiotic stress tolerance and lateral <span class="trait">root</span> formation in transgenic plants.</h2>
			<h3>PMID:21707801  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hao YJ(1), Wei W, Song QX, Chen HW, Zhang YQ, Wang F, Zou HF, Lei G, Tian AG, Zhang WK, Ma B, Zhang JS, Chen SY.
				<br/> 
				年份：2011 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						NAC transcription factors play important roles in plant growth, development and stress responses. Previously, we identified multiple NAC genes in soybean (Glycine max). Here, we identify the roles of two genes, <span class="gene">GmNAC11</span> and <span class="gene">GmNAC20,</span> in stress responses and other processes. The two genes were differentially induced by multiple abiotic stresses and plant hormones, and their transcripts were abundant in roots and cotyledons. Both genes encoded proteins that localized to the nucleus and bound to the core DNA sequence CGT[G/A]. In the protoplast assay system, <span class="gene">GmNAC11</span> acts as a transcriptional activator, whereas <span class="gene">GmNAC20</span> functions as a mild repressor; however, the C-terminal end of <span class="gene">GmANC20</span> has transcriptional activation <span class="trait">activity.</span> Over-expression of <span class="gene">GmNAC20</span> enhances salt and freezing tolerance in transgenic Arabidopsis plants; however, <span class="gene">GmNAC11</span> over-expression only improves salt tolerance. Over-expression of <span class="gene">GmNAC20</span> also promotes lateral <span class="trait">root</span> formation. <span class="gene">GmNAC20</span> may regulate stress tolerance through activation of the DREB/CBF-COR pathway, and may control lateral <span class="trait">root</span> development by altering auxin signaling-related genes. <span class="gene">GmNAC11</span> probably regulates DREB1A and other stress-related genes. The roles of the two <span class="gene">GmNAC</span> genes in stress tolerance were further analyzed in soybean transgenic hairy roots. These results provide a basis for genetic manipulation to improve the agronomic traits of important crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Promoters of orthologous Glycine max and Lotus japonicus nodulation autoregulation genes interchangeably drive phloem-specific expression in transgenic plants.</h2>
			<h3>PMID:17601165  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nontachaiyapoom S(1), Scott PT, Men AE, Kinkema M, Schenk PM, Gresshoff PM.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nodule autoregulation receptor kinase <span class="gene">(GmNARK)</span> of soybean (Glycine max) is essential for the systemic autoregulation of nodulation. Based on quantitative reverse-transcriptase polymerase chain reaction, <span class="gene">GmNARK</span> is ex-pressed to varying levels throughout the plant; the transcript was detected at high levels in mature leaves and roots but to a lesser extent in young leaves, shoot tips, and nodules. The transcript level was not significantly affected by Bradyrhizobium japonicum during the first week following inoculation. In addition, the activities of the promoters of <span class="gene">GmNARK</span> and Lotus japonicus HARI, driving a beta-glucuronidase (GUSPlus) reporter gene, were examined in stably transformed L. japonicus and transgenic hairy roots of soybean. Histochemical GUS <span class="trait">activity</span> in L. japonicus plants carrying either a 1.7-kb <span class="gene">GmNARKpr::GUS</span> or 2.0-kb LjHAR1pr::GUS construct was clearly localized to living cells within vascular bundles, especially phloem cells in leaves, stems, roots, and nodules. Phloem-specific expression also was detected in soybean hairy roots carrying these constructs. Our study suggests that regulatory elements required for the transcription of these orthologous genes are conserved. Moreover, rapid amplification of 5' cDNA ends (5' rapid amplification of cDNA ends) revealed two major transcripts of <span class="gene">GmNARK</span> potentially originating from two TATA boxes. Further analysis of the <span class="gene">GmNARK</span> promoter has confirmed that these two TATA boxes are functional. Deletion analysis also located a region controlling phloem-specific expression to a DNA sequence between 908 bp and 1.7 kb upstream of the translation start site of <span class="gene">GmNARK.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenylalanine ammonia-lyase2.1 contributes to the soybean response towards <italic> Phytophthora sojae </italic> infection</h2>
			<h3>PMID:28775360</h3>
				

				
					<h3>PMCID:5543151</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Chuanzhong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Xin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Junjiang </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qi </surname>  <given-names> Dongyue </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Xiaofei </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ninghui </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <address>  <email> xupengfei@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <address>  <email> zhangshuzhen@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 1136 </institution-id>  <institution-id institution-id-type="GRID"> grid.412243.2 </institution-id>  <institution> Soybean Research Institute, Key Laboratory of Soybean Biology of Chinese Education Ministry,  </institution>  <institution> Northeast Agricultural University,  </institution>  </institution-wrap> Harbin, Heilongjiang China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0482 9043 </institution-id>  <institution-id institution-id-type="GRID"> grid.473328.9 </institution-id>  <institution/>  <institution> Heilongjiang Academy of Land Reclamation Sciences,  </institution>  </institution-wrap> Harbin, Heilongjiang China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1797 9737 </institution-id>  <institution-id institution-id-type="GRID"> grid.412596.d </institution-id>  <institution/>  <institution> First Affiliated Hospital of Harbin Medical University,  </institution>  </institution-wrap> Harbin, Heilongjiang China  </aff>  <aff id="Aff4">  <label> 4 </label> Soybean Research Institute of Heilongjiang Academy of Agricultural Sciences, Key Laboratory of Soybean Cultivation of Ministry of Agriculture P. R. China, Harbin, Heilongjiang China  </aff>  <aff id="Aff5">  <label> 5 </label> Jiamusi Branch Academy of Heilongjiang Academy of Agricultural Sciences, Jiamusi, Heilongjiang China  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem rot of soybean [Glycine max (L.) Merr.] caused by Phytophthora sojae is a destructive disease worldwide. Phenylalanine ammonia-lyase (PAL) is one of the most extensively studied enzymes related to plant responses to biotic and abiotic stresses. However, the molecular mechanism of <span class="gene">PAL</span> in soybean in response to P. sojae is largely unclear. Here, we characterize a novel member of the soybean <span class="gene">PAL</span> gene family, <span class="gene">GmPAL2.1,</span> which is significantly induced by P. sojae. Overexpression and RNA interference analysis demonstrates that <span class="gene">GmPAL2.1</span> enhances <span class="trait">resistance</span> to P. sojae in transgenic soybean plants. In addition, the <span class="gene">PAL</span> <span class="trait">activity</span> in <span class="gene">GmPAL2.1-OX</span> transgenic soybean is significantly higher than that of non-transgenic plants after infection with P. sojae, while that in <span class="gene">GmPAL2.1-RNAi</span> soybean plants is lower. Further analyses show that the daidzein, genistein and salicylic acid (SA) levels and the relative content of glyceollins are markedly increased in <span class="gene">GmPAL2.1-OX</span> transgenic soybean. Taken together, these results suggest the important role of <span class="gene">GmPAL2.1</span> functioning as a positive regulator in the soybean response to P. sojae infection, possibly by enhancing the content of glyceollins, daidzein, genistein and SA.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmPAL2.1</span> enhances <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> in transgenic soybean roots. ( <bold> A </bold> ) Disease symptoms on the roots of the transgenic lines and non-transgenic lines treated with <italic> P </italic> . <italic> sojae </italic> at 7days. ( <bold> B </bold> ) qRT-PCR was used to determine the relative abundance of <italic> <span class="gene">GmPAL2</span> </italic> . <italic> 1 </italic> in three <italic> <span class="gene">GmPAL2</span> </italic> . <italic> 1 </italic> -overexpressing soybean plants (a) and three <italic> <span class="gene">GmPAL2</span> </italic> . <italic> 1 </italic> -RNAi soybean plants (b) Non-transgenic soybean plants were used as controls. For each sample, three biological replicates were analyzed with their respective three technical replicates and statistically analyzed using Student&#x02019;s t-test (*P&#x02009;&#x0003c;&#x02009;0.05, **P&#x02009;&#x0003c;&#x02009;0.01). Bars indicate the standard error of the mean.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmPAL2.1</span> enhances <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> in transgenic soybean cotyledons. ( <bold> A </bold> ) Disease symptoms on the living cotyledons of transgenic lines and non-transgenic lines treated with <italic> P </italic> . <italic> sojae </italic> inoculum at 48 h and 96&#x02009;h. ( <bold> B </bold> ) The relative lesion area of transgenic soybean cotyledon infection with <italic> P </italic> . <italic> sojae </italic> after 96&#x02009;h. The average lesion area of each independent transgenic line (n&#x02009;=&#x02009;3) was calculated, and their relative lesion areas are shown in columns after a comparison with the average lesion area on non-transgenic soybean. ( <bold> C </bold> ) Quantitative real-time PCR analysis of the <italic> P </italic> . <italic> sojae </italic> relative biomass in three <span class="gene">GmPAL2.1-overexpressing</span> soybean plants (a) and three <span class="gene">GmPAL2.1-RNAi</span> soybean plants (b) based on the transcript level of the <italic> P </italic> . <italic> sojae TEF1 </italic> gene. The experiment was performed on three biological replicates with their respective three technical replicates and statistically analyzed using Student&#x02019;s t-test (*P&#x02009;&#x0003c;&#x02009;0.05, **P&#x02009;&#x0003c;&#x02009;0.01). Bars indicate the standard error of the mean.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">PAL</span> <span class="trait">activity</span> in non-transgenic and transgenic soybean leaves treated with <italic> P </italic> . <italic> sojae </italic> inoculum at 36&#x02009;h. The non-transgenic soybean plants were used as controls. The experiment was performed on three biological replicates with their respective three technical replicates and statistically analyzed using Student&#x02019;s t-test (*P&#x02009;&#x0003c;&#x02009;0.05, **P&#x02009;&#x0003c;&#x02009;0.01). Bars indicate the standard error of the mean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The purple acid phosphatase <span class="gene">GmPAP21</span> enhances internal phosphorus utilization and possibly plays a role in symbiosis with rhizobia in soybean.</h2>
			<h3>PMID:27762446  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li C(1), Li C(1), Zhang H(2), Liao H(2), Wang X(1).
				<br/> 
				年份：2017 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Induction of secreted and intracellular purple acid phosphatases (PAPs; EC 3.1.3.2) is widely recognized as an adaptation of plants to phosphorus (P) deficiency. The secretion of PAPs plays important roles in P acquisition. However, little is known about the functions of intracellular <span class="gene">PAP</span> in plants and nodules. In this study, we identified a novel <span class="gene">PAP</span> gene <span class="gene">GmPAP21</span> in soybean. Expression of <span class="gene">GmPAP21</span> was induced by P limitation in nodules, roots and old leaves, and increased in roots with increasing duration of P starvation. Furthermore, the induction of <span class="gene">GmPAP21</span> in nodules and roots was more intensive than in leaves in both P-efficient genotype HN89 and P-inefficient genotype HN112 in response to P starvation, and the relative expression in the leaves and nodules of HN89 was significantly greater than that of HN112 after P deficiency treatment. Further functional analyses showed that over-expressing <span class="gene">GmPAP21</span> significantly enhanced both acid phosphatase <span class="trait">activity</span> and growth performance of hairy roots under P starvation condition, indicating that <span class="gene">GmPAP21</span> plays an important role in P utilization. Moreover, GUS expression driven by <span class="gene">GmPAP21</span> promoter was shown in the nodules besides roots. Overexpression of <span class="gene">GmPAP21</span> in transgenic soybean significantly inhibited nodule growth, and thereby affected plant growth after inoculation with rhizobia. This suggests that <span class="gene">GmPAP21</span> is also possibly involved in regulating P metabolism in nodules. Taken together, our results suggest that <span class="gene">GmPAP21</span> is a novel plant <span class="gene">PAP</span> that functions in the adaptation of soybean to P starvation, possibly through its involvement in P recycling in plants and P metabolism in nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPGIP3 enhanced <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> rot diseases in transgenic wheat.</h2>
			<h3>PMID:25487419  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang A(1), Wei X, Rong W, Dang L, Du LP, Qi L, Xu HJ, Shao Y, Zhang Z.
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Take-all (caused by the fungal pathogen Gaeumannomyces graminis var. tritici, Ggt) and common <span class="trait">root</span> rot (caused by Bipolaris sorokiniana) are devastating <span class="trait">root</span> diseases of wheat (Triticum aestivum L.). Development of resistant wheat cultivars has been a challenge since no resistant wheat accession is available. <span class="gene">GmPGIP3,</span> one member of polygalacturonase-inhibiting protein (PGIP) family in soybean (Glycine max), exhibited inhibition <span class="trait">activity</span> against fungal endopolygalacturonases (PGs) in vitro. In this study, the <span class="gene">GmPGIP3</span> transgenic wheat plants were generated and used to assess the effectiveness of <span class="gene">GmPGIP3</span> in protecting wheat from the infection of Ggt and B. sorokiniana. Four independent transgenic lines were identified by genomic PCR, Southern blot, and reverse transcription PCR (RT-PCR). The introduced <span class="gene">GmPGIP3</span> was integrated into the genomes of these transgenic lines and could be expressed. The expressing <span class="gene">GmPGIP3</span> protein in these transgenic wheat lines could inhibit the PGs produced by Ggt and B. sorokiniana. The disease response assessments postinoculation showed that the <span class="gene">GmPGIP3-expressing</span> transgenic wheat lines displayed significantly enhanced <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> rot diseases caused by the infection of Ggt and B. sorokiniana. These data suggested that <span class="gene">GmPGIP3</span> is an attractive gene resource in improving <span class="trait">resistance</span> to both take-all and common <span class="trait">root</span> rot diseases in wheat.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional Characterization of 14 <italic> Pht1 </italic> Family Genes in Yeast and Their Expressions in Response to Nutrient Starvation in Soybean</h2>
			<h3>PMID:23133521</h3>
				

				
					<h3>PMCID:3485015</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Qin </surname>  <given-names> Lu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Guo </surname>  <given-names> Yongxiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Liyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Ruikang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gu </surname>  <given-names> Mian </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Guohua </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Walk </surname>  <given-names> Thomas </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 9 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Phosphorus (P) is essential for plant growth and development. Phosphate (Pi) transporter genes in the Pht1 family play important roles in Pi uptake and translocation in plants. Although Pht1 family genes have been well studied in model plants, little is known about their functions in soybean, an important legume crop worldwide. PRINCIPAL FINDINGS: We identified and isolated a complete set of 14 Pi transporter genes <span class="gene">(GmPT1-14)</span> in the soybean genome and categorized them into two subfamilies based on phylogenetic analysis. Then, an experiment to elucidate Pi transport <span class="trait">activity</span> of the <span class="gene">GmPTs</span> was carried out using a yeast mutant defective in high-affinity Pi transport. Results showed that 12 of the 14 <span class="gene">GmPTs</span> were able to complement Pi uptake of the yeast mutant with Km values ranging from 25.7 to 116.3 µM, demonstrating that most of the <span class="gene">GmPTs</span> are high-affinity Pi transporters. Further results from qRT-PCR showed that the expressions of the 14 <span class="gene">GmPTs</span> differed not only in response to P availability in different tissues, but also to other nutrient stresses, including N, K and Fe deficiency, suggesting that besides functioning in Pi uptake and translocation, <span class="gene">GmPTs</span> might be involved in synergistic regulation of mineral nutrient homeostasis in soybean. CONCLUSIONS: The comprehensive analysis of Pi transporter function in yeast and expression responses to nutrition starvation of Pht1 family genes in soybean revealed their involvement in other nutrient homeostasis besides P, which could help to better understand the regulation network among ion homeostasis in plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We identified and isolated a complete set of 14 Pi transporter genes ( <italic> <span class="gene">GmPT1-14</span> </italic> ) in the soybean genome and categorized them into two subfamilies based on phylogenetic analysis. Then, an experiment to elucidate Pi transport <span class="trait">activity</span> of the <span class="gene">GmPTs</span> was carried out using a yeast mutant defective in high-affinity Pi transport. Results showed that 12 of the 14 <span class="gene">GmPTs</span> were able to complement Pi uptake of the yeast mutant with <italic> Km </italic> values ranging from 25.7 to 116.3 &#x000b5;M, demonstrating that most of the <span class="gene">GmPTs</span> are high-affinity Pi transporters. Further results from qRT-PCR showed that the expressions of the 14 <italic> <span class="gene">GmPTs</span> </italic> differed not only in response to P availability in different tissues, but also to other nutrient stresses, including N, K and Fe deficiency, suggesting that besides functioning in Pi uptake and translocation, <italic> <span class="gene">GmPTs</span> </italic> might be involved in synergistic regulation of mineral nutrient homeostasis in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Iron is the most studied nutrient element for interactions with P due to their strong precipitation <xref ref-type="bibr" rid="pone.0047726-Dalton1"> [60] </xref> . It is well accepted that P and Fe deficiency have similar effects on the differentiation of epidermal cells, and subsequently affect <span class="trait">root</span> growth, including lateral <span class="trait">root</span> and <span class="trait">root</span> hair formation <xref ref-type="bibr" rid="pone.0047726-LpezBucio1"> [61] </xref> , <xref ref-type="bibr" rid="pone.0047726-Nacry1"> [62] </xref> , <xref ref-type="bibr" rid="pone.0047726-Schmidt1"> [63] </xref> . In Arabidopsis, primary <span class="trait">root</span> growth was inhibited by P deficiency due to Fe toxicity in the <span class="trait">root</span> tip <xref ref-type="bibr" rid="pone.0047726-Ward1"> [64] </xref> . But no P and Fe interactions related to Pi transporters has been reported. Here we found that the expressions of 5 <italic> <span class="gene">GmPTs</span> </italic> highly responded to Fe deficiency ( <xref ref-type="fig" rid="pone-0047726-g006"> Figure 6 </xref> ), and thus provided the first evidence that Pi transporter genes might be involved in interactions between P and Fe signaling.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification and expression of a soybean nodule-enhanced PEP-carboxylase kinase gene (NE-PpcK) that shows striking up-/down-regulation in vivo.</h2>
			<h3>PMID:12753584  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu W(1), Zhou Y, Chollet R.
				<br/> 
				年份：2003 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Various isoforms of plant phosphoenolpyruvate carboxylase (PEPC (Ppc)) are controlled post-translationally by an intricate interaction between allosteric regulation and reversible protein phosphorylation. In leaves and <span class="trait">root</span> nodules of legumes, these changes in PEPC phosphorylation state are governed primarily by PEPC-kinase (PpcK), a novel, 'minimal but functional' Ser/Thr kinase. To date, this plant-specific kinase has been investigated in molecular terms exclusively in non-leguminous plants, such as Crassulacean-acid-metabolism (CAM) species and Arabidopsis. As an important extension of our earlier biochemical studies on this dedicated kinase and PEPC phosphorylation in soybean (Glycine max) nodules, we now report the molecular cloning of the first legume PpcK from a soybean nodule cDNA library, which encodes a functional, 31.0 kDa PpcK polypeptide. Besides displaying organ, developmental, and spatial expression properties that are strikingly up-regulated in mature nodules, the expression pattern of this transcript is distinct from that of a second soybean PpcK isogene <span class="gene">(GmPpcK).</span> The steady-state abundance of this former, nodule-enhanced transcript (NE-PpcK) is markedly influenced by photosynthate supply from the shoots. This latter up-/down-regulation of NE-PpcK transcript level occurs in vivo in concert with the corresponding changes in the nodule PpcK <span class="trait">activity,</span> the phosphorylation-state of PEPC, and the abundance of a previously identified, nodule-enhanced transcript <span class="gene">(GmPEPC7)</span> that encodes the target enzyme (NE-Ppc). Furthermore, genomic Southern analysis and inspection of the public database indicate that there are at least three distinct PpcK and Ppc isogenes in soybean. Collectively, these and recent findings with Arabidopsis implicate the existence of multiple PpcK-Ppc'expression-partners' in plants, exemplified by NE-PpcK and NE-Ppc in the soybean nodule.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mitigation of soil N <sub> 2 </sub> O emission by inoculation with a mixed culture of indigenous <italic> Bradyrhizobium diazoefficiens </italic></h2>
			<h3>PMID:27633524</h3>
				
					<h3>PMCID:5025649</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Akiyama </surname>  <given-names> Hiroko </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n4"> &#x000a7; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoshino </surname>  <given-names> Yuko Takada </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n4"> &#x000a7; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Itakura </surname>  <given-names> Manabu </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  <xref ref-type="author-notes" rid="n4"> &#x000a7; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shimomura </surname>  <given-names> Yumi </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n2"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamamoto </surname>  <given-names> Akinori </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n3"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tago </surname>  <given-names> Kanako </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakajima </surname>  <given-names> Yasuhiro </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Minamisawa </surname>  <given-names> Kiwamu </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hayatsu </surname>  <given-names> Masahito </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Institute for Agro-Environmental Sciences, National Agriculture and Food Research Organization (NARO) </institution> , 3-1-3, Kannondai, Tsukuba, Ibaraki 305-8604,  <country> Japan </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Graduate School of Life Sciences, Tohoku University </institution> , 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577,  <country> Japan </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Advanced Analysis Center, NARO </institution> , 3-1-3, Kannondai, Tsukuba, Ibaraki 305-8604,  <country> Japan </country>  </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 08 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Agricultural soil is the largest source of nitrous oxide (N <sub> 2 </sub> O), a greenhouse gas. Soybean is an important leguminous crop worldwide. Soybean hosts symbiotic nitrogen-fixing soil bacteria (rhizobia) in <span class="trait">root</span> nodules. In soybean ecosystems, N <sub> 2 </sub> O emissions often increase during decomposition of the <span class="trait">root</span> nodules. Our previous study showed that N <sub> 2 </sub> O reductase can be used to mitigate N <sub> 2 </sub> O emission from soybean fields during nodule decomposition by inoculation with <italic> <span class="gene">nosZ</span> </italic> ++ strains [mutants with increased N <sub> 2 </sub> O reductase (N <sub> 2 </sub> OR) <span class="trait">activity]</span> of <italic> Bradyrhizobium diazoefficiens </italic> . Here, we show that N <sub> 2 </sub> O emission can be reduced at the field scale by inoculation with a mixed culture of indigenous <italic> <span class="gene">nosZ</span> </italic> + strains of <italic> B. diazoefficiens </italic> USDA110 group isolated from Japanese agricultural fields. Our results also suggested that nodule nitrogen is the main source of N <sub> 2 </sub> O production during nodule decomposition. Isolating <italic> <span class="gene">nosZ</span> </italic> + strains from local soybean fields would be more applicable and feasible for many soybean-producing countries than generating mutants.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> [L.] Merr.) is one of the most important crops in the world. Soybean is grown on 6% of the world&#x02019;s arable land, and its production has dramatically increased from 26&#x02009;Mt in 1961 to 308&#x02009;Mt in 2014 (ref. <xref ref-type="bibr" rid="b3"> 3 </xref> ). The soybean production area is expected to increase more than that of other crops <xref ref-type="bibr" rid="b4"> 4 </xref> . As a leguminous crop, soybean hosts symbiotic nitrogen-fixing soil bacteria (rhizobia) that can also produce N <sub> 2 </sub> O in <span class="trait">root</span> nodules <xref ref-type="bibr" rid="b5"> 5 </xref> . In soybean ecosystems, increase of N <sub> 2 </sub> O emission during decomposition of the <span class="trait">root</span> nodules has often been reported <xref ref-type="bibr" rid="b6"> 6 </xref> . Organic nitrogen inside the decomposing nodules is mineralized to NH <sub> 4 </sub> <sup> + </sup> followed by nitrification and denitrification that produce N <sub> 2 </sub> O ( <xref ref-type="fig" rid="f1"> Fig. 1a </xref> ) <xref ref-type="bibr" rid="b7"> 7 </xref> <xref ref-type="bibr" rid="b8"> 8 </xref> . N <sub> 2 </sub> O is then emitted into the atmosphere or is further reduced to N <sub> 2 </sub> by N <sub> 2 </sub> O reductase (N <sub> 2 </sub> OR), which is encoded by the <italic> <span class="gene">nosZ</span> </italic> gene. <italic> Bradyrhizobium diazoefficiens </italic> is a nitrogen-fixing rhizobium that also possesses a denitrification pathway <xref ref-type="bibr" rid="b8"> 8 </xref> . Both <italic> B. diazoefficiens <span class="gene">nosZ</span> </italic> + (strains that have the <italic> <span class="gene">nosZ</span> </italic> gene) and <italic> <span class="gene">nosZ</span> </italic> &#x02212; (strains that do not have the <italic> <span class="gene">nosZ</span> </italic> gene) strains are found in soil <xref ref-type="bibr" rid="b9"> 9 </xref> . Denitrification by <italic> <span class="gene">nosZ</span> </italic> &#x02212; strains produces N <sub> 2 </sub> O because they lack <italic> <span class="gene">nosZ</span> </italic> , whereas <italic> <span class="gene">nosZ</span> </italic> + strains can reduce N <sub> 2 </sub> O to N <sub> 2 </sub> ( <xref ref-type="fig" rid="f1"> Fig 1a </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Enhancing microbial N <sub> 2 </sub> OR <span class="trait">activity</span> has been suggested as an N <sub> 2 </sub> O mitigation option <xref ref-type="bibr" rid="b10"> 10 </xref> , and mitigation of N <sub> 2 </sub> O using <italic> <span class="gene">nosZ</span> </italic> has been demonstrated on the laboratory scale. Pure culture and vermiculite pot experiments showed lower N <sub> 2 </sub> O emission by <italic> <span class="gene">nosZ</span> </italic> + strains <xref ref-type="bibr" rid="b11"> 11 </xref> and <italic> <span class="gene">nosZ</span> </italic> ++ strains (mutants with increased N <sub> 2 </sub> OR <span class="trait">activity)</span> <xref ref-type="bibr" rid="b12"> 12 </xref> <xref ref-type="bibr" rid="b13"> 13 </xref> of <italic> B. diazoefficiens </italic> than by <italic> <span class="gene">nosZ</span> </italic> &#x02212; strains. A pot experiment using soil confirmed these results <xref ref-type="bibr" rid="b14"> 14 </xref> . In addition to the use of <italic> B. diazoefficiens </italic> , transgenic plants expressing N <sub> 2 </sub> OR following introduction of soil bacterial <italic> <span class="gene">nosZ</span> </italic> were generated to reduce N <sub> 2 </sub> O emission <xref ref-type="bibr" rid="b15"> 15 </xref> . However, no field-scale study has been reported except our previous study, which showed that N <sub> 2 </sub> OR can be used to mitigate N <sub> 2 </sub> O emission by inoculation with <italic> <span class="gene">nosZ</span> </italic> ++ strains of <italic> B. diazoefficiens </italic> <xref ref-type="bibr" rid="b13"> 13 </xref> . Although it is an effective approach, generating <italic> <span class="gene">nosZ</span> </italic> ++ mutants requires time, cost, and technical skills, and the field use of genetically modified microbes is regulated in many countries. In contrast, isolating indigenous strains from field soil is easy and cost-effective in comparison with generating mutants. Moreover, isolated indigenous strains may be more competitive than mutants with native field strains. Here we report the mitigation of N <sub> 2 </sub> O emission from a soybean field by inoculation with a mixed culture of indigenous <italic> <span class="gene">nosZ</span> </italic> + strains of <italic> B. diazoefficiens </italic> isolated from agricultural fields, without the use of a mutant ( <xref ref-type="fig" rid="f1"> Fig. 1b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We conducted a two-year field experiment to test the effectiveness of C110 inoculation in reducing N <sub> 2 </sub> O emission in an Andosol field dominated by <italic> <span class="gene">nosZ</span> </italic> &#x02212; strains. In our previous study, postharvest N <sub> 2 </sub> O emission was significantly reduced by <italic> <span class="gene">nosZ</span> </italic> ++ (mutants with increased N <sub> 2 </sub> OR <span class="trait">activity)</span> inoculation, whereas the proportion of <italic> <span class="gene">nosZ</span> </italic> ++ nodules in the field experiment was only 23% (ref. <xref ref-type="bibr" rid="b13"> 13 </xref> ). We expected that increasing the proportion of inoculated strains in nodules might reduce more N <sub> 2 </sub> O from decomposition of nodules. In addition to the construction of C110, we improved our germination and inoculation methods to increase the proportion of inoculated strains of nodules. To increase the proportion of cotyledon emergence, soybean seeds were germinated in trays filled with moist vermiculite for one day instead of being seeded in soil-filled pots as in our previous study <xref ref-type="bibr" rid="b13"> 13 </xref> . With this change, the proportion of cotyledon emergence increased from approximately 30% for soil to 95% for vermiculite germination. The low proportion of cotyledon emergence for soil may have been observed because maintaining optimal soil water content for germination is much more difficult for small and water-permeable biodegradable pots filled with soil than for large trays filled with vermiculite. After soybean seeds were germinated in moist vermiculite for a day, they were transferred to biodegradable pots filled with soil. For the pots, soil were collected from a nearby Andosol orchard that showed a lower <italic> nod C </italic> copy number than the Andosol soil of the experimental field ( <xref ref-type="supplementary-material" rid="S1"> Fig. S1 </xref> ), instead of using soil from the experimental field as in Itakura <italic> et al. </italic> <xref ref-type="bibr" rid="b13"> 13 </xref> . Immediately after transfer to the pots, the seeds were inoculated with the mixed culture C110 ( <italic> <span class="gene">nosZ</span> </italic> +) or soil from the experimental field (native). The seedlings were grown for 10 days in a greenhouse and then were transplanted to the Andosol field. As a result of these changes in methods, the proportion of <italic> <span class="gene">nosZ</span> </italic> + nodules in <italic> <span class="gene">nosZ</span> </italic> + inoculated plots in this study were 71.4% to 82.8% from August to October ( <xref ref-type="table" rid="t1"> Table 1 </xref> ), much higher than the 23% for inoculated strain in our previous study <xref ref-type="bibr" rid="b13"> 13 </xref> . Our results also showed that the proportion of <italic> <span class="gene">nosZ</span> </italic> + nodules remained high from vegetative to full maturity stage in <italic> <span class="gene">nosZ</span> </italic> + inoculated plots ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Furthermore, the proportion of <italic> <span class="gene">nosZ</span> </italic> + outside pots in <italic> <span class="gene">nosZ</span> </italic> + inoculated plots were 38&#x02013;68%, significantly higher than that in native plots ( <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.001) on all sampling dates. This result indicated that C110 was able to infect soybean roots outside of pots where native rhizobia populations were high. In addition, C110 was more competitive with native strains than <italic> <span class="gene">nosZ</span> </italic> ++, which showed 0% of inoculated strain outside of the pots in our previous field experiment <xref ref-type="bibr" rid="b13"> 13 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Gene expression analysis showed that <italic> <span class="gene">nosZ</span> </italic> expression was significantly higher in nodules collected from <italic> <span class="gene">nosZ</span> </italic> + inoculated plots than in those from native plots ( <xref ref-type="table" rid="t2"> Table 2 </xref> ), suggesting that N <sub> 2 </sub> OR <span class="trait">activity</span> in <italic> <span class="gene">nosZ</span> </italic> + inoculated plots was higher than that in native plots. In contrast, no significant difference in <italic> <span class="gene">nirK</span> </italic> expression between the two treatments was found ( <xref ref-type="table" rid="t2"> Table 2 </xref> ), suggesting that the denitrification process before N <sub> 2 </sub> O reduction did not differ between the treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						N <sub> 2 </sub> O production rates from soil, <span class="trait">root,</span> and nodule samples collected from the experimental field were measured at different growth stages in 2013 ( <xref ref-type="fig" rid="f2"> Fig. 2a&#x02013;e </xref> ). At the vegetative stage, N <sub> 2 </sub> O was absorbed by nodules in both treatments, and the N <sub> 2 </sub> O uptake rate was significantly higher in the <italic> <span class="gene">nosZ</span> </italic> + treatment than in the native treatment ( <xref ref-type="fig" rid="f2"> Fig. 2a </xref> ; <italic> P </italic> &#x02009;&#x0003c;&#x02009; <italic> 0 </italic> . <italic> 05 </italic> ). Sameshima-Saito <italic> et al. </italic> <xref ref-type="bibr" rid="b11"> 11 </xref> also reported N <sub> 2 </sub> O uptake by nodules from USDA110 ( <italic> <span class="gene">nosZ</span> </italic> +)-inoculated plants, but no N <sub> 2 </sub> O uptake by nodules from <italic> <span class="gene">nosZ</span> </italic> &#x02212;mutant-inoculated plants at the vegetative stage. Nodule N <sub> 2 </sub> O production rates increased dramatically during the nodule decomposition period ( <xref ref-type="fig" rid="f2"> Fig. 2c,d,e </xref> ), and that in the <italic> <span class="gene">nosZ</span> </italic> + treatment was significantly lower than that in the native treatment in the two weeks before harvest ( <xref ref-type="fig" rid="f2"> Fig. 2c </xref> ; <italic> P </italic> &#x02009;&#x0003c;&#x02009; <italic> 0 </italic> . <italic> 05 </italic> ), whereas no differences were found in other periods ( <xref ref-type="fig" rid="f2"> Fig. 2d,e </xref> ). In contrast, N <sub> 2 </sub> O production rates of bulk soil, rhizosphere soil, and <span class="trait">root</span> remained low in all growth stages ( <xref ref-type="fig" rid="f2"> Fig. 2a&#x02013;e </xref> ). In 2014, N <sub> 2 </sub> O production rates were measured two weeks before harvest, and the results confirmed that nodule N <sub> 2 </sub> O production rates were much higher than those of bulk soil, rhizosphere soil, and roots ( <xref ref-type="fig" rid="f2"> Fig. 2f </xref> ). As in 2013, the nodule N <sub> 2 </sub> O production rate of the <italic> <span class="gene">nosZ</span> </italic> + treatment in 2014 was significantly lower than that of the native treatment two weeks before harvest ( <italic> P </italic> &#x02009;&#x0003c;&#x02009; <italic> 0 </italic> . <italic> 05 </italic> ). These results suggested that nodules were the main source of N <sub> 2 </sub> O emission from the soybean field during the nodule decomposition period. Although our previous study <xref ref-type="bibr" rid="b13"> 13 </xref> also suggested the importance of nodules as a N <sub> 2 </sub> O source, N <sub> 2 </sub> O production rates from soil and nodules were not measured in that study. Moreover, a lower nodule N <sub> 2 </sub> O production rate from <italic> <span class="gene">nosZ</span> </italic> + treatment than from the native treatment two weeks before harvest ( <xref ref-type="fig" rid="f2"> Fig. 2d,f </xref> ) suggested that the field-scale reduction of N <sub> 2 </sub> O emission in the <italic> <span class="gene">nosZ</span> </italic> + plot ( <xref ref-type="table" rid="t3"> Table 3 </xref> ) was due to a lower N <sub> 2 </sub> O production rate from the <italic> <span class="gene">nosZ</span> </italic> + nodules.
					</p>
				
					<p>&nbsp;&nbsp;
						In our previous report <xref ref-type="bibr" rid="b13"> 13 </xref> , we showed that inoculation with the <italic> <span class="gene">nosZ</span> </italic> ++ strain of <italic> B. diazoefficiens </italic> significantly decreased postharvest N <sub> 2 </sub> O emission. The <italic> <span class="gene">nosZ</span> </italic> ++ strain used in the field study was a genetically unmodified mutant generated using a proofreading-deficient technique <xref ref-type="bibr" rid="b12"> 12 </xref> . Although it was an effective approach for reducing N <sub> 2 </sub> O emissions from soybean fields, generating <italic> <span class="gene">nosZ</span> </italic> ++ mutants requires more time, cost, and technical skill than isolating indigenous <italic> <span class="gene">nosZ</span> </italic> + strains from soil as in the present study. Also, inoculation of soybean with indigenous strains has a long history and has been practiced commercially in many countries <xref ref-type="bibr" rid="b23"> 23 </xref> , whereas use of mutants, especially genetically modified mutants, may need to receive public acceptance before commercial use. In addition, we used a mixed culture of 63 <italic> <span class="gene">nosZ</span> </italic> + strains of USDA 110 group from agricultural fields from Japan, rather than selecting one strain from the <italic> <span class="gene">nosZ</span> </italic> + collection. The mixture of many strains provides more diversity and is accordingly expected to be more competitive than a single strain with native strains and also more adaptable to various environments and robust to extreme weather, such as <span class="trait">drought,</span> heat, and heavy rainfall. Moreover, using <italic> <span class="gene">nosZ</span> </italic> + strains isolated from local agricultural fields would have little effect on the ecosystem. Thus, isolating <italic> <span class="gene">nosZ</span> </italic> + strains from local soybean fields would be more applicable and feasible for many soybean-producing countries than generating mutants.
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> a </bold> ) Microbial pathway involved in N <sub> 2 </sub> O production from decomposition of <span class="trait">root</span> nodules. During decomposition of nodules, nitrogen becomes available for soil microorganisms. Using this nitrogen, <italic> B. diazoefficiens <span class="gene">nosZ</span> </italic> + strains sequentially reduce nitrogen oxides during denitrification (NO <sub> 3 </sub> <sup> &#x02212; </sup> &#x02009;&#x02192;&#x02009;NO <sub> 2 </sub> <sup> &#x02212; </sup> &#x02009;&#x02192;&#x02009;NO&#x02009;&#x02192;&#x02009;N <sub> 2 </sub> O&#x02009;&#x02192;&#x02009;N <sub> 2 </sub> ), with each step catalyzed by specific reductases encoded by denitrifying genes: <italic> <span class="gene">napA</span> </italic> (periplasmic nitrate reductase), <italic> <span class="gene">nirK</span> </italic> (copper-containing nitrite reductase), <italic> norCB </italic> (nitric oxide reductase), and <italic> <span class="gene">nosZ</span> </italic> (nitrous oxide reductase), respectively. However, denitrification by <italic> B. diazoefficiens <span class="gene">nosZ</span> </italic> &#x02212; strains produces N <sub> 2 </sub> O because they lack the <italic> <span class="gene">nosZ</span> </italic> gene that encodes N <sub> 2 </sub> O reductase (N <sub> 2 </sub> OR). Both <italic> <span class="gene">nosZ</span> </italic> + and <italic> <span class="gene">nosZ</span> </italic> &#x02212; strains are found in the soil. ( <bold> b </bold> ) Design of the experiment. First, soils were collected from 32 agricultural fields throughout Japan. Then, 125 indigenous <italic> <span class="gene">nosZ</span> </italic> + <italic> B. diazoefficiens </italic> strains were isolated (Shina <italic> et al. </italic> ) <xref ref-type="bibr" rid="b9"> 9 </xref> . From these 125 strains, 63 indigenous <italic> <span class="gene">nosZ</span> </italic> + strains of <italic> B. diazoefficiens </italic> USDA110 group were selected (C110), because nitrogen fixation in USDA110 is higher than that in other strains (Itakura <italic> et al. </italic> ) <xref ref-type="bibr" rid="b18"> 18 </xref> . C110 was cultured and inoculated onto soybean seeds in biodegradable pots. For control plots, soybean seeds were inoculated with soil from the experimental field. Soybean seedlings were grown for 10 days in a greenhouse and then transplanted into a <italic> nosZ- </italic> dominant Andosol field. Annual N <sub> 2 </sub> O flux was monitored and mitigation of N <sub> 2 </sub> O production by soybean nodules of inoculated strains was evaluated.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Association of extracellular dNTP utilization with a <span class="gene">GmPAP1-like</span> protein identified in cell wall proteomic analysis of soybean roots.</h2>
			<h3>PMID:29329437</h3>
				

				
			<p>
				作者：Wu W(1), Lin Y(1), Liu P(1)(2), Chen Q(1), Tian J(1), Liang C(1).
				<br/> 
				年份：2018 Jan 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant <span class="trait">root</span> cell walls are dynamic systems that serve as the first plant compartment responsive to soil conditions, such as phosphorus (P) deficiency. To date, evidence for the regulation of <span class="trait">root</span> cell wall proteins (CWPs) by P deficiency remains sparse. In order to gain a better understanding of the roles played by CWPs in the roots of soybean (Glycine max) in adaptation to P deficiency, we conducted an iTRAQ (isobaric tag for relative and absolute quantitation) proteomic analysis. A total of 53 CWPs with differential accumulation in response to P deficiency were identified. Subsequent qRT-PCR analysis correlated the accumulation of 21 of the 27 up-regulated proteins, and eight of the 26 down-regulated proteins with corresponding gene expression patterns in response to P deficiency. One up-regulated CWP, purple acid phosphatase 1-like <span class="gene">(GmPAP1-like),</span> was functionally characterized. Phaseolus vulgaris transgenic hairy roots overexpressing <span class="gene">GmPAP1-like</span> displayed an increase in root-associated acid phosphatase <span class="trait">activity.</span> In addition, relative growth and P content were significantly enhanced in <span class="gene">GmPAP1-like</span> overexpressing lines compared to control lines when deoxy-ribonucleotide triphosphate (dNTP) was applied as the sole external P source. Taken together, the results suggest that the modulation of CWPs may regulate complex changes in the <span class="trait">root</span> system in response to P deficiency, and that the cell wall-localized <span class="gene">GmPAP1-like</span> protein is involved in extracellular dNTP utilization in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The interactome of soybean <span class="gene">GmWRKY53</span> using yeast 2-hybrid library screening to saturation.</h2>
			<h3>PMID:26102586  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tripathi P(1), Rabara RC, Choudhary MK, Miller MA, Huang YS, Shen QJ, Blachon S, Rushton PJ.
				<br/> 
				年份：2015
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="gene">GmWRKY53</span> functions in both biotic and abiotic stress signaling. Using <span class="gene">GmWRKY53</span> as a bait yeast 2-hybrid library screening to saturation isolated multiple independent fragments for many interacting proteins, enabling delineation of minimal interacting domains and computation of a confidence score. Multiple independent clones coding for the LATE ELONGATED HYPOCOTYL clock protein <span class="gene">GmLCL2</span> (MYB114) were isolated and the binding site for <span class="gene">GmWRKY53</span> was mapped to 90 amino acids separate from the MYB domain. This suggests a direct input from the clock on <span class="gene">GmWRKY53</span> <span class="trait">activity.</span> The <span class="gene">GmWRKY53-interacting</span> proteins also included 3 water stress-inducible AP2/ERF transcription factors. One of these <span class="gene">(Glyma03g26310)</span> is one of the most strongly water stress induced genes in soybean roots, suggesting that <span class="gene">GmWRKY53/ERF</span> complexes regulate water stress responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmERF5,</span> a new member of the soybean EAR motif-containing ERF transcription factor, enhances <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:25779701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dong L(1), Cheng Y(1), Wu J(2), Cheng Q(1), Li W(1), Fan S(1), Jiang L(1), Xu Z(3), Kong F(4), Zhang D(3), Xu P(5), Zhang S(5).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora <span class="trait">root</span> and stem rot of soybean [Glycine max (L.) Merr.], caused by Phytophthora sojae Kaufmann and Gerdemann, is a destructive disease throughout the soybean planting regions in the world. Here, we report insights into the function and underlying mechanisms of a novel ethylene response factor (ERF) in soybean, namely <span class="gene">GmERF5,</span> in host responses to P. sojae. <span class="gene">GmERF5-overexpressing</span> transgenic soybean exhibited significantly enhanced <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the PR10, PR1-1, and PR10-1 genes. Sequence analysis suggested that <span class="gene">GmERF5</span> contains an AP2/ERF domain of 58 aa and a conserved ERF-associated amphiphilic repression (EAR) motif in its C-terminal region. Following stress treatments, <span class="gene">GmERF5</span> was significantly induced by P. sojae, ethylene (ET), abscisic acid (ABA), and salicylic acid (SA). The <span class="trait">activity</span> of the <span class="gene">GmERF5</span> promoter <span class="gene">(GmERF5P)</span> was upregulated in tobacco leaves with ET, ABA, Phytophthora nicotianae, salt, and <span class="trait">drought</span> treatments, suggesting that <span class="gene">GmERF5</span> could be involved not only in the induced defence response but also in the ABA-mediated pathway of salt and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> could bind to the GCC-box element and act as a repressor of gene transcription. It was targeted to the nucleus when transiently expressed in Arabidopsis protoplasts. <span class="gene">GmERF5</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> and eukaryotic translation initiation factor <span class="gene">(GmEIF)</span> both in yeast cells and in planta. To the best of our knowledge, <span class="gene">GmERF5</span> is the first soybean EAR motif-containing ERF transcription factor demonstrated to be involved in the response to pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A R2R3-MYB transcription factor, <span class="gene">GmMYB12B2,</span> affects the expression levels of flavonoid biosynthesis genes encoding key enzymes in transgenic Arabidopsis plants.</h2>
			<h3>PMID:24060295  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li XW(1), Li JW, Zhai Y, Zhao Y, Zhao X, Zhang HJ, Su LT, Wang Y, Wang QY.
				<br/> 
				年份：2013 Dec 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavones play diverse roles in plant-microbe interactions and are potentially important for human nutrition and health. To study the regulation of isoflavonoid synthesis in soybean, the R2R3-MYB transcription factor <span class="gene">GmMYB12B2</span> was isolated and characterized. Yeast expression experiments demonstrated that <span class="gene">GmMYB12B2</span> showed transcriptional <span class="trait">activity.</span> <span class="gene">GmMYB12B2</span> was localized in the nucleus when it was transiently expressed in onion epidermal cells. Real-time quantitative PCR analysis revealed that <span class="gene">GmMYB12B2</span> transcription was increased in roots and mature seeds compared with other organs. The gene expression level in immature embryos was consistent with the accumulation of isoflavones. CHS8 is a key enzyme in plant flavonoid biosynthesis. Transient expression experiments in soybean calli demonstrated that CHS8 was regulated by <span class="gene">GmMYB12B2</span> and produced more fluorescence. The expression levels of some key enzymes in flavonoid biosynthesis were examined in transgenic Arabidopsis lines. The results showed that the expression levels of PAL1, <span class="gene">CHS</span> and FLS in transgenic plants were significantly higher than those in wild type plants. However, the expression level of DFR was lower, and the expression levels of CHI, <span class="gene">F3H</span> and <span class="gene">F3'H</span> were the same in all lines. <span class="gene">GmMYB12B2</span> expression caused a constitutive increase in the accumulation of flavonoids in transgenic Arabidopsis lines compared with wild type plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inoculation- and nitrate-induced CLE peptides of soybean control NARK-dependent nodule formation.</h2>
			<h3>PMID:21198362  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reid DE(1), Ferguson BJ, Gresshoff PM.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Systemic autoregulation of nodulation in legumes involves a root-derived signal (Q) that is perceived by a CLAVATA1-like leucine-rich repeat receptor kinase (e.g. <span class="gene">GmNARK).</span> Perception of Q triggers the production of a shoot-derived inhibitor that prevents further nodule development. We have identified three candidate CLE peptide-encoding genes <span class="gene">(GmRIC1,</span> <span class="gene">GmRIC2,</span> and <span class="gene">GmNIC1)</span> in soybean (Glycine max) that respond to Bradyrhizobium japonicum inoculation or nitrate treatment. Ectopic overexpression of all three CLE peptide genes in transgenic roots inhibited nodulation in a <span class="gene">GmNARK-dependent</span> manner. The peptides share a high degree of amino acid similarity in a 12-amino-acid C-terminal domain, deemed to represent the functional ligand of <span class="gene">GmNARK.</span> <span class="gene">GmRIC1</span> was expressed early (12 h) in response to Bradyrhizobium-sp.-produced nodulation factor while <span class="gene">GmRIC2</span> was induced later (48 to 72 h) but was more persistent during later nodule development. Neither <span class="gene">GmRIC1</span> nor <span class="gene">GmRIC2</span> were induced by nitrate. In contrast, <span class="gene">GmNIC1</span> was strongly induced by nitrate (2 mM) treatment but not by Bradyrhizobium sp. inoculation and, unlike the other two <span class="gene">GmCLE</span> peptides, functioned locally to inhibit nodulation. Grafting demonstrated a requirement for <span class="trait">root</span> <span class="gene">GmNARK</span> <span class="trait">activity</span> for nitrate regulation of nodulation whereas Bradyrhizobium sp.-induced regulation was contingent on <span class="gene">GmNARK</span> function in the shoot.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The syntaxin 31-induced gene, LESION SIMULATING DISEASE1 (LSD1), functions in Glycine max defense to the <span class="trait">root</span> parasite Heterodera glycines.</h2>
			<h3>PMID:25530246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pant SR(1), Krishnavajhala A, McNeece BT, Lawrence GW, Klink VP.
				<br/> 
				年份：2015
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments show the membrane fusion genes α soluble NSF attachment protein (α-SNAP) and syntaxin 31 <span class="gene">(Gm-SYP38)</span> contribute to the ability of Glycine max to defend itself from infection by the plant parasitic nematode Heterodera glycines. Accompanying their expression is the transcriptional activation of the defense genes ENHANCED DISEASE SUSCEPTIBILITY1 (EDS1) and NONEXPRESSOR OF PR1 (NPR1) that function in salicylic acid (SA) signaling. These results implicate the added involvement of the antiapoptotic, environmental response gene LESION SIMULATING DISEASE1 (LSD1) in defense. Roots engineered to overexpress the G. max defense genes <span class="gene">Gm-α-SNAP,</span> SYP38, EDS1, NPR1, BOTRYTIS INDUCED KINASE1 (BIK1) and xyloglucan endotransglycosylase/hydrolase (XTH) in the susceptible genotype G. max[Williams 82/PI 518671] have induced <span class="gene">Gm-LSD1</span> <span class="gene">(Gm-LSD1-2)</span> transcriptional <span class="trait">activity.</span> In reciprocal experiments, roots engineered to overexpress <span class="gene">Gm-LSD1-2</span> in the susceptible genotype G. max[Williams 82/PI 518671] have induced levels of SYP38, EDS1, NPR1, BIK1 and XTH, but not α-SNAP prior to infection. In tests examining the role of <span class="gene">Gm-LSD1-2</span> in defense, its overexpression results in ∼52 to 68% reduction in nematode parasitism. In contrast, RNA interference (RNAi) of <span class="gene">Gm-LSD1-2</span> in the resistant genotype G. max[Peking/PI 548402] results in an 3.24-10.42 fold increased ability of H. glycines to parasitize. The results identify that <span class="gene">Gm-LSD1-2</span> functions in the defense response of G. max to H. glycines parasitism. It is proposed that LSD1, as an antiapoptotic protein, may establish an environment whereby the protected, living plant cell could secrete materials in the vicinity of the parasitizing nematode to disarm it. After the targeted incapacitation of the nematode the parasitized cell succumbs to its targeted demise as the infected <span class="trait">root</span> region is becoming fortified.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean Intrinsic Protein Gene, <span class="gene">GmTIP2;3,</span> Involved in Responding to Osmotic Stress.</h2>
			<h3>PMID:26779248</h3>
				

				
			<p>
				作者：Zhang D(1), Tong J(2), He X(1), Xu Z(1), Xu L(1), Wei P(1), Huang Y(1), Brestic M(3), Ma H(1), Shao H(4).
				<br/> 
				年份：2016 Jan 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water is essential for plant growth and development. Water deficiency leads to loss of <span class="trait">yield</span> and decreased crop quality. To understand water transport mechanisms in plants, we cloned and characterized a novel tonoplast intrinsic protein (TIP) gene from soybean with the highest similarity to TIP2-type from other plants, and thus designated <span class="gene">GmTIP2;3.</span> The protein sequence contains two conserved NPA motifs and six transmembrane domains. The expression analysis indicated that this gene was constitutively expressed in all detected tissues, with higher levels in the <span class="trait">root,</span> stem and pod, and the accumulation of <span class="gene">GmTIP2;3</span> transcript showed a significant response to osmotic stresses, including 20% PEG6000 (polyethylene glycol) and 100 μM ABA (abscisic acid) treatments. The promoter-GUS (glucuronidase) <span class="trait">activity</span> analysis suggested that <span class="gene">GmTIP2;3</span> was also expressed in the <span class="trait">root,</span> stem, and leaf, and preferentially expressed in the stele of <span class="trait">root</span> and stem, and the core promoter region was 1000 bp in length, located upstream of the ATG start codon. The GUS tissue and induced expression observations were consistent with the findings in soybean. In addition, subcellular localization showed that <span class="gene">GmTIP2;3</span> was a plasma membrane-localized protein. Yeast heterologous expression revealed that <span class="gene">GmTIP2;3</span> could improve tolerance to osmotic stress in yeast cells. Integrating these results, <span class="gene">GmTIP2;3</span> might play an important role in response to osmotic stress in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A comparison study of Agrobacterium-mediated transformation methods for root-specific promoter analysis in soybean.</h2>
			<h3>PMID:25097075  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li C(1), Zhang H, Wang X, Liao H.
				<br/> 
				年份：2014 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Both in vitro and in vivo hairy <span class="trait">root</span> transformation systems could not replace whole plant transformation for promoter analysis of root-specific and low-P induced genes in soybean. An efficient genetic transformation system is crucial for promoter analysis in plants. Agrobacterium-mediated transformation is the most popular method to produce transgenic hairy roots or plants. In the present study, first, we compared the two different Agrobacterium rhizogenes-mediated hairy <span class="trait">root</span> transformation methods using either constitutive CaMV35S or the promoters of root-preferential genes, <span class="gene">GmEXPB2</span> and <span class="gene">GmPAP21,</span> in soybean, and found the efficiency of in vitro hairy <span class="trait">root</span> transformation was significantly higher than that of in vivo transformation. We compared Agrobacterium rhizogenes-mediated hairy <span class="trait">root</span> and Agrobacterium tumefaciens-mediated whole plant transformation systems. The results showed that low-phosphorous (P) inducible <span class="gene">GmEXPB2</span> and <span class="gene">GmPAP21</span> promoters could not induce the increased expression of the GUS reporter gene under low P stress in both in vivo and in vitro transgenic hairy roots. Conversely, GUS <span class="trait">activity</span> of <span class="gene">GmPAP21</span> promoter was significantly higher at low P than high P in whole plant transformation. Therefore, both in vitro and in vivo hairy <span class="trait">root</span> transformation systems could not replace whole plant transformation for promoter analysis of root-specific and low-P induced genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a pathogenesis-related protein 10 gene <span class="gene">(GmPR10)</span> with induced expression in soybean (Glycine max) during infection with Phytophthora sojae.</h2>
			<h3>PMID:24737571  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xu P(1), Jiang L, Wu J, Li W, Fan S, Zhang S.
				<br/> 
				年份：2014 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In previous study, a cDNA library enriched for mRNAs encoding ESTs that increased in abundance during infection with Phytophthora sojae was constructed by suppression subtractive hybridization from leaf tissues of a high resistant soybean, and an EST homologous to the class 10 of pathogenesis-related (PR) proteins was identified to be up-regulated by microarray and real-time PCR. Here, the full-length cDNA (termed <span class="gene">GmPR10,</span> GenBank accession number FJ960440; ADC31789.1) of the EST was isolated by rapid amplification of cDNA ends, and contains an open reading frame of 474 bp. The <span class="gene">GmPR10</span> protein included a "P-loop'' motif. The constitutive transcript abundance of <span class="gene">GmPR10</span> in soybean was the highest in leaves, followed by roots and stems. Further analysis showed that <span class="gene">GmPR10</span> mRNA abundance was increased during infection with P. sojae following leaf treatments with gibberellin (GA3), hydrogen peroxide (H2O2), salicylic acid (SA), and abscisic acid (ABA). The dialytically renatured <span class="gene">GmPR10</span> protein significantly inhibited P. sojae hyphal growth and exhibited RNase <span class="trait">activity.</span> Transgenic tobacco and soybean plants overexpressing <span class="gene">GmPR10</span> showed increased <span class="trait">resistance</span> to P. nicotianae Breda and P. sojae, respectively. These results suggest that the <span class="gene">GmPR10</span> protein plays an important role in host defense against P. sojae infection. To the best of our knowledge, this is the first report on the functional characterization of a PR10 protein from soybean in defense against P. sojae.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rice ENOD40: isolation and expression analysis in rice and transgenic soybean <span class="trait">root</span> nodules.</h2>
			<h3>PMID:10363365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kouchi H(1), Takane K, So RB, Ladha JK, Reddy PM.
				<br/> 
				年份：1999 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The early nodulin <span class="gene">ENOD40</span> has been proposed as playing a pivotal role in the organogenesis of legume <span class="trait">root</span> nodules. We have isolated the <span class="gene">ENOD40</span> gene homologues <span class="gene">ObENOD40</span> and <span class="gene">OsENOD40</span> from the wild and cultivated rice genotypes Oryza brachyantha and Oryza sativa, respectively. Rice ENOD40s contain a sequence at the 5' end (region I) for encoding an oligopeptide that is highly conserved in all legume ENOD40s. Furthermore, at the 3' end (region II), the nucleotide sequence of rice ENOD40s exhibited a considerable homology to the corresponding region in legume ENOD40s. Among various organs of the rice plant, expression of <span class="gene">OsENOD40</span> was detected only in stems. In situ hybridization studies revealed that, within the stem, transcription of <span class="gene">OsENOD40</span> is confined to parenchyma cells surrounding the protoxylem during the early stages of development of lateral vascular bundles that conjoin an emerging leaf. Expression pattern of <span class="gene">OsENOD40</span> promoter-GUS fusion in nodules developed on transgenic hairy roots of soybean was also found to be restricted to peripheral cells of nodule vascular bundles, thus evidencing that rice <span class="gene">ENOD40</span> promoter <span class="trait">activity</span> is essentially the same as that of soybean ENOD40. Taken together, these results strongly suggest that <span class="gene">OsENOD40</span> and legume ENOD40s share common, if not identical, functions in differentiation and/or function of vascular bundles.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Roles of plant homologs of Rab1p and <span class="gene">Rab7p</span> in the biogenesis of the peribacteroid membrane, a subcellular compartment formed de novo during <span class="trait">root</span> nodule symbiosis.</h2>
			<h3>PMID:8223429  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cheon CI(1), Lee NG, Siddique AB, Bal AK, Verma DP.
				<br/> 
				年份：1993 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The peribacteroid membrane (PBM) in legume <span class="trait">root</span> nodules is derived from plasma membrane following endocytosis of Rhizobium by fusion of newly synthesized vesicles. We studied the roles of plant Rab1p and <span class="gene">Rab7p</span> homologs, the small GTP-binding proteins involved in vesicular transport, in the biogenesis of the PBM. Three cDNAs encoding legume homologs of mammalian Rab1p and <span class="gene">Rab7p</span> were isolated from soybean (sRab1p, sRab7p) and Vigna aconitifolia (vRab7p). sRab1p was confirmed to be a functional counterpart of yeast Ypt1p (Rab1p) by complementation of a yeast ypt1-1 mutant. Both srab1 and vrab7 genes are induced during nodulation with the level of vrab7 mRNA being 12 times higher than that in <span class="trait">root</span> meristem and leaves. This induction directly correlates with membrane proliferation in nodules. Antisense constructs of srab1 and vrab7, under a nodule-specific promoter (leghemoglobin, Lbc3), were made in a binary vector and transgenic nodules were developed on soybean hairy roots obtained through Agrobacterium rhizogenes-mediated transformation. Both antisense srab1 and vrab7 nodules were smaller in size and showed lower nitrogenase <span class="trait">activity</span> than controls. The antisense srab1 nodules showed lack of expansion of infected cells, fewer bacteroids per cell and their frequent release into vacuoles. In contrast, antisense vrab7 expressing nodules showed accumulation of late endosomal structure and multivesicular bodies in the perinuclear region. These data suggest that both Rab1p and <span class="gene">Rab7p</span> are essential for the development of the PBM compartment in effective symbiosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Small RNA profiles in soybean primary <span class="trait">root</span> tips under water deficit</h2>
			<h3>PMID:28105955</h3>
				
					<h3>PMCID:5249032</h3>
					<h4>期刊:BMC Systems Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Yun </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hivrale </surname>  <given-names> Vandana </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaotuo </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lelandais-Bri&#x000e8;re </surname>  <given-names> Christine </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farmer </surname>  <given-names> Andrew D. </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> May </surname>  <given-names> Gregory D. </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Crespi </surname>  <given-names> Martin </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> NguyenHenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Sunkar </surname>  <given-names> Ramanjulu </given-names>  </name>  <address>  <email> ramanjulu.sunkar@okstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 8571 108X </institution-id>  <institution-id institution-id-type="GRID"> grid.218292.2 </institution-id>  <institution> Faculty of Information Engineering and Automation,  </institution>  <institution> Kunming University of Science and Technology,  </institution>  </institution-wrap> Kunming, Yunnan 650500 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0721 7331 </institution-id>  <institution-id institution-id-type="GRID"> grid.65519.3e </institution-id>  <institution> Department of Biochemistry and Molecular Biology,  </institution>  <institution> Oklahoma State University,  </institution>  </institution-wrap> Stillwater, OK 74078 USA  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2162 3504 </institution-id>  <institution-id institution-id-type="GRID"> grid.134936.a </institution-id>  <institution> National Center for Soybean Biotechnology and Division of Plant Sciences,  </institution>  <institution> University of Missouri,  </institution>  </institution-wrap> Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2171 2558 </institution-id>  <institution-id institution-id-type="GRID"> grid.5842.b </institution-id>  <institution> Institut of Plant Sciences Paris-Saclay (IPS2), CNRS, INRA,  </institution>  <institution> University of &#x0201c;Paris-Sud&#x0201d;,  </institution>  </institution-wrap> Batiment 630, 91405 Orsay, France  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1788 6194 </institution-id>  <institution-id institution-id-type="GRID"> grid.469994.f </institution-id>  <institution> Institut of Plant Sciences Paris-Saclay (IPS2), CNRS, INRA,  </institution>  <institution> University of &#x0201c;Paris-Diderot&#x0201d;, Sorbonne Paris-Cit&#x000e9;, 91405 Orsay,,  </institution>  </institution-wrap> Paris, France  </aff>  <aff id="Aff6">  <label> 6 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2219 756X </institution-id>  <institution-id institution-id-type="GRID"> grid.419253.8 </institution-id>  <institution/>  <institution> National Center for Genome Resources,  </institution>  </institution-wrap> Santa Fe, New Mexico, NM 87505 USA  </aff>  <aff id="Aff7">  <label> 7 </label> Present address: Pioneer Hi-Bred International, Inc, Johnston, IA 50131 USA  </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs have emerged as critical regulators of gene expression which is important for normal growth and development of leaves, roots, flowers and other morphological traits in plants. Primary <span class="trait">root</span> apex is critical for <span class="trait">root</span> growth and development under normal conditions. To assess the importance of miRNAs in this <span class="trait">root</span> zone of soybean, we measured their levels. Indeed, the abundances of different miRNA families varied greatly in the primary <span class="trait">root</span> tips. On the basis of their normalized abundances, miRNA families were divided into high (100,000 RPTM and above), moderate (10,000 to 100,000 RPTM), low (1000 to 10,000 RPTM) and extremely low (1 to 1000 RPTM) abundant families. Of the 171 miRNA families, five (miR156, miR166, miR1507, miR1509 and miR1510) were categorized as highly abundant. Of these, miR1507 is the most abundant in primary <span class="trait">root</span> tips as its expression levels were as high as almost 800,000 RPTM accounting for almost one-twelfth of the total small RNAs sequenced in each of the four libraries. A very high abundance of these 5 miRNA families in this region indicates important gene regulatory roles for these miRNAs in primary <span class="trait">root</span> tips. miR156 regulates several <span class="gene">SBP</span> transcription factors and miR166 targets HD-ZIP transcription factors whereas miR1507, miR1509 and miR1510 largely found in legumes so far.
					</p>
				
					<p>&nbsp;&nbsp;
						By regulating the expression of <span class="gene">SBP</span> transcription factors in leaves, miR156 is known to regulate plant progression into reproductive phase from vegetative phase [ <xref ref-type="bibr" rid="CR25"> 25 </xref> &#x02013; <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. Besides in leaves, miR156 is abundantly expressed in primary <span class="trait">root</span> tips (this study) and the nodules [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ] suggesting a much broader role for miR156 family in plants. In different plant species, miR156 is largely upregulated under <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. Most recently, Stief et al. [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ] demonstrated that miR156 plays an important role in heat tolerance of Arabidopsis. Even in their study, only 3 variants (MIR156h, d and c) of miR156 family were found to respond to heat suggesting that the overall miR156 regulation would only be marginal and yet sufficient to make Arabidopsis plants to adapt to the recurrent heat by affecting the <span class="gene">SBP</span> factors expression [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. Of the seven miR156 variants we identified in the primary <span class="trait">root</span> tips, four are abundantly expressed and were unaffected by the treatment, while two variants that are expressed at low levels (miR156b/f and miR156r) are highly upregulated under stress (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ). This observation suggests a role for specific variants of miR156 under water stress, probably in specific cell types within the primary <span class="trait">root</span> tip.
					</p>
				
					<p>&nbsp;&nbsp;
						Being negative regulators of their target mRNAs, downregulation of a miRNA under stress could result in upregulation of the targets and the opposite is true for the upregulated miRNAs [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. The regulation of transcription factors was studied under water stress conditions in five-centimeter long <span class="trait">root</span> primary roots of soybean [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Most importantly, the study found the transcription factors (two CBF-B/NF-YA, a MYB33 gene and TEM1, TEMPRANILLO 1) [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ] that are targets of miRNAs were differentially regulated under <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Our miRNA analysis revealed that the miRNAs targeting above mentioned transcription factors were differentially regulated; miR482 was upregulated while miR169 was downregulated below the stringent threshold level of(log2) fold change&#x000a0;(Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> ). Our results are consistent with the reported upregulation of two NF-YA genes <span class="gene">(Glyma02g47380</span> and <span class="gene">Glyma10g10240)</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ] that are targeted by miR169 and decreased expression of TEM1 transcription factor [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ] that is targeted by miR482. These results demonstrate that the miRNA regulations affected their target genes under stress in the soybean primary <span class="trait">root</span> tips.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean primary roots, 37 transcripts associated with auxin synthesis and signaling were regulated by <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Of these, 17 are auxin response factors that include targets of miR167 <span class="gene">(Glyma16g02760.1)</span> and TAS3siRNAs <span class="gene">(Glyma06g00950.1),</span> which are downregulated especially under mild and severe stress conditions [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. The downregulation of these 2 ARFs <span class="gene">(Glyma16g02760.1</span> and <span class="gene">Glyma06g00950.1)</span> in primary <span class="trait">root</span> tips is consistent with the observed upregulation of miR167, which is induced by almost 2 fold (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). The moderate upregulation of miR390 under water deficit stress could lead to the enhanced accumulation of TAS3 siRNAs as observed in the present study (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> ), which in turn decreases the accumulation of ARFs (ARF2, ARF3 and ARF4). Similarly, the levels of miR393 were also elevated under stress and miR393 targets TIR1, an auxin receptor in plants. The elevated miR393 could decrease TIR1 levels under stress and this will results in decreased ubiquitination of AUX/IAA factors that act as transcriptional repressors. This is consistent with the hypothesis that miRNAs play a critical role in auxin-dependent growth modulation under stress [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. Overall, three miRNA families that are associated (miR167, miR393 and miR390 and TAS3 siRNAs all displayed either high or moderate upregulation) with auxin sensing and signalling are upregulated in response to water stress. Such a coordinated differential regulation of miRNAs and TAS3siRNAs that target auxin sensing and signaling suggests a more compelling role for these small RNAs under stress. This is somewhat consistent with the fact that the decelerated growth rate in the Region 2 of the primary <span class="trait">root</span> tips under <span class="trait">drought</span> could be mediated by three miRNAs for conserving water and energy.
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis, Cu/Zn SOD upregulation in response to oxidative stress is exclusively dependent on the downregulation of miR398 [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. miR398 in plants controls Cu/Zn SOD protein abundances not only at the posttranscriptional level by targeting Cu/Zn SOD transcripts but also at the posttranslational level by targeting the <span class="gene">CCS</span> gene, that supplies Cu to the Cu/Zn SODs in plants [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. Indeed miR398 levels were down-regulated under water stress in primary <span class="trait">root</span> tips of soybean (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). These results are consistent with previous reports that the protective proteins associated with the ROS toxicity were upregulated in this region of soybean primary roots exposed to water stress [ <xref ref-type="bibr" rid="CR10"> 10 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors (MYB TFs) play diverse roles in plant development and stress responses. However, the mechanisms underlying the actions of MYB TFs during stress response remain unclear. In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance.</span> Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> salt stress, H2O2 and ABA. Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary <span class="trait">root</span> length, greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content. We also found that ROS could induce SOD/POD/CAT <span class="trait">activity</span> in OE lines. In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary <span class="trait">root</span> elongation. Some ROS-related genes were up-regulated under abiotic stress in <span class="gene">GmMYB84</span> transgenic plants compared with the WT. Furthermore, electrophoretic mobility shift assay and luciferase reporter analysis demonstrated that <span class="gene">GmMYB84</span> binds directly to the promoter of <span class="gene">GmRBOHB-1</span> and <span class="gene">GmRBOHB-2</span> genes. Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control <span class="trait">root</span> growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of duplicated Symbiosis Receptor Kinase (SymRK) genes during nodulation and mycorrhizal infection in soybean (Glycine max).</h2>
			<h3>PMID:25617765  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Indrasumunar A(1), Wilde J(1), Hayashi S(1), Li D(1), Gresshoff PM(2).
				<br/> 
				年份：2015 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Association between legumes and rhizobia results in the formation of <span class="trait">root</span> nodules, where symbiotic nitrogen fixation occurs. The early stages of this association involve a complex of signalling events between the host and microsymbiont. Several genes dealing with early signal transduction have been cloned, and one of them encodes the leucine-rich repeat (LRR) receptor kinase (SymRK; also termed NORK). The Symbiosis Receptor Kinase gene is required by legumes to establish a <span class="trait">root</span> endosymbiosis with Rhizobium bacteria as well as mycorrhizal fungi. Using degenerate primer and BAC sequencing, we cloned duplicated SymRK homeologues in soybean called <span class="gene">GmSymRKα</span> and <span class="gene">GmSymRKβ.</span> These duplicated genes have high similarity of nucleotide (96%) and amino acid sequence (95%). Sequence analysis predicted a malectin-like domain within the extracellular domain of both genes. Several putative cis-acting elements were found in promoter regions of <span class="gene">GmSymRKα</span> and <span class="gene">GmSymRKβ,</span> suggesting a participation in lateral <span class="trait">root</span> development, cell division and peribacteroid membrane formation. The mutant of SymRK genes is not available in soybean; therefore, to know the functions of these genes, RNA interference (RNAi) of these duplicated genes was performed. For this purpose, RNAi construct of each gene was generated and introduced into the soybean genome by Agrobacterium rhizogenes-mediated hairy <span class="trait">root</span> transformation. RNAi of <span class="gene">GmSymRKβ</span> gene resulted in an increased reduction of nodulation and mycorrhizal infection than RNAi of <span class="gene">GmSymRKα,</span> suggesting it has the major <span class="trait">activity</span> of the duplicated gene pair. The results from the important crop legume soybean confirm the joint phenotypic action of <span class="gene">GmSymRK</span> genes in both mycorrhizal and rhizobial infection seen in model legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Deletions of the <span class="gene">SACPD-C</span> locus elevate seed stearic acid levels but also result in fatty acid and morphological alterations in nitrogen fixing nodules.</h2>
			<h3>PMID:24886084  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gillman JD(1), Stacey MG, Cui Y, Berg HR, Stacey G.
				<br/> 
				年份：2014 May 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max) seeds are the primary source of edible oil in the United States. Despite its widespread utility, soybean oil is oxidatively unstable. Until recently, the majority of soybean oil underwent chemical hydrogenation, a process which also generates trans fats. An alternative to chemical hydrogenation is genetic modification of seed oil through identification and introgression of mutant alleles. One target for improvement is the elevation of a saturated fat with no negative cardiovascular impacts, stearic acid, which typically constitutes a minute portion of seed oil (~3%). RESULTS: We examined radiation induced soybean mutants with moderately increased stearic acid (10-15% of seed oil, ~3-5 X the levels in wild-type soybean seeds) via comparative whole genome hybridization and genetic analysis. The deletion of one <span class="gene">SACPD</span> isoform encoding gene (SACPD-C) was perfectly correlated with moderate elevation of seed stearic acid content. However, <span class="gene">SACPD-C</span> deletion lines were also found to have altered nodule fatty acid composition and grossly altered morphology. Despite these defects, overall nodule accumulation and nitrogen fixation were unaffected, at least under laboratory conditions. CONCLUSIONS: Although no <span class="trait">yield</span> penalty has been reported for moderate elevated seed stearic acid content in soybean seeds, our results demonstrate that genetic alteration of seed traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and <span class="gene">SACPD</span> <span class="trait">activity</span> in particular, in the establishment and maintenance of symbiotic nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A UDP-glucose:isoflavone 7-O-glucosyltransferase from the roots of soybean (glycine max) seedlings. Purification, gene cloning, phylogenetics, and an implication for an alternative strategy of enzyme catalysis.</h2>
			<h3>PMID:17565994  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Noguchi A(1), Saito A, Homma Y, Nakao M, Sasaki N, Nishino T, Takahashi S, Nakayama T.
				<br/> 
				年份：2007 Aug 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavones, a class of flavonoids, play very important roles in plant-microbe interactions in certain legumes such as soybeans (Glycine max L. Merr.). G. max UDP-glucose:isoflavone 7-O-glucosyltransferase <span class="gene">(GmIF7GT)</span> is a key enzyme in the synthesis of isoflavone conjugates, which accumulate in large amounts in vacuoles and serve as an isoflavonoid pool that allows for interaction with microorganisms. In this study, the 14,000-fold purification of <span class="gene">GmIF7GT</span> from the roots of G. max seedlings was accomplished. The purified enzyme is a monomeric protein of 46 kDa, catalyzing regiospecific glucosyl transfer from UDP-glucose to isoflavones to produce isoflavone 7-O-beta-D-glucosides (k(cat) = 0.74 s(-1), K(m) for genistein = 3.6 microM, and K(m) for UDP-glucose = 190 microM). The <span class="gene">GmIF7GT</span> cDNA was isolated based on the amino acid sequence of the purified enzyme. Phylogenetic analysis showed that <span class="gene">GmIF7GT</span> is a novel member of glycosyltransferase family 1 and is distantly related to Glycyrrhiza echinata UDP-glucose:isoflavonoid 7-O-glucosyltransferase. The purified enzyme was unexpectedly devoid of the N-terminal 49-residue segment and thus lacks the histidine residue corresponding to the proposed catalytic residue of glycosyltransferases from Medicago truncatula (UGT71G1) and Vitis vinifera (VvGT1). The results of kinetic studies of site-directed mutants of <span class="gene">GmIF7GT</span> showed that both His-15 and Asp-125, which correspond to the catalytic residues of UGT71G1 and VvGT1, are not important for <span class="gene">GmIF7GT</span> <span class="trait">activity.</span> The results also suggest that an acidic residue at position 392 is very important for primary catalysis of <span class="gene">GmIF7GT.</span> These results led to the proposal that <span class="gene">GmIF7GT</span> utilizes a strategy of catalysis that is distinct from those proposed for UGT71G1 and VvGT1.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Modification of phenolic metabolism in soybean hairy roots through down regulation of chalcone synthase or isoflavone synthase.</h2>
			<h3>PMID:16924535  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lozovaya VV(1), Lygin AV, Zernova OV, Ulanov AV, Li S, Hartman GL, Widholm JM.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean hairy roots, transformed with the soybean chalcone synthase (CHS6) or isoflavone synthase (IFS2) genes, with dramatically decreased capacity to synthesize isoflavones were produced to determine what effects these changes would have on susceptibility to a fungal pathogen. The isoflavone and coumestrol concentrations were decreased by about 90% in most lines apparently due to gene silencing. The <span class="gene">IFS2</span> transformed lines had very low <span class="gene">IFS</span> enzyme <span class="trait">activity</span> in microsomal fractions as measured by the conversion of naringenin to genistein. The <span class="gene">CHS6</span> lines with decreased isoflavone concentrations had 5 to 20-fold lower <span class="gene">CHS</span> enzyme activities than the appropriate controls. Both <span class="gene">IFS2</span> and <span class="gene">CHS</span> transformed lines accumulated higher concentrations of both soluble and cell wall bound phenolic acids compared to controls with higher levels found in the <span class="gene">CHS6</span> lines indicating alterations in the lignin biosynthetic branch of the pathway. Induction of the soybean phytoalexin glyceollin, of which the precursor is the isoflavone daidzein, by the fungal pathogen Fusarium solani f. sp. glycines (FSG) that causes soybean sudden death syndrome (SDS) showed that the low isoflavone transformed lines did not accumulate glyceollin while the control lines did. The (iso)liquritigenin content increased upon FSG induction in the <span class="gene">IFS2</span> transformed roots indicating that the pathway reactions before this point can control isoflavonoid synthesis. The lowest fungal growth rate on hairy roots was found on the FSG partially resistant control roots followed by the SDS sensitive control roots and the low isoflavone transformants. The results indicate the importance of phytoalexin synthesis in <span class="trait">root</span> <span class="trait">resistance</span> to the pathogen.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean nodule sucrose synthase (nodulin-100): further analysis of its phosphorylation using recombinant and authentic root-nodule enzymes.</h2>
			<h3>PMID:10525291  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang XQ(1), Lund AA, Sarath G, Cerny RL, Roberts DM, Chollet R.
				<br/> 
				年份：1999 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS) is a known phosphoserine-containing enzyme in legume <span class="trait">root</span> nodules and various other plant "sink" tissues. In order to begin to investigate the possible physiological significance of this posttranslational modification, we have cloned a full-length soybean nodule <span class="gene">SS</span> (nodulin-100) cDNA and overexpressed it in Escherichia coli. Authentic nodule <span class="gene">SS</span> and recombinant wild-type and mutant forms of the enzyme were purified and characterized. We document that a conserved serine near the N-terminus (Ser(11)) is the primary phosphorylation site for a nodule Ca(2+)-dependent protein kinase (CDPK) in vitro. Related tryptic digestion and mass spectral analyses indicated that this target residue was also phosphorylated in planta in authentic nodulin-100. In addition, a secondary phosphorylation site(s) in recombinant nodule <span class="gene">SS</span> was implicated given that all active mutant enzyme forms (S11A, S11D, S11C, and N-terminal truncation between Ala(2) and Arg(13)) were phosphorylated, albeit weakly, by the CDPK. This secondary site(s) likely resides between Glu(14) and Met(193) as evidenced by CNBr cleavage and phosphopeptide mapping. Phosphorylation of the recombinant and authentic nodule Ser(11) enzymes in vitro by the nodule <span class="gene">CDPK</span> had no major effect on the sucrose-cleavage <span class="trait">activity</span> and/or kinetic properties. However, phosphorylation decreased the apparent surface hydrophobicity of the recombinant wild-type enzyme, suggesting that this covalent modification could potentially play some role in the documented partitioning of nodulin-100 between the nodule symbiosome/plasma membranes and cytosol in planta.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 1999 Academic Press.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A deletion mutation at the ep locus causes low seed coat peroxidase <span class="trait">activity</span> in soybean.</h2>
			<h3>PMID:9418041  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gijzen M(1).
				<br/> 
				年份：1997 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">Ep</span> locus severely affects the amount of peroxidase enzyme in soybean seed coats. Plants containing the dominant <span class="gene">Ep</span> allele accumulate large amounts of peroxidase in the hourglass cells of the sub-epidermis. Homozygous recessive epep genotypes do not accumulate peroxidase in the hourglass cells and are much reduced in total seed coat peroxidase <span class="trait">activity.</span> To isolate the gene encoding the seed coat peroxidase and to determine whether it corresponds to the <span class="gene">Ep</span> locus, a cDNA library was constructed from developing seed coats and an abundant 1.3 kb peroxidase transcript was cloned. The corresponding structural gene was also isolated from a genomic library. Sequence analysis shows that the seed coat peroxidase is translated as a 352 amino acid precursor protein of 38 kDa. Processing of a putative 26 amino acid signal sequence results in a mature protein of 326 residues with a calculated mass of 35 kDa and a pl of 4.4. Using probes derived from the cDNA, genomic DNA blot hybridization and polymerase chain reaction analysis detected polymorphisms that distinguished EpEp and epep genotypes. Co-segregation of the polymorphisms in an F2 population from a cross of EpEp and epep plants shows that the <span class="gene">Ep</span> locus encodes the seed coat peroxidase protein. Comparison of <span class="gene">Ep</span> and ep alleles indicates that the recessive gene lacks 87 bp of sequence encompassing the translation start codon. Analysis by RNA blot hybridization shows that epep plants have drastically reduced amounts of peroxidase transcript compared with EpEp plants. The peroxidase mRNA is abundant in seed coat tissues of EpEp plants during the late stages of seed maturation, and could also be detected in <span class="trait">root</span> tissues, but not in the flower, embryo, pod or leaf. The results indicate that the lack of peroxidase accumulation in seed coats of homozygous recessive epep plants is due to a mutation of the structural gene that reduces transcript abundance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bradyrhizobium japonicum delta-aminolevulinic acid dehydratase is essential for symbiosis with soybean and contains a novel metal-binding domain.</h2>
			<h3>PMID:8226669  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chauhan S(1), O'Brian MR.
				<br/> 
				年份：1993 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Bradyrhizobium japonicum <span class="gene">hemA</span> gene product delta-aminolevulinic acid (ALA) synthase is not required for symbiosis of that bacterium with soybean. Hence, the essentiality of the subsequent heme synthesis enzyme, ALA dehydratase, was examined. The B. japonicum ALA dehydratase gene, termed hemB, was isolated and identified on the basis of its ability to confer hemin prototrophy and enzyme <span class="trait">activity</span> on an Escherichia coli <span class="gene">hemB</span> mutant, and it encoded a protein that was highly homologous to ALA dehydratases from diverse organisms. A novel metal-binding domain in the B. japonicum ALA dehydratase was identified that is a structural composite of the Mg(2+)-binding domain found in plant ALA dehydratases and the Zn(2+)-binding region of nonplant ALA dehydratases. Enzyme <span class="trait">activity</span> in dialyzed extracts of cells that overexpressed the <span class="gene">hemB</span> gene was reconstituted by the addition of Mg2+ but not by addition of Zn2+, indicating that the B. japonicum ALA dehydratase is similar to the plant enzymes with respect to its metal requirement. Unlike the B. japonicum <span class="gene">hemA</span> mutant, the <span class="gene">hemB</span> mutant strain KP32 elicited undeveloped nodules on soybean, indicated by the lack of nitrogen fixation <span class="trait">activity</span> and plant hemoglobin. We conclude that the <span class="gene">hemB</span> gene is required for nodule development and propose that B. japonicum ALA dehydratase is the first essential bacterial enzyme for B. japonicum heme synthesis in soybean <span class="trait">root</span> nodules. In addition, we postulate that ALA is the only heme intermediate that can be translocated from the plant to the endosymbiont to support bacterial heme synthesis in nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmEXPB2, a Cell Wall β-Expansin, Affects Soybean Nodulation through Modifying <span class="trait">Root</span> Architecture and Promoting Nodule Formation and Development.</h2>
			<h3>PMID:26432877  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li X(1), Zhao J(1), Tan Z(1), Zeng R(1), Liao H(2).
				<br/> 
				年份：2015 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nodulation is an essential process for biological nitrogen (N2) fixation in legumes, but its regulation remains poorly understood. Here, a β-expansin gene, <span class="gene">GmEXPB2,</span> was found to be critical for soybean (Glycine max) nodulation. <span class="gene">GmEXPB2</span> was preferentially expressed at the early stage of nodule development. β-Glucuronidase staining further showed that <span class="gene">GmEXPB2</span> was mainly localized to the nodule vascular trace and nodule vascular bundles, as well as nodule cortical and parenchyma cells, suggesting that <span class="gene">GmEXPB2</span> might be involved in cell wall modification and extension during nodule formation and development. Overexpression of <span class="gene">GmEXPB2</span> dramatically modified soybean <span class="trait">root</span> architecture, increasing the size and number of cortical cells in the <span class="trait">root</span> meristematic and elongation zones and expanding <span class="trait">root</span> hair density and size of the <span class="trait">root</span> hair zone. Confocal microscopy with green fluorescent protein-labeled rhizobium USDA110 cells showed that the infection events were significantly enhanced in the <span class="gene">GmEXPB2-overexpressing</span> lines. Moreover, nodule primordium development was earlier in overexpressing lines compared with wild-type plants. Thereby, overexpression of <span class="gene">GmEXPB2</span> in either transgenic soybean hairy roots or whole plants resulted in increased nodule number, nodule mass, and nitrogenase <span class="trait">activity</span> and thus elevated plant N and phosphorus content as well as biomass. In contrast, suppression of <span class="gene">GmEXPB2</span> in soybean transgenic composite plants led to smaller infected cells and thus reduced number of big nodules, nodule mass, and nitrogenase <span class="trait">activity,</span> thereby inhibiting soybean growth. Taken together, we conclude that <span class="gene">GmEXPB2</span> critically affects soybean nodulation through modifying <span class="trait">root</span> architecture and promoting nodule formation and development and subsequently impacts biological N2 fixation and growth of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a <italic> <span class="gene">GmCnx1</span> </italic> Gene Enhanced <span class="trait">Activity</span> of Nitrate Reductase and Aldehyde Oxidase, and Boosted Mosaic Virus <span class="trait">Resistance</span> in Soybean</h2>
			<h3>PMID:25886067</h3>
				

				
					<h3>PMCID:4401665</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhou </surname>  <given-names> Zheng </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> He </surname>  <given-names> Hongli </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Luping </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Xiaoqian </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mi </surname>  <given-names> Qian </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pang </surname>  <given-names> Jingsong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Guixiang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Bao </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Molybdenum cofactor (Moco) is required for the activities of Moco-dependant enzymes. Cofactor for nitrate reductase and xanthine dehydrogenase (Cnx1) is known to be involved in the biosynthesis of Moco in plants. In this work, a soybean (Glycine max L.) Cnx1 gene <span class="gene">(GmCnx1)</span> was transferred into soybean using Agrobacterium tumefaciens-mediated transformation method. Twenty seven positive transgenic soybean plants were identified by coating leaves with phosphinothricin, bar protein quick dip stick and PCR analysis. Moreover, Southern blot analysis was carried out to confirm the insertion of <span class="gene">GmCnx1</span> gene. Furthermore, expression of <span class="gene">GmCnx1</span> gene in leaf and <span class="trait">root</span> of all transgenic lines increased 1.04-2.12 and 1.55-3.89 folds, respectively, as compared to wild type with <span class="gene">GmCnx1</span> gene and in line 10 , 22 showing the highest expression. The activities of Moco-related enzymes viz nitrate reductase (NR) and aldehydeoxidase (AO) of T1 generation plants revealed that the best line among the <span class="gene">GmCnx1</span> transgenic plants accumulated 4.25 μg g(-1) h(-1) and 30 pmol L(-1), respectively (approximately 2.6-fold and 3.9-fold higher than non-transgenic control plants).In addition, overexpression ofGmCnx1boosted the <span class="trait">resistance</span> to various strains of soybean mosaic virus (SMV). DAS-ELISA analysis further revealed that infection rate of <span class="gene">GmCnx1</span> transgenic plants were generally lower than those of non-transgenic plants among two different virus strains tested. Taken together, this study showed that overexpression of a <span class="gene">GmCnx1</span> gene enhanced NR and AO activities and SMV <span class="trait">resistance,</span> suggesting its important role in soybean genetic improvement.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						However, recent studies about the physiological and biochemical functions of <italic> <span class="gene">GmCnx1</span> </italic> Moco biosynthesis genes have been minute. Previous literatures have shown the importance of <italic> Cnx1 </italic> in forming a stable Moco, and that <italic> Cnx1 </italic> binds to actin filaments of plants cytoskeleton located on the membrane with N-terminal domains [ <xref rid="pone.0124273.ref014" ref-type="bibr"> 14 </xref> ]. In addition, expression of a senescence-associated gene <italic> Ntdin </italic> was related to synthesis of Moco in tobacco, resulting in the increase of the content of Moco. NR and XDH <span class="trait">activity</span> was also improved in transgenic <italic> Ntdin </italic> plants [ <xref rid="pone.0124273.ref015" ref-type="bibr"> 15 </xref> ]. It was also found that expression of the Mocosulfide enzyme in soybean enhanced <span class="trait">drought</span> <span class="trait">resistance,</span> which in turn improved its <span class="trait">yield</span> [ <xref rid="pone.0124273.ref016" ref-type="bibr"> 16 </xref> ].Furthermore, overexpression of <italic> Cnx1 </italic> gene was related to SMV infection based on whole-genome microarray analysis [ <xref rid="pone.0124273.ref017" ref-type="bibr"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although studies regarding the function of <italic> Cnx1 </italic> in Moco biosynthesis have been widely conducted, its specific functions in enhancing NR, AO activities and SMV <span class="trait">resistance</span> have not been explored yet. Therefore, the main aim of this study was to elucidate the potential functions of <italic> <span class="gene">GmCnx1</span> </italic> in enhancing NR, AO activities and SMV <span class="trait">resistance</span> using the <italic> Agrobacterium tumefaciens </italic> -mediated transformation, and characterization of the <italic> <span class="gene">GmCnx1</span> </italic> -overexpressing transgenic soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Genomic DNA was extracted from leaf of non-transgenic and the putative T <sub> 1 </sub> <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants (based on herbicide <span class="trait">resistance,</span> mentioned above) by the method described by Edwards et al.[ <xref rid="pone.0124273.ref022" ref-type="bibr"> 22 </xref> ] with minor modifications. Plasmid DNA and genomic DNA were digested with restriction enzymes <italic> EcoR </italic> I and <italic> Hind </italic> &#x02162;, respectively, at 37&#x000b0;C overnight. Digested DNA was separated on 0.8% (w/v) agarose gel and blotted onto Hybond N <sup> + </sup> nylon membrane. The membrane was hybridized with a DIG-labelled <italic> <span class="gene">GmCnx1</span> </italic> -specific probe (5&#x02019;-CGAGTATCCCGTCGTTG-3&#x02019; and 5&#x02019;-TTCCAGGTAGCCCAAAA-3&#x02019;) at 68&#x000b0;C for 16 h. The hybridized membrane was washed and detected according to the protocol of High Prime DNA Labeling and Detection Starter Kit I (Roche, USA).
					</p>
				
					<p>&nbsp;&nbsp;
						Fresh tissues of two-week-old T <sub> 1 </sub> transgenic and non-transgenic soybean plants were collected in liquid nitrogen before isolation of RNA. Total RNA was isolated leaf and <span class="trait">root</span> using Trizol reagent (TaKaRa, Dalian, China). First strand cDNA was synthesized with 1 &#x003bc;g of purified total RNA using PrimeScript RT reagent Kit with gDNA Eraser (TaKaRa). Real-time PCR reaction was carried out according to SYBR <italic> Premix Ex Taq </italic> II (TaKaRa). The gene-specific primers of <italic> <span class="gene">GmCnx1</span> </italic> (5&#x02019;-CCTGGAAACCCTAATGCTGT-3&#x02019; and 5&#x02019;-CACATCTGCAGGAACTGCTT-3&#x02019;) were used in the RT-PCR and gene&#x02014;specific primers of <italic> actin11 </italic> (5&#x02019;-ATCTTGACTGAGCGTGGTTATTCC-3&#x02019; and 5&#x02019;-GCTGGTCCTGGCTGTCTCC-3&#x02019;) were amplified as internal control to quantify the relative amounts of cDNA [ <xref rid="pone.0124273.ref023" ref-type="bibr"> 23 </xref> ]. The amplifications were performed on CFX96 Real Time System (Bio-Rad, Hercules, CA, USA) and the relative expression was calculated by 2 <sup> -&#x00394;&#x00394;Ct </sup> method [ <xref rid="pone.0124273.ref024" ref-type="bibr"> 24 </xref> ]. Significant differences of all data were analyzed using DPS software.
					</p>
				
					<p>&nbsp;&nbsp;
						NR <span class="trait">activity</span> was measured according to the method described previously[ <xref rid="pone.0124273.ref025" ref-type="bibr"> 25 </xref> ]. The NR <span class="trait">activity</span> of plant crude extracts was assayed using leaf of non-transgenic and T <sub> 1 </sub> <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants, which were homogenized in extraction buffer and centrifuged for 15 min at 4,000 rpm. The supernatants were collected and added to the reaction buffer. After incubation at 25&#x000b0;C for 30 min, the reaction was stopped by 1 ml 1% sulphanilamide. The mixture was centrifuged for 5 min at 5,000 rpm and the supernatant was used to detect nitrite production by reading the absorbance at 540 nm after addition of <italic> N </italic> -(1-naphthyl) ethylenediamine dihydrochloride. NO <sub> 2 </sub> <sup> - </sup> was quantified to calculate NR <span class="trait">activity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Vector contained selective maker gene ( <italic> bar </italic> ) coded phosphinothricin acetyltransferase (PAT) and showed <span class="trait">resistance</span> to the herbicide phosphinothricin, green fluorescent protein gene ( <italic> Egfp </italic> ) and <italic> <span class="gene">GmCnx1</span> </italic> gene. LB, left border; RB, right border; p35s, promoter; T35S, terminator. (B) T <sub> 1 </sub> transgenic lines were confirmed by coating leaves with Phosphinothricin. (C)T <sub> 1 </sub> transgenic lines were confirmed by bar protein quick dip strip. (D) T <sub> 1 </sub> transgenic lines were confirmed by PCR. (E) Southern blot analysis of transgene copy number in T <sub> 1 </sub> transgenic soybean and WT. Genomic DNA and plasmid DNA was digested with restriction enzyme <italic> EcoR </italic> Iand <italic> Hind </italic> &#x02162;. The probe was used for <italic> <span class="gene">GmCnx1</span> </italic> . M, DL2000 marker; +, positive control (plasmid DNA); WT, negative control, non-transgenic plants; 0, blank. L1, L2, L3, L4, L7, L10, L17, L18, L22 and L26 represent the <italic> <span class="gene">GmCnx1</span> </italic> transgenic line numbers.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression of <italic> <span class="gene">GmCnx1</span> </italic> was investigated by qRT-PCR in leaf and <span class="trait">root</span> under normal growth conditions. The results ( <xref rid="pone.0124273.g003" ref-type="fig"> Fig 3 </xref> ) showed that the expression of <italic> <span class="gene">GmCnx1</span> </italic> gene increased from 1.04 to 2.12 fold in leaf and 1.55 to 3.89 fold in <span class="trait">root</span> of all transgenic positive plants in comparison to the WT, with line 10 and 22 exhibiting the highest expression levels among the different transgenic lines. These results indicated that the <italic> <span class="gene">GmCnx1</span> </italic> gene expression was more significantly upregulated in <span class="trait">root</span> than in leaf.
					</p>
				
					<p>&nbsp;&nbsp;
						As NR and AO <span class="trait">activity</span> was causally linked with the biosynthesis of Moco, they were measured in leaves of WT and T <sub> 1 </sub> transgenic plants. The results showed that, NR and AO activities increased in <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants compared with WT ( <xref rid="pone.0124273.g004" ref-type="fig"> Fig 4 </xref> ). In particular, for NR <span class="trait">activity,</span> the <italic> <span class="gene">GmCnx1</span> </italic> transgenic soybean plants produced high level (4.25 &#x003bc;g g <sup> -1 </sup> h <sup> -1 </sup> ) in line 10, which were 2.6-fold higher than the non-transgenic plants (1.62 &#x003bc;g g <sup> -1 </sup> h <sup> -1 </sup> ), and for AO <span class="trait">activity,</span> the level in the leaf was 3.9-fold higher in <italic> <span class="gene">GmCnx1</span> </italic> transgenic soybean plants (30.00 pmol L <sup> -1 </sup> ) in line 10 compared with the non-transgenic plants (7.71 pmol L <sup> -1 </sup> ). That was consistent with the expression of <italic> <span class="gene">GmCnx1</span> </italic> gene in leaf and <span class="trait">root.</span> Taken together, it can be concluded that the different <italic> <span class="gene">GmCnx1</span> </italic> levels among the different transgenic lines correspond to similar differences at NR and AO activities in leaf.
					</p>
				
					<p>&nbsp;&nbsp;
						After being infected by SMV-1 and SMV-7 mosaic virus strains, non-transgenic soybean plants showed dwarfing ( <xref rid="pone.0124273.g005" ref-type="fig"> Fig 5a </xref> ), mosaic leaves, browning veins and crimple leaves ( <xref rid="pone.0124273.g005" ref-type="fig"> Fig 5b </xref> ). In contrast, leaves of T <sub> 1 </sub> transgenic soybean plants remained green, showing no mosaic, shrinkage leaves and other symptoms. <span class="trait">Resistance</span> analysis by DAS-ELISA ( <xref rid="pone.0124273.g006" ref-type="fig"> Fig 6 </xref> ) showed that infection rate of <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants were generally lower than those of non-transgenic plants. Specifically, for SMV-1, the infection rates of non-transgenic and10 lines of <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants were 87%, 47%, 40%, 33%, 53%, 33%, 33%, 47%, 40%, 33% and 47%, respectively. For SMV-7, the infection rates of non-transgenic and 10 lines of <italic> <span class="gene">GmCnx1</span> </italic> transgenic plants were 93%, 40%, 53%, 47%, 40%, 33%, 40%, 47%, 33%, 33% and 40%, respectively. The results above indicated that the <italic> <span class="gene">GmCnx1</span> </italic> transgenic soybean lines we generated showed higher degrees of <span class="trait">resistance</span> to two SMV strains tested.
					</p>
				
					<p>&nbsp;&nbsp;
						Although soybean is one of the most important food and feed crops worldwide, its transformation efficiency is relatively low. In comparison with other crops, there are still several problems that need to be tackled such as difficulties in the regeneration potential of the tissues, low reproducibility, and greater genotypic differences[ <xref rid="pone.0124273.ref028" ref-type="bibr"> 28 </xref> ].In the light of these issues, we optimized several physical and chemical parameters of the standard <italic> Agrobacterium tumefaciens </italic> -mediated transformation protocol for soybean, and obtained significantly improved results [ <xref rid="pone.0124273.ref029" ref-type="bibr"> 29 </xref> ].Here, in soybean, we successfully generated transgenic soybean plants for the <italic> <span class="gene">GmCnx1</span> </italic> gene, validated the previous function of the gene in enhancing NR <span class="trait">activity</span> and unveiled the novel function of the gene in enhancing AO <span class="trait">activity</span> and SMV <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Cnx1 is a functional protein with two domains, which are essential for catalyzing the insertion of Mo into MPT in plants [ <xref rid="pone.0124273.ref030" ref-type="bibr"> 30 </xref> ]. E and G domains that were detected in Arabidopsis were involved in the <span class="trait">activity</span> of Cnx1 [ <xref rid="pone.0124273.ref014" ref-type="bibr"> 14 </xref> ]. In the present study, the comparison of amino acid sequences of Cnx1 between Arabidopsis and soybean shows two highly conserved amino acid motifs involved in biosynthesis of molybdopterin cofactor present in soybean ( <xref rid="pone.0124273.g001" ref-type="fig"> Fig 1b </xref> ). Moreover, these two domains were suggested in binding with molybdopterin and form an alpha/beta structure. So it was predicted that these two domains would interact with each other for the function of <span class="gene">GmCnx1,</span> thus catalyzing the incorporation of Mo into MPT in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						According to Mendel et al. [ <xref rid="pone.0124273.ref031" ref-type="bibr"> 31 </xref> ], the third step in the biosynthesis of Moco is catalyzed by the two domain-containing proteins Cnx1 (Cnx1-G, Cnx1-E). Mature Moco can bind to NR and AO. Moco-deficient plant mutants showed a pleiotropic loss of all four Mo-enzymes activities NR, AO, XDH and SO. Moco-mutants have been described in numerous higher plants, for example, in tobacco [ <xref rid="pone.0124273.ref032" ref-type="bibr"> 32 </xref> , <xref rid="pone.0124273.ref033" ref-type="bibr"> 33 </xref> ], <italic> Nicotiana plumbaginigolia </italic> [ <xref rid="pone.0124273.ref034" ref-type="bibr"> 34 </xref> ] and barley [ <xref rid="pone.0124273.ref035" ref-type="bibr"> 35 </xref> ]. <italic> Cnx1 </italic> expression was found in all organs of Arabidopsis plants, and it was noted that transgenic <italic> Cnx1 </italic> gene could increase <span class="trait">activity</span> of NR and content of MPT [ <xref rid="pone.0124273.ref014" ref-type="bibr"> 14 </xref> ]. Moreover, blockage of Moco biosynthesis due to mutations led to the impairment of essential metabolic functions because of the loss of functions of all Mo-dependent enzymes, which ultimately led to death of the organism [ <xref rid="pone.0124273.ref036" ref-type="bibr"> 36 </xref> ]. On the other hand, there is paucity of literature regarding AO activities in plants. Early reports showed in potato the presence of two isoenzymes for AO that are able to use diverse aldehydes but not xanthine as substrates [ <xref rid="pone.0124273.ref037" ref-type="bibr"> 37 </xref> ]. The reduction of ABA levels due to the lack of AO <span class="trait">activity</span> had a direct impact on Moco-sulfurase deficiency [ <xref rid="pone.0124273.ref038" ref-type="bibr"> 38 </xref> , <xref rid="pone.0124273.ref039" ref-type="bibr"> 39 </xref> ]. A barley <italic> cnx </italic> mutant showed a thermo sensitive, wilty phenotype, which was attributed to an ABA deficiency due to the absence of <span class="trait">activity</span> of a Moco-dependent AO[ <xref rid="pone.0124273.ref040" ref-type="bibr"> 40 </xref> ].We found that overexpression of <italic> <span class="gene">GmCnx1</span> </italic> gene indeed enhanced activities of NR and AO in the transgenic soybean plants (Figs <xref rid="pone.0124273.g003" ref-type="fig"> 3 </xref> and <xref rid="pone.0124273.g004" ref-type="fig"> 4 </xref> ), consistent with previous studies in other plants. In addition, for evaluation of agronomic traits, three transgenic and three non-transgenic soybean plants were randomly sampled before flowering stage from the experimental greenhouse. Plant height, average leaf length, average leaf width, nodules number, fresh weight of nodules, <span class="trait">root</span> and shoot fresh weight, <span class="trait">root</span> and shoot dry weight and total biomass were measured ( <xref rid="pone.0124273.s002" ref-type="supplementary-material"> S1 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						SMV seriously affects <span class="trait">yield</span> and quality of soybean globally. Due to its wide distribution, great harmfulness, and difficulties in chemical control, cultivating and planting resistant varieties is an economical, safe and effective way to prevent SMV. In the present study we aimed at checking the relationship between <italic> <span class="gene">GmCnx1</span> </italic> gene and its function in plants with regards to SMV <span class="trait">resistance.</span> Based on a microarray analysis of gene expression differences between two contrasting genotypes (SMV resistant and SMV susceptible) upon SMV inoculation, <italic> <span class="gene">GmCnx1</span> </italic> was selected as a candidate SMV resistant gene [ <xref rid="pone.0124273.ref017" ref-type="bibr"> 17 </xref> ]. We therefore deduced that the <italic> <span class="gene">GmCnx1</span> </italic> gene expression in soybean is probably related to the positive regulation on the <span class="trait">resistance</span> to SMV. In this study, SMV-1 and SMV-7 were used to test for plant <span class="trait">resistance</span> to SMV. Consistent with the results from the microarray experiments, our results showed that transgenic plants indeed have higher SMV <span class="trait">resistance</span> compared with non-transgenic plants after the virus infection ( <xref rid="pone.0124273.g006" ref-type="fig"> Fig 6 </xref> ). We suspect that <span class="trait">resistance</span> to SMV might be due to the increase in NR and AO activities which are enhanced by the overexpression of <italic> <span class="gene">GmCnx1</span> </italic> gene in soybean ( <xref rid="pone.0124273.g007" ref-type="fig"> Fig 7 </xref> ). In any case, this is the first report about the novel function of <italic> <span class="gene">GmCnx1</span> </italic> against SMV, which has apparent breeding implications in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Follow-up studies are required to elucidate the underlying mechanism of <italic> <span class="gene">GmCnx1</span> </italic> function in response to NR, AO activities and SMV <span class="trait">resistance,</span> and in particular, the exact role of AO <span class="trait">activity</span> in post-invasive cell wall defense to SMV. Here, we observed that <italic> <span class="gene">GmCnx1</span> </italic> gene enhanced SMV <span class="trait">resistance</span> of plants, but whether <italic> Cnx2 </italic> , <italic> Cnx3 </italic> , <italic> Cnx6 </italic> and <italic> Cnx7 </italic> , which function in the initial steps of Moco biosynthesis ( <xref rid="pone.0124273.g007" ref-type="fig"> Fig 7 </xref> ), had similar effects on SMV <span class="trait">resistance</span> remain unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, we show that overexpression of <italic> <span class="gene">GmCnx1</span> </italic> gene enhanced NR, AO activities and SMV <span class="trait">resistance</span> in soybean. This report provided valuable information on molecular breeding and improved our understanding of the biological and molecular functions of <italic> <span class="gene">GmCnx1</span> </italic> . These data provide a novel route for initiating molecular breeding to improve NR, AO activities and SMV <span class="trait">resistance</span> in soybean, a significant crop worldwide.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of a novel group of legume-specific, Golgi apparatus-localized WRKY and Exo70 proteins from soybean</h2>
			<h3>PMID:25805717</h3>
				

				
					<h3>PMCID:4449531</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chi </surname>  <given-names> Yingjun </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many plant genes belong to families that arise from extensive proliferation and diversification allowing the evolution of functionally new proteins. Here we report the characterization of a group of proteins evolved from WRKY and exocyst complex subunit Exo70 proteins through fusion with a novel transmembrane (TM) domain in soybean (Glycine max). From the soybean genome, we identified a novel WRKY-related protein <span class="gene">(GmWRP1)</span> that contains a WRKY domain with no binding <span class="trait">activity</span> for W-box sequences. GFP fusion revealed that <span class="gene">GmWRP1</span> was targeted to the Golgi apparatus through its N-terminal TM domain. Similar Golgi-targeting TM domains were also identified in members of a new subfamily of Exo70J proteins involved in vesicle trafficking. The novel TM domains are structurally most similar to the endosomal cytochrome b561 from birds and close homologues of <span class="gene">GmWRP1</span> and <span class="gene">GmEx070J</span> proteins with the novel TM domain have only been identified in legumes. Transient expression of some <span class="gene">GmExo70J</span> proteins or the Golgi-targeting TM domain in tobacco altered the subcellular structures labelled by a fluorescent Golgi marker. <span class="gene">GmWRP1</span> transcripts were detected at high levels in roots, flowers, pods, and seeds, and the expression levels of <span class="gene">GmWRP1</span> and <span class="gene">GmExo70J</span> genes were elevated with increased age in leaves. The legume-specific, Golgi apparatus-localized <span class="gene">GmWRP1</span> and <span class="gene">GmExo70J</span> proteins are probably involved in Golgi-mediated vesicle trafficking of biological molecules that are uniquely important to legumes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						WRKY transcription factors are encoded by large gene families with more than 70 members in <italic> Arabidopsis </italic> ( <xref rid="CIT0042" ref-type="bibr"> Rushton <italic> et al. </italic> , 2010 </xref> ). Based on the number and amino acid sequence of the WRKY zinc-finger motifs, the WRKY family was originally divided into three groups. Group I WRKY proteins contains two C2H2 zinc-finger motifs, while groups II and III contain one C2H2 and one C2HC zinc-finger motif, respectively ( <xref rid="CIT0019" ref-type="bibr"> Eulgem <italic> et al. </italic> , 2000 </xref> ). Additional phylogenetic analyses, however, have shown that the group II proteins can be further divided into groups IIa&#x02013;e ( <xref rid="CIT0058" ref-type="bibr"> Zhang and Wang, 2005 </xref> ). Through extensive searching and sequence verification with the annotated soybean genome, we identified more than 170 WRKY genes from soybean, which is significantly more than the 133 members identified from a previously reported study with BLAST searching used <italic> Arabidopsis </italic> WRKY proteins as queries ( <xref rid="CIT0055" ref-type="bibr"> Yin <italic> et al. </italic> , 2013 </xref> ), but similar to the number identified from a more recent study through genome-wide annotation ( <xref rid="CIT0007" ref-type="bibr"> Bencke-Malato <italic> et al. </italic> , 2014 </xref> ). Phylogenetic analysis through multiple sequence alignment of the WRKY domains classifies the vast majority of soybean WRKY proteins into one of the seven groups. There were, however, a few exceptions, including <span class="gene">Glyma14g37960</span> ( <xref ref-type="fig" rid="F1"> Fig. 1A </xref> ). Comparison with the consensus WRKY-domain sequences of the seven groups of WRKY proteins ( <xref rid="CIT0042" ref-type="bibr"> Rushton <italic> et al. </italic> , 2010 </xref> ) indicated that the WRKY domain of <span class="gene">Glyma14g37960</span> is most closely related to the N-terminal WRKY domain of group I WRKY proteins ( <xref ref-type="fig" rid="F1"> Fig. 1B </xref> ). However, unlike typical group I WRKY proteins with two WRKY domains, <span class="gene">Glyma14g37960</span> does not contain the C-terminal WRKY domain, which is likely to be responsible for the sequence-specific DNA-binding <span class="trait">activity</span> of group I WRKY proteins ( <xref rid="CIT0012" ref-type="bibr"> de Pater <italic> et al. </italic> , 1996 </xref> ; <xref rid="CIT0027" ref-type="bibr"> Ishiguro and Nakamura, 1994 </xref> ). In addition, the almost invariant WRKYGQK sequence at the N-terminus of the WRKY domain in the vast majority of WRKY proteins was changed into WRKYEDK in <span class="gene">Glyma14g37960</span> ( <xref ref-type="fig" rid="F1"> Fig. 1A </xref> and <xref ref-type="fig" rid="F1"> 1B </xref> ). As will be described later, <span class="gene">Glyma14g37960</span> contains a novel domain at its N-terminus and is localized in the Golgi apparatus. Thus <span class="gene">Glyma14g37960</span> has undergone extensive structural modifications, most likely resulting in novel biological functions. Based on these observations, <span class="gene">Glyma14g37960</span> was named soybean WRKY-related protein 1 <span class="gene">(GmWRP1).</span>
					</p>
				
					<p>&nbsp;&nbsp;
						WRKY proteins bind the TTGACC/T W-box sequences that are present in the promoters of a large number of plant genes, particularly those involved in plant responses to biotic and abiotic stresses ( <xref rid="CIT0015" ref-type="bibr"> Dong <italic> et al. </italic> , 2003 </xref> ; <xref rid="CIT0018" ref-type="bibr"> Eulgem, 2006 </xref> ; <xref rid="CIT0042" ref-type="bibr"> Rushton <italic> et al. </italic> , 2010 </xref> ). To determine whether <span class="gene">GmWRP1</span> retains the most important molecular <span class="trait">activity</span> of WRKY proteins, we examined the sequence-specific DNA-binding <span class="trait">activity</span> of recombinant <span class="gene">GmWRP1</span> using electrophoretic mobility shift assay (EMSA). As a positive control, we included <italic> Arabidopsis </italic> AtWRKY33, a group I WRKY protein that is known to bind W-boxes in a sequence-specific manner ( <xref rid="CIT0059" ref-type="bibr"> Zheng <italic> et al. </italic> , 2006 </xref> ; <xref rid="CIT0033" ref-type="bibr"> Lai <italic> et al. </italic> , 2011 </xref> ). When recombinant AtWRKY33 proteins were incubated with a labelled Pchn0 probe, which contains two TTGACC W-box sequences ( <xref rid="CIT0054" ref-type="bibr"> Yang <italic> et al. </italic> , 1999 </xref> ), we detected a strong DNA-binding <span class="trait">activity</span> by AtWRKY33 based on the appearance of an intense band with reduced mobility ( <xref ref-type="fig" rid="F2"> Fig. 2 </xref> ). To determine whether AtWRKY33 specifically recognized the TTGACC W-box sequences, we also tested a mutant probe (mPchm) in which the two TTGACC sequences were mutated into TTGAAC. As shown in <xref ref-type="fig" rid="F2"> Fig. 2 </xref> , the mutant probe was not recognized by recombinant AtWRKY33. On the other hand, when recombinant <span class="gene">GmWRP1</span> proteins were incubated with Pchn or mPchn0, no retarded band was detected by EMSA. These results indicate that <span class="gene">GmWRP1</span> is unable to recognize W-box sequences.
					</p>
				
					<p>&nbsp;&nbsp;
						The lack of <span class="gene">GmWRP1</span> <span class="trait">activity</span> to bind W-box sequences raises the possibility of novel functions of the WRKY-related protein in soybean. To examine this possibility, we first analysed the subcellular localization of <span class="gene">GmWRP1.</span> We generated a <italic> <span class="gene">GmWRP1&#x02013;GFP</span> </italic> fusion gene ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/erv104/-/DC1"> Supplementary Figure S1 </ext-link> ) and transiently expressed it in <italic> N. benthamiana. </italic> Interestingly, in tobacco leaf epidermal cells expressing <italic> <span class="gene">GmWRP1&#x02013;GFP</span> </italic> , when viewed by confocal fluorescence microscopy we observed punctate fluorescent signals in the cytoplasm ( <xref ref-type="fig" rid="F3"> Fig. 3A </xref> ). To identify the subcellular structures relating to the punctate signals, we co-expressed <span class="gene">GmWRP1&#x02013;GFP</span> with an mRFP fused with the signal anchor of a rat sialyl transferase (ST-mRFP), a fluorescent Golgi apparatus marker ( <xref rid="CIT0030" ref-type="bibr"> Kim <italic> et al. </italic> , 2001 </xref> ). As shown in <xref ref-type="fig" rid="F3"> Fig. 3B </xref> , the co-localization analysis revealed that &#x0003e;90% of the <span class="gene">GmWRP1&#x02013;GFP</span> punctate signals were labelled by ST-mRFP. By contrast, co-localization analysis between <span class="gene">GmWRP1&#x02013;GFP</span> and a BIP-mCherry-HDEL fluorescent marker of endoplasmic reticulum showed little overlapping of labelled subcellular structures ( <xref ref-type="fig" rid="F3"> Fig. 3B </xref> ). We also co-expressed <span class="gene">GmWRP1&#x02013;GFP</span> with markers of the <italic> trans </italic> -Golgi network (TGN) (Syp41-mCherry) and multivesicular bodies (MVB) (Ara6-mCherry), both of which also exhibit punctate fluorescence signals when expressed in plant cells ( <xref rid="CIT0013" ref-type="bibr"> Dettmer <italic> et al. </italic> , 2006 </xref> ; <xref rid="CIT0017" ref-type="bibr"> Ebine <italic> et al. </italic> , 2011 </xref> ; <xref rid="CIT0036" ref-type="bibr"> Lin <italic> et al. </italic> , 2014 </xref> ). Again we observed little overlapping of the <span class="gene">GmWRP1&#x02013;GFP</span> signals with those of Syp41-mCherry or ARA6-mCherry ( <xref ref-type="fig" rid="F3"> Fig. 3B </xref> ). These results indicated that <span class="gene">GmWRP1</span> is primarily localized in Golgi complexes of plant cells.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a Soybean Ariadne-Like Ubiquitin Ligase Gene <italic> <span class="gene">GmARI1</span> </italic> Enhances Aluminum Tolerance in Arabidopsis</h2>
			<h3>PMID:25364908</h3>
				

				
					<h3>PMCID:4218711</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaolian </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ning </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Pei </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Mengmeng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Juge </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yufeng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tuanjie </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 9 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ariadne (ARI) subfamily of RBR (Ring Between Ring fingers) proteins have been found as a group of putative E3 ubiquitin ligases containing RING (Really Interesting New Gene) finger domains in fruitfly, mouse, human and Arabidopsis. Recent studies showed several RING-type E3 ubiquitin ligases play important roles in plant response to abiotic stresses, but the function of ARI in plants is largely unknown. In this study, an ariadne-like E3 ubiquitin ligase gene was isolated from soybean, Glycine max (L.) Merr., and designated as <span class="gene">GmARI1.</span> It encodes a predicted protein of 586 amino acids with a RBR supra-domain. Subcellular localization studies using Arabidopsis protoplast cells indicated <span class="gene">GmARI</span> protein was located in nucleus. The expression of <span class="gene">GmARI1</span> in soybean roots was induced as early as 2-4 h after simulated stress treatments such as aluminum, which coincided with the fact of aluminum toxicity firstly and mainly acting on plant roots. In vitro ubiquitination assay showed <span class="gene">GmARI1</span> protein has E3 ligase <span class="trait">activity.</span> Overexpression of <span class="gene">GmARI1</span> significantly enhanced the aluminum tolerance of transgenic Arabidopsis. These findings suggest that <span class="gene">GmARI1</span> encodes a RBR type E3 ligase, which may play important roles in plant tolerance to aluminum stress.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Ubiquitination is an enzymatic, protein post-translational modification by which proteins are selectively targeted for a variety of cellular processes including DNA transcription and repair, cell cycle and division, response to stresses and many others <xref rid="pone.0111120-Hershko1" ref-type="bibr"> [1] </xref> . This process is carried out by three types of enzyme, including an ubiquitin-activating enzyme (E1), an ubiquitin-conjugating enzyme (E2), and an ubiquitin protein ligase (E3) <xref rid="pone.0111120-Pickart1" ref-type="bibr"> [2] </xref> . Encoded by a large gene family of widely divergent isoforms <xref rid="pone.0111120-Stone1" ref-type="bibr"> [3] </xref> , E3 ligases play important roles in governing the ubiquitin signaling pathway by transferring ubiquitin from E2 conjugation to specific protein substrates. E3 ligases are generally divided into two families, with either a HECT or RING-finger domain(s) <xref rid="pone.0111120-Hershko1" ref-type="bibr"> [1] </xref> , <xref rid="pone.0111120-Moon1" ref-type="bibr"> [4] </xref> . The RING-type E3 ubiquitin ligases are generally identified by the presence of conserved cysteine- and histidine- rich RING finger motifs that coordinate zinc atoms <xref rid="pone.0111120-Barlow1" ref-type="bibr"> [5] </xref> . Recently several RING-type E3 ubiquitin ligases were found to play important roles in plant responses to abiotic and biotic stresses. The pepper E3 ubiquitin ligase RING1 gene, <italic> CaRING1 </italic> , is required for cell death and the salicylic acid (SA)-dependent defense response <xref rid="pone.0111120-Lee1" ref-type="bibr"> [6] </xref> . <italic> AtAIRP1 </italic> and <italic> AtAIRP2 </italic> play roles in abscisic acid (ABA)-mediated <span class="trait">drought</span> stress responses in Arabidopsis <xref rid="pone.0111120-Cho1" ref-type="bibr"> [7] </xref> . In soybean, a RING-finger protein encoded by <italic> <span class="gene">GmRFP1</span> </italic> was identified and shown to be involved in ABA signaling and stress responses through the ubiquitin-proteasome pathway <xref rid="pone.0111120-Du1" ref-type="bibr"> [8] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> [L.] Merr.) is widely grown as a major source of vegetable oil and protein. Soybean quality and <span class="trait">yield</span> are affected by various abiotic and biotic stresses. Soybean is also an important crop grown in South China, where acid soils comprise approximately 21% of the total land area <xref rid="pone.0111120-Huang1" ref-type="bibr"> [21] </xref> , <xref rid="pone.0111120-Liu1" ref-type="bibr"> [22] </xref> . Aluminum (Al) toxicity is a major limiting factor of plant growth and crop production on acidic soils. There is large variation in Al tolerance among soybean varieties, and most of the Al tolerant varieties are from China <xref rid="pone.0111120-Sapra1" ref-type="bibr"> [23] </xref> . Another study showed many Al tolerant varieties are from South China <xref rid="pone.0111120-Nian1" ref-type="bibr"> [24] </xref> . However, the genes underlying the Al tolerance in soybean remain largely unknown, except recently a soybean malate transporter gene <italic> <span class="gene">GmALMT1</span> </italic> which was shown to mediate <span class="trait">root</span> malate efflux which underlies soybean Al tolerance <xref rid="pone.0111120-Liang1" ref-type="bibr"> [25] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Homozygous T <sub> 3 </sub> seeds of the transgenic lines and wild type plants were used for Al-tolerance analysis. Seeds were surfaced-sterilized as described before <xref rid="pone.0111120-Ardie1" ref-type="bibr"> [33] </xref> and germinated on 1/2 <span class="gene">MS</span> medium for 7 days. The seedlings were transferred to 1/2 <span class="gene">MS</span> without Mg <sup> 2+ </sup> but with 0 or 15 &#x000b5;M Al(NO <sub> 3 </sub> ) <sub> 3 </sub> and 8% Agar, pH 4.3, and then put on the medium with the plates placed vertically. After 15 days, the lengths of the roots were measured, and the Relative <span class="trait">Root</span> Growth (RRG) in each line was calculated as: RRG (%)&#x0200a;=&#x0200a; (RL <sub> Alt </sub> - RL <sub> Al0 </sub> )/(RL <sub> ct </sub> - RL <sub> c0 </sub> ), in which RL <sub> Al0 </sub> represent the <span class="trait">root</span> length before Al treatment, RL <sub> Alt </sub> represent the <span class="trait">root</span> length after 15 days of Al treatment (15 &#x000b5;M Al(NO <sub> 3 </sub> ) <sub> 3 </sub> , pH 4.3), RL <sub> c0 </sub> represent the <span class="trait">root</span> length before growth on the control medium, and RL <sub> ct </sub> represent the <span class="trait">root</span> length after growth on the control medium (0 &#x000b5;M Al(NO <sub> 3 </sub> ) <sub> 3 </sub> , pH 4.3). The statistical analysis of the experimental data was conducted by t-test with SPSS <xref rid="pone.0111120-Norusis1" ref-type="bibr"> [34] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To test if <span class="gene">GmARI1</span> has the E3 ubiquitin ligase <span class="trait">activity,</span> a full-length <span class="gene">GmARI1</span> protein with maltose binding protein (6&#x000d7; His tag) was expressed in <italic> E. coli </italic> and subsequently affinity-purified (His <sub> 6 </sub> <span class="gene">-GmARI1)</span> from the soluble fraction ( <xref ref-type="supplementary-material" rid="pone.0111120.s001"> Fig. S1 </xref> ). The purified recombinant His <sub> 6 </sub> <span class="gene">-GmARI1</span> protein was about 66 kDa as expected, and the western blotting with anti-His <sub> 6 </sub> monoclonal antibody also showed the purified target recombinant protein had the right molecular weight of 66 kDa ( <xref ref-type="fig" rid="pone-0111120-g008"> Fig. 8A </xref> ). In vitro self-ubiquitination assays were performed in the presence of rabbit E1, human E2 (UbcH5b), and Ub ( <xref ref-type="fig" rid="pone-0111120-g008"> Fig. 8B </xref> ). Polyubiquitination was detected only in the presence of E1, E2, Ub and His <sub> 6 </sub> <span class="gene">-GmARI1.</span> A negative result was observed if either E1, E2, Ub or His <sub> 6 </sub> <span class="gene">-GmARI1</span> was omitted in the reaction. These results indicate that <span class="gene">GmARI1</span> has E3 ubiquitin ligase <span class="trait">activity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> A </bold> . Western blot using anti-His6 monoclonal antibody in the presence (+) or absence (&#x02212;) of the purified His6 tag-GmARI1 proteins. <bold> B </bold> . In vitro E3 ubiquitin ligase <span class="trait">activity</span> assay of <span class="gene">GmARI1</span> protein. His6 tag-GmARI1 fusion proteins were assayed for E3 <span class="trait">activity</span> in the presence of rabbit E1, human E2 (UbcH5b), and ubiquitin (Ub). The left numbers denote the molecular masses of marker proteins in kilodaltons.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene expression of <italic> <span class="gene">GmARI1</span> </italic> was induced early by Al stress, therefore we further studied its function in transgenic plants. An expression plasmid vector of <italic> pMDC83-GmARI1 </italic> was constructed and introduced into Arabidopsis plants using floral dip method. Transgenic T <sub> 3 </sub> Arabidopsis over-expressing <italic> <span class="gene">GmARI1</span> </italic> were generated and the positive transgenic lines were identified by RT-PCR ( <xref ref-type="supplementary-material" rid="pone.0111120.s002"> Fig. S2 </xref> ). Seeds of three T <sub> 3 </sub> homozygous transgenic lines ( <italic> <span class="gene">GmARI1-1,</span> <span class="gene">GmARI1-2</span> </italic> , and <italic> <span class="gene">GmARI1-3</span> </italic> ) and wild type Col-0 were germinated on 1/2 <span class="gene">MS</span> medium. After 10 days, the seedlings were transferred to 1/2 <span class="gene">MS</span> (pH4.3) containing 15 &#x000b5;M Al or 0 &#x000b5;M Al as a control. Fifteen days later, the <span class="trait">root</span> growth of the wild type Col-0 was severely inhibited by 15 &#x000b5;M Al as compared with the control medium (0 &#x000b5;M), while the transgenic lines were little affected by Al ( <xref ref-type="fig" rid="pone-0111120-g009"> Fig. 9 </xref> A). The relative <span class="trait">root</span> growth (RRG) of the transgenic lines was significantly ( <italic> p </italic> &#x0003c;0.01) longer than the wild type under Al treatment ( <xref ref-type="fig" rid="pone-0111120-g009"> Fig. 9 </xref> B). The relative abundance of <italic> <span class="gene">GmARI1</span> </italic> in the transgenic lines of <italic> <span class="gene">GmARI1-1</span> </italic> and <italic> <span class="gene">GmARI1-2</span> </italic> is higher than <italic> <span class="gene">GmARI1-3</span> </italic> , which coincided with RRG result ( <xref ref-type="supplementary-material" rid="pone.0111120.s003"> Fig. S3 </xref> and <xref ref-type="fig" rid="pone-0111120-g009"> Fig. 9 </xref> B).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> A </bold> . <span class="trait">Root</span> phenotypes of the <italic> 35S: <span class="gene">GmARI1</span> </italic> overexpression lines <italic> <span class="gene">GmARI1-3</span> </italic> and the wild type Col-0 grown on 15 &#x000b5;M Al in 1/2 <span class="gene">MS</span> (+Al, pH4.3), and 0 &#x000b5;M Al in 1/2 <span class="gene">MS</span> (-Al, pH4.3). <bold> B </bold> . Relative <span class="trait">root</span> growth (RRG, %) of the transgenic Arabidopsis lines ( <italic> <span class="gene">GmARI1-1,</span> <span class="gene">GmARI1-2,</span> <span class="gene">GmARI1-3</span> </italic> ) and the wild type (Col-0). RRG was calculated by the <span class="trait">root</span> growth length under Al treatment (15 &#x000b5;M Al in 1/2 MS, pH4.3) divided by the <span class="trait">root</span> growth length under control (0 &#x000b5;M Al in 1/2 MS, pH4.3). Error bars represent the standard error (SE), ** indicate the significance level of 0.01 by t-tests.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmARI1</span> </italic> gene was expressed ubiquitously in roots, stems, SAMs, leaves, flowers, and pods of soybean plants. The <span class="gene">GmARI1</span> protein was located in the nucleus of the cell. Real-time quantitative PCR showed that the expression level of <italic> <span class="gene">GmARI1</span> </italic> in soybean <span class="trait">root</span> under Al (10 &#x000b5;M, pH 4.3) treatment reached the peak during 2 to 4 hours, but no significant change was detected in the leaves. This coincides with the fact that Al toxicity was first and mainly acting on plant roots <xref rid="pone.0111120-Taylor1" ref-type="bibr"> [38] </xref> , limiting water and nutrition absorption <xref rid="pone.0111120-Barcelo1" ref-type="bibr"> [39] </xref> , which further inhibiting the development of whole plant and reducing <span class="trait">yield</span> <xref rid="pone.0111120-Foy1" ref-type="bibr"> [40] </xref> , <xref rid="pone.0111120-Ma1" ref-type="bibr"> [41] </xref> . Therefore in the early stage (less than 4 hours) of Al stress, induced gene expression of <italic> <span class="gene">GmARI1</span> </italic> in soybean roots may play important roles to trigger downstream signaling pathways to protect <span class="trait">root</span> cells from Al toxicity.
					</p>
				
					<p>&nbsp;&nbsp;
						It has been reported that Al induces oxidative stress and DNA damage in plant cells <xref rid="pone.0111120-Achary1" ref-type="bibr"> [42] </xref> &#x02013; <xref rid="pone.0111120-Yamamoto1" ref-type="bibr"> [44] </xref> . Reactive oxygen species (ROS) constantly attack DNA, leading to oxidative DNA damage <xref rid="pone.0111120-Markkanen1" ref-type="bibr"> [45] </xref> . The cell cycle checkpoint regulators could detect and respond to such damage, leading to inhibition of <span class="trait">root</span> growth <xref rid="pone.0111120-Nezames1" ref-type="bibr"> [46] </xref> . Protein ubiquitination is involved in DNA transcription and repair, cell cycle and division <xref rid="pone.0111120-Hershko1" ref-type="bibr"> [1] </xref> , and is emerging as a critical regulatory mechanism of DNA damage response <xref rid="pone.0111120-Pinder1" ref-type="bibr"> [47] </xref> . Several RING domain-containing E3 ubiquitin ligases play an essential role in response to DNA damage <xref rid="pone.0111120-Bartocci1" ref-type="bibr"> [48] </xref> <sup> . </sup> In vitro ubiquitination assay showed <span class="gene">GmARI1</span> has E3 ligase <span class="trait">activity</span> ( <xref ref-type="fig" rid="pone-0111120-g008"> Fig. 8 </xref> ), therefore we hypothesize that <span class="gene">GmARI</span> might be involved in the oxidative DNA damage repair to confer Al tolerance. We investigated the co-expression pattern of <italic> <span class="gene">GmARI</span> </italic> ( <ext-link ext-link-type="uri" xlink:href="http://bioinformatics.cau.edu.cn/SFGD/"> http://bioinformatics.cau.edu.cn/SFGD/ </ext-link> ), and found <span class="gene">Glyma02g15070</span> was on the top list of coexpression genes with <italic> <span class="gene">GmARI</span> </italic> . The homolog gene of <span class="gene">Glyma02g15070</span> in arabidopsis is AT1G49670, which was shown to be involved in oxidative stress tolerance ( <ext-link ext-link-type="uri" xlink:href="http://www.arabidopsis.org/"> http://www.arabidopsis.org/ </ext-link> ). Suppression of oxidative stress might help plants reduce the damage or <span class="trait">root</span> growth inhibition <xref rid="pone.0111120-Panda1" ref-type="bibr"> [49] </xref> , <xref rid="pone.0111120-Yin1" ref-type="bibr"> [50] </xref> . Therefore, the Al tolerance observed in <italic> <span class="gene">GmARI</span> </italic> overexpression lines might be due to the improved tolerance to oxidative stress, or/and other signaling cascades.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">SAT1</span> (Symbiotic Ammonium Transporter 1) encodes a bHLH transcription factor involved in nodule growth and NH4+ transport.</h2>
			<h3>PMID:24707045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chiasson DM(1), Loughlin PC, Mazurkiewicz D, Mohammadidehcheshmeh M, Fedorova EE, Okamoto M, McLean E, Glass AD, Smith SE, Bisseling T, Tyerman SD, Day DA, Kaiser BN.
				<br/> 
				年份：2014 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine max symbiotic ammonium transporter 1 was first documented as a putative ammonium (NH4(+)) channel localized to the symbiosome membrane of soybean <span class="trait">root</span> nodules. We show that Glycine max symbiotic ammonium transporter 1 is actually a membrane-localized basic helix-loop-helix (bHLH) DNA-binding transcription factor now renamed Glycine max bHLH membrane 1 <span class="gene">(GmbHLHm1).</span> In yeast, <span class="gene">GmbHLHm1</span> enters the nucleus and transcriptionally activates a unique plasma membrane NH4(+) channel Saccharomyces cerevisiae ammonium facilitator 1. Ammonium facilitator 1 homologs are present in soybean and other plant species, where they often share chromosomal microsynteny with bHLHm1 loci. <span class="gene">GmbHLHm1</span> is important to the soybean rhizobium symbiosis because loss of <span class="trait">activity</span> results in a reduction of nodule fitness and growth. Transcriptional changes in nodules highlight downstream signaling pathways involving circadian clock regulation, nutrient transport, hormone signaling, and cell wall modification. Collectively, these results show that <span class="gene">GmbHLHm1</span> influences nodule development and <span class="trait">activity</span> and is linked to a novel mechanism for NH4(+) transport common to both yeast and plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The acid phosphatase-encoding gene <span class="gene">GmACP1</span> contributes to soybean tolerance to low-phosphorus stress.</h2>
			<h3>PMID:24391523  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang D(1), Song H(2), Cheng H(2), Hao D(2), Wang H(2), Kan G(2), Jin H(2), Yu D(3).
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus (P) is essential for all living cells and organisms, and low-P stress is a major factor constraining plant growth and <span class="trait">yield</span> worldwide. In plants, P efficiency is a complex quantitative trait involving multiple genes, and the mechanisms underlying P efficiency are largely unknown. Combining linkage analysis, genome-wide and candidate-gene association analyses, and plant transformation, we identified a soybean gene related to P efficiency, determined its favorable haplotypes and developed valuable functional markers. First, six major genomic regions associated with P efficiency were detected by performing genome-wide associations (GWAs) in various environments. A highly significant region located on chromosome 8, qPE8, was identified by both GWAs and linkage mapping and explained 41% of the phenotypic variation. Then, a regional mapping study was performed with 40 surrounding markers in 192 diverse soybean accessions. A strongly associated haplotype (P = 10(-7)) consisting of the markers Sat_233 and BARC-039899-07603 was identified, and qPE8 was located in a region of approximately 250 kb, which contained a candidate gene <span class="gene">GmACP1</span> that encoded an acid phosphatase. <span class="gene">GmACP1</span> overexpression in soybean hairy roots increased P efficiency by 11-20% relative to the control. A candidate-gene association analysis indicated that six natural <span class="gene">GmACP1</span> polymorphisms explained 33% of the phenotypic variation. The favorable alleles and haplotypes of <span class="gene">GmACP1</span> associated with increased transcript expression correlated with higher enzyme <span class="trait">activity.</span> The discovery of the optimal haplotype of <span class="gene">GmACP1</span> will now enable the accurate selection of soybeans with higher P efficiencies and improve our understanding of the molecular mechanisms underlying P efficiency in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cinnamic Acid Increases Lignin Production and Inhibits Soybean <span class="trait">Root</span> Growth</h2>
			<h3>PMID:23922685</h3>
				

				
					<h3>PMCID:3724853</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Salvador </surname>  <given-names> Victor Hugo </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lima </surname>  <given-names> Rog&#x000e9;rio Barbosa </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> dos Santos </surname>  <given-names> Wanderley Dantas </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soares </surname>  <given-names> Anderson Ricardo </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000f6;hm </surname>  <given-names> Paulo Alfredo Feitoza </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marchiosi </surname>  <given-names> Rog&#x000e9;rio </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferrarese </surname>  <given-names> Maria de Lourdes Lucio </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferrarese-Filho </surname>  <given-names> Osvaldo </given-names>  </name>  <xref ref-type="corresp" rid="cor1"> * </xref>  <xref ref-type="aff" rid="aff1"/>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 6 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cinnamic acid is a known allelochemical that affects seed germination and plant <span class="trait">root</span> growth and therefore influences several metabolic processes. In the present work, we evaluated its effects on growth, indole-3-acetic acid (IAA) oxidase and cinnamate 4-hydroxylase (C4H) activities and lignin monomer composition in soybean (Glycine max) roots. The results revealed that exogenously applied cinnamic acid inhibited <span class="trait">root</span> growth and increased IAA oxidase and <span class="gene">C4H</span> activities. The allelochemical increased the total lignin content, thus altering the sum and ratios of the p-hydroxyphenyl (H), guaiacyl (G), and syringyl (S) lignin monomers. When applied alone or with cinnamic acid, piperonylic acid (PIP, a quasi-irreversible inhibitor of C4H) reduced <span class="gene">C4H</span> <span class="trait">activity,</span> lignin and the H, G, S monomer content compared to the cinnamic acid treatment. Taken together, these results indicate that exogenously applied cinnamic acid can be channeled into the phenylpropanoid pathway via the <span class="gene">C4H</span> reaction, resulting in an increase in H lignin. In conjunction with enhanced IAA oxidase <span class="trait">activity,</span> these metabolic responses lead to the stiffening of the cell wall and are followed by a reduction in soybean <span class="trait">root</span> growth.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We have demonstrated previously that hydroxycinnamic acids, including ferulic, <italic> p </italic> -coumaric and caffeic acids, impaired soybean <span class="trait">root</span> growth and are associated with premature lignification and changes in the lignin monomer composition [ <xref ref-type="bibr" rid="B15"> 15 </xref> &#x02013; <xref ref-type="bibr" rid="B17"> 17 </xref> ]. Based on these findings, a model of action has been proposed that indicates that these allelochemicals can be channeled into the phenylpropanoid pathway, which, in turn, may increase the lignin monomer amount, thus solidifying the cell wall and inhibiting the <span class="trait">root</span> growth of soybean plants. With this in mind, we hypothesized that exogenously applied cinnamic acid could act in a similar manner. The primary objective of this work was to analyze the effects of cinnamic acid on growth, <span class="gene">C4H</span> <span class="trait">activity,</span> and lignin and its monomer composition in soybean roots. Because the plant hormone indole-3-acetic acid (IAA) is required for plant growth, the <span class="trait">activity</span> of IAA oxidase was also evaluated.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> <named-content content-type="genus-species"> Glycine max </named-content> </italic> L. Merr. cv. BRS-184) seeds were surface-sterilized with 2% sodium hypochlorite for 3 min, rinsed extensively with deionized water and dark-germinated at 25&#x000b0;C on two sheets of moistened filter paper. Twenty-five 3-day-old seedlings of uniform size were supported by an adjustable acrylic plate and dipped into a 10&#x000d7;16 cm glass container filled with 200 mL of half-strength Hoagland&#x02019;s solution (pH 6.0), with or without 0.1 to 1.0 mM cinnamic acid. All nutrient solutions were buffered with a 67 mM potassium phosphate buffer to eliminate the effects of a very low pH. Additional experiments with 0.1 mM piperonylic acid (PIP) were conducted, as indicated in the figure legends. The containers were kept in a growth chamber for 24 h at 25&#x000b0;C, with a light/dark photoperiod of 12/12 h and a photon flux density of 280 &#x000b5;mol m <sup> &#x02212;2 </sup> s <sup> &#x02212;1 </sup> . The roots were measured before incubation and at the end of the experiment, and the difference in the lengths was calculated for all of the samples. Where indicated, the fresh <span class="trait">root</span> weight was determined immediately after incubation, and the dry weight was estimated after oven-drying at 80&#x000b0;C until a constant weight. <italic> trans- </italic> Cinnamic acid, <span class="gene">PIP</span> and IAA were purchased from Sigma Aldrich (St Louis, MO, USA), and all other reagents used were of the purest grade available or of chromatographic grade.
					</p>
				
					<p>&nbsp;&nbsp;
						Cinnamate 4-hydroxylase (C4H; E.C. 1.14.13.11) was extracted as described by Enkhardt and Pommer [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Fresh roots (0.5 g) were ground at 4&#x000b0;C in 1 mL of extraction medium containing 100 mM phosphate buffer (pH 7.6), 10 mM potassium chloride, 1 mM magnesium chloride, 8 mM EDTA and 15 mM 2-mercaptoethanol. The homogenates were centrifuged at (10,000&#x000d7;g, 10 min), and the supernatants were used as the enzyme preparations. For the <span class="gene">C4H</span> <span class="trait">activity</span> assay, the reaction mixture contained 100 mM phosphate buffer (pH 7.6), 1.33 mM cinnamic acid, 2 mM NADPH and a suitable amount of enzyme extract in a final volume of 1.5 mL. The mixture was incubated at 35&#x000b0;C, and the reaction was stopped after 40 min by the addition of 75 &#x000b5;L of 5 M HCl. The samples were filtered through a 0.45-&#x000b5;m disposable syringe filter (Hamilton <sup> &#x000ae; </sup> Co, Nevada, USA) and analyzed (20 &#x000b5;L) in an HPLC system (Shimadzu <sup> &#x000ae; </sup> 10AVP, Tokyo, Japan) equipped with a pump, a Rheodyne <sup> &#x000ae; </sup> manual sample injector, and an UV detector. A reversed-phase Shimpack <sup> &#x000ae; </sup> CLC-ODS column (150 mm &#x000d7; 4.6 mm, 5 &#x000b5;m) was used at room temperature together with an equivalent pre-column. The mobile phase was 70% methanol (v/v), with a flow rate of 0.7 mL min <sup> &#x02212;1 </sup> for an isocratic run of 15 min. UV detection was carried out at 275 nm. The substrate of C4H, <italic> t </italic> -cinnamate, was identified by comparing its retention time with a standard compound. <span class="gene">C4H</span> <span class="trait">activity</span> is expressed as nmol <italic> t </italic> -cinnamate consumed min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> fresh weight.
					</p>
				
					<p>&nbsp;&nbsp;
						In agreement with the effects observed on <span class="trait">root</span> growth, the enzyme <span class="trait">activity</span> of seedlings treated with cinnamic acid was also significantly different from those of controls. Roots exposed to cinnamic acid significantly increased IAA oxidase <span class="trait">activity</span> by 32% to 91% (0.1 to 0.5 mM); 76% (0.75 mM) and 30% (1.0 mM) in comparison with the control (2.96 &#x000b1; 0.31 nmols min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> fresh weight) ( <xref ref-type="fig" rid="pone-0069105-g001"> Figure 1) </xref> . Cinnamic acid treatments also increased the <span class="gene">C4H</span> activities by 79% (0.5 mM) to 129% (&#x02265; 0.75 mM) in comparison with the control (5.75 &#x000b1; 0.51 nmols min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> fresh weight) ( <xref ref-type="fig" rid="pone-0069105-g002"> Figure 2) </xref> . The allelochemical increased lignin content by 51% to 138% after 0.1 to 0.75 mM treatments, in comparison with the control (11.7 &#x000b1; 0.38 mg g <sup> &#x02212;1 </sup> dry weight) ( <xref ref-type="fig" rid="pone-0069105-g003"> Figure 3) </xref> . At a concentration of 1.0 mM, cinnamic acid increased the lignin content by 121% when compared with the control.
					</p>
				
					<p>&nbsp;&nbsp;
						Hypothesizing that cinnamic acid can be channeled into the phenylpropanoid pathway via the <span class="gene">C4H</span> reaction, soybean roots were treated with PIP, which is a <italic> quasi </italic> -irreversible inhibitor of this enzyme. <xref ref-type="fig" rid="pone-0069105-g004"> Figure 4 </xref> reveals that <span class="gene">PIP</span> reduced <span class="gene">C4H</span> <span class="trait">activity</span> by 32% (PIP alone) and 33% (cinnamic acid plus PIP) in comparison with the control (6.09 &#x000b1; 0.49 nmols min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> fresh weight). The treatment of roots with <span class="gene">PIP</span> alone or with the allelochemical plus <span class="gene">PIP</span> reduced the enzyme <span class="trait">activity</span> by 69% and 70%, respectively, in comparison with the cinnamic acid treatment (13.69 &#x000b1; 0.99 nmols min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> fresh weight).
					</p>
				
					<p>&nbsp;&nbsp;
						An interesting fact revealed in the current study is that the changes in <span class="trait">root</span> growth and IAA oxidase activities were accompanied by increases in <span class="gene">C4H</span> <span class="trait">activity</span> ( <xref ref-type="fig" rid="pone-0069105-g002"> Figure 2) </xref> and lignin content ( <xref ref-type="fig" rid="pone-0069105-g003"> Figure 3) </xref> . <span class="gene">C4H</span> is the first P450-dependent monooxygenase of the phenylpropanoid pathway, and it is widely expressed in various tissues, particularly in roots and cells undergoing lignification [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. In fact, Ye [ <xref ref-type="bibr" rid="B37"> 37 </xref> ] demonstrated a correlation between <span class="gene">C4H</span> <span class="trait">activity</span> and the deposition of lignin in phloem and xylem fiber cells of <named-content content-type="genus-species"> <named-content content-type="genus"> Zinnia </named-content> <italic> <named-content content-type="species"> elegans </named-content> </italic> </named-content> , while Blount et al. [ <xref ref-type="bibr" rid="B38"> 38 </xref> ] observed that reduced levels of lignin correlated with a pronounced decrease in <span class="gene">C4H</span> <span class="trait">activity</span> in transgenic lines of tobacco. As noted herein, the increased <span class="trait">activity</span> of <span class="gene">C4H</span> ( <xref ref-type="fig" rid="pone-0069105-g002"> Figure 2) </xref> correlated with a pronounced increase in lignin content ( <xref ref-type="fig" rid="pone-0069105-g003"> Figure 3) </xref> . These findings are of particular interest because a reduction in <span class="trait">root</span> growth has been considered one of the first effects of cinnamic acid derivatives that are is associated with the premature lignification of cell walls [ <xref ref-type="bibr" rid="B15"> 15 </xref> &#x02013; <xref ref-type="bibr" rid="B17"> 17 </xref> ]. These citations and the current findings strengthen the notion of a possible influx of these compounds into the phenylpropanoid pathway, followed by increases of lignin monomers that solidify the cell wall and reduce <span class="trait">root</span> growth. Therefore, a similar function can be attributed to cinnamic acid itself. In fact, 0.1&#x02013;1.0 mM cinnamic acid significantly increased the H lignin monomer and, as a result, altered both the sum (H+G+S) of the lignin monomers and the H: G:S ratios ( <xref ref-type="table" rid="tab2"> Table 2) </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm a supposed entry of exogenous cinnamic acid into the phenylpropanoid pathway, additional experiments were performed by growing soybean roots with PIP, a specific inhibitor of C4H. The findings reveal that alone or jointly with cinnamic acid, <span class="gene">PIP</span> reduced <span class="gene">C4H</span> <span class="trait">activity</span> ( <xref ref-type="fig" rid="pone-0069105-g004"> Figure 4) </xref> , the lignin content ( <xref ref-type="fig" rid="pone-0069105-g005"> Figure 5) </xref> and H, G and S monomers ( <xref ref-type="fig" rid="pone-0069105-g006"> Figure 6) </xref> in comparison with the cinnamic acid treatment. These data indicate that roots grown under cinnamic acid treatments produce more lignin, while roots grown with <span class="gene">PIP</span> or cinnamic acid plus <span class="gene">PIP</span> synthesize less lignin. These results are in agreement with the fact that <span class="gene">PIP</span> acts on the entry point of cinnamic acid in the pathway [ <xref ref-type="bibr" rid="B39"> 39 </xref> ]. Because <span class="gene">PIP</span> affects the <span class="gene">C4H</span> reaction, the exogenous allelochemical was blocked from accessing the phenylpropanoid pathway at this metabolic point. These results suggest that exogenously applied cinnamic acid can be channeled into the phenylpropanoid pathway and increase lignin production. In <named-content content-type="genus-species"> Robinia </named-content> ( <named-content content-type="genus-species"> <named-content content-type="genus"> Robinia </named-content> <italic> <named-content content-type="species"> pseudoacacia </named-content> </italic> </named-content> ), labeled ferulic acid, which is another phenylpropanoid allelochemical, was incorporated into G and S lignin monomers, and these incorporations increased the cell-wall lignification [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. In previous studies, we noted that cinnamic acid derivatives (i.e., ferulic, <italic> p </italic> -coumaric and caffeic acids) also increased the lignin content, inducing significant changes in the lignin monomer content [ <xref ref-type="bibr" rid="B15"> 15 </xref> &#x02013; <xref ref-type="bibr" rid="B17"> 17 </xref> ]. Complementarily, Hrubcov&#x000e1; et al. [ <xref ref-type="bibr" rid="B41"> 41 </xref> ] also observed a decline in the IAA level in alfalfa suspension cultures subjected to 2-aminoindan-2-phosphonic acid (AIP), a known inhibitor of the phenylalanine ammonia-lyase (PAL), which is the first enzyme of the phenylpropanoid pathway that is the main route to the lignin production ( <xref ref-type="fig" rid="pone-0069105-g007"> Figure 7) </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Our experimental findings have enabled us to provide a working model for the role of cinnamic acid as an allelochemical ( <xref ref-type="fig" rid="pone-0069105-g007"> Figure 7) </xref> . In this model, exogenous cinnamic acid is channeled into the phenylpropanoid pathway and converted to <italic> p </italic> -coumarate through the <span class="gene">C4H</span> reaction and is, thereafter, converted to <italic> p </italic> -coumaryl-CoA &#x02192; <italic> p </italic> -coumaraldehyde &#x02192; <italic> p </italic> -coumaryl alcohol, which is a monolignol. In a subsequent reaction, this monolignol is polymerized into lignin in the cell wall. In this way, the cinnamic acid-induced inhibition of soybean <span class="trait">root</span> growth can be due to the excessive production of lignin, particularly the H monomer. Concomitantly, an increase in IAA oxidase <span class="trait">activity</span> reduces the endogenous IAA level associated with a decrease in <span class="trait">root</span> growth. In brief, the cinnamic acid-induced inhibition of soybean <span class="trait">root</span> growth can be due to the formation of a complex network that solidifies the cell wall and restricts the plant growth.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">GmCLC1</span> protein from soybean functions as a chloride ion transporter.</h2>
			<h3>PMID:22921676  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wong TH(1), Li MW, Yao XQ, Lam HM.
				<br/> 
				年份：2013 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinization is a global issue that hampers agricultural production. Chloride is one of the prominent anions on saline land that cause toxicity to the plant. We previously identified the <span class="gene">GmCLC1</span> gene from soybean (Glycine max) that encodes a putative tonoplast-localized chloride transporter. In this study, using electrophysiological analysis, we demonstrated the chloride transport function of <span class="gene">GmCLC1.</span> Interestingly, this chloride transport <span class="trait">activity</span> is pH dependent, suggesting that <span class="gene">GmCLC1</span> is probably a chloride/proton antiporter. When the cDNA of <span class="gene">GmCLC1</span> was expressed in tobacco BY-2 cells under the control of a constitutive promoter, the protective effect against salinity stress in transgenic tobacco BY-2 cells was also found to be pH sensitive. In the native host soybean, the expression of <span class="gene">GmCLC1</span> gene is regulated by pH. All these findings support the notion that the function of <span class="gene">GmCLC1</span> is regulated by pH.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physiological responses of glyphosate-resistant and glyphosate-sensitive soybean to aminomethylphosphonic acid, a metabolite of glyphosate.</h2>
			<h3>PMID:21190714  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ding W(1), Reddy KN, Zablotowicz RM, Bellaloui N, Arnold Bruns H.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aminomethylphosphonic acid (AMPA) is formed in glyphosate-treated glyphosate-resistant (GR) and glyphosate-sensitive (GS) soybean [Glycine max (L.) Merr.] plants and is known to cause yellowing in soybean. Although, AMPA is less phytotoxic than glyphosate, its mode of action is different from that of glyphosate and is still unknown. Greenhouse studies were conducted at Stoneville, <span class="gene">MS</span> to determine the effects of AMPA on plant growth, chlorophyll content, photosynthesis, nodulation, nitrogenase <span class="trait">activity,</span> nitrate reductase <span class="trait">activity,</span> and shoot nitrogen content in <span class="gene">GR</span> and <span class="gene">GS</span> soybeans. AMPA was applied to one- to two-trifoliolate leaf stage soybeans at 0.1 and 1.0 kg ha(-1), representing a scenario of 10% and 100% degradation of glyphosate (1.0 kg ae ha(-1) use rate) to AMPA, respectively. Overall, AMPA effects were more pronounced at 1.0 kg ha(-1) than at 0.1 kg ha(-1) rate. Visual plant injury (18-27%) was observed on young leaves within 3d after treatment (DAT) with AMPA at the higher rate regardless of soybean type. AMPA injury peaked to 46-49% at 14 DAT and decreased to 17-18% by 28 DAT, in both soybean types. AMPA reduced the chlorophyll content by 37%, 48%, 66%, and 23% in <span class="gene">GR</span> soybean, and 17%, 48%, 57%, and 22% in <span class="gene">GS</span> soybean at 3, 7, 14, and 28 DAT, respectively. AMPA reduced the photosynthesis rate by 65%, 85%, and 77% in <span class="gene">GR</span> soybean and 59%, 88%, and 69% in <span class="gene">GS</span> soybean at 3, 7, and 14 DAT, respectively, compared to non-treated plants. Similarly, AMPA reduced stomatal conductance to water vapor and transpiration rates at 3, 7, and 14 DAT compared to non-treated plants in both soybean types. Photosynthesis rate, stomatal conductance, and transpiration rate recovered to the levels of non-treated plants by 28 DAT. Plant height and shoot dry weight at 28 DAT; nodulation, nitrogenase <span class="trait">activity</span> at 10 DAT, and nitrate reductase <span class="trait">activity</span> at 3 and 14 DAT were unaffected by AMPA. AMPA reduced <span class="trait">root</span> respiration and shoot nitrogen content at 10 DAT. These results suggest that a foliar application of AMPA could indirectly reduce photosynthesis through decreased chlorophyll content in <span class="gene">GR</span> and <span class="gene">GS</span> soybean up to 14 DAT, but affected plants can recover to normal growth by 28 DAT.
					</p>
				
					<p>&nbsp;&nbsp;
						Published by Elsevier Ltd.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glyphosate affects micro-organisms in rhizospheres of glyphosate-resistant soybeans.</h2>
			<h3>PMID:20880215  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zobiole LH(1), Kremer RJ, Oliveira RS Jr, Constantin J.
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						AIMS: Glyphosate-resistant (GR) soybean production increases each year because of the efficacy of glyphosate for weed management. A new or 'second' generation of <span class="gene">GR</span> soybean (GR2) is now commercially available for farmers that is being promoted as higher yielding relative to the previous, 'first generation' (GR1) cultivars. Recent reports show that glyphosate affects the biology and ecology of rhizosphere micro-organisms in <span class="gene">GR</span> soybean that affect <span class="trait">yield.</span> The objective of this research was to evaluate the microbiological interactions in the rhizospheres of GR2 and GR1 soybean and the performance of the cultivars with different rates of glyphosate applied at different growth stages. METHODS AND RESULTS: A greenhouse study was conducted using GR1 and GR2 soybean cultivars grown in a silt loam soil. Glyphosate was applied at V2, V4 and V6 growth stages at three rates. Plants harvested at R1 growth stage had high <span class="trait">root</span> colonization by Fusarium spp.; reduced rhizosphere fluorescent pseudomonads, Mn-reducing bacteria, and indoleacetic acid-producing rhizobacteria; and reduced shoot and <span class="trait">root</span> biomass. CONCLUSIONS: Glyphosate applied to <span class="gene">GR</span> soybean, regardless of cultivar, negatively impacts the complex interactions of microbial groups, biochemical <span class="trait">activity</span> and <span class="trait">root</span> growth that can have subsequent detrimental effects on plant growth and productivity. SIGNIFICANCE AND IMPACT OF THE STUDY: The information presented here will be crucial in developing strategies to overcome the potential detrimental effects of glyphosate in <span class="gene">GR</span> cropping systems.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of <span class="gene">CHS7</span> and CHS8 genes in soybean.</h2>
			<h3>PMID:20016991  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yi J(1), Derynck MR, Chen L, Dhaubhadel S.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chalcone synthase (CHS) catalyzes the first reaction specific for flavonoid and isoflavonoid biosynthesis. The soybean genome consists of nine copies of <span class="gene">CHS</span> genes (CHS1-CHS9) and a duplicate copy of CHS1. Even though the soybean <span class="gene">CHS</span> gene family members share a high degree of sequence similarity, they play different roles during plant development or in response to environmental stimuli. Our previous work on the comparison of a global gene expression in two soybean cultivars that differ in the level of total isoflavonoid accumulation has denoted the involvement of <span class="gene">CHS7</span> and CHS8 genes in isoflavonoid synthesis. We have extended our effort to understand expression patterns of these two genes in soybean and in transgenic Arabidopsis. Promoter regions of <span class="gene">CHS7</span> and CHS8 genes were isolated and in silico analysis performed to investigate potential transcription factor binding sites (TFBSs). The TFBSs were verified by DNase I footprint analysis. Some unique and several common TFBSs were identified in <span class="gene">CHS7</span> and CHS8 promoters. We cloned beta-glucuronidase (GUS) under <span class="gene">CHS7</span> and CHS8 promoters and monitored the tissue-specific GUS expression in transformed Arabidopsis. Differential GUS <span class="trait">activity</span> was observed in young leaves, roots, and mature pod walls of transgenic <span class="gene">CHS7</span> promoter-GUS and CHS8 promoter-GUS plants. The tissue-specific expression patterns of <span class="gene">CHS7</span> and CHS8 genes were determined in soybean by quantitative RT-PCR. Both <span class="gene">CHS7</span> and CHS8 genes were expressed at higher levels in roots; however, overall expression pattern of these genes varied in different tissues. The results suggest that the structural diversity within <span class="gene">CHS7</span> and CHS8 promoters may lead into differential activation of these genes by different inducers as well as developmental stage- and tissue-specific differences in gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Higher accumulation of gamma-aminobutyric acid induced by salt stress through stimulating the <span class="trait">activity</span> of diamine oxidases in Glycine max (L.) Merr. roots.</h2>
			<h3>PMID:17624796  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xing SG(1), Jun YB, Hau ZW, Liang LY.
				<br/> 
				年份：2007 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polyamines (PAs) are assumed to perform their functions through their oxidative product such as gamma-aminobutyric acid (GABA) formation. However, there is only limited information on the interrelation between PA degradation and GABA accumulation under salt stress. In order to reveal a quantitative correlation between PA oxidation and GABA accumulation, the effects of treatments with different NaCl concentrations, along with aminoguanidine (AG, a specific inhibitor of diamine oxidases (DAO; EC: 1.4.3.6)) and a recovery test from salt stress on endogenous free PAs, gamma-aminobutyric acid (GABA) accumulation and <span class="gene">DAO</span> <span class="trait">activity</span> were determined in roots of soybean [Glycine max (L.) Merr.] cultivar Suxie-1. The results showed that the levels of putrescine (Put), cadaverine (Cad), and spermidine (Spd) decreased significantly with increasing salt concentrations. This occurred because salt stress strongly promoted <span class="gene">DAO</span> <span class="trait">activity</span> to stimulate PA degradation. GABA accumulation increased with growing NaCl concentrations, about an 11- to 17-fold increase as compared with the control plants. AG treatment increased the accumulation of endogenous free PAs as a result of a strong retardation of <span class="gene">DAO</span> <span class="trait">activity,</span> but decreased GABA accumulation. The recovery for 6 days in 1/2 Hoagland solution from 100mM NaCl stress resulted in a decrease in <span class="gene">DAO</span> <span class="trait">activity,</span> a rebound of PA levels and a simultaneous reduction of GABA content. A close correlation was observed between the changes in <span class="gene">DAO</span> <span class="trait">activity</span> and GABA accumulation. The results indicated that higher GABA accumulation (about 39%) induced by salt stress could come from PA degradation, suggesting that PAs might perform their functions through GABA formation under salt stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Total Glutamine Synthetase <span class="trait">Activity</span> during Soybean Nodule Development Is Controlled at the Level of Transcription and Holoprotein Turnover.</h2>
			<h3>PMID:12226474</h3>
				

				
			<p>
				作者：Temple SJ(1), Kunjibettu S, Roche D, Sengupta-Gopalan C.
				<br/> 
				年份：1996 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Gln synthetase (GS) catalyzes the ATP-dependent condensation of ammonia with glutamate to <span class="trait">yield</span> Gln. In higher plants <span class="gene">GS</span> is an octameric enzyme and the subunits are encoded by members of a small multigene family. In soybeans (Glycine max), following the onset of N2 fixation there is a dramatic increase in <span class="gene">GS</span> <span class="trait">activity</span> in the <span class="trait">root</span> nodules. <span class="gene">GS</span> <span class="trait">activity</span> staining of native polyacrylamide gels containing nodule and <span class="trait">root</span> extracts showed a common band of <span class="trait">activity</span> (GSrs). The nodules also contained a slower-migrating, broad band of enzyme <span class="trait">activity</span> (GSns). The GSns <span class="trait">activity</span> band is a complex of many isozymes made up of different proportions of two kinds of <span class="gene">GS</span> subunits: GSr and GSn. <span class="trait">Root</span> nodules formed following inoculation with an Nif- strain of Bradyrhizobium japonicum showed the presence of <span class="gene">GS</span> isoenzymes (GSns1) with low enzyme <span class="trait">activity,</span> which migrated more slowly than GSns. Gsns1 is most likely made up predominantly of GSn subunits. Our data suggest that, whereas the class I <span class="gene">GS</span> genes encoding the GSr subunits are regulated by the availability of NH3, the class II <span class="gene">GS</span> genes coding for the GSn subunits are developmentally regulated. Furthermore, we have demonstrated that the GSns1 isozymes in the Nif- nodules are relatively more labile. Our overall conclusion is that GSns <span class="trait">activity</span> in soybean nodules is regulated by N2 fixation both at the level of transcription and at the level of holoprotein stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Rhizobial <span class="gene">hemA</span> Gene Is Required for Symbiosis in Species with Deficient [delta]-Aminolevulinic Acid Uptake <span class="trait">Activity.</span></h2>
			<h3>PMID:12228561</h3>
				

				
			<p>
				作者：McGinnis SD(1), O'Brian MR.
				<br/> 
				年份：1995 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most rhizobial <span class="gene">hemA</span> mutants induce <span class="trait">root</span> nodules on their respective legume hosts that lack nitrogen fixation <span class="trait">activity</span> and leghemoglobin expression. However, a Bradyrhizobium japonicum <span class="gene">hemA</span> mutant elicits effective nodules on soybean, and we proposed previously that synthesis and uptake of the heme precursor [delta]-aminolevulinic acid (ALA) by the plant and bacterial symbiont, respectively, allow mutant rescue (I. Sangwan, M.R. O'Brian [1991] Science 251: 1220-1222). In the present work, the B. japonicum <span class="gene">hemA</span> mutant MLG1 elicited normal nodules on three hosts, including cowpea, a plant that is not effectively nodulated by a <span class="gene">hemA</span> mutant of Rhizobium sp. These data indicate that B. japonicum rather than soybean possesses the unique trait that allows normal nodule development by a <span class="gene">hemA</span> mutant. Cowpea expressed glutamate-dependent ALA formation <span class="trait">activity</span> in nodules induced by B. japonicum strains I110 or MLG1 and by Rhizobium sp. ANU240. Exogenous ALA was taken up by B. japonicum bacteroids isolated from soybean or cowpea nodules, and the kinetics of uptake were biphasic. By comparison, Rhizobium sp. ANU240 had very low ALA uptake <span class="trait">activity.</span> In addition, ALA uptake was observed in cultured cells of B. japonicum but not in cultured cells of three other rhizobial species tested. We suggest that the differential success of legume-rhizobial <span class="gene">hemA</span> symbioses is due to an ALA uptake <span class="trait">activity</span> in B. japonicum that is deficient in other rhizobia, thereby further validating the ALA rescue hypothesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Seed Coat Peroxidase (A Comparison of High-Activity and Low-Activity Genotypes).</h2>
			<h3>PMID:12232000</h3>
				

				
			<p>
				作者：Gijzen M(1), Van Huystee R, Buzzell RI.
				<br/> 
				年份：1993 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peroxidase <span class="trait">activity</span> in the seed coats of soybean (Glycine max [L.] Merr.) is controlled by the <span class="gene">Ep</span> locus. We compared peroxidase <span class="trait">activity</span> in cell-free extracts from seed coat, <span class="trait">root,</span> and leaf tissues of three EpEp cultivars (Harosoy 63, Harovinton, and Coles) to three epep cultivars (Steele, Marathon, and Raiden). Extracts from the seed coats of EpEp cultivars were 100-fold higher in specific <span class="trait">activity</span> than those from epep cultivars, but there was no difference in specific <span class="trait">activity</span> in crude <span class="trait">root</span> or leaf extracts. Isoelectric focusing of <span class="trait">root</span> tissue extracts and staining for peroxidase <span class="trait">activity</span> showed that EpEp cultivars had a <span class="trait">root</span> peroxidase of identical isoelectric point to the seed coat peroxidase, whereas roots of the epep types were lacking that peroxidase, indicating that the <span class="gene">Ep</span> locus may also affect expression in the <span class="trait">root.</span> In seed coat extracts, peroxidase was the most abundant soluble protein in EpEp cultivars, whereas this enzyme was present only in trace amounts in epep genotypes, as determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis. Histochemical localization of peroxidase <span class="trait">activity</span> in seed coats of EpEp cultivars shows that the enzyme occurs predominately in the cytoplasm of hourglass cells of the subepidermis. No obvious difference in the gross or microscopic structure of the seed coat was observed to be associated with the <span class="gene">Ep</span> locus. These results suggest that soybean seed coat peroxidase may be involved in processes other than seed coat biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of Rhizobium japonicum <span class="gene">nifH</span> and nifDK operons can be activated by the Klebsiella pneumonia <span class="gene">nifA</span> protein but not by the product of ntrC.</h2>
			<h3>PMID:2862569  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Alvarez-Morales A, Hennecke H.
				<br/> 
				年份：1985
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizobium japonicum nifH'- and nifD'-'lacZ fusions were constructed using the translational fusion vector pMC1403. beta-Galactosidase activities from these fusion plasmids were measured in wild-type, ntrA- and delta(ntrBC) Escherichia coli strains carrying plasmids which overproduced the Klebsiella pneumoniae <span class="gene">nifA</span> or <span class="gene">ntrC</span> gene products. In contrast to results reported in R. meliloti (ref. in the text) neither <span class="gene">nifH</span> nor <span class="gene">nifD</span> promoters were activated by the <span class="gene">ntrC</span> product. In the presence of <span class="gene">nifA</span> gene product, however, beta-galactosidase <span class="trait">activity</span> from both <span class="gene">nifH</span> and <span class="gene">nifD</span> fusion plasmids increased substantially. NifA-mediated activation of these Rhizobium promoters was temperature sensitive and was dependent on the host <span class="gene">ntrA</span> product. In order to determine the point at which the fusion transcripts were initiated, RNA was extracted from the wild-type E. coli strain carrying each of the R. japonicum fusion plasmids plus the <span class="gene">nifA</span> overproducing plasmid. This RNA was used to perform S1 mapping experiments. NifA-mediated transcription from both R. japonicum promoters, began at the same point as previously determined in soybean root-nodule bacteroids (ref. in the text). The results obtained suggest that there may be differences in the mode of regulation between members of the fast- and slow-growing rhizobia. Also, the results of the S1 mapping experiments indicate that activation of the R. japonicum nitrogenase structural genes may be similar to the activation of nif genes in K. pneumoniae thus raising the possibility that R. japonicum may contain <span class="gene">nifA</span> and ntrA-like genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ectopic Expression of <italic> GsPPCK3 </italic> and <italic> SCMRP </italic> in <italic> Medicago sativa </italic> Enhances Plant Alkaline Stress Tolerance and Methionine Content</h2>
			<h3>PMID:24586886</h3>
				
					<h3>PMCID:3934933</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Sun </surname>  <given-names> Mingzhe </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Sun </surname>  <given-names> Xiaoli </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Chaoyue </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> DuanMu </surname>  <given-names> Huizi </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ji </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Yanming </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> a. </bold> The PEPC <span class="trait">activity</span> of the WT and <italic> GsPPCK3-SCMRP </italic> transgenic lines under alkali treatment. <bold> b. </bold> The net photosynthetic rate of the WT and <italic> GsPPCK3-SCMRP </italic> transgenic lines. <bold> c. </bold> The citric acid content of the WT and transgenic lines. <bold> d. </bold> Increased expression levels of <italic> PEPC </italic> in <italic> GsPPCK3-SCMRP </italic> transgenic plants under alkali stress (50 mM NaHCO <sub> 3 </sub> , pH 8.5). <bold> e. </bold> Increased expression levels of <italic> NADP-ME </italic> in <italic> GsPPCK3-SCMRP </italic> transgenic plants. <bold> f. </bold> Increased expression levels of <italic> H <sup> + </sup> -ATPase </italic> in <italic> GsPPCK3-SCMRP </italic> transgenic plants. <bold> g. </bold> Increased expression levels of <italic> CS </italic> in <italic> GsPPCK3-SCMRP </italic> transgenic plants. To explore expression patterns of stress-responsive genes, the 4-week-old WT and <italic> GsPPCK3-SCMRP </italic> transgenic seedlings after shoot cottage were treated with 50 mM NaHCO <sub> 3 </sub> (pH 8.5) for 0, 3, 6, 12 and 24 h, respectively. Relative transcript levels were determined by quantitative real-time PCR with <italic> <span class="gene">GAPDH</span> </italic> as internal reference, and were normalized to WT plants at 0 h. Values represented the means of three fully independent biological replicates, and three technological replicates for each.
					</p>
				
					<p>&nbsp;&nbsp;
						In a previous study, we isolated and characterized one of the alkali stress responsive PPCK genes <italic> GsPPCK1 </italic> , and found that overexpression of <italic> GsPPCK1 </italic> in alfalfa significantly improved plant tolerance to alkali stress <xref rid="pone.0089578-Wei1" ref-type="bibr"> [46] </xref> . In the present study, we focused on the expression pattern and biological function of <italic> GsPPCK3 </italic> . We cloned the full length <italic> GsPPCK3 </italic> , and found that it shared 90%, 84%, 69% and 61% sequence identity with <italic> <span class="gene">GmPPCK3</span> </italic> , <italic> <span class="gene">GmPPCK2</span> </italic> , <italic> <span class="gene">GmPPCK1</span> </italic> and <italic> <span class="gene">GmPPCK4</span> </italic> , respectively. Protein sequence analysis revealed a unique motif consisting of 30 amino acids in subdomain XI of GsPPCK3, which was not found in any soybean and Arabidopsis PPCKs. This unique sequence was encoded by the 90 bp length intron of <italic> <span class="gene">GmPPCK3</span> </italic> , indicating the existence of different transcripts in <italic> Glycine soja </italic> . Furthermore, GsPPCK3 protein contained all conserved subdomains required for kinase <span class="trait">activity</span> <xref rid="pone.0089578-Hanks1" ref-type="bibr"> [24] </xref> , <xref rid="pone.0089578-Yang1" ref-type="bibr"> [25] </xref> ( <xref ref-type="fig" rid="pone-0089578-g002"> Fig. 2 </xref> ), including a protein kinase ATP-binding signature, a Ser/Thr kinase active site signature <xref rid="pone.0089578-Hardie1" ref-type="bibr"> [26] </xref> and a conserved G-T/S-XX-Y/F-X-APE motif, as well as two transmembrane domains.
					</p>
				
					<p>&nbsp;&nbsp;
						It has been reported that PPCK genes showed spatial expression specificity. For example, <italic> OsPPCK1 </italic> and <italic> OsPPCK3 </italic> displayed obviously high expression levels in roots <xref rid="pone.0089578-Fukayama1" ref-type="bibr"> [13] </xref> , while <italic> <span class="gene">GmPPCK2</span> </italic> and <italic> <span class="gene">GmPPCK3</span> </italic> showed high transcription levels in <span class="trait">root</span> nodules <xref rid="pone.0089578-Xu3" ref-type="bibr"> [14] </xref> . Except for <span class="trait">root</span> nodules, <italic> <span class="gene">GmPPCK3</span> </italic> also showed relatively higher expression levels in stems, flowers and young leaves. In this study, we found that <italic> GsPPCK3 </italic> displayed the highest level in leaves but relatively lower in roots ( <xref ref-type="fig" rid="pone-0089578-g003"> Fig. 3B </xref> ). As we know, PEPC undergoes an important function in carbon fixation during photosynthesis, and its <span class="trait">activity</span> was largely regulated by PPCK phosphorylation <xref rid="pone.0089578-Vidala1" ref-type="bibr"> [28] </xref> , <xref rid="pone.0089578-Bakrim1" ref-type="bibr"> [29] </xref> . It is reasonable to speculate that the high levels of <italic> GsPPCK3 </italic> transcripts in leaves could make sure of the high PEPC <span class="trait">activity</span> and thereby the effective photosynthesis of plant leaves. Considering the difference in the transcript levels and alkali induction degrees of <italic> GsPPCK3 </italic> between roots and leaves, one could speculate that in leaves, most <italic> GsPPCK3 </italic> products were used for carbon fixation in photosynthesis process, but in roots, <italic> GsPPCK3 </italic> mainly functioned in alkali response.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Salicylic Acid Alleviates Aluminum Toxicity in Soybean Roots through Modulation of Reactive Oxygen Species Metabolism</h2>
			<h3>PMID:29164108</h3>
				

				
					<h3>PMCID:5681908</h3>
					<h4>期刊:Frontiers in Chemistry</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Ning </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Fengbin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Xiancan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/246678/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> You </surname>  <given-names> Jiangfeng </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Zhenming </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiangnan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/329752/overview"/>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 10 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						As an important signal molecule, salicylic acid (SA) improves plant tolerance to aluminum (Al) stress. The objective of this study was to investigate the effects of exogenous SA application on the dynamics of endogenous SA and reactive oxygen species in soybean (Glycine max L.) exposed to Al stress. The roots of soybean seedlings were exposed to a combination of AlCl3 (30 μM) and SA (10 μM)/PAC (100 μM, paclobutrazol, SA biosynthesis inhibitor) for 3, 6, 9, and 12 h. Al stress induced an increase in endogenous SA concentration in a time-dependent manner, also verified by the up-regulated expression of <span class="gene">GmNPR1,</span> an SA-responsive gene. Al stress increased the activities of phenylalanine ammonia-lyase (PAL) and benzoic acid 2-hydroxylase (BA2H), and the contents of SA, [Formula: see text] and malondialdehyde (MDA) in the <span class="trait">root</span> apex. The application of exogenous SA increased <span class="gene">PAL</span> and BA2H, and reduced [Formula: see text] and MDA contents in soybean roots under Al stress. <span class="gene">PAC</span> inhibited the SA induced increase in BA2H <span class="trait">activity.</span> In addition, the SA application resulted in a rapid increase in hydrogen peroxide (H2O2) concentration under Al stress, followed by a sharp decrease. Compared with the plants exposed to Al alone, Al+SA plants possessed higher activities of superoxide dismutase, peroxidase, and ascorbate peroxidase, and lower catalase <span class="trait">activity,</span> indicating that SA alleviated Al-induced oxidative damage. These results suggested that <span class="gene">PAL</span> and BA2H were involved in Al-induced SA production and showed that SA alleviated the adverse effects of Al toxicity by modulating the cellular H2O2 level and the antioxidant enzyme activities in the soybean <span class="trait">root</span> apex.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The objective of this study was to examine the effect of exogenous SA and of SA synthesis inhibitor paclobutrazol (PAC) on <span class="trait">root</span> elongation rate, and Al content, SA synthesis, membrane injury, and antioxidant system in soybean roots. We hypothesized that (1) exogenous SA would increase activities of <span class="gene">PAL</span> and BA2H and endogenous SA synthesis, thus alleviating the negative effect of Al stress on membrane injury and antioxidant system; (2) The effects of exogenous SA would be limited by adding the exogenous PAC, which act as inhibitor of endogenous SA synthesis.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Root</span> apices (0&#x02013;3 cm) were excised for the <span class="gene">PAL</span> <span class="trait">activity</span> assay, following Ferrarese et al. ( <xref rid="B9" ref-type="bibr"> 2000 </xref> ) with some modifications. Samples (1.0 g) were ground at 4&#x000b0;C in 0.1 M sodium borate buffer (pH 8.8), containing 15 mM &#x003b2;-mercaptoethanol, 5 mM EDTA, 1 mM phenylmethanesulfonyl fluoride, and 0.15% polyvinyl pyrrolidone (w/v). Homogenates were centrifuged (15,000 g, 4&#x000b0;C, 20 min), and the supernatant was used for enzyme preparation. The reaction mixture contained 0.1 M of borate buffer (pH 8.8), 15 mM L-phenylalanine and 1 mL enzyme extract in a 5 mL volume. The reaction proceeded for 1 h at 37&#x000b0;C and was interrupted by the addition of 0.5 mL 6 M HCl. The <span class="trait">activity</span> of <span class="gene">PAL</span> was determined spectrophotometrically at a wavelength of 290 nm.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Root</span> elongation rate <bold> (A) </bold> and Al content <bold> (B) </bold> of 1-cm <span class="trait">root</span> tips of soybean. Four-day-old seedlings were exposed to 0.5 mM CaCl <sub> 2 </sub> solution (pH 4.5) containing 10 &#x003bc;M SA and 100 &#x003bc;M <span class="gene">PAC</span> with or without 30 &#x003bc;M AlCl <sub> 3 </sub> for 12 h. Bars represent means &#x000b1; SD, <italic> n </italic> = 15 for <bold> (A) </bold> and <italic> n </italic> = 9 for <bold> (B) </bold> . Significant differences among the treatments are indicated by different lower case letters ( <italic> p </italic> &#x0003c; 0.05, Tukey's test).
					</p>
				
					<p>&nbsp;&nbsp;
						The alleviating effect of SA on Al-induced inhibition of <span class="trait">root</span> elongation suggested that the inhibition of <span class="trait">root</span> elongation by Al may be caused by the reductions in endogenous SA concentration in soybean roots. To test this hypothesis, the SA biosynthesis inhibitor <span class="gene">PAC</span> was used in the absence and presence of Al and SA. Treatment with 100 &#x003bc;M <span class="gene">PAC</span> for 12 h aggravated Al-induced <span class="trait">root</span> elongation inhibition, which could be reversed by exogenous SA supply (Figure <xref ref-type="fig" rid="F1"> 1A </xref> ). In addition, SA decreased, while <span class="gene">PAC</span> increased Al content in the <span class="trait">root</span> tips of the soybean seedlings under Al stress (Figure <xref ref-type="fig" rid="F1"> 1B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To confirm whether Al-induced inhibition of <span class="trait">root</span> elongation was related to endogenous SA concentration, the concentrations of total SA were investigated at 3-h intervals over a period of 12 h (Figure <xref ref-type="fig" rid="F2"> 2A </xref> ). The effect of SA on Al-induced endogenous SA concentrations in soybean roots was also examined under <span class="gene">PAC</span> treatment (Figure <xref ref-type="fig" rid="F2"> 2B </xref> ). In the presence of Al, both SA and <span class="gene">PAC</span> caused higher SA concentration as compared to the corresponding controls without Al. Exogenous SA markedly enhanced the Al-induced endogenous SA concentrations by 88.7% compared with the Al treatment without SA. However, the application of <span class="gene">PAC</span> decreased the SA content regardless of SA application.
					</p>
				
					<p>&nbsp;&nbsp;
						Endogenous SA production in the <span class="trait">root</span> tips of soybean seedlings. <bold> (A) </bold> Seven-day-old seedlings were exposed to 0 or 30 &#x003bc;M AlCl <sub> 3 </sub> with or without 10 &#x003bc;M SA in 0.5 mM CaCl <sub> 2 </sub> solution (pH 4.5) for 3, 6, 9, and 12 h. Then, <span class="trait">root</span> tips (0&#x02013;3 cm) were excised and analyzed to determine the SA content. <bold> (B) </bold> Seven-day-old seedlings were subjected to AlCl <sub> 3 </sub> (0, 30 &#x003bc;M), SA (0, 10 &#x003bc;M), and <span class="gene">PAC</span> (0, 100 &#x003bc;M) in 0.5 mM CaCl <sub> 2 </sub> solution (pH 4.5) for 12 h. <span class="trait">Root</span> tips (3 cm) were cut, and the SA content was determined. Bars represent means &#x000b1; SD of three individual replicates. Significant differences among the treatments are indicated by different lower case letters ( <italic> p </italic> &#x0003c; 0.05, Tukey's test).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">PAL</span> and BA2H serve as two key enzymes responsible for SA biosynthesis in plants. To determine the contribution of <span class="gene">PAL</span> and BA2H enzymatic pathways to SA production under Al stress, we analyzed the activities of the two enzymes at 3 and 12 h. As shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> , the <span class="gene">PAL</span> <span class="trait">activity</span> was increased after 3 h of Al treatment, while it was decreased slightly at 12 h. Al stress elevated the <span class="trait">activity</span> of BA2H at 3 and 12 h, i.e., by 2.8-fold and 4.7-fold, respectively, in relation to the control. Application of exogenous SA under Al stress further enhanced the <span class="gene">PAL</span> and BA2H activities at 3 and 12 h. It has been reported that <span class="gene">PAC</span> is an effective inhibitor of BA2H during SA biosynthesis. As expected, the <span class="trait">activity</span> of BA2H, but not PAL, was markedly inhibited in the <span class="gene">PAC</span> treatment under Al stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Activities of <span class="gene">PAL</span> <bold> (A) </bold> and BA2H <bold> (B) </bold> in <span class="trait">root</span> apices of soybean under Al stress. Seven-day-old seedlings were exposed to AlCl <sub> 3 </sub> (0, 30 &#x003bc;M), SA (0, 10 &#x003bc;M), and <span class="gene">PAC</span> (0, 100 &#x003bc;M) in 0.5 mM CaCl <sub> 2 </sub> solution (pH 4.5) for 3 and 12 h. <span class="trait">Root</span> tips (3 cm) were collected to determine the activities of <span class="gene">PAL</span> and BA2H. Bars represent means &#x000b1; SD of three individual replicates. Significant differences among the treatments are indicated by different lower case letters ( <italic> p </italic> &#x0003c; 0.05, Tukey's test).
					</p>
				
					<p>&nbsp;&nbsp;
						Relative expression level of the <italic> <span class="gene">GmNPR1</span> </italic> gene in 1-cm <span class="trait">root</span> tips of soybean. Seven-day-old seedlings were transferred to 0.5 mM CaCl <sub> 2 </sub> solution (pH 4.5) containing AlCl <sub> 3 </sub> (0, 30 &#x003bc;M) and SA (0, 10 &#x003bc;M) for 3, 6, 9, and 12 h. Total RNA was extracted from eight primary <span class="trait">root</span> tips. The expression levels of RNA were measured using qRT-PCR. For normalization of gene expression, the constitutively expressed &#x003b2; <italic> -tubulin </italic> gene of soybean was used as a reference gene. Significant differences among the times and treatments are indicated by different upper and lower case letters respectively ( <italic> p </italic> &#x0003c; 0.05, Tukey's test).
					</p>
				
					<p>&nbsp;&nbsp;
						The first hypothesis about exogenous SA effect was well supported by our results. It was found that exogenous Al caused obviously toxicity on <span class="trait">root,</span> i.e., it significantly decreased the <span class="trait">root</span> elongation rate and increased the Al content inside it; however, exogenous SA significantly ameliorated Al-induced inhibition of <span class="trait">root</span> elongation and reduced Al accumulation. Metal ion stress can enhance the SA level in a range of plant species (Metwally et al., <xref rid="B31" ref-type="bibr"> 2003 </xref> ; Yao et al., <xref rid="B54" ref-type="bibr"> 2012 </xref> ; Guan et al., <xref rid="B11" ref-type="bibr"> 2015 </xref> ). The first hypothesis was also supported by the effects of exogenous SA on the accumulation of endogenous SA under both conditions, i.e., with and without Al stress. Additionally, exogenous SA continuously increased endogenous SA content throughout the 12-h experiment, which was consistent with the results shown in a study by Wang et al. ( <xref rid="B50" ref-type="bibr"> 2004 </xref> ), who found that SA increased progressively up to 4-fold higher levels at the end of a 12 h exposure to Al in roots of <italic> C. tora </italic> . Exogenous SA had a synergistic and additive effect on endogenous SA under Al stress. These results indicated that changes in endogenous SA concentration in the <span class="trait">root</span> tips of soybean were closely associated with Al tolerance. Similar to the positive effects on endogenous SA content, exogenous SA also significantly increased <span class="gene">PAL</span> and BA2H activities, suggesting that endogenous SA content was strongly related to <span class="gene">PAL</span> and BA2H activities. Interestingly, without Al stress, exogenous SA did not show significant effects on <inline-formula> <mml:math id="M8"> <mml:msubsup> <mml:mrow> <mml:mtext> O </mml:mtext> </mml:mrow> <mml:mrow> <mml:mn> 2 </mml:mn> </mml:mrow> <mml:mrow> <mml:mo> - </mml:mo> </mml:mrow> </mml:msubsup> </mml:math> </inline-formula> and MDA contents, but significantly decreased their contents under Al stress. This may be because exogenous SA increased endogenous SA content under Al toxicity, then decreased the <inline-formula> <mml:math id="M9"> <mml:msubsup> <mml:mrow> <mml:mtext> O </mml:mtext> </mml:mrow> <mml:mrow> <mml:mn> 2 </mml:mn> </mml:mrow> <mml:mrow> <mml:mo> - </mml:mo> </mml:mrow> </mml:msubsup> </mml:math> </inline-formula> and MDA contents.
					</p>
				
					<p>&nbsp;&nbsp;
						The second hypothesis was also well supported by our results, i.e., that the addition of exogenous <span class="gene">PAC</span> brought about opposite effects as compared to exogenous SA supply, especially under Al stress. In fact, the addition of the SA biosynthesis inhibitor <span class="gene">PAC</span> exacerbated the Al-induced damage, leading to lower <span class="trait">root</span> elongation rates accompanied by a higher <span class="trait">root</span> Al content, thus again confirming the alleviation effect of Al toxicity by SA and especially by exogenous SA. Accordingly, under Al stress, the addition of <span class="gene">PAC</span> reduced the ameliorating effects caused by the administration of exogenous SA, reducing the levels of endogenous SA contents and also the BA2H <span class="trait">activity.</span> This strongly suggested that <span class="gene">PAC</span> may impair endogenous SA synthesis by specifically inhibiting BA2H <span class="trait">activity.</span> Consistent with this, it has been shown that <span class="gene">PAC</span> pre-treatment resulted in the disappearance of a conjugated SA peak in heat-acclimated pea leaves (Liu et al., <xref rid="B28" ref-type="bibr"> 2006 </xref> ). Some pharmacological and biochemical studies have demonstrated that <span class="gene">PAC</span> is an effective inhibitor of BA2H and is widely used as an inhibitor of SA biosynthesis (Ghai et al., <xref rid="B10" ref-type="bibr"> 2002 </xref> ; Zhou and Zhong, <xref rid="B56" ref-type="bibr"> 2011 </xref> ; Li et al., <xref rid="B26" ref-type="bibr"> 2015 </xref> ). In the present study, it was found that the inhibition of BA2H <span class="trait">activity</span> by <span class="gene">PAC</span> effectively blocked SA accumulation. Similarly, exogenous <span class="gene">PAC</span> also limited the effects of exogenous SA by increasing H <sub> 2 </sub> O <sub> 2 </sub> , <inline-formula> <mml:math id="M10"> <mml:msubsup> <mml:mrow> <mml:mtext> O </mml:mtext> </mml:mrow> <mml:mrow> <mml:mn> 2 </mml:mn> </mml:mrow> <mml:mrow> <mml:mo> - </mml:mo> </mml:mrow> </mml:msubsup> </mml:math> </inline-formula> and MDA contents.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results showed that endogenous SA content and the <span class="gene">PAL</span> and BA2H enzyme activities were enhanced by exogenous SA, while they were inhibited by exogenous <span class="gene">PAC</span> under Al stress. PAL, a pivotal enzyme, is involved in defense mechanisms in plants. It is not only responsible for SA and phenolic acid synthesis, but also lignification of cell walls. Ou-yang et al. ( <xref rid="B36" ref-type="bibr"> 2014 </xref> ) reported that <span class="gene">PAL</span> activities increased initially and then decreased with increasing Al concentration in cotyledons and radicles of <italic> Jatropha curcas </italic> seedlings. Al stress up-regulated SA-mediated <italic> <span class="gene">PAL</span> </italic> gene expression of potato during pathogen attack (Arasimowicz-Jelonek et al., <xref rid="B1" ref-type="bibr"> 2014 </xref> ). BA2H catalyzes the final step, i.e., the conversion of benzoic acid to SA, which acts as a switch that directly controls the synthesis of SA (Jayakannan et al., <xref rid="B20" ref-type="bibr"> 2015 </xref> ). The results that <span class="gene">PAC</span> treatment can aggravate Al-inhibited <span class="trait">root</span> elongation and reduce SA concentration in the <span class="trait">root</span> tips under Al stress, provide the evidence to explain the changes in the endogenous SA concentration caused by <span class="gene">PAL</span> and BA2H activities, which was associated with the Al toxicity syndrome in soybean (Liu et al., <xref rid="B29" ref-type="bibr"> 2012 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, the exogenous SA induced a transient increase in H <sub> 2 </sub> O <sub> 2 </sub> at 6 h after Al stress, then the H <sub> 2 </sub> O <sub> 2 </sub> content decreased with the elongation of treatment duration (9&#x02013;12 h). This finding showed that SA-induced an H <sub> 2 </sub> O <sub> 2 </sub> peak during the early stage of Al stress, suggesting that H <sub> 2 </sub> O <sub> 2 </sub> might play an important role in regulating soybean tolerance to Al stress. A rapid H <sub> 2 </sub> O <sub> 2 </sub> accumulation, also known as the oxidative burst, serves as a redox signal to regulate the plant tolerance to biotic and abiotic stresses (Hossain et al., <xref rid="B18" ref-type="bibr"> 2015 </xref> ; Saxena et al., <xref rid="B42" ref-type="bibr"> 2016 </xref> ). It was demonstrated that SA can cooperate with H <sub> 2 </sub> O <sub> 2 </sub> to form a self-amplifying feedback loop in SA-mediated defense responses (Vlot et al., <xref rid="B49" ref-type="bibr"> 2009 </xref> ; Miura and Tada, <xref rid="B33" ref-type="bibr"> 2014 </xref> ). Chao et al. ( <xref rid="B4" ref-type="bibr"> 2010 </xref> ) found that SA mediated H <sub> 2 </sub> O <sub> 2 </sub> accumulation and helped rice plants withstand Cd toxicity. Dynamics of H <sub> 2 </sub> O <sub> 2 </sub> content are triggered by SA under Al stress and are related to various antioxidant enzymes, such as, POD, SOD, APX, and CAT. The CAT serves as a key scavenging enzyme for H <sub> 2 </sub> O <sub> 2 </sub> (Willekens et al., <xref rid="B51" ref-type="bibr"> 1997 </xref> ; Lin et al., <xref rid="B27" ref-type="bibr"> 2012 </xref> ; Mhamdi et al., <xref rid="B32" ref-type="bibr"> 2012 </xref> ) <sub> . </sub> In the present study, SA decreased CAT <span class="trait">activity</span> under Al stress. However, SA and Al increased the activities of POD, SOD, and APX during the treatment. Therefore, the H <sub> 2 </sub> O <sub> 2 </sub> content was decreased under Al stress. SA treatment alleviated Al-induced oxidative stress as shown by the decrease in the content of MDA and <inline-formula> <mml:math id="M11"> <mml:msubsup> <mml:mrow> <mml:mtext> O </mml:mtext> </mml:mrow> <mml:mrow> <mml:mn> 2 </mml:mn> </mml:mrow> <mml:mrow> <mml:mo> - </mml:mo> </mml:mrow> </mml:msubsup> </mml:math> </inline-formula> . In rice, SA pretreatment enhanced Cd tolerance through elevated enzymatic and non-enzymatic antioxidants and the concentrations of glutathione and nonprotein sulfhydryl in roots and shoots, hence leading to alleviation of oxidative damage (Guo et al., <xref rid="B12" ref-type="bibr"> 2009 </xref> ). Our result was different from the results in a study on <italic> C. tora </italic> plants subjected to Al stress, where SA induced changes in H <sub> 2 </sub> O <sub> 2 </sub> levels by increasing the <span class="trait">activity</span> of POD, whereas the activities of CAT, APX, and <span class="gene">GR</span> remained unchanged (Wang et al., <xref rid="B50" ref-type="bibr"> 2004 </xref> ). SA suppressed the Al-induced enhancement in SOD, GPX, and APX activities but alleviated the Al-induced reduction in CAT <span class="trait">activity</span> in rice (Pandey et al., <xref rid="B37" ref-type="bibr"> 2013 </xref> ). In tomato, SA significantly reversed the Al-induced changes in the activities of SOD, POD, and CAT (Surapu et al., <xref rid="B47" ref-type="bibr"> 2014 </xref> ). These discrepancies may be caused by the different research methods, the time course of measurements and plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, it is suggested that Al stress induced a remarkable increase in endogenous SA levels in soybean <span class="trait">root</span> tips via the <span class="gene">PAL</span> and BA2H enzymatic pathways. SA treatment regulated the expression of the <italic> <span class="gene">GmNPR1</span> </italic> gene under Al stress. SA induced H <sub> 2 </sub> O <sub> 2 </sub> , which functions as a signal and activates antioxidant systems to remove the excess H <sub> 2 </sub> O <sub> 2 </sub> ; therefore, oxidative damage trigged by Al stress was partly alleviated.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A novel soybean (Glycine max) gene encoding a family 3 β-glucosidase has high isoflavone 7-O-glucoside-hydrolyzing <span class="trait">activity</span> in transgenic rice.</h2>
			<h3>PMID:25569564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hsu CC(1), Wu TM, Hsu YT, Wu CW, Hong CY, Su NW.
				<br/> 
				年份：2015 Jan 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A previous study demonstrated that purified Glycine max β-glucosidase <span class="gene">(GmBGL)</span> could hydrolyze glucosyl isoflavone to the aglyconic form. This study reports the cloning and functional characterization of a soybean cDNA encoding the β-glucosidase. <span class="gene">GmBGL</span> was isolated by use of a purified soybean N-terminal amino acid sequence and conserved sequences of β-glucosidase genes from other plants. Sequence analysis of <span class="gene">GmBGL</span> revealed an open reading frame of 1884 bp encoding a polypeptide of 627 amino acids with a calculated molecular mass of 69 kDa. Phylogenetic analysis classified the <span class="gene">GmBGL</span> into the glycosyl hydrolase 3 family. In soybean, the <span class="gene">GmBGL</span> transcript was predominantly accumulated in roots and leaves. To examine the enzymatic <span class="trait">activity</span> and substrate specificity, <span class="gene">GmBGL</span> was ectopically expressed in transgenic rice. Purified <span class="gene">GmBGL</span> protein from transgenic rice could catalyze the hydrolysis of genistin and daidzin to produce genistein and daidzein, respectively, which confirmed <span class="gene">GmBGL</span> as a functional β-glucosidase with isoflavone glucoside-hydrolyzing <span class="trait">activity.</span> This paper reveals that <span class="gene">GmBGL</span> is a key enzyme in transforming glucosyl isoflavones to aglycones in soybean, which may help in genetic manipulation of aglycone-rich soybean seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean oil biosynthesis: role of diacylglycerol acyltransferases.</h2>
			<h3>PMID:23322364  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li R(1), Hatanaka T, Yu K, Wu Y, Fukushige H, Hildebrand D.
				<br/> 
				年份：2013 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Diacylglycerol acyltransferase (DGAT) catalyzes the acyl-CoA-dependent acylation of sn-1,2-diacylglycerol to form seed oil triacylglycerol (TAG). To understand the features of genes encoding soybean (Glycine max) DGATs and possible roles in soybean seed oil synthesis and accumulation, two full-length cDNAs encoding type 1 diacylglycerol acyltransferases <span class="gene">(GmDGAT1A</span> and <span class="gene">GmDGAT1B)</span> were cloned from developing soybean seeds. These coding sequences share identities of 94 % and 95 % in protein and DNA sequences. The genomic architectures of <span class="gene">GmDGAT1A</span> and <span class="gene">GmDGAT1B</span> both contain 15 introns and 16 exons. Differences in the lengths of the first exon and most of the introns were found between <span class="gene">GmDGAT1A</span> and <span class="gene">GmDGAT1B</span> genomic sequences. Furthermore, detailed in silico analysis revealed a third predicted DGAT1, <span class="gene">GmDGAT1C.</span> <span class="gene">GmDGAT1A</span> and <span class="gene">GmDGAT1B</span> were found to have similar <span class="trait">activity</span> levels and substrate specificities. Oleoyl-CoA and sn-1,2-diacylglycerol were preferred substrates over vernoloyl-CoA and sn-1,2-divernoloylglycerol. Both transcripts are much more abundant in developing seeds than in other tissues including leaves, stem, roots, and flowers. Both soybean <span class="gene">DGAT1A</span> and <span class="gene">DGAT1B</span> are highly expressed at developing seed stages of maximal TAG accumulation with <span class="gene">DGAT1B</span> showing highest expression at somewhat later stages than DGAT1A. <span class="gene">DGAT1A</span> and <span class="gene">DGAT1B</span> show expression profiles consistent with important roles in soybean seed oil biosynthesis and accumulation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a soybean protein that interacts with GAGA element dinucleotide repeat DNA.</h2>
			<h3>PMID:12177492  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sangwan I(1), O'Brian MR.
				<br/> 
				年份：2002 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Dinucleotide repeat DNA with the pattern (GA)(n)/(TC)(n), so-called GAGA elements, control gene expression in animals, and are recognized by a specific regulatory protein. Here, a yeast one-hybrid screen was used to isolate soybean (Glycine max) cDNA encoding a GAGA-binding protein (GBP) that binds to (GA)(n)/(CT)(n) DNA. Soybean <span class="gene">GBP</span> was dissimilar from the GAGA factor of Drosophila melanogaster. Recombinant <span class="gene">GBP</span> protein did not bind to dinucleotide repeat sequences other than (GA)(n)/(CT)(n). <span class="gene">GBP</span> bound to the promoter of the heme and chlorophyll synthesis gene Gsa1, which contains a GAGA element. Removal of that GAGA element abrogated binding of <span class="gene">GBP</span> to the promoter. Furthermore, insertion of the GAGA element to a nonspecific DNA conferred GBP-binding <span class="trait">activity</span> on that DNA. Thus, the GAGA element of the Gsa1 promoter is both necessary and sufficient for <span class="gene">GBP</span> binding. Gbp mRNA was expressed in leaves and was induced in symbiotic <span class="trait">root</span> nodules elicited by the bacterium Bradyrhizobium japonicum. In addition, Gbp transcripts were much higher in leaves of dark-treated etiolated plantlets than in those exposed to light for 24 h. Homologs of <span class="gene">GBP</span> were found in other dicots and in the monocot rice (Oryza sativa), as well. We suggest that interaction between GAGA elements and GBP-like proteins is a regulatory feature in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Oxidative turnover of soybean <span class="trait">root</span> glutamine synthetase. In vitro and in vivo studies</h2>
			<h3>PMID:10198108</h3>
				

				
			<p>
				作者：Ortega JL(1), Roche D, Sengupta-Gopalan C.
				<br/> 
				年份：1999 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glutamine synthetase (GS) is the key enzyme in ammonia assimilation and catalyzes the ATP-dependent condensation of NH3 with glutamate to produce glutamine. <span class="gene">GS</span> in plants is an octameric enzyme. Recent work from our laboratory suggests that <span class="gene">GS</span> <span class="trait">activity</span> in plants may be regulated at the level of protein turnover (S.J. Temple, T.J. Knight, P.J. Unkefer, C. Sengupta-Gopalan [1993] Mol Gen Genet 236: 315-325; S.J. Temple, S. Kunjibettu, D. Roche, C. Sengupta-Gopalan [1996] Plant Physiol 112: 1723-1733; S.J. Temple, C. Sengupta-Gopalan [1997] In C.H. Foyer, W.P. Quick, eds, A Molecular Approach to Primary Metabolism in Higher Plants. Taylor & Francis, London, pp 155-177). Oxidative modification of <span class="gene">GS</span> has been implicated as the first step in the turnover of <span class="gene">GS</span> in bacteria. By incubating soybean (Glycine max) <span class="trait">root</span> extract enriched in <span class="gene">GS</span> in a metal-catalyzed oxidation system to produce the.OH radical, we have shown that <span class="gene">GS</span> is oxidized and that oxidized <span class="gene">GS</span> is inactive and more susceptible to degradation than nonoxidized GS. Histidine and cysteine protect <span class="gene">GS</span> from metal-catalyzed inactivation, indicating that oxidation modifies the <span class="gene">GS</span> active site and that cysteine and histidine residues are the site of modification. Similarly, ATP and particularly ATP/glutamate give the enzyme the greatest protection against oxidative inactivation. The roots of plants fed ammonium nitrate showed a 3-fold increase in the level of <span class="gene">GS</span> polypeptides and <span class="trait">activity</span> compared with plants not fed ammonium nitrate but without a corresponding increase in the <span class="gene">GS</span> transcript level. This would suggest either translational or posttranslational control of <span class="gene">GS</span> levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conserved regulation of the soybean early nodulin <span class="gene">ENOD2</span> gene promoter in determine and indeterminate transgenic <span class="trait">root</span> nodules.</h2>
			<h3>PMID:8220455  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lauridsen P(1), Franssen H, Stougaard J, Bisseling T, Marcker KA.
				<br/> 
				年份：1993 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The beta-glucuronidase (GUS) <span class="trait">activity</span> expressed from the soybean early nodulin ENOD2(B) gene promoter was localized histochemically in nodules of Lotus corniculatus and Trifolium repens. In both the determinate Lotus nodules and the indeterminate Trifolium nodules, <span class="trait">activity</span> was found in the parenchyma cells and especially in cells close to the vascular tissue of nodules. The characteristic cell-specific expression of the soybean <span class="gene">ENOD2</span> gene was therefore maintained by the ENOD2(B) promoter in the two developmentally different nodule types. Important DNA elements recognized in transgenic nodules were identified by deletion and hybrid promoter analysis in Lotus corniculatus. An indispensable positive element (PE) and a possible tissue specific element was defined between positions -1792 and -1582 from the transcription start site. Another qualitative control element located between -380 and -53 conferred the <span class="gene">ENOD2</span> characteristic cell type expression on hybrid promoters. This element contains the conserved nodulin gene sequences CTCTT and AAAGAT. In contrast to the ENOD2(B) promoter a chimeric leghemoglobin Ibc3-GUS gene was expressed in the infected cells of both types of nodules. In the indeterminate nodules expression was restricted to the interzone II-III and the active nitrogen-fixing zone III. Interchange of the distal strong positive element (SPE) of Ibc3 and the <span class="gene">ENOD2</span> positive element resulted in an expression pattern different from that observed for the Ibc3 and <span class="gene">ENOD2</span> genes, indicating that different interactions of trans-acting factors are required for regulation of early as well as late nodulin genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Pathogenesis-Related Class 10 Protein <italic> Gly m 4l, </italic> Increases <span class="trait">Resistance</span> upon <italic> Phytophthora sojae </italic> Infection in Soybean ( <italic> Glycine max </italic> [L.] Merr.)</h2>
			<h3>PMID:26474489</h3>
				
					<h3>PMCID:4608668</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Junjiang </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						RNA degradation assays were used to examine whether the recombinant <span class="gene">GmPru</span> ar 1 protein displayed RNase <span class="trait">activity</span> according to the method of Zubini et al. 2009 [ <xref rid="pone.0140364.ref048" ref-type="bibr"> 48 </xref> ]. The samples containing 10 &#x003bc;g total soybean RNA were incubated with the presence of 10 &#x003bc;g Gly m 4l protein at 37&#x000b0;C for 0, 2 and 4 h in 50 &#x003bc;L of 50 mM buffer at different pHs (citrate for pH 3, MES for pH 5, phosphate for pH 7, and CHES (N-cyclohexyl-2-amino-ethanesulfonic acid) for pH 9). At pH 7, RNase <span class="trait">activity</span> of the recombinant Gly m 4l protein was also checked in the presence of zeatin. Samples containing 10 &#x003bc;g total soybean RNA were incubated with 10 &#x003bc;g recombinant Gly m 4l protein and 10 mM zeatin at 37&#x000b0;C for 30 min, 1, 2, 3 and 4 h. The samples incubated with the recombinant Gly m 4l protein alone or boiled Gly m 4l protein and zeatin were used as the negative controls. The result was examined by 1% agarose gel electrophoresis (AGE) then visualized under UV light.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) The transcript abundance of <italic> Gly m 4l </italic> in the <span class="trait">root,</span> stem and leaf. The Ct value of each sample was normalized to the Ct value of <italic> <span class="gene">GmEF1B</span> </italic> . (B) The transcript abundance of <italic> Gly m 4l </italic> in response to <italic> P </italic> . <italic> sojae </italic> infection. The Ct value of each sample was normalized to the Ct value of <italic> <span class="gene">GmActin4</span> </italic> , and the relative expression of <italic> Gly m 4l </italic> was compared with mock plants at the same time point. (C) and (D), The transcript abundance of <italic> Gly m 4l </italic> in response to various stresses. Exogenous chemicals are SA (0.5 mM), MeJA (100 &#x003bc;M), ET (0.2 mM ethephon), ABA (50 mM), GA <sub> 3 </sub> (50 mg.L <sup> -1 </sup> ), NaCl (100 mM), PEG (20%), and cold (4&#x000b0;C). The Ct value of each sample was normalized to the Ct value of <italic> <span class="gene">GmActin4</span> </italic> and the relative expression of <italic> Gly m 4l </italic> was compared with mock plants at the same time point. The experiments were performed on three biological replicates with their respective three technical replicates and statistically analysed using Student&#x02019;s t-test (*P&#x0003c;0.05, **P&#x0003c;0.01). Bars indicate standard error of the mean (SE).
					</p>
				
					<p>&nbsp;&nbsp;
						Until recently, a variety of allergens have been identified into eight families in soybean ( <ext-link ext-link-type="uri" xlink:href="http://www.allergen.org/"> http://www.allergen.org </ext-link> ), namely Gly m 1, Gly m 2, Gly m 3, Gly m 4, Gly m 5, Gly m 6, Gly m 7 and Gly m 8. Gly m 4 is a member of the superfamily of PR10 proteins [ <xref rid="pone.0140364.ref053" ref-type="bibr"> 53 </xref> &#x02013; <xref rid="pone.0140364.ref055" ref-type="bibr"> 55 </xref> ], but there is almost no study on the disease <span class="trait">resistance</span> of this protein. Although soybean PR10 genes were first identified in 1992 [ <xref rid="pone.0140364.ref071" ref-type="bibr"> 71 </xref> ], there is limited information concerning the potential functions of these proteins <italic> in vitro </italic> and <italic> in vivo </italic> . In a previous study, we functionally characterized <italic> <span class="gene">GmPR10</span> </italic> (GenBank accession no. FJ960440) in soybean [ <xref rid="pone.0140364.ref014" ref-type="bibr"> 14 </xref> ]. In the present study, we provide the first description of Gly m 4l in soybean ( <italic> Glycine max </italic> [L.] Merr.), showing identities of 53.46% homology with Gly m 4 protein, a member of the PR10 family, containing a Bet v1-motif and P-loop motif. We also isolated and characterized the PR10 member Gly m 4l in response to pathogen ( <italic> P </italic> . <italic> sojae </italic> ) infection in soybean plants.The amino acid sequence, structure, subcellular location and other biological functions of PR10 proteins indicated that the major members of the PR10 family can be grouped into two different classes: intracellular pathogenesis-related proteins (IPR) with homology to ribonucleases [ <xref rid="pone.0140364.ref001" ref-type="bibr"> 1 </xref> ] and (S)-norcoclaurine synthases (NCS) [ <xref rid="pone.0140364.ref072" ref-type="bibr"> 72 </xref> ]. The sequence analysis indicated that Gly m 4l contained no signal peptide and the subcellular localization analysis showed that Gly m 4l protein was localized at cell membrane, suggesting that Gly m 4l is an intracellular protein similar to IPRs of the PR10 family. Gly m 4l also shared several conserved features of known IPR PR10 proteins, such as a small molecular mass, acidic pI and an intron of 76&#x02013;359 nucleotides [ <xref rid="pone.0140364.ref008" ref-type="bibr"> 8 </xref> , <xref rid="pone.0140364.ref073" ref-type="bibr"> 73 </xref> ]. Most <italic> PR10 </italic> genes are clustered in chromosomes [ <xref rid="pone.0140364.ref008" ref-type="bibr"> 8 </xref> , <xref rid="pone.0140364.ref014" ref-type="bibr"> 14 </xref> ], and the analysis of <italic> Gly m 4l </italic> based on data obtained from the internet ( <ext-link ext-link-type="uri" xlink:href="http://soybase.org/GlycineBlastPages/"> http://soybase.org/GlycineBlastPages/ </ext-link> ) indicated that a total of nine genes were clustered into nine linkage groups, namely Gm 01, 07, 08, 09, 10, 14, 15, 17, and 18.
					</p>
				
					<p>&nbsp;&nbsp;
						Members of PR10 family have been reported that they have antimicrobial activities in transgenic plants, as examples, <italic> VpPR10 </italic> . <italic> 2 </italic> gene from <italic> Vitis pseudoreticulata </italic> responded to <italic> Plasmopara viticola </italic> infection and over-expression of <italic> VpPR10 </italic> . <italic> 2 </italic> in the host plant enhanced <span class="trait">resistance</span> to <italic> Plasmopara viticola </italic> [ <xref rid="pone.0140364.ref065" ref-type="bibr"> 65 </xref> ], the over-expression of <italic> PgPR10-2 </italic> and <italic> PgPR10-4 </italic> conferred a tolerance against fungal infection [ <xref rid="pone.0140364.ref069" ref-type="bibr"> 69 </xref> , <xref rid="pone.0140364.ref082" ref-type="bibr"> 82 </xref> ], and over-expression of <italic> <span class="gene">GmPR10</span> </italic> in soybean enhanced <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> [ <xref rid="pone.0140364.ref014" ref-type="bibr"> 14 </xref> ]. In the present study, <italic> Gly m 4l </italic> over-expression transgenic soybean plants enhanced <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae in vivo </italic> and showed a lower biomass of <italic> P </italic> . <italic> sojae </italic> than that in non-transgenic soybean plants, suggesting that Gly m 4l could inhibit <italic> P </italic> . <italic> sojae </italic> infection and might be a useful tool for managing Phytophthora <span class="trait">root</span> and stem rot in soybean plants.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The intron and 5' distal region of the soybean <span class="gene">Gmubi</span> promoter contribute to very high levels of gene expression in transiently and stably transformed tissues.</h2>
			<h3>PMID:25292438  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：De La Torre CM(1), Finer JJ.
				<br/> 
				年份：2015 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: An extended version of an intron-containing soybean polyubiquitin promoter gave very high levels of gene expression using three different validation tools. The intron-containing Glycine max polyubiquitin promoter <span class="gene">(Gmubi)</span> is able to regulate expression levels five times higher than the widely used CaMV35S promoter. In this study, eleven <span class="gene">Gmubi</span> derivatives were designed and evaluated to determine which regions contributed to the high levels of gene expression, observed with this promoter. Derivative constructs regulating GFP were evaluated using transient expression in lima bean cotyledons and stable expression in soybean hairy roots. With both expression systems, removal of the intron in the 5'UTR led to reduced levels of gene expression suggesting a role of the intron in promoter <span class="trait">activity.</span> Promoter constructs containing an internal intron duplication and upstream translocations of the intron resulted in higher and similar expression levels to <span class="gene">Gmubi,</span> respectively, indicating the presence of enhancers within the intron. Evaluation of 5' distal extensions of the <span class="gene">Gmubi</span> promoter resulted in significantly higher levels of GFP expression, suggesting the presence of upstream regulatory elements. A twofold increase in promoter strength was obtained when <span class="gene">Gmubi</span> was extended 1.5 kb upstream to generate <span class="gene">GmubiXL</span> (2.4 kb total length). In stably transformed soybean plants containing GFP regulated by CaMV35S, <span class="gene">Gmubi</span> and <span class="gene">GmubiXL,</span> the <span class="gene">GmubiXL</span> promoter clearly produced the highest levels of gene expression, with especially high GFP fluorescence in the vascular tissue and <span class="trait">root</span> tips. Use of <span class="gene">GmubiXL</span> leads to very high levels of gene expression in soybean and represents a native soybean promoter, which may be useful for regulating transgene expression for both basic and applied research.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Functional Characterization of the Soybean <italic> <span class="gene">GmaPPO12</span> </italic> Promoter Conferring <italic> Phytophthora sojae </italic> Induced Expression</h2>
			<h3>PMID:23840763</h3>
				

				
					<h3>PMCID:3695865</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Chunyue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Yanling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Danyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Yuren </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hongjuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dou </surname>  <given-names> Daolong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 5 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Identification of pathogen-inducible promoters largely lags behind cloning of the genes for disease <span class="trait">resistance.</span> Here, we cloned the soybean <span class="gene">GmaPPO12</span> gene and found that it was rapidly and strongly induced by Phytophthorasojae infection. Computational analysis revealed that its promoter contained many known cis-elements, including several defense related transcriptional factor-binding boxes. We showed that the promoter could mediate induction of GUS expression upon infection in both transient expression assays in Nicotianabenthamiana and stable transgenic soybean hairy roots. Importantly, we demonstrated that pathogen-induced expression of the <span class="gene">GmaPPO12</span> promoter was higher than that of the soybean <span class="gene">GmaPR1a</span> promoter. A progressive 5' and 3' deletion analysis revealed two fragments that were essential for promoter <span class="trait">activity.</span> Thus, the cloned promoter could be used in transgenic plants to enhance <span class="trait">resistance</span> to phytophthora pathogens, and the identified fragment could serve as a candidate to produce synthetic pathogen-induced promoters.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A 1,500-bp fragment harboring the -1500 to +1 region (transcription start site of <italic> <span class="gene">GmaPPO12</span> </italic> as +1) was isolated from soybean as the candidate pathogen-induced promoter. This fragment ( <italic> <span class="gene">GmaPPO12</span> </italic> promoter) was submitted to PlantCARE [ <xref ref-type="bibr" rid="B28"> 28 </xref> ] to detect the <italic> cis </italic> -regulatory elements. Several elements that could be classified into three groups were identified within this promoter ( <xref ref-type="supplementary-material" rid="pone.0067670.s003"> Figure S2 </xref> ). The first group is the basal regulatory elements, including nine CAAT-box elements [ <xref ref-type="bibr" rid="B29"> 29 </xref> ]. The second group includes the well studied pathogen-inducible <italic> cis </italic> -acting elements, including six GT-1 elements [ <xref ref-type="bibr" rid="B30"> 30 </xref> ], two W-box elements [ <xref ref-type="bibr" rid="B31"> 31 </xref> ], three WRKY-binding site WRKY elements [ <xref ref-type="bibr" rid="B30"> 30 </xref> ], eight R response MYC binding sites [ <xref ref-type="bibr" rid="B32"> 32 </xref> ], one regulatory defense-related gene CACGTG motif [ <xref ref-type="bibr" rid="B33"> 33 </xref> ]. The third group is involved in abiotic stress or hormone response, such as the two GA-responsive GARE elements [ <xref ref-type="bibr" rid="B34"> 34 </xref> ], one central element of gibberellin response MYB [ <xref ref-type="bibr" rid="B35"> 35 </xref> ], one auxin response factor (ARF) binding sequence SURE [ <xref ref-type="bibr" rid="B36"> 36 </xref> ], three water stress responsive elements MYB2 elements [ <xref ref-type="bibr" rid="B37"> 37 </xref> ], one transcriptional activator ARR1AT [ <xref ref-type="bibr" rid="B38"> 38 </xref> ], three abscisic acid (ABA) -responsive complex RYR elements [ <xref ref-type="bibr" rid="B39"> 39 </xref> ], four ABA or stress responsive MYB1AT elements [ <xref ref-type="bibr" rid="B40"> 40 </xref> ], one light-activated transcription binding site T-box [ <xref ref-type="bibr" rid="B41"> 41 </xref> ], and one plant bZIP protein binding site SUREP [ <xref ref-type="bibr" rid="B42"> 42 </xref> ]. In conclusion, based on the predictive identification of putative <italic> cis </italic> -elements, we hypothesize that the transcriptional <span class="trait">activity</span> of <italic> <span class="gene">GmaPPO12</span> </italic> is induced by different biotic and abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						B. Enzymatic assay of GUS <span class="trait">activity</span> in expanded 6-week-old <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves. The leaves were infiltrated with <named-content content-type="genus-species"> Agrobacterium </named-content> harboring <italic> <span class="gene">GmaPPO12</span> </italic> or <italic> <span class="gene">GmaPR1a</span> </italic> promoter: GUS constructs, and then submerged in <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> zoospores 3 days after agroinfiltration. GUS <span class="trait">activity</span> levels were measured by &#x0fb02;uorimetric assays in protein extracts from treated leaves at 0 hpi, 0.5 hpi and 2 hpi. Each column represents the fold change of <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> induction at 0.5 hpi and 2 hpi to 0 hpi and normalized to 35S min. Labels are as follows: <italic> <span class="gene">GmaPR1a</span> </italic> promoter; <italic> <span class="gene">GmaPPO12</span> </italic> promoter; 35S minimal promoter; 35S full-length promoter fragments. Experiments were in triplicate and error bars show the standard error.
					</p>
				
					<p>&nbsp;&nbsp;
						C. Histochemical GUS staining of <italic> <span class="gene">GmaPPO12</span> </italic> and <italic> <span class="gene">GmaPR1a</span> </italic> : GUS constructs in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves. The leaves were treated as in B. GUS <span class="trait">activity</span> increased dramatically at <italic> <span class="gene">GmaPPO12</span> </italic> and <italic> <span class="gene">GmaPR1a</span> </italic> construct agroinfiltrated regions at 0.5 hpi and 2 hpi. No visible GUS <span class="trait">activity</span> was noted at 35S min agroinfiltrated regions. GUS <span class="trait">activity</span> was relatively strong at 35S (CaMV 35S full-length promoter fragments) agroinfiltrated regions.
					</p>
				
					<p>&nbsp;&nbsp;
						We examined induction <span class="trait">activity</span> upon inoculation with <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> zoospores after the <named-content content-type="genus-species"> Agrobacterium </named-content> strains were infiltrated into <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves. Quantitative GUS enzyme <span class="trait">activity</span> assays demonstrated that <italic> <span class="gene">GmaPPO12</span> </italic> showed almost 8.2- and 10.8-fold induction, while <italic> <span class="gene">PR1a</span> </italic> showed 2.1- and 4.6-fold induction at 0.5 and 2 hpi ( <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3B) </xref> , respectively. Histochemical staining revealed that <italic> <span class="gene">GmaPPO12</span> </italic> showed stronger induction than <italic> <span class="gene">PR1a</span> </italic> by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> , although untreated tobacco leaves containing <italic> <span class="gene">GmaPPO12</span> </italic> or <italic> <span class="gene">PR1a</span> </italic> showed weak background expression ( <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3C) </xref> . To further confirm the results, we used quantitative RT-PCR to measure GUS mRNA levels and found that the transcript expression of <italic> <span class="gene">GmaPPO12</span> </italic> was 6.1- and 9.5-fold at 0.5 and 2 hpi, and <italic> <span class="gene">PR1a</span> </italic> was 2.2- and 4.1-fold higher compared to uninfected tobacco leaves ( <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3D) </xref> , respectively. As expected, the leaves infiltrated with negative control (the construct only containing 35S minimal promoter) showed very low levels of GUS <span class="trait">activity</span> in all the three independent assays, while a full length 35S promoter fused to GUS exhibited high levels regardless of the treatment ( <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3B-C) </xref> . Importantly, the induction levels of GUS by <italic> <span class="gene">GmaPPO12</span> </italic> promoters were similar to the 35S promoter ( <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3B-C) </xref> , suggesting that the promoter is strongly and rapidly induced upon pathogen infection.
					</p>
				
					<p>&nbsp;&nbsp;
						To independently validate that <italic> <span class="gene">GmaPPO12</span> </italic> is an immediate-early <italic> Phytophthora </italic> infection gene, the vectors were introduced into soybean hairy roots by <named-content content-type="genus-species"> Agrobacterium </named-content> -mediated transformation. The multiple soybean hairy roots were inoculated with <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> zoospores, and analyzed by histochemical staining, enzymatic assay, and quantitative RT-PCR. Enzymatic assay showed that <italic> <span class="gene">GmaPPO12</span> </italic> was induced 6.3- and 10.8-fold and <italic> <span class="gene">PR1a</span> </italic> was induced 1.6- and 3.0-fold at 0.5 and 2 hpi ( <xref ref-type="fig" rid="pone-0067670-g004"> Figure 4A) </xref> , respectively. The histochemical staining shows that the GUS reporter <span class="trait">activity</span> driven by <italic> <span class="gene">GmaPPO12</span> </italic> was significantly higher than that of <italic> <span class="gene">PR1a</span> </italic> and non-treatment with <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> (0 hpi) in transgenic hairy roots ( <xref ref-type="fig" rid="pone-0067670-g004"> Figure 4B) </xref> . we also used quantitative RT-PCR to measure GUS mRNA levels and found that the transcript expression of <italic> <span class="gene">GmaPPO12</span> </italic> was 4.8- and 5.2-fold increases at 0.5 and 2 hpi, respectively, compared to that of <italic> <span class="gene">PR1a</span> </italic> with 1.4- and 2.8-fold increases at 0.5 and 2 hpi ( <xref ref-type="fig" rid="pone-0067670-g004"> Figure 4C) </xref> , respectively. These results demonstrate that the <italic> <span class="gene">GmaPPO12</span> </italic> promoter, compared to the <italic> <span class="gene">PR1a</span> </italic> promoter, was more strongly induced upon infection in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						B. Histochemical staining of GUS <span class="trait">activity</span> in transgenic soybean hairy roots. Roots were treated as in A. No visible GUS <span class="trait">activity</span> in 35S min transgenic soybean hairy roots; GUS <span class="trait">activity</span> was relatively strong in 35S (CaMV 35S full-length promoter fragments) transgenic soybean hairy roots; GUS <span class="trait">activity</span> increased dramatically in <italic> <span class="gene">GmaPPO12</span> </italic> transgenic soybean hairy roots at 0.5 hpi and 2 hpi.
					</p>
				
					<p>&nbsp;&nbsp;
						To locate the pathogen-responsive <italic> cis </italic> -regulatory regions, a series of progressive 5&#x02019; or 3&#x02019; truncated <italic> <span class="gene">GmaPPO12</span> </italic> promoters were fused to a minimal 35S promoter and the <italic> GUS </italic> reporter gene ( <xref ref-type="fig" rid="pone-0067670-g005"> Figure 5A-S3) </xref> . In <named-content content-type="genus-species"> Agrobacterium </named-content> -mediated transient expression of <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves, the potential expression patterns and pathogen-inducible activities of those promoter mutations were evaluated by histochemical staining and enzymatic assay after treatment with <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> zoospores for 2 h. Truncation of the promoter to -525 (NT2) significantly reduced GUS enzyme activities ( <xref ref-type="fig" rid="pone-0067670-g005"> Figure 5B-C) </xref> . Deletions beyond this position to -179 (NT4) resulted in no detectable GUS <span class="trait">activity</span> and transcription levels. Truncation of the promoter to -1012 (NT1) reduced the GUS enzyme <span class="trait">activity</span> to the same levels as the native <italic> <span class="gene">GmaPR1a</span> </italic> promoter ( <xref ref-type="fig" rid="pone-0067670-g005"> Figure 5B-C) </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						C. Histochemical GUS staining of different constructs in <named-content content-type="genus-species"> Nicotiana </named-content> transiently expressing leaves at 0 hpi and 2 hpi. The leaves were treated as in B. No visible induced GUS <span class="trait">activity</span> was detected at agroinfiltrated regions, including NT2, NT3, NT4, NT5, 35S min; GUS <span class="trait">activity</span> increased dramatically at the agroinfiltrated regions, including <italic> <span class="gene">GmaPPO12</span> </italic> , NT1, M1, <italic> <span class="gene">GmaPR1a</span> </italic> ; GUS <span class="trait">activity</span> was continuously high at agroinfiltrated regions, including 35S (CaMV 35S full-length promoter fragments).
					</p>
				
					<p>&nbsp;&nbsp;
						A series of progressive 3&#x02019; deletions of <span class="gene">Gma</span> <italic> PPO12 </italic> were evaluated. The deletion of the promoter sequence up to -1087 (CT1) reduced the GUS enzyme <span class="trait">activity</span> to around half of the native promoter. For other four progressive deletions up to -751 (CT2), -507 (CT3), -325 (CT4) and -174 (CT5), the GUS enzyme <span class="trait">activity</span> were significantly reduced (p &#x0003c; 0.05) ( <xref ref-type="fig" rid="pone-0067670-g006"> Figure 6A) </xref> . These results were also confirmed by a histochemical staining assay ( <xref ref-type="fig" rid="pone-0067670-g006"> Figure 6B) </xref> . Thus, 5&#x02019; and 3&#x02019; deletions together suggest that the fragment from -1012 to -413 is essential for promoter <span class="trait">activity,</span> although the fragment from -1500 to -1012 is partially required in response to pathogen inoculation.
					</p>
				
					<p>&nbsp;&nbsp;
						A. Enzymatic assay of C-terminal deletion constructs in <named-content content-type="genus-species"> Nicotiana </named-content> transiently expressing leaves. The expanded 6-week-old <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves were infiltrated with <named-content content-type="genus-species"> Agrobacterium </named-content> harboring constructs and inoculated with <named-content content-type="genus-species"> <named-content content-type="genus-species"> <named-content content-type="genus"> Phytophthora </named-content> <named-content content-type="species"> capsici </named-content> </named-content> </named-content> zoospores 3 days after infiltration. Enzymatic assay at 0 hpi and 2 hpi. Each column represents the fold change in <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> induction at 2 hpi compared to 0 hpi and normalized to 35S min. Labels are as follows: <italic> <span class="gene">GmaPPO12</span> </italic> promoter; C-terminal deletion mutants: CT1 (-1500 to -413), CT2 (-1500 to -746), CT3 (-1500 to -993), CT4 (-1500 to -1175), CT5 (-1500 to -1326); <italic> <span class="gene">GmaPR1a</span> </italic> promoter. Experiments were performed in triplicate. Means that have different letters at the top are significantly different (p &#x0003c; 0.05 Duncan&#x02019;s multiple range tests), lines designated with the same letter exhibit no significant difference of the GUS enzyme <span class="trait">activity.</span> Error bars show the standard error.
					</p>
				
					<p>&nbsp;&nbsp;
						B. Histochemical GUS staining of different constructs in <named-content content-type="genus-species"> Nicotiana </named-content> transiently expressing leaves at 0 hpi and 2 hpi. Leaves were treated as in A. No visibly induced GUS <span class="trait">activity</span> was detected at agroinfiltrated regions, including 35S min; GUS <span class="trait">activity</span> was relatively weak at agroinfiltrated regions, including CT4, CT5; GUS <span class="trait">activity</span> increased dramatically at agroinfiltrated regions, including CT1, CT2, CT3, M1, <italic> <span class="gene">GmaPPO12,</span> <span class="gene">GmaPR1a</span> </italic> ; GUS <span class="trait">activity</span> was continuously high at agroinfiltrated regions, including 35S (CaMV 35S full-length promoter fragments).
					</p>
				
					<p>&nbsp;&nbsp;
						B. Histochemical GUS staining of different constructs in <named-content content-type="genus-species"> Nicotiana </named-content> transiently expressing leaves at 0 hpi and 2 hpi. Leaves were treated as in A. No visible induced GUS <span class="trait">activity</span> was detected at the injection region in 35S min. GUS <span class="trait">activity</span> was relatively weak at the injection region in M1-113. GUS <span class="trait">activity</span> was increased dramatically at the injection region of <italic> <span class="gene">GmaPPO12</span> </italic> . GUS <span class="trait">activity</span> was continuously high at agroinfiltrated regions for 35S (CaMV 35S full-length promoter fragments). Experiments were done in triplicate with error bars showing the standard error.
					</p>
				
					<p>&nbsp;&nbsp;
						We are unaware of pathogen-inducible promoters have been identified and characterized in soybeans. Here, we screened the published microarray data [ <xref ref-type="bibr" rid="B25"> 25 </xref> ] and found that the <italic> <span class="gene">GmaPPO</span> </italic> 12 gene is strongly induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> . We elucidated its expression pattern by quantitative RT-PCR analysis in soybeans, transient expression in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> , and stable expression in soybean hairy roots. We found that the gene had three expressional characterizations. First, it is an immediate-early pathogen response gene because it can be induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection at 0.5 h in soybean, and its promoter also can mediate fast induction <span class="trait">activity</span> in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> capsici </named-content> </italic> </named-content> . Second, its induction is strong. The expression levels upon infection are higher than that of the <italic> <span class="gene">GmaPR1a</span> </italic> gene in soybean. Third, the expression is specific to infection. It cannot be induced by the tested hormones in soybean, and its background expression is almost undetectable in soybean and <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> . This promoter could be used in transgenic plants to enhance their <span class="trait">resistance</span> to <italic> Phytophthora </italic> and perhaps other oomycete diseases. Finally, we identified a small responsive region by progressive 5&#x02019; and 3&#x02019; deletions. The small functional segment could serve as a candidate to further identify novel pathogen responsive <italic> cis </italic> -elements or to produce synthetic pathogen-inducible promoters.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene is predicted to encode a member of the polyphenol oxidase (PPO) family that catalyzes the oxidation of phenolics to quinones [ <xref ref-type="bibr" rid="B26"> 26 </xref> , <xref ref-type="bibr" rid="B45"> 45 </xref> ]. In green plants, the <italic> PPO </italic> gene family is highly variable in both gene number and structure, except in the genus <named-content content-type="genus-species"> Arabidopsis </named-content> [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. The enzymes play many divergent roles in a variety of processes, including defense processes [ <xref ref-type="bibr" rid="B46"> 46 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]. In tomatoes, for example, PPO overexpression increased <span class="trait">resistance</span> to <italic> <named-content content-type="genus-species"> Pseudomonas syringae </named-content> </italic> and several insects [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. PPOs are also implicated in latex coagulation [ <xref ref-type="bibr" rid="B48"> 48 </xref> ], and the biosynthesis of aurones and lignans [ <xref ref-type="bibr" rid="B49"> 49 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Soybean contains 11 PPO isoforms [ <xref ref-type="bibr" rid="B26"> 26 </xref> ], which correlate with its abundant flavonoids. Here, we showed that <italic> <span class="gene">GmaPPO12</span> </italic> was rapidly and strongly induced by <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection. However, the role of <italic> <span class="gene">GmaPPO12</span> </italic> in disease <span class="trait">resistance</span> still needs to be addressed.
					</p>
				
					<p>&nbsp;&nbsp;
						Patterns of gene expression in soybean upon <named-content content-type="genus-species"> <named-content content-type="genus"> P </named-content> <italic> . <named-content content-type="species"> sojae </named-content> </italic> </named-content> infection have been investigated in several independent studies [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> , <xref ref-type="bibr" rid="B52"> 52 </xref> ]. Only 22 soybean genes were identified as upregulated early in infection (3 hrs). The genes included <italic> <span class="gene">GmaPR1a</span> </italic> and a gene encoding flavin adenine dinucleotide (FAD)-linked oxidoreductase [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Here, we identified <italic> <span class="gene">GmaPPO12</span> </italic> as being strongly inducing at 0.5 h postinfection. Its fold change was higher than that of <italic> <span class="gene">GmaPR1a</span> </italic> . Characterization of the <italic> <span class="gene">GmaPR1a</span> </italic> promoter in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> leaves and soybean hairy roots also supported the idea that the gene was rapidly and highly induced by <italic> Phytophthora </italic> pathogens. Moreover, the fold change in <italic> <span class="gene">GmaPPO12</span> </italic> promoter driven GUS expression was lower than that of the endogenous gene in soybean. Quantitative RT-PCR analysis showed that the gene was upregulated 53-fold at 0.5 hpi in soybean, but the <italic> GUS </italic> gene was only induced 6.1-fold in <named-content content-type="genus-species"> <named-content content-type="genus"> N </named-content> <italic> . <named-content content-type="species"> benthamiana </named-content> </italic> </named-content> and 4.8-fold in soybean hairy roots, respectively. This difference may be due to that the stress-induced expression of the endogenous gene driven by the native promoter not being similar to the heterologous <italic> GUS </italic> gene fused with the same promoter, as observed in previous studies [ <xref ref-type="bibr" rid="B53"> 53 </xref> ]. We have noted that the promoter driving GUS expression exhibited unwanted background expression, although the levels were low. One possible reason is that the agroinfiltration associated wounding induced gene expression. Further studies will be required to test the utilization of the cloned promoter to drive functional disease <span class="trait">resistance</span> genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Promoter deletion analysis revealed that two fragments appeared to be partially sufficient for the induction <span class="trait">activity</span> of the <italic> <span class="gene">GmaPPO12</span> </italic> promoter. The 599-bp and 113-bp fragments accounted for over 60% and 40%, respectively compared to <span class="trait">activity</span> of the full length promoter. The identified smaller fragment contained several putative <italic> cis </italic> -elements that are involved in defense responses, including two MYB binding sites, three MYC binding sites, and one DPBF-box. The MYB binding sites have been identified in the parsley <italic> <span class="gene">PAL1</span> </italic> promoter as the sites of fungal elicitor-inducible DNA&#x02013;protein interactions [ <xref ref-type="bibr" rid="B54"> 54 </xref> ], and the tobacco <italic> <span class="gene">PR1a</span> </italic> promoter [ <xref ref-type="bibr" rid="B55"> 55 </xref> ]. The MYC recognition sites play a critical role in MYC regulation of innate immune responses against mycobacterial infection [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]. Testing these elements from soybeans is now a priority so that we can design the synthetic promoters that are specific and highly induced by pathogens.
					</p>
				
					<p>&nbsp;&nbsp;
						Genomic DNA was extracted from 6&#x02013;7-day-old soybean roots, following the cetyltrimethylammonium bromide (CTAB) isolation procedure [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. The <italic> <span class="gene">GmaPPO12</span> </italic> and <italic> <span class="gene">GmaPR1a</span> </italic> promoters were isolated with the designed primers (Table. S1). The obtained promoters cover a sequence 1,500-bp in length upstream of the transcription start site. Five 5&#x02019; deletions, five 3&#x02019; deletions, and two 5&#x02019; and 3&#x02019; deletions of the <italic> <span class="gene">GmaPPO12</span> </italic> promoter region were generated by PCR, with forward primers containing a <italic> Kpn </italic> I restriction site and a reverse primer containing a <italic> Bgl </italic> II restriction site. The sequences of promoters used for the amplification of the deletion promoter fragments are summarized in <xref ref-type="supplementary-material" rid="pone.0067670.s004"> Figure S3 </xref> . All of the promoter fragments were digested with <italic> Kpn </italic> I and <italic> Bgl </italic> II and cloned into the <italic> Kpn </italic> I- <italic> Bgl </italic> II sites of pMDC162 [ <xref ref-type="bibr" rid="B59"> 59 </xref> ], a plant expression vector fused with the CaMV 35S minimal promoter [ <xref ref-type="bibr" rid="B60"> 60 </xref> ] and <italic> GUS </italic> reporter gene. A full description of plasmid construction can be found in <xref ref-type="fig" rid="pone-0067670-g003"> Figure 3A </xref> . The pMDC162 vector containing the CaMV 35S minimal promoter lacking <italic> cis </italic> -acting elements was constructed and used as a negative control. PBI121 contains the <italic> GUS </italic> (or <italic> uidA </italic> ) gene under the control of the CaMV 35S promoter as the positive control. Both plasmids contain the selectable marker gene <italic> nptII </italic> , which confers <span class="trait">resistance</span> to kanamycin. All primers used in this study are listed in Supplemental Table. 1. All plasmids containing expression cassettes were confirmed by sequence analysis and introduced into <named-content content-type="genus-species"> Agrobacterium </named-content> strains by electroporation for transformation.
					</p>
				
					<p>&nbsp;&nbsp;
						The introduction of expression constructs into soybean hairy roots using the <named-content content-type="genus-species"> <named-content content-type="genus"> Agrobacterium </named-content> <italic> <named-content content-type="species"> rhizogene </named-content> </italic> </named-content> K599 cucumopine strain [ <xref ref-type="bibr" rid="B63"> 63 </xref> ] was by electroporation as described by Ainsworth et al [ <xref ref-type="bibr" rid="B61"> 61 </xref> ]. Individual colonies were picked and verified by enzyme restriction analysis of DNA extracted from randomly selected clones. Plant inoculation was conducted according to Savka [ <xref ref-type="bibr" rid="B63"> 63 </xref> ] and modified as follows: soybean seeds were surface-sterilized with chlorine gas and germinated on Murashige and Skoog (MS) culture medium for 5-6 days in a 16: 8-h light/dark cycle incubator at 25&#x000b0;C. Cotyledons were wounded on the abaxial side with scalpel blades dipped in a culture of <named-content content-type="genus-species"> <named-content content-type="genus"> Agrobacterium </named-content> <italic> <named-content content-type="species"> rhizogenes </named-content> </italic> </named-content> K599 harboring the different constructs and cultured abaxial-side-up on <span class="gene">MS</span> culture medium. Carbenicillin and Cefotetan 250 &#x000b5;g/mL were added to inhibit the growth of <named-content content-type="genus-species"> <named-content content-type="genus"> A </named-content> <italic> . <named-content content-type="species"> rhizogenes </named-content> </italic> </named-content> and incubated at 25&#x000b0;C in the dark. The putative transformed hairy roots about 18-20 days after <span class="trait">root</span> emergence and at lengths of 4- to 6-cm were used for inoculations.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:14-3-3 proteins regulate the intracellular localization of the transcriptional activator <span class="gene">GmMYB176</span> and affect isoflavonoid synthesis in soybean.</h2>
			<h3>PMID:22404168  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li X(1), Chen L, Dhaubhadel S.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavonoids are legume-specific natural plant compounds that play important functions in nitrogen fixation as well as biotic and abiotic stress responses. Many clinical studies have suggested a role for isoflavonoids in human health and nutrition. We have recently identified an R1 MYB transcription factor <span class="gene">GmMYB176</span> that regulates CHS8 gene expression and isoflavonoid biosynthesis. Here we demonstrate that binding of 14-3-3 proteins to <span class="gene">GmMYB176</span> modulates this function. <span class="gene">GmMYB176</span> interacts with all 16 14-3-3 proteins (SGF14s) in soybean (Glycine max) with varying <span class="trait">activity.</span> The detailed analysis of 14-3-3-binding sites within <span class="gene">GmMYB176</span> identified a critical motif (D2) where Ser29 is potentially phosphorylated. Deletion of the D2 motif from <span class="gene">GmMYB176</span> or substitution of Ser29 with an alanine abolished binding with SGF14 proteins, which altered the subcellular localization of <span class="gene">GmMYB176.</span> Overexpression of SGF14l in soybean hairy roots did not affect the transcript level of <span class="gene">GmMYB176</span> but it reduced the expression levels of key isoflavonoid genes and isoflavonoid accumulation in soybean hairy <span class="trait">root.</span> Our results suggest that SGF14-GmMYB176 interaction regulates the intracellular localization of <span class="gene">GmMYB176,</span> thereby affecting isoflavonoid biosynthesis in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phylogeny of nodulation and nitrogen-fixation genes in Bradyrhizobium: supporting evidence for the theory of monophyletic origin, and spread and maintenance by both horizontal and vertical transfer.</h2>
			<h3>PMID:21357454  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Menna P(1), Hungria M.
				<br/> 
				年份：2011 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacteria belonging to the genus Bradyrhizobium are capable of establishing symbiotic relationships with a broad range of plants belonging to the three subfamilies of the family Leguminosae (=Fabaceae), with the formation of specialized structures on the roots called nodules, where fixation of atmospheric nitrogen takes place. Symbiosis is under the control of finely tuned expression of common and host-specific nodulation genes and also of genes related to the assembly and <span class="trait">activity</span> of the nitrogenase, which, in Bradyrhizobium strains investigated so far, are clustered in a symbiotic island. Information about the diversity of these genes is essential to improve our current poor understanding of their origin, spread and maintenance and, in this study, we provide information on 40 Bradyrhizobium strains, mostly of tropical origin. For the nodulation trait, common (nodA), Bradyrhizobium-specific (nodY/K) and host-specific (nodZ) nodulation genes were studied, whereas for fixation ability, the diversity of <span class="gene">nifH</span> was investigated. In general, clustering of strains in all nod and <span class="gene">nifH</span> trees was similar and the Bradyrhizobium group could be clearly separated from other rhizobial genera. However, the congruence of nod and nif genes with ribosomal and housekeeping genes was low. <span class="gene">nodA</span> and nodY/K were not detected in three strains by amplification or hybridization with probes using Bradyrhizobium japonicum and Bradyrhizobium elkanii type strains, indicating the high diversity of these genes or that strains other than photosynthetic Bradyrhizobium must have alternative mechanisms to initiate the process of nodulation. For a large group of strains, the high diversity of nod genes (with an emphasis on nodZ), the low relationship between nod genes and the host legume, and some evidence of horizontal gene transfer might indicate strategies to increase host range. On the other hand, in a group of five symbionts of Acacia mearnsii, the high congruence between nod and ribosomal/housekeeping genes, in addition to shorter nodY/K sequences and the absence of nodZ, highlights a co-evolution process. Additionally, in a group of B. japonicum strains that were symbionts of soybean, vertical transfer seemed to represent the main genetic event. In conclusion, clustering of <span class="gene">nodA</span> and <span class="gene">nifH</span> gives additional support to the theory of monophyletic origin of the symbiotic genes in Bradyrhizobium and, in addition to the analysis of nodY/K and nodZ, indicates spread and maintenance of nod and nif genes through both vertical and horizontal transmission, apparently with the dominance of one or other of these events in some groups of strains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Signaling LTPs: a new plant LTPs sub-family?</h2>
			<h3>PMID:20404561  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pii Y(1), Pandolfini T(1), Crimi M(2).
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Numerous plant non specific Lipid Transfer Protein (nsLTPs) have been characterized for their antimicrobial <span class="trait">activity,</span> suggesting for these proteins a direct role in the protection against pathogenic microorganisms. Another group of LTPs seems to be involved in structural events in the extracellular matrix through binding and transport of hydrophobic molecules. More recently, some LTPs putatively involved in the symbiotic interaction between legumes and rhizobia have been identified. We investigated the role of MtN5, a <span class="gene">LTP</span> from Medicago truncatula, which is specifically expressed in the roots and induced by both Sinorhizobium meliloti and a <span class="trait">root</span> pathogenic fungus. Once the symbiosis has been established, MtN5 is preferentially accumulated in the <span class="trait">root</span> nodule. The suppression of MtN5 transcript, obtained by means of an RNAi approach, resulted in a reduced nodulation, whereas its overexpression led to an increased number of nodules produced by S. meliloti. These observations demonstrated that MtN5 is required for an efficient nodulation. On the basis of the amino acid sequence, MtN5 has been included in the nsLTP-like sub-family, together with Arabidopsis thaliana DIR1, a protein playing a role in SAR signaling. The putative role(s) for this <span class="gene">LTP</span> in the symbiotic association are discussed in the present commentary. MtN5, together with DIR1 and other new LTPs, are proposed to form a new <span class="gene">LTP</span> subfamily involved in lipid signalling.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:1-Aminocyclopropane-1-carboxylic acid (ACC) concentration and ACC synthase expression in soybean roots, <span class="trait">root</span> tips, and soybean cyst nematode (Heterodera glycines)-infected roots.</h2>
			<h3>PMID:19861652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Xue P, Yang R.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Colonization of plant roots by <span class="trait">root</span> knot and cyst nematodes requires a functional ethylene response pathway. However, ethylene plays many roles in <span class="trait">root</span> development and whether its role in nematode colonization is direct or indirect, for example lateral <span class="trait">root</span> initiation or <span class="trait">root</span> hair growth, is not known. The temporal requirement for ethylene and localized synthesis of ethylene during the life span of soybean cyst nematode (SCN) on soybean roots was further investigated. Although a significant increase in ethylene evolution was not detected from SCN-colonized roots, the concentration of the immediate precursor to ethylene, 1-aminocyclopropane-1-carboxylic acid (ACC), was higher in SCN-colonized <span class="trait">root</span> pieces and <span class="trait">root</span> tips than in other parts of the <span class="trait">root.</span> Moreover, expression analysis of 17 ACC synthase (ACS) genes indicated that a select set of <span class="gene">ACS</span> genes is expressed in SCN-colonized <span class="trait">root</span> pieces that is clearly different from the set of genes expressed in non-colonized roots or <span class="trait">root</span> tips. Semi-quantitative real-time PCR indicated that <span class="gene">ACS</span> transcript accumulation correlates with the high concentration of ACC in <span class="trait">root</span> tips. In addition, an ACS-like sequence was found in the public SCN nucleotide database. Acquisition of a full-length sequence for this mRNA (accession GQ389647) and alignment with transcripts for other well-characterized <span class="gene">ACS</span> proteins indicated that the nematode sequence is missing a key element required for <span class="gene">ACS</span> <span class="trait">activity</span> and therefore probably is not a functional ACS. Moreover, no significant amount of ACC was found in any growth stage of SCN that was tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Light-responsive subtilisin-related protease in soybean seedling leaves.</h2>
			<h3>PMID:15283128  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Barnaby NG(1), He F, Liu X, Wilson KA, Wilson KA, Tan-Wilson A.
				<br/> 
				年份：2004 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Protease <span class="gene">C1</span> (E.C. 3.4.21.25), the soybean (Glycine max L. Merrill) proteolytic enzyme responsible for initiating the degradation of soybean storage proteins in seedling cotyledons appears at even higher levels in seedling leaves. This was manifested at the mRNA level through northern blot analysis, at the protein level through western blot analysis, through determination of enzyme <span class="trait">activity,</span> and also through isolation and partial sequencing of active leaf enzyme. Comparison of cDNA and amino acid sequences, as well as characterization of enzyme <span class="trait">activity,</span> is consistent with the leaf enzyme being identical to or highly similar to the cotyledon enzyme. Protease <span class="gene">C1</span> mRNA and protein are also present in stems of soybean seedlings, but is very low to absent in the roots. This presence in the aerial tissues is consistent with the higher steady state level of gene expression at both the mRNA and protein levels when the seedlings are grown in a 12-h light: 12-h dark photoperiod as compared to seedlings grown in continuous darkness. Transfer of dark-grown seedlings to light is followed by marked elevation in protease <span class="gene">C1</span> protein as seen in western blots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of nir, nor and nos denitrification genes from Bradyrhizobium japonicum in soybean <span class="trait">root</span> nodules.</h2>
			<h3>PMID:15032854</h3>
				

				
			<p>
				作者：Mesa S(1), Alché Jd, Bedmar E, Delgado MJ.
				<br/> 
				年份：2004 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Expression of Bradyrhizobium japonicum wild-type strain USDA110 nirK, <span class="gene">norC</span> and <span class="gene">nosZ</span> denitrification genes in soybean <span class="trait">root</span> nodules was studied by in situ histochemical detection of beta-galactosidase <span class="trait">activity.</span> Similarly, P(nirK)-lacZ, P(norC)-lacZ, and P(nosZ)-lacZ fusions were also expressed in bacteroids isolated from <span class="trait">root</span> nodules. Levels of beta-galactosidase <span class="trait">activity</span> were similar in both bacteroids and nodule sections from plants that were solely N(2)-dependent or grown in the presence of 4 mM KNO(3). These findings suggest that oxygen, and not nitrate, is the main factor controlling expression of denitrification genes in soybean nodules. In plants not amended with nitrate, B. japonicum mutant strains GRK308, GRC131, and GRZ25, that were altered in the structural nirK, <span class="gene">norC</span> and <span class="gene">nosZ</span> genes, respectively, showed a wild-type phenotype with regard to nodule number and nodule dry weight as well as plant dry weight and nitrogen content. In the presence of 4 mM KNO(3), plants inoculated with either GRK308 or GRC131 showed less nodules, and lower plant dry weight and nitrogen content, relative to those of strains USDA110 and GRZ25. Taken together, the present results revealed that although not essential for nitrogen fixation, mutation of either the structural <span class="gene">nirK</span> or <span class="gene">norC</span> genes encoding respiratory nitrite reductase and nitric oxide reductase, respectively, confers B. japonicum reduced ability for nodulation in soybean plants grown with nitrate. Furthermore, because nodules formed by each the parental and mutant strains exhibited nitrogenase <span class="trait">activity,</span> it is possible that denitrification enzymes play a role in nodule formation rather than in nodule function.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A pyramid of loci for partial <span class="trait">resistance</span> to Fusarium solani f. sp. glycines maintains Myo-inositol-1-phosphate synthase expression in soybean roots.</h2>
			<h3>PMID:12582889</h3>
				

				
			<p>
				作者：Iqbal J(1), Afzal J, Yaegashi S, Ruben E, Triwitayakorn K, Njiti N, Ahsan R, Wood J, Lightfoot A.
				<br/> 
				年份：2002 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Myo-inositol 1-phosphate synthase (MIPS; EC 5.5.1.4) converts glucose 6-phosphate to myo-inositol 1-phosphate in the presence of NAD(+). It catalyzes the first step in the synthesis of myo-inositol and pinitol, and is a rate limiting step in the de novo biosynthesis of inositol in eukaryotes. Therefore, <span class="gene">MIPS</span> is involved in biotic and abiotic stress via Ca(2+) signalling. Seedlings of four soybean genotypes were inoculated with Fusarium solani f. sp. glycines, the causative agent of sudden death syndrome (SDS), and differentially abundant mRNAs were identified by differential display. The genotypes carried either zero, two, four or six alleles of the quantitative trait loci (QTLs) that control <span class="trait">resistance</span> to SDS in an additive manner. The mRNA abundance of <span class="gene">MIPS</span> did not decrease following inoculation in a recombinant inbred line (RIL 23) containing all six <span class="trait">resistance</span> alleles of the QTLs conferring <span class="trait">resistance</span> to SDS of soybean. However, the abundance of <span class="gene">MIPS</span> mRNA was decreased in genotypes containing four, two or no <span class="trait">resistance</span> alleles. The specific <span class="trait">activity</span> of the <span class="gene">MIPS</span> enzyme in vitro followed the same pattern across genotypes. The IP(3) content in the inoculated roots of genotypes with two, four or six <span class="trait">resistance</span> alleles were higher compared to the non-inoculated <span class="trait">root.</span> The results suggests that a non-additive effect on transcription and translation of <span class="gene">MIPS</span> is established in RIL 23 roots by pyramiding six QTLs for <span class="trait">resistance</span> to SDS. A role of <span class="gene">MIPS</span> in the partial <span class="trait">resistance</span> or response of soybean roots to F. solani infection is suggested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Short-term metabolic responses of soybean <span class="trait">root</span> nodules to nitrate.</h2>
			<h3>PMID:11847240  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gordon AJ(1), Skøt L, James CL, Minchin FR.
				<br/> 
				年份：2002 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max L. Merr.) plants exposed to 10 mM KNO(3) for a 4 d period were used to test the correlation between nitrogenase <span class="trait">activity,</span> gene expression and sucrose metabolism. Nitrate caused the down-regulation of sucrose synthase (SS) transcripts within 1 d, although a decline in nodule <span class="gene">SS</span> <span class="trait">activity</span> and an increase in nodule sucrose content only occurred after 3-4 d. In a second experiment, plants were exposed to (15)N-labelled nitrate for 48 h to determine the time period during which nitrate was taken up, and to relate this to the decline in apparent nitrogenase <span class="trait">activity</span> (H(2) production in air) and the reduction in <span class="gene">SS</span> gene transcript levels. The peak of nitrate uptake appeared to be between 8 h and 14 h whilst apparent nitrogenase <span class="trait">activity</span> began to decline at about 17.5 h. The <span class="gene">SS</span> mRNA signal declined markedly between 14 h and 24 h. The correlative association of these factors is clear. However, <span class="gene">SS</span> <span class="trait">activity</span> per se does not appear to be related to the initial decline in apparent nitrogenase <span class="trait">activity</span> as a result of nitrate uptake. These findings, therefore, do not support the hypothesis that the regulation of nodule function is mediated by the regulation of <span class="gene">SS</span> <span class="trait">activity.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic characterization and regulation of the <span class="gene">nolA</span> gene of Bradyrhizobium japonicum.</h2>
			<h3>PMID:8810078  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Garcia M(1), Dunlap J, Loh J, Stacey G.
				<br/> 
				年份：1996 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two Bradyrhizobium japonicum <span class="gene">nolA</span> mutants were constructed and used to test the functional role of NolA in nodulation. Contrary to the previous hypothesis that NolA acts as a repressor of nod gene transcription, the expression of a nodD1-lacZ or nodY-lacZ fusion in the <span class="gene">nolA</span> mutant strains was similar to that found in the wild type. However, NolA does appear to act as a transcriptional regulatory protein since it is required for its own expression, as well as that of nodD2. Expression of NodD2 from a constitutive promoter led to a significant reduction in nodC-lacZ <span class="trait">activity.</span> Therefore, the repression of nod gene expression by NolA is likely an indirect effect, perhaps mediated by other genes (e.g., nodD2) that are regulated by NolA. When inoculated onto soybean roots, the <span class="gene">nolA</span> mutant strains showed only a slight delay in nodulation as compared to the wild type. However, the mutant strains were grossly defective in nodulation and nitrogen fixation on cowpea plants. Microscopic examination of soybean nodules induced by the <span class="gene">nolA</span> mutant strains showed developmental and morphological characteristics similar to nodules formed by the wild type with only a slight delay in bacteroid maturation. In contrast, cowpea nodules induced by the <span class="gene">nolA</span> mutant strains contained fewer infected cells and bacteroids were not found in a typical symbiosome structure. These results indicate that NolA is a transcriptional activator required for the expression of genes that play a role not only in the early stages of infection, but also during the later stages of bacteroid development and maintenance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Phytophthora sojae </italic> Avirulence Effector Avr3b is a Secreted NADH and ADP-ribose Pyrophosphorylase that Modulates Plant Immunity</h2>
			<h3>PMID:22102810</h3>
				
					<h3>PMCID:3213090</h3>
					<h4>期刊:PLoS Pathogens</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Suomeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">
				<br/> 
				年份：<day> 19 </day>  <month> 9 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Avr effectors are assumed to be important virulence factors that function by manipulating host immunity. However, only a few fungal and oomycete effectors have been functionally characterized <xref rid="ppat.1002353-Jones1" ref-type="bibr"> [1] </xref> , <xref rid="ppat.1002353-DeWit1" ref-type="bibr"> [2] </xref> . In the fungal pathogen <italic> Cladosporium fulvum </italic> , the Avr effector Avr2 is a cysteine protease inhibitor, and <span class="gene">Avr4</span> acts as a chitin-binding protein that protects <italic> C. fulvum </italic> against host chitinases <xref rid="ppat.1002353-vanEsse1" ref-type="bibr"> [24] </xref> , <xref rid="ppat.1002353-vanEsse2" ref-type="bibr"> [25] </xref> . The Avr-Pita effector is a zinc-dependent metalloprotease that is required by the fungus <italic> Magnaporthe oryzae </italic> for full virulence on rice <xref rid="ppat.1002353-Jia1" ref-type="bibr"> [26] </xref> , <xref rid="ppat.1002353-Bohnert1" ref-type="bibr"> [27] </xref> . The polyketide synthase <span class="trait">activity</span> of <italic> M. oryzae </italic> Ace1 is required for avirulence <xref rid="ppat.1002353-Bohnert1" ref-type="bibr"> [27] </xref> . In the flax <span class="trait">rust</span> <italic> Melampsora lini </italic> , AvrP123 effectors contain a Kazal protease inhibitor signature, but the <span class="trait">activity</span> has not been verified <xref rid="ppat.1002353-Catanzariti1" ref-type="bibr"> [28] </xref> . Recently, <italic> Fusarium oxysporum </italic> f. sp. <italic> lycopersici </italic> Avr1 was found to suppress Avr2- and Avr3-triggered ETI in tomato <xref rid="ppat.1002353-Houterman1" ref-type="bibr"> [29] </xref> . <span class="gene">Avr1b</span> and many other effectors from <italic> P. sojae </italic> could abolish cell death triggered in plants by BAX (a mammalian pro-apoptotic factor), by the PAMP infestin1 (INF1) and by several effectors <xref rid="ppat.1002353-Dou3" ref-type="bibr"> [30] </xref> , <xref rid="ppat.1002353-Wang1" ref-type="bibr"> [31] </xref> . Downy mildew <italic> H. arabidopsidis </italic> Avr effectors ATR1 and ATR13 could enhance bacterial virulence on susceptible hosts, while some ATR13 alleles could suppress bacterial PAMP-triggered callose deposition or reduce PAMP-triggered ROS production <xref rid="ppat.1002353-Sohn1" ref-type="bibr"> [32] </xref> . <italic> Phytophthora infestans </italic> <span class="gene">Avr3a</span> could interact with and stabilize host U-box E3 ligase CMPG1, preventing it from being degraded by the proteasome system and subsequently blocking triggered cell death triggered by INF1 <xref rid="ppat.1002353-Bos1" ref-type="bibr"> [33] </xref> . However, the virulence mechanism of most oomycete and fungal Avr effectors remains to be further explored.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine whether candidate effectors might match <italic> Avr3b </italic> , we conducted genetic mapping to identify effectors that co-segregated with the Avr3b avirulence phenotype. First, we crossed P6497 and P7076, and identified 3 F <sub> 1 </sub> individuals. All the 3 F <sub> 1 </sub> individuals showed avirulence on <italic> Rps </italic> 3b soybeans. Then, one F <sub> 1 </sub> individual was selected for self-crossing, resulting in a total of 71 F <sub> 2 </sub> individuals ( <italic> P. sojae </italic> is diploid and homothallic, and wild-type strains show low levels of heterozygosity, hence segregation for most loci occurs in the F <sub> 2 </sub> generation). Avirulence assays of these progeny on soybean plants carrying <span class="trait">resistance</span> gene <italic> Rps </italic> 3b distinguished 51 avirulent and 20 virulent F <sub> 2 </sub> individuals. The observed F <sub> 2 </sub> population segregation ratio for <italic> Avr3b </italic> fit a 3&#x02236;1 ratio (&#x003c7; <sup> 2 </sup> &#x0200a;=&#x0200a;0.38, p &#x0003e;0.05), suggesting that the <italic> Avr3b </italic> avirulence phenotype is conferred by a single dominant allele, consistent with a previous report <xref rid="ppat.1002353-May1" ref-type="bibr"> [34] </xref> . Cleaved amplified polymorphic (CAP) DNA markers were designed based on the sequence polymorphisms of the candidate effector genes ( <xref ref-type="supplementary-material" rid="ppat.1002353.s005"> Table S2 </xref> ). The <italic> <span class="gene">Avh</span> </italic> 307 genotype was found to perfectly match (0&#x00025; recombination) the Avr3b avirulence phenotype among the 71 F <sub> 2 </sub> progeny ( <xref ref-type="fig" rid="ppat-1002353-g001"> Figure 1A </xref> ). In contrast, <italic> <span class="gene">Avh</span> </italic> 238 (8.3&#x00025; recombination), <italic> <span class="gene">Avh</span> </italic> 288 (29&#x00025;), <italic> <span class="gene">Avh</span> </italic> 20 (33&#x00025;), <italic> <span class="gene">Avh</span> </italic> 6 (40&#x00025;), <italic> <span class="gene">Avh</span> </italic> 113 (35&#x00025;), and <italic> <span class="gene">Avh</span> </italic> 258 (31&#x00025;) did not match.
					</p>
				
					<p>&nbsp;&nbsp;
						To quantify suppression, and to determine whether the Nudix motif is required for suppression, an indirect and a direct assay were carried out ( <xref ref-type="fig" rid="ppat-1002353-g007"> Figure 7B </xref> ). For the indirect assay, the number of GUS-expressing spots surviving in the presence of <span class="gene">Avr1b</span> on Rps1b soybean was measured relative to a parallel GUS-only control in the presence or absence of Avr3b <sup> P6497 </sup> or Avr3b <sup> QQQQ </sup> . For the direct assay one barrel delivered DNA encoding <span class="gene">Avr1b</span> &#x0002b; Avr3b (Avr3b <sup> P6497 </sup> or Avr3b <sup> QQQQ </sup> ) and the other barrel delivered DNA encoding <span class="gene">Avr1b</span> alone. The results of both the direct and the indirect assays showed that the survival of GUS-spots from <span class="gene">Avr1b</span> &#x0002b; Avr3b <sup> P6497 </sup> was significantly higher than from <span class="gene">Avr1b</span> alone, confirming that Avr3b could suppress Avr1b-triggered cell death. The survival of GUS-spots from <span class="gene">Avr1b</span> &#x0002b; Avr3b <sup> QQQQ </sup> was significantly less than <span class="gene">Avr1b</span> &#x0002b; Avr3b <sup> P6497 </sup> and not significantly higher than from <span class="gene">Avr1b</span> alone indicating that Avr3b <sup> P6497 </sup> suppression of <span class="gene">Avr1b</span> mediated cell death is dependent on the nudix motif <span class="trait">activity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Phytophthora </italic> genomes encoded hundreds of RXLR effector genes, suggesting many of these effectors might be redundant in function <xref rid="ppat.1002353-Birch2" ref-type="bibr"> [57] </xref> . For examine, either overexpression or silencing of <italic> P. sojae </italic> Avr gene <italic> Avr3a/5 </italic> does not significantly effect <italic> P. sojae </italic> virulence on susceptible soybean <xref rid="ppat.1002353-Dong2" ref-type="bibr"> [19] </xref> . Overexpression of <italic> <span class="gene">Avr1b</span> </italic> in <italic> P. sojae </italic> makes transformants more aggressive on soybean <xref rid="ppat.1002353-Dou3" ref-type="bibr"> [30] </xref> , but the effector is naturally silenced in some isolates <xref rid="ppat.1002353-Shan1" ref-type="bibr"> [13] </xref> . On the other hand silencing of <italic> <span class="gene">Avr3a</span> </italic> impaired <italic> P. infestans </italic> pathogenicity on <italic> N. benthamiana </italic> <xref rid="ppat.1002353-Bos1" ref-type="bibr"> [33] </xref> , and silencing of Avh172 and Avh238 in <italic> P. sojae </italic> also impaired virulence <xref rid="ppat.1002353-Wang1" ref-type="bibr"> [31] </xref> , suggesting that these effectors are essential virulence factors. In this paper, transient silencing of <italic> Avr3b </italic> <sup> P6497 </sup> compromised the virulence of transformant recipient strains on susceptible soybean cultivar, identifying Avr3b also as an essential virulence factor. This is consistent with our finding that both Avr3b virulence and avirulence alleles are transcribed and both proteins retain Nudix enzyme <span class="trait">activity.</span> In conclusion, Avr3b plays dual roles (avirulence and virulence) in the <italic> P. sojae </italic> - soybean interaction. The virulence allele Avr3b <sup> P7076 </sup> may be considered to be a compromise between host recognition pressures and pathogen fitness.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean MAPK, <span class="gene">GMK1</span> is dually regulated by phosphatidic acid and hydrogen peroxide and translocated to nucleus during salt stress.</h2>
			<h3>PMID:22886763  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Im JH(1), Lee H, Kim J, Kim HB, An CS.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mitogen-activated protein kinase (MAPK) is activated by various biotic and abiotic stresses. Salt stress induces two well-characterized MAPK activating signaling molecules, phosphatidic acid (PA) via phospholipase D and phospholipase C, and reactive oxygen species (ROS) via nicotinamide adenine dinucleotide phosphate (NADPH)-oxidase. In our previous study, the <span class="trait">activity</span> of soybean MAPK, <span class="gene">GMK1</span> was strongly induced within 5 min of 300 mM NaCl treatment and this early <span class="trait">activity</span> was regulated by PA. In this study, we focused on the regulation of <span class="gene">GMK1</span> at the later stage of the salt stress, because its <span class="trait">activity</span> was strongly persistent for up to 30 min. H(2)O(2) activated <span class="gene">GMK1</span> even in the presence of PA generation inhibitors, but <span class="gene">GMK1</span> <span class="trait">activity</span> was greatly decreased in the presence of diphenyleneiodonium, an inhibitor of NADPH-oxidase after 5 min of the treatment. On the contrary, the n-butanol and neomycin reduced <span class="gene">GMK1</span> <span class="trait">activity</span> within 5 min of the treatment. Thus, <span class="gene">GMK1</span> <span class="trait">activity</span> may be sustained by H(2)O(2) 10 min after the treatment. Further, <span class="gene">GMK1</span> was translocated into the nucleus 60 min after NaCl treatment. In the relationship between <span class="gene">GMK1</span> and ROS generation, ROS generation was reduced by SB202190, a MAPK inhibitor, but was increased in protoplast overexpressing TESD-GMKK1. However, these effects were occurred at prolonged time of NaCl treatment. These data suggest that <span class="gene">GMK1</span> indirectly regulates ROS generation. Taken together, we propose that soybean <span class="gene">GMK1</span> is dually regulated by PA and H(2)O(2) at a time dependant manner and translocated to the nucleus by the salt stress signal.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Exogenous caffeic acid inhibits the growth and enhances the lignification of the roots of soybean (Glycine max).</h2>
			<h3>PMID:21489652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bubna GA(1), Lima RB, Zanardo DY, Dos Santos WD, Ferrarese Mde L, Ferrarese-Filho O.
				<br/> 
				年份：2011 Sep 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The allelopathic effect of caffeic acid was tested on <span class="trait">root</span> growth, phenylalanine ammonia-lyase (PAL) and peroxidase (POD) activities, hydrogen peroxide (H(2)O(2)) accumulation, lignin content and monomeric composition of soybean (Glycine max) roots. We found that exogenously applied caffeic acid inhibited <span class="trait">root</span> growth, decreased the <span class="gene">PAL</span> <span class="trait">activity</span> and H(2)O(2) content and increased the soluble and cell wall-bound POD activities. The p-hydroxyphenyl (H), guaiacyl (G), and syringyl (S) monomers and total lignin (H+G+S) increased in the caffeic acid-exposed roots. When applied in conjunction with piperonylic acid (PIP, an inhibitor of the cinnamate 4-hydroxylase, C4H), caffeic acid equalized the inhibitory effect of PIP, whereas the application of methylene dioxocinnamic acid (MDCA, an inhibitor of the 4-coumarate:CoA ligase, 4CL) plus caffeic acid decreased lignin production. These results indicate that exogenously applied caffeic acid can be channeled into the phenylpropanoid pathway via the <span class="gene">4CL</span> reaction, resulting in an increase of lignin monomers that solidify the cell wall and inhibit <span class="trait">root</span> growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular and biochemical characterization of AtPAP15, a purple acid phosphatase with phytase <span class="trait">activity,</span> in Arabidopsis.</h2>
			<h3>PMID:19633233  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kuang R(1), Chan KH, Yeung E, Lim BL.
				<br/> 
				年份：2009 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Purple acid phosphatase (PAP) catalyzes the hydrolysis of phosphate monoesters and anhydrides to release phosphate within an acidic pH range. Among the 29 PAP-like proteins in Arabidopsis (Arabidopsis thaliana), AtPAP15 (At3g07130) displays a greater degree of amino acid identity with soybean (Glycine max; <span class="gene">GmPHY)</span> and tobacco (Nicotiana tabacum) <span class="gene">PAP</span> (NtPAP) with phytase <span class="trait">activity</span> than the other AtPAPs. In this study, transgenic Arabidopsis that expressed an AtPAP15 promoterbeta-glucuronidase (GUS) fusion protein showed that AtPAP15 expression was developmentally and temporally regulated, with strong GUS staining at the early stages of seedling growth and pollen germination. The expression was also organ/tissue specific, with strongest GUS staining in the vasculature, pollen grains, and roots. The recombinant AtPAP purified from transgenic tobacco exhibited broad substrate specificity with moderate phytase <span class="trait">activity.</span> AtPAP15 T-DNA insertion lines exhibited a lower phytase and phosphatase <span class="trait">activity</span> in seedling and germinating pollen and lower pollen germination rate compared with the wild type and their complementation lines. Therefore, AtPAP15 likely mobilizes phosphorus reserves in plants, particularly during seed and pollen germination. Since AtPAP15 is not expressed in the <span class="trait">root</span> hair or in the epidermal cells, it is unlikely to play any role in external phosphorus assimilation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of stress-inducible <span class="gene">GmNAC</span> genes in soybean.</h2>
			<h3>PMID:19277718  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tran LS(1), Quach TN, Guttikonda SK, Aldrich DL, Kumar R, Neelakandan A, Valliyodan B, Nguyen HT.
				<br/> 
				年份：2009 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is detrimental to plant growth and development, and often results in significant losses to the yields of economically important crops such as soybeans (Glycine max L.). NAC transcription factors (TFs), which consist of a large family of plant-specific TFs, have been reported to enhance <span class="trait">drought</span> tolerance in a number of plants. In this study, 31 unigenes that contain the complete open reading frames encoding <span class="gene">GmNAC</span> proteins were identified and cloned from soybean. Analysis of C-terminal regulatory domain using yeast one-hybrid system indicated that among 31 <span class="gene">GmNAC</span> proteins, 28 have transcriptional activation <span class="trait">activity.</span> Expression analysis of these <span class="gene">GmNAC</span> genes showed that they are differentially expressed in different organs, suggesting that they have diverse functions during plant growth and development. To search for the drought-inducible <span class="gene">GmNAC</span> genes, we prescreened and re-confirmed by quantitative real-time PCR analysis that nine <span class="gene">GmNAC</span> genes are induced by dehydration stress with differential induction levels in both shoot and <span class="trait">root.</span> The expression profiles of these nine <span class="gene">GmNAC</span> genes were also examined under other stresses such as high salinity, cold and with abscisic acid hormone treatments. Phylogenetic analysis of the <span class="gene">GmNAC</span> proteins with previously reported drought-inducible NAC proteins of Arabidopsis and rice revealed that the nine drought-inducible <span class="gene">GmNAC</span> proteins belong to the "stress-inducible" NAC group. The results of this systematic analysis of the <span class="gene">GmNAC</span> family will provide novel tools and resources for the development of improved <span class="trait">drought</span> tolerant transgenic soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The 3' untranslated region of a soybean cytosolic glutamine synthetase (GS1) affects transcript stability and protein accumulation in transgenic alfalfa.</h2>
			<h3>PMID:16460515  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ortega JL(1), Moguel-Esponda S, Potenza C, Conklin CF, Quintana A, Sengupta-Gopalan C.
				<br/> 
				年份：2006 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Higher plants assimilate nitrogen in the form of ammonia through the concerted <span class="trait">activity</span> of glutamine synthetase (GS) and glutamate synthase (GOGAT). The <span class="gene">GS</span> enzyme is either located in the cytoplasm (GS1) or in the chloroplast (GS2). Glutamine synthetase 1 is regulated in different plants at the transcriptional level and there are some reports of regulation at the level of protein stability. Here we present data that clearly establish that GS1 in plants is also regulated at the level of transcript turnover and at the translational level. Using a Glycine max (soybean) GS1 transgene, with and without its 3' untranslated region (UTR), driven by the constitutive CaMV 35S promoter in Medicago sativa (alfalfa) and Nicotiana tabacum (tobacco), we show that the 3' UTR plays a major role in both transcript turnover and translation repression in both the leaves and the nodules. Our data suggest that the 3' UTR mediated turnover of the transcript is regulated by a nitrogen metabolite or carbon/nitrogen ratios. We also show that the 3' UTR of the gene for the soybean GS1 confers post-transcriptional regulation on a reporter gene. Our dissection of post-transcriptional and translational levels of regulation of <span class="gene">GS</span> in plants shows that the situation in plants strongly resembles that in other organisms where <span class="gene">GS</span> is regulated at almost all levels. Multistep regulation of <span class="gene">GS</span> shows the high priority given by organisms to regulating and ensuring optimal control of nitrogen substrates and preventing overproduction of glutamine and drainage of the glutamate pool.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="trait">root-</span> and hypocotyl-specific gene coding for copper-containing amine oxidase is related to cell expansion in soybean seedlings.</h2>
			<h3>PMID:16291800  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Delis C(1), Dimou M, Flemetakis E, Aivalakis G, Katinakis P.
				<br/> 
				年份：2006
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polyamines are considered to participate in various processes of plant development. In this study, the possible implication of putrescine catabolism by the copper-containing amine oxidases (CuAOs, EC 1.4.3.6) in the development of roots and hypocotyls was examined. For this purpose, two cDNA clones of Glycine max (L.) Merr. cv. Williams, designated as <span class="gene">GmCuAO1</span> and <span class="gene">GmCuAO2,</span> exhibiting extensive similarity with previously characterized CuAO clones from other plants, have been isolated and characterized. The expression of the <span class="gene">GmCuAO1</span> gene is <span class="trait">root-</span> and hypocotyl-specific, while <span class="gene">GmCuAO2</span> seems not to be expressed in a tissue-specific manner. Moreover, the <span class="gene">GmCuAO1</span> gene is predominantly expressed in tissues which are characterized by rapid extension growth, such as the apical segments of etiolated hypocotyls. Using convex and concave segments of the etiolated hypocotyl apical hook it has been demonstrated that <span class="gene">GmCuAO1</span> is strongly expressed in expanding cells of the concave part when exposed to light, while the same pattern is also followed by the <span class="trait">activity</span> of enzymes involved in putrescine catabolism. In dark and photoperiodically grown hypocotyls, <span class="trait">activity</span> measurements of the enzymes involved in putrescine catabolism have shown that the <span class="trait">activity</span> of these enzymes is several-fold higher in rapidly growing tissues. Furthermore, the cellular and tissue distribution of <span class="gene">GmCuAO1</span> gene transcripts in the <span class="trait">root</span> axis and in hypocotyls confirmed their abundance in developing tissues and expanding cells. The results provide evidence suggesting that a tissue-specific gene coding for CuAO is correlated with cell expansion in fast-growing tissues of <span class="trait">root</span> and hypocotyls.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterogeneity of sucrose synthase genes in bean (Phaseolus vulgaris L.): evidence for a nodule-enhanced sucrose synthase gene.</h2>
			<h3>PMID:12554718  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Silvente S(1), Camas A, Lara M.
				<br/> 
				年份：2003 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS), the key sucrose hydrolytic enzyme (EC 2.4.1.13), plays an important role in N(2)-fixing nodule metabolism. It has also been proposed that N(2) fixation in soybean nodules could be mediated by the potential to metabolize sucrose. The isolation and characterization of a nodule-enhanced <span class="gene">SS</span> full-length cDNA clone from the bean Phaseolus vulgaris is reported here. Southern blot analysis indicated that there are at least two <span class="gene">SS</span> genes in beans. Using a 3' specific probe from this <span class="gene">SS</span> cDNA clone, it was possible to identify a nodule-enhanced <span class="gene">SS</span> gene (PvSSn), which is expressed almost exclusively in nodules. A second gene (PvSS), which is expressed in all tissues tested, was detected using a coding region probe. Nodule-enhanced PvSSn transcript levels, but not the enzyme <span class="trait">activity</span> or protein amount, is reduced during nodule development. These data indicated that this reduction could be due to a limitation on the carbon availability in the nodule. PvSSn expression is reduced in the asparagine-treated nodules. By contrast, PvSSn transcript levels in nodules increased in the presence of glutamine, allantoin and allopurinol. This result suggests a relationship between ureide transport and <span class="gene">SS</span> regulation and could help in understanding why the ureide transport mechanism is activated during nitrogen fixation in bean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Abscisic acid induces a decline in nitrogen fixation that involves leghaemoglobin, but is independent of sucrose synthase <span class="trait">activity.</span></h2>
			<h3>PMID:11283173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：González EM(1), Gálvez L, Arrese-Igor C.
				<br/> 
				年份：2001 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sucrose synthase (SS) <span class="trait">activity</span> has been suggested to be a key point of regulation in nodule metabolism since this enzyme is down-regulated in response to different stresses which lead to decreased nitrogen fixation. In soybean, a dramatic decline of <span class="gene">SS</span> transcripts has been observed within 1 d from the onset of <span class="trait">drought.</span> Such a quick response suggests mediation by a signal transduction molecule. Abscisic acid (ABA) is a likely candidate to act as such a molecule as it mediates in a significant number of plant responses to environmental constraints. The hypothesis of ABA controlling nodule metabolism was approached in this work by assessing nodule responses to exogenous ABA supply in pea. Under the experimental conditions, ABA did not affect plant biomass, nodule numbers or dry weight. However, nitrogen fixation rate was reduced by 70% within 5 d and by 80% after 9 d leading to a reduced plant organic nitrogen content. Leghaemoglobin (Lb) content declined in parallel with that of nitrogen fixation. <span class="gene">SS</span> <span class="trait">activity,</span> however, was not affected by ABA treatment, and neither were the activities of the enzymes aspartate amino transferase, alkaline invertase, malate dehydrogenase, glutamate synthase, uridine diphosphoglucose pyrophosphorylase, isocitrate dehydrogenase, and glutamine synthetase. Nodule bacteroid-soluble protein content was reduced in nodules only after 9 d of ABA treatment. These results do not support the hypothesis that ABA directly regulates <span class="gene">SS</span> <span class="trait">activity.</span> However, they do suggest the occurrence of at least two different control pathways in nodules under environmental constraints, which include ABA being involved in a Lb/oxygen-related control of nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Bradyrhizobium japonicum proline biosynthesis gene <span class="gene">proC</span> is essential for symbiosis.</h2>
			<h3>PMID:11097929  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：King ND(1), Hojnacki D, O'Brian MR.
				<br/> 
				年份：2000 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant host-derived proline is proposed to serve as an energy source for rhizobia in the rhizosphere and in symbiotic <span class="trait">root</span> nodules. The Bradyrhizobium japonicum <span class="gene">proC</span> gene was isolated, and a <span class="gene">proC</span> mutant strain that behaved as a strict proline auxotroph in culture was constructed. The <span class="gene">proC</span> strain elicited undeveloped nodules on soybeans that lacked nitrogen fixation <span class="trait">activity</span> and plant hemoglobin. We conclude that the <span class="gene">proC</span> gene is essential for symbiosis and suggest that the mutant does not obtain an exogenous supply of proline in association with soybeans sufficient to satisfy its auxotrophy.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of <span class="gene">GmNRT2</span> expression and nitrate transport <span class="trait">activity</span> in roots of soybean (Glycine max).</h2>
			<h3>PMID:9715532  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Amarasinghe BH(1), de Bruxelles GL, Braddon M, Onyeocha I, Forde BG, Udvardi MK.
				<br/> 
				年份：1998 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA, <span class="gene">GmNRT2,</span> encoding a putative high-affinity nitrate transporter was isolated from a Glycine max (L.) <span class="trait">root</span> cDNA library and sequenced. The deduced <span class="gene">GmNRT2</span> protein is 530 amino acids in length and contains 12 putative membrane-spanning domains and a long, hydrophilic C-terminal domain. <span class="gene">GmNRT2</span> is related to high-affinity nitrate transporters in the eukaryotes Chlamydomonas reinhardtii and Aspergillus nidulans, and to putative high-affinity nitrate transporters in barley and tobacco. Southern blot analysis indicated that <span class="gene">GmNRT2</span> is part of a small, multigene family in soybean. Expression of <span class="gene">GmNRT2</span> in roots was regulated by the type of nitrogen source provided to plants: <span class="gene">GmNRT2</span> mRNA levels were barely detectable in ammonium-grown plants, higher in nitrogen-deprived plants, and highest in nitrate-grown plants. Induction of <span class="gene">GmNRT2</span> mRNA levels in roots occurred within 1 h after exposure of plants to nitrate. Nitrate induction of <span class="gene">GmNRT2</span> mRNA levels was accompanied by a fourfold increase in net nitrate uptake by soybean roots at 100 microM external nitrate. The molecular and physiological evidence indicates that <span class="gene">GmNRT2</span> is probably a high-affinity nitrate transporter involved in nitrate uptake by soybean roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and functional characterization of a cDNA encoding nucleosome assembly protein 1 (NAP-1) from soybean.</h2>
			<h3>PMID:8544812  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yoon HW(1), Kim MC, Lee SY, Hwang I, Bahk JD, Hong JC, Ishimi Y, Cho MJ.
				<br/> 
				年份：1995 Dec 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						NAP-1, a protein first isolated from mammalian cells, can introduce supercoils into relaxed circular DNA in the presence of purified core histones. Based on its in vitro <span class="trait">activity,</span> it has been suggested that NAP-1 may be involved in nucleosome assembly in vivo. We isolated a cDNA clone encoding a soybean NAP-1 homolog, SNAP-1. The <span class="gene">SNAP-1</span> cDNA contains an open reading frame of 358 amino acids residues with a calculated molecular weight of 41 kDa. The deduced amino acid sequence of <span class="gene">SNAP-1</span> shares sequence similarity with yeast NAP-1 (38%) and human hNRP (32%). Notable features of the deduced sequence are two extended acidic regions thought to be involved in histone binding. <span class="gene">SNAP-1</span> expressed in Escherichia coli induces supercoiling in relaxed circular DNA, suggesting that <span class="gene">SNAP-1</span> may have nucleosome assembly <span class="trait">activity.</span> The specific <span class="trait">activity</span> of <span class="gene">SNAP-1</span> is comparable to that of HeLa NAP-1 in an in vitro assay. Western analysis reveals that <span class="gene">SNAP-1</span> is expressed in the immature and young tissues that were examined, while mature tissues such as old leaves and roots, show very little or no expression. NAP-1 homologs also appear to be present in other plant species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aspartate Aminotransferase in Alfalfa <span class="trait">Root</span> Nodules : III. Genotypic and Tissue Expression of Aspartate Aminotransferase in Alfalfa and Other Species.</h2>
			<h3>PMID:16667896</h3>
				

				
			<p>
				作者：Farnham MW(1), Griffith SM, Miller SS, Vance CP.
				<br/> 
				年份：1990 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aspartate aminotransferase (AAT) plays an important role in nitrogen metabolism in all plants and is particularly important in the assimilation of fixed N derived from the legume-Rhizoblum symbiosis. Two isozymes of <span class="gene">AAT</span> (AAT-1 and AAT-2) occur in alfalfa (Medicago sativa L.). Antibodies against alfalfa nodule AAT-2 do not recognize AAT-1, and these antibodies were used to study AAT-2 expression in different tissues and genotypes of alfalfa and also in other legume and nonlegume species. Rocket immunoelectrophoresis indicated that nodules of 38-day-old alfalfa plants contained about eight times more AAT-2 than did nodules of 7-day-old plants, confirming the nodule-enhanced nature of this isozyme. AAT-2 was estimated to make up 16, 15, 5, and 8 milligrams per gram of total soluble protein in mature nodules, roots, stems, and leaves, respectively, of effective N(2)-fixing alfalfa. The concentration of AAT-2 in nodules of ineffective non-N(2)-fixing alafalfa genotypes was about 70% less than that of effective nodules. Western blots of soluble protein from nodules of nine legume species indicated that a 40-kilodalton polypeptide that reacts strongly with AAT-2 antibodies is conserved in legumes. Nodule AAT-2 immunoprecipitation data suggested that amide- and ureide-type legumes may differ in expression and regulation of the enzyme. In addition, Western blotting and immunoprecipitations of <span class="gene">AAT</span> <span class="trait">activity</span> demonstrated that antibodies against alfalfa AAT-2 are highly cross-reactive with <span class="gene">AAT</span> enzyme protein in leaves of soybean (Glycine max L.), wheat (Triticum aestivum L.), and maize (Zea mays L.) and in roots of maize, but not with <span class="gene">AAT</span> in soybean and wheat roots. Results from this study indicate that AAT-2 is structurally conserved and localized in similar tissues among diverse species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification and Functional Characterization of UDP-Glucosyltransferase Genes Involved in Flavonoid Biosynthesis in Glycine max.</h2>
			<h3>PMID:28633497</h3>
				

				
			<p>
				作者：Yin Q(1), Shen G(2), Di S(1), Fan C(1), Chang Z(3), Pang Y(1).
				<br/> 
				年份：2017 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flavonoids, natural products abundant in the model legume Glycine max, confer benefits to plants and to animal health. Flavonoids are present in soybean mainly as glycoconjugates. However, the mechanisms of biosynthesis of flavonoid glycosides are largely unknown in G. max. In the present study, 212 putative UDP-glycosyltransferase (UGT) genes were identified in G. max by genome-wide searching. The <span class="gene">GmUGT</span> genes were distributed differentially among the 20 chromosomes, and they were expressed in various tissues with distinct expression profiles. We further analyzed the enzymatic activities of 11 <span class="gene">GmUGTs</span> that are potentially involved in flavonoid glycosylation, and found that six of them (UGT72X4, UGT72Z3, UGT73C20, UGT88A13, UGT88E19 and UGT92G4) exhibited <span class="trait">activity</span> toward flavonol, isoflavone, flavone and flavanol aglycones with different kinetic properties. Among them, UGT72X4, UGT72Z3 and UGT92G4 are flavonol-specific UGTs, and UGT73C20 and UGT88E19 exhibited <span class="trait">activity</span> toward both flavonol and isoflavone aglycones. In particular, UGT88A13 exhibited <span class="trait">activity</span> toward epicatechin, but not for the flavonol aglycones kaempferol and quercetin. Overexpression of these six <span class="gene">GmUGT</span> genes significantly increased the contents of isoflavone and flavonol glucosides in soybean hairy roots. In addition, overexpression of these six <span class="gene">GmUGT</span> genes also affected flavonol glycoside contents differently in seedlings and seeds of transgenic Arabidopsis thaliana. We provide valuable information on the identification of all UGT genes in soybean, and candidate <span class="gene">GmUGT</span> genes for potential metabolic engineering of flavonoid compounds in both Escherichia coli and plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Auxin enhances aluminium-induced citrate exudation through upregulation of <span class="gene">GmMATE</span> and activation of the plasma membrane H+-ATPase in soybean roots.</h2>
			<h3>PMID:27474509</h3>
				

				
			<p>
				作者：Wang P(1), Yu W(1), Zhang J(1), Rengel Z(2), Xu J(3), Han Q(1), Chen L(1), Li K(1), Yu Y(4), Chen Q(5).
				<br/> 
				年份：2016 Jul 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIMS: Aluminium (Al) toxicity is a limiting factor for plant growth and crop production in acidic soils. Citrate exudation and activation of the plasma membrane H+-ATPase are involved in soybean responses to Al stress. Auxin has crucial functions in plant growth and stress responses. However, little is known about possible interactions between auxin and citrate exudation under Al stress. In this study, we elucidated the regulatory roles of IAA in Al-induced citrate exudation in soybean roots. METHODS: We measured IAA content, Al concentration, citrate exudation, plasma membrane H+-ATPase <span class="trait">activity,</span> expression of the relevant genes and phosphorylation of the plasma membrane H+-ATPase by integrating physiological characterization and molecular analysis using hydroponically grown soybean. KEY RESULT: s The concentration of IAA was increased by 25 and 50 μm Al, but decreased to the control level at 200 μm Al. External addition of 50 μm IAA to the <span class="trait">root</span> medium containing 25, 50 or 200 μm Al decreased <span class="trait">root</span> Al concentration and stimulated Al-induced citrate exudation and the plasma membrane H+-ATPase <span class="trait">activity.</span> Reverse transcription-PCR analysis showed that exogenous IAA enhanced the expression of citrate exudation transporter <span class="gene">(GmMATE)</span> but not the plasma membrane H+-ATPase gene. The western blot results suggested that IAA enhanced phosphorylation of the plasma membrane H+-ATPase under Al stress. CONCLUSIONS: Auxin enhanced Al-induced citrate exudation through upregulation of <span class="gene">GmMATE</span> and an increase in phosphorylation of the plasma membrane H+-ATPase in soybean roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of alternative transcripts of the soybean <span class="gene">Rj2</span> gene that restricts nodulation with specific rhizobial strains.</h2>
			<h3>PMID:26848549  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tang F(1), Yang S(1), Zhu H(1).
				<br/> 
				年份：2016 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">Rj2</span> gene is a TIR-NBS-LRR-type <span class="trait">resistance</span> gene in soybean (Glycine max) that restricts <span class="trait">root</span> nodule symbiosis with a group of Bradyrhizobium japonicum strains including USDA122. <span class="gene">Rj2</span> generates two distinct transcript variants in its expression profile through alternative splicing. Alternative splicing of <span class="gene">Rj2</span> is caused by the retention of the 86-bp intron 4. Inclusion of intron 4 in mature mRNA introduces an in-frame stop codon; as such, the alternative transcript is predicted to encode a truncated protein consisting of the entire portion of the TIR, NBS and LRR domains but missing the C-terminal domain of the full-length <span class="gene">Rj2</span> protein encoded by the regular transcript. Since alternative splicing has been shown to be essential for full <span class="trait">activity</span> of several plant R genes, we attempted to test whether the alternative splicing is required for Rj2-mediated nodulation restriction. Here we demonstrated that the Rj2-mediated nodulation restriction does not require the combined presence of the regular and alternative transcripts, and the expression of the regular transcript alone is sufficient to confer nodulation restriction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of alanine aminotransferase in various organs of soybean ( <italic> Glycine max </italic> ) and in dependence of different nitrogen fertilisers during hypoxic stress</h2>
			<h3>PMID:20414691</h3>
				

				
					<h3>PMCID:2945468</h3>
					<h4>期刊:Amino Acids</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Rocha </surname>  <given-names> Marcio </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sodek </surname>  <given-names> Ladaslav </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Licausi </surname>  <given-names> Francesco </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hameed </surname>  <given-names> Muhammad Waqar </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dornelas </surname>  <given-names> Marcelo Carnier </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> van Dongen </surname>  <given-names> Joost T. </given-names>  </name>  <address>  <phone> +49-331-5678353 </phone>  <email> dongen@mpimp-golm.mpg.de </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Energy Metabolism Research Group, Max Planck Institute of Molecular Plant Physiology, Am M&#x000fc;hlenberg 1, 14476 Potsdam-Golm, Germany  </aff>  <aff id="Aff2">  <label> 2 </label> Departamento de Fisiologia Vegetal, Instituto de Biologia, Universidade Estadual de Campinas, C.P. 6109, Campinas, SP 13083-970 Brazil  </aff>  <aff id="Aff3">  <label> 3 </label> Plant Lab, Scuola Superiore Sant&#x02019;Anna, Piazza Martiri della Liberta 33, 56127 Pisa, Italy  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 4 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Alanine aminotransferase (AlaAT) catalyses the reversible conversion of pyruvate and glutamate into alanine and oxoglutarate. In soybean, two subclasses were identified, each represented by two highly similar members. To investigate the role of AlaAT during hypoxic stress in soybean, changes in transcript level of both subclasses were analysed together with the enzyme <span class="trait">activity</span> and alanine content of the tissue. Moreover, the dependency of AlaAT <span class="trait">activity</span> and gene expression was investigated in relation to the source of nitrogen supplied to the plants. Using semi-quantitative PCR, <span class="gene">GmAlaAT</span> genes were determined to be highest expressed in roots and nodules. Under normal growth conditions, enzyme <span class="trait">activity</span> of AlaAT was detected in all organs tested, with lowest <span class="trait">activity</span> in the roots. Upon waterlogging-induced hypoxia, AlaAT <span class="trait">activity</span> increased strongly. Concomitantly, alanine accumulated. During re-oxygenation, AlaAT <span class="trait">activity</span> remained high, but the transcript level and the alanine content decreased. Our results show a role for AlaAT in the catabolism of alanine during the initial period of re-oxygenation following hypoxia. <span class="gene">GmAlaAT</span> also responded to nitrogen availability in the solution during waterlogging. Ammonium as nitrogen source induced both gene expression and enzyme <span class="trait">activity</span> of AlaAT more than when nitrate was supplied in the nutrient solution. The work presented here indicates that AlaAT might not only be important during hypoxia, but also during the recovery phase after waterlogging, when oxygen is available to the tissue again.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The role of AlaAT during the recovery phase after hypoxic stress is probably equally important for the understanding of flooding tolerance (Fan et al. <xref ref-type="bibr" rid="CR10"> 1988 </xref> ) as its biochemical behaviour during hypoxia. de Sousa and Sodek ( <xref ref-type="bibr" rid="CR8"> 2003 </xref> ) demonstrated that AlaAT <span class="trait">activity</span> increased during hypoxia more so after the accumulation of alanine had reached its maximum. Since the level of alanine returned to pre-hypoxic levels within 24&#x000a0;h of return to normoxia, it was suggested that AlaAT has a function during the recovery phase also. Miyashita et al <italic> . </italic> ( <xref ref-type="bibr" rid="CR21"> 2007 </xref> ) confirmed this by showing that the <italic> Arabidopsis </italic> <italic> <span class="gene">AlaAT1</span> </italic> knock-out mutant ( <italic> alaat1 </italic> - <italic> 1 </italic> ) was able to accumulate alanine during hypoxia like wild type plants, whereas the decrease of the levels of alanine during the re-oxygenation phase was delayed. It was suggested that during the hypoxic phase, the plant prepares itself for a rapid recovery once oxygen becomes available again. This hypothesis is mainly based on the observation that many genes that encode proteins with an important function during the recovery phase are already expressed during the hypoxic stress (Drew <xref ref-type="bibr" rid="CR9"> 1997 </xref> ). Furthermore, most metabolites recover rapidly to their normal level once hypoxia is over (Barret-Lennard et al. <xref ref-type="bibr" rid="CR2"> 1988 </xref> ; <span class="gene">Fan</span> et al. <xref ref-type="bibr" rid="CR10"> 1988 </xref> ; Albrecht et al. <xref ref-type="bibr" rid="CR1"> 1993 </xref> ; de Sousa and Sodek <xref ref-type="bibr" rid="CR8"> 2003 </xref> ; Rocha et al. <xref ref-type="bibr" rid="CR25"> 2010 </xref> ). AlaAT has exactly this kind of regulation pattern as its gene expression is up-regulated during hypoxia, and during the re-oxygenation phase, the high levels of AlaAT enzyme can ensure the rapid conversion of accumulated alanine back into glutamate (Miyashita et al. <xref ref-type="bibr" rid="CR21"> 2007 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In order to determine changes of <italic> AlaAT </italic> gene expression, plant material of soybean was ground in liquid nitrogen using a pestle and mortar. For RNA extraction 0.9&#x000a0;g of <span class="trait">root</span> or 0.6&#x000a0;g of nodule, leaf or pod was mixed with 3&#x000a0;mL of TRIzol Reagent (Invitrogen, Carlsbad, USA) following the manufacturer&#x02019;s recommendations. The RNA samples were stored at &#x02212;80&#x000b0;C until further processing. For the synthesis of cDNA, 5.0&#x000a0;&#x003bc;g of total RNA was first treated with Turbo DNA-free (Ambion, Austin, USA) to remove DNA contamination. Subsequently, cDNA was synthesised using oligo(dt) 12&#x02013;18 primers and Superscript II reverse transcriptase (Invitrogen, Carlsbad, USA). Subsequently, the cDNA was used as template for a polymerase chain reaction (PCR) in a total volume of 25 &#x003bc;l with the following reaction conditions: an initial step of 3&#x000a0;min at 96&#x000b0;C was followed by a repeating cycle of 1&#x000a0;min at 94&#x000b0;C, 1&#x000a0;min at 60&#x000b0;C and 1&#x000a0;min at 72&#x000b0;C. Primer selection was performed using the programme <italic> Lasergene </italic> (DNASTAR, Madison, USA) against <italic> <span class="gene">GmAlaAT1</span> </italic> (EU165371) and <italic> <span class="gene">GmAlaAT2</span> </italic> EU165372cDNA sequences from soybean which are representative for the gene subfamilies A and B, respectively. The primer sequences were as follows: <italic> <span class="gene">GmAlaAT1</span> </italic> : sense: CTTCTGCGCCACCGTCACCA antisense: CAGGGCTTGCACCATCAGTCAT; <italic> <span class="gene">GmAlaAT2</span> </italic> : sense: TGGCCACAATTGAAGGACGAG, antisense: GCATCAGCAGGGAATAGCAGT; <italic> Actin </italic> : sense: GCTCCTAGGGCTGTCTTTCC, antisense: CTCAGCAGAGGTGGTGAACA. Preliminary experiments were performed to identify the optimal cycle number at which non-saturated signal intensities were obtained for all ethidium bromide stained PCR products after electrophoresis in a 1% agarose gel TAE buffer. The number of cycles that was finally chosen is indicated next to the gel figures shown in the results section. Actin was used as a loading control to be used for normalisation of the data.
					</p>
				
					<p>&nbsp;&nbsp;
						The distribution of AlaAT in adult soybean plants was analysed by measuring gene expression and by determining the <span class="trait">activity</span> of AlaAT (E.C.2.6.1.2.) in nodules, leaves and pods of both nodulated and non-nodulated plants. Furthermore, the amount of alanine was determined. Alanine levels varied strongly between the various organs, as well as between nodulated and non-nodulated plants (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> a). In non-nodulated plants, leaves contained the highest levels of alanine on a dry weight basis, whereas in nodulated plants the pods had the highest alanine content. Independent of the nodulation status, the level of alanine was lowest in roots. <fig id="Fig3"> <label> Fig.&#x000a0;3 </label> <caption> <p> Analysis of alanine content, AlaAT transcript levels and enzyme <span class="trait">activity</span> in various organs of soybean. <bold> a </bold> Alanine content in pods, leaves, roots and nodules (nod) of nodulated and non-nodulated soybean plants. <bold> b </bold> AlaAT enzyme <span class="trait">activity</span> measured in pods, leaves, roots and nodules (nod) of nodulated and non-nodulated soybean plants. <bold> c </bold> Semi-quantitive RT-PCR analysis of <span class="gene">GmAlaAT</span> transcript levels in pods, leaves, roots and nodules (nod) of nodulated and non-nodulated soybean plants. <span class="gene">GmAlaAT-A</span> represents transcript levels from the genes belonging to subfamily A ( <italic> <span class="gene">GmAlaAT1</span> </italic> and - <italic> 4 </italic> ), whereas <span class="gene">GmALaAT-B</span> indicate transcripts from subfamily B ( <italic> <span class="gene">GmAlaAT2</span> </italic> and - <italic> 3 </italic> ). Due to the very high sequence similarity, it was not possible to distinguish between individual genes within each subfamily. The constitutively expressed actin gene was used as loading control to normalise the samples. The <italic> bars </italic> indicate mean value&#x000a0;&#x000b1;&#x000a0;SD ( <italic> n </italic> &#x000a0;=&#x000a0;4). Values that differ significantly according to a one-way ANOVA ( <italic> P </italic> &#x000a0;&#x0003c;&#x000a0;0.05) and Tukey posthoc test are marked with <italic> different letters </italic> . <italic> Small letters </italic> are used to indicate differences between tissues from either nodulated or non-nodulated plants. <italic> Capital letters </italic> indicate differences within the same kind of tissue from either nodulated or non-nodulated plants
					</p>
				
					<p>&nbsp;&nbsp;
						It was suggested previously that AlaAT is likely involved in nitrogen mobilisation during the recovery period following waterlogging-induced hypoxic stress (de Sousa and Sodek <xref ref-type="bibr" rid="CR8"> 2003 </xref> ; Miyashita et al. <xref ref-type="bibr" rid="CR21"> 2007 </xref> ). Therefore, an experiment was performed to analyse changes of the alanine content, the <span class="trait">activity</span> of AlaAT and the changes in transcript abundance of the <italic> <span class="gene">GmAlaAT</span> </italic> homologues during and after a 3-day waterlogging treatment. Roots of both non-nodulated and nodulated plants were investigated before, during and after the 3-day period of waterlogging.
					</p>
				
					<p>&nbsp;&nbsp;
						The content of alanine in <span class="trait">root</span> tissue increased during the 3-day period of hypoxia in both nodulated and non-nodulated plants. In the roots of non-nodulated plants, alanine increased 45 times and in <span class="trait">root</span> tissue of nodulated plants the increase was 4.6 times only (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> a). Independent of the nodulation status, the alanine values in roots recovered within the 3-day period of re-oxygenation to values close to those measured before the hypoxic treatment. The amount of alanine within nodules did not change during or after the waterlogging treatment. <fig id="Fig5"> <label> Fig.&#x000a0;5 </label> <caption> <p> Changes in alanine content, AlaAT <span class="trait">activity</span> and transcript levels in soybean tissues before and during a 3-day waterlogging treatment and subsequent recovery. <bold> a </bold> The amount of alanine as determined in roots and nodules of nodulated and non-nodulated soybean plants. <bold> b </bold> The <span class="trait">activity</span> of AlaAT as measured in extracts from nodulated and non-nodulated soybean plants. <bold> c </bold> Semiquantitive RT-PCR analysis of <italic> <span class="gene">GmAlaAT</span> </italic> transcript levels in roots and nodules of soybean. PCR conditions and sequence specificity are as described in Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> . Samples were taken before waterlogging ( <italic> C </italic> control), after 3&#x000a0;days of waterlogging ( <italic> W </italic> ), and 3&#x000a0;days after the waterlogging treatment was ceased ( <italic> R </italic> recovery). The <italic> bars </italic> indicate mean value&#x000a0;&#x000b1;&#x000a0;SD as determined from four biological replicates. <italic> Small letters </italic> indicate differences between the different treatments, <italic> C </italic> , <italic> W </italic> and <italic> R </italic> , within the same tissue. <italic> Capital letters </italic> indicate differences between nodulated and non-nodulated tissue within <italic> C </italic> , <italic> W </italic> or <italic> R </italic> treated samples
					</p>
				
					<p>&nbsp;&nbsp;
						The regulation of AlaAT enzyme <span class="trait">activity</span> (E.C. 2.6.1.2) was investigated in soybean. First, the soybean genome sequencing data (Schmutz et al. <xref ref-type="bibr" rid="CR26"> 2010 </xref> ) were explored to identify <italic> AlaAT </italic> homologues encoded by the soybean genome. The analysis revealed the existence of four genes. Comparison between these sequences and homologues from other plant species revealed a subdivision into two subfamilies (Figs.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> , <xref rid="Fig2" ref-type="fig"> 2 </xref> ). The first subfamily A contained the soybean homologues <italic> <span class="gene">GmAlaAT1</span> </italic> and - <italic> 4 </italic> , whereas both <italic> <span class="gene">GmAlaAT2</span> </italic> and <italic> <span class="gene">GmAlaAT3</span> </italic> were included in the subfamily B. The very high similarity in coding as well as non-coding sequences of the <italic> AlaAT </italic> genes indicated that the two members of each subfamily are derived from a recent genome duplication event.
					</p>
				
					<p>&nbsp;&nbsp;
						In Arabidopsis, four <italic> AlaAT </italic> homologues were described also. Detailed functional analysis of these genes had revealed different substrate selectivity of the various enzymes that are encoded by these genes (Igarashi et al. <xref ref-type="bibr" rid="CR17"> 2003 </xref> ; Liepman and Olsen <xref ref-type="bibr" rid="CR19"> 2003 </xref> ). Two of them, <italic> AtAlaAT1 </italic> and <italic> AtAlaAT2 </italic> , were annotated to encode true alanine aminotransferases (E.C. 2.6.1.2), whereas two others (A <italic> tGGAT1 </italic> and A <italic> tGGAT2 </italic> ) were shown to have glutamate:glyoxylate aminotransferase (GGAT) <span class="trait">activity</span> (EC. 2.6.1.4; Igarashi et al. <xref ref-type="bibr" rid="CR17"> 2003 </xref> ; Liepman and Olsen <xref ref-type="bibr" rid="CR19"> 2003 </xref> ). It is tempting to speculate that the two genes <italic> <span class="gene">AlaAT2</span> </italic> and - <italic> 3 </italic> from soybean that have highest sequence similarity to the Arabidopsis genes <italic> AtGGAT1 </italic> and - <italic> 2 </italic> might share the same function in catalysing the glutamate:glyoxylate aminotransferase reaction (E.C.2.6.1.4) rather than being alanine aminotransferases (E.C.2.6.1.2). However, this prediction of enzyme function should be taken with care as it is based on functional analysis of very few members of the gene family only.
					</p>
				
					<p>&nbsp;&nbsp;
						Using semi-quantitative RT-PCR, the changes in <italic> <span class="gene">GmAlaAT</span> </italic> gene expression were analysed. Due to the very high sequence similarity between the homologues from one subfamily, the expression signals were interpreted to represent the changes within the entire subfamily. Changes in the <span class="trait">activity</span> of AlaAT (E.C.2.6.1.2) in roots correlated well with the level of transcript of subfamily A ( <italic> <span class="gene">GmAlaAT1</span> </italic> and - <italic> 4 </italic> ) (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ), indicating that these genes indeed encode for enzymes with true alanine aminotransferase <span class="trait">activity</span> (E.C.2.6.1.2) as was suggested earlier for <italic> Arabidopsis </italic> homologues from the same subfamily (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ; Miyashita et al. <xref ref-type="bibr" rid="CR21"> 2007 </xref> ). However, in other tissues than roots, the correlation between AlaAT <span class="trait">activity</span> and the expression of genes from subfamily A was not so obvious. For example, AlaAT <span class="trait">activity</span> was highest in pods and leaves, but transcript levels of each of the <italic> <span class="gene">GmAlaAT</span> </italic> homologues were very low. Apparently, under these steady state conditions, the <span class="trait">activity</span> of AlaAT enzyme was not determined by the transcript level which is easily explained when the AlaAT enzyme is stable under the given conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						It should be noted that the role of AlaAT is not limited to hypoxic conditions only. After the waterlogging treatment was ceased, the high amount of transcript from <span class="gene">GmAlaAT</span> subfamily A decreased rapidly, but the <span class="trait">activity</span> of the AlaAT enzyme remained at a level that was significantly above that found before waterlogging (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Probably, by catalysing the reversed reaction from alanine to pyruvate, the remaining <span class="trait">activity</span> of AlaAT can explain why the amount of alanine declined so rapidly during the recovery from waterlogging. de Sousa and Sodek ( <xref ref-type="bibr" rid="CR8"> 2003 </xref> ) came to a similar conclusion after measuring a detailed time-course of both AlaAT <span class="trait">activity</span> and the level of alanine in roots of soybean plants during waterlogging and on return to normoxia. They observed that much of the increase in AlaAT <span class="trait">activity</span> during waterlogging took place after most of the increase in alanine had occurred. On return to normoxia, the decrease in the level of alanine was very rapidly. It was concluded that the increase in AlaAT <span class="trait">activity</span> during waterlogging is particularly important to prepare the plant for the time that oxygen is available again. By catalysing the reaction from alanine to pyruvate, AlaAT would enable the tissue to re-utilise the alanine that accumulated during waterlogging. de Sousa and Sodek ( <xref ref-type="bibr" rid="CR8"> 2003 </xref> ) also observed high amounts of alanine being transported by the xylem from waterlogged roots to the normoxic shoot of the plant. This observation provides a further indication that alanine acts as an agent to recycle carbon and nitrogen efficiently within a waterlogged plant.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GH3::GUS reflects cell-specific developmental patterns and stress-induced changes in wood anatomy in the poplar stem.</h2>
			<h3>PMID:18595842  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Teichmann T(1), Bolu-Arianto WH, Olbrich A, Langenfeld-Heyser R, Göbel C, Grzeganek P, Feussner I, Hänsch R, Polle A.
				<br/> 
				年份：2008 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GH3</span> genes related to the auxin-inducible Glycine max (L.) Merr. <span class="gene">GmGH3</span> gene encode enzymes that conjugate amino acids to auxin. To investigate the role of <span class="gene">GH3</span> enzymes in stress responses and normal wood development, Populus x canescens (Ait.) was transformed with the promoter-reporter construct GH3::GUS containing a <span class="gene">GH3</span> promoter and the 5' UTR from soybean. beta-Glucuronidase (GUS) <span class="trait">activity</span> was present in the vascular tissues of leaves and in developing lateral roots and was inducible in silent tissues by external auxin application. A decrease in GUS <span class="trait">activity</span> from the stem apex to the bottom corresponded to decreases in auxin concentrations in these tissues. High auxin concentration and high GH3::GUS <span class="trait">activity</span> were present in the pith tissue, which may provide storage for auxin compounds. <span class="gene">GH3</span> reporter was active in ray cells, paratracheal parenchyma cells, maturing vessels and in cells surrounding maturing phloem fibers but not in the cambium and immature phloem, despite high auxin concentrations in the latter tissues. However, the <span class="gene">GH3</span> promoter in these tissues became active when the plants were exposed to abiotic stresses, like bending or salinity, causing changes in wood anatomy. We suggest that adjustment of the internal auxin balance in wood in response to environmental cues involves <span class="gene">GH3</span> auxin conjugate synthases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Modification of soybean sucrose synthase by S-thiolation with <span class="gene">ENOD40</span> peptide A.</h2>
			<h3>PMID:15541370  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Röhrig H(1), John M, Schmidt J.
				<br/> 
				年份：2004 Dec 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The gene <span class="gene">ENOD40</span> is expressed at an early stage of <span class="trait">root</span> nodule organogenesis and has been postulated to play a central regulatory role in the Rhizobium-legume interaction. In vitro translation of soybean <span class="gene">ENOD40</span> mRNA showed that the gene encodes two peptides of 12 and 24aa residues (peptides A and B) that bind to sucrose synthase. Here we show that the small Cys-containing peptide A binds to sucrose synthase by disulfide bond formation, which may represent a novel form of posttranslational modification of this important metabolic enzyme. Assays using nanomolar concentrations of peptide A revealed that the monomeric reduced form of this peptide binds to purified sucrose synthase. Using a cysteinyl capture strategy combined with MALDI-TOF <span class="gene">MS</span> analysis we identified the Cys residue C264 of soybean sucrose synthase as the binding site of peptide A. Modification of sucrose synthase with <span class="gene">ENOD40</span> peptide A activates sucrose cleavage <span class="trait">activity</span> whereas the synthesis <span class="trait">activity</span> of the enzyme is unaffected. The results are discussed in relation to the role of sucrose synthase in the control of sucrose utilization in nitrogen-fixing nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Does lowering glutamine synthetase <span class="trait">activity</span> in nodules modify nitrogen metabolism and growth of Lotus japonicus?</h2>
			<h3>PMID:12970491  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Harrison J(1), Pou de Crescenzo MA, Sené O, Hirel B.
				<br/> 
				年份：2003 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA encoding cytosolic glutamine synthetase (GS) from Lotus japonicus was fused in the antisense orientation relative to the nodule-specific LBC3 promoter of soybean (Glycine max) and introduced into L. japonicus via transformation with Agrobacterium tumefaciens. Among the 12 independent transformed lines into which the construct was introduced, some of them showed diminished levels of GS1 mRNA and lower levels of <span class="gene">GS</span> <span class="trait">activity.</span> Three of these lines were selected and their T(1) progeny was further analyzed both for plant biomass production and carbon and nitrogen (N) metabolites content under symbiotic N-fixing conditions. Analysis of these plants revealed an increase in fresh weight in nodules, roots and shoots. The reduction in <span class="gene">GS</span> <span class="trait">activity</span> was found to correlate with an increase in amino acid content of the nodules, which was primarily due to an increase in asparagine content. Thus, this study supports the hypothesis that when <span class="gene">GS</span> becomes limiting, other enzymes (e.g. asparagine synthetase) that have the capacity to assimilate ammonium may be important in controlling the flux of reduced N in temperate legumes such as L. japonicus. Whether these alternative metabolic pathways are important in the control of plant biomass production still remains to be fully elucidated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the early nodulin, ENOD40, in soybean roots in response to various lipo-chitin signal molecules.</h2>
			<h3>PMID:8758977  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Minami E(1), Kouchi H, Cohn JR, Ogawa T, Stacey G.
				<br/> 
				年份：1996 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The lipo-chitin (LCO) nodulation signal (nod signal) purified from Bradyrhizobium japonicum induced nodule primordia on soybean (i.e. Glycine soja) roots. These primordia were characterized by a bifurcated vascular connection, cortical cell division, and the accumulation of mRNA of the early nodulin gene, ENOD40. A chemically synthesized LCO identical in structure to the Nod signal purified from B. japonicum cultures showed the same <span class="trait">activity</span> when inoculated on to soybean roots. Surprisingly, synthetic LCO or chitin pentamer, inactive in inducing <span class="trait">root</span> hair curling (HAD) or cortical cell division (NOI) in G. soja, induced the transient accumulation of <span class="gene">ENOD40</span> mRNA. In roots inoculated with such LCO, <span class="gene">ENOD40</span> mRNA was abundant at 40 h after inoculation but decreased to the background levels 6 days after inoculation. In contrast, nod signals active in inducing HAD and NOI induced high levels of <span class="gene">ENOD40</span> accumulation at 40 h and 6 days after inoculation. In situ hybridization analysis showed that <span class="gene">ENOD40</span> mRNA accumulated in the pericycle of the vascular bundle at 24 h after <span class="trait">root</span> inoculation with nod signal. At 6 days post-inoculation with nod signal, <span class="gene">ENOD40</span> expression was seen in dividing subepidermal cortical cells. These results provide morphological and molecular evidence that nodule induction in soybean in response to purified or synthetic nod signal is similar, if not identical, to nodule formation induced by bacterial inoculation. Surprisingly, <span class="gene">ENOD40</span> mRNA accumulation occurs in response to non-specific chitin signals. This suggests that, in the case of ENOD40, nodulation specificity is not determined at the level of initial gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hairy roots of Brassica napus: II. Glutamine synthetase overexpression alters ammonia assimilation and the response to phosphinothricin.</h2>
			<h3>PMID:24194225</h3>
				

				
			<p>
				作者：Downs CG(1), Christey MC, Davies KM, King GA, Seelye JF, Sinclair BK, Stevenson DG.
				<br/> 
				年份：1994 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Hairy roots of Brassica napus (rape cv. Giant) have been produced that contain the cytosolic glutamine synthetase (GS) gene from Glycine max (soybean). Leaf explants were cocultivated with Agrobacterium rhizogenes strain A4T harbouring the binary vector pLN16. This vector was constructed by inserting a soybean cytosolic <span class="gene">GS</span> cDNA into the multiple cloning site of pGA643, placing it under the control of the CaMV promoter. In addition, the T-DNA region of pLN16 contained a NPTII gene for selection of transformed cells. Transgenic hairy roots grew prolifically on hormone-free media containing a selective level of kanamycin. Southern and northern analyses confirmed the presence of soybean <span class="gene">GS</span> DNA and transcripts, respectively. These transformed hairy roots also have a greater abundance of the <span class="gene">GS</span> polypeptide, approximately 3-6 fold greater <span class="gene">GS</span> <span class="trait">activity</span> and lower levels of endogenous ammonia. Hairy roots provide a useful system for studying responses to phosphinothricin (PPT). Hairy roots grown in media containing PPT had lower <span class="gene">GS</span> <span class="trait">activity,</span> greater ammonia accumulation and slower growth than controls. The presence of the soybean <span class="gene">GS</span> gene in the hairy roots reduced these PPT-induced effects and resulted in higher <span class="gene">GS</span> <span class="trait">activity,</span> lower ammonia levels and faster growth than in PPT-treated controls. Greater tolerance of PPT was also seen in shoots regenerated from the hairy roots displaying elevated levels of <span class="gene">GS</span> <span class="trait">activity.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the soybean (Glycine max) glutamate 1-semialdehyde aminotransferase gene in symbiotic <span class="trait">root</span> nodules.</h2>
			<h3>PMID:8278535  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sangwan I(1), O'Brian MR.
				<br/> 
				年份：1993 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Extracts of soybean (Glycine max) <span class="trait">root</span> nodules and greening etiolated leaves catalyzed radiolabeled delta-aminolevulinic acid (ALA) formation from 3,4-[3H]glutamate but not from 1-[14C]glutamate. Nevertheless, those tissue extracts expressed the <span class="trait">activity</span> of glutamate 1-semialdehyde (GSA) aminotransferase, the <span class="gene">C5</span> pathway enzyme that catalyzes ALA synthesis from GSA for tetrapyrrole formation. A soybean nodule cDNA clone that conferred ALA prototrophy, GSA aminotransferase <span class="trait">activity,</span> and glutamate-dependent ALA formation <span class="trait">activity</span> on an Escherichia coli GSA aminotransferase mutant was isolated. The deduced product of the nodule cDNA shared 79% identity with the GSA aminotransferase expressed in barley leaves, providing, along with the complementation data, strong evidence that the cDNA encodes GSA aminotransferase. GSA aminotransferase mRNA and enzyme <span class="trait">activity</span> were expressed in nodules but not in uninfected roots, indicating that the Gsa gene is induced in the symbiotic tissue. The Gsa gene was strongly expressed in leaves of etiolated plantlets independently of light treatment and, to a much lesser extent, in leaves of mature plants. We conclude that GSA aminotransferase, and possibly the <span class="gene">C5</span> pathway, is expressed in a nonphotosynthetic plant organ for nodule heme synthesis and that Gsa is a regulated gene in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:cDNA sequence and expression of a phosphoenolpyruvate carboxylase gene from soybean.</h2>
			<h3>PMID:1450389  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sugimoto T(1), Kawasaki T, Kato T, Whittier RF, Shibata D, Kawamura Y.
				<br/> 
				年份：1992 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA encoding a subunit of phosphoenolpyruvate carboxylase (PEPC) was isolated from a developing seed expression library of the <span class="gene">C3</span> plant Glycine max. The corresponding mRNA is present at similar levels in leaf, stem, <span class="trait">root</span> and developing seed. Two potential start codons exist, and the <span class="trait">activity</span> of protein initiated from the first such codon could be subject to regulation by protein kinase. Sequence comparison shows a similar upstream start codon in the case of the Ppc2 gene from Mesembryanthemum crystallinum, previously assumed to lack the sequences necessary for phosphorylation. The soybean encoded protein tends to resemble other 'C3-type' PEPC proteins more closely than those implicated in <span class="gene">C4</span> or crassulacean acid metabolism.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Visualization of bioluminescence as a marker of gene expression in rhizobium-infected soybean <span class="trait">root</span> nodules.</h2>
			<h3>PMID:24277587</h3>
				

				
			<p>
				作者：O'Kane DJ(1), Lingle WL, Wampler JE, Legocki M, Legocki RP, Szalay AA.
				<br/> 
				年份：1988 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The linked structural genes lux A and lux B, encoding bacterial luciferase of a marine bacterium Vibrio harveyi, were fused with the nitrogenase <span class="gene">nifD</span> promoter from Bradyrhizobium japonicum and with the <span class="gene">P1</span> promoter of pBR322. Both fusions were integrated into the B. japonicum chromosome by site-specific recombination. Soybean roots infected with the two types of rhizobium transconjugants formed nitrogen-fixing nodules that produced bright blue-green light. Cells containing the <span class="gene">P1</span> promoter/lux AB fusion resulted in continuously expressed bioluminescence in both free-living rhizobium and in nodule bacteriods. However, when under control of the <span class="gene">nifD</span> promoter, luciferase <span class="trait">activity</span> was found only in introgen-fixing nodules. Light emission from bacteroids allowed us to visualize and to photograph nodules expressing this marker gene fusion in vivo at various levels of resolution, including within single, living plant cells. Localization of host cells containing nitrogen-fixing bacteroids within nodule tissue was accomplished using low-light video microscopy aided by realtime image processing techniques developed specifically to enhance extreme low-level luminescent images.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cellulose binding protein from the parasitic nematode Heterodera schachtii interacts with Arabidopsis pectin methylesterase: cooperative cell wall modification during parasitism.</h2>
			<h3>PMID:19001564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hewezi T(1), Howe P, Maier TR, Hussey RS, Mitchum MG, Davis EL, Baum TJ.
				<br/> 
				年份：2008 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-parasitic cyst nematodes secrete a complex of cell wall-digesting enzymes, which aid in <span class="trait">root</span> penetration and migration. The soybean cyst nematode Heterodera glycines also produces a cellulose binding protein (Hg CBP) secretory protein. To determine the function of CBP, an orthologous cDNA clone (Hs CBP) was isolated from the sugar beet cyst nematode Heterodera schachtii, which is able to infect Arabidopsis thaliana. <span class="gene">CBP</span> is expressed only in the early phases of feeding cell formation and not during the migratory phase. Transgenic Arabidopsis expressing Hs <span class="gene">CBP</span> developed longer roots and exhibited enhanced susceptibility to H. schachtii. A yeast two-hybrid screen identified Arabidopsis pectin methylesterase protein 3 (PME3) as strongly and specifically interacting with Hs CBP. Transgenic plants overexpressing PME3 also produced longer roots and exhibited increased susceptibility to H. schachtii, while a pme3 knockout mutant showed opposite phenotypes. Moreover, <span class="gene">CBP</span> overexpression increases PME3 <span class="trait">activity</span> in planta. Localization studies support the mode of action of PME3 as a cell wall-modifying enzyme. Expression of <span class="gene">CBP</span> in the pme3 knockout mutant revealed that PME3 is required but not the sole mechanism for <span class="gene">CBP</span> overexpression phenotype. These data indicate that <span class="gene">CBP</span> directly interacts with PME3 thereby activating and potentially targeting this enzyme to aid cyst nematode parasitism.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean cytosolic glutamine synthetase gene linked to organ-specific promoters in pea plants grown in different concentrations of nitrate.</h2>
			<h3>PMID:12520339  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fei H(1), Chaillou S, Hirel B, Mahon JD, Vessey JK.
				<br/> 
				年份：2003 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A glutamine synthetase gene ( GS15) coding for soybean cytosolic glutamine synthetase (GS1) fused to a constitutive promoter (CaMV 35S), a putative nodule-specific promoter (LBC(3)) and a putative root-specific promoter (rolD) was transformed into Pisum sativum L. cv. Greenfeast. Four lines with single copies of GS15 (one 35S-GS15 line, one LBC (3) -GS15 line, and two rolD-GS15 lines) were tested for the expression of GS15, levels of GS1, <span class="gene">GS</span> <span class="trait">activity,</span> N accumulation, N(2) fixation, and plant growth at different levels of nitrate. Enhanced levels of GS1 were detected in leaves of three transformed lines (the 35S-GS15 and rolD-GS15 transformants), in nodules of three lines (the LBC (3) -GS15 and rolD-GS15 transformants), and in roots of all four transformants. Despite increased levels of GS1 in leaves and nodules, there were no differences in <span class="gene">GS</span> <span class="trait">activity</span> in these tissues or in whole-plant N content, N(2) fixation, or biomass accumulation among all the transgenic lines and the wild-type control. However, the rolD-GS15 transformants, which displayed the highest levels of GS1 in the roots of all the transformants, had significantly higher <span class="gene">GS</span> <span class="trait">activity</span> in roots than the wild type. In one of the rolD-GS15 transformed lines (Line 8), increased <span class="trait">root</span> <span class="gene">GS</span> <span class="trait">activity</span> resulted in a lower N content and biomass accumulation, supporting the findings of earlier studies with Lotus japonicus (Limami et al. 1999 ). However, N content and biomass accumulation was not negatively affected in the other rolD-GS15 transformant (Line 9) and, in fact, these parameters were positively affected in the 0.1 mM treatment. These findings indicate that overexpression of GS15 in various tissues of pea does not consistently result in increases in <span class="gene">GS</span> <span class="trait">activity.</span> The current study also indicates that the increase in <span class="trait">root</span> <span class="gene">GS</span> <span class="trait">activity</span> is not always consistent with decreases in plant N and biomass accumulation and that further investigation of the relationship between <span class="trait">root</span> <span class="gene">GS</span> <span class="trait">activity</span> and growth responses is warranted.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phosphatidylcholine levels in Bradyrhizobium japonicum membranes are critical for an efficient symbiosis with the soybean host plant.</h2>
			<h3>PMID:11251836  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Minder AC(1), de Rudder KE, Narberhaus F, Fischer HM, Hennecke H, Geiger O.
				<br/> 
				年份：2001 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphatidylcholine (PC), the major membrane phospholipid in eukaryotes, is found in only some bacteria including members of the family Rhizobiaceae. For this reason, it has long been speculated that rhizobial PC might be required for a successful interaction of rhizobia with their legume host plants in order to allow the formation of nitrogen-fixing <span class="trait">root</span> nodules. A major pathway for PC formation in prokaryotes involves a threefold methylation of the precursor phosphatidylethanolamine (PE). Here, we report on the isolation of a Bradyrhizobium japonicum gene (pmtA) encoding the phospholipid N-methyltransferase PmtA. Upon expression of the bradyrhizobial <span class="gene">pmtA</span> gene in Escherichia coli, predominantly monomethylphosphatidylethanolamine was formed from PE. PmtA-deficient B. japonicum mutants still produced low levels of PC by a second methylation pathway. The amount of PC formed in such mutants (6% of total phospholipids) was greatly decreased compared with the wild type (52% of total phospholipids). <span class="trait">Root</span> nodules of soybean plants infected with B. japonicum <span class="gene">pmtA</span> mutants showed a nitrogen fixation <span class="trait">activity</span> of only 18% of the wild-type level. The interior colour of the nodules was beige instead of red, suggesting decreased amounts of leghaemoglobin. Moreover, ultrastructure analysis of these nodules demonstrated a greatly reduced number of bacteroids within infected plant cells. These data suggest that the biosynthesis of wild-type amounts of PC are required to allow for an efficient symbiotic interaction of B. japonicum with its soybean host plant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean ENOD40(2) promoter is active in Arabidopsis thaliana and is temporally and spatially regulated.</h2>
			<h3>PMID:10080720  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mirabella R(1), Martirani L, Lamberti A, Iaccarino M, Chiurazzi M.
				<br/> 
				年份：1999 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">ENOD40</span> gene is induced early during Rhizobium-legume symbiosis and has probably a primary role in the nodule organogenesis. In this paper we show that the 1.7 kb 5'-flanking region of the <span class="gene">GmENOD40(2)</span> is able to drive the expression of a gusA-int marker in transgenic Arabidopsis thaliana. The promoter <span class="trait">activity</span> is developmentally regulated and the major <span class="trait">activity</span> is detected in the <span class="trait">root</span> and in the stigma.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Calcium/Calmodulin Activation of Soybean Glutamate Decarboxylase.</h2>
			<h3>PMID:12228492</h3>
				

				
			<p>
				作者：Snedden WA(1), Arazi T, Fromm H, Shelp BJ.
				<br/> 
				年份：1995 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recently, we provided preliminary evidence for calcium (Ca2+)/calmodulin (CaM) stimulation of plant glutamate decarboxylase (GAD; EC 4.1.1.15). In the present study, a detailed characterization of the phenomenon is described. <span class="gene">GAD</span> was partially purified from various soybean (Glycine max L. Merr.) tissues (developing seed coat and cotyledons, leaf, and <span class="trait">root)</span> in the presence of EDTA by a combination of ammonium sulfate precipitation and anion-exchange fast protein liquid chromatography. <span class="gene">GAD</span> <span class="trait">activity</span> showed a sharp optimum at pH 5.8, with about 12% of maximal <span class="trait">activity</span> at pH 7. It was stimulated 2- to 8-fold (depending on the tissue source) in the presence of Ca2+/CaM at pH 7 but not at pH 5.8. Furthermore, when the protease inhibitor phenylmethylsulfonyl fluoride was omitted from the purification procedure, <span class="gene">GAD</span> <span class="trait">activity</span> was insensitive to Ca2+/CaM but was similar in magnitude to CaM-stimulated <span class="trait">activity.</span> The stimulation by Ca2+/CaM was fully inhibited by the <span class="gene">CaM</span> antagonists N-(6-aminohexyl)-5-chloro-1-naphthalenesulfon-amide and trifluoperazine. With saturating <span class="gene">CaM</span> or Ca2+, the concentrations of Ca2+ and <span class="gene">CaM</span> required for half-maximal stimulation were about 7 to 11 [mu]M and 25 nM, respectively. The effect of Ca2+ and <span class="gene">CaM</span> appeared to be through a 2.4-fold stimulation of Vmax and a 55% reduction in Km. The results suggested that <span class="gene">GAD</span> is activated via Ca2+ signal transduction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular cloning and expression of 4-coumarate:coenzyme A ligase, an enzyme involved in the <span class="trait">resistance</span> response of soybean (Glycine max L.) against pathogen attack.</h2>
			<h3>PMID:8278545  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Uhlmann A(1), Ebel J.
				<br/> 
				年份：1993 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have isolated three classes of cDNAs that probably encode three 4-coumarate:coenzyme A ligase (4CL) isoenzymes in soybean (Glycine max L.). The deduced amino acid sequences reveal several regions of extended sequence identity among 4CLs of all plants analyzed to date. The sequences of two of these regions are consistent with a domain structure proposed for a group of enzymes catalyzing the ATP-dependent covalent binding of AMP to their substrates during the reaction sequence. By using two cDNA fragments that do not cross-hybridize under the conditions used, we demonstrate that <span class="gene">4CL</span> in soybean is very likely encoded by a small gene family. Members of this family are differentially expressed in soybean cell cultures treated with beta-glucan elicitors of Phytophthora megasperma f. sp. glycinea or in soybean roots infected with either an incompatible or compatible race of the fungus. These results are in agreement with our previous observation that elicitor treatment of soybean cells caused a preferential enhancement in the <span class="trait">activity</span> level of one of the <span class="gene">4CL</span> isoenzymes. In soybean, <span class="gene">4CL</span> isoenzymes possessing different substrate affinities for substituted cinnamic acids, and showing differential regulation to environmental stress, may play a pivotal role in distributing substituted cinnamate intermediates at a branch point of general phenylpropanoid metabolism into subsequent specific pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">SecG</span> is required for antibiotic activities of Pseudomonas sp. YL23 against Erwinia amylovora and Dickeya chrysanthemi.</h2>
			<h3>PMID:25708938  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu Y(1), Baird SM, Qiao J, Du Y, Lu SE.
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Strain YL23 was isolated from soybean <span class="trait">root</span> tips and identified to be Pseudomonas sp. This strain showed broad-spectrum antibacterial <span class="trait">activity</span> against bacterial pathogens that are economically important in agriculture. To characterize the genes dedicated to antibacterial activities against microbial phytopathogens, a Tn5-mutation library of YL23 was constructed. Plate bioassays revealed that the mutant YL23-93 lost its antibacterial activities against Erwinia amylovora and Dickeya chrysanthemi as compared with its wild type strain. Genetic and sequencing analyses localized the transposon in a homolog of the <span class="gene">secG</span> gene in the mutant YL23-93. Constitutive expression plasmid pUCP26-secG was constructed and electroporated into the mutant YL23-93. Introduction of the plasmid pUCP26-secG restored antibacterial activities of the mutant YL23-93 to E. amylovora and D. chrysanthemi. As expected, empty plasmid pUCP26 could not complement the phenotype of the antibacterial <span class="trait">activity</span> in the mutant. Thus the <span class="gene">secG</span> gene, belonging to the Sec protein translocation system, is required for antibacterial <span class="trait">activity</span> of strain YL23 against E. amylovora and D. chrysanthemi.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Methane-rich water induces cucumber adventitious rooting through heme oxygenase1/carbon monoxide and Ca(2+) pathways.</h2>
			<h3>PMID:25503851  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cui W(1), Qi F, Zhang Y, Cao H, Zhang J, Wang R, Shen W.
				<br/> 
				年份：2015 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Methane-rich water triggered adventitious rooting by regulating heme oxygenase1/carbon monoxide and calcium pathways in cucumber explants. Heme oxygenase1/carbon monoxide (HO1/CO) and calcium (Ca(2+)) were reported as the downstream signals in auxin-induced cucumber adventitious <span class="trait">root</span> (AR) formation. Here, we observed that application of methane-rich water (MRW; 80% saturation) obviously induced AR formation in IAA-depleted cucumber explants. To address the universality, we checked adventitious rooting in soybean and mung bean explants, and found that MRW (50 and 10% saturation, respectively) exhibited the similar inducing results. To further determine if the HO1/CO system participated in MRW-induced adventitious rooting, MRW, <span class="gene">HO1</span> inducer hemin, its <span class="trait">activity</span> inhibitor zinc protoporphyrin IX (ZnPP), and its catalytic by-products CO, bilirubin, and Fe(2+) were used to detect their effects on cucumber adventitious rooting in IAA-depleted explants. Subsequent results showed that MRW-induced adventitious rooting was blocked by ZnPP and further reversed by 20% saturation CO aqueous solution. However, the other two by-products of HO1, bilirubin and Fe(2+), failed to induce AR formation. Above responses were consistent with the MRW-induced increases of <span class="gene">HO1</span> transcript and corresponding protein level. Further molecular evidence indicted that expression of marker genes, including auxin signaling-related genes and cell cycle regulatory genes, were modulated by MRW alone but blocked by the cotreatment with ZnPP, the latter of which could be significantly rescued by the addition of CO. By using the Ca(2+)-channel blocker and Ca(2+) chelator, the involvement of Ca(2+) pathway in MRW-induced adventitious rooting was also suggested. Together, our results indicate that MRW might serve as a stimulator of adventitious rooting, which was partially mediated by HO1/CO and Ca(2+) pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a gene associated with sulfate assimilation in Sinorhizobium fredii WGF03.</h2>
			<h3>PMID:25182128  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song Z(1), Shen P, Ma T, Jiang C, Zhao H, Wu B.
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sulfur is an essential element for rhizobia, such as sulfated modified Nod factors and nitrogenase. To investigate the role of sulfur metabolism in Rhizobium-Soybean symbiosis, a transponson random insertional mutants' library was constructed and a sulfur assimilation-related gene was isolated and characterized. A mutant strain unable to utilized sulfate was screened from 11,000 random insertional mutants of Sinorhizobium fredii WGF03. Sequencing analysis showed that a sulfate assimilation-related gene (cysDN) was inserted by the Tn transponson. Mutants inactivated in <span class="gene">cysD</span> and <span class="gene">cysN</span> (SMcysDF and SMcysNF) were constructed by homologous recombination using the suicide plasmid pK18mob. The mutants SMcysDF and SMcysNF could no longer utilize sulfate as sulfur source. Phenotype analysis revealed that mutation of cysDN had multiple effects on S. fredii WGF03. <span class="trait">Root</span> hair deformation assay showed that the <span class="trait">activity</span> of Nod factors secreted by mutants SMcysDR and SMcysNR elicited minimal hair initiation only. Soybean plant tests indicated that the mutant strains delayed 1-2 days to nodulate and exhibited lower nodulation efficiency and symbiotic efficiency than the wild-type strain. The complementary strain of <span class="gene">cysD</span> and <span class="gene">cysN</span> (HcysDF and HcysNF) could restore the nodulation efficiency.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Activity of the Arabidopsis <span class="gene">RD29A</span> and RD29B promoter elements in soybean under water stress.</h2>
			<h3>PMID:22983672  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bihmidine S(1), Lin J, Stone JM, Awada T, Specht JE, Clemente TE.
				<br/> 
				年份：2013 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The constitutive and drought-induced activities of the Arabidopsis thaliana <span class="gene">RD29A</span> and RD29B promoters were monitored in soybean (Glycine max (L.) Merr.] via fusions with the visual marker gene β-glucuronidase (GUS). Physiological responses of soybean plants were monitored over 9 days of water deprivation under greenhouse conditions. Data were used to select appropriate time points to monitor the activities of the respective promoter elements. Qualitative and quantitative assays for GUS expression were conducted in <span class="trait">root</span> and leaf tissues, from plants under well-watered and dry-down conditions. Both <span class="gene">RD29A</span> and RD29B promoters were significantly activated in soybean plants subjected to dry-down conditions. However, a low level of constitutive promoter <span class="trait">activity</span> was also observed in both <span class="trait">root</span> and leaves of plants under well-watered conditions. GUS expression was notably higher in roots than in leaves. These observations suggest that the respective drought-responsive regulatory elements present in the RD29X promoters may be useful in controlling targeted transgenes to mitigate abiotic stress in soybean, provided the transgene under control of these promoters does not invoke agronomic penalties with leaky expression when no abiotic stress is imposed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distinct repressing modules on the distal region of the <span class="gene">SBP2</span> promoter contribute to its vascular tissue-specific expression in different vegetative organs.</h2>
			<h3>PMID:17710554  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Freitas RL(1), Carvalho CM, Fietto LG, Loureiro ME, Almeida AM, Fontes EP.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Glycine max sucrose binding protein <span class="gene">(GmSBP2)</span> promoter directs vascular tissue-specific expression of reporter genes in transgenic tobacco. Here we showed that an SBP2-GFP fusion protein under the control of the <span class="gene">GmSBP2</span> promoter accumulates in the vascular tissues of vegetative organs, which is consistent with the proposed involvement of <span class="gene">SBP</span> in sucrose transport-dependent physiological processes. Through gain-of-function experiments we confirmed that the tissue-specific determinants of the <span class="gene">SBP2</span> promoter reside in the distal cis-regulatory domain A, CRD-A (position -2000 to -700) that is organized into a modular configuration to suppress promoter <span class="trait">activity</span> in tissues other than vascular tissues. The four analyzed CRD-A sub-modules, designates Frag II (-1785/-1508), Frag III (-1507/-1237), Frag IV (-1236/-971) and Frag V (-970/-700), act independently to alter the constitutive pattern of -92pSBP2-mediated GUS expression in different organs. Frag V fused to -92pSBP2-GUS restored the tissue-specific pattern of the full-length promoter in the shoot apex, but not in other organs. Likewise, Frag IV confined GUS expression to the vascular bundle of leaves, whereas Frag II mediated vascular specific expression in roots. Strong stem expression-repressing elements were located at positions -1485 to -1212, as Frag III limited GUS expression to the inner phloem. We have also mapped a procambium silencer to the consensus sequence CAGTTnCaAccACATTcCT which is located in both distal and proximal upstream modules. Fusion of either repressing element-containing module to the constitutive -92pSBP2 promoter suppresses GUS expression in the elongation zone of roots. Together our results demonstrate the unusual aspect of distal sequences negatively controlling tissue-specificity of a plant promoter.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impact of glyphosate on the Bradyrhizobium japonicum symbiosis with glyphosate-resistant transgenic soybean: a minireview.</h2>
			<h3>PMID:15224916  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zablotowicz RM(1), Reddy KN.
				<br/> 
				年份：2004 May-Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glyphosate-resistant (GR) soybean [Glycine max (L.) Merr.] expressing an insensitive 5-enolpyruvylshikimic acid-3-phosphate synthase (EPSPS) gene has revolutionized weed control in soybean production. The soybean nitrogen fixing symbiont, Bradyrhizobium japonicum, possesses a glyphosate-sensitive enzyme and upon exposure to glyphosate accumulates shikimic acid and hydroxybenzoic acids such as protocatechuic acid (PCA), accompanied with B. japonicum growth inhibition and death at high concentrations. In a series of greenhouse and field experiments, glyphosate inhibited nodulation and nodule leghemoglobin content of <span class="gene">GR</span> soybean. Glyphosate accumulated in nodules of field-grown <span class="gene">GR</span> soybean, but its effect on nitrogenase <span class="trait">activity</span> of <span class="gene">GR</span> soybean was inconsistent in field studies. In greenhouse studies, nitrogenase <span class="trait">activity</span> of <span class="gene">GR</span> soybean following glyphosate application was transiently inhibited especially in early growth stages, with the greatest inhibition occurring under moisture stress. Studies using bacteroid preparations showed that the level of glyphosate inhibition of bacteroid nitrogenase <span class="trait">activity</span> was related to in vitro glyphosate sensitivity of the B. japonicum strains. These studies indicate the potential for reduced nitrogen fixation in the <span class="gene">GR</span> soybean system; however, <span class="trait">yield</span> reductions due to this reduced N2 fixation in early stages of growth have not been demonstrated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean gene encoding cytosolic glutamine synthetase in shoots of transgenic Lotus corniculatus L. plants triggers changes in ammonium assimilation and plant development.</h2>
			<h3>PMID:9151446  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vincent R(1), Fraisier V, Chaillou S, Limami MA, Deleens E, Phillipson B, Douat C, Boutin JP, Hirel B.
				<br/> 
				年份：1997
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A soybean cytosolic glutamine synthetase gene (GS15) was fused with the constitutive 35S cauliflower mosaic virus (CaMV) promoter in order to direct overexpression in Lotus corniculatus L. plants. Following transformation with Agrobacterium rhizogenes, eight independent Lotus transformants were obtained which synthesized additional cytosolic glutamine synthetase (GS) in the shoots. To eliminate any interference caused by the T-DNA from the Ri plasmid, three primary transformants were crossed with untransformed plants and progeny devoid of TL- and TR-DNA sequences were chosen for further analyses. These plants had a 50-80% increase in total leaf <span class="gene">GS</span> <span class="trait">activity.</span> Plants were grown under different nitrogen regimes (4 or 12 mM NH4+) and aspects of carbon and nitrogen metabolism were examined. In roots, an increase in free amino acids and ammonium was accompanied by a decrease in soluble carbohydrates in the transgenic plants cultivated with 12 mM NH4+ in comparison to the wild type grown under the same conditions. Labelling experiments using 15NH4+ were carried out in order to monitor the influx of ammonium and its subsequent incorporation into amino acids. This experiment showed that both ammonium uptake in the roots and the subsequent translocation of amino acids to the shoots was lower in plants overexpressing GS. It was concluded that the build up of ammonium and the increase in amino acid concentration in the roots was the result of shoot protein degradation. Moreover, following three weeks of hydroponic culture early floral development was observed in the transformed plants. As all these properties are characteristic of senescent plants, these findings suggest that expression of cytosolic <span class="gene">GS</span> in the shoots may accelerate plant development, leading to early senescence and premature flowering when plants are grown on an ammonium-rich medium.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Root Interactions in a Maize/Soybean Intercropping System Control Soybean Soil-Borne Disease, Red Crown Rot</h2>
			<h3>PMID:24810161</h3>
				
					<h3>PMCID:4014482</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gao </surname>  <given-names> Xiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Man </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ruineng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiurong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Ruqian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Hye-Ji </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The results displayed in <xref ref-type="fig" rid="pone-0095031-g003"> Figure 3 </xref> showed that transcript abundances of all tested <italic> PR </italic> genes in soybean roots were induced to a high level by inoculation with <italic> C. parasiticum </italic> , and this induction was enhanced by increasing <span class="trait">root</span> interactions. Except <italic> PR4 </italic> on day 5 and <italic> PR3 </italic> , <italic> PR10 </italic> , <italic> PR12 </italic> and <italic> <span class="gene">PAL</span> </italic> on day 1, the relative expression values of most tested <italic> PR </italic> genes were highest in the no barrier treatment, followed by the mesh barrier treatment. For example, <italic> PPO </italic> transcription abundance increased by 7.3- and 7.4- fold, and <italic> <span class="gene">PAL</span> </italic> by 12.9- and 4.1- fold in LP and HP, respectively, at 5 DAI in a comparison of the no barrier treatment with the solid barrier treatment ( <xref ref-type="fig" rid="pone-0095031-g003"> Fig. 3G and H </xref> ). Furthermore, the enzyme activities of PPO and <span class="gene">PAL</span> in soybean roots were significantly enhanced by <span class="trait">root</span> interactions with maize ( <xref ref-type="supplementary-material" rid="pone.0095031.s001"> Figure D in File S1 </xref> ). PPO <span class="trait">activity</span> increased in the no barrier treatment compared to the solid barrier treatment at 1 DAI by 114% and 99%, and in the mesh treatment by 66% and 43% in LP and HP, respectively. Also, <span class="gene">PAL</span> <span class="trait">activity</span> increased in the no barrier treatment compared to the solid barrier treatment at 1 DAI by 65% and 227%, and in the mesh treatment by 39% and 105% in LP and HP, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						In comparison with soybean monoculture (MS), the addition of <span class="trait">root</span> exudates from the maize/soybean intercropping (ISC) or maize monoculture (MC) dramatically inhibited pathogen growth ( <xref ref-type="fig" rid="pone-0095031-g004"> Fig. 4 </xref> ). At LP, colony diameter of <italic> C. parasiticum </italic> decreased by 54% and 38%, and sporulation declined by 37% and 27% when growth in <span class="gene">MS</span> exudates was compared to that in ISC or MC exudates, respectively. Furthermore, P supply also affected pathogen growth as indicated by the observed 34%, 31% and 49% increases in colony diameter in HP compared to LP when adding the <span class="trait">root</span> exudates from MS, MC and ISC, respectively. This suggests that maize roots can be triggered to secrete antagonistic substances against soybean pathogen growth in the intercropping system, which might be enhanced under low P supply.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> PR </italic> related genes included pathogenesis-related, antimicrobial protein genes <italic> PR1 </italic> (PR1a precursor), <italic> PR2 </italic> (&#x003b2; 1&#x02013;3 endoglucanase), <italic> PR3 </italic> (chitinase class I), <italic> PR4 </italic> (wound-induced protein), <italic> PR10 </italic> (ribonuclease-like protein), <italic> PR12 </italic> (defending-like protein), <italic> PPO </italic> (polyphenol oxidase) and <italic> <span class="gene">PAL</span> </italic> (phenylalanine ammonia-lyase) that are involved in phytoalexin biosynthesis <xref rid="pone.0095031-Robert1" ref-type="bibr"> [33] </xref> , <xref rid="pone.0095031-VanLoon1" ref-type="bibr"> [34] </xref> . Here we shown that the defense responses elicited by inoculation of pathogen and intercropping with maize. In legumes, many genes encoding PR proteins have been shown to be upregulated in plants following inoculation with several pathogens <xref rid="pone.0095031-Robert1" ref-type="bibr"> [33] </xref> . The upregulation of defense-related genes, such as <italic> PR1 </italic> and <italic> PR2 </italic> was associated with lesion limitation in soybean seedling roots inoculated with <italic> P. sojae </italic> and to the high level of partial <span class="trait">resistance</span> of cultivar Conrad <xref rid="pone.0095031-Robert1" ref-type="bibr"> [33] </xref> . Defense-related genes are upregulated in both compatible and incompatible interactions, but more rapidly and/or to higher levels in plant varieties with <italic> PR </italic> gene <span class="trait">resistance</span> <xref rid="pone.0095031-VanLoon1" ref-type="bibr"> [34] </xref> . In addition to producing allelopathic compounds directly, intercropped plants can promote specific plant defense reactions in neighboring plants to protect against phytopathogens <xref rid="pone.0095031-Zhu1" ref-type="bibr"> [2] </xref> , <xref rid="pone.0095031-Bais3" ref-type="bibr"> [31] </xref> . Consistent with this, the results herein show that direct interactions with maize roots significantly induce the expression of important soybean <italic> PR </italic> genes and increase the activities of corresponding PR enzymes in soybean roots ( <xref ref-type="fig" rid="pone-0095031-g003"> Fig. 3 </xref> , <xref ref-type="supplementary-material" rid="pone.0095031.s001"> Figure C in File S1 </xref> ). This elicitation, via <span class="trait">root</span> exudates or signal compounds for specific plant defense reactions can predispose soybean plants to an early response to attack by <span class="trait">root</span> pathogens <xref rid="pone.0095031-Trenbath1" ref-type="bibr"> [24] </xref> , <xref rid="pone.0095031-Bais3" ref-type="bibr"> [31] </xref> . Infecting soybean roots after pre-treatment with maize enhances PR gene expression and enzyme <span class="trait">activity,</span> and therefore, might improve soybean's capability to defend against disease invasion. Furthermore, salicylic acid is a reported enhancer of plant defense pathways <xref rid="pone.0095031-Wees1" ref-type="bibr"> [35] </xref> . In this study, more salicylic acid was measured in <span class="trait">root</span> exudates from the maize/soybean intercropping system than in monoculture, which suggests that salicylic acid or a related component might also contribute to disease control through promotion of defense pathways.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Supporting figures. </bold> <bold> Figure A. </bold> Planting arrangement of diagram and actual pictures in monoculture or intercropping of soybean and maize. <bold> A </bold> , Planting arrangement in the field. <bold> B </bold> , Three <span class="trait">root</span> barriers were set-up in sand culture, either with a solid barrier (left) to eliminate <span class="trait">root</span> interactions and exudates movement, a nylon mesh barrier (middle) to prevent <span class="trait">root</span> intermingling of two species while permitting <span class="trait">root</span> exudates exchange, and no barrier (right) to allow roots and exudates to completely interact. <bold> Figure B. </bold> Soybean growth and grain <span class="trait">yield</span> as affected by red crown rot. HP: 80 kg P <sub> 2 </sub> O <sub> 5 </sub> ha <sup> &#x02212;1 </sup> added as calcium superphosphate, LP: no P fertilizer added. The reduction of growth parameters were calculated as follows: reduction (%) &#x0200a;=&#x0200a; (yield/biomass of healthy plants- yield/biomass of infected plants) yield/biomass of healthy plants &#x000d7;100. Each bar represents the mean of four replicates &#x000b1; SE. <bold> Figure C. </bold> Disease severity of soybean red crown rot in sand culture (Second round). A, disease incidence, B, disease index, C, CFU. LP, 15 &#x000b5;M P; HP, 500 &#x000b5;M P. All of the data are the means of four replicates &#x000b1;SE. Bars with different letter(s) vary significantly among treatments as determined by Duncan's multiple range test ( <italic> P </italic> &#x0003c;0.05). <bold> Figure D. </bold> Activities of the enzyme PPO (A) and <span class="gene">PAL</span> (B) in soybean roots. LP, 15 &#x000b5;M P; HP, 500 &#x000b5;M P. Except for control, all the roots were inoculated with <italic> C. parasiticum </italic> (see Materials and Methods for details). The solid barrier eliminated <span class="trait">root</span> contact and exudates movement, the nylon mesh (30 &#x000b5;m) barrier prevented <span class="trait">root</span> intermingling for the two species while permitting <span class="trait">root</span> exudates exchange, and no <span class="trait">root</span> barrier permitted roots and exudates to completely interact. Each bar represents the mean of three replicates &#x000b1; SE. <bold> Figure E. </bold> HPLC scan of soybean and/or maize <span class="trait">root</span> exudates. <bold> (a and b </bold> ) HPLC scan of soybean <span class="trait">root</span> exudates, ( <bold> c and d </bold> ) HPLC scan of maize <span class="trait">root</span> exudates, ( <bold> e and f </bold> ) HPLC scan of the <span class="trait">root</span> exudates from maize/soybean intercropping, ( <bold> g </bold> ) HPLC analysis of eight standard phenolic acids. LP, 15 &#x000b5;M P (a, c and e); HP, 500 &#x000b5;M P (b, d and f). 1, Gallic acid; 2, <italic> P </italic> -coumaric; 3, Phthalic acid; 4, Vanillic acid; 5, Syringic acid; 6, Ferulic acid; 7, Salicylic acid; 8, Cinnamic acid.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nodule-Enriched GRETCHEN HAGEN 3 Enzymes Have Distinct Substrate Specificities and Are Important for Proper Soybean Nodule Development</h2>
			<h3>PMID:29182530</h3>
				

				
					<h3>PMCID:5751150</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Damodaran </surname>  <given-names> Suresh </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-02547"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Westfall </surname>  <given-names> Corey S. </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-02547"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kisely </surname>  <given-names> Brian A. </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-02547"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jez </surname>  <given-names> Joseph M. </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-02547"> 2 </xref>  <xref rid="c1-ijms-18-02547" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="true"> https://orcid.org/0000-0002-7403-1139 </contrib-id>  <name>  <surname> Subramanian </surname>  <given-names> Senthil </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-02547"> 1 </xref>  <xref ref-type="aff" rid="af3-ijms-18-02547"> 3 </xref>  <xref rid="c1-ijms-18-02547" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume <span class="trait">root</span> nodules develop as a result of a symbiotic relationship between the plant and nitrogen-fixing rhizobia bacteria in soil. Auxin <span class="trait">activity</span> is detected in different cell types at different stages of nodule development; as well as an enhanced sensitivity to auxin inhibits, which could affect nodule development. While some transport and signaling mechanisms that achieve precise spatiotemporal auxin output are known, the role of auxin metabolism during nodule development is unclear. Using a soybean <span class="trait">root</span> lateral organ transcriptome data set, we identified distinct nodule enrichment of three genes encoding auxin-deactivating GRETCHEN HAGEN 3 (GH3) indole-3-acetic acid (IAA) amido transferase enzymes: <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14</span> and <span class="gene">GmGH3-15.</span> In vitro enzymatic assays showed that each of these <span class="gene">GH3</span> proteins preferred IAA and aspartate as acyl and amino acid substrates, respectively. <span class="gene">GmGH3-15</span> showed a broad substrate preference, especially with different forms of auxin. Promoter:GUS expression analysis indicated that <span class="gene">GmGH3-14</span> acts primarily in the <span class="trait">root</span> epidermis and the nodule primordium where as <span class="gene">GmGH3-15</span> might act in the vasculature. Silencing the expression of these <span class="gene">GH3</span> genes in soybean composite plants led to altered nodule numbers, maturity, and size. Our results indicate that these GH3s are needed for proper nodule maturation in soybean, but the precise mechanism by which they regulate nodule development remains to be explained.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Members of the GRETCHEN HAGEN3 (GH3) family of acyl amido transferase enzymes can conjugate IAA to amino acids [ <xref rid="B2-ijms-18-02547" ref-type="bibr"> 2 </xref> ]. The first <italic> <span class="gene">GH3</span> </italic> gene was identified in soybean through a screen for auxin responsive gene expression [ <xref rid="B16-ijms-18-02547" ref-type="bibr"> 16 </xref> ]. Subsequently, <italic> <span class="gene">GH3</span> </italic> family members were identified in other plant species including Arabidopsis, and found to play critical roles in plant development through the conjugation of various plant hormones [ <xref rid="B2-ijms-18-02547" ref-type="bibr"> 2 </xref> ]. For example, a change in local auxin pool is achieved at the site of organ development or in response to biotic/abiotic interaction through conjugation of IAA by <span class="gene">GH3</span> proteins [ <xref rid="B17-ijms-18-02547" ref-type="bibr"> 17 </xref> , <xref rid="B18-ijms-18-02547" ref-type="bibr"> 18 </xref> , <xref rid="B19-ijms-18-02547" ref-type="bibr"> 19 </xref> , <xref rid="B20-ijms-18-02547" ref-type="bibr"> 20 </xref> ]. A gain of function mutation in an Arabidopsis <italic> <span class="gene">GH3</span> </italic> gene, <italic> wes1-D </italic> conferred <span class="trait">resistance</span> against multiple factors and a loss of function mutation in the same gene led to reduced <span class="trait">resistance</span> [ <xref rid="B19-ijms-18-02547" ref-type="bibr"> 19 </xref> ]. A gain-of-function mutation in another Arabidopsis <italic> <span class="gene">GH3</span> </italic> gene, <italic> ydk1-D </italic> led to reduced <span class="trait">root</span> length and lateral <span class="trait">root</span> density because of altered auxin <span class="trait">activity</span> [ <xref rid="B21-ijms-18-02547" ref-type="bibr"> 21 </xref> ]. An activation-tagged Arabidopsis line with increased expression of GH3.9 exhibited increased sensitivity to IAA, resulting in reduced <span class="trait">root</span> growth [ <xref rid="B22-ijms-18-02547" ref-type="bibr"> 22 </xref> ]. Recently, the X-ray crystal structures of IAA- and jasmonate-conjugating <span class="gene">GH3</span> proteins were determined. This has revealed key features of substrate recognition and to the re-classification of the <span class="gene">GH3</span> enzyme family into different groups based on the preference of the acyl acid substrate [ <xref rid="B23-ijms-18-02547" ref-type="bibr"> 23 </xref> , <xref rid="B24-ijms-18-02547" ref-type="bibr"> 24 </xref> ]. Group II <span class="gene">GH3</span> proteins catalyze IAA-amino acid conjugation and alter the free IAA pool to regulate various plant developmental programs in Arabidopsis and other plant species [ <xref rid="B25-ijms-18-02547" ref-type="bibr"> 25 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Determinate and indeterminate nodules also display similarities and differences in the overall distribution of auxin <span class="trait">activity</span> during nodule development. As mentioned above, local auxin <span class="trait">activity</span> indicated by marker gene expression occurs in the nodule initials and nodule primordia of determinate nodules (soybean and <italic> L. japonicus </italic> ), as well as indeterminate nodules (white clover and <italic> M. truncatula </italic> ) [ <xref rid="B28-ijms-18-02547" ref-type="bibr"> 28 </xref> , <xref rid="B35-ijms-18-02547" ref-type="bibr"> 35 </xref> , <xref rid="B37-ijms-18-02547" ref-type="bibr"> 37 </xref> , <xref rid="B39-ijms-18-02547" ref-type="bibr"> 39 </xref> ]. Auxin responsive gene expression is significantly diminished/absent in the infection zone of determinate nodules; however, the nodule meristem and invasion zone of indeterminate nodules continue to display auxin response gene expression. In mature nodules, auxin <span class="trait">activity</span> is detectable in the vasculatures of both determinate and indeterminate nodules (e.g., [ <xref rid="B28-ijms-18-02547" ref-type="bibr"> 28 </xref> , <xref rid="B35-ijms-18-02547" ref-type="bibr"> 35 </xref> ]). Therefore, precise regulation of auxin <span class="trait">activity</span> appears to occur during nodule development. While auxin transport appears to dictate auxin distribution during initiation of indeterminate nodules, it is unclear what mechanisms contribute to it during determinate nodule initiation. Multiple microRNA-regulated AUXIN RESPONSE FACTORs (ARFs) that might act in concert to dictate precise spatiotemporal auxin sensitivity during nodule development are also known [ <xref rid="B26-ijms-18-02547" ref-type="bibr"> 26 </xref> , <xref rid="B28-ijms-18-02547" ref-type="bibr"> 28 </xref> , <xref rid="B29-ijms-18-02547" ref-type="bibr"> 29 </xref> , <xref rid="B30-ijms-18-02547" ref-type="bibr"> 30 </xref> , <xref rid="B31-ijms-18-02547" ref-type="bibr"> 31 </xref> ]. However, the role of auxin metabolism in regulating auxin homeostasis during nodule development remains unclear. Flavonoids that accumulate at the sites of nodule initiation can inhibit peroxidases capable of degrading auxin and this has been suggested as a possible mechanism for auxin accumulation in these tissues [ <xref rid="B43-ijms-18-02547" ref-type="bibr"> 43 </xref> ]. Transient induction of <italic> TRYPTOPHAN AMINOTRANSFERASE RELATED1 </italic> , a paralog of <italic> TAA </italic> , occurs in response to rhizobium inoculation in <italic> L. japonicus </italic> [ <xref rid="B38-ijms-18-02547" ref-type="bibr"> 38 </xref> ]. In soybean, we have shown enrichment of <italic> YUCCA </italic> , <italic> <span class="gene">GH3</span> </italic> , and IAA oxidase gene expression in emerging nodules (Damodaran et al. unpublished data [ <xref rid="B44-ijms-18-02547" ref-type="bibr"> 44 </xref> ]). In <italic> M. truncatula </italic> , the expression of several <italic> <span class="gene">GH3</span> </italic> genes is induced in <italic> Sinorhizobium meliloti </italic> treated roots [ <xref rid="B45-ijms-18-02547" ref-type="bibr"> 45 </xref> ]. Similarly, rhizobium-responsive expression of auxin conjugate hydrolases capable of hydrolyzing the ester bonds of IAA-glucose and thus releasing free IAA have also been reported [ <xref rid="B46-ijms-18-02547" ref-type="bibr"> 46 </xref> ]. Rhizobia are also capable of synthesizing auxin [ <xref rid="B47-ijms-18-02547" ref-type="bibr"> 47 </xref> , <xref rid="B48-ijms-18-02547" ref-type="bibr"> 48 </xref> ]. Therefore, expression of auxin-modifying enzymes during nodule development is likely to enable the plant to efficiently regulate rhizobia-derived auxin as well. While these observations suggested that local auxin metabolism might contribute to auxin output during nodule initiation and development, no functional evidence existed for this hypothesis. We sought to identify the roles of auxin-conjugating <span class="gene">GH3</span> proteins in soybean nodule development.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we identified three <italic> <span class="gene">GH3</span> </italic> genes with preferential expression during nodule development and characterized their enzymatic <span class="trait">activity</span> through in vitro assays. We also evaluated their expression patterns in roots and nodules of soybean, and their functional significance during nodule development by knocking down their expression using artificial microRNAs. We show that these <span class="gene">GH3</span> proteins have distinct expression patterns in soybean, and show highest <span class="trait">activity</span> towards IAA-Asp conjugation, but have distinct specificities especially for other acyl substrates. Suppression of <span class="gene">GH3</span> protein <span class="trait">activity</span> led to alterations in nodule number and nodule size indicating that these enzymes play important roles in soybean nodule development likely via their effect on auxin homeostasis.
					</p>
				
					<p>&nbsp;&nbsp;
						Nodule-enriched <italic> <span class="gene">GH3</span> </italic> genes in soybean were identified from our RNA-seq dataset on emerging nodules (EN), mature nodules (MN), emerging lateral roots (ELR), and young lateral roots (YLR) (Damodaran et al. unpublished data [ <xref rid="B44-ijms-18-02547" ref-type="bibr"> 44 </xref> ]; <xref ref-type="app" rid="app1-ijms-18-02547"> Table S1 </xref> ). Adjacent <span class="trait">root</span> segments above and below these organs were used as age- and rhizobium inoculation-appropriate controls to determine <italic> <span class="gene">GH3</span> </italic> genes specifically enriched in nodules versus lateral roots at two different stages of development ( <xref ref-type="app" rid="app1-ijms-18-02547"> Table S1 </xref> ). Among a total of five <italic> <span class="gene">GH3</span> </italic> genes that showed enrichment in nodule tissues, three with high expression and enrichment in either emerging or mature nodule tissues were selected (highlighted in <xref ref-type="app" rid="app1-ijms-18-02547"> Table S1 </xref> ; <xref ref-type="fig" rid="ijms-18-02547-f001"> Figure 1 </xref> A). These three showed the highest expression values with two of the three GH3s showing nodule-specific enrichment. The three genes were named as <italic> <span class="gene">GmGH3-11/12</span> </italic> <span class="gene">(Glyma11g05510</span> (a1. v1.1), <span class="gene">Glyma.11g051600</span> (a2. v1.1)), <italic> <span class="gene">GmGH3-14</span> </italic> <span class="gene">(Glyma01g39780</span> (a1. v1.1), <span class="gene">Glyma.01g190600</span> (a2. v1.1)), and <italic> <span class="gene">GmGH3-15</span> </italic> <span class="gene">(Glyma12g17510</span> (a1. v1.1), <span class="gene">Glyma.12g141000</span> (a2. v1.1)) based on the nomenclature/classification of the 25 soybean <italic> <span class="gene">GH3</span> </italic> genes previously [ <xref rid="B23-ijms-18-02547" ref-type="bibr"> 23 </xref> ]. Gene IDs in parenthesis correspond to those of soybean genome assembly release a1.v1.1 and a2.v1.1 ( <uri xlink:href="www.phytozome.net"> www.phytozome.net </uri> ). The three <italic> <span class="gene">GmGH3</span> </italic> genes used in this study were classified under group II GH3s that catalyze IAA conjugation.
					</p>
				
					<p>&nbsp;&nbsp;
						Enzymatic activities of the nodule enriched <span class="gene">GH3</span> proteins were evaluated using in vitro enzyme kinetics assays. Full-length proteins were expressed in bacterial cells, and purified for biochemical assays in which the conjugation of the 20 amino acids to IAA were evaluated. <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14,</span> and <span class="gene">GmGH3-15</span> all displayed a clear preference for conjugation of IAA to aspartate ( <xref ref-type="fig" rid="ijms-18-02547-f002"> Figure 2 </xref> ). <italic> <span class="gene">GmGH3-11/12</span> </italic> had a specific <span class="trait">activity</span> of 296.2 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> with aspartate and much lower specific activities with tryptophan (51.12 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ) and methionine (26.05 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ) ( <xref ref-type="fig" rid="ijms-18-02547-f002"> Figure 2 </xref> A). <span class="gene">GmGH3-14</span> had a specific <span class="trait">activity</span> of 305.9 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> with aspartate, and much lower rates with methionine (51.8 nmol&#x000b7;min <sup> &#x02212;1 </sup> mg protein <sup> &#x02212;1 </sup> ) and tryptophan (44.4 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ) ( <xref ref-type="fig" rid="ijms-18-02547-f002"> Figure 2 </xref> B). The specific <span class="trait">activity</span> profile of <span class="gene">GmGH3-15</span> was similar with conjugation of IAA to aspartate (377.8 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ) as the primary function, although methionine (26.1 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ), cysteine (24.4 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ), and tryptophan (24.9 nmol&#x000b7;min <sup> &#x02212;1 </sup> &#x000b7;mg&#x000b7;protein <sup> &#x02212;1 </sup> ) were accepted as amino acid substrates ( <xref ref-type="fig" rid="ijms-18-02547-f002"> Figure 2 </xref> C).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GH3</span> proteins are capable of generating conjugates of different plant hormones including jasmonic acid, IAA and other auxins, and benzoate-derived compounds [ <xref rid="B23-ijms-18-02547" ref-type="bibr"> 23 </xref> ]. Therefore, steady-state kinetic assays were performed using IAA, the ethylene precursor 1-aminocyclopropane carboxylic acid (ACC), abscisic acid (ABA), jasmonic acid (JA), and salicylic acid (SA) to further examine substrate preference ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> A). <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14,</span> and <span class="gene">GmGH3-15</span> exhibited little to no <span class="trait">activity</span> with ACC, ABA, JA, and SA ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> A).
					</p>
				
					<p>&nbsp;&nbsp;
						Although IAA is the primary auxin in many plants, several different forms of auxin are present in plant tissues and the levels of auxin analogs vary between species and between different tissues [ <xref rid="B12-ijms-18-02547" ref-type="bibr"> 12 </xref> , <xref rid="B49-ijms-18-02547" ref-type="bibr"> 49 </xref> ]. To determine the substrate preference of the three <span class="gene">GmGH3</span> proteins with different auxins, kinetic assays were performed using most abundant natural forms of auxin, IAA, phenyl acetic acid (PAA), and indole butyric acid (IBA), and the synthetic auxin, naphthalene acetic acid (NAA). As mentioned above, all three <span class="gene">GmGH3s</span> showed high catalytic efficiency towards IAA ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> B; <xref ref-type="app" rid="app1-ijms-18-02547"> Table S2 </xref> ). The catalytic efficiencies (k <sub> cat </sub> /K <sub> m </sub> ) of <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14,</span> and <span class="gene">GmGH3-15</span> with IAA were 2950 M <sup> &#x02212;1 </sup> &#x000b7;s <sup> &#x02212;1 </sup> , 2640 M <sup> &#x02212;1 </sup> &#x000b7;s <sup> &#x02212;1 </sup> , and 2840 M <sup> &#x02212;1 </sup> &#x000b7;s <sup> &#x02212;1 </sup> , respectively. Each of the soybean <span class="gene">GH3</span> proteins were also capable of using PAA as substrate, although not as efficiently as IAA. Of the three proteins, <span class="gene">GmGH3-15</span> displayed a 3-fold higher k <sub> cat </sub> /K <sub> m </sub> for PAA compared to the other two enzymes ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> B; <xref ref-type="app" rid="app1-ijms-18-02547"> Table S2 </xref> ). <span class="gene">GmGH3-15</span> also used IBA (k <sub> cat </sub> /K <sub> m </sub> = 592 M <sup> &#x02212;1 </sup> &#x000b7;s <sup> &#x02212;1 </sup> ) and NAA (k <sub> cat </sub> /K <sub> m </sub> = 207 M <sup> &#x02212;1 </sup> &#x000b7;s <sup> &#x02212;1 </sup> ) as substrates, whereas the other two GH3s did not show any <span class="trait">activity</span> with these auxins ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> B; <xref ref-type="app" rid="app1-ijms-18-02547"> Table S2 </xref> ). These results suggest that all these <span class="gene">GH3</span> proteins likely conjugate IAA with aspartate to mark IAA for degradation in soybean. While <span class="gene">GmGH3-11/12</span> and <span class="gene">GmGH3-14</span> had comparable substrate preferences, <span class="gene">GmGH3-15</span> showed a broader auxin substrate preference.
					</p>
				
					<p>&nbsp;&nbsp;
						We characterized in detail the expression patterns and functional roles of <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-15</span> </italic> genes in soybean roots and nodules. Technical difficulties in cloning the promoter region precluded the characterization of <italic> <span class="gene">GmGH3-11/12</span> </italic> expression patterns. The promoter region upstream (~1900 bp) of the coding sequences of both <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-15</span> </italic> were fused to bacterial uidA gene encoding a &#x003b2;-glucuronidase (GUS) and the transcriptional fusions were expressed in soybean hairy <span class="trait">root</span> composite plants. The expression patterns of <span class="gene">GmGH3-14p:GUS</span> and <span class="gene">GmGH3-15p:GUS</span> were monitored at 0, 10, and 14 days post rhizobium inoculation (dpi) through histochemical staining for GUS <span class="trait">activity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						At 0 dpi, <span class="gene">GmGH3-14p:GUS</span> was expressed primarily in the <span class="trait">root</span> epidermis above the meristematic region ( <xref ref-type="fig" rid="ijms-18-02547-f004"> Figure 4 </xref> A). There was no detectable gene expression in the <span class="trait">root</span> tip, including the <span class="trait">root</span> cap, quiescent center, and the <span class="trait">root</span> meristem, until the differentiation zone. In mature regions of the <span class="trait">root,</span> the expression of the construct was primarily in the <span class="trait">root</span> epidermis and was more prominent in the lateral <span class="trait">root</span> primordia ( <xref ref-type="fig" rid="ijms-18-02547-f004"> Figure 4 </xref> B). As the lateral <span class="trait">root</span> emerges, the expression of <span class="gene">GmGH3-14p:GUS</span> is not detectable in the ELR at the <span class="trait">root</span> tip similar to that of the primary <span class="trait">root</span> tips ( <xref ref-type="fig" rid="ijms-18-02547-f004"> Figure 4 </xref> C). The epidermal expression of <span class="gene">GmGH3-14p:GUS</span> made it difficult to clearly image the early cortical cell division during nodule development, but in emerging nodules, GUS expression was observed in the nodule primordia ( <xref ref-type="fig" rid="ijms-18-02547-f004"> Figure 4 </xref> D). As the nodule matures the expression of <span class="gene">GmGH3-14p:GUS</span> was localized to the nodule parenchyma including the nodule vasculature ( <xref ref-type="fig" rid="ijms-18-02547-f004"> Figure 4 </xref> E).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmGH3-15</span> </italic> was expressed in the <span class="trait">root</span> meristematic region, specifically above the quiescent center cells and in the elongating cells of the <span class="trait">root</span> vasculature ( <xref ref-type="fig" rid="ijms-18-02547-f005"> Figure 5 </xref> A). GUS staining was absent in the <span class="trait">root</span> cap, as well as young epidermal and cortex cells of the <span class="trait">root</span> meristem. In the mature regions of the <span class="trait">root,</span> <span class="gene">GmGH3-15p:GUS</span> expression was detectable in the <span class="trait">root</span> epidermis and was prominent in the vasculature ( <xref ref-type="fig" rid="ijms-18-02547-f005"> Figure 5 </xref> B). Similar to <italic> <span class="gene">GmGH3-14</span> </italic> , the promoter of <italic> <span class="gene">GmGH3-15</span> </italic> was also active in the lateral <span class="trait">root</span> primordia ( <xref ref-type="fig" rid="ijms-18-02547-f005"> Figure 5 </xref> B). In emerging nodules, <span class="gene">GmGH3-15p:GUS</span> expression was observed at the junction of <span class="trait">root</span> and nodule where initiation of nodule vasculature development occurs ( <xref ref-type="fig" rid="ijms-18-02547-f005"> Figure 5 </xref> C). There was no detectable expression in the nodule primordium or other nodule tissues. As the nodule matured, the expression was primarily localized in the parenchyma region and tissues surrounding the sclerid layer ( <xref ref-type="fig" rid="ijms-18-02547-f005"> Figure 5 </xref> D). Expression was largely absent in parenchyma cells closest to the infection zone, unlike that of <span class="gene">GmGH3-14p:GUS,</span> which was expressed throughout the parenchyma. Overall, <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-15</span> </italic> have distinct spatiotemporal expression patterns in <span class="trait">root</span> tips and emerging nodules. Both genes were generally expressed in the nodule parenchyma of mature nodules with subtle differences.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate the role of <span class="gene">GmGH3</span> proteins in soybean nodule development, we sought to knock down their expression in soybean composite plants. High sequence similarity among family members precluded the use of RNAi; therefore, artificial miRNAs to independently silence <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-15</span> </italic> were designed ([ <xref rid="B50-ijms-18-02547" ref-type="bibr"> 50 </xref> ]; <xref ref-type="app" rid="app1-ijms-18-02547"> Figure S1 </xref> ). The high sequence similarity between <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-11/12</span> </italic> made it difficult to design a specific artificial miRNA construct against <italic> <span class="gene">GmGH3-14</span> </italic> . Therefore, the amiRNA against <italic> <span class="gene">GmGH3-14</span> </italic> was expected to silence both <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> and was named GH3-amiR12n14. The amiRNA targeting <italic> <span class="gene">GmGH3-15</span> </italic> was named GH3-amiR15. The amiRNA sequences were synthesized using <italic> gma-miR164 </italic> pri-miRNA as backbone ( <xref ref-type="app" rid="app1-ijms-18-02547"> Figure S1 </xref> ) and expressed using the constitutive CsVMV promoter [ <xref rid="B51-ijms-18-02547" ref-type="bibr"> 51 </xref> ] in soybean hairy <span class="trait">root</span> composite plants. The &#x0201c;empty vector&#x0201d;, pCAMGFP-CsVMV:GW was used to generate vector control hairy <span class="trait">root</span> composite plants. To evalute amiRNA-mediated gene silencing, the expression of <span class="gene">GmGH3</span> genes were quantified using RT-qPCR ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> ). The expression levels of the corresponding targets were significantly reduced in roots expressing GH3-amiR12n14 and GH3-amiR15 compared to the vector control roots ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> A,B). However, the amiRNAs also led to the reduction in expression levels of non-target <italic> <span class="gene">GH3</span> </italic> genes ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> A,B). GH3-amiR12n14 led to reduction in the levels of <italic> <span class="gene">GmGH3-15</span> </italic> ; and GH3-amiR15 led to a significant reduction in the expression levels of <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> A,B). GH3-amiR15 led to &#x0003e;95% reduction in <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> expression where as GH3-amiR12n14 led to ~60&#x02013;70% reduction of these genes. Despite the silencing of non-target <italic> <span class="gene">GH3</span> </italic> genes, we expected that suppression of <italic> <span class="gene">GH3</span> </italic> expression in these roots might lead to a reduction in IAA-Asp formation, resulting in an increased active auxin pool. As a proxy for increased active auxin levels, we measured <span class="trait">root</span> length and lateral <span class="trait">root</span> density (number of lateral roots/cm of primary <span class="trait">root)</span> in these roots. There was no significant differences in these phenotypes in either of the GH3-amiR expressing roots relative to the vector control roots ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> C,D). We also assayed the expression of auxin response marker <italic> <span class="gene">GH3</span> </italic> (not targeted by the amiRNA) and <italic> INDOLE ACETIC ACID1 </italic> ( <italic> IAA1 </italic> ) as a proxy for increased auxin levels. We observed 2.4-fold and 72-fold increases in expression of auxin-responsive <italic> <span class="gene">GH3</span> </italic> in GH3-amiR12n14 roots and GH3-amiR15 roots, respectively. However, The differences were not statistically significant due to high variation between biological replicates ( <xref ref-type="fig" rid="ijms-18-02547-f006"> Figure 6 </xref> E,F). <italic> IAA1 </italic> showed a statistically significant 2-fold higher expression in GH3-amiR12n14 roots, but no change in GH3-amiR15 roots compared to vector control roots. While physiological assays such as <span class="trait">root</span> length and lateral <span class="trait">root</span> density are likely to indicate cumulative effects of potential changes in auxin levels, gene expression markers are typically indicative of responses at the time of tissue harvest. This is likely the reason for inconsistency between markers, and large variation among replicates.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate the effect of suppressing <italic> <span class="gene">GmGH3</span> </italic> genes on nodule morphology, median cross sections of mature nodules perpendicular to the <span class="trait">root</span> were imaged, and nodule and infection zone area were measured using ImageJ ( <xref ref-type="fig" rid="ijms-18-02547-f007"> Figure 7 </xref> B&#x02013;D and <xref ref-type="app" rid="app1-ijms-18-02547"> Figure S2 </xref> ). In roots over-expressing GH3-amiR12n14, there was no significant change in either the nodule area or the infection zone area compared to the nodules from vector control roots ( <xref ref-type="fig" rid="ijms-18-02547-f007"> Figure 7 </xref> B). In roots overexpressing GH3-amiR15, there was a significant reduction in both the nodule and infection zone area ( <xref ref-type="fig" rid="ijms-18-02547-f007"> Figure 7 </xref> C). The nodule sections were also stained with phloroglucinol and evaluated for nodule vasculature development by counting the number of visible vasculature branches at the nodule-root junction and in the nodule parenchyma ( <xref ref-type="fig" rid="ijms-18-02547-f007"> Figure 7 </xref> D and <xref ref-type="app" rid="app1-ijms-18-02547"> Figure S2 </xref> ). Typically 1&#x02013;2 vascular strands are visible at the nodule-root junction, and 3&#x02013;5 strands are visible in the parenchyma indicating <span class="trait">branching</span> of the vasculature in nodule tissues. There was no significant difference in the number of vasculature branches at either position in GH3-amiR12n14 or GH3-amiR15 over-expressing roots. Overall, our results suggest that <span class="gene">GmGH3</span> proteins regulate nodule number, infection zone size, and nodule size.
					</p>
				
					<p>&nbsp;&nbsp;
						Two issues made it difficult for us to clearly interpret these results: non-specific silencing of GH3s by the amiRNAs, and broad-substrate specificity of <span class="gene">GmGH3-15.</span> Despite bioinformatics predictions and careful design, both amiRNAs significantly reduced the expression of all three <span class="gene">GH3</span> proteins. GH3-amiR15 plants had a &#x0003e;95% reduction in expression levels of <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> where as it was ~60&#x02013;70% in GH3-amiR12n14 plants; however, the level of suppression of <italic> <span class="gene">GmGH3-15</span> </italic> was comparable between GH3-amiR12n14 and GH3-amiR15 plants. Therefore, the phenotypic difference between GH3-amiR12n14 and GH3-amiR15 plants is likely to have resulted from difference in suppression of <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> . Promoter:GUS assays showed that <italic> <span class="gene">GmGH3-14</span> </italic> is highly expressed in the <span class="trait">root</span> epidermis, and soybean gene expression atlas showed that the expression of both <italic> <span class="gene">GmGH3-11/12</span> </italic> and <italic> <span class="gene">GmGH3-14</span> </italic> are induced in <span class="trait">root</span> hairs upon rhizobium inoculation ( <xref ref-type="app" rid="app1-ijms-18-02547"> Table S1 </xref> ). Reduced expression of these genes is likely to have resulted in an increase in free auxin levels in <span class="trait">root</span> hairs upon rhizobium inoculation. We speculate that this would have resulted in increased infection and nodule formation because increased auxin response appears to promote rhizobial infection at least in soybean [ <xref rid="B27-ijms-18-02547" ref-type="bibr"> 27 </xref> ]. <italic> <span class="gene">GmGH3-14</span> </italic> is also expressed in the nodule primordium, and its suppression in these cells should have led to more free auxin and suppression of nodule development. The apparent contradiction might have resulted from suppression of more than one <span class="gene">GH3</span> with distinct expression patterns by the amiRNA constructs. For example, the construct also silenced <italic> <span class="gene">GmGH3-12</span> </italic> which is highly expressed and enriched in mature nodules. Nevertheless, we observed a reduction in number of mature nodules in both GH3-amiR expressing roots. In GH3-amiR15 roots where the expression levels of all three <italic> <span class="gene">GH3</span> </italic> genes were strongly reduced, we also observed reduction in nodule size. Together these data indicate that the <span class="gene">GmGH3s</span> evaluated in this study play a key role in nodule maturation and contribute to nodule size. It was also interesting to note that <italic> <span class="gene">GmGH3-14</span> </italic> and <italic> <span class="gene">GmGH3-15</span> </italic> were expressed in vascular tissues where typically high auxin <span class="trait">activity</span> is observed. It is possible that these genes act to establish threshold auxin levels for vascular differentiation. Generation of specific knock-outs in each <span class="gene">GH3</span> through CRISPR/Cas-mediated gene editing might offer a more clear answer to the role of each of these <italic> <span class="gene">GH3</span> </italic> genes in nodule development.
					</p>
				
					<p>&nbsp;&nbsp;
						Secondly, <span class="gene">GmGH3-15</span> displayed a broad substrate specificity and much higher catalytic efficiency than other characterized GH3s. Since <span class="gene">GmGH3-15</span> showed substantial <span class="trait">activity</span> towards PAA, and IBA, it is possible that the <span class="trait">activity</span> of more than one auxin and even other hormones might have been affected in the GH3-amiR roots (see below). The ability of <span class="gene">GmGH3-15</span> to utilize different forms of auxin such as IAA, PAA, IBA, and NAA was reminiscent of the broad substrate specificity of the Arabidopsis GH3.5 (AtGH3.5) protein [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> ]. Indeed, phylogenetic analyses indicate that both <italic> <span class="gene">GmGH3-15</span> </italic> and <italic> AtGH3.5 </italic> belong to the same orthoclade [ <xref rid="B53-ijms-18-02547" ref-type="bibr"> 53 </xref> ]. <span class="gene">GmGH3-15</span> had a much higher catalytic efficiency on IAA ( <xref ref-type="fig" rid="ijms-18-02547-f003"> Figure 3 </xref> B) compared to AtGH3.5 [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> ]. Similarly, while AtGH3.5 had near equal catalytic efficiencies between IAA and PAA, <span class="gene">GmGH3-15</span> was about 3-fold more efficient with IAA over PAA. The abundance of PAA in plants is near equal or even higher than that of IAA, although the former is relatively less active than IAA [ <xref rid="B54-ijms-18-02547" ref-type="bibr"> 54 </xref> ]. In Arabidopsis, over-expression of <italic> AtGH3.5 </italic> or gain of function mutations resulted in reduced free IAA and PAA levels and increased IAA-Asp and PAA-Asp levels [ <xref rid="B19-ijms-18-02547" ref-type="bibr"> 19 </xref> , <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> , <xref rid="B55-ijms-18-02547" ref-type="bibr"> 55 </xref> ], but the relative ratio of PAA-Asp vs. PAA was much higher than that of IAA-Asp vs. IAA. It was suggested that PAA-Asp might be more stable or a storage form [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> ]. Therefore, we speculate that silencing of <italic> <span class="gene">GmGH3-15</span> </italic> might have resulted in altered PAA accumulation as well in GH3-amiR roots. It is possible that PAA in addition to IAA might play a role in soybean nodule development.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmGH3-15</span> also displayed high catalytic efficiency towards benzoic acid (BA), and 4-hydroxy benzoic acid (4-HBA), and low, but detectable <span class="trait">activity</span> towards SA ( <xref ref-type="app" rid="app1-ijms-18-02547"> Table S3 </xref> ). Arabidopsis <italic> GH3.5 </italic> gain of functions mutants ( <italic> wes1-D </italic> and <italic> gh3.5-1D </italic> ) accumulate higher levels of SA during pathogen challenge, and over-expression of <italic> AtGH3.5 </italic> also led to increase in SA and SA-Asp [ <xref rid="B19-ijms-18-02547" ref-type="bibr"> 19 </xref> , <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> , <xref rid="B55-ijms-18-02547" ref-type="bibr"> 55 </xref> ]. It has been suggested that at least part of this SA might have been derived through conversion of BA or BA-Asp to SA [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> ]. Therefore, we speculate that <span class="gene">GmGH3-15</span> might regulate SA levels in soybean. SA inhibits nodule development, but its site of action is unclear. Exogenous SA clearly inhibited both rhizobial association with <span class="trait">root</span> hairs and nodule primordium formation in indeterminate nodule forming legumes, but not in determinate nodule forming legumes [ <xref rid="B56-ijms-18-02547" ref-type="bibr"> 56 </xref> ]. However, reduction in endogenous SA levels by expressing <italic> <span class="gene">nahG</span> </italic> (a bacterial SA hydroxylase gene) increased rhizobial infection as well as nodule formation in both determinate and indeterminate nodule forming legumes. When plants were co-treated with nod factors and SA, <span class="trait">root</span> hair deformation responses were unaffected, but primordium initiation was significantly reduced [ <xref rid="B56-ijms-18-02547" ref-type="bibr"> 56 </xref> , <xref rid="B57-ijms-18-02547" ref-type="bibr"> 57 </xref> ] suggesting that SA might primarily inhibit cortical cell responses during nodule development. Given that gain of GH3.5 function in Arabidopsis led to increased SA accumulation in Arabidopsis, one might expect reduced SA and BA accumulation in <italic> <span class="gene">GmGH3-15</span> </italic> -silenced soybean roots. This is also plausible explanation for increased emerging nodule formation in these roots. While PAA has not been directly implicated in legume nodule development, a balance between positive effect of PAA and negative effect of SA has been suggested during actinorhizal nodule development [ <xref rid="B58-ijms-18-02547" ref-type="bibr"> 58 </xref> ]. It is possible that <span class="gene">GmGH3-15</span> influences nodule development through its action on more than one plant hormone. Precise tissue-specific assays of the target hormones and conjugates are expected to clarify the specific role of <span class="gene">GmGH3-15</span> in soybean nodule development. In conclusion, our results clearly show that these <span class="gene">GH3</span> proteins are important for proper nodulation in soybean while the precise mechanism by which they regulate nodule development remains to be explained.
					</p>
				
					<p>&nbsp;&nbsp;
						The coding sequences of <italic> <span class="gene">GmGH3-12</span> </italic> , <italic> <span class="gene">GmGH3-14</span> </italic> , and <italic> <span class="gene">GmGH3-15</span> </italic> were amplified by PCR using high fidelity polymerase enzymes from soybean ( <italic> Glycine max </italic> cv. Williams82) <span class="trait">root</span> cDNA as template. Amplicons were cloned into a pET-28a bacterial expression vector and verified by sequencing. The coding sequence of <italic> <span class="gene">GmGH3-15</span> </italic> had a silent mutation (T101T caused by <span class="gene">ACT</span> &#x0003e; ACC) and <italic> <span class="gene">GmGH3-11</span> </italic> had a S492P mutation (TCT &#x0003e; CCT) compared to the reference sequence in multiple independent clones suggesting that these were not PCR artifacts. The N-terminally His-tagged fusion proteins of the GH3s were expressed in <italic> Escherichia coli </italic> BL21-CodonPlus-RP cells (Stratagene/Agilent, Santa Clara, CA, USA). The fusion protein was purified following cell lysis by sonication using nickel-based affinity purification, and size-exclusion chromatography, as described for other <span class="gene">GH3</span> proteins [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> , <xref rid="B59-ijms-18-02547" ref-type="bibr"> 59 </xref> ]. The enzymatic <span class="trait">activity</span> of the three purified <span class="gene">GH3</span> enzymes were assayed spectrophotometrically as previously described [ <xref rid="B52-ijms-18-02547" ref-type="bibr"> 52 </xref> , <xref rid="B59-ijms-18-02547" ref-type="bibr"> 59 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the silencing of target <italic> <span class="gene">GH3</span> </italic> gene expression by artificial miRNAs and to measure the expression of auxin response genes, <italic> <span class="gene">GH3</span> </italic> and <italic> IAA1 </italic> , <span class="trait">root</span> tips were collected from un-inoculated roots of vector control and artificial miRNA-expressing roots in triplicate, and frozen in liquid nitrogen. Total RNA was isolated from these tissues, and gene expression was assayed using RT-qPCR as previously described [ <xref rid="B28-ijms-18-02547" ref-type="bibr"> 28 </xref> , <xref rid="B29-ijms-18-02547" ref-type="bibr"> 29 </xref> ]. Gene expression levels were normalized to that of house-keeping genes <italic> CONS7 </italic> , <italic> CONS15 </italic> , <italic> ACTIN </italic> , or <italic> CONS6 </italic> independently [ <xref rid="B64-ijms-18-02547" ref-type="bibr"> 64 </xref> ]. Data shown are relative to that of <italic> ACTIN </italic> . Results obtained using other house-keeping genes yielded similar conclusions. The statistical significance of any difference in gene expression was determined using Mann-Whitney-Wilcox test. The sequences of primers used in this study are presented in <xref ref-type="app" rid="app1-ijms-18-02547"> Table S4 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of <italic> <span class="gene">GmGH3</span> </italic> genes in soybean <span class="trait">root</span> lateral organs. ( <bold> A </bold> ) Expression levels of <span class="gene">GmGH3s</span> in emerging nodule (EN), mature nodule (MN), emerging lateral <span class="trait">root</span> (ELR), and young lateral <span class="trait">root</span> (YLR) tissues. Data shown are average expression values from three biological replicates obtained using RNA-seq. Normalized gene expression levels based on RNA-seq read counts are shown in FPKM (fragments per kilobase of transcript per million mapped reads). Error bars indicate SD; ( <bold> B </bold> ) Enrichment of <italic> <span class="gene">GmGH3</span> </italic> gene expression in the same four tissue types relative to adjacent <span class="trait">root</span> tissues. Data shown are statistically significant log <sub> 2 </sub> fold change values vs. the respective control <span class="trait">root</span> segments from three biological replicates. See <xref ref-type="app" rid="app1-ijms-18-02547"> Table S1 </xref> for additional details.
					</p>
				
					<p>&nbsp;&nbsp;
						Acyl substrate preference of nodule-enriched <span class="gene">GmGH3s.</span> ( <bold> A </bold> ) Specific <span class="trait">activity</span> of <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14,</span> and <span class="gene">GmGH3-15</span> with aspartate as aminoacid substrate, and the plant hormones indole-3-acetic acid (IAA), aminocyclopropane carboxylic acid (ACC), jasmonic acid (JA), salicylic acid (SA), and abscisic acid (ABA) as acyl substrate. Data shown are averages of three replicate assays and error bars indicate SD; ( <bold> B </bold> ) Catalytic efficiency of <span class="gene">GmGH3-11/12,</span> <span class="gene">GmGH3-14,</span> and <span class="gene">GmGH3-15</span> with aspartate as amino acid substrate, and different forms of auxin: IAA, phenyl acetic acid (PAA), indole butyric acid (IBA), and naphthalene acetic acid (NAA) as acyl substrate. See <xref ref-type="app" rid="app1-ijms-18-02547"> Table S2 </xref> for additional details.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression patterns of <span class="gene">GmGH3-14p:GUS</span> in soybean roots and nodules. ( <bold> A </bold> &#x02013; <bold> C </bold> ) Expression of <span class="gene">GmGH3-14p:GUS</span> in ( <bold> A </bold> ) <span class="trait">root</span> tips; ( <bold> B </bold> ) mature <span class="trait">root</span> region with a lateral <span class="trait">root</span> primordium; and ( <bold> C </bold> ) mature <span class="trait">root</span> region with a young lateral <span class="trait">root;</span> ( <bold> D </bold> , <bold> E </bold> ) Expression of <span class="gene">GmGH3-14p:GUS</span> in ( <bold> D </bold> ) emerging nodule (transverse section at 10 dpi); and ( <bold> E </bold> ) mature nodule (transverse section at 17 dpi). NPR-nodule primordium; IZ&#x02013;infection zone; Arrowheads indicate nodule vascular bundles. The number of independent transgenic roots/nodules showing the representative staining pattern out of the number of roots/nodules examined is indicated in each panel. Scale bars: ( <bold> A </bold> , <bold> B </bold> , <bold> E </bold> ) 100 &#x003bc;m; ( <bold> C </bold> ) 200 &#x003bc;m; ( <bold> D </bold> ) 50 &#x003bc;m.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression patterns of <span class="gene">GmGH3-15p:GUS</span> in soybean roots and nodules. ( <bold> A </bold> , <bold> B </bold> ) Expression of <span class="gene">GmGH3-14p:GUS</span> in ( <bold> A </bold> ) <span class="trait">root</span> tips; and ( <bold> B </bold> ) mature <span class="trait">root</span> region with an emerging lateral <span class="trait">root;</span> ( <bold> C </bold> , <bold> D </bold> ) Expression of <span class="gene">GmGH3-15p:GUS</span> in ( <bold> C </bold> ) emerging nodule at 10 dpi and ( <bold> E </bold> ) mature nodule (transverse section at 17 dpi). IZ&#x02013;infection zone; Arrowheads indicate nodule vascular bundles. The number of independent transgenic roots/nodules showing the representative staining pattern out of the number of roots/nodules examined is indicated in each panel. Scale bars: 100 &#x003bc;m.
					</p>
				
					<p>&nbsp;&nbsp;
						Suppression of <italic> <span class="gene">GmGH3</span> </italic> expression by artificial microRNAs. ( <bold> A </bold> , <bold> B </bold> ) Expression of target <span class="gene">GmGH3</span> genes in roots expressing ( <bold> A </bold> ) GH3-amiR12n14 and ( <bold> B </bold> ) GH3-amiR15 relative to vector control roots; ( <bold> C </bold> ) <span class="trait">Root</span> length and ( <bold> D </bold> ) lateral <span class="trait">root</span> density of vector control roots and roots expressing GH3-amiRs. Data shown are averages ( <italic> n </italic> = 21) and error bars indicate SEM. No significant difference observed using Student&#x02019;s <italic> t </italic> -test; ( <bold> E </bold> , <bold> F </bold> ) Expression of auxin response marker genes <span class="gene">GH3</span> and IAA1 in roots expressing ( <bold> E </bold> ) GH3-amiR12n14; and ( <bold> F </bold> ) GH3-amiR15, relative to vector control roots. Expression levels shown in ( <bold> A </bold> , <bold> B </bold> , <bold> E </bold> , <bold> F </bold> ) were assayed by RT-qPCR and normalized to that of Actin in each sample. Data shown are average relative expression values (fold change vs. vector control) from three biological replicates and error bars indicate the range of possible value based on SD between replicates. ** <italic> p </italic> &#x0003c; 0.01, *** <italic> p </italic> &#x0003c; 0.001, Wilcoxon-Mann-Whitney test.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Three cis-Regulatory Motifs, AuxRE, MYCRS1 and MYCRS2, are Required for Modulating the Auxin- and Mycorrhiza-Responsive Expression of a Tomato <span class="gene">GH3</span> Gene.</h2>
			<h3>PMID:28339724  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen X(1), Liao D(1), Yang X(1), Ji M(1), Wang S(1), Gu M(1), Chen A(1), Xu G(1)(2).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Auxin is well known to be a key regulator that acts in almost all physiological processes during plant growth, and in interactions between plants and microbes. However, to date, the regulatory mechanisms underlying auxin-mediated plant-arbuscular mycorrhizal (AM) fungi symbiosis have not been well deciphered. Previously we identified a <span class="gene">GH3</span> gene, SlGH3.4, strongly responsive to both auxin induction and mycorrhizal symbiosis. Here, we reported a refined dissection of the SlGH3.4 promoter <span class="trait">activity</span> using the β-glucuronidase (GUS) reporter. The SlGH3.4 promoter could drive GUS expression strongly in mycorrhizal roots of soybean and rice plants, and in IAA-treated soybean roots, but not in IAA-treated rice roots. A promoter deletion assay revealed three cis-acting motifs, i.e. the auxin-responsive element, AuxRE, and two newly identified motifs named MYCRS1 and MYCRS2, involved in the activation of auxin- and AM-mediated expression of SlGH3.4. Deletion of the AuxRE from the SlGH3.4 promoter caused almost complete abolition of GUS staining in response to external IAA induction. Seven repeats of AuxRE fused to the Cauliflower mosaic virus (CaMV) 35S minimal promoter could direct GUS expression in both IAA-treated and AM fungal-colonized roots of tobacco plants. Four repeats of MYCRS1 or MYCRS2 fused to the CaMV35S minimal promoter was sufficient to drive GUS expression in arbuscule-containing cells, but not in IAA-treated tobacco roots. In summary, our results offer new insights into the molecular mechanisms underlying the potential cross-talk between the auxin and the AM regulatory pathways in modulating the expression of AM-responsive <span class="gene">GH3</span> genes in diverse mycorrhizal plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Leader Intron of a Soybean <italic> Elongation Factor 1A </italic> ( <italic> eEF1A </italic> ) Gene Interacts with Proximal Promoter Elements to Regulate Gene Expression in Synthetic Promoters</h2>
			<h3>PMID:27806110</h3>
				

				
					<h3>PMCID:5091777</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Ning </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McHale </surname>  <given-names> Leah K. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Finer </surname>  <given-names> John J. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 10 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Introns, especially the first intron in the 5' untranslated region (5'UTR), can significantly impact gene expression via intron-mediated enhancement (IME). In this study, we demonstrate the leader intron of a soybean elongation factor 1A (eEF1A) gene <span class="gene">(GmScreamM8)</span> was essential for the high <span class="trait">activity</span> of the native promoter. Furthermore, the interaction of the <span class="gene">GmScreamM8</span> leader intron with regulatory element sequences from several soybean eEF1A promoters was studied using synthetic promoters, which consisted of element tetramers upstream of a core promoter used to regulate a green fluorescent protein (gfp) reporter gene. Element tetramers, placed upstream of a <span class="gene">GmScreamM8</span> core promoter, showed very high <span class="trait">activity</span> using both transient expression in lima bean cotyledons and stable expression in soybean hairy roots, only if the native leader intron was included, suggesting an interaction between intronic sequences and promoter elements. Partial deletions of the leader intron showed that a 222 bp intronic sequence significantly contributed to very high levels of GFP expression. Generation of synthetic intron variants with a monomeric or trimeric repeat of the 222 bp intronic sequence, yielded almost two-fold higher expression compared to the original intron, while partial deletion of the 222 bp intronic repeated sequence significantly decreased gene expression, indicating that this intronic sequence was essential for the intron-element interaction enhancement.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Introns, especially the first intron in the 5&#x02019; untranslated region (5&#x02019;UTR), can significantly impact gene expression via intron-mediated enhancement (IME). In this study, we demonstrate the leader intron of a soybean <italic> elongation factor 1A </italic> ( <italic> eEF1A </italic> ) gene <span class="gene">(GmScreamM8)</span> was essential for the high <span class="trait">activity</span> of the native promoter. Furthermore, the interaction of the <span class="gene">GmScreamM8</span> leader intron with regulatory element sequences from several soybean <italic> eEF1A </italic> promoters was studied using synthetic promoters, which consisted of element tetramers upstream of a core promoter used to regulate a <italic> green fluorescent protein </italic> ( <italic> gfp </italic> ) reporter gene. Element tetramers, placed upstream of a <span class="gene">GmScreamM8</span> core promoter, showed very high <span class="trait">activity</span> using both transient expression in lima bean cotyledons and stable expression in soybean hairy roots, only if the native leader intron was included, suggesting an interaction between intronic sequences and promoter elements. Partial deletions of the leader intron showed that a 222 bp intronic sequence significantly contributed to very high levels of GFP expression. Generation of synthetic intron variants with a monomeric or trimeric repeat of the 222 bp intronic sequence, yielded almost two-fold higher expression compared to the original intron, while partial deletion of the 222 bp intronic repeated sequence significantly decreased gene expression, indicating that this intronic sequence was essential for the intron-element interaction enhancement.
					</p>
				
					<p>&nbsp;&nbsp;
						(a) Schematic of intron partial deletion constructs in the <span class="gene">GmScreamM8</span> promoter. (b) Transient expression profiles for intron partial deletion constructs in lima bean cotyledons. The order of constructs in the figure caption key reflects the highest to lowest <span class="trait">activity</span> in the curves. (c) GFP intensity value (&#x000b1;SEM) in soybean hairy roots regulated by different intron partial deletion constructs. Images of <span class="trait">root</span> tips on top of columns represent the average intensity of GFP expression driven by the corresponding promoter construct. Overlapping columns show no significant differences between constructs by the t test (LSD) at p&#x0003c;0.05. (d) Left: RT-PCR. Right: conventional PCR using plasmid DNA of each construct as template.
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate the effect of the <span class="gene">GmScreamM8</span> leader intron on the high <span class="trait">activity</span> of the <span class="gene">GmScreamM8</span> promoter, intron deletion and translocation constructs were generated and validated using both transient expression in lima bean cotyledons and stable expression in soybean hairy roots ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1 </xref> ). The intron-containing <span class="gene">GmScreamM8</span> promoter construct gave much higher transient GFP expression, compared to the intron-less <span class="gene">GmSM8ni</span> promoter construct ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1b </xref> ). Translocation of the intron sequence upstream of the pre-intronic sequence of the <span class="gene">GmScreamM8</span> promoter in both sense <span class="gene">(GmSM8InfP)</span> and antisense <span class="gene">(GmSM8InrP)</span> orientations yielded lower levels of transient expression compared with the <span class="gene">GmScreamM8</span> promoter, with somewhat higher expression from the upstream placement of the intron in the sense orientation ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1b </xref> ). Stably-transformed soybean hairy <span class="trait">root</span> events from the intron-less <span class="gene">(GmSM8ni)</span> and intron translocation constructs <span class="gene">(GmSM8InfP;</span> <span class="gene">GmSM8InrP)</span> all showed similar expression levels that were not significantly different from each other ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1c </xref> ). The <span class="gene">GmScreamM8</span> promoter, with the full intron in its native position, gave the highest expression in soybean hairy roots ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1c </xref> ). The expression patterns were the same for all hairy roots, as all roots displayed higher expression in the <span class="trait">root</span> elongation zone and <span class="trait">root</span> tips ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1c </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To further dissect the intronic sequences that enhanced gene expression, the full-length native promoter was evaluated with four intron internal deletions ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2 </xref> ). Three of the four intron internal deletion constructs ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2a </xref> ), <span class="gene">GmSM8InD2,</span> <span class="gene">GmSM8InD3,</span> and <span class="gene">GmSM8InD4,</span> gave rise to a level of transient GFP expression lower than the full intron-containing <span class="gene">GmScreamM8</span> promoter but higher than the intron-less construct <span class="gene">GmSM8ni</span> ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2b </xref> ). In stably transformed soybean hairy roots, the three deletion constructs drove levels of GFP expression significantly lower than the <span class="gene">GmScreamM8</span> promoter but higher than <span class="gene">GmSM8ni</span> ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2c </xref> ). Another large intron deletion construct, <span class="gene">GmSM8InD1,</span> in which the majority of the intron sequence (parts 2, 3, and 4) was removed, displayed very weak but detectable <span class="trait">activity</span> in both transient and stable expression analyses ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2 </xref> ). Reverse transcription PCR revealed that intron splicing took place with all the constructs except <span class="gene">GmSM8InD1,</span> where the intron was not processed, leading to a larger amplicon ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2d </xref> , left). This was further confirmed by sequencing.
					</p>
				
					<p>&nbsp;&nbsp;
						To simplify the identification of the intronic sequence that contributed to high gene expression, a synthetic promoter consisting of an element tetramer and a core promoter was developed ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3 </xref> ). The element tetramer, EF4, was selected as it showed the highest <span class="trait">activity</span> among several element-containing fragments from <span class="gene">GmScream</span> promoters ( <xref ref-type="supplementary-material" rid="pone.0166074.s001"> S1 Fig </xref> ). Different combinations of the EF4 tetramer from <span class="gene">GmScreamM1</span> with the 35S core or <span class="gene">GmScreamM8</span> core promoter, with and without the <span class="gene">GmScreamM8</span> intron, showed that the tetramer with the <span class="gene">GmScreamM8</span> core and <span class="gene">GmScreamM8</span> intron gave the highest <span class="trait">activity</span> ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3 </xref> ). This high <span class="trait">activity</span> was observed using both transient expression in lima bean cotyledons ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3b </xref> ) and stable expression in transgenic soybean hairy roots ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3c </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Using transient expression analysis, the construct containing the <span class="gene">GmScreamM8</span> core plus the native leader intron (4xEF4-M8C) showed 33-fold higher expression compared to its counterpart without the intron (4xEF4-M8Core). Another intron-containing construct (4xEF4-35Score-M8Intron) with the 35S core promoter rather than the <span class="gene">GmScreamM8</span> core promoter also displayed significantly enhanced gene expression in comparison with its intron-less counterpart (4xEF4-35Score) ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3b </xref> ). In addition, the <span class="gene">GmScreamM8</span> core promoter showed seven-fold higher <span class="trait">activity</span> when the intron was present <span class="gene">(GmSM8C),</span> even if no additional tetrameric promoter sequences were upstream. However, use of the leader intron did not show any enhancement of gene expression when it was cloned downstream of the 35S core promoter without the EF4 tetramer ( <xref ref-type="fig" rid="pone.0166074.g003"> Fig 3b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Using transient expression analysis of constructs containing the EF4 tetramer, the <span class="gene">GmScreamM8</span> core, and three different intron deletions, reduced expression was obtained only when part 3 of the intron (4xEF4-M8CInD3; <xref ref-type="fig" rid="pone.0166074.g004"> Fig 4a </xref> ) was deleted, while deletion of part 2 (4xEF4-M8CInD2) or part 4 (4xEF4-M8CInD4) did not significantly influence the intensity of gene expression compared to the control 4xEF4-M8C, which possessed a full intron ( <xref ref-type="fig" rid="pone.0166074.g004"> Fig 4b </xref> ). In stably transformed hairy roots, GFP expression driven by 4xEF4-M8CInD3 was significantly lower than all other constructs. For the other deletion constructs, 4xEF4-M8CInD2 displayed a significant lower <span class="trait">activity</span> than the control 4xEF4-M8C, while 4xEF4-M8CInD4 did not show a significant difference ( <xref ref-type="fig" rid="pone.0166074.g004"> Fig 4c </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Use of the automated image collection and analysis system with GFP allowed for continual monitoring of promoter <span class="trait">activity</span> over 100 h (Figs <xref ref-type="fig" rid="pone.0166074.g001"> 1 </xref> &#x02013; <xref ref-type="fig" rid="pone.0166074.g007"> 7 </xref> ), generating consistent results for measurement of promoter strength. This approach has been used previously with constitutively expressed promoters [ <xref rid="pone.0166074.ref026" ref-type="bibr"> 26 </xref> , <xref rid="pone.0166074.ref039" ref-type="bibr"> 39 </xref> ] as well as wound-inducible promoters [ <xref rid="pone.0166074.ref039" ref-type="bibr"> 39 </xref> ] and promoter elements [ <xref rid="pone.0166074.ref038" ref-type="bibr"> 38 </xref> ]. This same transient expression approach was also used for rapid validation of the <span class="gene">GmScream</span> promoters [ <xref rid="pone.0166074.ref034" ref-type="bibr"> 34 </xref> ], and showed that the <span class="gene">GmScreamM1,</span> <span class="gene">GmScreamM4</span> and <span class="gene">GmScreamM8</span> promoters were the most highly active promoters in this group. The specific <italic> gfp </italic> gene that was used in this research is an earlier version of the gene and the protein displays moderate fluorescence and stability [ <xref rid="pone.0166074.ref045" ref-type="bibr"> 45 </xref> ], compared with other green fluorescent proteins, which have been modified for increased fluorescence and higher stability [ <xref rid="pone.0166074.ref046" ref-type="bibr"> 46 </xref> &#x02013; <xref rid="pone.0166074.ref047" ref-type="bibr"> 47 </xref> ]. The GFP fluorescence reported here should therefore be more indicative of promoter <span class="trait">activity</span> than other reporter genes, which give rise to more stable products and may still be measured, long after the promoter becomes inactive. Although it may have been useful to evaluate either GUS or luciferase <span class="trait">activity,</span> regulated by a constitutive promoter, as an internal standard [ <xref rid="pone.0166074.ref048" ref-type="bibr"> 48 </xref> ], this approach was not feasible using a continual monitoring system as GUS and luciferase assays are destructive and can only be used for single time point assays. In this research, GFP expression was evaluated and quantified over time based on image analysis of a defined area of GFP-expressing cells in bombarded tissue. GFP levels were calculated based on background-corrected average gray value in the green and red channels across thousands of transiently expressing cells [ <xref rid="pone.0166074.ref042" ref-type="bibr"> 42 </xref> ], in 6&#x02013;9 cotyledons, from 2&#x02013;3 separate experiments. This approach has been useful for quantification of the effects of promoters [ <xref rid="pone.0166074.ref039" ref-type="bibr"> 39 </xref> ], intron-variants [ <xref rid="pone.0166074.ref026" ref-type="bibr"> 26 </xref> ] and <italic> cis </italic> -regulatory elements [ <xref rid="pone.0166074.ref038" ref-type="bibr"> 38 </xref> ], and seems to be useful for evaluation of the promoter and intron variants described in this research.
					</p>
				
					<p>&nbsp;&nbsp;
						Removal of the leader intron from the original <span class="gene">GmScreamM8</span> promoter <span class="gene">(GmSM8ni,</span> <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1 </xref> ) resulted in 60% decrease in gene expression, suggesting that the leader intron significantly contributed to the high <span class="trait">activity</span> of the ~1.5 kb promoter ( <xref ref-type="fig" rid="pone.0166074.g001"> Fig 1 </xref> ). The <span class="gene">GmScreamM8</span> leader intron is located only 77 bp downstream of the TSS of the native soybean <italic> eEF1A </italic> gene. As with other stimulating introns, the close proximity of the 5&#x02019;UTR intron to the TSS may allow better access of the transcriptional initiation machinery to the intron [ <xref rid="pone.0166074.ref051" ref-type="bibr"> 51 </xref> &#x02013; <xref rid="pone.0166074.ref052" ref-type="bibr"> 52 </xref> ], enhancing transcription initiation. In addition, the first introns relative to other introns of the same gene usually display significant enrichment of active chromatin marks, further indicating certain regions in first introns have potential regulatory roles [ <xref rid="pone.0166074.ref053" ref-type="bibr"> 53 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of intron partial deletion constructs with the full-length promoter showed that the partial deletions reduced promoter <span class="trait">activity</span> ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2 </xref> ). The largest intron deletion construct <span class="gene">GmSM8InD1</span> ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2 </xref> ), with only a 68 bp intron sequence remaining, showed very low but detectable GFP expression in both transient and stable expression assays. This was the only intron-containing construct, where the intron was not processed ( <xref ref-type="fig" rid="pone.0166074.g002"> Fig 2d </xref> ). The failure in intron processing with the <span class="gene">GmSM8InD1</span> construct likely affected mRNA stability and translation efficiency by interruption of the reading frame [ <xref rid="pone.0166074.ref054" ref-type="bibr"> 54 </xref> ], or by triggering nonsense-mediated mRNA decay (NMD) [ <xref rid="pone.0166074.ref056" ref-type="bibr"> 56 </xref> ]. Splicing failure of <span class="gene">GmSM8InD1</span> could have resulted from a short intron size [ <xref rid="pone.0166074.ref057" ref-type="bibr"> 57 </xref> ], where removal of the majority of the intronic regulatory elements/enhancers resulted in very inefficient splicing. However, the role of splicing on IME is still not clear. Splicing alone is not sufficient to induce IME [ <xref rid="pone.0166074.ref058" ref-type="bibr"> 58 </xref> ] since not all spliceosomal introns can enhance gene expression [ <xref rid="pone.0166074.ref025" ref-type="bibr"> 25 </xref> ]. But, splicing is often necessary for IME [ <xref rid="pone.0166074.ref032" ref-type="bibr"> 32 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Antioxidant responses in soybean and alfalfa plants grown in DDTs contaminated soils: Useful variables for selecting plants for soil phytoremediation?</h2>
			<h3>PMID:27155479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mitton FM(1), Ribas Ferreira JL(2), Gonzalez M(3), Miglioranza KSB(4), Monserrat JM(2).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytoremediation is a low-cost alternative technology based on the use of plants to remove pollutants from the environment. Persistent organic pollutants such as DDTs with a long half-life in soils are attractive candidates for remediation. This study aimed to determine the potential of antioxidant response use in the evaluation of plants' tolerance for selecting species in phytoremediation purposes. Alfalfa and soybean plants were grown in DDT contaminated soils. After 60days, growth, protein content, antioxidant capacity, <span class="gene">GST</span> <span class="trait">activity,</span> concentration of proteic and non-proteic thiol groups, chlorophyll content and carotenoid content were measured in plant tissues. Results showed no effect on alfalfa or soybean photosynthetic pigments but different responses in the protein content, antioxidant capacity, <span class="gene">GST</span> <span class="trait">activity</span> and thiol groups on roots, stems and leaves, indicating that DDTs affected both species. Soybean showed higher susceptibility than alfalfa plants due to the lower antioxidant capacity and <span class="gene">GST</span> <span class="trait">activity</span> in leaves, in spite of having the lowest DDT accumulation. This study provides new insights into the role of oxidative stress as an important component of the plant's response to DDT exposure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A bacterial gene <span class="gene">codA</span> encoding cytosine deaminase is an effective conditional negative selectable marker in Glycine max.</h2>
			<h3>PMID:26082433  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shao M(1), Michno JM(2), Hotton SK(1), Blechl A(3), Thomson J(4).
				<br/> 
				年份：2015 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Research describes the practical application of the <span class="gene">codA</span> negative selection marker in Soybean. Conditions are given for <span class="gene">codA</span> selection at both the shooting and rooting stages of regeneration. Conditional negative selection is a powerful technique whereby the absence of a gene product allows survival in otherwise lethal conditions. In plants, the Escherichia coli gene <span class="gene">codA</span> has been employed as a negative selection marker. Our research demonstrates that <span class="gene">codA</span> can be used as a negative selection marker in soybean, Glycine max. Like most plants, soybean does not contain cytosine deaminase <span class="trait">activity</span> and we show here that wild-type seedlings are not affected by inclusion of 5-FC in growth media. In contrast, transgenic G. max plants expressing <span class="gene">codA</span> and grown in the presence of more than 200 μg/mL 5-FC exhibit reductions in hypocotyl and taproot lengths, and severe suppression of lateral <span class="trait">root</span> development. We also demonstrate a novel negative selection-rooting assay in which codA-expressing aerial tissues or shoot cuttings are inhibited for <span class="trait">root</span> formation in media containing 5-FC. Taken together these techniques allow screening during either the regeneration or rooting phase of tissue culture.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of nitrous oxide reductase from <italic> Pseudomonas stutzeri </italic> in transgenic tobacco roots using the root-specific <italic> rolD </italic> promoter from <italic> Agrobacterium rhizogenes </italic></h2>
			<h3>PMID:22423324</h3>
				

				
					<h3>PMCID:3298943</h3>
					<h4>期刊:Ecology and Evolution</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wan </surname>  <given-names> Shen </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Johnson </surname>  <given-names> Amanda M </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Altosaar </surname>  <given-names> Illimar </given-names>  </name>  </contrib>  <aff>  <institution> Department of Biochemistry, Microbiology and Immunology, Center for Research on Environmental Microbiology &#x02013; CREM, Faculty of Medicine, University of Ottawa </institution>  <addr-line> 451 Smyth Road, Ottawa, Ontario, K1H 8M5, Canada </addr-line>  </aff>
				<br/> 
				年份：<day> 07 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nitrous oxide (N(2)O) reduction pathway from a soil bacterium, Pseudomonas stutzeri, was engineered in plants to reduce N(2)O emissions. As a proof of principle, transgenic plants expressing nitrous oxide reductase (N(2)OR) from P. stutzeri, encoded by the <span class="gene">nosZ</span> gene, and other transgenic plants expressing N(2)OR along with the more complete operon from P. stutzeri, encoded by nosFLZDY, were generated. Gene constructs were engineered under the control of a root-specific promoter and with a secretion signal peptide. Expression and rhizosecretion of the transgene protein were achieved, and N(2)OR from transgenic Nicotiana tabacum proved functional using the methyl viologen assay. Transgenic plant line 1.10 showed the highest specific <span class="trait">activity</span> of 16.7 µmol N(2)O reduced min(-1) g(-1) <span class="trait">root</span> protein. Another event, plant line 1.9, also demonstrated high specific <span class="trait">activity</span> of N(2)OR, 13.2 µmol N(2)O reduced min(-1) g(-1) <span class="trait">root</span> protein. The availability now of these transgenic seed stocks may enable canopy studies in field test plots to monitor whole rhizosphere N flux. By incorporating one bacterial gene into genetically modified organism <span class="gene">(GMO)</span> crops (e.g., cotton, corn, and soybean) in this way, it may be possible to reduce the atmospheric concentration of N(2)O that has continued to increase linearly (about 0.26% year(-1)) over the past half-century.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The nitrous oxide (N <sub> 2 </sub> O) reduction pathway from a soil bacterium, <italic> Pseudomonas stutzeri </italic> , was engineered in plants to reduce N <sub> 2 </sub> O emissions. As a proof of principle, transgenic plants expressing nitrous oxide reductase (N <sub> 2 </sub> OR) from <italic> P. stutzeri </italic> , encoded by the <italic> <span class="gene">nosZ</span> </italic> gene, and other transgenic plants expressing N <sub> 2 </sub> OR along with the more complete operon from <italic> P. stutzeri </italic> , encoded by <italic> nosFLZDY </italic> , were generated. Gene constructs were engineered under the control of a root-specific promoter and with a secretion signal peptide. Expression and rhizosecretion of the transgene protein were achieved, and N <sub> 2 </sub> OR from transgenic <italic> Nicotiana tabacum </italic> proved functional using the methyl viologen assay. Transgenic plant line 1.10 showed the highest specific <span class="trait">activity</span> of 16.7 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> protein. Another event, plant line 1.9, also demonstrated high specific <span class="trait">activity</span> of N <sub> 2 </sub> OR, 13.2 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> protein. The availability now of these transgenic seed stocks may enable canopy studies in field test plots to monitor whole rhizosphere N flux. By incorporating one bacterial gene into genetically modified organism <span class="gene">(GMO)</span> crops (e.g., cotton, corn, and soybean) in this way, it may be possible to reduce the atmospheric concentration of N <sub> 2 </sub> O that has continued to increase linearly (about 0.26% year <sup> &#x02212;1 </sup> ) over the past half-century.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant roots are in direct contact with the microbial community in the rhizosphere. They secrete a number of chemicals into the rhizosphere, having a large impact on soil chemistry ( <xref ref-type="bibr" rid="b31"> Philippot et al. 2009 </xref> ). Tobacco plant roots have been used as a recombinant protein production system using root-specific promoters for the gene of interest ( <xref ref-type="bibr" rid="b13"> Drake et al. 2003 </xref> ). Promoting complete denitrification in the rhizosphere in this way may eliminate N <sub> 2 </sub> O emissions at the source. The substrate, N <sub> 2 </sub> O, produced by denitrifiers in the rhizosphere could potentially bind to the catalytic enzyme, N <sub> 2 </sub> OR, secreted by transgenic tobacco plant roots. N <sub> 2 </sub> O reduction would occur, resulting in the release of N <sub> 2 </sub> gas into the soil air pockets and ultimately into the atmosphere. To test this hypothesis, in an attempt to achieve N <sub> 2 </sub> OR expression <italic> in planta </italic> , <italic> Nicotiana tabacum </italic> cv. Xanthi nc. plants were transformed with the single <italic> <span class="gene">nosZ</span> </italic> gene. A second set of transgenic plants were also transformed with the more complete operon, <italic> nosFLZDY </italic> . Transgenic plants were analyzed to confirm transgene incorporation, transgene expression, protein expression, and protein <span class="trait">activity.</span> Surprisingly, extracts from these tobacco plants, isolated from their <span class="trait">root</span> tissue and from the medium surrounding their roots, when analyzed by the methyl viologen assay provided evidence of N <sub> 2 </sub> O reduction capacity.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Pseudomonas stutzeri </italic> genomic DNA was used as a template for polymerase chain reaction (PCR) amplification of the <italic> <span class="gene">nosZ</span> </italic> gene or <italic> nosFLZDY </italic> genes. Primers were gene-specific, and were designed to introduce restriction sites so the resulting segments could be fused (Table S1). The root-specific promoter of the <italic> rolD </italic> gene from <italic> Agrobacterium rhizogenes </italic> was chosen to direct root-specific expression of N <sub> 2 </sub> OR. The promoter sequence 426D ( <xref ref-type="bibr" rid="b15"> Elmayan and Tepfer 1995 </xref> ), containing a 463 bp segment of the upstream untranscribed region of <italic> rolD </italic> , was amplified from the plasmid pLJ1 provided by D. Tepfer. The nopaline synthase polyadenylation sequence was used as terminator (NOSter). The leader sequence of the <italic> Daucus carota </italic> (carrot) <italic> extensin </italic> (ex) gene was amplified from the pHBV-CO plasmid using forward primer, ExSigZF/ExSigDF, and reverse primer, ExSigZR/ExSigDR ( <xref ref-type="bibr" rid="b7"> Chen and Varner 1985 </xref> ; <xref ref-type="bibr" rid="b1"> Alli et al. 2002 </xref> ). The <italic> Phaseolus vulgaris </italic> ( <italic> Pv </italic> ) alpha amylase inhibitor-1 signal sequence was amplified using forward and reverse primers PvSigF and PvSigR, respectively ( <xref ref-type="bibr" rid="b33"> Prescott et al. 2005 </xref> ). Purified PCR products were cloned into the pPCR-Script Amp SK (+) cloning vector system (Stratagene, La Jolla, CA). For assembly of expression vector prolD- <italic> <span class="gene">nosZ</span> </italic> , segments of NOSter, nosZ, ex, and rolD were ligated together and cloned into the binary vector pRD400 ( <xref ref-type="bibr" rid="b11"> Datla et al. 1992 </xref> ) (Fig. S1) following digestion with <italic> Kpn </italic> I and <italic> Bam </italic> HI. Plasmid pRD400 carries the neomycin phosphotransferase II ( <italic> nptII </italic> ) gene, which confers <span class="trait">resistance</span> to the antibiotic kanamycin. The resulting prolD- <italic> <span class="gene">nosZ</span> </italic> plasmid was sequenced at StemCore Laboratories (Ottawa, Canada) to verify accuracy of the DNA sequence. The megacassette prolD- <italic> nosFLZDY </italic> was constructed in a similar fashion, ligating individual expression constructs for <italic> nosF </italic> , <italic> nosD </italic> , <italic> nosY </italic> , and <italic> nosL </italic> and cloning the resulting sequence into pRD400 (Fig. S2).
					</p>
				
					<p>&nbsp;&nbsp;
						A portion of the <span class="trait">root</span> tissue (100 mg) was collected from each 6-week-old plant and genomic DNA was extracted using the DNeasy Plant Mini Kit (Qiagen, Mississauga, Canada). PCR was performed in a 25 &#x000b5;L reaction mixture containing 50 ng of tobacco genomic DNA, 25 pmol of each primer, 2.5 &#x000b5;L of 10 &#x000d7; PCR buffer (with 1.5 mM MgCl <sub> 2 </sub> ), 250 &#x000b5;M of each dNTP, 0.5 unit Taq DNA polymerase, with the following cycling condition: initial denaturation at 94&#x000b0;C for 4 min; 30 cycles with denaturation at 94&#x000b0;C for 30 sec, annealing at 55&#x000b0;C for 1 min, and elongation at 72&#x000b0;C for 2 min; followed by a final elongation at 72&#x000b0;C for 10 min. DNA from nontransgenic (NT) plant roots was used as negative control, while either the prolD- <italic> <span class="gene">nosZ</span> </italic> or prolD- <italic> nosFLZDY </italic> plasmid was used as positive control. The following primers were used to detect the <italic> <span class="gene">nosZ</span> </italic> gene in genomic DNA from prolD- <italic> <span class="gene">nosZ</span> </italic> transgenic plants: forward primer (ExSigZF) 5&#x02032;-TAGGTACCTACTCGAGATGGGAAGAATTGCTAGAGG-3&#x02032; and reverse primer (NosZR) 5&#x02032;-TAGGATCCAACATATGTTAGGCCGGCTCGACCATC-3&#x02032;. For prolD- <italic> nosFLZDY </italic> transgenic plants, the specific primers used to detect the <italic> <span class="gene">nosZ</span> </italic> gene were forward primer (NosZF) 5&#x02032;-GACTAGTCAGGCCGTCAAGGAGTCCAAG-3&#x02032; and reverse primer (NosZR) 5&#x02032;-TAGGATCCAACATATGTTAGGCCGGCTCGACCATC-3&#x02032;.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Root</span> tissue (100 mg) was collected from each 6-week-old plant and total RNA was extracted using the RNeasy Plant Mini Kit (Qiagen). After treatment with RNase-free DNase (Promega, Madison, WI), RT-PCR was performed according to the SuperScript II Reverse Transcriptase system protocol (Invitrogen, Carlsbad, CA). In a 10 &#x000b5;L reaction mixture, 250 ng of total RNA was reverse transcribed at 70&#x000b0;C for 10 min with an oligo (dT) primer. The conditions for cDNA synthesis were 10 min at 25&#x000b0;C, 60 min at 37&#x000b0;C, and 5 min at 95&#x000b0;C. Each PCR reaction used 2 &#x000b5;L cDNA as a template and <italic> nosZ- </italic> specific primers. DNA from NT plant roots was used as negative control, while either the prolD- <italic> <span class="gene">nosZ</span> </italic> or prolD- <italic> nosFLZDY </italic> plasmid was used as positive control. For prolD- <italic> <span class="gene">nosZ</span> </italic> transgenic plants, the following primers were used to detect the <italic> <span class="gene">nosZ</span> </italic> gene: forward primer (ExSigZF) 5&#x02032;-TAGGTACCTACTCGAGATGGGAAGAATTGCTAGAGG-3&#x02032; and reverse primer (NosZR) 5&#x02032;-TAGGATCCAACATATGTTAGGCCGGCTCGACCATC-3&#x02032;. For prolD- <italic> nosFLZDY </italic> transgenic plants, the following primers were used to detect the <italic> <span class="gene">nosZ</span> </italic> gene: forward primer (NosZF) 5&#x02032;-GACTAGTCAGGCCGTCAAGGAGTCCAAG-3&#x02032; and reverse primer (NosZR) 5&#x02032;-TAGGATCCAACATATGTTAGGCCGGCTCGACCATC-3&#x02032;. A control PCR, with total RNA not treated with reverse transcriptase, was performed to ensure that no DNA was present. Expression of other genes in the <italic> nos </italic> cassette ( <italic> nosF </italic> , <italic> no </italic> s <italic> L </italic> , <italic> nosD </italic> , and <italic> nosY </italic> ) was also analyzed using RT-PCR in the same way, with gene-specific primers.
					</p>
				
					<p>&nbsp;&nbsp;
						Hydroponic cultures were established for the harvest of recombinant protein from <span class="trait">root</span> tissue and the rhizosphere according to the method of <xref ref-type="bibr" rid="b13"> Drake et al. (2003) </xref> . Transgenic tobacco seeds containing either the rolD- <italic> <span class="gene">nosZ</span> </italic> or the rolD- <italic> FLZDY </italic> expression cassette were surface-sterilized using 20% v/v bleach, washed in sterile distilled water, and sown onto a 9-cm petri dish containing 0.7% w/v agar-solidified <span class="gene">MS</span> medium ( <xref ref-type="bibr" rid="b27"> Murashige and Skoog 1962 </xref> ). When seedlings reached 1 cm in height, they were transferred to liquid <span class="gene">MS</span> medium by placing the shoot through a perforation in a plastic platform in a 50-mL container containing 25 mL liquid <span class="gene">MS</span> medium. The roots of the seedling were immersed in the liquid medium and the shoots were supported above the platform. Seedlings were maintained at 25&#x000b0;C with 16 h daylight and 8 h dark, and were grown to a height of 6 cm before further analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Rhizosecreted protein from each plant culture was collected over a period of 25 days. Aliquots of <span class="gene">MS</span> medium (500 &#x000b5;L) from both rolD-transgenic and NT plant cultures were collected at regular intervals. Samples were concentrated to a volume of 20 &#x000b5;L by means of YM50 Microcon filters (Millipore, Billerica, MA) and stored at &#x02013;20&#x000b0;C. Protein samples were pooled together prior to western blotting and N <sub> 2 </sub> OR <span class="trait">activity</span> analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						To demonstrate the expression of the N <sub> 2 </sub> OR in tobacco <span class="trait">root</span> from T <sub> 0 </sub> transgenic plants, an immunoblot assay of total soluble protein extracts was performed using a polyclonal antibody against the N <sub> 2 </sub> OR antigen. <xref ref-type="fig" rid="fig02"> Fig. 2 </xref> displays the immunoblots for rolD- <italic> <span class="gene">nosZ</span> </italic> and rolD- <italic> nosFLZDY </italic> samples, showing a single band of the expected molecular mass of 72 kDa. The positive control, N <sub> 2 </sub> OR obtained from <italic> P. stutzeri </italic> , also yielded a signal band at 72 kDa. There was no signal for the presence of N <sub> 2 </sub> OR in nontransformed plant extracts. The images were scanned using ImageJ software ( <ext-link ext-link-type="uri" xlink:href="http://rsbweb.nih.gov/ij/"> http://rsbweb.nih.gov/ij/ </ext-link> ) to semiquantify the expression of N <sub> 2 </sub> OR. The corresponding histogram shows <span class="trait">yield</span> of recombinant N <sub> 2 </sub> OR in &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> protein. Recombinant protein <span class="trait">yield</span> varied among rolD- <italic> <span class="gene">nosZ</span> </italic> samples, with the lowest <italic> <span class="gene">nosZ</span> </italic> <span class="trait">yield</span> of 0.032 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> protein and the highest 0.079 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> protein. On average, the rolD- <italic> <span class="gene">nosZ</span> </italic> plants expressed N <sub> 2 </sub> OR at levels 10-fold higher than rolD- <italic> nosFLZDY </italic> plants, which varied from 0.003 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> protein to 0.009 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> protein.
					</p>
				
					<p>&nbsp;&nbsp;
						Western immunoblot analysis detecting N <sub> 2 </sub> OR in total soluble protein extracted from transgenic plant <span class="trait">root</span> tissue. (A) Histogram showing the <span class="trait">yield</span> of recombinant N <sub> 2 </sub> OR in <italic> <span class="gene">nosZ</span> </italic> -expressing transgenic plants, with the corresponding Western immunoblot. (B) Histogram showing the <span class="trait">yield</span> of recombinant N <sub> 2 </sub> OR in <italic> nosFLZDY </italic> -expressing transgenic plants, with the corresponding Western immunoblot. + = positive control, N <sub> 2 </sub> OR protein purified from <italic> P. stutzeri </italic> ; NT = negative control, protein extracted from a nontransformed plant; transformed plants are identified by number.
					</p>
				
					<p>&nbsp;&nbsp;
						A Western immunoblot was performed to test for the expression and subsequent excretion of recombinant N <sub> 2 </sub> OR into the rhizosphere. Fractions of hydroponic medium collected over a 25-day period from individual plants were pooled, and the crude protein extracted. The resulting immunoblot is shown in <xref ref-type="fig" rid="fig04"> Fig. 4 </xref> . A 72-kDa band coinciding with the size of the protein from <italic> P. stutzeri </italic> is present in the exudates from all transgenic lines. The nontransformed control did not exhibit a signal for the presence of N <sub> 2 </sub> OR. <span class="trait">Yield</span> of recombinant N <sub> 2 </sub> OR was determined based on the chemiluminescent signal and ranged from 0.016 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> secreted protein (rolD: <italic> <span class="gene">nosZ</span> </italic> -5.8) to 0.044 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g crude <span class="trait">root</span> secreted protein (rolD: <italic> <span class="gene">nosZ</span> </italic> -1.7).
					</p>
				
					<p>&nbsp;&nbsp;
						Western immunoblot analysis detecting N <sub> 2 </sub> OR in the rhizosphere surrounding transgenic plant <span class="trait">root</span> tissue. Histogram showing the <span class="trait">yield</span> of recombinant N <sub> 2 </sub> OR in <italic> <span class="gene">nosZ</span> </italic> -expressing and <italic> nosFLZDY </italic> -expressing transgenic plants, with the corresponding Western immunoblot. + = positive control, N <sub> 2 </sub> OR protein purified from <italic> P. stutzeri </italic> ; NT = negative control, protein extracted from a nontransformed plant; transformed plants are identified by number.
					</p>
				
					<p>&nbsp;&nbsp;
						The crude protein extracted from the hydroponic culture medium was assayed for specific <span class="trait">activity</span> using the methyl viologen-linked assay. <xref ref-type="fig" rid="fig05"> Fig. 5 </xref> reports the calculated specific <span class="trait">activity</span> of the recombinant enzyme, expressed as &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> secreted protein. The nontransformed control showed a background specific <span class="trait">activity</span> (1.72 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> secreted protein). The purified enzyme itself serving as a positive control, N <sub> 2 </sub> OR from <italic> P. stutzeri </italic> , gave a specific <span class="trait">activity</span> of 454 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> mg <sup> &#x02212;1 </sup> protein. N <sub> 2 </sub> OR from the rolD: <italic> <span class="gene">nosZ</span> </italic> -1.7, rolD: <italic> nosZ- </italic> 1.9, and rolD: <italic> nosZ- </italic> 1.10 transgenic lines showed the highest specific activities of 7.03, 6.27, and 5.66 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> secreted protein, respectively. This is less than half the specific activities of N <sub> 2 </sub> OR located in the <span class="trait">root</span> tissue of the same transgenic lines (see <xref ref-type="fig" rid="fig03"> Fig. 3 </xref> ). In contrast, the specific <span class="trait">activity</span> of N <sub> 2 </sub> OR extracted from the hydroponic media of rolD: <italic> nosFLZDY </italic> -5.8 plants was slightly higher than N <sub> 2 </sub> OR compartmentalized in the roots (compare 2.19 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> secreted protein vs. 1.91 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> protein <bold> ) </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						In calculating recombinant protein <span class="trait">yield</span> from these transgenic plant roots, two patterns became evident. First, the <span class="trait">yield</span> is positively correlated to rolD: <italic> <span class="gene">nosZ</span> </italic> expression levels determined by RT-PCR. For example, plant line 1.10 has very strong transcription despite the apparent low content of the transgene ( <xref ref-type="fig" rid="fig01"> Fig. 1A </xref> ). This 1.10 mRNA is correspondingly translated into the most abundant level of protein detected by anti-N <sub> 2 </sub> OR antibodies by western hybridization ( <xref ref-type="fig" rid="fig02"> Fig. 2A </xref> ). Plant line 1.10 extracts correspondingly contained the highest amount of protein, 0.079 &#x000b5;g N <sub> 2 </sub> OR/100 &#x000b5;g <span class="trait">root</span> protein. And finally, this plant line 1.10 had the highest specific enzyme <span class="trait">activity,</span> 16.7 &#x000b5;mol N <sub> 2 </sub> O reduced min <sup> &#x02212;1 </sup> g <sup> &#x02212;1 </sup> <span class="trait">root</span> protein ( <xref ref-type="fig" rid="fig03"> Fig. 3 </xref> ). The second pattern is the higher N <sub> 2 </sub> OR expression in rolD: <italic> <span class="gene">nosZ</span> </italic> relative to rolD: <italic> nosFLZDY </italic> transgenics. Plants expressing rolD: <italic> <span class="gene">nosZ</span> </italic> yielded 10-fold more N <sub> 2 </sub> OR than those expressing rolD: <italic> nosFLZDY </italic> . A possible explanation for the lower expressing rolD: <italic> nosFLZDY </italic> plants is gene silencing. Since gene silencing increases with increasing transcript size ( <xref ref-type="bibr" rid="b22"> Johnston et al. 2011 </xref> ; <xref ref-type="bibr" rid="b26"> Melnyk et al. 2011 </xref> ), the larger rolD: <italic> nosFLZDY </italic> construct may have been subject to gene silencing in the tobacco plant, whereas the rolD: <italic> <span class="gene">nosZ</span> </italic> construct was not.
					</p>
				
					<p>&nbsp;&nbsp;
						N <sub> 2 </sub> OR from rolD: <italic> <span class="gene">nosZ</span> </italic> and rolD: <italic> nosFLZDY </italic> plant <span class="trait">root</span> tissue was shown to be correctly assembled. Western blot analysis showed, by electrophoretic mobility, that N <sub> 2 </sub> OR purified from <italic> P. stutzeri </italic> and N <sub> 2 </sub> OR in crude protein extract isolated from plant <span class="trait">root</span> tissue were the same size ( <xref ref-type="fig" rid="fig02"> Fig. 2 </xref> ). Likewise, in hydroponic culture experiments, we detected N <sub> 2 </sub> OR in transgenic <span class="trait">root</span> exudates, but in a lower quantity than in the <span class="trait">root</span> tissue ( <xref ref-type="fig" rid="fig04"> Fig. 4 </xref> ). These results contrast those from previous experiments, which revealed more recombinant protein in the hydroponic medium than in the tobacco <span class="trait">root</span> tissue ( <xref ref-type="bibr" rid="b2"> Borisjuk et al. 1999 </xref> ). This is due to inadequate transport of the 140-kDa recombinant N <sub> 2 </sub> OR dimer from <span class="trait">root</span> cells across the epidermis and into the rhizosphere. Increasing the permeability of the <span class="trait">root</span> cell wall by supplying the plant with a plant growth regulator such as auxin may facilitate better secretion of N <sub> 2 </sub> OR ( <xref ref-type="bibr" rid="b14"> Drake et al. 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The present results indicate that transgenic tobacco plants stably expressing the <italic> <span class="gene">nosZ</span> </italic> transgene are capable of producing functional recombinant N <sub> 2 </sub> OR. Further analysis will include examination of the ability of these plants to reduce N <sub> 2 </sub> O in field trials using various soil types and environments together with single plant canopy enclosures to measure nitrogen flux. Once the reduction of N <sub> 2 </sub> O by the transgenic plant lines has been proven effective in test plots, studies to improve recombinant enzyme <span class="trait">yield,</span> as well as stability and <span class="trait">activity</span> of the enzyme will be of interest.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Production of nitric oxide and nitrosylleghemoglobin complexes in soybean nodules in response to flooding.</h2>
			<h3>PMID:20367476  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sánchez C(1), Gates AJ, Meakin GE, Uchiumi T, Girard L, Richardson DJ, Bedmar EJ, Delgado MJ.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitric oxide (NO) has gained interest as a major signaling molecule during plant development and in response to environmental cues. Formation of NO during symbiotic interactions has been reported, but the role and sources of NO in nodules remain unclear. In this work, the involvement of denitrification, performed by the symbiont Bradyrhizobium japonicum, in NO formation in soybean nodules in response to flooding conditions has been investigated by inoculating plants with napA-, nirK-, or norC-deficient mutants. Levels of nitrosylleghemoglobin (LbNO) in flooded <span class="gene">nirK</span> and <span class="gene">norC</span> nodules were significantly higher than those observed in wild-type nodules. In addition, <span class="gene">nirK</span> and <span class="gene">norC</span> nodules accumulated more nitrite and NO, respectively, than wild-type nodules. By contrast, levels of LbNO, nitrite, and NO in flooded <span class="gene">napA</span> nodules were lower than in wild-type nodules. These results suggest that LbNO formation in soybean nodules in response to flooding conditions is caused by nitrite and NO generated from periplasmic nitrate reductase (Nap) and also containing nitrite reductase (NirK) denitrification enzymes. Flooding caused a decrease of <span class="gene">nifH</span> expression and nitrogenase <span class="trait">activity</span> in wild-type and <span class="gene">norC</span> nodules but not in <span class="gene">napA</span> or <span class="gene">nirK</span> nodules. Incubation of wild-type and <span class="gene">norC</span> nodules with a NO scavenger counteracted the effect of flooding. Under free-living conditions, beta-galactosidase <span class="trait">activity</span> from a nifD'-'lacZ fusion decreased in a <span class="gene">norC</span> mutant, which also accumulated NO in the medium. These results suggest that NO formed by Cu-containing nitrite reductase in soybean nodules in response to flooding has a negative effect on expression of nitrogenase. We propose that Lb has a major role in detoxifying NO and nitrite produced by bacteroidal denitrification in response to flooding conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and characterization of a novel soybean receptor-like protein kinase gene <span class="gene">GmRLCK.</span></h2>
			<h3>PMID:16944594  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li PL(1), Ma YY, Gan R, Zhang LW, Li XP, Wang Y, Wang NN.
				<br/> 
				年份：2006 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Protein kinases play key roles in transduction of both environmental and developmental signals in higher plants. Here reported full-length cDNA cloning and preliminary structural and functional analysis of a novel soybean receptor-like protein kinase (RLK) gene (GenBank Accession No.AY687390). The deduced protein product of this gene contains a transmembrane region, an intracellular catalytic domain possessing serine/threonine kinase <span class="trait">activity,</span> and an extracellular domain which lacks the N-terminus signal peptide. Bioinformatic analyses revealed highly similarity between this soybean gene and a group of the Arabidopsis <span class="gene">RLK</span> genes which all lack of N-terminus signal sequence and belong to RLCK subfamily. Thus, it was designated <span class="gene">GmRLCK.</span> Some residues of <span class="gene">GmRLCK</span> that showed high probability to be phosphorylated were also predicted. RT-PCR analysis revealed higher expression levels of <span class="gene">GmRLCK</span> in soybean cotyledons, roots, flowers and young pods; whereas lower expression levels were detected in radicles, stems and mature leaves at vegetative stage. Phylogeny analysis suggested that <span class="gene">GmRLCK</span> was more close to and might have a common ancestor with several senescence-associated RLKs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Auxin distribution in Lotus japonicus during <span class="trait">root</span> nodule development.</h2>
			<h3>PMID:14682616  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pacios-Bras C(1), Schlaman HR, Boot K, Admiraal P, Langerak JM, Stougaard J, Spaink HP.
				<br/> 
				年份：2003 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For this work, Lotus japonicus transgenic plants were constructed expressing a fusion reporter gene consisting of the genes beta-glucuronidase (gus) and green fluorescent protein (gfp) under control of the soybean auxin-responsive promoter GH3. These plants expressed GUS and GFP in the vascular bundle of shoots, roots and leafs. <span class="trait">Root</span> sections showed that in mature parts of the roots GUS is mainly expressed in phloem and vascular parenchyma of the vascular cylinder. By detecting GUS <span class="trait">activity,</span> we describe the auxin distribution pattern in the <span class="trait">root</span> of the determinate nodulating legume L. japonicus during the development of nodulation and also after inoculation with purified Nod factors, N-naphthylphthalamic acid (NPA) and indoleacetic acid (IAA). Differently than white clover, which forms indeterminate nodules, L. japonicus presented a strong GUS <span class="trait">activity</span> at the dividing outer cortical cells during the first nodule cell divisions. This suggests different auxin distribution pattern between the determinate and indeterminate nodulating legumes that may be responsible of the differences in nodule development between these groups. By measuring of the GFP fluorescence expressed 21 days after treatment with Nod factors or bacteria we were able to quantify the differences in <span class="gene">GH3</span> expression levels in single living roots. In order to correlate these data with auxin transport capacity we measured the auxin transport levels by a previously described radioactive method. At 48 h after inoculation with Nod factors, auxin transport showed to be increased in the middle <span class="trait">root</span> segment. The results obtained indicate that L. japonicus transformed lines expressing the GFP and GUS reporters under the control of the <span class="gene">GH3</span> promoter are suitable for the study of auxin distribution in this legume.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant and fungal lipoxygenases.</h2>
			<h3>PMID:12432926  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Oliw EH(1).
				<br/> 
				年份：2002 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Lipoxygenases (LOX) are ubiquitous in plants, and some are important for <span class="trait">resistance</span> to pathogens. The soybean <span class="gene">LOX</span> (SLOX) remain the prototypes for studying lipoxygenation. The 3-D structures of SLOX-1 and -3 reveal an N-terminal beta-barrel and a C-terminal catalytic domain. The beta-barrel with 30 kDa of the N-terminal peptide chain of SLOX-1 can be separated from the catalytic domain (60 kDa), which is a functional lipoxygenase designated "mini-LOX." The 3-D structures have made it possible to predict and interpret effects of point mutations on catalytic parameters and on the substrate and position specificity and to identify the catalytic base as Fe3+-OH-. The first catalytic step, hydrogen abstraction at C-11 of linoleic acid by Fe3+-OH-, is associated with a very large isotope effect (20-80), which can be explained by the quantum-mechanical concept of hydrogen tunneling "through a 2-D potential barrier." The prosthetic iron of SLOX-3 can be extracted and the apoenzyme can incorporate Fe3+ and regain <span class="gene">LOX</span> <span class="trait">activity.</span> Plant <span class="gene">LOX</span> can be elicited by fungal pathogens. One fungal <span class="gene">LOX</span> has been cloned and characterized. It is secreted by the take-all fungus, and the enzyme may contribute to its detrimental actions on wheat roots. The primary structure of Mn-LOX shows that it belongs to the <span class="gene">LOX</span> gene family, the metal ligands appear to be homologous, and electron paramagnetic resonance (EPR) spectra suggest redox cycling between Mn2+ and Mn3+. The hypothetical catalytic base, Mn3+-OH-, abstracts the pro-S hydrogen at C-11, and molecular oxygen is inserted in a suprafacial way at C-11 and -13 in a 1:3 ratio. Mn-LOX catalyses the conversion of the 11S-hydroperoxide to 13R-hydroperoxylinoleic acid. Mn-LOX is a novel tool for studying lipoxygenation and plant-fungal interactions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Glyphosate on Metabolism of Phenolic Compounds: V. l-alpha-AMINOOXY-beta-PHENYLPROPIONIC ACID AND GLYPHOSATE EFFECTS ON PHENYLALANINE AMMONIA-LYASE IN SOYBEAN SEEDLINGS.</h2>
			<h3>PMID:16661135</h3>
				

				
			<p>
				作者：Duke SO(1), Hoagland RE, Elmore CD.
				<br/> 
				年份：1980 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The phenylalanine ammonia-lyase (PAL) inhibitor l-alpha-aminooxy-beta-phenylpropionic acid (AOPP) was root-fed to light-exposed soybean seedlings alone or with glyphosate [N-(phosphonomethyl)glycine] to test further the hypothesis that <span class="gene">PAL</span> <span class="trait">activity</span> is involved in the mode of action of glyphosate. Extractable <span class="gene">PAL</span> <span class="trait">activity</span> was increased by 0.01 and 0.1 millimolar AOPP. AOPP reduced total soluble hydroxyphenolic compound levels and increased phenylalanine and tyrosine levels, indicating that in vivo <span class="gene">PAL</span> <span class="trait">activity</span> was inhibited by AOPP. The increase in extractable <span class="gene">PAL</span> caused by AOPP may be a result of decreased feedback inhibition of <span class="gene">PAL</span> synthesis by cinnamic acid and/or its derivatives. AOPP alone had no effect on growth (fresh weight and elongation) at either concentration, but at 0.1 millimolar it slightly alleviated growth (fresh weight) inhibition caused by 0.5 millimolar glyphosate after 4 days. Reduction of the free pool of phenylalanine by glyphosate was reversed by AOPP. These results indicate that glyphosate exerts some of its effects through reduction of aromatic amino acid pools through increases in <span class="gene">PAL</span> <span class="trait">activity</span> and that not all growth effects of glyphosate are due to reductions of aromatic amino acids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the Hydrogen Uptake Gene Cluster for Chemolithoautotrophic Growth and Symbiosis Hydrogen Uptake in <italic> Bradyrhizobium Diazoefficiens </italic></h2>
			<h3>PMID:26911707</h3>
				
					<h3>PMCID:4791120</h3>
					<h4>期刊:Microbes and Environments</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Masuda </surname>  <given-names> Sachiko </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  <xref rid="c1-31_76" ref-type="corresp"> * </xref>  <xref rid="fn1-31_76" ref-type="author-notes"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saito </surname>  <given-names> Masaki </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sugawara </surname>  <given-names> Chiaki </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Itakura </surname>  <given-names> Manabu </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Eda </surname>  <given-names> Shima </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Minamisawa </surname>  <given-names> Kiwamu </given-names>  </name>  <xref ref-type="aff" rid="af1-31_76"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, the mutation of <italic> <span class="gene">hupS</span> </italic> <italic> <sub> 2 </sub> </italic> <italic> L </italic> <italic> <sub> 2 </sub> </italic> did not change nodule H <sub> 2 </sub> production from that by wild-type USDA110 ( <xref rid="f2-31_76" ref-type="fig"> Fig. 2C </xref> ), even though some genes on <italic> hup </italic> locus II were up-regulated in symbiotic bacteroids ( <xref rid="b1-31_76" ref-type="bibr"> 1 </xref> , <xref rid="b6-31_76" ref-type="bibr"> 6 </xref> ). <italic> Hup </italic> locus I contains a complete set of the <italic> hup </italic> - <italic> hyp </italic> - <italic> hox </italic> cluster, and is missing from <italic> hup </italic> locus II ( <xref rid="b11-31_76" ref-type="bibr"> 11 </xref> ). Thus, <italic> <span class="gene">hupS</span> </italic> <italic> <sub> 2 </sub> </italic> <italic> L </italic> <sub> 2 </sub> in locus II may not be fully induced without the <italic> hup </italic> gene assemblage, resulting in no or weak Hup <span class="trait">activity</span> by the <italic> <span class="gene">hupS</span> </italic> <italic> <sub> 2 </sub> </italic> <italic> L </italic> <sub> 2 </sub> genes. On the other hand, the <italic> hup </italic> gene cluster outside the symbiosis island, which we identified as the primary <italic> hup </italic> gene cluster contributing to Hup <span class="trait">activity</span> in free-living and symbiotic cells, is located on a typical genomic island (trnM element) of <italic> B. diazoefficiens </italic> USDA110. The trnM element is likely acquired in the USDA110 lineage after the divergence of strains USDA110 and USDA6 <sup> T </sup> because <italic> B. japonicum </italic> USDA6 <sup> T </sup> completely lacks this element ( <xref rid="b10-31_76" ref-type="bibr"> 10 </xref> , <xref rid="b11-31_76" ref-type="bibr"> 11 </xref> , <xref rid="b12-31_76" ref-type="bibr"> 12 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean ureide transporters play a critical role in nodule development, function and nitrogen export.</h2>
			<h3>PMID:22725647  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Collier R(1), Tegeder M.
				<br/> 
				年份：2012 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legumes can access atmospheric nitrogen through a symbiotic relationship with nitrogen-fixing bacteroids that reside in <span class="trait">root</span> nodules. In soybean, the products of fixation are the ureides allantoin and allantoic acid, which are also the dominant long-distance transport forms of nitrogen from nodules to the shoot. Movement of nitrogen assimilates out of the nodules occurs via the nodule vasculature; however, the molecular mechanisms for ureide export and the importance of nitrogen transport processes for nodule physiology have not been resolved. Here, we demonstrate the function of two soybean proteins - <span class="gene">GmUPS1-1</span> (XP_003516366) and <span class="gene">GmUPS1-2</span> (XP_003518768) - in allantoin and allantoic acid transport out of the nodule. Localization studies revealed the presence of both transporters in the plasma membrane, and expression in nodule cortex cells and vascular endodermis. Functional analysis in soybean showed that repression of <span class="gene">GmUPS1-1</span> and <span class="gene">GmUPS1-2</span> in nodules leads to an accumulation of ureides and decreased nitrogen partitioning to roots and shoot. It was further demonstrated that nodule development, nitrogen fixation and nodule metabolism were negatively affected in RNAi UPS1 plants. Together, we conclude that export of ureides from nodules is mediated by UPS1 proteins, and that <span class="trait">activity</span> of the transporters is not only essential for shoot nitrogen supply but also for nodule development and function.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of glyphosate application on seed iron and <span class="trait">root</span> ferric (III) reductase in soybean cultivars.</h2>
			<h3>PMID:19780538  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bellaloui N(1), Reddy KN, Zablotowicz RM, Abbas HK, Abel CA.
				<br/> 
				年份：2009 Oct 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous research demonstrated that plant nutrient assimilation was reduced by glyphosate (Gly). A 2 year field experiment investigated the effects of Gly at drift rate (12.5% of commercial use rate) on Fe concentrations in leaves and seeds of Gly-sensitive (GS) soybean, and a greenhouse experiment evaluated Gly effects on Fe assimilation using <span class="trait">root</span> in vivo ferric reductase <span class="trait">activity</span> (FRA) in two <span class="gene">GS</span> and one Gly-resistant (GR) soybean cultivars. Field studies showed that Gly drift rates resulted in a significant decrease in the Fe concentration in seeds and leaves compared to the nontreated plants. In greenhouse studies, leaf Fe and FRA were inhibited in <span class="gene">GS</span> cultivars Hutcheson and DP 5110 and the <span class="gene">GR</span> cultivar AG 4604RR and leaf Fe was positively correlated with <span class="trait">root</span> FRA (p &lt; 0.0001). These results indicate that Gly can interfere with Fe assimilation in both <span class="gene">GS</span> and <span class="gene">GR</span> soybean. Understanding the implication of Gly on Fe nutrition in soybean seed would help soybean agronomists and breeders seeking to improve seed mineral nutrition qualities.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The distribution of aquaporin subtypes (PIP1, <span class="gene">PIP2</span> and gamma-TIP) is tissue dependent in soybean (Glycine max) <span class="trait">root</span> nodules.</h2>
			<h3>PMID:15987697  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fleurat-Lessard P(1), Michonneau P, Maeshima M, Drevon JJ, Serraj R.
				<br/> 
				年份：2005 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND AND AIMS The inner cortical cells (IC-cells) of legume <span class="trait">root</span> nodules have been previously shown to regulate the <span class="trait">resistance</span> to nodule O2 diffusion by a rapid contraction/expansion mechanism, which controls the volume of intercellular spaces and their occlusion by a liquid phase. The expression of aquaporins in IC-cells was also found to be involved in this nodule O2 diffusion mechanism. The aim of this study was to compare the expression of plasma membrane intrinsic proteins (PIP) aquaporin isoforms with tonoplast intrinsic protein (gamma-TIP) in both IC-cells and adjacent cell types.METHODS: Using immunogold labelling in ultra-thin sections of Glycine max nodules, the expression of two <span class="gene">PIP</span> isoforms was observed and compared with the gamma-TIP pattern. KEY RESULTS: The plasma membrane aquaporins PIP1 and <span class="gene">PIP2</span> were expressed more in IC-cells and endodermis than in pericycle and infected cells. The tonoplast aquaporin gamma-TIP has shown a distribution pattern similar to that of the PIPs. CONCLUSIONS: PIPs and gamma-TIP aquaporins are highly expressed in both plasmalemma and tonoplast of nodule IC-cells. This distribution is consistent with the putative role of water fluxes associated with the regulation of nodule conductance to O2 diffusion and the subsequent ATP-dependent nitrogenase <span class="trait">activity.</span> In the endodermis, these aquaporins might also be involved in nutrient transport between the infected zone and vascular traces.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthate metabolism in the source leaves of n(2)-fixing soybean plants.</h2>
			<h3>PMID:16668977</h3>
				

				
			<p>
				作者：de Veau EJ(1), Robinson JM, Warmbrodt RD, Kremer DF.
				<br/> 
				年份：1992 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean plants (Glycine max [L.] Merr. cv Williams), which were symbiotic with Bradyrhizobium japonicum, and which grew well upon reduced nitrogen supplied solely through N(2) fixation processes, often exhibited excess accumulation of starch and sucrose and diminished soluble protein in their source leaves. Nitrate and ammonia, when supplied to the nodulated roots of N(2)-fixing plants, mediated a reduction of foliar starch accumulation and a corresponding increase in soluble protein in the source leaves. This provided an opportunity to examine the potential metabolic adjustments by which NO(3) (-) and NH(4) (+) (N) sufficiency or deficiency exerted an influence upon soybean leaf starch synthesis. When compared with soybean plants supplied with N, elevated starch accumulation was focused in leaf palisade parenchyma tissue of N(2)-fixing plants. Foliar activities of starch synthesis pathway enzymes including fructose-1,6-bisphosphate phosphatase, phosphohexoisomerase, phosphoglucomutase (PGM), as well as adenosine diphosphate glucose pyrophosphorylase (in some leaves) exhibited highest activities in leaf extracts of N(2)-fixing plants when expressed on a leaf protein basis. This was interpreted to mean that there was an adaptation of these enzyme activities in the leaves of N(2)-fixing plants, and this contributed to an increase in starch accumulation. Another major causal factor associated with increased starch accumulation was the elevation in foliar levels of fructose-6-phosphate, glucose-6-phosphate, and glucose-1-phosphate (G1P), which had risen to chloroplast concentrations considerably in excess of the K(m) values for their respective target enzymes associated with starch synthesis, e.g. elevated G1P with respect to adenosine diphosphate glucose pyrophosphorylase (ADPG-PPiase) binding sites. The cofactor glucose-1,6-bisphosphate (G1,6BP) was found to be obligate for maximal <span class="gene">PGM</span> <span class="trait">activity</span> in soybean leaf extracts of N(2)-fixing as well as N-supplemented plants, and G1,6BP levels in N(2)-fixing plant leaves was twice that of levels in N-supplied treatments. However the concentration of chloroplastic G1,6BP in illuminated leaves was computed to be saturating with respect to <span class="gene">PGM</span> in both N(2)-fixing and N-supplemented plants. This suggested that the higher level of this cofactor in N(2)-fixing plant leaves did not confer any higher <span class="gene">PGM</span> activation and was not a factor in higher starch synthesis rates. Relative to plants supplied with NO(3) (-) and NH(4) (+), the source leaf glycerate-3-phosphate (3-PGA) and orthophosphate (Pi) concentrations in leaves of N(2)-fixing plants were two to four times higher. Although Pi is a physiological competitive inhibitor of leaf chloroplast ADPG-PPiase, and hence, starch synthesis, elevated chloroplast 3-PGA levels in N(2)-fixing plant leaves apparently prevented interference of Pi with ADPG-PPiase catalysis and starch synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Appearance and accumulation of nodulin mRNAs and their relationship to the effectiveness of <span class="trait">root</span> nodules.</h2>
			<h3>PMID:24310256</h3>
				

				
			<p>
				作者：Fuller F(1), Verma DP.
				<br/> 
				年份：1984 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cloned cDNAs corresponding to mRNAs which accumulate in nitrogen-fixing <span class="trait">root</span> nodules of soybean (nodulin mRNAs) were used as probes to investigate the sizes, sequence relationships, tissue specificities and developmental accumulations of individual nodulin mRNA sequences. Northern blot analysis indicated that the NodB, NodC and NodD mRNA sequences are 1 150, 770, and 3 150 nucleotides long, respectively, which is consistent with the previously determined sizes of the hybrid-selected translation products (27 000, 24 000 and 100 000 MW, respectively). The <span class="gene">NodA</span> clones pNodA15 and pNodA25 hybridized to two mRNAs of lengths 1 600 and 1 100 nucleotides, indicating that they contain significant sequence homologies. However, increasing the hybridization stringency showed that the pNodA15 clone encodes the 1 600 nucleotide mRNA corresponding to the major <span class="gene">NodA</span> hybrid-selected translation product (44 000 MW) while pNodA25 encodes an mRNA of 1 100 nucleotides. The latter probably corresponds to one of two smaller (23 500 and 24 500 MW) in vitro translation products. RNA dot-blot hybridizations indicated that nodulin and leghemoglobin mRNAs began to appear and accumulate in Rhizobium infected <span class="trait">root</span> tissue very early (day 3 to 5) and reached fully induced levels by day 11. This accumulation was specific for nodule tissue (except for the NodD sequence) and preceded the accumulation of nitrogen fixation <span class="trait">activity.</span> Nodules produced by different effective Rhizobium strains accumulated similar levels of leghemoglobin and nodulin mRNAs while ineffective strains had a pleiotropic affect. While one ineffective strain (61A24) gave reduced levels of all these mRNAs, the other (SM5) gave levels which were nearly normal by the time nitrogen fixation <span class="trait">activity</span> should have reached its maximal level (day 17). Thus, leghemoglobin and nodulin genes are switched on soon after infection, prior to nodule morphogenesis, and the switch occurs prior to and is independent of nitrogen fixation <span class="trait">activity.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structural analysis of urate oxidase in complex with its natural substrate inhibited by cyanide: Mechanistic implications</h2>
			<h3>PMID:18638417</h3>
				
					<h3>PMCID:2490695</h3>
					<h4>期刊:BMC Structural Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" equal-contrib="yes" contrib-type="author">  <name>  <surname> Gabison </surname>  <given-names> Laure </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> laure.gabison@univ-paris5.fr </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Prang&#x000e9; </surname>  <given-names> Thierry </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> thierry.prange@univ-paris5.fr </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Colloc'h </surname>  <given-names> Nathalie </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> colloch@cyceron.fr </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> El Hajji </surname>  <given-names> Mohamed </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> mohamed.el-hajji@sanofi-aventis.com </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Castro </surname>  <given-names> Bertrand </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> bertrand.castro@sanofi-aventis.com </email>  </contrib>  <contrib id="A6" equal-contrib="yes" corresp="yes" contrib-type="author">  <name>  <surname> Chiadmi </surname>  <given-names> Mohamed </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> mohammed.chiadmi@univ-paris5.fr </email>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 7 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Two distinct enzymatic intermediates were proposed in the initial steps of the urate oxidase reaction, the urate dianion and 5-hydroperoxyisourate (5-HPIU), based on their spectral signatures [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] &#x02013; Scheme 1. At the end of the first step, the elimination of hydrogen peroxide from 5-HPIU would <span class="trait">yield</span> to dehydrourate that is not experimentally observed [ <xref ref-type="bibr" rid="B18"> 18 </xref> ], probably because it does not accumulate during the enzymatic turnover. The final step of the reaction is the rapid nucleophilic addition of a hydroxyl group deriving from water <span class="gene">W1</span> onto dehydrourate to give 5-HIU and a re-protonation by the water molecule <span class="gene">W2</span> (Figures <xref ref-type="fig" rid="F1"> 1 </xref> and <xref ref-type="fig" rid="F3"> 3 </xref> ). The characterization of discrete intermediates during the urate oxidase reaction ruled out any concerted mechanism in which <span class="gene">W1</span> would attack at <span class="gene">C5</span> simultaneously with direct electron transfer from urate to oxygen.
					</p>
				
					<p>&nbsp;&nbsp;
						It is known that <span class="gene">UOX</span> is inhibited in solution by cyanide with a loss of <span class="trait">activity</span> of 90 % [ <xref ref-type="bibr" rid="B11"> 11 </xref> , <xref ref-type="bibr" rid="B24"> 24 </xref> , <xref ref-type="bibr" rid="B25"> 25 </xref> ]. The same holds for azide, but at a lesser extent [ <xref ref-type="bibr" rid="B25"> 25 </xref> ], questioning at that time about the presence of a heavy metal in the active site, as many metallo-oxidases are analogously poisoned by cyanide. For example, irreversible inhibition is observed when xanthine oxidase is incubated with cyanide in the absence of substrate. By analogy, cyanide was assumed to interact with the dioxygen site in UOX, although not involved in the formation of any intermediate structure [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. However, compared to xanthine oxidase, the inhibition of <span class="gene">UOX</span> by cyanide is non competitive <italic> versus </italic> urate, increases with time, and is perfectly reversible [ <xref ref-type="bibr" rid="B11"> 11 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:MucR Is Required for Transcriptional Activation of Conserved Ion Transporters to Support Nitrogen Fixation of Sinorhizobium fredii in Soybean Nodules.</h2>
			<h3>PMID:26883490  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jiao J(1)(2)(3), Wu LJ(1)(2)(3), Zhang B(1), Hu Y(1)(2)(3), Li Y(1)(2)(3), Zhang XX(1)(2)(3), Guo HJ(1)(2)(3), Liu LX(1)(2)(3), Chen WX(1)(2)(3), Zhang Z(1), Tian CF(1)(2)(3).
				<br/> 
				年份：2016 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To achieve effective symbiosis with legume, rhizobia should fine-tune their background regulation network in addition to activating key genes involved in nodulation (nod) and nitrogen fixation (nif). Here, we report that an ancestral zinc finger regulator, MucR1, other than its paralog, MucR2, carrying a frameshift mutation, is essential for supporting nitrogen fixation of Sinorhizobium fredii CCBAU45436 within soybean nodules. In contrast to the chromosomal mucR1, <span class="gene">mucR2</span> is located on symbiosis plasmid, indicating its horizontal transfer potential. A MucR2 homolog lacking the frameshift mutation, such as the one from S. fredii NGR234, can complement phenotypic defects of the <span class="gene">mucR1</span> mutant of CCBAU45436. RNA-seq analysis revealed that the MucR1 regulon of CCBAU45436 within nodules exhibits significant difference compared with that of free-living cells. MucR1 is required for active expression of transporters for phosphate, zinc, and elements essential for nitrogenase <span class="trait">activity</span> (iron, molybdenum, and sulfur) in nodules but is dispensable for transcription of key genes (nif/fix) involved in nitrogen fixation. Further reverse genetics suggests that S. fredii uses high-affinity transporters to meet the demand for zinc and phosphate within nodules. These findings, together with the horizontal transfer potential of the mucR homolog, imply an intriguing evolutionary role of this ancestral regulator in supporting nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean miR172c targets the repressive <span class="gene">AP2</span> transcription factor NNC1 to activate <span class="gene">ENOD40</span> expression and regulate nodule initiation.</h2>
			<h3>PMID:25549672  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang Y(1), Wang L(2), Zou Y(1), Chen L(1), Cai Z(2), Zhang S(2), Zhao F(1), Tian Y(1), Jiang Q(2), Ferguson BJ(3), Gresshoff PM(3), Li X(4).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MicroRNAs are noncoding RNAs that act as master regulators to modulate various biological processes by posttranscriptionally repressing their target genes. Repression of their target mRNA(s) can modulate signaling cascades and subsequent cellular events. Recently, a role for miR172 in soybean (Glycine max) nodulation has been described; however, the molecular mechanism through which miR172 acts to regulate nodulation has yet to be explored. Here, we demonstrate that soybean miR172c modulates both rhizobium infection and nodule organogenesis. miR172c was induced in soybean roots inoculated with either compatible Bradyrhizobium japonicum or lipooligosaccharide Nod factor and was highly upregulated during nodule development. Reduced <span class="trait">activity</span> and overexpression of miR172c caused dramatic changes in nodule initiation and nodule number. We show that soybean miR172c regulates nodule formation by repressing its target gene, Nodule Number Control1, which encodes a protein that directly targets the promoter of the early nodulin gene, ENOD40. Interestingly, transcriptional levels of miR172c were regulated by both Nod Factor Receptor1α/5α-mediated activation and by autoregulation of nodulation-mediated inhibition. Thus, we established a direct link between miR172c and the Nod factor signaling pathway in addition to adding a new layer to the precise nodulation regulation mechanism of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Activation of phenylpropanoid pathway in legume plants exposed to heavy metals. Part I. Effects of cadmium and lead on phenylalanine ammonia-lyase gene expression, enzyme <span class="trait">activity</span> and lignin content.</h2>
			<h3>PMID:21503278  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pawlak-Sprada S(1), Arasimowicz-Jelonek M, Podgórska M, Deckert J.
				<br/> 
				年份：2011
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Species-specific changes in expression of phenylalanine ammonia-lyase (PAL) and lignin content were detected in roots of soybean (Glycine max L.) and lupine (Lupinus luteus L.) seedlings treated with different concentrations of cadmium (Cd(2+), 0-25 mg/l) or lead (Pb(2+), 0-350 mg/l). The stimulatory effect of both metals was observed in mRNA coding for <span class="gene">PAL</span> in soybean. In the case of lupine, changes of <span class="gene">PAL</span> mRNA level were dependent on the metal used: Cd(2+) caused a decrease, whereas Pb(2+) an increase of <span class="gene">PAL</span> transcript level. The <span class="trait">activity</span> of <span class="gene">PAL</span> was enhanced in both plant species at higher metal concentrations (15-25 mg/l of Cd(2+) or 150-350 mg/l of Pb(2+)); however it was not directly correlated with <span class="gene">PAL</span> mRNA. This suggests a transcriptional and posttranscriptional control of <span class="gene">PAL</span> expression under heavy metals stress. In soybean, Cd(2+) or Pb(2+) treatment increased lignin content, while in lupine the effect was opposite. The decreased lignin accumulation in lupine roots in response to heavy metals, despite an increased <span class="gene">PAL</span> <span class="trait">activity,</span> suggests that the activated phenylpropanoid pathway was involved in the synthesis of secondary metabolites other than lignin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Suppression of hypernodulation in soybean by a leaf-extracted, NARK- and Nod factor-dependent, low molecular mass fraction.</h2>
			<h3>PMID:20100211  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin YH(1), Ferguson BJ, Kereszt A, Gresshoff PM.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						*Legumes regulate the number of nodules they form via a process called autoregulation of nodulation (AON). This involves a shoot-derived inhibitor (SDI) molecule that is synthesized in the shoots and is transported down to the roots where it inhibits further nodule development. *To characterize SDI, we developed a novel feeding bioassay. This involved feeding aqueous leaf extracts directly into the petiole of hypernodulating and supernodulating nark mutant plants of Glycine max (soybean). These mutants normally exhibit an increased nodulation phenotype because SDI is not produced and thus AON is nonfunctional. *Feeding wild-type leaf extracts presumed to contain SDI was successful in suppressing the increased nodulation phenotype, whereas feeding with <span class="gene">Gmnark</span> leaf extracts did not. Suppression <span class="trait">activity</span> was inoculation-dependent, Nod factor-dependent, required <span class="gene">GmNARK</span> <span class="trait">activity,</span> and was heat-, Proteinase K- and ribonuclease A-resistant. Wild-type extracts maintained suppressive <span class="trait">activity</span> even at a ninefold dilution. Sinorhizobium meliloti-inoculated Medicago truncatula leaf extracts from wild-type, but not from supernodulating mutant Mtsunn, suppressed hypernodulation in soybean. *Our results demonstrate that the petiole feeding bioassay is an efficient and effective technique to introduce aqueous extracts into plants. They also demonstrate that SDI is a small compound with an apparent molecular mass of &lt; 1000 Da and is unlikely to be a protein or an RNA molecule.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of the overexpression of a soybean cytosolic glutamine synthetase gene (GS15) linked to organ-specific promoters on growth and nitrogen accumulation of pea plants supplied with ammonium.</h2>
			<h3>PMID:17067806  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Fei H(1), Chaillou S, Hirel B, Polowick P, Mahon JD, Vessey JK.
				<br/> 
				年份：2006 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A soybean cytosolic glutamine synthetase gene (GS15) fused to a constitutive promoter (CaMV 35S), a putative nodule-specific promoter (LBC(3)), or a putative root-specific promoter (rolD) was transformed into Pisum sativum L. cv. Greenfeast. Four lines with single copies (Lines 1, 7, 8 and 9) and four lines with two copies each of GS15 (Lines 2, 4, 6 and 11) were compared to the wild-type (WT) parental line for levels of cytosolic glutamine synthetase (GS1), glutamine synthetase (GS) <span class="trait">activity,</span> N accumulation, N derived form the atmosphere (NDFA), and biomass of plants grown on 0.0, 0.1, 1.0 or 10.0 mM NH(4)(+). Enhanced levels of GS1 were detected in leaves of one of the two lines transformed with the 35S-GS15 construct, and all three lines containing the rolD-GS15 construct. All three lines containing the LBC(3)-GS15 construct had increased levels of GS1 in nodules. Despite the increased levels of GS1 in many transformants, only the roots of lines containing the rolD-GS15 construct consistently demonstrated enhanced levels of <span class="gene">GS</span> <span class="trait">activity</span> (up to 12-fold). Positive responses in plant N content, NDFA, and biomass were rare, but increases in plant biomass and N content of up to 17% and 54%, respectively, occurred in some of the rolD-GS15 lines at certain levels of ammonium. In general, GS15 copy number did not seem to differentially affect phenotype of the transformants, and transformants respond to ammonium concentrations in similar patterns to that previously observed with nitrate. Despite the fact that the rolD-GS15 transformants consistently resulted in increased <span class="gene">GS</span> <span class="trait">activity</span> in roots and resulted in some occurrences of increases in biomass and plant N content, the lack of consistent positive growth effect across all transformants indicates that the generalized overexpression of GS1 in tissues holds little potential for positive growth responses in pea.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Symbiotic Bradyrhizobium japonicum reduces N2O surrounding the soybean <span class="trait">root</span> system via nitrous oxide reductase.</h2>
			<h3>PMID:16597953  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sameshima-Saito R(1), Chiba K, Hirayama J, Itakura M, Mitsui H, Eda S, Minamisawa K.
				<br/> 
				年份：2006 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						N(2)O reductase <span class="trait">activity</span> in soybean nodules formed with Bradyrhizobium japonicum was evaluated from N(2)O uptake and conversion of (15)N-N(2)O into (15)N-N(2). Free-living cells of USDA110 showed N(2)O reductase <span class="trait">activity,</span> whereas a <span class="gene">nosZ</span> mutant did not. Complementation of the <span class="gene">nosZ</span> mutant with two cosmids containing the nosRZDFYLX genes of B. japonicum USDA110 restored the N(2)O reductase <span class="trait">activity.</span> When detached soybean nodules formed with USDA110 were fed with (15)N-N(2)O, they rapidly emitted (15)N-N(2) outside the nodules at a ratio of 98.5% of (15)N-N(2)O uptake, but nodules inoculated with the <span class="gene">nosZ</span> mutant did not. Surprisingly, N(2)O uptake by soybean roots nodulated with USDA110 was observed even in ambient air containing a low concentration of N(2)O (0.34 ppm). These results indicate that the conversion of N(2)O to N(2) depends exclusively on the respiratory N(2)O reductase and that soybean roots nodulated with B. japonicum carrying the nos genes are able to remove very low concentrations of N(2)O.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative analysis of phytoestrogens in kudzu-root, soy and spiked serum samples by high-pressure liquid chromatography.</h2>
			<h3>PMID:15273977  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Benlhabib E(1), Baker JI, Keyler DE, Singh AK.
				<br/> 
				年份：2004 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A sensitive and reliable HPLC method that allows simultaneous quantification of phytoestrogens extracted from kudzu-root and soy preparations, and serum samples has been developed. Kudzu-root and soy preparations were mixed with 5 microg flavone and 15 microg rutin (internal standards) and the phytoestrogens were extracted by using solid-phase (C18) extraction cartridges. Blank or spiked serum samples were extracted by using either C18 cartridges or trichloroacetic acid-methanol extraction. The extracts were analyzed by the HPLC equipped with a reverse-phase (250 x 4 mm, C18) column and UV, diode-array or <span class="gene">MS</span> detector. A linear gradient of acetic acid and acetonitrile provided excellent separation of glycoside and aglycone-phytoestrogens from kudzu <span class="trait">root</span> and soy preparations. The C18 cartridge extraction of serum yielded excellent recovery of both glycoside- and aglycone-phytoestrogens, while the trichloroacetic acid-methanol extraction yielded excellent recovery of glycoside but poor recovery of aglycone compounds. UV and <span class="gene">MS</span> detectors were suitable for phytoestrogen analysis in plant and serum samples, while the diode-array detector was suitable for generating the UV absorbance curve for phytoestrogens.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential transcription of the two glutamine synthetase genes of Bradyrhizobium japonicum.</h2>
			<h3>PMID:2445733  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carlson TA(1), Martin GB, Chelm BK.
				<br/> 
				年份：1987 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bradyrhizobium japonicum induces the formation of nitrogen-fixing symbiotic <span class="trait">root</span> nodules on soybean plants. The B. japonicum genome encodes two isoforms of glutamine synthetase (GS). One form, GSI, encoded by the gene glnA, is similar in structure and <span class="trait">activity</span> to the enzyme found in all other bacteria. The second form, GSII, encoded by glnII, is structurally related to the eucaryotic enzyme. Genetic analyses indicate that <span class="gene">glnA</span> or <span class="gene">glnII</span> alone is sufficient to provide glutamine prototrophy, whereas the double mutation <span class="gene">glnA</span> <span class="gene">glnII</span> produces glutamine auxotrophy. The <span class="gene">glnA</span> gene is transcribed from a single promoter that has a structure most similar to that of the bacterial consensus promoter. The level of transcription of <span class="gene">glnA</span> is not specifically affected by nitrogen limitation of growth. The <span class="gene">glnII</span> gene is also transcribed from a single promoter; however, this promoter has structural features characteristic of promoters controlled by the nitrogen regulation system. In contrast to glnA, physiological studies indicate that <span class="gene">glnII</span> transcription is regulated in response to nitrogen source availability. Under aerobic growth conditions, expression of <span class="gene">glnII</span> is induced when growth is limited by nitrogen source depletion as expected for regulation by the nitrogen regulation system.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:cDNA cloning and developmental expression of pea nodulin genes.</h2>
			<h3>PMID:24301264</h3>
				

				
			<p>
				作者：Govers F(1), Nap JP, Moerman M, Franssen HJ, van Kammen A, Bisseling T.
				<br/> 
				年份：1987 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA library prepared from pea nodule poly(A)(+) RNA was screened by differential hybridization with cDNA probes synthesized from <span class="trait">root</span> and nodule RNA respectively. From the cDNA clones that hybridized exclusively with the nodule probe five clones, designated pPsNod 6, 10, 11, 13 and 14 and each containing unique sequences, were further characterized together with one leghemoglobin and one "root-specific" cDNA clone. In vitro translation of RNA selected by the pPsNod clones showed that the corresponding genes encode nodulins with molecular weights ranging from 5 800 to 19 000. During pea <span class="trait">root</span> nodule development expression of the five PsNod genes starts more or less concomitantly with the onset of nitrogen fixing <span class="trait">activity</span> in the nodules and the time course of appearance and accumulation of the nodulin mRNAs is similar to that of leghemoglobin mRNA. In ineffective pea <span class="trait">root</span> nodules expression of the PsNod genes is induced but the final accumulation levels of the mRNAs are markedly reduced to various degrees. The expression of another nodulin gene, designated ENOD2, was followed using a heterologous soybean cDNA clone as probe. In pea <span class="trait">root</span> nodules the <span class="gene">ENOD2</span> gene is expressed at least five days before the PsNod and leghemoglobin genes, and in contrast to the PsNod mRNAs the concentration of the <span class="gene">ENOD2</span> mRNA is the same in wild type and fix (-) nodules. The results described suggest that in <span class="trait">root</span> nodules several regulatory mechanisms exist which determine the final nodulin mRNA amounts accumulating in the <span class="trait">root</span> nodule.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Activation of the Bradyrhizobium japonicum <span class="gene">nifH</span> and nifDK operons is dependent on promoter-upstream DNA sequences.</h2>
			<h3>PMID:3086837  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Alvarez-Morales A, Betancourt-Alvarez M, Kaluza K, Hennecke H.
				<br/> 
				年份：1986 May 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous analysis of B. japonicum nifH'- and nifD'-'lacZ translational fusions showed that these promoters could be activated by the K. pneumoniae <span class="gene">nifA</span> plus the E. coli <span class="gene">ntrA</span> gene products. To study the functions of the DNA 5' to these promoters, plasmids carrying deletions in this region were constructed and analyzed in vivo in a heterologous system consisting of an E. coli (NtrA+) background with a plasmid that constitutively expresses the K. pneumoniae <span class="gene">nifA</span> gene. Activation of the B. japonicum promoters was completely dependent on sequences located between positions -165 and -100, relative to the start of transcription. Some of the <span class="gene">nifD</span> deletion-fusions were mobilized to the wild-type B. japonicum and the exconjugants tested in an ex planta micro-aerobic system, and also used to infect soybean seedlings. The time course of derepression was followed by assaying beta-galactosidase <span class="trait">activity</span> from samples withdrawn from the microaerobic cultures or from root-nodule extracts. The results conclusively show that in the homologous system the sequences upstream of the promoter are required to achieve wild-type <span class="trait">activity.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Sweetpotato Geranylgeranyl Pyrophosphate Synthase Gene, <italic> IbGGPS </italic> , Increases Carotenoid Content and Enhances Osmotic Stress Tolerance in <italic> Arabidopsis thaliana </italic></h2>
			<h3>PMID:26376432</h3>
				
					<h3>PMCID:4574098</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chen </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> He </surname>  <given-names> Shaozhen </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Degao </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant B. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhai </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Feibing </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stephenson </surname>  <given-names> Troy J. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yannan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Bing </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Qingchang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The formation of GGPP is a key step in biosynthetic pathway of carotenoids and many other terpenes. This step is catalyzed by geranylgeranyl pyrophosphate synthase (GGPS) [ <xref rid="pone.0137623.ref003" ref-type="bibr"> 3 </xref> , <xref rid="pone.0137623.ref004" ref-type="bibr"> 4 </xref> ]. <italic> GGPS </italic> cDNAs have been cloned from multiple plant species. A gene expression study using sunflower seedlings showed that the <italic> HaGGPS </italic> was expressed after 2 days of seed imbibition [ <xref rid="pone.0137623.ref005" ref-type="bibr"> 5 </xref> ]. The expression of <italic> GGPS </italic> cloned from <italic> Ipomoea sp </italic> . was found to change dramatically during flower development [ <xref rid="pone.0137623.ref006" ref-type="bibr"> 6 </xref> ]. Recently, the expression of <italic> GGPS </italic> in sweet orange was characterized to be regulated by a miRNA [ <xref rid="pone.0137623.ref007" ref-type="bibr"> 7 </xref> ]. In addition, the expression of <italic> <span class="gene">GPPS</span> </italic> has been found to associate with <span class="trait">resistance</span> of plants to different stresses. The silencing of <italic> NaGGPS </italic> in <italic> Nicotiana attenuate </italic> was found to decrease <span class="trait">resistance</span> to hornworm [ <xref rid="pone.0137623.ref008" ref-type="bibr"> 8 </xref> ]. The expression of GGPS in <italic> Arabidopsis thaliana </italic> has been found to associate with protection against photo-oxidative stress [ <xref rid="pone.0137623.ref009" ref-type="bibr"> 9 </xref> ]. In addition, molecular and function characterization of GGPS has been reported in multiple plant species, such as <italic> Taxus canadensis </italic> [ <xref rid="pone.0137623.ref010" ref-type="bibr"> 10 </xref> ], <italic> A </italic> . <italic> thaliana </italic> [ <xref rid="pone.0137623.ref011" ref-type="bibr"> 11 </xref> , <xref rid="pone.0137623.ref012" ref-type="bibr"> 12 </xref> ], <italic> Scoparia dulcis </italic> and <italic> Croton sublyratus </italic> [ <xref rid="pone.0137623.ref013" ref-type="bibr"> 13 </xref> ], <italic> Helianthus annuus </italic> [ <xref rid="pone.0137623.ref005" ref-type="bibr"> 5 </xref> ], <italic> Gentiana lutea </italic> [ <xref rid="pone.0137623.ref014" ref-type="bibr"> 14 </xref> ], <italic> Hevea brasiliensis </italic> [ <xref rid="pone.0137623.ref015" ref-type="bibr"> 15 </xref> ] and <italic> Solanum lycopersicon </italic> [ <xref rid="pone.0137623.ref016" ref-type="bibr"> 16 </xref> ]. Furthermore, A <italic> GlGGPS </italic> cDNA from <italic> G </italic> . <italic> lutea </italic> was used to rescue a <italic> ggps E </italic> . <italic> coli </italic> mutant [ <xref rid="pone.0137623.ref014" ref-type="bibr"> 14 </xref> ], indicating a functional conservation.
					</p>
				
					<p>&nbsp;&nbsp;
						Polyethylene glycol (PEG)-infused (-0.7 MPa) plates were prepared as described by Verslues et al. [ <xref rid="pone.0137623.ref028" ref-type="bibr"> 28 </xref> ]. Briefly, plates were made by dissolving solid PEG-8000 into a solution of basal <span class="gene">MS</span> medium and adjusting the pH to 5.7. This PEG solution was then added on the surface of an agar-solidified basal medium (3:2, v/v) in plates. All plates were placed on the bench at room temperature for 12 h, during which the PEG solution was immersed into the agar. The left excessive PEG solution on the top of agar was removed. The sterilized seeds of <italic> Arabidopsis </italic> WT and transgenic T <sub> 3 </sub> plants were inoculated on the top of PEG-fused <span class="gene">MS</span> medium. All materials were also be cultured on <span class="gene">MS</span> medium without PEG as the control. Plates were placed vertically on a shelf for at 22&#x000b0;C under 16 h photoperiod supplied with cool-white fluorescent light at 50 &#x003bc;M m <sup> -2 </sup> s <sup> -1 </sup> . After 10 days of germination of seeds, the <span class="trait">root</span> length of seedlings was measured.
					</p>
				
					<p>&nbsp;&nbsp;
						The ORF of IbGGPS was ectopically expressed in <italic> A </italic> . <italic> thaliana </italic> (Col-0) using the binary vector pMDC32-35S- <italic> IbGGPS </italic> ( <xref rid="pone.0137623.g005" ref-type="fig"> Fig 5A </xref> ). Multiple lines were obtained from Hyg <span class="trait">resistance</span> selection. Four transgenic lines (namely L10, L11, L13 and L14) were further demonstrated to highly express the <italic> IbGGPS </italic> transgene by sqRT-PCR analysis ( <xref rid="pone.0137623.g005" ref-type="fig"> Fig 5B </xref> ), thus selected for physiological evaluation. The transgenic plants and WT showed the similar <span class="trait">root</span> growth on <span class="gene">MS</span> medium without PEG ( <xref rid="pone.0137623.g006" ref-type="fig"> Fig 6A and 6B </xref> ). Transgenic plants were more tolerant to PEG-infused (-0.7 MPa) condition than WT. Seeds from L10, L11, L13, L14 and WT were geminated on the same PEG-infused agar-MS medium. The germination times of seeds were similar. After one week of germination, <span class="trait">root</span> growth differentiation phenotypes were observed among the five genotypes. The roots of the 4 transgenic lines were significantly longer than those of WT ( <xref rid="pone.0137623.g006" ref-type="fig"> Fig 6C and 6D </xref> ). In comparison, L13 produced the longest roots ( <xref rid="pone.0137623.g006" ref-type="fig"> Fig 6C and 6D </xref> ). In addition, L11 and L13 lines formed obviously less but longer roots on PEG-infused medium ( <xref rid="pone.0137623.g006" ref-type="fig"> Fig 6C and 6D </xref> ) than on medium without PEG ( <xref rid="pone.0137623.g006" ref-type="fig"> Fig 6C and 6B </xref> ). Osmotic stress might promote the <span class="trait">root</span> elongation in these two lines to get more water, indicating the two lines had the higher osmotic stress tolerance than WT and other transgenic lines. No other morphological or developmental changes were observed in transgenic plans.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Seedlings grown on <span class="gene">MS</span> medium without PEG. (B) Average <span class="trait">root</span> length of two-week old seedlings grown on <span class="gene">MS</span> medium without PEG. (C) Seedlings grown on <span class="gene">MS</span> medium containing PEG. (D) Average <span class="trait">root</span> length of two-week old seedlings grown on <span class="gene">MS</span> medium containing PEG. WT: Col-0; L10, L11, L13 and L14: 4 transgenic <italic> Arabidopsis </italic> lines. Data are presented as means &#x000b1; SE (n = 3). &#x0201c;*&#x0201d; indicates a significant difference from that of WT at P &#x0003c; 0.05 by Student&#x02019;s <italic> t </italic> -test.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Targeted mutagenesis in soybean using the CRISPR-Cas9 system</h2>
			<h3>PMID:26022141</h3>
				

				
					<h3>PMCID:4448504</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Xianjun </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Zheng </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Rui </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Qiyang </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Guohua </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hui </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xi </surname>  <given-names> Yajun </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> College of Agronomy, Northwest A&#x00026;F University </institution> , Yangling, Shaanxi 712100,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> National Key Facilities for Crop Genetic Resources and Improvement, Institute of Crop Sciences, Chinese Academy of Agricultural Sciences </institution> , Beijing 100081,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Tianjin Institute of Agricultural Quality Standard and Testing Technology, Tianjin Academy of Agricultural Sciences </institution> , Tianjin 300381,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 10 </day>  <month> 04 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genome editing is a valuable technique for gene function analysis and crop improvement. Over the past two years, the CRISPR-Cas9 system has emerged as a powerful tool for precisely targeted gene editing. In this study, we predicted 11 U6 genes in soybean (Glycine max L.). We then constructed two vectors (pCas9-GmU6-sgRNA and pCas9-AtU6-sgRNA) using the soybean U6-10 and Arabidopsis U6-26 promoters, respectively, to produce synthetic guide RNAs (sgRNAs) for targeted gene mutagenesis. Three genes, <span class="gene">Glyma06g14180,</span> <span class="gene">Glyma08g02290</span> and <span class="gene">Glyma12g37050,</span> were selected as targets. Mutations of these three genes were detected in soybean protoplasts. The vectors were then transformed into soybean hairy roots by Agrobacterium rhizogenes infection, resulting in efficient target gene editing. Mutation efficiencies ranged from 3.2-9.7% using the pCas9-AtU6-sgRNA vector and 14.7-20.2% with the pCas9-GmU6-sgRNA vector. Biallelic mutations in <span class="gene">Glyma06g14180</span> and <span class="gene">Glyma08g02290</span> were detected in transgenic hairy roots. Off-target activities associated with <span class="gene">Glyma06g14180</span> and <span class="gene">Glyma12g37050</span> were also detected. Off-target <span class="trait">activity</span> would improve mutation efficiency for the construction of a saturated gene mutation library in soybean. Targeted mutagenesis using the CRISPR-Cas9 system should advance soybean functional genomic research, especially that of genes involved in the roots and nodules.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To detect the <span class="trait">activity</span> of these two vectors in soybean, we selected three genes ( <italic> <span class="gene">Glyma06g14180</span> </italic> , <italic> <span class="gene">Glyma08g02290</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> ) as targets for gene editing in soybean. For each gene, we designed a different sgRNA seed with a restriction site in the vicinity of the PAM ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S2 </xref> ). A total of six binary vectors were therefore generated to evaluate targeted mutagenesis in this study.
					</p>
				
					<p>&nbsp;&nbsp;
						We first verified the <span class="trait">activity</span> of the CRISPR-Cas9 system in soybean protoplasts. The vectors were transformed into soybean protoplasts using the polyethylene glycol (PEG)-mediated transformation method. After 48&#x02009;h of incubation in darkness at room temperature, the transformed protoplasts were collected for genomic DNA extraction. A restriction enzyme PCR (RE-PCR) assay was used to detect mutations in the targeted genes. The genomic DNAs for various targeted mutant genes were completely digested with restriction enzymes. The mutant genes were not digested as they lost the enzyme sites and could be amplified using the gene-specific primers. The PCR results confirmed that all six vectors were able to induce targeted gene mutations ( <xref ref-type="fig" rid="f3"> Fig. 3a </xref> ). Sequence analysis revealed that nucleotide substitutions had occurred in <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> ( <xref ref-type="fig" rid="f3"> Figs. 3b,c </xref> ), suggesting that the DSBs in these two genes were repaired by the HR pathway in the soybean protoplasts. One nucleotide deletion and one substitution were found in <italic> <span class="gene">Glyma08g02290</span> </italic> ( <xref ref-type="fig" rid="f3"> Figs. 3b,c </xref> ). The DSBs of <italic> <span class="gene">Glyma08g02290</span> </italic> were repaired through both the HR and NHEJ pathways in soybean protoplasts.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Agrobacterium rhizogenes </italic> ( <italic> A. rhizogenes </italic> )-mediated transformation is a rapid, efficient, simple and inexpensive method for the studying soybean <span class="trait">root</span> biology <xref ref-type="bibr" rid="b34"> 34 </xref> . To detect the targeted gene mutations in soybean roots, we introduced the six binary vectors into <italic> A. rhizogenes </italic> strain K599 and then infected soybean seedling hypocotyls to induce hairy roots. Genomic DNA was collected and extracted for further detection of the target gene mutations from the hairy roots for each of the six vectors. Soybean is a diploid plant and genes have two copies in the homologous chromosomes. The target gene induced by the CRISPR-Cas9 system has three types in the hairy roots. Type I is no mutation of the target gene. Type II is a monoallelic mutation where one gene is mutated and the other allelic gene is no mutated. Type III is a biallelic mutation where both of the two allelic genes are mutated ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figure S1 </xref> ). The gene is amplified using gene specific primers and then digested completely with the restriction enzyme (PCR-RE assay). When the gene mutation is induced by the CRISPR/Cas9 system, the restriction enzyme site in the gene is destroyed. The results of PCR-RE assay for the non-mutation show two digested bands. For the monoallelic mutation, the results are three bands with one undigested band from the mutated gene and two digested bands from non-mutated allelic gene. For the biallelic mutation, both of the two allelic genes are mutated and the PCR-RE assay shows only a single undigested band ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figure S1 </xref> ). The PCR-RE assay shows that gene mutations were induced using all six vectors ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S2 </xref> &#x02013; <xref ref-type="supplementary-material" rid="S1"> S7 </xref> ). The undigested bands from the PCR-RE assay were cloned and sequenced to confirm the mutations. Sequence analysis indicated that the types of mutations differed between the genes ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S2 </xref> &#x02013; <xref ref-type="supplementary-material" rid="S1"> S7 </xref> ). Most <italic> <span class="gene">Glyma06g14180</span> </italic> mutations were single nucleotide insertions, whereas the majority of the detected mutations in <italic> <span class="gene">Glyma08g02290</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> involved multiple-nucleotide deletions. Although rare in the soybean hairy roots, nucleotide substitutions were the major type of mutation induced in soybean protoplasts using the CRISPR-Cas9 system. Mutation efficiencies differed between the pCas9-GmU6-sgRNA and pCas9-AtU6-sgRNA vectors ( <xref ref-type="table" rid="t1"> Table 1 </xref> ), with markedly higher efficiencies obtained with all three genes using the pCas9-GmU6-sgRNA vector. Mutation efficiencies with the pCas9-GmU6-sgRNA vector for <italic> <span class="gene">Glyma06g14180</span> </italic> , <italic> <span class="gene">Glyma08g02290</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> were 14.7, 20.2 and 17.9%, respectively, with corresponding efficiencies of 6.6, 3.2 and 9.7% using the pCas9-AtU6-sgRNA vector.
					</p>
				
					<p>&nbsp;&nbsp;
						Biallelic mutations can be detected in T <sub> 0 </sub> transgenic plants using the CRISPR-Cas9 system <xref ref-type="bibr" rid="b16"> 16 </xref> <xref ref-type="bibr" rid="b19"> 19 </xref> . We detected several biallelic mutations of <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma08g02290</span> </italic> using the PCR-RE assay ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S3 </xref> &#x02013; <xref ref-type="supplementary-material" rid="S1"> S5 </xref> ). A higher frequency of biallelic mutants was observed in <italic> <span class="gene">Glyma08g02290</span> </italic> . Twelve of 19 <italic> <span class="gene">Glyma08g02290</span> </italic> mutants generated using the pCas9-GmU6-sgRNA vector and 2 of 3 <italic> <span class="gene">Glyma08g02290</span> </italic> mutants induced by the pCas9-AtU6-sgRNA vector were biallelic ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Sequencing of several gene clones from independent biallelic mutant roots revealed a variety of mutations per <span class="trait">root</span> ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> ), which suggests that the CRISPR-Cas9 system continued to modify the genes during hairy <span class="trait">root</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						The CRISPR-Cas9 system can tolerate several mismatches between the sgRNA seed and its target, especially in the first 12 nucleotides at the 5&#x02019; end of the sgRNA seed <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b36"> 36 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> , which suggests that off-target <span class="trait">activity</span> is common with the CRISPR-Cas9 system. We accordingly searched the soybean genome for homologs of the three targeted genes in this study. We found that <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma04g40610</span> </italic> had the same target sequence and that the sequences of <italic> <span class="gene">Glyma08g02290</span> </italic> and <italic> <span class="gene">Glyma05g37270</span> </italic> were also identical to one another. <italic> <span class="gene">Glyma12g37050</span> </italic> and <italic> <span class="gene">Glyma09g00490</span> </italic> differed by a single nucleotide at the PAM site (AGG vs. ATG). Mutations in <italic> <span class="gene">Glyma04g40610</span> </italic> and <italic> <span class="gene">Glyma09g00490</span> </italic> induced by the CRISPR-CAS9 system using primers for <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> were detected in protoplasts and hairy roots ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S3 </xref> , <xref ref-type="supplementary-material" rid="S1"> S6 </xref> and <xref ref-type="supplementary-material" rid="S1"> S7 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Biallelic mutations can be detected and their phenotypes observed in T <sub> 0 </sub> transgenic plants using the CRISPR-Cas9 system <xref ref-type="bibr" rid="b16"> 16 </xref> <xref ref-type="bibr" rid="b19"> 19 </xref> . In a study by Ron <italic> et al. </italic> <xref ref-type="bibr" rid="b39"> 39 </xref> , the CRISPR-Cas9 system mediated by <italic> A. rhizogenes </italic> was used to produce a targeted mutation in the <italic> SHORT-ROOT </italic> ( <italic> SHR </italic> ) gene in tomato transgenic hairy roots. The phenotype of the resulting mutant was consistent with Arabidopsis <italic> shr </italic> mutants. In our study, biallelic mutations in <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma08g02290</span> </italic> were detected in transgenic hairy roots ( <xref ref-type="fig" rid="f5"> Fig. 5 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S3 </xref> - <xref ref-type="supplementary-material" rid="S1"> S5 </xref> ). <italic> <span class="gene">Glyma08g02290</span> </italic> had a higher number of biallelic mutations, with 12 of 19 <span class="trait">root</span> samples showing mutations ( <xref ref-type="table" rid="t1"> Table 1 </xref> ). Biallelic mutants can be detected easily using the PCR-RE assay ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S3 </xref> - <xref ref-type="supplementary-material" rid="S1"> S5 </xref> ). Biallelic mutants are the ideal materials for researching gene function. Compared to the inefficient and time-consuming transformation mediated by <italic> Agrobacterium tumefaciens </italic> ( <italic> A. tumefaciens </italic> ), transformation mediated by <italic> A. rhizogenes </italic> is easy, quick and efficient in soybean <xref ref-type="bibr" rid="b34"> 34 </xref> . Transgenic hairy roots can be obtained within one month with transformation efficiencies up to 80%. A large number of genes involved in the roots and nodules have been identified in soybean by next-generation sequencing <xref ref-type="bibr" rid="b40"> 40 </xref> . It would be easy to generate the target gene mutants using the CRISPR-Cas9 system mediated by <italic> A. rhizogenes </italic> , which would lead to advances in soybean <span class="trait">root</span> biology research.
					</p>
				
					<p>&nbsp;&nbsp;
						Off-target <span class="trait">activity</span> is common using the CRISPR-Cas9 system. In our study, we detected off-target gene mutations for <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma12g37050</span> </italic> ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figures S3 </xref> , <xref ref-type="supplementary-material" rid="S1"> S6 </xref> and <xref ref-type="supplementary-material" rid="S1"> S7 </xref> ). Off-target <span class="trait">activity</span> limits the application of the CRISPR-Cas9 system, but several methods are available to reduce this impediment. Decreasing sgRNA-Cas9 concentrations can increase on-target specificity <italic> in vitro </italic> <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> . Off-target <span class="trait">activity</span> can be reduced 50- to 1500-fold using double-nicking mediated by a <italic> Cas9 </italic> nickase mutant ( <italic> Cas9n </italic> ) <xref ref-type="bibr" rid="b41"> 41 </xref> . Use of truncated gRNAs (tru-gRNAs), a shorter sgRNA seed (typically 17 or 18 nucleotides) complementary to the target, can also decrease off-target <span class="trait">activity</span> by 5000-fold or more <xref ref-type="bibr" rid="b42"> 42 </xref> . Although these methods can effectively reduce off-target <span class="trait">activity,</span> the best strategy is identification of gene-specific sgRNA seeds. Fortunately, 97.3% of annotated transcription units (TUs) have specific sgRNA seeds in soybean; these TU-specific sgRNA seeds can be identified by searching the CRISPR-PLANT database ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.genome.arizona.edu/crispr"> http://www.genome.arizona.edu/crispr </ext-link> ) <xref ref-type="bibr" rid="b43"> 43 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Some mutation libraries have been developed by chemical agents and physical treatments in soybean <xref ref-type="bibr" rid="b44"> 44 </xref> <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b46"> 46 </xref> , but the mutants induced by these treatments are random and complex. T-DNA-induced mutagenesis has been widely applied in model plants such as Arabidopsis and rice <xref ref-type="bibr" rid="b47"> 47 </xref> <xref ref-type="bibr" rid="b48"> 48 </xref> . Successful T-DNA insertion mainly depends on efficient of <italic> A. tumefaciens </italic> -mediated transformation. In soybean, the creation of large numbers of mutants using T-DNA insertion is not feasible, as transformation efficiency mediated by <italic> A. tumefaciens </italic> is low in this species. Nevertheless, the acquisition of target mutants is still time-consuming and inefficient because T-DNA-based mutagenesis is random. Off-target <span class="trait">activity</span> can be exploited for the construction of a saturated gene mutation library in soybean. The CRISPR-Cas9 system can tolerate several mismatches between the sgRNA seed and its target, especially in the first 12 nucleotides at the 5&#x02019; end of the sgRNA seed <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b36"> 36 </xref> <xref ref-type="bibr" rid="b49"> 49 </xref> . With respect to these 12 nucleotides, sgRNA seeds having fewer than four mismatches with other sequences in our study were considered to be non-specific sgRNA seeds. A total of 13,103,481 sgRNA seeds were predicted in soybean genes, of which 5,631,730 were specific and 7,469,546 were non-specific ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figure S8 </xref> ). The number of specific sgRNA seeds as well as their coverage (99.5% of soybean genes) is consistent with results obtained by Xie <italic> et al </italic> . <xref ref-type="bibr" rid="b43"> 43 </xref> . The huge quantity of non-specific sgRNA seeds allows the targeting of two or more genes in one transformation in soybean ( <xref ref-type="fig" rid="f6"> Fig. 6a </xref> ). Off-target <span class="trait">activity</span> produces numerous mutations covering different genes in T <sub> 0 </sub> transgenic soybeans. The resulting mutants can be segregated to produce unique mutations in the progeny, which, similar to the application of Ac/Ds transposons or Tnt1 retrotransposons in T-DNA transformations, improves mutation efficiency <xref ref-type="bibr" rid="b50"> 50 </xref> <xref ref-type="bibr" rid="b51"> 51 </xref> . In our study, the seeds of <italic> <span class="gene">Glyma06g14180</span> </italic> or <italic> <span class="gene">Glyma12g37050</span> </italic> were detected to produce two gene mutations ( <italic> <span class="gene">Glyma06g14180</span> </italic> and <italic> <span class="gene">Glyma04g40610</span> </italic> , <italic> <span class="gene">Glyma12g37050</span> </italic> and <italic> <span class="gene">Glyma09g00490</span> </italic> ) respectively in one transgenic plant ( <xref ref-type="supplementary-material" rid="S1"> Figure S3, S6 and S7 </xref> ). By exploiting off-target <span class="trait">activity,</span> the number of transgenic soybean plants required to produce a saturated mutation library can be reduced dramatically ( <xref ref-type="fig" rid="f6"> Fig. 6b </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Agrobacterium rhizogenes transformed soybean roots differ in their nodulation and nitrogen fixation response to genistein and salt stress.</h2>
			<h3>PMID:23430716  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dolatabadian A(1), Modarres Sanavy SA, Ghanati F, Gresshoff PM.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We evaluated response differences of normal and transformed (so-called 'hairy') roots of soybean (Glycine max L. (Merr.), cv L17) to the Nod-factor inducing isoflavone genistein and salinity by quantifying growth, nodulation, nitrogen fixation and biochemical changes. Composite soybean plants were generated using Agrobacterium rhizogenes-mediated transformation of non-nodulating mutant nod139 <span class="gene">(GmNFR5α</span> minus) with complementing A. rhizogenes K599 carrying the wild-type <span class="gene">GmNFR5α</span> gene under control of the constitutive CaMV 35S promoter. We used genetic complementation for nodulation ability as only nodulated roots were scored. After hairy <span class="trait">root</span> emergence, primary roots were removed and composite plants were inoculated with Bradyrhizobium japonicum (strain CB1809) pre-induced with 10 μM genistein and watered with NaCl (0, 25, 50 and 100 mM). There were significant differences between hairy roots and natural roots in their responses to salt stress and genistein application. In addition, there were noticeable nodulation and nitrogen fixation differences. Composite plants had better growth, more <span class="trait">root</span> volume and chlorophyll as well as more nodules and higher nitrogenase <span class="trait">activity</span> (acetylene reduction) compared with natural roots. Decreased lipid peroxidation, proline accumulation and catalase/peroxidase activities were found in 'hairy' roots under salinity stress. Genistein significantly increased nodulation and nitrogen fixation and improved roots and shoot growth. Although genistein alleviated lipid peroxidation under salinity stress, it had no significant effect on the <span class="trait">activity</span> of antioxidant enzymes. In general, composite plants were more competitive in growth, nodulation and nitrogen fixation than normal non-transgenic even under salinity stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of systemic responses in soybean nodulation by xylem sap feeding and complete transcriptome sequencing reveal a novel component of the autoregulation pathway.</h2>
			<h3>PMID:22624681  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reid DE(1), Hayashi S, Lorenc M, Stiller J, Edwards D, Gresshoff PM, Ferguson BJ.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Establishment of the nitrogen-fixing nodulation symbiosis between legumes and rhizobia requires plant-wide reprogramming to allow infection and development of nodules. Nodulation is regulated principally via a mechanism called autoregulation of nodulation (AON). AON is dependent on shoot and <span class="trait">root</span> factors and is maintained by the nodulation autoregulation receptor kinase (NARK) in soybean. We developed a bioassay to detect root-derived signalling molecules in xylem sap of soybean plants which may function in AON. The bioassay involves feeding of xylem extracts via the cut hypocotyl of soybean seedlings and monitoring of molecular markers of AON in the leaf. Transcript abundance changes occurring in the leaf in response to feeding were used to determine the biological <span class="trait">activity</span> of the extracts. To identify transcript abundance changes that occur during AON, which may also be used in the bioassay, we used an RNA-seq-based transcriptomics approach. We identified changes in the leaves of bioassay plants fed with xylem extracts derived from either Bradyrhizobium japonicum-inoculated or uninoculated plants. Differential expression responses were detected for genes involved in jasmonic acid metabolism, pathogenesis and receptor kinase signalling. We identified an inoculation- and NARK-dependent candidate gene <span class="gene">(GmUFD1a)</span> that responds in both the bioassay and intact, inoculated plants. <span class="gene">GmUFD1a</span> is a component of the ubiquitin-dependent protein degradation pathway and provides new insight into the molecular responses occurring during AON. It may now also be used in our feeding bioassay as a molecular marker to assist in identifying the factors contributing to the systemic regulation of nodulation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Disparate pathways for the biogenesis of cytochrome oxidases in Bradyrhizobium japonicum.</h2>
			<h3>PMID:20335176  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bühler D(1), Rossmann R, Landolt S, Balsiger S, Fischer HM, Hennecke H.
				<br/> 
				年份：2010 May 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This work addresses the biogenesis of heme-copper terminal oxidases in Bradyrhizobium japonicum, the nitrogen-fixing <span class="trait">root</span> nodule symbiont of soybean. B. japonicum has four quinol oxidases and four cytochrome oxidases. The latter include the aa(3)- and cbb(3)-type oxidases. Although both have a Cu(B) center in subunit I, the subunit II proteins differ in having either a Cu(A) center (in aa(3)) or a covalently bound heme c (in cbb(3)). Two biogenesis factors were genetically studied here, the periplasmically exposed CoxG and ScoI proteins, which are the respective homologs of the mitochondrial copper-trafficking chaperones Cox11 and Sco1 for the formation of the Cu(B) center in subunit I and the Cu(A) center in subunit II of cytochrome aa(3). We could demonstrate copper binding to ScoI in vitro, a process for which the thiols of cysteine residues 74 and 78 in the ScoI polypeptide were shown to be essential. Knock-out mutations in the B. japonicum <span class="gene">coxG</span> and scoI genes led to loss of cytochrome aa(3) assembly and <span class="trait">activity</span> in the cytoplasmic membrane, whereas the cbb(3)-type cytochrome oxidase apparently remained unaffected. This suggests that subunit I of the cbb(3)-type oxidase obtains its copper cofactor via a different pathway than cytochrome aa(3). In contrast to the <span class="gene">coxG</span> mutation, the scoI mutation caused a decreased symbiotic nitrogen fixation <span class="trait">activity.</span> We hypothesize that a periplasmic B. japonicum protein other than any of the identified Cu(A) proteins depends on ScoI and is required for an effective symbiosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Integrated roles of BclA and DD-carboxypeptidase 1 in <italic> Bradyrhizobium </italic> differentiation within NCR-producing and NCR-lacking <span class="trait">root</span> nodules</h2>
			<h3>PMID:28831061</h3>
				
					<h3>PMCID:5567381</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Barri&#x000e8;re </surname>  <given-names> Quentin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Guefrachi </surname>  <given-names> Ibtissem </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gully </surname>  <given-names> Djamel </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lamouche </surname>  <given-names> Florian </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pierre </surname>  <given-names> Olivier </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fardoux </surname>  <given-names> Jo&#x000eb;l </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chaintreuil </surname>  <given-names> Cl&#x000e9;mence </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Alunni </surname>  <given-names> Beno&#x000ee;t </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Timchenko </surname>  <given-names> Tatiana </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Giraud </surname>  <given-names> Eric </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Mergaert </surname>  <given-names> Peter </given-names>  </name>  <address>  <email> peter.mergaert@i2bc.paris-saclay.fr </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.457334.2 </institution-id>  <institution> Institute for Integrative Biology of the Cell,  </institution>  <institution> UMR9198, CNRS, Universit&#x000e9; Paris-Sud, CEA,  </institution>  </institution-wrap> Gif-sur-Yvette, France  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.442508.f </institution-id>  <institution> Research Unit Biodiversity &#x00026; Valorization of Arid Areas Bioressources (BVBAA),  </institution>  <institution> Faculty of Sciences, Gab&#x000e8;s University, Erriadh-Zrig,  </institution>  </institution-wrap> 6072 Gab&#x000e8;s, Tunisia  </aff>  <aff id="Aff3">  <label> 3 </label> Laboratoire des Symbioses Tropicales et M&#x000e9;diterran&#x000e9;ennes, Institut pour la Recherche et le D&#x000e9;veloppement, UMR IRD/SupAgro/INRA/UM2/CIRAD, Campus International de Baillarguet, TA A-82/J, 34398 Montpellier Cedex 5, France  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2289 818X </institution-id>  <institution-id institution-id-type="GRID"> grid.5571.6 </institution-id>  <institution/>  <institution> Present Address: Universit&#x000e9; de Pau et des Pays de l&#x02019;Adour,  </institution>  </institution-wrap> Pau, France  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0385 8766 </institution-id>  <institution-id institution-id-type="GRID"> grid.435437.2 </institution-id>  <institution/>  <institution> Present Address: Institut Sophia AgroBiotech,  </institution>  </institution-wrap> Sophia-Antipolis, France  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 7 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The USDA110 <italic> bclA </italic> gene confers <span class="trait">resistance</span> to antimicrobial NCR peptides and mediates the uptake of FITC-NCR247. ( <bold> a </bold> ) <italic> S. meliloti </italic> strain Sm1021&#x00394; <italic> <span class="gene">bacA</span> </italic> derivatives expressing no <italic> <span class="gene">bacA</span> </italic> -related gene (empty vector) or the USDA110 <italic> bclA </italic> gene were incubated with NCR247, NCR335 or NCR035 or with water (control) and the surviving bacteria were counted and expressed as % from the control treatment. Error bars in all panels are standard deviations. ( <bold> b </bold> ) FITC-NCR247 uptake by <italic> S. meliloti </italic> strain Sm1021&#x00394; <italic> <span class="gene">bacA</span> </italic> or its derivative expressing the USDA110 <italic> bclA </italic> gene was measured by flow cytometry in the presence of trypan blue to quench extracellular fluorescence. FITC-positive cells are marked with a green box. ( <bold> c </bold> ) FITC is not taken up by <italic> S. meliloti </italic> derivatives. FITC-treated bacteria were analyzed by flow cytometry in the presence of trypan blue to quench extracellular fluorescence. No FITC-positive cells were detected.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">rbcS</span> SRS4 promoter from Glycine max and its expression <span class="trait">activity</span> in transgenic tobacco.</h2>
			<h3>PMID:26214418  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cui XY(1), Chen ZY(2), Wu L(1), Liu XQ(1), Dong YY(3), Wang FW(3), Li HY(3).
				<br/> 
				年份：2015 Jul 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The regulatory region of the ribulose-1,5-bisphosphate carboxylase small subunit gene SRS4 from soybean (Glycine max) was cloned using TAIL-PCR and general PCR, and named the <span class="gene">rbcS</span> promoter. The promoter was fused with the GUS gene and introduced into Nicotiana tabacum via Agrobacterium-mediated leaf disk transformation. In 4-week-old transgenic tobacco plants, the highest GUS expression levels were observed in the leaves, GUS <span class="trait">activity</span> was 7.13- and 7.40-fold higher in leaves than in stems and roots, respectively. Moreover, GUS <span class="trait">activity</span> was stimulated by light. In conclusion, spatial and light regulation of the soybean <span class="gene">rbcS</span> promoter was observed in N. tabacum, thus illustrating a leaf-specific and light-induced promoter.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Four allantoinase genes are expressed in nitrogen-fixing soybean.</h2>
			<h3>PMID:22476036  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Duran VA(1), Todd CD.
				<br/> 
				年份：2012 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max L. [Merr]) plants export nitrogen from the nodules as ureides during symbiotic biological nitrogen fixation. Ureides also play a role as nitrogen storage compounds in the seeds and are broken down in germinating seedlings. In this work we identified four soybean genes encoding allantoinase (E.C. 3.5.2.5), an enzyme involved in both ureide production in nodules and ureide catabolism in leaves and other sink tissues. We examined ureide content, allantoinase enzyme <span class="trait">activity</span> and expression of these genes, which we term <span class="gene">GmALN1</span> through <span class="gene">GmALN4,</span> in germinating seedlings and in vegetative tissues from 45 day old soybean plants. <span class="gene">GmALN1</span> and <span class="gene">GmALN2</span> transcripts were measured in all tissues, but similar levels of expression of <span class="gene">GmALN3</span> and <span class="gene">GmALN4</span> was only observed in nodules. The soybean allantoinase genes seem to have arisen through tandem gene duplication followed by a whole genome duplication. We looked for evidence of the tandem duplication in common bean (Phaseolus vulgaris L.) and present evidence that it occured sometime in the bean lineage before these two species diverged, but before soybean became a tetraploid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of soybean coumestrol on Bradyrhizobium japonicum nodulation ability, biofilm formation, and transcriptional profile.</h2>
			<h3>PMID:22307307  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lee HI(1), Lee JH, Park KH, Sangurdekar D, Chang WS.
				<br/> 
				年份：2012 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flavonoids, secondary plant metabolites which mainly have a polyphenolic structure, play an important role in plant-microbe communications for nitrogen-fixing symbiosis. Among 10 polyphenolic compounds isolated from soybean roots in our previous study, coumestrol showed the highest antioxidant <span class="trait">activity.</span> In this study, its effect on the soybean nodulation was tested. The soybean symbiont Bradyrhizobium japonicum USDA110 pretreated with 20 μM coumestrol enhanced soybean nodulation by increasing the number of nodules 1.7-fold compared to the control. We also tested the effect of coumestrol on B. japonicum biofilm formation. At a concentration of 2 μM, coumestrol caused a higher degree of biofilm formation than two major soybean isoflavonoids, genistein and daidzein, although no biofilm formation was observed at a concentration of 20 μM each compound. A genome-wide transcriptional analysis was performed to obtain a comprehensive snapshot of the B. japonicum response to coumestrol. When the bacterium was incubated in 20 μM coumestrol for 24 h, a total of 371 genes (139 upregulated and 232 downregulated) were differentially expressed at a 2-fold cutoff with a q value of less than 5%. No common nod gene induction was found in the microarray data. However, quantitative reverse transcription-PCR (qRT-PCR) data showed that incubation for 12 h resulted in a moderate induction (ca. 2-fold) of <span class="gene">nodD1</span> and nodABC, indicating that soybean coumestrol is a weak inducer of common nod genes. In addition, disruption of <span class="gene">nfeD</span> (bll4952) affected the soybean nodulation by an approximate 30% reduction in the average number of nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A mutant Bradyrhizobium japonicum delta-aminolevulinic acid dehydratase with an altered metal requirement functions in situ for tetrapyrrole synthesis in soybean <span class="trait">root</span> nodules.</h2>
			<h3>PMID:7649992  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chauhan S(1), O'Brian MR.
				<br/> 
				年份：1995 Aug 25
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The tetrapyrrole synthesis enzyme delta-aminolevulinic acid (ALA) dehydratase requires Mg2+ for catalytic <span class="trait">activity</span> in photosynthetic organisms and in Bradyrhizobium japonicum, a bacterium that can reside symbiotically within plant cells of soybean <span class="trait">root</span> nodules or as a free-living organism. ALA dehydratase from animals and other non-photosynthetic organisms is a Zn(2+)-dependent enzyme. A modified B. japonicum ALA dehydratase, ALAD*, was constructed by site-directed mutagenesis of <span class="gene">hemB</span> in which three proximal amino acids conserved in plant dehydratases were changed to cysteine residues as is found in the Zn(2+)-dependent enzyme of animals. These substitutions resulted in an enzyme that required Zn2+ rather than Mg2+ for catalytic <span class="trait">activity,</span> and therefore a region of the ALA dehydratase from B. japonicum, and probably from plants, was identified that is involved in Mg2+ dependence. In addition, the data show that a change in only a few residues is sufficient to change a Mg(2+)-dependent ALA dehydratase to a Zn(2+)-dependent one. B. japonicum strains were constructed that contained a single copy of either <span class="gene">hemB</span> or the altered gene hemB* integrated into the genome of a hemB- mutant. Cultures of the hemB* strain KPZn3 had Zn(2+)-dependent ALA dehydratase <span class="trait">activity</span> that functioned in vivo as discerned by its heme prototrophy and expression of wild type levels of cellular hemes. Strain KPZn3 elicited <span class="trait">root</span> nodules on soybean that contained viable bacteria and exhibited traits of normally developed nodules, and the symbiotic bacteria expressed nearly wild type levels of cellular hemes. We conclude that the Zn(2+)-dependent ALAD* can function and support bacterial tetrapyrrole synthesis within the plant milieu of <span class="trait">root</span> nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The symbiotic nitrogen fixation regulatory operon (fixRnifA) of Bradyrhizobium japonicum is expressed aerobically and is subject to a novel, nifA-independent type of activation.</h2>
			<h3>PMID:3313281  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Thöny B(1), Fischer HM, Anthamatten D, Bruderer T, Hennecke H.
				<br/> 
				年份：1987 Oct 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Bradyrhizobium japonicum N2 fixation regulatory gene, nifA, was sequenced and its transcription start site determined. Between the start of transcription and the <span class="gene">nifA</span> gene an open reading frame of 278 codons was found and named fixR. A deletion in <span class="gene">fixR</span> which allowed transcription into <span class="gene">nifA</span> resulted in a 50% reduced Fix <span class="trait">activity.</span> The fixRnifA operon was expressed in soybean <span class="trait">root</span> nodules, in cultures grown anaerobically with nitrate as terminal electron acceptor, in microaerobic cultures, and in aerobic cultures. The transcription start site (+1) was preceded by a characteristic nif(-24/-12)-type promoter consensus sequence. Double base-pair exchanges in the -12 but not in the -24 region resulted in a 'promoter-down' phenotype. A promoter-upstream DNA region between -50 and -148 was essential for maximal promoter <span class="trait">activity.</span> Expression from the promoter was not dependent on nifA. We conclude that the fixRnifA promoter is positively controlled, and that it requires a newly postulated transcriptional factor in order to become activated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of nodule-specific glutamine synthetase genes during nodule development in soybeans.</h2>
			<h3>PMID:24302304</h3>
				

				
			<p>
				作者：Sengupta-Gopalan C(1), Pitas JW.
				<br/> 
				年份：1986 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A cDNA clone (pcPvNGS-01) to glutamine synthetase (GS) mRNA from <span class="trait">root</span> nodules of Phaseolus vulgaris showed cross-hybridization to <span class="gene">GS</span> and mRNA from soybean <span class="trait">root</span> nodules, thus allowing its use as a probe to study the expression of <span class="gene">GS</span> genes during <span class="trait">root</span> nodule development in soybeans. Hybrid-select translation of <span class="trait">root</span> and nodule RNA of soybean with DNA from pcPvNGS-01, followed by 2D gel electrophoresis, showed six peptides in the <span class="trait">root</span> and an additional four peptides in the nodule which represent nodule-specific glutamine synthetase (GSn) gene products. The GSn gene products appeared for the first time between day 11 and 12 after infection, either concomitant with the onset of nitrogenase <span class="trait">activity</span> or immediately following it. The levels of expression of the GSn and leghemoglobin genes were not affected in young Fix(-) nodules formed by Bradyrhizobium japonicum strains that are defective in nitrogenase <span class="trait">activity,</span> suggesting that the induction of these two sets of host genes take place independent of nitrogenase <span class="trait">activity.</span> However, in Fix(-) nodules that are incapable of maintaining the peribacteroid membrane, GSn gene products were not detected while 1ba, 1bc2 and 1bc3 appeared. In both the timing of appearance during <span class="trait">root</span> nodule development and the effect of different bacterial mutations on the expression, GSn genes differ from most other nodulin genes examined (30), suggesting different regulatory mechanisms.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect on epidermal cell of soybean protein-degraded products and structural determination of the <span class="trait">root</span> hair promoting peptide.</h2>
			<h3>PMID:17671783  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matsumiya Y(1), Sumiyoshi S, Matsukura T, Kubo M.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peptide(s) produced from degraded soybean protein by an alkaline protease from Bacillus circulans HA12 (degraded soybean-meal products; DSP) increased the number of both the <span class="trait">root</span> hair cells (trichoblasts) and hairless cells (atrichoblasts) of Brassica rapa by about 4.4 times and 1.9 times, respectively. To identify the <span class="trait">root</span> hair-promoting peptide(s) in DSP, the origin protein of the <span class="trait">root</span> hair-promoting peptide(s) was identified as Kunitz trypsin inhibitor (KTI). The <span class="trait">root</span> hair-promoting peptide in the degraded products of <span class="gene">KTI</span> was purified and produced a signal of 1,198.2 Da with matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) analysis. A search of the amino acid sequence of <span class="gene">KTI</span> located the peptide GGIRAAPTGNER, which had a molecular weight identical to 1,198.2 Da. The peptide GGIRAAPTGNER was chemically synthesized, and the synthetic peptide possessed <span class="trait">root</span> hair-promoting <span class="trait">activity.</span> Thus, it is concluded that this peptide in DSP is the foreign bioactive peptide promoting the differentiation of <span class="trait">root</span> hairs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improved soybean <span class="trait">root</span> association of N-starved Bradyrhizobium japonicum.</h2>
			<h3>PMID:11717284  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：López-García SL(1), Vázquez TE, Favelukes G, Lodeiro AR.
				<br/> 
				年份：2001 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, we addressed the effects of N limitation in Bradyrhizobium japonicum for its association with soybean roots. The wild-type strain LP 3001 grew for six generations with a growth rate of 1.2 day(-1) in a minimal medium with 28 mM mannitol as the carbon source and with the N source [(NH(4))(2)SO(4)] limited to only 20 microM. Under these conditions, the glutamine synthetase (GS) <span class="trait">activity</span> was five to six times higher than in similar cultures grown with 1 or 0.1 mM (NH(4))(2)SO(4). The NtrBC-inducible GSII form of this enzyme accounted for 60% of the specific <span class="trait">activity</span> in N-starved rhizobia, being negligible in the other two cultures. The exopolysaccharide (EPS) and capsular polysaccharide (CPS) contents relative to cell protein were significantly higher in the N-starved cultures, but on the other hand, the poly-3-hydroxybutyrate level did not rise in comparison with N-sufficient cultures. In agreement with the accumulation of CPS in N-starved cultures, soybean lectin (SBL) binding as well as stimulation of rhizobial adsorption to soybean roots by SBL pretreatment were higher. The last effect was evident only in cultures that had not entered stationary phase. We also studied <span class="gene">nodC</span> gene induction in relation to N starvation. In the chromosomal nodC::lacZ fusion Bj110-573, <span class="gene">nodC</span> gene expression was induced by genistein 2.7-fold more in N-starved young cultures than in nonstarved ones. In stationary-phase cultures, <span class="gene">nodC</span> gene expression was similarly induced in N-limited cultures, but induction was negligible in cultures limited by another nutrient. Nodulation profiles obtained with strain LP 3001 grown under N starvation indicated that these cultures nodulated faster. In addition, as culture age increased, the nodulation efficiency decreased for two reasons: fewer nodules were formed, and nodulation was delayed. However, their relative importance was different according to the nutrient condition: in older cultures the overall decrease in the number of nodules was the main effect in N-starved cultures, whereas a delay in nodulation was more responsible for a loss in efficiency of N-sufficient cultures. Competition for nodulation was studied with young cultures of two wild-type strains differing only in their antibiotic <span class="trait">resistance,</span> the N-starved cultures being the most competitive.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phosphoenolpyruvate carboxylase protein kinase from soybean <span class="trait">root</span> nodules: partial purification, characterization, and up/down-regulation by photosynthate supply from the shoots.</h2>
			<h3>PMID:9224739  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang XQ(1), Chollet R.
				<br/> 
				年份：1997 Jul 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphoenolpyruvate carboxylase (PEPC) kinase was partially purified about 3000-fold from soybean <span class="trait">root</span> nodules by a fast-protein liquid chromatography protocol. This protein-serine kinase has an apparent native molecular mass of about 30,000 as estimated by size-exclusion chromatography. Following electrophoresis of this partially purified PEPC-kinase preparation in a denaturing gel containing dephospho maize leaf PEPC as substrate, the in situ renaturation and assay of protein kinase <span class="trait">activity</span> revealed two, PEPC-dependent kinase polypeptides with molecular masses of about 32 and 37 kDa. The approximately 32-kDa polypeptide was significantly more active than the approximately 37-kDa catalytic subunit. The <span class="trait">activity</span> of this partially purified PEPC kinase, and a less purified sample, was Ca2+-insensitive. This protein kinase preparation was able to phosphorylate purified PEPCs from soybean nodules, maize leaves, and a sorghum recombinant <span class="gene">C4</span> PEPC. In contrast, this PEPC kinase was unable to phosphorylate a phosphorylation-site mutant form of sorghum <span class="gene">C4</span> PEPC (S8Y), two other soybean nodule phosphoproteins [nodulin-26 and nodulin-100 (sucrose synthase)], bovine serum albumin, and histone III-S. Following in vitro phosphorylation of purified dephospho soybean nodule PEPC from stem-girdled plants by the partially purified nodule PEPC kinase, the former's <span class="trait">activity</span> and sensitivity to L-malate inhibition increased and decreased, respectively. Notably, the Ca2+-independent PEPC kinase <span class="trait">activity</span> in nodules from illuminated plants was markedly greater than that in nodules harvested from plants subjected to stem girdling or prolonged darkness. Furthermore, the kinase <span class="trait">activity</span> in nodules was controlled reversibly by illumination and extended darkness pretreatments of the parent plants, suggesting that photosynthate supply from the shoots is likely responsible for these striking changes in PEPC kinase <span class="trait">activity</span> observed in planta in the legume nodule.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genes for a microaerobically induced oxidase complex in Bradyrhizobium japonicum are essential for a nitrogen-fixing endosymbiosis.</h2>
			<h3>PMID:8386371  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preisig O(1), Anthamatten D, Hennecke H.
				<br/> 
				年份：1993 Apr 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We report the discovery of a Bradyrhizobium japonicum gene cluster (fixNOQP) in which mutations resulted in defective soybean root-nodule bacteroid development and symbiotic nitrogen fixation. The predicted, DNA-derived protein sequences suggested that FixN is a heme b and copper-binding oxidase subunit, FixO a monoheme cytochrome c, FixQ a polypeptide of 54 amino acids, and FixP a diheme cytochrome c and that they are all membrane-bound. The isolation and analysis of membrane proteins from B. japonicum wild-type and mutant cells revealed two c-type cytochromes of 28 and 32 kDa as the likely products of the <span class="gene">fixO</span> and <span class="gene">fixP</span> genes and showed that both were synthesized only under oxygen-limited growth conditions. Furthermore, <span class="gene">fixN</span> insertion and fixNO deletion mutants grown microaerobically or anaerobically (with nitrate) exhibited a strong decrease in whole-cell oxidase <span class="trait">activity</span> as compared with the wild type. The data suggest that the fixNOQP gene products are induced at low oxygen concentrations and constitute a member of the bacterial heme/copper cytochrome oxidase superfamily. The described features are compatible with the postulate that this oxidase complex is specifically required to support bacterial respiration in endosymbiosis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combined Effects of Elevated O <sub> 3 </sub> Concentrations and Enhanced UV-B Radiation of the Biometric and Biochemical Properties of Soybean Roots</h2>
			<h3>PMID:28955360</h3>
				
					<h3>PMCID:5600998</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mao </surname>  <given-names> Bing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tian-Hong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/380144/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Rong-Rong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Jia-Shu </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/442010/overview"/>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Catalase (CAT; EC 1.11.1.6) <span class="trait">activity</span> was extracted by homogenizing 500 mg frozen roots at 4&#x000b0;C in 100 mol L <sup> -1 </sup> cold phosphate buffer (pH 7.2) containing 0.5% Triton-X. <span class="trait">Activity</span> of CAT was assessed in 100 &#x003bc;L phosphate buffer, 400 &#x003bc;L 200 mmol L <sup> -1 </sup> H <sub> 2 </sub> O <sub> 2 </sub> and 100 &#x003bc;L enzyme extract. A decrease in H <sub> 2 </sub> O <sub> 2 </sub> was measured at 240 nm, and <span class="trait">activity</span> of CAT was measured according to the method of <xref rid="B1" ref-type="bibr"> Aebi (1984) </xref> . Phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) <span class="trait">activity</span> was determined by the method of <xref rid="B33" ref-type="bibr"> Havir and Hanson (1968) </xref> and <xref rid="B15" ref-type="bibr"> Bradford (1976) </xref> . The calculation was based on the extinction coefficient (9500 M <sup> -1 </sup> cm <sup> -1 </sup> ) for <italic> trans </italic> -cinnamic acid. One unit of <span class="trait">activity</span> for <span class="gene">PAL</span> was defined as the amount of enzyme which caused the formation of 1 &#x003bc;mol <italic> trans </italic> -cinnamic acid per hour. Lipoxygenase (LOX) was determined by the method of <xref rid="B18" ref-type="bibr"> Buranasompob et al. (2007) </xref> . 500 mg frozen roots were used for the extraction of <span class="gene">LOX</span> in phosphate buffer (pH 7.5). After added solid ammonium sulfate and centrifugation, the supernatant was used to determine APX <span class="trait">activity.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Elevated O <sub> 3 </sub> , enhanced UV-B radiation and O <sub> 3 </sub> + UV-B significantly inhibited SOD <span class="trait">activity</span> at flowering stage, and POD <span class="trait">activity</span> at podding stage, and APX <span class="trait">activity</span> during the whole period of soybean growth, and CAT <span class="trait">activity</span> at podding stage ( <bold> Table <xref ref-type="table" rid="T3"> 3 </xref> </bold> ). Concerning <span class="gene">PAL</span> <span class="trait">activity,</span> elevated O <sub> 3 </sub> showed significant negative effects, and enhanced UV-B radiation showed significant positive effects during the whole period of soybean growth. Elevated O <sub> 3 </sub> , enhanced UV-B radiation and O <sub> 3 </sub> + UV-B showed significant positive effects on <span class="gene">LOX</span> <span class="trait">activity</span> during the whole period of soybean growth.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of flower development related gene GsLFY from Glycine soja.</h2>
			<h3>PMID:28115306  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo WY(1), Cui YM(1), Wang TT(1), Yu DY(1), Huang F(1).
				<br/> 
				年份：2017 Jan 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						LEAFY/FLORICAULA (LFY/FLO) is a family of plant-specific transcription factors, which plays an important role(s) in the regulation of floral organ formation and development. So far, <span class="gene">LFY</span> regulation on floral development in wild soybean has not been reported in the literature. In this study, the <span class="gene">LFY</span> gene, GsLFY, has been isolated from Glycine soja, and characterized with molecular and transgenic techniques. The cDNA for GsLFY gene is 1224 bp in length and contains an open reading frame encoding a polypeptide of 407 amino acids. Quantitative RT-PCR analysis shows that GsLFY is prominently expressed in various tissues, including roots, flowers and seeds. Among the four floral organs, GsLFY is expressed highly in sepals and stamens while weakly in the petals and carpels. Yeast two-hybrid experiments show that GsLFY possesses transactivation <span class="trait">activity</span> while transient expression analysis with Arabidopsis thaliana protoplasts shows that GsLFY protein is localized in the nucleus, supporting the notion that GsLFY is a transcription factor. The GsLFY transgenic tobacco plants flower about 29 days earlier than the wild-type plants, thereby providing a potential rationale for developing new soybean varieties with altered flowering time.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two <span class="gene">MATE</span> proteins play a role in iron efficiency in soybean.</h2>
			<h3>PMID:19342121  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rogers EE(1), Wu X, Stacey G, Nguyen HT.
				<br/> 
				年份：2009 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Iron is a necessary but often limiting nutrient for plant growth and development. Soybeans grown on the high-pH calcareous soils are especially prone to developing iron deficiency <span class="trait">chlorosis</span> and suffering the resultant <span class="trait">yield</span> losses. Once iron is transported into the <span class="trait">root,</span> it must be translocated from the <span class="trait">root</span> to the shoot where it is needed for photosynthesis and other processes. Previous work has indicated that iron is likely to move through the xylem as ferric citrate. In Arabidopsis thaliana, citrate is effluxed into the xylem by the ferric reductase defective3 (FRD3) protein. Here, we present the identification and characterization of two soybean genes, <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b,</span> with similar sequence and function to AtFRD3. The expression of both <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b</span> is induced by iron deficiency in the iron-efficient reference cultivar Williams 82. <span class="gene">GmFRD3b,</span> but not <span class="gene">GmFRD3a,</span> is expressed at higher levels in the iron-efficient cultivar Clark than in its iron-inefficient near isogenic line iso-Clark (iso), likely accounting for the higher xylem citrate levels in Clark. Increased xylem citrate levels lead to increased solubility of ferric iron in Clark xylem exudate as compared to iso-Clark exudate. These results support the hypothesis that high xylem citrate levels are needed for efficient <span class="trait">root</span> to shoot translocation of iron. Along with efficient ferric chelate reductase <span class="trait">activity</span> and <span class="trait">root</span> iron uptake <span class="trait">activity,</span> high expression levels of FRD3 genes are also proposed as a target for future iron efficiency breeding projects.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Generation and analysis of soybean plastid transformants expressing Bacillus thuringiensis Cry1Ab protoxin.</h2>
			<h3>PMID:16158241  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dufourmantel N(1), Tissot G, Goutorbe F, Garçon F, Muhr C, Jansens S, Pelissier B, Peltier G, Dubald M.
				<br/> 
				年份：2005 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We describe the generation of fertile and homoplasmic soybean plastid transformants, expressing the Bacillus thuringiensis insecticidal protoxin Cry1Ab. Transgenes were targeted in the intergenic region of Glycine max plastome, between the rps12/7 and trnV genes and selection was carried out using the <span class="gene">aadA</span> gene encoding spectinomycin <span class="trait">resistance.</span> Molecular analysis confirmed the integration of the cry1Ab and <span class="gene">aadA</span> expression cassettes at the expected location in the soybean plastome, and the transmission of the transgenes to the next generation. Western blot analyses showed that the Cry1Ab protoxin is highly expressed in leaves, stems and seeds, but not in roots. Its expression confers strong insecticidal <span class="trait">activity</span> to the generated transgenic soybean, as exemplified with velvetbean caterpillar (Anticarsia gemmatalis).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induction of Glutamine Synthetase <span class="trait">Activity</span> in Nonnodulated Roots of Glycine max, Phaseolus vulgaris, and Pisum sativum.</h2>
			<h3>PMID:16652993</h3>
				

				
			<p>
				作者：Hoelzle I(1), Finer JJ, McMullen MD, Streeter JG.
				<br/> 
				年份：1992 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrate or ammonium fertilization significantly increased glutamine synthetase (GS) <span class="trait">activity</span> in nonnodulated roots of French bean (Phaseolus vulgaris), soybean (Glycine max), and pea (Pisum sativum). Western analysis revealed substantial <span class="gene">GS</span> antibody-positive protein in <span class="trait">root</span> extracts that had minimal <span class="gene">GS</span> <span class="trait">activity,</span> indicating that an inactive form of <span class="gene">GS</span> may be present in nonfertilized plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of constitutive iron reductase ( <italic> AtFRO2 </italic> ) expression on mineral accumulation and distribution in soybean ( <italic> Glycine max. </italic> L)</h2>
			<h3>PMID:24765096</h3>
				
					<h3>PMCID:3982063</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vasconcelos </surname>  <given-names> Marta W. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/74696"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Clemente </surname>  <given-names> Thomas E. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/125499"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grusak </surname>  <given-names> Michael A. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/55795"/>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						RNA from roots, leaves, seeds, <span class="trait">root</span> tips, pod walls, and petals of control and transgenic plants grown hydroponically under various iron concentrations (see above) was isolated using the RNA-easy kit (Qiagen Inc., Valencia, CA, USA) according to the manufacturer&#x02019;s instructions. RNA was isolated from 3 plants in each treatment. The experiments were repeated twice, and PCR reactions were performed at least in triplicate. Possible contaminating genomic DNA was removed with the TURBO DNA- <italic> free </italic> <sup> TM </sup> kit from Ambion (Ambion Inc., Austin, TX, USA) following the manufacturer&#x02019;s instructions. Total RNA (0.5 &#x003bc;g) were subjected to reverse transcription (RT) with an anchored oligo (dT) primer and 200 units superscript II reverse transcriptase (Invitrogen, Carlsbad, CA, USA) in a volume of 20 &#x003bc;l according to the manufacturer&#x02019;s instructions. PCR reactions were carried out with 1.5 &#x003bc;l of the RT reaction solutions. Additional reaction components were: 10 mM polymerase buffer, 1 mM dNTP&#x02019;s, 0.1 units Taq polymerase (Clontech, Palo Alto, CA, USA) and 1 &#x003bc;M specific primers. All primers surrounded an intron so that genomic DNA was clearly distinguished from cDNA-derived products. The following primer sets were designed: <italic> ferritin </italic> ( <italic> <span class="gene">GmFer</span> </italic> ), 5&#x02032;-ACTTGCTCTGTTTCTCTGAGC-3&#x02032; (forward), 5&#x02032;&#x02013;CGCTAGACGGTGTGACACGT-3&#x02032; (reverse); <italic> ubiquitin </italic> ( <italic> <span class="gene">GmUbq</span> </italic> ), 5&#x02032;-GGGTTTTAAGCTCGTTGT-3&#x02032; (forward) and 5&#x02032;-GGACACATTGAGTTCAAC-3&#x02032; (reverse). The number of cycles in each PCR reaction was 28, with 58&#x000b0;C annealing temperature. Amplified products from 10 &#x003bc;l of PCR reaction were visualized on a 1% TAE agarose gel containing ethidium bromide. Bands were photographed using the Quantity One 4.5.1 Chemidoc EQ <sup> TM </sup> Software System (Biorad, CA, USA).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A toolbox of genes, proteins, metabolites and promoters for improving <span class="trait">drought</span> tolerance in soybean includes the metabolite coumestrol and stomatal development genes</h2>
			<h3>PMID:26861168</h3>
				

				
					<h3>PMCID:4746818</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Tripathi </surname>  <given-names> Prateek </given-names>  </name>  <address>  <email> tprateek@dornsife.usc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff6"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Rabara </surname>  <given-names> Roel C. </given-names>  </name>  <address>  <email> roel.rabara@tamu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff7"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reese </surname>  <given-names> R. Neil </given-names>  </name>  <address>  <email> Neil.Reese@sdstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Miller </surname>  <given-names> Marissa A. </given-names>  </name>  <address>  <email> Marissa.Miller@ag.tamu.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rohila </surname>  <given-names> Jai S. </given-names>  </name>  <address>  <email> jai.rohila@sdstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Subramanian </surname>  <given-names> Senthil </given-names>  </name>  <address>  <email> senthil.subramanian@sdstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Qingxi J. </given-names>  </name>  <address>  <email> jeffery.shen@unlv.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Morandi </surname>  <given-names> Dominique </given-names>  </name>  <address>  <email> dmorandi@dijon.inra.fr </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000fc;cking </surname>  <given-names> Heike </given-names>  </name>  <address>  <email> Heike.Bucking@sdstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shulaev </surname>  <given-names> Vladimir </given-names>  </name>  <address>  <email> Vladimir.Shulaev@unt.edu </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Rushton </surname>  <given-names> Paul J. </given-names>  </name>  <address>  <phone> +1 972 952 9246 </phone>  <email> wrkypaul1@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <aff id="Aff1">  <label/> Department of Biology and Microbiology, South Dakota State University, Brookings, SD57007 USA  </aff>  <aff id="Aff2">  <label/> Texas A&#x00026;M AgriLife Research and Extension Center, Dallas, TX 75252 USA  </aff>  <aff id="Aff3">  <label/> School of Life Sciences, University of Nevada, Las Vegas, 89154 USA  </aff>  <aff id="Aff4">  <label/> INRA, UMR 1347 Agro&#x000e9;cologie, 17 rue Sully, BP 86510, 21065 Dijon, CEDEX France  </aff>  <aff id="Aff5">  <label/> Department of Biological Sciences, University of North Texas, Denton, TX 76203 USA  </aff>  <aff id="Aff6">  <label/> Current address, Molecular and Computational Biology, Dana &#x00026; David Dornsife College of Letters, Arts and Sciences, University of Southern California, Los Angeles, CA 90089 USA  </aff>  <aff id="Aff7">  <label/> Current address: Texas A&#x00026;M AgriLife Research and Extension Center, Dallas, TX 75252 USA  </aff>  <aff id="Aff8">  <label/> Current address, 22nd Century Group Inc., 9530 Main Street Clarence, New York, 14031 USA  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 1 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The purpose of this project was to identify metabolites, proteins, genes, and promoters associated with water stress responses in soybean. A number of these may serve as new targets for the biotechnological improvement of <span class="trait">drought</span> responses in soybean (Glycine max). RESULTS: We identified metabolites, proteins, and genes that are strongly up or down regulated during rapid water stress following removal from a hydroponics system. 163 metabolites showed significant changes during water stress in roots and 93 in leaves. The largest change was a root-specific 160-fold increase in the coumestan coumestrol making it a potential biomarker for <span class="trait">drought</span> and a promising target for improving <span class="trait">drought</span> responses. Previous reports suggest that coumestrol stimulates mycorrhizal colonization and under certain conditions mycorrhizal plants have improved <span class="trait">drought</span> tolerance. This suggests that coumestrol may be part of a call for help to the rhizobiome during stress. About 3,000 genes were strongly up-regulated by <span class="trait">drought</span> and we identified regulators such as ERF, MYB, NAC, bHLH, and WRKY transcription factors, receptor-like kinases, and calcium signaling components as potential targets for soybean improvement as well as the jasmonate and abscisic acid biosynthetic genes JMT, LOX1, and ABA1. <span class="trait">Drought</span> stressed soybean leaves show reduced mRNA levels of stomatal development genes including FAMA-like, MUTE-like and SPEECHLESS-like bHLH transcription factors and leaves formed after <span class="trait">drought</span> stress had a reduction in stomatal density of 22.34 % and stomatal index of 17.56 %. This suggests that reducing stomatal density may improve <span class="trait">drought</span> tolerance. MEME analyses suggest that ABRE (CACGT/CG), CRT/DRE (CCGAC) and a novel GTGCnTGC/G element play roles in transcriptional activation and these could form components of synthetic promoters to drive expression of transgenes. Using transformed hairy roots, we validated the increase in promoter <span class="trait">activity</span> of <span class="gene">GmWRKY17</span> and <span class="gene">GmWRKY67</span> during dehydration and after 20 μM ABA treatment. CONCLUSIONS: Our toolbox provides new targets and strategies for improving soybean <span class="trait">drought</span> tolerance and includes the coumestan coumestrol, transcription factors that regulate stomatal density, water stress-responsive WRKY gene promoters and a novel DNA element that appears to be enriched in water stress responsive promoters.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We identified metabolites, proteins, and genes that are strongly up or down regulated during rapid water stress following removal from a hydroponics system. 163 metabolites showed significant changes during water stress in roots and 93 in leaves. The largest change was a root-specific 160-fold increase in the coumestan coumestrol making it a potential biomarker for <span class="trait">drought</span> and a promising target for improving <span class="trait">drought</span> responses. Previous reports suggest that coumestrol stimulates mycorrhizal colonization and under certain conditions mycorrhizal plants have improved <span class="trait">drought</span> tolerance. This suggests that coumestrol may be part of a call for help to the rhizobiome during stress. About 3,000 genes were strongly up-regulated by <span class="trait">drought</span> and we identified regulators such as ERF, MYB, NAC, bHLH, and WRKY transcription factors, receptor-like kinases, and calcium signaling components as potential targets for soybean improvement as well as the jasmonate and abscisic acid biosynthetic genes <italic> JMT </italic> , <italic> LOX1 </italic> , and <italic> ABA1 </italic> . <span class="trait">Drought</span> stressed soybean leaves show reduced mRNA levels of stomatal development genes including <italic> FAMA </italic> -like, <italic> MUTE </italic> -like and <italic> SPEECHLESS </italic> -like bHLH transcription factors and leaves formed after <span class="trait">drought</span> stress had a reduction in stomatal density of 22.34 % and stomatal index of 17.56 %. This suggests that reducing stomatal density may improve <span class="trait">drought</span> tolerance. MEME analyses suggest that ABRE (CACGT/CG), CRT/DRE (CCGAC) and a novel GTGCnTGC/G element play roles in transcriptional activation and these could form components of synthetic promoters to drive expression of transgenes. Using transformed hairy roots, we validated the increase in promoter <span class="trait">activity</span> of <italic> <span class="gene">GmWRKY17</span> </italic> and <italic> <span class="gene">GmWRKY67</span> </italic> during dehydration and after 20&#x000a0;&#x003bc;M ABA treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> responses have previously been studied in soybean using both pot-based systems (PSys) and hydroponics systems (HSys) [ <xref ref-type="bibr" rid="CR5"> 5 </xref> &#x02013; <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. PSys are more similar to field conditions with a slower rate of water loss that allows acclimation to the stress [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ]. On the other hand, the rapid stress associated with the removal of soybean plants from a hydroponics solution results in a more uniform response to the stress and this may facilitate gene, protein, and metabolite discovery. Expression profile analyses of both systems show that although there are differences, many genes appear to show similar expression characteristics, for example <italic> <span class="gene">GmaxADC2-like</span> </italic> and <italic> <span class="gene">GmaxADC2-like1</span> </italic> [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Receptor-like kinases (RLKs) are important components of early signaling including <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. Analysis of <span class="gene">RLK</span> genes reveals a marked difference between <span class="trait">root</span> and leaf tissue (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Both tissues differentially regulate many <span class="gene">RLK</span> genes but leaf tissues show a striking down-regulation of about half of the LRR subfamily III genes (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Among these are <italic> <span class="gene">Glyma15g13840</span> </italic> and <italic> <span class="gene">Glyma09g02881</span> </italic> , two soybean orthologues of the Arabidopsis <italic> GUARD CELL HYDROGEN PEROXIDE-RESISTANT1 </italic> ( <italic> GHR1 </italic> ) gene. GHR1 mediates ABA and hydrogen peroxide-regulated stomatal movement under <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] and GHR1 is a critical early component in ABA signaling. Within 30&#x000a0;min, the levels of mRNA encoding <span class="gene">Glyma15g13840</span> and <span class="gene">Glyma09g02881</span> fall to about one third of their non-stressed levels and reach a 7-9-fold reduction after 3&#x02013;5 h (Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> : Table S2). This parallels a similar rapid drop in stomatal conductance. These data identify the soybean orthologs of <italic> GHR1 </italic> as potential targets for improving <span class="trait">drought</span> tolerance via their effect on stomatal movement. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Visualization of differentially expressed receptor-like kinase genes with MapMan. The receptor kinase bin 30.2 is illustrated. <bold> a </bold> Leaf and ( <bold> b </bold> ) Roots. Subfamilies III and XI are indicated. The figure was constructed using log2-transformed ratios of induced versus control. The most intense colors represent 8-fold change. Blue denotes increase and red decrease
					</p>
				
					<p>&nbsp;&nbsp;
						FAMA, SPEECHLESS and MUTE regulate the last steps in the stomatal development signaling pathway (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> ) but upstream components of the pathway are also known. In total 58 putative soybean orthologs of Arabidopsis stomatal development genes were identified and 24 of these showed differential expression (at least 5-fold change in mRNA level) (Additional file <xref rid="MOESM16" ref-type="media"> 16 </xref> : Table S12). Strikingly, only one gene was up-regulated and only two genes showed significant variations in mRNA levels in <span class="trait">root.</span> STOMAGEN is an intracellular signaling peptide that is a positive regulator of stomatal patterning and a striking reduction of over 40-fold in the mRNA level of the <italic> STOMAGEN </italic> -like gene <italic> <span class="gene">Glyma08g45890</span> </italic> was observed. These data reveal that orthologues of genes that regulate stomatal development are among the most strongly down-regulated soybean genes during <span class="trait">drought.</span> This suggests that differentiation of stomata is reduced as a long-term response of soybean to <span class="trait">drought.</span> We validated the expression changes of fourteen genes from the oligo array using qRT-PCR (Additional file <xref rid="MOESM17" ref-type="media"> 17 </xref> : Table S13). This included several stomatal development genes including <italic> GLYMA16g02020 </italic> (FAMA-like), <italic> GLYMA11g02520 </italic> (YODA MAP Kinase Kinase Kinase-like), and <italic> GLYMA19g35200 </italic> (Stomatal Density and Distribution-1).
					</p>
				
					<p>&nbsp;&nbsp;
						Our previous work in tobacco has shown that several WRKY gene promoters (notably <italic> NtWRKY69 </italic> ) direct water stress-inducible expression as shown by promoter:GFP or promoter:GUS analyses [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. We therefore sought to validate and further characterize the inducibility of WRKY promoters from soybean because they could be good candidates for driving transgenes [ <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. Promoter:GFP constructs of two promising WRKY genes, <italic> <span class="gene">GmWRKY17</span> </italic> (GLYMA06g06530) and <italic> <span class="gene">GmWRKY67</span> </italic> (GLYMA13g44730) were transformed into soybean roots via hairy-root transformation and transgenic roots were subjected to <span class="trait">drought.</span> GFP quantification confirmed <span class="trait">drought</span> inducibility and revealed that the promoter of <italic> <span class="gene">GmWRKY17</span> </italic> directed 12.7-fold inducible expression and the <italic> <span class="gene">GmWRKY67</span> </italic> promoter 4.8-fold (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ). Additionally, the <italic> <span class="gene">GmWRKY17</span> </italic> promoter responded to ABA (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9 </xref> ) and both promoters were responsive to cold. This suggests that the promoters from <italic> <span class="gene">GmWRKY17</span> </italic> and <italic> <span class="gene">GmWRKY67</span> </italic> may prove useful for driving transgenes in projects aimed at improving <span class="trait">drought</span> responses. We have previously shown that the <italic> <span class="gene">GmWRKY53</span> </italic> and <italic> <span class="gene">GmWRKY112</span> </italic> promoters respond positively to exogenous application of salt and PEG [ <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. <fig id="Fig9"> <label> Fig. 9 </label> <caption> <p> Validation of promoter <span class="trait">activity</span> of <italic> <span class="gene">GmWRKY17</span> </italic> and <italic> <span class="gene">GmWRKY67</span> </italic> in soybean hairy-roots during dehydration and 20&#x000a0;&#x003bc;M ABA treatment. Validation of promoter <span class="trait">activity</span> via visualization and quantification of a promoter:GFP construct of <italic> <span class="gene">GmWRKY17</span> </italic> and <italic> <span class="gene">GmWRKY67</span> </italic> in soybean hairy-roots during ( <bold> a </bold> ) dehydration ( <bold> b </bold> ) 20&#x000a0;&#x003bc;M ABA treatment and ( <bold> c </bold> ) cold. The time points in hours are shown. The graph shows mean&#x02009;&#x000b1;&#x02009;standard error for 9 independent plants for each time point. The fold inducibilities are indicated
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Flooding of the <span class="trait">root</span> system in soybean: biochemical and molecular aspects of N metabolism in the nodule during stress and recovery.</h2>
			<h3>PMID:26825550  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Souza SC(1), Mazzafera P(2), Sodek L(2).
				<br/> 
				年份：2016 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen fixation of the nodule of soybean is highly sensitive to oxygen deficiency such as provoked by waterlogging of the <span class="trait">root</span> system. This study aimed to evaluate the effects of flooding on N metabolism in nodules of soybean. Flooding resulted in a marked decrease of asparagine (the most abundant amino acid) and a concomitant accumulation of γ-aminobutyric acid (GABA). Flooding also resulted in a strong reduction of the incorporation of (15)N2 in amino acids. Nodule amino acids labelled before flooding rapidly lost (15)N during flooding, except for GABA, which initially increased and declined slowly thereafter. Both nitrogenase <span class="trait">activity</span> and the expression of <span class="gene">nifH</span> and <span class="gene">nifD</span> genes were strongly decreased on flooding. Expression of the asparagine synthetase genes SAS1 and SAS2 was reduced, especially the former. Expression of genes encoding the enzyme glutamic acid decarboxylase (GAD1, GAD4, GAD5) was also strongly suppressed except for GAD2 which increased. Almost all changes observed during flooding were reversible after draining. Possible changes in asparagine and GABA metabolism that may explain the marked fluctuations of these amino acids during flooding are discussed. It is suggested that the accumulation of GABA has a storage role during flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of a glutathione S-transferase gene, GsGST, from wild soybean (Glycine soja) enhances <span class="trait">drought</span> and salt tolerance in transgenic tobacco.</h2>
			<h3>PMID:20383560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ji W(1), Zhu Y, Li Y, Yang L, Zhao X, Cai H, Bai X.
				<br/> 
				年份：2010 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glycine soja is a species of soybean that survives in adverse environments including high salt and <span class="trait">drought</span> conditions. We constructed a cDNA library from G. soja seedlings treated with NaCl and isolated a glutathione S-transferase gene (GsGST: GQ265911) from the library. The cDNA encoding GsGST contains an open reading frame of 660 bp and the predicted protein belongs to the tau class of <span class="gene">GST</span> family proteins. Tobacco plants over-expressing the GsGST gene showed sixfold higher <span class="gene">GST</span> <span class="trait">activity</span> than wild-type plants. Transgenic tobacco plants exhibited enhanced dehydration tolerance. T(2) transgenic tobacco plants showed higher tolerance at the seedling stage than wild-type plants to salt and mannitol as demonstrated by longer <span class="trait">root</span> length and less growth retardation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Homologous soybean and Arabidopsis genes share responsiveness to cyst nematode infection.</h2>
			<h3>PMID:20565617</h3>
				

				
			<p>
				作者：Mazarei M(1), Lennon KA, Puthoff DP, Rodermel SR, Baum TJ.
				<br/> 
				年份：2004 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SUMMARY We previously isolated a partial soybean cDNA clone (D17.1) whose corresponding transcript increases in susceptible roots 1 day post inoculation (dpi) with the soybean cyst nematode, Heterodera glycines. Here we isolated the corresponding full-length cDNA from a soybean cDNA library and designated this gene of unknown function <span class="gene">Gm17.1.</span> Time course RNA gel blot analyses revealed that <span class="gene">Gm17.1</span> mRNA steady-state levels were elevated in soybean roots following H. glycines infection up to at least 6 dpi. For further in-depth study we identified a homologous Arabidopsis thaliana gene and designated this gene At17.1. Arabidopsis is successfully infected by the sugar beet cyst nematode (H. schachtii), a close relative of H. glycines. We isolated the At17.1 promoter, fused it to the beta-glucuronidase (GUS) reporter gene, and transformed this construct into Arabidopsis plants as well as soybean hairy roots. Histochemical analysis of plant materials containing the At17.1::GUS construct revealed that the At17.1 promoter is functional in Arabidopsis as well as in soybean and that during normal plant development the At17.1 promoter directs GUS expression predominantly to the vascular tissues and <span class="trait">root</span> tips of both plant species. When At17.1::GUS Arabidopsis plants and soybean hairy roots were inoculated with cyst nematodes, strong GUS <span class="trait">activity</span> was detected within the cyst nematode-induced feeding structures. Further tests of At17.1 promoter <span class="trait">activity</span> in Arabidopsis revealed that this promoter was induced by auxin, jasmonic acid, mannitol and dehydration. Quantitative real-time reverse transcription-polymerase chain reaction assays of At17.1 expression confirmed the observed promoter characteristics. Based on our expression data and the observation that both the soybean and the Arabidopsis homologues behaved in a similar fashion following cyst nematode infection, it is likely that these genes are closely associated with cyst nematode parasitism of plants, potentially with hormone and osmotic changes occurring in the developing nematode feeding cells. Furthermore, these data provide additional insights into the strengths of the Arabidopsis-H. schachtii pathosystem to study cyst nematode-plant interactions in lieu of less tractable pathosystems. This finding is supported by the fact that the Arabidopsis promoter tested here produced similar results in Arabidopsis and soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxidase and phenylalanine ammonia-lyase activities, phenolic acid contents, and allelochemicals-inhibited <span class="trait">root</span> growth of soybean.</h2>
			<h3>PMID:12125206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Herrig V(1), Ferrarese Mde L, Suzuki LS, Rodrigues JD, Ferrarese-Filho O.
				<br/> 
				年份：2002
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The influence of the allelochemicals ferulic (FA) and vanillic (VA) acids on peroxidase (POD, EC 1.11.1.7) and phenylalanine ammonia-lyase (PAL, EC 4.3.1.5) activities and their relationships with phenolic acid (PhAs) contents and <span class="trait">root</span> growth of soybean (Glycine max (L.) Merr.) were examined. Three-day-old seedlings were cultivated in nutrient solution containing FA or VA (0.1 to 1 mM) for 48 h. Both compounds (at 0.5 and 1 mM) decreased <span class="trait">root</span> length (RL), fresh weight (FW) and dry weight (DW) and increased PhAs contents. At 0.5 and 1 mM, FA increased soluble POD <span class="trait">activity</span> (18% and 47%, respectively) and cell wall (CW)-bound POD <span class="trait">activity</span> (61% and 34%), while VA increased soluble POD <span class="trait">activity</span> (33% and 17%) but did not affect CW-bound POD <span class="trait">activity.</span> At 1 mM, FA increased (82%) while VA reduced (32%) <span class="gene">PAL</span> activities. The results are discussed on the basis of the role of these compounds on phenylpropanoid metabolism and <span class="trait">root</span> growth and suggest that the effects caused on POD and <span class="gene">PAL</span> activities are some of the many mechanisms by which allelochemicals influence plant growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Ammonia-regulated expression of a soybean gene encoding cytosolic glutamine synthetase in transgenic Lotus corniculatus.</h2>
			<h3>PMID:1688099  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Miao GH(1), Hirel B, Marsolier MC, Ridge RW, Verma DP.
				<br/> 
				年份：1991 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A full-length cDNA clone encoding cytosolic glutamine synthetase (GS), expressed in roots and <span class="trait">root</span> nodules of soybean, was isolated by direct complementation of an Escherichia coli gln A- mutant. This sequence is induced in roots by the availability of ammonia. A 3.5-kilobase promoter fragment of a genomic clone (lambda GS15) corresponding to this cDNA was isolated and fused with a reporter [beta-glucuronidase (GUS)] gene. The GS-GUS fusion was introduced into a legume (Lotus corniculatus) and a nonlegume (tobacco) plant by way of Agrobacterium-mediated transformations. This chimeric gene was found to be expressed in a root-specific manner in both tobacco and L. corniculatus, the expression being restricted to the growing <span class="trait">root</span> apices and the vascular bundles of the mature <span class="trait">root.</span> Treatment with ammonia increased the expression of this chimeric gene in the legume background (i.e., L. corniculatus); however, no induction was observed in tobacco roots. Histochemical localization of GUS <span class="trait">activity</span> in ammonia-treated transgenic L. corniculatus roots showed a uniform distribution across all cell types. These data suggest that the tissue specificity of the soybean cytosolic <span class="gene">GS</span> gene is conserved in both tobacco and L. corniculatus; however, in the latter case, this gene is ammonia inducible. Furthermore, the ammonia-enhanced <span class="gene">GS</span> gene expression in L. corniculatus is due to an increase in transcription. That this gene is directly regulated by externally supplied or symbiotically fixed nitrogen is also evident from the expression of GS-GUS in the infection zone, including the uninfected cells, and the inner cortex of transgenic L. corniculatus nodules, where a flux of ammonia is encountered by this tissue. The lack of expression of GS-GUS in the outer cortex of the nodules suggests that ammonia may not be able to diffuse outside the endodermis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structural and transcriptional characterization of a novel member of the soybean urease gene family.</h2>
			<h3>PMID:26874294  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wiebke-Strohm B(1), Ligabue-Braun R(2), Rechenmacher C(3), De Oliveira-Busatto LA(4), Carlini CR(5), Bodanese-Zanettini MH(6).
				<br/> 
				年份：2016 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In plants, ureases have been related to urea degradation, to defense against pathogenic fungi and phytophagous insects, and to the soybean-Bradyrhizobium japonicum symbiosis. Two urease isoforms have been described for soybean: the embryo-specific, encoded by Eu1 gene, and the ubiquitous urease, encoded by Eu4. A third urease-encoding locus exists in the completed soybean genome. The gene was designated Eu5 and the putative product of its ORF as SBU-III. Phylogenetic analysis shows that 41 plant, moss and algal ureases have diverged from a common ancestor protein, but ureases from monocots, eudicots and ancient species have evolved independently. Genomes of ancient organisms present a single urease-encoding gene and urease-encoding gene duplication has occurred independently along the evolution of some eudicot species. SBU-III has a shorter amino acid sequence, since many gaps are found when compared to other sequences. A mutation in a highly conserved amino acid residue suggests absence of ureolytic <span class="trait">activity,</span> but the overall protein architecture remains very similar to the other ureases. The expression profile of urease-encoding genes in different organs and developmental stages was determined by RT-qPCR. Eu5 transcripts were detected in seeds one day after dormancy break, roots of young plants and embryos of developing seeds. Eu1 and <span class="gene">Eu4</span> transcripts were found in all analyzed organs, but <span class="gene">Eu4</span> expression was more prominent in seeds one day after dormancy break whereas Eu1 predominated in developing seeds. The evidence suggests that SBU-III may not be involved in nitrogen availability to plants, but it could be involved in other biological role(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene transcript accumulation and in situ mRNA hybridization of two putative glutamate dehydrogenase genes in etiolated Glycine max seedlings.</h2>
			<h3>PMID:25922975  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dimou M(1), Tsaniklidis G, Aivalakis G, Katinakis P.
				<br/> 
				年份：2015
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glutamate dehydrogenase (EC 1.4.1.2) is a multimeric enzyme that catalyzes the reversible amination of α-ketoglutarate to form glutamate. We characterized cDNA clones of two Glycine max sequences, <span class="gene">GmGDH1</span> and <span class="gene">GmGDH2,</span> that code for putative α- and β-subunits, respectively, of the NADH dependent enzyme. Temporal and spatial gene transcript accumulation studies using semiquantitative RT-PCR and in situ hybridization have shown an overlapping gene transcript accumulation pattern with differences in relative gene transcript accumulation in the organs examined. Detection of NADH-dependent glutamate dehydrogenase <span class="trait">activity</span> in situ using a histochemical method showed concordance with the spatial gene transcript accumulation patterns. Our findings suggest that although the two gene transcripts are co-localized in roots of etiolated soybean seedlings, the ratio of the two subunits of the active holoenzyme may vary among tissues.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cysteine protease and cystatin expression and <span class="trait">activity</span> during soybean nodule development and senescence</h2>
			<h3>PMID:25404209</h3>
				

				
					<h3>PMCID:4243279</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> van Wyk </surname>  <given-names> Stefan George </given-names>  </name>  <address>  <email> stefan.vanwyk@fabi.up.ac.za </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du Plessis </surname>  <given-names> Magdeleen </given-names>  </name>  <address>  <email> magdeleen.duplessis@fabi.up.ac.za </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cullis </surname>  <given-names> Christoper Ashley </given-names>  </name>  <address>  <email> cac5@case.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kunert </surname>  <given-names> Karl Josef </given-names>  </name>  <address>  <email> karl.kunert@fabi.up.ac.za </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Vorster </surname>  <given-names> Barend Juan </given-names>  </name>  <address>  <email> juan.vorster@fabi.up.ac.za </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Plant Production and Soil Science, Forestry and Agricultural Biotechnology Institute, University of Pretoria, Pretoria, 0002 South Africa  </aff>  <aff id="Aff2">  <label/> Department of Biology, Case Western Reserve University Cleveland, Cleveland, OH 44106 USA  </aff>  <aff id="Aff3">  <label/> Department of Plant Science, Forestry and Agricultural Biotechnology Institute, University of Pretoria, Pretoria, 0002 South Africa  </aff>
				<br/> 
				年份：<day> 17 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Nodules play an important role in fixing atmospheric nitrogen for soybean growth. Premature senescence of nodules can negatively impact on nitrogen availability for plant growth and, as such, we need a better understanding of nodule development and senescence. Cysteine proteases are known to play a role in nodule senescence, but knowledge is still fragmented regarding the function their inhibitors (cystatins) during the development and senescence of soybean nodules. This study provides the first data with regard to cystatin expression during nodule development combined with biochemical characterization of their inhibition strength. RESULTS: Seventy nine non-redundant cysteine protease gene sequences with homology to papain, belonging to different subfamilies, and several legumain-like cysteine proteases (vacuole processing enzymes) were identified from the soybean genome assembly with eighteen of these cysteine proteases actively transcribed during nodule development and senescence. In addition, nineteen non-redundant cystatins similar to oryzacystatin-I and belonging to cystatin subgroups A and C were identified from the soybean genome assembly with seven actively transcribed in nodules. Most cystatins had preferential affinity to cathepsin L-like cysteine proteases. Transcription of cystatins <span class="gene">Glyma05g28250,</span> <span class="gene">Glyma15g12211,</span> <span class="gene">Glyma15g36180</span> particularly increased during onset of senescence, possibly regulating proteolysis when nodules senesce and undergo programmed cell death. Both actively transcribed and non-actively transcribed nodule cystatins inhibited cathepsin-L- and B-like activities in different age nodules and they also inhibited papain and cathepsin-L <span class="trait">activity</span> when expressed and purified from bacterial cells. CONCLUSIONS: Overlap in activities and specificities of actively and non-actively transcribed cystatins raises the question if non-transcribed cystatins provide a reservoir for response to particular environments. This data might be applicable to the development of strategies to extend the active life span of nodules or prevent environmentally induced senescence.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seventy nine non-redundant cysteine protease gene sequences with homology to papain, belonging to different subfamilies, and several legumain-like cysteine proteases (vacuole processing enzymes) were identified from the soybean genome assembly with eighteen of these cysteine proteases actively transcribed during nodule development and senescence. In addition, nineteen non-redundant cystatins similar to oryzacystatin-I and belonging to cystatin subgroups A and C were identified from the soybean genome assembly with seven actively transcribed in nodules. Most cystatins had preferential affinity to cathepsin L-like cysteine proteases. Transcription of cystatins <span class="gene">Glyma05g28250,</span> <span class="gene">Glyma15g12211,</span> <span class="gene">Glyma15g36180</span> particularly increased during onset of senescence, possibly regulating proteolysis when nodules senesce and undergo programmed cell death. Both actively transcribed and non-actively transcribed nodule cystatins inhibited cathepsin-L- and B-like activities in different age nodules and they also inhibited papain and cathepsin-L <span class="trait">activity</span> when expressed and purified from bacterial cells.
					</p>
				
					<p>&nbsp;&nbsp;
						The Phytozome database ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net"> www.phytozome.net </ext-link> ) currently contains over 300 cystatin-like sequences from the Viridiplantae kingdom, 706 <span class="gene">C1</span> cysteine protease sequences and 362 C13 cysteine protease (VPE-type) sequences. The recent release of the complete soybean genome [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ] as well as the release of a RNAseq atlas of genes expressed in fourteen different soybean tissues including nodules [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ] has further allowed identification and characterization of all 19 soybean cystatins, irrespective of transcriptional <span class="trait">activity,</span> and 18 active cysteine proteases. Accurate studies are now possible to determine the cystatin and cysteine protease classes expressed in nodules and also to investigate if endogenous cystatins preferentially interact with specific target cysteine proteases in nodules. Our study was therefore aimed to provide a first insight into such interactions by identifying and characterizing all members of the cystatin and cysteine protease gene families in soybean nodules. We included both actively and non-actively transcribed cystatins and cysteine proteases identified through homology searches in the soybean genomic database. The nodule transcription profiles were developed with the technique of RNAseq [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] which allowed us to determine the expression of all oryzacystatin I-like cystatins, papain-like cysteine proteases, as well as vacuole VPE-type cysteine proteases in determinate soybean crown nodules during nodule development and senescence. Such VPE cysteine proteases resemble mammalian caspases and they contribute to the senescence process and PCD (Programmed Cell Death) [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ], but might further activate pre-proteases by post-translational modification [ <xref ref-type="bibr" rid="CR19"> 19 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In our characterization, we were also interested to determine to which families and functional groups nodule cystatins and cysteine proteases belong as well as the cystatin substrate preference by testing <italic> in vitro </italic> produced cystatin proteins with various cysteine protease-containing extracts. Cystatins are part of subfamily B of the I25 cystatin family and in cereals they can be divided into various functional groups (A, B and C) with most cystatins belonging to groups A and C [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. Group A cystatins, which efficiently inhibit cathepsin L-like cysteine-proteases, are preferentially expressed in dry and germinating seeds whereas group <span class="gene">C1</span> cystatins, which are potent inhibitors of C1A peptidases, are mostly expressed in developing seed endosperms. Cysteine proteases cluster into different subfamilies [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] with cysteine proteases closest to papain clustering with subfamily XCP1 represented by the <italic> Arabidopsis thaliana </italic> genes <span class="gene">At1g20850</span> and At4g35350. Cysteine proteases with cathepsin-L-like <span class="trait">activity</span> can closely cluster with subfamily RD21 consisting of RD21A ( <italic> A. thaliana </italic> gene <span class="gene">At1g47128),</span> RD21B (At5g43060) and RD21C (At3g19390). A C-terminal granulin domain is characteristic of the RD21 subfamily. Cysteine proteases with cathepsin-L-like <span class="trait">activity</span> can further cluster with the <span class="gene">SAG12</span> subfamily. Cysteine proteases with cathepsin-F-like <span class="trait">activity</span> cluster with subfamily RD19 with members RD19A (At4g39090), RD19B (At2g21430) and RD19C (At4g16190) and RD19 members have a characteristic ERFNAQ motif in the pro-domain. Cysteine proteases with cathepsin-H-like <span class="trait">activity</span> cluster with members of the AALP (At5g60360) and ALP2 (At3g45310) subfamily.
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 99 cysteine protease sequences with homology (1E&#x02009;&#x02264;&#x02009;&#x02212;1.0) to the model cysteine protease papain (E.C.3.4.22.2) were further identified from the soybean genome assembly (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Several sequences were alleles, paralogos and orthologos of other cysteine protease gene sequences. From these we identified 79 non-redundant cysteine protease gene sequences which had similarity to members of eight different cysteine protease sub-families. Seven sub-families were distinguished from our expression data and we identified confidently five functional groups (Figure&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). However, none of the identified soybean cysteine proteases clustered with papain (subfamily XCP1). Cysteine proteases with cathepsin-L-like <span class="trait">activity</span> included <span class="gene">Glyma04g03090</span> (closely clustering with subfamily RD21), as well as the two proteases <span class="gene">Glyma14g09440</span> and <span class="gene">Glyma17g35720</span> (similar to subfamily RD21 members). We also confirmed the C-terminal granulin domain, characteristic of the RD21 subfamily, in these cysteine proteases. <span class="gene">Glyma04g04400</span> (cathepsin-L-like <span class="trait">activity)</span> had highest similarity to RDL2 (Arabidopsis gene At3g19400) and closely clustered with the RD21 subfamily members. Finally, <span class="gene">Glyma04g36470</span> and <span class="gene">Glyma06g18390</span> (cathepsin-L-like <span class="trait">activity)</span> were highly similar to members of the <span class="gene">SAG12</span> subfamily despite absence of the additional C amino acid in the CG <italic> C </italic> CWAFS motif. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> <bold> Mapping of transcribed cysteine proteases to sub-families and functional groups with similarity to the <span class="gene">C1</span> cysteine protease papain. </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Seven proteases with cathepsin-F-like <span class="trait">activity</span> <span class="gene">(Glyma04g03020,</span> <span class="gene">Glyma06g03050,</span> <span class="gene">Glyma10g35100,</span> <span class="gene">Glyma11g12130,</span> <span class="gene">Glyma12g04340,</span> <span class="gene">Glyma14g40670,</span> <span class="gene">Glyma17g37400)</span> were highly similar to subfamily RD19 members. However, the ERFNAQ motif (instead of the ERFNIN motif in the pro-domain) characteristic of the RD19 subfamily, was absent. <span class="gene">Glyma08g12340,</span> which had no significant similarity to any specific subfamily, was closest to the two subfamilies RD19 or CTB3. Further cysteine proteases with cathepsin-H-like <span class="trait">activity</span> included <span class="gene">Glyma09g08100,</span> <span class="gene">Glyma15g19580</span> and <span class="gene">Glyma17g05670,</span> which had high similarity to AALP and ALP2. The three proteases also had an N-terminal NPIR vacuolar targeting signal and other RD21 subfamily motifs (except that the ATC motif was lacking in <span class="gene">Glyma09g08100).</span> Although <span class="gene">Glyma03g38520</span> and <span class="gene">Glyma19g41120</span> had similarity to this subfamily, they contained an ECGIE motif in the C terminus, characteristic of subfamily CTB3.
					</p>
				
					<p>&nbsp;&nbsp;
						From the initial 99 putative cysteine protease sequences homologous to the model <span class="gene">C1</span> cysteine protease papain, 18 cysteine proteases were transcriptionally active in nodules during at least one time point (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B). <span class="gene">Glyma15g19580</span> (cathepsin-H like <span class="trait">activity)</span> was the most abundant cysteine protease in 4&#x000a0;weeks old nodules with <span class="gene">Glyma17g37400</span> (cathepsin-F like <span class="trait">activity)</span> the most abundant at 14&#x000a0;weeks. Transcription of the majority of cysteine proteases increased with the onset of senescence, with five cysteine proteases <span class="gene">(Glyma04g04400,</span> <span class="gene">Glyma08g12340,</span> <span class="gene">Glyma10g35100,</span> <span class="gene">Glyma11g12130</span> and <span class="gene">Glyma17g05670)</span> highly expressed in 4 and 8&#x000a0;weeks old nodules. None of the cysteine protease transcription changed significantly (p &#x02265;0.05) except <span class="gene">Glyma06g18390</span> transcription, with a very low relative abundance, which changed (p &#x02264;0.05) due to senescence (Figure&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> B).
					</p>
				
					<p>&nbsp;&nbsp;
						In a next step, we carried out cysteine protease <span class="trait">activity</span> measurements with nodule extracts to determine potency of transcribed cystatins. Fluorometric interaction assays were used with either commercially available cathepsin-L or cathepsin-B as well as isolated nodule protein extracts representing the total proteolytic complement active in nodules. To establish a preferential binding for each cystatin, we first tested cystatin potency with commercially available enzyme preparations for cathepsin-L and cathepsin-B. Cystatins transcribed in nodules had generally stronger affinity for cathepsin-L than cathepsin-B, with <span class="gene">Glyma13g27980</span> and <span class="gene">Glyma14g04250</span> equally effective in preventing both cathepsin activities (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). Further, <span class="gene">Glyma15g36180</span> inhibited cathepsin-L, but was unable to inhibit cathepsin-B, even when an inhibitor concentration of 1&#x000a0;mM was used. In contrast, cystatins not transcriptionally active in nodules showed higher inhibition rates of cathepsin-L, with <span class="gene">Glyma18g12240</span> inhibiting both cathepsin-L and -B. <span class="gene">Glyma14g04260&#x02019;s</span> second domain and both domains of <span class="gene">Glyma14g04291</span> were further unable to inhibit cathepsin-B, even at a concentration of 1&#x000a0;mM (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). <table-wrap id="Tab1"> <label> Table 1 </label> <caption> <p> <bold> Inhibition (%) of protease <span class="trait">activity</span> by actively and non-actively transcribed cystatins during nodule life-span </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						We then tested cystatin potency against various nodule extracts (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). We first used the model rice cystatin OC-I as well as the cysteine protease inhibitor E64. OC-I and E64 both prevented cathepsin-L-like <span class="trait">activity</span> in 4&#x000a0;weeks old nodules but were less efficient against extracts derived from 8 and 14&#x000a0;weeks old nodules (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). Both inhibitors also prevented cathepsin-B-like <span class="trait">activity</span> in an extract of 4&#x000a0;weeks old nodules. We then compared OC-I and E64 potency with the potency of various recombinant soybean cystatins either actively transcribed or non-active in nodules (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). Cystatins tested were generally more active against extracts from younger nodules (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). Five of the cystatins actively transcribed in nodules blocked cysteine protease <span class="trait">activity</span> in nodule extracts. However, only <span class="gene">Glyma05g2850</span> inhibited cathepsin-L-like <span class="trait">activity</span> in nodule extracts from all three time points (4, 8, and 12&#x000a0;weeks) and cathepsin-B-like <span class="trait">activity</span> in extracts derived from 4 and 8&#x000a0;weeks old nodules. The most potent cystatin among the expressed cystatins was <span class="gene">Glyma15g36180</span> and potency of this cystatin was comparable to OC-I and E64 when either cathepsin&#x02013;L or B <span class="trait">activity</span> was measured in an extract derived from 4&#x000a0;weeks old nodules. <table-wrap id="Tab2"> <label> Table 2 </label> <caption> <p> <bold> Expression and inhibitory potency of cystatins against proteases from different aged nodules </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Finally, we were also interested in testing cystatins not actively transcribed during nodule development. These cystatins were generally more active against nodule extracts than cystatins actively transcribed in nodules (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). All non-transcribed cystatins had potency comparable to OC-I and E64 when tested against an extract derived from 4&#x000a0;weeks old nodules. Among them, <span class="gene">Glyma14g04260</span> domain 1 and <span class="gene">Glyma18g12240</span> had highest inhibition of all tested cystatins with 58.9% and 54% inhibition respectively. Three cystatins <span class="gene">(Glyma04g10360,</span> <span class="gene">Glyma07g39590</span> and <span class="gene">Glyma18g12240)</span> inhibited cathepsin-L as well as cathepsin-B like <span class="trait">activity</span> in extracts derived from all three time points. <span class="gene">Glyma07g39590</span> was further the most potent of all tested cystatins against cathepsin-L and B <span class="trait">activity</span> in an extract derived from 14&#x000a0;weeks old senescent nodules.
					</p>
				
					<p>&nbsp;&nbsp;
						We identified 8 cystatin genes expressed in soybean nodules using RNAseq. Since we carried out macro-dissection of crown nodule tissue, and not micro-dissection with selected tissues, RNAseq data represented transcription profiles of the entire nodule and also contained transcripts from areas surrounding the senescing nodule cortex. When we compared their transcription with already published RNAseq data from various other tissue types [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ], none of the identified nodule cystatins was uniquely transcribed. Several cystatins were further actively transcribed during nodule development and senescence but not exclusively transcribed at a particular time point like senescence. Only <span class="gene">Glyma05g28250</span> was actively transcribed, and also inhibited cathepsin-L-like <span class="trait">activity</span> in nodule extracts, at all three selected time points. The cystatin very likely plays a maintenance role and regulates cysteine protease <span class="trait">activity</span> throughout nodule development and senescence. The other actively transcribed cystatins were only capable of inhibiting specific types of cysteine proteases <span class="trait">activity</span> (cathepsin L or B) at specific time points. Cathepsin B is a member of the peptidase <span class="gene">C1</span> family and this cysteine protease is required for PCD involved in the plant disease <span class="trait">resistance</span> hypersensitive response [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. Transcription of cystatins <span class="gene">Glyma05g28250,</span> <span class="gene">Glyma15g12211,</span> <span class="gene">Glyma15g36180</span> increased by about two-fold during the onset of senescence with concurrent co-induction of several cysteine proteases. These cystatins very likely regulate proteolysis when nodules senesce and undergo PCD to maintain nitrogen fixation in determinate soybean nodules for as long as possible.
					</p>
				
					<p>&nbsp;&nbsp;
						The nodule group A cystatin cluster also contained two cystatins, <span class="gene">Glyma13g25870</span> and <span class="gene">Glyma15g36180,</span> with a C-terminal extension. Such an extension was also found in <span class="gene">Glyma05g28250,</span> highly similar to group B cystatins. Plant cystatins with a carboxy-terminal extension contain a SNSL amino acid motif and inhibit cysteine proteases of the legumain C13 family (VPEs) [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. Their consistent transcription during nodule development and increase of transcription found for <span class="gene">Glyma15g36180</span> and <span class="gene">Glyma05g28250</span> when nodules senesce, indicates that they are very likely produced to tightly control cell disruption and activation of any cysteine proteases that may compromise nitrogen fixation. VPE proteases resemble mammalian caspases and they contribute to the senescence process and PCD by contributing to the collapse of the vacuole membrane with release of proteases into the cell [ <xref ref-type="bibr" rid="CR18"> 18 </xref> ]. There is also evidence that VPEs play a regulatory role activating pre-proteases by post-translational modification, leading to maturation and proteolytic <span class="trait">activity</span> upon removal on the I19 inhibitory domain [ <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. Cysteine proteases, expressed as pre-proteins, consist of an I29 inhibitor domain preventing non-specific <span class="trait">activity</span> [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. In our study, transcription of the entire set of nodule VPE cysteine proteases strongly increased coinciding with the progression of senescence. VPEs are therefore predominantly transcribed in senescent nodules and might play an important role in the activation of cysteine proteases. These activated cysteine proteases finally degrade both the bacteroids and nodule cells [ <xref ref-type="bibr" rid="CR28"> 28 </xref> - <xref ref-type="bibr" rid="CR32"> 32 </xref> ] and correlates with nitrogenase <span class="trait">activity</span> decrease [ <xref ref-type="bibr" rid="CR8"> 8 </xref> ] as well as decrease in both crown nodule biomass and nodule number [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To analyze any endogenous cystatin function in nodules, it is crucial to identify their possible endogenous target cysteine proteases. Only little is so far known about any possible direct interaction between cystatins and their target proteases <italic> in vivo </italic> [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. We therefore searched cystatin and cysteine proteases sequences for signal peptides to obtain some information about their cellular localisation. Cystatins <span class="gene">Glyma05g28250,</span> <span class="gene">Glyma07g39590</span> and <span class="gene">Glyma13g25870</span> are likely localised in the secretory pathway. The ER has a vast protein storage protein capacity and relatively low proteolytic <span class="trait">activity</span> and cystatins might contribute to low proteolytic <span class="trait">activity</span> in this organelle [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. Nodule cystatins, such as <span class="gene">Glyma05g28250,</span> <span class="gene">Glyma07g39590</span> and <span class="gene">Glyma13g25870</span> with signal peptides, would be able to interact with C1A proteases that also have secretion sequences in their gene sequences [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. MJ Chrispeels and NV Raikhel [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ] further predicted that the open reading frame of two cysteine proteases contain a putative vacuolar targeting signal. We also found such signal for <span class="gene">Glyma04g36470</span> and <span class="gene">Glyma06g18390,</span> despite that <span class="gene">Glyma06g18390</span> was explicitly expressed during nodule senescence which raises the question of the particular <span class="gene">Glyma06g18390</span> function when targeted to the vacuole. One possible explanation is that this vacuolar targeting signal directs these cysteine proteases to the bacteroid-containing symbiosome compartment for bacteroid protein degradation affecting peribacteriod membrane stability. Rupture of the peribacteriod membrane then eliminates the microbial partner [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Model representative sequences for the 8 different cysteine proteases subfamilies described by [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ] were RD21A <span class="gene">(At1g47128),</span> RD21B (At5g43060), RD21C (At3g19390), RDL2 (At3g19400), XBCP3 <span class="gene">(At1g09850),</span> XCP1 (At4g35350), XCP1 <span class="gene">(At1g20850),</span> THI1 <span class="gene">(At1g06260),</span> <span class="gene">SAG12</span> (At5g45890), RD19A (At4g39090), RD19B, (At2g21430), RD19C (At4g16190), AALP (At5g60360). ALP2 (At3g45310) and CTB3 (At4g1610) were also included in the phylogenetic trees to infer possible functional <span class="trait">activity</span> of the proteases. Out-group used for the <span class="gene">C1</span> cysteine protease phylogenetic analysis was OCI (Os01g58890) and a further I25B cystatin from <italic> Vigna unguiculata </italic> (Q06445).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The contribution of bacteroidal nitrate and nitrite reduction to the formation of nitrosylleghaemoglobin complexes in soybean <span class="trait">root</span> nodules.</h2>
			<h3>PMID:17259612  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Meakin GE(1), Bueno E, Jepson B, Bedmar EJ, Richardson DJ, Delgado MJ.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is becoming recognized that leghaemoglobin constitutes an important buffer for the cytotoxic nitric oxide radical (NO(*)) in <span class="trait">root</span> nodules, although the sources of this NO(*) within nodules are unclear. In Bradyrhizobium japonicum bacteroids, NO(*) can be produced through the denitrification process, during which nitrate is reduced to nitrite by the periplasmic nitrate reductase Nap, and nitrite is reduced to NO(*) by the respiratory nitrite reductase NirK. To assess the contribution of bacteroidal denitrification to the NO(*) within nitrate-treated soybean nodules, electron paramagnetic resonance and UV-visible spectroscopy were employed to study the presence of nitrosylleghaemoglobin (LbNO) within nodules from plants inoculated with wild-type, <span class="gene">napA</span> or <span class="gene">nirK</span> B. japonicum strains. Since it has been found that hypoxia induces NO(*) production in plant <span class="trait">root</span> tissue, and that plant roots can be subjected to hypoxic stress during <span class="trait">drought</span> and flooding, the effect of hypoxic stress on the formation of LbNO complexes within nodules was also investigated. Maximal levels of LbNO were observed in nodules from plants treated with nitrate and subjected to hypoxic conditions. It is shown that, in the presence of nitrate, all of the LbNO within normoxic nodules arises from nitrate reduction by the bacteroidal periplasmic nitrate reductase, whereas Nap <span class="trait">activity</span> is only responsible for half of the LbNO within hypoxic nodules. In contrast to Nap, NirK is not essential for LbNO formation under any condition tested.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Abnormally high digestive enzyme <span class="trait">activity</span> and gene expression explain the contemporary evolution of a <italic> Diabrotica </italic> biotype able to feed on soybeans</h2>
			<h3>PMID:22957201</h3>
				
					<h3>PMCID:3434003</h3>
					<h4>期刊:Ecology and Evolution</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Curzi </surname>  <given-names> Mat&#x000ed;as J </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zavala </surname>  <given-names> Jorge A </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Spencer </surname>  <given-names> Joseph L </given-names>  </name>  <xref ref-type="aff" rid="au3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seufferheld </surname>  <given-names> Manfredo J </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  </contrib>  <aff id="au1">  <label> 1 </label>  <institution> Department of Crop Sciences, University of Illinois at Urbana-Champaign </institution>  <addr-line> Urbana, IL, 61801 </addr-line>  </aff>  <aff id="au2">  <label> 2 </label>  <institution> C&#x000e1;tedra de Bioqu&#x000ed;mica/INBA (CONICET/UBA), Facultad de Agronom&#x000ed;a, Universidad de Buenos Aires, Universidad Cat&#x000f3;lica Argentina </institution>  <addr-line> Buenos Aires, C1417DSE, Argentina </addr-line>  </aff>  <aff id="au3">  <label> 3 </label>  <institution> Illinois Natural History Survey, University of Illinois at Urbana-Champaign </institution>  <addr-line> Champaign, IL, 61820 </addr-line>  </aff>
				<br/> 
				年份：<day> 19 </day>  <month> 6 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Broadly, Coleopterans respond to high dietary CystPI <span class="trait">activity</span> by changing their feeding site on the plant or via biochemical changes in the gut (Zhu-Salzman et al. <xref ref-type="bibr" rid="b52"> 2003 </xref> ; Zavala et al. <xref ref-type="bibr" rid="b50"> 2009 </xref> ). Colorado potato beetle ( <italic> Leptinotarsa decemlineata </italic> ) and the cowpea bruchid ( <italic> Callosobruchus maculatus </italic> ) compensate for cysteine protease inhibition by increasing the expression and <span class="trait">activity</span> of gut digestive proteases (Bolter <xref ref-type="bibr" rid="b4"> 1995 </xref> ; Cloutier et al. <xref ref-type="bibr" rid="b10"> 2000 </xref> ; Zhu-Salzman et al. <xref ref-type="bibr" rid="b52"> 2003 </xref> ; Gruden et al. <xref ref-type="bibr" rid="b14"> 2004 </xref> ; Moon et al. <xref ref-type="bibr" rid="b34"> 2004 </xref> ). A diet containing soybean CystPI strongly induced cathepsin B-like protease transcripts in cowpea bruchids (Koo et al. <xref ref-type="bibr" rid="b23"> 2008 </xref> ). Protease <span class="trait">activity</span> in the WCR gut is regulated by at least five cathepsin L-like and two cathepsin B-like protease genes, which are susceptible to inhibition by soybean CystPIs (Koiwa et al. <xref ref-type="bibr" rid="b22"> 2000 </xref> ; Bown et al. <xref ref-type="bibr" rid="b6"> 2004 </xref> ; Siegfried et al. <xref ref-type="bibr" rid="b41"> 2005 </xref> ). We predict that compared with rotation-susceptible wild-type (WT) adults that rarely enter soybean fields, RR WCR should have constitutively high gut cysteine protease <span class="trait">activity</span> or the ability to significantly increase <span class="trait">activity</span> while feeding on soybean foliage. In the last 15 years, previous work to document differences between the WT and RR phenotypes of WCR has described only behavioral differences related to greater motility (Levine and Oloumi-Sadeghi <xref ref-type="bibr" rid="b26"> 1996 </xref> ; Levine et al. <xref ref-type="bibr" rid="b27"> 2002 </xref> ; Mabry and Spencer <xref ref-type="bibr" rid="b28"> 2003 </xref> ; Spencer et al. <xref ref-type="bibr" rid="b44"> 2005 </xref> ; Knolhoff et al. <xref ref-type="bibr" rid="b19"> 2006 </xref> , <xref ref-type="bibr" rid="b20"> 2010a </xref> ). Garabagi et al. ( <xref ref-type="bibr" rid="b12"> 2008 </xref> ) tested whether expression of the cyclic <span class="gene">GMP-dependent</span> protein kinase gene (a homolog of the <italic> foraging </italic> gene implicated in movement and foraging in <italic> Drosophila </italic> and honeybees) was associated with RR WCR behavior; however, the differences in kinase gene expression showed no direct link between RR and behavior of lab-reared insects. Further attempts to identify eco-physiological, biochemical, and molecular differences between phenotypes and to explain how RR WCR can feed on soybean foliage have proved elusive (Miller et al. <xref ref-type="bibr" rid="b31"> 2006 </xref> , <xref ref-type="bibr" rid="b32"> 2007 </xref> ; Knolhoff et al. <xref ref-type="bibr" rid="b21"> 2010b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Protease <span class="trait">activity</span> and gene expression in guts of RR and WT WCR. (A) Cathepsin L-like protease <span class="trait">activity</span> in guts of WCR adults fed on soybeans in field conditions. (B) Cathepsin L-like protease <span class="trait">activity</span> in guts of WCR adults fed on soybean foliage in laboratory conditions. Wild-type (WT) populations are represented with open markers and dotted lines, whereas rotation-resistant (RR) populations are represented by filled markers and solid lines. Insects were dissected and gut samples were collected at 8, 24, 36, and 72 h after the initiation of the experiment. Time 0 corresponds to pretreatment beetles fed on corn silks. Markers represent the mean &#x000b1; SE of three independent samples each containing guts from 10 WCR. Lines bearing the same letter are not significantly different between populations at <italic> P </italic> &#x0003c; 0.05. (C) Quantitative RT-PCR expression analysis of cathepsin L-like (protease gene DvRS40; accession number AJ583508) in guts of WCR adults fed for 0, 8, and 24 h on soybean foliage. Time points for each population represent three replicates, each derived from five beetles. Values represent the band intensity generated by image analysis software for the results of quantitative RT-PCR relative to <span class="gene">EF-1a</span> (mean &#x000b1; SE). Bars bearing the same letter within each time point are not significantly different between populations at <italic> P </italic> &#x0003c; 0.05.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Early nodulin genes are induced in alfalfa <span class="trait">root</span> outgrowths elicited by auxin transport inhibitors.</h2>
			<h3>PMID:16594017</h3>
				

				
			<p>
				作者：Hirsch AM(1), Bhuvaneswari TV, Torrey JG, Bisseling T.
				<br/> 
				年份：1989 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizobium nod genes are essential for <span class="trait">root</span> hair deformation and cortical cell division, early stages in the development of nitrogen-fixing <span class="trait">root</span> nodules. Nod(-) mutants are unable to initiate nodules on legume roots. We observed that N-(1-naphthyl)phthalamic acid and 2,3,5-triiodobenzoic acid, compounds known to function as auxin transport inhibitors, induced nodule-like structures on alfalfa roots. The nodule-like structures (pseudonodules) were white, devoid of bacteria, and resembled nodules elicited by Rhizobium meliloti exopolysaccharide (exo) mutants at both the histological and molecular level. Two nodulin genes, <span class="gene">ENOD2</span> and Nms-30, were expressed. RNA isolated from the nodule-like structures hybridized to pGmENOD2, a soybean early nodulin cDNA clone. RNA isolated from roots did not hybridize. We determined by in vitro translations of total RNA that the alfalfa nodulin transcript Nms-30 was also expressed in the nodule-like structures. The late expressed nodulin genes, such as the leghemoglobin genes, were not transcribed. Because N-(1-naphthyl)phthalamic acid and 2,3,5-triiodobenzoic acid induce the development of nodules on alfalfa roots, we suggest that the auxin transport inhibitors mimic the <span class="trait">activity</span> of compound(s) made upon the induction of the Rhizobium nod genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of a chickpea (Cicer arietinum L.) NAC family gene, CarNAC5, which is both developmentally- and stress-regulated.</h2>
			<h3>PMID:19800808  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Peng H(1), Cheng HY, Yu XW, Shi QH, Zhang H, Li JG, Ma H.
				<br/> 
				年份：2009 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been documented that the plant-specific NAC (for NAM, ATAF1,2 and CUC2) transcription factors play an important role in plant development and stress responses. In this study, a chickpea NAC gene CarNAC5 (for Cicer arietinum L. NAC gene 5) was isolated from a cDNA library from chickpea leaves treated by polyethylene glycol (PEG). CarNAC5, as a single/low copy gene, contained three exons and two introns within genomic DNA sequence and encoded a polypeptide with 291 amino acids. CarNAC5 protein had a conserved NAC domain in the N-terminus and showed high similarity to other NACs, especially ATAF subgroup members. The CarNAC5:GFP fusion protein was localized in the nucleus of onion epidermal cells. Furthermore, CarNAC5 protein activated the reporter genes LacZ and <span class="gene">HIS3</span> in yeast. The transactivation <span class="trait">activity</span> was mapped to the C-terminal region. The transcripts of CarNAC5 appeared in many chickpea tissues including seedling leaves, stems, roots, flowers, seeds and pods, but mostly accumulated in flowers. Meanwhile, CarNAC5 was strongly expressed during seed maturation and in embryos of the early germinating seeds. It was also significantly induced by <span class="trait">drought,</span> heat, wounding, salicylic acid (SA), and indole-3-acetic acid (IAA) treatments. Our results suggest that CarNAC5 encodes a novel NAC-domain protein and acts as a transcriptional activator involved in plant developmental regulation and various stress responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Border sequences of Medicago truncatula CLE36 are specifically cleaved by endoproteases common to the extracellular fluids of Medicago and soybean.</h2>
			<h3>PMID:21633083  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Djordjevic MA(1), Oakes M, Wong CE, Singh M, Bhalla P, Kusumawati L, Imin N.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						CLE (CLAVATA3/ESR-related) peptides are developmental regulators that are secreted into the apoplast. Little is known about the role of the sequences that flank CLE peptides in terms of their biological <span class="trait">activity</span> or how they are targeted by proteases that are known to liberate the final active CLE peptides from their precursor sequences. The biological <span class="trait">activity</span> of Medicago truncatula CLE36, which possesses broadly conserved border sequences flanking the putative final active CLE36 peptide product, was assessed. Using in vitro <span class="trait">root</span> growth assays and an in vitro <span class="trait">root</span> and callus formation assay it is shown that CLE36 peptides of different lengths possess differential biological activities. Using mass spectrometry, Glycine max and Medicago extracellular fluids were each shown to possess an endoproteolytic <span class="trait">activity</span> that recognizes and cleaves at border sequences in a synthetic 31 amino acid CLE36 'propeptide bait' to liberate biologically active peptide products. Inhibitor studies suggest that a subtilisin, in combination with a carboxypeptidase, liberated and trimmed CLE36, respectively, to form biologically relevant 11-15 amino acid cleavage products. The 15 amino acid cleavage product is more biologically potent on Arabidopsis than shorter or longer CLE peptides. In situ hybridization shows that the soybean orthologue of CLE36 <span class="gene">(GmCLE34)</span> is expressed in the provascular tissue. The results suggest that secreted subtilisins can specifically recognize the border sequences of CLE36 propeptides and liberate biologically active cleavage products. These secreted proteases may affect the stability and biological <span class="trait">activity</span> of CLE peptides in the apoplast or be involved in CLE36 processing.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant delta-aminolevulinic acid dehydratase. Expression in soybean <span class="trait">root</span> nodules and evidence for a bacterial lineage of the Alad gene.</h2>
			<h3>PMID:8016269  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kaczor CM(1), Smith MW, Sangwan I, O'Brian MR.
				<br/> 
				年份：1994 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We isolated a soybean (Glycine max) cDNA encoding the heme and chlorophyll synthesis enzyme delta-aminolevulinic acid (ALA) dehydratase by functional complementation of an Escherichia coli <span class="gene">hemB</span> mutant, and we designated the gene Alad. ALA dehydratase was strongly expressed in nodules but not in uninfected roots, although Alad mRNA was only 2- to 3-fold greater in the symbiotic tissue. Light was not essential for expression of Alad in leaves of dark-grown etiolated plantlets as discerned by mRNA, protein, and enzyme <span class="trait">activity</span> levels; hence, its expression in subterranean nodules was not unique in that regard. The data show that soybean can metabolize the ALA it synthesizes in nodules, which argues in favor of tetrapyrrole formation by the plant host in that organ. Molecular phylogenetic analysis of ALA dehydratases from 11 organisms indicated that plant and bacterial enzymes have a common lineage not shared by animals and yeast. We suggest that plant ALA dehydratase is descended from the bacterial endosymbiont ancestor of chloroplasts and that the Alad gene was transferred to the nucleus during plant evolution.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis and expression profiling of the <italic> PIN </italic> auxin transporter gene family in soybean ( <italic> Glycine max </italic> )</h2>
			<h3>PMID:26572792</h3>
				

				
					<h3>PMCID:4647520</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Yongqin </given-names>  </name>  <address>  <email> wangyq@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chai </surname>  <given-names> Chenglin </given-names>  </name>  <address>  <email> chaic@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maupin </surname>  <given-names> Christine </given-names>  </name>  <address>  <email> clm3n7@mail.missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Annen </surname>  <given-names> Brad </given-names>  </name>  <address>  <email> bsahf7@mail.missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Division of Plant Sciences and National Center for Soybean Biotechnology, University of Missouri, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The plant phytohormone auxin controls many aspects of plant growth and development, which largely depends on its uneven distribution in plant tissues. Transmembrane proteins of the PIN family are auxin efflux facilitators. They play a key role in polar auxin transport and are associated with auxin asymmetrical distribution in plants. PIN genes have been characterized in several plant species, while comprehensive analysis of this gene family in soybean has not been reported yet. RESULTS: In this study, twenty-three members of the PIN gene family were identified in the soybean genome through homology searches. Analysis of chromosome distribution and phylogenetic relationships of the soybean PIN genes indicated nine pairs of duplicated genes and a legume specific subfamily. Organ/tissue expression patterns and promoter <span class="trait">activity</span> assays of the soybean PINs suggested redundant functions for most duplicated genes and complementary and tissue-specific functions during development for non-duplicated genes. The soybean PIN genes were differentially regulated by various abiotic stresses and phytohormone stimuli, implying crosstalk between auxin and abiotic stress signaling pathways. This was further supported by the altered auxin distribution under these conditions as revealed by DR5::GUS transgenic soybean hairy <span class="trait">root.</span> Our data indicates that <span class="gene">GmPIN9,</span> a legume-specific PIN gene, which was responsive to several abiotic stresses, might play a role in auxin re-distribution in soybean <span class="trait">root</span> under abiotic stress conditions. CONCLUSIONS: This study provided the first comprehensive analysis of the soybean PIN gene family. Information on phylogenetic relationships, gene structure, protein profiles and expression profiles of the soybean PIN genes in different tissues and under various abiotic stress treatments helps to identity candidates with potential roles in specific developmental processes and/or environmental stress conditions. Our study advances our understanding of plant responses to abiotic stresses and serves as a basis for uncovering the biological role of PIN genes in soybean development and adaption to adverse environments.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this study, twenty-three members of the <italic> PIN </italic> gene family were identified in the soybean genome through homology searches. Analysis of chromosome distribution and phylogenetic relationships of the soybean <italic> PIN </italic> genes indicated nine pairs of duplicated genes and a legume specific subfamily. Organ/tissue expression patterns and promoter <span class="trait">activity</span> assays of the soybean <italic> PIN </italic> s suggested redundant functions for most duplicated genes and complementary and tissue-specific functions during development for non-duplicated genes. The soybean <italic> PIN </italic> genes were differentially regulated by various abiotic stresses and phytohormone stimuli, implying crosstalk between auxin and abiotic stress signaling pathways. This was further supported by the altered auxin distribution under these conditions as revealed by <italic> DR5::GUS </italic> transgenic soybean hairy <span class="trait">root.</span> Our data indicates that <italic> <span class="gene">GmPIN9</span> </italic> , a legume-specific <italic> PIN </italic> gene, which was responsive to several abiotic stresses, might play a role in auxin re-distribution in soybean <span class="trait">root</span> under abiotic stress conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to investigate the tissue-specific expression pattern of <italic> <span class="gene">GmPIN</span> </italic> s, a heat-map showing the expression of 22 <italic> <span class="gene">GmPIN</span> </italic> s in seven tissues (shoot apical meristem, flower, green pod, leaf, <span class="trait">root,</span> <span class="trait">root</span> tip and nodule) was constructed (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a </xref> ) using the publically available soybean RNA-Seq data [ <xref ref-type="bibr" rid="CR24"> 24 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. No related data was found for <italic> <span class="gene">GmPIN9b</span> </italic> , and barely detectable or no expression was observed for <italic> <span class="gene">GmPIN8c</span> </italic> , <italic> <span class="gene">GmPIN8d</span> </italic> , <italic> <span class="gene">GmPIN9a</span> </italic> , and <italic> <span class="gene">GmPIN9c</span> </italic> . Gene expression was detected in at least one tissue for all other <italic> <span class="gene">GmPIN</span> </italic> s. Based on the RNA-Seq data, qRT-PCR (quantitative Reverse Transcription-Polymerase Chain Reaction) analysis was carried out to study the expression patterns of 17 <italic> <span class="gene">GmPIN</span> </italic> s in eight tissues, including <span class="trait">root,</span> stem, mature leaf, immature leaf, flower, pod, and seed at 14 and 21&#x000a0;days after flowering. Results of the qRT-PCR analyses were shown in Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5b </xref> . The expression data in Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5a </xref> and <xref rid="Fig5" ref-type="fig"> 5b </xref> revealed overall similar expression patterns (but at different expression levels) for duplicated <italic> <span class="gene">GmPIN</span> </italic> s or genes from the same PIN group, but very different expression profiles for <italic> <span class="gene">GmPIN</span> </italic> s from different PIN groups. Genes from the PIN1 and PIN3 groups were differentially expressed in all or most tissues tested. Transcripts for several genes from the PIN1 group were abundant in shoot apical meristem, <span class="trait">root</span> tip and stem, but barely detectable in nodule. <italic> <span class="gene">GmPIN3a</span> </italic> and <italic> <span class="gene">GmPIN3b</span> </italic> were highly expressed in flower and leaf, while <italic> <span class="gene">GmPIN3c</span> </italic> and <italic> <span class="gene">GmPIN3d</span> </italic> were expressed at relatively low levels in many tissues. <italic> <span class="gene">GmPIN2a</span> </italic> and <italic> <span class="gene">GmPIN2b</span> </italic> were predominantly expressed in soybean <span class="trait">root,</span> especially in the <span class="trait">root</span> tip, while low levels of <italic> <span class="gene">GmPIN2b</span> </italic> expression were also detected in flower and seed. Tissue-specific and relatively low-level gene expression was detected for genes from groups PIN5, PIN6, PIN8 and PIN9, e.g. <italic> <span class="gene">GmPIN5b</span> </italic> , <italic> <span class="gene">GmPIN8a</span> </italic> and <italic> <span class="gene">GmPIN8b</span> </italic> in leaf and flower, <italic> <span class="gene">GmPIN6a</span> </italic> and <italic> <span class="gene">GmPIN6b</span> </italic> in shoot apical meristem, green pod and <span class="trait">root,</span> and <italic> <span class="gene">GmPIN5a</span> </italic> in leaf and nodule. <italic> <span class="gene">GmPIN9d</span> </italic> was mainly expressed in <span class="trait">root,</span> seed and flower. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Tissue-specific expression profiles of <italic> <span class="gene">GmPIN</span> </italic> genes. ( <bold> a </bold> ) Expression profiles of <italic> <span class="gene">GmPIN</span> </italic> s in seven tissues according to RNA-Seq data [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. The Reads/Kb/Million (RPKM) normalized values were visualized as a heat map. ( <bold> b </bold> ) Quantitative RT-PCR analysis of selected <italic> <span class="gene">GmPIN</span> </italic> s in eight soybean tissues. Relative expression levels of <italic> <span class="gene">GmPIN</span> </italic> s compared to an ubiquitin gene <span class="gene">(Glyma20g27950.1)</span> were multiplied by 1,000 and shown as a heat map. Data represented the means of three biological and two technical replicates
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , a different combination of <italic> <span class="gene">GmPIN</span> </italic> expression was observed in each tissue types, such as members of <italic> <span class="gene">GmPIN1</span> </italic> , <italic> <span class="gene">GmPIN3</span> </italic> and <italic> <span class="gene">GmPIN6</span> </italic> in shoot apical meristem, and the homologous genes of PIN1 and PIN2 in <span class="trait">root</span> tip. This suggests that these <italic> <span class="gene">GmPIN</span> </italic> s work cooperatively. The dynamic temporal and spatial expression of these <italic> <span class="gene">GmPIN</span> </italic> s may be critical for the developmental process of each tissue. Though expressed in the same tissue and at the same time, the cell type-specific expression pattern, subcellular polarity and subcellular localization of those genes can be different, which has been elegantly evidenced by experiments in Arabidopsis [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. The similar expression patterns of duplicated genes or genes from the same PIN group indicate functional redundancy, while synergistic expression of <italic> <span class="gene">GmPIN</span> </italic> s from different group suggests functional complementation. Both might contribute to the flexibility and variation during soybean evolution. Functional redundancy could reduce the selection pressure on duplicated genes, which might lead to novel function, loss of function, or loss of expression (such as <italic> <span class="gene">GmPIN8c</span> </italic> , <italic> <span class="gene">GmPIN8d</span> </italic> , <italic> <span class="gene">GmPIN9a</span> </italic> , <italic> <span class="gene">GmPIN9b</span> </italic> , and <italic> <span class="gene">GmPIN9c</span> </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In order to explore the possible involvement of <italic> <span class="gene">GmPIN</span> </italic> s in response to water deficit conditions at the transcription level, expression profiles of 18 <italic> <span class="gene">GmPIN</span> </italic> s in soybean seedlings under <span class="trait">drought,</span> salt and dehydration treatments were analyzed by qRT-PCR (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6a </xref> ). Significant differential expressions (fold change&#x02009;&#x02265;&#x02009;2, p-value&#x02009;&#x02264;&#x02009;0.05) of 16 genes were detected under one or more treatments except for <italic> <span class="gene">GmPIN1a</span> </italic> and <italic> <span class="gene">GmPIN1c</span> </italic> . Seven <italic> <span class="gene">GmPIN</span> </italic> s responded to only one stress, four genes responded to two stresses, and five genes were regulated by all three stresses ( <italic> <span class="gene">GmPIN1d</span> </italic> , <italic> <span class="gene">GmPIN2b</span> </italic> , <italic> <span class="gene">GmPIN3a</span> </italic> , <italic> <span class="gene">GmPIN5b</span> </italic> , and <italic> <span class="gene">GmPIN9d</span> </italic> ) (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6b </xref> ). More <italic> <span class="gene">GmPIN</span> </italic> s were involved in response to <span class="trait">drought</span> (15 genes) than to salt stress (8 genes) and dehydration (7 genes). The number of <italic> <span class="gene">GmPIN</span> </italic> s down-regulated by salt or dehydration was larger than that of up-regulated. Of the 15 drought-responsive <italic> <span class="gene">GmPIN</span> </italic> s, five were regulated by both mild and moderate <span class="trait">drought</span> stresses. Interestingly, more genes were differentially regulated in <span class="trait">root</span> than in shoot upon either mild or moderate <span class="trait">drought</span> treatments (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6b </xref> ). All of the differentially expressed genes in shoot tissue were induced by <span class="trait">drought</span> (mild or moderate), and most of those in <span class="trait">root</span> were up-regulated by mild <span class="trait">drought</span> (8 out of 9), but down-regulated by moderate <span class="trait">drought</span> (5 out of 8). <fig id="Fig6"> <label> Fig. 6 </label> <caption> <p> Expression profiles of <italic> <span class="gene">GmPIN</span> </italic> s under <span class="trait">drought,</span> salt and dehydration conditions. ( <bold> a </bold> ) Fold changes of <italic> <span class="gene">GmPIN</span> </italic> gene expression under <span class="trait">drought,</span> salt and dehydration stresses. Data represented the means of three biological and two technical replicates. The asterisks indicate an absolute fold change&#x02009;&#x02265;&#x02009;2 and P value &#x0003c;0.05 by Student's t-test. ( <bold> b </bold> ) Venn diagram analysis of <italic> <span class="gene">GmPIN</span> </italic> gene expression under <span class="trait">drought,</span> salt and dehydration stresses. MS-L, mild <span class="trait">drought</span> stress shoots; MS-R, mild <span class="trait">drought</span> stress roots; SS-L, moderate <span class="trait">drought</span> stress shoots; SS-R, moderate <span class="trait">drought</span> stress roots; S-1&#x000a0;h, salt 1&#x000a0;hour; S-5&#x000a0;h, salt 5&#x000a0;hour; S-10&#x000a0;h, salt 10&#x000a0;hour; S-24&#x000a0;h, salt 24&#x000a0;hour; D-1&#x000a0;h, dehydration 1&#x000a0;hour; D-5&#x000a0;h, dehydration 5&#x000a0;hour; D-10&#x000a0;h, dehydration 10&#x000a0;hour
					</p>
				
					<p>&nbsp;&nbsp;
						Our data demonstrated most <italic> <span class="gene">GmPIN</span> </italic> s were responsive to certain water deficit conditions at the transcriptional level, generally in a tissue-specific, time- and stress magnitude-sensitive mode, suggesting that soybean responds to water deficit stress through a very complex regulation network, which necessitates coordinated regulation of most <italic> <span class="gene">GmPIN</span> </italic> s. As in soybean, many <italic> PIN </italic> genes in sorghum and maize were found to be transcriptionally responsive to various abiotic stresses, including salt and <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> ]. <italic> PIN </italic> genes might be commonly used for plants from different species to adapt to various abiotic stress conditions. Although a huge body of evidence has demonstrated the importance of PIN proteins in many developmental processes and in response to environmental signals such as light and gravity [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR2"> 2 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> ], there is limited information on their role in abiotic stresses and the underlying molecular mechanisms. Recently, it was reported that PIN2 in Arabidopsis was required to maintain <span class="trait">root</span> growth under alkaline stress conditions by modulate proton [H <sup> + </sup> ] secretion [ <xref ref-type="bibr" rid="CR29"> 29 </xref> ]. Another study indicated that the affected intracellular trafficking of PIN2 and PIN3 proteins in Arabidopsis might be responsible for the inhibited auxin polar transport under cold stress conditions [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The plant phytohormone ABA is pivotal for plant growth and development at its basal level and plant response to biotic and abiotic stresses at elevated levels [ <xref ref-type="bibr" rid="CR31"> 31 </xref> , <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. Emerging evidence indicates that auxin might play the role as mediator of environmental adaptions in plants [ <xref ref-type="bibr" rid="CR33"> 33 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. In addition, auxin feedback regulation of <italic> PIN </italic> gene expression at the transcriptional level is a key mechanism in auxin distribution in Arabidopsis [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR6"> 6 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. Quantitative RT-PCR analysis was carried out to investigate the effect of ABA and auxin on <italic> <span class="gene">GmPIN</span> </italic> gene expression in shoot and <span class="trait">root</span> tissues of soybean seedlings (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7a </xref> ). All of the 18 <italic> <span class="gene">GmPIN</span> </italic> s detected were differentially expressed upon ABA treatment, with 11 of them responsive in both shoot and <span class="trait">root,</span> and 7 in <span class="trait">root</span> only (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7b </xref> ). Seventeen out of 18 <italic> <span class="gene">GmPIN</span> </italic> s were responsive to indole-3-acetic acid (IAA) treatment, with 12 overlapping in shoot and <span class="trait">root.</span> While most of the <italic> <span class="gene">GmPIN</span> </italic> s were up-regulated in shoot and/or <span class="trait">root</span> by the two hormones, expression of <italic> <span class="gene">GmPIN5a</span> </italic> and <italic> <span class="gene">GmPIN5a</span> </italic> was repressed by ABA and IAA, and decreased expression of <italic> <span class="gene">GmPIN1a</span> </italic> , <italic> <span class="gene">GmPIN2b</span> </italic> and <italic> <span class="gene">GmPIN6b</span> </italic> was observed in shoot or <span class="trait">root</span> at some time point(s) after IAA treatment. <fig id="Fig7"> <label> Fig. 7 </label> <caption> <p> Expression patterns of <italic> <span class="gene">GmPIN</span> </italic> s under ABA and auxin treatments. ( <bold> a </bold> ) Fold changes of <italic> <span class="gene">GmPIN</span> </italic> gene expression under ABA and IAA treatments. Data represented the means of three biological and two technical replicates in qRT-PCR analysis. Asterisks indicated an absolute fold change&#x02009;&#x02265;&#x02009;2 and P value &#x0003c;0.05 by Student's t-test. ( <bold> b </bold> ) Venn diagram analysis of <italic> <span class="gene">GmPIN</span> </italic> gene expression under ABA and IAA treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to examine gene expression pattern in soybean <span class="trait">root,</span> promoters of 10 <italic> <span class="gene">GmPIN</span> </italic> s were cloned and histochemical detection of promoter <italic> ::GUS </italic> <span class="trait">activity</span> in transgenic soybean hairy roots was carried out (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8 </xref> ). Promoter <span class="trait">activity</span> was detected in soybean <span class="trait">root</span> for all of the cloned promoters in various patterns and with variable strengths, while genes from the same PIN group showed higher similarity. Strong GUS signals were observed in <span class="trait">root</span> cap and stele for <italic> <span class="gene">GmPIN1b</span> </italic> transgenic primary <span class="trait">root</span> and lateral <span class="trait">root</span> (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8a </xref> and <xref rid="Fig8" ref-type="fig"> 8b </xref> ), <italic> <span class="gene">GmPIN1c</span> </italic> promoter showed a similar pattern of <span class="trait">activity</span> but weaker than <italic> <span class="gene">GmPIN1b</span> </italic> (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5c </xref> and <xref rid="Fig8" ref-type="fig"> 8d </xref> ); while a much stronger <italic> <span class="gene">GmPIN1e::GUS</span> </italic> signal was found in stele than in <span class="trait">root</span> cap (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8e </xref> and <xref rid="Fig8" ref-type="fig"> 8f </xref> ). For <italic> <span class="gene">GmPIN2a</span> </italic> and <italic> <span class="gene">GmPIN2b</span> </italic> , promoter <span class="trait">activity</span> was high in <span class="trait">root</span> cap, meristem region, lateral <span class="trait">root</span> tip and lateral <span class="trait">root</span> primordia, and weak in stele (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8g </xref> - <xref rid="Fig8" ref-type="fig"> 8j </xref> ). The <italic> <span class="gene">GmPIN3a</span> </italic> promoter <span class="trait">activity</span> was strong in lateral <span class="trait">root</span> cap and vascular tissue of mature <span class="trait">root</span> region but weak in <span class="trait">root</span> cap and stele of the <span class="trait">root</span> tip region (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8k </xref> -8l); and a similar but weak staining pattern as that of <italic> <span class="gene">GmPIN3a</span> </italic> was found in <italic> <span class="gene">GmPIN3b::GUS</span> </italic> transgenic roots (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8m </xref> - <xref rid="Fig8" ref-type="fig"> 8n </xref> ). For <italic> <span class="gene">GmPIN6a</span> </italic> , only weak <span class="trait">activity</span> was detected in stele and lateral <span class="trait">root</span> tips (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8o </xref> - <xref rid="Fig8" ref-type="fig"> 8p </xref> ); while no GUS signals were observed in <italic> <span class="gene">GmPIN6b::GUS</span> </italic> transgenic roots after staining for 4&#x000a0;hours (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8q </xref> - <xref rid="Fig8" ref-type="fig"> 8r </xref> ), but a weak signal appeared in stele of the <span class="trait">root</span> tip region after overnight-staining (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8s </xref> ). For <italic> <span class="gene">GmPIN9d</span> </italic> , strong promoter <span class="trait">activity</span> was found in the vascular tissue of mature region of <span class="trait">root,</span> and weak <span class="trait">activity</span> was observed in <span class="trait">root</span> cap and stele of <span class="trait">root</span> tips (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8t </xref> -8u). <fig id="Fig8"> <label> Fig. 8 </label> <caption> <p> Promoter activities of <italic> <span class="gene">GmPIN</span> </italic> s in transgenic soybean hairy roots. At least 10 independent transgenic roots for each <italic> <span class="gene">GmPIN::GUS</span> </italic> construct were subjected to GUS staining. Images show representative results in <span class="trait">root</span> tips and lateral roots after four-hour staining for all roots except (S), which was stained overnight. ( <bold> a </bold> ) and ( <bold> b </bold> ), <italic> <span class="gene">GmPIN1b</span> </italic> ; ( <bold> c </bold> ) and ( <bold> d </bold> ), <italic> <span class="gene">GmPIN1c</span> </italic> ; ( <bold> e </bold> ) and ( <bold> f </bold> ), <italic> <span class="gene">GmPIN1e</span> </italic> ; ( <bold> g </bold> ) and ( <bold> h </bold> ), <italic> <span class="gene">GmPIN2a</span> </italic> ; ( <bold> i </bold> ) and ( <bold> j </bold> ), <italic> <span class="gene">GmPIN2b</span> </italic> ; ( <bold> k </bold> ) and ( <bold> l </bold> ), <italic> <span class="gene">GmPIN3a</span> </italic> ; ( <bold> m </bold> ) and ( <bold> n </bold> ), <italic> <span class="gene">GmPIN3b</span> </italic> ; ( <bold> o </bold> ) and ( <bold> p </bold> ), <italic> <span class="gene">GmPIN6a</span> </italic> ; ( <bold> q </bold> ), ( <bold> r </bold> ) and ( <bold> s </bold> ), <italic> <span class="gene">GmPIN6b</span> </italic> ; ( <bold> t </bold> ) and ( <bold> u </bold> ), <italic> <span class="gene">GmPIN9d</span> </italic> . Scale bar, 1&#x000a0;mm
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean roots, the overall similar patterns in promoter <span class="trait">activity</span> of the duplicated genes, such as <italic> <span class="gene">GmPIN1b</span> </italic> - <italic> 1c </italic> , <italic> <span class="gene">GmPIN2a</span> </italic> - <italic> 2b </italic> , and <italic> <span class="gene">GmPIN3a</span> </italic> - <italic> 3b </italic> , further suggest their function redundancy. Members from PIN1-3 groups and <italic> <span class="gene">GmPIN9d</span> </italic> may play very important roles in soybean <span class="trait">root</span> development due to their relatively strong expression in this tissue. Notably, the patterns of promoter <span class="trait">activity</span> in soybean roots for the detected <italic> <span class="gene">GmPIN</span> </italic> s were different from those of their orthologous genes in Arabidopsis and rice [ <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. The large gene number, various tissue-specific expression patterns and versatile regulatory modes under internal and external cues make it very complicated to explore the specific role of a certain <italic> <span class="gene">GmPIN</span> </italic> gene in soybean development. Further cellular and subcellular localization of their proteins in plant and gene-specific or a group of duplicated gene-specific knockout transgenic analysis may be helpful to unravel their functions.
					</p>
				
					<p>&nbsp;&nbsp;
						The auxin responsive <italic> DR5::GUS </italic> reporter system was used to monitor the auxin distribution in soybean <span class="trait">root</span> in response to water stress conditions and auxin and ABA treatments (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9a </xref> - <xref rid="Fig9" ref-type="fig"> 9e </xref> ). Promoter <span class="trait">activity</span> of <italic> <span class="gene">GmPIN9d</span> </italic> , which is a legume specific PIN9 group member and responded to various treatments at the transcriptional level (Figs.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6 </xref> and <xref rid="Fig7" ref-type="fig"> 7 </xref> ), was also investigated under these conditions (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9f </xref> - <xref rid="Fig9" ref-type="fig"> 9j </xref> ). Under control condition (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9a </xref> ), the maximum signal of <italic> DR5 </italic> promoter <span class="trait">activity</span> exhibited in the <span class="trait">root</span> cap region, strong signal was also found in stele, but signal was barely detected in the region approximately 0.5 to 1.5&#x000a0;mm from the <span class="trait">root</span> tip. Upon PEG (polyethylene glycol) (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9b </xref> ) and salt (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9c </xref> ) treatments, staining was strongly induced in stele and other cells. GUS signal increased in the mature <span class="trait">root</span> region under IAA treatment (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9d </xref> ), but decreased upon ABA treatment (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9e </xref> ). As for <italic> <span class="gene">GmPIN9</span> </italic> , compared to the no-treatment control (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9f </xref> ), promoter <span class="trait">activity</span> was greatly induced in <span class="trait">root</span> cap, stele and other cell types upon treatments with PEG and salt (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9g </xref> and <xref rid="Fig9" ref-type="fig"> 9h </xref> ). It was also induced in <span class="trait">root</span> cap and stele by IAA (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9i </xref> ), and slightly induced by ABA (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9j </xref> ). <fig id="Fig9"> <label> Fig. 9 </label> <caption> <p> Promoter activities of <italic> DR5 </italic> and <italic> <span class="gene">GmPIN9d</span> </italic> in transgenic soybean hairy roots under different treatments. Images show representative results of at least 5 biological replicates. Scale bar, 0.2&#x000a0;mm
					</p>
				
					<p>&nbsp;&nbsp;
						Changes in <italic> DR5 </italic> promoter <span class="trait">activity</span> reflected altered auxin distribution or signaling. <italic> DR5 </italic> and <italic> <span class="gene">GmPIN9</span> </italic> showed overall similar patterns in changes of promoter <span class="trait">activity</span> under PEG and salt treatments. This strongly indicates that <italic> <span class="gene">GmPIN9</span> </italic> might play a role in auxin re-distribution under these conditions, probably working together with other <italic> <span class="gene">GmPIN</span> </italic> s and auxin transporters from other gene families. Besides auxin transport, some auxin signaling components might also be involved in these responses. For example, some members of the auxin response factor transcription factor family in soybean were found transcriptionally regulated under water-deficit conditions [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, the soybean <italic> PIN </italic> auxin transporter gene family was comprehensively analyzed, including phylogeny, chromosomal distribution, gene structure, protein profiles, expression profiles in various tissues and under various abiotic stress conditions and hormone treatments, and promoter <span class="trait">activity</span> assay in transgenic soybean <span class="trait">root.</span> Eighteen out of the 23 members in the soybean <italic> PIN </italic> gene family exist as duplicated gene pairs originating from the glycine-specific whole-genome duplication event. High potential functional redundancy of duplicated genes or genes from the same PIN group was implied from high similarities of encoded amino acids, gene structure, tissue-specific expression pattern, and promoter <span class="trait">activity</span> in soybean <span class="trait">root.</span> However, the versatile differential expression modes under abiotic stress conditions indicated specific gene function at certain environmental conditions. The soybean <italic> PIN </italic> genes were responsive to a complex network of internal and external signals, and thereby controlled the auxin re-distribution, which finally will lead to the adjustment of growth and developmental processes to adapt to the ever-changing environments. Further in-depth functional analysis of the biological roles of <italic> <span class="gene">GmPIN</span> </italic> genes will enhance our understanding of plant response to abiotic stresses and aid in development of stress resistant crops.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean cultivar, Williams 82, was used in this study for the expression profiling analysis, promoter cloning and hairy <span class="trait">root</span> transformation. Plants were grown under the same greenhouse conditions as reported [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. For tissue/organ-specific expression profiling analysis, roots, mature leaves, immature leaves and stems were collected from <span class="gene">V1</span> stage seedlings, and flowers, young pods (0.5 to 2&#x000a0;cm in length), and seeds at 14 and 21&#x000a0;days after flowering were collected from R3 to R6 growth stages. The same methods were followed for <span class="trait">drought,</span> dehydration, salt (250&#x000a0;mM) and ABA (150&#x000a0;&#x003bc;M) treatments [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. IAA treatments were conducted using the same method as ABA treatments except 50&#x000a0;&#x003bc;M IAA was used instead of ABA. Shoots and roots were collected separately and tissues from three plants were pooled as one sample after <span class="trait">drought</span> treatments. Whole plants were collected at different time points after dehydration and salt treatments. For hormone treatments, shoots and roots from single plants were collected separately at 0.5&#x000a0;hour (h), 1&#x000a0;h, 3&#x000a0;h and 5&#x000a0;h after treatments. Samples were frozen immediately in liquid nitrogen after collection and kept at &#x02212;80&#x000a0;&#x000b0;C until use.
					</p>
				
					<p>&nbsp;&nbsp;
						GUS staining was performed according to the standard protocols [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ]. The staining time was 30&#x000a0;minutes and 4&#x000a0;hours for <italic> DR5::GUS </italic> and <italic> <span class="gene">GmPIN</span> </italic> promoter <italic> ::GUS </italic> transgenic roots, respectively. <span class="trait">Root</span> images were developed using a Leica S6 D stereomicroscope (Leica Microsystems, Switzerland) and Leica EC3 digital camera (Leica Microsystems, Switzerland).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The tobacco <span class="gene">Cel7</span> gene promoter is auxin-responsive and locally induced in nematode feeding sites of heterologous plants.</h2>
			<h3>PMID:20507511</h3>
				

				
			<p>
				作者：Wang X(1), Replogle A, Davis EL, Mitchum MG.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SUMMARY: Emerging evidence suggests that plant cell-wall-modifying enzymes induced by root-parasitic nematodes play important roles in feeding cell formation. We previously identified a tobacco endo-beta-1,4-glucanase (cellulase) gene, NtCel7, that was strongly induced in both root-knot and cyst nematode feeding cells. To characterize further the developmental and nematode-responsive regulation of NtCel7, we isolated the NtCel7 promoter and analysed its expression over a time course of nematode infection and in response to auxin, gibberellin, ethylene and sucrose in soybean and tomato hairy roots and in Arabidopsis containing the NtCel7 promoter fused to the beta-glucuronidase (GUS) reporter gene. Histochemical analyses of transgenic plant materials revealed that the NtCel7 promoter exhibited a unique organ-specific expression pattern during plant development suggestive of important roles for NtCel7 in both vegetative and reproductive growth. In all plant species tested, strong GUS expression was observed in <span class="trait">root</span> tips and lateral <span class="trait">root</span> primordia of uninfected roots with weaker expression in the <span class="trait">root</span> vasculature. Further analyses of transgenic Arabidopsis plants revealed expression in shoot and <span class="trait">root</span> meristems and the vasculature of most organs during plant development. We also determined that the NtCel7 promoter was induced by auxin, but not gibberellin, ethylene or sucrose. Moreover, strong GUS <span class="trait">activity</span> was observed in both cyst and root-knot nematode-induced feeding sites in transgenic roots of soybean, tomato and Arabidopsis. The conserved developmental and nematode-responsive expression of the NtCel7 promoter in heterologous plants indicates that motifs of this regulatory element play a fundamental role in regulating NtCel7 gene expression within nematode feeding sites and that this regulation may be mediated by auxin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Arabinosylation Modulates the Growth-Regulating <span class="trait">Activity</span> of the Peptide Hormone CLE40a from Soybean.</h2>
			<h3>PMID:28943356  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Corcilius L(1), Hastwell AH(2), Zhang M(2), Williams J(1), Mackay JP(3), Gresshoff PM(2), Ferguson BJ(4), Payne RJ(5).
				<br/> 
				年份：2017 Nov 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Small post-translationally modified peptide hormones mediate crucial developmental and regulatory processes in plants. CLAVATA/ENDOSPERM-SURROUNDING REGION (CLE) genes are found throughout the plant kingdom and encode for 12-13 amino acid peptides that must often undergo post-translational proline hydroxylation and glycosylation with O-β1,2-triarabinose moieties before they become functional. Apart from a few recent examples, a detailed understanding of the structure and function of most CLE hormones is yet to be uncovered. This is mainly owing to difficulties in isolating mature homogeneously modified CLE peptides from natural plant sources. In this study, we describe the efficient synthesis of a synthetic Araf3Hyp glycosylamino acid building block that was used to access a hitherto uninvestigated CLE hormone from soybean called <span class="gene">GmCLE40a.</span> Through the development and implementation of a novel in vivo <span class="trait">root</span> growth assay, we show that the synthetic triarabinosylated glycopeptide suppresses primary <span class="trait">root</span> growth in this important crop species.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The PhyR-sigma(EcfG) signalling cascade is involved in stress response and symbiotic efficiency in Bradyrhizobium japonicum.</h2>
			<h3>PMID:19555458  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gourion B(1), Sulser S, Frunzke J, Francez-Charlot A, Stiefel P, Pessi G, Vorholt JA, Fischer HM.
				<br/> 
				年份：2009 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						PhyR is an unusual type of response regulator consisting of a receiver domain and an extracytoplasmic function (ECF) sigma factor-like domain. It was recently described as a master regulator of general stress response in Methylobacterium extorquens. Orthologues of this regulator are present in essentially all free-living Alphaproteobacteria. In most of them, <span class="gene">phyR</span> is genetically closely linked to a gene encoding an ECF sigma factor. Here, we investigate the role of these two regulators in the soybean symbiont Bradyrhizobium japonicum USDA110. Using deletion mutants and phenotypic assays, we showed that PhyR and the ECF sigma factor sigma(EcfG) are involved in heat shock and desiccation <span class="trait">resistance</span> upon carbon starvation. Both mutants had symbiotic defects on the plant hosts Glycine max (soybean) and Vigna radiata (mungbean). They induced fewer nodules than the wild type and these nodules were smaller, less pigmented, and their specific nitrogenase <span class="trait">activity</span> was drastically reduced 2 or 3 weeks after inoculation. Four weeks after infection, soybean nodule development caught up to a large extent whereas most mungbean nodules remained defective even 5 weeks after infection. Remarkably, both mutants triggered aberrant nodules on the different host plants with ectopically emerging roots. Microarray analysis revealed that PhyR and sigma(EcfG) control congruent regulons suggesting both regulators are part of the same signalling cascade. This finding was further substantiated by in vitro protein-protein interaction studies which are in line with a partner-switching mechanism controlling gene regulation triggered by phosphorylation of PhyR. The large number of genes of unknown function present in the PhyR/sigma(EcfG) regulon and the conspicuous symbiotic phenotype suggest that these regulators are involved in the Bradyrhizobium-legume interaction via yet undisclosed mechanisms.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The miR172c-NNC1 module modulates <span class="trait">root</span> plastic development in response to salt in soybean</h2>
			<h3>PMID:29191158</h3>
				
					<h3>PMCID:5709930</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Sahito </surname>  <given-names> Zulfiqar Ali </given-names>  </name>  <address>  <email> sahito_zulfiqarali@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Lixiang </given-names>  </name>  <address>  <email> wanglixiang2@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Zhengxi </given-names>  </name>  <address>  <email> szx-1988@163.com </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Qiqi </given-names>  </name>  <address>  <email> 648443557@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xingke </given-names>  </name>  <address>  <email> 464046562@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Qiong </given-names>  </name>  <address>  <email> 1241110474@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ullah </surname>  <given-names> Ihteram </given-names>  </name>  <address>  <email> ihterampbg@mails.ucas.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Tong </surname>  <given-names> Yiping </given-names>  </name>  <address>  <email> yptong@genetics.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Xia </given-names>  </name>  <address>  <email> xli@mail.hzau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1790 4137 </institution-id>  <institution-id institution-id-type="GRID"> grid.35155.37 </institution-id>  <institution> State Key Laboratory of Agricultural Microbiology,  </institution>  <institution> College of Plant Science and Technology, Huazhong Agricultural University,  </institution>  </institution-wrap> Wuhan, 430070 People&#x02019;s Republic of China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0596 2989 </institution-id>  <institution-id institution-id-type="GRID"> grid.418558.5 </institution-id>  <institution> Institute of Genetics and Developmental Biology, Chinese Academy of Sciences,  </institution>  </institution-wrap> 286 Huaizhong Road, Shijiazhuang, Hebei 050021 People&#x02019;s Republic of China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1797 8419 </institution-id>  <institution-id institution-id-type="GRID"> grid.410726.6 </institution-id>  <institution> University of Chinese Academy of Sciences,  </institution>  </institution-wrap> Beijing, 100049 People&#x02019;s Republic of China  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000000119573309 </institution-id>  <institution-id institution-id-type="GRID"> grid.9227.e </institution-id>  <institution> State Key Laboratory of Plant Cell and Chromosome Engineering,  </institution>  <institution> Institute of Genetic and Developmental Biology, Chinese Academy of Sciences,  </institution>  </institution-wrap> Beijing, 100101 China  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Relative transcript levels of <italic> THI </italic> , <italic> RD22 </italic> , <italic> NCED3 </italic> and <italic> SALT3 </italic> in transgenic hairy roots. Seven day-old composite transgenic plants were treated with 75&#x000a0;mM NaCl and the <span class="trait">root</span> samples were harvested for RNA extraction. Relative transcript levels of <italic> THI </italic> , <italic> RD22 </italic> , <italic> NCED3 </italic> and <italic> SALT3 </italic> in <italic> 35S::NNC1m6 </italic> and <italic> RNAi-NNC1 </italic> transgenic roots compared to empty vector roots under normal and salt stress conditions. <italic> <span class="gene">GmELF1b</span> </italic> was used as the internal control. The data are the means &#x000b1; SD of three replicates, and asterisks indicate a significant difference compared with the corresponding controls ( <italic> *P </italic> &#x02009;&#x0003c;&#x02009;0.05; <italic> **P </italic> &#x02009;&#x0003c;&#x02009;0.01; <italic> ***P </italic> &#x02009;&#x0003c;&#x02009;0.005)
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A high-affinity cbb3-type cytochrome oxidase terminates the symbiosis-specific respiratory chain of Bradyrhizobium japonicum.</h2>
			<h3>PMID:8626278  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preisig O(1), Zufferey R, Thöny-Meyer L, Appleby CA, Hennecke H.
				<br/> 
				年份：1996 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It has been a long-standing hypothesis that the endosymbiotic rhizobia (bacteroids) cope with a concentration of 10 to 20 nM free O2 in legume <span class="trait">root</span> nodules by the use of a specialized respiratory electron transport chain terminating with an oxidase that ought to have a high affinity for O2. Previously, we suggested that the microaerobically and anaerobically induced fixNOQP operon of Bradyrhizobium japonicum might code for such a special oxidase. Here we report the biochemical characteristics of this terminal oxidase after a 27-fold enrichment from membranes of anaerobically grown B. japonicum wild-type cells. The purified oxidase has TMPD (N,N,N',N'-tetramethyl-p-phenylenediamine) oxidase <span class="trait">activity</span> as well as cytochrome c oxidase <span class="trait">activity.</span> N-terminal amino acid sequencing of its major constituent subunits confirmed that presence of the fixN,fixO, and <span class="gene">fixP</span> gene products. FixN is a highly hydrophobic, heme B-binding protein. FixO and FixP are membrane-anchored c-type cytochromes (apparent Mrs of 29,000 and 31,000, respectively), as shown by their peroxidase activities in sodium dodecyl sulfate-polyacrylamide gels. All oxidase properties are diagnostic for it to be a member of the cbb3-type subfamily of heme-copper oxidases. The FixP protein was immunologically detectable in membranes isolated from <span class="trait">root</span> nodule bacteroids, and 85% of the total cytochrome c oxidase <span class="trait">activity</span> in bacteroid membranes was contributed by the cbb3-type oxidase. The Km values for O2 of the purified enzyme and of membranes from different B. japonicum wild-type and mutant strains were determined by a spectrophotometric method with oxygenated soybean leghemoglobin as the sole O2 delivery system. The derived Km value for O2 of the cbb3-type oxidase in membranes was 7 nM, which is six- to eightfold lower than that determined for the aerobic aa3-type cytochrome c oxidase. We conclude that the cbb3-type oxidase supports microaerobic respiration in endosymbiotic bacteroids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic white clover. Studies with the auxin-responsive promoter, GH3, in <span class="trait">root</span> gravitropism and lateral <span class="trait">root</span> development.</h2>
			<h3>PMID:11539555  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Larkin PJ(1), Gibson JM, Mathesius U, Weinman JJ, Gartner E, Hall E, Tanner GJ, Rolfe BG, Djordjevic MA.
				<br/> 
				年份：1996 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We report improved method for white clover (Trifolium repens) transformation using Agrobacterium tumefaciens. High efficiencies of transgenic plant production were achieved using cotyledons of imbibed mature seed. Transgenic plants were recovered routinely from over 50% of treated cotyledons. The bar gene and phosphinothricin selection was shown to be a more effective selection system than nptII (kanamycin selection) or <span class="gene">aadA</span> (spectinomycin selection). White clover was transformed with the soybean auxin responsive promoter, GH3, fused to the GUS gene (beta-glucuronidase) to study the involvement of auxin in <span class="trait">root</span> development. Analysis of 12 independent transgenic plants showed that the location and pattern of GUS expression was consistent but the levels of expression varied. The level of GH3:GUS expression in untreated plants was enhanced specifically by auxin-treatment but the pattern of expression was not altered. Expression of the GH3:GUS fusion was not enhanced by other phytohormones. A consistent GUS expression pattern was evident in untreated plants presumably in response to endogenous auxin or to differences in auxin sensitivity in various clover tissues. In untreated plants, the pattern of GH3:GUS expression was consistent with physiological responses which are regarded as being auxin-mediated. For the first time it is shown that localised spots of GH3:GUS <span class="trait">activity</span> occurred in <span class="trait">root</span> cortical tissue opposite the sites where lateral roots subsequently were initiated. Newly formed lateral roots grew towards and through these islands of GH3:GUS expression, implying the importance of auxin in controlling lateral <span class="trait">root</span> development. Similarly, it is demonstrated for the first time that gravistimulated roots developed a rapid (within 1 h) induction of GH3:GUS <span class="trait">activity</span> in tissues on the non-elongating side of the responding <span class="trait">root</span> and this induction occurred concurrently with <span class="trait">root</span> curvature. These transgenic plants could be useful tools in determining the physiological and biochemical changes that occur during auxin-mediated responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rust Secreted Protein Ps87 Is Conserved in Diverse Fungal Pathogens and Contains a RXLR-like Motif Sufficient for Translocation into Plant Cells</h2>
			<h3>PMID:22076138</h3>
				

				
					<h3>PMCID:3208592</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gu </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kale </surname>  <given-names> Shiv D. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Qinhu </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dinghe </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Qiaona </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Hua </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meng </surname>  <given-names> Yuling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Zhensheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tyler </surname>  <given-names> Brett M. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> Weixing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Effector proteins of biotrophic plant pathogenic fungi and oomycetes are delivered into host cells and play important roles in both disease development and disease <span class="trait">resistance</span> response. How obligate fungal pathogen effectors enter host cells is poorly understood. The Ps87 gene of Puccinia striiformis encodes a protein that is conserved in diverse fungal pathogens. Ps87 homologs from a clade containing <span class="trait">rust</span> fungi are predicted to be secreted. The aim of this study is to test whether Ps87 may act as an effector during Puccinia striiformis infection. METHODOLOGY/PRINCIPAL FINDINGS: Yeast signal sequence trap assay showed that the <span class="trait">rust</span> protein Ps87 could be secreted from yeast cells, but a homolog from Magnaporthe oryzae that was not predicted to be secreted, could not. Cell re-entry and protein uptake assays showed that a region of Ps87 containing a conserved RXLR-like motif [K/R]RLTG was confirmed to be capable of delivering oomycete effector <span class="gene">Avr1b</span> into soybean leaf cells and carrying GFP into soybean <span class="trait">root</span> cells. Mutations in the Ps87 motif (KRLTG) abolished the protein translocation ability. CONCLUSIONS/SIGNIFICANCE: The results suggest that Ps87 and its secreted homologs could utilize similar protein translocation machinery as those of oomycete and other fungal pathogens. Ps87 did not show direct suppression <span class="trait">activity</span> on plant defense responses. These results suggest Ps87 may represent an "emerging effector" that has recently acquired the ability to enter plant cells but has not yet acquired the ability to alter host physiology.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Yeast signal sequence trap assay showed that the <span class="trait">rust</span> protein Ps87 could be secreted from yeast cells, but a homolog from <italic> Magnaporthe oryzae </italic> that was not predicted to be secreted, could not. Cell re-entry and protein uptake assays showed that a region of Ps87 containing a conserved RXLR-like motif [K/R]RLTG was confirmed to be capable of delivering oomycete effector <span class="gene">Avr1b</span> into soybean leaf cells and carrying GFP into soybean <span class="trait">root</span> cells. Mutations in the Ps87 motif (KRLTG) abolished the protein translocation ability.
					</p>
				
					<p>&nbsp;&nbsp;
						Two predicted RXLR-like translocation motifs, RRLQ and KRLTG, were identified based on functional RXLR variants validated in the <italic> P. sojae </italic> effector <span class="gene">Avr1b</span> <xref ref-type="bibr" rid="pone.0027217-Kale1"> [24] </xref> , suggesting that Ps87 might be able to enter plant cells. To validate the prediction, the N-terminus of Ps87, encompassing the two motifs, was inserted between the secretory leader and the C-terminus of Avr1b, replacing the RXLR domain of <span class="gene">Avr1b</span> ( <xref ref-type="fig" rid="pone-0027217-g003"> Figure 3A </xref> ). Transient expression of the <italic> Ps87 </italic> - <italic> <span class="gene">Avr1b</span> </italic> fusion in soybean leaves carrying <italic> Rps1b </italic> triggered cell death, indicating that the N-terminus of Ps87 is capable of translocating the <span class="gene">Avr1b</span> C-terminal reporter back into soybean leaf cells ( <xref ref-type="fig" rid="pone-0027217-g003"> Figure 3B </xref> ). The RRLQ and DDDEER motifs in Ps87 appeared quite similar to the RXLR-dEER bipartite protein translocation motif of oomycete effectors, but the alanine substitutions in these two motifs did not abolish the cell killing <span class="trait">activity</span> of the <span class="gene">Avr1b</span> reporter as measured by the reduction of beta-glucuronidase (GUS) positive spots. However, mutations in the motif KRLTG did lead to reduced Avr1b-mediated cell death in <italic> Rps1b </italic> leaves, suggesting that KRLTG was required for re-entry of the fusion protein. The interaction of the leaderless Ps87 fused <span class="gene">Avr1b</span> with Rps1b soybean was unaffected by mutations in the KRLTG motif, indicating that the KRLTG motif present in the Ps87 N-terminus was not involved in the interaction between <span class="gene">Avr1b</span> and Rps1b. KRLTG is well conserved in other fungal homologs, typically as RRLTG ( <xref ref-type="fig" rid="pone-0027217-g001"> Figure 1A </xref> ). Another conserved motif, RGQ, did not appear to be involved in translocation.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Structure of recombined <span class="gene">Avr1b</span> protein. Predicted Ps87 translocation domain (in bold) was fused to <span class="gene">Avr1b</span> by replacement of the <span class="gene">Avr1b</span> RXLR-dEER domain. Residues subjected to mutagenesis are labeled in red. (B) Cell re-entry activities of Ps87 fused <span class="gene">Avr1b</span> proteins carrying (s) or lacking (m) the secretory leader and/or with mutations in RRLQ (RRLQ to aaaa), DDDEER (DYCVDKWDEMMKER to aYCVaKWaaMMKaa), TG-RGQ (TGKPRGQ to aaKPaaa) and KRLTG (KRLTG to aaaaa) motifs. Effector re-entry activities were measured by double-barreled particle bombardment in which parallel bombardments with a beta-glucuronidase (GUS) reporter gene, with and without the <italic> <span class="gene">Avr1b</span> </italic> fusion were compared for the reduction of cell death in the presence of <span class="trait">resistance</span> gene <italic> Rps1b </italic> (cultivar L77-1863) or its absence (rps; cultivar Williams). Averages and standard deviations shown are from 14&#x02013;16 pairs of bombardments. <italic> P </italic> values were calculated using the Wilcoxon rank sum test.
					</p>
				
					<p>&nbsp;&nbsp;
						A <span class="trait">root</span> uptake assay employing green fluorescent protein (GFP) fused to candidate effectors was previously employed to visualize specific translocation <span class="trait">activity</span> of the effectors <xref ref-type="bibr" rid="pone.0027217-Dou1"> [23] </xref> , <xref ref-type="bibr" rid="pone.0027217-Kale1"> [24] </xref> . The predicted host-targeting signals of Ps87 and <span class="gene">Avr1b</span> were fused to GFP, expressed in <italic> Escherichia coli </italic> and partially purified ( <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4J </xref> ). As shown in <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4 </xref> , both the RXLR domain of <span class="gene">Avr1b</span> and the N-terminus of Ps87 containing KRLTG carried the GFP fusion protein into soybean <span class="trait">root</span> tip cells in the absence of pathogens ( <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4C, D, F, G </xref> ). The GFP protein accumulated inside of several layers of the <span class="trait">root</span> tip cells, whereas buffer and GFP alone did not produce any fluorescence, confirming that uptake was specific ( <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4A, B </xref> ). The characteristic accumulation of GFP in the nuclei of the roots cells verifies that the GFP is located inside the cells. Furthermore, the GFP signal remained in the cytoplasm and nuclei rather than the intercellular spaces when the soybean <span class="trait">root</span> tissues were plasmolyzed with 3 M NaCl ( <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4I </xref> ). Mutations in the KRLTG motif of Ps87 as well as the RXLR motifs of <span class="gene">Avr1b</span> abolished accumulation of GFP fusion protein signals inside the soybean <span class="trait">root</span> cells ( <xref ref-type="fig" rid="pone-0027217-g004"> Figure 4E, H </xref> ). Therefore the results of the GFP fusion protein uptake assay confirm the results from the bombardment experiments.
					</p>
				
					<p>&nbsp;&nbsp;
						Entry of wild-type and mutant Ps87-GFP or Avr1b-GFP fusion proteins into soybean <span class="trait">root</span> cells. The paired light micrographs and fluorescence optical sections are from the same <span class="trait">root</span> tips in each case. All expressed GFP proteins fluoresce normally under UV illumination. Fluorescent micrographs were taken with the same photographic exposure in every case. Left panels, fluorescence images; right panels, light micrographs. (A&#x02013;B) The buffer (A) and GFP (B) alone. Scale bars, 200 &#x000b5;m. (C) The <span class="gene">Avr1b</span> N-terminal uptake signal fused GFP. Scale bars, 100 &#x000b5;m. (D) The <span class="gene">Avr1b</span> N-terminal uptake signal fused GFP. Scale bars, 200 &#x000b5;m. (E) The <span class="gene">Avr1b</span> rslr <sup> &#x02212; </sup> and rflr <sup> &#x02212; </sup> nonfunctional mutant. Scale bars, 50 &#x000b5;m. (F) The Ps87 N-terminal uptake signal fused GFP. Scale bars, 100 &#x000b5;m. (G) The Ps87 N-terminal uptake signal fused GFP. Scale bars, 50 &#x000b5;m. (H) The Ps87 krltg <sup> &#x02212; </sup> nonfunctional mutant. Scale bars, 50 &#x000b5;m. (I) The Ps87 N-terminal uptake signal fused GFP. Photograph was taken 15 min after plasmolysis of the roots with 3 M NaCl. Scale bars, 100 &#x000b5;m. (J) Protein gel electrophoresis analysis of GFP fusion proteins partially purified from <italic> E. coli </italic> cells: lane 1, GFP alone; lane 2, GFP fused to the N-terminus of mature wild-type <span class="gene">Avr1b</span> protein; lane 3, same as lane 2 with RXLR motifs mutation; lane 4, GFP fused to the N-terminus of mature wild-type Ps87 protein; lane 5, same as lane 4 with KRLTG mutation; The left lane contained molecular mass markers (M). Size markers: # 50 kDa; &#x0002a; 40 kDa; &#x0002b;30 kDa.
					</p>
				
					<p>&nbsp;&nbsp;
						Two lines of evidence support the hypothesis that Ps87 can enter plant cells via an RXLR-like motif, [K/R]RLTG, which forms part of the conserved central core of the Ps87 protein family. In one series of experiments we used particle bombardment assays to test the ability of Ps87 to carry <span class="gene">Avr1b</span> protein back into soybean leaf cells. In the second series of experiments, we showed that Ps87 could deliver GFP into soybean <span class="trait">root</span> cells. Both assays revealed that the motif KRLTG, which is conserved in the other family members as RRLTG, is required for the cell entry <span class="trait">activity</span> of Ps87. We note that these experiments were done in soybean, where these assays are relatively easy to carry out. Future validation using wheat leaves will be important to confirm this <span class="trait">activity</span> of Ps87.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:FvSNF1, the sucrose non-fermenting protein kinase gene of Fusarium virguliforme, is required for cell-wall-degrading enzymes expression and sudden death syndrome development in soybean.</h2>
			<h3>PMID:28132080</h3>
				

				
			<p>
				作者：Islam KT(1)(2), Bond JP(1), Fakhoury AM(3).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium virguliforme is a soil-borne pathogenic fungus that causes sudden death syndrome (SDS) in soybean. Its pathogenicity is believed to require the <span class="trait">activity</span> of cell-wall-degrading enzymes (CWDEs). The sucrose non-fermenting protein kinase 1 gene (SNF1) is a key component of the glucose de-repression pathway in yeast, and a regulator of gene expression for CWDEs in some plant pathogenic fungi. To elucidate the functional role of the <span class="gene">SNF1</span> homolog in F. virguliforme, FvSNF1 was disrupted using a split-marker strategy. Disruption of FvSNF1 in F. virguliforme abolishes galactose utilization and causes poor growth on xylose, arabinose and sucrose. However, the resulting Fvsnf1 mutant grew similar to wild-type and ectopic transformants on glucose, fructose, maltose, or pectin as the main source of carbon. The Fvsnf1 mutant displayed no expression of the gene-encoding galactose oxidase (GAO), a secretory enzyme that catalyzes oxidation of D-galactose. It also exhibited a significant reduction in the expression of several CWDE-coding genes in contrast to the wild-type strain. Greenhouse pathogenicity assays revealed that the Fvsnf1 mutant was severely impaired in its ability to cause SDS on challenged soybean plants. Microscopy and microtome studies on infected roots showed that the Fvsnf1 mutant was defective in colonizing vascular tissue of infected plants. Cross and longitudinal sections of infected roots stained with fluorescein-labeled wheat germ agglutinin and Congo red showed that the Fvsnf1 mutant failed to colonize the xylem vessels and phloem tissue at later stages of infection. Quantification of the fungal biomass in inoculated roots further confirmed a reduced colonization of roots by the Fvsnf1 mutant when compared to the wild type. These findings suggest that FvSNF1 regulates the expression of CWDEs in F. virguliforme, thus affecting the virulence of the fungus on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Fe-deficient conditions on Fe uptake and utilization in P-efficient soybean.</h2>
			<h3>PMID:28012287  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Qiu W(1), Dai J(1), Wang N(1), Guo X(1), Zhang X(1), Zuo Y(2).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus (P)-efficient soybean (Glycine max) plants absorb and utilize P with high efficiency. To investigate the effects of iron (Fe)-deficient conditions on the absorption and utilization of Fe in P-efficient soybean plants, two soybean cultivars with different P efficiency, the 03-3 (P-efficient variety) and Bd-2 (P-inefficient variety), were used in this study. The two soybean cultivars were grown in nutrient solution containing Fe concentrations of 0 (Fe0), 20 (Fe20), 40 (Fe40), or 80 (Fe80) μM for 7 days. The Fe reductase <span class="trait">activity</span> of roots was higher in 03-3 plants grown under the Fe0, Fe20, and Fe40 treatments than in Bd-2 plants and the total Fe uptake was greater in 03-3 plants under the Fe40 treatment. <span class="gene">GmFRD3a</span> was much more highly expressed in the stem of 03-3 than in that of Bd-2, and significantly more iron was transported to 03-3 plant shoots during Fe0 treatment. <span class="trait">Chlorosis</span> in young leaves caused by Fe deficiency under the Fe0 and Fe20 treatments was alleviated by increased Fe concentration in shoots. Increased levels of active Fe in young 03-3 leaves under Fe-deprivation conditions (Fe0) and maintenance of stable Fe concentrations in 03-3 shoots subjected to Fe20, Fe40, and Fe80 treatments suggested that the P-efficient 03-3 cultivar is also Fe-efficient. It is suggested that 03-3 soybean cultivar should be a good resource for application to farm field.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation and inheritance of iron reductase <span class="trait">activity</span> in the roots of common bean ( <italic> Phaseolus vulgaris </italic> L.) and association with seed iron accumulation QTL</h2>
			<h3>PMID:20923552</h3>
				
					<h3>PMCID:3095315</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Blair </surname>  <given-names> Matthew W </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> m.blair@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Knewtson </surname>  <given-names> Sharon JB </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> sknewston@bcm.tmc.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Astudillo </surname>  <given-names> Carolina </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> c.astudillo@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Li </surname>  <given-names> Chee-Ming </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> cmli@bcm.tmc.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Fernandez </surname>  <given-names> Andrea C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> a.c.fernandez@cgiar.org </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Grusak </surname>  <given-names> Michael A </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mgrusak@bcm.tmc.edu </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 10 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Surprisingly, neither of the QTL identified in this study for iron reductase <span class="trait">activity</span> aligned with the positions of the markers for <italic> Pv </italic> FRO orthologs that were successfully mapped on chromosomes b06 and b07. This may have been expected, since in other research the control of iron reductase <span class="trait">activity</span> has been postulated to be under the control of a shoot to <span class="trait">root</span> signal that induces reductase expression [ <xref ref-type="bibr" rid="B49"> 49 </xref> ]. It was not surprising that we found two map locations for the FRO orthologs since iron reductases are known to be an eight-member gene family in <italic> Arabidopsis thaliana </italic> , where paralogs exist and some genes have different functionality and expression patterns in roots versus several diverse above ground organs [ <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Although the family has not been characterized in soybean we found one homologous copy of FRO on chromosome <span class="gene">Gm10</span> with the equivalent positions in common bean being on chromosome b07 where the CAPS marker mapped to. Medicago synteny confirmed this position.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Roots Retain the Seed Urease Isozyme Synthesized during Embryo Development.</h2>
			<h3>PMID:16667765</h3>
				

				
			<p>
				作者：Torisky RS(1), Polacco JC.
				<br/> 
				年份：1990 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Roots of young soybean (Glycine max [L.] Merr.) plants (up to 25 days old) contain two distinct urease isozymes, which are separable by hydroxyapatite chromatography. These two urease species (URE1 and URE2) differ in: (a) electrophoretic mobility in native gels, (b) pH dependence, and (c) recognition by a monoclonal antibody specific for the seed ("embryo-specific") urease. By these parameters <span class="trait">root</span> URE1 urease is similar to the abundant embryo-specific urease isozyme, while <span class="trait">root</span> URE2 resembles the "ubiquitous" urease which has previously been found in all soybean tissues examined (leaf, embryo, seed coat, and cultured cells). The embryo-specific and ubiquitous urease isozymes are products of the Eu1 and <span class="gene">Eu4</span> structural genes, respectively. Roots of the eu1-sun/eu1-sun genotype, which lacks the embryo-specific urease (i.e. ;seed urease-null'), contain no URE1 urease <span class="trait">activity.</span> Roots of eu4/eu4, which lacks ubiquitous urease, lack the URE2 (leaflike) urease <span class="trait">activity.</span> From these genetic and biochemical criteria, then, we conclude that URE1 and URE2 are the embryo-specific and ubiquitous ureases, respectively. Adventitious roots generated from cuttings of any urease genotype lack URE1 <span class="trait">activity.</span> In seedling roots the seedlike (URE1) <span class="trait">activity</span> declines during development. Roots of 3-week-old plants contain 5% of the total URE1 <span class="trait">activity</span> of the radicle of 4-day-old seedlings, which, in turn, has approximately the same urease <span class="trait">activity</span> level as the dormant embryonic axis. The embryo-specific urease incorporates label from [(35)S]methionine during embryo development but not during germination, indicating that there is no de novo synthesis of the embryo-specific (URE1) urease in the germinating <span class="trait">root.</span> We conclude that the seedlike urease (URE1) found in roots of young soybean plants is a remnant of the Eu1-encoded, abundant, embryo-specific urease which accumulates in the embryonic <span class="trait">root</span> axis during seed development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A single amino acid substitution in soybean VSPalpha increases its acid phosphatase <span class="trait">activity</span> nearly 20-fold.</h2>
			<h3>PMID:15278453  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Leelapon O(1), Sarath G, Staswick PE.
				<br/> 
				年份：2004 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] contains two proteins called vegetative storage proteins (VSPs) that function as temporary storage reserves, but are also closely related to plant acid phosphatases of the haloacid dehalogenase (HAD) superfamily. This study examined the biochemical basis for the relatively low catalytic <span class="trait">activity</span> previously reported for these VSPs. The specific <span class="trait">activity</span> of purified recombinant VSPalpha on <span class="gene">GMP</span> was about 40-fold lower than for a related soybean <span class="trait">root</span> nodule acid phosphatase (APase), which had a specific <span class="trait">activity</span> of 845 U mg(-1) protein. Conversion of Ser106 to Asp increased VSPalpha <span class="trait">activity</span> about 20-fold. This Asp residue is present in nodule APase and is a highly conserved nucleophile in the HAD superfamily. Related VSPs from cultivated soybean and from three wild perennial soybeans, as well as a pod storage protein (PSP) from Phaseolus vulgaris L. all lack the catalytic Asp, suggesting they too are catalytically inefficient. Phylogenetic analysis showed the VSPs and PSP are more closely related to each other than to 21 other VSP-like proteins from several plant species, all of which have the nucleophilic Asp. This study suggests that loss of catalytic <span class="trait">activity</span> may be a requirement for the VSPs and PSP to function as storage proteins in legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Iron: an essential micronutrient for the legume-rhizobium symbiosis</h2>
			<h3>PMID:24062758</h3>
				

				
					<h3>PMCID:3772312</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Brear </surname>  <given-names> Ella M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Day </surname>  <given-names> David A. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> Penelope M. C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legumes, which develop a symbiosis with nitrogen-fixing bacteria, have an increased demand for iron. Iron is required for the synthesis of iron-containing proteins in the host, including the highly abundant leghemoglobin, and in bacteroids for nitrogenase and cytochromes of the electron transport chain. Deficiencies in iron can affect initiation and development of the nodule. Within <span class="trait">root</span> cells, iron is chelated with organic acids such as citrate and nicotianamine and distributed to other parts of the plant. Transport to the nitrogen-fixing bacteroids in infected cells of nodules is more complicated. Formation of the symbiosis results in bacteroids internalized within <span class="trait">root</span> cortical cells of the legume where they are surrounded by a plant-derived membrane termed the symbiosome membrane (SM). This membrane forms an interface that regulates nutrient supply to the bacteroid. Consequently, iron must cross this membrane before being supplied to the bacteroid. Iron is transported across the SM as both ferric and ferrous iron. However, uptake of Fe(II) by both the symbiosome and bacteroid is faster than Fe(III) uptake. Members of more than one protein family may be responsible for Fe(II) transport across the SM. The only Fe(II) transporter in nodules characterized to date is <span class="gene">GmDMT1</span> (Glycine max divalent metal transporter 1), which is located on the SM in soybean. Like the <span class="trait">root</span> plasma membrane, the SM has ferric iron reductase <span class="trait">activity.</span> The protein responsible has not been identified but is predicted to reduce ferric iron accumulated in the symbiosome space prior to uptake by the bacteroid. With the recent publication of a number of legume genomes including Medicago truncatula and G. max, a large number of additional candidate transport proteins have been identified. Members of the NRAMP (natural resistance-associated macrophage protein), YSL (yellow stripe-like), VIT (vacuolar iron transporter), and ZIP (Zrt-, Irt-like protein) transport families show enhanced expression in nodules and are expected to play a role in the transport of iron and other metals across symbiotic membranes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Iron transport into <italic> Lotus japonicus </italic> infected cells is enhanced by efflux of citrate via LjMATE1 ( <italic> Lotus japonicus </italic> multidrug and toxic compound extrusion 1; <xref ref-type="bibr" rid="B93"> Takanashi et al., 2013 </xref> ; <bold> Figure <xref ref-type="fig" rid="F3"> 3 </xref> </bold> ), suggesting that it occurs as a ferric citrate complex. LjMATE1 is expressed exclusively in infected cells early in development and catalyses efflux of citrate when expressed in <italic> Xenopus </italic> oocytes ( <xref ref-type="bibr" rid="B93"> Takanashi et al., 2013 </xref> ). When expression of LjMATE1 was reduced by RNAi, nitrogenase <span class="trait">activity</span> and leghemoglobin concentration were significantly decreased compared to control nodules ( <xref ref-type="bibr" rid="B93"> Takanashi et al., 2013 </xref> ), apparently as a result of less iron in infected cells. This decrease in leghemoglobin synthesis and nitrogenase <span class="trait">activity</span> was accompanied by increased concentrations of iron at the nodule&#x02013;root junction and the vascular bundle of nodules. The early expression of LjMATE1 and its importance to concentrating iron within the infected region of nodules, suggests that citrate efflux into the apoplast of the infected region is important for iron import into these cells. In this context, LjMATE1 may play a similar role to FRD3, a <span class="gene">MATE</span> family member in <italic> Arabidopsis </italic> ( <xref ref-type="bibr" rid="B78"> Roschzttardtz et al., 2011 </xref> ). <italic> frd3 </italic> mutants show iron deficiency in leaves even though iron uptake from the soil is constitutively active. FRD3 is a citrate effluxer and is thought to release citrate into the apoplast, to chelate iron and make it more soluble, to enable transport into the cytoplasm ( <xref ref-type="bibr" rid="B78"> Roschzttardtz et al., 2011 </xref> ). The protein/s responsible for transport across the plasma membrane of infected cells in soybean and other legumes are not known. A member of the Zrt-, Irt-like protein (ZIP) or NRAMP family may be involved but this would require reduction of Fe(III) by a ferric-chelate reductase before uptake ( <bold> Figure <xref ref-type="fig" rid="F3"> 3 </xref> </bold> ). As iron is likely to be imported into infected cells from the apoplast in <italic> M. truncatula </italic> ( <xref ref-type="bibr" rid="B77"> Rodriguez-Haas et al., 2013 </xref> ), it is likely that similar transporter families are involved in uptake in indeterminant nodules. Transcriptome studies of <italic> M. truncatula </italic> suggest that expression of members of these families is enhanced in nodules ( <xref ref-type="bibr" rid="B9"> Benedito et al., 2008 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Iron uptake into the infected cell and transport into the symbiosome. </bold> Ferrous iron transport into the infected cells, across the plasma membrane (PM) could me mediated by transporters belonging to the NRAMP, ZIP, or YSL family. A citrate effluxer, LjMATE1 is expressed in infected cells, it is predicted to release citrate (blue arc) into the apoplast surrounding infected cells to aid in iron uptake. It is likely to make iron in the apoplast more soluble. Both ferrous and ferric iron are transported across the symbiosome membrane (SM) and bacteroid membrane (BM). However, ferrous iron is taken up more rapidly. A ferric iron transporter, <span class="gene">GmDMT1,</span> is present on the SM, but it may not be the only ferrous iron transporter on the SM. Members of the VIT and ZIP transporter families are possible ferrous iron transporters on the SM. Although ferric iron is thought to cross the SM, no plant transporters are known to transport iron-citrate. Ferric chelate reductase <span class="trait">activity</span> (FCR) is present on the SM, however, the orientation on the SM has not been confirmed. Within the symbiosome space (SS) ferric iron could chelate siderophores produced by the bacteroid. Ferrous and ferric iron are transported across the bacteroid membrane (BM), but the transporters responsible have not been identified.
					</p>
				
					<p>&nbsp;&nbsp;
						Iron can cross the SM as both Fe(III) and Fe(II) ( <xref ref-type="bibr" rid="B62"> Moreau et al., 1995 </xref> , <xref ref-type="bibr" rid="B61"> 1998 </xref> ; <xref ref-type="bibr" rid="B49"> LeVier et al., 1996 </xref> ; <bold> Figure <xref ref-type="fig" rid="F3"> 3 </xref> </bold> ). This was indicated by uptake of radiolabeled ferric and ferrous iron by isolated soybean symbiosomes. Inhibition of ferrous iron transport into symbiosomes by Cu(II) suggests that the protein responsible may not be specific for iron. The uptake of ferrous iron [Fe(II)] was faster than that of ferric iron [Fe(III); <xref ref-type="bibr" rid="B61"> Moreau et al., 1998 </xref> ). Whether uptake of ferrous iron is favored by the symbiosome within the natural nodule environment is yet to be determined and will most likely depend on the concentrations of Fe(II) and Fe(III) in the infected cell cytosol. Unlike the soil environment, where iron is found in an oxidized state, the nodule cytosol provides conditions for maintaining iron in its reduced ferrous state ( <xref ref-type="bibr" rid="B62"> Moreau et al., 1995 </xref> ). This environment is created by the slightly acidic pH, microaerobic environment and the abundance of reducing molecules such as ascorbate and glutathione in the nodule infected cells. Ferric chelate reductase <span class="trait">activity</span> has been identified on the SM ( <xref ref-type="bibr" rid="B49"> LeVier et al., 1996 </xref> ). Initially it was thought that the reductase <span class="trait">activity</span> occurred on the cytoplasmic side of the SM, where Fe(III)-citrate, which is present at high concentrations within the nodule, is reduced before uptake into the symbiosome ( <xref ref-type="bibr" rid="B49"> LeVier et al., 1996 </xref> ). However, <xref ref-type="bibr" rid="B61"> Moreau et al. (1998) </xref> postulated that the reductase <span class="trait">activity</span> was present within the symbiosome space based on the discovery of ferrous iron transport across the SM and the orientation of the plasma membrane reductase (outside the plasma membrane). The protein responsible for the ferric chelate reductase <span class="trait">activity</span> has not been identified to date, but two out of nine genes encoding ferric chelate reductase proteins <span class="gene">(Glyma15g13090</span> and <span class="gene">Glyma16g03770)</span> are expressed in nodules at a higher level compared to <span class="trait">root</span> tissue ( <xref ref-type="bibr" rid="B51"> Libault et al., 2010 </xref> ; <xref ref-type="bibr" rid="B84"> Severin et al., 2010 </xref> ; see <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ). Due to their nodule expression, these proteins are potential candidates for the ferric chelate reductase <span class="trait">activity</span> of the SM. It will be interesting to determine their localization and role in iron movement within the nodule. The current model of iron transport across the SM postulates that both ferrous and ferric iron can be transported across the SM, although a transporter for Fe(III)-citrate has not been characterized in dicot plants ( <bold> Figure <xref ref-type="fig" rid="F3"> 3 </xref> </bold> ). According to this model, ferric iron is reduced to ferrous iron within the symbiosome space by the SM ferric chelate reductase. The resulting ferrous iron could either be transported out of the symbiosome space into the bacteroid or into the infected cell cytosol ( <xref ref-type="bibr" rid="B61"> Moreau et al., 1998 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Jasmonic Acid Modulates the Physio-Biochemical Attributes, Antioxidant Enzyme <span class="trait">Activity,</span> and Gene Expression in <italic> Glycine max </italic> under Nickel Toxicity</h2>
			<h3>PMID:27242811</h3>
				
					<h3>PMCID:4864666</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sirhindi </surname>  <given-names> Geetika </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/319119/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mir </surname>  <given-names> Mudaser Ahmad </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/329914/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abd-Allah </surname>  <given-names> Elsayed Fathi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ahmad </surname>  <given-names> Parvaiz </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/132896/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gucel </surname>  <given-names> Salih </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/281949/overview"/>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						H <sub> 2 </sub> O <sub> 2 </sub> is a potent ROS and increases with increasing concentration of Ni ( <bold> Figures <xref ref-type="fig" rid="F2"> 2A,B </xref> </bold> ) and the findings coincides with the reports of <xref rid="B38" ref-type="bibr"> Hao et al. (2006) </xref> in <italic> Triticum </italic> <span class="trait">root.</span> <xref rid="B31" ref-type="bibr"> Gajewska and Sklodowska (2007) </xref> have also reported that Ni stress induced the accumulation of H <sub> 2 </sub> O <sub> 2 </sub> in wheat leaves. Accumulation of H <sub> 2 </sub> O <sub> 2 </sub> under heavy metal stress is also reported by many workers ( <xref rid="B8" ref-type="bibr"> Ahmad et al., 2015a </xref> , <xref rid="B10" ref-type="bibr"> b </xref> ). MDA is a product of lipid peroxidation and is commonly considered as oxidative stress indicator ( <xref rid="B9" ref-type="bibr"> Ahmad et al., 2012b </xref> , <xref rid="B8" ref-type="bibr"> 2015a </xref> ). Ni is reported to increase the hydrogen peroxide and <span class="gene">LOX</span> (Lipoxygenase is an oxidative enzyme) <span class="trait">activity,</span> which induced lipid peroxidation. <xref rid="B68" ref-type="bibr"> Pandey and Rajeev (2010) </xref> also observed the increase in MDA content with increase in Ni concentration in eggplant. Furthermore, variety of metals increased the MDA content in <italic> Bruguiera gymnorrhiza </italic> and is thus regarded as the biomarker of metal stress ( <xref rid="B97" ref-type="bibr"> Zhang et al., 2007 </xref> ). Application of JA minimizes the production of H <sub> 2 </sub> O <sub> 2 </sub> and other ROS, which directly affect the membrane lipids. The MDA decreases with external supply JA was also reported in <italic> Kandelia obovata </italic> under Cd stress ( <xref rid="B22" ref-type="bibr"> Chen et al., 2014 </xref> ). How JA is decreasing the H <sub> 2 </sub> O <sub> 2 </sub> and MDA production is still unclear. However, it is assumed that JA enhanced the scavenging capacity of antioxidants that might lead to low production of H <sub> 2 </sub> O <sub> 2 </sub> and ultimately decrease in lipid peroxidation. Another reason might be JA induced other endogenous phytohormones that could directly or indirectly impart tolerance to plants through low production of ROS ( <xref rid="B46" ref-type="bibr"> Kang et al., 2005 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Haem oxygenase delays programmed cell death in wheat aleurone layers by modulation of hydrogen peroxide metabolism</h2>
			<h3>PMID:20797999</h3>
				
					<h3>PMCID:2993913</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Mingzhu </given-names>  </name>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Jingjing </given-names>  </name>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Sheng </given-names>  </name>  <xref ref-type="author-notes" rid="fn1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ling </surname>  <given-names> Tengfang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Yanjie </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Wenbiao </given-names>  </name>  <xref ref-type="corresp" rid="cor1"> &#x02020; </xref>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 8 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ameliorating effects of the HO-1 inducer haematin or the CO aqueous solution on GA-induced death of wheat aleurone cells described above prompted us to assess if induction of HO is associated with the above responses. In the following experiments, wheat aleurone layer cells were tested for HO accumulation, through semi-quantitative RT-PCR, enzymatic <span class="trait">activity</span> determination, and immunoblot analysis. Based on the results, GA treatment significantly decreased the level of HO-1 mRNA with respect to the ABA-treated sample, by contrast, haematin or CO plus GA reversed the effect of GA ( <xref ref-type="fig" rid="fig3"> Fig. 3A </xref> ). HO <span class="trait">activity</span> decreased as much as 44.6% after 12 h of incubation in GA-treated aleurone layers compared with ABA-treated samples. An increase close to 83.9% was assessed in incubations carried out in the presence of GA plus haematin. Application of GA plus the CO aqueous solution brought about 62.1% higher HO <span class="trait">activity</span> compared with the sample that was treated with GA alone ( <xref ref-type="fig" rid="fig3"> Fig. 3B </xref> ). Western blot analysis for HO-1 ( <xref ref-type="fig" rid="fig3"> Fig. 3C </xref> ) showed only a single band with a molecular mass of 26 kDa as determined using molecular mass markers (data not shown). This mass is approximately consistent to a previously reported result for mature <span class="gene">HO1</span> of pea and <italic> Arabidopsis </italic> plants ( <xref ref-type="bibr" rid="bib37"> Muramoto <italic> et al. </italic> , 1999 </xref> ; <xref ref-type="bibr" rid="bib32"> Linley <italic> et al. </italic> , 2006 </xref> ). This assay also demonstrated a positive correlation among HO-1 transcript, enzyme <span class="trait">activity,</span> and HO-1 protein expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the HO-1 knockout mice and inhibitor test, <xref ref-type="bibr" rid="bib35"> Matsumoto <italic> et al. </italic> (2006) </xref> have confirmed that CO and BV produced from HO-1 suppresses ROS production generated from NADPH oxidase. Meanwhile, CO can suppress the catalytic <span class="trait">activity</span> of cytochrome <span class="gene">P450</span> monooxygenases that are responsible for the endogenous generation of ROS ( <xref ref-type="bibr" rid="bib27"> Kyokane <italic> et al. </italic> , 2001 </xref> ). In our subsequent experiment, results revealed that, in wheat aleurone layers, application of exogenous haematin and CO aqueous solution not only induced HO expression ( <xref ref-type="fig" rid="fig3"> Figs 3 </xref> , <xref ref-type="fig" rid="fig4"> 4 </xref> ), but also slowed down the GA-induced decrease of CAT and APX activities or their transcripts in a time-dependent manner ( <xref ref-type="table" rid="tbl1"> Table 1 </xref> ; <xref ref-type="fig" rid="fig7"> Fig. 7 </xref> ), thus leading to the significant decrease in H <sub> 2 </sub> O <sub> 2 </sub> production ( <xref ref-type="fig" rid="fig8"> Fig. 8D </xref> ). These findings are consistent with those reported in plants ( <xref ref-type="bibr" rid="bib53"> Xie <italic> et al. </italic> , 2008 </xref> ) and animals by <xref ref-type="bibr" rid="bib51"> Turkseven <italic> et al. </italic> (2005) </xref> , in which they found that the vascular cytoprotective mechanism of HO-1 against oxidative stress requires an increase in extracellular SOD and CAT expression, thus leading to an improvement in vascular relaxation in diabetic rats. Meanwhile, the delay of PCD was also observed ( <xref ref-type="fig" rid="fig4"> Fig. 4C </xref> ). Overall, an increased ability to scavenge H <sub> 2 </sub> O <sub> 2 </sub> may contribute to the delay of GA-induced PCD in wheat aleurone layer cells treated with haematin or CO aqueous solution by reducing the degree of oxidative stress, because ROS, such as H <sub> 2 </sub> O <sub> 2 </sub> and superoxide anion, play a central role in promoting PCD ( <xref ref-type="bibr" rid="bib9"> Bethke and Jones, 2001 </xref> ; <xref ref-type="bibr" rid="bib31"> Ling <italic> et al. </italic> , 2009 </xref> ), and ROS are likely targets of BHT, NO, haematin, CO, and BR in plant and animal cells ( <xref ref-type="fig" rid="fig8"> Fig. 8D </xref> ; <xref ref-type="bibr" rid="bib5"> Beligni <italic> et al. </italic> , 2002 </xref> ). Our results also suggested that BR rather than Fe <sup> 2+ </sup> , can significantly suppress the increased H <sub> 2 </sub> O <sub> 2 </sub> generation in GA-treated wheat aleurone layers by enhancing APX and CAT activities at both 12 h and 48 h of incubation ( <xref ref-type="fig" rid="fig8"> Fig. 8 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The interaction of salicylic acid and Ca(2+) alleviates aluminum toxicity in soybean (Glycine max L.).</h2>
			<h3>PMID:26691059  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lan T(1), You J(1), Kong L(1), Yu M(1), Liu M(1), Yang Z(2).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Both calcium ion (Ca(2+)) and salicylic acid (SA) influence various stress responses in plants. In acidic soils, aluminum (Al) toxicity adversely affects crop <span class="trait">yield.</span> In this study, we determined the influences of Ca(2+) and SA on <span class="trait">root</span> elongation, Al accumulation, and citrate secretion in soybean plant. We also investigated the <span class="trait">activity</span> of antioxidative enzymes in Al-exposed soybean roots. <span class="trait">Root</span> elongation was severally inhibited when the roots were exposed to 30 μM Al. The Al-induced inhibition of <span class="trait">root</span> elongation was ameliorated by Ca(2+) and SA but aggravated by Ca(2+) channel inhibitor (VP), <span class="gene">CaM</span> antagonists (TFP), Ca(2+) chelator (EGTA), and SA biosynthesis inhibitor (PAC). Furthermore, 1.0 mM CaCl2 and 10 μM SA reduced the accumulation of Al in roots, but their inhibitors stimulated the accumulation of Al in roots. Citrate secretion from these roots increased with the addition of either 1.0 mM CaCl2 or 10 μM SA but did not increase significantly when treated with higher Ca(2+) concentration. Enzymatic analysis showed that Ca(2+) and SA stimulated the activities of superoxidase (SOD), peroxidase (POD), and ascorbate peroxidase (APX) in Al-treated roots. In addition, SA restored the inhibition of Ca(2+) inhibitors on <span class="trait">root</span> elongation and Al content. Thus, both Ca(2+) and SA contribute to Al tolerance in soybean. Furthermore, Ca(2+) supplements rapidly increased Al-induced accumulation of free-SA or conjugated SA (SAG), while Ca(2+) inhibitors delayed the accumulation of SA for more than 8 h. Within 4 h of treatment, SA increased cytosolic Ca(2+) concentration in Al-treated roots, and upregulated the expression of four genes that possibly encode calmodulin-like (CML) proteins. These findings indicate that SA is involved in Ca(2+)-mediated signal transduction pathways in Al tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:miR172 regulates soybean nodulation.</h2>
			<h3>PMID:23980625  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan Z, Hossain MS, Wang J, Valdés-López O, Liang Y, Libault M, Qiu L, Stacey G.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Micro-RNAs (miRNAs) play a pivotal role in the control of gene expression and regulate plant developmental processes. miRNA 172 (miR172) is a conserved miRNA in plants reported to control the expression of genes involved in developmental phase transition, floral organ identity, and flowering time. However, the specific role of miR172 in legume nodulation is undefined. Ectopic expression of soybean miR172 resulted in an increase in nodule numbers in transgenic roots and an increase in the expression of both symbiotic leghemoglobin and nonsymbiotic hemoglobin. These nodules showed higher levels of nitrogenase <span class="trait">activity.</span> Further analysis revealed a complex regulatory circuit in which miR156 regulates miR172 expression and controls the level of an <span class="gene">AP2</span> transcription factor. The latter, either directly or indirectly, controls the expression of nonsymbiotic hemoglobin, which is essential for regulating the levels of nodulation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A low temperature-inducible protein AtSRC2 enhances the ROS-producing <span class="trait">activity</span> of NADPH oxidase AtRbohF.</h2>
			<h3>PMID:23872431  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kawarazaki T(1), Kimura S(1), Iizuka A(1), Hanamata S(1), Nibori H(1), Michikawa M(1), Imai A(1), Abe M(2), Kaya H(3), Kuchitsu K(4).
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reactive oxygen species (ROS) produced by NADPH oxidases play critical roles in plant environmental responses. Arabidopsis thaliana NADPH oxidase AtRbohF-mediated ROS-production is involved in abiotic stress responses. Because overproduction of ROS is highly toxic to cells, the <span class="trait">activity</span> of AtRbohF needs to be tightly regulated in response to diverse stimuli. The ROS-producing <span class="trait">activity</span> of AtRbohF is activated by Ca(2+) and protein phosphorylation, but other regulatory factors for AtRbohF are mostly unknown. In this study, we screened for proteins that interact with the N-terminal cytosolic region of AtRbohF by a yeast two-hybrid screen, and isolated AtSRC2, an A. thaliana homolog of <span class="gene">SRC2</span> (soybean gene regulated by cold-2). A co-immunoprecipitation assay revealed that AtSRC2 interacts with the N-terminal region of AtRbohF in plant cells. Intracellular localization of GFP-tagged AtSRC2 was partially overlapped with that of GFP-tagged AtRbohF at the cell periphery. Co-expression of AtSRC2 enhanced the Ca(2+)-dependent ROS-producing <span class="trait">activity</span> of AtRbohF in HEK293T cells, but did not affect its phosphorylation-dependent activation. Low-temperature treatment induced expression of the AtSRC2 gene in Arabidopsis roots in proportion to levels of ROS production that was partially dependent on AtRbohF. Our findings suggest that AtSRC2 is a novel activator of Ca(2+)-dependent AtRbohF-mediated ROS production and may play a role in cold responses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aggressiveness of Fusarium species and impact of <span class="trait">root</span> infection on growth and <span class="trait">yield</span> of soybeans.</h2>
			<h3>PMID:23514263  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Arias MM(1), Leandro LF, Munkvold GP.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium spp. are commonly isolated from soybean roots but the pathogenic <span class="trait">activity</span> of most species is poorly documented. Aggressiveness and <span class="trait">yield</span> impact of nine species of Fusarium were determined on soybean in greenhouse (50 isolates) and field microplot (19 isolates) experiments. <span class="trait">Root</span> rot severity and shoot and <span class="trait">root</span> dry weights were compared at growth stages <span class="gene">V3</span> or R1. <span class="trait">Root</span> systems were scanned and digital image analysis was conducted; <span class="trait">yield</span> was measured in microplots. Disease severity and <span class="trait">root</span> morphology impacts varied among and within species. Fusarium graminearum was highly aggressive <span class="trait">(root</span> rot severity &gt;90%), followed by F. proliferatum and F. virguliforme. Significant variation in damping-off (20 to 75%) and <span class="trait">root</span> rot severity (&lt;20 to &gt;60%) was observed among F. oxysporum isolates. In artificially-infested microplots, <span class="trait">root</span> rot severity was low (&lt;25%) and mean <span class="trait">yield</span> was not significantly reduced. However, there were significant linear relationships between <span class="trait">yield</span> and <span class="trait">root</span> symptoms for some isolates. <span class="trait">Root</span> morphological characteristics were more consistent indicators of <span class="trait">yield</span> loss than <span class="trait">root</span> rot severity. This study provides the first characterization of aggressiveness and <span class="trait">yield</span> impact of Fusarium <span class="trait">root</span> rot species on soybean at different plant stages and introduces <span class="trait">root</span> image analysis to assess the impact of <span class="trait">root</span> pathogens on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant antimicrobial peptides: an overview of SuperSAGE transcriptional profile and a functional review.</h2>
			<h3>PMID:20088771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kido EA(1), Pandolfi V, Houllou-Kido LM, Andrade PP, Marcelino FC, Nepomuceno AL, Abdelnoor RV, Burnquist WL, Benko-Iseppon AM.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Defensin, thionin and lipid transfer protein (LTP) gene families, which antimicrobial <span class="trait">activity</span> has an attractive use in protein engineering and transgenic production of agronomical important plants, have been here functionally reviewed. Also, a transcriptional overview of a set of plant SuperSAGE libraries and analysis looking for 26 bp tags possibly annotated for those families is presented. Tags differentially expressed (p = 0.05) or constitutively transcribed were identified from leaves or roots SuperSAGE libraries from important Brazilian plant species [cowpea (Vigna unguiculata (L.) Walp.), soybean (Glycine max (L.) Merr.) and modern sugarcane hybrids (Saccharum spp.)] submitted to abiotic [salt (100 mM NaCl) or <span class="trait">drought]</span> or biotic stresses [fungus inoculation (Phakopsora pachyrhizi; Asiatic Soyben <span class="trait">Rust</span> phytopathogen)]. The diverse transcriptional patterns observed, probably related to the variable range of targets and functions involved, could be the first step to unravel the antimicrobial peptide world and the plant stress response relationship. Moreover, SuperSAGE opens the opportunity to find some SNPs or even rare transcript that could be important on plant stress <span class="trait">resistance</span> mechanisms. Putative defensin or <span class="gene">LTP</span> identified by SuperSAGE following a specific plant treatment or physiological condition could be useful for future use in genetic improvement of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean (Glycine max) <span class="trait">root</span> lignification induced by ferulic acid. The possible mode of action.</h2>
			<h3>PMID:18626717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：dos Santos WD(1), Ferrarese ML, Nakamura CV, Mourão KS, Mangolin CA, Ferrarese-Filho O.
				<br/> 
				年份：2008 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ferulic acid, in the form of feruloyl CoA, occupies a central position as an intermediate in the phenylpropanoid pathway. Due to the allelopathic function, its effects were tested on <span class="trait">root</span> growth, H(2)O(2) and lignin contents, and activities of cinnamyl alcohol dehydrogenase (CAD, EC 1.1.1.195) and peroxidase (POD, EC 1.11.1.7) from soybean (Glycine max (L.) Merr.) <span class="trait">root</span> seedlings. Three-day-old seedlings were cultivated in half-strength Hoagland's solution (pH 6.0), with or without 1.0 mM ferulic acid in a growth chamber (25 degrees C, 12/12 hr light/dark photoperiod, irradiance of 280 micromol m(-2) s(-1)) for 24 or 48 hr. Exogenously supplied ferulic acid induced premature cessation of <span class="trait">root</span> growth, with disintegration of the <span class="trait">root</span> cap, compression of cells in the quiescent center, increase of the vascular cylinder diameter, and earlier lignification of the metaxylem. Moreover, the allelochemical decreased CAD <span class="trait">activity</span> and H(2)O(2) level and increased the anionic isoform PODa5 <span class="trait">activity</span> and lignin content. The lignin monomer composition of ferulic acid-exposed roots revealed a significant increase of guaiacyl (G) units. When applied jointly with piperonylic acid (an inhibitor of the cinnamate 4-hydroxylase, C4H), ferulic acid increased lignin content. By contrast, the application of 3,4-(methylenedioxy) cinnamic acid (an inhibitor of the 4-coumarate:CoA ligase, 4CL) with ferulic acid did not. Taken together, these results suggest that ferulic acid may be channeled into the phenylpropanoid pathway (by the <span class="gene">4CL</span> reaction) and, further, may increase the lignin monomer amount solidifying the cell wall and restricting the <span class="trait">root</span> growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of glyphosate and foliar amendments on <span class="trait">activity</span> of microorganisms in the soybean rhizosphere.</h2>
			<h3>PMID:17365326  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Means NE(1), Kremer RJ, Ramsier C.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field study was conducted to determine the effects of glyphosate on microbial <span class="trait">activity</span> in the rhizosphere of glyphosate-resistant (GR) soybean and to evaluate interactions with foliar amendments. Glyphosate at 0.84 kg ae ha(-1) was applied <span class="gene">GR</span> soybean at the V4-V5 development stages. Check treatments included a conventional herbicide tank mix (2003 study only) and no herbicides (hand-weeded). Ten days after herbicide application, a commercially available biostimulant and a urea solution (21.0% N) were applied to soybean foliage at 33.5 mL ha(-1) and 9.2 kg ha(-1), respectively. Soil and plant samples were taken 0, 5, 10, 15, 20 and 25 days after herbicide application then assayed for enzyme and respiration activities. Soil respiration and enzyme <span class="trait">activity</span> increased with glyphosate and foliar amendment applications during the 2002 growing season; however, similar increases were not observed in 2003. Contrasting cumulative rainfall between 2002 and 2003 likely accounted for differences in soil microbial activities. Increases in soil microbial <span class="trait">activity</span> in 2002 suggest that adequate soil water and glyphosate application acted together to increase microbial <span class="trait">activity.</span> Our study suggests that general soil microbial properties including those involving C and N transformations are not sensitive enough to detect effects of glyphosate on rhizosphere microbial <span class="trait">activity.</span> Measurements of soil-plant-microbe relationships including specific microbial groups (i.e., root-associated Fusarium spp.) are likely better indicators of impacts of glyphosate on soil microbial ecology.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of composted textile sludge on growth, nodulation and nitrogen fixation of soybean and cowpea.</h2>
			<h3>PMID:16782329  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Araújo AS(1), Monteiro RT, Carvalho EM.
				<br/> 
				年份：2007 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of composted textile sludge on growth, nodulation and nitrogen fixation of soybean and cowpea was evaluated in a greenhouse experiment. The compost was incorporated into soil at 0, 9.5, 19 and 38 t ha(-1) (bases upon the N requirement of the crops, i.e., 0, 50, 100 and 200 kg available N ha(-1)). Growth, nodulation and shoot accumulation of nitrogen were evaluated 36 and 63 days after plant emergence. Nodule glutamine synthetase (GS) <span class="trait">activity</span> and leghemoglobin content were evaluated 63 days after emergence. Composted textile sludge did not show negative effects on nodule number and weight, nodule <span class="gene">GS</span> <span class="trait">activity</span> and leghemoglobin content. Nitrogen accumulation in shoot dry matter in soybean and cowpea was higher than other treatments with application of 19 t ha(-1) of compost. Composting can be an alternate technology for the management of solid textile mill sludge. This study verifies that the composted textile sludge was not harmful to growth, nodulation and nitrogen fixation of soybean and cowpea.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In Vivo Regulatory Phosphorylation of Soybean Nodule Phosphoenolpyruvate Carboxylase.</h2>
			<h3>PMID:12228563</h3>
				

				
			<p>
				作者：Zhang XQ(1), Li B, Chollet R.
				<br/> 
				年份：1995 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this report we provide evidence that cytosolic phosphoenolpyruvate carboxylase (PEPC) in soybean (Glycine max L.) <span class="trait">root</span> nodules is regulated in vivo by a seryl-phosphorylation cycle, as with the C4, Crassulacean acid metabolism, and <span class="gene">C3</span> leaf isoforms. Pretreatment of parent plants by stem girdling for 5 or 14 h caused a significant decrease in the apparent phosphorylation state of nodule PEPC, as indicated by the 50% inhibition constant (L-malate) and specific <span class="trait">activity</span> values assayed at suboptimal conditions, whereas short-term darkness alone was without effect. However, extended (26 h) darkness led to the formation of a relatively dephosphorylated nodule PEPC, an effect that was reversed by illuminating the darkened plants for 3 h. This reversal of the apparent phosphorylation state in the light was prevented by concomitant stem girdling. In contrast, the optimal <span class="trait">activity</span> of nodule PEPC and its protein level showed little or no change in all pretreated plants. These results suggest that the phosphorylation state of PEPC in soybean <span class="trait">root</span> nodules is possibly modulated by photosynthate transported recently from the shoots. In situ [32P]orthophosphate labeling, immunoprecipitation, and phosphoamino acid analyses confirmed directly that PEPC in detached intact soybean nodules is phosphorylated on a serine residue(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bradyrhizobium japonicum TlpA, a novel membrane-anchored thioredoxin-like protein involved in the biogenesis of cytochrome aa3 and development of symbiosis.</h2>
			<h3>PMID:8253065  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Loferer H(1), Bott M, Hennecke H.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We report the discovery of a bacterial gene, tlpA, that codes for a hitherto unknown type of thioredoxin-like protein. The gene was found in the course of studying a Tn5 insertion mutant of the soybean <span class="trait">root</span> nodule symbiont Bradyrhizobium japonicum. The TlpA protein shared up to 31% amino acid sequence identity with various eukaryotic and prokaryotic thioredoxins and protein disulfide isomerases, and possessed a characteristic active-site sequence, Trp-Cys-Val-Pro-Cys. In contrast to all members of the thioredoxin family known to date, TlpA was shown to be anchored to the cytoplasmic membrane by means of an N-terminal transmembrane domain, while the active site-containing part of the protein faced the periplasm. The <span class="gene">tlpA</span> mutant had a pleiotropic phenotype in that it was defective in the development of a nitrogen fixing endosymbiosis and exhibited a strongly decreased oxidase <span class="trait">activity,</span> as compared with the wild-type. Holocytochrome aa3 was spectroscopically undetectable in the mutant, whereas the apoprotein of subunit one (CoxA) of this oxidase was still synthesized and incorporated into the cytoplasmic membrane. Since cytochrome aa3 is not a prerequisite for the development of symbiosis, the results suggest that TlpA is involved in at least two independent cellular processes, one of which is an essential periplasmic step in the maturation of cytochrome aa3.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of soya bean flakes and liquorice <span class="trait">root</span> extract on enzyme induction and toxicity in B6C3F1 mice.</h2>
			<h3>PMID:8505019  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mirsalis JC(1), Hamilton CM, Schindler JE, Green CE, Dabbs JE.
				<br/> 
				年份：1993 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Both soya bean flakes (SBF) and liquorice <span class="trait">root</span> extract (LRE) have previously been reported to have anticarcinogenic properties, which have been thought to be related to an increased <span class="trait">activity</span> of specific enzymes responsible for the detoxification of chemical carcinogens. 30- and 90-day studies were conducted in male B6C3F1 mice to determine which, if any, of several detoxification enzymes are induced by SBF or LRE. Mice fed 8 and 25% LRE showed a variety of adverse clinical signs, poor weight gain and 30% mortality. Significant increases in liver:body weight ratios were observed in both the SBF and LRE groups. No significant treatment-related gross autopsy findings were observed in any of the SBF groups. A number of abnormalities were observed in the LRE groups, including lesions of the kidney, liver, spleen and thymus. Liver samples from the 90-day study were analysed for 7-ethoxycoumarin O-deethylase (7-ECOD), benzo[a]pyrene hydroxylase (BPH), superoxide dismutase (SOD), glutathione S-transferase (GST) and UDP-glucuronyl transferase (UDPGT) at 90 days, and at an interim 30-day autopsy. No treatment-related increases were observed for BPH or SOD. Both SBF and LRE induced modest increases in UDPGT <span class="trait">activity.</span> SBF induced modest increases in <span class="gene">GST</span> <span class="trait">activity,</span> but LRE decreased this <span class="trait">activity.</span> 7-ECOD <span class="trait">activity</span> was significantly increased by LRE and decreased by SBF. Samples from a 30-day study in which both LRE and SBF were administered at various dose levels were examined for UDPGT <span class="trait">activity;</span> all dose groups showed decreases in UDPGT <span class="trait">activity</span> relative to controls. The results suggest that both SBF and LRE may alter the activities of specific enzymes involved in the detoxification of chemical carcinogens; however, the combination of these two foodstuffs may not produce an additive effect in B6C3F1 mice.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Urease-null and hydrogenase-null phenotypes of a phylloplane bacterium reveal altered nickel metabolism in two soybean mutants.</h2>
			<h3>PMID:16668768</h3>
				

				
			<p>
				作者：Holland MA(1), Polacco JC.
				<br/> 
				年份：1992 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mutation at either of two genetic loci (Eu2 or Eu3) in soybean (Glycine max [L.] Merr.) results in a pleiotropic elimination of the <span class="trait">activity</span> of both major urease isozymes. Surprisingly, the phenotype of a phylloplane bacterium, Methylobacterium mesophilicum, living on the leaves of eu2/eu2 or eu3-e1/eu3-e1 mutants is also affected by these plant mutations. The bacteria isolated from leaves of these soybean mutants have transient urease- and hydrogenase-deficient phenotypes that can be corrected by the addition of nickel to free-living cultures. The same bacterium growing on wild-type soybeans or on urease mutants eu1-sun/eu1-sun or eu4/eu4, each deficient in only one urease isozyme, are urease-positive. These results suggest that the bacterium living on the eu2/eu2 or eu3-e1/eu3-e1 mutant is unable to produce an active urease or hydrogenase because it is effectively starved for nickel. We infer that mutations at Eu2 or <span class="gene">Eu3</span> result in defects in nickel metabolism but not in Ni(2+) uptake or transport, because eu2/eu2 and eu3-e1/eu3-e1 mutants exhibit normal uptake of (63)NiCl(2). Moreover, wild-type plants grafted on mutant rootstocks produce seeds with fully active urease, indicating unimpeded transport of nickel through mutant roots and stems.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Utilization of nitrate by bacteroids of Bradyrhizobium japonicum in the soybean <span class="trait">root</span> nodule.</h2>
			<h3>PMID:24221417</h3>
				

				
			<p>
				作者：Giannakis C(1), Nicholas DJ, Wallace W.
				<br/> 
				年份：1988 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bacteroids of Bradyrhizobium japonicum strain CB1809, unlike CC705, do not have a high level of constitutive nitrate reductase (NR; EC 1.7.99.4) in the soybean (Glycine max. Merr.) nodule. Ex planta both strains have a high <span class="trait">activity</span> of NR when cultured on 5 mM nitrate at 2% O2 (v/v). Nitrite reductase (NiR) was active in cultured cells of bradyrhizobia, but <span class="trait">activity</span> with succinate as electron donor was not detected in freshly-isolated bacteroids. A low <span class="trait">activity</span> was measured with reduced methyl viologen. When bacteroids of CC705 were incubated with nitrate there was a rapid production of nitrite which resulted in repression of NR. Subsequently when <span class="gene">NiR</span> was induced, nitrite was utilized and NR <span class="trait">activity</span> recovered. Nitrate reductase was induced in bacteroids of strain CB1809 when they were incubated in-vitro with nitrate or nitrite. Increase in NR <span class="trait">activity</span> was prevented by rifampicin (10 μg· ml(-1)) or chloramphenicol (50 μg·ml(-1)). Nitrite-reductase <span class="trait">activity</span> in bacteroids of strain CB1809 was induced in parallel with NR. When nitrate was supplied to soybeans nodulated with strain CC705, nitrite was detected in nodule extracts prepared in aqueous media and it accumulated during storage (1°C) and on further incubation at 25°C. Nitrite was not detected in nodule extracts prepared in ethanol. Thus nitrite accumulation in nodule tissue appears to occur only after maceration and although bacteroids of some strains of B. japonicum have a high level of a constitutive NR, they do not appear to reduce nitrate in the nodule because this anion does not gain access to the bacteroid zone. Soybeans nodulated with strains CC705 and CB1809 were equally sensitive to nitrate inhibition of N2 fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic Analysis of <italic> Rhizoctonia solani </italic> Identifies Infection-specific, Redox Associated Proteins and Insight into Adaptation to Different Plant Hosts <xref ref-type="fn" rid="FN1"> * </xref> <xref ref-type="fn" rid="FN2"> <sup> <inline-graphic xlink:href="sbox.jpg"/> </sup> </xref></h2>
			<h3>PMID:26811357</h3>
				
					<h3>PMCID:4824849</h3>
					<h4>期刊:Molecular &#x00026; Cellular Proteomics : MCP</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> Jonathan P. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> &#x02021; </xref>  <xref ref-type="aff" rid="aff2"> &#x000a7; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hane </surname>  <given-names> James K. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> &#x02021; </xref>  <xref ref-type="author-notes" rid="FN3"> ** </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stoll </surname>  <given-names> Thomas </given-names>  </name>  <xref ref-type="aff" rid="aff3"> &#x000b6; </xref>  <xref ref-type="author-notes" rid="FN4"> &#x02021;&#x02021; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pain </surname>  <given-names> Nicholas </given-names>  </name>  <xref ref-type="aff" rid="aff1"> &#x02021; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hastie </surname>  <given-names> Marcus L. </given-names>  </name>  <xref ref-type="aff" rid="aff3"> &#x000b6; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kaur </surname>  <given-names> Parwinder </given-names>  </name>  <xref ref-type="aff" rid="aff1"> &#x02021; </xref>  <xref ref-type="author-notes" rid="FN5"> &#x000a7;&#x000a7; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoogland </surname>  <given-names> Christine </given-names>  </name>  <xref ref-type="aff" rid="aff3"> &#x000b6; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gorman </surname>  <given-names> Jeffrey J. </given-names>  </name>  <xref ref-type="aff" rid="aff3"> &#x000b6; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Karam B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> &#x02021; </xref>  <xref ref-type="aff" rid="aff2"> &#x000a7; </xref>  <xref ref-type="corresp" rid="cor1"> &#x02016; </xref>  </contrib>  <aff id="aff1"> From the &#x02021;CSIRO Agriculture, Floreat, Western Australia;  </aff>  <aff id="aff2"> &#x000a7;The University of Western Australia Institute of Agriculture, Crawley, Western Australia;  </aff>  <aff id="aff3"> &#x000b6;QIMR Berghofer Medical Research Institute, Herston, QLD, Australia </aff>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 15 proteins were chosen for assessment for a role in pathogenicity based on their identification in infection conditions, gene expression during infection and/or the putative functions assigned by interproscan. As no genetic transformation system for gene knock out is presently available for <italic> R. solani </italic> , we introduced the open reading frames of candidate effectors under the control of the CaMV 35s promoter into <italic> Nicotiana benthamiana </italic> using agroinfiltration and lesion size from subsequent infection with <italic> R. solani </italic> was measured. Several <italic> R. solani </italic> isolates are known pathogens of tobacco, causing leaf spot and <span class="trait">root</span> rot and inoculation of <italic> N. benthamiana </italic> with <italic> R. solani </italic> in the present study caused the development of water soaked lesions similar to those previously described ( <xref rid="B44" ref-type="bibr"> 44 </xref> ). All expression vectors also contained a cassette for constitutive expression of GFP which was used to confirm transformation and transgene expression had occurred prior to <italic> R. solani </italic> inoculation (data not shown). Expression of RSAG8_08836, a thaumatin protein, increased the size of lesions formed following <italic> R. solani </italic> inoculation in comparison to control areas expressing <italic> GFP </italic> only or <italic> GFP </italic> and <italic> DsRed </italic> ( <xref ref-type="fig" rid="F6"> Fig. 6 </xref> <italic> A </italic> , <xref ref-type="fig" rid="F6"> 6 </xref> <italic> B </italic> ). Positive control infiltrations with the <italic> <span class="gene">NLP1</span> </italic> gene from <italic> Fusarium oxysporum </italic> (Fo5176) which is known to induce plant cell death, showed similar increased lesion size ( <xref ref-type="fig" rid="F6"> Fig. 6 </xref> ). All other infiltrated proteins did not alter lesion size ( <xref ref-type="fig" rid="F6"> Fig. 6 </xref> and data not shown).
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>