<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Root length development</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Root length development</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Functional characterization of soybean strigolactone biosynthesis and signaling genes in Arabidopsis MAX mutants and <span class="gene">GmMAX3</span> in soybean nodulation</h2>
			<h3>PMID:29268717</h3>
				

				
					<h3>PMCID:5740752</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Haq </surname>  <given-names> Basir UI </given-names>  </name>  <address>  <email> basirulhaq3star@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Ahmad </surname>  <given-names> Muhammad Zulfiqar </given-names>  </name>  <address>  <email> zulfiqar_nuas@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> ur Rehman </surname>  <given-names> Naveed </given-names>  </name>  <address>  <email> naveed.urrehman@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Junjie </given-names>  </name>  <address>  <email> junjiewang88@sina.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Penghui </given-names>  </name>  <address>  <email> lphui2012@126.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongqin </given-names>  </name>  <address>  <email> ldq@mail.hzau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-4416-7334 </contrib-id>  <name>  <surname> Zhao </surname>  <given-names> Jian </given-names>  </name>  <address>  <phone> +86 27 87385199 </phone>  <email> jzhao2@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1790 4137 </institution-id>  <institution-id institution-id-type="GRID"> grid.35155.37 </institution-id>  <institution> National Key Laboratory of Crop Genetic Improvement,  </institution>  <institution> Huazhong Agricultural University,  </institution>  </institution-wrap> Wuhan, 430075 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1760 4804 </institution-id>  <institution-id institution-id-type="GRID"> grid.411389.6 </institution-id>  <institution> State Key Laboratory of Tea Plant Biology and Utilization, College of Tea and Food Science and Technology,  </institution>  <institution> Anhui Agricultural University,  </institution>  </institution-wrap> Hefei, 230036 China  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Relative expression of these <italic> <span class="gene">GmMAXs</span> </italic> to <italic> <span class="gene">GmACTIN</span> </italic> was measured with samples from soybean cultivar &#x0201c;Tianlong 1&#x0201d; at different <span class="trait">development</span> stages by using qRT-PCR.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> a </bold> The seedlings of <italic> atmax1&#x02013;1, </italic> Col-0, and <italic> atmax1&#x02013;1 </italic> complementation plant with <italic> <span class="gene">GmMAX1a</span> </italic> ( <italic> atmax1/GmMAX1a </italic> ) with different primary <span class="trait">root</span> lengths (left panel).
					</p>
				
					<p>&nbsp;&nbsp;
						Quantification of silique <span class="trait">length</span> (cm) of Col-0, <italic> atmax1&#x02013;1, </italic> complementation of ( <italic> atmax1/GmMAX1a </italic> ) and <italic> <span class="gene">GmMAX1a-OE</span> </italic> plants (right panel).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> The different primary <span class="trait">root</span> lengths seedlings of <italic> atmax3/GmMAX3b </italic> complementation <italic> , atmax3, </italic> and Col-0 plant with <italic> <span class="gene">GmMAX3b</span> </italic> ( <italic> atmax3/GmMAX3b </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> b </bold> The different primary <span class="trait">root</span> lengths seedlings (left panel) of <italic> atmax4 </italic> complementation plant with <italic> <span class="gene">GmMAX4a</span> </italic> ( <italic> atmax4/GmMAX4a </italic> ), <italic> atmax4 </italic> and Col-0 plants.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> d </bold> <span class="trait">Root</span> hair patterns in the soybean hairy roots overexpressing <italic> <span class="gene">GmMAX3b</span> </italic> or <italic> GUS </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> Hairy <span class="trait">root</span> fresh weight (g) and nodule numbers ratio in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> control lines.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> e </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> as control hairy <span class="trait">root</span> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> b </bold> Validation of auxin biosynthetic genes <italic> TAA </italic> s and <italic> YUC </italic> s in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> Heat map analysis of hormone biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> d </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b</span> </italic> -OE, <italic> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines.
					</p>
				
					<p>&nbsp;&nbsp;
						Relative expression of these <italic> <span class="gene">GmMAXs</span> </italic> to <italic> <span class="gene">GmACTIN</span> </italic> was measured with samples from soybean cultivar &#x0201c;Tianlong 1&#x0201d; at different <span class="trait">development</span> stages by using qRT-PCR
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> a </bold> The seedlings of <italic> atmax1&#x02013;1, </italic> Col-0, and <italic> atmax1&#x02013;1 </italic> complementation plant with <italic> <span class="gene">GmMAX1a</span> </italic> ( <italic> atmax1/GmMAX1a </italic> ) with different primary <span class="trait">root</span> lengths (left panel)
					</p>
				
					<p>&nbsp;&nbsp;
						Quantification of silique <span class="trait">length</span> (cm) of Col-0, <italic> atmax1&#x02013;1, </italic> complementation of ( <italic> atmax1/GmMAX1a </italic> ) and <italic> <span class="gene">GmMAX1a-OE</span> </italic> plants (right panel)
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> The different primary <span class="trait">root</span> lengths seedlings of <italic> atmax3/GmMAX3b </italic> complementation <italic> , atmax3, </italic> and Col-0 plant with <italic> <span class="gene">GmMAX3b</span> </italic> ( <italic> atmax3/GmMAX3b </italic> )
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> b </bold> The different primary <span class="trait">root</span> lengths seedlings (left panel) of <italic> atmax4 </italic> complementation plant with <italic> <span class="gene">GmMAX4a</span> </italic> ( <italic> atmax4/GmMAX4a </italic> ), <italic> atmax4 </italic> and Col-0 plants
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> d </bold> <span class="trait">Root</span> hair patterns in the soybean hairy roots overexpressing <italic> <span class="gene">GmMAX3b</span> </italic> or <italic> GUS </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> Hairy <span class="trait">root</span> fresh weight (g) and nodule numbers ratio in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> control lines
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> e </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> as control hairy <span class="trait">root</span> lines
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> b </bold> Validation of auxin biosynthetic genes <italic> TAA </italic> s and <italic> YUC </italic> s in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> c </bold> Heat map analysis of hormone biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b-OE,</span> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> d </bold> Validation of SL biosynthesis and signaling genes in <italic> <span class="gene">GmMAX3b</span> </italic> -OE, <italic> <span class="gene">GmMAX3b-KD</span> </italic> and <italic> GUS </italic> hairy <span class="trait">root</span> lines
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> salt stress, H2O2 and ABA.
					</p>
				
					<p>&nbsp;&nbsp;
						Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary <span class="trait">root</span> <span class="trait">length,</span> greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content.
					</p>
				
					<p>&nbsp;&nbsp;
						In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary <span class="trait">root</span> elongation.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control <span class="trait">root</span> growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in high salt and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on germination and seedling <span class="trait">root</span> arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						While the <span class="trait">root</span> <span class="trait">length</span> and <span class="trait">root</span> surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings.
					</p>
				
					<p>&nbsp;&nbsp;
						When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and salt stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of flower <span class="trait">development</span> related gene GsLFY from Glycine soja.</h2>
			<h3>PMID:28115306  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo WY(1), Cui YM(1), Wang TT(1), Yu DY(1), Huang F(1).
				<br/> 
				年份：2017 Jan 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						So far, <span class="gene">LFY</span> regulation on floral <span class="trait">development</span> in wild soybean has not been reported in the literature.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean Intrinsic Protein Gene, <span class="gene">GmTIP2;3,</span> Involved in Responding to Osmotic Stress.</h2>
			<h3>PMID:26779248</h3>
				

				
			<p>
				作者：Zhang D(1), Tong J(2), He X(1), Xu Z(1), Xu L(1), Wei P(1), Huang Y(1), Brestic M(3), Ma H(1), Shao H(4).
				<br/> 
				年份：2016 Jan 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The expression analysis indicated that this gene was constitutively expressed in all detected tissues, with higher levels in the <span class="trait">root,</span> stem and pod, and the accumulation of <span class="gene">GmTIP2;3</span> transcript showed a significant response to osmotic stresses, including 20% PEG6000 (polyethylene glycol) and 100 μM ABA (abscisic acid) treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						The promoter-GUS (glucuronidase) activity analysis suggested that <span class="gene">GmTIP2;3</span> was also expressed in the <span class="trait">root,</span> stem, and leaf, and preferentially expressed in the stele of <span class="trait">root</span> and stem, and the core promoter region was 1000 bp in <span class="trait">length,</span> located upstream of the ATG start codon.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of soybean miR172c confers tolerance to water deficit and salt stress, but increases ABA sensitivity in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:26466661  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li W(1), Wang T(1), Zhang Y(1), Li Y(2).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The overexpression of gma-miR172c in Arabidopsis promoted earlier flowering compared with the wild type through modulation of the expression of flowering genes, such as FT and <span class="gene">LFY</span> during long days, especially under <span class="trait">drought</span> conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">Glyma01g39520</span> weakened ABA sensitivity and reduced the tolerance to <span class="trait">drought</span> stress in the snz mutant of Arabidopsis by reducing the expression of ABI3 and ABI5.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning and expression analysis of a stress-induced <span class="gene">GmIMT1</span> gene in soybean (Glycine max).</h2>
			<h3>PMID:24615045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang HT(1), Guo N(1), Zhao JM(1), Karthikeyan A(1), Xue D(1), Xue CC(1), Xu JY(1), Xu ZH(1), Gai JY(1), Xing H(1).
				<br/> 
				年份：2014 Feb 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The results showed that the <span class="trait">root,</span> leaves, and stems exhibited high level of <span class="gene">GmIMT1</span> gene expression, whereas there was no expression in the seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <italic> <span class="gene">GmRAV</span> </italic> Ortholog Is Involved in Photoperiod and Sucrose Control of Flowering Time in Soybean</h2>
			<h3>PMID:24551235</h3>
				

				
					<h3>PMCID:3925180</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Qingyao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Lin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Diqiu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhan </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 1 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmRAV</span> </italic> -i soybeans and <italic> Arabidopsis rav </italic> mutants showed more sensitive hypocotyl elongation responses when compared with wild-type seedlings, and <italic> <span class="gene">GmRAV</span> </italic> -ox overevpressed in tobacco revealed no sensitive changes in hypocotyl <span class="trait">length</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, <italic> <span class="gene">GmRAV</span> </italic> was inferred to have a fundamental function in photoperiod, darkness, and sucrose signaling responses to regulate plant <span class="trait">development</span> and flowering induction.
					</p>
				
					<p>&nbsp;&nbsp;
						A detailed phenotypic characterization, along with genetic and physiological analysis, indicated that <italic> <span class="gene">GmRAV</span> </italic> was inferred to be a signaling component involved in regulation of plant <span class="trait">development</span> and flowering time.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmRAV</span> </italic> mRNA was present in all organs examined, including leaf (Tl), trifoliate leaf (TL), stem (St), <span class="trait">root</span> (RO), pod (PO), flower bud (FB), and immature seed (IS).
					</p>
				
					<p>&nbsp;&nbsp;
						The transcript abundance of <italic> <span class="gene">GmRAV</span> </italic> was affected by day <span class="trait">length</span> in soybean leaves, the diurnal phase of <italic> <span class="gene">GmRAV</span> </italic> mRNA expression was regulated by the circadian clock, and higher levels of <italic> <span class="gene">GmRAV</span> </italic> mRNA were accumulated under SDs than under LDs.
					</p>
				
					<p>&nbsp;&nbsp;
						The early-flowering phenotype soybean mutant <italic> <span class="gene">GmRAV-i</span> </italic> was observed despite of the day <span class="trait">length</span> in both LDs and SDs compared to the WT ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Apart from their flowering-time phenotype, <italic> <span class="gene">GmRAV-i</span> </italic> soybeans also displayed complex pleiotropic alterations of vegetative <span class="trait">development</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Sensitivity of <italic> Arabidopsis rav </italic> mutants and <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco seedlings in response to sucrose was tested in <span class="trait">root</span> and hypocotyl by growth inhibition assays.
					</p>
				
					<p>&nbsp;&nbsp;
						The hypocotyl and <span class="trait">root</span> lengths of <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and WT plants were inhibited by both concentrations of sucrose (2% and 4%) ( <xref ref-type="fig" rid="pone-0089145-g006"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Likewise, the growth in soybean <italic> <span class="gene">GmRAV-i</span> </italic> and WT plants in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots was all inhibited by both concentrations of sucrose ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Hypocotyl, <span class="trait">root</span> <span class="trait">length,</span> and the number of lateral roots of <italic> <span class="gene">GmRAV-i</span> </italic> soybeans were less inhibited by sucrose than in WT plants ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, <italic> <span class="gene">GmRAV-i</span> </italic> seedlings exhibited insensitive phenotypes, as compared with the WT plant, during sucrose-mediated <span class="trait">root</span> and hypocotyl growth inhibition.
					</p>
				
					<p>&nbsp;&nbsp;
						Overall, in agreement with the <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco results, the Arabidopsis <italic> rav </italic> mutants and the <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings showed a significantly decreased sensitivity to sucrose by <span class="trait">root</span> and hypocotyl growth inhibition assays.
					</p>
				
					<p>&nbsp;&nbsp;
						The behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth inhibition assay ( <xref ref-type="fig" rid="pone-0089145-g009"> Fig.
					</p>
				
					<p>&nbsp;&nbsp;
						However, <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> that exhibited insensitive phenotypes were less inhibited by 200 mM sugars (6.
					</p>
				
					<p>&nbsp;&nbsp;
						9A, C&#x02013;E </xref> ), which indicated <italic> <span class="gene">RAV</span> </italic> gene played a negative role in sucrose-mediated regulation of hypocotyl and <span class="trait">root</span> elongation due to metabolic effects rather than osmotic effects.
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, Arabidopsis <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> which exhibited insensitive phenotypes were less inhibited by 200 mM sugars.
					</p>
				
					<p>&nbsp;&nbsp;
						But we found the behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmRAV</span> </italic> -i soybeans and <italic> Arabidopsis rav </italic> mutants showed more sensitive hypocotyl elongation responses when compared with wild-type seedlings, and <italic> <span class="gene">GmRAV</span> </italic> -ox overevpressed in tobacco revealed no sensitive changes in hypocotyl <span class="trait">length</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, <italic> <span class="gene">GmRAV</span> </italic> was inferred to have a fundamental function in photoperiod, darkness, and sucrose signaling responses to regulate plant <span class="trait">development</span> and flowering induction
					</p>
				
					<p>&nbsp;&nbsp;
						A detailed phenotypic characterization, along with genetic and physiological analysis, indicated that <italic> <span class="gene">GmRAV</span> </italic> was inferred to be a signaling component involved in regulation of plant <span class="trait">development</span> and flowering time
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmRAV</span> </italic> mRNA was present in all organs examined, including leaf (Tl), trifoliate leaf (TL), stem (St), <span class="trait">root</span> (RO), pod (PO), flower bud (FB), and immature seed (IS)
					</p>
				
					<p>&nbsp;&nbsp;
						The transcript abundance of <italic> <span class="gene">GmRAV</span> </italic> was affected by day <span class="trait">length</span> in soybean leaves, the diurnal phase of <italic> <span class="gene">GmRAV</span> </italic> mRNA expression was regulated by the circadian clock, and higher levels of <italic> <span class="gene">GmRAV</span> </italic> mRNA were accumulated under SDs than under LDs
					</p>
				
					<p>&nbsp;&nbsp;
						The early-flowering phenotype soybean mutant <italic> <span class="gene">GmRAV-i</span> </italic> was observed despite of the day <span class="trait">length</span> in both LDs and SDs compared to the WT ( <xref ref-type="fig" rid="pone-0089145-g002"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Apart from their flowering-time phenotype, <italic> <span class="gene">GmRAV-i</span> </italic> soybeans also displayed complex pleiotropic alterations of vegetative <span class="trait">development</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Sensitivity of <italic> Arabidopsis rav </italic> mutants and <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco seedlings in response to sucrose was tested in <span class="trait">root</span> and hypocotyl by growth inhibition assays
					</p>
				
					<p>&nbsp;&nbsp;
						The hypocotyl and <span class="trait">root</span> lengths of <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and WT plants were inhibited by both concentrations of sucrose (2% and 4%) ( <xref ref-type="fig" rid="pone-0089145-g006"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Likewise, the growth in soybean <italic> <span class="gene">GmRAV-i</span> </italic> and WT plants in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots was all inhibited by both concentrations of sucrose ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Hypocotyl, <span class="trait">root</span> <span class="trait">length,</span> and the number of lateral roots of <italic> <span class="gene">GmRAV-i</span> </italic> soybeans were less inhibited by sucrose than in WT plants ( <xref ref-type="fig" rid="pone-0089145-g008"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, <italic> <span class="gene">GmRAV-i</span> </italic> seedlings exhibited insensitive phenotypes, as compared with the WT plant, during sucrose-mediated <span class="trait">root</span> and hypocotyl growth inhibition
					</p>
				
					<p>&nbsp;&nbsp;
						Overall, in agreement with the <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco results, the Arabidopsis <italic> rav </italic> mutants and the <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings showed a significantly decreased sensitivity to sucrose by <span class="trait">root</span> and hypocotyl growth inhibition assays
					</p>
				
					<p>&nbsp;&nbsp;
						The behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth inhibition assay ( <xref ref-type="fig" rid="pone-0089145-g009"> Fig
					</p>
				
					<p>&nbsp;&nbsp;
						However, <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> that exhibited insensitive phenotypes were less inhibited by 200 mM sugars (6
					</p>
				
					<p>&nbsp;&nbsp;
						9A, C&#x02013;E </xref> ), which indicated <italic> <span class="gene">RAV</span> </italic> gene played a negative role in sucrose-mediated regulation of hypocotyl and <span class="trait">root</span> elongation due to metabolic effects rather than osmotic effects
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, Arabidopsis <italic> rav </italic> mutants and soybean <italic> <span class="gene">GmRAV-i</span> </italic> in terms of hypocotyl lengths, main <span class="trait">root</span> <span class="trait">length</span> and the number of lateral roots of <span class="trait">development</span> which exhibited insensitive phenotypes were less inhibited by 200 mM sugars
					</p>
				
					<p>&nbsp;&nbsp;
						But we found the behavior of Arabidopsis <italic> rav </italic> mutant, <italic> <span class="gene">GmRAV-ox</span> </italic> tobacco and <italic> <span class="gene">GmRAV-i</span> </italic> soybean seedlings with 200 mM sugar alcohols on <span class="gene">MS</span> media showed no difference from that of the WT seedlings in hypocotyl, <span class="trait">root</span> and the number of lateral roots growth
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Functional Analysis of Water Stress-Responsive Soybean <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> Transcription Factors in Lateral <span class="trait">Root</span> <span class="trait">Development</span> in <italic> Arabidopsis </italic></h2>
			<h3>PMID:24465446</h3>
				

				
					<h3>PMCID:3900428</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Quach </surname>  <given-names> Truyen N. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Hanh TM. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Rajesh </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn3">  <sup> &#x000a4;c </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neelakandan </surname>  <given-names> Anjanasree K. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn4">  <sup> &#x000a4;d </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guttikonda </surname>  <given-names> Satish Kumar </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn5">  <sup> &#x000a4;e </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharp </surname>  <given-names> Robert E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, the role of water stress&#x02013;inducible soybean <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the enhancement of lateral <span class="trait">root</span> <span class="trait">development</span> under water deficit conditions was investigated.
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> showed an increase in lateral <span class="trait">root</span> number and <span class="trait">length</span> under non-stress conditions and maintained higher lateral <span class="trait">root</span> number and <span class="trait">length</span> under mild water stress conditions compared to the wild-type (WT), while the transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> did not show any response.
					</p>
				
					<p>&nbsp;&nbsp;
						However, LR <span class="trait">development</span> of <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						In the treatment with ABA, LR density of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> was less suppressed than that of the WT, suggesting that <italic> <span class="gene">GmNAC004</span> </italic> counteracts ABA-induced inhibition of lateral <span class="trait">root</span> <span class="trait">development</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the treatment with 2,4-D, lateral <span class="trait">root</span> density was enhanced in both <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants but the promotion was higher in the transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Conversely, in the treatment with naphthylphthalamic acid (NPA), lateral <span class="trait">root</span> density was inhibited and there was no difference in the phenotype of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants, indicating that auxin is required for the action of <italic> <span class="gene">GmNAC004</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript analysis for a number of known auxin and ABA related genes showed that <italic> <span class="gene">GmNAC004</span> </italic> 's role may suppress ABA signaling but promote auxin signaling to increase lateral <span class="trait">root</span> <span class="trait">development</span> in the <italic> Arabidopsis </italic> heterologous system.
					</p>
				
					<p>&nbsp;&nbsp;
						Consistently, down-regulation of <italic> <span class="gene">NAC1</span> </italic> reduced LR <span class="trait">development</span> in transgenic <italic> Arabidopsis </italic> overexpressing the <italic> TIR1 </italic> gene <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> were also shown to specifically express in roots, suggesting that they might play a role in the regulation of growth and <span class="trait">development</span> of the <span class="trait">root</span> system in response to water deficit conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						With the hypothesis that water stress-inducible <italic> <span class="gene">GmNAC</span> </italic> genes might have a role in the enhanced <span class="trait">development</span> of LRs in soybean under mild water stress, the objective of the present study was to investigate the functions of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in LR <span class="trait">development</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, the <italic> Arabidopsis </italic> model has been used successfully for functional characterization of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 and soybean <span class="gene">GmNAC20</span> in stimulating lateral <span class="trait">root</span> <span class="trait">development</span> <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, we found that <italic> <span class="gene">GmNAC004</span> </italic> and <italic> <span class="gene">GmNAC003</span> </italic> had an increased expression level under auxin treatment and promoted LR number and total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants under well-watered conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> maintained a higher LR number and greater total LR <span class="trait">length</span> compared to the WT under mild water deficit, but there was no promotion of LR <span class="trait">development</span> under water deficit conditions when compared to the well-watered control.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC004</span> appears to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic> under normal growth conditions via suppression of ABA signaling and promotion of auxin signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						For transcript analysis of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in response to <span class="trait">drought,</span> abiotic stresses and hormonal treatments, nine soybean internal controls ( <xref ref-type="supplementary-material" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						s003"> Table S2 </xref> ) and transcript profiling was performed for <span class="trait">root</span> samples collected from plants grown on &#x000bc;-strength <span class="gene">MS</span> nutrient agar plates.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the response of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> to water deficit in various soybean organs, a greenhouse experiment was conducted in which water was withheld to allow progressive water stress <span class="trait">development</span> to mimic slow soil-drying under <span class="trait">drought</span> conditions in the field.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> have high sequence similarity (67% identical amino acid sequences for full <span class="trait">length</span> and 92% for the 170-amino acid conserved NAC domain).
					</p>
				
					<p>&nbsp;&nbsp;
						To explore the function of <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> in LR <span class="trait">development</span> we overexpressed the two genes in the <italic> Arabidopsis </italic> model.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, because <italic> <span class="gene">GmNAC004</span> </italic> was induced by water stress and by auxin treatment, we wanted to investigate whether this gene is involved in the enhancement of LR <span class="trait">development</span> under water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						This water deficit range, therefore, was considered suitable for the investigation of the roles of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the regulation of LR growth and <span class="trait">development</span> under water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Constitutive overexpression of <italic> <span class="gene">GmNAC004</span> </italic> resulted in an increase in the number and total <span class="trait">length</span> of LRs in transgenic <italic> Arabidopsis </italic> plants compared to the WT ( <xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> and <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> ), while that of <italic> <span class="gene">GmNAC003</span> </italic> had no effect on these traits ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Similar to LR number, total LR <span class="trait">length</span> was also increased in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants compared to the WT under both well-watered (increased by 112%) and mild water stress (increased by 87&#x02013;123%) conditions ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						0084886-Tran6"> [47] </xref> ), overexpression of <italic> <span class="gene">GmNAC003</span> </italic> did not result in promotion of LR number or total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants in comparison with the WT.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the <italic> Arabidopsis </italic> lines, transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> did not show significant changes in primary <span class="trait">root</span> <span class="trait">length</span> when compared to the WT at any tested water potentials ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4D </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Because the <italic> <span class="gene">GmNAC004</span> </italic> gene was induced by both ABA and water stress ( <xref ref-type="fig" rid="pone-0084886-g001"> Figures 1 </xref> and <xref ref-type="fig" rid="pone-0084886-g002"> 2 </xref> ), <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> via an ABA-signaling pathway.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of <italic> <span class="gene">GmNAC004</span> </italic> in counteracting the inhibition of seed germination by ABA ( <xref ref-type="fig" rid="pone-0084886-g006"> Figure 6 </xref> ) raises the possibility that the <span class="gene">GmNAC004</span> protein may also counteract ABA inhibition of LR <span class="trait">development,</span> which has been reported under both normal and stress conditions <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						This hypothesis was tested by examining the effects of applied ABA on LR <span class="trait">development</span> in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants under well-watered conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						To study the interaction of <span class="gene">GmNAC004</span> and auxin signaling on LR <span class="trait">development,</span> we applied the synthetic auxin 2,4-D and the polar auxin transport inhibitor NPA to the growing media and measured <span class="trait">root</span> growth traits.
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, these results demonstrate that the stimulation of LR <span class="trait">development</span> by <span class="gene">GmNAC004</span> requires auxin and that <span class="gene">GmNAC004</span> stimulates LR <span class="trait">development</span> via ABA-dependent and auxin-dependent signaling pathways.
					</p>
				
					<p>&nbsp;&nbsp;
						To further elucidate the involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development,</span> expression of a number of representative genes in ABA and auxin signaling pathways were quantified.
					</p>
				
					<p>&nbsp;&nbsp;
						Alteration of auxin and ABA related genes indicates that <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> through auxin and ABA signaling.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 transcription factors stimulate LR <span class="trait">development</span> through auxin signaling <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						0084886-Hao1"> [37] </xref> recently reported that a stress inducible <span class="gene">GmNAC20</span> stimulated LR <span class="trait">development</span> and stress tolerance in the <italic> Arabidopsis </italic> heterologous system.
					</p>
				
					<p>&nbsp;&nbsp;
						These results suggest that <span class="gene">GmNAC004</span> transcription factor have a role in regulation of plant <span class="trait">development</span> in response to environmental stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Using <italic> Arabidopsis in planta </italic> study, a role of <span class="gene">GmNAC004</span> in governing LR number and <span class="trait">development</span> was apparent.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> &#x00026; <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> show that <span class="gene">GmNAC004</span> significantly promoted LR number under non-stress conditions and that the transgenic plants were able to maintain higher LR numbers and total LR <span class="trait">length</span> than the WT under mild water deficit conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						The involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> through the ABA- and auxin-signaling pathways was also evident.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> is supported by the report of De Smet <italic> et al.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AIR3 </italic> was an auxin responsive gene which acts downstream of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and soybean <span class="gene">GmNAC20</span> in promotion of LR <span class="trait">development</span> in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Collectively, these data indicate that <span class="gene">GmNAC004</span> may enhance auxin signaling but suppress ABA signaling to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Although transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> had greater LR number and <span class="trait">length</span> than the WT under both well-watered and mild water stress conditions, they did not show promotion of LR in response to water stress when compared to the well-watered condition ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						There are two possibilities: (i) the soybean <span class="gene">GmNAC004</span> protein does not have a function in increased LR <span class="trait">development</span> in soybean under water stress; or (ii) it has a function, but it is unable to increase LR number in <italic> Arabidopsis </italic> under water stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, the findings of this study indicate that the soybean <span class="gene">GmNAC004</span> TF has a role in stimulating LR number and total LR <span class="trait">length</span> under non-stress conditions and maintains greater LR <span class="trait">development</span> under mild water stress, compared to the WT.
					</p>
				
					<p>&nbsp;&nbsp;
						However, LR <span class="trait">development</span> of <span class="gene">GmNAC004</span> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						Nevertheless, <italic> <span class="gene">GmNAC004</span> </italic> can potentially be used for genetic engineering for a higher lateral <span class="trait">root</span> number, at least under normal growth conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, the role of water stress&#x02013;inducible soybean <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the enhancement of lateral <span class="trait">root</span> <span class="trait">development</span> under water deficit conditions was investigated
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> showed an increase in lateral <span class="trait">root</span> number and <span class="trait">length</span> under non-stress conditions and maintained higher lateral <span class="trait">root</span> number and <span class="trait">length</span> under mild water stress conditions compared to the wild-type (WT), while the transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> did not show any response
					</p>
				
					<p>&nbsp;&nbsp;
						However, LR <span class="trait">development</span> of <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment
					</p>
				
					<p>&nbsp;&nbsp;
						In the treatment with ABA, LR density of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> was less suppressed than that of the WT, suggesting that <italic> <span class="gene">GmNAC004</span> </italic> counteracts ABA-induced inhibition of lateral <span class="trait">root</span> <span class="trait">development</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In the treatment with 2,4-D, lateral <span class="trait">root</span> density was enhanced in both <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants but the promotion was higher in the transgenic plants
					</p>
				
					<p>&nbsp;&nbsp;
						Conversely, in the treatment with naphthylphthalamic acid (NPA), lateral <span class="trait">root</span> density was inhibited and there was no difference in the phenotype of the <italic> <span class="gene">GmNAC004</span> </italic> transgenic <italic> Arabidopsis </italic> and WT plants, indicating that auxin is required for the action of <italic> <span class="gene">GmNAC004</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript analysis for a number of known auxin and ABA related genes showed that <italic> <span class="gene">GmNAC004</span> </italic> 's role may suppress ABA signaling but promote auxin signaling to increase lateral <span class="trait">root</span> <span class="trait">development</span> in the <italic> Arabidopsis </italic> heterologous system
					</p>
				
					<p>&nbsp;&nbsp;
						Consistently, down-regulation of <italic> <span class="gene">NAC1</span> </italic> reduced LR <span class="trait">development</span> in transgenic <italic> Arabidopsis </italic> overexpressing the <italic> TIR1 </italic> gene <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> were also shown to specifically express in roots, suggesting that they might play a role in the regulation of growth and <span class="trait">development</span> of the <span class="trait">root</span> system in response to water deficit conditions
					</p>
				
					<p>&nbsp;&nbsp;
						With the hypothesis that water stress-inducible <italic> <span class="gene">GmNAC</span> </italic> genes might have a role in the enhanced <span class="trait">development</span> of LRs in soybean under mild water stress, the objective of the present study was to investigate the functions of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in LR <span class="trait">development</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, the <italic> Arabidopsis </italic> model has been used successfully for functional characterization of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 and soybean <span class="gene">GmNAC20</span> in stimulating lateral <span class="trait">root</span> <span class="trait">development</span> <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, we found that <italic> <span class="gene">GmNAC004</span> </italic> and <italic> <span class="gene">GmNAC003</span> </italic> had an increased expression level under auxin treatment and promoted LR number and total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants under well-watered conditions
					</p>
				
					<p>&nbsp;&nbsp;
						Transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> maintained a higher LR number and greater total LR <span class="trait">length</span> compared to the WT under mild water deficit, but there was no promotion of LR <span class="trait">development</span> under water deficit conditions when compared to the well-watered control
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC004</span> appears to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic> under normal growth conditions via suppression of ABA signaling and promotion of auxin signaling
					</p>
				
					<p>&nbsp;&nbsp;
						For transcript analysis of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> in response to <span class="trait">drought,</span> abiotic stresses and hormonal treatments, nine soybean internal controls ( <xref ref-type="supplementary-material" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						s003"> Table S2 </xref> ) and transcript profiling was performed for <span class="trait">root</span> samples collected from plants grown on &#x000bc;-strength <span class="gene">MS</span> nutrient agar plates
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate the response of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> to water deficit in various soybean organs, a greenhouse experiment was conducted in which water was withheld to allow progressive water stress <span class="trait">development</span> to mimic slow soil-drying under <span class="trait">drought</span> conditions in the field
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> have high sequence similarity (67% identical amino acid sequences for full <span class="trait">length</span> and 92% for the 170-amino acid conserved NAC domain)
					</p>
				
					<p>&nbsp;&nbsp;
						To explore the function of <span class="gene">GmNAC003</span> and <span class="gene">GmNAC004</span> in LR <span class="trait">development</span> we overexpressed the two genes in the <italic> Arabidopsis </italic> model
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, because <italic> <span class="gene">GmNAC004</span> </italic> was induced by water stress and by auxin treatment, we wanted to investigate whether this gene is involved in the enhancement of LR <span class="trait">development</span> under water stress
					</p>
				
					<p>&nbsp;&nbsp;
						This water deficit range, therefore, was considered suitable for the investigation of the roles of <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> genes in the regulation of LR growth and <span class="trait">development</span> under water stress
					</p>
				
					<p>&nbsp;&nbsp;
						Constitutive overexpression of <italic> <span class="gene">GmNAC004</span> </italic> resulted in an increase in the number and total <span class="trait">length</span> of LRs in transgenic <italic> Arabidopsis </italic> plants compared to the WT ( <xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> and <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> ), while that of <italic> <span class="gene">GmNAC003</span> </italic> had no effect on these traits ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Similar to LR number, total LR <span class="trait">length</span> was also increased in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants compared to the WT under both well-watered (increased by 112%) and mild water stress (increased by 87&#x02013;123%) conditions ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4C </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						0084886-Tran6"> [47] </xref> ), overexpression of <italic> <span class="gene">GmNAC003</span> </italic> did not result in promotion of LR number or total LR <span class="trait">length</span> in transgenic <italic> Arabidopsis </italic> plants in comparison with the WT
					</p>
				
					<p>&nbsp;&nbsp;
						Among the <italic> Arabidopsis </italic> lines, transgenic plants overexpressing <italic> <span class="gene">GmNAC003</span> </italic> and <italic> <span class="gene">GmNAC004</span> </italic> did not show significant changes in primary <span class="trait">root</span> <span class="trait">length</span> when compared to the WT at any tested water potentials ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4D </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Because the <italic> <span class="gene">GmNAC004</span> </italic> gene was induced by both ABA and water stress ( <xref ref-type="fig" rid="pone-0084886-g001"> Figures 1 </xref> and <xref ref-type="fig" rid="pone-0084886-g002"> 2 </xref> ), <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> via an ABA-signaling pathway
					</p>
				
					<p>&nbsp;&nbsp;
						The role of <italic> <span class="gene">GmNAC004</span> </italic> in counteracting the inhibition of seed germination by ABA ( <xref ref-type="fig" rid="pone-0084886-g006"> Figure 6 </xref> ) raises the possibility that the <span class="gene">GmNAC004</span> protein may also counteract ABA inhibition of LR <span class="trait">development,</span> which has been reported under both normal and stress conditions <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						This hypothesis was tested by examining the effects of applied ABA on LR <span class="trait">development</span> in the <italic> <span class="gene">GmNAC004</span> </italic> transgenic plants under well-watered conditions
					</p>
				
					<p>&nbsp;&nbsp;
						To study the interaction of <span class="gene">GmNAC004</span> and auxin signaling on LR <span class="trait">development,</span> we applied the synthetic auxin 2,4-D and the polar auxin transport inhibitor NPA to the growing media and measured <span class="trait">root</span> growth traits
					</p>
				
					<p>&nbsp;&nbsp;
						Taken together, these results demonstrate that the stimulation of LR <span class="trait">development</span> by <span class="gene">GmNAC004</span> requires auxin and that <span class="gene">GmNAC004</span> stimulates LR <span class="trait">development</span> via ABA-dependent and auxin-dependent signaling pathways
					</p>
				
					<p>&nbsp;&nbsp;
						To further elucidate the involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development,</span> expression of a number of representative genes in ABA and auxin signaling pathways were quantified
					</p>
				
					<p>&nbsp;&nbsp;
						Alteration of auxin and ABA related genes indicates that <span class="gene">GmNAC004</span> may regulate LR <span class="trait">development</span> through auxin and ABA signaling
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> <span class="gene">NAC1</span> and AtNAC2 transcription factors stimulate LR <span class="trait">development</span> through auxin signaling <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						0084886-Hao1"> [37] </xref> recently reported that a stress inducible <span class="gene">GmNAC20</span> stimulated LR <span class="trait">development</span> and stress tolerance in the <italic> Arabidopsis </italic> heterologous system
					</p>
				
					<p>&nbsp;&nbsp;
						These results suggest that <span class="gene">GmNAC004</span> transcription factor have a role in regulation of plant <span class="trait">development</span> in response to environmental stress
					</p>
				
					<p>&nbsp;&nbsp;
						Using <italic> Arabidopsis in planta </italic> study, a role of <span class="gene">GmNAC004</span> in governing LR number and <span class="trait">development</span> was apparent
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0084886-g004"> Figures 4B </xref> &#x00026; <xref ref-type="fig" rid="pone-0084886-g005"> 5 </xref> show that <span class="gene">GmNAC004</span> significantly promoted LR number under non-stress conditions and that the transgenic plants were able to maintain higher LR numbers and total LR <span class="trait">length</span> than the WT under mild water deficit conditions
					</p>
				
					<p>&nbsp;&nbsp;
						The involvement of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> through the ABA- and auxin-signaling pathways was also evident
					</p>
				
					<p>&nbsp;&nbsp;
						The role of <span class="gene">GmNAC004</span> in regulation of LR <span class="trait">development</span> is supported by the report of De Smet <italic> et al
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> AIR3 </italic> was an auxin responsive gene which acts downstream of <italic> Arabidopsis </italic> <span class="gene">NAC1</span> and soybean <span class="gene">GmNAC20</span> in promotion of LR <span class="trait">development</span> in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Collectively, these data indicate that <span class="gene">GmNAC004</span> may enhance auxin signaling but suppress ABA signaling to regulate LR <span class="trait">development</span> in <italic> Arabidopsis </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Although transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmNAC004</span> </italic> had greater LR number and <span class="trait">length</span> than the WT under both well-watered and mild water stress conditions, they did not show promotion of LR in response to water stress when compared to the well-watered condition ( <xref ref-type="fig" rid="pone-0084886-g004"> Figure 4B </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						There are two possibilities: (i) the soybean <span class="gene">GmNAC004</span> protein does not have a function in increased LR <span class="trait">development</span> in soybean under water stress; or (ii) it has a function, but it is unable to increase LR number in <italic> Arabidopsis </italic> under water stress
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, the findings of this study indicate that the soybean <span class="gene">GmNAC004</span> TF has a role in stimulating LR number and total LR <span class="trait">length</span> under non-stress conditions and maintains greater LR <span class="trait">development</span> under mild water stress, compared to the WT
					</p>
				
					<p>&nbsp;&nbsp;
						However, LR <span class="trait">development</span> of <span class="gene">GmNAC004</span> transgenic <italic> Arabidopsis </italic> plants was not enhanced in the water-stressed compared to the well-watered treatment
					</p>
				
					<p>&nbsp;&nbsp;
						Nevertheless, <italic> <span class="gene">GmNAC004</span> </italic> can potentially be used for genetic engineering for a higher lateral <span class="trait">root</span> number, at least under normal growth conditions
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean salicylic acid methyltransferase gene confers <span class="trait">resistance</span> to soybean cyst nematode.</h2>
			<h3>PMID:24034273  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M, Zhao N, Zhu JJ, Zhuang X, Liu W, Pantalone VR, Arelli PR, Stewart CN Jr, Chen F.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To determine the role of <span class="gene">GmSAMT1</span> in soybean defence against SCN, transgenic hairy roots overexpressing <span class="gene">GmSAMT1</span> were produced and tested for SCN <span class="trait">resistance</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <span class="gene">GmSAMT1</span> in SCN-susceptible backgrounds significantly reduced the <span class="trait">development</span> of SCN, indicating that overexpression of <span class="gene">GmSAMT1</span> in the transgenic hairy <span class="trait">root</span> system could confer <span class="trait">resistance</span> to SCN.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The effects of dopamine on <span class="trait">root</span> growth and enzyme activity in soybean seedlings.</h2>
			<h3>PMID:23838960  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guidotti BB(1), Gomes BR, Siqueira-Soares Rde C, Soares AR, Ferrarese-Filho O.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In general, the <span class="trait">length,</span> fresh weight and dry weight of roots, cell viability, <span class="gene">PAL</span> and POD activities decreased, while SOD activities increased after dopamine treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protocol for augmented shoot organogenesis in selected variety of soybean [Glycine max L. (Merr.)].</h2>
			<h3>PMID:23214267  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Devi MK(1), Sakthivelu G, Giridhar P, Ravishankar GA.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Using cotyledonary node explants significant organogenic responses, mean shoot number and shoot <span class="trait">length</span> were observed when these were incubated on <span class="gene">MS</span> medium supplemented with 0.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide survey and expression analysis of the plant-specific NAC transcription factor family in soybean during <span class="trait">development</span> and dehydration stress.</h2>
			<h3>PMID:21685489  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Le DT(1), Nishiyama R, Watanabe Y, Mochida K, Yamaguchi-Shinozaki K, Shinozaki K, Tran LS.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Our systematic analysis has identified excellent tissue-specific and/or dehydration-responsive candidate <span class="gene">GmNAC</span> genes for in-depth characterization and future <span class="trait">development</span> of improved drought-tolerant transgenic soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich protein) gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage.
					</p>
				
					<p>&nbsp;&nbsp;
						Our data suggest that <span class="gene">GmPGYRP</span> genes encoding a class of conservative XYPPX-repeat proteins probably play an important role in plant <span class="trait">development</span> as well as in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean root-specific protein kinase <span class="gene">GmWNK1</span> regulates stress-responsive ABA signaling on the <span class="trait">root</span> system architecture.</h2>
			<h3>PMID:20735771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang Y(1), Suo H, Zheng Y, Liu K, Zhuang C, Kahle KT, Ma H, Yan X.
				<br/> 
				年份：2010 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmWNK1</span> expression was detected in the <span class="trait">root,</span> specifically in <span class="trait">root</span> cells associated with lateral <span class="trait">root</span> formation, and was down-regulated by abscisic acid (ABA), as well as by mannitol, sucrose, polyethylene glycol and NaCl.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, 35S-GmWNK1 transgenic soybean plants had reduced lateral <span class="trait">root</span> number and <span class="trait">length</span> compared with wild-type, suggesting a role of <span class="gene">GmWNK1</span> in the regulation of <span class="trait">root</span> system architecture.
					</p>
				
					<p>&nbsp;&nbsp;
						We propose that <span class="gene">GmWNK1</span> functions to fine-tune ABA-dependent ABA homeostasis, thereby mediating the regulation of the <span class="trait">root</span> system architecture by ABA and osmotic signals.
					</p>
				
					<p>&nbsp;&nbsp;
						The study has revealed a new function of a plant <span class="gene">WNK1</span> gene from the important staple crop soybean, and has identified a new component of a regulatory pathway that is involved not only in ABA signaling, but also in the repression of lateral <span class="trait">root</span> formation by an ABA-dependent mechanism distinct from known ABA signaling pathways.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cloning and characterization of an aldehyde dehydrogenase gene <span class="gene">GmALDH3-1</span> from soybean].</h2>
			<h3>PMID:20466639  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huang F(1), Chi YJ, He H, Yu DY.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmALDH3-1</span> contains a complete open reading frame of 1485 bp in <span class="trait">length,</span> which encodes for a peptide of 494 amino acids.
					</p>
				
					<p>&nbsp;&nbsp;
						We further analyzed <span class="gene">GmALDH3-1</span> expression during the course of seed <span class="trait">development</span> based on publicly available microarray data and found that <span class="gene">GmALDH3-1</span> was highly expressed in the seed endothelium, epidermis, outer integument, and hilum.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:1-Aminocyclopropane-1-carboxylic acid (ACC) concentration and ACC synthase expression in soybean roots, <span class="trait">root</span> tips, and soybean cyst nematode (Heterodera glycines)-infected roots.</h2>
			<h3>PMID:19861652  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tucker ML(1), Xue P, Yang R.
				<br/> 
				年份：2010
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Moreover, expression analysis of 17 ACC synthase (ACS) genes indicated that a select set of <span class="gene">ACS</span> genes is expressed in SCN-colonized <span class="trait">root</span> pieces that is clearly different from the set of genes expressed in non-colonized roots or <span class="trait">root</span> tips.
					</p>
				
					<p>&nbsp;&nbsp;
						Semi-quantitative real-time PCR indicated that <span class="gene">ACS</span> transcript accumulation correlates with the high concentration of ACC in <span class="trait">root</span> tips.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A RAV-like transcription factor controls photosynthesis and senescence in soybean.</h2>
			<h3>PMID:18297307  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao L(1), Luo Q, Yang C, Han Y, Li W.
				<br/> 
				年份：2008 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transgenic tobacco overexpressing <span class="gene">GmRAV</span> showed morphological and physiological alterations such as slower plant growth rate (dwarfing), reduced <span class="trait">root</span> elongation, delayed flowering time and reduced photosynthetic rate, reduced chlorophyll contents in leaves.
					</p>
				
					<p>&nbsp;&nbsp;
						The analyses of soybean leaf, <span class="trait">root</span> and stem organs showed that <span class="gene">GmRAV</span> mRNA abundances were higher in SDs than in LDs.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the enhanced expression of <span class="gene">GmRAV</span> in SDs compared to LDs may have caused the inhibited growth of soybean leaf, <span class="trait">root</span> and stem.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:L-DOPA increases lignification associated with Glycine max <span class="trait">root</span> growth-inhibition.</h2>
			<h3>PMID:17195115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Soares AR(1), Ferrarese Mde L, Siqueira Rde C, Böhm FM, Ferrarese-Filho O.
				<br/> 
				年份：2007 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In general, the <span class="trait">length,</span> fresh weight, and dry weight of the roots decreased, whereas <span class="gene">PAL</span> and POD activities and phenolic compound and lignin content increased after L-DOPA treatments.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure, expression, and mapping of two nodule-specific genes identified by mining public soybean EST databases.</h2>
			<h3>PMID:16973305  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong SC(1), Yang K, Park JY, Han KS, Yu S, Hwang TY, Hur CG, Kim SH, Park PB, Kim HM, Park YI, Liu JR.
				<br/> 
				年份：2006 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genetic mapping located <span class="gene">GmPGN</span> near the known quantitative trait locus conferring <span class="trait">resistance</span> to soybean cyst nematode on soybean molecular linkage group (MLG) B1, and <span class="gene">GmEKN</span> on MLG A2.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lignification and related enzymes in Glycine max <span class="trait">root</span> growth-inhibition by ferulic acid.</h2>
			<h3>PMID:15303323  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：dos Santos WD(1), Ferrarese Mde L, Finger A, Teixeira AC, Ferrarese-Filho O.
				<br/> 
				年份：2004 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Length,</span> fresh weight, and dry weight of roots decreased, while soluble and cell wall bound POD activity, <span class="gene">PAL</span> activity, and lignin content increased after ferulic acid treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peroxidase and phenylalanine ammonia-lyase activities, phenolic acid contents, and allelochemicals-inhibited <span class="trait">root</span> growth of soybean.</h2>
			<h3>PMID:12125206  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Herrig V(1), Ferrarese Mde L, Suzuki LS, Rodrigues JD, Ferrarese-Filho O.
				<br/> 
				年份：2002
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The results are discussed on the basis of the role of these compounds on phenylpropanoid metabolism and <span class="trait">root</span> growth and suggest that the effects caused on POD and <span class="gene">PAL</span> activities are some of the many mechanisms by which allelochemicals influence plant growth.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome co-expression analysis of soybean cytochrome <span class="gene">P450</span> genes identifies nodulation-specific <span class="gene">P450</span> monooxygenases</h2>
			<h3>PMID:21062474</h3>
				

				
					<h3>PMCID:3095325</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Guttikonda </surname>  <given-names> Satish K </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> SKGuttikonda@dow.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Trupti </surname>  <given-names> Joshi </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> joshitr@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Bisht </surname>  <given-names> Naveen C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> NBisht@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Chen </surname>  <given-names> Hui </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> huchen@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> An </surname>  <given-names> Yong-Qiang C </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> yong-qiang.an@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Pandey </surname>  <given-names> Sona </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> SPandey@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xudong@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Yu </surname>  <given-names> Oliver </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> oyu@danforthcenter.org </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 11 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Similarly, <italic> CYP736A34 </italic> was highly co-expressed with lipoxygenase, lectin and <italic> <span class="gene">CYP83D1</span> </italic> , all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development</span>
					</p>
				
					<p>&nbsp;&nbsp;
						We were able to identify tissue-specific <span class="gene">P450</span> genes that may play roles in biological process like nodulation, floral <span class="trait">development,</span> and seed maturation
					</p>
				
					<p>&nbsp;&nbsp;
						More specifically, <italic> <span class="gene">IFS1</span> </italic> is mainly expressed in the <span class="trait">root</span> and seed coat; while <italic> <span class="gene">IFS2</span> </italic> is mainly expressed in embryos and pods
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of soybean <span class="gene">P450</span> genes in <span class="trait">root</span> hair infection process according to the analysis of Illumina transcriptome dataset
					</p>
				
					<p>&nbsp;&nbsp;
						The most abundant <span class="gene">P450</span> genes in each tissue, as shown by the calculated tag sequence numbers, are <italic> CYP83G4 </italic> (nodules), <italic> CYP76X6 </italic> (apical meristem, green pods and leaves), <italic> CYP83E12 </italic> (flowers), and <italic> CYP93C5 </italic> (roots and <span class="trait">root</span> tips)
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> qRT-PCR confirmation of soybean <span class="gene">P450</span> genes highly expressed in <span class="trait">root</span> and nodule as revealed by the analysis of Microarray and Solexa transcriptome datasets
					</p>
				
					<p>&nbsp;&nbsp;
						Co-expression analysis showed that the expression of this gene is highly correlated with lipoxygenase, lectin and <italic> <span class="gene">CYP83D1</span> </italic> , all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> )
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We were able to identify tissue-specific <span class="gene">P450</span> genes that may play roles in biological process like nodulation, floral <span class="trait">development,</span> and seed maturation.
					</p>
				
					<p>&nbsp;&nbsp;
						More specifically, <italic> <span class="gene">IFS1</span> </italic> is mainly expressed in the <span class="trait">root</span> and seed coat; while <italic> <span class="gene">IFS2</span> </italic> is mainly expressed in embryos and pods.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression of soybean <span class="gene">P450</span> genes in <span class="trait">root</span> hair infection process according to the analysis of Illumina transcriptome dataset.
					</p>
				
					<p>&nbsp;&nbsp;
						The most abundant <span class="gene">P450</span> genes in each tissue, as shown by the calculated tag sequence numbers, are <italic> CYP83G4 </italic> (nodules), <italic> CYP76X6 </italic> (apical meristem, green pods and leaves), <italic> CYP83E12 </italic> (flowers), and <italic> CYP93C5 </italic> (roots and <span class="trait">root</span> tips).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> qRT-PCR confirmation of soybean <span class="gene">P450</span> genes highly expressed in <span class="trait">root</span> and nodule as revealed by the analysis of Microarray and Solexa transcriptome datasets.
					</p>
				
					<p>&nbsp;&nbsp;
						Co-expression analysis showed that the expression of this gene is highly correlated with lipoxygenase, lectin and <italic> <span class="gene">CYP83D1</span> </italic> , all of which are involved in <span class="trait">root</span> and nodule <span class="trait">development</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:RNA-seq analyses of multiple meristems of soybean: novel and alternative transcripts, evolutionary and functional implications</h2>
			<h3>PMID:24939556</h3>
				

				
					<h3>PMCID:4070088</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Wang </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wangleibio@gmail.com </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Cao </surname>  <given-names> Chenlong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> 11210700144@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Ma </surname>  <given-names> Qibin </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> maqibin@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Zeng </surname>  <given-names> Qiaoying </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> zengqiaoying123@163.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Wang </surname>  <given-names> Haifeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> whfwind@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Cheng </surname>  <given-names> Zhihao </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> zhihaocheng1@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Zhu </surname>  <given-names> Genfeng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> gfzhu@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Qi </surname>  <given-names> Ji </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> qij@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Ma </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> hongma@fudan.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> hnian@scau.edu.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A11">  <name>  <surname> Wang </surname>  <given-names> Yingxiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> yx_wang@fudan.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, as shown in Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> a, the 7th exon of <italic> <span class="gene">Gm15g15960</span> </italic> , showed &#x003a8; with 88% in <span class="trait">root</span> tip and 6% in cotyledon (in Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Table S11), suggesting divergent functions between <span class="trait">root</span> tip and cotyledon
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> (c) </bold> Alternative splicing in <italic> <span class="gene">Gm05g28120</span> </italic> was regulated during SAM <span class="trait">development</span>
					</p>
				
					<p>&nbsp;&nbsp;
						We detected several other auxin-related genes, including the pair of <italic> <span class="gene">Gm09g38700</span> </italic> and <italic> <span class="gene">Glyma18g47630</span> </italic> paralogs that are homologs of <italic> Arabidopsis PIN-FORMED 5 (PIN5) </italic> , which is required for auxin homoeostasis and gametophyte <span class="trait">development</span> [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						We also found homologs ( <italic> <span class="gene">Gm14g15820</span> </italic> and <italic> <span class="gene">Gm7g30920</span> </italic> ) of genes for auxin synthesis and response, such as <italic> YUCCA4 </italic> , in accordance with the fact that the <italic> Arabidopsis YUCCA4 </italic> expression is restricted to the SAM and flower meristems or young floral primordia [ <xref ref-type="bibr" rid="B55"> 55 </xref> ], as well as 20 genes related to auxin-responsive genes regulating SAM <span class="trait">development</span> [ <xref ref-type="bibr" rid="B56"> 56 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, the <italic> Arabidopsis DREB1B </italic> is one of the critical regulators for cold responses, and is also widely expressed [ <xref ref-type="bibr" rid="B58"> 58 </xref> ], whereas the soybean homologs ( <italic> <span class="gene">Gm11g19340</span> </italic> and <italic> <span class="gene">Gm12g09130</span> </italic> ) showed special expression in AM, IBM and IAM, but not in other vegetative tissues, suggesting it might have gained novel functions in reproductive <span class="trait">development</span> in soybean
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, one homolog of <italic> DREB1A </italic> ( <italic> <span class="gene">Gm17g14110</span> </italic> ) was also identified, consistent with a recent novel discovery that the <italic> Arabidopsis DREB1A </italic> gene is important for flower <span class="trait">development</span> especially under unfavorable conditions [ <xref ref-type="bibr" rid="B59"> 59 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Further classification of family-preferential expression showed that G1 mainly includes families of HMG, FHA, ZF-HD, SBP, TCP, C2C2 (Zn) <span class="gene">GATA</span> and PHD (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), indicating that early SAM <span class="trait">development</span> largely requires those transcription factor families
					</p>
				
					<p>&nbsp;&nbsp;
						For example, <italic> SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) </italic> proteins are a family of plant-specific TFs having a conserved <span class="gene">SBP</span> ( <italic> SQUAMOSA </italic> promoter binding protein) domain, and play multiple roles in plant growth and <span class="trait">development</span> [ <xref ref-type="bibr" rid="B63"> 63 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Our data detected 101 MADS-box genes during flower <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), such as <italic> <span class="gene">Gm01g08150,</span> <span class="gene">Gm04g42420,</span> <span class="gene">Gm08g12730</span> </italic> and <italic> <span class="gene">Gm08g27670</span> </italic> , which are homologs of <italic> AP1, PI, AG </italic> and <italic> SEP2 </italic> , respectively, consistent with their known function in floral organ identity
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized gene expression level was calculated as Reads Per Kilo-base of mRNA <span class="trait">length</span> per Millions of mapped reads (RPKM) by the GFOLD <span class="gene">V1</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> (c) </bold> Alternative splicing in <italic> <span class="gene">Gm05g28120</span> </italic> was regulated during SAM <span class="trait">development</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						We detected several other auxin-related genes, including the pair of <italic> <span class="gene">Gm09g38700</span> </italic> and <italic> <span class="gene">Glyma18g47630</span> </italic> paralogs that are homologs of <italic> Arabidopsis PIN-FORMED 5 (PIN5) </italic> , which is required for auxin homoeostasis and gametophyte <span class="trait">development</span> [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						We also found homologs ( <italic> <span class="gene">Gm14g15820</span> </italic> and <italic> <span class="gene">Gm7g30920</span> </italic> ) of genes for auxin synthesis and response, such as <italic> YUCCA4 </italic> , in accordance with the fact that the <italic> Arabidopsis YUCCA4 </italic> expression is restricted to the SAM and flower meristems or young floral primordia [ <xref ref-type="bibr" rid="B55"> 55 </xref> ], as well as 20 genes related to auxin-responsive genes regulating SAM <span class="trait">development</span> [ <xref ref-type="bibr" rid="B56"> 56 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, the <italic> Arabidopsis DREB1B </italic> is one of the critical regulators for cold responses, and is also widely expressed [ <xref ref-type="bibr" rid="B58"> 58 </xref> ], whereas the soybean homologs ( <italic> <span class="gene">Gm11g19340</span> </italic> and <italic> <span class="gene">Gm12g09130</span> </italic> ) showed special expression in AM, IBM and IAM, but not in other vegetative tissues, suggesting it might have gained novel functions in reproductive <span class="trait">development</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, one homolog of <italic> DREB1A </italic> ( <italic> <span class="gene">Gm17g14110</span> </italic> ) was also identified, consistent with a recent novel discovery that the <italic> Arabidopsis DREB1A </italic> gene is important for flower <span class="trait">development</span> especially under unfavorable conditions [ <xref ref-type="bibr" rid="B59"> 59 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Further classification of family-preferential expression showed that G1 mainly includes families of HMG, FHA, ZF-HD, SBP, TCP, C2C2 (Zn) <span class="gene">GATA</span> and PHD (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), indicating that early SAM <span class="trait">development</span> largely requires those transcription factor families.
					</p>
				
					<p>&nbsp;&nbsp;
						For example, <italic> SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) </italic> proteins are a family of plant-specific TFs having a conserved <span class="gene">SBP</span> ( <italic> SQUAMOSA </italic> promoter binding protein) domain, and play multiple roles in plant growth and <span class="trait">development</span> [ <xref ref-type="bibr" rid="B63"> 63 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Our data detected 101 MADS-box genes during flower <span class="trait">development</span> (Figure&#x000a0; <xref ref-type="fig" rid="F7"> 7 </xref> b), such as <italic> <span class="gene">Gm01g08150,</span> <span class="gene">Gm04g42420,</span> <span class="gene">Gm08g12730</span> </italic> and <italic> <span class="gene">Gm08g27670</span> </italic> , which are homologs of <italic> AP1, PI, AG </italic> and <italic> SEP2 </italic> , respectively, consistent with their known function in floral organ identity.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized gene expression level was calculated as Reads Per Kilo-base of mRNA <span class="trait">length</span> per Millions of mapped reads (RPKM) by the GFOLD <span class="gene">V1</span>.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Novel QTL Governing <span class="trait">Root</span> Architectural Traits in an Interspecific Soybean Population</h2>
			<h3>PMID:25756528</h3>
				

				
					<h3>PMCID:4355624</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Manavalan </surname>  <given-names> Lakshmi P. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Prince </surname>  <given-names> Silvas J. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Musket </surname>  <given-names> Theresa A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chaky </surname>  <given-names> Julian </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> James C. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Specht </surname>  <given-names> James E. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						a) Variation in <span class="trait">root</span> architecture in soybean accessions 12 days after sowing (das); b) Seed coat color of parents (Dunbar, <span class="gene">P1</span> and PI 326582A, P2) and RILs c) Cone system for <span class="trait">root</span> screening d) Dunbar 12 das e) PI 326582A 12 das
					</p>
				
					<p>&nbsp;&nbsp;
						The selected BILs and parental lines were grown to the <span class="gene">V1</span> stage and the <span class="trait">root</span> tissues were collected for RNA isolation
					</p>
				
					<p>&nbsp;&nbsp;
						The nuclear transport factor 2 ( <italic> <span class="gene">Glyma08g11070</span> </italic> ) showed more than a 300-fold increased expression in <span class="trait">root</span> tissues
					</p>
				
					<p>&nbsp;&nbsp;
						Although 12 transcription factor (TF) families were identified in this QTL interval, only a few TFs including Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ), GRAS ( <italic> <span class="gene">Glyma08g10140</span> </italic> ), WRKY ( <italic> <span class="gene">Glyma08g12460</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed more than a 50- fold expression in soybean <span class="trait">root</span> tissues ( <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Based on Affymetrix gene chip data, six Transcription factors (MYB HD; <italic> <span class="gene">Glyma08g12320</span> </italic> , TPR: <italic> <span class="gene">Glyma08g09550</span> </italic> , C2H2 Zn: <italic> <span class="gene">Glyma08g11800</span> </italic> , BZIP: <italic> <span class="gene">Glyma08g12170</span> </italic> , GRAS: <italic> <span class="gene">Glyma08g10140</span> </italic> and Ring finger: <italic> <span class="gene">Glyma08g13900</span> </italic> ) only showed higher expression on soybean <span class="trait">root</span> tissues ( <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Based on public soybean Affymetrix expression data, two TFs, AP2-EREBP ( <italic> <span class="gene">Glyma08g14600</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed higher expression in <span class="trait">root</span> tissues ( <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						This <span class="gene">AP2</span> TF also showed higher expression in <span class="trait">root</span> tissues (37- fold) and in nodules (13- fold) within the transcriptome data ( <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Key cell wall expansion-related genes ( <italic> <span class="gene">Glyma08g11300</span> </italic> and <italic> <span class="gene">Glyma08g09940</span> </italic> ) encoding xyloglucan endo-transglycosylases (XET) with high expression in both <span class="trait">root</span> and shoot tissues were identified
					</p>
				
					<p>&nbsp;&nbsp;
						However, the gene <italic> <span class="gene">Glyma08g11300</span> </italic> showed a higher expression in <span class="trait">root</span> tissues than <italic> <span class="gene">Glyma08g09940</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						In particular, two transcription factors like Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ) and C3H type 1 ( <italic> <span class="gene">Glyma08g09630</span> </italic> ) showed high expression in <span class="trait">root</span> tissues in both Affymetrix ( <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						6 </xref> ) the <span class="gene">AP2</span> TF as being highly expressed in shoot, <span class="trait">root,</span> and nodule tissues
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The selected BILs and parental lines were grown to the <span class="gene">V1</span> stage and the <span class="trait">root</span> tissues were collected for RNA isolation.
					</p>
				
					<p>&nbsp;&nbsp;
						The nuclear transport factor 2 ( <italic> <span class="gene">Glyma08g11070</span> </italic> ) showed more than a 300-fold increased expression in <span class="trait">root</span> tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						Although 12 transcription factor (TF) families were identified in this QTL interval, only a few TFs including Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ), GRAS ( <italic> <span class="gene">Glyma08g10140</span> </italic> ), WRKY ( <italic> <span class="gene">Glyma08g12460</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed more than a 50- fold expression in soybean <span class="trait">root</span> tissues ( <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on Affymetrix gene chip data, six Transcription factors (MYB HD; <italic> <span class="gene">Glyma08g12320</span> </italic> , TPR: <italic> <span class="gene">Glyma08g09550</span> </italic> , C2H2 Zn: <italic> <span class="gene">Glyma08g11800</span> </italic> , BZIP: <italic> <span class="gene">Glyma08g12170</span> </italic> , GRAS: <italic> <span class="gene">Glyma08g10140</span> </italic> and Ring finger: <italic> <span class="gene">Glyma08g13900</span> </italic> ) only showed higher expression on soybean <span class="trait">root</span> tissues ( <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on public soybean Affymetrix expression data, two TFs, AP2-EREBP ( <italic> <span class="gene">Glyma08g14600</span> </italic> ) and bZIP ( <italic> <span class="gene">Glyma08g12170</span> </italic> ) showed higher expression in <span class="trait">root</span> tissues ( <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						This <span class="gene">AP2</span> TF also showed higher expression in <span class="trait">root</span> tissues (37- fold) and in nodules (13- fold) within the transcriptome data ( <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Key cell wall expansion-related genes ( <italic> <span class="gene">Glyma08g11300</span> </italic> and <italic> <span class="gene">Glyma08g09940</span> </italic> ) encoding xyloglucan endo-transglycosylases (XET) with high expression in both <span class="trait">root</span> and shoot tissues were identified.
					</p>
				
					<p>&nbsp;&nbsp;
						However, the gene <italic> <span class="gene">Glyma08g11300</span> </italic> showed a higher expression in <span class="trait">root</span> tissues than <italic> <span class="gene">Glyma08g09940</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						In particular, two transcription factors like Homeobox domain ( <italic> <span class="gene">Glyma08g14130</span> </italic> ) and C3H type 1 ( <italic> <span class="gene">Glyma08g09630</span> </italic> ) showed high expression in <span class="trait">root</span> tissues in both Affymetrix ( <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						6 </xref> ) the <span class="gene">AP2</span> TF as being highly expressed in shoot, <span class="trait">root,</span> and nodule tissues.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>