<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>rust</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>rust</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Discovery of a seventh Rpp soybean <span class="trait">rust</span> <span class="trait">resistance</span> locus in soybean accession PI 605823.</h2>
			<h3>PMID:28980046</h3>
				

				
			<p>
				作者：Childs SP(1), King ZR(1), Walker DR(2)(3), Harris DK(1), Pedley KF(4), Buck JW(5), Boerma HR(1), Li Z(6).
				<br/> 
				年份：2018 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: A novel Rpp gene from PI 605823 for <span class="trait">resistance</span> to Phakopsora pachyrhizi was mapped on chromosome 19. Soybean <span class="trait">rust,</span> caused by the obligate biotrophic fungal pathogen Phakopsora pachyrhizi Syd. & P. Syd, is a disease threat to soybean production in regions of the world with mild winters. Host plant <span class="trait">resistance</span> conditioned by <span class="trait">resistance</span> to P. pachyrhizi (Rpp) genes has been found in numerous soybean accessions, and at least 10 Rpp genes or alleles have been mapped to six genetic loci. Identifying additional disease-resistance genes will facilitate development of soybean cultivars with durable <span class="trait">resistance.</span> PI 605823, a plant introduction from Vietnam, was previously identified as resistant to US populations of P. pachyrhizi in greenhouse and field trials. In this study, bulked segregant analysis using an F2 population derived from 'Williams 82' × PI 605823 identified a genomic region associated with <span class="trait">resistance</span> to P. pachyrhizi isolate GA12, which had been collected in the US State of Georgia in 2012. To further map the <span class="trait">resistance</span> locus, linkage mapping was carried out using single-nucleotide polymorphism markers and phenotypic data from greenhouse assays with an F2:3 population derived from Williams 82 × PI 605823 and an F4:5 population derived from '5601T' × PI 605823. A novel <span class="trait">resistance</span> gene, Rpp7, was mapped to a 154-kb interval <span class="gene">(Gm19:</span> 39,462,291-39,616,643 Glyma.Wm82.a2) on chromosome 19 that is different from the genomic locations of any previously reported Rpp genes. This new gene could be incorporated into elite breeding lines to help provide more durable <span class="trait">resistance</span> to soybean <span class="trait">rust.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Small Cysteine-Rich Protein from the Asian Soybean <span class="trait">Rust</span> Fungus, <italic> Phakopsora pachyrhizi </italic> , Suppresses Plant Immunity</h2>
			<h3>PMID:27676173</h3>
				

				
					<h3>PMCID:5038961</h3>
					<h4>期刊:PLoS Pathogens</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Qi </surname>  <given-names> Mingsheng </given-names>  </name>  <xref ref-type="aff" rid="aff001">
				<br/> 
				年份：<day> 26 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Asian soybean <span class="trait">rust</span> fungus, Phakopsora pachyrhizi, is an obligate biotrophic pathogen causing severe soybean disease epidemics. Molecular mechanisms by which P. pachyrhizi and other <span class="trait">rust</span> fungi interact with their host plants are poorly understood. The genomes of all <span class="trait">rust</span> fungi encode many small, secreted cysteine-rich proteins (SSCRP). While these proteins are thought to function within the host, their roles are completely unknown. Here, we present the characterization of P. pachyrhizi effector candidate 23 (PpEC23), a SSCRP that we show to suppress plant immunity. Furthermore, we show that PpEC23 interacts with soybean transcription factor <span class="gene">GmSPL12l</span> and that soybean plants in which <span class="gene">GmSPL12l</span> is silenced have constitutively active immunity, thereby identifying <span class="gene">GmSPL12l</span> as a negative regulator of soybean defenses. Collectively, our data present evidence for a virulence function of a <span class="trait">rust</span> SSCRP and suggest that PpEC23 is able to suppress soybean immune responses and physically interact with soybean transcription factor <span class="gene">GmSPL12l,</span> a negative immune regulator.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The Asian soybean <span class="trait">rust</span> fungus, <italic> Phakopsora pachyrhizi </italic> , is an obligate biotrophic pathogen causing severe soybean disease epidemics. Molecular mechanisms by which <italic> P </italic> . <italic> pachyrhizi </italic> and other <span class="trait">rust</span> fungi interact with their host plants are poorly understood. The genomes of all <span class="trait">rust</span> fungi encode many small, secreted cysteine-rich proteins (SSCRP). While these proteins are thought to function within the host, their roles are completely unknown. Here, we present the characterization of <italic> P </italic> . <italic> pachyrhizi </italic> effector candidate 23 ( <italic> Pp </italic> EC23), a SSCRP that we show to suppress plant immunity. Furthermore, we show that <italic> Pp </italic> EC23 interacts with soybean transcription factor <italic> Gm </italic> SPL12l and that soybean plants in which <italic> <span class="gene">GmSPL12l</span> </italic> is silenced have constitutively active immunity, thereby identifying <italic> Gm </italic> SPL12l as a negative regulator of soybean defenses. Collectively, our data present evidence for a virulence function of a <span class="trait">rust</span> SSCRP and suggest that <italic> Pp </italic> EC23 is able to suppress soybean immune responses and physically interact with soybean transcription factor <italic> Gm </italic> SPL12l, a negative immune regulator.
					</p>
				
					<p>&nbsp;&nbsp;
						We are interested in identifying <italic> Pp </italic> ECs that suppress immunity and characterizing the mechanisms by which they interfere with soybean immune signaling. To initiate this work, we performed functional screens designed to identify <italic> Pp </italic> ECs that suppress defense responses elicited by a bacterial pathogen, <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> ( <italic> Pst </italic> ). One SSCRP, which we named <italic> Pp </italic> EC23, was unique in its ability to suppress HR and basal defense responses. The major feature of <italic> Pp </italic> EC23 is a tandem repeat of a ten-cysteine motif that appears in the predicted secretomes of other <span class="trait">rust</span> fungi [ <xref rid="ppat.1005827.ref001" ref-type="bibr"> 1 </xref> , <xref rid="ppat.1005827.ref003" ref-type="bibr"> 3 </xref> , <xref rid="ppat.1005827.ref008" ref-type="bibr"> 8 </xref> ], and it interacts with soybean transcription factor <italic> Gm </italic> SPL12l (SQUAMOSA promoter-binding-like protein 12 like). RNA silencing of <italic> <span class="gene">GmSPL12l</span> </italic> indicates that this transcription factor functions as a negative regulator of soybean defense responses. While prior work documented <italic> avirulence </italic> functions for a few SSCRPs, <italic> i </italic> . <italic> e </italic> . their detection by the plant immune system leading to plant defenses, the current study establishes that SSCRP <italic> Pp </italic> EC23 has functions consistent with a virulence factor that regulates host plant immunity.
					</p>
				
					<p>&nbsp;&nbsp;
						The elevated expression of <italic> <span class="gene">PR1a</span> </italic> in <italic> <span class="gene">GmSPL12l</span> </italic> -silenced and <italic> NbSPL1- </italic> silenced plants is consistent with our conclusion that <italic> Pp </italic> EC23 targets a protein upstream of defense gene expression at a point of convergence between HR and basal defense, which is conserved between non-host and host plants. However, we do not know if <italic> Gm </italic> SPL12l or <italic> Nb </italic> SPL1 directly or indirectly regulate the expression of defense marker genes such as <italic> <span class="gene">PR1a</span> </italic> . In general, the genes that are direct transcriptional targets of the SPLs are not yet known. Studies to investigate this will be aided by stable mutations that knock out <italic> Gm </italic> SPL12l function and development of resources needed for performing chromatin immunoprecipitation coupled with RNA-seq. On the <span class="trait">rust</span> side of the interaction, the cluster 112 protein family characterized by the signature ten-cysteine motif is widely present in <span class="trait">rust</span> fungal species for which sequence information is available [ <xref rid="ppat.1005827.ref008" ref-type="bibr"> 8 </xref> ]. The presence of this motif suggests that <italic> Pp </italic> EC23 could be a representative of a conserved protein-protein interaction platform that <span class="trait">rust</span> fungi use to manipulate host immune responses. It will be interesting to further explore the functions of this family of SSCRPs as virulence factors in <span class="trait">rust</span> pathogen-plant interactions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Competitive Fitness of Phakopsora pachyrhizi Isolates with Mutations in the CYP51 and <span class="gene">CYTB</span> Genes.</h2>
			<h3>PMID:27359265  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Klosowski AC(1), Brahm L(1), Stammler G(1), De Mio LL(1).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">rust</span> (Phakopsora pachyrhizi) in Brazil is mainly controlled with applications of fungicides, including demethylation inhibitors (DMI) and quinone outside inhibitors (QoI). Isolates with less sensitivity to DMI and QoI have been reported, and these have been found to have mutations in the CYP51 and <span class="gene">CYTB</span> genes, respectively. There have been no reports of fitness costs in isolates with mutations in CYP51 and CYTB, and the aim of this work was to compare the competitive ability of isolates with lower DMI or QoI sensitivities with that of sensitive (wild-type) isolates. Urediniospores of sensitive wild-type isolates and isolates with different CYP51 or <span class="gene">CYTB</span> alleles were mixed and inoculated on detached soybean leaves. After 3 weeks, urediniospores were harvested and used as inoculum for the next disease cycle. Frequencies of relevant target site mutations were monitored using the pyrosequencing method over four disease cycles. Isolates with lower DMI sensitivity and different CYP51 alleles had competitive disadvantages compared with a DMI-sensitive, wild-type CYP51 isolate. In contrast, the isolate with the F129L mutation in the <span class="gene">CYTB</span> gene competed equally well with a QoI-sensitive, wild-type <span class="gene">CYTB</span> isolate under the conditions of this experiment. The CYP51 and <span class="gene">CYTB</span> alleles were stable in all isolates over four disease cycles when cultivated alone.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of a synthetic <span class="trait">rust</span> fungal virus cDNA in yeast.</h2>
			<h3>PMID:26497180  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cooper B(1), Campbell KB(2), Garrett WM(3).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mycoviruses are viruses that infect fungi. Recently, mycovirus-like RNAs were sequenced from the fungus Phakopsora pachyrhizi, the causal agent of soybean <span class="trait">rust.</span> One of the RNAs appeared to represent a novel mycovirus and was designated Phakopsora pachyrhizi virus 2383 (PpV2383). The genome of PpV2383 resembles Saccharomyces cerevisiae virus L-A, a double-stranded (ds) RNA mycovirus of yeast. PpV2383 encodes two major, overlapping open reading frames with similarity to gag (capsid protein) and <span class="gene">pol</span> (RNA-dependent RNA polymerase), and a -1 ribosomal frameshift is necessary for the translation of a <span class="gene">gag-pol</span> fusion protein. Phylogenetic analysis of <span class="gene">pol</span> relates PpV2383 to members of the family Totiviridae, including L-A. Because the obligate biotrophic nature of P. pachyrhizi makes it genetically intractable for in vivo analysis and because PpV2383 is similar to L-A, we synthesized a DNA clone of PpV2383 and tested its infectivity in yeast cells. PpV2383 RNA was successfully expressed in yeast, and mass spectrometry confirmed the translation of gag and <span class="gene">gag-pol</span> fusion proteins. There was, however, no production of PpV2383 dsRNA, the evidence of viral replication. Neither the presence of endogenous L-A nor the substitution of the 5' and 3' untranslated regions with those from L-A was sufficient to rescue replication of PpV2383. Nevertheless, the proof of transcription and translation from the clone in vivo are steps toward confirming that PpV2383 is a mycovirus. Further development of a surrogate biological system for the study of <span class="trait">rust</span> mycoviruses is necessary, and such research may facilitate biological control of <span class="trait">rust</span> diseases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Reference Genes in the Pathosystem <italic> Phakopsora pachyrhizi </italic> / Soybean Suitable for Normalization in Transcript Profiling</h2>
			<h3>PMID:26404265</h3>
				

				
					<h3>PMCID:4613351</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hirschburger </surname>  <given-names> Daniela </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-ijms-16-23057"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> M&#x000fc;ller </surname>  <given-names> Manuel </given-names>  </name>  <xref ref-type="author-notes" rid="fn1-ijms-16-23057"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Voegele </surname>  <given-names> Ralf T. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Link </surname>  <given-names> Tobias </given-names>  </name>  <xref rid="c1-ijms-16-23057" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phakopsora pachyrhizi is a devastating pathogen on soybean, endangering soybean production worldwide. Use of Host Induced Gene Silencing (HIGS) and the study of effector proteins could provide novel strategies for pathogen control. For both approaches quantification of transcript abundance by RT-qPCR is essential. Suitable stable reference genes for normalization are indispensable to obtain accurate RT-qPCR results. According to the Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines and using algorithms geNorm and NormFinder we tested candidate reference genes from P. pachyrhizi and Glycine max for their suitability in normalization of transcript levels throughout the infection process. For P. pachyrhizi we recommend a combination of <span class="gene">CytB</span> and PDK or <span class="gene">GAPDH</span> for in planta experiments. Gene expression during in vitro stages and over the whole infection process was found to be highly unstable. Here, RPS14 and UbcE2 are ranked best by geNorm and NormFinder. Alternatively <span class="gene">CytB</span> that has the smallest Cq range (Cq: quantification cycle) could be used. We recommend specification of gene expression relative to the germ tube stage rather than to the resting urediospore stage. For studies omitting the resting spore and the appressorium stages a combination of Elf3 and RPS9, or PKD and <span class="gene">GAPDH</span> should be used. For normalization of soybean genes during <span class="trait">rust</span> infection Ukn2 and cons7 are recommended.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Phakopsora pachyrhizi </italic> is a devastating pathogen on soybean, endangering soybean production worldwide. Use of Host Induced Gene Silencing (HIGS) and the study of effector proteins could provide novel strategies for pathogen control. For both approaches quantification of transcript abundance by RT-qPCR is essential. Suitable stable reference genes for normalization are indispensable to obtain accurate RT-qPCR results. According to the <italic> M </italic> inimum <italic> I </italic> nformation for Publication of <italic> Q </italic> uantitative Real-Time PCR <italic> E </italic> xperiments (MIQE) guidelines and using algorithms geNorm and NormFinder we tested candidate reference genes from <italic> P. pachyrhizi </italic> and <italic> Glycine max </italic> for their suitability in normalization of transcript levels throughout the infection process. For <italic> P. pachyrhizi </italic> we recommend a combination of <italic> <span class="gene">CytB</span> </italic> and <italic> PDK </italic> or <italic> <span class="gene">GAPDH</span> </italic> for <italic> in planta </italic> experiments. Gene expression during <italic> in vitro </italic> stages and over the whole infection process was found to be highly unstable. Here, <italic> RPS14 </italic> and <italic> UbcE2 </italic> are ranked best by geNorm and NormFinder. Alternatively <italic> <span class="gene">CytB</span> </italic> that has the smallest <italic> C </italic> q range ( <italic> C </italic> q: quantification cycle) could be used. We recommend specification of gene expression relative to the germ tube stage rather than to the resting urediospore stage. For studies omitting the resting spore and the appressorium stages a combination of <italic> Elf3 </italic> and <italic> RPS9 </italic> , or <italic> PKD </italic> and <italic> <span class="gene">GAPDH</span> </italic> should be used. For normalization of soybean genes during <span class="trait">rust</span> infection <italic> Ukn2 </italic> and <italic> cons7 </italic> are recommended.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results show that there are enormous changes in gene expression even among housekeeping genes during early development of soybean <span class="trait">rust.</span> There is only little gene expression in resting spores and there may be problems with viability of <italic> in vitro </italic> structures in the appressorium stage. We therefore recommend to specify gene expression data relative to the germ tube stage rather than the resting urediospore stage. We also suggest that gene expression changes in the appressorium should only be considered relevant when they are a thousand fold or higher. We could identify reference genes that are suitable for expression profiling during the infection process <italic> in planta </italic> and combined with the germ tube stage: <italic> <span class="gene">CytB</span> </italic> / <italic> PDK </italic> or <italic> <span class="gene">CytB</span> </italic> / <italic> <span class="gene">GAPDH</span> </italic> and <italic> Elf3 </italic> / <italic> RPS9 </italic> or <italic> PKD </italic> / <italic> <span class="gene">GAPDH</span> </italic> are suitable reference gene combinations.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of four soybean bZIP genes during Phakopsora pachyrhizi infection.</h2>
			<h3>PMID:26013145  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Alves MS(1), Soares ZG(1), Vidigal PM(2), Barros EG(3), Poddanosqui AM(4), Aoyagi LN(4), Abdelnoor RV(4), Marcelino-Guimarães FC(4), Fietto LG(5).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Asian soybean <span class="trait">rust</span> (ASR), caused by the obligate biotrophic fungus Phakopsora pachyrhizi, is one of most important diseases in the soybean (Glycine max (L.) Merr.) agribusiness. The identification and characterization of genes related to plant defense responses to fungal infection are essential to develop ASR-resistant plants. In this work, we describe four soybean genes, <span class="gene">GmbZIP62,</span> <span class="gene">GmbZIP105,</span> <span class="gene">GmbZIPE1,</span> and <span class="gene">GmbZIPE2,</span> which encode transcription factors containing a basic leucine zipper (bZIP) domain from two divergent classes, and that are responsive to P. pachyrhizi infection. Molecular phylogenetic analyses demonstrated that these genes encode proteins similar to bZIP factors responsive to pathogens. Yeast transactivation assays showed that only <span class="gene">GmbZIP62</span> has strong transactivation activity in yeast. In addition, three of the bZIP transcription factors analyzed were also differentially expressed by plant defense hormones, and all were differentially expressed by fungal attack, indicating that these proteins might participate in response to ASR infection. The results suggested that these bZIP proteins are part of the plant defense response to P. pachyrhizi infection, by regulating the gene expression related to ASR infection responses. These bZIP genes are potential targets to obtain new soybean genotypes resistant to ASR.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Intersubgeneric hybridization between Glycine max and G. tomentella: production of F₁, amphidiploid, BC₁, BC₂, BC₃, and fertile soybean plants.</h2>
			<h3>PMID:25835560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Singh RJ(1), Nelson RL.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: This paper describes methods for unlocking genetic treasure from wild perennial Glycine species of Australia for soybean improvement. The genetic resources of the ca. 26 species of the genus Glycine subgenus Glycine have not been exploited to broaden the genetic base of soybean (Glycine max; 2n = 40). The objectives of this study were to develop methods for producing F1, amphidiploid, BC1, BC2, BC3, and fertile soybean plants from crosses of soybean and the genus Glycine subgenus Glycine species, in order to utilize the subgenus Glycine germplasm in soybean breeding. Soybean cultivars were hybridized with six accessions of 78-chromosome G. tomentella as well as one accession each of 40-chromosome G. tomentella, G. argyrea and G. latifolia. They were chosen because they exhibit <span class="trait">resistance</span> to soybean <span class="trait">rust.</span> We were successful in producing fertile soybean from soybean cv. 'Dwight' and 78-chromosome G. tomentella accession PI 441001, while other hybrids were discontinued either at F1 or amphidiploid stage. The F1 seeds aborted prior to reaching maturity, so developing seeds from 19 to 21 day old pods were cultured aseptically in various media formulations. Seed maturation and multiple embryo generation media were developed. F1 plants with shoots and roots (2n = 59) were transplanted to pots in greenhouse. Amphidiploid (2n = 118) plants were backcrossed to 'Dwight'. <span class="gene">BC1</span> (2n = 79) plants were propagated through in vitro and 43 mature BC2F1 seeds were harvested. Fifteen surviving BC2F1 plants were morphologically distinct, sterile, and had chromosome numbers ranging 2n = 56-59. Chromosome numbers of the BC3F1 plants ranged 2n = 40-49. Derived fertile soybeans were first planted in the field in 2008 and are being evaluated for <span class="trait">yield,</span> <span class="trait">resistance</span> to pathogens and pests and tolerance to salt through material transfer agreement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis and molecular mapping of <span class="trait">resistance</span> gene to Phakopsora pachyrhizi in soybean germplasm SX6907.</h2>
			<h3>PMID:25673142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen H(1), Zhao S, Yang Z, Sha A, Wan Q, Zhang C, Chen L, Yuan S, Qiu D, Chen S, Shan Z, Zhou XA.
				<br/> 
				年份：2015 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: In this study, Rpp6907, a novel <span class="trait">resistance</span> gene/allele to Phakopsora pachyrhizi in soybean, was mapped in a 111.9-kb region, including three NBS-LRR type predicted genes, on chromosome 18. Soybean <span class="trait">rust</span> caused by Phakopsora pachyrhizi Sydow has been reported in numerous soybean-growing regions worldwide. The development of rust-resistant varieties is the most economical and environmentally safe method to control the disease. The Chinese soybean germplasm SX6907 is resistant to P. pachyrhizi and exhibits immune reaction compared with the known Rpp genes. These characteristics suggest that SX6907 may carry at least one novel Rpp gene/allele. Three F2 populations from the crosses of SX6907 (resistant) and Tianlong 1, Zhongdou40, and Pudou11 (susceptible) were used to map the Rpp gene. Three <span class="trait">resistance</span> responses (immune, red-brown, and tan-colored lesion) were observed from the F2 individuals. The segregation follows a ratio of 1(resistance):2(heterozygous):1(susceptible), indicating that the <span class="trait">resistance</span> in SX6907 is controlled by a single incomplete dominant gene (designated as Rpp6907). Results showed that Rpp6907 was mapped on soybean chromosome 18 (molecular linkage group G, MLG G) flanked by simple sequence repeat (SSR) markers SSR24 and SSR40 at a distance of 111.9 kb. Among the ten genes marked within this 111.9-kb region between the two markers, three genes <span class="gene">(Glyma18g51930,</span> <span class="gene">Glyma18g51950,</span> and <span class="gene">Glyma18g51960)</span> are nucleotide-binding site and leucine-rich repeat-type genes. These genes may be involved in recognizing the presence of pathogens and ultimately conferring <span class="trait">resistance.</span> Based on <span class="trait">resistance</span> spectrum analysis and mapping results, we inferred that Rpp6907 is a novel gene different from Rpp1 in PI 200492, PI 561356, PI 587880A, PI 587886, and PI 594538A, or a new Rpp1-b allele.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Potential toxic effect of trifloxystrobin on cellular microstructure, mRNA expression and antioxidant enzymes in Chlorella vulgaris.</h2>
			<h3>PMID:24762415  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shen YF(1), Liu L(1), Gong YX(2), Zhu B(1), Liu GL(3), Wang GX(4).
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This study investigated the effects of trifloxystrobin that one strobilurin used widely in the world as an effective fungicidal agent to control Asian soybean <span class="trait">rust</span> on aquatic unicellular algae Chlorella vulgaris. We determined the potential toxic effect of trifloxystrobin on C. vulgaris, and found median inhibition concentration (IC(50)) value 255.58 (95% confidence interval, 207.81-330.29)μgL(-1). In addition, the algal cells were obviously depressed or shrunk at different concentrations by electron microscopy. In the study, a real-time polymerase chain reaction (PCR) assay showed changes in transcript abundances of three photosynthetic genes, psaB, psbC, and rbcL, and one energy gene, ATPs. The results showed that trifloxystrobin reduced the transcript abundances of the three genes and enhanced expression of ATPs after 48 and 96 h. The lowest abundances of psaB, <span class="gene">psbC</span> and <span class="gene">rbcL</span> transcripts in response to trifloxystrobin exposure were 58%, 79% and 60% of those of the control, respectively. For the potential toxic influences, trifloxystrobin could decrease the soluble protein and total antioxidant contents (T-AOC), and increase superoxide dismutase (SOD) and peroxidase (POD) activity with a gradual concentration-response relationship. Overall, the present study demonstrated that trifloxystrobin could affect the activities of antioxidant enzymes, disrupts photosynthesis in C. vulgaris, and damage cellular structure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the soybean <italic> HyPRP </italic> family and specific gene response to Asian soybean <span class="trait">rust</span> disease</h2>
			<h3>PMID:23885204</h3>
				

				
					<h3>PMCID:3715288</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Neto </surname>  <given-names> Lauro B&#x000fc;cker </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Rafael Rodrigues </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wiebke-Strohm </surname>  <given-names> Beatriz </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bencke </surname>  <given-names> Marta </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weber </surname>  <given-names> Ricardo Lu&#x000ed;s Mayer </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cabreira </surname>  <given-names> Caroline </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo Vilela </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-214">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino </surname>  <given-names> Francismar Correa </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-214">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zanettini </surname>  <given-names> Maria Helena Bodanese </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Passaglia </surname>  <given-names> Luciane Maria Pereira </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-214">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="c1-gmb-36-214"/>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 12 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merril], one of the most important crop species in the world, is very susceptible to abiotic and biotic stress. Soybean plants have developed a variety of molecular mechanisms that help them survive stressful conditions. Hybrid proline-rich proteins (HyPRPs) constitute a family of cell-wall proteins with a variable N-terminal domain and conserved C-terminal domain that is phylogenetically related to non-specific lipid transfer proteins. Members of the HyPRP family are involved in basic cellular processes and their expression and activity are modulated by environmental factors. In this study, microarray analysis and real time RT-qPCR were used to identify putative HyPRP genes in the soybean genome and to assess their expression in different plant tissues. Some of the genes were also analyzed by time-course real time RT-qPCR in response to infection by Phakopsora pachyrhizi, the causal agent of Asian soybean <span class="trait">rust</span> disease. Our findings indicate that the time of induction of a defense pathway is crucial in triggering the soybean <span class="trait">resistance</span> response to P. pachyrhizi. This is the first study to identify the soybean HyPRP group B family and to analyze disease-responsive <span class="gene">GmHyPRP</span> during infection by P. pachyrhizi.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The soybean plant reaction to ASR was evaluated by inoculating a field population of <italic> P. pachyrhizi </italic> spores initially collected from Brazilian soybean fields and maintained on a susceptible cultivar under greenhouse conditions until use. The experiment was done at Embrapa Soja (Londrina, PR, Brazil). Briefly, soybean plants were grown in a pot-based system and maintained in a greenhouse at 28 &#x000b1;1 &#x000b0;C on a 16/8 h light/dark cycle at a light intensity of 22.5 &#x003bc;Em <sup> &#x02212;2 </sup> /s. The Embrapa-48 genotype was used as susceptible host as it develops a tan lesion after infection by ASR ( <xref rid="b49-gmb-36-214" ref-type="bibr"> van de Mortel <italic> et al. </italic> , 2007 </xref> ), and the PI561356 genotype was used as a resistant host in which the <span class="trait">resistance</span> to soybean <span class="trait">rust</span> is mapped on linkage group G (Abdelnoor R.V., personal communication). Uredospores were harvested from infected leaves with sporulating uredia and diluted in distilled water with 0.05% Tween-20 to a final concentration of 3 &#x000d7; 10 <sup> 5 </sup> spores/mL. The spore suspension was sprayed onto three plants per pot at the <span class="gene">V2</span> to <span class="gene">V3</span> stage of growth. The <span class="gene">V2</span> stage consists of a fully developed trifoliolate leaf at a node above the unifoliolate nodes and <span class="gene">V3</span> stage is characterized by three nodes on the main stem, with fully developed leaves beginning with the unifoliolate nodes ( <xref ref-type="bibr" rid="b12-gmb-36-214"> Fehr and Caviness, 1977 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						This work is the first to identify the soybean HyPRP group B family and to analyze disease-responsive <span class="gene">GmHyPRP</span> during infection by <italic> P. pachyrhizi </italic> . Our results indicate that the time of induction of a defense pathway is crucial to triggering the soybean <span class="trait">resistance</span> response to <italic> P. pachyrhizi </italic> , the causal agent of ASR. Future studies will improve our understanding of the relationship between the proteins described here and their role(s) in adaptation to biotic stress. Such information will provide a valuable genetic resource for engineering tolerance in soybean crops.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:UDP-glucosyltransferase UGT84A2/BRT1 is required for Arabidopsis nonhost <span class="trait">resistance</span> to the Asian soybean <span class="trait">rust</span> pathogen Phakopsora pachyrhizi.</h2>
			<h3>PMID:23356583  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Langenbach C(1), Campe R, Schaffrath U, Goellner K, Conrath U.
				<br/> 
				年份：2013 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nonhost <span class="trait">resistance</span> (NHR) of plants to fungal pathogens comprises different defense layers. Epidermal penetration <span class="trait">resistance</span> of Arabidopsis to Phakopsora pachyrhizi requires functional PEN1, PEN2 and PEN3 genes, whereas post-invasion <span class="trait">resistance</span> in the mesophyll depends on the combined functionality of PEN2, <span class="gene">PAD4</span> and SAG101. Other genetic components of Arabidopsis post-invasion mesophyll <span class="trait">resistance</span> remain elusive. We performed comparative transcriptional profiling of wild-type, pen2 and pen2 pad4 sag101 mutants after inoculation with P. pachyrhizi to identify a novel trait for mesophyll NHR. Quantitative reverse transcription-polymerase chain reaction (RT-qPCR) analysis and microscopic analysis confirmed the essential role of the candidate gene in mesophyll NHR. UDP-glucosyltransferase UGT84A2/bright trichomes 1 (BRT1) is a novel component of Arabidopsis mesophyll NHR to P. pachyrhizi. BRT1 is a putative cytoplasmic enzyme in phenylpropanoid metabolism. BRT1 is specifically induced in pen2 with post-invasion <span class="trait">resistance</span> to P. pachyrhizi. Silencing or mutation of BRT1 increased haustoria formation in pen2 mesophyll. Yet, the brt1 mutation did not affect NHR to P. pachyrhizi in wild-type plants. We assign a novel function to BRT1, which is important for post-invasion NHR of Arabidopsis to P. pachyrhizi. BRT1 might serve to confer durable <span class="trait">resistance</span> against P. pachyrhizi to soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a soybean nuclear localized type-III <span class="gene">DnaJ</span> domain-containing HSP40 reveals its roles in cell death and disease <span class="trait">resistance.</span></h2>
			<h3>PMID:23289813  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu JZ(1), Whitham SA.
				<br/> 
				年份：2013 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heat-shock proteins such as <span class="gene">HSP70</span> and <span class="gene">HSP90</span> are important molecular chaperones that play critical roles in biotic and abiotic stress responses; however, the involvement of their co-chaperones in stress biology remains largely uninvestigated. In a screen for candidate genes stimulating cell death in Glycine max (soybean), we transiently overexpressed full-length cDNAs of soybean genes that are highly induced during soybean <span class="trait">rust</span> infection in Nicotiana benthamiana leaves. Overexpression of a type-III <span class="gene">DnaJ</span> domain-containing HSP40 <span class="gene">(GmHSP40.1),</span> a co-chaperone of HSP70, caused hypersensitive response (HR)-like cell death. The HR-like cell death was dependent on MAPKKKα and WIPK, because silencing each of these genes suppressed the HR. Consistent with the presence of a nuclear localization signal (NLS) motif within the <span class="gene">GmHSP40.1</span> coding sequence, GFP-GmHSP40.1 was exclusively present in nuclear bodies or speckles. Nuclear localization of <span class="gene">GmHSP40.1</span> was necessary for its function, because deletion of the NLS or addition of a nuclear export signal abolished its HR-inducing ability. <span class="gene">GmHSP40.1</span> co-localized with HcRed-SE, a protein involved in pri-miRNA processing, which has been shown to be co-localized with SR33-YFP, a protein involved in pre-mRNA splicing, suggesting a possible role for <span class="gene">GmHSP40.1</span> in mRNA splicing or miRNA processing, and a link between these processes and cell death. Silencing <span class="gene">GmHSP40.1</span> enhanced the susceptibility of soybean plants to Soybean mosaic virus, confirming its positive role in pathogen defense. Together, the results demonstrate a critical role of a nuclear-localized <span class="gene">DnaJ</span> domain-containing <span class="gene">GmHSP40.1</span> in cell death and disease <span class="trait">resistance</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overall picture of expressed Heat Shock Factors in <italic> Glycine max </italic> , <italic> Lotus japonicus </italic> and <italic> Medicago truncatula </italic></h2>
			<h3>PMID:22802710</h3>
				

				
					<h3>PMCID:3392877</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Soares-Cavalcanti </surname>  <given-names> Nina M. </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-247">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Belarmino </surname>  <given-names> Lu&#x000ed;s C. </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-247">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kido </surname>  <given-names> Ederson A. </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-247">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pandolfi </surname>  <given-names> Valesca </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-247">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar C. </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-247">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana A. </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-247">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pereira </surname>  <given-names> Gon&#x000e7;alo A.G. </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-247">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Benko-Iseppon </surname>  <given-names> Ana M. </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-247">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="c1-gmb-35-1-sup-247"/>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Heat shock (HS) leads to the activation of molecular mechanisms, known as HS-response, that prevent damage and enhance survival under stress. Plants have a flexible and specialized network of Heat Shock Factors (HSFs), which are transcription factors that induce the expression of heat shock proteins. The present work aimed to identify and characterize the Glycine max <span class="gene">HSF</span> repertory in the Soybean Genome Project (GENOSOJA platform), comparing them with other legumes (Medicago truncatula and Lotus japonicus) in view of current knowledge of Arabidopsis thaliana. The <span class="gene">HSF</span> characterization in leguminous plants led to the identification of 25, 19 and 21 candidate ESTs in soybean, Lotus and Medicago, respectively. A search in the SuperSAGE libraries revealed 68 tags distributed in seven <span class="gene">HSF</span> gene types. From the total number of obtained tags, more than 70% were related to root tissues (water deficit stress libraries vs. controls), indicating their role in abiotic stress responses, since the root is the first tissue to sense and respond to abiotic stress. Moreover, as heat stress is related to the pressure of dryness, a higher <span class="gene">HSF</span> expression was expected at the water deficit libraries. On the other hand, expressive <span class="gene">HSF</span> candidates were obtained from the library inoculated with Asian Soybean <span class="trait">Rust,</span> inferring crosstalk among genes associated with abiotic and biotic stresses. Evolutionary relationships among sequences were consistent with different <span class="gene">HSF</span> classes and subclasses. Expression profiling indicated that regulation of specific genes is associated with the stage of plant development and also with stimuli from other abiotic stresses pointing to the maintenance of <span class="gene">HSF</span> expression at a basal level in soybean, favoring its activation under heat-stress conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Heat shock (HS) leads to the activation of molecular mechanisms, known as HS-response, that prevent damage and enhance survival under stress. Plants have a flexible and specialized network of Heat Shock Factors (HSFs), which are transcription factors that induce the expression of heat shock proteins. The present work aimed to identify and characterize the <italic> Glycine max </italic> <span class="gene">HSF</span> repertory in the Soybean Genome Project (GENOSOJA platform), comparing them with other legumes ( <italic> Medicago truncatula </italic> and <italic> Lotus japonicus </italic> ) in view of current knowledge of <italic> Arabidopsis thaliana </italic> . The <span class="gene">HSF</span> characterization in leguminous plants led to the identification of 25, 19 and 21 candidate ESTs in soybean, <italic> Lotus </italic> and <italic> Medicago </italic> , respectively. A search in the SuperSAGE libraries revealed 68 tags distributed in seven <span class="gene">HSF</span> gene types. From the total number of obtained tags, more than 70% were related to root tissues (water deficit stress libraries <italic> vs. </italic> controls), indicating their role in abiotic stress responses, since the root is the first tissue to sense and respond to abiotic stress. Moreover, as heat stress is related to the pressure of dryness, a higher <span class="gene">HSF</span> expression was expected at the water deficit libraries. On the other hand, expressive <span class="gene">HSF</span> candidates were obtained from the library inoculated with Asian Soybean <span class="trait">Rust,</span> inferring crosstalk among genes associated with abiotic and biotic stresses. Evolutionary relationships among sequences were consistent with different <span class="gene">HSF</span> classes and subclasses. Expression profiling indicated that regulation of specific genes is associated with the stage of plant development and also with stimuli from other abiotic stresses pointing to the maintenance of <span class="gene">HSF</span> expression at a basal level in soybean, favoring its activation under heat-stress conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Heat stress is one of the major factors limiting the productivity and adaptation of crops, especially when temperature extremes coincide with critical stages of plant development. The major developmental performance of plants occurs at a temperature regime between 10&#x000b0; and 40 &#x000b0;C. Temperatures below or above this range generally cause temperature-induced stresses ( <xref ref-type="bibr" rid="b35-gmb-35-1-sup-247"> Treshow, 1970 </xref> ; <xref ref-type="bibr" rid="b12-gmb-35-1-sup-247"> Hsu <italic> et al. </italic> , 2010 </xref> ). In the case of heat stress, both the rate of temperature change and the duration and degree of high temperatures contribute to the intensity of heat stress. The degree of inherent adaptedness to heat stress of a plant is an important determinant of its ability to survive a stress period ( <xref ref-type="bibr" rid="b5-gmb-35-1-sup-247"> Efeoglu, 2009 </xref> ). However, the expression of <span class="gene">HSF</span> and HSP genes has been also observed under other abiotic and biotic stresses, as cited by <xref ref-type="bibr" rid="b26-gmb-35-1-sup-247"> Pirkkala <italic> et al. </italic> (2001) </xref> . In response to various inducers such as elevated temperatures, salinity, <span class="trait">drought,</span> oxidants, heavy metals, bacterial and viral infections, most HSFs acquire DNA binding activity to the heat shock element (HSE), thereby mediating transcription of the heat shock factor genes, which results in accumulation of heat shock proteins (HSPs). Among important transcription factors, heat shock factors (HSFs) are essential for the transcription of many HSP coding genes that are active in response to sublethal heat stress leading to increased tolerance against a subsequent, otherwise lethal, heat shock ( <xref ref-type="bibr" rid="b35-gmb-35-1-sup-247"> Treshow, 1970 </xref> ; <xref ref-type="bibr" rid="b12-gmb-35-1-sup-247"> Hsu <italic> et al. </italic> , 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="bibr" rid="b13-gmb-35-1-sup-247"> Hu <italic> et al. </italic> (2009) </xref> identified rice <italic> <span class="gene">HSF</span> </italic> and <italic> HSP </italic> genes and analyzed their expression profiles under different abiotic stresses. A whole-genome microarray analysis was carried out to investigate expression changes of rice <italic> HSFs </italic> and <italic> HSPs </italic> genes in response to heat stress. By comparing their experimental data with other expression data under salt, cold, and <span class="trait">drought</span> conditions, <xref ref-type="bibr" rid="b13-gmb-35-1-sup-247"> Hu <italic> et al. </italic> (2009) </xref> found that the rice <span class="gene">HSF</span> and HSP families responded to different stresses in an overlapping relationship. The analysis also indicated that some <italic> <span class="gene">HSF</span> </italic> and <italic> HSP </italic> genes exhibited specific expression patterns in response to distinct stress types.
					</p>
				
					<p>&nbsp;&nbsp;
						The GENOSOJA databank is comprised of six SuperSAGE libraries and allowed the generation of three comparisons, including two from root tissues subjected to water deficit stress and one inoculated with Asian Soybean <span class="trait">Rust</span> fungus ( <italic> Phakopsora pachyrhizi </italic> ). For the water deficit libraries, seeds of a <span class="trait">drought</span> tolerant cultivar (Embrapa 48) and a <span class="trait">drought</span> susceptible cultivar (BR 16) were germinated on filter paper for four days in a growth chamber at 25 &#x000b1; 1 &#x000b0;C and 100% relative humidity (RH). Seedlings were placed in 36 L boxes containing 50% Hoagland&#x02019;s solution ( <xref ref-type="bibr" rid="b11-gmb-35-1-sup-247"> Hoagland and Arnon, 1950 </xref> ) continuously aerated and replaced on a weekly basis. These boxes were then transferred to a greenhouse under natural photoperiod of approximately 12/12 h light/dark cycle, temperature of 30 &#x000b1; 5 &#x000b0;C and 60 &#x000b1; 10% RH. The plants were allowed to grow until the V4 stage ( <xref ref-type="bibr" rid="b7-gmb-35-1-sup-247"> Fehr <italic> et al. </italic> , 1971 </xref> ). The experimental plan was a randomized complete block 2x7 factorial design with three repetitions. The treatments included two cultivars (BR 16 and Embrapa 48) and seven water deficit periods (0, 25, 50, 75, 100, 125 and 150 min). Water stress was applied by removing the plants from the hydroponic solution and leaving them in boxes without nutrient solution for up to 150 min under ambient-air exposure. For each stress exposure time, roots from 10 plants were collected, pooled and frozen in liquid nitrogen before storage at &#x02212;80 &#x000b0;C. The above mentioned exposure times were bulked together generating a library from <span class="trait">drought</span> tolerant genotype Embrapa 48 after stress as compared with the negative control (T0); the same procedure was also applied to the <span class="trait">drought</span> sensitive genotype (BR16 cultivar). The comparison regarding Asian Soybean <span class="trait">Rust</span> infection was generated from leaves of the resistant cultivar PI561356 collected at different times (12, 24 and 48 h) after spraying with a <italic> P. pachyrhizi </italic> spore suspension (6 x 10 <sup> 5 </sup> uredospores.mL <sup> &#x02212;1 </sup> ). The urediniospores were collected from <italic> Phakopsora pachyrhizi </italic> infected soybean fields in the state of Mato Grosso, Brazil, and maintained for over 10 generations on the susceptible cv. BRSMS-Bacuri. The suspension of spores was sprayed onto three plants per pot at the <span class="gene">V2</span> to <span class="gene">V3</span> growth stage ( <xref ref-type="bibr" rid="b8-gmb-35-1-sup-247"> Fehr and Caviness, 1977 </xref> ). The same solution without the spores was used for the false inoculations (Mock). The different times were bulked together to form a single resistant library, which was compared with the false inoculated negative control collected at the same time points.
					</p>
				
					<p>&nbsp;&nbsp;
						Regarding SuperSAGE, 68 different tags could be identified, including 26 tags unique to water deficit experiments with the tolerant comparison (water deficit stressed Embrapa 48 <italic> vs. </italic> control), 28 tags unique to water deficit experiments with the susceptible comparison (water deficit stressed BR16 <italic> vs. </italic> control) and 14 regarding Asian Soybean <span class="trait">Rust</span> (PI561356 inoculated <italic> vs. </italic> control) ( <xref ref-type="table" rid="t2-gmb-35-1-sup-247"> Table 2 </xref> ; <xref ref-type="fig" rid="f1-gmb-35-1-sup-247"> Figure 1 </xref> ). No common tags were identified. It is important to note that among 25 <span class="gene">HSF</span> EST clusters, 18 had no representative in the tags database, while five clusters were represented in all libraries. The sequence <span class="gene">Gmax_HSFB1_SJ09-E1-R06-064-B09-UC.F</span> was not identified in &#x02018;Embrapa 48&#x02019; and &#x02018;PI561356&#x02019; libraries, and <span class="gene">Gmax_HSFB3_Contig20961</span> was present in the water deficit stressed libraries only. When looked at from a different point of view, from the 14 <span class="gene">HSF</span> types compared, only six <span class="gene">HSF</span> types (HSFB1, HSFA1E, HSFB2A, HSFB3, HSFA8 and HSFA4A) were identified ( <xref ref-type="table" rid="t2-gmb-35-1-sup-247"> Table 2 </xref> ; <xref ref-type="fig" rid="f1-gmb-35-1-sup-247"> Figure 1 </xref> ), indicating their induction during the stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Despite the small number of identified sequences in the Asian <span class="trait">rust</span> &#x02018;PI561356&#x02019; stress analysis, when compared to water deficit experiments, the presence of <span class="gene">HSF</span> representatives indicates the involvement of HS-response also during biotic stresses. The stress condition by itself can activate non-specific stress-responsive-pathways, due to the debility caused to plants by biotic stressful conditions, which can activate a crosstalk among different stress related pathways, as observed in other plants ( <xref ref-type="bibr" rid="b9-gmb-35-1-sup-247"> Glombitza <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="b14-gmb-35-1-sup-247"> Kido <italic> et al. </italic> , 2011 </xref> ). Moreover, it is important to consider the tissue from which the library was generated, since leaves are among the first organs to present stress symptoms (especially to abiotic ones). These are necessary for the maintenance of photosynthesis and evapotranspiration processes to ensure plant survival. Moreover, the <span class="gene">Gmax_HSFB3_Contig20961</span> gene seems to be expressed specifically under abiotic stress, such as water deficit.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a new soybean <span class="trait">rust</span> <span class="trait">resistance</span> gene in PI 567102B.</h2>
			<h3>PMID:22374138  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li S(1), Smith JR, Ray JD, Frederick RD.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">rust</span> (SBR) caused by Phakopsora pachyrhizi Syd. and P. Syd. is one of the most economically important diseases of soybean (Glycine max (L.) Merr.). Durable <span class="trait">resistance</span> to P. pachyrhizi is the most effective long-term strategy to control SBR. The objective of this study was to investigate the genetics of <span class="trait">resistance</span> to P. pachyrhizi in soybean accession PI 567102B. This accession was previously identified as resistant to SBR in Paraguay and to P. pachyrhizi isolates from seven states in the USA (Alabama, Florida, Georgia, Louisiana, Mississippi, South Carolina, and Texas). Analysis of two independent populations, one in which F(2) phenotypes were inferred from F(2)-derived F(3) (F(2:3)) families and the other in which F(2) plants had phenotypes measured directly, showed that the <span class="trait">resistance</span> in PI 567102B was controlled by a single dominant gene. Two different isolates (MS06-1 and LA04-1) at different locations (Stoneville, <span class="gene">MS</span> and Ft. Detrick, MD) were used to independently assay the two populations. Linkage analysis of both populations indicated that the <span class="trait">resistance</span> locus was located on chromosome 18 (formerly linkage group G), but at a different location than either Rpp1 or Rpp4, which were previously mapped to this linkage group. Therefore, the SBR <span class="trait">resistance</span> in PI 567102B appeared to be conditioned by a previously unreported locus, with an underlying single dominant gene inferred. We propose this gene to be designated Rpp6. Incorporating Rpp6 into improved soybean cultivars may have wide benefits as PI 567102B has been shown to provide <span class="trait">resistance</span> to P. pachyrhizi isolates from Paraguay and the US.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Rust secreted protein Ps87 is conserved in diverse fungal pathogens and contains a RXLR-like motif sufficient for translocation into plant cells.</h2>
			<h3>PMID:22076138  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gu B(1), Kale SD, Wang Q, Wang D, Pan Q, Cao H, Meng Y, Kang Z, Tyler BM, Shan W.
				<br/> 
				年份：2011
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Effector proteins of biotrophic plant pathogenic fungi and oomycetes are delivered into host cells and play important roles in both disease development and disease <span class="trait">resistance</span> response. How obligate fungal pathogen effectors enter host cells is poorly understood. The Ps87 gene of Puccinia striiformis encodes a protein that is conserved in diverse fungal pathogens. Ps87 homologs from a clade containing <span class="trait">rust</span> fungi are predicted to be secreted. The aim of this study is to test whether Ps87 may act as an effector during Puccinia striiformis infection. METHODOLOGY/PRINCIPAL FINDINGS: Yeast signal sequence trap assay showed that the <span class="trait">rust</span> protein Ps87 could be secreted from yeast cells, but a homolog from Magnaporthe oryzae that was not predicted to be secreted, could not. Cell re-entry and protein uptake assays showed that a region of Ps87 containing a conserved RXLR-like motif [K/R]RLTG was confirmed to be capable of delivering oomycete effector <span class="gene">Avr1b</span> into soybean leaf cells and carrying GFP into soybean root cells. Mutations in the Ps87 motif (KRLTG) abolished the protein translocation ability. CONCLUSIONS/SIGNIFICANCE: The results suggest that Ps87 and its secreted homologs could utilize similar protein translocation machinery as those of oomycete and other fungal pathogens. Ps87 did not show direct suppression activity on plant defense responses. These results suggest Ps87 may represent an "emerging effector" that has recently acquired the ability to enter plant cells but has not yet acquired the ability to alter host physiology.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of a second Asian soybean <span class="trait">rust</span> <span class="trait">resistance</span> gene in Hyuuga soybean.</h2>
			<h3>PMID:21244223  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kendrick MD(1), Harris DK, Ha BK, Hyten DL, Cregan PB, Frederick RD, Boerma HR, Pedley KF.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						ABSTRACT Asian soybean <span class="trait">rust</span> (ASR) is an economically significant disease caused by the fungus Phakopsora pachyrhizi. The soybean genes Rpp3 and Rpp?(Hyuuga) confer <span class="trait">resistance</span> to specific isolates of the pathogen. Both genes map to chromosome 6 <span class="gene">(Gm06)</span> (linkage group [LG] C2). We recently identified 12 additional soybean accessions that harbor ASR <span class="trait">resistance</span> mapping to <span class="gene">Gm06,</span> within 5 centimorgans of Rpp3 and Rpp?(Hyuuga). To further characterize genotypes with <span class="trait">resistance</span> on <span class="gene">Gm06,</span> we used a set of eight P. pachyrhizi isolates collected from geographically diverse areas to inoculate plants and evaluate them for differential phenotypic responses. Three isolates elicited different responses from soybean accessions PI 462312 (Ankur) (Rpp3) and PI 506764 (Hyuuga) (Rpp?[Hyuuga]). In all, 11 of the new accessions yielded responses identical to either PI 462312 or Hyuuga and 1 of the new accessions, PI 417089B (Kuro daizu), differed from all others. Additional screening of Hyuuga-derived recombinant inbred lines indicated that Hyuuga carries two <span class="trait">resistance</span> genes, one at the Rpp3 locus on <span class="gene">Gm06</span> and a second, unlinked ASR <span class="trait">resistance</span> gene mapping to <span class="gene">Gm03</span> (LG-N) near Rpp5. These findings reveal a natural case of gene pyramiding for ASR <span class="trait">resistance</span> in Hyuuga and underscore the importance of utilizing multiple isolates of P. pachyrhizi when screening for ASR <span class="trait">resistance.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of the Asian soybean <span class="trait">rust</span> <span class="trait">resistance</span> pathway mediated by Rpp2.</h2>
			<h3>PMID:20977308  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pandey AK(1), Yang C, Zhang C, Graham MA, Horstman HD, Lee Y, Zabotina OA, Hill JH, Pedley KF, Whitham SA.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Asian soybean <span class="trait">rust</span> is an aggressive foliar disease caused by the obligate biotrophic fungus Phakopsora pachyrhizi. On susceptible plants, the pathogen penetrates and colonizes leaf tissue, resulting in the formation of necrotic lesions and the development of numerous uredinia. The soybean Rpp2 gene confers <span class="trait">resistance</span> to specific isolates of P. pachyrhizi. Rpp2-mediated <span class="trait">resistance</span> limits the growth of the pathogen and is characterized by the formation of reddish-brown lesions and few uredinia. Using virus-induced gene silencing, we screened 140 candidate genes to identify those that play a role in Rpp2 <span class="trait">resistance</span> toward P. pachyrhizi. Candidate genes included putative orthologs to known defense-signaling genes, transcription factors, and genes previously found to be upregulated during the Rpp2 <span class="trait">resistance</span> response. We identified 11 genes that compromised Rpp2-mediated <span class="trait">resistance</span> when silenced, including <span class="gene">GmEDS1,</span> <span class="gene">GmNPR1,</span> <span class="gene">GmPAD4,</span> <span class="gene">GmPAL1,</span> five predicted transcription factors, an O-methyl transferase, and a cytochrome <span class="gene">P450</span> monooxygenase. Together, our results provide new insight into the signaling and biochemical pathways required for <span class="trait">resistance</span> against P. pachyrhizi.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic analysis of germinating urediniospores of Phakopsora pachyrhizi, causal agent of Asian soybean <span class="trait">rust.</span></h2>
			<h3>PMID:20821732  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Luster DG(1), McMahon MB, Carter ML, Fortis LL, Nuñez A.
				<br/> 
				年份：2010 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phakopsora pachyrhizi is an obligate pathogen that causes Asian soybean <span class="trait">rust.</span> Asian soybean <span class="trait">rust</span> has an unusually broad host range and infects by direct penetration through the leaf cuticle. In order to understand the early events in the infection process, it is important to identify and characterize proteins in P. pachyrhizi. Germination of the urediniospore is the first stage in the infection process and represents a critical life stage applicable to studies with this obligate pathogen. We have applied a 2-DE and <span class="gene">MS</span> approach to identify 117 proteins from the National Center of Biotechnology Information nonredundant protein database and a custom database of Basidiomycota EST sequences. Proteins with roles in primary metabolism, energy transduction, stress, cellular regulation and signaling were identified in this study. This data set is accessible at http://world-2dpage.expasy.org/repository/database=0018.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of the complete mitochondrial genome sequences of the soybean <span class="trait">rust</span> pathogens phakopsora pachyrhizi and p. meibomiae.</h2>
			<h3>PMID:20648755  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Stone CL(1), Buitrago ML, Boore JL, Frederick RD.
				<br/> 
				年份：2010 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The mitochondrial (mt) genomes of two soybean <span class="trait">rust</span> pathogens, Phakopsora pachyrhizi and P. meibomiae, have been sequenced. The mt genome of P. pachyrhizi is a circular 31 825-bp molecule with a mean GC content of 34.6%, while P. meibomiae possesses a 32 520-bp circular molecule with a mean GC content of 34.9%. Both mt genomes contain the genes encoding ATP synthase subunits 6, 8 and 9 (atp6, <span class="gene">atp8</span> and atp9), cytochrome oxidase subunits I, II and III (cox1, <span class="gene">cox2</span> and cox3), apocytochrome b (cob), reduced nicotinamide adenine dinucleotide ubiquinone oxidoreductase subunits (nad1, nad2, nad3, nad4, nad4L, <span class="gene">nad5</span> and nad6), the large and small mt ribosomal RNA genes, one ORF coding for a ribosomal protein (rps3), and a set of 24 tRNA genes that recognize codons for all amino acids. The order of the protein-coding genes and tRNA is identical in the two Phakopsora species, and all genes are transcribed from the same DNA strand clockwise. Introns were identified in the cox1, <span class="gene">cob</span> and mnl genes of both species, with three of the introns having ORFs with motifs similar to the LAGLIDADG endonucleases of other fungi. Phylogenetic analysis of the 14 shared protein-coding genes agrees with commonly accepted fungal taxonomy.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SNP assay to detect the &#x02018;Hyuuga&#x02019; red-brown lesion <span class="trait">resistance</span> gene for Asian soybean <span class="trait">rust</span></h2>
			<h3>PMID:20532750</h3>
				

				
					<h3>PMCID:2938421</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Monteros </surname>  <given-names> Maria J. </given-names>  </name>  <address>  <phone> +1-580-2246810 </phone>  <fax> +1-580-2246802 </fax>  <email> mjmonteros@noble.org </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ha </surname>  <given-names> Bo-Keun </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Phillips </surname>  <given-names> Daniel V. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boerma </surname>  <given-names> H. Roger </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> The Samuel Roberts Noble Foundation, 2510 Sam Noble Parkway, Ardmore, OK 73401 USA  </aff>  <aff id="Aff2">  <label> 2 </label> Center for Applied Genetic Technologies, The University of Georgia, 111 Riverbend Rd, Athens, GA 30602 USA  </aff>  <aff id="Aff3">  <label> 3 </label> Department of Plant Pathology, The University of Georgia, Griffin, GA 30223 USA   </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 5 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Asian soybean <span class="trait">rust</span> (ASR), caused by Phakopsora pachyrhizi Syd., has the potential to become a serious threat to soybean, Glycine max L. Merr., production in the USA. A novel <span class="trait">rust</span> <span class="trait">resistance</span> gene, Rpp?(Hyuuga), from the Japanese soybean cultivar Hyuuga has been identified and mapped to soybean chromosome 6 <span class="gene">(Gm06).</span> Our objectives were to fine-map the Rpp?(Hyuuga) gene and develop a high-throughput single nucleotide polymorphism (SNP) assay to detect this ASR <span class="trait">resistance</span> gene. The integration of recombination events from two different soybean populations and the ASR reaction data indicates that the Rpp?(Hyuuga) locus is located in a region of approximately 371 kb between STS70887 and STS70923 on chromosome <span class="gene">Gm06.</span> A set of 32 ancestral genotypes which is predicted to contain 95% of the alleles present in current elite North American breeding populations and the sources of the previously reported ASR <span class="trait">resistance</span> genes (Rpp1, Rpp2, Rpp3, Rpp4, Rpp5, and rpp5) were genotyped with five SNP markers. We developed a SimpleProbe assay based on melting curve analysis for SNP06-44058 which is tighly linked to the Rpp?(Hyuuga) gene. This SNP assay can differentiate plants/lines that are homozygous/homogeneous or heterozygous/heterogeneous for the resistant and susceptible alleles at the Rpp?(Hyuuga) locus.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Asian soybean <span class="trait">rust</span> (ASR), caused by <italic> Phakopsora pachyrhizi </italic> Syd., has the potential to become a serious threat to soybean, <italic> Glycine max </italic> L. Merr., production in the USA. A novel <span class="trait">rust</span> <span class="trait">resistance</span> gene, <italic> Rpp </italic> ?(Hyuuga), from the Japanese soybean cultivar Hyuuga has been identified and mapped to soybean chromosome 6 ( <italic> <span class="gene">Gm06</span> </italic> ). Our objectives were to fine-map the <italic> Rpp </italic> ?(Hyuuga) gene and develop a high-throughput single nucleotide polymorphism (SNP) assay to detect this ASR <span class="trait">resistance</span> gene. The integration of recombination events from two different soybean populations and the ASR reaction data indicates that the <italic> Rpp </italic> ?(Hyuuga) locus is located in a region of approximately 371&#x000a0;kb between STS70887 and STS70923 on chromosome <italic> <span class="gene">Gm06</span> </italic> . A set of 32 ancestral genotypes which is predicted to contain 95% of the alleles present in current elite North American breeding populations and the sources of the previously reported ASR <span class="trait">resistance</span> genes ( <italic> Rpp1 </italic> , <italic> Rpp2 </italic> , <italic> Rpp3 </italic> , <italic> Rpp4 </italic> , <italic> Rpp5 </italic> , and <italic> rpp5 </italic> ) were genotyped with five SNP markers. We developed a SimpleProbe assay based on melting curve analysis for SNP06-44058 which is tighly linked to the <italic> Rpp </italic> ?(Hyuuga) gene. This SNP assay can differentiate plants/lines that are homozygous/homogeneous or heterozygous/heterogeneous for the resistant and susceptible alleles at the <italic> Rpp </italic> ?(Hyuuga) locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Asian soybean <span class="trait">rust</span> (ASR) caused by <italic> Phakopsora pachyrhizi </italic> Syd. is a widespread disease of soybean [ <italic> Glycine max </italic> (L.) Merr.] and has the potential to cause serious economic losses in soybean production (Sinclair and Hartman <xref ref-type="bibr" rid="CR37"> 1996 </xref> ; Grau et al. <xref ref-type="bibr" rid="CR12"> 2004 </xref> ). The pathogen was recently reported for the first time in North America (Schneider et al. <xref ref-type="bibr" rid="CR33"> 2005 </xref> ). Researchers during the 1970s and 1980s identified four unique dominant ASR <span class="trait">resistance</span> genes. PI 200492 (&#x02018;Komata&#x02019;) has a single dominant gene ( <italic> Rpp1 </italic> ) that confers <span class="trait">resistance</span> to an Australian Asian <span class="trait">rust</span> isolate (McLean and Byth <xref ref-type="bibr" rid="CR27"> 1980 </xref> ; Bromfield and Hartwig <xref ref-type="bibr" rid="CR3"> 1980 </xref> ). PI 230970 (&#x02018;Ankur&#x02019;) has <italic> Rpp2 </italic> (Singh and Thapliyal <xref ref-type="bibr" rid="CR38"> 1977 </xref> ), and PI 462312 has the ASR <span class="trait">resistance</span> gene <italic> Rpp3 </italic> (Hartwig and Bromfield <xref ref-type="bibr" rid="CR17"> 1983 </xref> ). The cultivar Bing Nan (PI 459025) from China possesses a dominant <span class="trait">resistance</span> allele ( <italic> Rpp4 </italic> ) at a locus different from the other three <span class="trait">resistance</span> alleles (Hartwig <xref ref-type="bibr" rid="CR16"> 1986 </xref> ). Recently, a new dominant <span class="trait">resistance</span> allele ( <italic> Rpp5 </italic> ) and a recessive <span class="trait">resistance</span> allele ( <italic> rpp5 </italic> ) were reported by Garcia et al. ( <xref ref-type="bibr" rid="CR9"> 2008 </xref> ). A novel <span class="trait">rust</span> <span class="trait">resistance</span> gene, <italic> Rpp? </italic> (Hyuuga) from the Japanese cultivar Hyuuga, produces a red-brown (RB) or resistant lesion type (LT) with current <span class="trait">rust</span> isolates prevalent in the USA. The <span class="trait">resistance</span> gene was identified and mapped to a 3.5&#x000a0;cM region on soybean chromosome <italic> <span class="gene">Gm06</span> </italic> [linkage group (LG) C2] (Monteros et al. <xref ref-type="bibr" rid="CR30"> 2007 </xref> ). The use of native Asian soybean <span class="trait">rust</span> <span class="trait">resistance</span> genes should be an important management component for the disease, but to date no commercial cultivars possessing any ASR <span class="trait">resistance</span> genes are currently available in the USA.
					</p>
				
					<p>&nbsp;&nbsp;
						Additional recombinant events between SNP06-43621 and Staga001 were identified in the Benning&#x000a0;&#x000d7;&#x000a0;Hyuuga RSLs. After identification of the four initial recombinant classes, more <span class="gene">STS</span> markers identified by Hyten et al. ( <xref ref-type="bibr" rid="CR21"> 2009 </xref> ) were used to further refine the genomic location of recombination events. Plants in Class I contain a recombinant event between STS70919 and STS70923 and have a tan or susceptible ARS phenotype, while Class II plants contain a recombinant event between STS70887 and STS70891 and have a RB or resistant phenotype. These data suggested that <italic> Rpp </italic> ?(Hyuuga) is located in the region of approximately 371&#x000a0;kb between STS70887 at nucleotide 43746435 and STS70923 at nucleotide 44118117. Seven genes showing a similarity with NBS-LRR disease <span class="trait">resistance</span> proteins were annotated in this 371&#x000a0;kb sequence from the Williams 82 genomic sequence (Soybean Genome Project, DoE Joint Genome Institute) (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). <table-wrap id="Tab3"> <label> Table&#x000a0;3 </label> <caption> <p> Candidate <span class="trait">resistance</span> genes showing a similarity with NBS-LRR family in the <italic> Rpp </italic> ?(Hyuuga) region on soybean chromosome <italic> <span class="gene">Gm06</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP genotypes for the 32 ancestral lines, Dillon, Hyuuga, and the other eight previously reported sources of ASR <span class="trait">resistance</span> genes at five SNP markers are shown in Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> . All 32 ancestral soybean accessions and Dillon possess a G allele at nucleotide 44058927 (SNP06-44058) on chromosome <italic> <span class="gene">Gm06</span> </italic> , while Hyuuga contains a T allele. &#x02018;Arksoy&#x02019;, &#x02018;Ralsoy&#x02019;, and &#x02018;Mejiro&#x02019; showed the same SNP genotypes as Hyuuga at the other four markers, SNP06-43621, SNP06-43731, SNP06-43983, and SNP06-44421. Hyuuga, PI 462312 ( <italic> Rpp3 </italic> ), PI 200487 ( <italic> Rpp5 </italic> ), and PI 471904 ( <italic> Rpp5 </italic> ) showed the same SNP genotypes for all five SNPs. The SNP marker SNP06-44058 can be used to distinguish the <italic> Rpp </italic> ?(Hyuuga) allele from the <italic> rpp </italic> ?(Hyuuga) allele in these 32 ancestral genotypes and the previously reported sources of ASR <span class="trait">resistance</span> other than PI 462312, PI 200487, and PI 471904. We developed melting curve analysis with SimpleProbe chemistry for use in MAS for <italic> Rpp </italic> ?(Hyuuga). The plants containing the Hyuuga allele yielded melting peaks at 60&#x000b0;C, and the plants containing the Dillon allele yielded melting peaks at 54&#x000b0;C (data not shown). The probe annealed with the Hyuuga sequence and resulted in a 6&#x000b0;C higher <italic> T </italic> <sub> m </sub> of the probe with the target sequence compared with the <italic> T </italic> <sub> m </sub> of the probe with the mismatch in the Dillon sequence. The heterozygous samples generated both peaks. This assay was further used to genotype the two parents and 100 RILs from the cross of Dillon&#x000a0;&#x000d7;&#x000a0;Hyuuga. Linkage analysis from the 100 RILs from the cross of Dillon&#x000a0;&#x000d7;&#x000a0;Hyuuga mapped the SNP marker to the interval between Satt079 and Staga001 (data not shown). The location of the SNP marker from the genetic mapping analysis is in agreement with its physical location on chromosome <italic> <span class="gene">Gm06</span> </italic> . <table-wrap id="Tab4"> <label> Table&#x000a0;4 </label> <caption> <p> The SNP haplotypes of ancestral soybean germplasm and the original sources of other known ASR <span class="trait">resistance</span> genes near the <italic> Rpp </italic> ?(Hyuuga) gene on chromosome <italic> <span class="gene">Gm06</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						The susceptibility of currently available elite soybean cultivars to ASR and the presence of ASR in most soybean-growing regions in the world make the development of ASR-resistant soybean cultivars an important breeding objective. The detection of additional SNP and SSR markers in the 3.5&#x000a0;cM interval on chromosome <italic> <span class="gene">Gm06</span> </italic> where <italic> Rpp </italic> ?(Hyuuga) was previously mapped allowed us to increase the resolution in mapping this <span class="trait">resistance</span> gene. A similar approach of combining the genotypes of RSLs with their phenotypic response to defoliating insects was used by Zhu et al. ( <xref ref-type="bibr" rid="CR43"> 2006 </xref> ) to fine map a major insect <span class="trait">resistance</span> QTL on chromosome <italic> <span class="gene">Gm07</span> </italic> (LG-M). A higher density of markers in this region of chromosome <italic> <span class="gene">Gm06</span> </italic> reduces the likelihood of recombination between markers and the target loci during the breeding process and should increase the feasibility of fine mapping and map-based cloning of this ASR <span class="trait">resistance</span> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, an increase in the marker density allowed us to identify RSLs in the region of the <italic> Rpp </italic> ?(Hyuuga) locus. The integration of the SNP genotypes from selected RSLs and the ASR reaction data allowed us to narrow down the location of the <italic> Rpp </italic> ?(Hyuuga) locus to a 371&#x000a0;kb segment on chromosome <italic> <span class="gene">Gm06</span> </italic> . Based on the gene annotation in the Soybean Genome Project, seven candidate genes containing TIR-NBS-LRR groups were identified in the sequence between STS70887 and STS70923 on chromosome <italic> <span class="gene">Gm06</span> </italic> (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). These results should increase the feasibility of cloning the <italic> Rpp </italic> ?(Hyuuga) gene following a similar approach used recently by Meyer et al. ( <xref ref-type="bibr" rid="CR28"> 2009 </xref> ) to identify the <italic> Rpp4 </italic> candidate gene from the resistant genotype PI459025B. They found that this candidate gene is a member of the CC-NBS-LRR family of disease <span class="trait">resistance</span> genes. This report suggests that other ASR genes may also belong to the NBS-LRR family of disease <span class="trait">resistance</span> genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Five SNP markers produce a distinct haplotype which can be used to distinguish Hyuuga from other reported sources of ARS <span class="trait">resistance</span> genes including <italic> Rpp1 </italic> , <italic> Rpp2 </italic> , <italic> Rpp4 </italic> , <italic> Rpp5 </italic> from PI 200526, and <italic> rpp5 </italic> , as well as from the soybean ancestors evaluated which contribute a high percent of the genetic variation to elite soybean breeding populations in the USA (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ). Both the <italic> Rpp1 </italic> and the <italic> Rpp4 </italic> loci have been mapped to chromosome <italic> <span class="gene">Gm18</span> </italic> (LG-G), and the <italic> Rpp2 </italic> locus has been mapped to chromosome <italic> <span class="gene">Gm16</span> </italic> (LG-J) using SSR markers (Hyten et al. <xref ref-type="bibr" rid="CR20"> 2007 </xref> ; Silva et al. <xref ref-type="bibr" rid="CR36"> 2008 </xref> ) indicating that <italic> Rpp </italic> ?(Hyuuga) is different from <italic> Rpp1, Rpp2, </italic> and <italic> Rpp4 </italic> . In our study the sources of <italic> Rpp3 </italic> (PI 462312) and <italic> Rpp5 </italic> (PI 200487 and PI 471904) had the same genotype at the five SNP markers as Hyuuga. Hyten et al. ( <xref ref-type="bibr" rid="CR21"> 2009 </xref> ) used a F <sub> 2:3 </sub> population from the cross of Williams 82&#x000a0;&#x000d7;&#x000a0;PI 462312 to map the <italic> Rpp3 </italic> gene to a similar region on chromosome <italic> Gm </italic> 06 as <italic> Rpp </italic> ?(Hyuuga). They also found that Hyuuga (PI 506764) and PI 463312 had the same SNP genotypes at 273 of 275 SNPs in this region. However, when PI 462312 and Hyuuga were simultaneously evaluated using Brazilian isolates of ASR, PI 462312 showed a susceptible tan lesion type while Hyuuga produced the resistant RB lesion type (Garcia et al. <xref ref-type="bibr" rid="CR9"> 2008 </xref> ). Also, <italic> Rpp3 </italic> was defeated by the <italic> P. pachyrhizi </italic> MT isolate just 2&#x000a0;years after ASR was first detected in Brazil (Pierozzi et al. <xref ref-type="bibr" rid="CR31"> 2008 </xref> ). Current efforts include screening Hyuuga and PI 462312 with multiple ASR isolates in Bio-Safety Level 3 containment facilities at Ft. Detrick to identify an isolate of <italic> P. pachyrhizi </italic> that differentiates <italic> Rpp </italic> ?(Hyuuga) from <italic> Rpp3 </italic> (Reid Fredrick, Fort Detrick MD, personal communication).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Plant antimicrobial peptides: an overview of SuperSAGE transcriptional profile and a functional review.</h2>
			<h3>PMID:20088771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kido EA(1), Pandolfi V, Houllou-Kido LM, Andrade PP, Marcelino FC, Nepomuceno AL, Abdelnoor RV, Burnquist WL, Benko-Iseppon AM.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Defensin, thionin and lipid transfer protein (LTP) gene families, which antimicrobial activity has an attractive use in protein engineering and transgenic production of agronomical important plants, have been here functionally reviewed. Also, a transcriptional overview of a set of plant SuperSAGE libraries and analysis looking for 26 bp tags possibly annotated for those families is presented. Tags differentially expressed (p = 0.05) or constitutively transcribed were identified from leaves or roots SuperSAGE libraries from important Brazilian plant species [cowpea (Vigna unguiculata (L.) Walp.), soybean (Glycine max (L.) Merr.) and modern sugarcane hybrids (Saccharum spp.)] submitted to abiotic [salt (100 mM NaCl) or <span class="trait">drought]</span> or biotic stresses [fungus inoculation (Phakopsora pachyrhizi; Asiatic Soyben <span class="trait">Rust</span> phytopathogen)]. The diverse transcriptional patterns observed, probably related to the variable range of targets and functions involved, could be the first step to unravel the antimicrobial peptide world and the plant stress response relationship. Moreover, SuperSAGE opens the opportunity to find some SNPs or even rare transcript that could be important on plant stress <span class="trait">resistance</span> mechanisms. Putative defensin or <span class="gene">LTP</span> identified by SuperSAGE following a specific plant treatment or physiological condition could be useful for future use in genetic improvement of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Generation of Phaseolus vulgaris ESTs and investigation of their regulation upon Uromyces appendiculatus infection.</h2>
			<h3>PMID:19397807  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Thibivilliers S(1), Joshi T, Campbell KB, Scheffler B, Xu D, Cooper B, Nguyen HT, Stacey G.
				<br/> 
				年份：2009 Apr 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Phaseolus vulgaris (common bean) is the second most important legume crop in the world after soybean. Consequently, <span class="trait">yield</span> losses due to fungal infection, like Uromyces appendiculatus (bean <span class="trait">rust),</span> have strong consequences. Several resistant genes were identified that confer <span class="trait">resistance</span> to bean <span class="trait">rust</span> infection. However, the downstream genes and mechanisms involved in bean <span class="trait">resistance</span> to infection are poorly characterized. RESULTS: A subtractive bean cDNA library composed of 10,581 unisequences was constructed and enriched in sequences regulated by either bean <span class="trait">rust</span> race 41, a virulent strain, or race 49, an avirulent strain on cultivar Early Gallatin carrying the <span class="trait">resistance</span> gene Ur-4. The construction of this library allowed the identification of 6,202 new bean ESTs, significantly adding to the available sequences for this plant. Regulation of selected bean genes in response to bean <span class="trait">rust</span> infection was confirmed by qRT-PCR. Plant gene expression was similar for both race 41 and 49 during the first 48 hours of the infection process but varied significantly at the later time points (72-96 hours after inoculation) mainly due to the presence of the <span class="gene">Avr4</span> gene in the race 49 leading to a hypersensitive response in the bean plants. A biphasic pattern of gene expression was observed for several genes regulated in response to fungal infection. CONCLUSION: The enrichment of the public database with over 6,000 bean ESTs significantly adds to the genomic resources available for this important crop plant. The analysis of these genes in response to bean <span class="trait">rust</span> infection provides a foundation for further studies of the mechanism of fungal disease <span class="trait">resistance.</span> The expression pattern of 90 bean genes upon <span class="trait">rust</span> infection shares several features with other legumes infected by biotrophic fungi. This finding suggests that the P. vulgaris-U. appendiculatus pathosystem could serve as a model to explore legume-rust interaction.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The control of Asian <span class="trait">rust</span> by glyphosate in glyphosate-resistant soybeans.</h2>
			<h3>PMID:18080285  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng PC(1), Clark C, Andrade GC, Balbi MC, Caldwell P.
				<br/> 
				年份：2008 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Glyphosate is a widely used broad-spectrum herbicide. Recent studies in glyphosate-resistant (GR) crops have shown that, in addition to its herbicidal activity, glyphosate exhibits activity against fungi, thereby providing disease control benefits. In <span class="gene">GR</span> wheat, glyphosate has shown both preventive and curative activities against Puccinia striiformis f. sp. tritici (Erikss) CO Johnston and Puccinia triticina Erikss, which cause stripe and leaf rusts respectively. RESULTS: Laboratory studies confirmed earlier observations that glyphosate has activity against Phakopsora pachyrhizi Syd & P Syd which causes Asian soybean <span class="trait">rust</span> (ASR) in <span class="gene">GR</span> soybeans. The results showed that glyphosate at rates between 0.84 and 1.68 kg ha(-1) delayed the onset of ASR in <span class="gene">GR</span> soybeans. However, field trials conducted in Argentina and Brazil under natural infestations showed variable ASR control from application of glyphosate in <span class="gene">GR</span> soybeans. Further field studies are ongoing to define the activity of glyphosate against ASR. CONCLUSIONS: These results demonstrate the disease control activities of glyphosate against <span class="trait">rust</span> diseases in <span class="gene">GR</span> wheat and <span class="gene">GR</span> soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright (c) 2007 Society of Chemical Industry.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation, genetic variation and expression of TIR-NBS-LRR <span class="trait">resistance</span> gene analogs from western white pine ( Pinus monticola Dougl. ex. D. Don.).</h2>
			<h3>PMID:14586641  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu JJ(1), Ekramoddoullah AK.
				<br/> 
				年份：2003 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Western white pine ( Pinus monticola Dougl. ex. D. Don., WWP) shows genetic variation in disease <span class="trait">resistance</span> to white pine blister <span class="trait">rust</span> ( Cronartium ribicola). Most plant disease <span class="trait">resistance</span> (R) genes encode proteins that belong to a superfamily with nucleotide-binding site domains (NBS) and C-terminal leucine-rich repeats (LRR). In this work a PCR strategy was used to clone R gene analogs (RGAs) from WWP using oligonucleotide primers based on the conserved sequence motifs in the NBS domain of angiosperm NBS-LRR genes. Sixty-seven NBS sequences were cloned from disease-resistant trees. BLAST searches in GenBank revealed that they shared significant identity to well-characterized R genes from angiosperms, including L and M genes from flax, the tobacco N gene and the soybean gene LM6. Sequence alignments revealed that the RGAs from WWP contained the conserved motifs identified in angiosperm NBS domains, especially those motifs specific for TIR-NBS-LRR proteins. Phylogenic analysis of plant R genes and RGAs indicated that all cloned WWP RGAs can be grouped into one major branch together with well-known R proteins carrying a <span class="gene">TIR</span> domain, suggesting they belong to the subfamily of TIR-NBS-LRR genes. In one phylogenic tree, WWP RGAs were further subdivided into fourteen clusters with an amino acid sequence identity threshold of 75%. cDNA cloning and RT-PCR analysis with gene-specific primers demonstrated that members of 10 of the 14 RGA classes were expressed in foliage tissues, suggesting that a large and diverse NBS-LRR gene family may be functional in conifers. These results provide evidence for the hypothesis that conifer RGAs share a common origin with R genes from angiosperms, and some of them may play important roles in defense mechanisms that confer disease <span class="trait">resistance</span> in western white pine. Ratios of non-synonymous to synonymous nucleotide substitutions (Ka/Ks) in the WWP NBS domains were greater than 1 or close to 1, indicating that diversifying selection and/or neutral selection operate on the NBS domains of the WWP RGA family.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide annotation of the soybean WRKY family and functional characterization of genes involved in response to <italic> Phakopsora pachyrhizi </italic> infection</h2>
			<h3>PMID:25201117</h3>
				

				
					<h3>PMCID:4172953</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bencke-Malato </surname>  <given-names> Marta </given-names>  </name>  <address>  <email> mbencke@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cabreira </surname>  <given-names> Caroline </given-names>  </name>  <address>  <email> carol.cabreira@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wiebke-Strohm </surname>  <given-names> Beatriz </given-names>  </name>  <address>  <email> strohm@sinos.net </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> B&#x000fc;cker-Neto </surname>  <given-names> Lauro </given-names>  </name>  <address>  <email> laurobucker@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mancini </surname>  <given-names> Estefania </given-names>  </name>  <address>  <email> soyestepi@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Osorio </surname>  <given-names> Marina B </given-names>  </name>  <address>  <email> miaosorio@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Homrich </surname>  <given-names> Milena S </given-names>  </name>  <address>  <email> mihomrich@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Turchetto-Zolet </surname>  <given-names> Andreia Carina </given-names>  </name>  <address>  <email> aturchetto@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Carvalho </surname>  <given-names> Mayra CCG </given-names>  </name>  <address>  <email> mayragallo@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stolf </surname>  <given-names> Renata </given-names>  </name>  <address>  <email> renatastolf@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Weber </surname>  <given-names> Ricardo LM </given-names>  </name>  <address>  <email> rlmweber@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Westergaard </surname>  <given-names> Gast&#x000f3;n </given-names>  </name>  <address>  <email> gastongw@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Castagnaro </surname>  <given-names> At&#x000ed;lio P </given-names>  </name>  <address>  <email> atilio@eeaoc.org.ar </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo V </given-names>  </name>  <address>  <email> ricardo@cnpso.embrapa.br </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar C </given-names>  </name>  <address>  <email> francmg@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Margis-Pinheiro </surname>  <given-names> M&#x000e1;rcia </given-names>  </name>  <address>  <email> marcia.margis@ufrgs.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Bodanese-Zanettini </surname>  <given-names> Maria Helena </given-names>  </name>  <address>  <email> mhbzanettini@yahoo.com.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Programa de P&#x000f3;s-Gradua&#x000e7;&#x000e3;o em Gen&#x000e9;tica e Biologia Molecular, Universidade Federal do Rio Grande do Sul (UFRGS), Porto Alegre, Brazil  </aff>  <aff id="Aff2">  <label/> Instituto de Agrobiotecnologia Rosario SA, Rosario, Argentina  </aff>  <aff id="Aff3">  <label/> Empresa Brasileira de Pesquisa Agropecu&#x000e1;ria (Embrapa Soja), Londrina, Brazil  </aff>  <aff id="Aff4">  <label/> Estaci&#x000f3;n Experimental Agroindustrial Obispo Colombres (EEAOC), Tucum&#x000e1;n, Argentina  </aff>
				<br/> 
				年份：<day> 29 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Some of the genes that presented differential expression profiles in response to the fungus were randomly selected from each classification group for more detailed analyses. <italic> <span class="gene">GmWRKY27</span> </italic> <span class="gene">(Glyma15g00570)</span> and <italic> <span class="gene">GmWRKY125</span> </italic> <span class="gene">(Glyma09g41050)</span> were differentially expressed in three of the four experiments, while <italic> <span class="gene">GmWRKY56</span> </italic> <span class="gene">(Glyma08g23380),</span> <italic> <span class="gene">GmWRKY106</span> </italic> <span class="gene">(Glyma07g02630)</span> and <italic> <span class="gene">GmWRKY20</span> </italic> <span class="gene">(Glyma08g02580)</span> in the two microarrays. <italic> <span class="gene">GmWRKY139</span> </italic> <span class="gene">(Glyma13g44730),</span> <italic> <span class="gene">GmWRKY46</span> </italic> <span class="gene">(Glyma05g36970),</span> <italic> <span class="gene">GmWRKY57</span> </italic> <span class="gene">(Glyma18g44560)</span> were also analyzed because they were closely related to at least one of the genes evaluated above. Interestingly, none of these genes was expressed in <span class="trait">rust</span> infection lesions at ten days after fungus inoculation (RNA-Seq).
					</p>
				
					<p>&nbsp;&nbsp;
						To determine which soybean <italic> WRKY </italic> genes are involved in plant defense against <italic> P. pachyrhizi </italic> infection, we performed a series of analyses to examine their expression patterns after infection. We initially compared the microarray data available in the literature [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ] with the results obtained from two additional experiments: SuperSAGE and RNA-Seq. Many genes were differentially expressed in only one library, while a few of them showed differential expression in more than one library. The modulation in the transcript levels of eight genes was validated, showing the reliability of data mining. The similar expression patterns in response to <italic> P. pachyrhizi </italic> infection was observed among closely related genes (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ), such as <italic> <span class="gene">GmWRKY61</span> </italic> <span class="gene">(Glyma06g15220)</span> and <italic> <span class="gene">GmWRKY21</span> </italic> <span class="gene">(Glyma04g39650),</span> <italic> <span class="gene">GmWRKY143</span> </italic> <span class="gene">(Glyma14g11920)</span> and <italic> <span class="gene">GmWRKY63</span> </italic> <span class="gene">(Glyma17g33920),</span> <italic> <span class="gene">GmWRKY106</span> </italic> and <italic> <span class="gene">GmWRKY56</span> </italic> , <italic> <span class="gene">GmWRKY58</span> </italic> <span class="gene">(Glyma04g40130)</span> and <italic> <span class="gene">GmWRKY97</span> </italic> <span class="gene">(Glyma06g14720).</span> This similar expression pattern suggests that these genes might share similar functions in disease <span class="trait">resistance.</span> The redundant function of <italic> <span class="gene">GmWRKY</span> </italic> genes might be beneficial in protecting the cell or organism under various stress conditions and eliciting multiple pathways that lead to the wide array of physiological responses that occur following pathogen infections [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Global expression data have suggested that the timing and the degree of induction of the defense pathway are determinants for the induction of soybean <span class="trait">resistance</span> to <italic> P. pachyrhizi </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. In our study, the induced expression of <italic> <span class="gene">GmWRKY20,</span> <span class="gene">GmWRKY27,</span> <span class="gene">GmWRKY46,</span> <span class="gene">GmWRKY57</span> </italic> , <italic> <span class="gene">GmWRKY56,</span> <span class="gene">GmWRKY106,</span> <span class="gene">GmWRKY125</span> </italic> and <italic> <span class="gene">GmWRKY139</span> </italic> in response to <italic> P. pachyrhizi </italic> was earlier and <bold> / </bold> or stronger in the resistant genotype. The expression of most genes analyzed peaked at 12 hai in the resistant genotype; therefore, we propose that these genes might be involved in non-specific defense responses. Van de Mortel et al. [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and Schneider et al. [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ] reported that <italic> P. pachyrhizi </italic> infections induce biphasic global expression. Gene expression initially peaked at 12 hai, which corresponded with the early infection processes of appressoria formation and epidermal cell penetration. The authors suggested that this peak corresponded to a non-specific defense response similar to pathogen-triggered immunity. A second phase of gene expression, which began at 72 hai and continued until 288 hai, is coincident with haustoria formation and effector protein secretion. The authors suggested that this response is consistent with the activation of <italic> RPP2 </italic> and <italic> RPP3 </italic> -mediated <span class="trait">resistance.</span> It has been shown that gene expression is rapid and increased in the incompatible interaction [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The closely related genes <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> are putative orthologues of <italic> AtWRKY40, AtWRKY18 </italic> and <italic> At </italic> <span class="gene">WRKY60</span> <italic> Arabidopsis </italic> genes. In both species, these genes were classified into group IIa. The three <italic> Arabidopsis </italic> WRKYs are involved in stress responses, which include <span class="trait">resistance</span> against the bacteria <italic> Pseudomonas syringae </italic> and fungus <italic> Botrytis cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> , <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. <italic> At </italic> <span class="gene">WRKY18</span> is a salicylic acid-induced gene that positively regulates SAR [ <xref ref-type="bibr" rid="CR63"> 63 </xref> , <xref ref-type="bibr" rid="CR64"> 64 </xref> ] and modulates PR gene expression; <italic> At </italic> W <italic> RKY18 </italic> overexpression increases <span class="trait">resistance</span> to <italic> P. syringae </italic> [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. <italic> At </italic> WRKY40 and <italic> At </italic> <span class="gene">WRKY60</span> proteins antagonize <italic> At </italic> <span class="gene">WRKY18</span> during <italic> P. syringae </italic> infection. The gain or loss of gene function in single, double or triple combination mutants resulted in increased susceptibility to <italic> B. cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. Some rice, barley and <italic> Brassica napus </italic> WRKY members from group IIa are also involved in the response to fungal and bacterial pathogens, as demonstrated using expression studies. O <italic> sWRKY62 </italic> and <italic> OsWRKY76 </italic> are upregulated in <italic> Magnaporthe grisea </italic> infected-leaves <italic> and </italic> downregulated in <italic> Xanthomonas oryzae </italic> -inoculated leaves [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. <italic> HvWRKY1 </italic> and <italic> HvWRKY2 </italic> play an important role in response to <italic> Blumeria graminis </italic> infection [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ], and <italic> BnWRKY18 and BnWRKY40 </italic> play a role in the response to <italic> Sclerotinia sclerotiorum </italic> and <italic> Alternaria brassicae </italic> infections [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Most available information concerning soybean gene function is based on data obtained from heterologous expression systems. However, as the activity of many proteins frequently depends on specific interactions that are only found in homologous backgrounds, the present study was based on a homologous expression system. An RNA interference approach was used for the silencing of four soybean homologous genes ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ). The quadruple silencing is an advantage because a single knockout of transcription factors rarely results in altered phenotypes due to functional redundancy among closely related members [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. The transgenic RNAi line used in this study generated a significant reduction in the transcript levels of the four target genes. When infected with <italic> P. pachyrhizi </italic> , the transgenic line showed increased susceptibility to the fungus. Taken together, the results strongly suggest that at least one of the four genes might be involved in the soybean <span class="trait">resistance</span> phenotype.
					</p>
				
					<p>&nbsp;&nbsp;
						Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] silenced 64 soybean WRKYs individually using virus-induced gene silencing (VIGS) to test their involvement in <italic> Rpp2- </italic> mediated <span class="trait">resistance</span> against <italic> P. pachyrhizi </italic> infection. Three of these genes ( <italic> <span class="gene">GmWRKY45</span> </italic> , <italic> <span class="gene">GmWRKY40</span> </italic> and <italic> <span class="gene">GmWRKY36</span> </italic> ) compromised the <span class="trait">resistance</span> phenotype when silenced. Phenotypic alterations were not evidenced when <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> genes were individually silenced <italic> . </italic> However, in the present study, an increased susceptibility to <italic> P. pachyrhizi </italic> infection was observed in the quadruple-silenced ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139,</span> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ) line, suggesting that this phenotype is a consequence of <italic> <span class="gene">GmWRKY27</span> </italic> and/or <italic> <span class="gene">GmWRKY139</span> </italic> silencing. Moreover, the four genes analyzed in this study could also play a synergistic role in the pathogenic defense response.
					</p>
				
					<p>&nbsp;&nbsp;
						A previous study showed that <italic> <span class="gene">GmWRKY27</span> </italic> is also strongly induced under conditions of <span class="trait">drought</span> and salt stress in the soybean [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Altogether, these data suggest that this gene is probably involved in a non-specific response that occurs upstream of biotic and abiotic stress defense routes, in contrast with the specific <italic> Rpp2- </italic> response of the genes identified by Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] in response to the fungal infection.
					</p>
				
					<p>&nbsp;&nbsp;
						a) SuperSAGE: The libraries were constructed using the leaves of a soybean resistant genotype (PI561356), which carries the <italic> Rpp1 </italic> <span class="trait">resistance</span> gene, infected with <italic> P. pachyrhizi </italic> vs. uninfected leaves (mock inoculation/control) collected at 12, 24 and 48 hours after inoculation (hai). A Plant RNeasy kit (Qiagen) was used for RNA extraction and equal amounts of RNA from each sample were used to construct the RNA pools. The libraries (inoculated and mock) were constructed at GenXPro <span class="gene">GmbH</span> (Frankfurt, Germany) using previously described methods [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ] and subsequently sequenced using the Illumina Genome Analyzer IIx. The SuperSAGE tags were analyzed using the DiscoverySpace software v.4.01 [ <xref ref-type="bibr" rid="CR79"> 79 </xref> ] to identify unique (unitags) and differentially expressed tags (p&#x02009;&#x02264;&#x02009;0.05). The libraries were constructed as part of the GenoSoja project (Brazilian Soybean Genome Consortium), and the results are available in the LGE (Laborat&#x000f3;rio de Gen&#x000f4;mica e Express&#x000e3;o, UNICAMP) Soybean Genome database [ <xref ref-type="bibr" rid="CR80"> 80 </xref> ] for members of the consortium.
					</p>
				
					<p>&nbsp;&nbsp;
						b) RNA-Seq of lesion LCM (Laser Capture Microdissection): foliar segments (1 cm <sup> 2 </sup> ) containing <italic> P. pachyrhizi </italic> lesions from two soybean resistant (PI561356) and susceptible BRS231 [ <xref ref-type="bibr" rid="CR81"> 81 </xref> ] genotypes at the <span class="gene">V2</span> growth stage were collected at 10 days after infection. The leaf segments were immediately fixed on ice in Farmer&#x02019;s solution [ <xref ref-type="bibr" rid="CR82"> 82 </xref> ], dehydrated and embedded on paraffin in accord with the methods of Cai and Lashbrook [ <xref ref-type="bibr" rid="CR83"> 83 </xref> ]. Serial sections of 12-&#x003bc;m in thickness were generated using a rotary microtome and transferred to microscope membrane slides. Twenty sections containing a variable number of <span class="trait">rust</span> lesions were prepared for each biological replicate/treatment. The PixCell II LCM system (Arcturus) and CapSure Macro LCM (Arcturus) were used to collect the foliar cells within the lesion. Total RNA was extracted using the PicoPure RNA Isolation Kit (Arcturus) from the cells collected at a variable number of infection sites for each biological replicate. The synthesis of cDNA was conducted, and high-performance paired-end (108 bp) sequencing was performed on the Illumina genome analyzer GAAllx. Low-quality RNA-Seq reads were discarded. The reads (a total of 86,301,242) were aligned against the soybean genome, and the corresponding genes were predicted using the TopHat [ <xref ref-type="bibr" rid="CR84"> 84 </xref> ] and SOAP2 [ <xref ref-type="bibr" rid="CR85"> 85 </xref> ] alignment programs. Gene expression was calculated using the FPKM (fragments per kilobase of exon per million fragments mapped) value [ <xref ref-type="bibr" rid="CR86"> 86 </xref> ]. To identify differentially expressed genes, a pair-wise comparison between the FPKM values of both genotypes was performed using a <italic> t </italic> -test at the 99% confidence level. This library was constructed as part of the Biotecsur Consortium and the results are available [ <xref ref-type="bibr" rid="CR87"> 87 </xref> ] for members of the consortium.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean plants were grown in a pot-based system maintained in greenhouse conditions at 28&#x02009;&#x000b1;&#x02009;1&#x000b0;C under a 16/8 h light/dark cycle with a light intensity of 22.5 &#x003bc;Em <sup> -2 </sup> s <sup> -1 </sup> in Embrapa Soja, Londrina, PR, Brazil. The Embrapa-48 genotype, which develops a &#x0201c;tan&#x0201d; lesion [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ], was used as the susceptible standard, and the PI561356 genotype, which carries the <italic> Rpp1 </italic> <span class="trait">resistance</span> gene [ <xref ref-type="bibr" rid="CR88"> 88 </xref> ], was used as the resistant standard. ASR isolated from Brazilian fields was maintained in a susceptible cultivar. Spores harvested from leaves exhibiting sporulating uredinia and diluted in distilled water containing 0.05% Tween-20 to a final concentration of 3 &#x000d7; 10 <sup> 5 </sup> spores/mL. The spore suspension was sprayed onto plantlets at the <span class="gene">V2</span> developmental stage. The same solution without spores was used for the mock inoculation. Subsequently, the water-misted bags were placed over each pot for one day. One trifoliate leaf from each plant was collected at 1, 12, 24, 48, 96 and 192 hai, frozen in liquid nitrogen, and stored at -80&#x000b0;C. Three biological replicates from each genotype/treatment were analyzed.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Some of the genes that presented differential expression profiles in response to the fungus were randomly selected from each classification group for more detailed analyses. <italic> <span class="gene">GmWRKY27</span> </italic> <span class="gene">(Glyma15g00570)</span> and <italic> <span class="gene">GmWRKY125</span> </italic> <span class="gene">(Glyma09g41050)</span> were differentially expressed in three of the four experiments, while <italic> <span class="gene">GmWRKY56</span> </italic> <span class="gene">(Glyma08g23380),</span> <italic> <span class="gene">GmWRKY106</span> </italic> <span class="gene">(Glyma07g02630)</span> and <italic> <span class="gene">GmWRKY20</span> </italic> <span class="gene">(Glyma08g02580)</span> in the two microarrays. <italic> <span class="gene">GmWRKY139</span> </italic> <span class="gene">(Glyma13g44730),</span> <italic> <span class="gene">GmWRKY46</span> </italic> <span class="gene">(Glyma05g36970),</span> <italic> <span class="gene">GmWRKY57</span> </italic> <span class="gene">(Glyma18g44560)</span> were also analyzed because they were closely related to at least one of the genes evaluated above. Interestingly, none of these genes was expressed in <span class="trait">rust</span> infection lesions at ten days after fungus inoculation (RNA-Seq).
					</p>
				
					<p>&nbsp;&nbsp;
						To determine which soybean <italic> WRKY </italic> genes are involved in plant defense against <italic> P. pachyrhizi </italic> infection, we performed a series of analyses to examine their expression patterns after infection. We initially compared the microarray data available in the literature [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ] with the results obtained from two additional experiments: SuperSAGE and RNA-Seq. Many genes were differentially expressed in only one library, while a few of them showed differential expression in more than one library. The modulation in the transcript levels of eight genes was validated, showing the reliability of data mining. The similar expression patterns in response to <italic> P. pachyrhizi </italic> infection was observed among closely related genes (Figure&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ), such as <italic> <span class="gene">GmWRKY61</span> </italic> <span class="gene">(Glyma06g15220)</span> and <italic> <span class="gene">GmWRKY21</span> </italic> <span class="gene">(Glyma04g39650),</span> <italic> <span class="gene">GmWRKY143</span> </italic> <span class="gene">(Glyma14g11920)</span> and <italic> <span class="gene">GmWRKY63</span> </italic> <span class="gene">(Glyma17g33920),</span> <italic> <span class="gene">GmWRKY106</span> </italic> and <italic> <span class="gene">GmWRKY56</span> </italic> , <italic> <span class="gene">GmWRKY58</span> </italic> <span class="gene">(Glyma04g40130)</span> and <italic> <span class="gene">GmWRKY97</span> </italic> <span class="gene">(Glyma06g14720).</span> This similar expression pattern suggests that these genes might share similar functions in disease <span class="trait">resistance.</span> The redundant function of <italic> <span class="gene">GmWRKY</span> </italic> genes might be beneficial in protecting the cell or organism under various stress conditions and eliciting multiple pathways that lead to the wide array of physiological responses that occur following pathogen infections [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Global expression data have suggested that the timing and the degree of induction of the defense pathway are determinants for the induction of soybean <span class="trait">resistance</span> to <italic> P. pachyrhizi </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> , <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. In our study, the induced expression of <italic> <span class="gene">GmWRKY20,</span> <span class="gene">GmWRKY27,</span> <span class="gene">GmWRKY46,</span> <span class="gene">GmWRKY57</span> </italic> , <italic> <span class="gene">GmWRKY56,</span> <span class="gene">GmWRKY106,</span> <span class="gene">GmWRKY125</span> </italic> and <italic> <span class="gene">GmWRKY139</span> </italic> in response to <italic> P. pachyrhizi </italic> was earlier and <bold> / </bold> or stronger in the resistant genotype. The expression of most genes analyzed peaked at 12 hai in the resistant genotype; therefore, we propose that these genes might be involved in non-specific defense responses. Van de Mortel et al. [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] and Schneider et al. [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ] reported that <italic> P. pachyrhizi </italic> infections induce biphasic global expression. Gene expression initially peaked at 12 hai, which corresponded with the early infection processes of appressoria formation and epidermal cell penetration. The authors suggested that this peak corresponded to a non-specific defense response similar to pathogen-triggered immunity. A second phase of gene expression, which began at 72 hai and continued until 288 hai, is coincident with haustoria formation and effector protein secretion. The authors suggested that this response is consistent with the activation of <italic> RPP2 </italic> and <italic> RPP3 </italic> -mediated <span class="trait">resistance.</span> It has been shown that gene expression is rapid and increased in the incompatible interaction [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The closely related genes <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> are putative orthologues of <italic> AtWRKY40, AtWRKY18 </italic> and <italic> At </italic> <span class="gene">WRKY60</span> <italic> Arabidopsis </italic> genes. In both species, these genes were classified into group IIa. The three <italic> Arabidopsis </italic> WRKYs are involved in stress responses, which include <span class="trait">resistance</span> against the bacteria <italic> Pseudomonas syringae </italic> and fungus <italic> Botrytis cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> , <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. <italic> At </italic> <span class="gene">WRKY18</span> is a salicylic acid-induced gene that positively regulates SAR [ <xref ref-type="bibr" rid="CR63"> 63 </xref> , <xref ref-type="bibr" rid="CR64"> 64 </xref> ] and modulates PR gene expression; <italic> At </italic> W <italic> RKY18 </italic> overexpression increases <span class="trait">resistance</span> to <italic> P. syringae </italic> [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. <italic> At </italic> WRKY40 and <italic> At </italic> <span class="gene">WRKY60</span> proteins antagonize <italic> At </italic> <span class="gene">WRKY18</span> during <italic> P. syringae </italic> infection. The gain or loss of gene function in single, double or triple combination mutants resulted in increased susceptibility to <italic> B. cinerea </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. Some rice, barley and <italic> Brassica napus </italic> WRKY members from group IIa are also involved in the response to fungal and bacterial pathogens, as demonstrated using expression studies. O <italic> sWRKY62 </italic> and <italic> OsWRKY76 </italic> are upregulated in <italic> Magnaporthe grisea </italic> infected-leaves <italic> and </italic> downregulated in <italic> Xanthomonas oryzae </italic> -inoculated leaves [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. <italic> HvWRKY1 </italic> and <italic> HvWRKY2 </italic> play an important role in response to <italic> Blumeria graminis </italic> infection [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ], and <italic> BnWRKY18 and BnWRKY40 </italic> play a role in the response to <italic> Sclerotinia sclerotiorum </italic> and <italic> Alternaria brassicae </italic> infections [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Most available information concerning soybean gene function is based on data obtained from heterologous expression systems. However, as the activity of many proteins frequently depends on specific interactions that are only found in homologous backgrounds, the present study was based on a homologous expression system. An RNA interference approach was used for the silencing of four soybean homologous genes ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139</span> </italic> , <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ). The quadruple silencing is an advantage because a single knockout of transcription factors rarely results in altered phenotypes due to functional redundancy among closely related members [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ]. The transgenic RNAi line used in this study generated a significant reduction in the transcript levels of the four target genes. When infected with <italic> P. pachyrhizi </italic> , the transgenic line showed increased susceptibility to the fungus. Taken together, the results strongly suggest that at least one of the four genes might be involved in the soybean <span class="trait">resistance</span> phenotype.
					</p>
				
					<p>&nbsp;&nbsp;
						Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] silenced 64 soybean WRKYs individually using virus-induced gene silencing (VIGS) to test their involvement in <italic> Rpp2- </italic> mediated <span class="trait">resistance</span> against <italic> P. pachyrhizi </italic> infection. Three of these genes ( <italic> <span class="gene">GmWRKY45</span> </italic> , <italic> <span class="gene">GmWRKY40</span> </italic> and <italic> <span class="gene">GmWRKY36</span> </italic> ) compromised the <span class="trait">resistance</span> phenotype when silenced. Phenotypic alterations were not evidenced when <italic> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> genes were individually silenced <italic> . </italic> However, in the present study, an increased susceptibility to <italic> P. pachyrhizi </italic> infection was observed in the quadruple-silenced ( <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmWRKY139,</span> <span class="gene">GmWRKY56</span> </italic> and <italic> <span class="gene">GmWRKY106</span> </italic> ) line, suggesting that this phenotype is a consequence of <italic> <span class="gene">GmWRKY27</span> </italic> and/or <italic> <span class="gene">GmWRKY139</span> </italic> silencing. Moreover, the four genes analyzed in this study could also play a synergistic role in the pathogenic defense response.
					</p>
				
					<p>&nbsp;&nbsp;
						A previous study showed that <italic> <span class="gene">GmWRKY27</span> </italic> is also strongly induced under conditions of <span class="trait">drought</span> and salt stress in the soybean [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. Altogether, these data suggest that this gene is probably involved in a non-specific response that occurs upstream of biotic and abiotic stress defense routes, in contrast with the specific <italic> Rpp2- </italic> response of the genes identified by Pandey et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] in response to the fungal infection.
					</p>
				
					<p>&nbsp;&nbsp;
						a) SuperSAGE: The libraries were constructed using the leaves of a soybean resistant genotype (PI561356), which carries the <italic> Rpp1 </italic> <span class="trait">resistance</span> gene, infected with <italic> P. pachyrhizi </italic> vs. uninfected leaves (mock inoculation/control) collected at 12, 24 and 48 hours after inoculation (hai). A Plant RNeasy kit (Qiagen) was used for RNA extraction and equal amounts of RNA from each sample were used to construct the RNA pools. The libraries (inoculated and mock) were constructed at GenXPro <span class="gene">GmbH</span> (Frankfurt, Germany) using previously described methods [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ] and subsequently sequenced using the Illumina Genome Analyzer IIx. The SuperSAGE tags were analyzed using the DiscoverySpace software v.4.01 [ <xref ref-type="bibr" rid="CR79"> 79 </xref> ] to identify unique (unitags) and differentially expressed tags (p&#x02009;&#x02264;&#x02009;0.05). The libraries were constructed as part of the GenoSoja project (Brazilian Soybean Genome Consortium), and the results are available in the LGE (Laborat&#x000f3;rio de Gen&#x000f4;mica e Express&#x000e3;o, UNICAMP) Soybean Genome database [ <xref ref-type="bibr" rid="CR80"> 80 </xref> ] for members of the consortium.
					</p>
				
					<p>&nbsp;&nbsp;
						b) RNA-Seq of lesion LCM (Laser Capture Microdissection): foliar segments (1 cm <sup> 2 </sup> ) containing <italic> P. pachyrhizi </italic> lesions from two soybean resistant (PI561356) and susceptible BRS231 [ <xref ref-type="bibr" rid="CR81"> 81 </xref> ] genotypes at the <span class="gene">V2</span> growth stage were collected at 10 days after infection. The leaf segments were immediately fixed on ice in Farmer&#x02019;s solution [ <xref ref-type="bibr" rid="CR82"> 82 </xref> ], dehydrated and embedded on paraffin in accord with the methods of Cai and Lashbrook [ <xref ref-type="bibr" rid="CR83"> 83 </xref> ]. Serial sections of 12-&#x003bc;m in thickness were generated using a rotary microtome and transferred to microscope membrane slides. Twenty sections containing a variable number of <span class="trait">rust</span> lesions were prepared for each biological replicate/treatment. The PixCell II LCM system (Arcturus) and CapSure Macro LCM (Arcturus) were used to collect the foliar cells within the lesion. Total RNA was extracted using the PicoPure RNA Isolation Kit (Arcturus) from the cells collected at a variable number of infection sites for each biological replicate. The synthesis of cDNA was conducted, and high-performance paired-end (108 bp) sequencing was performed on the Illumina genome analyzer GAAllx. Low-quality RNA-Seq reads were discarded. The reads (a total of 86,301,242) were aligned against the soybean genome, and the corresponding genes were predicted using the TopHat [ <xref ref-type="bibr" rid="CR84"> 84 </xref> ] and SOAP2 [ <xref ref-type="bibr" rid="CR85"> 85 </xref> ] alignment programs. Gene expression was calculated using the FPKM (fragments per kilobase of exon per million fragments mapped) value [ <xref ref-type="bibr" rid="CR86"> 86 </xref> ]. To identify differentially expressed genes, a pair-wise comparison between the FPKM values of both genotypes was performed using a <italic> t </italic> -test at the 99% confidence level. This library was constructed as part of the Biotecsur Consortium and the results are available [ <xref ref-type="bibr" rid="CR87"> 87 </xref> ] for members of the consortium.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean plants were grown in a pot-based system maintained in greenhouse conditions at 28&#x02009;&#x000b1;&#x02009;1&#x000b0;C under a 16/8 h light/dark cycle with a light intensity of 22.5 &#x003bc;Em <sup> -2 </sup> s <sup> -1 </sup> in Embrapa Soja, Londrina, PR, Brazil. The Embrapa-48 genotype, which develops a &#x0201c;tan&#x0201d; lesion [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ], was used as the susceptible standard, and the PI561356 genotype, which carries the <italic> Rpp1 </italic> <span class="trait">resistance</span> gene [ <xref ref-type="bibr" rid="CR88"> 88 </xref> ], was used as the resistant standard. ASR isolated from Brazilian fields was maintained in a susceptible cultivar. Spores harvested from leaves exhibiting sporulating uredinia and diluted in distilled water containing 0.05% Tween-20 to a final concentration of 3 &#x000d7; 10 <sup> 5 </sup> spores/mL. The spore suspension was sprayed onto plantlets at the <span class="gene">V2</span> developmental stage. The same solution without spores was used for the mock inoculation. Subsequently, the water-misted bags were placed over each pot for one day. One trifoliate leaf from each plant was collected at 1, 12, 24, 48, 96 and 192 hai, frozen in liquid nitrogen, and stored at -80&#x000b0;C. Three biological replicates from each genotype/treatment were analyzed.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of beta-tubulin from <italic> Phakopsora pachyrhizi </italic> , the causal agent of Asian soybean <span class="trait">rust</span></h2>
			<h3>PMID:21637494</h3>
				
					<h3>PMCID:3036848</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="no">  <name>  <surname> Maciel </surname>  <given-names> Talles Eduardo Ferreira </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Freire </surname>  <given-names> Ma&#x000ed;ra Cristina Menezes </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> de Almeida </surname>  <given-names> &#x000c1;lvaro M. R. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> de Oliveira </surname>  <given-names> Luiz Orlando </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 1 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To the best of our knowledge, no studies have examined the relationship between amino acid substitutions and benzimidazole <span class="trait">resistance</span> in <italic> P. pachyrhizi </italic> . As shown here, the amino acid substitutions in &#x003b2;-tubulin associated with benzimidazole <span class="trait">resistance</span> in closely related organisms were absent in the predicted Pp <span class="gene">TUB</span> sequences from the <italic> P. pachyrhizi </italic> samples collected in seven Brazilian soybean fields. Nevertheless, we found a high level of sequence identity between Pp <span class="gene">TUB</span> and &#x003b2;-tubulins of model organisms such as <italic> A. nidulans </italic> and <italic> N. crassa </italic> (Figures <xref ref-type="fig" rid="fig1"> 1 </xref> and <xref ref-type="fig" rid="fig2"> 2 </xref> ), for which there is a clear relationship between amino acid substitutions at given sites and benzimidazole <span class="trait">resistance</span> ( <xref ref-type="bibr" rid="Jungetal1992"> Jung <italic> et al. </italic> , 1992 </xref> ; <xref ref-type="bibr" rid="CooleyandCaten1993"> Cooley and Caten, 1993 </xref> ; <xref ref-type="bibr" rid="Oakley2004"> Oakley, 2004 </xref> ). We hypothesize that the emergence of <span class="trait">resistance</span> in <italic> P. pachyrhizi </italic> is likely to follow a similar pattern of amino acid substitutions in Pp TUB. However, we cannot exclude the possibility that there were no benzimidazole resistant mutants in the soybean fields at the time of sampling. Because of the limitations inherent to our sampling and sequencing strategies, we can only imply that if mutants were indeed present, their frequencies were extremely low. The appearance of natural mutants with amino acid substitutions in Pp TUB, especially at His <sub> 6 </sub> , Glu <sub> 198 </sub> and Phe <sub> 200 </sub> , should be closely monitored since the continual use of benzimidazoles for controlling Asian soybean <span class="trait">rust</span> may well increase the selective pressure under field conditions in the near future.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phytohormone signaling pathway analysis method for comparing hormone responses in plant-pest interactions</h2>
			<h3>PMID:22846705</h3>
				
					<h3>PMCID:3460778</h3>
					<h4>期刊:BMC Research Notes</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Studham </surname>  <given-names> Matthew E </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> matthew.studham@scilifelab.se </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A2">  <name>  <surname> MacIntosh </surname>  <given-names> Gustavo C </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> gustavo@iastate.edu </email>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						At 168 hours, during the second phase, SA appeared to be the main regulator of ASR defense in both genotypes (Figure <xref ref-type="fig" rid="F1"> 1 </xref> , and Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). In the resistant response, ET and JA were induced at about half the SA induction level, while ABA was only weakly induced. In the susceptible response the only notable result was weak induction of SA (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). In the original analysis of this dataset, van de Mortel <italic> et al. </italic> also concluded that the second phase of the response to this pathogen is faster and stronger in resistant plants ( <italic> Rpp2) </italic> than in susceptible plants. While no direct conclusion was put forward on the role of hormones in these responses, genes regulated by SA and by JA were reported as differentially regulated in response to ASR [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. Regulation of SA and JA related genes was also observed in an experiment that analyzed ASR-induced changes in gene expression in susceptible plants at different growth stages [ <xref ref-type="bibr" rid="B39"> 39 </xref> ], and in work that used laser capture microdissection followed by microarray analyses to study the response of susceptible soybean palisade mesophyll cells infected with ASR [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. Finally, the functional role of SA was confirmed by virus induced gene silencing experiments that targeted <italic> <span class="gene">GmNPR1</span> </italic> and other genes involved in SA signaling [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. When these genes were silenced, <italic> Rpp2 </italic> -mediated <span class="trait">resistance</span> was compromised.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Probing Protein Sequences as Sources for Encrypted Antimicrobial Peptides</h2>
			<h3>PMID:23029273</h3>
				
					<h3>PMCID:3461044</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Brand </surname>  <given-names> Guilherme D. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Magalh&#x000e3;es </surname>  <given-names> Mariana T. Q. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tinoco </surname>  <given-names> Maria L. P. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arag&#x000e3;o </surname>  <given-names> Francisco J. L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nicoli </surname>  <given-names> Jacques </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kelly </surname>  <given-names> Sharon M. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cooper </surname>  <given-names> Alan </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bloch </surname>  <given-names> Carlos </given-names>  <suffix> Jr </suffix>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 8 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In order to evaluate if peptides encrypted in <italic> G. max </italic> proteins can confer <span class="trait">resistance</span> to the plant&#x02019;s natural pathogens, two novel IAPs were filtered and synthesized. <span class="gene">Gm0025x00667(75&#x02013;100),</span> primary structure RWRFLRKISSVHMFSVKALDDFRQL, is a fragment of the enzyme flavonoid 3-hydroxylase, while <span class="gene">Gm0026x00785(77&#x02013;103),</span> primary structure HKMDLHWYLRTLEEVVIRALQRFQFR, is derived from the lipoate-protein ligase B. They both inhibited the <italic> in vitro </italic> growth of <italic> X. axonopodis </italic> pv. <italic> glycines </italic> , the causative agent of the bacterial pustule disease, at 5 and 10 &#x000b5;M, respectively. IAPs were also tested <italic> ex vivo </italic> for the inhibition of the fixation of asian <span class="trait">rust</span> spores on <italic> G. max </italic> leaves. In general, IAPs co-incubated with spores of <italic> P. pachyrhizi </italic> on the surface of leaves of <italic> G. max </italic> decreased significantly the area occupied by uredias at micromolar concentrations ( <xref ref-type="fig" rid="pone-0045848-g006"> Fig. 6a </xref> ). <span class="gene">Gm0025x00667(75&#x02013;100)</span> reduced the infected area to approximately 50% of the control at 8 &#x000b5;g/mL (3 &#x000b5;M).
					</p>
				
					<p>&nbsp;&nbsp;
						(a) AMPs and IAPs at 8 to 128 &#x000b5;g/mL were co-incubated with 3.5&#x000d7;10 <sup> 5 </sup> /ml <italic> P. pachyrhizi </italic> spores for seven days on the surface of <italic> G. max </italic> leaves, followed by the estimation of the infected area (represented as a histogram graph). Bars correspond to the sample standard deviation ( <italic> n </italic> &#x0200a;=&#x0200a;3). All peptides at all tested concentrations resulted in significantly different pustule areas at a 95% confidence interval except for those marked with an asterisk (*). Wild type and <italic> G. max </italic> plants transformed with a vector containing gb|ABM17058.1|(213&#x02013;231) were grown to the <span class="gene">V3</span> stage and sprayed with a suspension of <italic> P. pachyrhizi </italic> spores (10 <sup> 6 </sup> spores/mL). (b) Transformed plants (72.14.5 and 72.14.9) were co-cultivated with control plants (wt/+) for 15 days. (c) Control lineages, designated as wt/&#x02212; and wt/+, correspond to wild-type plants subjected to spraying with water alone or the Asian <span class="trait">rust</span> spore suspension, respectively. Intragenic lineages 72.14.5 and 72.14.9 presented a significant reduction in the number of uredia per foliar area (see text).
					</p>
				
					<p>&nbsp;&nbsp;
						To test whether IAPs confer <span class="trait">resistance</span> to the Asian <span class="trait">rust</span> when expressed by plants, <italic> G. max </italic> was transformed with a vector encoding the IAP gb|ABM17058.1|(213&#x02013;231) <xref ref-type="bibr" rid="pone.0045848-Rech1"> [30] </xref> , <xref ref-type="bibr" rid="pone.0045848-Aragao1"> [31] </xref> . This molecule was conceived by serendipity prior to the systematic attempts to filter, optimize and validate the IAPs described herein. Transformed soybean plants were cultivated along with control lineages to the <span class="gene">V3</span> stage and sprinkled with a suspension of Asian <span class="trait">rust</span> spores ( <xref ref-type="fig" rid="pone-0045848-g006"> Fig. 6b </xref> ). Two lineages presented a significant decrease in the number of uredia per cm <sup> 2 </sup> of foliar area based on the number of pustules observed on expanded leaves 15 days after inoculation ( <xref ref-type="fig" rid="pone-0045848-g006"> Fig. 6b </xref> ). The transformed lineages 72.14.5 and 72.14.9 had a reduced number of uredia per foliar area when compared to control plants, from 45&#x000b1;8 pustules/cm <sup> 2 </sup> to 7&#x000b1;3 pustules/cm <sup> 2 </sup> and 12&#x000b1;3 pustules/cm <sup> 2 </sup> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						The gene encoding the peptide gb|ABM17058.1|(213&#x02013;231), primary structure MIKAFKEATKVDKVVVLWTA, was synthesized by Epoch Life Science Inc. (Sugar Land, TX, USA) and cloned into the vector pBluKSPOXDCAHAS replacing the <italic> oxdc </italic> gene <xref ref-type="bibr" rid="pone.0045848-Cunha1"> [53] </xref> . The vector was used to transform soybean plants as previously described <xref ref-type="bibr" rid="pone.0045848-Rech1"> [30] </xref> , <xref ref-type="bibr" rid="pone.0045848-Aragao1"> [31] </xref> . Six intragenic soybean lines were tested for tolerance to Asian <span class="trait">rust</span> ( <italic> Phakopsora pachyrhizi </italic> ). The plants (20 plants per line at <span class="gene">V3</span> stage <xref ref-type="bibr" rid="pone.0045848-Fehr1"> [54] </xref> ) were sprayed with a spore suspension (10 <sup> 6 </sup> spores/mL) and maintained in the greenhouse at room temperature and sprinkling 4 times a day. Leaves were photographed after 15 days and the number of pustules per square centimeter counted.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nonhost <span class="trait">resistance</span> to <span class="trait">rust</span> pathogens &#x02013; a continuation of continua</h2>
			<h3>PMID:25566270</h3>
				
					<h3>PMCID:4263244</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bettgenhaeuser </surname>  <given-names> Jan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/142498"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gilbert </surname>  <given-names> Brian </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ayliffe </surname>  <given-names> Michael </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/149214"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moscou </surname>  <given-names> Matthew J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://community.frontiersin.org/people/u/144416"/>  </contrib>
				<br/> 
				年份：<day> 07 </day>  <month> 11 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						While in some instances very few genes confer apparent immunity to a pathogen species, in other instances it appears that NHR is polygenic. The loss of a single NHR gene in this latter case is unlikely to result in susceptibility of the donor gene species. Redundancy in NHR mechanisms to mildew pathogens has been well-established in <italic> Arabidopsis thaliana </italic> with penetration <span class="trait">resistance</span> mechanisms, contributed to by the <italic> PEN </italic> proteins, preceding a second layer of <span class="trait">resistance</span> dependent upon SA signaling, as described above ( <xref rid="B34" ref-type="bibr"> <span class="gene">Fan</span> and Doerner, 2012 </xref> ). Other forms of polygenic NHR do not involve multiple independent layers of <span class="trait">resistance,</span> but rather multiple loci, each conferring minor additive affects, as observed in the interaction between barley and <italic> Puccinia triticina </italic> ( <xref rid="B74" ref-type="bibr"> Neu et al., 2003 </xref> ; <xref rid="B48" ref-type="bibr"> Jafary et al., 2008 </xref> ; <xref rid="B78" ref-type="bibr"> Niks, 2014 </xref> ). This latter <span class="trait">resistance,</span> however, is unlikely to be of practical use due to the genetic complexity required to achieve useful levels of <span class="trait">resistance.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptomic and metabolomic analyses identify a role for chlorophyll catabolism and phytoalexin during <italic> Medicago </italic> nonhost <span class="trait">resistance</span> against Asian soybean <span class="trait">rust</span></h2>
			<h3>PMID:26267598</h3>
				
					<h3>PMCID:4533520</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ishiga </surname>  <given-names> Yasuhiro </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rao Uppalapati </surname>  <given-names> Srinivasa </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  <xref ref-type="author-notes" rid="n2"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gill </surname>  <given-names> Upinder S. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huhman </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Yuhong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mysore </surname>  <given-names> Kirankumar S. </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Plant Biology Division, The Samuel Roberts Noble Foundation </institution> , Ardmore, OK 73401,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Faculty of Life and Environmental Sciences, University of Tsukuba </institution> , 1-1-1 Tennodai, Tsukuba, Ibaraki 305-8572,  <country> Japan </country>  </aff>
				<br/> 
				年份：<day> 13 </day>  <month> 07 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several genes related to phytohormones including salicylic acid (SA), jasmonic acid (JA), ethylene (ET), auxin, cytokinin and gibberellin were differently regulated during nonhost interactions ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S4; Supplementary Tables S5, S6, S7 and S8 </xref> ). Several SA-regulated <italic> Pathogenesis-related </italic> ( <italic> PR </italic> ) genes were also induced during nonhost <span class="trait">resistance</span> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Tables S5 and S7 </xref> ). Further, several genes involved in ethylene biosynthesis were induced during nonhost interactions ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Tables S5 and S7 </xref> ). These genes included <italic> 1-Aminocyclopropane-1-carboxylate synthase </italic> ( <italic> ACC synthase </italic> ) and <italic> 1-Aminocyclopropane-1-carboxylate oxidase </italic> ( <italic> ACC oxidase </italic> ). <italic> P. pachyrhizi </italic> also induced the expression of genes encoding ET-responsive transcription factors (AP2/ERF family members) ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S4; Supplementary Tables S6 and S7 </xref> ). In addition, several genes involved in JA biosynthesis were induced during nonhost interactions. These genes included <italic> Lipoxygenases </italic> ( <italic> <span class="gene">LOX</span> </italic> ), <italic> Allene oxide synthase </italic> ( <italic> AOS </italic> ) and <italic> 12-Oxophytodienoic acid 10,11-reductase </italic> ( <italic> OPR </italic> ) ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S1 </xref> ). However, the expression of several other members of JA-related genes including <italic> <span class="gene">LOX</span> </italic> , <italic> Allene oxide cyclase </italic> ( <italic> AOC </italic> ) and <italic> OPR </italic> were downregulated during nonhost <span class="trait">resistance</span> ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S2; Supplementary Tables S6 and S8 </xref> ). These results indicate a correlation between up-regulation of SA-, ET- and JA-mediated signaling pathways and nonhost <span class="trait">resistance</span> of <italic> M. truncatula </italic> against ASR.
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in <xref ref-type="fig" rid="f2"> Fig. 2 </xref> and <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs S1 and S2 </xref> , microarray analysis revealed a significant induction of transcripts derived from genes encoding proteins associated with phenylpropanoid, flavonoid and isoflavonoid metabolic pathways in <italic> M. truncatula </italic> leaves at 12 and 24 hpi with <italic> P. pachyrhizi </italic> ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). Interestingly, the genes involved in the phytoalexin production pathway were induced and the genes involved in the terpenoid production pathway were suppressed in response to <italic> P. pachyrhizi </italic> inoculation ( <xref ref-type="fig" rid="f2"> Fig. 2 </xref> ). Furthermore, real-time quantitative RT-PCR (RT-qPCR) analysis confirmed the induction of genes involved in the phytoalexin medicarpin metabolic pathway, including <italic> Phenylalanine ammonia-lyase </italic> ( <italic> <span class="gene">PAL</span> </italic> ), <italic> Chalcone synthase </italic> ( <italic> <span class="gene">CHS</span> </italic> ), <italic> Chalcone reductase </italic> ( <italic> CHR </italic> ), <italic> Chalcone isomerase </italic> ( <italic> <span class="gene">CHI</span> </italic> ), <italic> Isoflavone synthase </italic> ( <italic> <span class="gene">IFS</span> </italic> ), <italic> Isoflavone 4&#x02032;-O-methyltransferase </italic> ( <italic> HI4&#x02032;OMT </italic> ), <italic> 2-Hydroxyisoflavanone dehydratase </italic> ( <italic> 2HID </italic> ), <italic> Isoflavone 2&#x02032;-hydroxylase </italic> ( <italic> I2&#x02032;H </italic> ), <italic> Isoflavone reductase </italic> ( <italic> IFR </italic> ) and <italic> Vestitone reductase </italic> ( <italic> VR </italic> ) ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ). The expression profiles of these genes in response to <italic> P. pachyrhizi </italic> inoculation were almost identical. The transcripts of these genes were significantly induced within 12 hpi and slightly decreased until 72 hpi with <italic> P. pachyrhizi </italic> . These results indicate a correlation between phytoalexin medicarpin pathway and nonhost <span class="trait">resistance</span> against ASR.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcriptome analysis revealed downregulation of a large number of genes involved in photosynthesis in response to <italic> P. pachyrhizi </italic> inoculation ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs S2 and S3 </xref> ). At the same time, many genes with exception of <italic> Red chlorophyll catabolite reductase </italic> ( <italic> RCCR </italic> ) that encode enzymes involved in the chlorophyll catabolism pathway, including <italic> Chlorophyll b reductase </italic> ( <italic> NYC1 </italic> ), <italic> Chlorophyllase </italic> ( <italic> <span class="gene">CHL</span> </italic> ), <italic> Pheophorbide a oxygenase </italic> ( <italic> PAO </italic> ) and <italic> Stay-green </italic> ( <italic> SGR </italic> ), were upregulated in response to <italic> P. pachyrhizi </italic> inoculation ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table. S9 </xref> ). Previous studies have demonstrated an important link between light and disease <span class="trait">resistance,</span> based on the mutants involved in the chlorophyll degradation pathway in Arabidopsis <xref ref-type="bibr" rid="b32"> 32 </xref> <xref ref-type="bibr" rid="b33"> 33 </xref> . To further investigate the molecular mechanism of the chlorophyll catabolism pathway in nonhost <span class="trait">resistance</span> of <italic> M. truncatula </italic> against ASR, expression analysis of <italic> NYC1 </italic> , <italic> <span class="gene">CHL</span> </italic> , <italic> PAO </italic> , <italic> RCCR </italic> and <italic> SGR </italic> was performed by RT-qPCR. Consistent with the microarray results ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S9 </xref> ), the expression of <italic> NYC1 </italic> , <italic> <span class="gene">CHL</span> </italic> , <italic> PAO </italic> and <italic> SGR </italic> was significantly induced by ASR ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ). Interestingly, the expression of <italic> RCCR </italic> was suppressed 12&#x02009;hours after inoculation with ASR ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S9 </xref> ). Among these genes, <italic> NYC1 </italic> , <italic> <span class="gene">CHL</span> </italic> , <italic> PAO </italic> and <italic> RCCR </italic> are known to encode enzymes involved in chlorophyll degradation <xref ref-type="bibr" rid="b34"> 34 </xref> . Although <italic> SGR </italic> is believed to regulate the enzymes involved in chlorophyll degradation and involved in disease symptom development <xref ref-type="bibr" rid="b35"> 35 </xref> <xref ref-type="bibr" rid="b36"> 36 </xref> <xref ref-type="bibr" rid="b37"> 37 </xref> , the precise function of <italic> SGR </italic> in plant defense is still unknown. Therefore, we decided to further characterize the role of <italic> SGR </italic> in nonhost <span class="trait">resistance</span> against ASR.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we characterized a model pathosystem between <italic> M. truncatula </italic> and ASR to investigate the molecular mechanism of nonhost <span class="trait">resistance</span> ( <xref ref-type="fig" rid="f1"> Fig. 1 </xref> ). We identified the key genes or pathways for nonhost disease <span class="trait">resistance</span> of <italic> M. truncatula </italic> against ASR and demonstrated that a large number of genes involved in phenylpropanoid and (iso) flavonoid biosynthetic pathways that produce phytoalexin medicarpin were upregulated after inoculation with <italic> P. pachyrhizi </italic> ( <xref ref-type="fig" rid="f2"> Figs 2 </xref> and <xref ref-type="fig" rid="f3"> 3 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S3 </xref> ). Furthermore, a significant accumulation of the metabolite medicarpin was detected 72&#x02009;hours after inoculation with <italic> P. pachyrhizi </italic> ( <xref ref-type="fig" rid="f7"> Fig. 7 </xref> ). Consistent with the results from transcriptome and metabolome analyses during nonhost <span class="trait">resistance,</span> phytoalexin medicarpin significantly inhibited the pre-infection structure formation of <italic> P. pachyrhizi </italic> ( <xref ref-type="fig" rid="f8"> Fig. 8 </xref> ). These results clearly suggest a correlation between phenylpropanoid and (iso) flavonoid pathways leading to medicarpin production and nonhost <span class="trait">resistance</span> in <italic> M. truncatula </italic> against fungal pathogens such as <italic> P. pachyrhizi </italic> . Interestingly, similar results were observed in soybean transcriptome analysis for <italic> Rpp2 </italic> -mediated disease <span class="trait">resistance</span> against ASR wherein the expression of phenylpropanoid pathway genes, such as <italic> <span class="gene">PAL</span> </italic> , was induced during infection <xref ref-type="bibr" rid="b39"> 39 </xref> . It was reported that <italic> <span class="gene">PAL1</span> </italic> -silenced soybean plants compromised <italic> Rpp2 </italic> -mediated disease <span class="trait">resistance</span> against ASR <xref ref-type="bibr" rid="b40"> 40 </xref> . In addition, the expression of genes involved in the phenylpropanoid pathway was also reported to be induced in <italic> Rpp3 </italic> -mediated disease <span class="trait">resistance</span> in soybean <xref ref-type="bibr" rid="b41"> 41 </xref> . It is known that the phenylpropanoid pathway has an important role in providing precursors not only for the production of antimicrobial metabolites, such as phytoalexins, but also for lignin biosynthesis <xref ref-type="bibr" rid="b42"> 42 </xref> . Together with our present study, these results suggest that both <italic> R </italic> -gene-mediated <span class="trait">resistance</span> and nonhost <span class="trait">resistance</span> utilize the phenylpropanoid pathway to produce antimicrobial metabolites.
					</p>
				
					<p>&nbsp;&nbsp;
						In recent years, the importance of chloroplasts in plant immunity has been reported <xref ref-type="bibr" rid="b50"> 50 </xref> . There are numerous reports showing that pathogens target host chloroplast to cause disease <xref ref-type="bibr" rid="b51"> 51 </xref> <xref ref-type="bibr" rid="b52"> 52 </xref> <xref ref-type="bibr" rid="b53"> 53 </xref> . Furthermore, there is an important link between light and disease <span class="trait">resistance.</span> The intermediate metabolites derived from chlorophyll degradation are known to generate ROS which function as signaling molecules to cause cell death, that play a role in disease <span class="trait">resistance</span> <xref ref-type="bibr" rid="b32"> 32 </xref> <xref ref-type="bibr" rid="b54"> 54 </xref> <xref ref-type="bibr" rid="b55"> 55 </xref> . Consistent with these reports, the present study demonstrated that the expression of genes involved in chlorophyll catabolism, including <italic> NYC1 </italic> , <italic> <span class="gene">CHL</span> </italic> , <italic> PAO </italic> and <italic> SGR </italic> , was significantly induced by inoculation with <italic> P. pachyrhizi </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ; <xref ref-type="supplementary-material" rid="S1"> Supplementary Table S9 </xref> ). On the other hand, the expression of genes related to photosynthetic pathways was suppressed during <italic> P. pachyrhizi </italic> infection ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Figs S2 and S3 </xref> ). These results suggest that optimization of chlorophyll catabolism could be essential for protecting plants against the toxicity of ROS, especially when the photosynthetic activity is suppressed by stresses such as pathogen infection. Interestingly, similar results were observed in a previous study of Arabidopsis <italic> SGR </italic> . It was reported that the expression of <italic> AtSGR </italic> was induced after inoculation with an avirulent bacterial pathogen, <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> DC3000 ( <italic> Pst </italic> DC3000) carrying <italic> AvrRpm1 </italic> <xref ref-type="bibr" rid="b55"> 55 </xref> , suggesting that <italic> SGR </italic> may play a role in plant defense. Furthermore, transcriptome analysis for soybean demonstrated that the expression of genes involved in chlorophyll catabolism including <italic> SGR </italic> and <italic> <span class="gene">CHL</span> </italic> was induced in the compatible interactions with <italic> P </italic> . <italic> pachyrhizi </italic> <xref ref-type="bibr" rid="b41"> 41 </xref> . However, the induction of <italic> SGR </italic> and <italic> <span class="gene">CHL</span> </italic> was ~2.5-fold in contrary to ~40-fold induction of <italic> SGR </italic> and <italic> <span class="gene">CHL</span> </italic> during nonhost <span class="trait">resistance</span> in <italic> M </italic> . <italic> truncatula </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4 </xref> ). Consistent with the fact that pathogen effectors modulate host defense machinery, we speculate that <italic> P. pachyrhizi </italic> utilizes effectors to interfere with the expression of chlorophyll catabolism pathway genes, especially <italic> SGR </italic> , to suppress plant defense.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fighting Asian Soybean <span class="trait">Rust</span></h2>
			<h3>PMID:27375652</h3>
				
					<h3>PMCID:4894884</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Langenbach </surname>  <given-names> Caspar </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/251240/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Campe </surname>  <given-names> Ruth </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Beyer </surname>  <given-names> Sebastian F. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mueller </surname>  <given-names> Andr&#x000e9; N. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Conrath </surname>  <given-names> Uwe </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/350862/overview"/>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptome analysis disclosed different components of <italic> Rpp2 </italic> -mediated <span class="trait">resistance</span> to SBR in soybean ( <xref rid="B146" ref-type="bibr"> van de Mortel et al., 2007 </xref> ; <xref rid="B117" ref-type="bibr"> Pandey et al., 2011 </xref> ). Of 140 candidates tested by VIGS, eleven genes clearly contributed to <italic> Rpp2 </italic> -mediated SBR <span class="trait">resistance.</span> The genes encompassed <italic> <span class="gene">GmEDS1,</span> <span class="gene">GmPAD4</span> </italic> , and <italic> <span class="gene">GmNPR1</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						EDS1 and <span class="gene">PAD4</span> are key regulators of several types of plant disease <span class="trait">resistance</span> (basal, <italic> R </italic> gene-mediated, and NHR). The two proteins are required for accumulation of SA, and they control various SA-dependent defense pathways ( <xref rid="B44" ref-type="bibr"> Falk et al., 1999 </xref> ; <xref rid="B70" ref-type="bibr"> Jirage et al., 1999 </xref> ; <xref rid="B113" ref-type="bibr"> Nawrath et al., 2002 </xref> ; <xref rid="B88" ref-type="bibr"> Lipka et al., 2005 </xref> ; <xref rid="B152" ref-type="bibr"> Wiermer et al., 2005 </xref> ; <xref rid="B80" ref-type="bibr"> Langenbach et al., 2013 </xref> ; <xref rid="B149" ref-type="bibr"> Wang et al., 2014 </xref> ). Because silencing of <italic> <span class="gene">GmEDS1</span> </italic> or <italic> <span class="gene">GmPAD4</span> </italic> lead to susceptibility of otherwise resistant soybean lines carrying <italic> Rpp2 </italic> , EDS1 and <span class="gene">PAD4</span> seem to control also <italic> Rpp2 </italic> -mediated SBR <span class="trait">resistance</span> in soybean ( <xref rid="B117" ref-type="bibr"> Pandey et al., 2011 </xref> ). SA accumulation is thus likely to limit the growth and reproduction of <italic> P. pachyrhizi </italic> in soybean. Because <italic> <span class="gene">PAD4</span> </italic> is also required for <italic> Arabidopsis </italic> postinvasion NHR to <italic> P. pachyrhizi </italic> ( <xref rid="B80" ref-type="bibr"> Langenbach et al., 2013 </xref> ), SA-associated defense responses seem to be highly effective in antagonizing SBR disease. However, overexpression of SA biosynthesis genes is likely not to provide a realistic agronomical solution for SBR control because constitutive SA accumulation often causes dwarfism ( <xref rid="B16" ref-type="bibr"> Bowling et al., 1994 </xref> ; <xref rid="B86" ref-type="bibr"> Li et al., 2001 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The importance of TFs in conferring SBR <span class="trait">resistance</span> became obvious when <xref rid="B146" ref-type="bibr"> van de Mortel et al. (2007) </xref> and <xref rid="B129" ref-type="bibr"> Schneider et al. (2011) </xref> found that TF genes are being overrepresented among genes whose expression is activated in the biphasic transcriptional response in SBR-resistant soybean genotypes harboring <italic> Rpp2 </italic> or <italic> Rpp3 </italic> . Amongst others, genes encoding WRKY, bHLH, and MYB TFs were activated in incompatible, but not compatible, soybean- <italic> P. pachyrhizi </italic> interactions. When <italic> <span class="gene">GmWRKY36,</span> <span class="gene">GmWRKY40,</span> <span class="gene">GmWRKY45</span> </italic> , and <italic> <span class="gene">GmMYB84</span> </italic> were individually silenced using VIGS, <italic> Rpp2 </italic> -mediated SBR <span class="trait">resistance</span> was gone ( <xref rid="B117" ref-type="bibr"> Pandey et al., 2011 </xref> ). Several other studies also revealed differential expression of TFs in incompatible or compatible soybean- <italic> P. pachyrhizi </italic> interactions ( <xref rid="B118" ref-type="bibr"> Panthee et al., 2009 </xref> ; <xref rid="B109" ref-type="bibr"> Morales et al., 2013 </xref> ; <xref rid="B2" ref-type="bibr"> Aoyagi et al., 2014 </xref> ). In fact, there seems to be considerable overlap of TF activity in <italic> Rpp2, Rpp3 </italic> , and <italic> Rpp4 </italic> -mediated soybean disease <span class="trait">resistance</span> ( <xref rid="B109" ref-type="bibr"> Morales et al., 2013 </xref> ). Therefore, these TFs seem to be excellent candidates for engineering SBR <span class="trait">resistance.</span> However, manipulation of TF balance may affect agronomic traits because TFs regulate a diverse array of loci.
					</p>
				
					<p>&nbsp;&nbsp;
						In another approach, <xref rid="B37" ref-type="bibr"> Cooper et al. (2011) </xref> compared nuclear proteome changes in a resistant vs. susceptible genotype at 24 h after inoculation with <italic> P. pachyrhizi </italic> . Their analysis disclosed more than 200 proteins that specifically accumulated in the nucleus of SBR-resistant soybean plants harboring <italic> Rpp1 </italic> ( <xref rid="B37" ref-type="bibr"> Cooper et al., 2011 </xref> ). Silencing two predicted soybean TFs <span class="gene">(Glyma14g11400,</span> PHD superfamily and <span class="gene">Glyma12g30600,</span> zinc finger TF) via VIGS partially compromised <italic> Rpp1 </italic> -conferred SBR <span class="trait">resistance</span> ( <xref rid="B38" ref-type="bibr"> Cooper et al., 2013 </xref> ). Similarly, <xref rid="B7" ref-type="bibr"> Bencke-Malato et al. (2014) </xref> demonstrated that accumulation of mRNA transcripts for several WRKY TFs was faster and more robust in a resistant than susceptible soybean accession. Consistently, the simultaneous silencing of four identified <italic> WRKY </italic> genes rendered soybean plants more susceptible to SBR disease. Because the authors did not succeed in producing <italic> WRKY </italic> -overexpressing soybean lines ( <xref rid="B7" ref-type="bibr"> Bencke-Malato et al., 2014 </xref> ), the potential of <italic> WRKY </italic> overexpression for providing SBR <span class="trait">resistance</span> to susceptible soybean genotypes remained unclear.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants can halt or slow down infection by constitutive or inducible accumulation of antimicrobial and/or cell wall-fortifying secondary metabolites ( <xref rid="B34" ref-type="bibr"> Chiang and Norris, 1983 </xref> ; <xref rid="B59" ref-type="bibr"> Hahlbrock and Scheel, 1989 </xref> ; <xref rid="B31" ref-type="bibr"> Chang et al., 1995 </xref> ; <xref rid="B42" ref-type="bibr"> Dixon et al., 2002 </xref> ; <xref rid="B12" ref-type="bibr"> Boerjan et al., 2003 </xref> ; <xref rid="B79" ref-type="bibr"> La Camera et al., 2004 </xref> ; <xref rid="B148" ref-type="bibr"> Vogt, 2010 </xref> ). Secondary metabolites also contribute to the outcome of the soybean&#x02013; <italic> P. pachyrhizi </italic> interaction. Daidzein, genistein, and glyceollin are isoflavonoids that accumulate in both resistant and susceptible soybean genotypes upon <italic> P. pachyrhizi </italic> infection ( <xref rid="B94" ref-type="bibr"> Lygin et al., 2009 </xref> ). Glyceollin efficiently reduces <italic> P. pachyrhizi </italic> uredospore germination <italic> in vitro </italic> ( <xref rid="B94" ref-type="bibr"> Lygin et al., 2009 </xref> ). Further evidence for a role of phytoalexins in SBR <span class="trait">resistance</span> was provided by <xref rid="B11" ref-type="bibr"> Bilgin et al. (2009) </xref> . The authors disclosed that SBR <span class="trait">resistance</span> in a <italic> Glycine tomentella </italic> accession correlated with the presence of a flavonoid that also inhibited <italic> P. pachyrhizi </italic> spore germination ( <xref rid="B36" ref-type="bibr"> Chung and Singh, 2008 </xref> ). The high potential of phytoalexins in defeating SBR is further supported by medicarpin accumulating in <italic> P. pachyrhizi </italic> -infected <italic> Medicago truncatula </italic> , a non-host of <italic> P. pachyrhizi </italic> . Consistently, medicarpin inhibits <italic> P. pachyrhizi </italic> spore germination ( <xref rid="B66" ref-type="bibr"> Ishiga et al., 2015 </xref> ). Providing such comparative large-scale metabolic profiles from resistant vs. susceptible soybean varieties, or other SBR-resistant species would likely identify more secondary metabolites inhibiting SBR. Genes in their biosynthesis pathways could be used to engineer SBR <span class="trait">resistance</span> in transgenic soybean. Alternatively, the compound(s) themselves could serve as natural fungicides in spray application, especially if they can be produced at low costs and in sufficient quantities for use in agriculture. In a variety of studies, genes in the phenylpropanoid and flavonoid metabolism were overrepresented when analyzing the transcriptional response of infected soybean genotypes with SBR <span class="trait">resistance</span> ( <xref rid="B146" ref-type="bibr"> van de Mortel et al., 2007 </xref> ; <xref rid="B35" ref-type="bibr"> Choi et al., 2008 </xref> ; <xref rid="B118" ref-type="bibr"> Panthee et al., 2009 </xref> ; <xref rid="B129" ref-type="bibr"> Schneider et al., 2011 </xref> ). Overall, activation of these genes was faster and stronger in SBR-resistant accessions than in susceptible ones ( <xref rid="B146" ref-type="bibr"> van de Mortel et al., 2007 </xref> ; <xref rid="B129" ref-type="bibr"> Schneider et al., 2011 </xref> ). Functional evidence for the importance of phenylpropanoid pathway genes in soybean&#x02019;s SBR <span class="trait">resistance</span> was provided by <xref rid="B117" ref-type="bibr"> Pandey et al. (2011) </xref> . The authors demonstrated that silencing of soybean phenylalanine ammonia-lyase ( <italic> <span class="gene">GmPAL</span> </italic> ) or <italic> O </italic> -methyl transferase1 ( <italic> <span class="gene">GmOMT1</span> </italic> ) compromised <italic> Rpp2 </italic> -mediated SBR <span class="trait">resistance.</span> <italic> OMT1 </italic> silencing also partially impaired <italic> Rpp1 </italic> -mediated SBR <span class="trait">resistance</span> ( <xref rid="B38" ref-type="bibr"> Cooper et al., 2013 </xref> ) and significantly decreased lignin content ( <xref rid="B117" ref-type="bibr"> Pandey et al., 2011 </xref> ). The latter result points to an important role of lignification in rejecting SBR.
					</p>
				
					<p>&nbsp;&nbsp;
						Another option for identifying soybean <italic> S </italic> genes to SBR is via sequence homology search to known <italic> S </italic> genes. Functional analysis can be done using, for example, soybean insertion mutants ( <xref rid="B99" ref-type="bibr"> Mathieu et al., 2009 </xref> ), performing VIGS ( <xref rid="B162" ref-type="bibr"> Zhang and Ghabrial, 2006 </xref> ; <xref rid="B161" ref-type="bibr"> Zhang et al., 2010 </xref> , <xref rid="B163" ref-type="bibr"> 2013 </xref> ; <xref rid="B117" ref-type="bibr"> Pandey et al., 2011 </xref> ), TILLING ( <xref rid="B39" ref-type="bibr"> Cooper et al., 2008 </xref> ), or applying targeted genome editing techniques such as CRISPR/Cas9 ( <xref rid="B68" ref-type="bibr"> Jacobs et al., 2015 </xref> ). However, currently only one gene [the Cys(2)His(2) zinc finger TF palmate-like pentafoliata1, <italic> <span class="gene">PALM1</span> </italic> ] that would classify as an SBR <italic> S </italic> gene is known from <italic> M. truncatula </italic> ( <xref rid="B145" ref-type="bibr"> Uppalapati et al., 2012 </xref> ). Alternatively, fungal effectors might serve as guides to identify novel <italic> S </italic> genes in soybean and other plants since several effectors of bacteria, fungi, or oomycetes were shown to target plant <italic> S </italic> genes (reviewed by <xref rid="B48" ref-type="bibr"> Gawehns et al., 2013 </xref> ). Although various analyses identified stage-specific <span class="trait">rust</span> proteins that might have bona-fide effector function ( <xref rid="B90" ref-type="bibr"> Loehrer and Schaffrath, 2011 </xref> ; <xref rid="B140" ref-type="bibr"> Stone et al., 2012 </xref> ; <xref rid="B87" ref-type="bibr"> Link et al., 2014 </xref> ), their role as virulence factors awaits functional confirmation. Identification of effector proteins and corresponding <italic> S </italic> gene targets was likely hampered by missing <italic> P. pachyrhizi </italic> genome information ( <xref rid="B91" ref-type="bibr"> Loehrer et al., 2014 </xref> ). Transformation protocols enabling generation of <italic> P. pachyrhizi </italic> knockout mutants are also missing.
					</p>
				
					<p>&nbsp;&nbsp;
						To identify more genes that function in <italic> Arabidopsis </italic> NHR to SBR, <xref rid="B81" ref-type="bibr"> Langenbach et al. (2016) </xref> searched for genes co-regulated with <italic> BRT1 </italic> . Upon confirming the genes&#x02019; importance in <italic> Arabidopsis </italic> postinvasion <span class="trait">resistance,</span> the authors expressed these genes in soybean. Four so-called postinvasion-induced NHR genes ( <italic> PINGs </italic> ) indeed reduced SBR disease severity. The supposed function of individual PING proteins is quite diverse and includes an EARLI4-like phospholipase (PING4), a group I receptor-like kinase (PING5), a GDSL-like lipase (PING7), and a germin-like protein (PING9). The exact mode of action of PINGs in conferring <span class="trait">resistance</span> to <italic> P. pachyrhizi </italic> has remained elusive ( <xref rid="B81" ref-type="bibr"> Langenbach et al., 2016 </xref> ). However, the study discloses that interspecies gene transfer is a promising strategy for conferring SBR <span class="trait">resistance</span> to soybean. Gene donor and receiver plant obviously do not need to be closely related, although it is likely that the successful transfer of a protein&#x02019;s function from one species to another implies conservation or convergence of its physiological environment (e.g., signaling networks). Thus, employing phylogenetically related non-hosts might further enhance the success of interspecies NHR gene transfer as a means for SBR <span class="trait">resistance.</span> Because <italic> P. pachyrhizi </italic> infects many plants, non-hosts to the fungus are rare, especially in the legume family of plants. <italic> M. truncatula </italic> is the only reported leguminous non-host as sporulation of <italic> P. pachyrhizi </italic> has not been observed on this plant ( <xref rid="B145" ref-type="bibr"> Uppalapati et al., 2012 </xref> ; <xref rid="B66" ref-type="bibr"> Ishiga et al., 2015 </xref> ). The former authors did a forward genetic screen to identify <italic> M. truncatula </italic> mutants with altered <span class="trait">resistance</span> to <italic> P. pachyrhizi </italic> . Because of its diploid genome, <italic> M. truncatula </italic> is better suited for forward genetic screening than the allopolyploid soybean crop ( <xref rid="B50" ref-type="bibr"> Gill et al., 2009 </xref> ). Furthermore, there is highly conserved microsynteny between soybean and <italic> M. truncatula </italic> ( <xref rid="B157" ref-type="bibr"> Yan et al., 2003 </xref> ). The screen by <xref rid="B145" ref-type="bibr"> Uppalapati et al. (2012) </xref> identified an <italic> inhibitor of <span class="trait">rust</span> germ tube differentiation </italic> ( <italic> irg </italic> ) <italic> 1 </italic> mutant on which <italic> P. pachyrhizi </italic> failed to promote preinfection structures. It turned out that the loss of abaxial epicuticular wax crystals and the reduced surface hydrophobicity inhibited fungal development on <italic> irg1 </italic> ( <xref rid="B145" ref-type="bibr"> Uppalapati et al., 2012 </xref> ). The mutation was mapped to <italic> <span class="gene">PALM1</span> </italic> encoding a Cys(2)His(2) zinc finger TF controlling the expression of genes involved in long-chain fatty acid biosynthesis and transport ( <xref rid="B145" ref-type="bibr"> Uppalapati et al., 2012 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular Characterization of <span class="trait">Resistance</span> to Soybean <span class="trait">Rust</span> ( <italic> Phakopsora pachyrhizi </italic> Syd. &#x00026; Syd.) in Soybean Cultivar DT 2000 (PI 635999)</h2>
			<h3>PMID:27935940</h3>
				
					<h3>PMCID:5147787</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Walker </surname>  <given-names> David R. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Binh T. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Tuyet T. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dinh </surname>  <given-names> Hoan X. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hyten </surname>  <given-names> David L. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sleper </surname>  <given-names> David A. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Jeong D. </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> James G. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 9 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Results of the selective genotyping analysis inspired us to <span class="gene">try</span> mapping the genes using phenotypic data from the entire RIL population evaluated at the two locations. Similar to the results of selective genotyping, two genomic locations significantly associated with variation in the AUDPC values, SBR severity, and sporulation from uredinia were detected and mapped on Chrs. 6 and 18 in the regions containing the <italic> Rpp3 </italic> and <italic> Rpp4 </italic> loci, respectively. Moreover, it allowed the 4.4-cM and 4.2-cM marker intervals identified by selective genotyping ( <xref ref-type="fig" rid="pone.0164493.g002"> Fig 2 </xref> ) to be narrowed to less than 0.1 cM ( <xref ref-type="fig" rid="pone.0164493.g003"> Fig 3 </xref> ). CIM also resulted in higher LOD scores and <italic> R </italic> <sup> <italic> 2 </italic> </sup> values ( <xref ref-type="table" rid="pone.0164493.t004"> Table 4 </xref> ). In addition to the BARC_SNPs, polymorphic SSR markers from the vicinities of the <italic> Rpp3 </italic> and <italic> Rpp4 </italic> loci were also included in the analysis. Genetic positions of these genetic markers were in an agreement with the consensus linkage map [ <xref rid="pone.0164493.ref054" ref-type="bibr"> 54 </xref> ]. In particular, SSR markers in this study and their genetic positions on Chrs. 6 and 18 were consistent with the results of previous studies to map <italic> Rpp </italic> genes [ <xref rid="pone.0164493.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0164493.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0164493.ref047" ref-type="bibr"> 47 </xref> , <xref rid="pone.0164493.ref053" ref-type="bibr"> 53 </xref> ]. Because of the nature of the pathogenicity of soybean <span class="trait">rust</span> and the host <span class="trait">resistance</span> in diverse soybean germplasm, it is desirable to monitor disease progress over time [ <xref rid="pone.0164493.ref005" ref-type="bibr"> 5 </xref> , <xref rid="pone.0164493.ref007" ref-type="bibr"> 7 </xref> ]. Moreover, it has been recommended that the employment of new sources with partial <span class="trait">resistance</span> in soybean breeding programs could potentially overcome the ineffectiveness of race-specific monogenic <span class="trait">resistance</span> [ <xref rid="pone.0164493.ref007" ref-type="bibr"> 7 </xref> , <xref rid="pone.0164493.ref055" ref-type="bibr"> 55 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Marker-Assisted Molecular Profiling, Deletion Mutant Analysis, and RNA-Seq Reveal a Disease <span class="trait">Resistance</span> Cluster Associated with <italic> Uromyces appendiculatus </italic> Infection in Common Bean <italic> Phaseolus vulgaris </italic> L.</h2>
			<h3>PMID:28545258</h3>
				
					<h3>PMCID:5485933</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Todd </surname>  <given-names> Antonette R. </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-01109"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Donofrio </surname>  <given-names> Nicole </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-18-01109"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sripathi </surname>  <given-names> Venkateswara R. </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-01109"> 1 </xref>  <xref ref-type="aff" rid="af3-ijms-18-01109"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClean </surname>  <given-names> Phillip E. </given-names>  </name>  <xref ref-type="aff" rid="af4-ijms-18-01109"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Rian K. </given-names>  </name>  <xref ref-type="aff" rid="af4-ijms-18-01109"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pastor-Corrales </surname>  <given-names> Marcial </given-names>  </name>  <xref ref-type="aff" rid="af5-ijms-18-01109"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kalavacharla </surname>  <given-names> Venu (Kal) </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-18-01109"> 1 </xref>  <xref ref-type="aff" rid="af6-ijms-18-01109"> 6 </xref>  <xref rid="c1-ijms-18-01109" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plant-pathogen interactions have been a necessary study to understand the relationships and mechanisms that drive disease <span class="trait">resistance</span> [ <xref rid="B34-ijms-18-01109" ref-type="bibr"> 34 </xref> ]. For more than a hundred years, people have used methods such as breeding to <span class="gene">try</span> to control immune receptors in plants [ <xref rid="B35-ijms-18-01109" ref-type="bibr"> 35 </xref> ]. The number of cloned <italic> R </italic> genes has greatly increased over the last 20 years. As more plant <italic> R </italic> genes are isolated and cloned, there is new promise that scientists may obtain a better understanding of pathogenicity and gene function, which can ultimately be used to create more resistant plant varieties. The identification and isolation of <italic> R </italic> genes is advantageous, as developing disease resistant varieties in plants may be a more suitable approach than the use of pesticides to manage virulent pathogens on crops. Since the long-term effects of pesticide use are not completely known, many consumers now opt for organically raised crops. While humans have been purposely studying <italic> R </italic> genes for a hundred years, pathogens have been evolving along with their hosts for much longer [ <xref rid="B35-ijms-18-01109" ref-type="bibr"> 35 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Disease <span class="trait">resistance</span> genes usually contain signature motifs that are recognizable as such. It is understood that NBS, LRR, and <span class="gene">TIR</span> domains are normally present in <italic> R </italic> genes, making them easy to identify in a group of amino acid sequences. However, recent work by two groups, [ <xref rid="B36-ijms-18-01109" ref-type="bibr"> 36 </xref> , <xref rid="B37-ijms-18-01109" ref-type="bibr"> 37 </xref> ], reveals unusual domains anchoring themselves to nucleotide binding oligomerization domain (NOD)-like receptors (NLRs) [ <xref rid="B15-ijms-18-01109" ref-type="bibr"> 15 </xref> ]. One of these unusual domains is the WRKY-like or WRKY transcription factor-like domain that is fused to the <italic> RPS1 </italic> gene in <italic> Arabidopsis </italic> . The RPS1 and RPS4 complex is paired with the WRKY domain to provide recognition of two bacterial effectors. However, when the WRKY-like domain anchors itself to this complex instead, the bacterial effectors never interact with the true target. Therefore, instead of pathogen effectors being intercepted by true WRKY domains, they are intercepted by an imposter. Hence the WRKY-like domain attracts the pathogen effectors and suppresses the immune response [ <xref rid="B36-ijms-18-01109" ref-type="bibr"> 36 </xref> , <xref rid="B37-ijms-18-01109" ref-type="bibr"> 37 </xref> ]. Current plant disease perspectives seek to use the pathogen effectors as a means to exploit <italic> R </italic> gene mediated <span class="trait">resistance</span> [ <xref rid="B38-ijms-18-01109" ref-type="bibr"> 38 </xref> ]. Initially, pathogen effectors were believed to directly interact with host <italic> R </italic> genes. It is now understood that the primary role of pathogen effectors is to alter the host&#x02019;s cellular function and to create a better environment for itself [ <xref rid="B39-ijms-18-01109" ref-type="bibr"> 39 </xref> ]. The hemibiotrophic oomycete pathogen <italic> Phytophthora infestans </italic> secretes the effector protein AVR3a into the host. There are two forms of AVR3a, which are AVR3aKI and AVR3aEM. AVR3aKI strongly suppresses infestin 1 (INF1) induced cell death (ICD) in the potato interacting with the R3a protein. AVR3aEM weakly suppresses INF1 ICD when interacting with R3a [ <xref rid="B40-ijms-18-01109" ref-type="bibr"> 40 </xref> ]. This interaction suppresses HR by recognition of the R3a protein in the host by AVR3a [ <xref rid="B41-ijms-18-01109" ref-type="bibr"> 41 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>