<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Salt resistance</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Salt resistance</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Identification and analysis of the germin-like gene family in soybean</h2>
			<h3>PMID:21059215</h3>
				
					<h3>PMCID:3011853</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Lu </surname>  <given-names> Mo </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> xiaomo521@msn.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Han </surname>  <given-names> Ying-Peng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> hyp234286@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Gao </surname>  <given-names> Ji-Guo </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> gaojiguo1961@hotmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Wang </surname>  <given-names> Xiang-Jing </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> wangxiangjing2008@yahoo.com.cn </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A5">  <name>  <surname> Li </surname>  <given-names> Wen-Bin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wenbinli@neau.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 8 </day>  <month> 11 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the present study, twenty-one germin gene members in soybean cultivar 'Maple Arrow' (partial <span class="trait">resistance</span> to <italic> Sclerotinia </italic> stem rot of soybean) were identified by in silico identification and RACE method <span class="gene">(GmGER</span> 1 to <span class="gene">GmGER</span> 21). A genome-wide analyses of these germin-like protein genes using a bioinformatics approach showed that the genes located on chromosomes 8, 1, 15, 20, 16, 19, 7, 3 and 10, on which more disease-resistant genes were located on. Sequence comparison revealed that the genes encoded three germin-like domains. The phylogenetic relationships and functional diversity of the germin gene family of soybean were analyzed among diverse genera. The expression of the <span class="gene">GmGER</span> genes treated with exogenous IAA suggested that <span class="gene">GmGER</span> genes might be regulated by auxin. Transgenic tobacco that expressed the <span class="gene">GmGER</span> 9 gene exhibited high tolerance to the <span class="trait">salt</span> stress. In addition, the <span class="gene">GmGER</span> mRNA increased transiently at darkness and peaked at a time that corresponded approximately to the critical night length. The mRNA did not accumulate significantly under the constant light condition, and did not change greatly under the SD and LD treatments.
					</p>
				
					<p>&nbsp;&nbsp;
						This study provides a complex overview of the <span class="gene">GmGER</span> genes in soybean. Phylogenetic analysis suggested that the germin and germin-like genes of the plant species that had been founded might be evolved by independent gene duplication events. The experiment indicated that germin genes exhibited diverse expression patterns during soybean development. The different time courses of the mRNAs accumulation of <span class="gene">GmGER</span> genes in soybean leaves appeared to have a regular photoperiodic reaction in darkness. Also the <span class="gene">GmGER</span> genes were proved to response to abiotic stress (such as auxin and <span class="trait">salt),</span> suggesting that these paralogous genes were likely involved in complex biological processes in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast to the advanced knowledge of the structure, cell biology and expression features of barley and wheat germins and GLPs, less was known about soybean germin and germin-like genes by 2010. In the present work, nearly 50,000 sequenced and annotated ESTs of soybean were analyzed to find <span class="gene">GmGER-like</span> sequences by amino acid sequence similarity, and <span class="gene">GmGER-like</span> sequences were elongated by RACE. Their locations were determined and compared to disease <span class="trait">resistance</span> QTL, and both cluster and phylogenetic analyses of the germin-like proteins were performed to describe the variations in the soybean gene family. The abiotic factors (auxin-IAA, <span class="trait">salt,</span> light treatments) was tested on <span class="gene">GmGER</span> genes to evaluate the complex biological processes related to soybean development.
					</p>
				
					<p>&nbsp;&nbsp;
						The common structural features of germin-like proteins included: conserved structural elements, secretory transit peptides, protein glycosylation sites and regions of conserved sequence similarity [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. According to the review of Carter and Thornburg [ <xref ref-type="bibr" rid="B36"> 36 </xref> ], the most important conserved structural feature among the GLPs is a conserved amino acid sequence termed the germin box. The consensus pattern is: GxxxxHxHPxAxEh, where x is any amino acid and h is a hydrophobic amino acid. Mature germins and GLPs all contain approximately 200 amino acids in length and the germin box occurs near the middle for all proteins in this family. In soybean, the conserved region B was found in all the 21 germin genes. Conserved region A enriched in hydrophobic amino acids always followed this peptide (GxxxxHxHPxAxEh) [Figure <xref ref-type="fig" rid="F2"> 2 </xref> ]. Another conserved region A was localized in the amino terminal part of the mature proteins, and strongly conserved in all the germins but not in the spherulins. It is likely that the heptapeptide sequence [L/V]QDFCV[A/G], found in all germin-like proteins examined, was of importance in the biochemical functions of these proteins. A third putative conserved region C underlined in Figure <xref ref-type="fig" rid="F2"> 2 </xref> was [V/M][F/K]P[Q/K/I]G[L][V/I/L]HFQ[K/L/Q/I]N[V/N/I]G. Two His residues and a Glu residue in Motif A, together with a His residue in Motif C, act as ligands for the binding of a manganese ion at the active site of the archetypal germin, cupin [ <xref ref-type="bibr" rid="B42"> 42 </xref> ]. In our result, motif A was HTHPRATEILTVLEGTLYVGF with 21 amino acids, motif B was KVLNKGDVFVFPEGLIHFQFN with 21 amino acids, and motif C was [N/S]SQNPGIVFVPLTLFG with 16 amino acids. The 21 <span class="gene">GmGER</span> genes all have the germin activity, <bold> e </bold> xhibiting diverse expression patterns during soybean development, a regular photoperiodical reaction in darkness, response to abiotic stress (such as auxin and <span class="trait">salt),</span> as we have proved in this paper.
					</p>
				
					<p>&nbsp;&nbsp;
						Germin and the GLP gene family could be divided into two distinct groups. The members in one group (germins) had relatively homogeneous sequences [ <xref ref-type="bibr" rid="B44"> 44 </xref> ], meanwhile, the members in another group (GLPs) were much more numerous and showed high sequence divergence [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. GLPs could be further divided into three subgroups based upon sequence conservation [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. In this paper, the <span class="gene">GmGER</span> genes were also divided into three subgroups based upon sequence conservation. Systematic investigation of soybean germin gene family would be useful for defining origins of the germin and germin-like gene family. Although the functional significance of all these elements remained to be tested, a variety of regulatory mechanisms acting on the transcript abundance were found. The germin genes in rice have been confirmed to be expressed in all types of tissues and could be induced by biotic or abiotic stresses. Many of the stress-induced germin genes were physically co-localized with quantitative trait loci (QTL) for disease <span class="trait">resistance,</span> and the emerged evidence suggested that microRNAs might regulate their transcript abundances [ <xref ref-type="bibr" rid="B46"> 46 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]. Oxalate oxidase could confer to enhance <span class="trait">resistance</span> to <italic> Sclerotinia </italic> blight in peanut [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. In transformed sunflower, the expression of oxalate oxidase resulted in the induction of plant defense proteins [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. In tissue sections derived from pea nodules, PsGER1 was shown to be the first known germin-like protein with superoxide dismutase activity [ <xref ref-type="bibr" rid="B49"> 49 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A significant promotion of the expression of <span class="gene">GmGER</span> mRNA was achieved by quantitative real-time RT-PCR under the supplement of IAA in medium. As IAA is involved mainly in the regulation of cell elongation and stimulating cell division [ <xref ref-type="bibr" rid="B50"> 50 </xref> ], this result suggested that <span class="gene">GmGER</span> genes might mediate the stimulating effect of auxin on cell division. It should be noticed that, although the <span class="gene">GmGER</span> gene responded to IAA, the affinity was unknown, which made it difficult to be certain that the <span class="gene">GmGER</span> gene was involved in auxin signaling, especially without additional physiologic evidence. Therefore, it should be investigated furthermore. Meanwhile, the assay of <span class="trait">salt</span> stress indicated that the transgenic seedlings of tobacco with <span class="gene">GmGER</span> 9 gene showed improved <span class="trait">salt</span> tolerance compared to the WT plants (Figure <xref ref-type="fig" rid="F7"> 7A, 7B, 7C, 7D, 7E </xref> and <xref ref-type="fig" rid="F7"> 7F </xref> ), confirming that the <span class="gene">GmGER</span> 9 gene had a positive response to the <span class="trait">salt</span> condition. <span class="trait">Salt</span> stress could cause oxidative damage in plant, such as protein oxidation and lipid peroxidation [ <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Therefore, <span class="gene">GmGER</span> 9 gene remained a potential to improve the <span class="trait">salt</span> tolerance in transgenic plants.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A putative soybean <span class="gene">GmsSOS1</span> confers enhanced <span class="trait">salt</span> tolerance to transgenic Arabidopsis sos1-1 mutant.</h2>
			<h3>PMID:24934653  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nie WX(1), Xu L, Yu BJ.
				<br/> 
				年份：2015 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cDNA of <span class="gene">GmsSOS1,</span> a putative plasma membrane Na(+)/H(+) antiporter gene isolated from Glycine max, Glycine soja, and their hybrid, was constructed into plant expression vector pCAMBIA 1300 and then transformed with Agrobacterium tumefaciens under the control of CaMV 35S promoter to Arabidopsis thaliana wild-type (WT) and mutant (atsos1-1) plants. By hygromycin <span class="trait">resistance</span> detection and PCR analysis, transgenic plants (WT35S:GmsSOS1 and atsos1-1 35S:GmsSOS1) were obtained. Seed germination, seedling growth, and Na(+) contents in roots and shoots were analytically compared among WT, atsos1-1 mutant, and their transgenic lines under <span class="trait">salt</span> stress. The results showed that when <span class="gene">GmsSOS1</span> was integrated into the genome of A. thaliana, the inhibitions of <span class="trait">salt</span> stress on seed germination and seedling growth were all significantly improved, and enhanced <span class="trait">salt</span> tolerance was displayed, which may be attributed to the decrease of Na(+) absorption in roots and transportation in shoots of the transgenic lines, especially for that of atsos1-1 mutant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmERF5,</span> a new member of the soybean EAR motif-containing ERF transcription factor, enhances <span class="trait">resistance</span> to Phytophthora sojae in soybean.</h2>
			<h3>PMID:25779701  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dong L(1), Cheng Y(1), Wu J(2), Cheng Q(1), Li W(1), Fan S(1), Jiang L(1), Xu Z(3), Kong F(4), Zhang D(3), Xu P(5), Zhang S(5).
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytophthora root and stem rot of soybean [Glycine max (L.) Merr.], caused by Phytophthora sojae Kaufmann and Gerdemann, is a destructive disease throughout the soybean planting regions in the world. Here, we report insights into the function and underlying mechanisms of a novel ethylene response factor (ERF) in soybean, namely <span class="gene">GmERF5,</span> in host responses to P. sojae. <span class="gene">GmERF5-overexpressing</span> transgenic soybean exhibited significantly enhanced <span class="trait">resistance</span> to P. sojae and positively regulated the expression of the PR10, PR1-1, and PR10-1 genes. Sequence analysis suggested that <span class="gene">GmERF5</span> contains an AP2/ERF domain of 58 aa and a conserved ERF-associated amphiphilic repression (EAR) motif in its C-terminal region. Following stress treatments, <span class="gene">GmERF5</span> was significantly induced by P. sojae, ethylene (ET), abscisic acid (ABA), and salicylic acid (SA). The activity of the <span class="gene">GmERF5</span> promoter <span class="gene">(GmERF5P)</span> was upregulated in tobacco leaves with ET, ABA, Phytophthora nicotianae, <span class="trait">salt,</span> and <span class="trait">drought</span> treatments, suggesting that <span class="gene">GmERF5</span> could be involved not only in the induced defence response but also in the ABA-mediated pathway of <span class="trait">salt</span> and <span class="trait">drought</span> tolerance. <span class="gene">GmERF5</span> could bind to the GCC-box element and act as a repressor of gene transcription. It was targeted to the nucleus when transiently expressed in Arabidopsis protoplasts. <span class="gene">GmERF5</span> interacted with a basic helix-loop-helix transcription factor <span class="gene">(GmbHLH)</span> and eukaryotic translation initiation factor <span class="gene">(GmEIF)</span> both in yeast cells and in planta. To the best of our knowledge, <span class="gene">GmERF5</span> is the first soybean EAR motif-containing ERF transcription factor demonstrated to be involved in the response to pathogen infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A physical map of a gene-dense region in soybean linkage group A2 near the black seed coat and Rhg (4) loci.</h2>
			<h3>PMID:12582695</h3>
				

				
			<p>
				作者：Lewers K(1), Heinz R, Beard H, Marek L, Matthews B.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( Glycine max L. Merrill) linkage group A2 contains a major <span class="trait">resistance</span> gene to the soybean cyst nematode ( Heterodera glycines Ichinohe) at the Rhg (4) locus near a gene encoding aspartokinase homoserine dehydrogenase (AK-HSDH) and also near the I locus affecting seed coat color. To identify clones related to this region of the genome, we used a PCR assay using primers designed from a gene encoding <span class="gene">AK-HSDH</span> to screen approximately 40,000 clones from a bacterial artificial chromosome (BAC) library constructed from genomic DNA of the susceptible cv. Williams 82. The identified BACs were screened with a second PCR assay using primers designed from DNA sequence associated with the I locus to confirm the location of the BACs. Only BAC <span class="gene">Gm_ISb001_056_G02</span> (56G2) was positive for both assays. BAC 56G2 contains several genes previously associated with stress or defense response including genes with high sequence similarity to those encoding chalcone synthase, glucosyl-transferase, a heat-shock transcription factor, a membrane-associated salt-inducible protein, adenosyl homocysteinase, a protein kinase, and a G10-like protein. The map contributes to our understanding of the organization of the soybean genome and to the completion of a physical map of the soybean genome. In addition, the genes identified provide landmarks to identify BAC clones near the Rhg (4) locus in resistant soybean genomic libraries and provide a foundation for comparison of soybean cyst nematode resistant and -susceptible DNA sequences in this region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of the <span class="gene">RAV</span> Family in Soybean and Functional Identification of <italic> <span class="gene">GmRAV-03</span> </italic> Involvement in <span class="trait">Salt</span> and <span class="trait">Drought</span> Stresses and Exogenous ABA Treatment</h2>
			<h3>PMID:28634481</h3>
				

				
					<h3>PMCID:5459925</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Shu-Ping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/442690/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao-Shi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/203437/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Wei-Jun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Wan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan-Xia </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Tai-Fei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/394729/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Yong-Bin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Min </surname>  <given-names> Dong-Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> You-Zhi </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Shou-Cheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiao-Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/418301/overview"/>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcription factors play vital roles in plant growth and in plant responses to abiotic stresses. The <span class="gene">RAV</span> transcription factors contain a B3 DNA binding domain and/or an APETALA2 (AP2) DNA binding domain. Although genome-wide analyses of <span class="gene">RAV</span> family genes have been performed in several species, little is known about the family in soybean (Glycine max L.). In this study, a total of 13 <span class="gene">RAV</span> genes, named as <span class="gene">GmRAVs,</span> were identified in the soybean genome. We predicted and analyzed the amino acid compositions, phylogenetic relationships, and folding states of conserved domain sequences of soybean <span class="gene">RAV</span> transcription factors. These soybean <span class="gene">RAV</span> transcription factors were phylogenetically clustered into three classes based on their amino acid sequences. Subcellular localization analysis revealed that the soybean <span class="gene">RAV</span> proteins were located in the nucleus. The expression patterns of 13 <span class="gene">RAV</span> genes were analyzed by quantitative real-time PCR. Under <span class="trait">drought</span> stresses, the <span class="gene">RAV</span> genes expressed diversely, up- or down-regulated. Following NaCl treatments, all <span class="gene">RAV</span> genes were down-regulated excepting <span class="gene">GmRAV-03</span> which was up-regulated. Under abscisic acid (ABA) treatment, the expression of all of the soybean <span class="gene">RAV</span> genes increased dramatically. These results suggested that the soybean <span class="gene">RAV</span> genes may be involved in diverse signaling pathways and may be responsive to abiotic stresses and exogenous ABA. Further analysis indicated that <span class="gene">GmRAV-03</span> could increase the transgenic lines <span class="trait">resistance</span> to high <span class="trait">salt</span> and <span class="trait">drought</span> and result in the transgenic plants insensitive to exogenous ABA. This present study provides valuable information for understanding the classification and putative functions of the <span class="gene">RAV</span> transcription factors in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> L.) is an economically important crop that is used in human diets, animal feeds and for biodiesel production. However, soybean growth and productivity are greatly affected by environmental stresses such as <span class="trait">drought</span> and high soil salinity ( <xref rid="B18" ref-type="bibr"> Kang et al., 2009 </xref> ; <xref rid="B19" ref-type="bibr"> Kidokoro et al., 2015 </xref> ). Plants can resist abiotic stresses by regulating the expression of a variety of genes. Under stress conditions, transcriptome changes are the earliest responses before biosynthesis, modification, and interaction of related expression proteins in plants ( <xref rid="B50" ref-type="bibr"> Yamaguchi-Shinozaki and Shinozaki, 2006 </xref> ). The expression of genes encoding transcription factors are induced or repressed and these function to further regulate plant responses to environmental stresses ( <xref rid="B13" ref-type="bibr"> Hao and O&#x02019;Shea, 2012 </xref> ). Many transcription factors involved in plant stress-resistance pathways have been identified, among which the &#x02018;related to ABI3/VP1 (RAV)&#x02019; family of transcription factors are one of them. The <span class="gene">RAV</span> family is part of the B3 superfamily, which also contains the ARF, LAV, and REM families. The B3 superfamily is defined by the B3 domain, which consists of about 110 amino acids ( <xref rid="B45" ref-type="bibr"> Swaminathan et al., 2008 </xref> ). The <span class="gene">RAV</span> family proteins contain a B3 domain and/or an <span class="gene">AP2</span> domain. Thus, the <span class="gene">RAV</span> family members could reasonably be classified as members of either the B3 superfamily or the AP2/EREBP family ( <xref rid="B37" ref-type="bibr"> Mat&#x000ed;as-Hern&#x000e1;ndez et al., 2014 </xref> ). The <span class="gene">AP2</span> domain as a DNA-binding domain was first identified in the <italic> Arabidopsis </italic> ( <italic> Arabidopsis thaliana </italic> ) <span class="gene">AP2</span> protein ( <xref rid="B16" ref-type="bibr"> Jofuku et al., 1994 </xref> ). The B3 domain is a DNA-binding domain ( <xref rid="B8" ref-type="bibr"> Finkelstein et al., 1998 </xref> ) that was initially named due to its position in the third basic domain of the maize gene <italic> VIVIPAROUS1 </italic> ( <italic> VP1 </italic> ) ( <xref rid="B38" ref-type="bibr"> McCarty et al., 1991 </xref> ). Unlike the first and second basic domains (B1 and B2) which are specific to the VP1-like proteins, the B3 domain is widespread in plant genomes ( <xref rid="B44" ref-type="bibr"> Suzuki et al., 1997 </xref> ). B3 domains consist of seven &#x003b2;-barrels and two short &#x003b1;-helices ( <xref rid="B51" ref-type="bibr"> Yamasaki et al., 2004 </xref> ; <xref rid="B46" ref-type="bibr"> Waltner et al., 2005 </xref> ). <italic> Arabidopsis </italic> RAV1 and RAV2, which each contain both a B3 domain and an <span class="gene">AP2</span> domain, were the first identified members of the <span class="gene">RAV</span> family ( <xref rid="B17" ref-type="bibr"> Kagaya et al., 1999 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						It was reported that <span class="gene">RAV</span> transcription factors function in the regulation of plant responses to plant pathogens. Tomato <span class="gene">RAV</span> transcription factors may act as intermediate transcription factors that somehow connect <italic> AtCBF1 </italic> and pathogenesis-related genes, conferring enhanced tolerance to bacterial wilt ( <xref rid="B27" ref-type="bibr"> Li et al., 2011 </xref> ). Overexpression of the pepper <italic> CARAV1 </italic> gene in transgenic <italic> Arabidopsis </italic> plants was demonstrated to induce the expression of some pathogenesis-related genes, and to confer <span class="trait">resistance</span> to <italic> Pseudomonas syringae </italic> pv. <italic> tomato </italic> DC3000 to confer tolerance to osmotic stress ( <xref rid="B25" ref-type="bibr"> Lee and Hwang, 2006 </xref> ; <xref rid="B43" ref-type="bibr"> Sohn et al., 2006 </xref> ). Therefore, it is clear that the <span class="gene">RAV</span> family members mediate plant growth and developmental process and that these proteins are responsive to diverse hormone/pathogenic bacteria stimuli.
					</p>
				
					<p>&nbsp;&nbsp;
						Meanwhile, the expression levels of several <span class="gene">RAV</span> family members were affected by various plant hormones. <span class="gene">RAV</span> proteins are known to be involved in ethylene and brassinosteroid responses ( <xref rid="B2" ref-type="bibr"> Alonso et al., 2003 </xref> ; <xref rid="B15" ref-type="bibr"> Hu et al., 2004 </xref> ). In <italic> Arabidopsis TEMPRANILLO </italic> (TEM) genes repressed the expression of GA4 biosynthetic genes, resulting in retarded-growth phenotypes of transgenic plants ( <xref rid="B40" ref-type="bibr"> Osnato et al., 2012 </xref> ). <italic> RAV1 </italic> ( <xref rid="B15" ref-type="bibr"> Hu et al., 2004 </xref> ) and AT3G25730 ( <italic> RAV1L) </italic> in <italic> Arabidopsis </italic> were down-regulated by 24-epibrassinolide (epiBL) and ABA, respectively. And the <italic> RAV1 </italic> -overexpressing transgenic <italic> Arabidopsis </italic> were insensitive to ABA ( <xref rid="B9" ref-type="bibr"> Fu et al., 2014 </xref> ). ABA as the important one of plant hormones plays vital roles in controlling plant abiotic stress responses. The production of ABA can be triggered by <span class="trait">drought</span> and high salinity, and ABA activates some drought-inducible genes ( <xref rid="B52" ref-type="bibr"> Yang et al., 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Nevertheless, to date, little has been reported regarding the function(s) <span class="gene">RAV</span> proteins in plant responses to abiotic stress. In cotton ( <italic> Gossypium hirsutum </italic> ), overexpression AtRAV1/2 could improve the <span class="trait">drought</span> <span class="trait">resistance</span> of cotton ( <xref rid="B37" ref-type="bibr"> Mat&#x000ed;as-Hern&#x000e1;ndez et al., 2014 </xref> ). The pepper <italic> CARAV1 </italic> which was induced by abiotic stresses functioned as a transcriptional activator triggering tolerance to osmotic stresses ( <xref rid="B43" ref-type="bibr"> Sohn et al., 2006 </xref> ). <xref rid="B24" ref-type="bibr"> Lee et al. (2010) </xref> also reported that the overexpression <italic> Arabidopsis </italic> of pepper <italic> CARAV1 </italic> increased tolerance to high salinity stresses. These results indicate that <span class="gene">RAV</span> proteins may play roles in plant responses to various abiotic stresses. Given the potential importance of <span class="gene">RAV</span> genes in plant responses to abiotic stress environments, we carried out a genome-wide analysis of the soybean <italic> <span class="gene">RAV</span> </italic> gene family and investigated the potential functions of <italic> <span class="gene">RAV</span> </italic> genes in plant responses to various experimental stimuli.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmRAV-03</span> </italic> gene, which is induced by the <span class="trait">drought,</span> <span class="trait">salt,</span> and exogenous ABA, was selected to confirm gene functions. Fragment of <italic> <span class="gene">GmRAV-03</span> </italic> was ligated into the pCAMBIA1302 vector under control of the CaMV 35S promoter using the primers in Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S1 </xref> . The expression vector pCAMBIA1302::GmRAV-03 was transformed into the <italic> Agrobacterium tumefaciens </italic> (GV3101), followed by <italic> Arabidopsis </italic> transformation using the floral dipping method ( <xref rid="B5" ref-type="bibr"> Clough and Bent, 1998 </xref> ; <xref rid="B9" ref-type="bibr"> Fu et al., 2014 </xref> ). Transgenic <italic> Arabidopsis </italic> lines were selected using <span class="gene">MS</span> medium (Supplementary Table <xref ref-type="supplementary-material" rid="SM5"> S2 </xref> ) containing 30 mg/L hygromycin. Further experiments were performed with homozygous lines in T3 generation. And based on the expression levels of various lines containing four transgenic lines and wild line, we selected OE-2 and OE-3 for further experiments ( <bold> Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Abiotic stresses and phytohormones are known to affect a great number of physiological processes. To investigate the potential functions of the soybean <italic> <span class="gene">RAV</span> </italic> genes in response to different stimuli, the expression patterns of these genes in plants that had been treated with ABA or grown in <span class="trait">drought</span> and <span class="trait">salt</span> stress conditions were analyzed by qRT-PCR ( <bold> Figure <xref ref-type="fig" rid="F7"> 7 </xref> </bold> ). Under <span class="trait">drought</span> stress, two genes were obviously up-regulated (&#x0003e;2-fold); others were down-regulated slightly or obviously ( <bold> Figure <xref ref-type="fig" rid="F7"> 7A </xref> </bold> ). Following <span class="trait">salt</span> treatment, all genes, excepting <italic> <span class="gene">GmRAV-03</span> </italic> which was up-regulated, were down-regulated ( <bold> Figure <xref ref-type="fig" rid="F7"> 7B </xref> </bold> ). Exogenous ABA also affected the transcription of the soybean <italic> <span class="gene">RAV</span> </italic> gene. Under ABA treatment, the expression of all of the soybean <italic> <span class="gene">RAV</span> </italic> genes increased markedly (&#x0003e;2-fold). In particular, the expression of <italic> <span class="gene">GmRAV-01</span> </italic> , <italic> <span class="gene">GmRAV-03</span> </italic> , and <italic> <span class="gene">GmRAV-13</span> </italic> reached levels that were more than 10 times higher than the control ( <bold> Figure <xref ref-type="fig" rid="F7"> 7C </xref> </bold> ). Therefore, soybean <span class="gene">RAV</span> genes may be involved in ABA signal transduction pathways in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Relative expression of soybean <italic> <span class="gene">RAV</span> </italic> genes in plants treated with <span class="trait">drought,</span> NaCl, and exogenous application of ABA. qRT-PCR analyses of plants treated with <span class="trait">drought</span> <bold> (A) </bold> , NaCl <bold> (B), </bold> and ABA <bold> (C) </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the role of <italic> <span class="gene">GmRAV-03</span> </italic> in plant growth and development under high salinity, <span class="trait">salt</span> tolerance of the transgenic lines was also tested. For germination arrays, seeds of the transgenic lines and WT germinated on 1/2 <span class="gene">MS</span> medium containing various concentrations of NaCl, and the germination rates of both WT and transgenic seeds were determined. When germinated on 1/2 <span class="gene">MS</span> medium, all lines showed similar germination rate ( <bold> Figure <xref ref-type="fig" rid="F10"> 10A </xref> </bold> ). However, in the presence of NaCl, the germination of both WT and <italic> <span class="gene">GmRAV-03</span> </italic> overexpression transgenic seeds was inhibited, but the degree of inhibition in the WT was much greater than that of transgenic seeds. The ultimate germination rate of <italic> <span class="gene">GmRAV-03</span> </italic> transgenic seeds (especially OE-2) was a litter higher than that of WT ( <bold> Figure <xref ref-type="fig" rid="F10"> 10B </xref> </bold> ). With 100 mM NaCl treatment, the results were similar to that with 80 mM NaCl treatment ( <bold> Figure <xref ref-type="fig" rid="F10"> 10C </xref> </bold> and Supplementary Table <xref ref-type="supplementary-material" rid="SM8"> S5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						For phenotype comparison, seedlings grown on 1/2 <span class="gene">MS</span> medium for 5 days were transferred to 1/2 <span class="gene">MS</span> medium with or without NaCl. All lines showed similar phenotypes when grown on 1/2 <span class="gene">MS</span> medium. With 150 mM NaCl treatment for 10 days, the cotyledon of WT became yellow, even die, while the cotyledon of transgenic <italic> Arabidopsis </italic> still kept green ( <bold> Figure <xref ref-type="fig" rid="F10"> 10D </xref> </bold> ). Similarly, the root length and the root surface of <italic> <span class="gene">GmRAV-03</span> </italic> overexpression transgenic seedlings were longer than that of WT with 150 mM NaCl treatment ( <bold> Figures <xref ref-type="fig" rid="F10"> 10E,F </xref> </bold> and Supplementary Table <xref ref-type="supplementary-material" rid="SM7"> S4 </xref> ). Moreover, in seedling stage, survival rate, and chlorophyll content of the transgenic plants were determined and measured using WT as control, respectively. The results revealed that both survival rate and chlorophyll content of transgenic plants were higher than those of WT ( <bold> Figure <xref ref-type="fig" rid="F11"> 11 </xref> </bold> ). Collectively, the results suggested that <italic> <span class="gene">GmRAV-03</span> </italic> may participate in plant response to high salinity, and increase the transgenic plants tolerance to <span class="trait">salt</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the role of <italic> <span class="gene">GmRAV-03</span> </italic> in <span class="trait">drought</span> stresses, we observed the phenotype of overexpression <italic> <span class="gene">GmRAV-03</span> </italic> transgenic <italic> Arabidopsis </italic> seedlings grown on medium and soil under PEG and <span class="trait">drought</span> treatment, respectively. In phenotype comparison, all lines showed similar phenotypes when grown on 1/2 <span class="gene">MS</span> medium. However, with 8% PEG for 2 weeks, the phenotypes of wild and transgenic <italic> Arabidopsis </italic> showed obvious difference that the root length of transgenic lines was longer than that of wild lines especially OE-3 line ( <bold> Figures <xref ref-type="fig" rid="F12"> 12A,B </xref> </bold> and Supplementary Table <xref ref-type="supplementary-material" rid="SM7"> S4 </xref> ). On the other hand, the total root surface of transgenic lines was also higher than that of wild lines ( <bold> Figure <xref ref-type="fig" rid="F12"> 12C </xref> </bold> ). And the seedlings grown in soil also showed the <span class="trait">resistance</span> for <span class="trait">drought</span> ( <bold> Figure <xref ref-type="fig" rid="F13"> 13A </xref> </bold> ). The survival rate of transgenic lines was higher than that of wild lines ( <bold> Figure <xref ref-type="fig" rid="F13"> 13B </xref> </bold> ). Given these, <italic> <span class="gene">GmRAV-03</span> </italic> may have the potential function to increase the transgenic plants tolerance to <span class="trait">drought</span> stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> tolerances in <italic> Arabidopsis </italic> overexpressing <italic> <span class="gene">GmRAV-03</span> </italic> and wild type. <bold> (A) </bold> Performance of WT and <italic> <span class="gene">GmRAV-03</span> </italic> -overexpressing lines OE2 and OE3 before and after <span class="trait">drought</span> stress at the seedling stage. <bold> (B) </bold> Survival rate of WT and <italic> <span class="gene">GmRAV-03</span> </italic> -overexpressing lines after <span class="trait">drought</span> stress ( <italic> n </italic> = 3). The data are shown as mean &#x000b1; SD ( <italic> n </italic> = 45). Independent <italic> t </italic> -tests demonstrated that there was significant difference ( <sup> &#x02217; </sup> <italic> P </italic> &#x0003c; 0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Cis- </italic> elements in the promoters of genes play very important roles in the regulation of plant growth, development, and interaction with the environment. We analyzed the <italic> cis </italic> -acting elements in promoter regions of the <italic> <span class="gene">RAV</span> </italic> genes. This analysis revealed that the soybean <italic> <span class="gene">RAV</span> </italic> genes contain MYB, TC-rich repeats, and MYC elements and that the promoters of some of these genes contain ABRE, E-BOX, MBS, HSE, GT-1, and/or LTRE elements ( <bold> Table <xref ref-type="table" rid="T2"> 2 </xref> </bold> ). ABRE elements are known to be involved in plant responses to <span class="trait">drought</span> and ABA through their interaction with ABRE binding proteins ( <xref rid="B30" ref-type="bibr"> Li et al., 2012a </xref> ). It was reported that the bZIP transcription factors AREB1, AREB2, and ABF3 co-regulate plant responses to osmotic stress by interacting with the ABRE elements in the <italic> <span class="gene">DREB2A</span> </italic> promoter ( <xref rid="B20" ref-type="bibr"> Kim et al., 2011 </xref> ). <xref rid="B57" ref-type="bibr"> Zhu et al. (2005) </xref> mentioned that the MYB element is involved in stresses-induced <span class="trait">drought,</span> low temperature, <span class="trait">salt,</span> ABA, and GA responses ( <xref rid="B57" ref-type="bibr"> Zhu et al., 2005 </xref> ). LTRE elements are thought to contribute primarily to low temperature responses ( <xref rid="B36" ref-type="bibr"> Maestrini et al., 2009 </xref> ). Therefore, we concluded that the soybean <italic> <span class="gene">RAV</span> </italic> genes are very likely involved in plant responses to <span class="trait">drought,</span> <span class="trait">salt,</span> and ABA in plants. Our qRT-PCR results also showed the different induced mechanism in soybean <span class="gene">RAV</span> family for <span class="trait">drought,</span> <span class="trait">salt,</span> and ABA stresses. All of the soybean <italic> <span class="gene">RAV</span> </italic> genes were transcriptionally up-regulated by ABA and that most of the soybean <italic> <span class="gene">RAV</span> </italic> genes were down-regulated by NaCl treatment.
					</p>
				
					<p>&nbsp;&nbsp;
						It was reported that soybean <italic> <span class="gene">Glyma10g34760</span> </italic> , a <italic> RAV2-like </italic> orthologous gene, plays a key role in cytokinin signaling and photoperiod regulation ( <xref rid="B54" ref-type="bibr"> Zhao et al., 2012 </xref> ). Another RAV-like transcription factor has been reported to have a role in controlling photosynthesis and senescence in soybean ( <xref rid="B55" ref-type="bibr"> Zhao et al., 2008 </xref> ). Two <italic> Arabidopsis <span class="gene">RAV</span> </italic> genes ( <italic> AtRAV1L and AtRAV2 </italic> ) are transcriptionally down-regulated by <span class="trait">drought</span> and <span class="trait">salt</span> stress ( <xref rid="B9" ref-type="bibr"> Fu et al., 2014 </xref> ). However, <italic> Brassica napus BnaRAV-1-HY15 </italic> expression is actually induced by NaCl and PEG treatments, even though it shares high identity with the <italic> AtRAV2 </italic> of <italic> Arabidopsis </italic> ( <xref rid="B58" ref-type="bibr"> Zhuang et al., 2011 </xref> ). These studies emphasize that the functions of <italic> <span class="gene">RAV</span> </italic> genes in different species appear to be divergent.
					</p>
				
					<p>&nbsp;&nbsp;
						Abscisic acid plays a crucial role in modulating plant responses to abiotic stresses. The <italic> Arabidopsis </italic> RAV1 transcription factor plays an important role in ABA signaling; it is a negative regulator of ABA signaling during seed germination and during early seedling development ( <xref rid="B7" ref-type="bibr"> Feng et al., 2014 </xref> ). In cotton, expression of <italic> GhRAV1 </italic> is induced by ABA ( <xref rid="B29" ref-type="bibr"> Li et al., 2015 </xref> ), and overexpression of pepper <italic> CARAV1 </italic> in <italic> Arabidopsis </italic> increases plant tolerance to <span class="trait">drought</span> and <span class="trait">salt</span> and increasing the ABA sensitivity of transgenic plants ( <xref rid="B43" ref-type="bibr"> Sohn et al., 2006 </xref> ). The present study revealed that all of the soybean <italic> <span class="gene">RAV</span> </italic> genes are up-regulated under ABA stress ( <bold> Figure <xref ref-type="fig" rid="F7"> 7C </xref> </bold> ). In contrast, most of the soybean <italic> <span class="gene">RAV</span> </italic> genes had down-regulated expression under <span class="trait">drought</span> and <span class="trait">salt</span> stress conditions ( <bold> Figures <xref ref-type="fig" rid="F7"> 7A,B </xref> </bold> ). These different results emphasize the functional diversity of the soybean RAVs. Although <span class="trait">drought-</span> and salt-stress responses in plants are often associated with ABA-signaling pathway ( <xref rid="B56" ref-type="bibr"> Zhu, 2002 </xref> ), Zhu reported that some osmotic stress responsive genes are induced completely dependent or independent of ABA and others are only partially ABA dependent. Perhaps, <span class="gene">RAV</span> gene members showed different repercussion to ABA and osomotic stresses. And this need to further be discussed and investigated in the future.
					</p>
				
					<p>&nbsp;&nbsp;
						To further examine the role of RAVs, we generated the transgenic <italic> Arabidopsis </italic> plants overexpressing <italic> <span class="gene">GmRAV-03</span> </italic> . In this study, our results revealed that the expression of <italic> <span class="gene">GmRAV-03</span> </italic> was induced by NaCl, PEG and ABA, suggesting that <italic> <span class="gene">GmRAV-03</span> </italic> may play a role in plant response to abiotic stress and ABA signaling. Under <span class="trait">salt,</span> PEG, and ABA treatments, we found both seed germination, root length and root surface were increased in the transgenic lines, compared with those of WT. This may indicate that <italic> <span class="gene">GmRAV-03</span> </italic> may take part in plant response to <span class="trait">salt</span> and <span class="trait">drought</span> stresses in an ABA-independent manner.
					</p>
				
					<p>&nbsp;&nbsp;
						It found that <italic> CARAV1 </italic> enhanced the transgenic plant <span class="trait">resistance</span> to NaCl, but sensitive to ABA ( <xref rid="B43" ref-type="bibr"> Sohn et al., 2006 </xref> ). On the contrary, our data in this study revealed that <italic> <span class="gene">GmRAV-03</span> </italic> transgenic <italic> Arabidopsis </italic> were insensitive to both high salinity and exogenous ABA, which is consist with the previous report that the <italic> Arabidopsis </italic> RAV1 overexpressing lines were ABA-insensitive in terms of seed germination, root growth, and chlorophyll synthesis ( <xref rid="B7" ref-type="bibr"> Feng et al., 2014 </xref> ). Under <span class="trait">salt</span> and ABA stresses, the transgenic plants present higher germination rates, root length and root surface than those of WT, indicating that <italic> <span class="gene">GmRAV-03</span> </italic> plays an important role during seed germination and early seedling development. Considering the previous reports and the results of qRT-PCR in this study, the function of soybean RAVs may be diverse and there may be some gene-specific functions among them.
					</p>
				
					<p>&nbsp;&nbsp;
						Thirteen soybean <italic> <span class="gene">RAV</span> </italic> genes were identified and classified after searching the soybean genome sequence. Based on expression analyses, we conclude that the soybean <span class="gene">RAV</span> genes might be involved in plant responses to ABA treatment or <span class="trait">salt,</span> <span class="trait">drought</span> stresses in plants. Especially, overexpression of <italic> <span class="gene">GmRAV-03</span> </italic> could increase the transgenic lines <span class="trait">resistance</span> against high <span class="trait">salt</span> and <span class="trait">drought</span> and result in the transgenic plants insensitive to exogenous ABA in <italic> Arabidopsis </italic> . These results further indicate that soybean <italic> <span class="gene">RAV</span> </italic> genes play important roles in plant responses to abiotic and exogenous ABA stresses.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmCaM4</span> in soybean enhances <span class="trait">resistance</span> to pathogens and tolerance to <span class="trait">salt</span> stress.</h2>
			<h3>PMID:24118726  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rao SS(1), El-Habbak MH, Havens WM, Singh A, Zheng D, Vaughn L, Haudenshield JS, Hartman GL, Korban SS, Ghabrial SA.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant diseases inflict heavy losses on soybean <span class="trait">yield,</span> necessitating an understanding of the molecular mechanisms underlying biotic/abiotic stress responses. Ca(2) (+) is an important universal messenger, and protein sensors, prominently calmodulins (CaMs), recognize cellular changes in Ca(2) (+) in response to diverse signals. Because the development of stable transgenic soybeans is laborious and time consuming, we used the Bean pod mottle virus (BPMV)-based vector for rapid and efficient protein expression and gene silencing. The present study focuses on the functional roles of the gene encoding the soybean <span class="gene">CaM</span> isoform <span class="gene">GmCaM4.</span> Overexpression of <span class="gene">GmCaM4</span> in soybean resulted in enhanced <span class="trait">resistance</span> to three plant pathogens and increased tolerance to high <span class="trait">salt</span> conditions. To gain an understanding of the underlying mechanisms, we examined the potential defence pathways involved. Our studies revealed activation/increased expression levels of pathogenesis-related (PR) genes in <span class="gene">GmCaM4-overexpressing</span> plants and the accumulation of jasmonic acid (JA). Silencing of <span class="gene">GmCaM4,</span> however, markedly repressed the expression of PR genes. We confirmed the in vivo interaction between <span class="gene">GmCaM4</span> and the <span class="gene">CaM</span> binding transcription factor Myb2, which regulates the expression of salt-responsive genes, using the yeast two-hybrid (Y2H) system and bimolecular fluorescence complementation assays. <span class="gene">GmCaM4</span> and Glycine max <span class="gene">CaM</span> binding receptor-like kinase <span class="gene">(GmCBRLK)</span> did not interact in the Y2H assays, but the interaction between <span class="gene">GmCaM2</span> and <span class="gene">GmCBRLK</span> was confirmed. Thus, a <span class="gene">GmCaM2-GmCBRLK-mediated</span> <span class="trait">salt</span> tolerance mechanism, similar to that reported in Glycine soja, may also be functional in soybean. Confocal microscopy showed subcellular localization of the green fluorescent protein (GFP)-GmCaM4 fusion protein in the nucleus and cytoplasm.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification and transcription analysis of soybean carotenoid oxygenase genes during abiotic stress treatments.</h2>
			<h3>PMID:23666055  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang RK(1), Wang CE, Fei YY, Gai JY, Zhao TJ.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Carotenoid oxygenase is a key enzyme in carotenoid metabolism leading to the synthesis of two phytohormones, abscisic acid (ABA) and strigolactone, as well as norisoprenoids. Few studies have analyzed inter-relationship of the metabolic networks of these three substances. In this present paper, soybean carotenoid oxygenase genes were identified to reveal their phylogenetic relationships, and the transcriptional response of these genes to four abiotic stresses (NaCl, PEG, high and low temperature) and ABA treatment were investigated to characterize their potential roles in plant <span class="trait">resistance.</span> Positive selection was found in the branches of carotenoid cleavage dioxygenase (CCD1), <span class="gene">CCD8</span> and NCED (9-cis-epoxycarotenoid oxygenase), indicating an adaptive evolution in these clades. In soybean eight carotenoid oxygenase genes were identified. The transcriptional responses of almost all of them under stress and ABA conditions were significantly altered when assessed by quantitative polymerase chain reaction. Notably, CCD1 and CCD4, previously known as the key genes in norisoprenoids metabolism, showed especially strong responses to the abiotic stresses and ABA treatment. Furthermore, transcription levels of <span class="gene">CCD7</span> and CCD8, key genes for the strigolactone pathway, highly increased during ABA treatment providing further evidence that ABA is involved in regulating strigolactone metabolism. All of the carotenoid oxygenase genes in soybean are involved in plant abiotic stress physiology, and ABA is presumed to be a core regulatory substance. These findings provide some insights into the mechanisms that underlie the regulation of tolerance response to abiotic stresses in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel EAR-motif-containing gene <span class="gene">GmERF4</span> from soybean (Glycine max L.).</h2>
			<h3>PMID:19597961  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Chen X, Xu Z, Li L, Guo J, Ma Y.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transcriptional repressors are emerging as central regulators of development and stress responses in different organisms. The ERF-associated amphiphilic repression (EAR) motif was identified as essential for transcriptional repression. To gain a better understanding of this type of protein, we reported here a novel <span class="gene">GmERF4</span> protein from soybean. Sequence alignment showed that <span class="gene">GmERF4</span> contains one AP2/ERF domain, two putative nuclear localization signal regions and one EAR motif. The <span class="gene">GmERF4</span> protein was preferentially localized to the nucleus of onion epidermis cells and bound specifically to the GCC box and DRE/CRT element in vitro. Furthermore, the expression of <span class="gene">GmERF4</span> was induced by ethylene, JA, SA, cold, <span class="trait">salt,</span> <span class="trait">drought,</span> and soybean mosaic virus, and repressed by ABA. Constitutive expression of <span class="gene">GmERF4</span> in transgenic tobacco plants increased tolerance to <span class="trait">salt</span> and <span class="trait">drought</span> stresses compared with wild-type plants, but did not exhibit detectable <span class="trait">resistance</span> against bacterial infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of the soybean <span class="gene">GmERF3</span> gene, an AP2/ERF type transcription factor for increased tolerances to <span class="trait">salt,</span> <span class="trait">drought,</span> and diseases in transgenic tobacco.</h2>
			<h3>PMID:19602544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang G(1), Chen M, Li L, Xu Z, Chen X, Guo J, Ma Y.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A new member of the AP2/ERF transcription factor family, <span class="gene">GmERF3,</span> was isolated from soybean. Sequence analysis showed that <span class="gene">GmERF3</span> contained an AP2/ERF domain of 58 amino acids and two putative nuclear localization signal (NLS) domains. It belonged to a group IV protein in the ERF (ethylene response factor) subfamily as typified by a conserved N-terminal motif [MCGGAI(I/L)]. Expression of <span class="gene">GmERF3</span> was induced by treatments with high salinity, <span class="trait">drought,</span> abscisic acid (ABA), salicylic acid (SA), jasmonic acid (JA), ethylene (ET), and soybean mosaic virus (SMV), whereas there was no significant <span class="gene">GmERF3</span> mRNA accumulation under cold stress treatment. <span class="gene">GmERF3</span> could bind to the GCC box and DRE/CRT element, and was targeted to the nucleus when transiently expressed in onion epidermal cells. The <span class="gene">GmERF3</span> protein fused to the GAL4 DNA-binding domain to activate transcription of reporter genes in yeast. Ectopic expression of the <span class="gene">GmERF3</span> gene in transgenic tobacco plants induced the expression of some PR genes and enhanced <span class="trait">resistance</span> against infection by Ralstonia solanacearum, Alternaria alternata, and tobacco mosaic virus (TMV), and gave tolerance to high salinity and dehydration stresses. Furthermore, overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions. The overall results suggested that <span class="gene">GmERF3</span> as an AP2/ERF transcription factor may play dual roles in response to biotic and abiotic stresses in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> in Glycine max L. that enhances abiotic stress tolerance in Saccharomyces cerevisiae.</h2>
			<h3>PMID:20155384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang C(1), Zhao H, Liu Y, Li Q, Liu X, Tan H, Yuan C, Dong Y.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel glycogen synthase kinase-3 gene, <span class="gene">GmGSK,</span> was isolated from Glycine max. It is 1,596 bp in length with one ORF of 410 amino acids. Southern blot analysis revealed that it has at least two copies in the G. max genome. <span class="gene">GmGSK,</span> when transiently expressed in Nicotiana tabacum leaves, was localized in both cell membrane and cytoplasm. Northern blot analysis indicated that <span class="gene">GmGSK</span> is expressed in all tissues, with highest expression in the root. <span class="gene">GmGSK</span> can be induced by various abiotic stresses. When transformed with <span class="gene">GmGSK,</span> Saccharomyces cerevisiae exhibited enhanced <span class="trait">resistance</span> to <span class="trait">salt</span> and <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide analysis of the YABBY family in soybean and functional identification of <span class="gene">GmYABBY10</span> involvement in high <span class="trait">salt</span> and <span class="trait">drought</span> stresses.</h2>
			<h3>PMID:28866235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao SP(1), Lu D(2), Yu TF(3), Ji YJ(4), Zheng WJ(3), Zhang SX(5), Chai SC(3), Chen ZY(6), Cui XY(7).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						YABBY family is a plant specific transcription factor family, with the typical N-terminal C2C2 type zinc finger domain and the C-terminal YABBY conservative structure domain, which plays important biological roles in plant growth, development and morphogenesis. In this study, a total of 17 YABBY genes were identified in the soybean genome. The results of this research showed that 17 soybean YABBY genes were located on 11 chromosomes. Analysis of putative cis-acting elements showed that soybean YABBY genes contained lots of MYB and MYC elements. Quantitative Real-time PCR (qRT-PCR) showed that the expressions of <span class="gene">GmYABBY3,</span> <span class="gene">GmYABBY10</span> and <span class="gene">GmYABBY16</span> were more highly sensitive in <span class="trait">drought,</span> NaCl and ABA stresses. And the transient expression in Arabidopsis protoplasts showed that <span class="gene">GmYABBY3</span> protein distributed uniformly the whole cells, while <span class="gene">GmYABBY10</span> protein was mainly localized in the membranes and cytoplasm and <span class="gene">GmYABBY16</span> protein was localized the nucleus and membranes. To further identify the function of <span class="gene">GmYABBY10,</span> we obtained the transgenic Arabidopsis overexpression <span class="gene">GmYABBY10.</span> Based on germination and seedling root arrays in transgenic Arabidopsis, we found that the rates of wild type seeds was a litter higher than that of <span class="gene">GmYABBY10</span> transgenic seeds under both PEG and NaCl treatment. While the root length and root surface of wild type seedlings were bigger than those of <span class="gene">GmYABBY10</span> transgenic seedlings. When seedlings were grown in soil, the survival rates of wild type were higher than those of transgenic plants under both PEG and NaCl treatment, which indicated that <span class="gene">GmYABBY10</span> may be a negatively regulator in plant resistances to <span class="trait">drought</span> and <span class="trait">salt</span> stresses. This study provided valuable information regarding the classification and functions of YABBY genes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsSNAP33, a novel Glycine soja SNAP25-type protein gene: Improvement of plant <span class="trait">salt</span> and <span class="trait">drought</span> tolerances in transgenic Arabidopsis thaliana.</h2>
			<h3>PMID:28841544  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nisa ZU(1), Mallano AI(2), Yu Y(3), Chen C(4), Duan X(5), Amanullah S(6), Kousar A(7), Baloch AW(8), Sun X(9), Tabys D(10), Zhu Y(11).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The N-ethylmaleimide-sensitive factor attachment protein receptor (SNARE) superfamily, specifically the SNAP25-type proteins and t-SNAREs, have been proposed to regulate cellular processes and plant <span class="trait">resistance</span> mechanisms. However, little is known about the role of SNAP25-type proteins in combating abiotic stresses, specifically in wild soybean. In the current study, the isolation and functional characterization of the putative synaptosomal-associated SNAP25-type protein gene GsSNAP33 from wild soybean (Glycine soja) were performed. GsSNAP33 has a molecular weight of 33,311 Da and comprises 300 amino acid residues along with Qb-Qc SNARE domains. Multiple sequence alignment revealed the highest similarity of the GsSNAP33 protein to <span class="gene">GmSNAP33</span> (91%), VrSNAP33 (89%), PvSNAP33 (86%) and AtSNAP33 (63%). Phylogenetic studies revealed the abundance of SNAP33 proteins mostly in dicotyledons. Quantitative real-time PCR assays confirmed that GsSNAP33 expression can be induced by <span class="trait">salt,</span> alkali, ABA and PEG treatments and that GsSNAP33 is highly expressed in the pods, seeds and roots of Glycine soja. Furthermore, the overexpression of the GsSNAP33 gene in WT Arabidopsis thaliana resulted in increased germination rates, greater root lengths, improved photosynthesis, lower electrolyte leakage, higher biomass production and up-regulated expression levels of various stress-responsive marker genes, including KINI, COR15A, P5Cs, RAB18, <span class="gene">RD29A</span> and COR47 in transgenic lines compared with those in WT lines. Subcellular localization studies revealed that the GsSNAP33-eGFP fusion protein was localized to the plasma membrane, while eGFP was distributed throughout whole cytoplasm of onion epidermal cells. Collectively, our findings suggest that GsSNAP33, a novel plasma membrane protein gene of Glycine soja, might be involved in improving plant responses to <span class="trait">salt</span> and <span class="trait">drought</span> stresses in Arabidopsis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Drought Tolerance Conferred in Soybean (Glycine max. L) by <span class="gene">GmMYB84,</span> a Novel R2R3-MYB Transcription Factor.</h2>
			<h3>PMID:29016915</h3>
				

				
			<p>
				作者：Wang N(1), Zhang W(1), Qin M(1), Li S(1), Qiao M(2), Liu Z(1), Xiang F(1).
				<br/> 
				年份：2017 Oct 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						MYB-type transcription factors (MYB TFs) play diverse roles in plant development and stress responses. However, the mechanisms underlying the actions of MYB TFs during stress response remain unclear. In this study we identified a R2R3-MYB TF in soybean (Glycine max), denoted <span class="gene">GmMYB84,</span> which contributes to <span class="trait">drought</span> <span class="trait">resistance.</span> Expression of <span class="gene">GmMYB84</span> was induced by <span class="trait">drought,</span> <span class="trait">salt</span> stress, H2O2 and ABA. Compared with the wild type (WT), <span class="gene">GmMYB84-overexpressing</span> soybean mutants (OE lines) exhibited enhanced <span class="trait">drought</span> <span class="trait">resistance</span> with a higher survival rate, longer primary root length, greater proline and reactive oxygen species (ROS) contents, higher antioxidant enzyme activities [peroxidase (POD), catalase (CAT) and superoxide dismutase (SOD)], a lower dehydration rate and reduced malondialdehyde (MDA) content. We also found that ROS could induce SOD/POD/CAT activity in OE lines. In particular, we found that the optimal level of ROS is required for <span class="gene">GmMYB84</span> to modulate primary root elongation. Some ROS-related genes were up-regulated under abiotic stress in <span class="gene">GmMYB84</span> transgenic plants compared with the WT. Furthermore, electrophoretic mobility shift assay and luciferase reporter analysis demonstrated that <span class="gene">GmMYB84</span> binds directly to the promoter of <span class="gene">GmRBOHB-1</span> and <span class="gene">GmRBOHB-2</span> genes. Based on this evidence, we propose a model for how <span class="gene">GmMYB84,</span> H2O2 and antioxidant enzymes work together to control root growth under both optimal and <span class="trait">drought</span> stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel <italic> Glycine soja </italic> homeodomain-leucine zipper (HD-Zip) I gene, <italic> Gshdz4 </italic> , positively regulates bicarbonate tolerance and responds to osmotic stress in <italic> Arabidopsis </italic></h2>
			<h3>PMID:27553065</h3>
				
					<h3>PMCID:4995822</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Lei </given-names>  </name>  <address>  <email> caoleifirst@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Yang </given-names>  </name>  <address>  <email> yuyang0302pbl@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> DuanMu </surname>  <given-names> Huizi </given-names>  </name>  <address>  <email> duanmuhuizi@sina.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Chao </given-names>  </name>  <address>  <email> chenchaochenchao09@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Duan </surname>  <given-names> Xiangbo </given-names>  </name>  <address>  <email> duanxbo@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Pinghui </given-names>  </name>  <address>  <email> zhupinghui@outlook.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ranran </given-names>  </name>  <address>  <email> gdsanyecao@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Qiang </given-names>  </name>  <address>  <email> lstrong@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhu </surname>  <given-names> Yanming </given-names>  </name>  <address>  <email> ymzhu2001@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Ding </surname>  <given-names> Xiaodong </given-names>  </name>  <address>  <email> xiaodong.ding@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Key Laboratory of Agricultural Biological Functional Genes, Northeast Agricultural University, Harbin, 150030 China  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To our surprise, the negative regulation of <span class="trait">drought</span> stress by <italic> Gshdz4 </italic> was observed. Phenotypic experiments at both germination and seedling stages revealed that transgenic plants showed more sensitivity to osmotic stress than the WT, implying that <italic> Gshdz4 </italic> regulates alkaline and osmotic stresses by different ways. To better understand the molecular basis of the impaired <span class="trait">drought</span> stress tolerance of the <italic> Gshdz4 </italic> transgenic plants, the expression patterns of the stress-related genes, such as <italic> <span class="gene">P5CS</span> </italic> , <italic> RD22 </italic> and <italic> <span class="gene">RD29A</span> </italic> , were investigated. These genes have been used as convenient markers for monitoring the ABA and stress-responsive pathways in plants [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. However, there was no significant difference observed in the experiment, implying that <italic> Gshdz4 </italic> may not regulate response to <span class="trait">drought</span> stress via these genes.
					</p>
				
					<p>&nbsp;&nbsp;
						We selected three T <sub> 3 </sub> homozygous overexpression lines (# 14, # 20, and # 23) to study their <span class="trait">resistance</span> to various stresses. The WT and homozygous transgenic <italic> Arabidopsis </italic> seeds were sterilized and were sown on 0.5&#x000d7; <span class="gene">MS</span> agar medium, which were supplemented with 0, 6, 7&#x000a0;mM NaHCO <sub> 3 </sub> , or 6&#x000a0;mM KHCO <sub> 3 </sub> or 325&#x000a0;mM mannitol or KOH to make the medium at pH&#x000a0;7.5 or pH&#x000a0;8.2. For root length and fresh weight assays, the seeds were germinated on 0.5&#x000d7; <span class="gene">MS</span> agar medium for 7&#x000a0;days and then were transferred to fresh medium supplemented with 0, 6, 8&#x000a0;mM NaHCO <sub> 3 </sub> , 6&#x000a0;mM KHCO <sub> 3 </sub> , 325&#x000a0;mM mannitol, or KOH to adjust the medium to pH&#x000a0;7.5, pH&#x000a0;8.2. The seedlings were grown on the medium for 12&#x02013;14 days and the lengths of roots were measured. All experiments were performed 3 completely independent biological replicates at least. All statistical analysis was done with EXCEL2010 (Microsoft, <ext-link ext-link-type="uri" xlink:href="http://www.microsoftstore.com.cn/"> http://www.microsoftstore.com.cn/ </ext-link> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comprehensive insights on how 2,4-dichlorophenoxyacetic acid retards senescence in post-harvest citrus fruits using transcriptomic and proteomic approaches</h2>
			<h3>PMID:24215076</h3>
				
					<h3>PMCID:3883282</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="no">  <name>  <surname> Ma </surname>  <given-names> Qiaoli </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Three proteins involved in pathogen&#x02013;plant interactions, chitinase (spot a25), miraculin-like protein 2 (spot a26), and osmotin 34 (spot d1), were induced by 2,4-D treatment. HSP abundances also changed. For example, HSP19 class II (spot c3), mitochondrial HSP23.5 (spots e1 and e2), and chaperonins (spot e3 and a30) were differentially accumulated, but HSA32 and HSP21 (spots b4 and e5) decreased after treatment. The different expression modes of these stress-responsive proteins may relate to their special functions in preserving fruits from various biotic and abiotic stresses during the storage period. <span class="gene">C2</span> domain-containing proteins have Ca <sup> 2+ </sup> -dependent membrane-binding motifs and are involved in plant <span class="trait">resistance</span> to pathogen infection, and osmotic, dehydration, and oxidative stresses ( <xref ref-type="bibr" rid="CIT0021"> Kim <italic> et al. </italic> , 2008 </xref> ; <xref ref-type="bibr" rid="CIT0045"> Yokotani <italic> et al. </italic> , 2009 </xref> ). In this study, <span class="gene">C2</span> domain-containing proteins (spots a13, a1, and a19) and a calcium-binding EF hand family protein (spot a7) significantly accumulated after treatment. This indicated that calcium signalling may be involved in 2,4-D signal transduction and may promote defence responses in 2,4-D-treated fruits.
					</p>
				
					<p>&nbsp;&nbsp;
						The processes of fruit development, ripening, and senescence are determined by dynamic changes in hormone levels. When entering the ripening stage, the auxin content decreases and ethylene release or ABA biosynthesis gradually increases. Olinda orange fruit treated with 2,4-D has very high levels of auxin. 2,4-D levels rose to 101.6ng g <sup> &#x02013;1 </sup> at 6 HPT, reached a maximum of 326.5ng g <sup> &#x02013;1 </sup> at 24 HPT, and then remained at ~200ng g <sup> &#x02013;1 </sup> after 48 HPT ( <xref ref-type="fig" rid="F7"> Fig. 7A </xref> ). The accumulation of 2,4-D within 24h after treatment was probably caused by two factors. First, it was actively transported into cells, but was not pumped out by efflux carriers, which could not use the 2,4-D as a substrate ( <xref ref-type="bibr" rid="CIT0010"> Delbarre <italic> et al. </italic> , 1996 </xref> ). Secondly, <span class="gene">GH3</span> subfamily family II members, such as <italic> GH3.1 </italic> , which encode enzymes catalysing the formation of many inactive IAA&#x02013;amino acid conjugates when plant cell auxin levels are high, have a low affinity for 2,4-D ( <xref ref-type="bibr" rid="CIT0039"> Staswick <italic> et al. </italic> , 2005 </xref> ). It is worth noting that the expression of multidrug <span class="trait">resistance</span> transporter genes involved in plant detoxification and adaption to high levels of 2,4-D, such as pleiotropic drug <span class="trait">resistance</span> 6 ( <italic> PDR6 </italic> ), <italic> PDR11 </italic> , <italic> <span class="gene">PDR12</span> </italic> , multidrug <span class="trait">resistance</span> 4 ( <italic> MDR4 </italic> ), <italic> MDR17 </italic> , multidrug <span class="trait">resistance</span> protein 2 ( <italic> MRP2 </italic> ), and <italic> MRP5 </italic> , was mainly up-regulated at 24 HPT ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S2 </ext-link> at <italic> JXB </italic> online). These genes may be involved in extruding 2,4-D from cells and preventing fruit damage caused by the overaccumulation of 2,4-D. Consistently, the 2,4-D concentration reached its peak level at 24 HPT and then declined and reached an equilibrium at 48 HPT ( <xref ref-type="fig" rid="F7"> Fig. 7A </xref> ). In addition, <italic> <span class="gene">GH3</span> </italic> is one of the three families of early auxin-responsive genes that can be immediately induced by IAA or by synthetic auxins within a few minutes. The persistent induction of <italic> GH3.1 </italic> indicated that 2,4-D had long-lasting rather than instantaneous effects.
					</p>
				
					<p>&nbsp;&nbsp;
						Post-harvest fruit faces various biotic and abiotic stresses during storage. Water loss and pathogen infection are two of the most important factors that affect fruit commercial quality during long-term storage. In this study, fruit drenched with 2,4-D showed significant decreases in the incidence of decay ( <xref ref-type="fig" rid="F1"> Fig. 1A </xref> ) and a high water content in the peel ( <xref ref-type="fig" rid="F1"> Fig. 1D </xref> ). The &#x02018;omics&#x02019; data revealed that the defence hormone, SA, accumulated in treated fruit peel, and many genes, proteins, and secondary metabolites downstream of the defence signalling pathway were also up-regulated by 2,4-D. TFs that participate in plant defence signalling, such as the WRKY, NAC, and AP2-EREBP family members ( <xref ref-type="bibr" rid="CIT0047"> Zhang <italic> et al. </italic> , 2009 </xref> ; <xref ref-type="bibr" rid="CIT0023"> Li <italic> et al. </italic> , 2011 </xref> ), were also significantly up-regulated. The NAC and WRKY TFs are the largest TF families in plants. They are involved in the regulation of many different plant processes, such as biotic or abiotic stresses and plant senescence ( <xref ref-type="bibr" rid="CIT0037"> Rushton <italic> et al. </italic> , 2010 </xref> ; <xref ref-type="bibr" rid="CIT0002"> Besseau <italic> et al. </italic> , 2012 </xref> ; <xref ref-type="bibr" rid="CIT0027"> Nakashima <italic> et al. </italic> , 2012 </xref> ). It was found that <italic> <span class="gene">WRKY11</span> </italic> , <italic> <span class="gene">WRKY30</span> </italic> , <italic> WRKY33 </italic> , <italic> <span class="gene">WRKY41</span> </italic> , <italic> <span class="gene">WRKY42</span> </italic> , <italic> WRKY65 </italic> , and <italic> WRKY70 </italic> were differentially up-regulated during the earlier time periods (12 and 24 HPT). Studies in other species have shown that overexpression of <italic> <span class="gene">WRKY30</span> </italic> or <italic> <span class="gene">WRKY11</span> </italic> dramatically increased osmotic stress tolerance ( <xref ref-type="bibr" rid="CIT0024"> Liu <italic> et al. </italic> , 2011 </xref> ; <xref ref-type="bibr" rid="CIT0038"> Shen <italic> et al. </italic> , 2012 </xref> ), and <xref ref-type="bibr" rid="CIT0043"> &#x000dc;lker <italic> et al. </italic> (2007) </xref> reported that the <italic> WRKY70 </italic> TF in <italic> Arabidopsis </italic> negatively influenced plant senescence and changed the defence signalling pathways. In addition, stress-responsive genes, especially for biotic stresses, were over-represented during the late time periods (48 and 72 HPT) ( <xref ref-type="fig" rid="F4"> Fig. 4B </xref> ). Moreover, a quarter of the 76 proteins with differential expression between the control and the treated samples were related to stress defence ( <xref ref-type="fig" rid="F6"> Fig. 6B </xref> ). For example, chitinases that were probably involved in hydrolysing the cell wall of fungal hyphae ( <xref ref-type="bibr" rid="CIT0001"> Ballester <italic> et al. </italic> , 2010 </xref> ) were induced by 2,4-D at both the RNA ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Tablel S2 </ext-link> at <italic> JXB </italic> online) and protein levels ( <xref ref-type="fig" rid="F6"> Fig. 6 </xref> ; <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spot a25). A gene encoding a member of the PR5 protein family (thaumatin-like protein) that was involved in membrane permeabilization, glucan hydrolysis, and apoptosis constantly increased after treatment. Correspondingly, the protease inhibitor, miraculin-like protein 2, which contains a thaumatin motif and has antifungal activity ( <xref ref-type="bibr" rid="CIT0042"> Tsukuda <italic> et al. </italic> , 2006 </xref> ), was highly expressed at the protein level ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spot a26). It is worth noting that miraculin-like protein and osmotin 34 ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spot 1) were involved in plant tolerance to pathogen infections and <span class="trait">resistance</span> to water deficiency ( <xref ref-type="bibr" rid="CIT0016"> Gimeno <italic> et al. </italic> , 2009 </xref> ; <xref ref-type="bibr" rid="CIT0044"> Viktorova <italic> et al. </italic> , 2012 </xref> ). HSPs play important roles in the re-establishment of cellular homeostasis by partially binding to the denatured proteins and preventing irreversible protein inactivation and aggregation under raised heat conditions. They are also involved in plant <span class="trait">resistance.</span> The present data show that 13 of the 32 HSPs, including HSP22.0, HSP21, <span class="gene">HSP26.5-P,</span> HSC70-5, HSP91, and <span class="gene">DNAJ</span> heat shock family proteins, were differentially expressed at the transcriptional level and were up-regulated at the proteome level at 24 and 72 HPT. Seven of the 19 stress response protein spots corresponded to HSPs, including five low molecular mass HSPs ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spots c3, e1, e2, b4, and e5) and chaperonins HSP60 ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spots e3 and e30). Previous studies have shown that chaperonins play an important role in assisting chloroplast proteins, such as Rubisco assembling proteins ( <xref ref-type="bibr" rid="CIT0004"> Boston <italic> et al. </italic> , 1996 </xref> ). In this study, with the exception of chaperonins, Rubisco proteins ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert344/-/DC1"> Supplementary Table S6 </ext-link> , spot a10 and a15) also significantly accumulated in treated fruit peel, which might help explain the fact that 2,4-D could keep the fruit calyx green after prolonged storage. Generally, the expression changes in these defence genes and proteins will enhance fruit <span class="trait">resistance</span> to pathogen infection, and the fruit decay incidence results after 2 years of storage ( <xref ref-type="fig" rid="F1"> Fig. 1A </xref> ) further verified that 2,4-D enhanced the orange fruit antifungal activity.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing cyclic electron flow is related to Na <sup> + </sup> sequestration into vacuoles for <span class="trait">salt</span> tolerance in soybean</h2>
			<h3>PMID:26276865</h3>
				

				
					<h3>PMCID:4623694</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Yi </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In land plants, the NAD(P)H dehydrogenase (NDH) complex reduces plastoquinones and drives cyclic electron flow (CEF) around PSI. It also produces extra ATP for photosynthesis and improves plant fitness under conditions of abiotic environmental stress. To elucidate the role of CEF in <span class="trait">salt</span> tolerance of the photosynthetic apparatus, Na(+) concentration, chlorophyll fluorescence, and expression of NDH B and H subunits, as well as of genes related to cellular and vacuolar Na(+) transport, were monitored. The salt-tolerant Glycine max (soybean) variety S111-9 exhibited much higher CEF activity and ATP accumulation in light than did the salt-sensitive variety Melrose, but similar leaf Na(+) concentrations under <span class="trait">salt</span> stress. In S111-9 plants, <span class="gene">ndhB</span> and <span class="gene">ndhH</span> were highly up-regulated under <span class="trait">salt</span> stress and their corresponding proteins were maintained at high levels or increased significantly. Under <span class="trait">salt</span> stress, S111-9 plants accumulated Na(+) in the vacuole, but Melrose plants accumulated Na(+) in the chloroplast. Compared with Melrose, S111-9 plants also showed higher expression of some genes associated with Na(+) transport into the vacuole and/or cell, such as genes encoding components of the CBL10 (calcineurin B-like protein 10)-CIPK24 (CBL-interacting protein kinase 24)-NHX (Na(+)/H(+) antiporter) and CBL4 (calcineurin B-like protein 4)-CIPK24-SOS1 <span class="trait">(salt</span> overly sensitive 1) complexes. Based on the findings, it is proposed that enhanced NDH-dependent CEF supplies extra ATP used to sequester Na(+) in the vacuole. This reveals an important mechanism for <span class="trait">salt</span> tolerance in soybean and provides new insights into plant <span class="trait">resistance</span> to <span class="trait">salt</span> stress.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						It was further investigated whether the increased ATP content in S111-9 under <span class="trait">salt</span> stress could be attributed to enhanced CEF caused by increased accumulation of the NDH complex. The amounts of NDH-B and NDH-H are often used to monitor the accumulation of the NDH complex ( <xref rid="CIT0012" ref-type="bibr"> Hashimoto <italic> et al. </italic> , 2003 </xref> ; <xref rid="CIT0033" ref-type="bibr"> Munshi <italic> et al. </italic> , 2005 </xref> ). Accordingly immunoblotting was performed with antibodies raised against NDH-B and NDH-H to estimate the amount of NDH complex in S111-9 and Melrose. As shown in <xref ref-type="fig" rid="F5"> Fig. 5A </xref> , the constitutive levels of NDH-B and NDH-H proteins were much higher in S111-9 than in Melrose. Both NDH-B and NDH-H protein levels were induced by <span class="trait">salt</span> stress in Melrose. In S111-9, a salt-mediated increase was observed only in NDH-H ( <xref ref-type="fig" rid="F5"> Fig. 5A </xref> ). During the 2 d <span class="trait">salt</span> treatment, NDH-B and NDH-H protein levels were generally higher in S111-9 than in Melrose. Moreover, the results of immunogold labelling for NDH-B and NDH-H protein ( <xref ref-type="fig" rid="F5"> Fig. 5C </xref> , <xref ref-type="fig" rid="F5"> D </xref> ) are in accordance with the above immunoblotting analyses. Furthermore, the relative mRNA abundance of <italic> <span class="gene">ndhB</span> </italic> and <italic> <span class="gene">ndhH</span> </italic> was higher in S111-9 than in Melrose, both before and after <span class="trait">salt</span> stress ( <xref ref-type="fig" rid="F5"> Fig. 5B </xref> ), especially for <italic> <span class="gene">ndhH</span> </italic> , consistent with the results of immunoblotting. The mRNA levels of <italic> <span class="gene">ndhB</span> </italic> and <italic> <span class="gene">ndhH</span> </italic> were up-regulated by <span class="trait">salt</span> stress in both Melrose and S111-9. These findings implied that in S111-9, the higher abundance of the NDH complex might result in faster CEF and more ATP accumulation in the light.
					</p>
				
					<p>&nbsp;&nbsp;
						Effect of <span class="trait">salt</span> stress on the protein and mRNA abundance of <italic> <span class="gene">ndhB</span> </italic> and <italic> <span class="gene">ndhH</span> </italic> . (A) Changes in NDH-H and NDH-B content were analysed using immunoblotting in the leaves of Melrose and S111-9 treated with 150mM NaCl for 0h, 4h, 1 d, and 2 d. Equal loading is shown by Coomassie-stained gels below each blot. (B) Effect of <span class="trait">salt</span> stress on the mRNA expression of <italic> ndh </italic> B and <italic> ndh </italic> H. RNA gel blotting hybridization analysis of <italic> ndh </italic> B and <italic> ndh </italic> H was performed using the total RNA from Melrose and S111-9 treated with 150mM NaCl stress for 0h, 4h, 1d, and 2 d. The RNA samples loaded were visualized by ethidium bromide staining of 28S rRNA (lower panel). (C and D) Immunogold labelling of NDH-B and NDH-H in the chloroplast organelles of the mature leaves of soybean ( <italic> Glycine max </italic> ) under 150mM <span class="trait">salt</span> stress. Seven to eight individual cells of palisade and spongy layers were examined in several immunolabelled sections. Numbers of gold particles per unit area (&#x003bc;m <sup> 2 </sup> ) are given as mean &#x000b1;SD. (C) Effect of <span class="trait">salt</span> stress on label density of NDH-B (&#x003bc;m <sup> 2 </sup> ). (D) Effect of <span class="trait">salt</span> stress on label density of NDH-H (&#x003bc;m <sup> 2 </sup> ). Different letters indicate significantly different values ( <italic> P </italic> &#x0003c;0.05) by Tukey&#x02019;s test.
					</p>
				
					<p>&nbsp;&nbsp;
						To assess which Na <sup> + </sup> , K <sup> + </sup> , and Ca <sup> 2+ </sup> transport-related genes would respond to <span class="trait">salt</span> stress, transcript levels were measured by reverse transcription and quantitative PCR (RT&#x02013;qPCR) for a series of Na <sup> + </sup> , K <sup> + </sup> , and Ca <sup> 2+ </sup> transport-related genes. Among these genes, <italic> CBL4 </italic> (calcineurin B-like protein 4), <italic> CIPK24 </italic> (CBL-interacting protein kinase 24), and <italic> NHX </italic> (Na <sup> + </sup> /H <sup> + </sup> antiporter) were significantly up-regulated in response to <span class="trait">salt</span> stress in S111-9, but they were unchanged and even down-regulated in Melrose ( <xref ref-type="fig" rid="F9"> Fig. 9A </xref> &#x02013; <xref ref-type="fig" rid="F9"> C </xref> ). Transcript levels of <italic> AKT </italic> (Arabidopsis K <sup> + </sup> transporter), <italic> KAT2 </italic> (K <sup> + </sup> channel in Arabidopsis thaliana 2), <italic> KEA </italic> (K efflux antiporter), and <italic> <span class="gene">SOS1</span> </italic> <span class="trait">(salt</span> overly sensitive 1) in S111-9 were also significantly higher than those in Melrose after 2 d <span class="trait">salt</span> stress ( <xref ref-type="fig" rid="F9"> Fig. 9D </xref> &#x02013; <xref ref-type="fig" rid="F9"> G </xref> ). Therefore, these genes might be partly positively regulated by <span class="trait">salt</span> stress. Transcript levels of <italic> CIPK24 </italic> and <italic> NHX </italic> were ~4- and 3-fold higher, respectively, in S111-9 than in Melrose ( <xref ref-type="fig" rid="F9"> Fig. 9B </xref> , <xref ref-type="fig" rid="F9"> C </xref> ), which further indicated that <italic> CIPK24 </italic> , which is involved in Ca <sup> 2+ </sup> transport, and <italic> NHX </italic> , which is involved in Na <sup> + </sup> transport and accumulation in vacuoles, could directly respond to <span class="trait">salt</span> stress in S111-9. In addition, transcript levels of <italic> CBL3 </italic> (calcineurin B-like protein 3), <italic> CBL10 </italic> (calcineurin B-like protein 10), and <italic> CIPK23 </italic> (CBL-interacting protein kinase 23) increased significantly in both varieties after <span class="trait">salt</span> stress, but their transcript levels were significantly higher in Melrose than in S111-9 ( <xref ref-type="fig" rid="F9"> Fig. 9H </xref> &#x02013; <xref ref-type="fig" rid="F9"> J </xref> ). Collectively, these results revealed a greater difference between two varieties in transcription of key genes related to Na <sup> + </sup> , K <sup> + </sup> , and Ca <sup> 2+ </sup> transport, which could be a response to <span class="trait">salt</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Salt</span> stress-induced expression of genes related to Na <sup> + </sup> transport. Transcript levels of <span class="trait">salt</span> stress-related genes, relative to <italic> ELF1B </italic> and <italic> CYP2 </italic> expression, in the leaves of plants treated with 150mM NaCl for 0h, 4h, 1 d, and 2 d (mean &#x000b1;SD, <italic> n </italic> =3). Different letters indicate significantly different values ( <italic> P </italic> &#x0003c;0.05) by Tukey&#x02019;s test; <italic> n </italic> refers to the number of biological replicates. <italic> NHX </italic> , <italic> KEA </italic> , <italic> <span class="gene">SOS1</span> </italic> , <italic> ERA1 </italic> , and <italic> PP2C </italic> are Na <sup> + </sup> transport-associated genes. <italic> NHX </italic> and <italic> <span class="gene">SOS1</span> </italic> are involved in the accumulation of Na <sup> + </sup> in vacuoles and in shoots, respectively. <italic> KEA </italic> is involved in regulating chloroplast function. <italic> ERA1 </italic> and <italic> PP2C </italic> function in the regulation of photosynthesis. <italic> AKT </italic> and <italic> KAT2 </italic> are K <sup> + </sup> transport-associated genes, and both of them are related to K <sup> + </sup> influx. <italic> CBL4 </italic> , <italic> CBL3 </italic> , and <italic> CBL10 </italic> are calcineurin B-like protein genes (CBLs); <italic> CIPK24 </italic> and <italic> CIPK23 </italic> are CBL-interacting protein kinase genes; all of them are Ca <sup> 2+ </sup> transport-associated genes that may assist in Na <sup> + </sup> transport regulation by Ca <sup> 2+ </sup> concentration in the cell.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of the NDH complex in adaptation to <span class="trait">salt</span> stress was reported in studies of an <italic> <span class="gene">ndhB</span> </italic> -inactivated mutant of the cyanobacterium <italic> Synechocystis </italic> sp. PCC 6803 ( <xref rid="CIT0046" ref-type="bibr"> Tanaka <italic> et al. </italic> , 1997 </xref> ), which showed that enhanced CEF from the cytosol to PSI via NDH is essential for the adaptation of cyanobacteria to <span class="trait">salt</span> stress. A previous study demonstrated that <span class="trait">salt</span> stress could induce significant up-regulation of CEF in soybean ( <xref rid="CIT0052" ref-type="bibr"> Yang <italic> et al. </italic> , 2007 </xref> ). In addition, the <italic> hcef1 crr2-2 </italic> double mutant, which is deficient in thylakoid NDH complex and FBPase, is highly sensitive to light and lacks elevated CEF1, further suggesting that NDH might play a direct role in catalysing or regulating CEF ( <xref rid="CIT0025" ref-type="bibr"> Livingston <italic> et al. </italic> , 2010 <italic> a </italic> , <italic> b </italic> </xref> ). The reduction of P700 <sup> + </sup> has been taken as a good indicator of NDH activity ( <xref rid="CIT0042" ref-type="bibr"> Shikanai <italic> et al. </italic> , 1998 </xref> ; <xref rid="CIT0015" ref-type="bibr"> Horvath <italic> et al. </italic> , 2000 </xref> ; <xref rid="CIT0040" ref-type="bibr"> Rumeau <italic> et al. </italic> , 2007 </xref> ), and the present results suggested that <span class="trait">salt</span> stress induces a remarkable increase in NDH activity in S111-9, greater than that in Melrose ( <xref ref-type="fig" rid="F3"> Fig. 3 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Salinity stress <span class="trait">resistance</span> offered by endophytic fungal interaction between Penicillium minioluteum LHL09 and glycine max. L.</h2>
			<h3>PMID:21952365  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Khan AL(1), Hamayun M, Ahmad N, Hussain J, Kang SM, Kim YH, Adnan M, Tang DS, Waqas M, Radhakrishnan R, Hwang YH, Lee IJ.
				<br/> 
				年份：2011 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Endophytic fungi are little known for their role in gibberellins (GAs) synthesis and abiotic stress <span class="trait">resistance</span> in crop plants. We isolated 10 endophytes from the roots of field-grown soybean and screened their culture filtrates (CF) on the GAs biosynthesis mutant rice line - Waito-C. CF bioassay showed that endophyte <span class="gene">GMH-1B</span> significantly promoted the growth of Waito-C compared with controls. <span class="gene">GMH-1B</span> was identified as Penicillium minioluteum LHL09 on the basis of ITS regions rDNA sequence homology and phylogenetic analyses. GC/MS-SIM analysis of CF of P. minioluteum revealed the presence of bioactive GA(4) and GA(7). In endophyte-soybean plant interaction, P. minioluteum association significantly promoted growth characteristics (shoot length, shoot fresh and dry biomasses, chlorophyll content, and leaf area) and nitrogen assimilation, with and without sodium chloride (NaCl)-induced salinity (70 and 140 mM) stress, as compared with control. Field-emission scanning electron microcopy showed active colonization of endophyte with host plants before and after stress treatments. In response to salinity stress, low endogenous abscisic acid and high salicylic acid accumulation in endophyte-associated plants elucidated the stress mitigation by P. minioluteum. The endophytic fungal symbiosis of P. minioluteum also increased the daidzein and genistein contents in the soybean as compared with control plants, under <span class="trait">salt</span> stress. Thus, P. minioluteum ameliorated the adverse effects of abiotic salinity stress and rescued soybean plant growth by influencing biosynthesis of the plant's hormones and flavonoids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Global Analysis of <italic> WRKY </italic> Genes and Their Response to Dehydration and <span class="trait">Salt</span> Stress in Soybean</h2>
			<h3>PMID:26870047</h3>
				

				
					<h3>PMCID:4740950</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Hui </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/283284/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Pengfei </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hou </surname>  <given-names> Lei </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Shuzhen </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Chuanzhi </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/309031/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Han </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/295355/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Pengcheng </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Ye </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bian </surname>  <given-names> Xiaotong </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xingjun </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/49133/overview"/>  </contrib>
				<br/> 
				年份：<day> 07 </day>  <month> 1 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						WRKY proteins are plant specific transcription factors involved in various developmental and physiological processes, especially in biotic and abiotic stress <span class="trait">resistance.</span> Although previous studies suggested that WRKY proteins in soybean (Glycine max var. Williams 82) involved in both abiotic and biotic stress responses, the global information of WRKY proteins in the latest version of soybean genome (Wm82.a2v1) and their response to dehydration and <span class="trait">salt</span> stress have not been reported. In this study, we identified 176 <span class="gene">GmWRKY</span> proteins from soybean Wm82.a2v1 genome. These proteins could be classified into three groups, namely group I (32 proteins), group II (120 proteins), and group III (24 proteins). Our results showed that most <span class="gene">GmWRKY</span> genes were located on Chromosome 6, while chromosome 11, 12, and 20 contained the least number of this gene family. More <span class="gene">GmWRKY</span> genes were distributed on the ends of chromosomes to compare with other regions. The cis-acting elements analysis suggested that <span class="gene">GmWRKY</span> genes were transcriptionally regulated upon dehydration and <span class="trait">salt</span> stress. RNA-seq data analysis indicated that three <span class="gene">GmWRKY</span> genes responded negatively to dehydration, and 12 genes positively responded to <span class="trait">salt</span> stress at 1, 6, and 12 h, respectively. We confirmed by qRT-PCR that the expression of <span class="gene">GmWRKY47</span> and <span class="gene">GmWRKY</span> 58 genes was decreased upon dehydration, and the expression of <span class="gene">GmWRKY92,</span> 144 and 165 genes was increased under <span class="trait">salt</span> treatment.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> ), as one of the important protein and oil crop, is planted worldwide. Zhou et al. ( <xref rid="B74" ref-type="bibr"> 2008 </xref> ) identified 64 <italic> <span class="gene">GmWRKY</span> </italic> genes before the soybean genome was sequenced, and confirmed that <italic> <span class="gene">GmWRKY13,</span> 21 </italic> , and <italic> 54 </italic> genes were involved in abiotic stresses. Bencke-Malato et al. ( <xref rid="B3" ref-type="bibr"> 2014 </xref> ) identified 182 <italic> <span class="gene">GmWRKY</span> </italic> genes including 33 pseudogenes using the whole genome sequence information (Wm82.a1.v1; Schmutz et al., <xref rid="B45" ref-type="bibr"> 2010 </xref> ). Among the 149 non-pseudogenized <italic> <span class="gene">GmWRKY</span> </italic> genes, 72 genes were differentially expressed during fungal infection based on SAGE, RNA-seq and microarray experiments (Bencke-Malato et al., <xref rid="B3" ref-type="bibr"> 2014 </xref> ). However, the role of <italic> <span class="gene">GmWRKY</span> </italic> gene in dehydration and <span class="trait">salt</span> stress are largely unknown. We carried out a comprehensive analysis of <italic> <span class="gene">GmWRKY</span> </italic> genes base on the newly released genome Wm82.a2v1, and investigated their response to dehydration and <span class="trait">salt</span> stress. Totally, we identified 176 putative <italic> <span class="gene">GmWRKY</span> </italic> genes from soybean Wm82.a2v1 genome using bioinformatics approach. The chromosomal location, codon usage bias, <italic> cis </italic> -elements, and gene expression in different tissues and under dehydration and <span class="trait">salt</span> stress were analyzed. These results provided new insight into the roles of soybean <italic> WRKY </italic> genes in abiotic stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized data (Reads/kb/Million, RPKM) for six different tissues from different growth periods was reported by Severin et al. ( <xref rid="B46" ref-type="bibr"> 2010 </xref> ), and available from SoyBase website. A gene was considered expressed if the RPKM value was greater than or equal to two in an expression atlas (Belamkar et al., <xref rid="B2" ref-type="bibr"> 2014 </xref> ). The RPKM normalized read count data of expressed genes was log <sub> 2 </sub> -transformed and displayed in the form of heatmaps in R script. To survey the involvement of <italic> <span class="gene">GmWRKY</span> </italic> genes in dehydration and <span class="trait">salt</span> stress responses, transcriptome sequencing data of soybean under dehydration, and <span class="trait">salt</span> stress at three time points (1, 6, and 12 h) was downloaded from described by Belamkar et al. ( <xref rid="B2" ref-type="bibr"> 2014 </xref> ). The satisfied criteria of differentially expressed genes was as follows: (1) <italic> P </italic> -value adjusted for multiple testing correction using Benjamini and Hochberg method (Benjamini and Hochberg, <xref rid="B4" ref-type="bibr"> 1995 </xref> ) to be &#x0003c;0.05, (2) two fold or greater fold change, (3) residual variance quotients of both the control and treatment samples of &#x0003c;20.
					</p>
				
					<p>&nbsp;&nbsp;
						Publicly available ESTs were considered as a useful source for gene expression study (Ohlrogge and Benning, <xref rid="B37" ref-type="bibr"> 2000 </xref> ). A total of 1,468,526 soybean ESTs were downloaded from GenBank. A total of 127 <italic> <span class="gene">GmWRKY</span> </italic> genes were obtained from these ESTs which were generated from leaf, seed, and other tissues (Table <xref ref-type="supplementary-material" rid="SM3"> S3 </xref> ). These ESTs were sequenced from soybean plants under different stresses. We found that the expression of <italic> <span class="gene">GmWRKY4,</span> 5, 6, 9, 31, 46, 50, 56, 96, 106, 155 </italic> , and <italic> 160 </italic> genes was responsive to water deficit stress, and <italic> <span class="gene">GmWRKY10,</span> 49, 121 </italic> , and <italic> 155 </italic> genes showed altered expression in response to <span class="trait">salt</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						RNA-seq data was used to screen <italic> <span class="gene">GmWRKY</span> </italic> genes that are responsive to dehydration and <span class="trait">salt</span> stress. A total number of 31 and 65 <italic> <span class="gene">GmWRKY</span> </italic> genes are considered differentially expressed at least at one of the three time points under dehydration or NaCl treatment, respectively (Tables <xref ref-type="supplementary-material" rid="SM4"> S4 </xref> , <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). Dehydration induced down-regulation of most <italic> <span class="gene">GmWRKY</span> </italic> genes except <italic> <span class="gene">GmWRKY56,</span> 106, 120 </italic> , and <italic> 139 </italic> genes at 1 h (Table <xref ref-type="supplementary-material" rid="SM4"> S4 </xref> ). <italic> <span class="gene">GmWRKY47,</span> <span class="gene">GmWRKY58</span> </italic> , and <italic> <span class="gene">GmWRKY60</span> </italic> genes were differentially expressed at least at one of three time points under dehydration. These genes were significantly down-regulated (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ). NaCl treatment resulted in up-regulation of most <italic> <span class="gene">GmWRKY</span> </italic> genes (Figure <xref ref-type="fig" rid="F9"> 9 </xref> and Table <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). Twelve <italic> <span class="gene">GmWRKY</span> </italic> genes were differentially expressed at all three time points under <span class="trait">salt</span> (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ). Most of the differentially expressed genes belong to group II (dehydration: 17/31; <span class="trait">salt:</span> 45/65), and followed by group III (dehydration: 9/31; <span class="trait">salt:</span> 13/65). Less group I genes (dehydration: 5/31; <span class="trait">salt:</span> 7/65) were found responsive to <span class="trait">salt</span> and dehydration stresses. These results were consistent with results from cotton <italic> WRKY </italic> that most group II and III <italic> GhWRKY </italic> genes are highly expressed under stress condition (Dou et al., <xref rid="B12" ref-type="bibr"> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Different expression of <italic> <bold> <span class="gene">GmWRKY</span> </bold> </italic> genes in three time point under <span class="trait">salt</span> stress based on RNA-seq data </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						Ninety-five <italic> <span class="gene">GmWRKY</span> </italic> genes are differentially expressed under dehydration and/or <span class="trait">salt</span> stresses (Tables <xref ref-type="supplementary-material" rid="SM4"> S4 </xref> , <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). Removal of three genes ( <italic> <span class="gene">GmWRKY175,</span> <span class="gene">GmWRKY53</span> </italic> , and <italic> 155 </italic> ) with incomplete sequences or low quality sequences, we extracted 2000 bp promoter regions of 92 <italic> <span class="gene">GmWRKY</span> </italic> genes. Various types of <italic> cis </italic> -acting elements were detected in the promoter region of 92 <italic> WRKY </italic> genes, suggesting that the same type of <italic> <span class="gene">GmWRKY</span> </italic> could perform different functions. Most of W-box elements could be distributed in promoters of <italic> <span class="gene">GmWRKY</span> </italic> (Tables <xref ref-type="supplementary-material" rid="SM7"> S7 </xref> , <xref ref-type="supplementary-material" rid="SM8"> S8 </xref> ). Some <italic> WRKY </italic> genes, contained majority of W-box elements, are auto-regulated by itself and cross-regulated, indicating a self-feedback or mutual manipulation channel might exist among <italic> WRKY </italic> genes (Chi et al., <xref rid="B8" ref-type="bibr"> 2013 </xref> ). <italic> AtWRKY18, 40 </italic> , and <italic> 60 </italic> genes were reported to be self- and cross-regulated based on W-box elements (Yan et al., <xref rid="B65" ref-type="bibr"> 2013 </xref> ). Similarly, <italic> OsWRKY24, 53 </italic> , and <italic> 70 </italic> genes were predicted self- and cross-regulated according to the presence of W-box clusters in their promoters (Zhang et al., <xref rid="B71" ref-type="bibr"> 2015 </xref> ). Based on these reports, we speculated that the presence of W-box elements in <italic> <span class="gene">GmWRKY</span> </italic> gene might have the similar regulatory mode. ABRE and MBS elements that respond to dehydration or <span class="trait">salt</span> stress were distributed in promoter region of most <italic> <span class="gene">GmWRKY</span> </italic> genes (Tables <xref ref-type="supplementary-material" rid="SM7"> S7 </xref> , <xref ref-type="supplementary-material" rid="SM8"> S8 </xref> ). These results suggested that <italic> <span class="gene">GmWRKY</span> </italic> genes were transcriptionally regulated upon dehydration and <span class="trait">salt</span> stress.
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the expression patterns of <italic> <span class="gene">GmWRKY</span> </italic> genes revealed by RNA-seq, 15 <italic> <span class="gene">GmWRKY</span> </italic> genes were selected for expression analysis by qRT-PCR. The RNA-seq results showed that 3 and 12 <italic> <span class="gene">GmWRKY</span> </italic> genes were responsive to dehydration and <span class="trait">salt</span> stress at all three time points, respectively. The qRT-PCR results showed the expression of <italic> <span class="gene">GmWRKY47</span> </italic> and <italic> 58 </italic> genes was down-regulated at 6, 12, 24, and 48 h under dehydration stress (Figure <xref ref-type="fig" rid="F10"> 10 </xref> ). Expression of <italic> <span class="gene">GmWRKY60</span> </italic> gene was significantly increased at 6 and 24 h, but significantly suppressed at 48 h under dehydration stress. The results showed that <italic> <span class="gene">GmWRKY47</span> </italic> and <italic> 58 </italic> genes were consistent with RNA-seq data while <italic> <span class="gene">GmWRKY60</span> </italic> gene was not. The expression of <italic> <span class="gene">GmWRKY92,</span> 144 </italic> , and <italic> 165 </italic> genes was enhanced at all time points using qRT-PCR (Figure <xref ref-type="fig" rid="F11"> 11 </xref> ). The results were consistent with the RNA-seq data. Unexpectedly, the expression of remaining nine <italic> <span class="gene">GmWRKY</span> </italic> genes was not consistent with RNA-seq data. For example, the expression of <italic> <span class="gene">GmWRKY56,</span> 96 </italic> , and <italic> 106 </italic> genes was reduced at all four time points (Figure <xref ref-type="fig" rid="F11"> 11 </xref> ), but they showed up-regulation in RNA-seq analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression pattern of selected <italic> <bold> <span class="gene">GmWRKY</span> </bold> </italic> genes under <span class="trait">salt</span> stress </bold> . The Y-axis indicates the relative expression; X-axis (0, 6, 12, 24, and 48 h) indicates hours of <span class="trait">salt</span> treatment. The standard errors are plotted using vertical lines. <sup> * </sup> significant difference at <italic> P </italic> &#x0003c; 0.05, <sup> ** </sup> significant difference ( <italic> P </italic> &#x0003c; 0.01).
					</p>
				
					<p>&nbsp;&nbsp;
						Compared to other plants, soybean genome contained the highest number of WRKY members. The expansion of WRKY proteins in soybean might be due to the following reasons. (1) soybean genome experienced at least three rounds of whole genome duplication (WGD) events that could produce a large number of paralogs (Conant and Wolfe, <xref rid="B9" ref-type="bibr"> 2008 </xref> ). The dicotyledons, such as <italic> Arabidopsis </italic> , grape, soybean, and <italic> M. truncatula </italic> , share a general &#x0201c;gamma&#x0201d; genome triplication event about 117 million years ago (Mya; Schmutz et al., <xref rid="B45" ref-type="bibr"> 2010 </xref> ). Subsequently, soybean and <italic> Medicago </italic> experience a common legume-specific WGD event around 59 Mya, and soybean has undergone an additional glycine-specific genome duplication event approximately 13 Mya (Schmutz et al., <xref rid="B45" ref-type="bibr"> 2010 </xref> ). (2) The number of duplicated genes was mainly determined by segmental duplication events because genes generated by segmental duplicated events have more chance to be retained (Wang et al., <xref rid="B60" ref-type="bibr"> 2005 </xref> ). Tandem duplication events play an important role in generating new duplicated genes (Cannon et al., <xref rid="B5" ref-type="bibr"> 2004 </xref> ), whereas segmental duplication events may widely distribute duplicated genes across the genome (Baumgarten et al., <xref rid="B1" ref-type="bibr"> 2003 </xref> ). Segmental duplication events could result in lost of many functional redundant genes to avoid fitness cost (Song et al., <xref rid="B51" ref-type="bibr"> 2014 </xref> ). Yin et al. ( <xref rid="B68" ref-type="bibr"> 2013 </xref> ) found that <italic> <span class="gene">GmWRKY</span> </italic> genes were generated mainly through segmental duplication events, which may lead to neofunctionalization or subfunctionalization (Moore and Purugganan, <xref rid="B36" ref-type="bibr"> 2005 </xref> ). Gene duplication events could improve plant <span class="trait">resistance</span> to pathogens by allowing the functional diversification of genes (Moore and Purugganan, <xref rid="B36" ref-type="bibr"> 2005 </xref> ). It was reported that 75 <italic> <span class="gene">GmWRKY</span> </italic> genes were involved in response to fungal infection (Bencke-Malato et al., <xref rid="B3" ref-type="bibr"> 2014 </xref> ). (3) Positive selection play a key role in preserving duplicated genes, and can act at very early stage of gene duplication process (Moore and Purugganan, <xref rid="B36" ref-type="bibr"> 2005 </xref> ). Site model and branch-site model analysis showed that group I, IIc, IIe, and III <span class="gene">GmWRKY</span> underwent positive selection (Yin et al., <xref rid="B68" ref-type="bibr"> 2013 </xref> ). Positive selection promotes constant expansion of <span class="gene">GmWRKY.</span> Similarly, group IIc and III WRKY from eggplant and turkey berry were speculated to undergo positive selection (Yang et al., <xref rid="B66" ref-type="bibr"> 2015 </xref> ). In contrast, Group III WRKY from <italic> L. japonicus </italic> (Song et al., <xref rid="B51" ref-type="bibr"> 2014 </xref> ), <italic> M. truncatula </italic> (Song and Nan, <xref rid="B49" ref-type="bibr"> 2014 </xref> ), and <italic> Cucumis sativus </italic> (Ling et al., <xref rid="B28" ref-type="bibr"> 2011 </xref> ) appear to be under purifying selection. Purifying selection may generate genes with conserved functions or pseudogenization (Zhang, <xref rid="B70" ref-type="bibr"> 2003 </xref> ). The genome size and the number of WRKY family members are not necessarily correlated. For example, the genome size of soybean is 978 Mb containing 176 WRKY, while the genome size of <italic> Gossypium arboretum </italic> is 1746 Mb containing 109 WRKY. The genome size of <italic> G. arboreum </italic> (1746 Mb) was about three times greater than the <italic> Populus trichocarpa </italic> genome size (485 Mb). These two plants have approximately same number of WRKY (109 vs. 104; Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Stomata and roots were involved in plant responses to dehydration and <span class="trait">salt</span> stress (Song et al., <xref rid="B52" ref-type="bibr"> 2009 </xref> ; Chen et al., <xref rid="B7" ref-type="bibr"> 2012 </xref> ; Belamkar et al., <xref rid="B2" ref-type="bibr"> 2014 </xref> ). The RNA-seq data showed that 15 <italic> <span class="gene">GmWRKY</span> </italic> genes were differentially expressed under dehydration and <span class="trait">salt</span> stress in root. Zhou et al. ( <xref rid="B74" ref-type="bibr"> 2008 </xref> ) found that <italic> <span class="gene">GmWRKY54</span> </italic> genes confer tolerance to <span class="trait">salt</span> and <span class="trait">drought</span> stresses in <italic> Arabidopsis </italic> , possibly through the regulation of <italic> <span class="gene">DREB2A</span> </italic> and <italic> STZ/Zat10 </italic> genes. Heterologous expression of <italic> <span class="gene">GmWRKY13</span> </italic> genes could increase the sensitivity of <italic> Arabidopsis </italic> to <span class="trait">salt</span> tolerance (Zhou et al., <xref rid="B74" ref-type="bibr"> 2008 </xref> ). However, we failed to detect the response of these two genes to dehydration and <span class="trait">salt</span> stress. The possible explanation might be due to tissue-specific regulation; <italic> <span class="gene">GmWRKY13</span> </italic> and <italic> 54 </italic> genes were cloned from leaf, while the RNA-seq data were from root.
					</p>
				
					<p>&nbsp;&nbsp;
						Under dehydration stress, <italic> <span class="gene">GmWRKY47,</span> 58 </italic> , and <italic> 60 </italic> genes were considered as differentially expressed genes. Their orthologous genes in <italic> Arabidopsis </italic> are <italic> AtWRKY11, 41 </italic> , and <italic> 70 </italic> , respectively (Table <xref ref-type="supplementary-material" rid="SM2"> S2 </xref> ). <italic> <span class="gene">WRKY11</span> </italic> from <italic> Vitis vinifera </italic> is orthologous gene of <italic> AtWRKY11 </italic> . Transgenic <italic> Arabidopsis </italic> expressing <italic> VvWRKY11 </italic> showed higher tolerance to <span class="trait">drought</span> stress, indicating its involvement in response to dehydration stress (Liu et al., <xref rid="B30" ref-type="bibr"> 2011 </xref> ). Overexpression of <italic> GsWRKY20 </italic> (orthologous of <italic> AtWRKY 70 </italic> ) in <italic> Arabidopsis </italic> and <italic> Medicago sativa </italic> could increase <span class="trait">drought</span> tolerance of the transgenic <italic> Arabidopsis </italic> , and enhance <span class="trait">salt</span> and <span class="trait">drought</span> tolerance of transgenic <italic> Medicago </italic> (Luo et al., <xref rid="B33" ref-type="bibr"> 2013a </xref> ; Tang et al., <xref rid="B55" ref-type="bibr"> 2014 </xref> ). <italic> VvWRKY11 </italic> and <italic> GsWRKY20 </italic> promoted dehydration tolerance. However, our results from qRT-PCR and RNA-seq data showed that <italic> <span class="gene">GmWRKY47</span> </italic> ( <italic> AtWRKY11 </italic> ) and <italic> 60 </italic> ( <italic> AtWRKY70 </italic> ) genes were negatively regulated by dehydration. <italic> AtWRKY41 </italic> (orthologous of <italic> <span class="gene">GmWRKY58</span> </italic> ) could promote disease <span class="trait">resistance</span> (Higashi et al., <xref rid="B18" ref-type="bibr"> 2008 </xref> ). It could promote seed dormancy through regulation of <italic> ABI3 </italic> gene (Ding et al., <xref rid="B11" ref-type="bibr"> 2014 </xref> ). We first reported the observation that <italic> <span class="gene">GmWRKY58</span> </italic> gene was involved in dehydration response.
					</p>
				
					<p>&nbsp;&nbsp;
						Twelve <italic> <span class="gene">GmWRKY</span> </italic> genes were differentially expressed under <span class="trait">salt</span> stress. Their orthologous genes are <italic> AtWRKY6, 30, 40, 50, 51 </italic> , and <italic> 70 </italic> in <italic> Arabidopsis </italic> , respectively (Table <xref ref-type="supplementary-material" rid="SM2"> S2 </xref> ). The orthologous genes of <italic> AtWRKY40 </italic> are <italic> <span class="gene">GmWRKY17,</span> 56, 106, 139 </italic> , and <italic> 144 </italic> ; Both of <italic> <span class="gene">GmWRKY36</span> </italic> and <italic> 105 </italic> , which are orthologous genes of <italic> AtWRKY6 </italic> ; and <italic> <span class="gene">GmWRKY57</span> </italic> and <italic> 125 </italic> shared a common orthologous gene, <italic> AtWRKY70 </italic> . <italic> AtWRKY6 </italic> was identified as target gene of <italic> AtWRKY53 </italic> (Miao et al., <xref rid="B35" ref-type="bibr"> 2004 </xref> ). The expression of <italic> AtWRKY53 </italic> was up-regulated in <italic> Arabidopsis sos2 </italic> mutant under <span class="trait">salt</span> stress (Kamei et al., <xref rid="B23" ref-type="bibr"> 2005 </xref> ), indicating its involvement in <span class="trait">salt</span> tolerance. Scarpeci et al. ( <xref rid="B44" ref-type="bibr"> 2013 </xref> ) showed that overexpression of <italic> AtWRKY30 </italic> enhanced <span class="trait">salt</span> tolerance in <italic> Arabidopsis </italic> during early growth stages. ChIP experiments showed that <italic> AtWRKY40 </italic> directly targeted a number of known ABA-responsive genes, including <italic> ABI4, ABI5, ABF4, MYB2, DREB1A </italic> , and <italic> RAB18 </italic> genes (Shang et al., <xref rid="B47" ref-type="bibr"> 2010 </xref> ), indicating that <italic> AtWRKY40 </italic> could promote <span class="trait">salt</span> tolerance. Previous study showed that <italic> AtWRKY50 </italic> and <italic> 51 </italic> played crucial roles in jasmonic acid (JA) pathway (Chen et al., <xref rid="B7" ref-type="bibr"> 2012 </xref> ), which was a key component in pathogen tolerance. We found the expression of <italic> <span class="gene">GmWRKY92</span> </italic> (orthologous of <italic> AtWRKY51 </italic> ) and <italic> 165 </italic> (orthologous of <italic> AtWRKY50 </italic> ) was up-regulated under 100 mM NaCl treatment (Figure <xref ref-type="fig" rid="F11"> 11 </xref> ), indicating their involvement in multiple stress responses.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we identified 176 <span class="gene">GmWRKY</span> proteins in soybean genome (Wm82.a2v1) using bioinformatics approach. There are more WRKY proteins in soybean genome than other plant species. We found that no positive correlation exists between the genome size and the number of WRKY. Expression analysis showed that some <italic> <span class="gene">GmWRKY</span> </italic> genes were involved in response to dehydration and <span class="trait">salt</span> stress. Our results will be helpful for understanding the roles of <italic> WRKY </italic> gene family in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> <bold> <span class="gene">GmWRKY</span> </bold> </italic> genes differentially expressed under <span class="trait">salt</span> stress at 1, 6, and 12 h </bold> .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of a pepper <span class="gene">C2</span> domain-containing <span class="gene">SRC2</span> protein implicated in <span class="trait">resistance</span> against host and non-host pathogens and abiotic stresses.</h2>
			<h3>PMID:18204857  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim YC(1), Kim SY, Choi D, Ryu CM, Park JM.
				<br/> 
				年份：2008 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plants guard themselves against pathogen attack using multi-layered defense mechanism. Calcium represents an important secondary messenger during such defense responses. Upon examination of a pepper cDNA library, we observed that the gene CaSRC2-1 (Capsicum annum SRC2-1) was upregulated significantly in response to infection with the type II non-host pathogen Xanthomonas axonopodis pv. glycines 8 ra, which elicits a hypersensitive response. CaSRC2-1 encodes a protein that contains a <span class="gene">C2</span> domain and it exhibits a high degree of homology to the protein Soybean genes regulated by cold 2 (SRC2). However, little is known about how <span class="gene">SRC2</span> expression is elicited by biotic stresses such as pathogen challenge. Further sequence analysis indicated that the CaSRC2-1 <span class="gene">C2</span> domain is unique and contain certain amino acids that are conserved within the <span class="gene">C2</span> domains of other plants and animals. CaSRC2-1 transcription was up-regulated under both biotic and abiotic stress conditions, including bacterial and viral pathogen infection, CaCl(2) and cold treatment, but unaffected by treatment with plant defense-related chemicals such as salicylic acid, methyl jasmonic acid, ethephone, and abscisic acid. Intriguingly, under steady state conditions, CaSRC2-1 was expressed only in the root system. A CaSRC2-1-GFP fusion protein was used to determine localization to the plasma membrane. A fusion protein lacking the <span class="gene">C2</span> domain failed to target the membrane but remained in the cytoplasm, indicating that the <span class="gene">C2</span> domain plays a critical role in localization. Thus, CaSRC2-1 encodes a novel <span class="gene">C2</span> domain-containing protein that targets the plasma membrane and plays a critical role in the abiotic stress and defense responses of pepper plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsLRPK, a novel cold-activated leucine-rich repeat receptor-like protein kinase from Glycine soja, is a positive regulator to cold stress tolerance.</h2>
			<h3>PMID:24388511  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang L(1), Wu K(1), Gao P(2), Liu X(1), Li G(3), Wu Z(4).
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant LRR-RLKs serve as protein interaction platforms, and as regulatory modules of protein activation. Here, we report the isolation of a novel plant-specific <span class="gene">LRR-RLK</span> from Glycine soja (termed GsLRPK) by differential screening. GsLRPK expression was cold-inducible and shows Ser/Thr protein kinase activity. Subcellular localization studies using GFP fusion protein indicated that GsLRPK is localized in the plasma membrane. Real-time PCR analysis indicated that temperature, <span class="trait">salt,</span> <span class="trait">drought,</span> and ABA treatment can alter GsLRPK gene transcription in G. soja. However, just protein induced by cold stress not by salinity and ABA treatment in tobacco was found to possess kinase activity. Furthermore, we found that overexpression of GsLRPK in yeast and Arabidopsis can enhance <span class="trait">resistance</span> to cold stress and increase the expression of a number of cold responsive gene markers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sinorhizobium fredii HH103 cgs mutants are unable to nodulate determinate- and indeterminate nodule-forming legumes and overproduce an altered EPS.</h2>
			<h3>PMID:19348575  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Crespo-Rivas JC(1), Margaret I, Hidalgo A, Buendía-Clavería AM, Ollero FJ, López-Baena FJ, del Socorro Murdoch P, Rodríguez-Carvajal MA, Soria-Díaz ME, Reguera M, Lloret J, Sumpton DP, Mosely JA, Thomas-Oates JE, van Brussel AA, Gil-Serrano A, Vinardell JM, Ruiz-Sainz JE.
				<br/> 
				年份：2009 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sinorhizobium fredii HH103 produces cyclic beta glucans (CG) composed of 18 to 24 glucose residues without or with 1-phosphoglycerol as the only substituent. The S. fredii HH103-Rifr cgs gene (formerly known as ndvB) was sequenced and mutated with the lacZ-gentamicin <span class="trait">resistance</span> cassette. Mutant SVQ562 did not produce CG, was immobile, and grew more slowly in the hypoosmotic GYM medium, but its survival in distilled water was equal to that of HH103-Rifr. Lipopolysaccharides and K-antigen polysaccharides produced by SVQ562 were not apparently altered. SVQ562 overproduced exopolysaccharides (EPS) and its <span class="gene">exoA</span> gene was transcribed at higher levels than in HH103-Rifr. In GYM medium, the EPS produced by SVQ562 was of higher molecular weight and carried higher levels of substituents than that produced by HH103-Rifr. The expression of the SVQ562 cgsColon, two colonslacZ fusion was influenced by the pH and the osmolarity of the growth medium. The S. fredii cgs mutants SVQ561 (carrying cgs::Omega) and SVQ562 only formed pseudonodules on Glycine max (determinate nodules) and on Glycyrrhiza uralensis (indeterminate nodules). Although nodulation factors were detected in SVQ561 cultures, none of the cgs mutants induced any macroscopic response in Vigna unguiculata roots. Thus, the nodulation process induced by S. fredii cgs mutants is aborted at earlier stages in V. unguiculata than in Glycine max.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Ecophysiological Study of <span class="trait">Salt</span> Stress for Wild and Cultivated Soybean Species from the Yellow River Delta, China</h2>
			<h3>PMID:24999494</h3>
				
					<h3>PMCID:4066866</h3>
					<h4>期刊:The Scientific World Journal</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Gang </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 21 </day>  <month> 4 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Compared to the ion levels changing, the NMT gave us a more ocular and quicker detection, and the wild soybean exerted more outstanding superiority. Both in young and in mature seedlings, the Na <sup> + </sup> and Cl <sup> &#x02212; </sup> were ejected to the extracellular more quickly, and the H <sup> + </sup> was imported into cells faster in wild soybean ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> ). The K <sup> + </sup> on the other hand was discharged from cells more slowly, reflecting that the wild soybean had better abilities to maintain K <sup> + </sup> ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> ). These results were coinciding with the emulative absorptions of K <sup> + </sup> and Na <sup> + </sup> [ <xref rid="B15" ref-type="bibr"> 15 </xref> ]. Unlike animal, the plant cells make use of the H <sup> + </sup> -ATPases not the Na <sup> + </sup> -ATPases, which they do not have, to create a proton-motive force to drive the afflux of the Na <sup> + </sup> . Therefore, the enhanced H <sup> + </sup> influx in wild soybean in fact was an indirect reflection of its <span class="trait">salt</span> <span class="trait">resistance.</span> Under 300&#x02009;mM NaCl in specific, the flux of H <sup> + </sup> in cultivated soybean almost arrived at 0 ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> ) reflecting that the stress has produced serious damage, while the flux in wild soybean still maintained a steady rising trend to the contrary. The conditions of Cl <sup> &#x02212; </sup> were the same. So we can realize that the wild soybean had better <span class="trait">salt</span> tolerance, and the capability was more high light under high <span class="trait">salt</span> concentration. With the increasing H <sup> + </sup> entering the cells, more and more Na <sup> + </sup> was transferred out of the cells by H <sup> + </sup> /Na <sup> + </sup> at the same time, and the theory had already been elaborated in poplar [ <xref rid="B52" ref-type="bibr"> 52 </xref> ]. In addition, it has been reported that the activity of the plasma membrane (PM) Na <sup> + </sup> /H <sup> + </sup> antiporters can be induced by NaCl, and many related researches on transporters such as <span class="gene">SOS1</span> and NHXs have also acquired similar conclusions [ <xref rid="B53" ref-type="bibr"> 53 </xref> &#x02013; <xref rid="B63" ref-type="bibr"> 63 </xref> ]. For the mature seedlings, there was the same conclusion. The most obvious differences between them still emerged on 300&#x02009;mM. Nonetheless, the gap was not as large as young seedlings, especially for the flux of K <sup> + </sup> [ <xref rid="B59" ref-type="bibr"> 59 </xref> &#x02013; <xref rid="B63" ref-type="bibr"> 63 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Intersubgeneric hybridization between Glycine max and G. tomentella: production of F₁, amphidiploid, BC₁, BC₂, BC₃, and fertile soybean plants.</h2>
			<h3>PMID:25835560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Singh RJ(1), Nelson RL.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: This paper describes methods for unlocking genetic treasure from wild perennial Glycine species of Australia for soybean improvement. The genetic resources of the ca. 26 species of the genus Glycine subgenus Glycine have not been exploited to broaden the genetic base of soybean (Glycine max; 2n = 40). The objectives of this study were to develop methods for producing F1, amphidiploid, BC1, BC2, BC3, and fertile soybean plants from crosses of soybean and the genus Glycine subgenus Glycine species, in order to utilize the subgenus Glycine germplasm in soybean breeding. Soybean cultivars were hybridized with six accessions of 78-chromosome G. tomentella as well as one accession each of 40-chromosome G. tomentella, G. argyrea and G. latifolia. They were chosen because they exhibit <span class="trait">resistance</span> to soybean <span class="trait">rust.</span> We were successful in producing fertile soybean from soybean cv. 'Dwight' and 78-chromosome G. tomentella accession PI 441001, while other hybrids were discontinued either at F1 or amphidiploid stage. The F1 seeds aborted prior to reaching maturity, so developing seeds from 19 to 21 day old pods were cultured aseptically in various media formulations. Seed maturation and multiple embryo generation media were developed. F1 plants with shoots and roots (2n = 59) were transplanted to pots in greenhouse. Amphidiploid (2n = 118) plants were backcrossed to 'Dwight'. <span class="gene">BC1</span> (2n = 79) plants were propagated through in vitro and 43 mature BC2F1 seeds were harvested. Fifteen surviving BC2F1 plants were morphologically distinct, sterile, and had chromosome numbers ranging 2n = 56-59. Chromosome numbers of the BC3F1 plants ranged 2n = 40-49. Derived fertile soybeans were first planted in the field in 2008 and are being evaluated for <span class="trait">yield,</span> <span class="trait">resistance</span> to pathogens and pests and tolerance to <span class="trait">salt</span> through material transfer agreement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean Ferritin Expression in Saccharomyces cerevisiae Modulates Iron Accumulation and <span class="trait">Resistance</span> to Elevated Iron Concentrations.</h2>
			<h3>PMID:26969708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：de Llanos R(1), Martínez-Garay CA(1), Fita-Torró J(1), Romero AM(1), Martínez-Pastor MT(2), Puig S(3).
				<br/> 
				年份：2016 May 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fungi, including the yeast Saccharomyces cerevisiae, lack ferritin and use vacuoles as iron storage organelles. This work explored how plant ferritin expression influenced baker's yeast iron metabolism. Soybean seed ferritin <span class="gene">H1</span> (SFerH1) and SFerH2 genes were cloned and expressed in yeast cells. Both soybean ferritins assembled as multimeric complexes, which bound yeast intracellular iron in vivo and, consequently, induced the activation of the genes expressed during iron scarcity. Soybean ferritin protected yeast cells that lacked the Ccc1 vacuolar iron detoxification transporter from toxic iron levels by reducing cellular oxidation, thus allowing growth at high iron concentrations. Interestingly, when simultaneously expressed in ccc1Δ cells, SFerH1 and SFerH2 assembled as heteropolymers, which further increased iron <span class="trait">resistance</span> and reduced the oxidative stress produced by excess iron compared to ferritin homopolymer complexes. Finally, soybean ferritin expression led to increased iron accumulation in both wild-type and ccc1Δ yeast cells at certain environmental iron concentrations.IMPORTANCE: Iron deficiency is a worldwide nutritional disorder to which women and children are especially vulnerable. A common strategy to combat iron deficiency consists of dietary supplementation with inorganic iron salts, whose bioavailability is very low. Iron-enriched yeasts and cereals are alternative strategies to diminish iron deficiency. Animals and plants possess large ferritin complexes that accumulate, detoxify, or buffer excess cellular iron. However, the yeast Saccharomyces cerevisiae lacks ferritin and uses vacuoles as iron storage organelles. Here, we explored how soybean ferritin expression influenced yeast iron metabolism, confirming that yeasts that express soybean seed ferritin could be explored as a novel strategy to increase dietary iron absorption.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant antimicrobial peptides: an overview of SuperSAGE transcriptional profile and a functional review.</h2>
			<h3>PMID:20088771  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kido EA(1), Pandolfi V, Houllou-Kido LM, Andrade PP, Marcelino FC, Nepomuceno AL, Abdelnoor RV, Burnquist WL, Benko-Iseppon AM.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Defensin, thionin and lipid transfer protein (LTP) gene families, which antimicrobial activity has an attractive use in protein engineering and transgenic production of agronomical important plants, have been here functionally reviewed. Also, a transcriptional overview of a set of plant SuperSAGE libraries and analysis looking for 26 bp tags possibly annotated for those families is presented. Tags differentially expressed (p = 0.05) or constitutively transcribed were identified from leaves or roots SuperSAGE libraries from important Brazilian plant species [cowpea (Vigna unguiculata (L.) Walp.), soybean (Glycine max (L.) Merr.) and modern sugarcane hybrids (Saccharum spp.)] submitted to abiotic <span class="trait">[salt</span> (100 mM NaCl) or <span class="trait">drought]</span> or biotic stresses [fungus inoculation (Phakopsora pachyrhizi; Asiatic Soyben <span class="trait">Rust</span> phytopathogen)]. The diverse transcriptional patterns observed, probably related to the variable range of targets and functions involved, could be the first step to unravel the antimicrobial peptide world and the plant stress response relationship. Moreover, SuperSAGE opens the opportunity to find some SNPs or even rare transcript that could be important on plant stress <span class="trait">resistance</span> mechanisms. Putative defensin or <span class="gene">LTP</span> identified by SuperSAGE following a specific plant treatment or physiological condition could be useful for future use in genetic improvement of plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>