<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Sclerotinia sclerotiorum</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Sclerotinia sclerotiorum</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:The pathogenic development of <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> in soybean requires specific host NADPH oxidases.</h2>
			<h3>PMID:28378935</h3>
				

				
			<p>
				作者：Ranjan A(1), Jayaraman D(2), Grau C(1), Hill JH(3), Whitham SA(3), Ané JM(2)(4), Smith DL(1), Kabbage M(1).
				<br/> 
				年份：2017 Apr 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The plant membrane-localized NADPH oxidases, also known as respiratory burst oxidase homologues (RBOHs), play crucial roles in various cellular activities, including plant disease responses, and are a major source of reactive oxygen species (ROS). <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> is a cosmopolitan fungal pathogen that causes <span class="trait">Sclerotinia</span> stem rot (SSR) in soybean. Via a key virulence factor, oxalic acid, it induces programmed cell death (PCD) in the host plant, a process that is reliant on ROS generation. In this study, using protein sequence similarity searches, we identified 17 soybean RBOHs <span class="gene">(GmRBOHs)</span> and studied their contribution to SSR disease development, <span class="trait">drought</span> tolerance and nodulation. We clustered the soybean RBOH genes into six groups of orthologues based on phylogenetic analysis with their Arabidopsis counterparts. Transcript analysis of all 17 <span class="gene">GmRBOHs</span> revealed that, of the six identified groups, group VI <span class="gene">(GmRBOH-VI)</span> was specifically and drastically induced following S. <span class="trait">sclerotiorum</span> challenge. Virus-induced gene silencing (VIGS) of <span class="gene">GmRBOH-VI</span> using Bean pod mottle virus (BPMV) resulted in enhanced <span class="trait">resistance</span> to S. <span class="trait">sclerotiorum</span> and markedly reduced ROS levels during disease development. Coincidently, <span class="gene">GmRBOH-VI-silenced</span> plants were also found to be <span class="trait">drought</span> tolerant, but showed a reduced capacity to form nodules. Our results indicate that the pathogenic development of S. <span class="trait">sclerotiorum</span> in soybean requires the active participation of specific host RBOHs, to induce ROS and cell death, thus leading to the establishment of disease.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The complete genome sequence of a novel Fusarium graminearum RNA virus in a new proposed family within the order Tymovirales.</h2>
			<h3>PMID:27376377  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen X(1), He H(1), Yang X(1), Zeng H(1), Qiu D(1), Guo L(2).
				<br/> 
				年份：2016 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete nucleotide sequence of Fusarium graminearum deltaflexivirus 1 (FgDFV1), a novel positive single-stranded (+ss) RNA mycovirus, was sequenced and analyzed. The complete genome of FgDFV1/BJ59 was shown to be 8246 nucleotides (nt) long excluding the poly(A) tail. FgDFV1/BJ59 was predicted to contain a large open reading frame (ORF 1) and four smaller ORFs (2-5). <span class="gene">ORF1</span> encodes a putative replication-associated <span class="gene">polyprotein</span> (RP) of 2042 amino acids (aa) and contains three conserved domains, viral RNA methyltransferase (Mtr), viral RNA helicase (Hel) and RNA-dependent RNA polymerase (RdRp). ORFs 2-5 encode four putative small hypothetical proteins (12-18 kDa) with unknown biological functions. Phylogenetic analysis based on RP sequences indicated that FgDFV1 is phylogenetically related to soybean leaf-associated mycoflexivirus 1 (SlaMyfV1) and <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> deltaflexivirus 1 (SsDFV1), which form a well-supported and independent group belonging to a newly proposed family Deltaflexiviridae within the order Tymovirales. However, FgDFV1 is markedly different from SsDFV1 and SlaMyfV1 in genome organization and nucleotide sequence. FgDFV1 may represent an additional species in the new genus Deltaflexivirus or possibly a new genus in the proposed family Deltaflexiviridae.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Co-infection of a hypovirulent isolate of <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> with a new botybirnavirus and a strain of a mitovirus.</h2>
			<h3>PMID:27267756  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ran H(1), Liu L(1), Li B(1), Cheng J(1), Fu Y(1), Jiang D(1), Xie J(2).
				<br/> 
				年份：2016 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum,</span> a notorious plant fungal pathogen, causes <span class="trait">yield</span> loss of many crops and vegetables, and is a natural host of a diverse viruses with positive-sense RNA (+ssRNA), negative-sense RNA (-ssRNA), double-stranded RNA (dsRNA), or DNA genomes. Mixed-infection with multiple related or unrelated mycoviruses is a common phenomenon in S. <span class="trait">sclerotiorum.</span> However, a single strain co-infected with dsRNA and + ssRNA viruses has not been reported in S. <span class="trait">sclerotiorum.</span> RESULTS: We report two unrelated viruses, <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> botybirnavirus 2 (SsBRV2) with a bipartite dsRNA genome and <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> mitovirus 4 (SsMV4/AH16) with a + ssRNA genome, which were originally detected in a single hypovirulent strain AH16 of S. <span class="trait">sclerotiorum.</span> SsMV4/AH16 has a typical genome of mitovirus and is a strain of mitovirus SsMV4. The genome of SsBRV2 consists of two separated dsRNA segments. The large dsRNA segment is 6159 bp in length and only has a single open reading frame (ORF) encoding a putative 1868-aa <span class="gene">polyprotein</span> with a conserved RNA dependent RNA polymerase (RdRp) domain. The small dsRNA segment is 5872 bp in length and encodes a putative 1778-aa protein. Phylogenetic analysis using RdRp conserved domain sequences revealed that SsBRV2 is phylogenetically related to the previously reported three bipartite viruses SsBRV1, Botrytis porri RNA virus 1 (BpRV1), and soybean leaf-associated botybirnavirus 1 (SlaBRV1). Electron microscopy demonstrated that SsBRV2 forms rigid spherical virions with a diameter of approximately 40 nm in infected mycelia. The virion of SsBRV2 was successfully introduced into a virus-free strain, which provides conclusive evidence that SsBRV2 confers hypovirulence on phytopathogenic fungus S. <span class="trait">sclerotiorum.</span> CONCLUSIONS: A bisegmented dsRNA virus (SsBRV2/AH16) and a nonsegmented + ssRNA virus (SsMV4/AH16) were characterized in a hypovirulent strain AH16 of S. <span class="trait">sclerotiorum.</span> SsMV4/AH16 is a strain of a reported mitovirus, whereas SsBRV2 is a new botybirnavirus. SsBRV2 is the causal agent of hypovirulence on S. <span class="trait">sclerotiorum.</span> Our findings supplied a first evidence that a single S. <span class="trait">sclerotiorum</span> strain is co-infected by dsRNA and + ssRNA mycoviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loci and candidate gene identification for <span class="trait">resistance</span> to <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> in soybean (Glycine max L. Merr.) via association and linkage maps.</h2>
			<h3>PMID:25736370  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao X(1), Han Y, Li Y, Liu D, Sun M, Zhao Y, Lv C, Li D, Yang Z, Huang L, Teng W, Qiu L, Zheng H, Li W.
				<br/> 
				年份：2015 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean white mold (SWM), caused by <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> ((Lib.) W. Phillips), is currently considered to be the second most important cause of soybean <span class="trait">yield</span> loss due to disease. Research is needed to identify SWM-resistant germplasm and gain a better understanding of the genetic and molecular basis of SWM <span class="trait">resistance</span> in soybean. Stem pigmentation after treatment with oxaloacetic acid is an effective indicator of <span class="trait">resistance</span> to SWM. A total of 128 recombinant inbred lines (RILs) derived from a cross of 'Maple Arrow' (partial resistant to SWM) and 'Hefeng 25' (susceptible) and 330 diverse soybean cultivars were screened for the soluble pigment concentration of their stems, which were treated with oxalic acid. Four quantitative trait loci (QTLs) underlying soluble pigment concentration were detected by linkage mapping of the RILs. Three hundred and thirty soybean cultivars were sequenced using the whole-genome encompassing approach and 25 179 single-nucleotide polymorphisms (SNPs) were detected for the fine mapping of SWM <span class="trait">resistance</span> genes by genome-wide association studies. Three out of five SNP markers representing a linkage disequilibrium (LD) block and a single locus on chromosome 13 <span class="gene">(Gm13)</span> were significantly associated with the soluble pigment content of stems. Three more SNPs that represented three minor QTLs for the soluble pigment content of stems were identified on another three chromosomes by association mapping. A major locus with the largest effect on <span class="gene">Gm13</span> was found both by linkage and association mapping. Four potential candidate genes involved in disease response or the anthocyanin biosynthesis pathway were identified at the locus near the significant SNPs (&lt;60 kbp). The beneficial allele and candidate genes should be useful in soybean breeding for improving <span class="trait">resistance</span> to SWM.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:LIK1, A CERK1-Interacting Kinase, Regulates Plant Immune Responses in <italic> Arabidopsis </italic></h2>
			<h3>PMID:25036661</h3>
				

				
					<h3>PMCID:4103824</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Le </surname>  <given-names> Mi Ha </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Yangrong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xue-Cheng </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stacey </surname>  <given-names> Gary </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chitin, an integral component of the fungal cell wall, is one of the best-studied microbe-associated molecular patterns. Previous work identified a LysM receptor-like kinase (LysM-RLK1/CERK1) as the primary chitin receptor in Arabidopsis. In order to identify proteins that interact with CERK1, we conducted a yeast two-hybrid screen using the intracellular kinase domain of CERK1 as the bait. This screen identified 54 putative CERK1-interactors. Screening mutants defective in 43 of these interacting proteins identified only two, a calmodulin like protein (At3g10190) and a leucine-rich repeat receptor like kinase (At3g14840), which differed in their response to pathogen challenge. In the present work, we focused on characterizing the <span class="gene">LRR-RLK</span> gene where mutations altered responses to chitin elicitation. This <span class="gene">LRR-RLK</span> was named LysM RLK1-interacting kinase 1 (LIK1). The interaction between CERK1 and LIK1 was confirmed by co-immunoprecipitation using protoplasts and transgenic plants. In vitro experiments showed that LIK1 was directly phosphorylated by CERK1. In vivo phosphorylation assays showed that Col-0 wild-type plants have more phosphorylated LIK1 than cerk1 mutant plants, suggesting that LIK1 may be directly phosphorylated by CERK1. Lik1 mutant plants showed an enhanced response to both chitin and flagellin elicitors. In comparison to the wild-type plants, lik1 mutant plants were more resistant to the hemibiotrophic pathogen Pseudomonas syringae, but more susceptible to the necrotrophic pathogen <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum.</span> Consistent with the enhanced susceptibility to necrotrophs, lik1 mutants showed reduced expression of genes involved in jasmonic acid and ethylene signaling pathways. These data suggest that LIK1 directly interacts with CERK1 and regulates MAMP-triggered innate immunity.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In order to identify novel components that directly interact with AtCERK1, a yeast two-hybrid screen was used to identify 54 putative AtCERK1 interactors of which we were able to identify mutant lines in 43 genes. We initially screened these 43 lines by monitoring the accumulation of ROS in response to chitin elicitation and then subsequently for their pathogen response. This resulted in the identification of only two mutant lines that showed a significantly altered response to pathogen inoculation; that is mutations in At3g10190, encoding a calmodulin-like protein, and At3g14840, encoding a LRR-RLK. Since mutants lacking the calmodulin-like protein retained the ability to respond to chitin, albeit at a significantly lower level than the wild-type, we focused our attention on the <span class="gene">LRR-RLK</span> protein, LIK1, where mutations enhanced the response to chitin elicitaion. The interaction between AtCERK1 and LIK1 was confirmed by co-immunoprecipitation. An <italic> in vitro </italic> kinase assay showed that LIK1 has very low, but measurable kinase activity. Wild-type LIK1 protein, as well as a mutant form lacking kinase activity, were phosphorylated <italic> in vitro </italic> upon the addition of CERK1. Interestingly, <italic> lik1 </italic> mutant plants showed increased <span class="trait">resistance</span> to the hemibiotrophic pathogen, <italic> P.syringae </italic> pv. <italic> tomato </italic> DC3000, but increased susceptibility to the necrotrophic fungal pathogen, <italic> S. <span class="trait">sclerotiorum</span> </italic> . Collectively, these data suggest that LIK1 is part of the AtCERK1 receptor complex and negatively regulates chitin-induced immunity.
					</p>
				
					<p>&nbsp;&nbsp;
						Two parallel hormone pathways are known to mediate distinct mechanisms of defense against either hemibiotrophic or necrotrophic pathogens. The salicylic acid (SA)-dependent signaling pathway primarily targets biotrophs and hemibiotrophs; whereas the jasmonic acid (JA) and ethylene (ET)-dependent signaling pathways primarily mediate <span class="trait">resistance</span> to necrotrophic pathogens <xref rid="pone.0102245-Glazebrook1" ref-type="bibr"> [21] </xref> . As mentioned above, analysis of the <italic> lik1 </italic> mutant lines suggests that LIK1 normally suppresses the SA pathway (consistent with the greater <span class="trait">resistance</span> of the mutants to <italic> P. syringae </italic> ) and activates JA and/or ET pathways (consistent with the greater susceptibility of the mutants to <italic> S. <span class="trait">sclerotiorum</span> </italic> ). In order to test this hypothesis, the expression of key genes in the SA and JA/ET pathways was examined in wild-type and <italic> lik1 </italic> mutant plants either mock-treated or treated with chitooctaose. <xref ref-type="fig" rid="pone-0102245-g006"> Figure 6 </xref> shows that the expression of key JA/ET signaling pathway genes, <italic> JAR1 </italic> , <italic> ACO2 </italic> and <italic> COI1, </italic> as measured by quantitative real-time-PCR (qRT-PCR). Expression of <italic> JAR1 </italic> , a gene involved in formation of JA-amino acid conjugates <xref rid="pone.0102245-Staswick1" ref-type="bibr"> [22] </xref> , is low in <italic> lik1 </italic> mutants ( <xref ref-type="fig" rid="pone-0102245-g006"> Fig 6A </xref> ), indicating a suppression of the JA signaling pathway in the mutant. <italic> ACO2 </italic> encodes ACC oxidase 2, a vital enzyme involved in ET synthesis. In <italic> lik1 </italic> mutant plants, this gene is expressed at a consistently lower level relative to wild-type plants ( <xref ref-type="fig" rid="pone-0102245-g006"> Fig 6B </xref> ). <span class="gene">COI1</span> is a co-receptor for JA <xref rid="pone.0102245-Sheard1" ref-type="bibr"> [23] </xref> . In <italic> lik1 </italic> mutant plants, <italic> <span class="gene">COI1</span> </italic> expression was consistently lower than in the wild-type plants ( <xref ref-type="fig" rid="pone-0102245-g006"> Fig 6C </xref> ). In contrast to these results, measurements of gene expression of key SA pathway genes, such as <italic> PR1 </italic> , <italic> PR2 </italic> and <italic> SID2 </italic> , showed no effect of the <italic> lik1 </italic> gene mutations (data not shown). Therefore, the data support the postulated role of LIK1 as a positive regulator of the JA and ET pathways.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The <span class="gene">pgip</span> family in soybean and three other legume species: evidence for a birth-and-death model of evolution</h2>
			<h3>PMID:25034494</h3>
				

				
					<h3>PMCID:4115169</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Kalunke </surname>  <given-names> Raviraj M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> rkalunke@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Cenci </surname>  <given-names> Alberto </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> alberto.cenci@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Volpi </surname>  <given-names> Chiara </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> volpichiara1980@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> O&#x02019;Sullivan </surname>  <given-names> Donal M </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> d.m.osullivan@reading.ac.uk </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Sella </surname>  <given-names> Luca </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> luca.sella@unipd.it </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Favaron </surname>  <given-names> Francesco </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> francesco.favaron@unipd.it </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Cervone </surname>  <given-names> Felice </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> felice.cervone@uniroma1.it </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> De Lorenzo </surname>  <given-names> Giulia </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> giulia.delorenzo@uniroma1.it </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A9">  <name>  <surname> D&#x02019;Ovidio </surname>  <given-names> Renato </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dovidio@unitus.it </email>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 7 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Polygalacturonase-inhibiting proteins (PGIPs) are leucine-rich repeat (LRR) plant cell wall glycoproteins involved in plant immunity. They are typically encoded by gene families with a small number of gene copies whose evolutionary origin has been poorly investigated. Here we report the complete characterization of the full complement of the <span class="gene">pgip</span> family in soybean (Glycine max [L.] Merr.) and the characterization of the genomic region surrounding the <span class="gene">pgip</span> family in four legume species. RESULTS: BAC clone and genome sequence analyses showed that the soybean genome contains two <span class="gene">pgip</span> loci. Each locus is composed of three clustered genes that are induced following infection with the fungal pathogen <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> (Lib.) de Bary, and remnant sequences of <span class="gene">pgip</span> genes. The analyzed homeologous soybean genomic regions (about 126 Kb) that include the <span class="gene">pgip</span> loci are strongly conserved and this conservation extends also to the genomes of the legume species Phaseolus vulgaris L., Medicago truncatula Gaertn. and Cicer arietinum L., each containing a single <span class="gene">pgip</span> locus. Maximum likelihood-based gene trees suggest that the genes within the <span class="gene">pgip</span> clusters have independently undergone tandem duplication in each species. CONCLUSIONS: The paleopolyploid soybean genome contains two <span class="gene">pgip</span> loci comprised in large and highly conserved duplicated regions, which are also conserved in bean, M. truncatula and C. arietinum. The genomic features of these legume <span class="gene">pgip</span> families suggest that the forces driving the evolution of <span class="gene">pgip</span> genes follow the birth-and-death model, similar to that proposed for the evolution of <span class="trait">resistance</span> (R) genes of NBS-LRR-type.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						BAC clone and genome sequence analyses showed that the soybean genome contains two <italic> <span class="gene">pgip</span> </italic> loci. Each locus is composed of three clustered genes that are induced following infection with the fungal pathogen <italic> <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> </italic> (Lib.) de Bary, and remnant sequences of <italic> <span class="gene">pgip</span> </italic> genes. The analyzed homeologous soybean genomic regions (about 126 Kb) that include the <italic> <span class="gene">pgip</span> </italic> loci are strongly conserved and this conservation extends also to the genomes of the legume species <italic> Phaseolus vulgaris </italic> L., <italic> Medicago truncatula </italic> Gaertn. and <italic> Cicer arietinum </italic> L., each containing a single <italic> <span class="gene">pgip</span> </italic> locus. Maximum likelihood-based gene trees suggest that the genes within the <italic> <span class="gene">pgip</span> </italic> clusters have independently undergone tandem duplication in each species.
					</p>
				
					<p>&nbsp;&nbsp;
						The paleopolyploid soybean genome contains two <italic> <span class="gene">pgip</span> </italic> loci comprised in large and highly conserved duplicated regions, which are also conserved in bean, <italic> M. truncatula </italic> and <italic> C. arietinum </italic> . The genomic features of these legume <italic> <span class="gene">pgip</span> </italic> families suggest that the forces driving the evolution of <italic> <span class="gene">pgip</span> </italic> genes follow the birth-and-death model, similar to that proposed for the evolution of <span class="trait">resistance</span> (R) genes of NBS-LRR-type.
					</p>
				
					<p>&nbsp;&nbsp;
						We have previously reported variation in the expression pattern of <italic> <span class="gene">Gmpgip</span> </italic> genes ( <italic> <span class="gene">Gmpgip1</span> </italic> , <italic> <span class="gene">Gmpgip2</span> </italic> , <italic> <span class="gene">Gmpgip3</span> </italic> and <italic> <span class="gene">Gmpgip4</span> </italic> ) following infection of soybean hypocotyls with the necrotrophic fungal pathogen <italic> <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> </italic> [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]]. In this work, we extended the study to <italic> <span class="gene">Gmpgip5</span> </italic> and <italic> <span class="gene">Gmpgip7</span> </italic> . qRT-PCR analysis showed that <italic> <span class="gene">Gmpgip5</span> </italic> and <italic> <span class="gene">Gmpip7</span> </italic> are expressed in soybean hypocotyls and are induced following the infection with <italic> S. <span class="trait">sclerotiorum</span> </italic> . The basal transcript levels of both <italic> <span class="gene">Gmpgip5</span> </italic> and <italic> <span class="gene">Gmpgip7</span> </italic> are much lower than that of <italic> <span class="gene">Gmpgip3</span> </italic> , used as control because it is the most highly expressed <italic> <span class="gene">Gmpgip</span> </italic> gene in soybean hypocotyls [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]]; moreover, basal expression of <italic> <span class="gene">Gmpgip7</span> </italic> was higher than that of <italic> <span class="gene">Gmpgip5</span> </italic> (Table&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> ). Expression of <italic> <span class="gene">Gmpgip5</span> </italic> decreases during the first 24&#x000a0;hour post infection (hpi) with <italic> S. <span class="trait">sclerotiorum</span> </italic> , to greatly increase by more than 1000 fold at 48 hpi, the last time point analyzed, when the tissue is almost completely macerated (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). Upon infection, expression of <italic> <span class="gene">Gmpgip7</span> </italic> shows a moderate increase during the first 24 hpi to reach high levels at 48 hpi (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). <italic> <span class="gene">Gmpgip3</span> </italic> showed different kinetics of transcript accumulation, with a maximum of five fold increase at 24 hpi and no further increase at 48&#x000a0;h (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Time-course expression analysis </bold> <sup> <bold> a </bold> </sup> <bold> of </bold> <bold> <italic> <span class="gene">Gmpgip</span> </italic> </bold> <bold> genes in etiolated soybean hypocotyls infected with </bold> <bold> <italic> S </italic> </bold> <bold> . </bold> <bold> <italic> <span class="trait">sclerotiorum</span> </italic> </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Total protein extracts of <italic> N. benthamiana </italic> leaves infected with the PVX-Gmpgip7 were then used to test the inhibitory activity of <span class="gene">GmPGIP7</span> against the fungal PGs of <italic> S. <span class="trait">sclerotiorum</span> </italic> , <italic> Fusarium graminearum </italic> , <italic> Colletotrichum acutatum </italic> and <italic> Aspergillus niger </italic> (data not shown) <italic> . </italic> No inhibition was observed; on the contrary, all the PGs examined were inhibited by <span class="gene">GmPGIP3,</span> used as control (Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In the homeologous regions containing the <italic> <span class="gene">pgip</span> </italic> clusters, the striking conservation is interrupted only by the remarkable sequence variability in the intergenic regions between the <italic> <span class="gene">pgip</span> </italic> genes. This low sequence conservation includes the proximal 5&#x02032; flanking regions, suggesting a differential regulation of the different <italic> <span class="gene">pgip</span> </italic> genes. Indeed, the six <italic> <span class="gene">pgip</span> </italic> genes show variation in <italic> cis </italic> -acting elements known to regulate defense response genes, and their expression patterns following pathogen infection show clear differences. The two novel <italic> <span class="gene">pgip</span> </italic> genes, <italic> <span class="gene">Gmpgip5</span> </italic> and <italic> <span class="gene">Gmpgip7</span> </italic> , are poorly expressed in soybean hypocotyls but are strongly induced at late stages of infection with the fungal pathogen <italic> S. <span class="trait">sclerotiorum.</span> </italic> This expression pattern is similar to that of <italic> <span class="gene">Gmpgip2</span> </italic> and differs from that of <italic> <span class="gene">Gmpgip1,</span> <span class="gene">Gmpgip3</span> </italic> and <italic> <span class="gene">Gmpgip4</span> </italic> , which all show a more prompt induction following <italic> S. <span class="trait">sclerotiorum</span> </italic> infection [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]]. Diversification of gene regulation following fungal pathogen infections or stress stimuli has been reported for other <italic> <span class="gene">pgip</span> </italic> gene families, including those of <italic> Arabidopsis </italic> [[ <xref ref-type="bibr" rid="B3"> 3 </xref> ]], bean [[ <xref ref-type="bibr" rid="B16"> 16 </xref> ]] and <italic> B. napus </italic> [[ <xref ref-type="bibr" rid="B15"> 15 </xref> ]], and suggests adaptation against stresses. However, this possibility in soybean is still poorly supported at the protein level, since, at present, inhibition activity against fungal PGs has been shown <italic> in vitro </italic> only for the product encoded by <italic> <span class="gene">Gmpgip3</span> </italic> , which is also the most expressed soybean family member [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]]. In fact, although <span class="gene">GmPGIP1,</span> <span class="gene">GmPGIP2</span> and <span class="gene">GmPGIP4</span> [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]] and now also <span class="gene">GmPGIP7</span> were expressed in <italic> N. benthamiana </italic> using PVX as a vector, they did not show any inhibitory activity. The lack of <italic> in vitro </italic> inhibition activity of <span class="gene">GmPGIPs</span> does not exclude the possibility that they can inhibit PGs from different sources not yet examined, or that only the <italic> in planta </italic> environment provides a suitable context to support the interaction with PGs. This last possibility has been suggested by Joubert et al. [[ <xref ref-type="bibr" rid="B6"> 6 </xref> ]] who found a reduction of symptoms caused by the activity of <italic> Botrytis cinerea </italic> BcPG2 on plant tissue when co-infiltrated with <italic> Vitis vinifera </italic> VvPGIP1, although no interaction between VvPGIP1 and BcPG2 was detected <italic> in vitro </italic> . Moreover, as suggested previously [[ <xref ref-type="bibr" rid="B18"> 18 </xref> ]], the lack of inhibition activity by <span class="gene">GmPGIP1,</span> <span class="gene">GmPGIP2</span> <span class="gene">GmPGIP4</span> and now <span class="gene">GmPGIP7</span> towards fungal PGs may reflect a different physiological role <italic> in planta </italic> . This possibility is supported by several observations in different plant species. For example, OsFOR1, a rice protein possessing PG inhibiting capabilities, affects the formation and/or maintenance of floral organ primordia in rice [[ <xref ref-type="bibr" rid="B36"> 36 </xref> ]]; levels of PGIP1 determine the timing of radicle protrusion in Arabidopsis [[ <xref ref-type="bibr" rid="B37"> 37 </xref> ]], and <italic> Vvpgip1 </italic> can affect gene expression and cell wall structure in transgenic tobacco plants [[ <xref ref-type="bibr" rid="B38"> 38 </xref> ],[ <xref ref-type="bibr" rid="B39"> 39 </xref> ]].
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Western blot of total protein extract from </bold> <bold> <italic> N. benthamiana </italic> </bold> <bold> plants inoculated with PVX-pgip constructs and agarose diffusion assay for PGIP inhibition. </bold> A) Western blot analysis was performed using total protein extract from <italic> N. benthamiana </italic> plants inoculated with individual PVX 201-based constructs for the expression of <span class="gene">GmPGIP3</span> or <span class="gene">GmPGIP7</span> or the empty vector, as a control. 1, protein ladder; 2, PVX 201 (empty vector); 3, <span class="gene">GmPGIP7</span> (5&#x000a0;&#x003bc;g); 4, <span class="gene">GmPGIP7</span> (10&#x000a0;&#x003bc;g); 5, <span class="gene">GmPGIP7</span> (20&#x000a0;&#x003bc;g); 6, <span class="gene">GmPGIP3</span> (10&#x000a0;&#x003bc;g). B) Agarose diffusion assay using crude protein extract from <italic> N. benthamiana </italic> plants inoculated with the PVX-Gmpgip3 or PVX-Gmpgip7 constructs or the empty vector, as a control. The assay was performed using 0.011 reducing units of <italic> S. <span class="trait">sclerotiorum</span> </italic> endopolygalacturonase (SsPG). The absence of halo indicates the inhibition of PG activity. 1, SsPG; 2, SsPG plus <span class="gene">GmPGIP3</span> (1&#x000a0;&#x003bc;g); 3, SsPG plus boiled <span class="gene">GmPGIP</span> (1&#x000a0;&#x003bc;g); 4, SsPG plus <span class="gene">GmPGIP7</span> (20&#x000a0;&#x003bc;g); 5, SsPG plus boiled <span class="gene">GmPGIP7</span> (20&#x000a0;&#x003bc;g); 6, SsPG plus empty PVX 201 vector (20&#x000a0;&#x003bc;g); 7, SsPG plus boiled empty PVX 201 vector (20&#x000a0;&#x003bc;g). Similar results were obtained with the PG of <italic> F. graminearum </italic> , <italic> C. acutatum </italic> and <italic> A. niger. </italic> <span class="gene">GmPGIP3</span> inhibited to completion all four PGs, whereas <span class="gene">GmPGIP7</span> did not show any inhibition activity (data not shown).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Suppression of <span class="trait">sclerotinia</span> stem rot of soybean by lactofen herbicide treatment.</h2>
			<h3>PMID:18944696</h3>
				

				
			<p>
				作者：Dann EK, Diers BW, Hammerschmidt R.
				<br/> 
				年份：1999 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						ABSTRACT Severity of <span class="trait">Sclerotinia</span> stem rot of soybean after treatment with lactofen (Cobra) and other herbicides was assessed in field experiments conducted in Michigan from 1995 to 1997. At sites where disease pressure was high, disease severity was reduced 40 to 60% compared with controls when lactofen was applied at the <span class="gene">V3</span> (1995 and 1996) or R1 (1997) growth stages. Corresponding seed yields were unchanged or up to 20% greater when lactofen was applied at the R1 stage in 1997. Disease severity was not reduced by lactofen treatments in years and at sites where disease pressure was low to medium, and corresponding yields often were reduced by 10%. High levels of glyceollin accumulated in lactofen-injured leaves collected from field plots in 1996 and 1997. High glyceollin content in lactofen-treated leaves was associated with significant reductions in lesion size when leaves were challenge-inoculated with <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The characterization of the soybean polygalacturonase-inhibiting proteins (Pgip) gene family reveals that a single member is responsible for the activity detected in soybean tissues.</h2>
			<h3>PMID:16501991  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：D'Ovidio R(1), Roberti S, Di Giovanni M, Capodicasa C, Melaragni M, Sella L, Tosi P, Favaron F.
				<br/> 
				年份：2006 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polygalacturonase-inhibiting proteins (PGIPs) are leucine-rich repeat (LRR) proteins that inhibit fungal endopolygalacturonases (PGs). They are encoded by multigene families whose members show functional redundancy and subfunctionalization for recognition of fungal PGs. In order to expand the information on the structure and functional features of legume PGIP, we have isolated and characterized four members of the soybean Pgip gene family and determined the properties of the encoded protein products. Sequence analysis showed that these genes form two clusters: one cluster of about 5 kbp containing <span class="gene">Gmpgip1</span> and <span class="gene">Gmpgip2,</span> and the other containing <span class="gene">Gmpgip3</span> and <span class="gene">Gmpgip4</span> within a 60 kb fragment of a separate BAC clone. Sequence diversification of the four members resides mainly in the xxLxLxx region that includes residues forming the beta-sheet B1. When compared with other legume Pgip genes, <span class="gene">Gmpgip3</span> groups with the bean genes Pvpgip1 and Pvpgip2, suggesting that these genes are closer to the ancestral gene. At the protein level, only <span class="gene">GmPGIP3</span> shows the capability to inhibit fungal PGs. The spectrum of inhibition of <span class="gene">GmPGIP3</span> against eight different fungal PGs mirrors that of the PGIP purified from soybean tissues and is similar to that of the bean PvPGIP2, one of the most efficient inhibitors so far characterized. We also report that the four <span class="gene">Gmpgip</span> genes are differentially regulated after wounding or during infection with the fungal pathogen <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum.</span> Following fungal infection <span class="gene">Gmpgip3</span> is up regulated promptly, while <span class="gene">Gmpgip2</span> is delayed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Relationships among endo-polygalacturonase, oxalate, pH, and plant polygalacturonase-inhibiting protein (PGIP) in the interaction between <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> and soybean.</h2>
			<h3>PMID:15597746  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Favaron F(1), Sella L, D'Ovidio R.
				<br/> 
				年份：2004 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The necrotrophic fungal pathogen <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> secretes oxalic acid and endo-polygalacturonase (endo-PG) in host plants. Oxalic acid acidifies the plant tissue to values more suitable to endo-PG activity. However, we observed that the infected soybean seedlings possessed a pH of 3.8, which is below that optimal for endo-PG activity (4.5 to 5.0). We investigated, therefore, the effects of pH (from 5.0 to 3.6) and oxalate (5 to 20 mM) on the activity of the major basic endo-PG (PGb) and towards an acidic endo-PG (PGa) secreted by S. <span class="trait">sclerotiorum</span> during soybean infection. We verified that only PGb activity is stimulated by oxalate, while at the lowest pH levels, PGa escapes the inhibition of a soybean polygalacturonase-inhibiting protein (PGIP). These results, performed on polygalacturonic acid, were apparently consistent with data obtained from studies with soybean hypocotyl segments, in which PGb activity was increased by oxalate and PGa maintained its activity also at pH 3.6, possibly because at this pH the PGIP contained in the plant tissue is inactive. Reverse transcription-polymerase chain reaction analysis showed that, during soybean infection, the expression of the putative <span class="gene">pga</span> gene is delayed in comparison to the basic one. The different temporal expressions of the two endo-PGs and their differing responses to pH, oxalate, and PGIP seem to be consistent with a possible maximization of the fungal PG activity in the host tissue.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Meta-QTL for <span class="trait">resistance</span> to white mold in common bean</h2>
			<h3>PMID:28199342</h3>
				
					<h3>PMCID:5310892</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vasconcellos </surname>  <given-names> Renato C. C. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oraguzie </surname>  <given-names> O. Blessing </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soler </surname>  <given-names> Alvaro </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Arkwazee </surname>  <given-names> Haidar </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Myers </surname>  <given-names> James R. </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferreira </surname>  <given-names> Juan J. </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClean </surname>  <given-names> Phil </given-names>  </name>  <xref ref-type="aff" rid="aff007">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Miklas </surname>  <given-names> Phillip N. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						White mold, caused by the fungus <italic> <span class="trait">Sclerotinia</span> <span class="trait">sclerotiorum</span> </italic> (Lib.) de Bary, is a major disease that limits common bean production and quality worldwide. The host-pathogen interaction is complex, with partial <span class="trait">resistance</span> in the host inherited as a quantitative trait with low to moderate heritability. Our objective was to identify meta-QTL conditioning partial <span class="trait">resistance</span> to white mold from individual QTL identified across multiple populations and environments. The physical positions for 37 individual QTL were identified across 14 recombinant inbred bi-parental populations (six new, three re-genotyped, and five from the literature). A meta-QTL analysis of the 37 QTL was conducted using the genetic linkage map of Stampede x Red Hawk population as the reference. The 37 QTL condensed into 17 named loci (12 previously named and five new) of which nine were defined as meta-QTL WM1.1, WM2.2, WM3.1, WM5.4, WM6.2, WM7.1, WM7.4, WM7.5, and WM8.3. The nine meta-QTL had confidence intervals ranging from 0.65 to 9.41 Mb. Candidate genes shown to express under <italic> S </italic> . <italic> <span class="trait">sclerotiorum</span> </italic> infection in other studies, including cell wall receptor kinase, <italic> <span class="gene">COI1</span> </italic> , ethylene responsive transcription factor, peroxidase, and MYB transcription factor, were found within the confidence interval for five of the meta-QTL. The nine meta-QTL are recommended as potential targets for MAS for partial <span class="trait">resistance</span> to white mold in common bean.
					</p>
				
					<p>&nbsp;&nbsp;
						A single QTL WM1.2 for straw test <span class="trait">resistance</span> was observed in the O12 population at the 37.65&#x02013;40.83 Mb interval (Fig M in <xref ref-type="supplementary-material" rid="pone.0171685.s001"> S1 File </xref> ). This location better coincides with the proximal orientation for the WM1.2 QTL from the GC population and occurs outside the <span class="gene">CI</span> for the WM1.1 QTL. A new QTL WM1.3 conditioning straw test <span class="trait">resistance</span> was identified in AO population in a distinct location (17.30&#x02013;13.81 Mb) from the previous named QTL on Pv01.
					</p>
				
					<p>&nbsp;&nbsp;
						Another candidate gene within WM1.1 is Phvul.001G240400 (50.03 Mb) which is homologous with a gene in <italic> A </italic> . <italic> thaliana </italic> that encodes a coronatine-insensitive protein 1 ( <italic> <span class="gene">COI1</span> </italic> ) which is a jasmonate receptor known to facilitate plant defense responses against necrotrophic pathogens. Loss of function mutant plants of <italic> A </italic> . <italic> thaliana </italic> that didn`t expressed <italic> <span class="gene">COI1</span> </italic> showed increased susceptibility to <italic> S </italic> . <italic> <span class="trait">sclerotiorum</span> </italic> and other necrotrophic pathogens [ <xref rid="pone.0171685.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0171685.ref056" ref-type="bibr"> 56 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although WM3.1 is associated with disease avoidance traits and the <span class="gene">CI</span> is seemingly too wide to effectively parse through hundreds of genes, there is one candidate gene, Phvul.003G164600 (37.29 Mb), found to encode a peroxidase that should be mentioned. Peroxidases are ubiquitous enzymes present in plants, animals and microorganisms. In plants, peroxidases are involved in physiological activities such as growth and development, through formation of lignin or oxidative carboxylation of indole-3-acetic acid (IAA) required for auxin metabolism [ <xref rid="pone.0171685.ref065" ref-type="bibr"> 65 </xref> , <xref rid="pone.0171685.ref066" ref-type="bibr"> 66 </xref> ]. Additionally, peroxidases play a crucial role in the generation of reactive oxygen species (ROS) during biotic and abiotic stress. Upon pathogen attack, several peroxidase genes are induced leading to over accumulation of ROS in plants which in turn causes oxidation of membranes and proteins, nucleic acid damage, onset of hypersensitive response and downstream signaling pathways, that could restrict pathogen spread from the site of infection [ <xref rid="pone.0171685.ref065" ref-type="bibr"> 65 </xref> ]. Furthermore, up regulation of peroxidase genes lead to cell wall fortification through formation of lignin and suberin which reinforce the plant cell wall [ <xref rid="pone.0171685.ref067" ref-type="bibr"> 67 </xref> ]. Leite et al. [ <xref rid="pone.0171685.ref068" ref-type="bibr"> 68 </xref> ] showed that common bean lines that were more resistant to <italic> S </italic> . <italic> <span class="trait">sclerotiorum</span> </italic> also had a more intense activity of peroxidases.
					</p>
				
					<p>&nbsp;&nbsp;
						The WM5.1 QTL in PX population is near RFLP D1157 (36.95 Mb) in the consensus map [ <xref rid="pone.0171685.ref005" ref-type="bibr"> 5 </xref> ] which is outside the <span class="gene">CI</span> for WM5.4 and between the QTL (39.83&#x02013;40.00 Mb) conditioning straw test <span class="trait">resistance</span> identified in AO. This QTL in AO is named WM5.5.
					</p>
				
					<p>&nbsp;&nbsp;
						Using a sequence based introgression mapping approach [ <xref rid="pone.0171685.ref016" ref-type="bibr"> 16 </xref> ] the WM8.3 interval was significantly narrowed to 2.36 Mb (44.35&#x02013;46.71 Mb). The WM8.3 meta-QTL has a slightly broader <span class="gene">CI</span> (40.62&#x02013;46.60 Mb), but nonetheless it validates the narrowed location for the WM8.3 QTL [ <xref rid="pone.0171685.ref016" ref-type="bibr"> 16 </xref> ]. Mamidi et al. [ <xref rid="pone.0171685.ref016" ref-type="bibr"> 16 </xref> ] suggested Phvul.008G173600 (45.62 Mb) as a candidate gene for WM8.3. This gene is a common bean receptor-like protein (RLP) which can recognize external signals and transmit a response signal. Some <italic> Arabidopsis </italic> RLPs are involved in disease defense [ <xref rid="pone.0171685.ref074" ref-type="bibr"> 74 </xref> ], and one RLP, <italic> AtRLP30 </italic> , is known to mediate <span class="trait">resistance</span> to <italic> <span class="trait">Sclerotinia</span> </italic> [ <xref rid="pone.0171685.ref075" ref-type="bibr"> 75 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Six new RIL populations were mapped in this study. The O83 RIL population was developed to verify WM7.3 and introgress it into susceptible great northern bean. The R31-83 parent of O83 population is a RIL with straw test <span class="trait">resistance</span> from the R31 population. The WM7.3 QTL was originally identified in the R31 population with a large straw test effect (51 PVE-percent phenotypic variation explained) [ <xref rid="pone.0171685.ref005" ref-type="bibr"> 5 </xref> ]. Validation of WM7.3, though, failed as it was not detected in the R31 (re-genotyped) or O83 populations. Instead, the WM7.5 was detected as in R31 as a large effect straw test QTL (36 PVE) at the proximal end of the chromosome (2.67&#x02013;4.62 Mb). For O83, the large effect straw test QTL (26 PVE) on Pv07 was derived from the susceptible parent Orion. It mapped (9.33&#x02013;11.44 Mb) adjacent WM7.5 but to a different location and was named a new QTL&#x02013;WM7.6. Although, our effort to transfer and validate the WM7.3 QTL failed, a narrower <span class="gene">CI</span> for the WM7.5 QTL from R31, will facilitate another attempt to transfer this QTL with similar strong straw test effect into susceptible great northern or pinto bean using I9365-31 instead of a RIL as the source of the QTL. The importance of WM7.5 is further supported by its identification as a meta-QTL due to detection in the straw test in the M25 population.
					</p>
				
					<p>&nbsp;&nbsp;
						The I9365-25 parent of M25 is similarly derived from an interspecific <italic> P </italic> . <italic> vulgaris </italic> x <italic> P </italic> . <italic> coccineus </italic> population as is the I9365-31 parent in the R31 population. This parentage supports the conclusion that WM5.4 and WM7.5, detected in both populations, are derived from <italic> P </italic> . <italic> coccineus </italic> . The U25 population also has I9365-25 as a parent, but only one QTL WM5.4 is found in both U25 and M25 populations. The detection of only a single common QTL between mapping populations with a common resistant parent attests to the complexity of the partial <span class="trait">resistance</span> response to white mold and reinforces the continuous need to validate QTL that are identified in only one population or environment. The WM7.1 QTL detected in M25 RIL population does not necessarily fit within the Andean origin for this QTL, but it has a relatively wide <span class="gene">CI</span> (4.62&#x02013;15.77 Mb) which upon fine mapping may shift position of the QTL into the <span class="gene">CI</span> for WM7.5 or WM7.6. The proximal end of Pv07 in general is an important genomic region for white mold <span class="trait">resistance</span> because meta-QTL WM7.1 (4.59&#x02013;6.38 Mb) and WM7.5 (1.81&#x02013;4.06 Mb) are closely linked. The tight linkage in repulsion represents a challenge to breeders trying to combine these QTL, a challenge that is more manageable with robust markers.
					</p>
				
					<p>&nbsp;&nbsp;
						New genomic resources in common bean enabled us to compare 37 individual QTL conditioning partial <span class="trait">resistance</span> to white mold from across 14 RIL populations. Knowledge of the physical position and interval for the 37 QTL facilitated the identification of nine meta-QTL and verifies the complexity of the quantitative <span class="trait">resistance</span> response to <italic> S </italic> . <italic> <span class="trait">sclerotiorum</span> </italic> in common bean. Breeders have struggled using MAS for QTL conferring partial <span class="trait">resistance</span> in the past because sparse maps and unknown physical positions contributed to the selection of large genomic intervals leading to negative linkage drag for <span class="trait">yield</span> and other traits. Moreover, the derivation of the <span class="trait">resistance</span> from exotic sources and wide crosses such as Middle American x Andean, dry bean x snap bean, and cultivated x landrace or wild, contributes to negative linkage drag effects on <span class="trait">yield</span> and other traits. To overcome these constraints it is important to identify stable QTL by continuously validating and characterizing the effects of the individual QTL in more populations and environments, and by different methods. Verified meta-QTL should then be fine mapped using near-isogenic populations or other approaches to develop MAS for a smaller genomic interval, and to facilitate the search for the actual gene(s) underpinning the <span class="trait">resistance.</span> Viable candidate genes (cell wall receptor kinase, coronatine-insensitive protein 1&#x02014; <italic> <span class="gene">COI1</span> </italic> , ethylene responsive transcription factor, peroxidase, and MYB transcription factor) with differential expression under <italic> S </italic> . <italic> <span class="trait">sclerotiorum</span> </italic> infection in other studies were found within the narrower <span class="gene">CI</span> for five of the nine meta-QTL identified in this study. Going forward, the nine meta-QTL provide targets for further characterization and manipulation by breeders. As new information is incorporated, the genomic confidence interval for the individual meta-QTL can be refined and their utility for breeding reassessed.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>