<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed Number</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed Number</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:The phytochrome gene family in soybean and a dominant negative effect of a soybean <span class="gene">PHYA</span> transgene on endogenous Arabidopsis PHYA.</h2>
			<h3>PMID:24013793  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu FQ(1), Fan CM, Zhang XM, Fu YF.
				<br/> 
				年份：2013 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The evolutionary origin of the phytochrome genes in soybean was analyzed. The expression profiles of <span class="gene">PHYA</span> paralogs were characterized. The heterologous expression of <span class="gene">GmPHYA1</span> in Arabidopsis resulted in longer hypocotyls. The phytochromes (PHY) are a small family of red/far-red light photoreceptors which regulate a <span class="trait">number</span> of important developmental responses in plants. So far, the members of the PHY gene family in soybean (Glycine max) remain unclear and an understanding of each member's physiological functions is limited. Our present in silico analysis revealed that the soybean genome harbors four PHYA, two <span class="gene">PHYB</span> and two PHYE, totally four pairs of eight PHY loci. The phylogenetic analysis suggested that the four PHY paralogous pairs originated from the latest round of genome duplication (~13 million years ago) and the four copies of <span class="gene">PHYA</span> were remnants of the two rounds of genome duplication (~58 and ~13 million years ago). A possible evolutionary history of <span class="gene">PHYA</span> homologs in the three legume species (soybean, Medicago truncatula, and Lotus japonicus) was proposed and the fate of duplicate soybean <span class="gene">PHYA</span> genes following polyploidization was discussed. The expression profiles of a soybean <span class="gene">PHYA</span> paralogous pair <span class="gene">(GmPHYA1</span> and <span class="gene">GmPHYA2)</span> showed that the transcript abundance was highest in the aerial organs of young plants. The physiological role of <span class="gene">GmPHYA1</span> was explored by observing the de-etiolation phenotype of transgenic Arabidopsis plants constitutively expressing <span class="gene">GmPHYA1.</span> The <span class="gene">GmPHYA1</span> protein interfered with the function of endogenous <span class="gene">PHYA</span> with respect to de-etiolation in a dominant negative manner when exogenously expressed in Arabidopsis. The elucidation of the PHY gene family members in soybean provide us with a general description and understanding of the photoreceptor gene family in this important crop plant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the <span class="gene">FAD2</span> Gene Family in Soybean Reveals the Limitations of Gel-Based TILLING in Genes with High Copy <span class="trait">Number.</span></h2>
			<h3>PMID:28348573</h3>
				

				
			<p>
				作者：Lakhssassi N(1), Zhou Z(1), Liu S(1), Colantonio V(1), AbuGhazaleh A(2), Meksem K(1).
				<br/> 
				年份：2017 Mar 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> oil typically contains 18-20% oleic acid. Increasing the content of oleic acid is beneficial for health and biodiesel production. Mutations in <span class="gene">FAD2-1</span> genes have been reported to increase <span class="trait">seed</span> oleic acid content. A subset of 1,037 mutant families from a mutagenized soybean cultivar (cv.) Forrest population was screened using reverse genetics (TILLING) to identify mutations within <span class="gene">FAD2</span> genes. Although no fad2 mutants were identified using gel-based TILLING, four fad2-1A and one fad2-1B mutants were identified to have high <span class="trait">seed</span> oleic acid content using forward genetic screening and subsequent target sequencing. TILLING has been successfully used as a non-transgenic reverse genetic approach to identify mutations in genes controlling important agronomic traits. However, this technique presents limitations in traits such as oil composition due to gene copy <span class="trait">number</span> and similarities within the soybean genome. In soybean, <span class="gene">FAD2</span> are present as two copies, <span class="gene">FAD2-1</span> and FAD2-2. Two <span class="gene">FAD2-1</span> members: <span class="gene">FAD2-1A</span> and FAD2-1B; and three <span class="gene">FAD2-2</span> members: FAD2-2A, FAD2-2B, and FAD2-2C have been reported. Syntenic, phylogenetic, and in silico analysis revealed two additional members constituting the <span class="gene">FAD2</span> gene family: <span class="gene">GmFAD2-2D</span> and <span class="gene">GmFAD2-2E,</span> located on chromosomes 09 and 15, respectively. They are presumed to have diverged from other <span class="gene">FAD2-2</span> members localized on chromosomes 19 <span class="gene">(GmFAD2-2A</span> and <span class="gene">GmFAD2-2B)</span> and 03 <span class="gene">(GmFAD2-2C).</span> This work discusses alternative solutions to the limitations of gel-based TILLING in functional genomics due to high copy <span class="trait">number</span> and multiple paralogs of the <span class="gene">FAD2</span> gene family in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean mycorrhiza-inducible phosphate transporter gene, <span class="gene">GmPT7,</span> also shows localized expression at the tips of vein endings of senescent leaves.</h2>
			<h3>PMID:25282557  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Inoue Y(1), Kobae Y(2), Omoto E(1), Tanaka A(1), Banba M(1), Takai S(1), Tamura Y(1), Hirose A(3), Komatsu K(3), Otagaki S(1), Matsumoto S(1), Taniguchi M(1), Masuta C(4), Ishimoto M(5), Hata S(6).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmPT7</span> was originally identified as an arbuscular mycorrhiza-inducible gene of soybean that encodes a member of subfamily I in the PHOSPHATE TRANSPORTER 1 family. In the present study, we established conditions under which a <span class="trait">number</span> of dwarf soybean plants complete their life cycles in a growth chamber. Using this system, we grew transgenic soybean with a <span class="gene">GmPT7</span> promoter-β-glucuronidase fusion gene and evaluated <span class="gene">GmPT7</span> expression in detail. <span class="gene">GmPT7</span> was highly expressed in mature, but not in collapsed, arbuscule-containing cortical cells, suggesting its importance in the absorption of fungus-derived phosphate and/or arbuscule development. <span class="gene">GmPT7</span> was also expressed in the columella cells of root caps and in the lateral root primordia of non-mycorrhizal roots. The expression of <span class="gene">GmPT7</span> occurred only in the late stage of phosphorus translocation from leaves to seeds, after water evaporation from the leaves ceased, and later than the expression of <span class="gene">GmUPS1-2,</span> <span class="gene">GmNRT1.7a</span> and <span class="gene">GmNRT1.7b,</span> which are possibly involved in nitrogen export. <span class="gene">GmPT7</span> expression was localized in a pair of tracheid elements at the tips of vein endings of senescent leaves. Transmission electron microscopy revealed that the tip tracheid elements in yellow leaves were still viable and had intact plasma membranes. Thus, we think that <span class="gene">GmPT7</span> on the plasma membranes transports phosphate from the apoplast into the tip elements. <span class="gene">GmPT7</span> knockdown resulted in no significant effects, the function of <span class="gene">GmPT7</span> remaining to be clarified. We propose a working model in which phosphate incorporated in vein endings moves to seeds via xylem to phloem transfer.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cloning and characterization of an aldehyde dehydrogenase gene <span class="gene">GmALDH3-1</span> from soybean].</h2>
			<h3>PMID:20466639  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huang F(1), Chi YJ, He H, Yu DY.
				<br/> 
				年份：2010 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To research the mechanism of soybean reproductive development, we identified a <span class="trait">number</span> of flower development related genes in soybean by microarray hybridization. A gene predominately expressed in soybean flowers was chosen for further analysis. Through bioinformatic and RT-PCR approaches, the full-length gene was cloned from soybean flowers. The results of BLAST searching indicated that this gene encoded for an aldehyde dehydrogenase and was named as <span class="gene">GmALDH3-1.</span> <span class="gene">GmALDH3-1</span> contains a complete open reading frame of 1485 bp in length, which encodes for a peptide of 494 amino acids. The product encoded by <span class="gene">GmALDH3-1</span> shows 83% similarity and 68% identity with Populus tomentosa PtALDH3, respectively, and 39% and 59% with human ALDH3B. Phylogenetic analysis shows that <span class="gene">GmALDH3-1</span> and other ALDH3 subfamily members are grouped into the same branch and <span class="gene">GmALDH3-1</span> is close to PtALDH3 and Arabidopsis AtALDH3F1. Real-time RT-PCR analysis demonstrated that the highest expression level of <span class="gene">GmALDH3-1</span> occurred in flowers, but the expression of this gene was almost undetectable in leaves and roots. We further analyzed <span class="gene">GmALDH3-1</span> expression during the course of <span class="trait">seed</span> development based on publicly available microarray data and found that <span class="gene">GmALDH3-1</span> was highly expressed in the <span class="trait">seed</span> endothelium, epidermis, outer integument, and hilum.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the Arabidopsis thaliana BBX32 gene in soybean increases grain <span class="trait">yield.</span></h2>
			<h3>PMID:22363475  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preuss SB(1), Meister R, Xu Q, Urwin CP, Tripodi FA, Screen SE, Anil VS, Zhu S, Morrell JA, Liu G, Ratcliffe OJ, Reuber TL, Khanna R, Goldman BS, Bell E, Ziegler TE, McClerren AL, Ruff TG, Petracek ME.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Crop <span class="trait">yield</span> is a highly complex quantitative trait. Historically, successful breeding for improved grain <span class="trait">yield</span> has led to crop plants with improved source capacity, altered plant architecture, and increased <span class="trait">resistance</span> to abiotic and biotic stresses. To date, transgenic approaches towards improving crop grain <span class="trait">yield</span> have primarily focused on protecting plants from herbicide, insects, or disease. In contrast, we have focused on identifying genes that, when expressed in soybean, improve the intrinsic ability of the plant to <span class="trait">yield</span> more. Through the large scale screening of candidate genes in transgenic soybean, we identified an Arabidopsis thaliana B-box domain gene (AtBBX32) that significantly increases soybean grain <span class="trait">yield</span> year after year in multiple transgenic events in multi-location field trials. In order to understand the underlying physiological changes that are associated with increased <span class="trait">yield</span> in transgenic soybean, we examined phenotypic differences in two AtBBX32-expressing lines and found increases in plant height and node, flower, pod, and <span class="trait">seed</span> <span class="trait">number.</span> We propose that these phenotypic changes are likely the result of changes in the timing of reproductive development in transgenic soybean that lead to the increased duration of the pod and <span class="trait">seed</span> development period. Consistent with the role of BBX32 in A. thaliana in regulating light signaling, we show that the constitutive expression of AtBBX32 in soybean alters the abundance of a subset of gene transcripts in the early morning hours. In particular, AtBBX32 alters transcript levels of the soybean clock genes <span class="gene">GmTOC1</span> and LHY-CCA1-like2 <span class="gene">(GmLCL2).</span> We propose that through the expression of AtBBX32 and modulation of the abundance of circadian clock genes during the transition from dark to light, the timing of critical phases of reproductive development are altered. These findings demonstrate a specific role for AtBBX32 in modulating soybean development, and demonstrate the validity of expressing single genes in crops to deliver increased agricultural productivity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Validation of a newly developed hexaplex real-time PCR assay for screening for presence of <span class="gene">GMOs</span> in food, feed and <span class="trait">seed.</span></h2>
			<h3>PMID:20101506  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bahrdt C(1), Krech AB, Wurz A, Wulff D.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For years, an increasing <span class="trait">number</span> and diversity of genetically modified plants has been grown on a commercial scale. The need for detection and identification of these genetically modified organisms <span class="gene">(GMOs)</span> calls for broad and at the same time flexible high throughput testing methods. Here we describe the development and validation of a hexaplex real-time polymerase chain reaction (PCR) screening assay covering more than 100 approved <span class="gene">GMOs</span> containing at least one of the <span class="gene">GMO</span> targets of the assay. The assay comprises detection systems for Cauliflower Mosaic Virus 35S promoter, Agrobacterium tumefaciens NOS terminator, Figwort Mosaic Virus 34S promoter and two construct-specific sequences present in novel genetically modified soybean and maize that lack common screening elements. Additionally a detection system for an internal positive control (IPC) indicating the presence or absence of PCR inhibiting substances was included. The six real-time PCR systems were allocated to five detection channels showing no significant crosstalk between the detection channels. As part of an extensive validation, a limit of detection (LOD(abs)) &lt; or = ten target copies was proven in hexaplex format. A sensitivity &lt; or = ten target copies of each <span class="gene">GMO</span> detection system was still shown in highly asymmetric target situations in the presence of 1,000 copies of all other <span class="gene">GMO</span> targets of each detection channel. Furthermore, the applicability to a broad sample spectrum and reliable indication of inhibition by the IPC system was demonstrated. The presented hexaplex assay offers sensitive and reliable detection of <span class="gene">GMOs</span> in processed and unprocessed food, feed and <span class="trait">seed</span> samples with high efficiency.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Localization of myo-inositol phosphate synthase <span class="gene">(GmMIPS-1)</span> during the early stages of soybean <span class="trait">seed</span> development.</h2>
			<h3>PMID:17525076  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chiera J(1), Grabau E.
				<br/> 
				年份：2007
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Erratum in J Exp Bot. 2007;58(9):2389.
					</p>
				
					<p>&nbsp;&nbsp;
						As a precursor to a large variety of compounds, myo-inositol is a central molecule required for cell metabolism and plant growth. The de novo synthesis of myo-inositol requires the activity of the enzyme D-myo-inositol-3-phosphate synthase (MIPS). <span class="gene">MIPS</span> cDNAs encoding one or more isoforms have been cloned from a <span class="trait">number</span> of species, nevertheless, little is known about the regulation of <span class="gene">MIPS</span> expression in developing <span class="trait">seed.</span> Seed-specific expression of a soybean isoform <span class="gene">(GmMIPS-1)</span> has been demonstrated, but tissue-specific localization during embryo development has not been reported. Using immunolocalization techniques, a specialized area of <span class="gene">GmMIPS-1</span> expression was identified in the outer integumentary layer during early soybean <span class="trait">seed</span> development. In addition, localization data provided evidence that <span class="gene">MIPS</span> was associated with oxalate crystal idioblasts.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional characterization of <span class="gene">GmBZL2</span> (AtBZR1 like gene) reveals the conserved BR signaling regulation in Glycine max.</h2>
			<h3>PMID:27498784</h3>
				

				
			<p>
				作者：Zhang Y(1)(2), Zhang YJ(3), Yang BJ(4), Yu XX(2)(5), Wang D(5), Zu SH(3), Xue HW(4), Lin WH(3)(6).
				<br/> 
				年份：2016 Aug 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BRs) play key roles in plant growth and development, and regulate various agricultural traits. Enhanced BR signaling leads to increased <span class="trait">seed</span> <span class="trait">number</span> and <span class="trait">yield</span> in Arabidopsis bzr1-1D (AtBZR1(P234L), gain-of-function mutant of the important transcription factor in BR signaling/effects). BR signal transduction pathway is well elucidated in Arabidopsis but less known in other species. Soybean is an important dicot crop producing edible oil and protein. Phylogenetic analysis reveals AtBZR1-like genes are highly conserved in angiosperm and there are 4 orthologues in soybean <span class="gene">(GmBZL1-4).</span> We here report the functional characterization of <span class="gene">GmBZL2</span> (relatively highly expresses in flowers). The P234 site in AtBZR1 is conserved in <span class="gene">GmBZL2</span> (P216) and mutation of <span class="gene">GmBZL2(P216L)</span> leads to <span class="gene">GmBZL2</span> accumulation. <span class="gene">GmBZL2(P216L)</span> <span class="gene">(GmBZL2*)</span> in Arabidopsis results in enhanced BR signaling; including increased <span class="trait">seed</span> <span class="trait">number</span> per silique. <span class="gene">GmBZL2*</span> partially rescued the defects of bri1-5, further demonstrating the conserved function of <span class="gene">GmBZL2</span> with AtBZR1. BR treatment promotes the accumulation, nuclear localization and dephosphorylation/phosphorylation ratio of <span class="gene">GmBZL2,</span> revealing that <span class="gene">GmBZL2</span> activity is regulated conservatively by BR signaling. Our studies not only indicate the conserved regulatory mechanism of <span class="gene">GmBZL2</span> and BR signaling pathway in soybean, but also suggest the potential application of <span class="gene">GmBZL2</span> in soybean <span class="trait">seed</span> <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution and association analysis of <span class="gene">GmCYP78A10</span> gene with <span class="trait">seed</span> size/weight and pod <span class="trait">number</span> in soybean.</h2>
			<h3>PMID:25324172  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Zhang H, Sun G, Zhang W, Qiu L.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed-size/weight traits, controlled by multiple genes in soybean, play an important role in determining <span class="trait">seed</span> <span class="trait">yield.</span> However, the molecular mechanisms controlling the <span class="trait">seed</span> size and weight in soybean remain unclear. In Arabidopsis, P450/CYP78A gene family has been proved extremely relevant to <span class="trait">seed</span> size (such as AtCYP78A5, AtCYP78A6 and AtCYP78A9). We found that a soybean <span class="gene">GmCYP78A10</span> gene underwent artificial selection during soybean breeding. The <span class="gene">GmCYP78A10a</span> allele mainly distributed in wild soybean (Glycine soja), but has been eliminated in the cultivars during early stage of soybean breeding, while the <span class="gene">GmCYP78A10b</span> allele has been accumulated and become the predominant allele in cultivated soybean (G. max). ANOVA analysis showed that the mean <span class="trait">seed</span> weight, <span class="trait">seed</span> width and <span class="trait">seed</span> thickness of soybean varieties with <span class="gene">GmCYP78A10b</span> allele was significantly heavier/bigger than those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01). The allele could explain 7.2 % variation in <span class="trait">seed</span> weight. The pod <span class="trait">number</span> of the soybeans with <span class="gene">GmCYP78A10b</span> allele significantly decreased compared to those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01, R(2) = 5.8 %), while other agronomic traits including <span class="trait">seed</span> weight/plant were not significantly affected by these two alleles. We speculated that during the early stage of soybean breeding, breeders selected big <span class="trait">seed</span> carrying <span class="gene">GmCYP78A10b</span> allele, but lowered pod <span class="trait">number</span> simultaneously. Overall, the selection did not cause the significantly change in soybean <span class="trait">seed</span> <span class="trait">yield.</span> Our results suggests that the soybean <span class="gene">GmCYP78A10</span> gene may have a similar function to those genes belonging to P450/CYP78A subfamily in Arabidopsis and provides new information for the genetic control of <span class="trait">seed</span> size in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heat-induced aggregation and sulphydryl/disulphide reaction products of soy protein with different sulphydryl contents.</h2>
			<h3>PMID:24629932  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ruan Q(1), Chen Y(1), Kong X(1), Hua Y(2).
				<br/> 
				年份：2014 Aug 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, soy proteins were reduced with 0.1-10mM dithiothreitol (DTT) to obtain an increasing <span class="trait">number</span> of sulphydryl groups (SH) with a similar particle size. Aggregation was promoted by increasing the degree of reduction when heated (100°C, 30min), resulting in larger sized aggregates (from 40 to 70nm) and a higher viscosity of the aggregate dispersion. The disulphide bond (SS) content decreased and the less <span class="gene">SS</span> linked polymer, which was composed of acidic (A) polypeptide of glycinin, basic (B) polypeptides of glycinin, and a small amount of α' and α subunits of β-conglycinin, was formed with increasing reduction degree, suggesting that SH/SS polymerisation was not the driving force for aggregation. The larger aggregates with increasing degrees of reduction were composed of more B of glycinin and β of β-conglycinin, suggesting that the A and the small amount of α' and α in the <span class="gene">SS</span> linked polymer have an inhibiting effect on protein aggregates formation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lipid transfer protein cross-reactivity assessed in vivo and in vitro in the office: pros and cons.</h2>
			<h3>PMID:21462803  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Asero R(1).
				<br/> 
				年份：2011
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Few studies analyze cross-reactivity between lipid transfer proteins (LTP) from a large spectrum of botanically unrelated plant-derived foods using routine diagnostic tests. OBJECTIVE: To assess the clinical usefulness of currently available in vivo and in vitro tests in LTP-hypersensitive patients. METHODS: An in vitro and in vivo study was performed of 15 peach-allergic adults monosensitized to <span class="gene">LTP</span> in order to analyze their allergy and hypersensitivity to apple, hazelnut, walnut, peanut, soybean, lentil, maize, celery, carrot, banana, melon, tomato, kiwi, buckwheat, and sunflower, poppy, mustard, and sesame seeds. RESULTS: The study revealed that 8, 7, 10, 5, 3, 2, 1, 1, and 1 patients were allergic to apple, hazelnut, walnut, peanut, tomato, kiwi, melon, lentil, and maize, respectively. Immunoglobulin (Ig) E levels for peach were strongly associated with the total <span class="trait">number</span> of offending foods other than peach and with levels of IgE specific for all the study foods except carrot. Both in vivo and in vitro tests showed excellent sensitivity and negative predictive value, but poor specificity and positive predictive value. Sensitized but tolerant patients showed lower IgE levels than those with a history of local or systemic symptoms, although the difference between the 3 subsets was not statistically significant. CONCLUSION: This study confirms that peach is the primary sensitizer to <span class="gene">LTP</span> and that the level of IgE to peach <span class="gene">LTP</span> is the main factor associated with cross-reactivity (and clinical allergy) to non-Rosaceae foods. Clinically irrelevant sensitization is common in LTP-hypersensitive patients, and positive in vivo and/or in vitro test results are of little help in detecting potential clinical reactors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of a Stokesia laevis epoxygenase gene.</h2>
			<h3>PMID:15587702  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hatanaka T(1), Shimizu R, Hildebrand D.
				<br/> 
				年份：2004 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Epoxy fatty acids have a <span class="trait">number</span> of important uses and there is interest in enzymes catalyzing their synthesis from renewable sources. Both cytochrome <span class="gene">P450</span> monooxygenases and divergent forms of di-iron desaturases are known to produce epoxy fatty acids in plants. Degenerate primers based on conserved sequences of delta12 desaturase-like genes led to the isolation of an epoxygenase gene from Stokesia laevis. The cDNA is 1.4 kb and it encodes 378 amino acids. The similarities of this gene at the amino acid sequence level with epoxygenases of Vernonia and Crepis, and the delta12 desaturases of soybean, <span class="gene">FAD2-1</span> and FAD2-2, are 84%, 69%, 49%, and 55%, respectively. When the vector, pYES2, was used to transform yeast, epoxy fatty acid formation was observed in the cells. The effects of electron donors in the yeast expression system were tested but cytochrome b5 and cytochrome b5 reductase genes from Arabidopsis thaliana co-expressed with the epoxygenase had little effect on vernolic acid accumulation in the yeast. Finally, this gene, driven by a seed-specific phaseolin promoter, was cloned into a TDNA-vector and transferred into Arabidopsis plants. The results showed that T2 seeds of transgenic Arabidopsis expressing the Stokesia gene accumulated vernolic acid but no vernolic acid was detected in control plants. Northern blot analysis indicates this S. laevis epoxygenase gene is expressed mainly in developing seeds and no transcript was detected in leaves or roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An acidic amino acid-specific protease from germinating soybeans.</h2>
			<h3>PMID:8688170  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tan-Wilson AL(1), Liu X, Chen R, Qi X, Wilson KA.
				<br/> 
				年份：1996 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The degradation of the beta-conglycinin protein reserves in soybean seeds during germination and early growth begins with the proteolysis of its alpha and <span class="gene">alpha'</span> subunits by an enzyme called Protease C1. In the pathway, a <span class="trait">number</span> of proteolytic intermediates are produced and subsequently degraded. Determination of the N-terminal sequences of these intermediates provides insight regarding the requirements of the cleavage sites. The N-terminal sequence of three such proteolytic intermediates has been determined. The sequence has been located in the published sequences of the beta-conglycinin subunits. Comparing these cleavage sites, plus those of two others previously delineated, shows that the P1' and P4' positions always bear either a Glu or an Asp residue while the <span class="gene">P1</span> position always bears either a Glu or a Gln residue. In addition, other sites from P3 to P7' are also rich in either Glu or Asp, and the whole region is predicted to be in a alpha-helix. Consistent with the observation, synthetic poly-L-Glu inhibits the Protease C1-catalysed degradation of the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin. Poly-L-Glu (av. M(r) = 1000) at 12.5 mM was more effective at inhibiting the reaction than poly-L-Glu (av. M(r) = 600) or poly-L-Glu (av. M(r) = 14,300) at the same concentration. Comparing large synthetic polypeptides at 12.5mM, inhibition by poly-L-Asp (av. M(r) = 15,000) is as effective as poly-L-Glu (av. M(r) = 14,300), while poly-L-Ser (av. M(r) = 15,000) had no effect at all. Poly-D-Glu (av. M(r) = 15,000) is a better inhibitor than poly-L-Glu of the same size. A serine protease of similar molecular weight as Protease <span class="gene">C1</span> and also capable of catalysing the proteolysis of the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin to generate proteolytic intermediates of the same size has been found in mung bean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Confirmation of the pleiotropic control of leaflet shape and <span class="trait">number</span> of seeds per pod by the <span class="gene">Ln</span> gene in induced soybean mutants.</h2>
			<h3>PMID:29085246</h3>
				

				
			<p>
				作者：Sayama T(1)(2), Tanabata T(3), Saruta M(2)(4), Yamada T(1), Anai T(5), Kaga A(1), Ishimoto M(1).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most soybean cultivars possess broad leaflets; however, a recessive allele on the <span class="gene">Ln</span> locus is known to cause the alteration of broad to narrow leaflets. The recessive allele ln has also been considered to increase the <span class="trait">number</span> of seeds per pod (NSP) and has the potential to improve <span class="trait">yield.</span> Recently, <span class="gene">Gm-JAG1</span> <span class="gene">(Glyma20g25000),</span> a gene controlling Ln, has been shown to complement leaf shape and silique length in Arabidopsis mutants. However, whether <span class="gene">Gm-JAG1</span> is responsible for those traits in soybean is not yet known. In this study, we investigated the pleiotropic effect of soybean <span class="gene">Ln</span> gene on leaflet shape and NSP by using two independent soybean <span class="gene">Gm-jag1</span> mutants and four ln near isogenic lines (NILs). The leaflet shape was evaluated using a leaf image analysis software, SmartLeaf, which was customized from SmartGrain. The leaflets of both the <span class="gene">Gm-jag1</span> mutants were longer and narrower than those of the wild-type plants. Interestingly, the image analysis results clarified that the perimeter of the mutant leaflets did not change, although their leaflet area decreased. Furthermore, one mutant line with narrow leaflets showed significantly higher NSP than that in the wild (or Ln) genotype, indicating that soybean <span class="gene">Ln</span> gene pleiotropically controls leaflet shape and NSP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of a WRKY gene between wild and cultivated soybeans correlates to <span class="trait">seed</span> size.</h2>
			<h3>PMID:28472462  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gu Y(1)(2), Li W(3), Jiang H(4), Wang Y(1), Gao H(1)(2), Liu M(3), Chen Q(4), Lai Y(3), He C(1)(2).
				<br/> 
				年份：2017 May 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) probably originated from the wild soybean (Glycine soja). Glycine max has a significantly larger <span class="trait">seed</span> size, but the underlying genomic changes are largely unknown. Candidate regulatory genes were preliminarily proposed by data co-localizing RNA sequencing with the quantitative loci (QTLs) for <span class="trait">seed</span> size. The soybean gene locus SoyWRKY15a and its orthologous genes from G. max <span class="gene">(GmWRKY15a)</span> and G. soja (GsWRKY15a) were analyzed in detail. The coding sequences were nearly identical between the two orthologs, but <span class="gene">GmWRKY15a</span> was significantly more highly expressed than GsWRKY15a. Four haplotypes (H1-H4) were found and they varied in the size of a CT-core microsatellite locus in the 5'-untranslated region of this gene. <span class="gene">H1</span> (with six CT-repeats) was the only allelic version found in G. max, while <span class="gene">H3</span> (with five CT-repeats) was the dominant G. soja allele. Differential expression of this gene in soybean pods was correlated with CT-repeat variation, and manipulation of the CT copy <span class="trait">number</span> altered the reporter gene expression, suggesting a regulatory role for the simple sequence repeats. <span class="trait">Seed</span> weight of wild soybeans harboring <span class="gene">H1</span> was significantly greater than that of soybeans having haplotypes H2, H3, or H4, and <span class="trait">seed</span> weight was correlated with gene expression, suggesting the influence of GsWRKY15a in controlling <span class="trait">seed</span> size. However, the <span class="trait">seed</span> size might be refractory to increased SoyWRKY15a expression in cultivated soybeans. The evolutionary significance of SoyWRKY15a variation in soybean <span class="trait">seed</span> domestication is discussed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog.</h2>
			<h3>PMID:26786479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao C(1), Takeshima R(2), Zhu J(3), Xu M(4), Sato M(5), Watanabe S(6), Kanazawa A(7), Liu B(8), Kong F(9), Yamada T(10), Abe J(11).
				<br/> 
				年份：2016 Jan 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Understanding the molecular mechanisms of flowering and maturity is important for improving the adaptability and <span class="trait">yield</span> of <span class="trait">seed</span> crops in different environments. In soybean, a facultative short-day plant, genetic variation at four maturity genes, <span class="gene">E1</span> to E4, plays an important role in adaptation to environments with different photoperiods. However, the molecular basis of natural variation in time to flowering and maturity is poorly understood. Using a cross between early-maturing soybean cultivars, we performed a genetic and molecular study of flowering genes. The progeny of this cross segregated for two maturity loci, <span class="gene">E1</span> and E9. The latter locus was subjected to detailed molecular analysis to identify the responsible gene. RESULTS: Fine mapping, sequencing, and expression analysis revealed that E9 is FT2a, an ortholog of Arabidopsis FLOWERING LOCUS T. Regardless of daylength conditions, the e9 allele was transcribed at a very low level in comparison with the E9 allele and delayed flowering. Despite identical coding sequences, a <span class="trait">number</span> of single nucleotide polymorphisms and insertions/deletions were detected in the promoter, untranslated regions, and introns between the two cultivars. Furthermore, the e9 allele had a Ty1/copia-like retrotransposon, SORE-1, inserted in the first intron. Comparison of the expression levels of different alleles among near-isogenic lines and photoperiod-insensitive cultivars indicated that the SORE-1 insertion attenuated FT2a expression by its allele-specific transcriptional repression. SORE-1 was highly methylated, and did not appear to disrupt FT2a RNA processing. CONCLUSIONS: The soybean maturity gene E9 is FT2a, and its recessive allele delays flowering because of lower transcript abundance that is caused by allele-specific transcriptional repression due to the insertion of SORE-1. The FT2a transcript abundance is thus directly associated with the variation in flowering time in soybean. The e9 allele may maintain vegetative growth in early-flowering genetic backgrounds, and also be useful as a long-juvenile allele, which causes late flowering under short-daylength conditions, in low-latitude regions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmHs1-1, encoding a calcineurin-like protein, controls hard-seededness in soybean.</h2>
			<h3>PMID:26098868  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun L(1), Miao Z(1), Cai C(2), Zhang D(1), Zhao M(1), Wu Y(2), Zhang X(2), Swarm SA(3), Zhou L(4), Zhang ZJ(4), Nelson RL(5), Ma J(6).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Loss of seed-coat impermeability was essential in the domestication of many leguminous crops to promote the production of their highly nutritious seeds. Here we show that seed-coat impermeability in wild soybean is controlled by a single gene, <span class="gene">GmHs1-1,</span> which encodes a calcineurin-like metallophosphoesterase transmembrane protein. <span class="gene">GmHs1-1</span> is primarily expressed in the Malpighian layer of the <span class="trait">seed</span> coat and is associated with calcium content. The transition from impermeability to permeability in domesticated soybean was caused by artificial selection of a point mutation in <span class="gene">GmHs1-1.</span> Interestingly, a <span class="trait">number</span> of soybean landraces evaded selection for permeability because of an alternative selection for seed-coat cracking that also enables <span class="trait">seed</span> imbibition. Despite the single origin of the mutant allele <span class="gene">Gmhs1-1,</span> the distribution pattern of allelic variants in the context of soybean population structure and the detected signature of genomic introgression between wild and cultivated soybeans suggest that <span class="gene">Gmhs1-1</span> may have experienced reselection for seed-coat permeability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic and biochemical analyses of the cotyledon and root of flooding-stressed soybean plants.</h2>
			<h3>PMID:23799004  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Komatsu S(1), Makino T, Yasue H.
				<br/> 
				年份：2013 Jun 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Flooding significantly reduces the growth and grain <span class="trait">yield</span> of soybean plants. Proteomic and biochemical techniques were used to determine whether the function of cotyledon and root is altered in soybean under flooding stress. RESULTS: Two-day-old soybean plants were flooded for 2 days, after which the proteins from root and cotyledon were extracted for proteomic analysis. In response to flooding stress, the abundance of 73 and 28 proteins was significantly altered in the root and cotyledon, respectively. The accumulation of only one protein, 70 kDa heat shock protein (HSP70) <span class="gene">(Glyma17g08020.1),</span> increased in both organs following flooding. The ratio of protein abundance of <span class="gene">HSP70</span> and biophoton emission in the cotyledon was higher than those detected in the root under flooding stress. Computed tomography and elemental analyses revealed that flooding stress decreases the <span class="trait">number</span> of calcium oxalate crystal the cotyledon, indicating calcium ion was elevated in the cotyledon under flooding stress. CONCLUSION: These results suggest that calcium might play one role through <span class="gene">HSP70</span> in the cotyledon under flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combinations of mutant <span class="gene">FAD2</span> and <span class="gene">FAD3</span> genes to produce high oleic acid and low linolenic acid soybean oil.</h2>
			<h3>PMID:22476873  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pham AT(1), Shannon JG, Bilyeu KD.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High oleic acid soybeans were produced by combining mutant <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> genes. Despite having a high oleic acid content, the linolenic acid content of these soybeans was in the range of 4-6 %, which may be high enough to cause oxidative instability of the oil. Therefore, a study was conducted to incorporate one or two mutant <span class="gene">FAD3</span> genes into the high oleic acid background to further reduce the linolenic acid content. As a result, soybean lines with high oleic acid and low linolenic acid (HOLL) content were produced using different sources of mutant <span class="gene">FAD2-1A</span> genes. While oleic acid content of these HOLL lines was stable across two testing environments, the reduction of linolenic acid content varied depending on the <span class="trait">number</span> of mutant <span class="gene">FAD3</span> genes combined with mutant <span class="gene">FAD2-1</span> genes, on the severity of mutation in the <span class="gene">FAD2-1A</span> gene, and on the testing environment. Combination of two mutant <span class="gene">FAD2-1</span> genes and one mutant <span class="gene">FAD3</span> gene resulted in less than 2 % linolenic acid content in Portageville, Missouri (MO) while four mutant genes were needed to achieve the same linolenic acid in Columbia, MO. This study generated non-transgenic soybeans with the highest oleic acid content and lowest linolenic acid content reported to date, offering a unique alternative to produce a fatty acid profile similar to olive oil.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The PA domain is crucial for determining optimum substrate length for soybean protease C1: structure and kinetics correlate with molecular function.</h2>
			<h3>PMID:22285412  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tan-Wilson A(1), Bandak B, Prabu-Jeyabalan M.
				<br/> 
				年份：2012 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A subtilisin-like enzyme, soybean protease <span class="gene">C1</span> (EC 3.4.21.25), initiates the degradation of the β-conglycinin storage proteins in early seedling growth. Previous kinetic studies revealed a nine-residue (P5-P4') length requirement for substrate peptides to attain optimum cleavage rates. This modeling study used the crystal structure of tomato subtilase (SBT3) as a starting model to explain the length requirement. The study also correlates structure to kinetic studies that elucidated the amino acid preferences of soybean protease <span class="gene">C1</span> for P1, P1' and P4' locations of the cleavage sequence. The interactions of a <span class="trait">number</span> of protease <span class="gene">C1</span> residues with P5, P4 and P4' residues of its substrate elucidated by this analysis can explain why the enzyme only hydrolyzes peptide bonds outside of soybean storage protein's core double β-barrel cupin domains. The findings further correlate with the literature-reported hypothesis for the subtilisin-specific protease-associated (PA) domain to play a critical role. Residues of the SBT3 PA domain also interact with the P2' residue on the substrate's carboxyl side of the scissile bond, while those on protease <span class="gene">C1</span> interact with its substrate's P4' residue. This stands in contrast with the subtilisin BPN' that has no PA domain, and where the enzyme makes stronger interaction with residues on the amino side of the cleaved bond. The variable patterns of interactions between the substrate models and PA domains of tomato SBT3 and soybean protease <span class="gene">C1</span> illustrate a crucial role for the PA domain in molecular recognition of their substrates.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of glycinin genes in soybean mutants for development of gene-specific markers.</h2>
			<h3>PMID:21959908  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jegadeesan S(1), Yu K, Woodrow L, Wang Y, Shi C, Poysa V.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean mutant lines that differ in 11S glycinin and 7S β-conglycinin <span class="trait">seed</span> storage protein subunit compositions were developed. These proteins have significant influence on tofu quality. The molecular mechanisms underlying the mutant lines are unknown. In this study, gene-specific markers for five of the glycinin genes (Gy1 to Gy5) were developed using three 11S null lines, two A(4) null Japanese cultivars, Enrei and Raiden, and a control cultivar, Harovinton. Whereas gene-specific primers produced the appropriate products in the control cultivar for the Gy1, Gy2, Gy3 and <span class="gene">Gy5</span> genes, they did not amplify in mutants missing the A(1a)B(2), A(2)B(1a), A(1b) B(1b), and A(3)B(4) subunits. However, ecotype targeting induced local lesions in genomes (EcoTILLING) and sequencing analysis revealed that the absence of the A(4) peptide in the mutants is due to the same point mutation as that in Enrei and Raiden. Selection efficiency of the gene-specific primer pairs was tested using a <span class="trait">number</span> of breeding lines segregating for the different subunits. Primer pairs specific to each of the Gy1, Gy2, Gy3, and <span class="gene">Gy5</span> genes can be used to detect the presence or absence of amplification in normal or mutant lines. The <span class="gene">Gy4</span> null allele can be selected for by temperature-switch PCR (TS-PCR) for identification of the A(4) (G4) null genotypes. In comparison to protein analysis by SDS-PAGE, gene-specific markers are easier, faster and more accurate for analysis, they do not have to use <span class="trait">seed,</span> and can be analyzed at any plant growth stage for marker-assisted selection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An intron sense suppression construct targeting soybean <span class="gene">FAD2-1</span> requires a double-stranded RNA-producing inverted repeat T-DNA insert.</h2>
			<h3>PMID:20424004  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mroczka A(1), Roberts PD, Fillatti JJ, Wiggins BE, Ulmasov T, Voelker T.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We demonstrate that the transformation of soybean (Glycine max) with sense suppression constructs using intron sequences from the fatty acid oleyl Delta12 desaturase gene <span class="gene">FAD2-1A</span> leads to efficient and specific reduction of <span class="gene">FAD2-1</span> transcripts in developing seeds, increased oleic acid, and decreased polyunsaturated fatty acids. The related <span class="gene">FAD2-2</span> transcripts are only marginally affected. Despite screening a large <span class="trait">number</span> of independent transformants, no single-copy efficacious transformants could be found. Invariably, all the least complex transgenic loci have two T-DNA copies in an inverted repeat configuration, centered at the right borders. We show that this T-DNA configuration produces an inverted repeat transcript and that small interfering RNAs accumulate against the target sequence.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Herbicides, glyphosate <span class="trait">resistance</span> and acute mammalian toxicity: simulating an environmental effect of glyphosate-resistant weeds in the USA.</h2>
			<h3>PMID:18069659  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gardner JG(1), Nelson GC.
				<br/> 
				年份：2008 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: With the emergence of glyphosate-resistant (GR) weeds, the environmental consequences of alternatives to <span class="gene">GR</span> technology are of increasing importance. A well-known acute mammalian toxicity measure, the LD(50) dose for rats, is used to assess one potential environmental impact of the loss of <span class="gene">GR</span> technology. A new dataset with this index is used to estimate and simulate the effects for corn, soybeans and cotton. RESULTS: With conventional tillage it is found that the use of <span class="gene">GR</span> seeds reduces the <span class="trait">number</span> of LD(50) doses applied per hectare by 17-98% depending on crop. With no-till, the use of <span class="gene">GR</span> seeds reduces LD(50) doses only in corn. If farmers switch to conventional seeds because of <span class="gene">GR</span> weeds but maintain the same tillage practice, the present simulations suggest that LD(50) doses could increase by as much as 100 LD(50) doses per hectare in soybeans, and 500 LD(50) doses per hectare in cotton, or 11.4 and 19.8% respectively. CONCLUSIONS: This is the first study to use field-level data to assess <span class="gene">GR</span> technology with a mammalian toxicity environmental indicator. It has been found that <span class="gene">GR</span> crops have a positive environmental effect, and that alternatives to <span class="gene">GR</span> technology increase toxicity.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright (c) 2007 Society of Chemical Industry.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic and genomic characterization of Kunitz trypsin inhibitors in wild and cultivated soybean genotypes.</h2>
			<h3>PMID:16884824  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Natarajan S(1), Xu C, Bae H, Bailey BA.
				<br/> 
				年份：2007 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, we investigated protein and genetic profiles of Kunitz trypsin inhibitors (KTIs) in seeds of 16 different soybean genotypes that included four groups consisting of wild soybean (Glycine soja), the cultivated soybean (G. max) ancestors of modern N. American soybean cultivars (old), modern N. American soybean (elite), and Asian cultivated soybean landraces that were the immediate results of domestication from the wild soybean. Proteins were well separated by two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) and stained protein cut from a 2D-PAGE indicated that <span class="gene">KTI</span> exists as multiple isoforms (spots) in soybean. Protein spots of <span class="gene">KTI</span> were identified and characterized using matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF-MS). Although overall distribution patterns of the <span class="gene">KTI</span> protein spots appeared similar, the <span class="trait">number</span> and intensity of the protein spots between wild and cultivated genotypes varied. Three <span class="gene">KTI</span> peptides were identified in three of the wild genotypes, PI 393551, PI 407027 and PI 407282, in which <span class="gene">KTI3</span> peptide showed highest intensity. The remaining wild genotype, PI 366120, showed four protein spots. In contrast, the ancestors, modern and Asian landrace genotypes showed only two protein spots corresponding to KTI. On the basis of DNA blot analysis, there is one copy of the <span class="gene">KTI3</span> gene in all 16 genotypes. Polymorphism was detected in one of the wild genotypes (PI 366120) both in proteomic and genomic analyses. Our data suggest that the major variation of protein profiles were between wild and cultivated soybean genotypes rather than among genotypes in the same group. Genetic variation of KTI1, <span class="gene">KTI2</span> and KTI3-related genes were detected within and between groups.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene amplification of the Hps locus in Glycine max.</h2>
			<h3>PMID:16536872  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gijzen M(1), Kuflu K, Moy P.
				<br/> 
				年份：2006 Mar 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Hydrophobic protein from soybean (HPS) is an 8 kD cysteine-rich polypeptide that causes asthma in persons allergic to soybean dust. <span class="gene">HPS</span> is synthesized in the pod endocarp and deposited on the <span class="trait">seed</span> surface during development. Past evidence suggests that the protein may mediate the adherence or dehiscence of endocarp tissues during maturation and affect the lustre, or glossiness of the <span class="trait">seed</span> surface. RESULTS: A comparison of soybean germplasm by genomic DNA blot hybridization shows that the copy <span class="trait">number</span> and structure of the Hps locus is polymorphic among soybean cultivars and related species. Changes in Hps gene copy <span class="trait">number</span> were also detected by comparative genomic DNA hybridization using cDNA microarrays. The Hps copy <span class="trait">number</span> polymorphisms co-segregated with <span class="trait">seed</span> lustre phenotype and <span class="gene">HPS</span> surface protein in a cross between dull- and shiny-seeded soybeans. In soybean cultivar Harosoy 63, a minimum of 27 +/- 5 copies of the Hps gene were estimated to be present in each haploid genome. The isolation and analysis of genomic clones indicates that the core Hps locus is comprised of a tandem array of reiterated units, with each 8.6 kb unit containing a single <span class="gene">HPS</span> open reading frame. CONCLUSION: This study shows that polymorphisms at the Hps locus arise from changes in the gene copy <span class="trait">number</span> via gene amplification. We present a model whereby Hps copy <span class="trait">number</span> modulates protein expression levels and <span class="trait">seed</span> lustre, and we suggest that gene amplification may result from selection pressures imposed on crop plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean sucrose binding protein gene family: genomic organization, gene copy <span class="trait">number</span> and tissue-specific expression of the <span class="gene">SBP2</span> promoter.</h2>
			<h3>PMID:14585823  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contim LA(1), Waclawovsky AJ, Delú-Filho N, Pirovani CP, Clarindo WR, Loureiro ME, Carvalho CR, Fontes EP.
				<br/> 
				年份：2003 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The sucrose binding protein (SBP) from soybean has been implicated as an important component of the sucrose uptake system. Two <span class="gene">SBP</span> genomic clones, gsS641.1 and gsS641.2, which correspond to allelic forms of the <span class="gene">GmSBP2/S64</span> gene, have been isolated and characterized. As a member of the <span class="trait">seed</span> storage protein superfamily, it has been shown that the <span class="gene">SBP</span> gene structure is similar to vicilin genes with intron/exon boundaries at conserved positions. Fluores cence in situ hybridization (FISH) suggested that the soybean <span class="gene">SBP</span> gene family is represented by at least two non-allelic genes corresponding to the previously isolated <span class="gene">GmSBP1</span> and <span class="gene">GmSBP2/S64</span> cDNAs. These two cDNAs share extensive sequence similarity but are located at different loci in the soybean genome. To investigate transcriptional activation of the <span class="gene">GmSBP2</span> gene, 2 kb 5'-flanking sequences of gsS641.1 and gsS641.2 were fused to the beta-glucuronidase (GUS) reporter gene and to the green fluorescent protein (GFP) reporter gene and inde pendently introduced into Nicotiana tabacum by Agrobacterium tumefaciens-mediated transformation. The <span class="gene">SBP2</span> promoter directed expression of both GUS and GFP reporter genes with high specificity to the phloem of leaves, stems and roots. Thus, the overall pattern of SBP-GUS or SBP-GFP expression is consistent with the involvement of <span class="gene">SBP</span> in sucrose translocation-dependent physiological processes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structure-physicochemical function relationships of soybean beta-conglycinin heterotrimers.</h2>
			<h3>PMID:12105965  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Maruyama N(1), Mohamed Salleh MR, Takahashi K, Yagasaki K, Goto H, Hontani N, Nakagawa S, Utsumi S.
				<br/> 
				年份：2002 Jul 17
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We purified four single molecular species of beta-conglycinin heterotrimers consisting of the alpha and beta subunits or the <span class="gene">alpha'</span> and beta subunits from mutant soybean cultivars lacking the alpha or <span class="gene">alpha'</span> subunit, respectively, and examined their structural features and physicochemical functions. The extent of the hydrophobicities of the heterotrimers was related to the <span class="trait">number</span> of the alpha or <span class="gene">alpha'</span> subunit. The thermal stabilities of the heterotrimers were mainly conferred by the subunit which had lower thermal stability. Solubilities at low ionic strength (mu = 0.08) of the heterotrimers containing the alpha or <span class="gene">alpha'</span> subunit were very similar to those of the alpha and <span class="gene">alpha'</span> homotrimers, respectively. Emulsifying abilities and heat-induced associations of the heterotrimers containing one beta subunit were similar to those of the alpha or <span class="gene">alpha'</span> homotrimer, whereas those of the heterotrimers containing two beta subunits were similar to those of the beta homotrimer.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Localization of peroxidase mRNAs in soybean seeds by in situ hybridization.</h2>
			<h3>PMID:10561068  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gijzen M(1), Miller SS, Bowman LA, Batchelor AK, Boutilier K, Miki BL.
				<br/> 
				年份：1999 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean <span class="gene">Ep</span> gene encodes an anionic peroxidase enzyme that accumulates in large amounts in <span class="trait">seed</span> coat tissues. We have isolated a second peroxidase gene, Prx2, that is also highly expressed in developing <span class="trait">seed</span> coat tissues. Sequence analysis of <span class="gene">Prx2</span> cDNA indicates that this transcript encodes a cationic peroxidase isozyme that is far removed from <span class="gene">Ep</span> in peroxidase phylogeny. To determine the expression patterns for these two peroxidases in developing seeds, the abundance and localization of the <span class="gene">Ep</span> and <span class="gene">Prx2</span> transcripts were compared by in situ hybridization. Results show the expression of <span class="gene">Ep</span> begins in a small <span class="trait">number</span> of cells flanking the vascular bundle in the <span class="trait">seed</span> coat, spreads to encircle the <span class="trait">seed,</span> and then migrates to the hourglass cells as they develop. Expression of <span class="gene">Prx2</span> occurs throughout development in all cell layers of the <span class="trait">seed</span> coat, and is also evident in the pericarp and embryo. Nonetheless, the Ep-encoded enzyme accounts for virtually all of the peroxidase activity detected in mature <span class="trait">seed</span> coats. The <span class="gene">Prx2</span> enzyme is either insoluble in a catalytically inactive form, or is subject to degradation during <span class="trait">seed</span> maturation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of exogenous auxins on expression of lipoxygenases in cultured soybean embryos.</h2>
			<h3>PMID:16668538</h3>
				

				
			<p>
				作者：Liu W(1), Hildebrand DF, Grayburn WS, Phillips GC, Collins GB.
				<br/> 
				年份：1991 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The expression of lipoxygenases (LOXs) is known to be developmentally regulated in soybeans (Glycine max. [L.] Merr.). Hormones have been firmly established as being involved in the growth and developmental processes of a <span class="trait">number</span> of plant species. Correlation between the expression of LOXs and the development and germination of soybean embryos suggests that plant hormones may affect the expression of LOXs. The present studies were conducted to investigate the effects of exogenous auxins on the expression of <span class="gene">LOX</span> isozymes and <span class="gene">LOX</span> activities in cultured cotyledon tissues of immature soybean seeds. The results revealed that at least one of the more acidic nonembryo <span class="gene">LOX</span> isozymes was induced by either alpha-naphthaleneacetic acid or indoleacetic acid but not by 2,4-dichlorophenoxyacetic acid after 4 days' exposure. Levels of LOX-1, -2, and -3 proteins and activities were significantly decreased by 2,4-dichlorophenoxyacetic acid 10 days after explanting. S1 analysis showed that embryo <span class="gene">LOX</span> messenger RNAs were detectable in the tissues treated with each of the auxins. The reduced levels of the embryo <span class="gene">LOX</span> proteins may, therefore, be regulated at the levels of translation, posttranslational modification, or degradation. The more acidic isozymes induced by alpha-naphthaleneacetic acid showed enzymatic activity and shared the same molecular mass and isoelectric point values as the germination-associated <span class="gene">LOX</span> isozymes found in hypocotyls and radicles, suggesting that those LOXs are involved in germination competency of soybean embryos.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis of the Lf1 gene that controls leaflet <span class="trait">number</span> in soybean.</h2>
			<h3>PMID:28516383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong SC(1), Kim JH(2), Bae DN(2).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: A high-resolution genetic map that was constructed for the Lf1 -residing region will provide valuable information for map-based cloning and genetic improvement efforts in soybean. Changes in leaf architecture as photosynthesis factories remain a major challenge for the improvement of crop productivity. Unlike most soybeans, which have compound leaves comprising three leaflets, the soybean Lf1 mutant has a high frequency of compound leaves with five leaflets in a partially dominant manner. Here, we generated a fine genetic map to determine the genetic basis of this multifoliolate leaf trait. A five-leaflet variant Dusam was found in a recently collected landrace cultivar. Phenotypic data were collected from the F2 population of a cross between the Dusam and three-leaflet cultivar V94-5152. The mapping results generated using public markers indicated that the five-leaflet determining gene in Dusam is an allele of the previously studied Lf1 gene on chromosome 8. A high-resolution map delimited the genomic region controlling the leaflet <span class="trait">number</span> trait to a sequence length of 49 kb. <span class="gene">AP2</span> domain-containing <span class="gene">Glyma.08g281900</span> annotated in this 49 kb region appeared to be a strong candidate for the Lf1-encoding gene, as members of the AP2-type transcription factor family regulate lateral organ development. Dusam additionally exhibits visually distinct phenotypes for shattering and seed-coat cracking traits. However, the two traits were clearly unlinked to the Lf1 gene in our mapping population. Interestingly, the mapping results suggest that the Lf1 gene most likely exerts a pleiotropic effect on the <span class="trait">number</span> of seeds per pod. Thus, our results provide a strong foundation towards the cloning of this compound leaf development gene and marker-assisted selection of the seeds per pod trait.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of large DNA deletions affecting oil quality traits in soybean seeds through transcriptome sequencing analysis.</h2>
			<h3>PMID:27179525  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Goettel W(1), Ramirez M(2), Upchurch RG(2), An YQ(3).
				<br/> 
				年份：2016 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Identification and characterization of a 254-kb genomic deletion on a duplicated chromosome segment that resulted in a low level of palmitic acid in soybean seeds using transcriptome sequencing. A large <span class="trait">number</span> of soybean genotypes varying in <span class="trait">seed</span> oil composition and content have been identified. Understanding the molecular mechanisms underlying these variations is important for breeders to effectively utilize them as a genetic resource. Through design and application of a bioinformatics approach, we identified nine co-regulated gene clusters by comparing <span class="trait">seed</span> transcriptomes of nine soybean genotypes varying in oil composition and content. We demonstrated that four gene clusters in the genotypes M23, Jack and N0304-303-3 coincided with large-scale genome rearrangements. The co-regulated gene clusters in M23 and Jack mapped to a previously described 164-kb deletion and a copy <span class="trait">number</span> amplification of the <span class="gene">Rhg1</span> locus, respectively. The coordinately down-regulated gene clusters in N0304-303-3 were caused by a 254-kb deletion containing 19 genes including a fatty acyl-ACP thioesterase B gene (FATB1a). This deletion was associated with reduced palmitic acid content in seeds and was the molecular cause of a previously reported nonfunctional <span class="gene">FATB1a</span> allele, fap nc . The M23 and N0304-304-3 deletions were located in duplicated genome segments retained from the Glycine-specific whole genome duplication that occurred 13 million years ago. The homoeologous genes in these duplicated regions shared a strong similarity in both their encoded protein sequences and transcript accumulation levels, suggesting that they may have conserved and important functions in seeds. The functional conservation of homoeologous genes may result in genetic redundancy and gene dosage effects for their associated <span class="trait">seed</span> traits, explaining why the large deletion did not cause lethal effects or completely eliminate palmitic acid in N0304-303-3.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association study, genomic prediction and marker-assisted selection for <span class="trait">seed</span> weight in soybean (Glycine max).</h2>
			<h3>PMID:26518570  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang J(1), Song Q(2), Cregan PB(2), Jiang GL(3).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Twenty-two loci for soybean SW and candidate genes conditioning <span class="trait">seed</span> development were identified; and prediction accuracies of <span class="gene">GS</span> and MAS were estimated through cross-validation and validation with unrelated populations. Soybean (Glycine max) is a major crop for plant protein and oil production, and <span class="trait">seed</span> weight (SW) is important for <span class="trait">yield</span> and quality in food/vegetable uses of soybean. However, our knowledge of genes controlling SW remains limited. To better understand the molecular mechanism underlying the trait and explore marker-based breeding approaches, we conducted a genome-wide association study in a population of 309 soybean germplasm accessions using 31,045 single nucleotide polymorphisms (SNPs), and estimated the prediction accuracy of genomic selection (GS) and marker-assisted selection (MAS) for SW. Twenty-two loci of minor effect associated with SW were identified, including hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> The mixed model containing these loci explained 83.4% of phenotypic variation. Candidate genes with Arabidopsis orthologs conditioning SW were also proposed. The prediction accuracies of <span class="gene">GS</span> and MAS by cross-validation were 0.75-0.87 and 0.62-0.75, respectively, depending on the <span class="trait">number</span> of SNPs used and the size of training population. <span class="gene">GS</span> also outperformed MAS when the validation was performed using unrelated panels across a wide range of maturities, with an average prediction accuracy of 0.74 versus 0.53. This study convincingly demonstrated that soybean SW is controlled by numerous minor-effect loci. It greatly enhances our understanding of the genetic basis of SW in soybean and facilitates the identification of genes controlling the trait. It also suggests that <span class="gene">GS</span> holds promise for accelerating soybean breeding progress. The results are helpful for genetic improvement and genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<span class="gene">Ln</span> is a key regulator of leaflet shape and <span class="trait">number</span> of seeds per pod in soybean.</h2>
			<h3>PMID:23243125  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong N(1), Suh SJ, Kim MH, Lee S, Moon JK, Kim HS, Jeong SC.
				<br/> 
				年份：2012 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Narrow leaflet soybean (Glycine max) varieties tend to have more seeds per pod than broad leaflet varieties. Narrow leaflet in soybean is conferred by a single recessive gene, ln. Here, we show that the transition from broad (Ln) to narrow leaflet (ln) is associated with an amino acid substitution in the EAR motif encoded by a gene (designated <span class="gene">Gm-JAGGED1)</span> homologous to Arabidopsis JAGGED (JAG) that regulates lateral organ development and the variant exerts a pleiotropic effect on fruit patterning. The genomic region that regulates both the traits was mapped to a 12.6-kb region containing only one gene, <span class="gene">Gm-JAG1.</span> Introducing the <span class="gene">Gm-JAG1</span> allele into a loss-of-function Arabidopsis jagged mutant partially restored the wild-type JAG phenotypes, including leaf shape, flower opening, and fruit shape, but the <span class="gene">Gm-jag1</span> (ln) and EAR-deleted <span class="gene">Gm-JAG1</span> alleles in the jagged mutant did not result in an apparent phenotypic change. These observations indicate that despite some degree of functional change of <span class="gene">Gm-JAG1</span> due to the divergence from Arabidopsis JAG, <span class="gene">Gm-JAG1</span> complemented the functions of JAG in Arabidopsis thaliana. However, the <span class="gene">Gm-JAG1</span> homoeolog, <span class="gene">Gm-JAG2,</span> appears to be sub- or neofunctionalized, as revealed by the differential expression of the two genes in multiple plant tissues, a complementation test, and an allelic analysis at both loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript profiling reveals expression differences in wild-type and glabrous soybean lines.</h2>
			<h3>PMID:22029708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hunt M(1), Kaur N, Stromvik M, Vodkin L.
				<br/> 
				年份：2011 Oct 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trichome hairs affect diverse agronomic characters such as <span class="trait">seed</span> weight and <span class="trait">yield,</span> prevent insect damage and reduce loss of water but their molecular control has not been extensively studied in soybean. Several detailed models for trichome development have been proposed for Arabidopsis thaliana, but their applicability to important crops such as cotton and soybean is not fully known. RESULTS: Two high throughput transcript sequencing methods, Digital Gene Expression (DGE) Tag Profiling and RNA-Seq, were used to compare the transcriptional profiles in wild-type (cv. Clark standard, CS) and a mutant (cv. Clark glabrous, i.e., trichomeless or hairless, CG) soybean isoline that carries the dominant <span class="gene">P1</span> allele. DGE data and RNA-Seq data were mapped to the cDNAs (Glyma models) predicted from the reference soybean genome, Williams 82. Extending the model length by 250 bp at both ends resulted in significantly more matches of authentic DGE tags indicating that many of the predicted gene models are prematurely truncated at the 5' and 3' UTRs. The genome-wide comparative study of the transcript profiles of the wild-type versus mutant line revealed a <span class="trait">number</span> of differentially expressed genes. One highly-expressed gene, <span class="gene">Glyma04g35130,</span> in wild-type soybean was of interest as it has high homology to the cotton gene GhRDL1 gene that has been identified as being involved in cotton fiber initiation and is a member of the BURP protein family. Sequence comparison of <span class="gene">Glyma04g35130</span> among Williams 82 with our sequences derived from CS and CG isolines revealed various SNPs and indels including addition of one nucleotide C in the CG and insertion of ~60 bp in the third exon of CS that causes a frameshift mutation and premature truncation of peptides in both lines as compared to Williams 82. CONCLUSION: Although not a candidate for the <span class="gene">P1</span> locus, a BURP family member <span class="gene">(Glyma04g35130)</span> from soybean has been shown to be abundantly expressed in the CS line and very weakly expressed in the glabrous CG line. RNA-Seq and DGE data are compared and provide experimental data on the expression of predicted soybean gene models as well as an overview of the genes expressed in young shoot tips of two closely related isolines.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High level transgenic expression of soybean (Glycine max) <span class="gene">GmERF</span> and <span class="gene">Gmubi</span> gene promoters isolated by a novel promoter analysis pipeline.</h2>
			<h3>PMID:21050446  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hernandez-Garcia CM(1), Bouchard RA, Rushton PJ, Jones ML, Chen X, Timko MP, Finer JJ.
				<br/> 
				年份：2010 Nov 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Although numerous factors can influence gene expression, promoters are perhaps the most important component of the regulatory control process. Promoter regions are often defined as a region upstream of the transcriptional start. They contain regulatory elements that interact with regulatory proteins to modulate gene expression. Most genes possess their own unique promoter and large numbers of promoters are therefore available for study. Unfortunately, relatively few promoters have been isolated and characterized; particularly from soybean (Glycine max). RESULTS: In this research, a bioinformatics approach was first performed to identify members of the <span class="gene">Gmubi</span> (G.max ubiquitin) and the <span class="gene">GmERF</span> (G. max Ethylene Response Factor) gene families of soybean. Ten <span class="gene">Gmubi</span> and ten <span class="gene">GmERF</span> promoters from selected genes were cloned upstream of the gfp gene and successfully characterized using rapid validation tools developed for both transient and stable expression. Quantification of promoter strength using transient expression in lima bean (Phaseolus lunatus) cotyledonary tissue and stable expression in soybean hairy roots showed that the intensity of gfp gene expression was mostly conserved across the two expression systems. Seven of the ten <span class="gene">Gmubi</span> promoters yielded from 2- to 7-fold higher expression than a standard CaMV35S promoter while four of the ten <span class="gene">GmERF</span> promoters showed from 1.5- to 2.2-times higher GFP levels compared to the CaMV35S promoter. Quantification of GFP expression in stably-transformed hairy roots of soybean was variable among roots derived from different transformation events but consistent among secondary roots, derived from the same primary transformation events. Molecular analysis of hairy root events revealed a direct relationship between copy <span class="trait">number</span> and expression intensity; higher copy <span class="trait">number</span> events displayed higher GFP expression. CONCLUSION: In this study, we present expression intensity data on 20 novel soybean promoters from two different gene families, ubiquitin and ERF. We also demonstrate the utility of lima bean cotyledons and soybean hairy roots for rapid promoter analyses and provide novel insights towards the utilization of these expression systems. The soybean promoters characterized here will be useful for production of transgenic soybean plants for both basic research and commercial plant improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genistein.</h2>
			<h3>PMID:12031439  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dixon RA(1), Ferreira D.
				<br/> 
				年份：2002 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genistein (4',5,7-trihydroxyisoflavone) is a common precursor in the biosynthesis of antimicrobial phytoalexins and phytoanticipins in legumes, and an important nutraceutical molecule found in soybean seeds. Genistein is a phytoestrogen with a wide variety of pharmacological effects in animal cells, including tyrosine kinase inhibition, and dietary genistein ingestion has been linked, through epidemiological and animal model studies, with a range of potential health beneficial effects. These include chemoprevention of breast and prostate cancers, cardiovascular disease and post-menopausal ailments. In spite of an extensive literature on the effects of dietary genistein, questions still exist as to its potential overall benefits as a component of the human diet. Genistein can be synthesized chemically via the deoxybenzoin or chalcone route. Genistein is synthesized in plants from the flavanone naringenin by a novel ring migration reaction catalyzed by the cytochrome <span class="gene">P450</span> enzyme isoflavone synthase (IFS). <span class="gene">IFS</span> genes have recently been cloned from a <span class="trait">number</span> of plant species, and production of genistein can be now achieved in non-legumes by recombinant DNA approaches.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic production of epoxy fatty acids by expression of a cytochrome <span class="gene">P450</span> enzyme from Euphorbia lagascae <span class="trait">seed.</span></h2>
			<h3>PMID:11842164  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cahoon EB(1), Ripp KG, Hall SE, McGonigle B.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> oils of a <span class="trait">number</span> of Asteraceae and Euphorbiaceae species are enriched in 12-epoxyoctadeca-cis-9-enoic acid (vernolic acid), an unusual 18-carbon Delta(12)-epoxy fatty acid with potential industrial value. It has been previously demonstrated that the epoxy group of vernolic acid is synthesized by the activity of a Delta(12)-oleic acid desaturase-like enzyme in seeds of the Asteraceae Crepis palaestina and Vernonia galamensis. In contrast, results from metabolic studies have suggested the involvement of a cytochrome <span class="gene">P450</span> enzyme in vernolic acid synthesis in seeds of the Euphorbiaceae species Euphorbia lagascae. To clarify the biosynthetic origin of vernolic acid in E. lagascae <span class="trait">seed,</span> an expressed sequence tag analysis was conducted. Among 1,006 randomly sequenced cDNAs from developing E. lagascae seeds, two identical expressed sequence tags were identified that encode a cytochrome <span class="gene">P450</span> enzyme classified as CYP726A1. Consistent with the seed-specific occurrence of vernolic acid in E. lagascae, mRNA corresponding to the CYP726A1 gene was abundant in developing seeds, but was not detected in leaves. In addition, expression of the E. lagascae CYP726A1 cDNA in Saccharomyces cerevisiae was accompanied by production of vernolic acid in cultures supplied with linoleic acid and an epoxy fatty acid tentatively identified as 12-epoxyoctadeca-9,15-dienoic acid (12-epoxy-18:2Delta(9,15)) in cultures supplied with alpha-linolenic acid. Consistent with this, expression of CYP726A1 in transgenic tobacco (Nicotiana tabacum) callus or somatic soybean (Glycine max) embryos resulted in the accumulation of vernolic acid and 12-epoxy-18:2Delta(9,15). Overall, these results conclusively demonstrate that Asteraceae species and the Euphorbiaceae E. lagascae have evolved structurally unrelated enzymes to generate the Delta(12)-epoxy group of vernolic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the subunits of beta-conglycinin.</h2>
			<h3>PMID:3840670  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Coates JB, Medeiros JS, Thanh VH, Nielsen NC.
				<br/> 
				年份：1985 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Four subunits of beta-conglycinin were purified from soybean cultivar CX 635-1-1-1, and were designated alpha, alpha', beta, and beta' in accordance with nomenclature proposed by Thanh and Shibasaki [(1977) Biochim. Biophys. Acta 490, 370-384]. Of these subunits, beta' has not previously been reported or characterized. Consistent with the low levels of methionine in these proteins, cyanogen bromide cleavage of alpha', alpha, and beta' subunits produced only a few fragments. The beta subunit contains no methionine and was not cleaved by cyanogen bromide. The NH2-terminal amino acid sequences of the alpha and <span class="gene">alpha'</span> subunits are homologous, and each has valine at its amino terminus. The beta subunit has a very different NH2-terminal sequence from those of the alpha and <span class="gene">alpha'</span> subunits, and has leucine at its amino terminus. The NH2-terminal sequence of the beta' subunit could not be determined, as it appeared to be blocked to Edman degradation. Although alpha and <span class="gene">alpha'</span> subunits have similar NH2-terminal sequences, they differ in the <span class="trait">number</span> of methionine residues and so yielded different numbers of cyanogen bromide fragments. Two cyanogen bromide fragments (CB-1 and CB-2) were purified from the alpha subunit. CB-1 originated from the NH2-terminal end of the subunit. The amino acid sequence of CB-2 was identical to that predicted from the nucleotide sequence of cDNA clone pB36. The insert in pB36 encoded 216 amino acids from the COOH-terminal end of the alpha subunit and contained a 138-bp trailer sequence which was followed by a poly-(A) tail. Maps showing the relative positions of methionine residues and carbohydrate moieties in the alpha and <span class="gene">alpha'</span> subunits were drawn, based on primary sequence data, and the size and carbohydrate content of the CNBr fragments derived from the subunits.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in the levels of off-flavor generation in soybean through biotic elicitor treatments.</h2>
			<h3>PMID:25552290  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kumari S(1), Dahuja A, Vinutha T, Lal SK, Kar A, Rai RD.
				<br/> 
				年份：2015 Jan 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The nutritional benefits of soybean remain underutilized as the off-flavor present in it limits the consumption and acceptability among people. The aim of the present study was to unveil the effect of the phytohormones methyl jasmonate (MJ: 0, 50 μM, 1 mM, and 15 mM) and salicylic acid (SA: 0, 50 μM, 0.1 mM, and 10 mM) as elicitors on two contrasting off-flavor soybean varieties at different growth stages (1, bloom; 2, pod development; 3, <span class="trait">seed</span> development). The effects of two elicitors varied widely and were found to be dose dependent and growth stage independent. SA reduces the lipoxygenase (LOX) and hydroperoxide lyase (HPL) activity, which in turn resulted in reduction in the TBA <span class="trait">number</span> and carbonyl value in contrast to MJ. SA 0.1 mM is the most effective dose in reduction of off-flavor determining parameters and protein oxidation, and it reduces the <span class="gene">LOX</span> and <span class="gene">HPL</span> activity by 2.3- and 2.4-fold, respectively in "high off-flavor" cultivar 'Bragg' compared to "low off-flavor" cultivar 'DS 2706' which showed 1.4- and 2.1-fold, respectively. This reduction in protein oxidation is also supported by enhanced content of antioxidant enzymes. Thus, phytohormone SA can be used in reduction of off-flavor generation, more effectively than MJ treatments, in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protocol for augmented shoot organogenesis in selected variety of soybean [Glycine max L. (Merr.)].</h2>
			<h3>PMID:23214267  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Devi MK(1), Sakthivelu G, Giridhar P, Ravishankar GA.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Development of a reproducible, versatile and efficient in vitro plant regeneration system is highly warranted for Indian soybean varieties for their mass multiplication in view of their commercial significance. Accordingly a protocol for direct shoot organogenesis in soybean variety JS 335 has been developed. Using cotyledonary node explants significant organogenic responses, mean shoot <span class="trait">number</span> and shoot length were observed when these were incubated on <span class="gene">MS</span> medium supplemented with 0.89 microM Benzyladenine (BA) and 5 microg/L triacontanol (TRIA) where in 9.3 +/- 0.5 shoots were obtained. TRIA at 5 microg/L able to produce 6.8 +/- 0.5 shoot buds in presence of 0.98 microM IBA and 0.89 microM BA. Highest mean shoot buds (14.0 +/- 0.5 and 9.0 +/- 0.5) and mean shoot length (4.6 +/- 0.3 and 10.0 +/- 0.7) were obtained when cotyledonary node and shoot tip explants were cultured on <span class="gene">MS</span> medium containing 0.14 microM gibberellic acid (GA3), 0.89 microM BA and 5 microg/L TRIA. Moreover, TRIA supported highest mean root <span class="trait">number</span> (6.3 +/- 0.5) and root length (21.5 +/- 0.57 cm). Field survival of in vitro derived plants of TRIA treatment was 70% and the overall growth and <span class="trait">seed</span> <span class="trait">yield</span> was also significantly better than control plants. This protocol may be used for improving the in vitro regeneration of soybean variety JS 335 for transformation studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GsZFP1, a new Cys2/His2-type zinc-finger protein, is a positive regulator of plant tolerance to cold and <span class="trait">drought</span> stress.</h2>
			<h3>PMID:22160567  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Luo X(1), Bai X, Zhu D, Li Y, Ji W, Cai H, Wu J, Liu B, Zhu Y.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant acclimation to environmental stress is controlled by a complex network of regulatory genes that compose distinct stress-response regulons. The C2H2-type zinc-finger proteins (ZFPs) have been implicated in different cellular processes involved in plant development and stress responses. Through microarray analysis, an alkaline (NaHCO(3))-responsive ZFP gene GsZFP1 was identified and subsequently cloned from Glyycine soja. GsZFP1 encodes a 35.14 kDa protein with one C2H2-type zinc-finger motif. The QALGGH domain, conserved in most plant C2H2-type ZFPs, is absent in the GsZFP1 protein sequence. A subcellular localization study using a GFP fusion protein indicated that GsZFP1 is localized to the nucleus. Real-time RT-PCR analysis showed that GsZFP1 was induced in the leaf by ABA (100 μM), salt (200 mM NaCl), and cold (4°C), and in the root by ABA (100 μM), cold (4°C), and <span class="trait">drought</span> (30% PEG 6000). Over-expression of GsZFP1 in transgenic Arabidopsis resulted in a greater tolerance to cold and <span class="trait">drought</span> stress, a decreased water loss rate, and an increase in proline irrespective of environmental conditions. The over-expression of GsZFP1 also increased the expression of a <span class="trait">number</span> of stress-response marker genes, including CBF1, CBF2, CBF3, NCED3, COR47, and <span class="gene">RD29A</span> in response to cold stress and RAB18, NCED3, P5CS, RD22, and <span class="gene">RD29A</span> in response to <span class="trait">drought</span> stress, especially early during stress treatments. Our studies suggest that GsZFP1 plays a crucial role in the plant response to cold and <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular evolution of glycinin and β-conglycinin gene families in soybean (Glycine max L. Merr.).</h2>
			<h3>PMID:20668431  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li C(1), Zhang YM.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						There are two main classes of multi-subunit <span class="trait">seed</span> storage proteins, glycinin (11S) and β-conglycinin (7S), which account for approximately 70% of the total protein in a typical soybean <span class="trait">seed.</span> The subunits of these two protein classes are encoded by a <span class="trait">number</span> of genes. The genomic organization of these genes follows a complex evolutionary history. This research was designed to describe the origin and maintenance of genes in each of these gene families by analyzing the synteny, phylogenies, selection pressure and duplications of the genes in each gene family. The ancestral glycinin gene initially experienced a tandem duplication event; then, the genome underwent two subsequent rounds of whole-genome duplication, thereby resulting in duplication of the glycinin genes, and finally a tandem duplication likely gave rise to the <span class="gene">Gy1</span> and <span class="gene">Gy2</span> genes. The β-conglycinin genes primarily originated through the more recent whole-genome duplication and several tandem duplications. Purifying selection has had a key role in the maintenance of genes in both gene families. In addition, positive selection in the glycinin genes and a large deletion in a β-conglycinin exon contribute to the diversity of the duplicate genes. In summary, our results suggest that the duplicated genes in both gene families prefer to retain similar function throughout evolution and therefore may contribute to phenotypic robustness.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The critical period of weed control in soybean (Glycine max (L.) Merr.) in north of Iran conditions.</h2>
			<h3>PMID:18817174  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Keramati S(1), Pirdashti H, Esmaili MA, Abbasian A, Habibi M.
				<br/> 
				年份：2008 Feb 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field study was conducted in 2006 at Sari Agricultural and Natural Resources University, in order to determine the best time for weed control in soybean promising line, 033. Experiment was arranged in randomized complete block design with 4 replications and two series of treatments. In the first series, weeds were kept in place until crop reached <span class="gene">V2</span> (second trifoliolate), V4 (fourth trifoliolate), V6 (sixth trifoliolate), R1 (beginning bloom, first flower), R3 (beginning pod), R5 (beginning <span class="trait">seed)</span> and were then removed and the crop kept weed-free for the rest of the season. In the second series, crops were kept weed-free until the above growth stages after which weeds were allowed to grow in the plots for the rest of the season. Whole season weedy and weed-free plots were included in the experiment for <span class="trait">yield</span> comparison. The results showed that among studied traits, grain <span class="trait">yield,</span> pod numbers per plant and weed biomass were affected significantly by control and interference treatments. The highest <span class="trait">number</span> of pods per plant was obtained from plots which kept weed-free for whole season control. Results showed that weed control should be carried out between <span class="gene">V2</span> (26 day after planting) to R1 (63 day after planting) stages of soybean to provide maximum grain <span class="trait">yield.</span> Thus, it is possible to optimize the timing of weed control, which can serve to reduce the costs and side effects of intensive chemical weed control.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic characterization of a novel Tib-derived variant of soybean Kunitz trypsin inhibitor detected in wild soybean (Glycine soja).</h2>
			<h3>PMID:15060597  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang KJ(1), Yamashita T, Watanabe M, Takahata Y.
				<br/> 
				年份：2004 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A novel variant of soybean Kunitz trypsin inhibitor (SKTI) was detected in 530 lines of wild soybean (Glycine soja). This variant showed an intermediate electrophoretic mobility between the Tia and <span class="gene">Tic</span> types. In isoelectric focusing polyacrylamide gel electrophoresis gels containing urea, this variant had a similar isoelectric point as that of Tia. The genetic analysis of SKTI bands in F2 seeds from crosses of the new variant type with Tia or <span class="gene">Tic</span> type showed that this variant type is controlled by a codominant allele at the SKTI locus. We propose the genetic symbol <span class="gene">Tif</span> for this novel variant. When the nucleotide sequence of the <span class="gene">Tif</span> gene was compared with those of other types of SKTI genes (Tia, Tib, and Tic), the sequence of <span class="gene">Tif</span> was identical to that of Tib with the exception of one A--&gt;G transitional mutation occurring at position 676 of Tif. This mutation resulted in an amino acid change from Lys to Glu at the 178 residue. These results suggest that this variant is derived from Tib through a point mutation. In addition, we settled an inconsistency in the <span class="trait">number</span> of amino acid differences between Tia and Tib (eight or nine). Analysis of nucleotide and amino acid sequences revealed that Tib was different from Tia by nine amino acids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine genetic mapping of the genomic region controlling leaflet shape and <span class="trait">number</span> of seeds per pod in the soybean.</h2>
			<h3>PMID:21104397  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong N(1), Moon JK, Kim HS, Kim CG, Jeong SC.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Narrow leaflet cultivars tend to have more seeds per pod than broad leaflet cultivars in soybean [Glycine max (L.) Merr.], which suggests that the leaflet-shape trait locus is tightly linked to or cosegregates with the trait locus controlling the <span class="trait">number</span> of seeds per pod (NSPP). Here, we attempted to further elucidate the relationship between leaflet shape and NSPP. A BC(3)F(2) population from a cross between the 'Sowon' (narrow leaflets and high NSPP) and 'V94-5152' (broad leaflets and low NSPP) variants was used. The results of the molecular genetic analyses indicated that, although the NSPP characteristic, in particular, the occurrence of 4-seeded pods, is governed by additional modifying genes that are likely present in Sowon, the two traits cosegregate in the BC(3)F(2) population. The mapping results generated using public markers demonstrated that the narrow leaflet-determining gene in Sowon is an allele of the previously highly studied ln gene on chromosome 20. A high-resolution map delimited the genomic region controlling both the leaflet shape and NSPP traits to a sequence length of 66 kb, corresponding to 0.7 cM. Among the three genes annotated in this 66 kb region, <span class="gene">Glyma20g25000.1</span> appeared to be a good candidate for the Ln-encoding gene, owing to its 47.8% homology with the protein encoding for the JAGGED gene that regulates lateral organ development in Arabidopsis. Taken together, our results suggested that phenotypic variations for narrow leaflet and NSPP are predominantly from the pleiotropic effects of the ln gene. Thus, our results should provide a molecular framework for soybean breeding programs with the objective of improving soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:"Potential health benefits of lunasin: a multifaceted soy-derived bioactive peptide".</h2>
			<h3>PMID:25627564  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lule VK(1), Garg S, Pophaly SD, Hitesh, Tomar SK.
				<br/> 
				年份：2015 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bioactive peptides are small protein fragments derived from enzymatic hydrolysis of food proteins, fermentation with proteolytic starter cultures, and gastrointestinal digestion. These peptides have positive impacts on a <span class="trait">number</span> of physiological functions in living beings. Lunasin, a soy-derived bioactive peptide, is one of the most promising among them. Lunasin encoded within 2S albumin <span class="gene">(GM2S-1)</span> gene, identified as a novel peptide extracted from soybean <span class="trait">seed.</span> It is composed of 43 amino acid residues with a molecular weight of 5.5 kDa. Extensive scientific studies have shown that lunasin possesses inherent antioxidative, anti-inflammatory, anticancerous properties and could also play a vital role in regulating of cholesterol biosynthesis in the body. Its high bioavailability and heat stable nature allow its potential use as dietary supplement. The present review summarizes some of the potential health and therapeutic benefits of lunasin reported hitherto.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean <span class="trait">seed</span> oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased <span class="trait">seed</span> <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] <span class="trait">seed</span> oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as <span class="trait">seed</span> <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between <span class="trait">seed</span> oil concentration and important agronomic and <span class="trait">seed</span> quality traits, including <span class="trait">seed</span> <span class="trait">yield,</span> 100-seed weight, protein concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with <span class="trait">seed</span> oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the <span class="trait">number</span> of QTL that were co-localized with other important traits QTL were six for protein concentration, four for <span class="trait">seed</span> <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with <span class="trait">seed</span> protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with <span class="trait">seed</span> <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with <span class="trait">seed</span> oil concentration, were identified for the selected traits in this study. The <span class="trait">number</span> of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for protein concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as <span class="trait">seed</span> <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher <span class="trait">seed</span> <span class="trait">yield</span> and oil concentration or higher <span class="trait">seed</span> protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plastid transformation in soybean.</h2>
			<h3>PMID:24599865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dubald M(1), Tissot G, Pelissier B.
				<br/> 
				年份：2014
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The biotechnological potential of plastid genetic engineering has been illustrated in a limited <span class="trait">number</span> of higher plant species. We have developed a reproducible method to generate plastid transformants in soybean (Glycine max), a crop of major agronomic importance. The transformation vectors are delivered to embryogenic cultures by the particle gun method and selection performed using the <span class="gene">aadA</span> antibiotic <span class="trait">resistance</span> gene. Homoplasmy is established rapidly in the selected events without the need for further selection or regeneration cycles, and genes of interest can be expressed at a high level in green tissues. This is a significant step toward the commercial application of this technology.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Soybean LATERAL ORGAN BOUNDARIES Domain-Containing Genes: A Functional Investigation of <span class="gene">GmLBD12.</span></h2>
			<h3>PMID:28464070</h3>
				

				
			<p>
				作者：Yang H, Shi G, Du H, Wang H, Zhang Z, Hu D, Wang J, Huang F, Yu D.
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-specific () genes play critical roles in various plant growth and development processes. However, the <span class="trait">number</span> and characteristics of genes in soybean [ (L.) Merr.] remain unknown. Here, we identified 90 homologous genes in the soybean genome that phylogenetically clustered into two classes (I and II). The majority of the genes were evenly distributed across all 20 soybean chromosomes, and 77 (81.11%) of them were detected in segmental duplicated regions. Furthermore, the exon-intron organization and motif composition for each were analyzed. A close phylogenetic relationship was identified between the soybean genes and 41 previously reported genes of different plants in the same group, providing insights into their putative functions. Expression analysis indicated that more than half of the genes were expressed, with the two gene classes showing differential tissue expression characteristics; in addition, they were differentially induced by biotic and abiotic stresses. To further explore the functions of genes in soybean, was selected for functional characterization. <span class="gene">GmLBD12</span> was mainly localized to the nucleus and showed high expression in root and <span class="trait">seed</span> tissues. Overexpressing in (L.) Heynh resulted in increases in lateral root (LR) <span class="trait">number</span> and plant height. Quantitative real-time polymerase chain reaction (qRT-PCR) analysis demonstrated that was induced by <span class="trait">drought,</span> salt, cold, indole acetic acid (IAA), abscisic acid (ABA), and salicylic acid SA treatments. This study provides the first comprehensive analysis of the soybean gene family and a valuable foundation for future functional studies of genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome-wide transcript profiling to identify differentially expressed genes associated with <span class="trait">seed</span> field emergence in two soybean low phytate mutants.</h2>
			<h3>PMID:28100173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan F(1), Yu X(1), Dong D(1), Yang Q(1), Fu X(1), Zhu S(1), Zhu D(2).
				<br/> 
				年份：2017 Jan 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Seed</span> germination is important to soybean (Glycine max) growth and development, ultimately affecting soybean <span class="trait">yield.</span> A lower <span class="trait">seed</span> field emergence has been the main hindrance for breeding soybeans low in phytate. Although this reduction could be overcome by additional breeding and selection, the mechanisms of <span class="trait">seed</span> germination in different low phytate mutants remain unknown. In this study, we performed a comparative transcript analysis of two low phytate soybean mutants (TW-1 and TW-1-M), which have the same mutation, a 2 bp deletion in <span class="gene">GmMIPS1,</span> but show a significant difference in <span class="trait">seed</span> field emergence, TW-1-M was higher than that of TW-1 . RESULTS: Numerous genes analyzed by RNA-Seq showed markedly different expression levels between TW-1-M and TW-1 mutants. Approximately 30,000-35,000 read-mapped genes and ~21000-25000 expressed genes were identified for each library. There were ~3900-9200 differentially expressed genes (DEGs) in each contrast library, the <span class="trait">number</span> of up-regulated genes was similar with down-regulated genes in the mutant TW-1and TW-1-M. Gene ontology functional categories of DEGs indicated that the ethylene-mediated signaling pathway, the abscisic acid-mediated signaling pathway, response to hormone, ethylene biosynthetic process, ethylene metabolic process, regulation of hormone levels, and oxidation-reduction process, regulation of flavonoid biosynthetic process and regulation of abscisic acid-activated signaling pathway had high correlations with <span class="trait">seed</span> germination. In total, 2457 DEGs involved in the above functional categories were identified. Twenty-two genes with 20 biological functions were the most highly up/down- regulated (absolute value Log2FC &gt;5) in the high field emergence mutant TW-1-M and were related to metabolic or signaling pathways. Fifty-seven genes with 36 biological functions had the greatest expression abundance (FRPM &gt;100) in germination-related pathways. CONCLUSIONS: <span class="trait">Seed</span> germination in the soybean low phytate mutants is a very complex process, which involves a series of physiological, morphological and transcriptional changes. Compared with TW-1, TW-1-M had a very different gene expression profile, which included genes related to plant hormones, antioxidation, anti-stress and energy metabolism processes. Our research provides a molecular basis for understanding germination mechanisms, and is also an important resource for the genetic analysis of germination in low phytate crops. Plant hormone- and antioxidation-related genes might strongly contribute to the high germination rate in the TW-1-M mutant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peanut gene expression profiling in developing seeds at different reproduction stages during Aspergillus parasiticus infection.</h2>
			<h3>PMID:18248674  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo B(1), Chen X, Dang P, Scully BT, Liang X, Holbrook CC, Yu J, Culbreath AK.
				<br/> 
				年份：2008 Feb 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Peanut (Arachis hypogaea L.) is an important crop economically and nutritionally, and is one of the most susceptible host crops to colonization of Aspergillus parasiticus and subsequent aflatoxin contamination. Knowledge from molecular genetic studies could help to devise strategies in alleviating this problem; however, few peanut DNA sequences are available in the public database. In order to understand the molecular basis of host <span class="trait">resistance</span> to aflatoxin contamination, a large-scale project was conducted to generate expressed sequence tags (ESTs) from developing seeds to identify resistance-related genes involved in defense response against Aspergillus infection and subsequent aflatoxin contamination. RESULTS: We constructed six different cDNA libraries derived from developing peanut seeds at three reproduction stages (R5, R6 and R7) from a resistant and a susceptible cultivated peanut genotypes, 'Tifrunner' (susceptible to Aspergillus infection with higher aflatoxin contamination and resistant to TSWV) and 'GT-C20' (resistant to Aspergillus with reduced aflatoxin contamination and susceptible to TSWV). The developing peanut <span class="trait">seed</span> tissues were challenged by A. parasiticus and <span class="trait">drought</span> stress in the field. A total of 24,192 randomly selected cDNA clones from six libraries were sequenced. After removing vector sequences and quality trimming, 21,777 high-quality EST sequences were generated. Sequence clustering and assembling resulted in 8,689 unique EST sequences with 1,741 tentative consensus EST sequences (TCs) and 6,948 singleton ESTs. Functional classification was performed according to <span class="gene">MIPS</span> functional catalogue criteria. The unique EST sequences were divided into twenty-two categories. A similarity search against the non-redundant protein database available from NCBI indicated that 84.78% of total ESTs showed significant similarity to known proteins, of which 165 genes had been previously reported in peanuts. There were differences in overall expression patterns in different libraries and genotypes. A <span class="trait">number</span> of sequences were expressed throughout all of the libraries, representing constitutive expressed sequences. In order to identify resistance-related genes with significantly differential expression, a statistical analysis to estimate the relative abundance (R) was used to compare the relative abundance of each gene transcripts in each cDNA library. Thirty six and forty seven unique EST sequences with threshold of R &gt; 4 from libraries of 'GT-C20' and 'Tifrunner', respectively, were selected for examination of temporal gene expression patterns according to EST frequencies. Nine and eight resistance-related genes with significant up-regulation were obtained in 'GT-C20' and 'Tifrunner' libraries, respectively. Among them, three genes were common in both genotypes. Furthermore, a comparison of our EST sequences with other plant sequences in the TIGR Gene Indices libraries showed that the percentage of peanut EST matched to Arabidopsis thaliana, maize (Zea mays), Medicago truncatula, rapeseed (Brassica napus), rice (Oryza sativa), soybean (Glycine max) and wheat (Triticum aestivum) ESTs ranged from 33.84% to 79.46% with the sequence identity &gt;/= 80%. These results revealed that peanut ESTs are more closely related to legume species than to cereal crops, and more homologous to dicot than to monocot plant species. CONCLUSION: The developed ESTs can be used to discover novel sequences or genes, to identify resistance-related genes and to detect the differences among alleles or markers between these resistant and susceptible peanut genotypes. Additionally, this large collection of cultivated peanut EST sequences will make it possible to construct microarrays for gene expression studies and for further characterization of host <span class="trait">resistance</span> mechanisms. It will be a valuable genomic resource for the peanut community. The 21,777 ESTs have been deposited to the NCBI GenBank database with accession numbers ES702769 to ES724546.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multiple organ gigantism caused by mutation in VmPPD gene in blackgram (Vigna mungo).</h2>
			<h3>PMID:28588392</h3>
				

				
			<p>
				作者：Naito K(1), Takahashi Y(1), Chaitieng B(2), Hirano K(3), Kaga A(1), Takagi K(4), Ogiso-Tanaka E(5), Thavarasook C(6), Ishimoto M(5), Tomooka N(1).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> size is one of the most important traits in leguminous crops. We obtained a recessive mutant of blackgram that had greatly enlarged leaves, stems and seeds. The mutant produced 100% bigger leaves, 50% more biomass and 70% larger seeds though it produced 40% less <span class="trait">number</span> of seeds. We designated the mutant as multiple-organ-gigantism (mog) and found the mog phenotype was due to increase in cell numbers but not in cell size. We also found the mog mutant showed a rippled leaf (rl) phenotype, which was probably caused by a pleiotropic effect of the mutation. We performed a map-based cloning and successfully identified an 8 bp deletion in the coding sequence of VmPPD gene, an orthologue of Arabidopsis PEAPOD (PPD) that regulates arrest of cell divisions in meristematic cells. We found no other mutations in the neighboring genes between the mutant and the wild type. We also knocked down <span class="gene">GmPPD</span> genes and reproduced both the mog and rl phenotypes in soybean. Controlling PPD genes to produce the mog phenotype is highly valuable for breeding since larger <span class="trait">seed</span> size could directly increase the commercial values of grain legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="gene">GmAOX2b</span> antisense gene compromises vegetative growth and <span class="trait">seed</span> production in soybean.</h2>
			<h3>PMID:22307678  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alternative oxidase mediates the cyanide-resistant respiratory pathway in plant mitochondria. In non-thermogenic plants, the role of alternative oxidase in plant growth and development is not well understood. Soybean (Glycine max) lines carrying a <span class="gene">GmAOX2b</span> antisense gene had compromised vegetative growth and reproductive performance under typical glasshouse growth conditions. The reduction in vegetative growth was demonstrated by reduction in shoot height, the <span class="trait">number</span> of leaves per plant and the green leaf area. Antisense plants also had decreased pod formation and <span class="trait">seed</span> to pod ratios, which together led to a reduction in the <span class="trait">number</span> and total mass of <span class="trait">seed</span> produced. The negative effects of the antisense gene on pod set, <span class="trait">seed</span> set, ovule availability and total <span class="trait">seed</span> mass were primarily confined to the branches, rather than the main stem. The preferential effect of alternative oxidase suppression in the branches is discussed in relation to the reproductive potential of soybean under stress. Taken together, these results demonstrate that alternative oxidase provides the benefit of sustaining plant vegetative growth and reproductive capacity in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean <span class="trait">seed</span> set and size using RNA sequencing analysis.</h2>
			<h3>PMID:28087653</h3>
				

				
			<p>
				作者：Du J(1), Wang S(1), He C(1), Zhou B(2), Ruan YL(3), Shou H(1).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To understand the gene expression networks controlling soybean <span class="trait">seed</span> set and size, transcriptome analyses were performed in three early <span class="trait">seed</span> developmental stages, using two genotypes with contrasting <span class="trait">seed</span> size. The two-dimensional data set provides a comprehensive and systems-level view on dynamic gene expression networks underpinning soybean <span class="trait">seed</span> set and subsequent development. Using pairwise comparisons and weighted gene coexpression network analyses, we identified modules of coexpressed genes and hub genes for each module. Of particular importance are the discoveries of specific modules for the large <span class="trait">seed</span> size variety and for <span class="trait">seed</span> developmental stages. A large <span class="trait">number</span> of candidate regulators for <span class="trait">seed</span> size, including those involved in hormonal signaling pathways and transcription factors, were transiently and specifically induced in the early developmental stages. The soybean homologs of a brassinosteroid signaling receptor kinase, a brassinosteroid-signaling kinase, were identified as hub genes operating in the <span class="trait">seed</span> coat network in the early <span class="trait">seed</span> maturation stage. Overexpression of a candidate <span class="trait">seed</span> size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased <span class="trait">seed</span> size and <span class="trait">seed</span> weight. Together, these analyses identified a large <span class="trait">number</span> of potential key regulators controlling soybean <span class="trait">seed</span> set, <span class="trait">seed</span> size, and, consequently, <span class="trait">yield</span> potential, thereby providing new insights into the molecular networks underlying soybean <span class="trait">seed</span> development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, Heterodera glycines Ichinohe). Although this approach successfully mapped a large <span class="trait">number</span> of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse set of 553 soybean plant introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as <span class="trait">seed</span> coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of protein engineering of canola procruciferin on its physicochemical and functional properties.</h2>
			<h3>PMID:15506820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tandang MR(1), Adachi M, Inui N, Maruyama N, Utsumi S.
				<br/> 
				年份：2004 Nov 3
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The primary structure of Brassica napus procruciferin 2/3a was engineered to elucidate structure-function relationships and to improve the functionality of cruciferin. The following mutants were constructed: (1) C287T, (2) DeltaII, variable region II was deleted; (3) C287T/DeltaII, mutation involving (1) and (2); (4) DeltaIV + A1aIV; and (5) DeltaIV + A3IV, variable region IV was replaced with variable region IV containing many charged residues from soybean glycinin A1aB1b and <span class="gene">A3B4</span> subunits. Differential scanning calorimetry analysis revealed that the A1aIV region has a more favorable interaction with the procruciferin molecule than does A3IV as well as the original regions. On the basis of heat-induced precipitation analysis, it was concluded that replacement of the free cysteine residue with threonine (C287T) and insertion of charged regions (DeltaIV + A1aIV and DeltaIV + A3IV) could lead procruciferin to form soluble aggregates after heating. Low solubility was observed in mutants DeltaIV + A3IV, DeltaII, and C287T/DeltaII, especially between pH 4 and 6 at mu = 0.08, but not in DeltaIV + A1aIV, indicating that the <span class="trait">number</span> of acidic amino acid residues and the high <span class="trait">number</span> of glutamine residues are important factors for solubility at mu = 0.08. None of the mutants showed any improvements in emulsifying ability, indicating that destabilization and addition of the hydrophilic region are not effective for emulsification. The insertion of the A1aIV region in procruciferin made the molecule more susceptible to alpha-chymotrypsin.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Measurement of emissions from air pollution sources. 4. C1-C27 organic compounds from cooking with <span class="trait">seed</span> oils.</h2>
			<h3>PMID:11883419  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schauer JJ(1), Kleeman MJ, Cass GR, Simoneit BR.
				<br/> 
				年份：2002 Feb 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The emission rates of gas-phase, semivolatile, and particle-phase organic compounds ranging in carbon <span class="trait">number</span> from <span class="gene">C1</span> to C27 were measured from institutional-scale food cooking operations that employ <span class="trait">seed</span> oils. Two cooking methods and three types of <span class="trait">seed</span> oils were examined: vegetables stir-fried in soybean oil, vegetables stir-fried in canola oil, and potatoes deep fried in hydrogenated soybean oil. The emission rates of 99 organic compounds were quantified, and these include n-alkanes, branched alkanes, alkenes, n-alkanoic acids, n-alkenoic acids, carbonyls, aromatics, polycyclic aromatic hydrocarbons (PAH), and lactones. Carbonyls and fatty acids (n-alkanoic and n-alkenoic acids) make up a significant portion of the organic compounds emitted from all three <span class="trait">seed</span> oil cooking procedures. The compositional differences in the organic compound emissions between the different cooking operations are consistent with the differences in the organic composition of the various cooking oils used. The distribution of the n-alkanoic acids between the gas and particle phases was found to be in good agreement with gas/particle partitioning theory. The relative importance of emissions from commercial deep frying operations to the total emissions of C16 and C18 n-alkanoic acids in the Los Angeles urban area was estimated using the available information and is estimated to account for approximately 7% of the total primary emissions of these acids. Additional emissions of these n-alkanoic acids from stir-frying and grill frying operations are expected. Estimates also indicate that <span class="trait">seed</span> oil cooking may make up a significant fraction of the emissions of lighter n-alkanoic acids such as nonanoic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of the acquisition of longevity during <span class="trait">seed</span> maturation in soybean.</h2>
			<h3>PMID:28700604  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pereira Lima JJ(1)(2), Buitink J(2), Lalanne D(2), Rossi RF(1)(2), Pelletier S(2), da Silva EAA(1), Leprince O(2).
				<br/> 
				年份：2017 Jul 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> longevity, defined as the ability to remain alive during storage, is an important agronomic factor. Poor longevity negatively impacts seedling establishment and consequently crop <span class="trait">yield.</span> This is particularly problematic for soybean as seeds have a short lifespan. While the economic importance of soybean has fueled a large <span class="trait">number</span> of transcriptome studies during embryogenesis and <span class="trait">seed</span> filling, the mechanisms regulating <span class="trait">seed</span> longevity during late maturation remain poorly understood. Here, a detailed physiological and molecular characterization of late <span class="trait">seed</span> maturation was performed in soybean to obtain a comprehensive overview of the regulatory genes that are potentially involved in longevity. Longevity appeared at physiological maturity at the end of <span class="trait">seed</span> filling before maturation drying and progressively doubled until the seeds reached the dry state. The increase in longevity was associated with the expression of genes encoding protective chaperones such as heat shock proteins and the repression of nuclear and chloroplast genes involved in a range of chloroplast activities, including photosynthesis. An increase in the raffinose family oligosaccharides (RFO)/sucrose ratio together with changes in RFO metabolism genes was also associated with longevity. A gene co-expression network analysis revealed 27 transcription factors whose expression profiles were highly correlated with longevity. Eight of them were previously identified in the longevity network of Medicago truncatula, including homologues of ERF110, HSF6AB, NFXL1 and members of the <span class="gene">DREB2</span> family. The network also contained several transcription factors associated with auxin and developmental cell fate during flowering, organ growth and differentiation. A transcriptional transition occurred concomitant with <span class="trait">seed</span> chlorophyll loss and detachment from the mother plant, suggesting the activation of a post-abscission program. This transition was enriched with AP2/EREBP and WRKY transcription factors and genes associated with growth, germination and post-transcriptional processes, suggesting that this program prepares the <span class="trait">seed</span> for the dry quiescent state and germination.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole-Genome Resequencing Identifies the Molecular Genetic Cause for the Absence of a <span class="gene">Gy5</span> Glycinin Protein in Soybean PI 603408.</h2>
			<h3>PMID:28592556</h3>
				

				
			<p>
				作者：Gillman JD(1)(2), Kim WS(2), Song B(3), Oehrle NW(1), Tawari NR(4), Liu S(3), Krishnan HB(5)(2).
				<br/> 
				年份：2017 Jul 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						During ongoing proteomic analysis of the soybean (Glycine max (L.) Merr) germplasm collection, PI 603408 was identified as a landrace whose seeds lack accumulation of one of the major <span class="trait">seed</span> storage glycinin protein subunits. Whole genomic resequencing was used to identify a two-base deletion affecting glycinin 5 The newly discovered deletion was confirmed to be causative through immunological, genetic, and proteomic analysis, and no significant differences in total <span class="trait">seed</span> protein content were found to be due to the glycinin 5 loss-of-function mutation per se In addition to focused studies on this one specific glycinin subunit-encoding gene, a total of 1,858,185 nucleotide variants were identified, of which 39,344 were predicted to affect protein coding regions. In order to semiautomate analysis of a large <span class="trait">number</span> of soybean gene variants, a new SIFT 4G (Sorting Intolerant From Tolerated 4 Genomes) database was designed to predict the impact of nonsynonymous single nucleotide soybean gene variants, potentially enabling more rapid analysis of soybean resequencing data in the future.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Silencing of Soybean Raffinose Synthase Gene Reduced Raffinose Family Oligosaccharides and Increased True Metabolizable Energy of Poultry Feed.</h2>
			<h3>PMID:28559898</h3>
				

				
			<p>
				作者：Valentine MF(1), De Tar JR(1), Mookkan M(1), Firman JD(2), Zhang ZJ(1)(2).
				<br/> 
				年份：2017 May 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is the <span class="trait">number</span> one oil and protein crop in the United States, but the <span class="trait">seed</span> contains several anti-nutritional factors that are toxic to both humans and livestock. RNA interference technology has become an increasingly popular technique in gene silencing because it allows for both temporal and spatial targeting of specific genes. The objective of this research is to use RNA-mediated gene silencing to down-regulate the soybean gene raffinose synthase 2 (RS2), to reduce total raffinose content in mature <span class="trait">seed.</span> Raffinose is a trisaccharide that is indigestible to humans and monogastric animals, and as monogastric animals are the largest consumers of soy products, reducing raffinose would improve the nutritional quality of soybean. An RNAi construct targeting <span class="gene">RS2</span> was designed, cloned, and transformed to the soybean genome via Agrobacterium-mediated transformation. Resulting plants were analyzed for the presence and <span class="trait">number</span> of copies of the transgene by PCR and Southern blot. The efficiency of mRNA silencing was confirmed by real-time quantitative PCR. Total raffinose content was determined by HPLC analysis. Transgenic plant lines were recovered that exhibited dramatically reduced levels of raffinose in mature <span class="trait">seed,</span> and these lines were further analyzed for other phenotypes such as development and <span class="trait">yield.</span> Additionally, a precision-fed rooster assay was conducted to measure the true metabolizable energy (TME) in full-fat soybean meal made from the wild-type or transgenic low-raffinose soybean lines. Transgenic low-raffinose soy had a measured TME of 2,703 kcal/kg, an increase as compared with 2,411 kcal/kg for wild-type. As low digestible energy is a major limiting factor in the percent of soybean meal that can be used in poultry diets, these results may substantiate the use of higher concentrations of low-raffinose, full-fat soy in formulated livestock diets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soya bean seeds accumulating β-carotene exhibit the collateral enhancements of oleate and protein content traits.</h2>
			<h3>PMID:25400247  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schmidt MA(1), Parrott WA, Hildebrand DF, Berg RH, Cooksey A, Pendarvis K, He Y, McCarthy F, Herman EM.
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transgenic soya bean (Glycine max) plants overexpressing a seed-specific bacterial phytoene synthase gene from Pantoea ananatis modified to target to plastids accumulated 845 μg β carotene g(-1) dry <span class="trait">seed</span> weight with a desirable 12:1 ratio of β to α. The β carotene accumulating seeds exhibited a shift in oil composition increasing oleic acid with a concomitant decrease in linoleic acid and an increase in <span class="trait">seed</span> protein content by at least 4% (w/w). Elevated β-carotene accumulating soya bean cotyledons contain 40% the amount of abscisic acid compared to nontransgenic cotyledons. Proteomic and nontargeted metabolomic analysis of the mid-maturation β-carotene cotyledons compared to the nontransgenic did not reveal any significant differences that would account for the altered phenotypes of both elevated oleate and protein content. Transcriptomic analysis, confirmed by RT-PCR, revealed a <span class="trait">number</span> of significant differences in ABA-responsive transcripton factor gene expression in the <span class="gene">crtB</span> transgenics compared to nontransgenic cotyledons of the same maturation stage. The altered <span class="trait">seed</span> composition traits seem to be attributed to altered ABA hormone levels varying transcription factor expression. The elevated β-carotene, oleic acid and protein traits in the β-carotene soya beans confer a substantial additive nutritional quality to soya beans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant growth-promotion (PGP) activities and molecular characterization of rhizobacterial strains isolated from soybean (Glycine max L. Merril) plants against charcoal rot pathogen, Macrophomina phaseolina.</h2>
			<h3>PMID:21833548  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choudhary DK(1).
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Charcoal rot disease, caused by the fungus Macrophomina phaseolina, leads to significant <span class="trait">yield</span> losses of soybean crops. One strategy to control charcoal rot is the use of antagonistic, root-colonizing bacteria. Rhizobacteria A(5)F and FPT(7)21 and Pseudomonas sp. strain GRP(3) were characterized for their plant growth-promotion activities against the pathogen. Rhizobacterium FPT(7)21 exhibited higher antagonistic activity against the pathogen on dual plate assay compared to strain A(5)F and GRP(3). FPT(7)21 and GRP(3) gave decreased disease intensity in terms of average <span class="trait">number</span> of pathogen-infested plants. Lipoxygenase (LOX), phenylalanine ammonia-lyase (PAL), and peroxidase (POD) activities were estimated in extracts of plants grown from seeds that were treated with rhizobacteria, and inoculated with spore suspension of M. phaseolina. The activity of these enzymes after challenge with the test pathogen increased. Strains FPT(7)21 and GRP(3) exhibited maximum increases in LOX, <span class="gene">PAL</span> and POD activity (U mg(-1) fresh leaf wt) compared to strain A(5)F.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL underlying somatic embryogenesis capacity of immature embryos in soybean (Glycine max (L.) Merr.).</h2>
			<h3>PMID:19960195  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song X(1), Han Y, Teng W, Sun G, Li W.
				<br/> 
				年份：2010 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High embryogenesis capacity of soybean (Glycine max (L.) Merr.) in vitro possessed potential for effective genetic engineering and tissue culture. The objects of this study were to identify quantitative trait loci (QTL) underlying embryogenesis traits and to identify genotypes with higher somatic embryogenesis capacity. A mapping population, consisting of 126 F(5:6) recombinant inbred lines, was advanced by single-seed-descent from cross between Peking (higher primary and secondary embryogenesis) and Keburi (lower primary and secondary embryogenesis). This population was evaluated for primary embryogenesis capacity from immature embryo cultures by measuring the frequency of somatic embryogenesis (FSE), the somatic embryo <span class="trait">number</span> per explant (EPE) and the efficiency of somatic embryogenesis (ESE). A total of 89 simple sequence repeat markers were used to construct a genetic linkage map. Six QTL were associated with somatic embryogenesis. Two QTL for FSE were found, QFSE-1 (Satt307) and QFSE-2 (Satt286), and both were located on linkage group <span class="gene">C2</span> that explained 45.21 and 25.97% of the phenotypic variation, respectively. Four QTL for EPE (QEPE-1 on MLG H, QEPE-2 on MLG G and QEPE-3 on MLG G) were found, which explained 7.11, 7.56 and 6.12% of phenotypic variation, respectively. One QTL for ESE, QESE-1 (Satt427), was found on linkage group G that explained 6.99% of the phenotypic variation. QEPE-2 and QESE-1 were located in the similar region of MLG G. These QTL provide potential for marker assistant selection of genotypes with higher embryogenesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Collaborative trial validation studies of real-time PCR-based <span class="gene">GMO</span> screening methods for detection of the bar gene and the ctp2-cp4epsps construct.</h2>
			<h3>PMID:19807158  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grohmann L(1), Brünen-Nieweler C, Nemeth A, Waiblinger HU.
				<br/> 
				年份：2009 Oct 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polymerase Chain Reaction (PCR)-based screening methods targeting genetic elements commonly used in genetically modified (GM) plants are important tools for the detection of GM materials in food, feed, and <span class="trait">seed</span> samples. To expand and harmonize the screening capability of enforcement laboratories, the German Federal Office of Consumer Protection and Food Safety conducted collaborative trials for interlaboratory validation of real-time PCR methods for detection of the phosphinothricin acetyltransferase (bar) gene from Streptomyces hygroscopicus and a construct containing the 5-enolpyruvylshikimate-3-phosphate synthase gene from Agrobacterium tumefaciens sp. strain CP4 (ctp2-cp4epsps), respectively. To assess the limit of detection, precision, and accuracy of the methods, laboratories had to analyze two sets of 18 coded genomic DNA samples of events LLRice62 and MS8 with the bar method and NK603 and GT73 with the ctp2-cp4epsps method at analyte levels of 0, 0.02, and 0.1% GM content, respectively. In addition, standard DNAs were provided to the laboratories to generate calibration curves for copy <span class="trait">number</span> quantification of the bar and ctp2-cp4epsps target sequences present in the test samples. The study design and the results obtained are discussed with respect to the difficult issue of developing general guidelines and concepts for the collaborative trial validation of qualitative PCR screening methods.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of palm kernel meal and corn distillers grains in corn silage-based diets for lactating dairy cows.</h2>
			<h3>PMID:16772589  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carvalho LP(1), Cabrita AR, Dewhurst RJ, Vicente TE, Lopes ZM, Fonseca AJ.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of increasing levels of solvent-extracted palm kernel meal (SPKM) and corn distillers dried grains (CDG) in corn silage-based diets on feed intake and milk production were examined in 2 experiments. In Experiment 1, 20 Holstein cows averaging 100 d in milk (DIM) (SD = 61.5) at the start of the experiment were utilized in an 11-wk randomized complete block design with 4 treatments in 5 blocks to study effects of increasing levels of SPKM in the diet. During a 3-wk preliminary period, cows were fed a standard diet. At the end of the preliminary period, cows were blocked by 4% fat-corrected milk <span class="trait">yield,</span> parity <span class="trait">number</span> (primiparous and multiparous), and DIM, and were assigned randomly to 1 of 4 experimental diets. The total mixed ration (TMR) consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of SPKM were achieved by replacing protein sources and citrus pulp with SPKM and urea (0, 5, 10, and 15% SPKM and 0.06, 0.22, 0.38, and 0.55% urea for SPKM0, SPKM5, SPKM10, and SPKM15, respectively). In Experiment 2, 18 Holstein cows averaging 93 <span class="gene">DIM</span> (SD = 49.1) at the start of the experiment were utilized in an 11-wk randomized complete block design with 3 treatments in 6 blocks to study effects of increasing levels of CDG in the diet. The preliminary period lasted for 2 wk. Assignment of cows to treatments was the same as in Experiment 1. The TMR consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of CDG were achieved by replacing soybean meal and citrus pulp with CDG and urea (0, 7, and 14% CDG and 0, 0.22, and 0.49% urea for CDG0, CDG7, and CDG14, respectively). There were no significant treatment effects on dry matter intake, milk <span class="trait">yield,</span> or milk composition in Experiment 1. Inclusion of SPKM tended to increase protein and lactose contents of milk. The SPKM0 diet promoted body weight loss. There were no treatment effects in Experiment 2, except for milk protein content, which was decreased by CDG. Plasma Lys concentration tended to be affected by SPKM and CDG inclusions. Plasma concentrations of 3-methylhistidine and Leu seemed to be related to body protein degradation/synthesis. The feeding of SPKM up to 15% in the diet decreased feed costs without detrimental effects on productive responses and tended to increase milk protein content. The inclusion of CDG in diets based on corn silage and corn byproducts might decrease milk protein content due to an unbalanced supply of AA, particularly Lys.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, <span class="trait">number</span> of nodes on main stem, <span class="trait">lodging,</span> <span class="trait">number</span> of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being <span class="trait">seed</span> oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Vegetative and Seed-Specific Forms of Tonoplast Intrinsic Protein in the Vacuolar Membrane of Arabidopsis thaliana.</h2>
			<h3>PMID:16668923</h3>
				

				
			<p>
				作者：Höfte H(1), Hubbard L, Reizer J, Ludevid D, Herman EM, Chrispeels MJ.
				<br/> 
				年份：1992 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reports from a <span class="trait">number</span> of laboratories describe the presence of a family of proteins (the major intrinsic protein family) in a variety of organisms. These proteins are postulated to form channels that function in metabolite transport. In plants, this family is represented by the product of NOD26, a nodulation gene in soybean that encodes a protein of the peribacteroid membrane, and tonoplast intrinsic protein (TIP), an abundant protein in the tonoplast of protein storage vacuoles of bean seeds (KD Johnson, H Höfte, MJ Chrispeels [1990] Plant Cell 2: 525-532). Other homologs that are induced by water stress in pea and in Arabidopsis thaliana and that are expressed in the roots of tobacco have been reported, but the location of the proteins they encode is not known. We now report the presence and derived amino acid sequences of two different <span class="gene">TIP</span> proteins in A. thaliana. alpha-TIP is a seed-specific protein that has 68% amino acid sequence identity with bean <span class="trait">seed</span> TIP; gamma-TIP is expressed in the entire vegetative body of A. thaliana and has 58% amino acid identity with bean <span class="trait">seed</span> TIP. Both proteins are associated with the tonoplast. Comparisons of the derived amino acid sequences of the seven known plant proteins in the major intrinsic protein family show that genes with similar expression patterns (e.g. water stress-induced or <span class="trait">seed</span> specific) are more closely related to each other than the three A. thaliana homologs are related. We propose that the nonoverlapping gene expression patterns reported here, and the evolutionary relationships indicated by the phylogenetic tree, suggest a functional specialization of these proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Somatic embryogenesis and plantlet regeneration in the soybean Glycine max.</h2>
			<h3>PMID:24254079</h3>
				

				
			<p>
				作者：Li BJ(1), Langridge WH, Szalay AA.
				<br/> 
				年份：1985 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A tissue culture procedure for the regeneration of somatic embryos and plantlets from somatic cells of the soybean Glycine max is described. Bean pods of soybean cv. TGM119 were immersed in liquid nitrogen for 20 minutes. Young embryos were excised from the immature seeds and cultured to form calli. Calli grown from the young embryos were incubated in liquid culture for two weeks. The liquid suspension culture was filtered to obtain single cells. The soybean cells were cultured for one month in a liquid medium in hanging drop cultures for development into proembryoids. The proembryoids were maintained on a solid growth medium for 40 days. The resultant callus tissue was transferred into <span class="gene">MS</span> media containing selected combinations and concentrations of 2,4-Dichlorophenoxyacetic acid, Naphthaleneacetic acid, Kinetin, Benzyladenine and Indoleacetic acid. In the presence of Benzyladenine (0.2 mg/l) and Indoleacetic acid (0.01 mg/l), globular and heart shaped somatic embryos were formed on the surface of the calli. Calli containing somatic embryos were transferred into liquid medium and incubated under low light conditions. After six months further incubation, more than 1,000 plantlets and a large <span class="trait">number</span> of somatic embryoids at various developmental stages were obtained per flask.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>