<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed Set</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed Set</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Loss-of-function mutations affecting a specific <italic> Glycine max </italic> R2R3 MYB transcription factor result in brown hilum and brown <span class="trait">seed</span> coats</h2>
			<h3>PMID:22070454</h3>
				

				
					<h3>PMCID:3229458</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Gillman </surname>  <given-names> Jason D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Jason.Gillman@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Tetlow </surname>  <given-names> Ashley </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> altz8b@mail.mizzou.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lee </surname>  <given-names> Jeong-Deong </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> jdlee@knu.ac.kr </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Shannon </surname>  <given-names> J Grover </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ShannonG@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Bilyeu </surname>  <given-names> Kristin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Kristin.Bilyeu@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Although modern soybean cultivars feature yellow <span class="trait">seed</span> coats, with the only color variation found at the hila, the ancestral condition is black <span class="trait">seed</span> coats. Both <span class="trait">seed</span> coat and hila coloration are due to the presence of phenylpropanoid pathway derivatives, principally anthocyanins. The genetics of soybean <span class="trait">seed</span> coat and hilum coloration were first investigated during the resurgence of genetics during the 1920s, following the rediscovery of Mendel's work. Despite the inclusion of this phenotypic marker into the extensive genetic maps developed for soybean over the last twenty years, the genetic basis behind the phenomenon of brown <span class="trait">seed</span> coats (the R locus) has remained undetermined until now. RESULTS: In order to identify the gene responsible for the r gene effect (brown hilum or <span class="trait">seed</span> coat color), we utilized bulk segregant analysis and identified recombinant lines derived from a population segregating for two phenotypically distinct alleles of the R locus. Fine mapping was accelerated through use of a novel, bioinformatically determined <span class="trait">set</span> of Simple Sequence Repeat (SSR) markers which allowed us to delimit the genomic region containing the r gene to less than 200 kbp, despite the use of a mapping population of only 100 F6 lines. Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype. We observed a near perfect correlation between the mRNA expression levels of the functional R gene candidate and an UDP-glucose:flavonoid 3-O-glucosyltransferase (UF3GT) gene, which is responsible for the final step in anthocyanin biosynthesis. In contrast, when a null allele of <span class="gene">Glyma09g36990</span> is expressed no upregulation of the UF3GT gene was found. CONCLUSIONS: We discovered an allelic series of four loss of function mutations affecting our R locus gene candidate. The presence of any one of these mutations was perfectly correlated with the brown <span class="trait">seed</span> coat/hilum phenotype in a broadly distributed survey of soybean cultivars, barring the presence of the epistatic dominant I allele or gray pubescence, both of which can mask the effect of the r allele, resulting in yellow or buff hila. These findings strongly suggest that loss of function for one particular <span class="trait">seed</span> coat-expressed R2R3 MYB gene is responsible for the brown <span class="trait">seed</span> coat/hilum phenotype in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In order to identify the gene responsible for the <italic> r </italic> gene effect (brown hilum or <span class="trait">seed</span> coat color), we utilized bulk segregant analysis and identified recombinant lines derived from a population segregating for two phenotypically distinct alleles of the <italic> R </italic> locus. Fine mapping was accelerated through use of a novel, bioinformatically determined <span class="trait">set</span> of Simple Sequence Repeat (SSR) markers which allowed us to delimit the genomic region containing the <italic> r </italic> gene to less than 200 kbp, despite the use of a mapping population of only 100 F <sub> 6 </sub> lines. Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype. We observed a near perfect correlation between the mRNA expression levels of the functional <italic> R </italic> gene candidate and an <italic> UDP-glucose:flavonoid 3-O-glucosyltransferase (UF3GT) </italic> gene, which is responsible for the final step in anthocyanin biosynthesis. In contrast, when a null allele of <span class="gene">Glyma09g36990</span> is expressed no upregulation of the <italic> UF3GT </italic> gene was found.
					</p>
				
					<p>&nbsp;&nbsp;
						The visual appearance of the soybean <span class="trait">seed</span> itself has also been altered as a result of domestication: All <italic> Glycine soja </italic> accessions in the USDA GRIN germplasm collection possess black <span class="trait">seed</span> coats, whereas the majority of <italic> Glycine max </italic> germplasm (12880/18585 Soybean entries, accessed 06/07/2011) possess yellow <span class="trait">seed</span> coats. Although a small market exists for black soybeans, all modern high yielding cultivars feature yellow <span class="trait">seed</span> coats, with a range of hila colors present (brown, black, imperfect black, buff, yellow). Cultivars with pale hila are highly prized for natto and tofu production [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Because hilum coloration is controlled by a small number of genes [ <xref ref-type="bibr" rid="B6"> 6 </xref> ], this trait is frequently used by breeders as a readily assayed visible marker for the presence of "off-types" in soybean <span class="trait">seed</span> lots. <span class="trait">Seed</span> coat and hilum color are relatively simple epistatic multi-genic traits, and variation in hilum and <span class="trait">seed</span> coat pigmentation appears to be due to the interaction of four independent loci: <italic> Inhibitor </italic> ( <italic> I </italic> ), <italic> Tawny </italic> ( <italic> T </italic> ), an unnamed locus termed <italic> R </italic> , and the flower color locus <italic> <span class="gene">W1</span> </italic> [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ](Table <xref ref-type="table" rid="T1"> 1 </xref> ). Other loci with minor effects have been described, but these have not been mapped and the genetics have been incompletely discerned [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Two anthocyanin glycosides form the predominant colored compounds in black <span class="trait">seed</span> coats: cyanidin-3-monoglucoside and delphinidin-3-monoglucoside [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. These are formed through the action of UDP-glucose:flavonoid 3-O-glucosyltransferase (UF3GT) enzymes, which specifically transfer a glucose moiety from UTP to the 3' position of cyanidin and delphinidin (recently reviewed in [ <xref ref-type="bibr" rid="B13"> 13 </xref> ], Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). This glycosylation is thought to increase the stability and solubility of the cyanidin molecule [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. In lines with brown <span class="trait">seed</span> coats ( <italic> r </italic> ), cyanidin accumulates, though high levels of proanthocyanidins are also present [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Recently, two highly similar co-expressed <italic> UF3GT </italic> genes <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130)</span> were determined to be expressed in <span class="trait">seed</span> coats of black seeded soybean lines, and these genes have been demonstrated to specifically transfer a glucose moiety to the cyanidin molecule at the 3'-hydroxyl group, resulting in the formation of cyanidin-3-glucoside [ <xref ref-type="bibr" rid="B15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> coat color is primarily under control of the <italic> Inhibitor </italic> locus, which has at least four classically defined genetic alleles [ <xref ref-type="bibr" rid="B8"> 8 </xref> ], listed here from the most dominant to the least: <italic> I </italic> (largely colorless seeds) &#x0003e; <italic> i <sup> i </sup> </italic> (color restricted to hilum) &#x0003e; <italic> i <sup> k </sup> </italic> ("saddle;" color in hilum and spreading slightly beyond the hilum) &#x0003e; <italic> i </italic> (seeds completely black). <italic> Inhibitor </italic> acts in a dominant, gain-of-function manner with maternal-effect inheritance, and results in <span class="trait">seed</span> coats appearing pale yellow due to the absence of anthocyanins [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. Both the dominant <italic> Inhibitor </italic> allele ( <italic> I </italic> ) and the <italic> i <sup> i </sup> </italic> alleles have been shown to be due to naturally occurring, gene-silencing effects derived from linked but independent Chalcone Synthase (CHS) gene clusters (chromosome 8, LG A2) that generate siRNA which target <span class="gene">CHS</span> gene transcripts specifically within the <span class="trait">seed</span> coat for degradation [ <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Lines which have the dominant <italic> I </italic> allele can still exhibit some traces of color within the hilum, with the specific hilum coloration due to the allelic status at three other genetic loci: <italic> Tawny </italic> , <italic> R </italic> , and <italic> <span class="gene">W1</span> </italic> [ <xref ref-type="bibr" rid="B6"> 6 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ] (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Hilum tissue is not maternally-derived, in contrast to the <span class="trait">seed</span> coat [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. In lines with the recessive ( <italic> i </italic> ) allele, <span class="trait">seed</span> coat color is brown, imperfect black, buff or black, dependent on the allelic status of the <italic> Tawny, R and <span class="gene">W1</span> </italic> loci (Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Tawny </italic> locus has two pleiotropic effects: homozygosity for the gray ( <italic> t </italic> ) allele results in gray pubescence at maturity and, in lines carrying the combination of the <italic> i <sup> i </sup> </italic> allele of the <italic> Inhibitor </italic> locus, a functional <italic> R </italic> gene, and purple flowers ( <italic> <span class="gene">W1</span> </italic> ), <span class="trait">seed</span> which feature "imperfect black" hila (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Alternatively, gray pubescent ( <italic> t </italic> ) lines carrying the <italic> i <sup> i </sup> </italic> allele of the <italic> Inhibitor </italic> locus, a functional or nonfunctional <italic> R </italic> , and white flowers ( <italic> w1 </italic> ) produce <span class="trait">seed</span> which feature buff hila [ <xref ref-type="bibr" rid="B8"> 8 </xref> ] (Table <xref ref-type="table" rid="T1"> 1 </xref> ). The phenotypic effects of the recessive allele of <italic> Tawny </italic> have been discerned to be due to loss of function mutations affecting a <italic> flavonoid 3' hydroxylase </italic> gene <span class="gene">(Glyma06g21920)</span> [ <xref ref-type="bibr" rid="B24"> 24 </xref> ]. At the chemical level, this is the result of a reduction in the accumulation of anthocyanins within the hilum, and the presence of pelargonidin (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ), which does not accumulate in lines carrying the wild type version of the <italic> Tawny </italic> locus [ <xref ref-type="bibr" rid="B10"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Another locus, classically termed <italic> R </italic> , also interacts epistatically with the <italic> Tawny </italic> and <italic> Inhibitor </italic> loci (as well as the <italic> <span class="gene">W1</span> </italic> locus) to control hilum and <span class="trait">seed</span> coat colors [ <xref ref-type="bibr" rid="B8"> 8 </xref> ] (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Lines with a functional <italic> Tawny </italic> gene and homozygous for the recessive allele of the <italic> R </italic> locus possess either brown <span class="trait">seed</span> coats or brown hilum, dependent on the allelic status of the <italic> Inhibitor </italic> locus ( <italic> i </italic> or <italic> i <sup> i </sup> </italic> respectively). Although the genetics behind this trait were well resolved shortly after the rediscovery of Mendel's work in the 1920s [ <xref ref-type="bibr" rid="B6"> 6 </xref> ], the molecular genetic basis has not been ascertained. Despite this, the ease of phenotyping has resulted in the inclusion of this locus in the development of genetic maps for soybean [ <xref ref-type="bibr" rid="B25"> 25 </xref> - <xref ref-type="bibr" rid="B27"> 27 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Epistatic and pleiotropic interactions are the norm for genes involved in soybean coloration (Table <xref ref-type="table" rid="T1"> 1 </xref> ). For example, loss of function mutations affecting a <italic> flavonoid 3'5'-hydroxylase </italic> gene ( <italic> w1 </italic> , <italic> F3'5'H </italic> , Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) have been demonstrated to result in two phenotypes: white flowers and loss of purple pigment in hypocotyls [ <xref ref-type="bibr" rid="B28"> 28 </xref> ]. The allelic status of the <italic> <span class="gene">W1</span> </italic> locus, when combined with the recessive gray allele of the Tawny locus, determines if <span class="trait">seed</span> coats or hila are colored "imperfect black" or "buff" (Table <xref ref-type="table" rid="T1"> 1 </xref> ) [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Loss of function mutations affecting a gene involved in the terminal end of the anthocyanin biosynthetic pathway have been suggested as the cause of the recessive brown <span class="trait">seed</span> coat/hilum phenotype (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). Possible candidates have included <italic> UF3GT </italic> , <italic> Anthocyanidin Synthase </italic> ( <italic> <span class="gene">ANS</span> </italic> ) and/or <italic> Dihydroxyflavone Reductase </italic> ( <italic> DFR </italic> ) genes. However, no correlation has been found between the genomic locations of any <italic> UF3GT </italic> , <italic> DFR </italic> or <italic> <span class="gene">ANS</span> </italic> gene and the location of the <italic> R </italic> gene [ <xref ref-type="bibr" rid="B29"> 29 </xref> ]. Alternately, a transcription factor or other regulatory element could be responsible for the brown hilum/seed coat phenomenon. The objective of this work was to identify the specific gene and causative basis behind the phenomenon of brown hilum/seed coat coloration, historically defined as the R locus, in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The generation of the F <sub> 6 </sub> RIL mapping population, derived from a cross between Jake X PI 283327, was previously described [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. Jake (PI 643912) has tawny pubescence, purple flowers, and shiny yellow <span class="trait">seed</span> with black hila ( <italic> i <sup> i </sup> T R <span class="gene">W1</span> </italic> )[ <xref ref-type="bibr" rid="B31"> 31 </xref> ]. The brown hila line, PI 283327 has tawny pubescence, purple flowers, and yellow <span class="trait">seed</span> with brown hila ( <italic> i <sup> i </sup> , T, r, <span class="gene">W1</span> </italic> ) (USDA GRIN germplasm collection, accessed 06/22/2011 ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/npgs/"> http://www.ars-grin.gov/npgs/ </ext-link> ). The reference cultivar Williams 82, for which the genome sequence was determined [ <xref ref-type="bibr" rid="B32"> 32 </xref> ], has tawny pubescence, white flowers and yellow <span class="trait">seed</span> with black hila ( <italic> i <sup> i </sup> , T, R, w1 </italic> ) [ <xref ref-type="bibr" rid="B33"> 33 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Three primer pairs from the novel SSR <span class="trait">set</span> (BARCSOYSSR 09_1475, 09_1501 and 09_1566 were examined for all 100 RIL lines. For the majority, the hilum color phenotype was correlated with the expected parental polymorphic band. We also observed seven individual RILs which possessed recombination events within the region identified on chromosome <span class="gene">Gm09/LG</span> K (Figure <xref ref-type="fig" rid="F2"> 2A </xref> ). We examined these seven RILs using all novel polymorphic SSRs markers within this region, and compared the marker genotype to the RIL phenotype (Table <xref ref-type="table" rid="T2"> 2 </xref> Figure <xref ref-type="fig" rid="F2"> 2A </xref> ). Our methodology allowed us to fine-map the location of the <italic> R </italic> gene to a predicted region of less than 200 kbp with only 100 RIL lines. This region in Williams 82 contains 23 predicted open reading frames, with another 3 genes annotated as pseudogenes (Figure <xref ref-type="fig" rid="F2"> 2B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						BLAST searches using the 26 candidate genes were performed against NCBI ( <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/"> http://www.ncbi.nlm.nih.gov/ </ext-link> ) and TAIR ( <ext-link ext-link-type="uri" xlink:href="http://www.arabidopsis.org"> http://www.arabidopsis.org </ext-link> ) databases to search for candidate genes. BLAST searches revealed four tandem genes which featured homology to the R2R3 MYB transcription factor gene family: <span class="gene">Glyma09g36970,</span> <span class="gene">Glyma09g36980,</span> 09g36990 and <span class="gene">Glyma09g37010.</span> R2R3 MYB genes have been shown to control flux through the phenylpropanoid pathway, and mutants in multiple species are associated with changes in fruit, flower and/or <span class="trait">seed</span> color (recently reviewed in [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]). These four tandem R2R3 MYB genes are highly similar (~80-90% nucleotide identity, excluding presumed intronic sequence) and may have arisen due to a tandem gene amplification event(s). Strikingly, none of these genes appears to have been identified in recent <span class="trait">seed</span> focused studies using RNAseq methods [ <xref ref-type="bibr" rid="B42"> 42 </xref> , <xref ref-type="bibr" rid="B43"> 43 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Because soybean hilum tissue is extremely small and difficult to accurately dissect from seeds in non-pigmented stages, we utilized a large seeded soybean line with brown <span class="trait">seed</span> coats (PI 567115 B) and a large seeded line with black <span class="trait">seed</span> coats (PI 84970) to examine mRNA expression. In order to assess whether a subset of these four tandem genes were pseudogenes and/or expressed in <span class="trait">seed</span> coat tissue <span class="gene">(Glyma09g36970</span> is annotated as a pseudogene in the current whole genome shotgun sequence build), we utilized qRT-PCR. Only one of these candidate R2R3 MYB genes, <span class="gene">Glyma09g36990,</span> was expressed in any of the tissues examined (leaf, <span class="trait">seed</span> cotyledons, and <span class="trait">seed</span> coats). Gene transcripts from <span class="gene">Glyma09g36990</span> were present in the <span class="trait">seed</span> coats of both a brown seeded and a black seeded cultivar. However, this gene was not expressed in either cotyledon tissue (Figure <xref ref-type="fig" rid="F3"> 3A </xref> ) or in leaves (data not shown). It is not clear if the other three R2R3 MYB genes in the cluster are expressed in other tissues. Nor is the role these genes play in soybean physiology known, if any.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Quantitative RT-PCR of RNA isolated from <span class="trait">seed</span> coat and cotyledon tissue at four stages of development </bold> . Each data point represents the average gene expression for two biological replicates, with three technical replicates for each biological replicate. Vertical bars represent one standard deviation. X-axis indicates days prior to <span class="trait">seed</span> maturity. Y-axis indicates gene expression relative to CONS6. <bold> 3A: </bold> qRT-PCR of <italic> R </italic> gene candidate <span class="gene">Glyma09g36990,</span> expressed as a relative measure of CONS6. <bold> 3B: </bold> qRT-PCR of anthocyanidin synthase gene expression ( <italic> <span class="gene">ANS</span> </italic> , non-gene specific), relative to CONS6. <bold> 3C: </bold> qRT-PCR of <italic> UDP-glucose: flavonoid 3-O-glucosyltransferase </italic> ( <italic> UF3GT </italic> , <span class="gene">Glyma08g07130)</span> gene expression, relative to CONS6.
					</p>
				
					<p>&nbsp;&nbsp;
						PCR and Sanger sequencing of exons (and partial intronic sequence) was used to evaluate the <span class="gene">Glyma09g36990</span> gene for polymorphisms in a selection of lines: Jake (black hilum), PI 283327 (brown hilum), Williams 82 (black hilum), PI 84970 (black <span class="trait">seed</span> coats) and PI 567115 B (brown <span class="trait">seed</span> coats). We discovered a single-base deletion within exon 2 in PI 283327 and PI 567115 B that results in a frameshift mutation (C377-, relative to the start codon) (Figure <xref ref-type="fig" rid="F4"> 4 </xref> , details in Additional File <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). The open reading frame for <span class="gene">Glyma09g36990</span> was allelic between Williams 82, Jake and PI 84970.
					</p>
				
					<p>&nbsp;&nbsp;
						In all cases where we observed an intact open reading frame, we noted the phenotype of imperfect black hilum ( <italic> i <sup> i </sup> R t <span class="gene">W1</span> </italic> ), buff hilum ( <italic> i <sup> i </sup> R t w1 </italic> ), black hilum ( <italic> i <sup> i </sup> R T </italic> ) or black <span class="trait">seed</span> coat ( <italic> i R T </italic> ), dependent on the allelic status of the <italic> Inhibitor </italic> and <italic> Tawny </italic> loci (Additional File <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Any of these four loss of function alleles resulted in either brown hilum ( <italic> i <sup> i </sup> r T </italic> ), brown <span class="trait">seed</span> coat <italic> (i r T </italic> ) or buff hila ( <italic> i <sup> i </sup> r t </italic> ). In all cases, we observed a perfect association between the presence of one of the four loss of function alleles and brown hilum or brown <span class="trait">seed</span> coats, barring the presence of the epistatic dominant <italic> I </italic> allele or gray pubescence, both of which can mask the effect of the <italic> r </italic> allele, resulting in yellow or buff hila (Additional File <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). These epistatic interactions (and masking in the case of <italic> Inhibitor </italic> ) are due to the placement of the step affected by the R2R3 MYB gene at the terminal end of the anthocyanin biosynthesis pathway (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). Any one of the loss of function mutations affecting the <italic> R </italic> gene are necessary and sufficient for brown <span class="trait">seed</span> coat and/or hilum coloration. However, the phenotypic effect can be masked or modulated by the presence of certain alleles of the Inhibitor and Tawny loci (Table <xref ref-type="table" rid="T1"> 1 </xref> Additional File <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						If the candidate <italic> R </italic> gene is controlling expression of a gene which forms a rate limited step in anthocyanin production, we hypothesized that a correlation would exist between 1) <italic> R </italic> gene expression levels, 2) the appearance of color compounds, and 3) the expression of <italic> <span class="gene">ANS</span> </italic> and/or <italic> UF3GT </italic> genes in developing <span class="trait">seed</span> coats. We examined a time course of <span class="trait">seed</span> coat and <span class="trait">seed</span> cotyledons by qRT-PCR (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ) for expression of three genes: the <italic> R </italic> gene candidate, <italic> <span class="gene">ANS</span> </italic> , and <italic> UF3GT </italic> . <span class="trait">Seed</span> coats from the large seeded line with brown <span class="trait">seed</span> coats (PI 567115 B) and one with black <span class="trait">seed</span> coats (PI 84970) were investigated for quantitation of steady state transcripts. We selected four time-points corresponding to the development of pigmentation during <span class="trait">seed</span> growth and maturation for PI 84970 (black <span class="trait">seed</span> coats) and PI 567115 B (brown <span class="trait">seed</span> coats) (Additional File <xref ref-type="supplementary-material" rid="S4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Although there are apparently two <italic> UF3GT </italic> genes expressed in <span class="trait">seed</span> coats in soybean <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130),</span> only one of these genes <span class="gene">(Glyma08g07130)</span> is not expressed in cotyledon tissue [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. We elected to focus on this gene for qRT-PCR, as we noted a virtual absence of <italic> <span class="gene">ANS</span> </italic> or <italic> R </italic> gene expression in cotyledons (Figure <xref ref-type="fig" rid="F3"> 3A </xref> and <xref ref-type="fig" rid="F3"> 3B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We observed a near-perfect coefficient of correlation (R <sup> 2 </sup> = 0.96) between the level of expression (relative to an internal control CONS6) of the putative <italic> R </italic> gene and a <italic> UF3GT </italic> gene <span class="gene">(Glyma08g07130)</span> (Figure <xref ref-type="fig" rid="F3"> 3A </xref> and <xref ref-type="fig" rid="F3"> 3C </xref> ). In contrast, we observed a weak correlation between expression of the <italic> R </italic> gene and <italic> <span class="gene">ANS</span> </italic> gene expression (R <sup> 2 </sup> = 0.66) in the black <span class="trait">seed</span> coat line (Figure <xref ref-type="fig" rid="F3"> 3A </xref> and <xref ref-type="fig" rid="F3"> 3B </xref> ). In the brown seeded line PI 567115 B, no significant correlation was found between <italic> R </italic> gene expression levels and either <italic> <span class="gene">ANS</span> </italic> or <italic> UF3GT </italic> expression levels (Figure <xref ref-type="fig" rid="F3"> 3A-C </xref> ). During early and mid-development stages <italic> R </italic> gene expression is similar in both black and brown <span class="trait">seed</span> coat lines, though <italic> R </italic> gene expression declined during the last stages of development of the brown seeded line, in contrast to the high expression noted for the black <span class="trait">seed</span> coat lines (Figure <xref ref-type="fig" rid="F3"> 3A </xref> ). In striking contrast to the increase in expression of <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> during <span class="trait">seed</span> coat maturation of the black <span class="trait">seed</span> coat line, only negligible <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression was observed in the brown <span class="trait">seed</span> coat line as seeds approached maturity (Figure <xref ref-type="fig" rid="F3"> 3B </xref> and <xref ref-type="fig" rid="F3"> 3C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						One of the characteristic features of the accumulation of plant pigments that has emerged is the regulation of critical structural genes by R2R3 MYB transcription factors in complexes with bHLH transcription factors and WD40 proteins [ <xref ref-type="bibr" rid="B48"> 48 </xref> ]. R2R3 MYB genes tend to display limited homology (aside from the highly conserved DNA binding region), and the code by which R2R3 MYB genes bind to specific sequences has not been well elucidated [ <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. These difficulties can complicate phylogenetic analysis and the assignment of genes to paralogous functions. Nevertheless, the soybean <italic> R </italic> gene candidate <span class="gene">Glyma09g36990</span> shows homology to R2R3 MYB genes (Additional File <xref ref-type="supplementary-material" rid="S3"> 3 </xref> ). In the past few years a plethora of R2R3 genes have been found which directly impact expression of <italic> UF3GT </italic> and/or phenylpropanoid pathway derived color compound accumulation in <span class="trait">seed</span> coats [ <xref ref-type="bibr" rid="B49"> 49 </xref> ], fruits [ <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> - <xref ref-type="bibr" rid="B52"> 52 </xref> ], flowers [ <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B53"> 53 </xref> , <xref ref-type="bibr" rid="B54"> 54 </xref> ] and other tissues [ <xref ref-type="bibr" rid="B55"> 55 </xref> - <xref ref-type="bibr" rid="B57"> 57 </xref> ]. Aside from the aesthetic appeal of colored compounds, many of these color compounds may have roles as nutraceuticals [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. Loss of function mutations within R2R3 genes have also been discerned as causative for loss of anthocyanin accumulation in other plant species [ <xref ref-type="bibr" rid="B57"> 57 </xref> , <xref ref-type="bibr" rid="B58"> 58 </xref> ]. Although an R2R3 MYB gene(s) would be logical <italic> a priori </italic> candidates for the underlying basis of the <italic> R </italic> locus, the low level of overall homology among R2R3 MYB genes, the presence of at least 448 MYB genes within the soybean genome [ <xref ref-type="bibr" rid="B59"> 59 </xref> ] and the relatively poorly defined genetic map location for the <italic> R </italic> locus [ <xref ref-type="bibr" rid="B25"> 25 </xref> - <xref ref-type="bibr" rid="B27"> 27 </xref> ] precluded candidate gene analysis prior to our fine-mapping effort.
					</p>
				
					<p>&nbsp;&nbsp;
						Here we used genetic mapping and candidate gene association in a RIL population and a panel of soybean lines with defined coloration <span class="trait">(seed</span> coat and hilum, pubescence, and flower) to determine the <italic> R </italic> gene controlling black or brown <span class="trait">seed</span> coat in soybean is the R2R3 MYB gene <span class="gene">Glyma09g36990.</span> Indirect evidence supports a model in which a functional <italic> R </italic> gene acts to promote transcription of the anthocyanidin late pathway structural genes <italic> U3FGT </italic> as well as <italic> <span class="gene">ANS</span> </italic> . These results are consistent with many other instances of a transcriptional regulatory activation control point for genes in the anthocyanidin pathway [ <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> - <xref ref-type="bibr" rid="B58"> 58 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						We performed bulk segregant analysis (BSA) [ <xref ref-type="bibr" rid="B40"> 40 </xref> ] on a F <sub> 6 </sub> -RIL population which had segregated for hilum color [ <xref ref-type="bibr" rid="B30"> 30 </xref> ], derived from a cross between a commercial cultivar with black hila (Jake) and a plant introduction line with brown hila (PI 283327). We utilized a novel <span class="trait">set</span> of bioinformatically derived SSR markers [ <xref ref-type="bibr" rid="B35"> 35 </xref> ] to fine map the <italic> R </italic> gene to less than 200 kilobasepairs, despite using a RIL population of less than 100 individual F <sub> 6 </sub> lines. Analysis of the Williams 82 whole genome shotgun sequence [ <xref ref-type="bibr" rid="B32"> 32 </xref> ] corresponding to this region revealed four tandem R2R3 MYB genes as likely candidates for the authentic <italic> R </italic> gene. R2R3 MYB transcription factors are one of the largest transcription factor families in plants [ <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B44"> 44 </xref> ], and specific R2R3 genes have been identified in a number of species which activate phenylpropanoid biosynthetic genes [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> , <xref ref-type="bibr" rid="B54"> 54 </xref> , <xref ref-type="bibr" rid="B56"> 56 </xref> , <xref ref-type="bibr" rid="B60"> 60 </xref> , <xref ref-type="bibr" rid="B61"> 61 </xref> ]. Only one of the four candidate R2R3 MYB transcription factor genes <span class="gene">(Glyma09g36990)</span> in the genomic region containing <italic> R </italic> proved to be expressed in any of the tissues we examined. The seed-coat specific expression of the functional version of this gene was strongly correlated with the level of expression of a <italic> UF3GT </italic> gene <span class="gene">(Glyma08g07130),</span> which encodes a gene product that carries out the final step in anthocyanin biosynthesis [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. We discovered an allelic series of loss of function mutations affecting our R2R3 gene candidate, and the presence of any of the four loss of function mutations was perfectly correlated with the brown <span class="trait">seed</span> coat/hilum phenotype in a broad distribution of soybean cultivars divergent in <span class="trait">seed</span> coat, hilum and flower color. These findings strongly suggest that loss of function for this particular R2R3 MYB gene is responsible for the brown <span class="trait">seed</span> coat/hilum phenotype in soybean. The presence of multiple independent alleles suggests that this gene was selected during domestication either directly for brown coloration or indirectly for pale hilum colors (due to its epistatic effects with <italic> Inhibitor </italic> and <italic> Tawny </italic> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:SNPs discovery and CAPS marker conversion in soybean.</h2>
			<h3>PMID:20859693  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shu Y(1), Li Y, Zhu Z, Bai X, Cai H, Ji W, Guo D, Zhu Y.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To discover new SNPs and develop an easy assay method in soybean, we compared the high-throughput pyrosequencing ESTs with whole genome sequences in different soybean varieties and identified 3899 SNPs. Transitions were found to be much more frequent than transversions in these SNPs. We found that SNPs were widely distributed in the soybean genome, targeting numerous genes involved in various physiological and biochemical processes influencing important agronomic traits. A <span class="trait">set</span> of 16 SNPs were validated in nine soybean varieties, and seven SNPs were converted into CAPS. From functional gene association analysis, the marker CAPS282 on the 3'-UTR of gene <span class="gene">Glyma07g03490</span> was identified as associated with 100-seed weight in soybean. The SNP discovery and CAPS markers conversion system developed in this study is fast and cost effective, and holds great promise for molecular-assisted breeding of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq Atlas of <italic> Glycine max </italic> : A guide to the soybean transcriptome</h2>
			<h3>PMID:20687943</h3>
				
					<h3>PMCID:3017786</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Severin </surname>  <given-names> Andrew J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> severin@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Woody </surname>  <given-names> Jenna L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jlwoody@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Bolon </surname>  <given-names> Yung-Tsi </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yung-tsi.bolon@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Joseph </surname>  <given-names> Bindu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> bindu@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Diers </surname>  <given-names> Brian W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> bdiers@uiuc.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Farmer </surname>  <given-names> Andrew D </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> adf@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Muehlbauer </surname>  <given-names> Gary J </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> muehl003@umn.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Nelson </surname>  <given-names> Rex T </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> rex.nelson@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Grant </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> david.grant@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Specht </surname>  <given-names> James E </given-names>  </name>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> jspecht1@unl.edu </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Graham </surname>  <given-names> Michelle A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> michelle.graham@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Cannon </surname>  <given-names> Steven B </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> steven.cannon@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A13">  <name>  <surname> May </surname>  <given-names> Gregory D </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> gdm@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A14">  <name>  <surname> Vance </surname>  <given-names> Carroll P </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> carroll.vance@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A15">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> randy.shoemaker@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 8 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A Z-score analysis was conducted to gain insight into the gene expression patterns of each tissue [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. The Z-score numerical value is calculated by taking for each gene and tissue the (RPKM)-normalized log <sub> 2 </sub> -transformed transcript count, subtracting the normalized mean transcript count of all tissues and dividing by the standard deviation of the normalized transcript count of all tissues. The Z-score numerical value measures the number of standard deviations the expression level of a gene in a specific tissue is from the mean expression level in all tissues. The Z-score analysis revealed that aerial and underground tissues are distinguished from <span class="trait">seed</span> tissue by a bimodal expression pattern with more genes from aerial and underground tissues shifted toward higher expression values (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Transcription values in non-seed tissues are less similar than transcription values in stages of <span class="trait">seed</span> development resulting in a greater distribution of Z-score values and a noticeable portion of genes with Z-score values near the positive extreme between 3.4 and 3.6 indicating a high specificity for the tissue. We provide a supplementary list of genes with Z-scores in the 3.4 to 3.6 value range for each tissue (Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> ). To examine the validity of tissue specificity using Z-score analysis, we inspected the gene annotations based on the Dana Farber tentative consensus sequences [ <xref ref-type="bibr" rid="B29"> 29 </xref> ] for all genes greater than 5000 (RPKM) normalized count in nodule tissues. Of the ten genes with this level of expression found between a Z-score value of 3.4 and 3.6 in nodules, all genes had an annotation. We identified four genes: <span class="gene">Glyma10g34290,</span> <span class="gene">Glyma10g34280,</span> <span class="gene">Glyma20g33290</span> and <span class="gene">Glyma10g34260</span> as leghemoglobin A, leghemoglobin C1, leghemoglobin <span class="gene">C2</span> and leghemoglobin C3, respectively. We identified another five genes: <italic> <span class="gene">Glyma13g40400</span> </italic> , <italic> <span class="gene">Glyma14g05690</span> </italic> , <italic> <span class="gene">Glyma15g05010</span> </italic> , <italic> <span class="gene">Glyma19g22210</span> </italic> and <italic> <span class="gene">Glyma13g44100</span> </italic> as nodulin 20, nodulin 22, nodulin 24, nodulin 26B and nodulin 44, respectively. The last gene, <italic> <span class="gene">Glyma08g14020</span> </italic> , was identified as a nodule specific extensin gene based on the PANTHER classification system [ <xref ref-type="bibr" rid="B30"> 30 </xref> ]. These gene annotations support the validity of using high Z-score values for identifying tissue specific genes from RNA-Seq data.
					</p>
				
					<p>&nbsp;&nbsp;
						An inspection of the legume specific genes found only in <span class="trait">seed</span> development revealed three genes with similar expression profiles that vary between no expression in <span class="trait">seed</span> 10-DAF and some of the highest expression seen in the heatmap in <span class="trait">seed</span> 42-DAF. The first two genes, <italic> <span class="gene">Glyma06g08290</span> </italic> and <italic> <span class="gene">Glyma04g08220</span> </italic> are oleosins (based on annotations in the Dana Farber tentative consensus sequences). The third gene, <italic> <span class="gene">Glyma02g01590</span> </italic> , has an annotated of "lectin precursor 1" [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. These three genes were also identified in the heatmap for the highest expressed genes. Oleosins are membrane proteins found in <span class="trait">seed</span> oil bodies [ <xref ref-type="bibr" rid="B36"> 36 </xref> ] whereas lectin precursor 1 is localized to protein storage vacuoles [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]. A more in-depth analysis may be warranted for these genes to determine how their similar expression profiles in <span class="trait">seed</span> development is affected by the negative correlation between protein and oil <span class="trait">seed</span> content.
					</p>
				
					<p>&nbsp;&nbsp;
						As a final example to demonstrate the power of combining a RNA-Seq atlas with the genomic sequence, consider the soybean lipoxygenase genes (LOXs) [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Lipoxygenase enzymes act on polyunsaturated fatty acids to form polyunsaturated fatty acid hydroperoxides that can be converted to aldehydes and alcohols, which result in a lower flavor quality in soybeans [ <xref ref-type="bibr" rid="B46"> 46 </xref> , <xref ref-type="bibr" rid="B47"> 47 </xref> ]. Null genotypes were identified in gamma-irradiation experiments that knocked out the three lipoxygenase genes: LOX1, <span class="gene">LOX2</span> and LOX3, that expressed during <span class="trait">seed</span> development [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> ]. LOX1 and <span class="gene">LOX2</span> are linked and found on Chromosome 13, while LOX3 is located on Chromosome 15 [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. The <italic> G. max </italic> Seq-atlas confirms that for the 72 lipoxygenase genes (Additional file <xref ref-type="supplementary-material" rid="S20"> 20 </xref> ) identified in the soybean genome and designated with a GOslim molecular function of lipoxygenase activity (GO:0016165), only 3 genes are highly and significantly expressed during <span class="trait">seed</span> development based on a Fisher's exact test with a FDR correction of 0.05 during <span class="trait">seed</span> development. The genes are: <italic> <span class="gene">Glyma13g42310</span> </italic> , <italic> <span class="gene">Glyma13g42320</span> </italic> and <italic> <span class="gene">Glyma15g03030</span> </italic> (Figure <xref ref-type="fig" rid="F6"> 6 </xref> : numbers in circles). The Seq-Atlas data and the latest genome release support the tight linkage between LOX1 and <span class="gene">LOX2</span> on chromosome 13 - only approximately 7000 base pairs separate the two genes. Although the identities of these lipoxygenase genes were determined prior to knowledge of the genomic sequence and access to next generation sequencing [ <xref ref-type="bibr" rid="B50"> 50 </xref> ], it is not difficult to imagine how the RNA-Seq atlas could be used to increase the efficiency of scientific discovery.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression dynamics and genome distribution of osmoprotectants in soybean: identifying important components to face abiotic stress</h2>
			<h3>PMID:23369061</h3>
				
					<h3>PMCID:3548699</h3>
					<h4>期刊:BMC Bioinformatics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Kido </surname>  <given-names> Ederson A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kido.ufpe@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Ferreira Neto </surname>  <given-names> Jos&#x000e9; RC </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> netocostaferreira@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Silva </surname>  <given-names> Roberta LO </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lane.roberta@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Belarmino </surname>  <given-names> Luis C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lucabellarmino@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Bezerra Neto </surname>  <given-names> Jo&#x000e3;o P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> pacifico.joao@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Soares-Cavalcanti </surname>  <given-names> Nina M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ninamotasc@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Pandolfi </surname>  <given-names> Valesca </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> valescapandolfi@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Silva </surname>  <given-names> Manass&#x000e9;s D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> manassesdaniel@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre L </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> alexlimanepo@gmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Benko-Iseppon </surname>  <given-names> Ana M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ana.iseppon@gmail.com </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The workflow applied resulted in a <span class="trait">set</span> including 1,996 <span class="trait">seed</span> sequences that allowed the identification of 36 differentially expressed genes related to the biosynthesis of osmoprotectants [Proline ( <italic> <span class="gene">P5CS</span> </italic> : 4, <italic> P5CR </italic> : 2), Trehalose ( <italic> TPS1 </italic> : 9, <italic> TPPB </italic> : 1), Glycine betaine ( <italic> BADH </italic> : 4) and <italic> Myo- </italic> inositol ( <italic> <span class="gene">MIPS</span> </italic> : 7, <italic> INPS1 </italic> : 8)], also mapped <italic> in silico </italic> in the soybean genome (25 loci). Another approach considered matches using Arabidopsis full length sequences as <span class="trait">seed</span> sequences, and allowed the identification of 124 osmoprotectant-related sequences, matching ~10.500 tags anchored in the soybean virtual chromosomes. Osmoprotectant-related genes appeared clustered in all soybean chromosomes, with higher density in some subterminal regions and synteny among some chromosome pairs.
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 77 osmoprotectants OSMTL sequences presented significant similarity (BLASTx, e-value cut-off e <sup> -10 </sup> ) to <italic> G. max </italic> sequences from four loci coding BADHs and annotated as Aldehyde Dehydrogenase Family 10A (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ). Out of four loci, one <span class="gene">(Glyma06g19820)</span> was associated with four HT-SuperSAGE tags in BLASTn alignments, tolerating at most a single mismatch (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). From these, two <span class="gene">(GmDr_44</span> and 3640) were induced after stress in both tolerant and sensible accessions (Embrapa 48 and BR-16), being mapped in the 3'UTRs of all three alternative transcripts of the locus <span class="gene">Glyma06g19820</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Other two tags <span class="gene">(GmDr_2643</span> and 55655) were induced only in the <span class="trait">drought</span> sensible accession BR-16 after stress regarding the same three transcripts, whereas one of them <span class="gene">(GmDr_2443)</span> was mapped in the 3'UTR and another in the CDS <span class="gene">(GmDr_55655)</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). The tag <span class="gene">GmDr_55655</span> also mapped in the transcript <span class="gene">Glyma11g27100.1</span> with a mismatch in the CDS region, but no 3'UTR was identified for this transcript (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Despite its induction in the sensible accession in relation to the control, the normalized frequency was only six tmp (tags per million; Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Thus, the locus <span class="gene">Glyma06g19820</span> emerged as a likely BADH candidate gene induced in response to the water deficit stress in the studied libraries.
					</p>
				
					<p>&nbsp;&nbsp;
						Significant upregulation (RTqPCR) in leaves of PvP5CS (from common bean <italic> Phaseolus vulgaris </italic> ) was demonstrated with transcription increase after 4d <span class="trait">drought</span> stress (2.5 times the control level), 2 h post-treatment (200 mM NaCl) of salt stress (about 16.3 times the control) and 2 h after of cold stress (11.7-fold). Another <span class="gene">P5CS</span> (PvP5CS2) also from common bean [ <xref ref-type="bibr" rid="B41"> 41 </xref> ] presented predicted amino acid sequence showing 83.7% identity with PvP5CS and an overall 93.2% identity with <span class="gene">GmP5CS</span> [ <italic> G. max </italic> P5CS], suggesting PvP5CS2 represented a soybean <span class="gene">P5CS</span> homolog gene. Indels (insertion and deletion events) and SNPs (single nucleotide polymorphisms) were found in the cloned PvP5CS2 genome sequence when the authors compared different accessions, helping in the development of a molecular marker in the chromosome b01. The association of molecular markers and phenotypes, in this case Pro accumulation is highly applicable for genetic improvement of plants and germplasm screening.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">seed</span> sequences OSMTL431, 432 and 434 were similar to P5CR polypeptides from two soybean loci <span class="gene">(Glyma19g31230</span> and <span class="gene">Glyma03g28480;</span> Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ), which transcripts were associated to SuperSAGE tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Of these one was repressed in the sensible accession <span class="gene">(GmDr_4445,</span> mapped at the 3'UTR region of the transcript <span class="gene">Glyma19g31230.1)</span> after stress (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). A second one <span class="gene">(GmDr_42728,</span> mapped in the 3'UTR of <span class="gene">Glyma03g28480.1</span> and CDS of <span class="gene">Glyma03g28480.1)</span> was not significantly modulated in the tolerant accession under stress as compared with the respective control, but presented a significant difference when compared to the sensible accession under stress (fold change of 12,0) (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 13 OSMTL <span class="trait">seed</span> sequences (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ) presented similarities to polypeptides from three soybean <italic> Myo- </italic> inositol sequences. With exception of the transcript <span class="gene">Glyma08g14670.1</span> that matched with <span class="gene">MIPS1</span> the other two transcripts, matching <span class="gene">MIPS2</span> and MIPS3, were associated to tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). The tag <span class="gene">GmDr_37</span> (mapped at the 3'UTR in all four alternative transcripts of <span class="gene">Glyma18g02210)</span> was the most frequent tag (615-1446 tpm) being DR in both stressed accessions (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). The other tag <span class="gene">(GmDr_3907)</span> presented a perfect match with the 3'UTR of all three alternative transcripts of <span class="gene">Glyma05g31450,</span> with DR expression in the sensible accession under stress (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Another tag, <span class="gene">GmDr_5821</span> (mapped at the 3'UTR in four alternative transcripts of <span class="gene">Glyma18g02210)</span> was induced (UR) in the tolerant accession Embrapa 48 under stress when compared with the respective control (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Considering all transcripts identified the locus <span class="gene">Glyma18g02210</span> (MIPS2) seems to be the most interesting candidate for future validation and transgenic expression (in detriment to <span class="gene">Glyma05g31450,</span> MIPS3).
					</p>
				
					<p>&nbsp;&nbsp;
						The confirmation of such a differential expression regarding <span class="gene">MIPS</span> is useful for plant breeding as highlighted by Kaur <italic> et al. </italic> [ <xref ref-type="bibr" rid="B45"> 45 </xref> ] that observed two divergent genes encoding <span class="gene">MIPS1</span> and <span class="gene">MIPS2</span> (isolated from a drought-tolerant plant) in chickpea with differential expression but discrete overlapping roles, despite their pronounced divergence in respect to their &#x000ed;ntrons composition, at the same time retaining 85% identity to their exons. Expression analysis showed both genes being expressed in all organs except <span class="trait">seed,</span> where only <span class="gene">MIPS2</span> transcript was detected. Under environmental stresses (high temperature and salinity), only <span class="gene">MIPS2</span> was induced whereas <span class="gene">MIPS1</span> expression remained the same. Also, in those conditions of high temperature and salinity <span class="gene">MIPS2</span> retained higher activity than MIPS1.
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 12 <span class="trait">seed</span> sequences (OSMTL61-66, OSMTL331-335 and OSMTL94) presented similarities with annotated IMP polypeptides regarding 10 <italic> G. max </italic> loci (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ), for those 19 SuperSAGE tags were identified. From the differentially expressed tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ), <span class="gene">GmDr_3452</span> mapped at the last CATG of the <span class="gene">Glyma08g19430.1,</span> with bases in the CDS and 3'UTR and was induced after stress in both accessions. Similarly other tags were induced in the tolerant accession under stress <span class="gene">(GmDr_23844,</span> at the 3'UTR of <span class="gene">Glyma16g28310.1</span> and <span class="gene">GmDr_32375</span> at the 3'UTRs of both <span class="gene">Glyma07g30110.1</span> and <span class="gene">Glyma08g07200.1)</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). By the other hand, the tag <span class="gene">GmDr_5543</span> was mapped at the 3'UTR of three alternative transcripts of <span class="gene">Glyma04g01170,</span> being upregulated in the sensible accession and downregulated in the tolerant accession under stress (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). Also the tag <span class="gene">GmDr_25343</span> (mapped at the 3'UTR of <span class="gene">Glyma15g07240.1)</span> was downregulated in the tolerant accession after stress (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). The abundance and differential expression of various IMP candidates in diverse comparisons indicate an important role in soybean water deficit. Despite of that and of the known role of these osmoprotectant-related genes, it is interesting that few expression essays or transgenic approaches have been carried using these candidates up to date.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Biological material, experimental design and stress application - </bold> Soybean HT-SuperSAGE libraries were generated according to the procedures described by Matsumura <italic> et al. </italic> [ <xref ref-type="bibr" rid="B57"> 57 </xref> ] at GenXPro <span class="gene">GmbH,</span> with posterior SOLEXA sequencing of the tags. The generated tags are distributed into four libraries (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ) including root tissues subjected to dehydration: two libraries from the <span class="trait">drought</span> tolerant cultivar Embrapa 48 [Tolerant after stress (TS) and negative control (TC)] and two libraries from a <span class="trait">drought</span> sensible cultivar BR-16 [Sensible after Stress (SS) and negative control (SC)]. The conditions for the generation of the mentioned libraries, time frame experiments, and laboratory protocols used are described in Soares-Cavalcanti <italic> et al. </italic> [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. The generated sequences are available at the GENOSOJA database (Brazilian Soybean Genome Consortium) [ <xref ref-type="bibr" rid="B35"> 35 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						EAK, VP and <span class="gene">ALN</span> generated the HT-SuperSAGE libraries. EAK and JRCFN carried out the identification of tags and differential expression analysis. RLOS, LCB, JPBN, NMSC and MDS generated and curated the <span class="trait">seed</span> sequence bank, while JPBN and AMBI generated the data for genome anchoring. AMBI coordinated the research.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Development of SNP Genotyping Assays for <span class="trait">Seed</span> Composition Traits in Soybean</h2>
			<h3>PMID:28630621</h3>
				
					<h3>PMCID:5463095</h3>
					<h4>期刊:International Journal of Plant Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 30 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Commodity soybean <span class="trait">seed</span> contains palmitic (11%), stearic (4%), oleic (23%), linoleic (54%), and linolenic acid (8%). Soybean oil with elevated oleic acid (&#x0003e;75%) is desirable for nutrition, flavor, and improvement of shelf life. Moreover, higher oleic acid content is desired in biodiesel production to improve the oxidative stability of oil while augmenting cold flow [ <xref rid="B14" ref-type="bibr"> 10 </xref> ]. On the other hand, the higher concentration of polyunsaturated fatty acids (linolenic and linoleic) reduces the oxidative stability of the oil that decreases flavor and storage quality. Soybean oil used for cooking purposes is often partially hydrogenated, resulting in <italic> trans </italic> -fat in the oil, which is unhealthy for the heart. Several genetic studies have characterized genes controlling monounsaturated (oleic acid) and polyunsaturated fatty acids (linoleic and linolenic acid) in soybean [ <xref rid="B5" ref-type="bibr"> 11 </xref> &#x02013; <xref rid="B42" ref-type="bibr"> 13 </xref> ]. During <span class="trait">seed</span> development stage, the two oleate desaturase genes, <span class="gene">FAD2-1A</span> <italic> <span class="gene">(Glyma10g42470)</span> </italic> and <span class="gene">FAD2-1B</span> <italic> <span class="gene">(Glyma20g24530)</span> </italic> , and the three linoleate desaturase genes, <span class="gene">FAD3A</span> <italic> <span class="gene">(Glyma14g37350)</span> </italic> , <span class="gene">FAD3B</span> <italic> <span class="gene">(Glyma02g39230)</span> </italic> , and <span class="gene">FAD3C</span> <italic> <span class="gene">(Glyma18g06950)</span> </italic> , are the major contributors controlling oleic and linolenic acid levels. The combination of mutant alleles of <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> produces up to 80% oleic acid in soybean <span class="trait">seed</span> [ <xref rid="B35" ref-type="bibr"> 14 </xref> ]. Similarly, a combination of FAD3A, FAD3B, and <span class="gene">FAD3C</span> mutant alleles can produce ultralow (&#x0003c;1%) linolenic acid.
					</p>
				
					<p>&nbsp;&nbsp;
						While high oleic and low linolenic acid content are of importance to the commodity soybean, the manipulation of raffinose family oligosaccharides (RFOs) is also an important aspect to improving soybean <span class="trait">seed</span> components [ <xref rid="B26" ref-type="bibr"> 15 </xref> ]. In soybean, raffinose and stachyose are indigestible components that are undesirable in diets of poultry and swine [ <xref rid="B13" ref-type="bibr"> 16 </xref> ]. The key step in raffinose and stachyose biosynthesis is mediated by the enzyme raffinose synthase (RS). Kerr and Sebastian (2000) reported PI 200508, with reduced levels of RFOs and elevated levels of sucrose due to variation in RS gene <italic> <span class="gene">(Glyma06g18890)</span> </italic> . Recently, Qiu et al. [ <xref rid="B37" ref-type="bibr"> 17 </xref> ] reported a 33&#x02009;bp deletion in the exon 4 of stachyose synthase gene (STS gene, <italic> <span class="gene">Glyma19g40550</span> </italic> ) of PI 603176A having ultralow stachyose content (0.5%).
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> storage protein consists of approximately 6% proteinase inhibitors, Kunitz trypsin inhibitor (KTi, 21&#x02009;kDa), and Bowman&#x02013;Birk trypsin inhibitor (BBTi, 7-8&#x02009;kDa), which also contributes to indigestibility [ <xref rid="B51" ref-type="bibr"> 18 </xref> ]. Mutation in KTi gene <italic> <span class="gene">(Glyma08g45530)</span> </italic> prevents the accumulation of Kunitz trypsin inhibitor protein during <span class="trait">seed</span> development and improves tryptic activity [ <xref rid="B20" ref-type="bibr"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to undesirable traits mentioned above, the beany or grassy flavor of soybean <span class="trait">seed</span> products is a major factor that limits human consumption of soybean [ <xref rid="B38" ref-type="bibr"> 19 </xref> ]. The grassy and beany flavor in soybean and other legumes seeds is due to the oxidation products resulting from <span class="trait">seed</span> lipoxygenase activity. Mature soybean <span class="trait">seed</span> consists of three distinct lipoxygenase isozymes, lipoxygenase 1 <italic> <span class="gene">(Glyma13g42320)</span> </italic> , lipoxygenase 2 <italic> <span class="gene">(Glyma13g42310)</span> </italic> , and lipoxygenase 3 <italic> <span class="gene">(Glyma13g42330)</span> </italic> . Mutation in these genes leads to reduced lipoxygenase enzyme activity and results in better quality soybean meal [ <xref rid="B23" ref-type="bibr"> 7 </xref> , <xref rid="B47" ref-type="bibr"> 8 </xref> , <xref rid="B52" ref-type="bibr"> 20 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A <span class="trait">set</span> of populations obtained from biparental crosses was used for validation of oleic and linolenic acid assays (Supplementary Table S1 in Supplementary Material available online at <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1155/2017/6572969"> https://doi.org/10.1155/2017/6572969 </ext-link> ). In addition, leaf tissue and FTA cards (Whatman Inc., Clifton, NJ, USA) were (at <span class="gene">V2</span> stage) collected to validate <span class="gene">FAD2-1B</span> (MU-HO-4) genotyping assay. FTA is a paper-based system designed to fix and store nucleic acids directly from fresh tissues pressed into the treated paper (Fehr and Caviness, 1997 [ <xref rid="B27" ref-type="bibr"> 51 </xref> ]).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inference of Transcription Regulatory Network in Low Phytic Acid Soybean Seeds</h2>
			<h3>PMID:29250090</h3>
				
					<h3>PMCID:5714895</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Redekar </surname>  <given-names> Neelam </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/340447/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pilot </surname>  <given-names> Guillaume </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/11124/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Raboy </surname>  <given-names> Victor </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/58055/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Song </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/45869/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saghai Maroof </surname>  <given-names> M. A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/458642/overview"/>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A dominant loss of function mutation in <italic> myo </italic> -inositol phosphate synthase ( <italic> <span class="gene">MIPS</span> </italic> ) gene and recessive loss of function mutations in two multidrug resistant protein type-ABC transporter genes not only reduce the <span class="trait">seed</span> phytic acid levels in soybean, but also affect the pathways associated with <span class="trait">seed</span> development, ultimately resulting in low emergence. To understand the regulatory mechanisms and identify key genes that intervene in the <span class="trait">seed</span> development process in low phytic acid crops, we performed computational inference of gene regulatory networks in low and normal phytic acid soybeans using a time course transcriptomic data and multiple network inference algorithms. We identified a <span class="trait">set</span> of putative candidate transcription factors and their regulatory interactions with genes that have functions in myo-inositol biosynthesis, auxin-ABA signaling, and <span class="trait">seed</span> dormancy. We evaluated the performance of our unsupervised network inference method by comparing the predicted regulatory network with published regulatory interactions in <italic> Arabidopsis </italic> . Some contrasting regulatory interactions were observed in low phytic acid mutants compared to non-mutant lines. These findings provide important hypotheses on expression regulation of <italic> myo </italic> -inositol metabolism and phytohormone signaling in developing low phytic acid soybeans. The computational pipeline used for unsupervised network learning in this study is provided as open source software and is freely available at <ext-link ext-link-type="uri" xlink:href="https://lilabatvt.github.io/LPANetwork/"> https://lilabatvt.github.io/LPANetwork/ </ext-link> .
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> development is a complex metabolic process, which involves both synthesis and breakdown of macromolecules for growth and maintenance of the embryo (Weber et al., <xref rid="B59" ref-type="bibr"> 2005 </xref> ; Le et al., <xref rid="B25" ref-type="bibr"> 2007 </xref> ). During <span class="trait">seed</span> development, glucose-6-phosphaste is converted to myo-inositol, an intracellular signaling molecule, which is phosphorylated several times to form phytic acid (Raboy, <xref rid="B42" ref-type="bibr"> 1997 </xref> ). Seeds with reduced phytic acid content are commercially more valuable because consumption of low phytic acid seeds by monogastric animals alleviates mineral deficiency and reduces phosphorus pollution from animal waste (Raboy, <xref rid="B43" ref-type="bibr"> 2007 </xref> ). Mutations that block the phytic acid biosynthesis pathway have been shown to alter the <span class="trait">seed</span> metabolite levels in soybean, rice, maize, and other plant species (Wilcox et al., <xref rid="B60" ref-type="bibr"> 2000 </xref> ; Shi et al., <xref rid="B54" ref-type="bibr"> 2003 </xref> , <xref rid="B53" ref-type="bibr"> 2005 </xref> ; Stevenson-Paulik et al., <xref rid="B57" ref-type="bibr"> 2005 </xref> ; Raboy, <xref rid="B43" ref-type="bibr"> 2007 </xref> ; Glover, <xref rid="B11" ref-type="bibr"> 2011 </xref> ; Jervis et al., <xref rid="B19" ref-type="bibr"> 2015 </xref> ). For example, a mutation in <italic> myo- </italic> inositol phosphate synthase ( <italic> <span class="gene">MIPS</span> </italic> ) gene results in reduced phytic acid, stachyose, raffinose, and elevated sucrose, and low <span class="trait">seed</span> emergence in soybean (Hitz et al., <xref rid="B16" ref-type="bibr"> 2002 </xref> ; Saghai Maroof and Buss, <xref rid="B46" ref-type="bibr"> 2008 </xref> ). Other non-biosynthetic pathway genes such as multi-drug <span class="trait">resistance</span> protein ( <italic> MRP </italic> ) genes encoding ATP-binding cassette transporters that are believed to be involved in the transport of phytic acid to storage vacuoles, are also known to regulate phytic acid levels and affect <span class="trait">seed</span> emergence (Shi et al., <xref rid="B52" ref-type="bibr"> 2007 </xref> ; Nagy et al., <xref rid="B39" ref-type="bibr"> 2009 </xref> ; Saghai Maroof et al., <xref rid="B47" ref-type="bibr"> 2009 </xref> ; Xu et al., <xref rid="B62" ref-type="bibr"> 2009 </xref> ; Jervis et al., <xref rid="B19" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the connections between transcription regulation and metabolic pathways that were altered in the <italic> 3mlpa </italic> and <italic> 1mlpa </italic> mutant lines, we downloaded the metabolic pathway annotation from the SoyCyc 7.0 database. We mapped genes in myo-inositol metabolism, stachyose metabolism and sucrose metabolism to different gene modules, because these metabolites are altered as a result of <italic> lpa </italic> mutations (Supplementary Table <xref ref-type="supplementary-material" rid="SM3"> 3 </xref> ). We found that there were 64 genes involved in these metabolic pathways, which mapped to 34 gene modules (Supplementary Table <xref ref-type="supplementary-material" rid="SM3"> 3 </xref> ). Twelve of the 64 genes were involved in stachyose (or raffinose family oligosaccharides (RFOs)) biosynthesis and eight were associated with inferred gene regulatory networks (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). In module 29, which is up-regulated in the first stage of <span class="trait">seed</span> development, two genes <span class="gene">(Glyma.02G303300</span> and <span class="gene">Glyma.14G010500,</span> both encoding raffinose synthase / <span class="trait">seed</span> imbibition protein 1) were found to be regulated by a bZIP transcription factor <span class="gene">(Glyma.02G131700).</span> This bZIP transcription factor is homologous to <italic> ABF1 </italic> (ABA response element-binding factor 1) in <italic> Arabidopsis </italic> , providing a potential connection between ABA response and stachyose biosynthesis (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Some of the enzymes involved in <italic> myo </italic> -inositol biosynthesis are regulated similarly in mutant and non-mutant lines. For example, inositol-polyphosphate 5-phosphatase <span class="gene">(Glyma.20G170500)</span> and inositol-phosphate phosphatase <span class="gene">(Glyma.09G011100)</span> are found in module 16. Genes in this module are up-regulated during mid-stages of <span class="trait">seed</span> development. Module 16 is predicted to be regulated by two transcription factors, a bHLH transcription factor <span class="gene">(Glyma.13G040100)</span> and a C2H2 transcription factor <span class="gene">(Glyma.13G327500).</span> The bHLH transcription factor is homologous to SPCH, which regulates stomatal lineage specification during embryo development (Danzer et al., <xref rid="B7" ref-type="bibr"> 2015 </xref> ). Some regulatory interactions are only found in non-mutants. For example, genes from module 40 are predicted to be regulated by <span class="gene">Glyma.17G085600,</span> a MYB-related transcription factor, and this module contains two genes related to <italic> myo </italic> -inositol biosynthesis <span class="gene">(Glyma.07G107000</span> encoding inositol-polyphosphate 5-phosphatase and <span class="gene">Glyma.05G180600</span> encoding inositol-1-phosphate synthase). This module is highly expressed in the early stage of <span class="trait">seed</span> development and the MYB-related transcription factor is similar to <italic> RSM1 </italic> in Arabidopsis, which has been found to be related to auxin signaling in early morphogenesis (Hamaguchi et al., <xref rid="B13" ref-type="bibr"> 2008 </xref> ). Several target genes associated with regulation of phytic acid biosynthesis pathway matched with those discovered in co-expression network of developing maize kernel with low phytic acid content (Zhang et al., <xref rid="B63" ref-type="bibr"> 2016 </xref> ). These predictions provide genotype-specific, testable hypotheses that may connect gene expression patterns with putative regulatory TFs and hormone regulations during different <span class="trait">seed</span> developmental stages.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Myo- </italic> inositol is an essential signaling molecule with multifunctional properties including gene regulation, chromatin modeling, mRNA transport, signal transduction, cell death, pathogen <span class="trait">resistance,</span> vesicle trafficking, plasma membrane, and cell wall formation (Martin, <xref rid="B37" ref-type="bibr"> 1998 </xref> ; Stevenson et al., <xref rid="B56" ref-type="bibr"> 2000 </xref> ; Chen and Xiong, <xref rid="B5" ref-type="bibr"> 2010 </xref> ). It is synthesized in a two-step pathway (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ), where glucose-6-phosphate is first converted to inositol monophosphate (IMP), a rate-limiting step catalyzed by <span class="gene">MIPS1</span> enzyme, followed by dephosphorylation of IMP to form <italic> myo </italic> -inositol (Loewus and Murthy, <xref rid="B32" ref-type="bibr"> 2000 </xref> ). Upon synthesis, <italic> myo </italic> -inositol is utilized in and recycled from multiple metabolic pathways such as biosynthesis of phytic acid and RFOs (such as stachyose and raffinose), inositol and phosphatidylinositol intermediates, auxin-inositol conjugates and glucuronic acid.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regeneration of fertile plants from protoplasts of soybean (Glycine max L. Merr.): genotypic differences in culture response.</h2>
			<h3>PMID:24203141</h3>
				

				
			<p>
				作者：Dhir SK(1), Dhir S, Widholm JM.
				<br/> 
				年份：1992 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fourteen soybean (Glycine max L.) genotypes were evaluated for their regenerability from protoplasts using a procedure previously descibed for the cultivar Clark 63. Protoplasts were isolated from immature cotyledon tissue and were cultured in liquid or agarose gelled KP8, <span class="gene">MS</span> or B5 medium with different sugars. Significant differences were observed in plating efficiency, which was as high as 63% in Jack and A-2396, and as low as 38% in X-3337. Upon regular dilution with K8 medium, 1-2 mm diameter colonies were formed in 5-6 weeks with all the genotypes tested. These colonies were then transferred onto MSB (MS salts; Murashige & Skoog, 1962 + B5 organics; Gamborg et al., 1968) medium with 0.5 mg l(-1) each of 2,4-D, BA and KN and 500 mg l(-1) CH for further growth. Once the colonies had become green, compact and nodular, and were 8-10 mm in size, they were transferred to regeneration medium. Upon regular subculturing, calli of six genotypes; A-2396, Chamberlain, Heilong-26, Jack, Resnick and XP-3015 developed shoots, with the regeneration frequency being highest 27% in Jack (52 calli out of 192 produced 8-12 shoots). The regenerated shoots from different genotypes were elongated and rooted. So far, sixty three complete plants have been obtained, including twelve A-2396, nineteen Chamberlain, fifteen Jack, nine Resnick and eight XP-3015. A total of thirty five plants have been transplanted into pots in the greenhouse. Sixteen have <span class="trait">set</span> seeds and others are producing flowers and pods.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plantlet regeneration from immature cotyledon protoplasts of soybean (Glycine max L.).</h2>
			<h3>PMID:24226162</h3>
				

				
			<p>
				作者：Dhir SK(1), Dhir S, Widholm JM.
				<br/> 
				年份：1991 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Protoplasts were isolated from immature cotyledons of Glycine max L. Merr. cv. Clark 63 and cultured in liquid or in agarose-gelled modified KP8 medium. Plating efficiencies of 45-50% were obtained in liquid medium and 55-60% in 1.2% (w/v) agarose beads. Upon regular dilution with K8 medium rapidly growing green microcalli (1-2 mm in size) were obtained in 5-6 weeks, which upon transfer to MSB medium with 0.5 mg 1(-1) each of 2,4-D, BA, Kn and 500 mg 1(-1) CH produced compact green calli in 4-6 weeks. After 3-4 regular subcultures of 14 days each on MSB medium containing 0.5 mg 1(-1) each of BA, Kn, ZT, 0.1 mg 1(-1) NAA and 500 mg 1(-1) CH, about 21% of the compact calli formed multiple shoots. Addition of glutamine, asparagine and GA3 enhanced shoot regeneration up to 30%. Shoots of 0.5-1.0 cm length were transferred to 1/2 <span class="gene">MS</span> medium with 0.01 mg 1(-1) TH and 0.5 mg 1(-1) GA3 for elongation. In 2 to 3 weeks, approximately 60% of the shoots were 2-3 cm in length. These shoots were rooted on 1/2 <span class="gene">MS</span> with 1% sucrose and 0.2 mg 1(-1) IBA or 0.5 mg 1(-1) NAA. So far, twenty six plants have been transferred to the greenhouse, where they all have <span class="trait">set</span> <span class="trait">seed.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Microarrays for global expression constructed with a low redundancy <span class="trait">set</span> of 27,500 sequenced cDNAs representing an array of developmental stages and physiological conditions of the soybean plant</h2>
			<h3>PMID:15453914</h3>
				
					<h3>PMCID:526184</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Khanna </surname>  <given-names> Anupama </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> anupama.khanna@epicentre.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Shealy </surname>  <given-names> Robin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Clough </surname>  <given-names> Steven J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I8"> 8 </xref>  <email> sjclough@uiuc.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Gonzalez </surname>  <given-names> Delkin Orlando </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dogonzal@uiuc.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Philip </surname>  <given-names> Reena </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I9"> 9 </xref>  <email> philipreena@yahoo.com </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> g-zabala@uiuc.edu </email>  </contrib>  <contrib id="A8" contrib-type="author">  <name>  <surname> Thibaud-Nissen </surname>  <given-names> Fran&#x000e7;oise </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I10"> 10 </xref>  <email> fthibaud@tigr.org </email>  </contrib>  <contrib id="A9" contrib-type="author">  <name>  <surname> Sidarous </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sidarous@uiuc.edu </email>  </contrib>  <contrib id="A10" contrib-type="author">  <name>  <surname> Str&#x000f6;mvik </surname>  <given-names> Martina V </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I11"> 11 </xref>  <email> martina.stromvik@mcgill.ca </email>  </contrib>  <contrib id="A11" contrib-type="author">  <name>  <surname> Shoop </surname>  <given-names> Elizabeth </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I12"> 12 </xref>  <email> schoop@macalester.edu </email>  </contrib>  <contrib id="A12" contrib-type="author">  <name>  <surname> Schmidt </surname>  <given-names> Christina </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> shoop@macalesler.edu </email>  </contrib>  <contrib id="A13" contrib-type="author">  <name>  <surname> Retzel </surname>  <given-names> Ernest </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ernest@mail.ahc.umn.edu </email>  </contrib>  <contrib id="A14" contrib-type="author">  <name>  <surname> Erpelding </surname>  <given-names> John </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> rcsshoe@iastate.edu </email>  </contrib>  <contrib id="A15" contrib-type="author">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> rcsshoe@iastate.edu </email>  </contrib>  <contrib id="A16" contrib-type="author">  <name>  <surname> Rodriguez-Huete </surname>  <given-names> Alicia M </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <xref ref-type="aff" rid="I13"> 13 </xref>  <email> PolaccoJ@missouri.edu </email>  </contrib>  <contrib id="A17" contrib-type="author">  <name>  <surname> Polacco </surname>  <given-names> Joseph C </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> PolaccoJ@missouri.edu </email>  </contrib>  <contrib id="A18" contrib-type="author">  <name>  <surname> Coryell </surname>  <given-names> Virginia </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> Paul.Keim@nau.edu </email>  </contrib>  <contrib id="A19" contrib-type="author">  <name>  <surname> Keim </surname>  <given-names> Paul </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> Paul.Keim@nau.edu </email>  </contrib>  <contrib id="A20" contrib-type="author">  <name>  <surname> Gong </surname>  <given-names> George </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> ligong@uiuc.edu </email>  </contrib>  <contrib id="A21" contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> leiliu@uiuc.edu </email>  </contrib>  <contrib id="A22" contrib-type="author">  <name>  <surname> Pardinas </surname>  <given-names> Jose </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A23" contrib-type="author">  <name>  <surname> Schweitzer </surname>  <given-names> Peter </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I14"> 14 </xref>  <email> pas48@cornell.edu </email>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 9 </month>  <year> 2004 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Figure <xref ref-type="fig" rid="F1"> 1 </xref> illustrates an overview of the data generation and analysis used to create a low redundancy 'unigene' <span class="trait">set</span> and its use in construction of cDNA microarrays. The 5' EST sequence information was used as the raw material including the addition of over 30 new cDNA libraries and more than 160,000 5' sequences since an initial report [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. In total now, over 80 libraries and over 280,000 5' EST have been generated from many tissue and organ systems of various stages of development, ranging from roots, shoots, leaves, stems, pods, cotyledons, germinating shoot tips, flower meristems, tissue culture derived embryos, and pathogen challenged tissues. These libraries, with the one exception of library <span class="gene">Gm-r1030</span> described below, were non-normalized. Thus, the mRNAs that are more abundant in various tissue and organ systems will be more highly represented in the EST collection. To remove redundancy and identify unique sequences, the ESTs were assembled using the Phrap program [ <xref ref-type="bibr" rid="B7"> 7 </xref> ] into contiguous regions (contigs) representing overlapping sequences based on EST sequence similarity. In this way, longer overlapping sequences of expressed genes are assembled. Identical sequences that represent redundant mRNAs of various sizes are also recognized and the number of sequences in a contig in a non-normalized cDNA library is a rough approximation of the relative abundance of that particular mRNA within that tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Certain tissues contain large amounts of specialized transcripts. Developing soybean cotyledons, for example, contain large amounts of RNA transcripts representing highly expressed storage protein genes. In order to select for some of the weakly expressed cDNA clones from a mid-maturation stage cotyledon library, we used a virtual subtraction approach using high densisty filters. A total of 18,000 bacterial clones containing cDNAs from library <span class="gene">Gm-c1007</span> (immature cotyledons of 100&#x02013;300 mg fresh weigh range from Williams variety) was printed in high density on nylon filters and probed with <sup> 33 </sup> P-cDNA produced by reverse transcription of total mRNA isolated from immature soybean cotyledons. Figure <xref ref-type="fig" rid="F2"> 2A </xref> shows the highly complex hybridization pattern resulting from using total mRNA from immature cotyledons to probe the high density filter. The intensity of each dot represents the hybridization signal and the relative abundance of that cDNA in the message population. The phosphorimager pattern was quantified by image analysis software and 5000 of the lowest expressing clones were selected. The cDNA clones were physically reracked to a new <span class="trait">set</span> of 384 well plates to form the filter-normalized reracked library designated <span class="gene">Gm-r1030.</span> The 5' end of these clones were then sequenced at Washington University. Figure <xref ref-type="fig" rid="F2"> 2B </xref> shows that 1,528, or 85%, of the 1799 sequences within the filter-driven reracked library <span class="gene">Gm-r1030</span> were novel and not found in any of the 931 sequenced clones from the <span class="gene">Gm-c1007</span> source cDNA library. Thus, the filter normalization method was an effective way to identify cDNAs with low expression and increase gene discovery in libraries that contain large numbers of transcripts from highly expressed genes. The virtual subtraction method using high density cDNA filters compares favorably with other mRNA subtraction methods used to create normalized libraries during the cloning process [ <xref ref-type="bibr" rid="B10"> 10 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To enable global expression studies, the ideal would be to have each gene represented at least once on an array. Toward this aim, we selected 27,513 of the cDNA clones by four successive clustering assemblies of the soybean cDNAs performed as the 5' EST data accumulated. Table <xref ref-type="table" rid="T1"> 1 </xref> shows the four reracked sets of cDNA clones <span class="gene">(Gm-r1021,</span> <span class="gene">Gm-r1070,</span> <span class="gene">Gm-r1083,</span> and <span class="gene">Gm-r1088)</span> and the level of uniqueness within them as defined by Phrap and CAP3 analysis [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ]. Thus, the clustering was conducted periodically as the number of 5' EST input sequences grew in size. After each clustering, the previously selected and reracked cDNAs were excluded from subsequent reracking lists. In order to develop a 'unigene' <span class="trait">set</span> for soybean, a single representative of each contig was chosen. To select a representative from each contig, we chose the cDNA clone corresponding to the EST that was found at the furthest 5' region of each contig. Thus, we are selecting the cDNA clones most likely to be near full-length.
					</p>
				
					<p>&nbsp;&nbsp;
						The current 'unigene' collection (or tentatively unique sequences) represents low redundancy sets of cDNA clones. We have processed all of these cDNAs for microarrays, as outlined in the Methods section, into three sets of 9,216 cDNAs per array. As shown in Table <xref ref-type="table" rid="T3"> 3 </xref> , these include reracked libraries <span class="gene">Gm-r1070</span> (a <span class="trait">set</span> of 9,216 cDNA clones from libraries of various developmental stages of immature cotyledons, flowers, pods, and <span class="trait">seed</span> coats); <span class="gene">Gm-r1021</span> plus <span class="gene">Gm-r1083</span> (a <span class="trait">set</span> of approximately 9,216 cDNA clones from 8-day old seedling roots, seedling roots inoculated with <italic> Bradyrhozobium japonicum, </italic> 2-month old roots, and whole seedlings); and <span class="gene">Gm-r1088</span> (a collection of 9,216 cDNA clones from a number of libraries made from cotyledons and hypocotyls of germinating seedlings and leaves and other plant parts subjected to various pathogens or environmental stress conditions and also from tissue-culture derived somatic embryos). As an example, the <span class="gene">Gm-r1070</span> <span class="trait">set</span> contains 3,938 tentatively unique cDNAs that are directly derived from two flower cDNA libraries <span class="gene">(Gm-c1015</span> and <span class="gene">Gm-c1016)</span> that were sequenced deeply with over 14,000 5' ESTs obtained from these two libraries. In addition, a total of 2,639 cDNAs on the array are directly derived from source libraries made from the immature stages of cotyledon development and representing over 11,000 input sequences from the cotyledon stages of <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						The 9,216 amplified inserts of each <span class="trait">set</span> were singly spotted onto glass slides as outlined in the Methods section. A <span class="trait">set</span> of 64 control or 'choice' clones was assembled by hand into one 96-well plate (designated <span class="gene">Gm-b10BB)</span> and printed eight times repetitively throughout each array. Thus, the total number of spots on the array is 9,728 consisting of 9,216 cDNAs from the unigene <span class="trait">set</span> plus 512 (64 cDNAs &#x000d7; 8 repeats) from the choice clones. The choice clones were selected for various reasons. Some represent constitutively expressed genes (such as ubiquitin and EF1). Some are cDNAs whose expression is restricted to a subset of specific plant tissues (such as Rubisco or <span class="trait">seed</span> storage proteins). Some are clones of enzymes representing commonly used antibiotic <span class="trait">resistance</span> markers in transgenic plants (as hygromycin or kanamycin <span class="trait">resistance),</span> and 32 are cDNAs that represent at least 13 different enzymes of the flavonoid pathway. The flavonoid pathway was chosen because the corresponding genes often respond to many biotic and abiotic stress conditions and it has been widely studied in plant systems.
					</p>
				
					<p>&nbsp;&nbsp;
						Figure <xref ref-type="fig" rid="F3"> 3 </xref> illustrates an example of the reliability of the soybean microarray approach using dual labeled RNA probes from two near isogenic lines. These data illustrate the potential to discover novel genes by analysis of contrasting probes from mutant and normal lines. In this experiment, we compared two isogenic soybean lines that differ only at the <italic> T </italic> locus. The <italic> T </italic> locus controls the color of the pigment in the trichome hairs on the stems, leaves, and pods of the plant and also modifies the composition of the flavonoids and the color of the <span class="trait">seed</span> coats. Total RNA extracted from developing <span class="trait">seed</span> coats of line XB22A ( <italic> T </italic> / <italic> T </italic> genotype) was labeled with Cy3 and compared to RNA extracted from the same stage of developing <span class="trait">seed</span> coats from an isoline containing the spontaneous mutation 37609 ( <italic> t </italic> */ <italic> t </italic> * genotype) that was labeled with Cy5. A replicate experiment with a dye swap was also performed. Hybridizations were performed to microarrays constructed with the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1070</span> representing cDNAs from seeds, <span class="trait">seed</span> coats, and flowers. Figure <xref ref-type="fig" rid="F3"> 3 </xref> shows both replicates before and after the flagging and normalization procedures conducted as described in the Methods section. As shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> , the normalization procedure serves to compensate within slide differences between the Cy3 and Cy5 intensity levels to shift the majority of spots to the line of equivalent expression between the isogenic lines. Also, as shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> , very few of the 9,728 cDNAs on the array were reproducibly found to be expressed differentially in the two genotypes at levels higher or lower level than two-fold. A group of 16 of these (encircled by a line) are overexpressed in the <italic> T </italic> / <italic> T </italic> line relative to the <italic> t </italic> */ <italic> t </italic> * line by approximately three-fold. These cDNAs correspond to the flavonoid 3' hydroxylase cDNAs that were repetitively spotted on the array as members of the 'choice' clone <span class="trait">set.</span> Table <xref ref-type="table" rid="T4"> 4 </xref> shows the Cy5 and Cy3 intensities and the ratios of the two replicate slides for all cDNA spots that exceeded a two-fold difference after normalizations within each slide and between the replicate slides. Only 23 cDNAs were found to have values that meet the criteria of exceeding a two-fold differences in both of the replicate slides. Of these 16 were the repetitively spotted flavonoid 3' hydroxylase cDNAs. Only an additional 22 cDNAs (known as partial hits) had values exceeding two-fold levels in one but not both of the slides replicates. Thus, out of over 9200 cDNAs represented on the array, there are relatively few that show differential expression between the RNAs in the normal and mutant lines.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast to the results with near isogenic lines of the <italic> T </italic> locus which showed that relatively few cDNAs showed differential expression between the two very closely related lines, the soybean microarrays reveal larger numbers of cDNAs showing differential mRNA abundance levels in different tissue types of the same plants. For illustration, Figure <xref ref-type="fig" rid="F4"> 4 </xref> shows one of the two replicates of a dual labeling experiment using the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1088</span> of 9,216 cDNAs. The Cy3 labeled probe in this experiment was RNA from roots of hydroponically grown soybean plants, and the Cy5 probe was RNA from leaves of the same plants. The upper ratio threshold is 2.0 and the lower threshold is 0.5.
					</p>
				
					<p>&nbsp;&nbsp;
						Our publicly available <span class="gene">Gm-r1021</span> soybean unigene subset containing 4,098 cDNAs has also been used to examine differential gene expression in roots and shoots of older soybean plants [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. We have previously utilized microarrays containing 9,216 clones of the <span class="gene">Gm-r1070</span> <span class="trait">set</span> (representing many cDNAs from developing seeds, <span class="trait">seed</span> coats, flowers, and pods) to carry out a detailed analysis of induction of somatic embryos during culture of cotyledons on auxin-containing media [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. The resulting transcript profiles were subjected to a cluster analysis and revealed the process of reprogramming of the cotyledons cells during the induction process. The 495 cDNAs (5.3% of the cDNAs on the array) that were differentially expressed were clustered into 11 sets using a non-hierarchical method (K-means) to reveal cDNAs with similar profiles in either the adaxial or the abaxial side of the embryos from 0 to 28 days in 7 day intervals. Among other conclusions, these global expression studies indicated that auxin induces dedifferentiation of the cotyledon and provokes a surge of cDNAs involved in cell division and oxidative burst.
					</p>
				
					<p>&nbsp;&nbsp;
						All arrays contained 32 grids of spots arranged in an 8 &#x000d7; 4 matrix. Each grid had 19 rows and 16 columns of spots for a total of 9,728 spots per array. A total of 9,216 spots were the cDNAs prepared from the 'unigene' <span class="trait">set</span> to form 18 of the 19 rows with 288 spots per grid. After all of the 9,216 cDNAs were printed, an additional row of 16 spots was printed as the first row of each grid for a total of 32 grids &#x000d7; 16 spots = 512 additional spots. These cDNAs were printed from the choice clone spotting plate designated <span class="gene">Gm-b10BB</span> which contained 64 hand-picked clones. Thus, the 64 hand-picked, choice clones were printed 8 times each, i.e., each clone was printed in twice in four separate grids. In addition, since the <span class="gene">Gm-r1021</span> library contained only 4089 cDNAs, an additional 135 were repeated in order to obtain an even 9216 cDNAs for printing when combined with the <span class="gene">Gm-r1083</span> unigene <span class="trait">set.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						LOV led the unigene and microarray development, coordinated the project, and drafted the manuscript; AK constructed multiple cDNA libraries included in the unigene cDNAs, performed over 18,000 PCR reactions, performed the library normalization by filter screening shown in Figure <xref ref-type="fig" rid="F1"> 1 </xref> , the array hybridization reactions including that of Fig <xref ref-type="fig" rid="F4"> 4 </xref> , and drafted protocols; RS led the informatics and sample tracking efforts for array printing, high throughput PCR, cDNA clone reracking, printed arrays, developed and drafted protocols for analysis of array data; SJC initiated the PCR and array hybridization protocols for the project and constructed several cDNA libraries; DOG performed 10,000 PCR reactions, printed arrays, and participated in clone rearraying; RP contributed to protocol development, PCR production, and gel analysis; GZ accumulated the 'choice clone" cDNAs and performed the isoline hybridization analysis shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> ; <span class="gene">MS</span> contributed to informatics and sample tracking and gel analysis of PCR products; MVS performed EST cluster analysis for development of the unigene <span class="trait">set</span> and drafted sections of the paper; ES and CS performed EST analysis and clustering for the unigene <span class="trait">set;</span> ER coordinated EST processing and informatics; JE constructed multiple cDNA libraries used in selecting the unigene <span class="trait">set;</span> RS led the public EST project for library construction and 5' sequencing; AR-H and JCP provided hydroponically grown plant material and RNAs for Figure <xref ref-type="fig" rid="F4"> 4 </xref> and constructed a cDNA library; VC and PC constructed multiple cDNA libraries; GG and LL performed annotations of the 27,500 unigene <span class="trait">set;</span> JP and PS led or performed the 3' sequencing of the 27,500 unigene clones.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich protein) gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The genes coding for PGYRPs (proline-, glycine- and tyrosine-rich proteins) are widely distributed across eukaryotes and have been proposed to have critical role in plant development, especially in response to environmental stresses. In this study, total of 12 soybean PGYRPs <span class="gene">(GmPGYRP1-12)</span> were identified from the soybean genome database for the first time and full-length cDNA and DNA sequences of <span class="gene">GmPGYRP7</span> was cloned. <span class="gene">GmPGYRP1-12</span> genes encoded a <span class="trait">set</span> of small predicted proteins (&lt;120 aa) with molecular mass of 7.20-13.29 kDa and isoelectric point of 4.06-6.57. All <span class="gene">GmPGYRPs</span> contained three exons and two introns with fixed occurring sites within genomic DNA sequences. In the putative <span class="gene">GmPGYRP</span> sequences, 4 amino acids (proline, glycine, tyrosine, and glutamine) account for more than 39% of the total protein composition. <span class="gene">GmPGYRPs</span> had a relatively flexible GYPPX motif followed by a highly conserved cysteine-rich domain (GCLAAXCCCCXLXC) and showed high similarity to other known PGYRPs, especially in C-terminal region. Most of PGYRPs can be divided into five subgroups according to phylogenetic analysis. The transcripts of <span class="gene">GmPGYRP1,</span> 3, 5, and 7, representing different PGYRP subgroups, appeared in different organs including seedling leaves, stems, roots, flowers, and developing seeds, but mainly accumulated in seedling roots. Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage. Our data suggest that <span class="gene">GmPGYRP</span> genes encoding a class of conservative XYPPX-repeat proteins probably play an important role in plant development as well as in response to abiotic stresses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The male sterility locus ms3 is present in a <span class="trait">fertility</span> controlling gene cluster in soybean.</h2>
			<h3>PMID:19617521  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cervantes-Martinez I(1), Sandhu D, Xu M, Ortiz-Pérez E, Kato KK, Horner HT, Palmer RG.
				<br/> 
				年份：2009 Sep-Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is self-pollinated. To produce large quantities of hybrid <span class="trait">seed,</span> insect-mediated cross-pollination is necessary. An efficient nuclear male-sterile system for hybrid <span class="trait">seed</span> production would benefit from molecular and/or phenotypic markers linked to male fertility/sterility loci to facilitate early identification of phenotypes. Nuclear male-sterile, female-fertile ms3 mutant is a single recessive gene and displays high outcrossed <span class="trait">seed</span> <span class="trait">set</span> with pollinators. Our objective was to map the ms3 locus. A segregating population of 150 F(2) plants from Minsoy (PI 27890) x T284H, Ms3ms3 (A00-68), was screened with 231 simple sequence repeat markers. The ms3 locus mapped to molecular linkage group (MLG) D1b <span class="gene">(Gm02)</span> and is flanked by markers Satt157 and Satt542, with a distance of 3.7 and 12.3 cM, respectively. Female-partial sterile-1 (Fsp1) and the Midwest Oilseed male-sterile (msMOS) mutants previously were located on MLG D1b. msMOS and Fsp1 are independent genes located very close to each other. All 3 genes are located in close proximity of Satt157. We believe that this is the first report of clustering of fertility-related genes in plants. Characterization of these closely linked genes may help in understanding the evolutionary relationship among them.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="gene">GmAOX2b</span> antisense gene compromises vegetative growth and <span class="trait">seed</span> production in soybean.</h2>
			<h3>PMID:22307678  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alternative oxidase mediates the cyanide-resistant respiratory pathway in plant mitochondria. In non-thermogenic plants, the role of alternative oxidase in plant growth and development is not well understood. Soybean (Glycine max) lines carrying a <span class="gene">GmAOX2b</span> antisense gene had compromised vegetative growth and reproductive performance under typical glasshouse growth conditions. The reduction in vegetative growth was demonstrated by reduction in shoot height, the number of leaves per plant and the green leaf area. Antisense plants also had decreased pod formation and <span class="trait">seed</span> to pod ratios, which together led to a reduction in the number and total mass of <span class="trait">seed</span> produced. The negative effects of the antisense gene on pod <span class="trait">set,</span> <span class="trait">seed</span> <span class="trait">set,</span> ovule availability and total <span class="trait">seed</span> mass were primarily confined to the branches, rather than the main stem. The preferential effect of alternative oxidase suppression in the branches is discussed in relation to the reproductive potential of soybean under stress. Taken together, these results demonstrate that alternative oxidase provides the benefit of sustaining plant vegetative growth and reproductive capacity in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthetic performance and <span class="trait">fertility</span> are repressed in <span class="gene">GmAOX2b</span> antisense soybean.</h2>
			<h3>PMID:20097793  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alternative oxidase (AOX) is a cyanide-resistant oxidase that provides an alternative outlet for electrons from the respiratory electron transport chain embedded in the inner membrane of plant mitochondria. Examination of soybean (Glycine max) plants carrying a <span class="gene">GmAOX2b</span> antisense gene showed AOX to have a central role in reproductive development and fecundity. In three independently transformed antisense lines, <span class="trait">seed</span> <span class="trait">set</span> was reduced by 16% to 43%, whereas ovule abortion increased by 1.2- to 1.7-fold when compared with nontransgenic transformation control plants. Reduced fecundity was associated with reductions in whole leaf cyanide-resistant, salicylhydroxamic acid-sensitive respiration and net photosynthesis, but there was no change in total respiration in the dark. The frequency of potential fertilization events was reduced by at least one-third in the antisense plants as a likely consequence of prefertilization defects. Pistils of the antisense plants contained a higher proportion of immature-sized, nonfertile embryo sacs compared with nontransgenic control plants. Increased rates of pollen abortion in vivo and reduced rates of pollen germination in vitro suggested that the antisense gene compromised pollen development and function. Reciprocal crosses between antisense and nontransgenic plants revealed that pollen produced by antisense plants was less active in fertilization. Taken together, the results presented here indicate that AOX expression has an important role in determining normal gametophyte development and function.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Allele-specific marker development and selection efficiencies for both flavonoid 3&#x02032;-hydroxylase and flavonoid 3&#x02032;,5&#x02032;-hydroxylase genes in soybean subgenus <italic> soja </italic></h2>
			<h3>PMID:23463490</h3>
				

				
					<h3>PMCID:3664743</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <address>  <phone> +86-10-82105843 </phone>  <fax> +86-10-82105840 </fax>  <email> qiu_lijuan@263.net </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI), Institute of Crop Science, Chinese Academy of Agricultural Sciences, No.12 Zhongguancun South Street, Haidian District, 100081 Beijing, People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Color is one of the phenotypic markers mostly used to study soybean (Glycine max L. Merr.) genetic, molecular and biochemical processes. Two P450-dependent mono-oxygenases, flavonoid 3'-hydroxylase (F3'H; EC1.14.3.21) and flavonoid 3',5'-hydroxylase (F3'5'H, EC1.14.13.88), both catalyzing the hydroxylation of the B-ring in flavonoids, play an important role in coloration. Previous studies showed that the T locus was a gene encoding <span class="gene">F3'H</span> and the <span class="gene">W1</span> locus co-segregated with a gene encoding F3'5'H in soybean. These two genetic loci have identified to control <span class="trait">seed</span> coat, flower and pubescence colors. However, the allelic distributions of both <span class="gene">F3'H</span> and F3'5'H genes in soybean were unknown. In this study, three novel alleles were identified (two of four alleles for <span class="gene">GmF3'H</span> and one of three alleles for <span class="gene">GmF3'5'H).</span> A <span class="trait">set</span> of gene-tagged markers was developed and verified based on the sequence diversity of all seven alleles. Furthermore, the markers were used to analyze soybean accessions including 170 cultivated soybeans (G. max) from a mini core collection and 102 wild soybeans (G. soja). For both <span class="gene">F3'H</span> and F3'5'H, the marker selection efficiencies for pubescence color and flower color were determined. The results showed that one <span class="gene">GmF3'H</span> allele explained 92.2 % of the variation in tawny and two gmf3'h alleles explained 63.8 % of the variation in gray pubescence colors. In addition, two <span class="gene">GmF3'5'H</span> alleles and one gmF3'5'h allele explained 94.0 % of the variation in purple and 75.3 % in white flowers, respectively. By the combination of the two loci, <span class="trait">seed</span> coat color was determined. In total, 90.9 % of accessions possessing both the gmf3'h-b and gmf3'5'h alleles had yellow <span class="trait">seed</span> coats. Therefore, <span class="trait">seed</span> coat colors are controlled by more than two loci.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The chemistry and genetics of pigmentation in seeds and flowers has also been extensively studied in soybean (Palmer and Stelly <xref ref-type="bibr" rid="CR25"> 1979 </xref> ; Stephens and Nickell <xref ref-type="bibr" rid="CR33"> 1992 </xref> ; Todd and Vodkin <xref ref-type="bibr" rid="CR42"> 1993 </xref> ), Thus far, alleles of at least five loci ( <italic> I, T, W1, R, </italic> and <italic> O </italic> ) are known to function epistatically to control <span class="trait">seed</span> coat pigmentation, six loci ( <italic> W1, W2, W3, W4, Wm, </italic> and <italic> Wp </italic> ) control flower pigmentation and two loci ( <italic> T </italic> and <italic> Td </italic> ) control pubescence color in soybean (Palmer et al. <xref ref-type="bibr" rid="CR26"> 2004 </xref> ; Stephens and Nickell <xref ref-type="bibr" rid="CR33"> 1992 </xref> ). So far, the molecular biology of these soybean loci has been studied but only a few loci have been identified at the molecular level due to the complexity of the soybean genome. Classic genetics showed that <italic> T </italic> conferred brown and <italic> t </italic> conferred gray pubescence color. Further, <italic> T </italic> generally darkens hilum and/or <span class="trait">seed</span> coat color in combination with genotypes at the <italic> I </italic> , <italic> <span class="gene">W1</span> </italic> , <italic> R </italic> and <italic> O </italic> loci (Palmer et al. <xref ref-type="bibr" rid="CR26"> 2004 </xref> ). Cloning and mapping of the soybean <italic> sf3&#x02032;h1 </italic> genomic and cDNA sequences identified the <italic> F3&#x02032;H </italic> gene at the <italic> T </italic> locus on linkage group <span class="gene">C2</span> (chromosome 6) (Toda et al. <xref ref-type="bibr" rid="CR40"> 2002 </xref> ; Zabala and Vodkin <xref ref-type="bibr" rid="CR46"> 2003 </xref> ). Silencing of <italic> sf3&#x02032;h1 </italic> resulted in decreased levels of quercetin and loss of the tawny pigmentation in pubescence (Nagamatsu et al. <xref ref-type="bibr" rid="CR23"> 2009 </xref> ; Nagamatsu et al. <xref ref-type="bibr" rid="CR22"> 2007 </xref> ), suggesting that the <italic> sf3&#x02032;h1 </italic> gene is responsible for the production of quercetin and involved in the control of pigmentation in pubescence. Moreover, the relationship between the <italic> T </italic> locus and chill-tolerance has also been studied (Takahashi and Asanuma <xref ref-type="bibr" rid="CR34"> 1996 </xref> ; Takahashi et al. <xref ref-type="bibr" rid="CR35"> 2005 </xref> ; Toda et al. <xref ref-type="bibr" rid="CR41"> 2012 </xref> ), indicating the importance of the soybean <italic> F3&#x02032;H </italic> gene in pigment biosynthesis and physiological function. The <italic> <span class="gene">W1</span> </italic> locus has a pleiotropic effect on flower and hypocotyl color, in which soybean accessions with purple/white flowers have purple/green hypocotyls. It was confirmed that <italic> <span class="gene">W1</span> </italic> locus co-segregated with a gene encoding F3&#x02032;5&#x02032;H by using near isogenic lines (Zabala and Vodkin <xref ref-type="bibr" rid="CR47"> 2007 </xref> ). In addition, the light-purple flower of the wild soybean B09121 is controlled by a new allele of the <italic> <span class="gene">W1</span> </italic> locus encoding F3&#x02032;5&#x02032;H (Takahashi et al. <xref ref-type="bibr" rid="CR36"> 2010 </xref> ), suggesting that more alleles of these genes remain to be identified.
					</p>
				
					<p>&nbsp;&nbsp;
						The sequences of <italic> F3&#x02032;H </italic> and <italic> F3&#x02032;5&#x02032;H </italic> have been cloned from some soybean accessions; however, no functional markers and allelic diversity analysis of these genes have been published so far. Due to the abundant variation in the soybean genome, whether <italic> F3&#x02032;H </italic> and <italic> F3&#x02032;5&#x02032;H </italic> have other alleles and their distribution in accessions need to be revealed. The core collection and mini core collection of soybean provide an effective platform for genetic diversity studies, novel gene identification, and allele distribution analysis (Song et al. <xref ref-type="bibr" rid="CR32"> 2010 </xref> ; Wang et al. <xref ref-type="bibr" rid="CR43"> 2006 </xref> ). In this study, three novel alleles, two of four alleles for <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and one of three alleles for <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> , were found on the basis of the sequence variation of these two genes in different soybean accessions. A <span class="trait">set</span> of gene-tagged markers were developed and verified. The genetic effects of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> were also detected by genotyping 272 accessions including part of the mini core collection of cultivated soybeans and annual wild soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies showed that the <italic> T </italic> locus was a gene encoding F3&#x02032;H and the <italic> <span class="gene">W1</span> </italic> locus co-segregated with a gene encoding F3&#x02032;5&#x02032;H in soybean, which belonged to the genetic loci controlling <span class="trait">seed</span> coat, flower and pubescence color. The genetic effect of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> alleles on pubescence color and <span class="trait">seed</span> coat was estimated (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ). The results showed that &#x0003e;80&#x000a0;% of accessions possessing the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> (86.3&#x000a0;%) or <italic> gmf3&#x02032;h </italic> - <italic> a2 </italic> (83.3&#x000a0;%) alleles had a tawny pubescence color while &#x0003e;80&#x000a0;% of those possessing the <italic> gmf3&#x02032;h </italic> - <italic> a1 </italic> (84.6&#x000a0;%) or <italic> gmf3&#x02032;h </italic> - <italic> b </italic> (83.3&#x000a0;%) alleles had a gray pubescence color. As to <span class="trait">seed</span> coat color, ~45.4&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;H</span> </italic> had a black <span class="trait">seed</span> coat, which was the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele. However, most accessions possessing <italic> gmf3&#x02032;h </italic> - <italic> a1, gmf3&#x02032;h </italic> - <italic> a2 </italic> or <italic> gmf3&#x02032;h </italic> - <italic> b </italic> had a yellow <span class="trait">seed</span> coat while few had other <span class="trait">seed</span> coat colors. On the other hand, these results also suggested that one <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele explained 177/192 (92.2&#x000a0;%) of tawny and two <italic> gmf3&#x02032;h </italic> alleles ( <italic> gmf3&#x02032;h </italic> - <italic> a1 </italic> and <italic> gmf3&#x02032;h </italic> - <italic> b </italic> ) explained 51/80 (63.8&#x000a0;%) of gray colors. <table-wrap id="Tab4"> <label> Table&#x000a0;4 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and pubescence and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						For the <italic> F3&#x02032;5&#x02032;H </italic> locus, 92.3&#x000a0;% of accessions possessing <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> and 88.9&#x000a0;% possessing <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> b </italic> had a purple flower while 82.1&#x000a0;% of those possessing <italic> gmf3&#x02032;5&#x02032;h </italic> had a white flower (Table&#x000a0; <xref rid="Tab5" ref-type="table"> 5 </xref> ). With regard to <span class="trait">seed</span> coat color, ~54.9&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> had a black <span class="trait">seed</span> coat, which was also the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> allele. However, the most widespread <span class="trait">seed</span> coat color of the <italic> gmf3&#x02032;5&#x02032;h </italic> allele was yellow (73.1&#x000a0;%). The relationship between <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> b </italic> and <span class="trait">seed</span> coat color was not very close and the frequency of each color was similar. Two <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> alleles and one <italic> gmF3&#x02032;5&#x02032;h </italic> allele explained 187/199 (94.0&#x000a0;%) of purple and 55/73 (75.3&#x000a0;%) of white flowers. <table-wrap id="Tab5"> <label> Table&#x000a0;5 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and flower and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						When the genetic effect of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> alleles were combined, the results showed that these loci had a high association with <span class="trait">seed</span> coat color (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> ). Sixty-five percent of accessions possessing both <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> had a black seed-coat and 90.9&#x000a0;% of those possessing both <italic> gmf3&#x02032;h </italic> - <italic> b </italic> and <italic> gmf3&#x02032;5&#x02032;h </italic> had a yellow seed-coat, contributions higher than the single locus. <table-wrap id="Tab6"> <label> Table&#x000a0;6 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						Although the genetic diversity of <italic> G. soja </italic> was reduced 50&#x000a0;% by the bottleneck of domestication (Hyten et al. <xref ref-type="bibr" rid="CR14"> 2006 </xref> ), it appears that selection for <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> did not cause erosion of diversity. This was inferred by the finding of four <italic> <span class="gene">GmF3&#x02032;H</span> </italic> alleles among cultivated soybeans, whereas <italic> G. soja </italic> only contained the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele. With regard to <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> , nearly all wild soybeans (92.2&#x000a0;%) contained the <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> allele while three <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> alleles occurred among cultivated soybeans. The distribution of these alleles in cultivated and wild soybean was similar to an artificial selection locus <italic> <span class="gene">GmTfl1</span> </italic> , which is associated with growth habit (Tian et al. <xref ref-type="bibr" rid="CR39"> 2010 </xref> ). These results suggested that the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> loci might also have undergone artificial selection along with wild soybeans to landraces and then cultivars. However, this kind of selection might go with the selection of <span class="trait">seed</span> coat color since these two genes are all associated with <span class="trait">seed</span> coat color.
					</p>
				
					<p>&nbsp;&nbsp;
						Functional marker development is very important for gene discovery in crops. A polymorphism study of the <italic> <span class="gene">IFS1</span> </italic> and <italic> <span class="gene">IFS2</span> </italic> genes indicated that three SNPs in <italic> <span class="gene">IFS1</span> </italic> and two SNPs in <italic> <span class="gene">IFS2</span> </italic> were closely associated with all individual types and total <span class="trait">seed</span> isoflavone concentrations (Cheng et al. <xref ref-type="bibr" rid="CR3"> 2008 </xref> ). In the present study, gene-tagged markers of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> were developed and verified. Analysis of the genotype in 272 soybeans including the partial mini core collection showed that <italic> <span class="gene">GmF3&#x02032;H</span> </italic> was correlated with pubescence color and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> was correlated with flower color. Therefore, these gene-tagged markers can be used to genotype the cultivars on the target locus. By PCR and/or digestion, the alleles of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> could be clearly separated, indicating that nearly all SNPs or InDels could be used for developing CAPS/dCAPs markers. However, the phenotypes of some accessions were still not correlated with the genotypes, differing from previous reports (Toda et al. <xref ref-type="bibr" rid="CR40"> 2002 </xref> ; Zabala and Vodkin <xref ref-type="bibr" rid="CR47"> 2007 </xref> ). This was due to the existence of other loci controlling pubescence, flower and <span class="trait">seed</span> coat color in soybean and/or the existence of other alleles of these two genes. For example, another locus <italic> Td </italic> is also related to the control of pubescence color in soybean, and some of the cultivars may have the <italic> Td/td </italic> allele that is near grey as exemplified in this study. All wild soybeans possessed the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele, indicating that the <italic> Td </italic> locus may not have effect in the <italic> G. soja </italic> accessions. In addition, the dominant <italic> I </italic> allele exhibits a completely colorless <span class="trait">seed</span> coat phenotype due to dominance inhibition possibly via a posttranscriptional mode of gene silencing (Senda et al. <xref ref-type="bibr" rid="CR30"> 2004 </xref> ). The <italic> T </italic> and <italic> R </italic> loci determine specific <span class="trait">seed</span> coat color only in combination with the recessive <italic> i </italic> allele. Therefore, this interaction also reduced the frequency of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> alleles in the surveyed accessions. Using partial mini core collection, three new alleles, <italic> gmf3&#x02032;h </italic> - <italic> a1 </italic> , <italic> gmf3&#x02032;h </italic> - <italic> a2 </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> b, </italic> were identified, and their frequencies were 4.4, 2.6, and 23.2&#x000a0;%, which indicated that the materials we used for genotyping had abundant genetic variation.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Protease C2, a cysteine endopeptidase involved in the continuing mobilization of soybean beta-conglycinin <span class="trait">seed</span> proteins.</h2>
			<h3>PMID:11342045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo S(1), Tan-Wilson A, Wilson KA.
				<br/> 
				年份：2001 Feb 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The protease that degrades the beta subunit of the soybean (Glycine max (L.) Merrill) storage protein beta-conglycinin was purified from the cotyledons of seedlings grown for 12 days. The enzyme was named protease <span class="gene">C2</span> because it is the second enzyme to cleave the beta-conglycinin storage protein, the first (protease C1) being one that degrades only the <span class="gene">alpha'</span> and alpha subunits of the storage protein to products similar in size and sequence to the remaining intact beta subunit. Protease <span class="gene">C2</span> activity is not evident in vivo until 4 days after imbibition of the <span class="trait">seed.</span> The 31 kDa enzyme is a cysteine protease with a pH optimum with beta-conglycinin as substrate of 5.5. The action of protease <span class="gene">C2</span> on native beta-conglycinin produces a <span class="trait">set</span> of large fragments (52-46 kDa in size) and small fragments (29-25 kDa). This is consistent with cleavage of all beta-conglycinin subunits at the region linking their N- and C-domains. Protease <span class="gene">C2</span> also cleaves phaseolin, the Phaseolus vulgaris vicilin homologous to beta-conglycinin, to fragments in the 25-28 kDa range. N-Terminal sequences of isolated beta-conglycinin and phaseolin products show that protease <span class="gene">C2</span> cleaves at a bond within a very mobile surface loop connecting the two compact structural domains of each subunit. The protease <span class="gene">C2</span> cleavage specificity appears to be dictated by the substrate's three-dimensional structure rather than a specificity for a particular amino acid or sequence.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative estimation of sampling uncertainties for mycotoxins in cereal shipments.</h2>
			<h3>PMID:22651820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bourgeois FS(1), Lyman GJ.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many countries receive shipments of bulk cereals from primary producers. There is a volume of work that is on-going that seeks to arrive at appropriate standards for the quality of the shipments and the means to assess the shipments as they are out-loaded. Of concern are mycotoxin and heavy metal levels, pesticide and herbicide residue levels, and contamination by genetically modified organisms <span class="gene">(GMOs).</span> As the ability to quantify these contaminants improves through improved analytical techniques, the sampling methodologies applied to the shipments must also keep pace to ensure that the uncertainties attached to the sampling procedures do not overwhelm the analytical uncertainties. There is a need to understand and quantify sampling uncertainties under varying conditions of contamination. The analysis required is statistical and is challenging as the nature of the distribution of contaminants within a shipment is not well understood; very limited data exist. Limited work has been undertaken to quantify the variability of the contaminant concentrations in the flow of grain coming from a ship and the impact that this has on the variance of sampling. Relatively recent work by Paoletti et al. in 2006 [Paoletti C, Heissenberger A, Mazzara M, Larcher S, Grazioli E, Corbisier P, Hess N, Berben G, Lübeck PS, De Loose M, et al. 2006. Kernel lot distribution assessment (KeLDA): a study on the distribution of <span class="gene">GMO</span> in large soybean shipments. Eur Food Res Tech. 224:129-139] provides some insight into the variation in <span class="gene">GMO</span> concentrations in soybeans on cargo out-turn. Paoletti et al. analysed the data using correlogram analysis with the objective of quantifying the sampling uncertainty (variance) that attaches to the final cargo analysis, but this is only one possible means of quantifying sampling uncertainty. It is possible that in many cases the levels of contamination passing the sampler on out-loading are essentially random, negating the value of variographic quantitation of the sampling variance. <span class="gene">GMOs</span> and mycotoxins appear to have a highly heterogeneous distribution in a cargo depending on how the ship was loaded (the grain may have come from more than one terminal and <span class="trait">set</span> of storage silos) and mycotoxin growth may have occurred in transit. This paper examines a statistical model based on random contamination that can be used to calculate the sampling uncertainty arising from primary sampling of a cargo; it deals with what is thought to be a worst-case scenario. The determination of the sampling variance is treated both analytically and by Monte Carlo simulation. The latter approach provides the entire sampling distribution and not just the sampling variance. The sampling procedure is based on rules provided by the Canadian Grain Commission (CGC) and the levels of contamination considered are those relating to allowable levels of ochratoxin A (OTA) in wheat. The results of the calculations indicate that at a loading rate of 1000 tonnes h(-1), primary sample increment masses of 10.6 kg, a 2000-tonne lot and a primary composite sample mass of 1900 kg, the relative standard deviation (RSD) is about 1.05 (105%) and the distribution of the mycotoxin (MT) level in the primary composite samples is highly skewed. This result applies to a mean MT level of 2 ng g(-1). The rate of false-negative results under these conditions is estimated to be 16.2%. The corresponding contamination is based on initial average concentrations of MT of 4000 ng g(-1) within average spherical volumes of 0.3 m diameter, which are then diluted by a factor of 2 each time they pass through a handling stage; four stages of handling are assumed. The Monte Carlo calculations allow for variation in the initial volume of the MT-bearing grain, the average concentration and the dilution factor. The Monte Carlo studies seek to show the effect of variation in the sampling frequency while maintaining a primary composite sample mass of 1900 kg. The overall results are presented in terms of operational characteristic curves that relate only to the sampling uncertainties in the primary sampling of the grain. It is concluded that cross-stream sampling is intrinsically unsuited to sampling for mycotoxins and that better sampling methods and equipment are needed to control sampling uncertainties. At the same time, it is shown that some combination of cross-cutting sampling conditions may, for a given shipment mass and MT content, <span class="trait">yield</span> acceptable sampling performance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comprehensive <span class="gene">GMO</span> detection using real-time PCR array: single-laboratory validation.</h2>
			<h3>PMID:22649939  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Mano J(1), Harada M, Takabatake R, Furui S, Kitta K, Nakamura K, Akiyama H, Teshima R, Noritake H, Hatano S, Futo S, Minegishi Y, Iizuka T.
				<br/> 
				年份：2012 Mar-Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have developed a real-time PCR array method to comprehensively detect genetically modified (GM) organisms. In the method, genomic DNA extracted from an agricultural product is analyzed using various qualitative real-time PCR assays on a 96-well PCR plate, targeting for individual GM events, recombinant DNA (r-DNA) segments, taxon-specific DNAs, and donor organisms of the respective r-DNAs. In this article, we report the single-laboratory validation of both DNA extraction methods and component PCR assays constituting the real-time PCR array. We selected some DNA extraction methods for specified plant matrixes, i.e., maize flour, soybean flour, and ground canola seeds, then evaluated the DNA quantity, DNA fragmentation, and PCR inhibition of the resultant DNA extracts. For the component PCR assays, we evaluated the specificity and LOD. All DNA extraction methods and component PCR assays satisfied the criteria <span class="trait">set</span> on the basis of previous reports.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two classes of differentially regulated glutamine synthetase genes are expressed in the soybean nodule: a nodule-specific class and a constitutively expressed class.</h2>
			<h3>PMID:8104530  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roche D(1), Temple SJ, Sengupta-Gopalan C.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have characterized two sets of cDNA clones representing the glutamine synthetase (GS) mRNA in soybean nodules. Using the 3'-untranslated regions of a representative member of each <span class="trait">set,</span> as gene member(s) specific probes, we have shown that one <span class="trait">set</span> of the <span class="gene">GS</span> genes are expressed in a nodule-specific manner, while the other <span class="trait">set</span> is expressed in other tissues, besides the nodules. The nodule-specific <span class="gene">GS</span> genes are expressed in a developmentally regulated manner in the nodules, independent of the onset of nitrogen fixation. The other class of <span class="gene">GS</span> genes is expressed constitutively in all tissues tested, but its expression level is dramatically enhanced in nodules following onset of N2 fixation. The latter <span class="trait">set</span> of genes is also expressed in cotyledons of germinating seedlings in a developmentally regulated manner. Analysis of hybrid select translation products and genomic Southern blots suggests that multiple gene members in each class are expressed in the nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of different constitutive and inducible promoters for the overexpression of transgenes in Arabidopsis thaliana.</h2>
			<h3>PMID:8541491  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Holtorf S(1), Apel K, Bohlmann H.
				<br/> 
				年份：1995 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We compared the organ specificity and the strength of different constitutive (CaMV-35S, CaMV-35Somega, Arabidopsis ubiquitin UBQ1, and barley leaf thionin BTH6 promoter) and one inducible promoter (soybean heat-shock promoter <span class="gene">Gmhsp17.3)</span> in stably transformed Arabidopsis thaliana plants. For this purpose we constructed a <span class="trait">set</span> of plant expression vectors equipped with the different promoters. Using the uidA reporter gene we could show that the CaMV-35S promoter has the highest expression level which was enhanced two- to threefold by the addition of a translational enhancer (TMV omega element) without altering the organ specificity of the promoter. The barley leaf thionin promoter was almost inactive in the majority of lines whereas the ubiquitin promoter exhibited an intermediate strength. The heat-shock promoter was inducible up to 18-fold but absolute levels were lower than in the case of the ubiquitin promoter. Conclusive quantitative results for different organs and developmental stages were obtained by the analysis of 24 stably transformed lines per promoter construct.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A <span class="trait">set</span> of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being <span class="trait">seed</span> oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fatty acid profiling of soybean cotyledons by near-infrared spectroscopy.</h2>
			<h3>PMID:17132452  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roberts CA(1), Ren C, Beuselinck PR, Benedict HR, Bilyeu K.
				<br/> 
				年份：2006 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genetically improved soybean grain often contains altered fatty acid profiles. Such alterations can have deleterious effects on <span class="trait">seed</span> germination and seedling development, making it necessary to monitor fatty acid profiles in follow-up physiological studies. The objective of this research was to quantify the five fatty acids in soybean (Glycine max) cotyledons using near-infrared (NIR) spectroscopy. Soybean cotyledon samples were dried, ground, and scanned with visible and <span class="gene">NIR</span> radiation from 400 to 2500 nm, and reflectance was recorded. Samples were also analyzed by gas chromatography (GC) for palmitic, stearic, oleic, linoleic, and linolenic acids and total oil; GC data, expressed as actual concentration and proportion of total oil, were regressed against spectral data to develop calibration equations. Equation statistics indicated that four of the five fatty acids could be predicted accurately by <span class="gene">NIR</span> spectroscopy; the fifth fatty acid could be determined by subtraction. Principal component analysis revealed that most of the spectral variation in this population was due to chlorophyll absorbance in the visible region. Therefore, the spectra were trimmed to include the <span class="gene">NIR</span> region only (1100-2500 nm), and a second <span class="trait">set</span> of equations was developed. Equations based exclusively on <span class="gene">NIR</span> spectra had equal or greater precision than equations based on visible and <span class="gene">NIR</span> spectra. Principal component analysis and partial least squares analysis revealed that even after trimming, at least 90% of the spectral variation was unrelated to fatty acid, though variation from fatty acid was identified in the second and third principal components. This research provides an <span class="gene">NIR</span> method for complete fatty acid profiling of soybean cotyledons. Equations were achieved with <span class="gene">NIR</span> spectra only, so spectrophotometers that analyze both the visible and <span class="gene">NIR</span> regions are not needed for this analysis. In addition, equations were possible with a 250 mg sample, which is one-tenth the normal sample size for this analysis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, Heterodera glycines Ichinohe). Although this approach successfully mapped a large number of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse <span class="trait">set</span> of 553 soybean plant introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as <span class="trait">seed</span> coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T <span class="trait">set</span> was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less pod wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant <span class="trait">seed</span> weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher <span class="trait">seed</span> weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean <span class="trait">seed</span> <span class="trait">set</span> and size using RNA sequencing analysis.</h2>
			<h3>PMID:28087653</h3>
				

				
			<p>
				作者：Du J(1), Wang S(1), He C(1), Zhou B(2), Ruan YL(3), Shou H(1).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To understand the gene expression networks controlling soybean <span class="trait">seed</span> <span class="trait">set</span> and size, transcriptome analyses were performed in three early <span class="trait">seed</span> developmental stages, using two genotypes with contrasting <span class="trait">seed</span> size. The two-dimensional data <span class="trait">set</span> provides a comprehensive and systems-level view on dynamic gene expression networks underpinning soybean <span class="trait">seed</span> <span class="trait">set</span> and subsequent development. Using pairwise comparisons and weighted gene coexpression network analyses, we identified modules of coexpressed genes and hub genes for each module. Of particular importance are the discoveries of specific modules for the large <span class="trait">seed</span> size variety and for <span class="trait">seed</span> developmental stages. A large number of candidate regulators for <span class="trait">seed</span> size, including those involved in hormonal signaling pathways and transcription factors, were transiently and specifically induced in the early developmental stages. The soybean homologs of a brassinosteroid signaling receptor kinase, a brassinosteroid-signaling kinase, were identified as hub genes operating in the <span class="trait">seed</span> coat network in the early <span class="trait">seed</span> maturation stage. Overexpression of a candidate <span class="trait">seed</span> size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased <span class="trait">seed</span> size and <span class="trait">seed</span> weight. Together, these analyses identified a large number of potential key regulators controlling soybean <span class="trait">seed</span> <span class="trait">set,</span> <span class="trait">seed</span> size, and, consequently, <span class="trait">yield</span> potential, thereby providing new insights into the molecular networks underlying soybean <span class="trait">seed</span> development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Foliar and <span class="trait">Seed</span> Application of Amino Acids Affects the Antioxidant Metabolism of the Soybean Crop.</h2>
			<h3>PMID:28377778</h3>
				

				
			<p>
				作者：Teixeira WF(1), Fagan EB(2), Soares LH(2), Umburanas RC(1), Reichardt K(3), Neto DD(1).
				<br/> 
				年份：2017 Mar 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In recent years, the application of natural substances on crops has been intensified in order to increase the <span class="trait">resistance</span> and <span class="trait">yield</span> of the soybean crop. Among these products are included plant biostimulants that may contain algae extracts, amino acids, and plant regulators in their composition. However, there is little information on the isolated effect of each of these constituents. The objective of this research was to evaluate the effect of the application of isolated amino acids on the antioxidant metabolism of the soybean crop. Experiments were carried out in a greenhouse and in the field with the application of the amino acids glutamate, phenylalanine, cysteine, glycine in <span class="trait">seed</span> treatment, and foliar application at V4 growth stage. Antioxidant metabolism constituents evaluated were superoxide dismutase, catalase, peroxidase, hydrogen peroxide content, proline, and lipid peroxidation. In addition, <span class="trait">resistance</span> enzymes as polyphenol oxidase and phenylalanine ammonia-lyase (PAL) were evaluated. In both experiments, the use of cysteine, only in <span class="trait">seed</span> treatment and in both <span class="trait">seed</span> treatment and foliar application increased the activity of the enzyme <span class="gene">PAL</span> and catalase. Also in both experiments, the use of phenylalanine increased the activity of the enzyme <span class="gene">PAL</span> when the application was carried out as foliar application or both in <span class="trait">seed</span> treatment and foliar application. In the field experiment, the application of glutamate led to an increase in the activity of the catalase and <span class="gene">PAL</span> enzymes for <span class="trait">seed</span> treatment and foliar application. The use of the <span class="trait">set</span> of amino acids was only efficient in foliar application, which led to a greater activity of the enzymes peroxidase, PAL, and polyphenol oxidase. The other enzymes as well as lipid peroxidation and hydrogen peroxide presented different results according to the experiment. Therefore, glutamate, cysteine, phenylalanine, and glycine can act as signaling amino acids in soybean plants, since small doses are enough to increase the activity of the antioxidant enzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel role of the soybean clock gene <italic> LUX ARRHYTHMO </italic> in male reproductive development</h2>
			<h3>PMID:28878247</h3>
				

				
					<h3>PMCID:5587693</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liew </surname>  <given-names> Lim Chee </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Mohan B. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-2910-0393 </contrib-id>  <name>  <surname> Bhalla </surname>  <given-names> Prem L. </given-names>  </name>  <address>  <email> premlb@unimelb.edu.au </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2179 088X </institution-id>  <institution-id institution-id-type="GRID"> grid.1008.9 </institution-id>  <institution> Plant Molecular Biology and Biotechnology Laboratory,  </institution>  <institution> Faculty of Veterinary and Agricultural Sciences, The University of Melbourne,  </institution>  </institution-wrap> Parkville, Victoria 3010 Australia  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2342 0938 </institution-id>  <institution-id institution-id-type="GRID"> grid.1018.8 </institution-id>  <institution> Present Address: Department of Animal, Plant and Soil Science,  </institution>  <institution> School of Life Science, La Trobe University,  </institution>  </institution-wrap> Bundoora, VIC 3086 Australia  </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The evening complex of ELF4-ELF3-LUX proteins is an integral component of a plant circadian clock. LUX ARRHYTHMO (LUX) is one of the key components of the evening complex, and that play a key role in circadian rhythms and flowering. Here, we report that diverged soybean LUX has the additional role in male reproductive development. We studied diurnal and circadian rhythms of soybean LUX <span class="gene">(GmLUXa,</span> <span class="gene">GmLUXb,</span> and <span class="gene">GmLUXc)</span> using qRT-PCR, and show its nuclear localisation by particle bombardment. Yeast-two hybrid (Y2H) studies indicate that both <span class="gene">GmLUXb</span> and <span class="gene">GmLUXc</span> form an evening complex with <span class="gene">GmELF4b</span> and <span class="gene">GmELF3a,</span> respectively. Ectopic expression of <span class="gene">GmLUXb</span> in Arabidopsis lux mutants can complement functions of AtLUX, whereas <span class="gene">GmLUXc</span> generates novel phenotypes of serrated leaves, stunted plants, shortened anther filament, and low <span class="trait">seed</span> <span class="trait">set.</span> Overall, our results suggest that the LUX gene has diverged in soybean where <span class="gene">GmLUXb</span> and <span class="gene">GmLUXc</span> share the role to control flowering time, but <span class="gene">GmLUXc</span> has evolved to regulate anther filament growth and <span class="trait">seed</span> <span class="trait">set</span> by regulating the Gibberellin hormone biosynthesis pathway.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Ectopic expression of <italic> <span class="gene">GmLUXc</span> </italic> in <italic> Arabidopsis lux-4 </italic> mutant shows novel phenotypes. <bold> (A) </bold> Representative plants grown in LD (16&#x02009;L: 8D) and SD (10&#x02009;L: 14D), showed compact and stunted plants of <italic> lux-4 35S::GmLUXc </italic> . ( <bold> B) </bold> <italic> lux-4 35S::GmLUXc </italic> plants have compact inflorescence, short siliques, and low <span class="trait">seed</span> <span class="trait">set</span> as compared to <italic> lux-4 </italic> mutant. <bold> (C) </bold> Flowers of <italic> lux-4 and lux-4 35S::GmLUXc </italic> at different stages of flower development (stage 11&#x02013;15), showing shortened filament (red arrows) during all the developmental stages. <bold> (D) </bold> Alexander&#x02019;s staining and DAPI staining of anthers and pollen of <italic> Columbia </italic> ( <italic> Col </italic> ) and <italic> lux-4 35S::GmLUXc </italic> plants, showing normal pollen. <bold> (E) </bold> Pollen attachment and retention assay of <italic> Columbia </italic> ( <italic> Col </italic> ) and <italic> lux-4 35S::GmLUXc </italic> plants, showing impaired pollen attachment of <italic> lux-4 35S::GmLUXc </italic> plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Gibberellin Pathway is one of the downstream targets of <span class="gene">GmLUXc.</span> <bold> (A) </bold> RT-PCR of downstream genes shows decrease transcript in <italic> AtMYB21 </italic> and <italic> AtMYB24 </italic> and splice variants (red arrows) in <italic> AtGA20x1 </italic> and <italic> AtGA20ox2 </italic> . The data is representative of three biological replicates. Gibberellin (GA <sub> 3 </sub> or GA <sub> 4 </sub> ) treatment rescue. <bold> (B) </bold> stunted phenotypes <bold> (C) </bold> filament length (red arrows). <bold> (D) </bold> <span class="trait">seed</span> <span class="trait">set</span> of <italic> lux-4 35S::GmLUXc </italic> plants. Photos of siliques are from representative plant (from left to right) of <italic> Col </italic> , <italic> lux-4 </italic> , <italic> lux-4 35S::GmLUXc </italic> mock, <italic> lux-4 35S::GmLUXc </italic> GA3 treated, and <italic> lux-4 35S::GmLUXc </italic> GA4 treated. <span class="trait">Seed</span> <span class="trait">set</span> is measured by seeds per siliques and total <span class="trait">seed</span> weight. Data are mean&#x02009;&#x000b1;&#x02009;SE for n&#x02009;=&#x02009;10&#x02013;12. Representative plants treated with GA <sub> 3 </sub> or GA <sub> 4 </sub> or mock (0.02% ethanol) are shown.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>