<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed isoflavone content</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed isoflavone content</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Construction of a high-density genetic map based on large-scale markers developed by specific length amplified fragment sequencing (SLAF-seq) and its application to QTL analysis for <span class="trait">isoflavone</span> <span class="trait">content</span> in <italic> Glycine max </italic></h2>
			<h3>PMID:25494922</h3>
				
					<h3>PMCID:4320444</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Bin </given-names>  </name>  <address>  <email> libin02@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Tian </surname>  <given-names> Ling </given-names>  </name>  <address>  <email> tianling159659@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhang </surname>  <given-names> Jingying </given-names>  </name>  <address>  <email> zhangjycaas@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Huang </surname>  <given-names> Long </given-names>  </name>  <address>  <email> huangl@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Fenxia </given-names>  </name>  <address>  <email> hanfenxia@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Shurong </given-names>  </name>  <address>  <email> yanshurong@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Lianzheng </given-names>  </name>  <address>  <email> wanglianzheng@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Sun </surname>  <given-names> Junming </given-names>  </name>  <address>  <email> sunjunming@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> The National Key Facility for Crop Gene Resources and Genetic Improvement, NFCRI, MOA Key Laboratory of Soybean Biology (Beijing), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 12 Zhongguancun South Street, Beijing, 100081 China  </aff>  <aff id="Aff2">  <label/> Biomarker Technologies Corporation, Beijing, 101300 China  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 11 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A high-density genetic map was constructed by using the SLAF-seq genotyping data. The 5,785 SLAF markers were grouped into 20 linkage groups (LGs) and the order of these markers was arranged (Figures&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , <xref rid="Fig3" ref-type="fig"> 3 </xref> and <xref rid="Fig4" ref-type="fig"> 4 </xref> ). The total genetic distance of this map was 2255.18 cM. The average distance between adjacent markers was 0.43 cM. The largest LG was <span class="gene">Gm11</span> with 62 SLAF markers and a length of 134.05 cM. The smallest LG was <span class="gene">Gm18</span> with 647 SLAF markers and a length of 74.90 cM. The mean chromosome region length was 112.76 cM. We also found that approximately 90.8% of the intervals between adjacent markers were less than 1 cM. There were a total of 40 gaps that were 5 to 10 cM in length and four gaps of&#x02009;&#x0003e;&#x02009;10 cM. The largest gap was mapped to <span class="gene">Gm15</span> with 13.06 cM in length. The detail characters of these 20 LGs are shown in Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> . To assess the quality of this genetic map, two dimensional heat maps of the 20 LGs were generated separately by using pair-wise recombination values for the 5,785 SLAF markers. A heat map for <span class="gene">Gm20</span> is shown as an example in Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S1. These heat maps indicated that the construction of this map was accurate, as the recombination frequency was considerably low among adjacent markers. The collinearity of each LG with the soybean reference genome was also analyzed. As shown in Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> and Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , a relatively high collinearity was observed between 20 LGs and the reference genome. An example collinearity map for <span class="gene">Gm20</span> is also shown in Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S2. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> <bold> The soybean high-density genetic map: Linkage groups <span class="gene">Gm01-Gm07.</span> </bold> The SLAF markers and their location are shown on the right and left side, respectively. The 11 QTLs identified across various environments are depicted on the right side of each linkage group as black ovals. The name of each QTL, shown near each oval, is a composite of the influenced trait: genistin (G), daidzin (D), glycitin (GL), malonyldaidzin (MD), malonylgenistin (MG), malonylglycitin (MGL) and total <span class="trait">isoflavone</span> <span class="trait">content</span> (TOT) followed by the chromosome number. For QTLs underlying the contents of multiple <span class="trait">isoflavone</span> components, the name is a composite of isoflavones (IF) followed by the chromosome number.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The soybean high-density genetic map: Linkage groups <span class="gene">Gm08-Gm14.</span> </bold> The SLAF markers and their location are shown on the right and left side, respectively. The 11 QTLs identified across various environments are depicted on the right side of each linkage group as black ovals. The name of each QTL, shown near each oval, is a composite of the influenced trait: genistin (G), daidzin (D), glycitin (GL), malonyldaidzin (MD), malonylgenistin (MG), malonylglycitin (MGL) and total <span class="trait">isoflavone</span> <span class="trait">content</span> (TOT) followed by the chromosome number. For QTLs underlying the contents of multiple <span class="trait">isoflavone</span> components, the name is a composite of isoflavones (IF) followed by the chromosome number.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The soybean high-density genetic map: Linkage groups <span class="gene">Gm15-Gm20.</span> </bold> The SLAF markers and their location are shown on the right and left side, respectively. The 11 QTLs identified across various environments are depicted on the right side of each linkage group as black ovals. The name of each QTL, shown near each oval, is a composite of the influenced trait: genistin (G), daidzin (D), glycitin (GL), malonyldaidzin (MD), malonylgenistin (MG), malonylglycitin (MGL) and total <span class="trait">isoflavone</span> <span class="trait">content</span> (TOT) followed by the chromosome number. For QTLs underlying the contents of multiple <span class="trait">isoflavone</span> components, the name is a composite of isoflavones (IF) followed by the chromosome number.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the high-density genetic map, the QTLs underlying the <span class="trait">isoflavone</span> <span class="trait">content</span> were identified. The threshold of logarithm of odds (LOD) scores for evaluating the statistical significance of QTL effects was determined using 1,000 permutations. As a result, intervals with a LOD value above 2.5 were detected as effective QTLs using the QTL ICIMapping V3.3 software. According to the threshold, 89 QTLs were detected for individual and total <span class="trait">isoflavone</span> contents. The 89 QTLs overlapped and can be classified into 41 loci (shown in Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). Of these loci, 11 QTLs were detected across various environments (shown in Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> , Figures&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> , <xref rid="Fig3" ref-type="fig"> 3 </xref> and <xref rid="Fig4" ref-type="fig"> 4 </xref> and Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S4). These QTLs may represent the genetic basis of the <span class="trait">isoflavone</span> accumulation and were thus focused in our subsequent analysis. As described in Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> , the 11 QTLs were mapped to <span class="gene">Gm4,</span> <span class="gene">Gm7,</span> <span class="gene">Gm8,</span> <span class="gene">Gm13,</span> <span class="gene">Gm14,</span> <span class="gene">Gm16</span> and <span class="gene">Gm20.</span> The average phenotypic variance explained by individual QTL of the 11 loci ranged from 8.61% to 28.00% and the average LOD score ranged from 3.50 to 12.40. The additive effects of <italic> qIF8 </italic> , <italic> qD4 </italic> , <italic> qIF7 </italic> , <italic> qGL8-2 </italic> , <italic> qMGL14 </italic> , <italic> qMG14 </italic> and <italic> qIF20-2 </italic> were derived mainly from the female parent ( <italic> cv. </italic> Luheidou2), while those of <italic> qMGL7 </italic> , <italic> qIF13 </italic> , <italic> qIF16-1 </italic> and <italic> qIF16-2 </italic> were derived mainly from the male parent ( <italic> cv. </italic> Nanhuizao). Our comparative analysis showed that 18 of the total 41 loci overlapped with previously reported QTLs (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). For the 11 consistent QTLs, five loci ( <italic> qD4 </italic> , <italic> qIF7 </italic> , <italic> qGL18-2 </italic> , <italic> qIF13 </italic> and <italic> qIF16-1 </italic> ) overlapped with previously reported QTLs (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). The remaining six QTLs ( <italic> qMGL7 </italic> , <italic> qIF8 </italic> , <italic> qMGL14 </italic> , <italic> qMG14 </italic> , <italic> qIF16-2 </italic> and <italic> qIF20-2 </italic> ) were not found in previous reports, thus they may present novel loci for the <span class="trait">isoflavone</span> <span class="trait">content.</span> Noticeably, <italic> qIF20-2 </italic> was associated with daidzin, genistin, malonyldaidzin, malonylgenistin, and total <span class="trait">isoflavone</span> <span class="trait">content</span> across multiple environments. The LOD score of this locus ranged from 2.86 to 18.89 for individual <span class="trait">isoflavone</span> components among various environments, and it could explain 8.67% to 35.29% of phenotypic variance (Table&#x000a0; <xref rid="Tab5" ref-type="table"> 5 </xref> ). Therefore, this QTL may be the major locus for <span class="trait">isoflavone</span> accumulation. As an example, the LOD curves of <italic> qIF20 </italic> for daidzin were shown in Figure&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> . <table-wrap id="Tab4"> <label> Table 4 </label> <caption> <p> <bold> The characters of 11 QTLs associated with individual and total <span class="trait">isoflavone</span> contents across various environments </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						<sup> 2 </sup> The Effect of QTL is composite of the particular <span class="trait">isoflavone</span> component followed by the specific environments. It represents the particular <span class="trait">isoflavone</span> components [i. e., genistin (G), daidzin (D), glycitin (GL), malonyldaidzin (MD), malonylgenistin (MG), malonylglycitin (MGL)] and total isoflavones (TOT) that are controlled by this QTL in specific environments [i. e., <span class="gene">E1</span> (2009 at Changping), E2 (2009 at Shunyi), E3 (2010 at Shunyi), and E4 (2011 at Shunyi)].
					</p>
				
					<p>&nbsp;&nbsp;
						<sup> 2 </sup> The Effect of QTL is composite of the particular <span class="trait">isoflavone</span> component followed by the specific environments. It represents the particular <span class="trait">isoflavone</span> components [i. e., genistin (G), daidzin (D), glycitin (GL), malonyldaidzin (MD), malonylgenistin (MG), malonylglycitin (MGL)] and total isoflavones (TOT) that are controlled by this QTL in specific environments [i. e., <span class="gene">E1</span> (2009 Changping), E2 (2009 Shunyi), E3 (2010 Shunyi), and E4 (2011 Shunyi)].
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the QTL mapping results, the genome interval regions within these 41 QTLs were also compared with the soybean reference genome sequences. Potential genes within the QTL intervals were annotated and analyzed. As depicted in Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2, 13 genes encoding enzymes involved in the <span class="trait">isoflavone</span> biosynthetic pathway were discovered including <italic> 4-coumarate: coenzyme A ligase </italic> ( <italic> <span class="gene">4CL</span> </italic> ), <italic> chalcone isomerase </italic> ( <italic> <span class="gene">CHI</span> </italic> ), <italic> chalcone reductase </italic> ( <italic> CHR </italic> ), <italic> <span class="trait">isoflavone</span> synthase </italic> ( <italic> <span class="gene">IFS</span> </italic> ) and <italic> O-methyltransferase </italic> ( <italic> OMT </italic> ). Moreover, of these 13 genes, three genes were identified within the 11 consistent QTLs. The inclusion of these genes suggests a high efficiency and accuracy of this map in QTL mapping for <span class="trait">isoflavone</span> <span class="trait">content.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies have investigated the QTLs underlying <span class="trait">isoflavone</span> <span class="trait">content</span> in soybean [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR37"> 37 </xref> &#x02013; <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Approximately 45 QTLs have been identified. In this study, 41 QTLs were identified for individual and total <span class="trait">isoflavone</span> contents (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2), and approximately half of these loci have previously been associated with <span class="trait">isoflavone</span> <span class="trait">content</span> (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). Moreover, many genes encoding <span class="trait">isoflavone</span> biosynthetic enzymes were identified within the genomic region of these loci (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S2). For example, both <span class="trait">isoflavone</span> synthase (IFS) genes ( <italic> <span class="gene">IFS1</span> </italic> and <italic> <span class="gene">IFS2</span> </italic> ) that are active in soybean were found within the <italic> qGL7 </italic> and <italic> qIF13 </italic> regions, respectively. The <span class="gene">IFS</span> catalyzes the first committed step in the <span class="trait">isoflavone</span> biosynthetic pathway, which is a branch of the phenylpropanoid pathway. In this pathway, <span class="gene">IFS</span> redirects the intermediates from the flavonoid pathway to corresponding isoflavones and thus plays a key role in <span class="trait">isoflavone</span> biosynthesis [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. The discovery of previously reported QTLs and genes encoding <span class="trait">isoflavone</span> biosynthetic enzymes suggests that this genetic map has a high resolution and accuracy for QTL mapping.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparative genomic analysis showed that five of these 11 consistent QTLs had been reported in previous studies, while the other six QTLs were identified for the first time. Interestingly, one of these six novel QTLs, <italic> qIF 20-2 </italic> , had a significant effect on almost all <span class="trait">isoflavone</span> components (daidzin, genistin, malonylgenistin, and malonyldaidzin) and total <span class="trait">isoflavone</span> contents across various environments. The average phenotypic variance explained by this QTL ranged from 8.67% to 35.29 % with an average of 19.62%, and the LOD values ranged from 2.86 to 18.89, with an average of 9.11. A previous study reported another major QTLon <span class="gene">Gm05,</span> which also had a significant influence on individual and total <span class="trait">isoflavone</span> contents and could explain over 30.0% of phenotypic variance of <span class="trait">isoflavone</span> <span class="trait">content</span> [ <xref ref-type="bibr" rid="CR7"> 7 </xref> ]. These results demonstrate that despite the strong effect of environmental factors on the accumulation of isoflavones, there major QTLs or genes for <span class="trait">isoflavone</span> accumulation may exist.
					</p>
				
					<p>&nbsp;&nbsp;
						Additional file 1: Figure S1: Heat map showing a matrix of pair-wise recombination values for SLAF markers along <span class="gene">Gm20.</span> The colors represent the strength of linkage in recombination values between all pairs of markers. The grey color indicates the lowest recombination scores, which suggest a strong linkage between markers. The red color represents the highest recombination scores, which suggest no linkage between markers. The yellow color represents middle recombination scores and some amount of linkage between markers. <bold> Figure S2. </bold> The collinearity map of <span class="gene">Gm20.</span> The yellow bar indicates the linkage group <span class="gene">Gm20,</span> while the blue bar indicates the corresponding chromosome of the soybean reference genome. The SLAF markers plus their location in centimorgan (cM) and kilo base pairs (Kb) with respect to the first marker in the LG are indicated on each side. <bold> Figure S3. </bold> Frequency distributions of six <span class="trait">isoflavone</span> components for the 110 RILs planted at Changping experimental station in 2009. <bold> Figure S4. </bold> Details of SLAF markers for 11 QTLs underlying individual and total <span class="trait">isoflavone</span> contents across various environments. The SLAF markers for these QTLs were indicated on the right side of LGs, while the genetic distance between adjacent SLAF markers were shown on the left side. <bold> Table S1. </bold> The correlation analysis of total <span class="trait">isoflavone</span> contents among four environments. <bold> Table S2. </bold> The characters of 41 QTLs associated with <span class="trait">isoflavone</span> <span class="trait">content.</span> (PDF 616 KB)
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping <span class="trait">Isoflavone</span> QTL with Main, Epistatic and QTL &#x000d7; Environment Effects in Recombinant Inbred Lines of Soybean</h2>
			<h3>PMID:25738957</h3>
				
					<h3>PMCID:4349890</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="econtrib001">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="econtrib001">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yongguang </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhan </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Over fifty QTLs underlying individual and/or individual and total <span class="trait">isoflavone</span> <span class="trait">content</span> have been reported [ <xref rid="pone.0118447.ref014" ref-type="bibr"> 14 </xref> &#x02013; <xref rid="pone.0118447.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0118447.ref031" ref-type="bibr"> 31 </xref> &#x02013; <xref rid="pone.0118447.ref037" ref-type="bibr"> 37 </xref> ]. Among them, two QTL with main effects, located in <span class="gene">Gm05</span> (soybean chromosome 5, LGA1) and <span class="gene">Gm08</span> (LGA2), respectively, consistently affected <span class="trait">isoflavone</span> <span class="trait">content</span> across multi-environments [ <xref rid="pone.0118447.ref030" ref-type="bibr"> 30 </xref> , <xref rid="pone.0118447.ref035" ref-type="bibr"> 35 </xref> ]. Striking eQTL hotspots associated with soybean seed-specific expression on Gm 20, 7, and 13 were discovered by Bolon et al. [ <xref rid="pone.0118447.ref038" ref-type="bibr"> 38 </xref> ]. Wang et al. [ <xref rid="pone.0118447.ref039" ref-type="bibr"> 39 </xref> ] identified seven candidate genes on <span class="gene">Gm13</span> (LG F) and assumed <span class="gene">Gm13</span> could be a hotspot of gene cluster that regulated <span class="trait">seed</span> <span class="trait">isoflavone</span> <span class="trait">content.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						qDZF_2, qGTF_1 and qTIF_2 between Satt144-Satt569, and qGCM_1, qGTM_1 and qTIM_1 between Satt540-Sat_244, were identified in multiple environments ( <xref rid="pone.0118447.g001" ref-type="fig"> Fig. 1 </xref> ). These QTL detected by Satt540 and QTL detected by Satt144 on LG M and on LG F in this study were the same or similar to that of our previous studies [ <xref rid="pone.0118447.ref031" ref-type="bibr"> 31 </xref> &#x02013; <xref rid="pone.0118447.ref034" ref-type="bibr"> 34 </xref> , <xref rid="pone.0118447.ref039" ref-type="bibr"> 39 </xref> ], which provided a valuable resource for MAS to develop soybean varieties with high <span class="trait">seed</span> <span class="trait">isoflavone</span> <span class="trait">content.</span> Previously, two major QTL consistently affected <span class="trait">isoflavone</span> <span class="trait">content</span> across multiple environments were mapped on <span class="gene">Gm05</span> (LG A1) and <span class="gene">Gm08</span> (LG A2) by Gutierrez et al. [ <xref rid="pone.0118447.ref030" ref-type="bibr"> 30 </xref> ] and Yang et al. [ <xref rid="pone.0118447.ref035" ref-type="bibr"> 35 </xref> ], respectively. Here, we mapped three new QTL (qTIA2_1 located in Sct_067-Satt470; qGCA2_1 and qGTA2_1 located in Sat_040-Satt233) associated with individual and total <span class="trait">isoflavone.</span> The qGTA2_1 was detected in four environments and explained 3.51%-11.58% of the phenotypic variation. This QTL was near to the major locus identified by Yang et al. [ <xref rid="pone.0118447.ref035" ref-type="bibr"> 35 </xref> ] that controlled the same trait GT, suggesting that qGTA2_1 might be an enzyme-related locus. The qGCA2_1 and qGTA2_1 were identified between the same marker interval of Sat_040-Satt233, implying that there could be some genetic factors regulating the accumulation of GC and GT. These stable QTL were responsible for large <italic> a </italic> effects ( <xref rid="pone.0118447.t002" ref-type="table"> Table 2 </xref> ). As suggested by Tanksley [ <xref rid="pone.0118447.ref048" ref-type="bibr"> 48 </xref> ], QTL with higher <italic> a </italic> effects are more likely to be stable across multiple environments. Most of the QTL discussed above with higher <italic> a </italic> effects (Significant at P = 0.01) were stable across at least three environments (Tables <xref rid="pone.0118447.t001" ref-type="table"> 1 </xref> and <xref rid="pone.0118447.t002" ref-type="table"> 2 </xref> ). These seven novel major QTL were selected to do the GGE Bioplot analysis, and only explaineding 59% of the G and G &#x000d7; E variation of <span class="trait">seed</span> <span class="trait">isoflavone,</span> which was lower than other&#x02019;s studies [ <xref rid="pone.0118447.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0118447.ref031" ref-type="bibr"> 31 </xref> ]. This could be caused by few QTL involved. The contribution of G and G &#x000d7; E to <span class="trait">seed</span> <span class="trait">isoflavone</span> phenotypic variation could be increased to 79% after the excluding of E4 (maybe a mega environment), indicating that E4 had significant influence on <span class="trait">isoflavone</span> <span class="trait">content</span> ( <xref rid="pone.0118447.s001" ref-type="supplementary-material"> S1 Fig </xref> .). Moreover, in order to examine the accuracy of the results by GGE Biplot analysis, QTLNetwork 2.0 software [ <xref rid="pone.0118447.ref049" ref-type="bibr"> 49 </xref> ] was used to analyze the interaction between QTL and environments, and the result was similar to GGE Biplot results ( <xref rid="pone.0118447.s002" ref-type="supplementary-material"> S1 Table </xref> ), indicating that QTL detected in multiple environments were more stable.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the newly identified QTL, the qTIF_1 shared the same marker Satt569 with qDZF_2, qGTF_1 and qTIF_2 in multi-environments, suggesting that there are some genetic elements, such as genes or factors could affect the accumulation of DZ, GT and TI ( <xref rid="pone.0118447.t001" ref-type="table"> Table 1 </xref> , <xref rid="pone.0118447.g001" ref-type="fig"> Fig. 1 </xref> ). Additionally, six pairs of QTL overlapped with each other and shared the same marker interval, inferring that some genetic elements could regulate the accumulation of different <span class="trait">isoflavone</span> components in these intervals ( <xref rid="pone.0118447.t001" ref-type="table"> Table 1 </xref> ). Among the 23 newly identified QTL, five QTL intervals were completely overlapped with our previously reported eQTL and a total of eleven candidate genes within the overlapped eQTL and QTL were identified [ <xref rid="pone.0118447.ref039" ref-type="bibr"> 39 </xref> ]. For example, the newly identified QTL (qGCE_1, qGTE_1 and qTIE_1) located in the interval of Sat_124- Sat_380 shared the same marker Sat_380 with the eQTL qF3HE_1, implying the <italic> <span class="gene">F3H</span> </italic> gene in the phenylpropanoid pathway could affect the accumulation of GC, GT and TI.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of Soybean <span class="trait">Isoflavone</span> Reductase ( <italic> <span class="gene">GmIFR</span> </italic> ) Enhances <span class="trait">Resistance</span> to <italic> Phytophthora sojae </italic> in Soybean</h2>
			<h3>PMID:26635848</h3>
				

				
					<h3>PMCID:4655237</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ninghui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Lidong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dayong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Sujie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Liangyu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Pengfei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Shuzhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257714/overview"/>  </contrib>
				<br/> 
				年份：<day> 05 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Isoflavone</span> reductase (IFR) is an enzyme involved in the biosynthetic pathway of isoflavonoid phytoalexin in plants. IFRs are unique to the plant kingdom and are considered to have crucial roles in plant response to various biotic and abiotic environmental stresses. Here, we report the characterization of a novel member of the soybean <span class="trait">isoflavone</span> reductase gene family <span class="gene">GmIFR.</span> Overexpression of <span class="gene">GmIFR</span> transgenic soybean exhibited enhanced <span class="trait">resistance</span> to Phytophthora sojae. Following stress treatments, <span class="gene">GmIFR</span> was significantly induced by P. sojae, ethephon (ET), abscisic acid (placeCityABA), salicylic acid (SA). It is located in the cytoplasm when transiently expressed in soybean protoplasts. The daidzein levels reduced greatly for the seeds of transgenic plants, while the relative <span class="trait">content</span> of glyceollins in transgenic plants was significantly higher than that of non-transgenic plants. Furthermore, we found that the relative expression levels of reactive oxygen species (ROS) of transgenic soybean plants were significantly lower than those of non-transgenic plants after incubation with P. sojae, suggesting an important role of <span class="gene">GmIFR</span> might function as an antioxidant to reduce ROS in soybean. The enzyme activity assay suggested that <span class="gene">GmIFR</span> has <span class="trait">isoflavone</span> reductase activity.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In a previous study, a cDNA library enriched for mRNAs encoding ESTs that increased in abundance during infection with <italic> Phytophthora sojae </italic> was constructed by suppression subtractive hybridization from leaf tissues of a high resistant soybean cultivar &#x0201c;Suinong 10,&#x0201d; and an EST homologous to an <span class="trait">isoflavone</span> reductase from white lupin ( <italic> Lupinus albus </italic> ) was identified to be up-regulated by microarray and real-time PCR (Xu et al., <xref rid="B57" ref-type="bibr"> 2012 </xref> ). In this study, the full-length EST, designated <italic> <span class="gene">GmIFR</span> </italic> (GenBank accession no. <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="NM_001254100"> NM_001254100 </ext-link> , NCBI protein no. <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="NP_001241029"> NP_001241029 </ext-link> ), was isolated through RT-PCR from &#x0201c;Suinong 10&#x0201d; soybean, and the transgenic soybean plants over-expressing <italic> <span class="gene">GmIFR</span> </italic> gene under the control of 35S promoter were produced. The expression patterns of <italic> <span class="gene">GmIFR</span> </italic> induced under biotic stresses were also examined. Moreover, the function of <italic> <span class="gene">GmIFR</span> </italic> and the <span class="trait">content</span> of daidzein, genistein, glycitein, the relative <span class="trait">content</span> of glyceollins and the ROS in transgenic plants were investigated. <span class="gene">GmIFR</span> protein could catalyze a distinct NADPH-dependent oxidoreductase reaction by enzyme activity assay, suggesting that <span class="gene">GmIFR</span> has <span class="trait">isoflavone</span> reductase activity. Together, we report insights into the function of an <span class="trait">isoflavone</span> reductase (IFR) in soybean, namely <span class="gene">GmIFR,</span> in host responses to <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						A suppression subtractive hybridization library coupled with cDNA microarrays was queried using a soybean EST encoding an EST homologous to an <span class="trait">isoflavone</span> reductase from white lupin ( <italic> L. albus </italic> ), previously shown to be up-regulated in &#x0201c;Suinong 10&#x0201d; soybean inoculated with <italic> P. sojae </italic> (Xu et al., <xref rid="B57" ref-type="bibr"> 2012 </xref> ). Here, the full-length cDNA (tremed <italic> <span class="gene">GmIFR</span> </italic> , GenBank accession no. <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="NM_001254100"> NM_001254100 </ext-link> , NCBI protein no. <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="NP_001241029"> NP_001241029 </ext-link> ) of the EST was amplified by RT-PCR from cDNA of &#x0201c;Suinong 10&#x0201d; using the primer pairs <italic> <span class="gene">GmIFRF</span> </italic> and <italic> <span class="gene">GmIFRR</span> </italic> (Supplementary Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). PCR was performed as follows: 94&#x000b0;C for 5 min, followed 30 cycles of 94&#x000b0;C for 30 s, 60&#x000b0;C for 30 s, and 72&#x000b0;C for 90 s, with a final extension at 72&#x000b0;C for 10 min. The amplification product was gel purified and cloned into the pMD18-T vector (TaKaRa, Dalian, China). An analysis of protein structure was performed using Smart ( <ext-link ext-link-type="uri" xlink:href="http://smart.embl-heidelberg.de/"> http://smart.embl-heidelberg.de/ </ext-link> ). Sequence alignments were performed using DNAMAN software ( <ext-link ext-link-type="uri" xlink:href="http://www.lynnon.com/"> http://www.lynnon.com/ </ext-link> ). A phylogenetic analysis of <italic> <span class="gene">GmIFR</span> </italic> and various heterologous IFR proteins was performed using MEGA4 software (Tamura et al., <xref rid="B52" ref-type="bibr"> 2007 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In order to determine whether <span class="gene">GmIFR</span> has <span class="trait">isoflavone</span> reductase activity, the enzyme assays of <span class="gene">GmIFR</span> was analyzed. The fusion protein was used in the enzyme assays. <span class="gene">GmIFR</span> activity was analyzed according to the method described by Paiva et al. ( <xref rid="B34" ref-type="bibr"> 1991 </xref> ) with minor modifications. High Performance Liquid Chromatography (HPLC) was used to separate the substrate and product.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to determine whether <span class="gene">GmIFR</span> has <span class="trait">isoflavone</span> reductase activity, the enzyme assays of <span class="gene">GmIFR</span> was analyzed. The reductase reactions in the presence of NADPH were measured by reversed-phase HPLC: the formation of the product from the substrate. As shown in Figures <xref ref-type="fig" rid="F4"> 4B,C </xref> , the protein purified from <italic> Transetta </italic> (DE3) expressing the fusion protein showed clear <span class="trait">isoflavone</span> reductase activity, with 2&#x02032;-hydroxyformononetin as the substrate. These results proved that <span class="gene">GmIFR</span> is an <span class="trait">isoflavone</span> reductase.
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate whether overexpression of <italic> <span class="gene">GmIFR</span> </italic> in soybean has an effect on Phytophthora root rot <span class="trait">resistance,</span> the living cotyledons of three T8 transgenic soybean plants (T8-80, T8-88, T8-96) were selected by Real-time PCR (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Resistance</span> analysis of <italic> <span class="gene">GmIFR</span> </italic> transgenic soybean plants </bold> . <bold> (A) </bold> qRT-PCR determining the relative abundance of <italic> <span class="gene">GmIFR</span> </italic> (lines T8-80, T8-88, and T8-96) in the transgenic soybean plants. The non-transgenic soybean plants were used as controls. For each sample, three biological replicates were analyzed with their respective three technical replicates and statistically analyzed using Student's <italic> t </italic> -test ( <sup> ** </sup> <italic> P </italic> &#x0003c; 0.01). Bars indicate standard error of the mean. <bold> (B) </bold> Disease symptoms after infection with <italic> P. sojae </italic> . Lesions on living cotyledon at 72 h with <italic> P. sojae </italic> isolate race 1. For infection assays, three biological replicates were analyzed with their respective technical replicates. And do the same with control. <bold> (C) </bold> Relative lesion area of transgenic soybean cotyledon infection with <italic> P. sojae </italic> after 72 h. Seventy-two hours a represents the cotyledon of transgenic soybean and non-transgenic soybean infected with V8 juice agar, and 72 h-b represents the cotyledon were treated with a <italic> P. sojae </italic> inoculum. The average lesion area of each independent transgenic line ( <italic> n </italic> = 3) was calculated and their relative lesion areas are shown in columns after comparison with the average lesion area on non-transgenic soybean. The statistically analyzed using Student's <italic> t </italic> -test ( <sup> ** </sup> <italic> P </italic> &#x0003c; 0.01). Bars indicate standard error of the mean.
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in Figure <xref ref-type="fig" rid="F5"> 5B </xref> , the cotyledons of the non-transgenic soybean plants detached and exhibited clear and large water-soaked lesions compared with those of the transgenic plants after 72 h of incubation with <italic> P. sojae </italic> (Figure <xref ref-type="fig" rid="F5"> 5B </xref> ). The lesion area of the three transgenic lines was significantly ( <italic> P </italic> &#x0003c; 0.01) smaller than that of non-transgenic soybean plants at 72 h after inoculation (Figure <xref ref-type="fig" rid="F5"> 5C </xref> ). These results indicated that constitutive expression of the <italic> <span class="gene">GmIFR</span> </italic> enhances <span class="trait">resistance</span> toward <italic> P. sojae </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						It is well known that daidzein, genistein, and glycitein were the essential components of isoflavones (Wang et al., <xref rid="B55" ref-type="bibr"> 2014 </xref> ) and IFR was a crucial enzyme involved in the synthesis of the glyceollins from daidzein (Graham et al., <xref rid="B15" ref-type="bibr"> 1990 </xref> ; Oliver et al., <xref rid="B33" ref-type="bibr"> 2003 </xref> ). Thus, changes in the <italic> IFR </italic> expression level may cause change of isoflavonoid and glyceollins <span class="trait">content</span> in soybean. To study the relationship between isoflavonoid and glyceollins <span class="trait">content</span> in soybean seeds and the <italic> <span class="gene">GmIFR</span> </italic> gene expression level, the isoflavonoid <span class="trait">content</span> and the relative <span class="trait">content</span> of the glyceollins were measured in the seeds of transgenic soybean plants and non-transgenic soybean plants. As shown in Figure <xref ref-type="fig" rid="F7"> 7A </xref> , the daidzein levels in the seeds (numbered S-T8-80, S-T8-88, and S-T8-96) developed from the three independent transformed lines T8-80, T8-88, and T8-96 reduced greatly, while levels of genistein and glycitein had little change compared to those of control (Figures <xref ref-type="fig" rid="F7"> 7B,C </xref> ). The relative <span class="trait">content</span> of glyceollins in the transgenic plants was significantly higher than that of non-transgenic plants (Figure <xref ref-type="fig" rid="F7"> 7D </xref> ), suggesting that an important role of <italic> <span class="gene">GmIFR</span> </italic> involved in the synthesis of the glyceollins from daidzein in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we identified a novel <italic> <span class="gene">GmIFR</span> </italic> gene that encodes for a NAD(P)H -dependant oxidoreductase, enhances <span class="trait">resistance</span> to <italic> P. sojae </italic> when over-expressed in soybean. The gene encoding isoflavonoid reductase (IFR), one of the key enzymes in isoflavonoid phytoalexin biosynthesis, was first cloned from alfalfa ( <italic> Medicago sativa </italic> L.; Paiva et al., <xref rid="B34" ref-type="bibr"> 1991 </xref> ). However, there is little knowledge about the biological function of IFR in soybean. Here, we report for the first time that <italic> <span class="gene">GmIFR</span> </italic> transgenic soybean plants inoculated with <italic> P. sojae </italic> display significantly altered responses to pathogen infection.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants encounter a range of environmental stresses in their natural environments and have evolved a wide range of mechanisms to cope with them (Dixon and Paiva, <xref rid="B9" ref-type="bibr"> 1995 </xref> ; Zhang et al., <xref rid="B62" ref-type="bibr"> 2008 </xref> ). There are multiple stress perception and signaling pathways, some of which are specific, whereas others cross-talk at various steps (Kunkel and Brooks, <xref rid="B23" ref-type="bibr"> 2002 </xref> ; Chinnusamy et al., <xref rid="B6" ref-type="bibr"> 2004 </xref> ; Fujita et al., <xref rid="B13" ref-type="bibr"> 2006 </xref> ; R&#x000ed;podas et al., <xref rid="B40" ref-type="bibr"> 2013 </xref> ). It has been reported that higher levels of IFR determining fungal <span class="trait">resistance</span> in response to <italic> A. rabiei </italic> in chickpea (Daniel et al., <xref rid="B8" ref-type="bibr"> 1990 </xref> ). In the present study, we demonstrated that overexpression of the <italic> <span class="gene">GmIFR</span> </italic> gene improved <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean. Moreover, the genes with high sequence similarity to IFRs have been identified in non-leguminous plants and are called <span class="trait">isoflavone</span> reductases like (IRL; Shoji et al., <xref rid="B44" ref-type="bibr"> 2002 </xref> ). In most cases, IRLs might involve in responses to biotic or abiotic stresses (Shoji et al., <xref rid="B44" ref-type="bibr"> 2002 </xref> ; Zhu et al., <xref rid="B64" ref-type="bibr"> 2009 </xref> ; Cheng et al., <xref rid="B5" ref-type="bibr"> 2013 </xref> ). A recent study suggested that GbIRL is involved in regulating ABA, JA, and ET stress responses in Ginkgo (Cheng et al., <xref rid="B5" ref-type="bibr"> 2013 </xref> ). In this study, the expression of <italic> <span class="gene">GmIFR</span> </italic> following various stress treatments was analyzed. The results showed that inoculation with <italic> P. sojae </italic> as biotic stress and woundings as abiotic stress significantly increased the transcript levels of <italic> <span class="gene">GmIFR</span> </italic> in soybean plants. The expression of <italic> <span class="gene">GmIFR</span> </italic> was also induced by treatments with SA, ET, JA, and ABA. ABA is a phytohormone that is extensively involved in responses to abiotic stresses such as <span class="trait">drought</span> and low temperature, as well as osmotic stress (Skriver and Mundy, <xref rid="B46" ref-type="bibr"> 1990 </xref> ). In contrast, the phytohormones SA, JA, and ET play central roles in biotic stress signaling following pathogen infection (Pieterse et al., <xref rid="B38" ref-type="bibr"> 2009 </xref> ; Robert-Seilaniantz et al., <xref rid="B41" ref-type="bibr"> 2011 </xref> ; Sugano et al., <xref rid="B49" ref-type="bibr"> 2013 </xref> ). Therefore, it is possible to speculate that <italic> <span class="gene">GmIFR</span> </italic> likely plays an important role in responsive to biotic stresses in soybean. Moreover, it has been reported that IFR proteins as the key enzymes can catalyze reductase reactions (Paiva et al., <xref rid="B34" ref-type="bibr"> 1991 </xref> ; Gang et al., <xref rid="B14" ref-type="bibr"> 1999 </xref> ). For instance, IFR can convert 2&#x02032;-hydroxyformononetin of the isoflavonoid substrates to (3R)-vestitone in alfalfa (Paiva et al., <xref rid="B34" ref-type="bibr"> 1991 </xref> ). In Ginkgo, the recombinant GbIRL1 protein could catalyze the formation of the TDDC, IDDDC from DDDC, DDC (Cheng et al., <xref rid="B5" ref-type="bibr"> 2013 </xref> ). In this work, we determined the recombinant <span class="gene">GmIFR</span> protein could catalyze oxidoreductase reaction using enzyme assays. Therefore, it is proven that <span class="gene">GmIFR</span> has <span class="trait">isoflavone</span> reductase activity.
					</p>
				
					<p>&nbsp;&nbsp;
						There have been some reports about the inhibition of glyceollin on several lines of pathogen species (Lyne et al., <xref rid="B28" ref-type="bibr"> 1976 </xref> ; Kim et al., <xref rid="B20" ref-type="bibr"> 2010a </xref> , <xref rid="B19" ref-type="bibr"> 2012 </xref> ; Nwachukwu et al., <xref rid="B32" ref-type="bibr"> 2013 </xref> ). More specifically, the glyceollin can resist to <italic> Phytophthora megasperma </italic> var. <italic> sojae </italic> in soybean (Lyne et al., <xref rid="B28" ref-type="bibr"> 1976 </xref> ; Hahn et al., <xref rid="B17" ref-type="bibr"> 1985 </xref> ; Lygin et al., <xref rid="B26" ref-type="bibr"> 2010 </xref> , <xref rid="B27" ref-type="bibr"> 2013 </xref> ). In addition, Boue and Raina ( <xref rid="B4" ref-type="bibr"> 2003 </xref> ) noted that the fungi <italic> Aspergillus flavus, Aspergillus niger, Aspergillus oryzae </italic> , and <italic> Aspergillus flavus </italic> were all able to induce glyceollin in soybean. Glyceollin revealed a remarkable antimicrobial effect against <italic> Phytophthora capsici </italic> and <italic> Sclerotinia scrotiorum </italic> by Kim et al. ( <xref rid="B20" ref-type="bibr"> 2010a </xref> ). Moreover, It has been reported that IFR was a key enzyme involved in the synthesis of the glyceollins from daidzein which was the essential component of the isoflavones (Graham et al., <xref rid="B15" ref-type="bibr"> 1990 </xref> ; Oliver et al., <xref rid="B33" ref-type="bibr"> 2003 </xref> ). In this work, we detected the <span class="trait">content</span> of daidzein, genistein, glycitein and the relative <span class="trait">content</span> of glyceollins in transgenic soybean seeds and non-transgenic soybean seeds. The results showed that the daidzein <span class="trait">content</span> greatly reduced in transgenic soybean seeds, while levels of genistein and glycitein had little change compared to those of the non-transgenic soybean seeds. The relative <span class="trait">content</span> of glyceollins in the transgenic plants was significantly higher than that of non-transgenic plants. Therefore, we suggested that <italic> <span class="gene">GmIFR</span> </italic> might improve the <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean when overexpression likely by increasing the accumulation of glyceollins. It has been reported that the biosynthesis of glyceollin via the isoflavonoid branch of the phenylpropanoid pathway (Ng et al., <xref rid="B31" ref-type="bibr"> 2011 </xref> ) and glyceollin as a phytoalexin could response to pathogen invasion (Hahn et al., <xref rid="B17" ref-type="bibr"> 1985 </xref> ; Lygin et al., <xref rid="B26" ref-type="bibr"> 2010 </xref> ; Kim et al., <xref rid="B19" ref-type="bibr"> 2012 </xref> ). PAL, 4CL, and <span class="gene">CHS</span> play an important role in the isoflavonoid branch of the phenylpropanoid pathway (Vogt, <xref rid="B53" ref-type="bibr"> 2010 </xref> ; Yi et al., <xref rid="B58" ref-type="bibr"> 2010 </xref> ). Here, we studied the transcript levels change of the three genes ( <italic> <span class="gene">GmPAL,</span> <span class="gene">GmCHS,</span> <span class="gene">Gm4CL</span> </italic> ) after incubation with <italic> P. sojae </italic> in transgenic and non-transgenic soybean plants. We found that these genes were induced after incubation with <italic> P. sojae </italic> and there was up-regulation of the transcript in transgenic soybean plants. We speculated that the three genes ( <italic> <span class="gene">GmPAL,</span> <span class="gene">GmCHS,</span> <span class="gene">Gm4CL</span> </italic> ) might play a cooperation role on the biosynthesis of glyceollin which might improve <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Plants experience a variety of environmental stresses likely leading to the generation of ROS (Sies, <xref rid="B45" ref-type="bibr"> 1991 </xref> ). Although ROS may be essential to maintain homoeostasis, the generation of ROS, within certain boundaries, is harmful at the high concentration (Kim et al., <xref rid="B19" ref-type="bibr"> 2012 </xref> ). However, glyceollins as major isoflavonoid phytoalexin in leguminous plants, showed strong antioxidant activity and ROS scavenging potential when assessed by an <italic> in vitro </italic> model (Kim et al., <xref rid="B20" ref-type="bibr"> 2010a </xref> , <xref rid="B19" ref-type="bibr"> 2012 </xref> ). Moreover, Kim et al. ( <xref rid="B21" ref-type="bibr"> 2010b </xref> ) reported that overexpression of <italic> OsIRL </italic> in transgenic rice plants promotes <span class="trait">resistance</span> to ROS. A possible interpretation is that the IRLs contain a putative NAD (P) domain related to oxidation/reduction properties (Babiychuk et al., <xref rid="B1" ref-type="bibr"> 1995 </xref> ; Petrucco et al., <xref rid="B37" ref-type="bibr"> 1996 </xref> ). The different roles of ROS in plants interactions with special emphasis on fungal and oomycete pathogens have also been reported (Shetty et al., <xref rid="B43" ref-type="bibr"> 2008 </xref> ). For example, <italic> Blumeria graminis </italic> f. sp. <italic> hordei </italic> can induce ROS accumulation in barley (H&#x000fc;ckelhoven and Kogel, <xref rid="B18" ref-type="bibr"> 2003 </xref> ) and <italic> Septoria tritici </italic> can induce ROS accumulation in wheat (Shetty et al., <xref rid="B42" ref-type="bibr"> 2003 </xref> ). Thus, it can be deduced that the plants can improve <span class="trait">resistance</span> to pathogen by scavenge excess ROS. Consistent with these, we detected the relative expression levels of ROS in transgenic soybean plants and non-transgenic plants after incubation with <italic> P. sojae </italic> , and found that the relative expression levels of ROS in transgenic plants were significantly lower than those of control plants. Overexpression of <italic> <span class="gene">GmIFR</span> </italic> in transgenic soybean plants can increase the <span class="trait">content</span> of glyceollins or <span class="gene">GmIFR</span> itself behave as an antioxidant to scavenge ROS, which might improve <span class="trait">resistance</span> to <italic> P. sojae </italic> in soybean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Ameliorative symbiosis of endophyte (Penicillium funiculosum LHL06) under salt stress elevated plant growth of Glycine max L.</h2>
			<h3>PMID:21458283  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Khan AL(1), Hamayun M, Kim YH, Kang SM, Lee IJ.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments were conducted to investigate the role of a newly isolated endophytic fungus <span class="gene">GMC-2A</span> on physiology of host plant (Glycine max. L cv. Hwangkeum-kong) growing under salinity stress. <span class="gene">GMC-2A</span> was identified as a new strain of Penicillium funiculosum on the basis of sequence homology and phylogenetic analysis of D1/D2 regions of 28S rDNA. Preliminary screening experiment showed that the culture filtrate (CF) of <span class="gene">GMC-2A</span> promoted the growth of Waito-C, a dwarf gibberellin (GA) biosynthesis mutant rice cultivar. Analysis of fungal CF revealed the presence of GAs (GA₁ 1.53 ng/ml; GA₄ 9.34 ng/ml; GA₈ 1.21 ng/ml; GA₉ 37.87 ng/ml) and indole acetic acid (14.85 μg/ml). <span class="gene">GMC-2A</span> also showed high phosphate solubilization of tricalcium phosphate. Besides that, <span class="gene">GMC-2A</span> application enhanced soybean <span class="trait">seed</span> germination as compared to control. Under salinity stress (70 and 140 mM), <span class="gene">GMC-2A</span> significantly promoted the soybean growth attributes (shoot length, shoot fresh/dry biomass, chlorophyll <span class="trait">content,</span> photosynthesis rate and leaf area) in comparison to control treatments. We also observed low endogenous abscisic acid and elevated jasmonic acid contents in <span class="gene">GMC-2A</span> treated plants under salt stress. <span class="gene">GMC-2A</span> treatment significantly enhanced levels of isoflavones (34.22% and 75.37%) under salinity stress as compared to control. In conclusion, P. funiculosum LHL06 has significantly ameliorated the adverse effects of salinity induced abiotic stress, and re-programmed soybean to higher growth and <span class="trait">isoflavone</span> biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome analysis reveals a critical role of <span class="gene">CHS7</span> and CHS8 genes for isoflavonoid synthesis in soybean seeds.</h2>
			<h3>PMID:17098860  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dhaubhadel S(1), Gijzen M, Moy P, Farhangkhoee M.
				<br/> 
				年份：2007 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We have used cDNA microarray analysis to examine changes in gene expression during embryo development in soybean (Glycine max) and to compare gene expression profiles of two soybean cultivars that differ in <span class="trait">seed</span> isoflavonoid <span class="trait">content.</span> The analysis identified 5,910 genes that were differentially expressed in both soybean cultivars grown at two different locations for two consecutive years in one of the five different stages of embryo development. An ANOVA analysis with P value &lt; 0.05 and &lt; 0.01 indicated that gene expression changes due to environmental factors are greater than those due to cultivar differences. Most changes in gene expression occurred at the stages when the embryos were at 30 or 70 d after pollination. A significantly larger fraction of genes (48.5%) was expressed throughout the development and showed little or no change in expression. Transcript accumulation for genes related to the biosynthesis of storage components in soybean embryos showed several unique temporal expressions. Expression patterns of several genes involved in isoflavonoid biosynthesis, such as Phenylalanine Ammonia-Lyase, Chalcone Synthase (CHS) 7, CHS8, and <span class="trait">Isoflavone</span> Synthase2, were higher at 70 d after pollination in both the cultivars. Thus, expression of these genes coincides with the onset of accumulation of isoflavonoids in the embryos. A comparative analysis of genes involved in isoflavonoid biosynthesis in RCAT Angora (high <span class="trait">seed</span> isoflavonoid cultivar) and Harovinton (low <span class="trait">seed</span> isoflavonoid cultivar) revealed that <span class="gene">CHS7</span> and CHS8 were expressed at significantly greater level in RCAT Angora than in Harovinton. Our study provides a detailed transcriptome profiling of soybean embryos during development and indicates that differences in the level of <span class="trait">seed</span> isoflavonoids between these two cultivars could be as a result of differential expression of <span class="gene">CHS7</span> and CHS8 during late stages of <span class="trait">seed</span> development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:LC/UV/ESI-MS analysis of isoflavones in Edamame and Tofu soybeans.</h2>
			<h3>PMID:15137811  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu Q(1), Wang M, Sciarappa WJ, Simon JE.
				<br/> 
				年份：2004 May 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High-performance liquid chromatography coupled with ultraviolet and electrospray ionization mass spectrometry (HPLC/UV/ESI-MSD) was applied to the study of isoflavones in both Edamame and Tofu soy varieties, from which the immature fresh soybeans or the mature soybean seeds are consumed, respectively. Positive atmospheric pressure interface (API) <span class="gene">MS</span> and MS/MS were used to provide molecular mass information and led to the identification of a total 16 isoflavones, including three aglycones, three glycosides, two glycoside acetates, and eight glycoside malonates. The major isoflavones in soybean seeds were daidzein and genistein glycoside and their malonate conjugates. Trace levels of daidzein and genistein acetyl glycosides were found only in the mature dry soybean seeds. To facilitate quantitative analysis, acid hydrolysis during extraction of soy samples was selected to convert the various phytoestrogen conjugates into their respective <span class="trait">isoflavone</span> aglycones, allowing accurate quantitation of total phytoestrogens as aglycones. On the basis of HPLC combined with UV and <span class="gene">MS</span> detection, all three targeted soy <span class="trait">isoflavone</span> aglycones, daidzein, genistein and glycitein in hydrolyzed extracts were successfully quantified within 25 min with formononetin used as the internal standard. The standard curves of UV detection were fitted in the range of 14.16-29000 ng/mL for daidzein, 15.38-31500 ng/mL for genistein, and 11.72-24000 ng/mL for glycitein. For <span class="gene">MS</span> detection, the standard curves were established in the range of 3.54-1812.5 ng/mL for daidzein, 3.85-1968.75 ng/mL for genistein, and 2.93-1500 ng/mL for glycitein. Good linearities (r(2) &gt; 0.999 for UV and r(2) &gt; 0.99 for MS) for standard curves were achieved for each <span class="trait">isoflavone.</span> The accuracy and precision (RSD) were within 10% for UV detection and 15% for <span class="gene">MS</span> detection (n = 10). Using this method, the phytoestrogen levels of total <span class="trait">isoflavone</span> aglycones from 30 soybean <span class="trait">seed</span> varieties were then evaluated for confirmation of the technique. Total isoflavones ranged across the varieties from 0.02 to 0.12% in the Edamame varieties, which are harvested while the seeds are still immature, and from 0.16 to 0.25% in Tofu varieties, harvested when the seeds are physiologically mature. While the literature has focused on the <span class="trait">isoflavone</span> <span class="trait">content</span> of soy products and processing soy, this report provides a reliable analytical technique for screening of authenticated fresh immature Edamame soybeans and Tofu soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of <span class="trait">Isoflavone</span> Biosynthesis by miRNAs in Two Contrasting Soybean Genotypes at Different <span class="trait">Seed</span> Developmental Stages.</h2>
			<h3>PMID:28450878</h3>
				

				
			<p>
				作者：Gupta OP(1), Nigam D(2), Dahuja A(1), Kumar S(2), Vinutha T(1), Sachdev A(1), Praveen S(1).
				<br/> 
				年份：2017 Apr 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Owing to the presence of nutritionally important, health-promoting bioactive compounds, especially isoflavones, soybean has acquired the status of a functional food. miRNAs are tiny riboregulator of gene expression by either decreasing and/or increasing the expression of their corresponding target genes. Despite several works on identification and functional characterization of plant miRNAs, the role of miRNAs in the regulation of isoflavones metabolism is still a virgin field. In the present study, we identified a total of 31 new miRNAs along with their 245 putative target genes from soybean seed-specific ESTs using computational approach. The Kyoto Encyclopedia of Genes and Genomes pathway analyses indicated that miRNA putatively regulates metabolism and genetic information processing. Out of that, a total of 5 miRNAs <span class="gene">(Gma-miRNA12,</span> <span class="gene">Gma-miRNA24,</span> <span class="gene">Gma-miRNA26,</span> <span class="gene">Gma-miRNA28,</span> and <span class="gene">Gma-miRNA29)</span> were predicted and validated for their probable role during <span class="trait">isoflavone</span> biosynthesis. We also validated their five target genes using RA-PCR, which is as good as 5'RLM-RACE. Temporal regulation [35 days after flowering, 45, 55, and 65 DAF] of miRNAs and their targets showed differential expression schema. Differential expression of <span class="gene">Gma-miR26</span> and <span class="gene">Gma-miRNA28</span> along with their corresponding target genes <span class="gene">(Glyma.10G197900</span> and <span class="gene">Glyma.09G127200)</span> showed a direct relationship with the total <span class="trait">isoflavone</span> <span class="trait">content.</span> Therefore, understanding the miRNA-based genetic regulation of <span class="trait">isoflavone</span> pathway would assist in selection and manipulation to get high-performing soybean genotypes with better <span class="trait">isoflavone</span> <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genistein production in rice <span class="trait">seed</span> via transformation with soybean <span class="gene">IFS</span> genes.</h2>
			<h3>PMID:24467893  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sohn SI(1), Kim YH(2), Kim SL(2), Lee JY(3), Oh YJ(4), Chung JH(5), Lee KR(3).
				<br/> 
				年份：2014 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To produce genistein in rice, the <span class="trait">isoflavone</span> synthase (IFS) genes, SpdIFS1 and SpdIFS2 were cloned from the Korean soybean cultivar, Sinpaldalkong II as it has a higher genistein <span class="trait">content</span> than other soybean varieties. SpdIFS1 and SpdIFS2 show a 99.6% and 98.2% identity at the nucleotide level and 99.4% and 97.9% identity at the amino acid level, respectively, with <span class="gene">IFS1</span> and <span class="gene">IFS2</span> from soybean (GenBank accession Nos. AF195798 and AF195819). Plant expression vectors were constructed harboring SpdIFS1 or SpdIFS2 under the control of a rice globulin promoter that directs <span class="trait">seed</span> specific expression, and used to transform two rice varieties, Heugnam, a black rice, and Nakdong, a normal rice cultivar without anthocyanin pigment. Because naringenin, the substrate of SpdIFS1 and SpdIFS2, is on the anthocyanin biosynthesis pathway, the relative production rate of genistein was compared between SpdIFS-expressing transgenic Heugnam and Nakdong. Southern blot analysis of eight of the resulting transgenic rice plants revealed that the T0 plants had one to three copies of the SpdIFS1 or SpdIFS2 gene. The highest level of genistein <span class="trait">content</span> found in rice seeds was 103 μg/g. These levels were about 30-fold higher in our transgenic rice lines than the genistein aglycon <span class="trait">content</span> of a non-leguminous IFS-expressing transgenic tobacco petal, equaling about 12% of total genistein <span class="trait">content</span> of Sinpaldalkong II. There were no significant differences found between the genistein <span class="trait">content</span> in Heugnam and Nakdong transgenic rice plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physicochemical and sensory properties of soy bread made with germinated, steamed, and roasted soy flour.</h2>
			<h3>PMID:23768388  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shin DJ(1), Kim W, Kim Y.
				<br/> 
				年份：2013 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For the development of healthful gluten-free soy bread acceptable to consumers, we evaluated the effects of various processing procedures for soy flour on bread quality, in terms of beany flavour and texture. We pretreated soy flour by both non-heating (raw:NS and germinated:GS) and heating (steamed:SS and roasted:RS) methods. In addition, to improve the loaf volume, we added 1% hydroxypropyl-methylcellulose (HPMC) to RS flour. Lipoxygenase activity was retained in the non-heat-treated flours (279 U/g for NS and 255 U/g for GS), but was significantly reduced in the heat-treated flours (106U/g for <span class="gene">SS</span> and 69 U/g for RS). Moreover, heat-treated flour had higher <span class="trait">isoflavone</span> and ferric reducing antioxidant power than had non-heat-treated flour. However, RS flour had the lowest moisture <span class="trait">content</span> and lowest L value. The <span class="gene">GS</span> bread had the highest specific loaf volume (3.53 cm(3)/g), followed by NS (2.96 cm(3)/g), RS (2.25c m(3)/g), and <span class="gene">SS</span> (1.81 cm(3)/g) bread. <span class="gene">GS</span> bread had the lowest hardness (1.53N), followed by NS (1.65 N), RS (2.00 N), and <span class="gene">SS</span> (3.75 N) bread. The addition of 1% HPMC to RS increased the loaf volume (2.44 cm(3)/g), but decreased the bread's hardness (1.80N). As to the sensory properties, the bread with heat-treated flour was perceived to have a less beany odour and taste than was the bread with non-heat-treated flour. However, the latter had a better appearance than the former. These results indicated that soy flour pretreatment could enhance the loaf volume and reduce the beany flavour of whole soy bread.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of flavonoid 6-hydroxylase candidate genes in normal and mutant soybean genotypes for glycitein <span class="trait">content.</span></h2>
			<h3>PMID:23695494  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Artigot MP(1), Baes M, Daydé J, Berger M.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean seeds accumulate large amounts of isoflavones (genistein, daidzein and glycitein), secondary metabolites known for their phytoestrogenic activities. <span class="trait">Isoflavone</span> composition depends on the <span class="trait">seed</span> part and glycitein is almost found exclusively in hypocotyls. Moreover, two major phenotypes are encountered in soybean cultivars, with either low (35 %) or high (55 %) levels of glycitein in their hypocotyls. This trait was under a quasi-mendelian heredity, implicating at most one or two genes. A <span class="gene">CYP71D9</span> cDNA displaying a flavonoid 6-hydroxylase (F6H) activity had previously been isolated from elicitor-induced soybean (Glycine max L.) cells. This enzyme allows the synthesis of the glycitein flavanone intermediate (6,7,4'- trihydroxyflavanone) by catalyzing the A-ring hydroxylation of liquiritigenin. In this study, the <span class="gene">CYP71D9</span> gene (F6H1) and two other candidates (F6H2 and F6H3) were studied using contrasted soybean cultivars for glycitein <span class="trait">content</span> (0, 35, 55 and 80 %). Their expression was observed in chitosan elicited leaves. They encode <span class="gene">P450</span> proteins of 496, 469 and 481 amino acids respectively and were expressed in leaves with or without elicitation. The expression patterns of these three genes were performed in cotyledons and hypocotyls at different developmental stages. <span class="gene">F6H1</span> and <span class="gene">F6H2</span> were not expressed in the developing <span class="trait">seed.</span> <span class="gene">F6H3</span> was only expressed in hypocotyls. Its expression levels did not correlate with hypocotyls glycitein <span class="trait">content,</span> but it was not expressed in the null mutant for glycitein. Thus, this <span class="gene">F6H3</span> gene is a good potential candidate for glycitein biosynthesis in soybean <span class="trait">seed.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Differential expression of <span class="trait">isoflavone</span> biosynthetic genes in soybean during water deficits.</h2>
			<h3>PMID:20430761  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gutierrez-Gonzalez JJ(1), Guttikonda SK, Tran LS, Aldrich DL, Zhong R, Yu O, Nguyen HT, Sleper DA.
				<br/> 
				年份：2010 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Numerous environmental factors influence <span class="trait">isoflavone</span> accumulation and have long hampered their genetic dissection. Temperature and water regimes are two of the most significant abiotic factors. However, while the effects of temperature have been widely studied, little is known about how water scarcity might affect <span class="trait">isoflavone</span> concentration in seeds. Studies have shown that accumulation of isoflavones is promoted by well-watered conditions, but the molecular basis remains elusive. The length and severity of the water stress required to induce changes are also still unknown. In the present work, several intensities of water stress were evaluated at various critical stages for soybean [Glycine max (L.) Merr.] <span class="trait">seed</span> development, in both field and controlled environments. The results suggested that only long-term progressive <span class="trait">drought,</span> spanning most of the <span class="trait">seed</span> developmental stages, significantly decreased <span class="trait">isoflavone</span> <span class="trait">content</span> in seeds. The reduction is proportional to the intensity of the stress and appears to occur in a genotype-dependent manner. However, regardless of water regime, <span class="trait">isoflavone</span> compounds were mainly accumulated in the later <span class="trait">seed</span> developmental stages. Transcripts of the most important genes for <span class="trait">isoflavone</span> biosynthesis were also quantified from samples collected at key <span class="trait">seed</span> developmental stages under well-watered and long-term water deficit conditions. Expression of CHS7, CHS8 and <span class="gene">IFS2</span> correlated with <span class="trait">isoflavone</span> accumulation under well-watered conditions. Interestingly, we found that the two <span class="trait">isoflavone</span> synthase genes in soybean (IFS1 and IFS2) showed different patterns of expression. The abundance of <span class="gene">IFS1</span> transcripts was maintained at a constant rate, whereas <span class="gene">IFS2</span> was down-regulated and highly correlated with <span class="trait">isoflavone</span> accumulation under both water deficit and well-watered conditions, suggesting <span class="gene">IFS2</span> as a main contributor to <span class="trait">isoflavone</span> diminution under <span class="trait">drought.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of two low phytic acid mutations on <span class="trait">seed</span> quality and nutritional traits in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:19323582  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan FJ(1), Zhu DH, Deng B, Fu XJ, Dong DK, Zhu SL, Li BQ, Shu QY.
				<br/> 
				年份：2009 May 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reduction of phytic acid in soybean seeds has the potential to improve the nutritional value of soybean meal and lessen phosphorus pollution in large scale animal farming. The objective of this study was to assess the effect of two new low phytic acid (LPA) mutations on <span class="trait">seed</span> quality and nutritional traits. Multilocation/season comparative analyses showed that the two mutations did not affect the concentration of crude protein, any of the individual amino acids, crude oil, and individual saturated fatty acids. Among other traits, <span class="gene">Gm-lpa-TW75-1</span> had consistently higher sucrose contents (+47.4-86.1%) and lower raffinose contents (-74.2 to -84.3%) than those of wild type (WT) parent Taiwan 75; <span class="gene">Gm-lpa-ZC-2</span> had higher total <span class="trait">isoflavone</span> contents (3038.8-4305.4 microg/g) than its parent Zhechun # 3 (1583.6-2644.9 microg/g) in all environments. Further tests of homozygous F(3) progenies of the cross <span class="gene">Gm-lpa-ZC-2</span> x Wuxing # 4 (WT variety) showed that LPA lines had a mean <span class="trait">content</span> of total <span class="trait">isoflavone</span> significantly higher than WT lines. This study demonstrated that two LPA mutant genes have no negative effects on <span class="trait">seed</span> quality and nutritional traits; they instead have the potential to improve a few other properties. Therefore, these two mutant genes are valuable genetic resources for breeding high quality soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isoflavonoid biosynthesis and accumulation in developing soybean seeds.</h2>
			<h3>PMID:15082922  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dhaubhadel S(1), McGarvey BD, Williams R, Gijzen M.
				<br/> 
				年份：2003 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Isoflavonoids are biologically active natural products that accumulate in soybean seeds during development. The amount of isoflavonoids present in soybean <span class="trait">seed</span> is variable, depending on genetic and environmental factors that are not fully understood. Experiments were conducted to determine whether isoflavonoids are synthesized within <span class="trait">seed</span> tissues during development, or made in other plant organs and transported to the seeds where they accumulate. An analysis of isoflavonoids by HPLC detected the compounds in all organs of soybean plant, but the amount of isoflavonoids present varied depending on the tissue and developmental stage. The greatest concentrations were found in mature seeds and leaves. The 2-hydroxyisoflavanone synthase genes <span class="gene">IFS1</span> and <span class="gene">IFS2</span> were studied to determine their pattern of expression in different tissues and developmental stages. The highest level of expression of <span class="gene">IFS1</span> was observed in the root and <span class="trait">seed</span> coat, while <span class="gene">IFS2</span> was most highly expressed in embryos and pods, and in elicitor-treated or pathogen-challenged tissues. Incorporation of radiolabel into isoflavonoids was observed when developing embryos and other plant organs were fed with [(14)C]phenylalanine. Embryos excised from developing soybean seeds also accumulated isoflavonoids from a synthetic medium. A maternal effect on <span class="trait">seed</span> isoflavonoid <span class="trait">content</span> was noted in reciprocal crosses between soybean cultivars that differ in <span class="trait">seed</span> isoflavonoids. From these results, we propose that developing soybean embryos have an ability to synthesize isoflavonoids de novo, but that transport from maternal tissues may in part contribute to the accumulation of these natural products in the <span class="trait">seed.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isoflavone composition and antioxidant capacity of modified-lipoxygenase soybeans grown in Maryland.</h2>
			<h3>PMID:22098380  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Whent M(1), Lv J, Luthria DL, Kenworthy W, Yu LL.
				<br/> 
				年份：2011 Dec 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Maryland-grown soybean lines modified for low lipoxygenase-1 (LOX-1) <span class="trait">content</span> and a traditional nonmodified cultivar were analyzed for fatty acid composition, total phenolic <span class="trait">content</span> (TPC), <span class="trait">isoflavone</span> composition, relative DPPH• scavenging capacity (RDSC), and hydroxyl radical scavenging capacity (HOSC). Soybean lines included black, brown, and yellow soybeans. TPC of all soybean lines ranged from 2.84 to 4.74 mg gallic acid equiv (GAE)/g flour. Total isoflavones were between 2.78 and 8.66 μmol/g flour. RDSC of all lines was between 0.48 and 14.62 μmol Trolox equiv (TE)/g flour, and HOSC ranged from 53.57 to 135.52 μmol TE/g flour. Some modified-LOX genotypes demonstrated antioxidant capacity and/or <span class="trait">isoflavone</span> <span class="trait">content</span> similar to or higher than those of the nonmodified cultivar (P &lt; 0.05). Black soybeans demonstrated higher TPC and RDSC than most yellow soybean lines, although they did not have higher <span class="trait">isoflavone</span> <span class="trait">content.</span> The results demonstrate that modification of the <span class="gene">LOX</span> trait did not necessarily alter the antioxidant capacity or chemical composition of the experimental soybean lines when compared with a nonmodified cultivar. These soybean lines may be studied further for nutraceutical properties and use in functional foods.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genomic regions that underlie soybean <span class="trait">seed</span> <span class="trait">isoflavone</span> <span class="trait">content</span></h2>
			<h3>PMID:12488625</h3>
				
					<h3>PMCID:79679</h3>
					<h4>期刊:Journal of Biomedicine and Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Meksem </surname>  <given-names> K. </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Njiti </surname>  <given-names> V. N </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Banz </surname>  <given-names> W. J </given-names>  </name>  <xref rid="I2" ref-type="aff"> 2 </xref>  <email> Tw3a@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Iqbal </surname>  <given-names> M. J </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Kassem </surname>  <given-names> My. M </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Hyten </surname>  <given-names> D. L </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Yuang </surname>  <given-names> J. </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Winters </surname>  <given-names> T. A </given-names>  </name>  <xref rid="I2" ref-type="aff"> 2 </xref>  <email> Tw3a@siu.edu </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Lightfoot </surname>  <given-names> D. A </given-names>  </name>  <xref rid="I1" ref-type="aff"> 1 </xref>  <email> Ga4082@siu.edu </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 400 microsatellite markers were tested and 133 were found to be polymorphic within the E &#x000d7; F population. The present report summarizes the data from the 133 SSR loci. One hundred and seven markers were mapped to 18 linkage groups encompassing about 2823 cM with an average distance of 26 cM between markers plus 26 markers that were not linked but map to known locations in the soybean genome (see [ <xref rid="B15" ref-type="bibr"> 15 </xref> ]). The recombination distances and orders of markers in linkage groups agree with values reported for the soybean genome of about 3000 cM within 20 linkage groups (see [ <xref rid="B12" ref-type="bibr"> 12 </xref> , <xref rid="B33" ref-type="bibr"> 33 </xref> , <xref rid="B15" ref-type="bibr"> 15 </xref> , <xref rid="B34" ref-type="bibr"> 34 </xref> , <xref rid="B29" ref-type="bibr"> 29 </xref> ]). There was an average of six to seven markers per linkage group. The actual number of markers per linkage group ranged from three on linkage group A1 to 16 on linkage group G. The large number of markers on linkage group G was the result of marker saturation around the <italic> <span class="gene">Rhg1</span> </italic> gene for the soybean cyst nematode ( <italic> H. glycines </italic> 1) <span class="trait">resistance.</span> The F5:10 lines were heterogeneous at 12&#x00025; of loci scored by codominant markers.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Definition of Soybean Genomic Regions That Control <span class="trait">Seed</span> Phytoestrogen Amounts</h2>
			<h3>PMID:15123888</h3>
				
					<h3>PMCID:545653</h3>
					<h4>期刊:Journal of Biomedicine and Biotechnology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="no">  <name>  <surname> Kassem </surname>  <given-names> My A. </given-names>  </name>  <xref ref-type="aff" rid="a1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Meksem </surname>  <given-names> K. </given-names>  </name>  <xref ref-type="aff" rid="a1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Iqbal </surname>  <given-names> M. J. </given-names>  </name>  <xref ref-type="aff" rid="a1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Njiti </surname>  <given-names> V. N. </given-names>  </name>  <xref ref-type="aff" rid="a2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Banz </surname>  <given-names> W. J. </given-names>  </name>  <xref ref-type="aff" rid="a3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Winters </surname>  <given-names> T. A. </given-names>  </name>  <xref ref-type="aff" rid="a3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" corresp="no">  <name>  <surname> Wood </surname>  <given-names> A. </given-names>  </name>  <xref ref-type="aff" rid="a4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lightfoot </surname>  <given-names> D. A. </given-names>  </name>  <xref ref-type="aff" rid="a1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 11 </month>  <year> 2003 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A partial diagram of the phenylpropanoid pathway showing intermediates and enzymes involved in <span class="trait">isoflavone</span> synthesis, as well as some branched pathways. The enzymes in bold are encoded by genes expressed as transgenes in Yu et al [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. Genes encoding the underlined enzymes have been shown to be activated in maize by <span class="gene">C1</span> and R. Dotted arrows represent multiple steps. Enzymes are indicated in italics (adapted from [ <xref ref-type="bibr" rid="B24"> 24 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> ]). Glycitein (a glyceollin) is hypothesized to be made from daidzein in a multistep process (hydroxylation, oxidation, and acetylation) see <xref ref-type="fig" rid="F4"> Figure 4 </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>