<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed oil content</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed oil content</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Soybean (Glycine max) WRINKLED1 transcription factor, <span class="gene">GmWRI1a,</span> positively regulates <span class="trait">seed</span> <span class="trait">oil</span> accumulation.</h2>
			<h3>PMID:29138932</h3>
				

				
			<p>
				作者：Chen L(1), Zheng Y(2), Dong Z(3), Meng F(1), Sun X(1), Fan X(1), Zhang Y(1), Wang M(1), Wang S(4).
				<br/> 
				年份：2017 Nov 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is the world's most important leguminous crop producing high-quality protein and <span class="trait">oil.</span> Elevating <span class="trait">oil</span> accumulation in soybean <span class="trait">seed</span> is always many researchers' goal. WRINKLED1 (WRI1) encodes a transcription factor of the APETALA2/ethylene responsive element-binding protein (AP2/EREBP) family that plays important roles during plant <span class="trait">seed</span> <span class="trait">oil</span> accumulation. In this study, we isolated and characterized three distinct orthologues of <span class="gene">WRI1</span> in soybean (Glycine max) that display different organ-specific expression patterns, among which <span class="gene">GmWRI1a</span> was highly expressed in maturing soybean <span class="trait">seed.</span> Electrophoretic mobility shift assays and yeast one-hybrid experiments demonstrated that the <span class="gene">GmWRI1a</span> protein was capable of binding to AW-box, a conserved sequence in the proximal upstream regions of many genes involved in various steps of <span class="trait">oil</span> biosynthesis. Transgenic soybean seeds overexpressing <span class="gene">GmWRI1a</span> under the control of the seed-specific napin promoter showed the increased total <span class="trait">oil</span> and fatty acid <span class="trait">content</span> and the changed fatty acid composition. Furthermore, basing on the activated expressions in transgenic soybean seeds and existence of AW-box element in the promoter regions, direct downstream genes of <span class="gene">GmWRI1a</span> were identified, and their products were responsible for fatty acid production, elongation, desaturation and export from plastid. We conclude that <span class="gene">GmWRI1a</span> transcription factor can positively regulate <span class="trait">oil</span> accumulation in soybean <span class="trait">seed</span> by a complex gene expression network related to fatty acid biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Silencing of Soybean Raffinose Synthase Gene Reduced Raffinose Family Oligosaccharides and Increased True Metabolizable Energy of Poultry Feed.</h2>
			<h3>PMID:28559898</h3>
				

				
			<p>
				作者：Valentine MF(1), De Tar JR(1), Mookkan M(1), Firman JD(2), Zhang ZJ(1)(2).
				<br/> 
				年份：2017 May 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is the number one <span class="trait">oil</span> and protein crop in the United States, but the <span class="trait">seed</span> contains several anti-nutritional factors that are toxic to both humans and livestock. RNA interference technology has become an increasingly popular technique in gene silencing because it allows for both temporal and spatial targeting of specific genes. The objective of this research is to use RNA-mediated gene silencing to down-regulate the soybean gene raffinose synthase 2 (RS2), to reduce total raffinose <span class="trait">content</span> in mature <span class="trait">seed.</span> Raffinose is a trisaccharide that is indigestible to humans and monogastric animals, and as monogastric animals are the largest consumers of soy products, reducing raffinose would improve the nutritional quality of soybean. An RNAi construct targeting <span class="gene">RS2</span> was designed, cloned, and transformed to the soybean genome via Agrobacterium-mediated transformation. Resulting plants were analyzed for the presence and number of copies of the transgene by PCR and Southern blot. The efficiency of mRNA silencing was confirmed by real-time quantitative PCR. Total raffinose <span class="trait">content</span> was determined by HPLC analysis. Transgenic plant lines were recovered that exhibited dramatically reduced levels of raffinose in mature <span class="trait">seed,</span> and these lines were further analyzed for other phenotypes such as development and <span class="trait">yield.</span> Additionally, a precision-fed rooster assay was conducted to measure the true metabolizable energy (TME) in full-fat soybean meal made from the wild-type or transgenic low-raffinose soybean lines. Transgenic low-raffinose soy had a measured TME of 2,703 kcal/kg, an increase as compared with 2,411 kcal/kg for wild-type. As low digestible energy is a major limiting factor in the percent of soybean meal that can be used in poultry diets, these results may substantiate the use of higher concentrations of low-raffinose, full-fat soy in formulated livestock diets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the <span class="gene">FAD2</span> Gene Family in Soybean Reveals the Limitations of Gel-Based TILLING in Genes with High Copy Number.</h2>
			<h3>PMID:28348573</h3>
				

				
			<p>
				作者：Lakhssassi N(1), Zhou Z(1), Liu S(1), Colantonio V(1), AbuGhazaleh A(2), Meksem K(1).
				<br/> 
				年份：2017 Mar 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> <span class="trait">oil</span> typically contains 18-20% oleic acid. Increasing the <span class="trait">content</span> of oleic acid is beneficial for health and biodiesel production. Mutations in <span class="gene">FAD2-1</span> genes have been reported to increase <span class="trait">seed</span> oleic acid <span class="trait">content.</span> A subset of 1,037 mutant families from a mutagenized soybean cultivar (cv.) Forrest population was screened using reverse genetics (TILLING) to identify mutations within <span class="gene">FAD2</span> genes. Although no fad2 mutants were identified using gel-based TILLING, four fad2-1A and one fad2-1B mutants were identified to have high <span class="trait">seed</span> oleic acid <span class="trait">content</span> using forward genetic screening and subsequent target sequencing. TILLING has been successfully used as a non-transgenic reverse genetic approach to identify mutations in genes controlling important agronomic traits. However, this technique presents limitations in traits such as <span class="trait">oil</span> composition due to gene copy number and similarities within the soybean genome. In soybean, <span class="gene">FAD2</span> are present as two copies, <span class="gene">FAD2-1</span> and FAD2-2. Two <span class="gene">FAD2-1</span> members: <span class="gene">FAD2-1A</span> and FAD2-1B; and three <span class="gene">FAD2-2</span> members: FAD2-2A, FAD2-2B, and FAD2-2C have been reported. Syntenic, phylogenetic, and in silico analysis revealed two additional members constituting the <span class="gene">FAD2</span> gene family: <span class="gene">GmFAD2-2D</span> and <span class="gene">GmFAD2-2E,</span> located on chromosomes 09 and 15, respectively. They are presumed to have diverged from other <span class="gene">FAD2-2</span> members localized on chromosomes 19 <span class="gene">(GmFAD2-2A</span> and <span class="gene">GmFAD2-2B)</span> and 03 <span class="gene">(GmFAD2-2C).</span> This work discusses alternative solutions to the limitations of gel-based TILLING in functional genomics due to high copy number and multiple paralogs of the <span class="gene">FAD2</span> gene family in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plasticity and innovation of regulatory mechanisms underlying <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> mediated by duplicated genes in the palaeopolyploid soybean.</h2>
			<h3>PMID:28295817  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang D(1), Zhao M(1), Li S(2), Sun L(1), Wang W(1), Cai C(2), Dierking EC(3), Ma J(1)(2).
				<br/> 
				年份：2017 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many plants have undergone whole genome duplication (WGD). However, how regulatory networks underlying a particular trait are reshaped in polyploids has not been experimentally investigated. Here we show that the regulatory pathways modulating <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content,</span> which involve WRINKLED1 (WRI1), LEAFY COTYLEDON1 (LEC1), and LEC2 in Arabidopsis, have been modified in the palaeopolyploid soybean. Such modifications include functional reduction of <span class="gene">GmWRI1b</span> of the <span class="gene">GmWRI1a/GmWRI1b</span> homoeologous pair relevant to WRI1, complementary non-allelic dosage effects of the <span class="gene">GmLEC1a/GmLEC1b</span> homoeologous pair relevant to LEC1, pseudogenization of the singleton <span class="gene">GmLEC2</span> relevant to LEC2, and the rise of the LEC2-like function of <span class="gene">GmABI3b,</span> contrasting to its homoeolog <span class="gene">GmABI3a,</span> which maintains the ABSCISIC ACID INSENSITIVE 3 (ABI3)-like function in modulating <span class="trait">seed</span> maturation and dormancy. The function of <span class="gene">GmABI3b</span> in modulating <span class="trait">seed</span> <span class="trait">oil</span> biosynthesis was fulfilled by direct binding to a RY (CATGCA) cis-regulatory element in the <span class="gene">GmWRI1a</span> promoter, which was absent in the <span class="gene">GmWRI1b</span> promoter, resulting in reduction of the <span class="gene">GmWRI1b</span> expression. Nevertheless, the three regulators each exhibited similar intensities of purifying selection to their respective duplicates since these pairs were formed by a WGD event that is proposed to have occurred approximately 13 million years ago (mya), suggesting that the differentiation in spatiotemporal expression between the duplicated genes is more likely to be the outcome of neutral variation in regulatory sequences. This study thus exemplifies the plasticity, dynamics, and novelty of regulatory networks mediated by WGD.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of an efficient acyl-CoA: diacylglycerol acyltransferase 1 (DGAT1) gene from the microalga Chlorella ellipsoidea.</h2>
			<h3>PMID:28222675  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo X(1)(2), Fan C(1), Chen Y(1), Wang J(3), Yin W(1), Wang RR(4), Hu Z(5)(6)(7).
				<br/> 
				年份：2017 Feb 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Oil</span> in the form of triacylglycerols (TAGs) is quantitatively the most important storage form of energy for eukaryotic cells. Diacylglycerol acyltransferase (DGAT) is considered the rate-limiting enzyme for TAG accumulation. Chlorella, a unicellular eukaryotic green alga, has attracted much attention as a potential feedstock for renewable energy production. However, the function of <span class="gene">DGAT1</span> in Chlorella has not been reported. RESULTS: A full-length cDNA encoding a putative diacylglycerol acyltransferase 1 (DGAT1, EC 2.3.1.20) was obtained from Chlorella ellipsoidea. The 2,142 bp open reading frame of this cDNA, designated CeDGAT1, encodes a protein of 713 amino acids showing no more than 40% identity with DGAT1s of higher plants. Transcript analysis showed that the expression level of CeDGAT1 markedly increased under nitrogen starvation, which led to significant triacylglycerol (TAG) accumulation. CeDGAT1 activity was confirmed in the yeast quadruple mutant strain H1246 by restoring its ability to produce TAG. Upon expression of CeDGAT1, the total fatty acid <span class="trait">content</span> in wild-type yeast (INVSc1) increased by 142%, significantly higher than that transformed with DGAT1s from higher plants, including even the <span class="trait">oil</span> crop soybean. The over-expression of CeDGAT1 under the NOS promoter in wild-type Arabidopsis thaliana and Brassica napus var. Westar significantly increased the <span class="trait">oil</span> <span class="trait">content</span> by 8-37% and 12-18% and the average 1,000-seed weight by 9-15% and 6-29%, respectively, but did not alter the fatty acid composition of the <span class="trait">seed</span> <span class="trait">oil.</span> The net increase in the 1,000-seed total lipid <span class="trait">content</span> was up to 25-50% in both transgenic Arabidopsis and B. napus. CONCLUSIONS: We identified a gene encoding <span class="gene">DGAT1</span> in C. ellipsoidea and confirmed that it plays an important role in TAG accumulation. This is the first functional analysis of <span class="gene">DGAT1</span> in Chlorella. This information is important for understanding lipid synthesis and accumulation in Chlorella and for genetic engineering to enhance <span class="trait">oil</span> production in microalgae and <span class="trait">oil</span> plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Selection for a Zinc-Finger Protein Contributes to <span class="trait">Seed</span> <span class="trait">Oil</span> Increase during Soybean Domestication.</h2>
			<h3>PMID:28184009  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li QT(1)(2)(3), Lu X(1)(2)(3), Song QX(1)(2)(3), Chen HW(1)(2)(3), Wei W(1)(2)(3), Tao JJ(1)(2)(3), Bian XH(1)(2)(3), Shen M(1)(2)(3), Ma B(1)(2)(3), Zhang WK(1)(2)(3), Bi YD(1)(2)(3), Li W(1)(2)(3), Lai YC(1)(2)(3), Lam SM(1)(2)(3), Shui GH(1)(2)(3), Chen SY(4)(5)(6), Zhang JS(4)(5)(6).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">oil</span> is a momentous agronomical trait of soybean (Glycine max) targeted by domestication in breeding. Although multiple oil-related genes have been uncovered, knowledge of the regulatory mechanism of <span class="trait">seed</span> <span class="trait">oil</span> biosynthesis is currently limited. We demonstrate that the seed-preferred gene <span class="gene">GmZF351,</span> encoding a tandem CCCH zinc finger protein, is selected during domestication. Further analysis shows that <span class="gene">GmZF351</span> facilitates <span class="trait">oil</span> accumulation by directly activating WRINKLED1, BIOTIN CARBOXYL CARRIER PROTEIN2, 3-KETOACYL-ACYL CARRIER PROTEIN SYNTHASE III, DIACYLGLYCEROL O-ACYLTRANSFERASE1, and OLEOSIN2 in transgenic Arabidopsis (Arabidopsis thaliana) seeds. Overexpression of <span class="gene">GmZF351</span> in transgenic soybean also activates lipid biosynthesis genes, thereby accelerating <span class="trait">seed</span> <span class="trait">oil</span> accumulation. The ZF351 haplotype from the cultivated soybean group and the wild soybean (Glycine soja) subgroup III correlates well with high gene expression level, <span class="trait">seed</span> <span class="trait">oil</span> contents and promoter activity, suggesting that selection of <span class="gene">GmZF351</span> expression leads to increased <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> in cultivated soybean. Our study provides novel insights into the regulatory mechanism for <span class="trait">seed</span> <span class="trait">oil</span> accumulation, and the manipulation of <span class="gene">GmZF351</span> may have great potential in the improvement of <span class="trait">oil</span> production in soybean and other related crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">GmDREBL</span> Increases Lipid <span class="trait">Content</span> in Seeds of Transgenic <italic> Arabidopsis </italic></h2>
			<h3>PMID:27694917</h3>
				

				
					<h3>PMCID:5046110</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yu-Qin </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Xiang </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Fei-Yi </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Qing-Tian </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Niu </surname>  <given-names> Su-Ling </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Wan-Ke </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shou-Yi </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jin-Song </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> State Key Laboratory of Plant Genomics, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Chaoyang District </institution> , Beichen West Road, Campus #1, No. 2, Beijing 100101,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> School of Bioengineering &#x00026; Biotechnology, Tianshui Normal University </institution> , Tianshui, Gansu 741000,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 09 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A DREB-type transcription factor gene <span class="gene">GmDREBL</span> has been characterized for its functions in <span class="trait">oil</span> accumulation in seeds. The gene is specifically expressed in soybean seeds. The <span class="gene">GmDREBL</span> is localized in nucleus and has transcriptional activation ability. Overexpression of <span class="gene">GmDREBL</span> increased the fatty acid <span class="trait">content</span> in the seeds of transgenic Arabidopsis plants. <span class="gene">GmDREBL</span> can bind to the promoter region of <span class="gene">WRI1</span> to activate its expression. Several other genes in the fatty acid biosynthesis pathway were also enhanced in the <span class="gene">GmDREBL-transgenic</span> plants. The <span class="gene">GmDREBL</span> can be up-regulated by <span class="gene">GmABI3</span> and <span class="gene">GmABI5.</span> Additionally, overexpression of <span class="gene">GmDREBL</span> significantly promoted <span class="trait">seed</span> size in transgenic plants compared to that of WT plants. Expression of the DREBL is at higher level on the average in cultivated soybeans than that in wild soybeans. The promoter of the DREBL may have been subjected to selection during soybean domestication. Our results demonstrate that <span class="gene">GmDREBL</span> participates in the regulation of fatty acid accumulation by controlling the expression of <span class="gene">WRI1</span> and its downstream genes, and manipulation of the gene may increase the <span class="trait">oil</span> contents in soybean plants. Our study provides novel insights into the function of DREB-type transcription factors in <span class="trait">oil</span> accumulation in addition to their roles in stress response.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A <italic> DREB </italic> -type transcription factor gene <italic> <span class="gene">GmDREBL</span> </italic> has been characterized for its functions in <span class="trait">oil</span> accumulation in seeds. The gene is specifically expressed in soybean seeds. The <span class="gene">GmDREBL</span> is localized in nucleus and has transcriptional activation ability. Overexpression of <italic> <span class="gene">GmDREBL</span> </italic> increased the fatty acid <span class="trait">content</span> in the seeds of transgenic <italic> Arabidopsis </italic> plants. <span class="gene">GmDREBL</span> can bind to the promoter region of <italic> <span class="gene">WRI1</span> </italic> to activate its expression. Several other genes in the fatty acid biosynthesis pathway were also enhanced in the <italic> <span class="gene">GmDREBL</span> </italic> -transgenic plants. The <italic> <span class="gene">GmDREBL</span> </italic> can be up-regulated by <italic> <span class="gene">GmABI3</span> </italic> and <italic> <span class="gene">GmABI5</span> </italic> . Additionally, overexpression of <italic> <span class="gene">GmDREBL</span> </italic> significantly promoted <span class="trait">seed</span> size in transgenic plants compared to that of WT plants. Expression of the <italic> DREBL </italic> is at higher level on the average in cultivated soybeans than that in wild soybeans. The promoter of the <italic> DREBL </italic> may have been subjected to selection during soybean domestication. Our results demonstrate that <italic> <span class="gene">GmDREBL</span> </italic> participates in the regulation of fatty acid accumulation by controlling the expression of <italic> <span class="gene">WRI1</span> </italic> and its downstream genes, and manipulation of the gene may increase the <span class="trait">oil</span> contents in soybean plants. Our study provides novel insights into the function of <italic> DREB </italic> -type transcription factors in <span class="trait">oil</span> accumulation in addition to their roles in stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is an important economic crop and provides <span class="trait">oil</span> and proteins for human and animals. Increasing the FA (fatty acid) contents and improving the <span class="trait">oil</span> quality are closely related to our daily life. So far, numerous efforts have been made to meet the needs of human food and industry production by changing the fatty acid <span class="trait">content</span> in seeds <xref ref-type="bibr" rid="b1"> 1 </xref> . However, the extracted fatty acids from the existing <span class="trait">oil</span> plants are far from enough, and hence traditional breeding methods and transgenic approaches manipulating fatty acid biosynthesis pathway are used to increase <span class="trait">oil</span> <span class="trait">content</span> in soybean. In plants, the pathways for lipid biosynthesis and <span class="trait">oil</span> accumulation had been studied and the genes related to fatty acid biosynthesis have been characterized. There are several key genes in the process of fatty acid biosynthesis. One is <italic> ACCase </italic> encoding acetyl CoA carboxylase in the first key step of fatty acid biosynthesis, and malonyl-CoA is produced <xref ref-type="bibr" rid="b2"> 2 </xref> . The second one is <italic> KASIII </italic> , which encodes 3-ketoacyl-ACP synthase III to catalyze the formation of a 4-carbon product <xref ref-type="bibr" rid="b3"> 3 </xref> <xref ref-type="bibr" rid="b4"> 4 </xref> . The carbon number of fatty acid is increased by two in acyl chain, and elongation of the acyl chain from six to 16 carbon molecules is catalyzed by an enzyme named KAS1 <xref ref-type="bibr" rid="b5"> 5 </xref> . Without KAS1, FA contents would be sharply reduced, and plant growth and development would be strongly affected <xref ref-type="bibr" rid="b6"> 6 </xref> . The genes related to FA biosynthesis such as <italic> Pl-PK&#x003b2;1 (pyruvate kinase </italic> ), <italic> PDHE1 </italic> &#x003b1; ( <italic> pyruvate dehydrogenase <span class="gene">E1</span> alpha subunit </italic> ), <italic> BCCP2 (acetyl-CoA carboxylase </italic> ), <italic> ACP1 (acyl carrier protein </italic> ), and <italic> KAS1 </italic> have similar expression pattern with <italic> <span class="gene">WRI1</span> (WRINKLED1 </italic> ), and the FA biosynthesis-related genes were up-regulated in the <italic> <span class="gene">WRI1</span> </italic> -overexpressing plants <xref ref-type="bibr" rid="b7"> 7 </xref> . <span class="gene">WRI1</span> is an AP2-type transcription factor (TF) with two <span class="gene">AP2</span> DNA-binding domains <xref ref-type="bibr" rid="b8"> 8 </xref> , and it appears to be a master regulator of <italic> FAS </italic> (fatty acid synthesis) genes in expression level. There is a specific sequence motif AW-box in the promoter regions of the <italic> FAS </italic> genes, and <span class="gene">WRI1</span> binds to this motif in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="b9"> 9 </xref> . Overexpression of <italic> <span class="gene">WRI1</span> </italic> enhanced the <span class="trait">oil</span> <span class="trait">content</span> in transgenic <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="b6"> 6 </xref> <xref ref-type="bibr" rid="b8"> 8 </xref> <xref ref-type="bibr" rid="b9"> 9 </xref> <xref ref-type="bibr" rid="b10"> 10 </xref> <xref ref-type="bibr" rid="b11"> 11 </xref> and maize <xref ref-type="bibr" rid="b12"> 12 </xref> <xref ref-type="bibr" rid="b13"> 13 </xref> . In Castor bean, there are <span class="gene">WRI1</span> binding consensus sites in the promoter region of <italic> RcBCCP2 </italic> and <italic> RcKAS1 </italic> , and RcWRI1 possibly binds to these sites to play a pivotal role in fatty acid biosynthesis <xref ref-type="bibr" rid="b14"> 14 </xref> . Overexpression of a single transcription factor gene <italic> WRI </italic> can increase the <span class="trait">seed</span> <span class="trait">oil</span> contents while manipulating a single fatty acid biosynthesis gene had only very limited effect on the <span class="trait">oil</span> <span class="trait">content</span> <xref ref-type="bibr" rid="b15"> 15 </xref> <xref ref-type="bibr" rid="b16"> 16 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription factors can regulate expression of genes involved in a wide range of plant processes and have a cascade amplification effect <xref ref-type="bibr" rid="b8"> 8 </xref> <xref ref-type="bibr" rid="b17"> 17 </xref> . Therefore, transcription factors are the promising targets to improve <span class="trait">oil</span> contents in plants. Several candidate transcription factors involved in fatty acid biosynthesis and accumulation have been characterized, including <italic> <span class="gene">WRI1</span> </italic> <xref ref-type="bibr" rid="b8"> 8 </xref> <xref ref-type="bibr" rid="b18"> 18 </xref> <xref ref-type="bibr" rid="b19"> 19 </xref> and <italic> LEC2 (leafy cotyledon2 </italic> ) <xref ref-type="bibr" rid="b20"> 20 </xref> in <italic> Arabidopsis. <span class="gene">WRI1</span> </italic> is a target of LEC2 <xref ref-type="bibr" rid="b19"> 19 </xref> . The transcription factors regulating fatty acid contents have been identified from soybean in our lab. Two Dof-type (DNA-binding one zinc finger) genes <italic> <span class="gene">GmDof4</span> </italic> and <italic> <span class="gene">GmDof11</span> </italic> were found to increase the <span class="trait">content</span> of total fatty acids in their transgenic <italic> Arabidopsis </italic> seeds by activating the ACCase and ACSL (long-chain-acyl CoA synthetase) genes respectively <xref ref-type="bibr" rid="b21"> 21 </xref> . Through microarray analysis, a MYB-type gene <italic> <span class="gene">GmMYB73</span> </italic> was identified and this gene can suppress expression of <italic> GL2 (GLABRA 2 </italic> ), a negative regulator of <span class="trait">oil</span> accumulations <xref ref-type="bibr" rid="b22"> 22 </xref> . Overexpression of <italic> <span class="gene">GmMYB73</span> </italic> enhanced lipid contents in seeds of transgenic <italic> Arabidopsis </italic> through release of GL2-inhibited <italic> PLD&#x003b1;1 </italic> (phospholipase D) expression <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b23"> 23 </xref> <xref ref-type="bibr" rid="b24"> 24 </xref> . Overexpression of <italic> <span class="gene">GmbZIP123</span> </italic> also enhanced lipid <span class="trait">content</span> and <span class="trait">oil</span> accumulation by regulating two sucrose transporter genes <italic> SUC1 </italic> and <italic> SUC5 </italic> , and three cell-wall invertase genes <italic> cwINV1 </italic> , <italic> cwINV3 </italic> and <italic> cwINV6 </italic> <xref ref-type="bibr" rid="b25"> 25 </xref> . Recently, through RNA-seq analysis, gene co-expression networks have been identified for soybean <span class="trait">seed</span> trait regulation and <italic> <span class="gene">GmNFYA</span> (nuclear transcription factor Y alpha </italic> ) is found to enhance <span class="trait">seed</span> <span class="trait">oil</span> contents in transgenic <italic> Arabidopsis </italic> plants <xref ref-type="bibr" rid="b26"> 26 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, a DREB-type (dehydration-responsive element-binding) transcription factor gene <italic> <span class="gene">GmDREBL,</span> </italic> was cloned and found to increase the <span class="trait">seed</span> lipid <span class="trait">content</span> in the transgenic plants. <span class="gene">GmDREBL</span> directly activates the expression of <italic> <span class="gene">WRI1</span> </italic> to promote fatty acid accumulation. Our study provides a novel viewpoint into the regulation of the fatty acid accumulation in seeds and should add more understanding of the function for the DREB-type transcription factor gene in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Because the RNA-seq-identified transcription factors are roughly correlated with fatty acid accumulation during soybean <span class="trait">seed</span> development <xref ref-type="bibr" rid="b25"> 25 </xref> , we examined whether the <italic> <span class="gene">GmDREBL</span> </italic> alters the fatty acid accumulation in the overexpressing transgenic lines. Among these lines, 15 overexpression lines exhibited significantly higher total fatty acid <span class="trait">content</span> in seeds than vector control (DL-3 and DL-18) and Col-0 ( <xref ref-type="fig" rid="f3"> Fig. 3D </xref> ). The left lines also showed increases in <span class="trait">seed</span> total fatty acid contents on average. This result suggests that <italic> <span class="gene">GmDREBL</span> </italic> has a positive role on total fatty acid accumulation in seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						To further decipher which composition of fatty acid could be regulated by <italic> <span class="gene">GmDREBL</span> </italic> , we selected three homozygous T3 transgenic lines DL-2, DL-4 and DL-25 for further analysis. The profile and <span class="trait">content</span> of fatty acids in the seeds of Col-0 and the three lines were compared. The result showed that overexpression of <italic> <span class="gene">GmDREBL</span> </italic> significantly enhanced the contents of C18:1, C18:2, C18:3 in all the three transgenic lines and the C20:1 <span class="trait">content</span> in DL-4 overexpression lines ( <xref ref-type="fig" rid="f3"> Fig. 3E </xref> ). The profile and <span class="trait">content</span> of fatty acid were also measured in leaves of the transgenic plants and Col-0 plants, and no significant difference was observed between transgenic lines and Col-0 ( <xref ref-type="fig" rid="f3"> Fig. 3F </xref> ). These results indicate that <italic> <span class="gene">GmDREBL</span> </italic> promotes accumulation of fatty acids specifically in seeds, but not in leaves of the transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						We further examined the effects of <italic> <span class="gene">GmDREBL</span> </italic> overexpression on seedling growth and other seed-related traits. We found that there was no difference in rosette size between overexpression lines and Col-0 at different stages of seedling growth ( <xref ref-type="fig" rid="f3"> Fig. 3G </xref> ). The <span class="trait">seed</span> size and the 1000 seed-weight were substantially greater in the overexpressing lines than that in Col-0 ( <xref ref-type="fig" rid="f3"> Fig. 3H&#x02013;J </xref> ). The <span class="gene">GmDREBL-GFP</span> protein can be detected in the <span class="trait">seed</span> integument cells ( <xref ref-type="supplementary-material" rid="S1"> Figure S1A </xref> ). However, the <span class="trait">seed</span> <span class="trait">yield</span> per plant did not change ( <xref ref-type="fig" rid="f3"> Fig. 3K </xref> ). The length of silique in the three transgenic lines was not significantly different from that in Col-0 either ( <xref ref-type="supplementary-material" rid="S1"> Figure S1B,C </xref> ). These results suggest that the <italic> <span class="gene">GmDREBL</span> </italic> increases the <span class="trait">seed</span> size/weight but not the total <span class="trait">seed</span> <span class="trait">yield</span> per plant in transgenic <italic> Arabidopsis </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Arabidopsis </italic> ABI3 and ABI5 are key regulators for fatty acid accumulation <xref ref-type="bibr" rid="b21"> 21 </xref> <xref ref-type="bibr" rid="b27"> 27 </xref> , and ABI3 contains <span class="gene">AP2</span> domain whereas ABI5 belongs to bZIP family. There are putative <italic> cis </italic> -elements in the promoter region of <italic> <span class="gene">GmDREBL</span> </italic> for possible AP2-domain protein and bZIP protein binding. We then determined if <italic> <span class="gene">GmDREBL</span> </italic> has any co-expressions with <italic> <span class="gene">GmABI3</span> <span class="gene">(Glyma08g47240</span> </italic> ) and <italic> <span class="gene">GmABI5</span> <span class="gene">(Glyma10g08370</span> </italic> ) from soybean, whose identity with ABI3 and ABI5 is 71.7% and 81.3%, respectively. The expression pattern of <italic> <span class="gene">GmABI3</span> </italic> , <italic> <span class="gene">GmABI5</span> </italic> and <italic> <span class="gene">GmDREBL</span> </italic> was determined during soybean <span class="trait">seed</span> development by real-time PCR, and these genes have similar expression pattern ( <xref ref-type="fig" rid="f5"> Fig. 5A&#x02013;C </xref> ), consistent with the accumulation trend of fatty acids in our previous report <xref ref-type="bibr" rid="b25"> 25 </xref> . These results imply that <italic> <span class="gene">GmDREBL</span> </italic> may be regulated by <span class="gene">GmABI3</span> and <span class="gene">GmABI5.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Our above study indicates that overexpression of <italic> <span class="gene">GmDREBL</span> </italic> promotes accumulation of fatty acid <span class="trait">content</span> in seeds of transgenic plants ( <xref ref-type="fig" rid="f3"> Fig. 3 </xref> ). Considering that cultivated soybeans usually had much higher <span class="trait">oil</span> contents compared to the wild soybeans, we investigated whether there is any difference in the expression levels of <italic> <span class="gene">GmDREBL</span> </italic> from cultivated soybean and <italic> GsDREBL </italic> from wild type soybean. It can be seen that the average expression level of <italic> <span class="gene">GmDREBL</span> </italic> in cultivated soybeans is significantly higher than that of <italic> GsDREBL </italic> in wild soybeans ( <xref ref-type="fig" rid="f6"> Fig. 6A </xref> ), suggesting that the <italic> DREBL </italic> expression may have been subjected to selection during domestication of soybean. We further determined the relationship between expression levels of <italic> DREBL </italic> and <span class="trait">seed</span> <span class="trait">oil</span> contents in 76 wild and cultivated soybeans, and found that the <span class="trait">seed</span> <span class="trait">oil</span> contents were positively correlated with the <italic> DREBL </italic> gene expressions with a coefficiency of 0.4378 ( <xref ref-type="fig" rid="f6"> Fig. 6B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Since the <italic> DREBL </italic> gene expression is relatively higher in the cultivated soybeans, and the <span class="trait">oil</span> contents are roughly correlated with the <italic> DREBL </italic> gene expression levels among all the soybeans tested, we determined the sequences of <italic> DREBL </italic> promoter regions to see if there is any natural variations. The promoters of <italic> <span class="gene">GmDREBL</span> </italic> from 33 cultivated varieties and <italic> GsDREBL </italic> from 43 wild type soybeans were cloned and sequenced. The phylogenetic tree analysis was performed and we found that most of the wild soybeans were clustered together ( <xref ref-type="fig" rid="f7"> Fig. 7 </xref> , names in black), however, some of the wild soybeans clustered with cultivated soybeans ( <xref ref-type="fig" rid="f7"> Fig. 7 </xref> , names in yellow), possibly suggesting a close relationship of the cultivated soybeans with the few wild soybeans in the same group.
					</p>
				
					<p>&nbsp;&nbsp;
						Our previous studies have demonstrated that transcription factors <span class="gene">GmDof4,</span> <span class="gene">GmDof11,</span> <span class="gene">GmMYB73,</span> <span class="gene">GmbZIP123</span> and <span class="gene">GmNFYA</span> play important roles in regulation of seed-related traits and fatty acid biosynthesis <xref ref-type="bibr" rid="b22"> 22 </xref> <xref ref-type="bibr" rid="b25"> 25 </xref> <xref ref-type="bibr" rid="b26"> 26 </xref> <xref ref-type="bibr" rid="b27"> 27 </xref> . Presently, we further found that an AP2-domain protein <span class="gene">GmDREBL</span> promotes fatty acid accumulation in seeds of the transgenic <italic> Arabidopsis </italic> through up-regulation of the master regulatory gene <italic> <span class="gene">WRI1</span> </italic> and other genes related to fatty acid biosynthesis. The <span class="gene">GmDREBL</span> also enhances <span class="trait">seed</span> size and <span class="trait">seed</span> weight. The expression of the <italic> DREBL </italic> gene is relatively higher in developing seeds of the cultivated soybeans compared to that in wild soybeans, and the expression level is roughly positively correlated with the <span class="trait">oil</span> contents in all the soybean accessions. The promoter sequences were further analyzed and the genetic variations in these sequences showed correlations with fatty acid accumulation in soybean seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes in <italic> de novo </italic> synthesis of FAs was further examined and the expression of <italic> KAS1 </italic> (b-Ketoacyl-[acylcarrier protein] synthase I) was significantly increased in <italic> <span class="gene">GmDREBL</span> </italic> -overexpressing transgenic lines compared to WT plants ( <xref ref-type="fig" rid="f4"> Fig. 4A </xref> ). KAS1 is responsible for the elongation of fatty acid (FA) synthesis from <span class="gene">C4</span> to C16 <xref ref-type="bibr" rid="b6"> 6 </xref> . Lacking of <italic> KAS1 </italic> leads to an obvious change in the polar lipid component, abnormal embryo development before the globular stage, and sharp decrease in FA levels in seeds. Genes such as <italic> Pl-PKb1 </italic> , <italic> PDHE1a </italic> , <italic> <span class="gene">FAD2</span> </italic> , <italic> <span class="gene">FAD3</span> </italic> , <italic> BCCP2 </italic> , <italic> ACP1 </italic> , <italic> oleosin </italic> , and <italic> KAS1 </italic> in fatty acid synthesis have similar expression patterns with <italic> <span class="gene">WRI1</span> </italic> during the <span class="trait">seed</span> developmental process <xref ref-type="bibr" rid="b7"> 7 </xref> . These gene expressions increased in <italic> <span class="gene">WRI1</span> </italic> -overexpressing plants but decreased in mutants with down-regulated <italic> <span class="gene">WRI1</span> </italic> expression. <span class="gene">WRI1</span> is a key regulator in FA biosynthetic pathway. Two BnWRI1 genes, <italic> BnWRI1-1 </italic> and <italic> BnWRI1-2, </italic> promoted <span class="trait">seed</span> <span class="trait">oil</span> increase by 10&#x02013;40% in transgenic plants overexpressing <italic> BnWRI1-1 </italic> or <italic> BnWRI1-2 </italic> , and the <span class="trait">seed</span> size was also enlarged <xref ref-type="bibr" rid="b11"> 11 </xref> . Overexpression of <italic> BnSTM </italic> resulted in the induction of genes relevant to FA synthesis including <italic> BnLEC1 </italic> , <italic> BnLEC2 </italic> and <italic> BnWRI1 </italic> , and <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> was also increased in the transgenic plants <xref ref-type="bibr" rid="b30"> 30 </xref> . In <italic> Arabidopsis </italic> , <span class="gene">WRI1</span> regulated the expression of <italic> BCCP2 </italic> <xref ref-type="bibr" rid="b19"> 19 </xref> . BCCP2 is involved in fatty acid biosynthetic process and has acetyl-CoA carboxylase activity. <italic> ZmWRI1a </italic> , the homologue of <italic> AtWRI1 </italic> in maize, complemented the reduced fatty acid <span class="trait">content</span> of <italic> Atwri1-4 </italic> mutant, and overexpression of <italic> ZmWRI1a </italic> dramatically increased the fatty acid <span class="trait">content</span> in maize grain <xref ref-type="bibr" rid="b13"> 13 </xref> . Studies also reported that overexpression of <italic> ZmWRI1 </italic> resulted in an <span class="trait">oil</span> increase <xref ref-type="bibr" rid="b12"> 12 </xref> . In the present study, we proved that <italic> AtWRI1 </italic> or its soybean homologue was upregulated in transgenic plants/transgenic hairy roots overexpressing <italic> <span class="gene">GmDREBL</span> </italic> ( <xref ref-type="fig" rid="f4"> Fig. 4A,H </xref> ). ChIP-qPCR, EMSA and transient transcriptional activation experiments in tobacco leaves ( <xref ref-type="fig" rid="f4"> Fig. 4B&#x02013;F </xref> ) further demonstrate that <span class="gene">GmDREBL</span> can directly bind to the promoter and activate <italic> <span class="gene">WRI1</span> </italic> expressions, finally leading to FA synthesis and <span class="trait">oil</span> accumulation.
					</p>
				
					<p>&nbsp;&nbsp;
						DREB-like transcription factors are generally stress-related and could be induced by abiotic stresses such as <span class="trait">drought</span> and high salt <xref ref-type="bibr" rid="b32"> 32 </xref> <xref ref-type="bibr" rid="b33"> 33 </xref> <xref ref-type="bibr" rid="b34"> 34 </xref> . We also found that <italic> <span class="gene">GmDREBL</span> </italic> could be sharply induced by <span class="trait">drought</span> and high salt ( <xref ref-type="supplementary-material" rid="S1"> Figure S2A,B </xref> ), and overexpressing <italic> <span class="gene">GmDREBL</span> </italic> increased the survival rate of transgenic plants under 200&#x02009;mM NaCl treatment ( <xref ref-type="supplementary-material" rid="S1"> Figure S2C,D </xref> ). The <span class="gene">GmDREBL</span> could directly bind to the promoter region of stress-related gene <italic> LEA </italic> and activate <italic> LEA </italic> expression for stress tolerance ( <xref ref-type="supplementary-material" rid="S1"> Figure S3 </xref> ). The <span class="gene">GmDREBL</span> also promoted expression of the <italic> AtDREB2B (AT3G1102 </italic> 0) ( <xref ref-type="supplementary-material" rid="S1"> Figure S1D </xref> ), which may further contribute to stress tolerance of the transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, overexpressing <italic> <span class="gene">GmDREBL</span> </italic> increased fatty acid accumulation in seeds of transgenic plants, likely through the control of a key regulator <italic> <span class="gene">WRI1</span> </italic> for FA synthesis. Besides, <italic> <span class="gene">GmDREBL</span> </italic> can be regulated by <span class="gene">GmABI3</span> and <span class="gene">GmABI5.</span> Our study provides novel insight to the function of <italic> DREB-type </italic> genes in both <span class="trait">oil</span> accumulation and <span class="trait">seed</span> size control in addition to their roles in stress response. Further manipulation of the <italic> <span class="gene">GmDREBL</span> </italic> in soybean and evaluation in field test should shed light on its roles in regulation of <span class="trait">oil</span> contents in seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						The pGWB405- <italic> <span class="gene">GmDREBL-GFP</span> </italic> vector was transfected into <italic> A. tumefaciens </italic> GV3101 by electroporation, then introduced into <italic> Arabidopsis </italic> Col-0 plants by the vacuum infiltration method <xref ref-type="bibr" rid="b37"> 37 </xref> . Homozygous transgenic lines were obtained and the lines DL-2, DL-4 and DL-25 with different expression levels were selected for further analysis. The subcellular localization of the <span class="gene">GmDREBL</span> protein in integument cells of <span class="trait">seed</span> was visualized under a confocal microscope (Leica TCS SP5). Positiveness of transgenic plant was examined by PCR using plant DNA as a template, and an AtActin-2 gene was amplified as a control. Western blot was performed using GFP-Tag mouse antibody (Abmart) and plant &#x003b2; actin mouse monoclonal antibody (Cwbiotech).
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> How to cite this article </bold> : Zhang, Y.-Q. <italic> et al. </italic> Soybean <span class="gene">GmDREBL</span> Increases Lipid <span class="trait">Content</span> in Seeds of Transgenic <italic> Arabidopsis. Sci. Rep. </italic> <bold> 6 </bold> , 34307; doi: 10.1038/srep34307 (2016).
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> A </bold> ) Phylogenetic tree analysis of <span class="gene">GmDREBL</span> protein and other DREB proteins. The phylogenetic tree was created with the high similarity sequences of <span class="gene">AP2</span> domains from 6 <italic> G. max </italic> proteins and 11 <italic> Arabidopsis thaliana </italic> proteins using PHYLIP, and the bootstrap analyses were conducted with 200 replicates. The bootstrap value was showed in the branch and the length of each branch represents the distance between the sequences. ( <bold> B </bold> ) The organ-specific expression of <italic> <span class="gene">GmDREBL</span> </italic> in <italic> G. max </italic> cv. HN44. The roots, shoots and leaves of two weeks seedling were sampled. The pods with 2 cm length and the opening flowers of adult plant were sampled. Two developmental stages of soybean seeds were selected. Weight of a <span class="gene">H1</span> developing <span class="trait">seed</span> is only 4% of highest <span class="trait">seed</span> fresh weight. Weight of a H5 developing <span class="trait">seed</span> is 35% of highest <span class="trait">seed</span> fresh weight. ( <bold> C </bold> ) The organ-specific expression of <italic> AtDREB2A </italic> and <italic> AtDREB2B in Arabidopsis thaliana </italic> . The homologues of <span class="gene">GmDREBL</span> with high identity in <italic> Arabidopsis </italic> were AtDREB2A and AtDREB2B (e-value&#x02009;&#x0003c;&#x02009;1.3E-30). Their expression was evaluated by publicly available data of microarray in AtGenExpress of TAIR ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.arabidopsis.org"> http://www.arabidopsis.org </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> A </bold> ) Expression of <italic> <span class="gene">GmDREBL</span> </italic> in transgenic plants. The relative expression level of <italic> <span class="gene">GmDREBL</span> </italic> was normalized using <italic> AtACTIN7 </italic> as an internal control. Bars indicate SD (n&#x02009;=&#x02009;4). ( <bold> B </bold> ) Examination of <italic> <span class="gene">GmDREBL</span> </italic> presence by PCR in transgenic plant using genomic DNA as a template. An actin 2 gene was also amplified as the control. ( <bold> C </bold> ) Detection of <span class="gene">GmDREBL-GFP</span> in transgenic plants by Western blot analysis using mouse anti-GFP antibody. And the detection of &#x003b2; actin was used as the control. ( <bold> D </bold> ) Overexpression of <italic> <span class="gene">GmDREBL</span> </italic> increased total fatty acid <span class="trait">content</span> of transgenic <italic> Arabidopsis </italic> seeds. Bars indicate SD (n&#x02009;=&#x02009;4). ( <bold> E </bold> ) The profile of fatty acids in dry seeds of Col-0 and three lines DL-2, DL-4 and DL-25. Bars indicate SD (n&#x02009;=&#x02009;4). ( <bold> F </bold> ) The profile of fatty acids in fresh leaves of 12-day-old seedlings from Col-0 and three lines DL-2, DL-4 and DL-25. Bars indicate SD (n&#x02009;=&#x02009;4). ( <bold> G </bold> ) The rosette phenotypes of Col-0 and three homozygous lines DL-2, DL-4 and DL-25 in two and three weeks. ( <bold> H </bold> ) The scanning electron micrograph of the <span class="trait">seed</span> coat of Col-0 and transgenic plants with <italic> <span class="gene">GmDREBL</span> </italic> . The white line stands for 200&#x02009;&#x003bc;m. ( <bold> G </bold> ) The statistical result of <span class="trait">seed</span> length and <span class="trait">seed</span> width of wild type Col-0 plant and transgenic plants with <italic> <span class="gene">GmDREBL</span> </italic> . Bars indicate SD (n&#x02009;=&#x02009;12). ( <bold> I </bold> ) Overexpression of <italic> <span class="gene">GmDREBL</span> </italic> increased 1000-seed weight of transgenic <italic> Arabidopsis </italic> seeds. Bars indicate SD (n&#x02009;=&#x02009;4). ( <bold> J </bold> ) The <span class="trait">seed</span> <span class="trait">yield</span> per plant of Col-0 and three homozygous lines DL-2, DL-4 and DL-25. Bars indicate SD (n&#x02009;=&#x02009;10). All the asterisks indicate significant difference compared to the corresponding controls (p-value&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						( <bold> A </bold> ) The expression level of <italic> <span class="gene">GmDREBL</span> </italic> in different stages of <span class="trait">seed</span> development. Six developmental stages of soybean seeds were selected according to the ratio of <span class="trait">seed</span> weight of the stage to highest <span class="trait">seed</span> fresh weight, namely 4, 8, 12, 20, 35, and 80% (H1&#x02013;H6). ( <bold> B </bold> ) The expression level of <italic> <span class="gene">GmABI3</span> </italic> in different stages of <span class="trait">seed</span> development. ( <bold> C </bold> ) The expression level of <italic> <span class="gene">GmABI5</span> </italic> in different stages of <span class="trait">seed</span> development. ( <bold> D </bold> ) The expression of <italic> <span class="gene">GmABI3</span> </italic> in soybean transgenic hairy roots harboring <italic> <span class="gene">GmABI3</span> </italic> . ( <bold> E </bold> ) The expression of <italic> <span class="gene">GmABI5</span> </italic> in soybean transgenic hairy roots harboring <italic> <span class="gene">GmABI5</span> </italic> . ( <bold> F </bold> ) The expression of <italic> <span class="gene">GmDREBL</span> </italic> in soybean transgenic hairy roots expressing <italic> <span class="gene">GmABI3</span> </italic> or <italic> <span class="gene">GmABI5</span> </italic> . Bars indicate SD (n&#x02009;=&#x02009;4). All the asterisks indicate significant difference compared to the corresponding controls (p-value&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A Vernonia Diacylglycerol Acyltransferase Can Increase Renewable <span class="trait">Oil</span> Production.</h2>
			<h3>PMID:27578203  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hatanaka T(1), Serson W(2), Li R(3), Armstrong P(4), Yu K(2), Pfeiffer T(2), Li XL(2), Hildebrand D(2).
				<br/> 
				年份：2016 Sep 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the production of plant oils such as soybean <span class="trait">oil</span> as a renewable resource for food and fuel is valuable. Successful breeding for higher <span class="trait">oil</span> levels in soybean, however, usually results in reduced protein, a second valuable <span class="trait">seed</span> component. This study shows that by manipulating a highly active acyl-CoA:diacylglycerol acyltransferase (DGAT) the hydrocarbon flux to <span class="trait">oil</span> in oilseeds can be increased without reducing the protein component. Compared to other plant DGATs, a <span class="gene">DGAT</span> from Vernonia galamensis (VgDGAT1A) produces much higher <span class="trait">oil</span> synthesis and accumulation activity in yeast, insect cells, and soybean. Soybean lines expressing VgDGAT1A show a 4% increase in <span class="trait">oil</span> <span class="trait">content</span> without reductions in <span class="trait">seed</span> protein contents or <span class="trait">yield</span> per unit land area. Incorporation of this trait into 50% of soybeans worldwide could result in an increase of 850 million kg oil/year without new land use or inputs and be worth ∼U.S.$1 billion/year at 2012 production and market prices.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An Improved Variant of Soybean Type 1 Diacylglycerol Acyltransferase Increases the <span class="trait">Oil</span> <span class="trait">Content</span> and Decreases the Soluble Carbohydrate <span class="trait">Content</span> of Soybeans.</h2>
			<h3>PMID:27208257  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roesler K(1), Shen B(2), Bermudez E(2), Li C(2), Hunt J(2), Damude HG(2), Ripp KG(2), Everard JD(2), Booth JR(2), Castaneda L(2), Feng L(2), Meyer K(1).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Kinetically improved diacylglycerol acyltransferase (DGAT) variants were created to favorably alter carbon partitioning in soybean (Glycine max) seeds. Initially, variants of a type 1 <span class="gene">DGAT</span> from a high-oil, high-oleic acid plant <span class="trait">seed,</span> Corylus americana, were screened for high <span class="trait">oil</span> <span class="trait">content</span> in Saccharomyces cerevisiae Nearly all <span class="gene">DGAT</span> variants examined from high-oil strains had increased affinity for oleoyl-CoA, with S0.5 values decreased as much as 4.7-fold compared with the wild-type value of 0.94 µm Improved soybean <span class="gene">DGAT</span> variants were then designed to include amino acid substitutions observed in promising C. americana <span class="gene">DGAT</span> variants. The expression of soybean and C. americana <span class="gene">DGAT</span> variants in soybean somatic embryos resulted in <span class="trait">oil</span> contents as high as 10% and 12%, respectively, compared with only 5% and 7.6% <span class="trait">oil</span> achieved by overexpressing the corresponding wild-type DGATs. The affinity for oleoyl-CoA correlated strongly with <span class="trait">oil</span> <span class="trait">content.</span> The soybean <span class="gene">DGAT</span> variant that gave the greatest <span class="trait">oil</span> increase contained 14 amino acid substitutions out of a total of 504 (97% sequence identity with native). Seed-preferred expression of this soybean <span class="gene">DGAT1</span> variant increased <span class="trait">oil</span> <span class="trait">content</span> of soybean seeds by an average of 3% (16% relative increase) in highly replicated, single-location field trials. The <span class="gene">DGAT</span> transgenes significantly reduced the soluble carbohydrate <span class="trait">content</span> of mature seeds and increased the <span class="trait">seed</span> protein <span class="trait">content</span> of some events. This study demonstrated that engineering of the native <span class="gene">DGAT</span> enzyme is an effective strategy to improve the <span class="trait">oil</span> <span class="trait">content</span> and value of soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of large DNA deletions affecting <span class="trait">oil</span> quality traits in soybean seeds through transcriptome sequencing analysis.</h2>
			<h3>PMID:27179525  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Goettel W(1), Ramirez M(2), Upchurch RG(2), An YQ(3).
				<br/> 
				年份：2016 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Identification and characterization of a 254-kb genomic deletion on a duplicated chromosome segment that resulted in a low level of palmitic acid in soybean seeds using transcriptome sequencing. A large number of soybean genotypes varying in <span class="trait">seed</span> <span class="trait">oil</span> composition and <span class="trait">content</span> have been identified. Understanding the molecular mechanisms underlying these variations is important for breeders to effectively utilize them as a genetic resource. Through design and application of a bioinformatics approach, we identified nine co-regulated gene clusters by comparing <span class="trait">seed</span> transcriptomes of nine soybean genotypes varying in <span class="trait">oil</span> composition and <span class="trait">content.</span> We demonstrated that four gene clusters in the genotypes M23, Jack and N0304-303-3 coincided with large-scale genome rearrangements. The co-regulated gene clusters in M23 and Jack mapped to a previously described 164-kb deletion and a copy number amplification of the <span class="gene">Rhg1</span> locus, respectively. The coordinately down-regulated gene clusters in N0304-303-3 were caused by a 254-kb deletion containing 19 genes including a fatty acyl-ACP thioesterase B gene (FATB1a). This deletion was associated with reduced palmitic acid <span class="trait">content</span> in seeds and was the molecular cause of a previously reported nonfunctional <span class="gene">FATB1a</span> allele, fap nc . The M23 and N0304-304-3 deletions were located in duplicated genome segments retained from the Glycine-specific whole genome duplication that occurred 13 million years ago. The homoeologous genes in these duplicated regions shared a strong similarity in both their encoded protein sequences and transcript accumulation levels, suggesting that they may have conserved and important functions in seeds. The functional conservation of homoeologous genes may result in genetic redundancy and gene dosage effects for their associated <span class="trait">seed</span> traits, explaining why the large deletion did not cause lethal effects or completely eliminate palmitic acid in N0304-303-3.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An Integrated Bioinformatics Analysis Reveals Divergent Evolutionary Pattern of <span class="trait">Oil</span> Biosynthesis in High- and Low-Oil Plants</h2>
			<h3>PMID:27159078</h3>
				

				
					<h3>PMCID:4861283</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Shi-Bo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Qi-Gang </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Yu-Qi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zheng </surname>  <given-names> Huan-Quan </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dunwell </surname>  <given-names> Jim M. </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yuan-Ming </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> oils provide a renewable source of food, biofuel and industrial raw materials that is important for humans. Although many genes and pathways for acyl-lipid metabolism have been identified, little is known about whether there is a specific mechanism for high-oil <span class="trait">content</span> in high-oil plants. Based on the distinct differences in <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> between four high-oil dicots (20~50%) and three low-oil grasses (&lt;3%), comparative genome, transcriptome and differential expression analyses were used to investigate this mechanism. Among 4,051 dicot-specific soybean genes identified from 252,443 genes in the seven species, 54 genes were shown to directly participate in acyl-lipid metabolism, and 93 genes were found to be associated with acyl-lipid metabolism. Among the 93 dicot-specific genes, 42 and 27 genes, including CBM20-like SBDs and GPT2, participate in carbohydrate degradation and transport, respectively. 40 genes highly up-regulated during <span class="trait">seed</span> <span class="trait">oil</span> rapid accumulation period are mainly involved in initial fatty acid synthesis, triacylglyceride assembly and oil-body formation, for example, ACCase, PP, DGAT1, PDAT1, OLEs and STEROs, which were also found to be differentially expressed between high- and low-oil soybean accessions. Phylogenetic analysis revealed distinct differences of oleosin in patterns of gene duplication and loss between high-oil dicots and low-oil grasses. In addition, seed-specific <span class="gene">GmGRF5,</span> ABI5 and <span class="gene">GmTZF4</span> were predicted to be candidate regulators in <span class="trait">seed</span> <span class="trait">oil</span> accumulation. This study facilitates future research on lipid biosynthesis and potential genetic improvement of <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">oil</span> is not only the major source of carbon and energy for germination and seedling growth but also provides humans with renewable sources of food, biofuel and industrial raw materials. Up to now acyl-lipid metabolism in Arabidopsis has been well studied and more than 600 genes have been predicted to encode the enzymes and regulatory factors associated with this process [ <xref rid="pone.0154882.ref007" ref-type="bibr"> 7 </xref> , <xref rid="pone.0154882.ref008" ref-type="bibr"> 8 </xref> ]. Among these predicted genes, some have been shown to be associated with changes in <span class="trait">seed</span> <span class="trait">oil</span> accumulation [ <xref rid="pone.0154882.ref006" ref-type="bibr"> 6 </xref> ]. The over-expression of many individual key enzymes altered <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> in various plants. For example, ACCase in potato [ <xref rid="pone.0154882.ref009" ref-type="bibr"> 9 </xref> ], <italic> Brassica napus </italic> [ <xref rid="pone.0154882.ref010" ref-type="bibr"> 10 </xref> ] and <italic> Escherichia coli </italic> [ <xref rid="pone.0154882.ref011" ref-type="bibr"> 11 </xref> ]; glycerol-3-phosphate dehydrogenase (GPDH) in <italic> B </italic> . <italic> napus </italic> [ <xref rid="pone.0154882.ref012" ref-type="bibr"> 12 </xref> ]; glycerol-3-phosphate acyltransferase (GPAT) in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref013" ref-type="bibr"> 13 </xref> ]; 2-lysophosphatidic acid acyltransferase (LPAAT) in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref014" ref-type="bibr"> 14 </xref> ]; and acyl-CoA: diacylglycerol acytransferase (DGAT) in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0154882.ref016" ref-type="bibr"> 16 </xref> ], <italic> Glycine max </italic> [ <xref rid="pone.0154882.ref017" ref-type="bibr"> 17 </xref> ], <italic> B </italic> . <italic> napus </italic> [ <xref rid="pone.0154882.ref018" ref-type="bibr"> 18 </xref> ] and maize [ <xref rid="pone.0154882.ref019" ref-type="bibr"> 19 </xref> ]. In addition to these key enzymes that participate in lipid synthesis, the expression of transcription factors (TFs) directly or indirectly regulating genes involved in carbohydrate and lipid metabolism can also evidently change <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content;</span> such TFs include <italic> WRINKLED1 </italic> ( <italic> <span class="gene">WRI1</span> </italic> ) [ <xref rid="pone.0154882.ref020" ref-type="bibr"> 20 </xref> ], <italic> LEAFY COTYLEDON1 </italic> ( <italic> LEC1 </italic> ) [ <xref rid="pone.0154882.ref021" ref-type="bibr"> 21 </xref> , <xref rid="pone.0154882.ref022" ref-type="bibr"> 22 </xref> ], <italic> LEAFY COTYLEDON2 </italic> ( <italic> LEC2 </italic> ) [ <xref rid="pone.0154882.ref023" ref-type="bibr"> 23 </xref> ], <italic> FUSCA3 </italic> ( <italic> FUS3 </italic> ) [ <xref rid="pone.0154882.ref024" ref-type="bibr"> 24 </xref> ], <italic> <span class="gene">GmbZIP123</span> </italic> [ <xref rid="pone.0154882.ref025" ref-type="bibr"> 25 </xref> ], <italic> <span class="gene">GmMYB73</span> </italic> [ <xref rid="pone.0154882.ref026" ref-type="bibr"> 26 </xref> ] and <italic> ABSCISIC ACID INSENSITIVE3 </italic> ( <italic> ABI3 </italic> ) [ <xref rid="pone.0154882.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0154882.ref028" ref-type="bibr"> 28 </xref> ]. However, each of the above studies only focused on a single enzyme or TF involved in lipid metabolism. In reality, <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> is affected by multiple genes [ <xref rid="pone.0154882.ref029" ref-type="bibr"> 29 </xref> , <xref rid="pone.0154882.ref030" ref-type="bibr"> 30 </xref> ] or multiple reactions [ <xref rid="pone.0154882.ref031" ref-type="bibr"> 31 </xref> ]. For example, specific combination of expression of <italic> <span class="gene">WRI1</span> </italic> , <italic> <span class="gene">DGAT</span> </italic> and triacylglycerol lipase <italic> SUGAR-DEPENDENT1 </italic> resulted in a higher percentage <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> than that obtained by manipulation of each gene individually [ <xref rid="pone.0154882.ref032" ref-type="bibr"> 32 </xref> ]. More importantly, the <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> is influenced by multiple metabolic pathways, such as sucrose catabolism, glycolysis, pentose phosphate pathway, and related pathways which rely on the supply of carbon [ <xref rid="pone.0154882.ref033" ref-type="bibr"> 33 </xref> &#x02013; <xref rid="pone.0154882.ref038" ref-type="bibr"> 38 </xref> ]. Therefore, it is necessary to consider the involvement of multiple genes and the interaction of multiple related pathways in order to understand the mechanism of high-oil <span class="trait">content</span> in high-oil plants. The recent sequencing of many plant genomes has provided an opportunity to investigate this mechanism using comparative genome and transcriptome analyses.
					</p>
				
					<p>&nbsp;&nbsp;
						Fourteen of the above 40 genes were observed in FA synthesis, and 10 of the 14 genes were involved in the initiation of FA synthesis. Among the 10 genes, five encoded two subunits of the plastidial pyruvate dehydrogenase complex (PDHC) that promotes pyruvate decarboxylation to acetyl-CoA and is the key enzyme linking carbohydrate metabolism to FA synthesis [ <xref rid="pone.0154882.ref004" ref-type="bibr"> 4 </xref> ]; and the others encoded two subunits of ACCase. Of these 40 genes, two genes, those encoding ACP4 and LACS9, were involved in FA transportation. Eight of the above 40 genes were involved in TAG assembly; this group included 2, 3, 2, and 1 genes respectively coding phosphatidate phosphatase (PP), DGAT1, phospholipid: diacylglycerol acyltransferase (PDAT1), and PDAT2; these enzymes catalyze the consecutive steps after the second acylation of glycerol-3-phosphate [ <xref rid="pone.0154882.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0154882.ref019" ref-type="bibr"> 19 </xref> , <xref rid="pone.0154882.ref048" ref-type="bibr"> 48 </xref> ]. Three and six of the above 40 genes, respectively encoding steroleosins (STEROs) and oleosins (OLEs), were observed in oil-body formation, and they were up-regulated, mostly by some hundredfold, during stages from 5&#x02013;6 mg to 400&#x02013;500 mg in seeds ( <xref ref-type="supplementary-material" rid="pone.0154882.s004"> S4 Dataset </xref> ). The remaining genes in this group of 40 genes encoded the TFs WRI1, VAL1, FUS3, and ABI3. Kim et al. [ <xref rid="pone.0154882.ref049" ref-type="bibr"> 49 </xref> ] divided Arabidopsis oleosin genes into three groups on the basis of their tissue-specific expression. In this study, all the genes in the S type (genes expressed only in maturing <span class="trait">seed</span> (siliques)) and SM type (genes expressed in both maturing seeds and florets (microspores)) were used to conduct a phylogenetic analysis ( <xref ref-type="fig" rid="pone.0154882.g003"> Fig 3 </xref> ). The results showed distinct differences of OLEs in patterns of gene duplication and loss between high-oil dicots and low-oil grasses. Genes coding oleosins were preferentially retained in high-oil dicots. In two angiosperm groups, there was only one copy in each grass species, while there were many duplicates in dicots. Remarkably, two high-oil dicot-specific groups were identified, which were resulted from gene loss in grasses after whole genome duplication of ancestor angiosperm. In dicot-specific group 1, <italic> <span class="gene">Glyma04g08220</span> </italic> and <italic> <span class="gene">Glyma06g08290</span> </italic> were up-regulated by a thousand-fold during stages from 5&#x02013;6 mg to 400&#x02013;500 mg ( <xref ref-type="supplementary-material" rid="pone.0154882.s004"> S4 Dataset </xref> ) and may play an important role in soybean <span class="trait">seed</span> <span class="trait">oil</span> accumulation. In dicot-specific group 2, <italic> At3G01570 </italic> , <italic> At3G27660 </italic> and <italic> At5G40420 </italic> in Arabidopsis were shown to be S type genes [ <xref rid="pone.0154882.ref049" ref-type="bibr"> 49 </xref> ]. Meanwhile, <italic> <span class="gene">Glyma05g08880</span> </italic> and <italic> <span class="gene">Glyma19g00400</span> </italic> in this group were highly up-regulated in hundred-fold in the dry whole <span class="trait">seed</span> ( <xref ref-type="supplementary-material" rid="pone.0154882.s004"> S4 Dataset </xref> ), indicating that this dicot-specific group may perform some specific functions in <span class="trait">seed</span> development. Furthermore, highly up-regulated and abundant OLEs probably play major roles in <span class="trait">oil</span> accumulation and/or <span class="trait">oil</span> body development, and are associated with high-oil <span class="trait">content</span> in seeds [ <xref rid="pone.0154882.ref050" ref-type="bibr"> 50 </xref> , <xref rid="pone.0154882.ref051" ref-type="bibr"> 51 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Several important transcription factors (TFs) have been found to participate in the regulation of <span class="trait">seed</span> <span class="trait">oil</span> accumulation. These TFs are distributed in the TF families like AP2, basic leucine zipper (bZIP), B3, NF-YB and Dof [ <xref rid="pone.0154882.ref021" ref-type="bibr"> 21 </xref> , <xref rid="pone.0154882.ref022" ref-type="bibr"> 22 </xref> , <xref rid="pone.0154882.ref024" ref-type="bibr"> 24 </xref> &#x02013; <xref rid="pone.0154882.ref027" ref-type="bibr"> 27 </xref> , <xref rid="pone.0154882.ref052" ref-type="bibr"> 52 </xref> , <xref rid="pone.0154882.ref053" ref-type="bibr"> 53 </xref> ]. In soybean, there were 3,714 genes coding TFs [ <xref rid="pone.0154882.ref054" ref-type="bibr"> 54 </xref> ], and 19 of these were found to be known lipid-related TFs ( <xref ref-type="supplementary-material" rid="pone.0154882.s005"> S5 Dataset </xref> ). To identify additional potential TFs related to <span class="trait">seed</span> <span class="trait">oil</span> accumulation, we conducted a PLC network analysis between all the TFs in soybean and the above 156 genes coding core lipid synthetic enzymes. As a result, 327 TFs were found to be co-expressed with at least two genes coding core lipid synthetic enzymes (P-value&#x0003c;1e-4) ( <xref ref-type="supplementary-material" rid="pone.0154882.s005"> S5 Dataset </xref> ), and were classified into 42 gene families. Among the 327 TFs, 67 were distributed in high-oil dicot-specific clusters ( <xref ref-type="supplementary-material" rid="pone.0154882.s005"> S5 Dataset </xref> ), and 70 had expression levels greater than twice the average level in at least one of stages from 5&#x02013;6 mg to 400&#x02013;500 mg. These 70 TFs were compared with the 69 seed-specific TFs described by Song et al. [ <xref rid="pone.0154882.ref025" ref-type="bibr"> 25 </xref> ]. Seven genes were found to be seed-specific, respectively coding tandem CCCH zinc finger protein 4 ( <italic> <span class="gene">GmTZF4</span> </italic> ; <italic> <span class="gene">Glyma12g13300</span> </italic> , <italic> <span class="gene">Glyma06g44440</span> </italic> , <italic> <span class="gene">Glyma12g33320</span> </italic> ), growth-regulating factor 5 ( <italic> <span class="gene">GmGRF5</span> </italic> ; <italic> <span class="gene">Glyma07g04290</span> </italic> ), ABI5 ( <italic> <span class="gene">Glyma10g08370</span> </italic> ), ABI3 ( <italic> <span class="gene">Glyma18g38490</span> </italic> ) and a sequence-specific DNA binding TF ( <italic> <span class="gene">Glyma03g34730</span> </italic> ) ( <xref ref-type="table" rid="pone.0154882.t002"> Table 2 </xref> ). Since these seven seed-specific TFs were not only highly up-regulated during the rapid <span class="trait">oil</span> accumulation phase of <span class="trait">seed</span> development but also co-expressed with key enzymes of lipid synthesis genes, we deduced that they possibly play roles in regulating <span class="trait">seed</span> <span class="trait">oil</span> accumulation. It should be noted that ABI3 has been confirmed to be a regulator in oleosin gene expression [ <xref rid="pone.0154882.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0154882.ref028" ref-type="bibr"> 28 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						We investigated co-expression networks of the above 19 known lipid-related TFs with 156 genes coding core lipid synthetic enzymes. As a result, 14 lipid-related TFs were co-expressed with more than 10 lipid synthesis genes (P-value&#x0003c;0.05) ( <xref ref-type="supplementary-material" rid="pone.0154882.s005"> S5 Dataset </xref> ), such as two genes ( <italic> <span class="gene">Glyma15g34770</span> </italic> and <italic> <span class="gene">Glyma08g24420</span> </italic> ) coding <span class="gene">WRI1</span> were found to be co-expressed with 19 lipid synthesis genes ( <xref ref-type="table" rid="pone.0154882.t003"> Table 3 </xref> ). Among the 19 lipid synthesis genes, genes coding BCCP2, BC, ENR and &#x003b1;-PDH have been verified to be regulated by <italic> <span class="gene">WRI1</span> </italic> in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref053" ref-type="bibr"> 53 </xref> , <xref rid="pone.0154882.ref020" ref-type="bibr"> 20 </xref> , <xref rid="pone.0154882.ref023" ref-type="bibr"> 23 </xref> ], and 18 genes have an AW-box in their promoter ( <xref ref-type="supplementary-material" rid="pone.0154882.s007"> S2 Fig </xref> ) that has been shown to be a direct target of <span class="gene">WRI1</span> in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref053" ref-type="bibr"> 53 </xref> ]. Note that <italic> <span class="gene">WRI1</span> </italic> has a high co-expression relationship with <italic> <span class="gene">Glyma06g11860</span> </italic> ( <italic> LACS9 </italic> ; r = 0.9801, P-value = 2.32e-5), and <italic> LACS9 </italic> is found to impact the biosynthesis of <span class="trait">seed</span> storage lipids in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref055" ref-type="bibr"> 55 </xref> ], and is considered as the major LACS isoform involved in plastidial FA export for TAG formation [ <xref rid="pone.0154882.ref056" ref-type="bibr"> 56 </xref> ]. We surmised that <italic> <span class="gene">WRI1</span> </italic> up-regulated <italic> LACS9 </italic> and this regulation could lead to an increased export of FAs to the endoplasmic reticulum (ER), and a subsequent increase in the rate of FA synthesis and triacylglycerol synthesis.
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, the expression of <italic> <span class="gene">WRI1</span> </italic> could increase <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> in both dicots and grasses, such as in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref020" ref-type="bibr"> 20 </xref> ], oilseed rape [ <xref rid="pone.0154882.ref057" ref-type="bibr"> 57 </xref> ], <span class="trait">oil</span> palm [ <xref rid="pone.0154882.ref058" ref-type="bibr"> 58 </xref> , <xref rid="pone.0154882.ref059" ref-type="bibr"> 59 </xref> ] and maize [ <xref rid="pone.0154882.ref060" ref-type="bibr"> 60 </xref> ]. However, distinct differences exist between dicots and grasses. These differences include low sequence similarity that results in dicot-specific and grass-specific clusters ( <xref ref-type="supplementary-material" rid="pone.0154882.s001"> S1 Dataset </xref> ), different gene structures, and a low evolutionary rate in dicots (&#x003c9;1 = 0.0861) as compared with that in grasses (&#x003c9;0 = 0.2809) ( <xref ref-type="supplementary-material" rid="pone.0154882.s008"> S3 Fig </xref> ). More importantly, <italic> <span class="gene">WRI1</span> </italic> is included in different regulatory networks in dicots and grasses [ <xref rid="pone.0154882.ref061" ref-type="bibr"> 61 </xref> ]. In <italic> Arabidopsis </italic> , the expression of <italic> <span class="gene">WRI1</span> </italic> is up-regulated by <italic> LEC1 </italic> , <italic> LEC2 </italic> and <italic> FUS3 </italic> , and <italic> <span class="gene">WRI1</span> </italic> is a direct target of <italic> LEC2 </italic> [ <xref rid="pone.0154882.ref023" ref-type="bibr"> 23 </xref> ], and possibly of <italic> FUS3 </italic> [ <xref rid="pone.0154882.ref062" ref-type="bibr"> 62 </xref> ]. In maize, however, no ortholog of <italic> AtLEC2 </italic> was identified [ <xref rid="pone.0154882.ref060" ref-type="bibr"> 60 </xref> ], and <italic> <span class="gene">WRI1</span> </italic> is able to regulate amino acid biosynthesis [ <xref rid="pone.0154882.ref063" ref-type="bibr"> 63 </xref> ]. In addition, <italic> LEC1 </italic> and <italic> FUS3 </italic> in <italic> Arabidopsis </italic> have low sequence similarity compared with those in grasses.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 27 dicot-specific genes in transport, four genes ( <italic> <span class="gene">Glyma07g38830</span> </italic> , <italic> <span class="gene">Glyma13g27680</span> </italic> , <italic> <span class="gene">Glyma15g11270</span> </italic> and <italic> <span class="gene">Glyma18g08740</span> </italic> ) belonged to <italic> PF03151 </italic> (Triose-phosphate Transporter family). The first three genes were annotated as genes encoding the glucose 6-phosphate/phosphate translocator 2 (GPT2), which is located in the envelope of the plastids for the importation of Glc6P [ <xref rid="pone.0154882.ref071" ref-type="bibr"> 71 </xref> ]. In non-green plastids, glucose 6-phosphate (Glc6P) can be used as a precursor for starch biosynthesis and FA synthesis and also as a substrate for the Oxidative Pentose Phosphate Pathway (OPPP) which supplies reducing power to drive FA synthesis [ <xref rid="pone.0154882.ref004" ref-type="bibr"> 4 </xref> , <xref rid="pone.0154882.ref072" ref-type="bibr"> 72 </xref> ]. Although <italic> AtGPT2 </italic> has no obvious effects on plant development [ <xref rid="pone.0154882.ref073" ref-type="bibr"> 73 </xref> , <xref rid="pone.0154882.ref074" ref-type="bibr"> 74 </xref> ], its over-expression could increase the net import of Glc6P from cytosol to chloroplast and accelerate the accumulation of soluble sugars in <italic> Arabidopsis </italic> [ <xref rid="pone.0154882.ref075" ref-type="bibr"> 75 </xref> , <xref rid="pone.0154882.ref076" ref-type="bibr"> 76 </xref> ]. Moreover, the expression of <italic> GPT2 </italic> in <span class="trait">oil</span> palm increased notably during fruit ripening and was significantly higher than in date palm, a low <span class="trait">oil</span> <span class="trait">content</span> palm [ <xref rid="pone.0154882.ref077" ref-type="bibr"> 77 </xref> ]. More importantly, <italic> <span class="gene">Glyma15g11270</span> </italic> coding GPT2 was also differentially expressed in high- and low-oil soybean accessions ( <xref ref-type="table" rid="pone.0154882.t001"> Table 1 </xref> and <xref ref-type="supplementary-material" rid="pone.0154882.s012"> S4 Table </xref> ). In that case, the specific GPT2 in high-oil <span class="trait">content</span> dicots implies a strong funneling of carbon toward pyruvate in the plastid, a significant increase for FA synthesis and ultimately an increase in the <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> [ <xref rid="pone.0154882.ref077" ref-type="bibr"> 77 </xref> ]. The remaining 23 transport genes had unclear functions ( <xref ref-type="table" rid="pone.0154882.t001"> Table 1 </xref> and <xref ref-type="supplementary-material" rid="pone.0154882.s012"> S4 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 23 dicot-specific genes encoding key enzymes of FA synthesis in clusters 3 to 5 ( <xref ref-type="supplementary-material" rid="pone.0154882.s010"> S2 Table </xref> ), 10 are involved in FA synthesis, i.e., <italic> <span class="gene">Glyma05g36450</span> </italic> and <italic> <span class="gene">Glyma08g03120</span> </italic> encode Biotin Carboxylase ( <italic> BC </italic> ); <italic> <span class="gene">Glyma13g06080</span> </italic> , <italic> <span class="gene">Glyma18g50020</span> </italic> and <italic> <span class="gene">Glyma19g03530</span> </italic> encode Biotin Carboxyl Carrier Protein 2 ( <italic> BCCP2 </italic> ); <italic> <span class="gene">Glyma18g42280</span> </italic> and <italic> <span class="gene">Glyma18g42300</span> </italic> encode Carboxyltransferase alpha subunit ( <italic> &#x003b1;-CT </italic> ); <italic> <span class="gene">Glyma15g34770</span> </italic> and <italic> <span class="gene">Glyma08g24420</span> </italic> belong to the <italic> AP2/EREBP </italic> family and encode soybean <italic> <span class="gene">WRI1</span> </italic> ; and <italic> <span class="gene">Glyma16g05480</span> </italic> encodes the FUSCA3 (FUS3). All these genes except the two genes encoding BC were highly up-regulated during the rapid <span class="trait">oil</span> accumulation phase of <span class="trait">seed</span> development. <italic> <span class="gene">Glyma19g03530</span> </italic> encoding BCCP2 was up-regulated in high-oil soybean accessions compared with the two low-oil accessions ( <xref ref-type="fig" rid="pone.0154882.g002"> Fig 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">WRI1</span> </italic> plays an important role in controlling the rate of carbon flow from carbohydrate metabolism to lipid synthesis, and is capable of affecting the <span class="trait">seed</span> <span class="trait">oil</span> accumulation by regulating a set of genes involved in lipid synthesis, glycolysis and photosynthesis [ <xref rid="pone.0154882.ref080" ref-type="bibr"> 80 </xref> , <xref rid="pone.0154882.ref057" ref-type="bibr"> 57 </xref> , <xref rid="pone.0154882.ref047" ref-type="bibr"> 47 </xref> ]. Although <italic> <span class="gene">WRI1</span> </italic> could regulate <span class="trait">seed</span> <span class="trait">oil</span> accumulation in both grasses and high-oil dicots, different gene structures and divergent evolutionary rates between these two lineages were observed ( <xref ref-type="supplementary-material" rid="pone.0154882.s008"> S3 Fig </xref> ). We deduced that different evolutionary mechanisms of <italic> <span class="gene">WRI1</span> </italic> in high-oil dicots and grasses might lead to different regulatory networks [ <xref rid="pone.0154882.ref061" ref-type="bibr"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although up-regulation of key enzymes in lipid synthesis like ACCase alone slightly increases <span class="trait">oil</span> <span class="trait">content</span> [ <xref rid="pone.0154882.ref011" ref-type="bibr"> 11 </xref> ], its effect on increasing the <span class="trait">oil</span> <span class="trait">content</span> was much less than that by up-regulation of <italic> <span class="gene">WRI1</span> </italic> or other TFs [ <xref rid="pone.0154882.ref047" ref-type="bibr"> 47 </xref> , <xref rid="pone.0154882.ref059" ref-type="bibr"> 59 </xref> ]. This partly indicates that <span class="trait">oil</span> accumulation is a complex biological process and increasing the expression of TFs may be an effective approach to significantly improve <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In the initial period of FA synthesis, apart from five dicot-specific genes coding ACCase, five genes coding PDHC were also found to be highly up-regulated during the rapid <span class="trait">oil</span> accumulation phase of <span class="trait">seed</span> development ( <xref ref-type="fig" rid="pone.0154882.g002"> Fig 2 </xref> ). In several species, the expression of <italic> PDHC </italic> has been reported to be associated with <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> [ <xref rid="pone.0154882.ref081" ref-type="bibr"> 81 </xref> &#x02013; <xref rid="pone.0154882.ref083" ref-type="bibr"> 83 </xref> ]. In this study, <italic> <span class="gene">Glyma07g05550</span> </italic> encoding a subunit of PDHC was found to be a DEG between high- and low-oil soybean accessions ( <xref ref-type="fig" rid="pone.0154882.g002"> Fig 2 </xref> ). A similar phenomenon between high- and low-oil accessions in oat was also observed by Hayden et al. [ <xref rid="pone.0154882.ref084" ref-type="bibr"> 84 </xref> ]. Therefore, we assumed that the up-regulated expression of <italic> PDHC </italic> and <italic> ACCase </italic> in soybean likely resulted in the increase of carbon flux to FA synthesis, and then to an increase of the efficiency of FA synthesis [ <xref rid="pone.0154882.ref084" ref-type="bibr"> 84 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 54 dicot-specific genes involved in acyl-lipid metabolism, 12 genes encoded lipid transfer proteins (LTPs), including 5 DEGs between high- and low-oil soybean accessions ( <xref ref-type="supplementary-material" rid="pone.0154882.s010"> S2 Table </xref> ). Apart from the above 12 high-oil dicot-specific LTPs, we also identified nine additional DEGs coding LTPs ( <xref ref-type="supplementary-material" rid="pone.0154882.s010"> S2 Table </xref> ). Wang et al. [ <xref rid="pone.0154882.ref064" ref-type="bibr"> 64 </xref> ] hypothesizes that an increase of the number of <italic> LTP1 </italic> genes in sesame might enhance <span class="trait">oil</span> accumulation by strengthening the transport of FAs, acyl-CoAs, and other lipid molecules. On this basis, we proposed that abundant <italic> <span class="gene">LTP</span> </italic> genes in <span class="trait">oil</span> plants might possibly benefit <span class="trait">oil</span> accumulation. Among the 40 up-regulated genes during the stages from 5&#x02013;6 mg to 400&#x02013;500 mg ( <xref ref-type="fig" rid="pone.0154882.g002"> Fig 2 </xref> ), two genes <italic> <span class="gene">Glyma08g01180</span> </italic> and <italic> <span class="gene">Glyma06g11860</span> </italic> encoded ACP4 and LACS9, respectively. In addition, two genes ( <italic> <span class="gene">Glyma13g11700</span> </italic> and <italic> <span class="gene">Glyma20g07280</span> </italic> ) that belong to LACS family are up-regulated DEGs in high-oil accession ( <xref ref-type="fig" rid="pone.0154882.g004"> Fig 4B </xref> ). Therefore, dicot-specific or up-regulation of <italic> <span class="gene">LTP</span> </italic> , <italic> ACP4 </italic> and <italic> LACS </italic> in dicots could possibly increase the efficiency of plastidial fatty acid export for TAG synthesis and then consequently regulate <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> [ <xref rid="pone.0154882.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0154882.ref085" ref-type="bibr"> 85 </xref> , <xref rid="pone.0154882.ref086" ref-type="bibr"> 86 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In TAG assembly, eight genes were highly up-regulated. Among these eight, <italic> <span class="gene">Glyma04g04060</span> </italic> encoding PP, <italic> <span class="gene">Glyma09g07520</span> </italic> and <italic> <span class="gene">Glyma13g16560</span> en </italic> coding DGAT1, and <italic> <span class="gene">Glyma13g16790</span> </italic> encoding PDAT1 were also differentially expressed in high- and low-oil soybean accessions ( <xref ref-type="fig" rid="pone.0154882.g002"> Fig 2 </xref> ). <italic> <span class="gene">DGAT1</span> </italic> has a principal role in TAG biosynthesis [ <xref rid="pone.0154882.ref087" ref-type="bibr"> 87 </xref> ] and over-expression of <italic> <span class="gene">DGAT1</span> </italic> had been shown to enhance <span class="trait">oil</span> accumulation [ <xref rid="pone.0154882.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0154882.ref032" ref-type="bibr"> 32 </xref> , <xref rid="pone.0154882.ref088" ref-type="bibr"> 88 </xref> ]. In <italic> B </italic> . <italic> napus </italic> , two domestication-related genes <italic> BnaA01g32210D </italic> and <italic> BnaAnng30990D </italic> , encoding PP and DGAT1, respectively [ <xref rid="pone.0154882.ref089" ref-type="bibr"> 89 </xref> ], possibly played roles in increasinging <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span> In oil-body formation, three and six genes respectively encoding STEROs and OLEs were identified and were up-regulated by some hundredfold between the stages from 5&#x02013;6 mg to 400&#x02013;500 mg ( <xref ref-type="fig" rid="pone.0154882.g003"> Fig 3 </xref> and <xref ref-type="supplementary-material" rid="pone.0154882.s004"> S4 Dataset </xref> ). Phylogenetic analysis showed more copies of genes coding OLEs in the dicots than in the grasses ( <xref ref-type="fig" rid="pone.0154882.g003"> Fig 3 </xref> ). Highly up-regulated and abundant OLEs possibly play vital roles in regulating <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> in dicots. Evidently, Siloto et al. [ <xref rid="pone.0154882.ref090" ref-type="bibr"> 90 </xref> ] and Miquel et al. [ <xref rid="pone.0154882.ref051" ref-type="bibr"> 51 </xref> ] showed that the size and spatial distribution of <span class="trait">oil</span> bodies affect the total lipid <span class="trait">content</span> and <span class="trait">oil</span> body proteins have specific functions in lipid accumulation.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 54 dicot-specific genes involved in acyl-lipid metabolism ( <xref ref-type="supplementary-material" rid="pone.0154882.s010"> S2 Table </xref> ), <italic> <span class="gene">Glyma07g01310</span> </italic> , <italic> <span class="gene">Glyma08g20710</span> </italic> and <italic> <span class="gene">Glyma15g02710</span> </italic> encode Phospholipase D&#x003b1; ( <italic> PLD&#x003b1; </italic> ), which is able to hydrolyze phospholipids, producing signalling molecule phosphatidic acid [ <xref rid="pone.0154882.ref091" ref-type="bibr"> 91 </xref> ]. A suppression of the expression of <italic> PLD&#x003b1; </italic> led to a significant decrease in triacylglycerol levels in <italic> Arabidopsis </italic> leaves [ <xref rid="pone.0154882.ref026" ref-type="bibr"> 26 </xref> ] and could also slow the conversion of phosphatidylcholine to TAG in soybean seeds [ <xref rid="pone.0154882.ref092" ref-type="bibr"> 92 </xref> ]. Two transgenic <italic> B </italic> . <italic> napus </italic> cultivars expressing an <italic> Arabidopsis PLD&#x003b1;1 </italic> both demonstrated a 9% increase in <span class="trait">seed</span> total <span class="trait">oil</span> <span class="trait">content</span> [ <xref rid="pone.0154882.ref093" ref-type="bibr"> 93 </xref> ]. In addition, <italic> PLD&#x003b1; </italic> is regulated by <italic> <span class="gene">GmMYB73</span> </italic> , which has been shown to be an important TF regulating lipid <span class="trait">content</span> [ <xref rid="pone.0154882.ref026" ref-type="bibr"> 26 </xref> ]. Although these three dicot-specific <italic> PLD&#x003b1; </italic> are not DEGs in high- and low-oil soybean accessions, two of their homologous genes, <italic> <span class="gene">Glyma13g44170</span> </italic> and <italic> <span class="gene">Glyma08g22600</span> </italic> , are found to be differentially expressed genes ( <xref ref-type="supplementary-material" rid="pone.0154882.s010"> S2 Table </xref> ). We suspect that up-regulated dicot-specific <italic> PLD&#x003b1; </italic> and the other two DEGs of <italic> PLD&#x003b1; </italic> would possibly have a significant effect on the <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 93 dicot-specific genes associated with acyl-lipid metabolism, 11 genes were in the category GO:0007165 (signal transduction) ( <xref ref-type="table" rid="pone.0154882.t001"> Table 1 </xref> and <xref ref-type="supplementary-material" rid="pone.0154882.s012"> S4 Table </xref> ). Among these 11 genes, three ( <italic> <span class="gene">Glyma09g07090</span> </italic> , <italic> <span class="gene">Glyma15g18380</span> </italic> and <italic> <span class="gene">Glyma17g06290</span> </italic> ) encode <span class="gene">GATA</span> TFs, which play a role in light-mediated transcriptional regulation [ <xref rid="pone.0154882.ref094" ref-type="bibr"> 94 </xref> ]. The three genes were homologous to the <italic> Arabidopsis </italic> gene <italic> At5g56860 </italic> that is closely associated with lipid metabolism in green algae [ <xref rid="pone.0154882.ref095" ref-type="bibr"> 95 </xref> ] and is capable of regulating carbon and nitrogen metabolism [ <xref rid="pone.0154882.ref096" ref-type="bibr"> 96 </xref> ]. In this study, <italic> <span class="gene">Glyma15g18380</span> </italic> was validated to be a DEG in high- and low-oil soybean accessions. We deduced that these specific <span class="gene">GATA</span> TFs might affect the accumulation of <span class="trait">seed</span> <span class="trait">oil</span> in dicots. Two genes, <italic> <span class="gene">Glyma09g39570</span> </italic> and <italic> <span class="gene">Glyma10g12130</span> </italic> , are included in category PWY-5035 (gibberellin biosynthesis III) and might be related to the synthesis of the hormone gibberellin. Similarly, five dicot-specific genes, <italic> <span class="gene">Glyma04g02660</span> </italic> , <italic> <span class="gene">Glyma06g02690</span> </italic> , <italic> <span class="gene">Glyma14g40400</span> </italic> , <italic> <span class="gene">Glyma17g37750</span> </italic> and <italic> <span class="gene">Glyma17g37760</span> </italic> in GO:0005975 (carbohydrate metabolism) were annotated as PF02704 (Gibberellin regulated protein). The expression of genes encoding gibberellin regulated proteins is up-regulated by gibberellin [ <xref rid="pone.0154882.ref097" ref-type="bibr"> 97 </xref> ]. It is known that the gibberellin signalling pathway is related to FA <span class="trait">content</span> [ <xref rid="pone.0154882.ref098" ref-type="bibr"> 98 </xref> ]. More importantly, four genes ( <italic> <span class="gene">Glyma04g02660</span> </italic> , <italic> <span class="gene">Glyma06g02690</span> </italic> , <italic> <span class="gene">Glyma17g37750</span> </italic> and <italic> <span class="gene">Glyma17g37760</span> </italic> ) encoding gibberellin regulated proteins were also validated to be DEGs in high- and low-oil soybean accessions. Therefore, the above six gibberellin related genes may play roles in FA synthesis and thereby affect the <span class="trait">oil</span> <span class="trait">content.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Apart from the above specific genes directly participating in acyl-lipid metabolism, we also found that five dicot-specific genes in soybean were putatively involved in the lipid metabolic process (GO:0006629) ( <xref ref-type="table" rid="pone.0154882.t001"> Table 1 </xref> and <xref ref-type="supplementary-material" rid="pone.0154882.s012"> S4 Table </xref> ). Among these genes, <italic> <span class="gene">Glyma02g45680</span> </italic> and <italic> <span class="gene">Glyma14g03130</span> </italic> encoded cytochrome P450, which plays a role in the FA metabolism in plants [ <xref rid="pone.0154882.ref099" ref-type="bibr"> 99 </xref> , <xref rid="pone.0154882.ref100" ref-type="bibr"> 100 </xref> ]); and <italic> <span class="gene">Glyma18g45250</span> </italic> encoded a protein participating in glyceollin biosynthesis, which was related to lipid peroxidation [ <xref rid="pone.0154882.ref101" ref-type="bibr"> 101 </xref> ]. Since these genes were highly expressed and coordinated with genes participating in acyl lipid synthesis, we hypothesized that these genes and their associated pathways possibly play some roles in <span class="trait">seed</span> <span class="trait">oil</span> accumulation.
					</p>
				
					<p>&nbsp;&nbsp;
						Ninety-three dicot-specific genes, including 42 and 27 genes respectively in carbohydrate degradation and transport, were predicted to be candidate genes associated with acyl-lipid metabolism pathways. And seed-specific TF genes <italic> <span class="gene">GmGRF5</span> </italic> , <italic> ABI5 </italic> and <italic> <span class="gene">GmTZF4</span> </italic> were also predicted to play roles in regulating <span class="trait">seed</span> <span class="trait">oil</span> accumulation. Furthermore, <italic> ACCase </italic> , <italic> <span class="gene">DGAT1</span> </italic> , <italic> PP </italic> , <italic> OLEs </italic> and <italic> STEROs </italic> were highly up-regulated not only in specific stages of <span class="trait">seed</span> development but also in high-oil accessions, which indicates that enzymes in initial fatty acid synthesis, downstream of TAG biosynthesis and oil-body formation, played vital roles in the variation of <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span> In particular, highly up-regulated and abundant <italic> OLEs </italic> possibly play vital roles in determining <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span> Most of the above key genes were further confirmed by differential gene expression analysis between high-oil cultivated and low-oil wild soybeans.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Microsomal Omega-3 Fatty Acid Desaturase Genes in Low Linolenic Acid Soybean Line RG10 and Validation of Major Linolenic Acid QTL</h2>
			<h3>PMID:27066063</h3>
				

				
					<h3>PMCID:4810023</h3>
					<h4>期刊:Frontiers in Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Reinprecht </surname>  <given-names> Yarmilla </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/71871/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pauls </surname>  <given-names> K. Peter </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/106404/overview"/>  </contrib>
				<br/> 
				年份：<day> 07 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High levels of linolenic acid (80 g kg(-1)) are associated with the development of off-flavors and poor stability in soybean <span class="trait">oil.</span> The development of low linolenic acid lines such as RG10 (20 g kg(-1) linolenic acid) can reduce these problems. The level of linolenic acid in <span class="trait">seed</span> <span class="trait">oil</span> is determined by the activities of microsomal omega-3 fatty acid desaturases (FAD3). A major linolenic acid QTL (&gt;70% of variation) on linkage group B2 (chromosome <span class="gene">Gm14)</span> was previously detected in a recombinant inbred line population from the RG10 × OX948 cross. The objectives of this study were to validate the major linolenic acid QTL in an independent population and characterize all the soybean <span class="gene">FAD3</span> genes. Four <span class="gene">FAD3</span> genes were sequenced and localized in RG10 and OX948 and compared to the genes in the reference Williams 82 genome. The <span class="gene">FAD3A</span> gene sequences mapped to the locus <span class="gene">Glyma.14g194300</span> [on the chromosome <span class="gene">Gm14</span> (B2)], which is syntenic to the <span class="gene">FAD3B</span> gene (locus <span class="gene">Glyma.02g227200)</span> on the chromosome <span class="gene">Gm02</span> (D1b). The location of the <span class="gene">FAD3A</span> gene is the same as was previously determined for the fan allele, that conditions low linolenic acid <span class="trait">content</span> and several linolenic acid QTL, including Linolen 3-3, mapped previously with the RG10 × OX948 population and confirmed in the PI 361088B × OX948 population as Linolen-PO (FAD3A). The <span class="gene">FAD3B</span> gene-based marker, developed previously, was mapped to the chromosome <span class="gene">Gm02</span> (D1b) in a region containing a newly detected linolenic acid QTL [Linolen-RO(FAD3B)] in the RG10 × OX948 genetic map and corresponds well with the in silico position of the <span class="gene">FAD3B</span> gene sequences. <span class="gene">FAD3C</span> and <span class="gene">FAD3D</span> gene sequences, mapped to syntenic regions on chromosomes <span class="gene">Gm18</span> (locus <span class="gene">Glyma.18g062000)</span> and <span class="gene">Gm11</span> (locus <span class="gene">Glyma.11g227200),</span> respectively. Association of linolenic acid QTL with the desaturase genes <span class="gene">FAD3A</span> and FAD3B, their validation in an independent population, and development of <span class="gene">FAD3</span> gene-specific markers should simplify and accelerate breeding for low linolenic acid soybean cultivars.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						High levels of linolenic acid (80 g kg <sup> &#x02212;1 </sup> ) are associated with the development of off-flavors and poor stability in soybean <span class="trait">oil.</span> The development of low linolenic acid lines such as RG10 (20 g kg <sup> &#x02212;1 </sup> linolenic acid) can reduce these problems. The level of linolenic acid in <span class="trait">seed</span> <span class="trait">oil</span> is determined by the activities of microsomal omega-3 fatty acid desaturases (FAD3). A major linolenic acid QTL (&#x0003e;70% of variation) on linkage group B2 (chromosome <span class="gene">Gm14)</span> was previously detected in a recombinant inbred line population from the RG10 &#x000d7; OX948 cross. The objectives of this study were to validate the major linolenic acid QTL in an independent population and characterize all the soybean <italic> <span class="gene">FAD3</span> </italic> genes. Four <italic> <span class="gene">FAD3</span> </italic> genes were sequenced and localized in RG10 and OX948 and compared to the genes in the reference Williams 82 genome. The <italic> <span class="gene">FAD3A</span> </italic> gene sequences mapped to the locus <span class="gene">Glyma.14g194300</span> [on the chromosome <span class="gene">Gm14</span> (B2)], which is syntenic to the <italic> <span class="gene">FAD3B</span> </italic> gene (locus <span class="gene">Glyma.02g227200)</span> on the chromosome <span class="gene">Gm02</span> (D1b). The location of the <italic> <span class="gene">FAD3A</span> </italic> gene is the same as was previously determined for the <italic> fan </italic> allele, that conditions low linolenic acid <span class="trait">content</span> and several linolenic acid QTL, including Linolen 3-3, mapped previously with the RG10 &#x000d7; OX948 population and confirmed in the PI 361088B &#x000d7; OX948 population as Linolen-PO (FAD3A). The <italic> <span class="gene">FAD3B</span> </italic> gene-based marker, developed previously, was mapped to the chromosome <span class="gene">Gm02</span> (D1b) in a region containing a newly detected linolenic acid QTL [Linolen-RO(FAD3B)] in the RG10 &#x000d7; OX948 genetic map and corresponds well with the <italic> in silico </italic> position of the <italic> <span class="gene">FAD3B</span> </italic> gene sequences. <italic> <span class="gene">FAD3C</span> </italic> and <italic> <span class="gene">FAD3D</span> </italic> gene sequences, mapped to syntenic regions on chromosomes <span class="gene">Gm18</span> (locus <span class="gene">Glyma.18g062000)</span> and <span class="gene">Gm11</span> (locus <span class="gene">Glyma.11g227200),</span> respectively. Association of linolenic acid QTL with the desaturase genes <italic> <span class="gene">FAD3A</span> </italic> and <italic> <span class="gene">FAD3</span> </italic> B, their validation in an independent population, and development of <italic> <span class="gene">FAD3</span> </italic> gene-specific markers should simplify and accelerate breeding for low linolenic acid soybean cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean <span class="trait">seed,</span> linolenic acid is synthesized by consecutive desaturation [the creation of a double bond (at the delta position) by the enzymatic removal of hydrogen from a methylene group in an acyl chain (Shanklin and Cahoon, <xref rid="B56" ref-type="bibr"> 1998 </xref> )] of oleic acid catalyzed by fatty acid desaturases (FADs, Cherif et al., <xref rid="B15" ref-type="bibr"> 1975 </xref> ). Thus, two sets of genes might control linolenic acid <span class="trait">content,</span> which may act in additive, possibly sequential ways. The <italic> <span class="gene">FAD2</span> </italic> genes encode omega (&#x003c9;)-6 FADs and control the conversion of oleic acid into linoleic acid. The <italic> <span class="gene">FAD3</span> </italic> gene family encodes &#x003c9;-3 FADs, which catalyzes transformation of linoleic acid into linolenic acid (Yadav et al., <xref rid="B72" ref-type="bibr"> 1993 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The level of linolenic acid in soybean <span class="trait">seed</span> is determined by three <italic> <span class="gene">FAD3</span> </italic> genes ( <italic> FAD3A, <span class="gene">FAD3B</span> </italic> , and <italic> FAD3C; </italic> Bilyeu et al., <xref rid="B5" ref-type="bibr"> 2003 </xref> ). At least three major loci ( <italic> Fan, Fan2 </italic> , and <italic> Fan3 </italic> ) and potentially a fourth ( <italic> <span class="gene">Fanx</span> </italic> ), have been identified with multiple alleles, including some with the reduced levels of linolenic acid (Rajcan et al., <xref rid="B43" ref-type="bibr"> 2005 </xref> ; Lee et al., <xref rid="B33" ref-type="bibr"> 2007 </xref> ). The <italic> <span class="gene">Fan</span> </italic> locus was mapped between the RFLP markers B124_1 and B194_1 on the linkage group (LG) B2 or chromosome <span class="gene">Gm14</span> (Brummer et al., <xref rid="B10" ref-type="bibr"> 1995 </xref> ). The first <italic> <span class="gene">FAD3</span> </italic> gene was mapped to chromosome <span class="gene">Gm14</span> (or linkage group B2, Byrum et al., <xref rid="B11" ref-type="bibr"> 1995 </xref> ). A reduced level of linolenic acid in A5 was associated with full or partial deletion of a microsomal omega-3 desaturase gene (Byrum et al., <xref rid="B11" ref-type="bibr"> 1995 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic basis of low linolenic acid soybean was associated with mutations in one or more <italic> <span class="gene">FAD3</span> </italic> genes (Bilyeu et al., <xref rid="B5" ref-type="bibr"> 2003 </xref> , <xref rid="B7" ref-type="bibr"> 2005 </xref> , <xref rid="B6" ref-type="bibr"> 2006 </xref> ; Anai et al., <xref rid="B2" ref-type="bibr"> 2005 </xref> ; Chappell and Bilyeu, <xref rid="B12" ref-type="bibr"> 2006 </xref> , <xref rid="B13" ref-type="bibr"> 2007 </xref> ; Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ). For example, PI 361088B is a naturally-occurring mutant line which contains over 4% <span class="trait">seed</span> linolenic acid (Rennie et al., <xref rid="B49" ref-type="bibr"> 1988 </xref> ). A two nucleotide insertion in the coding sequence of <italic> <span class="gene">FAD3A</span> </italic> gene resulted in a frameshift and a premature stop codon (Chappell and Bilyeu, <xref rid="B13" ref-type="bibr"> 2007 </xref> ). The 3% of linolenic acid <span class="trait">content</span> in the <span class="trait">seed</span> <span class="trait">oil</span> of the line 10&#x02013;73 (derived from CX1512- 44) was associated with mutations in <italic> <span class="gene">FAD3A</span> </italic> and <italic> <span class="gene">FAD3C</span> </italic> genes (Bilyeu et al., <xref rid="B7" ref-type="bibr"> 2005 </xref> ). Low linolenic acid line RG10, also contains less than 3% linolenic acid. The line was developed by remutagenesis (Stoj&#x00161;in et al., <xref rid="B61" ref-type="bibr"> 1998 </xref> ) of the 4% linolenic acid line C1640 (Wilcox and Cavins, <xref rid="B66" ref-type="bibr"> 1985 </xref> ). In addition to the <italic> <span class="gene">FAD3A</span> </italic> mutation from C1640 (Chappell and Bilyeu, <xref rid="B12" ref-type="bibr"> 2006 </xref> ), RG10 contains a novel null allele of the <italic> <span class="gene">FAD3B</span> </italic> gene (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ). By combining mutations in all three <italic> <span class="gene">FAD3</span> </italic> genes from an RG10-derived soybean line (RCAT 0716L, <italic> <span class="gene">GmFAD3aabbCC</span> </italic> ) and 10&#x02013;73 ( <italic> <span class="gene">GmFAD3aaBBcc</span> </italic> ), Bilyeu et al. ( <xref rid="B4" ref-type="bibr"> 2011 </xref> ) were able to produce novel soybean lines ( <italic> <span class="gene">GmFAD3aabbcc</span> </italic> ) containing only 1% linolenic acid in the <span class="trait">seed</span> <span class="trait">oil.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						We developed a molecular linkage map with 118 markers and identified QTL for linolenic acid <span class="trait">content</span> and <span class="trait">seed</span> lipoxygenases based on the RG10 &#x000d7; OX948 recombinant inbred line (RIL) population. Based on composite interval mapping (CIM), a major linolenic acid QTL (Linolen 3-3) that accounted for 72&#x02013;78% of linolenic acid variability, was positioned on LG B2 <span class="gene">(Gm14,</span> flanked by SSR marker Satt534 and <italic> <span class="gene">FAD3</span> </italic> gene-specific marker FAD3i6; Reinprecht et al., <xref rid="B46" ref-type="bibr"> 2006 </xref> ; SoyBase). Although strongly expressed in all three Ontario locations (Harrow, Ridgetown, and Woodslee in 2000) this QTL needed to be validated in different genetic backgrounds in order to be useful for marker-assisted selection (MAS) of soybean lines with reduced levels of linolenic acid. QTL regions are generally large and can contain thousands of potential candidate genes for the trait. The most robust marker for a QTL is located in the gene that determines the phenotype. In our previous work, <italic> <span class="gene">FAD3A</span> </italic> gene-based markers were mapped to the region of the major linolenic acid QTL on the chromosome <span class="gene">Gm14</span> (B2) in the RG10 &#x000d7; OX948 population (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Reciprocal crosses were made between the low linolenic acid mutant lines (RG10 and PI 361088B) and OX948 <span class="trait">(seed</span> lipoxygenase triple null line). PI 361088B is a naturally-occurring low linolenic acid source (Rennie et al., <xref rid="B49" ref-type="bibr"> 1988 </xref> ) and line RG10 was developed at the University of Guelph, Ridgetown Campus (Stoj&#x00161;in et al., <xref rid="B61" ref-type="bibr"> 1998 </xref> ) by treating the low linolenic acid line C1640 with ethyl methanesulfonate (EMS). The allele at the <italic> <span class="gene">Fan</span> </italic> locus in RG10 is designated as <italic> fan </italic> -b, for a very low level of linolenic acid (&#x0003c;25 g kg <sup> &#x02212;1 </sup> ). In line PI 361088B the <italic> <span class="gene">Fan</span> </italic> allele is designated as <italic> fan </italic> (PI 361088B) for low level of linolenic acid (~39 g kg <sup> &#x02212;1 </sup> ) (Rennie et al., <xref rid="B49" ref-type="bibr"> 1988 </xref> ). OX948 is a selection from a backcross between Harovinton and a triple lipoxygenase null (3lx) F <sub> 2 </sub> plant, and has high linolenic acid <span class="trait">content</span> (&#x0003e;80 g kg <sup> &#x02212;1 </sup> ). The 3lx genotype, produced by gamma irradiation, was obtained from Dr. Kitamura (National Agricultural Research Centre, Ministry of Agriculture, Forestry and Fisheries, Tsukuba, Japan).
					</p>
				
					<p>&nbsp;&nbsp;
						PCR amplification and sequencing of the <italic> <span class="gene">FAD3</span> </italic> genes are described in Reinprecht et al. ( <xref rid="B44" ref-type="bibr"> 2009 </xref> ). Briefly, sequences for soybean <span class="trait">seed</span> <italic> <span class="gene">FAD3</span> </italic> genes, available in public databases at the initiation of this study [ <italic> G. max </italic> microsomal &#x003c9;-3 FAD mRNAs for <italic> <span class="gene">FAD3A</span> </italic> (GenBank accession <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="AY204710"> AY204710 </ext-link> ), <italic> FAD </italic> 3 <italic> B </italic> (GenBank accession <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="AY204711"> AY204711 </ext-link> ), <italic> FAD </italic> 3 <italic> C </italic> (GenBank accession <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="AY204712"> AY204712 </ext-link> ), reported by Bilyeu et al. ( <xref rid="B5" ref-type="bibr"> 2003 </xref> ) and <italic> <span class="gene">FAD3D</span> </italic> ( <italic> <span class="gene">FAD3-2b</span> </italic> ) (GenBank accession <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="AB188198"> AB188198 </ext-link> , Anai et al., <xref rid="B2" ref-type="bibr"> 2005 </xref> )], were aligned with the <italic> Arabidopsis thaliana </italic> microsomal <italic> <span class="gene">FAD3</span> </italic> gene (GenBank accession <ext-link ext-link-type="DDBJ/EMBL/GenBank" xlink:href="D26508"> D26508 </ext-link> ) sequence (Nishiuchi et al., <xref rid="B38" ref-type="bibr"> 1994 </xref> ) using the multiple sequence alignment program ClustalW (Chenna et al., <xref rid="B14" ref-type="bibr"> 2003 </xref> ), at the European Bioinformatic Institute (EBI, <ext-link ext-link-type="uri" xlink:href="http://www.ebi.ac.uk/clustalw"> http://www.ebi.ac.uk/clustalw </ext-link> ). Based on differences among the alignments, primers to specifically amplify six to nine overlapping fragments from each of the four <italic> <span class="gene">FAD3</span> </italic> genes were designed [using Primer3 (Rozen and Skaletsky, <xref rid="B51" ref-type="bibr"> 2000 </xref> ) and synthesized by Sigma-Aldrich Canada (Oakville, ON, Canada)]. In addition, the 1 kb flanking sequences of <italic> <span class="gene">FAD3A</span> </italic> and <italic> <span class="gene">FAD3B</span> </italic> , and 2 kb of <italic> <span class="gene">FAD3C</span> </italic> and <italic> <span class="gene">FAD3D</span> </italic> were used to amplify the 5&#x02032; and 3&#x02032; untranslated regions (UTR) in <italic> <span class="gene">FAD3</span> </italic> genes. The criteria for primer design included a length of 18 to 34 base pairs (bp), a GC <span class="trait">content</span> of 40 to 60%, fewer than four contiguous identical bases, and a melting temperature between 55 and 65&#x000b0;C. The <italic> <span class="gene">FAD3</span> </italic> gene-specific primers used to amplify overlapping fragments for sequencing are listed in a Table <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the linolenic acid QTL detected with the RG10 &#x000d7; OX948 population, markers on chromosomes <span class="gene">Gm02,</span> <span class="gene">Gm11,</span> <span class="gene">Gm14,</span> and <span class="gene">Gm18</span> were scored for individuals from the PI 361088B &#x000d7; OX948 population. A relatively small set of SSR markers on four chromosomes was shared between the two populations. In the PI 361088B &#x000d7; OX948 population, chromosome <span class="gene">Gm14</span> (B2) contains eight markers, four of which are shared with the mapping RG10 &#x000d7; OX948 population. A major QTL (LOD 8), which explains 59% variability for the linolenic acid <span class="trait">content</span> in the PI 361088B &#x000d7; OX948 population was detected between the markers Satt534 and Satt560. This is the same region as the location of the major linolenic acid QTL Linolen 3-3 (Reinprecht et al., <xref rid="B46" ref-type="bibr"> 2006 </xref> ; SoyBase) in the RG10 &#x000d7; OX948 mapping population (Figure <xref ref-type="fig" rid="F3"> 3A </xref> ) and associated RG10 and OX948 <italic> <span class="gene">FAD3A</span> </italic> gene sequences (locus <span class="gene">Glyma.14g194300).</span>
					</p>
				
					<p>&nbsp;&nbsp;
						A SNP marker FAD3B-i5SNP, developed previously for the mutation in the <italic> <span class="gene">FAD3B</span> </italic> gene (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ), mapped initially to the end of the top portion of the chromosome <span class="gene">Gm02</span> (D1b, 21.4 cM from the SSR marker Sat_135) in the RG10 &#x000d7; OX948 map (not shown). To fine map this region, 21 SSR markers were selected in the region of the <italic> <span class="gene">FAD3B</span> </italic> gene and 12 were placed on the RG10 &#x000d7; OX948 map. The addition of new markers allowed fragments of chromosome <span class="gene">Gm02</span> (D1b) to be joined. The current size of this chromosome-based linkage group in the RG10 &#x000d7; OX948 map is 131.1 cM and it contains 21 markers. The <italic> <span class="gene">FAD3B</span> </italic> gene/marker FAD3B-i5SNP is positioned between the SSR markers Sat_139 (3.9 cM) and Satt172 (11.7 cM) in the region of the newly identified linolenic acid QTL [Linolen-RO(FAD3B), Figure <xref ref-type="fig" rid="F4"> 4A </xref> ] and corresponds well with the <italic> in silico </italic> position of the RG10 and OX948 <italic> <span class="gene">FAD3B</span> </italic> gene sequences at the locus <span class="gene">Glyma.02g227200</span> ( <italic> <span class="gene">FAD3B</span> </italic> ). This QTL (LOD 6.12) explained approximately 10% variability for linolenic acid <span class="trait">content</span> and was consistently expressed in all three Ontario locations (Harrow 8%, Woodslee 9%, and Ridgetown 11%) with an additive effect ranging from &#x02212;9.74 g kg <sup> &#x02212;1 </sup> (Harrow) to &#x02212;11.14 g kg <sup> &#x02212;1 </sup> (Woodslee).
					</p>
				
					<p>&nbsp;&nbsp;
						The addition of a single SSR marker to the chromosome <span class="gene">Gm14</span> (B2) did not change the position of the previously mapped <italic> <span class="gene">FAD3A</span> </italic> gene (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ). This gene was mapped as a <italic> <span class="gene">FAD3A</span> </italic> gene-based marker to the region of a major linolenic acid QTL (Linolen 3-3) on the chromosome <span class="gene">Gm14</span> (B2) in the RG10 &#x000d7; OX948 map (Reinprecht et al., <xref rid="B46" ref-type="bibr"> 2006 </xref> ; SoyBase), which is the same genomic region as the previously mapped <italic> fan </italic> allele that conditions low linolenic acid <span class="trait">content</span> in C1640 (Brummer et al., <xref rid="B10" ref-type="bibr"> 1995 </xref> ). This is also consistent with the previous placement of the microsomal &#x003c9;-3 <italic> FAD </italic> gene on the chromosome <span class="gene">Gm14</span> (B2) (Byrum et al., <xref rid="B11" ref-type="bibr"> 1995 </xref> ) and corresponds well with the <italic> in silico </italic> position of the RG10 and OX948 <italic> <span class="gene">FAD3A</span> </italic> gene sequences (this study).
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, significant work was done on mapping genomic regions associated with the <span class="trait">seed</span> linolenic acid <span class="trait">content.</span> Currently, 44 linolenic acid QTL (identified in 10 independent studies) have been deposited in SoyBase (23 Sep 2015). However, information for several QTL from these studies are unavailable since the markers associated with them were not present on the consensus and/or composite genetic maps. Xie et al. ( <xref rid="B71" ref-type="bibr"> 2012 </xref> ) developed several <italic> <span class="gene">FAD3</span> </italic> gene-based markers and mapped six linolenic acid QTL in a F <sub> 5:7 </sub> population derived from the cross He-Feng 25 &#x000d7; Dongnong L-5 over several environments. However, QTL Linolen 11-3 (marker FAD3b-1) and Linolen 11-4 (marker FAD3c-2) were not reported in SoyBase since these markers are not present on the <span class="gene">GmComposite_2003</span> map. In addition, QTL Linolen 11-2 was associated with SSR marker Satt063 (SoyBase) but not FAD3a-4 marker that was reported originally by Xie et al. ( <xref rid="B71" ref-type="bibr"> 2012 </xref> ). In the current study, sequences of these markers were BLASTed against soybean genome, but because of absence of any significant hits they were not placed on the sequence map. There are two additional linolenic acid QTL studies not cataloged in SoyBase. Brummer et al. ( <xref rid="B10" ref-type="bibr"> 1995 </xref> ) mapped the <italic> <span class="gene">Fan</span> </italic> locus between RFLP markers B124_1 and B194_1 on linkage group B2 <span class="gene">(Gm14)</span> in an F <sub> 2 </sub> population developed from the <italic> G. max </italic> (C1694) &#x000d7; <italic> G. soja </italic> (PI 479750) cross. The QTL at this position explained most of the variation (85%) for the linolenic acid <span class="trait">content</span> in that population. Recently, Akond et al. ( <xref rid="B1" ref-type="bibr"> 2014 </xref> ) identified six QTL associated with the levels of linolenic acid in an F <sub> 5:7 </sub> population developed from the MD96-5722 &#x000d7; Spencer cross. These QTL were mapped on four chromosomes <span class="gene">(Gm13,</span> <span class="gene">Gm14,</span> <span class="gene">Gm15,</span> and <span class="gene">Gm16)</span> and explained 5 to 23% variability for the linolenic acid <span class="trait">content</span> in this population. However, linolenic acid QTL reported in both studies as well as three linolenic acid QTL (Linolen 8-1, Linolen 8-2, and Linolen 8-3; SoyBase) reported by Kim et al. ( <xref rid="B29" ref-type="bibr"> 2010 </xref> ) should be considered as putative QTL since their identification was based on a single environment data.
					</p>
				
					<p>&nbsp;&nbsp;
						Although numerous QTL for different traits were identified in soybean, very few of them were verified in independent populations. For example, QTL for soybean cyst nematode <span class="trait">resistance,</span> initially identified in an F <sub> 2:3 </sub> population, were confirmed in F <sub> 6:7 </sub> RILs developed from the same Magellan (susceptible) &#x000d7; PI 438489B (resistant) cross (Vuong et al., <xref rid="B63" ref-type="bibr"> 2011 </xref> ), while QTL for <span class="trait">seed</span> weight, protein and <span class="trait">oil</span> contents were confirmed in different genetic backgrounds by two research groups (Fasoula et al., <xref rid="B19" ref-type="bibr"> 2004 </xref> ; Pathan et al., <xref rid="B41" ref-type="bibr"> 2013 </xref> ). Seven QTL for linolenic acid were mapped on the chromosome <span class="gene">Gm14</span> (B2) in five different populations by four research groups (SoyBase). Four of these QTL (Linolen 10-1, Linolen 10-2, Linolem 11-2) were identified in the region (86.3&#x02013;98.9 cM, <span class="gene">GmComposite_2003)</span> of the <italic> <span class="gene">Fan</span> </italic> locus. The major linolenic acid QTL (Linolen 3-3) was detected in all three testing locations (Harrow, Ridgetown, and Woodslee, ON) and explained very similar amounts of the total phenotypic variability ( <italic> R </italic> <sup> 2 </sup> = 0.76) for linolenic acid <span class="trait">content</span> in the RG10 &#x000d7; OX948 mapping population (Reinprecht et al., <xref rid="B46" ref-type="bibr"> 2006 </xref> ). This is an example of the most important type of QTL for breeders because it was associated with the <italic> <span class="gene">FAD3A</span> </italic> gene (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ) and confirmed in the PI 361088B &#x000d7; OX948 population ( <italic> R </italic> <sup> 2 </sup> = 0.59), which may lead to the stability in breeding for low linolenic acid trait. However, because of the small size of the PI 361088B &#x000d7; OX948 confirmation population ( <italic> N </italic> = 45) additional work is required to verify that. In this study, we identified a new QTL associated with linolenic acid <span class="trait">content</span> in the RG10 &#x000d7; OX948 population. It mapped to the chromosome <span class="gene">Gm02</span> (D1b), which explained ~10% variation for the trait and was associated with the <italic> <span class="gene">FAD3B</span> </italic> gene. The usefulness of the previously developed <italic> <span class="gene">FAD3A</span> </italic> and <italic> <span class="gene">FAD3B</span> </italic> gene mutation-based markers was confirmed with a number of RG10-derived low linolenic acid lines and cultivars with normal level of linolenic acid (Reinprecht et al., <xref rid="B44" ref-type="bibr"> 2009 </xref> ). In the current work, we sequenced all four microsomal <italic> <span class="gene">FAD3</span> </italic> genes and identified numerous variations between parental lines of mapping population (RG10 and OX948) as well as reference Wm82. In the future, these polymorphisms could be used to develop additional <italic> <span class="gene">FAD3</span> </italic> gene-based markers. To be useful for the soybean breeders globally, these markers need to be validated in diverse germplasm.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Gamma radiation and magnetic field mediated delay in effect of accelerated ageing of soybean.</h2>
			<h3>PMID:26243899</h3>
				

				
			<p>
				作者：Kumar M(1), Singh B(2), Ahuja S(2), Dahuja A(3), Anand A(4).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean seeds were exposed to gamma radiation (0.5, 1, 3 and 5 kGy), static magnetic field (50, 100 and 200 mT) and a combination of gamma radiation and magnetic energy (0.5 kGy + 200 mT and 5 kGy + 50 mT) and stored at room temperature for six months. These seeds were later subjected to accelerated ageing treatment at 42 °C temperature and 95-100 % relative humidity and were compared for various physical and biochemical characteristics between the untreated and the energized treatments. Energy treatment protected the quality of stored seeds in terms of its protein and <span class="trait">oil</span> <span class="trait">content</span> . Accelerated aging conditions, however, affected the <span class="trait">oil</span> and protein quantity and quality of <span class="trait">seed</span> negatively. Antioxidant enzymes exhibited a decline in their activity during aging while the <span class="gene">LOX</span> activity, which reflects the rate of lipid peroxidation, in general, increased during the aging. Gamma irradiated (3 and 5 kGy) and magnetic field treated seeds (100 and 200 mT) maintained a higher catalase and ascorbate peroxidase activity which may help in efficient scavenging of deleterious free radical produced during the aging. Aging caused peroxidative changes to lipids, which could be contributed to the loss of <span class="trait">oil</span> quality. Among the electromagnetic energy treatments, a dose of 1-5 kGy of gamma and 100 mT, 200 mT magnetic field effectively slowed the rate of biochemical degradation and loss of cellular integrity in seeds stored under conditions of accelerated aging and thus, protected the deterioration of <span class="trait">seed</span> quality. Energy combination treatments did not <span class="trait">yield</span> any additional protection advantage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel <span class="gene">GmFAD3-2a</span> mutant allele developed through TILLING reduces α-linolenic acid <span class="trait">content</span> in soybean <span class="trait">seed</span> <span class="trait">oil.</span></h2>
			<h3>PMID:25914592</h3>
				

				
			<p>
				作者：Hoshino T(1), Watanabe S(2), Takagi Y(2), Anai T(2).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) <span class="trait">oil</span> typically contains 8% α-linolenic acid that is highly unstable and easily oxidized. This property is undesirable in many food and industrial applications. Genetic strategies for reducing α-linolenic acid <span class="trait">content</span> would enhance the commercial value. However, genetic resources for low α-linolenic acid <span class="trait">content</span> are limited among natural soybean variations. Microsomal omega-3-fatty acid desaturase (FAD3) is responsible for the synthesis of α-linolenic acid in the polyunsaturated fatty acid pathway. There are four <span class="gene">FAD3</span> homologs <span class="gene">(Glyma02g39230,</span> <span class="gene">Glyma11g27190,</span> <span class="gene">Glyma14g37350</span> and <span class="gene">Glyma18g06950)</span> in the soybean genome. While non-functional alleles have been reported for <span class="gene">Glyma02g39230</span> <span class="gene">(GmFAD3-1a)</span> and <span class="gene">Glyma14g37350</span> <span class="gene">(GmFAD3-1b),</span> little variation is seen in <span class="gene">Glyma18g06950</span> <span class="gene">(GmFAD3-2a).</span> We isolated seven mutant <span class="gene">GmFAD3-2a</span> alleles, each containing a single-nucleotide substitution, from 39,100 independent mutant lines by using targeting induced local lesions in genomes (TILLING). Analysis of <span class="gene">GmFAD3-2a</span> transcripts and enzyme activities revealed that one missense mutant, 'P1-A9', contains a non-functional allele of <span class="gene">GmFAD3-2a.</span> By combining three non-functional alleles <span class="gene">(GmFAD3-1a,</span> <span class="gene">GmFAD3-1b,</span> and <span class="gene">GmFAD3-2a),</span> we generated soybean lines containing &lt;2% α-linolenic acid in their seeds. The reverse-genetics-based development of novel mutant alleles in the fatty acid metabolic pathway will allow the improvement of soybean with better <span class="trait">oil</span> quality through conventional breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Metabolic engineering of oilseed crops to produce high levels of novel acetyl glyceride oils with reduced viscosity, freezing point and calorific value.</h2>
			<h3>PMID:25756355  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu J(1), Rice A(1), McGlew K(1), Shaw V(1), Park H(2), Clemente T(2), Pollard M(1), Ohlrogge J(1), Durrett TP(3).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> oils have proved recalcitrant to modification for the production of industrially useful lipids. Here, we demonstrate the successful metabolic engineering and subsequent field production of an oilseed crop with the highest accumulation of unusual <span class="trait">oil</span> achieved so far in transgenic plants. Previously, expression of the Euonymus alatus diacylglycerol acetyltransferase (EaDAcT) gene in wild-type Arabidopsis seeds resulted in the accumulation of 45 mol% of unusual 3-acetyl-1,2-diacyl-sn-glycerols (acetyl-TAGs) in the <span class="trait">seed</span> <span class="trait">oil</span> (Durrett et al., 2010 PNAS 107:9464). Expression of EaDAcT in dgat1 mutants compromised in their ability to synthesize regular triacylglycerols increased acetyl-TAGs to 65 mol%. Camelina and soybean transformed with the EaDAcT gene accumulate acetyl-triacylglycerols (acetyl-TAGs) at up to 70 mol% of <span class="trait">seed</span> <span class="trait">oil.</span> A similar strategy of coexpression of EaDAcT together with RNAi suppression of <span class="gene">DGAT1</span> increased acetyl-TAG levels to up to 85 mol% in field-grown transgenic Camelina. Additionally, total moles of triacylglycerol (TAG) per <span class="trait">seed</span> increased 20%. Analysis of the acetyl-TAG fraction revealed a twofold reduction in very long chain fatty acids (VLCFA), consistent with their displacement from the sn-3 position by acetate. <span class="trait">Seed</span> germination remained high, and seedlings were able to metabolize the stored acetyl-TAGs as rapidly as regular triacylglycerols. Viscosity, freezing point and caloric <span class="trait">content</span> of the Camelina acetyl-TAG oils were reduced, enabling use of this <span class="trait">oil</span> in several nonfood and food applications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soya bean seeds accumulating β-carotene exhibit the collateral enhancements of oleate and protein <span class="trait">content</span> traits.</h2>
			<h3>PMID:25400247  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schmidt MA(1), Parrott WA, Hildebrand DF, Berg RH, Cooksey A, Pendarvis K, He Y, McCarthy F, Herman EM.
				<br/> 
				年份：2015 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Transgenic soya bean (Glycine max) plants overexpressing a seed-specific bacterial phytoene synthase gene from Pantoea ananatis modified to target to plastids accumulated 845 μg β carotene g(-1) dry <span class="trait">seed</span> weight with a desirable 12:1 ratio of β to α. The β carotene accumulating seeds exhibited a shift in <span class="trait">oil</span> composition increasing oleic acid with a concomitant decrease in linoleic acid and an increase in <span class="trait">seed</span> protein <span class="trait">content</span> by at least 4% (w/w). Elevated β-carotene accumulating soya bean cotyledons contain 40% the amount of abscisic acid compared to nontransgenic cotyledons. Proteomic and nontargeted metabolomic analysis of the mid-maturation β-carotene cotyledons compared to the nontransgenic did not reveal any significant differences that would account for the altered phenotypes of both elevated oleate and protein <span class="trait">content.</span> Transcriptomic analysis, confirmed by RT-PCR, revealed a number of significant differences in ABA-responsive transcripton factor gene expression in the <span class="gene">crtB</span> transgenics compared to nontransgenic cotyledons of the same maturation stage. The altered <span class="trait">seed</span> composition traits seem to be attributed to altered ABA hormone levels varying transcription factor expression. The elevated β-carotene, oleic acid and protein traits in the β-carotene soya beans confer a substantial additive nutritional quality to soya beans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Changes in oleic Acid <span class="trait">content</span> of transgenic soybeans by antisense RNA mediated posttranscriptional gene silencing.</h2>
			<h3>PMID:25197629</h3>
				

				
			<p>
				作者：Zhang L(1), Yang XD(1), Zhang YY(1), Yang J(1), Qi GX(1), Guo DQ(1), Xing GJ(1), Yao Y(1), Xu WJ(1), Li HY(1), Li QY(1), Dong YS(1).
				<br/> 
				年份：2014
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Delta-12 oleate desaturase gene (FAD2-1), which converts oleic acid into linoleic acid, is the key enzyme determining the fatty acid composition of <span class="trait">seed</span> <span class="trait">oil.</span> In this study, we inhibited the expression of endogenous Delta-12 oleate desaturase <span class="gene">GmFad2-1b</span> gene by using antisense RNA in soybean Williams 82. By employing the soybean cotyledonary-node method, a part of the cDNA of soybean <span class="gene">GmFad2-1b</span> 801 bp was cloned for the construction of a pCAMBIA3300 vector under the soybean <span class="trait">seed</span> promoter BCSP. Leaf painting, LibertyLink strip, PCR, Southern blot, qRT-PCR, and fatty acid analysis were used to detect the insertion and expression of <span class="gene">GmFad2-1b</span> in the transgenic soybean lines. The results indicate that the metabolically engineered plants exhibited a significant increase in oleic acid (up to 51.71%) and a reduction in palmitic acid (to &lt;3%) in their <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span> No structural differences were observed between the fatty acids of the transgenic and the nontransgenic <span class="trait">oil</span> extracts.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Deletions of the <span class="gene">SACPD-C</span> locus elevate <span class="trait">seed</span> stearic acid levels but also result in fatty acid and morphological alterations in nitrogen fixing nodules.</h2>
			<h3>PMID:24886084  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gillman JD(1), Stacey MG, Cui Y, Berg HR, Stacey G.
				<br/> 
				年份：2014 May 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max) seeds are the primary source of edible <span class="trait">oil</span> in the United States. Despite its widespread utility, soybean <span class="trait">oil</span> is oxidatively unstable. Until recently, the majority of soybean <span class="trait">oil</span> underwent chemical hydrogenation, a process which also generates trans fats. An alternative to chemical hydrogenation is genetic modification of <span class="trait">seed</span> <span class="trait">oil</span> through identification and introgression of mutant alleles. One target for improvement is the elevation of a saturated fat with no negative cardiovascular impacts, stearic acid, which typically constitutes a minute portion of <span class="trait">seed</span> <span class="trait">oil</span> (~3%). RESULTS: We examined radiation induced soybean mutants with moderately increased stearic acid (10-15% of <span class="trait">seed</span> <span class="trait">oil,</span> ~3-5 X the levels in wild-type soybean seeds) via comparative whole genome hybridization and genetic analysis. The deletion of one <span class="gene">SACPD</span> isoform encoding gene (SACPD-C) was perfectly correlated with moderate elevation of <span class="trait">seed</span> stearic acid <span class="trait">content.</span> However, <span class="gene">SACPD-C</span> deletion lines were also found to have altered nodule fatty acid composition and grossly altered morphology. Despite these defects, overall nodule accumulation and nitrogen fixation were unaffected, at least under laboratory conditions. CONCLUSIONS: Although no <span class="trait">yield</span> penalty has been reported for moderate elevated <span class="trait">seed</span> stearic acid <span class="trait">content</span> in soybean seeds, our results demonstrate that genetic alteration of <span class="trait">seed</span> traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and <span class="gene">SACPD</span> activity in particular, in the establishment and maintenance of symbiotic nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mutations in <span class="gene">SACPD-C</span> result in a range of elevated stearic acid concentration in soybean <span class="trait">seed.</span></h2>
			<h3>PMID:24846334  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carrero-Colón M(1), Abshire N(2), Sweeney D(2), Gaskin E(2), Hudson K(1).
				<br/> 
				年份：2014 May 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">oil</span> has a wide variety of uses, and stearic acid, which is a relatively minor component of soybean <span class="trait">oil</span> is increasingly desired for both industrial and food applications. New soybean mutants containing high levels of the saturated fatty acid stearate in seeds were recently identified from a chemically mutagenized population. Six mutants ranged in stearate <span class="trait">content</span> from 6-14% stearic acid, which is 1.5 to 3 times the levels contained in wild-type <span class="trait">seed</span> of the Williams 82 cultivar. Candidate gene sequencing revealed that all of these lines carried amino acid substitutions in the gene encoding the delta-9-stearoyl-acyl-carrier protein desaturase enzyme (SACPD-C) required for the conversion of stearic acid to oleic acid. Five of these missense mutations were in highly conserved residues clustered around the predicted di-iron center of the <span class="gene">SACPD-C</span> enzyme. Co-segregation analysis demonstrated a positive association of the elevated stearate trait with the <span class="gene">SACPD-C</span> mutation for three populations. These missense mutations may provide additional alleles that may be used in the development of new soybean cultivars with increased levels of stearic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of transcript polymorphisms in soybean lines varying in <span class="trait">oil</span> composition and <span class="trait">content.</span></h2>
			<h3>PMID:24755115  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Goettel W, Xia E, Upchurch R, Wang ML, Chen P, An YQ(1).
				<br/> 
				年份：2014 Apr 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Variation in <span class="trait">seed</span> <span class="trait">oil</span> composition and <span class="trait">content</span> among soybean varieties is largely attributed to differences in transcript sequences and/or transcript accumulation of <span class="trait">oil</span> production related genes in seeds. Discovery and analysis of sequence and expression variations in these genes will accelerate soybean <span class="trait">oil</span> quality improvement. RESULTS: In an effort to identify these variations, we sequenced the transcriptomes of soybean seeds from nine lines varying in <span class="trait">oil</span> composition and/or total <span class="trait">oil</span> <span class="trait">content.</span> Our results showed that 69,338 distinct transcripts from 32,885 annotated genes were expressed in seeds. A total of 8,037 transcript expression polymorphisms and 50,485 transcript sequence polymorphisms (48,792 SNPs and 1,693 small Indels) were identified among the lines. Effects of the transcript polymorphisms on their encoded protein sequences and functions were predicted. The studies also provided independent evidence that the lack of <span class="gene">FAD2-1A</span> gene activity and a non-synonymous SNP in the coding sequence of FAB2C caused elevated oleic acid and stearic acid levels in soybean lines M23 and FAM94-41, respectively. CONCLUSIONS: As a proof-of-concept, we developed an integrated RNA-seq and bioinformatics approach to identify and functionally annotate transcript polymorphisms, and demonstrated its high effectiveness for discovery of genetic and transcript variations that result in altered <span class="trait">oil</span> quality traits. The collection of transcript polymorphisms coupled with their predicted functional effects will be a valuable asset for further discovery of genes, gene variants, and functional markers to improve soybean <span class="trait">oil</span> quality.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mass spectrometry characterisation of fatty acids from metabolically engineered soybean seeds.</h2>
			<h3>PMID:24652150  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Murad AM(1), Vianna GR, Machado AM, da Cunha NB, Coelho CM, Lacerda VA, Coelho MC, Rech EL.
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Improving the quality and performance of soybean <span class="trait">oil</span> as biodiesel depends on the chemical composition of its fatty acids and requires an increase in monounsaturated acids and a reduction in polyunsaturated acids. Despite its current use as a source of biofuel, soybean <span class="trait">oil</span> contains an average of 25 % oleic acid and 13 % palmitic acid, which negatively impacts its oxidative stability and freezing point, causing a high rate of nitrogen oxide emission. Gas chromatography and ion mobility mass spectrometry were conducted on soybean fatty acids from metabolically engineered <span class="trait">seed</span> extracts to determine the nature of the structural oleic and palmitic acids. The soybean genes <span class="gene">FAD2-1</span> and FatB were placed under the control of the 35SCaMV constitutive promoter, introduced to soybean embryonic axes by particle bombardment and down-regulated using RNA interference technology. Results indicate that the metabolically engineered plants exhibited a significant increase in oleic acid (up to 94.58 %) and a reduction in palmitic acid (to &lt;3 %) in their <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content.</span> No structural differences were observed between the fatty acids of the transgenic and non-transgenic <span class="trait">oil</span> extracts.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:BnWRI1 coordinates fatty acid biosynthesis and photosynthesis pathways during <span class="trait">oil</span> accumulation in rapeseed.</h2>
			<h3>PMID:24393360  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wu XL(1), Liu ZH, Hu ZH, Huang RZ.
				<br/> 
				年份：2014 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosynthesis in "green" seeds, such as rapeseed, soybean, and Arabidopsis, plays a substantial role in the improved efficiency of <span class="trait">oil</span> accumulation. However, the molecular mechanism underpinning the coordinated expression of fatty acid (FA) biosynthesis- and photosynthesis-related genes in such developing seeds remains to be elucidated. Here, we found that seed-specific overexpression of BnWRI1, a <span class="gene">WRI1</span> homolog from rapeseed (Brassica napus cv. ZGY2), results in enhanced chlorophyll <span class="trait">content</span> in developing seeds and increased <span class="trait">oil</span> <span class="trait">content</span> and <span class="trait">seed</span> mass in matured seeds. BnWRI1 was co-expressed with BnBCCP and BnCAB, two marker genes of FA biosynthesis and photosynthesis during <span class="trait">seed</span> development, respectively. Overexpression of BnWRI1 increased expression of both marker genes. Further, the nuclear-localized BnWRI1 protein was found to act as a transcription activator. It could bind to the GT1-element and/or GCC-box, which are widespread in the upstream regions of genes involved in FA biosynthesis and photosynthesis pathways. Accordingly, BnWRI1 could interact with promoters of BCCP2 and LHB1B2 in vivo. These results suggested that BnWRI1 may coordinate FA biosynthesis and photosynthesis pathways in developing seeds via directly stimulating expression of GT1-element and/or GCC-box containing genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of Δ9-stearoyl-ACP-desaturase-C mutants in a high oleic background on soybean <span class="trait">seed</span> <span class="trait">oil</span> composition.</h2>
			<h3>PMID:24193357  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ruddle P 2nd(1), Whetten R, Cardinal A, Upchurch RG, Miranda L.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Two new sources of elevated <span class="trait">seed</span> stearic acid were identified and the feasibility of an elevated stearic acid, high oleic acid germplasm was studied. Soybean [Glycine max (L.) Merr.] <span class="trait">oil</span> typically contains 2-4% stearic acid. <span class="trait">Oil</span> with at least 20% stearic acid is desirable because of its improved baking properties and health profile. This study identifies two new sources of high stearic acid and evaluates the interaction of high stearic and oleic acid alleles. TCHM08-1087 and TCHM08-755, high stearic acid 'Holladay' mutants, were crossed to FAM94-41-3, a line containing a point mutation in a seed-specific isoform of a Δ9-stearoyl-acyl carrier protein-desaturase (SACPD-C). F2-derived lines were evaluated for fatty acid <span class="trait">content</span> in four field environments. Sequencing of SACPDs in TCHM08-1087 and TCHM08-755 revealed distinct deletions of at least one megabase encompassing <span class="gene">SACPD-C</span> in both lines. After genotyping, the additive effect for stearic acid was estimated at +1.8% for the <span class="gene">SACPD-C</span> point mutation and +4.1% for the <span class="gene">SACPD-C</span> deletions. Average stearic acid in lines homozygous for the deletions was 12.2%. A FAM94-41-3-derived line and TCHM08-1087-11, a selection from TCHM08-1087, were crossed to S09-2902-145, a line containing missense mutations in two fatty acid desaturases (FAD2-1A and FAD2-1B). F1 plants were grown in a greenhouse and individual F2 <span class="trait">seed</span> were genotyped and phenotyped. No interaction was observed between either <span class="gene">FAD2-1A</span> or <span class="gene">FAD2-1B</span> and any of the <span class="gene">SACPD-C</span> mutant alleles. <span class="trait">Seed</span> homozygous mutant for SACPD-C/FAD2-1A/FAD2-1B contained 12.7% stearic acid and 65.5% oleic acid while <span class="trait">seed</span> homozygous for the <span class="gene">SACPD-C</span> deletion and mutant for <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> averaged 10.4% stearic acid and 75.9% oleic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping the low palmitate fap1 mutation and validation of its effects in soybean <span class="trait">oil</span> and agronomic traits in three soybean populations.</h2>
			<h3>PMID:24132738  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cardinal AJ(1), Whetten R, Wang S, Auclair J, Hyten D, Cregan P, Bachlava E, Gillman J, Ramirez M, Dewey R, Upchurch G, Miranda L, Burton JW.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: fap 1 mutation is caused by a G174A change in <span class="gene">GmKASIIIA</span> that disrupts a donor splice site recognition and creates a GATCTG motif that enhanced its expression. Soybean <span class="trait">oil</span> with reduced palmitic acid <span class="trait">content</span> is desirable to reduce the health risks associated with consumption of this fatty acid. The objectives of this study were: to identify the genomic location of the reduced palmitate fap1 mutation, determine its molecular basis, estimate the amount of phenotypic variation in fatty acid composition explained by this locus, determine if there are epistatic interactions between the fap1 and fap nc loci and, determine if the fap1 mutation has pleiotropic effects on <span class="trait">seed</span> <span class="trait">yield,</span> <span class="trait">oil</span> and protein <span class="trait">content</span> in three soybean populations. This study detected two major QTL for 16:0 <span class="trait">content</span> located in chromosome 5 <span class="gene">(GmFATB1a,</span> fap nc) and chromosome 9 near BARCSOYSSR_09_1707 that explained, with their interaction, 66-94 % of the variation in 16:0 <span class="trait">content</span> in the three populations. Sequencing results of a putative candidate gene, <span class="gene">GmKASIIIA,</span> revealed a single unique polymorphism in the germplasm line C1726, which was predicted to disrupt the donor splice site recognition between exon one and intron one and produce a truncated KASIIIA protein. This G to A change also created the GATCTG motif that enhanced gene expression of the mutated <span class="gene">GmKASIIIA</span> gene. Lines homozygous for the <span class="gene">GmKASIIIA</span> mutation (fap1) had a significant reduction in 16:0, 18:0, and <span class="trait">oil</span> <span class="trait">content;</span> and an increase in unsaturated fatty acids <span class="trait">content.</span> There were significant epistatic interactions between <span class="gene">GmKASIIIA</span> (fap1) and fap nc for 16:0 and <span class="trait">oil</span> contents, and <span class="trait">seed</span> <span class="trait">yield</span> in two populations. In conclusion, the fap1 phenotype is caused by a single unique SNP in the <span class="gene">GmKASIIIA</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of genetically modified soybean in crude soybean <span class="trait">oil.</span></h2>
			<h3>PMID:24128586  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nikolić Z(1), Vasiljević I, Zdjelar G, Ðorđević V, Ignjatov M, Jovičić D, Milošević D.
				<br/> 
				年份：2014 Feb 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In order to detect presence and quantity of Roundup Ready (RR) soybean in crude <span class="trait">oil</span> extracted from soybean <span class="trait">seed</span> with a different percentage of <span class="gene">GMO</span> <span class="trait">seed</span> two extraction methods were used, CTAB and DNeasy Plant Mini Kit. The amplifications of lectin gene, used to check the presence of soybean DNA, were not achieved in all CTAB extracts of DNA, while commercial kit gave satisfactory results. Comparing actual and estimated <span class="gene">GMO</span> <span class="trait">content</span> between two extraction methods, root mean square deviation for kit is 0.208 and for CTAB is 2.127, clearly demonstrated superiority of kit over CTAB extraction. The results of quantification evidently showed that if the <span class="trait">oil</span> samples originate from soybean <span class="trait">seed</span> with varying percentage of RR, it is possible to monitor the <span class="gene">GMO</span> <span class="trait">content</span> at the first stage of processing crude <span class="trait">oil.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">GmbZIP123</span> gene enhances lipid <span class="trait">content</span> in the seeds of transgenic Arabidopsis plants.</h2>
			<h3>PMID:23963672  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song QX(1), Li QT, Liu YF, Zhang FX, Ma B, Zhang WK, Man WQ, Du WG, Wang GD, Chen SY, Zhang JS.
				<br/> 
				年份：2013 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is one of most important <span class="trait">oil</span> crops and a significant increase in lipid <span class="trait">content</span> in soybean seeds would facilitate vegetable <span class="trait">oil</span> production in the world. Although the pathways for lipid biosynthesis in higher plants have been uncovered, our understanding of regulatory mechanism controlling lipid accumulation is still limited. In this study, we identified 87 transcription factor genes with a higher abundance at the stage of lipid accumulation in soybean seeds. One of these genes, <span class="gene">GmbZIP123,</span> was selected to further study its function in regulation of lipid accumulation. Overexpression of <span class="gene">GmbZIP123</span> enhanced lipid <span class="trait">content</span> in the seeds of transgenic Arabidopsis thaliana plants. The <span class="gene">GmbZIP123</span> transgene promoted expression of two sucrose transporter genes (SUC1 and SUC5) and three cell-wall invertase genes (cwINV1, cwINV3, and cwINV6) by binding directly to the promoters of these genes. Consistently, the cell-wall invertase activity and sugar translocation were all enhanced in siliques of <span class="gene">GmbZIP123</span> transgenic plants. Higher levels of glucose, fructose, and sucrose were also found in seeds of <span class="gene">GmbZIP123</span> transgenic plants. These results suggest that <span class="gene">GmbZIP123</span> may participate in regulation of lipid accumulation in soybean seeds by controlling sugar transport into seeds from photoautotrophic tissues. This study provides novel insights into the regulatory mechanism for lipid accumulation in seeds and may facilitate improvements in <span class="trait">oil</span> production in soybean and other <span class="trait">oil</span> crops through genetic manipulation of the <span class="gene">GmbZIP123</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of β-expansin gene <span class="gene">GmEXPB2</span> improves phosphorus efficiency in soybean.</h2>
			<h3>PMID:23773128  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou J(1), Xie J, Liao H, Wang X.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is an important <span class="trait">oil</span> crop in agricultural production, but low phosphorus (P) availability limits soybean growth and production. Expansin is a family of plant cell wall proteins and involved in a variety of physiological processes, including cell division and enlargement, root growth and leaf development. To test the potential effects of expansins on crop production, we have developed soybean transgenic plants overexpressing a soybean β-expansin gene <span class="gene">GmEXPB2,</span> which was significantly induced by phosphate (Pi) starvation. The results indicated that constitutive overexpression of <span class="gene">GmEXPB2</span> promoted leaf expansion, sequentially stimulated root growth and consequently resulted in improved P efficiency in the transgenic plants under P-limited conditions in hydroponics. In particular, when tested in calcareous (CS) and acid soils (AS), the two <span class="gene">GmEXPB2</span> transgenic soybean lines showed above 25 and 40% increases in plant dry weight and P <span class="trait">content,</span> respectively to wild-type plants in low-P CS, but not in AS. To our knowledge, this is the first report in which improvement of P efficiency could be achieved through constitutive overexpression of an endogenous EXPB gene in soybean. These findings suggest that genetic modification of root and leaf traits might be a suitable strategy for improving crop production in low-P soils.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Classification of distinct <span class="trait">seed</span> carbohydrate profiles in soybean.</h2>
			<h3>PMID:23317449  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hagely KB(1), Palmquist D, Bilyeu KD.
				<br/> 
				年份：2013 Feb 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybeans are an important source of protein-rich meal for livestock feed formulations. Recent changes in the cost of commodity-based sources of metabolizable energy (ME) inputs has put pressure on soybean meal to deliver both protein and ME in feed formulations. The non-oil fraction of soybean contains approximately 12% soluble carbohydrates, principally sucrose, raffinose, and stachyose. Of these carbohydrates, only sucrose is positive for ME. Both raffinose and stachyose, belonging to the raffinose family of oligosaccharides (RFOs), are considered antinutritional because of the negative consequences of their fermentation in the gut of monogastric animals when RFOs are consumed in the diet. Therefore, there is an interest in improving soybean <span class="trait">seed</span> composition so that it contains higher ME and fewer antinutritional components by increasing the sucrose <span class="trait">content</span> while lowering the RFOs. Several soybean lines have been discovered that contain altered levels of RFOs, and recent molecular genetic investigations have shown the phenotype to be caused by mutations in a raffinose synthase 2 (RS2) gene encoding the enzyme that is the committed step for RFO biosynthesis. The objective of this research was to determine the variation in carbohydrate profile for different soybean lines grown in a single location containing one of three different alleles of the <span class="gene">RS2</span> gene. The results indicate that, although there is variation in the carbohydrate profiles for each line, different lines with the same <span class="gene">RS2</span> genotype tend to produce a characteristic carbohydrate profile. Although the carbohydrate profile for each <span class="gene">RS2</span> genotype class was consistent in different genetic backgrounds under two conditions grown at one location, more research will be necessary to determine the environmental stability of the carbohydrate profiles in multiple locations over different years.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of two high-palmitic-acid mutant loci induced by X-ray irradiation in soybean.</h2>
			<h3>PMID:23136502</h3>
				

				
			<p>
				作者：Anai T(1), Hoshino T, Imai N, Takagi Y.
				<br/> 
				年份：2012 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Palmitic acid is the most abundant (approx. 11% of total fatty acids) saturated fatty acid in conventional soybean <span class="trait">seed</span> <span class="trait">oil.</span> Increasing the saturated acid <span class="trait">content</span> of soybean <span class="trait">oil</span> improves its oxidative stability and plasticity. We have developed three soybean mutants with high palmitic acid <span class="trait">content</span> by X-ray irradiation. In this study, we successfully identified the mutated sites of two of these high-palmitic-acid mutants, J10 and M22. PCR-based mutant analysis revealed that J10 has a 206,203-bp-long deletion that includes the <span class="gene">GmKASIIA</span> gene and 16 other predicted genes, and M22 has a 26-bp-long deletion in the sixth intron of <span class="gene">GmKASIIB.</span> The small deletion in M22 causes mis-splicing of <span class="gene">GmKASIIB</span> transcripts, which should result in nonfunctional products. In addition, we designed co-dominant marker sets for these mutant alleles and confirmed the association of genotypes and palmitic acid contents in F(2) seeds of J10 X M22. This information will be useful in breeding programs to develop novel soybean cultivars with improved palmitic acid <span class="trait">content.</span> However, in the third mutant, KK7, we found no polymorphism in either <span class="gene">GmKASIIA</span> or <span class="gene">GmKASIIB,</span> which suggests that several unknown genes in addition to <span class="gene">GmKASIIA</span> and <span class="gene">GmKASIIB</span> may be involved in elevating the palmitic acid <span class="trait">content</span> of soybean <span class="trait">seed</span> <span class="trait">oil.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of the quantitative trait loci (QTL) underlying water soluble protein <span class="trait">content</span> in soybean.</h2>
			<h3>PMID:23052024  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu W(1), Wen Z, Li H, Yuan D, Li J, Zhang H, Huang Z, Cui S, Du W.
				<br/> 
				年份：2013 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water soluble protein <span class="trait">content</span> (SPC) plays an important role in the functional efficacy of protein in food products. Therefore, for the identification of quantitative trait loci (QTL) associated with SPC, 212 F(2:9) lines of the recombinant inbred line (RIL) population derived from the cross of ZDD09454 × Yudou12 were grown along with the parents, in six different environments (location × year) to determine inheritance and map solubility-related genes. A linkage map comprising of 301 SSR markers covering 3,576.81 cM was constructed in the RIL population. <span class="trait">Seed</span> SPC was quantified with a macro-Kjeldahl procedure in samples collected over multiple years from three locations (Nantong in 2007 and 2008, Zhengzhou in 2007 and 2008, and Xinxiang in 2008 and 2009). SPC demonstrated transgressive segregation, indicating a complementary genetic structure between the parents. Eleven putative QTL were associated with SPC explaining 4.5-18.2 % of the observed phenotypic variation across the 6 year/location environments. Among these, two QTL (qsp8-4, qsp8-5) near <span class="gene">GMENOD2B</span> and Sat_215 showed an association with SPC in multiple environments, suggesting that they were key QTL related to protein solubility. The QTL × environment interaction demonstrated the complex genetic mechanism of SPC. These SPC-associated QTL and linked markers in soybean will provide important information that can be utilized by breeders to improve the functional quality of soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combinations of mutant <span class="gene">FAD2</span> and <span class="gene">FAD3</span> genes to produce high oleic acid and low linolenic acid soybean <span class="trait">oil.</span></h2>
			<h3>PMID:22476873  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pham AT(1), Shannon JG, Bilyeu KD.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						High oleic acid soybeans were produced by combining mutant <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> genes. Despite having a high oleic acid <span class="trait">content,</span> the linolenic acid <span class="trait">content</span> of these soybeans was in the range of 4-6 %, which may be high enough to cause oxidative instability of the <span class="trait">oil.</span> Therefore, a study was conducted to incorporate one or two mutant <span class="gene">FAD3</span> genes into the high oleic acid background to further reduce the linolenic acid <span class="trait">content.</span> As a result, soybean lines with high oleic acid and low linolenic acid (HOLL) <span class="trait">content</span> were produced using different sources of mutant <span class="gene">FAD2-1A</span> genes. While oleic acid <span class="trait">content</span> of these HOLL lines was stable across two testing environments, the reduction of linolenic acid <span class="trait">content</span> varied depending on the number of mutant <span class="gene">FAD3</span> genes combined with mutant <span class="gene">FAD2-1</span> genes, on the severity of mutation in the <span class="gene">FAD2-1A</span> gene, and on the testing environment. Combination of two mutant <span class="gene">FAD2-1</span> genes and one mutant <span class="gene">FAD3</span> gene resulted in less than 2 % linolenic acid <span class="trait">content</span> in Portageville, Missouri (MO) while four mutant genes were needed to achieve the same linolenic acid in Columbia, MO. This study generated non-transgenic soybeans with the highest oleic acid <span class="trait">content</span> and lowest linolenic acid <span class="trait">content</span> reported to date, offering a unique alternative to produce a fatty acid profile similar to olive <span class="trait">oil.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In silico identification and comparative genomics of candidate genes involved in biosynthesis and accumulation of <span class="trait">seed</span> <span class="trait">oil</span> in plants.</h2>
			<h3>PMID:22312320</h3>
				

				
			<p>
				作者：Sharma A(1), Chauhan RS.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genes involved in fatty acids biosynthesis, modification and <span class="trait">oil</span> body formation are expected to be conserved in structure and function in different plant species. However, significant differences in the composition of fatty acids and total <span class="trait">oil</span> contents in seeds have been observed in different plant species. Comparative genomics was performed on 261 genes involved in fatty acids biosynthesis, TAG synthesis, and <span class="trait">oil</span> bodies formation in Arabidopsis, Brassica rapa, castor bean and soybean. In silico expression analysis revealed that stearoyl desaturase, FatB, FAD2, oleosin and <span class="gene">DGAT</span> are highly abundant in seeds, thereby considered as ideal candidates for mining of favorable alleles in natural population. Gene structure analysis for major genes, ACCase, FatA, FatB, FAD2, <span class="gene">FAD3</span> and DGAT, which are known to play crucial role in <span class="trait">oil</span> synthesis revealed that there are uncommon variations (SNPs and INDELs) which lead to varying <span class="trait">content</span> and composition of fatty acids in <span class="trait">seed</span> <span class="trait">oil.</span> The predicted variations can provide good targets for <span class="trait">seed</span> <span class="trait">oil</span> QTL identification, understanding the molecular mechanism of <span class="trait">seed</span> <span class="trait">oil</span> accumulation, and genetic modification to enhance <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">yield</span> in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Vernonia DGATs can complement the disrupted <span class="trait">oil</span> and protein metabolism in epoxygenase-expressing soybean seeds.</h2>
			<h3>PMID:22107928  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li R(1), Yu K, Wu Y, Tateno M, Hatanaka T, Hildebrand DF.
				<br/> 
				年份：2012 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant oils can be useful chemical feedstocks such as a source of epoxy fatty acids. High seed-specific expression of a Stokesia laevis epoxygenase (SlEPX) in soybeans only results in 3-7% epoxide levels. SlEPX-transgenic soybean seeds also exhibited other phenotypic alterations, such as altered <span class="trait">seed</span> fatty acid profiles, reduced <span class="trait">oil</span> accumulation, and variable protein levels. SlEPX-transgenic seeds showed a 2-5% reduction in total <span class="trait">oil</span> <span class="trait">content</span> and protein levels of 30.9-51.4%. To address these pleiotrophic effects of SlEPX expression on other traits, transgenic soybeans were developed to co-express SlEPX and <span class="gene">DGAT</span> (diacylglycerol acyltransferase) genes (VgDGAT1 & 2) isolated from Vernonia galamensis, a high accumulator of epoxy fatty acids. These side effects of SlEPX expression were largely overcome in the <span class="gene">DGAT</span> co-expressing soybeans. Total <span class="trait">oil</span> and protein contents were restored to the levels in non-transgenic soybeans, indicating that both VgDGAT1 and VgDGAT2 could complement the disrupted phenotypes caused by over-expression of an epoxygenase in soybean seeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel <span class="gene">FAD2-1</span> A allele in a soybean plant introduction offers an alternate means to produce soybean <span class="trait">seed</span> <span class="trait">oil</span> with 85% oleic acid <span class="trait">content.</span></h2>
			<h3>PMID:21681491  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pham AT(1), Lee JD, Shannon JG, Bilyeu KD.
				<br/> 
				年份：2011 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alteration of fatty acid profiles in soybean to improve soybean <span class="trait">oil</span> quality has been a long-time goal of soybean researchers. Soybean <span class="trait">oil</span> with elevated oleic acid is desirable because this monounsaturated fatty acid improves the nutrition and oxidative stability of soybean <span class="trait">oil</span> compared to other oils. In the lipid biosynthetic pathway, the enzyme fatty acid desaturase 2 (FAD2) is responsible for the conversion of oleic acid precursors to linoleic acid precursors in developing soybean seeds. Two genes encoding <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> were identified to be expressed specifically in seeds during embryogenesis and have been considered to hold an important role in controlling the <span class="trait">seed</span> oleic acid <span class="trait">content.</span> A total of 22 soybean plant introduction (PI) lines identified to have an elevated oleic acid <span class="trait">content</span> were characterized for sequence mutations in the FAD 2-1A and <span class="gene">FAD2-1B</span> genes. PI 603452 was found to contain a deletion of a nucleotide in the second exon of FAD2-1A. These important SNPs were used in developing molecular marker genotyping assays. The assays appear to be a reliable and accurate tool to identify the FAD 2-1A and <span class="gene">FAD2-1B</span> genotype of wild-type and mutant plants. PI 603452 was subsequently crossed with PI 283327, a soybean line that has a mutation in FAD2-1B. Interestingly, soybean lines carrying both homozygous insertion/deletion mutation (indel) <span class="gene">FAD2-1A</span> alleles and mutant <span class="gene">FAD2-1B</span> alleles have an average of 82-86% oleic acid <span class="trait">content,</span> compared to 20% in conventional soybean, and low levels of linoleic and linolenic acids. The newly identified indel mutation in the <span class="gene">FAD2-1A</span> gene offers a simple method for the development of high oleic acid commercial soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Dynamic QTL analysis of linolenic acid <span class="trait">content</span> in different developmental stages of soybean <span class="trait">seed.</span></h2>
			<h3>PMID:21344183  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Han Y(1), Xie D, Teng W, Zhang S, Chang W, Li W.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Linolenic acid (LN) in soybean (Glycine max L. Merr.) <span class="trait">seed</span> mainly contributes to the undesirable odors and flavors commonly associated with poor <span class="trait">oil</span> quality. LN deposition at various stages of soybean <span class="trait">seed</span> development had not been reported by 2010. The objects of this study were (1) to identify and measure quantitative trait loci (QTL) underlying LN <span class="trait">content</span> and (2) to estimate the QTL effects expressed from earlier <span class="trait">seed</span> developmental stages to drying <span class="trait">seed</span> of soybean. One hundred and twenty-five F(5:8) and F(5:9) recombinant inbred lines derived from the cross of soybean cultivars 'Hefeng 25' and 'Dongnong L5' were used for the identification of QTL underlying LN <span class="trait">content</span> from the 37 day (D) to 86D stages after flowering, at Harbin in 2008 and 2009. QTL × Environment interactions (QE) effects were evaluated using a mixed genetic model (Zhu in J Zhejiang Univ (Natural Science) 33:327-335, 1999). Twelve unconditional QTL and 12 conditional QTL associated with LN <span class="trait">content</span> were identified at different developmental stages. Most of the QTL explained &lt;10% of phenotypic variation of LN <span class="trait">content.</span> Unconditional QTL QLNF-1, QLNC2-1, QLND1b-1, QLNA2-1 and QLNH-1 influenced LN <span class="trait">content</span> across different development stages and environments. Conditional QTL QLNF-1, QLNC2-1 and QLNH-1 were identified in multiple developmental stages and environments. Conditional and unconditional QTL clustered in neighboring intervals on linkage groups A2, <span class="gene">C2</span> and D1b. Ten QTL with conditional additive main effects (a) and/or conditional additive × environment interaction effects (ae) at specific developmental stage were identified on nine linkage groups. Of them, six QTL only possessed additive main effects and seven QTL had significant ae effects in different developmental stages. A total of 13 epistatic pairwise QTL were identified by conditional mapping in different developmental stages. Two pairs of QTL only showed aa effects and five pairs of QTL only showed aae effects at different developmental stages. QTL with aa effects, as well as their environmental interaction effects, appeared to vary at different developmental stages.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic and genomic analyses of a fast neutron mutant population resource in soybean.</h2>
			<h3>PMID:21321255  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bolon YT(1), Haun WJ, Xu WW, Grant D, Stacey MG, Nelson RT, Gerhardt DJ, Jeddeloh JA, Stacey G, Muehlbauer GJ, Orf JH, Naeve SL, Stupar RM, Vance CP.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mutagenized populations have become indispensable resources for introducing variation and studying gene function in plant genomics research. In this study, fast neutron (FN) radiation was used to induce deletion mutations in the soybean (Glycine max) genome. Approximately 120,000 soybean seeds were exposed to FN radiation doses of up to 32 Gray units to develop over 23,000 independent M2 lines. Here, we demonstrate the utility of this population for phenotypic screening and associated genomic characterization of striking and agronomically important traits. Plant variation was cataloged for <span class="trait">seed</span> composition, maturity, morphology, pigmentation, and nodulation traits. Mutants that showed significant increases or decreases in <span class="trait">seed</span> protein and <span class="trait">oil</span> <span class="trait">content</span> across multiple generations and environments were identified. The application of comparative genomic hybridization (CGH) to lesion-induced mutants for deletion mapping was validated on a midoleate x-ray mutant, M23, with a known <span class="gene">FAD2-1A</span> (for fatty acid desaturase) gene deletion. Using CGH, a subset of mutants was characterized, revealing deletion regions and candidate genes associated with phenotypes of interest. Exome resequencing and sequencing of PCR products confirmed FN-induced deletions detected by CGH. Beyond characterization of soybean FN mutants, this study demonstrates the utility of CGH, exome sequence capture, and next-generation sequencing approaches for analyses of mutant plant genomes. We present this FN mutant soybean population as a valuable public resource for future genetic screens and functional genomics research.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of two low phytic acid mutations on <span class="trait">seed</span> quality and nutritional traits in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:19323582  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan FJ(1), Zhu DH, Deng B, Fu XJ, Dong DK, Zhu SL, Li BQ, Shu QY.
				<br/> 
				年份：2009 May 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reduction of phytic acid in soybean seeds has the potential to improve the nutritional value of soybean meal and lessen phosphorus pollution in large scale animal farming. The objective of this study was to assess the effect of two new low phytic acid (LPA) mutations on <span class="trait">seed</span> quality and nutritional traits. Multilocation/season comparative analyses showed that the two mutations did not affect the concentration of crude protein, any of the individual amino acids, crude <span class="trait">oil,</span> and individual saturated fatty acids. Among other traits, <span class="gene">Gm-lpa-TW75-1</span> had consistently higher sucrose contents (+47.4-86.1%) and lower raffinose contents (-74.2 to -84.3%) than those of wild type (WT) parent Taiwan 75; <span class="gene">Gm-lpa-ZC-2</span> had higher total isoflavone contents (3038.8-4305.4 microg/g) than its parent Zhechun # 3 (1583.6-2644.9 microg/g) in all environments. Further tests of homozygous F(3) progenies of the cross <span class="gene">Gm-lpa-ZC-2</span> x Wuxing # 4 (WT variety) showed that LPA lines had a mean <span class="trait">content</span> of total isoflavone significantly higher than WT lines. This study demonstrated that two LPA mutant genes have no negative effects on <span class="trait">seed</span> quality and nutritional traits; they instead have the potential to improve a few other properties. Therefore, these two mutant genes are valuable genetic resources for breeding high quality soybean varieties.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean Dof-type transcription factor genes, <span class="gene">GmDof4</span> and <span class="gene">GmDof11,</span> enhance lipid <span class="trait">content</span> in the seeds of transgenic Arabidopsis plants.</h2>
			<h3>PMID:17877700  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang HW(1), Zhang B, Hao YJ, Huang J, Tian AG, Liao Y, Zhang JS, Chen SY.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is one of the most important leguminous <span class="trait">seed</span> crops among the <span class="trait">oil</span> crops. Although the pathways for lipid biosynthesis have been identified, the factors that regulate the biosynthetic pathways at the transcriptional level are largely unknown. Here, we report our findings on the involvement of soybean Dof-type transcription factor genes in the regulation of the lipid <span class="trait">content</span> in soybean seeds. We identified 28 Dof-type transcription factor genes in soybean plants, and these genes displayed diverse patterns of expression in various organs. Seven flower/pod-specific genes and one constitutively expressed gene were further investigated. The proteins encoded by these seven genes were localized in the nucleus, and exhibited different abilities for transcriptional activation and DNA binding. Two genes, <span class="gene">GmDof4</span> and <span class="gene">GmDof11,</span> were found to increase the <span class="trait">content</span> of total fatty acids and lipids in <span class="gene">GmDof4</span> and <span class="gene">GmDof11</span> transgenic Arabidopsis seeds. We also found that the 1000-seed weight was increased in the <span class="gene">GmDof4</span> and <span class="gene">GmDof11</span> transgenic plants. Using microarray and DNA binding analysis, we found that the two Dof-like proteins, <span class="gene">GmDof4</span> and <span class="gene">GmDof11,</span> activated the acetyl CoA carboxylase gene and long-chain-acyl CoA synthetase gene, respectively, by direct binding to the cis-DNA elements in their promoter regions. In addition, both proteins downregulated the storage protein gene, CRA1, through direct binding. These results suggest that the two <span class="gene">GmDof</span> genes may augment the lipid <span class="trait">content</span> of soybean seeds by upregulating genes that are associated with the biosynthesis of fatty acids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Functional expression of an omega-3 fatty acid desaturase gene from Glycine max in Saccharomyces cerevisiae].</h2>
			<h3>PMID:16572837  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang HT(1), Yang JS, Shan L, Bi YP.
				<br/> 
				年份：2006 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Alpha-linolenic acid(ALA, C18:3delta9,12,15 ) is an essential fatty acid which has many sanitary functions to human. However, its contents in diets are often not enough. In plants, omega-3 fatty acid desaturases(FAD) catalyze linoleic acid(LA, C18:2delta9,12) into ALA. The <span class="trait">seed</span> <span class="trait">oil</span> of Glycine max contains high level of ALA. To investigate the functions of Glycine max omega-3FAD, the cDNA of <span class="gene">GmFAD3</span> C was amplified by RT-PCR from immature seeds, then cloned into the shuttle expression vector p416 to generate the recombinant vector p4GFAD3C. The resulting vector was transformed into Saccharomyces cerevisiae K601 throuth LiAc method. The positive clones were screened on the CM(Ura-) medium and identified by PCR, and then cultured in CM (Ura-) liquid medium with exogenous LA in 20 degrees C for three days. The intracellular fatty acid composition of the engineering strain Kp416 and Kp4GFAD3C was analyzed by gas chromatography (GC). A novel peak in strain Kp4GFAD3C was detected,which was not detectable in control, Comparison of the retention times of the newly yielded peak with that of authentic standard indicated that the fatty acid is ALA. The <span class="trait">content</span> of ALA reached to 3.1% of the total fatty acid in recombinant strain, the <span class="trait">content</span> of LA correspondingly decreased from 22% to 16.2% by contrast. It was suggested that the protein encoded by <span class="gene">GmFAD3</span> C can specifically catalyze 18 carbon PUFA substrate of LA into ALA by taking off hydrogen atoms at delta15 location. In this study, we expressed a Glycine max omega-3 fatty acid desaturase gene in S. cerevisiae; An efficient and economical yeast expressing system(K601-p416 system) which is suitable for the expression of FAD was built.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Cereal bars with soy protein and wheat germ, physicochemical characteristics and texture during the storage].</h2>
			<h3>PMID:16454057  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Castro Freitas DD(1).
				<br/> 
				年份：2005 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Studies analyzing cereal bars have reported on consumer characteristics and preferences in sensory analyses and on their market growth, however little has been published on their physicochemical data and texture properties. Thus the objective of this research was to provide information about the storage of a cereal <span class="gene">bar</span> formulation with high protein and vitamin levels based on soy protein and wheat germ, packaged in 3 different films (A: PET/PEBD; B: PETmet/PEBD; C: PET/PEBD/AL/PEBD), during 6 months under environmental conditions of temperature (25 +/- 2 degrees C) and relative humidity (56%). The moisture <span class="trait">content,</span> water activity, pH and total acidity of the cereal bars were determined. The textural measurements accompanied during storage were breaking strength, hardness and cohesiveness. The cereal bars presented variations in water activity (Aw), moisture <span class="trait">content</span> and total acidity during storage. The moisture <span class="trait">content</span> of the bars tended to increase, which led to a significant (p = 0.05) influence on the texture characteristics of breaking strength and hardness, in the different packaging films tested. The increase in the values for breaking strength (A: 4756,5N; B: 5093,0N; C: 5575,6N) at 45 days of storage was attributed to a possible crystallization of the agglutinating syrup used for the bars. The textured soy protein used in the formulation could also have contributed to this fact due to its hygroscopic character, also interfering in the decrease in the cohesiveness measurements (deformation) with time. The effect of the different barrier properties of the packaging films tested was significant (p &lt; 0.05) in the stability of the cereal bars during storage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of dehulling of rapeseed on feed value and nutrient digestibility of rape products in pigs.</h2>
			<h3>PMID:15595622  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kracht W(1), Dänicke S, Kluge H, Keller K, Matzke W, Hennig U, Schumann W.
				<br/> 
				年份：2004 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In the presented study the influence of dehulling rapeseed on the composition of rapeseed meal (RM) and rapeseed cake (RC) and on its feed value for piglets and growing-finishing pigs was investigated. Before withdrawal of <span class="trait">oil,</span> rapeseed (variety Express) was dehulled applying a procedure developed by SKET <span class="gene">GmbH</span> Magdeburg and the Section Food-Technology of the University Essen. The steps of the dehulling procedure were described. For RM the <span class="trait">oil</span> was removed by the prepress-solvent procedure till a crude fat <span class="trait">content</span> of 2.1% in DM. RC was produced by pressing only resulting approximately 13% crude fat in DM. The RM and RC from not dehulled (ND) and dehulled (D) rapeseed were examined analytically. Crude nutrients, sugar and fibre substances, amino acids, some minerals and trace elements, fatty acids, glucosinolates and sinapine, and phytate were determined. By dehulling the <span class="trait">seed</span> the crude fibre <span class="trait">content</span> was decreased in RM and RC by approximately 40%. The ADF <span class="trait">content</span> declined by 35 and 39%, and the NDF <span class="trait">content</span> by 28% and 40% in RM and RC, respectively. The decrease in ADL <span class="trait">content</span> amounted to 50% and 65% for RM and RC, respectively. On the other hand, the CP <span class="trait">content</span> of RM and RC was increased by 7% and 13%, respectively, by dehulling the <span class="trait">seed</span> while the amino acid <span class="trait">content</span> of rape protein increased only slightly. The contents of glucosinolates and sinapine were also increased by dehulling, while the contents of phytate and phytate P were decreased. In digestibility and balance experiments with piglets and intact hybrid breeds of growing-finishing pigs, the digestibility of organic matter and of crude nutrients and the contents of digestible energy and metabolizable energy were estimated. Furthermore, the precaecal digestibility of crude nutrients and amino acids was determined with fistulated mini-pigs. By dehulling the seeds the digestibility of organic matter from RM and RC was improved in piglets and adult pigs by approximately 10%, and the ME contents increased by 13-15%. The precaecal digestibility of the sum of amino acids was increased by approximately 3 and 6 units in RM and RC, respectively. The precaecal digestibility of lysine in RM and RC reached that of soybean <span class="trait">oil</span> meal from not dehulled beans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein <span class="trait">content,</span> <span class="trait">oil</span> <span class="trait">content,</span> 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content)</span> and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improvement of rice (Oryza sativa L.) <span class="trait">seed</span> <span class="trait">oil</span> quality through introduction of a soybean microsomal omega-3 fatty acid desaturase gene.</h2>
			<h3>PMID:12835909  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Anai T(1), Koga M, Tanaka H, Kinoshita T, Rahman SM, Takagi Y.
				<br/> 
				年份：2003 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Microsomal omega-3 fatty acid desaturase is an essential enzyme in the production of the n-3 polyunsaturated fatty acid alpha-linolenic acid during the <span class="trait">seed</span> developing stage. We have constructed a chimeric gene consisting of a maize Ubi1-P-int and a soybean <span class="gene">GmFAD3</span> cDNA, which was introduced into rice plants by Agrobacterium-mediated transformation. Ten transformants containing the chimeric gene were established and expression subsequently confirmed by Northern blotting. Furthermore, alpha-linolenic acid <span class="trait">content</span> of the T(1) seeds increased dramatically up to tenfold that of the control, and this phenotype was also stably inherited in the T(2) and T(3) progenies. These results demonstrate that the alpha-linolenic acid <span class="trait">content</span> of rice <span class="trait">seed</span> <span class="trait">oil</span> can easily be altered using the combination of a high-activity promoter and a <span class="gene">GmFAD3</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biplot Analysis of Test Sites and Trait Relations of Soybean in Ontario.</h2>
			<h3>PMID:11756248</h3>
				

				
			<p>
				作者：Yan W(1), Rajcan I.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Superior crop cultivars must be identified through multi-environment trials (MET) and on the basis of multiple traits. The objectives of this paper were to describe two types of biplots, the GGE biplot and the GT biplot, which graphically display genotype by environment data and genotype by trait data, respectively, and hence facilitate cultivar evaluation on the basis of <span class="gene">MET</span> data and multiple traits. Genotype main effect plus genotype by environment interaction effect (GGE) biplot analysis of the soybean [Glycine max (L.) Merr.] <span class="trait">yield</span> data for the 2800 crop heat unit area of Ontario for <span class="gene">MET</span> in the period 1994-1999 revealed yearly crossover genotype by site interactions. The eastern Ontario site Winchester showed a different genotype response pattern from the three southwestern Ontario sites in four of the six years. The interactions were not large enough to divide the area into different mega-environments as when analyzed over years, a single cultivar yielded the best in all four sites. The southwestern site, St. Pauls, was found to always group together with at least one of the other three sites; it did not provide unique information on genotype performance. Therefore, in future cultivar evaluations, Winchester should always be used but St. Pauls can be dismissed. Applying GT biplot to the 1994-1999 multiple trait data illustrated that GT biplots graphically displayed the interrelationships among <span class="trait">seed</span> <span class="trait">yield,</span> <span class="trait">oil</span> <span class="trait">content,</span> protein <span class="trait">content,</span> plant height, and days to maturity, among other traits, and facilitated visual cultivar comparisons and selection. It was found that selection for <span class="trait">seed</span> <span class="trait">yield</span> alone was not only the simplest, but also the most effective strategy in the early stages of soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cosuppression of the alpha subunits of beta-conglycinin in transgenic soybean seeds induces the formation of endoplasmic reticulum-derived protein bodies.</h2>
			<h3>PMID:11340189  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kinney AJ(1), Jung R, Herman EM.
				<br/> 
				年份：2001 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The expression of the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin was suppressed by sequence-mediated gene silencing in transgenic soybean <span class="trait">seed.</span> The resulting seeds had similar total <span class="trait">oil</span> and protein <span class="trait">content</span> and ratio compared with the parent line. The decrease in beta-conglycinin protein was apparently compensated by an increased accumulation of glycinin. In addition, proglycinin, the precursor of glycinin, was detected as a prominent polypeptide band in the protein profile of the transgenic <span class="trait">seed</span> extract. Electron microscopic analysis and immunocytochemistry of maturing transgenic soybean seeds indicated that the process of storage protein accumulation was altered in the transgenic line. In normal soybeans, the storage proteins are deposited in pre-existing vacuoles by Golgi-derived vesicles. In contrast, in transgenic <span class="trait">seed</span> with reduced beta-conglycinin levels, endoplasmic reticulum (ER)-derived vesicles were observed that resembled precursor accumulating-vesicles of pumpkin seeds and the protein bodies accumulated by cereal seeds. Their ER-derived membrane of the novel vesicles did not contain the protein storage vacuole tonoplast-specific protein alpha-TIP, and the sequestered polypeptides did not contain complex glycans, indicating a preGolgi and nonvacuolar nature. Glycinin was identified as a major component of these novel protein bodies and its diversion from normal storage protein trafficking appears to be related to the proglycinin buildup in the transgenic <span class="trait">seed.</span> The stable accumulation of proteins in a protein body compartment instead of vacuolar accumulation of proteins may provide an alternative intracellular site to sequester proteins when soybeans are used as protein factories.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Developmental and growth temperature regulation of two different microsomal omega-6 desaturase genes in soybeans.</h2>
			<h3>PMID:8587990  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heppard EP(1), Kinney AJ, Stecca KL, Miao GH.
				<br/> 
				年份：1996 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The polyunsaturated fatty acid <span class="trait">content</span> is one of the major factors influencing the quality of vegetable oils. Edible oils rich in monounsaturated fatty acid provide improved <span class="trait">oil</span> stability, flavor, and nutrition for human and animal consumption. In plants, the microsomal omega-6 desaturase-catalyzed pathway is the primary route of production of polyunsaturated lipids. We report the isolation of two different cDNA sequences, <span class="gene">FAD2-1</span> and FAD2-2, encoding microsomal omega-6 desaturase in soybeans and the characterization of their developmental and temperature regulation. The <span class="gene">FAD2-1</span> gene is strongly expressed in developing seeds, whereas the <span class="gene">FAD2-2</span> gene is constitutively expressed in both vegetative tissues and developing seeds. Thus, the <span class="gene">FAD2-2</span> gene-encoded omega-6 desaturase appears to be responsible for production of polyunsaturated fatty acids within membrane lipids in both vegetative tissues and developing seeds. The seed-specifically expressed <span class="gene">FAD2-1</span> gene is likely to play a major role in controlling conversion of oleic acid to linoleic acid within storage lipids during <span class="trait">seed</span> development. In both soybean <span class="trait">seed</span> and leaf tissues, linoleic acid and linolenic acid levels gradually increase as temperature decreases. However, the levels of transcripts for FAD2-1, FAD2-2, and the plastidial omega-6 desaturase gene (FAD 6) do not increase at low temperature. These results suggest that the elevated polyunsaturated fatty acid levels in developing soybean seeds grown at low temperature are not due to the enhanced expression of omega-6 desaturase genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of Major Quantitative Trait Loci for <span class="trait">Seed</span> <span class="trait">Oil</span> <span class="trait">Content</span> in Soybeans by Combining Linkage and Genome-Wide Association Mapping</h2>
			<h3>PMID:28747922</h3>
				
					<h3>PMCID:5506190</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Yongce </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/435444/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shuguang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Zili </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Fangguo </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Jiejie </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289861/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tuanjie </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/345840/overview"/>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">oil</span> <span class="trait">content</span> has a wide range of genetic variation among soybean accessions ( <xref rid="B57" ref-type="bibr"> Wilson, 2004 </xref> ) and there is also a strong negative correlation with <span class="trait">seed</span> protein <span class="trait">content</span> ( <xref rid="B55" ref-type="bibr"> Wilcox, 1998 </xref> ). Therefore, several QTLs can be related to both <span class="trait">seed</span> protein and <span class="trait">oil</span> <span class="trait">content.</span> Two major QTLs for <span class="trait">seed</span> oil/protein <span class="trait">content</span> have been mapped in many soybean populations: one was mapped at an interval from 20 to 40 centimorgan (cM) on chromosome (Chr.) 20 of the Soybean <span class="gene">GmConsensus</span> 4.0 Map and the other one was mapped at an interval from 10 to 30 cM on Chr. 15 <sup> <xref ref-type="fn" rid="fn01"> 1 </xref> </sup> . Many studies have focused on these two QTLs. The QTL on Chr. 20 was narrowed down to a small region ( <xref rid="B34" ref-type="bibr"> Nichols et al., 2006 </xref> ; <xref rid="B5" ref-type="bibr"> Bolon et al., 2010 </xref> ; <xref rid="B20" ref-type="bibr"> Hwang et al., 2014 </xref> ; <xref rid="B48" ref-type="bibr"> Vaughn et al., 2014 </xref> ); and the QTL on Chr. 15 was fine mapped at a 535 kb interval ( <xref rid="B24" ref-type="bibr"> Kim et al., 2016 </xref> ). In contrast, fewer studies have focused on refining the position of the QTL for <span class="trait">oil</span> <span class="trait">content</span> on Chr. 05, which has been reported many times within a large genomic region ( <xref rid="B32" ref-type="bibr"> Mansur et al., 1996 </xref> ; <xref rid="B7" ref-type="bibr"> Brummer et al., 1997 </xref> ; <xref rid="B36" ref-type="bibr"> Orf et al., 1999 </xref> ; <xref rid="B38" ref-type="bibr"> Pathan et al., 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, the distribution of <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> followed a normal distribution in both the natural soybean population and in RILs, indicating that the genetic control of <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> is complex (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). Although <span class="trait">seed</span> <span class="trait">oil</span> <span class="trait">content</span> had a high <italic> h </italic> <sup> 2 </sup> , the environmental instability of this trait made it difficult to detect stable QTLs in different environments. However, there were some major loci detected in different environments using one method in our study. In linkage mapping, <italic> qOil-10-1 </italic> and <italic> qOil-14-1 </italic> were detected in four environments, had adjacent or physically overlapping QTLs, as reported by previous studies, and might actually be the same QTL ( <xref rid="B47" ref-type="bibr"> Tajuddin et al., 2003 </xref> ; <xref rid="B52" ref-type="bibr"> Wang et al., 2014 </xref> ). In the GWAS, the significant SNPs on Chr. 20 were found to be associated over the 2 years investigated. The peak marker, <span class="gene">Gm20_31164168,</span> was similar to the one identified by <xref rid="B48" ref-type="bibr"> Vaughn et al. (2014) </xref> at 31150279 bp on Chr. 20. Therefore, our results were in good agreement with previous studies, and the QTLs identified can be treated as important targets for the identification of candidate genes involved in the modification of <span class="trait">oil</span> in future studies.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>