<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed coat color</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed coat color</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Mutations in Argonaute5 Illuminate Epistatic Interactions of the K1 and I Loci Leading to Saddle <span class="trait">Seed</span> <span class="trait">Color</span> Patterns in Glycine max.</h2>
			<h3>PMID:28351993  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cho YB(1), Jones SI(1), Vodkin LO(2).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The soybean (Glycine max) <span class="trait">seed</span> <span class="trait">coat</span> has distinctive, genetically programmed patterns of pigmentation, and the recessive k1 mutation can epistatically overcome the dominant I and ii alleles, which inhibit <span class="trait">seed</span> <span class="trait">color</span> by producing small interfering RNAs (siRNAs) targeting chalcone synthase (CHS) mRNAs. Small RNA sequencing of dissected regions of immature <span class="trait">seed</span> coats demonstrated that <span class="gene">CHS</span> siRNA levels cause the patterns produced by the ii and ik alleles of the I locus, which restrict pigment to the hilum or saddle region of the <span class="trait">seed</span> <span class="trait">coat,</span> respectively. To identify the K1 locus, we compared RNA-seq data from dissected regions of two Clark isolines having similar saddle phenotypes mediated by <span class="gene">CHS</span> siRNAs but different genotypes (homozygous ik K1 versus homozygous ii k1). By examining differentially expressed genes, mapping information, and genome resequencing, we identified a 129-bp deletion in <span class="gene">Glyma.11G190900</span> encoding Argonaute5 (AGO5), a member of the Argonaute family. Amplicon sequencing of several independent saddle pattern mutants from different genetic backgrounds revealed independent lesions affecting AGO5, thus establishing <span class="gene">Glyma.11G190900</span> as the K1 locus. Nonfunctional <span class="gene">AGO5</span> from k1 alleles leads to altered distributions of <span class="gene">CHS</span> siRNAs, thus explaining how the k1 mutation reverses the phenotype of the <span class="trait">seed</span> <span class="trait">coat</span> regions from yellow to pigmented, even in the presence of the normally dominant I or ii alleles.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterizing Two Inter-specific Bin Maps for the Exploration of the QTLs/Genes that Confer Three Soybean Evolutionary Traits</h2>
			<h3>PMID:27602037</h3>
				

				
					<h3>PMCID:4994327</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Wubin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/338561/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Meifeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/368787/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yufeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/347210/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xuliang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/367596/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Shixuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Liping </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/347211/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Zheping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Jiejie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tuanjie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/345840/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289861/overview"/>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Annual wild soybean (Glycine soja Sieb. and Zucc.), the wild progenitor of the cultivated soybean [Glycine max (L.) Merr.], is valuable for improving the later. The construction of a linkage map is crucial for studying the genetic differentiation between these species, but marker density is the main factor limiting the accuracy of such a map. Recent advances in next-generation sequencing technologies allow for the generation of high-density linkage maps. Here, two sets of inter-specific recombinant inbred line populations, named NJIRNP and NJIR4P, composed of 284 and 161 lines, respectively, were generated from the same wild male parent, PI 342618B, and genotyped by restriction-site-associated DNA sequencing. Two linkage maps containing 5,728 and 4,354 bins were constructed based on 89,680 and 80,995 single nucleotide polymorphisms, spanning a total genetic distance of 2204.6 and 2136.7 cM, with an average distance of 0.4 and 0.5 cM between neighboring bins in NJRINP and NJRI4P, respectively. With the two maps, seven well-studied loci, B1 for <span class="trait">seed</span> bloom; G and I for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color;</span> E2, E3, qDTF16.1 and two linked FLOWERING LOCUS T for days to flowering, were detected. In addition, two SB and two DTF loci were newly identified in wild soybean. Using two high-density maps, the mapping resolution was enhanced, e.g., G was narrowed to a region of 0.4 Mb on chromosome 1, encompassing 54 gene models, among which only <span class="gene">Glyma01g40590</span> was predicted to be involved in anthocyanin accumulation, and its interaction with I was verified in both populations. In addition, five genes, <span class="gene">Glyma16g03030,</span> orthologous to Arabidopsis Phytochrome A (PHYA); <span class="gene">Glyma13g28810,</span> <span class="gene">Glyma13g29920,</span> and <span class="gene">Glyma13g30710</span> predicted to encode the APETALA 2 (AP2) domain; and <span class="gene">Glyma02g00300,</span> involved in response to red or far red light, might be candidate DTF genes. Our results demonstrate that RAD-seq is a cost-effective approach for constructing high-density and high-quality bin maps that can be used to map QTLs/genes into such small enough regions that their candidate genes can be predicted.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Annual wild soybean ( <italic> Glycine soja </italic> Sieb. and Zucc.), the wild progenitor of the cultivated soybean [ <italic> Glycine max </italic> (L.) Merr.], is valuable for improving the later. The construction of a linkage map is crucial for studying the genetic differentiation between these species, but marker density is the main factor limiting the accuracy of such a map. Recent advances in next-generation sequencing technologies allow for the generation of high-density linkage maps. Here, two sets of inter-specific recombinant inbred line populations, named NJIRNP and NJIR4P, composed of 284 and 161 lines, respectively, were generated from the same wild male parent, PI 342618B, and genotyped by restriction-site-associated DNA sequencing. Two linkage maps containing 5,728 and 4,354 bins were constructed based on 89,680 and 80,995 single nucleotide polymorphisms, spanning a total genetic distance of 2204.6 and 2136.7 cM, with an average distance of 0.4 and 0.5 cM between neighboring bins in NJRINP and NJRI4P, respectively. With the two maps, seven well-studied loci, <italic> B1 </italic> for <span class="trait">seed</span> bloom; <italic> G </italic> and <italic> I </italic> for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color;</span> <italic> E2 </italic> , <italic> E3 </italic> , <italic> qDTF16.1 </italic> and two linked <italic> FLOWERING LOCUS T </italic> for days to flowering, were detected. In addition, two SB and two DTF loci were newly identified in wild soybean. Using two high-density maps, the mapping resolution was enhanced, e.g., <italic> G </italic> was narrowed to a region of 0.4 Mb on chromosome 1, encompassing 54 gene models, among which only <italic> <span class="gene">Glyma01g40590</span> </italic> was predicted to be involved in anthocyanin accumulation, and its interaction with <italic> I </italic> was verified in both populations. In addition, five genes, <italic> <span class="gene">Glyma16g03030</span> </italic> , orthologous to <italic> Arabidopsis Phytochrome A </italic> ( <italic> <span class="gene">PHYA</span> </italic> ); <italic> <span class="gene">Glyma13g28810</span> </italic> , <italic> <span class="gene">Glyma13g29920</span> </italic> , and <italic> <span class="gene">Glyma13g30710</span> </italic> predicted to encode the <italic> APETALA 2 </italic> ( <italic> <span class="gene">AP2</span> </italic> ) domain; and <italic> <span class="gene">Glyma02g00300</span> </italic> , involved in response to red or far red light, might be candidate DTF genes. Our results demonstrate that RAD-seq is a cost-effective approach for constructing high-density and high-quality bin maps that can be used to map QTLs/genes into such small enough regions that their candidate genes can be predicted.
					</p>
				
					<p>&nbsp;&nbsp;
						Meanwhile, the marker densities of linkage maps were low, except for those of <span class="gene">GmComposite2003</span> ( <xref rid="B37" ref-type="bibr"> Song et al., 2004 </xref> ) and <span class="gene">GmConsensus40</span> ( <xref rid="B17" ref-type="bibr"> Hyten et al., 2010 </xref> ), which integrated five and three different populations, respectively. Recent advances in next-generation sequencing technologies have provided effective platforms for the detection of high-quality single nucleotide polymorphism (SNP) markers for the genotyping of mapping populations ( <xref rid="B40" ref-type="bibr"> Varshney et al., 2009 </xref> ), and genotyping-by-sequencing (GBS) has been successfully used for genetic studies in a variety of species ( <xref rid="B45" ref-type="bibr"> Wu et al., 2010 </xref> ; <xref rid="B8" ref-type="bibr"> Chen et al., 2014 </xref> ; <xref rid="B21" ref-type="bibr"> Li et al., 2015 </xref> ). However, the limitations to GBS include a relatively large proportion of missing data, and a small but rarely corrected percentage of SNP genotyping sequencing errors always occur during GBS. One option for imputing missing SNP data is the sliding-window approach, where adjacent SNPs with the same genotype in an interval are combined into bins that demarcate recombination locations across the whole population ( <xref rid="B16" ref-type="bibr"> Huang et al., 2009 </xref> ). The bin-map method is more powerful for detecting QTL than traditional methods and has been employed for the fine mapping of yield-associated loci in rice ( <xref rid="B52" ref-type="bibr"> Yu et al., 2011 </xref> ), maize ( <xref rid="B8" ref-type="bibr"> Chen et al., 2014 </xref> ), and others. In soybean, <xref rid="B47" ref-type="bibr"> Xu et al. (2013) </xref> constructed the first bin maps using 246 RILs derived from a <italic> G. max </italic> cross between Magellan and PI 438489B and fine mapped a root-knot nematode <span class="trait">resistance</span> QTL into a bin of 29.7 kb. However, the advantages of high-density linkage mapping have not yet been demonstrated in the soybean inter-specific cross.
					</p>
				
					<p>&nbsp;&nbsp;
						Annual wild soybean ( <italic> Glycine soja </italic> Sieb. and Zucc.) is acknowledged as the wild progenitor of cultivated soybean ( <xref rid="B3" ref-type="bibr"> Broich and Palmer, 1980 </xref> ). During evolution, the <span class="trait">seed</span> bloom (SB) in the wild soybean disappeared, and the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> (SCC) became yellow in the released cultivars. During adaptation to their environments, soybeans with variable days to flowering (DTF) were selected. So far, the genetic bases of these three evolutionary traits, SB, SCC and DTF, have been well-studied in cultivated soybeans. For SB, three genes, <italic> B1 </italic> , <italic> B2 </italic> and <italic> B3 </italic> , have been reported ( <xref rid="B30" ref-type="bibr"> Palmer et al., 2004 </xref> ), while only <italic> B1 </italic> has been mapped to a large region of chromosome (Chr.) 13 ( <xref rid="B7" ref-type="bibr"> Chen and Shoemaker, 1998 </xref> ). At least five genetic loci ( <italic> I </italic> , <italic> T </italic> , <italic> <span class="gene">W1</span> </italic> , <italic> R </italic> , and <italic> O </italic> ) are known to control SCC ( <xref rid="B30" ref-type="bibr"> Palmer et al., 2004 </xref> ), but they represent large intervals, except for <italic> I </italic> , which was located in a 103-kb gene-rich region harboring a cluster of chalcone synthase genes on Chr. 08 ( <xref rid="B10" ref-type="bibr"> Clough et al., 2004 </xref> ). For DTF, remarkable progress has been made in identifying genes, including <italic> <span class="gene">E1</span> </italic> &#x02013; <italic> E8 </italic> ( <xref rid="B1" ref-type="bibr"> Bernard, 1971 </xref> ; <xref rid="B5" ref-type="bibr"> Buzzell, 1971 </xref> ; <xref rid="B4" ref-type="bibr"> Buzzell and Voldeng, 1980 </xref> ; <xref rid="B25" ref-type="bibr"> Mcblain and Bernard, 1987 </xref> ; <xref rid="B2" ref-type="bibr"> Bonato and Vello, 1999 </xref> ; <xref rid="B12" ref-type="bibr"> Cober and Voldeng, 2001 </xref> ; <xref rid="B11" ref-type="bibr"> Cober et al., 2010 </xref> ), <italic> J </italic> ( <xref rid="B32" ref-type="bibr"> Ray et al., 1995 </xref> ), and QTLs ( <xref rid="B24" ref-type="bibr"> Mansur et al., 1996 </xref> ; <xref rid="B29" ref-type="bibr"> Orf et al., 1999 </xref> ; <xref rid="B48" ref-type="bibr"> Yamanaka et al., 2000 </xref> ; <xref rid="B38" ref-type="bibr"> Tasma et al., 2001 </xref> ; <xref rid="B14" ref-type="bibr"> Funatsuki et al., 2005 </xref> ; <xref rid="B34" ref-type="bibr"> Reinprecht et al., 2006 </xref> ) <sup> <xref ref-type="fn" rid="fn01"> 1 </xref> </sup> . However, the allele differentiations between wild and cultivated soybeans in the above evolutionary traits are largely unknown.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> pigmentation is induced by the deposition of a number of flavonoids in the <span class="trait">seed</span> <span class="trait">coat</span> of soybean, and the synthesis of these compounds mainly derives from an anthocyanin biosynthesis branch of the phenylpropanoid pathway of secondary metabolism ( <xref rid="B50" ref-type="bibr"> Yang et al., 2010 </xref> ). According to the mapping results in the present study, there were a total of 54 gene models in the interval of <italic> gSCC1 </italic> ( <italic> G </italic> ) ( <bold> Table <xref ref-type="table" rid="T5"> 5 </xref> </bold> ), among which only one gene model of <italic> <span class="gene">Glyma01g40590</span> </italic> was predicted to involve in anthocyanin accumulation and thus might be the most likely candidate gene of <italic> gSCC1 </italic> ( <italic> G </italic> ). According to the gene annotation database at SoyBase <sup> 1 </sup> , the small physical interval of <italic> gSCC8 </italic> in the NJRINP population encompassed only 63 gene models, among which <italic> <span class="gene">Glyma08g11520</span> </italic> , <italic> <span class="gene">Glyma08g11530</span> </italic> , <italic> <span class="gene">Glyma08g11610</span> </italic> , <italic> <span class="gene">Glyma08g11620</span> </italic> , <italic> <span class="gene">Glyma08g11630</span> </italic> , and <italic> <span class="gene">Glyma08g11650</span> </italic> were predicted to encode <italic> Chalcone </italic> and <italic> Stilbene synthase </italic> family proteins and thus might be the candidate genes of <italic> gSCC8 </italic> ( <italic> I </italic> ) ( <bold> Table <xref ref-type="table" rid="T5"> 5 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The key genes and distinct pathways involved in flowering have been identified through many <italic> Arabidopsis </italic> studies. <italic> Phytochromes </italic> are red-light- and far-red-light-absorbing photoreceptors in the leaves of most plants ( <xref rid="B6" ref-type="bibr"> Chen et al., 2004 </xref> ). Mutations in the <italic> phytochrome </italic> genes cause altered flowering phenotypes in <italic> Arabidopsis </italic> and soybean ( <xref rid="B33" ref-type="bibr"> Reed et al., 1994 </xref> ; <xref rid="B23" ref-type="bibr"> Liu et al., 2008 </xref> ). In the region of the novel locus <italic> qDTF16.1 </italic> detected in the wild soybean, there was only one soybean gene ( <italic> <span class="gene">Glyma16g03030</span> </italic> ) orthologous to <italic> Arabidopsis SPA1 </italic> (suppressor of <italic> <span class="gene">phyA</span> </italic> protein family). <italic> APETALA 2 </italic> ( <italic> <span class="gene">AP2</span> </italic> ) is related to various aspects of plant development, including flowering and <span class="trait">seed</span> mass control ( <xref rid="B18" ref-type="bibr"> Jofuku et al., 2005 </xref> ). In the interval of <italic> qDTF13 </italic> , three gene models, <italic> <span class="gene">Glyma13g28810</span> </italic> , <italic> <span class="gene">Glyma13g29920</span> </italic> and <italic> <span class="gene">Glyma13g30710</span> </italic> , were predicted to encode the <italic> <span class="gene">AP2</span> </italic> domain. A total of 19 gene models were located in the interval of <italic> qDTF02 </italic> , and although there were no homologs of genes for <italic> Arabidopsis </italic> flowering, one gene model, <italic> <span class="gene">Glyma02g00300</span> </italic> , was predicted to be involved in the response to red or far red light. These candidate genes are worth investigating in future research.
					</p>
				
					<p>&nbsp;&nbsp;
						The classic characteristics of the wild soybean <span class="trait">seed</span> <span class="trait">coat</span> are bloom and black <span class="trait">color.</span> Through evolution and artificial selection, the SB disappeared and SCC became yellow, but the location of these wild alleles is largely unknown. Although, three genes of SB, <italic> B1 </italic> , <italic> B2 </italic> and <italic> B3 </italic> , have been reported ( <xref rid="B30" ref-type="bibr"> Palmer et al., 2004 </xref> ), only <italic> B1 </italic> has been mapped to a large region of Chr. 13 ( <xref rid="B7" ref-type="bibr"> Chen and Shoemaker, 1998 </xref> ), while the position of other two loci are unknown. In the present study, three genome regions had allelic differentiation between wild and cultivated soybean in SB, among which <italic> gSB13 </italic> was located in the region of <italic> B1 </italic> , while the other two genes may be in the region of <italic> B2 </italic> or <italic> B3 </italic> . So far, at least five genetic loci ( <italic> I </italic> , <italic> T </italic> , <italic> <span class="gene">W1</span> </italic> , <italic> R </italic> , and <italic> O </italic> ) are known to control SCC ( <xref rid="B30" ref-type="bibr"> Palmer et al., 2004 </xref> ), but only two loci that correspond to <italic> G </italic> and <italic> I </italic> were detected to have allelic differentiation between the wild and cultivated parents in the present study. The two loci explained almost the entire genetic variation of RILs with effects of green and black SCC, but the gene associated with brown SCC was not detected in the two RIL populations, which might be due to the low frequency of brown SCC. Meanwhile, we first detected the interaction between <italic> G </italic> and <italic> I </italic> in both populations at the molecular level.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:CACTA-superfamily transposable element is inserted in MYB transcription factor gene of soybean line producing variegated seeds.</h2>
			<h3>PMID:26360633  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yan F(1), Di S(1), Takahashi R(1)(2).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The R gene of soybean, presumably encoding a MYB transcription factor, controls <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color.</span> The gene consists of multiple alleles, R (black), r-m (black spots and (or) concentric streaks on brown <span class="trait">seed),</span> and r (brown <span class="trait">seed).</span> This study was conducted to determine the structure of the MYB transcription factor gene in a near-isogenic line (NIL) having r-m allele. PCR amplification of a fragment of the candidate gene <span class="gene">Glyma.09G235100</span> generated a fragment of about 1 kb in the soybean cultivar Clark, whereas a fragment of about 14 kb in addition to fragments of 1 and 1.4 kb were produced in L72-2040, a Clark 63 NIL with the r-m allele. Clark 63 is a NIL of Clark with the rxp and Rps1 alleles. A DNA fragment of 13 060 bp was inserted in the intron of <span class="gene">Glyma.09G235100</span> in L72-2040. The fragment had the CACTA motif at both ends, imperfect terminal inverted repeats (TIR), inverse repetition of short sequence motifs close to the 5' and 3' ends, and a duplication of three nucleotides at the site of integration, indicating that it belongs to a CACTA-superfamily transposable element. We designated the element as Tgm11. Overall nucleotide sequence, motifs of TIR, and subterminal repeats were similar to those of Tgm1 and Tgs1, suggesting that these elements comprise a family.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean.</h2>
			<h3>PMID:26263897  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vuong TD(1), Sonah H(2)(3), Meinhardt CG(4), Deshmukh R(5)(6), Kadam S(7), Nelson RL(8), Shannon JG(9), Nguyen HT(10).
				<br/> 
				年份：2015 Aug 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Bi-parental mapping populations have been commonly utilized to identify and characterize quantitative trait loci (QTL) controlling <span class="trait">resistance</span> to soybean cyst nematode (SCN, Heterodera glycines Ichinohe). Although this approach successfully mapped a large number of SCN <span class="trait">resistance</span> QTL, it captures only limited allelic diversity that exists in parental lines, and it also has limitations for genomic resolution. In this study, a genome-wide association study (GWAS) was performed using a diverse set of 553 soybean plant introductions (PIs) belonging to maturity groups from III to V to detect QTL/genes associated with SCN <span class="trait">resistance</span> to HG Type 0. RESULTS: Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip (http// www.soybase.org ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the rhg1 and <span class="gene">Rhg4</span> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color,</span> flower <span class="trait">color,</span> pubescence <span class="trait">color,</span> and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS. CONCLUSIONS: The study showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Wide genetic variation in phenolic compound content of <span class="trait">seed</span> coats among black soybean cultivars.</h2>
			<h3>PMID:25914597</h3>
				

				
			<p>
				作者：Phommalath S(1), Teraishi M(1), Yoshikawa T(1), Saito H(1), Tsukiyama T(1), Nakazaki T(1), Tanisaka T(1), Okumoto Y(1).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Black soybeans have been used as a food source and also in traditional medicine because their <span class="trait">seed</span> coats contain natural phenolic compounds such as proanthocyanidin and anthocyanin. The objective of this research is to reveal the genetic variation in the phenolic compound contents (PCCs) of <span class="trait">seed</span> coats in 227 black soybean cultivars, most of which were Japanese landraces and cultivars. Total phenolics were extracted from <span class="trait">seed</span> coats using an acidic acetone reagent and the proanthocyanidin content, monomeric anthocyanin content, total flavonoids content, total phenolics content, and radical scavenging activity were measured. The cultivars showed wide genetic variation in PCCs. Each of the contents was highly correlated with one another, and was closely associated with radical scavenging activity. PCCs were also moderately associated by flowering date but not associated by <span class="trait">seed</span> weight. Cultivars with purple flowers had a tendency to produce higher PCCs compared with cultivars with white flowers, suggesting that the <span class="gene">W1</span> locus for flower <span class="trait">color</span> can affect phenolic compound composition and content. Our results suggest that developing black soybean cultivars with high functional phenolic compounds activity is feasible.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Methylation Affects Transposition and Splicing of a Large CACTA Transposon from a MYB Transcription Factor Regulating Anthocyanin Synthase Genes in Soybean <span class="trait">Seed</span> Coats</h2>
			<h3>PMID:25369033</h3>
				

				
					<h3>PMCID:4219821</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We determined the molecular basis of three soybean lines that vary in <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> at the R locus which is thought to encode a MYB transcription factor. RM55-r(m) is homozygous for a mutable allele (r(m)) that specifies black and brown striped seeds; RM30-R* is a stable black revertant isoline derived from the mutable line; and RM38-r has brown <span class="trait">seed</span> coats due to a recessive r allele shown to translate a truncated MYB protein. Using long range PCR, 454 sequencing of amplicons, and whole genome re-sequencing, we determined that the variegated RM55-r(m) line had a 13 kb CACTA subfamily transposon insertion (designated TgmR*) at a position 110 bp from the beginning of Intron2 of the R locus, <span class="gene">Glyma09g36983.</span> Although the MYB encoded by R was expressed at only very low levels in older <span class="trait">seed</span> coats of the black revertant RM30-R* line, it upregulated expression of anthocyanidin synthase genes (ANS2, ANS3) to promote the synthesis of anthocyanins. Surprisingly, the RM30-R* revertant also carried the 13 kb TgmR* insertion in Intron2. Using RNA-Seq, we showed that intron splicing was accurate, albeit at lower levels, despite the presence of the 13 kb TgmR* element. As determined by whole genome methylation sequencing, we demonstrate that the TgmR* sequence was relatively more methylated in RM30-R* than in the mutable RM55-r(m) progenitor line. The stabilized and more methylated RM30-R* revertant line apparently lacks effective binding of a transposae to its subterminal repeats, thus allowing intron splicing to proceed resulting in sufficient MYB protein to stimulate anthocyanin production and thus black <span class="trait">seed</span> coats. In this regard, the TgmR* element in soybean resembles McClintock's Spm-suppressible and change-of-state alleles of maize. This comparison explains the opposite effects of the TgmR* element on intron splicing of the MYB gene in which it resides depending on the methylation state of the element.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We have identified the molecular basis of some of the classical loci leading to anthocyanin production in soybean ( <italic> Glycine max </italic> ) <span class="trait">seed</span> and plant parts. The majority of the cultivated soybean varieties have yellow seeds with hila of various colors but ancestral soybeans ( <italic> Glycine soja </italic> ) have pigmented <span class="trait">seed</span> coats. The pigmentation in <span class="trait">seed</span> <span class="trait">coat,</span> hilum and pubescence (trichome hairs) is determined by three independent loci, <italic> I </italic> ( <italic> Inhibitor </italic> ), <italic> R </italic> , and <italic> T </italic> ( <italic> Tawny </italic> ). The <italic> I </italic> locus controls distribution of anthocyanin and proanthocyanidin pigments and comprises the multigenic, inverted repeat region of chalcone synthase (CHS) genes ( <italic> <span class="gene">CHS1</span> </italic> , <italic> <span class="gene">CHS3</span> </italic> and <italic> CHS4 </italic> ) <xref rid="pone.0111959-Todd1" ref-type="bibr"> [1] </xref> . In its dominant form <italic> I </italic> , silences the expression of all nine <span class="gene">CHS</span> gene family via short interfering RNAs in a tissue-specific manner in the <span class="trait">seed</span> <span class="trait">coat</span> <xref rid="pone.0111959-Tuteja1" ref-type="bibr"> [2] </xref> . Since <span class="gene">CHS</span> is the first committed enzyme in the anthocyanin pathway, the <italic> I </italic> genotypes result in yellow <span class="trait">seed</span> coats. Colored soybeans have recessive <italic> I </italic> alleles and the various colors of the <span class="trait">seed</span> coats are influenced by the <italic> R </italic> and <italic> T </italic> loci. Black ( <italic> i,R,T </italic> ), imperfect-black ( <italic> i,R,t </italic> ), brown ( <italic> i,r,T </italic> ) and buff ( <italic> i,r,t </italic> ) <span class="trait">seed</span> coats. The <italic> T </italic> locus encodes a flavonoid 3&#x02032;hydroxylase ( <italic> F3&#x02032;H </italic> ) gene, the expression of which drives the synthesis of the anthocyanin cyanidin pathway branch <xref rid="pone.0111959-Zabala1" ref-type="bibr"> [3] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Recently it has been proposed that <italic> R </italic> maps to a locus encoding a MYB transcription factor which may positively regulate the expression of a <italic> UDP-glucose: flavonoid 3-O-glucosyltransferase </italic> ( <italic> UF3GT </italic> ) gene that functions in the last step of anthocyanin synthesis <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> . These authors identified the recessive, brown <italic> r </italic> allele to affect the expression of a <span class="trait">seed</span> coat-specific R2R3 MYB transcription factor gene, <span class="gene">Glyma09g36983</span> from the sequenced soybean genome as a strong candidate for the <italic> R </italic> locus. In addition, their qRT-PCR analysis supported a correlation between the mRNA levels of the putative <italic> R </italic> gene and a UDP-glucose: flavonoid 3-O-glucosyltransferase ( <italic> UF3GT </italic> ) gene <span class="gene">(Glyma08g07130)</span> that catalyzes the glycosylation of cyanidins to anthocyanins. The recessive <italic> r </italic> (brown) allele at <span class="gene">Glyma09g36983</span> contained a single base deletion that would result in a truncated protein and did not upregulate this <italic> UF3GT </italic> gene. Based on a combined analysis of transcriptome and metabolome data, another recent study identified 20 anthocyanin, flavonoid and phenylpropanoid isogenes that were differentially expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) seeded soybean isolines <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> . This same study suggested that <italic> R </italic> locus candidate might be a transcription factor at the distal end of chromosome <span class="gene">Gm09</span> at or near <span class="gene">Glyma09g36840,</span> 120 kb from the <span class="gene">Glyma09g36983</span> MYB factor mapped by Gillman et al <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Using primers based on the <span class="gene">Glyma09g36983</span> as the <italic> R </italic> locus <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> , we here report finding a large 13 kb CACTA transposable element insertion in the second intron of the MYB factor in the RM55- <italic> r <sup> m </sup> </italic> mutable soybean line along with a 929 bp PCR product representing excision products that restore the MYB transcription factor sequence. The transposon represents a second CACTA subfamily in soybean based on its transposase and subterminal repeat sequences that most resemble the first CACTA defective element described in soybean, <italic> Tgm1 </italic> <xref rid="pone.0111959-Vodkin1" ref-type="bibr"> [7] </xref> , and is more distant to <italic> Tgmt </italic> <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> and <italic> Tgm9 </italic> <xref rid="pone.0111959-Xu1" ref-type="bibr"> [9] </xref> . Surprisingly the 13 kb CACTA transposon (designated <italic> TgmR* </italic> ) is retained in the <italic> R* </italic> allele of a stable black-seeded revertant isoline. Using quantitative RNA-Seq data from a developmental series of <span class="trait">seed</span> coats from the black-seeded RM30- <italic> R </italic> * revertant and the RM38- <italic> r </italic> brown-seeded isoline, we also demonstrate that the MYB factor encoded by the <italic> R </italic> gene appears to induce the expression of anthocyanidin synthase ( <italic> <span class="gene">ANS</span> </italic> ) genes at late stages of <span class="trait">seed</span> development in the black RM30- <italic> R* </italic> line. Thus, our data further support <span class="gene">Glyma09g3698</span> as the <italic> R </italic> locus in soybean and show that it regulates <italic> <span class="gene">ANS</span> </italic> genes and potentially other genes in the anthocyanin pathway despite a very low level of <italic> R </italic> gene transcripts. The <italic> R </italic> encoded MYB protein possesses four of the amino acids present in the subgroup five of MYB proteins that are involved in anthocyanin regulation by transcriptional activation <xref rid="pone.0111959-Stracke1" ref-type="bibr"> [10] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Agarose gel displaying a &#x0223c;14 kb PCR fragment in the RM55- <italic> r <sup> m </sup> </italic> mutable and RM30 <italic> -R </italic> * revertant alleles of the <italic> R </italic> locus. A small 929 bp fragment was the only amplification product from the <italic> r </italic> allele of the RM38- <italic> r </italic> brown-seeded line and it was the expected size of the DNA fragment comprised between the two oligo DNA primers (FP, forward primer: R6990FP1 and RP, reverse primer: R6990RPB) designed to amplify the 5&#x02032;end portion of the <span class="gene">Glyma09g36983</span> gene encoding a putative R2R3 MYB transcription factor. The 929 bp fragment was also an amplification product from the DNA of the mutable RM55- <italic> r <sup> m </sup> </italic> . These results predict an insertion of 13 kb in one allele of the mutable RM55- <italic> r <sup> m </sup> </italic> line that is maintained in the revertant RM30- <italic> R </italic> * black <span class="trait">seed</span> isoline. (B) Agarose gel displaying a 1.45 kb PCR fragment of the <span class="gene">Glyma09g36983</span> gene amplified from four other soybean varieties with the standard <italic> R </italic> allele as well as from two soybean lines with an <italic> r </italic> allele (see <xref ref-type="table" rid="pone-0111959-t001"> Table 1 </xref> for full genotypes). The reverse primer in these instances was R6990RP1, situated 491 bp downstream from R6990RPB used in (A). The size marker &#x0201c;M&#x0201d; is a 1 kb ladder.
					</p>
				
					<p>&nbsp;&nbsp;
						We compared the expression <span class="gene">Glyma09g36983</span> MYB transcription factor in the <span class="trait">seed</span> coats of the black revertant RM30- <italic> R* </italic> soybean line to that in the defective brown RM38- <italic> r </italic> soybean line at various stages of <span class="trait">seed</span> development. Total RNA was extracted from <span class="trait">seed</span> coats at five stages of <span class="trait">seed</span> development ( <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4A </xref> ) following a modified method that prevents RNA adhesion to the proanthocyanidins (tannins) that accumulate in the vacuoles of pigmented <span class="trait">seed</span> coats <xref rid="pone.0111959-Wang1" ref-type="bibr"> [17] </xref> , <xref rid="pone.0111959-Wang2" ref-type="bibr"> [18] </xref> . The five stages of <span class="trait">seed</span> development were chosen based on the fresh weight of entire seeds: 100&#x02013;200, 200&#x02013;300, 300&#x02013;400 and 400&#x02013;500 mg with green cotyledons, and a later stage in which <span class="trait">seed</span> desiccation had initiated judging by the yellowing of cotyledons and a lower fresh weight of 300&#x02013;400 mg.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Soybean <span class="trait">seed</span> developmental stages for the RM38- <italic> r </italic> (brown), RM55- <italic> r <sup> m </sup> </italic> (variegated) and RM30- <italic> R* </italic> stable black revertant. Encircled in red are the five developmental stages (in mg <span class="trait">seed</span> fresh weight) chosen for the RNA-Seq analysis. See <xref ref-type="supplementary-material" rid="pone.0111959.s006"> Table S1 </xref> for the sequence read counts of each sample which ranged from from 30 to 77 million. (B) Expression of <span class="gene">Glyma09g36983</span> in RPKMs plotted against the five stages of <span class="trait">seed</span> development in mg <span class="trait">seed</span> fresh weight as shown in (A) above. 1: 100&#x02013;200 mg, 2: 200&#x02013;300 mg, 3: 300&#x02013;400 mg, 4: 400&#x02013;500 mg and 5: 300&#x02013;400 mg as the seeds enter desiccation. The solid red line represents transcripts derived from <span class="trait">seed</span> coats of the RM38- <italic> r </italic> brown-seeded line without the <italic> TgmR* </italic> but with a &#x0201c;C&#x0201d;-nt deletion in Exon2. The blue line represent the expression in <span class="trait">seed</span> coats of the RM30- <italic> R* </italic> black revertant line with the <italic> TgmR* </italic> insertion in Intron2.
					</p>
				
					<p>&nbsp;&nbsp;
						The RNA-Seq results reflecting the level of transcript reads aligning to the <span class="gene">Glyma09g36983</span> <span class="gene">(Glyma09g36990</span> in Phytozome <italic> G. max </italic> _109 v.1a-1.0) sequence are given in RPKMs (reads per kilobase of gene model per million mapped reads) as shown in in <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> (top row) and graphically in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4B </xref> . This MYB transcription factor was not expressed at the early stages examined (100&#x02013;200 mg) and at very low levels later in development when the anthocyanins and proanthocyanins start pigmenting the <span class="trait">seed</span> <span class="trait">coat.</span> The quantitative levels were 1.07 RPKM in the RM30- <italic> R* </italic> black revertant that contains the <italic> TgmR* </italic> insertion and only 2.19 in the RM38- <italic> r </italic> brown isoline. This RPKM ratio of 0.49 (p-value: 0.069) indicates no significant differences of expression of the <span class="gene">Glyma09g36983</span> gene in the RM30- <italic> R </italic> * (black) and RM38- <italic> r </italic> (brown) <span class="trait">seed</span> coats at the peak of its measured expression in the 400&#x02013;500 mg <span class="trait">seed</span> weight.
					</p>
				
					<p>&nbsp;&nbsp;
						RPKMs are shown for <span class="trait">seed</span> coats from <span class="trait">seed</span> of the five <span class="trait">seed</span> fresh weight ranges denoted 100&#x02013;200, 200&#x02013;300, 300&#x02013;400, 400&#x02013;500 mg and 300&#x02013;400 mg desiccating <span class="trait">seed</span> coats as shown in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4A </xref> . Ratios and p-values are shown for the 400&#x02013;500 mg weight range, and date for the <span class="gene">ANS</span> genes at this stage are in Bold.
					</p>
				
					<p>&nbsp;&nbsp;
						We examined the expression of three MYB transcription factors, <span class="gene">Glyma09g36966,</span> <span class="gene">Glyma09g37010,</span> and <span class="gene">Glyma18g49670</span> that have transcript sequence similarity of 90%, 87%, and 84% respectively to <span class="gene">Glyma09g36983.</span> The level of expression of these three genes as shown in <xref ref-type="table" rid="pone-0111959-t002"> Tables 2 </xref> and <xref ref-type="table" rid="pone-0111959-t003"> 3 </xref> is even lower or nil, and the RPKM values associated with <span class="gene">Glyma09g36966</span> may reflect the number of sequence reads aligning to the 530-nt stretch with 90% transcript sequence similarity to <span class="gene">Glyma09g36983.</span> Three mismatches was the cutoff used to determine the number of RNA sequence reads aligning to a given gene transcript using Bowtie. A more stringent alignment search with zero mismatches supported the above statement and strongly suggested that none of the three related MYB transcription factors are expressed in the soybean <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						Once we had evidence that the <italic> TgmR </italic> * insertion slightly suppressed the expression of the <italic> R </italic> locus gene, <span class="gene">Glyma09g36983,</span> at late stages of <span class="trait">seed</span> development, we questioned at which point in the anthocyanin biosynthetic pathway the encoded MYB transcription factor exerted its regulatory control. Towards that end, we compared the level of mRNA expression of well characterized genes in the pathway in the RNAseq data sets obtained from the black and brown <span class="trait">seed</span> coats of the RM30- <italic> R </italic> * revertant and RM38- <italic> r </italic> defective line, at all five stages of <span class="trait">seed</span> development under study ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Contrary to the difference observed for the expression of the MYB gene in the two lines in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4B </xref> , where higher levels were measured in the brown RM38- <italic> r </italic> <span class="trait">seed</span> coats, all other pathway genes analyzed had higher RPKM expression values in the black RM30- <italic> R </italic> * line at the late stages of <span class="trait">seed</span> development ( <xref ref-type="fig" rid="pone-0111959-g005"> Figures 5 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> 6 </xref> , <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ). These increases occurred at the developmental time when the <italic> R </italic> locus gene, <span class="gene">Glyma09g36983,</span> is activated. One exception is the <italic> ANR </italic> genes for which the low level of expression at the late stages of <span class="trait">seed</span> development was maintained in the <span class="trait">seed</span> coats of the black RM30- <italic> R </italic> * line as well as in the brown RM38- <italic> r </italic> line ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ). This is consistent with the function of this gene's product in directing the synthesis of proanthocyanidins now in competition for its substrate that is being diverted towards anthocyanin synthesis in the black <span class="trait">seed</span> coats at the late stages of <span class="trait">seed</span> development as shown in <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript levels are in RPKMs plotted against the same five stages of <span class="trait">seed</span> <span class="trait">coat</span> development as shown in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4 </xref> for RM30- <italic> R </italic> * where the <italic> R* </italic> allele is interrupted by the 13 kb <italic> TgmR* </italic> insertion (profile in blue) and the RM38- <italic> r </italic> line where the <italic> r </italic> allele is not interrupted by <italic> TgmR* </italic> but has a &#x0201c;C&#x0201d;-nt deletion in Exon2 (red profile). Graphs have different scales. The Glyma models for the indicated genes are as follows: <italic> <span class="gene">CHS7</span> </italic> <span class="gene">(Glyma01g43880.1);</span> <italic> CHS8 </italic> <span class="gene">(Glyma11g01350.1);</span> <italic> <span class="gene">CHI1A</span> </italic> <span class="gene">(Glyma20g38560.1);</span> <italic> CHI2 </italic> <span class="gene">(Glyma20g38580.1);</span> <italic> F3&#x02032;H </italic> <span class="gene">(Glyma06g21920.1);</span> <italic> F3&#x02032;5&#x02032;H </italic> <span class="gene">(Glyma13g04210.1);</span> <italic> <span class="gene">F3H</span> </italic> <span class="gene">(Glyma02g05450.1);</span> <italic> <span class="gene">DFR1</span> </italic> <span class="gene">(Glyma14g07940.1);</span> <italic> <span class="gene">DFR2</span> </italic> <span class="gene">(Glyma17g37060.1.</span> See <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> for the pathway abbreviations.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcript levels are in RPKMs plotted against the same five stages of <span class="trait">seed</span> <span class="trait">coat</span> development as shown in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4 </xref> for RM30- <italic> R </italic> * where the <italic> R* </italic> allele is interrupted by the 13 kb <italic> TgmR* </italic> insertion (profile in blue) and the RM38- <italic> r </italic> line where the <italic> r </italic> allele is not interrupted by <italic> TgmR* </italic> but has a &#x0201c;C&#x0201d;-nt deletion in Exon2 (red profile). Graphs have different scales. The Glyma models for the indicated genes are as following. (A) <span class="gene">ANS3</span> <span class="gene">(Glyma11g03010.1);</span> (B) UFGT2 <span class="gene">(Glyma08g07130.1);</span> (C) <span class="gene">ANR1</span> <span class="gene">(Glyma08g06630.1)</span> and (D) AOMT <span class="gene">(Glyma05g36210.1.</span> See <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> for the pathway abbreviations.
					</p>
				
					<p>&nbsp;&nbsp;
						In display are the three branches leading to the synthesis of the purple delphinidins, pink pelargonidins, red cyanidins as well as the brown proanthocyanidins. The major anthocyanin in black soybean <span class="trait">seed</span> coats is cyanidin-3-glycoside (C3G) at &#x0223c;70%. The brown <span class="trait">seed</span> coats contain mostly proanthocyanidins. Enzymes and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> loci are abbreviated in uppercase letters: <italic> I </italic> , chalcone synthase (CHS); <italic> <span class="gene">W1</span> </italic> , flavonoid 3&#x02032;5&#x02032;-hydroxylase (F3&#x02032;5&#x02032;H); <italic> T </italic> , flavonoid 3&#x02032;-hydroxylase (F3&#x02032;H); <italic> Wp </italic> , flavanone 3-hydroxylase (F3H); <italic> <span class="gene">W3</span> </italic> , dihydroflavonol-4-reductase (DFR); Annthocyanidin synthase (ANS) also called leucoanthocyanidin dioxygenase (LDOX); UDP-flavonoid glucosyltransferase (UFGT); Leucoanthocyanidin reductase (LAR); Anthocyanidin reductase (ANR). The MYB transcription factor encoded by the <italic> R* </italic> allele <span class="gene">Glyma09g36983</span> gene enhances expression/accumulation of <span class="gene">ANS</span> transcripts as shown in <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Several of the genes involved in the anthocyanin pathway ( <italic> <span class="gene">CHS</span> </italic> , <italic> F3&#x02032;H </italic> , <italic> <span class="gene">F3H</span> </italic> , <italic> DFR </italic> , <italic> <span class="gene">ANS</span> </italic> , <italic> <span class="gene">ANR1</span> </italic> and <italic> AOMT </italic> ) are expressed at higher level early in <span class="trait">seed</span> development (100&#x02013;200 mg <span class="trait">seed</span> fresh weight) and the amount of transcripts decline in <span class="trait">seed</span> coats of both lines as <span class="trait">seed</span> growth advances ( <xref ref-type="fig" rid="pone-0111959-g005"> Figures 5 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> 6 </xref> ). However, this decline is arrested or reversed for all these genes in the <span class="trait">seed</span> coats of the black-seeded RM30- <italic> R </italic> * line at the third and fourth stages of <span class="trait">seed</span> development. For those genes that are not expressed or expressed at very low level earlier in development ( <italic> <span class="gene">CHI</span> </italic> , <italic> F3&#x02032;5&#x02032;H </italic> and <italic> UFGT </italic> ), the expression increased to a slightly higher level in the <span class="trait">seed</span> coats of the black RM30- <italic> R </italic> * seeds during the late stages of <span class="trait">seed</span> development. Despite the two <italic> UFGT </italic> genes ( <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">UGT78K1</span> </italic> ) transcript sequences being 92% similar, the sequence reads from the <span class="trait">seed</span> <span class="trait">coat</span> RNA samples aligned better to the <italic> <span class="gene">UGT78K2</span> </italic> transcript and differentially between the two lines ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> ). This suggests that the <italic> <span class="gene">UGT78K1</span> </italic> gene may not be expressed in the <span class="trait">seed</span> <span class="trait">coat</span> but rather in the cotyledons as it has been reported previously <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Although all the mentioned genes appear to be differentially expressed in the <span class="trait">seed</span> coats of the black and brown seeds, the ones that show the highest RPKM ratios and lowest p-values are the <italic> <span class="gene">ANS</span> </italic> genes with an RM30/RM38 of 10.17 (p-value 2.12 E-38) and 10.51 (p-value 2.71 E-40) respectively, at the 400&#x02013;500 mg <span class="trait">seed</span> weight ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ). The transcript sequences of the two <italic> <span class="gene">ANS</span> </italic> genes are 94% similar and it is not surprising that the numbers of RNA sequence reads for each gene ( <italic> ANS2 </italic> and <italic> <span class="gene">ANS3</span> </italic> ) are very similar and show parallel patterns of expression in the five samples measured from the two lines (RM30- <italic> R </italic> * and RM38- <italic> r </italic> ). <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> shows the differences in the number of sequence reads in RPKMs for the <italic> <span class="gene">ANS3</span> </italic> gene in the RM30- <italic> R </italic> * black and the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						We inspected the alignment of the RNA-Seq reads from the Rm30- <italic> R* </italic> and RM38- <italic> r </italic> lines at all five developmental stages ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ) against the 714-nt (3-exon gene model) transcript sequence with Bowtie 1. The results presented in <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> show clearly the numbers of sequence reads and their alignments with a total coverage of the <span class="gene">Glyma09g36983</span> gene transcript sequence in both lines. The expression level of the <italic> R </italic> gene is low in black and brown <span class="trait">seed</span> coats with the only significant difference manifested in <span class="trait">seed</span> coats of the 400&#x02013;500 mg seeds, with the brown <span class="trait">seed</span> coats accumulating slightly more than double the level in the black <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> ) as previously indicated in <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> with the RM30- <italic> R </italic> */RM38- <italic> r </italic> ratio of 0.49. As stated earlier, these alignment results also show that the <span class="gene">Glyma09g36983</span> gene is transcribed all the way to the 3&#x02032;end in the RM30- <italic> R* </italic> black <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> ) despite the large transposon insertion at 110 nt from the beginning of Intron2. These data indicate that despite a very low level of transcripts, they are cleanly processed at the intron exon boundaries.
					</p>
				
					<p>&nbsp;&nbsp;
						The pair of graphs in each row represent the distribution of non-normalized RNAseq reads that align to the 714 bp transcript sequence from the <span class="gene">Glyma09g36983</span> gene at each of the five stages of <span class="trait">seed</span> <span class="trait">coat</span> development indicated in the margin to the right (described in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4 </xref> ). The plots in the left column represent the distributions of sequence reads derived from the <span class="trait">seed</span> coats of the RM30- <italic> R </italic> * that is interrupted by the <italic> TgmR* </italic> transposon and the plots in the right column represent the RNAseq reads derived from the RM38- <italic> r </italic> line with the uninterrupted <italic> R </italic> gene but containing the &#x0201c;C&#x0201d;-nt deletion at 222 bp of the transcript. The numbers of RNAseq reads matching a given position of the <italic> R </italic> gene transcript are plotted against the position along the reference <span class="gene">Glyma09g36983</span> transcript sequence containing the three exons and no introns. No expression was detected in the early 100&#x02013;200 mg <span class="trait">seed</span> developmental stage in the <italic> RM30-R* </italic> line. The red arrow points to the location of the <italic> r </italic> allele &#x0201c;C&#x0201d;-nt deletion at 222 bp of the transcript.
					</p>
				
					<p>&nbsp;&nbsp;
						On the other hand, the alignment of RNA-Seq reads from the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats to the 714-nt sequence representing the standard <span class="gene">Glyma09g36983</span> <italic> R </italic> allele transcript, shows a valley observed at 222 bp, the location of the deleted &#x0201c;C&#x0201d;-nt ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> red arrow). Bowtie 1 does not align any insertion or deletions, thus explaining the lack of alignments at the location of an intron or even a single nucleotide insertion or deletion. These data are additional evidence that RNA sequence reads from the RM38- <italic> r </italic> allele were missing a C at position 222. This conclusion was also directly confirmed by visual inspection showing that sequences aligned properly to either side of the C at nucleotide 222 but none contained the C at position 222. In addition, we performed the alignment of the RM38- <italic> r </italic> RNA-Seq reads to the mutant transcript sequence (with the C deletion) and the valley observed at position 222 in the graph of alignments was no longer present as shown in <xref ref-type="supplementary-material" rid="pone.0111959.s001"> Figure S1 </xref> indicating proper alignment over the full length transcript.
					</p>
				
					<p>&nbsp;&nbsp;
						Using the Bowtie 1 alignments, we determined the distribution of sequence reads along the length of the 15,450-nt genomic sequence obtained from the black <span class="trait">seed</span> RM30- <italic> R* </italic> line that comprises the <italic> R </italic> locus gene <span class="gene">Glyma09g36983</span> interrupted by the 13,024-nt <italic> TgmR* </italic> insertion in Intron2. A representation of the distribution of sequence reads at the five stages of <span class="trait">seed</span> development in the black and brown <span class="trait">seed</span> lines is shown in <xref ref-type="supplementary-material" rid="pone.0111959.s002"> Figure S2 </xref> . It is clear that sequences aligned all through the <italic> TgmR </italic> * sequence portion in RNA samples from both lines at all five stages of <span class="trait">seed</span> <span class="trait">coat</span> development examined including a low level from ORF2. Whether these sequences originate from <italic> TgmR* </italic> or from the other genome regions containing similar elements ( <xref ref-type="fig" rid="pone-0111959-g003"> Figure 3 </xref> ) cannot be determined from the alignments. Manual inspection of the transcript sequences did not find any chimeric sequences at the intron junctions or the boundaries of the <italic> TgmR* </italic> element, which again indicates clean processing of the introns from the gene with complete and precise removal of the 13 kb <italic> TgmR* </italic> element located within Intron2 through the normal intron processing.
					</p>
				
					<p>&nbsp;&nbsp;
						The genomic sequences of the alleles from the variegated ( <italic> r <sup> m </sup> </italic> ) and black ( <italic> R* </italic> ) <span class="trait">seed</span> lines reported here showed that both contained the <italic> TgmR* </italic> transposon insertion in Intron2. Could the instability of these alleles be dictated by their methylation levels in the two isolines? To answer this question we sequenced the genomes of three soybean lines, RM55- <italic> r <sup> m </sup> </italic> (variegated <span class="trait">seed),</span> RM30- <italic> R* </italic> (black <span class="trait">seed)</span> and UC44- <italic> R </italic> (black <span class="trait">seed</span> with no <italic> TgmR* </italic> insertion), using the bisulfite method (BS-seq) first described by Cokus et al, 2008 <xref rid="pone.0111959-Cokus1" ref-type="bibr"> [29] </xref> to <span class="trait">yield</span> 226, 209, and 235 million total reads respectively ( <xref ref-type="supplementary-material" rid="pone.0111959.s006"> Table S1 </xref> ). The resulting bisulfite sequences were processed through the three-letter aligner program, Bismark <xref rid="pone.0111959-Krueger1" ref-type="bibr"> [30] </xref> to determine the type and degree of &#x0201c;C&#x0201d;-methylation of the transposon and the upstream (35,513 bp) and the downstream (25,417 bp) regions at the <italic> R </italic> <span class="gene">(Glyma09g36983)</span> loci. The results were compiled in <xref ref-type="table" rid="pone-0111959-t004"> Table 4 </xref> and revealed a higher degree of methylation for the transposon ( <italic> TgmR* </italic> ) sequences, most notably, in the RM30- <italic> R* </italic> black-seeded revertant line with an almost double (5.8%) level of methylated C in the CHH context compared to 3% in the RM55- <italic> r <sup> m </sup> </italic> (variegated <span class="trait">seed)</span> and 2.9% in the UC44- <italic> R </italic> (black <span class="trait">seed</span> with no <italic> TgmR* </italic> insertion). The percentage of CpG methylation in sequences with similarity to <italic> TgmR* </italic> from both RM30- <italic> R* </italic> (86.2%) and UC44- <italic> R </italic> (84.7%) were substantially higher than in RM55- <italic> r <sup> m </sup> </italic> (77.6%). CHG methylation in sequences with similarity to <italic> TgmR* </italic> was also higher for RM30- <italic> R* </italic> (74.3%) compared to RM55- <italic> r <sup> m </sup> </italic> (63.1%). Although UC44- <italic> R </italic> does not contain <italic> TgmR* </italic> in the <italic> R </italic> locus, there are highly repeated regions with sequence similarity to the transposon elsewhere in the genome as previously discussed.
					</p>
				
					<p>&nbsp;&nbsp;
						All graphs were created by the sequence alignment tool of the Integrative Genomics Viewer (IGV) browser. (A) <italic> TgmR* </italic> methylation regions. The 13 bp CACTA inverted repeat ends and subterminal repeats are marked IR. Lower levels of methylated sequences (marked by red arrows) are found aligning to the subterminal repeats of the <italic> TgmR </italic> * element and the 3&#x02032;-end of the predicted <span class="gene">ORF2</span> transcript in the striped mutable RM55- <italic> r <sup> m </sup> </italic> line. The black-seeded RM30- <italic> R* </italic> line containing the <italic> TgmR* </italic> insertion and the black-seeded UC44- <italic> R </italic> , line which lacks the <italic> TgmR* </italic> insertion, present nearly identical patterns of methylation at those three differentiated areas of the <italic> TgmR </italic> * element. (B) Upstream regions and (C) downstream regions of the <italic> TgmR* </italic> insertion site in the <italic> R </italic> gene. A few methylation differences appear between the three lines in the upstream and downstream regions, but the most distinct are those for UC44- <italic> R </italic> (Williams self-black <span class="trait">seed)</span> which may represent varietal differences. Gene represented by green arrows are: A <span class="gene">(Glyma09g36941);</span> B <span class="gene">(Glyma09g36950);</span> C <span class="gene">(Glyma09g36966);</span> D <span class="gene">(Glyma09g36983);</span> E <span class="gene">(Glyma09g37000)</span> and F <span class="gene">(Glyma09g37010).</span> The upstream and downstream schematics are not drawn to scale.
					</p>
				
					<p>&nbsp;&nbsp;
						We have previously identified and characterized the expression of several key loci controlling the synthesis of anthocyanins in the soybean <span class="trait">seed</span> coats. The <italic> I </italic> locus was found to be a cluster of <italic> <span class="gene">CHS</span> </italic> genes that produce small RNAs that downregulate <span class="gene">CHS</span> transcripts <xref rid="pone.0111959-Tuteja1" ref-type="bibr"> [2] </xref> , the <italic> T </italic> locus encodes a flavonoid 3&#x02032;hydroxylase ( <italic> F3&#x02032;H </italic> ) gene <xref rid="pone.0111959-Zabala1" ref-type="bibr"> [3] </xref> , and the <italic> Wp </italic> is a flavanone 3-hydroxylase ( <italic> <span class="gene">F3H</span> </italic> ) gene <xref rid="pone.0111959-Zabala3" ref-type="bibr"> [32] </xref> ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ). However, the <italic> R </italic> locus that with <italic> T </italic> determines the various colors of the <span class="trait">seed</span> <span class="trait">coat</span> in genotypes with a recessive <italic> I </italic> allele ( <italic> i </italic> ) is not completely understood. Recently, it was proposed that <italic> R </italic> maps to a locus, <span class="gene">Glyma09g36983,</span> encoding a MYB transcription factor that may positively regulate the expression of a <italic> UF3GT </italic> gene, which catalyzes the glycosylation of cyanidins to anthocyanins in the last step of anthocyanin synthesis <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> . However, another report suggested a different transcription factor location 120 kb away on the same chromosome <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The results we present here support the location of the <italic> R </italic> locus to be the MYB encoded by <span class="gene">Glyma09g36983</span> as determined recently by Gillman et al., 2011 <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> based on mapping polymorphisms associated with standard <italic> R </italic> and <italic> r </italic> alleles, one of which was a &#x0201c;C&#x0201d; deletion that truncates the MYB protein in the standard brown <italic> r </italic> allele. Using primers designed to <span class="gene">Glyma09g36983,</span> our analysis of two isolines including a mutable RM55- <italic> r <sup> m </sup> </italic> line with black and brown striped <span class="trait">seed</span> phenotype and RM30- <italic> R* </italic> , a stable black-seeded revertant isoline, revealed the existence of a 13 kb CACTA transposon insertion ( <italic> TgmR* </italic> ) at the beginning of the <span class="gene">Glyma09g36983</span> gene Intron2 in both of these lines ( <xref ref-type="fig" rid="pone-0111959-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g003"> 3 </xref> ). The transposon was absent in the brown-seeded RM38- <italic> r </italic> line. Thus, our data directly implicate the structural variations in <span class="gene">Glyma09g36983</span> as producing the phenotypic differences of the <italic> R </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Based on differences in the subterminal repeats, we previously predicted the existence of two CACTA subfamilies in soybean <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> . The novel <italic> TgmR </italic> * element is 13 kb in length and has all the characteristic features of a CACTA transposon based on the three base duplication (ATG) at the site of insertion, the CACTA ends and the direct and reverse repeats at the subterminal ends that can fold into complex secondary structures ( <xref ref-type="fig" rid="pone-0111959-g003"> Figure 3A </xref> , <xref ref-type="supplementary-material" rid="pone.0111959.s004"> Files S2 </xref> and <xref ref-type="supplementary-material" rid="pone.0111959.s005"> S3 </xref> ). <italic> TgmR* </italic> confirms the proposed existence of two subfamilies of CACTA transposons in soybean which was based on the two distinct types of repeated motif at the subterminal ends ( <xref ref-type="fig" rid="pone-0111959-g003"> Figure 3B </xref> ). The motif in <italic> TgmR* </italic> is more closely related to the one in <italic> Tgm1 </italic> , a defective element interrupting a soybean <span class="trait">seed</span> lectin gene <xref rid="pone.0111959-Vodkin1" ref-type="bibr"> [7] </xref> , <xref rid="pone.0111959-Rhodes1" ref-type="bibr"> [14] </xref> , than that of three other previously characterized elements ( <italic> Tgmt*, Tgmw4m, Tgm-Express1 </italic> ) <xref rid="pone.0111959-Zabala3" ref-type="bibr"> [32] </xref> , <xref rid="pone.0111959-Zabala2" ref-type="bibr"> [8] </xref> . In addition, the <italic> TgmR* </italic> 13 kb element encodes two genes, <span class="gene">ORF1</span> (Tnp2-like transposase) and <span class="gene">ORF2</span> of unknown function. The transposase region of <italic> TgmR* </italic> is only 71% similar to the transposase in <italic> Tgmt* </italic> ( <italic> Tgm9 </italic> ). Thus, we propose that <italic> TgmR* </italic> is an active element of the <italic> Tgm1 </italic> type family of elements. Based on RNA-Seq data, both ORFs of <italic> TgmR* </italic> or related element are expressed at low levels in the <span class="trait">seed</span> coats of both black RM30- <italic> R* </italic> and RM38- <italic> r </italic> lines ( <xref ref-type="supplementary-material" rid="pone.0111959.s002"> Figure S2 </xref> ). Because DNA sequences with high similarity to the <span class="gene">ORF1</span> gene are highly repeated throughout the entire genome, the <span class="gene">ORF1</span> RNAs detected in both lines could be derived from sequences other than the one inserted in the <italic> R* </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						The RNA-Seq analysis at multiple stages of <span class="trait">seed</span> development was crafted to determine whether the <italic> TgmR* </italic> insertion interfered with the expression of the <italic> R </italic> <span class="gene">(Glyma09g36983)</span> gene. We contrasted the normalized RPKM reads from the black RM30- <italic> R </italic> * revertant line to those of the brown RM38- <italic> r </italic> line that aligned to the <span class="gene">Glyma09g36983.</span> The results were congruent with <italic> TgmR* </italic> having a negative effect on the expression of the <italic> R* </italic> allele, since a higher number of RNAseq reads were found to align to <span class="gene">Glyma09g36983</span> from the RM38- <italic> r </italic> line than from the RM30- <italic> R </italic> * line at late stages of <span class="trait">seed</span> development ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g004"> Figures 4 </xref> and <xref ref-type="fig" rid="pone-0111959-g009"> 9 </xref> ). The level of expression was very low in both lines, but this is common for other transcription factors. The difference in <italic> R </italic> expression between the two lines is more marked at the 400&#x02013;500 mg <span class="trait">seed</span> developmental stage with a RM30- <italic> R </italic> */RM38- <italic> r </italic> ratio of 0.49 and a p-value of 0.069 ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Although many of the genes in the anthocyanin pathway examined showed differences in expression between the RM30- <italic> R* </italic> black revertant and the RM38- <italic> r </italic> brown isoline, the most significant differences detected were for the <italic> ANS2 </italic> and <italic> <span class="gene">ANS3</span> </italic> genes at the 400&#x02013;500 mg <span class="trait">seed</span> developmental stage ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6A </xref> ). In another study in which the transcript profiles of <span class="trait">seed</span> coats from a standard black and brown-seeded near-isogenic pair were compared at different stages of <span class="trait">seed</span> development, significant differences in <italic> ANS2/ANS3 </italic> and <italic> <span class="gene">UGT78K1</span> </italic> expression were also seen between the two isolines with higher levels in the black <span class="trait">seed</span> coats <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> . For <italic> ANS2/ANS3, </italic> the significant differential expression (P&#x0003c;0.01) manifested early in <span class="trait">seed</span> development, (75&#x02013;100 mg), while the <italic> <span class="gene">UGT78K1</span> </italic> differences were first detected at 200&#x02013;300 mg weight range and reaching maximum at 300&#x02013;400 mg <span class="trait">seed</span> size. Gillman et al, (2011) <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> measured <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression via qRT-PCR in <span class="trait">seed</span> coats of a brown- (PI 567115 B) and black- (PI 84970) seeded lines with similar results. As in those instances, our results show that the <italic> UFGT </italic> genes are expressed at very low level at early stages of <span class="trait">seed</span> development, while <italic> <span class="gene">ANS</span> </italic> expression is very high at those early stages and tapers down as <span class="trait">seed</span> development advances. This decrease is slowed down or reversed in the black-seeded phenotypes. The higher level of <italic> <span class="gene">ANS</span> </italic> gene expression at early stages of <span class="trait">seed</span> development can be explained by an additional role of the <span class="gene">ANS</span> enzyme in directing the synthesis towards the flavanol and proanthocyanidin branches of the flavonoid pathway ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ). In contrast, the UFGT enzyme function is limited to the last step in the synthesis of anthocyanins ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The structural genes of the flavonoid metabolic pathways are regulated for the most part, at the transcriptional level and, with some exceptions, the majority of the regulatory genes have been found to be members of a large family of MYB transcription factors. Most of the MYB genes were predicted to function as transcriptional activators of the flavonoid biosynthetic pathway but in a few instances transcriptional repression guided by MYB transcription factor has been demonstrated <xref rid="pone.0111959-Aharoni1" ref-type="bibr"> [21] </xref> . In maize, <italic> C1-I </italic> was found to be a dominant inhibitor allele of the transcription activator <italic> C1-myb- </italic> like gene <xref rid="pone.0111959-PazAres1" ref-type="bibr"> [34] </xref> , <xref rid="pone.0111959-Goff1" ref-type="bibr"> [35] </xref> . A different <italic> <span class="gene">C1</span> </italic> homologue of maize, <italic> Zm38 </italic> , was found to be structurally similar to two other R2R3 MYB genes, <italic> AmMYB308 </italic> and <italic> AtMYB4 </italic> , that are transcriptional repressors of <italic> 4Cl </italic> (b-coumaroyl 4-CoAligase) and the <italic> <span class="gene">C4H</span> </italic> (cinnamate 4-hydroxylase) genes in <italic> Antirrhinum </italic> and <italic> Arabidopsis </italic> , respectively <xref rid="pone.0111959-Tamagnone1" ref-type="bibr"> [36] </xref> , <xref rid="pone.0111959-Jin1" ref-type="bibr"> [22] </xref> . Another TF with the motif shown to be involved in repression of transcription in <italic> Arabidopsis </italic> ( <italic> AtMYB4 </italic> ) is the <italic> FaMYB1 </italic> gene expressed during fruit maturation in strawberry <xref rid="pone.0111959-Aharoni1" ref-type="bibr"> [21] </xref> . <italic> AmMYB308 </italic> , like <italic> C1-I </italic> , competes out and inhibits the action of their respective transcriptional activator alleles. In contrast, <italic> AtMYB4 </italic> and <italic> FaMYB1 </italic> are direct repressors of their respective target genes and contain sequence motifs that recognize specific promoter repressor domains. Although little is known about repression mechanisms in plants, in both those instances, AtMYB4 and FaMYB1 C-termini possess a peptide motif, pdLNL <sup> D </sup> / <sub> E </sub> Lxi <sup> G </sup> / <sub> S </sub> , shown to be required for suppression activity <xref rid="pone.0111959-Jin1" ref-type="bibr"> [22] </xref> , <xref rid="pone.0111959-Aharoni1" ref-type="bibr"> [21] </xref> , <xref rid="pone.0111959-Kranz1" ref-type="bibr"> [20] </xref> . The soybean <italic> R </italic> locus MYB factor has the core portion of such motif, <bold> LLDL </bold> , at its C-terminus which may suffice to function as a direct repressor of late genes in the anthocyanin biosynthetic pathway. Likewise, the <italic> R </italic> gene possesses four of the amino acids in the motif, <bold> KP </bold> R <bold> PR </bold> [S/T]F, present in the subgroup five (AtMYB113) involved in anthocyanin regulation by activation <xref rid="pone.0111959-Stracke1" ref-type="bibr"> [10] </xref> . The differential expression of the <italic> R </italic> MYB gene and <italic> <span class="gene">ANS</span> </italic> genes supports its role as an inducer of anthocyanidin synthase in the black soybean <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						The RNAseq expression results also demonstrate that the activation of the anthocyanin pathway in RM30- <italic> R </italic> * is due to the <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> transposon and not to either of the two other closely related MYB transcription factor genes, <span class="gene">Glyma09g36966</span> and <span class="gene">Glyma09g37010,</span> since these two genes appear not to be expressed in the <span class="trait">seed</span> coats of either the RM30- <italic> R* </italic> line with the black <span class="trait">seed</span> and the RM38- <italic> r </italic> line with the brown <span class="trait">seed</span> phenotypes. This eliminates the likelihood that the <italic> TgmR </italic> * insertion occurred in any of those two related genes that could in turn affect the expression of the <span class="gene">Glyma09g36983</span> gene in a similar fashion as the <italic> <span class="gene">C1</span> </italic> homologue of maize, <italic> Zm38 </italic> , that was found to be a transcriptional repressor of <italic> <span class="gene">C1</span> </italic> <xref rid="pone.0111959-Franken1" ref-type="bibr"> [37] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean line with the mutable <italic> r <sup> m </sup> </italic> allele (RM55- <italic> r <sup> m </sup> </italic> ) and striped <span class="trait">seed</span> phenotype showed instability with the frequent generation of revertant <span class="trait">seed</span> genotypes some being all black which in subsequent generations would switch back to variegated at a low frequency <xref rid="pone.0111959-Chandlee1" ref-type="bibr"> [6] </xref> . To find out whether methylation contributes to the variegated or stable phenotype, we compared the BS-sequences from the striped RM55- <italic> r <sup> m </sup> </italic> and the stable revertant isoline RM30- <italic> R* </italic> . As shown in <xref ref-type="table" rid="pone-0111959-t004"> Table 4 </xref> , the level of methylated C residues in the CHH context for <italic> TgmR* </italic> in the RM30- <italic> R* </italic> stable line was double the percentage of that found in RM55- <italic> r <sup> m </sup> </italic> . The more prevalent methylated CpG and CHG regions were also higher in comparison to RM55- <italic> r <sup> m </sup> </italic> . More interestingly, distribution of the hypomethylated regions in the element were concentrated near the subterminal repeats and a region of the 3&#x02032; end of <span class="gene">ORF2</span> ( <xref ref-type="fig" rid="pone-0111959-g010"> Figure 10 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The brown background in the RM55- <italic> r <sup> m </sup> </italic> <span class="trait">seed</span> coats being the result of the 14 kb amplicon 13 kb <italic> TgmR* </italic> (ie the 14kb amplicon) is harder to explain especially in light of the fact that we have shown that the revertant RM30- <italic> R* </italic> line containing the 13 kb <italic> TgmR* </italic> element in its intron has black <span class="trait">seed</span> coats. This would suggest that the phenotype of the allele (black or brown) is determined by the level of methylation of the <italic> TgmR* </italic> element. When the element residing in Intron2 is less methylated as in the RM55- <italic> r <sup> m </sup> </italic> , the sectors are brown implying the MYB gene is not expressed. When <italic> TgmR* </italic> is more methylated, as in the RM30- <italic> R* </italic> allele, the <span class="trait">seed</span> coats are black and the MYB gene is expressed as we have shown experimentally. We propose that the mechanism for this phenenomon is similar to that shown for the <italic> Spm </italic> -suppressible alleles of maize in that the phenotype of the affected allele can depend upon where the transposon sits in the gene and whether an active element is present (reviewed in <xref rid="pone.0111959-Wessler1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0111959-Fedoroff1" ref-type="bibr"> [13] </xref> ). The CACTA ends of <italic> Spm </italic> elements as well as the Activator (Ac) elements sometimes mimic splicing sites and are recognized as intron ends <xref rid="pone.0111959-Wessler1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0111959-Menssen1" ref-type="bibr"> [38] </xref> . In some cases as shown in maize, binding of transposase from an active element will interfere with intron processing if the element is near the Exon-Intron junction leading to the gene not being processed correctly. In the absence of transposase, the introns are processed and the gene is expressed, although sometimes at a lower level. In maize, it has been shown that the autonomous <italic> Spm </italic> element itself can switch between an active state (less methylated with transposase produced) and an inactive state (more methylated and transposase not produced) based on the level of its methylation <xref rid="pone.0111959-Banks1" ref-type="bibr"> [11] </xref> . Thus, in the hypomethylated RM55- <italic> r <sup> m </sup> </italic> line, transposase from the <italic> TgmR* </italic> element in the MYB gene or other locations could bind to the element ends and interfere with intron processing leading to the brown background <span class="trait">color.</span> On the other hand, in the stable black line RM30- <italic> R* </italic> , the opposite effect occurs as the element is more methylated and transposase is lacking or cannot bind the element termini and interfere with intron processing. Thus, the element is spliced out with the intron and the MYB gene is expressed at a low level leading to activation of the <span class="gene">ANS</span> genes and production of the anthocyanin pigments. There is one phenotypic difference between the standard <italic> R </italic> allele and the <italic> R* </italic> allele in the RM30 line in that the hilum region, which is the point of attachment of the <span class="trait">seed</span> to the pod, is less pigmented than in the standard <italic> R </italic> allele and retain a brownish cast <xref rid="pone.0111959-Chandlee1" ref-type="bibr"> [6] </xref> . A genetic difference between the two alleles is that the <italic> r <sup> m </sup> </italic> is normally recessive to the standard <italic> R </italic> allele producing progeny with black <span class="trait">seed,</span> but it appears to be dominant to the <italic> R* </italic> allele with variegated <span class="trait">seed</span> predominating and an increased rate of somatic and germinal mutability results from crossing RM30- <italic> R* </italic> and RM55- <italic> r <sup> m </sup> </italic> <xref rid="pone.0111959-Wang3" ref-type="bibr"> [39] </xref> . The activation <italic> in trans </italic> of the <italic> TgmR* </italic> element within the RM30- <italic> R* </italic> allele by the transposase from the hypomethylated <italic> r <sup> m </sup> </italic> allele is consistent with its dominance over the <italic> R* </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, our results confirm the soybean <italic> R </italic> locus to be <span class="gene">Glyma09g36983</span> and demonstrate that the transcription of the <italic> R </italic> gene in the black <span class="trait">seed</span> coats of the RM30- <italic> R </italic> * revertant line, may be hindered by the processing of the very large Intron2 containing the <italic> TgmR </italic> * (13kb) insertion, but it does not render the processed transcript non-functional. Instead, the translated MYB protein drives an increase in transcript accumulation of the <italic> ANS2/3 </italic> , <italic> UFGT2 </italic> and <italic> AOMT </italic> genes whose products function in the late steps of anthocyanin synthesis, particularly in production of the cyanidin-3-glycoside that accounts for the black pigmentation of the <span class="trait">seed</span> coats. In contrast, the observed higher level of transcripts found for the defective <italic> r </italic> allele in RM38- <italic> r </italic> <span class="trait">seed</span> coats may translate into truncated peptides that are non-functional and do not activate the transcription of the late anthocyanin pathway genes resulting in brown seeds which reflects accumulation of only proanthocyanidins. The <italic> TgmR* </italic> found in the striped RM55- <italic> r <sup> m </sup> </italic> and black-seeded RM30- <italic> R* </italic> isolines is a transposon of a second CACTA subfamily that encodes a full length <italic> Tnp2 </italic> transposase gene that may drive the observed instability of the isolines with the <italic> r <sup> m </sup> </italic> and <italic> R* </italic> alleles. The level of instability of the RM55- <italic> r <sup> m </sup> </italic> and RM30- <italic> R* </italic> isolines is influenced by the total level of C methylation and also methylated C in the CHH context, judging by the bisulfite sequencing results of three genomes. The <italic> TgmR* </italic> element in the variegated RM55- <italic> r <sup> m </sup> </italic> line was hypomethylated especially at the subterminal repeat ends compared to the RM30- <italic> R* </italic> element. Stress and other environmental factors may in turn affect the methylation level of the germinal and somatic tissues and consequently lead to the variegated phenotype and affect the stability of the mutable ( <italic> r <sup> m </sup> </italic> ) and revertant ( <italic> R* </italic> ) alleles by a mechanism similar to that elucidated for CACTA elements in maize including the <italic> Spm </italic> -suppressible alleles first described by Barbara McClintock (reviewed in <xref rid="pone.0111959-Wessler1" ref-type="bibr"> [12] </xref> , <xref rid="pone.0111959-Fedoroff1" ref-type="bibr"> [13] </xref> ). This explains the opposite phenotypes resulting from the influence of the <italic> TgmR* </italic> transposase on intron processing of the MYB protein when the element is less methylated in the RM55- <italic> r <sup> m </sup> </italic> line (brown <span class="trait">coat</span> sectors) versus the more methylated element in the RM30- <italic> R* </italic> revertant line with black <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						The choices of Glyma models representing the different genes of the anthocyanin pathways and listed in <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> were based on previous gene characterization studies. Here we provide the Glyma models, GenBank accession numbers and references for each one of the genes whose expression we analyzed in the soybean lines used for the current study. The <italic> I </italic> locus <span class="gene">(Glyma01g43880.1)</span> <italic> chalcone synthase 7 </italic> ( <italic> <span class="gene">CHS7</span> </italic> ) Ac.N.: M98871 <xref rid="pone.0111959-Akada1" ref-type="bibr"> [45] </xref> and <span class="gene">(Glyma11g01350.1)</span> <italic> CHS8 </italic> Ac.No.: AY237728; <span class="gene">Glyma20g38560.1</span> <italic> chalcone isomerase 1A </italic> ( <italic> <span class="gene">CHI1A</span> </italic> ) Ac.No.: AY595413 <xref rid="pone.0111959-Ralston1" ref-type="bibr"> [46] </xref> and <span class="gene">Glyma20g38580.1</span> ( <italic> CHI2 </italic> ) Ac.No.: AY595415 <xref rid="pone.0111959-Ralston1" ref-type="bibr"> [46] </xref> ; <italic> T </italic> locus <span class="gene">(Glyma06g21920.1)</span> <italic> flavonoid 3&#x02032;hydroxylase </italic> ( <italic> F3&#x02032;H </italic> ) Ac.No.: EU190438 <xref rid="pone.0111959-Zabala1" ref-type="bibr"> [3] </xref> ; <italic> <span class="gene">W1</span> </italic> locus <span class="gene">(Glyma13g04210.1)</span> <italic> flavonoid 3&#x02032;-5&#x02032;-hydroxylase </italic> ( <italic> F3&#x02032;5&#x02032;H </italic> ) Ac.No.: EF174665 <xref rid="pone.0111959-Zabala4" ref-type="bibr"> [47] </xref> ; <italic> Wp </italic> locus <span class="gene">(Glyma02g05450.1)</span> <italic> flavanone 3-hydroxylase </italic> ( <italic> <span class="gene">F3H</span> </italic> ) Ac.No.: AY669325, <xref rid="pone.0111959-Zabala4" ref-type="bibr"> [47] </xref> ; <italic> <span class="gene">W3</span> </italic> locus <span class="gene">(Glyma14g07940.1)</span> <italic> dihydroflavonol-4-reductase 1 </italic> ( <italic> <span class="gene">DFR1</span> </italic> ) Ac.No.: AF167556; <italic> <span class="gene">W4</span> </italic> locus <span class="gene">(Glyma17g37060.1)</span> <italic> <span class="gene">DFR2</span> </italic> Ac.No.: EF187612 <xref rid="pone.0111959-Xu1" ref-type="bibr"> [9] </xref> ; <span class="gene">Glyma01g42350.1</span> <italic> anthocyanidin synthase 2 </italic> ( <italic> ANS2 </italic> ) Ac.No.: AY382829.1 and <span class="gene">Glyma11g03010.1</span> ( <italic> <span class="gene">ANS3</span> </italic> ) Ac.No.: AY382830; <span class="gene">Glyma08g07130,</span> <italic> UDP-glucose:flavonoid 3-O-glucosyltransferase </italic> ( <italic> UF3GT </italic> ) ( <italic> <span class="gene">UGT78K2</span> </italic> ) Ac.No.: HM591298.1 <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> and <span class="gene">Glyma07g30180</span> ( <italic> <span class="gene">UGT78K1</span> </italic> ) Ac.No.: GU434274.1 <xref rid="pone.0111959-Kovinich2" ref-type="bibr"> [48] </xref> . The latter, <italic> <span class="gene">UGT78K1</span> </italic> , was found to be expressed in the cotyledons in addition to the <span class="trait">seed</span> coats <xref rid="pone.0111959-Gillman1" ref-type="bibr"> [4] </xref> . <span class="gene">Glyma08g06630.1,</span> <italic> anthocyanidin reductase </italic> ( <italic> <span class="gene">ANR1</span> </italic> ) Ac.No.: NM_001254984.1 and <span class="gene">Glyma08g06640.1</span> ( <italic> ANR2 </italic> ) <span class="gene">(LOC100817081)</span> Ac.No.: NM_001256143.1 <xref rid="pone.0111959-Kovinich3" ref-type="bibr"> [49] </xref> . The <italic> ANR </italic> genes function in diverting the anthocyanidins towards the synthesis of proanthocyanidins (tannins) which provide the <span class="trait">color</span> to the brown <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0111959-g001"> Figure 1 </xref> ). Lastly, <span class="gene">Glyma05g36210.1,</span> <italic> anthocyanin O-methyltransferase </italic> ( <italic> AOMT </italic> ) Ac.No.: NM_001255526.1 <xref rid="pone.0111959-Kovinich1" ref-type="bibr"> [5] </xref> which methylates the anthocyanins.
					</p>
				
					<p>&nbsp;&nbsp;
						Distribution of RNA-Seq Reads from <span class="trait">Seed</span> Coats of the RM30- <italic> R </italic> * and RM38- <italic> r </italic> when Aligned to the Genomic Sequence of <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> Insertion in Intron2.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:<italic> In silico </italic> comparison of genomic regions containing genes coding for enzymes and transcription factors for the phenylpropanoid pathway in <italic> Phaseolus vulgaris </italic> L. and <italic> Glycine max </italic> L. Merr</h2>
			<h3>PMID:24046770</h3>
				

				
					<h3>PMCID:3763686</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Reinprecht </surname>  <given-names> Yarmilla </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yadegari </surname>  <given-names> Zeinab </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Perry </surname>  <given-names> Gregory E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Siddiqua </surname>  <given-names> Mahbuba </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wright </surname>  <given-names> Lori C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClean </surname>  <given-names> Phillip E. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pauls </surname>  <given-names> K. Peter </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legumes contain a variety of phytochemicals derived from the phenylpropanoid pathway that have important effects on human health as well as <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color,</span> plant disease <span class="trait">resistance</span> and nodulation. However, the information about the genes involved in this important pathway is fragmentary in common bean (Phaseolus vulgaris L.). The objectives of this research were to isolate genes that function in and control the phenylpropanoid pathway in common bean, determine their genomic locations in silico in common bean and soybean, and analyze sequences of the <span class="gene">4CL</span> gene family in two common bean genotypes. Sequences of phenylpropanoid pathway genes available for common bean or other plant species were aligned, and the conserved regions were used to design sequence-specific primers. The PCR products were cloned and sequenced and the gene sequences along with common bean gene-based (g) markers were BLASTed against the Glycine max v.1.0 genome and the P. vulgaris v.1.0 (Andean) early release genome. In addition, gene sequences were BLASTed against the OAC Rex (Mesoamerican) genome sequence assembly. In total, fragments of 46 structural and regulatory phenylpropanoid pathway genes were characterized in this way and placed in silico on common bean and soybean sequence maps. The maps contain over 250 common bean g and SSR (simple sequence repeat) markers and identify the positions of more than 60 additional phenylpropanoid pathway gene sequences, plus the putative locations of <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes. The majority of cloned phenylpropanoid pathway gene sequences were mapped to one location in the common bean genome but had two positions in soybean. The comparison of the genomic maps confirmed previous studies, which show that common bean and soybean share genomic regions, including those containing phenylpropanoid pathway gene sequences, with conserved synteny. Indels identified in the comparison of Andean and Mesoamerican common bean <span class="gene">4CL</span> gene sequences might be used to develop inter-pool phenylpropanoid pathway gene-based markers. We anticipate that the information obtained by this study will simplify and accelerate selections of common bean with specific phenylpropanoid pathway alleles to increase the contents of beneficial phenylpropanoids in common bean and other legumes.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed,</span> flower and pubescence colors have utility in soybean breeding programs as hybridity markers. Commercial soybean has yellow <span class="trait">seed,</span> although natural variation in <span class="trait">seed</span> pigmentation exists. <span class="trait">Seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean is determined by three independent loci ( <italic> I, T </italic> and <italic> R </italic> ) distinct from flower color-controlling loci (Owen, <xref ref-type="bibr" rid="B48"> 1928 </xref> ; Palmer et al., <xref ref-type="bibr" rid="B49"> 2004 </xref> ), some of which are associated with genes coding for enzymes of anthocyanin biosynthesis. For example, the <italic> I </italic> (inhibitor) locus (light, dark or saddled hilum <span class="trait">color)</span> was associated with six inverted chalcone synthase (CHS) repeats on chromosome <span class="gene">Gm08</span> (Todd and Vodkin, <xref ref-type="bibr" rid="B62"> 1996 </xref> ) and the <italic> T </italic> locus (tawny brown or gray pubescence <span class="trait">color)</span> co-localized with flavonoid 3&#x02032; hydroxylase (F3&#x02032;H) on chromosome <span class="gene">Gm06</span> (Toda et al., <xref ref-type="bibr" rid="B61"> 2002 </xref> ; Zabala and Vodkin, <xref ref-type="bibr" rid="B71"> 2003 </xref> ). However, the <italic> R </italic> locus (black, brown or striped <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color)</span> mapped on chromosome <span class="gene">Gm09</span> could not be linked with known anthocyanidin synthase (ANS), dihydroflavonol 4-reductase (DFR), or UDP-flavonoid glucosyltransferase genes. Findings by Gillman et al. ( <xref ref-type="bibr" rid="B23"> 2011 </xref> ) suggest that loss of function mutation in an R2R3 MYB transcription factor results in brown hilum and brown <span class="trait">seed</span> <span class="trait">coat.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Large genetic diversity exists between and within the Mesoamerican (Mesoamerica, Durango and Jalisco) and Andean (Nueva Granada, Chile, and Peru) gene pools of the common bean. They are distinguishable not only by geographic origin but also by <span class="trait">seed</span> size, growth habit, environmental adaptation, <span class="trait">fertility</span> barriers, <span class="trait">resistance</span> to diseases, phaseolin types, isozyme, and molecular marker diversities (Kwak and Gepts, <xref ref-type="bibr" rid="B34"> 2009 </xref> ). In common bean, a number of genes control the colors ( <italic> P, R, J, D, G, B, V </italic> , and <italic> Rk </italic> ) and patterns ( <italic> T, Z, L, J, Bip </italic> , and <italic> Ana </italic> ) of <span class="trait">seed</span> coats (Prakken, <xref ref-type="bibr" rid="B53"> 1970 </xref> , <xref ref-type="bibr" rid="B54"> 1972 </xref> ). Interactions among these genes are important in defining each of the major common bean market classes such as white, black, pinto or cranberry (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ). Although none of the biochemical functions of the genes have been defined (Hosfield, <xref ref-type="bibr" rid="B31"> 2001 </xref> ), it is believed that flavonoids are the pigments responsible for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> in common bean (Beninger et al., <xref ref-type="bibr" rid="B3"> 1998 </xref> ). Markers based on the genes that control <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> would greatly facilitate germplasm utilization in common bean breeding programs. RAPD (random amplified polymorphic DNA) and <span class="gene">STS</span> (sequence-tagged sites) markers associated with <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes have been identified and mapped on several linkage groups (Brady et al., <xref ref-type="bibr" rid="B5"> 1998 </xref> ; Erdmann et al., <xref ref-type="bibr" rid="B17"> 2002 </xref> ; McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ). Also, gene-based markers for several enzymes of the flavonoid/anthocyanin pathway (F3H, DFR, and ANS) have been developed (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ), but their connection with <span class="trait">color</span> genes has not been established.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Phenylpropanoid pathway gene sequence-based common bean-soybean comparative map </bold> . Marker and phenylpropanoid pathway gene names (abbreviated plus accession number) are shown on right of each chromosome. Feature position [bp, start nt shown (Phytozome)] is indicated on left of each chromosome. Syntenic fragments of soybean chromosomes (each in different <span class="trait">color)</span> are shown on left (copy &#x0201c;1&#x0201d;) and right (copy &#x0201c;2&#x0201d;) of common bean chromosome (black). Phenylpropanoid pathway gene fragments cloned in this study are shown in red, bold, and underlined. Tentative positions of <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes are shown in black blocks. Enzyme abbreviations: PAL, phenylalanine ammonia lyase; C4H, cinnamate 4-hydroxylase; 4CL, 4-coumarate:CoA ligase; C3H, 4-coumarate 3-hydroxylase; COMT, caffeate O-methyltransferase; F5H, ferulic acid 5-hydroxylase; CCoOMT, caffeoyl CoA O-methyltransferase; CCR, cinnamoyl CoA-reductase; HCT, hydroxycinnamoyl-CoA: shikimate/guinate hydroxycinnamoyltransferase; CAD, cinnamyl alcohol dehydrogenase; LAC, laccase; PER, lignin peroxydase; PLR, pinoresinol/lariciresinol reductase; DP, dirigent protein; STS, stilbene synthase; CHS, chalcone synthase; CHI, chalcone isomerase; FNS, flavone synthase; F3H, flavanone 3-hydroxylase; F3&#x02032;H, flavonoid 3&#x02032;-hydroxylase; F3&#x02032;5&#x02032;H, flavonoid 3&#x02032;5&#x02032;-hydroxylase; FLS, flavonol synthase; DFR, dihydroflavanol 4-reductase; <span class="gene">ANS</span> (LDOX), anthocyanidin synthase; UF3GT, UDP glucose flavonoid 3-O-glucosyltransferase; 3RT, anthocyanidin-3-glucoside rhamnosyl transferase; AAT, anthocyanin 5-aromatic acyltransferase; AMT, anthocyanin methyltransferase; GST, glutathione S-transferase; VT, vacuolar transporter; LAR, leucoanthocyanidin reductase; ANR, anthocyanidin 4-reductase; PPO, polyphenol oxidase; CHR, chalcone reductase; IFS, 2-hydroxyisoflavanone synthase; HIDH, 2-hydroxyisoflavanone dehydratase; 7IOMT, isoflavone 7-O-methyltransferase; I2&#x02032;H, isoflavone 2&#x02032;-hydroxylase; IFR, isoflavanone reductase; F6H, flavonoid 6-hydroxylase; HI4&#x02032;OMT, isoflavone 4-O-methyltransferase; IF7MaT, isoflavone 7-O-methyltransferase; VR, vestitone reductase; DMID, dihydroxy-methoxy-isoflavanol dehydratase; G4DT, pterocarpan 4-dimethylallyltransferase; PTS, pterocarpan synthase.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv2. </italic> </bold> This study confirmed the existence of two major syntenic blocks between common bean chromosome Pv2 and soybean chromosomes <span class="gene">Gm01</span> (22 features)/Gm11 (22 features) and <span class="gene">Gm05</span> (27 features)/Gm08 (43 features), that were identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ; Galeano et al., <xref ref-type="bibr" rid="B21"> 2011 </xref> ). In addition, one to 12 sequences were shared between Pv2 and eight soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Twenty phenylpropanoid pathway gene sequences were found on chromosome Pv2, six of them cloned in this study ( <italic> <span class="gene">CHS</span> </italic> -CV670744, <italic> CHS-B-Z, F5H </italic> -CV670742, <italic> KAP2 </italic> -CV670752, <italic> ANR </italic> -CW652104, and <italic> CAD </italic> -CV670740). A cluster containing four <italic> <span class="gene">CHS</span> </italic> genes was found on the common bean chromosome Pv2 (at 3,726,784 nt; 3,733,338 nt; 3,765,494 nt; 3,775,079 nt). The first gene was syntenic to soybean <italic> <span class="gene">CHS7</span> </italic> on chromosome <span class="gene">Gm01</span> and <italic> CHS8 </italic> on chromosome <span class="gene">Gm11.</span> The fifth common bean <italic> <span class="gene">CHS</span> </italic> gene was found on position 33,749,564 nt (sequence start) and was syntenic to a single soybean <italic> <span class="gene">CHS</span> </italic> on chromosome <span class="gene">Gm05</span> ( <italic> <span class="gene">CHS2</span> </italic> ) but corresponds to a cluster of six <italic> <span class="gene">CHS</span> </italic> genes on chromosome <span class="gene">Gm08.</span> The locations of the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes <italic> Rk </italic> and <italic> B </italic> that were mapped previously to the end of chromosome Pv2, are indicated (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ). Interestingly, soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes <italic> I </italic> [factor which inhibits pigment formation in the <span class="trait">seed</span> <span class="trait">coat,</span> associated with six <italic> <span class="gene">CHS</span> </italic> repeats (Tuteja et al., <xref ref-type="bibr" rid="B64"> 2004 </xref> )] and <italic> O </italic> [factor for dull brown <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color,</span> may be associated with anthocyanin reductase (ANR, Yang et al., <xref ref-type="bibr" rid="B69"> 2010 </xref> )] were located on chromosome <span class="gene">Gm08,</span> which are syntenic to the regions of Pv2 chromosome containing common bean <italic> <span class="gene">CHS</span> </italic> and <italic> ANR </italic> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv3. </italic> </bold> Blocks of soybean chromosomes <span class="gene">Gm02</span> (16 sequence features) and <span class="gene">Gm05</span> (eight features)/Gm17 (21 features) and <span class="gene">Gm02/Gm16</span> (six features) were syntenic to common bean chromosome Pv3 (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ; Galeano et al., <xref ref-type="bibr" rid="B21"> 2011 </xref> ). In addition, one to 11 sequences were shared between Pv3 and many (11) soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Eleven phenylpropanoid pathway genes were identified on chromosome Pv3, including five cloned in this study ( <italic> <span class="gene">IFS</span> </italic> -CV670762, <italic> <span class="gene">4CL1</span> </italic> -CV670737, <italic> <span class="gene">F3H</span> </italic> -CW652103, <italic> IFR </italic> -CV670750, and <italic> 3RT </italic> -CV670748). Two copies of the <italic> <span class="gene">4CL1</span> </italic> gene were found 13,788 bp apart on the common bean chromosome Pv3 ( <italic> 4CL1-2 </italic> at the start position 34,628,464 nt and <italic> 4CL1-1 </italic> at the start position 34,642,252 nt, respectively). Soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> Wp </italic> , (purple flower and hypocotyl <span class="trait">color</span> and the production of black pigment in the <span class="trait">seed</span> <span class="trait">coat)</span> associated previously with flavanone 3-hydroxylase (F3H) on chromosome <span class="gene">Gm02</span> (Zabala and Vodkin, <xref ref-type="bibr" rid="B72"> 2005 </xref> ), was mapped to the expected region that was syntenic to common bean <italic> <span class="gene">F3H</span> </italic> . Previously, RAPD marker OAM10 was associated with common bean <span class="trait">seed</span> <span class="trait">coat</span> pattern gene <italic> Z </italic> (zonal) and was positioned between markers Bng12 and Bng165 on chromosome Pv3 (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ). However, in the current study there were no significant BLAST hits with sequence of this marker and the <italic> Z </italic> gene was tentatively positioned in the region between the closest SSRs, Bng75 and Bng165.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv4. </italic> </bold> Blocks of soybean chromosomes <span class="gene">Gm13</span> (five features)/Gm19 (eight features) were syntenic with common bean chromosome Pv4, as detected previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). A new syntenic block was identified with soybean chromosomes <span class="gene">Gm16</span> (12 features) and <span class="gene">Gm09</span> (13 sequence features). In addition, one to four sequences were shared between Pv4 and several (seven) additional soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Six phenylpropanoid pathway genes were identified on common bean chromosome Pv4, including chalcone reductase ( <italic> CHR </italic> -CW652099) cloned in this study. There were no significant BLAST hits with markers OU14, OAP3, and OAP14 (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ) associated with the common bean <span class="trait">seed</span> <span class="trait">coat</span> gene <italic> G </italic> (yellow <span class="trait">seed</span> <span class="trait">coat,</span> from <italic> Gelbe </italic> in German) and the gene was tentatively positioned in the region between the closest SSRs, BM68, and Bng224.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv6. </italic> </bold> Blocks of soybean chromosomes <span class="gene">Gm08</span> (15 sequence features)/Gm18 (nine features), <span class="gene">Gm13</span> (10 features)/Gm15, <span class="gene">Gm08/Gm15,</span> <span class="gene">Gm15/Gm09</span> (11 features), and <span class="gene">Gm19</span> (three features) and <span class="gene">Gm12</span> (five features) were syntenic with common bean chromosome Pv6, as identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). In addition, one to two sequences were shared between Pv6 and six additional soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Nine phenylpropanoid pathway genes were found on the common bean chromosome Pv6, including four cloned in this study ( <italic> Myb4 </italic> -CW652106, <italic> PER-FBP1 </italic> -CW652094, <italic> 7IOMT </italic> -CV670751, and <italic> PER-FBP4 </italic> -CW652095). Two copies of the common bean transcription factor gene <italic> Myb4 </italic> (23,032,616 nt) were identified on soybean chromosomes <span class="gene">Gm19</span> and <span class="gene">Gm16,</span> respectively. A cluster of 10 PERs ( <italic> FBP1 </italic> -AF149277) found at the positions 24,384,794 nt to 24,475,020 nt was syntenic to six PERs on the soybean chromosome <span class="gene">Gm15</span> and six genes on <span class="gene">Gm09,</span> both clusters in reverse order. Another example is a single PER gene ( <italic> FBP4 </italic> -AF149279) on the common bean chromosome Pv6 (30,963,524 nt) associated with two clusters of four soybean peroxidases on the chromosomes <span class="gene">Gm15</span> and three genes on the chromosome <span class="gene">Gm08.</span> The sequence of the marker OD12 that was associated with the common bean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> <italic> V </italic> (violet factor, changing white or pale lilac flowers into violet and causing violet to black <span class="trait">seed</span> <span class="trait">coat</span> colors) gene (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ) mapped between <italic> F3&#x02032;5&#x02032;H </italic> -AY17551 and <italic> F6H </italic> -XM_003551443 at the top of the chromosome Pv6. In soybean, the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> <span class="gene">W1</span> </italic> is associated with F3&#x02032;5&#x02032;H (Zabala and Vodkin, <xref ref-type="bibr" rid="B73"> 2007 </xref> ). Several F3&#x02032;5&#x02032;H gene-based markers (Yang et al., <xref ref-type="bibr" rid="B69"> 2010 </xref> ) mapped to the location of F3&#x02032;5&#x02032;H on the soybean chromosome <span class="gene">Gm13.</span> This map position was syntenic to the region of common bean <italic> F3&#x02032;5&#x02032;H </italic> gene located at the top of the chromosome Pv6.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv7. </italic> </bold> Blocks of soybean chromosome <span class="gene">Gm10</span> (39 sequence features)/Gm20 (25 features) and <span class="gene">Gm10/Gm13</span> (six features) were syntenic with common bean chromosome Pv7, as identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). In addition, one to six sequences were shared between Pv7 and five soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Thirteen phenylpropanoid pathway genes were identified on the common bean chromosome Pv7, including six cloned in this study ( <italic> <span class="gene">CHI</span> </italic> -CV670745, <italic> <span class="gene">C4H</span> </italic> -CV670736, <italic> TT2-1-L, Myb15 </italic> -CW652102/ <italic> Myb15-Z, <span class="gene">PAL2</span> </italic> -CV670734/ <italic> PAL2-Z </italic> , and <italic> DFR-Z/DFR1-M </italic> ). Cinnamate 4-hydroxylase gene ( <italic> <span class="gene">C4H</span> </italic> -J09447) on the common bean chromosome Pv7 was associated with one copy on soybean chromosome <span class="gene">Gm20</span> and second on the chromosome <span class="gene">Gm10.</span> A BLAST search with the RAPD marker sequence OU3, which is tightly linked with the common bean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> <italic> P </italic> gene (pigment ground factor) on chromosome Pv7 (Erdmann et al., <xref ref-type="bibr" rid="B17"> 2002 </xref> ), had no significant hits in our search. The <italic> P </italic> gene was tentatively placed between two closest SSR markers, Bng42, and BM210.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv8. </italic> </bold> Two large syntenic blocks were found between common bean chromosome Pv8 and soybean chromosome blocks from <span class="gene">Gm18</span> (34 sequence features)/[Gm02 (18 features), <span class="gene">Gm8</span> (10 features), <span class="gene">Gm09</span> (17 features), <span class="gene">Gm07</span> (6 features)], and <span class="gene">Gm14</span> (19 features)/Gm02, as identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). In addition, one to six sequences were shared between Pv8 and eight other soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Fifteen phenylpropanoid pathway genes were identified on the common bean chromosome Pv8, including six cloned in the current study ( <italic> HD </italic> -CV670753, <italic> GST1 </italic> -CW652097, <italic> WIP </italic> -CW652101, <italic> <span class="gene">GST2</span> </italic> -CV670756, <italic> VT </italic> -CV670747, and <italic> PAL3 </italic> -CV670735/ <italic> PAL3-Z </italic> ). Most of these sequences were associated with two copies on two soybean chromosomes. The only exceptions were polyphenol oxidase ( <italic> PPO </italic> -ABL67928) with a single sequence and isoflavonoid glucosyltransferase ( <italic> IFGT </italic> -EV280635) with two sequences on chromosome <span class="gene">Gm18.</span> The sequence of RAPD marker OW17, which was associated with the common bean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> Gy </italic> (greenish-yellow factor; McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ) was mapped at the top portion of the chromosome Pv8 close to a cluster of Myb transcription factors. Soybean SSR marker Sat_293, associated with the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> <italic> R </italic> gene (Yang et al., <xref ref-type="bibr" rid="B69"> 2010 </xref> ), was also mapped to this region of the chromosome Pv8 and to the syntenic region on the soybean chromosome <span class="gene">Gm09.</span> A second marker (OAP2; McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ) sequence, linked to <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> C </italic> (required for complete <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color)</span> mapped between SSR markers BM165 and BM153, close to the glutathione S-transferase (GST-AY220095) sequence on the chromosome Pv8.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv9. </italic> </bold> Common bean chromosome Pv9 was syntenic to sequence blocks of soybean chromosomes <span class="gene">Gm04</span> (28 shared sequence features)/Gm06 (33 features), as identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). The region at the lower arm of common bean chromosome Pv9 containing six g markers was syntenic to regions in soybean chromosomes <span class="gene">Gm09</span> and <span class="gene">Gm15.</span> In addition, one to two sequences were shared between Pv9 and four other soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Eight phenylpropanoid pathway genes were identified on the common bean chromosome Pv9, including three cloned in this study ( <italic> COMT </italic> -CV670741, <italic> LIM </italic> -CW652092, and <italic> F3&#x02032;H </italic> -CV707760). Common bean chalcone isomerase (CHI-AY595417) positioned at 20,929,174 nt was associated with a single copy on the soybean chromosome <span class="gene">Gm06</span> but was missing from duplicated region on the chromosome <span class="gene">Gm04.</span> The RAPD marker sequence OM19, linked to <span class="trait">seed</span> <span class="trait">coat</span> pattern <italic> T </italic> gene (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ), mapped on the common bean chromosome Pv9 between pterocarpan reductase ( <italic> PTR </italic> -TC433083) and LIM transcription factor ( <italic> LIM </italic> -BU764417). Several soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> <italic> T </italic> gene-based markers [associated with F3&#x02032;H (Yang et al., <xref ref-type="bibr" rid="B69"> 2010 </xref> )] were mapped to the F3&#x02032;H location on chromosome <span class="gene">Gm06</span> and to the syntenic region on common bean chromosome Pv9.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <italic> Pv10. </italic> </bold> Common bean chromosome Pv10 was syntenic to soybean chromosome blocks of <span class="gene">Gm07</span> (16 shared sequence features)/Gm08 (nine features) and <span class="gene">Gm16</span> (four features), as identified previously (McClean et al., <xref ref-type="bibr" rid="B43"> 2010 </xref> ). In addition, one to four sequences were shared between chromosome Pv10 and seven soybean chromosomes (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Six phenylpropanoid pathway gene sequences mapped to the common bean chromosome Pv10, none of which were cloned in the current study. Several transcription factors were found on the common bean chromosome Pv10. Interestingly, for common bean transcription factor Myb11-NM_116126 (29,502,170 nt) only one copy was found on the soybean chromosome <span class="gene">Gm19.</span> Sequences of RAPD markers associated with three <span class="trait">seed</span> <span class="trait">coat</span> pattern genes (McClean et al., <xref ref-type="bibr" rid="B42"> 2002 </xref> ) were mapped on the chromosome Pv10. The markers OM9, linked with the gene <italic> Ana </italic> , and OJ17 linked with the <italic> Bip </italic> gene, mapped to the top arm of chromosome Pv10, between markers BM157 and g1029. Marker OL4, linked to the <span class="trait">seed</span> <span class="trait">coat</span> pattern <italic> L </italic> gene, mapped to the bottom arm of chromosome Pv10 close to the Myb123 transcription factor ( <italic> Myb123 </italic> -AJ2994452).
					</p>
				
					<p>&nbsp;&nbsp;
						To illustrate potential application of common bean and soybean whole genome sequences, we selected 4-coumarate:CoA ligase (4CL), a key enzyme of general phenylpropanoid pathway, positioned at the <span class="trait">branching</span> point leading to biosynthesis of lignin/lignans, flavonoids/anthocyanins, isoflavonoids or stilbenes for a detailed analysis. Two common bean <italic> <span class="gene">4CL</span> </italic> gene fragments homologous to soybean genes <italic> <span class="gene">4CL1</span> </italic> and <italic> 4CL2 </italic> , respectively, were isolated in this study (Table <xref ref-type="table" rid="T2"> 2 </xref> ) and <italic> in silico </italic> mapped in both common bean and soybean (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The impetus for the current work was the lack of systematic information for the genes of the phenylpropanoid pathway in common bean. This is a real limitation for research dealing with the synthesis of a number of important secondary metabolites, including isoflavonoids, anthocyanins and lignin. At the beginning of this study only few phenylpropanoid pathway gene sequences were available in public databases for common bean. In the general phenylpropanoid pathway, all three genes coding for phenylalanine ammonia lyase ( <italic> <span class="gene">PAL1</span> </italic> , Edwards et al., <xref ref-type="bibr" rid="B16"> 1985 </xref> ; <italic> <span class="gene">PAL2</span> </italic> and <italic> PAL3 </italic> , Cramer et al., <xref ref-type="bibr" rid="B9"> 1989 </xref> ) and a single gene for cinnamate 4-hydroxylase ( <italic> <span class="gene">C4H</span> </italic> , Nedelkina et al., <xref ref-type="bibr" rid="B47"> 1999 </xref> ) were previously isolated in common bean. A number of enzymes are involved in the lignin/lignan branch of the phenylpropanoid pathway, in most cases, encoded by small gene families. Only few sequences for lignin peroxidases were available for common bean at the beginning of this study (Blee et al., <xref ref-type="bibr" rid="B4"> 2001 </xref> ). The flavonoid/anthocyanin branch, which encodes the enzymes involved in the production of anthocyanins and proanthocyanidins (condensed tannins) associated with <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> as well as different flavonoids involved in nodulation and plant defense, is probably the most studied segment of the phenylpropanoid pathway. Numerous enzymes, coded mostly by small gene families, catalyze the synthesis of these compounds. However, only gene sequences for chalcone synthase ( <italic> <span class="gene">CHS</span> </italic> , Ryder et al., <xref ref-type="bibr" rid="B56"> 1987 </xref> ) and chalcone isomerase ( <italic> <span class="gene">CHI</span> </italic> , Mehdy et al., unpublished, sequence available at <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/nuccore/Z15046.1"> http://www.ncbi.nlm.nih.gov/nuccore/Z15046.1 </ext-link> ) were available for common bean in databases before the initiation of this study. The isoflavonoid branch is unique to legumes and poorly understood in common bean. No sequence information was available in common bean for the genes coding for the enzymes of this branch of the phenylpropanoid pathway, except a <italic> P. coccineus </italic> EST corresponding to isoflavone reductase-like protein (IFR-l, Bui et al., unpublished, sequence available at <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/nucest/ca914441"> http://www.ncbi.nlm.nih.gov/nucest/ca914441 </ext-link> ) at the initiation of this study. The regulation of the phenylpropanoid pathway by transcription factors is well characterized in maize, Arabidopsis, snapdragon, or petunia and numerous genes coding for transcription factors have been isolated from these plants (Chandler et al., <xref ref-type="bibr" rid="B8"> 1989 </xref> ; Martin et al., <xref ref-type="bibr" rid="B40"> 1991 </xref> ; Menting et al., <xref ref-type="bibr" rid="B45"> 1994 </xref> ; Kranz et al., <xref ref-type="bibr" rid="B33"> 1998 </xref> ). However, at the beginning of this study only two common bean gene sequences for regulatory genes of the phenylpropanoid pathway [ <italic> KAP2 </italic> (Lindsay et al., <xref ref-type="bibr" rid="B38"> 2002 </xref> ) and bZIP <italic> TGA1 </italic> (Tucker et al., <xref ref-type="bibr" rid="B63"> 2002 </xref> )] were available.
					</p>
				
					<p>&nbsp;&nbsp;
						Associations between <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes and enzymes of anthocyanin biosynthetic pathway have been identified in soybean. For example, the <italic> I </italic> locus was associated with a <italic> <span class="gene">CHS</span> </italic> gene cluster on chromosome <span class="gene">Gm08</span> (Todd and Vodkin, <xref ref-type="bibr" rid="B62"> 1996 </xref> ) while the <italic> T </italic> locus was associated with the <italic> F3&#x02032;H </italic> gene on chromosome <span class="gene">Gm06</span> (Toda et al., <xref ref-type="bibr" rid="B61"> 2002 </xref> ; Zabala and Vodkin, <xref ref-type="bibr" rid="B71"> 2003 </xref> ). However, in common bean none of the nine <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes has been associated with a phenylpropanoid pathway genes. Caldas and Blair ( <xref ref-type="bibr" rid="B7"> 2009 </xref> ) identified a number of QTL for condensed tannins on several linkage groups and associated them with putative <span class="trait">color</span> genes. For example, QTL Cst1a (closest marker Y04.2) on linkage group Pv7 was associated with <italic> P </italic> gene, Cst2a (OAM10) on Pv3 with <italic> Z </italic> gene, Cst3b (B1DB1D) on Pv10 with <italic> Bip </italic> while QTL Cst1c (L041.05) on Pv8 was associated with <italic> C </italic> gene. The current study identified potential associations between several <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes and phenylpropanoid pathway genes in common bean. In particular, <span class="trait">seed</span> <span class="trait">coat</span> pattern gene <italic> Z </italic> (zonal) mapped close to <italic> <span class="gene">4CL1</span> </italic> gene on chromosome Pv3, <italic> V </italic> gene (violet factor) close to F3&#x02032;5&#x02032;H on Pv6 and <italic> P </italic> gene (pigment ground factor) was placed close to the transcription factor Myb15 on chromosome Pv7. In a number of cases regions of common bean and soybean chromosomes that share synteny also contained mapped <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes. For example, Pv2 ( <italic> Rk, B </italic> ) and <span class="gene">Gm08</span> ( <italic> I, O </italic> ), Pv3 ( <italic> Z </italic> ) and <span class="gene">Gm02</span> ( <italic> Wp </italic> ), Pv6 ( <italic> V </italic> ) and <span class="gene">Gm13</span> ( <italic> <span class="gene">W1</span> </italic> ), Pv08 ( <italic> Gy, C </italic> ) and Gm ( <italic> R </italic> ), and Pv09 ( <italic> T </italic> ) and <span class="gene">Gm06</span> ( <italic> T </italic> ). The phenylpropanoid pathway genes that are located in these regions would be interesting to explore more thoroughly as candidates for these classical <span class="trait">color</span> genes. The information gained from studies in one species may help to narrow the search in the other.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Allele-specific marker development and selection efficiencies for both flavonoid 3'-hydroxylase and flavonoid 3',5'-hydroxylase genes in soybean subgenus soja.</h2>
			<h3>PMID:23463490  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo Y(1), Qiu LJ.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Color</span> is one of the phenotypic markers mostly used to study soybean (Glycine max L. Merr.) genetic, molecular and biochemical processes. Two P450-dependent mono-oxygenases, flavonoid 3'-hydroxylase (F3'H; EC1.14.3.21) and flavonoid 3',5'-hydroxylase (F3'5'H, EC1.14.13.88), both catalyzing the hydroxylation of the B-ring in flavonoids, play an important role in coloration. Previous studies showed that the T locus was a gene encoding <span class="gene">F3'H</span> and the <span class="gene">W1</span> locus co-segregated with a gene encoding F3'5'H in soybean. These two genetic loci have identified to control <span class="trait">seed</span> <span class="trait">coat,</span> flower and pubescence colors. However, the allelic distributions of both <span class="gene">F3'H</span> and F3'5'H genes in soybean were unknown. In this study, three novel alleles were identified (two of four alleles for <span class="gene">GmF3'H</span> and one of three alleles for <span class="gene">GmF3'5'H).</span> A set of gene-tagged markers was developed and verified based on the sequence diversity of all seven alleles. Furthermore, the markers were used to analyze soybean accessions including 170 cultivated soybeans (G. max) from a mini core collection and 102 wild soybeans (G. soja). For both <span class="gene">F3'H</span> and F3'5'H, the marker selection efficiencies for pubescence <span class="trait">color</span> and flower <span class="trait">color</span> were determined. The results showed that one <span class="gene">GmF3'H</span> allele explained 92.2 % of the variation in tawny and two gmf3'h alleles explained 63.8 % of the variation in gray pubescence colors. In addition, two <span class="gene">GmF3'5'H</span> alleles and one gmF3'5'h allele explained 94.0 % of the variation in purple and 75.3 % in white flowers, respectively. By the combination of the two loci, <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> was determined. In total, 90.9 % of accessions possessing both the gmf3'h-b and gmf3'5'h alleles had yellow <span class="trait">seed</span> coats. Therefore, <span class="trait">seed</span> <span class="trait">coat</span> colors are controlled by more than two loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Progress in genes related to seed-coat <span class="trait">color</span> in soybean].</h2>
			<h3>PMID:22698739  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song J(1), Guo Y, Yu LJ, Qiu LJ.
				<br/> 
				年份：2012 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed-coat <span class="trait">color</span> has changed from black to yellow during natural and artificial selection of cultivated soybean from wild soybean, and it is also an important morphological marker. Therefore, discovering genes related to the soybean seed-coat <span class="trait">color</span> will play a very important role in breeding and evolutionary study. Different seed-coat colors caused by deposition of various anthocyanin pigments. Although pigmentation has been well dissected at molecular level in several plant species, the genes controlling natural variation of seed-coat <span class="trait">color</span> in soybean remain to be unknown. Genes related to seed-coat <span class="trait">color</span> in soybean were discussed in this paper, including 5 genetic loci (I, T, W1, R and O). Locus I is located in a region that riches in chalcone synthase (CHS) genes on chromosome 8. Gene <span class="gene">CHS</span> is a multi-gene family with highly conserved sequences in soybean. Locus T located on chromosome 6 has been cloned and verified, which encodes a flavon-oid-3'-hydroxylase. Mutant of <span class="gene">F3'H</span> can not interact with the heme-binding domain due to lack of conservative domain GGEK caused by a nucleotide deletion in the coding region of F3'H. Locus R is located between A668-1 and K387-1 on chromosome 9 (linkage group K). This locus may encode a R2R3 MYB transcription factor or a UDP flavonoid 3-O glyco-syltransferase. Locus O is located between Satt207 and Satt493 on chromosome 8 (linkage group A2) and its molecular characteristics has not been characterized. Locus <span class="gene">W1</span> may be a homology of F3'5'H gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">F3'H</span> protein is localized to the tonoplast in the <span class="trait">seed</span> <span class="trait">coat</span> hilum.</h2>
			<h3>PMID:22258749  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Toda K(1), Kuroiwa H, Senthil K, Shimada N, Aoki T, Ayabe S, Shimada S, Sakuta M, Miyazaki Y, Takahashi R.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We previously isolated a soybean (Glycine max (L.) Merr.) flavonoid 3'-hydroxylase (F3'H) gene (sf3'h1) corresponding to the T locus, which controls pubescence and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color,</span> from two near-isogenic lines (NILs), To7B (TT) and To7G (tt). The T allele is also associated with chilling tolerance. Here, Western-blot analysis shows that the <span class="gene">sf3'h1</span> protein was predominantly detected in the hilum and funiculus of the immature <span class="trait">seed</span> <span class="trait">coat</span> in To7B, whereas <span class="gene">sf3'h1</span> was not detected in To7G. A truncated <span class="gene">sf3'h1</span> protein isolated from To7G was detected only upon enrichment by immunoprecipitation. An analysis using diphenylboric acid 2-aminoethyl ester (DBPA) staining revealed that flavonoids accumulated in the hilum and the funiculus in both To7B and To7G. Further, the scavenging activity of the 1,1-diphenyl-2-picrylhydrazyl (DPPH) radical in methanol extracts from the funiculus and hilum of To7B was higher than that of To7G. Moreover, the enzymatic activity of <span class="gene">F3'H</span> was detected using microsomal fractions from yeast transformed with <span class="gene">sf3'h1</span> from To7B, but not from To7G. These results indicate that <span class="gene">sf3'h1</span> is involved in flavonoid biosynthesis in the <span class="trait">seed</span> <span class="trait">coat</span> and affects the antioxidant properties of those tissues. As shown by immunofluorescence microscopy, the <span class="gene">sf3'h1</span> protein was detected primarily around the vacuole in the parenchymatic cells of the hilum in To7B. Further immunoelectron microscopy detected <span class="gene">sf3'h1</span> protein on the membranous structure of the vacuole. Based on these observations, we conclude that F3'H, which is a cytochrome <span class="gene">P450</span> monooxygenase and has been found to be localized to the ER in other plant systems, is localized in the tonoplast in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of two anthocyanidin reductase genes and three red-brown soybean accessions with reduced anthocyanidin reductase 1 mRNA, activity, and <span class="trait">seed</span> <span class="trait">coat</span> proanthocyanidin amounts.</h2>
			<h3>PMID:22107112  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kovinich N(1), Saleem A, Arnason JT, Miki B.
				<br/> 
				年份：2012 Jan 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Anthocyanidin reductase (ANR; EC 1.3.1.77) catalyzes a key step in the biosynthesis of proanthocyanidins (PAs; also known as condensed tannins), flavonoid metabolites responsible for the brown pigmentation of seeds. Here, two ANR genes (ANR1 and ANR2) from the <span class="trait">seed</span> <span class="trait">coat</span> of brown soybean (Glycine max (L.) Merr.) have been isolated and their enzymatic function confirmed for the reduction of cyanidin to (-)-epicatechin in vitro. Biochemical and genetic comparisons of soybean lines differing in <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> revealed three red-brown lines to exhibit major reductions in the amounts of soluble PAs in the <span class="trait">seed</span> <span class="trait">coat</span> compared to brown soybean lines. Two spontaneous mutants with red-brown grain <span class="trait">color</span> had reduced <span class="gene">ANR1</span> gene expression in the <span class="trait">seed</span> <span class="trait">coat,</span> and an EMS-mutagenized red-brown mutant had nonsynonymous substitutions that resulted in slightly reduced <span class="gene">ANR1</span> activity in vitro. These results suggest that defects in the <span class="gene">ANR1</span> gene can be associated with red-brown soybean grain <span class="trait">color.</span> These results suggest that suppressing <span class="gene">ANR1</span> gene expression or activity may be a rational approach toward engineering <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> to enable the visual identification of genetically modified soybean grains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Coloring genetically modified soybean grains with anthocyanins by suppression of the proanthocyanidin genes <span class="gene">ANR1</span> and ANR2.</h2>
			<h3>PMID:22083247  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kovinich N(1), Saleem A, Rintoul TL, Brown DC, Arnason JT, Miki B.
				<br/> 
				年份：2012 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Detection and quantification of the levels of adventitious presence of genetically modified (GM) soybeans in non-GM grain shipments currently requires sophisticated tests that can have issues with their reproducibility. We show here that pigment biosynthesis in the soybean <span class="trait">seed</span> <span class="trait">coat</span> can be manipulated to provide a distinct <span class="trait">color</span> that would enable the simple visible detection of the GM soybean grain. We observed that a distinct red-brown grain <span class="trait">color</span> could be engineered by the simultaneous suppression of two proanthocyanidin (PA) genes, ANTHOCYANIDIN REDUCTASE1 (ANR1) and ANR2. Multiple reaction monitoring by liquid chromatography tandem mass spectrometry was used to quantify differentially accumulated <span class="trait">seed</span> <span class="trait">coat</span> metabolites, and revealed the redirection of metabolic flux into the anthocyanin pigment pathway and unexpectedly the flavonol-3-O-glucoside pathway. The upregulations of anthocyanin isogenes (DFR1 and GST26) and the anthocyanin/flavonol-3-O-glycosyltransferase (UGT78K2) were identified by quantitative RT-PCR to be endogenous feedback and feedforward responses to overaccumulation of upstream flavonoid intermediates resulting from <span class="gene">ANR1</span> and ANR2 suppressions. These results suggested the transcription of flavonoid genes to be a key component of the mechanism responsible for the redirection of metabolite flux. This report identifies the suppression of PA genes to be a novel approach for engineering pigmentation in soybean grains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loss-of-function mutations affecting a specific Glycine max R2R3 MYB transcription factor result in brown hilum and brown <span class="trait">seed</span> coats.</h2>
			<h3>PMID:22070454  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gillman JD(1), Tetlow A, Lee JD, Shannon JG, Bilyeu K.
				<br/> 
				年份：2011 Nov 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Although modern soybean cultivars feature yellow <span class="trait">seed</span> coats, with the only <span class="trait">color</span> variation found at the hila, the ancestral condition is black <span class="trait">seed</span> coats. Both <span class="trait">seed</span> <span class="trait">coat</span> and hila coloration are due to the presence of phenylpropanoid pathway derivatives, principally anthocyanins. The genetics of soybean <span class="trait">seed</span> <span class="trait">coat</span> and hilum coloration were first investigated during the resurgence of genetics during the 1920s, following the rediscovery of Mendel's work. Despite the inclusion of this phenotypic marker into the extensive genetic maps developed for soybean over the last twenty years, the genetic basis behind the phenomenon of brown <span class="trait">seed</span> coats (the R locus) has remained undetermined until now. RESULTS: In order to identify the gene responsible for the r gene effect (brown hilum or <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color),</span> we utilized bulk segregant analysis and identified recombinant lines derived from a population segregating for two phenotypically distinct alleles of the R locus. Fine mapping was accelerated through use of a novel, bioinformatically determined set of Simple Sequence Repeat (SSR) markers which allowed us to delimit the genomic region containing the r gene to less than 200 kbp, despite the use of a mapping population of only 100 F6 lines. Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype. We observed a near perfect correlation between the mRNA expression levels of the functional R gene candidate and an UDP-glucose:flavonoid 3-O-glucosyltransferase (UF3GT) gene, which is responsible for the final step in anthocyanin biosynthesis. In contrast, when a null allele of <span class="gene">Glyma09g36990</span> is expressed no upregulation of the UF3GT gene was found. CONCLUSIONS: We discovered an allelic series of four loss of function mutations affecting our R locus gene candidate. The presence of any one of these mutations was perfectly correlated with the brown <span class="trait">seed</span> coat/hilum phenotype in a broadly distributed survey of soybean cultivars, barring the presence of the epistatic dominant I allele or gray pubescence, both of which can mask the effect of the r allele, resulting in yellow or buff hila. These findings strongly suggest that loss of function for one particular <span class="trait">seed</span> coat-expressed R2R3 MYB gene is responsible for the brown <span class="trait">seed</span> coat/hilum phenotype in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combined analysis of transcriptome and metabolite data reveals extensive differences between black and brown nearly-isogenic soybean ( <italic> Glycine max </italic> ) <span class="trait">seed</span> coats enabling the identification of pigment isogenes</h2>
			<h3>PMID:21801362</h3>
				

				
					<h3>PMCID:3163566</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Kovinich </surname>  <given-names> Nik </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Nikola.Kovinich@agr.gc.ca </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Saleem </surname>  <given-names> Ammar </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> Ammar.Saleem@uOttawa.ca </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Arnason </surname>  <given-names> John T </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> John.Arnason@uottawa.ca </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Miki </surname>  <given-names> Brian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Brian.Miki@agr.gc.ca </email>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: The R locus controls the <span class="trait">color</span> of pigmented soybean (Glycine max) seeds. However information about its control over <span class="trait">seed</span> <span class="trait">coat</span> biochemistry and gene expressions remains limited. The <span class="trait">seed</span> coats of nearly-isogenic black (iRT) and brown (irT) soybean (Glycine max) were known to differ by the presence or absence of anthocyanins, respectively, with genes for only a single enzyme (anthocyanidin synthase) found to be differentially expressed between isolines. We recently identified and characterized a UDP-glycose:flavonoid-3-O-glycosyltransferase (UGT78K1) from the <span class="trait">seed</span> <span class="trait">coat</span> of black (iRT) soybean with the aim to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> by suppression of an anthocyanin-specific gene. However, it remained to be investigated whether <span class="gene">UGT78K1</span> was overexpressed with anthocyanin biosynthesis in the black (iRT) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown (irT) tissue.In this study, we performed a combined analysis of transcriptome and metabolite data to elucidate the control of the R locus over <span class="trait">seed</span> <span class="trait">coat</span> biochemistry and to identify pigment biosynthesis genes. Two differentially expressed late-stage anthocyanin biosynthesis isogenes were further characterized, as they may serve as useful targets for the manipulation of soybean grain <span class="trait">color</span> while minimizing the potential for unintended effects on the plant system. RESULTS: Metabolite composition differences were found to not be limited to anthocyanins, with specific proanthocyanidins, isoflavones, and phenylpropanoids present exclusively in the black (iRT) or the brown (irT) <span class="trait">seed</span> <span class="trait">coat.</span> A global analysis of gene expressions identified <span class="gene">UGT78K1</span> and 19 other anthocyanin, (iso)flavonoid, and phenylpropanoid isogenes to be differentially expressed between isolines. A combined analysis of metabolite and gene expression data enabled the assignment of putative functions to biosynthesis and transport isogenes. The recombinant enzymes of two genes were validated to catalyze late-stage steps in anthocyanin biosynthesis in vitro and expression profiles of the corresponding genes were shown to parallel anthocyanin biosynthesis during black (iRT) <span class="trait">seed</span> <span class="trait">coat</span> development. CONCLUSION: Metabolite composition and gene expression differences between black (iRT) and brown (irT) <span class="trait">seed</span> coats are far more extensive than previously thought. Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black (iRT) and brown (irT) <span class="trait">seed</span> coats, and <span class="gene">UGT78K2</span> and <span class="gene">OMT5</span> were validated to code UDP-glycose:flavonoid-3-O-glycosyltransferase and anthocyanin 3'-O-methyltransferase proteins in vitro, respectively. Duplicate gene copies for several enzymes were overexpressed in the black (iRT) <span class="trait">seed</span> <span class="trait">coat</span> suggesting more than one isogene may have to be silenced to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> using RNA interference.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> R </italic> locus controls the <span class="trait">color</span> of pigmented soybean ( <italic> Glycine max </italic> ) seeds. However information about its control over <span class="trait">seed</span> <span class="trait">coat</span> biochemistry and gene expressions remains limited. The <span class="trait">seed</span> coats of nearly-isogenic black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) soybean ( <italic> Glycine max </italic> ) were known to differ by the presence or absence of anthocyanins, respectively, with genes for only a single enzyme (anthocyanidin synthase) found to be differentially expressed between isolines. We recently identified and characterized a UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase ( <italic> <span class="gene">UGT78K1</span> </italic> ) from the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean with the aim to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> by suppression of an anthocyanin-specific gene. However, it remained to be investigated whether <italic> <span class="gene">UGT78K1</span> </italic> was overexpressed with anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolite composition differences were found to not be limited to anthocyanins, with specific proanthocyanidins, isoflavones, and phenylpropanoids present exclusively in the black ( <italic> iRT </italic> ) or the brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat.</span> A global analysis of gene expressions identified <italic> <span class="gene">UGT78K1</span> </italic> and 19 other anthocyanin, (iso)flavonoid, and phenylpropanoid isogenes to be differentially expressed between isolines. A combined analysis of metabolite and gene expression data enabled the assignment of putative functions to biosynthesis and transport isogenes. The recombinant enzymes of two genes were validated to catalyze late-stage steps in anthocyanin biosynthesis <italic> in vitro </italic> and expression profiles of the corresponding genes were shown to parallel anthocyanin biosynthesis during black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolite composition and gene expression differences between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats are far more extensive than previously thought. Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively. Duplicate gene copies for several enzymes were overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> suggesting more than one isogene may have to be silenced to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> using RNA interference.
					</p>
				
					<p>&nbsp;&nbsp;
						Six genetic loci ( <italic> I </italic> , <italic> R </italic> , <italic> T </italic> , <italic> Wp </italic> , <italic> <span class="gene">W1</span> </italic> , and <italic> O </italic> ) [ <xref ref-type="bibr" rid="B6"> 6 </xref> ] identified by classical genetics control flavonoid-based <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean. The <italic> I </italic> locus controls the presence or absence and spatial distribution of flavonoid pigments and has four alleles ( <italic> I </italic> , <italic> I <sup> i </sup> </italic> , <italic> I <sup> k </sup> </italic> , <italic> i </italic> ); <italic> I </italic> gives completely non-pigmented <span class="trait">seed</span> <span class="trait">coat,</span> <italic> I <sup> i </sup> </italic> restricts pigment to the hilum and <italic> I <sup> k </sup> </italic> to a saddle-shaped region, whereas the <italic> i </italic> allele results in a fully pigmented <span class="trait">seed</span> <span class="trait">coat</span> [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. The recessive <italic> i </italic> allele results from spontaneous deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences and results in the abolishment of a posttranscriptional RNA silencing mechanism that results in the increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">W1</span> </italic> locus controls flower <span class="trait">color</span> and affects <span class="trait">seed</span> <span class="trait">color</span> only in an <italic> iRt </italic> background; where <italic> <span class="gene">W1</span> </italic> and <italic> <span class="gene">w1</span> </italic> alleles give imperfect black and buff <span class="trait">seed</span> <span class="trait">coat</span> colors, respectively [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]. The <italic> <span class="gene">W1</span> </italic> allele for purple flower <span class="trait">color</span> was shown to code flavonoid 3',5'-hydroxylase (F3'5'H), as a 65-bp insertion in the gene ( <italic> F3'5'H </italic> ) co-segregated with white flower <span class="trait">color</span> ( <italic> <span class="gene">w1</span> </italic> ) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The Wp locus was suggested to code the flavonone 3-hydroxylase gene ( <italic> <span class="gene">F3H1</span> </italic> ) by microarray analysis as high levels of <italic> <span class="gene">F3H1</span> </italic> transcripts co-segregated with purple flower <span class="trait">color</span> ( <italic> Wp </italic> ), and low levels with pink ( <italic> wp </italic> ) flowers [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. In the <span class="trait">seed</span> <span class="trait">coat,</span> recessive <italic> wp </italic> resulted in a change from black ( <italic> iRTWp </italic> ) to a lighter grayish ( <italic> iRTwp </italic> ) <span class="trait">color</span> [ <xref ref-type="bibr" rid="B14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Finally, the <italic> R </italic> locus controls the presence ( <italic> R </italic> ) or absence ( <italic> r </italic> ) of anthocyanins in black ( <italic> iRT </italic> ) or brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, respectively [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]. Despite the identification of several pigment genes from soybean, only transcripts for anthocyanidin synthase (ANS) genes ( <italic> ANS23-1 </italic> , <italic> ANS27-1 </italic> and <italic> ANS100 </italic> ) have been shown to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the brown ( <italic> irT </italic> ) nearly-isogenic line by northern blot [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]. The upregulation of several <span class="gene">ANS</span> genes suggests the <italic> R </italic> locus to code a regulatory factor, and raises the possibility that other isogenes for anthocyanin biosynthesis may be upregulated.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, a cDNA coding the UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase (UF3GT) gene ( <italic> <span class="gene">UGT78K1</span> </italic> ) was isolated from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and shown to function in anthocyanin biosynthesis <italic> in vitro </italic> and by complementation of a gene mutation in Arabidopsis [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]. However <italic> <span class="gene">UGT78K1</span> </italic> expressions have not been investigated in relation to <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Among the flavonoid genes identified from soybean <span class="trait">seed</span> coats, only <italic> <span class="gene">ANS</span> </italic> transcripts were found to be overexpressed in the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean compared to the brown ( <italic> irT </italic> ) isoline, however no RFLP difference was observed when probing <italic> <span class="gene">ANS</span> </italic> genes [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]. This suggests that <italic> <span class="gene">ANS</span> </italic> overexpression is not the only molecular difference between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats. To examine whether other anthocyanin genes are upregulated, transcript profiles of the recently identified <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] and the <italic> <span class="gene">DFR1</span> </italic> gene [GenBank: <ext-link ext-link-type="gen" xlink:href="AF167556"> AF167556 </ext-link> ] were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA by quantitative RT-PCR (qRT-PCR) using <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> transcripts [GenBank: <ext-link ext-link-type="gen" xlink:href="AY382829"> AY382829 </ext-link> /GenBank: <ext-link ext-link-type="gen" xlink:href="AY382830"> AY382830 </ext-link> ] as a positive control. Phosphenol pyruvate carboxylase ( <italic> PEPC </italic> ) [GenBank: <ext-link ext-link-type="gen" xlink:href="D10717"> D10717 </ext-link> ] was used as an endogenous reference for normalization of the qRT-PCR measurements of target genes across <span class="trait">seed</span> <span class="trait">coat</span> samples, as PEPC is expressed at similar levels in a wide range of soybean tissues and has been used previously as a reference for gene expressions during soybean <span class="trait">seed</span> <span class="trait">coat</span> development [ <xref ref-type="bibr" rid="B8"> 8 </xref> , <xref ref-type="bibr" rid="B24"> 24 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR demonstrated <italic> <span class="gene">UGT78K1</span> </italic> to be significantly overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at the transition stage (300 mg - 400 mg FSW) and early maturation stage (400 - 500 mg FSW) of <span class="trait">seed</span> development compared to the brown ( <italic> irT </italic> ) isoline, and to be almost undetectable at earlier stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). Stereomicroscopy and photospectroscopic measurements confirmed these developmental stages to coincide with the stages of anthocyanin biosynthesis in the black ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). In contrast to <italic> <span class="gene">UGT78K1</span> </italic> , <italic> <span class="gene">DFR1</span> </italic> was not differentially expressed at the stages of anthocyanin biosynthesis (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> mRNAs were confirmed to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at most stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) as shown previously [ <xref ref-type="bibr" rid="B17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">seed</span> <span class="trait">coat</span> palisade cells of black ( <italic> iRT </italic> ) soybean are unusual compared to those of other model plants used for the study of anthocyanin biosynthesis (e.g. Arabidopsis, Maize, and Petunia) because black ( <italic> iRT </italic> ) soybean palisade cells synthesize both anthocyanins and proanthocyanidins (PAs). ANR and LAR enzymes have been identified to catalyze the conversion of cyanidin to epicatechin and leucocyanidin to catechin, respectively, which are the putative initial biochemical steps of PA biosynthesis [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> ]. To investigate how a <span class="trait">seed</span> tissue may coordinate the biosynthesis of these parallel pigment pathways, the expressions of putative PA genes <italic> <span class="gene">ANR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433915"> JF433915 </ext-link> ] and <italic> <span class="gene">LAR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433916"> JF433916 </ext-link> ] were examined in relation to anthocyanin gene expressions. qRT-PCR demonstrated <italic> <span class="gene">ANR1</span> </italic> expressions to decrease throughout <span class="trait">seed</span> <span class="trait">coat</span> development in both genotypes and to be almost undetectable by the stages of <italic> <span class="gene">UGT78K1</span> </italic> overexpressions and anthocyanin biosynthesis (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). By contrast, <italic> <span class="gene">LAR1</span> </italic> expressions were downregulated at the stages of anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, these results show that the recently identified <italic> <span class="gene">UGT78K1</span> </italic> is upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> in addition to <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> , and that there is a simultaneous downregulation of the PA gene <italic> <span class="gene">LAR1</span> </italic> at the stages of anthocyanin biosynthesis. These results demonstrate that expression differences between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats are not limited to anthocyanin genes, and may suggest extensive differences in gene expressions and phenolic compositions.
					</p>
				
					<p>&nbsp;&nbsp;
						The structures of anthocyanins from only a few (Korean) black soybean varieties have been described [ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ] and information regarding other phenolic compositions may be limited to proanthocyanidins (PAs), where acid-catalyzed hydrolysis of black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats showed the PAs to be 3',4'-hydroxylated [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. A more extensive analysis of phenolic composition differences between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats could greatly improve our understanding of <span class="trait">seed</span> pigmentation, and may suggest gene activities that function in pigment biosynthesis. For these reasons, black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) Clark <span class="trait">seed</span> <span class="trait">coat</span> metabolites were extracted at the transition stage (300 mg - 400 mg FSW) of <span class="trait">seed</span> development with multiple solvents and analyzed by targeted high performance liquid chromatography-diode array detection (HPLC-DAD) and non-targeted HPLC-tandem mass spectrometry (HPLC-MS/MS). To ensure <span class="trait">seed</span> coats were at the same stage of development, the days post anthesis, pod length, pod <span class="trait">color,</span> embryo morphology, and transcript levels of the developmental marker gene <span class="gene">Gm-r1083-1191,</span> a putative cutin biosynthesis gene [ <xref ref-type="bibr" rid="B29"> 29 </xref> ], and the metabolism gene <italic> <span class="gene">DFR1</span> </italic> were ensured to be equivalent between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) tissues (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						Nearly 75% of genes in the soybean genome are present in multiple copies [ <xref ref-type="bibr" rid="B19"> 19 </xref> ], and previously only transcripts for <italic> <span class="gene">ANS</span> </italic> isogenes were found to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]. For a global analysis of isogene expressions, black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) Clark <span class="trait">seed</span> coats were dissected at the transition stage of <span class="trait">seed</span> development (300 - 400 mg FSW) for microarray analysis using the Affymetrix Soybean GeneChip. <span class="trait">Seed</span> coats were ensured to be at the same stage of development as described above for the metabolite analyses (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						A BLASTP search of the soybean genome sequence Glyma1 ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/soybean"> http://www.phytozome.net/soybean </ext-link> ) using the conserved plant secondary product glycosyltransferase signature sequence (PSPG box) [ <xref ref-type="bibr" rid="B32"> 32 </xref> ] identified 214 annotated Family 1 glycosyltransferases (UGTs)(Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> : Table S3). Of the 214 UGTs, only 2 were found to be upregulated more than 2-fold from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat;</span> the recently identified UF3GT gene <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] was upregulated more than 2.8-fold and a second uncharacterized glycosyltransferase, which we named <italic> <span class="gene">UGT78K2</span> </italic> in accordance with UGT nomenclature [ <xref ref-type="bibr" rid="B33"> 33 </xref> ], was upregulated more than 3-fold (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A search of the soybean genome sequence with the conserved OMT-domain sequence (PLN02589; NCBI) and the ontology ' <italic> O </italic> -methyltransferase', identified 84 annotated OMTs (Additional file <xref ref-type="supplementary-material" rid="S6"> 6 </xref> : Table S4). Of the 84 OMT genes, only probe sets for OMT-like and <span class="gene">OMT5</span> were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 2.4-fold and 2.5-fold, respectively (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 52 probe sets were downregulated more than 2-fold in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (i.e. black/brown &#x0003c; 0.5 fold); among these, 2 probe sets by more than 10-fold, 12 probe sets by more than 3-fold, with the remainder between 2- and 3-fold (Table <xref ref-type="table" rid="T4"> 4 </xref> ; Additional file <xref ref-type="supplementary-material" rid="S7"> 7 </xref> : Table S5). Classification of down-regulated genes using the Gene-Bins ontology tool demonstrated the majority to be "unclassified without homology" (45.5%) or "unclassified with homology" (12.1%), followed by "biosynthesis of secondary metabolites" (9.1%) (Additional file <xref ref-type="supplementary-material" rid="S8"> 8 </xref> : Figure S3). The latter class consisted primarily of characterized and putative (iso)flavonoid isogenes. Isogenes for common flavonoid biosynthesis (CHS1, CHS9, F3'H), PA biosynthesis (LAR1), and isoflavone biosynthesis (IFR1-like) were all down-regulated more than 2-fold in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 35-fold in the case of the <span class="gene">CHS1</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> ). The flavonoid genes <span class="gene">CHS9</span> and <span class="gene">F3'H</span> were down-regulated more than 21-fold and 2.5-fold, respectively, the putative isoflavone gene IFR1-like by more than 3.6-fold, and the putative PA gene <span class="gene">LAR1</span> by more than 2.2-fold (Table <xref ref-type="table" rid="T4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To validate the microarray results, 28 differentially-regulated genes were selected and their relative expression profiles were measured by semi-quantitative RT-PCR (semi-qRT-PCR) using black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> cDNA as templates. Phosphenol pyruvate carboxylase (PEPC) was used as the endogenous control to ensure equal loading and the flavonoid genes <italic> <span class="gene">CHS7</span> </italic> / <italic> CHS8 </italic> and the putative PA gene <italic> <span class="gene">ANR1</span> </italic> were used as negative controls as they were not found to be differentially expressed in the microarray analysis. Semi-qRT-PCR confirmed the differential-expressions of all 28 genes and demonstrated similar expressions for <italic> <span class="gene">CHS7</span> </italic> / <italic> CHS8 </italic> and <italic> <span class="gene">ANR1</span> </italic> (Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> : Figure S4). Furthermore, as the upregulated probe set <span class="gene">GmaAffx.42116.1.S1</span> did not distinguish between <italic> CHS4 </italic> and <italic> <span class="gene">CHS5</span> </italic> expressions, semi-qRT-PCR was performed with gene specific primers to show the <italic> CHS4 </italic> gene, and not the <italic> <span class="gene">CHS5</span> </italic> gene to be upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> : Figure S4).
					</p>
				
					<p>&nbsp;&nbsp;
						Anthocyanins in the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean are glycosylated at the 3-position of the C-ring (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). The glycosyltransferase <span class="gene">UGT78K2</span> is assigned to code a UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferase (UF3GT) protein (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ) and may have functional redundancies in anthocyanin biosynthesis with the previously characterized <span class="gene">UGT78K1</span> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ], which was also upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Two anthocyanins (petunidin-3- <italic> O </italic> -glucoside and peonidin-3- <italic> O </italic> -glucoside) and one deoxyanthocyanidin (diosmetinidin) from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> are methylated at the 3-position of the B-ring (the 3'-position) (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). The two upregulated <italic> O </italic> -methyltransferases were assigned to code anthocyanin 3'- <italic> O </italic> -methyltransferase (AOMT) proteins (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ) and may have redundant function in anthocyanin biosynthesis this tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Reducing the expressions of late-stage anthocyanin genes by RNA interference (RNAi) in the <span class="trait">seed</span> <span class="trait">coat</span> of black soybean may be an effective strategy to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> while minimizing unintended effects on other flavonoid subpathways. However, functional redundancies may require that multiple genes be silenced in order to reduce enzyme activity enough to reduce pigment levels. Two glycosyltransferases ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> relative to the nearly-isogenic brown ( <italic> irT </italic> ) tissue (Table <xref ref-type="table" rid="T3"> 3 </xref> ) and the upregulation paralleled the accumulation of anthocyanins with 3- <italic> O </italic> -glycosylated structures (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), suggesting the function of these genes as UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferases (UF3GTs). We have characterized <span class="gene">UGT78K1</span> previously and shown it to possess UF3GT activity towards anthocyanidins <italic> in vitro </italic> and <italic> in vivo </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]. To determine the catalytic function of <italic> <span class="gene">UGT78K2</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="HM591298"> HM591298 </ext-link> ], the ORF was cloned into an N-terminal hexahistidine fusion tag vector and expressed in <italic> E </italic> . <italic> coli </italic> for analysis of the coded recombinant protein <italic> in vitro </italic> . The recombinant protein (rUGT78K2) was purified by ion-metal-affinity chromatography (IMAC) had an apparent mass of 50.9 kDa that matched well with the calculated mass of the native <span class="gene">UGT78K2</span> protein (49.04 kDa) in addition to the 2.04 kDa N-terminal hexahistidine fusion tag encoded by the pET-14b vector (Figure <xref ref-type="fig" rid="F7"> 7A </xref> ). In assays containing UDP-glucose as the sugar donor and cyanidin as the acceptor, the recombinant enzyme transferred glucose to the 3-position of cyanidin to form cyanidin-3- <italic> O </italic> -glucoside (Figure <xref ref-type="fig" rid="F7"> 7B, C </xref> ), whereas the boiled enzyme and bacteria expressing the corresponding empty vector could not catalyze this reaction (not shown). Similarly, in assays containing UDP-galactose as the sugar donor and cyanidin as the acceptor, the recombinant enzyme transferred galactose to the 3-position of cyanidin to form cyanidin-3- <italic> O </italic> -galactoside (not shown). Cyanidin-3- <italic> O </italic> -glucoside and cyanidin-3- <italic> O </italic> -galactoside from recombinant enzyme assays were identified by HPLC-DAD in comparison to the authentic standards, and confirmed by HPLC-MS/MS with the characteristic parent ions 449.0 [cyanidin-3- <italic> O </italic> -glucoside or cyanidin-3- <italic> O </italic> -galactoside] <sup> +1 </sup> and the MS/MS fragment 287.3 [cyanidin] <sup> +1 </sup> corresponding to the loss of glucose (-162 amu).
					</p>
				
					<p>&nbsp;&nbsp;
						Methylation of the B-ring of anthocyanins has a reddening effect on flower <span class="trait">color</span> (reviewed by [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]) and thus engineering the expression of 3'- <italic> O </italic> -methyltransferases could change the redness of <span class="trait">seed</span> <span class="trait">color.</span> However, despite the identification of several flavonoid OMTs from soybean, none have been shown to accept anthocyanins as substrates. Our microarray analysis identified two <italic> O </italic> -methyltransferases (OMT5 and OMT-like) to be upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> ) and the upregulation paralleled the accumulation of 3'- <italic> O </italic> -methylated anthocyanins in this tissue (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), suggesting the function of these genes as anthocyanin 3'- <italic> O </italic> -methyltransferases (AOMTs). Following the approach described above, the ORF of <italic> <span class="gene">OMT5</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="HQ856048"> HQ856048 </ext-link> ] was cloned for expression and functional characterization of the corresponding recombinant protein <italic> in vitro </italic> . The purified recombinant protein had an apparent mass of 30.0 kDa that matched well with the calculated mass (27.78 kDa) in addition to the 2.04 kDa N-terminal hexahistidine fusion tag encoded by the pET-14b vector (Figure <xref ref-type="fig" rid="F8"> 8A </xref> ). In assays containing S-adenosyl-L-methionine (SAM) as the methyl donor and cyanidin-3- <italic> O </italic> -glucoside as the acceptor, the recombinant enzyme transferred a methyl group to the 3'-position of the acceptor to form peonidin-3- <italic> O </italic> -glucoside (Figure <xref ref-type="fig" rid="F8"> 8B, C </xref> ), whereas the boiled enzyme and bacteria expressing the corresponding empty vector could not catalyze this reaction (not shown). Peonidin-3- <italic> O </italic> -glucoside from recombinant enzyme assays was identified by HPLC-DAD in comparison to the authentic standard, and confirmed by HPLC-MS/MS with the characteristic parent ion 463.1 [peonidin-3-O-glucoside] <sup> +1 </sup> and the MS/MS fragment 301.0 [peonidin] <sup> +1 </sup> corresponding to the loss of glucose (-162 amu).
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the function these genes, <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expression profiles were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA throughout black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) soybean <span class="trait">seed</span> development by quantitative RT-PCR (qRT-PCR) (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ) and expressions were compared to anthocyanin accumulation profiles throughout <span class="trait">seed</span> development (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Similar to <italic> <span class="gene">UGT78K1</span> </italic> , <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expressions peaked at the transition and early maturation stages of <span class="trait">seed</span> development (300 mg - 400 mg FSW, 400 mg - 500 mg FSW) in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ), the stages at which anthocyanins accumulate (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), and were low or almost undetectable at early stages of <span class="trait">seed</span> development in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and at all stages in the brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Seed</span> <span class="trait">coat</span> transcript profiles of <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> during <span class="trait">seed</span> development measured by qRT-PCR </bold> . Black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (black bars) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (white bars). Student's <italic> t </italic> test significant at P &#x0003c; 0.01 (**), student's t test significant at P &#x0003c; 0.05 (*).
					</p>
				
					<p>&nbsp;&nbsp;
						Together the qRT-PCR and recombinant enzyme assays suggest that <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> code UF3GT and AOMT proteins, respectively, that function in anthocyanin pigment biosynthesis in the <span class="trait">seed</span> of black ( <italic> iRT </italic> ) soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The anthocyanins identified from variety Clark were mainly similar to those reported previously from various Korean varieties [ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ] however Clark was the first shown to contain a 3-deoxyanthocyanidin (diosmetinidin) in the <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T2"> 2 </xref> ). 3-deoxyanthocyanidins are relatively rare intensely coloured pigments found in <italic> Sorghum bicolour </italic> in response to fungal attack [ <xref ref-type="bibr" rid="B36"> 36 </xref> ], and from the cotyledons of yellow soybean in response to UV-C treatment [ <xref ref-type="bibr" rid="B37"> 37 </xref> ]. The biosynthesis of 3-deoxyanthocyanidins from flavonones has been shown to be a minor activity of a DFR isogene in Sorghum [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. It would be interesting to determine whether the DFR gene that was highly overexpressed in the black Clark <span class="trait">seed</span> <span class="trait">coat</span> ( <italic> <span class="gene">DFR2</span> </italic> ) had similar catalytic activity, and whether the Korean varieties contain genetic alterations related to <italic> <span class="gene">DFR2</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> transcriptome compared to that of the nearly-isogenic brown ( <italic> irT </italic> ) tissue identified 80 probe sets to be upregulated (Table <xref ref-type="table" rid="T3"> 3 </xref> ). This is almost double the number of probe sets upregulated by overexpression of the MYB transcription factor <span class="gene">PAP1</span> in Arabidopsis (38 probe sets) [ <xref ref-type="bibr" rid="B21"> 21 </xref> ], similar to the number upregulated by overexpression of the anthocyanin factor LAP1 in <italic> M. truncatula </italic> (61 probe sets), and less than half the number upregulated by overexpression of LAP1 in <italic> M. sativa </italic> (270 probe sets) [ <xref ref-type="bibr" rid="B39"> 39 </xref> ]. The higher numbers of genes upregulated in anthocyanin-overaccumulating tissues of black soybean, <italic> M. truncatula </italic> and <italic> M. sativa </italic> likely reflect the fact that legumes have undergone genome duplication and that genes with multiple copies are frequently retained [ <xref ref-type="bibr" rid="B19"> 19 </xref> ]. Our data supported this hypothesis as duplicate copies of several genes were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (e.g. <italic> 4CL2 </italic> and <italic> 4CL-like </italic> , <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">UGT78K1</span> </italic> , <italic> <span class="gene">OMT5</span> </italic> and <italic> OMT </italic> - <italic> like </italic> , and <italic> GST21 </italic> and <italic> GST26 </italic> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Transcriptome analysis has previously demonstrated a critical role of <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes for isoflavonoid biosynthesis in developing embryos of soybean [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. However, our microarray and semi-quantitative RT-PCR data demonstrated that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> were not differentially regulated in black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Interestingly, it was <italic> CHS4 </italic> that was upregulated and <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> that were downregulatedin the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> and Table <xref ref-type="table" rid="T4"> 4 </xref> respectively). As <italic> CHS4 </italic> is upregulated with anthocyanin genes, it likely has a role in anthocyanin biosynthesis in this tissue. Consistent with this, spontaneous mutations of the dominant <italic> i <sup> i </sup> </italic> allele to the recessive <italic> i </italic> allele have been shown to affect the promoter region of the <italic> CHS4 g </italic> ene [ <xref ref-type="bibr" rid="B7"> 7 </xref> ]. By contrast, the downregulation of <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> may suggest that they have a role in the parallel downregulated biochemical pathways, such as flavonol and isoflavonoid biosynthesis, as these metabolites, and transcript levels of genes for their biosynthesis, were reduced or not present in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ). These results emphasize the complexity of isogene expressions that exist to achieve a single enzyme function in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Microarray analysis identified differentially regulated probe sets for only 3 gene functions (LAR, GST, and MATE) that have been previously shown to be involved in PA biosynthesis and transport. Recombinant LAR enzymes from several legume species have been shown to convert leucocyanidin to (+)-catechin <italic> in vitro </italic> [ <xref ref-type="bibr" rid="B25"> 25 </xref> , <xref ref-type="bibr" rid="B26"> 26 </xref> , <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B42"> 42 </xref> ]. However, the <italic> in vivo </italic> role of LAR in PA biosynthesis remains questionable as endogenous expression has been shown not to correlate with PA levels in <italic> M. truncatula </italic> and heterologous expression has failed to increase PA accumulations [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. Similarly, our metabolite data showed that soybean PAs lack catechin subunits (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Figure S1) and that <italic> <span class="gene">LAR1</span> </italic> expressions (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) do not parallel PA accumulations during <span class="trait">seed</span> development (not shown). Interestingly, 4 PAs were found to be exclusive to the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T2"> 2 </xref> ) however the total subunit compositions, the mean degree of polymerization (mDP), and the total amounts of soluble and solvent-insoluble PAs were not different compared to the brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Figure S1). These results suggest the PA oligomers of black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats may differ only in their subunit linkages. However, the potential influence of LAR or other PA genes on PA subunit linkages (if any) remains to be determined.
					</p>
				
					<p>&nbsp;&nbsp;
						From the set of differentially regulated genes identified by microarray, two putative late-stage anthocyanin genes were selected and the functions of their coded enzymes were determined <italic> in vitro </italic> . <italic> <span class="gene">UGT78K2</span> </italic> was found to code a UF3GT enzyme for anthocyanin biosynthesis (Figure <xref ref-type="fig" rid="F7"> 7 </xref> ). A UF3GT cDNA ( <italic> <span class="gene">UGT78K1</span> </italic> ) has recently been isolated from the black <span class="trait">seed</span> <span class="trait">coat</span> and functionally characterized [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]. <italic> <span class="gene">UGT78K2</span> </italic> was 93% similar to <italic> <span class="gene">UGT78K1</span> </italic> but possessed increased catalytic activity towards cyanidin (Figure <xref ref-type="fig" rid="F7"> 7D </xref> ). The identification gene redundancy has important implications for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> engineering by RNA interference, as both copies of the gene may have to be silenced to achieve a significant reduction in UF3GT activity. Similarly, two OMT genes ( <italic> <span class="gene">OMT5</span> </italic> and <italic> OMT-like </italic> ) were found to be overexpressed in the black <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> ). <italic> <span class="gene">OMT5</span> </italic> was cloned and found to code an AOMT enzyme <italic> in vitro </italic> (Figure <xref ref-type="fig" rid="F8"> 8 </xref> ). UF3GT and OMT genes may be ideal candidates for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> engineering as suppression of UF3GT genes ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) may result in a reduction in anthocyanin levels similar to that caused by a T-DNA insertion in the Arabidopsis UF3GT [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] and upregulation of an AOMT may result in a more red <span class="trait">seed</span> <span class="trait">coat,</span> as methylation of the B-ring of anthocyanins has a reddening effect on flower <span class="trait">color</span> (reviewed by [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]). However, it remains to be determined whether altering the expressions of these genes could be use to modify <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> without causing unintended effects on alternative biochemical pathways.
					</p>
				
					<p>&nbsp;&nbsp;
						Prior to the present study, currently known soybean genes did not map between markers A668_1 and K387_1 on chromosome 9 of the soybean physical genomic map, where the R locus is located, leaving no obvious candidates for the R locus gene [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. In our study, 14 genes identified to be differentially regulated in the black ( <italic> iRT </italic> ) soybean <span class="trait">seed</span> <span class="trait">coat</span> transcriptome were found to be located on chromosome 9 of the soybean genome sequence Glyma1 (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). Among the gene families represented, only WD40 and MYB have known roles in flavonoid regulation. However the genes demonstrated did not cluster with known flavonoid regulators by phylogenetic analysis (not shown). A Mob1/phocein gene was commonly upregulated in the black ( <italic> iRT </italic> ) soybean <span class="trait">seed</span> <span class="trait">coat</span> and Arabidopsis seedlings overexpressing <span class="gene">PAP1</span> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]. Other regulatory gene families represented on chromosome 9 were AP2/ERF, serine/threonine protein kinase, calcium/calmodulin-dependent protein kinase, and ethylene-responsive element-binding protein. Interestingly, all upregulated and downregulated genes on chromosome 9 were located in 5.16 Mb region that includes the nearest sequence markers to the R locus. Only the downregulated AP2/ERF transcription factor, the serine carboxypeptidase gene, and the gene moderately similar to the <italic> A. thaliana </italic> polyamine oxidase ATPAO1 were located between these sequence markers (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). As a result our study provides a short list of gene candidates that may serve as the focus of future efforts to identify the R locus gene.
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolite composition and gene expression differences between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats are far more extensive than previously thought. Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively. Duplicate gene copies for several enzymes were overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> suggesting more than one isogene may have to be silenced to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> using RNA interference.
					</p>
				
					<p>&nbsp;&nbsp;
						To clone the full-length coding sequences (CDSs) of <span class="gene">UGT78K2</span> and OMT5, PCR primers were designed based on the transcript sequence predicted for <span class="gene">Glyma08g07130</span> ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/soybean"> http://www.phytozome.net/soybean </ext-link> ) and based on the mRNA sequence of BT098523, respectively. Full-length CDSs for <span class="gene">UGT78K2</span> and <span class="gene">OMT5</span> were amplified from black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> cDNA by end-to-end PCR using primers UHF/UAR and HO5F/HO5R, respectively (Additional file <xref ref-type="supplementary-material" rid="S12"> 12 </xref> : Table S7) and a high-fidelity DNA polymerase (Invitrogen). The resulting <span class="gene">UGT78K2</span> and <span class="gene">OMT5</span> amplicons (1466 bp and 836 bp, respectively) were cloned into NdeI and BamHI sites, and NdeI and XhoI sites of the pET-14b vector (Novagen), respectively, and sequenced to confirm their identities.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Supplementary Figure S4 </bold> . Semi-qRT-PCR validation of the expressions of select genes found to be differentially expressed in black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats by microarray analysis. An asterix (*) represents genes located on Chromosome <span class="gene">Gm09</span> of the soybean Glyma1 genome sequence. Genes and corresponding differentially regulated probe sets: 4CL-L, <span class="gene">Gma.7423.2.S1_a_at;</span> 4CL-2, <span class="gene">Gma.8472.1.S1_at;</span> CHS4 and CHS5, <span class="gene">GmaAffx.42116.1.S1_at;</span> LAR1, <span class="gene">GmaAffx.34868.1.A1_at;</span> DFR2, <span class="gene">GmaAffx.80720.1.S1_at;</span> ANS2/ANS3, <span class="gene">Gma.1163.1.S1_at;</span> UGT78K1, <span class="gene">Gma.1002.2.S1_at;</span> UGT78K2, <span class="gene">GmaAffx.71999.1.S1_at;</span> OMT-like, <span class="gene">Gma.9647.1.S1_at;</span> OMT5, <span class="gene">GmaAffx.57777.1.S1_at;</span> GST26, <span class="gene">GmaAffx.71212.1.A1_at;</span> GST21, <span class="gene">Gma.5139.1.S1_at;</span> MYB50, <span class="gene">GmaAffx.81605.1.S1_at;</span> MYB159, <span class="gene">GmaAffx.39483.1.A1_at;</span> C2H2 ZF, <span class="gene">Gma.17736.1.S1_at;</span> WD40, <span class="gene">GmaAffx.45454.1.S1_at;</span> SCOF-1, <span class="gene">Gma.235.1.S1_at;</span> AP2, <span class="gene">GmaAffx.2469.1.S1_at;</span> EF-hand, <span class="gene">Gma.15972.1.A1_at;</span> ProtK, <span class="gene">GmaAffx.90491.1.A1_s_at;</span> LRR, <span class="gene">GmaAffx.12723.1.A1_at;</span> G4DT, <span class="gene">Gma.5621.1.S1_at;</span> PAO1-L, <span class="gene">Gma.3745.1.S1_at;</span> Am Oxy, <span class="gene">Gma.3745.1.S1_at;</span> PCT, <span class="gene">GmaAffx.78720.2.S1_at;</span> PUB22, <span class="gene">Gma.4530.1.A1_s_at;</span> 9O12a, <span class="gene">Gma.2605.1.S1_at;</span> 9012b, <span class="gene">Gma.2605.2.S1_at;</span> Lipase, <span class="gene">GmaAffx.90450.1.S1_at.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis of genes controlling natural variation of <span class="trait">seed</span> <span class="trait">coat</span> and flower colors in soybean.</h2>
			<h3>PMID:20584753  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang K(1), Jeong N, Moon JK, Lee YH, Lee SH, Kim HM, Hwang CH, Back K, Palmer RG, Jeong SC.
				<br/> 
				年份：2010 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean exhibits natural variation in flower and <span class="trait">seed</span> <span class="trait">coat</span> colors via the deposition of various anthocyanin pigments in the respective tissues. Although pigmentation in seeds or flowers has been well dissected at molecular level in several plant species, the genes controlling natural variation in anthocyanin traits in the soybean are not completely understood. To evaluate the genetic correlation between genetic loci and genes, 8 enzyme-encoding gene families and a transcription factor were localized in a soybean genome-wide genetic map. Among the <span class="trait">seed</span> <span class="trait">coat</span> color-controlling loci, the genetic location of the gene encoding for <span class="gene">W1</span> was substantiated in the context of the current soybean molecular genetic map and O was postulated to correspond to anthocyanidin reductase. Among the genetic loci that regulate flower pigmentation, the genetic locations of the genes encoding for W1, W4, and Wp were identified, <span class="gene">W3</span> was mapped on soybean linkage group B2 (chromosome 14), and <span class="gene">W2</span> was postulated to correspond to an MYB transcription factor. Correlation studies between the developed markers and 3 color-controlling loci provided important empirical data that should prove useful in the design of marker-assisted breeding schemes as well as future association studies involving soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Difference in chilling-induced flavonoid profiles, antioxidant activity and chilling tolerance between soybean near-isogenic lines for the pubescence <span class="trait">color</span> gene.</h2>
			<h3>PMID:20428921  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Toda K(1), Takahashi R, Iwashina T, Hajika M.
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Chilling tolerance is an important trait of soybeans [Glycine max (L.) Merr.] produced in cool climates. We previously isolated a soybean flavonoid 3' hydroxylase (F3'H) gene corresponding to the T locus, which controls pubescence and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color.</span> A genetic link between the T gene and chilling tolerance has been reported, although the exact underlying mechanisms remain unclear. Using the soybean near-isogenic lines (NILs) To7B (TT) and To7G (tt), we examined the relationship between chilling injury, antioxidant activity and flavonoid profiles associated with chilling treatment (15°C). Chilling injury was more severe in the second trifoliate leaves of To7G than in those of To7B. Hydrogen peroxide accumulation and lipid peroxidation were enhanced by chilling in To7G. Chilling-induced enhancement of antioxidant activity was more prominent in To7B than in To7G. High performance liquid chromatography analysis indicated that the contents of quercetin glycosides and isorhamnetin glycosides (3',4'-dihydroxylated flavonol derivatives) increase in the second trifoliate leaves of To7B after chilling treatment, whereas the same treatment increased kaempferol glycoside (4'-monohydroxylated flavonol derivatives) content in the corresponding leaves of To7G. Histochemical staining also demonstrated chilling-induced flavonoid accumulation. Microarray analysis and real-time reverse transcription-PCR demonstrated that the transcript levels of soybean <span class="gene">F3'H</span> are upregulated by chilling. The differences in chilling injury, antioxidant activity and flavonoid species between the two NILs support the notion that soybean <span class="gene">F3'H</span> affects chilling tolerance by increasing antioxidant activity via production of 3',4'-dihydroxylated flavonol derivatives.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL analysis of seed-flooding tolerance in soybean (Glycine max [L.] Merr.).</h2>
			<h3>PMID:26493141</h3>
				

				
			<p>
				作者：Sayama T(1), Nakazaki T(1), Ishikawa G(1), Yagasaki K(2), Yamada N(2), Hirota N(1), Hirata K(1), Yoshikawa T(1), Saito H(1), Teraishi M(1), Okumoto Y(1), Tsukiyama T(1), Tanisaka T(3).
				<br/> 
				年份：2009 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In soybean (Glycine max [L.] Merr.), varieties with seed-flooding tolerance at the geminating stage are desirable for breeding in countries with much rainfall at sowing time. Our study revealed great intervarietal variation in seed-flooding tolerance as evaluated by germination rate (GR) and normal seedling rate (NS). Pigmented <span class="trait">seed</span> <span class="trait">coat</span> and small <span class="trait">seed</span> weight tended to give a positive effect on seed-flooding tolerance. Subsequently, QTL analysis of <span class="gene">GR</span> and NS were performed and a total of four QTLs were detected. Among them, Sft1 on the linkage group H (LG_H) exhibited a large effect on <span class="gene">GR</span> after a 24-h treatment; however, Sft2 near the I locus on LG_A2 involved in <span class="trait">seed</span> <span class="trait">coat</span> pigmentation exhibited the largest effect on seed-flooding tolerance. Sft1, Sft3 and Sft4 were independent of <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> and <span class="trait">seed</span> weight. Based on the results, we discussed the physiological effects of genetic factors responsible for seed-flooding tolerance in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of several gy4 nulls from the USDA soybean germplasm collection provides new genetic resources for the development of high-quality tofu cultivars.</h2>
			<h3>PMID:18991447  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kim WS(1), Ho HJ, Nelson RL, Krishnan HB.
				<br/> 
				年份：2008 Dec 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Tofu, a cheese-like food made by curdling soy milk, is a major dietary staple of Asian countries. Consumption of tofu and other soy products is steadily increasing in North America due to its well-known health benefits. Soybean A(5), A(4), and B(3) peptide null lines 'Enrei' and 'Raiden' are commonly utilized in breeding programs to develop high-quality tofu cultivars. To expand the genetic diversity it is desirable to identify and utilize other A(5), A(4), and B(3) null genotypes in the development of improved tofu cultivars that are adapted to North American conditions. In this study were screened diverse soybean accessions from the USDA Soybean Germplasm Collection to identify <span class="gene">Gy4</span> mutants, the locus that controls A(5), A(4), and B(3) peptide production. Analysis of total <span class="trait">seed</span> proteins from 485 soybean lines by SDS-PAGE enabled the identification of 38 accessions that lacked the A(5), A(4), and B(3) peptides. These accessions showed marked differences in <span class="trait">seed</span> size and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> and represented different maturity groups ranging from 0 to IX. To ascertain the molecular basis for the lack of A(5), A(4), and B(3) peptides in the newly identified <span class="gene">Gy4</span> mutants, the nucleotide sequence of a portion of the <span class="gene">Gy4</span> gene was determined from eight soybean accessions representing different maturity groups. These eight <span class="gene">Gy4</span> mutants revealed a single point mutation that changed the translation initiation codon ATG to ATA, resulting in the A(5), A(4), and B(3) null phenotype. The newly identified <span class="gene">Gy4</span> mutants from this study will enable plant breeders to expand the genetic diversity of North American food-quality soybeans and also aid in the development of hypoallergenic soybeans.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structural features of <span class="gene">GmIRCHS,</span> candidate of the I gene inhibiting <span class="trait">seed</span> <span class="trait">coat</span> pigmentation in soybean: implications for inducing endogenous RNA silencing of chalcone synthase genes.</h2>
			<h3>PMID:17497082  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kasai A(1), Kasai K, Yumoto S, Senda M.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most commercial soybean varieties have yellow seeds due to loss of pigmentation in the <span class="trait">seed</span> <span class="trait">coat.</span> The I gene inhibits pigmentation over the entire <span class="trait">seed</span> <span class="trait">coat,</span> resulting in a uniform yellow <span class="trait">color</span> of mature harvested seeds. We previously demonstrated that the inhibition of <span class="trait">seed</span> <span class="trait">coat</span> pigmentation by the I gene results from post-transcriptional gene silencing (PTGS) of chalcone synthase (CHS) genes. Little is known about the structure of the I gene and the mechanism by which it induces PTGS of <span class="gene">CHS</span> genes. Here, we report a candidate of the I gene, <span class="gene">GmIRCHS,</span> which consists of a 5'-portion of a DnaJ-like gene containing a promoter region and a perfect inverted repeat (IR) of 1.1-kb truncated <span class="gene">CHS3</span> sequences (5'-DeltaCHS3 and 3'-DeltaCHS3). RT-PCRs and RNase protection assay indicated the existence of the read-through product from 5'-DeltaCHS3 to 3'-DeltaCHS3 and the dsRNA region of DeltaCHS3, suggesting that dsRNA of DeltaCHS3 could be transcribed from <span class="gene">GmIRCHS</span> and could induce PTGS of <span class="gene">CHS</span> genes. Moreover, the IR structure of DeltaCHS3 in <span class="gene">GmIRCHS</span> was lost in the soybean mutants in which I was changed to i, supporting the conclusion that <span class="gene">GmIRCHS</span> is the I gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning of the pleiotropic T locus in soybean and two recessive alleles that differentially affect structure and expression of the encoded flavonoid 3' hydroxylase.</h2>
			<h3>PMID:12586717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zabala G(1), Vodkin L.
				<br/> 
				年份：2003 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Three loci (I, R, and T) control pigmentation of the <span class="trait">seed</span> coats in Glycine max and are genetically distinct from those controlling flower <span class="trait">color.</span> The T locus also controls <span class="trait">color</span> of the trichome hairs. We report the identification and isolation of a flavonoid 3' hydroxylase gene from G. max <span class="gene">(GmF3'H)</span> and the linkage of this gene to the T locus. This <span class="gene">GmF3'H</span> gene was highly expressed in early stages of <span class="trait">seed</span> <span class="trait">coat</span> development and was expressed at very low levels or not at all in other tissues. Evidence that the <span class="gene">GmF3'H</span> gene is linked to the T locus came from the occurrence of multiple RFLPs in lines with varying alleles of the T locus, as well as in a population of plants segregating at that locus. <span class="gene">GmF3'H</span> genomic and cDNA sequence analysis of <span class="trait">color</span> mutant lines with varying t alleles revealed a frameshift mutation in one of the alleles. In another line derived from a mutable genetic stock, the abundance of the mRNAs for <span class="gene">GmF3'H</span> was dramatically reduced. Isolation of the <span class="gene">GmF3'H</span> gene and its identification as the T locus will enable investigation of the pleiotropic effects of the T locus on cell wall integrity and its involvement in the regulation of the multiple branches of the flavonoid pathway in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A physical map of a gene-dense region in soybean linkage group A2 near the black <span class="trait">seed</span> <span class="trait">coat</span> and Rhg (4) loci.</h2>
			<h3>PMID:12582695</h3>
				

				
			<p>
				作者：Lewers K(1), Heinz R, Beard H, Marek L, Matthews B.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( Glycine max L. Merrill) linkage group A2 contains a major <span class="trait">resistance</span> gene to the soybean cyst nematode ( Heterodera glycines Ichinohe) at the Rhg (4) locus near a gene encoding aspartokinase homoserine dehydrogenase (AK-HSDH) and also near the I locus affecting <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color.</span> To identify clones related to this region of the genome, we used a PCR assay using primers designed from a gene encoding <span class="gene">AK-HSDH</span> to screen approximately 40,000 clones from a bacterial artificial chromosome (BAC) library constructed from genomic DNA of the susceptible cv. Williams 82. The identified BACs were screened with a second PCR assay using primers designed from DNA sequence associated with the I locus to confirm the location of the BACs. Only BAC <span class="gene">Gm_ISb001_056_G02</span> (56G2) was positive for both assays. BAC 56G2 contains several genes previously associated with stress or defense response including genes with high sequence similarity to those encoding chalcone synthase, glucosyl-transferase, a heat-shock transcription factor, a membrane-associated salt-inducible protein, adenosyl homocysteinase, a protein kinase, and a G10-like protein. The map contributes to our understanding of the organization of the soybean genome and to the completion of a physical map of the soybean genome. In addition, the genes identified provide landmarks to identify BAC clones near the Rhg (4) locus in resistant soybean genomic libraries and provide a foundation for comparison of soybean cyst nematode resistant and -susceptible DNA sequences in this region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of the duplicated <span class="gene">CHS1</span> gene related to the suppression of the <span class="trait">seed</span> <span class="trait">coat</span> pigmentation in yellow soybeans.</h2>
			<h3>PMID:12582616</h3>
				

				
			<p>
				作者：Senda M(1), Jumonji A, Yumoto S, Ishikawa R, Harada T, Niizeki M, Akada S.
				<br/> 
				年份：2002 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean is controlled by the classically defined I ( Inhibitor) locus. The seeds of most commercial soybean varieties are yellow due to the presence of a dominant allele of the I locus ( I: yellow <span class="trait">seed</span> <span class="trait">coat,</span> or i(i) : pigmented hilum and yellow <span class="trait">seed</span> <span class="trait">coat),</span> which inhibits <span class="trait">seed</span> <span class="trait">coat</span> pigmentation. Analysis of spontaneous mutations from I (yellow <span class="trait">seed</span> <span class="trait">coat)</span> to i (pigmented <span class="trait">seed</span> <span class="trait">coat)</span> has shown that these mutations are correlated with the deletion of a duplicated chalcone synthase gene-1 ( CHS1) region. In the current study, we isolated the duplicated <span class="gene">CHS1</span> region from a soybean cultivar with a I/I genotype (cv Miyagi shirome) and determined its structure. The results showed that the duplicated <span class="gene">CHS1</span> contained intact regulatory and coding regions. We designated the duplicated <span class="gene">CHS1</span> as ICHS1. In the hypocotyls of Miyagi shirome, the cDNA derived from <span class="gene">ICHS1</span> mRNA was identified by reverse transcriptase-polymerase chain reaction (RT-PCR) analysis, whereas in the immature <span class="trait">seed</span> coats it was suggested that the amount of transcripts from <span class="gene">ICHS1</span> and/or another type of <span class="gene">CHS1</span> ( CHS1.1) was very low. Interestingly, in the Miyagi shirome genome with a I/I genotype, <span class="gene">ICHS1</span> was closely linked to the truncated CHS3, and sequence comparison showed that this cluster probably arose from the CHS1-CHS3 cluster by a 1.8-kb deletion event.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sequence divergence at chalcone synthase gene in pigmented <span class="trait">seed</span> <span class="trait">coat</span> soybean mutants of the Inhibitor locus.</h2>
			<h3>PMID:12441645  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Senda M(1), Kasai A, Yumoto S, Akada S, Ishikawa R, Harada T, Niizeki M.
				<br/> 
				年份：2002 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybeans is determined by the I (Inhibitor) locus. The dominant I allele inhibits <span class="trait">seed</span> <span class="trait">coat</span> pigmentation, and it has been suggested that there is a correlation between the inhibition of pigmentation by the I allele and chalcone synthase (CHS) gene silencing in the <span class="trait">seed</span> <span class="trait">coat.</span> Analysis of spontaneous mutations from I to i has shown that these mutations are closely related to the deletion of one of the <span class="gene">CHS</span> genes (designated ICHS1). In soybeans with the I/I genotype (cv. Miyagi shirome), a truncated form of the <span class="gene">CHS</span> gene (CHS3) is located in an inverse orientation 680 bp upstream of ICHS1, and it was previously suggested that the truncated CHS3- <span class="gene">ICHS1</span> cluster might be involved in <span class="gene">CHS</span> gene silencing in the <span class="trait">seed</span> <span class="trait">coat.</span> In the current study, the truncated CHS3- <span class="gene">ICHS1</span> cluster was compared with the corresponding region of pigmented <span class="trait">seed</span> <span class="trait">coat</span> mutants in which I had changed to i in Miyagi shirome and in the strain Karikei 584. In the Karikei 584 mutant, the truncated CHS3-ICHS1 cluster was retained and the sequence diverged at a point immediately upstream (32 bp) of this cluster. The sequences upstream of the points of divergence in both mutants almost perfectly matched a part of the registered sequence in a soybean BAC clone containing the soybean cyst nematode resistance-associated gene, and inspection of the sequences suggested that the sequence divergence of the <span class="gene">CHS</span> gene in the Karikei 584 and Miyagi shirome mutants was due to an unequal crossing-over via 4-bp or 5-bp short repeats, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A single-base deletion in soybean flavonoid 3'-hydroxylase gene is associated with gray pubescence <span class="trait">color.</span></h2>
			<h3>PMID:12175012  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Toda K(1), Yang D, Yamanaka N, Watanabe S, Harada K, Takahashi R.
				<br/> 
				年份：2002 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The T locus of soybean (Glycine max (L.) Merr.) controls pubescence and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> and is presumed to encode flavonoid 3'-hydroxylase (F3'H). The dominant T and the recessive t allele of the locus produce brown and gray pubescence, respectively. PCR primers were constructed based on the sequence of a soybean EST clone homologous to the <span class="gene">F3'H</span> gene. A putative full-length cDNA, <span class="gene">sf3'h1</span> was isolated by 3' and 5' RACE. Sequence analysis revealed that <span class="gene">sf3'h1</span> consists of 1690 nucleotides encoding 513 amino acids. It had 68% and 66% homology with corresponding <span class="gene">F3'H</span> protein sequences of petunia and Arabidopsis, respectively. A conserved amino acid sequence of <span class="gene">F3'H</span> proteins, GGEK, was found in the deduced polypeptide. Sequence analysis of the gene from a pair of near-isogenic lines for T, To7B (TT, brown) and To7G (tt, gray) revealed that they differed by a single C deletion in the coding region of To7G. The deletion changed the subsequent reading frame resulting in a truncated polypeptide lacking the GGEK consensus sequence and the heme-binding domain. Genomic Southern analysis probed by <span class="gene">sf3'h1</span> revealed restriction fragment length polymorphisms between cultivars with different pubescence <span class="trait">color.</span> Further, <span class="gene">sf3'h1</span> was mapped at the same position with T locus on LG3(c2). PCR-RFLP analysis was performed to detect the single-base deletion. To7B and three cultivars with brown pubescence exhibited shorter fragments, while To7G and three cultivars with gray pubescence had longer fragments due to the single-base deletion. The PCR-RFLP marker co-segregated with genotypes at the T locus in a F2 population segregating for the T locus. The above results strongly suggest that sJ3'h1 represents the T gene of soybean responsible for pubescence <span class="trait">color</span> and that the single-base deletion may be responsible for gray pubescence <span class="trait">color.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed <span class="trait">Coat</span> Cracking in Soybean Isolines for Pubescence <span class="trait">Color</span> and Maturity.</h2>
			<h3>PMID:11756256</h3>
				

				
			<p>
				作者：Yang D(1), Nakamura T, Ohtsubo N, Takahashi K, Abe J, Takahashi R.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> coats of soybean crack under various stress conditions. Cracking of <span class="trait">seed</span> coats degrades the external appearance of soybean seeds and reduces their commercial value. Previous studies revealed that the T gene responsible for pubescence <span class="trait">color,</span> and the maturity genes, <span class="gene">E1</span> and E5, had inhibitory effects on low-temperature induced <span class="trait">seed</span> <span class="trait">coat</span> cracking. The objective of this study was to evaluate the effects of the T gene and five maturity genes (E1 to E5) on the intensity of <span class="trait">seed</span> <span class="trait">coat</span> cracking induced by pod-removal treatments. Soybean cv. Harosoy (te1e2E3E4e5) and its near-isogenic lines for T and <span class="gene">E1</span> to E5 loci were used in the experiment. Cracking was induced by removing the upper 50% of pods on the plant 40 d after anthesis. Frequency and degree of cracking were not different among the isolines in the control group. In contrast, there were significant differences among isolines subjected to the pod-removal treatment. Frequency and degree of cracking was low in Harosoy, Harosoy-E1, e3, and e4, and high in Harosoy-T and E2. The results suggest that genotypes at T and E2 loci were associated with severity of <span class="trait">seed</span> <span class="trait">coat</span> cracking induced by pod-removal. There was a positive correlation (r = 0.90**) between individual <span class="trait">seed</span> weight and frequency of cracking among isolines in the pod-removal treatment. <span class="trait">Seed</span> <span class="trait">coat</span> cracking was probably exacerbated in part by the genes that allow enlargement of individual seeds in response to pod-removal. The differences among isolines suggest that the mechanism of <span class="trait">seed</span> <span class="trait">coat</span> cracking induced by pod removal may differ from that induced by low temperature treatment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A defective <span class="trait">seed</span> <span class="trait">coat</span> pattern (Net) is correlated with the post-transcriptional abundance of soluble proline-rich cell wall proteins.</h2>
			<h3>PMID:10480384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Percy JD(1), Philip R, Vodkin LO.
				<br/> 
				年份：1999 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The pigmented <span class="trait">seed</span> coats of several soybean (Glycine max (L.) Merr.) plant introductions and isolines have unusual defects that result in cracking of the mature <span class="trait">seed</span> <span class="trait">coat</span> exposing the endosperm and cotyledons. It has previously been shown that the T (tawny) locus that controls the <span class="trait">color</span> of trichomes on stems and leaves also has an effect on both the structure and pigmentation of the <span class="trait">seed</span> <span class="trait">coat.</span> Distribution of pigmentation on the <span class="trait">seed</span> <span class="trait">coat</span> is controlled by alleles of the I (inhibitor) locus. It was also found that total <span class="trait">seed</span> <span class="trait">coat</span> proteins were difficult to extract from pigmented <span class="trait">seed</span> coats with i T genotypes because they have procyanidins that exhibit tannin properties. We report that the inclusion of poly-L-proline in the extraction buffer out-competes proteins for binding to procyanidins. Once this problem was solved, we examined expression of the proline-rich cell wall proteins <span class="gene">PRP1</span> and <span class="gene">PRP2</span> in pigmented genotypes with the dominant T allele. We found that both homozygous i T and i t genotypes have reduced soluble <span class="gene">PRP1</span> levels. The epistatic interaction of the double recessive genotype at both loci is necessary to produce the pigmented, defective <span class="trait">seed</span> <span class="trait">coat</span> phenotype characteristic of <span class="trait">seed</span> coats with the double recessive i and t alleles. This implies a novel effect of an enzyme in the flavonoid pathway on <span class="trait">seed</span> <span class="trait">coat</span> structure in addition to its effect on flavonoids, anthocyanidins, and proanthocyanidins. No soluble <span class="gene">PRP1</span> polypeptides were detectable in pigmented <span class="trait">seed</span> coats (i T genotypes) of isolines that also display a net-like pattern of <span class="trait">seed</span> <span class="trait">coat</span> cracking, known as the Net defect. <span class="gene">PRP2</span> was also absent in one of the these lines. However, both <span class="gene">PRP1</span> and <span class="gene">PRP2</span> cytoplasmic mRNAs were found in the Net-defective <span class="trait">seed</span> coats. Together with in vitro translation studies, these results suggest that the absence of soluble PRP polypeptides in the defective Net lines is post-translational and could be due to a more rapid or premature insolubilization of PRP polypeptides within the cell wall matrix.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Duplications That Suppress and Deletions That Restore Expression from a Chalcone Synthase Multigene Family.</h2>
			<h3>PMID:12239396</h3>
				

				
			<p>
				作者：Todd JJ(1), Vodkin LO.
				<br/> 
				年份：1996 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean is determined by four alleles of the classically defined / (inhibitor) locus that controls the presence or absence as well as the spatial distribution of anthocyanin pigments in the <span class="trait">seed</span> <span class="trait">coat.</span> By analyzing spontaneous mutations of the / locus, we demonstrated that the / locus is a region of chalcone synthase (CHS) gene duplications. Paradoxically, deletions of <span class="gene">CHS</span> gene sequences allow higher levels of <span class="gene">CHS</span> mRNAs and restore pigmentation to the <span class="trait">seed</span> <span class="trait">coat.</span> The unusual nature of the / locus suggests that its dominant alleles may represent naturally occurring examples of homology-dependent gene silencing and that the spontaneous deletions erase the gene-silencing phenomena. Specifically, mutations from the dominant ii allele (yellow <span class="trait">seed</span> coats with pigmented hila) to the recessive i allele (fully pigmented) can be associated with the absence of a 2.3-kb Hindlll fragment that carries CHS4, a member of the multigene <span class="gene">CHS</span> family. Seven independent mutations exhibit deletions in the CHS4 promoter region. The dominant / allele (yellow <span class="trait">seed</span> coats) exhibits an extra 12.1-kb Hindlll fragment that hybridizes with both the <span class="gene">CHS</span> coding region and <span class="gene">CHS1</span> promoter-specific probes. Mutations of the dominant / allele to the recessive i allele (pigmented <span class="trait">seed</span> coats) give rise to 10.4- or 9.6-kb Hindlll <span class="gene">CHS</span> fragments that have lost the duplicated <span class="gene">CHS1</span> promoter. Finally, gene expression analysis demonstrated that heterozygous plants (I/i) with yellow <span class="trait">seed</span> coats have reduced mRNA levels, indicating that the 12.1-kb Hindlll <span class="gene">CHS</span> fragment associated with the dominant / allele inhibits pigmentation in a trans-dominant manner. Moreover, <span class="gene">CHS</span> gene-specific expression in <span class="trait">seed</span> coats shows that multiple <span class="gene">CHS</span> genes are expressed in <span class="trait">seed</span> coats.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation of proline rich cell wall proteins in soybean lines with anthocyanin mutations.</h2>
			<h3>PMID:8425044  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nicholas CD(1), Lindstrom JT, Vodkin LO.
				<br/> 
				年份：1993 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The I locus controls inhibition of anthocyanin accumulation in the epidermal cells of the soybean <span class="trait">seed</span> <span class="trait">coat</span> and affects abundance of PRP1, a proline-rich cell wall protein in the <span class="trait">seed</span> <span class="trait">coat.</span> Saline-soluble <span class="gene">PRP1</span> is abundant in the developing <span class="trait">seed</span> coats of cultivar Richland (homozygous I, yellow), while it is significantly decreased in the pigmented isogenic mutant T157 (homozygous i, imperfect black). In this report, we examined soluble <span class="gene">PRP1</span> in several cultivars containing alleles of the I locus which affect spatial distribution of pigmentation in the <span class="trait">seed</span> <span class="trait">coat.</span> We also characterized <span class="gene">PRP1</span> in isolines with allelic variants of several other loci involved in <span class="trait">seed</span> <span class="trait">coat</span> pigmentation, including T and Im. The T gene is pleiotropic and affects both pubesence <span class="trait">color</span> and <span class="trait">seed</span> <span class="trait">coat</span> pigmentation and structure. Soluble <span class="gene">PRP1</span> was abundant in the developing <span class="trait">seed</span> coats of lines with yellow <span class="trait">seed</span> (I or ii alleles) regardless of pubescence <span class="trait">color,</span> just as in Richland. Likewise, soluble <span class="gene">PRP1</span> was decreased in pigmented <span class="trait">seed</span> coats (ik or i alleles) with grey (t) pubescence, as in T157. However, the total <span class="trait">seed</span> <span class="trait">coat</span> proteins were not extractable from pigmented <span class="trait">seed</span> coats with tawny pubescence (i, T genotypes) because they have proanthocyanidins that exhibit tannin properties. The dominant Im allele inhibits <span class="trait">seed</span> <span class="trait">coat</span> mottling (irregular patches of pigmentation) that occurs if plants are infected with soybean mosaic virus. <span class="gene">PRP1</span> was 35 kDa in mottled (im) isolines and 34 kDa in non-mottled (Im) isolines. PRP2, which is expressed later in <span class="trait">seed</span> <span class="trait">coat</span> development and in the hypocotyl hooks of soybean seedlings, was also smaller in Im isolines. In summary, some of the anthocyanin mutations affect the quantity of soluble <span class="gene">PRP1</span> polypeptides. while others correlate with structural changes in developmentally regulated proline-rich proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean cell wall protein is affected by <span class="trait">seed</span> <span class="trait">color</span> genotype.</h2>
			<h3>PMID:1841717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lindstrom JT(1), Vodkin LO.
				<br/> 
				年份：1991 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The dominant I gene inhibits accumulation of anthocyanin pigments in epidermal cells of the soybean <span class="trait">seed</span> <span class="trait">coat.</span> We compared saline-soluble proteins extracted from developing <span class="trait">seed</span> coats and identified a 35-kilodalton protein that was abundant in Richland (genotype I/I, yellow) and much reduced in an isogenic mutant line T157 (genotype i/i, imperfect black <span class="trait">seed</span> coats). We purified the 35-kilodalton protein by a novel procedure using chromatography on insoluble polyvinylpolypyrrolidone. The 35-kilodalton protein was composed primarily of proline, hydroxyproline, valine, tyrosine, and lysine. Three criteria (N-terminal amino acid sequence, amino acid composition, and sequence of a cDNA) proved that the <span class="trait">seed</span> <span class="trait">coat</span> 35-kilodalton protein was PRP1, a member of a proline-rich gene family expressed in hypocotyls and other soybean tissues. The levels of soluble <span class="gene">PRP1</span> polypeptides and <span class="gene">PRP1</span> mRNA were reduced in young <span class="trait">seed</span> coats with the recessive i/i genotype. These data demonstrated an unexpected and novel correlation between an anthocyanin gene and the quantitative levels of a specific, developmentally regulated cell wall protein. In contrast, PRP2, a closely related cell wall protein, was synthesized later in <span class="trait">seed</span> <span class="trait">coat</span> development and was not affected by the genotype of the I locus.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Inheritance and Organization of Glycinin Genes in Soybean.</h2>
			<h3>PMID:12359892</h3>
				

				
			<p>
				作者：Cho TJ(1), Davies CS, Nielsen NC.
				<br/> 
				年份：1989 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Five genes (Gy1, through Gy5) encode most of the subunits that are assembled into glycinin, a predominant <span class="trait">seed</span> storage protein found in soybeans. Restriction fragment length polymorphisms are described that identify four of these five genes (Gy1/Gy2, Gy3, and Gy5). The fifth gene (Gy4) is characterized by two alleles, one of which (gy4) causes absence of the subunit. Genetic segregation studies indicate that the five genes are located at four genetic loci within the genome. <span class="gene">Gy1</span> and <span class="gene">Gy2</span> are in a direct tandem repeat at one locus, whereas there is a single glycinin gene at each of the other three loci. All four loci segregate independently from one another, and they also segregate independently from the genetic markers for tawny/grey pubescence (T/t), purple/white flower <span class="trait">color</span> (W1/w1), light/dark hilum pigmentation (l/ll), black/brown <span class="trait">seed</span> <span class="trait">coat</span> (R/r), and brown/tan pod <span class="trait">color</span> (I1I1L2L2/I1I1I2I2). The latter genetic markers are located on linkage groups 1 (t), 8 (w1), 7 (i), and 2 (r) in the soybean genome, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Transition from Primary siRNAs to Amplified Secondary siRNAs That Regulate Chalcone Synthase During Development of <italic> Glycine max </italic> <span class="trait">Seed</span> Coats</h2>
			<h3>PMID:24204712</h3>
				
					<h3>PMCID:3804491</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cho </surname>  <given-names> Young B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jones </surname>  <given-names> Sarah I. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> I </italic> locus is a 27-kb inverted repeat cluster of chalcone synthase genes <italic> CHS1-3-4 </italic> that mediates siRNA down-regulation of <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> target mRNAs during <span class="trait">seed</span> development leading to yellow <span class="trait">seed</span> coats lacking anthocyanin pigments. Here, we report small RNA sequencing of ten stages of <span class="trait">seed</span> development from a few days post fertilization through maturity, revealing the amplification from primary to secondary short interfering RNAs (siRNAs) occurring during development. The young <span class="trait">seed</span> populations had a higher proportion of siRNAs representing the <italic> CHS1-3-4 </italic> gene family members, consistent with this region as the origin of the primary siRNAs. More intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did the mid-maturation <span class="trait">seed.</span> We infer that the primary <italic> <span class="gene">CHS</span> </italic> siRNAs increase during development to levels sufficient to trigger amplification of secondary <italic> <span class="gene">CHS</span> </italic> siRNAs from the <italic> <span class="gene">CHS7</span> </italic> / <italic> 8 </italic> target mRNAs, enabling the total levels of 21-nt <italic> <span class="gene">CHS</span> </italic> siRNAs to rise dramatically. Further, we demonstrate that the soybean system exhibits tissue-specific <italic> <span class="gene">CHS</span> </italic> siRNA production because primary <italic> <span class="gene">CHS</span> </italic> siRNA levels are not sufficient to trigger secondary amplification in tissues other than the <span class="trait">seed</span> <span class="trait">coat.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> I </italic> locus was first identified as a region of duplicated and inverted <italic> <span class="gene">CHS</span> </italic> genes by DNA blotting and PCR mapping of 15 pairs of isogenic lines which showed that mutations of a dominant allele ( <italic> I </italic> or <italic> i <sup> i </sup> </italic> ) to the recessive allele ( <italic> i </italic> ) delete promoter sequences, paradoxically resulting in increased total <italic> <span class="gene">CHS</span> </italic> transcript levels and thereby pigmented (black) <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone.0076954-Todd1"> [1] </xref> . The inhibitory effects of this naturally occurring mutation appeared to parallel the co-suppressive phenotypes found in transgenic <italic> Petunia </italic> lines carrying multiple inserted copies of the <italic> <span class="gene">CHS</span> </italic> genes <xref ref-type="bibr" rid="pone.0076954-Napoli1"> [5] </xref> , <xref ref-type="bibr" rid="pone.0076954-VanderKrol1"> [6] </xref> . Sequencing bacterial artificial chromosomes (BACs) from the Williams soybean variety containing the dominant <italic> i <sup> i </sup> </italic> allele revealed that five ( <italic> CHS1, CHS3, CHS4, CHS5, </italic> and <italic> <span class="gene">CHS9</span> </italic> ) of the nine <italic> <span class="gene">CHS</span> </italic> gene family members are located in a 230-kb region <xref ref-type="bibr" rid="pone.0076954-Clough1"> [7] </xref> , <xref ref-type="bibr" rid="pone.0076954-Tuteja3"> [8] </xref> . <italic> CHS1, <span class="gene">CHS3</span> </italic> , and <italic> CHS4 </italic> are located in two 10.91-kb perfect and inverted repeat clusters separated by 5.87 kb of intervening sequence.
					</p>
				
					<p>&nbsp;&nbsp;
						The relative expression profiles of <italic> <span class="gene">CHS</span> </italic> gene family members were examined by quantitative RT-PCR in the <span class="trait">seed</span> coats of two near-isogenic pairs that result from independently occurring mutations of the dominant <italic> I </italic> allele to the recessive <italic> i </italic> allele or of the dominant <italic> i <sup> i </sup> </italic> allele to the recessive <italic> i </italic> allele <xref ref-type="bibr" rid="pone.0076954-Tuteja1"> [3] </xref> . Decreased expression of the <italic> CHS7/8 </italic> genes results in the lack of pigmentation in the yellow <span class="trait">seed</span> coats ( <italic> I </italic> and <italic> i <sup> i </sup> </italic> ). Both small RNA blots and high throughput small RNA sequencing from three genotypes ( <italic> I, i <sup> i </sup> </italic> and <italic> i </italic> alleles) of soybeans revealed that <italic> <span class="gene">CHS</span> </italic> siRNAs accumulated only in the yellow <span class="trait">seed</span> coats containing either dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and not in the pigmented <span class="trait">seed</span> coats with homozygous recessive <italic> i </italic> genotypes <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> . Interestingly, the <italic> <span class="gene">CHS</span> </italic> siRNAs were generated in a tissue-specific manner. <italic> <span class="gene">CHS</span> </italic> siRNAs did not accumulate in the cotyledons of the genotype with dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and yellow <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> , thus allowing isoflavones to be produced in the cotyledons <xref ref-type="bibr" rid="pone.0076954-Dhaubhadel1"> [9] </xref> since <italic> CHS7/8 </italic> mRNAs are not down-regulated in the cotyledons.
					</p>
				
					<p>&nbsp;&nbsp;
						The generation pathway of <italic> <span class="gene">CHS</span> </italic> siRNAs involved in silencing the <italic> I </italic> locus was proposed <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> . A putative double stranded (ds) RNA generated from within a 27-kb inverted <italic> <span class="gene">CHS</span> </italic> region comprised of two clusters of the <italic> CHS1-3-4 </italic> and <italic> CHS4-3-1 </italic> genes is cleaved into primary siRNAs representing both strands that are amplified by RNA-dependent RNA polymerase (RdRP) to generate secondary <italic> <span class="gene">CHS</span> </italic> siRNAs from the target <italic> CHS7/8 </italic> mRNAs, which are capable of down-regulating all members of the <italic> <span class="gene">CHS</span> </italic> gene family. On the other hand, <italic> CHS7/8 </italic> mRNAs are highly expressed in the pigmented <span class="trait">seed</span> coats in which <italic> <span class="gene">CHS</span> </italic> siRNA production has been abolished by deletions in the <italic> <span class="gene">CHS</span> </italic> cluster regions in the mutant <italic> i </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Small RNA sequencing of co-suppressed, non-pigmented transgenic petunia flowers with introduced <italic> <span class="gene">CHS</span> </italic> genes has shown that <italic> <span class="gene">CHS</span> </italic> siRNAs are the causative factor of silencing the pigment pathway in flowers <xref ref-type="bibr" rid="pone.0076954-DePaoli1"> [10] </xref> . There are many commonalities between the naturally occurring soybean <span class="trait">seed</span> <span class="trait">coat</span> and the transgenic petunia systems <xref ref-type="bibr" rid="pone.0076954-Eckardt1"> [11] </xref> . These similarities include <italic> <span class="gene">CHS</span> </italic> siRNAs that are predominantly 21 nt in size and match both sense and antisense strands primarily of exon 2 of the conserved target <italic> <span class="gene">CHS</span> </italic> genes, likely through amplification of the original signal through RdRP.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we sequenced small RNA populations from a developmental series representing ten stages of <span class="trait">seed</span> development from a few days post fertilization through <span class="trait">seed</span> maturity. Due to the sequence polymorphisms between the <italic> <span class="gene">CHS</span> </italic> siRNA origin and target genes, we were able to reveal the amplification from primary to secondary <italic> <span class="gene">CHS</span> </italic> siRNAs that occurred early in development. The primary <italic> <span class="gene">CHS</span> </italic> siRNAs were lower in abundance but composed of a higher proportion of 22-nt small RNAs, whereas the more abundant secondary <italic> <span class="gene">CHS</span> </italic> siRNAs were primarily 21 nt in size. A developmental amplification from primary to secondary siRNAs has not previously been tracked in either naturally occurring or transgenic plant systems. We also demonstrate that <italic> <span class="gene">CHS</span> </italic> siRNA production is specific to the <span class="trait">seed</span> coats as other tissues including cotyledons, roots, leaves, and stems do not produce sufficient primary siRNAs for amplification to secondary siRNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						Although <italic> CHS1, 2, 3, 4, 5, 6 </italic> and <italic> 9 </italic> share 93% to 98% pairwise identity by nucleotide sequence, they are only 82% similar to <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> . The dispersed nucleotide polymorphisms between these two groups of <italic> <span class="gene">CHS</span> </italic> genes present the opportunity to distinguish siRNAs representing the <italic> CHS7/8 </italic> mRNAs that are down-regulated <italic> in trans </italic> by <italic> CHS1/3/4 </italic> primary siRNAs originating from the <italic> I </italic> locus, which is defined by inverted repeats of <italic> CHS1, <span class="gene">CHS3</span> </italic> and <italic> CHS4 </italic> genes (referred to as the <italic> CHS1-3-4 </italic> clusters). To determine the pattern of <italic> <span class="gene">CHS</span> </italic> siRNAs during the entire span of <span class="trait">seed</span> development, we constructed small RNA libraries from ten stages of <span class="trait">seed</span> development of the cultivar Williams ( <italic> i <sup> i </sup> </italic> ) as shown in <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> . In the early developmental periods from 4 DAF (days after flowering) through 22&#x02013;24 DAF, whole seeds were used as the maternal <span class="trait">seed</span> <span class="trait">coat</span> is proportionally a larger part of the whole <span class="trait">seed,</span> which is too small for hand dissection. Beginning at the 5&#x02013;6 mg weight range, the developmental staging is more accurate using a combination of <span class="trait">seed</span> fresh weight and <span class="trait">color</span> changes. At that time the <span class="trait">seed</span> are large enough to dissect the <span class="trait">seed</span> <span class="trait">coat</span> free of the cotyledons and embryonic axis. <xref ref-type="table" rid="pone-0076954-t001"> Table 1 </xref> lists details on the small RNA sequencing, ranging generally from 10 to 30 million reads from each of the ten developmental stages. Two biological repeats were made for eight of the ten developmental stages.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart. RPM, reads per million. Numbers above the <span class="gene">bar</span> indicate the total counts of <italic> CHS4 </italic> (red) and <italic> <span class="gene">CHS7</span> </italic> (blue) siRNAs. Developmental stages are whole <span class="trait">seed</span> from (a) 4 DAF (Days After Flowering); (b) 12&#x02013;14 DAF; and (c) 22&#x02013;24 DAF; <span class="trait">seed</span> coats dissected from immature green <span class="trait">seed</span> of fresh weight (d) 5&#x02013;6 mg; (e) 10&#x02013;25 mg; (f) 25&#x02013;50 mg; (g) 50&#x02013;75 mg; (h) 75&#x02013;100 mg; (i) 200&#x02013;300 mg; and <span class="trait">seed</span> coats from (j) 300&#x02013;400 mg yellow, desiccating <span class="trait">seed.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> shows the normalized total counts in reads per million (RPM) of <italic> <span class="gene">CHS</span> </italic> siRNAs that map with 100% identity to each of the nine <italic> <span class="gene">CHS</span> </italic> genes during <span class="trait">seed</span> development. The normalized <italic> <span class="gene">CHS</span> </italic> siRNA counts were initially low in the 4 DAF <span class="trait">seed</span> at less than 100 RPM and also in the 12&#x02013;14 DAF and 22&#x02013;24 DAF <span class="trait">seed.</span> Then, they increased to about 900 RPM in <span class="trait">seed</span> coats from immature <span class="trait">seed</span> of 5&#x02013;6 mg fresh weight, after which they increased dramatically, reaching the highest levels of 8600 for <italic> <span class="gene">CHS7</span> </italic> in <span class="trait">seed</span> coats from 50&#x02013;75 mg immature <span class="trait">seed.</span> After this peak, the <italic> <span class="gene">CHS</span> </italic> siRNAs decreased as the <span class="trait">seed</span> increased in size and began desiccation. An analysis of <italic> <span class="gene">CHS</span> </italic> siRNAs from a biological repeat ( <xref ref-type="supplementary-material" rid="pone.0076954.s001"> Figure S1 </xref> ) of eight of the ten stages shows the same pattern as <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> with maximal <italic> <span class="gene">CHS</span> </italic> siRNAs found in the 50&#x02013;75 mg stage.
					</p>
				
					<p>&nbsp;&nbsp;
						Strikingly, the expression pattern of <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> and <xref ref-type="supplementary-material" rid="pone.0076954.s001"> Figure S1 </xref> enables us to distinguish the primary siRNAs originating from the <italic> CHS1-3-4 </italic> clusters of the <italic> I </italic> locus from those representing secondary siRNAs matching target mRNAs encoded by <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> . The secondary <italic> CHS7/8 </italic> siRNAs showed an increase in the 5&#x02013;6 mg <span class="trait">seed</span> coats and peaked at the 50&#x02013;75 mg stages. The predominance of secondary <italic> CHS7/8 </italic> siRNAs remained during the rest of development in <span class="trait">seed</span> coats dissected from the immature green <span class="trait">seed</span> of 75&#x02013;100 mg and 200&#x02013;300 mg, as well as from 300&#x02013;400 mg yellow <span class="trait">seed</span> coats that were undergoing desiccation. These data demonstrate that the amplification from primary to secondary <italic> <span class="gene">CHS</span> </italic> siRNAs occurs during <span class="trait">seed</span> <span class="trait">coat</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of the same ten developmental stages A-J as described in <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> were analyzed for their size distributions between 18 and 25 nt. The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						A comparison of total counts of 21-nt and 22-nt primary <italic> CHS4 </italic> siRNAs and 21-nt secondary <italic> <span class="gene">CHS7</span> </italic> siRNAs that are unique to either <italic> CHS4 </italic> or <italic> <span class="gene">CHS7</span> </italic> shows that 21-nt secondary siRNAs were highly amplified during <span class="trait">seed</span> <span class="trait">coat</span> development compared to those of primary <italic> <span class="gene">CHS</span> </italic> siRNAs ( <xref ref-type="fig" rid="pone-0076954-g003"> Figure 3 </xref> and <xref ref-type="supplementary-material" rid="pone.0076954.s003"> Figure S3 </xref> ). RdRPs are involved in RNA amplification of primary siRNAs and generate secondary siRNAs <xref ref-type="bibr" rid="pone.0076954-Chapman1"> [12] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0076954-Ghildiyal1"> [14] </xref> . In the soybean <span class="trait">seed</span> <span class="trait">coat,</span> we demonstrate that this amplification started at an early developmental stage around 5&#x02013;6 mg ( <xref ref-type="fig" rid="pone-0076954-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> ) likely at the stage when <italic> <span class="gene">CHS7</span> </italic> / <italic> 8 </italic> mRNAs begin to be expressed in the <span class="trait">seed</span> coats at higher levels. The <italic> CHS7/8 </italic> mRNAs are then susceptible to degradation by primary siRNAs, and amplification is guided by the primary <italic> CHS1/3/4 </italic> siRNAs that are produced initially from the triggering clusters of the <italic> CHS1-3-4 </italic> genes of the dominant <italic> i <sup> i </sup> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Using mRNA blots and small RNA blots, we have previously shown that silencing of <italic> <span class="gene">CHS</span> </italic> gene family members occurs only in the <span class="trait">seed</span> coats and not in other organs <xref ref-type="bibr" rid="pone.0076954-Tuteja1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> . Using sequencing, small RNAs were detected in significant numbers only in the dissected <span class="trait">seed</span> coats and not the cotyledons of the developing <span class="trait">seed</span> <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> from the 50&#x02013;75 mg weight range. Here, we investigated additional tissues such as leaf, root and germinating cotyledon with the power of deep sequencing to ascertain the tissue specificity of <italic> <span class="gene">CHS</span> </italic> siRNAs. As shown in <xref ref-type="table" rid="pone-0076954-t002"> Table 2 </xref> , total numbers of small RNA reads were from three million to thirty million from each of the tissues from Williams cultivar or other cultivars with <italic> i <sup> i </sup> </italic> genotype, providing sufficient data to show the tissue specificity of <italic> <span class="gene">CHS</span> </italic> siRNAs and to examine even low levels of <italic> <span class="gene">CHS</span> </italic> siRNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> demonstrates the tissue specificity clearly as <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> coats in the Williams <italic> i <sup> i </sup> </italic> cultivar. The normalized total counts of <italic> <span class="gene">CHS7</span> </italic> siRNAs in <span class="trait">seed</span> coats from 5&#x02013;6 mg is 925 (RPM). In contrast, the expression levels of <italic> <span class="gene">CHS7</span> </italic> siRNAs were less than 60 counts in four other tissues which is approximately a 15-fold difference ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ). Most of those tissues (immature cotyledon, germinated cotyledon, unifoliate, root, shoot tip, and stem) have more <italic> CHS4 </italic> siRNAs than <italic> <span class="gene">CHS7</span> </italic> siRNAs; however the counts of <italic> CHS4 </italic> siRNAs are very low compared to those of <span class="trait">seed</span> <span class="trait">coat</span> tissues ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ). These data show that both primary and secondary <italic> <span class="gene">CHS</span> </italic> siRNAs were not significantly produced in germinated cotyledon, immature cotyledon, leaf, root, shoot tip, and stem tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> tissue is close to the cotyledon physically during <span class="trait">seed</span> development. It is interesting that <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> <span class="trait">coat</span> and were not in immature cotyledons of three different stages of development, 5&#x02013;6 mg, 50&#x02013;75 mg and 100&#x02013;300 mg fresh <span class="trait">seed</span> weight. The immature cotyledons contained very few <italic> <span class="gene">CHS</span> </italic> siRNAs in these three developmental stages ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ). Because of the small size of the <span class="trait">seed</span> prior to the 5&#x02013;6 mg stage, we used whole <span class="trait">seed</span> through 24 DAF; thus, it is likely also that the low levels of <italic> <span class="gene">CHS</span> </italic> siRNAs from these young whole <span class="trait">seed</span> less than 24 DAF ( <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> ) originated solely from the maternal <span class="trait">seed</span> <span class="trait">coat</span> tissue and not from the very young developing cotyledons since the cotyledons at all succeeding stages did not accumulate <italic> <span class="gene">CHS</span> </italic> siRNAs at significant levels as shown in <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						We have previously presented evidence for a model of action for the naturally occurring dominant <italic> I </italic> and <italic> i <sup> i </sup> </italic> alleles in preventing pigment formation in soybean <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> based on knowledge of RNA interference mechanisms in other organisms <xref ref-type="bibr" rid="pone.0076954-Chapman1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0076954-Carthew1"> [13] </xref> . <xref ref-type="fig" rid="pone-0076954-g005"> Figure 5 </xref> summarizes the model and illustrates our data from this report on the developmental transition from primary to secondary siRNAs. The long inverted repeat of the <italic> I </italic> locus on Chromosome 8 contains cluster A ( <italic> CHS1-3-4 </italic> ) and cluster B ( <italic> CHS4-3-1 </italic> ) and forms the nascent <italic> <span class="gene">CHS</span> </italic> dsRNA although the exact size is unknown. The cleavage of <italic> <span class="gene">CHS</span> </italic> dsRNA by Dicer Like proteins (DCL) generates the primary <italic> <span class="gene">CHS</span> </italic> siRNAs, resulting in the degradation <italic> in trans </italic> of targeted <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> transcripts that originate from Chromosomes 1 and 11, respectively. After cleavage at the mRNA sites targeted by the primary <italic> <span class="gene">CHS</span> </italic> siRNAs, an RdRP synthesizes dsRNA from the cleaved <italic> <span class="gene">CHS</span> </italic> mRNA. The secondary <italic> <span class="gene">CHS</span> </italic> siRNAs generated from this dsRNA could target additional <italic> <span class="gene">CHS</span> </italic> mRNAs, amplifying the silencing response as well as spreading it over a larger region of the target. The roles for putative DCL and RdRP like functions in soybean are based on extrapolation from other systems and not by analysis of mutants of these functions in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Partially adapted from <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> . The long inverted repeat of the <italic> i <sup> i </sup> </italic> allele of the <italic> I </italic> locus on Chromosome 8 contains cluster A ( <italic> CHS1-3-4 </italic> ) and cluster B ( <italic> CHS4-3-1 </italic> ) and forms the nascent <italic> <span class="gene">CHS</span> </italic> dsRNA although the exact size is unknown. The cleavage of <italic> <span class="gene">CHS</span> </italic> dsRNA by putative Dicer Like proteins (DCL) generates predominantly 22-nt primary <italic> <span class="gene">CHS</span> </italic> siRNAs at the earliest developmental stages, resulting in the degradation <italic> in trans </italic> of targeted transcripts from <italic> <span class="gene">CHS7</span> </italic> <span class="gene">(Glyma01g43880)</span> and <italic> CHS8 </italic> <span class="gene">(Glyma11g01350)</span> located on Chromosomes 1 and 11, respectively. After cleavage at the mRNA sites targeted by the primary <italic> <span class="gene">CHS</span> </italic> siRNAs, a putative RdRP synthesizes dsRNA from the cleaved <italic> <span class="gene">CHS</span> </italic> mRNA. The 21-nt secondary <italic> <span class="gene">CHS</span> </italic> siRNAs likely generated by DCL4 homologs from this dsRNA can target additional <italic> <span class="gene">CHS</span> </italic> mRNAs, thus amplifying the silencing response as well as spreading it over a larger region of the target. The transition from predominantly 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in blue) representing the origin of the <italic> CHS1-3-4 </italic> siRNAs to the 21-nt secondary <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in red) representing the target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs occurs at the 5&#x02013;6 mg <span class="trait">seed</span> stage as indicated.
					</p>
				
					<p>&nbsp;&nbsp;
						In this report, we show that the unique attributes of the naturally occurring soybean <italic> I </italic> locus clearly revealed the amplification of the <italic> <span class="gene">CHS</span> </italic> siRNA population from primary to secondary siRNAs during <span class="trait">seed</span> development based on the polymorphisms between the siRNAs representing the origin ( <italic> CHS1-3-4 </italic> ) and target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes. <xref ref-type="supplementary-material" rid="pone.0076954.s005"> Table S1 </xref> shows that very few siRNAs match both the origin and target subgroups with 100% identity, verifying that the observed patterns ( <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> , <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> , <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> and Figures S1, S2, S3) are not due to multi-matching siRNA. The polymorphisms between the two sub-groups are widely dispersed throughout the sequence. For example, although <italic> CHS4 </italic> and <italic> <span class="gene">CHS7</span> </italic> are 82% identical <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> , there are no siRNAs which are longer than 20 nt matching both <italic> CHS4 </italic> and <italic> <span class="gene">CHS7</span> </italic> at 100% identity as shown in the alignment of <xref ref-type="supplementary-material" rid="pone.0076954.s004"> Figure S4 </xref> . Allowing up to two-base mismatches between the origin and target increases the number of siRNA sequences significantly ( <xref ref-type="supplementary-material" rid="pone.0076954.s005"> Table S1 </xref> ). Thus, the targeting of <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> by the <italic> CHS1-3-4 </italic> primary siRNAs is likely primarily through mismatching rather than perfectly matched siRNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						A developmental transition from primary to secondary siRNAs has not previously been elucidated in either naturally occurring or transgenic plant systems. In <italic> C. elegans </italic> , deep sequencing distinguished primary and secondary siRNAs using an exogenous RNAi construct containing mismatches from the wild type <italic> C. elegans </italic> target sequence positioned at 25 base intervals <xref ref-type="bibr" rid="pone.0076954-Pak1"> [15] </xref> . Here, by examining the entire <span class="trait">seed</span> developmental period including the very early stages from a few days post fertilization that are labor intensive to obtain in sufficient quantity for sequencing, we demonstrated the transition from a small population of primary <italic> <span class="gene">CHS</span> </italic> siRNAs to the accumulation of high levels of 21-nt secondary siRNAs ( <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> , <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> , <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> and Figures S1, S2, S3). The siRNA populations in the young <span class="trait">seed</span> had a higher proportion representing the <italic> CHS1-3-4 </italic> gene family members, consistent with this region as the origin of the primary siRNAs. <span class="trait">Seed</span> at 4 DAF contained a basal level of less than 100 RPM that aligned to any <italic> <span class="gene">CHS</span> </italic> gene. The levels of <italic> <span class="gene">CHS</span> </italic> siRNAs originating from the <italic> CHS1-3-4 </italic> clusters increased thereafter to roughly 400&#x02013;900 RPM very early in <span class="trait">seed</span> development from 12&#x02013;24 DAF to 5&#x02013;6 mg. By the 5&#x02013;6 mg stage, the <span class="trait">seed</span> coats contained roughly even proportions of siRNAs representing <italic> CHS1-3-4 </italic> compared to <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> but after that period, the <italic> CHS7/8 </italic> siRNAs increased dramatically, reaching peak levels of 8,000 to 10,000 RPMs in the biological repeats of the 50&#x02013;75 mg <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						Intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did <span class="trait">seed</span> from the 5&#x02013;6 mg stage and older. Recent studies in <italic> Arabidopsis thaliana </italic> <xref ref-type="bibr" rid="pone.0076954-Chen1"> [16] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0076954-Yoshikawa1"> [18] </xref> demonstrated that 22-nt miRNAs (microRNAs) are critical for biogenesis of secondary 21-nt siRNAs as they drive amplification of secondary siRNAs via RdRP to generate ta-siRNAs (transacting siRNAs). Our data support the conclusions that the primary siRNA population has a higher proportion of 22-nt siRNAs, and that the secondary population amplified from the <italic> CHS7/8 </italic> substrates by RdRP are predominantly 21 nt in size. However, another study recently claimed that the miRNA-duplex structure is critical to the production of secondary ta-siRNAs rather than the length of miRNA <xref ref-type="bibr" rid="pone.0076954-Manavella1"> [19] </xref> . Whether or not the 22-nt siRNAs are the only trigger to generate the 21-nt secondary siRNAs is unknown, since 21-nt primary <italic> <span class="gene">CHS</span> </italic> siRNAs were also present in the very young <span class="trait">seed.</span> The predominance of the 21-nt secondary siRNAs that represent <italic> CHS7/8 </italic> was maintained as the total levels of <italic> <span class="gene">CHS</span> </italic> siRNAs declined during the later stages of maturity.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs were found to be most abundant in the 50&#x02013;75 mg stage among ten developmental stages ( <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> and <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> ). Previously, it was shown with RNA blots that <italic> <span class="gene">CHS</span> </italic> mRNAs are prevalent at 10&#x02013;25 mg, 25&#x02013;50 mg and 50&#x02013;75 mg weight ranges of the <span class="trait">seed</span> coats of mid-maturation embryos with pigmented genotypes <xref ref-type="bibr" rid="pone.0076954-Tuteja1"> [3] </xref> . Also, using TaqMan real time RT-PCR, it was shown that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs are highest at the 50&#x02013;75 mg stage of Williams 55 pigmented <span class="trait">seed</span> coats with the homozygous recessive <italic> i </italic> genotype. In this regard, the profile of the <italic> <span class="gene">CHS</span> </italic> siRNAs resembled many other genes expressed during <span class="trait">seed</span> <span class="trait">coat</span> development, including many in the isoflavone and anthocyanin pathways <xref ref-type="bibr" rid="pone.0076954-Wang1"> [20] </xref> . The most abundant levels of <italic> <span class="gene">CHS</span> </italic> siRNAs in the <italic> i <sup> i </sup> </italic> genotype with yellow <span class="trait">seed</span> coats generally coincided with the developmental time of appearance of the <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> target mRNAs in the recessive <italic> i </italic> genotype that has pigmented <span class="trait">seed</span> coats. Thus, in contrast with the down-regulation of the pathway by <italic> <span class="gene">CHS</span> </italic> siRNA-targeted destruction of <italic> <span class="gene">CHS</span> </italic> mRNAs in the yellow <span class="trait">seed</span> coats, the <italic> <span class="gene">CHS</span> </italic> transcripts continued to produce <italic> <span class="gene">CHS</span> </italic> isoforms after the 50 mg stage with the resulting accumulation of large amounts of anthocyanins leading to the pigmented <span class="trait">seed</span> <span class="trait">coat.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						By examining the entire <span class="trait">seed</span> developmental period including the small <span class="trait">seed</span> at 4 DAF through 24 DAF, we found that the <italic> <span class="gene">CHS</span> </italic> primary siRNAs representing the <italic> CHS1-3-4 </italic> origin clusters began to increase before the secondary siRNAs representing the <italic> CHS7/8 </italic> target mRNAs (summarized in <xref ref-type="fig" rid="pone-0076954-g005"> Figure 5 </xref> ). These levels were higher than the basal level for <italic> <span class="gene">CHS</span> </italic> siRNAs found in non-seed tissues ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ). From these results, we deduce that there was active biogenesis of the nascent <italic> <span class="gene">CHS</span> </italic> dsRNA that began to occur early in development between 12&#x02013;24 DAF. It is not yet determined how early the <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs begin to be expressed in the non-silencing, pigmented mutant line Williams ( <italic> i </italic> , black). Although RNA-Seq data is available from the Williams ( <italic> i <sup> i </sup> </italic> , yellow) variety at these early stages <xref ref-type="bibr" rid="pone.0076954-Jones1"> [21] </xref> , it is not yet available from the pigmented mutant line Williams ( <italic> i </italic> , black).
					</p>
				
					<p>&nbsp;&nbsp;
						Several hypotheses have been put forward to explain the tissue-specific accumulation of <italic> <span class="gene">CHS</span> </italic> siRNAs <xref ref-type="bibr" rid="pone.0076954-Tuteja1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> . (1) One mechanism could involve the action of a cell or tissue-specific transcription factor or DNA binding protein that initiates sufficient production of the dsRNA progenitor molecules only in the <span class="trait">seed</span> coats and not in other tissues of varieties with the dominant <italic> I </italic> and <italic> i <sup> i </sup> </italic> alleles. Alternatively, (2) the dsRNA could be formed in other tissues but not cleaved properly by a DCL protein to <span class="trait">yield</span> enough primary siRNA to trigger secondary siRNAs, or (3) the primary <italic> <span class="gene">CHS</span> </italic> siRNAs might not be amplified to high levels of secondary siRNAs due to lack of an RdRP or other core components to generate secondary siRNAs in other tissues. Here, our results with high throughput sequencing of large populations of small RNAs from eight non-seed tissues revealed only very low basal levels of <italic> <span class="gene">CHS</span> </italic> siRNAs, generally in the range of 60 RPM. We propose that the level of the <italic> <span class="gene">CHS</span> </italic> primary siRNAs is too low to trigger the secondary amplification in these tissues, unlike the situation in the non-pigmented <span class="trait">seed</span> coats where our data clearly show a rise in the levels of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs to nearly 1000 RPM before the secondary siRNAs overtake them in abundance.
					</p>
				
					<p>&nbsp;&nbsp;
						Using RNAse protection experiments, a recent study has detected <italic> <span class="gene">CHS</span> </italic> dsRNA not only in the <span class="trait">seed</span> <span class="trait">coat</span> but also in the cotyledon and leaf tissues of lines with a dominant <italic> I </italic> allele <xref ref-type="bibr" rid="pone.0076954-Kurauchi1"> [22] </xref> . Although those results were not quantitative, they suggested that the biogenesis of <italic> <span class="gene">CHS</span> </italic> siRNA could be regulated in a tissue-specific manner after dsRNAs are generated by the failure to amplify secondary siRNAs because of a lack of a critical component in the amplification step, such as an RdRP activity. However, this would not explain why <italic> <span class="gene">CHS</span> </italic> dsRNAs that are formed in non-seed <span class="trait">coat</span> tissues would fail to be processed and amplified when many other miRNAs and siRNAs are fully processed and amplified in other tissues such as cotyledons, leaves, and roots of soybean with the <italic> I </italic> and <italic> i <sup> i </sup> </italic> alleles <xref ref-type="bibr" rid="pone.0076954-Tuteja2"> [4] </xref> , <xref ref-type="bibr" rid="pone.0076954-Joshi1"> [23] </xref> , <xref ref-type="bibr" rid="pone.0076954-Zabala1"> [24] </xref> including numerous phased siRNAs (phasiRNAs) resulting from 22-nt miRNAs in soybean roots that target the <italic> NB-LRR </italic> defense gene families <xref ref-type="bibr" rid="pone.0076954-Zhai1"> [25] </xref> . As shown in <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> , <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> , <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> , <xref ref-type="fig" rid="pone-0076954-g004"> 4 </xref> , the amplification of the secondary siRNAs starts in the 5&#x02013;6 mg <span class="trait">seed</span> coats. Recently, RNA-Seq data of <span class="trait">seed</span> <span class="trait">coat</span> and cotyledons from the 5&#x02013;6 mg stage has been reported <xref ref-type="bibr" rid="pone.0076954-Jones1"> [21] </xref> . The RNA-Seq data showed that the expression levels of Glyma models with annotations such as RdRP, DCL, and AGO (argonaute like protein), are very low and have no significant differences between <span class="trait">seed</span> <span class="trait">coat</span> and cotyledon (data not shown). For these reasons, we currently prefer hypothesis (1) that the level of the <italic> <span class="gene">CHS</span> </italic> siRNAs follows an increase in the nascent dsRNA to levels that are higher in the <span class="trait">seed</span> coats than in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						Whether the limiting factor proves to be the levels of the dsRNA or primary siRNAs in tissues other than the <span class="trait">seed</span> <span class="trait">coat,</span> it is clear that secondary amplification from the target <italic> <span class="gene">CHS</span> </italic> mRNAs does not occur in other tissues as seen by the lack of high levels of 21-nt <italic> <span class="gene">CHS</span> </italic> siRNAs that align to any of the <italic> <span class="gene">CHS</span> </italic> genes ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ). The absence of secondary siRNAs is not from the lack of <italic> <span class="gene">CHS</span> </italic> mRNA substrates as can be seen from a number of studies that report reasonable levels of <italic> <span class="gene">CHS</span> </italic> mRNAs in cotyledons, roots, leaves, and other tissues using RNA blots <xref ref-type="bibr" rid="pone.0076954-Tuteja1"> [3] </xref> . In addition, using RT-PCR, most of the <italic> <span class="gene">CHS</span> </italic> genes including <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> were shown to be induced to high levels in pathogen-challenged leaf tissues <xref ref-type="bibr" rid="pone.0076954-Zabala2"> [26] </xref> indicating that sufficient <italic> <span class="gene">CHS</span> </italic> mRNA substrates for secondary amplification from even a low level of primary <italic> <span class="gene">CHS</span> </italic> siRNAs in the leaves would be possible. It is clear that <italic> <span class="gene">CHS</span> </italic> is down-regulated by the <italic> <span class="gene">CHS</span> </italic> siRNAs only in the <span class="trait">seed</span> coats, though sufficient levels of <italic> <span class="gene">CHS</span> </italic> mRNAs exist in other tissues to serve as substrates for amplification of secondary siRNAs to high levels.
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, because of the unique properties of the soybean silencing system, we were able to reveal that the onset of production of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs in the very early <span class="trait">seed</span> stages contained a higher proportion of 22-nt siRNAs originating from the <italic> CHS1-3-4 </italic> clusters of the <italic> i <sup> i </sup> </italic> allele. Then, the transition during development to a very large population of 21-nt secondary siRNAs representing amplification from the target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes was apparent. Our results demonstrate regulation of the <italic> <span class="gene">CHS</span> </italic> inverted repeat genomic region is likely to rely on factors that occur during the very early <span class="trait">seed</span> developmental program and may reflect such changes. Further study of this system should provide more insight into the mechanisms of tissue specificity regulated by small RNAs and more broadly the blueprint of genes involved in development using the <span class="trait">seed</span> <span class="trait">coat</span> as a model.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs for Each </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> Gene Reveal Biogenesis of </bold> <bold> <italic> CHS7/8 </italic> </bold> <bold> siRNAs is Dramatically Increased during <span class="trait">Seed</span> <span class="trait">Coat</span> Development in Biological Repeats of Eight Stages. </bold> <italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart. Numbers above the <span class="gene">bar</span> indicate the total counts of <italic> CHS4 </italic> (red) and <italic> <span class="gene">CHS7</span> </italic> (blue) siRNAs. Developmental stages are whole <span class="trait">seed</span> from (a) 4 DAF (Days After Flowering); (b) 12&#x02013;14 DAF; and (c) 22&#x02013;24 DAF; <span class="trait">seed</span> coats dissected from immature green <span class="trait">seed</span> of fresh weight (d) 5&#x02013;6 mg; (e) 10&#x02013;25 mg, no repeat data; (f) 25&#x02013;50 mg, no repeat data; (g) 50&#x02013;75 mg; (h) 75&#x02013;100 mg; (i) 200&#x02013;300 mg; and <span class="trait">seed</span> coats from (j) 300&#x02013;400 mg yellow, desiccating <span class="trait">seed.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Size Distributions of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development. </bold> <italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of the same eight developmental stages as described in Supplemental <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> were analyzed for their size distributions between 18 and 25 nt. The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Normalized Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs that Match Either </bold> <bold> <italic> CHS4 </italic> </bold> <bold> or </bold> <bold> <italic> <span class="gene">CHS7</span> </italic> </bold> <bold> with 100% Identity in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development. </bold> The 21-nt secondary <italic> <span class="gene">CHS7</span> </italic> siRNAs are amplified to high levels relative to the <italic> CHS4 </italic> 21-nt or 22-nt siRNAs. Developmental stages are indicated.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome Re-Sequencing of Semi-Wild Soybean Reveals a Complex <italic> Soja </italic> Population Structure and Deep Introgression</h2>
			<h3>PMID:25265539</h3>
				
					<h3>PMCID:4181298</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Qiu </surname>  <given-names> Jie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Yu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Sanling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ying-Ying </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ye </surname>  <given-names> Chu-Yu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bai </surname>  <given-names> Xuefei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Zefeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Chenghai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Weidi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ziqiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Qingyao </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Jiahua </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Suk-Ha </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Longjiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Both natural and domestication selections target genes/loci controlling adaptive or agronomic traits and leave footprints of selection in the soybean genome. Several candidate domestication regions have been identified by recent genomic investigations of wild and cultivated soybeans <xref rid="pone.0108479-Li3" ref-type="bibr"> [25] </xref> , <xref rid="pone.0108479-Hyten1" ref-type="bibr"> [26] </xref> , <xref rid="pone.0108479-Chung1" ref-type="bibr"> [28] </xref> . Of the 282 genes located in the putative selective regions in this study ( <xref ref-type="supplementary-material" rid="pone.0108479.s008"> Table S7 </xref> ), at least 63 were also identified as putative selective genes by Li <italic> et al </italic> . <xref rid="pone.0108479-Li3" ref-type="bibr"> [25] </xref> or Chung <italic> et al </italic> . <xref rid="pone.0108479-Chung1" ref-type="bibr"> [28] </xref> ( <xref ref-type="supplementary-material" rid="pone.0108479.s008"> Table S7 </xref> ). The gene coding for ABSCISIC ACID-INSENSITIVE 5-like protein 3 <span class="gene">(Glyma13g03880)</span> was detected by both of their studies as well as our study. It is one of the <span class="trait">seed</span> <span class="trait">color</span> related genes based on QTL mapping <xref rid="pone.0108479-Yang1" ref-type="bibr"> [63] </xref> , and it suggests that this gene was targeted by the recent genetic improvement. Besides <span class="trait">seed</span> size, <span class="trait">seed</span> <span class="trait">color</span> is also a main target trait of domestication and subsequent improvement. In our study, a relatively strict criteria ( <italic> Z(H <sub> p </sub> ) </italic> &#x0003c;&#x02013;4) was applied, which may exclude some putative artificially selected genes. Moreover, as shown above, 14,157 large regions (&#x0003e;1 kb), or about 2.8% of the reference genome of cultivated Williams 82, were not mapped by any reads from the deep sequenced (&#x0223c;41&#x000d7;) semi-wild line Maliaodou ( <xref ref-type="supplementary-material" rid="pone.0108479.s006"> Table S5 </xref> ). These regions might be kept only in the cultivated soybean population during improvement. Taken together, the above-identified genes in the selective and un-mapping regions in this study that may relate to the artificial selection and important agronomic traits provide some candidate targets for further functional investigation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Validation of Loci Governing <span class="trait">Seed</span> <span class="trait">Coat</span> <span class="trait">Color</span> by Combining Association Mapping and Bulk Segregation Analysis in Soybean</h2>
			<h3>PMID:27404272</h3>
				
					<h3>PMCID:4942065</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Song </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Liu </surname>  <given-names> Zhangxiong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Hong </surname>  <given-names> Huilong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Yansong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Long </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xinxiu </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ying-Hui </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Rongxia </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> <span class="trait">color</span> has moderately complex inheritance which is controlled by multi-loci. At least five genetic loci ( <italic> I </italic> , <italic> R </italic> , <italic> T </italic> , <italic> <span class="gene">W1</span> </italic> , and <italic> O </italic> ) were identified by classical genetics, most of which were involved in flavonoid-based pigmentation pathway [ <xref rid="pone.0159064.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> ]. Among them, three ( <italic> I </italic> , <italic> R </italic> , and <italic> T </italic> ) are involved in the biosynthesis of the pigments while <italic> O </italic> and <italic> <span class="gene">W1</span> </italic> only influence the pigmentation under the background of recessive alleles of <italic> i r </italic> or <italic> i t </italic> , respectively [ <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> ]. There are four alleles (known as <italic> I </italic> , <italic> i </italic> <sup> <italic> i </italic> </sup> , <italic> i </italic> <sup> <italic> k </italic> </sup> , and <italic> i </italic> ) at <italic> I </italic> locus controlling the presence/absence and spatial distribution of anthocyanin and proanthocyanidin <italic> via </italic> posttranscriptional gene silencing. Soybeans possessing dominant <italic> I </italic> allele exhibit complete colorless of <span class="trait">seed</span> <span class="trait">coat</span> while soybeans with <italic> i </italic> allele give rise to colored <span class="trait">seed</span> <span class="trait">coat</span> [ <xref rid="pone.0159064.ref012" ref-type="bibr"> 12 </xref> ]. The other two alleles ( <italic> i </italic> <sup> <italic> i </italic> </sup> and <italic> i </italic> <sup> <italic> k </italic> </sup> ) restrict pigments to the hilum and saddle regions of the <span class="trait">seed</span> <span class="trait">coat</span> [ <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> ]. <italic> R </italic> and <italic> T </italic> loci control the type and abundance of pigments in <span class="trait">seed</span> <span class="trait">coat,</span> resulting in specific colors including black ( <italic> i </italic> , <italic> R </italic> , <italic> T </italic> ), imperfect black ( <italic> i </italic> , <italic> R </italic> , <italic> t </italic> ), brown ( <italic> i </italic> , <italic> r </italic> , <italic> T </italic> ), or buff ( <italic> i </italic> , <italic> r </italic> , <italic> t </italic> ) [ <xref rid="pone.0159064.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0159064.ref016" ref-type="bibr"> 16 </xref> ]. <italic> <span class="gene">W1</span> </italic> locus only affects <span class="trait">seed</span> <span class="trait">color</span> under <italic> iRt </italic> background and <italic> <span class="gene">W1</span> </italic> and <italic> <span class="gene">w1</span> </italic> alleles give imperfect black and buff colors, respectively. <italic> O </italic> locus affects <span class="trait">color</span> of brown <span class="trait">seed</span> and soybeans with the recessive <italic> o </italic> allele under <italic> irT </italic> background exhibit red-brown <span class="trait">seed</span> <span class="trait">coat</span> [ <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> ]. In addition, mutants with different combinations (single, double or triple mutants) of <italic> G </italic> , <italic> d1 </italic> and <italic> d2 </italic> loci give rise to green <span class="trait">seed</span> <span class="trait">color</span> and segregation of <italic> G1 </italic> , <italic> G2 </italic> , and <italic> G3 </italic> for green <span class="trait">color</span> has also been studied previously [ <xref rid="pone.0159064.ref017" ref-type="bibr"> 17 </xref> &#x02013; <xref rid="pone.0159064.ref019" ref-type="bibr"> 19 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Molecular cloning of these loci suggested that many of them were structural or regulatory genes involving in anthocyanin biosynthesis pathway. <italic> I </italic> locus was mapped to a region harboring a cluster of chalcone synthase (CHS) genes on chromosome 8 of soybean genome [ <xref rid="pone.0159064.ref020" ref-type="bibr"> 20 </xref> &#x02013; <xref rid="pone.0159064.ref022" ref-type="bibr"> 22 </xref> ]. The recessive <italic> i </italic> allele had a deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences, resulting in an increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> due to the abolishment of posttranscriptional RNA silencing [ <xref rid="pone.0159064.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0159064.ref024" ref-type="bibr"> 24 </xref> ]. Cloning of genomic and cDNA sequences of flavonoid 3&#x02019;-hydroxylase ( <italic> F3&#x02019;H </italic> ) gene suggested that this gene cosegregated with <italic> T </italic> locus [ <xref rid="pone.0159064.ref025" ref-type="bibr"> 25 </xref> , <xref rid="pone.0159064.ref026" ref-type="bibr"> 26 </xref> ]. Chromatographic experiments and genetic analysis also revealed that <italic> <span class="gene">W1</span> </italic> might encode a flavonoid 3&#x02019; 5&#x02019; hydroxylase (F3&#x02019;5&#x02019;H) as a 65-bp insertion in this gene cosegregated with the mutant phenotype [ <xref rid="pone.0159064.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0159064.ref027" ref-type="bibr"> 27 </xref> ]. <italic> R </italic> locus was initially mapped to LG K (chromosome 9) [ <xref rid="pone.0159064.ref028" ref-type="bibr"> 28 </xref> ] and then restricted to a region between molecular markers A668_1 and K387_1 [ <xref rid="pone.0159064.ref029" ref-type="bibr"> 29 </xref> ]. Candidate gene analysis suggested that loss function of a <span class="trait">seed</span> coat-expressed <italic> R2R3-MYB </italic> gene was responsible for recessive phenotype of <italic> R </italic> locus [ <xref rid="pone.0159064.ref030" ref-type="bibr"> 30 </xref> , <xref rid="pone.0159064.ref031" ref-type="bibr"> 31 </xref> ]. Furthermore, <italic> O </italic> locus has been found to correspond to an anthocyanidin reductase (ANR) gene, which needs to be further confirmed [ <xref rid="pone.0159064.ref013" ref-type="bibr"> 13 </xref> ]. Recently, cloning and characterization of <italic> D1 </italic> and <italic> D2 </italic> revealed that they were homologs of the <italic> STAY-GREEN </italic> ( <italic> SGR </italic> ) genes from other plant species and were duplicated as a result of the most recent whole genome duplication in soybean [ <xref rid="pone.0159064.ref032" ref-type="bibr"> 32 </xref> , <xref rid="pone.0159064.ref033" ref-type="bibr"> 33 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Combinations of different loci can be used to infer genetic effect of each locus for specific trait. Three SSR markers closely linked to <italic> qSC1 </italic> , <italic> qSC5/I </italic> , <italic> qSC7/R </italic> loci (BARCSOYSSR_1_1528, 8_466, and 9_1491) and a dCAPS marker of <italic> <span class="gene">GmF3&#x02019;H</span> </italic> gene for <italic> qSC2/T </italic> locus were used for genotyping entire F <sub> 2 </sub> population. The results revealed that all individuals possessing dominant <italic> qSC5/I </italic> allele showed green or yellow <span class="trait">seed</span> coats while soybeans with recessive <italic> qsc5/i </italic> allele showed black or brown coats. In the <italic> qSC5/I </italic> background, <span class="trait">seed</span> colors of all individuals possessing dominant <italic> qSC1 </italic> allele were green while soybeans with recessive <italic> qsc1 </italic> allele showed yellow <span class="trait">seed</span> coats. Furthermore, <span class="trait">seed</span> colors of individuals with recessive <italic> qsc2/t </italic> locus in the <italic> qsc5/i </italic> background were brown. However, when individuals possessed dominant allele of <italic> qSC2/T </italic> and recessive allele of <italic> qsc5/i </italic> , <italic> qSC7/R </italic> locus could be used for distinguishing black and brown <span class="trait">seed</span> <span class="trait">coat</span> ( <xref ref-type="table" rid="pone.0159064.t004"> Table 4 </xref> ). From these results the interaction of different loci can be concluded, in which <italic> qSC5/I </italic> locus controlled pigmentation of <span class="trait">seed</span> <span class="trait">coat</span> to dark colors and <italic> qSC1 </italic> governed further pigmentation of relative light <span class="trait">color</span> on the basis of <italic> qSC5/I </italic> locus. In addition, <italic> qSC2/T </italic> and <italic> qSC7/R </italic> loci were responsible for pigmentation of different degrees of dark colors and <italic> qSC2/T </italic> locus might function upstream of <italic> qSC7/R </italic> in this network.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> <span class="trait">coat</span> <span class="trait">color</span> is not only related to biochemical functions of secondary metabolism, antioxidant activity, and disease <span class="trait">resistance</span> but also a morphological trait for classification of germplasm and evolutionary analysis [ <xref rid="pone.0159064.ref003" ref-type="bibr"> 3 </xref> &#x02013; <xref rid="pone.0159064.ref005" ref-type="bibr"> 5 </xref> ]. Apart from five genetic loci controlling flavonoid-based pigmentation [ <xref rid="pone.0159064.ref013" ref-type="bibr"> 13 </xref> , <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> ], eight QTLs on five chromosomes have also been identified through QTL mapping but mapping regions were always too large due to limited number of markers used [ <xref rid="pone.0159064.ref078" ref-type="bibr"> 78 </xref> &#x02013; <xref rid="pone.0159064.ref080" ref-type="bibr"> 80 </xref> ]. Among them, two QTLs <span class="trait">(seed</span> <span class="trait">coat</span> color2-1 and 3&#x02013;1) were all close to <italic> I </italic> locus, but it was difficult to confirm whether they were the same QTL due to the large genomic regions in their studies. Moreover, some reports also revealed that combination of gene-based markers of <italic> T </italic> and <italic> <span class="gene">W1</span> </italic> loci or two SNP outliers could partially increase selection efficiency for <span class="trait">seed</span> colors [ <xref rid="pone.0159064.ref064" ref-type="bibr"> 64 </xref> , <xref rid="pone.0159064.ref081" ref-type="bibr"> 81 </xref> ]. Therefore, soybean <span class="trait">seed</span> <span class="trait">color</span> has a relative complex genetic basis and accessions with same <span class="trait">color</span> possibly having different genotypes at these loci.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous reports indicated that <italic> I </italic> locus had major effect on controlling pigmentation of <span class="trait">seed</span> <span class="trait">coat</span> [ <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> , <xref rid="pone.0159064.ref021" ref-type="bibr"> 21 </xref> , <xref rid="pone.0159064.ref022" ref-type="bibr"> 22 </xref> ]. Our results from association mapping, BSA of F <sub> 2 </sub> individuals and RHL populations all supported this conclusion as <italic> qSC5/I </italic> locus could be used for distinguishing dark and light colors of <span class="trait">seed</span> <span class="trait">coat.</span> Since the <span class="trait">seed</span> colors of wild soybeans and modern cultivars are mainly black and yellow respectively, <italic> qSC5/I </italic> locus may undergo selection during soybean domestication. Previous report on resequencing of wild and cultivated soybeans also indentified three genes in <italic> qSC5/I </italic> region with strong selection signals [ <xref rid="pone.0159064.ref007" ref-type="bibr"> 7 </xref> ]. <italic> qSC1 </italic> locus was proven to co-segregate with green <span class="trait">color</span> and dominant <italic> qSC1 </italic> allele could pigment light green <span class="trait">color</span> in <italic> qSC5/I </italic> allele background. Up to now, few reports illustrated the genetic basis of green <span class="trait">seed</span> <span class="trait">color</span> in soybean, partially because the segregation of this kind of individuals is more complex than others. There is also a possibility that the green <span class="trait">color</span> is fading out at maturity of soybean <span class="trait">seed</span> and becoming yellow under the control of <italic> v1 </italic> or <italic> g1 </italic> locus [ <xref rid="pone.0159064.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0159064.ref086" ref-type="bibr"> 86 </xref> ]. Since <italic> qSC5/I </italic> was proven to regulate the expression of <italic> <span class="gene">CHS</span> </italic> genes which had function in early step in flavonoids and anthocyanins biosynthesis [ <xref rid="pone.0159064.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0159064.ref024" ref-type="bibr"> 24 </xref> , <xref rid="pone.0159064.ref087" ref-type="bibr"> 87 </xref> , <xref rid="pone.0159064.ref088" ref-type="bibr"> 88 </xref> ], we postulated that <italic> qSC1 </italic> might affect the coloration of <span class="trait">seed</span> <span class="trait">coat</span> in an independent pathway.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, previous studies also revealed that <italic> T </italic> and <italic> R </italic> loci were associated with black and brown <span class="trait">seed</span> coats [ <xref rid="pone.0159064.ref014" ref-type="bibr"> 14 </xref> &#x02013; <xref rid="pone.0159064.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0159064.ref031" ref-type="bibr"> 31 </xref> ]. Characterization of <italic> R </italic> locus suggested that functional <italic> R </italic> gene acted to promote transcription of structural genes encoding U3FGT and <span class="gene">ANS</span> which were located in downstream of flavonoid 3&#x02019;-hydroxylase (encoded by <italic> <span class="gene">GmF3&#x02019;H</span> </italic> gene) in anthocyanin pathway [ <xref rid="pone.0159064.ref030" ref-type="bibr"> 30 </xref> ]. In this study, <italic> qSC7/R </italic> locus could be used for distinguish black and brown <span class="trait">seed</span> coats only under the background of dominant <italic> qSC2/T </italic> locus, also indicating that <italic> qSC7/R </italic> locus was involved in the downstream of <italic> qSC2/T </italic> . Therefore, further fine mapping and cloning of <italic> qSC1 </italic> will contribute to construct regulatory network of <span class="trait">seed</span> <span class="trait">coat</span> pigmentation in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) Estimated <span class="gene">ln</span> (probability of the data) calculated for K ranging from 2 to 9. (B) Population structure of soybean accessions, each accession was represented by a single vertical line and every <span class="trait">color</span> represented one cluster. The red <span class="trait">color</span> indicated Subgroup I and the green <span class="trait">color</span> indicated subgroup II.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>