<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed coat color</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed coat color</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Mutations in Argonaute5 Illuminate Epistatic Interactions of the K1 and I Loci Leading to Saddle <span class="trait">Seed</span> <span class="trait">Color</span> Patterns in Glycine max.</h2>
			<h3>PMID:28351993  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cho YB(1), Jones SI(1), Vodkin LO(2).
				<br/> 
				年份：2017 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Small RNA sequencing of dissected regions of immature <span class="trait">seed</span> coats demonstrated that <span class="gene">CHS</span> siRNA levels cause the patterns produced by the ii and ik alleles of the I locus, which restrict pigment to the hilum or saddle region of the <span class="trait">seed</span> <span class="trait">coat,</span> respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Nonfunctional <span class="gene">AGO5</span> from k1 alleles leads to altered distributions of <span class="gene">CHS</span> siRNAs, thus explaining how the k1 mutation reverses the phenotype of the <span class="trait">seed</span> <span class="trait">coat</span> regions from yellow to pigmented, even in the presence of the normally dominant I or ii alleles.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Wide genetic variation in phenolic compound content of <span class="trait">seed</span> coats among black soybean cultivars.</h2>
			<h3>PMID:25914597</h3>
				

				
			<p>
				作者：Phommalath S(1), Teraishi M(1), Yoshikawa T(1), Saito H(1), Tsukiyama T(1), Nakazaki T(1), Tanisaka T(1), Okumoto Y(1).
				<br/> 
				年份：2014 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cultivars with purple flowers had a tendency to produce higher PCCs compared with cultivars with white flowers, suggesting that the <span class="gene">W1</span> locus for flower <span class="trait">color</span> can affect phenolic compound composition and content.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Allele-specific marker development and selection efficiencies for both flavonoid 3'-hydroxylase and flavonoid 3',5'-hydroxylase genes in soybean subgenus soja.</h2>
			<h3>PMID:23463490  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Guo Y(1), Qiu LJ.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For both <span class="gene">F3'H</span> and F3'5'H, the marker selection efficiencies for pubescence <span class="trait">color</span> and flower <span class="trait">color</span> were determined.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The soybean <span class="gene">F3'H</span> protein is localized to the tonoplast in the <span class="trait">seed</span> <span class="trait">coat</span> hilum.</h2>
			<h3>PMID:22258749  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Toda K(1), Kuroiwa H, Senthil K, Shimada N, Aoki T, Ayabe S, Shimada S, Sakuta M, Miyazaki Y, Takahashi R.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Here, Western-blot analysis shows that the <span class="gene">sf3'h1</span> protein was predominantly detected in the hilum and funiculus of the immature <span class="trait">seed</span> <span class="trait">coat</span> in To7B, whereas <span class="gene">sf3'h1</span> was not detected in To7G.
					</p>
				
					<p>&nbsp;&nbsp;
						These results indicate that <span class="gene">sf3'h1</span> is involved in flavonoid biosynthesis in the <span class="trait">seed</span> <span class="trait">coat</span> and affects the antioxidant properties of those tissues.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of two anthocyanidin reductase genes and three red-brown soybean accessions with reduced anthocyanidin reductase 1 mRNA, activity, and <span class="trait">seed</span> <span class="trait">coat</span> proanthocyanidin amounts.</h2>
			<h3>PMID:22107112  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kovinich N(1), Saleem A, Arnason JT, Miki B.
				<br/> 
				年份：2012 Jan 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two spontaneous mutants with red-brown grain <span class="trait">color</span> had reduced <span class="gene">ANR1</span> gene expression in the <span class="trait">seed</span> <span class="trait">coat,</span> and an EMS-mutagenized red-brown mutant had nonsynonymous substitutions that resulted in slightly reduced <span class="gene">ANR1</span> activity in vitro.
					</p>
				
					<p>&nbsp;&nbsp;
						These results suggest that defects in the <span class="gene">ANR1</span> gene can be associated with red-brown soybean grain <span class="trait">color</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						These results suggest that suppressing <span class="gene">ANR1</span> gene expression or activity may be a rational approach toward engineering <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> to enable the visual identification of genetically modified soybean grains.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loss-of-function mutations affecting a specific Glycine max R2R3 MYB transcription factor result in brown hilum and brown <span class="trait">seed</span> coats.</h2>
			<h3>PMID:22070454  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gillman JD(1), Tetlow A, Lee JD, Shannon JG, Bilyeu K.
				<br/> 
				年份：2011 Nov 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Combined analysis of transcriptome and metabolite data reveals extensive differences between black and brown nearly-isogenic soybean ( <italic> Glycine max </italic> ) <span class="trait">seed</span> coats enabling the identification of pigment isogenes</h2>
			<h3>PMID:21801362</h3>
				

				
					<h3>PMCID:3163566</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Kovinich </surname>  <given-names> Nik </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Nikola.Kovinich@agr.gc.ca </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Saleem </surname>  <given-names> Ammar </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> Ammar.Saleem@uOttawa.ca </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Arnason </surname>  <given-names> John T </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> John.Arnason@uottawa.ca </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Miki </surname>  <given-names> Brian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Brian.Miki@agr.gc.ca </email>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We recently identified and characterized a UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase ( <italic> <span class="gene">UGT78K1</span> </italic> ) from the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean with the aim to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> by suppression of an anthocyanin-specific gene.
					</p>
				
					<p>&nbsp;&nbsp;
						However, it remained to be investigated whether <italic> <span class="gene">UGT78K1</span> </italic> was overexpressed with anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Six genetic loci ( <italic> I </italic> , <italic> R </italic> , <italic> T </italic> , <italic> Wp </italic> , <italic> <span class="gene">W1</span> </italic> , and <italic> O </italic> ) [ <xref ref-type="bibr" rid="B6"> 6 </xref> ] identified by classical genetics control flavonoid-based <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The recessive <italic> i </italic> allele results from spontaneous deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences and results in the abolishment of a posttranscriptional RNA silencing mechanism that results in the increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">W1</span> </italic> locus controls flower <span class="trait">color</span> and affects <span class="trait">seed</span> <span class="trait">color</span> only in an <italic> iRt </italic> background; where <italic> <span class="gene">W1</span> </italic> and <italic> w1 </italic> alleles give imperfect black and buff <span class="trait">seed</span> <span class="trait">coat</span> colors, respectively [ <xref ref-type="bibr" rid="B6"> 6 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">W1</span> </italic> allele for purple flower <span class="trait">color</span> was shown to code flavonoid 3',5'-hydroxylase (F3'5'H), as a 65-bp insertion in the gene ( <italic> F3'5'H </italic> ) co-segregated with white flower <span class="trait">color</span> ( <italic> w1 </italic> ) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The Wp locus was suggested to code the flavonone 3-hydroxylase gene ( <italic> <span class="gene">F3H1</span> </italic> ) by microarray analysis as high levels of <italic> <span class="gene">F3H1</span> </italic> transcripts co-segregated with purple flower <span class="trait">color</span> ( <italic> Wp </italic> ), and low levels with pink ( <italic> wp </italic> ) flowers [ <xref ref-type="bibr" rid="B14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, a cDNA coding the UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase (UF3GT) gene ( <italic> <span class="gene">UGT78K1</span> </italic> ) was isolated from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and shown to function in anthocyanin biosynthesis <italic> in vitro </italic> and by complementation of a gene mutation in Arabidopsis [ <xref ref-type="bibr" rid="B18"> 18 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						However <italic> <span class="gene">UGT78K1</span> </italic> expressions have not been investigated in relation to <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the flavonoid genes identified from soybean <span class="trait">seed</span> coats, only <italic> <span class="gene">ANS</span> </italic> transcripts were found to be overexpressed in the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean compared to the brown ( <italic> irT </italic> ) isoline, however no RFLP difference was observed when probing <italic> <span class="gene">ANS</span> </italic> genes [ <xref ref-type="bibr" rid="B17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						This suggests that <italic> <span class="gene">ANS</span> </italic> overexpression is not the only molecular difference between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						To examine whether other anthocyanin genes are upregulated, transcript profiles of the recently identified <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] and the <italic> <span class="gene">DFR1</span> </italic> gene [GenBank: <ext-link ext-link-type="gen" xlink:href="AF167556"> AF167556 </ext-link> ] were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA by quantitative RT-PCR (qRT-PCR) using <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> transcripts [GenBank: <ext-link ext-link-type="gen" xlink:href="AY382829"> AY382829 </ext-link> /GenBank: <ext-link ext-link-type="gen" xlink:href="AY382830"> AY382830 </ext-link> ] as a positive control.
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR demonstrated <italic> <span class="gene">UGT78K1</span> </italic> to be significantly overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at the transition stage (300 mg - 400 mg FSW) and early maturation stage (400 - 500 mg FSW) of <span class="trait">seed</span> development compared to the brown ( <italic> irT </italic> ) isoline, and to be almost undetectable at earlier stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> mRNAs were confirmed to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at most stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) as shown previously [ <xref ref-type="bibr" rid="B17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate how a <span class="trait">seed</span> tissue may coordinate the biosynthesis of these parallel pigment pathways, the expressions of putative PA genes <italic> <span class="gene">ANR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433915"> JF433915 </ext-link> ] and <italic> <span class="gene">LAR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433916"> JF433916 </ext-link> ] were examined in relation to anthocyanin gene expressions.
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR demonstrated <italic> <span class="gene">ANR1</span> </italic> expressions to decrease throughout <span class="trait">seed</span> <span class="trait">coat</span> development in both genotypes and to be almost undetectable by the stages of <italic> <span class="gene">UGT78K1</span> </italic> overexpressions and anthocyanin biosynthesis (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						By contrast, <italic> <span class="gene">LAR1</span> </italic> expressions were downregulated at the stages of anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, these results show that the recently identified <italic> <span class="gene">UGT78K1</span> </italic> is upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> in addition to <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> , and that there is a simultaneous downregulation of the PA gene <italic> <span class="gene">LAR1</span> </italic> at the stages of anthocyanin biosynthesis.
					</p>
				
					<p>&nbsp;&nbsp;
						To ensure <span class="trait">seed</span> coats were at the same stage of development, the days post anthesis, pod length, pod <span class="trait">color,</span> embryo morphology, and transcript levels of the developmental marker gene <span class="gene">Gm-r1083-1191,</span> a putative cutin biosynthesis gene [ <xref ref-type="bibr" rid="B29"> 29 </xref> ], and the metabolism gene <italic> <span class="gene">DFR1</span> </italic> were ensured to be equivalent between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) tissues (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Table S1).
					</p>
				
					<p>&nbsp;&nbsp;
						Nearly 75% of genes in the soybean genome are present in multiple copies [ <xref ref-type="bibr" rid="B19"> 19 </xref> ], and previously only transcripts for <italic> <span class="gene">ANS</span> </italic> isogenes were found to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue [ <xref ref-type="bibr" rid="B17"> 17 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 214 UGTs, only 2 were found to be upregulated more than 2-fold from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat;</span> the recently identified UF3GT gene <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] was upregulated more than 2.
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 84 OMT genes, only probe sets for OMT-like and <span class="gene">OMT5</span> were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 2.
					</p>
				
					<p>&nbsp;&nbsp;
						Isogenes for common flavonoid biosynthesis (CHS1, CHS9, F3'H), PA biosynthesis (LAR1), and isoflavone biosynthesis (IFR1-like) were all down-regulated more than 2-fold in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 35-fold in the case of the <span class="gene">CHS1</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						S1 did not distinguish between <italic> CHS4 </italic> and <italic> <span class="gene">CHS5</span> </italic> expressions, semi-qRT-PCR was performed with gene specific primers to show the <italic> CHS4 </italic> gene, and not the <italic> <span class="gene">CHS5</span> </italic> gene to be upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> : Figure S4).
					</p>
				
					<p>&nbsp;&nbsp;
						The glycosyltransferase <span class="gene">UGT78K2</span> is assigned to code a UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferase (UF3GT) protein (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ) and may have functional redundancies in anthocyanin biosynthesis with the previously characterized <span class="gene">UGT78K1</span> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ], which was also upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Two glycosyltransferases ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> relative to the nearly-isogenic brown ( <italic> irT </italic> ) tissue (Table <xref ref-type="table" rid="T3"> 3 </xref> ) and the upregulation paralleled the accumulation of anthocyanins with 3- <italic> O </italic> -glycosylated structures (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), suggesting the function of these genes as UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferases (UF3GTs).
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the function these genes, <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expression profiles were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA throughout black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) soybean <span class="trait">seed</span> development by quantitative RT-PCR (qRT-PCR) (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ) and expressions were compared to anthocyanin accumulation profiles throughout <span class="trait">seed</span> development (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Similar to <italic> <span class="gene">UGT78K1</span> </italic> , <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expressions peaked at the transition and early maturation stages of <span class="trait">seed</span> development (300 mg - 400 mg FSW, 400 mg - 500 mg FSW) in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ), the stages at which anthocyanins accumulate (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), and were low or almost undetectable at early stages of <span class="trait">seed</span> development in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and at all stages in the brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Seed</span> <span class="trait">coat</span> transcript profiles of <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> during <span class="trait">seed</span> development measured by qRT-PCR </bold>.
					</p>
				
					<p>&nbsp;&nbsp;
						Together the qRT-PCR and recombinant enzyme assays suggest that <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> code UF3GT and AOMT proteins, respectively, that function in anthocyanin pigment biosynthesis in the <span class="trait">seed</span> of black ( <italic> iRT </italic> ) soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						It would be interesting to determine whether the DFR gene that was highly overexpressed in the black Clark <span class="trait">seed</span> <span class="trait">coat</span> ( <italic> <span class="gene">DFR2</span> </italic> ) had similar catalytic activity, and whether the Korean varieties contain genetic alterations related to <italic> <span class="gene">DFR2</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						However, our microarray and semi-quantitative RT-PCR data demonstrated that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> were not differentially regulated in black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, it was <italic> CHS4 </italic> that was upregulated and <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> that were downregulatedin the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> and Table <xref ref-type="table" rid="T4"> 4 </xref> respectively).
					</p>
				
					<p>&nbsp;&nbsp;
						By contrast, the downregulation of <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> may suggest that they have a role in the parallel downregulated biochemical pathways, such as flavonol and isoflavonoid biosynthesis, as these metabolites, and transcript levels of genes for their biosynthesis, were reduced or not present in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, our metabolite data showed that soybean PAs lack catechin subunits (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Figure S1) and that <italic> <span class="gene">LAR1</span> </italic> expressions (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) do not parallel PA accumulations during <span class="trait">seed</span> development (not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						A UF3GT cDNA ( <italic> <span class="gene">UGT78K1</span> </italic> ) has recently been isolated from the black <span class="trait">seed</span> <span class="trait">coat</span> and functionally characterized [ <xref ref-type="bibr" rid="B18"> 18 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, two OMT genes ( <italic> <span class="gene">OMT5</span> </italic> and <italic> OMT-like </italic> ) were found to be overexpressed in the black <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						UF3GT and OMT genes may be ideal candidates for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> engineering as suppression of UF3GT genes ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) may result in a reduction in anthocyanin levels similar to that caused by a T-DNA insertion in the Arabidopsis UF3GT [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] and upregulation of an AOMT may result in a more red <span class="trait">seed</span> <span class="trait">coat,</span> as methylation of the B-ring of anthocyanins has a reddening effect on flower <span class="trait">color</span> (reviewed by [ <xref ref-type="bibr" rid="B35"> 35 </xref> ]).
					</p>
				
					<p>&nbsp;&nbsp;
						A Mob1/phocein gene was commonly upregulated in the black ( <italic> iRT </italic> ) soybean <span class="trait">seed</span> <span class="trait">coat</span> and Arabidopsis seedlings overexpressing <span class="gene">PAP1</span> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Full-length CDSs for <span class="gene">UGT78K2</span> and <span class="gene">OMT5</span> were amplified from black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> cDNA by end-to-end PCR using primers UHF/UAR and HO5F/HO5R, respectively (Additional file <xref ref-type="supplementary-material" rid="S12"> 12 </xref> : Table S7) and a high-fidelity DNA polymerase (Invitrogen).
					</p>
				
					<p>&nbsp;&nbsp;
						We recently identified and characterized a UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase ( <italic> <span class="gene">UGT78K1</span> </italic> ) from the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean with the aim to engineer <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> by suppression of an anthocyanin-specific gene
					</p>
				
					<p>&nbsp;&nbsp;
						However, it remained to be investigated whether <italic> <span class="gene">UGT78K1</span> </italic> was overexpressed with anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue
					</p>
				
					<p>&nbsp;&nbsp;
						Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively
					</p>
				
					<p>&nbsp;&nbsp;
						Six genetic loci ( <italic> I </italic> , <italic> R </italic> , <italic> T </italic> , <italic> Wp </italic> , <italic> <span class="gene">W1</span> </italic> , and <italic> O </italic> ) [ <xref ref-type="bibr" rid="B6"> 6 </xref> ] identified by classical genetics control flavonoid-based <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> in soybean
					</p>
				
					<p>&nbsp;&nbsp;
						The recessive <italic> i </italic> allele results from spontaneous deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences and results in the abolishment of a posttranscriptional RNA silencing mechanism that results in the increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> [ <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B8"> 8 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">W1</span> </italic> locus controls flower <span class="trait">color</span> and affects <span class="trait">seed</span> <span class="trait">color</span> only in an <italic> iRt </italic> background; where <italic> <span class="gene">W1</span> </italic> and <italic> w1 </italic> alleles give imperfect black and buff <span class="trait">seed</span> <span class="trait">coat</span> colors, respectively [ <xref ref-type="bibr" rid="B6"> 6 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">W1</span> </italic> allele for purple flower <span class="trait">color</span> was shown to code flavonoid 3',5'-hydroxylase (F3'5'H), as a 65-bp insertion in the gene ( <italic> F3'5'H </italic> ) co-segregated with white flower <span class="trait">color</span> ( <italic> w1 </italic> ) [ <xref ref-type="bibr" rid="B13"> 13 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The Wp locus was suggested to code the flavonone 3-hydroxylase gene ( <italic> <span class="gene">F3H1</span> </italic> ) by microarray analysis as high levels of <italic> <span class="gene">F3H1</span> </italic> transcripts co-segregated with purple flower <span class="trait">color</span> ( <italic> Wp </italic> ), and low levels with pink ( <italic> wp </italic> ) flowers [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, a cDNA coding the UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase (UF3GT) gene ( <italic> <span class="gene">UGT78K1</span> </italic> ) was isolated from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and shown to function in anthocyanin biosynthesis <italic> in vitro </italic> and by complementation of a gene mutation in Arabidopsis [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						However <italic> <span class="gene">UGT78K1</span> </italic> expressions have not been investigated in relation to <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Among the flavonoid genes identified from soybean <span class="trait">seed</span> coats, only <italic> <span class="gene">ANS</span> </italic> transcripts were found to be overexpressed in the <span class="trait">seed</span> <span class="trait">coat</span> of black ( <italic> iRT </italic> ) soybean compared to the brown ( <italic> irT </italic> ) isoline, however no RFLP difference was observed when probing <italic> <span class="gene">ANS</span> </italic> genes [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						This suggests that <italic> <span class="gene">ANS</span> </italic> overexpression is not the only molecular difference between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						To examine whether other anthocyanin genes are upregulated, transcript profiles of the recently identified <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] and the <italic> <span class="gene">DFR1</span> </italic> gene [GenBank: <ext-link ext-link-type="gen" xlink:href="AF167556"> AF167556 </ext-link> ] were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA by quantitative RT-PCR (qRT-PCR) using <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> transcripts [GenBank: <ext-link ext-link-type="gen" xlink:href="AY382829"> AY382829 </ext-link> /GenBank: <ext-link ext-link-type="gen" xlink:href="AY382830"> AY382830 </ext-link> ] as a positive control
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR demonstrated <italic> <span class="gene">UGT78K1</span> </italic> to be significantly overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at the transition stage (300 mg - 400 mg FSW) and early maturation stage (400 - 500 mg FSW) of <span class="trait">seed</span> development compared to the brown ( <italic> irT </italic> ) isoline, and to be almost undetectable at earlier stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> mRNAs were confirmed to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> at most stages (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) as shown previously [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						To investigate how a <span class="trait">seed</span> tissue may coordinate the biosynthesis of these parallel pigment pathways, the expressions of putative PA genes <italic> <span class="gene">ANR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433915"> JF433915 </ext-link> ] and <italic> <span class="gene">LAR1</span> </italic> [GenBank: <ext-link ext-link-type="gen" xlink:href="JF433916"> JF433916 </ext-link> ] were examined in relation to anthocyanin gene expressions
					</p>
				
					<p>&nbsp;&nbsp;
						qRT-PCR demonstrated <italic> <span class="gene">ANR1</span> </italic> expressions to decrease throughout <span class="trait">seed</span> <span class="trait">coat</span> development in both genotypes and to be almost undetectable by the stages of <italic> <span class="gene">UGT78K1</span> </italic> overexpressions and anthocyanin biosynthesis (Figure <xref ref-type="fig" rid="F1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						By contrast, <italic> <span class="gene">LAR1</span> </italic> expressions were downregulated at the stages of anthocyanin biosynthesis in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, these results show that the recently identified <italic> <span class="gene">UGT78K1</span> </italic> is upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> in addition to <italic> ANS2 </italic> / <italic> <span class="gene">ANS3</span> </italic> , and that there is a simultaneous downregulation of the PA gene <italic> <span class="gene">LAR1</span> </italic> at the stages of anthocyanin biosynthesis
					</p>
				
					<p>&nbsp;&nbsp;
						To ensure <span class="trait">seed</span> coats were at the same stage of development, the days post anthesis, pod length, pod <span class="trait">color,</span> embryo morphology, and transcript levels of the developmental marker gene <span class="gene">Gm-r1083-1191,</span> a putative cutin biosynthesis gene [ <xref ref-type="bibr" rid="B29"> 29 </xref> ], and the metabolism gene <italic> <span class="gene">DFR1</span> </italic> were ensured to be equivalent between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) tissues (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Table S1)
					</p>
				
					<p>&nbsp;&nbsp;
						Nearly 75% of genes in the soybean genome are present in multiple copies [ <xref ref-type="bibr" rid="B19"> 19 </xref> ], and previously only transcripts for <italic> <span class="gene">ANS</span> </italic> isogenes were found to be overexpressed in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> compared to the nearly-isogenic brown ( <italic> irT </italic> ) tissue [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 214 UGTs, only 2 were found to be upregulated more than 2-fold from the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat;</span> the recently identified UF3GT gene <italic> <span class="gene">UGT78K1</span> </italic> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] was upregulated more than 2
					</p>
				
					<p>&nbsp;&nbsp;
						Of the 84 OMT genes, only probe sets for OMT-like and <span class="gene">OMT5</span> were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 2
					</p>
				
					<p>&nbsp;&nbsp;
						Isogenes for common flavonoid biosynthesis (CHS1, CHS9, F3'H), PA biosynthesis (LAR1), and isoflavone biosynthesis (IFR1-like) were all down-regulated more than 2-fold in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat,</span> more than 35-fold in the case of the <span class="gene">CHS1</span> (Table <xref ref-type="table" rid="T4"> 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						S1 did not distinguish between <italic> CHS4 </italic> and <italic> <span class="gene">CHS5</span> </italic> expressions, semi-qRT-PCR was performed with gene specific primers to show the <italic> CHS4 </italic> gene, and not the <italic> <span class="gene">CHS5</span> </italic> gene to be upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Additional file <xref ref-type="supplementary-material" rid="S10"> 10 </xref> : Figure S4)
					</p>
				
					<p>&nbsp;&nbsp;
						The glycosyltransferase <span class="gene">UGT78K2</span> is assigned to code a UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferase (UF3GT) protein (Figure <xref ref-type="fig" rid="F6"> 6 </xref> ) and may have functional redundancies in anthocyanin biosynthesis with the previously characterized <span class="gene">UGT78K1</span> [ <xref ref-type="bibr" rid="B18"> 18 </xref> ], which was also upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Two glycosyltransferases ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) were upregulated in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> relative to the nearly-isogenic brown ( <italic> irT </italic> ) tissue (Table <xref ref-type="table" rid="T3"> 3 </xref> ) and the upregulation paralleled the accumulation of anthocyanins with 3- <italic> O </italic> -glycosylated structures (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), suggesting the function of these genes as UDP-glycose:flavonoid 3- <italic> O </italic> -glycosyltransferases (UF3GTs)
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the function these genes, <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expression profiles were measured from <span class="trait">seed</span> <span class="trait">coat</span> cDNA throughout black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) soybean <span class="trait">seed</span> development by quantitative RT-PCR (qRT-PCR) (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ) and expressions were compared to anthocyanin accumulation profiles throughout <span class="trait">seed</span> development (Figure <xref ref-type="fig" rid="F2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Similar to <italic> <span class="gene">UGT78K1</span> </italic> , <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> expressions peaked at the transition and early maturation stages of <span class="trait">seed</span> development (300 mg - 400 mg FSW, 400 mg - 500 mg FSW) in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F9"> 9 </xref> ), the stages at which anthocyanins accumulate (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ), and were low or almost undetectable at early stages of <span class="trait">seed</span> development in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> and at all stages in the brown ( <italic> irT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> <span class="trait">Seed</span> <span class="trait">coat</span> transcript profiles of <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> during <span class="trait">seed</span> development measured by qRT-PCR </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Together the qRT-PCR and recombinant enzyme assays suggest that <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> code UF3GT and AOMT proteins, respectively, that function in anthocyanin pigment biosynthesis in the <span class="trait">seed</span> of black ( <italic> iRT </italic> ) soybean
					</p>
				
					<p>&nbsp;&nbsp;
						It would be interesting to determine whether the DFR gene that was highly overexpressed in the black Clark <span class="trait">seed</span> <span class="trait">coat</span> ( <italic> <span class="gene">DFR2</span> </italic> ) had similar catalytic activity, and whether the Korean varieties contain genetic alterations related to <italic> <span class="gene">DFR2</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						However, our microarray and semi-quantitative RT-PCR data demonstrated that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> were not differentially regulated in black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats (Table <xref ref-type="table" rid="T3"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, it was <italic> CHS4 </italic> that was upregulated and <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> that were downregulatedin the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> and Table <xref ref-type="table" rid="T4"> 4 </xref> respectively)
					</p>
				
					<p>&nbsp;&nbsp;
						By contrast, the downregulation of <italic> <span class="gene">CHS1</span> </italic> and <italic> <span class="gene">CHS9</span> </italic> may suggest that they have a role in the parallel downregulated biochemical pathways, such as flavonol and isoflavonoid biosynthesis, as these metabolites, and transcript levels of genes for their biosynthesis, were reduced or not present in the black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> (Figure <xref ref-type="fig" rid="F6"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, our metabolite data showed that soybean PAs lack catechin subunits (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> : Figure S1) and that <italic> <span class="gene">LAR1</span> </italic> expressions (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ) do not parallel PA accumulations during <span class="trait">seed</span> development (not shown)
					</p>
				
					<p>&nbsp;&nbsp;
						A UF3GT cDNA ( <italic> <span class="gene">UGT78K1</span> </italic> ) has recently been isolated from the black <span class="trait">seed</span> <span class="trait">coat</span> and functionally characterized [ <xref ref-type="bibr" rid="B18"> 18 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Similarly, two OMT genes ( <italic> <span class="gene">OMT5</span> </italic> and <italic> OMT-like </italic> ) were found to be overexpressed in the black <span class="trait">seed</span> <span class="trait">coat</span> (Table <xref ref-type="table" rid="T3"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						UF3GT and OMT genes may be ideal candidates for <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> engineering as suppression of UF3GT genes ( <italic> <span class="gene">UGT78K1</span> </italic> and <italic> <span class="gene">UGT78K2</span> </italic> ) may result in a reduction in anthocyanin levels similar to that caused by a T-DNA insertion in the Arabidopsis UF3GT [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] and upregulation of an AOMT may result in a more red <span class="trait">seed</span> <span class="trait">coat,</span> as methylation of the B-ring of anthocyanins has a reddening effect on flower <span class="trait">color</span> (reviewed by [ <xref ref-type="bibr" rid="B35"> 35 </xref> ])
					</p>
				
					<p>&nbsp;&nbsp;
						A Mob1/phocein gene was commonly upregulated in the black ( <italic> iRT </italic> ) soybean <span class="trait">seed</span> <span class="trait">coat</span> and Arabidopsis seedlings overexpressing <span class="gene">PAP1</span> [ <xref ref-type="bibr" rid="B21"> 21 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Putative anthocyanin, proanthocyanidin, (iso)flavonoid, and phenylpropanoid isogenes were differentially-expressed between black ( <italic> iRT </italic> ) and brown ( <italic> irT </italic> ) <span class="trait">seed</span> coats, and <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">OMT5</span> </italic> were validated to code UDP-glycose:flavonoid-3- <italic> O </italic> -glycosyltransferase and anthocyanin 3'- <italic> O </italic> -methyltransferase proteins <italic> in vitro </italic> , respectively
					</p>
				
					<p>&nbsp;&nbsp;
						Full-length CDSs for <span class="gene">UGT78K2</span> and <span class="gene">OMT5</span> were amplified from black ( <italic> iRT </italic> ) <span class="trait">seed</span> <span class="trait">coat</span> cDNA by end-to-end PCR using primers UHF/UAR and HO5F/HO5R, respectively (Additional file <xref ref-type="supplementary-material" rid="S12"> 12 </xref> : Table S7) and a high-fidelity DNA polymerase (Invitrogen)
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis of genes controlling natural variation of <span class="trait">seed</span> <span class="trait">coat</span> and flower colors in soybean.</h2>
			<h3>PMID:20584753  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yang K(1), Jeong N, Moon JK, Lee YH, Lee SH, Kim HM, Hwang CH, Back K, Palmer RG, Jeong SC.
				<br/> 
				年份：2010 Nov-Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Among the <span class="trait">seed</span> <span class="trait">coat</span> color-controlling loci, the genetic location of the gene encoding for <span class="gene">W1</span> was substantiated in the context of the current soybean molecular genetic map and O was postulated to correspond to anthocyanidin reductase.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL analysis of seed-flooding tolerance in soybean (Glycine max [L.] Merr.).</h2>
			<h3>PMID:26493141</h3>
				

				
			<p>
				作者：Sayama T(1), Nakazaki T(1), Ishikawa G(1), Yagasaki K(2), Yamada N(2), Hirota N(1), Hirata K(1), Yoshikawa T(1), Saito H(1), Teraishi M(1), Okumoto Y(1), Tsukiyama T(1), Tanisaka T(3).
				<br/> 
				年份：2009 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Among them, Sft1 on the linkage group H (LG_H) exhibited a large effect on <span class="gene">GR</span> after a 24-h treatment; however, Sft2 near the I locus on LG_A2 involved in <span class="trait">seed</span> <span class="trait">coat</span> pigmentation exhibited the largest effect on seed-flooding tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional analysis of soybean genes involved in flavonoid biosynthesis by virus-induced gene silencing.</h2>
			<h3>PMID:17764520  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nagamatsu A(1), Masuta C, Senda M, Matsuura H, Kasai A, Hong JS, Kitamura K, Abe J, Kanazawa A.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						When soybean plants with brown <span class="trait">seed</span> coats were infected with a virus that contains the <span class="gene">CHS</span> gene sequence, the colour of the <span class="trait">seed</span> coats changed to yellow, which indicates that the naturally occurring RNA silencing is reproduced by VIGS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Structural features of <span class="gene">GmIRCHS,</span> candidate of the I gene inhibiting <span class="trait">seed</span> <span class="trait">coat</span> pigmentation in soybean: implications for inducing endogenous RNA silencing of chalcone synthase genes.</h2>
			<h3>PMID:17497082  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kasai A(1), Kasai K, Yumoto S, Senda M.
				<br/> 
				年份：2007 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cloning of the pleiotropic T locus in soybean and two recessive alleles that differentially affect structure and expression of the encoded flavonoid 3' hydroxylase.</h2>
			<h3>PMID:12586717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zabala G(1), Vodkin L.
				<br/> 
				年份：2003 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This <span class="gene">GmF3'H</span> gene was highly expressed in early stages of <span class="trait">seed</span> <span class="trait">coat</span> development and was expressed at very low levels or not at all in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmF3'H</span> genomic and cDNA sequence analysis of <span class="trait">color</span> mutant lines with varying t alleles revealed a frameshift mutation in one of the alleles.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of the duplicated <span class="gene">CHS1</span> gene related to the suppression of the <span class="trait">seed</span> <span class="trait">coat</span> pigmentation in yellow soybeans.</h2>
			<h3>PMID:12582616</h3>
				

				
			<p>
				作者：Senda M(1), Jumonji A, Yumoto S, Ishikawa R, Harada T, Niizeki M, Akada S.
				<br/> 
				年份：2002 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In the hypocotyls of Miyagi shirome, the cDNA derived from <span class="gene">ICHS1</span> mRNA was identified by reverse transcriptase-polymerase chain reaction (RT-PCR) analysis, whereas in the immature <span class="trait">seed</span> coats it was suggested that the amount of transcripts from <span class="gene">ICHS1</span> and/or another type of <span class="gene">CHS1</span> ( <span class="gene">CHS1</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Sequence divergence at chalcone synthase gene in pigmented <span class="trait">seed</span> <span class="trait">coat</span> soybean mutants of the Inhibitor locus.</h2>
			<h3>PMID:12441645  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Senda M(1), Kasai A, Yumoto S, Akada S, Ishikawa R, Harada T, Niizeki M.
				<br/> 
				年份：2002 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Miyagi shirome), a truncated form of the <span class="gene">CHS</span> gene (CHS3) is located in an inverse orientation 680 bp upstream of ICHS1, and it was previously suggested that the truncated CHS3- <span class="gene">ICHS1</span> cluster might be involved in <span class="gene">CHS</span> gene silencing in the <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						In the current study, the truncated CHS3- <span class="gene">ICHS1</span> cluster was compared with the corresponding region of pigmented <span class="trait">seed</span> <span class="trait">coat</span> mutants in which I had changed to i in Miyagi shirome and in the strain Karikei 584.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A single-base deletion in soybean flavonoid 3'-hydroxylase gene is associated with gray pubescence <span class="trait">color.</span></h2>
			<h3>PMID:12175012  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Toda K(1), Yang D, Yamanaka N, Watanabe S, Harada K, Takahashi R.
				<br/> 
				年份：2002 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genomic Southern analysis probed by <span class="gene">sf3'h1</span> revealed restriction fragment length polymorphisms between cultivars with different pubescence <span class="trait">color</span>.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed <span class="trait">Coat</span> Cracking in Soybean Isolines for Pubescence <span class="trait">Color</span> and Maturity.</h2>
			<h3>PMID:11756256</h3>
				

				
			<p>
				作者：Yang D(1), Nakamura T, Ohtsubo N, Takahashi K, Abe J, Takahashi R.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies revealed that the T gene responsible for pubescence <span class="trait">color,</span> and the maturity genes, <span class="gene">E1</span> and E5, had inhibitory effects on low-temperature induced <span class="trait">seed</span> <span class="trait">coat</span> cracking.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A defective <span class="trait">seed</span> <span class="trait">coat</span> pattern (Net) is correlated with the post-transcriptional abundance of soluble proline-rich cell wall proteins.</h2>
			<h3>PMID:10480384  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Percy JD(1), Philip R, Vodkin LO.
				<br/> 
				年份：1999 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						No soluble <span class="gene">PRP1</span> polypeptides were detectable in pigmented <span class="trait">seed</span> coats (i T genotypes) of isolines that also display a net-like pattern of <span class="trait">seed</span> <span class="trait">coat</span> cracking, known as the Net defect.
					</p>
				
					<p>&nbsp;&nbsp;
						However, both <span class="gene">PRP1</span> and <span class="gene">PRP2</span> cytoplasmic mRNAs were found in the Net-defective <span class="trait">seed</span> coats.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Duplications That Suppress and Deletions That Restore Expression from a Chalcone Synthase Multigene Family.</h2>
			<h3>PMID:12239396</h3>
				

				
			<p>
				作者：Todd JJ(1), Vodkin LO.
				<br/> 
				年份：1996 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Paradoxically, deletions of <span class="gene">CHS</span> gene sequences allow higher levels of <span class="gene">CHS</span> mRNAs and restore pigmentation to the <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, <span class="gene">CHS</span> gene-specific expression in <span class="trait">seed</span> coats shows that multiple <span class="gene">CHS</span> genes are expressed in <span class="trait">seed</span> coats.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation of proline rich cell wall proteins in soybean lines with anthocyanin mutations.</h2>
			<h3>PMID:8425044  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nicholas CD(1), Lindstrom JT, Vodkin LO.
				<br/> 
				年份：1993 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Saline-soluble <span class="gene">PRP1</span> is abundant in the developing <span class="trait">seed</span> coats of cultivar Richland (homozygous I, yellow), while it is significantly decreased in the pigmented isogenic mutant T157 (homozygous i, imperfect black).
					</p>
				
					<p>&nbsp;&nbsp;
						In this report, we examined soluble <span class="gene">PRP1</span> in several cultivars containing alleles of the I locus which affect spatial distribution of pigmentation in the <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						We also characterized <span class="gene">PRP1</span> in isolines with allelic variants of several other loci involved in <span class="trait">seed</span> <span class="trait">coat</span> pigmentation, including T and Im.
					</p>
				
					<p>&nbsp;&nbsp;
						Soluble <span class="gene">PRP1</span> was abundant in the developing <span class="trait">seed</span> coats of lines with yellow <span class="trait">seed</span> (I or ii alleles) regardless of pubescence <span class="trait">color,</span> just as in Richland.
					</p>
				
					<p>&nbsp;&nbsp;
						Likewise, soluble <span class="gene">PRP1</span> was decreased in pigmented <span class="trait">seed</span> coats (ik or i alleles) with grey (t) pubescence, as in T157.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean cell wall protein is affected by <span class="trait">seed</span> <span class="trait">color</span> genotype.</h2>
			<h3>PMID:1841717  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lindstrom JT(1), Vodkin LO.
				<br/> 
				年份：1991 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The levels of soluble <span class="gene">PRP1</span> polypeptides and <span class="gene">PRP1</span> mRNA were reduced in young <span class="trait">seed</span> coats with the recessive i/i genotype.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Transition from Primary siRNAs to Amplified Secondary siRNAs That Regulate Chalcone Synthase During Development of <italic> Glycine max </italic> <span class="trait">Seed</span> Coats</h2>
			<h3>PMID:24204712</h3>
				

				
					<h3>PMCID:3804491</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Cho </surname>  <given-names> Young B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jones </surname>  <given-names> Sarah I. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> I </italic> locus is a 27-kb inverted repeat cluster of chalcone synthase genes <italic> CHS1-3-4 </italic> that mediates siRNA down-regulation of <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> target mRNAs during <span class="trait">seed</span> development leading to yellow <span class="trait">seed</span> coats lacking anthocyanin pigments
					</p>
				
					<p>&nbsp;&nbsp;
						More intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did the mid-maturation <span class="trait">seed</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Further, we demonstrate that the soybean system exhibits tissue-specific <italic> <span class="gene">CHS</span> </italic> siRNA production because primary <italic> <span class="gene">CHS</span> </italic> siRNA levels are not sufficient to trigger secondary amplification in tissues other than the <span class="trait">seed</span> <span class="trait">coat</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> I </italic> locus was first identified as a region of duplicated and inverted <italic> <span class="gene">CHS</span> </italic> genes by DNA blotting and PCR mapping of 15 pairs of isogenic lines which showed that mutations of a dominant allele ( <italic> I </italic> or <italic> i <sup> i </sup> </italic> ) to the recessive allele ( <italic> i </italic> ) delete promoter sequences, paradoxically resulting in increased total <italic> <span class="gene">CHS</span> </italic> transcript levels and thereby pigmented (black) <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						The relative expression profiles of <italic> <span class="gene">CHS</span> </italic> gene family members were examined by quantitative RT-PCR in the <span class="trait">seed</span> coats of two near-isogenic pairs that result from independently occurring mutations of the dominant <italic> I </italic> allele to the recessive <italic> i </italic> allele or of the dominant <italic> i <sup> i </sup> </italic> allele to the recessive <italic> i </italic> allele <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Both small RNA blots and high throughput small RNA sequencing from three genotypes ( <italic> I, i <sup> i </sup> </italic> and <italic> i </italic> alleles) of soybeans revealed that <italic> <span class="gene">CHS</span> </italic> siRNAs accumulated only in the yellow <span class="trait">seed</span> coats containing either dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and not in the pigmented <span class="trait">seed</span> coats with homozygous recessive <italic> i </italic> genotypes <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs did not accumulate in the cotyledons of the genotype with dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and yellow <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						On the other hand, <italic> CHS7/8 </italic> mRNAs are highly expressed in the pigmented <span class="trait">seed</span> coats in which <italic> <span class="gene">CHS</span> </italic> siRNA production has been abolished by deletions in the <italic> <span class="gene">CHS</span> </italic> cluster regions in the mutant <italic> i </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						We also demonstrate that <italic> <span class="gene">CHS</span> </italic> siRNA production is specific to the <span class="trait">seed</span> coats as other tissues including cotyledons, roots, leaves, and stems do not produce sufficient primary siRNAs for amplification to secondary siRNAs
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the pattern of <italic> <span class="gene">CHS</span> </italic> siRNAs during the entire span of <span class="trait">seed</span> development, we constructed small RNA libraries from ten stages of <span class="trait">seed</span> development of the cultivar Williams ( <italic> i <sup> i </sup> </italic> ) as shown in <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> shows the normalized total counts in reads per million (RPM) of <italic> <span class="gene">CHS</span> </italic> siRNAs that map with 100% identity to each of the nine <italic> <span class="gene">CHS</span> </italic> genes during <span class="trait">seed</span> development
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized <italic> <span class="gene">CHS</span> </italic> siRNA counts were initially low in the 4 DAF <span class="trait">seed</span> at less than 100 RPM and also in the 12&#x02013;14 DAF and 22&#x02013;24 DAF <span class="trait">seed</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Then, they increased to about 900 RPM in <span class="trait">seed</span> coats from immature <span class="trait">seed</span> of 5&#x02013;6 mg fresh weight, after which they increased dramatically, reaching the highest levels of 8600 for <italic> <span class="gene">CHS7</span> </italic> in <span class="trait">seed</span> coats from 50&#x02013;75 mg immature <span class="trait">seed</span>
					</p>
				
					<p>&nbsp;&nbsp;
						After this peak, the <italic> <span class="gene">CHS</span> </italic> siRNAs decreased as the <span class="trait">seed</span> increased in size and began desiccation
					</p>
				
					<p>&nbsp;&nbsp;
						These data demonstrate that the amplification from primary to secondary <italic> <span class="gene">CHS</span> </italic> siRNAs occurs during <span class="trait">seed</span> <span class="trait">coat</span> development
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart
					</p>
				
					<p>&nbsp;&nbsp;
						A comparison of total counts of 21-nt and 22-nt primary <italic> CHS4 </italic> siRNAs and 21-nt secondary <italic> <span class="gene">CHS7</span> </italic> siRNAs that are unique to either <italic> CHS4 </italic> or <italic> <span class="gene">CHS7</span> </italic> shows that 21-nt secondary siRNAs were highly amplified during <span class="trait">seed</span> <span class="trait">coat</span> development compared to those of primary <italic> <span class="gene">CHS</span> </italic> siRNAs ( <xref ref-type="fig" rid="pone-0076954-g003"> Figure 3 </xref> and <xref ref-type="supplementary-material" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						In the soybean <span class="trait">seed</span> <span class="trait">coat,</span> we demonstrate that this amplification started at an early developmental stage around 5&#x02013;6 mg ( <xref ref-type="fig" rid="pone-0076954-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> ) likely at the stage when <italic> <span class="gene">CHS7</span> </italic> / <italic> 8 </italic> mRNAs begin to be expressed in the <span class="trait">seed</span> coats at higher levels
					</p>
				
					<p>&nbsp;&nbsp;
						Using mRNA blots and small RNA blots, we have previously shown that silencing of <italic> <span class="gene">CHS</span> </italic> gene family members occurs only in the <span class="trait">seed</span> coats and not in other organs <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> demonstrates the tissue specificity clearly as <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> coats in the Williams <italic> i <sup> i </sup> </italic> cultivar
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts of <italic> <span class="gene">CHS7</span> </italic> siRNAs in <span class="trait">seed</span> coats from 5&#x02013;6 mg is 925 (RPM)
					</p>
				
					<p>&nbsp;&nbsp;
						Most of those tissues (immature cotyledon, germinated cotyledon, unifoliate, root, shoot tip, and stem) have more <italic> CHS4 </italic> siRNAs than <italic> <span class="gene">CHS7</span> </italic> siRNAs; however the counts of <italic> CHS4 </italic> siRNAs are very low compared to those of <span class="trait">seed</span> <span class="trait">coat</span> tissues ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						It is interesting that <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> <span class="trait">coat</span> and were not in immature cotyledons of three different stages of development, 5&#x02013;6 mg, 50&#x02013;75 mg and 100&#x02013;300 mg fresh <span class="trait">seed</span> weight
					</p>
				
					<p>&nbsp;&nbsp;
						Because of the small size of the <span class="trait">seed</span> prior to the 5&#x02013;6 mg stage, we used whole <span class="trait">seed</span> through 24 DAF; thus, it is likely also that the low levels of <italic> <span class="gene">CHS</span> </italic> siRNAs from these young whole <span class="trait">seed</span> less than 24 DAF ( <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> ) originated solely from the maternal <span class="trait">seed</span> <span class="trait">coat</span> tissue and not from the very young developing cotyledons since the cotyledons at all succeeding stages did not accumulate <italic> <span class="gene">CHS</span> </italic> siRNAs at significant levels as shown in <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref>
					</p>
				
					<p>&nbsp;&nbsp;
						The transition from predominantly 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in blue) representing the origin of the <italic> CHS1-3-4 </italic> siRNAs to the 21-nt secondary <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in red) representing the target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs occurs at the 5&#x02013;6 mg <span class="trait">seed</span> stage as indicated
					</p>
				
					<p>&nbsp;&nbsp;
						In this report, we show that the unique attributes of the naturally occurring soybean <italic> I </italic> locus clearly revealed the amplification of the <italic> <span class="gene">CHS</span> </italic> siRNA population from primary to secondary siRNAs during <span class="trait">seed</span> development based on the polymorphisms between the siRNAs representing the origin ( <italic> CHS1-3-4 </italic> ) and target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes
					</p>
				
					<p>&nbsp;&nbsp;
						Here, by examining the entire <span class="trait">seed</span> developmental period including the very early stages from a few days post fertilization that are labor intensive to obtain in sufficient quantity for sequencing, we demonstrated the transition from a small population of primary <italic> <span class="gene">CHS</span> </italic> siRNAs to the accumulation of high levels of 21-nt secondary siRNAs ( <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> , <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> , <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> and Figures S1, S2, S3)
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> at 4 DAF contained a basal level of less than 100 RPM that aligned to any <italic> <span class="gene">CHS</span> </italic> gene
					</p>
				
					<p>&nbsp;&nbsp;
						The levels of <italic> <span class="gene">CHS</span> </italic> siRNAs originating from the <italic> CHS1-3-4 </italic> clusters increased thereafter to roughly 400&#x02013;900 RPM very early in <span class="trait">seed</span> development from 12&#x02013;24 DAF to 5&#x02013;6 mg
					</p>
				
					<p>&nbsp;&nbsp;
						By the 5&#x02013;6 mg stage, the <span class="trait">seed</span> coats contained roughly even proportions of siRNAs representing <italic> CHS1-3-4 </italic> compared to <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> but after that period, the <italic> CHS7/8 </italic> siRNAs increased dramatically, reaching peak levels of 8,000 to 10,000 RPMs in the biological repeats of the 50&#x02013;75 mg <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						Intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did <span class="trait">seed</span> from the 5&#x02013;6 mg stage and older
					</p>
				
					<p>&nbsp;&nbsp;
						Whether or not the 22-nt siRNAs are the only trigger to generate the 21-nt secondary siRNAs is unknown, since 21-nt primary <italic> <span class="gene">CHS</span> </italic> siRNAs were also present in the very young <span class="trait">seed</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Previously, it was shown with RNA blots that <italic> <span class="gene">CHS</span> </italic> mRNAs are prevalent at 10&#x02013;25 mg, 25&#x02013;50 mg and 50&#x02013;75 mg weight ranges of the <span class="trait">seed</span> coats of mid-maturation embryos with pigmented genotypes <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Also, using TaqMan real time RT-PCR, it was shown that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs are highest at the 50&#x02013;75 mg stage of Williams 55 pigmented <span class="trait">seed</span> coats with the homozygous recessive <italic> i </italic> genotype
					</p>
				
					<p>&nbsp;&nbsp;
						In this regard, the profile of the <italic> <span class="gene">CHS</span> </italic> siRNAs resembled many other genes expressed during <span class="trait">seed</span> <span class="trait">coat</span> development, including many in the isoflavone and anthocyanin pathways <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						The most abundant levels of <italic> <span class="gene">CHS</span> </italic> siRNAs in the <italic> i <sup> i </sup> </italic> genotype with yellow <span class="trait">seed</span> coats generally coincided with the developmental time of appearance of the <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> target mRNAs in the recessive <italic> i </italic> genotype that has pigmented <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, in contrast with the down-regulation of the pathway by <italic> <span class="gene">CHS</span> </italic> siRNA-targeted destruction of <italic> <span class="gene">CHS</span> </italic> mRNAs in the yellow <span class="trait">seed</span> coats, the <italic> <span class="gene">CHS</span> </italic> transcripts continued to produce <italic> <span class="gene">CHS</span> </italic> isoforms after the 50 mg stage with the resulting accumulation of large amounts of anthocyanins leading to the pigmented <span class="trait">seed</span> <span class="trait">coat</span>
					</p>
				
					<p>&nbsp;&nbsp;
						By examining the entire <span class="trait">seed</span> developmental period including the small <span class="trait">seed</span> at 4 DAF through 24 DAF, we found that the <italic> <span class="gene">CHS</span> </italic> primary siRNAs representing the <italic> CHS1-3-4 </italic> origin clusters began to increase before the secondary siRNAs representing the <italic> CHS7/8 </italic> target mRNAs (summarized in <xref ref-type="fig" rid="pone-0076954-g005"> Figure 5 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Alternatively, (2) the dsRNA could be formed in other tissues but not cleaved properly by a DCL protein to <span class="trait">yield</span> enough primary siRNA to trigger secondary siRNAs, or (3) the primary <italic> <span class="gene">CHS</span> </italic> siRNAs might not be amplified to high levels of secondary siRNAs due to lack of an RdRP or other core components to generate secondary siRNAs in other tissues
					</p>
				
					<p>&nbsp;&nbsp;
						We propose that the level of the <italic> <span class="gene">CHS</span> </italic> primary siRNAs is too low to trigger the secondary amplification in these tissues, unlike the situation in the non-pigmented <span class="trait">seed</span> coats where our data clearly show a rise in the levels of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs to nearly 1000 RPM before the secondary siRNAs overtake them in abundance
					</p>
				
					<p>&nbsp;&nbsp;
						Using RNAse protection experiments, a recent study has detected <italic> <span class="gene">CHS</span> </italic> dsRNA not only in the <span class="trait">seed</span> <span class="trait">coat</span> but also in the cotyledon and leaf tissues of lines with a dominant <italic> I </italic> allele <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						However, this would not explain why <italic> <span class="gene">CHS</span> </italic> dsRNAs that are formed in non-seed <span class="trait">coat</span> tissues would fail to be processed and amplified when many other miRNAs and siRNAs are fully processed and amplified in other tissues such as cotyledons, leaves, and roots of soybean with the <italic> I </italic> and <italic> i <sup> i </sup> </italic> alleles <xref ref-type="bibr" rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						For these reasons, we currently prefer hypothesis (1) that the level of the <italic> <span class="gene">CHS</span> </italic> siRNAs follows an increase in the nascent dsRNA to levels that are higher in the <span class="trait">seed</span> coats than in other tissues
					</p>
				
					<p>&nbsp;&nbsp;
						Whether the limiting factor proves to be the levels of the dsRNA or primary siRNAs in tissues other than the <span class="trait">seed</span> <span class="trait">coat,</span> it is clear that secondary amplification from the target <italic> <span class="gene">CHS</span> </italic> mRNAs does not occur in other tissues as seen by the lack of high levels of 21-nt <italic> <span class="gene">CHS</span> </italic> siRNAs that align to any of the <italic> <span class="gene">CHS</span> </italic> genes ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear that <italic> <span class="gene">CHS</span> </italic> is down-regulated by the <italic> <span class="gene">CHS</span> </italic> siRNAs only in the <span class="trait">seed</span> coats, though sufficient levels of <italic> <span class="gene">CHS</span> </italic> mRNAs exist in other tissues to serve as substrates for amplification of secondary siRNAs to high levels
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, because of the unique properties of the soybean silencing system, we were able to reveal that the onset of production of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs in the very early <span class="trait">seed</span> stages contained a higher proportion of 22-nt siRNAs originating from the <italic> CHS1-3-4 </italic> clusters of the <italic> i <sup> i </sup> </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						Our results demonstrate regulation of the <italic> <span class="gene">CHS</span> </italic> inverted repeat genomic region is likely to rely on factors that occur during the very early <span class="trait">seed</span> developmental program and may reflect such changes
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs for Each </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> Gene Reveal Biogenesis of </bold> <bold> <italic> CHS7/8 </italic> </bold> <bold> siRNAs is Dramatically Increased during <span class="trait">Seed</span> <span class="trait">Coat</span> Development in Biological Repeats of Eight Stages
					</p>
				
					<p>&nbsp;&nbsp;
						</bold> <italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Size Distributions of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Normalized Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs that Match Either </bold> <bold> <italic> CHS4 </italic> </bold> <bold> or </bold> <bold> <italic> <span class="gene">CHS7</span> </italic> </bold> <bold> with 100% Identity in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						More intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did the mid-maturation <span class="trait">seed</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Further, we demonstrate that the soybean system exhibits tissue-specific <italic> <span class="gene">CHS</span> </italic> siRNA production because primary <italic> <span class="gene">CHS</span> </italic> siRNA levels are not sufficient to trigger secondary amplification in tissues other than the <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> I </italic> locus was first identified as a region of duplicated and inverted <italic> <span class="gene">CHS</span> </italic> genes by DNA blotting and PCR mapping of 15 pairs of isogenic lines which showed that mutations of a dominant allele ( <italic> I </italic> or <italic> i <sup> i </sup> </italic> ) to the recessive allele ( <italic> i </italic> ) delete promoter sequences, paradoxically resulting in increased total <italic> <span class="gene">CHS</span> </italic> transcript levels and thereby pigmented (black) <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						The relative expression profiles of <italic> <span class="gene">CHS</span> </italic> gene family members were examined by quantitative RT-PCR in the <span class="trait">seed</span> coats of two near-isogenic pairs that result from independently occurring mutations of the dominant <italic> I </italic> allele to the recessive <italic> i </italic> allele or of the dominant <italic> i <sup> i </sup> </italic> allele to the recessive <italic> i </italic> allele <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Both small RNA blots and high throughput small RNA sequencing from three genotypes ( <italic> I, i <sup> i </sup> </italic> and <italic> i </italic> alleles) of soybeans revealed that <italic> <span class="gene">CHS</span> </italic> siRNAs accumulated only in the yellow <span class="trait">seed</span> coats containing either dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and not in the pigmented <span class="trait">seed</span> coats with homozygous recessive <italic> i </italic> genotypes <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs did not accumulate in the cotyledons of the genotype with dominant <italic> I </italic> or <italic> i <sup> i </sup> </italic> alleles and yellow <span class="trait">seed</span> coats <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						On the other hand, <italic> CHS7/8 </italic> mRNAs are highly expressed in the pigmented <span class="trait">seed</span> coats in which <italic> <span class="gene">CHS</span> </italic> siRNA production has been abolished by deletions in the <italic> <span class="gene">CHS</span> </italic> cluster regions in the mutant <italic> i </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						We also demonstrate that <italic> <span class="gene">CHS</span> </italic> siRNA production is specific to the <span class="trait">seed</span> coats as other tissues including cotyledons, roots, leaves, and stems do not produce sufficient primary siRNAs for amplification to secondary siRNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine the pattern of <italic> <span class="gene">CHS</span> </italic> siRNAs during the entire span of <span class="trait">seed</span> development, we constructed small RNA libraries from ten stages of <span class="trait">seed</span> development of the cultivar Williams ( <italic> i <sup> i </sup> </italic> ) as shown in <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> shows the normalized total counts in reads per million (RPM) of <italic> <span class="gene">CHS</span> </italic> siRNAs that map with 100% identity to each of the nine <italic> <span class="gene">CHS</span> </italic> genes during <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized <italic> <span class="gene">CHS</span> </italic> siRNA counts were initially low in the 4 DAF <span class="trait">seed</span> at less than 100 RPM and also in the 12&#x02013;14 DAF and 22&#x02013;24 DAF <span class="trait">seed</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Then, they increased to about 900 RPM in <span class="trait">seed</span> coats from immature <span class="trait">seed</span> of 5&#x02013;6 mg fresh weight, after which they increased dramatically, reaching the highest levels of 8600 for <italic> <span class="gene">CHS7</span> </italic> in <span class="trait">seed</span> coats from 50&#x02013;75 mg immature <span class="trait">seed</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						After this peak, the <italic> <span class="gene">CHS</span> </italic> siRNAs decreased as the <span class="trait">seed</span> increased in size and began desiccation.
					</p>
				
					<p>&nbsp;&nbsp;
						These data demonstrate that the amplification from primary to secondary <italic> <span class="gene">CHS</span> </italic> siRNAs occurs during <span class="trait">seed</span> <span class="trait">coat</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						A comparison of total counts of 21-nt and 22-nt primary <italic> CHS4 </italic> siRNAs and 21-nt secondary <italic> <span class="gene">CHS7</span> </italic> siRNAs that are unique to either <italic> CHS4 </italic> or <italic> <span class="gene">CHS7</span> </italic> shows that 21-nt secondary siRNAs were highly amplified during <span class="trait">seed</span> <span class="trait">coat</span> development compared to those of primary <italic> <span class="gene">CHS</span> </italic> siRNAs ( <xref ref-type="fig" rid="pone-0076954-g003"> Figure 3 </xref> and <xref ref-type="supplementary-material" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						In the soybean <span class="trait">seed</span> <span class="trait">coat,</span> we demonstrate that this amplification started at an early developmental stage around 5&#x02013;6 mg ( <xref ref-type="fig" rid="pone-0076954-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> ) likely at the stage when <italic> <span class="gene">CHS7</span> </italic> / <italic> 8 </italic> mRNAs begin to be expressed in the <span class="trait">seed</span> coats at higher levels.
					</p>
				
					<p>&nbsp;&nbsp;
						Using mRNA blots and small RNA blots, we have previously shown that silencing of <italic> <span class="gene">CHS</span> </italic> gene family members occurs only in the <span class="trait">seed</span> coats and not in other organs <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> demonstrates the tissue specificity clearly as <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> coats in the Williams <italic> i <sup> i </sup> </italic> cultivar.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts of <italic> <span class="gene">CHS7</span> </italic> siRNAs in <span class="trait">seed</span> coats from 5&#x02013;6 mg is 925 (RPM).
					</p>
				
					<p>&nbsp;&nbsp;
						Most of those tissues (immature cotyledon, germinated cotyledon, unifoliate, root, shoot tip, and stem) have more <italic> CHS4 </italic> siRNAs than <italic> <span class="gene">CHS7</span> </italic> siRNAs; however the counts of <italic> CHS4 </italic> siRNAs are very low compared to those of <span class="trait">seed</span> <span class="trait">coat</span> tissues ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						It is interesting that <italic> <span class="gene">CHS</span> </italic> siRNAs were limited to the <span class="trait">seed</span> <span class="trait">coat</span> and were not in immature cotyledons of three different stages of development, 5&#x02013;6 mg, 50&#x02013;75 mg and 100&#x02013;300 mg fresh <span class="trait">seed</span> weight.
					</p>
				
					<p>&nbsp;&nbsp;
						Because of the small size of the <span class="trait">seed</span> prior to the 5&#x02013;6 mg stage, we used whole <span class="trait">seed</span> through 24 DAF; thus, it is likely also that the low levels of <italic> <span class="gene">CHS</span> </italic> siRNAs from these young whole <span class="trait">seed</span> less than 24 DAF ( <xref ref-type="fig" rid="pone-0076954-g001"> Figure 1 </xref> ) originated solely from the maternal <span class="trait">seed</span> <span class="trait">coat</span> tissue and not from the very young developing cotyledons since the cotyledons at all succeeding stages did not accumulate <italic> <span class="gene">CHS</span> </italic> siRNAs at significant levels as shown in <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref>.
					</p>
				
					<p>&nbsp;&nbsp;
						The transition from predominantly 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in blue) representing the origin of the <italic> CHS1-3-4 </italic> siRNAs to the 21-nt secondary <italic> <span class="gene">CHS</span> </italic> siRNAs (shown in red) representing the target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs occurs at the 5&#x02013;6 mg <span class="trait">seed</span> stage as indicated.
					</p>
				
					<p>&nbsp;&nbsp;
						In this report, we show that the unique attributes of the naturally occurring soybean <italic> I </italic> locus clearly revealed the amplification of the <italic> <span class="gene">CHS</span> </italic> siRNA population from primary to secondary siRNAs during <span class="trait">seed</span> development based on the polymorphisms between the siRNAs representing the origin ( <italic> CHS1-3-4 </italic> ) and target <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, by examining the entire <span class="trait">seed</span> developmental period including the very early stages from a few days post fertilization that are labor intensive to obtain in sufficient quantity for sequencing, we demonstrated the transition from a small population of primary <italic> <span class="gene">CHS</span> </italic> siRNAs to the accumulation of high levels of 21-nt secondary siRNAs ( <xref ref-type="fig" rid="pone-0076954-g001"> Figures 1 </xref> , <xref ref-type="fig" rid="pone-0076954-g002"> 2 </xref> , <xref ref-type="fig" rid="pone-0076954-g003"> 3 </xref> and Figures S1, S2, S3).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> at 4 DAF contained a basal level of less than 100 RPM that aligned to any <italic> <span class="gene">CHS</span> </italic> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						The levels of <italic> <span class="gene">CHS</span> </italic> siRNAs originating from the <italic> CHS1-3-4 </italic> clusters increased thereafter to roughly 400&#x02013;900 RPM very early in <span class="trait">seed</span> development from 12&#x02013;24 DAF to 5&#x02013;6 mg.
					</p>
				
					<p>&nbsp;&nbsp;
						By the 5&#x02013;6 mg stage, the <span class="trait">seed</span> coats contained roughly even proportions of siRNAs representing <italic> CHS1-3-4 </italic> compared to <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> but after that period, the <italic> CHS7/8 </italic> siRNAs increased dramatically, reaching peak levels of 8,000 to 10,000 RPMs in the biological repeats of the 50&#x02013;75 mg <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						Intriguingly, the very young <span class="trait">seed</span> had a higher proportion of 22-nt <italic> <span class="gene">CHS</span> </italic> siRNAs than did <span class="trait">seed</span> from the 5&#x02013;6 mg stage and older.
					</p>
				
					<p>&nbsp;&nbsp;
						Whether or not the 22-nt siRNAs are the only trigger to generate the 21-nt secondary siRNAs is unknown, since 21-nt primary <italic> <span class="gene">CHS</span> </italic> siRNAs were also present in the very young <span class="trait">seed</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Previously, it was shown with RNA blots that <italic> <span class="gene">CHS</span> </italic> mRNAs are prevalent at 10&#x02013;25 mg, 25&#x02013;50 mg and 50&#x02013;75 mg weight ranges of the <span class="trait">seed</span> coats of mid-maturation embryos with pigmented genotypes <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Also, using TaqMan real time RT-PCR, it was shown that <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> mRNAs are highest at the 50&#x02013;75 mg stage of Williams 55 pigmented <span class="trait">seed</span> coats with the homozygous recessive <italic> i </italic> genotype.
					</p>
				
					<p>&nbsp;&nbsp;
						In this regard, the profile of the <italic> <span class="gene">CHS</span> </italic> siRNAs resembled many other genes expressed during <span class="trait">seed</span> <span class="trait">coat</span> development, including many in the isoflavone and anthocyanin pathways <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						The most abundant levels of <italic> <span class="gene">CHS</span> </italic> siRNAs in the <italic> i <sup> i </sup> </italic> genotype with yellow <span class="trait">seed</span> coats generally coincided with the developmental time of appearance of the <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> target mRNAs in the recessive <italic> i </italic> genotype that has pigmented <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						Thus, in contrast with the down-regulation of the pathway by <italic> <span class="gene">CHS</span> </italic> siRNA-targeted destruction of <italic> <span class="gene">CHS</span> </italic> mRNAs in the yellow <span class="trait">seed</span> coats, the <italic> <span class="gene">CHS</span> </italic> transcripts continued to produce <italic> <span class="gene">CHS</span> </italic> isoforms after the 50 mg stage with the resulting accumulation of large amounts of anthocyanins leading to the pigmented <span class="trait">seed</span> <span class="trait">coat</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						By examining the entire <span class="trait">seed</span> developmental period including the small <span class="trait">seed</span> at 4 DAF through 24 DAF, we found that the <italic> <span class="gene">CHS</span> </italic> primary siRNAs representing the <italic> CHS1-3-4 </italic> origin clusters began to increase before the secondary siRNAs representing the <italic> CHS7/8 </italic> target mRNAs (summarized in <xref ref-type="fig" rid="pone-0076954-g005"> Figure 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Alternatively, (2) the dsRNA could be formed in other tissues but not cleaved properly by a DCL protein to <span class="trait">yield</span> enough primary siRNA to trigger secondary siRNAs, or (3) the primary <italic> <span class="gene">CHS</span> </italic> siRNAs might not be amplified to high levels of secondary siRNAs due to lack of an RdRP or other core components to generate secondary siRNAs in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						We propose that the level of the <italic> <span class="gene">CHS</span> </italic> primary siRNAs is too low to trigger the secondary amplification in these tissues, unlike the situation in the non-pigmented <span class="trait">seed</span> coats where our data clearly show a rise in the levels of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs to nearly 1000 RPM before the secondary siRNAs overtake them in abundance.
					</p>
				
					<p>&nbsp;&nbsp;
						Using RNAse protection experiments, a recent study has detected <italic> <span class="gene">CHS</span> </italic> dsRNA not only in the <span class="trait">seed</span> <span class="trait">coat</span> but also in the cotyledon and leaf tissues of lines with a dominant <italic> I </italic> allele <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						However, this would not explain why <italic> <span class="gene">CHS</span> </italic> dsRNAs that are formed in non-seed <span class="trait">coat</span> tissues would fail to be processed and amplified when many other miRNAs and siRNAs are fully processed and amplified in other tissues such as cotyledons, leaves, and roots of soybean with the <italic> I </italic> and <italic> i <sup> i </sup> </italic> alleles <xref ref-type="bibr" rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						For these reasons, we currently prefer hypothesis (1) that the level of the <italic> <span class="gene">CHS</span> </italic> siRNAs follows an increase in the nascent dsRNA to levels that are higher in the <span class="trait">seed</span> coats than in other tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						Whether the limiting factor proves to be the levels of the dsRNA or primary siRNAs in tissues other than the <span class="trait">seed</span> <span class="trait">coat,</span> it is clear that secondary amplification from the target <italic> <span class="gene">CHS</span> </italic> mRNAs does not occur in other tissues as seen by the lack of high levels of 21-nt <italic> <span class="gene">CHS</span> </italic> siRNAs that align to any of the <italic> <span class="gene">CHS</span> </italic> genes ( <xref ref-type="fig" rid="pone-0076954-g004"> Figure 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear that <italic> <span class="gene">CHS</span> </italic> is down-regulated by the <italic> <span class="gene">CHS</span> </italic> siRNAs only in the <span class="trait">seed</span> coats, though sufficient levels of <italic> <span class="gene">CHS</span> </italic> mRNAs exist in other tissues to serve as substrates for amplification of secondary siRNAs to high levels.
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, because of the unique properties of the soybean silencing system, we were able to reveal that the onset of production of the primary <italic> <span class="gene">CHS</span> </italic> siRNAs in the very early <span class="trait">seed</span> stages contained a higher proportion of 22-nt siRNAs originating from the <italic> CHS1-3-4 </italic> clusters of the <italic> i <sup> i </sup> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results demonstrate regulation of the <italic> <span class="gene">CHS</span> </italic> inverted repeat genomic region is likely to rely on factors that occur during the very early <span class="trait">seed</span> developmental program and may reflect such changes.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> The Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs for Each </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> Gene Reveal Biogenesis of </bold> <bold> <italic> CHS7/8 </italic> </bold> <bold> siRNAs is Dramatically Increased during <span class="trait">Seed</span> <span class="trait">Coat</span> Development in Biological Repeats of Eight Stages.
					</p>
				
					<p>&nbsp;&nbsp;
						</bold> <italic> <span class="gene">CHS</span> </italic> siRNAs from small RNA libraries of ten developmental stages of the cultivar Williams <italic> i <sup> i </sup> </italic> were filtered to identify those with 100% identity to individual <italic> <span class="gene">CHS</span> </italic> genes as indicated by the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Size Distributions of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development.
					</p>
				
					<p>&nbsp;&nbsp;
						The normalized total counts in reads per million (RPM) are shown for each <italic> <span class="gene">CHS</span> </italic> gene according to the <span class="trait">color</span> chart.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Normalized Total Counts of </bold> <bold> <italic> <span class="gene">CHS</span> </italic> </bold> <bold> siRNAs that Match Either </bold> <bold> <italic> CHS4 </italic> </bold> <bold> or </bold> <bold> <italic> <span class="gene">CHS7</span> </italic> </bold> <bold> with 100% Identity in Biological Repeats of Eight Stages of <span class="trait">Seed</span> <span class="trait">Coat</span> Development.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:<italic> In silico </italic> comparison of genomic regions containing genes coding for enzymes and transcription factors for the phenylpropanoid pathway in <italic> Phaseolus vulgaris </italic> L. and <italic> Glycine max </italic> L. Merr</h2>
			<h3>PMID:24046770</h3>
				

				
					<h3>PMCID:3763686</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Reinprecht </surname>  <given-names> Yarmilla </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yadegari </surname>  <given-names> Zeinab </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Perry </surname>  <given-names> Gregory E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Siddiqua </surname>  <given-names> Mahbuba </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wright </surname>  <given-names> Lori C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClean </surname>  <given-names> Phillip E. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pauls </surname>  <given-names> K. Peter </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						For example, the <italic> I </italic> (inhibitor) locus (light, dark or saddled hilum <span class="trait">color)</span> was associated with six inverted chalcone synthase (CHS) repeats on chromosome <span class="gene">Gm08</span> (Todd and Vodkin, <xref ref-type="bibr" rid="B62"> 1996 </xref> ) and the <italic> T </italic> locus (tawny brown or gray pubescence <span class="trait">color)</span> co-localized with flavonoid 3&#x02032; hydroxylase (F3&#x02032;H) on chromosome <span class="gene">Gm06</span> (Toda et al
					</p>
				
					<p>&nbsp;&nbsp;
						However, the <italic> R </italic> locus (black, brown or striped <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color)</span> mapped on chromosome <span class="gene">Gm09</span> could not be linked with known anthocyanidin synthase (ANS), dihydroflavonol 4-reductase (DFR), or UDP-flavonoid glucosyltransferase genes
					</p>
				
					<p>&nbsp;&nbsp;
						RAPD (random amplified polymorphic DNA) and <span class="gene">STS</span> (sequence-tagged sites) markers associated with <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes have been identified and mapped on several linkage groups (Brady et al
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes <italic> I </italic> [factor which inhibits pigment formation in the <span class="trait">seed</span> <span class="trait">coat,</span> associated with six <italic> <span class="gene">CHS</span> </italic> repeats (Tuteja et al
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> Wp </italic> , (purple flower and hypocotyl <span class="trait">color</span> and the production of black pigment in the <span class="trait">seed</span> <span class="trait">coat)</span> associated previously with flavanone 3-hydroxylase (F3H) on chromosome <span class="gene">Gm02</span> (Zabala and Vodkin, <xref ref-type="bibr" rid="B72"> 2005 </xref> ), was mapped to the expected region that was syntenic to common bean <italic> <span class="gene">F3H</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> <span class="gene">W1</span> </italic> is associated with F3&#x02032;5&#x02032;H (Zabala and Vodkin, <xref ref-type="bibr" rid="B73"> 2007 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						In particular, <span class="trait">seed</span> <span class="trait">coat</span> pattern gene <italic> Z </italic> (zonal) mapped close to <italic> <span class="gene">4CL1</span> </italic> gene on chromosome Pv3, <italic> V </italic> gene (violet factor) close to F3&#x02032;5&#x02032;H on Pv6 and <italic> P </italic> gene (pigment ground factor) was placed close to the transcription factor Myb15 on chromosome Pv7
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						However, the <italic> R </italic> locus (black, brown or striped <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color)</span> mapped on chromosome <span class="gene">Gm09</span> could not be linked with known anthocyanidin synthase (ANS), dihydroflavonol 4-reductase (DFR), or UDP-flavonoid glucosyltransferase genes.
					</p>
				
					<p>&nbsp;&nbsp;
						RAPD (random amplified polymorphic DNA) and <span class="gene">STS</span> (sequence-tagged sites) markers associated with <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes have been identified and mapped on several linkage groups (Brady et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> genes <italic> I </italic> [factor which inhibits pigment formation in the <span class="trait">seed</span> <span class="trait">coat,</span> associated with six <italic> <span class="gene">CHS</span> </italic> repeats (Tuteja et al.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> Wp </italic> , (purple flower and hypocotyl <span class="trait">color</span> and the production of black pigment in the <span class="trait">seed</span> <span class="trait">coat)</span> associated previously with flavanone 3-hydroxylase (F3H) on chromosome <span class="gene">Gm02</span> (Zabala and Vodkin, <xref ref-type="bibr" rid="B72"> 2005 </xref> ), was mapped to the expected region that was syntenic to common bean <italic> <span class="gene">F3H</span> </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, the <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> gene <italic> <span class="gene">W1</span> </italic> is associated with F3&#x02032;5&#x02032;H (Zabala and Vodkin, <xref ref-type="bibr" rid="B73"> 2007 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In particular, <span class="trait">seed</span> <span class="trait">coat</span> pattern gene <italic> Z </italic> (zonal) mapped close to <italic> <span class="gene">4CL1</span> </italic> gene on chromosome Pv3, <italic> V </italic> gene (violet factor) close to F3&#x02032;5&#x02032;H on Pv6 and <italic> P </italic> gene (pigment ground factor) was placed close to the transcription factor Myb15 on chromosome Pv7.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Methylation Affects Transposition and Splicing of a Large CACTA Transposon from a MYB Transcription Factor Regulating Anthocyanin Synthase Genes in Soybean <span class="trait">Seed</span> Coats</h2>
			<h3>PMID:25369033</h3>
				

				
					<h3>PMCID:4219821</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Although the MYB encoded by <italic> R </italic> was expressed at only very low levels in older <span class="trait">seed</span> coats of the black revertant RM30- <italic> R* </italic> line, it upregulated expression of anthocyanidin synthase genes ( <italic> ANS2, <span class="gene">ANS3</span> </italic> ) to promote the synthesis of anthocyanins
					</p>
				
					<p>&nbsp;&nbsp;
						In its dominant form <italic> I </italic> , silences the expression of all nine <span class="gene">CHS</span> gene family via short interfering RNAs in a tissue-specific manner in the <span class="trait">seed</span> <span class="trait">coat</span> <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Since <span class="gene">CHS</span> is the first committed enzyme in the anthocyanin pathway, the <italic> I </italic> genotypes result in yellow <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						These authors identified the recessive, brown <italic> r </italic> allele to affect the expression of a <span class="trait">seed</span> coat-specific R2R3 MYB transcription factor gene, <span class="gene">Glyma09g36983</span> from the sequenced soybean genome as a strong candidate for the <italic> R </italic> locus
					</p>
				
					<p>&nbsp;&nbsp;
						Using quantitative RNA-Seq data from a developmental series of <span class="trait">seed</span> coats from the black-seeded RM30- <italic> R </italic> * revertant and the RM38- <italic> r </italic> brown-seeded isoline, we also demonstrate that the MYB factor encoded by the <italic> R </italic> gene appears to induce the expression of anthocyanidin synthase ( <italic> <span class="gene">ANS</span> </italic> ) genes at late stages of <span class="trait">seed</span> development in the black RM30- <italic> R* </italic> line
					</p>
				
					<p>&nbsp;&nbsp;
						We compared the expression <span class="gene">Glyma09g36983</span> MYB transcription factor in the <span class="trait">seed</span> coats of the black revertant RM30- <italic> R* </italic> soybean line to that in the defective brown RM38- <italic> r </italic> soybean line at various stages of <span class="trait">seed</span> development
					</p>
				
					<p>&nbsp;&nbsp;
						(B) Expression of <span class="gene">Glyma09g36983</span> in RPKMs plotted against the five stages of <span class="trait">seed</span> development in mg <span class="trait">seed</span> fresh weight as shown in (A) above
					</p>
				
					<p>&nbsp;&nbsp;
						069) indicates no significant differences of expression of the <span class="gene">Glyma09g36983</span> gene in the RM30- <italic> R </italic> * (black) and RM38- <italic> r </italic> (brown) <span class="trait">seed</span> coats at the peak of its measured expression in the 400&#x02013;500 mg <span class="trait">seed</span> weight
					</p>
				
					<p>&nbsp;&nbsp;
						Once we had evidence that the <italic> TgmR </italic> * insertion slightly suppressed the expression of the <italic> R </italic> locus gene, <span class="gene">Glyma09g36983,</span> at late stages of <span class="trait">seed</span> development, we questioned at which point in the anthocyanin biosynthetic pathway the encoded MYB transcription factor exerted its regulatory control
					</p>
				
					<p>&nbsp;&nbsp;
						Enzymes and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> loci are abbreviated in uppercase letters: <italic> I </italic> , chalcone synthase (CHS); <italic> <span class="gene">W1</span> </italic> , flavonoid 3&#x02032;5&#x02032;-hydroxylase (F3&#x02032;5&#x02032;H); <italic> T </italic> , flavonoid 3&#x02032;-hydroxylase (F3&#x02032;H); <italic> Wp </italic> , flavanone 3-hydroxylase (F3H); <italic> <span class="gene">W3</span> </italic> , dihydroflavonol-4-reductase (DFR); Annthocyanidin synthase (ANS) also called leucoanthocyanidin dioxygenase (LDOX); UDP-flavonoid glucosyltransferase (UFGT); Leucoanthocyanidin reductase (LAR); Anthocyanidin reductase (ANR)
					</p>
				
					<p>&nbsp;&nbsp;
						Several of the genes involved in the anthocyanin pathway ( <italic> <span class="gene">CHS</span> </italic> , <italic> F3&#x02032;H </italic> , <italic> <span class="gene">F3H</span> </italic> , <italic> DFR </italic> , <italic> <span class="gene">ANS</span> </italic> , <italic> <span class="gene">ANR1</span> </italic> and <italic> AOMT </italic> ) are expressed at higher level early in <span class="trait">seed</span> development (100&#x02013;200 mg <span class="trait">seed</span> fresh weight) and the amount of transcripts decline in <span class="trait">seed</span> coats of both lines as <span class="trait">seed</span> growth advances ( <xref ref-type="fig" rid="pone-0111959-g005"> Figures 5 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						For those genes that are not expressed or expressed at very low level earlier in development ( <italic> <span class="gene">CHI</span> </italic> , <italic> F3&#x02032;5&#x02032;H </italic> and <italic> UFGT </italic> ), the expression increased to a slightly higher level in the <span class="trait">seed</span> coats of the black RM30- <italic> R </italic> * seeds during the late stages of <span class="trait">seed</span> development
					</p>
				
					<p>&nbsp;&nbsp;
						Despite the two <italic> UFGT </italic> genes ( <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">UGT78K1</span> </italic> ) transcript sequences being 92% similar, the sequence reads from the <span class="trait">seed</span> <span class="trait">coat</span> RNA samples aligned better to the <italic> <span class="gene">UGT78K2</span> </italic> transcript and differentially between the two lines ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						This suggests that the <italic> <span class="gene">UGT78K1</span> </italic> gene may not be expressed in the <span class="trait">seed</span> <span class="trait">coat</span> but rather in the cotyledons as it has been reported previously <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Although all the mentioned genes appear to be differentially expressed in the <span class="trait">seed</span> coats of the black and brown seeds, the ones that show the highest RPKM ratios and lowest p-values are the <italic> <span class="gene">ANS</span> </italic> genes with an RM30/RM38 of 10
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> shows the differences in the number of sequence reads in RPKMs for the <italic> <span class="gene">ANS3</span> </italic> gene in the RM30- <italic> R </italic> * black and the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						As stated earlier, these alignment results also show that the <span class="gene">Glyma09g36983</span> gene is transcribed all the way to the 3&#x02032;end in the RM30- <italic> R* </italic> black <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> ) despite the large transposon insertion at 110 nt from the beginning of Intron2
					</p>
				
					<p>&nbsp;&nbsp;
						The pair of graphs in each row represent the distribution of non-normalized RNAseq reads that align to the 714 bp transcript sequence from the <span class="gene">Glyma09g36983</span> gene at each of the five stages of <span class="trait">seed</span> <span class="trait">coat</span> development indicated in the margin to the right (described in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						On the other hand, the alignment of RNA-Seq reads from the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats to the 714-nt sequence representing the standard <span class="gene">Glyma09g36983</span> <italic> R </italic> allele transcript, shows a valley observed at 222 bp, the location of the deleted &#x0201c;C&#x0201d;-nt ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> red arrow)
					</p>
				
					<p>&nbsp;&nbsp;
						Using the Bowtie 1 alignments, we determined the distribution of sequence reads along the length of the 15,450-nt genomic sequence obtained from the black <span class="trait">seed</span> RM30- <italic> R* </italic> line that comprises the <italic> R </italic> locus gene <span class="gene">Glyma09g36983</span> interrupted by the 13,024-nt <italic> TgmR* </italic> insertion in Intron2
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear that sequences aligned all through the <italic> TgmR </italic> * sequence portion in RNA samples from both lines at all five stages of <span class="trait">seed</span> <span class="trait">coat</span> development examined including a low level from <span class="gene">ORF2</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Using primers designed to <span class="gene">Glyma09g36983,</span> our analysis of two isolines including a mutable RM55- <italic> r <sup> m </sup> </italic> line with black and brown striped <span class="trait">seed</span> phenotype and RM30- <italic> R* </italic> , a stable black-seeded revertant isoline, revealed the existence of a 13 kb CACTA transposon insertion ( <italic> TgmR* </italic> ) at the beginning of the <span class="gene">Glyma09g36983</span> gene Intron2 in both of these lines ( <xref ref-type="fig" rid="pone-0111959-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g003"> 3 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The RNA-Seq analysis at multiple stages of <span class="trait">seed</span> development was crafted to determine whether the <italic> TgmR* </italic> insertion interfered with the expression of the <italic> R </italic> <span class="gene">(Glyma09g36983)</span> gene
					</p>
				
					<p>&nbsp;&nbsp;
						The results were congruent with <italic> TgmR* </italic> having a negative effect on the expression of the <italic> R* </italic> allele, since a higher number of RNAseq reads were found to align to <span class="gene">Glyma09g36983</span> from the RM38- <italic> r </italic> line than from the RM30- <italic> R </italic> * line at late stages of <span class="trait">seed</span> development ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g004"> Figures 4 </xref> and <xref ref-type="fig" rid="pone-0111959-g009"> 9 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Although many of the genes in the anthocyanin pathway examined showed differences in expression between the RM30- <italic> R* </italic> black revertant and the RM38- <italic> r </italic> brown isoline, the most significant differences detected were for the <italic> ANS2 </italic> and <italic> <span class="gene">ANS3</span> </italic> genes at the 400&#x02013;500 mg <span class="trait">seed</span> developmental stage ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6A </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						In another study in which the transcript profiles of <span class="trait">seed</span> coats from a standard black and brown-seeded near-isogenic pair were compared at different stages of <span class="trait">seed</span> development, significant differences in <italic> ANS2/ANS3 </italic> and <italic> <span class="gene">UGT78K1</span> </italic> expression were also seen between the two isolines with higher levels in the black <span class="trait">seed</span> coats <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						01) manifested early in <span class="trait">seed</span> development, (75&#x02013;100 mg), while the <italic> <span class="gene">UGT78K1</span> </italic> differences were first detected at 200&#x02013;300 mg weight range and reaching maximum at 300&#x02013;400 mg <span class="trait">seed</span> size
					</p>
				
					<p>&nbsp;&nbsp;
						0111959-Gillman1" ref-type="bibr"> [4] </xref> measured <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression via qRT-PCR in <span class="trait">seed</span> coats of a brown- (PI 567115 B) and black- (PI 84970) seeded lines with similar results
					</p>
				
					<p>&nbsp;&nbsp;
						As in those instances, our results show that the <italic> UFGT </italic> genes are expressed at very low level at early stages of <span class="trait">seed</span> development, while <italic> <span class="gene">ANS</span> </italic> expression is very high at those early stages and tapers down as <span class="trait">seed</span> development advances
					</p>
				
					<p>&nbsp;&nbsp;
						The higher level of <italic> <span class="gene">ANS</span> </italic> gene expression at early stages of <span class="trait">seed</span> development can be explained by an additional role of the <span class="gene">ANS</span> enzyme in directing the synthesis towards the flavanol and proanthocyanidin branches of the flavonoid pathway ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The differential expression of the <italic> R </italic> MYB gene and <italic> <span class="gene">ANS</span> </italic> genes supports its role as an inducer of anthocyanidin synthase in the black soybean <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						The RNAseq expression results also demonstrate that the activation of the anthocyanin pathway in RM30- <italic> R </italic> * is due to the <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> transposon and not to either of the two other closely related MYB transcription factor genes, <span class="gene">Glyma09g36966</span> and <span class="gene">Glyma09g37010,</span> since these two genes appear not to be expressed in the <span class="trait">seed</span> coats of either the RM30- <italic> R* </italic> line with the black <span class="trait">seed</span> and the RM38- <italic> r </italic> line with the brown <span class="trait">seed</span> phenotypes
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, our results confirm the soybean <italic> R </italic> locus to be <span class="gene">Glyma09g36983</span> and demonstrate that the transcription of the <italic> R </italic> gene in the black <span class="trait">seed</span> coats of the RM30- <italic> R </italic> * revertant line, may be hindered by the processing of the very large Intron2 containing the <italic> TgmR </italic> * (13kb) insertion, but it does not render the processed transcript non-functional
					</p>
				
					<p>&nbsp;&nbsp;
						The latter, <italic> <span class="gene">UGT78K1</span> </italic> , was found to be expressed in the cotyledons in addition to the <span class="trait">seed</span> coats <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Distribution of RNA-Seq Reads from <span class="trait">Seed</span> Coats of the RM30- <italic> R </italic> * and RM38- <italic> r </italic> when Aligned to the Genomic Sequence of <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> Insertion in Intron2
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In its dominant form <italic> I </italic> , silences the expression of all nine <span class="gene">CHS</span> gene family via short interfering RNAs in a tissue-specific manner in the <span class="trait">seed</span> <span class="trait">coat</span> <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Since <span class="gene">CHS</span> is the first committed enzyme in the anthocyanin pathway, the <italic> I </italic> genotypes result in yellow <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						These authors identified the recessive, brown <italic> r </italic> allele to affect the expression of a <span class="trait">seed</span> coat-specific R2R3 MYB transcription factor gene, <span class="gene">Glyma09g36983</span> from the sequenced soybean genome as a strong candidate for the <italic> R </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Using quantitative RNA-Seq data from a developmental series of <span class="trait">seed</span> coats from the black-seeded RM30- <italic> R </italic> * revertant and the RM38- <italic> r </italic> brown-seeded isoline, we also demonstrate that the MYB factor encoded by the <italic> R </italic> gene appears to induce the expression of anthocyanidin synthase ( <italic> <span class="gene">ANS</span> </italic> ) genes at late stages of <span class="trait">seed</span> development in the black RM30- <italic> R* </italic> line.
					</p>
				
					<p>&nbsp;&nbsp;
						We compared the expression <span class="gene">Glyma09g36983</span> MYB transcription factor in the <span class="trait">seed</span> coats of the black revertant RM30- <italic> R* </italic> soybean line to that in the defective brown RM38- <italic> r </italic> soybean line at various stages of <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						(B) Expression of <span class="gene">Glyma09g36983</span> in RPKMs plotted against the five stages of <span class="trait">seed</span> development in mg <span class="trait">seed</span> fresh weight as shown in (A) above.
					</p>
				
					<p>&nbsp;&nbsp;
						069) indicates no significant differences of expression of the <span class="gene">Glyma09g36983</span> gene in the RM30- <italic> R </italic> * (black) and RM38- <italic> r </italic> (brown) <span class="trait">seed</span> coats at the peak of its measured expression in the 400&#x02013;500 mg <span class="trait">seed</span> weight.
					</p>
				
					<p>&nbsp;&nbsp;
						Once we had evidence that the <italic> TgmR </italic> * insertion slightly suppressed the expression of the <italic> R </italic> locus gene, <span class="gene">Glyma09g36983,</span> at late stages of <span class="trait">seed</span> development, we questioned at which point in the anthocyanin biosynthetic pathway the encoded MYB transcription factor exerted its regulatory control.
					</p>
				
					<p>&nbsp;&nbsp;
						Enzymes and <span class="trait">seed</span> <span class="trait">coat</span> <span class="trait">color</span> loci are abbreviated in uppercase letters: <italic> I </italic> , chalcone synthase (CHS); <italic> <span class="gene">W1</span> </italic> , flavonoid 3&#x02032;5&#x02032;-hydroxylase (F3&#x02032;5&#x02032;H); <italic> T </italic> , flavonoid 3&#x02032;-hydroxylase (F3&#x02032;H); <italic> Wp </italic> , flavanone 3-hydroxylase (F3H); <italic> <span class="gene">W3</span> </italic> , dihydroflavonol-4-reductase (DFR); Annthocyanidin synthase (ANS) also called leucoanthocyanidin dioxygenase (LDOX); UDP-flavonoid glucosyltransferase (UFGT); Leucoanthocyanidin reductase (LAR); Anthocyanidin reductase (ANR).
					</p>
				
					<p>&nbsp;&nbsp;
						Several of the genes involved in the anthocyanin pathway ( <italic> <span class="gene">CHS</span> </italic> , <italic> F3&#x02032;H </italic> , <italic> <span class="gene">F3H</span> </italic> , <italic> DFR </italic> , <italic> <span class="gene">ANS</span> </italic> , <italic> <span class="gene">ANR1</span> </italic> and <italic> AOMT </italic> ) are expressed at higher level early in <span class="trait">seed</span> development (100&#x02013;200 mg <span class="trait">seed</span> fresh weight) and the amount of transcripts decline in <span class="trait">seed</span> coats of both lines as <span class="trait">seed</span> growth advances ( <xref ref-type="fig" rid="pone-0111959-g005"> Figures 5 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						For those genes that are not expressed or expressed at very low level earlier in development ( <italic> <span class="gene">CHI</span> </italic> , <italic> F3&#x02032;5&#x02032;H </italic> and <italic> UFGT </italic> ), the expression increased to a slightly higher level in the <span class="trait">seed</span> coats of the black RM30- <italic> R </italic> * seeds during the late stages of <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						Despite the two <italic> UFGT </italic> genes ( <italic> <span class="gene">UGT78K2</span> </italic> and <italic> <span class="gene">UGT78K1</span> </italic> ) transcript sequences being 92% similar, the sequence reads from the <span class="trait">seed</span> <span class="trait">coat</span> RNA samples aligned better to the <italic> <span class="gene">UGT78K2</span> </italic> transcript and differentially between the two lines ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						This suggests that the <italic> <span class="gene">UGT78K1</span> </italic> gene may not be expressed in the <span class="trait">seed</span> <span class="trait">coat</span> but rather in the cotyledons as it has been reported previously <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Although all the mentioned genes appear to be differentially expressed in the <span class="trait">seed</span> coats of the black and brown seeds, the ones that show the highest RPKM ratios and lowest p-values are the <italic> <span class="gene">ANS</span> </italic> genes with an RM30/RM38 of 10.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="fig" rid="pone-0111959-g006"> Figure 6 </xref> shows the differences in the number of sequence reads in RPKMs for the <italic> <span class="gene">ANS3</span> </italic> gene in the RM30- <italic> R </italic> * black and the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						As stated earlier, these alignment results also show that the <span class="gene">Glyma09g36983</span> gene is transcribed all the way to the 3&#x02032;end in the RM30- <italic> R* </italic> black <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> ) despite the large transposon insertion at 110 nt from the beginning of Intron2.
					</p>
				
					<p>&nbsp;&nbsp;
						The pair of graphs in each row represent the distribution of non-normalized RNAseq reads that align to the 714 bp transcript sequence from the <span class="gene">Glyma09g36983</span> gene at each of the five stages of <span class="trait">seed</span> <span class="trait">coat</span> development indicated in the margin to the right (described in <xref ref-type="fig" rid="pone-0111959-g004"> Figure 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						On the other hand, the alignment of RNA-Seq reads from the RM38- <italic> r </italic> brown <span class="trait">seed</span> coats to the 714-nt sequence representing the standard <span class="gene">Glyma09g36983</span> <italic> R </italic> allele transcript, shows a valley observed at 222 bp, the location of the deleted &#x0201c;C&#x0201d;-nt ( <xref ref-type="fig" rid="pone-0111959-g009"> Figure 9 </xref> red arrow).
					</p>
				
					<p>&nbsp;&nbsp;
						Using the Bowtie 1 alignments, we determined the distribution of sequence reads along the length of the 15,450-nt genomic sequence obtained from the black <span class="trait">seed</span> RM30- <italic> R* </italic> line that comprises the <italic> R </italic> locus gene <span class="gene">Glyma09g36983</span> interrupted by the 13,024-nt <italic> TgmR* </italic> insertion in Intron2.
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear that sequences aligned all through the <italic> TgmR </italic> * sequence portion in RNA samples from both lines at all five stages of <span class="trait">seed</span> <span class="trait">coat</span> development examined including a low level from <span class="gene">ORF2</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Using primers designed to <span class="gene">Glyma09g36983,</span> our analysis of two isolines including a mutable RM55- <italic> r <sup> m </sup> </italic> line with black and brown striped <span class="trait">seed</span> phenotype and RM30- <italic> R* </italic> , a stable black-seeded revertant isoline, revealed the existence of a 13 kb CACTA transposon insertion ( <italic> TgmR* </italic> ) at the beginning of the <span class="gene">Glyma09g36983</span> gene Intron2 in both of these lines ( <xref ref-type="fig" rid="pone-0111959-g002"> Figures 2 </xref> and <xref ref-type="fig" rid="pone-0111959-g003"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The RNA-Seq analysis at multiple stages of <span class="trait">seed</span> development was crafted to determine whether the <italic> TgmR* </italic> insertion interfered with the expression of the <italic> R </italic> <span class="gene">(Glyma09g36983)</span> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						The results were congruent with <italic> TgmR* </italic> having a negative effect on the expression of the <italic> R* </italic> allele, since a higher number of RNAseq reads were found to align to <span class="gene">Glyma09g36983</span> from the RM38- <italic> r </italic> line than from the RM30- <italic> R </italic> * line at late stages of <span class="trait">seed</span> development ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g004"> Figures 4 </xref> and <xref ref-type="fig" rid="pone-0111959-g009"> 9 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Although many of the genes in the anthocyanin pathway examined showed differences in expression between the RM30- <italic> R* </italic> black revertant and the RM38- <italic> r </italic> brown isoline, the most significant differences detected were for the <italic> ANS2 </italic> and <italic> <span class="gene">ANS3</span> </italic> genes at the 400&#x02013;500 mg <span class="trait">seed</span> developmental stage ( <xref ref-type="table" rid="pone-0111959-t002"> Table 2 </xref> , <xref ref-type="fig" rid="pone-0111959-g006"> Figure 6A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In another study in which the transcript profiles of <span class="trait">seed</span> coats from a standard black and brown-seeded near-isogenic pair were compared at different stages of <span class="trait">seed</span> development, significant differences in <italic> ANS2/ANS3 </italic> and <italic> <span class="gene">UGT78K1</span> </italic> expression were also seen between the two isolines with higher levels in the black <span class="trait">seed</span> coats <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						01) manifested early in <span class="trait">seed</span> development, (75&#x02013;100 mg), while the <italic> <span class="gene">UGT78K1</span> </italic> differences were first detected at 200&#x02013;300 mg weight range and reaching maximum at 300&#x02013;400 mg <span class="trait">seed</span> size.
					</p>
				
					<p>&nbsp;&nbsp;
						0111959-Gillman1" ref-type="bibr"> [4] </xref> measured <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression via qRT-PCR in <span class="trait">seed</span> coats of a brown- (PI 567115 B) and black- (PI 84970) seeded lines with similar results.
					</p>
				
					<p>&nbsp;&nbsp;
						As in those instances, our results show that the <italic> UFGT </italic> genes are expressed at very low level at early stages of <span class="trait">seed</span> development, while <italic> <span class="gene">ANS</span> </italic> expression is very high at those early stages and tapers down as <span class="trait">seed</span> development advances.
					</p>
				
					<p>&nbsp;&nbsp;
						The higher level of <italic> <span class="gene">ANS</span> </italic> gene expression at early stages of <span class="trait">seed</span> development can be explained by an additional role of the <span class="gene">ANS</span> enzyme in directing the synthesis towards the flavanol and proanthocyanidin branches of the flavonoid pathway ( <xref ref-type="fig" rid="pone-0111959-g007"> Figure 7 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The differential expression of the <italic> R </italic> MYB gene and <italic> <span class="gene">ANS</span> </italic> genes supports its role as an inducer of anthocyanidin synthase in the black soybean <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						The RNAseq expression results also demonstrate that the activation of the anthocyanin pathway in RM30- <italic> R </italic> * is due to the <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> transposon and not to either of the two other closely related MYB transcription factor genes, <span class="gene">Glyma09g36966</span> and <span class="gene">Glyma09g37010,</span> since these two genes appear not to be expressed in the <span class="trait">seed</span> coats of either the RM30- <italic> R* </italic> line with the black <span class="trait">seed</span> and the RM38- <italic> r </italic> line with the brown <span class="trait">seed</span> phenotypes.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, our results confirm the soybean <italic> R </italic> locus to be <span class="gene">Glyma09g36983</span> and demonstrate that the transcription of the <italic> R </italic> gene in the black <span class="trait">seed</span> coats of the RM30- <italic> R </italic> * revertant line, may be hindered by the processing of the very large Intron2 containing the <italic> TgmR </italic> * (13kb) insertion, but it does not render the processed transcript non-functional.
					</p>
				
					<p>&nbsp;&nbsp;
						The latter, <italic> <span class="gene">UGT78K1</span> </italic> , was found to be expressed in the cotyledons in addition to the <span class="trait">seed</span> coats <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Distribution of RNA-Seq Reads from <span class="trait">Seed</span> Coats of the RM30- <italic> R </italic> * and RM38- <italic> r </italic> when Aligned to the Genomic Sequence of <span class="gene">Glyma09g36983</span> with the <italic> TgmR* </italic> Insertion in Intron2.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Validation of Loci Governing <span class="trait">Seed</span> <span class="trait">Coat</span> <span class="trait">Color</span> by Combining Association Mapping and Bulk Segregation Analysis in Soybean</h2>
			<h3>PMID:27404272</h3>
				

				
					<h3>PMCID:4942065</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Song </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Liu </surname>  <given-names> Zhangxiong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Hong </surname>  <given-names> Huilong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Yansong </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tian </surname>  <given-names> Long </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xinxiu </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ying-Hui </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Rongxia </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <xref ref-type="corresp" rid="cor001"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">W1</span> </italic> locus only affects <span class="trait">seed</span> <span class="trait">color</span> under <italic> iRt </italic> background and <italic> <span class="gene">W1</span> </italic> and <italic> w1 </italic> alleles give imperfect black and buff colors, respectively
					</p>
				
					<p>&nbsp;&nbsp;
						The recessive <italic> i </italic> allele had a deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences, resulting in an increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> due to the abolishment of posttranscriptional RNA silencing [ <xref rid="pone
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, some reports also revealed that combination of gene-based markers of <italic> T </italic> and <italic> <span class="gene">W1</span> </italic> loci or two SNP outliers could partially increase selection efficiency for <span class="trait">seed</span> colors [ <xref rid="pone
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The recessive <italic> i </italic> allele had a deletion of <italic> CHS4 </italic> or <italic> <span class="gene">CHS1</span> </italic> promoter sequences, resulting in an increased accumulation of chalcone synthase (CHS) transcripts in the <span class="trait">seed</span> <span class="trait">coat</span> due to the abolishment of posttranscriptional RNA silencing [ <xref rid="pone.
					</p>
				
					<p>&nbsp;&nbsp;
						Moreover, some reports also revealed that combination of gene-based markers of <italic> T </italic> and <italic> <span class="gene">W1</span> </italic> loci or two SNP outliers could partially increase selection efficiency for <span class="trait">seed</span> colors [ <xref rid="pone.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Characterizing Two Inter-specific Bin Maps for the Exploration of the QTLs/Genes that Confer Three Soybean Evolutionary Traits</h2>
			<h3>PMID:27602037</h3>
				

				
					<h3>PMCID:4994327</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Wubin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/338561/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Meifeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/368787/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yufeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/347210/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xuliang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/367596/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Shixuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Liping </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/347211/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Zheping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Jiejie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tuanjie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/345840/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gai </surname>  <given-names> Junyi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289861/overview"/>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 8 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<italic> APETALA 2 </italic> ( <italic> <span class="gene">AP2</span> </italic> ) is related to various aspects of plant development, including flowering and <span class="trait">seed</span> mass control ( <xref rid="B18" ref-type="bibr"> Jofuku et al
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		
	</div>
</body>

</html>