<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed yield</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed yield</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Confirmation of the pleiotropic control of leaflet shape and number of seeds per pod by the <span class="gene">Ln</span> gene in induced soybean mutants.</h2>
			<h3>PMID:29085246</h3>
				

				
			<p>
				作者：Sayama T(1)(2), Tanabata T(3), Saruta M(2)(4), Yamada T(1), Anai T(5), Kaga A(1), Ishimoto M(1).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most soybean cultivars possess broad leaflets; however, a recessive allele on the <span class="gene">Ln</span> locus is known to cause the alteration of broad to narrow leaflets. The recessive allele ln has also been considered to increase the number of seeds per pod (NSP) and has the potential to improve <span class="trait">yield.</span> Recently, <span class="gene">Gm-JAG1</span> <span class="gene">(Glyma20g25000),</span> a gene controlling Ln, has been shown to complement leaf shape and silique length in Arabidopsis mutants. However, whether <span class="gene">Gm-JAG1</span> is responsible for those traits in soybean is not yet known. In this study, we investigated the pleiotropic effect of soybean <span class="gene">Ln</span> gene on leaflet shape and NSP by using two independent soybean <span class="gene">Gm-jag1</span> mutants and four ln near isogenic lines (NILs). The leaflet shape was evaluated using a leaf image analysis software, SmartLeaf, which was customized from SmartGrain. The leaflets of both the <span class="gene">Gm-jag1</span> mutants were longer and narrower than those of the wild-type plants. Interestingly, the image analysis results clarified that the perimeter of the mutant leaflets did not change, although their leaflet area decreased. Furthermore, one mutant line with narrow leaflets showed significantly higher NSP than that in the wild (or Ln) genotype, indicating that soybean <span class="gene">Ln</span> gene pleiotropically controls leaflet shape and NSP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine-mapping and identification of a novel locus Rsc15 underlying soybean <span class="trait">resistance</span> to Soybean mosaic virus.</h2>
			<h3>PMID:28825113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rui R(1)(2), Liu S(1)(2), Karthikeyan A(1)(2), Wang T(1)(2), Niu H(1)(2), Yin J(1)(2), Yang Y(1)(2), Wang L(1)(2), Yang Q(1)(2), Zhi H(3)(4), Li K(5)(6).
				<br/> 
				年份：2017 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Rsc15, a novel locus underlying soybean <span class="trait">resistance</span> to SMV, was fine mapped to a 95-kb region on chromosome 6. The Rsc15- mediated <span class="trait">resistance</span> is likely attributed to the gene <span class="gene">GmPEX14</span> , the relative expression of which was highly correlated with the accumulation of H 2 O 2 along with the activities of POD and CAT during the early stages of SMV infection in RN-9. Soybean mosaic virus (SMV) causes severe <span class="trait">yield</span> losses and <span class="trait">seed</span> quality deterioration in soybean [Glycine max (L.) Merr.] worldwide. A series of single dominant SMV <span class="trait">resistance</span> genes have been identified on respective soybean chromosomes 2, 13 and 14, while one novel locus, Rsc15, underlying <span class="trait">resistance</span> to the virulent SMV strain SC15 from soybean cultivar RN-9 has been recently mapped to a 14.6-cM region on chromosome 6. However, candidate gene has not yet been identified within this region. In the present study, we aimed to fine map the Rsc15 region and identify candidate gene(s) for this invaluable locus. High-resolution fine-mapping revealed that the Rsc15 gene was located in a 95-kb genomic region which was flanked by the two simple sequence repeat (SSR) markers SSR_06_17 and BARCSOYSSR_06_0835. Allelic sequence comparison and expression profile analysis of candidate genes inferred that the gene <span class="gene">Glyma.06g182600</span> (designated as <span class="gene">GmPEX14)</span> was the best candidate gene attributing for the <span class="trait">resistance</span> of Rsc15, and that genes encoding receptor-like kinase (RLK) (i.e., <span class="gene">Glyma.06g175100</span> and <span class="gene">Glyma.06g184400)</span> and serine/threonine kinase (STK) (i.e., <span class="gene">Glyma.06g182900</span> and <span class="gene">Glyma.06g183500)</span> were also potential candidates. High correlations were established between the relative expression level of <span class="gene">GmPEX14</span> and the hydrogen peroxide (H2O2) concentration and activities of catalase (CAT) and peroxidase (POD) during the early stages of SMV-SC15 infection in RN-9. The results of the present study will be useful in marker-assisted breeding for SMV <span class="trait">resistance</span> and will lead to further understanding of the molecular mechanisms of host <span class="trait">resistance</span> against SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The genetic architecture of water-soluble protein content and its genetic relationship to total protein content in soybean.</h2>
			<h3>PMID:28698580</h3>
				

				
			<p>
				作者：Zhang D(1), Lü H(2), Chu S(2), Zhang H(3), Zhang H(4), Yang Y(5), Li H(2), Yu D(6).
				<br/> 
				年份：2017 Jul 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water-soluble protein content (WSPC) is a critical factor in both soybean protein quality and functionality. However, the underlying genetic determinants are unclear. Here, we used 219 soybean accessions and 152 recombinant inbred lines genotyped with high-density markers and phenotyped in multi-environments to dissect the genetic architectures of WSPC and protein content (PC) using single- and multi-locus genome-wide association studies. In the result, a total of 32 significant loci, including 10 novel loci, significantly associated with WSPC and PC across multi-environments were identified, which were subsequently validated by linkage mapping. Among these loci, only four exhibited pleiotropic effects for PC and WSPC, explaining the low correlation coefficient between the two traits. The largest-effect WSPC-specific loci, GqWSPC8, was stably identified across all six environments and tagged to a linkage disequilibrium block comprising two promising candidate genes AAP8 and 2 S albumin, which might contribute to the high level of WSPC in some soybean varieties. In addition, two genes, <span class="gene">Glyma.13G123500</span> and <span class="gene">Glyma.13G194400</span> with relatively high expression levels at <span class="trait">seed</span> development stage compared with other tissues were regarded as promising candidates associated with the PC and WSPC, respectively. Our results provide new insights into the genetic basis of WSPC affecting soybean protein quality and <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of Isoflavone Biosynthesis by miRNAs in Two Contrasting Soybean Genotypes at Different <span class="trait">Seed</span> Developmental Stages.</h2>
			<h3>PMID:28450878</h3>
				

				
			<p>
				作者：Gupta OP(1), Nigam D(2), Dahuja A(1), Kumar S(2), Vinutha T(1), Sachdev A(1), Praveen S(1).
				<br/> 
				年份：2017 Apr 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Owing to the presence of nutritionally important, health-promoting bioactive compounds, especially isoflavones, soybean has acquired the status of a functional food. miRNAs are tiny riboregulator of gene expression by either decreasing and/or increasing the expression of their corresponding target genes. Despite several works on identification and functional characterization of plant miRNAs, the role of miRNAs in the regulation of isoflavones metabolism is still a virgin field. In the present study, we identified a total of 31 new miRNAs along with their 245 putative target genes from soybean seed-specific ESTs using computational approach. The Kyoto Encyclopedia of Genes and Genomes pathway analyses indicated that miRNA putatively regulates metabolism and genetic information processing. Out of that, a total of 5 miRNAs <span class="gene">(Gma-miRNA12,</span> <span class="gene">Gma-miRNA24,</span> <span class="gene">Gma-miRNA26,</span> <span class="gene">Gma-miRNA28,</span> and <span class="gene">Gma-miRNA29)</span> were predicted and validated for their probable role during isoflavone biosynthesis. We also validated their five target genes using RA-PCR, which is as good as 5'RLM-RACE. Temporal regulation [35 days after flowering, 45, 55, and 65 DAF] of miRNAs and their targets showed differential expression schema. Differential expression of <span class="gene">Gma-miR26</span> and <span class="gene">Gma-miRNA28</span> along with their corresponding target genes <span class="gene">(Glyma.10G197900</span> and <span class="gene">Glyma.09G127200)</span> showed a direct relationship with the total isoflavone content. Therefore, understanding the miRNA-based genetic regulation of isoflavone pathway would assist in selection and manipulation to get high-performing soybean genotypes with better isoflavone <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping of the Rsv1-h gene in the soybean cultivar Suweon 97 that confers <span class="trait">resistance</span> to two Chinese strains of the soybean mosaic virus.</h2>
			<h3>PMID:27544525  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ma FF(1), Wu XY(1), Chen YX(1), Liu YN(1), Shao ZQ(1), Wu P(1), Wu M(1), Liu CC(1), Wu WP(1), Yang JY(2), Li DX(3), Chen JQ(4), Wang B(5).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The Rsv1 - h gene in cultivar Suweon 97, which confers <span class="trait">resistance</span> to SMVs, was mapped to a 97.5-kb location (29,815,195-29,912,667 bp on chromosome 13) in the Rsv1 locus, thereby providing additional insights into the molecular nature underlying variations in <span class="trait">resistance</span> alleles in this particular locus. Soybean mosaic virus (SMV) is a well-known devastating pathogen of soybean (Glycine max (L.) Merrill.) causing significant <span class="trait">yield</span> losses and <span class="trait">seed</span> quality deterioration. A single dominant allele, Rsv1-h, which confers <span class="trait">resistance</span> to multiple SMV strains, was previously reported in the cultivar Suweon 97, but its exact location is unknown. In the present study, Suweon 97 was crossed with a SMV-sensitive cultivar, Williams 82. Inoculating 267 F 2 individuals with two Chinese SMV strains (SC6-N and SC7-N) demonstrated that one single dominant gene confers SMV <span class="trait">resistance.</span> Another 1,150 F 2 individuals were then screened for two simple sequence repeat (SSR) markers (BARCSOYSSR_13_1103 and BARCSOYSSR_13_1187) that flank the Rsv1 locus. Seventy-four recombinants were identified and 20 additional polymorphic SSR markers within the Rsv1 region were then employed in genotyping these recombinants. F 2:3 and F 3:4 recombinant lines were also inoculated with SC6-N and SC7-N to determine their phenotypes. The final data revealed that in Suweon 97, the Rsv1-h gene that confers <span class="trait">resistance</span> to SC6-N and SC7-N was flanked by BARCSOYSSR_13_1114 and BARCSOYSSR_13_1115, two markers that delimit a 97.5-kb region in the reference Williams 82 genome. In such region, eight genes were present, of which two, <span class="gene">Glyma13g184800</span> and <span class="gene">Glyma13g184900,</span> encode the characteristic CC-NBS-LRR type of <span class="trait">resistance</span> gene and were considered potential candidates for Rsv1-h.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic and biochemical analyses of the cotyledon and root of flooding-stressed soybean plants.</h2>
			<h3>PMID:23799004  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Komatsu S(1), Makino T, Yasue H.
				<br/> 
				年份：2013 Jun 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Flooding significantly reduces the growth and grain <span class="trait">yield</span> of soybean plants. Proteomic and biochemical techniques were used to determine whether the function of cotyledon and root is altered in soybean under flooding stress. RESULTS: Two-day-old soybean plants were flooded for 2 days, after which the proteins from root and cotyledon were extracted for proteomic analysis. In response to flooding stress, the abundance of 73 and 28 proteins was significantly altered in the root and cotyledon, respectively. The accumulation of only one protein, 70 kDa heat shock protein (HSP70) <span class="gene">(Glyma17g08020.1),</span> increased in both organs following flooding. The ratio of protein abundance of <span class="gene">HSP70</span> and biophoton emission in the cotyledon was higher than those detected in the root under flooding stress. Computed tomography and elemental analyses revealed that flooding stress decreases the number of calcium oxalate crystal the cotyledon, indicating calcium ion was elevated in the cotyledon under flooding stress. CONCLUSION: These results suggest that calcium might play one role through <span class="gene">HSP70</span> in the cotyledon under flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript profiling reveals expression differences in wild-type and glabrous soybean lines.</h2>
			<h3>PMID:22029708  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hunt M(1), Kaur N, Stromvik M, Vodkin L.
				<br/> 
				年份：2011 Oct 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trichome hairs affect diverse agronomic characters such as <span class="trait">seed</span> weight and <span class="trait">yield,</span> prevent insect damage and reduce loss of water but their molecular control has not been extensively studied in soybean. Several detailed models for trichome development have been proposed for Arabidopsis thaliana, but their applicability to important crops such as cotton and soybean is not fully known. RESULTS: Two high throughput transcript sequencing methods, Digital Gene Expression (DGE) Tag Profiling and RNA-Seq, were used to compare the transcriptional profiles in wild-type (cv. Clark standard, CS) and a mutant (cv. Clark glabrous, i.e., trichomeless or hairless, CG) soybean isoline that carries the dominant <span class="gene">P1</span> allele. DGE data and RNA-Seq data were mapped to the cDNAs (Glyma models) predicted from the reference soybean genome, Williams 82. Extending the model length by 250 bp at both ends resulted in significantly more matches of authentic DGE tags indicating that many of the predicted gene models are prematurely truncated at the 5' and 3' UTRs. The genome-wide comparative study of the transcript profiles of the wild-type versus mutant line revealed a number of differentially expressed genes. One highly-expressed gene, <span class="gene">Glyma04g35130,</span> in wild-type soybean was of interest as it has high homology to the cotton gene GhRDL1 gene that has been identified as being involved in cotton fiber initiation and is a member of the BURP protein family. Sequence comparison of <span class="gene">Glyma04g35130</span> among Williams 82 with our sequences derived from CS and CG isolines revealed various SNPs and indels including addition of one nucleotide C in the CG and insertion of ~60 bp in the third exon of CS that causes a frameshift mutation and premature truncation of peptides in both lines as compared to Williams 82. CONCLUSION: Although not a candidate for the <span class="gene">P1</span> locus, a BURP family member <span class="gene">(Glyma04g35130)</span> from soybean has been shown to be abundantly expressed in the CS line and very weakly expressed in the glabrous CG line. RNA-Seq and DGE data are compared and provide experimental data on the expression of predicted soybean gene models as well as an overview of the genes expressed in young shoot tips of two closely related isolines.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine genetic mapping of the genomic region controlling leaflet shape and number of seeds per pod in the soybean.</h2>
			<h3>PMID:21104397  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong N(1), Moon JK, Kim HS, Kim CG, Jeong SC.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Narrow leaflet cultivars tend to have more seeds per pod than broad leaflet cultivars in soybean [Glycine max (L.) Merr.], which suggests that the leaflet-shape trait locus is tightly linked to or cosegregates with the trait locus controlling the number of seeds per pod (NSPP). Here, we attempted to further elucidate the relationship between leaflet shape and NSPP. A BC(3)F(2) population from a cross between the 'Sowon' (narrow leaflets and high NSPP) and 'V94-5152' (broad leaflets and low NSPP) variants was used. The results of the molecular genetic analyses indicated that, although the NSPP characteristic, in particular, the occurrence of 4-seeded pods, is governed by additional modifying genes that are likely present in Sowon, the two traits cosegregate in the BC(3)F(2) population. The mapping results generated using public markers demonstrated that the narrow leaflet-determining gene in Sowon is an allele of the previously highly studied ln gene on chromosome 20. A high-resolution map delimited the genomic region controlling both the leaflet shape and NSPP traits to a sequence length of 66 kb, corresponding to 0.7 cM. Among the three genes annotated in this 66 kb region, <span class="gene">Glyma20g25000.1</span> appeared to be a good candidate for the Ln-encoding gene, owing to its 47.8% homology with the protein encoding for the JAGGED gene that regulates lateral organ development in Arabidopsis. Taken together, our results suggested that phenotypic variations for narrow leaflet and NSPP are predominantly from the pleiotropic effects of the ln gene. Thus, our results should provide a molecular framework for soybean breeding programs with the objective of improving soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmFT2a and <span class="gene">GmFT5a</span> Redundantly and Differentially Regulate Flowering through Interaction with and Upregulation of the bZIP Transcription Factor <span class="gene">GmFDL19</span> in Soybean</h2>
			<h3>PMID:24845624</h3>
				
					<h3>PMCID:4028237</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Nan </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Dayong </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Ying </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Sijia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Lili </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Xiaohui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 4 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Plants integrate various environmental signals, such as photoperiod and temperature, to ensure flowering under those conditions that optimize <span class="trait">seed</span> production. In <italic> Arabidopsis thaliana </italic> (Arabidopsis), multiple pathways converge on a small number of floral integrator genes, which include the floral promoters <italic> FLOWERING LOCUS T (FT) </italic> and <italic> TWIN SISTER OF FT (TSF) </italic> , to integrate photoperiod, temperature, vernalization, and light quality signaling <xref rid="pone.0097669-Amasino1" ref-type="bibr"> [1] </xref> . FT and TSF are members of a family of proteins similar to the mammalian phosphatidylethanolamine-binding domain protein (PEBP) <xref rid="pone.0097669-Kardailsky1" ref-type="bibr"> [2] </xref> , <xref rid="pone.0097669-Kobayashi1" ref-type="bibr"> [3] </xref> . In addition to the FT-like proteins, the plant PEBP family consists of two other phylogenetically distinct groups of proteins, the TERMINAL FLOWER 1 (TFL1)-like proteins and the MOTHER OF FT AND TFL (MFT)-like proteins <xref rid="pone.0097669-Bradley1" ref-type="bibr"> [4] </xref> &#x02013; <xref rid="pone.0097669-Yoo2" ref-type="bibr"> [8] </xref> . FT and TSF act redundantly to promote flowering under long-day (LD) photoperiods <xref rid="pone.0097669-Yamaguchi1" ref-type="bibr"> [7] </xref> , <xref rid="pone.0097669-Michaels1" ref-type="bibr"> [9] </xref> , <xref rid="pone.0097669-Jang1" ref-type="bibr"> [10] </xref> . Arabidopsis FT and TSF proteins produced in the phloem <xref rid="pone.0097669-Yamaguchi1" ref-type="bibr"> [7] </xref> , <xref rid="pone.0097669-Takada1" ref-type="bibr"> [11] </xref> and are transported to the shoot apex, where they dimerize with the bZIP transcription factor FD to activate the expression of <italic> SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1 </italic> ( <italic> SOC1 </italic> ) <xref rid="pone.0097669-Michaels1" ref-type="bibr"> [9] </xref> , <xref rid="pone.0097669-Yoo3" ref-type="bibr"> [12] </xref> and the floral meristem identity genes <italic> APETALA1 </italic> ( <italic> <span class="gene">AP1</span> </italic> ) and <italic> LEAFY </italic> ( <italic> <span class="gene">LFY</span> </italic> ) <xref rid="pone.0097669-Abe1" ref-type="bibr"> [13] </xref> , <xref rid="pone.0097669-Wigge1" ref-type="bibr"> [14] </xref> . FT-like proteins from various species function in a manner similar to that of FT regarding the induction of flowering, transport in the phloem, and interaction with FD-like proteins <xref rid="pone.0097669-Lifschitz1" ref-type="bibr"> [15] </xref> &#x02013; <xref rid="pone.0097669-Li1" ref-type="bibr"> [18] </xref> , suggesting that this general mechanism is likely widely conserved across flowering plants. However, the rice FT ortholog Hd3a interacts with OsFD1 indirectly through a 14-3-3 protein to form a ternary trimer known as the florigen activation complex in the nuclei of the shoot apex, where it activates the expression of <italic> OsMADS15 </italic> , an <italic> <span class="gene">AP1</span> </italic> homolog that regulates flowering <xref rid="pone.0097669-Taoka1" ref-type="bibr"> [19] </xref> , <xref rid="pone.0097669-Kobayashi2" ref-type="bibr"> [20] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						All flowering pathways converge onto floral integrators, including <italic> FT </italic> and <italic> SOC1 </italic> , and induce the expression of floral meristem identity genes, including <italic> <span class="gene">AP1</span> </italic> and <italic> <span class="gene">LFY</span> </italic> <xref rid="pone.0097669-Blzquez1" ref-type="bibr"> [44] </xref> , <xref rid="pone.0097669-Lee1" ref-type="bibr"> [45] </xref> . Several genes involved in the determination of flowering time have recently been isolated and characterized in soybean, including <italic> <span class="gene">GmAP1</span> </italic> , <italic> <span class="gene">GmSOC1</span> </italic> , <italic> <span class="gene">GmSOC1-like</span> </italic> and <italic> <span class="gene">GmLFY</span> </italic> <xref rid="pone.0097669-Na1" ref-type="bibr"> [38] </xref> , <xref rid="pone.0097669-Chi1" ref-type="bibr"> [39] </xref> , <xref rid="pone.0097669-Meng1" ref-type="bibr"> [46] </xref> . By searching the soybean reference genome using Phytozome ( <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/soybean"> http://www.phytozome.net/soybean </ext-link> ), four <italic> <span class="gene">AP1</span> </italic> homologs were identified and designated as <italic> <span class="gene">GmAP1b</span> </italic> <span class="gene">(Glyma01g08150),</span> <italic> <span class="gene">GmAP1c</span> </italic> <span class="gene">(Glyma08g36380),</span> <italic> <span class="gene">GmAP1d</span> </italic> <span class="gene">(Glyma02g13420)</span> and <italic> <span class="gene">GmAP1a</span> </italic> <span class="gene">(Glyma16g13070,</span> <italic> <span class="gene">GmAP1</span> </italic> ), the last of which has been characterized previously <xref rid="pone.0097669-Chi1" ref-type="bibr"> [39] </xref> . Spatial RT-PCR analyses suggested that <italic> <span class="gene">GmAP1a</span> </italic> , <italic> <span class="gene">GmAP1b</span> </italic> and <italic> <span class="gene">GmAP1c</span> </italic> were mainly transcribed in reproductive organs such as shoot apices, flower buds and flowers under SD (12L/12D) conditions in the cultivar Harosoy, with the transcription of <italic> <span class="gene">GmAP1a</span> </italic> being the most prominent; however, the transcription of <italic> <span class="gene">GmAP1d</span> </italic> was not detected in any tissues ( <xref ref-type="fig" rid="pone-0097669-g002"> Figure 2A </xref> ). Two <italic> <span class="gene">LFY</span> </italic> homologs, designated as <italic> <span class="gene">GmLFY1</span> </italic> <span class="gene">(Glyma04g37900)</span> <xref rid="pone.0097669-Meng1" ref-type="bibr"> [46] </xref> and <italic> <span class="gene">GmLFY2</span> </italic> <span class="gene">(Glyma06g17170),</span> were also identified in the soybean genome. <italic> <span class="gene">GmLFY1</span> </italic> was transcribed mainly in developing pods and seeds and was not detected in leaves or the SAM (shoot apex meristem) ( <xref ref-type="fig" rid="pone-0097669-g002"> Figure 2A </xref> ), suggesting that the gene might contribute to <span class="trait">seed</span> development in soybean instead of flowering, as was previously reported <xref rid="pone.0097669-Meng1" ref-type="bibr"> [46] </xref> . Similarly to the <italic> <span class="gene">AP1</span> </italic> homologs, <italic> <span class="gene">GmLFY2</span> </italic> was also transcribed in shoot apices, flower buds and flowers ( <xref ref-type="fig" rid="pone-0097669-g002"> Figure 2A </xref> ). Two soybean <italic> SOC1 </italic> homologs, <italic> <span class="gene">GmSOC1a</span> </italic> <span class="gene">(Glyma18g45780)</span> <xref rid="pone.0097669-Chi1" ref-type="bibr"> [39] </xref> and <italic> <span class="gene">GmSOC1b</span> </italic> <span class="gene">(Glyma09g40230)</span> <xref rid="pone.0097669-Na1" ref-type="bibr"> [38] </xref> , could be identified from the soybean genome. Both of these genes were highly expressed in shoot apices, leaves, flower buds and roots but were weakly expressed in flowers and pods ( <xref ref-type="fig" rid="pone-0097669-g002"> Figure 2A </xref> ), in agreement with the expression patterns observed for <italic> SOC1 </italic> in multiple organs of Arabidopsis <xref rid="pone.0097669-Lee2" ref-type="bibr"> [47] </xref> . In total, six floral meristem identity genes, <italic> <span class="gene">GmAP1a,</span> <span class="gene">GmAP1b,</span> <span class="gene">GmAP1c,</span> <span class="gene">GmSOC1a,</span> <span class="gene">GmSOC1b</span> </italic> and <italic> <span class="gene">GmLFY2</span> </italic> , were constantly expressed in the shoot apices of the cultivar Harosoy from 10 DAE to 25 DAE under SD (12L/12D) conditions before the floral bud formation stage (floral buds formed at 25 DAE), and <italic> <span class="gene">GmSOC1a</span> </italic> and <italic> <span class="gene">GmSOC1b</span> </italic> were also constantly expressed in the leaves of the cultivar ( <xref ref-type="fig" rid="pone-0097669-g002"> Figure 2B </xref> ). The constantly high expression levels of these genes in shoot apices before the floral bud formation stage most likely indicate their involvement in the flowering transition of soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of soybean <span class="trait">seed</span> developmental stage-specific and tissue-specific miRNA targets by degradome sequencing</h2>
			<h3>PMID:22799740</h3>
				
					<h3>PMCID:3410764</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Shamimuzzaman </surname>  <given-names> Md </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> shamimu2@illinois.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A2">  <name>  <surname> Vodkin </surname>  <given-names> Lila </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@illinois.edu </email>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Tissue specific miRNA and target identification is very important for understanding the regulation of gene expression in a spatial manner. In this study, we constructed cotyledon and <span class="trait">seed</span> coat libraries separately to identify miRNA targets both at younger (25&#x02013;50&#x02009;mg fresh weight <span class="trait">seed)</span> and older stages (100&#x02013;200&#x02009;mg and 300&#x02013;400&#x02009;mg fresh weight <span class="trait">seed)</span> of soybean <span class="trait">seed</span> development. Tissue specific siRNAs generated from a cluster of inverted repeat chalcone synthase (CHS) genes that downregulate <span class="gene">CHS</span> mRNAs and lead to lack of pigment on soybean <span class="trait">seed</span> coats have been described [ <xref ref-type="bibr" rid="B27"> 27 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> ], but very little is known about the miRNAs and their targets in developing <span class="trait">seed</span> tissues. We analyzed the degradome data from <span class="trait">seed</span> coats versus cotyledons and identified 25 miRNAs and their 32 different targets that were found only in the cotyledons and not the <span class="trait">seed</span> coats (Table&#x02009; <xref ref-type="table" rid="T3"> 3 </xref> ). Likewise, 12 miRNAs and their 18 targets are associated with the <span class="trait">seed</span> coats only (Table&#x02009; <xref ref-type="table" rid="T4"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Comparison of the data in our report with the first soybean degradome data reported by Song et al. [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] from a very early stage of soybean <span class="trait">seed</span> development in the cultivar Heinong44 revealed a number of miRNA targets in common among the data sets including many transcription factors such as ARF, MYB, TCP, NF-Y, Growth Regulatory Factor, HD-ZIP, PPR, <span class="gene">SBP</span> and NAC family protein. In contrast, we found some other miRNA targets such as Permease family protein, LRR domain containing protein, transmembrane protein 14&#x02009;C, Serine-Threonine kinase, BRE expressed protein and transcription factor TFIID only in our degradome libraries that represented later stages of <span class="trait">seed</span> development of the cultivar Williams. Overall there were 65 Glyma model targets in common between both of the data sets, 80 found only in the Heinong44 data set, and 118 found only in the Williams data sets.
					</p>
				
					<p>&nbsp;&nbsp;
						The 183 identified targets from our degradome analyses belonged to 57 different annotation groupings ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). Comparing those to the Song et al. paper, 11 annotation groups are in common, 18 are unique to the Heinong44 very young <span class="trait">seed,</span> and 46 are unique to the five libraries of the Williams data sets (Table&#x02009; <xref ref-type="table" rid="T5"> 5 </xref> ). Many of the identified soybean miRNA targets belong to diverse gene families of transcription factors such as ARFs, MYBs, TCPs, NACs, HD-ZIPs and NF-Y subunits (Tables&#x02009; <xref ref-type="table" rid="T2"> 2 </xref> <xref ref-type="table" rid="T3"> 3 </xref> and <xref ref-type="table" rid="T4"> 4 </xref> and <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). Many of these transcription factors are known to regulate diverse aspects of plant growth and development. Patterning and outgrowth of lateral organs in plants depend on the expression of HD-ZIP transcription factors that specify adaxial/upper cell fate [ <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> ]. We identified a number of HD-ZIP transcription factors as targets for gma-miR166 in our degradome libraries. MYB family members in rice, which are targeted by miR159, appear to play an important role in response to the presence of abscisic acid (ABA) during plant embryonic development, suggesting their roles in <span class="trait">seed</span> development [ <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B38"> 38 </xref> ]. In this study, we detected a number of MYB family transcription factors regulated by gma-miR159 (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> . The gma-miRNA156 family members target sites in numerous proteins containing the Squamosa Promoter Binding (SBP) domain. <span class="gene">SBP</span> and SBP-LIKE (SPLs) proteins play multiple roles in plant development [ <xref ref-type="bibr" rid="B39"> 39 </xref> , <xref ref-type="bibr" rid="B40"> 40 </xref> ]. In <italic> Arabidopsis </italic> , rice and in some other plants, miR156 regulates leaf development by targeting Squamosa-Promoter Binding protein-like (SBP) transcription factors [ <xref ref-type="bibr" rid="B31"> 31 </xref> , <xref ref-type="bibr" rid="B39"> 39 </xref> ]. The identification of <span class="gene">SBP</span> as a target of gma-miR160 may indicate the additional level of regulation for <span class="gene">SBP</span> during soybean <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						Our analyses of the early, mid and late maturation developmental stages of soybean show a number of targets similar to those found by Song et al., [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] in the very young seeds of the cultivar Heinong44 including the SPB transcription factors. One notable difference was the absence in our degradome data of miRNA172 targets which include members of the <span class="gene">AP2</span> transcription factor family. From inspection of sequenced small RNA populations from the 50&#x02013;75&#x02009;mg <span class="trait">seed</span> coats and cotyledons of Williams [ <xref ref-type="bibr" rid="B27"> 27 </xref> ], we find only a few occurrences of the miR172 family (less than 30 occurrences per million reads) while some family members of the miR156 family are highly abundant (99,000 per million) in the cotyledons. We speculate that miR172 and/or its targets may be more abundant in the very young <span class="trait">seed</span> used by the Song et al. group [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] and not prevalent in the mid-maturation <span class="trait">seed</span> that we have examined. In <italic> Arabidopsis </italic> , miR172 has been reported to be involved in the regulation of flowering time and floral development [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. Alternatively, the <span class="gene">AP2</span> factors may not be detected as targets in the degradome data if translational repression by miR172 is operative as has been shown in <italic> Arabidopsis </italic> flower development [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. Nuclear Factor Y (NF-Y) was shown to control a variety of agronomically important traits, including <span class="trait">drought</span> tolerance, flowering time, and <span class="trait">seed</span> development [ <xref ref-type="bibr" rid="B42"> 42 </xref> ]. We detected seven NF transcription factor YA subunit mRNAs specifically in <span class="trait">seed</span> coats that are targets of miR169 family members that occur in both <span class="trait">seed</span> coats and cotyledons ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ). These targets may indicate some specific regulation of NF-YA transcription factors during soybean <span class="trait">seed</span> development.
					</p>
				
					<p>&nbsp;&nbsp;
						Auxin is an important phytohormone in higher plants. It acts as a key player in plant development [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. As the transducer of auxin signaling, ARFs play vital roles in plant development, including shoot, root and flower formation [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B48"> 48 </xref> ]. In <italic> Arabidopsis </italic> , miR160 and miR167 are involved in auxin signaling via regulation of ARF genes [ <xref ref-type="bibr" rid="B1"> 1 </xref> ]. In rice, a number of ARF encoding genes have been identified which are regulated by osa-miR160 and osa-miR167, respectively [ <xref ref-type="bibr" rid="B33"> 33 </xref> , <xref ref-type="bibr" rid="B34"> 34 </xref> ]. In our study, we identified a large number of ARF genes as targets for different miRNAs such as gma-miR160 and miR167. In the cotyledon (100&#x02013;200&#x02009;mg) degradome library ( <xref ref-type="supplementary-material" rid="S1"> Additional file 1 </xref> ), we identified five targets annotated as Auxin Response Factors (ARFs) for gma-miR160, and four of these, <italic> <span class="gene">Glyma12g08110.1</span> </italic> <italic> <span class="gene">Glyma12g29720.1</span> </italic> <italic> <span class="gene">Glyma14g33730.1</span> </italic> and <italic> <span class="gene">Glyma11g20490.1,</span> </italic> were validated by RLM-5&#x02019;RACE showing precise cleavage as expected (Figure&#x02009; <xref ref-type="fig" rid="F2"> 2 </xref> ). These results suggested that gma-miR160 could participate in auxin signaling via down-regulation of ARFs during soybean <span class="trait">seed</span> developmental stages. The cleaved mRNAs captured by the degradome procedure indicate that the levels of the ARF mRNA targets are likely to be decreased, but qRTPCR or RNA sequencing data would be needed to directly confirm the effect on mRNA levels for a particular ARF target gene.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genomic differences between cultivated soybean, <italic> G. max </italic> and its wild relative <italic> G. soja </italic></h2>
			<h3>PMID:23368680</h3>
				
					<h3>PMCID:3549820</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> joshitr@missouri.edu </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> valliyodanb@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Wu </surname>  <given-names> Jeng-Hung </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> s19601060@ym.edu.tw </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Lee </surname>  <given-names> Suk-Ha </given-names>  </name>  <xref ref-type="aff" rid="I7"> 7 </xref>  <xref ref-type="aff" rid="I8"> 8 </xref>  <email> sukhalee@snu.ac.kr </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> xudong@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A6">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> nguyenhenry@missouri.edu </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean gene <span class="gene">Glyma04g12020.1</span> is annotated as WD40 repeat proteins and these proteins regulate biosynthesis of anthocyanins, proanthocyanidins (PAs), and mucilage in the <span class="trait">seed</span> and <span class="trait">seed</span> developmental stages [ <xref ref-type="bibr" rid="B22"> 22 </xref> ]. Functional classification of these 12 genes clearly shows their roles in major biochemical processes during <span class="trait">seed</span> development in plants. Various molecular functions of these genes such as sugar and carbohydrate metabolism, nitrogen assimilation and metabolism, transporters, trans-membrane, and even the regulation of ubiquitination of proteins [ <xref ref-type="bibr" rid="B23"> 23 </xref> ] are essential to regulate <span class="trait">seed</span> developmental processes and <span class="trait">seed</span> size in soybean. Above all, the <span class="gene">Glyma14g02210.1</span> encoding enzyme CDP-diacylglycerol synthase is a major enzyme in regulation of fatty acyl fluxes during triacylglycerol (TAG) biosynthesis [ <xref ref-type="bibr" rid="B24"> 24 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The sequences of 694 unique loci of known or genes considered to be involved in acyl lipid metabolism of <italic> Arabidopsis thaliana </italic> were compared against the soybean cDNA sequences to identify the corresponding homolog using Blast (criteria: identity &#x0003e; = 80% and e-value &#x0003c; 1e-3). When compared against the 425 soybean genes that are deleted in <italic> G. soja </italic> but present in <italic> G. max </italic> , 2 genes involved in acyl lipid metabolism were identified that are unique to <italic> G. max </italic> . 1,261 genes out of 1,266 unique hits to genes involved in acyl lipid metabolism were mapped to genes with ".1" gene models. As shown in Table <xref ref-type="table" rid="T7"> 7 </xref> , 3 genes were found in the list of 425 unique genes in <italic> G. max </italic> and one gene <span class="gene">(Glyma10g05750.1)</span> is novel when compared to the previous analysis. The gene <span class="gene">Glyma06g39750.1</span> is shown to overlap with oil, seed-weight and <span class="trait">yield</span> related traits QTLs. The gene <span class="gene">Glyma10g05750.1</span> is associated with southern root-knot nematode <span class="trait">resistance</span> trait and falls in its QTL. <span class="gene">Glyma10g05750.1</span> is dolichyldiphosphatase which is catalytic and membrane related. This enzyme is reported as cell cycle dependent in human and, dolichol and dolichyl phosphate phosphatase (polyphenols) were characterized from soybean previously [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. The lipid fractions of the lipo-sugar carriers are essential for the biosynthesis of N-linked glycoproteins. The gene <span class="gene">Glyma06g39750.1</span> encodes for 3-hydroxyacyl-coa dehydrogenase and this enzyme catalyzes the third reaction of the mitochondrial &#x003b2;-oxidation cascade in animals. The gene <span class="gene">Glyma04g12260</span> encodes for the fatty acyl-coenzyme A reductases. Earlier studies has confirmed that the heterologous expression of fatty acyl-coenzyme A reductases FAR1, FAR4, and FAR5 in yeast are active alcohol-forming FARs with distinct, but overlapping, chain length specificities ranging from C18:0 to C24:0 [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. This report indicates that Arabidopsis FAR1, FAR4, and FAR5 generate the fatty alcohols found in root, <span class="trait">seed</span> coat, and wound-induced leaf tissue. The soybean transcript for Phospholipase D <span class="gene">(Glyma07g08740)</span> belongs to a lipolytic enzyme subclass which catalyzes the hydrolysis and transesterification of glycerophospholipids at the terminal phosphodiester bond [ <xref ref-type="bibr" rid="B27"> 27 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						We performed some analysis and visualization of the QTLs and traits using the SoyKB web resource. The <italic> G. soja </italic> SNPs, inversions and deletions have been added to the SoyKB resource. The graphical chromosome visualizer is an innovative molecular breeding tool, developed as part of SoyKB Breeding Program Tools, which integrates the genomic variations data such as SNPs and GWAS alongside the traits, QTLs, molecular markers and underlying genes. It displays the data linearly for each chromosome along with other useful information such as QTLs for multiple traits, SNPs from multiple genotypes of soybean including <italic> G. soja </italic> , as well as insertion, inversion and deletions data from multiple genotypes and <italic> G. soja </italic> . The chromosome visualizer can be directly accessed via the Tools menu on SoyKB ( <ext-link ext-link-type="uri" xlink:href="http://soykb.org/visualizer.php"> http://soykb.org/visualizer.php </ext-link> ). This tool can be used for highly meaningful analysis for QTLs of specific traits and showing the overlapping genes and regions of deletion and SNPs in comparison to the Williams82 reference genome. The list of genes falling in any QTL regions for single or multiple traits can be saved using the "Save" option on the viewing panel or "Left-clicking" the highlighted QTL region. The panels can be dragged and dropped to rearrange the sequence of visualizing the data. We performed the combined analysis for all the 3 traits. For example, in <span class="trait">seed</span> trait QTLs, gene <span class="gene">Glyma01g17340.1</span> is deleted in <italic> G. soja </italic> and thus unique to <italic> G. max </italic> as shown in Figure <xref ref-type="fig" rid="F2"> 2 </xref> . There are 315 such other genes falling in QTLs and being unique to <italic> G. max </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Chromosome visualizer tool in SoyKB showing the gene <span class="gene">Glyma01g17340</span> </bold> . Chromosome visualizer tool in SoyKB shows the gene <span class="gene">Glyma01g17340.1,</span> which is unique in G. <italic> max </italic> , overlapping in QTL region for <span class="trait">seed</span> trait as well as the deletion region in <italic> G. soja </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						We selected the <span class="trait">seed,</span> oil and protein traits to visualize the QTLs for all three of them. We also selected the <italic> G. soja </italic> SNPs as well as inversion and deletions datasets from the bottom panels for viewing. Figure <xref ref-type="fig" rid="F3"> 3 </xref> shows the region of <span class="gene">Gm09</span> chromosome with the overlapping QTLs for all the 3 traits and also the region overlapping with the <italic> G. soja </italic> deletions region. This clearly shows an example of genes unique to <italic> G. max </italic> and to be involved important functions related to <span class="trait">seed,</span> oil and protein traits. This tool can be used by researchers to identify, confirm as well as generate lists of genes falling in the QTL regions.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Chromosome visualizer tool in SoyKB showing the region of chromosome <span class="gene">Gm09</span> </bold> . Chromosome visualizer tool in SoyKB shows the region of chromosome <span class="gene">Gm09</span> with overlapping QTL regions for multiple traits. Oil trait is shown in pink, protein trait in blue and <span class="trait">seed</span> trait in grey.
					</p>
				
					<p>&nbsp;&nbsp;
						Combined analysis of <span class="trait">seed</span> and oil traits shows a region of deletion in <italic> G. soja </italic> on chromosome <span class="gene">Gm01:19,434,434</span> - 19,498,249, which overlaps with the QTL regions for both the traits. This involves 7 consecutive genes <span class="gene">(Glyma09g16400,</span> <span class="gene">Glyma09g16410,</span> <span class="gene">Glyma09g16420,</span> <span class="gene">Glyma09g16430,</span> <span class="gene">Glyma09g16440,</span> <span class="gene">Glyma09g16470,</span> and <span class="gene">Glyma09g16480),</span> which are deleted in <italic> G. soja </italic> and are significant in <italic> G. max </italic> as they all fall in the QTL region for both oil and <span class="trait">seed</span> traits (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ), contributing to the differences in the oil and <span class="trait">seed</span> development between the two varieties. Combined analysis of oil and protein traits shows a QTL region on chromosome <span class="gene">Gm09</span> that contains several genes deleted in <italic> G. soja </italic> . Several genes in this region are significant since they overlap with some deleted regions in <italic> G. soja </italic> and are unique to <italic> G. max </italic> , in addition to falling in important QTL regions. In Figure <xref ref-type="fig" rid="F3"> 3 </xref> , the oil trait QTL is highlighted in pink and protein QTL in blue. The functional annotation and experimental data for all of these genes were acquired using the multiple gene search feature of SoyKB.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq Profiling of a Defective <span class="trait">Seed</span> Coat Mutation in <italic> Glycine max </italic> Reveals Differential Expression of Proline-Rich and Other Cell Wall Protein Transcripts</h2>
			<h3>PMID:24828743</h3>
				
					<h3>PMCID:4020777</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kour </surname>  <given-names> Anupreet </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boone </surname>  <given-names> Anne M. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 4 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The effect on <span class="gene">PRP1</span> transcript levels was assumed to be a downstream effect of the net pattern mutation. In this report, we present the global gene expression analysis of the net pattern cell wall mutant in soybean using transcriptome analyses of three different stages of <span class="trait">seed</span> development at 50&#x02013;100 mg, 100&#x02013;200 mg, and 400&#x02013;500 mg. By comparing RNA-Seq data from the two lines containing the defective <span class="trait">seed</span> coat trait compared to their standard counterpart isolines using Bowtie alignments <xref rid="pone.0096342-Langmead1" ref-type="bibr"> [11] </xref> to the soybean genes models from the sequenced soybean genome <xref rid="pone.0096342-Schmutz1" ref-type="bibr"> [12] </xref> , 364 significantly differentially expressed genes in common between the two isolines were revealed, many of which were involved in cell wall processes. These data aid in determining perturbations in the physiological development of the plant cell wall which affects agronomically important traits and is a source of bioenergy <xref rid="pone.0096342-PestanaCalsa1" ref-type="bibr"> [13] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						A selection of 29 of the 364 differentially expressed genes at the 100&#x02013;200 mg <span class="trait">seed</span> weight stage are shown in <xref ref-type="table" rid="pone-0096342-t003"> Table 3 </xref> (in Clark background) and <xref ref-type="table" rid="pone-0096342-t004"> Table 4 </xref> (in Harosoy background) along with their functional annotations, fold-changes in differential expression, and p-values. The data for all 364 genes are shown in <xref ref-type="supplementary-material" rid="pone.0096342.s021"> Table S3 </xref> . Six genes in <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> and <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> have cell wall structural protein annotations, including proline-rich proteins <span class="gene">PRP1</span> and <span class="gene">PRP2</span> and glycine-rich proteins.
					</p>
				
					<p>&nbsp;&nbsp;
						As presented in <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> &#x00026; <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> , <span class="gene">PRP1</span> and <span class="gene">PRP2</span> were two of the highly differentially expressed proline-rich protein genes in both Clark and Harosoy isolines. In the Harosoy isolines, both the RPKM levels and fold differences for these genes were higher as compared to Clark isolines. The expression of the <span class="gene">PRP1</span> was higher earlier in development and declined as shown in <xref ref-type="fig" rid="pone-0096342-g004"> Figure 4A </xref> . In contrast, <span class="gene">PRP2</span> was expressed only at the later stages of <span class="trait">seed</span> development as seen in <xref ref-type="fig" rid="pone-0096342-g004"> Figure 4B </xref> . The differential expression between the standard and defective <span class="trait">seed</span> coats was maximal for both <span class="gene">PRP1</span> and <span class="gene">PRP2</span> at the 100&#x02013;200 mg weight range in Clark and Harosoy isolines. <span class="gene">PRP1</span> transcripts appeared to decline in abundance faster in the standard <span class="trait">seed</span> coats of both Clark and Harosoy isolines leading to higher levels in the defective <span class="trait">seed</span> coats whereas <span class="gene">PRP2</span> transcripts were always less abundant in the defective <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression levels of <span class="gene">PRP1</span> and <span class="gene">PRP2</span> at different <span class="trait">seed</span> weight stages were previously demonstrated by <xref rid="pone.0096342-Percy1" ref-type="bibr"> [10] </xref> using RNA blots in the Clark isoline. Our previous RNA blots agree with our present RNA-Seq data showing that <span class="gene">PRP1</span> was expressed more abundantly in early development, whereas <span class="gene">PRP2</span> was present in the older <span class="trait">seed</span> coats. These two Glyma models were called as &#x0201c;low confidence&#x0201d; gene models with no annotation by the Glyma gene model version 1.0 at the Phytozome database and as extensin-related genes in Glyma model version 1.1, whereas the non-redundant database of the National Center for Biotechnology Information (NCBI) annotated them as proline-rich proteins. We conducted manual annotations in the following manner to confirm them as the classically defined soybean <span class="gene">PRP1</span> and <span class="gene">PRP2</span> genes which are in NCBI. The nucleotide and amino acid sequences of <span class="gene">PRP1</span> from NCBI (J02746) showed 100% match with <span class="gene">Glyma09g12200.1</span> from the soybean genome ( <xref ref-type="supplementary-material" rid="pone.0096342.s003"> Figure S3 </xref> ), and the nucleotide sequence of SbPRP2 gene from NCBI (J05208) showed 98.8% similarity with <span class="gene">Glyma09g12260.1</span> ( <xref ref-type="supplementary-material" rid="pone.0096342.s004"> Figure S4 </xref> ). There was a difference of 10 amino acids between the amino acid sequence of <span class="gene">PRP2</span> in cultivars Wayne (amino acid sequence from NCBI) and Williams (amino acid sequence from the soybean genome). Both <span class="gene">PRP1</span> and <span class="gene">PRP2</span> genes are linked and separation between these genes is approximately 146 kb ( <xref ref-type="supplementary-material" rid="pone.0096342.s005"> Figure S5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The graphical representations of the expression levels of the other differentially expressed genes shown in <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> and <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> at all three different stages of <span class="trait">seed</span> development is presented in <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figures S6 </xref> and <xref ref-type="supplementary-material" rid="pone.0096342.s007"> S7 </xref> , respectively, showing that all have the highest differential expression at the mid- developmental stages of 100&#x02013;200 mg <span class="trait">seed</span> weight. Included is the pattern for several other proline and glycine-rich proteins and that of the xylem serine proteinase that is overexpressed in the defective <span class="trait">seed</span> coats ( <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figure S6 </xref> -M). In many of these genes, those that are highly expressed in the 50&#x02013;100 mg <span class="trait">seed</span> coats and then decline show a pattern similar to <span class="gene">PRP1</span> in that they are overexpressed in the defective <span class="trait">seed</span> coats. Likewise, those that are overexpressed in the standard line are similar to <span class="gene">PRP2</span> in that they generally have higher RPKM expression levels later in development.
					</p>
				
					<p>&nbsp;&nbsp;
						It is clear from the 68 graphs of the gene expression patterns shown in <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figures S6 </xref> &#x02013; <xref ref-type="supplementary-material" rid="pone.0096342.s010"> S10 </xref> that the 100&#x02013;200 mg weight range shows the highest differential expression and that many of the genes can be classified as either declining in expression or increasing in expression during <span class="trait">seed</span> coat development as is the case for the <span class="gene">PRP1</span> and <span class="gene">PRP2</span> patterns. In many cases, the net pattern mutation appears to increase expression of many of the cell wall related genes in the 100&#x02013;200 mg weight range as for PRP1. In contrast, many of the genes that are overexpressed in the standard line appear to be increasing during late <span class="trait">seed</span> development in both standard and defective lines as does PRP2.
					</p>
				
					<p>&nbsp;&nbsp;
						There were 21 hormone regulated genes that were differentially expressed in the <span class="trait">seed</span> coat of wild type and defective isolines of both Clark and Harosoy background ( <xref ref-type="supplementary-material" rid="pone.0096342.s021"> Table S3 </xref> ) including representative auxin and gibberellin regulated genes that are shown in <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> &#x00026; <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> . One gene model <span class="gene">Glyma10g35870.1</span> showed the best match to the authentic <span class="gene">ADR12</span> (auxin down regulated) gene from NCBI (S58482). It showed approximately 16-fold lower expression in the Clark defective <span class="trait">seed</span> coats and 4-fold lower expression expression in Harosoy defective <span class="trait">seed</span> coats ( <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> &#x00026; <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> ). Auxin is an important phytohormone that plays several important roles in plant growth and development. The expression level of this gene at different stages of <span class="trait">seed</span> development in both standard and defective <span class="trait">seed</span> coat isolines is presented in <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5A </xref> for RNA-Seq data. In the Clark standard isoline, this gene showed higher expression in earlier stages, and there was continuous decrease in the expression level at later stages of <span class="trait">seed</span> development. There was significant differential expression at all the <span class="trait">seed</span> developmental stages ( <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5A </xref> , left). In the Harosoy standard isoline, expression of <span class="gene">ADR12</span> increased continuously with developmental stages and also increased in the defective <span class="trait">seed</span> coats at the two older stages, although the levels were significantly lower in the defective <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5A </xref> , right). The expression pattern of <span class="gene">ADR12</span> using RNA blots ( <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5B </xref> , top) coincides well with the RNA-Seq data in that the defective <span class="trait">seed</span> coats have reduced expression especially in the Clark isolines. In addition, the blots confirm that the <span class="gene">ADR12</span> is more highly expressed in standard <span class="trait">seed</span> coats of Clark than in Harosoy. As presented in the blots in <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5B </xref> , bottom panel, there was no differential expression observed from 4&#x02013;5 days old soybean hypocotyls in standard and defective isolines of Clark and Harosoy. Thus, the differential expression of this gene is likely confined to the <span class="trait">seed</span> coats that manifest the defective cell wall structure.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) The levels of <span class="gene">ADR12</span> (ADR12-2) transcripts (as RPKM) in <span class="trait">seed</span> coats of Clark (left) or Harosoy (right) isolines. The blue line in the graphs represents the expression level (RPKM) of this gene in <span class="trait">seed</span> coats of standard isolines and the red line represents expression levels in the defective <span class="trait">seed</span> coats at the 50&#x02013;100 mg, 100&#x02013;200 mg, and 400&#x02013;500 mg <span class="trait">seed</span> weight ranges. (B, Top) The Northern blots of RNA from 100&#x02013;200 mg soybean <span class="trait">seed</span> coats representing three independent extractions of the Clark and two independent samples of the Harsoy lines. (B, Bottom) The Northern blot of RNA from 4&#x02013;5 days old soybean hypocotyls extracted from two independent samples of each line. Each gel lane was loaded with 10 ug of total RNA, electrophoresed, stained with ethidium bromide to confirm equal loadings, and blotted. The RNA blots were probed with <span class="gene">Gm-r1088-6964,</span> a EST clone representing <span class="gene">ADR12</span> and corresponding to <span class="gene">Glyma10g35870.1.</span> Whereas there is differential expression in the <span class="trait">seed</span> coats of standard and defective lines, there was no difference in expression of <span class="gene">ADR12</span> in the hypocotyls. (Abbreviations: CS: Clark Standard, CD: Clark defective, HS: Harosoy standard, HD: Harosoy defective).
					</p>
				
					<p>&nbsp;&nbsp;
						The expression pattern of two other hormone regulated genes in <xref ref-type="table" rid="pone-0096342-t003"> Tables 3 </xref> and <xref ref-type="table" rid="pone-0096342-t004"> 4 </xref> that showed significant differential expression in the <span class="trait">seed</span> coat of wild type and defective isolines of both Clark and Harosoy backgrounds is presented in <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figures S6 </xref> -L and <xref ref-type="supplementary-material" rid="pone.0096342.s007"> S7 </xref> -N. The gene that showed a gibberellin regulated annotation had an expression pattern similar to <span class="gene">PRP1</span> with higher expression in defective isolines as compared to the standard <span class="trait">seed</span> coats at 100&#x02013;200 mg <span class="trait">seed</span> weight ( <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figure S6 </xref> -L), while the other gene that had an auxin related annotation showed a different expression pattern with under expression in the defective isoline at the 100&#x02013;200 mg <span class="trait">seed</span> weight stage ( <xref ref-type="supplementary-material" rid="pone.0096342.s007"> Figure S7 </xref> -N).
					</p>
				
					<p>&nbsp;&nbsp;
						Both <span class="gene">Glyma14g35140</span> (Figure A) and <span class="gene">Glyma13g01720</span> (Figure B) gene models had TT1 annotations. The blue line in the graphs represents expression level (RPKM) of these genes in <span class="trait">seed</span> coats of standard isolines and the red line represents expression levels in the defective <span class="trait">seed</span> coats at the 50&#x02013;100 mg, 100&#x02013;200 mg, and 400&#x02013;500 mg <span class="trait">seed</span> weight ranges. There was significant differential expression in these genes at 100&#x02013;200mg <span class="trait">seed</span> weight range. (Abbreviations: CS: Clark standard, CD: Clark defective, HS: Harosoy standard, HD: Harosoy defective).
					</p>
				
					<p>&nbsp;&nbsp;
						Along with the two TT1 gene models, there were 14 total transcription factor genes that showed differential expression in <span class="trait">seed</span> coats of standard and defective isolines in both of the Clark and Harosoy backgrounds at the 100&#x02013;200 mg stage ( <xref ref-type="fig" rid="pone-0096342-g007"> Figure 7 </xref> , <xref ref-type="supplementary-material" rid="pone.0096342.s021"> Table S3 </xref> ). The most prominent category had NAC-related annotations. The other important classes of transcription factor genes were bHLH, zinc finger and ethylene responsive factor genes. The expression of these transcription factor genes at three stages of <span class="trait">seed</span> development is presented in <xref ref-type="supplementary-material" rid="pone.0096342.s014"> Figure S14 </xref> and <xref ref-type="supplementary-material" rid="pone.0096342.s015"> S15 </xref> . There were two transcription factor genes (Myb and bHLH) that showed expression patterns similar to TT1 and <span class="gene">PRP1</span> ( <xref ref-type="supplementary-material" rid="pone.0096342.s014"> Figure S14 </xref> A &#x00026; C). Four of the other transcription factor genes showed expression patterns similar to <span class="gene">PRP2</span> ( <xref ref-type="supplementary-material" rid="pone.0096342.s015"> Figure S15 </xref> A, C, E &#x00026; H). Interestingly, there were no significantly differentially expressed transcription factor genes found in common at the 50&#x02013;100 mg or 400&#x02013;500 mg <span class="trait">seed</span> weight stages in both backgrounds.
					</p>
				
					<p>&nbsp;&nbsp;
						<xref ref-type="supplementary-material" rid="pone.0096342.s020"> Table S2 </xref> contains the data for transcription factors that were differentially expressed between standard and defective <span class="trait">seed</span> coats in only the Clark or the Harsoy background. At the 50&#x02013;100 mg <span class="trait">seed</span> weight stage, there were 59 transcription factor genes that showed differential expression in the seeds coats of only the Clark isolines with the maximum number of genes being related to the zinc finger category ( <xref ref-type="supplementary-material" rid="pone.0096342.s016"> Figure S16 </xref> ) whereas in the Harosoy isolines at the 50&#x02013;100 mg <span class="trait">seed</span> weight stage, <span class="gene">Glyma15g42810.1,</span> was the only transcription factor gene that showed differential expression. Likewise, <xref ref-type="supplementary-material" rid="pone.0096342.s017"> Figure S17 </xref> delineates the functional annotations of the 16 and 12 transcription factors that showed differential expression in the Clark background at the 100&#x02013;200 or 400&#x02013;500 mg stages, respectively, and <xref ref-type="supplementary-material" rid="pone.0096342.s018"> Figure S18 </xref> shows the 43 and 111 transcription factors that were differentially expressed only in the Harosoy line at the later stages of development.
					</p>
				
					<p>&nbsp;&nbsp;
						Auxin is a plant growth regulator that is required for cell expansion, division and differentiation. Auxin pathway related genes have been reported to regulate several cell wall related genes in Arabidopsis <xref rid="pone.0096342-Evans1" ref-type="bibr"> [35] </xref> , <xref rid="pone.0096342-Overvoorde1" ref-type="bibr"> [36] </xref> . Likewise gibberellins had been reported to alter the expression of cell wall loosening genes and to work in coordination with auxin and abscisic acid hormone pathways <xref rid="pone.0096342-Gou1" ref-type="bibr"> [37] </xref> . Genes related to the jasmonic acid pathway and ethylene response may also be related to cell wall formation <xref rid="pone.0096342-Denness1" ref-type="bibr"> [38] </xref> , <xref rid="pone.0096342-Creelman1" ref-type="bibr"> [39] </xref> . As presented in the results ( <xref ref-type="supplementary-material" rid="pone.0096342.s020"> Tables S2 </xref> &#x02013; <xref ref-type="supplementary-material" rid="pone.0096342.s022"> S4 </xref> ), there were differentially expressed genes in the standard and defective isolines that showed ethylene response related, auxin and gibberline hormone related/regulated annotations suggesting an effect of the <span class="trait">seed</span> coat mutation on these genes. The role of the auxin downregulated <span class="gene">ADR12</span> transcript that showed high differential expression in the standard and defective <span class="trait">seed</span> coats but not in the hypocotyls ( <xref ref-type="fig" rid="pone-0096342-g005"> Figure 5 </xref> ) is not known. The predicted peptide form the <span class="gene">ADR12</span> transcript is small at only 71 amino acids. Likely, many of the transcript differences for the affected pathways are confined to the <span class="trait">seed</span> coats as is the case for the <span class="gene">ADR12</span> transcript. Such an extensive disruption of cell wall metabolism by the suite of genes affected in the net pattern mutation that leads to producing tears in the walls would likely be detrimental to the growth of the seedlings but is tolerated in the latter stages of <span class="trait">seed</span> coat development.
					</p>
				
					<p>&nbsp;&nbsp;
						As described in our results, the expression of <span class="gene">PRP1</span> was dramatically higher in the yellow Harosoy isolines (homozygous for <italic> I </italic> alleles), as compared to black Clark isolines (homozygous for the <italic> i </italic> allele) ( <xref ref-type="fig" rid="pone-0096342-g004"> Figure 4 </xref> ). Likewise, another study showed that the <italic> I </italic> locus which controls inhibition of anthocyanin accumulation in the epidermal cells of the soybean <span class="trait">seed</span> coat also affects abundance of <span class="gene">PRP1</span> mRNA and protein in the <span class="trait">seed</span> coat. Interestingly, an epistatic interaction between the recessive <italic> i </italic> and <italic> t </italic> alleles also causes cracking of the pigmented <span class="trait">seed</span> coat <xref rid="pone.0096342-Nicholas1" ref-type="bibr"> [48] </xref> . This <span class="trait">seed</span> coat cracking is not related to the defective <span class="trait">seed</span> coats of the net pattern described in this report which is independent of <span class="trait">seed</span> color as shown in <xref ref-type="fig" rid="pone-0096342-g001"> Figure 1 </xref> . However, this genetic interaction implies an interaction of the flavonoid pathway with cell wall structure as the <italic> t </italic> locus is known to encode a flavonoid 3&#x02032; hydroxylase <xref rid="pone.0096342-Zabala1" ref-type="bibr"> [17] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Our RNA-Seq data ( <xref ref-type="fig" rid="pone-0096342-g004"> Figure 4 </xref> ) show excellent agreement with the previous RNA blots <xref rid="pone.0096342-Percy1" ref-type="bibr"> [10] </xref> indicating a delay in the decline of <span class="gene">PRP1</span> transcripts in the defective <span class="trait">seed</span> coats leading to higher levels in the defective <span class="trait">seed</span> coats at the middle weight range of 100&#x02013;200 mg. Despite the presence of significant levels of <span class="gene">PRP1</span> transcripts in the defective <span class="trait">seed</span> coats, no <span class="gene">PRP1</span> protein was detectable by immunoblotting in defective Clark <span class="trait">seed</span> coats at any stage of <span class="trait">seed</span> development but it was easily extractable from the standard, non-defective isoline at the same stages <xref rid="pone.0096342-Percy1" ref-type="bibr"> [10] </xref> . However, the similar <span class="gene">PRP2</span> protein was extractable from both standard and defective <span class="trait">seed</span> coats. These results implied that a major physiological event in the net pattern defective <span class="trait">seed</span> coats may be the irreversible cross-linking of <span class="gene">PRP1</span> into the cell wall occurring in the developing seeds.
					</p>
				
					<p>&nbsp;&nbsp;
						Transcription factors are important players for controlling the flow of genetic information from DNA to RNA and ultimately affecting the growth and physiology of the plant. In this study, there were approximately 240 differentially expressed transcription factor genes at different <span class="trait">seed</span> weight stages ( <xref ref-type="supplementary-material" rid="pone.0096342.s020"> Tables S2 </xref> &#x02013; <xref ref-type="supplementary-material" rid="pone.0096342.s022"> S4 </xref> ). These genes were divided into different classes as presented in <xref ref-type="fig" rid="pone-0096342-g007"> Figures 7 </xref> , <xref ref-type="supplementary-material" rid="pone.0096342.s016"> S16 </xref> , <xref ref-type="supplementary-material" rid="pone.0096342.s017"> S17 </xref> , and <xref ref-type="supplementary-material" rid="pone.0096342.s018"> S18 </xref> . The major classes of transcription factors genes which showed differential expression in our study were Myb, NAC, basic helix-loop-helix, WRKY, and zinc finger transcription factors. Transcription factors of these types been reported to affect cell wall integrity in other plant species. MYB domain containing transcription factors are involved in secondary cell wall biosynthesis, pollen wall composition, mucilage deposition, extrusion and lignin deposition in Arabidopsis <xref rid="pone.0096342-Zhong1" ref-type="bibr"> [49] </xref> , <xref rid="pone.0096342-Preston1" ref-type="bibr"> [50] </xref> , <xref rid="pone.0096342-Penfield1" ref-type="bibr"> [51] </xref> , <xref rid="pone.0096342-Western1" ref-type="bibr"> [52] </xref> . The MYB46 and <span class="gene">MYB83</span> transcription factors are thought to regulate secondary wall biosynthesis in Arabidopsis <xref rid="pone.0096342-Zhong2" ref-type="bibr"> [53] </xref> , <xref rid="pone.0096342-Zhou1" ref-type="bibr"> [54] </xref> . A basic helix-loop-helix transcription factor had been reported to play an important role in tapetal cell development <xref rid="pone.0096342-Xu1" ref-type="bibr"> [55] </xref> . The role of a MADS-box transcription factor as RIPENING INHIBITOR (RIN) is in cell wall metabolism and carotenoid biosynthesis <xref rid="pone.0096342-Martel1" ref-type="bibr"> [56] </xref> and a basic leucine zipper domain containing transcription factor had been reported to affect pollen coat patterning <xref rid="pone.0096342-Gibalov1" ref-type="bibr"> [57] </xref> . As demonstrated by Wang <italic> et al., </italic> WRKY transcription factors are involved in regulation of downstream genes encoding the NAM, ATAF1/2, and CUC2 (NAC) and CCCH type (C3H) zinc finger TFs that activate secondary wall synthesis <xref rid="pone.0096342-Wang1" ref-type="bibr"> [58] </xref> . NAC domain containing transcription factors were reported to be involved in secondary cell wall biosynthesis and in the regulation of cellulose and hemicellulose biosynthetic genes in addition to those involved in lignin polymerization and signaling in Arabidopsis and Eucalyptus <xref rid="pone.0096342-Hussey1" ref-type="bibr"> [59] </xref> , <xref rid="pone.0096342-Wang2" ref-type="bibr"> [60] </xref> . Rice and maize SWNs (Secondary wall NACs) and MYB46 had been reported as master transcriptional activators of the secondary wall biosynthetic program <xref rid="pone.0096342-Zhong3" ref-type="bibr"> [61] </xref> . The <span class="gene">AP2</span> domain containing family is an ethylene responsive group of transcription factors, and in <italic> Arabidopsis thaliana </italic> , these were expressed in specific cell types of roots, stems and seeds that undergo suberization <xref rid="pone.0096342-Lasserre1" ref-type="bibr"> [62] </xref> . The TT1 genes were another class of the significantly differentially expressed genes in this study ( <xref ref-type="fig" rid="pone-0096342-g006"> Figure 6 </xref> ). These transcription factor genes had been reported to be involved in <span class="trait">seed</span> coat pigmentation and integrity in Arabidopsis <xref rid="pone.0096342-Sagasser1" ref-type="bibr"> [63] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						It is likely that the defective <span class="trait">seed</span> coat mutation in soybean sets in motion changes in many pathways related to the cell wall. Some of the differential expression may be a temporal shift in developmental appearances of transcripts. Notably, in the case of the TT1 homolog, it appears that the decline in its transcript levels was delayed in the defective <span class="trait">seed</span> coats relative to the standard <span class="trait">seed</span> coats ( <xref ref-type="fig" rid="pone-0096342-g006"> Figure 6 </xref> ), thus leading to significant differential expression at the 100&#x02013;200 mg <span class="trait">seed</span> weight range. This profile is similar to the pattern exhibited by the <span class="gene">PRP1</span> protein ( <xref ref-type="fig" rid="pone-0096342-g004"> Figure 4A </xref> ) in that its decline in expression appears to be slowed. Thus, a developmental delay in the normal expression patterns of multiple genes appeared to be a feature of the effect of this mutation as shown in <xref ref-type="supplementary-material" rid="pone.0096342.s006"> Figures S6 </xref> &#x02013; <xref ref-type="supplementary-material" rid="pone.0096342.s010"> S10 </xref> and <xref ref-type="supplementary-material" rid="pone.0096342.s014"> S14 </xref> &#x02013; <xref ref-type="supplementary-material" rid="pone.0096342.s015"> S15 </xref> . These shifts may be one of the reasons for significant differential expression at the mid-seed weight range of 100&#x02013;200 mg and may indicate that the triggering event occurs at or before the 50 mg <span class="trait">seed</span> weight range.
					</p>
				
					<p>&nbsp;&nbsp;
						Total RNA was extracted from the frozen tissue including 4&#x02013;5 day old soybean hypocotyls minus their cotyledons and <span class="trait">seed</span> coats of 100&#x02013;200 mg seeds from four lines using a standard phenol chloroform method with lithium chloride precipitation <xref rid="pone.0096342-McCarty1" ref-type="bibr"> [64] </xref> or the modified method for pigmented tissues <xref rid="pone.0096342-Todd2" ref-type="bibr"> [65] </xref> , <xref rid="pone.0096342-Wang3" ref-type="bibr"> [66] </xref> . RNA samples were quantified by spectrophotometer and the integrity was confirmed using agarose gel electrophoresis. RNA was stored at &#x02212;80&#x000b0;C until further use. For RNA gel blot analysis, 10 &#x000b5;g of total RNA was electrophoresed through 1.2% agarose/1.1% formaldehyde gels <xref rid="pone.0096342-Sambrook1" ref-type="bibr"> [70] </xref> blotted onto nitrocellulose membranes (Schleicher &#x00026; Schuell, Keene, NH) via capillary action with 10&#x000d7; SSC (1.5 M NaCl and 0.15 M sodium citrate, pH&#x0200a;=&#x0200a;7) overnight. After blotting, RNA was cross-linked to the nitrocellulose membranes with UV radiation by a UV cross-linker (Stratagene, La Jolla, CA). Nitrocellulose RNA gel blots were then prehybridized, hybridized, washed, and exposed to Hyperfilm (Amersham, Piscataway,NJ) as described by Todd and Vodkin <xref rid="pone.0096342-Todd1" ref-type="bibr"> [14] </xref> . An <span class="gene">ADR12</span> gene EST was <span class="gene">(Gm-r1088-6964)</span> used as probe and labeled with [&#x003b1;- <sup> 32 </sup> P]dATP by random primer reaction method <xref rid="pone.0096342-Feinberg1" ref-type="bibr"> [71] </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Bayesian model for detection of high-order interactions among genetic variants in genome-wide association studies</h2>
			<h3>PMID:26607428</h3>
				
					<h3>PMCID:4660815</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Juexin </given-names>  </name>  <address>  <email> wangjue@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shi </surname>  <given-names> Haiying </given-names>  </name>  <address>  <email> shiha@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Yanchun </given-names>  </name>  <address>  <email> ycliang@jlu.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhang </surname>  <given-names> Jing </given-names>  </name>  <address>  <email> jzhang47@gsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> College of Computer Science and Technology, Jilin University, Changchun, Jilin China  </aff>  <aff id="Aff2">  <label/> Department of Computer Science, Informatics Institute, and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO USA  </aff>  <aff id="Aff3">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology (NCSB), University of Missouri, Columbia, MO USA  </aff>  <aff id="Aff4">  <label/> Department of Statistics, Computational Biology and Bioinformatics, Yale University, New Haven, CT USA  </aff>  <aff id="Aff5">  <label/> Department of Mathematics and Statistics, Georgia State University, Atlanta, GA USA  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this Strategy C, we ran BHIT on a subset of 799 SNPs from Soy50KSNP array, which overlapped with the protein coding regions and known QTL regions related to protein/oil contents (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Table S9). Then we ran BHIT for about 1000 times on this subset using either oil or protein phenotypes. Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> presents some interesting identified interactions. The first and most interesting interactions among them were identified in 4 loci across two chromosomes located in position 20,897,627; 20,954,490 of Chromosome eight, and 8,642,446; 12,051,017 of Chromosome 19. The first SNP (named as SNP293) is located in gene <span class="gene">Glyma08g26580.1</span> mapped, which has an Arabidopsis homology AT3G0140 (EC/6.3.2.19) and an ubiquitin-protein ligase. At the sequence level, the polymorphism makes the major allele nucleotide guanine (g) replaced by the minor allele nucleotide adenine (a), which causes the 73th amino acid of the protein change from glycine (G) to arginine (R). The added positive charged arginine may have significant impact on the protein conformation and function. The second SNP (named as SNP294) is located in gene <span class="gene">Glyma08g26680.1,</span> which causes the 31st amino acid change from alanine (A) to valine (V). The function of this gene is unknown. The third SNP (named as SNP792) is located in gene <span class="gene">Glyma19g07330.1,</span> which also causes amino acid change from glycine (G) to arginine (R). This gene has the Arabidopsis homolog AT3G48990.1, which encodes an oxalyl-CoA synthetase and is required for oxalate degradation and normal <span class="trait">seed</span> development processes. The fourth SNP (named as SNP794) is located in gene <span class="gene">Glyma19g10100.1,</span> which causes amino acid change from valine (V) to isoleucine (I). This gene has the Arabidopsis homology AT1G51310.1, a tRNA (5-methylaminomethyl&#x02212;2-thiouridylate)-methyltransferases. This transferase is involved in tRNA processing in chloroplast and cytoplasm. It was found that protein <span class="gene">Glyma08g26580.1</span> containing SNP293 and protein <span class="gene">Glyma19g07330.1</span> containing SNP792 were predicted to interact by ProprInt [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. We predicted their protein structures by MUFOLD [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ] . Then we docked the two predicted structures using GRAMMX [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. Interestingly, the distance between the residue containing SNP293 and the residue containing SNP792 was shorter than 0.0052&#x000a0;% of all the paired distances between the two structures, as shown in Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10 </xref> . This suggests that the epistatic interaction between the two SNPs may play a role in the interaction between the two proteins. <table-wrap id="Tab6"> <label> Table 6 </label> <caption> <p> SNPs identified by strategy C using BHIT in soybean data
					</p>
				
					<p>&nbsp;&nbsp;
						Many of the SNP-associated genes in Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> are highly related to oil content according to the literature. It is known that ubiquitin processes <span class="gene">(Glyma08g26580)</span> have significant influences on fatty acid compensation [ <xref ref-type="bibr" rid="CR28"> 28 </xref> , <xref ref-type="bibr" rid="CR29"> 29 </xref> ] . Intracellular composition of fatty acid could affect the processing and function of enzymes in connection with the ubiquitin&#x02013;proteasome pathway, which might be a common physiological approach to regulate protein degradation [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ]. Under the control of a corn ubiquitin promoter, positive expression of puroindoline a and b (PINA and PINB) proteins in transgenic corn could significantly increase the <span class="trait">seed</span> oil content [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. It is also well known that acyl-coenzyme A (CoA) synthase <span class="gene">(Glyma19g07330)</span> catalyses the conversion of free fatty acid to acyl-CoA ester, and is, therefore, necessary for many pathways of fatty acid and lipid metabolism [ <xref ref-type="bibr" rid="CR32"> 32 </xref> ]. Researchers have successfully improved fatty alcohol by manipulating the CoA synthase in fatty alcohol biosynthesis pathway in engineered <italic> E. coli </italic> [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ], and they also succeeded improving fatty acid ethyl esters in <italic> Saccharomyces cerevisiae </italic> [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. Although there is no direct report on the role of tRNA-methyltransferase <span class="gene">(Glyma19g10100)</span> in oil synthesis in plants, a product of tRNA-methyltransferase, S-adenosylhomocysteine was shown to have a metabolic link with the fatty acid metabolism in Alzheimer&#x02019;s disease [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. The potential interaction between <span class="gene">Glyma08g26580</span> and <span class="gene">Glyma19g07330</span> links the ubiquitin and acyl-CoA synthase through NAD(P)/FAD-dependent dehydrogenases [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and characterization of large DNA deletions affecting oil quality traits in soybean seeds through transcriptome sequencing analysis</h2>
			<h3>PMID:27179525</h3>
				
					<h3>PMCID:4943983</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Goettel </surname>  <given-names> Wolfgang </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ramirez </surname>  <given-names> Martha </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Upchurch </surname>  <given-names> Robert G. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> An </surname>  <given-names> Yong-qiang Charles </given-names>  </name>  <address>  <phone> 314-587-1669 </phone>  <email> yong-qiang.an@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> US Department of Agriculture, Agricultural Research Service, Midwest Area, Plant Genetics Research Unit, Donald Danforth Plant Science Center, 975 N. Warson Rd, St. Louis, MO 63132 USA  </aff>  <aff id="Aff2">  <label/> US Department of Agriculture, Agricultural Research Service, Soybean and Nitrogen Fixation Research, 2417 Gardner Hall, Raleigh, NC 27695 USA  </aff>
				<br/> 
				年份：<day> 28 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A large number of soybean genotypes varying in <span class="trait">seed</span> oil composition and content have been identified. Understanding the molecular mechanisms underlying these variations is important for breeders to effectively utilize them as a genetic resource. Through design and application of a bioinformatics approach, we identified nine co-regulated gene clusters by comparing <span class="trait">seed</span> transcriptomes of nine soybean genotypes varying in oil composition and content. We demonstrated that four gene clusters in the genotypes M23, Jack and N0304-303-3 coincided with large-scale genome rearrangements. The co-regulated gene clusters in M23 and Jack mapped to a previously described 164-kb deletion and a copy number amplification of the <italic> <span class="gene">Rhg1</span> </italic> locus, respectively. The coordinately down-regulated gene clusters in N0304-303-3 were caused by a 254-kb deletion containing 19 genes including a <italic> fatty acyl </italic> - <italic> ACP thioesterase B gene </italic> ( <italic> <span class="gene">FATB1a</span> </italic> ). This deletion was associated with reduced palmitic acid content in seeds and was the molecular cause of a previously reported nonfunctional <italic> <span class="gene">FATB1a</span> </italic> allele, <italic> fap </italic> <sub> <italic> nc </italic> </sub> . The M23 and N0304-304-3 deletions were located in duplicated genome segments retained from the <italic> Glycine </italic> -specific whole genome duplication that occurred 13&#x000a0;million years ago. The homoeologous genes in these duplicated regions shared a strong similarity in both their encoded protein sequences and transcript accumulation levels, suggesting that they may have conserved and important functions in seeds. The functional conservation of homoeologous genes may result in genetic redundancy and gene dosage effects for their associated <span class="trait">seed</span> traits, explaining why the large deletion did not cause lethal effects or completely eliminate palmitic acid in N0304-303-3.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is an allotetraploid species with a complex genome. The soybean genome has undergone two whole-genome duplication events that occurred 13 and 59&#x000a0;million years ago (Schmutz et al. <xref ref-type="bibr" rid="CR33"> 2010 </xref> ). It is estimated that about 75&#x000a0;% of genes are retained in multiple copies. Many of the duplicated genes have conserved protein functions (Lin et al. <xref ref-type="bibr" rid="CR21"> 2010 </xref> ; Roulin et al. <xref ref-type="bibr" rid="CR30"> 2012 </xref> ). The two whole genome duplications created genetic redundancy and dosage effects for many phenotypic traits in soybean, and make genetic engineering for fatty acid composition more complicated and less predictable. Enzymatic reactions in the fatty acid metabolic pathway are often determined by a combination of more than two homoeologous genes (Pham et al. <xref ref-type="bibr" rid="CR24"> 2010 </xref> ). For example, the delta-twelve fatty acid desaturase 2 enzymes (FAD2) are encoded by a multi-gene family. The <italic> <span class="gene">FAD2</span> </italic> - <italic> 2 </italic> sub-family, which consists of three genes, is mainly expressed in soybean vegetative tissues while the two microsomal <italic> <span class="gene">FAD2</span> </italic> - <italic> 1 </italic> genes [ <italic> <span class="gene">FAD2</span> </italic> - <italic> 1A </italic> <span class="gene">(Glyma10g42470)</span> and <italic> <span class="gene">FAD2</span> </italic> - <italic> 1B </italic> <span class="gene">(Glyma20g24530)]</span> are mainly expressed in developing seeds. It has been shown that soybean genotypes carrying both mutated alleles of <italic> <span class="gene">FAD2</span> </italic> - <italic> 1A </italic> and <italic> <span class="gene">FAD2</span> </italic> - <italic> 1B </italic> have an average of 82&#x02013;86&#x000a0;% oleic acid content, which is significantly higher than that of soybean genotypes containing each individual mutated allele (Pham et al. <xref ref-type="bibr" rid="CR25"> 2011 </xref> ). Understanding the contribution of each homoeologous gene to each enzymatic activity in soybean seeds facilitates the design of effective breeding strategies towards improvement of the soybean <span class="trait">seed</span> fatty acid profile.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, we applied next-generation sequencing technology to sequence <span class="trait">seed</span> transcriptomes from nine soybean genotypes varying in oil content and composition, and showed that genetic variation results in the expression change of thousands of genes (Goettel et al. <xref ref-type="bibr" rid="CR18"> 2014 </xref> ). To identify and characterize large structural genome rearrangements that affect the fatty acid composition of soybean seeds we further explored our transcriptome sequencing data. We developed a bioinformatics approach to identify several co-regulated gene clusters, and demonstrated that some of them are caused by large-scale genome segment deletions and amplifications. We identified a novel 254-kb deletion in genotype N0304-303-3 that includes the <italic> <span class="gene">FATB1a</span> </italic> gene encoding a fatty acyl-ACP thioesterase B. N0304-303-3 contains a reduced level of palmitic acid and also has altered levels of other major fatty acids (Goettel et al. <xref ref-type="bibr" rid="CR18"> 2014 </xref> ) (Suppl. Figure&#x000a0;1). We showed that the low palmitic acid content in N0304-303-3 was highly associated with the large deletion, suggesting that loss of <span class="gene">FATB1a</span> activity likely reduces palmitic acid in N0304-303-3. The deletion is located in a duplicated genome segment retained from the <italic> Glycine </italic> -specific whole genome duplication. Most genes in the deleted region shared a high similarity with their homoeologous genes in their expression patterns and protein sequences, which may have led to genetic redundancy and dosage effects of those homoeologs. We propose that loss of those genes due to the deletion event does not cause lethal and null phenotypes in N0304-303-3 because their homoeologous genes completely or partially complement their function.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean [ <italic> Glycine max </italic> (L.) Merrill] seeds for PI 90406-1, PI 92567, N69-2774, and N79-2077-12 were received from the USDA Soybean Germplasm Collection ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/npgs/"> http://www.ars-grin.gov/npgs/ </ext-link> ). Seeds for the remaining genotypes were provided by Dr. Joe Burton (USDA-ARS in Raleigh, NC). Genomic DNA was isolated from 100&#x000a0;mg of mature <span class="trait">seed</span> tissue using the DNeasy Plant Mini kit (Qiagen, Valencia, CA). PCR amplification reactions were conducted with the Expand High Fidelity Plus system from Roche (Mannheim, Germany) in a 25&#x000a0;&#x000b5;l volume containing 50&#x000a0;ng genomic DNA, 200&#x000a0;nM each primer, 2.5&#x000a0;mM MgCl <sub> 2 </sub> , and 200&#x000a0;&#x000b5;M deoxyribonucleotide phosphates (dNTPs). Amplification condition was 1 cycle at 94&#x000a0;&#x000b0;C for 4&#x000a0;min, followed by 30 cycles at 94&#x000a0;&#x000b0;C for 30&#x000a0;s, primer annealing temperature for 30&#x000a0;s, and 72&#x000a0;&#x000b0;C for 30&#x000a0;s. A final extension was conducted at 72&#x000a0;&#x000b0;C for 15&#x000a0;min. For sequencing, amplicons were cloned into vector pCR2.1 with the TOPO TA cloning kit supplied by Invitrogen (Carlsbad, CA). Cloned fragments were sequenced at the Iowa State University Biotechnology Center, Ames, IA. A primer pair was designed to amplify an 805-bp product from <italic> G. max </italic> N0304-303-3 in the region of chromosome 5 flanking the 254-kb deletion (forward primer GCCAGCTTAATGATCAGTTT, reverse primer TAGAACCACTCAGTGTTCTTTG, annealing temperature 57&#x000a0;&#x000b0;C). Another primer pair that amplifies a 530-bp region of <italic> <span class="gene">FATB1a</span> </italic> <span class="gene">(Glyma05g08060)</span> was designed to be used in conjunction with the amplification reactions above (forward primer CAAGTGGACACTTGGGTTTC, reverse primer GCCTCCGTGTTAGCTTATTC, annealing temperature 61&#x000a0;&#x000b0;C).
					</p>
				
					<p>&nbsp;&nbsp;
						In a recent study, we sequenced the <span class="trait">seed</span> transcriptomes of nine soybean genotypes that vary in oil composition and content and identified a set of transcript SNPs and small indels (Goettel et al. <xref ref-type="bibr" rid="CR18"> 2014 </xref> ). Here, we developed a computational method to identify co-regulated gene clusters and predict large genome segment deletions and amplifications by comparing transcriptome sequencing data from nine genotypes. In our approach, we used the <italic> Z </italic> score to measure expression differences among the genotypes for each gene. The genomic regions containing adjacent genes within a genotype with positive or negative <italic> Z </italic> score values for each transcript above or below a threshold indicates that those genes are coordinately up-regulated or down-regulated. Genomic regions containing adjacent down-regulated genes with little or no transcript accumulation are indicative of putative deletions. We applied the algorithm in the analysis of the nine genotypes and identified four genomic regions in three genotypes (M23, N0304-303-3 and R02-6268F) that each contained more than three adjacent down-regulated genes with a <italic> Z </italic> score value of &#x02264;&#x02212;2 (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The genomic region in M23 mapped to a previously identified 164-kb deletion on chromosome 10 (Bolon et al. <xref ref-type="bibr" rid="CR5"> 2011 </xref> ; Rahman et al. <xref ref-type="bibr" rid="CR26"> 1994 </xref> ). The two co-regulated regions in N0304-303-3 were caused by a 254-kb deletion (see below). Additionally, we identified five genomic regions in four genotypes (Jack, FAM94-41, R05-591 and R08-1450) that each contained more than three adjacent up-regulated genes with <italic> Z </italic> score values of more than &#x000a0;or equal to 2 (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The coordinately up-regulated genome region in Jack coincided with the <italic> <span class="gene">Rhg1</span> </italic> locus (for <underline> r </underline> esistance to <italic> <underline> H </underline> </italic> <italic> eterodera </italic> <italic> <underline> g </underline> </italic> <italic> lycines </italic> ), which is composed of four adjacent genes <span class="gene">(Glyma18g02580,</span> <span class="gene">Glyma18g02590,</span> <span class="gene">Glyma18g02600</span> and <span class="gene">Glyma18g02610).</span> This finding is consistent with our previous report showing the expression of the four genes at the <italic> <span class="gene">Rhg1</span> </italic> locus was tenfold up-regulated due to a tenfold genome segment amplification in Jack (Goettel et al. <xref ref-type="bibr" rid="CR18"> 2014 </xref> ). Thus, this computational algorithm could be used effectively to both identify genome regions containing coordinately regulated genes and to predict large structural genome variation such as deletions and amplifications. <table-wrap id="Tab1"> <label> Table&#x000a0;1 </label> <caption> <p> Co-regulated gene clusters identified in nine genotypes
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of the <italic> <span class="gene">FAD2</span> </italic> Gene Family in Soybean Reveals the Limitations of Gel-Based TILLING in Genes with High Copy Number</h2>
			<h3>PMID:28348573</h3>
				
					<h3>PMCID:5346563</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lakhssassi </surname>  <given-names> Naoufal </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/419843/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Zhou </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/403415/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/394633/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Colantonio </surname>  <given-names> Vincent </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/394642/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> AbuGhazaleh </surname>  <given-names> Amer </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/419729/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> Khalid </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/166396/overview"/>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="trait">seed</span> oil typically contains 18&#x02013;20% oleic acid. Increasing the content of oleic acid is beneficial for health and biodiesel production. Mutations in <italic> <span class="gene">FAD2-1</span> </italic> genes have been reported to increase <span class="trait">seed</span> oleic acid content. A subset of 1,037 mutant families from a mutagenized soybean cultivar (cv.) Forrest population was screened using reverse genetics (TILLING) to identify mutations within <italic> <span class="gene">FAD2</span> </italic> genes. Although no <italic> fad2 </italic> mutants were identified using gel-based TILLING, four <italic> fad2-1A </italic> and one <italic> fad2-1B </italic> mutants were identified to have high <span class="trait">seed</span> oleic acid content using forward genetic screening and subsequent target sequencing. TILLING has been successfully used as a non-transgenic reverse genetic approach to identify mutations in genes controlling important agronomic traits. However, this technique presents limitations in traits such as oil composition due to gene copy number and similarities within the soybean genome. In soybean, <italic> <span class="gene">FAD2</span> </italic> are present as two copies, <italic> <span class="gene">FAD2-1</span> </italic> and <italic> FAD2-2. Two <span class="gene">FAD2-1</span> </italic> members: <span class="gene">FAD2-1A</span> and FAD2-1B; and three <italic> <span class="gene">FAD2-2</span> </italic> members: <italic> FAD2-2A, FAD2-2B </italic> , and <italic> FAD2-2C </italic> have been reported. Syntenic, phylogenetic, and <italic> in silico </italic> analysis revealed two additional members constituting the <italic> <span class="gene">FAD2</span> </italic> gene family: <italic> <span class="gene">GmFAD2-2D</span> </italic> and <italic> <span class="gene">GmFAD2-2E</span> </italic> , located on chromosomes 09 and 15, respectively. They are presumed to have diverged from other <italic> <span class="gene">FAD2-2</span> </italic> members localized on chromosomes 19 ( <italic> <span class="gene">GmFAD2-2A</span> </italic> and <italic> <span class="gene">GmFAD2-2B</span> </italic> ) and 03 ( <italic> <span class="gene">GmFAD2-2C</span> </italic> ). This work discusses alternative solutions to the limitations of gel-based TILLING in functional genomics due to high copy number and multiple paralogs of the <italic> <span class="gene">FAD2</span> </italic> gene family in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Three sources of <italic> <span class="gene">FAD2-1A</span> </italic> mutants were previously described: M23 carrying a large genomic deletion of 160 Kb; KK21, a mutant with a single nucleotide deletion; and 17D (S117N) were combined with one <italic> <span class="gene">FAD2-1B</span> </italic> mutant (P137R) from PI283327 to produce high oleic acid lines (Dierking and Bilyeu, <xref rid="B16" ref-type="bibr"> 2009 </xref> ; Hoshino et al., <xref rid="B23" ref-type="bibr"> 2010 </xref> ; Pham et al., <xref rid="B35" ref-type="bibr"> 2010 </xref> ; Bolon et al., <xref rid="B4" ref-type="bibr"> 2011 </xref> ). However, many of these lines presented poor agronomic characteristics such as a reduction in oleic acid content when grown in cooler environments (17D), a reduction in yields, and/or susceptibility to pathogens (i.e., SCN, SDS, etc.) (Taylor et al., <xref rid="B46" ref-type="bibr"> 2002 </xref> ; Anai et al., <xref rid="B1" ref-type="bibr"> 2008 </xref> ; Pham et al., <xref rid="B35" ref-type="bibr"> 2010 </xref> ). The M23-derived lines were shown to have a reductions in <span class="trait">yield</span> that appear to be linked to the deleted portion of chromosome 10 (Taylor et al., <xref rid="B46" ref-type="bibr"> 2002 </xref> ). Thus, developed mutants like <italic> <span class="gene">FAD2-1A</span> </italic> from the M23 line with a large genomic deletion is agronomically impractical. Consequently, producing new EMS mutants with a single base pair mutation in each <italic> <span class="gene">FAD2-1</span> </italic> gene seems to be a better strategy to produce high oleic mutant alleles with stable oil content, particularly after combining both mutant alleles into one line (Pham et al., <xref rid="B36" ref-type="bibr"> 2011 </xref> ). This may result in soybean lines with more than 80% oleic acid content, as it was reported in two previous independent studies (Hoshino et al., <xref rid="B23" ref-type="bibr"> 2010 </xref> ; Pham et al., <xref rid="B36" ref-type="bibr"> 2011 </xref> ). Since then, several efforts have been accomplished to improve soybean <span class="trait">seed</span> composition traits (Dierking and Bilyeu, <xref rid="B16" ref-type="bibr"> 2009 </xref> ; Pham et al., <xref rid="B37" ref-type="bibr"> 2012 </xref> ; Lestari et al., <xref rid="B27" ref-type="bibr"> 2013 </xref> ; Chaudhary et al., <xref rid="B11" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A soybean ( <italic> Glycine max </italic> ) EMS mutagenized population employing the cultivar &#x0201c;Forrest&#x0201d; was used to identify soybean lines carrying mutations in both the <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> genes by reverse genetic and forward genetic approaches. Using gel-based TILLING, no mutant was identified within the <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> , however, a forward genetic approach based on gas chromatography screening identified five soybean lines with high oleic acid <span class="trait">seed</span> content. Through target sequencing, new alleles presenting mutations in either the <italic> <span class="gene">FAD2-1A</span> </italic> or <italic> <span class="gene">FAD2-1B</span> </italic> were identified.
					</p>
				
					<p>&nbsp;&nbsp;
						Five fatty acid desaturase ( <italic> <span class="gene">GmFAD2</span> </italic> ) gene family members were previously reported to be located on chromosomes 10 ( <italic> <span class="gene">GmFAD2-1A</span> </italic> ; <span class="gene">Glyma.10G278000),</span> 20 ( <italic> <span class="gene">GmFAD2-1B</span> </italic> ; <span class="gene">Glyma.20G111000),</span> 19 ( <italic> <span class="gene">GmFAD2-2A</span> </italic> ; <span class="gene">Glyma.19G147300),</span> 19 ( <italic> <span class="gene">GmFAD2-2B</span> </italic> ; <span class="gene">Glyma.19G147400),</span> and 03 ( <italic> <span class="gene">GmFAD2-2C</span> </italic> ; <span class="gene">Glyma.03G144500)</span> (Schlueter et al., <xref rid="B41" ref-type="bibr"> 2007 </xref> ; Pham et al., <xref rid="B35" ref-type="bibr"> 2010 </xref> ; Zhang et al., <xref rid="B50" ref-type="bibr"> 2014 </xref> ). The analysis of the <italic> G. max </italic> <sub> [Williams 82] </sub> genome using both soybase and phytozome databases indicates that two novel <italic> <span class="gene">FAD2-2</span> </italic> members were found on chromosomes 09 <span class="gene">(Glyma.09G111900)</span> and 15 <span class="gene">(Glyma.15G195200)</span> and named in this study as <italic> <span class="gene">GmFAD2-2D</span> </italic> and <italic> <span class="gene">GmFAD2-2E</span> </italic> , respectively (Table <xref ref-type="table" rid="T1"> 1 </xref> , Figures <xref ref-type="supplementary-material" rid="SM7"> S7 </xref> , <xref ref-type="supplementary-material" rid="SM9"> S9 </xref> ). In particular, <italic> <span class="gene">GmFAD2-1A</span> </italic> and <italic> <span class="gene">GmFAD2-1B</span> </italic> present the <span class="trait">seed</span> specific isoforms in soybean with very high expression levels in the seeds (Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The enzyme <span class="gene">FAD2</span> converts oleic acid to linoleic acid (Figure <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ). In the soybean genome, there are two seed-specific <italic> <span class="gene">FAD2</span> </italic> genes, <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> (Table <xref ref-type="table" rid="T1"> 1 </xref> ). In order to identify possible mutations in the <italic> <span class="gene">FAD2</span> </italic> genes and to test the direct correlation between the <italic> <span class="gene">FAD2</span> </italic> mutations and <span class="trait">seed</span> oleic acid phenotype alterations, the genomic sequences of the two <italic> <span class="gene">FAD2-1</span> </italic> isoforms of the three mutants (F1274, F1284, and F812) and the wild type Forrest were analyzed by target sequencing.
					</p>
				
					<p>&nbsp;&nbsp;
						One <span class="trait">seed</span> from each of the 1,037 M3 mutant families was screened for fatty acid content. A set of 14 lines within the M3 population with high oleic acid content was selected. However, only three mutant lines: F812 (FAD2-1B <sub> P163S </sub> ), F1274 (FAD2-1A <sub> L249 = </sub> ), and F1284 (FAD2-1A <sub> P284L </sub> ), characterized previously maintained a significantly high oleic acid content in the M4 generation. The remaining 11 mutants showed no oleic acid content changes. The results of fatty acid analysis showed that three M3 lines were segregating for their oleic acid content (Table <xref ref-type="supplementary-material" rid="SM12"> S2 </xref> ). The M3 mutants were then advanced to the M4 generation by planting one <span class="trait">seed</span> from each one of the 1,037 mutant families. It is likely that the selected <span class="trait">seed</span> from each of the 11 M3 lines that were identified to have high oleic acid were segregating for the non-mutant alleles when advanced to the M4 generation. For this reason, they did not present a higher oleic acid content. In order to support this hypothesis, another two seeds from each of these 11 mutant M2 and M3 families were re-screened when available. The fatty acid screening showed that two mutants from both the M2 and M3 generations maintained a high oleic acid content at least 1.5 times more than the level of oleic acid contained in the wild type Forrest ( <bold> Table 6 </bold> ). Six mutants presented higher oleic acid content in either the M2 or M3 generations, while the other three mutants did not show any significant changes in oleic acid content (Table <xref ref-type="supplementary-material" rid="SM13"> S3 </xref> ). In addition, only the two mutants F784 ( <italic> P </italic> &#x0003c; 0.01) and F1235 ( <italic> P </italic> &#x0003c; 0.05) maintained significantly higher <span class="trait">seed</span> oleic acid content in both M2 and M3 generations (Figure <xref ref-type="fig" rid="F5"> 5 </xref> ). These 11 mutants were also genotyped for both <italic> <span class="gene">FAD2-1</span> </italic> paralogs. Interestingly, target sequencing revealed that only the two mutants maintaining significantly higher content from both the M2 and M3 generations are new alleles in <italic> <span class="gene">FAD2-1A</span> </italic> presenting two missense mutations: <span class="gene">FAD2-1A</span> <sub> R22G </sub> and <span class="gene">FAD2-1A</span> <sub> L101F </sub> , respectively (Figures <xref ref-type="fig" rid="F3"> 3B </xref> , <xref ref-type="fig" rid="F6"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The purpose of this study was the identification of soybean lines with high oleic acid levels, by screening the developed EMS mutagenized Forrest library using a gel-based TILLLING approach. We developed several soybean EMS mutagenized populations and screened multiple soybean mutants using TILLING during the past 10 years (Cooper et al., <xref rid="B14" ref-type="bibr"> 2008 </xref> ; Meksem et al., <xref rid="B31" ref-type="bibr"> 2008 </xref> ; Liu et al., <xref rid="B29" ref-type="bibr"> 2011 </xref> , <xref rid="B28" ref-type="bibr"> 2012 </xref> ). Using gel-based TILLING we were able to identify different genes involved in non-complex traits like SCN <span class="trait">resistance</span> including a serine hydroxymethyltransferase (SHMT) and a Leucine Rich Repeat (LRR-RLK) (Liu et al., <xref rid="B29" ref-type="bibr"> 2011 </xref> , <xref rid="B28" ref-type="bibr"> 2012 </xref> ). However, this method presented limitations in functional gene analysis in the case of complex traits (i.e., oil) due to the high copy number and multiple paralogs of the <italic> <span class="gene">FAD2</span> </italic> gene family in the soybean genome. Most likely, during the initial steps of the PCR, although the designed primers were specific to the targeted gene, the possibility of amplification of the other <italic> <span class="gene">FAD2</span> </italic> copies was present because of the high similarities shared between those genes. Therefore, these non-targeted genes were present within the amplicons more than with just their initial copy number. In this case, the PCR products anneal with the other <italic> <span class="gene">FAD2</span> </italic> copies, and the heteroduplex formed contain both the targeted amplicon from the PCR product and the non-specific amplicons. Forming the heteroduplex during TILLING resulted in non-specific hybridizations leading to the detection of false positive mutants by the mismatch enzyme.
					</p>
				
					<p>&nbsp;&nbsp;
						The phylogenetic analysis within the <italic> Glycine max <span class="gene">FAD2</span> </italic> gene family indicates that <span class="gene">GmFAD2-1A</span> and <span class="gene">GmFAD2-1B</span> proteins form a separate clade from the five <span class="gene">GmFAD2-2</span> members (Figure <xref ref-type="fig" rid="F2"> 2 </xref> ). Our mutational analysis demonstrated that only mutations within <span class="gene">FAD2-1</span> members affect <span class="trait">seed</span> oleic acid content in soybean. In addition, data from RNAseq shows that the two microsomal <italic> <span class="gene">FAD2-1</span> </italic> desaturases <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> were mainly expressed in developing seeds and constituted the <span class="trait">seed</span> specific paralogs in the soybean genome (Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ). However, the expression profile of the five members constituting the <italic> <span class="gene">GmFAD2-2</span> </italic> gene family was different. While <italic> <span class="gene">GmFAD2-2B</span> </italic> and <italic> <span class="gene">GmFAD2-2C</span> </italic> were found to display ubiquitous expression in all the vegetative tissues of the soybean plant, <italic> <span class="gene">GmFAD2-2D</span> </italic> was expressed in the flower, <span class="trait">seed,</span> and nodule, while <italic> <span class="gene">GmFAD2-2E</span> </italic> expression was exclusively confined to the pod and <span class="trait">seed</span> with low levels of expression. The exception was <italic> <span class="gene">GmFAD2-2A</span> </italic> , in which no expression was detected. <italic> <span class="gene">GmFAD2-2A</span> </italic> has a deletion of 100 bp in the coding region and therefore was predicted to be non-functional (Pham et al., <xref rid="B35" ref-type="bibr"> 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Forward genetics screening was used to detect mutations in genes of interest. Gas chromatography was used for the identification of novel alleles with high oleic acid content and their associated mutations were identified by targeted gene sequencing. Phenomics is an important field of research that is consistently improving and can be used as an alternative strategy. Interestingly, using the forward genetic approach, high oleic acid soybean mutant lines were identified to carry missense mutations in either the <italic> <span class="gene">FAD2-1A</span> </italic> or <italic> <span class="gene">FAD2-1B</span> </italic> paralogs (Figures <xref ref-type="fig" rid="F3"> 3B </xref> , <xref ref-type="fig" rid="F4"> 4B </xref> ). Oleic acid content in the <span class="trait">seed</span> oil of the mutants was up to 50.86% (Table <xref ref-type="supplementary-material" rid="SM12"> S2 </xref> ). However, the levels of palmitic acid, stearic acid, and linolenic acid in <span class="trait">seed</span> oil were unchanged, with the exception of a small decrease in linoleic acid compared to the wild type Forrest (Tables <xref ref-type="table" rid="T5"> 5 </xref> , <xref ref-type="table" rid="T6"> 6 </xref> ). Acknowledging the role of <span class="gene">FAD2</span> in converting oleic acid to linoleic acid in the fatty acid biosynthetic pathway (Figure <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> ), the decrease in linoleic acid is possibly caused by the reduced activity of <span class="gene">FAD2</span> members. Reduction of linoleic acid in these mutants is regarded as an agronomic improvement. It has been demonstrated that high linoleic acid content in soybean oil is potentially negative due to the associated reduction of healthy omega-3 fatty acids (Clark et al., <xref rid="B12" ref-type="bibr"> 1992 </xref> ; Friesen and Innis, <xref rid="B19" ref-type="bibr"> 2010 </xref> ; Blasbalg et al., <xref rid="B3" ref-type="bibr"> 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The availability of high-throughput phenotyping methods that allow for the identification of mutants with high oleic acid levels provide us with an alternative way to identify these mutants. However, a forward-screening approach also presents some limitations, including the cost and time involved in the phenotyping process. In addition, only a fraction of these mutants could be linked to mutations within well-known genes for fatty acid biosynthesis. Stability in the desired traits was shown in some mutants when advanced to the next generation. Half of the mutants identified were not stable once advanced to the next generation because of two possibilities. The first one is that the seeds that were taken to advance the population from the M2 family were segregating for the desired high oleic level. This presents 78% of the identified mutants, from which the original M2 <span class="trait">seed</span> stock was retested in order to identify mutations linked to the fatty acid biosynthesis pathway. The rest of the seeds that were found to show high oleic acid content were not linked to <italic> <span class="gene">FAD2</span> </italic> genes. This was most likely due to the genotype, epistatic environmental interactions, or other genes that may contribute to fatty acid biosynthesis indirectly. The mutants carrying mutations within the <italic> <span class="gene">FAD2-1</span> </italic> genes were identified by target sequencing; therefore, it's suggested to use TILLING by sequencing to replace the gel-based system in future works.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we showed the benefit of using ethyl-methanesulfonate (EMS) mutagenesis, which alkylates DNA and can cause significant changes to the genes and gene networks underlying oil biosynthesis. These changes are hard to accomplish under conventional breeding programs, which require diverse genetic material, and are usually more expensive and time-consuming than mutation breeding. It has also been reported that the highest oleic acid <span class="trait">seed</span> content due to natural variations in the soybean germplasm was around 48% (Pham et al., <xref rid="B36" ref-type="bibr"> 2011 </xref> ). However, most of the tested lines were plant introductions (PIs) with poor agronomic performance. In this study, a cultivar was used to produce allele variants within <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> , with oleic acid levels up to 50%. The Forrest cultivar could be used to introgress the high oleic acid content trait into high-yielding lines without compromising their agronomic performance. EMS mutagenesis is also less time-consuming as two seasons were sufficient to develop an EMS-mutagenized population. This is a highly efficient technique for the characterization and identification of multiple beneficial agronomic traits in terms of time, cost, and productivity, not only in soybeans, but also in other plant crops.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression patterns of the six soybean <italic> <bold> <span class="gene">FAD2</span> </bold> </italic> gene members <italic> <bold> in planta </bold> </italic> , based on Soyseq resource available from RNAsequencing data ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org/soyseq"> <bold> http://www.soybase.org/soyseq </bold> </ext-link> ) </bold> . The figure shows that both <italic> <span class="gene">FAD2-1A</span> </italic> and <italic> <span class="gene">FAD2-1B</span> </italic> encode the <span class="trait">seed</span> specific paralogs of <italic> <span class="gene">FAD2-1</span> </italic> , while <italic> FAD2-2B </italic> and <italic> FAD2-2C </italic> paralogs are expressed in the vegetative tissues and the development of seeds. <italic> FAD2-2A </italic> expression was not detected. <italic> FAD2-2E </italic> expression was limited to the <span class="trait">seed</span> and pods, while <italic> FAD2-2D </italic> expression was detected in the flower, <span class="trait">seed,</span> and nodules.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional characterization of <span class="gene">GmBZL2</span> (AtBZR1 like gene) reveals the conserved BR signaling regulation in Glycine max.</h2>
			<h3>PMID:27498784</h3>
				

				
			<p>
				作者：Zhang Y(1)(2), Zhang YJ(3), Yang BJ(4), Yu XX(2)(5), Wang D(5), Zu SH(3), Xue HW(4), Lin WH(3)(6).
				<br/> 
				年份：2016 Aug 8
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BRs) play key roles in plant growth and development, and regulate various agricultural traits. Enhanced BR signaling leads to increased <span class="trait">seed</span> number and <span class="trait">yield</span> in Arabidopsis bzr1-1D (AtBZR1(P234L), gain-of-function mutant of the important transcription factor in BR signaling/effects). BR signal transduction pathway is well elucidated in Arabidopsis but less known in other species. Soybean is an important dicot crop producing edible oil and protein. Phylogenetic analysis reveals AtBZR1-like genes are highly conserved in angiosperm and there are 4 orthologues in soybean <span class="gene">(GmBZL1-4).</span> We here report the functional characterization of <span class="gene">GmBZL2</span> (relatively highly expresses in flowers). The P234 site in AtBZR1 is conserved in <span class="gene">GmBZL2</span> (P216) and mutation of <span class="gene">GmBZL2(P216L)</span> leads to <span class="gene">GmBZL2</span> accumulation. <span class="gene">GmBZL2(P216L)</span> <span class="gene">(GmBZL2*)</span> in Arabidopsis results in enhanced BR signaling; including increased <span class="trait">seed</span> number per silique. <span class="gene">GmBZL2*</span> partially rescued the defects of bri1-5, further demonstrating the conserved function of <span class="gene">GmBZL2</span> with AtBZR1. BR treatment promotes the accumulation, nuclear localization and dephosphorylation/phosphorylation ratio of <span class="gene">GmBZL2,</span> revealing that <span class="gene">GmBZL2</span> activity is regulated conservatively by BR signaling. Our studies not only indicate the conserved regulatory mechanism of <span class="gene">GmBZL2</span> and BR signaling pathway in soybean, but also suggest the potential application of <span class="gene">GmBZL2</span> in soybean <span class="trait">seed</span> <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soybean overexpressing <span class="gene">GmSAMT1</span> exhibits <span class="trait">resistance</span> to multiple-HG types of soybean cyst nematode Heterodera glycines.</h2>
			<h3>PMID:27064027  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin J(1), Mazarei M(1), Zhao N(2), Hatcher CN(1), Wuddineh WA(1), Rudis M(1), Tschaplinski TJ(2), Pantalone VR(1), Arelli PR(3), Hewezi T(1), Chen F(1), Stewart CN Jr(4).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) salicylic acid methyl transferase <span class="gene">(GmSAMT1)</span> catalyses the conversion of salicylic acid to methyl salicylate. Prior results showed that when <span class="gene">GmSAMT1</span> was overexpressed in transgenic soybean hairy roots, <span class="trait">resistance</span> is conferred against soybean cyst nematode (SCN), Heterodera glycines Ichinohe. In this study, we produced transgenic soybean overexpressing <span class="gene">GmSAMT1</span> and characterized their response to various SCN races. Transgenic plants conferred a significant reduction in the development of SCN HG type 1.2.5.7 (race 2), HG type 0 (race 3) and HG type 2.5.7 (race 5). Among transgenic lines, <span class="gene">GmSAMT1</span> expression in roots was positively associated with SCN <span class="trait">resistance.</span> In some transgenic lines, there was a significant decrease in salicylic acid titer relative to control plants. No significant <span class="trait">seed</span> <span class="trait">yield</span> differences were observed between transgenics and control soybean plants grown in one greenhouse with 22 °C day/night temperature, whereas transgenic soybean had higher <span class="trait">yield</span> than controls grown a warmer greenhouse (27 °C day/23 °C night) temperature. In a 1-year field experiment in Knoxville, TN, there was no significant difference in <span class="trait">seed</span> <span class="trait">yield</span> between the transgenic and nontransgenic soybean under conditions with negligible SCN infection. We hypothesize that <span class="gene">GmSAMT1</span> expression affects salicylic acid biosynthesis, which, in turn, attenuates SCN development, without negative consequences to soybean <span class="trait">yield</span> or other morphological traits. Thus, we conclude that <span class="gene">GmSAMT1</span> overexpression confers broad <span class="trait">resistance</span> to multiple SCN races, which would be potentially applicable to commercial production.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Arabidopsis KLU homologue <span class="gene">GmCYP78A72</span> regulates <span class="trait">seed</span> size in soybean.</h2>
			<h3>PMID:26482479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao B(1)(2), Dai A(2), Wei H(2), Yang S(3), Wang B(1), Jiang N(4), Feng X(5)(6).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is one of the most important crops in the world, and its <span class="trait">yield</span> is largely determined by grain weight and grain size. However, the genes that regulate soybean <span class="trait">seed</span> size have not been identified. CYP78A, which is highly conserved within terrestrial plants, regulates organ development. In Arabidopsis, AtCYP78A5/KLU has been shown to determine <span class="trait">seed</span> size. In the present study, soybean CYP78A72 <span class="gene">(GmCYP78A72),</span> one of the orthologs of KLU, was over-expressed in both Arabidopsis and soybean to examine its function in plant development. <span class="gene">GmCYP78A72</span> heterologous expression in Arabidopsis resulted in enlarged sepals, petals, seeds and carpel. Over-expression of <span class="gene">GmCYP78A72</span> in soybean resulted in increased pea size, which is an extremely desirable trait for enhancing productivity. Moreover, knock-down of <span class="gene">GmCYP78A72</span> does not reduce grain size. However, silencing of <span class="gene">GmCYP78A57,</span> <span class="gene">GmCYP78A70</span> and <span class="gene">GmCYP78A72</span> genes in triplet reduces the <span class="trait">seed</span> size significantly indicating functional redundancy of these three <span class="gene">GmCYP78A</span> genes. In conclusion, we investigated the role of CYP78A in soybean <span class="trait">seed</span> regulation, and our strategy can be effectively used to engineer large <span class="trait">seed</span> traits in soybean varieties as well as other crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean (Glycine max) SWEET gene family: insights through comparative genomics, transcriptome profiling and whole genome re-sequence analysis.</h2>
			<h3>PMID:26162601  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Patil G(1), Valliyodan B(2), Deshmukh R(3), Prince S(4), Nicander B(5), Zhao M(6)(7), Sonah H(8), Song L(9), Lin L(10), Chaudhary J(11), Liu Y(12), Joshi T(13), Xu D(14), Nguyen HT(15).
				<br/> 
				年份：2015 Jul 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: SWEET (MtN3_saliva) domain proteins, a recently identified group of efflux transporters, play an indispensable role in sugar efflux, phloem loading, plant-pathogen interaction and reproductive tissue development. The SWEET gene family is predominantly studied in Arabidopsis and members of the family are being investigated in rice. To date, no transcriptome or genomics analysis of soybean SWEET genes has been reported. RESULTS: In the present investigation, we explored the evolutionary aspect of the SWEET gene family in diverse plant species including primitive single cell algae to angiosperms with a major emphasis on Glycine max. Evolutionary features showed expansion and duplication of the SWEET gene family in land plants. Homology searches with BLAST tools and Hidden Markov Model-directed sequence alignments identified 52 SWEET genes that were mapped to 15 chromosomes in the soybean genome as tandem duplication events. Soybean SWEET <span class="gene">(GmSWEET)</span> genes showed a wide range of expression profiles in different tissues and developmental stages. Analysis of public transcriptome data and expression profiling using quantitative real time PCR (qRT-PCR) showed that a majority of the <span class="gene">GmSWEET</span> genes were confined to reproductive tissue development. Several natural genetic variants (non-synonymous SNPs, premature stop codons and haplotype) were identified in the <span class="gene">GmSWEET</span> genes using whole genome re-sequencing data analysis of 106 soybean genotypes. A significant association was observed between SNP-haplogroup and <span class="trait">seed</span> sucrose content in three gene clusters on chromosome 6. CONCLUSION: Present investigation utilized comparative genomics, transcriptome profiling and whole genome re-sequencing approaches and provided a systematic description of soybean SWEET genes and identified putative candidates with probable roles in the reproductive tissue development. Gene expression profiling at different developmental stages and genomic variation data will aid as an important resource for the soybean research community and can be extremely valuable for understanding sink unloading and enhancing carbohydrate delivery to developing seeds for improving <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Constitutive overexpression of soybean plasma membrane intrinsic protein <span class="gene">GmPIP1;6</span> confers salt tolerance.</h2>
			<h3>PMID:24998596  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhou L, Wang C, Liu R, Han Q, Vandeleur RK, Du J, Tyerman S, Shou H(1).
				<br/> 
				年份：2014 Jul 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Under saline conditions, plant growth is depressed via osmotic stress and salt can accumulate in leaves leading to further depression of growth due to reduced photosynthesis and gas exchange. Aquaporins are proposed to have a major role in growth of plants via their impact on root water uptake and leaf gas exchange. In this study, soybean plasma membrane intrinsic protein 1;6 <span class="gene">(GmPIP1;6)</span> was constitutively overexpressed to evaluate the function of <span class="gene">GmPIP1;6</span> in growth regulation and salt tolerance in soybean. RESULTS: <span class="gene">GmPIP1;6</span> is highly expressed in roots as well as reproductive tissues and the protein targeted to the plasma membrane in onion epidermis. Treatment with 100 mM NaCl resulted in reduced expression initially, then after 3 days the expression was increased in root and leaves. The effects of constitutive overexpression of <span class="gene">GmPIP1;6</span> in soybean was examined under normal and salt stress conditions. Overexpression in 2 independent lines resulted in enhanced leaf gas exchange, but not growth under normal conditions compared to wild type (WT). With 100 mM NaCl, net assimilation was much higher in the <span class="gene">GmPIP1;6-Oe</span> and growth was enhanced relative to WT. <span class="gene">GmPIP1;6-Oe</span> plants did not have higher root hydraulic conductance (Lo) under normal conditions, but were able to maintain Lo under saline conditions compared to WT which decreased Lo. <span class="gene">GmPIP1;6-Oe</span> lines grown in the field had increased <span class="trait">yield</span> resulting mainly from increased <span class="trait">seed</span> size. CONCLUSIONS: The general impact of overexpression of <span class="gene">GmPIP1;6</span> suggests that it may be a multifunctional aquaporin involved in root water transport, photosynthesis and <span class="trait">seed</span> loading. <span class="gene">GmPIP1;6</span> is a valuable gene for genetic engineering to improve soybean <span class="trait">yield</span> and salt tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping the low palmitate fap1 mutation and validation of its effects in soybean oil and agronomic traits in three soybean populations.</h2>
			<h3>PMID:24132738  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cardinal AJ(1), Whetten R, Wang S, Auclair J, Hyten D, Cregan P, Bachlava E, Gillman J, Ramirez M, Dewey R, Upchurch G, Miranda L, Burton JW.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: fap 1 mutation is caused by a G174A change in <span class="gene">GmKASIIIA</span> that disrupts a donor splice site recognition and creates a GATCTG motif that enhanced its expression. Soybean oil with reduced palmitic acid content is desirable to reduce the health risks associated with consumption of this fatty acid. The objectives of this study were: to identify the genomic location of the reduced palmitate fap1 mutation, determine its molecular basis, estimate the amount of phenotypic variation in fatty acid composition explained by this locus, determine if there are epistatic interactions between the fap1 and fap nc loci and, determine if the fap1 mutation has pleiotropic effects on <span class="trait">seed</span> <span class="trait">yield,</span> oil and protein content in three soybean populations. This study detected two major QTL for 16:0 content located in chromosome 5 <span class="gene">(GmFATB1a,</span> fap nc) and chromosome 9 near BARCSOYSSR_09_1707 that explained, with their interaction, 66-94 % of the variation in 16:0 content in the three populations. Sequencing results of a putative candidate gene, <span class="gene">GmKASIIIA,</span> revealed a single unique polymorphism in the germplasm line C1726, which was predicted to disrupt the donor splice site recognition between exon one and intron one and produce a truncated KASIIIA protein. This G to A change also created the GATCTG motif that enhanced gene expression of the mutated <span class="gene">GmKASIIIA</span> gene. Lines homozygous for the <span class="gene">GmKASIIIA</span> mutation (fap1) had a significant reduction in 16:0, 18:0, and oil content; and an increase in unsaturated fatty acids content. There were significant epistatic interactions between <span class="gene">GmKASIIIA</span> (fap1) and fap nc for 16:0 and oil contents, and <span class="trait">seed</span> <span class="trait">yield</span> in two populations. In conclusion, the fap1 phenotype is caused by a single unique SNP in the <span class="gene">GmKASIIIA</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the Arabidopsis thaliana BBX32 gene in soybean increases grain <span class="trait">yield.</span></h2>
			<h3>PMID:22363475  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preuss SB(1), Meister R, Xu Q, Urwin CP, Tripodi FA, Screen SE, Anil VS, Zhu S, Morrell JA, Liu G, Ratcliffe OJ, Reuber TL, Khanna R, Goldman BS, Bell E, Ziegler TE, McClerren AL, Ruff TG, Petracek ME.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Crop <span class="trait">yield</span> is a highly complex quantitative trait. Historically, successful breeding for improved grain <span class="trait">yield</span> has led to crop plants with improved source capacity, altered plant architecture, and increased <span class="trait">resistance</span> to abiotic and biotic stresses. To date, transgenic approaches towards improving crop grain <span class="trait">yield</span> have primarily focused on protecting plants from herbicide, insects, or disease. In contrast, we have focused on identifying genes that, when expressed in soybean, improve the intrinsic ability of the plant to <span class="trait">yield</span> more. Through the large scale screening of candidate genes in transgenic soybean, we identified an Arabidopsis thaliana B-box domain gene (AtBBX32) that significantly increases soybean grain <span class="trait">yield</span> year after year in multiple transgenic events in multi-location field trials. In order to understand the underlying physiological changes that are associated with increased <span class="trait">yield</span> in transgenic soybean, we examined phenotypic differences in two AtBBX32-expressing lines and found increases in plant height and node, flower, pod, and <span class="trait">seed</span> number. We propose that these phenotypic changes are likely the result of changes in the timing of reproductive development in transgenic soybean that lead to the increased duration of the pod and <span class="trait">seed</span> development period. Consistent with the role of BBX32 in A. thaliana in regulating light signaling, we show that the constitutive expression of AtBBX32 in soybean alters the abundance of a subset of gene transcripts in the early morning hours. In particular, AtBBX32 alters transcript levels of the soybean clock genes <span class="gene">GmTOC1</span> and LHY-CCA1-like2 <span class="gene">(GmLCL2).</span> We propose that through the expression of AtBBX32 and modulation of the abundance of circadian clock genes during the transition from dark to light, the timing of critical phases of reproductive development are altered. These findings demonstrate a specific role for AtBBX32 in modulating soybean development, and demonstrate the validity of expressing single genes in crops to deliver increased agricultural productivity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression quantitative trait loci analysis of two genes encoding rubisco activase in soybean.</h2>
			<h3>PMID:20032079  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin Z(1), Meng F, Song H, Wang X, Xu X, Yu D.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rubisco activase (RCA) catalyzes the activation of Rubisco in vivo and plays a crucial role in photosynthesis. However, until now, little was known about the molecular genetics of RCA in soybean (Glycine max), one of the most important legume crops. Here, we cloned and characterized two genes encoding the longer alpha -isoform and the shorter beta -isoform of soybean RCA <span class="gene">(GmRCA</span> alpha and <span class="gene">GmRCA</span> beta, respectively). The two corresponding cDNAs are divergent in both the translated and 3 ' untranslated regions. Analysis of genomic DNA sequences suggested that the corresponding mRNAs are transcripts of two different genes and not the products of a single alternatively splicing pre-mRNA. Two additional possible alpha -form RCA-encoding genes, <span class="gene">GmRCA03</span> and <span class="gene">GmRCA14,</span> and one additional beta -form RCA-encoding gene, <span class="gene">GmRCA11,</span> were also isolated. To examine the function and modulation of RCA genes in soybean, we determined the expression levels of <span class="gene">GmRCA</span> alpha and <span class="gene">GmRCA</span> beta, Rubisco initial activity, photosynthetic rate, and <span class="trait">seed</span> <span class="trait">yield</span> in 184 soybean recombinant inbred lines. Correlation of gene expression levels with three other traits indicates that RCA genes could play an important role in regulating soybean photosynthetic capacity and <span class="trait">seed</span> <span class="trait">yield.</span> Expression quantitative trait loci mapping revealed four trans-expression quantitative trait loci for <span class="gene">GmRCA</span> alpha and <span class="gene">GmRCA</span> beta. These results could provide a new approach for the modulation of RCA genes to improve photosynthetic rate and plant growth in soybean and other plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel phytase with sequence similarity to purple acid phosphatases is expressed in cotyledons of germinating soybean seedlings.</h2>
			<h3>PMID:11500558  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hegeman CE(1), Grabau EA.
				<br/> 
				年份：2001 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (myo-inositol hexakisphosphate) is the major storage form of phosphorus in plant seeds. During germination, stored reserves are used as a source of nutrients by the plant seedling. Phytic acid is degraded by the activity of phytases to <span class="trait">yield</span> inositol and free phosphate. Due to the lack of phytases in the non-ruminant digestive tract, monogastric animals cannot utilize dietary phytic acid and it is excreted into manure. High phytic acid content in manure results in elevated phosphorus levels in soil and water and accompanying environmental concerns. The use of phytases to degrade <span class="trait">seed</span> phytic acid has potential for reducing the negative environmental impact of livestock production. A phytase was purified to electrophoretic homogeneity from cotyledons of germinated soybeans (Glycine max L. Merr.). Peptide sequence data generated from the purified enzyme facilitated the cloning of the phytase sequence <span class="gene">(GmPhy)</span> employing a polymerase chain reaction strategy. The introduction of <span class="gene">GmPhy</span> into soybean tissue culture resulted in increased phytase activity in transformed cells, which confirmed the identity of the phytase gene. It is surprising that the soybean phytase was unrelated to previously characterized microbial or maize (Zea mays) phytases, which were classified as histidine acid phosphatases. The soybean phytase sequence exhibited a high degree of similarity to purple acid phosphatases, a class of metallophosphoesterases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Intricate environment-modulated genetic networks control isoflavone accumulation in soybean seeds</h2>
			<h3>PMID:20540761</h3>
				
					<h3>PMCID:3224685</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Gutierrez-Gonzalez </surname>  <given-names> Juan J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> jjg3k9@mail.missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Wu </surname>  <given-names> Xiaolei </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> wuxia@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Gillman </surname>  <given-names> Jason D </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> Jason.Gillman@ARS.USDA.GOV </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Lee </surname>  <given-names> Jeong-Dong </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> jdlee@knu.ac.kr </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Zhong </surname>  <given-names> Rui </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> rzhong@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Yu </surname>  <given-names> Oliver </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> OYu@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Shannon </surname>  <given-names> Grover </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ShannonG@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Ellersieck </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> ellersieckm@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> nguyenhenry@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Sleper </surname>  <given-names> David A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sleperd@missouri.edu </email>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 6 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Developing RILs may be time-consuming but allows for testing the same genetic background under different conditions. <span class="trait">Seed</span> isoflavone content is highly affected by the environment (Table <xref ref-type="table" rid="T1"> 1 </xref> ). Hence, it is not surprising that very distinct sets of QTL data have been reported when the same mapping populations were independently analyzed under different conditions [ <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> ]. In this study, data from four different environments were used to perform an integrated analysis by a mixed linear model. A total of 2460 (205 RILs &#x000d7; 3 replications &#x000d7; 4 locations) individual observations were used to run the mapping software. Running data from different locations and individual replicates together under a unique model has been proven to be a powerful tool for discovering minor-effect QTL in complex traits because of the contribution to a reduced experimental error associated with increased observations [ <xref ref-type="bibr" rid="B14"> 14 </xref> , <xref ref-type="bibr" rid="B41"> 41 </xref> ], and might also allow the identification of the most stable QTL over the pooled locations. As a result, ten additive main genomic regions were found to influence genistein accumulation in seeds (Table <xref ref-type="table" rid="T2"> 2 </xref> and Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). A major QTL was found on chromosome 5 <span class="gene">(Gm05,</span> following the new nomenclature for soybean chromosomes) named <italic> qGEN5 </italic> , with an estimated heritability of the additive effect, h <sup> 2 </sup> (a), of 5.5%. Other identified important regions were in <span class="gene">Gm02,</span> <span class="gene">Gm15,</span> <span class="gene">Gm13,</span> <span class="gene">Gm19</span> and <span class="gene">Gm07,</span> which were named <italic> qGEN2 </italic> , <italic> qGEN15 </italic> , <italic> qGEN13 </italic> , <italic> qGEN19 </italic> , and <italic> qGEN7 </italic> , respectively. All were previously reported [ <xref ref-type="bibr" rid="B14"> 14 </xref> ]. Importantly, two newly identified loci were found in <span class="gene">Gm12</span> and <span class="gene">Gm20,</span> <italic> qGEN12 </italic> and <italic> qGEN20 </italic> . Among them, <italic> qGEN5 </italic> , <italic> qGEN2 </italic> , and <italic> qGEN19 </italic> , also displayed an additive by environment (A&#x000d7;E) interaction effect, reflecting differences in accumulation over locations (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ). We also used each environment alone as input data for the mapping software. As a result, two other regions were found to explain genistein accumulation: <italic> qGEN4 </italic> and <italic> qGEN2_2 </italic> (at BREC in 2007). For daidzein, nine different regions were exerting influence on its accumulation (Table <xref ref-type="table" rid="T2"> 2 </xref> and Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). Two key loci were found in <span class="gene">Gm05</span> and <span class="gene">Gm12,</span> <italic> qDAI5 </italic> and <italic> qDAI12 </italic> . Other influencing loci were mapped in <span class="gene">Gm08,</span> <span class="gene">Gm04,</span> <span class="gene">Gm01,</span> <span class="gene">Gm02,</span> and <span class="gene">Gm16:</span> <italic> qDAI8 </italic> , <italic> qDAI4 </italic> , <italic> qDAI1 </italic> , <italic> qDAI2 </italic> , <italic> qDAI16 </italic> , respectively, and two in <span class="gene">Gm17,</span> <italic> qDAI7 </italic> and <italic> qDAI7_2 </italic> . Apart from their additive effect, <italic> qDAI2 </italic> , <italic> qDAI12 </italic> , and <italic> qDAI17_2 </italic> also showed a significant A&#x000d7;E interaction effect. Glycitein accumulation was found to be influenced by a total of six different genomic regions (Table <xref ref-type="table" rid="T3"> 3 </xref> and Figure <xref ref-type="fig" rid="F3"> 3 </xref> ). Two of them, <italic> qGLY15 </italic> and <italic> qGLY6 </italic> , explained a greater percentage of the variation than any other QTL. Newly found glycitein loci were detected in <span class="gene">Gm05,</span> <span class="gene">Gm06,</span> <span class="gene">Gm02,</span> and <span class="gene">Gm09</span> ( <italic> qGLY5 </italic> , <italic> qGLY6 </italic> , <italic> qGLY2 </italic> , and <italic> qGLY9 </italic> ). Reflecting the variability of this particular isoflavone over environments, five out of six loci had an A&#x000d7;E interaction effect. In the latter instance, total isoflavone content was mapped as the sum of the previous three, resulting in the discovery of ten genomic regions that influenced its content in seeds. The most influencing locus was again placed in <span class="gene">Gm05</span> ( <italic> qTOT5 </italic> ). This was previously reported, as well as the other five ( <italic> qTOT2 </italic> , <italic> qTOT15 </italic> , <italic> qTOT12 </italic> , <italic> qTOT19 </italic> , and <italic> qTOT7 </italic> ). Newly discovered regions were found in <span class="gene">Gm04</span> ( <italic> qTOT4 </italic> ), <span class="gene">Gm02</span> ( <italic> qTOT2_2 </italic> ), and <span class="gene">Gm16</span> ( <italic> qTOT16 </italic> ). Another QTL ( <italic> qTOT8 </italic> ) was detected only at BREC in 2006. Five of these QTLs also bared a component of A&#x000d7;E interaction effect. The total percentage of phenotypic variance accounted for additive QTLs were 24.5, 24.3, 7.7, and 25.9% for genistein, daidzein, glycitein and total isoflavones, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Accordingly, several matches were found between QTL and biosynthetic genes. In <span class="gene">Gm01,</span> <italic> qDAI1 </italic> and another locus reported somewhere else for glycitein synthesis [ <xref ref-type="bibr" rid="B11"> 11 </xref> ] overlapped a copy of chalcone synthase ( <italic> <span class="gene">CHS6</span> </italic> ). Another region at one of the ends of the chromosome was previously reported to account for glycitein <span class="trait">seed</span> content [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B12"> 12 </xref> ] and embraced three phenylpropanoid genes ( <italic> <span class="gene">CHS7</span> </italic> , <italic> <span class="gene">4CL4</span> </italic> , and <italic> <span class="gene">HIDH</span> </italic> ). Nearby the end of <span class="gene">Gm02,</span> two <italic> <span class="gene">F3H</span> </italic> copies matched the position of four QTLs ( <italic> qGEN2 </italic> , <italic> qDAI2 </italic> , <italic> qGLY2 </italic> , and <italic> qTOT2 </italic> ). In the same chromosome, two other QTL were found to explain genistein and total isoflavones ( <italic> qTOT_2 </italic> and <italic> qGEN2 </italic> ). A <italic> <span class="gene">C4</span> H </italic> gene is centered in their IC. Another <italic> <span class="gene">C4</span> H </italic> copy on <span class="gene">Gm14</span> is found within the interval of the QTL <italic> gen-B2 </italic> , and also of two other QTL for genistein and daidzein previously reported [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. In <span class="gene">Gm03</span> a glycitein QTL found by the same group [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ] coincides with a copy of another important gene, <italic> <span class="gene">PAL1</span> </italic> . At the beginning of <span class="gene">Gm04,</span> three QTL ( <italic> qGEN4 </italic> , <italic> qDAI4 </italic> , and <italic> qTOT4 </italic> ) were in the same region in which an isoflavone reductase copy ( <italic> <span class="gene">IFR2</span> </italic> ) was mapped. Several genomic regions in <span class="gene">Gm07</span> account for the synthesis of distinct isoflavones ( <italic> qGEN7 </italic> , <italic> qDAI7 </italic> , <italic> qTOT7 </italic> , and <italic> qDAI7_2 </italic> ). However, despite the observation that critical isoflavone genes ( <italic> <span class="gene">IFS1</span> </italic> and <italic> <span class="gene">4CL4</span> </italic> ) are present in this chromosome, those QTLs remain without a clear candidate. An extensive area of approximately 50 cM was shown to influence genistein, daidzein, glycitein and total isoflavone accumulation [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. This region was apparently reduced to ~15 cM by another group [ <xref ref-type="bibr" rid="B11"> 11 </xref> ], because the latter is included in the former and both accounted for the same isoflavone compounds. The <italic> <span class="gene">4CL4</span> </italic> gene is less than 5 cM apart from this region and about 10 cM from <italic> qGEN7 </italic> , <italic> qDAI7 </italic> , and <italic> qTOT7 </italic> . Numerous QTL reported for this chromosome suggests the presence of a large number of polymorphisms associated with isoflavone synthesis. In <span class="gene">Gm11,</span> a region identified to affect glycitein content [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ] was found to overlap a <italic> <span class="gene">4CL</span> </italic> homolog. A chalcone isomerase ( <italic> <span class="gene">CHI3</span> </italic> ) and isoflavone synthase ( <italic> <span class="gene">IFS2</span> </italic> ) genes were located in the region identified by <italic> qGEN13 </italic> , in <span class="gene">Gm13.</span> In this chromosome, another QTL was reported for glycitein [ <xref ref-type="bibr" rid="B12"> 12 </xref> ], with a <italic> <span class="gene">PAL</span> </italic> and <italic> <span class="gene">4CL</span> </italic> copies lying nearby the QTL IC. In <span class="gene">Gm14,</span> one of the two <italic> <span class="gene">C4</span> H </italic> homologous copies is within the genomic area delimited by <italic> gen-B2 </italic> [ <xref ref-type="bibr" rid="B14"> 14 </xref> ] and by two other QTL for genistein and daidzein [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. The only known copy of the isoflavone hydroxylase ( <italic> I2'H </italic> ) gene is in the region delimited by the IC of <italic> qGEN15 </italic> , <italic> qGLY15 </italic> , and <italic> qTOT15 </italic> , in <span class="gene">Gm15.</span> Two glycitein loci were previously reported by our group in <span class="gene">Gm17</span> ( <italic> gly-D2_1 </italic> and <italic> gly-D2_2 </italic> ), had a copy of <italic> <span class="gene">4CL1</span> </italic> and <italic> <span class="gene">DFR2</span> </italic> as candidate genes, respectively. Another QTL for glycitein in <span class="gene">Gm18</span> ( <italic> gly-G </italic> ) is nearby a <italic> IOMT </italic> and <italic> CHR </italic> copies, although not overlapping. A phenylalanine ammonia-lyase copy ( <italic> <span class="gene">PAL</span> </italic> ) located on <span class="gene">Gm19</span> is clearly a good candidate for <italic> qGEN19 </italic> , <italic> qTOT19 </italic> , and <italic> dai-L </italic> . Finally, in <span class="gene">Gm20,</span> a copy of the <italic> IOMT </italic> gene is within the region demarcated by <italic> qGEN20 </italic> . In addition, copies of <italic> CHI-1A </italic> , <italic> CHI-1B1 </italic> , and <italic> CHI2 </italic> are present in close proximity (&#x0003c;10 cM).
					</p>
				
					<p>&nbsp;&nbsp;
						A separate discussion of the <italic> <span class="gene">CHS</span> </italic> genes present on <span class="gene">Gm08</span> is merited. According to BAC sequencing, as many as 12 <italic> <span class="gene">CHS</span> </italic> -related genes are thought to be present on <span class="gene">Gm08,</span> organized in two clustered regions [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. One such region is composed of two perfect inverted repeated regions each containing <italic> <span class="gene">CHS1</span> </italic> , <italic> <span class="gene">CHS3</span> </italic> , <italic> CHS4 </italic> genes. This region is commonly referred to as the <italic> I </italic> locus, and although the specific silencing RNA is unclear, appears to act through siRNA-mediated gene silencing to inhibit <span class="trait">seed</span> coat pigmentation by specific degradation of <span class="trait">seed</span> coat specific <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> transcripts [ <xref ref-type="bibr" rid="B58"> 58 </xref> - <xref ref-type="bibr" rid="B60"> 60 </xref> ]. The <italic> <span class="gene">CHS6</span> </italic> , <italic> <span class="gene">CHS7</span> </italic> and <italic> CHS8 </italic> genes are not apparently present in clusters on <span class="gene">Gm08</span> but rather in single copy regions on other chromosomes [ <xref ref-type="bibr" rid="B58"> 58 </xref> ]. In addition, at least four additional <italic> <span class="gene">CHS</span> </italic> genes ( <italic> CHS5, CHS3, <span class="gene">CHS1</span> </italic> , and <italic> <span class="gene">CHS9</span> </italic> ) are thought to be present in another <italic> <span class="gene">CHS</span> </italic> cluster on <span class="gene">Gm08,</span> for a total of 12 <italic> <span class="gene">CHS</span> </italic> genes on <span class="gene">Gm08.</span> The current Glyma 1.01 assembly of <span class="gene">Gm08</span> contains only 7 <italic> <span class="gene">CHS</span> </italic> -corresponding regions (6 <italic> <span class="gene">CHS</span> </italic> gene models, and an additional matching region which does not have an annotated gene model, Additional file <xref ref-type="supplementary-material" rid="S5"> 5 </xref> ) as determined by BLAST searches using <italic> <span class="gene">CHS</span> </italic> sequence as queries <ext-link ext-link-type="uri" xlink:href="http://www.phytozome.net/soybean"> http://www.phytozome.net/soybean </ext-link> . It seems likely that a miss-assembly has occurred in this region and the number of <italic> <span class="gene">CHS</span> </italic> genes present on <span class="gene">Gm08</span> remains to be resolved. It is not currently clear to what extent other similar regions with gene repeats may have impacted on the assembly of the whole genome shotgun sequence. Furthermore, as many as ten regions were found sharing homology to the <italic> IFR </italic> gene. Although several of them may be pseudogenes, genes inactivated by the accumulation of deleterious mutations, it may be quite difficult to ascertain in which tissues, if any, these genes are expressed if they share a high percentage of homology. In addition, we currently have only a single sequenced genome (Williams 82) for reference. It is highly likely that other cultivars will have differing numbers and dispositions of phenylpropanoid genes. In particular, large scale rearrangements and deletions have been noted for the <italic> <span class="gene">CHS</span> </italic> gene clusters present on <span class="gene">Gm08</span> [ <xref ref-type="bibr" rid="B61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The network of genes and interactions appears to have several interconnected neuralgic centers. The <italic> <span class="gene">CHS</span> </italic> gene cluster in <span class="gene">Gm08</span> emerges as key node controlling the synthesis and accumulation of all individual isoflavones. The cluster rests within the IC of two important QTL for daidzein and total isoflavone content. In addition, it has connections with two other influential QTL and candidate genes: in <span class="gene">Gm05,</span> and <span class="gene">Gm19(</span> <italic> <span class="gene">PAL</span> </italic> ). Importantly, all three isoflavones have at least one epistatic line converging in the <italic> <span class="gene">CHS</span> </italic> cluster. One of those pair wise interactions connected the cluster with the focal locus at the end of <span class="gene">Gm05,</span> and explained both genistein and daidzein, which is a rare phenomenon and further validated the interaction. This locus itself accounted for genistein, daidzein, glycitein, and total isoflavones ( <italic> qGEN5 </italic> , <italic> qDAI5 </italic> , <italic> qGLY5 </italic> and <italic> qTOT5 </italic> ), and it is by far the most principal of all QTL reported herein in terms of explained percentage of variance and additive value. However, for this QTL of largest effect, no known biosynthetic gene was found to be located within this region. Perhaps a heretofore unknown isoflavone biosynthetic gene is within this region. Alternately, a <italic> trans </italic> -acting factor affecting the expression or activity of isoflavone biosynthetic genes may be present (transcription factor, ubiquitin-related protein, etc). Consistent with this hypothesis, the epistatic interactions that this locus has with another central locus <span class="gene">(Gm08/</span> <italic> <span class="gene">CHS</span> </italic> cluster) reinforces the hypothesis of an action through <italic> trans </italic> -regulatory control. Whether this region contains such a factor remains for future fine-mapping work. Strengthening its central role, the <italic> <span class="gene">CHS</span> </italic> cluster is also epistaticaly connected with two other regions: a genistein interaction with the locus <span class="gene">Gm19/(</span> <italic> <span class="gene">PAL</span> </italic> ), and a glycitein interaction <span class="gene">(Gm11)</span> with a locus in the proximity of a QLT implicated in glycitein production [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ]. A third strategic node in <span class="gene">Gm07</span> accounts for genistein, daidzein and the sum of all isoflavones, and it is also connected with two other important areas in the genome. First, a daidzein epistatic interaction with the region in <span class="gene">Gm01/(</span> <italic> <span class="gene">CHS6</span> </italic> ) of <italic> qDAI1 </italic> and a QTL reported for glycitein [ <xref ref-type="bibr" rid="B11"> 11 </xref> ]. The second is an interaction for genistein with <span class="gene">Gm13/(</span> <italic> <span class="gene">IFS2</span> </italic> , <italic> <span class="gene">CHI3</span> </italic> ), which itself also explains genistein <span class="trait">seed</span> concentrations in an additive manner. Three loci located in <span class="gene">Gm02/(</span> <italic> <span class="gene">F3H1</span> </italic> , <italic> <span class="gene">F3H2</span> </italic> <span class="gene">)-Gm05-Gm15/(</span> <italic> I2'H </italic> ) formed another key three-node subnet for isoflavone synthesis. If a single genomic region harbors QTL for genistein, daidzein, glycitein and total isoflavones, it is certainly a good candidate to be considered for marker assisted selection. The underlying gene could be an enzyme acting early in the pathway or a <italic> trans </italic> -acting factor, which impacts expression or activity of one or more of these genes. One such region is found at the beginning of <span class="gene">Gm02</span> corresponding to two <italic> F3 H </italic> genes ( <italic> <span class="gene">F3H1</span> </italic> and <italic> <span class="gene">F3H2</span> </italic> ). This focal locus also featured two epistatic interactions accounting for daidzein and total isoflavones, respectively, with another locus in <span class="gene">Gm05</span> for daidzein accumulation [ <xref ref-type="bibr" rid="B10"> 10 </xref> ]. In addition, this <span class="gene">Gm02</span> locus was also linked by means of an epistatic interaction for total isoflavones to a QTL in <span class="gene">Gm15</span> for genistein, glycitein, and total <span class="trait">seed</span> isoflavone content, which is also connected with <span class="gene">Gm10(</span> <italic> <span class="gene">CHI</span> </italic> - <italic> 1B </italic> ). Finally, two other important additive nodes, in <span class="gene">Gm04</span> ( <italic> qGEN4 </italic> , <italic> qDAI4 </italic> , <italic> qTOT4 </italic> ) and <span class="gene">Gm16</span> ( <italic> qDAI16 </italic> , <italic> qTOT16 </italic> ), were connected by a daidzein epistatic union.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T set was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less pod wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant <span class="trait">seed</span> weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher <span class="trait">seed</span> weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and functional characterization of the sulfate transporter gene <span class="gene">GmSULTR1;2b</span> in soybean.</h2>
			<h3>PMID:27206527  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ding Y(1), Zhou X(1), Zuo L(1), Wang H(2), Yu D(3).
				<br/> 
				年份：2016 May 20
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is a major source of oil and protein in the human diet and in animal feed. However, as soybean is deficient in sulfur-containing amino acids, its nutritional value is limited. Increasing sulfate assimilation and utilization efficiency is a valuable approach to augment the concentration of sulfur-containing amino acids in soybean seeds, and sulfate transporters play important roles in both sulfate uptake and translocation within plants. RESULTS: In this study, we isolated and characterized a soybean sulfate transporter gene: <span class="gene">GmSULTR1;2b.</span> The gene was found to be specifically expressed in root tissues and induced by low-sulfur stress. In addition, <span class="gene">GmSULTR1;2b</span> expression in yeast could complement deficiency in the sulfate transporter genes SUL1 and SUL2. Under +S conditions, <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco plants accumulated higher levels of organic matter and exhibited enhanced biomass and <span class="trait">seed</span> weight compared to control plants. Under -S conditions, acclimation of <span class="gene">GmSULTR1;2b-overexpressing</span> plants was much better than control plants. <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, but no significant difference was observed under salt stress. Transcriptome analysis revealed 515 genes with at least a 2-fold change in expression in the leaves of tobacco plants overexpressing <span class="gene">GmSULTR1;2b.</span> Of these, 227 gene annotations were classified into 12 functional categories associated with 123 relevant pathways, including biosynthesis and metabolism-related proteins, stress-related proteins, and transporters. CONCLUSIONS: The findings reported here indicate that the increased biomass and <span class="trait">seed</span> <span class="trait">yield</span> observed in transgenic tobacco plants could have resulted from greater nutrient uptake and transport capability as well as enhanced development and accumulation of organic matter. Taken together, our results indicate that <span class="gene">GmSULTR1;2b</span> plays an important role in sulfur uptake and could alter the sulfur status of plants. Our study suggests that overexpressing <span class="gene">GmSULTR1;2b</span> may enhance plant <span class="trait">yield</span> under +S conditions, reduce plant production loss under -S conditions, and improve plant tolerance to sulfur deficiency stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Dynamics of Soybean Leaf and Shoot Apical Meristem Transcriptome Undergoing Floral Initiation Process</h2>
			<h3>PMID:23762343</h3>
				
					<h3>PMCID:3675103</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wong </surname>  <given-names> Chui E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Mohan B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhalla </surname>  <given-names> Prem L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Putative <italic> TPS </italic> and <italic> TPP </italic> transcripts were represented in both leaf and SAM dataset. For the leaf dataset, there were seven putative <italic> TPS </italic> transcripts that were up-regulated after 1 short-day (fold change ranging from 2 to 12) and only two putative <italic> TPPs </italic> with significant expression changes but down-regulated on 1 short-day, in contrast to <italic> TPS </italic> s ( <xref ref-type="fig" rid="pone-0065319-g005"> Figure 5 </xref> ). The high expression of <italic> TPS </italic> after 1short-day was maintained throughout the time course investigated. There is thus a possible sustained increase of T6P level in the leaf under short-day, which could interfere with carbon allocation and growth in a mechanism involving ABA metabolism. While T6P may serve the primary role of coordinating the metabolic shift in the leaf under short-day photoperiod, it could also play an important role in initiating flowering process as <italic> tps1 </italic> mutant are unable to flower <xref ref-type="bibr" rid="pone.0065319-vanDijken1"> [43] </xref> . The temporal profile of <italic> TPS </italic> and <italic> TPP </italic> transcripts in the SAM markedly differs from that of the leaf with increased expression detected only on 4short-day coinciding with the induction of floral meristem identity genes such as <italic> <span class="gene">GmAP1</span> </italic> . <italic> TPP </italic> gene is linked to inflorescent <span class="trait">branching</span> in maize presumably through the alteration of trehalose level <xref ref-type="bibr" rid="pone.0065319-SatohNagasawa1"> [45] </xref> . The increased expression of trehalose-related sequences on 4short-day in the SAM is consistent with this function.
					</p>
				
					<p>&nbsp;&nbsp;
						Hormones may play predominant roles in the short-day-induced floral initiation process ( <xref ref-type="fig" rid="pone-0065319-g008"> Figure 8B </xref> ). Both GA and cytokinin have been shown to induce flowering via FT or its paralogue, TSF <xref ref-type="bibr" rid="pone.0065319-Wilson1"> [65] </xref> , <xref ref-type="bibr" rid="pone.0065319-Hisamatsu2"> [66] </xref> , <xref ref-type="bibr" rid="pone.0065319-DAloia1"> [67] </xref> . Our hypothesis that they could be additional factors as florigenic signals is consistent with the fact that <italic> ft </italic> mutants are only late-flowering <xref ref-type="bibr" rid="pone.0065319-Corbesier1"> [4] </xref> . Once in the SAM, GA or cytokinin may exert their floral promoting roles by directly activating <italic> <span class="gene">GmSOC1</span> </italic> as well as contributing to a low cytokinin and high GA environment. The balance of cytokinin and GA (high cytokinin and low GA) is known to be critical in the maintenance of vegetative meristem homeostasis <xref ref-type="bibr" rid="pone.0065319-Perilli1"> [60] </xref> . Our transcriptome analysis however alludes to low cytokinin and high GA level being critical for the conversion of vegetative meristem to IM. Multiple mechanisms are potentially in place to result in such hormonal regime as there is likely an influx of GA from the leaf as well as increased local production in the SAM that give rise to increased GA level. The induction of CKX possibly via auxin to breakdown cytokinin is to reinforce the low cytokinin to GA ratio especially since there is also a strong likelihood of cytokinin import to the SAM from the leaf under short-days. Interactions among these hormones form the basis in generating a specific signal to the SAM to eventuate in flowering. Meanwhile, germination is another developmental transition that responds to similar environmental cues as flowering and a recent study has unraveled common elements shared between genetic pathways regulating these two transitions <xref ref-type="bibr" rid="pone.0065319-Chiang1"> [68] </xref> . As both ABA and GA are known to play roles in regulating <span class="trait">seed</span> germination and GA is known to regulate flowering, it is not unprecedented that our study has consistently implicated ABA roles in the floral initiation process.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In silico identification and comparative genomics of candidate genes involved in biosynthesis and accumulation of <span class="trait">seed</span> oil in plants.</h2>
			<h3>PMID:22312320</h3>
				

				
			<p>
				作者：Sharma A(1), Chauhan RS.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genes involved in fatty acids biosynthesis, modification and oil body formation are expected to be conserved in structure and function in different plant species. However, significant differences in the composition of fatty acids and total oil contents in seeds have been observed in different plant species. Comparative genomics was performed on 261 genes involved in fatty acids biosynthesis, TAG synthesis, and oil bodies formation in Arabidopsis, Brassica rapa, castor bean and soybean. In silico expression analysis revealed that stearoyl desaturase, FatB, FAD2, oleosin and <span class="gene">DGAT</span> are highly abundant in seeds, thereby considered as ideal candidates for mining of favorable alleles in natural population. Gene structure analysis for major genes, ACCase, FatA, FatB, FAD2, <span class="gene">FAD3</span> and DGAT, which are known to play crucial role in oil synthesis revealed that there are uncommon variations (SNPs and INDELs) which lead to varying content and composition of fatty acids in <span class="trait">seed</span> oil. The predicted variations can provide good targets for <span class="trait">seed</span> oil QTL identification, understanding the molecular mechanism of <span class="trait">seed</span> oil accumulation, and genetic modification to enhance <span class="trait">seed</span> oil <span class="trait">yield</span> in plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of broiler performance when fed diets containing event DP-3O5423-1, nontransgenic near-isoline control, or commercial reference soybean meal, hulls, and oil.</h2>
			<h3>PMID:19038811  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：McNaughton J(1), Roberts M, Smith B, Rice D, Hinds M, Sanders C, Layton R, Lamb I, Delaney B.
				<br/> 
				年份：2008 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DP-3Ø5423-1 (305423) is a genetically modified soybean that was produced by biolistic insertion of the gm-fad2-1 gene fragment and gm-hra genes into the germline of soybean seeds. Expression of gm-fad2-1 results in greater concentrations of oleic acid (18:1) by suppressing expression of the endogenous <span class="gene">FAD2-1</span> gene, which encodes an n-6 fatty acid desaturase enzyme that catalyzes desaturation of 18:1 to linoleic acid (18:2). The <span class="gene">GM-HRA</span> protein expressed by the gm-hra gene is a modified version of the soybean acetolactate synthase enzyme that is used as a selectable marker during transformation. A 42-d feeding trial was conducted with broiler chickens to compare the nutritional performance of 305423 soybeans with nontransgenic soybeans. Diets were prepared using processed fractions (meal, hulls, and oil) from 305423 soybean plants. For comparison, additional diets were produced with soybean fractions obtained from a nontransgenic near-isoline (control) and nontransgenic commercial Pioneer brand varieties (93B86, 93B15, and 93M40). Diets were fed to Ross x Cobb broilers (n = 120/group, 50% male and 50% female) in 3 phases. Starter, grower, and finisher diets contained 26.5, 23, and 21.5% soybean meal, respectively. Soybean hulls and oil were added at 1.0 and 0.5%, respectively, across all diets in each phase. No statistically significant differences were observed in growth performance (BW, mortality, feed efficiency), organ <span class="trait">yield</span> (liver and kidney), or carcass <span class="trait">yield</span> (breast, thigh, leg, wing, and abdominal fat) variables between broilers consuming diets prepared with isolated fractions from 305423 or near-isoline control soybean. Additionally, all performance and carcass variables from control and 305423 soybean treatment groups fell within tolerance intervals constructed for each response variable using data from broilers fed diets prepared with reference soybean fractions. Based on the results from this study, it was concluded that 305423 soybeans were nutritionally equivalent to non-transgenic control soybeans with a comparable genetic background.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A genome-wide association study of <span class="trait">seed</span> composition traits in wild soybean ( <italic> Glycine soja </italic> )</h2>
			<h3>PMID:28056769</h3>
				
					<h3>PMCID:5217241</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Leamy </surname>  <given-names> Larry J. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hengyou </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Changbao </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Chen </surname>  <given-names> Charles Y. </given-names>  </name>  <address>  <email> cyc0002@auburn.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-3537-7783 </contrib-id>  <name>  <surname> Song </surname>  <given-names> Bao-Hua </given-names>  </name>  <address>  <email> bsong5@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Department of Biological Sciences, the University of North Carolina at Charlotte, Charlotte, NC 28223 USA  </aff>  <aff id="Aff2">  <label> 2 </label> Double Haploid Optimization Group, Monsanto Company, Chesterfield, MO 63017 USA  </aff>  <aff id="Aff3">  <label> 3 </label> Department of Crop, Soil and Environmental Sciences, Auburn University, Auburn, AL 36849 USA  </aff>
				<br/> 
				年份：<day> 7 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Oleic acid is a monounsaturated fatty acid that at high levels is associated with increased health benefits as well as oxidative stability [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. In soybean, this fatty acid typically averages 20&#x02013;25% [ <xref ref-type="bibr" rid="CR46"> 46 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> ], although its level in our wild soybean sample was considerably lower than this (overall average&#x02009;=&#x02009;15%), especially in accessions from Japan. This difference might partially be explained by selection for overall oil content in domesticated soybeans that has increased oleic acid levels as well. Consistent with this, oleic acid in our wild soybean sample was the only one of the five fatty acids to exhibit a significant, positive (although low) correlation with total oil content (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). Direct selection for increased oleic acid itself also has been practiced in domesticated soybeans, especially with the discovery of mutant <italic> <span class="gene">FAD2</span> </italic> genes that can increase its level to as much as 80% or more [ <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. This approach appears very promising, although in some populations these mutant alleles may negatively impact <span class="trait">yield</span> [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Vernonia Diacylglycerol Acyltransferase Can Increase Renewable Oil Production.</h2>
			<h3>PMID:27578203  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hatanaka T(1), Serson W(2), Li R(3), Armstrong P(4), Yu K(2), Pfeiffer T(2), Li XL(2), Hildebrand D(2).
				<br/> 
				年份：2016 Sep 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the production of plant oils such as soybean oil as a renewable resource for food and fuel is valuable. Successful breeding for higher oil levels in soybean, however, usually results in reduced protein, a second valuable <span class="trait">seed</span> component. This study shows that by manipulating a highly active acyl-CoA:diacylglycerol acyltransferase (DGAT) the hydrocarbon flux to oil in oilseeds can be increased without reducing the protein component. Compared to other plant DGATs, a <span class="gene">DGAT</span> from Vernonia galamensis (VgDGAT1A) produces much higher oil synthesis and accumulation activity in yeast, insect cells, and soybean. Soybean lines expressing VgDGAT1A show a 4% increase in oil content without reductions in <span class="trait">seed</span> protein contents or <span class="trait">yield</span> per unit land area. Incorporation of this trait into 50% of soybeans worldwide could result in an increase of 850 million kg oil/year without new land use or inputs and be worth ∼U.S.$1 billion/year at 2012 production and market prices.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Using the candidate gene approach for detecting genes underlying <span class="trait">seed</span> oil concentration and <span class="trait">yield</span> in soybean.</h2>
			<h3>PMID:23568222  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the oil concentration in soybean seeds has been given more attention in recent years because of demand for both edible oil and biodiesel production. Oil concentration in soybean is a complex quantitative trait regulated by many genes as well as environmental conditions. To identify genes governing <span class="trait">seed</span> oil concentration in soybean, 16 putative candidate genes of three important gene families (GPAT: acyl-CoA:sn-glycerol-3-phosphate acyltransferase, DGAT: acyl-CoA:diacylglycerol acyltransferase, and PDAT: phospholipid:diacylglycerol acyltransferase) involved in triacylglycerol (TAG) biosynthesis pathways were selected and their sequences retrieved from the soybean database ( http://www.phytozome.net/soybean ). Three sequence mutations were discovered in either coding or noncoding regions of three <span class="gene">DGAT</span> soybean isoforms when comparing the parents of a 203 recombinant inbreed line (RIL) population; OAC Wallace and OAC Glencoe. The RIL population was used to study the effects of these mutations on <span class="trait">seed</span> oil concentration and other important agronomic and <span class="trait">seed</span> composition traits, including <span class="trait">seed</span> <span class="trait">yield</span> and protein concentration across three field locations in Ontario, Canada, in 2009 and 2010. An insertion/deletion (indel) mutation in the <span class="gene">GmDGAT2B</span> gene in OAC Wallace was significantly associated with reduced <span class="trait">seed</span> oil concentration across three environments and reduced <span class="trait">seed</span> <span class="trait">yield</span> at Woodstock in 2010. A mutation in the 3' untranslated (3'UTR) region of <span class="gene">GmDGAT2C</span> was associated with <span class="trait">seed</span> <span class="trait">yield</span> at Woodstock in 2009. A mutation in the intronic region of <span class="gene">GmDGAR1B</span> was associated with <span class="trait">seed</span> <span class="trait">yield</span> and protein concentration at Ottawa in 2010. The genes identified in this study had minor effects on either <span class="trait">seed</span> <span class="trait">yield</span> or oil concentration, which was in agreement with the quantitative nature of the traits. However, the novel gene-specific markers designed in the present study can be used in soybean breeding for marker-assisted selection aimed at increasing <span class="trait">seed</span> <span class="trait">yield</span> and oil concentration with no significant impact on <span class="trait">seed</span> protein concentration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean <span class="trait">seed</span> oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased <span class="trait">seed</span> <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] <span class="trait">seed</span> oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as <span class="trait">seed</span> <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between <span class="trait">seed</span> oil concentration and important agronomic and <span class="trait">seed</span> quality traits, including <span class="trait">seed</span> <span class="trait">yield,</span> 100-seed weight, protein concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with <span class="trait">seed</span> oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for protein concentration, four for <span class="trait">seed</span> <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with <span class="trait">seed</span> protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with <span class="trait">seed</span> <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with <span class="trait">seed</span> oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for protein concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as <span class="trait">seed</span> <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher <span class="trait">seed</span> <span class="trait">yield</span> and oil concentration or higher <span class="trait">seed</span> protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome Analysis of Yellow Horn ( <italic> Xanthoceras sorbifolia </italic> Bunge): A Potential Oil-Rich <span class="trait">Seed</span> Tree for Biodiesel in China</h2>
			<h3>PMID:24040247</h3>
				
					<h3>PMCID:3770547</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yulin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Zhedong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ao </surname>  <given-names> Yan </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhixiang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 2 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						As has been reported previously, overexpression of ACCase, a crucial enzyme in fatty acid synthesis, can alter the fatty acid composition of seeds and increase the fatty acid content, which would lead to an increased oleic acid content and <span class="trait">seed</span> <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0074441-Roesler1"> [41] </xref> , <xref ref-type="bibr" rid="pone.0074441-Madoka1"> [42] </xref> . Most research on FAS has concentrated on ACP and KAS. Functional expression of an ACP from <italic> Azospirillum brasilense </italic> in <italic> Brassica juncea </italic> can improve the content of 18&#x02236;1 and 18&#x02236;2 in seeds, enhance the ratio of monounsaturated (18&#x02236;1) to saturated fatty acids, increase the ratio of 18&#x02236;2 to 18&#x02236;3 and reduce the erucic acid content (22&#x02236;1) <xref ref-type="bibr" rid="pone.0074441-Jha1"> [43] </xref> . In plastids, three types of KAS are found: KAS I, KAS II and KAS III. KAS II catalyses 16&#x02236;0-ACP to elongate to 18&#x02236;0-ACP and KAS III condenses acetyl-CoA with malonyl-ACP to form 4&#x02236;0-ACP. Using hairpin RNAi to reduce the activity of KAS II can lead to an increase in 16&#x02236;0 accumulation, up to about 53% of the total, but some transgenic offspring are deformed during early embryonic development <xref ref-type="bibr" rid="pone.0074441-Pidkowich1"> [44] </xref> . Overexpression of KAS III can also improve 16&#x02236;0 accumulation, but the rate of lipid synthesis is reduced <xref ref-type="bibr" rid="pone.0074441-Dehesh1"> [45] </xref> . In our study, we did not find any unigenes that encoded KAS I, which is highly active with acyl-ACP with chain lengths from <span class="gene">C2</span> to C14, is far less effective for 16&#x02236;0-ACP and almost inactive for 18&#x02236;0-ACP <xref ref-type="bibr" rid="pone.0074441-Shimakata1"> [46] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of a plastidial safflower GPAT and an <italic> Escherichia coli </italic> GPAT in <italic> Arabidopsis </italic> can improve the <span class="trait">seed</span> oil content, with average increases of 22 and 15%, respectively <xref ref-type="bibr" rid="pone.0074441-Jain1"> [49] </xref> . Similarly, substantial increases of 8 to 48% in <span class="trait">seed</span> oil content and increases in both overall proportions and amounts of very-long-chain fatty acids in <span class="trait">seed</span> TAGs were obtained by overexpression of a mutant form of yeast LPAT in <italic> Arabidopsis </italic> and <italic> Brassica napus </italic> <xref ref-type="bibr" rid="pone.0074441-Zou1"> [50] </xref> . <span class="gene">DGAT</span> is a key enzyme regulating the rate of the Kennedy pathway. It has four types, and we detected <span class="gene">DGAT1</span> in this study. Ectopic expression of <span class="gene">DGAT</span> will improve the oil content in seeds, which has been confirmed in <italic> Arabidopsis </italic> , soybean and maize <xref ref-type="bibr" rid="pone.0074441-Lardizabal1"> [51] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0074441-Zheng1"> [53] </xref> . In addition, compared to the single unigene that encoded GPAT1, the finding of four unigenes that encoded PDAT1 in our study provides further evidence that yellow horn has the potential to channel fatty acids that are incorporated in membrane lipids, such as PC, into TAG biosynthesis.
					</p>
				
					<p>&nbsp;&nbsp;
						In previous studies, a set of TFs, including LEC1, LEC2, ABI3, FUS3 and <span class="gene">WRI1</span> which play key roles in <span class="trait">seed</span> oil synthesis and deposition, were identified <xref ref-type="bibr" rid="pone.0074441-Wang2"> [63] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0074441-Tan1"> [65] </xref> . These TFs were reviewed by Fobert and made available online ( <ext-link ext-link-type="uri" xlink:href="http://lipidlibrary.aocs.org/plantbio/transfactors/index.htm"> http://lipidlibrary.aocs.org/plantbio/transfactors/index.htm </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We summed the SSRs, SNPs and InDels that were located in the 281 unigenes involved in the FA and TAG biosynthesis and metabolism pathways, which encoded oleosin, caleosin, steroleosin and TFs that regulated <span class="trait">seed</span> oil deposition. As a result, 26 SSRs, 194 SNPs and 60 InDels were found in 74 unigenes, which covered 31 enzymes or proteins and two TFs (ABI3 and HSI2-L1) ( <xref ref-type="supplementary-material" rid="pone.0074441.s004"> Table S4 </xref> , <xref ref-type="supplementary-material" rid="pone.0074441.s005"> Table S5 </xref> and <xref ref-type="supplementary-material" rid="pone.0074441.s006"> Table S6 </xref> ). The remaining enzymes, such as MAT, EAR, FATA, ADH, ALDH, PCH, FAD6, PP and TGL4, and TFs, such as L1L, ADOF1, EMF2, HSI2, <span class="gene">AP2</span> and GL2, contained no SSRs, SNPs or InDels.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Genome-Wide Association Study for Agronomic Traits in Soybean Using SNP Markers and SNP-Based Haplotype Analysis.</h2>
			<h3>PMID:28152092  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contreras-Soto RI(1), Mora F(2), de Oliveira MA(3), Higashi W(3), Scapim CA(1), Schuster I(4).
				<br/> 
				年份：2017 Feb 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mapping quantitative trait loci through the use of linkage disequilibrium (LD) in populations of unrelated individuals provides a valuable approach for dissecting the genetic basis of complex traits in soybean (Glycine max). The haplotype-based genome-wide association study (GWAS) has now been proposed as a complementary approach to intensify benefits from LD, which enable to assess the genetic determinants of agronomic traits. In this study a GWAS was undertaken to identify genomic regions that control 100-seed weight (SW), plant height (PH) and <span class="trait">seed</span> <span class="trait">yield</span> (SY) in a soybean association mapping panel using single nucleotide polymorphism (SNP) markers and haplotype information. The soybean cultivars (N = 169) were field-evaluated across four locations of southern Brazil. The genome-wide haplotype association analysis (941 haplotypes) identified eleven, seventeen and fifty-nine SNP-based haplotypes significantly associated with SY, SW and PH, respectively. Although most marker-trait associations were environment and trait specific, stable haplotype associations were identified for SY and SW across environments (i.e., haplotypes <span class="gene">Gm12_Hap12).</span> The haplotype block 42 on Chr19 <span class="gene">(Gm19_Hap42)</span> was confirmed to be associated with PH in two environments. These findings enable us to refine the breeding strategy for tropical soybean, which confirm that haplotype-based GWAS can provide new insights on the genetic determinants that are not captured by the single-marker approach.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association study, genomic prediction and marker-assisted selection for <span class="trait">seed</span> weight in soybean (Glycine max).</h2>
			<h3>PMID:26518570  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang J(1), Song Q(2), Cregan PB(2), Jiang GL(3).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Twenty-two loci for soybean SW and candidate genes conditioning <span class="trait">seed</span> development were identified; and prediction accuracies of <span class="gene">GS</span> and MAS were estimated through cross-validation and validation with unrelated populations. Soybean (Glycine max) is a major crop for plant protein and oil production, and <span class="trait">seed</span> weight (SW) is important for <span class="trait">yield</span> and quality in food/vegetable uses of soybean. However, our knowledge of genes controlling SW remains limited. To better understand the molecular mechanism underlying the trait and explore marker-based breeding approaches, we conducted a genome-wide association study in a population of 309 soybean germplasm accessions using 31,045 single nucleotide polymorphisms (SNPs), and estimated the prediction accuracy of genomic selection (GS) and marker-assisted selection (MAS) for SW. Twenty-two loci of minor effect associated with SW were identified, including hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> The mixed model containing these loci explained 83.4% of phenotypic variation. Candidate genes with Arabidopsis orthologs conditioning SW were also proposed. The prediction accuracies of <span class="gene">GS</span> and MAS by cross-validation were 0.75-0.87 and 0.62-0.75, respectively, depending on the number of SNPs used and the size of training population. <span class="gene">GS</span> also outperformed MAS when the validation was performed using unrelated panels across a wide range of maturities, with an average prediction accuracy of 0.74 versus 0.53. This study convincingly demonstrated that soybean SW is controlled by numerous minor-effect loci. It greatly enhances our understanding of the genetic basis of SW in soybean and facilitates the identification of genes controlling the trait. It also suggests that <span class="gene">GS</span> holds promise for accelerating soybean breeding progress. The results are helpful for genetic improvement and genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Scanning and Filling: Ultra-Dense SNP Genotyping Combining Genotyping-By-Sequencing, SNP Array and Whole-Genome Resequencing Data</h2>
			<h3>PMID:26161900</h3>
				
					<h3>PMCID:4498655</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Torkamaneh </surname>  <given-names> Davoud </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Belzile </surname>  <given-names> Francois </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To determine if the enhanced SNP catalogs obtained through imputation could provide increased power in genome-wide association scans, a subset composed of 139 soybean lines was used to perform an association analysis for <span class="trait">seed</span> oil content. This subset was used because phenotypic data were available only for these lines. One analysis was conducted using a &#x0201c;basic&#x0201d; GBS catalog of 7,152 SNPs obtained at MaxMD = 20% and MinMAF = 0.05, while the other was performed using an enhanced catalog resulting from imputation of missing GBS data (at MaxMD = 80%) and untyped loci from the SoySNP50K dataset. At MAF&#x02265;0.05, a total of 83,532 SNPs were retained within this combined dataset. As can be seen in <xref rid="pone.0131533.g004" ref-type="fig"> Fig 4A </xref> , using the &#x0201c;basic&#x0201d; SNP catalog, a single SNP marker on <span class="gene">Gm19</span> showed a significant association ( <italic> p </italic> = 9.6&#x000d7;10 <sup> &#x02212;3 </sup> and <italic> q </italic> = 0.09) with <span class="trait">seed</span> oil content. In contrast, using the enhanced SNP catalog ( <xref rid="pone.0131533.g004" ref-type="fig"> Fig 4B </xref> ) and a multi-locus mixed-model implementation, a total of 11 markers were in significant association with this trait despite the fact that the significance threshold increased from 3.4 to 5.3 (-Log <sub> 10 </sub> <italic> p- </italic> value). Interestingly, the peak SNP in both cases was the same <span class="gene">(Gm19_41742182),</span> but its association with oil content exhibited a much higher <italic> p </italic> -value (3.1&#x000d7;10 <sup> &#x02212;7 </sup> ) and lower <italic> q </italic> -value (0.01). This demonstrates that the increased number of informative SNP loci, obtained through the imputation of both missing GBS data and untyped loci from additional sources of SNP haplotype information, can prove highly beneficial in studying the genetic architecture of complex traits.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Enhancing soybean photosynthetic CO2 assimilation using a cyanobacterial membrane protein, ictB.</h2>
			<h3>PMID:28273517  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hay WT(1), Bihmidine S(2), Mutlu N(3), Hoang KL(4), Awada T(2), Weeks DP(3), Clemente TE(5), Long SP(6).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="gene">C3</span> photosynthesis can suffer a severe loss in efficiency due to photorespiration and the lack of a carbon concentrating mechanism (CCM) such as those present in other plant species or cyanobacteria. Transgenic soybean (Glycine max cv. Thorne) plants constitutively expressing cyanobacterial ictB (inorganic carbon transporter B) gene were generated using Agrobacterium-mediated transformation. Although more recent data suggest that ictB does not actively transport HCO3-/CO2, there is nevertheless mounting evidence that transformation with this gene can increase higher plant photosynthesis. The hypothesis that expression of the ictB gene would improve photosynthesis, biomass production and <span class="trait">seed</span> <span class="trait">yield</span> in soybean was tested, in two independent replicated greenhouse and field trials. Results showed significant increases in photosynthetic CO2 uptake (Anet) and dry mass in transgenic relative to wild type (WT) control plants in both the greenhouse and field trials. Transgenic plants also showed increased photosynthetic rates and biomass production during a <span class="trait">drought</span> mimic study. The findings presented herein demonstrate that ictB, as a single-gene, contributes to enhancement in various <span class="trait">yield</span> parameters in a major commodity crop and point to the significant role that biotechnological approaches to increasing photosynthetic efficiency can play in helping to meet increased global demands for food.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of cyanobacterial FBP/SBPase in soybean prevents <span class="trait">yield</span> depression under future climate conditions.</h2>
			<h3>PMID:28204603</h3>
				

				
			<p>
				作者：Köhler IH(1)(2), Ruiz-Vera UM(1), VanLoocke A(3), Thomey ML(1)(2), Clemente T(4), Long SP(4), Ort DR(1)(2)(5), Bernacchi CJ(1)(2)(5).
				<br/> 
				年份：2017 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Predictions suggest that current crop production needs to double by 2050 to meet global food and energy demands. Based on theory and experimental studies, overexpression of the photosynthetic enzyme sedoheptulose-1,7-bisphosphatase (SBPase) is expected to enhance <span class="gene">C3</span> crop photosynthesis and yields. Here we test how expression of the cyanobacterial, bifunctional fructose-1,6/sedoheptulose-1,7-bisphosphatase (FBP/SBPase) affects carbon assimilation and <span class="trait">seed</span> <span class="trait">yield</span> (SY) in a major crop (soybean, Glycine max). For three growing seasons, wild-type (WT) and FBP/SBPase-expressing (FS) plants were grown in the field under ambient (400 μmol mol−1) and elevated (600 μmol mol−1) CO2 concentrations [CO2] and under ambient and elevated temperatures (+2.7 °C during daytime, +3.4 °C at night) at the SoyFACE research site. Across treatments, FS plants had significantly higher carbon assimilation (4–14%), Vc,max (5–8%), and Jmax (4–8%). Under ambient [CO2], elevated temperature led to significant reductions of SY of both genotypes by 19–31%. However, under elevated [CO2] and elevated temperature, FS plants maintained SY levels, while the WT showed significant reductions between 11% and 22% compared with plants under elevated [CO2] alone. These results show that the manipulation of the photosynthetic carbon reduction cycle can mitigate the effects of future high CO2 and high temperature environments on soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Is there potential to adapt soybean (Glycine max Merr.) to future [CO₂]? An analysis of the <span class="trait">yield</span> response of 18 genotypes in free-air CO₂ enrichment.</h2>
			<h3>PMID:25211487  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bishop KA(1), Betzelberger AM(1), Long SP(1)(2), Ainsworth EA(1)(3).
				<br/> 
				年份：2015 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rising atmospheric [CO2] is a uniform, global change that increases <span class="gene">C3</span> photosynthesis and could offset some of the negative effects of global climate change on crop yields. Genetic variation in <span class="trait">yield</span> responsiveness to rising [CO2] would provide an opportunity to breed more responsive crop genotypes. A multi-year study of 18 soybean (Glycine max Merr.) genotypes was carried out to identify variation in responsiveness to season-long elevated [CO2] (550 ppm) under fully open-air replicated field conditions. On average across 18 genotypes, elevated [CO2] stimulated total above-ground biomass by 22%, but <span class="trait">seed</span> <span class="trait">yield</span> by only 9%, in part because most genotypes showed a reduction in partitioning of energy to seeds. Over four years of study, there was consistency from year to year in the genotypes that were most and least responsive to elevated [CO2], suggesting heritability of CO2 response. Further analysis of six genotypes did not reveal a photosynthetic basis for the variation in <span class="trait">yield</span> response. Although partitioning to <span class="trait">seed</span> was decreased, cultivars with the highest partitioning coefficient in current [CO2 ] also had the highest partitioning coefficient in elevated [CO2]. The results show the existence of genetic variation in soybean response to elevated [CO2], which is needed to breed soybean to the future atmospheric environment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Global food insecurity. treatment of major food crops with elevated carbon dioxide or ozone under large-scale fully open-air conditions suggests recent models may have overestimated future yields.</h2>
			<h3>PMID:16433090  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Long SP(1), Ainsworth EA, Leakey AD, Morgan PB.
				<br/> 
				年份：2005 Nov 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Predictions of <span class="trait">yield</span> for the globe's major grain and legume arable crops suggest that, with a moderate temperature increase, production may increase in the temperate zone, but decline in the tropics. In total, global food supply may show little change. This security comes from inclusion of the direct effect of rising carbon dioxide (CO2) concentration, [CO2], which significantly stimulates <span class="trait">yield</span> by decreasing photorespiration in <span class="gene">C3</span> crops and transpiration in all crops. Evidence for a large response to [CO2] is largely based on studies made within chambers at small scales, which would be considered unacceptable for standard agronomic trials of new cultivars or agrochemicals. Yet, predictions of the globe's future food security are based on such inadequate information. Free-Air Concentration Enrichment (FACE) technology now allows investigation of the effects of rising [CO2] and ozone on field crops under fully open-air conditions at an agronomic scale. Experiments with rice, wheat, maize and soybean show smaller increases in <span class="trait">yield</span> than anticipated from studies in chambers. Experiments with increased ozone show large <span class="trait">yield</span> losses (20%), which are not accounted for in projections of global food security. These findings suggest that current projections of global food security are overoptimistic. The fertilization effect of CO2 is less than that used in many models, while rising ozone will cause large <span class="trait">yield</span> losses in the Northern Hemisphere. Unfortunately, FACE studies have been limited in geographical extent and interactive effects of CO2, ozone and temperature have yet to be studied. Without more extensive study of the effects of these changes at an agronomic scale in the open air, our ever-more sophisticated models will continue to have feet of clay.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthesis and increased production of protein.</h2>
			<h3>PMID:727014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Platt SG, Bassham JA.
				<br/> 
				年份：1978
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosynthesis, the use of light energy in the conversion of CO2 and inorganic nutrients into plant material, is the ultimate source of the food protein necessary to man's existence. Given certain assumptions, the overall maximal theoretical photosynthetic efficiency of agricultural plants can be calculated. Actual measured maximal growth rates of plants are equivalent to efficiency levels well below that theoretical maximum. In air, <span class="gene">C4</span> plants can some closer to the theoretical value than <span class="gene">C3</span> plants, perhaps because <span class="gene">C4</span> plants avoid the occurrence of measurable photorespiration and oxygen inhibition of photosynthesis. Alfalfa, a <span class="gene">C3</span> legume, is an extremely productive protein source. Its protein <span class="trait">yield</span> per acre can surpass that of commonly grown <span class="gene">C4</span> crops (corn, sorghum) and <span class="gene">C3</span> <span class="trait">seed</span> crops (soybean, wheat, rice). Alfalfa leaf protein is of high nutritional quality and can apparently be used directly in the human diet, eliminating the protein loss involved in animal production. Plant protein productivity can be raised as part of an increase in overall crop <span class="trait">yield.</span> The growth of plants in atmospheres with elevated CO2 levels can result in increased yields. In <span class="gene">C3</span> plants this is due, at least in part, to the suppression of photorespiration and oxygen inhibition of photosynthesis. We have investigated the effect of CO2 concentration on alfalfa photosynthetic metabolism. Our results support the contention that alfalfa productivity can be increased by an environment of elevated CO2. A second approach toward increased plant protein productivity is through regulation of carbon flow during photosynthesis so as to increase portein production relative to that of other plant constituents. In particular, we have investigated whether ammonia (the form in which plants first incorporate nitrogen) can act to regulate leaf carbon metabolism. Our results indicate that NH4+, in part through stimulation of pyruvate kinase, brings about increased production of amino acids at the expense of sucrose production in alfalfa. The effect may be of considerable importance in the regulation of green leaf protein synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping QTLs for <span class="trait">seed</span> <span class="trait">yield</span> and <span class="trait">drought</span> susceptibility index in soybean (Glycine max L.) across different environments.</h2>
			<h3>PMID:20129399  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Wang M, Fu S, Yu D.
				<br/> 
				年份：2009 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress has long been a major constraint in maintaining <span class="trait">yield</span> stability of soybean (Glycine max (L.) Merr.) in rainfed ecosystems. The identification of consistent quantitative trait loci (QTL) involving <span class="trait">seed</span> <span class="trait">yield</span> per plant (YP) and <span class="trait">drought</span> susceptibility index (DSI) in a population across different environments would therefore be important in molecular marker-assisted breeding of soybean cultivars suitable for rainfed regions. The YP of a recombinant line population of 184 F(2:7:11) lines from a cross of Kefeng1 and Nannong1138-2 was studied under water-stressed (WS) and well-watered (WW) conditions in field (F) and greenhouse (G) trials, and DSI for <span class="trait">yield</span> was calculated in two trials. Nineteen QTLs associated with YP-WS and YP-WW, and 10 QTLs associated with DSI, were identified. Comparison of these QTL locations with previous findings showed that the majority of these regions control one or more traits related to <span class="trait">yield</span> and other agronomic traits. One QTL on molecular linkage group (MLG) K for YP-F, and two QTLs on MLG <span class="gene">C2</span> for YP-G, remained constant across different water regimes. The regions on MLG <span class="gene">C2</span> for YP-WW-F and MLG H for YP-WS-F had a pleiotropic effect on DSI-F, and MLG A1 for YP-WS-G had a pleiotropic effect on DSI-G. The identification of consistent QTLs for YP and DSI across different environments will significantly improve the efficiency of selecting for <span class="trait">drought</span> tolerance in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						2009 Institute of Genetics and Developmental Biology and the Genetics Society of China. Published by Elsevier Ltd. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of quantitative trait loci for <span class="trait">yield</span> and <span class="trait">drought</span> tolerance traits in soybean using a recombinant inbred line population.</h2>
			<h3>PMID:19723246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Yu D, Fu S.
				<br/> 
				年份：2009 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the genetic basis of <span class="trait">drought</span> tolerance in soybean (Glycine max L. Merr.) a recombinant inbred population with 184 F(2:7:11) lines developed from a cross between Kefeng1 <span class="trait">(drought</span> tolerant) and Nannong1138-2 <span class="trait">(drought</span> sensitive) were tested under water-stressed and well-watered conditions in field and greenhouse trials. Traits measured included leaf wilting coefficient, excised leaf water loss and relative water content as indicators of plant water status and <span class="trait">seed</span> <span class="trait">yield.</span> A total of 40 quantitative trait loci (QTLs) were identified: 17 for leaf water status traits under <span class="trait">drought</span> stress and 23 for <span class="trait">seed</span> <span class="trait">yield</span> under well-watered and drought-stressed conditions in both field and greenhouse trials. Two <span class="trait">seed</span> <span class="trait">yield</span> QTLs were detected under both well-watered and drought-stressed conditions in the field on molecular linkage group H and D1b, while two <span class="trait">seed</span> <span class="trait">yield</span> QTLs on molecular linkage group <span class="gene">C2</span> were found under greenhouse conditions. Several QTLs for traits associated with plant water status were identified in both field and greenhouse trials, including two leaf wilting coefficient QTLs on molecular linkage group A2 and one excised leaf water loss QTL on molecular linkage group H. Phenotypic correlations of traits suggested several QTLs had pleiotropic or location-linked associations. These results will help to elucidate the genetic basis of <span class="trait">drought</span> tolerance in soybean, and could be incorporated into a marker-assisted selection breeding program to develop high-yielding soybean cultivars with improved tolerance to <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean <span class="trait">seed</span> isoflavone content: importance of statistical model and epistasis in complex traits</h2>
			<h3>PMID:19626310</h3>
				
					<h3>PMCID:2755750</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name name-style="western">  <surname> Gutierrez-Gonzalez </surname>  <given-names> Juan Jose </given-names>  </name>  <address>  <email> juangutierrez@mizzou.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Wu </surname>  <given-names> Xiaolei </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Zhang </surname>  <given-names> Juan </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Lee </surname>  <given-names> Jeong-Dong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Ellersieck </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Yu </surname>  <given-names> Oliver </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name name-style="western">  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name name-style="western">  <surname> Sleper </surname>  <given-names> David A. </given-names>  </name>  <address>  <phone> +1-573-8827320 </phone>  <fax> +1-573-8849676 </fax>  <email> Sleperd@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Division of Plant Sciences, National Center for Soybean Biotechnology, University of Missouri, Life Sciences Building, Columbia, MO 65201 USA  </aff>  <aff id="Aff2">  <label> 2 </label> Donald Danforth Plant Science Center, 975 North Warson Road, Saint Louis, MO 63132 USA  </aff>  <aff id="Aff3">  <label> 3 </label> Department of Statistics, University of Missouri, 146 Middlebush Hall, Columbia, MO 65211 USA  </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 6 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Glycitein is the most variable isoflavone in soybean (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). This complicates QTL discovery because it requires larger mapping populations and advanced statistical methods that can take into account most of that variation. Six loci were found to control glycitein accumulation when considering all mapping algorithms together (Table&#x000a0; <xref rid="Tab5" ref-type="table"> 5 </xref> ; Fig.&#x000a0; <xref rid="Fig1" ref-type="fig"> 1 </xref> ). Two of them viz. one in LG <span class="gene">C2</span> ( <italic> gly-C2 </italic> ), and the other in LG E ( <italic> gly-E) </italic> , were revealed by MCIM model with data from BREC and DRC combined. Locus <italic> gly-C2 </italic> was the major glycitein QTL reported herein. However, IM could not map a single QTL at BREC, and just <italic> gly-C2 </italic> at DRC. This same region was also detected with CIM in DRC, and with MIM in both locations. Two other loci in LG D2 ( <italic> gly-D2_1 </italic> and <italic> gly-D2_2 </italic> ) were found to be significant only in BREC. On the other hand, <italic> gly-G </italic> and <italic> gly-K </italic> were detected at DRC solely. Interestingly, no one of the QTLs for glycitein discovered in the present study was previously reported. <table-wrap id="Tab5"> <label> Table&#x000a0;5 </label> <caption> <p> QTL bearing additive effect detected for glycitein <span class="trait">seed</span> content by different models
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evalution of DNA extraction methods in order to monitor genetically modified materials in soy foodstuffs and feeds commercialised in Turkey by multiplex real-time PCR.</h2>
			<h3>PMID:24816675  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Turkec A(1), Kazan H, Baykut A, Lucas SJ.
				<br/> 
				年份：2015 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is one of the most important biotech crops, widely used as an ingredient in both foodstuffs and feed. DNA extraction methods have been evaluated to detect the presence of genetically modified (GM) materials in soya-containing food and feed products commercialised in Turkey. RESULTS: All extraction methods performed well for the majority of soya foods and feed products analysed. However, the most successful method varied between different products; the Foodproof, Genespin and the cetyltrimethylammonium bromide (CTAB) methods each produced the highest DNA <span class="trait">yield</span> and purity for different soya foodstuffs and feeds. Of the samples tested, 20% were positive for the presence of at least two GM elements (35S/NOS) while 11% contained an additional GM element (35S/NOS/FMV). Of the tested products, animal feeds showed a larger prevalence of GM material (50%) than the soya-containing foodstuffs (13%). CONCLUSION: The best performing extraction methods proved to be the Foodproof, Genespin and CTAB methods for soya-containing food and feed products. The results obtained herein clearly demonstrate the presence of GM soybean in the Turkish market, and that the Foodproof <span class="gene">GMO</span> Screening Kit provides reliable screening of soy-containing food and feed products.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative estimation of sampling uncertainties for mycotoxins in cereal shipments.</h2>
			<h3>PMID:22651820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bourgeois FS(1), Lyman GJ.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many countries receive shipments of bulk cereals from primary producers. There is a volume of work that is on-going that seeks to arrive at appropriate standards for the quality of the shipments and the means to assess the shipments as they are out-loaded. Of concern are mycotoxin and heavy metal levels, pesticide and herbicide residue levels, and contamination by genetically modified organisms <span class="gene">(GMOs).</span> As the ability to quantify these contaminants improves through improved analytical techniques, the sampling methodologies applied to the shipments must also keep pace to ensure that the uncertainties attached to the sampling procedures do not overwhelm the analytical uncertainties. There is a need to understand and quantify sampling uncertainties under varying conditions of contamination. The analysis required is statistical and is challenging as the nature of the distribution of contaminants within a shipment is not well understood; very limited data exist. Limited work has been undertaken to quantify the variability of the contaminant concentrations in the flow of grain coming from a ship and the impact that this has on the variance of sampling. Relatively recent work by Paoletti et al. in 2006 [Paoletti C, Heissenberger A, Mazzara M, Larcher S, Grazioli E, Corbisier P, Hess N, Berben G, Lübeck PS, De Loose M, et al. 2006. Kernel lot distribution assessment (KeLDA): a study on the distribution of <span class="gene">GMO</span> in large soybean shipments. Eur Food Res Tech. 224:129-139] provides some insight into the variation in <span class="gene">GMO</span> concentrations in soybeans on cargo out-turn. Paoletti et al. analysed the data using correlogram analysis with the objective of quantifying the sampling uncertainty (variance) that attaches to the final cargo analysis, but this is only one possible means of quantifying sampling uncertainty. It is possible that in many cases the levels of contamination passing the sampler on out-loading are essentially random, negating the value of variographic quantitation of the sampling variance. <span class="gene">GMOs</span> and mycotoxins appear to have a highly heterogeneous distribution in a cargo depending on how the ship was loaded (the grain may have come from more than one terminal and set of storage silos) and mycotoxin growth may have occurred in transit. This paper examines a statistical model based on random contamination that can be used to calculate the sampling uncertainty arising from primary sampling of a cargo; it deals with what is thought to be a worst-case scenario. The determination of the sampling variance is treated both analytically and by Monte Carlo simulation. The latter approach provides the entire sampling distribution and not just the sampling variance. The sampling procedure is based on rules provided by the Canadian Grain Commission (CGC) and the levels of contamination considered are those relating to allowable levels of ochratoxin A (OTA) in wheat. The results of the calculations indicate that at a loading rate of 1000 tonnes h(-1), primary sample increment masses of 10.6 kg, a 2000-tonne lot and a primary composite sample mass of 1900 kg, the relative standard deviation (RSD) is about 1.05 (105%) and the distribution of the mycotoxin (MT) level in the primary composite samples is highly skewed. This result applies to a mean MT level of 2 ng g(-1). The rate of false-negative results under these conditions is estimated to be 16.2%. The corresponding contamination is based on initial average concentrations of MT of 4000 ng g(-1) within average spherical volumes of 0.3 m diameter, which are then diluted by a factor of 2 each time they pass through a handling stage; four stages of handling are assumed. The Monte Carlo calculations allow for variation in the initial volume of the MT-bearing grain, the average concentration and the dilution factor. The Monte Carlo studies seek to show the effect of variation in the sampling frequency while maintaining a primary composite sample mass of 1900 kg. The overall results are presented in terms of operational characteristic curves that relate only to the sampling uncertainties in the primary sampling of the grain. It is concluded that cross-stream sampling is intrinsically unsuited to sampling for mycotoxins and that better sampling methods and equipment are needed to control sampling uncertainties. At the same time, it is shown that some combination of cross-cutting sampling conditions may, for a given shipment mass and MT content, <span class="trait">yield</span> acceptable sampling performance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural variability in abundance of prevalent soybean proteins.</h2>
			<h3>PMID:20709130  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Natarajan SS(1).
				<br/> 
				年份：2010 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is an inexpensive source of protein for humans and animals. Genetic modifications <span class="gene">(GMO)</span> to soybean have become inevitable on two fronts, both quality and <span class="trait">yield</span> will need to improve to meet increasing global demand. To ensure the safety of the crop for consumers it is important to determine the natural variation in <span class="trait">seed</span> protein constituents as well as any unintended changes that may occur in the <span class="gene">GMO</span> as a result of genetic modification. Understanding the natural variation of <span class="trait">seed</span> proteins in wild and cultivated soybeans that have been used in conventional soybean breeding programs is critical for determining unintended protein expression in <span class="gene">GMO</span> soybeans. In recent years, proteomic technologies have been used as an effective analytical tool for examining modifications of protein profiles. We have standardized and applied these technologies to determine and quantify the spectrum of proteins present in soybean <span class="trait">seed.</span> We used two-dimensional polyacrylamide gel electrophoresis (2D-PAGE), matrix-assisted laser desorption/ionization time of flight mass spectrometry (MALDI-TOF-MS), and liquid chromatography mass spectrometry (LC-MS) for the separation, quantification, and identification of different classes of soybean <span class="trait">seed</span> proteins. We have observed significant variations in different classes of proteins, including storage, allergen and anti-nutritional protein profiles, between non-GMO cultivated and wild soybean varieties. This information is useful for scientists and regulatory agencies to determine whether the unintended expression of proteins found in transgenic soybean is within the range of natural variation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Real-time quantification of wild-type contaminants in glyphosate tolerant soybean</h2>
			<h3>PMID:19267904</h3>
				
					<h3>PMCID:2656496</h3>
					<h4>期刊:BMC Biotechnology</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Battistini </surname>  <given-names> Elena </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ebattistini@agrsci.unibo.it </email>  </contrib>  <contrib id="A2" corresp="yes" contrib-type="author">  <name>  <surname> Noli </surname>  <given-names> Enrico </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> enrico.noli@unibo.it </email>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 3 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Real-time quantitative PCR (rtqPCR) has found numerous applications in research and diagnostics, both in the animal and in the plant sciences [ <xref ref-type="bibr" rid="B7"> 7 </xref> - <xref ref-type="bibr" rid="B9"> 9 </xref> ]. For the plant biotech industry it represents a useful tool for the determination of copy number [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ] and zygosity [ <xref ref-type="bibr" rid="B12"> 12 </xref> , <xref ref-type="bibr" rid="B13"> 13 </xref> ] of the inserted genes in the development of transgenic varieties. For regulatory purposes, this technology, in particular when associated with hydrolysis probes, is commonly utilized in the food/feed as well as in the <span class="trait">seed</span> sector for the detection and quantification of AP in conventional material [see Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ] [ <xref ref-type="bibr" rid="B14"> 14 </xref> - <xref ref-type="bibr" rid="B16"> 16 </xref> ]. As a matter of fact, for this purpose, rtqPCR methods are not only widely used, but they are also required by the EU <span class="gene">GMO</span> legislation. EC/787/2004 recommends the use of DNA based quantification of the <span class="gene">GMO</span> content in seeds, food and feed and this has to be done by means of rtq-PCR. Methods are submitted as a part of the authorization procedure to the Community Reference Laboratory (CRL) which coordinates the validation process. Despite the great abundance of publications [ <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B6"> 6 </xref> ] and the availability of on-line databases [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B18"> 18 </xref> ] largely dedicated to rtqPCR methods for AP testing, up to now there were no reports on the application of rtqPCR to assess the purity of GM seedlots or of derived agricultural products. In fact, due to the exponential nature of PCR, the rtqPCR assays designed so far, targeted to the amplification of the transgenic sequence, are not suitable for checking trait purity in GM seedlots. For example, in the case of a glyphosate tolerant soybean variety, assuming that all the seeds with the trait were homozygous, only one threshold cycle (Ct) difference would actually be expected between a perfectly pure (100% herbicide tolerant) and a "very bad" (50% herbicide tolerant) seedlot [see Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Like tests designed to evaluate zygosity [ <xref ref-type="bibr" rid="B22"> 22 </xref> , <xref ref-type="bibr" rid="B27"> 27 </xref> ], the assays here described, when used on single seeds or plants, can be valuable tools in the conversion of conventional varieties [ <xref ref-type="bibr" rid="B28"> 28 </xref> ] by marker assisted backcrossing and in quality control at the early stages of <span class="trait">seed</span> increase, replacing laborious progeny testing. However, the rtqPCR format represents a true novelty since it provides an accurate estimate of the proportion of wt gametes that will contribute to the following generation, facilitating maintenance breeding of GM homozygous varieties and, in the case of hybrid crops, of inbred lines. Moreover it is the only possible approach for the quantification of trait purity in materials not formed of discrete particles (derived products, e.g. flour, extracts, etc.), as it could be in the next future for high value products derived from "pharma crops", or for the preparation of pure reference materials. As an obvious extension, this approach could be applied to the purity determination of any trait for which allelic discrimination is possible at a specific locus, like in the case of mutants or naturally occurring variants not strictly classifiable as <span class="gene">GMOs.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL for increased fibrous roots in soybean.</h2>
			<h3>PMID:21165732  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Lee GJ, Boerma RH.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects soybean at various developmental stages, which collectively results in <span class="trait">yield</span> reduction. Unpredictable rainfall has been reported to contribute about 36% to variation of <span class="trait">yield</span> difference between the rain-fed and irrigated fields. Among the <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms, <span class="trait">drought</span> avoidance in genotypes with fibrous roots was recognized to be associated with <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Plant introduction PI416937 was shown to possess fibrous roots and has been used as a parent in breeding programs to improve soybean productivity. Little information is available on relative contribution and chromosomal location of quantitative trait loci (QTL) conditioning fibrous roots in soybean. To identify the genomic locations and genetic bases of this trait, a recombinant inbred line population was derived from a cross between PI416937 and 'Benning'. To detect associated QTLs, phenotypic data were collected and analyzed for 2 years under rain-fed field conditions. The selective genotyping approach was used to reduce the costs and work associated with conducting the QTL analysis. A total of five QTLs were identified on chromosomes <span class="gene">Gm01</span> (Satt383), <span class="gene">Gm03</span> (Satt339), <span class="gene">Gm04</span> (Sct_191), <span class="gene">Gm08</span> (Satt429), and <span class="gene">Gm20</span> (Sat_299), and together explained 51% of the variation in root score. Detected QTLs were co-localized with QTLs related to root morphology, suggesting that fibrous roots QTL may be associated with other morpho-physiological traits and <span class="trait">seed</span> <span class="trait">yield</span> in soybean. Genetic dissection of the fibrous roots trait at the individual marker loci will allow for marker-assisted selection to develop soybean genotypes with enhanced levels of fibrous roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Foliar and <span class="trait">Seed</span> Application of Amino Acids Affects the Antioxidant Metabolism of the Soybean Crop.</h2>
			<h3>PMID:28377778</h3>
				

				
			<p>
				作者：Teixeira WF(1), Fagan EB(2), Soares LH(2), Umburanas RC(1), Reichardt K(3), Neto DD(1).
				<br/> 
				年份：2017 Mar 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In recent years, the application of natural substances on crops has been intensified in order to increase the <span class="trait">resistance</span> and <span class="trait">yield</span> of the soybean crop. Among these products are included plant biostimulants that may contain algae extracts, amino acids, and plant regulators in their composition. However, there is little information on the isolated effect of each of these constituents. The objective of this research was to evaluate the effect of the application of isolated amino acids on the antioxidant metabolism of the soybean crop. Experiments were carried out in a greenhouse and in the field with the application of the amino acids glutamate, phenylalanine, cysteine, glycine in <span class="trait">seed</span> treatment, and foliar application at V4 growth stage. Antioxidant metabolism constituents evaluated were superoxide dismutase, catalase, peroxidase, hydrogen peroxide content, proline, and lipid peroxidation. In addition, <span class="trait">resistance</span> enzymes as polyphenol oxidase and phenylalanine ammonia-lyase (PAL) were evaluated. In both experiments, the use of cysteine, only in <span class="trait">seed</span> treatment and in both <span class="trait">seed</span> treatment and foliar application increased the activity of the enzyme <span class="gene">PAL</span> and catalase. Also in both experiments, the use of phenylalanine increased the activity of the enzyme <span class="gene">PAL</span> when the application was carried out as foliar application or both in <span class="trait">seed</span> treatment and foliar application. In the field experiment, the application of glutamate led to an increase in the activity of the catalase and <span class="gene">PAL</span> enzymes for <span class="trait">seed</span> treatment and foliar application. The use of the set of amino acids was only efficient in foliar application, which led to a greater activity of the enzymes peroxidase, PAL, and polyphenol oxidase. The other enzymes as well as lipid peroxidation and hydrogen peroxide presented different results according to the experiment. Therefore, glutamate, cysteine, phenylalanine, and glycine can act as signaling amino acids in soybean plants, since small doses are enough to increase the activity of the antioxidant enzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant growth-promotion (PGP) activities and molecular characterization of rhizobacterial strains isolated from soybean (Glycine max L. Merril) plants against charcoal rot pathogen, Macrophomina phaseolina.</h2>
			<h3>PMID:21833548  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choudhary DK(1).
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Charcoal rot disease, caused by the fungus Macrophomina phaseolina, leads to significant <span class="trait">yield</span> losses of soybean crops. One strategy to control charcoal rot is the use of antagonistic, root-colonizing bacteria. Rhizobacteria A(5)F and FPT(7)21 and Pseudomonas sp. strain GRP(3) were characterized for their plant growth-promotion activities against the pathogen. Rhizobacterium FPT(7)21 exhibited higher antagonistic activity against the pathogen on dual plate assay compared to strain A(5)F and GRP(3). FPT(7)21 and GRP(3) gave decreased disease intensity in terms of average number of pathogen-infested plants. Lipoxygenase (LOX), phenylalanine ammonia-lyase (PAL), and peroxidase (POD) activities were estimated in extracts of plants grown from seeds that were treated with rhizobacteria, and inoculated with spore suspension of M. phaseolina. The activity of these enzymes after challenge with the test pathogen increased. Strains FPT(7)21 and GRP(3) exhibited maximum increases in LOX, <span class="gene">PAL</span> and POD activity (U mg(-1) fresh leaf wt) compared to strain A(5)F.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induction of Systemic <span class="trait">Resistance</span> of Benzothiadiazole and Humic Acid in Soybean Plants Against Fusarium Wilt Disease</h2>
			<h3>PMID:22783118</h3>
				
					<h3>PMCID:3385127</h3>
					<h4>期刊:Mycobiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Abdel-Monaim </surname>  <given-names> Montaser Fawzy </given-names>  </name>  <xref ref-type="aff" rid="A1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ismail </surname>  <given-names> Mamdoh Ewis </given-names>  </name>  <xref ref-type="aff" rid="A2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Morsy </surname>  <given-names> Kadry Mohamed </given-names>  </name>  <xref ref-type="aff" rid="A1"> 1 </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 8 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						BTH and HA induce disease <span class="trait">resistance</span> and increase <span class="trait">yield</span> in a number of plants including soybean and other legumes against a broad range of pathogens [ <xref ref-type="bibr" rid="B5"> 5 </xref> , <xref ref-type="bibr" rid="B7"> 7 </xref> , <xref ref-type="bibr" rid="B23"> 23 </xref> , <xref ref-type="bibr" rid="B24"> 24 </xref> ]. These compounds have no direct antimicrobial activity against many fungal and bacterial pathogens [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. We suggest, that, for soybean plants, BTH and HA may stimulate inherent defense mechanisms that enable plants to respond more quickly against invading and colonizing fungi. It seems likely that the increased activity of enzymes involved in defense reactions may be one of the basic ways that BTH and HA induce <span class="trait">resistance</span> in soybeans against wilt disease. This view is entirely consistent with the present observations of increased activity of PO, PPO, and <span class="gene">PAL</span> in soybean plants treated with BTH and/or HA. On the other hand, the susceptibility of the four soybean cultivars was positively correlated with the activity of these enzymes; the Giza 111 cultivar, which was more resistant to <italic> F. oxysporum </italic> , displayed the highest activities of the enzymes, while the Giza 21 cultivar, which was highly susceptible, displayed the lowest enzyme activities in treated or untreated plants. Also, the total phenolic content was markedly increased in plants treated with the inducers compared with untreated ones in all the tested cultivars, with the BTH + HA combination producing more of an increase than their individual application.
					</p>
				
					<p>&nbsp;&nbsp;
						Concerning the role of BTH as an <span class="trait">resistance</span> inducer, it has been reported that induction of SAR gene expression by BTH does not require the contribution of salicylic acid (SA), which suggests that this compound could act as a secondary messenger analog capable of activating SAR signal transduction pathway independent of the accumulation of other signal molecules [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Application of BTH to a variety of plants before pathogen challenge triggers a set of plant defense reactions that result in the creation of a fungitoxic environment, which protects them by different physical and/or chemical mechanisms [ <xref ref-type="bibr" rid="B23"> 23 </xref> ]. Also, treatment with BTH and HA presently led to an increase in PO, PPO, and <span class="gene">PAL</span> activity, consistent with the previous demonstration that the expression of <span class="trait">resistance</span> is often accompanied by the activation of phenol-oxidizing enzymes such as PO, PPO, and <span class="gene">PAL</span> [ <xref ref-type="bibr" rid="B25"> 25 </xref> ]. Increased PO and PPO activity may contribute to defense through the production of oxidized forms of quinones, which can inactivate pectinolytic enzymes produced by pathogens. This suggests that PO and PPO could play an effective role in the observed <span class="trait">resistance.</span> PO and PPO have been associated with induced <span class="trait">resistance</span> and are involved in several plant defense mechanisms, such as lignin biosynthesis, oxidative cross-linking of plant cell walls, and generation of active oxygen species [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. <span class="gene">PAL</span> is a key enzyme of the phenylpropanoid pathway that leads to a variety of defense-related plant secondary metabolites such as SA, phytoalexins, and lignin-like polymers [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. <span class="gene">PAL</span> has been shown to play a critical role in acibenzolar-s-methyl (ASM)-mediated <span class="trait">resistance,</span> as its expression was primed early by ASM in Japanese pear [ <xref ref-type="bibr" rid="B26"> 26 </xref> ] and cucumber [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. However, the direct role of <span class="gene">PAL</span> in <span class="trait">resistance</span> induced by ASM comes from the work of Stadnik and Buchenauer [ <xref ref-type="bibr" rid="B28"> 28 </xref> ] who showed that chemical inhibition of <span class="gene">PAL</span> abolished <span class="trait">resistance</span> in wheat induced against <italic> Blumeria graminis </italic> f. sp. <italic> tritici </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional properties and expression quantitative trait loci for phosphate transporter <span class="gene">GmPT1</span> in soybean.</h2>
			<h3>PMID:23889314  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song H(1), Yin Z, Chao M, Ning L, Zhang D, Yu D.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphate (Pi) remobilization within a plant is critical for plant survival under Pi-limiting conditions. In this paper, a soybean Pi transporter gene, <span class="gene">GmPT1,</span> was characterized. A marked induction of <span class="gene">GmPT1</span> transcript was observed in young leaves, mature leaves and lateral roots during long-term Pi starvation. Transgenic tobacco plants containing the <span class="gene">GmPT1</span> gene were obtained using an Agrobacterium-mediated transformation system. Compared with wild-type plants, transgenic plants showed significant increases in phosphorus-use efficiency (PUE), photosystem II (PSII) function, total dry weight and <span class="trait">seed</span> weight under Pi-deficient conditions. <span class="gene">GmPT1</span> expression levels and PUE were determined in a soybean recombinant inbred line population during a pot experiment that was conducted to measure chlorophyll fluorescence parameters, photosynthetic rate (PN ) and <span class="trait">seed</span> <span class="trait">yield.</span> Correlation analysis revealed that <span class="gene">GmPT1</span> expression levels had significantly positive correlations with <span class="trait">seed</span> <span class="trait">yield,</span> PUE, PN and the quantum <span class="trait">yield</span> of PSII primary photochemistry (ΦPSII ). Expression quantitative trait loci (eQTL) mapping for <span class="gene">GmPT1</span> revealed two eQTLs, one of which coincided with both the physical location of <span class="gene">GmPT1</span> and a QTL associated with <span class="trait">seed</span> <span class="trait">yield.</span> These results suggest that <span class="gene">GmPT1</span> plays a role in Pi remobilization, and it may be possible to improve soybean <span class="trait">seed</span> yields under Pi-limiting conditions by modulating <span class="gene">GmPT1</span> expression levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic structure of populations of Rhizoctonia solani anastomosis group-1 IA from soybean in Brazil.</h2>
			<h3>PMID:18943212  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ciampi MB(1), Meyer MC, Costa MJ, Zala M, McDonald BA, Ceresini PC.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Basidiomycete fungus Rhizoctonia solani anastomosis group (AG)-1 IA is a major pathogen of soybean in Brazil, where the average <span class="trait">yield</span> losses have reached 30 to 60% in some states in Northern Brazil. No information is currently available concerning levels of genetic diversity and population structure for this pathogen in Brazil. A total of 232 isolates of R. solani <span class="gene">AG1</span> IA were collected from five soybean fields in the most important soybean production areas in central-western, northern, and northeastern Brazil. These isolates were genotyped using 10 microsatellite loci. Most of the multilocus genotypes (MLGTs) were site-specific, with few MLGTs shared among populations. Significant population subdivision was evident. High levels of admixture were observed for populations from Mato Grosso and Tocantins. After removing admixed genotypes, three out of five field populations (Maranhao, Mato Grosso, and Tocantins), were in Hardy-Weinberg (HW) equilibrium, consistent with sexual recombination. HW and gametic disequilibrium were found for the remaining soybean-infecting populations. The findings of low genotypic diversity, departures from HW equilibrium, gametic disequilibrium, and high degree of population subdivision in these R. solani AG-1 IA populations from Brazil are consistent with predominantly asexual reproduction, short-distance dispersal of vegetative propagules (mycelium or sclerotia), and limited long-distance dispersal, possibly via contaminated <span class="trait">seed.</span> None of the soybean-infecting populations showed a reduction in population size (bottleneck effect). We detected asymmetric historical migration among the soybean-infecting populations, which could explain the observed levels of subdivision.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Generation and characterization of two novel low phytate mutations in soybean (Glycine max L. Merr.).</h2>
			<h3>PMID:17701395  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan FJ(1), Zhao HJ, Ren XL, Zhu SL, Fu XJ, Shu QY.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (PA, myo-inositol 1, 2, 3, 4, 5, 6 hexakisphosphate) is important to the nutritional quality of soybean meal. Organic phosphorus (P) in PA is indigestible in humans and non-ruminant animals, which affects nutrition and causes P pollution of ground water from animal wastes. Two novel soybean [(Glycine max L. (Merr.)] low phytic acid (lpa) mutations were isolated and characterized. <span class="gene">Gm-lpa-TW-1</span> had a phytic acid P (PA-P) reduction of 66.6% and a sixfold increase in inorganic P (Pi), and <span class="gene">Gm-lpa-ZC-2</span> had a PA-P reduction of 46.3% and a 1.4-fold increase in Pi, compared with their respective non-mutant progenitor lines. The reduction of PA-P and increase of Pi in <span class="gene">Gm-lpa-TW-1</span> were molar equivalent; the decrease of PA-P in <span class="gene">Gm-lpa-ZC-2,</span> however, was accompanied by the increase of both Pi and lower inositol phosphates. In both mutant lines, the total P content remained similar to their wild type parents. The two lpa mutations were both inherited in a single recessive gene model but were non-allelic. Sequence data and progeny analysis indicate that <span class="gene">Gm-lpa-TW-1</span> lpa mutation resulted from a 2 bp deletion in the soybean D: -myo-inositol 3-phosphate synthase (MIPS1 EC 5.5.1.4) gene 1 (MIPS1). The lpa mutation in <span class="gene">Gm-lpa-ZC-2</span> was mapped on LG B2, closely linked with microsatellite loci Satt416 and Satt168, at genetic distances of approximately 4.63 and approximately 9.25 cM, respectively. Thus this mutation probably represents a novel soybean lpa locus. The <span class="trait">seed</span> emergence rate of <span class="gene">Gm-lpa-ZC-2</span> was similar to its progenitor line and was not affected by <span class="trait">seed</span> source and its lpa mutation. However, <span class="gene">Gm-lpa-TW-1</span> had a significantly reduced field emergence when seeds were produced in a subtropic environment. Field tests of the mutants and their progenies further demonstrated that the lpa mutation in <span class="gene">Gm-lpa-ZC-2</span> does not negatively affect plant <span class="trait">yield</span> traits. These results will advance understanding of the genetic, biochemical and molecular control of PA synthesis in soybean. The novel lpa mutation in <span class="gene">Gm-lpa-ZC-2,</span> together with linked simple sequence repeat (SSR) markers, will be of value for breeding productive lpa soybeans, with meal high in digestible Pi eventually to improve animal nutrition and lessen environmental pollution.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression, purification and analysis of the anti-HIV Cyanovirin-N produced in transgenic soybeans seeds</h2>
			<h3>PMID:未知</h3>
				
					<h3>PMCID:4211091</h3>
					<h4>期刊:BMC Proceedings</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Murad </surname>  <given-names> Andre </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Cunha </surname>  <given-names> Nicolau </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lacorte </surname>  <given-names> Cristiano </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Coelho </surname>  <given-names> Marly </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Vianna </surname>  <given-names> Giovanni </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Rech </surname>  <given-names> Elibio </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The fragment of 306bp corresponding to CVN coding region was amplified by PCR from pET30b-CVN (BioSyn) vector and cloned into bombardment vector pbCong. This vector contains the promoter and the complete signal peptide of the soybean b-conglycinin gene with a CaMV35S terminator. The final vectors pbCongCVN and pAC321 (containing Imazapyr herbicide <span class="trait">resistance</span> gene) were co-bombarded in a 1:1 ratio into the apical meristem of somatic embryonic axes from mature soybean seeds cv. BR-16, utilizing a particle bombardment device [ <xref ref-type="bibr" rid="B4"> 4 </xref> ]. Transgenic R0 plants were cultivated under green-house conditions to produce <span class="trait">seed.</span> R1 seeds were analyzed by PCR and ELISA-gp120 followed by nanoUPLC-MS <sup> E </sup> [ <xref ref-type="bibr" rid="B5"> 5 </xref> ] to characterize and quantify the expression levels of recombinant CVN. Seeds were grinded and proteins were extracted using 50mM Tris-HCl pH 8.0 in 1:20 (w:v) ratio. Total soluble proteins (TSP) were separated using gel filtration sephadex s200 followed by HPLC <span class="gene">C4</span> reversed phase. Fraction containing semi-purified recombinant CVN were used for anti-HIV assay.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase propeptides are functional signal peptides and increase the <span class="trait">yield</span> of a foreign protein.</h2>
			<h3>PMID:20535473  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schnell JA(1), Han S, Miki BL, Johnson DA.
				<br/> 
				年份：2010 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elements that contribute to the high, stable <span class="trait">yield</span> of soybean peroxidase (SBP) in soybean <span class="trait">seed</span> coats can be exploited in the development of this tissue as a protein production platform. <span class="gene">SBP</span> contains an N-terminal and a C-terminal propeptide that are predicted to direct vacuolar targeting; this may be one factor that contributes to its high <span class="trait">yield</span> and stability. We characterized the function of the <span class="gene">SBP</span> propeptides and investigated their ability to increase the <span class="trait">yield</span> of a foreign protein in a heterologous plant system. <span class="gene">SBP</span> propeptides are functional signal peptides capable of directing vacuolar transport in Arabidopsis. The use of these propeptides as well as an endoplasmic reticulum (ER)-retention signal to direct a foreign protein to the apoplast, ER, or vacuole can significantly increase <span class="trait">yield</span> and will therefore be useful for the development of the <span class="trait">seed</span> coat as a protein production platform. We also demonstrate that growth conditions may have a significant impact on the <span class="trait">yield</span> of a foreign protein and that this may be subcellular compartment-specific.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase as an effective bromination catalyst*</h2>
			<h3>PMID:10713205</h3>
				

				
			<p>
				作者：Munir IZ(1), Dordick JS.
				<br/> 
				年份：2000 Mar 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean peroxidase (SBP), an acidic peroxidase isolated from the <span class="trait">seed</span> coat, has been shown to be an effective catalyst for the oxidation of a variety of organic compounds. In the present study, we demonstrate that <span class="gene">SBP</span> can catalyze halogenation reactions. In the presence of H(2)O(2), <span class="gene">SBP</span> catalyzed the oxidation of bromide and iodide but not chloride. Veratryl alcohol (3,4-dimethoxybenzyl alcohol) served as a useful substrate for SBP-catalyzed halogenations yielding the 6-bromo derivative. Halogenation of veratryl alcohol was optimal at pHs below 2.5 with rates of 2.4 µm/min, achieving complete conversions of 150-µm veratryl alcohol in 24 h. The enzyme showed essentially no brominating activity at pHs above 5.5. SBP-catalyzed bromination of veratryl alcohol proceeded with a maximum reaction velocity, (V(max))(apparent), of 5.8 x 10(-1) µm/min, a K(m) of 78 µm and a catalytic efficiency (k(cat)/K(m) of 1.37 x 10(5) M/min at pH 4.0, assuming all of the enzyme's active sites participate in the reaction. <span class="gene">SBP</span> also catalyzed the bromination of several other organic substrates including pyrazole to produce a single product 1-bromopyrazole, indole to <span class="trait">yield</span> both 5-bromoindole and 5-hydroxyindole, and the decarboxylative bromination of 3,4 dimethoxy-trans-cinnamic acid to trans-2-bromo-1-(3,4 dimethoxyphenyl)ethylene. A catalytic mechanism for SBP-catalyzed bromination has been proposed based on experimental results in this and related studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular strategies to improve the nutritional quality of legume proteins.</h2>
			<h3>PMID:10335390  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：de Lumen BO(1), Galvez AF, Revilleza MJ, Krenz DC.
				<br/> 
				年份：1999
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume proteins are relatively deficient in the sulfur amino acids. Among the different strategies to increase the sulfur amino acid contents of legumes, we have chosen to increase the biosynthesis of endogenous, non-abundant sulfur-rich proteins we have identified in soybean <span class="trait">seed.</span> We identified and isolated an 8 kDa sulfur-rich protein from the albumin fraction of soybean <span class="trait">seed.</span> Subsequently, a cDNA designated <span class="gene">Gm2S-1,</span> has been isolated and characterized. The <span class="gene">Gm2S-1</span> cDNA encodes a pre-proprotein that contains a signal peptide and a precursor protein that undergoes post-translational processing to <span class="trait">yield</span> the mature 8 kDa protein containing 7.8% methionine and 7.8% cysteine. The <span class="gene">Gm2S-1</span> gene is an ideal candidate for overexpression to improve the nutritional quality of soybean and other legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of soybean mosaic virus-encoded proteins in <span class="trait">seed</span> and aphid transmission in soybean.</h2>
			<h3>PMID:23927427  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jossey S(1), Hobbs HA, Domier LL.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean mosaic virus (SMV) is <span class="trait">seed</span> and aphid transmitted and can cause significant reductions in <span class="trait">yield</span> and <span class="trait">seed</span> quality in soybean (Glycine max). The roles in <span class="trait">seed</span> and aphid transmission of selected SMV-encoded proteins were investigated by constructing mutants in and chimeric recombinants between SMV 413 (efficiently aphid and <span class="trait">seed</span> transmitted) and an isolate of SMV G2 (not aphid or <span class="trait">seed</span> transmitted). As previously reported, the DAG amino acid sequence motif near the amino terminus of the coat protein (CP) was the major determinant in differences in aphid transmissibility of the two SMV isolates, and helper component proteinase (HC-Pro) played a secondary role. <span class="trait">Seed</span> transmission of SMV was influenced by P1, HC-Pro, and CP. Replacement of the <span class="gene">P1</span> coding region of SMV 413 with that of SMV G2 significantly enhanced <span class="trait">seed</span> transmissibility of SMV 413. Substitution in SMV 413 of the two amino acids that varied in the CPs of the two isolates with those from SMV G2, G to D in the DAG motif and Q to P near the carboxyl terminus, significantly reduced <span class="trait">seed</span> transmission. The Q-to-P substitution in SMV 413 also abolished virus-induced seed-coat mottling in plant introduction 68671. This is the first report associating P1, CP, and the DAG motif with <span class="trait">seed</span> transmission of a potyvirus and suggests that HC-Pro interactions with CP are important for multiple functions in the virus infection cycle.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Effects of plant polysaccharide compound agents on the photosynthetic characteristics and dry matter of soybean].</h2>
			<h3>PMID:23173460  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bai WB(1), Song JQ, Guo JY, Liu XH, Li JH.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field experiment was conducted to study the effects of foliar spraying three compound agents [plant polysaccharides (P1), plant polysaccharides and 5-aminolevulinic acid (P2), and plant polysaccharides and 5-aminolevulinic acid and dimethylpiperidinium chloride (P3)] at the initial flowering stage of soybean on its leaf chlorophyll content, photosynthesis and transpiration, dry matter accumulation and allocation, and grain <span class="trait">yield.</span> Within 35 days after spraying the three compound agents, the leaf chlorophyll content had obvious increase, and its decreasing trend with plant growth had somewhat delay. Compared with the control, spraying <span class="gene">P1</span> and P3 increased the leaf photosynthetic rate and water use efficiency by more than 13.2% and 10.3%, respectively. With the spraying of the three compound agents, the dry matter accumulation in aerial part increased, and the allocation of dry matter from leaf to pod was also enhanced, with the contribution of post-anthesis assimilates to grain <span class="trait">yield</span> increased by more than 17.1%. The 100-grain mass and the pods and seeds per plant increased significantly after spraying <span class="gene">P1</span> and P3, but had no significant increase after spraying P2. The grain <span class="trait">yield</span> of soybean treated with the three compound agents increased by more than 5.9%, compared with the control. This study showed that the three plant polysaccharide compound agents could increase the leaf chlorophyll content, delay the leaf-senescence, improve the leaf photosynthetic capacity and water status, effectively control the dry matter accumulation and post-anthesis assimilates allocation, and increase the grain <span class="trait">yield</span> of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nitrogen redistribution and its relationship with the expression of <span class="gene">GmATG8c</span> during <span class="trait">seed</span> filling in soybean.</h2>
			<h3>PMID:26871505  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Islam MM(1), Ishibashi Y(2), Nakagawa AC(3), Tomita Y(3), Iwaya-Inoue M(2), Arima S(4), Zheng SH(5).
				<br/> 
				年份：2016 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is well known that some nitrogen in the vegetative organs is redistributed to the seeds during <span class="trait">seed</span> filling in soybean (Glycine max [L.] Merrill). This redistribution is considered to affect the <span class="trait">seed</span> <span class="trait">yield</span> of soybean. However, it is still not clear when the nitrogen moves from the vegetative part to the seeds, and the relationship between nitrogen redistribution and leaf senescence has not been clarified. The soybean variety Fukuyutaka was grown in the experimental field of Saga University, Japan from 22 July to 31 October, 2014. After the first flower stage (R1), the plant samples were collected weekly and were separated into leaf, petiole, stem, podshell and <span class="trait">seed.</span> The nitrogen concentrations in each plant part were determined. Fresh leaf samples were provided for the determination of soluble protein and autophagy gene <span class="gene">GmATG8c</span> expression. The nitrogen that accumulated in the vegetative parts reached its highest level at 60days after sowing (DAS), then began to decrease at 73DAS (R6). This decrease is considered to be the consequence of nitrogen redistribution from the vegetative parts to the seeds. The movement of nitrogen from the vegetative parts to the seeds was estimated to occur at around 73DAS (R6). At this stage, leaf SPAD values, leaf nitrogen, and soluble protein concentrations began to decrease simultaneously, suggesting the onset of leaf senescence. Furthermore, the expression of the autophagy gene <span class="gene">GmATG8c</span> in the leaves increased dramatically from 73 to 85DAS, which is the duration of nitrogen redistribution. The results suggest that the nitrogen redistribution from the vegetative parts to the seeds could be one of the initiating factors of leaf senescence, and the autophagy gene <span class="gene">GmATG8c</span> was associated with this process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterologous expression of ATG8c from soybean confers tolerance to nitrogen deficiency and increases <span class="trait">yield</span> in Arabidopsis.</h2>
			<h3>PMID:22629371  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xia T(1), Xiao D, Liu D, Chai W, Gong Q, Wang NN.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen is an essential element for plant growth and <span class="trait">yield.</span> Improving Nitrogen Use Efficiency (NUE) of crops could potentially reduce the application of chemical fertilizer and alleviate environmental damage. To identify new NUE genes is therefore an important task in molecular breeding. Macroautophagy (autophagy) is an intracellular process in which damaged or obsolete cytoplasmic components are encapsulated in double membraned vesicles termed autophagosomes, then delivered to the vacuole for degradation and nutrient recycling. One of the core components of autophagosome formation, ATG8, has been shown to directly mediate autophagosome expansion, and the transcript of which is highly inducible upon starvation. Therefore, we postulated that certain homologs of Saccharomyces cerevisiae ATG8 (ScATG8) from crop species could have potential for NUE crop breeding. A soybean (Glycine max, cv. Zhonghuang-13) ATG8, <span class="gene">GmATG8c,</span> was selected from the 11 family members based on transcript analysis upon nitrogen deprivation. <span class="gene">GmATG8c</span> could partially complement the yeast atg8 mutant. Constitutive expression of <span class="gene">GmATG8c</span> in soybean callus cells not only enhanced nitrogen starvation tolerance of the cells but accelerated the growth of the calli. Transgenic Arabidopsis over-expressing <span class="gene">GmATG8c</span> performed better under extended nitrogen and carbon starvation conditions. Meanwhile, under optimum growth conditions, the transgenic plants grew faster, bolted earlier, produced larger primary and axillary inflorescences, eventually produced more seeds than the wild-type. In average, the <span class="trait">yield</span> was improved by 12.9%. We conclude that <span class="gene">GmATG8c</span> may serve as an excellent candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean <span class="trait">seed</span> set and size using RNA sequencing analysis.</h2>
			<h3>PMID:28087653</h3>
				

				
			<p>
				作者：Du J(1), Wang S(1), He C(1), Zhou B(2), Ruan YL(3), Shou H(1).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To understand the gene expression networks controlling soybean <span class="trait">seed</span> set and size, transcriptome analyses were performed in three early <span class="trait">seed</span> developmental stages, using two genotypes with contrasting <span class="trait">seed</span> size. The two-dimensional data set provides a comprehensive and systems-level view on dynamic gene expression networks underpinning soybean <span class="trait">seed</span> set and subsequent development. Using pairwise comparisons and weighted gene coexpression network analyses, we identified modules of coexpressed genes and hub genes for each module. Of particular importance are the discoveries of specific modules for the large <span class="trait">seed</span> size variety and for <span class="trait">seed</span> developmental stages. A large number of candidate regulators for <span class="trait">seed</span> size, including those involved in hormonal signaling pathways and transcription factors, were transiently and specifically induced in the early developmental stages. The soybean homologs of a brassinosteroid signaling receptor kinase, a brassinosteroid-signaling kinase, were identified as hub genes operating in the <span class="trait">seed</span> coat network in the early <span class="trait">seed</span> maturation stage. Overexpression of a candidate <span class="trait">seed</span> size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased <span class="trait">seed</span> size and <span class="trait">seed</span> weight. Together, these analyses identified a large number of potential key regulators controlling soybean <span class="trait">seed</span> set, <span class="trait">seed</span> size, and, consequently, <span class="trait">yield</span> potential, thereby providing new insights into the molecular networks underlying soybean <span class="trait">seed</span> development.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution and association analysis of <span class="gene">GmCYP78A10</span> gene with <span class="trait">seed</span> size/weight and pod number in soybean.</h2>
			<h3>PMID:25324172  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Zhang H, Sun G, Zhang W, Qiu L.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed-size/weight traits, controlled by multiple genes in soybean, play an important role in determining <span class="trait">seed</span> <span class="trait">yield.</span> However, the molecular mechanisms controlling the <span class="trait">seed</span> size and weight in soybean remain unclear. In Arabidopsis, P450/CYP78A gene family has been proved extremely relevant to <span class="trait">seed</span> size (such as AtCYP78A5, AtCYP78A6 and AtCYP78A9). We found that a soybean <span class="gene">GmCYP78A10</span> gene underwent artificial selection during soybean breeding. The <span class="gene">GmCYP78A10a</span> allele mainly distributed in wild soybean (Glycine soja), but has been eliminated in the cultivars during early stage of soybean breeding, while the <span class="gene">GmCYP78A10b</span> allele has been accumulated and become the predominant allele in cultivated soybean (G. max). ANOVA analysis showed that the mean <span class="trait">seed</span> weight, <span class="trait">seed</span> width and <span class="trait">seed</span> thickness of soybean varieties with <span class="gene">GmCYP78A10b</span> allele was significantly heavier/bigger than those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01). The allele could explain 7.2 % variation in <span class="trait">seed</span> weight. The pod number of the soybeans with <span class="gene">GmCYP78A10b</span> allele significantly decreased compared to those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01, R(2) = 5.8 %), while other agronomic traits including <span class="trait">seed</span> weight/plant were not significantly affected by these two alleles. We speculated that during the early stage of soybean breeding, breeders selected big <span class="trait">seed</span> carrying <span class="gene">GmCYP78A10b</span> allele, but lowered pod number simultaneously. Overall, the selection did not cause the significantly change in soybean <span class="trait">seed</span> <span class="trait">yield.</span> Our results suggests that the soybean <span class="gene">GmCYP78A10</span> gene may have a similar function to those genes belonging to P450/CYP78A subfamily in Arabidopsis and provides new information for the genetic control of <span class="trait">seed</span> size in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue-Specific Regulation of <span class="gene">Gma-miR396</span> Family on Coordinating Development and Low Water Availability Responses.</h2>
			<h3>PMID:28694817</h3>
				

				
			<p>
				作者：Liu W(1), Zhou Y(1), Li X(1), Wang X(1), Dong Y(1), Wang N(1), Liu X(1), Chen H(1), Yao N(1), Cui X(1), Jameel A(1), Wang F(1), Li H(1).
				<br/> 
				年份：2017 Jun 26
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previously, it was reported that miR396s interact with growth-regulating factors (GRFs) to modulate plant growth, development, and stress <span class="trait">resistance.</span> In soybean, 11 gma-miR396 precursors (Pre-miR396a-k) were found, and 24 <span class="gene">GmGRFs</span> were predicted as targets of seven mature gma-miR396s (gma-miR396a/b/c/e/h/i/k). To explore the roles of the miR396-GRF module in low water availability response of soybean, we analyzed the expression of Pre-miR396a-k, and found that Pre-miR396a/i/bdgk/e/h were up-regulated in leaves and down-regulated in roots; on the contrary, <span class="gene">GmGRF5/6/7/8/15/17/21</span> were down-regulated in leaves and <span class="gene">GmGRF1/2/17/18/19/20/21/22/23/24</span> were up-regulated in roots of low water potential stressed soybean. Any one of gma-miR396a/b/c/e/h/i/k was able to interact with 20 <span class="gene">GmGRFs</span> <span class="gene">(GmGRF1/2/6-11/13-24),</span> confirming that this module represents a multi-to-multi network interaction. We generated Arabidopsis plants over-expressing each of the 11 gma-miR396 precursors (Pre-miR396a-k), and seven of them (miR396a/b/c/e/h/i/k-OE transgenic Arabidopsis) showed altered development. The low water availability of miR396a/b/c/e/h/i/k-OE was enhanced in leaves but reduced in seeds and roots. Contrary to previous reports, miR396a/b/c/i-OE seedlings showed lower survival rate than WT when recovering after rewatering under soil drying. In general, we believe our findings are valuable to understand the role of gma-miR396 family in coordinating development and low water availability responses, and can provide potential strategies and directions for soybean breeding programs to improve <span class="trait">seed</span> <span class="trait">yield</span> and plant <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Prediction of novel miRNAs and associated target genes in <italic> Glycine max </italic></h2>
			<h3>PMID:20122185</h3>
				
					<h3>PMCID:3009485</h3>
					<h4>期刊:BMC Bioinformatics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> joshitr@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Yan </surname>  <given-names> Zhe </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yanzh@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Libault </surname>  <given-names> Marc </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> libaultm@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Jeong </surname>  <given-names> Dong-Hoon </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> jeong@dbi.udel.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Park </surname>  <given-names> Sunhee </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> spark@dbi.udel.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Green </surname>  <given-names> Pamela J </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> green@dbi.udel.edu </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Sherrier </surname>  <given-names> D Janine </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> sherrier@dbi.udel.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Farmer </surname>  <given-names> Andrew </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> adf@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> May </surname>  <given-names> Greg </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> gdm@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Meyers </surname>  <given-names> Blake C </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> meyers@dbi.udel.edu </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> xudong@missouri.edu </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A12">  <name>  <surname> Stacey </surname>  <given-names> Gary </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> staceyg@missouri.edu </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Illumina's SBS sequencing technology was utilized to generate high-quality reads from four different soybean tissues, including root, <span class="trait">seed,</span> flower, and nodule. The <span class="gene">Gmax1.01</span> release version genomic sequences and gene model predictions of Williams 82 soybean genome were acquired from Phytozome [ <xref ref-type="bibr" rid="B15"> 15 </xref> ] and used as a reference genome.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic architecture of cyst nematode <span class="trait">resistance</span> revealed by genome-wide association study in soybean</h2>
			<h3>PMID:26263897</h3>
				
					<h3>PMCID:4533770</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" equal-contrib="yes">  <name>  <surname> Vuong </surname>  <given-names> T. D. </given-names>  </name>  <address>  <email> vuongt@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Sonah </surname>  <given-names> H. </given-names>  </name>  <address>  <email> sonahh@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meinhardt </surname>  <given-names> C. G. </given-names>  </name>  <address>  <email> meinhardtc@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> R. </given-names>  </name>  <address>  <email> deshmukhr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kadam </surname>  <given-names> S. </given-names>  </name>  <address>  <email> kadamsu@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> R. L. </given-names>  </name>  <address>  <email> rlnelson@illinois.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. G. </given-names>  </name>  <address>  <email> shannong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> H. T. </given-names>  </name>  <address>  <email> nguyenhr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Division of Plant Sciences and National Center for Soybean Biotechnology (NCSB), University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Soybean Maize Germplasm, Pathology, and Genetics Research Unit, USDA, Agricultural Research Service, and Department of Crop Sciences University of Illinois, Urbana, IL 61801 USA  </aff>  <aff id="Aff3">  <label/> Division of Plant Sciences and NCSB, University of Missouri, Portageville, MO 63873 USA  </aff>  <aff id="Aff4">  <label/> Present address: D&#x000e9;partement de Phytologie, Facult&#x000e9; des Sciences de l&#x02019;Agriculture et de l&#x02019;Alimentation, Centre de Recherche en Horticulture, Universit&#x000e9; Laval, Quebec, Canada  </aff>
				<br/> 
				年份：<day> 3 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Over 45,000 single nucleotide polymorphism (SNP) markers generated by the SoySNP50K iSelect BeadChip ( <italic> http// </italic> <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> <italic> www.soybase.org </italic> </ext-link> ) were utilized for analysis. GWAS identified 14 loci distributed over different chromosomes comprising 60 SNPs significantly associated with SCN <span class="trait">resistance.</span> Results also confirmed six QTL that were previously mapped using bi-parental populations, including the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> loci. GWAS identified eight novel QTL, including QTL on chromosome 10, which we have previously mapped by using a bi-parental population. In addition to the known loci for four simple traits, such as <span class="trait">seed</span> coat color, flower color, pubescence color, and stem growth habit, two traits, like <span class="trait">lodging</span> and pod shattering, having moderately complex inheritance have been confirmed with great precision by GWAS.
					</p>
				
					<p>&nbsp;&nbsp;
						Early studies of the inheritance of <span class="trait">resistance</span> to SCN indicated that SCN <span class="trait">resistance</span> was genetically controlled by different recessive or dominant genes, designated as <italic> rhg1 </italic> , <italic> <span class="gene">rhg2</span> </italic> , <italic> rhg3 </italic> [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ], <italic> <span class="gene">Rhg4</span> </italic> [ <xref ref-type="bibr" rid="CR5"> 5 </xref> ], and <italic> Rhg5 </italic> [ <xref ref-type="bibr" rid="CR6"> 6 </xref> ]. However, further genetic studies of new <span class="trait">resistance</span> sources have showed that SCN <span class="trait">resistance</span> was a complex trait genetically controlled by quantitative trait loci (QTL) [ <xref ref-type="bibr" rid="CR7"> 7 </xref> , <xref ref-type="bibr" rid="CR8"> 8 </xref> ]. In a comprehensive review, Concibido et al. [ <xref ref-type="bibr" rid="CR9"> 9 </xref> ] has summarized 31 putative QTL associated with <span class="trait">resistance</span> to various SCN HG types, which were mapped to 17 of the 20 soybean chromosomes. With new <span class="trait">resistance</span> sources, many efforts have been made to identify novel QTL, which were mapped on new loci [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> ]. In addition to the identification of new QTL, genetic analysis also confirmed many QTL previously reported [ <xref ref-type="bibr" rid="CR11"> 11 </xref> , <xref ref-type="bibr" rid="CR12"> 12 </xref> ]. Recently, two major genomic loci, <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , which have been identified and consistently mapped on chromosomes (Chrs.) 18 and 8, respectively, were successfully cloned [ <xref ref-type="bibr" rid="CR13"> 13 </xref> , <xref ref-type="bibr" rid="CR14"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						GWAS was performed using generalized linear model (GLM) identified 223 SNPs distributed over 19 different chromosomes and associated with <span class="trait">resistance</span> to SCN HG Type 0 (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6a </xref> ). These SNPs represent a minimum allele frequency (MAF) ranging from 0.05 to 0.45 and with a highest <italic> p </italic> value of 1.7E-9.0. A Q-Q plot representing expected and observed probability of getting association of SNPs with a phenotype showed possibility of high number of false positive associations (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6b </xref> ). Therefore, a mixed linear model (MLM), which is one of the most effective methods for controlling false positives in GWAS, was used for further analysis. The efficient mixed-model association (EMMA) model was used in the analysis to correct for confounding effects due to subpopulation structure and relatedness between individuals. The MLM identified 41 SNPs distributed over 16 loci on 14 different chromosomes that were significantly associated with <span class="trait">resistance</span> to SCN HG Type 0 (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> , Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ). The genomic region on Chr. 10 showed a higher peak level of significance ( <italic> p </italic> -value&#x02009;=&#x02009;3.29E-07, 2.42E-06) comprising two SNPs. The most significant SNP on Chr. 10 showed 51&#x000a0;% phenotypic variation, significantly higher than the variation (47&#x000a0;%) estimated without the SNP (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). On Chr. 7, two loci at 36.5&#x000a0;Mb and 43.0&#x000a0;Mb comprising five and four SNPs, respectively, were found to be associated with SCN <span class="trait">resistance.</span> The most significant SNP at these loci contributed 50&#x000a0;% phenotypic variation. Interestingly, the known loci <italic> rhg1 </italic> on Chr. 18 and <italic> <span class="gene">Rhg4</span> </italic> on Chr. 8, were also identified in this study. Highly significant SNPs on Chr. 8 between 7.5 to 8.6&#x000a0;Mb and on Chr. 18 between 1.2 to 6.6&#x000a0;Mb were associated with SCN <span class="trait">resistance.</span> These loci did not show a high level of significance even though these loci harbor a very high level of <span class="trait">resistance.</span> Since the number of genotypes carrying the <span class="trait">resistance</span> allele for either of these genes was very few which affects the level of significance in GWAS. <fig id="Fig6"> <label> Fig. 6 </label> <caption> <p> Significantly associated single nucleotide polymorphism (SNP) with SCN <span class="trait">resistance</span> in soybean identified by genome-wide association study (GWAS). <bold> a </bold> Manhattan plot; <bold> b </bold> Q-Q plot using generalized linear model (GLM)
					</p>
				
					<p>&nbsp;&nbsp;
						Annotation information of the soybean genome sequence assembly suggested 2,352 genes at the 16 GWAS loci identified in the present study. Functional categorization of genes based on gene ontology showed the highest number of genes involved in transcription factor/DNA binding activity followed by catalytic activity in the molecular function category (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Figure S1). Furthermore, information of significantly over-represented gene ontology (GO) categories was used for sorting the list as per priority. A total of 158 <span class="trait">resistance</span> gene analogs (RGA) and disease <span class="trait">resistance</span> genes were identified (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> : Table S2). Out of these, only 106 genes were observed to be expressed in RNA-seq data available for 14 different soybean tissues. Analysis of microarray experimental data available for SCN ( <italic> H. glycine </italic> ) infected cells captured using laser micro-dissection (E-MEXP-876) and infected root tissue (E-MEXP-808) showed differential expression for most of the candidate genes [ <xref ref-type="bibr" rid="CR53"> 53 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ] (Additional file <xref rid="MOESM4" ref-type="media"> 4 </xref> : Table S3). The GWAS loci identified on Chr. 8 and Chr. 18 showed a presence of the previously known and well characterized <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Besides the recent identification of two major genes <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , very little has been known about the <span class="trait">resistance</span> mechanism involved in SCN <span class="trait">resistance</span> and genetic variation that exists in the soybean germplasm. The host-pathogen interaction is very complex involving multiple genes, which trigger the molecular signaling and subsequent responses. The identification of loci governing <span class="trait">resistance</span> not only helps the genetic improvement of cultivars but also facilitates the identification of genes and the understanding of molecular mechanisms involved in the <span class="trait">resistance</span> process. The 16 GWAS loci for SCN <span class="trait">resistance</span> identified in the present study provided the molecular basis to understand the variable genetic responses observed in soybean germplasm. Apart from the conventional QTL mapping, which was typically based on a segregation of <span class="trait">resistance</span> in narrow genetic backgrounds, GWAS captures simultaneously the vast genetic variation existing in soybean germplasm. Most of the GWAS loci confirmed the previously identified QTL for SCN <span class="trait">resistance</span> (Additional file <xref rid="MOESM6" ref-type="media"> 6 </xref> : Table S4). Of these, a novel QTL recently identified on Chr. 10 [ <xref ref-type="bibr" rid="CR4"> 4 </xref> ] was also observed with this GWAS. We believe that these are the major 16 loci which define the genetic architecture of SCN <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, we report the identification and confirmation of QTL significantly associated with <span class="trait">resistance</span> to SCN HG Type 0 (race 3) in a diverse panel of 553 soybean germplasm accessions in maturity groups from III to V. It included the known QTL, such as the <italic> rhg1 </italic> and <italic> <span class="gene">Rhg4</span> </italic> , and also novel QTL, such as Chr. 10-QTL, which was recently reported. GWAS using the appropriate analysis model enabled us to identify several SNP markers significantly associated with QTL. The availability and accessibility of the reference soybean genome sequence and gene annotation also facilitated the identification of candidate genes, leading to the functionality analysis. The results showed that GWAS can be employed as an effective strategy for identifying complex traits in soybean and for narrowing GWAS-defined genomic regions, which facilitates positional cloning of the causal gene(s).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Performance of hybrid progeny formed between genetically modified herbicide-tolerant soybean and its wild ancestor</h2>
			<h3>PMID:26507568</h3>
				
					<h3>PMCID:4670487</h3>
					<h4>期刊:AoB Plants</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Zheng-Jun </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="aff" rid="af2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Peng-Fei </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mi </surname>  <given-names> Xiang-Cheng </given-names>  </name>  <xref ref-type="aff" rid="af1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kang </surname>  <given-names> Ding-Ming </given-names>  </name>  <xref ref-type="aff" rid="af3"> 3 </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="af4"> 4 </xref>  </contrib>  <aff id="af1">  <label> 1 </label>  <addr-line> State Key Laboratory of Vegetation and Environmental Change </addr-line> ,  <institution> Institute of Botany, Chinese Academy of Sciences </institution> ,  <addr-line> Beijing 100093 </addr-line> ,  <country> China </country>  </aff>  <aff id="af2">  <label> 2 </label>  <addr-line> Department of Life Sciences </addr-line> ,  <institution> Yuncheng University </institution> ,  <addr-line> Yuncheng, Shanxi 044000 </addr-line> ,  <country> China </country>  </aff>  <aff id="af3">  <label> 3 </label>  <addr-line> College of Agriculture and Biotechnology </addr-line> ,  <institution> China Agricultural University </institution> ,  <addr-line> Beijing 100193 </addr-line> ,  <country> China </country>  </aff>  <aff id="af4">  <label> 4 </label>  <addr-line> Nanjing Institute of Environmental Sciences </addr-line> ,  <institution> Ministry of Environmental Protection of China </institution> ,  <addr-line> Nanjing, Jiangsu 210042 </addr-line> ,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 11 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						At the three-to-four leaf stage, only one <span class="gene">GM-F</span> <sub> 2 </sub> plant carrying the EPSPS gene or one wild soybean plant was retained at each spot, according to PCR identification. Due to unexpected dry climate conditions and/or potential <span class="trait">seed</span> dormancy, especially in wild soybean, which possesses a hard <span class="trait">seed</span> coat ( <xref rid="PLV121C31" ref-type="bibr"> Sun <italic> et al </italic> . 2015 </xref> ), the five plant types were unequally represented in the field samples. We harvested 9 wild soybean Miyun plants, 14 wild soybean Pinggu plants, 28 GM F <sub> 2 </sub> -Miyun plants, 38 GM F <sub> 2 </sub> -Pinggu plants and 41 non-GM F <sub> 2 </sub> -Miyun plants (Table&#x000a0; <xref ref-type="table" rid="PLV121TB1"> 1 </xref> ). The seeds harvested from F <sub> 2 </sub> plants in the field were considered as F <sub> 3 </sub> seeds.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of <span class="trait">seed</span> <span class="trait">yield</span> and agronomic characters by photoperiod sensitivity and growth habit genes in soybean.</h2>
			<h3>PMID:20012856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cober ER(1), Morrison MJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes are adapted to narrow bands of latitude due to photoperiod sensitivity. There are several photoperiod-sensitive loci (E1, E2, E3, E4, E5, E6, E7, E8). Determinate and indeterminate growth habits are controlled by a single locus. The objective of our research was to examine the effects of photoperiod sensitivity and growth habit alleles on <span class="trait">seed</span> <span class="trait">yield</span> and other agronomic characters using isogenic lines. Twenty 'Harosoy' isolines with 11 photoperiod-sensitive genotypes many with both indeterminate and determinate growth habits were grown in the field at Ottawa, ON, from 2003 to 2007. Maturity ranged from 97 to 127 days, and <span class="trait">seed</span> <span class="trait">yield</span> increased linearly with maturity until about 112 days when it plateaued. Determinate lines were always shorter than indeterminate lines of equivalent maturity. <span class="trait">Seed</span> <span class="trait">yield</span> was associated with plant height, maturity, <span class="trait">seed</span> sugar concentration, <span class="trait">seed</span> weight and <span class="trait">lodging.</span> Effects of alleles at individual loci, and additive and epistatic effects across multiple loci were examined. At a single locus, photoperiod-insensitive alleles produced isolines that matured 8-11 days earlier, yielded less, and had shorter plants with reduced <span class="trait">lodging.</span> In multiple loci analyses, additive effects explained most of the variation in agronomic characters since additive models with E1, E3, E4, E7 and <span class="gene">Dt1</span> loci included compared well to additive plus epistatic models and genotype-based models. Variation in photoperiod sensitivity and growth habit alleles results in a range of maturity, with pleiotropic effects on <span class="trait">seed</span> <span class="trait">yield</span> and agronomic characteristics, and play an important role in providing adaptation across latitudes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL affecting fitness of hybrids between wild and cultivated soybeans in experimental fields</h2>
			<h3>PMID:23919159</h3>
				
					<h3>PMCID:3728954</h3>
					<h4>期刊:Ecology and Evolution</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kuroda </surname>  <given-names> Yosuke </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kaga </surname>  <given-names> Akito </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tomooka </surname>  <given-names> Norihiko </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yano </surname>  <given-names> Hiroshi </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takada </surname>  <given-names> Yoshitake </given-names>  </name>  <xref ref-type="aff" rid="au2"> 2 </xref>  <xref ref-type="aff" rid="au3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kato </surname>  <given-names> Shin </given-names>  </name>  <xref ref-type="aff" rid="au3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vaughan </surname>  <given-names> Duncan </given-names>  </name>  <xref ref-type="aff" rid="au1"> 1 </xref>  </contrib>  <aff id="au1">  <label> 1 </label>  <institution> National Institute of Agrobiological Sciences </institution>  <addr-line> 2-1-2 Kannondai, Tsukuba, Ibaraki, 305-8602, Japan </addr-line>  </aff>  <aff id="au2">  <label> 2 </label>  <institution> Western Region Agricultural Research Center, NARO </institution>  <addr-line> 6-12-1 Nishifukatsu, Fukuyama, Hiroshima, 721-8514, Japan </addr-line>  </aff>  <aff id="au3">  <label> 3 </label>  <institution> Tohoku Agricultural Research Center, NARO </institution>  <addr-line> 297 Uenodai Kariwano, Daisen, Akita, 019-2112, Japan </addr-line>  </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To confirm the effect of <italic> G. max </italic> genes in a predominantly <italic> G. soja </italic> background, backcross (BC) populations were developed for both <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 combinations using <italic> G. soja </italic> as the recurrent parent (Table <xref ref-type="table" rid="tbl1"> 1 </xref> , Fig. <xref ref-type="fig" rid="fig01"> 1 </xref> ). Two BC <sub> 1 </sub> F <sub> 1 </sub> populations (W1 &#x000d7; D1: 68 individuals; <span class="gene">W2</span> &#x000d7; D2: 160 individuals) were obtained from crossing each F <sub> 1 </sub> hybrid (donor plant) to the corresponding <italic> G. soja </italic> accession (recurrent parent). The success of the crossing was confirmed using 60 simple sequence repeat (SSR) markers composed of three markers per linkage group for all 20 linkage groups. Furthermore, two BC <sub> 2 </sub> F <sub> 1 </sub> populations (W1 &#x000d7; D1: 60 individuals; <span class="gene">W2</span> &#x000d7; D2: 40 individuals) were developed by crossing selected BC <sub> 1 </sub> F <sub> 1 </sub> plants (one plant per population) to <italic> G. soja </italic> . The selection of the BC <sub> 1 </sub> F <sub> 1 </sub> plants, having major <italic> G. max </italic> QTLs for <span class="trait">seed</span> dormancy and total <span class="trait">seed</span> number identified in F <sub> 2 </sub> populations, was based on the genotypes of the BC <sub> 1 </sub> F <sub> 1 </sub> populations. To investigate the fitness of the populations after an additional generation of self-pollination, the seeds obtained from self-pollination of the selected BC <sub> 1 </sub> F <sub> 1 </sub> plants were used to develop two BC <sub> 1 </sub> F <sub> 2 </sub> populations (W1 &#x000d7; D1: 150 individuals; <span class="gene">W2</span> &#x000d7; D2: 150 individuals).
					</p>
				
					<p>&nbsp;&nbsp;
						Among 204 F <sub> 2 </sub> plants in the <span class="gene">W1</span> &#x000d7; D1 population, 104 F <sub> 2 </sub> plants, together with the 10 parents and several F <sub> 1 </sub> hybrid (Table <xref ref-type="table" rid="tbl1"> 1 </xref> ), were grown at 1 m &#x000d7; 1 m spacing at the Tohoku Agricultural Research Center (39.5&#x000b0;N, 140.4&#x000b0;E, Akita Prefecture, northern Japan, <xref ref-type="supplementary-material" rid="SD1"> Appendices A1 and A2 </xref> ), hereafter referred to as the &#x0201c;north field.&#x0201d; The other 100 F <sub> 2 </sub> plants, together with the parents and F <sub> 1 </sub> hybrid, were grown at the same density at the Western Region Agricultural Research Center (34.5&#x000b0;N, 133.4&#x000b0;E, Hiroshima prefecture, southern Japan, <xref ref-type="supplementary-material" rid="SD1"> Appendices A1 and A2 </xref> ), hereafter referred to as the &#x0201c;south field.&#x0201d; The <span class="gene">W2</span> &#x000d7; D2 population, composed of 204 F <sub> 2 </sub> plants, was grown in the north and south fields in the same manner as the <span class="gene">W1</span> &#x000d7; D1 population (Table <xref ref-type="table" rid="tbl1"> 1 </xref> ). As maintenance and evaluation of many climbing plants with a higher <italic> G. soja </italic> background are difficult, backcross populations (BC <sub> 1 </sub> F <sub> 1 </sub> , BC <sub> 2 </sub> F <sub> 1 </sub> , and BC <sub> 1 </sub> F <sub> 2 </sub> ) were only grown at 1 m &#x000d7; 1 m density at the National Institute of Agrobiological Sciences (36.0&#x000b0;N, 140.1&#x000b0;E, Ibaraki Prefecture, central Japan, <xref ref-type="supplementary-material" rid="SD1"> Appendices A1 and A2 </xref> ), hereafter referred to as the &#x02018;central field&#x02019;. <span class="trait">Seed</span> coats were scratched with a razor blade and germinated in a small pot at the beginning of July in 2005, at the middle of June in 2006, and at the end of May for in 2007. The seedlings were transplanted to the field in the middle of July every year. Three stakes with a net strung between the stakes per plant were used to guide twining stems. During October to November, mature pods with seeds were harvested by hand twice a week. Standard agricultural practices such as applications of fertilizer (650 kg/ha of 3 parts nitrogen, 10 parts phosphate, and 10 parts potassium; 1000 kg/ha of fused magnesium phosphate; 1000 kg/ha of limestone), weeding, insecticides to control stink bug and common cutworm, were conducted.
					</p>
				
					<p>&nbsp;&nbsp;
						Of a total of 11 fitness-related traits (Table <xref ref-type="table" rid="tbl2"> 2 </xref> ), 10 were treated as quantitative traits and one <span class="trait">(seed</span> coat color) was treated as a qualitative trait. Two <span class="trait">seed</span> dormancy&#x02013;related traits, namely <span class="trait">seed</span> winter survival (DORM_1) and <span class="trait">seed</span> hardness (DORM_2), were evaluated using the seeds from individual plants. As germination of all the hard seeds from randomly selected lines were confirmed by the mechanical abrasion on the wetted filter paper or soil at room temperature, hard seeds were treated as viable nongerminated seeds. <span class="trait">Seed</span> production&#x02013;related traits, namely total <span class="trait">seed</span> number (PROD_1), <span class="trait">seed</span> total weight (PROD_2), 100-seed weight (PROD_3), total pod number (PROD_4), stem dry weight (PROD_5), and stem length (PROD_6) were evaluated for each plant (Table <xref ref-type="table" rid="tbl2"> 2 </xref> ). Those traits were recorded on a per plant basis after the seeds of each plant had matured. As flowering of the two southern accessions (W2 and D2) as well as F <sub> 1 </sub> plants and most of the <span class="gene">W2</span> &#x000d7; D2 F <sub> 2 </sub> plants were late flowering in the north field, those whole plants (7 of 10 W2, all 5 D2, all 5 of the <span class="gene">W2</span> &#x000d7; D2 F <sub> 1 </sub> , and 73 of 104 <span class="gene">W2</span> &#x000d7; D2 F <sub> 2 </sub> plants) were taken from the field before the first snowfall and dried in the greenhouse to obtain mature seeds. The methods of trait evaluation in the backcrossing populations were the same as for the F <sub> 2 </sub> populations. The number of days from sowing to first flowering was recorded as FLOW. The total number of seeds expected to germinate in the following year (SURV) was estimated as PROD_1 multiplied by DORM_1. The mean and standard deviation for each trait and the correlation coefficient between each pair of traits were calculated. Differences in mean values between <italic> G. soja </italic> , <italic> G. max </italic> , and F <sub> 1 </sub> hybrids were analyzed separately for each field location in each year with the Mann&#x02013;Whitney <italic> U </italic> -test or Kruskal&#x02013;Wallis test. The median and range instead of mean and standard deviation are reported for segregating populations. All statistical analyses were conducted using R version 2.9.2 (R Development Core Team <xref ref-type="bibr" rid="b58"> 2009 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Total DNA of each putative F <sub> 1 </sub> <span class="trait">seed</span> was extracted from a small piece of cotyledon tissue using an EZ1 DNA Tissue kit (Qiagen, Tokyo, Japan). Total DNA of F <sub> 2, </sub> BC <sub> 1 </sub> F <sub> 1, </sub> BC <sub> 2 </sub> F <sub> 1 </sub> , and BC <sub> 1 </sub> F <sub> 2 </sub> individuals was extracted from 100 mg of fresh leaf tissue. DNA concentration was adjusted between 5 and 25 ng/&#x003bc;L by comparing with known concentrations of standard &#x003bb; DNA on a 1.5% agarose gel. A total of 720 SSR markers from SoyBase ( <ext-link ext-link-type="uri" xlink:href="http://soybase.org/"> http://soybase.org/ </ext-link> ) were screened to detect polymorphisms between the parents. Five markers were also included to track the three classical soybean loci, <italic> I </italic> , <italic> T </italic> , and <italic> <span class="gene">Dt1</span> </italic> ( <xref ref-type="supplementary-material" rid="SD1"> Appendix A3 </xref> ). Three markers, <italic> dCHS1 </italic> (Matsumura et al. <xref ref-type="bibr" rid="b47"> 2005 </xref> ), <italic> AY262686B </italic> , and <italic> AY262686Z, </italic> were used to track the <italic> I </italic> locus, which controls <span class="trait">seed</span> coat color and might be related to <span class="trait">seed</span> winter survival. A single-base indel marker <italic> sF3&#x02032;H1 </italic> reported by Toda et al. ( <xref ref-type="bibr" rid="b70"> 2002 </xref> ) was used to detect the <italic> T </italic> locus, a locus that controls pubescence color and interacts with the <italic> I </italic> locus. A SSR marker <italic> LFsoy3 </italic> was designed to track the <italic> <span class="gene">Dt1</span> </italic> locus, which might be related to stem length and <span class="trait">seed</span> total number. These markers were amplified by using KOD-plus polymerase (Toyobo, Osaka, Japan), based on the manufacturer's guide, in a GeneAmp 9700 PCR system (Applied Biosystems, Tokyo, Japan). Polymorphisms were scored by using banding patterns in 12% polyacrylamide gel.
					</p>
				
					<p>&nbsp;&nbsp;
						Because <italic> G. soja </italic> had higher PROD_1 and DORM_1 than <italic> G. max </italic> , SURV of <italic> G. soja </italic> was higher than that of <italic> G. max </italic> in both the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 combinations in all three backcross generations (Table <xref ref-type="table" rid="tbl3"> 3 </xref> , Fig. <xref ref-type="fig" rid="fig02"> 2 </xref> C&#x02013;H). The medians of SURV in the BC <sub> 1 </sub> F <sub> 1 </sub> and BC <sub> 2 </sub> F <sub> 1 </sub> generations were very close to <italic> G. soja </italic> ; still, there was variation in both DORM_1 and PROD_1 in the BC <sub> 1 </sub> F <sub> 1 </sub> and BC <sub> 2 </sub> F <sub> 1 </sub> generations (Fig. <xref ref-type="fig" rid="fig02"> 2 </xref> C&#x02013;H). Some individuals had the potential to <span class="trait">yield</span> large numbers of dormant <span class="trait">seed</span> because the number of seeds (PROD_1) was greater than <italic> G. soja </italic> and the <span class="trait">seed</span> dormancy (DORM_1) was similar.
					</p>
				
					<p>&nbsp;&nbsp;
						In total, 28 and 27 QTLs related to <span class="trait">seed</span> dormancy, <span class="trait">seed</span> production, and flowering phenology were detected in the F <sub> 2 </sub> generation of <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations, respectively (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). Among them, QTLs in three regions (LG-A2, -C2, and -D1b) had large effects on <span class="trait">seed</span> dormancy and QTL in one region (LG-L) had a significant effect on <span class="trait">seed</span> production.
					</p>
				
					<p>&nbsp;&nbsp;
						Eight and 6 QTLs associated with <span class="trait">seed</span> dormancy were detected in the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations, respectively (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). The <italic> G. max </italic> alleles at all of those QTLs had additive effects (Add.) of decreasing DORM_1 (Add, &#x02013;3 to &#x02013;37%; PVE, 6.4&#x02013;76.2%) and DORM_2 (Add, &#x02013;1 to &#x02013;26%; PVE, 6.2&#x02013;42.7%). Three major QTLs, which were located on LG-A2, -C2, and -D1b, were associated with DORM_1 and DORM_2 in both populations (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A). The <italic> G. max </italic> allele at the QTL on LG-A2 had larger additive effects in the <span class="gene">W2</span> &#x000d7; D2 population than in the <span class="gene">W1</span> &#x000d7; D1 population. The QTL on LG-A2 was located near the <italic> I </italic> locus, and the QTL on LG-C2 was close to the <italic> T </italic> locus. The additive effect of the QTL on LG-A2 detected in seeds harvested from the south field tended to be higher than that for seeds harvested from the north field. In contrast, the additive effect of the QTL on LG-C2 and LG-D1b detected in seeds harvested from the south field tended to be lower than that for seeds harvested from the north field.
					</p>
				
					<p>&nbsp;&nbsp;
						Sixteen and 17 QTLs related to <span class="trait">seed</span> production were detected in <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations, respectively (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). Most <italic> G. max </italic> alleles at those QTLs had additive effects of decreasing PROD_1 (Add, &#x02013;73 to &#x02013;496 seeds; PVE, 8.8&#x02013;33.8%), PROD_2 (Add, &#x02013;5.1 to &#x02013;42.5 g; PVE, 6.8&#x02013;33.5%), PROD_4 (Add, &#x02013;74 to &#x02013;229 pods; PVE, 7.5&#x02013;33%), PROD_5 (Add, &#x02013;6.7 to &#x02013;18.6 g; PVE, 10.2&#x02013;28.8%), and PROD_6 (Add, &#x02013;4 to &#x02013;50 cm; PVE, 11.5&#x02013;40.2%) and increasing PROD_3 (Add, +0.6 to 1.1 g; PVE, 7.8&#x02013;28.2%).
					</p>
				
					<p>&nbsp;&nbsp;
						QTLs with large effect on <span class="trait">seed</span> production&#x02013;related traits such as PROD_1, PROD_2, PROD_3, PROD_4, PROD_5, and PROD_6 were located near a marker <italic> LFsoy3 </italic> in both the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> B). The <italic> G. max </italic> alleles at those QTLs, except for PROD_3, had additive effects of decreasing the phenotypic values for those traits, but the magnitude of effect differed depending on the test location ( <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). For both populations, the additive effects for PROD_1, PROD_2, and PROD_5 were greater than those in the north field. Although the effects of QTLs for PROD_4 would be expected to be consistent, they did not seem to be related in the populations. The frequency of pods with only one or two seeds on plants in the north field was greater than for plants in the south field (data not shown), which may explain this inconsistency.
					</p>
				
					<p>&nbsp;&nbsp;
						Three QTLs for <span class="trait">seed</span> dormancy were detected in each population (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A, <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). Although the QTLs for DORM_2 on LG-A2 were detected only in the <span class="gene">W2</span> &#x000d7; D2 population, the QTLs for DORM_1 on LG-A2 were detected across combinations (W1 &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2), suggesting that the <italic> G. max </italic> alleles have a consistent genetic effect even within a high percentage of wild genetic background. The <italic> G. max </italic> allele at this QTL on LG-A2 had a large negative effect on DORM_1 (Add,&#x02013;7% to &#x02013;8%; PVE, 21.2&#x02013;22.3%).
					</p>
				
					<p>&nbsp;&nbsp;
						Three and 15 QTLs related to <span class="trait">seed</span> production were detected in the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 BC <sub> 1 </sub> F <sub> 1 </sub> populations, respectively (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). No QTL was common between the F <sub> 2 </sub> and BC <sub> 1 </sub> F <sub> 1 </sub> generations. For traits PROD_1, PROD_2, PROD_4, and PROD_5, no QTLs were detected in the <span class="gene">W1</span> &#x000d7; D1 population, but several QTLs different from those found in the F <sub> 2 </sub> generation were identified in the <span class="gene">W2</span> &#x000d7; D2 population. As in the F <sub> 2 </sub> generation, most <italic> G. max </italic> alleles had the effect of decreasing <span class="trait">seed</span> production&#x02013;related traits, that is, at QTLs for PROD_5 (Add, &#x02013;8.6 to &#x02013;11.6 g; PVE, 5.1&#x02013;9.4%) and PROD_6 (Add, &#x02013;21 to &#x02013;62 cm; PVE, 7.5&#x02013;20.2%). However, some <italic> G. max </italic> alleles at those QTLs from the <span class="gene">W2</span> &#x000d7; D2 population had the effect of increasing PROD_1 on LG-A2 (Add, +354 seeds; PVE, 5.5%), PROD_2 on LG-A2 (Add, +17.9 g; PVE, 7.4%), PROD_4 on LG-E (Add, +113 pods; PVE, 5.1%) and on LG-M (Add, +112 pods; PVE, 4.9%), and PROD_6 on LG-A2 and LG-O (Add, +20 to +21 cm; PVE, 6.9&#x02013;7.9%). The <italic> G. max </italic> alleles at all QTLs detected had the effect of increasing PROD_3.
					</p>
				
					<p>&nbsp;&nbsp;
						In the BC <sub> 1 </sub> F <sub> 2 </sub> generation, which had a similar percentage of <italic> G. soja </italic> background to the BC <sub> 1 </sub> F <sub> 1 </sub> generation but was homozygous for selected fitness-related alleles from <italic> G. max </italic> , 19 and 17 QTLs were detected in the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations, respectively (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). The major QTLs for <span class="trait">seed</span> dormancy on LG-A2, C2, and D1b (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A) and for <span class="trait">seed</span> number on LG-L (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> B) were well conserved between the F <sub> 2 </sub> and BC <sub> 1 </sub> F <sub> 2 </sub> generation, except for the DORM_1 QTL on LG-A2 in the <span class="gene">W1</span> &#x000d7; D1 population, which was present in the F <sub> 2 </sub> but not detected in the BC <sub> 1 </sub> F <sub> 2 </sub> generation.
					</p>
				
					<p>&nbsp;&nbsp;
						Seedling emergence represents the interface between two demographic events: <span class="trait">seed</span> production and seedling recruitment. Because <span class="trait">seed</span> dormancy&#x02013;related traits determine the timing of seedling emergence, the physiology of <span class="trait">seed</span> dormancy has a large effect on fitness. Good water permeability is an important trait for uniform and rapid germination in <italic> G. max </italic> cultivation and food processing. Conversely, rapid water uptake is known to lead to cell damage in the cotyledon (Powell and Matthews <xref ref-type="bibr" rid="b56"> 1978 </xref> ) and is disadvantageous to survival of <italic> G. soja </italic> during winter in natural habitats. The physiological difference has been characterized by many researchers who have measured traits such as <span class="trait">seed</span> water imbibition or <span class="trait">seed</span> hardness during several days under germinable conditions. However, evaluation of <span class="trait">seed</span> dormancy is generally quite different between artificial and natural conditions in terms of time, water, and temperature conditions. Even <italic> G. max </italic> <span class="trait">seed,</span> which imbibes water during winter, could survive winter in 2006 (Table <xref ref-type="table" rid="tbl3"> 3 </xref> ), indicating that water imbibition does not always lead to loss of <span class="trait">seed</span> viability. In this study, three major QTLs affecting both DORM_1 and DORM_2, which are located on LG-A2, -C2, and -D1b (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A), were generally consistent over generations and crossing combinations. A significant high correlation between DORM_1 and DORM_2 was observed ( <xref ref-type="supplementary-material" rid="SD1"> Appendix A4 </xref> ): seeds from hybrid derivatives that had <italic> G. max </italic> alleles at those QTLs imbibed water easily and appear to have rotted in the soil over the winter. In particular, the <italic> G. max </italic> allele for DORM_1 on LG-A2 was found to be partially dominant to the <italic> G. soja </italic> allele because its effect appeared in BC <sub> 1 </sub> F <sub> 1 </sub> progenies and it had a large effect of reducing survival rate in the <span class="gene">W2</span> &#x000d7; D2 population ( <xref ref-type="supplementary-material" rid="SD1"> Appendix A5B </xref> ). Therefore, the effect of such strong <italic> G. max </italic> alleles may lead to reduced winter survival of the seeds produced by an F <sub> 1 </sub> hybrid plant as well as by later-generation progenies.
					</p>
				
					<p>&nbsp;&nbsp;
						Nevertheless, the magnitudes of allele effects at the three major DORM_1/DORM_2 QTLs were slightly different depending on the cross combination. For example, the effect of the QTL on LG-A2 was strongest among the three QTLs in the <span class="gene">W2</span> &#x000d7; D2 population, whereas it was similar to that of the other two QTLs in the <span class="gene">W1</span> &#x000d7; D1 population (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A). This explains the different level of <span class="trait">seed</span> winter survival between the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 combinations. All the previously reported QTLs had the effect of causing water imbibition when the alleles at those loci were from <italic> G. max </italic> . In a <italic> G. max </italic> &#x000d7; <italic> G. soja </italic> population, Keim et al. ( <xref ref-type="bibr" rid="b32"> 1990 </xref> ) detected four QTLs on LG-A2, -L, and -D1b by evaluating imbibition of F <sub> 4 </sub> seeds for 7 days at room temperature. In contrast, Sakamoto et al. ( <xref ref-type="bibr" rid="b63"> 2004 </xref> ) and Liu et al. ( <xref ref-type="bibr" rid="b44"> 2007 </xref> ) identified two QTLs, located in LG-C2 and -D1b, by evaluating imbibition of seeds for 12 h and 24 h at room temperature, respectively. <italic> Glycine gracilis </italic> is an intermediate form between <italic> G. max </italic> and <italic> G. soja </italic> that originated in northeastern China (Hymowitz <xref ref-type="bibr" rid="b26"> 2004 </xref> ). Three QTLs (on LG-C2, -D1b, and -I) were identified in a <italic> G. max </italic> &#x000d7; <italic> G. gracilis </italic> population by testing imbibition of seeds for 24 h at 25&#x000b0;C (Watanabe et al. <xref ref-type="bibr" rid="b74"> 2004 </xref> ). These results indicate that QTLs on LG-C2 and -D1b are common among <italic> G. max </italic> &#x000d7; <italic> G. soja </italic> populations, but that a QTL on LG-A2 is not consistently detected in such populations. Similarly, in this study, no QTL for <span class="trait">seed</span> hardness (DORM_2) was detected on LG-A2 in the <span class="gene">W1</span> &#x000d7; D1 population (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A). It is very interesting that QTLs for <span class="trait">seed</span> winter survival (DORM_1), which required a long-term evaluation in the field, were successfully identified in the <span class="gene">W1</span> &#x000d7; D1 combination in approximately the same region on LG-A2 where QTLs for DORM_1 and DORM_2 were detected in the <span class="gene">W2</span> &#x000d7; D2 combination. One possible explanation of this finding is that the effect of a QTL on LG-A2 may appear when seeds imbibe water during long-term evaluation if the <span class="trait">seed</span> coat of <italic> G. max </italic> has <span class="trait">resistance</span> to water imbibition. The slow imbibition rate seen for D1 parent also supports this explanation and suggests that there is allelic variation within <italic> G. max </italic> for a <span class="trait">seed</span> hardness QTL on LG-A2.
					</p>
				
					<p>&nbsp;&nbsp;
						The genes for domestication-related traits, which differentiate between crops and their wild relatives, are not randomly distributed across crop genomes (Ross-Ibarra <xref ref-type="bibr" rid="b61"> 2005 </xref> ; Kaga et al. <xref ref-type="bibr" rid="b30"> 2008 </xref> ). In this study, QTLs with high contributions to <span class="trait">seed</span> production&#x02013;related traits, representing distinct differences between <italic> G. soja </italic> and <italic> G. max </italic> , tended to be concentrated in a particular genomic region on LG-L (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> B). Those QTLs were common between different cross combinations (W1 &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2) as well as across different generations. One possible reason for the positive, high correlation of total number of <span class="trait">seed</span> (PROD_1) with traits related to plant size such as stem dry weight (PROD_5) and stem length (PROD_6 [ <xref ref-type="supplementary-material" rid="SD1"> Appendix A4 </xref> ]) would be a gene related to stem elongation. Classically, stem termination in soybean is known to be controlled by two loci, <italic> Dt </italic> 1 and <italic> Dt </italic> 2 (Bernard <xref ref-type="bibr" rid="b3"> 1972 </xref> ). The determinate stem type ( <italic> dt </italic> 1 allele) shows little growth in stem length after flowering, whereas the indeterminate stem type ( <italic> Dt </italic> 1 allele) continues to elongate even after flowering. An intermediate phenotype, called semideterminate, is conditioned by the <italic> Dt </italic> 2 locus (Bernard <xref ref-type="bibr" rid="b3"> 1972 </xref> ). Because <italic> Dt </italic> 1 and <italic> Dt </italic> 2 have been mapped on LG-L and LG-G, respectively (Cregan et al. <xref ref-type="bibr" rid="b10"> 1999 </xref> ), the QTL with a strong contribution to stem length (PROD_6) on LG-L in this study is likely to be the <italic> Dt </italic> 1 locus (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> B). Our results indicate that the <italic> G. max </italic> allele at this locus has the effect of reducing the number of seeds produced by hybrids between <italic> G. max </italic> and <italic> G. soja </italic> , as previously reported by Wang et al. ( <xref ref-type="bibr" rid="b72"> 2004 </xref> ). Intriguingly, QTLs for <span class="trait">seed</span> weight (PROD_3) as well as other <span class="trait">seed</span> production&#x02013;related traits were closely linked to marker <italic> LFsoy </italic> 3, which was designed to detect a soybean homolog of <italic> PsTFL1a </italic> , a gene-controlling stem termination in <italic> Pisum </italic> (Foucher et al. <xref ref-type="bibr" rid="b15"> 2003 </xref> ). Further studies are necessary to clarify the pleiotropic effect of soybean <italic> TFL1a </italic> on these traits. The <italic> G. max </italic> allele at the QTL for PROD_6 on LG-L was confirmed to have a moderate negative effect in the BC <sub> 1 </sub> F <sub> 1 </sub> and BC <sub> 2 </sub> F <sub> 1 </sub> populations, but it had no effect on PROD_1 as was found in the progenies from self-pollination (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> B, <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ). A QTL for both PROD_6 and PROD_1 was identified again on LG-L in the BC <sub> 1 </sub> F <sub> 2 </sub> population. These results indicate that the <italic> G. max </italic> allele is recessive to the <italic> G. soja </italic> allele because its effects were detected only in progenies generated by self-pollination.
					</p>
				
					<p>&nbsp;&nbsp;
						Photosensitivity is also an important plant response that is heavily involved in the control of flowering as well as in successful <span class="trait">seed</span> production. There were clear differences between the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations in terms of both days to first flower (FLOW) (Table <xref ref-type="table" rid="tbl3"> 3 </xref> ). The <span class="gene">W1</span> &#x000d7; D1 population, representing northern Japanese germplasm, had shorter FLOW than the <span class="gene">W2</span> &#x000d7; D2 population, representing southern Japanese germplasm. This difference reflects the adaptive strategy of <italic> G. soja </italic> and <italic> G. max </italic> in Japan. In northern Japan, the growing season is relatively short; thus, the <span class="gene">W1</span> &#x000d7; D1 population might respond to warm temperatures and start to produce seeds during the short period of moderate climate even if the plants are not large. In contrast, the <span class="gene">W2</span> &#x000d7; D2 population might respond to photoperiod and start to produce seeds only after the plants have grown large because autumn is relatively long in southern Japan.
					</p>
				
					<p>&nbsp;&nbsp;
						Until recently, little has been known about the effect of <italic> G. max </italic> alleles within a predominantly <italic> G. soja </italic> genetic background. In this study, the genetic effects of those <italic> G. max </italic> alleles were not expressed as phenotypes in the BC <sub> 1 </sub> F <sub> 1 </sub> and BC <sub> 2 </sub> F <sub> 1 </sub> generations, indicating that most <italic> G. soja </italic> alleles are dominant to <italic> G. max </italic> alleles; one notable exception was the QTL for <span class="trait">seed</span> dormancy on LG-A2 (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> A). Snow et al. ( <xref ref-type="bibr" rid="b66"> 1999 </xref> ) indicated that after two or three generation of backcrossing, hybrid derivatives in which crop alleles have been introgressed can be just as competitive and successful as wild plants. In this study, PROD_1 and DORM_1 in the BC <sub> 1 </sub> F <sub> 1 </sub> and BC <sub> 2 </sub> F <sub> 1 </sub> generation approached the values for <italic> G. soja </italic> as the proportion of <italic> G. soja </italic> genetic background increased (Table <xref ref-type="table" rid="tbl3"> 3 </xref> ). Although QTLs at which <italic> G. max </italic> alleles had the increasing effect on PROD_1 and DORM_1 were not consistent over generations and crossing combinations (Fig. <xref ref-type="fig" rid="fig03"> 3 </xref> , <xref ref-type="supplementary-material" rid="SD1"> Appendix A5 </xref> ), these alleles may have the potential to increase the fitness of hybrid derivatives. Individual plants that had higher fitness than <italic> G. soja </italic> in terms of SURV could be found in most generations of both the <span class="gene">W1</span> &#x000d7; D1 and <span class="gene">W2</span> &#x000d7; D2 populations (Fig. <xref ref-type="fig" rid="fig02"> 2 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmFtsH9 expression correlates with in vivo photosystem II function: chlorophyll a fluorescence transient analysis and eQTL mapping in soybean.</h2>
			<h3>PMID:21638036  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin Z(1), Meng F, Song H, Wang X, Chao M, Zhang G, Xu X, Deng D, Yu D.
				<br/> 
				年份：2011 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Filamentation temperature-sensitive H (FtsH) is an ATP-dependent zinc metalloprotease involved in diverse biological functions. There are 12 FtsH proteins in Arabidopsis, among which AtFtsH2 plays an important role in regulating the turnover of photosystem II (PSII) reaction center D1 protein and the development of the photosynthetic apparatus. Here, we have identified 11 FtsH genes in the soybean genome by a bioinformatics approach. These soybean FtsH genes corresponded to seven Arabidopsis FtsH genes, suggesting that the main characteristics of soybean FtsH genes were formed before the evolutionary split of soybean and Arabidopsis. Phylogenetic analyses allowed us to clone a soybean AtFtsH2-like gene designated as <span class="gene">GmFtsH9.</span> The predicted protein of <span class="gene">GmFtsH9</span> consists of 690 amino acids and contains three typical FtsH proteins conserved domains. The expression level of <span class="gene">GmFtsH9</span> was determined in a soybean recombinant inbred line population under a pot experiment conducted for measuring chlorophyll a fluorescence transient parameters, photosynthetic CO(2) fixation rate (P (N)), and <span class="trait">seed</span> <span class="trait">yield.</span> Expression quantitative trait loci (eQTL) mapping revealed two trans-acting eQTLs for <span class="gene">GmFtsH9.</span> The significant correlation of gene expression level with chlorophyll a fluorescence transient parameters and the presence of overlapping eQTL (QTL) between gene expression level and chlorophyll a fluorescence transient parameters indicated that <span class="gene">GmFtsH9</span> could be involved in regulating PSII function. These results further lead to the understanding of the mechanism underlying FtsH gene expression, and contribute to the development of marker-assisted selection breeding programs for modulating soybean FtsH gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being <span class="trait">seed</span> oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Variation in Rubisco activase (RCAβ) gene promoters and expression in soybean [Glycine max (L.) Merr].</h2>
			<h3>PMID:24170743  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chao M(1), Yin Z, Hao D, Zhang J, Song H, Ning A, Xu X, Yu D.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Understanding the genetic basis of Rubisco activase (RCA) gene regulation and altering its expression levels to optimize Rubisco activation may provide an approach to enhance plant productivity. However, the genetic mechanisms and the effect of RCA expression on phenotype are still unknown in soybean. This work analysed the expression of RCA genes and demonstrated that two RCA isoforms presented different expression patterns. Compared with <span class="gene">GmRCAα,</span> <span class="gene">GmRCAβ</span> was expressed at higher mRNA and protein levels. In addition, <span class="gene">GmRCAα</span> and <span class="gene">GmRCAβ</span> were positively correlated with chlorophyll fluorescence parameters and <span class="trait">seed</span> <span class="trait">yield,</span> suggesting that changes in expression of RCA has a potential applicability in breeding for enhanced soybean productivity. To identify the genetic factors that cause expression level variation of <span class="gene">GmRCAβ,</span> expression quantitative trait loci (eQTL) mapping was combined with allele mining in a natural population including 219 landraces. The eQTL mapping showed that a combination of both cis- and trans-acting eQTLs might control <span class="gene">GmRCAβ</span> expression. As promoters can affect both cis- and trans-acting eQTLs by altering cis-acting regulatory elements or transcription factor binding sites, this work subsequently focused on the promoter region of <span class="gene">GmRCAβ.</span> Single-nucleotide polymorphisms in the <span class="gene">GmRCAβ</span> promoter were identified and shown to correlate with expression level diversity. These SNPs were classified into two groups, A and B. Further transient expression showed that GUS expression driven by the group A promoter was stronger than that by the group B promoter, suggesting that promoter sequence types could influence gene expression levels. These results would improve understanding how variation within promoters affects gene expression and, ultimately, phenotypic diversity in natural populations.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotyping-by-Sequencing-Based Investigation of the Genetic Architecture Responsible for a ∼Sevenfold Increase in Soybean <span class="trait">Seed</span> Stearic Acid.</h2>
			<h3>PMID:27866151  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heim CB(1), Gillman JD(2)(3).
				<br/> 
				年份：2017 Jan 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean oil is highly unsaturated but oxidatively unstable, rendering it nonideal for food applications. Until recently, the majority of soybean oil underwent partial chemical hydrogenation, which produces trans fats as an unavoidable consequence. Dietary intake of trans fats and most saturated fats are conclusively linked to negative impacts on cholesterol levels and cardiovascular health. Two major soybean oil breeding targets are: (1) to reduce or eliminate the need for chemical hydrogenation, and (2) to replace the functional properties of partially hydrogenated soybean oil. One potential solution is the elevation of <span class="trait">seed</span> stearic acid, a saturated fat which has no negative impacts on cardiovascular health, from 3 to 4% in typical cultivars to &gt; 20% of the <span class="trait">seed</span> oil. We performed QTL analysis of a population developed by crossing two mutant lines, one with a missense mutation affecting a stearoyl-acyl-carrier protein desaturase gene resulting in ∼11% <span class="trait">seed</span> stearic acid crossed to another mutant, A6, which has 24-28% <span class="trait">seed</span> stearic acid. Genotyping-by-sequencing (GBS)-based QTL mapping identified 21 minor and major effect QTL for six <span class="trait">seed</span> oil related traits and plant height. The inheritance of a large genomic deletion affecting chromosome 14 is the basis for largest effect QTL, resulting in ∼18% <span class="trait">seed</span> stearic acid. This deletion contains <span class="gene">SACPD-C</span> and another gene(s); loss of both genes boosts <span class="trait">seed</span> stearic acid levels to ≥ 18%. Unfortunately, this genomic deletion has been shown in previous studies to be inextricably correlated with reduced <span class="trait">seed</span> <span class="trait">yield.</span> Our results will help inform and guide ongoing breeding efforts to improve soybean oil oxidative stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Deletions of the <span class="gene">SACPD-C</span> locus elevate <span class="trait">seed</span> stearic acid levels but also result in fatty acid and morphological alterations in nitrogen fixing nodules.</h2>
			<h3>PMID:24886084  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gillman JD(1), Stacey MG, Cui Y, Berg HR, Stacey G.
				<br/> 
				年份：2014 May 27
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max) seeds are the primary source of edible oil in the United States. Despite its widespread utility, soybean oil is oxidatively unstable. Until recently, the majority of soybean oil underwent chemical hydrogenation, a process which also generates trans fats. An alternative to chemical hydrogenation is genetic modification of <span class="trait">seed</span> oil through identification and introgression of mutant alleles. One target for improvement is the elevation of a saturated fat with no negative cardiovascular impacts, stearic acid, which typically constitutes a minute portion of <span class="trait">seed</span> oil (~3%). RESULTS: We examined radiation induced soybean mutants with moderately increased stearic acid (10-15% of <span class="trait">seed</span> oil, ~3-5 X the levels in wild-type soybean seeds) via comparative whole genome hybridization and genetic analysis. The deletion of one <span class="gene">SACPD</span> isoform encoding gene (SACPD-C) was perfectly correlated with moderate elevation of <span class="trait">seed</span> stearic acid content. However, <span class="gene">SACPD-C</span> deletion lines were also found to have altered nodule fatty acid composition and grossly altered morphology. Despite these defects, overall nodule accumulation and nitrogen fixation were unaffected, at least under laboratory conditions. CONCLUSIONS: Although no <span class="trait">yield</span> penalty has been reported for moderate elevated <span class="trait">seed</span> stearic acid content in soybean seeds, our results demonstrate that genetic alteration of <span class="trait">seed</span> traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and <span class="gene">SACPD</span> activity in particular, in the establishment and maintenance of symbiotic nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome-wide transcript profiling to identify differentially expressed genes associated with <span class="trait">seed</span> field emergence in two soybean low phytate mutants.</h2>
			<h3>PMID:28100173  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan F(1), Yu X(1), Dong D(1), Yang Q(1), Fu X(1), Zhu S(1), Zhu D(2).
				<br/> 
				年份：2017 Jan 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Seed</span> germination is important to soybean (Glycine max) growth and development, ultimately affecting soybean <span class="trait">yield.</span> A lower <span class="trait">seed</span> field emergence has been the main hindrance for breeding soybeans low in phytate. Although this reduction could be overcome by additional breeding and selection, the mechanisms of <span class="trait">seed</span> germination in different low phytate mutants remain unknown. In this study, we performed a comparative transcript analysis of two low phytate soybean mutants (TW-1 and TW-1-M), which have the same mutation, a 2 bp deletion in <span class="gene">GmMIPS1,</span> but show a significant difference in <span class="trait">seed</span> field emergence, TW-1-M was higher than that of TW-1 . RESULTS: Numerous genes analyzed by RNA-Seq showed markedly different expression levels between TW-1-M and TW-1 mutants. Approximately 30,000-35,000 read-mapped genes and ~21000-25000 expressed genes were identified for each library. There were ~3900-9200 differentially expressed genes (DEGs) in each contrast library, the number of up-regulated genes was similar with down-regulated genes in the mutant TW-1and TW-1-M. Gene ontology functional categories of DEGs indicated that the ethylene-mediated signaling pathway, the abscisic acid-mediated signaling pathway, response to hormone, ethylene biosynthetic process, ethylene metabolic process, regulation of hormone levels, and oxidation-reduction process, regulation of flavonoid biosynthetic process and regulation of abscisic acid-activated signaling pathway had high correlations with <span class="trait">seed</span> germination. In total, 2457 DEGs involved in the above functional categories were identified. Twenty-two genes with 20 biological functions were the most highly up/down- regulated (absolute value Log2FC &gt;5) in the high field emergence mutant TW-1-M and were related to metabolic or signaling pathways. Fifty-seven genes with 36 biological functions had the greatest expression abundance (FRPM &gt;100) in germination-related pathways. CONCLUSIONS: <span class="trait">Seed</span> germination in the soybean low phytate mutants is a very complex process, which involves a series of physiological, morphological and transcriptional changes. Compared with TW-1, TW-1-M had a very different gene expression profile, which included genes related to plant hormones, antioxidation, anti-stress and energy metabolism processes. Our research provides a molecular basis for understanding germination mechanisms, and is also an important resource for the genetic analysis of germination in low phytate crops. Plant hormone- and antioxidation-related genes might strongly contribute to the high germination rate in the TW-1-M mutant.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Peanut gene expression profiling in developing seeds at different reproduction stages during <italic> Aspergillus parasiticus </italic> infection</h2>
			<h3>PMID:18248674</h3>
				
					<h3>PMCID:2257936</h3>
					<h4>期刊:BMC Developmental Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Baozhu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> baozhu.guo@ars.usda.gov </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Xiaoping </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xpchen@uga.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Dang </surname>  <given-names> Phat </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> phat.dang@ars.usda.gov </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Scully </surname>  <given-names> Brian T </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> brian.scully@ars.usda.gov </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Liang </surname>  <given-names> Xuanqiang </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> liang804@yahoo.com </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Holbrook </surname>  <given-names> C Corley </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> corley.holbrook@ars.usda.gov </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Jiujiang </given-names>  </name>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> jiuyu@srrc.ars.usda.gov </email>  </contrib>  <contrib id="A8" contrib-type="author">  <name>  <surname> Culbreath </surname>  <given-names> Albert K </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> spotwilt@uga.edu </email>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 2 </month>  <year> 2008 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We constructed six different cDNA libraries derived from developing peanut seeds at three reproduction stages (R5, R6 and R7) from a resistant and a susceptible cultivated peanut genotypes, 'Tifrunner' (susceptible to <italic> Aspergillus </italic> infection with higher aflatoxin contamination and resistant to TSWV) and 'GT-C20' (resistant to <italic> Aspergillus </italic> with reduced aflatoxin contamination and susceptible to TSWV). The developing peanut <span class="trait">seed</span> tissues were challenged by <italic> A. parasiticus </italic> and <span class="trait">drought</span> stress in the field. A total of 24,192 randomly selected cDNA clones from six libraries were sequenced. After removing vector sequences and quality trimming, 21,777 high-quality EST sequences were generated. Sequence clustering and assembling resulted in 8,689 unique EST sequences with 1,741 tentative consensus EST sequences (TCs) and 6,948 singleton ESTs. Functional classification was performed according to <span class="gene">MIPS</span> functional catalogue criteria. The unique EST sequences were divided into twenty-two categories. A similarity search against the non-redundant protein database available from NCBI indicated that 84.78% of total ESTs showed significant similarity to known proteins, of which 165 genes had been previously reported in peanuts. There were differences in overall expression patterns in different libraries and genotypes. A number of sequences were expressed throughout all of the libraries, representing constitutive expressed sequences. In order to identify resistance-related genes with significantly differential expression, a statistical analysis to estimate the relative abundance ( <italic> R </italic> ) was used to compare the relative abundance of each gene transcripts in each cDNA library. Thirty six and forty seven unique EST sequences with threshold of <italic> R </italic> &#x0003e; 4 from libraries of 'GT-C20' and 'Tifrunner', respectively, were selected for examination of temporal gene expression patterns according to EST frequencies. Nine and eight resistance-related genes with significant up-regulation were obtained in 'GT-C20' and 'Tifrunner' libraries, respectively. Among them, three genes were common in both genotypes. Furthermore, a comparison of our EST sequences with other plant sequences in the TIGR Gene Indices libraries showed that the percentage of peanut EST matched to <italic> Arabidopsis thaliana </italic> , maize ( <italic> Zea mays </italic> ), <italic> Medicago truncatula </italic> , rapeseed ( <italic> Brassica napus </italic> ), rice ( <italic> Oryza sativa </italic> ), soybean ( <italic> Glycine max </italic> ) and wheat ( <italic> Triticum aestivum </italic> ) ESTs ranged from 33.84% to 79.46% with the sequence identity &#x02265; 80%. These results revealed that peanut ESTs are more closely related to legume species than to cereal crops, and more homologous to dicot than to monocot plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						To understand the molecular basis of host <span class="trait">resistance</span> <italic> to A. flavus/parasiticus </italic> and consequent aflatoxin contamination, we monitored the transcript changes at these three developmental stages in developing seeds. The 8,689 unique ESTs were categorized into different functional groups based on the <span class="gene">MIPS</span> criteria [ <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ]. The highly expressed overlapping ESTs also helped in assembling full-length unique transcripts expressed in peanut <span class="trait">seed,</span> such as the putative allergen protein (iso-Ara h3, GenBank accession no. <ext-link ext-link-type="gen" xlink:href="DQ855115"> DQ855115 </ext-link> ). The putative functions of those identified unique ESTs have been predicted by similarity search according to <span class="gene">MIPS</span> (Fig. <xref ref-type="fig" rid="F4"> 4 </xref> ). Comparing to the <italic> Arabidopsis </italic> sequence data, 65.99% of total peanut unique ESTs matched <italic> Arabidopsis </italic> protein sequences with a known function and 17.58% had significant similarity to <italic> Arabidopsis </italic> protein sequences with unknown function. About 16.43% of the total unique ESTs showed no significant similarity to <italic> Arabidopsis </italic> al all. Those peanut ESTs matched <italic> Arabidopsis </italic> know functions were divided into nineteen categories [ <xref ref-type="bibr" rid="B29"> 29 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> ]. A major portion of these genes with known functions fall in the category of metabolism (24.47%) followed by transcription (8.85%, Fig. <xref ref-type="fig" rid="F4"> 4 </xref> ). To further identify novel peanut sequences, a comprehensive similarity search against GenBank non-redudant (nr) database using the stand-alone BLASTx algorithm was performed and resulted in the identification of an additional 967 putative novel sequences including 165 unique peanut ESTs matching reported known peanut genes. The BLAST result revealed that significant number of unique peanut <span class="trait">seed</span> ESTs match soybean (396), <italic> Arabidopsis </italic> (2952), rice (682), and other plant species.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, some previously reported defense-related genes have been confirmed to be expressed. Desiccation-related proteins could be induced by <span class="trait">drought</span> stress and were relatively sensitive to cellular dehydration [ <xref ref-type="bibr" rid="B35"> 35 </xref> , <xref ref-type="bibr" rid="B36"> 36 </xref> ]. The LEA (late embryogenesis abundant) proteins are known to be involved in protecting higher plants from damage caused by environmental stresses, especially dehydration from <span class="trait">drought</span> [ <xref ref-type="bibr" rid="B37"> 37 </xref> - <xref ref-type="bibr" rid="B39"> 39 </xref> ]. Serine protease inhibitors are involved in plant defense against pathogens and could be induced in response to infection by pathogens [ <xref ref-type="bibr" rid="B40"> 40 </xref> - <xref ref-type="bibr" rid="B42"> 42 </xref> ]. These three different classes of genes were up-regulated in the three reproduction stages of both 'GT-C20' and 'Tifrunner' libraries. Other related-genes with significant differential expression were present either in 'GT-C20' or in 'Tifrunner'. For example, the PR10 protein family is induced by plants in response to pathogen infection as well as abiotic stress, and showed transcriptional up-regulation upon biotic and abiotic stresses [ <xref ref-type="bibr" rid="B43"> 43 </xref> - <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Calmodulin (CaM) is a ubiquitous Ca <sup> 2+ </sup> sensor found in all eukaryotes and has been shown to participate in the regulation of diverse calcium-dependent physiological processes [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. Calmodulin plays an important role in sensing and transducing changes in cellular Ca <sup> 2+ </sup> concentration in response to several biotic and abiotic stresses [ <xref ref-type="bibr" rid="B47"> 47 </xref> ]. <span class="gene">CaM</span> has been implicated in plant-pathogen interactions [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> ]. PR10 and Calmodulin were significantly up-regulated in 'GT-C20' libraries but not in 'Tifrunner' (Table <xref ref-type="table" rid="T6"> 6 </xref> ). In contrast, two heat shock proteins, synthesized in response to heat stress [ <xref ref-type="bibr" rid="B50"> 50 </xref> - <xref ref-type="bibr" rid="B52"> 52 </xref> ], were detected up-regulated in 'Tifrunner' libraries but not in 'GT-C20' (Table <xref ref-type="table" rid="T6"> 6 </xref> ). This raises questions of why certain genes are present or absent or show differential expression in different genotypes, such as 'GT-C20' and 'Tifrunner'. There are two possible hypothetic explanations. One is that in this study we randomly selected clones for cDNA sequencing and might have missed some clones that could be in 'GT-C20' or 'Tifrunner' libraries. The other is that the presence, absence or significantly differential expressions of certain genes, especially defense-related genes, are a result of the genetic differences <span class="trait">(resistance</span> and susceptibility) of these two genotypes. In order to verify the assumption that variability of expression might be a result of genetic differences in disease <span class="trait">resistance</span> or stresses tolerance, two genes (an allergen protein <italic> iso ara h3 </italic> , highly abundant and a constitutively expressed genes, and an LEA 4, a up-regulated and defense-related gene) were selected for sequence similarity analysis. As expected, the similarity of <italic> iso ara h3 </italic> between 'GT-C20' and 'Tifrunner' was 97%, however, <italic> LEA 4 </italic> sequences shared only 91% identity over 709 bases. For <italic> iso ara h3 </italic> , among 1,692 consensus sequences, 6 gaps were found. For <italic> LEA 4 </italic> , among 709 consensus sequences, 19 gaps were found (data not shown). The results implied that the allelic differences of defense-related genes were higher than that of constitutively expressed genes. Further investigations are necessary to characterize their gene functions and to analyze the patterns of their gene expressions.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a <span class="gene">GmGBP1</span> ortholog of soybean enhances the responses to flowering, stem elongation and heat tolerance in transgenic tobaccos.</h2>
			<h3>PMID:23636865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao L(1), Wang Z, Lu Q, Wang P, Li Y, Lv Q, Song X, Li D, Gu Y, Liu L, Li W.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a typical short-day crop, and its photoperiodic and gibberellin (GA) responses for the control of flowering are critical to <span class="trait">seed</span> <span class="trait">yield.</span> The <span class="gene">GmGBP1</span> mRNA abundance in leaves was dramatically increased in short-days (SDs) compared to that in long-days in which it was consistently low at all time points from 0 to 6 days (days after transfer to SDs). <span class="gene">GmGBP1</span> was highly expressed in leaves and exhibited a circadian rhythm in SDs. Ectopic overexpression of <span class="gene">GmGBP1</span> in tobaccos caused photoperiod-insensitive early flowering by increasing NtCO mRNA levels. <span class="gene">GmGBP1</span> mRNA abundance was also increased by GAs. Transgenic <span class="gene">GmGBP1</span> overexpressing (-ox) tobacco plants exhibited increased GA signaling-related phenotypes including flowering and plant height promotion. Furthermore, the hypocotyl elongation, early-flowering and longer internode phenotypes were largely accelerated by GA3 application in the <span class="gene">GmGBP1-ox</span> tobacco seedlings. Being consistent, overexpression of <span class="gene">GmGBP1</span> resulted in significantly enhanced GA signaling (evidenced suppressed expression of NtGA20ox) both with and without GA treatments. <span class="gene">GmGBP1</span> was a positive regulator of both photoperiod and GA-mediated flowering responses. In addition, <span class="gene">GmGBP1-ox</span> tobaccos were hypersensitive to ABA, salt and osmotic stresses during <span class="trait">seed</span> germination. Heat-inducible <span class="gene">GmGBP1</span> also enhanced thermotolerance in transgenic <span class="gene">GmGBP1-ox</span> tobaccos during <span class="trait">seed</span> germination and growth. <span class="gene">GmGBP1</span> protein was localized in the nucleus. Analyses of a series of 5'-deletions of the <span class="gene">GmGBP1</span> promoter suggested that several cis-acting elements, including P-BOX, TCA-motif and three HSE elements necessary to induce gene expression by GA, salicic acid and heat stress, were specifically localized in the <span class="gene">GmGBP1</span> promoter region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome Resilience and Prevalence of Segmental Duplications Following Fast Neutron Irradiation of Soybean</h2>
			<h3>PMID:25213171</h3>
				
					<h3>PMCID:4224183</h3>
					<h4>期刊:Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bolon </surname>  <given-names> Yung-Tsi </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="author-notes" rid="afn2">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stec </surname>  <given-names> Adrian O. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Michno </surname>  <given-names> Jean-Michel </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Roessler </surname>  <given-names> Jeffrey </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhaskar </surname>  <given-names> Pudota B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="author-notes" rid="afn3">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ries </surname>  <given-names> Landon </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="author-notes" rid="afn4">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dobbels </surname>  <given-names> Austin A. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Campbell </surname>  <given-names> Benjamin W. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Young </surname>  <given-names> Nathan P. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> Justin E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grant </surname>  <given-names> David M. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Orf </surname>  <given-names> James H. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Naeve </surname>  <given-names> Seth L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Muehlbauer </surname>  <given-names> Gary J. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vance </surname>  <given-names> Carroll P. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff4">  <sup> &#x000a7; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Stupar </surname>  <given-names> Robert M. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 4 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108  </aff>  <aff id="aff2">  <label> &#x02020; </label> Corn Insects and Crop Genetics Research Unit, United States Department of Agriculture-Agricultural Research Service, Ames, Iowa 50011  </aff>  <aff id="aff3">  <label> &#x02021; </label> Department of Plant Biology, University of Minnesota, St. Paul, Minnesota 55108  </aff>  <aff id="aff4">  <label> &#x000a7; </label> Plant Science Research Unit, United States Department of Agriculture-Agricultural Research Service, St. Paul, Minnesota 55108 </aff>
				<br/> 
				年份：<day> 02 </day>  <month> 9 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean FN mutant FN0172932 was crossed to lines &#x02018;Anoka&#x02019; (PI 548508) and &#x02018;Minsoy&#x02019; (PI 548389) and backcrossed to cv. &#x02018;M92-220&#x02019; to create mapping populations. In subsequent generations, soybean seeds were analyzed for <span class="trait">seed</span> protein and oil composition by <span class="gene">NIR</span> using a Perten DA7200 diode array instrument equipped with calibration equations developed by Perten in cooperation with the University of Minnesota. Genomic DNA samples from nine individuals at tail ends of the <span class="trait">seed</span> composition phenotype were isolated using the Qiagen DNeasy kit protocol and assayed on the Illumina Goldengate platform for genotyping using 1536 soybean Universal SNP BARC markers ( <xref rid="bib20" ref-type="bibr"> Hyten <italic> et al. </italic> 2010 </xref> ). SNP markers were subject to BLAST ( <xref rid="bib1" ref-type="bibr"> Altschul <italic> et al. </italic> 1990 </xref> ) analysis to identify the corresponding physical SNP position along the reference genome sequence, and polymorphic SNP locations were examined for FN-induced structural variations. PCR confirmation of the deletion in FN0172932 was performed on DNA from 81 segregating backcrossed progeny at each end of the <span class="trait">seed</span> composition phenotype distribution using flanking primer sequences 5&#x02032;-GCTGGAGCTAGCTGAGGAACCTG-3&#x02032; and 5&#x02032;-GGGGGAGATTGTGGAAGCAATGACT-3&#x02032; and interior primer sequences 5&#x02032;-TGGCCGATGTGATGGACTTCTTCC-3&#x02032; and 5&#x02032;-GGGGGAGATTGTGGAAGCAATGACT-3&#x02032;.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Organic acids for control of <italic> Salmonella </italic> in different feed materials</h2>
			<h3>PMID:23597100</h3>
				
					<h3>PMCID:3646707</h3>
					<h4>期刊:BMC Veterinary Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Koyuncu </surname>  <given-names> Sevinc </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sevinc.koyuncu@sva.se </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Andersson </surname>  <given-names> Mats Gunnar </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> gunnar.andersson@sva.se </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> L&#x000f6;fstr&#x000f6;m </surname>  <given-names> Charlotta </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> chalo@food.dtu.dk </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Skandamis </surname>  <given-names> Panagiotis N </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> pskan@aua.gr </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Gounadaki </surname>  <given-names> Antonia </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> agounadaki@aua.gr </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Zentek </surname>  <given-names> J&#x000fc;rgen </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> zentek.juergen@vetmed.fu-berlin.de </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A7">  <name>  <surname> H&#x000e4;ggblom </surname>  <given-names> Per </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> per.haggblom@sva.se </email>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The initial survival analysis in trial 1 was performed in rape <span class="trait">seed</span> meal, since pilot studies indicated that the effect of acid in this commodity was intermediate. The results showed that 1 h after addition of acid, there was a decrease in <italic> Salmonella </italic> counts compared with the control (Figure&#x000a0; <xref ref-type="fig" rid="F1"> 1 </xref> ). However, the immediate acid effect is difficult to interpret, since masking effects, imperfect recovery and sublethally injured <italic> Salmonella </italic> on the <span class="gene">TSA</span> plates might have contributed to these results.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gene amplification of the <italic> Hps </italic> locus in <italic> Glycine max </italic></h2>
			<h3>PMID:16536872</h3>
				
					<h3>PMCID:1421400</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Gijzen </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> gijzenm@agr.gc.ca </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Kuflu </surname>  <given-names> Kuflom </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kufluk@agr.gc.ca </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Moy </surname>  <given-names> Pat </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> moyp@agr.gc.ca </email>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 3 </month>  <year> 2006 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Hydrophobic protein from soybean (HPS) is an 8 kD cysteine-rich polypeptide that causes asthma in persons allergic to soybean dust. <span class="gene">HPS</span> is synthesized in the pod endocarp and deposited on the <span class="trait">seed</span> surface during development. Past evidence suggests that the protein may mediate the adherence or dehiscence of endocarp tissues during maturation and affect the lustre, or glossiness of the <span class="trait">seed</span> surface.
					</p>
				
					<p>&nbsp;&nbsp;
						A comparison of soybean germplasm by genomic DNA blot hybridization shows that the copy number and structure of the <italic> Hps </italic> locus is polymorphic among soybean cultivars and related species. Changes in <italic> Hps </italic> gene copy number were also detected by comparative genomic DNA hybridization using cDNA microarrays. The <italic> Hps </italic> copy number polymorphisms co-segregated with <span class="trait">seed</span> lustre phenotype and <span class="gene">HPS</span> surface protein in a cross between dull- and shiny-seeded soybeans. In soybean cultivar Harosoy 63, a minimum of 27 &#x000b1; 5 copies of the <italic> Hps </italic> gene were estimated to be present in each haploid genome. The isolation and analysis of genomic clones indicates that the core <italic> Hps </italic> locus is comprised of a tandem array of reiterated units, with each 8.6 kb unit containing a single <span class="gene">HPS</span> open reading frame.
					</p>
				
					<p>&nbsp;&nbsp;
						It is not known what molecules control the adherence of endocarp to the <span class="trait">seed</span> surface, but one likely factor is an 8 kDa cysteine-rich protein named <span class="gene">HPS</span> ( <underline> h </underline> ydrophobic <underline> p </underline> rotein from <underline> s </underline> oybean). Past studies indicate that <span class="gene">HPS</span> is synthesized in the endocarp and deposited on the <span class="trait">seed</span> surface during development [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. The presence of <span class="gene">HPS</span> on the <span class="trait">seed</span> surface is a trait that cosegregates with the <span class="trait">seed</span> lustre determinant <italic> B </italic> [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. These facts along with other evidence suggest that <span class="gene">HPS</span> can mediate the attachment of endocarp tissues to the <span class="trait">seed</span> surface and thereby affect the <span class="trait">seed</span> lustre.
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">HPS</span> protein has also been named Gly m 1 because it is the major allergen that causes asthma in persons allergic to soybean dust [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Epidemic outbreaks of asthma caused by the presence of soybean dust have been documented in many cities [ <xref ref-type="bibr" rid="B9"> 9 </xref> ]. The occurrence of relatively large amounts of <span class="gene">HPS</span> on the <span class="trait">seed</span> surface results in the release of aerosols containing the protein during <span class="trait">seed</span> handling. Airborne <span class="gene">HPS</span> can be detected in ports where soybeans are transferred and even in regions where soybeans are grown, during the harvesting season [ <xref ref-type="bibr" rid="B10"> 10 </xref> , <xref ref-type="bibr" rid="B11"> 11 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Here we demonstrate that genetic polymorphisms that affect the copy number of the <italic> Hps </italic> gene are prevalent in soybean germplasm. We show that <italic> Hps </italic> genes are clustered in a tandem array at a single genetic locus, and we suggest that a process of gene amplification has led to this structural arrangement. Finally, we propose that changes in <italic> Hps </italic> gene copy number modulate <span class="gene">HPS</span> protein expression levels and <span class="trait">seed</span> lustre phenotypes.
					</p>
				
					<p>&nbsp;&nbsp;
						Past studies have pointed to a role for <span class="gene">HPS</span> in the control of <span class="trait">seed</span> lustre in soybean cultivars [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Now, we have conducted an extensive study of <italic> Hps </italic> copy number polymorphisms in a range of soybean lines and related legume species. The structure of the <italic> Hps </italic> gene was investigated by isolating and characterizing clones from the genomic region. The results have led us to propose a model to account for variation of <span class="trait">seed</span> lustre controlled by <italic> Hps </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						All <italic> Glycine max </italic> lines that were tested contained multiple copies of the <italic> Hps </italic> gene, but there were large differences in the number of copies of <italic> Hps </italic> depending on the cultivar examined. We observed a good correlation between the apparent <italic> Hps </italic> copy number, as judged by hybridization intensity on DNA blots, and <span class="trait">seed</span> lustre. This is especially true for dull- and shiny-seeded phenotypes and for intermediates between these types. This relationship was not apparent for bloom phenotypes, an exception that has been noted in past studies that correlated the occurrence of <span class="gene">HPS</span> protein to <span class="trait">seed</span> lustre [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ]. Two bloom phenotypes analyzed, Clark B1 and Sooty, produced contrasting patterns of <italic> Hps </italic> hybridization. This can be accounted for by tracing the pedigree of Clark B1. The cv Clark is a dull phenotype with a high-copy <italic> Hps </italic> RFLP pattern, whereas Sooty is a bloom phenotype with a low-copy RFLP pattern. Clark B1 is an isoline derived from a cross between Clark and Sooty, with Clark as the recurrent parent. This indicates that the bloom phenotype ( <italic> B1 </italic> ) is controlled by genes that are independent of <italic> B </italic> and <italic> Hps </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In the cross between soybean lines OX281 and Mukden, <italic> Hps </italic> copy number polymorphisms cosegregated with <span class="trait">seed</span> lustre phenotype <italic> B </italic> and associated genetic markers. This result was expected because past studies have shown that <italic> B </italic> cosegregates with the presence of <span class="gene">HPS</span> protein on the <span class="trait">seed</span> surface, and with a DNA marker derived from the <italic> Hps </italic> cDNA sequence [ <xref ref-type="bibr" rid="B2"> 2 </xref> ]. The multiple copies of <italic> Hps </italic> that are present in OX281 segregate in a Mendelian fashion, indicating that they occur at a single genetic locus and are not distributed throughout the genome. The analysis and assembly of <italic> Hps </italic> genomic clones substantiates the inheritance results, since the clones could be aligned to produce a reiterated array of <italic> Hps </italic> genes. All of the evidence therefore points to a tandem array of <italic> Hps </italic> genes occurring in a structural configuration arising from gene amplification.
					</p>
				
					<p>&nbsp;&nbsp;
						The results from this study together with past work [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B5"> 5 </xref> ] can be integrated into a model, whereby <italic> Hps </italic> genomic copy number operates as a genetic rheostat to control transcriptional and translational flux and the resulting quantity of <span class="gene">HPS</span> protein synthesized by the endocarp. Variation in <span class="gene">HPS</span> protein levels expressed in the endocarp could then account for the variable pattern of attachment of this tissue to the <span class="trait">seed</span> surface, and the resulting <span class="trait">seed</span> lustre phenotypes. Alternative explanations cannot be excluded, but the evidence so far tends to favour this gene amplification-based hypothesis. What kind of selection pressure could cause this to occur? The size, shape, colour, and general appearance of the <span class="trait">seed</span> are traits that are under intense selective pressure for crop plants, especially so for legumes. Even today certain markets may favour dull- or shiny-seeded soybeans for particular uses, so it is not unreasonable to suppose that selection for various lustre phenotypes has accompanied the development and expansion of this crop since its domestication some 3,000 years ago [ <xref ref-type="bibr" rid="B24"> 24 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						This study demonstrates that copy number polymorphisms of the <italic> Hps </italic> gene are common in soybean cultivars and lines. In some cultivars, in excess of 27 &#x000b1; 5 copies of <italic> Hps </italic> occur in a tandem array at a single locus. From these results, together with past studies on the occurrence and inheritance of the <span class="gene">HPS</span> protein, we developed a model to account for variation in <span class="trait">seed</span> lustre controlled by the <italic> B </italic> locus. The model proposes that <italic> Hps </italic> copy number changes provide a mechanism to modulate <span class="gene">HPS</span> protein levels expressed in the pod endocarp. Variable <span class="gene">HPS</span> expression in the endocarp likewise generates variation in the quantity and pattern of attachment of the endocarp to the <span class="trait">seed</span> surface, thereby affecting the <span class="trait">seed</span> lustre. Experiments in the future can be designed to test this model, and to investigate additional genetic loci controlling <span class="trait">seed</span> lustre that are independent of <italic> B </italic> and <italic> Hps </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of secretory IgA antibodies in plants.</h2>
			<h3>PMID:11566600  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Larrick JW(1), Yu L, Naftzger C, Jaiswal S, Wycoff K.
				<br/> 
				年份：2001 Oct 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Functional antibodies produced in tobacco plants were first reported over a decade ago (1989). The basic protocol used to generate these 'plantibodies' involved the independent cloning of H and L chain antibody genes in Agrobacterium tumefaciens vectors, the transformation of plant tissue in vitro with the recombinant bacterium, the reconstitution of whole plants expressing individual chains, and their sexual cross. In a 'Mendelian' fashion, a fully assembled and functional antibody was recovered from plant tissue in some double-transgenic plants. In mammalian cells, the antibody H and L chains are produced as precursor proteins that are translocated into the endoplasmic reticulum (ER), under the guidance of signal sequences. Within the ER, the signal peptides are proteolytically cleaved, and several stress proteins act as chaperonins to bind the unassembled antibody chains, and direct subsequent folding and tetramer formation. A similar process occurs in plant cells, and expression can be directed via signal sequences (even of foreign origin) into the aqueous environment of the apoplasm, or to be accumulated in other specific plant tissues, including tubers, fruit, or <span class="trait">seed.</span> Plants can facilely assemble secretory IgA, which is comprised of four chains, H and L chains, J chain and secretory component. Plant 'bioreactors' are expected to <span class="trait">yield</span> over 10 kg of therapeutic antibody/acre in tobacco, maize, soybean, and alfalfa [(Ann. NY Acad. Sci.)721(1994)235; (Biotechnol. Bioeng.)20(1999)135]. Compared with conventional steel tank bioreactors using mammalian cells, or microorganisms, the costs of <span class="gene">GMP</span> plantibodies are expected to perhaps one tenth. The differences in glycosylation patterns of plant and mammalian cell produced antibodies apparently have no effect on antigen-binding or specificity, but there is some concern about potential immunogenicity in humans. N-linked glycans of plants differ from human by having fucose-linked alpha 1,3 and the sugar xylose. No adverse effects or human anti-mouse antibodies (HAMA) have been observed in &gt;40 patients receiving topical oral application of a plant produced secretory IgA specific to Streptococcus mutans, for the control of caries [(Nat. Med.)4(1998)601]. The progressive improvement of expression vectors for plantibodies, and purification strategies, as well as the increase in transformable crop species, is expected to lead to almost limitless availability of inexpensive (even edible forms of) recombinant immunoglobulins free of human pathogens for human and animal therapy, and for novel industrial applications (e.g. catalytic antibodies).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biplot Analysis of Test Sites and Trait Relations of Soybean in Ontario.</h2>
			<h3>PMID:11756248</h3>
				

				
			<p>
				作者：Yan W(1), Rajcan I.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Superior crop cultivars must be identified through multi-environment trials (MET) and on the basis of multiple traits. The objectives of this paper were to describe two types of biplots, the GGE biplot and the GT biplot, which graphically display genotype by environment data and genotype by trait data, respectively, and hence facilitate cultivar evaluation on the basis of <span class="gene">MET</span> data and multiple traits. Genotype main effect plus genotype by environment interaction effect (GGE) biplot analysis of the soybean [Glycine max (L.) Merr.] <span class="trait">yield</span> data for the 2800 crop heat unit area of Ontario for <span class="gene">MET</span> in the period 1994-1999 revealed yearly crossover genotype by site interactions. The eastern Ontario site Winchester showed a different genotype response pattern from the three southwestern Ontario sites in four of the six years. The interactions were not large enough to divide the area into different mega-environments as when analyzed over years, a single cultivar yielded the best in all four sites. The southwestern site, St. Pauls, was found to always group together with at least one of the other three sites; it did not provide unique information on genotype performance. Therefore, in future cultivar evaluations, Winchester should always be used but St. Pauls can be dismissed. Applying GT biplot to the 1994-1999 multiple trait data illustrated that GT biplots graphically displayed the interrelationships among <span class="trait">seed</span> <span class="trait">yield,</span> oil content, protein content, plant height, and days to maturity, among other traits, and facilitated visual cultivar comparisons and selection. It was found that selection for <span class="trait">seed</span> <span class="trait">yield</span> alone was not only the simplest, but also the most effective strategy in the early stages of soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Introduction of DPR, an enterostatin fragment peptide, into soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis.</h2>
			<h3>PMID:14745196  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Takenaka Y(1), Doyama N, Maruyama N, Utsumi S, Yoshikawa M.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DPR, a fragment peptide of enterostatin (VPDPR) having hypocholesterolemic activity, was introduced into the three homologous sites, EPR, DYR, and DPI, in the soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis. The modified beta-conglycinin was expressed in Escherichia coli and recovered in the soluble fraction. After purification on ion-exchange HPLC, the modified beta-conglycinin was digested by trypsin to release integrated DPR. The <span class="trait">yield</span> of DPR from 1 mole of the modified beta-conglycinin was 1.2 mole.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of palm kernel meal and corn distillers grains in corn silage-based diets for lactating dairy cows.</h2>
			<h3>PMID:16772589  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carvalho LP(1), Cabrita AR, Dewhurst RJ, Vicente TE, Lopes ZM, Fonseca AJ.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of increasing levels of solvent-extracted palm kernel meal (SPKM) and corn distillers dried grains (CDG) in corn silage-based diets on feed intake and milk production were examined in 2 experiments. In Experiment 1, 20 Holstein cows averaging 100 d in milk (DIM) (SD = 61.5) at the start of the experiment were utilized in an 11-wk randomized complete block design with 4 treatments in 5 blocks to study effects of increasing levels of SPKM in the diet. During a 3-wk preliminary period, cows were fed a standard diet. At the end of the preliminary period, cows were blocked by 4% fat-corrected milk <span class="trait">yield,</span> parity number (primiparous and multiparous), and DIM, and were assigned randomly to 1 of 4 experimental diets. The total mixed ration (TMR) consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of SPKM were achieved by replacing protein sources and citrus pulp with SPKM and urea (0, 5, 10, and 15% SPKM and 0.06, 0.22, 0.38, and 0.55% urea for SPKM0, SPKM5, SPKM10, and SPKM15, respectively). In Experiment 2, 18 Holstein cows averaging 93 <span class="gene">DIM</span> (SD = 49.1) at the start of the experiment were utilized in an 11-wk randomized complete block design with 3 treatments in 6 blocks to study effects of increasing levels of CDG in the diet. The preliminary period lasted for 2 wk. Assignment of cows to treatments was the same as in Experiment 1. The TMR consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of CDG were achieved by replacing soybean meal and citrus pulp with CDG and urea (0, 7, and 14% CDG and 0, 0.22, and 0.49% urea for CDG0, CDG7, and CDG14, respectively). There were no significant treatment effects on dry matter intake, milk <span class="trait">yield,</span> or milk composition in Experiment 1. Inclusion of SPKM tended to increase protein and lactose contents of milk. The SPKM0 diet promoted body weight loss. There were no treatment effects in Experiment 2, except for milk protein content, which was decreased by CDG. Plasma Lys concentration tended to be affected by SPKM and CDG inclusions. Plasma concentrations of 3-methylhistidine and Leu seemed to be related to body protein degradation/synthesis. The feeding of SPKM up to 15% in the diet decreased feed costs without detrimental effects on productive responses and tended to increase milk protein content. The inclusion of CDG in diets based on corn silage and corn byproducts might decrease milk protein content due to an unbalanced supply of AA, particularly Lys.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed <span class="trait">yield</span> of near-isogenic soybean lines with introgressed quantitative trait loci conditioning <span class="trait">resistance</span> to corn earworm (Lepidoptera: Noctuidae) and soybean looper (Lepidoptera: Noctuidae) from PI 229358.</h2>
			<h3>PMID:18767762  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Warrington CV(1), Zhu S, Parrott WA, All JN, Boerma HR.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The development of superior soybean, Glycine max (L.) Merr., cultivars exhibiting <span class="trait">resistance</span> to insects has been hindered due to linkage drag, a common phenomenon when introgressing alleles from exotic germplasm. Simple-sequence repeat (SSR) markers were used previously to map soybean insect <span class="trait">resistance</span> (SIR) quantitative trait loci (QTLs) in a'Cobb' X PI 229358 population, and subsequently used to create near-isogenic lines (NILs) with <span class="gene">SIR</span> QTL i n a 'Benning' genetic background. <span class="gene">SIR</span> QTLs were mapped on linkage groups (LGs) M (SIRQTL-M), G (SIRQTL-G), and H (SIRQTL-H). The objectives of this study were to 1) evaluate linkage drag for <span class="trait">seed</span> <span class="trait">yield</span> by using Benning-derived NILs selected for SIRQTL-M, SIRQTL-H, and SIRQTL-G; 2) assess the amount of PI 229358 genome surrounding the <span class="gene">SIR</span> QTL in each Benning NIL; and 3) evaluate the individual effects these three QTLs on antibiosis and antixenosis to corn earworm, Helicoverpa zea (Boddie), and soybean looper, Pseudoplusia includens (Walker). <span class="trait">Yield</span> data collected in five environments indicated that a significant <span class="trait">yield</span> reduction is associated with SIRQTL-G compared with NILs without <span class="gene">SIR</span> QTL. Overall, there was no <span class="trait">yield</span> reduction associated with SIRQTL-M or SIRQTL-H. A significant antixenosis and antibiosis effect was detected for SIRQTL-M in insect feeding assays, with no effect detected in antixenosis or antibiosis assays for SIRQTL-G or SIRQTL-H without the presence of PI 229358 alleles at SIRQTL-M. These results support recent findings concerning these loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The critical period of weed control in soybean (Glycine max (L.) Merr.) in north of Iran conditions.</h2>
			<h3>PMID:18817174  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Keramati S(1), Pirdashti H, Esmaili MA, Abbasian A, Habibi M.
				<br/> 
				年份：2008 Feb 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field study was conducted in 2006 at Sari Agricultural and Natural Resources University, in order to determine the best time for weed control in soybean promising line, 033. Experiment was arranged in randomized complete block design with 4 replications and two series of treatments. In the first series, weeds were kept in place until crop reached <span class="gene">V2</span> (second trifoliolate), V4 (fourth trifoliolate), V6 (sixth trifoliolate), R1 (beginning bloom, first flower), R3 (beginning pod), R5 (beginning <span class="trait">seed)</span> and were then removed and the crop kept weed-free for the rest of the season. In the second series, crops were kept weed-free until the above growth stages after which weeds were allowed to grow in the plots for the rest of the season. Whole season weedy and weed-free plots were included in the experiment for <span class="trait">yield</span> comparison. The results showed that among studied traits, grain <span class="trait">yield,</span> pod numbers per plant and weed biomass were affected significantly by control and interference treatments. The highest number of pods per plant was obtained from plots which kept weed-free for whole season control. Results showed that weed control should be carried out between <span class="gene">V2</span> (26 day after planting) to R1 (63 day after planting) stages of soybean to provide maximum grain <span class="trait">yield.</span> Thus, it is possible to optimize the timing of weed control, which can serve to reduce the costs and side effects of intensive chemical weed control.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-environment mapping and meta-analysis of 100-seed weight in soybean.</h2>
			<h3>PMID:22740134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun YN(1), Pan JB, Shi XL, Du XY, Wu Q, Qi ZM, Jiang HW, Xin DW, Liu CY, Hu GH, Chen QS.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						100-Seed weight (100-SW) of soybean is an important but complicated quantitative trait to <span class="trait">yield.</span> This study was focus on the quantitative trait loci (QTLs) of soybean 100-SW from 2006 to 2010, using recombination inbred lines population that was derived from a cross between Charleston and Dongnong 594. A total of 23 QTLs for 100-SW were detected in the linkage group C2, D1a, F, G and O. Nine QTLs were identified by composite interval mapping including one QTL with the minimum confidence interval (CI) of 1.3 cM, while 14 QTLs by multiple interval mapping. Furthermore, 94 reported QTLs of 100-SW were integrated with our QTL mapping results using BioMercator. As a result, 15 consensus QTLs and their corresponding markers were identified. The minimum <span class="gene">CI</span> was reduced to 1.52 cM by the combination of meta-analysis. These findings may merit fine-mapping of these QTL in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protocol for augmented shoot organogenesis in selected variety of soybean [Glycine max L. (Merr.)].</h2>
			<h3>PMID:23214267  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Devi MK(1), Sakthivelu G, Giridhar P, Ravishankar GA.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Development of a reproducible, versatile and efficient in vitro plant regeneration system is highly warranted for Indian soybean varieties for their mass multiplication in view of their commercial significance. Accordingly a protocol for direct shoot organogenesis in soybean variety JS 335 has been developed. Using cotyledonary node explants significant organogenic responses, mean shoot number and shoot length were observed when these were incubated on <span class="gene">MS</span> medium supplemented with 0.89 microM Benzyladenine (BA) and 5 microg/L triacontanol (TRIA) where in 9.3 +/- 0.5 shoots were obtained. TRIA at 5 microg/L able to produce 6.8 +/- 0.5 shoot buds in presence of 0.98 microM IBA and 0.89 microM BA. Highest mean shoot buds (14.0 +/- 0.5 and 9.0 +/- 0.5) and mean shoot length (4.6 +/- 0.3 and 10.0 +/- 0.7) were obtained when cotyledonary node and shoot tip explants were cultured on <span class="gene">MS</span> medium containing 0.14 microM gibberellic acid (GA3), 0.89 microM BA and 5 microg/L TRIA. Moreover, TRIA supported highest mean root number (6.3 +/- 0.5) and root length (21.5 +/- 0.57 cm). Field survival of in vitro derived plants of TRIA treatment was 70% and the overall growth and <span class="trait">seed</span> <span class="trait">yield</span> was also significantly better than control plants. This protocol may be used for improving the in vitro regeneration of soybean variety JS 335 for transformation studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular basis of a shattering <span class="trait">resistance</span> boosting global dissemination of soybean.</h2>
			<h3>PMID:25468966  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Funatsuki H(1), Suzuki M(2), Hirose A(3), Inaba H(2), Yamada T(4), Hajika M(4), Komatsu K(3), Katayama T(5), Sayama T(6), Ishimoto M(6), Fujino K(7).
				<br/> 
				年份：2014 Dec 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pod dehiscence (shattering) is essential for the propagation of wild plant species bearing seeds in pods but is a major cause of <span class="trait">yield</span> loss in legume and crucifer crops. Although natural genetic variation in pod dehiscence has been, and will be, useful for plant breeding, little is known about the molecular genetic basis of shattering <span class="trait">resistance</span> in crops. Therefore, we performed map-based cloning to unveil a major quantitative trait locus (QTL) controlling pod dehiscence in soybean. Fine mapping and complementation testing revealed that the QTL encodes a dirigent-like protein, designated as Pdh1. The gene for the shattering-resistant genotype, pdh1, was defective, having a premature stop codon. The functional gene, Pdh1, was highly expressed in the lignin-rich inner sclerenchyma of pod walls, especially at the stage of initiation in lignin deposition. Comparisons of near-isogenic lines indicated that <span class="gene">Pdh1</span> promotes pod dehiscence by increasing the torsion of dried pod walls, which serves as a driving force for pod dehiscence under low humidity. A survey of soybean germplasm revealed that pdh1 was frequently detected in landraces from semiarid regions and has been extensively used for breeding in North America, the world's leading soybean producer. These findings point to a new mechanism for pod dehiscence involving the dirigent protein family and suggest that pdh1 has played a crucial role in the global expansion of soybean cultivation. Furthermore, the orthologs of pdh1, or genes with the same role, will possibly be useful for crop improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Intersubgeneric hybridization between Glycine max and G. tomentella: production of F₁, amphidiploid, BC₁, BC₂, BC₃, and fertile soybean plants.</h2>
			<h3>PMID:25835560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Singh RJ(1), Nelson RL.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: This paper describes methods for unlocking genetic treasure from wild perennial Glycine species of Australia for soybean improvement. The genetic resources of the ca. 26 species of the genus Glycine subgenus Glycine have not been exploited to broaden the genetic base of soybean (Glycine max; 2n = 40). The objectives of this study were to develop methods for producing F1, amphidiploid, BC1, BC2, BC3, and fertile soybean plants from crosses of soybean and the genus Glycine subgenus Glycine species, in order to utilize the subgenus Glycine germplasm in soybean breeding. Soybean cultivars were hybridized with six accessions of 78-chromosome G. tomentella as well as one accession each of 40-chromosome G. tomentella, G. argyrea and G. latifolia. They were chosen because they exhibit <span class="trait">resistance</span> to soybean <span class="trait">rust.</span> We were successful in producing fertile soybean from soybean cv. 'Dwight' and 78-chromosome G. tomentella accession PI 441001, while other hybrids were discontinued either at F1 or amphidiploid stage. The F1 seeds aborted prior to reaching maturity, so developing seeds from 19 to 21 day old pods were cultured aseptically in various media formulations. <span class="trait">Seed</span> maturation and multiple embryo generation media were developed. F1 plants with shoots and roots (2n = 59) were transplanted to pots in greenhouse. Amphidiploid (2n = 118) plants were backcrossed to 'Dwight'. <span class="gene">BC1</span> (2n = 79) plants were propagated through in vitro and 43 mature BC2F1 seeds were harvested. Fifteen surviving BC2F1 plants were morphologically distinct, sterile, and had chromosome numbers ranging 2n = 56-59. Chromosome numbers of the BC3F1 plants ranged 2n = 40-49. Derived fertile soybeans were first planted in the field in 2008 and are being evaluated for <span class="trait">yield,</span> <span class="trait">resistance</span> to pathogens and pests and tolerance to salt through material transfer agreement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Gamma radiation and magnetic field mediated delay in effect of accelerated ageing of soybean.</h2>
			<h3>PMID:26243899</h3>
				

				
			<p>
				作者：Kumar M(1), Singh B(2), Ahuja S(2), Dahuja A(3), Anand A(4).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean seeds were exposed to gamma radiation (0.5, 1, 3 and 5 kGy), static magnetic field (50, 100 and 200 mT) and a combination of gamma radiation and magnetic energy (0.5 kGy + 200 mT and 5 kGy + 50 mT) and stored at room temperature for six months. These seeds were later subjected to accelerated ageing treatment at 42 °C temperature and 95-100 % relative humidity and were compared for various physical and biochemical characteristics between the untreated and the energized treatments. Energy treatment protected the quality of stored seeds in terms of its protein and oil content . Accelerated aging conditions, however, affected the oil and protein quantity and quality of <span class="trait">seed</span> negatively. Antioxidant enzymes exhibited a decline in their activity during aging while the <span class="gene">LOX</span> activity, which reflects the rate of lipid peroxidation, in general, increased during the aging. Gamma irradiated (3 and 5 kGy) and magnetic field treated seeds (100 and 200 mT) maintained a higher catalase and ascorbate peroxidase activity which may help in efficient scavenging of deleterious free radical produced during the aging. Aging caused peroxidative changes to lipids, which could be contributed to the loss of oil quality. Among the electromagnetic energy treatments, a dose of 1-5 kGy of gamma and 100 mT, 200 mT magnetic field effectively slowed the rate of biochemical degradation and loss of cellular integrity in seeds stored under conditions of accelerated aging and thus, protected the deterioration of <span class="trait">seed</span> quality. Energy combination treatments did not <span class="trait">yield</span> any additional protection advantage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog.</h2>
			<h3>PMID:26786479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao C(1), Takeshima R(2), Zhu J(3), Xu M(4), Sato M(5), Watanabe S(6), Kanazawa A(7), Liu B(8), Kong F(9), Yamada T(10), Abe J(11).
				<br/> 
				年份：2016 Jan 19
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Understanding the molecular mechanisms of flowering and maturity is important for improving the adaptability and <span class="trait">yield</span> of <span class="trait">seed</span> crops in different environments. In soybean, a facultative short-day plant, genetic variation at four maturity genes, <span class="gene">E1</span> to E4, plays an important role in adaptation to environments with different photoperiods. However, the molecular basis of natural variation in time to flowering and maturity is poorly understood. Using a cross between early-maturing soybean cultivars, we performed a genetic and molecular study of flowering genes. The progeny of this cross segregated for two maturity loci, <span class="gene">E1</span> and E9. The latter locus was subjected to detailed molecular analysis to identify the responsible gene. RESULTS: Fine mapping, sequencing, and expression analysis revealed that E9 is FT2a, an ortholog of Arabidopsis FLOWERING LOCUS T. Regardless of daylength conditions, the e9 allele was transcribed at a very low level in comparison with the E9 allele and delayed flowering. Despite identical coding sequences, a number of single nucleotide polymorphisms and insertions/deletions were detected in the promoter, untranslated regions, and introns between the two cultivars. Furthermore, the e9 allele had a Ty1/copia-like retrotransposon, SORE-1, inserted in the first intron. Comparison of the expression levels of different alleles among near-isogenic lines and photoperiod-insensitive cultivars indicated that the SORE-1 insertion attenuated FT2a expression by its allele-specific transcriptional repression. SORE-1 was highly methylated, and did not appear to disrupt FT2a RNA processing. CONCLUSIONS: The soybean maturity gene E9 is FT2a, and its recessive allele delays flowering because of lower transcript abundance that is caused by allele-specific transcriptional repression due to the insertion of SORE-1. The FT2a transcript abundance is thus directly associated with the variation in flowering time in soybean. The e9 allele may maintain vegetative growth in early-flowering genetic backgrounds, and also be useful as a long-juvenile allele, which causes late flowering under short-daylength conditions, in low-latitude regions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Introgression of null allele of Kunitz trypsin inhibitor through marker-assisted backcross breeding in soybean (Glycine max L. Merr.).</h2>
			<h3>PMID:27407019  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Maranna S(1)(2), Verma K(2), Talukdar A(3), Lal SK(2), Kumar A(2), Mukherjee K(2).
				<br/> 
				年份：2016 Jul 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Presence of Kunitz trypsin inhibitor (KTI) in soybean seeds necessitates pre-heat treatment of the soy-flour for its inactivation before using it in food and feed products. The heat treatment not only enhances processing costs of the soy-based foods and feeds but also affects seed-protein quality and solubility. Genetic elimination of <span class="gene">KTI</span> is an important and effective approach. Therefore, molecular marker-assisted backcross breeding (MABB) approach was adopted for genetic elimination of <span class="gene">KTI</span> from two popular soybean genotypes, DS9712 and DS9814. PI542044, an exotic germplasm line was used as donor of the kti allele which inhibits functional <span class="gene">KTI</span> peptide production. RESULTS: Foreground selection for the kti allele was performed with three closely linked SSR markers while background selection was done with 93 polymorphic SSR markers. Plants in the BC1F1 generation were found to recover 70.4-87.63 % and 60.26-73.78 % of the recurrent parent genome (RPG) of DS9712 and DS9814, respectively. Similarly, selected plants in the BC2F1 generation had 93.01-98.92 % and 83.3-91.67 % recovery of their respective RPGs. Recombinant selection was performed so as to identify plants with minimal linkage drag. Biochemical analysis of the seeds of the selected plants (ktikti) confirmed absence of <span class="gene">KTI</span> peptides in the seeds. Phenotypically, the selected plants were comparable to the respective recurrent parent in <span class="trait">yield</span> and other traits. CONCLUSIONS: MABB approach helped in speedy development of 6 <span class="gene">KTI</span> free breeding lines of soybean. Such lines will be suitable for the farmers and the soybean industries to use in production of soy-based foods and feeds without pre-heat treatment of the soy-flour. It would contribute towards wider acceptability of soy-based foods and feeds.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing <span class="trait">seed</span> size and quality by manipulating BIG SEEDS1 in legume species.</h2>
			<h3>PMID:27791139</h3>
				

				
			<p>
				作者：Ge L(1), Yu J(2), Wang H(1), Luth D(3)(4), Bai G(2)(5), Wang K(3)(4), Chen R(6).
				<br/> 
				年份：2016 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant organs, such as seeds, are primary sources of food for both humans and animals. <span class="trait">Seed</span> size is one of the major agronomic traits that have been selected in crop plants during their domestication. Legume seeds are a major source of dietary proteins and oils. Here, we report a conserved role for the BIG SEEDS1 (BS1) gene in the control of <span class="trait">seed</span> size and weight in the model legume Medicago truncatula and the grain legume soybean (Glycine max). <span class="gene">BS1</span> encodes a plant-specific transcription regulator and plays a key role in the control of the size of plant organs, including seeds, <span class="trait">seed</span> pods, and leaves, through a regulatory module that targets primary cell proliferation. Importantly, down-regulation of <span class="gene">BS1</span> orthologs in soybean by an artificial microRNA significantly increased soybean <span class="trait">seed</span> size, weight, and amino acid content. Our results provide a strategy for the increase in <span class="trait">yield</span> and <span class="trait">seed</span> quality in legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Silencing of Soybean Raffinose Synthase Gene Reduced Raffinose Family Oligosaccharides and Increased True Metabolizable Energy of Poultry Feed.</h2>
			<h3>PMID:28559898</h3>
				

				
			<p>
				作者：Valentine MF(1), De Tar JR(1), Mookkan M(1), Firman JD(2), Zhang ZJ(1)(2).
				<br/> 
				年份：2017 May 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is the number one oil and protein crop in the United States, but the <span class="trait">seed</span> contains several anti-nutritional factors that are toxic to both humans and livestock. RNA interference technology has become an increasingly popular technique in gene silencing because it allows for both temporal and spatial targeting of specific genes. The objective of this research is to use RNA-mediated gene silencing to down-regulate the soybean gene raffinose synthase 2 (RS2), to reduce total raffinose content in mature <span class="trait">seed.</span> Raffinose is a trisaccharide that is indigestible to humans and monogastric animals, and as monogastric animals are the largest consumers of soy products, reducing raffinose would improve the nutritional quality of soybean. An RNAi construct targeting <span class="gene">RS2</span> was designed, cloned, and transformed to the soybean genome via Agrobacterium-mediated transformation. Resulting plants were analyzed for the presence and number of copies of the transgene by PCR and Southern blot. The efficiency of mRNA silencing was confirmed by real-time quantitative PCR. Total raffinose content was determined by HPLC analysis. Transgenic plant lines were recovered that exhibited dramatically reduced levels of raffinose in mature <span class="trait">seed,</span> and these lines were further analyzed for other phenotypes such as development and <span class="trait">yield.</span> Additionally, a precision-fed rooster assay was conducted to measure the true metabolizable energy (TME) in full-fat soybean meal made from the wild-type or transgenic low-raffinose soybean lines. Transgenic low-raffinose soy had a measured TME of 2,703 kcal/kg, an increase as compared with 2,411 kcal/kg for wild-type. As low digestible energy is a major limiting factor in the percent of soybean meal that can be used in poultry diets, these results may substantiate the use of higher concentrations of low-raffinose, full-fat soy in formulated livestock diets.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of the acquisition of longevity during <span class="trait">seed</span> maturation in soybean.</h2>
			<h3>PMID:28700604  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pereira Lima JJ(1)(2), Buitink J(2), Lalanne D(2), Rossi RF(1)(2), Pelletier S(2), da Silva EAA(1), Leprince O(2).
				<br/> 
				年份：2017 Jul 12
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> longevity, defined as the ability to remain alive during storage, is an important agronomic factor. Poor longevity negatively impacts seedling establishment and consequently crop <span class="trait">yield.</span> This is particularly problematic for soybean as seeds have a short lifespan. While the economic importance of soybean has fueled a large number of transcriptome studies during embryogenesis and <span class="trait">seed</span> filling, the mechanisms regulating <span class="trait">seed</span> longevity during late maturation remain poorly understood. Here, a detailed physiological and molecular characterization of late <span class="trait">seed</span> maturation was performed in soybean to obtain a comprehensive overview of the regulatory genes that are potentially involved in longevity. Longevity appeared at physiological maturity at the end of <span class="trait">seed</span> filling before maturation drying and progressively doubled until the seeds reached the dry state. The increase in longevity was associated with the expression of genes encoding protective chaperones such as heat shock proteins and the repression of nuclear and chloroplast genes involved in a range of chloroplast activities, including photosynthesis. An increase in the raffinose family oligosaccharides (RFO)/sucrose ratio together with changes in RFO metabolism genes was also associated with longevity. A gene co-expression network analysis revealed 27 transcription factors whose expression profiles were highly correlated with longevity. Eight of them were previously identified in the longevity network of Medicago truncatula, including homologues of ERF110, HSF6AB, NFXL1 and members of the <span class="gene">DREB2</span> family. The network also contained several transcription factors associated with auxin and developmental cell fate during flowering, organ growth and differentiation. A transcriptional transition occurred concomitant with <span class="trait">seed</span> chlorophyll loss and detachment from the mother plant, suggesting the activation of a post-abscission program. This transition was enriched with AP2/EREBP and WRKY transcription factors and genes associated with growth, germination and post-transcriptional processes, suggesting that this program prepares the <span class="trait">seed</span> for the dry quiescent state and germination.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Robust RNAi-mediated <span class="trait">resistance</span> to infection of seven potyvirids in soybean expressing an intron hairpin <span class="gene">NIb</span> RNA.</h2>
			<h3>PMID:28840434</h3>
				

				
			<p>
				作者：Yang X(1), Niu L(1), Zhang W(1), He H(1), Yang J(1), Xing G(1), Guo D(1), Du Q(1), Qian X(1), Yao Y(1), Li Q(2), Dong Y(3).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Viral pathogens, such as soybean mosaic virus (SMV), are a major constraint in soybean production and often cause significant <span class="trait">yield</span> loss and quality deterioration. Engineering <span class="trait">resistance</span> by RNAi-mediated gene silencing is a powerful strategy for controlling viral diseases. In this study, a 248-bp inverted repeat of the replicase (nuclear inclusion b, NIb) gene was isolated from the SMV SC3 strain, driven by the leaf-specific rbcS2 promoter from Phaseolus vulgaris, and introduced into soybean. The transgenic lines had significantly lower average disease indices (ranging from 2.14 to 12.35) than did the non-transformed (NT) control plants in three consecutive generations, exhibiting a stable and significantly enhanced <span class="trait">resistance</span> to the SMV SC3 strain under field conditions. Furthermore, <span class="trait">seed</span> mottling did not occur in transgenic seeds, whereas the NT plants produced ~90% mottled seeds. Virus <span class="trait">resistance</span> spectrum screening showed that the greenhouse-grown transgenic lines exhibited robust <span class="trait">resistance</span> to five SMV strains (SC3, SC7, SC15, SC18, and a recombinant SMV), bean common mosaic virus, and watermelon mosaic virus. Nevertheless, no significantly enhanced <span class="trait">resistance</span> to bean pod mottle virus (BPMV, Comovirus) was observed in the transgenic lines relative to their NT counterparts. Consistent with the results of <span class="trait">resistance</span> evaluation, the accumulation of each potyvirid (but not of BPMV) was significantly inhibited in the transgenic plants relative to the NT controls as confirmed by quantitative real-time (qRT-PCR) and double antibody sandwich enzyme-linked immunosorbent assay (DAS-ELISA). These results demonstrate that robust RNAi-mediated <span class="trait">resistance</span> to multiple potyvirids in soybean was conferred by expressing an intron hairpin SMV <span class="gene">NIb</span> RNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>