<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed height</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed height</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Overexpression of a <span class="gene">GmGBP1</span> ortholog of soybean enhances the responses to flowering, stem elongation and heat tolerance in transgenic tobaccos.</h2>
			<h3>PMID:23636865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao L(1), Wang Z, Lu Q, Wang P, Li Y, Lv Q, Song X, Li D, Gu Y, Liu L, Li W.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a typical short-day crop, and its photoperiodic and gibberellin (GA) responses for the control of flowering are critical to <span class="trait">seed</span> <span class="trait">yield.</span> The <span class="gene">GmGBP1</span> mRNA abundance in leaves was dramatically increased in short-days (SDs) compared to that in long-days in which it was consistently low at all time points from 0 to 6 days (days after transfer to SDs). <span class="gene">GmGBP1</span> was highly expressed in leaves and exhibited a circadian rhythm in SDs. Ectopic overexpression of <span class="gene">GmGBP1</span> in tobaccos caused photoperiod-insensitive early flowering by increasing NtCO mRNA levels. <span class="gene">GmGBP1</span> mRNA abundance was also increased by GAs. Transgenic <span class="gene">GmGBP1</span> overexpressing (-ox) tobacco plants exhibited increased GA signaling-related phenotypes including flowering and plant <span class="trait">height</span> promotion. Furthermore, the hypocotyl elongation, early-flowering and longer internode phenotypes were largely accelerated by GA3 application in the <span class="gene">GmGBP1-ox</span> tobacco seedlings. Being consistent, overexpression of <span class="gene">GmGBP1</span> resulted in significantly enhanced GA signaling (evidenced suppressed expression of NtGA20ox) both with and without GA treatments. <span class="gene">GmGBP1</span> was a positive regulator of both photoperiod and GA-mediated flowering responses. In addition, <span class="gene">GmGBP1-ox</span> tobaccos were hypersensitive to ABA, salt and osmotic stresses during <span class="trait">seed</span> germination. Heat-inducible <span class="gene">GmGBP1</span> also enhanced thermotolerance in transgenic <span class="gene">GmGBP1-ox</span> tobaccos during <span class="trait">seed</span> germination and growth. <span class="gene">GmGBP1</span> protein was localized in the nucleus. Analyses of a series of 5'-deletions of the <span class="gene">GmGBP1</span> promoter suggested that several cis-acting elements, including P-BOX, TCA-motif and three HSE elements necessary to induce gene expression by GA, salicic acid and heat stress, were specifically localized in the <span class="gene">GmGBP1</span> promoter region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the Arabidopsis thaliana BBX32 gene in soybean increases grain <span class="trait">yield.</span></h2>
			<h3>PMID:22363475  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preuss SB(1), Meister R, Xu Q, Urwin CP, Tripodi FA, Screen SE, Anil VS, Zhu S, Morrell JA, Liu G, Ratcliffe OJ, Reuber TL, Khanna R, Goldman BS, Bell E, Ziegler TE, McClerren AL, Ruff TG, Petracek ME.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Crop <span class="trait">yield</span> is a highly complex quantitative trait. Historically, successful breeding for improved grain <span class="trait">yield</span> has led to crop plants with improved source capacity, altered plant architecture, and increased <span class="trait">resistance</span> to abiotic and biotic stresses. To date, transgenic approaches towards improving crop grain <span class="trait">yield</span> have primarily focused on protecting plants from herbicide, insects, or disease. In contrast, we have focused on identifying genes that, when expressed in soybean, improve the intrinsic ability of the plant to <span class="trait">yield</span> more. Through the large scale screening of candidate genes in transgenic soybean, we identified an Arabidopsis thaliana B-box domain gene (AtBBX32) that significantly increases soybean grain <span class="trait">yield</span> year after year in multiple transgenic events in multi-location field trials. In order to understand the underlying physiological changes that are associated with increased <span class="trait">yield</span> in transgenic soybean, we examined phenotypic differences in two AtBBX32-expressing lines and found increases in plant <span class="trait">height</span> and node, flower, pod, and <span class="trait">seed</span> number. We propose that these phenotypic changes are likely the result of changes in the timing of reproductive development in transgenic soybean that lead to the increased duration of the pod and <span class="trait">seed</span> development period. Consistent with the role of BBX32 in A. thaliana in regulating light signaling, we show that the constitutive expression of AtBBX32 in soybean alters the abundance of a subset of gene transcripts in the early morning hours. In particular, AtBBX32 alters transcript levels of the soybean clock genes <span class="gene">GmTOC1</span> and LHY-CCA1-like2 <span class="gene">(GmLCL2).</span> We propose that through the expression of AtBBX32 and modulation of the abundance of circadian clock genes during the transition from dark to light, the timing of critical phases of reproductive development are altered. These findings demonstrate a specific role for AtBBX32 in modulating soybean development, and demonstrate the validity of expressing single genes in crops to deliver increased agricultural productivity.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:SUMO E3 Ligases <span class="gene">GmSIZ1a</span> and <span class="gene">GmSIZ1b</span> regulate vegetative growth in soybean .</h2>
			<h3>PMID:27762067  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cai B(1), Kong X(1), Zhong C(2), Sun S(2), Zhou XF(3), Jin YH(1), Wang Y(4), Li X(4), Zhu Z(2), Jin JB(1).
				<br/> 
				年份：2017 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						SIZ1 is a small ubiquitin-related modifier (SUMO) E3 ligase that mediates post-translational SUMO modification of target proteins and thereby regulates developmental processes and hormonal and environmental stress responses in Arabidopsis. However, the role of SUMO E3 ligases in crop plants is largely unknown. Here, we identified and characterized two Glycine max (soybean) SUMO E3 ligases, <span class="gene">GmSIZ1a</span> and <span class="gene">GmSIZ1b.</span> Expression of <span class="gene">GmSIZ1a</span> and <span class="gene">GmSIZ1b</span> was induced in response to salicylic acid (SA), heat, and dehydration treatment, but not in response to cold, abscisic acid (ABA), and NaCl treatment. Although <span class="gene">GmSIZ1a</span> was expressed at higher levels than <span class="gene">GmSIZ1b,</span> both genes encoded proteins with SUMO E3 ligase activity in vivo. Heterologous expression of <span class="gene">GmSIZ1a</span> or <span class="gene">GmSIZ1b</span> rescued the mutant phenotype of Arabidopsis siz1-2, including dwarfism, constitutively activated expression of pathogen-related genes, and ABA-sensitive <span class="trait">seed</span> germination. Simultaneous downregulation of <span class="gene">GmSIZ1a</span> and <span class="gene">GmSIZ1b</span> <span class="gene">(GmSIZ1a/b)</span> using RNA interference (RNAi)-mediated gene silencing decreased heat shock-induced SUMO conjugation in soybean. Moreover, <span class="gene">GmSIZ1RNAi</span> plants exhibited reduced plant <span class="trait">height</span> and leaf size. However, unlike Arabidopsis siz1-2 mutant plants, flowering time and SA levels were not significantly altered in <span class="gene">GmSIZ1RNAi</span> plants. Taken together, our results indicate that <span class="gene">GmSIZ1a</span> and <span class="gene">GmSIZ1b</span> mediate SUMO modification and positively regulate vegetative growth in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Genome-Wide Association Study for Agronomic Traits in Soybean Using SNP Markers and SNP-Based Haplotype Analysis.</h2>
			<h3>PMID:28152092  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Contreras-Soto RI(1), Mora F(2), de Oliveira MA(3), Higashi W(3), Scapim CA(1), Schuster I(4).
				<br/> 
				年份：2017 Feb 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mapping quantitative trait loci through the use of linkage disequilibrium (LD) in populations of unrelated individuals provides a valuable approach for dissecting the genetic basis of complex traits in soybean (Glycine max). The haplotype-based genome-wide association study (GWAS) has now been proposed as a complementary approach to intensify benefits from LD, which enable to assess the genetic determinants of agronomic traits. In this study a GWAS was undertaken to identify genomic regions that control 100-seed weight (SW), plant <span class="trait">height</span> (PH) and <span class="trait">seed</span> <span class="trait">yield</span> (SY) in a soybean association mapping panel using single nucleotide polymorphism (SNP) markers and haplotype information. The soybean cultivars (N = 169) were field-evaluated across four locations of southern Brazil. The genome-wide haplotype association analysis (941 haplotypes) identified eleven, seventeen and fifty-nine SNP-based haplotypes significantly associated with SY, SW and PH, respectively. Although most marker-trait associations were environment and trait specific, stable haplotype associations were identified for SY and SW across environments (i.e., haplotypes <span class="gene">Gm12_Hap12).</span> The haplotype block 42 on Chr19 <span class="gene">(Gm19_Hap42)</span> was confirmed to be associated with PH in two environments. These findings enable us to refine the breeding strategy for tropical soybean, which confirm that haplotype-based GWAS can provide new insights on the genetic determinants that are not captured by the single-marker approach.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biplot Analysis of Test Sites and Trait Relations of Soybean in Ontario.</h2>
			<h3>PMID:11756248</h3>
				

				
			<p>
				作者：Yan W(1), Rajcan I.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Superior crop cultivars must be identified through multi-environment trials (MET) and on the basis of multiple traits. The objectives of this paper were to describe two types of biplots, the GGE biplot and the GT biplot, which graphically display genotype by environment data and genotype by trait data, respectively, and hence facilitate cultivar evaluation on the basis of <span class="gene">MET</span> data and multiple traits. Genotype main effect plus genotype by environment interaction effect (GGE) biplot analysis of the soybean [Glycine max (L.) Merr.] <span class="trait">yield</span> data for the 2800 crop heat unit area of Ontario for <span class="gene">MET</span> in the period 1994-1999 revealed yearly crossover genotype by site interactions. The eastern Ontario site Winchester showed a different genotype response pattern from the three southwestern Ontario sites in four of the six years. The interactions were not large enough to divide the area into different mega-environments as when analyzed over years, a single cultivar yielded the best in all four sites. The southwestern site, St. Pauls, was found to always group together with at least one of the other three sites; it did not provide unique information on genotype performance. Therefore, in future cultivar evaluations, Winchester should always be used but St. Pauls can be dismissed. Applying GT biplot to the 1994-1999 multiple trait data illustrated that GT biplots graphically displayed the interrelationships among <span class="trait">seed</span> <span class="trait">yield,</span> oil content, protein content, plant <span class="trait">height,</span> and days to maturity, among other traits, and facilitated visual cultivar comparisons and selection. It was found that selection for <span class="trait">seed</span> <span class="trait">yield</span> alone was not only the simplest, but also the most effective strategy in the early stages of soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Analysis of Soybean LATERAL ORGAN BOUNDARIES Domain-Containing Genes: A Functional Investigation of <span class="gene">GmLBD12.</span></h2>
			<h3>PMID:28464070</h3>
				

				
			<p>
				作者：Yang H, Shi G, Du H, Wang H, Zhang Z, Hu D, Wang J, Huang F, Yu D.
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant-specific () genes play critical roles in various plant growth and development processes. However, the number and characteristics of genes in soybean [ (L.) Merr.] remain unknown. Here, we identified 90 homologous genes in the soybean genome that phylogenetically clustered into two classes (I and II). The majority of the genes were evenly distributed across all 20 soybean chromosomes, and 77 (81.11%) of them were detected in segmental duplicated regions. Furthermore, the exon-intron organization and motif composition for each were analyzed. A close phylogenetic relationship was identified between the soybean genes and 41 previously reported genes of different plants in the same group, providing insights into their putative functions. Expression analysis indicated that more than half of the genes were expressed, with the two gene classes showing differential tissue expression characteristics; in addition, they were differentially induced by biotic and abiotic stresses. To further explore the functions of genes in soybean, was selected for functional characterization. <span class="gene">GmLBD12</span> was mainly localized to the nucleus and showed high expression in root and <span class="trait">seed</span> tissues. Overexpressing in (L.) Heynh resulted in increases in lateral root (LR) number and plant <span class="trait">height.</span> Quantitative real-time polymerase chain reaction (qRT-PCR) analysis demonstrated that was induced by <span class="trait">drought,</span> salt, cold, indole acetic acid (IAA), abscisic acid (ABA), and salicylic acid SA treatments. This study provides the first comprehensive analysis of the soybean gene family and a valuable foundation for future functional studies of genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean <span class="trait">seed</span> oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased <span class="trait">seed</span> <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] <span class="trait">seed</span> oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as <span class="trait">seed</span> <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between <span class="trait">seed</span> oil concentration and important agronomic and <span class="trait">seed</span> quality traits, including <span class="trait">seed</span> <span class="trait">yield,</span> 100-seed weight, protein concentration, plant <span class="trait">height,</span> and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with <span class="trait">seed</span> oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for protein concentration, four for <span class="trait">seed</span> <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant <span class="trait">height.</span> The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with <span class="trait">seed</span> protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with <span class="trait">seed</span> <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with <span class="trait">seed</span> oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for protein concentration, and one for plant <span class="trait">height.</span> The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as <span class="trait">seed</span> <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher <span class="trait">seed</span> <span class="trait">yield</span> and oil concentration or higher <span class="trait">seed</span> protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="gene">GmAOX2b</span> antisense gene compromises vegetative growth and <span class="trait">seed</span> production in soybean.</h2>
			<h3>PMID:22307678  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The alternative oxidase mediates the cyanide-resistant respiratory pathway in plant mitochondria. In non-thermogenic plants, the role of alternative oxidase in plant growth and development is not well understood. Soybean (Glycine max) lines carrying a <span class="gene">GmAOX2b</span> antisense gene had compromised vegetative growth and reproductive performance under typical glasshouse growth conditions. The reduction in vegetative growth was demonstrated by reduction in shoot <span class="trait">height,</span> the number of leaves per plant and the green leaf area. Antisense plants also had decreased pod formation and <span class="trait">seed</span> to pod ratios, which together led to a reduction in the number and total mass of <span class="trait">seed</span> produced. The negative effects of the antisense gene on pod set, <span class="trait">seed</span> set, ovule availability and total <span class="trait">seed</span> mass were primarily confined to the branches, rather than the main stem. The preferential effect of alternative oxidase suppression in the branches is discussed in relation to the reproductive potential of soybean under stress. Taken together, these results demonstrate that alternative oxidase provides the benefit of sustaining plant vegetative growth and reproductive capacity in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotyping-by-Sequencing-Based Investigation of the Genetic Architecture Responsible for a ∼Sevenfold Increase in Soybean <span class="trait">Seed</span> Stearic Acid.</h2>
			<h3>PMID:27866151  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Heim CB(1), Gillman JD(2)(3).
				<br/> 
				年份：2017 Jan 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean oil is highly unsaturated but oxidatively unstable, rendering it nonideal for food applications. Until recently, the majority of soybean oil underwent partial chemical hydrogenation, which produces trans fats as an unavoidable consequence. Dietary intake of trans fats and most saturated fats are conclusively linked to negative impacts on cholesterol levels and cardiovascular health. Two major soybean oil breeding targets are: (1) to reduce or eliminate the need for chemical hydrogenation, and (2) to replace the functional properties of partially hydrogenated soybean oil. One potential solution is the elevation of <span class="trait">seed</span> stearic acid, a saturated fat which has no negative impacts on cardiovascular health, from 3 to 4% in typical cultivars to &gt; 20% of the <span class="trait">seed</span> oil. We performed QTL analysis of a population developed by crossing two mutant lines, one with a missense mutation affecting a stearoyl-acyl-carrier protein desaturase gene resulting in ∼11% <span class="trait">seed</span> stearic acid crossed to another mutant, A6, which has 24-28% <span class="trait">seed</span> stearic acid. Genotyping-by-sequencing (GBS)-based QTL mapping identified 21 minor and major effect QTL for six <span class="trait">seed</span> oil related traits and plant <span class="trait">height.</span> The inheritance of a large genomic deletion affecting chromosome 14 is the basis for largest effect QTL, resulting in ∼18% <span class="trait">seed</span> stearic acid. This deletion contains <span class="gene">SACPD-C</span> and another gene(s); loss of both genes boosts <span class="trait">seed</span> stearic acid levels to ≥ 18%. Unfortunately, this genomic deletion has been shown in previous studies to be inextricably correlated with reduced <span class="trait">seed</span> <span class="trait">yield.</span> Our results will help inform and guide ongoing breeding efforts to improve soybean oil oxidative stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of <span class="trait">seed</span> <span class="trait">yield</span> and agronomic characters by photoperiod sensitivity and growth habit genes in soybean.</h2>
			<h3>PMID:20012856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cober ER(1), Morrison MJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes are adapted to narrow bands of latitude due to photoperiod sensitivity. There are several photoperiod-sensitive loci (E1, E2, E3, E4, E5, E6, E7, E8). Determinate and indeterminate growth habits are controlled by a single locus. The objective of our research was to examine the effects of photoperiod sensitivity and growth habit alleles on <span class="trait">seed</span> <span class="trait">yield</span> and other agronomic characters using isogenic lines. Twenty 'Harosoy' isolines with 11 photoperiod-sensitive genotypes many with both indeterminate and determinate growth habits were grown in the field at Ottawa, ON, from 2003 to 2007. Maturity ranged from 97 to 127 days, and <span class="trait">seed</span> <span class="trait">yield</span> increased linearly with maturity until about 112 days when it plateaued. Determinate lines were always shorter than indeterminate lines of equivalent maturity. <span class="trait">Seed</span> <span class="trait">yield</span> was associated with plant <span class="trait">height,</span> maturity, <span class="trait">seed</span> sugar concentration, <span class="trait">seed</span> weight and <span class="trait">lodging.</span> Effects of alleles at individual loci, and additive and epistatic effects across multiple loci were examined. At a single locus, photoperiod-insensitive alleles produced isolines that matured 8-11 days earlier, yielded less, and had shorter plants with reduced <span class="trait">lodging.</span> In multiple loci analyses, additive effects explained most of the variation in agronomic characters since additive models with E1, E3, E4, E7 and <span class="gene">Dt1</span> loci included compared well to additive plus epistatic models and genotype-based models. Variation in photoperiod sensitivity and growth habit alleles results in a range of maturity, with pleiotropic effects on <span class="trait">seed</span> <span class="trait">yield</span> and agronomic characteristics, and play an important role in providing adaptation across latitudes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant <span class="trait">height,</span> number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being <span class="trait">seed</span> oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant <span class="trait">height,</span> <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>