<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seeds per plant</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seeds per plant</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmFDL19</span> enhances tolerance to <span class="trait">drought</span> and salt stresses in soybean.</h2>
			<h3>PMID:28640834  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li Y(1)(2), Chen Q(2), Nan H(2), Li X(2)(3), Lu S(2)(4), Zhao X(2)(4), Liu B(2)(4), Guo C(1), Kong F(2)(4), Cao D(2).
				<br/> 
				年份：2017 Jun 22
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The basic leucine zipper (bZIP) family of transcription factors plays an important role in the growth and developmental process as well as responds to various abiotic stresses, such as <span class="trait">drought</span> and high salinity. Our previous work identified <span class="gene">GmFDL19,</span> a bZIP transcription factor, as a flowering promoter in soybean, and the overexpression of <span class="gene">GmFDL19</span> caused early flowering in transgenic soybean plants. Here, we report that <span class="gene">GmFDL19</span> also enhances tolerance to <span class="trait">drought</span> and salt stress in soybean. <span class="gene">GmFDL19</span> was determined to be a group A member, and its transcription expression was highly induced by abscisic acid (ABA), polyethylene glycol (PEG 6000) and high salt stresses. Overexpression of <span class="gene">GmFDL19</span> in soybean enhanced <span class="trait">drought</span> and salt tolerance at the seedling stage. The relative <span class="trait">plant</span> height (RPH) and relative shoot dry weight (RSDW) of transgenic plants were significantly higher than those of the WT after PEG and salt treatments. In addition, the germination rate and <span class="trait">plant</span> height of the transgenic soybean were also significantly higher than that of WT plants after various salt treatments. Furthermore, we also found that <span class="gene">GmFDL19</span> could reduce the accumulation of Na+ ion content and up-regulate the expression of several ABA/stress-responsive genes in transgenic soybean. We also found that <span class="gene">GmFDL19</span> overexpression increased the activities of several antioxidative enzyme and chlorophyll content but reduced malondialdehyde content. These results suggested that <span class="gene">GmFDL19</span> is involved in soybean abiotic stress responses and has potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of Polyhydroxybutyrate Synthesis in the Soil Bacterium Bradyrhizobium diazoefficiens.</h2>
			<h3>PMID:27208130  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Quelas JI(1), Mesa S(2), Mongiardini EJ(1), Jendrossek D(3), Lodeiro AR(4).
				<br/> 
				年份：2016 Jun 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polyhydroxybutyrate (PHB) is a carbon and energy reserve polymer in various prokaryotic species. We determined that, when grown with mannitol as the sole carbon source, Bradyrhizobium diazoefficiens produces a homopolymer composed only of 3-hydroxybutyrate units (PHB). Conditions of oxygen limitation (such as microoxia, oxic stationary phase, and bacteroids inside legume nodules) were permissive for the synthesis of PHB, which was observed as cytoplasmic granules. To study the regulation of PHB synthesis, we generated mutations in the regulator gene <span class="gene">phaR</span> and the phasin genes phaP1 and phaP4 Under permissive conditions, mutation of <span class="gene">phaR</span> impaired PHB accumulation, and a phaP1 phaP4 double mutant produced more PHB than the wild type, which was accumulated in a single, large cytoplasmic granule. Moreover, PhaR negatively regulated the expression of phaP1 and phaP4 as well as the expression of phaA1 and phaA2 (encoding a 3-ketoacyl coenzyme A [CoA] thiolases), phaC1 and phaC2 (encoding PHB synthases), and <span class="gene">fixK2</span> (encoding a cyclic AMP receptor protein [CRP]/fumarate and nitrate reductase regulator [FNR]-type transcription factor of genes for microoxic lifestyle). In addition to the depressed PHB cycling, <span class="gene">phaR</span> mutants accumulated more extracellular polysaccharides and promoted higher <span class="trait">plant</span> shoot dry weight and competitiveness for nodulation than the wild type, in contrast to the phaC1 mutant strain, which is defective in PHB synthesis. These results suggest that <span class="gene">phaR</span> not only regulates PHB granule formation by controlling the expression of phasins and biosynthetic enzymes but also acts as a global regulator of excess carbon allocation and symbiosis by controlling <span class="gene">fixK2</span> IMPORTANCE: In this work, we investigated the regulation of polyhydroxybutyrate synthesis in the soybean-nodulating bacterium Bradyrhizobium diazoefficiens and its influence in bacterial free-living and symbiotic lifestyles. We uncovered a new interplay between the synthesis of this carbon reserve polymer and the network responsible for microoxic metabolism through the interaction between the gene regulators <span class="gene">phaR</span> and <span class="gene">fixK2</span> These results contribute to the understanding of the physiological conditions required for polyhydroxybutyrate biosynthesis. The interaction between these two main metabolic pathways is also reflected in the symbiotic phenotypes of soybeans inoculated with <span class="gene">phaR</span> mutants, which were more competitive for nodulation and enhanced dry matter production by the plants. Therefore, this knowledge may be applied to the development of superior strains to be used as improved inoculants for soybean crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmPAP4, a novel purple acid phosphatase gene isolated from soybean (Glycine max), enhanced extracellular phytate utilization in Arabidopsis thaliana.</h2>
			<h3>PMID:24595918  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kong Y(1), Li X, Ma J, Li W, Yan G, Zhang C.
				<br/> 
				年份：2014 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: <span class="gene">GmPAP4</span> , a novel <span class="trait">plant</span> <span class="gene">PAP</span> gene in soybean, has phytase activity. Over-expressing <span class="gene">GmPAP4</span> can enhance Arabidopsis growth when phytate is the sole P source in culture. Phosphorus (P) is an important macronutrient for <span class="trait">plant</span> growth and development. However, most of the total P in soils is fixed into organic phosphate (Po). Purple acid phosphatase (PAP) can hydrolyze Po in the soil to liberate inorganic phosphate and enhance <span class="trait">plant</span> P utilization. We isolated a novel <span class="gene">PAP</span> gene, <span class="gene">GmPAP4,</span> from soybean (Glycine max). It had an open reading frame of 1,329 bp, encoding 442 amino acid residues. Sequence alignment and phylogenetics analysis indicated that <span class="gene">GmPAP4</span> was similar to other <span class="trait">plant</span> PAPs with large molecular masses. Quantitative real-time PCR analysis showed that the induced expression of <span class="gene">GmPAP4</span> was greater in P-efficient genotype Zhonghuang15 (ZH15) than in P-inefficient genotype Niumaohuang (NMH) during the periods of flowering (28-35 days post phytate stress; DPP) and pod formation (49-63 DPP). Moreover, peak expression, at 63 DPP, was about 3-fold higher in 'ZH15' than in 'NMH'. Sub-cellular localization showed that <span class="gene">GmPAP4</span> might be on plasma membrane or in cytoplasm. Over-expressing <span class="gene">GmPAP4</span> in Arabidopsis resulted in significant rises in P acquisition and utilization compared with the wild-type (WT). Under phytate condition, transgenic Arabidopsis plants showed increases of approximately 132.7 % in dry weight and 162.6 % in shoot P content compared with the WT. Furthermore, when phytate was added as the sole P source in cultures, the activity of acid phosphatase was significantly higher in transgenic plants. Therefore, <span class="gene">GmPAP4</span> is a novel <span class="gene">PAP</span> gene that functions in plant's utilization of organic phosphate especially under phytate condition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Symbiotic effectiveness of inoculation with Bradyrhizobium isolates on soybean [Glycine max (L.) Merrill] genotypes with different maturities.</h2>
			<h3>PMID:26034703</h3>
				

				
			<p>
				作者：Argaw A(1).
				<br/> 
				年份：2014 Dec 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The influence of soybean genotypes with different maturity groups on the symbiotic effectiveness of Bradyrhizobium <span class="gene">spp</span> under high native soil N is not well known. Therefore, the objective of this work was to evaluate the influence of maturity time of soybean genotypes on the symbiotic effectiveness of Bradyrhizobium spp. at higher soil N. Three isolates of Bradyrhizobium spp. (UK-isolate, TAL-379 isolate and local-isolate) and six soybean genotypes, three late maturing (Wogayen, TGx-1336424 and Belsa) and three medium maturing (GIZA, Afgat and Gishame) were used for greenhouse experiment. Only GIZA and TGx-1336424 were selected for field experiment. The result of the experiments showed that significantly (P &lt; 0.05) differences in all investigated traits, except total <span class="trait">plant</span> tissue N, was observed in TGx-1336424 with UK-Isolate and Local-Isolate.TAL-379 inoculation performed better in all investigated traits of GIZA genotype than other inoculation treatments. N-fertilization in the greenhouse experiment significantly (P &lt; 0.05) improved the shoot biomass of Wogayen and Belsa-95, but did not observe in GIZA and Gishame. The regression analysis obtained between nodule number and nodule dry weight with that of grain <span class="trait">yield</span> indicated generally higher R(2) value for late maturing than that of the medium maturing genotypes. This indicates high importance of nodulation for improving the GY of late maturing genotypes. Hence, this study proves the need for inoculation to improve the production and productivity of soybean sustainably in Ethiopia, with particularly pronounced effect on late maturing genotypes of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Physiological responses of glyphosate-resistant and glyphosate-sensitive soybean to aminomethylphosphonic acid, a metabolite of glyphosate.</h2>
			<h3>PMID:21190714  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ding W(1), Reddy KN, Zablotowicz RM, Bellaloui N, Arnold Bruns H.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aminomethylphosphonic acid (AMPA) is formed in glyphosate-treated glyphosate-resistant (GR) and glyphosate-sensitive (GS) soybean [Glycine max (L.) Merr.] plants and is known to cause yellowing in soybean. Although, AMPA is less phytotoxic than glyphosate, its mode of action is different from that of glyphosate and is still unknown. Greenhouse studies were conducted at Stoneville, <span class="gene">MS</span> to determine the effects of AMPA on <span class="trait">plant</span> growth, chlorophyll content, photosynthesis, nodulation, nitrogenase activity, nitrate reductase activity, and shoot nitrogen content in <span class="gene">GR</span> and <span class="gene">GS</span> soybeans. AMPA was applied to one- to two-trifoliolate leaf stage soybeans at 0.1 and 1.0 kg ha(-1), representing a scenario of 10% and 100% degradation of glyphosate (1.0 kg ae ha(-1) use rate) to AMPA, respectively. Overall, AMPA effects were more pronounced at 1.0 kg ha(-1) than at 0.1 kg ha(-1) rate. Visual <span class="trait">plant</span> injury (18-27%) was observed on young leaves within 3d after treatment (DAT) with AMPA at the higher rate regardless of soybean type. AMPA injury peaked to 46-49% at 14 DAT and decreased to 17-18% by 28 DAT, in both soybean types. AMPA reduced the chlorophyll content by 37%, 48%, 66%, and 23% in <span class="gene">GR</span> soybean, and 17%, 48%, 57%, and 22% in <span class="gene">GS</span> soybean at 3, 7, 14, and 28 DAT, respectively. AMPA reduced the photosynthesis rate by 65%, 85%, and 77% in <span class="gene">GR</span> soybean and 59%, 88%, and 69% in <span class="gene">GS</span> soybean at 3, 7, and 14 DAT, respectively, compared to non-treated plants. Similarly, AMPA reduced stomatal conductance to water vapor and transpiration rates at 3, 7, and 14 DAT compared to non-treated plants in both soybean types. Photosynthesis rate, stomatal conductance, and transpiration rate recovered to the levels of non-treated plants by 28 DAT. <span class="trait">Plant</span> height and shoot dry weight at 28 DAT; nodulation, nitrogenase activity at 10 DAT, and nitrate reductase activity at 3 and 14 DAT were unaffected by AMPA. AMPA reduced root respiration and shoot nitrogen content at 10 DAT. These results suggest that a foliar application of AMPA could indirectly reduce photosynthesis through decreased chlorophyll content in <span class="gene">GR</span> and <span class="gene">GS</span> soybean up to 14 DAT, but affected plants can recover to normal growth by 28 DAT.
					</p>
				
					<p>&nbsp;&nbsp;
						Published by Elsevier Ltd.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping quantitative trait loci associated with aluminum toxin tolerance in NJRIKY recombinant inbred line population of soybean (Glycine max).</h2>
			<h3>PMID:18844777  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Qi B(1), Korir P, Zhao T, Yu D, Chen S, Gai J.
				<br/> 
				年份：2008 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the genetic mechanism of Al-tolerance in soybean, a recombinant inbred line population (RIL) with 184 F(2:7:11) lines derived from the cross of Kefeng No.1 x Nannong 1138-2 (Al-tolerant x Al-sensitive) were tested in pot experiment with sand culture medium in net room in Nanjing. Four traits, i.e. <span class="trait">plant</span> height, number of leaves, shoot dry weight and root dry weight at seedling stage, were evaluated and used to calculate the average membership index (FAi) as the indicator of Al-tolerance. The composite interval mapping (CIM) under WinQTL Cartographer v. 2.5 detected five QTLs (i.e. qFAi-1, qFAi-2, qFAi-3, qFAi-4 and qFAi-5), explaining 5.20%-9.07% of the total phenotypic variation individually. While with the multiple interval mapping (MIM) of the same software, five QTLs (qFAi-1, qFAi-5, qFAi-6, qFAi-7, and qFAi-8) explaining 5.7%-24.60% of the total phenotypic variation individually were mapped. Here qFAi-1 and qFAi-5 were detected by both CIM and MIM with the locations in a same flanking marker region, <span class="gene">GMKF046-GMKF080</span> on B1 and satt278-sat_95 on L, respectively. While qFAi-2 under CIM and qFAi-6 under MIM both on D1b2 were located in neighboring regions with their confidence intervals overlapped and might be the same locus. Segregation analysis under major gene plus polygene inheritance model showed that Al-tolerance was controlled by two major genes (h(2) (mg) = 33.05%) plus polygenes (h(2) (pg) = 52.73%). Both QTL mapping and segregation analysis confirmed two QTLs responsible for Al-tolerance with relatively low heritability, and there might be a third QTL, confounded with the polygenes in segregation analysis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:NAD-Malic Enzyme Affects Nitrogen Fixing Activity of Bradyrhizobium japonicum USDA 110 Bacteroids in Soybean Nodules.</h2>
			<h3>PMID:21558711</h3>
				

				
			<p>
				作者：Dao TV(1), Nomura M, Hamaguchi R, Kato K, Itakura M, Minamisawa K, Sinsuwongwat S, Le HT, Kaneko T, Tabata S, Tajima S.
				<br/> 
				年份：2008
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The NAD(+)-dependent malic enzyme (DME) has been reported to play a key role supporting nitrogenase activity in bacteroids of Sinorhizobium meliloti. Genetic evidence for a similar role in Bradyrhizobium japonicum USDA110 was obtained by constructing a <span class="gene">dme</span> mutant. Soybean plants inoculated with a <span class="gene">dme</span> mutant did not show delayed nodulation, but formed small root nodules and exhibited significant nitrogen-deficiency symptoms. Nodule numbers and the acetylene reducting activity <span class="trait">per</span> nodule as a dry weight value 14 and 28 days after inoculation with the <span class="gene">dme</span> mutant were comparable to those of plants inoculated with wild-type B. japonicum. However, shoot dry weight and acetylene reducting activity <span class="trait">per</span> nodule decreased to ca. 30% of the values in plants with wild-type B. japonicum. The sucrose and organic acid (malate, succinate, acetate, α-ketoglutarate and lactate) contents of the nodules were investigated. Amounts of sucrose, malate and a-ketoglutarate increased on inoculation with the <span class="gene">dme</span> mutant, suggesting that the decreased DME and nitrogenase activities in the bacteroids resulted in a reduction in the consumption of these respiratory metabolites by the nodules. The data suggest that the DME activity of B. japonicum bacteroids plays a role in nodule metabolism and supports nitrogen fixation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glyphosate-resistant and -susceptible soybean (Glycine max) and canola (Brassica napus) dose response and metabolism relationships with glyphosate.</h2>
			<h3>PMID:17417871  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Nandula VK(1), Reddy KN, Rimando AM, Duke SO, Poston DH.
				<br/> 
				年份：2007 May 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Experiments were conducted to determine (1) dose response of glyphosate-resistant (GR) and -susceptible (non-GR) soybean [Glycine max (L.) Merr.] and canola (Brassica napus L.) to glyphosate, (2) if differential metabolism of glyphosate to aminomethyl phosphonic acid (AMPA) is the underlying mechanism for differential <span class="trait">resistance</span> to glyphosate among <span class="gene">GR</span> soybean varieties, and (3) the extent of metabolism of glyphosate to AMPA in <span class="gene">GR</span> canola and to correlate metabolism to injury from AMPA. GR50 (glyphosate dose required to cause a 50% reduction in <span class="trait">plant</span> dry weight) values for <span class="gene">GR</span> (Asgrow 4603RR) and non-GR (HBKC 5025) soybean were 22.8 kg ae ha-1 and 0.47 kg ha-1, respectively, with <span class="gene">GR</span> soybean exhibiting a 49-fold level of <span class="trait">resistance</span> to glyphosate as compared to non-GR soybean. Differential reduction in chlorophyll by glyphosate was observed between <span class="gene">GR</span> soybean varieties, but there were no differences in shoot fresh weight reduction. No significant differences were found between <span class="gene">GR</span> varieties in metabolism of glyphosate to AMPA, and in shikimate levels. These results indicate that <span class="gene">GR</span> soybean varieties were able to outgrow the initial injury from glyphosate, which was previously caused at least in part by AMPA. GR50 values for <span class="gene">GR</span> (Hyola 514RR) and non-GR (Hyola 440) canola were 14.1 and 0.30 kg ha-1, respectively, with <span class="gene">GR</span> canola exhibiting a 47-fold level of <span class="trait">resistance</span> to glyphosate when compared to non-GR canola. Glyphosate did not cause reduction in chlorophyll content and shoot fresh weight in <span class="gene">GR</span> canola, unlike <span class="gene">GR</span> soybean. Less glyphosate <span class="trait">(per</span> unit leaf weight) was recovered in glyphosate-treated <span class="gene">GR</span> canola as compared to glyphosate-treated <span class="gene">GR</span> soybean. External application of AMPA caused similar injury in both <span class="gene">GR</span> and non-GR canola. The presence of a bacterial glyphosate oxidoreductase gene in <span class="gene">GR</span> canola contributes to breakdown of glyphosate to AMPA. However, the AMPA from glyphosate breakdown could have been metabolized to nonphytotoxic metabolites before causing injury to <span class="gene">GR</span> canola. Injury in <span class="gene">GR</span> and non-GR canola from exogenous application of AMPA was similar.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effect of composted textile sludge on growth, nodulation and nitrogen fixation of soybean and cowpea.</h2>
			<h3>PMID:16782329  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Araújo AS(1), Monteiro RT, Carvalho EM.
				<br/> 
				年份：2007 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of composted textile sludge on growth, nodulation and nitrogen fixation of soybean and cowpea was evaluated in a greenhouse experiment. The compost was incorporated into soil at 0, 9.5, 19 and 38 t ha(-1) (bases upon the N requirement of the crops, i.e., 0, 50, 100 and 200 kg available N ha(-1)). Growth, nodulation and shoot accumulation of nitrogen were evaluated 36 and 63 days after <span class="trait">plant</span> emergence. Nodule glutamine synthetase (GS) activity and leghemoglobin content were evaluated 63 days after emergence. Composted textile sludge did not show negative effects on nodule number and weight, nodule <span class="gene">GS</span> activity and leghemoglobin content. Nitrogen accumulation in shoot dry matter in soybean and cowpea was higher than other treatments with application of 19 t ha(-1) of compost. Composting can be an alternate technology for the management of solid textile mill sludge. This study verifies that the composted textile sludge was not harmful to growth, nodulation and nitrogen fixation of soybean and cowpea.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[A study of the effects on the symbiotic nitrogen fixation of Sinorhizobium fredii with the introduction of dctABD and <span class="gene">nifA</span> genes].</h2>
			<h3>PMID:11902003  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li YG(1), Zhou JC.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A recombinant plasmid pHN307 containing C4-dicarboxylic acid transport genes (dctABD) from Sinorhizobium meliloti, <span class="gene">nifA</span> genes from Klebsiella pneumoniae and reporter genes luxAB from pDB30 was constructed by using pTR102 as the vector. The pHN307 was then introduced into the S. fredii HN01, YC4 and GR3 by tri-parental mating, and the stability of pHN307 in the transconjugants under free-living and symbiotic condition was also investigated. The results of <span class="trait">plant</span> pot experiment indicated that the introduction of pHN307 in the transconjugants could significantly increase the nodule fresh weight, shoot dry weight (biomass) and total nitrogen content of the symbiotic plants with soybean variety of Heilong 33. When the transconjugants were in symbiotic with soybean variety of Chuanzao No. 1, HN01 (pHN307) could significantly increase its root nodule number and fresh weight; GR3 (pHN307) could significantly increase its root nodule number, nodule fresh weight, shoot dry weight and total nitrogen content, but YC4(pHN307) demonstrated negative effect under the same condition. The results of this study suggested that the introduction of dctABD and <span class="gene">nifA</span> could improve the symbiotic nitrogen fixation efficiency and nodulation ability of the rhizobia tested, respectively, and its effect was relevant to the characteristics of recipient rhizobia and soybean variety.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A dual positional specific lipoxygenase functions in the generation of flavor compounds during climacteric ripening of apple</h2>
			<h3>PMID:26504564</h3>
				
					<h3>PMCID:4595979</h3>
					<h4>期刊:Horticulture Research</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Schiller </surname>  <given-names> Doreen </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Contreras </surname>  <given-names> Carolina </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vogt </surname>  <given-names> J&#x000f6;rg </given-names>  </name>  <xref ref-type="aff" rid="aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dunemann </surname>  <given-names> Frank </given-names>  </name>  <xref ref-type="aff" rid="aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Defilippi </surname>  <given-names> Bruno G </given-names>  </name>  <xref ref-type="aff" rid="aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Beaudry </surname>  <given-names> Randolph </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schwab </surname>  <given-names> Wilfried </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  <xref ref-type="corresp" rid="caf1"> * </xref>  </contrib>  <aff id="aff1">  <label> 1 </label>  <institution> Biotechnology of Natural Products, Technische Universit&#x000e4;t M&#x000fc;nchen </institution> , Liesel-Beckmann-Str. 1, D-85354 Freising,  <country> Germany </country>  </aff>  <aff id="aff2">  <label> 2 </label>  <institution> Michigan State University, Department of Horticulture </institution> , 1066 Bogue Street, East Lansing, MI 48824,  <country> USA </country>  </aff>  <aff id="aff3">  <label> 3 </label>  <institution> Julius K&#x000fc;hn-Institut (JKI) &#x02013; Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Fruit Crops, Pillnitzer Platz 3a </institution> , D-01326 Dresden,  <country> Germany </country>  </aff>  <aff id="aff4">  <label> 4 </label>  <institution> Julius K&#x000fc;hn-Institut (JKI) &#x02013; Federal Research Centre for Cultivated Plants, Institute for Breeding Research on Horticultural Crops </institution> , Erwin-Baur-Str. 27, D-06484 Quedlinburg,  <country> Germany </country>  </aff>  <aff id="aff5">  <label> 5 </label>  <institution> INIA La Platina, PO Box 439-9 </institution> , Santiago, Chile </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 01 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The formation of primary metabolites for inclusion into fruit esters occurs via several pathways. <sup> <xref ref-type="bibr" rid="bib6"> 6 </xref> </sup> Fatty acid degradation is thought to contribute to the formation of short chain fatty acids and aldehydes to <span class="trait">yield</span> straight-chain esters. <sup> <xref ref-type="bibr" rid="bib7 bib8 bib9"> 7&#x02013;9 </xref> </sup> The leucine/isoleucine/valine pathway provides precursors to branched-chain esters <sup> <xref ref-type="bibr" rid="bib9"> 9 </xref> </sup> and may contribute to the formation of short-chain fatty acids. <sup> <xref ref-type="bibr" rid="bib10"> 10 </xref> </sup> Saturated fatty acids are degraded by the enzymatic steps of &#x003b2;-oxidation whereas linoleic (LA) and linolenic acid (LnA) are either catabolized through &#x003b2;-oxidation or the lipoxygenase (LOX) pathway. The latter involves the sequential action of LOX, hydroperoxide lyase and alcohol dehydrogenase (ADH). <sup> <xref ref-type="bibr" rid="bib6"> 6 </xref> </sup> The resulting alcohols are linked to an acyl moiety by alcohol acyl-CoA transferases (AATs) to form esters. Unsaturated straight-chain esters appear to arise mainly by the <span class="gene">LOX</span> pathway in apple. <sup> <xref ref-type="bibr" rid="bib8"> 8 </xref> </sup>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">LOX</span> (linoleate oxygen oxidoreductase, EC 1.13.11.12) are non-heme iron-containing dioxygenases. They catalyse the regio- and stereospecific introduction of molecular dioxygen into polyunsaturated fatty acids, containing a <italic> Z </italic> , <italic> Z </italic> -1,4-pentadiene system such as LA and LnA to form hydroperoxy octadecadienoic acid (HpODE) and hydroperoxy octadecatrienoic acid, respectively. <sup> <xref ref-type="bibr" rid="bib11"> 11 </xref> </sup> <span class="trait">Plant</span> <span class="gene">LOX</span> are classified according to the position of oxygenation in the carbon backbone of LA. <sup> <xref ref-type="bibr" rid="bib12"> 12 </xref> </sup> This occurs either at carbon atom 9 (9-LOX) or 13 (13-LOX), forming the corresponding hydroperoxy derivatives, respectively ( <xref ref-type="fig" rid="fig1"> Figure 1 </xref> ). However, <span class="gene">LOX</span> enzymes are not perfectly specific and biocatalysts that produce more than 10% of the alternative regio-isomer are called dual positional specific LOX. <sup> <xref ref-type="bibr" rid="bib14"> 14 </xref> </sup> Both 9- and 13-hydroperoxides can be catabolized to aroma-active volatile aldehydes by hydroperoxide lyase. However, only 13-LOX activity contributes to the apple aroma due to the formation of precursors of C6 volatile compounds. <sup> <xref ref-type="bibr" rid="bib15"> 15 </xref> </sup> Alternatively, <italic> <span class="gene">LOX</span> </italic> genes are classified based on comparison of <italic> <span class="gene">LOX</span> </italic> primary structures. <sup> <xref ref-type="bibr" rid="bib13"> 13 </xref> </sup> Most <span class="trait">plant</span> <italic> <span class="gene">LOX</span> </italic> share high sequence similarity of about 70% and are therefore grouped into the so-called <italic> type-1 </italic> <span class="gene">LOX</span> class. Members of the <italic> type-2 </italic> <span class="gene">LOX</span> class show only moderate sequence similarity but exhibit a putative chloroplast transit peptide. <sup> <xref ref-type="bibr" rid="bib12"> 12 </xref> </sup> TomloxC, a chloroplast-targeted <span class="gene">LOX</span> from tomato, has been identified as key <span class="gene">LOX</span> specifically involved in the generation of fatty acid-derived C6 aldehydes and alcohols upon tissue disruption. <sup> <xref ref-type="bibr" rid="bib16"> 16 </xref> </sup>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Plant</span> LOXs form multi-gene families comprising varying numbers of genes. While five <italic> <span class="gene">LOX</span> </italic> genes have been cloned from the transcriptome of tomato, <sup> <xref ref-type="bibr" rid="bib16"> 16 </xref> </sup> up to 20 <italic> <span class="gene">LOX</span> </italic> s have been reported for soybean <sup> <xref ref-type="bibr" rid="bib17"> 17 </xref> </sup> and grapevine. <sup> <xref ref-type="bibr" rid="bib18"> 18 </xref> </sup> It has been postulated that <span class="trait">plant</span> <italic> <span class="gene">LOX</span> </italic> may be involved in <span class="trait">plant</span> growth, development, senescence, stress, defence and response to wounding. <sup> <xref ref-type="bibr" rid="bib13"> 13 </xref> </sup> <span class="gene">LOX</span> enzymes catalyze the initial reaction in the biosynthetic pathways of diverse defense molecules such as jasmonates and traumatin, and of volatile compounds such as hexanal, hexenal and hexenol ( <xref ref-type="fig" rid="fig1"> Figure 1 </xref> ). <italic> <span class="gene">LOX</span> </italic> genes with potential roles during fruit ripening have been described in tomato, <sup> <xref ref-type="bibr" rid="bib19"> 19 </xref> </sup> apricot, <sup> <xref ref-type="bibr" rid="bib20"> 20 </xref> </sup> peach <sup> <xref ref-type="bibr" rid="bib21"> 21 </xref> , <xref ref-type="bibr" rid="bib22"> 22 </xref> </sup> and kiwifruit. <sup> <xref ref-type="bibr" rid="bib23"> 23 </xref> </sup>
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">LOX</span> activity was determined by continuously monitoring the formation of conjugated dienes at 234&#x02005;nm. The standard assay mixture consisted of 500&#x02005;&#x000b5;L 50&#x02005;mM sodium phosphate buffer (pH&#x02005;7.5) and the appropriate amount of purified protein (0.2 to 2&#x02005;&#x000b5;g). Reactions were started by addition of 0.3&#x02005;mM substrate and incubated at 25&#x02005;&#x000b0;C with constant shaking for 1&#x02005;h. The reaction products were extracted with dichloromethane, reduced by addition of 400&#x02005;nmol triphenyl phosphine and evaporated to dryness under a constant flow of nitrogen. After resuspension in 30% (v/v) methanol samples were submitted to HPLC analysis. Reaction buffers were prepared with 0.5 pH unit increments as follows: 50&#x02005;mM sodium citrate buffer (pH&#x02005;3.0&#x02013;6.0), 50&#x02005;mM sodium phosphate buffer (pH&#x02005;6.0&#x02013;8.0) and 50&#x02005;mM Tris HCl buffer (pH&#x02005;8.0&#x02013;10.0). For determination of the pH optimum of the recombinant proteins, the reaction was run across a pH range of 3.0&#x02013;10.0 at 25&#x02005;&#x000b0;C. The optimum temperature was determined by assaying <span class="gene">LOX</span> activity rates at temperatures of 5&#x02013;55&#x02005;&#x000b0;C with 5&#x02005;&#x000b0;C increments at a pH of 7.0. Kinetic properties of recombinant <span class="gene">LOX</span> were determined by monitoring hydroperoxide formation across a range of concentrations of LA and LnA with a Nicolet evolution 100 spectrophotometer (Thermo, Cambridge, UK). The standard assay was performed at 45&#x02005;&#x000b0;C with 0&#x02013;100&#x02005;&#x000b5;M substrate in 50&#x02005;mM sodium citrate buffer (pH&#x02005;6.0) for LOX1:Md:1a and Gly567Ala or 50&#x02005;mM sodium phosphate buffer (pH&#x02005;7.5) for LOX1:Md:1c, respectively. Each reaction was run in triplicate. The first order kinetic rates were estimated as changes in absorbance at 234&#x02005;nm s <sup> &#x02212;1 </sup> and converted into &#x000b5;mol min <sup> &#x02212;1 </sup> mg <sup> &#x02212;1 </sup> , using the extinction coefficient <italic> e </italic> =23 000&#x02005;M <sup> &#x02212;1 </sup> cm <sup> &#x02212;1 </sup> for LA and LnA hydroperoxides. One unit of <span class="gene">LOX</span> activity is defined as the amount of enzyme catalysing the formation of 1&#x02005;&#x000b5;mol of hydroperoxide <span class="trait">per</span> minute under optimal reaction conditions. The data were then fitted using Michaelis&#x02013;Menten equation to determine the kinetic parameters for all enzymes.
					</p>
				
					<p>&nbsp;&nbsp;
						Subcellular localization of deduced <span class="gene">LOX</span> proteins was predicted through the identification of signal peptide sequences using several prediction programs such as ProtComp ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.softberry.com/berry.phtml"> http://www.softberry.com/berry.phtml </ext-link> ), PSORT (psort.nibb.ac.jp/). <sup> <xref ref-type="bibr" rid="bib36"> 36 </xref> </sup> TargetP ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/TargetP/"> http://www.cbs.dtu.dk/services/TargetP/ </ext-link> ), ChloroP ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/ChloroP/"> http://www.cbs.dtu.dk/services/ChloroP/ </ext-link> ), Signal-3L ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://www.csbio.sjtu.edu.cn/bioinf/Signal-3L/"> http://www.csbio.sjtu.edu.cn/bioinf/Signal-3L/ </ext-link> ) and Predotar ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="http://urgi.versailles.inra.fr/predotar/predotar.html"> http://urgi.versailles.inra.fr/predotar/predotar.html </ext-link> ). <sup> <xref ref-type="bibr" rid="bib37"> 37 </xref> </sup> Chimeric YFP constructs were produced for <italic> MdLOX2a </italic> , <italic> MdLOX4a </italic> and <italic> MDLOX6b </italic> in the 13-LOX group and <italic> MdLOX1a </italic> , <italic> MdLOX7a </italic> and <italic> MdLOX7c </italic> in the 9-LOX group. All were designed to express the first 166&#x02013;300 amino-acid residues of the protein including, where present, the putative chloroplast transit peptide. <sup> <xref ref-type="bibr" rid="bib38"> 38 </xref> </sup> The YFP sequence was fused to the C-terminus of each <span class="gene">LOX</span> sequence. Phusion <sup> &#x000ae; </sup> High-Fidelity DNA polymerase (New England Biolabs, Arlington, VA, USA) was used for PCR and cloning applications. Directional cloning was performed using the P-ENTR/D-TOPO entry vector kit (Life Technologies, Carlsbad, CA, USA). After sequencing, purified plasmids were digested with <italic> Mlu </italic> -I restriction enzyme (New England Biolabs) and a LR clonase recombination reaction between the entry clone and the pEarleyGate 101destination vector (35S-Gateway-YFP-HA tag-OCS 3&#x02032;, ABRC stock no. CD3-683 at The Ohio State University) <sup> <xref ref-type="bibr" rid="bib39"> 39 </xref> </sup> was performed as described by the manufacturer (Life Technologies). Tobacco ( <italic> Nicotiana tabacum </italic> cv Petit havana) plants, grown for five weeks at 22&#x02005;&#x000b0;C in a greenhouse, were used for <italic> Agrobacterium tumefaciens </italic> -mediated transient expression. Briefly, each expression vector was introduced into <italic> Agrobacterium tumefaciens </italic> strain GV3101 by heat shock. A single transformed colony was used to inoculate 3&#x02005;mL of LB medium containing kanamycin (50&#x02005;&#x000b5;g mL <sup> &#x02212;1 </sup> ), gentamycin (50&#x02005;&#x000b5;g mL <sup> &#x02212;1 </sup> ) and rifampicin (10&#x02005;&#x000b5;g mL <sup> &#x02212;1 </sup> ). The bacterial culture was incubated at 28&#x02005;&#x000b0;C with agitation for 16&#x02005;h. An aliquot (0.2&#x02005;mL) of bacterial culture then was pelleted in a 1.5-mL Eppendorf tube by centrifugation at 4000 <italic> g </italic> <sub> n </sub> for 5&#x02005;min at room temperature. The pellet was resuspended in 0.5&#x02005;mL of infiltration medium (50&#x02005;mM MES, pH&#x02005;5.6, 2&#x02005;mM NaH <sub> 2 </sub> PO <sub> 4 </sub> , 200&#x02005;&#x000b5;M aceto-syringone and 5&#x02005;mg mL <sup> &#x02212;1 </sup> glucose). The bacterial suspension was diluted with infiltration buffer to adjust the inoculum concentration to a final OD <sub> 600 </sub> value range of 0.05&#x02013;0.1. The bacterial suspension was infiltrated using a 1-mL syringe without a needle by gentle pressure through the stomata on the abaxial surface. Transformed plants then were incubated under normal growth conditions for 72&#x02005;h. <sup> <xref ref-type="bibr" rid="bib40"> 40 </xref> , <xref ref-type="bibr" rid="bib41"> 41 </xref> </sup> Confocal imaging was performed using an inverted laser scanning microscope (IX81 model FV1000; Olympus, Melville, NY, USA) with &#x000d7;20 and &#x000d7;40 air objectives. Excised <span class="trait">plant</span> tissues were mounted in water between a microscope slide and cover slip and examined. To detect EYFP fluorescence, an argon ion laser was used to produce excitation at a wavelength of 488&#x02005;nm and florescence was detected in the range of 505&#x02013;530&#x02005;nm. Chlorophyll fluorescence was detected in the range of 670&#x02013;730&#x02005;nm using an excitation laser with a wavelength of 633&#x02005;nm. All images were acquired from single optical sections. Post-acquisition image processing was done using Olympus Fluoview FV1000 software.
					</p>
				
					<p>&nbsp;&nbsp;
						The oligonucleotides for amplification of <italic> MdLOX1 </italic> and <italic> MdLOX2 </italic> genes were designed based on manually curated <italic> Malus </italic> &#x000d7; <italic> domestica </italic> predicted protein sequences. <sup> <xref ref-type="bibr" rid="bib24"> 24 </xref> </sup> The <italic> MdLOX1 </italic> gene cluster comprises five putative functional <italic> <span class="gene">LOX</span> </italic> sequences, whereas the <italic> MdLOX2 </italic> gene cluster seemed to consist of only two <span class="gene">LOX</span> isoforms. Six candidate genes ( <italic> MdLOX1a </italic> , <italic> MdLOX1b </italic> , <italic> MdLOX1c </italic> , <italic> MdLOX1d </italic> , <italic> MdLOX2a </italic> and <italic> MdLOX2b </italic> ) contained all functional motifs of <span class="trait">plant</span> <span class="gene">LOX</span> enzymes and were chosen for primer design ( <xref ref-type="supplementary-material" rid="xob1"> Supplementary Table S1 </xref> ). The bioinformatically predicted open reading frame of <italic> MdLOX1e </italic> (MDP0000312394) exhibited large sequence deletions and lacked the stop codon immediately downstream of the highly conserved C-terminal motif. <sup> <xref ref-type="bibr" rid="bib42"> 42 </xref> </sup> Also, manual evaluation of the Golden Delicious genome did not succeed in a functional full-length <italic> <span class="gene">LOX</span> </italic> sequence for <italic> MdLOX1e </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Additionally, we tested for substrate conversion of AA by all four apple LOXs. Arachidonic acid is of minor physiological importance in plants and is therefore often a poor substrate for <span class="trait">plant</span> LOXs. Even so, all wild-type apple <span class="gene">LOX</span> enzymes were able to convert AA at least to some extent to hydroperoxy eicosatetraenoic acid (HpETE). LOX2:Md:2b showed highest product specificity as it formed solely 15-HpETE ( <xref ref-type="supplementary-material" rid="xob1"> Supplementary Table S4 </xref> ). LOX1:Md:1a and LOX2:Md:2a produced considerable amounts of 15-HpETE (42%&#x02013;45%), but also 12-HpETE (16%&#x02013;17%), 9-HpETE (12%&#x02013;16%) and small amounts of 11-, 8- and 5-HpETE from AA. The major AA products formed by LOX1:Md:1c was 11-HpETE (53%) and 5-HpETE (18%).
					</p>
				
					<p>&nbsp;&nbsp;
						The C6 volatiles such as hexanal, <italic> Z </italic> -3-hexenal, <italic> E </italic> -2-hexenal and <italic> Z </italic> -3-hexenol are important for the characteristic aroma of different fruit products and are formed by <span class="gene">LOX</span> action and subsequent enzymatic reactions during maceration of leaf and fruit tissue. The unsaturated alcohols and aldehydes are abundant in the headspace or extracts of fruit juice or pulped tissue, whereas saturated esters predominate in the headspace volatiles above intact fruits such as apples. <sup> <xref ref-type="bibr" rid="bib8"> 8 </xref> , <xref ref-type="bibr" rid="bib52"> 52 </xref> </sup> In most <span class="trait">plant</span> species (except tomato), aldehydes are more abundant in unripe fruit and esters in ripe fruit as esters accumulate when the fruit undergoes a climacteric rise in ethylene production. <sup> <xref ref-type="bibr" rid="bib53"> 53 </xref> </sup> Thus, in apples and other fruits, most of the studies on aroma formation have focused on the last step of ester formation, in which <span class="gene">AAT</span> enzymes have a key role. <sup> <xref ref-type="bibr" rid="bib54"> 54 </xref> </sup> Ester biosynthesis in apple is also limited by the levels of alcohols, which implies that a critical step for ester production may be regulated at the level of ADH and LOX. <sup> <xref ref-type="bibr" rid="bib5"> 5 </xref> , <xref ref-type="bibr" rid="bib52"> 52 </xref> , <xref ref-type="bibr" rid="bib55"> 55 </xref> , <xref ref-type="bibr" rid="bib56"> 56 </xref> </sup> In the following, we provide strong evidence that LOX1:Md:1a is involved in the formation of C6 precursor substrates for <span class="gene">AAT</span> and is essential for ester production during climacteric ripening of apples whereas LOX1:Md:1c, LOX2:Md:2a and LOX2:Md:2b probably contribute to C6 aldehyde production in immature fruit after decompartmentation.
					</p>
				
					<p>&nbsp;&nbsp;
						Initially, the involvement of the <span class="gene">LOX</span> pathway in the generation of fruit esters was discovered by feeding experiments using unlabeled and isotopically labeled precursors. <sup> <xref ref-type="bibr" rid="bib8"> 8 </xref> </sup> Although &#x003b2;-oxidation of fatty acids has been established as one of the primary processes contributing to the pool of alcohols and acyl-CoA for ester production, unsaturated straight-chain esters appear to arise only by the <span class="gene">LOX</span> pathway. Besides, precursors for butyl and possibly also propyl and pentyl ester may be formed from unsaturated fatty acids by sequential activity of LOX/HPL followed by &#x003b2;-oxidation. As ester composition and levels differed in apples treated with different signal molecules such as salicylic acid, ethephon (ethylene precursor) and methyl jasmonate, it was supposed that regulation of the pathway upstream of <span class="gene">AAT</span> activity depends on the regulation of <span class="gene">LOX</span> and ADH activity by the signal molecules. <sup> <xref ref-type="bibr" rid="bib57"> 57 </xref> </sup> Consistent with this, the suppression of ethylene biosynthesis <sup> <xref ref-type="bibr" rid="bib58"> 58 </xref> </sup> and perception <sup> <xref ref-type="bibr" rid="bib59"> 59 </xref> </sup> in apple caused a major reduction in all group of volatiles, especially in esters and alcohols. Exposure of apple fruit (in which ethylene synthesis had been suppressed) to ethylene led to a recovery in the levels of alcohols and aldehydes. <sup> <xref ref-type="bibr" rid="bib58"> 58 </xref> </sup> While <span class="gene">AAT</span> enzyme activity displayed a clear pattern concomitant with ethylene regulation, total <span class="gene">LOX</span> enzyme activity (mixture of all LOX) seems to be independent of ethylene modulation. <sup> <xref ref-type="bibr" rid="bib58"> 58 </xref> </sup> However, <italic> MdLOX1a </italic> is the only <italic> <span class="gene">LOX</span> </italic> gene that underwent a sustained increase in expression level concomitant with the onset of the rise in ripening-related ethylene and the climacteric rise in CO <sub> 2 </sub> production and hexyl ester formation. Given that no <italic> Z </italic> -3-hexenal, <italic> Z </italic> -3-hexenol or esters derived from <italic> Z </italic> -3-hexenal are detected in emissions from intact fruit, <sup> <xref ref-type="bibr" rid="bib7"> 7 </xref> </sup> <span class="gene">LOX</span> metabolism of fatty acid substrates is apparently limited to LA in intact tissues, which would <span class="trait">yield</span> hexanal, and subsequently, hexanol and hexyl esters. This conclusion is consistent with the observations that free LnA levels are quite low in ripening apple fruit, while LA levels are substantially higher. <sup> <xref ref-type="bibr" rid="bib60"> 60 </xref> </sup> Given the low expression of <italic> MdLOX1a </italic> in immature fruit, other <italic> <span class="gene">LOX</span> </italic> genes are more likely to contribute to C6 aldehyde synthesis prior to ripening.
					</p>
				
					<p>&nbsp;&nbsp;
						In our study, three out of four characterized apple <span class="gene">LOX</span> were able to form 13-hydroperoxides from LA and LnA. The major fraction of 13-HpODE detected for LOX2:Md:2b was in ( <italic> S </italic> )-configuration, whereas LOX1:Md:1a and LOX2:Md:2a formed mainly 13( <italic> R </italic> )-HpODE. In contrast to the <italic> in silico </italic> prediction, LOX1:Md:1a was identified as ( <italic> R </italic> )-specific dual positional LOX. ( <italic> R </italic> )-LOX specificity is a rare trait among <span class="trait">plant</span> LOXs and has so far only been described for <italic> type-2 </italic> <span class="gene">LOX</span> from soybean and pea as well as an olive 9/13-LOX. <sup> <xref ref-type="bibr" rid="bib68"> 68 </xref> , <xref ref-type="bibr" rid="bib69"> 69 </xref> </sup> Similar to LOX1:Md:1a, olive 9/13-LOX harbors a glycine residue (Gly568) in the sequence position determining stereoselectivity of fatty acid oxygenation. In contrast, LOX2:Md:2a harbors alanine (Ala609), which is conserved among ( <italic> S </italic> )-LOXs. <sup> <xref ref-type="bibr" rid="bib47"> 47 </xref> </sup> It has been suggested that <italic> type-2 </italic> LOXs with an alanine residue are able to mimic ( <italic> R </italic> )-LOX specificity depending on varying substrate concentrations and pH. <sup> <xref ref-type="bibr" rid="bib48"> 48 </xref> </sup> However, taking recent theories on stereo- and regiospecificity of LOXs into account, this would lead to the formation of 9( <italic> R </italic> )- and 13( <italic> S </italic> )-hydroperoxides. Based on the current knowledge it is difficult to predict which mode of substrate binding into the active site would cause the unexpected product specificity of LOX2:Md:2a. Factors that determine regio- and stereospecificity of LOX1:Md:1a are discussed in <xref ref-type="supplementary-material" rid="xob1"> Supplementary Figure S9 </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="gene">LOX</span> pathway. The dioxygenation of PUFAs by 9- and 13-LOX activity forms precursors for important phytooxylipins with functions in <span class="trait">plant</span> defense, wound signaling, senescence and fruit ripening. The formation of fruit esters from 13-hydroperoxide occurs by the <span class="gene">HPL</span> pathway and the action of AAT. AAT, alcohol acyl-CoA transferase; ADH, alcohol dehydrogenase; AOS, allene oxid synthase; DES, divinyl ether synthase; HPL, hydroperoxide lyase; POX, peroxygenase, PUFA, polyunsaturated fatty acid. Adapted from Feussner and Wasternack. <sup> <xref ref-type="bibr" rid="bib13"> 13 </xref> </sup>
					</p>
				
					<p>&nbsp;&nbsp;
						Overview of the substrate binding pocket of LOX1:Md:1a. Homology model showing positions of amino-acid residues described to be essential for <span class="gene">LOX</span> activity and specificity (gray, blue) as well as positions chosen for site-directed mutagenesis (orange, blue). A 13( <italic> S </italic> )-HpODE molecule (green) illustrates position and depth of the substrate binding pocket. The bottom of the cavity is marked by Arg732, which is essential for enzyme activity and therefore highly conserved among <span class="trait">plant</span> <span class="gene">LOX</span> enzymes. However, the catalytic iron atom (red sphere) is coordinated by five conserved residues (His524, His529, His715, Asn719, Ile863) near the entrance. The residues Thr581 and Val582 align with the so-called Sloane position described for mammalian <span class="gene">LOX</span> and are known to control regiospecificity of fatty acid oxygenation. In addition, Gly567 has been described to influence both regio- and stereoselectivity of <span class="gene">LOX</span> enzymes.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Expression Analysis of Cytokinin Metabolic Genes in Soybean under Normal and <span class="trait">Drought</span> Conditions in Relation to Cytokinin Levels</h2>
			<h3>PMID:22900018</h3>
				
					<h3>PMCID:3416864</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Le </surname>  <given-names> Dung Tien </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishiyama </surname>  <given-names> Rie </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Yasuko </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vankova </surname>  <given-names> Radomira </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tanaka </surname>  <given-names> Maho </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seki </surname>  <given-names> Motoaki </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ham </surname>  <given-names> Le Huy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi-Shinozaki </surname>  <given-names> Kazuko </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shinozaki </surname>  <given-names> Kazuo </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tran </surname>  <given-names> Lam-Son Phan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Cytokinins (CKs) mediate cellular responses to <span class="trait">drought</span> stress and targeted control of CK metabolism can be used to develop drought-tolerant plants. Aiming to manipulate CK levels to improve <span class="trait">drought</span> tolerance of soybean cultivars through genetic engineering of CK metabolic genes, we surveyed the soybean genome and identified 14 CK biosynthetic (isopentenyltransferase, <italic> <span class="gene">GmIPT</span> </italic> ) and 17 CK degradative (CK dehydrogenase, <italic> <span class="gene">GmCKX</span> </italic> ) genes. Comparative analyses of <span class="gene">GmIPTs</span> and <span class="gene">GmCKXs</span> with <italic> Arabidopsis </italic> counterparts revealed their similar architecture. The average numbers of abiotic stress-inducible <italic> cis </italic> -elements <span class="trait">per</span> promoter were 0.4 and 1.2 for <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes, respectively, suggesting that upregulation of <italic> <span class="gene">GmCKX</span> </italic> s, thereby reduction of CK levels, maybe the major events under abiotic stresses. Indeed, the expression of 12 <italic> <span class="gene">GmCKX</span> </italic> genes was upregulated by dehydration in R2 roots. Overall, the expressions of soybean CK metabolic genes in various tissues at various stages were highly responsive to <span class="trait">drought.</span> CK contents in various organs at the reproductive (R2) stage were also determined under well-watered and <span class="trait">drought</span> stress conditions. Although tRNA-type <italic> <span class="gene">GmIPT</span> </italic> genes were highly expressed in soybean, <italic> cis </italic> -zeatin and its derivatives were found at low concentrations. Moreover, reduction of total CK content in R2 leaves under <span class="trait">drought</span> was attributable to the decrease in dihydrozeatin levels, suggesting a role of this molecule in regulating soybean's responses to <span class="trait">drought</span> stress. Our systematic analysis of the <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> families has provided an insight into CK metabolism in soybean under <span class="trait">drought</span> stress and a solid foundation for in-depth characterization and future development of improved drought-tolerant soybean cultivars by manipulation of CK levels via biotechnological approach.
					</p>
				
					<p>&nbsp;&nbsp;
						Taking into account the importance of CKs and CK signaling in the regulation of stress tolerance, which provides multiple biotechnological strategies for agronomy, we have previously identified and characterized expression profiles of each TCS member in soybean seedlings under dehydration stress <xref ref-type="bibr" rid="pone.0042411-Le1"> [36] </xref> , <xref ref-type="bibr" rid="pone.0042411-Mochida1"> [37] </xref> . In this report, we have identified and systematically characterized all of the <span class="gene">IPT</span> and CKX encoding genes in soybean. We have found that a large number of putative <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes may be the result of genome duplication. Since there is a wealth of structural and functional information for <italic> Arabidopsis </italic> IPTs and CKXs, we performed sequence analyses and phylogenetic relationship studies of IPTs and CKXs of soybean and <italic> Arabidopsis </italic> to classify the functions of <span class="gene">GmIPT</span> and <span class="gene">GmCKX</span> proteins in CK metabolism based on their sequence architecture. To clarify the regulation of CK metabolism in soybean during normal growth and <span class="trait">drought</span> stress, we have analyzed the expression patterns of <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes under normal and <span class="trait">drought</span> stress conditions in a tissue-specific fashion. Expression profiles of <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes were examined in various tissues of both dehydrated young seedlings and soil-dried plants at vegetative and flowering stages. Additionally, we have investigated the correlation between the <span class="trait">drought</span> stress-dependent alterations of CK metabolic gene expression and CK biosynthesis by determining the endogenous CK levels in soybean drought-treated leaf tissue.
					</p>
				
					<p>&nbsp;&nbsp;
						Our analysis has indicated that soybean genes encoding <span class="gene">GmIPTs</span> and <span class="gene">GmCKXs</span> are distributed on various chromosomes, with exception of chromosomes I, V, XVI and XX ( <xref ref-type="fig" rid="pone-0042411-g003"> Figure 3 </xref> ). Gene duplication is also of interest because it is the source of genetic material for diversification <xref ref-type="bibr" rid="pone.0042411-Zhang1"> [47] </xref> . Thus, we subsequently investigated the duplication patterns of <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes. We defined the genes as tandem duplicates if they were located within 20 loci from each other. For segmental duplicates, we analyzed the synteny blocks using an online tool ( <ext-link ext-link-type="uri" xlink:href="http://chibba.agtec.uga.edu/duplication/"> http://chibba.agtec.uga.edu/duplication/ </ext-link> ) <xref ref-type="bibr" rid="pone.0042411-Tang1"> [40] </xref> . We found three tandem duplicated pairs among the <italic> <span class="gene">GmCKXs</span> </italic> ( <xref ref-type="fig" rid="pone-0042411-g003"> Figure 3 </xref> ), namely <italic> <span class="gene">GmCKX03</span> </italic> and <italic> <span class="gene">GmCKX04</span> </italic> on chromosome IX; <italic> <span class="gene">GmCKX05</span> </italic> and <italic> <span class="gene">GmCKX06</span> </italic> on chromosome XIII; and <italic> <span class="gene">GmCKX08</span> </italic> and <italic> <span class="gene">GmCKX09</span> </italic> on chromosome XVII. No tandem duplication was found among the <italic> <span class="gene">GmIPTs</span> </italic> . Segmental duplicates were found in both <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> gene families. As evidenced by synteny analysis, four pairs of <italic> <span class="gene">GmIPT</span> </italic> s were formed by segmental duplication: <italic> <span class="gene">GmIPT04</span> </italic> (Chr. III) and <italic> <span class="gene">GmIPT07</span> </italic> (Chr. XIX); <italic> <span class="gene">GmIPT10</span> </italic> (Chr. VII) and <italic> <span class="gene">GmIPT08</span> </italic> (Chr. XVII); <italic> <span class="gene">GmIPT12</span> </italic> (Chr. VIII) and <italic> <span class="gene">GmIPT11</span> </italic> (Chr. XVIII); and <italic> <span class="gene">GmIPT13</span> </italic> (Chr. XIII) and <italic> <span class="gene">GmIPT09</span> </italic> (Chr. XV) ( <xref ref-type="supplementary-material" rid="pone.0042411.s003"> Figure S3A </xref> ). Among 17 <italic> <span class="gene">GmCKX</span> </italic> genes, five pairs were formed by segmental duplication: <italic> <span class="gene">GmCKX02</span> </italic> (Chr. III) and <italic> <span class="gene">GmCKX01</span> </italic> (Chr. XIX); <italic> <span class="gene">GmCKX11</span> </italic> (Chr. IV) and <italic> <span class="gene">GmCKX10</span> </italic> (Chr. VI); <italic> <span class="gene">GmCKX12</span> </italic> (Chr. IX) and <italic> <span class="gene">GmCKX14</span> </italic> (Chr. XII); <italic> <span class="gene">GmCKX05</span> </italic> (Chr. XIII) and <italic> <span class="gene">GmCKX06</span> </italic> (Chr. XIII); and <italic> <span class="gene">GmCKX08</span> </italic> (Chr. XVII) and <italic> <span class="gene">GmCKX09</span> </italic> (Chr. XVII), the last two pairs being first formed by tandem duplication and then by segmental duplication ( <xref ref-type="fig" rid="pone-0042411-g003"> Figure 3 </xref> ; <xref ref-type="supplementary-material" rid="pone.0042411.s003"> Figure S3B </xref> ). The differences in duplication patterns of the two gene families may reflex their contrasting functions in regulating CK levels in soybean <span class="trait">plant.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Cis </italic> -regulatory elements, which are located in the upstream regions of genes and act as the binding sites for TFs, have essential roles in determining the tissue-specific or stress-responsive expression patterns of genes. Over the years, extensive promoter analyses have identified a number of stress-responsive <italic> cis </italic> -elements, which are important molecular switches involved in the transcriptional regulation of a dynamic network of gene activities controlling abiotic stress responses <xref ref-type="bibr" rid="pone.0042411-YamaguchiShinozaki1"> [4] </xref> , <xref ref-type="bibr" rid="pone.0042411-Tran5"> [48] </xref> , <xref ref-type="bibr" rid="pone.0042411-Nakashima1"> [49] </xref> . Increasing evidence has demonstrated a positive correlation between multi-stimulus responsive genes and <italic> cis </italic> -element density in upstream regions <xref ref-type="bibr" rid="pone.0042411-Walther1"> [50] </xref> - <xref ref-type="bibr" rid="pone.0042411-Zhang2"> [52] </xref> . Therefore, in order to identify possible candidates among <italic> <span class="gene">GmIPT</span> </italic> and <italic> <span class="gene">GmCKX</span> </italic> genes which are involved in abiotic stress responses in soybean plants, we performed a search for the existence of the eleven known stress-responsive <italic> cis </italic> element(s) in the 1000-bp promoter region upstream of the transcription start site of each <span class="gene">GmIPT</span> and <span class="gene">GmCKX</span> encoding gene. As shown in <xref ref-type="supplementary-material" rid="pone.0042411.s005"> Table S1 </xref> , three of the known abiotic-stress responsive <italic> cis- </italic> elements were found in the promoters of <italic> <span class="gene">GmIPT</span> </italic> gene family members. Specifically, MYCR (MYC recognition site) was found in five <italic> <span class="gene">IPT</span> </italic> members ( <italic> <span class="gene">GmIPT04,</span> 05, 08, 11 </italic> and <italic> 12 </italic> ), ZFHDR (zinc finger homeodomain recognition site) was found in <italic> <span class="gene">GmIPT01</span> </italic> and ICEr2 (Inducer of <span class="gene">CBF</span> expression 2 recognition site) was found in <italic> <span class="gene">GmIPT07</span> </italic> . Among the promoters of the <italic> <span class="gene">GmCKX</span> </italic> family, we detected five abiotic-stress responsive <italic> cis- </italic> elements, which are ABRE (ABA-responsive element), MYBR (MYB recognition site), MYCR, ZFHDR and ICEr2 ( <xref ref-type="supplementary-material" rid="pone.0042411.s005"> Table S1 </xref> ). Although distributed on promoters of both <italic> <span class="gene">GmIPT</span> </italic> s and <italic> <span class="gene">GmCKX</span> </italic> s, on the average there were 0.4 and 1.2 abiotic stress-inducible <italic> cis </italic> -elements <span class="trait">per</span> promoter of genes encoding <span class="gene">GmIPT</span> and <span class="gene">GmCKX,</span> respectively. This evidence suggested that upregulation of <italic> <span class="gene">GmCKX</span> </italic> s, thereby reduction of CK levels, could be the major event in soybean plants under abiotic stresses. In agreement with our data, in <italic> Arabidopsis </italic> three out of seven <italic> AtCKX </italic> genes were found to be upregulated by salt stress <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> . The upregulated <italic> CKX </italic> genes, either in <italic> Arabidopsis </italic> or soybean, may contribute to faster degradation of the accumulated CKs upon exposure to environmental stresses, thereby enhancing <span class="trait">plant</span> adaptation to adverse stress conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Tissue-specific and development stage-related expression data are useful in the identification of genes that are involved in defining the precise nature of individual tissues in a given developmental stage. Moreover, the mechanisms controlling the response to <span class="trait">drought</span> stress may be associated with root- and/or shoot-related traits. For instance, suppression of shoot growth and/or promotion of primary root growth are considered morphological adjustments enabling plants to adapt better to <span class="trait">drought</span> stress <xref ref-type="bibr" rid="pone.0042411-Manavalan1"> [1] </xref> , <xref ref-type="bibr" rid="pone.0042411-Thao1"> [3] </xref> , <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0042411-Werner1"> [12] </xref> . CKs are well-known to positively regulate shoot growth but negatively regulate root growth. As a result, the appropriate control of shoot- and root-related morphological traits, via the modulation of endogenous CK levels prior to the occurrence of a stress, as a preventive measure, is a promising approach for developing economically important drought-tolerant crops <xref ref-type="bibr" rid="pone.0042411-Werner1"> [12] </xref> , <xref ref-type="bibr" rid="pone.0042411-Ha1"> [32] </xref> . Apart from their biochemical characteristics, tissue-specific and development stage-related expression of the CK metabolic genes indicate their functional specification and potential utility for the genetic engineering of specific traits. A well-known example is that among the seven <italic> Arabidopsis </italic> ATP/ADP <italic> <span class="gene">IPT</span> </italic> genes, <italic> IPT1 </italic> , <italic> 3 </italic> , <italic> 5 </italic> and <italic> 7 </italic> are expressed in the vegetative phase and <italic> IPT4 </italic> , <italic> 6 </italic> and <italic> 8 </italic> are not. Thus, the <italic> ipt1,3,5,7 </italic> quadruple mutant has reduced active CK levels which results in morphological adjustment (shorter shoot and longer primary root), hypersensitivity to ABA and enhanced cell membrane integrity contributing to enhanced drought-tolerant phenotypes <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0042411-Miyawaki2"> [19] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Among the <italic> <span class="gene">IPT</span> </italic> transcripts expressed in remarkably high abundance, the <italic> <span class="gene">GmIPT08</span> </italic> transcript was consistently increased in the leaves and young seedling shoots under <span class="trait">drought</span> or dehydration conditions ( <xref ref-type="fig" rid="pone-0042411-g005"> Figures 5A </xref> , <xref ref-type="fig" rid="pone-0042411-g006"> 6A </xref> and <xref ref-type="fig" rid="pone-0042411-g007"> 7A </xref> ). The induction level of this transcript was the highest in young seedling shoots (&#x0223c;300-fold, <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5A </xref> ). In the V6 leaves, the induction levels were correlated with the age of the trifoliate leaves; the older the leaf is the higher the induction ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6A </xref> ). In R2 leaves, <italic> <span class="gene">GmIPT08</span> </italic> was the only gene whose expression was significantly induced (&#x0223c;7-fold) by <span class="trait">drought</span> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ). Other <italic> <span class="gene">GmIPT</span> </italic> genes, which were significantly induced by <span class="trait">drought</span> in the V6-stage leaves, are <italic> <span class="gene">GmIPT09</span> </italic> and <italic> 13 </italic> , and the degree of induction of these genes was higher in the younger trifoliate leaves ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6A </xref> ). As for the drought-repressible <italic> <span class="gene">IPT</span> </italic> genes identified in the leaf tissues, the <italic> <span class="gene">GmIPT05</span> </italic> transcripts were repressed by <span class="trait">drought</span> in the leaves of various stages ( <xref ref-type="fig" rid="pone-0042411-g005"> Figures 5A </xref> , <xref ref-type="fig" rid="pone-0042411-g006"> 6A </xref> and <xref ref-type="fig" rid="pone-0042411-g007"> 7A </xref> ). In the R2-stage leaves, expression of almost all highly expressed <italic> <span class="gene">GmIPT</span> </italic> genes, except that of <italic> <span class="gene">GmIPT08</span> </italic> , was repressed by <span class="trait">drought.</span> The most substantial repression was observed in case of the <italic> <span class="gene">GmIPT07</span> </italic> gene ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ). In the roots of young seedlings, the most abundant <italic> <span class="gene">GmIPT08</span> </italic> transcript ( <xref ref-type="fig" rid="pone-0042411-g004"> Figure 4A </xref> ) was slightly induced by dehydration after 2 h of treatment, being subsequently repressed after 10 h of dehydration ( <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5A </xref> ). In the R2 auxiliary roots, the transcripts with high abundance did not change significantly upon dehydration. These data are in contrast with the observation in root hairs where all major transcripts, such as <italic> <span class="gene">GmIPT01</span> </italic> , <italic> 07 </italic> , <italic> 08 </italic> , <italic> 09 </italic> and <italic> 13 </italic> , were induced upon 5-h dehydration ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7A </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						(A) <italic> <span class="gene">GmIPT</span> </italic> genes. (B) <italic> <span class="gene">GmCKX</span> </italic> genes. Black bars; expression under normal condition; white bars, expression under <span class="trait">drought</span> condition.
					</p>
				
					<p>&nbsp;&nbsp;
						(A) <italic> <span class="gene">GmIPT</span> </italic> genes. (B) <italic> <span class="gene">GmCKX</span> </italic> genes. Black bars; expression under normal condition; white bars, expression under <span class="trait">drought</span> condition.
					</p>
				
					<p>&nbsp;&nbsp;
						As for the <italic> <span class="gene">GmCKX</span> </italic> genes, five <italic> <span class="gene">GmCKX</span> </italic> s ( <italic> <span class="gene">GmCKX01</span> </italic> , <italic> 02 </italic> , <italic> 06 </italic> , <italic> 13 </italic> and <italic> 15 </italic> ) were found to be severely reduced at transcriptional level by <span class="trait">drought</span> in the V6 leaves ( <xref ref-type="fig" rid="pone-0042411-g006"> Figure 6B </xref> ), and four of which, except <italic> <span class="gene">GmCKX06</span> </italic> , were also significantly downregulated in R2 leaves under <span class="trait">drought</span> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ). Nevertheless, in young seedling shoots, the expression of <italic> <span class="gene">GmCKX01</span> </italic> and <italic> 02 </italic> was induced by dehydration ( <xref ref-type="fig" rid="pone-0042411-g005"> Figure 5B </xref> ). In addition, the <italic> <span class="gene">GmCKX14</span> </italic> transcript was found to be significantly induced by <span class="trait">drought</span> in various tissues at different stages. Furthermore, most of the <italic> <span class="gene">GmCKX</span> </italic> s were induced by <span class="trait">drought</span> in the R2 roots ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ). Among 17 <italic> <span class="gene">GmCKX</span> </italic> genes, only <italic> <span class="gene">GmCKX16</span> </italic> , whose encoded protein might be involved in degradation of <italic> c </italic> Z-type CKs as its AtCKX7 ortholog <xref ref-type="bibr" rid="pone.0042411-Gajdosova1"> [46] </xref> , was significantly upregulated by <span class="trait">drought</span> in the roots of the R2 stage soybean plants ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Strong lines of evidence have also suggested that CKs play an important role in the development of reproductive organs and seed <span class="trait">yield.</span> Disruption of <italic> <span class="gene">CKX3</span> </italic> and <italic> CKX5 </italic> genes in <italic> Arabidopsis </italic> resulted in higher CK levels, which subsequently led to larger inflorescences and floral meristems, increased size of the WUSCHEL expression domain, supernumerary ovules and increased seed <span class="trait">yield</span> of the <italic> ckx3,5 </italic> double mutant plants <xref ref-type="bibr" rid="pone.0042411-Bartrina2"> [55] </xref> . An increase in CK accumulation caused by a null mutation in the <italic> OsCKX2 </italic> gene was also shown to enhance the size of inflorescence meristems and increase the number of reproductive organs, resulting in enhanced grain <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0042411-Ashikari1"> [56] </xref> . Therefore, to gain an insight into the CK metabolism in reproductive organs, we analyzed the expression of CK metabolic genes in flowers, full pods and R5 <span class="trait">seeds.</span> Results shown in <xref ref-type="fig" rid="pone-0042411-g008"> Figure 8A </xref> indicated that <italic> <span class="gene">GmIPT02</span> </italic> is ubiquitously expressed in all three reproductive tissues examined while five other <italic> <span class="gene">GmIPT</span> </italic> s ( <italic> <span class="gene">GmIPT03</span> </italic> , <italic> 05 </italic> , <italic> 08 </italic> , <italic> 10 </italic> and <italic> 12 </italic> ) were not expressed. <italic> <span class="gene">GmIPT02</span> </italic> was the major transcript in flowers, <italic> <span class="gene">GmIPT01</span> </italic> and <italic> 02 </italic> expressed in the greatest abundance in full pods while <italic> <span class="gene">GmIPT01</span> </italic> , <italic> 02 </italic> and <italic> 11 </italic> were the most abundant transcripts in R5 <span class="trait">seeds.</span> The variation in the <italic> <span class="gene">GmIPT</span> </italic> transcript levels in flowers, pods and R5 <span class="trait">seeds</span> suggested that each of these organs required different <italic> <span class="gene">GmIPT</span> </italic> genes for CK biosynthesis. <italic> <span class="gene">GmIPT02</span> </italic> and <italic> <span class="gene">GmIPT11</span> </italic> might play the most important role in flowers and R5 <span class="trait">seeds,</span> respectively, while <italic> <span class="gene">GmIPT01</span> </italic> and <italic> 02 </italic> appear to be equally important in full pods as judged by their abundant expression.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression levels of <italic> <span class="gene">GmCKX</span> </italic> genes were also determined in flowers, pods and R5 <span class="trait">seeds</span> ( <xref ref-type="fig" rid="pone-0042411-g008"> Figure 8B </xref> ). In flowers, five out of 17 <italic> <span class="gene">GmCKX</span> </italic> mRNAs were dominant, including <italic> <span class="gene">GmCKX04</span> </italic> , <italic> 07 </italic> , <italic> 08 </italic> , <italic> 12 </italic> and <italic> 16 </italic> . Unlike in flowers, <italic> <span class="gene">GmCKX08</span> </italic> transcript was in the highest abundance measured in pods. In R5 <span class="trait">seeds,</span> <italic> <span class="gene">GmCKX08</span> </italic> was still the most highly abundant transcript, while the second most abundant one was <italic> <span class="gene">GmCKX16</span> </italic> . It is noteworthy to mention that the expression of nine out of 17 <italic> <span class="gene">GmCKX</span> </italic> s was hardly detected in the three reproductive tissues examined ( <xref ref-type="fig" rid="pone-0042411-g008"> Figure 8B </xref> ), including <italic> <span class="gene">GmCKX13</span> </italic> , which were highly expressed in various vegetative tissues ( <xref ref-type="fig" rid="pone-0042411-g004"> Figure 4B </xref> ). Our data suggested that <italic> <span class="gene">GmCKX08</span> </italic> is perhaps the major regulator of CK levels in reproductive organs. On the other hand, in flower tissues, the concerted action of at least five <span class="gene">GmCKXs</span> is required for maintaining CK homeostasis.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain an overall image of CK functions in the development of soybean plants, we quantified the CK metabolites in various tissues/organs collected from soybean plants at various stages as described in <xref ref-type="sec" rid="s2"> Materials and Methods </xref> ( <xref ref-type="supplementary-material" rid="pone.0042411.s008"> Table S4 </xref> ). The most remarkable result is that compared with the levels of <italic> t </italic> Z-type and iP-type CKs, the levels of <italic> c </italic> Z-type CKs were significantly lower in almost all of the tissues/organs examined ( <xref ref-type="fig" rid="pone-0042411-g009"> Figure 9 </xref> ; <xref ref-type="supplementary-material" rid="pone.0042411.s008"> Table S4 </xref> ), despite the fact that tRNA-type <span class="gene">GmIPT</span> encoding genes were found to be highly expressed ( <xref ref-type="fig" rid="pone-0042411-g004"> Figure 4 </xref> ). These data are similar to the situation observed in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0042411-Nishiyama1"> [11] </xref> , <xref ref-type="bibr" rid="pone.0042411-Miyawaki2"> [19] </xref> . Our results added up to a recent study by Gajdosova <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0042411-Gajdosova1"> [46] </xref> which indicated that <italic> c </italic> Z-type CKs occur ubiquitously across <span class="trait">plant</span> kingdom and their abundance is, perhaps, correlated with life strategy rather than with evolutionary complexity. In the vegetative organs, such as leaves, the total content of CKs, as well as the CK compositions, also varied with age. For example, the total CK content in young R2 leaves was higher than that in fully developed R2 leaves. This increase was mainly result of an increase in the DZR and DZRP ( <xref ref-type="fig" rid="pone-0042411-g009"> Figure 9, left panel </xref> ), suggesting that the DZ-type CKs are also significantly produced in young leaves of soybean, in addition to dormant <span class="trait">seeds</span> and apical buds as observed in bean <xref ref-type="bibr" rid="pone.0042411-Martin1"> [57] </xref> , <xref ref-type="bibr" rid="pone.0042411-Mok1"> [58] </xref> . Increasing evidence suggests that several CK receptor kinases, such as the AHK3 of <italic> Arabidopsis </italic> or the ZmHK2 of maize, have affinity to DZ-type CKs, <xref ref-type="bibr" rid="pone.0042411-Spichal1"> [59] </xref> - <xref ref-type="bibr" rid="pone.0042411-Lomin1"> [61] </xref> suggesting that the DZ-type CKs might have biological functions in plants. Previously, we showed that in soybean there are two <span class="gene">GmHKs,</span> the <span class="gene">GmHK12</span> and 13, which have high homology to the AHK3 <xref ref-type="bibr" rid="pone.0042411-Mochida1"> [37] </xref> . Taken together, the DZ-type CKs might be biologically active in soybean and their variation in levels may reflex their active roles in regulating <span class="trait">plant</span> growth and development of soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Since an increasing amount of evidence suggests that CKs play an important role in the regulation of the <span class="trait">drought</span> response, we examined the effects of <span class="trait">drought</span> stress on overall CK metabolism in soybean plants by comparing the CK content in R2 leaves under control and <span class="trait">drought</span> conditions. A CK analysis was performed in leaves collected at the R2 stage, because this stage is considered critical, where <span class="trait">drought</span> stress may cause detrimental effects on soybean productivity ( <ext-link ext-link-type="uri" xlink:href="http://www.uwex.edu/ces/ag/issues/drought2003/soybeansrespondstress.html"> http://www.uwex.edu/ces/ag/issues/drought2003/soybeansrespondstress.html </ext-link> ). The results shown in <xref ref-type="fig" rid="pone-0042411-g009"> Figure 9 </xref> (right panel) indicated that CK content was lower in <span class="trait">drought</span> stressed leaves, which is in agreement with the results observed in other <span class="trait">plant</span> species <xref ref-type="bibr" rid="pone.0042411-Havlova1"> [25] </xref> , <xref ref-type="bibr" rid="pone.0042411-Kudoyarova1"> [62] </xref> . The most pronounced changes were observed with active CKs, <italic> t </italic> Z and DZ, which were significantly reduced in the R2 leaves upon <span class="trait">drought</span> treatment. The total content of <italic> t </italic> Z-type CKs was relatively unchanged. Our data suggest that a decrease in DZ content, which is associated with the downregulation of the majority of the <italic> <span class="gene">GmIPT</span> </italic> genes and the upregulation of <italic> <span class="gene">GmCKX03</span> </italic> and <italic> <span class="gene">GmCKX14</span> </italic> ( <xref ref-type="fig" rid="pone-0042411-g007"> Figure 7 </xref> ), contributes significantly to the overall reduction of CK content in <span class="trait">drought</span> stressed soybean leaves. Additionally, DZ may be involved in regulation of <span class="trait">drought</span> stress responses through <span class="gene">GmHK12,</span> which has high homology to AHK3. Both <italic> <span class="gene">GmHK12</span> </italic> and <italic> AHK3 </italic> were upregulated by <span class="trait">drought</span> stress, and AHK3 was shown to act as a negative regulator of <span class="trait">drought</span> stress signaling in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0042411-Tran2"> [5] </xref> , <xref ref-type="bibr" rid="pone.0042411-Le1"> [36] </xref> , <xref ref-type="bibr" rid="pone.0042411-Mochida1"> [37] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Research in the last several years has indicated that CKs play an essential role in the regulation of <span class="trait">plant</span> adaptation to various environmental stresses, including <span class="trait">drought</span> <xref ref-type="bibr" rid="pone.0042411-Peleg2"> [63] </xref> . Repression of CK metabolism under adverse stress conditions, which leads to a downregulation of CK signaling, is known as one of the mechanisms used by plants to adapt to adverse environmental conditions <xref ref-type="bibr" rid="pone.0042411-Ha1"> [32] </xref> . The results of this study provided the first insight into the previously uncharacterized CK metabolic genes encoding <span class="gene">GmIPTs,</span> which are involved in the rate-limiting step of CK biosynthesis. In addition, we also investigated <italic> <span class="gene">GmCKX</span> </italic> s, encoding the main CK degrading enzymes, which contribute to maintenance or reestablishment of CK homeostasis. Throughout our investigation, we placed a particular emphasis on their tissue-specific and/or drought-responsive expression. Collectively, these data enable us to understand the molecular mechanisms regulating CK homeostasis in various tissues/organs at different developmental stages under both normal and <span class="trait">drought</span> stress conditions.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Genetic Architecture of Seed Composition in Soybean Is Refined by Genome-Wide Association Scans Across Multiple Populations</h2>
			<h3>PMID:25246241</h3>
				
					<h3>PMCID:4232554</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vaughn </surname>  <given-names> Justin N. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> Randall L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Center for Applied Genetic Technologies and Department of Crop and Soil Sciences, University of Georgia, Athens, Georgia 30602 </aff>  <aff id="aff2">  <label> &#x02020; </label> Soybean Maize Germplasm, Pathology, and Genetics Research Unit, USDA, Agricultural Research Service, and Department of Crop Sciences University of Illinois, Urbana, Illinois 61801 </aff>  <aff id="aff3">  <label> &#x02021; </label> Soybean Genomics and Improvement Laboratory, USDA, Agricultural Research Service, Beltsville, Maryland 20705 </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 9 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genotypic data were derived from the large-scale effort to genotype the USDA Soybean Germplasm Collection using SoySNP50K iSelect BeadChips ( <xref rid="bib39" ref-type="bibr"> Song <italic> et al. </italic> 2013 </xref> ). The data were accessed from <ext-link ext-link-type="uri" xlink:href="http://soybase.org/data_distribution/soybase_soy50K_snp_all_cultivars_and_snps.gz"> http://soybase.org/data_distribution/soybase_soy50K_snp_all_cultivars_and_snps.gz </ext-link> on 6 February 2014. Missing data in the total genotyped data set were imputed using command-line TASSEL 3.0 and default <italic> -impute </italic> options ( <xref rid="bib4" ref-type="bibr"> Bradbury <italic> et al. </italic> 2007 </xref> ). To reduce computation demands, imputation was performed using 1000 genotypes at a time. For a given trait and population, genotypes were filtered from the total data set and markers with a minor allele frequency of &#x0003c;0.05 were removed. Generally, filtering resulted in &#x0223c;32,000 SNP markers <span class="trait">per</span> population, although the number of SNP markers for the MS-2000 population was 28,622 ( <ext-link ext-link-type="uri" xlink:href="http://www.g3journal.org/content/suppl/2014/09/22/g3.114.013433.DC1/TableS1.pdf"> Table S1 </ext-link> ). Physical distances described in this manuscript are based on genome assembly version Glyma.Wm82.a1 <span class="gene">(Gmax1.01);</span> the distances are shifted in version Glyma.Wm82.a2 <span class="gene">(Gmax2.0)</span> ( <xref rid="bib36" ref-type="bibr"> Schmutz <italic> et al. </italic> 2010 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-Population Selective Genotyping to Identify Soybean [ <italic> Glycine max </italic> (L.) Merr.] Seed Protein and Oil QTLs</h2>
			<h3>PMID:27172185</h3>
				
					<h3>PMCID:4889660</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Phansak </surname>  <given-names> Piyaporn </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="author-notes" rid="afn2">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soonsuwon </surname>  <given-names> Watcharin </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="author-notes" rid="afn3">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hyten </surname>  <given-names> David L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Graef </surname>  <given-names> George L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Specht </surname>  <given-names> James E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 3 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583-0915 </aff>  <aff id="aff2">  <label> &#x02020; </label> Soybean Genomics and Improvement Laboratory, USDA-ARS, Beltsville, Maryland 20705-2325 </aff>
				<br/> 
				年份：<day> 25 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The F <sub> 2:3 </sub> seed progenies, the F <sub> 1:2 </sub> seed progenies, and parental seed of a given mating, plus seed of four checks ( <italic> i.e. </italic> , breeding lines known to be low or high in seed protein), were evaluated for seed protein, oil, and moisture content using a near-infrared reflectance (NIR) analyzer (Infratec model 1255 <span class="gene">NIR</span> Food and Feed Grain Analyzer, Ultra Tec Manufacturing Inc. Santa Ana, CA). The four check samples were used at the beginning and end of each day to confirm that the <span class="gene">NIR</span> instrument was operating during the day within its performance standards. Seed protein and oil values were output on a zero <span class="trait">per</span> cent seed moisture basis.
					</p>
				
					<p>&nbsp;&nbsp;
						One complete replicate of the NIR-measured protein (and oil) data was obtained for all available F <sub> 2:3 </sub> progenies in each of the 48 populations. Though each population required about 2&#x000a0;hr of assay time, only two (and on occasion, three) 2-hr assays could be conducted on a given day due to worker availability, instrument warm-up and prep time, etc. Thus, this 48-population <span class="gene">NIR</span> assay effort required about five contiguous weeks of workdays to complete. The F <sub> 2:3 </sub> seed progeny in each mating were then ranked from lowest to highest based on their measured seed protein value. After completing a second replicate of NIR-assays of all progenies in just two populations ( <italic> i.e. </italic> , matings 43 and 44; <xref ref-type="table" rid="t1"> Table 1 </xref> ), it was determined that the F <sub> 2:3 </sub> seed progenies present in highest and lowest 10% fractions of the first and second replicate assays were essentially the same progenies (data not shown). Thus, to reduce the phenotyping effort and time required to identify the F <sub> 2:3 </sub> progenies occupying just the lowest and highest 10% fractions, a second replicate of <span class="gene">NIR</span> measurement was performed only on the highest and lowest 20% fractions in each of the remaining 46 populations. In each low and high 20% of 2-rep means, those F <sub> 2:3 </sub> seed progenies ranking at the extreme ends of those 20% fractions were selected to become the corresponding 10% tail fractions of the seed protein distribution. Leaf tissue samples of the F <sub> 2 </sub> <span class="trait">plant</span> progenitors of just these extreme progenies ( <italic> i.e. </italic> , 22 high and 22 low protein) were subsequently used for SG.
					</p>
				
					<p>&nbsp;&nbsp;
						Individual F <sub> 2 </sub> plants (and the F <sub> 3 </sub> seed progeny each produced) were the experimental units in this experiment. Because F <sub> 2 </sub> plants cannot be naturally replicated to obtain an estimate of environmental variance, <span class="gene">NIR</span> assays were performed on the seed progenies harvested from the multiple homozygous female and male parent plants that had been grown in interspersed sections of the same nursery row containing F <sub> 2 </sub> plants. Parental assay data were used to obtain an indirect estimate of the environmental variance using the following equation: <disp-formula id="eq"> <mml:math id="me1"> <mml:mrow> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mtext> e </mml:mtext> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:mo> = </mml:mo> <mml:mo stretchy="false"> ( </mml:mo> <mml:mn> 1 </mml:mn> <mml:mo> / </mml:mo> <mml:mn> 2 </mml:mn> <mml:mo stretchy="false"> ) </mml:mo> <mml:mo stretchy="false"> ( </mml:mo> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mrow> <mml:mtext> pFem </mml:mtext> <mml:mo> + </mml:mo> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mrow> <mml:mtext> pMal </mml:mtext> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:mo stretchy="false"> ) </mml:mo> </mml:mrow> </mml:math> </disp-formula> where <inline-formula> <mml:math id="me2"> <mml:mrow> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mrow> <mml:mtext> pFem </mml:mtext> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:mrow> </mml:math> </inline-formula> and <inline-formula> <mml:math id="me3"> <mml:mrow> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mrow> <mml:mtext> pMal </mml:mtext> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:mtext> &#x000a0; </mml:mtext> </mml:mrow> </mml:math> </inline-formula> were the respective phenotypic variances in the seed protein for the seed produced by the high protein female parent, and by the high <span class="trait">yield</span> (but ordinary protein) male parent, respectively. The genetic variance component of the F <sub> 2:3 </sub> progeny phenotypic variance was then estimated by subtraction, using this formula: <disp-formula id="eq___1"> <mml:math id="me4"> <mml:mrow> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mtext> g </mml:mtext> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:mo> = </mml:mo> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mtext> p </mml:mtext> <mml:mn> 2 </mml:mn> </mml:msubsup> <mml:mo> &#x02212; </mml:mo> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mtext> e </mml:mtext> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:mrow> </mml:math> </disp-formula> where <inline-formula> <mml:math id="me5"> <mml:mrow> <mml:msubsup> <mml:mtext> &#x003c3; </mml:mtext> <mml:mtext> p </mml:mtext> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:mrow> </mml:math> </inline-formula> was the F <sub> 2:3 </sub> progeny phenotypic variance.
					</p>
				
					<p>&nbsp;&nbsp;
						Our initial goal was to generate at least 250 F <sub> 2:3 </sub> seed progenies in each mating, which was reached in most matings ( <ext-link ext-link-type="uri" xlink:href="http://www.g3journal.org/cgi/data/g3.116.027656/DC1/5"> Table S2 </ext-link> ), but not in some later MGs, though sufficient F <sub> 2 </sub> <span class="trait">plant</span> numbers were raised <span class="trait">per</span> mating. Progeny numbers averaged 224 over the 48 matings, but ranged from 278 to 115. In later MG matings, many F <sub> 2 </sub> plants produced too few F <sub> 3 </sub> seed (due to pod shattering) to meet the minimum seed sample requirement of the <span class="gene">NIR</span> instrument.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Whole-Genome Resequencing Identifies the Molecular Genetic Cause for the Absence of a <span class="gene">Gy5</span> Glycinin Protein in Soybean PI 603408</h2>
			<h3>PMID:28592556</h3>
				
					<h3>PMCID:5499141</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gillman </surname>  <given-names> Jason D. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Won-Seok </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oehrle </surname>  <given-names> Nathan W. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tawari </surname>  <given-names> Nilesh R. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> &#x000a7; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Krishnan </surname>  <given-names> Hari B. </given-names>  </name>  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-6437-3672 </contrib-id>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> United States Department of Agriculture - Agricultural Research Service Plant Genetics Research Unit, Columbia, Missouri </aff>  <aff id="aff2">  <label> &#x02020; </label> Plant Science Division, University of Missouri, Columbia, Missouri 65211, </aff>  <aff id="aff3">  <label> &#x02021; </label> Key Laboratory of Soybean Biology at the Chinese Ministry of Education, Northeast Agricultural University, Harbin 150030, China </aff>  <aff id="aff4">  <label> &#x000a7; </label> Computational and Systems Biology Group, Genome Institute of Singapore, Agency for Science, Technology and Research, 138672 Singapore </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seed of line PI 603408 was obtained from the USDA-GRIN collection. PI 603408 was donated by the Chinese Academy of Agricultural Sciences to the USDA-GRIN collection in 1998, marked as originating from Liaoning province. Seed of line &#x0201c;Patriot&#x0201d; was obtained from David Sleper of the University of Missouri. PI 603408 was crossed during the summer of 2013 to Patriot and to line &#x0201c;10/81b,&#x0201d; which is a <italic> gy1a/gy1a </italic> and <italic> gy4/gy4 </italic> F <sub> 4:5 </sub> line (PI 605781 B &#x000d7; Patriot) produced as part of a previous study ( <xref rid="bib12" ref-type="bibr"> Kim <italic> et al. </italic> 2013 </xref> ). F <sub> 1 </sub> seed from the two crosses were advanced two generations by single seed descent at a winter nursery. In 2014, F <sub> 3 </sub> <span class="trait">seeds</span> from crosses were planted at South Farm Experimental farm (Columbia, MO, Latitude 38.908189, Longitude &#x02212;92.278693, Mexico silt loam soil) and in 2015 at Hinkson Field (Columbia, MO, Latitude 38.928015, Longitude &#x02212;92.351425, Haymond silt loam soil). <span class="trait">Seeds</span> were planted in 15 ft rows with a 3 ft <span class="gene">gap</span> between plots, a spacing of &#x0223c;2 in between plants, and row spacing of 30 in between rows. In 2014, individual plants were tagged and leaf tissue from individual plants sampled, freeze-dried, and crude DNA was isolated as previously described ( <xref rid="bib35" ref-type="bibr"> Xin <italic> et al. </italic> 2003 </xref> ) or high-quality DNA was isolated with a DNeasy <span class="trait">plant</span> mini kit (QIAGEN, Valencia, CA). When plants had matured in 2014, individual plants were single <span class="trait">plant</span> threshed and <span class="trait">seeds</span> were stored at 4&#x000b0; and 39% relative humidity. Seed was harvested from individual plants in 2014 and 20 <span class="trait">seeds</span> were planted as five ft single plots in 2015, with spacing as described above. At maturity, each plot in 2015 was bulk harvested and stored at conditions described above until analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Through proteomic analysis, an unimproved soybean landrace, PI 603408, was identified whose <span class="trait">seeds</span> lacked a glycinin protein subunit. Through whole genomic resequencing at a very high coverage depth (&#x0223c;59-fold), the molecular genetic cause was determined to be a two-base deletion that introduces a frameshift mutation in <italic> Glycinin 5 </italic> . This was confirmed by cosegregation of the mutation with the absence of <span class="gene">Gy5</span> protein in two independent segregating populations. The two-nucleotide deletion was found to have no significant effect on seed protein in field experiments over 2 yr. In addition, a total of 1,858,185 nucleotide variants were detected by resequencing PI 603408, as compared to the reference genome Williams 82, and 39,344 variants were predicted to result in coding region changes, affecting 13,590 genes. A newly developed SIFT 4G database was used to predict the effect of the SNV using ancestral conservation scoring across a range of diverse species. We anticipate that the new SIFT 4G database, as well as the extremely high coverage depth (average 59.1-fold) resequencing information for PI 603408, will prove useful in future soybean gene diversity and gene function studies.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>