<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Seed Set</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>Seed Set</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:A novel role of the soybean clock gene LUX ARRHYTHMO in male reproductive development.</h2>
			<h3>PMID:28878247</h3>
				

				
			<p>
				作者：Liew LC(1)(2), Singh MB(1), Bhalla PL(3).
				<br/> 
				年份：2017 Sep 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Ectopic expression of <span class="gene">GmLUXb</span> in Arabidopsis lux mutants can complement functions of AtLUX, whereas <span class="gene">GmLUXc</span> generates novel phenotypes of serrated leaves, stunted plants, shortened anther filament, and low <span class="trait">seed</span> <span class="trait">set</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Overall, our results suggest that the LUX gene has diverged in soybean where <span class="gene">GmLUXb</span> and <span class="gene">GmLUXc</span> share the role to control flowering time, but <span class="gene">GmLUXc</span> has evolved to regulate anther filament growth and <span class="trait">seed</span> <span class="trait">set</span> by regulating the Gibberellin hormone biosynthesis pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Foliar and <span class="trait">Seed</span> Application of Amino Acids Affects the Antioxidant Metabolism of the Soybean Crop.</h2>
			<h3>PMID:28377778</h3>
				

				
			<p>
				作者：Teixeira WF(1), Fagan EB(2), Soares LH(2), Umburanas RC(1), Reichardt K(3), Neto DD(1).
				<br/> 
				年份：2017 Mar 21
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In both experiments, the use of cysteine, only in <span class="trait">seed</span> treatment and in both <span class="trait">seed</span> treatment and foliar application increased the activity of the enzyme <span class="gene">PAL</span> and catalase.
					</p>
				
					<p>&nbsp;&nbsp;
						Also in both experiments, the use of phenylalanine increased the activity of the enzyme <span class="gene">PAL</span> when the application was carried out as foliar application or both in <span class="trait">seed</span> treatment and foliar application.
					</p>
				
					<p>&nbsp;&nbsp;
						In the field experiment, the application of glutamate led to an increase in the activity of the catalase and <span class="gene">PAL</span> enzymes for <span class="trait">seed</span> treatment and foliar application.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean <span class="trait">seed</span> <span class="trait">set</span> and size using RNA sequencing analysis.</h2>
			<h3>PMID:28087653</h3>
				

				
			<p>
				作者：Du J(1), Wang S(1), He C(1), Zhou B(2), Ruan YL(3), Shou H(1).
				<br/> 
				年份：2017 Apr 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Overexpression of a candidate <span class="trait">seed</span> size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased <span class="trait">seed</span> size and <span class="trait">seed</span> weight.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl- Exclusion Prior to Na+ Exclusion But Does Not Improve Early Vigor under Salinity.</h2>
			<h3>PMID:27746805</h3>
				

				
			<p>
				作者：Liu Y(1), Yu L(1), Qu Y(2), Chen J(1), Liu X(1), Hong H(1), Liu Z(1), Chang R(1), Gilliham M(2), Qiu L(1), Guan R(1).
				<br/> 
				年份：2016 Sep 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene.
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast, under saline field conditions the NIL-T lines had significantly greater plant <span class="trait">seed</span> weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher <span class="trait">seed</span> weight under saline stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative estimation of sampling uncertainties for mycotoxins in cereal shipments.</h2>
			<h3>PMID:22651820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bourgeois FS(1), Lyman GJ.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GMOs</span> and mycotoxins appear to have a highly heterogeneous distribution in a cargo depending on how the ship was loaded (the grain may have come from more than one terminal and <span class="trait">set</span> of storage silos) and mycotoxin growth may have occurred in transit.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A <span class="gene">GmAOX2b</span> antisense gene compromises vegetative growth and <span class="trait">seed</span> production in soybean.</h2>
			<h3>PMID:22307678  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loss-of-function mutations affecting a specific <italic> Glycine max </italic> R2R3 MYB transcription factor result in brown hilum and brown <span class="trait">seed</span> coats</h2>
			<h3>PMID:22070454</h3>
				

				
					<h3>PMCID:3229458</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Gillman </surname>  <given-names> Jason D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Jason.Gillman@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Tetlow </surname>  <given-names> Ashley </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> altz8b@mail.mizzou.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Lee </surname>  <given-names> Jeong-Deong </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> jdlee@knu.ac.kr </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Shannon </surname>  <given-names> J Grover </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> ShannonG@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Bilyeu </surname>  <given-names> Kristin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Kristin.Bilyeu@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 9 </day>  <month> 11 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> coat and hilum color are relatively simple epistatic multi-genic traits, and variation in hilum and <span class="trait">seed</span> coat pigmentation appears to be due to the interaction of four independent loci: <italic> Inhibitor </italic> ( <italic> I </italic> ), <italic> Tawny </italic> ( <italic> T </italic> ), an unnamed locus termed <italic> R </italic> , and the flower color locus <italic> <span class="gene">W1</span> </italic> [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ](Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, two highly similar co-expressed <italic> UF3GT </italic> genes <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130)</span> were determined to be expressed in <span class="trait">seed</span> coats of black seeded soybean lines, and these genes have been demonstrated to specifically transfer a glucose moiety to the cyanidin molecule at the 3'-hydroxyl group, resulting in the formation of cyanidin-3-glucoside [ <xref ref-type="bibr" rid="B15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Both the dominant <italic> Inhibitor </italic> allele ( <italic> I </italic> ) and the <italic> i <sup> i </sup> </italic> alleles have been shown to be due to naturally occurring, gene-silencing effects derived from linked but independent Chalcone Synthase (CHS) gene clusters (chromosome 8, LG A2) that generate siRNA which target <span class="gene">CHS</span> gene transcripts specifically within the <span class="trait">seed</span> coat for degradation [ <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B22"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In lines with the recessive ( <italic> i </italic> ) allele, <span class="trait">seed</span> coat color is brown, imperfect black, buff or black, dependent on the allelic status of the <italic> Tawny, R and <span class="gene">W1</span> </italic> loci (Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Tawny </italic> locus has two pleiotropic effects: homozygosity for the gray ( <italic> t </italic> ) allele results in gray pubescence at maturity and, in lines carrying the combination of the <italic> i <sup> i </sup> </italic> allele of the <italic> Inhibitor </italic> locus, a functional <italic> R </italic> gene, and purple flowers ( <italic> <span class="gene">W1</span> </italic> ), <span class="trait">seed</span> which feature "imperfect black" hila (Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Another locus, classically termed <italic> R </italic> , also interacts epistatically with the <italic> Tawny </italic> and <italic> Inhibitor </italic> loci (as well as the <italic> <span class="gene">W1</span> </italic> locus) to control hilum and <span class="trait">seed</span> coat colors [ <xref ref-type="bibr" rid="B8"> 8 </xref> ] (Table <xref ref-type="table" rid="T1"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The allelic status of the <italic> <span class="gene">W1</span> </italic> locus, when combined with the recessive gray allele of the Tawny locus, determines if <span class="trait">seed</span> coats or hila are colored "imperfect black" or "buff" (Table <xref ref-type="table" rid="T1"> 1 </xref> ) [ <xref ref-type="bibr" rid="B8"> 8 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Jake (PI 643912) has tawny pubescence, purple flowers, and shiny yellow <span class="trait">seed</span> with black hila ( <italic> i <sup> i </sup> T R <span class="gene">W1</span> </italic> )[ <xref ref-type="bibr" rid="B31"> 31 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The brown hila line, PI 283327 has tawny pubescence, purple flowers, and yellow <span class="trait">seed</span> with brown hila ( <italic> i <sup> i </sup> , T, r, <span class="gene">W1</span> </italic> ) (USDA GRIN germplasm collection, accessed 06/22/2011 ( <ext-link ext-link-type="uri" xlink:href="http://www.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to assess whether a subset of these four tandem genes were pseudogenes and/or expressed in <span class="trait">seed</span> coat tissue <span class="gene">(Glyma09g36970</span> is annotated as a pseudogene in the current whole genome shotgun sequence build), we utilized qRT-PCR.
					</p>
				
					<p>&nbsp;&nbsp;
						Only one of these candidate R2R3 MYB genes, <span class="gene">Glyma09g36990,</span> was expressed in any of the tissues examined (leaf, <span class="trait">seed</span> cotyledons, and <span class="trait">seed</span> coats).
					</p>
				
					<p>&nbsp;&nbsp;
						Gene transcripts from <span class="gene">Glyma09g36990</span> were present in the <span class="trait">seed</span> coats of both a brown seeded and a black seeded cultivar.
					</p>
				
					<p>&nbsp;&nbsp;
						PCR and Sanger sequencing of exons (and partial intronic sequence) was used to evaluate the <span class="gene">Glyma09g36990</span> gene for polymorphisms in a selection of lines: Jake (black hilum), PI 283327 (brown hilum), Williams 82 (black hilum), PI 84970 (black <span class="trait">seed</span> coats) and PI 567115 B (brown <span class="trait">seed</span> coats).
					</p>
				
					<p>&nbsp;&nbsp;
						In all cases where we observed an intact open reading frame, we noted the phenotype of imperfect black hilum ( <italic> i <sup> i </sup> R t <span class="gene">W1</span> </italic> ), buff hilum ( <italic> i <sup> i </sup> R t w1 </italic> ), black hilum ( <italic> i <sup> i </sup> R T </italic> ) or black <span class="trait">seed</span> coat ( <italic> i R T </italic> ), dependent on the allelic status of the <italic> Inhibitor </italic> and <italic> Tawny </italic> loci (Additional File <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						If the candidate <italic> R </italic> gene is controlling expression of a gene which forms a rate limited step in anthocyanin production, we hypothesized that a correlation would exist between 1) <italic> R </italic> gene expression levels, 2) the appearance of color compounds, and 3) the expression of <italic> <span class="gene">ANS</span> </italic> and/or <italic> UF3GT </italic> genes in developing <span class="trait">seed</span> coats.
					</p>
				
					<p>&nbsp;&nbsp;
						We examined a time course of <span class="trait">seed</span> coat and <span class="trait">seed</span> cotyledons by qRT-PCR (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ) for expression of three genes: the <italic> R </italic> gene candidate, <italic> <span class="gene">ANS</span> </italic> , and <italic> UF3GT </italic>.
					</p>
				
					<p>&nbsp;&nbsp;
						Although there are apparently two <italic> UF3GT </italic> genes expressed in <span class="trait">seed</span> coats in soybean <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130),</span> only one of these genes <span class="gene">(Glyma08g07130)</span> is not expressed in cotyledon tissue [ <xref ref-type="bibr" rid="B15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In striking contrast to the increase in expression of <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> during <span class="trait">seed</span> coat maturation of the black <span class="trait">seed</span> coat line, only negligible <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression was observed in the brown <span class="trait">seed</span> coat line as seeds approached maturity (Figure <xref ref-type="fig" rid="F3"> 3B </xref> and <xref ref-type="fig" rid="F3"> 3C </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Here we used genetic mapping and candidate gene association in a RIL population and a panel of soybean lines with defined coloration <span class="trait">(seed</span> coat and hilum, pubescence, and flower) to determine the <italic> R </italic> gene controlling black or brown <span class="trait">seed</span> coat in soybean is the R2R3 MYB gene <span class="gene">Glyma09g36990</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						Candidate gene analysis identified a loss of function mutation affecting a <span class="trait">seed</span> coat-specific expressed R2R3 MYB transcription factor gene <span class="gene">(Glyma09g36990)</span> as a strong candidate for the brown hilum phenotype
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Seed</span> coat and hilum color are relatively simple epistatic multi-genic traits, and variation in hilum and <span class="trait">seed</span> coat pigmentation appears to be due to the interaction of four independent loci: <italic> Inhibitor </italic> ( <italic> I </italic> ), <italic> Tawny </italic> ( <italic> T </italic> ), an unnamed locus termed <italic> R </italic> , and the flower color locus <italic> <span class="gene">W1</span> </italic> [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ](Table <xref ref-type="table" rid="T1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, two highly similar co-expressed <italic> UF3GT </italic> genes <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130)</span> were determined to be expressed in <span class="trait">seed</span> coats of black seeded soybean lines, and these genes have been demonstrated to specifically transfer a glucose moiety to the cyanidin molecule at the 3'-hydroxyl group, resulting in the formation of cyanidin-3-glucoside [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Both the dominant <italic> Inhibitor </italic> allele ( <italic> I </italic> ) and the <italic> i <sup> i </sup> </italic> alleles have been shown to be due to naturally occurring, gene-silencing effects derived from linked but independent Chalcone Synthase (CHS) gene clusters (chromosome 8, LG A2) that generate siRNA which target <span class="gene">CHS</span> gene transcripts specifically within the <span class="trait">seed</span> coat for degradation [ <xref ref-type="bibr" rid="B16"> 16 </xref> - <xref ref-type="bibr" rid="B22"> 22 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In lines with the recessive ( <italic> i </italic> ) allele, <span class="trait">seed</span> coat color is brown, imperfect black, buff or black, dependent on the allelic status of the <italic> Tawny, R and <span class="gene">W1</span> </italic> loci (Table <xref ref-type="table" rid="T1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> Tawny </italic> locus has two pleiotropic effects: homozygosity for the gray ( <italic> t </italic> ) allele results in gray pubescence at maturity and, in lines carrying the combination of the <italic> i <sup> i </sup> </italic> allele of the <italic> Inhibitor </italic> locus, a functional <italic> R </italic> gene, and purple flowers ( <italic> <span class="gene">W1</span> </italic> ), <span class="trait">seed</span> which feature "imperfect black" hila (Table <xref ref-type="table" rid="T1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Another locus, classically termed <italic> R </italic> , also interacts epistatically with the <italic> Tawny </italic> and <italic> Inhibitor </italic> loci (as well as the <italic> <span class="gene">W1</span> </italic> locus) to control hilum and <span class="trait">seed</span> coat colors [ <xref ref-type="bibr" rid="B8"> 8 </xref> ] (Table <xref ref-type="table" rid="T1"> 1 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The allelic status of the <italic> <span class="gene">W1</span> </italic> locus, when combined with the recessive gray allele of the Tawny locus, determines if <span class="trait">seed</span> coats or hila are colored "imperfect black" or "buff" (Table <xref ref-type="table" rid="T1"> 1 </xref> ) [ <xref ref-type="bibr" rid="B8"> 8 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Jake (PI 643912) has tawny pubescence, purple flowers, and shiny yellow <span class="trait">seed</span> with black hila ( <italic> i <sup> i </sup> T R <span class="gene">W1</span> </italic> )[ <xref ref-type="bibr" rid="B31"> 31 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						The brown hila line, PI 283327 has tawny pubescence, purple flowers, and yellow <span class="trait">seed</span> with brown hila ( <italic> i <sup> i </sup> , T, r, <span class="gene">W1</span> </italic> ) (USDA GRIN germplasm collection, accessed 06/22/2011 ( <ext-link ext-link-type="uri" xlink:href="http://www
					</p>
				
					<p>&nbsp;&nbsp;
						In order to assess whether a subset of these four tandem genes were pseudogenes and/or expressed in <span class="trait">seed</span> coat tissue <span class="gene">(Glyma09g36970</span> is annotated as a pseudogene in the current whole genome shotgun sequence build), we utilized qRT-PCR
					</p>
				
					<p>&nbsp;&nbsp;
						Only one of these candidate R2R3 MYB genes, <span class="gene">Glyma09g36990,</span> was expressed in any of the tissues examined (leaf, <span class="trait">seed</span> cotyledons, and <span class="trait">seed</span> coats)
					</p>
				
					<p>&nbsp;&nbsp;
						Gene transcripts from <span class="gene">Glyma09g36990</span> were present in the <span class="trait">seed</span> coats of both a brown seeded and a black seeded cultivar
					</p>
				
					<p>&nbsp;&nbsp;
						PCR and Sanger sequencing of exons (and partial intronic sequence) was used to evaluate the <span class="gene">Glyma09g36990</span> gene for polymorphisms in a selection of lines: Jake (black hilum), PI 283327 (brown hilum), Williams 82 (black hilum), PI 84970 (black <span class="trait">seed</span> coats) and PI 567115 B (brown <span class="trait">seed</span> coats)
					</p>
				
					<p>&nbsp;&nbsp;
						In all cases where we observed an intact open reading frame, we noted the phenotype of imperfect black hilum ( <italic> i <sup> i </sup> R t <span class="gene">W1</span> </italic> ), buff hilum ( <italic> i <sup> i </sup> R t w1 </italic> ), black hilum ( <italic> i <sup> i </sup> R T </italic> ) or black <span class="trait">seed</span> coat ( <italic> i R T </italic> ), dependent on the allelic status of the <italic> Inhibitor </italic> and <italic> Tawny </italic> loci (Additional File <xref ref-type="supplementary-material" rid="S2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						If the candidate <italic> R </italic> gene is controlling expression of a gene which forms a rate limited step in anthocyanin production, we hypothesized that a correlation would exist between 1) <italic> R </italic> gene expression levels, 2) the appearance of color compounds, and 3) the expression of <italic> <span class="gene">ANS</span> </italic> and/or <italic> UF3GT </italic> genes in developing <span class="trait">seed</span> coats
					</p>
				
					<p>&nbsp;&nbsp;
						We examined a time course of <span class="trait">seed</span> coat and <span class="trait">seed</span> cotyledons by qRT-PCR (Figure <xref ref-type="fig" rid="F3"> 3 </xref> ) for expression of three genes: the <italic> R </italic> gene candidate, <italic> <span class="gene">ANS</span> </italic> , and <italic> UF3GT </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Although there are apparently two <italic> UF3GT </italic> genes expressed in <span class="trait">seed</span> coats in soybean <span class="gene">(Glyma07g30180</span> and <span class="gene">Glyma08g07130),</span> only one of these genes <span class="gene">(Glyma08g07130)</span> is not expressed in cotyledon tissue [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In striking contrast to the increase in expression of <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> during <span class="trait">seed</span> coat maturation of the black <span class="trait">seed</span> coat line, only negligible <italic> <span class="gene">ANS</span> </italic> and <italic> UF3GT </italic> expression was observed in the brown <span class="trait">seed</span> coat line as seeds approached maturity (Figure <xref ref-type="fig" rid="F3"> 3B </xref> and <xref ref-type="fig" rid="F3"> 3C </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Here we used genetic mapping and candidate gene association in a RIL population and a panel of soybean lines with defined coloration <span class="trait">(seed</span> coat and hilum, pubescence, and flower) to determine the <italic> R </italic> gene controlling black or brown <span class="trait">seed</span> coat in soybean is the R2R3 MYB gene <span class="gene">Glyma09g36990</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Molecular analysis of the PGYRP (proline-, glycine- and tyrosine-rich protein) gene family in soybean.</h2>
			<h3>PMID:21104142  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Feng Y(1), Peng H, Liang S.
				<br/> 
				年份：2011 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="gene">GmPGYRP1-12</span> genes encoded a <span class="trait">set</span> of small predicted proteins (&lt;120 aa) with molecular mass of 7.
					</p>
				
					<p>&nbsp;&nbsp;
						Furthermore, the expression of <span class="gene">GmPGYRP1,</span> 3, 5, and 7 was significantly regulated by <span class="trait">drought,</span> salt and cold, but obviously repressed by abscisic acid (ABA) at early stage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthetic performance and <span class="trait">fertility</span> are repressed in <span class="gene">GmAOX2b</span> antisense soybean.</h2>
			<h3>PMID:20097793  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chai TT(1), Simmonds D, Day DA, Colmer TD, Finnegan PM.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						未知
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fatty acid profiling of soybean cotyledons by near-infrared spectroscopy.</h2>
			<h3>PMID:17132452  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roberts CA(1), Ren C, Beuselinck PR, Benedict HR, Bilyeu K.
				<br/> 
				年份：2006 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Therefore, the spectra were trimmed to include the <span class="gene">NIR</span> region only (1100-2500 nm), and a second <span class="trait">set</span> of equations was developed.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protease C2, a cysteine endopeptidase involved in the continuing mobilization of soybean beta-conglycinin <span class="trait">seed</span> proteins.</h2>
			<h3>PMID:11342045  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seo S(1), Tan-Wilson A, Wilson KA.
				<br/> 
				年份：2001 Feb 9
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Protease <span class="gene">C2</span> activity is not evident in vivo until 4 days after imbibition of the <span class="trait">seed</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						The action of protease <span class="gene">C2</span> on native beta-conglycinin produces a <span class="trait">set</span> of large fragments (52-46 kDa in size) and small fragments (29-25 kDa).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Two classes of differentially regulated glutamine synthetase genes are expressed in the soybean nodule: a nodule-specific class and a constitutively expressed class.</h2>
			<h3>PMID:8104530  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Roche D(1), Temple SJ, Sengupta-Gopalan C.
				<br/> 
				年份：1993 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Using the 3'-untranslated regions of a representative member of each <span class="trait">set,</span> as gene member(s) specific probes, we have shown that one <span class="trait">set</span> of the <span class="gene">GS</span> genes are expressed in a nodule-specific manner, while the other <span class="trait">set</span> is expressed in other tissues, besides the nodules.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Microarrays for global expression constructed with a low redundancy <span class="trait">set</span> of 27,500 sequenced cDNAs representing an array of developmental stages and physiological conditions of the soybean plant</h2>
			<h3>PMID:15453914</h3>
				

				
					<h3>PMCID:526184</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Khanna </surname>  <given-names> Anupama </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> anupama.khanna@epicentre.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Shealy </surname>  <given-names> Robin </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Clough </surname>  <given-names> Steven J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I8"> 8 </xref>  <email> sjclough@uiuc.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Gonzalez </surname>  <given-names> Delkin Orlando </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dogonzal@uiuc.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Philip </surname>  <given-names> Reena </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I9"> 9 </xref>  <email> philipreena@yahoo.com </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> Zabala </surname>  <given-names> Gracia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> g-zabala@uiuc.edu </email>  </contrib>  <contrib id="A8" contrib-type="author">  <name>  <surname> Thibaud-Nissen </surname>  <given-names> Fran&#x000e7;oise </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I10"> 10 </xref>  <email> fthibaud@tigr.org </email>  </contrib>  <contrib id="A9" contrib-type="author">  <name>  <surname> Sidarous </surname>  <given-names> Mark </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> sidarous@uiuc.edu </email>  </contrib>  <contrib id="A10" contrib-type="author">  <name>  <surname> Str&#x000f6;mvik </surname>  <given-names> Martina V </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I11"> 11 </xref>  <email> martina.stromvik@mcgill.ca </email>  </contrib>  <contrib id="A11" contrib-type="author">  <name>  <surname> Shoop </surname>  <given-names> Elizabeth </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I12"> 12 </xref>  <email> schoop@macalester.edu </email>  </contrib>  <contrib id="A12" contrib-type="author">  <name>  <surname> Schmidt </surname>  <given-names> Christina </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> shoop@macalesler.edu </email>  </contrib>  <contrib id="A13" contrib-type="author">  <name>  <surname> Retzel </surname>  <given-names> Ernest </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> ernest@mail.ahc.umn.edu </email>  </contrib>  <contrib id="A14" contrib-type="author">  <name>  <surname> Erpelding </surname>  <given-names> John </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> rcsshoe@iastate.edu </email>  </contrib>  <contrib id="A15" contrib-type="author">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> rcsshoe@iastate.edu </email>  </contrib>  <contrib id="A16" contrib-type="author">  <name>  <surname> Rodriguez-Huete </surname>  <given-names> Alicia M </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <xref ref-type="aff" rid="I13"> 13 </xref>  <email> PolaccoJ@missouri.edu </email>  </contrib>  <contrib id="A17" contrib-type="author">  <name>  <surname> Polacco </surname>  <given-names> Joseph C </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> PolaccoJ@missouri.edu </email>  </contrib>  <contrib id="A18" contrib-type="author">  <name>  <surname> Coryell </surname>  <given-names> Virginia </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> Paul.Keim@nau.edu </email>  </contrib>  <contrib id="A19" contrib-type="author">  <name>  <surname> Keim </surname>  <given-names> Paul </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> Paul.Keim@nau.edu </email>  </contrib>  <contrib id="A20" contrib-type="author">  <name>  <surname> Gong </surname>  <given-names> George </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> ligong@uiuc.edu </email>  </contrib>  <contrib id="A21" contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> leiliu@uiuc.edu </email>  </contrib>  <contrib id="A22" contrib-type="author">  <name>  <surname> Pardinas </surname>  <given-names> Jose </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A23" contrib-type="author">  <name>  <surname> Schweitzer </surname>  <given-names> Peter </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <xref ref-type="aff" rid="I14"> 14 </xref>  <email> pas48@cornell.edu </email>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 9 </month>  <year> 2004 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cDNA clones were physically reracked to a new <span class="trait">set</span> of 384 well plates to form the filter-normalized reracked library designated <span class="gene">Gm-r1030</span>
					</p>
				
					<p>&nbsp;&nbsp;
						As shown in Table <xref ref-type="table" rid="T3"> 3 </xref> , these include reracked libraries <span class="gene">Gm-r1070</span> (a <span class="trait">set</span> of 9,216 cDNA clones from libraries of various developmental stages of immature cotyledons, flowers, pods, and <span class="trait">seed</span> coats); <span class="gene">Gm-r1021</span> plus <span class="gene">Gm-r1083</span> (a <span class="trait">set</span> of approximately 9,216 cDNA clones from 8-day old seedling roots, seedling roots inoculated with <italic> Bradyrhozobium japonicum, </italic> 2-month old roots, and whole seedlings); and <span class="gene">Gm-r1088</span> (a collection of 9,216 cDNA clones from a number of libraries made from cotyledons and hypocotyls of germinating seedlings and leaves and other plant parts subjected to various pathogens or environmental stress conditions and also from tissue-culture derived somatic embryos)
					</p>
				
					<p>&nbsp;&nbsp;
						As an example, the <span class="gene">Gm-r1070</span> <span class="trait">set</span> contains 3,938 tentatively unique cDNAs that are directly derived from two flower cDNA libraries <span class="gene">(Gm-c1015</span> and <span class="gene">Gm-c1016)</span> that were sequenced deeply with over 14,000 5' ESTs obtained from these two libraries
					</p>
				
					<p>&nbsp;&nbsp;
						A <span class="trait">set</span> of 64 control or 'choice' clones was assembled by hand into one 96-well plate (designated <span class="gene">Gm-b10BB)</span> and printed eight times repetitively throughout each array
					</p>
				
					<p>&nbsp;&nbsp;
						Hybridizations were performed to microarrays constructed with the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1070</span> representing cDNAs from seeds, <span class="trait">seed</span> coats, and flowers
					</p>
				
					<p>&nbsp;&nbsp;
						For illustration, Figure <xref ref-type="fig" rid="F4"> 4 </xref> shows one of the two replicates of a dual labeling experiment using the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1088</span> of 9,216 cDNAs
					</p>
				
					<p>&nbsp;&nbsp;
						We have previously utilized microarrays containing 9,216 clones of the <span class="gene">Gm-r1070</span> <span class="trait">set</span> (representing many cDNAs from developing seeds, <span class="trait">seed</span> coats, flowers, and pods) to carry out a detailed analysis of induction of somatic embryos during culture of cotyledons on auxin-containing media [ <xref ref-type="bibr" rid="B16"> 16 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, since the <span class="gene">Gm-r1021</span> library contained only 4089 cDNAs, an additional 135 were repeated in order to obtain an even 9216 cDNAs for printing when combined with the <span class="gene">Gm-r1083</span> unigene <span class="trait">set</span>
					</p>
				
					<p>&nbsp;&nbsp;
						LOV led the unigene and microarray development, coordinated the project, and drafted the manuscript; AK constructed multiple cDNA libraries included in the unigene cDNAs, performed over 18,000 PCR reactions, performed the library normalization by filter screening shown in Figure <xref ref-type="fig" rid="F1"> 1 </xref> , the array hybridization reactions including that of Fig <xref ref-type="fig" rid="F4"> 4 </xref> , and drafted protocols; RS led the informatics and sample tracking efforts for array printing, high throughput PCR, cDNA clone reracking, printed arrays, developed and drafted protocols for analysis of array data; SJC initiated the PCR and array hybridization protocols for the project and constructed several cDNA libraries; DOG performed 10,000 PCR reactions, printed arrays, and participated in clone rearraying; RP contributed to protocol development, PCR production, and gel analysis; GZ accumulated the 'choice clone" cDNAs and performed the isoline hybridization analysis shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> ; <span class="gene">MS</span> contributed to informatics and sample tracking and gel analysis of PCR products; MVS performed EST cluster analysis for development of the unigene <span class="trait">set</span> and drafted sections of the paper; ES and CS performed EST analysis and clustering for the unigene <span class="trait">set;</span> ER coordinated EST processing and informatics; JE constructed multiple cDNA libraries used in selecting the unigene <span class="trait">set;</span> RS led the public EST project for library construction and 5' sequencing; AR-H and JCP provided hydroponically grown plant material and RNAs for Figure <xref ref-type="fig" rid="F4"> 4 </xref> and constructed a cDNA library; VC and PC constructed multiple cDNA libraries; GG and LL performed annotations of the 27,500 unigene <span class="trait">set;</span> JP and PS led or performed the 3' sequencing of the 27,500 unigene clones
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						As shown in Table <xref ref-type="table" rid="T3"> 3 </xref> , these include reracked libraries <span class="gene">Gm-r1070</span> (a <span class="trait">set</span> of 9,216 cDNA clones from libraries of various developmental stages of immature cotyledons, flowers, pods, and <span class="trait">seed</span> coats); <span class="gene">Gm-r1021</span> plus <span class="gene">Gm-r1083</span> (a <span class="trait">set</span> of approximately 9,216 cDNA clones from 8-day old seedling roots, seedling roots inoculated with <italic> Bradyrhozobium japonicum, </italic> 2-month old roots, and whole seedlings); and <span class="gene">Gm-r1088</span> (a collection of 9,216 cDNA clones from a number of libraries made from cotyledons and hypocotyls of germinating seedlings and leaves and other plant parts subjected to various pathogens or environmental stress conditions and also from tissue-culture derived somatic embryos).
					</p>
				
					<p>&nbsp;&nbsp;
						As an example, the <span class="gene">Gm-r1070</span> <span class="trait">set</span> contains 3,938 tentatively unique cDNAs that are directly derived from two flower cDNA libraries <span class="gene">(Gm-c1015</span> and <span class="gene">Gm-c1016)</span> that were sequenced deeply with over 14,000 5' ESTs obtained from these two libraries.
					</p>
				
					<p>&nbsp;&nbsp;
						A <span class="trait">set</span> of 64 control or 'choice' clones was assembled by hand into one 96-well plate (designated <span class="gene">Gm-b10BB)</span> and printed eight times repetitively throughout each array.
					</p>
				
					<p>&nbsp;&nbsp;
						Hybridizations were performed to microarrays constructed with the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1070</span> representing cDNAs from seeds, <span class="trait">seed</span> coats, and flowers.
					</p>
				
					<p>&nbsp;&nbsp;
						For illustration, Figure <xref ref-type="fig" rid="F4"> 4 </xref> shows one of the two replicates of a dual labeling experiment using the low redundancy <span class="trait">set</span> <span class="gene">Gm-r1088</span> of 9,216 cDNAs.
					</p>
				
					<p>&nbsp;&nbsp;
						We have previously utilized microarrays containing 9,216 clones of the <span class="gene">Gm-r1070</span> <span class="trait">set</span> (representing many cDNAs from developing seeds, <span class="trait">seed</span> coats, flowers, and pods) to carry out a detailed analysis of induction of somatic embryos during culture of cotyledons on auxin-containing media [ <xref ref-type="bibr" rid="B16"> 16 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In addition, since the <span class="gene">Gm-r1021</span> library contained only 4089 cDNAs, an additional 135 were repeated in order to obtain an even 9216 cDNAs for printing when combined with the <span class="gene">Gm-r1083</span> unigene <span class="trait">set</span>.
					</p>
				
					<p>&nbsp;&nbsp;
						LOV led the unigene and microarray development, coordinated the project, and drafted the manuscript; AK constructed multiple cDNA libraries included in the unigene cDNAs, performed over 18,000 PCR reactions, performed the library normalization by filter screening shown in Figure <xref ref-type="fig" rid="F1"> 1 </xref> , the array hybridization reactions including that of Fig <xref ref-type="fig" rid="F4"> 4 </xref> , and drafted protocols; RS led the informatics and sample tracking efforts for array printing, high throughput PCR, cDNA clone reracking, printed arrays, developed and drafted protocols for analysis of array data; SJC initiated the PCR and array hybridization protocols for the project and constructed several cDNA libraries; DOG performed 10,000 PCR reactions, printed arrays, and participated in clone rearraying; RP contributed to protocol development, PCR production, and gel analysis; GZ accumulated the 'choice clone" cDNAs and performed the isoline hybridization analysis shown in Figure <xref ref-type="fig" rid="F3"> 3 </xref> ; <span class="gene">MS</span> contributed to informatics and sample tracking and gel analysis of PCR products; MVS performed EST cluster analysis for development of the unigene <span class="trait">set</span> and drafted sections of the paper; ES and CS performed EST analysis and clustering for the unigene <span class="trait">set;</span> ER coordinated EST processing and informatics; JE constructed multiple cDNA libraries used in selecting the unigene <span class="trait">set;</span> RS led the public EST project for library construction and 5' sequencing; AR-H and JCP provided hydroponically grown plant material and RNAs for Figure <xref ref-type="fig" rid="F4"> 4 </xref> and constructed a cDNA library; VC and PC constructed multiple cDNA libraries; GG and LL performed annotations of the 27,500 unigene <span class="trait">set;</span> JP and PS led or performed the 3' sequencing of the 27,500 unigene clones.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:RNA-Seq Atlas of <italic> Glycine max </italic> : A guide to the soybean transcriptome</h2>
			<h3>PMID:20687943</h3>
				

				
					<h3>PMCID:3017786</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Severin </surname>  <given-names> Andrew J </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> severin@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Woody </surname>  <given-names> Jenna L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jlwoody@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Bolon </surname>  <given-names> Yung-Tsi </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> yung-tsi.bolon@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Joseph </surname>  <given-names> Bindu </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> bindu@iastate.edu </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Diers </surname>  <given-names> Brian W </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> bdiers@uiuc.edu </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Farmer </surname>  <given-names> Andrew D </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> adf@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Muehlbauer </surname>  <given-names> Gary J </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> muehl003@umn.edu </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Nelson </surname>  <given-names> Rex T </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> rex.nelson@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Grant </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> david.grant@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A10">  <name>  <surname> Specht </surname>  <given-names> James E </given-names>  </name>  <xref ref-type="aff" rid="I7"> 7 </xref>  <email> jspecht1@unl.edu </email>  </contrib>  <contrib contrib-type="author" id="A11">  <name>  <surname> Graham </surname>  <given-names> Michelle A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> michelle.graham@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A12">  <name>  <surname> Cannon </surname>  <given-names> Steven B </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> steven.cannon@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A13">  <name>  <surname> May </surname>  <given-names> Gregory D </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> gdm@ncgr.org </email>  </contrib>  <contrib contrib-type="author" id="A14">  <name>  <surname> Vance </surname>  <given-names> Carroll P </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> carroll.vance@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" id="A15">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> randy.shoemaker@ars.usda.gov </email>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 8 </month>  <year> 2010 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Null genotypes were identified in gamma-irradiation experiments that knocked out the three lipoxygenase genes: LOX1, <span class="gene">LOX2</span> and LOX3, that expressed during <span class="trait">seed</span> development [ <xref ref-type="bibr" rid="B48"> 48 </xref> , <xref ref-type="bibr" rid="B49"> 49 </xref> ]
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Expression dynamics and genome distribution of osmoprotectants in soybean: identifying important components to face abiotic stress</h2>
			<h3>PMID:23369061</h3>
				

				
					<h3>PMCID:3548699</h3>
					<h4>期刊:BMC Bioinformatics</h4>
				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Kido </surname>  <given-names> Ederson A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> kido.ufpe@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Ferreira Neto </surname>  <given-names> Jos&#x000e9; RC </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> netocostaferreira@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Silva </surname>  <given-names> Roberta LO </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lane.roberta@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Belarmino </surname>  <given-names> Luis C </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> lucabellarmino@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Bezerra Neto </surname>  <given-names> Jo&#x000e3;o P </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> pacifico.joao@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Soares-Cavalcanti </surname>  <given-names> Nina M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ninamotasc@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Pandolfi </surname>  <given-names> Valesca </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> valescapandolfi@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A8">  <name>  <surname> Silva </surname>  <given-names> Manass&#x000e9;s D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> manassesdaniel@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A9">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre L </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> alexlimanepo@gmail.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A10">  <name>  <surname> Benko-Iseppon </surname>  <given-names> Ana M </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ana.iseppon@gmail.com </email>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The workflow applied resulted in a <span class="trait">set</span> including 1,996 <span class="trait">seed</span> sequences that allowed the identification of 36 differentially expressed genes related to the biosynthesis of osmoprotectants [Proline ( <italic> <span class="gene">P5CS</span> </italic> : 4, <italic> P5CR </italic> : 2), Trehalose ( <italic> TPS1 </italic> : 9, <italic> TPPB </italic> : 1), Glycine betaine ( <italic> BADH </italic> : 4) and <italic> Myo- </italic> inositol ( <italic> <span class="gene">MIPS</span> </italic> : 7, <italic> INPS1 </italic> : 8)], also mapped <italic> in silico </italic> in the soybean genome (25 loci)
					</p>
				
					<p>&nbsp;&nbsp;
						Other two tags <span class="gene">(GmDr_2643</span> and 55655) were induced only in the <span class="trait">drought</span> sensible accession BR-16 after stress regarding the same three transcripts, whereas one of them <span class="gene">(GmDr_2443)</span> was mapped in the 3'UTR and another in the CDS <span class="gene">(GmDr_55655)</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">seed</span> sequences OSMTL431, 432 and 434 were similar to P5CR polypeptides from two soybean loci <span class="gene">(Glyma19g31230</span> and <span class="gene">Glyma03g28480;</span> Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ), which transcripts were associated to SuperSAGE tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						Expression analysis showed both genes being expressed in all organs except <span class="trait">seed,</span> where only <span class="gene">MIPS2</span> transcript was detected
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Other two tags <span class="gene">(GmDr_2643</span> and 55655) were induced only in the <span class="trait">drought</span> sensible accession BR-16 after stress regarding the same three transcripts, whereas one of them <span class="gene">(GmDr_2443)</span> was mapped in the 3'UTR and another in the CDS <span class="gene">(GmDr_55655)</span> (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <span class="trait">seed</span> sequences OSMTL431, 432 and 434 were similar to P5CR polypeptides from two soybean loci <span class="gene">(Glyma19g31230</span> and <span class="gene">Glyma03g28480;</span> Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> ), which transcripts were associated to SuperSAGE tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Expression analysis showed both genes being expressed in all organs except <span class="trait">seed,</span> where only <span class="gene">MIPS2</span> transcript was detected.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Allele-specific marker development and selection efficiencies for both flavonoid 3&#x02032;-hydroxylase and flavonoid 3&#x02032;,5&#x02032;-hydroxylase genes in soybean subgenus <italic> soja </italic></h2>
			<h3>PMID:23463490</h3>
				

				
					<h3>PMCID:3664743</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <address>  <phone> +86-10-82105843 </phone>  <fax> +86-10-82105840 </fax>  <email> qiu_lijuan@263.net </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI), Institute of Crop Science, Chinese Academy of Agricultural Sciences, No.12 Zhongguancun South Street, Haidian District, 100081 Beijing, People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 2 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Further, <italic> T </italic> generally darkens hilum and/or <span class="trait">seed</span> coat color in combination with genotypes at the <italic> I </italic> , <italic> <span class="gene">W1</span> </italic> , <italic> R </italic> and <italic> O </italic> loci (Palmer et al
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies showed that the <italic> T </italic> locus was a gene encoding F3&#x02032;H and the <italic> <span class="gene">W1</span> </italic> locus co-segregated with a gene encoding F3&#x02032;5&#x02032;H in soybean, which belonged to the genetic loci controlling <span class="trait">seed</span> coat, flower and pubescence color
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic effect of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> alleles on pubescence color and <span class="trait">seed</span> coat was estimated (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						4&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;H</span> </italic> had a black <span class="trait">seed</span> coat, which was the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab4"> <label> Table&#x000a0;4 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and pubescence and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						9&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> had a black <span class="trait">seed</span> coat, which was also the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> allele
					</p>
				
					<p>&nbsp;&nbsp;
						The relationship between <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> b </italic> and <span class="trait">seed</span> coat color was not very close and the frequency of each color was similar
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab5"> <label> Table&#x000a0;5 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and flower and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						When the genetic effect of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> alleles were combined, the results showed that these loci had a high association with <span class="trait">seed</span> coat color (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> )
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab6"> <label> Table&#x000a0;6 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and <span class="trait">seed</span> coat color of soybean
					</p>
				
					<p>&nbsp;&nbsp;
						A polymorphism study of the <italic> <span class="gene">IFS1</span> </italic> and <italic> <span class="gene">IFS2</span> </italic> genes indicated that three SNPs in <italic> <span class="gene">IFS1</span> </italic> and two SNPs in <italic> <span class="gene">IFS2</span> </italic> were closely associated with all individual types and total <span class="trait">seed</span> isoflavone concentrations (Cheng et al
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies showed that the <italic> T </italic> locus was a gene encoding F3&#x02032;H and the <italic> <span class="gene">W1</span> </italic> locus co-segregated with a gene encoding F3&#x02032;5&#x02032;H in soybean, which belonged to the genetic loci controlling <span class="trait">seed</span> coat, flower and pubescence color.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic effect of <italic> <span class="gene">GmF3&#x02032;H</span> </italic> alleles on pubescence color and <span class="trait">seed</span> coat was estimated (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						4&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;H</span> </italic> had a black <span class="trait">seed</span> coat, which was the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab4"> <label> Table&#x000a0;4 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and pubescence and <span class="trait">seed</span> coat color of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						9&#x000a0;% accessions possessing <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> a </italic> had a black <span class="trait">seed</span> coat, which was also the most widespread <span class="trait">seed</span> coat color of the <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> allele.
					</p>
				
					<p>&nbsp;&nbsp;
						The relationship between <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> - <italic> b </italic> and <span class="trait">seed</span> coat color was not very close and the frequency of each color was similar.
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab5"> <label> Table&#x000a0;5 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and flower and <span class="trait">seed</span> coat color of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						When the genetic effect of the <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> alleles were combined, the results showed that these loci had a high association with <span class="trait">seed</span> coat color (Table&#x000a0; <xref rid="Tab6" ref-type="table"> 6 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<table-wrap id="Tab6"> <label> Table&#x000a0;6 </label> <caption> <p> Frequency of allelic variation in <italic> <span class="gene">GmF3&#x02032;H</span> </italic> and <italic> <span class="gene">GmF3&#x02032;5&#x02032;H</span> </italic> and <span class="trait">seed</span> coat color of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						A polymorphism study of the <italic> <span class="gene">IFS1</span> </italic> and <italic> <span class="gene">IFS2</span> </italic> genes indicated that three SNPs in <italic> <span class="gene">IFS1</span> </italic> and two SNPs in <italic> <span class="gene">IFS2</span> </italic> were closely associated with all individual types and total <span class="trait">seed</span> isoflavone concentrations (Cheng et al.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Development of SNP Genotyping Assays for <span class="trait">Seed</span> Composition Traits in Soybean</h2>
			<h3>PMID:28630621</h3>
				

				
					<h3>PMCID:5463095</h3>
					<h4>期刊:International Journal of Plant Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 30 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						During <span class="trait">seed</span> development stage, the two oleate desaturase genes, <span class="gene">FAD2-1A</span> <italic> <span class="gene">(Glyma10g42470)</span> </italic> and <span class="gene">FAD2-1B</span> <italic> <span class="gene">(Glyma20g24530)</span> </italic> , and the three linoleate desaturase genes, <span class="gene">FAD3A</span> <italic> <span class="gene">(Glyma14g37350)</span> </italic> , <span class="gene">FAD3B</span> <italic> <span class="gene">(Glyma02g39230)</span> </italic> , and <span class="gene">FAD3C</span> <italic> <span class="gene">(Glyma18g06950)</span> </italic> , are the major contributors controlling oleic and linolenic acid levels
					</p>
				
					<p>&nbsp;&nbsp;
						The combination of mutant alleles of <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> produces up to 80% oleic acid in soybean <span class="trait">seed</span> [ <xref rid="B35" ref-type="bibr"> 14 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Mutation in KTi gene <italic> <span class="gene">(Glyma08g45530)</span> </italic> prevents the accumulation of Kunitz trypsin inhibitor protein during <span class="trait">seed</span> development and improves tryptic activity [ <xref rid="B20" ref-type="bibr"> 9 </xref> ]
					</p>
				
					<p>&nbsp;&nbsp;
						Mature soybean <span class="trait">seed</span> consists of three distinct lipoxygenase isozymes, lipoxygenase 1 <italic> <span class="gene">(Glyma13g42320)</span> </italic> , lipoxygenase 2 <italic> <span class="gene">(Glyma13g42310)</span> </italic> , and lipoxygenase 3 <italic> <span class="gene">(Glyma13g42330)</span> </italic>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The combination of mutant alleles of <span class="gene">FAD2-1A</span> and <span class="gene">FAD2-1B</span> produces up to 80% oleic acid in soybean <span class="trait">seed</span> [ <xref rid="B35" ref-type="bibr"> 14 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Mutation in KTi gene <italic> <span class="gene">(Glyma08g45530)</span> </italic> prevents the accumulation of Kunitz trypsin inhibitor protein during <span class="trait">seed</span> development and improves tryptic activity [ <xref rid="B20" ref-type="bibr"> 9 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Mature soybean <span class="trait">seed</span> consists of three distinct lipoxygenase isozymes, lipoxygenase 1 <italic> <span class="gene">(Glyma13g42320)</span> </italic> , lipoxygenase 2 <italic> <span class="gene">(Glyma13g42310)</span> </italic> , and lipoxygenase 3 <italic> <span class="gene">(Glyma13g42330)</span> </italic>.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Inference of Transcription Regulatory Network in Low Phytic Acid Soybean Seeds</h2>
			<h3>PMID:29250090</h3>
				

				
					<h3>PMCID:5714895</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Redekar </surname>  <given-names> Neelam </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/340447/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pilot </surname>  <given-names> Guillaume </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/11124/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Raboy </surname>  <given-names> Victor </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/58055/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Song </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/45869/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saghai Maroof </surname>  <given-names> M. A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/458642/overview"/>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A dominant loss of function mutation in <italic> myo </italic> -inositol phosphate synthase ( <italic> <span class="gene">MIPS</span> </italic> ) gene and recessive loss of function mutations in two multidrug resistant protein type-ABC transporter genes not only reduce the <span class="trait">seed</span> phytic acid levels in soybean, but also affect the pathways associated with <span class="trait">seed</span> development, ultimately resulting in low emergence
					</p>
				
					<p>&nbsp;&nbsp;
						For example, a mutation in <italic> myo- </italic> inositol phosphate synthase ( <italic> <span class="gene">MIPS</span> </italic> ) gene results in reduced phytic acid, stachyose, raffinose, and elevated sucrose, and low <span class="trait">seed</span> emergence in soybean (Hitz et al
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						For example, a mutation in <italic> myo- </italic> inositol phosphate synthase ( <italic> <span class="gene">MIPS</span> </italic> ) gene results in reduced phytic acid, stachyose, raffinose, and elevated sucrose, and low <span class="trait">seed</span> emergence in soybean (Hitz et al.
					</p>
				
			

			<hr/> 
		
	</div>
</body>

</html>