<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>shoot weight, dry</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>shoot weight, dry</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Overexpression of <italic> <span class="gene">GmFDL19</span> </italic> enhances tolerance to <span class="trait">drought</span> and salt stresses in soybean</h2>
			<h3>PMID:28640834</h3>
				
					<h3>PMCID:5480881</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Yuanyuan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chen </surname>  <given-names> Quanzhen </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Nan </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiaoming </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Sijia </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Xiaohui </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Changhong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0003-3392-486X </contrib-id>  <name>  <surname> Cao </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 30 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The basic leucine zipper (bZIP) family of transcription factors plays an important role in the growth and developmental process as well as responds to various abiotic stresses, such as <span class="trait">drought</span> and high salinity. Our previous work identified <span class="gene">GmFDL19,</span> a bZIP transcription factor, as a flowering promoter in soybean, and the overexpression of <italic> <span class="gene">GmFDL19</span> </italic> caused early flowering in transgenic soybean plants. Here, we report that <italic> <span class="gene">GmFDL19</span> </italic> also enhances tolerance to <span class="trait">drought</span> and salt stress in soybean. <span class="gene">GmFDL19</span> was determined to be a group A member, and its transcription expression was highly induced by abscisic acid (ABA), polyethylene glycol (PEG 6000) and high salt stresses. Overexpression of <italic> <span class="gene">GmFDL19</span> </italic> in soybean enhanced <span class="trait">drought</span> and salt tolerance at the seedling stage. The relative plant height (RPH) and relative <span class="trait">shoot</span> <span class="trait">dry</span> weight (RSDW) of transgenic plants were significantly higher than those of the WT after PEG and salt treatments. In addition, the germination rate and plant height of the transgenic soybean were also significantly higher than that of WT plants after various salt treatments. Furthermore, we also found that <italic> <span class="gene">GmFDL19</span> </italic> could reduce the accumulation of Na <sup> + </sup> ion content and up-regulate the expression of several ABA/stress-responsive genes in transgenic soybean. We also found that <italic> <span class="gene">GmFDL19</span> </italic> overexpression increased the activities of several antioxidative enzyme and chlorophyll content but reduced malondialdehyde content. These results suggested that <italic> <span class="gene">GmFDL19</span> </italic> is involved in soybean abiotic stress responses and has potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The bZIP family of transcription factors plays an important role in developmental processes and responds to various abiotic stresses such as <span class="trait">drought</span> and high salinity stresses [ <xref rid="pone.0179554.ref017" ref-type="bibr"> 17 </xref> ]. Our previous work showed that overexpression of <italic> <span class="gene">GmFDL19</span> </italic> , a bZIP transcription factor, caused early flowering in soybean [ <xref rid="pone.0179554.ref018" ref-type="bibr"> 18 </xref> ]. To further analyze whether <italic> <span class="gene">GmFDL19</span> </italic> has a function in abiotic stress in soybean, we investigated the <span class="trait">drought</span> and salt stress tolerance in seedlings. In addition, we evaluated the expression pattern of <italic> <span class="gene">GmFDL19</span> </italic> in soybean seedlings under PEG, NaCl and ABA treatments. Furthermore, the regulation of Na <sup> + </sup> ion accumulation in transgenic soybean plants and the transcription of several ABA/stress-responsive genes in soybean that overexpressed <italic> <span class="gene">GmFDL19</span> </italic> were also investigated. Ultimately, the results of this study facilitate our understanding of <italic> <span class="gene">GmFDL19</span> </italic> in response to abiotic stress as well as the genetic engineering of stress-tolerant soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic soybean plants that overexpressed <italic> <span class="gene">GmFDL19</span> </italic> and wild type (WT) were used in this study. All plants were grown in soil and vermiculite at a 1:1 ratio in an artificial climate chamber under long day conditions (16 h light/ 8 h dark) at 25&#x000b0;C with an average light fluency of 200&#x02013;300 &#x003bc;mol m <sup> -2 </sup> s <sup> -1 </sup> . Light was regulated by Master TL5 lamps (Philips). When the first trifoliate leaves fully expanded (one-week-old), the seedling roots were immersed in various stress treatments, i.e. PEG-simulated <span class="trait">drought</span> (15% PEG 6000), salinity (150 mM NaCl) and ABA (100 &#x003bc;M ABA). The seedlings were incubated for 0, 1, 6 and 12 h, and the first trifoliate leaves were collected at each time point. Three biological replicates were performed for each stress treatment. All samples were immediately frozen in liquid nitrogen and stored at &#x02212;80&#x000b0;C until analyzed. Total RNA was isolated and reverse transcribed to obtain cDNA, which was used as a template for real-time quantitative RT-PCR.
					</p>
				
					<p>&nbsp;&nbsp;
						Since <italic> <span class="gene">GmFDL19</span> </italic> may be a member of ABF/AREB and its expression was induced by PEG treatment, the function of transgenic soybean plants overexpressing <italic> <span class="gene">GmFDL19</span> </italic> were investigated under PEG treatment. The phenotype of soybean plants that overexpress <italic> <span class="gene">GmFDL19</span> </italic> was examined under PEG treatment. As shown in <xref ref-type="fig" rid="pone.0179554.g002"> Fig 2A </xref> , no significant difference in phenotype was observed for the transgenic plants when compared with the wild type (WT) plants under normal conditions. The one-week-old soybean seedlings were treated with 15% PEG 6000 for 4 weeks. During this time, the <span class="trait">shoot</span> of transgenic plants grew stronger and higher, whereas the leaves of WT plants curled and exhibited signs of dwarfism ( <xref ref-type="fig" rid="pone.0179554.g002"> Fig 2B </xref> ). Moreover, the RPH and RSDW of transgenic plants were significantly higher than those of the WT ( <xref ref-type="fig" rid="pone.0179554.g002"> Fig 2C and 2D </xref> ). Similar results were observed under natural <span class="trait">dry</span> treatment ( <xref ref-type="supplementary-material" rid="pone.0179554.s001"> S1 Fig </xref> ). These results suggested that <italic> <span class="gene">GmFDL19</span> </italic> overexpression could improve the response to <span class="trait">drought</span> stress in transgenic soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						To further test the salt-tolerant phenotype of soybean plants that overexpress <italic> <span class="gene">GmFDL19</span> </italic> , the one-week-old soybean seedlings were treated with 150 mM NaCl for 4 weeks. The WT plants displayed <span class="trait">chlorosis</span> after 150 mM NaCl treatment. Additionally, a general growth inhibition was observed, specifically less plant height and <span class="trait">dry</span> mass, when compared with the controls ( <xref ref-type="fig" rid="pone.0179554.g004"> Fig 4 </xref> ). Although the inhibitory effect was also observed in transgenic plants, the transgenic plants had a significantly higher RPH and RSDW than the WT ( <xref ref-type="fig" rid="pone.0179554.g004"> Fig 4 </xref> ). The RPH and RSDW of the transgenic plants were 0.78 and 0.79, respectively, whereas the RPH and RSDW of WT were 0.61 and 0.59, respectively ( <xref ref-type="fig" rid="pone.0179554.g004"> Fig 4C and 4D </xref> ). In addition, the WT plants died after 3 weeks with salt treatment at 200 mM NaCl, whereas transgenic plants still survived well with less inhibition ( <xref ref-type="supplementary-material" rid="pone.0179554.s002"> S2 Fig </xref> ). These results further confirmed that <italic> <span class="gene">GmFDL19</span> </italic> overexpression could improve the salt tolerance in transgenic soybean seedlings.
					</p>
				
					<p>&nbsp;&nbsp;
						Since the <italic> <span class="gene">GmFDL19</span> </italic> gene could improve salinity tolerance response in transgenic plants, <italic> <span class="gene">GmFDL19</span> </italic> may also have a role in regulating the Na <sup> + </sup> absorption. Under control treatment (CK), both transgenic plants and WT plants taken up less than 1.0 mg Na <sup> + </sup> per g of <span class="trait">dry</span> weight (DW) in shoots and roots ( <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5A </xref> ). Under NaCl treatment, Na <sup> + </sup> content in shoots and roots of transgenic and WT plants were increased above 20 mg Na <sup> + </sup> per g of DW. However, <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5A </xref> showed significantly lower Na <sup> + </sup> content in <span class="trait">shoot</span> and root of transgenic plants than that of the WT plants under NaCl treatment. WT plants took up 28.3 and 34.7 mg Na <sup> + </sup> per g of DW in shoots and roots respectively, whereas, the transgenic plants took up 25.6 and 31.2 mg Na <sup> + </sup> per g of DW in shoots and roots respectively under NaCl treatment ( <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5A </xref> ). The transgenic plants and WT plants had a higher K <sup> + </sup> contents in control than that in salt stress conditions ( <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5B </xref> ). This were consistent with the result that both transgenic and WT plants had a low plant height under salt stress ( <xref ref-type="fig" rid="pone.0179554.g004"> Fig 4 </xref> ). No significant differences in K <sup> + </sup> content were observed between the WT and transgenic plants ( <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5B </xref> ). These results indicated that <italic> <span class="gene">GmFDL19</span> </italic> plays a role in regulating absorption of Na <sup> + </sup> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">GmFDL19</span> </italic> gene might have a role in improving stress tolerance through the regulation of downstream genes. As shown in <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6A </xref> , the transcript abundance of five reported salt-responsive genes, such as <italic> <span class="gene">GmCHX1</span> </italic> [ <xref rid="pone.0179554.ref020" ref-type="bibr"> 20 </xref> , <xref rid="pone.0179554.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0179554.ref024" ref-type="bibr"> 24 </xref> ], <italic> <span class="gene">GmSOS1</span> </italic> [ <xref rid="pone.0179554.ref025" ref-type="bibr"> 25 </xref> ], <italic> <span class="gene">GmPIP1;6</span> </italic> [ <xref rid="pone.0179554.ref026" ref-type="bibr"> 26 </xref> ], <italic> <span class="gene">GmNHX1</span> </italic> [ <xref rid="pone.0179554.ref027" ref-type="bibr"> 27 </xref> ] and <italic> <span class="gene">GmNKT1;4</span> </italic> [ <xref rid="pone.0179554.ref028" ref-type="bibr"> 28 </xref> ], increased significantly in the transgenic soybean plants compared with that in WT under salt stress conditions. In addition, seven TF genes, including <italic> <span class="gene">GmbZIP1</span> </italic> , <italic> <span class="gene">GmNAC11</span> </italic> , <italic> <span class="gene">GmNAC29</span> </italic> , <italic> <span class="gene">GmDERB2A;2</span> </italic> , <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmERF5</span> and <span class="gene">GmMYB174</span> </italic> , which have been involved in salt stress responses in soybean [ <xref rid="pone.0179554.ref029" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> ], also increased significantly in the transgenic soybean plants compared with that in WT after the 150 mM NaCl treatment ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6A </xref> ). Similar to the results observed under the salt stress condition, the expression of all seven TF genes in transgenic plants was higher than that in WT plants under PEG treatment ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6B </xref> ). The relative expression of <italic> <span class="gene">GmUBC2</span> </italic> , which might improve <span class="trait">drought</span> and salt tolerance in soybean [ <xref rid="pone.0179554.ref034" ref-type="bibr"> 34 </xref> ], was similar in WT and transgenic plants after the 150mM NaCl treatment ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6A </xref> ), whereas, the expression of <italic> <span class="gene">GmUBC2</span> </italic> was lower in WT than in transgenic plants after the PEG treatment ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6B </xref> ). These results indicated that <italic> <span class="gene">GmFDL19</span> </italic> regulated a common set of ABA and stress responsive genes in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In maize [ <xref rid="pone.0179554.ref037" ref-type="bibr"> 37 </xref> , <xref rid="pone.0179554.ref038" ref-type="bibr"> 38 </xref> ], wheat [ <xref rid="pone.0179554.ref039" ref-type="bibr"> 39 </xref> ] and rice [ <xref rid="pone.0179554.ref040" ref-type="bibr"> 40 </xref> ], studies have shown that overexpressing the stress-responsive <italic> bZIP </italic> genes can improve <span class="trait">drought</span> and salinity tolerance in transgenic plant. In soybean, the <italic> <span class="gene">GmbZIP</span> </italic> genes, such as <italic> <span class="gene">GmbZIP44</span> </italic> , <italic> <span class="gene">GmbZIP62</span> </italic> , <italic> <span class="gene">GmbZIP78</span> </italic> , <italic> <span class="gene">GmbZIP132</span> </italic> , <italic> <span class="gene">GmbZIP110</span> </italic> and <italic> <span class="gene">GmbZIP1</span> </italic> , were responsive to multiple stress treatment [ <xref rid="pone.0179554.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0179554.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> , <xref rid="pone.0179554.ref041" ref-type="bibr"> 41 </xref> ]. In the present study, the results showed that the <span class="gene">GmFDL19,</span> which has a high amino acid sequence similar to the <italic> Arabidopsis </italic> ABF/AREB subfamily members, was determined to be a group A member ( <xref ref-type="fig" rid="pone.0179554.g001"> Fig 1A </xref> ). In <italic> Arabidopsis </italic> , the ABF/AREB subfamily genes were involved in ABA and stress signaling pathways [ <xref rid="pone.0179554.ref017" ref-type="bibr"> 17 </xref> , <xref rid="pone.0179554.ref022" ref-type="bibr"> 22 </xref> ]. Here, we also found that the expression of <italic> <span class="gene">GmFDL19</span> </italic> was induced by multiple stresses, including PEG, high salt and ABA ( <xref ref-type="fig" rid="pone.0179554.g001"> Fig 1B </xref> ). These results were consistent with the expression of <italic> <span class="gene">GmbZIP1</span> </italic> , which is also a group A member in soybean and highly induced by ABA, <span class="trait">drought,</span> high salt, and low temperature [ <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> ]. These results indicated that the <italic> <span class="gene">GmFDL19</span> </italic> , a novel group A bZIP gene, might play a role in abiotic stress tolerance in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						The transgenic soybean plants overexpressing <italic> <span class="gene">GmFDL19</span> </italic> showed a greater tolerance to PEG and salt treatment compared to WT plants (Figs <xref ref-type="fig" rid="pone.0179554.g002"> 2 </xref> &#x02013; <xref ref-type="fig" rid="pone.0179554.g004"> 4 </xref> ). Similar phenotypic parameters were observed under PEG and salt treatment using transgenic line overexpressing <italic> <span class="gene">GmFDL06</span> </italic> , another member of ABF/AREB family [ <xref rid="pone.0179554.ref042" ref-type="bibr"> 42 </xref> ]. Our data showed that both <italic> <span class="gene">GmFDL19</span> </italic> overexpression line and <italic> <span class="gene">GmFDL06</span> </italic> overexpression line showed an improvement of salt and <span class="trait">drought</span> tolerance compared with the wild type [ <xref rid="pone.0179554.ref042" ref-type="bibr"> 42 </xref> , Figs <xref ref-type="fig" rid="pone.0179554.g002"> 2 </xref> &#x02013; <xref ref-type="fig" rid="pone.0179554.g004"> 4 </xref> ]. Thus, we thought that the enhancement of tolerance to <span class="trait">drought</span> and salt stresses in transgenic soybean overexpressing <italic> <span class="gene">GmFDL19</span> </italic> , was due to the <italic> <span class="gene">GmFDL19</span> </italic> overexpression, not other genes in soybean genome. Moreover, these results were consistent with the description that the overexpression of <italic> <span class="gene">GmbZIP1</span> </italic> improved tolerance to high salt and <span class="trait">drought</span> in transgenic plants [ <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> ]. In addition, previous studies have found that the <italic> Arabidopsis </italic> transgenic plants that overexpress three non-AREB subfamily bZIP genes ( <italic> <span class="gene">GmbZIP44</span> </italic> , <italic> <span class="gene">GmbZIP62</span> </italic> or <italic> <span class="gene">GmbZIP78</span> </italic> ) exhibited enhanced salt tolerance [ <xref rid="pone.0179554.ref015" ref-type="bibr"> 15 </xref> ]. Recently, <italic> <span class="gene">GmbZIP110</span> </italic> that is also induced by salt stress, has been identified in soybean, and an enhanced salt tolerance of composite seedling and transgenic <italic> Arabidopsis </italic> was observed [ <xref rid="pone.0179554.ref016" ref-type="bibr"> 16 </xref> ]. The transgenic <italic> Arabidopsis </italic> that overexpress <italic> <span class="gene">GmbZIP110</span> </italic> took up significantly less Na <sup> + </sup> ion, while no significant change in K <sup> + </sup> ion uptake was measured [ <xref rid="pone.0179554.ref016" ref-type="bibr"> 16 </xref> ]. Our results showed that the transgenic soybean overexpressing <italic> <span class="gene">GmFDL19</span> </italic> also have a significantly lower Na <sup> + </sup> ion content, with no significant change in K <sup> + </sup> and Cl <sup> - </sup> content when compared to WT plants ( <xref ref-type="fig" rid="pone.0179554.g005"> Fig 5 </xref> ). Thus, <italic> <span class="gene">GmFDL19</span> </italic> might function in regulating absorption of Na <sup> + </sup> , and in turn, might improve salt tolerance in soybean. This suggests that different bZIP subfamilies in soybean may play similar roles in resisting abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">GmbZIP</span> </italic> genes can up-regulate the expression levels of a number of ABA/stress-responsive genes in <italic> Arabidopsis </italic> [ <xref rid="pone.0179554.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0179554.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> ]. However, the function of <italic> <span class="gene">GmbZIP</span> </italic> genes on the regulation of target genes remains unclear in soybean. Our findings showed that the overexpression of <italic> <span class="gene">GmFDL19</span> </italic> up-regulated the expression of salt-related genes and some TF genes in transgenic soybean plants after salt and PEG treatments ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6 </xref> ). The <italic> <span class="gene">GmCHX1</span> </italic> gene plays an important role in Na <sup> + </sup> transportation and salt tolerance in soybean [ <xref rid="pone.0179554.ref020" ref-type="bibr"> 20 </xref> , <xref rid="pone.0179554.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0179554.ref024" ref-type="bibr"> 24 </xref> ]. The overexpression of <italic> <span class="gene">GmSOS1</span> </italic> , <italic> <span class="gene">GmPIP1;6</span> </italic> , <italic> <span class="gene">GmNHX1</span> </italic> and <italic> <span class="gene">GmHKT1;4</span> </italic> reduced Na <sup> + </sup> uptake and improved salt tolerance in transgenic plants [ <xref rid="pone.0179554.ref025" ref-type="bibr"> 25 </xref> &#x02013; <xref rid="pone.0179554.ref028" ref-type="bibr"> 28 </xref> ]. In order to dissect the reduction of Na <sup> + </sup> uptake and enhancement of salt tolerance at the molecular level, the expression of these salt-related genes were analyzed. The results showed that the salt-related genes, including <italic> <span class="gene">GmCHX1</span> </italic> , <italic> <span class="gene">GmSOS1</span> </italic> , <italic> <span class="gene">GmPIP1;6</span> </italic> , <italic> <span class="gene">GmNHX1</span> </italic> , and <italic> <span class="gene">GmNKT1;4</span> </italic> , increased in transgenic soybean post salt treatment ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6A </xref> ). In addition, overexpression of <italic> <span class="gene">GmUBC2</span> </italic> enhanced <span class="trait">drought</span> and salt tolerance of <italic> Arabidopsis </italic> transgenic plants through modulating expression of abiotic stress-responsive genes [ <xref rid="pone.0179554.ref034" ref-type="bibr"> 34 </xref> ]. Furthermore, some TF genes, including <italic> <span class="gene">GmbZIP1</span> </italic> , <italic> <span class="gene">GmNAC11</span> </italic> , <italic> <span class="gene">GmNAC29</span> </italic> , <italic> <span class="gene">GmDERB2A;2</span> </italic> , <italic> <span class="gene">GmWRKY27</span> </italic> , <italic> <span class="gene">GmERF5</span> and <span class="gene">GmMYB174</span> </italic> , are involved in plant abiotic stress responses in ABA-dependent pathways [ <xref rid="pone.0179554.ref029" ref-type="bibr"> 29 </xref> &#x02013; <xref rid="pone.0179554.ref033" ref-type="bibr"> 33 </xref> ]. The expression of these TF genes was also analyzed, and we found that these genes also increased in transgenic soybean plants that overexpress <italic> <span class="gene">GmFDL19</span> </italic> ( <xref ref-type="fig" rid="pone.0179554.g006"> Fig 6 </xref> ). These results indicated that <italic> <span class="gene">GmFDL19</span> </italic> might be involved in ABA dependent pathways and is also related to ion transport similar to <italic> Arabidopsis </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						In plants, salt stress can overproduced ROS, such as H <sub> 2 </sub> O <sub> 2 </sub> and MDA, which could cause many adverse effects to the plant cell [ <xref rid="pone.0179554.ref035" ref-type="bibr"> 35 </xref> , <xref rid="pone.0179554.ref043" ref-type="bibr"> 43 </xref> ]. In soybean, one possible mechanism of salt tolerance is to enhance the contents and activities of antioxidative enzyme to restore the oxidative balance and minimize the cellular damage by secondary oxidative stress [ <xref rid="pone.0179554.ref007" ref-type="bibr"> 7 </xref> ]. In soybean, the purple acid phosphatase 3 <span class="gene">(GmPAP3),</span> encoding a purple acid phosphatase that localizes in mitochondria, was shown to be induced by salinity, osmotic and oxidative stresses [ <xref rid="pone.0179554.ref044" ref-type="bibr"> 44 </xref> , <xref rid="pone.0179554.ref045" ref-type="bibr"> 45 </xref> ]. Ectopic expression of <italic> <span class="gene">GmPAP3</span> </italic> could alleviate the oxidative damage caused by salinity and osmotic stresses [ <xref rid="pone.0179554.ref044" ref-type="bibr"> 44 </xref> ]. Silencing two <italic> <span class="gene">GmFNSII</span> </italic> genes in soybean plants reduced the production of flavones aglycones, resulting in more MDA and H <sub> 2 </sub> O <sub> 2 </sub> accumulation, and then hypersensitivity to salt stress as compared with control plants [ <xref rid="pone.0179554.ref046" ref-type="bibr"> 46 </xref> ]. Recent research showed that <span class="gene">GmWRKY27</span> improves salt and <span class="trait">drought</span> tolerance in soybean; partly via reduce ROS level [ <xref rid="pone.0179554.ref029" ref-type="bibr"> 29 </xref> ]. <span class="gene">GmWRKY27</span> interacts with <span class="gene">GmMYB174,</span> and the two TFs cooperatively inhibit transcription of <italic> <span class="gene">GmNAC29</span> </italic> , whose product may be involved in ROS production by directly increasing the expression of <italic> <span class="gene">GmSPOD1</span> </italic> , leads to reduced intracellular ROS levels [ <xref rid="pone.0179554.ref029" ref-type="bibr"> 29 </xref> ]. Our data showed that the transgenic soybean plants overexpressing <italic> <span class="gene">GmFDL19</span> </italic> displayed lower level MDA and higher level of antioxidant enzymes activities and chlorophyll contents ( <xref ref-type="fig" rid="pone.0179554.g007"> Fig 7 </xref> ), which is consistent with the salt tolerance phenotype ( <xref ref-type="fig" rid="pone.0179554.g004"> Fig 4 </xref> ). Our results suggested that <italic> <span class="gene">GmFDL19</span> </italic> may reduce the ROS and then increase salt tolerance in soybean. Further studies are needed to better understand how <italic> <span class="gene">GmFDL19</span> </italic> reduces the ROS in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, overexpression of <italic> <span class="gene">GmFDL19</span> </italic> improved the <span class="trait">drought</span> and salt tolerance in soybean. <span class="gene">GmFDL19</span> might directly bind to ACGT core elements and up-regulate several ABA/stress-responsive genes. In addition, overexpression of <italic> <span class="gene">GmFDL19</span> </italic> regulated the absorption of Na <sup> + </sup> but not K <sup> + </sup> or Cl <sup> - </sup> , in transgenic plants. Our data suggests that <italic> <span class="gene">GmFDL19</span> </italic> is involved in soybean abiotic stress responses and has a potential utilization to improve multiple stress tolerance in transgenic soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Relationship between cellulose synthesis metabolism and <span class="trait">lodging</span> <span class="trait">resistance</span> in intercropping soybean at seedling stage].</h2>
			<h3>PMID:27396119  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Deng YC, Liu WG, Yuan XQ, Yuan J, Zou JL, Du JB, Yang WY.
				<br/> 
				年份：2016 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Physical characteristics of stem are closely relative to the crop <span class="trait">lodging.</span> Increase of stem strength is conducive to resolve the problem of <span class="trait">lodging.</span> Three soybean cultivars with different shade tolerance were planted under maize-soybean intercropping and soybean monocropping, respectively. Physiological and biochemical indices including cellulose, soluble sugar, sucrose, starch contents and enzyme activity were investigated to assess the snapping <span class="trait">resistance</span> and <span class="trait">lodging</span> <span class="trait">resistance</span> of the stems of soybean seedling, and snapping- and lodging-resistance indices were calculated for further verification. Furthermore, relationship analyses between these factors and the <span class="trait">lodging</span> of inter-cropped soybean showed that the intercropping soybean lodged seriously, the snapping <span class="trait">resistance,</span> <span class="trait">lodging</span> <span class="trait">resistance</span> index, contents of cellulose, soluble sugar, sucrose, starch and activities of the related enzymes were significantly lower than monocropping soybean at seedling stage. The three soybean cultivars showed different phenotypes in intercropping condition. The snapping-resistant Nandou12 with strong shade-tolerant traits was the most lodging-resistant phenotype, and it also harbored high contents of cellulose, soluble sugar, sucrose, starch and active enzymes. The <span class="trait">lodging</span> <span class="trait">resistance</span> index, cellulose content of the stems of intercropped soybean seedling were significantly positively correlated with the snapping <span class="trait">resistance,</span> and were significantly negatively correlated with the actual <span class="trait">lodging</span> percentage. The activities of sucrose phosphate synthase (SPS) , sucrose synthase (SS) and neutral invertase (NI) were positively correlated with sucrose is content, but not the acid invertase (AI). The activities of SPS, NI and <span class="gene">SS</span> were positively correlated with cellulose content, but not Al. In a word, the high activities of SPS and <span class="gene">SS</span> in the soybean stem were the enzymatic basis to maintain relatively higher cellulose and sucrose content, which is conducive to improve the stem-sfrength and enhance the <span class="trait">lodging</span> <span class="trait">resistance</span> ability in intercropping condition. Effects of different light conditions on cellulose metabolic mechanism of soybean seedling stems, <span class="trait">lodging</span> resistant characteristics of soybean seedlings studied in the corn-soybean intercropping system provided a basis for screening more shade-tolerant soybean variety.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biochar Treatment Resulted in a Combined Effect on Soybean Growth Promotion and a Shift in Plant Growth Promoting Rhizobacteria</h2>
			<h3>PMID:26941730</h3>
				
					<h3>PMCID:4766286</h3>
					<h4>期刊:Frontiers in Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Egamberdieva </surname>  <given-names> Dilfuza </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/240452/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wirth </surname>  <given-names> Stephan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/308449/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Behrendt </surname>  <given-names> Undine </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/301753/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abd_Allah </surname>  <given-names> Elsayed F. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/192675/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Berg </surname>  <given-names> Gabriele </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/27010/overview"/>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 2 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To test whether bacterial isolates were capable of stimulating plant growth, a pot experiment was conducted in the greenhouse using loamy sand. The seeds were surface-sterilized and inoculated with bacterial strains as described above. The sterility of the seeds was previously tested on <span class="gene">TSA</span> agar by incubating the plates for 3 days at 28&#x000b0;C. No contaminants were found, indicating that the surface-sterilization was effective. Two hundred bacterial strains isolated from the rhizosphere of soybeans were grown overnight in Tryptic Soy Broth (TSB), and one milliliter of each culture was pelleted by centrifugation (10,000 &#x000d7; g for 10 min); the supernatant was discarded. Non-inoculated plants were used as negative controls. Cell pellets were washed with 1 ml of PBS and re-suspended in PBS. Cell suspensions corresponded to a cell density of 10 <sup> 7 </sup> cells/ml. Germinated seeds were placed in the bacterial suspension with sterile forceps and shaken gently. After approximately 10 min, the inoculated seeds were aseptically planted into the potting soil. Three seeds were sown per plastic pot (12 cm diameter; 10 cm deep) to a depth of approximately 1.5 cm. After germination, plants were thinned to one per pot, and pots were set-up in a randomized design with six replications. The plants were grown under greenhouse conditions (day temperature 24&#x000b0;C/night 16&#x000b0;C; humidity 50&#x02013;60%, day length 12 h) for 1 month. At harvest, the plants were removed from the pots, and the <span class="trait">dry</span> weights of roots and shoots were determined. A total 32 strains from control soil and 43 strains from HTC char amended soil were selected based on their plant growth promoting abilities and were further identified and characterized.
					</p>
				
					<p>&nbsp;&nbsp;
						The identification of isolated strains was performed using whole-cell matrix assisted laser desorption/ionization (MALDI)&#x02013;time of flight (TOF) mass spectrometry. Sample preparation was carried out according to the ethanol/formic acid extraction protocol recommended by Bruker Daltonics (Bremen, Germany). The isolates were cultured on tryptic soy agar (TSA, Difco Laboratories, Detroit, Michigan, USA) for 24 h, and approximately 10 mg of cell mass was suspended in 300 &#x003bc;L of water and vortexed to generate a homogenous suspension. The suspension was mixed with 900 &#x003bc;L of ethanol and centrifuged. The pellet was resuspended in 50 &#x003bc;L of 70% formic acid and subsequently carefully mixed with 50 &#x003bc;L of acetonitrile. After centrifugation, aliquots of 1 &#x003bc;L of supernatant were placed immediately on spots of a MALDI target. Each spot was allowed to <span class="trait">dry</span> and subsequently overlaid with 1 &#x003bc;L of matrix (&#x003b1;-ciano-4-hydroxycinnamic acid in 50% aqueous acetonitrile containing 2.5% trifluoroacetic acid). Mass spectra were acquired using a MALDI-TOF <span class="gene">MS</span> spectrometer in a linear positive mode (Microflex&#x02122;LT, Bruker Daltonics, Bermen, Germany) in a mass range of 2&#x02013;20 kDa. A bacterial test standard (BTS, Bruker Daltonics, Bremen, Germany) was used for instrument calibration. The raw spectra were imported into MALDI Biotyper&#x02122; software (Bruker Daltonics, Germany) and then processed and analyzed using standard pattern matching against the reference spectra in the MALDI Biotyper&#x02122; reference database (version 3.0, Bruker Daltonics, Germany).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Beneficial rhizobacteria immobilized in nanofibers for potential application as soybean seed bioinoculants</h2>
			<h3>PMID:28472087</h3>
				
					<h3>PMCID:5417607</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> De Gregorio </surname>  <given-names> Priscilla Romina </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Michavila </surname>  <given-names> Gabriela </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ricciardi Muller </surname>  <given-names> Lenise </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Souza Borges </surname>  <given-names> Clarissa </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pomares </surname>  <given-names> Mar&#x000ed;a Fernanda </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saccol de S&#x000e1; </surname>  <given-names> Enilson Luiz </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pereira </surname>  <given-names> Claudio </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vincent </surname>  <given-names> Paula Andrea </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<sup> e </sup> <italic> Vigor index </italic> = ( <italic> mean root length </italic> + <italic> mean <span class="trait">shoot</span> length </italic> ) &#x000d7; <italic> <span class="gene">GR</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Although the use of <italic> Burkholderia </italic> spp. strains such as PGPR is limited by the risk they can cause to human health; this is mainly attributable to the <italic> B </italic> . <italic> cepacia </italic> complex species [ <xref rid="pone.0176930.ref062" ref-type="bibr"> 62 </xref> , <xref rid="pone.0176930.ref063" ref-type="bibr"> 63 </xref> ]. Frickmann <italic> et al </italic> . [ <xref rid="pone.0176930.ref064" ref-type="bibr"> 64 </xref> ] and Pan <italic> et al </italic> . [ <xref rid="pone.0176930.ref065" ref-type="bibr"> 65 </xref> ] carried out phylogenetic studies using <italic> <span class="gene">rpsU</span> </italic> and 16S rRNA gene sequences, respectively, and demonstrated that <italic> B </italic> . <italic> caribensis </italic> specie strains are not closely related to the <italic> B </italic> . <italic> cepacia </italic> complex species. Moreover, Paungfoo-Lonhienne <italic> et al </italic> . [ <xref rid="pone.0176930.ref063" ref-type="bibr"> 63 </xref> ] reported <italic> B </italic> . <italic> caribensis </italic> strains, such as plant-beneficial-environmental <italic> Burkholderia </italic> . Taking the above references into account, it was concluded that the use of the <italic> B </italic> . <italic> caribesis </italic> species, such as a PGPR, could be biosafe. On the other hand, several works have also reported the potential of <italic> B </italic> . <italic> caribensis </italic> strains as PGPR [ <xref rid="pone.0176930.ref043" ref-type="bibr"> 43 </xref> , <xref rid="pone.0176930.ref045" ref-type="bibr"> 45 </xref> , <xref rid="pone.0176930.ref066" ref-type="bibr"> 66 </xref> , <xref rid="pone.0176930.ref067" ref-type="bibr"> 67 </xref> ]. Chen <italic> et al </italic> . [ <xref rid="pone.0176930.ref066" ref-type="bibr"> 66 </xref> ] identified numerous strains of <italic> B </italic> . <italic> caribensis </italic> , such as nitrogen-fixing legume symbionts. Similarly, Roy <italic> et al </italic> . [ <xref rid="pone.0176930.ref067" ref-type="bibr"> 67 </xref> ] and Parra-Cota <italic> et al </italic> . [ <xref rid="pone.0176930.ref045" ref-type="bibr"> 45 </xref> ] demonstrated that <italic> B </italic> . <italic> caribensis </italic> strains promote growth and increase <span class="trait">yield</span> in rice and grain amaranth, respectively, by improving plant nitrogen uptake. In the present work, <italic> B </italic> . <italic> caribensis </italic> ISIB40 was also able to fix nitrogen ( <xref ref-type="fig" rid="pone.0176930.g001"> Fig 1D </xref> ). Thus, this ability of <italic> B </italic> . <italic> caribensis </italic> ISIB40 could be suggested as one of the possible mechanisms that induced higher leaf number and <span class="trait">dry</span> weight of shoots in the plants from the ISIB40-PVA and ISIB40-treated seeds. However, further studies will be necessary to confirm this hypothesis.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>